"
Page
Discussion
View source
History
teams
Log in
 
Team:NUDT CHINA/Notebook
From 2014.igem.org
The Transformation of Our iGEM Ideas
Originally, we were attracted by the Shortest Path Problem, which aims to find the shortest path in a directed graph that can link the beginning node and ending node. Be inspired, we decided to design a cascade pathway in vivo which can help us read out the answer of Shortest Path Problem directly by the life stage of E. coli.
The main problem in the design are:
how to describe a route in E. coli;
how to build different routes in E. coli;
how to compare the length of different pathways.
Home
Team
Project
Parts
Modeling
Notebook
Safety Policy & Practices
Attributions
To solve the first problem, we built devices that contains promoter, RBS, target gene (CDS) and terminator to simulate the nodes and lines in directed graph: the promoters are nodes and the target fragment are lines. When the promoter is activated, the target gene can coding protein which then activate the next promoter as regulatory element. This regulatory process can describe how a man the walking in a directed graph: he departed from the node P1, went through the line L and finally arrived at the spot P2. In this way, theoretically, we can describe any given directed graph in E. coli. (For further details about the coding of the graph, please refer to the project profile of our wiki) (Fig. 1)
Fig. 1 One promoter and one target gene can simulate one node and one line
After building one complete graph in E. coli, different route should be labelled and tested independently so that we can get the length indirectly by the time of whole travel from beginning to ending. Then we can selected the quickest one as the shortest route. However, we are stuck in the second problem: how to build different route.
First design:
In each regulatory unit, three different restrict enzyme sites are put between the promoter and RBS, target gene and terminator, terminator and next promoter, respectively. (Fig. 2)
Fig. 2 the position of three different restrict enzyme sites
In this way, we can cut the RBS and target gene out which means this route does not have line L (we link the 1, 2 (or 1, 3) to keep the circle structure of plasmid). Then we can test this route by the time of arriving at the ending node which reported by the report gene (we used the GFP).
Second stage:
But we found that it was not easy get so many different good restrict enzyme. So we change the design slightly. (Fig. 3)
Fig. 3 the position of two same restrict enzyme sites
In this way, we can still finish the function of first stage but save my restrict enzymes.
Third stage:
The last problem is how to compare the length of different pathway. Firstly, we decide to build all the possible route and test them. That is to say, we need to cut out 1, 2, 3,…,device(s) so that we can test the entire route, which is named exhaustion, quite time-consuming. Later, we found a better way where we can only cut one device which means we only need to build n graph if the graph has n lines.
Provided that we have 5 lines in a graph like Fig. 4.
Fig. 4 a graph of 5 lines, where node 1 is the beginning node 4 is the ending
According our method to find the shortest path, the first step is to build 6 different systems. Then we tested the “finished time” of each systems. Here is the systems and their “finish time”.
System
Array
Time (Theoretically)
11111
2 units
01111
2 units
10111
2 units
11011
2 units
11101
3 units
11110
unreachable
Noticed that without line L5, the ending is inaccessible which implicates the L5 is indispensable. Though it does not block the accessibility, the absence of L4 slows down the arrival, which implies that L4 is on the shortest path. Thus, from the date of the chart above, we can pick out the shortest path is:
Protocol
Here are the recapitulative protocols of the experiments that we have used.
Polymerase Chain Reaction (PCR)
Most of our PCRs was performed with the Recombinant Taq DNA Polymerase TaKaRa Taq(TM) or PrimeSTAR(R) HS DNA Polymerase and dNTP mixture (2.5mM each) under the standard 3-step PCR protocol. Touch-down RCR procedures were also used for several times to inhibit the nonspecific amplification. (Refer to the instruction for more details).
Plasmid Preparation
We used the TIANprep Mini Plasmid Kit (TIANGEN) to perform all of our plasmid preparation on bacteria with target plasmid. (Refer to the TIANGEN instruction for more details).
Gel Electrophoresis & Gel Extraction
After the gel electrophoresis (normal experiment), we used the TIANgel Midi Plasmid Kit (TIANGEN) to extract all our target fragment in gel. (Refer to the TIANGEN instruction for more details).
Endonuclease Digestion & Ligases
The enzymes we had used were EcoR I, Spe I, Xba I, Pst I, Cla I, Sal I, Kpn I (digestion), T4 DNA ligase (ligase) all of them were bought from TaKaRaTM. The digestion and ligase systems were built following the instructions of TaKaRa(TM).
Fluorescent Fusion Protein Gene Expression
Fluorometric measurement were processed with Fluoroskan(R) Ascent FL from ThermoTM using the filters of 485nm and 538nm. The E.coli to be tested was cultured in liquid LB media containing 35μg/mL chloramphenicol to OD0.6, than split into 96 well plates (from ThermoTM) by 150μL/well. The plate was then put into the Fluoroskan(R) Ascent FL and cultured in 25℃ and background shake of 90rpms. The Fluorometric measurement was processed with the interval time of 10 minutes.
Blue print
According our design of the whole cascade regulatory pathway as Fig. 1 showed, we can solve the Shortest Path Problem. However, building and standardizing these devices and systems means drawing the blue print.
Systems
Functional fragments at plasmid
System “111”
System “011”
System “101”
System “110”
Fig. 1 regulatory pathways
Thus, we drew a picture of our experiment procedure. (Fig. 2)
Fig. 2 experiment procedure
Week Events
Following our big plan, we started our daily experiment.
Type
Week
Operation
Result
Learn
2013.12 - 2014.4
Learing Molecular Cloning
Developing suitable protocols for Molecular Cloning
Build
2014.4.19 - 5.1
Amplifying and endonuclease digest plasmids of BBa_C0012, BBa_B0015, BBa_C0080, BBa_K082003 and BBa_C0062 (from Nanjing University)
Getting plasmid of BBa_C0012, BBa_B0015, BBa_C0080 & BBa_K082003  but finding BBa_C0062 invalid
Learn
Developing suitable protocols for the operation of PSB1C3 plasmid
Build
2014.5.2 - 5.18
Connecting BBa_C0012 & BBa_B0015, colony PCR
Tested positive
Build
2014.5.24 - 6.7
Adding [T7+RBS] to the [BBa_C0012+BBa_B0015] and sequencing it
Mutation sequenced
Build
2014.6.20 - 6.21
Adding [T7+RBS] to the [BBa_C0012+BBa_B0015] and sequencing it
Correct [T7+RBS + BBa_C0012+BBa_B0015]
Test
2014.6.28 - 8.17
Testing the function of [T7+RBS + BBa_C0012+BBa_B0015] by SDS-PAGE
Getting weak stripe from SDS-PAGE (due to low expression with LVA tag)
Build
Picking out BBa_C0062 by PCR valid plasmid
Standardized BBa_C0062
Build
Standardizing three restrict enzyme sites: Cla I, Sal I & Kpn I (basic parts)
Standardized Cla I, Sal I & Kpn I and Getting correct them with name BBa_K1532000, BBa_K1532001, BBa_K1532002
Build
Adding K09100 to [T7+RBS+ BBa_C0012+BBa_B0015]
Fluorescent expression curve of [T7+RBS+BBa_C0012+BBa_B0015+K09100]
Build
Build device [BBa_R0010 + RBS + C0080 + BBa_B0015]
Sequenced correct
Learn
Developing protocol for technique of fluorescent fusion protein gene expression (in BL21(DE3))
Fluorescent expression curve of [T7+RBS+BBa_C0012+BBa_B0015+K09100]. Valid bacteria with Lux I plasmid
Build
2014.8.29 - 9.1
Amplifying, transforming Lux I (from Wuhan University) and SDS-PAGE
Other
Confecting culture medium with AHL
Test
2014.9.1 - 9.16
Testing the interaction of IPTG and AHL
Unusual phenomenon that needs further experiment and discussion
Build
2014.9.17 - 9.22
Synthesizing the devices of Syn_EGFP, Syn_LuxR, Syn_LuxRI and Syn_Arac and sub-cloning them into E.coli PSB1C3
Getting correct Syn_EGFP, Syn_LuxR, Syn_LuxRI and Syn_Arac and respectively named BBa_K1532003~ BBa_K1532006
Build
2014.9.23 - 9.26
Building [PJ23104+LacI]
Getting sequenced (but untested) [PJ23104+LacI] (named BBa_K1532007)
Other
2014.9.27 - 9.28
Sending BBa_K1532000~ BBa_K1532006
Arriving at 2014.9.30
Build
2014.9.23 - 10.7
Building system of “111”, ”011”, ”101” and ”110”
Finished at 2014.10.15
Other
2014.10.4
Sending BBa_K1532007
Test
2014.10.13
Testing the function of [PJ23104+LacI]
Unfinished
Click here to edit this page! Click here to upload!
Retrieved from " http://2014.igem.org/Team:NUDT_CHINA/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
