<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NUDT_CHINA_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NUDT_CHINA/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NUDT_CHINA/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NUDT_CHINA/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NUDT_CHINA/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NUDT_CHINA/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NUDT CHINA/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H3>The Transformation of Our iGEM Ideas</H3><P>
Originally, we were attracted by the <I>Shortest Path Problem</I>, which aims to find the shortest path in a directed graph that can link the beginning node and ending node. Be inspired, we decided to design a cascade pathway <I>in vivo</I> which can help us read out the answer of Shortest Path Problem directly by the life stage of <I>E. coli</I>. 
</P><P>The main problem in the design are:</P><UL><LI>how to describe a route in <I>E. coli</I>;</LI><LI>how to build different routes in <I>E. coli</I>;</LI><LI>how to compare the length of different pathways.</LI></UL><UL><LI><A href="https://2014.igem.org/Team:NUDT_CHINA">Home</A></LI><LI><A href="https://2014.igem.org/Team:NUDT_CHINA/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:NUDT_CHINA/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:NUDT_CHINA/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:NUDT_CHINA/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:NUDT_CHINA/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:NUDT_CHINA/Safety">Safety Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:NUDT_CHINA/Attributions">Attributions</A></LI></UL><P>
To solve the first problem, we built devices that contains promoter, RBS, target gene (CDS) and terminator to simulate the nodes and lines in directed graph: the promoters are nodes and the target fragment are lines. When the promoter is activated, the target gene can coding protein which then activate the next promoter as regulatory element. This regulatory process can describe how a man the walking in a directed graph: he departed from the node P1, went through the line L and finally arrived at the spot P2. In this way, theoretically, we can describe any given directed graph in <I>E. coli</I>. (For further details about the coding of the graph, please refer to the project profile of our wiki) (Fig. 1)</P><P><CENTER>
Fig. 1 One promoter and one target gene can simulate one node and one line
</CENTER></P><P>
After building one complete graph in <I>E. coli</I>, different route should be labelled and tested independently so that we can get the length indirectly by the time of whole travel from beginning to ending. Then we can selected the quickest one as the shortest route. However, we are stuck in the second problem: how to build different route.</P><H5>First design:</H5><P>
In each regulatory unit, three different restrict enzyme sites are put between the promoter and RBS, target gene and terminator, terminator and next promoter, respectively. (Fig. 2)
</P><P><CENTER>Fig. 2 the position of three different restrict enzyme sites</CENTER></P><P>
In this way, we can cut the RBS and target gene out which means this route does not have line L (we link the 1, 2 (or 1, 3) to keep the circle structure of plasmid). Then we can test this route by the time of arriving at the ending node which reported by the report gene (we used the GFP). 
</P><H5>Second stage:</H5><P>But we found that it was not easy get so many different good restrict enzyme. So we change the design slightly. (Fig. 3) </P><P><CENTER>Fig. 3 the position of two same restrict enzyme sites</CENTER></P><P>In this way, we can still finish the function of first stage but save my restrict enzymes.</P><H5>Third stage:</H5><P>The last problem is how to compare the length of different pathway. Firstly, we decide to build all the possible route and test them. That is to say, we need to cut out 1, 2, 3,…,device(s) so that we can test the entire route, which is named exhaustion, quite time-consuming. Later, we found a better way where we can only cut one device which means we only need to build <I>n</I> graph if the graph has <I>n</I> lines.</P><P>Provided that we have 5 lines in a graph like Fig. 4.</P><P><CENTER>Fig. 4 a graph of 5 lines, where node 1 is the beginning node 4 is the ending</CENTER></P><P>According our method to find the shortest path, the first step is to build 6 different systems. Then we tested the “finished time” of each systems. Here is the systems and their “finish time”.</P><P><CENTER><TABLE width="50%" boder="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD width="20%"><STRONG>System</STRONG></TD><TD width="40%"><STRONG>Array</STRONG></TD><TD width="40%">Time (Theoretically)</TD></TR><TR><TD>11111</TD><TD>2 units</TD></TR><TR><TD>01111</TD><TD>2 units</TD></TR><TR><TD>10111</TD><TD>2 units</TD></TR><TR><TD>11011</TD><TD>2 units</TD></TR><TR><TD>11101</TD><TD>3 units</TD></TR><TR><TD>11110</TD><TD>unreachable</TD></TR></TBODY></TABLE></CENTER></P><P>Noticed that without line L<FONT size="-3">5</FONT>, the ending is inaccessible which implicates the L5 is indispensable. Though it does not block the accessibility, the absence of L<FONT size="-3">4</FONT> slows down the arrival, which implies that L4 is on the shortest path. Thus, from the date of the chart above, we can pick out the shortest path is:</P><H3>Protocol</H3><P>Here are the recapitulative protocols of the experiments that we have used.</P><H5>Polymerase Chain Reaction (PCR)</H5><P>Most of our PCRs was performed with the Recombinant Taq DNA Polymerase <I>TaKaRa Taq</I>(TM) or PrimeSTAR(R) HS DNA Polymerase and dNTP mixture (2.5mM each) under the standard 3-step PCR protocol. Touch-down RCR procedures were also used for several times to inhibit the nonspecific amplification. (Refer to the instruction for more details).</P><H5>Plasmid Preparation</H5><P>We used the TIANprep Mini Plasmid Kit (TIANGEN) to perform all of our plasmid preparation on bacteria with target plasmid. (Refer to the TIANGEN instruction for more details).</P><H5>Gel Electrophoresis &amp; Gel Extraction</H5><P>After the gel electrophoresis (normal experiment), we used the TIANgel Midi Plasmid Kit (TIANGEN) to extract all our target fragment in gel. (Refer to the TIANGEN instruction for more details).</P><H5>Endonuclease Digestion &amp; Ligases</H5><P>The enzymes we had used were EcoR I, Spe I, Xba I, Pst I, Cla I, Sal I, Kpn I (digestion), T4 DNA ligase (ligase) all of them were bought from TaKaRaTM. The digestion and ligase systems were built following the instructions of TaKaRa(TM).</P><H5>Fluorescent Fusion Protein Gene Expression</H5><P>Fluorometric measurement were processed with Fluoroskan(R) Ascent FL from ThermoTM using the filters of 485nm and 538nm. The E.coli to be tested was cultured in liquid LB media containing 35μg/mL chloramphenicol to OD0.6, than split into 96 well plates (from ThermoTM) by 150μL/well. The plate was then put into the Fluoroskan(R) Ascent FL and cultured in 25℃ and background shake of 90rpms. The Fluorometric measurement was processed with the interval time of 10 minutes. </P><H3>Blue print</H3><P>According our design of the whole cascade regulatory pathway as Fig. 1 showed, we can solve the Shortest Path Problem. However, building and standardizing these devices and systems means drawing the blue print.</P><P><CENTER><TABLE width="70%" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD width="12%"><STRONG>Systems</STRONG></TD><TD width="88%"><STRONG>Functional fragments at plasmid</STRONG></TD></TR><TR><TD>System “111”</TD></TR><TR><TD>System “011”</TD></TR><TR><TD>System “101”</TD></TR><TR><TD>System “110”</TD></TR></TBODY></TABLE></CENTER></P><P>
Fig. 1 regulatory pathways</P><P>Thus, we drew a picture of our experiment procedure. (Fig. 2) </P><P><CENTER>Fig. 2 experiment procedure</CENTER></P><H3>Week Events</H3><P>Following our big plan, we started our daily experiment.</P><P><CENTER><TABLE width="80%" cellpadding="0" cellspacing="0" border="1"><TBODY><TR><TD width="15%"><STRONG>Type</STRONG></TD><TD width="25%"><STRONG>Week</STRONG></TD><TD width="40%"><STRONG>Operation</STRONG></TD><TD width="20%"><STRONG>Result</STRONG></TD></TR><TR><TD rowspan="2">Learn</TD><TD rowspan="2">2013.12 - 2014.4</TD><TD>Learing Molecular Cloning</TD></TR><TR><TD>Developing suitable protocols for Molecular Cloning</TD></TR><TR><TD>Build</TD><TD rowspan="2">2014.4.19 - 5.1</TD><TD>Amplifying and endonuclease digest plasmids of BBa_C0012, BBa_B0015, BBa_C0080, BBa_K082003 and BBa_C0062 (from Nanjing University)</TD><TD>Getting plasmid of BBa_C0012, BBa_B0015, BBa_C0080 &amp; BBa_K082003  but finding BBa_C0062 invalid</TD></TR><TR><TD>Learn</TD><TD>Developing suitable protocols for the operation of PSB1C3 plasmid</TD></TR><TR><TD>Build</TD><TD>2014.5.2 - 5.18</TD><TD>Connecting BBa_C0012 &amp; BBa_B0015, colony PCR</TD><TD>Tested positive</TD></TR><TR><TD>Build</TD><TD>2014.5.24 - 6.7</TD><TD>Adding [T7+RBS] to the [BBa_C0012+BBa_B0015] and sequencing it</TD><TD>Mutation sequenced</TD></TR><TR><TD>Build</TD><TD>2014.6.20 - 6.21</TD><TD>Adding [T7+RBS] to the [BBa_C0012+BBa_B0015] and sequencing it</TD><TD>Correct [T7+RBS + BBa_C0012+BBa_B0015]</TD></TR><TR><TD>Test</TD><TD rowspan="6">2014.6.28 - 8.17</TD><TD>Testing the function of [T7+RBS + BBa_C0012+BBa_B0015] by SDS-PAGE</TD><TD>Getting weak stripe from SDS-PAGE (due to low expression with LVA tag)</TD></TR><TR><TD>Build</TD><TD>Picking out BBa_C0062 by PCR valid plasmid</TD><TD>Standardized BBa_C0062</TD></TR><TR><TD>Build</TD><TD>Standardizing three restrict enzyme sites: Cla I, Sal I &amp; Kpn I (basic parts)</TD><TD>Standardized Cla I, Sal I &amp; Kpn I and Getting correct them with name BBa_K1532000, BBa_K1532001, BBa_K1532002</TD></TR><TR><TD>Build</TD><TD>Adding K09100 to [T7+RBS+ BBa_C0012+BBa_B0015]</TD><TD>Fluorescent expression curve of [T7+RBS+BBa_C0012+BBa_B0015+K09100]</TD></TR><TR><TD>Build</TD><TD>Build device [BBa_R0010 + RBS + C0080 + BBa_B0015]</TD><TD>Sequenced correct</TD></TR><TR><TD>Learn</TD><TD>Developing protocol for technique of fluorescent fusion protein gene expression (in BL21(DE3))</TD><TD rowspan="2">Fluorescent expression curve of [T7+RBS+BBa_C0012+BBa_B0015+K09100]. Valid bacteria with Lux I plasmid</TD></TR><TR><TD>Build</TD><TD>2014.8.29 - 9.1</TD><TD>Amplifying, transforming Lux I (from Wuhan University) and SDS-PAGE</TD></TR><TR><TD>Other</TD><TD>Confecting culture medium with AHL</TD></TR><TR><TD>Test</TD><TD>2014.9.1 - 9.16</TD><TD>Testing the interaction of IPTG and AHL</TD><TD>Unusual phenomenon that needs further experiment and discussion</TD></TR><TR><TD>Build</TD><TD>2014.9.17 - 9.22</TD><TD>Synthesizing the devices of Syn_EGFP, Syn_LuxR, Syn_LuxRI and Syn_Arac and sub-cloning them into E.coli PSB1C3</TD><TD>Getting correct Syn_EGFP, Syn_LuxR, Syn_LuxRI and Syn_Arac and respectively named BBa_K1532003~ BBa_K1532006</TD></TR><TR><TD>Build</TD><TD>2014.9.23 - 9.26</TD><TD>Building [PJ23104+LacI]</TD><TD>Getting sequenced (but untested) [PJ23104+LacI] (named BBa_K1532007)</TD></TR><TR><TD>Other</TD><TD>2014.9.27 - 9.28</TD><TD>Sending BBa_K1532000~ BBa_K1532006</TD><TD>Arriving at 2014.9.30</TD></TR><TR><TD>Build</TD><TD>2014.9.23 - 10.7</TD><TD>Building system of “111”, ”011”, ”101” and ”110”</TD><TD>Finished at 2014.10.15</TD></TR><TR><TD>Other</TD><TD>2014.10.4</TD><TD>Sending BBa_K1532007</TD></TR><TR><TD>Test</TD><TD>2014.10.13</TD><TD>Testing the function of [PJ23104+LacI]</TD><TD>Unfinished</TD></TR></TBODY></TABLE></CENTER></P><DIV align="right"><A href="https://2014.igem.org/wiki/index.php?title=Team:NUDT_CHINA/Notebook&amp;action=edit"> Click here to edit this page!</A><A href="https://2014.igem.org/Special:Upload">Click here to upload! </A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:NUDT_CHINA/Notebook">http://2014.igem.org/Team:NUDT_CHINA/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NUDT_CHINA/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NUDT_CHINA/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NUDT_CHINA/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NUDT_CHINA/Notebook&amp;oldid=393600" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>