<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_DTU-Denmark_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:DTU-Denmark/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="windowcontent"><DIV class="row"><DIV class="maincontainer"><DIV class="colp-10 rmvpadmarg"><H1 class="bottomborder">Results</H1><P>The AMC experiment showed that we are able to detect a significant difference between the Bitis species and the <I>Naja nigricollis</I>. The initial AMC substrate experiment led to a more comprehensive substrate screening experiment that resulted in multiple substrate candidates. We managed to assemble the peptide sequences into the plasmid backbone (pSB1C3). The venom degradation test of the reporter molecules amilCP and β-galactosidase showed no reduction in the colorimetric or enzymatic properties. We improved part BBa_K592009 by adding a His-tag, the expression of color was not reduced and the color protein could easily be retained by a His-tag purification. To further improve our diagnostic device a reduction on the response time for the result was undertaken. Instead of the amilCP the reporter enzyme β-galactosidase were to be attached to the substrate linker. However, there were no successful assembly of ScAvidin with the linker to the β-galactosidase.

Visit <A href="https://2017.igem.org/Team:DTU-Denmark/ExperimentalDesign">experimental design page</A> for the theory and design behind our experiments, and <A href="https://2017.igem.org/Team:DTU-Denmark/Protocols">protocols</A> for the protocols of our experiments.
          </P><H2 id="amc">AMC Experiment</H2><P>In this experiment, we used the AMC fluorescent molecule, coupled with a peptide sequence (A-L-K) known to be cleaved by serine proteases from literature. The molecule emits fluorescence when it is released from the peptide it is coupled with.  For that reason, we expected to see fluorescence when the peptide was cleaved by proteases in the snake venom. We first generated a standard curve of the AMC molecule without the peptide sequence as seen in figure 1. The raw data for the standard curve can be found <A href="https://static.igem.org/mediawiki/2017/5/5d/T--DTU-Denmark--DATA_AMC_standardcurve.xlsx">here</A>.
          </P><FIGURE><FIGCAPTION>Figure 1: AMC standardcurve.The AMC molecule is not connected with the substrate in this part of the experiment.</FIGCAPTION></FIGURE><P>We tested the AMC-substrate peptide against the venom from our three snakes of interest, <I>Bitis arietans</I>, <I>Bitis gabonica</I> and <I>Naja nigricolis</I>. We made measurements in five different timepoints, with five different venom concentrations. The background noise was deducted from all measurements. The experiment showed that this particular substrate is cleaved significantly by the two venoms from <I>Bitis arietans</I> and <I>Bitis gabonica</I>, but not by the venom of <I>Naja nigricolis</I> as seen in figure 2-4. Different concentrations of venom had great effect in the fluorescence intensity. You can find the raw data <A href="2017.igem.org/wiki/images/6/6f/T--DTU-Denmark--results-AMC-1.xlsx">here</A>. 
          </P><FIGURE><FIGCAPTION>Figure 2:  ALK-AMC incubated with different concentrations of <I>Bitis arietans</I> venom.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 3: ALK-AMC incubated with different concentrations of <I>Bitis gabonica</I> venom.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 4: ALK-AMC incubated with different concentrations of <I>Naja nigricollis</I> venom.</FIGCAPTION></FIGURE><P>We repeated the experiment with two different substrate concentrations, along with three different venom concentrations, similar results were obtained. The bigger substrate concentration produced much higher fluorescence intensity as seen in figures 5-7. You can find the raw data <A href="https://static.igem.org/mediawiki/2017/3/3f/T--DTU-Denmark--DATA_AMC_experiment2.xlsx">here</A>.
          </P><FIGURE><FIGCAPTION>Figure 5: ALK-AMC incubated with two different substrate concentrations. The venom concentration was 100 μg/ml.
</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 6: ALK-AMC incubated with two different substrate concentrations. The venom concentration was 10 μg/ml.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 7: ALK-AMC incubated with two different substrate concentrations. The venom concentration was 1 μg/ml.</FIGCAPTION></FIGURE><P> In conclusion we were able to detect a significant difference from the Bitis species and the <I>Naja nigricollis</I>.</P><H2 id="peptidesubstratescreening">Peptide substrate screening</H2><P>In order to find more suitable substrates, we conducted screening experiments using JPT Peptide Technologies’ Protease Substrate Sets. They consisted of 360 oligopeptides with cleavage sites described in scientific literature, flanked by a fluorescent molecule (EDANS) and a quencher (DABCYL). Fluorescence is obtained when the fluorescent molecule is released from the complex due to cleavage by proteases.
          </P><P>The plates with the peptides were incubated with the three different venoms. Background fluorescence was deducted from the measurements. A great number of wells exhibited different fluorescence patterns, showing that some peptides had different specificities depending on the venom. The highest cleavage activity was observed when incubating with the venom of <I>Bitis arietans</I>. As expected, no peptide showed unique specificity for the venom of <I>Naja nigricollis</I> as seen in figure 8. From these peptides, the most significant ones that can be used for distinguishing between the three venoms were selected and submitted as parts. The plots with fluorescence intensity when incubating the three different venoms of the submitted peptides is shown below.
          </P><FIGURE><FIGCAPTION>Figure 8: Substrates with sequence DEMDEKSE (well J12) and LDEDEEDL (well D20), incubated with the three different venoms.</FIGCAPTION></FIGURE><P>As seen in figure 9, this peptide is not cleaved by any of the three venoms, and can be consequently used as a negative control.
          </P><FIGURE><FIGCAPTION>Figure 9: Substrate with sequence IDIRGPRL (well G2), incubated with the three different venoms.</FIGCAPTION></FIGURE><P>The peptides below, are cleaved by <I>Bitis arietans</I> and <I>Bitis gabonica</I>, as seen in figure 10. They can be used to distinguish between these two Bitis snakes and <I>Naja nigricollis</I>.
          </P><FIGURE><FIGCAPTION>Figure 10: Substrates with sequences PEDDGYFV (well D19), AAGVAPLS (well B11), DDSDAATF (well N21), incubated with the three different venoms.</FIGCAPTION></FIGURE><P>Three peptides were specifically cleaved only by <I>Bitis arietans</I> see figure 11.
          </P><FIGURE><FIGCAPTION>Figure 11: Substrates with sequences RQRYGKRS (well B17), PAPDAPLK (well K23), RKRRAVLT (well N17)  incubated with the three different venoms.</FIGCAPTION></FIGURE><P>The peptide in figure 12 is cleaved only by <I>Bitis gabonica</I> and can be used as a single positive response for that snake.
          </P><FIGURE><FIGCAPTION>Figure 12: Substrate with sequence GGVRGPRV (well O22) incubated with the three different venoms.</FIGCAPTION></FIGURE><P>You can find the raw data from this experiment <A href="https://static.igem.org/mediawiki/2017/6/68/T--DTU-Denmark--results-peptide-screening.xlsx">here</A>. For more on the analysis of this experiment, click <A href="https://2017.igem.org/Team:DTU-Denmark/Model">here</A>. 
          </P><H2 id="assemblyamilCP">Assembly of amilCP with and without His-tag</H2><P>As part of our collaboration with the Biosensor project one of the response genes used in the kit was amilCP (BBa_K592009), which we decided to improve by adding a His-tag. 
The results of the assembly of amilCP (BBa_K592009) with promoter and RBS (BBa_K608003) is shown in figure 13 by a digestion. The expected bands for a double digestion with EcoRI and PstI are 2029 bp for the backbone and 774 bp for the amilCP. For the single digests the bands should be 2803 bp which corresponds to the bands on the picture.
          </P><FIGURE><FIGCAPTION>Figure 13: Verification by digestion of amilCP (BBa_K592009) with promoter and RBS (BBa_K608003) from two colonies. Colony #1 is displayed in well 2-5.  Well 1 shows 1kb ladder. Well 2 shows the part double digested with EcoRI and PstI. Well 3 shows the part single digested with EcoRI. Well 4 shows the part single digested with PstI. Well 5 is the undigested amilCP before verification. Colony #2 is displayed in well 7-10. Well 7 is the amilCP double digested with EcoRI and PstI. Well 8 is single digested with EcoRI and well 8 is single digested with PstI. Well 10 is undigested amilCP. </FIGCAPTION></FIGURE><P>The His-tagged amilCP sequence was ordered as a gBlock and assembled into the pSB1C3 backbone. The results of the modified amilCP with the His-tag is shown in figure 14. The expected bands for a double digestion with EcoRI and PstI are 2029 bp for the backbone and 792 bp for the amilCP with His-tag. For the single digests the bands should be 2821 bp which corresponds to the bands on the picture.
          </P><FIGURE><FIGCAPTION>Figure 14: Verification of amilCP with His-tag (BBa_K2355002) by digestion. 
Well 1 shows the 1 kb ladder. Well 2 is amilCP with His-tag double digested with EcoRI and PstI. Well 3 is amilCP with His-tag single digested with EcoRI and well 4 is single digested with PstI. Well 5 is undigested amilCP with His-tag
</FIGCAPTION><P>A PCR was run on the amilCP with His-tag the results is shown in figure 15. Well number 4 and 5 was amilCP with His-tag (BBa_K2355002) and it showed clear bands at 1065 bp. 
          </P><FIGURE><FIGCAPTION>Figure 15: Verification of amilCP with His-tag by PCR. The PCR was run with the iGEM verification primers: BBa_G00100 and BBa_G00101. Well 4 and 5 shows amilCP with His-tag (BBa_K2355002). Well 2-3 is not related to the amilCP part.</FIGCAPTION></FIGURE><P>Below two pictures of amilCP with His-tag (BBa_K2355002) is displayed see figure 16. The left picture is amilCP with His-tag spun down in LB media. The picture on the right is a picture of amilCP with His-tag after the lysis procedure by sonication. It is clear that the expression of the blue/purple color is very strong. In conclusion the amilCP has been successfully modified by adding a His-tag without compromising the protein. 
          </P><FIGURE><FIGCAPTION>Figure 16: The picture to the left is of amilCP with His-tag spun down in LB media. The picture to the right is a picture of amilCP with His-tag after the lysis procedure by sonication. </FIGCAPTION></FIGURE><H2 id="hispurification">AmilCP with His-tag purification test</H2><P>To test whether the amilCP with His-tag worked as expected, we purified two solutions of amilCP; one with His-tagged amilCP, and one with just amilCP. Cells producing the His-tagged amilCP were lysed in accordance with the cell lysis protocol before being used in this experiment. The non His-tagged amilCP producing cells were not lysed because they secreted the amilCP out of the cells and into the media.  
          </P><P>Both solutions of amilCP were diluted to an equal absorbance before the solutions were loaded onto Ni-NTA columns to be purified. The purification is based on loading the columns, washing away residual proteins, and eluting the target protein.  
          </P><P>As shown in table 1 below, the vast majority of amilCP without His-tag seems to run right through the column, as high concentrations can be detected in the flow through after adding the samples to the column. In addition, the absorbance of the eluant is not far from that of the blank, indicating that no amilCP was retained in the column and eluted. It seems like it makes no difference to run the sample through the column multiple times before washing and eluting, which makes sense if the amilCP is not bound to the column. 
          </P><FIGURE><FIGCAPTION>Table 1: The absorbance of solutions of amilCP without His-tag, isolated from the different steps in the His-tag purification. The absorbance was measured at 588 nm.</FIGCAPTION></FIGURE><P>Table 2 shows a different pattern for the His-tagged amilCP. For the His-tagged amilCP, the flow-through after sample loading yields concentrations of amilCP that are lower or equal to that of the eluates. The number of times the sample is run through the column seems to make a big difference, especially in the 2nd elution, where it seems there are a linear correlation between the number of times cell lysate were loaded onto the column, and the concentration of eluted amilCP with His-tag.   
          </P><FIGURE><FIGCAPTION>Table 2: The absorbance of amilCP with His-tag isolated from the different steps in the His-tag purification. The absorbance was measured at 588 nm.</FIGCAPTION></FIGURE><P>The difference between the purification of amilCP with and without His-tag is illustrated by figure 16. It is clear that amilCP without His-tag just runs directly through the column, whereas the His-tagged amilCP actually binds to the column and elutes as expected. 
          </P><FIGURE><FIGCAPTION>Figure 16: Comparison of how amilCP with or without His-tag is purified. The data used graph is from the samples with the highest amount of sample loading (5 times). The absorbance was measured at 588 nm. </FIGCAPTION></FIGURE><P>From these results we can conclude that our his-tag purification of amilCP has worked.</P><H2 id="amilcpdegradation">AmilCP degradation by venom  </H2><P>The amilCP without His-tag (BBa_K592009) was obtained from Biosensor. Apparently the cells had produced an excessive amount of amilCP and exported it out of the cells. This means that it was not necessary to lyse the cells. A simple spin down of the cells left the amilCP in the supernatant and it was used for the amilCP degradation experiment.
          </P><P>AmilCP were incubated with the three different snake venoms: <I>Bitis arietans</I>, <I>Bitis gabonica</I> and <I>Naja nigricollis</I>.  
          </P><P>After 70 min of incubation time at 37°C, a spectrophotometer was used to measure the absorbance at 588 nm. There was no loss of amilCP during the incubation with snake venom.  
The reason for the slight fluctuation could be small pipetting errors, or that the spectrophotometer did not have an optimal readout at 588 nm. The results of the experiment is shown in table 3. 
          </P><FIGURE><FIGCAPTION>Table 3: The table shows the measurements for amilCP (BBa_K592009) at 588 nm. 
The dilution factor of the lysate was 10. 
The dilution factor for lysate with venom was 15. 
</FIGCAPTION><TABLE><TBODY><TR><TH>Measured absorbance</TH><TH>Calculated absorbance</TH></TR><TR><TH>Before addition of snake venom </TH></TR><TR><TD><SPAN>Lysate </SPAN></TD><TD><SPAN>0.106</SPAN></TD><TD><SPAN>1.06</SPAN></TD></TR><TR><TH>After addition of snake venom </TH></TR><TR><TD><SPAN>Lysate 100 ug/mL w/ <I>Bitis arietans</I> venom  </SPAN></TD><TD><SPAN>0.082</SPAN></TD><TD><SPAN>1.24</SPAN></TD></TR><TR><TD><SPAN>Lysate 10 ug/mL w/ <I>Bitis arietans</I> venom </SPAN></TD><TD><SPAN>0.093</SPAN></TD><TD><SPAN>1.39</SPAN></TD></TR><TR><TD><SPAN>Lysate 100 ug/mL w/ <I>Bitis gabonica</I> venom  </SPAN></TD><TD><SPAN>0.096</SPAN></TD><TD><SPAN>1.44</SPAN></TD></TR><TR><TD><SPAN>Lysate 10 ug/mL w/ <I>Bitis gabonica</I> venom  </SPAN></TD><TD><SPAN>0.089</SPAN></TD><TD><SPAN>1.34</SPAN></TD></TR><TR><TD><SPAN>Lysate 100 ug/mL w/ <I>Naja nigricollis</I> venom  </SPAN></TD><TD><SPAN>0.097</SPAN></TD><TD><SPAN>1.44</SPAN></TD></TR><TR><TD><SPAN>Lysate 10 ug/mL w/ <I>Naja nigricollis</I> venom  </SPAN></TD><TD><SPAN>0.101</SPAN></TD><TD><SPAN>1.51</SPAN></TD></TR><TR><TD><SPAN>Lysate control </SPAN></TD><TD><SPAN>0.086</SPAN></TD><TD><SPAN>1.29</SPAN></TD></TR></TBODY></TABLE></FIGURE><P>Since the measured absorbance was not lower that the control with no venom, it is clear from table 3 that the amilCP was not degraded by any of the three snake venoms. In conclusion this means that it is safe to use amilCP as a reporter molecule in our device with snake venom. 
          </P><H2 id="laczdegradation">Test of lacZ degradation by venom</H2><P>As in the test for amilCP stability in venom, the reporter enzyme β-galactosidase, which is used in the composite part (BBa_K2355313), was incubated in venom to test whether or not venom incubation would degrade or otherwise interfere with the activity of the enzyme.
After 70 minutes of incubation at 37°C, the activity of venom-incubated lacZ was measured using ONPG (1.5 mg/mL) and a spectrophotometer. The result can be seen in figure 17.  
          </P><FIGURE><FIGCAPTION>Figure 17: the activity of β-galactosidase is measured in the presence of different snake venoms. NN: <I>Naja nigricollis</I>, BA: <I>Bitis arietans</I>, BG: <I>Bitis gabonica</I>. Brij is the buffer that the snake venom was dissolved in. </FIGCAPTION></FIGURE><P>The activity of β-galactosidase is not decreased in the presence of the chosen snake venoms as seen in figure 17, suggesting that it is not degraded by the protease activity of the venoms. This makes it a suitable candidate as an amplification molecule for our device.</P><H2 id="linkerassembly">Assembly of linkers derived from substrate screening and literature</H2><P>The linkers found by screening was ordered as oligonucleotides from IDT. The oligos were annealed to each other by heating it up to 94°C for 2 minutes and then gradually cooled. 
The linker part BBa_K2355001 was ordered as a gBlock from IDT. The linkers were assembled by digestion, ligation and transformation.  
          </P><P>For verification a digestion of the purified plasmid was made.  
All linkers were single digested first with EcoRI and then with PstI. Uncut plasmid was also run on the gel to see the difference between cut and uncut.  
          </P><P>All linkers obtained from the substrate screening has a length of 24 nucleotides. The oligos that we received had a total length of 60 bp because of the desired restriction sites at the ends. 
When the linkers are assembled correctly into the plasmid backbone (pSB1C3) and cut with either EcoRI or PstI the length obtained is 2094 bp. 
          </P><P>The only linker not obtained by the substrate screening BBa_K2355001 has a length of 72 bp. When assembled correctly into the plasmid backbone (pSB1C3) and cut with either EcoRI or PstI the length obtained is 2142 bp. 
The uncut plasmid will run faster in the gel due to the supercoiled structure. The results of the digest is displayed in figure 18, figure 19, figure 20 and figure 21. 
          </P><FIGURE><FIGCAPTION>Figure 18: Digest verification of linkers: BBa_K2355006 well 2-7, BBa_K2355008 well 8-13 and BBa_K2355014 well 14-19. 1Kb ladder. Well 4, 7, 10, 13, 16 and 19 uncut. The rest of the wells are single cuts with either EcoRI or PstI.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 19: Digest verification of linkers: BBa_K2355015 well 2-7, BBa_K2355025 well 8-13 and BBa_K2355016 well 14-19. 1Kb ladder. Well 4, 7, 10, 13, 16 and 19 uncut. The rest of the wells are single cuts with either EcoRI or PstI.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 20: Digest verification of linkers: BBa_K2355032 well 2-7, BBa_K2355033 well 8-13 and BBa_K2355037 well 14-19. 1Kb ladder. Well 4, 7, 10, 13, 16 and 19 uncut. The rest of the wells are single cuts with either EcoRI or PstI.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 21: Digest verification of linkers: BBa_K2355039 well 2-7 and BBa_K2355001 well 8-13. 1Kb ladder. Well 4, 7, 10 and 13 uncut. The rest of the wells are single cuts with either EcoRI or PstI.</FIGCAPTION></FIGURE><P>TFrom the digestion it seems that the assembly of the linkers had succeeded. To confirm it we decided to run a PCR with the iGEM verification primers: BBa_G00100 and BBa_G00101. The expected bands for the ten small linkers are supposed to be 338 bp and for the one large linker it is 386 bp which corresponds well to the gel pictures displayed in figure 22 and 23. 
          </P><FIGURE><FIGCAPTION>Figure 22: verfication of linkers by PCR. 100 bp ladder. Linkers: BBa_K2355006 well 2-3, BBa_K2355008 well 4-5, BBa_K2355014 6-7, BBa_K2355015 8-9, BBa_K2355025 10-11. Wells 12-16 negative controls.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 23: verfication of linkers by PCR. 100 bp ladder. Linkers BBa_K2355026 well 2-3, BBa_K2355032 well 4-5, BBa_K2355033 well 6-7, BBa_K2355037 well 8-9, BBa_K2355039 well 10-11, BBa_K2355001 well 12-13. Wells 14-16 negative controls.</FIGCAPTION></FIGURE><P>Both of the verification methods showed that all 11 linkers submitted are present in the backbone (pSB1C3). </P><P>We also decided to send the linkers for sequencing the results of the sequencing is available <A href="https://static.igem.org/mediawiki/2017/2/2a/T--DTU-Denmark--results-linker.pdf">here</A>. </P><P>In conclusion the assembly of all linkers succeeded. </P><H2 id="composite">Assembly of composite part</H2><P>An unregistered composite part was ordered as an IDT gBlock. The gBlock was assembled by digestion, ligation and transformation. It was done in several attempts with different protocols. The parameters that were changed during the troubleshooting was the ligation ratio, incubation temperature and time. Unfortunately despite these alterations the assembly was unsuccessful. 
          </P><P>The composite part BBa_K2355302 was ordered as an IDT gBlock but delivered very late in the process. It was assembled by digestion, ligation and transformation. 
The first verification of the part was done by digestion of the purified plasmid. Six colonies were chosen and digested with EcoRI and PstI the desired bands were 2029 bp and 2658 bp. The results of the digestion is shown in figure 24. 
          </P><FIGURE><FIGCAPTION>Figure 24: Digest verification of 6 potential colonies of composite part BBa_K2355302. 1kb ladder. Wells 2,5,8,11,14,17 double cut with EcoRI and PstI. wells 3,6,9,12,15,18 single cut with EcoRI. The rest of the wells are uncut.  </FIGCAPTION></FIGURE><P>None of the digestions gave the expected bands. Therefore, a PCR was also run on BBa_K2355302 with iGEM verification primers: BBa_G00100 and BBa_G00101. But showed no bands of the expected length. 
          </P><P>The results of the digestion and PCR showed that the assembly of BBa_K2355302 had not worked. 
          </P><H2 id="laczassembly">Assembly of composite lacZ part (BBa_2355313)</H2><P>Using basic digestion/ligation based assembly of parts, we were able to assemble a composite part, consisting of a coding sequence for ScAvidin with BBa_B0030 RBS, and the coding sequence for an empirical protein linker. This has been confirmed by sequencing. We were also able to attach a promoter (BBa_R0010) to this composite part, and we were able to PCR-amplify a lacZ coding sequence to prepare the BBa_I732005 lacZ part for assembly using the RFC[25] standard.  
          </P><P>However, while assembling the lacZ part with the RBS + ScAvidin + linker part, we might have encountered some STAR activity from our restriction enzymes, as the finalised lacZ part only contains fractions of the ScAvidin molecule, in contrast to the intermediates, where all of the ScAvidin was present. 
          </P><FIGURE><FIGCAPTION>Figure 25: Picture of 1% agarose gel with 1 kb NEB ladder. It shows the PCR product of a lacZ part (BBa_I732005). The bands correspond well with the fact that lacZ is ~3000 bp long.</FIGCAPTION></FIGURE><P>The finalised composite lacZ plasmid was digested with BseRI to verify that the Scavidin part + RBS was there. If inserted correctly, two bands would be visible at ~6000 bp and ~1000 bp respectively. The bands on the gel (see figure 26) fit quite nicely with this description, except for the ~1000 bp band, which looks to be ~2000 bp long. However, since the plasmid only is about 7000 bp in length, it made no sense for such a large band to appear, and we assumed that the Scavidin would be inserted correctly, or the 6000 bp band wouldn’t appear. It thus appeared like the lacZ composite part was assembled correctly, but after sequencing the part, it was apparent that the positive result from the BseRI digest were due to the BseRI sites from Scavidin being recombined within the plasmid.  </P><FIGURE><FIGCAPTION>Figure 26: 1 % agarose gel with the result of BseRI digest. The ladder used was a 1 kb ladder from NEB. Wells 3-7 was loaded with BseRI lacZ device plasmids, and well 2 and 8 was used as negative and positive control respectively. </FIGCAPTION></FIGURE><H2 id="lacztest">Test of composite lacZ part: ScAvidin binding to biotin</H2><P>To test if our composite lacZ part was able to attach to biotinylated beads, a cell lysate from BBa_K2355313 transformants was added to tubes containing biotinylated magnetic beads. After separation on a magnetic separation rack, the supernatant was removed from tubes, and tubes were washed with washing buffer (PBS TWEEN-20 0.05%) a couple of times. For each washing cycle, samples were put aside for testing. The different samples were tested by adding ONPG, and measuring the activity of β-galactosidase. A plot of the measured average β-galactosidase for each purification step is shown in figure 27. Along with samples of washing buffer, the biotinylated beads themselves were measured for β-galactosidase activity, both before (Beads 1) and after (Beads 2) the washing steps. </P><FIGURE><FIGCAPTION>Figure 27: A plot of the activity of β-galactosidase for different purification steps using biotinylated magnetic beads. </FIGCAPTION></FIGURE><P>Figure 27 shows that first fraction, which is the supernatant from the first magnetic separation following the addition of cell lysate to the beads, has the highest enzymatic activity of all the fractions measured, indicating that most of the β-galactosidase is washed out immediately, and not bound to the biotin beads. This is true for both the lacZ device with and without ScAvidin, which suggests that the ScAvidin of the composite lacZ part doesn’t work as intended. The fractions from the secondary and tertiary washing steps showed an enzyme activity close to zero; as did the enzyme activity of the isolated beads. </P><P>The fact that the lacZ part with ScAvidin didn’t bind to the biotin beads underlines the fact that a fully working ScAvidin might not be present at in the composite lacZ part. </P><H2 id="laczcomparison">Comparison of composite part with β-galactosidase</H2><P>To test if our composite lacZ part (BBa_K2355313) would be cleaved by the snake venom, we used the biotinylated beads from the ScAvidin-binding experiment, and incubated them in snake venom for approx. 70 minutes. Besides having solutions of snake venom in Brij (0.01%), human blood serum was used to make solutions of snake venom that would emulate the blood samples of patients being envenomed by snakes.  </P><P>As with the ScAvidin binding experiment, the presence of β-galactosidase is determined by measuring the conversion of ONPG in a spectrophotometer. 
Figure 28 shows the result of the β-galactosidase activity measurements for the venom solutions with human blood serum. 
 </P><FIGURE><FIGCAPTION>Figure 28: Cleavage shown by β-galactosidase activity in samples with serum diluted snake venom. Positive control + is diluted cell lysate with the ScAvidin+LacZ composite part (BBa_K2355313). Positive control - is diluted cell lysate with LacZ but without ScAvidin. </FIGCAPTION></FIGURE><P>As seen in figure 28, no noticeable activity is observed in any of the venom solutions. Ideally, we would have seen more β-galactosidase activity in the solutions with venoms that are able to cleave the empirical linker (i.e. the two <I>Bitis</I> venoms) as the venoms would have cleaved the linker, thus releasing the β-galactosidase into the supernatant that we were collecting in this experiment. The test with Brij-diluted venom had the same profile as figure 28, with no β-galactosidase activity from the venom solutions.     
 </P><P>Since the ScAvidin binding experiment demonstrated an insignificant amount of ScAvidin binding to biotinylated beads, the concentrations of our composite lacZ part on the beads were probably too low for us to measure any cleavage of the linker sequence. </P></FIGURE></DIV><DIV class="colp-2 rmvpadmarg"><DIV class="rightnavbar rmvpadmarg"><UL><LI class="rightnavbarbtn"><A href="#results">Results</A></LI><LI class="rightnavbarbtn"><A href="#amc">AMC Experiment</A></LI><LI class="rightnavbarbtn"><A href="#peptidesubstratescreening">Peptide Substrate Screening</A></LI><LI class="rightnavbarbtn"><A href="#assemblyamilCP">Assembly of amilCP with and without His-tag</A></LI><LI class="rightnavbarbtn"><A href="#hispurification">AmilCP with His-tag purification test</A></LI><LI class="rightnavbarbtn"><A href="#amilcpdegradation">AmilCP degradation by venom </A></LI><LI class="rightnavbarbtn"><A href="#laczdegradation">Test of lacZ degradation by venom</A></LI><LI class="rightnavbarbtn"><A href="#linkerassembly">Assembly of linkers derived from substrate screening and literature</A></LI><LI class="rightnavbarbtn"><A href="#composite">Assembly of composite part</A></LI><LI class="rightnavbarbtn"><A href="#laczassembly">Assembly of composite lacZ part (BBa_2355313)</A></LI><LI class="rightnavbarbtn"><A href="#lacztest">Test of composite lacZ part: ScAvidin binding to biotin</A></LI><LI class="rightnavbarbtn"><A href="#laczcomparison">Comparison of composite part with β-galactosidase</A></LI></UL></DIV></DIV></DIV></DIV><FOOTER class="rmvpadmarg" id="footer"><DIV class="footercontainer"><DIV class="colp-5 rmvpadmarg"><DIV class="rmvlinkstyle footercolstyle leftfootercol"><P>FIND US AT</P><SPAN>DTU BIOBUILDERS</SPAN><SPAN>DENMARK</SPAN><SPAN>DTU - SØLTOFTS PLADS, BYGN. 221/006</SPAN><SPAN>2800 KGS. LYNGBY</SPAN></DIV></DIV><DIV class="colp-5 rmvpadmarg"><DIV class="rmvlinkstyle footercolstyle rightfootercol"><P>MAIN SPONSORS</P></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>