<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Wetlab_Genome_Reduction"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Wetlab/Genome_Reduction">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Wetlab/Genome_Reduction&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Genome_Reduction&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Genome_Reduction&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Wetlab/Genome_Reduction" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Wetlab/Genome Reduction</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="background:#FFFFFF ; border:3.5px solid #60AFFE; padding: 1em; margin: auto ; width:98.5%"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Genome_reduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Genome reduction</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Goal"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Goal</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Method"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Proof_of_concept"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Proof of concept</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Extracting_modelling_parameters_from_the_proof_of_concept_experiment"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Extracting modelling parameters from the proof of concept experiment</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Modification_of_proof_of_concept"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Modification of proof of concept</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#SceI_restriction_enzyme"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">SceI restriction enzyme</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#T4_ligase"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">T4 ligase</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-9"><A href="#Lab_results"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Lab results</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Modified_proof_of_concept"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Modified proof of concept</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#SceI_restriction_enzyme_and_T4_ligase"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">SceI restriction enzyme and T4 ligase</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#References"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Genome_reduction"> Genome reduction </SPAN></H2><H3><SPAN class="mw-headline" id="Goal"> Goal </SPAN></H3><P>In this section we discuss about how to randomly delete chromosomal fragments of <I>E. coli</I> in order to reduce the physical amount of genomic DNA.
</P><H3><SPAN class="mw-headline" id="Method"> Method </SPAN></H3><P>To reach our goal of random deletion of chromosomal fragments, we want to synthesize a frequently cutting restriction enzyme along with the simultaneous, shortly delayed, or even continuous synthesis of a ligase. The restriction enzyme will cut genomic DNA in a random fashion <I>in vivo</I>, while the ligase performs its job of religation. Assuming that the genomic DNA is cut at several sites within one cell, religation will lead to the exclusion of chromosomal fragments in some cells. Multiple rounds of restriction and religation will therefore lead to a markedly reduced genome.
</P><H3><SPAN class="mw-headline" id="Proof_of_concept"> Proof of concept </SPAN></H3><P>In 1997 Ren et al. showed that <I>in vivo</I> religation of linearized vector and insert is possible by overexpression of the T4 ligase <A href="https://2008.igem.org/Team:ETH_Zurich/Wetlab/Genome_Reduction#References" class="external text" rel="nofollow">(1)</A>. Additionally, Heitman et al. proposed that intracellular DNA which has been cut <I>in vivo</I> by EcoRI is religated by the ligase T4 <A href="https://2008.igem.org/Team:ETH_Zurich/Wetlab/Genome_Reduction#References" class="external text" rel="nofollow">(2)</A>. Our idea, however, relies on the assumption that <I>in vivo</I> restriction and religation is possible and leads to the exclusion of chromosomal DNA. We are trying to verify these assumptions in several experiments which will from now on be refered to as our “proof of concept”.
</P><P>Our proof of concept relies on the following construct (not shown are ribosomal binding sites behind both SceI restriction sites and a terminator following the RFP):
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:297px;"><DIV class="thumbcaption">Figure 1: Construct for proof of concept</DIV></DIV></DIV></DIV><P>
The above construct was ordered at GeneArt and is supposed to be integrated into the genome of a wild-type <I>E. coli</I> strain (MG1655). We are planning to use the lambda red recombination system established by Wanner and Datsenko <A href="https://2008.igem.org/Team:ETH_Zurich/Wetlab/Genome_Reduction#References" class="external text" rel="nofollow">(3)</A> to integrate our construct into the tryptophanase A gene resulting in the knockout of the latter.   
</P><P>Additionally, we ordered DNA encoding the T4 ligase and the SceI restriction enzyme. Transformation of these plasmid-encoded enzymes into cells carrying the proof of concept construct is supposed to yield bacteria that can be induced to express T4 and SceI.
</P><P>Before synthesis of T4 and SceI, cells carrying the proof of concept construct do not synthesize RFP. After synthesis of the restriction enzyme, SceI will cut the bacterial chromosome at the sites indicated above. The ligase will then religate the construct, leading to the exclusion of the sensitivity gene and the terminator in some of the cells. Hence, these cells would synthesize the RFP reporter protein and could easily be identified:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:312px;"><DIV class="thumbcaption">Figure 2: Proof of concept</DIV></DIV></DIV></DIV><P>
Additionally, adding sucrose to the medium would kill all cells that have not eliminated the sacB sensitivity gene. Hence, only RFP-synthesizing bacteria would survive, which would presumably facilitate detection.
</P><H4><SPAN class="mw-headline" id="Extracting_modelling_parameters_from_the_proof_of_concept_experiment"> Extracting modelling parameters from the proof of concept experiment </SPAN></H4><P>A very important parameter in our reduction strategy approach is undoubtly the efficiency on which the process of cutting, fragment removal and re-ligation is going to occure <I>in-vivo</I>. Our proof of concept construct and experiment were designed in order to provide us two important indexes of efficiency, namely the cutting efficiency (CE) and the removal and ligation efficiency (RLE). The CE parameter represents the probability that at least one cut is made into the chromosome of a cell for a pulse of expression of the restriction enzyme. The RLE represents the probability that after a cutting a fragment is lost and the chromosome is relagated. This efficiency frequencies are going to help us understand the behaviour of the single cell when expression of the restriction enzyme is induced. In order to calculate the two parameters from the experiment we planned to:
</P><UL><LI> take pictures of the plate with the microscope before inducing the expression of restriction enzyme and, manually or with help of specialized software, estimate the total number of cells in the population (TOT_POP)
</LI><LI> after having induced the restriction enzyme expression and the ligase, take pictures of the plate in a similar way and extimate: the number of cells left (SURVIVED_POP) and the number of cells showing fluorescence (FLUO_POP). 
</LI></UL><P>From these three population counts we can set up the system of two equations:</P><P>After solving the equations we can use the parameters for modelling the probability of one fragment to be removed, as we used in the modelling section <A href="/Team:ETH_Zurich/Modeling/Genome-Scale_Model" title="Team:ETH Zurich/Modeling/Genome-Scale Model">Genome Scale Model</A>.
</P><H4><SPAN class="mw-headline" id="Modification_of_proof_of_concept"> Modification of proof of concept </SPAN></H4><P>Unfortunately, up to now we have not received our construct for the proof of concept. Therefore, we are working on a modified construct:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:265px;"><DIV class="thumbcaption">Figure 3: Modified construct for proof of concept</DIV></DIV></DIV></DIV><P>
For this purpose, a commonly used RFP reporter plasmid is used. SceI restriction sites are introduced by cutting the plasmid in front of and after the RFP gene and ligating oligonucleotide duplexes containing the desired restriction sites. 
</P><P>After transformation of bacteria with the construct shown above and plasmids containing the genes coding for SceI restriction enzyme and the T4 ligase, induction of gene expression will lead to <I>in vivo</I> cutting and religation of the RFP-encoding plasmid, leading to the loss of the RFP gene from the plasmid. Therefore, if cells lose the RFP signal after induction of the restriction enzyme and ligase, we have proven that <I>in vivo</I> restriction and ligation is possible and can lead to the exclusion of DNA fragments:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:265px;"><DIV class="thumbcaption">Figure 4: Modified proof of concept</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="SceI_restriction_enzyme"> SceI restriction enzyme </SPAN></H4><P>SceI is a site-specific homing endonuclease. It is extremely rare-cutting as it recognizes an 18 bp sequence. These properties make SceI a perfect restriction enzyme for our proof of concept for which the restriction enzyme should only cut the sites of our constructs indicated above. However, for our goal of minimizing <I>E. coli</I>’s genome we will need to use a frequently cutting restriction enzyme which will potentially lead to severe damage of the cell’s genome. In order to limit this damage, we want to be able to pulse the expression of the restriction enzyme. Therefore, we are cloning an inducible promoter in front of SceI.
</P><H4><SPAN class="mw-headline" id="T4_ligase"> T4 ligase </SPAN></H4><P>T4 is the commonly used ligase for <I>in vitro</I> cloning. For <I>in vivo</I> cloning, high levels of T4 are advantageous not only for improving the efficiency of religation leading to the exclusion of chromosomal fragments, but also for limiting DNA damage. However, constitutive overexpression of T4 might lead to immediate religation without the exclusion of chromosomal fragments. Therefore, we are cloning T4 behind an IPTG-inducible promoter (R0010), and, as an alternative approach, behind several constitutive promoter of differing strengths. Another idea would be to clone both SceI and T4 behind the same promoter, so that induction would lead to simultaneous expression of both enzymes.
</P><H3><SPAN class="mw-headline" id="Lab_results"> Lab results </SPAN></H3><P><B>Goals</B></P><P>For the proof of concept we want to generate the following constructs:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 5: Plasmids for proof of concept</DIV></DIV></DIV></DIV><P>These two plasmids will be transformed into the same host cells.
The RFP generator flanked by two SceI-restriction sites will be on a low-copy plasmid, like pCK01, to reduce the number of cutting targets for SceI. The T4 ligase and the SceI restriction enzyme will be placed on a high-copy plasmid. Expression of T4 and SceI will be induced to cut out the RFP generator and religate the digested plasmid <I>in vivo</I>. This can be easily detected by the disappearance of the red fluorescence of the cells. We have chosen to use RFP, since on agar plates RFP-synthesizing colonies are easily distinguishable from the ones which have lost the RFP gene. By using antibiotics we will assure that only colonies which contain the plasmids encoding the two resistance genes will survive.
</P><H4><SPAN class="mw-headline" id="Modified_proof_of_concept"> Modified proof of concept </SPAN></H4><P>For the RFP generator we used the Biobrick BBa_J04450 and cloned SceI-restriction sites into its EcoRI- and PstI-sites.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:391px;"><DIV class="thumbcaption">Figure 6: Plasmids for proof of concept</DIV></DIV></DIV></DIV><P>For the SceI-restriction sites we ordered oligos containing the SceI-recognition site and, at its ends, the overhangs for EcoRI- and PstI-sites, respectively. The oligos were hybridized and cloned into BBa_J04450. Successful cloning was verified by digestion and gel electrophoresis.
</P><P>Oligos for SceI-site integrated at EcoRI-site:
</P><PRE>       EcoR_SceI_up 
       AATTCAGTTACGCTAGGGATAACAGGGTAATATAGC 
 
       EcoR_SceI_down 
       AATTGCTATATTACCCTGTTATCCCTAGCGTAACTG
</PRE><P>
Oligos for SceI-site integrated at PstI-site:
</P><PRE>       Pst_site_up
       AGTTACGCTAGGGATAACAGGGTAATATAGCTGCA

       Pst_site_down
       GCTATATTACCCTGTTATCCCTAGCGTAACTTGCA
</PRE><P>
Hybridized Oligos:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 7: Hybridized oligos for insertion of SceI-restriction sites</DIV></DIV></DIV></DIV><P>In future work we are planning to subclone this construct (RFP with two SceI-restriction sites) onto a low-copy plasmid.
</P><H4><SPAN class="mw-headline" id="SceI_restriction_enzyme_and_T4_ligase"> SceI restriction enzyme and T4 ligase</SPAN></H4><P>We performed the following cloning to bring SceI and T4 under the control of different promoters by standard Biobrick assembly:
</P><UL><LI><A href="http://partsregistry.org/Part:BBa_K142205" class="external text" rel="nofollow">BBa_K142205</A>: SceI under the lac-Promoter
</LI></UL><UL><LI><A href="http://partsregistry.org/Part:BBa_K142204" class="external text" rel="nofollow">BBa_K142204</A>: T4 under the lac-Promoter
</LI></UL><UL><LI><A href="http://partsregistry.org/Part:BBa_K142207" class="external text" rel="nofollow">BBa_K142207</A>: T4 under a medium constitutive promoter
</LI></UL><UL><LI><A href="http://partsregistry.org/Part:BBa_K142206" class="external text" rel="nofollow">BBa_K142206</A>: T4 under a strong constitutive promoter
</LI></UL><H3><SPAN class="mw-headline" id="References"> References </SPAN></H3><P>(1) <A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6T39-3V4BTWM-CH&amp;_user=791130&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_version=1&amp;_urlVersion=0&amp;_userid=791130&amp;md5=5f7b6177feaf9c0d06cd0e9b334a43ce" class="external text" rel="nofollow">Ren Z. J., Baumann R. G., Black L. W. (1997): Cloning of linear DNAs in vivo by overexpressed T4 DNA ligase: construction of a T4 phage <I>hoc</I> gene display vector. Gene 22 195 (2):303-11.</A></P><P>(2) <A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=2648397" class="external text" rel="nofollow">Heitman J., Zinder N. D., Model P. (1989): Repair of the Escherichia coli chromosome after in vivo scission by the EcoRI endonuclease. Proc Natl Acad Sci 86 (7):2281-5.</A></P><P>(3) <A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=10829079" class="external text" rel="nofollow">Datsenko K. A., Wanner B. L. (2000): One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proc Natl Acad Sci 97 (12):6640-5.</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:ETH_Zurich/Wetlab/Genome_Reduction">http://2008.igem.org/Team:ETH_Zurich/Wetlab/Genome_Reduction</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Wetlab/Genome_Reduction" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Wetlab/Genome_Reduction" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Genome_Reduction&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Genome_Reduction&amp;oldid=104821" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>