<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Project_Approach"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Project/Approach">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Project/Approach&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Project/Approach&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Project/Approach&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Project/Approach" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Project/Approach</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="background:#FFFFFF ; border:3.5px solid #60AFFE; padding: 1em; margin: auto ; width:98.5%"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Random_walk_towards_a_minimal_genome"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Random walk towards a minimal genome</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Implementation"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Implementation</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Modeling_framework"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Modeling framework</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Wetlab_proof_of_concept"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Wetlab proof of concept</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Random_walk_towards_a_minimal_genome"> Random walk towards a minimal genome </SPAN></H2><P>The concept we are trying to follow is based on a massively parallel random search. We have developed a top-down approach that will enable us to dramatically reduce the genome size of a living organism. As we described in the <A href="/Team:ETH_Zurich/Project/Background" title="Team:ETH Zurich/Project/Background">previous section</A> we are looking for a minimal subset of genes that are able to sustain life. The main problem when following this type of approach is the combinatorial possibilities that lead to a huge solution space, this corresponds to a classical NP-problem (non-polynomial) in computer science. The number of combinations of the original set of genes is given by:
</P><P>
This type of problem makes it infeasible to try all possible combinations sequentially and justifies a massively parallel approach as the one followed in this project.
</P><P>We propose the following algorithm as a method to find solutions to the presented problem:
</P><P>Evolutionary algorithm:
</P><P>1. Take the elements of current solution population (our populations in the chemostat).
</P><P>2. Mutate them according to some evolutionary rules (pulse restriction enzymes and ligases).
</P><P>3. Test the fitting of each solution (grow in continuous culture under selective conditions).
</P><P>4. Bring only the fitter ones to next generation.
</P><P>5. Go to step 1
</P><P>
The effect of this algorithm when implemented <I>in vivo</I> should be the following:
</P><P>
As can be seen in the previous figure, we start with our wild type genome and generate a large library of mutants by randomly deleting fragments of the chromosome as described in the <A href="/Team:ETH_Zurich/Wetlab/Genome_Reduction" title="Team:ETH Zurich/Wetlab/Genome Reduction">genome reduction</A> section.
</P><P>We then follow the new &quot;paths&quot; to test the fitness of our solutions and select for the ones that have successfully reduced the genome. These solutions are amplified by our continuous culture environment and represent the starting point for the subsequent round of mutations.
</P><P>By iterating this process we should be able to follow a biased &quot;random walk&quot; towards the minimal genome and gradually reduce the genome of our target organism.
</P><H2><SPAN class="mw-headline" id="Implementation"> Implementation </SPAN></H2><P>In order to prove the viability of the concept, we have developed a series of models and biological experiments that are introduced in this section. For details on the results of the single steps we refer to the specific parts in the <A href="/Team:ETH_Zurich/Wetlab/Overview" title="Team:ETH Zurich/Wetlab/Overview">wetlab</A> and the <A href="/Team:ETH_Zurich/Modeling/Overview" title="Team:ETH Zurich/Modeling/Overview">modeling</A> sections. The following figure gives an overview of the developed system and illustrates the final experimental setup.
</P><P>
We start by inoculating our continuous culture with a wild type strain with a knock out that makes it auxotroph for nucleotides. These nucleotides will be the limiting substrate in the chemostat and the concentration in our feed will be the parameter we can tune to set the selective pressure. See the <A href="/Team:ETH_Zurich/Wetlab/Chemostat_Selection" title="Team:ETH Zurich/Wetlab/Chemostat Selection"> chemostat selection</A> section for a detailed description of the selection process.
</P><P>The iteration of the following steps gradually reduces the size of the chromosome:
</P><P><B>1) Excision of genomic sequences by restriction enzymes and ligases.</B>  We have developed a <A href="/Team:ETH_Zurich/Modeling/Switch_Circuit" title="Team:ETH Zurich/Modeling/Switch Circuit">genetic circuit</A> that allows us to generate a controlled pulse to control the expression of restriction enzymes and ligases. Our assumption is that by carefully selecting the timing of the pulse, we can cut fragments of the chromosome, that in a number of cases will be lost when the chromosome is religated. To test this principle in vivo we have developed a proof of concept that will show the potential viability of this method <A href="/Team:ETH_Zurich/Wetlab/Genome_Reduction" title="Team:ETH Zurich/Wetlab/Genome Reduction">(genome reduction)</A>.
</P><P><B>2) Growth in continuous culture under selective pressure.</B> After the generation of the library of mutant we have to select for the strains that have successfully deleted part of the chromosome. Our <A href="/Team:ETH_Zurich/Modeling/Chemostat_Selection" title="Team:ETH Zurich/Modeling/Chemostat Selection">model</A> predicts the effectiveness of the devised approach and supports our assumptions regarding the applicability of this method.
</P><H2><SPAN class="mw-headline" id="Modeling_framework"> Modeling framework </SPAN></H2><P>A complete framework was developed to test our approach. We have implemented models of all steps of the presented method and integrated them in a framework that tries to capture the behavior of the system with the highest possible fidelity.
</P><P>The different models include:
</P><P>1) The genome <A href="/Team:ETH_Zurich/Modeling/Genome_Static_Analysis" title="Team:ETH Zurich/Modeling/Genome Static Analysis">static analysis</A>. Here we perform an exhaustive analysis of restriction enzymes based on a complete annotation of the e.coli K12 chromosome. We identify the different cutting characteristics and select for suitable candidates for subsequent simulations.
</P><P>2) A genome scale model of e.coli to perform <A href="/Team:ETH_Zurich/Modeling/Genome-Scale_Model" title="Team:ETH Zurich/Modeling/Genome-Scale Model">flux balance</A> analysis. With this part we simulate the effect of fragment deletions on viability and growth rate.
</P><P>3) A model of the <A href="/Team:ETH_Zurich/Modeling/Chemostat_Selection" title="Team:ETH Zurich/Modeling/Chemostat Selection">continuous culture</A> that takes the output of the FBA to simulate the selection mechanism.
</P><P>4) A dynamic model of the <A href="/Team:ETH_Zurich/Modeling/Switch_Circuit" title="Team:ETH Zurich/Modeling/Switch Circuit">pulse generator</A> circuit based on the SimBiology toolbox.
</P><H2><SPAN class="mw-headline" id="Wetlab_proof_of_concept"> Wetlab proof of concept </SPAN></H2><P>To assess the viability of our method in living organisms we have developed the following proofs of concept:
</P><P>1) <A href="/Team:ETH_Zurich/Wetlab/Genome_Reduction" title="Team:ETH Zurich/Wetlab/Genome Reduction">Genome Reduction</A>. Here we study the possibility of <I>in vivo</I> restriction and religation of the chromosome.
</P><P>2) Our selection mechanism is based on the possibility to select organisms with different chromosomal sizes by limiting nucleotide availability. To test this assumption we have performed a series of growth experiments, the results are shown in the section dedicated to the <A href="/Team:ETH_Zurich/Wetlab/Chemostat_Selection" title="Team:ETH Zurich/Wetlab/Chemostat Selection">chemostat selection</A> section.
</P><P>3) The controlled expression of restriction enzyme and ligase is of fundamental importance for our project. We present the results of the <A href="/Team:ETH_Zurich/Wetlab/Switch_Circuit" title="Team:ETH Zurich/Wetlab/Switch Circuit">pulse generator</A> implementation and the challenges we encountered.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:ETH_Zurich/Project/Approach">http://2008.igem.org/Team:ETH_Zurich/Project/Approach</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Project/Approach" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Project/Approach" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Project/Approach&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Project/Approach&amp;oldid=103412" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>