<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Wetlab_Overview"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Wetlab/Overview">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Wetlab/Overview&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Overview&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Overview&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Wetlab/Overview" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Wetlab/Overview</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="background:#FFFFFF ; border:3.5px solid #60AFFE; padding: 1em; margin: auto ; width:98.5%"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Genome_Reduction"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Genome Reduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Chemostat_selection"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Chemostat selection</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Switch_circuit"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Switch circuit</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Outline"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Outline</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>In order to approach our goal of creating an <I>E. coli</I> strain carrying a minimal genome, there are three main problems that have to be overcome:
</P><P><B>	Genome reduction:		show that <I>in vivo</I> restriction and religation is possible </B></P><P><B>	Chemostat selection:		introduce a limitation that confers a growth advantage to organisms with smaller genomes</B></P><P><B>	Switch circuit:		design a biobrick that provides for short-term synthesis of the desired gene products</B></P><H3><SPAN class="mw-headline" id="Genome_Reduction"><A href="/Team:ETH_Zurich/Wetlab/Genome_Reduction" title="Team:ETH Zurich/Wetlab/Genome Reduction">Genome Reduction</A></SPAN></H3><P>To prove that <I>in vivo</I> restriction and religation is possible is fundamental to our project which relies on short-term expression of a restriction enzyme and a ligase. While the restriction enzyme will randomly cut DNA, the simultaneous or shortly delayed synthesis of the ligase should religate the DNA. If the DNA is cut at several sites, religation will lead to exclusion of chromosomal fragments in a random manner.
</P><H3><SPAN class="mw-headline" id="Chemostat_selection"><A href="/Team:ETH_Zurich/Wetlab/Chemostat_Selection" title="Team:ETH Zurich/Wetlab/Chemostat Selection">Chemostat selection</A></SPAN></H3><P>In the continuous culture of a chemostat, those organisms with the highest rate of proliferation will overgrow those with a smaller growth rate. In order to bypass the need of selecting for those <I>E. coli</I> which have successfully reduced their genomes by massive screening of thousands of clones, we need to introduce a constraint that confers a growth advantage to organisms with smaller genomes. We have chosen to introduce mutations in the nucleotide synthesis pathway to achieve this goal. This will render DNA replication the rate-limiting step of proliferation and therefore be advantageous to organisms with small genomes.
</P><H3><SPAN class="mw-headline" id="Switch_circuit"><A href="/Team:ETH_Zurich/Wetlab/Switch_Circuit" title="Team:ETH Zurich/Wetlab/Switch Circuit">Switch circuit</A></SPAN></H3><P>Expression of restriction enzymes that cut genomic DNA inside the cell is likely to decrease viability. Actually, the <A href="/Team:Waterloo" title="Team:Waterloo">Waterloo iGEM team</A> is using restriction enzymes to kill cells in their project this year. Therefore, construction of a switch circuit, which allows to restrict expression of the restriction enzyme to a short period of time, is a crucial part of the project. The switch circuit allows expression of a gene under control of the lac repressor and rapid termination of lac-controlled expression despite presence of inducer by expression of an engineered LacI mutant, which is unresponsive to IPTG.
</P><P>
On the following pages, we will show a detailed description of how we are trying to achieve these three goals.
</P><H2><SPAN class="mw-headline" id="Outline"> Outline </SPAN></H2><TABLE border="1" cellpadding="20"><TBODY><TR><TD valign="top" align="center" width="450"><P><B>1)</B><A href="/Team:ETH_Zurich/Wetlab/Genome_Reduction" title="Team:ETH Zurich/Wetlab/Genome Reduction"><B>Genome Reduction</B></A></P><DIV style="text-align:justify;"><P><B>Questions:</B></P><UL><LI> Is <I>in vivo</I> restriction and religation possible without killing the cell?
</LI><LI> Does <I>in vivo</I> restriction and religation lead to the exclusion of chromosomal fragments?
</LI></UL><P><B>Method:</B>
For our proof of concept we ordered the above construct. Only if <I>in vivo</I> restriction by the endonuclease SceI and religation by the T4 ligase leads to the exclusion of the sacB sensitivity gene and the terminator, RFP will be synthesized. RFP-synthesizing cells can then be detected. Unfortunately, the construct has not arrived until today. Therefore, we are working on an alternative construct, which contains an RFP flanked by two SceI restriction sites. In this case, successful <I>in vivo</I> restriction and religation will lead to the loss of the RFP, which can also easily be detected.
<B>Results:</B>
We cloned a lac-inducible promoter in front of SceI and various inducible and constitutive promoters in front of the T4 ligase. During our attempts to design the construct coding for RFP flanked by SceI restriction sites, we managed to insert both restriction sites in front of and following the RFP by annealing oligonucleotide duplexes.</P></DIV></TD><TD valign="top" align="center" width="450"><P><B>2)</B><A href="/Team:ETH_Zurich/Wetlab/Chemostat_Selection" title="Team:ETH Zurich/Wetlab/Chemostat Selection"><B>Chemostat selection</B></A></P><DIV style="text-align:justify;"><P><B>Questions:</B></P><UL><LI> Do <I>E. coli</I> strains carrying differently sized genomes differ in growth rates?
</LI><LI> Can the growth rate of thymidylate synthase knockout strains be modified by regulating the external thymidine supply?
</LI><LI> Do thymidylate synthase knockout strains containing a reduced genome grow faster than strains carrying a larger genome under limiting thymidine concentrations?
</LI></UL><P><B>Method:</B>
In order to be able to examine the growth behaviors of <I>E. coli</I> strains carrying differently sized genomes, we ordered an <I>E. coli</I> strain (MDS42) whose genome had been reduced by 15 % using a targeted deletion approach. Also, we ordered the wild-type strain which the MDS42 had been derived from. In order to be able to keep track of the individual growth rates of the wild-type and MDS42 strains if grown in a mixed culture, we wanted to label these two strains. Finally, for knocking out the thymidylate synthase, we decided to use phage transduction.<B>Results:</B>
We successfully knocked out the thymidylate synthase, both in the wild-type and the MDS42 <I>E. coli</I> strains. Also, we managed to label the wild-type and the MDS42 <I>E. coli</I> strains by transformation of low-copy plasmids encoding different reporter proteins. Finally, we successfully performed growth experiments showing that the growth rate of thymidylate synthase knockout strains can be influenced by regulating the external thymidine supply.</P></DIV></TD></TR><TR><TD valign="top" align="center" width="450"><P><B>3)</B><A href="/Team:ETH_Zurich/Wetlab/Switch_Circuit" title="Team:ETH Zurich/Wetlab/Switch Circuit"><B>Switch circuit</B></A></P><DIV style="text-align:justify;"><P><B>Questions:</B></P><UL><LI> Can we construct a simple and at the same time robust pulse generator that allows pulses of variable duration without modification of the construct?
</LI><LI> Can we build a pulse generator that would work inside the chemostat?
</LI></UL><P><B>Method:</B>
We decided to build a simple pulse generator based on LacI IS mutants, which upon expression rapidly silence lac-controlled expression despite presence of IPTG. This allows us to induce expression with IPTG and terminate expression by addition of tetracyline. Since it is independent of removal of inducer by complete exchange of medium, it can be used inside the chemostat.<B>Results:</B>
Using PCR-based site-directed mutagenesis we generated eight different mutants of LacI and characterized them in a simple genetic experiment. As it turned out, all of them repressed lac-controlled expression even at high concentrations of IPTG. We constructed a pulse generator consisting of a tet-controlled LacI IS generator and a constitutive TetR expression cassette and demonstrated that leaky expression of the protein of interest as well as repression of lac-controlled expression due to leaky expression of LacI IS are not problematic. </P></DIV></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:ETH_Zurich/Wetlab/Overview">http://2008.igem.org/Team:ETH_Zurich/Wetlab/Overview</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Wetlab/Overview" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Wetlab/Overview" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Overview&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Overview&amp;oldid=103416" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>