"
Page
Discussion
View source
History
teams
Log in
 
Team:ETH Zurich/Modeling/Framework
From 2008.igem.org
Contents
1 Framework
1.1 Modeling Framework
1.2 Detailed description of the Modeling Framework
1.3 Summary of the Algorithm and Interplay of Frameworks Components
Framework
We propose a novel method of random gene deletion and chemostat-based selection of species with a reduced genome. For this we provide an algorithm described below.
Genome Reduction Algorithm
Modeling Framework
This algorithm requires a modeling framework consisting of four main parts:
Statistical Analysis of DNA fragments produced by Restriction Enzymes
Flux Balance Analysis on a Genome Scale Model
Growth Simulations in Chemostat
Switch circuit for short-time Restriction Enzyme Expression
Detailed description of the Modeling Framework
Restriction Enzyme Analysis
We want to identify a restriction enzyme for genome reduction that maximizes the probability of genome reduction per excision and minimizes the probability to hit an (known) essential gene.  Therefore, given a genome data of an E. coli strain, the genome is digested using ca. 700 different restriction enzymes (with different recognition patterns). The resulting fragments are analyzed using the available annotation of the genome. The number of genes disrupted is calculated for each fragment. Statistical measures of the restriction enzyme effects, such as average size of the fragments and its variance, and the average number of (all and only essential) genes and its variance are calculated. According to these results, the most suitable restriction enzyme is chosen.
Genome Scale Model and Flux Balance Analysis
We apply Flux Balance Analysis on the state-of-the-art genome scale model for E. coli iAF1260 (1260 genes included) in order to calculate the biomass yield (and therefore the corresponding growth rate) for wildtype and reduced strains. In order to apply a selective condition for reduced genome strain, the model is modified by constraining nucleotide availability: this means inactivation of a part of a thymidine synthesis pathway and external uptake of thymidine from the medium. The optimal external thymidine concentration is determined which enables the fastest change in growth rate of a reduced genome mutant.
Growth Simulations in Chemostat
We model a chemostat using a system of coupled differential equations for a population of different mutants with different growth rates competing for a limited external thymidine concentration. By varying the time intervals between two pulses of restriction enzymes (where only a small amount of previous population survived and new mutants are generated) we control the number of different mutants considering for further gene deletions after the next pulse event, consequently the diversity of a population.
Switch Circuit for short-time Restriction Enzyme Expression
Using a novel pulsing mechanism consisting of two signals – a start signal that initiates the synthesis of restriction enzymes, and a stop signal, which switches gene expression off - we are able to delete genome fragments in vivo. To simulate this system we developed and modeled a switch circuit which works as follows: protein gene expression is under the control of LacI and can be induced by the addition of IPTG. In order to stop gene expression, the IPTG-sensitive LacI is replaced by IPTG-insensitive LacIIs, which shuts restriction enzyme gene expression off again. The synthesis of LacIIs is started by the addition of Tetracyclin to the system, which binds to the tet repressor TetR and thus de-represses the expression of the LacIIs gene. This circuit is modeled by over 60 chemical reactions and is simulated in MATLAB.
Summary of the Algorithm and Interplay of Frameworks Components
First tree steps in the algorithm initialize and prepare the system for gene deletions and growth simulations.  In the first step the genome data are analyzed in order to find a most suitable restriction enzyme for random fragments deletion using the  “Restriction Enzyme Analysis”- procedure.  For the second step the state-of-the-art model is adjusted by introducing a selective pressure due to the genome size. Thirdly, initial population is set to consist of one type, namely wild-type.
The next steps of the algorithm perform in depth genome fragments deletion and growth simulation. First, we simulate the restriction enzyme expression and the consecutive population decline and occurrence of new mutants with reduced genome. The growth rates of the latter can be predicted using Flux Balance Analysis. For these simulations we used the framework parts: “Switch generator” and “Flux Balance Analysis on a Genome Scale Model “. Secondly, the growth simulations are performed using a chemostat model and the distribution of different mutant types according to the growth rates obtained. This simulation continues until no better mutants are being generated.
Eventually, the genome data of the fastest growing reduced genome mutant can be returned.
Retrieved from " http://2008.igem.org/Team:ETH_Zurich/Modeling/Framework "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
