<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Wetlab_Chemostat_Selection"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Wetlab/Chemostat_Selection">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Wetlab/Chemostat_Selection&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Chemostat_Selection&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Chemostat_Selection&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Wetlab/Chemostat_Selection" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Wetlab/Chemostat Selection</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="background:#FFFFFF ; border:3.5px solid #60AFFE; padding: 1em; margin: auto ; width:98.5%"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Chemostat_selection"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Chemostat selection</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Goal"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Goal</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Idea"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Idea</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Method"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Chemostat"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Chemostat</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Growth_constraint"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Growth constraint</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-7"><A href="#Lab_results"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Lab results</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Growth_experiments_with_E._coli_thymidine_auxotrophic_strain_from_Keio_library"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Growth experiments with E. coli thymidine auxotrophic strain from Keio library</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Growth_experiments_with_E._coli_strains_of_different_genomic_sizes"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Growth experiments with E. coli strains of different genomic sizes</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Labeling_of_E._coli_strains_of_different_genomic_sizes"><SPAN class="tocnumber">1.4.3</SPAN><SPAN class="toctext">Labeling of E. coli strains of different genomic sizes</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Thymidylate_synthase_knockout_using_phage_transduction"><SPAN class="tocnumber">1.4.4</SPAN><SPAN class="toctext">Thymidylate synthase knockout using phage transduction</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Growth_experiments_with_thymidylate_synthase_knockout_strains"><SPAN class="tocnumber">1.4.5</SPAN><SPAN class="toctext">Growth experiments with thymidylate synthase knockout strains</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Strains_used_in_the_experiments_of_this_section"><SPAN class="tocnumber">1.4.6</SPAN><SPAN class="toctext">Strains used in the experiments of this section</SPAN></A></LI><LI class="toclevel-4 tocsection-14"><A href="#Keio_thymidylate_synthase_knockout_strains"><SPAN class="tocnumber">1.4.6.1</SPAN><SPAN class="toctext">Keio thymidylate synthase knockout strains</SPAN></A></LI><LI class="toclevel-4 tocsection-15"><A href="#Phage_transduction_knockout_strains"><SPAN class="tocnumber">1.4.6.2</SPAN><SPAN class="toctext">Phage transduction knockout strains</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#References"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Chemostat_selection"> Chemostat selection </SPAN></H2><H3><SPAN class="mw-headline" id="Goal"> Goal </SPAN></H3><P>The goal of our project is to find the minimal genome that is able to sustain a living <I>E. coli</I> in a specific environmental condition. In the previous section, we have introduced the method we want to apply for reducing the genome. However, manually selecting for those bacteria that have successfully reduced their genomes is obviously impossible. Therefore, we have to find a mechanism that automatically selects cells possessing a reduced genome.
</P><H3><SPAN class="mw-headline" id="Idea"> Idea </SPAN></H3><P>Our idea is to set up a continuous bacterial culture in which cells with a reduced genome overgrow those maintaining more chromosomal DNA. 
</P><P>However, bacteria with a smaller genome do not automatically grow faster than those containing more DNA. This is due to the fact that the rate of proliferation is influenced not only by the replication velocity, but also by other factors such as protein synthesis, etc. Therefore, we have to introduce a constraint that renders DNA synthesis the limiting factor of the growth rate. In this scenario, reducing chromosomal DNA would speed up replication leading to fastest growth of those cells with the smallest genome. This way, we would not have to look for bacteria with reduced genomes, but we could let them do it for us!
</P><H3><SPAN class="mw-headline" id="Method"> Method </SPAN></H3><H4><SPAN class="mw-headline" id="Chemostat"> Chemostat </SPAN></H4><P>A chemostat is the instrument of choice for setting up a continuous culture. It is characterized by a continuous influx of medium, and an efflux of bacterial culture of the same volume. Since the volume of the continuous culture is kept constant, the growth rate of the population can be set by the dilution rate. Under these conditions, slowly growing (growth rate &lt; dilution rate) cells will be washed out of the culture, while the fastest growing clone will take the lead and finally dominate the entire culture.
</P><P>A model of the selection mechanism can be found in <A href="/Team:ETH_Zurich/Modeling/Chemostat_Selection" title="Team:ETH Zurich/Modeling/Chemostat Selection">Chemostat Selection</A>, where we estimate the initial parameters for optimal selection.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Figure 1: A chemostat</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Growth_constraint"> Growth constraint </SPAN></H4><P>As mentioned above, we want to introduce a constraint that renders DNA synthesis the limiting factor of the growth rate. DNA synthesis in bacterial cells is accomplished using the following biosynthetic pathway:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 2: DNA biosynthetic pathway</DIV></DIV></DIV></DIV><P>
The thymidylate synthase catalyzes the methylation of dUMP to yield dTMP. Phosphorylation then converts dTMP into dTTP, one of the four building blocks of DNA. In contrast to the other three nucleotides DNA is made of, dTTP is exclusively used for DNA, but not for RNA synthesis. Therefore, impairing the activity of the thymidylate synthase should interfere quite specifically with DNA synthesis.
</P><P>Our idea is to knockout the thymidylate synthase of the <I>E. coli</I> strain we use for genome reduction and then add thymidine to the continuous culture in a concentration that renders DNA synthesis the limiting factor for the growth rate. 
</P><P>The underlying assumption is that a thymidylate synthase knockout can no longer synthesize DNA from glucose and thus depends entirely on the availability of thymidine in the medium. Therefore, if the thymidine concentration is kept low, DNA replication should be slowed down. A decreased replication rate then results in a slower growth rate. This way, by fixing the thymidine supply rate, we can influence the growth rate of the knockout mutant.
</P><P>Supporting our assumptions, Escartin et al. showed that an <I>E. coli</I> strain expressing an enzymatically less active thymidylate synthase shows a significantly lower incorporation rate of radioactively labeled thymidine into DNA indicating a reduced replication velocity <A href="https://2008.igem.org/Team:ETH_Zurich/Wetlab/Chemostat_Selection#References" class="external text" rel="nofollow">(1)</A>. The same strain grows poorly in thymidine-deprived growth media. Other reports support that a decreased replication rate results in a slower growth rate and decreased replicative fitness <A href="https://2008.igem.org/Team:ETH_Zurich/Wetlab/Chemostat_Selection#References" class="external text" rel="nofollow">(2)</A>.
</P><H3><SPAN class="mw-headline" id="Lab_results"> Lab results </SPAN></H3><H4><SPAN class="mw-headline" id="Growth_experiments_with_E._coli_thymidine_auxotrophic_strain_from_Keio_library"> Growth experiments with <I>E. coli</I> thymidine auxotrophic strain from <A href="http://ecoli.naist.jp/gb6/Resources/deletion/deletion.html" class="external text" rel="nofollow">Keio library</A></SPAN></H4><P>Before we were able to perform the knockout of the <I>thyA</I> gene in our working strains (see section below), we decided to test the effect of external thymidine supply on growth rates by using an available thymidine auxotrophic strain. Our goal was to understand if growth rates could be controlled using different thymidine concentrations in the medium. We used the <I>thyA</I> knockout strain from the Keio library (5) and performed the experiment in a BioLector machinery. The Biolector instrument takes lightscattering online measuraments from a 96 wells titer plate every 5 minutes. The plate is continuously rotated while oxygen supply and humidity are kept constant. We devised an experiment in which different Keio strain populations were made to grow in different wells under different concentrations of thymidine. The series consisted of 8 different dilutions, with concentrations varying from 0.001, 0.050, 0.01, 0.5, 10, 50, 100 to 500 ug/ml. Our goal was to assess the strengths of the effects. We also included controls: the wild type strain was subjected to the very same concentrations as the auxotrophic strains without adding thymidine. The results are summarized in the growth profiles shown below.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Figure 3: Growth profiles of thymidine auxotrophic strain subjected to different thymidine concentrations.</DIV></DIV></DIV></DIV><P>The growth rates were minimal in all wells with 0.5 ug/ml thymidine or less. For the wells with 0.50 ug/ml thymidine or more, we found a regular growth pattern (partly due to errors of measurement caused by evaporation, and probably also due to thymidine which could have been left from the overnight colture). For the wells with a concentration of 0.10 ug/ml, we noticed a clear limitation in the growth rate resulting in a growth profile in between the other two conditions. This effect was confirmed by two technical replicates. The controls showed the expected behavior: the thymidine deprived auxotrophic strain did not show any growth response and wild type strains showed a regular growing profile for all thymidine concentrations.
These results led to two important conclusions: thymidine concentration could be used  effectively to control growth rate to a certain extent (extent yet to be fully characterized) and the experiment permitted us to identify the range of concentrations in which thymidine is supposed to be effective in controlling growth rates.
</P><H4><SPAN class="mw-headline" id="Growth_experiments_with_E._coli_strains_of_different_genomic_sizes"> Growth experiments with <I>E. coli</I> strains of different genomic sizes </SPAN></H4><P>To test if we can select for strains with smaller genome sizes in a chemostat setup, we have chosen to compare growth behaviors (growth rate and thymidine yield) of strains with different genome sizes.
For this purpose we used MG1655 and MDS42, two strains with the same genetic background, but with different genome sizes. MDS42 has a 14.3% smaller genome size than MG1655. Both strains were ordered at <A href="http://www.scarabgenomics.com/" class="external text" rel="nofollow">ScarabGenomics </A>. 
</P><TABLE style="width:90%; border=0;background-color: lightgrey;text-align: center;"><CAPTION> MG1655 vs MDS42 <A href="https://2008.igem.org/Team:ETH_Zurich/Wetlab/Chemostat_Selection#References" class="external text" rel="nofollow">(4)</A></CAPTION><TBODY><TR><TH> MG1655 </TH><TH> MDS42
</TH></TR><TR><TD> Total number of genes
</TD><TD> 4434
</TD><TD> 3730
</TD></TR><TR><TD> Genome size (bp)
</TD><TD> 4639675
</TD><TD> 3976359
</TD></TR><TR><TD> Total no. genes deleted
</TD><TD> 0
</TD><TD> 704
</TD></TR><TR><TD> Total bp DNA deleted
</TD><TD> 0
</TD><TD> 663316
</TD></TR><TR><TD> Percent of genome deleted
</TD><TD> 0
</TD><TD> 14.30%
</TD></TR></TBODY></TABLE><P>For a first characterization of the growth behaviors of these two strains, we performed growth experiments in LB medium.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 4: MG1655 and MDS42 were cultivated in 500ml baffled flasks with 100ml LB medium at 37°C and 220 rpm. Cell growth was assessed by measuring the optical density at 600 nm.</DIV></DIV></DIV></DIV><P>We observed a higher growth rate for MG1655 than for MDS42 in LB medium, while both grew to the same cell density.
</P><H4><SPAN class="mw-headline" id="Labeling_of_E._coli_strains_of_different_genomic_sizes"> Labeling of <I>E. coli</I> strains of different genomic sizes </SPAN></H4><P>Since individual growth rates of two strains in the same environment cannot be assessed by measuring the optical density, we decided to label them with two different fluorescent proteins. Our initial idea was to integrate these fluorescent proteins into the chromosomes of MG1655 and MDS42 using the lambda red recombination system established by Wanner and Datsenko (6). However, after successfully generating the GFP- and RFP-encoding PCR products needed for recombination, we repeatedly failed to integrate the PCR products into the chromosomes. Therefore, we opted for labeling via plasmids.
</P><P>To this end we have constructed a RFP generator and a GFP generator on low-copy plasmids (pCK01) and transformed them into MG1655 and MDS42. When labeling strains with different genome sizes to compare genome size specific behavior, it is important to use plasmids with the same copy-number. This is to keep the relative differences in nucleotide usage the same. We have chosen to use a low-copy plasmid to keep the added protein burden low. 
To ensure that the fluorescence signal is proportional to the cell concentration, we compared measurements of the optical density with the fluorescence signals. To this end, we measured the fluorescent signals of a dilution series of RFP-labeled MG1655 and GFP-labeled MDS42 with a <A href="http://las.perkinelmer.com/Catalog/ProductInfoPage.htm?ProductID=1420-050" class="external text" rel="nofollow">fluorescence reader</A>. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 5: We optimized the fluorescence reader to measure the amount of our fluorescent proteins by using specific emission and excitation filters. A dilution series of different amounts of cells show the linear correlation to the optical density of cells.</DIV></DIV></DIV></DIV><P>The fluorescent signal for both fluorescent proteins, RFP and GFP, showed a linear response between OD600 0.5 and 6, which is the range of optical density important to follow the growth of <I>E. coli</I>.
</P><P>In order to assure a correlating signal also during growth of <I>E. coli</I>, we followed the growth of MG1655 labeled with GFP by measuring the optical density and, at the same time, the fluorescence signal.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 6: MG1655 labeled with GFP were grown in 500ml shake flasks with 100ml LB medium at 37°C and 220rpm. Growth was assessed by measuring the optical density at 600nm and fluorescence intensity.</DIV></DIV></DIV></DIV><P>The expression of GFP in MG1655 correlates well with the measured optical density.
</P><H4><SPAN class="mw-headline" id="Thymidylate_synthase_knockout_using_phage_transduction"> Thymidylate synthase knockout using phage transduction </SPAN></H4><P>The knockouts of <I>thyA</I> in MG1655 and MDS42 were performed by P1 vir transduction <A href="https://2008.igem.org/Team:ETH_Zurich/Wetlab/Chemostat_Selection#References" class="external text" rel="nofollow">(3)</A>. For this purpose, we used a <I>thyA</I> mutant strain from the <A href="http://ecoli.naist.jp/gb6/Resources/deletion/deletion.html" class="external text" rel="nofollow">Keio collection</A> as donor cells and transferred its kanamycin-resistence gene into MG1655 and MDS42. The mutants from the Keio collection are single knockouts which each have one single gene replaced by a kanamycin-resistence gene. 
Different verification methods were performed to verify successful knockout of <I>thyA</I> in MG1655 and MDS42:
</P><UL><LI> We selected the transduced strains on kanamycin-plates. 
</LI><LI> We performed an XGal-test to distinguish colonies of the donor cells (Keio <I>thyA</I> mutant) from the colonies of the receptor cells (MG1655 and MDS42). Since BW25113, the strain of the Keio collection, is lac- (∆lacZ4787(::rrnB-3)) and MG1655 and MDS42 are lac+, XGal-screen is a fast method to distinguish residual donor cells in the phage lysate from the receptor cells. 
</LI><LI> We verified the knockout by growth on minimal glucose medium with and without supplemented thymidine (40ug/ml). On agar plates as well as in liquid 5 ml O/N cultures the mutants grew in medium with thymidine, but did not grow in medium without thymidine. 
</LI><LI> For further verification we will perform colony PCR of the mutants.
</LI></UL><H4><SPAN class="mw-headline" id="Growth_experiments_with_thymidylate_synthase_knockout_strains"> Growth experiments with thymidylate synthase knockout strains </SPAN></H4><P>We performed growth experiments with the knockout mutants in minimal glucose medium with different concentrations of supplemented thymidine.
</P><P>At a thymidine concentration of 80 ug/ml, MG1655 thyA- and MDS42 thyA- show similar growth behaviors.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 7: MG1655 thyA- and MDS42 thyA- were grown in 500ml baffled shake flasks with 100ml minimal glucose medium supplemented with 80 ug/ml thymidine at 37°C and 220rpm. Growth was assessed by measuring the optical density.</DIV></DIV></DIV></DIV><P>On the other hand, at a thymidine concentration of 10 ug/ml, MDS42 thyA- grows to a much higher cell concentration.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 8: MG1655 thyA- and MDS42 thyA- were grown in 500ml baffled shake flasks with 100ml minimal glucose medium supplemented with 10 ug/ml thymidine at 37°C and 220rpm. Growth was assessed by measuring the optical density.</DIV></DIV></DIV></DIV><P>This shows that at limiting thymidine concentrations MDS42 thyA-, which has a reduced genome size, has a growth advantage in comparison to MG1655 thyA-. This effect might select for cells with smaller genome sizes in a chemostat setup. 
</P><P>In future work, MG1655 thyA- and MDS42 thyA- will be labeled with RFP and GFP, respectively, to follow outgrow of one strain by the other when cultivated together in a chemostat.
</P><H4><SPAN class="mw-headline" id="Strains_used_in_the_experiments_of_this_section"> Strains used in the experiments of this section </SPAN></H4><H5><SPAN class="mw-headline" id="Keio_thymidylate_synthase_knockout_strains"> Keio thymidylate synthase knockout strains </SPAN></H5><UL><LI> BW25113:  ∆(araD-araB)567, ∆lacZ4787(::rrnB-3), lambda-, rph-1, ∆(rhaD-rhaB)568, hsdR514
</LI></UL><H5><SPAN class="mw-headline" id="Phage_transduction_knockout_strains"> Phage transduction knockout strains </SPAN></H5><UL><LI> MG16555 thyA: F-, lambda-, rph-1, thyA- 
</LI><LI> MDS42 thyA: F-, lambda-, rph-1, thyA-
</LI></UL><H3><SPAN class="mw-headline" id="References"> References </SPAN></H3><P>(1) <A href="http://www.pnas.org/content/105/29/9948" class="external text" rel="nofollow">Escartin F., Skouloubris S., Liebl U., Myllykallio H. (2008): Flavin-dependent thymidylate synthase X limits chromosomal DNA replication. Proc Natl Acad Sci 22 105(29):9948-52.</A></P><P>(2) Helmstetter C. (1996): Timing of synthetic activities in the cell cycle. Escherichia coli and
Salmonella: Cellular and Molecular Biology. eds Neidhardt FC et al. (Am Soc Microbiol,
Washington D. C.): 1627–1639.
</P><P>(3) <A href="http://mrw.interscience.wiley.com/emrw/9780471142720/cp/cpmb/article/mb0117/current/html" class="external text" rel="nofollow">Lynn C. et al. (2007): E. coli Genome Manipulation by P1 Transduction. Current protocols in molecular biology</A></P><P>(4) <A href="http://www.sciencemag.org/cgi/content/full/312/5776/1044" class="external text" rel="nofollow">Pósfai G, Plunkett G 3rd, Fehér T, Frisch D, Keil GM, Umenhoffer K, Kolisnychenko V, Stahl B, Sharma SS, de Arruda M, Burland V, Harcum SW, Blattner FR. Emergent Properties of Reduced-Genome <I>Escherichia coli</I>. Science. 2006 May 19;312(5776):1044-6. Epub 2006 Apr 27.</A></P><P>(5) Baba et al, Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection, Mol Syst Biol. 2006; 2: 2006.0008.
</P><P>(6) <A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=10829079" class="external text" rel="nofollow">Datsenko K. A., Wanner B. L. (2000): One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proc Natl Acad Sci 97 (12):6640-5.</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:ETH_Zurich/Wetlab/Chemostat_Selection">http://2008.igem.org/Team:ETH_Zurich/Wetlab/Chemostat_Selection</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Wetlab/Chemostat_Selection" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Wetlab/Chemostat_Selection" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Chemostat_Selection&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Chemostat_Selection&amp;oldid=105133" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>