"
Page
Discussion
View source
History
teams
Log in
 
Team:ETH Zurich/Wet Lab/Protocol
From 2008.igem.org
Home
Design & Implementation
Background
Applications
Introduction
Wet Lab
Protocol
Modelling Conclusions Team Page Organization
Introduction of GFP/RFP into MG1655/MD42 E. coli strains
using the Lambda Red System
 
Subcloning of GFP/RFP into pCK01 followed by PCR amplification
 
Day 1: Friday, July 11
digest p18GFP/BB_J14461RFP plasmids and pCK01 2.5 h
25 µL GFP construct/RFP construct/pCK01 plasmid
1 µL EcoRI
1 µL PstI
3 µL EcoRI buffer
 
→ 2 h at 37°C
→ pCK01 digest was heat-inactivated for 20 min at 80°C
 
run gel to confirm digest 1 h
0.8 % agarose gel:
lane 1 (large!) + 2: 26 µL RFP digest + 6 µL loading buffer (LB)
lane 3: 4 µL RFP digest + 2 µL LB
lane 4: 2.5 µL λ marker + 2 µL LB
lane 5 (large!): 26 µL GFP digest + 6 µL LB
lane 6: 4 µL GFP digest + 2 µL LB
 
→ run at 90 V for ca. 1 h
→ lanes 3/4 and lane 6 cut out and put in EtBr for ca. 30 min
 
pics (4 s exposure) saved under GFP_RFP_restr
 
cut out GFP/RFP bands, recover DNA 1 h
RFP showed 2 bands: one at ca. 1000 bp and one higher
(ca. 1000 bp + higher expected)
GFP showed 3 bands: one at ca. 1000 bp, one higher and one lower
(ca. 500, 1000 and 2200 bp expected)
 
→ bands at 1000 bp were cut out (ca. 250 mg)
 
gel extraction was performed according to protocol:
750 µL of buffer QG
250 µL isopropanol
elution with 30 µL EB buffer (wait 1 min before elution!)
 
ligate recovered DNA with digested pCK01 O/N
3 µL pCK01 vector
1 µL T4 buffer
0.5 µL T4
5.5 µL GFP insert/RFP insert/ water (control)
 
                        → spin down and put to 4°C O/N
 
Day 2: Saturday, July 12 (5 pm)
transform ligations (3 x) and GFP/RFP/pCK01 plasmids 2 h
(to get backups): 6 transformations!
3 ligations transformed:
10 µL ligation (GFP/RFP/control)
40 µL competent cells
3 backup plasmids transformed:
2 µL p18GFP/BB_J14461 RFP/pCK01 plasmids
40 µL competent cells
 
→ 20 min on ice
→ 90 sec at 42°C
→ 90 sec on ice
→ 200 µL CVM added
→ 1 h recovery at 37°C and 220 rpm
 
plate out transformations (6 x) and incubate at 37°C O/N
3 ligations (GFP-pCK01, RFP-pCK01, control) plated on
LB + chloramphenicol (cat) + x-gal agar plates
pCK01 plated on LB + chloramphenicol agar plates, p18GFP and BB_J14461 RFP plated on LB + ampicillin agar plates
 
→ put to 37°C O/N
 
Day 3: Sunday, July 13 (5 pm)
pick colonies and inoculate over night cultures: O/N
1 colony of GFP/RFP/pCK01 and lambda red
+ 2 x 5 of GFP-pCK01/RFP-pCK01 → 14 cultures
one GFP/RFP/lambda red colony was inoculated in 15 mL-Falcon
tubes containing 5 mL LB medium + ampicillin
5 colonies of GFP-pCK01/RFP-pCK01 were inoculated in 5 mL
LB medium + chloramphenicol
 
→ put to 37°C O/N
 
Day 4: Monday, July 14 (9 am)
do mini preps of GFP-pCK01, RFP-pCK01 (3 x each) and GFP/RFP/pCK01/lambda red plasmids (10 mini preps):
eluted in 50 µL EB buffer after 1 min incubation at RT
 
digest and gel to check cloning:
GFP/RFP plasmids digested:
                    2 µL DNA
                    1 µL EcoRI buffer
                    0.5 µL EcoRI
                    0.5 µL PstI
                    6 µL water
 
GFP-pCK01/RFP-pCK01/pCK01 plasmids digested:
                    8 µL DNA
                    1 µL EcoRI buffer
                    0.5 µL EcoRI
                    0.5 µL PstI
 
                    → put to 37°C for 2 h
 
run on 0.8 % gel:
                    add 2 µL LB
                    load 10 µL of each
                    run at 100 V
                    take pictures
 
                    → hardly any DNA visible
 
Inoculation of an additional 3 colonies of GFP-pCK01 and RFP-pCK01
inoculated in 5 mL chloramphenicol containing LB medium
incubated at 37°C O/N
 
Day 5: Tuesday, July 15
GFP-pCK01, RFP-pCK01 (3 x each) and GFP/RFP/pCK01/lambda red plasmids were run on 0.8% gel without digestion
          → double bands (several colonies picked?!)
Mini prep of 2 x 3 GFP-pCK01/RFP-pCK01 cultures
mini preps (2 x 2) were performed and run on 0.8% gel
                    → 3/4 positive
 
Day 6: Wednesday, July 16
do PCR of GFP-pCK01 and RFP-pCK01 constructs using the ordered primers
primers were dissolved in xxx µL/xxx µL ddH2O (forward/reverse)
Master-Mix (MM):
                    40 µL 5 x Taq buffer
                    32 µL 25 mM MgCl2
                    2 µL ATP
                    2 µL GTP
                    2 µL CTP
                    2 µL TTP    
                    8 µL forward primer (100 µM)
                    8 µL reverse primer (100 µM)
                    8 µL Taq polymerase
                    272 µL ddH2O
3 different amounts of GFP-pCK01 (no. 2, 15.07.08)/RFP-pCK01 (no. 1, 15.07.08) template were used:
                    49 µL MM + 1 µL template
                    48 µL MM + 2 µL template
                    46 µL MM + 4 µL template
 
                    → 6 PCRs
 
PCR program:
                    5 min 95°C
 
                    35 x
                    1:50 min 95°C
                    1:50 min 50°C
                    2 min 72°C
                    
                    5 min 72°C
                    4°C after finish; running time: ca. 4:30 h
 
check and purify PCR product on gel
10 µL LB added to all 6 PCR products
6 µL loaded for checking
rest (ca. 50 µL) loaded in large pockets for extraction
6 µL lambda marker loaded (3 µL marker + 3 µL LB)
run at 100 V on 0.8% gel
            
                → no bands, PCR needs to be repeated!
 
Day 7: Thursday, July 17
repitition of PCR with changed protocol
 2.5 µL high fidelity buffer (with MgCl2, no.2)
                     1 µL primer up (10 µM!)
                     1 µL primer down (10 µM!)
                     0.5 µL dNTPs (10 µM of each dATP, dGTP, dCTP, dTTP; is aliquoted)
                     0.5/1 µL template (GFP-cat 15.07.08 no. 3, RFP-cat 15.07.08 no. 1)
                     0.5 µL high fidelity polymerase 
                     19 µL ddH2O
 
                     → 4 PCRs a 25 µL
 
program:
                    5 min 94°C
                    
                    1 min 95°C
                    1 min 55°C
                    2 min 72°C
                    → 25 x
 
                    10 min 72°C
                    4°C till end
 
0.8% gel to check and extract PCR product
5 µL LB was added to each
run at 100 V
 
                    → bands at 2500 kb for all 4 samples
 
gel extraction
gel extraction was performed using 1100 µL buffer QG and 350 µL isopropanol                 
samples were eluted with 30 µL EB buffer (wait 1 min before centrifugation!)
 
Day 8: Friday, July 18
run extracted PCR products on 0.8 % gel to check extraction and size
Agarose gel electrophoresis of PCR products (2µl 100 bp Fermentas DNA Marker; 3µl PCR product GFP 0.5, GFP 1, RFP 0.5, RFP 1)
PCR product is larger than 1kb, yield exceeds 30ng/ul
 
                    → clear bands for all 4 PCR products!
 
<img alt="" id="khah" style="width: 389px; height: 211px;" src=" http://docs.google.com/File?id=ddsp7zr4_1fkdvbchq_b " />
→ ready for transformation into lambda red-transformed, competent MG1655/MD42 strains (see below)
 
 
 
Preparation of competent MG1655 and MD42 strains for introduction of GFP/RFP constructs
 
Day 1: Thursday, July 10
plate out glycerol stocks of MG1655 and MD42 done
 
scrape off some of icy glycerol stock and plate on LB agar
 
→ incubate at 37°C O/N
 
Day 2: Friday, July 11
put colonies to 4°C
colonies have grown and are ready to be picked!
 
Day 3: Monday, July 14
prepare competent MG1655/MDS42 cells
 2 colonies of MG1655 and MDS42 each inoculated in 5 mL LB medium (5 pm)
incubated at 37°C O/N
 
Day 4: Tuesday, July 15
prepare competent MG1655/MDS42 cells (continued)
50 mL LB medium were inoculated with 500 µL over night cultures and grown to OD(600) 0.488 (MG1655) and 0.503 (MDS42)
                    (MG1655 grew faster and was stored on ice while MDS42 was still growing)
cultures were transfered to 50 mL Falcon tube (on ice!) and spun down for 10 min at 2500 rpm and 4°C
discard supernatant
pellet was resuspended in 50 mL buffer 1 (pre-cooled!)
store on ice for 90 min
spin 10 min at 2500 rpm and 4°C
discard supernatant
resuspend pellet in 2 mL ice cold buffer 2
200 µL each were filled in 1.5 mL Eppis and stored at -80°C
 
transform competent MG1655/MDS42 cells with lambda red constructs (pKD46)
2 µL pKD46 were transformed into 50 µL of competent MG1655/MDS42 according to the protocol
heat shock at 42°C
200 µL CVM added
recovery at 30°C
plated on ampicillin containing agar plates at 30°C (!) O/N
 
 Day 5: Wednesday, July 16
prepare competent lambda red MG1655/MD42 pKD46 cells
inoculate 2 colonies of each MG1655/MDS42 pKD46 in 5 mL ampicillin containing LB medium
incubate at 30°C O/N at 200 rpm
 
Day 6: Thursday, July 17
prepare competent lambda red MG1655/MD42 pKD46 cells (continued)
protocol as described on day 4 (with MG1655/MD42 pKD46 cells!)
grown to OD(600) of 0.458 (MG1655)/0.536 (MDS42)
                    Again, MG1655 grew faster!
cells were stored on ice for 20 min before 1st centrifugation
aliquoted in 2 x 11 Eppis a 200 µL and stored at -80°C
 
Day 7: Friday, July 18
 
transform MG1655/MD42 pKD46 cells with GFP-pCK01/RFP-pCK01 PCR products (see above)
temperature shock transformation of MG1655/MD42 pKD46 cells with GFP-pCK01/RFP-pCK01 PCR products according to protocol:
                    For the transformation, 50ul of competent cells and 2ul of PCR product (amounting to approximately 100 ng) were used and streaked onto LB-Cat plates.
                    Incubation at 30°C for 12 hours to three days did not yield any colonies.
 
                    
                    → Transformation has failed and needs to be repeated.
 
Day 8: Tuesday, July 22
 
Repetition of PCR product transformations:
1 µL/5 µL/10 µL of GFP-cat PCR product (0.5) and 2 µL pCK01 (control) were transformed in 45 µL MG1655 + pKD46 cells
1 µL/5 µL/10 µL of RFP-cat PCR product (0.5) and 2 µL pCK01 (control) were transformed in 45 µL MDS42 + pKD46 cells
10 min on ice, heat shock at 42°C, recovery with 200 µL CVM at 37°C, plated on chloramphenicol containing agar plates, incubation at 37°C O/N
Day 9: Wednesday, July 23
Picking colonies for O/N cultures of MG1655/MD42 pKD46 in 5mL LB medium with ampicillin.
Day 9: Thursday, July 24
    Production of competent cells with induction of lambda-red-recombination system:
Inoculation with 500 uL O/N culture of MG1655/MD42 pKD46 in 50mLB medium with ampicillin and 500 uL 1M Arabinose. \
growth until OD(600nm) = ~ 0.4 in 30C! shaker (MG1655 grew slightly faster and were put on ice 15 min earlier).
cells on ice for 30 min
centrifuge cells for 10 min, 2500 rpm, 4°C, discard supernatant
wash cells twice with 10 mL ice-cold ddH2O and twice with 10 mL 10% Glycerol.
cells resuspended in 500 µL 10 % glzcerol and aliquoted to 50 µL per Eppi
stored at -80°C
    
    Transformation with insert through electroporation:
add to 50 uL competent cells (MG1655/MD42 pKD46) 5 uL/10 uL insert (GFP-cat-1/RFP-cat-1 from PCR of 17.7.08)
<tbody id="fz.:0">
 
Strain
Insert
 
 
 
Amount
Which
1
MG1655
5 uL
GFP-cat
2
MG1655
10 uL
GFP-cat
3
MD42
5 uL
RFP-cat
4
MD42
10 uL
RFP-cat
</tbody>
put into ice-cold electroporation-tube
pulse with 1.25 kV, add immediately 1 mL LB
put into a sterile epi and recover in 37C shaker for 1 h
centrifuge 1 min, 12 000 rpm, discard supernatant, resuspend in residual medium
plate on Cat-plates and incubate O/N at 37C
Day 10: Friday, July 25
    Result: Transformation has failed and needs to be repeated.
Day 11: Monday, 28 July
    Repetition of transformation through electroporation incl. control
add to 50 ul competent cells (MG1655/MD42) from 24. 6.08 insert (1 uL pUCP26, contains tet-resistence or 3ul RFP-cat/GFP-cat)
electroshock at 1.25kV, add immediately 1mL LB + 10 uL 1M arabinose
recover in 37C shaker for 3 hours
plate 50% and incubate O/N at 37C
let 50% of cells on the bench in 1ml LB O/N
   
<tbody id="o4ny0">
 
Strain
Insert
Electro shock time
Plate Type
1
MG1655
pUCP26
6ms
Tet
2
MD42
pUCP26
5.9ms
Tet
3
MD42
RFP-cat (0.5 PCR 17.7.08)
5.8ms
Cm
4
MG1655
GFP-cat (0.5 PCR 17.7.08)
5.9ms
Cm
</tbody>
Day 12: Tuesday, 29 July
    Results from electroporation from 28.7.08
control strains (transformation with pUCP26) are grown: => electroporation has been successful
GFP/RFP-cat insertion failed again => integration of insert into chromosome has failed
    Test of Insertion of GFP/RFP into Target gene TnaA
one colony found on plate from 24.7.08, MD42 transformed with 10uL RFP-cat 
inoculation of colony in 5 ml LB with Chloramphenicol, and incubation in 37C shaker (start 11am - about 3/4pm until OD = 2)
Tryptophan-Color-Test: positive ! addition of Kovac's Regent for indoles stayed yellow, and didnt turn red. Therefore TnaA is inactive, probably by insertion of RFP-cat.
Fluorescence microscopy: clearly moving particles were visible under LM, some red dots under FM
Inoculation of O/N culture in 5 ml LB with Chloramphenicol. 
Incubation of Backup plates from the one colony found in 37C oven.
    Growth experiment of MG1655/MD42 in LB
    Inoculate MG1655/MD42 in 5 ml LB 37 shaker over night.       
Day 13: Wednesday, 30 July
    Test of Insertion of GFP/RFP into Target gene TnaA
Indol-test with O/N culture from 29.7.08 OD = 4.605 again positiv: Medium stayed yellow upon addition of Kovac's Regent for indoles.
to do: FL after 2h in freezer
to do: Primer design for PCR to verify insertion
    Growth experiment of MG1655/MD42 in LB
Inoculation of 100ml media with O/N culture to OD(600) = 0.05: 2.358 mL MG1655 (preculture with OD(600) = 2.12) and 1.976 mL MD42 (preculture with OD(600) = 2.53)
OD measurements:
    
<img width="450" height="320" alt="" id="mqen" src=" http://docs.google.com/File?id=dhf5kqn7_5cx2xg4d6_b " />
<img alt="" id="e_n9" style="width: 450px; height: 320px;" src=" http://docs.google.com/File?id=dhf5kqn7_8cm2vzghd_b " />
=> max growth rate:
MG1655: u = 1.57 1/h
MDS42: u = 1.10 1/h
=> max OD600
MG1655: 4.55
MDS42: 4.55
    Digestion of Insert and pKD46
digestion of 8.5 uL pKD46(MG)/pKD46(MD) in 1 uL EcoR I Buffer and 0.5 uL EcoR I
digestion of 3 uL RFP-cat/GFP-cat in 5.5 uL ddH2O and 1 uL Buffer 0 and 0.5 uL Pst I
Digestion for 2h in 37°Oven
load with 2 uL LB, from left to right: lambda marker, pkD42 MG, pKD42 MD, GFP MG, RFP MD.
<img id="a6lq0" alt="" /><img id="a6lq1" alt="" /> <img width="137" height="333" alt="" id="w4fx" style="margin: 1em 0pt 0pt 1em; float: left;" src=" http://docs.google.com/File?id=dhf5kqn7_3ff2dk4ct_b " />
<img width="138" height="332" alt="" id="i5cz" src=" http://docs.google.com/File?id=dhf5kqn7_4c5wm22fb_b " />
 
Appendix
 
<img border="0" id="jgfp3" title="GeneRuler™ & O'GeneRuler™ 100 bp DNA Ladders" style="width: 300px; height: 434px;" alt="" src="
" />
Retrieved from " http://2008.igem.org/Team:ETH_Zurich/Wet_Lab/Protocol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
