"
Page
Discussion
View source
History
teams
Log in
 
Team:ETH Zurich/Wetlab/Genome Reduction
From 2008.igem.org
Contents
1 Genome reduction
1.1 Goal
1.2 Method
1.3 Proof of concept
1.3.1 Extracting modelling parameters from the proof of concept experiment
1.3.2 Modification of proof of concept
1.3.3 SceI restriction enzyme
1.3.4 T4 ligase
1.4 Lab results
1.4.1 Modified proof of concept
1.4.2 SceI restriction enzyme and T4 ligase
1.5 References
Genome reduction
Goal
In this section we discuss about how to randomly delete chromosomal fragments of E. coli in order to reduce the physical amount of genomic DNA.
Method
To reach our goal of random deletion of chromosomal fragments, we want to synthesize a frequently cutting restriction enzyme along with the simultaneous, shortly delayed, or even continuous synthesis of a ligase. The restriction enzyme will cut genomic DNA in a random fashion in vivo, while the ligase performs its job of religation. Assuming that the genomic DNA is cut at several sites within one cell, religation will lead to the exclusion of chromosomal fragments in some cells. Multiple rounds of restriction and religation will therefore lead to a markedly reduced genome.
Proof of concept
In 1997 Ren et al. showed that in vivo religation of linearized vector and insert is possible by overexpression of the T4 ligase (1) . Additionally, Heitman et al. proposed that intracellular DNA which has been cut in vivo by EcoRI is religated by the ligase T4 (2) . Our idea, however, relies on the assumption that in vivo restriction and religation is possible and leads to the exclusion of chromosomal DNA. We are trying to verify these assumptions in several experiments which will from now on be refered to as our “proof of concept”.
Our proof of concept relies on the following construct (not shown are ribosomal binding sites behind both SceI restriction sites and a terminator following the RFP):
Figure 1: Construct for proof of concept
The above construct was ordered at GeneArt and is supposed to be integrated into the genome of a wild-type E. coli strain (MG1655). We are planning to use the lambda red recombination system established by Wanner and Datsenko (3) to integrate our construct into the tryptophanase A gene resulting in the knockout of the latter.
Additionally, we ordered DNA encoding the T4 ligase and the SceI restriction enzyme. Transformation of these plasmid-encoded enzymes into cells carrying the proof of concept construct is supposed to yield bacteria that can be induced to express T4 and SceI.
Before synthesis of T4 and SceI, cells carrying the proof of concept construct do not synthesize RFP. After synthesis of the restriction enzyme, SceI will cut the bacterial chromosome at the sites indicated above. The ligase will then religate the construct, leading to the exclusion of the sensitivity gene and the terminator in some of the cells. Hence, these cells would synthesize the RFP reporter protein and could easily be identified:
Figure 2: Proof of concept
Additionally, adding sucrose to the medium would kill all cells that have not eliminated the sacB sensitivity gene. Hence, only RFP-synthesizing bacteria would survive, which would presumably facilitate detection.
Extracting modelling parameters from the proof of concept experiment
A very important parameter in our reduction strategy approach is undoubtly the efficiency on which the process of cutting, fragment removal and re-ligation is going to occure in-vivo. Our proof of concept construct and experiment were designed in order to provide us two important indexes of efficiency, namely the cutting efficiency (CE) and the removal and ligation efficiency (RLE). The CE parameter represents the probability that at least one cut is made into the chromosome of a cell for a pulse of expression of the restriction enzyme. The RLE represents the probability that after a cutting a fragment is lost and the chromosome is relagated. This efficiency frequencies are going to help us understand the behaviour of the single cell when expression of the restriction enzyme is induced. In order to calculate the two parameters from the experiment we planned to:
take pictures of the plate with the microscope before inducing the expression of restriction enzyme and, manually or with help of specialized software, estimate the total number of cells in the population (TOT_POP)
after having induced the restriction enzyme expression and the ligase, take pictures of the plate in a similar way and extimate: the number of cells left (SURVIVED_POP) and the number of cells showing fluorescence (FLUO_POP).
From these three population counts we can set up the system of two equations:
After solving the equations we can use the parameters for modelling the probability of one fragment to be removed, as we used in the modelling section Genome Scale Model .
Modification of proof of concept
Unfortunately, up to now we have not received our construct for the proof of concept. Therefore, we are working on a modified construct:
Figure 3: Modified construct for proof of concept
For this purpose, a commonly used RFP reporter plasmid is used. SceI restriction sites are introduced by cutting the plasmid in front of and after the RFP gene and ligating oligonucleotide duplexes containing the desired restriction sites.
After transformation of bacteria with the construct shown above and plasmids containing the genes coding for SceI restriction enzyme and the T4 ligase, induction of gene expression will lead to in vivo cutting and religation of the RFP-encoding plasmid, leading to the loss of the RFP gene from the plasmid. Therefore, if cells lose the RFP signal after induction of the restriction enzyme and ligase, we have proven that in vivo restriction and ligation is possible and can lead to the exclusion of DNA fragments:
Figure 4: Modified proof of concept
SceI restriction enzyme
SceI is a site-specific homing endonuclease. It is extremely rare-cutting as it recognizes an 18 bp sequence. These properties make SceI a perfect restriction enzyme for our proof of concept for which the restriction enzyme should only cut the sites of our constructs indicated above. However, for our goal of minimizing E. coli’s genome we will need to use a frequently cutting restriction enzyme which will potentially lead to severe damage of the cell’s genome. In order to limit this damage, we want to be able to pulse the expression of the restriction enzyme. Therefore, we are cloning an inducible promoter in front of SceI.
T4 ligase
T4 is the commonly used ligase for in vitro cloning. For in vivo cloning, high levels of T4 are advantageous not only for improving the efficiency of religation leading to the exclusion of chromosomal fragments, but also for limiting DNA damage. However, constitutive overexpression of T4 might lead to immediate religation without the exclusion of chromosomal fragments. Therefore, we are cloning T4 behind an IPTG-inducible promoter (R0010), and, as an alternative approach, behind several constitutive promoter of differing strengths. Another idea would be to clone both SceI and T4 behind the same promoter, so that induction would lead to simultaneous expression of both enzymes.
Lab results
Goals
For the proof of concept we want to generate the following constructs:
Figure 5: Plasmids for proof of concept
These two plasmids will be transformed into the same host cells. The RFP generator flanked by two SceI-restriction sites will be on a low-copy plasmid, like pCK01, to reduce the number of cutting targets for SceI. The T4 ligase and the SceI restriction enzyme will be placed on a high-copy plasmid. Expression of T4 and SceI will be induced to cut out the RFP generator and religate the digested plasmid in vivo. This can be easily detected by the disappearance of the red fluorescence of the cells. We have chosen to use RFP, since on agar plates RFP-synthesizing colonies are easily distinguishable from the ones which have lost the RFP gene. By using antibiotics we will assure that only colonies which contain the plasmids encoding the two resistance genes will survive.
Modified proof of concept
For the RFP generator we used the Biobrick BBa_J04450 and cloned SceI-restriction sites into its EcoRI- and PstI-sites.
Figure 6: Plasmids for proof of concept
For the SceI-restriction sites we ordered oligos containing the SceI-recognition site and, at its ends, the overhangs for EcoRI- and PstI-sites, respectively. The oligos were hybridized and cloned into BBa_J04450. Successful cloning was verified by digestion and gel electrophoresis.
Oligos for SceI-site integrated at EcoRI-site:
EcoR_SceI_up         AATTCAGTTACGCTAGGGATAACAGGGTAATATAGC           EcoR_SceI_down         AATTGCTATATTACCCTGTTATCCCTAGCGTAACTG
Oligos for SceI-site integrated at PstI-site:
Pst_site_up        AGTTACGCTAGGGATAACAGGGTAATATAGCTGCA         Pst_site_down        GCTATATTACCCTGTTATCCCTAGCGTAACTTGCA
Hybridized Oligos:
Figure 7: Hybridized oligos for insertion of SceI-restriction sites
In future work we are planning to subclone this construct (RFP with two SceI-restriction sites) onto a low-copy plasmid.
SceI restriction enzyme and T4 ligase
We performed the following cloning to bring SceI and T4 under the control of different promoters by standard Biobrick assembly:
BBa_K142205 : SceI under the lac-Promoter
BBa_K142204 : T4 under the lac-Promoter
BBa_K142207 : T4 under a medium constitutive promoter
BBa_K142206 : T4 under a strong constitutive promoter
References
(1) Ren Z. J., Baumann R. G., Black L. W. (1997): Cloning of linear DNAs in vivo by overexpressed T4 DNA ligase: construction of a T4 phage hoc gene display vector. Gene 22 195 (2):303-11.
(2) Heitman J., Zinder N. D., Model P. (1989): Repair of the Escherichia coli chromosome after in vivo scission by the EcoRI endonuclease. Proc Natl Acad Sci 86 (7):2281-5.
(3) Datsenko K. A., Wanner B. L. (2000): One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proc Natl Acad Sci 97 (12):6640-5.
Retrieved from " http://2008.igem.org/Team:ETH_Zurich/Wetlab/Genome_Reduction "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
