<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Modeling_Download"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Modeling/Download">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Modeling/Download&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Download&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Download&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Modeling/Download" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Modeling/Download</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="background:#FFFFFF ; border:3.5px solid #60AFFE; padding: 1em; margin: auto ; width:98.5%"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Restriction_Enzymes_Analysis"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Restriction Enzymes Analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Genome-scale_Model"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Genome-scale Model</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Chemostat_Selection"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Chemostat Selection</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Switch_Circuit"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Switch Circuit</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><P>
This page contains all the programming code functions and the data we have used in order to generate our results. You can download each of the fourth section as a stand-alone framework and run in order to reproduce the result or investigate our methodology. Some of them requires third part free software (such as the Cobra Toolbox), you can find the links to download them in the description section. Some of them requires data or models previously developed by other groups but freely accessible, we state where and when these object have been used. All the code is enriched with comments for an easily understanding of functions and computational steps.
</P><H2><SPAN class="mw-headline" id="Restriction_Enzymes_Analysis"> Restriction Enzymes Analysis </SPAN></H2><P><B>Download file:</B><A href="/File:ETHZ-iGEM2008-GenomeStaticAnalysis.zip" title="File:ETHZ-iGEM2008-GenomeStaticAnalysis.zip">File:ETHZ-iGEM2008-GenomeStaticAnalysis.zip</A></P><P><B>Compute:</B> This simulation downloads the genome annotation of <I>E. coli</I> K12 MG1655 from GenBank and restriction enzymes informations from REBASE. For each restriction enzyme, calculates the fragments generated by digesting the chromosome of the strain. It calculates statistics on various aspects, such as probability of having an essential gene per fragment.
</P><P><B>Output:</B> An array of records, each one containing statistical analysis for one of the restriction enzyme groups. Generates five plots that explain the statistical results.
</P><P><B>Softwared needed:</B> MATLAB with BioInformatics Toolbox.
</P><P><B>Third part code:</B> use of standard Bioinformatics Toolbox functions.
</P><P><B>Files and descriptions:</B></P><UL><LI> convertCuttingSitesIntoGeneNames.m - convert informations on cutting sites for restriction enzymes in file names as Blattner Ids and alphanumeric gene names.
</LI><LI> createAtomicKnockoutsForRestEnzymes.m - construct the structure containing fragment patterns for all the selected restriction enzymes.
</LI><LI> createAtomicKnockoutStruct.m - constructs a structure object containing fragment patterns for one restriction enzyme.
</LI><LI> digestionSimulation.m - perform the digestion of chromosome using all the restriction enzymes.
</LI><LI> genomeStaticAnalysis.m - main function, invoke the digestion procedure and plots the five graphs.
</LI><LI> essentialGenes.mat - list of essential genes (single knockouts are mortal), obtained from source cited in <A href="/Team:ETH_Zurich/Modeling/Genome_Static_Analysis" title="Team:ETH Zurich/Modeling/Genome Static Analysis">Restriction Enzymes Analysis page</A>.
</LI></UL><P><B>Author(s):</B> Luca and Georg.
</P><H2><SPAN class="mw-headline" id="Genome-scale_Model"> Genome-scale Model </SPAN></H2><P><B>Download file:</B><A href="/File:ETHZ-iGEM2008-GenomeScaleModel.zip" title="File:ETHZ-iGEM2008-GenomeScaleModel.zip">File:ETHZ-iGEM2008-GenomeScaleModel.zip</A></P><P><B>Compute:</B> This package contains the code for reproducing Figure 2,3,4,5,6 and the code needed to create Figure 7. The package initially loads the genome of <I>E. coli</I> K12 MG1655 from GeneBank and then construct a data structure of gene infos taken from the annotation. It then loads two different genome scale models (included in the zip file) that represent the <I>iAF1260</I> model thymidine auxotrophic, with an external uptake of thymidine, respectively in minimal and in rich medium. It then perform different simulations based on Flux Balance Analysis, such as testing the effect of thymidine feeding at different chromosomal size. It then simulate the cycles of reduction and selection by considering an <I>in silico</I> chromosome containing only the gene represented in the genome scale model. For detail please refer to the <A href="/Team:ETH_Zurich/Modeling/Genome-Scale_Model" title="Team:ETH Zurich/Modeling/Genome-Scale Model">Genome Scale Model page</A>.
</P><P><B>Output:</B> The figures reported in the corresponding section.
</P><P><B>Softwared needed:</B> MATLAB (The Mathworks) with the Bioinformatics Toolbox, the Cobra ToolBox from <A href="http://gcrg.ucsd.edu/|" class="external text" rel="nofollow">Palsson's Group at UCSD</A>. 
</P><P><B>Third part code:</B> the iAF1260 <I>in-silico</I> organism from <A href="http://gcrg.ucsd.edu/|" class="external text" rel="nofollow">Palsson's Group at UCSD</A>.
</P><P><B>Files and descriptions:</B></P><UL><LI> genomeScaleModel.m - main function that loads the needed data and calls the plots and simulation in sequence.
</LI><LI> changeModelForDNASize.m - functions that changes a genome scale model by setting a different need of nucleotides depending on the new chromosome size.
</LI><LI> convertCuttingSitesIntoGeneNames.m - functions that converts cutting site positions into the corresponding genes effected by disruption (alphanumeric name and Blattner Id supported). 
</LI><LI> create3DPlots.m - create the 3D plot that relates thymidine uptake rate and chromosomal size to growth rate.
</LI><LI> createAtomicKnockoutsForRestEnzymes.m - creates the list of all the chromosomal fragments generate after a digestion with several enzymes.
</LI><LI> createAtomicKnockoutStruct.m - function that creates the list of all the chromosomal fragments generate after a digestion of on enzyme.
</LI><LI> createPlotChromSizeVsGrowthRate.m - function that creates the plot of chromosomal size vs. growth rate.
</LI><LI> createPlotThymVsGrowthRate.m - function that creates the plot of thymidine uptake rate vs. growth rate.
</LI><LI> extractGenesInfoFromGenomeAnnotation.m - function that extractes information about each gene in <I>E. coli</I> K12 MG1665 
</LI><LI> simGeneGreedyRandomWalk.m - function that simulate in abstract an actual complete cycle of reduction and selection at the level of single gene and using a random walk combined with a greedy search.
</LI><LI> modelKnockoutMinMedium.mat - data file containing the <I>iAF1260</I> genome scale model, thymidine auxotrophic, on minimal medium plus external feeding of thymidine. 
</LI><LI> modelKnockoutRichMedium.mat - data file containing the <I>iAF1260</I> genome scale model, thymidine auxotrophic, on rich medium plus external feeding of thymidine. 
</LI></UL><P><B>Author(s):</B> Luca.
</P><H2><SPAN class="mw-headline" id="Chemostat_Selection"> Chemostat Selection </SPAN></H2><P><B>Download file:</B><A href="/File:ChemostatANDFramework.zip" title="File:ChemostatANDFramework.zip">File:ChemostatANDFramework.zip</A></P><P><B>Compute:</B> Growth rate simulation of different genome-reduced mutants in the chemostat
</P><P><B>Output:</B></P><UL><LI> graphical representation of growth simulations
</LI><LI> best found reduced strain  
</LI></UL><P><B>Software needed:</B> MATLAB
</P><P><B>Third part code:</B><A href="/File:ETHZ-iGEM2008-GenomeScaleModel.zip" title="File:ETHZ-iGEM2008-GenomeScaleModel.zip">File:ETHZ-iGEM2008-GenomeScaleModel.zip</A>, 
<A href="/File:ETHZ-iGEM2008-GenomeStaticAnalysis.zip" title="File:ETHZ-iGEM2008-GenomeStaticAnalysis.zip">File:ETHZ-iGEM2008-GenomeStaticAnalysis.zip</A> and CobraToolbox
</P><P><B>Files and descriptions:</B></P><UL><LI> chemo2.m: performs chemostat simulation and plots the results
</LI><LI> sceI_chem2.m: calculates the right right hand side of chemostat equations
</LI><LI> framework.m: link different modelling parts 
</LI><LI> performs gene deletion and growth rate calculation
</LI><LI> handles tree structure of emerging mutants
</LI><LI> simulates dynamics of chemostat population
</LI></UL><P><B>Author(s):</B> Sven and Julia
</P><H2><SPAN class="mw-headline" id="Switch_Circuit"> Switch Circuit </SPAN></H2><P><B>Download file:</B><A href="/File:ETHZ-iGEM2008-SwitchCircuitModel.zip" title="File:ETHZ-iGEM2008-SwitchCircuitModel.zip">File:ETHZ-iGEM2008-SwitchCircuitModel.zip</A></P><P><B>Compute:</B> This project-file contains the chemical reactions describing the switch circuit, together with all the kinetic parameters and initial concentrations. It converts the chemical reactions in a set of ordinary differential equations (ODEs) and solves them deterministically or stochastically depending on the selected ODE solver. It then plots the results. Furthermore the sensitivities of the equations can be computed. 
</P><P><B>Output:</B> Plots of the dynamic behavior in time of the switch circuit 
</P><P><B>Software needed:</B> MATLAB (The Mathworks) with the SimBiology Toolbox
</P><P><B>Third part code:</B> uses the Sundials ODE solver and the SSA (standard stochastic algorithm) solver (both included in the SimBiology Toolbox)
</P><P><B>Files and descriptions:</B></P><UL><LI> switch_circuit_model.sbproj - Symbiology-project-file containing the switch circuit model
</LI></UL><P><B>Author(s):</B> Nico and Julia
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:ETH_Zurich/Modeling/Download">http://2008.igem.org/Team:ETH_Zurich/Modeling/Download</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Modeling/Download" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Modeling/Download" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Download&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Download&amp;oldid=105319" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>