"
Page
Discussion
View source
History
teams
Log in
Â 
Team:ETH Zurich/Project/Conclusions
From 2008.igem.org
Conclusions
In this iGEM project we addressed the question of how the minimal genome of a particular organism, E.coli, could be identified. Our aim was to develop a minimal strain for researchers working in the field of synthetic biology. The motivations that made the search of a minimal genome strain appealing were twofold: the search for fundamental yet missing biological properties that are expected to be found in essential systems and the desire of providing a convenient chassis for synthetic biology. In this project we identified two requirements for our ideal minimal genome:
to be as simple as possible by achieving a maximal reduction in genome size and gene content.
to be viable, meaning that we were aiming for strains able to provide a reactive and productive background on which to add synthetic functionalities.
The approach we proposed  was based on two main considerations. First, that the solution space of possible minimal genomes is huge and intractable without taking a heuristic approach. Second, that evolution probably worked in the opposite direction, constructing complex organisms starting from a relatively small set of genes.
Combining the two concepts, we decided to take an evolutionary synthetic reductive approach. In order to do so, we had to invert two main biological mechanisms. First, loosing part of the genome should be made possible, while cells (for the evolutionary motivations discussed before) are indeed more equipped for uptaking DNA parts. Second, to give a fitness advantage to cells that have a reduced genome, things that to our knowledge have never been showed before. Moreover, these two mechanisms had to be implemented in a framework that permitted the sequential application of a mutation phase (reduction) and selection phase (fitness function) in order to form the cycle that is proper of an evolutionary algorithm. By bringing out the concept that cells are natural carriers of our possible solutions (they indeed carry a chromosome that can be reduced), we investigated the following solutions:
reduction of the chromosome could be achieved by controlled expression of restriction enzymes and ligase by using a genetic circuit.
reduced strains can be made fitter by penalizing large chromosomal size through a nucleotide limitation.
populations of our solutions (cells) can be repetitively subjected to reduction and selection phases by using a chemostat.
Our efforts were centered in trying to prove the feasibility of our assumptions from the experimental side (when possible) and using modeling techniques (when convenient). Here we report a brief summary of what we achieved with links to the detailed description.
Wet laboratory (experimental) results:
we designed and sent to be synthesized a novel proof of concept construct regarding to the restriction enzyme in-vivo activity. Moreover we designed experiments that would reveal key parameters of restriction enzyme in-vivo efficiency ( link ).
we successfully performed knockouts of thymidylate synthase as first step for selection method evaluation ( link ).
we showed evidence that is possible to control growth rate by constraining thymidine feeding, thus validating our proposed selection mechanism ( link ).
we designed and brought to late step of cloning a novel genetic switch circuit, able to control the expression of in-vivo restriction enzymes ( link ).
we construct and characterized a new biobrick, able to stop ribosomal functionality upon induction, as a spin-off from our main stream project ( link ).
Dry Laboratory (modeling) results:
we applied statistical analysis to evaluate optimality for our purpose of a large population of restriction enzymes ( link )
we used the state-of-the-art genome scale model for E.coli as framework for testing single cell response to our reduction and selection mechanisms. We performed modification of the model and simulations that permitted to quantitatively validate the feasibility of our selection method and to predict a reduction up to 71% of the E.coli genome ( link ).
we modeled the dynamical behavior of the chemostat machinery and evaluated parameters and feasibility range using  Ordinary Differential Equation modeling ( link ).
we proposed the Ordinary Differential Equation modeling of the genetic switch circuit, taking parameter values from literature and performing sensitivity analysis in order to analyze key variables in the model ( link ).
we provided all the software and data used to generate our results to the community ( link ).
Retrieved from " http://2008.igem.org/Team:ETH_Zurich/Project/Conclusions "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
