<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Modeling_Switch_Circuit"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Modeling/Switch_Circuit">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Modeling/Switch_Circuit&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Switch_Circuit&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Switch_Circuit&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Modeling/Switch_Circuit" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Modeling/Switch Circuit</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="background:#FFFFFF ; border:3.5px solid #60AFFE; padding: 1em; margin: auto ; width:98.5%"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Switch_Circuit"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Switch Circuit</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Implementation_and_Simulation"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Implementation and Simulation</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Implementation"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Implementation</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Simulation_Results"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Simulation Results</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Sensitivity_Analysis"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Sensitivity Analysis</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Detailed_Model"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Detailed Model</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#diffusion_of_IPTG"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">diffusion of IPTG</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#diffusion_of_tet"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">diffusion of tet</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#binding_of_IPTG_to_LacI"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">binding of IPTG to LacI</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#binding_of_tet_to_TetR"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">binding of tet to TetR</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#binding_of_TetR_to_LacIIs-promoter"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">binding of TetR to LacIIs-promoter</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#binding_of_LacI_and_LacIIs_to_GFP-promoter"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">binding of LacI and LacIIs to GFP-promoter</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#transcription_and_translation_of_LacI"><SPAN class="tocnumber">3.7</SPAN><SPAN class="toctext">transcription and translation of LacI</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#transcription_and_translation_of_LacIIs"><SPAN class="tocnumber">3.8</SPAN><SPAN class="toctext">transcription and translation of LacIIs</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#transcription_and_translation_of_TetR"><SPAN class="tocnumber">3.9</SPAN><SPAN class="toctext">transcription and translation of TetR</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#transcription_and_translation_of_GFP"><SPAN class="tocnumber">3.10</SPAN><SPAN class="toctext">transcription and translation of GFP</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#dimerization_and_tetramerization_of_LacI_and_LacIIs"><SPAN class="tocnumber">3.11</SPAN><SPAN class="toctext">dimerization and tetramerization of LacI and LacIIs</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#dimerization_of_tetR"><SPAN class="tocnumber">3.12</SPAN><SPAN class="toctext">dimerization of tetR</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Parameters"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Parameters</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Switch_Circuit">Switch Circuit</SPAN></H2><P>The designed switching circuit is driven by  two input signals – a start signal initiates the synthesis of a specific protein and a terminating signal switches gene expression off. The goal of this system is to control expression of restriction enzymes in order to delete genome fragments <I>in vivo</I>.
In order to do some preliminary experiments, the restriction enzyme has been substituted by a fluorescent protein.</P><P>The detailed mechanism is described <A href="/Team:ETH_Zurich/Wetlab/Switch_Circuit#Switch_Circuit" title="Team:ETH Zurich/Wetlab/Switch Circuit"> here</A>. It follows a summarized version of it's mode of operation:
Fluorescent protein gene expression is under control of LacI and can be induced by the addition of IPTG. In order to stop gene expression, the IPTG-sensitive LacI is replaced by IPTG-insensitive LacIIs, which shuts off fluorescent gene expression again. The synthesis of LacIIs is started by the addition of Tetracyclin (tet) to the system, which binds to the tet repressor TetR and thus de-represses the expression of the LacIIs gene. The fluorescent protein is tagged, so that it is degraded faster by the Clp protease and vanishes faster from the system as its expression is stopped.</P><P>This switching circuit is described by a set of 64 chemical reactions and 41 molecular species.
In order to do a computational analysis of the circuit, this model has been simplified and implemented in MATLAB. Then the system has been simulated using ODE and stochastic solvers.
</P><H2><SPAN class="mw-headline" id="Implementation_and_Simulation"> Implementation and Simulation </SPAN></H2><H3><SPAN class="mw-headline" id="Implementation"> Implementation </SPAN></H3><P>In the implementation, some minor parts that do not have significant effects on the aspired results have been neglected for the sake of simplicity. In the MATLAB-model of the different protein-expressions, the RNA-polymerase and the ribosomes have not been taken into account, in the transcription and translation respectively. Furthermore, the effects of the dimerization of TetR as well as the impacts of dimerization and tetramerization of LacI and LacIIs <A href="https://2008.igem.org/Team:ETH_Zurich/Modeling/Switch_Circuit#References" class="external text" rel="nofollow">(4)</A> have not been considered in the final implementation. </P><P>This simplified model still comprises more than 20 different molecular species and over 30 kinetic reactions and was implemented by using the SimBiology Toolbox in MATLAB.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">diagram view of the model</DIV></DIV></DIV></DIV><P>We performed deterministic and stochastic simulations based on Mass Action Kinetics. The stochastic simulations turned out to be computationally very exhaustive but generated no further significant information compared to the deterministic simulations.</P><H3><SPAN class="mw-headline" id="Simulation_Results"> Simulation Results </SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">Input signals: Start signal (Induction with IPTG) and Stop signal (induction with tet) after 10 min</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:430px;"><DIV class="thumbcaption">Output signal: expression of GFP and GFP_mRNA as a response to the input signals on the left</DIV></DIV></DIV><P>The simulations show that our system actually should create a nice pulse-shaped expression of the fluorescent protein (GFP). This expression can be initiated by inducing with IPTG and stopped by subsequent addition of tet into the medium. By tagging the protein it will be degraded much faster by the Clp protease, so that the overall concentration is bounded and, after activating the stop-signal, the remaining proteins disappear quickly.</P><P>Another fact that the simulations showed is that in order to get one single pulse the tet-concentration inside, the medium must not reach zero before all the IPTG is degraded too. Otherwise there would still be IPTG in the system inhibiting the binding of LacI to the GFP-promoter and leading to an unwanted expression of our protein of interest as the LacIIs degrades.</P><P>One way to overcome this problem is by simply inducing with a much higher quantity of tet than IPTG, so that it simply takes longer for it to completely degrade or being washed away. Still another way would be to wait a bit longer after the induction with IPTG so that it has already partly vanished until switching off the GFP-expression by inducing with tet.
</P><H3><SPAN class="mw-headline" id="Sensitivity_Analysis"> Sensitivity Analysis </SPAN></H3><P>We define the sensitivity as the change of the production of the desired fluorescence protein - which is the output of our system - depending on the change of the 
<A href="/Team:ETH_Zurich/Modeling/Switch_Circuit#Parameters" title="Team:ETH Zurich/Modeling/Switch Circuit">parameters</A>.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Sensitivity analysis - change in the GFP concentration depending on the change of the kinetic parameters</DIV></DIV></DIV></DIV><P>The sensitivity analysis shows, that the concentration of the fluorescent protein strongly depends on its decay rate (parameter 13) the decay rate of its mRNA (parameter 10) and of course the transcription and translation rates of the protein (parameters 29 and 27), which is no surprise.
We can also see that the decay rates of LacIIs (parameter 9) and LacIIs_mRNA (parameter 12) and the transcription rate of LacIIs (parameter 31) have an influence on the expression of the fluorescent protein.
</P><H2><SPAN class="mw-headline" id="Detailed_Model"> Detailed Model </SPAN></H2><H3><SPAN class="mw-headline" id="diffusion_of_IPTG"> diffusion of IPTG </SPAN></H3><P>In order to switch on the circuit, we induce with IPTG. When IPTG is added into the medium it diffuses reversibly between the medium and the cells, where it is slowly degraded. In a chemostat extracellular IPTG is washed away.</P><H3><SPAN class="mw-headline" id="diffusion_of_tet"> diffusion of tet </SPAN></H3><P>The second inducer which is used in our system is tet. This one also diffuses reversibly between the medium and the cells, where it is slowly degraded and is washed away in the medium.
</P><H3><SPAN class="mw-headline" id="binding_of_IPTG_to_LacI"> binding of IPTG to LacI </SPAN></H3><P>The first inducer IPTG can bind to the tetramerized LacI <A href="https://2008.igem.org/Team:ETH_Zurich/Modeling/Switch_Circuit#References" class="external text" rel="nofollow">(4)</A>.</P><H3><SPAN class="mw-headline" id="binding_of_tet_to_TetR"> binding of tet to TetR </SPAN></H3><P>The second inducer tet can bind to TetR.</P><H3><SPAN class="mw-headline" id="binding_of_TetR_to_LacIIs-promoter"> binding of TetR to LacIIs-promoter </SPAN></H3><P>TetR is constitutively expressed and binds to the LacIIs promoter, inhibiting its expression.</P><H3><SPAN class="mw-headline" id="binding_of_LacI_and_LacIIs_to_GFP-promoter"> binding of LacI and LacIIs to GFP-promoter </SPAN></H3><P>LacI which is constitutively expressed and LacIIs which is under the control of a tet repressor can bind both to the GFP promotor.</P><H3><SPAN class="mw-headline" id="transcription_and_translation_of_LacI"> transcription and translation of LacI </SPAN></H3><UL><LI> RNA polymerase binds to the LacI-promoter and transcribes it into LacI-mRNA
</LI><LI> RNA polymerase detaches from the LacI-mRNA
</LI><LI> degradation of LacI-mRNA
</LI><LI> ribosome binds to LacI-mRNA and translates it into LacI
</LI><LI> ribosome detaches from LacI
</LI><LI> degradation of LacI
</LI></UL><H3><SPAN class="mw-headline" id="transcription_and_translation_of_LacIIs"> transcription and translation of LacIIs </SPAN></H3><UL><LI> RNA polymerase binds to the LacIIs-promoter and transcribes it into LacIIs-mRNA
</LI><LI> RNA polymerase detaches from the LacIIs-mRNA
</LI><LI> degradation of LacIIs-mRNA
</LI><LI> ribosome binds to LacIIs-mRNA and translates it into LacIIs
</LI><LI> ribosome detaches from LacIIs
</LI><LI> degradation of LacIIs
</LI></UL><H3><SPAN class="mw-headline" id="transcription_and_translation_of_TetR"> transcription and translation of TetR </SPAN></H3><UL><LI> RNA polymerase binds to the TetR-promoter and transcribes it into TetR-mRNA
</LI><LI> RNA polymerase detaches from the TetR-mRNA
</LI><LI> degradation of TetR-mRNA
</LI><LI> ribosome binds to TetR-mRNA and translates it into TetR
</LI><LI> ribosome detaches from TetR
</LI><LI> degradation of TetR
</LI></UL><H3><SPAN class="mw-headline" id="transcription_and_translation_of_GFP"> transcription and translation of GFP </SPAN></H3><UL><LI> RNA polymerase binds to the GFP-promoter and transcribes it into GFP-mRNA
</LI><LI> RNA polymerase detaches from the GFP-mRNA
</LI><LI> degradation of GFP-mRNA
</LI><LI> ribosome binds to TGFP-mRNA and translates it into GFP
</LI><LI> ribosome detaches from GFP
</LI><LI> degradation of GFP
</LI></UL><H3><SPAN class="mw-headline" id="dimerization_and_tetramerization_of_LacI_and_LacIIs"> dimerization and tetramerization of LacI and LacIIs </SPAN></H3><H3><SPAN class="mw-headline" id="dimerization_of_tetR"> dimerization of tetR </SPAN></H3><H2><SPAN class="mw-headline" id="Parameters"> Parameters </SPAN></H2><P>In this section you can find all the parameters used in the simulation.</P><P>Because many of the in vivo rates of the biochemical reactions we simulated are unknown or could not be found in the literature, the kinetic parameters were mainly obtained from estimates <A href="https://2008.igem.org/Team:ETH_Zurich/Modeling/Switch_Circuit#References" class="external text" rel="nofollow">(2)</A> based on the values found in the supporting text to <A href="https://2008.igem.org/Team:ETH_Zurich/Modeling/Switch_Circuit#References" class="external text" rel="nofollow">(1)</A>.</P><TABLE border="1"><TBODY><TR><TH> # </TH><TH> Parameter name </TH><TH> Value </TH><TH> Units </TH><TH> Reference/Comment
</TH></TR><TR><TH> 1
</TH><TD> k_assoc(IPTG_LacI) </TD><TD> 5.0 </TD><TD> 1/(mole*second) </TD><TD> Estimate
</TD></TR><TR><TH> 2
</TH><TD> k_assoc(LacI) </TD><TD> 5.0 </TD><TD> 1/(mole*second) </TD><TD> Estimate
</TD></TR><TR><TH> 3
</TH><TD> k_assoc(LacIIs) </TD><TD> 5.0 </TD><TD> 1/(mole*second) </TD><TD> Estimate
</TD></TR><TR><TH> 4
</TH><TD> k_assoc(tet) </TD><TD> 5.0 </TD><TD> 1/(mole*second) </TD><TD> Estimate
</TD></TR><TR><TH> 5
</TH><TD> k_assoc(tetR) </TD><TD> 5.0 </TD><TD> 1/(mole*second) </TD><TD> Estimate
</TD></TR><TR><TH> 6
</TH><TD> k_dec(IPTG) </TD><TD> 0.0002 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 7
</TH><TD> k_dec(IPTG_ext) </TD><TD> 0.001 </TD><TD> 1/second </TD><TD> [*]
</TD></TR><TR><TH> 8
</TH><TD> k_dec(LacI) </TD><TD> 0.05 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 9
</TH><TD> k_dec(LacIIs) </TD><TD> 0.05 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 10
</TH><TD> k_dec(gfp) </TD><TD> 0.25 </TD><TD> 1/second </TD><TD> [**]
</TD></TR><TR><TH> 11
</TH><TD> k_dec(mRNA_LacI) </TD><TD> 0.05 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 12
</TH><TD> k_dec(mRNA_LacIIs) </TD><TD> 0.05 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 13
</TH><TD> k_dec(mRNA_gfp) </TD><TD> 0.1 </TD><TD> 1/second </TD><TD> [**]
</TD></TR><TR><TH> 14
</TH><TD> k_dec(mRNA_tetR) </TD><TD> 0.05 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 15
</TH><TD> k_dec(tetR) </TD><TD> 0.05 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 16
</TH><TD> k_dec(tet) </TD><TD> 0.0002 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 17
</TH><TD> k_dec(tet_ext) </TD><TD> 0.001 </TD><TD> 1/second </TD><TD> [*]
</TD></TR><TR><TH> 18
</TH><TD> k_diff(IPTG) </TD><TD> 0.1 </TD><TD> 1/second </TD><TD> (1)
</TD></TR><TR><TH> 19
</TH><TD> k_diff(tet) </TD><TD> 0.1 </TD><TD> 1/second </TD><TD> (1)
</TD></TR><TR><TH> 20
</TH><TD> k_dissoc(IPTG_LacI) </TD><TD> 1.0 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 21
</TH><TD> k_dissoc(LacI) </TD><TD> 1.0 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 22
</TH><TD> k_dissoc(LacIIs) </TD><TD> 1.0 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 23
</TH><TD> k_dissoc(tet) </TD><TD> 1.0 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 24
</TH><TD> k_dissoc(tetR) </TD><TD> 1.0 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 25
</TH><TD> k_tl(LacI) </TD><TD> 5.0 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 26
</TH><TD> k_tl(LacIIs) </TD><TD> 5.0 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 27
</TH><TD> k_tl(gfp) </TD><TD> 5.0 </TD><TD> 1/second </TD><TD> (1)
</TD></TR><TR><TH> 28
</TH><TD> k_tl(tetR) </TD><TD> 5.0 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR><TR><TH> 29
</TH><TD> k_tr(gfp) </TD><TD> 1.0 </TD><TD> 1/second </TD><TD> (1)
</TD></TR><TR><TH> 30
</TH><TD> k_tr(LacI) </TD><TD> 0.1 </TD><TD> 1/second </TD><TD>  [***]
</TD></TR><TR><TH> 31
</TH><TD> k_tr(LacIIs) </TD><TD> 0.1 </TD><TD> 1/second </TD><TD> [***]
</TD></TR><TR><TH> 32
</TH><TD> k_tr(tetR) </TD><TD> 1.0 </TD><TD> 1/second </TD><TD> Estimate
</TD></TR></TBODY></TABLE><P>[*]   the degradation constants of the two inducers are bigger outside the cell, the effect that they are washed away in the chemostat is taken into account in those parameters
[**]  degradation rates of gfp are higher because of the tagging 
[***] need to be that high to account for the autorepression of LacI / in order to get a low steady state concentration of LacI of about 50 proteins (3)
</P><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P>(1) <I>&quot;Spatiotemporal control of gene expression with pulse-generating networks&quot;</I>, Basu et al., <I>PNAS</I>, 2004
</P><P>(2) <I>&quot;Genetic circuit building blocks for cellular computation, communications, and signal processing&quot;</I>, Weiss et al.,  <I>Natural Computing</I>, 2003
</P><P>(3) <I>&quot;Predicting stochastic gene expression dynamics in single cells&quot;</I>, Mettetal et al., <I>PNAS</I>, 2006
</P><P>(4) <I>&quot;Engineered gene circuits&quot;</I>, Hasty et al., <I>Nature</I>, 2002
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:ETH_Zurich/Modeling/Switch_Circuit">http://2008.igem.org/Team:ETH_Zurich/Modeling/Switch_Circuit</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Modeling/Switch_Circuit" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Modeling/Switch_Circuit" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Switch_Circuit&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Switch_Circuit&amp;oldid=105306" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>