<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Wetlab_Switch_Circuit"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Wetlab/Switch_Circuit">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Wetlab/Switch_Circuit&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Switch_Circuit&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Switch_Circuit&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Wetlab/Switch_Circuit" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Wetlab/Switch Circuit</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="background:#FFFFFF ; border:3.5px solid #60AFFE; padding: 1em; margin: auto ; width:98.5%"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Switch_Circuit"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Switch Circuit</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Short_Introduction_and_Overview"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Short Introduction and Overview</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Introduction"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#System_Description"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">System Description</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Mechanism"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Mechanism</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-6"><A href="#General_Layout_of_the_Pulse_Generator"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">General Layout of the Pulse Generator</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Design_of_the_LacI_IS_Mutants"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Design of the LacI IS Mutants</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Estimate_of_LacI_IS_Repressor_Strength"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Estimate of LacI IS Repressor Strength</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Genetic_Experiments_using_Ribosome_Modulation_Factor_.28RMF.29"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Genetic Experiments using Ribosome Modulation Factor (RMF)</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Fluorescent_Protein_Experiments"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Fluorescent Protein Experiments</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Assembly_of_the_Pulse_Generator"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Assembly of the Pulse Generator</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Testing_of_the_Pulse_Generator"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Testing of the Pulse Generator</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Prevention_of_Leaky_Expression"><SPAN class="tocnumber">1.6.1</SPAN><SPAN class="toctext">Prevention of Leaky Expression</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Expression_Level_upon_Induction"><SPAN class="tocnumber">1.6.2</SPAN><SPAN class="toctext">Expression Level upon Induction</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Termination_of_Expression_upon_Addition_of_Tetracycline"><SPAN class="tocnumber">1.6.3</SPAN><SPAN class="toctext">Termination of Expression upon Addition of Tetracycline</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#References"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Switch_Circuit"> Switch Circuit </SPAN></H2><H3><SPAN class="mw-headline" id="Short_Introduction_and_Overview">Short Introduction and Overview</SPAN></H3><H4><SPAN class="mw-headline" id="Introduction"> Introduction </SPAN></H4><P>Many biological applications require a pulse mechanism, that enables on the one side a fast production and high accumulation of specific protein at one time and on the other side a fast elimination of this protein after a period of time has passed. Pulse generators that turn protein production on and off are essential parts of engineered genetic circuits, which allow to control the system of interest. With regard to our project, especially to  the in vivo chromosomal genome-deletion, it is crucial to express restriction enzymes only for a very short amount of time in order to increase  cell survival.
</P><P>Recent studies describe several engineered pulse-generating circuits, which are based on a feed-forward motif. These are parameter-sensitive and are difficult to construct on the biological level. We propose a novel switching mechanism which requires a start signal, realized by induction with IPTG, and a stop signal, realized by addition of tetracycline. This setup allows us to test genetic circuits with protein pulses of different duration. Moreover, because our setup relies on standard promoters and repressor binding sites, standard expression vectors such as pET system by F. W. Studier (1-3) are fully compatible with our design and can be used without modification. 
</P><H4><SPAN class="mw-headline" id="System_Description"> System Description </SPAN></H4><P>The system is based on two plasmids. The first plasmid carries the protein of interest that is under control of a standard lac repressor. The protein of interest is either fluorescent protein for testing or, in our experiments with genomic reduction, specific restriction enzyme to support the gene-deletion mechanism. The second plasmid contains lacI IS, an IPTG-insensitive mutant, that is under control of the tet repressor. Since E. coli does not produce TetR, a  source of constitutively expressed TetR is included. As the system will only work with both plasmids integrated, the first plasmid has ampicillin resistance while plasmid two has kanamycin resistance.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 1: Schematic representation of the target plasmid.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 2: Schematic representation of the pulse generator.</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Mechanism"> Mechanism </SPAN></H4><P>Expression of the lac operon in E. coli is tightly controlled by LacI, a protein, which binds to a repressor
binding site within the promotor and disables transcription by obscuring the promotor region.  When bound to DNA, lacI is in the tetrameric form, which consists of two dimers interacting at the end
distal from the DNA binding site.  Since our protein of interest is under control of LacI, there is no expression of GFP before induction. In order to turn the expression of GFP on, the start signal, induction by IPTG, is given. Upon binding of IPTG, the lacI  tetramer breaks down into two dimers and the affinity for the repressor binding site is greatly reduced: the lacI IPTG complex will
diffuse away from the repressor binding site, leaving the promotor accessible. Transcription and translation of GFP protein starts immediately.
For termination of expression, tetracycline is added. Since lacI IS is under a TetR-repressible promoter, lacI IS is not expressed until induction with tetracycline abolishes affinity of TetR for the repressor binding site. Transcription of LacI IS subsequently starts; since LacI IS binds to the repressor binding side of the lac-repressible promoter no matter whether IPTG is present or not, translation of the protein of interest is terminated.
</P><H3><SPAN class="mw-headline" id="General_Layout_of_the_Pulse_Generator">General Layout of the Pulse Generator</SPAN></H3><P>The basic idea for the pulse generator was to use the lac repressor to control the start of expression and then use a second protein under control of a different promoter, which in turn would shut down expression by binding tightly to a motif contained inside the lac repressor binding site after induction of it’s promoter with a second inducer. While there are numerous proteins, which bind tightly to specific DNA sequences and while it is even possible to engineer these proteins to bind to a DNA sequence of choice (4-7), we decided to use the IS lac repressor mutant, which binds to the lac repressor binding site like the wild type but does not lose affinity for the binding site in the presence of IPTG or lactose (8). Since expression of the mutant lacI has to be tightly controlled to avoid repression of lac-controlled gene expression despite presence of inducer, it was decided to put it under control of the tet repressor (9, 10). The pulse generator consists of two parts, a constitutively active TetR generator (unlike in the case of the lac repressor, which is expressed under control of its own promoter, E. coli does not produce tet repressor) and a lacI IS mutant gene under the control of the tet repressor. The constitutively active TetR generator supplies the TetR repressor, which binds to the repressor binding site in front of the lacI IS gene and releases the binding site upon induction with tetracycline. Induction with tetracycline leads to expression of the lacI IS mutant and this protein binds to the lac repressor binding site in front of the activated gene of interest despite presence of lactose or IPTG activator. With this system, IPTG induction initiates expression of the protein of interest while tetracycline induction rapidly terminates the expression of the protein.
</P><H3><SPAN class="mw-headline" id="Design_of_the_LacI_IS_Mutants">Design of the LacI IS Mutants</SPAN></H3><P>Since the discovery of the lac repressor over 40 years ago (11), it has been subjected to extensive genetic studies, which have identified numerous mutants with different defects in activity (8, 12-25). The determination of the atomic structure of the lac repressor by itself (26) and in complex with IPTG (27) by x-ray crystallography furthered the understanding of repressor activity on the molecular level and allowed to assign mutations to different functional regions of the molecule. Mutations in several regions of the lac repressor can lead to the lacI IS phenotype: Mutations at the N-terminal domain of the core, mutations of the dimerization interface and mutations of the residues, which establish the IPTG contact. We decided to create lacI IS mutants by mutation of two residues, which form part of the IPTG binding site: R197 and T276. 
</P><DIV class="thumb tnone"><DIV class="thumbinner" style="width:948px;"><DIV class="thumbcaption">Figure 3: A Lac repressor tetramer, residues R197 and T276 are shown in red. B IPTG bound to the inducer binding site of the lac repressor, residues R197 and T276 are shown in green. Molecular graphics was generated from coordinate set <A href="http://www.rcsb.org/pdb/explore.do?structureId=1LBH" class="external text" rel="nofollow">1lbh</A> (27) using  <A href="http://www.cgl.ucsf.edu/chimera/" class="external text" rel="nofollow">UCSF Chimera</A>.</DIV></DIV></DIV><P>We are not aware of quantitative studies that detail the strength of repression with respect to different mutations (in one genetic study (8) a repressor strength of &gt;200 has been determined for numerous IS mutants including mutations of R197 and T276) or with respect to the combination between different mutations and we therefore set out to replace either residue with alanine or phenylalanine and in addition we constructed double mutants with all combinations of the two mutations. Our set of lacI IS mutations therefore comprises eight different mutants: R197A; R197F; T276A;  T276F; R197A T276A; R197A T276F; R197F T276A; R197F T276F. In order to get an impression of the efficiency of repression by the LacI IS mutants generated, we performed a genetic experiment linking the viability of cells to the repressor strength of the LacI IS mutants in question.
</P><H3><SPAN class="mw-headline" id="Estimate_of_LacI_IS_Repressor_Strength">Estimate of LacI IS Repressor Strength</SPAN></H3><P>In order to characterize the lacI IS mutants generated – especially with respect to the double mutants – we performed a series of simple genetic experiments, which would allow us to identify promising mutations and reject mutations that allow significant induction at IPTG concentrations regularly used for induction (usually between 0.1mM and 1mM). Ideally, the experiment would link repression of a lac inducible gene to a marked change in cell growth or morphology. We decided to use ribosome modulation factor (RMF) in one series of experiments and fluorescent proteins in another series as an indicator of lac repressor activity.
</P><H4><SPAN class="mw-headline" id="Genetic_Experiments_using_Ribosome_Modulation_Factor_.28RMF.29"> Genetic Experiments using Ribosome Modulation Factor (RMF)</SPAN></H4><P>Ribosome modulation factor (RMF, Uniprot <A href="http://www.uniprot.org/uniprot/P0AFW2" class="external text" rel="nofollow">P0AFW2</A>) is a small protein, which is found in many bacteria. RMF is expressed when bacterial cultures reach the stationary phase (28, 29) and terminates protein synthesis efficiently by binding to ribosomes. Biochemical studies have shown that expression of RMF leads to dimerization of ribosomes (the bacterial 70S ribosomes form a complex with a sedimentation velocity of 100S) and chemical crosslinking studies elucidated that RMF binds to the peptidyl transferase center (30, 31). Since expression of RMF reversibly terminates protein expression and thereby stops bacterial growth we used RMF to determine the extent of lac repression by mutant lacI repressors.
</P><DIV class="thumb tnone"><DIV class="thumbinner" style="width:939px;"><DIV class="thumbcaption">Figure 4: A Schematic representation of vector pET28a. Note that the vector has a constitutive LacI expression cassette. B BL21 DE3 cells holding the empty pET28a vector at 10mM IPTG. The vector is obviously nontoxic to cells even at high concentrations of inducer. C Decreased viability of BL21 DE3 cells harboring vector pET28a-RMF wild type at 1mM IPTG and 10mM IPTG.</DIV></DIV></DIV><P>For our genetic experiment, we took advantage of two properties of the pET expression system. The pET expression system shows an extremely low degree of leaky expression and is therefore ideal for expression of toxic genes. In order to increase repression of lac-controlled expression in absence of inducer, the pET vector series contains a constitutive LacI expression cassette encoding a lacI gene that is identical to the lacI gene provided by the registry as part <A href="http://partsregistry.org/Part:BBa_I763026" class="external text" rel="nofollow">I763026</A> and can therefore be converted by simple site-directed mutagenesis into the LacI IS mutants we set out to study. In order to find out whether the LacI IS mutants were able to rescue a BL21 DE3 cell harboring the mutated pET28a-RMF plasmid at even elevated concentrations of inducer, we subcloned RMF into the expression vector and subjected the expression vector to site-directed mutagenesis. In addition, we tested the impact of empty plasmid and pET28a-RMF with wild-type lacI on the viability of cells at elevated IPTG concentrations.
</P><P>In order to perform this genetic experiment, we amplified RMF from E. coli strain DH5-alpha DNA using forward primer 5’-cgcggatccgaaaacctgtattttcagggcaagagacaaaaacgagatcgcctg and reverse primer 5’-ccgctcgagttattaggccattactaccctgtcc (reaction setup was 35ul water, 10ul <A href="http://www.finnzymes.com/highperformancepcr.html" class="external text" rel="nofollow">Finnzyme</A> Phusion HF buffer, 1ul 10mM dNTPs, 1ul 25uM forward primer, 1ul 25uM reverse primer, 0.5ul DH5-alpha genomic DNA, 0.5ul <A href="http://www.finnzymes.com/highperformancepcr.html" class="external text" rel="nofollow">Finnzyme</A> Phusion Hot-start polymerase; thermocycler program was 98C for 3min, cycle start: 98C for 10s, 55C for 30s, 72C for 10s, cycle end, 35 repeats, 72C for 30min, 4C hold). We subsequently digested the PCR product with XhoI and BamHI and subcloned it into the pET28a vector (<A href="http://www.merckbiosciences.com/g.asp?f=NVG/pETtable.html" class="external text" rel="nofollow">Novagen</A>) multiple cloning site. The sequence of the RMF insert was verified by sequencing. In order to determine whether the construct is expressed correctly and in order to verify the physiological reaction of E. coli cells to expression of RMF, we electroporated pET28a-RMF into BL21 DE3 cells (which hold a T7 polymerase gene on their genome and are therefore able to express genes under the T7 promoter) and plated the cells on LB-agar plates supplemented with kanamycin and found growth strongly diminished upon plating cells on IPTG containing plates. In order to verify the termination of growth after induction of RMF expression in liquid culture, we grew cells holding the plasmid in shake flasks and found that growth terminates rapidly upon induction. We prepared ribosomes from induced cells and measured their sedimentation velocity. Induced cells contain two poulations of ribosomes, one with a sedimentation velocity of 70S (suggesting monomeric ribosomes) and the other with a sedimentation velocity of 100S (suggesting dimers). Analysis of the different fractions by negative-stain transmission electron microscopy confirmed that both fractions consist of ribosomes although both of them appear monomeric in the electron microscope (probably the interaction is very weak and breaks up upon adsorption onto the carbon grid; an image of dimeric ribosomes seen in the electron microscope after the sample was treated with substantial amounts of glutaraldehyde has been published previously (31)). The two populations of ribosomes are equally found in uninduced cells, which have progressed into the stationary phase of growth, while the 100S fraction is diminished in cells during log phase and absent in RMF-knockouts (31).
</P><DIV class="thumb tnone"><DIV class="thumbinner" style="width:722px;"><DIV class="thumbcaption">Figure 5: A Growth curve of BL21 DE3 cells holding the pET28a-RMF plasmid without induction (uninduced) and after induction after 5 hours. Cells were grown in 1l of medium in 5l baffled flasks (37C, 95rpm) and induced with 1mM IPTG once they had reached the exponential phase.  B Ribosome profile of cells in the stationary phase and after RMF induction. Cells were harvested by centrifugation (Sorvall SLC-6000 rotor, 7200rpm, 10min, 4C) and disrupted in a cell disruptor (Constant systems). The lysate was cleared at 13000 rpm in a SLA-1500 rotor in a Sorvall refrigerated centrifuge. Supernatant was collected and layered on top of a 30% sucrose solution (50mM HEPES-KOH pH 7.6; 100mM KCl; 10mM MgCl2; 1mM DTT). After centrifugation at 50000rpm for 20 hours at 4C in the preparative ultracentrifuge using a Ti70 rotor (<A href="http://www.beckmancoulter.com" class="external text" rel="nofollow">Beckman Coulter</A>) the supernatant was decanted and the pellet resuspended in ribosome buffer (50mM HEPES-KOH pH 7.6; 100mM KCl; 10mM MgCl2; 1mM DTT). The resuspended ribosomes were layered on top of a 10 to 40% (w/w) sucrose gradient, which was centrifuged in the SW32 swing rotor (<A href="http://www.beckmancoulter.com" class="external text" rel="nofollow">Beckman Coulter</A>) at 28000 rpm for 7 hours. Ribosome bands were visualized by <A href="http://en.wikipedia.org/wiki/Tyndall_effect" class="external text" rel="nofollow">light scattering</A> and photographed with a Nikon D80 SLR.  C and D Electron micrographs of the 70S and 100S fraction show monomeric ribosomes in both fractions as the only macromolecular constituent. Ribosome samples were diluted to OD260 = 1. 7ul of ribosome solution was added to a glow discharged carbon grid (<A href="http://www.quantifoil.com" class="external text" rel="nofollow">Quantifoil</A>) and stained with uranyl acetate according to standard protocol (1min sample adsorption, three times washing with 1% uranyl acetate solution by floatation). Samples were imaged with a <A href="http://www.fei.com" class="external text" rel="nofollow">FEI</A> Morgagni 268 transmission electron microscope with tungsten emitter at an acceleration voltage of 100kV; 30000x magnification and recorded with a post column Gatan CCD camera.</DIV></DIV></DIV><P>We took advantage of the constitutive lac expression cassette in the pET28a vector and used PCR-based site-directed mutagenesis to introduce lacI IS muations into the lacI generator of the pET28a-RMF vector. In order to perform this mutagenesis, we used forward primers
</P><P>R197F forward 5’-CGGCGCGTCTGTTTCTGGCTGGCTG 
</P><P>R197A forward 5’-CGGCGCGTCTGGCGCTGGCTGGCTG 
</P><P>T276F forward 5’-GGATACGACGATTTTGAAGACAGCTC 
</P><P>T276A forward 5’-GGATACGACGATGCGGAAGACAGCTC 
</P><P>and reverse primers 
</P><P>R197F reverse 5’-CAGCCAGCCAGAAACAGACGCGCCG 
</P><P>R197A reverse 5’-CAGCCAGCCAGCGCCAGACGCGCCG 
</P><P>T276F reverse 5’-GAGCTGTCTTCAAAATCGTCGTATCC 
</P><P>T276A reverse 5’-GAGCTGTCTTCCGCATCGTCGTATCC 
</P><P>to introduce either a single mutation or double mutations. PCR was performed using vector pET28a-RMF as template according to the following protocol: Each reaction setup contained 35ul water, 10ul <A href="http://www.finnzymes.com/highperformancepcr.html" class="external text" rel="nofollow">Finnzyme</A> Phusion polymerase HF buffer, 1ul 10mM dNTPs, 5ul primer mix (primers at 100ng/ul), 1ul of diluted DNA template (approximate concentration after dilution 10ng/ul), 0.5ul <A href="http://www.finnzymes.com/highperformancepcr.html" class="external text" rel="nofollow">Finnzyme</A> Phusion Hot Start polymerase. The reaction was run with the following thermocycler program: 95C for 30sec, cycle start, 95C 30 sec, 55C 1min, 72C 3min, cycle end, 18 repeats, 72C for 30min, 4C hold. After the PCR reaction had gone to completion, 10ul of 50mM MgCl2 were added to each reaction tube to adjust the magnesium concentration for DpnI restriction digest and the contents were thoroughly mixed. 1ul of DpnI (<A href="http://www.neb.com" class="external text" rel="nofollow">NEB</A>, equivalent to 20 units) were added to each tube and the contents were thoroughly mixed. The reaction was incubated in the thermocycler: 37C for 120min, 4C hold. Samples were purified with Qiagen Qiaquick PCR purification kit and electroporated into BL21 DE3 cells. Transformation into BL21 DE3 cells yielded numerous colonies. For each construct eight colonies were streaked out on LB-agar supplemented with kanamycin. In order to test the repressor strength we streaked each of these clones onto plates of progressively higher IPTG concentration and monitored the growth. As a control, the unmodified vector was streaked out on plates without, with 1mM IPTG and with 10mM IPTG and IPTG toxicity was assayed by streaking cells holding an empty pET28a vector onto a 10mM IPTG plate. While empty pET28a vector is apparently not toxic for cells even at 10mM IPTG, pET28a-RMF wild type vector prevents growth even at low concentrations of IPTG as expected. Although single colonies can be seen even at 10mM IPTG, it can be assumed that they originate from point mutations that either mutated the T7 gene, the promoter of T7 or RMF or the RMF gene. It is obvious that the great majority of cells is unviable even at 1mM IPTG and the colonies are expected to arise from single aberrations. With the mutated lacI IS as a major source of repressor we assumed that cells would be viable even at high IPTG concentration, since expression of RMF would be repressed by the mutated, IPTG-insensitive LacI IS. Indeed, cells grew normally up to an IPTG concentration of 10mM, which is 10- to 100-fold the concentration usually used for induction and, as can be seen in the corresponding figure, growth was even more pronounced at IPTG concentrations of 1mM or even 10mM compared to growth without induction. This behavior is difficult to explain. The LacI expression cassette used in the pET vector series (accession number: <A href="http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?val=EF442785" class="external text" rel="nofollow">EF442785</A>) appears to be controlled by the standard promoter TCACTGCCCGCTTTCCAGTCGGGAAACCTGTCGTGCCAGCTGCATTA taken from E. coli. This promoter is a weak, constitutive promoter, which does not appear to be regulated by IPTG, since it does not show sequence motifs reminiscent of the lac repressor binding site. It can be assumed that the population of lac repressors in the cell is dominated by the repressor mutant rather than by the wild type, since the mutant is expressed from hundreds of plasmids, while the wildtype is expressed from a single gene on the E. coli chromosome. While the wild type repressor will dissociate from the repressor binding site upstream of the T7 gene, the LacI IS mutant will not. Since IPTG does not change the ratio of repressor to activator, it is difficult to explain why cells grew slightly denser at elevated IPTG concentration. 
</P><DIV class="thumb tnone"><DIV class="thumbinner" style="width:924px;"><DIV class="thumbcaption">Figure 6: Cell viability at increasing IPTG concentration. BL21 DE3 cells holding the pET28a-RMF wildtype plasmid (control) and mutations R197A; R197F; T276A;  T276F; R197A T276A; R197A T276F; R197F T276A; R197F T276F were plated on LB-agar with kanamycin without IPTG, with 1mM IPTG and with 10mM IPTG</DIV></DIV></DIV><P>Since all mutants restored viability of cells at 1mM and even at 10mM IPTG, we assume that all mutants we have generated strongly repress expression under lac control. Although we have not been able to obtain sequences of the mutations yet, we are confident that all described mutants strongly repress lac-controlled expression.
</P><P>Since RMF was deemed useful for several applications in synthetic biology, we decided to provide both, the coding sequence and an arabinose-inducible RMF generator as BioBrick. In order to produce BioBrick <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142040" class="external text" rel="nofollow">K142040</A> holding the coding sequence of RMF, we amplified RMF from pET28a-RMF using forward primer 5'-cgcggaattcgcggccgcttctagatgaagagacaaaaacgagatcgcctgg and reverse primer 5'-cgcgctgcagcggccgctactagtattattaggccattactaccctgtccgc and subcloned it into pSB1A7 using the BioBrick restriction sites. While the digest showed an insert of the expected size, sequencing unfortunately showed that the insert was not RMF. We were not able to repeat the experiment in time for submission of the BioBrick to the registry. The arabinose-inducible RMF generator <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142041" class="external text" rel="nofollow">K142041</A>, which could be used as a sort of “cellular stop switch”, has been ordered from Geneart, but has not been received in time for submission to the registry.
</P><H4><SPAN class="mw-headline" id="Fluorescent_Protein_Experiments">Fluorescent Protein Experiments</SPAN></H4><P>While the estimation of repressor strength of the lacI IS mutants in the genetic experiment with the pET28a-RMF mutants has yielded some qualitative data suggesting that all mutants repress expression of lac controlled genes even at high IPTG concentrations, we attempted to quantify the repressor strength of different lacI IS mutants. We cloned the constitutive lacI expression cassette, which had been constructed by the iGEM team of ETH Zurich in 2007, (BioBrick <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I739002" class="external text" rel="nofollow">BBa_I739002</A>) behind one of several lac-controlled fluorescent protein generators (BioBricks <A href="http://partsregistry.org/Part:BBa_I763004" class="external text" rel="nofollow">I763004</A>, <A href="http://partsregistry.org/Part:BBa_J04430" class="external text" rel="nofollow">J04430</A>, <A href="http://partsregistry.org/Part:BBa_I13601" class="external text" rel="nofollow">I13601</A>, <A href="http://partsregistry.org/Part:BBa_J5526" class="external text" rel="nofollow">J5526</A>). The order of the assembly was directed by the intention to minimize readthrough of the RNA polymerase from the constitutive expression cassette into the lac-controlled reporter gene. Placing the lacI generator behind the fluorescent protein generator was assumed to help decreasing the background of leaky reporter protein expression.
</P><P>The constructs holding GFP and RFP were selected for site-directed mutagenesis, since they are easily detectable with the filter set of our plate reader and – in the case of RFP – even detectable by eye. Site directed mutagenensis was carried out by PCR using forward primers
</P><P>R197F forward 5’-CGGCGCGTCTGTTTCTGGCTGGCTG 
</P><P>R197A forward 5’-CGGCGCGTCTGGCGCTGGCTGGCTG 
</P><P>T276F forward 5’-GGATACGACGATTTTGAAGACAGCTC 
</P><P>T276A forward 5’-GGATACGACGATGCGGAAGACAGCTC 
</P><P>and reverse primers 
</P><P>R197F reverse 5’-CAGCCAGCCAGAAACAGACGCGCCG 
</P><P>R197A reverse 5’-CAGCCAGCCAGCGCCAGACGCGCCG 
</P><P>T276F reverse 5’-GAGCTGTCTTCAAAATCGTCGTATCC 
</P><P>T276A reverse 5’-GAGCTGTCTTCCGCATCGTCGTATCC
</P><P>to achieve single mutations R197A; R197F; T276A;  T276F and double mutants R197A T276A; R197A T276F; R197F T276A; R197F T276F. DpnI digest was used to dispose of the template. Unfortunately, the PCR reaction failed and we were unable to repeat it before closing of the Wiki.
</P><H3><SPAN class="mw-headline" id="Assembly_of_the_Pulse_Generator">Assembly of the Pulse Generator</SPAN></H3><P>We decided to assemble the pulse generator from the constitutive TetR expression cassette <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I739001" class="external text" rel="nofollow">I739001</A> and the tet-controlled LacI generator <A href="http://partsregistry.org/Part:BBa_I763026" class="external text" rel="nofollow">I763026</A>. According to the information available at the time we started cloning, we assumed that cloning the TetR generator behind the LacI expression cassette and subsequent site-directed mutagenesis would be sufficient to construct the pulse generator. However, when the Caltech team sequenced BioBrick <A href="http://partsregistry.org/Part:BBa_I763026" class="external text" rel="nofollow">I763026</A>, they found that it did not contain a promoter, but rather started with the ribosome binding site. Therefore, we revised our cloning strategy and tried two different approaches. We firstly cloned the LacI generator behind the tet-controlled promoter <A href="http://partsregistry.org/Part:BBa_R0040" class="external text" rel="nofollow">R0040</A> in order to assemble a “real “ part <A href="http://partsregistry.org/Part:BBa_I763026" class="external text" rel="nofollow">I763026</A>, our part <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142046" class="external text" rel="nofollow">K142046</A>. In the attempt to try a different approach, we at the same time cloned the TetR-generator behind the promoterless LacI generator to build intermediate <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142048" class="external text" rel="nofollow">K142048</A>. In the next step, we cloned the TetR expression cassette behind the LacI generator with tet-controlled promoter and subjected the assembly to site-directed mutagenesis.
</P><DIV class="thumb tnone"><DIV class="thumbinner" style="width:912px;"><DIV class="thumbcaption">Figure 7: A 1% agarose gel of XbaI/PstI test digests of plasmids holding the lac generator <A href="http://partsregistry.org/Part:BBa_I763026" class="external text" rel="nofollow">I763026</A> behind the tet-controlled promoter <A href="http://partsregistry.org/Part:BBa_R0040" class="external text" rel="nofollow">R0040</A> (RL1, RL2, RL3) and of XbaI/PstI test digests of plasmids holding the constitutive tetR expression cassette <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I739001" class="external text" rel="nofollow">I739001</A> behind the promoterless lacI generator <A href="http://partsregistry.org/Part:BBa_I763026" class="external text" rel="nofollow">I763026</A> (TL1, TL2, TL3). The insert size agrees with the expected size in the case of RL1, RL2, RL3, TL1, TL2. TL3 apparently religated despite dephosphorylation treatment with alkaline phosphatase (CIP) and does not contain the aforementioned TetR expression cassette.  B 1% agarose gel of XbaI/SpeI test digests of assembled pulse generator precursor containing BioBricks <A href="http://partsregistry.org/Part:BBa_R0040" class="external text" rel="nofollow">R0040</A><A href="http://partsregistry.org/Part:BBa_I763026" class="external text" rel="nofollow">I763026</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I739001" class="external text" rel="nofollow">I739001</A>. Insert size agrees with the expected size in all cases.</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:503px;"><DIV class="thumbcaption">Figure 8: 1% agarose gel of PCR products of site directed mutagenesis of the final pulse generator assembly consisting of <A href="http://partsregistry.org/Part:BBa_R0040" class="external text" rel="nofollow">R0040</A><A href="http://partsregistry.org/Part:BBa_I763026" class="external text" rel="nofollow">I763026</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I739001" class="external text" rel="nofollow">I739001</A>. Template DNA of the pulse generator precursor <A href="http://partsregistry.org/Part:BBa_R0040" class="external text" rel="nofollow">R0040</A><A href="http://partsregistry.org/Part:BBa_I763026" class="external text" rel="nofollow">I763026</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I739001" class="external text" rel="nofollow">I739001</A> was amplified by PCR in eight different reactions using primers 276F forward 5’-GGATACGACGATTTTGAAGACAGCTC; T276A forward 5’-GGATACGACGATGCGGAAGACAGCTC; R197F reverse 5’-CAGCCAGCCAGAAACAGACGCGCCG ; R197A reverse 5’-CAGCCAGCCAGCGCCAGACGCGCCG; T276F reverse 5’-GAGCTGTCTTCAAAATCGTCGTATCC ; T276A reverse 5’-GAGCTGTCTTCCGCATCGTCGTATCC in combinations as to produce the desired mutations R197A; R197F; T276A;  T276F and double mutants R197A T276A; R197A T276F; R197F T276A; R197F T276F. Each reaction setup contained 35ul water, 10ul <A href="http://www.finnzymes.com/highperformancepcr.html" class="external text" rel="nofollow">Finnzyme</A> Phusion polymerase HF buffer, 1ul 10mM dNTPs, 5ul primer mix (primers at 100ng/ul), 1ul of diluted DNA template (approximate concentration after dilution 10ng/ul), 0.5ul <A href="http://www.finnzymes.com/highperformancepcr.html" class="external text" rel="nofollow">Finnzyme</A> Phusion Hot Start polymerase. The reaction was run with the following thermocycler program: 95C for 30sec, cycle start, 95C 30 sec, 55C 1min, 72C 2min, cycle end, 18 repeats, 72C for 30min, 4C hold. After the PCR reaction had gone to completion, 10ul of 50mM MgCl2 were added to each reaction tube and the contents were thoroughly mixed. 1ul of DpnI (equivalent to 20 units) were added to each tube and the contents were thoroughly mixed. The reaction was incubated in the thermocycler following the program: 37C for 120min, 4C hold. Samples were purified with Qiaquick PCR purification kit and separated on a 1% agarose gel stained with ethidium bromide. It is obvious that the PCR reaction failed in all cases except for reaction number 4, where a small amount of product was generated. Interestingly, amplification between the primers – as evidenced by the bands slightly above 200bp in lanes 5 to 8 – nevertheless took place in the double mutant setup.</DIV></DIV></DIV><P>Unfortunately only one of the eight mutagenensis expriments by PCR had worked by end of the Wiki deadline. However, since all mutants had shown considerable repressor strength in the previous genetic experiments, we assumed it safe to continue experimentation on this particular construct.
</P><P>In addition, since all assembly intermediates are potentially useful BioBricks for genetic circuit engineering, we decided to subject both, the lacI gene by itself (as contained in BioBrick <A href="http://partsregistry.org/Part:BBa_C0012" class="external text" rel="nofollow">C0012</A>) and the, as it later turned out, promoterless Biobrick <A href="http://partsregistry.org/Part:BBa_I763026" class="external text" rel="nofollow">I763026</A> to site-directed mutagenesis to produce a set of useful LacI IS mutants.
</P><DIV class="thumb tnone"><DIV class="thumbinner" style="width:856px;"><DIV class="thumbcaption">Figure 9: A 1% agarose gel of PCR products of site directed mutagenesis of C0012. Template DNA of BioBrick <A href="http://partsregistry.org/Part:BBa_C0012" class="external text" rel="nofollow">C0012</A> was amplified by PCR in eight different reactions using primers 276F forward 5’-GGATACGACGATTTTGAAGACAGCTC; T276A forward 5’-GGATACGACGATGCGGAAGACAGCTC; R197F reverse 5’-CAGCCAGCCAGAAACAGACGCGCCG ; R197A reverse 5’-CAGCCAGCCAGCGCCAGACGCGCCG; T276F reverse 5’-GAGCTGTCTTCAAAATCGTCGTATCC ; T276A reverse 5’-GAGCTGTCTTCCGCATCGTCGTATCC in combinations as to produce the desired mutations R197A; R197F; T276A;  T276F and double mutants R197A T276A; R197A T276F; R197F T276A; R197F T276F. Each reaction setup contained 35ul water, 10ul <A href="http://www.finnzymes.com/highperformancepcr.html" class="external text" rel="nofollow">Finnzyme</A> Phusion polymerase HF buffer, 1ul 10mM dNTPs, 5ul primer mix (primers at 100ng/ul), 1ul of diluted DNA template (approximate concentration after dilution 10ng/ul), 0.5ul <A href="http://www.finnzymes.com/highperformancepcr.html" class="external text" rel="nofollow">Finnzyme</A> Phusion Hot Start polymerase. The reaction was run with the following thermocycler program: 95C for 30sec, cycle start, 95C 30 sec, 55C 1min, 72C 2min, cycle end, 18 repeats, 72C for 30min, 4C hold. After the PCR reaction had gone to completion, 10ul of 50mM MgCl2 were added to each reaction tube and the contents were thoroughly mixed. 1ul of DpnI (equivalent to 20 units) were added to each tube and the contents were thoroughly mixed. The reaction was incubated in the thermocycler following the program: 37C for 120min, 4C hold. Samples were purified with Qiaquick PCR purification kit and separated on a 1% agarose gel stained with ethidium bromide. The additional bands at ~300bp visible in lanes 5, 7 and 8 are PCR products that span the distance between the two sets of primers of the double mutation.  B 1% agarose gel of PCR products of site directed mutagenesis of <A href="http://partsregistry.org/Part:BBa_I763026" class="external text" rel="nofollow">I763026</A>. Template DNA of BioBrick <A href="http://partsregistry.org/Part:BBa_I763026" class="external text" rel="nofollow">I763026</A> was amplified by PCR in eight different reactions using primers 276F forward 5’-GGATACGACGATTTTGAAGACAGCTC; T276A forward 5’-GGATACGACGATGCGGAAGACAGCTC; R197F reverse 5’-CAGCCAGCCAGAAACAGACGCGCCG ; R197A reverse 5’-CAGCCAGCCAGCGCCAGACGCGCCG; T276F reverse 5’-GAGCTGTCTTCAAAATCGTCGTATCC ; T276A reverse 5’-GAGCTGTCTTCCGCATCGTCGTATCC in combinations as to produce the desired mutations R197A; R197F; T276A;  T276F and double mutants R197A T276A; R197A T276F; R197F T276A; R197F T276F. Each reaction setup contained 35ul water, 10ul <A href="http://www.finnzymes.com/highperformancepcr.html" class="external text" rel="nofollow">Finnzyme</A> Phusion polymerase HF buffer, 1ul 10mM dNTPs, 5ul primer mix (primers at 100ng/ul), 1ul of diluted DNA template (approximate concentration after dilution 10ng/ul), 0.5ul <A href="http://www.finnzymes.com/highperformancepcr.html" class="external text" rel="nofollow">Finnzyme</A> Phusion Hot Start polymerase. The reaction was run with the following thermocycler program: 95C for 30sec, cycle start, 95C 30 sec, 55C 1min, 72C 2min, cycle end, 18 repeats, 72C for 30min, 4C hold. After the PCR reaction had gone to completion, 10ul of 50mM MgCl2 were added to each reaction tube and the contents were thoroughly mixed. 1ul of DpnI (equivalent to 20 units) were added to each tube and the contents were thoroughly mixed. The reaction was incubated in the thermocycler following the program: 37C for 120min, 4C hold. Since samples were transformed into chemically competent cells, purification was skipped and samples were separated on a 1% agarose gel stained with ethidium bromide. The additional bands at ~300bp visible in lanes 5, 6, 7 and 8 are PCR products that span the distance between the two sets of primers of the double mutation.</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Testing_of_the_Pulse_Generator">Testing of the Pulse Generator</SPAN></H3><P>A pulse generator for synthetic biology must fulfill several criteria in order to be considered functional. Before activation of protein expression, leaky expression of the target protein should be as low as possible. However, upon induction, protein expression must increase markedly and remain at a high level until expression is terminated by the second signal (tetracycline). Having completed one version of the pulse generator before freezing of the Wiki (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142027" class="external text" rel="nofollow">K142027</A>), we have been able to perform limited tests on the construct. There were two major concerns with respect to pulse generator function. Leaky expression from the target plasmid (we chose lac-controlled fluorescent protein generators) was obvious and we approached the problem by adding a LacI wild-type expression cassette to the target plasmid. (For a more advanced assembly the constitutive LacI wild-type expression cassette could be added to the pulse generator to reduce cloning effort for the user.) Another concern was that leaky expression of LacI IS could repress the lac-controlled promoter of the target gene despite absence of tetracycline making the pulse generator unresponsive to IPTG. 
</P><H4><SPAN class="mw-headline" id="Prevention_of_Leaky_Expression">Prevention of Leaky Expression</SPAN></H4><P>While we had originally intended to use fluorescent proteins under lac-controlled promotors by themselves as target vectors, we were concerned that even the relatively low copy number (10-12 per cell) plasmid <A href="http://partsregistry.org/Part:pSB3K3" class="external text" rel="nofollow">pSB3K3</A> would offer enough binding sites for LacI to titrate away the wild-type lac repressor. Indeed, leaky expression is rather noticeable in parts <A href="http://partsregistry.org/Part:BBa_I763004" class="external text" rel="nofollow">I763004</A>, <A href="http://partsregistry.org/Part:BBa_J04430" class="external text" rel="nofollow">J04430</A>, <A href="http://partsregistry.org/Part:BBa_I13601" class="external text" rel="nofollow">I13601</A> and <A href="http://partsregistry.org/Part:BBa_J5526" class="external text" rel="nofollow">J5526</A>, in some of them even with bare eyes. Therefore, we increased the number of wild-type LacI generators in the cell by including constitutive LacI expression cassettes in our target vectors. This effort resulted in devices <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142073" class="external text" rel="nofollow">K142073</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142074" class="external text" rel="nofollow">K142074</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142075" class="external text" rel="nofollow">K142075</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142076" class="external text" rel="nofollow">K142076</A>, which can still be induced with IPTG to yield rather strong fluorescence (except for part <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142075" class="external text" rel="nofollow">K142075</A>, which showed extremely weak fluorescence).
</P><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Figure 10: A 1% agarose gel of XbaI/PstI test restriction digests of plasmid <A href="http://partsregistry.org/Part:pSB3K3" class="external text" rel="nofollow">pSB3K3</A> holding fluorescent protein generators <A href="http://partsregistry.org/Part:BBa_I763004" class="external text" rel="nofollow">I763004</A>, <A href="http://partsregistry.org/Part:BBa_J04430" class="external text" rel="nofollow">J04430</A>, <A href="http://partsregistry.org/Part:BBa_I13601" class="external text" rel="nofollow">I13601</A> and <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">J04450</A>. After ligation four colonies were picked per construct and a test restriction digest was carried out to identify the plasmids holding the correct inserts. B Having subcloned <A href="http://partsregistry.org/Part:BBa_I763004" class="external text" rel="nofollow">I763004</A>, <A href="http://partsregistry.org/Part:BBa_J04430" class="external text" rel="nofollow">J04430</A>, <A href="http://partsregistry.org/Part:BBa_I13601" class="external text" rel="nofollow">I13601</A> and <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">J04450</A>, respectively, into plasmid <A href="http://partsregistry.org/Part:pSB3K3" class="external text" rel="nofollow">pSB3K3</A>, the resulting plasmid was linearized by SpeI/PstI restriction digest and the constitutive LacI expression cassette <A href="http://partsregistry.org/Part:BBa_I739002" class="external text" rel="nofollow">I739002</A> was cloned behind the fluorescent protein generators to build devices <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142073" class="external text" rel="nofollow">K142073</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142074" class="external text" rel="nofollow">K142074</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142075" class="external text" rel="nofollow">K142075</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142076" class="external text" rel="nofollow">K142076</A>. After transformation, devices <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142073" class="external text" rel="nofollow">K142073</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142074" class="external text" rel="nofollow">K142074</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142075" class="external text" rel="nofollow">K142075</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142076" class="external text" rel="nofollow">K142076</A> were streaked onto LB-agar plates containing 1mM IPTG and imaged under UV light with a Nikon D80 SLR</DIV></DIV></DIV><P>As shown in the next chapter, induction of cells co-transformed with pulse generator <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142027" class="external text" rel="nofollow">K142027</A> and target plasmid <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142073" class="external text" rel="nofollow">K142073</A> yielded a strong increase in fluorescence.
</P><H4><SPAN class="mw-headline" id="Expression_Level_upon_Induction">Expression Level upon Induction</SPAN></H4><P>Due to the remarkable strength of the LacI IS repressor evidenced in our genetic experiment, we were concerned that if LacI IS was transcribed in small amounts despite tet-repressor regulation, which is supposed to be extremely tight, it might repress expression of the target protein despite induction and result in a pulse generator that is unable to initiate expression of the target protein upon induction. In order to test induction of a lac-controlled gene in presence of the pulse generator, we co-transformed pulse generator <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142027" class="external text" rel="nofollow">K142027</A> and target plasmid <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142073" class="external text" rel="nofollow">K142073</A> into DH5-alpha cells. We plated the cells harboring the pulse generator as well as the target plasmid on LB-agar with and without IPTG and compared the fluorescence of the colonies.
</P><DIV class="thumb tnone"><DIV class="thumbinner" style="width:823px;"><DIV class="thumbcaption">Figure 11: Pulse generator construct  <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142027" class="external text" rel="nofollow">K142027</A> was produced from device <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142047" class="external text" rel="nofollow">K142047</A> using PCR-based site-directed mutagenesis as described previously. After transformation by electroporation of purified PCR product into DH5-alpha cells, eight colonies were selected and plasmids were purified using Qiagen Miniprep kit. The eight plasmids have been co-transformed together with target plasmid <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K142073" class="external text" rel="nofollow">K142073</A> into electrocompetent DH5-alpha cells and were plated on LB-agar supplemented with carbenicillin and kanamycin. Upon incubation overnight at 37C, numerous colonies appeared and one colony from each plate was streaked onto a LB-agar plate supplemented with carbenicillin and kanamycin and on a LB-agar plate supplemented with carbenicillin, kanamycin and 1mM IPTG. Cells were incubated overnight at 37C for the same duration and imaged with a Nikon D80 SLR on the UV lamp of a Kodak Gel Logic 200 imaging system.</DIV></DIV></DIV><P>While induction is not successful in all eight strains, we see a considerable increase of fluorescence in four of them (stars). Obviously, leaky expression of the LacI IS uninducible repressor has not occurred to the extent that would make induction impossible, since all eight strains can be seen expressing fluorescent protein under control of the lac repressor. Since we have not been able to get the construct sequenced yet, we have no definite proof of the functionality of the pulse generator, however, based on this experiment we can with great confidence exclude any complications arising from premature expression of LacI IS.
</P><H4><SPAN class="mw-headline" id="Termination_of_Expression_upon_Addition_of_Tetracycline">Termination of Expression upon Addition of Tetracycline</SPAN></H4><P>Unfortunately we have not been able to conduct experiments on termination of expression by induction of repressor with tetracycline before the Wiki freeze. However, we are optimistic about conducting the experiments before the Jamboree and we might be able to present results during our talk or on our poster.
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P>(1)	Studier, F. W., Rosenberg, A. H., Dunn, J. J., and Dubendorff, J. W. (1990) Use of T7 RNA polymerase to direct expression of cloned genes. Methods Enzymol 185, 60-89.
</P><P>(2)	Rosenberg, A. H., Lade, B. N., Chui, D. S., Lin, S. W., Dunn, J. J., and Studier, F. W. (1987) Vectors for selective expression of cloned DNAs by T7 RNA polymerase. Gene 56, 125-35.
</P><P>(3)	Studier, F. W., and Moffatt, B. A. (1986) Use of bacteriophage T7 RNA polymerase to direct selective high-level expression of cloned genes. J Mol Biol 189, 113-30.
</P><P>(4)	Beumer, K., Bhattacharyya, G., Bibikova, M., Trautman, J. K., and Carroll, D. (2006) Efficient gene targeting in Drosophila with zinc-finger nucleases. Genetics 172, 2391-403.
</P><P>(5)	Bibikova, M., Beumer, K., Trautman, J. K., and Carroll, D. (2003) Enhancing gene targeting with designed zinc finger nucleases. Science 300, 764.
</P><P>(6)	Carroll, D. (2008) Progress and prospects: Zinc-finger nucleases as gene therapy agents. Gene Ther.
</P><P>(7)	Carroll, D., Morton, J. J., Beumer, K. J., and Segal, D. J. (2006) Design, construction and in vitro testing of zinc finger nucleases. Nat Protoc 1, 1329-41.
</P><P>(8)	Suckow, J., Markiewicz, P., Kleina, L. G., Miller, J., Kisters-Woike, B., and Muller-Hill, B. (1996) Genetic studies of the Lac repressor. XV: 4000 single amino acid substitutions and analysis of the resulting phenotypes on the basis of the protein structure. J Mol Biol 261, 509-23.
</P><P>(9)	Saenger, W., Orth, P., Kisker, C., Hillen, W., and Hinrichs, W. (2000) The Tetracycline Repressor-A Paradigm for a Biological Switch. Angew Chem Int Ed Engl 39, 2042-2052.
</P><P>(10)	Hinrichs, W., Kisker, C., Duvel, M., Muller, A., Tovar, K., Hillen, W., and Saenger, W. (1994) Structure of the Tet repressor-tetracycline complex and regulation of antibiotic resistance. Science 264, 418-20.
</P><P>(11)	Gilbert, W., and Muller-Hill, B. (1966) Isolation of the Lac Repressor. Proc Natl Acad Sci U S A 56, 1891-1898.
</P><P>(12)	Calos, M. P., Galas, D., and Miller, J. H. (1978) Genetic studies of the lac repressor. VIII. DNA sequence change resulting from an intragenic duplication. J Mol Biol 126, 865-9.
</P><P>(13)	Coulondre, C., and Miller, J. H. (1977) Genetic studies of the lac repressor. III. Additional correlation of mutational sites with specific amino acid residues. J Mol Biol 117, 525-67.
</P><P>(14)	Coulondre, C., and Miller, J. H. (1977) Genetic studies of the lac repressor. IV. Mutagenic specificity in the lacI gene of Escherichia coli. J Mol Biol 117, 577-606.
</P><P>(15)	Farabaugh, P. J., Schmeissner, U., Hofer, M., and Miller, J. H. (1978) Genetic studies of the lac repressor. VII. On the molecular nature of spontaneous hotspots in the lacI gene of Escherichia coli. J Mol Biol 126, 847-57.
</P><P>(16)	Kleina, L. G., and Miller, J. H. (1990) Genetic studies of the lac repressor. XIII. Extensive amino acid replacements generated by the use of natural and synthetic nonsense suppressors. J Mol Biol 212, 295-318.
</P><P>(17)	Markiewicz, P., Kleina, L. G., Cruz, C., Ehret, S., and Miller, J. H. (1994) Genetic studies of the lac repressor. XIV. Analysis of 4000 altered Escherichia coli lac repressors reveals essential and non-essential residues, as well as &quot;spacers&quot; which do not require a specific sequence. J Mol Biol 240, 421-33.
</P><P>(18)	Miller, J. H. (1979) Genetic studies of the lac repressor. XI. On aspects of lac repressor structure suggested by genetic experiments. J Mol Biol 131, 249-58.
</P><P>(19)	Miller, J. H. (1984) Genetic studies of the lac repressor. XII. Amino acid replacements in the DNA binding domain of the Escherichia coli lac repressor. J Mol Biol 180, 205-12.
</P><P>(20)	Miller, J. H., Coulondre, C., Hofer, M., Schmeissner, U., Sommer, H., Schmitz, A., and Lu, P. (1979) Genetic studies of the lac repressor. IX. Generation of altered proteins by the suppression of nonsence mutations. J Mol Biol 131, 191-222.
</P><P>(21)	Miller, J. H., Ganem, D., Lu, P., and Schmitz, A. (1977) Genetic studies of the lac repressor. I. Correlation of mutational sites with specific amino acid residues: construction of a colinear gene-protein map. J Mol Biol 109, 275-98.
</P><P>(22)	Miller, J. H., and Schmeissner, U. (1979) Genetic studies of the lac repressor. X. Analysis of missense mutations in the lacI gene. J Mol Biol 131, 223-48.
</P><P>(23)	Schmeissner, U., Ganem, D., and Miller, J. H. (1977) Genetic studies of the lac repressor. II. Fine structure deletion map of the lacI gene, and its correlation with the physical map. J Mol Biol 109, 303-26.
</P><P>(24)	Schmitz, A., Coulondre, C., and Miller, J. H. (1978) Genetic studies of the lac repressor. V. Repressors which bind operator more tightly generated by suppression and reversion of nonsense mutations. J Mol Biol 123, 431-54.
</P><P>(25)	Sommer, H., Schmitz, A., Schmeissner, U., and Miller, J. H. (1978) Genetic studies of the lac repressor. VI. The B116 repressor: an altered lac repressor containing amino acid specified by both the trp and lacI leader regions. J Mol Biol 123, 457-69.
</P><P>(26)	Friedman, A. M., Fischmann, T. O., and Steitz, T. A. (1995) Crystal structure of lac repressor core tetramer and its implications for DNA looping. Science 268, 1721-7.
</P><P>(27)	Lewis, M., Chang, G., Horton, N. C., Kercher, M. A., Pace, H. C., Schumacher, M. A., Brennan, R. G., and Lu, P. (1996) Crystal structure of the lactose operon repressor and its complexes with DNA and inducer. Science 271, 1247-54.
</P><P>(28)	Yamagishi, M., Matsushima, H., Wada, A., Sakagami, M., Fujita, N., and Ishihama, A. (1993) Regulation of the Escherichia coli rmf gene encoding the ribosome modulation factor: growth phase- and growth rate-dependent control. Embo J 12, 625-30.
</P><P>(29)	Wada, A., Yamazaki, Y., Fujita, N., and Ishihama, A. (1990) Structure and probable genetic location of a &quot;ribosome modulation factor&quot; associated with 100S ribosomes in stationary-phase Escherichia coli cells. Proc Natl Acad Sci U S A 87, 2657-61.
</P><P>(30)	Yoshida, H., Yamamoto, H., Uchiumi, T., and Wada, A. (2004) RMF inactivates ribosomes by covering the peptidyl transferase centre and entrance of peptide exit tunnel. Genes Cells 9, 271-8.
</P><P>(31)	Yoshida, H., Maki, Y., Kato, H., Fujisawa, H., Izutsu, K., Wada, C., and Wada, A. (2002) The ribosome modulation factor (RMF) binding site on the 100S ribosome of Escherichia coli. J Biochem 132, 983-9.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:ETH_Zurich/Wetlab/Switch_Circuit">http://2008.igem.org/Team:ETH_Zurich/Wetlab/Switch_Circuit</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Wetlab/Switch_Circuit" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Wetlab/Switch_Circuit" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Switch_Circuit&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Wetlab/Switch_Circuit&amp;oldid=104091" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>