"
Page
Discussion
View source
History
teams
Log in
 
Team:ETH Zurich/Modeling/Switch Circuit
From 2008.igem.org
Contents
1 Switch Circuit
2 Implementation and Simulation
2.1 Implementation
2.2 Simulation Results
2.3 Sensitivity Analysis
3 Detailed Model
3.1 diffusion of IPTG
3.2 diffusion of tet
3.3 binding of IPTG to LacI
3.4 binding of tet to TetR
3.5 binding of TetR to LacIIs-promoter
3.6 binding of LacI and LacIIs to GFP-promoter
3.7 transcription and translation of LacI
3.8 transcription and translation of LacIIs
3.9 transcription and translation of TetR
3.10 transcription and translation of GFP
3.11 dimerization and tetramerization of LacI and LacIIs
3.12 dimerization of tetR
4 Parameters
5 References
Switch Circuit
The designed switching circuit is driven by  two input signals – a start signal initiates the synthesis of a specific protein and a terminating signal switches gene expression off. The goal of this system is to control expression of restriction enzymes in order to delete genome fragments in vivo. In order to do some preliminary experiments, the restriction enzyme has been substituted by a fluorescent protein.
The detailed mechanism is described here . It follows a summarized version of it's mode of operation:
Fluorescent protein gene expression is under control of LacI and can be induced by the addition of IPTG. In order to stop gene expression, the IPTG-sensitive LacI is replaced by IPTG-insensitive LacIIs, which shuts off fluorescent gene expression again. The synthesis of LacIIs is started by the addition of Tetracyclin (tet) to the system, which binds to the tet repressor TetR and thus de-represses the expression of the LacIIs gene. The fluorescent protein is tagged, so that it is degraded faster by the Clp protease and vanishes faster from the system as its expression is stopped.
This switching circuit is described by a set of 64 chemical reactions and 41 molecular species. In order to do a computational analysis of the circuit, this model has been simplified and implemented in MATLAB. Then the system has been simulated using ODE and stochastic solvers.
Implementation and Simulation
Implementation
In the implementation, some minor parts that do not have significant effects on the aspired results have been neglected for the sake of simplicity. In the MATLAB-model of the different protein-expressions, the RNA-polymerase and the ribosomes have not been taken into account, in the transcription and translation respectively. Furthermore, the effects of the dimerization of TetR as well as the impacts of dimerization and tetramerization of LacI and LacIIs (4) have not been considered in the final implementation.
This simplified model still comprises more than 20 different molecular species and over 30 kinetic reactions and was implemented by using the SimBiology Toolbox in MATLAB.
diagram view of the model
We performed deterministic and stochastic simulations based on Mass Action Kinetics. The stochastic simulations turned out to be computationally very exhaustive but generated no further significant information compared to the deterministic simulations.
Simulation Results
Input signals: Start signal (Induction with IPTG) and Stop signal (induction with tet) after 10 min
Output signal: expression of GFP and GFP_mRNA as a response to the input signals on the left
The simulations show that our system actually should create a nice pulse-shaped expression of the fluorescent protein (GFP). This expression can be initiated by inducing with IPTG and stopped by subsequent addition of tet into the medium. By tagging the protein it will be degraded much faster by the Clp protease, so that the overall concentration is bounded and, after activating the stop-signal, the remaining proteins disappear quickly.
Another fact that the simulations showed is that in order to get one single pulse the tet-concentration inside, the medium must not reach zero before all the IPTG is degraded too. Otherwise there would still be IPTG in the system inhibiting the binding of LacI to the GFP-promoter and leading to an unwanted expression of our protein of interest as the LacIIs degrades.
One way to overcome this problem is by simply inducing with a much higher quantity of tet than IPTG, so that it simply takes longer for it to completely degrade or being washed away. Still another way would be to wait a bit longer after the induction with IPTG so that it has already partly vanished until switching off the GFP-expression by inducing with tet.
Sensitivity Analysis
We define the sensitivity as the change of the production of the desired fluorescence protein - which is the output of our system - depending on the change of the parameters .
Sensitivity analysis - change in the GFP concentration depending on the change of the kinetic parameters
The sensitivity analysis shows, that the concentration of the fluorescent protein strongly depends on its decay rate (parameter 13) the decay rate of its mRNA (parameter 10) and of course the transcription and translation rates of the protein (parameters 29 and 27), which is no surprise. We can also see that the decay rates of LacIIs (parameter 9) and LacIIs_mRNA (parameter 12) and the transcription rate of LacIIs (parameter 31) have an influence on the expression of the fluorescent protein.
Detailed Model
diffusion of IPTG
In order to switch on the circuit, we induce with IPTG. When IPTG is added into the medium it diffuses reversibly between the medium and the cells, where it is slowly degraded. In a chemostat extracellular IPTG is washed away.
diffusion of tet
The second inducer which is used in our system is tet. This one also diffuses reversibly between the medium and the cells, where it is slowly degraded and is washed away in the medium.
binding of IPTG to LacI
The first inducer IPTG can bind to the tetramerized LacI (4) .
binding of tet to TetR
The second inducer tet can bind to TetR.
binding of TetR to LacIIs-promoter
TetR is constitutively expressed and binds to the LacIIs promoter, inhibiting its expression.
binding of LacI and LacIIs to GFP-promoter
LacI which is constitutively expressed and LacIIs which is under the control of a tet repressor can bind both to the GFP promotor.
transcription and translation of LacI
RNA polymerase binds to the LacI-promoter and transcribes it into LacI-mRNA
RNA polymerase detaches from the LacI-mRNA
degradation of LacI-mRNA
ribosome binds to LacI-mRNA and translates it into LacI
ribosome detaches from LacI
degradation of LacI
transcription and translation of LacIIs
RNA polymerase binds to the LacIIs-promoter and transcribes it into LacIIs-mRNA
RNA polymerase detaches from the LacIIs-mRNA
degradation of LacIIs-mRNA
ribosome binds to LacIIs-mRNA and translates it into LacIIs
ribosome detaches from LacIIs
degradation of LacIIs
transcription and translation of TetR
RNA polymerase binds to the TetR-promoter and transcribes it into TetR-mRNA
RNA polymerase detaches from the TetR-mRNA
degradation of TetR-mRNA
ribosome binds to TetR-mRNA and translates it into TetR
ribosome detaches from TetR
degradation of TetR
transcription and translation of GFP
RNA polymerase binds to the GFP-promoter and transcribes it into GFP-mRNA
RNA polymerase detaches from the GFP-mRNA
degradation of GFP-mRNA
ribosome binds to TGFP-mRNA and translates it into GFP
ribosome detaches from GFP
degradation of GFP
dimerization and tetramerization of LacI and LacIIs
dimerization of tetR
Parameters
In this section you can find all the parameters used in the simulation.
Because many of the in vivo rates of the biochemical reactions we simulated are unknown or could not be found in the literature, the kinetic parameters were mainly obtained from estimates (2) based on the values found in the supporting text to (1) .
#
Parameter name
Value
Units
Reference/Comment
1
k_assoc(IPTG_LacI)
5.0
1/(mole*second)
Estimate
2
k_assoc(LacI)
5.0
1/(mole*second)
Estimate
3
k_assoc(LacIIs)
5.0
1/(mole*second)
Estimate
4
k_assoc(tet)
5.0
1/(mole*second)
Estimate
5
k_assoc(tetR)
5.0
1/(mole*second)
Estimate
6
k_dec(IPTG)
0.0002
1/second
Estimate
7
k_dec(IPTG_ext)
0.001
1/second
[*]
8
k_dec(LacI)
0.05
1/second
Estimate
9
k_dec(LacIIs)
0.05
1/second
Estimate
10
k_dec(gfp)
0.25
1/second
[**]
11
k_dec(mRNA_LacI)
0.05
1/second
Estimate
12
k_dec(mRNA_LacIIs)
0.05
1/second
Estimate
13
k_dec(mRNA_gfp)
0.1
1/second
[**]
14
k_dec(mRNA_tetR)
0.05
1/second
Estimate
15
k_dec(tetR)
0.05
1/second
Estimate
16
k_dec(tet)
0.0002
1/second
Estimate
17
k_dec(tet_ext)
0.001
1/second
[*]
18
k_diff(IPTG)
0.1
1/second
(1)
19
k_diff(tet)
0.1
1/second
(1)
20
k_dissoc(IPTG_LacI)
1.0
1/second
Estimate
21
k_dissoc(LacI)
1.0
1/second
Estimate
22
k_dissoc(LacIIs)
1.0
1/second
Estimate
23
k_dissoc(tet)
1.0
1/second
Estimate
24
k_dissoc(tetR)
1.0
1/second
Estimate
25
k_tl(LacI)
5.0
1/second
Estimate
26
k_tl(LacIIs)
5.0
1/second
Estimate
27
k_tl(gfp)
5.0
1/second
(1)
28
k_tl(tetR)
5.0
1/second
Estimate
29
k_tr(gfp)
1.0
1/second
(1)
30
k_tr(LacI)
0.1
1/second
[***]
31
k_tr(LacIIs)
0.1
1/second
[***]
32
k_tr(tetR)
1.0
1/second
Estimate
[*]   the degradation constants of the two inducers are bigger outside the cell, the effect that they are washed away in the chemostat is taken into account in those parameters
[**]  degradation rates of gfp are higher because of the tagging
[***] need to be that high to account for the autorepression of LacI / in order to get a low steady state concentration of LacI of about 50 proteins (3)
References
(1) "Spatiotemporal control of gene expression with pulse-generating networks", Basu et al., PNAS, 2004
(2) "Genetic circuit building blocks for cellular computation, communications, and signal processing", Weiss et al., Natural Computing, 2003
(3) "Predicting stochastic gene expression dynamics in single cells", Mettetal et al., PNAS, 2006
(4) "Engineered gene circuits", Hasty et al., Nature, 2002
Retrieved from " http://2008.igem.org/Team:ETH_Zurich/Modeling/Switch_Circuit "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
