"
Page
Discussion
View source
History
teams
Log in
 
Team:Colorado State/Project
From 2012.igem.org
Home
Home
The Brew
Project Summary
Parts Submitted to the Registry
Results
Safety
Modeling
Team Notebook
Calendar
The Brewers
Meet the Students
Meet the Instructors
Meet the Advisors
Official Team Page
Sponsors
Attributions
Edit page
Project Summary
We are currently pursuing the concept of brewing a gluten-free beer using a yeast that secretes an enzyme to break down gluten.
As Fort Collins is a major brewing hub, it was natural for our team to gravitate toward a beer-related project. Knowing full well the problems caused by Celiac disease, and the affinity many others have for reducing gluten in their diets, we decided to design and create a yeast strain capable of both fermenting quality beer, and breaking down gluten. To accomplish this we had to address several issues:
What is the immunological basis for gluten intolerance i.e. gluten’s antigenic qualities?
Celiac disease is an autoimmune disorder that affects the digestive system. People with Celiac disease suffer from a severe reaction when exposed to gluten, a protein found in wheat, rye, and barley. Glutamine and proline rich regions of gluten family proteins, which include gliadin in wheat and hordein in barley, are incredibly stable and cannot be broken down by the stomach. The antigenicity of gluten seems to be due to these proline and glutamine rich peptides, the most prevalent one being the 33-mer: LQLQPFPQPQLPYPQPQLPYPQPQLPYPQPQPF, found in gliadin. The sequence seems to be fairly conserved across barley and wheat. Three distinct patient-specific T cell epitopes identified previously in T cell proliferation assays are present in this peptide, namely, PFPQPQLPY, PQPQLPYPQ (three copies), and PYPQPQLPY (two copies). Upon reaching the intestines, these peptides are processed by a Celiac’s immune system, producing a response that damages villi in the small intestine and interferes with absorption of nutrients from food. [1] [2]
Where can we find a viable enzyme?
Our search for an enzyme capable of breaking down gluten and neutralizing its toxicity led us first to the enzyme mutated by the 2011 UW iGEM team. The modified Kumamolisin-As has a maximal activity at a pH of 4 and would work well in the pH range of 5.2-5.5 found in beer. For expression in yeast we had to account for codon bias, and optimized the sequence so it could more easily be moved from a prokaryotic system to a eukaryotic one. Another promising enzyme was AN-PEP. This protease already cleaves after proline residues and is stable at low pH. Used in the cocktail known as Brewers Clarex, AN-PEP has been shown to reduce gluten levels in beer when it is added to a final product. Unfortunately, the enzyme is protected by patent and was unavailable to us.
How will the enzyme be secreted?
Yeast have two mating types, called “a” and “
$\alpha$
”. Yeast of the alpha mating type secrete mating factor-alpha (MF-
\alpha
), which is tagged with a secretion factor at the N-terminus of the protein. Experimental evidence shows that this signal sequence is cleaved in the golgi before MF-
\alpha
is exported from the cell. It is commonly used by researchers to mark foreign proteins for secretion in laboratory yeast. The tag’s sequence was placed directly upstream of the sequence for Kumamolisin. The DNA itself was synthesized by IDT, but we were also able to extract it from the yeast genome by PCR. [1] [2]
How will we move our system into yeast?
Yeast can take up plasmids from the environment, but without selective pressure will drop them quickly. This would not be ideal for our purposes, as it would be unrealistic and potentially harmful to add antibiotics to every batch of beer just to ensure plasmid retention. As a proof of concept we introduced our system into yeast using plasmids, but we were also in the process of developing an integrative plasmid that would insert our sequence directly into the yeast genome.
Two plasmids were used:
pCM189: centromeric yeast plasmid, marker URA3, tetracycline repressed expression of target gene under control of tetO2, low copy number
pCM190: episomal yeast plasmid, marker URA3, tetracycline repressed expression of target gene under control of tetO7, high copy number
The integrative plasmid was constructed by adding the resistance gene for geneticin to pCM189/MF-alpha/Kuma-max.
How will we assay the enzyme and screen for gluten?
Before we move our engineered yeast into beer, it will be necessary to determine whether or not the enzyme is performing as expected. In 2011 the UW iGEM team assayed mutated Kumamolisin-As using fluorescence. We hope to repeat their tests in the lab using the peptide sequence PQPQLP with attached fluorophore and quencher. When the sequence is cleaved, the fluorophore will be separated from the quencher and will fluoresce. Ideally our results will resemble Washington’s.
Once we have determined whether the enzyme is present, and how active it is, the next step will be to assess how well it can break down the naturally occurring peptides found in beer. Studies have shown that the brewing process breaks native gluten proteins into smaller peptides, some of which are still immunogenic. However, it will still be necessary to confirm that Kumamolisin-As can manage these peptides and neutralize their toxicity. To do so, we have prepared a competitive ELISA to detect the presence of gluten antigens after incubation with Kumamolisin-As.
What will be the effects of the secreted protein on beer?
Flavor/head/haze
Project Details
Yeast
Assemble plasmids
Integrative: industrial strain
Centromeric: Ethan
Episomal: Guy
His Tag Plasmids: Dave
C-terminus
N-terminus
Clone mating factor alpha: Steven
Bacteria
Assemble plasmids (3A): Steven
His Tag Plasmids: Dave
C-terminus
N-terminus
Secretion: Ethan
Functional Protease Assay
Flourophore/Quencher: Ryan
Mass Spec: Ryan/Guy
Elisa: Guy
Modeling: Dave/Ryan/Ethan
Retrieved from " http://2012.igem.org/Team:Colorado_State/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
