<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NCKU_Tainan_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NCKU Tainan/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="navbar navbar-expand-lg navbar-light fixed-top" style="background-color: #5DD789; padding-top: 15px;"><A class="navbar-brand" href="https://2018.igem.org/Team:NCKU_Tainan" style="font-size: 30px; color: white;">NCKU Tainan</A></NAV></P><DIV class="collapse navbar-collapse" id="navbarSupportedContent"><UL class="navbar-nav mr-auto"><LI class="nav-item dropdown"><A class="nav-link" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                                Project
                            </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Improve">Improve</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Protocol">Protocol</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Measurement">Measurement</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Parts">Parts</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                            Model
                            </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Model">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Kinetic_Law">Kinetic Law</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Analysis">Analysis</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                            Device
                            </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Software">Software</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Hardware">Hardware</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Demonstrate">Demonstrate</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/pH_meter">pH Meter</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Temperature">Thermometer</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/wi_fi">Wi-Fi Module</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/CO2">CO<SUB>2</SUB> Sensor</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:NCKU_Tainan/Safety">Safety</A></LI><LI class="nav-item dropdown"><A class="nav-link" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                            Human Practices
                            </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Human_Practices">About</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Integrated_Human_Practices">Integrated Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Public_Engagement">Education &amp; Public Engagement</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                            Product Design
                            </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Applied_Design">Product design</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Entrepreneurship">Entrepreneurship</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                            Team
                            </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Team">Team Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Collaborations">Collaborations</A><A class="dropdown-item" href="https://2018.igem.org/Team:NCKU_Tainan/Attributions">Attributions</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:NCKU_Tainan/Notebook">Notebook</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:NCKU_Tainan/Judge">Awards</A></LI></UL></DIV><DIV class="container content"><DIV class="headstyle"><H1 class="head">Design</H1></DIV><DIV class="righttitle"><H6 class="subtitle">Bring Solutions, Not Problems</H6></DIV><DIV class="navbar-example"><DIV class="row"><DIV class="col-2 side"><DIV id="sidelist" class="list-group"><A class="list-group-item list-group-item-action" href="#Overview">Overview</A><A class="list-group-item list-group-item-action" href="#Chassis_Organism">Chassis Organism</A><A class="list-group-item list-group-item-action" href="#Prk">Prk</A><A class="list-group-item list-group-item-action" href="#Rubisco">Rubisco</A><A class="list-group-item list-group-item-action" href="#CA">CA</A><A class="list-group-item list-group-item-action" href="#dual_plasmid_system">Dual plasmid system</A><A class="list-group-item list-group-item-action" href="#pH_sensing_system">pH sensing system</A><A class="list-group-item list-group-item-action" href="#Reference">References</A></DIV></DIV><DIV class="col-10"><DIV data-spy="scroll" data-target="#sidelist" data-offset="0" class="scrollspy-example"><DIV class="container"><DIV id="Overview"><H3>Overview of Designed Pathway</H3><P class="pcontent">Calvin-Benson cycle is one of the most important pathways for inorganic carbon to be converted into organic carbon in the carbon cycle. 
                                        Plant, algae, and cyanobacteria utilize light as energy source for Calvin-Benson cycle. 
                                        Taking the advantage of the pentose phosphate pathway, a native metabolic pathway of <I>E. coli</I>, 
                                        only two additional enzymes will be needed to reconstruct the pathway in <I>E. coli</I> -- PRK and Rubisco, 
                                        which we will describe more in detail. 
                                        The primary product of the pathway is pyruvate, which can be utilized to produce various valuable products.
                                    </P></DIV><DIV id="Chassis_Organism"><H3>Chassis Organism</H3><P class="pcontent">We would like to test our pathway in various <I>E. coli</I> strains to see the functionality of our construction. 
                                        We selected three different strains: BL21 (DE3), W3110, W3110 (L5T7). 
                                        BL21 (DE3) is a common expression strain that is widely used to express recombinant proteins using T7 polymerase. 
                                        We expected that the high production of protein may change the entire native metabolic pathway. 
                                        W3110 (K-12 laboratory strain) is reported to be resilient in a stressed environment. 
                                        We expected that W3110 will grow well even if the sole carbon source is xylose. 
                                        W3110 (L5T7) (provided by Dr. Ng) is a constructed lab strain based on W3110. 
                                        T7 polymerase was inserted into its genome.
                                    </P></DIV><DIV id="Prk"><H3>PRK</H3><H5 class="question">What is its function?</H5><P class="pcontent">We first introduced phosphoribulokinase (PRK), an enzyme from cyanobacterial Calvin cycle, 
                                        into the central carbon metabolic pathway of <I>E. coli</I>. 
                                        PRK catalyzes the conversion of ribulose-5-phosphate (Ru5P) from the pentose phosphate pathway of 
                                        the central carbon metabolism to ribulose-1,5-biphosphate (RuBP). 
                                        One ATP is required for this conversion. 
                                        The accumulation of RuBP in <I>E.coli</I> would cause cell growth arrest because <I>E. coli</I> can not metabolize RuBP.
                                    </P><DIV class="row"><A class="btn col-md-12" data-toggle="collapse" href="#PRK_how_to_construct" role="button" aria-expanded="false" aria-controls="multiCollapseExample1">
                                            How do we construct this part?
                                            </A></DIV><DIV class="collapse multi-collapse" id="PRK_how_to_construct"><DIV class="card card-body"><P class="pcontent">Steps involved in expressing PRK in <I>E. coli.</I>
                                                We initially confirm the gene sequence of <I>Synechococcus elongtus</I><I>prk</I> from NCBI gene database. 
                                                We then codon optimized the sequence so <I>E. coli</I> can express the protein properly. 
                                                The optimized sequence was sent to IDT for gene synthesis. 
                                                We PCR amplified the gene fragments and digest it with restriction enzymes HindIII and SpeI. 
                                                After digestion, we ligate the fragments into pSB3K3 plasmid with P<SUB>LacI</SUB>-rbs (B0034) located upstream of the fragment. 
                                                The plasmid was then transformed into DH5 alpha.
                                            </P></DIV></DIV><H5 class="question">How do we test its function?</H5><P class="pcontent">We initially decided to measure the concentration of RuBP by HPLC. 
                                        Our instructors pointed out some difficulties in HPLC measurement such as excessive noise signal in our sample. 
                                        We then designed an experiment to prove the function of PRK in an indirect manner: by measuring its growth rate. 
                                        RuBP, the product of PRK, is toxic to <I>E. coli</I>. 
                                        We expressed this protein independently in xylose M9 to check cell growth. 
                                        If the cell growth is arrested, we can indirectly conclude the function of PRK.
                                    </P></DIV><DIV id="Rubisco"><H3>Rubisco</H3><H5 class="question">What is its function?</H5><P class="pcontent">Ribulose-1,5-biphosphate carboxylase/oxygenase is one of the world’s most abundant enzyme. 
                                        It catalyzes the conversion of inorganic carbon into organic carbon. 
                                        In our designed pathway, the function of the Rubisco is to convert ribulose-1,5-biphosphate (RuBP) from the upper pathway and carbon dioxide into 3-phosphoglycerate (3PGA). 
                                        3PGA will then be converted to pyruvate by the native metabolic system of <I>E. coli</I>. 
                                        After mining information from various publications, 
                                        we selected Rubisco from <I>Synechococcus elongatus</I> PCC 7002, which is a well-studied cyanobacteria. 
                                        Its genome is completely sequenced and it is often used as a model organism for gene manipulation. 
                                        Previous research has utilized <I>E. coli</I> as a host of random mutagenesis to enhance the activity of <I>Synechococcus</I> Rubisco.
                                    </P><DIV class="row"><A class="btn col-md-12" data-toggle="collapse" href="#RuBisCO_how_to_construct" role="button" aria-expanded="false" aria-controls="multiCollapseExample1">
                                            How do we construct this part?
                                            </A></DIV><DIV class="collapse multi-collapse" id="RuBisCO_how_to_construct"><DIV class="card card-body"><P class="pcontent">Akin to the construction of <I>prk</I>, we codon optimized the sequence of three <I>rbc</I> subunit and 
                                                clone it into pSB1C3 plasmid with HindIII and SpeI. 
                                                The sequence and the size of <I>rbcL</I> is much larger than other subunit, 
                                                so we separated <I>rbcL</I> from <I>rbcX</I> and <I>rbcS</I> subunits. <I>rbcX</I> and <I>rbcS</I> is separated by a <I>rbs</I> (B0034) for the convenience of construction. 
                                                We attached two different promoters upstream of the <I>rbc</I>. They are P<SUB>LacI</SUB> and P<SUB>T7</SUB> promoter. 
                                                Since we would like to increase the expression of this protein in the metabolic pathway, 
                                                we would like to test various promoter combination to find out the most efficient combination for our pathway.
                                            </P></DIV></DIV><H5 class="question">How do we test its function?</H5><P class="pcontent">Measurement of 3PGA or pyruvate concentration could not directly reflect the activity of Rubisco 
                                        since both of them are important metabolites that will flow to downstream metabolic pathway. 
                                        We then decided to determine its function by a total solution test which we will mention below.
                                    </P></DIV><DIV id="CA"><H3>CA</H3><H5 class="question">What is its function?</H5><P class="pcontent">Rubisco is the rate-limiting enzyme in carbon fixation. 
                                        Oxygen competes with CO<SUB>2</SUB> as a substrate for Rubisco, giving rise to photorespiration. 
                                        To overcome this problem, some photosynthetic organisms have evolved their own carbon concentrating
                                        mechanism (CCM), which helps to maintain a sufficient amount of CO<SUB>2</SUB> around Rubisco.
                                    </P><P class="pcontent">We are inspired by the carbon concentrating mechanisms (CCM) of cyanobacteria. 
                                        In cyanobacteria, Rubisco and carbonic anhydrase (CA) is encapsulated in a microcompartment, the carboxysome. 
                                        Carbonic anhydrase, also known as carbonate dehydratase, is involved in the interconversion between CO<SUB>2</SUB> and HCO<SUB>3</SUB><SUP>-</SUP>. This enzyme can be found in most organisms, including <I>E. coli</I> but the difference is its catalyzing rate in hydration and dehydration of CO2. Therefore, 
                                        we will incorporate into our system the carbonic anhydrase gene from <I>Synechococcus elongatus</I> PCC 7002.
                                    </P><DIV class="row"><A class="btn col-md-12" data-toggle="collapse" href="#CA_how_to_construct" role="button" aria-expanded="false" aria-controls="multiCollapseExample1">
                                            How do we construct this part?
                                            </A></DIV><DIV class="collapse multi-collapse" id="CA_how_to_construct"><DIV class="card card-body"><P class="pcontent">We first codon optimized the sequence and insert it into the empty pSB1C3 plasmid with HindIII and 
                                                SpeI just as mentioned above. In our optimized sequence, we have already designed a P<SUB>T7</SUB> promoter 
                                                in front of <I>ca</I>, so we can directly ligate it into the plasmid. 
                                                The constructed basic part is then linked with other basic parts to complete our construction.
                                            </P></DIV></DIV><H5 class="question">How do we test its function?</H5><P class="pcontent">To measure the enzyme activity of CA, we compare the conversion rate of carbon dioxide to 
                                        bicarbonate ion. After saturated CO<SUB>2</SUB> solution is prepared, we add fixed amount of bacteria broth that 
                                        contains CA construction into the solution. We then measure the time taken for the pH value to decrease from 8.3 to 6.3. 
                                        We compare the measured time interval with the time interval that enzyme was not added to determine the enzyme activity of CA.
                                    </P></DIV><DIV id="dual_plasmid_system"><H3>Dual Plasmid System</H3><P class="pcontent">We decided to construct the whole pathway with the dual plasmid system. 
                                        Previously, every basic part was the backbone conserved in the backbone of pSB1C3. 
                                        We then link the construction together and 
                                        even change the backbone of some composite parts to pSB3K3 for a lower protein expression.
                                    </P><H5 class="question">Rubisco whole protein in pSB1C3</H5><P class="pcontent">We link each basic part together with biobrick standard method. 
                                        We link P<SUB>T7</SUB>-<I>rbcL</I> and P<SUB>T7</SUB>-<I>rbcX</I>-<I>rbcS</I> together. 
                                        The former, the insert, was digested with EcoRI and SpeI and the latter, the backbone, 
                                        is digested with EcoRI and XbaI. 
                                        We ligate the backbone with the insert to complete this composite part.
                                    </P><H5 class="question"><I>prk</I> gene into pSB3K3</H5><P class="pcontent">PRK catalyzes the reaction of converting Ru5P into RuBP. 
                                        Not native to the host, RuBP is, nonetheless, toxic to <I>E. coli</I>. 
                                        We hope that the expression of PRK could be lower in the host so we change the backbone of it into pSB3K3. 
                                        We selected J04450 from the distributed kit that under the backbone of pSB3K3, 
                                        which will express red color after the formation of the colony. We digest both backbone and insert with EcoRI and PstI and ligate both fragments. 
                                        We can then select the colony that does not present red color to prove that the ligation was conducted successfully.
                                    </P><H5 class="question">Link <I>prk</I> with <I>ccaA</I> into pSB3K3</H5><P class="pcontent">We also constructed the composite part that contains both <I>ccaA</I> and <I>prk</I>. 
                                        We construct it using the method mentioned in <I>rbc</I> whole construction. 
                                        We cloned the fragments into pSB3K3 for lower expression of PRK.
                                    </P><H5 class="question">Transformation</H5><P class="pcontent">After the construction of various composite parts, 
                                        we co-transform them into three <I>E. coli</I> strains: BL21 (DE3), W3110, and W3110 (L5T7). 
                                        Since BL21 (DE3) and W3110 (L5T7) contains T7 polymerase, 
                                        we co-transformed composite parts that contain T7 promoter into these strains. 
                                        We co-transform plasmid that only contains LacI promoter into W3110.
                                    </P><H5 class="question">How to prove our design?</H5><P class="pcontent">We designed a total solution test to verify the function of our whole construction. 
                                        We incubate the constructed strains in modified M9 medium that contains 4 (g/l) xylose as its sole carbon source. 
                                        The construction is designed to consume xylose as energy source and as a material for Calvin-Benson cycle. 
                                        We then measure the optical intensity (O.D. 600) to characterize the cell growth. At a fixed time interval, 
                                        we use DNS assay to measure the sugar consumption of the bacteria. 
                                        By comparing the experimental group to the control group, 
                                        we can prove that our engineered strain utilize carbon dioxide as its carbon source.
                                    </P></DIV><DIV id="pH_sensing_system"><H3>pH sensing system</H3><P class="pcontent">The pH sensing system, our side project, 
                                        is a system that allows us to monitor the pH in the surrounding medium 
                                        in our device at any time by observing the color change of the medium.
                                    </P><P class="pcontent">We selected two pH sensitive promoter from <I>E. coli</I>: 
                                        P<SUB>asr</SUB> and P<SUB>gadA</SUB>. P<SUB>gadA</SUB> will be induced under neutral condition while P<SUB>asr</SUB> will be induced under acidic condition. 
                                        We cloned a GFP and sfGFP gene downstream of these promoters respectively, whose product will express green fluorescence once the promoter has been activated. For the design of P<SUB>gadA</SUB> sensing system, we took the previous constructed P<SUB>gadA</SUB> biobrick <A href="http://parts.igem.org/Part:BBa_K1962013" style="color:#28ff28;">BBa_K1962013</A> from <A href="https://2016.igem.org/Team:Dundee" style="color:#28ff28;">2016 iGEM Dundee team</A> as our reference. We also improve the P<SUB>gadA</SUB> biobrick to enhance the expression of GFP.
                                    </P><P class="pcontent">In conclusion, when the color of the medium turns from turbid yellow to green, 
                                        it indicates the pH of the medium has altered so we can determine the pH condition of the medium. 
                                    </P><DIV class="row"><A class="btn col-md-12" data-toggle="collapse" href="#pH_how_to_construct" role="button" aria-expanded="false" aria-controls="multiCollapseExample1">
                                            How do we construct this part?
                                            </A></DIV><DIV class="collapse multi-collapse" id="pH_how_to_construct"><DIV class="card card-body"><P class="pcontent">We first extracted whole genome DNA from <I>E. coli</I> MG1655 and amplify both promoters by PCR 
                                                using primers that contains HindIII and SpeI. 
                                                We then exchanged the promoter with the previously constructed plasmid that contains P<SUB>T7</SUB> and GFP or sfGFP. 
                                                We initially transformed the constructed plasmid into DH5 alpha for colony screening. 
                                                We then transformed the plasmid into BL21 (DE3) to test its function.
                                                We also design another biobrick that contains riboJ (a signal amplify fragment) 
                                                at the downstream of P<SUB>gadA</SUB> to get the signal more clearly and enhance the specificity.
                                            </P></DIV></DIV><H5 class="question">How do we determine its function?</H5><P class="pcontent">We measure the fluorescence intensity of the plasmid in different pH environment to 
                                        determine its promoter activity. We incubate the bacteria in pH modified M9 medium and 
                                        measure the fluorescence intensity (absorbance: 480 nm, excitation: 510 nm).
                                    </P></DIV><DIV id="Reference"><H3>References</H3><OL><LI class="smallp">Z. Cai, G. Liu, J. Zhang, Protein Cell (2014) 5: 552.</LI><LI class="smallp">F. Gong, “Quantitative Analysis of an Engineered CO<SUB>2</SUB> -Fixing <I>Escherichia Coli</I> Reveals Great Potential of Heterotrophic CO<SUB>2</SUB> Fixation.” Biotechnology for Biofuels, BioMed Central, 18 June 2015</LI><LI class="smallp">“The Coupling of Glycolysis and the Rubisco-Based Pathway through the Non-Oxidative Pentose Phosphate Pathway to Achieve Low Carbon Dioxide Emission Fermentation.” NeuroImage, Academic Press, 25 Mar. 2015</LI><LI class="smallp">“Sugar Synthesis from CO<SUB>2</SUB> in <I>Escherichia Coli</I>.” NeuroImage, Academic Press, 23 June 2016</LI><LI class="smallp">H. Cheng, E. J. Yang, Y. L. Liu, F. Y.Chenm, Y. Ou, S. Y. Li. “The Comprehensive Profile of Fermentation Products during in Situ CO<SUB>2</SUB> Recycling by Rubisco-Based Engineered <I>Escherichia Coli</I>.” Microbial Cell Factories, BioMed Central, 2 Aug. 2016</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION style="background-color: #010E00"><FOOTER class="page-footer font-small pt-4" style="background-color: #010E00"><DIV class="container-fluid text-center text-md-left"><DIV class="row"><DIV class="col-md-6 mt-md-0 mt-3" style="text-align: center;"><H5 class="text-uppercase font-weight-bold">Follow us</H5></DIV><DIV class="col-md-6 mb-md-0 mb-3"><H5 class="text-uppercase font-weight-bold">Contact us</H5><P class="pcontent">   igem.ncku.tainan@gmail.com
                            
                            No.1, Daxue Rd., East Dist., Tainan City 701, Taiwan (R.O.C.)
                        </P></DIV></DIV></DIV></FOOTER></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>