Team:ULaval/Experiments
Home
Project
Description Design Results
Interlab
Parts
Overview Basic Parts
Notebook
Lab Journal Protocols Achievements
Human Practices
Human Practices Public Engagement Safety
Team
Team members Attributions Collaborations
Protocols
Here we present all the protocols that we used or designed to realize our project.
Culture Media
Culture media for bacteria (for 1L)
2YT+glucose (rich media)
1% Yeast extract (10g)
0.2% Glucose (2g)
0.5% NaCl (5g)
LB (complex media salt rich)
0.5% Yeast extract (5g)
1.0% Tryptone (10g)
1.0% NaCl (10g)
2LB-Lennox (Rich media)
1% Yeast extract (5g)
2.0% Tryptone (20g)
0.5% NaCl (5g)
LB-Lennox (Rich media)
0.5% Yeast extract (5g)
1.0% Tryptone (10g)
0.5% NaCl (5g)
Culture media for yeast (for 1L)
YPD
1% Yeast extract (10g)
2% Tryptone (20g)
2% Glucose (20g)
For solid media, (about 20mL/petri dish 100x15mm), add 2% agar before autoclaving media.
SC media
0.174% Yeast nitrogen base without amino acids, without ammonium sulfate (1.74g)
2% Glucose (20g)
X Drop-out mix (according to the drop-out used)
Add a nitrogen source according to your experience and to what your media will contain (for example, antibiotics):
0.5% ammonium sulfate (5g)
or
0.1% monosodium glutamate (MSG) (1g)
For solid media (about 20mL/Petri dish 100x15 mm or 35mL/Omnitray), autoclave 2% agar (20g/L) and combine to the rest once everything cooled down to 55°C.
Bacteria Competent Cells (CaCl2 method)
Methods:
Start 2x5mL precultures of the desired cells and incubate overnight at 37°C.
The day after, start 2x500mL cultures of 2YT with 5mL of preculture.
Reach a DO (600nm) of 1/mL (do not overreach).
Centrifugate cells at 5000 RPM for 20 min at 4°C.
Perform all the coming steps very gently.
Resuspend cells in 0.1M CaCl2 (100mL/pellet).
Let 3h on ice. (Start to resuspend after 1h30).
Centrifugate at 5000 RPM for 20 min.
Resuspend pellet in 0.1M CaCl2 + 15% glycerol (25mL/pellet).
Aliquot in 2mL tubes with screw caps and conserve at -80°C.
Notes:
Perform everything on ice because cells are very fragile.
Keep pipets at -20°C and tubes at -80°C the day before the experiment.
Sterilize centrifuge bottles.
Prepare 250mL of 0.1M CaCl2 and 100mL of 0.1M + 15% glycerol. Sterilize. Conserve at 4°C and transfer at -20°C 30 min before use.
Colony PCR (Bacteria)
Colony PCR works best with small fragments (200-500bp)
Methods:
Add 40µL of sterile water or 20nM NaOH in a PCR tube.
Touch gently the colony to test with a tip and resuspend in water (or NaOH).
Incubate 20 min at 95°C in the thermocycler.
Centrifuge 5 min at 4000 RPM.
Prepare the mastermix:
One reaction
(µL)
Sterile Water
17.35
Bioshop PCR Buffer [10X]
2.5
dNTP mix [10mM]
0.5
MgCL2 [25mM]
1.5
Reverse Primer [10µM]
0.5
Forward Primer [10µM]
0.5
Bioshop Taq DNA Polymerase [5U/µl]
0.15
Final volume
23
Transfer 2µL of supernatant in a new PCR tube in addition of the PCR mix.
Conditions:
5 min at 94°C
35 cycles of:
30 s at 94°C
30 s at 55°C (or the appropriate Tm of your oligos)
X min at 72°C (Time according to the length of your PCR product, 1 min/kb)
3 min at 72°C
Hold at 10°C
Colony PCR (Yeast)
Colony PCR works best with small fragments (200-500bp)
Methods:
Put 15µL of sterile water in a PCR tube.
Touch slightly the colony to test with a tip and resuspend in water.
Incubate in the PCR machine for 20 min at 95℃.
Centrifuge 4 min at 15000 RPM.
Prepare the mastermix:
One reaction
(µL)
Sterile Water
16.85
Bioshop PCR Buffer [10X]
2.5
dNTP mix [10mM]
0.5
MgCL2 [25mM]
1.5
Reverse Primer [10µM]
0.5
Forward Primer [10µM]
0.5
Bioshop Taq DNA Polymerase [5U/µl]
0.15
Final volume
22.5
Transfer 2.5µL of supernatant in a new PCR tube containing the PCR mix.
Conditions:
5 min at 94°C
35 cycles of:
30 s at 94°C
30 s at 55°C (or the appropriate Tm of your oligos)
X min at 72°C (Time according to the length of your PCR product, 1 min/kb)
3 min at 72°C
Hold at 10°C
Competent Cells (Yeast)
Methods:
5mL preculture overnight
Yeast cells inoculated from fresh preculture and grown overnight until OD 0.5-0.7 in YPD (100mL culture).
Split 50mL culture in Falcon tubes. Harvest cells by centrifuging at 500g for 5 min at RT (~1500 RPM).
Wash with 5mL sterile water.
Centrifuge at 500g for 5 min.
Wash with 5mL sterile SORB. Centrifuge again.
Remove SORB with aspiration (sterile tips!).
Resuspend cells in 360μL SORB and 40μL carrier DNA (salmon sperm DNA 10mg/mL - boiled for 5 min and cooled on ice) per 50mL culture.
Aliquot (20µL) and place in freezer (-80°C).
Stock solution
SORB 500mL
SORB 250mL
100mM LiOAc
1M
50
25
10mM Tris pH 8.0
1M
5mL
2.5mL
1mM EDTA
1M
1mL
0.5mL
1M sorbitol
182.17g/mol
91.0g
45.5g
Dissolve sorbitol in ~100mL water with stir bar on plate. Complete with above solutions and bring up to desired volume. Filter sterilize.
Gibson Cloning
Gibson cloning is used to clone one or more fragments in a vector of interest. In the lab, we generally use PCR amplified vector and fragments. Here is the general procedure in that specific case.
Amplification of vector:
Use a high fidelity polymerase (like the Kapa HiFi hotstart polymerase) to amplify your vector of interest. Your oligos should be around 20-24 nt (it depends on the Tm of the oligo - see following section) and should be designed as to amplify your vector from each side of the insertion site. PCR reaction must then be digested with DpnI to get rid of parental plasmid. Purify on column. A vector can also be digested and directly used for cloning. In that case, make a double-digestion and make sure it is complete.
Amplification of fragments (inserts):
Use a high fidelity polymerase (like the Kapa HiFi hotstart polymerase) to amplify your fragments of interest (from gDNA or plasmid). Design a forward primer with an overlap region (≥ 16 nt - usually 20 to 24 nt) at the 5´-end, which is complementary to the reverse primer of the vector. The 3´-end of the forward primer (around 20 nt - it depends on the Tm, try to have a 50% GC content) should contain insert-specific sequences to amplify the target fragment. The “≥ 16 nt” overlap region of the forward primer should be complementary to the 5´-end sequence of the vector which corresponds to the insertion site. The size of the overlap region is determined by the number of nucleotides needed to reach a Tm > 48°C. If necessary, one can also add nucleotides between the overlap region and gene-specific sequence region to ensure, for example, that the expressed protein is in frame. Similarly, the reverse primer should also contain an overlap region (≥ 16 nt - usually 20 to 24 nt) at the 5'-end and an insert-specific sequence at the 3'-end. If the amplification is made from a plasmid, digest the PCR reaction with DpnI and purify on column.
Manufacture Gibson mix:
Prepare 6mL of 5X ISO Buffer in a 15mL falcon tube as follows:
3mL 1M Tris-HCl pH 7.5
+ 150µL 2M MgCl2
+ 240µL 100 mM dNTP mix (25mM each: dGTP, dCTP, dATP, dTTP)
+ 300µL 1M DTT
+ 1.5g PEG-8000
+ 300µL 100mM NAD
+ dH20 to 6mL
Store at -20°C in 320µL aliquots.
Prepare 1.2mL of Gibson assembly master mix as follows:
320µL 5X ISO Buffer
+ 0.64µL 10U/µL T5 exonuclease (This is optimized for 20-150 bp sequence homology overlaps)
+ 20µL 2U/µL Phusion polymerase
+ 160µl 40U/µL Taq ligase
+ 700µL dH20
Store at -20°C in 7.5µL aliquots in PCR tubes (or strips).
Gibson cloning:
Thaw a 7.5µL aliquot of the Gibson assembly master mix, and keep on ice until use.
Measure the DNA concentration (ng/µL) of each assembly piece (on an agarose gel).
Add 50ng of the vector backbone (PCR or linearized) and a 1:2 (vector:insert) molar ratio amounts of the other assembly pieces to the thawed 7.5µL master mix in a 10µL total volume assembly reaction mixture as follows:
vector backbone (50ng)
each insert (to a 1:2 molar ratio with backbone)
7.5µL Gibson assembly master mix
ddH20 to 10µL
The vector:insert ratio must be adapted to the size of the fragments to clone. For smaller fragments, more inserts should be put in the reaction.
Incubate the assembly reaction at 50°C for 60 min, and then place on ice.
Transform 5µL of the assembly reaction into 100µL of competent E.coli and plate 100µL and 900µL on 2YT plates with the appropriate antibiotic(s).
Gibson cloning is hard to achieve with repeated sequences.
See the original brochure for more information about the general protocol and the procedure to design the fragments for assembly: File:Manual from NEB for Gibson cloning.pdf
Transformation of Competent Bacteria
Methods:
Transfer 0.1mL of competent cells in a 15mL tube cooled on ice.
Add 2-5µL of plasmid.
Incubate 15 min on ice.
Perform a heat shock at 42°C for 1 min.
Put on ice for 5 min.
Add 900µL of 2YT media without antibiotics.
Incubate 45 min at 37°C with agitation.
Spread 100µL of cells on 2YT+glucose+antibiotic media.
Centrifuge the remaining volume (900µL), resuspend pellet in 100µL then spread.
Incubate overnight at 37°C.
Transformation of Competent Yeast
Methods:
8μL of DNA (0.5-1μg/μL) and 20μL of thawed competent cells are placed into sterile tube (or PCR tubes in strips or plates if many transformations are done) and mixed.
100 μL of Plate mixture is added.
Cells are mixed thoroughly and incubated at room temperature for approximately 30 min (without shaking).
15μL DMSO is added.
The cells are placed at 42°C for 15-20 min in a water bath without shaking.
Centrifuge at 2000 RPM for 3 min and aspirate the supernatant.
Add 100μL YPD and incubate at 30°C for at least 2 hours (4 hours minimum in the case of a genomic integration) in a water bath without shaking.
The cells are plated on selective medium. In the case of multiple transformations, cells can be plated in a well of a 6-well plate previously filled with 4mL of media with agar. Make sure that all the liquid is well absorbed before removing the glass beads.
Place plates upside down in incubator at 30°C for 2–3 days.
If PCR plates are used, use aluminium seal to keep it sterile.
PLATE Mixture (combine all solutions previously autoclaved):
PEG 3350 45% sterile
90mL (final concentration 40%)
Li OAc 1M
10mL (final concentration 100mM)
Tris-HCl (pH 7.5) 1M
1mL (final concentration 10mM)
EDTA 0.5M
0.2mL (final concentration 1mM)
HPLC detection of standards and metabolites
Solutions to prepare:
Solvent : 0.1M HCL
Prepare a 1L bottle with 990mL of ultra pure water.
Add 10mL of 10M HCl (37%).
Stir gently. Strong acid solutions are hot: be careful and wear glasses and gloves.
Stock solutions of all standards:
Weight 100mg of standard powder.
Put in a 10mL volumetric flask.
Let the powder dissolve in solvent (don’t fill until the flask gauge). Mix by inversion.
Sonicate to help dissolution and remove bubbles.
Fill the flask to reach the gauge.
Filter with 0.2μm filter.
Working solution of all standards:
Perform serial dilutions of all standards.
Use well calibrated micropipettes or volumetric pipettes.
Figure 1: Serial dilution of the standards
HPLC programmation:
UV detector: 210nm
Mobile phase: Water/MeCN3/TFA: 90/10/0.1
Debit: 0.5mL/min
End of cycle: 10 min
Injection volume : see Figure 1
Mix of all standards :
Prepare a 10mL flask containing 5mL of solvent (HCl 0.1M).
Add 500μL of each standard (tyrosine, adrenaline, DOPA, L-DOPA) from working solutions (100ng/μL).
Fill the flask to reach the gauge.
Inject 10μL of the mix in the HPLC.
Detection of metabolites and standards:
Extraction from intracellular content:
Prepare 100mL of PBS 1X at pH 7.4 in a 250mL bottle.
Prepare a 5mL solution of zymolyase 70U/mL.
Centrifuge 1mL of 72 hours yeast culture in an eppendorf tube at 10000 RPM for 5 min.
Transfer 500μL of supernatant in a new eppendorf tube for extracellular content analysis.
Resuspend cells with 1mL of PBS 1X pH 7.4.
Centrifuge again at 3000RCF for 5 min. Discard supernatant.
Resuspend in 500μL of PBS1x pH7.4 containing 35U of zymolyase.
Incubate 1 hour at 37°C. Yeast cells become spheroplasts during that step.
After incubation, add 500μL of acetonitrile in order to break cells and precipitate proteins.
Vortex for 1 min then centrifuge at maximum speed for 1 min.
Extract supernatant while avoiding the pellet. With the help of a syringe and a 0.2μm filter: filter supernatant and transfer in a new eppendorf tube.
Concentrate 500μL of supernatant with nitrogen gaz: let dry then resuspend in 100μL of acetonitrile.
Extracellular content:
Add 250μL of 2M HClO4 to the supernatant.
Centrifuge 10 min at 13000 RPM.
Extract supernatant while avoiding the pellet. With the help of a syringe and a 0.2μm filter: filter supernatant and transfer in a new eppendorf tube.
Conserve supernatant until injection in the HPLC.
igem@bcm.ulaval.ca
