Team:ULaval/Results
Home
Project
Description Design Results
Interlab
Parts
Overview Basic Parts
Notebook
Lab Journal Protocols Achievements
Human Practices
Human Practices Public Engagement Safety
Team
Team members Attributions Collaborations
Results
Introduction
For our project AdrenaYeast, we planned on producing three different plasmids harboring different enzymes that allowed for the biosynthesis of adrenaline and some metabolic intermediates in the yeast S. cerevisiae. We, however, encountered many problems in the first weeks of work. As this is our first iGEM participation, we struggled to secure access to lab facilities at certain times. There were also some communication problems that resulted in some work being done in duplicate, or not at all. In the end, we did not manage to produce all the desired plasmids shown in Figure 1, but we were able to clone all our parts in the desired initial vectors, and to sequence confirm them. Here are the main steps that we worked through to produce the desired plasmids, and the steps we planned on doing to finish the plasmid and strain construction.
Figure 1. A) Produced plasmids B) Planned plasmids
Planning and DNA Preparation
First, we started by designing our parts. We did a literature review to assess the relative activity of the enzymes, to verify that they did not necessitate post-translational modifications (PTM) and to check which isoform was active in our chassis’ conditions. We also used the IDT optimization algorithm to codon-optimize the sequences for use in our chassis. After identifying the proper isoforms and assessing that the parts would likely be active and non-toxic, we chose yeast promoters and terminators from the iGEM registry to add to our parts and create our own parts. We managed to generate four parts using different promoter, coding sequence (CDS) and terminator combinations, all under 3Kb in length. This step was crucial, since we wanted to order the designed parts as gBlocks from IDT. This would allow us to skip part assembly and go directly to plasmid assembly. After a few synthesis problems and some sequence optimization to allow for proper synthesis, we received our gBlocks from IDT. Here, we encountered another problem. One of the students working in the laboratory did not use the proper solution to resuspend some of the gBlocks, meaning that we lost most of the DNA present in the tube. The Figure 2 shows a gel electrophoresis on 1% agarose of 2μL of the concentrated gBlocks. We can see that the gBlocks for LDDC and PNMT are much less concentrated than the other two.
Figure 2. gBlock rescue and migration to confirm quality
Blunt-End Cloning
Initially, we tried to rescue gBlocks by blunt-end cloning in a linearized blunt-end vector, as we ordered our gBlocks 5’ phosphorylated. Sadly, after many tries, we did not manage to get a single positive clone for gBlock insertion.
Restriction enzymes cloning
We then turned ourselves to directional restriction enzyme cloning. We designed and ordered primers with overhang that contained restriction sites to amplify our gBlocks with. Sadly, doing a PCR on gBlocks of above 1Kb is not recommended by IDT, and we had very low amplification efficiency, leading to very low cloning efficiency. Furthermore, restriction cloning necessitated double digestions and at least two rounds of column purification, meaning that we lost most of our DNA by the end of the process. We did not manage to get any positives using restriction enzyme cloning. As the Jamboree was approaching, we entrusted the remaining cloning and troubleshooting to a single person, hoping it would speed up the process.
Gibson Assembly
It was decided to continue using the Gibson assembly technique, as restriction cloning was inefficient. New primers for Gibson assembly were carefully designed. Gibson cloning was carried out with the goal of inserting each gBlock individually in either one of the available backbones, each carrying either a resistance to hygromicin or nourseotricin, pRS31H and pRS31N respectively. Using Gibson assembly and the new primers, we were able to generate the plasmids showed in Figure 1A with high efficiency (41/48 colonies tested were positive for gBlock insertion compared to 0/480 for other cloning methods). Surprisingly, we were only able to insert the fragments in pRS31N. The Figure 3 below shows, for all genes-gBlocks, the linearized plasmid pRS31N, the amplified composite gBlock and finally the linearized plasmid containing the amplified gBlock. The next step would be to design new Gibson assembly primers and carry out cloning to combine two gBlocks into one plasmid. Moreover, it would also be necessary to construct these plasmids in the pRS31H backbone to be able to co-transform both plasmids in a single yeast strain.
Figure 3. Confirmation of gBlocks inserts in pRS31N
HPLC
Sadly, we were not able to characterize our parts properly. In order to do so, we ordered a HPLC reverse-phase analytical column made by SIELC Technologies for the efficient separation of catecholamines. We wanted to use it to purify our different products and thus verify the activity and expression of our enzymes. As they are not natively present in our chassis, the detection of their reaction products would have been a confirmation of the enzyme’s expression and activity. After waiting for a month to receive the column, we learned that it was lost in the mail and that we would not receive it in time for proper part characterization.
igem@bcm.ulaval.ca
