<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tsinghua_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tsinghua/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:Tsinghua">HOME</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Team">TEAM</A></LI><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Design">
                            Design</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Results">                            Experiments &amp; Results</A></LI></UL><LI><A href="#">PARTS
            					</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Composite_Part">Composite Parts</A></LI><LI><A href="#">NOTEBOOK
                    </A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Notebook">Lablog</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Protocol">Protocol</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Hardware">HARDWARE</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Modeling">MODELING</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Safety">SAFETY</A></LI></DIV><DIV id="contentContainer"><TITLE>无标题文档</TITLE><P align="center"><STRONG>Protocols</STRONG></P><OL><LI><STRONG>Plasmid DNA purification</STRONG></LI><LI>Centrifuge the 5mL culture tube  for 10 minutes at 3000 rpm.</LI><LI>Remove the culture medium and  add 250uL buffer P1 (RNase added and 4°C stored) to resuspend the pelleted bacteria.</LI><LI>Transfer to a 1.5mL EP tube.</LI><LI>Add 250uL buffer P2 and gently  mix by inverting the tube 8 times.</LI><LI>Add 350uL buffer P3 and  immediately mix by inverting the tube 8 times.</LI><LI>Centrifuge the tube for 10  minutes at 12000 rpm.</LI><LI>Transfer the supernatant to a  pre-balanced (by buffer BL) column.</LI><LI>Centrifuge the column for 60  seconds at 12000 rpm.</LI><LI>Remove the flow-through and add  600uL buffer PW (alcohol added).</LI><LI>Centrifuge the column for 60  seconds at 12000 rpm.</LI><LI>Repeat step 9-10.</LI><LI>Remove the flow-through and  centrifuge for another time.</LI><LI>Place the column in a 1.5mL EP  tube and wait for 2 minutes at RT.</LI><LI>Add 60uL ddH2O onto  the column and centrifuge for 60 seconds at 12000 rpm.</LI><LI>Measure the concentration of  plasmid by Nanodrop.</LI></OL><OL><LI><STRONG>Gel extraction</STRONG></LI><LI>Remove the gel containing DNA  of interest and place it into a 1.5mL EP tube.</LI><LI>Add 500uL solution SN per 100mg  agarose gel and dissolve the gel for 5 minutes at 65°C.</LI><LI>Add 50uL solution B per 500uL  solution SN and mix them up.</LI><LI>Place the 3S column into the  collecting tube and transfer the solution mixture into the column.</LI><LI>Place the column for 2 minutes  at RT and then centrifuge for 1 minute at 10000 rpm.</LI><LI>Discard the flow-through and  add 600uL Wash solution.</LI><LI>Centrifuge for 1 minute at  10000 rpm.</LI><LI>Repeat step 6-7.</LI><LI>Place the column into a new  1.5mL EP tube and add 30uL ddH2O.</LI><LI>Place the column for 2 minutes  at RT and centrifuge for 1 minute at 10000 rpm.</LI><LI>Measure the concentration of  DNA by Nanodrop.</LI></OL><P> </P><OL><LI><STRONG>Chemical transformation</STRONG></LI><LI>Add 5uL plasmid into 50uL  competent DH5α and place the tube on ice for 30 minutes.</LI><LI>Heat up the tube for 90 seconds  at 42°C.</LI><LI>Place the tube on ice for 2.5  minutes.</LI><LI>Add 1mL LB (without  antibiotics) on a germ-free bench.</LI><LI>Recover the bacteria by shaking  the tube for 30 minutes at 37°C.</LI><LI>Centrifuge (if needed) for 3  minutes at 5000 rpm and remove most of the supernatant.</LI><LI>Resuspend the bacteria and add  100uL suspension onto the plate with antibiotics.</LI><LI>Culture overnight at 37°C.</LI></OL><P><STRONG>Transformation  protocol for distribution kit</STRONG></P><OL><LI>Insert a tip into the well containing  plasmid of interest.</LI><LI>Add 10uL ddH2O and  dissolve the plasmid for 5 minutes.</LI><LI>Use 5uL for transformation.</LI></OL><OL><LI><STRONG>Electro-transformation</STRONG></LI><LI>Produce electroporation-competent  cells.</LI></OL><P>Add 50uL bacteria  into 100mL LB and culture at 30°C until its optical density (OD600) reaches 0.05-0.1. Then add 1mM  L-arabinose until it reaches 0.6. Place the cells on ice for 1 hour and then  wash with overnight-stored cold H2O twice and cold 10% glycerol  once. Condense 100 times and dispense 50uL into each tube. Store at -80°C. </P><OL><LI>Electro-transform 10-100ng DNA  per 50uL cells using Eporator from Eppendorf with 1mm3 electric  shock cup (Electric shock is given at 1.8kV for 4-5 seconds).</LI><LI>Recover the bacteria in 1mL LB  at 37°C. </LI><LI>Plate the bacteria on the plate  with antibiotics. </LI></OL><OL><LI><STRONG>PCR</STRONG></LI></OL><P>Commonly used system:
  KOD buffer: 1uL
  dNTP: 1uL
  MgSO4: 1uL
  Forward primer: 0.2uL
  Reverse primer: 0.2uL
  KOD plus: 0.2uL
  Template: 0.2uL (it depends)
  ddH2O: add up to 10uL
  Total volume: 10uL</P><P>Commonly set up program:
  94°C 5 minutes 94°C 30 seconds
  60°C (it depends) 30  seconds    30 cycles (it depends)  
  72°C 30 seconds (it  depends) 
  72°C 7 minutes 
  12°C ∞</P><OL><LI><STRONG>RNA extraction</STRONG></LI><LI>Homogenize the tissue by  grinding the tissue in liquid nitrogen with motar and pestle.</LI><LI>Transfer the mixture into a  tube and add TRIzol to stabilize the RNA.</LI><LI>Incubate the sample for 5  minutes at RT.</LI><LI>Add chloroform (1:5 volume  ratio of chloroform to TRIzol is preferred) to the sample.  </LI><LI>Mix the sample by inverting for  8 times.</LI><LI>Incubate for 5 minutes at RT.</LI><LI>Centrifuge at 12000 g for 5  minutes at 2-8°C. </LI><LI>Transfer the  colorless aqueous upper layer to a new tube. </LI><LI>Add isopropanol (1:2 volume  ratio of isopropanol to TRIzol is preferred) to the sample.</LI><LI>Mix the sample by inverting for  8 times.</LI><LI>Incubate for 5 minutes at RT.</LI><LI>Centrifuge at 12000 g for 10  minutes at 4°C. </LI><LI>Remove supernatant and add 70%  ethanol.</LI><LI>Mix by vortexing shortly.</LI><LI>Centrifuge at 12000 g for 10  minutes at 4°C. </LI><LI>Air-dry the pellet for 5  minutes.</LI><LI>Dissolve the pellet with 50uL  DEPC H2O.</LI><LI>Mix the solution by inverting  for several times and heat for 10 minutes at 55°C. </LI><LI>Measure the concentration of  RNA by Nanodrop.</LI></OL><P><STRONG> </STRONG></P><OL><LI><STRONG>Reverse transcription PCR</STRONG></LI><LI>Template RNA is incubated with  primer pairs at 70°C to denature  secondary structures. </LI><LI>Quickly place the  mixture on ice. </LI><LI>dNTP, RNase  inhibitors, reverse transcriptase and reaction are then added. </LI><LI>Incubate the mixture  for 1 hour at 42°C. </LI><LI>Place the mixture at  70°C to inactivate the enzyme. </LI><LI>Use the newly synthesized DNA  as the template for future experiments.</LI></OL><OL><LI><STRONG>Enzymatic digestion</STRONG></LI></OL><P>Buffer: 1uL
  Template: 2uL (it depends)
  Restriction enzyme: 0.2-0.5uL
  ddH2O: add up to 10uL
  Total volume: 10uL
  Placed at 37°C for 1 hour </P><OL><LI><STRONG>Ligation</STRONG></LI></OL><P>10× T4 ligase buffer (NEB) 1uL
  100× BSA  0.1uL
  T4  ligase 0.2uL
  Linearized  backbone 0.1uL (it depends) 
  Insert 1uL (it depends)
  BsaI 0.5uL
  ddH2O add up to 10uL
  Total volume: 10uL
  Notes: The molecular weight ratio of insert  to backbone that is used commonly is 3:1.</P><OL><LI><STRONG>Seamless cloning</STRONG></LI><LI>Add homologous overlap (usually  15-30 bp) to DNA fragments.</LI><LI>Purify the DNA fragments and  measure its concentration.</LI><LI>Set up the assembly cloning  reaction with the following system:</LI></OL><P>DNA fragments + linearized vector 0.5-5uL
  2× Reagent mix 5uL
  ddH2O add up  to 10uL
  Total volume: 10uL</P><OL><LI>Place the reaction mix for 15  minutes at 50°C. </LI><LI>Use 5uL mix for  transformation. </LI></OL><P>Note: Ratio between inserts and ratio between  vector and inserts should be kept at 1:1 and 1:3 respectively. </P><OL><LI><STRONG>Bacterial genome one-step knock-out assay</STRONG></LI></OL><P>Adapted from method designed by Wanner.</P><OL><LI>Prepare the following plasmids:</LI></OL><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="138" valign="top"><P> </P></TD><TD width="138" valign="top"><P>Requirement from the host bacteria</P></TD><TD width="138" valign="top"><P>Resistance gene</P></TD><TD width="138" valign="top"><P>Temperature requirement</P></TD></TR><TR><TD width="138" valign="top"><P>pKD46</P></TD><TD width="138" valign="top"><P> </P></TD><TD width="138" valign="top"><P>Amp</P></TD><TD width="138" valign="top"><P>30°C </P></TD></TR><TR><TD width="138" valign="top"><P>pKD3/4/13</P></TD><TD width="138" valign="top"><P>BW25141</P></TD><TD width="138" valign="top"><P>Amp, Kan or Cm</P></TD><TD width="138" valign="top"><P> </P></TD></TR><TR><TD width="138" valign="top"><P>pCP20</P></TD><TD width="138" valign="top"><P> </P></TD><TD width="138" valign="top"><P>Amp, Cm</P></TD><TD width="138" valign="top"><P>30°C </P></TD></TR></TBODY></TABLE><OL><LI>Transform pKD46 (temperature-sensitive,  culture at 30°C) and screen on Amp+ plate.</LI><LI>Extract the plasmid and  confirm.</LI><LI>Amplify the Kanamycin gene to  be inserted with primers share homology (usually 39bp for homologous arm and  20bp for matching sequence of Kan) to the genome by PCR.</LI><LI>Electro-transform the fragment  after PCR and screen on Kan+ plate.</LI><LI>Pick multiple colonies and  preserve on Kan+ plate.</LI><LI>Meanwhile confirm the colonies  by colony PCR. </LI><LI>Culture in LB at 37°C without antibiotics overnight in order to  remove pKD46. </LI><LI>Culture on plate at  30°C without antibiotics. </LI><LI>Pick colony and screen positively  using Kan+ plate.</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>