Team:Tsinghua/Protocol
HOME
TEAM
PROJECT
Description
Design
Experiments & Results
PARTS
Team Parts
Basic Parts
Composite Parts
NOTEBOOK
Lablog
Protocol
HARDWARE
MODELING
HUMAN PRACTICES
COLLABORATIONS
ATTRIBUTIONS
SAFETY
无标题文档
Protocols
1. Plasmid DNA purification
(1)Centrifuge the 5mL culture tube  for 10 minutes at 3000 rpm.
(2)Remove the culture medium and  add 250uL buffer P1 (RNase added and 4°C stored) to resuspend the pelleted bacteria.
(3)Transfer to a 1.5mL EP tube.
(4)Add 250uL buffer P2 and gently  mix by inverting the tube 8 times.
(5)Add 350uL buffer P3 and  immediately mix by inverting the tube 8 times.
(6)Centrifuge the tube for 10  minutes at 12000 rpm.
(7)Transfer the supernatant to a  pre-balanced (by buffer BL) column.
(8)Centrifuge the column for 60  seconds at 12000 rpm.
(9)Remove the flow-through and add  600uL buffer PW (alcohol added).
(10)Centrifuge the column for 60  seconds at 12000 rpm.
(11)Repeat step 9-10.
(12)Remove the flow-through and  centrifuge for another time.
(13)Place the column in a 1.5mL EP  tube and wait for 2 minutes at RT.
(14)Add 60uL ddH2O onto  the column and centrifuge for 60 seconds at 12000 rpm.
(15)Measure the concentration of  plasmid by Nanodrop.
2. Gel extraction
(1)Remove the gel containing DNA  of interest and place it into a 1.5mL EP tube.
(2)Add 500uL solution SN per 100mg  agarose gel and dissolve the gel for 5 minutes at 65°C.
(3)Add 50uL solution B per 500uL  solution SN and mix them up.
(4)Place the 3S column into the  collecting tube and transfer the solution mixture into the column.
(5)Place the column for 2 minutes  at RT and then centrifuge for 1 minute at 10000 rpm.
(6)Discard the flow-through and  add 600uL Wash solution.
(7)Centrifuge for 1 minute at  10000 rpm.
(8)Repeat step 6-7.
(9)Place the column into a new  1.5mL EP tube and add 30uL ddH2O.
(10)Place the column for 2 minutes  at RT and centrifuge for 1 minute at 10000 rpm.
(11)Measure the concentration of  DNA by Nanodrop.
3. Chemical transformation
(1)Add 5uL plasmid into 50uL  competent DH5α and place the tube on ice for 30 minutes.
(2)Heat up the tube for 90 seconds  at 42°C.
(3)Place the tube on ice for 2.5  minutes.
(4)Add 1mL LB (without  antibiotics) on a germ-free bench.
(5)Recover the bacteria by shaking  the tube for 30 minutes at 37°C.
(6)Centrifuge (if needed) for 3  minutes at 5000 rpm and remove most of the supernatant.
(7)Resuspend the bacteria and add  100uL suspension onto the plate with antibiotics.
(8)Culture overnight at 37°C.
4. Transformation  protocol for distribution kit
(1)Insert a tip into the well containing  plasmid of interest.
(2)Add 10uL ddH2O and  dissolve the plasmid for 5 minutes.
(3) 5uL for transformation.
5. Electro-transformation
(1)Produce electroporation-competent  cells.
(2)Add 50uL bacteria  into 100mL LB and culture at 30°C until its optical density (OD600) reaches 0.05-0.1. Then add 1mM  L-arabinose until it reaches 0.6. Place the cells on ice for 1 hour and then  wash with overnight-stored cold H2O twice and cold 10% glycerol  once. Condense 100 times and dispense 50uL into each tube. Store at -80°C.
(3)Electro-transform 10-100ng DNA  per 50uL cells using Eporator from Eppendorf with 1mm3 electric  shock cup (Electric shock is given at 1.8kV for 4-5 seconds).
(4)Recover the bacteria in 1mL LB  at 37°C.
(5)Plate the bacteria on the plate  with antibiotics.
6. PCR
(1)Commonly used system:
KOD buffer: 1uL
dNTP: 1uL
MgSO4: 1uL
Forward primer: 0.2uL
Reverse primer: 0.2uL
KOD plus: 0.2uL
Template: 0.2uL (it depends)
ddH2O: add up to 10uL
Total volume: 10uL
(2)Commonly set up program:
94°C 5 minutes
94°C 30 seconds
60°C (it depends) 30 seconds    30 cycles (it depends) 
72°C 30 seconds(it depends)
72°C 7 minutes
12°C ∞
7. RNA extraction
(1)Homogenize the tissue by  grinding the tissue in liquid nitrogen with motar and pestle.
(2)Transfer the mixture into a  tube and add TRIzol to stabilize the RNA.
(3)Incubate the sample for 5  minutes at RT.
(4)Add chloroform (1:5 volume  ratio of chloroform to TRIzol is preferred) to the sample.
(5)Mix the sample by inverting for  8 times.
(6)Incubate for 5 minutes at RT.
(7)Centrifuge at 12000 g for 5  minutes at 2-8°C.
(8)Transfer the  colorless aqueous upper layer to a new tube.
(9)Add isopropanol (1:2 volume  ratio of isopropanol to TRIzol is preferred) to the sample.
(10)Mix the sample by inverting for  8 times.
(11)Incubate for 5 minutes at RT.
(12)Centrifuge at 12000 g for 10  minutes at 4°C.
(13)Remove supernatant and add 70%  ethanol.
(14)Mix by vortexing shortly.
(15)Centrifuge at 12000 g for 10  minutes at 4°C.
(16)Air-dry the pellet for 5  minutes.
(17)Dissolve the pellet with 50uL  DEPC H2O.
(18)Mix the solution by inverting  for several times and heat for 10 minutes at 55°C.
(19)Measure the concentration of  RNA by Nanodrop.
8. Reverse transcription PCR
(1)Template RNA is incubated with  primer pairs at 70°C to denature  secondary structures.
(2)Quickly place the  mixture on ice.
(3)dNTP, RNase  inhibitors, reverse transcriptase and reaction are then added.
(4)Incubate the mixture  for 1 hour at 42°C.
(5)Place the mixture at  70°C to inactivate the enzyme.
(6)Use the newly synthesized DNA  as the template for future experiments.
9. Enzymatic digestion
Buffer: 1uL
Template: 2uL (it depends)
Restriction enzyme: 0.2-0.5uL
ddH2O: add up to 10uL
Total volume: 10uL
Place at 37°C for 1 hour
10. Ligation
10×T4 ligase buffer (NEB) 1uL
100×BSA 0.1uL
T4 ligase 0.2uL
Linearized backbone 0.1uL (it depends)
Insert 1uL (it depends)
BsaI 0.5uL
ddH2O add up to 10uL
Total volume: 10uL
Notes: The molecular weight ratio of insert to backbone that is used commonly is 3:1.
11. Seamless cloning
(1)Add homologous overlap (usually  15-30 bp) to DNA fragments.
(2)Purify the DNA fragments and  measure its concentration.
(3)Set up the assembly cloning  reaction with the following system:
DNA fragments + linearized vector 0.5-5uL
2× Reagent mix 5uL
ddH2O add up  to 10uL
Total volume: 10uL
(4)Place the reaction mix for 15  minutes at 50°C.
(5)Use 5uL mix for  transformation.
Note: Ratio between inserts and ratio between vector and inserts should be kept at 1:1 and 1:3 respectively.
12. Bacterial genome one-step knock-out assay
Adapted from method designed by Wanner.
Prepare the following plasmids:
 
Requirement from the host bacteria
Resistance gene
Temperature requirement
pKD46
 
Amp
30°C
pKD3/4/13
BW25141
Amp, Kan or Cm
 
pCP20
 
Amp, Cm
30°C
Transform pKD46 (temperature-sensitive,  culture at 30°C) and screen on Amp+ plate.
Extract the plasmid and  confirm.
Amplify the Kanamycin gene to  be inserted with primers share homology (usually 39bp for homologous arm and  20bp for matching sequence of Kan) to the genome by PCR.
Electro-transform the fragment  after PCR and screen on Kan+ plate.
Pick multiple colonies and  preserve on Kan+ plate.
Meanwhile confirm the colonies  by colony PCR.
Culture in LB at 37°C without antibiotics overnight in order to  remove pKD46.
Culture on plate at  30°C without antibiotics.
Pick colony and screen positively  using Kan+ plate.
________________________________________________________________________________________________________________________
