<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tsinghua_Protocol skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tsinghua/Protocol</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:Tsinghua">HOME</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Team">TEAM</A></LI><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Design">
                            Design</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Results">                            Experiments &amp; Results</A></LI></UL><LI><A href="#">PARTS
            					</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Composite_Part">Composite Parts</A></LI><LI><A href="#">NOTEBOOK
                    </A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Notebook">Lablog</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Protocol">Protocol</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Hardware">HARDWARE</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Modeling">MODELING</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Tsinghua/Safety">SAFETY</A></LI></DIV><DIV id="contentContainer"><TITLE>无标题文档</TITLE><P align="center"><STRONG>Protocols</STRONG></P><P><STRONG>1. Plasmid DNA purification</STRONG>
  (1)Centrifuge the 5mL culture tube  for 10 minutes at 3000 rpm.
  (2)Remove the culture medium and  add 250uL buffer P1 (RNase added and 4°C stored) to resuspend the pelleted bacteria.
  (3)Transfer to a 1.5mL EP tube.
  (4)Add 250uL buffer P2 and gently  mix by inverting the tube 8 times.
  (5)Add 350uL buffer P3 and  immediately mix by inverting the tube 8 times.
  (6)Centrifuge the tube for 10  minutes at 12000 rpm.
  (7)Transfer the supernatant to a  pre-balanced (by buffer BL) column.
  (8)Centrifuge the column for 60  seconds at 12000 rpm.
  (9)Remove the flow-through and add  600uL buffer PW (alcohol added).
  (10)Centrifuge the column for 60  seconds at 12000 rpm.
  (11)Repeat step 9-10.
  (12)Remove the flow-through and  centrifuge for another time.
  (13)Place the column in a 1.5mL EP  tube and wait for 2 minutes at RT.
  (14)Add 60uL ddH2O onto  the column and centrifuge for 60 seconds at 12000 rpm.
  (15)Measure the concentration of  plasmid by Nanodrop.</P><P><STRONG>2. Gel extraction</STRONG>
  (1)Remove the gel containing DNA  of interest and place it into a 1.5mL EP tube.
  (2)Add 500uL solution SN per 100mg  agarose gel and dissolve the gel for 5 minutes at 65°C.
  (3)Add 50uL solution B per 500uL  solution SN and mix them up.
  (4)Place the 3S column into the  collecting tube and transfer the solution mixture into the column.
  (5)Place the column for 2 minutes  at RT and then centrifuge for 1 minute at 10000 rpm.
  (6)Discard the flow-through and  add 600uL Wash solution.
  (7)Centrifuge for 1 minute at  10000 rpm.
  (8)Repeat step 6-7.
  (9)Place the column into a new  1.5mL EP tube and add 30uL ddH2O.
  (10)Place the column for 2 minutes  at RT and centrifuge for 1 minute at 10000 rpm.
  (11)Measure the concentration of  DNA by Nanodrop.</P><P><STRONG>3. Chemical transformation</STRONG>
  (1)Add 5uL plasmid into 50uL  competent DH5α and place the tube on ice for 30 minutes.
  (2)Heat up the tube for 90 seconds  at 42°C.
  (3)Place the tube on ice for 2.5  minutes.
  (4)Add 1mL LB (without  antibiotics) on a germ-free bench.
  (5)Recover the bacteria by shaking  the tube for 30 minutes at 37°C.
  (6)Centrifuge (if needed) for 3  minutes at 5000 rpm and remove most of the supernatant.
  (7)Resuspend the bacteria and add  100uL suspension onto the plate with antibiotics.
  (8)Culture overnight at 37°C.</P><P><STRONG>4. Transformation  protocol for distribution kit</STRONG>
  (1)Insert a tip into the well containing  plasmid of interest.
  (2)Add 10uL ddH2O and  dissolve the plasmid for 5 minutes.
  (3) 5uL for transformation.</P><P><STRONG>5. Electro-transformation</STRONG>
  (1)Produce electroporation-competent  cells.
(2)Add 50uL bacteria  into 100mL LB and culture at 30°C until its optical density (OD600) reaches 0.05-0.1. Then add 1mM  L-arabinose until it reaches 0.6. Place the cells on ice for 1 hour and then  wash with overnight-stored cold H2O twice and cold 10% glycerol  once. Condense 100 times and dispense 50uL into each tube. Store at -80°C. 
  (3)Electro-transform 10-100ng DNA  per 50uL cells using Eporator from Eppendorf with 1mm3 electric  shock cup (Electric shock is given at 1.8kV for 4-5 seconds).
  (4)Recover the bacteria in 1mL LB  at 37°C. 
  (5)Plate the bacteria on the plate  with antibiotics. </P><P><STRONG>6. PCR</STRONG>
(1)Commonly used system:<LI>KOD buffer: 1uL</LI><LI>dNTP: 1uL</LI><LI>MgSO4: 1uL</LI><LI>Forward primer: 0.2uL</LI><LI>Reverse primer: 0.2uL</LI><LI>KOD plus: 0.2uL</LI><LI>Template: 0.2uL (it depends)</LI><LI>ddH2O: add up to 10uL
  Total volume: 10uL</LI></P><P>(2)Commonly set up program:<LI>94°C 5 minutes </LI><LI>94°C 30 seconds</LI><LI>60°C (it depends) 30 seconds    30 cycles (it depends)  </LI><LI>72°C 30 seconds(it depends) </LI><LI>72°C 7 minutes </LI><LI>12°C ∞</LI></P><P><STRONG>7. RNA extraction</STRONG>
  (1)Homogenize the tissue by  grinding the tissue in liquid nitrogen with motar and pestle.
  (2)Transfer the mixture into a  tube and add TRIzol to stabilize the RNA.
  (3)Incubate the sample for 5  minutes at RT.
  (4)Add chloroform (1:5 volume  ratio of chloroform to TRIzol is preferred) to the sample.
  (5)Mix the sample by inverting for  8 times.
  (6)Incubate for 5 minutes at RT.
  (7)Centrifuge at 12000 g for 5  minutes at 2-8°C.
  (8)Transfer the  colorless aqueous upper layer to a new tube.
  (9)Add isopropanol (1:2 volume  ratio of isopropanol to TRIzol is preferred) to the sample.
  (10)Mix the sample by inverting for  8 times.
  (11)Incubate for 5 minutes at RT.
  (12)Centrifuge at 12000 g for 10  minutes at 4°C.
  (13)Remove supernatant and add 70%  ethanol.
  (14)Mix by vortexing shortly.(15)Centrifuge at 12000 g for 10  minutes at 4°C.
  (16)Air-dry the pellet for 5  minutes.
  (17)Dissolve the pellet with 50uL  DEPC H2O.
  (18)Mix the solution by inverting  for several times and heat for 10 minutes at 55°C.
  (19)Measure the concentration of  RNA by Nanodrop.</P><P><STRONG>8. Reverse transcription PCR</STRONG>
  (1)Template RNA is incubated with  primer pairs at 70°C to denature  secondary structures.
  (2)Quickly place the  mixture on ice.
  (3)dNTP, RNase  inhibitors, reverse transcriptase and reaction are then added.
  (4)Incubate the mixture  for 1 hour at 42°C.
  (5)Place the mixture at  70°C to inactivate the enzyme.
  (6)Use the newly synthesized DNA  as the template for future experiments.</P><P><STRONG>9. Enzymatic digestion</STRONG><LI>Buffer: 1uL</LI><LI>Template: 2uL (it depends)</LI><LI>Restriction enzyme: 0.2-0.5uL</LI><LI>ddH2O: add up to 10uL
  Total volume: 10uL
  Place at 37°C for 1 hour</LI></P><P><STRONG>10. Ligation</STRONG><LI>10×T4 ligase buffer (NEB) 1uL</LI><LI>100×BSA 0.1uL</LI><LI>T4 ligase 0.2uL</LI><LI>Linearized backbone 0.1uL (it depends)</LI><LI>Insert 1uL (it depends)</LI><LI>BsaI 0.5uL</LI><LI>ddH2O add up to 10uL
  Total volume: 10uL
  Notes: The molecular weight ratio of insert to backbone that is used commonly is 3:1.</LI></P><P><STRONG>11. Seamless cloning</STRONG>
  (1)Add homologous overlap (usually  15-30 bp) to DNA fragments.
  (2)Purify the DNA fragments and  measure its concentration.
  (3)Set up the assembly cloning  reaction with the following system:<LI>DNA fragments + linearized vector 0.5-5uL</LI><LI>2× Reagent mix 5uL</LI><LI>ddH2O add up  to 10uL
  Total volume: 10uL</LI></P>
  (4)Place the reaction mix for 15  minutes at 50°C.
  (5)Use 5uL mix for  transformation.
Note: Ratio between inserts and ratio between vector and inserts should be kept at 1:1 and 1:3 respectively.<P><STRONG>12. Bacterial genome one-step knock-out assay</STRONG>
Adapted from method designed by Wanner.
Prepare the following plasmids:</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="138" valign="top"><P> </P></TD><TD width="138" valign="top"><P>Requirement from the host bacteria</P></TD><TD width="138" valign="top"><P>Resistance gene</P></TD><TD width="138" valign="top"><P>Temperature requirement</P></TD></TR><TR><TD width="138" valign="top"><P>pKD46</P></TD><TD width="138" valign="top"><P> </P></TD><TD width="138" valign="top"><P>Amp</P></TD><TD width="138" valign="top"><P>30°C </P></TD></TR><TR><TD width="138" valign="top"><P>pKD3/4/13</P></TD><TD width="138" valign="top"><P>BW25141</P></TD><TD width="138" valign="top"><P>Amp, Kan or Cm</P></TD><TD width="138" valign="top"><P> </P></TD></TR><TR><TD width="138" valign="top"><P>pCP20</P></TD><TD width="138" valign="top"><P> </P></TD><TD width="138" valign="top"><P>Amp, Cm</P></TD><TD width="138" valign="top"><P>30°C </P></TD></TR></TBODY></TABLE><OL><LI>Transform pKD46 (temperature-sensitive,  culture at 30°C) and screen on Amp+ plate.</LI><LI>Extract the plasmid and  confirm.</LI><LI>Amplify the Kanamycin gene to  be inserted with primers share homology (usually 39bp for homologous arm and  20bp for matching sequence of Kan) to the genome by PCR.</LI><LI>Electro-transform the fragment  after PCR and screen on Kan+ plate.</LI><LI>Pick multiple colonies and  preserve on Kan+ plate.</LI><LI>Meanwhile confirm the colonies  by colony PCR. </LI><LI>Culture in LB at 37°C without antibiotics overnight in order to  remove pKD46. </LI><LI>Culture on plate at  30°C without antibiotics. </LI><LI>Pick colony and screen positively  using Kan+ plate.</LI></OL><DIV type="container" align="center"><P>________________________________________________________________________________________________________________________</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>