"
Page
Discussion
View source
History
teams
Log in
 
Team:UGent/Experiments
From 2013.igem.org
Home
Team
Undergrads Instructors Official team page
Project
Literature Study Parts Results
Notebook
Labwork
Experiments CloneManager Labjournal
Social Aspects
Human Outreach Ethics Survey Press
Safety
Achievements
Attributions
Experiments
To test our idea we are conducting 6 experiments. These are described below.
Experiment 1
Experiment 1 is the knock-in (KI) of the construct containing ccdA, gfp and the homologous regions (this was constructed beforehand). This will be done by using the method of Datsenko & Wanner [PNAS 2000] , based on homologous recombination. The principle of the KO/KI method is depicted left.
Protocol experiment 1
Experiment 2
Next is finding a way to administer the toxin to the cells. This will be done by putting the gene coding for the toxin on a plasmid (a linear piece of DNA) and transferring this plasmid into the bacterial cell. In experiment 2 a plasmid containing ccdB (toxin) under control of a T7 promoter will be constructed. The T7 promoter allows us to control the expression of ccdB by regulating the amount of IPTG added to the cells.
During this experiment we encountered some unexpected mutations and used a reverse mutation protocol, based on Gibson Assembly, to restore the original plasmid sequence.
Protocol experiment 2
Protocol for Back Mutation (using Gibson Assembly)
Experiment 3
Once we have constructed the plasmid with ccdB and a T7 promoter, it has to be transferred into the cells in which CIChE will be performed. This “transformation” (the process of putting DNA into a cell) will be carried out in experiment 3.
Protocol experiment 3
Experiment 4
Strains constructed in experiment 3 will be used to perform CIChE. Tandem gene replication of reporter protein GFP will be induced by replicating the antitoxin CcdA as a response on titration of the toxin CcdB. Titration of CcdB under inducible T7-promoter will be accomplished by different levels of IPTG [0.01 mM – 0.5mM] and different plasmid copy numbers [p5, p10 and p20].
Protocol experiment 4
Experiment 5
Through experiment 4 we obtained strains that underwent different levels of chromosomal evolution and should contain different construct copy numbers. In experiment 5 we will assess the number of duplications by growing our colonies in microtiter plates and measuring fluorescence intensity and optical density (OD) inside the FLUOstar OPTIMA (BMG LABTECH). Bacteria with higher construct copy numbers also contain a higher number of gfp copies, resulting in higher fluorescence intensities.
Besides measuring fluorescence, we also measure OD as this an indicator for the growth of the bacteria. This has to be taken into account when analysing fluorescence results, since fluorescence is also dependent of bacterial density. To compare bacterial colonies, grown in different IPTG concentrations, this factor has to be eliminated. That is why for comparison we will use fluorescence/OD.
We used two protocols for this experiment. In the first version bacteria are grown for approximately 9 hours inside the FLUOstar and fluorescence and OD are measured each five minutes. The second protocol is less time consuming and consists of growing our bacteria in microtiter plates during 4 hours at 37°C (not in FLUOstar) after which fluorescence and OD are measured one time each inside the FLUOstar.
Protocol experiment 5.1
Protocol experiment 5.2
Experiment 6
In this experiment, a new part , called BBa_K1105000 , will be constructed by cloning LacIq-T7ccdB with standard Biobrick prefix and suffix in pSB1C3. Laciq-T7ccdB is derived from the plasmid p10-LacIq-T7ccdB, which originally comes from a mini F-plasmid positive strains (such as E. coli F+). The function of this part is to produce CcdB (in control of a T7 promotor), which interferes with the topoisomerase unit GyrA.
During this experiment we encountered some unexpected mutations and used a reverse mutation protocol, based on Gibson Assembly, to restore the original plasmid sequence.
Protocol experiment 6
Protocol for Back Mutation (using Gibson Assembly)
Project: NetSite...JavaScript Countdown Clock
Days:
Hours:
Mins:
Secs:
Until Jamboree Lyon
Tweets van @iGEM_UGent
We thank following sponsors for their support
Retrieved from " http://2013.igem.org/Team:UGent/Experiments "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
