<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UGent_LiteratureStudy"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UGent/LiteratureStudy">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UGent/LiteratureStudy&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UGent/LiteratureStudy&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UGent/LiteratureStudy&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UGent/LiteratureStudy" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UGent/LiteratureStudy</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><TABLE cellspacing="0" cellpadding="0" border="0"><TBODY><TR align="center" bgcolor="#F9BD05"><TD colspan="2" width="1200px"><UL id="sddm"><LI><A href="https://2013.igem.org/Team:UGent">Home</A></LI><LI><A href="https://2013.igem.org/Team:UGent/Team" onmouseover="mopen('m2')" onmouseout="mclosetime()">Team</A><DIV id="m2" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2013.igem.org/Team:UGent/Undergrads">Undergrads</A><A href="https://2013.igem.org/Team:UGent/Instructors">Instructors</A><A href="https://igem.org/Team.cgi">Official team page</A></DIV></LI><LI><A href="https://2013.igem.org/Team:UGent/Project" onmouseover="mopen('m3')" onmouseout="mclosetime()">Project</A><DIV id="m3" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2013.igem.org/Team:UGent/LiteratureStudy">Literature Study</A><A href="https://2013.igem.org/Team:UGent/Parts">Parts</A><A href="https://2013.igem.org/Team:UGent/Results">Results</A></DIV></LI><LI><A href="https://2013.igem.org/Team:UGent/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:UGent/Labwork" onmouseover="mopen('m4')" onmouseout="mclosetime()">Labwork</A><DIV id="m4" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2013.igem.org/Team:UGent/Experiments">Experiments</A><A href="https://2013.igem.org/Team:UGent/CloneManager">CloneManager</A><A href="https://2013.igem.org/Team:UGent/Labjournal">Labjournal</A></DIV></LI><LI><A href="https://2013.igem.org/Team:UGent/SocialAspects" onmouseover="mopen('m5')" onmouseout="mclosetime()">Social Aspects</A><DIV id="m5" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2013.igem.org/Team:UGent/HumanOutreach">Human Outreach</A><A href="https://2013.igem.org/Team:UGent/Ethics">Ethics</A><A href="https://2013.igem.org/Team:UGent/Survey">Survey</A><A href="https://2013.igem.org/Team:UGent/Press">Press</A></DIV></LI><LI><A href="https://2013.igem.org/Team:UGent/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:UGent/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:UGent/Attributions">Attributions</A></LI></UL></TD></TR><TR><TD valign="top"><P>You can read our entire literature study about our new model for stabilised gene duplication <A href="https://static.igem.org/mediawiki/2013/d/dc/UGent_2013_Bachelorproef.pdf" target="_blank">here</A>. Below you can read the most important aspects of it.</P><H1>Introduction: high gene expression in industrial biotechnology</H1><P>The main goal of industrial biotechnology is to increase the yield of biochemical products using microorganisms as production hosts. This includes engineering large synthetic pathways and improving their expression. <B>Overexpression of genes</B> has mainly been achieved by using high or medium copy plasmids. However, studies have demonstrated that plasmid-bearing cells lose their productivity fairly quickly as a result of genetic instability.</P><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
 Read more about plasmids 
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><P>In industrial biotechnology, a common technique to express new synthetic products and pathways is the use of plasmids as vectors. Plasmids are circular pieces of DNA. They are easy to insert into cells and replicate independently from the genome, allowing strong gene expression. Overexpression is easily achieved by using plasmids with a medium or high copy number, different promoter systems, ribosome binding sites (RBS), etc. Thanks to plasmids, the industrial biotechnology has grown substantially over the past years. However, the use of plasmids entails some important disadvantages: plasmid maintenance imposes a metabolic burden on cells and plasmids suffer from genetic instability.</P><H3>Metabolic burden</H3><P>When plasmids are present in cells and replicate, they create a metabolic burden. This is defined by Bentley <I>et al.</I> (1990) as <I>“the amount of resources that are taken from the host cell metabolism for foreign DNA maintenance and replication”</I>. As a result, metabolic load causes many alterations to the physiology and metabolism of the cell and reduces the cellular fitness. The most common change is a delayed growth. This can be caused by the fact that new pathways for energy generation are activated due to the competition between cell propagation and plasmid replication. This growth retardation leads to a lower yield of the desired product.</P><H3>Genetic instability</H3><P>Plasmids are genetically instable due to three processes: segregational instability, structural instability and allele segregation. 
<UL><LI><I>Segregational instability</I> is caused by unequal distribution of plasmids to daughter cells, which leads to cells devoid of any plasmids. This problem has been solved by using selectable markers (e.g. antibiotic resistance) or post-segregational killing to remove plasmid-lacking cells.</LI><LI><I>Structural instability</I> leads to an incorrect expression of proteins. This problem finds its cause in plasmids with a changed DNA sequence. However, these mutations occur at a low frequency.</LI></UL></P></DIV></DIV></TD></TR><TR><TD><P><LI><I>Allele segregation</I> up to now remains mainly unaddressed. When a mutation occurs in the gene of interest, but not in the selectable marker, cells can emerge that are resistant to the selection, but not productive. As a result they cannot be removed by the use of selectable markers or post-segregational killing (<B>Figure</B>). After the mutation occurs, the plasmids are replicated and divided over daughters cells. The mutated plasmids can be divided in two different ways: either each daughter cell receives one mutated plasmid, or only one daughter cell receives both. In the latter case, the cell receiving both mutated plasmids produces less of the desired product and grows faster than the other cell. This growth advantage is due to the fact that the new synthetic pathway that has been inserted places a heavy metabolic burden on cells and reduces the cellular fitness. Therefore, cells with mutated plasmids accumulate and lead to a great productivity loss.</LI></P></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Allele segregation results in a rapid loss of productivity in plasmids</DIV></DIV></DIV></TD></TR></TBODY></TABLE><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><P>Therefore a new method was developed for the overexpression of a gene of interest in the bacterial chromosome: Chemically Inducible Chromosomal evolution (CIChE). In this technique the chromosome is evolved to contain a higher number of gene copies by adding a chemical inducer. </P><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
 Read more about CIChE
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><P>In 2009, Tyo <I>et al.</I> developed a technique for the stable, high copy expression of a gene of interest in <I>E. coli</I> without the use of high copy number plasmids, thus avoiding their previously stated negative characteristics. They called this plasmid-free, high gene copy expression system ‘chemically inducible chromosomal evolution’. In this method, the gene of interest is integrated in the microbial genome and then amplified to achieve multiple copies and reach the desired expression level. Genomic integration guarantees ordered inheritance, resolving the problem of allele segregation.</P><P>CIChE works as follows: First a construct, containing the gene(s) of interest and the antibiotic marker <I>chloramphenicol acetyl transferase</I> (<I>cat</I>) flanked by homologous regions, is delivered to and subsequently integrated into the <I>E. coli</I> genome. The construct can be amplified in the genome through tandem gene duplication by <I>recA</I> homologous recombination. Then the strain is cultured in increasing concentrations of chloramphenicol, providing a growth advantage for cells with increased repeats of the construct and thereby selecting for bacteria with a higher gene copy number (<B>Figure</B>).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Tyo et al., 2009</DIV></DIV></DIV></DIV><P>This process is called chromosomal evolution. When the desired gene copy number is reached <I>recA</I> is deleted, thereby fixing the copy number.</P><P>The use of CIChE-strains poses several advantages. They require no selection markers and can be cultured without the addition of antibiotics to the medium. This is in contrast to strains containing plasmids, which still need antibiotic selection. Also, in CIChE-strains yields can be tuned by varying the antibiotic concentration during chromosomal evolution.</P><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><P>It has been shown that approximately 40 and even <B>up to 50 gene copies can be attained using chromosomal evolution</B>. It has also been demonstrated that, while plasmid-bearing strains lose their productivity after 40 generations due to allele segregation, gene copy number and productivity of CIChE-strains remain <B>stable even after 70 generations</B>. This genetic stability is considered to be the most important asset of CIChE. </P><P>The <B>original model</B> for CIChE, however, results in bacterial strains containing a <B>large number of antibiotic resistance genes</B>. To make this valuable technique more widely applicable in the industry, <B>we developed</B> a model for chromosomal evolution based on a <B>toxin-antitoxin system</B> instead of antibiotic resistance.</P><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
 Read more about antibiotic resistance
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><P>Antibiotic resistance is the resistance of a bacterium to an antibacterial medicine a.k.a. antibiotic to which it was originally sensitive. When a pathogenic bacterium has acquired resistance to a certain antibiotic, infections of this bacterium can no longer be treated with the antibiotic in question since the bacterium has become insensitive to the antibiotic. 
Overuse and misuse of antibiotics has accelerated the emergence and spread of resistant bacteria. Today antibiotic resistance has become a fairly well-known term as a result of the recurring appearance of multi-drug resistant bacteria such as MRSA and VRE in media, awareness campaigns etc. These bacteria have become resistant to the commonly used antibiotics. As a result infections are very difficult to treat. The emergence of these hard-to-kill pathogenic bacteria poses a serious risk to public health. 
</P><P>
In biotechnology antibiotic resistance genes (the genes that make a bacterium resistant to a certain antibiotic) are often used as selectable markers. The best known application is probably the selection of plasmids. In the original CIChE technique a chloramphenicol resistance gene is used as selectable marker. During chromosomal evolution the antibiotic resistance gene is duplicated since it is the driving pressure for the tandem duplication of the CIChE-construct. As a result the final bacterial strain carries multiple antibiotic resistance genes. The creation of such a strain raises several safety concerns as bacteria can pass resistance genes on to other, possibly pathogenic, bacteria in the environment through a process called <I>horizontal gene transfer</I>. Although it is very unlikely to occur the possibility of horizontal gene transfer cannot be entirely ruled out. </P><P>The fact that the strain eventually contains multiple antibiotic resistance genes withholds this CIChE technique from being applicable in for example food industry, given several regulations as well as consumer’s mind-set. We will try to replace the use of antibiotic resistance genes with a toxin-antitoxin system. By eliminating the need for antibiotic resistance genes, CIChE could be applied in the industry without having to worry about the possibility of horizontal gene transfer and the spread of antibiotic resistance.</P><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><H1>Our model</H1><H2>Considered alternatives</H2><P>We examined several alternative mechanisms which may be able to replace the system with antibiotics and an antibiotic resistance gene. What we were looking for was a mechanism to put pressure on cells and to create an advantage for cells with increased copy numbers. For each of the following mechanisms we examined the advantages and disadvantages for its use in chromosomal evolution:</P><UL><LI>Triclosan and <I>fabI</I> overexpression</LI><LI>Operator-repressor titration</LI><LI>Solvents</LI><LI>Cationic antimicrobial peptides </LI><LI>Toxin-Antitoxin systems
</LI></UL><P>Considering all these alternatives the use of TA systems seemed to be the best option. Some important disadvantages of the other systems made us decide against them: the toxicity of <I>fabI</I> and the link between triclosan and antibiotic resistance, the uncertainty of sufficient repressor titration levels in operator-repressor titration, destabilization of the membrane using efflux pumps for solvents and the similarity of cationic antimicrobial peptides to antibiotics.</P><P> You can read the complete discussion of all alternatives in <A href="https://static.igem.org/mediawiki/2013/d/dc/UGent_2013_Bachelorproef.pdf" target="_blank">our paper</A> (page 6-20).  </P><H2>Toxin-Antitoxin systems</H2><P>In our opinion it is possible to eliminate the need for antibiotics by using a toxin-antitoxin system. These systems are widely distributed among bacteria and archaea. The toxins produced by these systems can slow down or stop cell growth by interfering with certain molecules that are essential in cellular processes like DNA replication, cell wall synthesis, ATP synthesis etc. Under normal conditions, the toxin is inhibited by the antitoxin, which is encoded in the same operon as the toxin. Research revealed different types of TA systems.</P><TABLE><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Mechanism of type II TA systems.</B> The antitoxin protein inhibits the toxin protein by forming a sstable complex with the toxin. This complex (as well as the antitoxin itself) is part of a feedback inhibition of the promoter.</DIV></DIV></DIV></DIV></TD><TD><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
 Read more about the different types of TA systems 
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><H3>Type I</H3><P>
The first category of TA systems is characterised by an antisense mechanism that regulates the expression of the toxin gene. This antisense RNA is transcribed from the same toxin region, but in reversed orientation, and encodes the antitoxin. This antitoxin anneals to the toxin mRNA and forms double-stranded RNA across the ribosome binding site. This induces the degradation of the toxin mRNA or the blocking of the ribosome binding site. As a result, the amount of toxin in the cell is reduced.</P><H3>Type II</H3><P><I>(Figure)</I>Antitoxins in type II TA systems are proteins and inhibit the corresponding toxin by forming a stable TA protein complex. These systems are encoded by an operon that contains the two genes encoding the toxin and antitoxin. The antitoxin has to be continuously produced, as it is less stable than the toxin. Because of this feature, the cell has a feedback mechanism to prevent growth arrest. The TA complex functions as a repressor that binds the promoter and thereby inhibits transcription. Thus, when the antitoxin is degraded and the concentration of the TA complex decreases, the TA operon is derepressed and produces more toxin and antitoxin. Finally, the concentration of the antitoxin is reestablished. The rank of the two genes differs, which can contribute to different toxin-antitoxin ratios.</P><P>The toxin and antitoxin are dependent, as they are both necessary for survival. Without the antidote the cell dies, but an efficient toxin might be vital to maintain a functional antitoxin.</P><H3>Type III</H3><P>In the last category of TA systems, an RNA antitoxin directly inhibits the toxin. These RNA antitoxins are pseudoknots, which contain internal stemloops. The toxin and its antidote are once more encoded by the same operon. A transcriptional terminator between the two genes, regulates the ratio of toxin and antitoxin in the cell.</P><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV></TD></TR></TBODY></TABLE><H3> CcdA/CcdB </H3><P>The F plasmid <I>ccd</I> (control of cell death or coupled cell division) TA module encodes the toxin CcdB and its antidote CcdA. This type II TA system is the first identified and best studied of all TA systems, which is why we have chosen to use it in our model. The <B>target</B> of the CcdB protein is the A subunit of <B>DNA gyrase</B>. This gyrase is an <B>essential</B> type II topoisomerase, occurring <B>in all bacteria but not in eukaryotes</B>. Bacterial gyrases have the property to introduce negative supercoils into DNA, which makes them unique among topoisomerases. As they are not present in eukaryotes, the toxin does not pose any risk for humans.</P><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
 Read more about the <I>ccd</I> module and our use of it 
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><P>In the absence of its antidote CcdA, CcdB causes reduced DNA synthesis, activation of the SOS pathway, cell filamentation and eventually cell death. When CcdB interacts with a gyrase:DNA complex, it stabilises the cleavable complex. In this mode of action, CcdB acts as a poison, promoting DNA breakage mediated by gyrase. The antidote CcdA can inhibit CcdB by the formation of a tight non-covalent complex. This prevents CcdB from making a covalent complex with gyrase. But even after CcdB has formed a complex with gyrase, CcdA can still reverse the blocking CcdB caused.</P><P>As CcdA has no toxic effects in the absence of CcdB, the <I>ccdA</I> gene will be used as selectable marker integrated in the CIChE construct, under the control of a constitutive promoter. By adding a plasmid bearing the ccdB gene under the control of a chemically inducible promoter, we suggest it is possible to regulate the amount of CcdB present in the cell titrating the chemical inducer. If this CIChE system works, cells with a higher gene copy number could be attained by titrating higher concentrations of the chemical inducer, as these have more CcdA to compensate for the toxic CcdB.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption"><B>Interaction between CcdB and the A subunit of DNA gyrase (GyrA).</B> (a) Free GyrA59 (the 59 kDa amino-terminal breaking rejoining domain of GyrA) in its closed conformation. (b) Opening up of the GyrA dimer creates space for a CcdB dimer. When CcdB binds to this conformation, normal working mechanism of gyrase is prohibited.</DIV></DIV></DIV></DIV><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><H2>Construct</H2><P> A construct containing the homologous regions, the antitoxin <I>ccdA</I> and <I>gfp</I> will be inserted in the genome of an <I>E. coli</I> strain which contains <I>recA</I>. <I>recA</I> is necessary to perform the tandem gene duplication by homologous recombination. When the desired copy number is reached, <I>recA</I> is deleted. A plasmid containing the <I>ccdB</I> toxin gene is transformed into the cell. Expression of <I>ccdB</I> is controlled by the T7-promoter. The <B>T7 promoter</B> is not recognised by the <I>E. coli</I> RNA polymerase. By using an inducible promoter to regulate the production of T7 RNAP, it is possible to change the amount of transcription of the <I>ccdB</I> toxin. By raising IPTG (inducer) concentrations, the selection pressure increases and chromosomal evolution is achieved.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Our model for CIChE</DIV></DIV></DIV></DIV><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
 Read why we chose <I>gfp</I> as gene of interest and T7 as inducible promoter 
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><H3>Gene of interest</H3><P>For our experiment, it would be useful to select a gene of interest that can easily be identified. Using a fluorescent reporter gene such as <I>gfp</I> is the obvious choice. When GFP is brought to expression in <I>E. coli</I> this results in the production of green light when illuminated with an ultraviolet source. A great advantage of GFP is the fact that no substrate is needed for its expression. Hence, GFP is ideal to measure gene expression, as it is not limited by substrate availability. The intensity of the fluorescence is proportional to the GFP presence in the cell, providing a first indication of the number of <I>gfp</I> genes present in the cell. Consequently, fluorescence intensity could give us a first insight in the number of construct copies attained through chromosomal evolution.</P><H3>Inducible promoter for <I>ccdB</I></H3><P>To increase the gene copy number in the chromosome, we have to be able to regulate the amount of CcdB in the cell. To achieve this, an inducible promoter is used. An inducible promoter is a combination of an operator and a promoter. The operator may be switched on or off which controls the transcription of the gene. Sometimes it is possible to regulate transcription according to the amount of stimulus the promoter receives. </P><P>The <B>T7 promoter</B> is not recognised by the <I>E. coli</I> RNA polymerase. This allows for good regulation of the transcription of genes, because there is no transcription as long as there is no T7 RNAP present. By using an inducible promoter to regulate the production of T7 RNAP, it is possible to change the amount of transcription of the desired protein. This is usually done by an IPTG inducible promoter.</P><P>Other options we considered were the <B>arabinose promoter</B> and the <B>lactose promoter</B>. We chose the T7 promoter because it shows lower leaky expression than these two. Leaky expression is transcription that occurs when no stimulus is given to the promoter. For the transcription of a toxic protein as in our case, leaky expression has to be avoided because even a small amount of transcription can be a burden to the cell and restrict cell growth.</P><P>The T7 promoter can be induced by IPTG. By using different concentrations we try to find a correlation between the applied IPTG concentration and the amount of gene copy numbers in the genome.</P><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><H1>Literature study</H1><P> If you would like to know the whole story, you can read our <A href="https://static.igem.org/mediawiki/2013/d/dc/UGent_2013_Bachelorproef.pdf" target="_blank">paper</A>!
All references can be found in our paper.
</P><P><TITLE>Project: NetSite...JavaScript Countdown Clock</TITLE></P><FORM name="clock"><TABLE border="1" frame="void" rules="none"><TBODY><TR><TD align="CENTER" width="31%" bgcolor="#F9BD05"><FONT color="#FFFFFF"><B>Days:</B></FONT></TD><TD align="CENTER" width="23%" bgcolor="#F9BD05"><FONT color="#FFFFFF"><B>Hours:</B></FONT></TD><TD align="CENTER" width="23%" bgcolor="#F9BD05"><FONT color="#FFFFFF"><B>Mins:</B></FONT></TD><TD align="CENTER" width="23%" bgcolor="#F9BD05"><FONT color="#FFFFFF"><B>Secs:</B></FONT></TD></TR><TR><TD colspan="4" bgcolor="#F9BD05"><CENTER><FONT face="Verdana, Arial,sans-serif" size="+1.5" color="#FFFFFF">Until Jamboree Lyon</FONT></CENTER></TD></TR></TBODY></TABLE></FORM><A class="twitter-timeline" href="https://twitter.com/iGEM_UGent" data-widget-id="373064939419607041">Tweets van @iGEM_UGent</A><TABLE border="0" cellspacing="0" cellpadding="0" width="70%" align="center"><DIV align="right"><H1>We thank following sponsors for their support</H1></DIV></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UGent/LiteratureStudy">http://2013.igem.org/Team:UGent/LiteratureStudy</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UGent/LiteratureStudy" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UGent/LiteratureStudy" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UGent/LiteratureStudy&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UGent/LiteratureStudy&amp;oldid=313336" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>