"
Page
Discussion
View source
History
teams
Log in
 
Team:UGent/Labjournal
From 2013.igem.org
Home
Team
Undergrads Instructors Official team page
Project
Literature Study Parts Results
Notebook
Labwork
Experiments CloneManager Labjournal
Social Aspects
Human Outreach Ethics Survey Press
Safety
Achievements
Attributions
Journal
July
Week 4: july 22 - 28
Introduction given by our lab instructors
General techniques: plasmid/PCR purification, inoculation, gel extraction, restriction & ligation.
Safety and waste disposal training
Introduction to CloneManager
General preparations: sterile mQ, sterile eppendorf
Close
August
Week 1: july 29 - august 4
Close
Week 2: august 5 - 11
Experiment 1
Inoculate E. coli DH5a + pTGD-ccdA-Pmb1GFP-CmFRT
Purify plasmid pTGD-ccdA-Pmb1GFP-CmFRT using Qiagen spin mini kit: Nanodrop -- 362.4 ng/µL
Preparative Restriction Digest (RD) of purified plasmid: BspHI & BstAPI: expected fragments of 5541bp and 903 bp
Gel purify RD-fragment of 5541 bp using Qiagen Qiaquick gel extraction kit: Nanodrop -- 70.4 ng/µL
PCR on RD (HR-ccdA-Pmb1GFP-HR)and on plasmid pTGD-ccdA-PMbAFGP-CmFRT  with MDM588 & MDM589
->HiFi PCR using Primestar polymerase. Analytical gel: negative
->PCR using Q5 polymerase. Analytical gel: negative
->PCR using Roche. Analytical gel: negative
->Touchdown PCR using Q5 polymerase. Analytical gel: negative
Control plasmid pTGD-ccdA-PMbAFGP-CmFRT: RD with AclI. Expected fragments: 758 bp, 1730 bp and 3956 bp. Analytical gel: negative --> Problem with plasmid pTGD-ccdA-PMbAFGP-CmFRT
Transformation: Knock in with linear back-up DNA by electroporation. Incubation of the transformed cells and transfer the culture on a Cm plate.
Colony PCR on 48 colonies using crimson taq polymerase with two primer pairs: MDM0141/MDM0010 and MDM0046/MDM123. Expected fragments: 550bp & 2450 bp. Analytic gel: 4 positives
Colony PCR on 4 positive and 4 negative colonies using crimson taq polymerase with out primers: MDM0046/MDM0010. Expected fragment: ca. 5500 bp. Analytic gel: negative .
2 Colony PCR on 4 positive and 4 negative colonies using Taq and Phire polymerase with out primers: MDM0046/MDM0010. Expected fragment: ca. 5500 bp. Analytic gel: negative .
Colony PCR on 4 positive and 4 negative colonies using Emerald polymerase with out primers: MDM0046/MDM0010. Expected fragment: ca. 5500 bp. Analytic gel: 1 positive: colonie 35 .
Experiment 2
Inoculate E. coli DH5a; + p5SpFRT-T7ccdB
Inoculate E. coli DH5a; + p10SpFRT-T7ccdB
Inoculate E. coli DH5a; + p20SpFRT-T7ccdB
Purify plasmids using Qiagen spin mini kit: nanodrop
p5SpFRT-T7ccdB: 119.6 ng/µl
p10SpFRT-T7ccdB: 156.3 ng/µl
p20SpFRT-T7ccdB: 392.9 ng/µl
CcdB operon:
-> HiFi PCR of plasmids p5SpFRT-T7ccdB, p10SpFRT-T7ccdB and p20SpFRT-T7ccdB with MDM0586/MDM0587 to amplify ccdB operon
-> Purification of CcdB operon PCR fragment using Qiagen Qiaquick PCR purification kit and checked on analytical gel (expected fragment of 2300 bp): nanodrop
p5: 174.6 ng/µl
p10: 24.5 ng/µl (consistent with small band on the analytival gel)
p20: 104.3 ng/µl
-> Redo of inoculation E. coli DH5a + p5SpFRT-T7ccdB, E. coli DH5a + p10SpFRT-T7ccdB and E. coli DH5a + p20SpFRT-T7ccdB
-> Redo of purification of plasmids p5SpFRT-T7ccdB, p10SpFRT-T7ccdB and p20SpFRT-T7ccdB using Qiagen Spin minikit: nanodrop
p5SpFRT-T7ccdB: 21.1 ng/µl
p10SpFRT-T7ccdB: 25.1 ng/µl
p20SpFRT-T7ccdB: 24.6 ng/µl
Vectors pSB4A5 (plate 5, well 5I), pSB3T5 (plate 2, well 8D) and pSB6A1 (plate 2, well 2L):
-> Resuspend plasmids from the iGEM kit
-> Transform in E. coli Top10 subcloning cells using elektroporation
-> Plate pSB4A5 and pSB6A1 on ampicillin plate and pSB3T5 on tetracyclin  and grow overnight at 37°C
-> Plates with transformants: pSB6A1 lots of colonies, pSB3T5 sufficient colonies and pSB4A5 no colonies
Vectors pSB4A5 (plate 5, well 5I):
-> Resuspend plasmids from the iGEM kit
-> Transform in E. coli Top10 subcloning cells using elektroporation
-> Plate on ampicillin plate and grow overnight at 37°C (1 plate 150 µl, 1 plate 50 µl)
-> Plates with transformants: pSB4A5 no colonies
Inoculation colonies of pSB3T5 and pSB6A1 -> at the end of the day replaced from 37°C to 30°C
Vector pSB4A5 (plate 5, well 1I and plate 2, well 2J as backup) and pSB6A1 (plate 5, well 1K as backup):
-> Resuspend plasmid pSB4A5 and pSB6A1 from the iGEM kit
-> Transform in E. coli Top10 subcloning cells (heat shock)
-> Plate transformation on ampicillin plate and grow overnight at 37°C (3 plates: 10-2,10-1 and 100)
-> Plates with transformants: pSB4A5 (plate 2, well 2J) colonies, pSB4A5 (plate 5, well 1I) no colonies and pSB6A1 (plate 2, 2L) few colonies
Inoculate pSB3T5 and pSB6A1  again and in warm chamber (37°C).
Inoculation colonies of pSB4A5 (plate 2, well 2J) and pSB6A1 (plate 2, well 2L) (2 times: one for further work and one for cryovials)
Purification of plasmids pSB3T5, pSB6A1 and pSB4A5
Generation of restriction digest fragments of T7-ccdB insert (from p5SpFRT-T7ccdB and p20SpFRT-T7ccdB) and vector (pSB3T5 (2x), pSB4A5 (from plate 2, well 2J) and pSB6A1 (1x from plate 5, well 1K and 1x from plate 2, well 2L) with XbaI and PstI-HF (gel purify RD-fragments using Qiagen Qiaquick gel extraction kit (expected fragment of 2157 bp for T7-ccdB, 3229 bp for pSB3T5, 3372 bp for pSB4A5 and 3999 bp for pSB6A1): Nanodrop
p5: 43.7 ng/µl
p20: 24.2 ng/µl
pSB3T5 (inoculation 1): 23.5 ng/µl
pSB3T5 (inoculation 2): 23 ng/µl
pSB4A5: 40.5 ng/µl
pSB6A1 (plate 2, 2L): 40.1 ng/µl
pSB6A1 (plate 5, 1K): 30.4 ng/µl
Ligation of T7-ccdB from p5SpFRT-T7ccdB and pSB6A1 (1x from plate 5, well 1K and 1x from plate 2, well 2L), p20SpFRT-T7ccdB and pSB3T5, p20SpFRT-T7ccdB and pSB4A5 (from plate 2, well 2J)  (3(insert)/1(plasmid) ratio)
Transformation in E. coli Top10 (heat shock)
Plate transformation on ampicillin (pSB4A5 and pSB6A1) and tetracycline plate (pSB3T5) and grow overnight at 37°C (2 plates: 10-2, 10-1)
-> Plates with transformants: no colonies
New transformation in E. coli Top10 from pSB4A5 and pSB6A1 using heat shock
-> Plates with transformants: pSB6A1 2 colonies, pSB4A5 few colonies
New ligation with RD-fragments created before (vectors pSB3T5, pSB4A5, pSB6A1 (plate 5, well 1K) and insert T7-ccdB created in experiment 6), at 22.5°C o/n)
CcdB operon:
-> HiFi PCR of plasmids p5SpFRT-T7ccdB, p10SpFRT-T7ccdB and p20SpFRT-T7ccdB with MDM0586 & MDM0587 to amplify ccdB operon
Experiment 6
CcdB operon:
-> HiFi PCR of plasmids p20SpFRT-T7ccdB with MDM0586/MDM0587 to amplify ccdB operon
nanodrop: 301,5 ng/µl
Vector pSB1C3 (plate 1, well 23O):
-> Resuspend plasmids from the iGEM kit
-> Transform pSB1C3 in E. coli Top10 subcloning cells, using heat shock (42°C)
-> Plate on chloramphenicol plate and grow overnight at 37°C
-> Inoculate E. coli Top10 + pSB1C3
-> Purify plasmids using Qiagen Qiaprep Spin minikit: nanodrop: 200,2 ng/µl
Restriction of CcdB operon and pSB1C3 with XbaI and PstI
Preparative gel:
restriction digest of T7-ccdB (expected to be 2044bp) and restriction digest of backbone of pSB1C3 (expected length is 2232 bp)
-> Gel purify fragments using Qiagen Qiaquick gel extraction kit: nanodrop:
Restriction digest of T7-ccdB: 16.9 ng/µl
Restriction digest of pSB1C3: 12.3 ng/µl
Ligation of CcdB operon and pSB1C3
Transformation in E. coli Top10 subcloning cells, using heat shock (42°C)
-> Plate on chloramphenicol plate and grow overnight at 37°C: negative
Close
Week 3: august 12 - 18
Experiment 1
Inoculation positive colonie on cm plate.
Colony PCR on 8 kolonies using emerald polymerase with out primers: MDM0046/MDM0010. Expected fragment: ca. 5500 bp. Analytic gel: positive: 5 & 8 .
Experiment 2
PCR purification of CcdB operon: nanodrop
p5: 54.8 ng/µl
p10: 65.8 ng/µl
p20: 21.1 ng/µl
cPCR with Taq polymerase of colonies pSB6A1 (primers: MDM0096 and CLG0019) and pSB4A5 (primers: MDM0095 and CLG0019): negative
New transformation with pSB3T5, pSB4A5 and pSB6A1 using heat shock in E. coli DH5a
-> Plate on normal plates and glucose plates (100 ml 2% glucose, to avoid leaky expressing of ccdB)
-> Plates with transformants: pSB6A1 some colonies, pSB4A5 some colonies and pSB3T5 no colonies
Gel elektroforesis on RD-fragments from plasmids: show expected fragments
cPCR on the pSB4A5 (primers: MDM0095 and CLG0019) and pSB6A1 (primers: MDM0096 and CLG0019) colonies with Taq polymerase: one positive colony of pSB6A1-T7ccdB (fragment of 437 bp in lane 14)
inoculation of positive pSB6A1-T7ccdB colony from back up plate (3x: one for sequencing, 1 for cryovial and 1 for experiment 3)
Plate transformation mixtures again on glucose plates and grow overnight at 30°C
->Plates with transformants: pSB4A5 colonies, pSB3T5 no colonies
cPCR on pSB4A5 colonies (primers: MDM0095 and CLG0019) with Taq polymerase: negative
New ligation with rest of RD-fragments of plasmids pSB3T5 (2 times) and pSB4A5 and CcdB from experiment 6 and transformation in DH5a using heat shock
Due to lack of success: start from the beginning.
Inoculation of pSB3T5, pSB4A5, p5SpFRT-T7ccdB, p10SpFRT-T7ccdB and p10SpFRT-T7ccdB from cryovials.
Also resuspend pSB3T5 (plate 5, well 7C) from iGEM kit, transform in E. coli DH5a using heat shock, plate on tetracycline plates and incubate overnight at 37°C
-> no colonies
-> transform again in DH5a using heat shock
-> again no colonies
Purification of plasmids using Qiagen spin minikit: concentrations deduced from gel
p5SpFRT-T7ccdB: +/- 40 ng/µl
p10SpFRT-T7ccdB: +/- 40 ng/µl
p20SpFRT-T7ccdB: +/- 40 ng/µl
pSB3T4: +/- 250 ng/µl
pSB4A5 (1): +/- 400 ng/µl
pSB4A5 (2): +/- 350 ng/µl
PCR on CcdB operon using Q5 polymerase (primers: MDM0586_Fw-Trc-ccdB-G00000 and MDM0587_Rv-Trc-ccdB-G00001)
cPCR on pSB3T5 colonies from old plates using Taq polymerase (primers: MDM0602 and CLG0019, expected fragment of 503 bp): negative
Restriction of ccdB operon, pSB3T5 and pSB4A5 using PstI-HF and XbaI (gel purify RD-fragments using Qiagen Qiaquick gel extraction kit (expected fragment of 2157 bp for T7-ccdB, 3229 bp for pSB3T5 and 3372 bp for pSB4A5): concentrations deduced from gel
Ligation of pSB3T5 and T7-ccdB (from p5SpFRT-T7ccdB), pSB4A5 and T7-ccdB (from p5SpFRT-T7ccdB) and pSB4A5 and T7-ccdB (from p20SpFRT-T7ccdB) with T4 DNA ligase at 16°C overnight
Transformation of ligation mixtures in DH5a using heat shock
-> no colonies
Experiment 3
Inoculation KI-strain 5 & 8 from experiment 1
Transformation of pSB6A1 - T7-ccdB in KI-strains 5 and 8
Colony PCR on kolonies from transformed strains using emerald polymerase with out primers to check KI: MDM0046/MDM0010. Expected fragment: ca. 5500 bp. Analytic gel: positive
Colony PCR on kolonies from transformed strains using emerald polymerase with MDM0096 & CLG0019 to check if plasmid is present. Expected fragment: 437 bp. Analytic gel: positive
Experiment 6
New transformation in E. coli Top10 subcloning cells, using heat shock (42°C)
-> Plate on chloramphenicol plate and grow overnight at 37°C: negative
CcdB operon: HiFi PCR of plasmids p20SpFRT-T7ccdB with MDM0586/MDM0587 to amplify CcdB operon
New restriction of CcdB operon and pSB1C3 with Xba I and PstI
-> Gel purify fragments using Qiagen Qiaquick gel extraction kit: nanodrop:
Restriction Digest of T7-ccdB: 16.6 ng/µl
Restriction Digest of pSB1C3: 17.7 ng/µl
Ligation of CcdB operon and pSB1C3: overnight at 16°C
Transformation in E. coli DH5a, using heat shock (42°C)
-> Plate on chloramphenicol + glucose plate and grow overnight at 30°C: negative
Restriction of CcdB operon and pSB1C3 with XbaI and PstI
Because a preparative gel purification cause low concentrations, we will use DpnI
-> Purification by using minElute reaction cleanup kit
Control of restriction fragments: positive, so we hope that ligation will succeed
Ligation of CcdB operon and pSB1C3: overnight at 16°C
Transformation in E. coli DH5a, using heat shock (42°C)
-> Plate on chloramphenicol plate and grow overnight at 37°C: negative
(it does not work, maby our transformation is not succeed)
Close
Week 4: august 19 - 25
Experiment 2
pSB6A1-T7ccdB sent for sequencing with primers MDM0096 (A471969), MDM0060 (A471682) and MDM0039 (A471681)
-> correct sequence
Transform ligation mixtures again in DH5a, using heat shock (42°C),
-> no colonies
PCR on CcdB operon with Q5 polymerase (primers: MDM0586_Fw-Trc-ccdB-G00000 & MDM0587_Rv-Trc-ccdB-G00001) and PCR purification using Qiagen Qiaquick PCR purification kit: nanodrop
p5: 15.2 ng/µl
p10: 18.6 ng/µl
p20: 43.5 ng/µl
Inoculation of pSB3T5 and pSB4A5 from cryovials and plasmid purification using Qiagen Spin minikit: nanodrop
pSB3T5: 75.7 ng/µl
pSB4A5: 190.0 ng/µl
Restriction of CcdB operon, pSB3T5, pSB4A5 and the pSB6A1-T7ccdB transformant with PstI-HF and XbaI (gel purify restriction digest-fragments using Qiagen Qiaquick gel extraction kit (expected fragment of 2157 bp for T7-ccdB, 3229 bp for pSB3T5 and 3372 bp for pSB4A5)
Ligation
Transformation in E. coli DH5a
-> no colonies
-> plate transformants on plates with a lower antibiotic concentration (50% lower)
-> no colonies
Design primers for Gibson assembly
Experiment 3
Colony PCR with higher annealing temperature to minimize false priming on kolonies from transformed strains using emerald polymerase with out primers to check KI: MDM0046/MDM0010. Expected fragment: ca. 5500 bp. Analytic gel: positive
HiFi PCR using Q5 polymerase with two primer pairs generating overlapping fragments of the ccdA.gfp.cat construct : MDM0046 (out) & MDM0123: 2130 bp and MEMO1237 & MDM0010 (out): 3400 bp. Analytic gel: negative
Experiment 4
Lysate preparation:
-> Dilution of o/n culture of BW26547 in 5 mL LB+MgCL2+CaCl2+glucose
-> Add phage lysate (10,20,40 & 80 µL)
-> Wait untill lysis visible
Perform CIChE with 8 + pSB6A1 & 5 + pSB6A1 (we will do 2 strains parallel): 0,0mM - 0,05mM IPTG
Transduction of CIChe strains: no positive colonies after UV-test
Experiment 6
New transformation in E. coli DH5a, using heat shock (42°C)
(because we are sure that the fragments were present for ligation and so will lead to succesful results)
-> Plate on chloramphenicol plate and grow overnight at 37°C: negative (it still does not work)
Control of ligations by using PCR with primers MDM0606 and MDM0607: negative
Ligation again of CcdB operon and pSB1C3: 15 minutes at 16°C
Transformation in E. coli DH5a, using heat shock (42°C)
-> Plate on chloramphenicol plate and grow overnight at 37°C: negative
CcdB operon: HiFi PCR of plasmids p20SpFRT-T7ccdB with MDM0586/MDM0587 to amplify CcdB operon
because we have no more T7-ccdB fragments
New restriction of CcdB operon and pSB1C3 with XbaI and PstI
-> Gel purify fragments using Qiagen Qiaquick gel extraction kit: nanodrop:
Restriction Digest of T7-ccdB: 15.3 ng/µl
Restriction Digest of pSB1C3: 16.8 ng/µl
Ligation of CcdB operon and pSB1C3: 1 hour at room temperature
Transformation in E. coli DH5a, using heat shock (42°C)
-> Plate on chloramphenicol plate and grow overnight at 37°C: positive (finally)
Colony PCR (Taq): negative
It seems that restriction-ligation does not work, thus we will use another technique, called Gibson Assembly
Designing primers for Gibson Assembly for the insert T7-ccdB and the backbone pSB1C3
Close
Week 5: august 26 - september 1
Experiment 2
Inoculation of pSB3T5, pSB4A5, pSB6A1-T7ccdB, p5SpFRT-T7ccdB, p10pFRT-T7ccdB and p20pFRT-T7ccdB and plasmid purification using Qiagen Spin minikit: nanodrop
p10pFRT-T7ccdB: 62.8 ng/µl
p20pFRT-T7ccdB: 286.1 ng/µl
PCR on CcdB operon using Q5 polymerase (primers: MDM0586_Fw-Trc-ccdB-G00000 and MDM0587_Rv-Trc-ccdB-G00001): nanodrop
p10: 29.7 ng/µl
p20: 31.4 ng/µl
Restriction of pSB3T5, pSB4A5 and CcdB operon with PstI-HF and XbaI (gel purify RD-fragments using Qiagen Qiaquick gel extraction kit (expected fragment of 2157 bp for T7-ccdB, 3229 bp for pSB3T5 and 3372 bp for pSB4A5): nanodrop:
pSB3T5: 2.7 ng/µl
pSB4A5: 7.8 ng/µl
T7-ccdB: 10.5 ng/µl
Q5 PCR of CcdB operon and plasmids (pSB3T5 and pSB4A5), using the designed Gibson primers
-> Purification of PCR-product after DpnI treatment by using minElute reaction cleanup kit
-> Control of fragments on a gel => Backbones are present, CcdB operons not
PrimeStar PCR of CcdB operons, using the designed Gibson primers
-> Purification of PCR-product after DpnI treatment by using minElute reaction cleanup kit
-> Control of fragments on a gel => CcdB operons are still not present
Q5 PCR of CcdB operons on a linear CcdB operon, using the designed Gibson primers
-> Purification of PCR-product, using Qiagen Qiaquick PCR purification kit,
-> Control of fragments on a gel => CcdB operon is visible
Gibson Assembly (1h at 50°C)
Transformation in E. coli DH5a, using heat shock (42°C)
-> Plate pSB3T5 on tetracyclin plate and grow overnight at 37°C: positive
-> Plate pSB4A5 on ampicillin plate and grow overnight at 37°C: positive
Strange phenomena: most colonies are red, let’s control it
Colony PCR of colonies derived from Gibson Assembly: negative
Experiment 3
Transformation of plasmids:p5SpFRT-T7ccdB, p10SpFRT-T7ccdB, p20SpFRT-T7ccdB in KI-strain (8) and back-up strain 5
Colony PCR on 5 + pSB6A1/p5/p10/p20 using emerald polymerase with primerpairs MDM0046/MDM010 to check KI. Expected fragment: 5500bp. Analytic gel: positive
Colony PCR on 5 + pSB6A1/p5/p10/p20 using emerald polymerase with primerpairs MDM0096/CGL0019 for pSB6A1 and MDM0039/MDM0060 for p5/p10/p20. Expected fragment: 437bp & 910 bp. Analytic gel: positive for p5/p10/p20
New transformation 5 +pSB6A1 and check KI + plasmid as described above. Analytic gel: positive .
Experiment 4
Abort CIChe with 8+pSB6A1 & 5+pSB6A1 because of mutation in pSB6A1
Lysate preparation
Transduction: on CIChE strains described above
Test lysate by performing transduction on strain 5 with different concentrations of lysate: negative
Experiment 6
While waiting for the Gibson Assembly primers
Again restriction of CcdB operon and pSB1C3 with XbaI and PstI
-> Gel purify fragments using Qiagen Qiaquick gel extraction kit: nanodrop:
Restriction Digest of T7-ccdB: 10.5 ng/µl
Restriction Digest of pSB1C3: 7.2 ng/µl
Ligation of CcdB operon and pSB1C3: overnight at 16°C
Transformation in E. coli DH5a, using heat shock (42°C),
-> Plate on chloramphenicol plate and grow overnight at 37°C: positive
Colony PCR (Taq) of these colonies: negative
After receiving the primers for Gibson Assembly
Q5 PCR of CcdB operon and pSB1C3, using the designed Gibson primers
-> Purification of PCR-product, after DpnI treatment, by using minElute reaction cleanup kit
-> Control of fragments on a gel => Backbone is present, CcdB operon not
PrimeStar PCR of CcdB operon, using the designed Gibson primers
-> Purification of PCR-product, after DpnI treatment, by using minElute reaction cleanup kit
-> Control of fragments on a gel => CcdB operon is not present
Q5 PCR of CcdB operon on a linear CcdB operon, using the designed Gibson primers
-> Purification of PCR-product, using Qiagen Qiaquick PCR purification kit
-> Control of fragments on a gel => CcdB operon is visible
Gibson Assembly (1h at 50°C)
Transformation in E. coli DH5a, using heat shock (42°C),
-> Plate on chloramphenicol plate and grow overnight at 37°C: positive
Strange phenomena: colonies are red, let’s control it
Colony PCR (Taq) of colonies derived from Gibson Assembly: negative
We think that some of the original plasmid has to be present
and the bacteria love that plasmid more than the plasmid with T7-ccdB
Close
September
Week 1: september 2 - 8
Experiment 2
Restriction of CcdB operon and pSB3T5 with EcoRI and SpeI
-> Gel purify fragments using Qiagen Qiaquick gel extraction kit: nanodrop:
Restriction Digest of T7-ccdB: 14.4 ng/µl
Restriction Digest of pSB3T5 :  7.2 ng/µl
Restriction of CcdB operon and pSB4A5 with EcoRI and SpeI
-> Gel purify fragments using Qiagen Qiaquick gel extraction kit: nanodrop:
Restriction Digest of T7-ccdB: 14.4 ng/µl
Restriction Digest of pSB4A5: 7.8 ng/µl
Ligation of CcdB operon and pSB3T5 and of CcdB operon and pSB4A5: 25 minutes at room temperature
Transformation in E. coli DH5a using heat shock
-> Plate pSB3T5 on tetracyclin plate and grow overnight at 37°C: negative
-> Plate pSB4A5 on ampicillin plate and grow overnight at 37°C: negative
Gibson Assembly (1h at 50°C) of pSB3T5 and CcdB operon
Transformation in E. coli Top10 subcloning cells, using electroporation (Time Constant: 4.3)
-> Plate on tetracyclin plate and grow overnight at 37°C: negative
Gibson Assembly (1h at 50°C) of pSB4A5 and CcdB operon
Transformation in E. coli Top10 subcloning cells, using electroporation (Time Constant: 4.4)
-> Plate on ampicillin plate and grow overnight at 37°C:negative
Experiment 4
CIChE with strain 8 + pSB6A1/p5/p10&p20:
--> Inoculated without antibiotic & plated out on antibiotic: 0mM - 0.5mM IPTG
--> Inoculated in antibiotic - sequentially: 0mM - 0.5mM IPTG
--> Inoculated in antibiotic - parallel: 0mM -0.5 mM IPTG
Transduction: on CIChE strains described above --> colonies on control plate
Experiment 5
Test evaluation GFP with LB-medium:
-> Wild type
-> DH5a + pTGD-ccd-Pmb1GFP-CmFRT
-> Strains with GFP: 8 / 5 + pSB6A1/p5/p10/p20 -- 0mM & 0.01 mM IPTG
Test evaluation GFP with EZrich-medium:
-> Wild type
-> DH5a + pTGD-ccd-Pmb1GFP-CmFRT
-> Strains with GFP: 8 / 5 + pSB6A1/p5/p10/p20 -- 0mM/0.01mM/0.05mM & 0.2mMM IPTG
Experiment 6
Restriction of CcdB operon and pSB1C3 with EcoRI and SpeI
-> Gel purify fragments using Qiagen Qiaquick gel extraction kit: nanodrop:
Restriction Digest T7-ccdB: 14.4 ng/µl
Restriction Digest pSB1C3: 4.4 ng/µl
Ligation of CcdB operon and pSB1C3: 25 minutes at room temperature
Transformation in E. coli DH5a, using heat shock (42°C),
-> Plate on chloramphenicol plate and grow overnight at 37°C : negative
Q5 PCR of pSB1C3, using the designed Gibson primers and the linearised plasmid
-> Purification of PCR-product, using Qiagen Qiaquick PCR purification kit: nanodrop:
pSB1C3-backbone: 109.9 ng/µl
Gibson Assembly (1h at 50°C)
Transformation in E. coli Top10 subcloning cells, using electroporation (Time Constant: 4.4)
-> Plate on chloramphenicol plate and grow overnight at 37°C: positive
Colony PCR (Crimson) using primers MDM0606 and MDM0607:
6 colonies can possible be positive (but it is not sure)
-> Inoculate the 6 colonies (overnight)
Control of fragment of pSB1C3 on a gel => Backbone is not present
Plasmid purification of the 6 colonies + restriction to control of they are the good colonies (on a gel)
There is a possibility that they are the good ones, so we will send it to sequenate
-> pSB1C3-T7ccdB sent for sequencing with primers MDM0606, MDM0607, MDM0060 and MDM0039: negative
Q5 PCR of pSB1C3, using the designed Gibson primers and the linearised plasmid
-> Purification of PCR-product, using Qiagen Qiaquick PCR purification kit:
nanodrop: pSB1C3-backbone: 58.0 ng/µl
-> Control of fragment of pSB1C3 on a gel => Backbone is not present
Q5 PCR of pSB1C3, using the designed Gibson primers and the restriction digest of pSB1C3
-> Purification of PCR-product, using Qiagen Qiaquick PCR purification kit:
nanodrop: pSB1C3-backbone: 76.2 ng/µl
-> Control of fragment of pSB1C3 on a gel => Backbone is present
Gibson Assembly (1h at 50°C)
Transformation in E. coli Top10 subcloning cells, using electroporation (Time Constant: 4.3)
-> Plate on chloramphenicol plate and grow overnight at 37°C: positive => many colonies
Colony PCR (Crimson):
We tested 48 colonies (using primers MDM0606 and MDM0607) and 4 colonies show the right fragment (2520 bp)
The first 24 colonies
, and the second 24 colonies
-> Inoculate these 4 good colonies
Plasmid purification of the 4 colonies, using Qiagen Qiaprep Spin minikit: nanodrop:
-> pSB1C3 strain 1:  94.2 ng/µl
-> pSB1C3 strain 2: 163.4 ng/µl
-> pSB1C3 strain 3: 160.2 ng/µl
-> pSB1C3 strain 4: 124.3 ng/µl
Restriction with BspHI to control on a gel of they are the right colonies
(expected fragments are 1028bp and 3248 bp): positive
-> Strains 2, 3 and 4 seems to be the correct colonies
pSB1C3-T7ccdB of these 3 colonies sent for sequencing with primers MDM0606, MDM0607, MDM0060 and MDM0039
-> sequence shows a mutation in the stop-codon, so we have to start over again
Close
Week 2: september 9 - 15
Experiment 2
Q5 PCR for amplification of T7-ccdB-fragment, using the designed Gibson primers, on p5SpFRT-T7ccdB, p10SpFRT-T7ccdB and p20SpFRT-T7ccdB
-> Using DpnI at 37°C for 1 hour
-> Purification of PCR-product by using minElute reaction cleanup kit
-> Control on gel: negative
Plasmid pSB3T5
Gibson Assembly (1h at 50°C) of pSB3T5 and CcdB operon that we already amplified before
Transformation in E. coli Top10 subcloning cells, using electroporation (Time Constant: 3.0)
-> Plate on tetracyclin plate and grow overnight at 37°C: negative, there are oly red colonies
Plasmid pSB4A5
Gibson Assembly (1h at 50°C) of pSB4A5 and CcdB operon that we already amplified before
Transformation in E. coli Top10 subcloning cells, using electroporation (Time Constant: 4.0)
-> Plate on ampicillin plate and grow overnight at 37°C: positive
Colony PCR (Crimson): there is only 1 colony. We tested this colony (using primers MDM0606 and MDM0607) and it show the right fragment (2486 bp) on gel
-> Inoculate these colonies
Plasmid purification of the colonies, using Qiagen Qiaprep Spin minikit
Seqenate this plasmid: the result is a totally different sequence than expected
Plasmid pSB6A1
We controled the pSB6A1 on mutations (by sequencing) and we found also a mutation on in the ccdB gene
Try to design Gibson primers to undo the mutations
(We will use primers to amplify the biggest part of the plasmid (PCR) and afterwards using Gibson Assembly with oligos, which undo the founded mutation)
Q5 PCR for amplification of biggest part of the plasmid pSB6A1 (using the designed Gibson primers).
-> Using DpnI at 37°C for 1 hour
-> Purification of PCR-product by using minElute reaction cleanup kit
-> Control on gel: positive (fragment length is 6192bp)
Gibson Assembly
-> melt oligos first: 5 minutes at 95°C
-> 6192bp-fragment and oligos at 50°C (1 h)
Transformation in E. coli Top10 subcloning cells, using electroporation  (Time Constant: 2.4)
-> Plate on ampicillin plate and grow overnight at 37°C: positive
We made our own competent cells which have the ccdA gene in the genome, so they will be more tolerant to CcdB. We hope on no mutations this time.
-> Q5 PCR: Control of Knock-In
-> Control on gel: positive
-> Further on we will call these cells competent strain 8
Second transformation in competent strain 8, using electroporation  (Time Constant: 1.6)
-> Plate on ampicillin plate and grow overnight at 37°C: negative
Gibson Assembly
-> melting oligos first: 5 minutes at 95°C + infinite at 50°C
-> 6192bp-fragment and oligos at 50°C (1 h)
Transformation in competent strain 8, using electroporation (Time Constant: 1.7)
-> Plate on ampicillin plate and grow overnight at 37°C: negative
Experiment 4
CIChE with strain 8 + p5/p10/p20:
-> STEP: 0mM - 0.1mM IPTG
-> JUMP: 0mM - 0.5mM IPTG
CIChE with strains 5 + p5/p10/p20:
-> STEP + JUMP: 0 - 0.01mM
Experiment 5
MTP experiment: version 1 (using precultures)
(Evaluation GFP 4 colonies/strain -  3 repeats)
-> 8+pSB6A1 - sequential: 0mM - 0.5mM IPTG
-> 8+pSB6A1 - strains grown without antibiotics: 0mM - 0.5mM IPTG
-> 8+p10 - sequential: 0mM - 0.05mM  IPTG
Measure the OD and the GFP (using FLUOstar)
Experiment 6
Q5 PCR for amplification of T7-ccdB-fragment, using the designed Gibson primers, on p5SpFRT-T7ccdB, p10SpFRT-T7ccdB and p20SpFRT-T7ccdB
-> Using DpnI at 37°C for 1 hour
-> Purification of PCR-product by using minElute reaction cleanup kit
-> Control on gel: negative
Gibson Assembly (1h at 50°C), using a T7-ccdB we already amplified before
Transformation in E. coli Top10 subcloning cells, using electroporation (Time Constant: 4.0)
-> Plate on chloramphenicol plate and grow overnight at 37°C: negative
Try to design Gibson primers to undo the mutations, found in the ccdB gene
(We will use primers to amplify the biggest part of the plasmid (PCR) and afterwards using Gibson Assembly with oligos, which undo the founded mutation)
Q5 PCR for amplification of biggest part of the plasmid (using the designed Gibson primers).
We will do this on the 2 best colonies of last time.
-> Using DpnI at 37°C for 1 hour
-> Purification of PCR-product by using minElute reaction cleanup kit
-> Control on gel: positive (fragment length is 4233bp)
Gibson Assembly
-> melt oligos first: 5 minutes at 95°C
-> 4233bp-fragment and oligos at 50°C (1 h)
Transformation in E. coli Top10 subcloning cells, using electroporation
(Time Constant for colony 1: 4.0, for colony 2: 4.1)
-> Plate on chloramphenicol plate and grow overnight at 37°C: positive
We made our own competent cells which have the ccdA gene in the genome = strain 8, so they will be more tolerant to CcdB.
We hope on no mutations this time
-> Q5 PCR: Control of Knock-In
-> Control on gel: positive
Second transformation in competent strain 8, using electroporation
(Time Constant for colony 1: 1.5, for colony 2: 1.4)
-> Plate on chloramphenicol plate and grow overnight at 37°C: positive
Gibson Assembly
-> melting oligos first: 5 minutes at 95°C + infinite at 50°C
-> 4233bp-fragment and oligos at 50°C (1 h)
Transformation in competent strain 8, using electroporation (Time Constant for colony 1: 1.5, for colony 2: 1.4)
-> Plate on chloramphenicol plate and grow overnight at 37°C: positive
Close
Week 3: september 16 - 22
Experiment 2
Plasmid pSB3T5
Transformation in competent strain 8, using electroporation  (Time Constant: 1.4)
-> Plate on tetracyclin plate and grow overnight at 37°C: negative
Transformation in competent strain 8, using electroporation
(Time Constant: pSB3T5+T7-ccdB of p10: 4.1,   pSB3T5+T7-ccdB of p20: 2.2)
-> Plate on tetracyclin plate and grow overnight at 37°C: positive, but also red colonies
Colony PCR (Crimson):
We tested 15 colonies (using primers MDM0606 and MDM0607) : negative (expected fragment: 2486 bp)
Plasmid pSB4A5
Transformation in competent strain 8, using electroporation  (Time Constant: 1.5)
-> Plate on ampicillin plate and grow overnight at 37°C: negative
Transformation in competent strain 8, using electroporation
(Time Constant: pSB3T5+T7-ccdB of p10: 2.7,  pSB3T5+T7-ccdB of p20: 2.6)
-> Plate on ampicillin plate and grow overnight at 37°C: positive, but also red colonies
Colony PCR (Crimson):
We tested the few colonies, we had (using primers MDM0606 and MDM0607) : negative (expected fragment: 2486 bp)
Plasmid pSB6A1
Colony PCR (Crimson):
We tested 24 colonies of pSB6A1(using primers MDM0606 and MDM0607) and  we found many good colonies
(expected length is 4387 bp)
-> We chose 4 colonies to sequenate
->  Inoculate these 4 good colonies
Plasmid purification of the 4 colonies, using Qiagen Qiaprep Spin minikit: nanodrop:
-> pSB6A1 strain 4 : 399.4 ng/µl
-> pSB6A1 strain 13: 412.0 ng/µl
-> pSB6A1 strain 16: 365.5 ng/µl
-> pSB6A1 strain 19: 420.3 ng/µl
Sequencing shows in 2 colonies other mutations
and the other 2 colonies shows the right sequence
-> send this 2 good strains to iGEM (BioBrick)
Experiment 3
Transformation
Experiment 4
CIChE with strains 8 + p5/p10/p20:
-> STEP: 0.1mM - 1.0mM IPTG
-> JUMP: 0.5mM -  3mM IPTG
CIChE with strains 5 + p5/p10/p20:
-> STEP: 0.01mM - 0.5mM IPTG
-> JUMP: 0.5mM - 1.0mM IPTG
CIChE with strains 8 + pSB6A1 (New plasmids from colony 4 & 16 (exp 2)):
-> STEP: 0mM - 0.2mM IPTG
-> JUMP: 0.5mM - 5.0mM IPTG
Experiment 6
Colony PCR (Crimson): We tested 24 colonies (using primers MDM0606 and MDM0607) and we found many good colonies
(expected length is 2520 bp)
-> We chose 4 colonies to sequenate
->  Inoculate these 4 good colonies
Plasmid purification of the 4 colonies, using Qiagen Qiaprep Spin minikit: nanodrop:
-> pSB1C3 strain 10: 181.1 ng/µl
-> pSB1C3 strain 19: 169.1 ng/µl
-> pSB1C3 strain 20: 184.8 ng/µl
-> pSB1C3 strain 24: 184.5 ng/µl
Sequencing shows the same mutation at the end codon again
Close
Week 4: september 23 - 29
Experiment 4
CIChE with strains 5 + p5/p10/P20:
-> STEP: 1.0mM - 4.0mM IPTG
CIChE with strain 8 + pSB6A1:
-> STEP: 0.2mM - 2.0mM IPTG
-> JUMP: 5mM - 120mM IPTG
Experiment 5
MTP experiment: version 1 (using precultures)
-> Plate1: 8 + p5;STEP; 0mM Day1 - 0.5mM Day2
-> Plate2: 8 + p10;STEP; 0mM Day1 - 0.5mM Day2
-> Plate3: 8 + p20;STEP; 0mM Day1 - 0.5mM Day2
-> Plate4: 8 + pSB6A1-16;STEP; 0mM Day1 - 0.05mM Day2
-> Plate5: 8 + p5;JUMP; 0mM Day1 - 2mM Day2
-> Plate6: 8 + p10;JUMP; 0mM Day1 - 2mM Day2
-> Plate7: 8 + p20;JUMP; 0mM Day1 - 2mM Day2
Measure the OD and the GFP of all previous plates (using FLUOstar)
MTP experiment: Test version 2
(to know how long they have to grow until they reach a sufficient OD: 4 - 5 hours)
MTP experiment: version 2
-> Plate1: 8 + p5;STEP; 0mM Day1 - 0.5mM Day2
-> Plate2: 8 + p10;STEP; 0mM Day1 - 0.5mM Day2
-> Plate3: 8 + p20;STEP; 0mM Day1 - 0.5mM Day2
-> Plate4: 8 + pSB6A1-16;STEP; 0mM Day1 - 0.05mM Day2
-> Plate5: 8 + p5;JUMP; 0mM Day1 - 2mM Day2
-> Plate6: 8 + p10;JUMP; 0mM Day1 - 2mM Day2
-> Plate7: 8 + p20;JUMP; 0mM Day1 - 2mM Day2
-> Plate8: 8 + pSB6A1-16;STEP; 0.1mM Day1 - 0.2mM Day1 & JUMP;0mM Day1 - 5mM Day1
-> Plate9: 8 + pSB6A1-4;STEP; 0mM Day1 - 0.5mM Day2
-> Plate10: 8 + pSB6A1-4;STEP; 1.0mM Day1 - 2.0mM Day1 & JUMP; 0mM Day1 - 1.5mM Day1
-> Plate11: 8 + pSB6A1-4;JUMP; 1.5mM Day2 - 30.0mM Day2
-> Plate12: 8 + pSB6A1-4;JUMP; 60.0mM Day2 - 120.0mM Day1 & 5 + p5;STEP;0mM Day1 - 0.2mM Day1
-> Plate13: 5 + p5;STEP; 0.2mM Day2 - 3.0mM Day2
-> Plate14: 5 + p5;STEP; 3.5mM Day1 & JUMP;0mM Day1- 1mM Day1
& 5 + p10;STEP;0mM Day1 - 0,01mM Day1
-> Plate15: 5 + p10;STEP; 0.01mM Day2 - 1.0mM Day2
-> Plate16: 5 + p10;STEP; 2mM Day1 - 3.5mM Day1 & JUMP;0mM Day1 - 0.1mM Day1
-> Plate17: 5 + p10;JUMP; 0.1mM Day2 - 1.0mM Day1 & 5 + p20;STEP; 0mM Day1 - 0.1mM Day2
-> Plate18: 5 + p20;STEP; 0.2mM Day1 - 3.0mM Day1
-> Plate19: 5 + p20;STEP; 3.0mM Day1 - 3.5mM Day1 & JUMP;0mM Day1 - 1mM day1
Measure the OD and the GFP of all plates (using FLUOstar)
Close
October
Week 1: september 30 - october 4
Experiment 4
CIChE with strains 5 + p5/p10/P20:
-> STEP: 4.0mM - 5.0mM IPTG
CIChE with strain 8 + pSB6A1:
-> STEP: 2.0mM - 3.0mM IPTG
-> JUMP: 120.0mM - 240mM IPTG
Experiment 5
MTP experiment: version 2
-> Plate20: 8 + pSB6A1-4;STEP; 2.5mM Day2 - 3.0mM Day2 & JUMP; 120mM Day2 - 240mM Day2
-> Plate21: 5 + p5;STEP; 3.5mM Day2 - 5mM Day2 & 5 + p10;STEP; 3.5mM Day2 - 5mM Day2
-> Plate22: 5 + p20;STEP; 3.5mM Day2 - 5mM Day2 & 5 + p5;STEP; 1mM Day1 and 3mM Day1
& 5 + p10;STEP; 1mM Day1 and 3mM Day1 & 5 + p20;STEP; 1mM Day1 and 3mM Day1
Measure the OD and the GFP of the 3 plates (using FLUOstar)
Close
Project: NetSite...JavaScript Countdown Clock
Days:
Hours:
Mins:
Secs:
Until Jamboree Lyon
Tweets van @iGEM_UGent
We thank following sponsors for their support
Retrieved from " http://2013.igem.org/Team:UGent/Labjournal "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
