<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UGent_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UGent/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UGent/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UGent/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UGent/Results&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UGent/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UGent/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><TABLE cellspacing="0" cellpadding="0" border="0"><TBODY><TR align="center" bgcolor="#F9BD05"><TD colspan="2" width="1200px"><UL id="sddm"><LI><A href="https://2013.igem.org/Team:UGent">Home</A></LI><LI><A href="https://2013.igem.org/Team:UGent/Team" onmouseover="mopen('m2')" onmouseout="mclosetime()">Team</A><DIV id="m2" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2013.igem.org/Team:UGent/Undergrads">Undergrads</A><A href="https://2013.igem.org/Team:UGent/Instructors">Instructors</A><A href="https://igem.org/Team.cgi">Official team page</A></DIV></LI><LI><A href="https://2013.igem.org/Team:UGent/Project" onmouseover="mopen('m3')" onmouseout="mclosetime()">Project</A><DIV id="m3" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2013.igem.org/Team:UGent/LiteratureStudy">Literature Study</A><A href="https://2013.igem.org/Team:UGent/Parts">Parts</A><A href="https://2013.igem.org/Team:UGent/Results">Results</A></DIV></LI><LI><A href="https://2013.igem.org/Team:UGent/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:UGent/Labwork" onmouseover="mopen('m4')" onmouseout="mclosetime()">Labwork</A><DIV id="m4" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2013.igem.org/Team:UGent/Experiments">Experiments</A><A href="https://2013.igem.org/Team:UGent/CloneManager">CloneManager</A><A href="https://2013.igem.org/Team:UGent/Labjournal">Labjournal</A></DIV></LI><LI><A href="https://2013.igem.org/Team:UGent/SocialAspects" onmouseover="mopen('m5')" onmouseout="mclosetime()">Social Aspects</A><DIV id="m5" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2013.igem.org/Team:UGent/HumanOutreach">Human Outreach</A><A href="https://2013.igem.org/Team:UGent/Ethics">Ethics</A><A href="https://2013.igem.org/Team:UGent/Survey">Survey</A><A href="https://2013.igem.org/Team:UGent/Press">Press</A></DIV></LI><LI><A href="https://2013.igem.org/Team:UGent/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:UGent/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:UGent/Attributions">Attributions</A></LI></UL></TD></TR><TR><TD valign="top"><H1> Strains with <I>ccdA-gfp</I> construct </H1><P>
The following strain containing the construct to be duplicated (<I>ccdA-gfp</I>) was constructed (KI experiment 1):
<CENTER><B><I>E. coli</I> MG1655 ΔendA DE3 ccdA-Pmb1GFP-CmFRT</B></CENTER></P></TD></TR><TR><TD><UL><LI> Colony PCR on 48 colonies using crimson taq polymerase with two primer pairs: MDM0141/MDM0010 and MDM0046/MDM123. Expected fragments: 550bp &amp; 2450 bp. Analytic gel: <B> 4 positives: colonies 26, 28, 35 and 36 </B></LI></UL></TD></TR><TR><TD><UL><LI> Colony PCR on 4 positive and 4 negative colonies using Emerald polymerase with out primers: MDM0046/MDM0010. Expected fragment: ca. 5500 bp. Analytic gel: <B> 1 positive: colonie 35 </B>.</LI></UL></TD></TR><TR><TD><UL><LI> After inoculation of c35 on cm plate: Colony PCR on 8 kolonies using emerald polymerase with out primers: MDM0046/MDM0010. Expected fragment: ca. 5500 bp. Analytic gel: <B> positive: 5 &amp; 8 </B>.</LI></UL></TD></TR></TBODY></TABLE><P>We have two versions of this strain: 5 and 8. The two strains were used in our further experiments as to test possible differences between the two versions.
</P><H1> Plasmids containing T7-<I>ccdB</I></H1><P>
We constructed plasmid pSB6A1-T7<I>ccdB</I> for use in CIChE and pSB1C3-T7<I>ccdB</I> to submit in the biobrick-registry. Through these plasmids, pressure can be put on the CIChE strains simply by adding IPTG. We started with a PCR on p20SpFRT-T7<I>ccdB</I> with primers MDM0586 and MDM0587. We then used restriction enzymes XbaI and PstI to cut the obtained T7-<I>ccdB</I> and the plasmid backbones (pSB6A1 and pSB1C3). After ligating these parts, the DNA was transformed into <I>E. coli</I> Top10 subcloning cells, using heat shock. This experiment delivered pSB6A1 with a deletion of cytosine at position 5929 (for pSB6A1), but failed multiple times for pSB1C3. We then tried to create the plasmid using the Gibson Assembly technique. This was done using T7-<I>ccdB</I> and backbone constructed with specific primers. The result of this transformation (using electroporation) was pSB1C3 with a deletion of adenosine at position 3988 (for pSB1C3). A backmutation was performed on both plasmids using specially constructed primers. This process was <B>succesfull for pSB6A1-T7<I>ccdB</I></B>, but not for pSB1C3-T7<I>ccdB</I>. 




We also purified plasmids p5SpFRT-T7<I>ccdB</I>, p10SpFRT-T7<I>ccdB</I>, p20SpFRT-T7<I>ccdB</I> from strains we obtained from InBio. These plasmids have different copy numbers and they too can be used in chromosomal evolution.

Having plasmids with different copy numbers at our disposal, we can test CIChE with different degrees of toxin-pressure.
</P><H1> CIChE Strains</H1><P>
The plasmids and strains mentioned above were used in transformations (experiment 3) to obtain strains containing all elements necessary to perform chromosomal evolution. The following strains were constructed:<TABLE><TBODY><TR><TD width="270"><UL><LI>8+pSB6A1-T7<I>ccdB</I></LI></UL></TD><TD><B>Colony PCR</B> on 8 + pSB6A1 using emerald polymerase with primerpairs MDM0046/MDM010 to check KI. Expected fragment: 5500bp. Analytic gel: <B> positive </B><B>Colony PCR</B> on 8 + pSB6A1 using Q5 polymerase to test for pSB6A1 <B>after backmutation</B> with primers MDM0606 and MDM0607). Expected fragment: 4387 bp. Analytic gel: <B> positive </B></TD></TR></TBODY></TABLE><UL><LI>8+p5SpFRT-T7<I>ccdB</I></LI><LI>8+p10SpFRT-T7<I>ccdB</I></LI><LI>8+p20SpFRT-T7<I>ccdB</I></LI><LI>5+p5SpFRT-T7<I>ccdB</I></LI><LI>5+p10SpFRT-T7<I>ccdB</I></LI><LI>5+p20SpFRT-T7<I>ccdB</I></LI></UL><B>Colony PCR</B> on 5 and 8 + pSB6A1/p5/p10/p20 using emerald polymerase with primerpairs MDM0046/MDM010 to check KI. Expected fragment: 5500bp. Analytic gel: <B> positive </B><B>Colony PCR</B> on 5 and 8 + pSB6A1/p5/p10/p20 using emerald polymerase with primerpairs MDM0096/CGL0019 for pSB6A1 and MDM0039/MDM0060 for p5/p10/p20. Expected fragment: 437bp &amp; 910 bp. Analytic gel: <B> positive for p5/p10/p20</B></P><H1> UV Test </H1><P> Phage transduction was used to perform the deletion of the <I>recA</I> gene after chromosomal evolution. To check whether the gene was successfully knocked out, a UV test was developed. The UV light puts a certain amount of stress on the bacterial cells. In respons to this stress, an SOS pathway is triggered in which <I>recA</I> plays an important role. Cells in which the <I>recA</I> gene is deleted will not survive on the UV-treated plate. Colonies in which <I>recA</I> was successfully deleted still can be found on a non-treated backup plate. 
<B>RESULT: </B><TABLE><TBODY><TR><TD><UL><LI><I>recA</I> positive strain grows on both parts (UV and non-UV)</LI><LI><I>recA</I> negative strain</LI><UL class="a"><LI>10’’ – no visible differences </LI><LI>15’’ – still some colonies on UV-part of the plate</LI><LI>20’’ – one colony visible on UV-part of the plate</LI><LI>30'' – no growth on UV-part of the plate</LI></UL></UL></TD></TR></TBODY></TABLE></P><P><B>CONCLUSION</B>
With a 30’’ UV exposure, the cells in which <I>recA</I> is successfully deleted will not survive on the UV-treated plate. </P><H1> CIChE </H1><P>
The practical execution of CIChE was as follows: a strain was inoculated with the right concentration of IPTG (day 1 = D1), these strains were grown overnight at 37°C. The next day the strains were inoculated with the same IPTG concentration (day 2 = D2) and with an increased IPTG level (D1). IPTG concentration was raised with a small “step” or with a larger “jump”. These strains were also grown overnight at 37°C. This was repeated over the next days. Because the outcome was difficult to predict, a lot of different IPTG concentrations and ranges were tested.

To test all of our CIChE strains we used 22 microtiter plates (the distribution of the different strains in the plates is documented in <A href="https://static.igem.org/mediawiki/2013/2/22/UGent_2013_Plates_complete.pdf" target="_blank">this pdf file</A>). To spare some time we used the second, faster protocol (see <A href="https://2013.igem.org/Team:UGent/Experiments" target="_blank">experiment 5</A>). Bacteria were grown in microtiter plates during 4 hours at 37°C on shakers. Afterwards the OD (optical density) and fluorescence intensity (F) were measured inside the FLUOstar. Strains with more copies of the <I>gfp</I> gene should result in higher fluorescence intensity, indicating a higher construct copy number. Because the fluorescence not only depends on the gene copy number, but also on the OD, we divided the fluorescence by the OD (F/OD) to analyse our results.

The plates also contained blank wells and reference colonies. These were necessary for two different reasons. 
</P><UL><LI>We subtracted the OD of all the wells with the average OD of the blank and did the same for the fluorescence intensity. This was done to standardize the measurements. </LI><LI>Because we used many different plates (and thus some strains were spread over two or more plates), we added a reference strain (strain 8) to all the plates. This strain contains one copy of our construct and thus should give the same F/OD in every well. We used various statistical tests to check if there was a significant difference in average F/OD between the plates of one strain. If these tests resulted in a significant difference of the 8 strain, we could not compare the measurements. Luckily this only happened for a few strains. The extensive execution of these tests can be checked in <A href="https://static.igem.org/mediawiki/2013/d/db/UGent_2013_statistics.pdf" target="_blank">this pdf file</A>.</LI></UL><P>
Thereafter we calculated the F/OD for every well. If the data showed some abnormalities, we tested them for outliers. If the values of the outliers were significantly different than the values of the rest of the data, we excluded these observations and did not use them in our further analysis.

Finally we plotted the F/OD in function of the IPTG concentration. The results can be seen in figure 1-9 (you can click on the graphs to enlarge).

In every graph the average F/OD of strain 8 is added as reference. The average is calculated using the results of 8 corresponding to the correct plates. If our model for CIChE works, our strains (containing more gfp copies) should have an average F/OD higher than the one of strain 8. We also expected that the F/OD would rise with increasing IPTG concentrations. Looking at the graphs we see that most of our strains result in an average F/OD lower than or about the same as the average result of strain 8. This is not at all what we expected/hoped. <B>According to the obtained data, we must conclude that no duplication of the construct occurred, hence no chromosomal evolution was achieved. However, we have a pretty good idea what lead to these results. Our hypothesis can be read below.</B></P><H1>CIChE and its stumbling blocks: our hypothesis</H1><P>For the construction of our plasmid we chose to work with a T7 promoter for the <I>ccdB</I> gene as to avoid leaky expression. Cloning was performed in Top10 cells which do not contain T7 RNA polymerase. Taking into account the troubles we had with the construction of the plasmids with the T7-<I>ccdB</I> insert, we can almost certainly conclude that despite this the cells did experience hindering of the CcdB toxin. Often we obtained no colonies after transformation and if we did, there were mutations in the toxic part of the <I>ccdB</I> gene (see further). We presume that this can only be caused if the <I>ccdB</I> was transcribed by other polymerases who are not specific for the T7 promoter. This probably only occurs at very low frequencies, but as in these strains there is no  CcdA present at all even the lowest amount of working CcdB can be lethal for these cells.
We performed CIChE with the plasmid we constructed ourselves (pSB6A1-T7<I>ccdB</I>) as well as with the original plasmids from which we constructed this part: p5SpFRT-T7<I>ccdB</I>, p10SpFRT-T7<I>ccdB</I> and p20SpFRT-T7-<I>ccdB</I>. According to the GFP measurements we performed, none of these resulted in duplication of the <I>ccdA-gfp</I> construct in the genome. We presume that this is caused by the malfunction of CcdB due to mutations. But how is it possible that there’s so many of these mutations?
It is known from previous research that the part essential for CcdB’s toxic activity is situated at the last three amino acids (Trp-Gly-Ile). Interaction between CcdB and the DNA gyrase A subunit (GyrA) takes place at Trp99 from CcdB and Arg462 of GyrA59 (the 59 kDa amino-terminal breaking-rejoining domain of GyrA). Random mutations in the final part of the <I>ccdB</I> gene affecting this toxic site will consequently provide these cells with a great growth advantage over cells with intact <I>ccdB</I> genes. We assume this is a possible reason why mutation frequency of the <I>ccdB </I>gene seems to be so high in our project.
Furthermore, in the absence of its antidote CcdA, CcdB causes reduced DNA synthesis, activation of the SOS pathway, cell filamentation and eventually cell death. One of the aspects of this SOS pathway is the enhancing of capacity of mutagenesis, such as point and frameshift mutations. So if for example there is leaky expression of the promoter and CcdB is being formed without a sufficient amount of CcdA present, SOS pathway will be activated causing higher mutation frequencies. We postulate that these mutations can also affect the ccdB gene itself and thus restoring ‘normal’ cell life. In combination with what we discussed in the previous paragraph this could explain the amount of mutations we had to deal with over the course of our project.
We chose some strains at random during the CIChE experiment (8 + pSB6A1-T7<I>ccdB</I>, 2 mM IPTG day 2, Step)  and let the plasmid get sequenced.  When we compared the ccdB sequence to the native one we saw our plasmid had a deletion of a cytosine residue at position 297 (out of a total of 306 base pairs). <B>This causes a frameshift, resulting in a prolonged protein</B> of 154 amino acids instead of the native structure with 102 amino acids. <I>(Note: CcdB normally contains 101 amino acids, but the sequence we used has an extra Alanine after the start codon for cloning purposes.)</I> Likewise, all pSB1C3-T7<I>ccdB </I>plasmids we sent for sequencing contained a mutation at the end: all of them had a deletion of the A residu of the stop codon, also leading to a frameshift and prolonging of the protein.
As mentioned before, the toxic part of CcdB is situated at the last three amino acids. None of the sequencing results showed a mutation anywhere in the beginning or middle of the <I>ccdB </I>gene. <B>This shows the mutations are really all linked with the toxicity and cells lacking toxic CcdB easily overrule the cells that do contain toxic CcdB. If experimental conditions are not adapted there’s little chance of success.</B></P><H2>Clone Manager and I-TASSER</H2><P>In order to examine the effect of these mutations on the protein structure, we determined the amino acid sequence of the mutated plasmid 8 + pSB6A1-T7<I>ccdB</I>, 2 mM IPTG day 2, Step using Clone Manager:</P><P>
MAQFKVYTYKRESRYRLFVDVQSDIIDTPGRRMVIPLASARLLSDKVSRELYPVVHIGDESWRMMTTDMASVPVSVI
GEEVADLSHRENDIKNAINLMFGEYNLGEVKRRSADGSVGSPHARVGNCQASNKTKGSVERLGLSFYLLFVGERSPE</P><P>
At the position of the Tryptophan residue responsible for interaction with Arg462 and consequently essential for toxic activity, there is now a Glycine residue. Also the following two amino acids essential for toxicity are mutated. Furthermore chances are that the following part of the protein will sterically hinder interaction with DNA gyrase. <B>As a conclusion this amino acid sequence is a good indication that this mutated CcdB protein won’t be able to execute its toxic function.</B>
To further investigate the consequences of the frameshift we used I-TASSER to predict the structure of this protein, resulting in following top 5 of predicted models:</P><TABLE><TBODY><TR><TD><CENTER>C-score = -2.27 </CENTER></TD><TD><CENTER>C-score = -2.76</CENTER></TD><TD><CENTER>C-score = -2.75</CENTER></TD><TD><CENTER>C-score = -2.69</CENTER></TD><TD><CENTER>C-score = -2.93</CENTER></TD></TR></TBODY></TABLE><P><I>Note: C-score is typically in the range of [-5,2], where a C-score of higher value signifies a model with a high confidence and vice-versa.</I>
Whereas native CcdB has an α helix at its C-terminal end, two of these end with a β strand and three with an undetermined structure.
Five of the top 10 templates used by I-TASSER are CcdB from E. coli and four of them are CcdB from <I>V. fischeri</I>. This shows that to a great extent there still is resemblance to the native <I>ccdB</I> sequence. The tenth template used by I-TASSER is the A subunit of DNA gyrase, the target of CcdB. 
When comparing the proposed structure of our mutated plasmid to the native CcdB structure (pdb entry 3VUB) using I-TASSER this is what we get, with <B>in red the native CcdB structure and in blue our mutated protein</B>. The red, native, part overlaps practically completely with the blue one. The rest of the blue structure is the extension caused by the frameshift. </P><H1>Further research perspectives </H1><P><B> Assemble T7-<I>ccdB</I> plasmid inside CcdB-survival strains </B>
When assembling the plasmid containing T7-<I>ccdB</I> in cells resistant to CcdB, mutations as a result of stress are less likely to occur at this stage. We experienced difficulties cloning the T7-<I>ccdB</I> construct in our plasmids probably because of leaky expression of the T7-promoter.
<B>Cloning T7-<I>ccdB</I> plasmid in <I>ccdA</I>-strain</B>
If the plasmids are cloned in the strains that already contain the <I>ccdA</I> construct, leaky expression of <I>ccdB</I> might be neutralized by the CcdA already present. This would lower the toxin pressure, making mutations less likely to occur.
<B>Adding multiple <I>ccdA</I> copies to the <I>ccdA-gfp</I> construct</B>
If multiple <I>ccdA</I> copies were incorporated in the CIChE-construct, cells would be able to handle higher amounts of CcdB toxin. They would not feel pressured as much to mutate.
<B>Knock out of alternative polymerases</B><I>E. coli</I> possesses five DNA polymerases. Polymerase I and III see to respectively the maturation of Okazaki fragments and replication under vegetative conditions. Polymerase II, IV and V are the so called alternative polymerases or error prone DNA polymerases. They can be induced under a variety of environmental stress factors and mediate stress-induced mutagenesis. They also mediate translesion synthesis and enable efficient replication past DNA damage that would otherwise halt replication. Unfortunately this happens with significantly reduced fidelity, resulting in a higher mutation rate.  A possible solution to reduce this rate could be the knock out (and maybe the knock in of high fidelity polymerases) of these low fidelity polymerases.<B>Other alternatives</B>
In our literature study we discussed some more alternatives, you can read about them <A href="https://static.igem.org/mediawiki/2013/d/dc/UGent_2013_Bachelorproef.pdf" target="_blank">here</A> (page 6-20).

</P><H1>Sources</H1> 
Dao-Thi, M.H. et al. Molecular basis of gyrase poisoning by the addiction toxin CcdB. <I>Journal of Molecular Biology</I><B>348</B>, 1091-1102 (2005).
Couturier, M., Bahassi, E. &amp; Van Melderen, L. Bacterial death by DNA gyrase poisoning. <I>Trends in Microbiology</I><B>6</B>, 269-275 (1998).
Corzett, C.H., Goodman, M.F. &amp; Finkel, S.E. Competitive Fitness During Feast and Famine: How SOS DNA Polymerases Influence Physiology and Evolution in <I>Escherichia coli</I>. <I>Genetics</I><B>194</B>, 409-420 (2013).





<P><TITLE>Project: NetSite...JavaScript Countdown Clock</TITLE></P><FORM name="clock"><TABLE border="1" frame="void" rules="none"><TBODY><TR><TD align="CENTER" width="31%" bgcolor="#F9BD05"><FONT color="#FFFFFF"><B>Days:</B></FONT></TD><TD align="CENTER" width="23%" bgcolor="#F9BD05"><FONT color="#FFFFFF"><B>Hours:</B></FONT></TD><TD align="CENTER" width="23%" bgcolor="#F9BD05"><FONT color="#FFFFFF"><B>Mins:</B></FONT></TD><TD align="CENTER" width="23%" bgcolor="#F9BD05"><FONT color="#FFFFFF"><B>Secs:</B></FONT></TD></TR><TR><TD colspan="4" bgcolor="#F9BD05"><CENTER><FONT face="Verdana, Arial,sans-serif" size="+1.5" color="#FFFFFF">Until Jamboree Lyon</FONT></CENTER></TD></TR></TBODY></TABLE></FORM><A class="twitter-timeline" href="https://twitter.com/iGEM_UGent" data-widget-id="373064939419607041">Tweets van @iGEM_UGent</A><TABLE border="0" cellspacing="0" cellpadding="0" width="70%" align="center"><DIV align="right"><H1>We thank following sponsors for their support</H1></DIV></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UGent/Results">http://2013.igem.org/Team:UGent/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UGent/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UGent/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UGent/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UGent/Results&amp;oldid=319233" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>