<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UGent_Labjournal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UGent/Labjournal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UGent/Labjournal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UGent/Labjournal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UGent/Labjournal&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UGent/Labjournal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UGent/Labjournal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><TABLE cellspacing="0" cellpadding="0" border="0"><TBODY><TR align="center" bgcolor="#F9BD05"><TD colspan="2" width="1200px"><UL id="sddm"><LI><A href="https://2013.igem.org/Team:UGent">Home</A></LI><LI><A href="https://2013.igem.org/Team:UGent/Team" onmouseover="mopen('m2')" onmouseout="mclosetime()">Team</A><DIV id="m2" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2013.igem.org/Team:UGent/Undergrads">Undergrads</A><A href="https://2013.igem.org/Team:UGent/Instructors">Instructors</A><A href="https://igem.org/Team.cgi">Official team page</A></DIV></LI><LI><A href="https://2013.igem.org/Team:UGent/Project" onmouseover="mopen('m3')" onmouseout="mclosetime()">Project</A><DIV id="m3" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2013.igem.org/Team:UGent/LiteratureStudy">Literature Study</A><A href="https://2013.igem.org/Team:UGent/Parts">Parts</A><A href="https://2013.igem.org/Team:UGent/Results">Results</A></DIV></LI><LI><A href="https://2013.igem.org/Team:UGent/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:UGent/Labwork" onmouseover="mopen('m4')" onmouseout="mclosetime()">Labwork</A><DIV id="m4" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2013.igem.org/Team:UGent/Experiments">Experiments</A><A href="https://2013.igem.org/Team:UGent/CloneManager">CloneManager</A><A href="https://2013.igem.org/Team:UGent/Labjournal">Labjournal</A></DIV></LI><LI><A href="https://2013.igem.org/Team:UGent/SocialAspects" onmouseover="mopen('m5')" onmouseout="mclosetime()">Social Aspects</A><DIV id="m5" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2013.igem.org/Team:UGent/HumanOutreach">Human Outreach</A><A href="https://2013.igem.org/Team:UGent/Ethics">Ethics</A><A href="https://2013.igem.org/Team:UGent/Survey">Survey</A><A href="https://2013.igem.org/Team:UGent/Press">Press</A></DIV></LI><LI><A href="https://2013.igem.org/Team:UGent/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:UGent/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:UGent/Attributions">Attributions</A></LI></UL></TD></TR><TR><TD valign="top"><H1> Journal</H1><H2> July </H2><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
Week 4: july 22 - 28  
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><P align="justify"><UL><LI> Introduction given by our lab instructors</LI><UL class="a"><LI> General techniques: plasmid/PCR purification, inoculation, gel extraction, restriction &amp; ligation. </LI><LI> Safety and waste disposal training  </LI><LI> Introduction to CloneManager </LI></UL><LI> General preparations: sterile mQ, sterile eppendorf </LI></UL></P><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><H2> August </H2><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
Week 1: july 29 - august 4  
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
Week 2: august 5 - 11  
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><P align="justify"><UL><LI> Experiment 1 </LI><UL class="a"><LI><B>Inoculate</B><I> E. coli </I>DH5a + pTGD-<I>ccdA</I>-Pmb1GFP-CmFRT </LI><LI><B>Purify plasmid pTGD-<I>ccdA</I>-Pmb1GFP-CmFRT</B> using Qiagen spin mini kit: Nanodrop -- <B>362.4 ng/µL</B></LI><LI><B>Preparative Restriction Digest (RD)</B> of purified plasmid: BspHI &amp; BstAPI: expected fragments of 5541bp and 903 bp </LI><LI><B>Gel purify RD-fragment of 5541 bp</B> using Qiagen Qiaquick gel extraction kit: Nanodrop -- <B>70.4 ng/µL</B></LI><LI><B>PCR on RD (HR-<I>ccdA</I>-Pmb1GFP-HR)and on plasmid pTGD-<I>ccdA</I>-PMbAFGP-CmFRT  with MDM588 &amp; MDM589</B> -&gt;HiFi PCR using Primestar polymerase. Analytical gel: <B>negative </B> -&gt;PCR using Q5 polymerase. Analytical gel: <B> negative </B> -&gt;PCR using Roche. Analytical gel: <B> negative </B> -&gt;Touchdown PCR using Q5 polymerase. Analytical gel: <B> negative </B></LI><LI><B>Control plasmid pTGD-<I>ccdA</I>-PMbAFGP-CmFRT: RD with AclI.</B> Expected fragments: 758 bp, 1730 bp and 3956 bp. Analytical gel: <B> negative </B> --&gt; <U> Problem with plasmid pTGD-<I>ccdA</I>-PMbAFGP-CmFRT </U></LI><LI><B> Transformation</B>: Knock in with <U>linear back-up</U> DNA by electroporation. Incubation of the transformed cells and transfer the culture on a Cm plate. </LI><LI> Colony PCR on 48 colonies using crimson taq polymerase with two primer pairs: MDM0141/MDM0010 and MDM0046/MDM123. Expected fragments: 550bp &amp; 2450 bp. Analytic gel: <B> 4 positives </B></LI><LI> Colony PCR on 4 positive and 4 negative colonies using crimson taq polymerase with out primers: MDM0046/MDM0010. Expected fragment: ca. 5500 bp. Analytic gel: <B> negative </B>.</LI><LI> 2 Colony PCR on 4 positive and 4 negative colonies using Taq and Phire polymerase with out primers: MDM0046/MDM0010. Expected fragment: ca. 5500 bp. Analytic gel: <B> negative </B>.</LI><LI> Colony PCR on 4 positive and 4 negative colonies using Emerald polymerase with out primers: MDM0046/MDM0010. Expected fragment: ca. 5500 bp. Analytic gel: <B> 1 positive: colonie 35 </B>.
</LI></UL><LI> Experiment 2 </LI><UL class="a"><LI> Inoculate <I>E. coli</I> DH5a; + p5SpFRT-T7<I>ccdB</I> Inoculate <I>E. coli</I> DH5a; + p10SpFRT-T7<I>ccdB</I> Inoculate <I>E. coli</I> DH5a; + p20SpFRT-T7<I>ccdB</I></LI><LI> Purify plasmids using Qiagen spin mini kit: nanodrop  p5SpFRT-T7<I>ccdB</I>: 119.6 ng/µl  p10SpFRT-T7<I>ccdB</I>: 156.3 ng/µl  p20SpFRT-T7<I>ccdB</I>: 392.9 ng/µl </LI><LI><B>CcdB operon</B>: 
 -&gt; HiFi PCR of plasmids p5SpFRT-T7<I>ccdB</I>, p10SpFRT-T7<I>ccdB</I> and p20SpFRT-T7<I>ccdB</I> with MDM0586/MDM0587 to amplify <I>ccdB</I> operon 
 -&gt; Purification of CcdB operon PCR fragment using Qiagen Qiaquick PCR purification kit and checked on analytical gel (expected fragment of 2300 bp): nanodrop
 p5: 174.6 ng/µl
 p10: 24.5 ng/µl (consistent with small band on the analytival gel)
 p20: 104.3 ng/µl


 -&gt; Redo of inoculation <I>E. coli</I> DH5a + p5SpFRT-T7<I>ccdB</I>, <I>E. coli</I> DH5a + p10SpFRT-T7<I>ccdB</I> and <I>E. coli</I> DH5a + p20SpFRT-T7<I>ccdB</I> -&gt; Redo of purification of plasmids p5SpFRT-T7<I>ccdB</I>, p10SpFRT-T7<I>ccdB</I> and p20SpFRT-T7<I>ccdB</I> using Qiagen Spin minikit: nanodrop  p5SpFRT-T7<I>ccdB</I>: 21.1 ng/µl  p10SpFRT-T7<I>ccdB</I>: 25.1 ng/µl  p20SpFRT-T7<I>ccdB</I>: 24.6 ng/µl 
</LI><LI><B>Vectors pSB4A5</B> (plate 5, well 5I), <B>pSB3T5</B> (plate 2, well 8D) and <B>pSB6A1</B> (plate 2, well 2L):
 -&gt; Resuspend plasmids from the iGEM kit
 -&gt; Transform in <I>E. coli </I> Top10 subcloning cells using elektroporation
 -&gt; Plate pSB4A5 and pSB6A1 on ampicillin plate and pSB3T5 on tetracyclin 
and grow overnight at 37°C
 -&gt; Plates with transformants: pSB6A1 lots of colonies, pSB3T5 sufficient colonies and pSB4A5 no colonies 
</LI><LI><B>Vectors pSB4A5 </B>(plate 5, well 5I):
 -&gt; Resuspend plasmids from the iGEM kit
 -&gt; Transform in <I>E. coli</I> Top10 subcloning cells using elektroporation
 -&gt; Plate on ampicillin plate and grow overnight at 37°C (1 plate 150 µl, 1 plate 50 µl)
 -&gt; Plates with transformants: pSB4A5 no colonies</LI><LI>Inoculation colonies of pSB3T5 and pSB6A1 -&gt; at the end of the day replaced from 37°C to 30°C</LI><LI><B>Vector pSB4A5 </B>(plate 5, well 1I and plate 2, well 2J as backup) and <B>pSB6A1 </B>(plate 5, well 1K as backup):
 -&gt; Resuspend plasmid pSB4A5 and pSB6A1 from the iGEM kit 
 -&gt; Transform in <I>E. coli</I> Top10 subcloning cells (heat shock)
 -&gt; Plate transformation on ampicillin plate and grow overnight at 37°C (3 plates: 10<SUP>-2</SUP>,10<SUP>-1</SUP> and 10<SUP>0</SUP>)
 -&gt; Plates with transformants: pSB4A5 (plate 2, well 2J) colonies, pSB4A5 (plate 5, well 1I) no colonies and pSB6A1 (plate 2, 2L) few colonies
</LI><LI> Inoculate pSB3T5 and pSB6A1  again and in warm chamber (37°C). </LI><LI> Inoculation colonies of pSB4A5 (plate 2, well 2J) and pSB6A1 (plate 2, well 2L) (2 times: one for further work and one for cryovials)
</LI><LI> Purification of plasmids pSB3T5, pSB6A1 and pSB4A5 </LI><LI> Generation of <B>restriction</B> digest fragments of T7-<I>ccdB</I> insert (from p5SpFRT-T7<I>ccdB</I> and p20SpFRT-T7<I>ccdB</I>) and vector (pSB3T5 (2x), pSB4A5 (from plate 2, well 2J) and pSB6A1 (1x from plate 5, well 1K and 1x from plate 2, well 2L) with XbaI and PstI-HF (gel purify RD-fragments using Qiagen Qiaquick gel extraction kit (expected fragment of 2157 bp for T7-<I>ccdB</I>, 3229 bp for pSB3T5, 3372 bp for pSB4A5 and 3999 bp for pSB6A1): Nanodrop
p5: 43.7 ng/µl
p20: 24.2 ng/µl
pSB3T5 (inoculation 1): 23.5 ng/µl
pSB3T5 (inoculation 2): 23 ng/µl
pSB4A5: 40.5 ng/µl
pSB6A1 (plate 2, 2L): 40.1 ng/µl
pSB6A1 (plate 5, 1K): 30.4 ng/µl </LI><LI><B>Ligation</B> of T7-<I>ccdB</I> from p5SpFRT-T7<I>ccdB</I> and pSB6A1 (1x from plate 5, well 1K and 1x from plate 2, well 2L), p20SpFRT-T7<I>ccdB</I> and pSB3T5, p20SpFRT-T7<I>ccdB</I> and pSB4A5 (from plate 2, well 2J)  (3(insert)/1(plasmid) ratio) </LI><LI><B>Transformation</B> in <I>E. coli</I> Top10 (heat shock)</LI><LI> Plate transformation on ampicillin (pSB4A5 and pSB6A1) and tetracycline plate (pSB3T5) and grow overnight at 37°C (2 plates: 10<SUP>-2</SUP>, 10<SUP>-1</SUP>)
 -&gt; Plates with transformants: no colonies</LI><LI> New <B>transformation</B> in <I>E. coli</I> Top10 from pSB4A5 and pSB6A1 using heat shock 
 -&gt; Plates with transformants: pSB6A1 2 colonies, pSB4A5 few colonies </LI><LI> New <B>ligation</B> with RD-fragments created before (vectors pSB3T5, pSB4A5, pSB6A1 (plate 5, well 1K) and insert T7-<I>ccdB</I> created in experiment 6), at 22.5°C o/n) </LI><LI><B>CcdB operon</B>:
 -&gt; HiFi PCR of plasmids p5SpFRT-T7<I>ccdB</I>, p10SpFRT-T7<I>ccdB</I> and p20SpFRT-T7<I><I>ccdB</I></I> with MDM0586 &amp; MDM0587 to amplify <I>ccdB</I> operon </LI></UL><LI> Experiment 6 </LI><UL class="a"><LI><B>CcdB operon</B>: 
 -&gt; HiFi PCR of plasmids p20SpFRT-T7<I>ccdB</I> with MDM0586/MDM0587 to amplify <I>ccdB</I> operon
 nanodrop: 301,5 ng/µl </LI><LI><B>Vector pSB1C3</B> (plate 1, well 23O):
 -&gt; Resuspend plasmids from the iGEM kit
 -&gt; Transform pSB1C3 in <I>E. coli</I> Top10 subcloning cells, using heat shock (42°C)
 -&gt; Plate on chloramphenicol plate and grow overnight at 37°C
 -&gt; Inoculate <I>E. coli</I> Top10 + pSB1C3
 -&gt; Purify plasmids using Qiagen Qiaprep Spin minikit: nanodrop: 200,2 ng/µl</LI><LI><B>Restriction</B> of CcdB operon and pSB1C3 with XbaI and PstI
 Preparative gel:   restriction digest of T7-<I>ccdB</I> (expected to be 2044bp) and restriction digest of backbone of pSB1C3 (expected length is 2232 bp)
 -&gt; Gel purify fragments using Qiagen Qiaquick gel extraction kit: nanodrop:
 Restriction digest of T7-<I>ccdB</I>: 16.9 ng/µl
 Restriction digest of pSB1C3: 12.3 ng/µl</LI><LI><B>Ligation</B> of CcdB operon and pSB1C3</LI><LI><B>Transformation</B> in <I> E. coli </I> Top10 subcloning cells, using heat shock (42°C)
 -&gt; Plate on chloramphenicol plate and grow overnight at 37°C: <B>negative</B></LI></UL></UL></P><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
Week 3: august 12 - 18   
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><P align="justify"><UL><LI> Experiment 1 </LI><UL class="a"><LI>Inoculation positive colonie on cm plate. </LI><LI>Colony PCR on 8 kolonies using emerald polymerase with out primers: MDM0046/MDM0010. Expected fragment: ca. 5500 bp. Analytic gel: <B> positive: 5 &amp; 8 </B>.</LI></UL><LI> Experiment 2 </LI><UL class="a"><LI> PCR purification of <B> CcdB operon</B>: nanodrop
p5: 54.8 ng/µl
p10: 65.8 ng/µl
p20: 21.1 ng/µl</LI><LI><B>cPCR</B> with Taq polymerase of colonies pSB6A1 (primers: MDM0096 and CLG0019) and pSB4A5 (primers: MDM0095 and CLG0019): <B>negative</B></LI><LI> New <B>transformation</B> with pSB3T5, pSB4A5 and pSB6A1 using heat shock in <I>E. coli</I><B>DH5a</B> -&gt; Plate on normal plates and glucose plates (100 ml 2% glucose, to avoid leaky expressing of <I>ccdB</I>)
 -&gt; Plates with transformants: pSB6A1 some colonies, pSB4A5 some colonies and pSB3T5 no colonies </LI><LI>Gel elektroforesis on RD-fragments from plasmids: show expected fragments</LI><LI><B>cPCR</B> on the pSB4A5 (primers: MDM0095 and CLG0019) and pSB6A1 (primers: MDM0096 and CLG0019) colonies with Taq polymerase: one <B>positive</B> colony of pSB6A1-T7<I>ccdB</I> (fragment of 437 bp in lane 14) </LI><LI><B>inoculation</B> of positive pSB6A1-T7<I>ccdB</I> colony from back up plate (3x: one for sequencing, 1 for cryovial and 1 for experiment 3)</LI><LI>Plate <B>transformation mixtures</B> again on glucose plates and grow overnight at 30°C
-&gt;Plates with transformants: pSB4A5 colonies, pSB3T5 no colonies</LI><LI><B>cPCR</B> on pSB4A5 colonies (primers: MDM0095 and CLG0019) with Taq polymerase: <B>negative</B></LI><LI>New <B>ligation</B> with rest of RD-fragments of plasmids pSB3T5 (2 times) and pSB4A5 and CcdB from experiment 6 and <B>transformation</B> in DH5a using heat shock</LI><LI>Due to lack of success: <B>start from the beginning</B>.</LI><LI><B>Inoculation</B> of pSB3T5, pSB4A5, p5SpFRT-T7<I>ccdB</I>, p10SpFRT-T7<I>ccdB</I> and p10SpFRT-T7<I>ccdB</I> from cryovials. 
</LI><LI>Also resuspend pSB3T5 (plate 5, well 7C) from iGEM kit, <B>transform</B> in <I>E. coli</I> DH5a using heat shock, plate on tetracycline plates and incubate overnight at 37°C 
 -&gt; no colonies
 -&gt; transform again in DH5a using heat shock
 -&gt; again no colonies</LI><LI>Purification of plasmids using Qiagen spin minikit: concentrations deduced from gel
p5SpFRT-T7<I>ccdB</I>: +/- 40 ng/µl
p10SpFRT-T7<I>ccdB</I>: +/- 40 ng/µl
p20SpFRT-T7<I>ccdB</I>: +/- 40 ng/µl
pSB3T4: +/- 250 ng/µl
pSB4A5 (1): +/- 400 ng/µl
pSB4A5 (2): +/- 350 ng/µl </LI><LI>PCR on <B>CcdB operon</B> using Q5 polymerase (primers: MDM0586_Fw-Trc-<I>ccdB</I>-G00000 and MDM0587_Rv-Trc-<I>ccdB</I>-G00001) </LI><LI><B>cPCR</B> on pSB3T5 colonies from old plates using Taq polymerase (primers: MDM0602 and CLG0019, expected fragment of 503 bp): <B>negative</B></LI><LI><B>Restriction</B> of ccdB operon, pSB3T5 and pSB4A5 using PstI-HF and XbaI (gel purify RD-fragments using Qiagen Qiaquick gel extraction kit (expected fragment of 2157 bp for T7-<I>ccdB</I>, 3229 bp for pSB3T5 and 3372 bp for pSB4A5): concentrations deduced from gel</LI><LI><B>Ligation</B> of pSB3T5 and T7-<I>ccdB</I> (from p5SpFRT-T7<I>ccdB</I>), pSB4A5 and T7-<I>ccdB</I> (from p5SpFRT-T7<I>ccdB</I>) and pSB4A5 and T7-<I>ccdB</I> (from p20SpFRT-T7<I>ccdB</I>) with T4 DNA ligase at 16°C overnight</LI><LI><B>Transformation</B> of ligation mixtures in DH5a using heat shock
-&gt; no colonies</LI></UL><LI> Experiment 3 </LI><UL class="a"><LI> Inoculation KI-strain 5 &amp; 8 from experiment 1 </LI><LI><B> Transformation of pSB6A1 - T7-<I>ccdB</I></B> in KI-strains 5 and 8 </LI><LI><B> Colony PCR </B>on kolonies from transformed strains using emerald polymerase with out primers to check KI: MDM0046/MDM0010. Expected fragment: ca. 5500 bp. Analytic gel: <B> positive </B></LI><LI><B> Colony PCR </B>on kolonies from transformed strains using emerald polymerase with MDM0096 &amp; CLG0019 to check if plasmid is present. Expected fragment: 437 bp. Analytic gel: <B> positive </B></LI></UL><LI> Experiment 6 </LI><UL class="a"><LI> New <B>transformation</B> in <I> E. coli </I> Top10 subcloning cells, using heat shock (42°C)
 -&gt; Plate on chloramphenicol plate and grow overnight at 37°C: <B>negative</B></LI><LI><B>CcdB operon</B>: HiFi PCR of plasmids p20SpFRT-T7<I>ccdB</I> with MDM0586/MDM0587 to amplify CcdB operon</LI><LI> New <B>restriction</B> of CcdB operon and pSB1C3 with Xba
I and PstI
 -&gt; Gel purify fragments using Qiagen Qiaquick gel extraction kit: nanodrop:
 Restriction Digest of T7-<I>ccdB</I>: 16.6 ng/µl
 Restriction Digest of pSB1C3: 17.7 ng/µl</LI><LI><B>Ligation</B> of CcdB operon and pSB1C3: overnight at 16°C</LI><LI><B>Transformation</B> in <I>E. coli</I> DH5a, using heat shock (42°C)
 -&gt; Plate on chloramphenicol + glucose plate and grow overnight at 30°C: <B>negative</B></LI><LI><B>Restriction</B> of CcdB operon and pSB1C3 with XbaI and PstI
 Because a preparative gel purification cause low concentrations, we will use DpnI
 -&gt; Purification by using minElute reaction cleanup kit</LI><LI> Control of restriction fragments: <B>positive</B>, so we hope that ligation will succeed</LI><LI><B>Ligation</B> of CcdB operon and pSB1C3: overnight at 16°C</LI><LI><B>Transformation</B> in <I>E. coli</I> DH5a, using heat shock (42°C) 
 -&gt; Plate on chloramphenicol plate and grow overnight at 37°C: <B>negative</B> (it does not work, maby our transformation is not succeed) 
</LI></UL></UL></P><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
Week 4: august 19 - 25   
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><P align="justify"><UL><LI> Experiment 2 </LI><UL class="a"><LI>pSB6A1-T7<I>ccdB</I> sent for sequencing with primers MDM0096 (A471969), MDM0060 (A471682) and MDM0039 (A471681)
-&gt; <B>correct sequence</B></LI><LI><B>Transform</B> ligation mixtures again in DH5a, using heat shock (42°C),
-&gt; no colonies</LI><LI><B>PCR on CcdB operon</B> with Q5 polymerase (primers: MDM0586_Fw-Trc-<I>ccdB</I>-G00000 &amp; MDM0587_Rv-Trc-<I>ccdB</I>-G00001) and PCR purification using Qiagen Qiaquick PCR purification kit: nanodrop
p5: 15.2 ng/µl
p10: 18.6 ng/µl
p20: 43.5 ng/µl </LI><LI>Inoculation of pSB3T5 and pSB4A5 from cryovials and plasmid purification using Qiagen Spin minikit: nanodrop
pSB3T5: 75.7 ng/µl
pSB4A5: 190.0 ng/µl </LI><LI><B>Restriction</B> of CcdB operon, pSB3T5, pSB4A5 and the pSB6A1-T7<I>ccdB</I> transformant with PstI-HF and XbaI (gel purify restriction digest-fragments using Qiagen Qiaquick gel extraction kit (expected fragment of 2157 bp for T7-<I>ccdB</I>, 3229 bp for pSB3T5 and 3372 bp for pSB4A5)</LI><LI><B>Ligation</B></LI><LI><B>Transformation</B> in <I>E. coli</I> DH5a 
-&gt; no colonies
-&gt; plate transformants on plates with a lower antibiotic concentration (50% lower)
-&gt; no colonies</LI><LI>Design primers for <B>Gibson assembly</B></LI></UL><LI> Experiment 3 </LI><UL class="a"><LI><B> Colony PCR with higher annealing temperature to minimize false priming</B> on kolonies from transformed strains using emerald polymerase with out primers to check KI: MDM0046/MDM0010. Expected fragment: ca. 5500 bp. Analytic gel: <B> positive </B></LI><LI><B> HiFi PCR using Q5 polymerase with two primer pairs generating overlapping fragments of the ccdA.gfp.cat construct </B>: MDM0046 (out) &amp; MDM0123: 2130 bp and MEMO1237 &amp; MDM0010 (out): 3400 bp. Analytic gel: <B>negative</B></LI></UL><LI> Experiment 4 </LI><UL class="a"><LI><B> Lysate preparation</B>:
 -&gt; Dilution of o/n culture of BW26547 in 5 mL LB+MgCL2+CaCl2+glucose 
 -&gt; Add phage lysate (10,20,40 &amp; 80 µL)
 -&gt; Wait untill lysis visible </LI><LI><B> Perform CIChE with 8 + pSB6A1 &amp; 5 + pSB6A1 </B> (we will do 2 strains parallel): 0,0mM - 0,05mM IPTG </LI><LI><B> Transduction</B> of CIChe strains: no positive colonies after UV-test </LI></UL><LI> Experiment 6 </LI><UL class="a"><LI> New <B>transformation</B> in <I>E. coli</I> DH5a, using heat shock (42°C) 
 (because we are sure that the fragments were present for ligation and so will lead to succesful results)
 -&gt; Plate on chloramphenicol plate and grow overnight at 37°C: <B>negative</B> (it still does not work) </LI><LI> Control of ligations by using PCR with primers MDM0606 and MDM0607: <B>negative</B></LI><LI><B>Ligation</B> again of CcdB operon and pSB1C3: 15 minutes at 16°C</LI><LI><B>Transformation</B> in <I>E. coli</I> DH5a, using heat shock (42°C)
 -&gt; Plate on chloramphenicol plate and grow overnight at 37°C: <B>negative</B></LI><LI><B>CcdB operon</B>: HiFi PCR of plasmids p20SpFRT-T7<I>ccdB</I> with MDM0586/MDM0587 to amplify CcdB operon
 because we have no more T7-<I>ccdB</I> fragments</LI><LI> New <B>restriction</B> of CcdB operon and pSB1C3 with XbaI and PstI
 -&gt; Gel purify fragments using Qiagen Qiaquick gel extraction kit: nanodrop:
 Restriction Digest of T7-<I>ccdB</I>: 15.3 ng/µl
 Restriction Digest of pSB1C3: 16.8 ng/µl</LI><LI><B>Ligation</B> of CcdB operon and pSB1C3: 1 hour at room temperature</LI><LI><B>Transformation</B> in <I>E. coli</I> DH5a, using heat shock (42°C)
 -&gt; Plate on chloramphenicol plate and grow overnight at 37°C: <B>positive</B> (finally)</LI><LI><B>Colony PCR</B> (Taq): <B>negative</B> It seems that restriction-ligation does not work, thus we will use another technique, called Gibson Assembly </LI><LI> Designing primers for Gibson Assembly for the insert T7-<I>ccdB</I> and the backbone pSB1C3  </LI></UL></UL></P><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
Week 5: august 26 - september 1   
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><P align="justify"><UL><LI> Experiment 2 </LI><UL class="a"><LI> Inoculation of pSB3T5, pSB4A5, pSB6A1-T7<I>ccdB</I>, p5SpFRT-T7<I>ccdB</I>, p10pFRT-T7<I>ccdB</I> and p20pFRT-T7<I>ccdB</I> and plasmid purification using Qiagen Spin minikit: nanodrop
p10pFRT-T7<I>ccdB</I>: 62.8 ng/µl
p20pFRT-T7<I>ccdB</I>: 286.1 ng/µl</LI><LI><B>PCR on CcdB operon</B> using Q5 polymerase (primers: MDM0586_Fw-Trc-<I>ccdB</I>-G00000 and MDM0587_Rv-Trc-<I>ccdB</I>-G00001): nanodrop
p10: 29.7 ng/µl
p20: 31.4 ng/µl</LI><LI><B>Restriction</B> of pSB3T5, pSB4A5 and CcdB operon with PstI-HF and XbaI (gel purify RD-fragments using Qiagen Qiaquick gel extraction kit (expected fragment of 2157 bp for T7-<I>ccdB</I>, 3229 bp for pSB3T5 and 3372 bp for pSB4A5): nanodrop:
pSB3T5: 2.7 ng/µl
pSB4A5: 7.8 ng/µl
 T7-<I>ccdB</I>: 10.5 ng/µl</LI><LI><B>Q5 PCR</B> of CcdB operon and plasmids (pSB3T5 and pSB4A5), using the designed Gibson primers 
 -&gt; Purification of PCR-product after DpnI treatment by using minElute reaction cleanup kit
 -&gt; Control of fragments on a gel =&gt; Backbones are present, CcdB operons not </LI><LI><B>PrimeStar PCR</B> of CcdB operons, using the designed Gibson primers 
 -&gt; Purification of PCR-product after DpnI treatment by using minElute reaction cleanup kit
 -&gt; Control of fragments on a gel =&gt; CcdB operons are still not present </LI><LI><B>Q5 PCR</B> of CcdB operons on a linear CcdB operon, using the designed Gibson primers 
 -&gt; Purification of PCR-product, using Qiagen Qiaquick PCR purification kit,
 -&gt; Control of fragments on a gel =&gt; CcdB operon is visible </LI><LI><B>Gibson Assembly</B> (1h at 50°C)</LI><LI><B>Transformation</B> in <I>E. coli</I> DH5a, using heat shock (42°C)
 -&gt; Plate pSB3T5 on tetracyclin plate and grow overnight at 37°C: <B>positive</B> -&gt; Plate pSB4A5 on ampicillin plate and grow overnight at 37°C:  <B>positive</B> Strange phenomena: most colonies are red, let’s control it </LI><LI><B>Colony PCR</B> of colonies derived from Gibson Assembly: <B>negative</B></LI></UL><LI> Experiment 3 </LI><UL class="a"><LI><B> Transformation </B> of plasmids:p5SpFRT-T7<I>ccdB</I>, p10SpFRT-T7<I>ccdB</I>, p20SpFRT-T7<I>ccdB</I> in KI-strain (8) and back-up strain 5 </LI><LI><B>Colony PCR</B> on 5 + pSB6A1/p5/p10/p20 using emerald polymerase with primerpairs MDM0046/MDM010 to check KI. Expected fragment: 5500bp. Analytic gel: <B> positive </B></LI><LI><B>Colony PCR</B> on 5 + pSB6A1/p5/p10/p20 using emerald polymerase with primerpairs MDM0096/CGL0019 for pSB6A1 and MDM0039/MDM0060 for p5/p10/p20. Expected fragment: 437bp &amp; 910 bp. Analytic gel: <B> positive for p5/p10/p20</B></LI><LI><B> New transformation </B> 5 +pSB6A1 and check KI + plasmid as described above. Analytic gel: <B> positive </B>.</LI></UL><LI> Experiment 4 </LI><UL class="a"><LI><B> Abort CIChe with 8+pSB6A1 &amp; 5+pSB6A1 </B> because of mutation in pSB6A1 </LI><LI> Lysate preparation </LI><LI><B> Transduction:  </B> on CIChE strains described above </LI><LI> Test lysate by performing transduction on strain 5 with different concentrations of lysate: <B>negative</B></LI></UL><LI> Experiment 6 </LI><UL class="a"><LI><B>While waiting for the Gibson Assembly primers  </B></LI><UL class="a"><LI>  Again <B>restriction</B> of CcdB operon and pSB1C3 with XbaI and PstI
 -&gt; Gel purify fragments using Qiagen Qiaquick gel extraction kit: nanodrop:
 Restriction Digest of T7-<I>ccdB</I>: 10.5 ng/µl
 Restriction Digest of pSB1C3: 7.2 ng/µl</LI><LI><B>Ligation</B> of CcdB operon and pSB1C3: overnight at 16°C</LI><LI><B>Transformation</B> in <I>E. coli</I> DH5a, using heat shock (42°C),
 -&gt; Plate on chloramphenicol plate and grow overnight at 37°C: <B>positive</B></LI><LI><B>Colony PCR</B> (Taq) of these colonies: <B>negative</B></LI></UL><LI><B>After receiving the primers for Gibson Assembly</B></LI><LI><B>Q5 PCR</B> of CcdB operon and pSB1C3, using the designed Gibson primers 
 -&gt; Purification of PCR-product, after DpnI treatment, by using minElute reaction cleanup kit
 -&gt; Control of fragments on a gel =&gt; Backbone is present, CcdB operon not </LI><LI><B>PrimeStar PCR</B> of CcdB operon, using the designed Gibson primers 
 -&gt; Purification of PCR-product, after DpnI treatment, by using minElute reaction cleanup kit
 -&gt; Control of fragments on a gel =&gt; CcdB operon is not present </LI><LI><B>Q5 PCR</B> of CcdB operon on a linear CcdB operon, using the designed Gibson primers 
 -&gt; Purification of PCR-product, using Qiagen Qiaquick PCR purification kit
 -&gt; Control of fragments on a gel =&gt; CcdB operon is visible </LI><LI><B>Gibson Assembly</B> (1h at 50°C)</LI><LI><B>Transformation</B> in <I>E. coli</I> DH5a, using heat shock (42°C),
 -&gt; Plate on chloramphenicol plate and grow overnight at 37°C: <B>positive</B> Strange phenomena: colonies are red, let’s control it </LI><LI><B>Colony PCR</B> (Taq) of colonies derived from Gibson Assembly: <B>negative</B> We think that some of the original plasmid has to be present 
 and the bacteria love that plasmid more than the plasmid with T7-<I>ccdB</I></LI></UL></UL></P><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><H2> September </H2><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
Week 1: september 2 - 8 
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><P align="justify"><UL><LI> Experiment 2 </LI><UL class="a"><LI><B>Restriction</B> of CcdB operon and pSB3T5 with EcoRI and SpeI 
-&gt; Gel purify fragments using Qiagen Qiaquick gel extraction kit: nanodrop: 
Restriction Digest of T7-<I>ccdB</I>: 14.4 ng/µl 
Restriction Digest of pSB3T5 :  7.2 ng/µl
</LI><LI><B>Restriction</B> of CcdB operon and pSB4A5 with EcoRI and SpeI 
-&gt; Gel purify fragments using Qiagen Qiaquick gel extraction kit: nanodrop: 
Restriction Digest of T7-<I>ccdB</I>: 14.4 ng/µl 
Restriction Digest of pSB4A5: 7.8 ng/µl
</LI><LI><B>Ligation</B> of CcdB operon and pSB3T5 and of CcdB operon and pSB4A5: 25 minutes at room temperature </LI><LI><B>Transformation</B> in <I>E. coli</I> DH5a using heat shock 
 -&gt; Plate pSB3T5 on tetracyclin plate and grow overnight at 37°C: <B>negative</B> -&gt; Plate pSB4A5 on ampicillin plate and grow overnight at 37°C: <B>negative</B></LI><LI><B>Gibson Assembly</B> (1h at 50°C) of pSB3T5 and CcdB operon</LI><LI><B>Transformation</B> in <I>E. coli</I> Top10 subcloning cells, using electroporation (Time Constant: 4.3) 
-&gt; Plate on tetracyclin plate and grow overnight at 37°C: <B>negative</B></LI><LI><B>Gibson Assembly</B> (1h at 50°C) of pSB4A5 and CcdB operon</LI><LI><B>Transformation</B> in <I>E. coli</I> Top10 subcloning cells, using electroporation (Time Constant: 4.4) 
-&gt; Plate on ampicillin plate and grow overnight at 37°C:<B>negative</B></LI></UL><LI> Experiment 4 </LI><UL class="a"><LI><B> CIChE with strain 8 + pSB6A1/p5/p10&amp;p20:</B> --&gt; Inoculated without antibiotic &amp; plated out on antibiotic: 0mM - 0.5mM IPTG
 --&gt; Inoculated in antibiotic - sequentially: 0mM - 0.5mM IPTG
 --&gt; Inoculated in antibiotic - parallel: 0mM -0.5 mM IPTG </LI><LI><B> Transduction: </B> on CIChE strains described above --&gt; colonies on control plate </LI></UL><LI> Experiment 5 </LI><UL class="a"><LI> Test evaluation GFP with LB-medium: 
 -&gt; Wild type
 -&gt; DH5a + pTGD-ccd-Pmb1GFP-CmFRT
 -&gt; Strains with GFP: 8 / 5 + pSB6A1/p5/p10/p20 -- 0mM &amp; 0.01 mM IPTG </LI><LI> Test evaluation GFP with EZrich-medium: 
 -&gt; Wild type
 -&gt; DH5a + pTGD-ccd-Pmb1GFP-CmFRT
 -&gt; Strains with GFP: 8 / 5 + pSB6A1/p5/p10/p20 -- 0mM/0.01mM/0.05mM &amp; 0.2mMM IPTG </LI></UL><LI> Experiment 6 </LI><UL class="a"><LI><B>Restriction</B> of CcdB operon and pSB1C3 with EcoRI and SpeI 
-&gt; Gel purify fragments using Qiagen Qiaquick gel extraction kit: nanodrop: 
Restriction Digest T7-<I>ccdB</I>: 14.4 ng/µl 
Restriction Digest pSB1C3: 4.4 ng/µl
</LI><LI><B>Ligation</B> of CcdB operon and pSB1C3: 25 minutes at room temperature </LI><LI><B>Transformation</B> in <I>E. coli</I> DH5a, using heat shock (42°C), 
-&gt; Plate on chloramphenicol plate and grow overnight at 37°C : <B>negative</B></LI><LI><B>Q5 PCR</B> of pSB1C3, using the designed Gibson primers and the linearised plasmid
 -&gt; Purification of PCR-product, using Qiagen Qiaquick PCR purification kit: nanodrop: 
 pSB1C3-backbone: 109.9 ng/µl</LI><LI><B>Gibson Assembly</B> (1h at 50°C)</LI><LI><B>Transformation</B> in <I>E. coli</I> Top10 subcloning cells, using electroporation (Time Constant: 4.4) 
-&gt; Plate on chloramphenicol plate and grow overnight at 37°C: <B>positive</B></LI><LI><B>Colony PCR</B> (Crimson) using primers MDM0606 and MDM0607: 
 6 colonies can possible be <B>positive</B>  (but it is not sure)
-&gt; Inoculate the 6 colonies (overnight)</LI><LI>Control of fragment of pSB1C3 on a gel =&gt; Backbone is not present</LI><LI>Plasmid purification of the 6 colonies + restriction to control of they are the good colonies (on a gel)
 There is a possibility that they are the good ones, so we will send it to sequenate 
 -&gt; pSB1C3-T7<I>ccdB</I> sent for <B>sequencing</B> with primers MDM0606, MDM0607, MDM0060 and MDM0039: <B>negative</B></LI><LI><B>Q5 PCR</B>  of pSB1C3, using the designed Gibson primers and the linearised plasmid
-&gt; Purification of PCR-product, using Qiagen Qiaquick PCR purification kit: 
 nanodrop: pSB1C3-backbone: 58.0 ng/µl
 -&gt; Control of fragment of pSB1C3 on a gel =&gt; Backbone is not present</LI><LI><B>Q5 PCR</B>  of pSB1C3, using the designed Gibson primers and the restriction digest of pSB1C3
-&gt; Purification of PCR-product, using Qiagen Qiaquick PCR purification kit: 
nanodrop: pSB1C3-backbone: 76.2 ng/µl
 -&gt; Control of fragment of pSB1C3 on a gel =&gt; Backbone is present</LI><LI><B>Gibson Assembly</B> (1h at 50°C)</LI><LI><B>Transformation</B> in <I>E. coli</I> Top10 subcloning cells, using electroporation (Time Constant: 4.3) </LI>
-&gt; Plate on chloramphenicol plate and grow overnight at 37°C: <B>positive</B> =&gt; many colonies<LI><B>Colony PCR</B> (Crimson):
 We tested 48 colonies (using primers MDM0606 and MDM0607) and 4 colonies show the right fragment (2520 bp)
 The first 24 colonies , and the second 24 colonies  -&gt; Inoculate these 4 good colonies</LI><LI> Plasmid purification of the 4 colonies, using Qiagen Qiaprep Spin minikit: nanodrop:
-&gt; pSB1C3 strain 1:  94.2 ng/µl
-&gt; pSB1C3 strain 2: 163.4 ng/µl
-&gt; pSB1C3 strain 3: 160.2 ng/µl
-&gt; pSB1C3 strain 4: 124.3 ng/µl</LI><LI><B>Restriction</B> with BspHI to control on a gel of they are the right colonies 
(expected fragments are 1028bp and 3248 bp): <B>positive</B>-&gt; Strains 2, 3 and 4 seems to be the correct colonies</LI><LI> pSB1C3-T7<I>ccdB</I> of these 3 colonies sent for <B>sequencing</B> with primers MDM0606, MDM0607, MDM0060 and MDM0039 
-&gt; sequence shows a mutation in the stop-codon, so we have to start over again</LI></UL></UL></P><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
Week 2: september 9 - 15 
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><P align="justify"><UL><LI> Experiment 2 </LI><UL class="a"><LI><B>Q5 PCR</B> for amplification of T7-<I>ccdB</I>-fragment, using the designed Gibson primers, on p5SpFRT-T7<I>ccdB</I>, p10SpFRT-T7<I>ccdB</I> and p20SpFRT-T7<I>ccdB</I> -&gt; Using DpnI at 37°C for 1 hour
 -&gt; Purification of PCR-product by using minElute reaction cleanup kit
 -&gt; Control on gel: <B>negative</B></LI><LI> Plasmid pSB3T5</LI><UL class="a"><LI><B>Gibson Assembly</B> (1h at 50°C) of pSB3T5 and CcdB operon that we already amplified before</LI><LI><B>Transformation</B> in <I>E. coli</I> Top10 subcloning cells, using electroporation (Time Constant: 3.0) 
-&gt; Plate on tetracyclin plate and grow overnight at 37°C: <B> negative</B>, there are oly red colonies</LI></UL><LI> Plasmid pSB4A5</LI><UL class="a"><LI><B>Gibson Assembly</B> (1h at 50°C) of pSB4A5 and CcdB operon that we already amplified before</LI><LI><B>Transformation</B> in <I>E. coli</I> Top10 subcloning cells, using electroporation (Time Constant: 4.0) 
-&gt; Plate on ampicillin plate and grow overnight at 37°C: <B>positive</B></LI><LI><B>Colony PCR</B> (Crimson): there is only 1 colony. We tested this colony (using primers MDM0606 and MDM0607) and it show the right fragment (2486 bp) on gel
-&gt; Inoculate these colonies</LI><LI> Plasmid purification of the colonies, using Qiagen Qiaprep Spin minikit</LI><LI> Seqenate this plasmid: the result is a totally different sequence than expected </LI></UL><LI> Plasmid pSB6A1</LI><UL class="a"><LI> We controled the pSB6A1 on mutations (by sequencing) and we found also a mutation on in the <I>ccdB</I> gene</LI><LI> Try to design Gibson primers to undo the mutations 
(We will use primers to amplify the biggest part of the plasmid (PCR) and afterwards using Gibson Assembly with oligos, which undo the founded mutation) </LI><LI><B>Q5 PCR</B> for amplification of biggest part of the plasmid pSB6A1 (using the designed Gibson primers). 
 -&gt; Using DpnI at 37°C for 1 hour
 -&gt; Purification of PCR-product by using minElute reaction cleanup kit
 -&gt; Control on gel: <B>positive</B> (fragment length is 6192bp)</LI><LI><B>Gibson Assembly</B> -&gt; melt oligos first: 5 minutes at 95°C 
 -&gt; 6192bp-fragment and oligos at 50°C (1 h)</LI><LI><B>Transformation</B> in <I>E. coli</I> Top10 subcloning cells, using electroporation  (Time Constant: 2.4) 
-&gt; Plate on ampicillin plate and grow overnight at 37°C: <B> positive</B></LI><LI> We made our own competent cells which have the <I>ccdA</I> gene in the genome, so they will be more tolerant to CcdB. We hope on no mutations this time.
-&gt; <B>Q5 PCR</B>: Control of Knock-In
 -&gt; Control on gel: <B>positive</B> -&gt; Further on we will call these cells competent strain 8
</LI><LI> Second <B>transformation</B> in competent strain 8, using electroporation  (Time Constant: 1.6) 
-&gt; Plate on ampicillin plate and grow overnight at 37°C: <B> negative</B></LI><LI><B>Gibson Assembly</B> -&gt; melting oligos first: 5 minutes at 95°C + infinite at 50°C
 -&gt; 6192bp-fragment and oligos at 50°C (1 h)</LI><LI><B>Transformation</B> in competent strain 8, using electroporation (Time Constant: 1.7) 
-&gt; Plate on ampicillin plate and grow overnight at 37°C: <B> negative</B></LI></UL></UL><LI> Experiment 4 </LI><UL class="a"><LI> CIChE with strain 8 + p5/p10/p20: 
 -&gt; STEP: 0mM - 0.1mM IPTG
 -&gt; JUMP: 0mM - 0.5mM IPTG </LI><LI> CIChE with strains 5 + p5/p10/p20:
 -&gt; STEP + JUMP: 0 - 0.01mM </LI></UL><LI> Experiment 5 </LI><UL class="a"><LI><B> MTP experiment: version 1 (using precultures) </B> (Evaluation GFP 4 colonies/strain -  3 repeats) 
 -&gt; 8+pSB6A1 - sequential: 0mM - 0.5mM IPTG 
 -&gt; 8+pSB6A1 - strains grown without antibiotics: 0mM - 0.5mM IPTG 
 -&gt; 8+p10 - sequential: 0mM - 0.05mM  IPTG </LI><LI> Measure the OD and the GFP (using FLUOstar)</LI></UL><LI> Experiment 6 </LI><UL class="a"><LI><B>Q5 PCR</B> for amplification of T7-<I>ccdB</I>-fragment, using the designed Gibson primers, on p5SpFRT-T7<I>ccdB</I>, p10SpFRT-T7<I>ccdB</I> and p20SpFRT-T7<I>ccdB</I> -&gt; Using DpnI at 37°C for 1 hour
 -&gt; Purification of PCR-product by using minElute reaction cleanup kit
 -&gt; Control on gel: <B>negative</B></LI><LI><B>Gibson Assembly</B> (1h at 50°C), using a T7-<I>ccdB</I> we already amplified before</LI><LI><B>Transformation</B> in <I>E. coli</I> Top10 subcloning cells, using electroporation (Time Constant: 4.0) 
-&gt; Plate on chloramphenicol plate and grow overnight at 37°C: <B> negative</B></LI><LI> Try to design Gibson primers to undo the mutations, found in the <I>ccdB</I> gene 
(We will use primers to amplify the biggest part of the plasmid (PCR) and afterwards using Gibson Assembly with oligos, which undo the founded mutation) </LI><LI><B>Q5 PCR</B> for amplification of biggest part of the plasmid (using the designed Gibson primers). 
 We will do this on the 2 best colonies of last time.
 -&gt; Using DpnI at 37°C for 1 hour
 -&gt; Purification of PCR-product by using minElute reaction cleanup kit
 -&gt; Control on gel: <B>positive</B> (fragment length is 4233bp)</LI><LI><B>Gibson Assembly</B> -&gt; melt oligos first: 5 minutes at 95°C 
 -&gt; 4233bp-fragment and oligos at 50°C (1 h)</LI><LI><B>Transformation</B> in <I>E. coli</I> Top10 subcloning cells, using electroporation 
(Time Constant for colony 1: 4.0, for colony 2: 4.1) 
-&gt; Plate on chloramphenicol plate and grow overnight at 37°C: <B> positive</B></LI><LI> We made our own competent cells which have the <I>ccdA</I> gene in the genome = strain 8, so they will be more tolerant to CcdB.
 We hope on no mutations this time
-&gt; <B>Q5 PCR</B>: Control of Knock-In
 -&gt; Control on gel: <B>positive</B></LI><LI> Second <B>transformation</B> in competent strain 8, using electroporation 
(Time Constant for colony 1: 1.5, for colony 2: 1.4) 
-&gt; Plate on chloramphenicol plate and grow overnight at 37°C: <B> positive</B></LI><LI><B>Gibson Assembly</B> -&gt; melting oligos first: 5 minutes at 95°C + infinite at 50°C
 -&gt; 4233bp-fragment and oligos at 50°C (1 h)</LI><LI><B>Transformation</B> in competent strain 8, using electroporation (Time Constant for colony 1: 1.5, for colony 2: 1.4) 
-&gt; Plate on chloramphenicol plate and grow overnight at 37°C: <B> positive</B></LI></UL></UL></P><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
Week 3: september 16 - 22 
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><P align="justify"><UL><LI> Experiment 2 </LI><UL class="a"><LI> Plasmid pSB3T5</LI><UL class="a"><LI><B>Transformation</B> in competent strain 8, using electroporation  (Time Constant: 1.4) 
-&gt; Plate on tetracyclin plate and grow overnight at 37°C: <B> negative</B></LI><LI><B>Transformation</B> in competent strain 8, using electroporation 
 (Time Constant: pSB3T5+T7-<I>ccdB</I> of p10: 4.1,   pSB3T5+T7-<I>ccdB</I> of p20: 2.2) 
-&gt; Plate on tetracyclin plate and grow overnight at 37°C: <B> positive</B>, but also red colonies </LI><LI><B>Colony PCR</B> (Crimson):
 We tested 15 colonies (using primers MDM0606 and MDM0607) : <B> negative</B> (expected fragment: 2486 bp)</LI></UL><LI> Plasmid pSB4A5</LI><UL class="a"><LI><B>Transformation</B> in competent strain 8, using electroporation  (Time Constant: 1.5) 
-&gt; Plate on ampicillin plate and grow overnight at 37°C: <B> negative</B></LI><LI><B>Transformation</B> in competent strain 8, using electroporation 
 (Time Constant: pSB3T5+T7-<I>ccdB</I> of p10: 2.7,  pSB3T5+T7-<I>ccdB</I> of p20: 2.6) 
-&gt; Plate on ampicillin plate and grow overnight at 37°C: <B> positive</B>, but also red colonies  </LI><LI><B>Colony PCR</B> (Crimson):
  We tested the few colonies, we had (using primers MDM0606 and MDM0607) : <B> negative</B> (expected fragment: 2486 bp)</LI></UL><LI> Plasmid pSB6A1</LI><UL class="a"><LI><B>Colony PCR</B> (Crimson):
 We tested 24 colonies of pSB6A1(using primers MDM0606 and MDM0607) and  we found many good colonies
 (expected length is 4387 bp) 
-&gt; We chose 4 colonies to sequenate 
-&gt;  Inoculate these 4 good colonies</LI><LI> Plasmid purification of the 4 colonies, using Qiagen Qiaprep Spin minikit: nanodrop:
-&gt; pSB6A1 strain 4 : 399.4 ng/µl
-&gt; pSB6A1 strain 13: 412.0 ng/µl
-&gt; pSB6A1 strain 16: 365.5 ng/µl
-&gt; pSB6A1 strain 19: 420.3 ng/µl</LI><LI><B>Sequencing</B> shows in 2 colonies other mutations  
and the other 2 colonies shows the right sequence
-&gt; send this 2 good strains to iGEM (<B>BioBrick</B>)</LI></UL></UL><LI> Experiment 3 </LI><UL class="a"><LI>Transformation
</LI></UL><LI> Experiment 4 </LI><UL class="a"><LI> CIChE with strains 8 + p5/p10/p20:
 -&gt; STEP: 0.1mM - 1.0mM IPTG
 -&gt; JUMP: 0.5mM -  3mM IPTG </LI><LI> CIChE with strains 5 + p5/p10/p20:
 -&gt; STEP: 0.01mM - 0.5mM IPTG
 -&gt; JUMP: 0.5mM - 1.0mM IPTG </LI><LI> CIChE with strains 8 + pSB6A1 (New plasmids from colony 4 &amp; 16 (exp 2)):
 -&gt; STEP: 0mM - 0.2mM IPTG
 -&gt; JUMP: 0.5mM - 5.0mM IPTG </LI></UL><LI> Experiment 6 </LI><UL class="a"><LI><B>Colony PCR</B> (Crimson): We tested 24 colonies (using primers MDM0606 and MDM0607) and we found many good colonies
 (expected length is 2520 bp)   
-&gt; We chose 4 colonies to sequenate 
-&gt;  Inoculate these 4 good colonies</LI><LI> Plasmid purification of the 4 colonies, using Qiagen Qiaprep Spin minikit: nanodrop:
-&gt; pSB1C3 strain 10: 181.1 ng/µl
-&gt; pSB1C3 strain 19: 169.1 ng/µl
-&gt; pSB1C3 strain 20: 184.8 ng/µl
-&gt; pSB1C3 strain 24: 184.5 ng/µl</LI><LI><B>Sequencing</B> shows the same mutation at the end codon again</LI></UL></UL></P><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
Week 4: september 23 - 29 
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><P align="justify"><UL><LI> Experiment 4 </LI><UL class="a"><LI> CIChE with strains 5 + p5/p10/P20:
 -&gt; STEP: 1.0mM - 4.0mM IPTG</LI><LI> CIChE with strain 8 + pSB6A1:
 -&gt; STEP: 0.2mM - 2.0mM IPTG
 -&gt; JUMP: 5mM - 120mM IPTG</LI></UL><LI> Experiment 5 </LI><UL class="a"><LI><B> MTP experiment: version 1 (using precultures) </B> -&gt; Plate1: 8 + p5;STEP; 0mM Day1 - 0.5mM Day2
 -&gt; Plate2: 8 + p10;STEP; 0mM Day1 - 0.5mM Day2
 -&gt; Plate3: 8 + p20;STEP; 0mM Day1 - 0.5mM Day2
 -&gt; Plate4: 8 + pSB6A1-16;STEP; 0mM Day1 - 0.05mM Day2
 -&gt; Plate5: 8 + p5;JUMP; 0mM Day1 - 2mM Day2
 -&gt; Plate6: 8 + p10;JUMP; 0mM Day1 - 2mM Day2
 -&gt; Plate7: 8 + p20;JUMP; 0mM Day1 - 2mM Day2
</LI><LI> Measure the OD and the GFP of all previous plates (using FLUOstar)</LI><LI><B> MTP experiment: Test version 2 </B> (to know how long they have to grow until they reach a sufficient OD: 4 - 5 hours)
</LI><LI><B> MTP experiment: version 2 </B> -&gt; Plate1: 8 + p5;STEP; 0mM Day1 - 0.5mM Day2
 -&gt; Plate2: 8 + p10;STEP; 0mM Day1 - 0.5mM Day2
 -&gt; Plate3: 8 + p20;STEP; 0mM Day1 - 0.5mM Day2
 -&gt; Plate4: 8 + pSB6A1-16;STEP; 0mM Day1 - 0.05mM Day2
 -&gt; Plate5: 8 + p5;JUMP; 0mM Day1 - 2mM Day2
 -&gt; Plate6: 8 + p10;JUMP; 0mM Day1 - 2mM Day2
 -&gt; Plate7: 8 + p20;JUMP; 0mM Day1 - 2mM Day2
 -&gt; Plate8: 8 + pSB6A1-16;STEP; 0.1mM Day1 - 0.2mM Day1 &amp; JUMP;0mM Day1 - 5mM Day1
 -&gt; Plate9: 8 + pSB6A1-4;STEP; 0mM Day1 - 0.5mM Day2
 -&gt; Plate10: 8 + pSB6A1-4;STEP; 1.0mM Day1 - 2.0mM Day1 &amp; JUMP; 0mM Day1 - 1.5mM Day1
 -&gt; Plate11: 8 + pSB6A1-4;JUMP; 1.5mM Day2 - 30.0mM Day2
 -&gt; Plate12: 8 + pSB6A1-4;JUMP; 60.0mM Day2 - 120.0mM Day1 <B>&amp;</B> 5 + p5;STEP;0mM Day1 - 0.2mM Day1
 -&gt; Plate13: 5 + p5;STEP; 0.2mM Day2 - 3.0mM Day2
 -&gt; Plate14: 5 + p5;STEP; 3.5mM Day1 &amp; JUMP;0mM Day1- 1mM Day1 
<B>&amp;</B> 5 + p10;STEP;0mM Day1 - 0,01mM Day1
 -&gt; Plate15: 5 + p10;STEP; 0.01mM Day2 - 1.0mM Day2
 -&gt; Plate16: 5 + p10;STEP; 2mM Day1 - 3.5mM Day1 &amp; JUMP;0mM Day1 - 0.1mM Day1
 -&gt; Plate17: 5 + p10;JUMP; 0.1mM Day2 - 1.0mM Day1 <B>&amp;</B> 5 + p20;STEP; 0mM Day1 - 0.1mM Day2
 -&gt; Plate18: 5 + p20;STEP; 0.2mM Day1 - 3.0mM Day1
 -&gt; Plate19: 5 + p20;STEP; 3.0mM Day1 - 3.5mM Day1 &amp; JUMP;0mM Day1 - 1mM day1
</LI><LI> Measure the OD and the GFP of all plates (using FLUOstar)</LI></UL></UL></P><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV><H2> October </H2><P class="trigger"><A class="toggle_text" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961">
Week 1: september 30 - october 4 
</A></P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 95%;"><P align="justify"><UL><LI> Experiment 4 </LI><UL class="a"><LI> CIChE with strains 5 + p5/p10/P20:
 -&gt; STEP: 4.0mM - 5.0mM IPTG </LI><LI> CIChE with strain 8 + pSB6A1:
 -&gt; STEP: 2.0mM - 3.0mM IPTG
 -&gt; JUMP: 120.0mM - 240mM IPTG</LI></UL><LI> Experiment 5 </LI><UL class="a"><LI><B> MTP experiment: version 2 </B> -&gt; Plate20: 8 + pSB6A1-4;STEP; 2.5mM Day2 - 3.0mM Day2 &amp; JUMP; 120mM Day2 - 240mM Day2
 -&gt; Plate21: 5 + p5;STEP; 3.5mM Day2 - 5mM Day2 <B>&amp;</B> 5 + p10;STEP; 3.5mM Day2 - 5mM Day2
 -&gt; Plate22: 5 + p20;STEP; 3.5mM Day2 - 5mM Day2 <B>&amp;</B> 5 + p5;STEP; 1mM Day1 and 3mM Day1
<B>&amp;</B> 5 + p10;STEP; 1mM Day1 and 3mM Day1<B>&amp;</B> 5 + p20;STEP; 1mM Day1 and 3mM Day1
</LI><LI> Measure the OD and the GFP of the 3 plates (using FLUOstar)</LI></UL></UL></P><A class="toggle_close" href="#" style="font-size: 13px;font-weight:bold; color: #F7B961" align="left">Close</A></DIV></DIV></TD><TD bgcolor="#F9BD05" valign="top" align="center" width="210px"><TITLE>Project: NetSite...JavaScript Countdown Clock</TITLE></TD></TR><TR><TD align="CENTER" width="31%" bgcolor="#F9BD05"><FONT color="#FFFFFF"><B>Days:</B></FONT></TD><TD align="CENTER" width="23%" bgcolor="#F9BD05"><FONT color="#FFFFFF"><B>Hours:</B></FONT></TD><TD align="CENTER" width="23%" bgcolor="#F9BD05"><FONT color="#FFFFFF"><B>Mins:</B></FONT></TD><TD align="CENTER" width="23%" bgcolor="#F9BD05"><FONT color="#FFFFFF"><B>Secs:</B></FONT></TD></TR><TR><TD colspan="4" bgcolor="#F9BD05"><CENTER><FONT face="Verdana, Arial,sans-serif" size="+1.5" color="#FFFFFF">Until Jamboree Lyon</FONT></CENTER></TD></TR></TBODY></TABLE><A class="twitter-timeline" href="https://twitter.com/iGEM_UGent" data-widget-id="373064939419607041">Tweets van @iGEM_UGent</A><TABLE border="0" cellspacing="0" cellpadding="0" width="70%" align="center"><DIV align="right"><H1>We thank following sponsors for their support</H1></DIV></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UGent/Labjournal">http://2013.igem.org/Team:UGent/Labjournal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UGent/Labjournal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UGent/Labjournal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UGent/Labjournal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UGent/Labjournal&amp;oldid=313003" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>