"
Page
Discussion
View source
History
teams
Log in
 
Team:UT-Tokyo/Sudoku assay LeakSw
From 2010.igem.org
UT-Tokyo
team
project
parts
protocols
online
sponsor
consideration
Project
Sudoku
Introduction
System
Modeling
Experiments
Perspective
Reference
Sudoku
Introduction
System
Modeling
Experiments
Perspective
Reference
=>-Assay- Terminator Leak/ Location Sequence / Phage MS2
Terminator Leak
Abstract
To construct 4C3 leak-switch, which is indispensable to determine the numbers when solving Sudoku, the following system is required: do not express enough amount of Cre gene that has two or more terminators on the upstream, and  express enough amount of Cre gene that has only one terminator there. For this purpose, it is neccessary to use appropriate terminator.
The aim of terminator leakiness assay is to make sure whether our terminator is appropriate or not. We measured the fluorescence of GFP expressed when Cre protein recombine DNA construct. When the fluorescence of expressed GFP depends on the existence of Cre gene in this construct, our system works properly. However, in the first trial, the fluorescence of GFP cannot be observed in our construct. We considered whether lox sequence, cre coding sequence or both of them have some errors, and performed some experiments to make sure our hypothesis. As a result, it elucidated that lox sequence is not functional.
INDEX
(experiment1-4)
Profiling of the gene expression by T7 RNA polymerase
(experiment5,6)
Verification of the recombination activity of Cre protein
(experiment7)
Verification of the activity of the lox sequence in our construct
Experiment 1
The construct of experimant 1,2,3,4,5
Profiling of the gene expression by T7 RNA polymerase
First, we did some experiments to profile the gene expression by T7 RNA polymerase. The part 1-24 (T7 promoter – GFP unit) plasmid was transformed to BL21 (DE3) strain, and incubated on the LB plate at 37 oC.One colony was isolated and cultivated in two LB mediums. One sample was added IPTG (final conc. 1mM) and incubated at 37 oC. The samples were centrifuged and the supernatant were removed. We checked fluorescence from pellets on our eyes.
DE3 strain is one of the E.coli strain which codes the gene of T7 RNA polymerase at the downstream of lac operon on its genome.
As a result, we could not observe the increasing of the GFP flourescence by addition of IPTG.The cultivated bacteria without the addition of IPTG turned green and those with the addition of IPTG did not turn green.
From this result, the following possibility was suggested. Green bacteria without the addition of IPTG → T7 RNA polymerase was expressed weakly and transcribed the mRNA coding gfp.  White bacteria with the addition of IPTG → Overexpressed GFP made inclusion body.
Experiment 2
From the previous result, we expected that reduced IPTG concentration will  prevent the formation of inclution body and performed experiment 1 in low IPTG concentration.We checked the GFP fluorescence of both samples on our eyes.
The GFP fluorescence of both samples were measured and there was no difference between the two samples .
It was suggested that BL21(DE3) is not proper to control the expression of gene expression.
We desided to use Rossetta (DE3) pLysS, that can repress the activity of the leaked T7 RNA polymerase.
Experiment 3
The plasmid containing part 1-24 was transformed to Rosetta (DE3) plysS.
Experiment 4
We improved the protocol to the following.
We used full growth samples preserved in glycerol made in experiment three. We added samples I-V into LB medium and cultured at 37oC. One colony was isolated and cultured in LB medium in duplicate. We added IPTG into one and cultured at room temperature. Then we observed  green fluorescence. (The concentration of IPTG was 0.1mM)
As a result, we confirmed that IPTG is able to induce expression in samples I-III. Regarding IV, a small amount of green fluorescence was detected in the sample without IPTG too.  We were not able to detect any green fluorescence in V whether IPTG was added or not.
Experiment 5
The result of experimant 5
Verification of the recombination activity of Cre protein
We assayed how the fluorescent intensity of GFP changes over time for sample 1, the sample we were able to induce in the previous assay..
I:The plasmid containing part 1-24 transformed to Rosetta (DE3) plysS
When the OD reached 0.6, we added IPTG to a final concentration of 0.1 mM.
We measured the fluorescence and graphed the results:
Result
We observed fluorescence without IPTG. However, fluorescence intensity was half compared to that after IPTG induction.
Experiment 6
The construct of experimant 6
We make four constructs. The first, “Aoi,” does not have a terminator between the promoter and the ribosome binding site. The second, “Murasaki,” has one terminator, and “Rokujo” has two terminators. “Char,” the fourth, expresses GFP when the lox sites are recombined. We combine these to make three units.
1. Aoi + Char unit
2. Murasaki + Char unit
3. Rokujo + Char unit
Cre protein is placed downstream of the RBS.
The terminator is placed between the two lox sites. GFP is expressed when terminators between the lox sites are excised by the Cre protein.
GFP is expressed when the terminators between the lox sites are excised by Cre protein.
A terminator exists between the promoter and the rbs, so cre is not translated. However, this terminator is leaky and transcribes a small amount of cre, so the terminators are  gradually excised and GFP is expressed.
Terminators exist between the promoter and the rbs, so cre is not translated. The terminator is not excised and GFP is not expressed.
We transformed E.coli containing Char, Aoi&Char, Murasaki&Char and Rokujo&Char to Rosetta(DE3) plyS.We cultured these and measured the fluorescence intensity of GFP after IPTG induction.
With Char, we were not able to observe GFP fluorescence because terminator is not excised in this construct.(negative control)
With Murasaki & Char, IPTG induces the expression of Cre and the terminators are gradually excised, so the ratio of plasmids able to produce GFP increases.
Therefore, the intensity of GFP fluorescence increases gradually.Murasaki & Char express Cre at low levels, so the terminator is excised infrequently.Therefore, the intensity of GFP fluorescence increases slowly.
With Rokujo & Char, Cre is seldom expressed, so the intensity of GFP should be similar to that of Char.
Result
The result of experimant 6
We were only able to observe fluorescence native to bacterial proteins. We deciphered this result as the following.
The sequence of our parts were identical to those reported on the registry, so the possibility of errors in our sequence was eliminated.
1. We used BBa_I71801（lox66）and (lox71)BBa_I718017 as our lox sites. There is the possibility that these sequences were dysfunctional.
2. We used BBa_J61047 as our Cre protein. There is the possibility that the sequence of this Cre protein was incorrect and Cre is dysfunctional.
Experiment 7
on-CMV plasmid map
Resulting band by electrophoresis
Verification of the activity of the lox sequence in our construct
We planned the following experiment to certify whether 1(in experiment 6) was true or not.
We dperformed experiments in vitro using purified the Cre protein. We enzymatically digested CMV and Char and adjusted the base length so that we could observe the base length alteration caused by Cre. CMV is a positive control previously reported to be recognized by Cre recombinase as a target of homologous recombination.
We cut this by Cre in vitro and observed the resulting band by electrophoresis.
As a result, we were able to confirm that Cre cuts on-CMV.
In contrast we were unable to observe a cut in the sample using Char.
Discussion
We concluded that lox66 and lox 71 incorporated  into Char are dysfunctional or have very low excision efficiency and furthermore,on-CMV plasmid can be a good read-out system for Cre activity assay.
Experiment 8planing
abstract
Evaluation of the Cre generating ability of the construct "Aoi"
Here we used a construct "on-CMV", which contains a GFP coding region sandwiched between effective loxP sites (about 1000bp). We ligased "on-CMV" and "Aoi"(BBa_K31008), then transformated this construct into T7 expression type of E.coli, Rosetta(DE3)pLysS, and observed if excision of 1000bp was occurred (this indicates the efficiency of "Aoi"). We extracted plasmids from once cultivated on-CMV - Aoi colony (excision was occurred during this course), and electrophoresis this to check the bands.
In addition to it, we used other Cre generating constructs (which were used in terminator assay) in the place of "Aoi", which may show the differences of the probability of terminator leak and, will be the basis of the 4C3 leak-switch which we are planning to construct.
Detail protocols
Experiment1
We examine whether IPTG can induce transcription and translation of downstream of T7promoter.
Method
1. Transform BL21(DE3) with 1-24 (T7promoter-GFPunit (figure)) , and cultivate on the agar medium on 37oC.
2. Inoculate bacteria.
3. Incubate the bacteria for 6 hours on 37oC
4. Add IPTG on one side. (Concentration of IPTG is 1mM)
5. Centrifuge culture at 14000rpm for 2min on 4oC.
6. Judge visually whether the pellet contains GFP.
Result
Downstream of T7promoter did not be transcribed or translated on the samples which contains IPTG.
On the contrary, GFP was found in non-IPTG samples.
Discussion
From this result, the following possibility was suggested. Green bacteria without the addition of IPTG → T7 RNA polymerase was expressed weakly and transcribed the mRNA coding gfp. White bacteria with the addition of IPTG → Overexpressed GFP made inclusion body.
Experiment2
Use less concentration of IPTG and observed whether gfp of BL21(DE3) is expressed.
Method
1. Prepare Full-grown BL21(DE3)  which is transformed with 1-24 .
a. Full growth 300ul ; LB browth 1700ul ;  ampicillin 100ng/ml ;  IPTG 0.5mM
b. Full growth 300ul ; LB browth 1700ul ;  ampicillin 100ng/ml ;  IPTG 0.1mM
c. Full growth 300ul ; LB browth 1700ul ;  ampicillin 100ng/ml ;  IPTG 0.05mM
d. Full growth 300ul ; LB browth 1700ul ;  ampicillin 100ng/ml ;  IPTG 0.0mM
2. Incubate a-d for 20hours on 37oC.
3. Centrifuge culture at 14000rpm for 2min on 4oC.
4. Judge visually whether the pellet contains GFP.
Result
The GFP fluorescence of both samples was measured and there was no difference between the two samples.
Discussion
It is suggested that BL21(DE3) is not proper to control the gene expression.  We desided to use Rossetta (DE3) pLysS that can repress the activity of the leaked T7 RNA polymerase.
Experiment4
We execute the almost same method as exp1. This time, we use Rosetta(DE3) instead of BL21(DE3). We used full grown samples preserved in glycerol.
Method
1. Transform Rosetta (DE3) with 1-24 (T7promoter-GFPunit), and cultivate on the agar medium on 37oC.
2. Inoculate bacteria. (5samples, I-V)
3. Incubate the bacteria on 37oC till reach full growth.
4. Stock full grown I-V in glycerol.
5. Add I-V to 2mL of LB browth (with Ampicillin 100ng/ml)
6. Divide each sample into 2 other test tube.
7. Incubate the bacteria in shaking incubator on 37oC for 6 hours.
8. Add IPTG one side of each pair of samples (I-V). (Concentration of IPTG is 1mM)
9. Incubate the bacteria in shaking incubator for 12 hours on room temperature.
10. Centrifuge culture at 14000rpm for 2min on 4oC.
11. Judge visually whether the pellet contains GFP.
Result
As a result, we concluded that IPTG is able to induce expression in samples I-III. As for IV, a small amount of green fluorescence was detected in the sample without IPTG too. We were not able to detect any green fluorescence in V whether IPTG was added or not.
Experiment5
We observed the green fluorescence by measuring the amount of GFP expression in samples induction was confirmed. In this observation, we confirmed the induction of IPTG again.
I : The plasmid containing part 1-24 transformed to Rosetta (DE3) plysS
Method
Prepare E.coli that contains the following two constructs:
 pSB1A2/  1-24(T7promoter-gfp unit)
 pSB1Ak3/ 1-4(T7promoter-double terminator)
1. Transform  1-4 plasmids to Rosetta(DE3) plysS competent cells. Cultivated pellet stocked in glycerol and the 1-4 colony transformed in 1.(15 ml LB broth in 200ml)
2. Incubate sample at 37 oC with constant shaking at 180 rpm for more than 12 hours (until the medium become the state of full growth).
We made some samples as follows and cultivated them at room temperature.
1 Add LB medium 49.5ml into 200ml flask(amp100ng/ml)1-24full growth culture 0.5ml
2 Add LB medium 49.5ml into 200ml flask(amp100ng/ml)1-24full growth culture 0.5ml
3 Add LB medium 49.5ml into 200ml flask(amp100ng/ml)1-24full growth culture 0.5ml
4 Add LB medium 49.5ml into 200ml flask(amp100ng/ml)1-24full growth culture 0.5ml
5 Add LB medium 49.5ml into 200ml flask(amp100ng/ml)1-24full growth culture 0.5ml
6 Add LB medium 49.5ml into 200ml flask(amp100ng/ml)1-4full growth culture 0.5ml
7 Add LB medium 49.5ml into 200ml flask(amp100ng/ml)1-4full growth culture 0.5ml
We added IPTG only to 1-3,6,7 to be 0.1mM when OD became 0.6, 350min after cultivation started.  The results were as follows.
After the induction of IPTG, we take 1000ml samples every hour. We made these samples pellet by centrifuging for 2min on 14000rpm for 4 times.
Then we measured the strength of fluorescence of pellets by the following protocols.
1. Add 100 ul 8M urea buffer into each pellet and make suspension solution.
2. Incubate sample solutions at room temperature for 30 minutes.
3. Sonicate the cells. (10 seconds, 10% power.)
4. Cool the samples on ice and cool it down.
5. Repeat steps 3 and 4.
6. Centrifuge the tube at 150rpm, 2 minutes, 4 oC.
7. Measure the fluorescence.
Result
The number of E.coli varies by each sample. To correct this, we divided this fluorescence by OD. Then we made the graph of these results. (Figure)
Figure gfp induced : Add IPTG when OD is 0.6（0.1mM） gfp uninduced:  non-IPTG Background: non-gfp
Discussion
We observed fluorescence without IPTG. However, fluorescence intensity was half compared to that after IPTG induction. So, we could surely confirm the expression induction of IPTG. Furthermore, we could conclude that intensity of GFP correlate with time.
Experiment6
As for Experiment6, Method and Result have already detailed and discussed in above.
Please refer to "Abstract".
Experiment7
materials
cre recombinase
x10 cre buffer
Hikari/pSB1C3…BBa_  concentration ng/ul  / date:
onCMV plasmid 86ng/ul
1%agarose gel
x10Mbuffer
XbaI
PstI
EcoRI
Promega Wizard SV Gel and PCR Clean-Up System
Procedure
Linearization of plasmids
mixing following reagents and incubation under 37℃
I. Hikari plasmid 3.8ul(ngDNA)
x10Mbuffer 3ul
XbaI  1ul
PstI  1ul
MilliQ  22.2ｕl
Total  30ul
II. onCMV plasmid 1ul(ngDNA)
x10Hbuffer 2ul
EcoRI  1ul
MilliQ  16ul
Total  20ul
After incubated for several hours, 1uL of each was checked by 1% agarose gel electrophoresis. Since having got expected band patterns (following result session in detail), we purified DNA fragments under column purification, used 50 uL of Nuclease-free water for elution. We recorded UV absorbance spectrum and concentration.
Result Following DNA fragments (purified) was extracted under above procedures.
Hikari XPcut fragments  9.4ng/ul
onCMV Ecut fragments    42.3ng/ul
Band patterns of electrophoresis
OnCMV-- 5300bp
Hikari-- 2000bp, 1150bp and faint 3100bp band (indicated one-cut fragments)
(no image)
These were used for next recombination process.
Recombination procedure
We mixed following solutions and incubated under 37℃
Mix I Hikari-XPcut 3.8ul
x10cre buffer 3ul
cre recombinase 1ul
MilliQ  22.2ul
total  30ul
Mix II Hikari-XPcut 3.8ul
x10cre buffer 2ul
cre recombinase 2ul
MilliQ  21.2ul
Total  30ul
Mix III onCMV-Ecut 3.8ul
x10cre buffer 2ul
cre recombinase 2ul
MilliQ  22.2ul
total  30ul
After incubated for several hours, added x10loding dye and checked by electrophoresis
The order of loading was:
lambda-Hmarker / onCMV-Ecut / hikariXPcut /  Mix I/ MixII/ MixIII
Experiment8planning
abstract
Evaluation of the Cre generating ability of the construct "Aoi"
Here we used a construct "on-CMV", which contains a GFP coding region sandwiched between effective loxP sites (about 1000bp). We ligased "on-CMV" and "Aoi"(BBa_K31008), then transformated this construct into T7 expression type of E.coli, Rosetta(DE3)pLysS, and observed if excision of 1000bp was occurred (this indicates the efficiency of "Aoi"). We extracted plasmids from once cultivated on-CMV - Aoi colony (excision was occurred during this course), and electrophoresis this to check the bands.
In addition to it, we used other Cre generating constructs (which were used in terminator assay) in the place of "Aoi", which may show the differences of the probability of terminator leak and, will be the basis of the 4C3 leak-switch which we are planning to construct.
Copyright © 2010 iGEM UT-Tokyo. All rights reserved.
Site map
Retrieved from " http://2010.igem.org/Team:UT-Tokyo/Sudoku_assay_LeakSw "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
