<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NRP-UEA-Norwich_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NRP-UEA-Norwich/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>NRP-UEA-NORWICH iGEM</TITLE></P><DIV id="container"><DIV id="menu"><UL class="unselectable"><LI style="width:80px;"><DIV><A href="https://2016.igem.org/Team:NRP-UEA-Norwich">Home</A></DIV></LI><LI style="width:80px;"><DIV><A href="https://2016.igem.org/Team:NRP-UEA-Norwich/Team">Team</A></DIV></LI><LI style="width:80px;"><DIV><A href="#">Project</A></DIV></LI><LI><A href="https://2016.igem.org/Team:NRP-UEA-Norwich/Background">Background</A></LI><LI><A href="https://2016.igem.org/Team:NRP-UEA-Norwich/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:NRP-UEA-Norwich/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:NRP-UEA-Norwich/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:NRP-UEA-Norwich/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:NRP-UEA-Norwich/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:NRP-UEA-Norwich/GoldenGate">Golden Gate</A></LI></UL><LI style="width:80px;"><DIV><A href="https://2016.igem.org/Team:NRP-UEA-Norwich/Parts">Parts</A></DIV></LI><LI style="margin-left:-15px;"><DIV><A href="https://2016.igem.org/Team:NRP-UEA-Norwich/Collaborations">Collaborations</A></DIV></LI><LI><DIV><A href="https://2016.igem.org/Team:NRP-UEA-Norwich/Attributions">Attributions</A></DIV></LI><LI><DIV><A href="https://2016.igem.org/Team:NRP-UEA-Norwich/Integrated_Practices">Human Practices</A></DIV></LI><LI style="width: 120px;"><DIV><A href="https://2016.igem.org/Team:NRP-UEA-Norwich/Awards">Achievements</A></DIV></LI></DIV><DIV id="contentWrapper"><DIV id="timeline"><DIV class="pageSection"><H2><SPAN class="mw-headline" id="Notebook">Notebook</SPAN></H2><P><SPAN style="color:blue; text-decoration;"><B>Group 1: Cloning</B></SPAN></P><P><SPAN style="color:green; text-decoration;"><B>Group 2: Protein and Conjugation/Transformation into <I> Shewanella oneidensis</I></B></SPAN></P><P><SPAN style="color:orange; text-decoration;"><B>Group 3: Electrochemistry</B></SPAN></P><DIV class="timeline_vline"><DIV class="timeline_bubble"><DIV class="timeline_event_header" data-side="right" style="z-index: 14;"><DIV class="timeline_event_header_container"><P>						Week 1 4.07-8.07
</P></DIV><DIV class="timeline_event_container"><DIV class="timeline_event_inner"><P><SPAN style="color:blue; text-decoration;">
							At the start of the week, team members completed a safety induction for the lab. The first experiment used the iGEM standard RFP BioBrick (BBa_J04450) to transform <I>E. coli</I>. For our planned outreach activity, seven coloured BioBricks from the iGEM 2016 distribution kit were identified. <I>E. coli</I> was transformed with these BioBricks, and then plated to determine which cells had been transformed successfully. </SPAN></P></DIV></DIV></DIV></DIV><DIV class="timeline_bubble"><DIV class="timeline_event_header" data-side="left" style="z-index: 13;"><DIV class="timeline_event_header_container"><P>						Week 2 11.07-15.07
</P></DIV><DIV class="timeline_event_container"><DIV class="timeline_event_inner"><P><SPAN style="color:blue; text-decoration;"> Successful transformants were selected and used to inoculate LB media, these samples were to be used for the children's Summer School. Consequently, they were mini prepped to obtain the plasmid DNA. Alongside this, the PCR protocol was optimised by experimenting with annealing temperatures. The <I>HydC/Fdh, HydA, HydB</I> C- and N-terminus genes and RFP BioBricks (Bba_J04450) were digested with <I>EcoR1</I> and <I>Pst1</I>. Samples were run on a gel to observe complete digestion and then extracted and purified. </SPAN></P></DIV></DIV></DIV></DIV><DIV class="timeline_bubble"><DIV class="timeline_event_header" data-side="right" style="z-index: 12;"><DIV class="timeline_event_header_container"><P>						Week 3 18.07-22.07
</P></DIV><DIV class="timeline_event_container"><DIV class="timeline_event_inner"><P><SPAN style="color:blue; text-decoration;"> For the Summer School: Blue, Pink, RFP (Red Fluorescent Protein) and GFP (Green Fluorescent Protein) colonies were inoculated. Various drawing utensils were tested on plates to determine which one would be best for the children to use. After speaking to the student leader of UCC (<SPAN class="plainlinks"><A rel="nofollow" class="external text" href="https://2016.igem.org/Team:UCC_Ireland">iGEM Cork</A></SPAN>) it was decided to go with cotton buds. For our main project, the cleaned-up PCR products were run on a gel to check the PCR amplification was successful. 2.5µL DMSO was added to the <I>HyaC</I> N- and C-terminus DNA to minimise non-specific amplification, but did not make any difference. The FeFe gene inserts were digested along with RFP to obtain the BioBrick vector. Gel extraction and purification of the gene inserts yielded low concentrations, these were repeated. </SPAN></P></DIV></DIV></DIV></DIV><DIV class="timeline_bubble"><DIV class="timeline_event_header" data-side="left" style="z-index: 11;"><DIV class="timeline_event_header_container"><P>						Week 4 25.07-29.07
</P></DIV><DIV class="timeline_event_container"><DIV class="timeline_event_inner"><P><SPAN style="color:blue; text-decoration;"> Vials of the ‘paint’ and cotton buds were prepared for the human practices activity. GFP, RFP, Pink and Blue were inoculated and grown overnight to use the following day for the activity. The PCR amplification was repeated for FeFe gene inserts. These were then digested, extracted and purified from the agarose gel. FeFe genes inserts were ligated with the BioBrick vector. The ligations reactions were set up with different ratios of vector and insert (1:1 and 1:3) and incubated  overnight. These were transformed into <I>α-select</I> Gold Efficiency <I>E.coli</I> cells by heat shock the following day and plated out. Four plates showed colonies and successful ligations i.e. <I>HydC/Fdh</I> 1:1, <I>HydC/Fdh</I> 1:3, <I>HydA</I> 1:3 and <I>HydB</I> C-terminus 1:3. Digestion reactions were set up for the NiFe genes: <I>HyaC</I>, <I>HyaA</I> and <I>HyaB</I> N- and C-terminus followed by ligations. </SPAN></P></DIV></DIV></DIV></DIV><DIV class="timeline_bubble"><DIV class="timeline_event_header" data-side="right" style="z-index: 10;"><DIV class="timeline_event_header_container"><P>						Week 5 1.08-5.08
</P></DIV><DIV class="timeline_event_container"><DIV class="timeline_event_inner"><P><SPAN style="color:blue; text-decoration;">The successful colonies for <I>HydC/Fdh, HydA</I> and <I>HydB</I> C-terminus were inoculated in 10ml LB  overnight. Following mini-prep plasmid purification, PCR confirmed correct ligations. The gel results showed unexpected sizes for some of the samples and suggested that some of the samples had been mis-labelled. DNA from new overnight cultures of <I>HydC/Fdh, HydA</I> and <I>HydB</I> C-terminus were obtained by mini-prep and a restriction digest was carried out to verify the results. Digests of <I>HydB</I> N-terminus, <I>HyaC, HyaA, HyaB</I> N-terminus and <I>HyaB</I> C-terminus were set up, however following gel purification concentrations were too low, thus transformations did not yield results. </SPAN></P></DIV></DIV></DIV></DIV><DIV class="timeline_bubble"><DIV class="timeline_event_header" data-side="left" style="z-index: 9;"><DIV class="timeline_event_header_container"><P>						Week 6 8.08-12.08
</P></DIV><DIV class="timeline_event_container"><DIV class="timeline_event_inner"><P><SPAN style="color:blue; text-decoration;"> We repeated the PCR amplification of FeFe units along with NiFe genes inserts. With new primers for <I>HyaB</I> N- and C-terminus gene insert. Gel showed successful amplification of FeFe gene inserts and <I>HyaB</I> N-terminus. The FeFe subunits were digested alongside samples of RFP DNA. Samples were extracted and purified from the gel, however nanodrop results showed extremely low concentration for <I>HydA</I>. It was decided this would be repeated. Afterwards, <I>HydA</I> and <I>HydB</I> N-terminus were ligated into the pSB1C3 vector and frozen for later transformation. More J04450 was inoculated and mini-prepped as a source of vector. <I>HydC/Fdh, HydB</I> C-terminus and <I>HyaA</I> was successfully ligated into pSB1C3 and digest confirmed a mix-up in sample labels however bands did correspond to correct gene inserts. </SPAN></P><P><SPAN style="color:green; text-decoration;"> A practice tri-parental conjugation trial was conducted following protocol, using samples obtained from a PhD student in the lab </SPAN></P><P><SPAN style="color:orange; text-decoration;"> We conducted experiments to practise growing bacteria for creating anaerobic growth curves and to practise using the gas chromatograph. We grew wildtype MR-1 and double knock out (∆<I>HydABC,HyaABC</I>) <I>Shewanella</I> strains in anaerobic conditions for 24 hours in hungate tubes then analysed the headspace gases using gas chromatography to calculate the hydrogen yield. </SPAN></P></DIV></DIV></DIV></DIV><DIV class="timeline_bubble"><DIV class="timeline_event_header" data-side="right" style="z-index: 8;"><DIV class="timeline_event_header_container"><P>						Week 7 15.08-19.08
</P></DIV><DIV class="timeline_event_container"><DIV class="timeline_event_inner"><P><SPAN style="color:blue; text-decoration;">Due to the uncertainty of the identity of these potential BioBricks, <I>HydC/Fdh, HydB</I> N-terminus and <I>HyaA</I> plasmid DNA samples were sent for DNA sequencing confirmation. The results proved that these samples were successful BioBricks, but that samples had been mislabelled, samples were subsequently correctly labelled. The frozen samples of ligated <I>HydA</I> and <I>HydB</I> N-terminus were transformed successfully into <I>α-select E. coli</I> and incubated overnight. A colony from each was used to inoculate LB, incubated overnight and mini-prepped the following day. </SPAN></P><P><SPAN style="color:green; text-decoration;">We conducted a practice arabinose induced protein expression trial using samples from a PhD student in the lab. SDS-PAGE gels were set up and used to practice analysing protein expression. The tri-parental conjugation of RFP (J04450) Biobrick into <I>S. oneidensis</I> was set up for the following week. </SPAN></P><P><SPAN style="color:orange; text-decoration;"> We grew MR1 and ∆<I>HydABC,HyaABC</I><I>Shewanella</I> strains in anaerobic conditions for 24 hours in hungate tubes and analysed the amount of hydrogen produced using gas chromatography. This was to serve as a control value for our later experiments. </SPAN></P></DIV></DIV></DIV></DIV><DIV class="timeline_bubble"><DIV class="timeline_event_header" data-side="left" style="z-index: 7;"><DIV class="timeline_event_header_container"><P>						Week 8 22.08-26.08
</P></DIV><DIV class="timeline_event_container"><DIV class="timeline_event_inner"><P><SPAN style="color:blue; text-decoration;">Golden Gate Cloning of FeFe Hydrogenase subunits was now possible. The use of the arabinose inducible pBAD vector allowed controlled expression of the FeFe Hydrogenase. Additionally, using pBAD allowed us to get around the reported issues of <I>S.oneidensis</I> not being compatible with pSB1C3. <SPAN class="plainlinks"><A rel="nofollow" class="external text" href="https://2008.igem.org/Team:Harvard">iGEM Harvard 2008</A></SPAN> and <SPAN class="plainlinks"><A rel="nofollow" class="external text" href="https://2015.igem.org/Team:NTU-Singapore">NTU Singapore 2015</A></SPAN> have reported similar issues in conjugating BioBricks into <I>S.oneidensis</I> due to conflicts in origin of replication. This was also reported in a paper “Replication of plasmids with the p15A origin in <I>Shewanella putrefaciens</I>” (Myers and Myers ,1997). The Golden Gate cloning for FeFe hydrogenase (<I>Fdh/HydC, HydA</I> and <I>HydB</I> N-terminus) cluster was successful, allowing us to characterise our construct in <I>S.oneidensis</I>. Only <I>HyaC</I> and <I>HyaB</I> N-terminus were PCR amplified successfully, and cloned into pSB1C3. </SPAN></P><P><SPAN style="color:green; text-decoration;">Tri-parental conjugation of RFP (J04450) and GFP (K584001) BioBricks into S. oneidensis was attempted but was unsuccessful. </SPAN></P></DIV></DIV></DIV></DIV><DIV class="timeline_bubble"><DIV class="timeline_event_header" data-side="right" style="z-index: 6;"><DIV class="timeline_event_header_container"><P>						Week 9 29.08-2.09
</P></DIV><DIV class="timeline_event_container"><DIV class="timeline_event_inner"><P><SPAN style="color:blue; text-decoration;">We obtained the <I>HydABC</I> FeFe hydrogenase gene cluster in our expression vector: pBAD, from the Golden Gate cloning. This was then transformed into <I>E.coli</I> and the overnight culture mini-prepped to obtain DNA, which was sent off for sequencing. Some of the remaining DNA was run on a gel against the pBAD vector to observe correct ligation. <I>HydABC</I> in our expression vector, pBAD, was transformed back into <I>α-select E.coli</I> for more cells. The NiFe gene inserts were transformed and plated, but was unsuccessful. Additional RFP digests showed high amounts of smearing, thus an overnight digest was done and used to ligate the NiFe gene inserts.</SPAN></P><P><SPAN style="color:green; text-decoration;">Wild Type <I>S. oneidensis</I> MR-1, ∆<I>HydABC,HyaABC</I> (NiFe and FeFe hydrogenase double knockout), <I>LS498</I> (FeFe hydrogenase knockout) and <I>pRK2013</I> (<I>E.coli</I> helper strain) were inoculated in preparation for transforming the plasmid containing <I>HydABC</I> into <I>S. oneidensis</I>. Triparental conjugation was then performed to transform 8A5 (pBAD vector with <I>HydABC</I> C-terminally tagged) and 9A5 (pBAD vector with <I>HydABC</I> N-terminally tagged) into all three strains of <I>S. oneidensis</I>. </SPAN></P></DIV></DIV></DIV></DIV><DIV class="timeline_bubble"><DIV class="timeline_event_header" data-side="left" style="z-index: 5;"><DIV class="timeline_event_header_container"><P>						Week 10 5.09-9.09
</P></DIV><DIV class="timeline_event_container"><DIV class="timeline_event_inner"><P><SPAN style="color:blue; text-decoration;">Ligated <I>HyaA</I> and <I>HyaB</I> N-terminus did not yield any colonies. Stocks of DNA were made by inoculating and mini-prepping more samples of the <I>HydABC</I>. </SPAN></P><P><SPAN style="color:green; text-decoration;">The triparental conjugation was continued from last week, however there were no <I>S. oneidensis</I> colonies containing the plasmid. Attempted electroporation to deliver the plasmid with <I>HydABC</I> into <I>S. oneidensis</I>. The first attempt did not yield any results, but the second attempt gave colonies of <I>S.oneidensis</I> MR-1 and ∆<I>HydABC,HyaABC</I>strains <I>S. oneidensis</I> with both 8A5 and 9A5. </SPAN></P><P><SPAN style="color:orange; text-decoration;">We grew∆<I>HydABC,HyaABC</I> and wildtype MR-1 strains for 24 hours in anaerobic conditions. We then used an electrochemical cell setup to analyse the activation of the hydrogenase enzymes, and the difference between the two strains. This would then serve as a control for later experiments. </SPAN></P></DIV></DIV></DIV></DIV><DIV class="timeline_bubble"><DIV class="timeline_event_header" data-side="right" style="z-index: 4;"><DIV class="timeline_event_header_container"><P>						Week 11 12.09-16.09
</P></DIV><DIV class="timeline_event_container"><DIV class="timeline_event_inner"><P><SPAN style="color:blue; text-decoration;">Attempted to start again with NiFe catalytic subunit cloning. PCR amplified with new primers and digested with a new source of <I>EcoR1</I> and <I>Pst1</I> as digests with our enzymes showed high amounts of smearing. Digested gel showed a band for <I>HyaB</I> N-terminus but not for <I>HyaB</I> C-terminus. Ligations, at ratios of 1:3, 1:5 and 1:7, were set up to be transformed the following week. Alongside <I>HyaB</I> N- and C-terminus, it was decided to excise <I>HydABC</I> out of the pBAD vector and ligate that into pSB1C3.</SPAN></P><P><SPAN style="color:green; text-decoration;">Mini-prepped colonies of the electroporation were sent off for sequencing. Results showed that only wild type <I>S. oneidensis</I> MR-1 has successfully taken up the <I>HydABC</I> C-terminally tagged i.e. 8A5, construct. Triparental conjugation and electroporation was continued with other strains but did not yield results. Protein expression trials (induced with 0mM, 1mM and 10mM arabinose) of the WT with 8A5, <I>E.coli</I> with 8A5 and 9A5 were performed using SDS-PAGE and western blot techniques. </SPAN></P></DIV></DIV></DIV></DIV><DIV class="timeline_bubble"><DIV class="timeline_event_header" data-side="left" style="z-index: 3;"><DIV class="timeline_event_header_container"><P>						Week 12 19.09-23.09
</P></DIV><DIV class="timeline_event_container"><DIV class="timeline_event_inner"><P><SPAN style="color:blue; text-decoration;">NiFe Hydrogenase subunits and <I>HydABC</I> were ligated and transformed into <I>E.coli</I> cells. All plates had colonies, however the digested gel samples showed that our insert was not in them this was most likely due to incomplete digestion of Bba_J04450. The DNA concentrations of <I>HyaB</I> C- and N-terminus were low. Thus, we decided to order new DNA, in order to re-attempt cloning of <I>HyaB</I> C- and N-terminus into pSB1C3 </SPAN></P><P><SPAN style="color:green; text-decoration;">The triparental conjugations and electroporations from the previous week did not give any positive results. Protein expression trials (induced with 0mM, 1mM and 10mM arabinose) using SDS-PAGE and Western Blot were repeated for Mr1:<I>HydABC</I> and <I>E.coli</I> with 8A5 and 9A5. </SPAN></P></DIV></DIV></DIV></DIV><DIV class="timeline_bubble"><DIV class="timeline_event_header" data-side="right" style="z-index: 2;"><DIV class="timeline_event_header_container"><P>						Week 13 26.09-30.09 
</P></DIV><DIV class="timeline_event_container"><DIV class="timeline_event_inner"><P><SPAN style="color:blue; text-decoration;">The digests of RFP were repeated to obtain a source of vector for new ligations for NiFe gene inserts and <I>HydABC</I> cluster. There were transformed into <I>E.coli</I> cells. All plates showed the ligations were successful as colonies had grown. The colonies were used to inoculate LB and mini prepped to perform a diagnostic colony PCR. Results showed correct ligations of all NiFe gene inserts, yielding more BioBricks. </SPAN></P></DIV></DIV></DIV></DIV><DIV class="timeline_bubble"><DIV class="timeline_event_header" data-side="left" style="z-index: 1;"><DIV class="timeline_event_header_container"><P>						Week 14 3.10-8.10
</P></DIV><DIV class="timeline_event_container"><DIV class="timeline_event_inner"><P><SPAN style="color:orange; text-decoration;">We grew wildtype MR-1 and MR-1 FeFe overexpression strains (our modified strain i.e. <I>HydABC</I> ligated into pBAD by Golden Gate cloning) in anaerobic conditions with varying concentrations of arabinose added. This showed the difference in expression between arabinose concentrations and also the effect of adding our FeFe <I>HydABC</I> hydrogenase cluster into the bacteria. This data could also be compared to MR1 without arabinose.</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="sponContainer" class="pageSection"><H2>Sponsors</H2></DIV><DIV id="footer">
		Copyright UEA 2016 . All rights reserved.
		</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>