Team:NRP-UEA-Norwich/Results/ProteinModeling
NRP-UEA-NORWICH iGEM
Home
Team
Project
Background
Safety
Demonstrate
Notebook
Results
Protocols
Golden Gate
Parts
Collaborations
Attributions
Human Practices
Achievements
PROTEIN MODELING
Aim
Apart from MtrC , the crystal structures of the hydrogenase and Mtr proteins are currently unknown. To help the general public to understand our project in a visual way using virtual reality, on posters and presentations, we decided to model the proteins.
Method
Genbank was used to find the sequence of the gene for each of the proteins. The DNA sequence was translated into an amino acid sequence using the web programme ExPASy. All modelling was carried out using the Phyre2 software, this works by aligning the amino acid sequence of the protein to be modelled against other proteins with solved structures in the Protein Data Bank.  For the fumarate reductase FccA and small tetraheme cytochrome (STC), the existing PDB files were used for the figure.
Results
Although the sequence identity and coverage was low for many of the proteins (see table), the models are the most accurate possible approximations of these proteins available through homology modelling.
Table 1. Details of the structures Phyre2 used, the proteins used are the most accurate from the Phyre2 software.
Protein name
Organism and protein model used
PDB file
% Sequence identity
% Sequence coverage
MtrA
Wolinella succinogenes octaheme sulfite reductase MccA, form I
4RKM
14
76
MtrB
Pseudomonas aeruginosa membrane bound ferripyoverdine receptor
2IAH
10
91
MtrC
Shewanella oneidensis MR-1 crystal structure of outer membrane decaheme cytochrome MtrC
4LM8
100
93
HyaA
Escherichia coli hydrogenase-1 complexed with cytochrome b
4GD3
46
79
HyaB
Salmonella enterica NiFe hydrogenase
4C3O
45
96
HyaC
Escherichia coli formate dehydrogenase
1KQF
16
86
HydA
Desulfovibrio desulfuricans Fe-only hydrogenase
1HFE
49
93
HydB
Desulfovibrio desulfuricans small Fe-only subunit
1HFE
39
58
HydC
Escerichia coli formate dehydrogenase
1KQF
22
91
References
ExPASy- Gasteiger E., Gattiker A., Hoogland C., Ivanyi I., Appel R.D., Bairoch A. ExPASy: the proteomics server for in-depth protein knowledge and analysis Nucleic Acids Res. 31:3784-3788 (2003).
Genbank - Nucleic Acids Research, 28: 235-242. (www.rcsb.org)
PDB - H.M. Berman, J. Westbrook, Z. Feng, G. Gilliland, T.N. Bhat, H. Weissig, I.N. Shindyalov, P.E. Bourne (2000) The Protein Data Bank
Phyre2 - The Phyre2 web portal for protein modelling, prediction and analysis. Kelley LA et al. Nature Protocols 10, 845-858 (2015)
Pymol - The PyMOL Molecular Graphics System, Version 1.8 Schr√∂dinger, LLC.
Sponsors
Copyright UEA 2016 . All rights reserved.
