Team:NRP-UEA-Norwich/GoldenGate
NRP-UEA-NORWICH iGEM
Home
Team
Project
Background
Safety
Demonstrate
Notebook
Results
Protocols
Golden Gate
Parts
Collaborations
Attributions
Human Practices
Achievements
Golden Gate Cloning
Conventional cloning methods utilise restriction enzymes that recognise symmetrical or palindromic DNA cut sites. This site is then cleaved, exposing sticky ends for use in re-ligation. Our basic BioBricks containing the subunit proteins for our hydrogenases were formed using this method. These genes are naturally found in a single gene cluster in Shewanella oneidensis, transcribed together to ensure that the subunits are expressed at the same level.
Once we had successfully cloned our individual subunit parts for each hydrogenase into the BioBrick vector, we had to combine these genes into a single vector under the same promoter. This meant removing the inserts from pSB1C3, connecting them in the correct order, and re-ligating them into our vector. To do this we used an alternative method known as ‘Golden Gate cloning’
Golden gate cloning utilises type IIS instead of type IIP restriction enzymes. These are restriction enzymes that recognise a DNA sequence outside of the cut site. A cut site which is non-specific, allowing for custom overhangs. This means that during ligation between the inserts and their vector, the recognition site for these enzymes will be removed, eliminating the scaring that occurs during cloning with conventional restriction enzymes. In designing our individual gene parts, it was important to include cut sites within our BioBrick prefix/suffix that would allow for ligation of the parts in the correct order in the final gene cluster; while ensuring that no enzyme recognition sites were present within the sequence between cut sites.
Preparation for this method started with our design for gene synthesis by IDT. Restriction enzyme sites were added to either side of each individual subunit gene that would result in complementary sticky ends for adjacent genes. Ensuring they would be re-ligated in the correct order of the cluster. Golden Gate recognition sites for the BsaI enzyme were removed from internal sites by base substitutions that did not alter the amino acid sequence. In each gene part the iGEM BioBrick prefix and suffix was also included.
For example, once cloning of the FeFe hydrogenase subunit genes HydC, HydB and HydA into BioBricks was complete, we had to create our FeFe gene cluster. Due to cleavage at the golden gate sites removing the recognition sequence, we were able to undergo the whole process of digesting and re-ligating with all three BioBricks in a one-pot reaction.
Alongside the personalised cut sites, primers were designed. These primers contained additional bases that are complimentary to the consecutive gene insert, including the BsaI cut site. This ensures correct annealing and ligation of our inserts into the expression vector.
By cycling the temperature of the reaction between 16⁰C for digestion and 37⁰C for ligation we maximised the chances of the full cluster being formed. This vector was transformed into E. coli (transformation into Shewanella via conjugation requires transformed E. coli) and successful colonies were observed. Inoculations were made, incubated, mini-prepped and our purified constructed vector was DNA sequenced to confirm our cloning was successful.
References
Kirchmaier S, Lust K, Wittbrodt J (2013) ‘Golden GATEway Cloning – A Combinatorial Approach to Generate Fusion and Recombination Constructs’. PLoS ONE 8(10): e76117. doi:10.1371/journal.pone.0076117
Sponsors
Copyright UEA 2016 . All rights reserved.
