"
Page
Discussion
View source
History
teams
Log in
 
Team:Bilkent UNAM Turkey/Experiment
From 2011.igem.org
We ordered synthetic nfsI gene from a company. We optimized codon usage because of gene taken from Enterobacter cloacae.
Codon Usage . We optimized our nfsI gene according to Chlamydomonas reinhardtii codon bias table provided by DNA 2.0. This program enables to change synonymous codons without changing amino acid sequence. You can also check the restriction sites while optimizing codons.  We also avoided from forbidden restriction sites (EcoRI, XbaI, SpeI, PstI) therefore; we have chosen second frequently used codons for 34., 35., 38. and 45. amino acids.
1.Gene Designer DNA 2.0.
Then we did restriction digestion and gel electrophoresis .
This image is the best result of our trials
2.The picture of restricted with EcoRI and PstI, nfsI gene.
3.Our nfsI gene design. It contains XhoI and BamHI therefore; we can ligate into pRbcBRL protein expression vector.
4.Confirmation of Ligation nfsI gene with pRbcBRL backbone. This pRbcNFSI construct was done because we have received our biobrick prefix and suffix gene lately as a result of an ordering problem. We need to go on transfection into C. reinhardtii and TNT tolerance and biodegradation capacity of algae
We used standard gel purification kit Our ligation style is really similar to iGEM's one.
Our one;
• insert volume(µl):X
• Cut Vector Volume(µl):X
• DEPC-water(µl):8,5-2X
• T4 DNA ligase(µl):0,5
• Ligation Buffer(µl):1
• Total Volume(µl):10
We transformed our plasmid into E.coli with a protocol.
We used invitrogen purification kit to get our plasmid. We sent genes to sequencing and sequencing data should be available on their link .
Polymerase Chain Reaction  of GFP from pKScGFP
For 1.0µl of DNA we added,
• 1.0µl of forward primer
• 1.0µl of reverse primer
• 2.5µl of 10X PCR Buffer
• 0.5µl of MgSO4
• 0.5µl of dNTP mix
• 18.375µl of ddH2O
• 0.125µl of Taq polymerase
Parameters in thermal cycler were chosen as below:
Denaturation is 95oC for 4min
Annealing is 30 cycles:
• 95oC 1min
• 65oC 1min
• 72oC 2min
Extension 72oC for 10min
4oC hold
GFP fusion
GFP fusion with nfsI gene on our Biobrick is our plannned future work.
Restriction Enzyme Digestion
We obtained nfsI gene synthetically with prefix and suffixes. Synthetic nfsI gene was restriction digested via XhoI/BamHI to create sticky ends at prefix and suffixes.
We used pRbcRL as our backbone and as an algal expression plasmid. We restriction digested luciferase gene via XhoI/BamHI restriction enzymes and created a cut vector backbone with sticky ends.
DNA ligation
Then we ligated nfsI gene to the cut vector of pRbcRL (pRbc plasmid backbone) with insert:backbone ratios of 10:1, 3:1 and 1:1. All of them gave good results.
For 10:1 nfsI:pRbc ligation tube,
Solution
Volume (µl)
pRbc backbone
1.7
Insert nfsI
16.1
T4 ligase buffer (10X)
1
T4 DNA ligase
0.5
ddH2O
0.7
Total
20
For 3:1 nfsI:pRbc ligation tube,
Solution
Volume (µl)
pRbc backbone
1.7
Insert nfsI
4.8
T4 ligase buffer (10X)
1
T4 DNA ligase
0.5
ddH2O
2
Total
10
For 1:1 nfsI:pRbc ligation tube,
Solution
Volume (µl)
pRbc backbone
1.7
Insert nfsI
1.6
T4 ligase buffer (10X)
1
T4 DNA ligase
0.5
ddH2O
5.2
Total
10
Competent Escherichia coli DH5α were transformed with pRbcnfsI ligated products and plated to LB ampicillin plates.
Colonies were collected after overnight incubation at 37oC, and miniculture was started from each colony. Plasmid DNA isolation was performed by using PureLink™ Quick Plasmid Miniprep Kit of InvitrogenTM.
Nanodrop DNA concentrations after minipreps were as below:
Nucleic Acid Conc.
Unit
A260
A280
260/280
260/230
Sample Type
361.4
ng/µl
7.228
4.334
1.67
1.82
DNA
384.9
ng/µl
7.698
4.554
1.69
1.82
DNA
558.7
ng/µl
11.174
7.232
1.55
1.65
DNA
We proceeded to transformation of this plasmids to microalgae. We co-transformed Chlamydomonas reinhardtii to Tris-acetate-phosphate plus neomycine agar plates with pRbcnfsI and pKS-aphVIII-lox plasmid in order to select colonies based on the arginine deficiency in mutant strain cc-425 of Chlamydomonas reinhardtii. Next step was to collect colonies from plate. But due to time constraints we could not obtain colonies from this experiment yet. After collecting colonies, we plan to subculture colonies in the presence of trinitrotoluene (TNT) and check for change in its concentration.
There is a problem with poping up if you see this note.
Retrieved from " http://2011.igem.org/Team:Bilkent_UNAM_Turkey/Experiment "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
