"
Page
Discussion
View source
History
teams
Log in
 
Team:XMU-China/Project
From 2011.igem.org
Home
Team
Project
Model
Result
Data
Safety
Human Practice
Partner
Description
Approach
Background
Protocols
Wetlab Journal
Contents
1 Project Description
2 Approach
3 Background
3.1 Quorum Sensing
3.1.1 LuxI :  Acyl-homoserine-lactone synthase
3.1.2 LuxR: Transcriptional activator protein luxR
3.1.3 AHL: N-acyl-homoserine lactone
3.1.4 IPTG: Isopropyl β-D-1-thiogalactopyranoside
3.2 toxin ccdB
3.3 LacZalpha-ccdB
3.4 ccdB vs cell DEATH
3.5 RBS: Ribosome Binding Sites
3.6 Lux pR
3.7 IR-GFP
4 Protocols
4.1 1.Isolation of Plasmid
4.2 2.Reaction system of restriction endonuclease
4.3 Standard BioBrick Assembly
4.4 Suffix Insertion
4.5 Ligation
4.6 Transformation
4.7 Restriction analysis
4.8 Determining fluorescence intensity
4.9 Cell growth
4.10 Site Directed Mutagenesis
5 Wetlab journal
5.1 Week 1(3rd Apr.—9th Apr.)
5.2 Week 2(10th Apr.—16th Apr.)
5.3 Week 3(17th Apr.—23rd Apr.)
5.4 Week 4(24th Apr.—30th Apr.)
5.5 Week 5-8(1st May—28th May)
5.6 Week 9-17(29th May—30th Jul.)
5.7 Week 18-21(31st Jul.—27th Aug.)
5.8 Week 20-22(28th Aug.—17th Sep.)
5.9 Week 23(18th Sep.—24th Sep.)
Project Description
i-ccdB: intelligent Control of Cell Density in Bacteria
We have developed a series of devices which program a bacteria population to maintain at different cell densities. We have designed and characterized the genetic circuit to establish a bacterial ‘population-control’ device in E. coli based on the well-known quorum-sensing system from Vibrio fischeri, which autonomously regulates the density of an E. coli population. The cell density however is influenced by the expression levels of a killer gene (ccdB) in our device. As such, we have successfully controlled the expression levels of ccdB by site-directed mutagenesis of a luxR promoter (lux pr) and error-prone PCR of gene luxR, and finally we have built a database for a series of mutation sites corresponding to different cell densities. An artificial neural network has then been built to model and predict the cell density of an E. coli population. This work can serve as a foundation for future advances involving fermentation industry and information processing.
Approach
Background
Quorum Sensing
Quorum sensing is a method of communication between bacteria that enables the coordination of group-based behavior based on population density. [3] It was first observed in Vibrio fischeri, a bioluminiscent bacterium that lives in the ocean. Bacteria that use quorum sensing constantly produce and secrete certain signaling molecules (called autoinducers or pheromones). These bacteria also have a receptor that can specifically detect the signaling molecule (inducer). When the inducer binds the receptor, it activates transcription of certain genes, including those for inducer synthesis. There is a low likelihood of a bacterium detecting its own secreted inducer. Thus, in order for gene transcription to be activated, the cell must encounter signaling molecules secreted by other cells in its environment. When only a few other bacteria of the same kind are in the vicinity, diffusion reduces the concentration of the inducer in the surrounding medium to almost zero, so the bacteria produce little inducer. However, as the population grows, the concentration of the inducer passes a threshold, causing more inducer to be synthesized. This forms a positive feedback loop, and the receptor becomes fully activated. Activation of the receptor induces the up-regulation of other specific genes, causing all of the cells to begin transcription at approximately the same time. This coordinated behavior of bacterial cells can be useful in a variety of situations. For instance, the bioluminescent luciferase produced by V. fischeri would not be visible if it were produced by a single cell. By using quorum sensing to limit the production of luciferase to situations when cell populations are large, V. fischeri cells are able to avoid wasting energy on the production of useless product.
LuxI : Acyl-homoserine-lactone synthase
LuxI is required for the synthesis of OHHL (N-(3-oxohexanoyl)-L-homoserine lactone) also known as VAI or N-(beta-ketocaproyl)homoserine lactone or 3-oxo-N-(tetrahydro-2-oxo-3-furanyl)-hexanamide, an autoinducer molecule which binds to luxR and thus acts in bioluminescence regulation.[8]
Catalytic activity
An acyl-[acyl-carrier-protein] + S-adenosyl-L-methionine = [acyl-carrier-protein] + S-methyl-5'-thioadenosine + an N-acyl-L-homoserine lactone.[8]
LuxR: Transcriptional activator protein luxR
The function of LuxR homologues as quorum sensors is mediated by the binding of N-acyl-L-homoserine lactone (AHL) signal molecules to the N-terminal receptor site of the proteins.[1] It is a transcriptional activator of the bioluminescence operon. It binds to the AHL autoinducer.[7]
AHL: N-acyl-homoserine lactone
AHL is a kind of signaling molecule involved in bacterial quorum sensing.[2] In Vibrio fischeri, AHL binds to the protein product of the LuxR gene and activates it. The C-terminal domain of activated LuxR relieves the repression exerted by H-NS nucleoid proteins that bind to the promoters of LuxR, LuxI and the LuxCDABEG operon, as well as to A-T-rich stretches within that operon and other genomic regions. The product of LuxI catalyses the synthesis of AHL. Thus, AHL acts as an autoinducer. Transcription of the LuxCDABEG operon results in luminescence due to the expression of LuxA and LuxB, which form a protein known as a luciferase and the expression of LuxC, D, E, and G, which are involved in the synthesis of the luciferase's substrate, tetradecanal.[11]
IPTG: Isopropyl β-D-1-thiogalactopyranoside
This compound is used as a molecular mimic of allolactose, a lactose metabolite that triggers transcription of the lac operon. Many regulatory elements of the lac operon are used in inducible recombinant protein systems.[14]
iGEM-Team XMU-China has designed a series of circuits driven by The PLlac 0-1 promoter (BBa_R0011). IPTG was used as an inducer to activate these circuits.
toxin ccdB
Toxin ccdB is a component of toxin-antitoxin (TA) module, functioning in plasmid maintenance. Cell killing by CcdB is accompanied by filamentation, defects in chromosome and plasmid segregation, defects in cell division, formation of anucleate cells, decreased DNA synthesis and plasmid loss.
LacZalpha-ccdB
The LacZ alpha-CcdB fusion protein has retained both the CcdB killer activity and the ability to alpha-complement the truncated LacZ delta M15.
ccdB vs cell DEATH
iGEM-Team XMU-China has designed and constructed a ccdB producer driven by promoter lux pR. We assumed that for circuit-regulated growth, the cell death rate is proportional to the intracellular concentration of the killer protein CcdB. For this reason, we controlled the expression level of the killer gene ccdB in three ways: (1) by using different RBSs (RBS1.0, RBS0.6, RBS0.3, RBS0.07); (2)by using different LuxR promoters(site-directed mutagenesis); (3) by using different LuxR(error-prone PCR).
RBS: Ribosome Binding Sites
A Ribosome Binding Site (RBS) is an RNA sequence found in mRNA to which ribosomes can bind and initiate translation. In order to control the expression of the killer protein ccdB, our team designed a series of bacteria population-control devices using RBSs with different strength. The cell growth and fluorescent curves corresponding to different RBSs illustrate that the bacteria population was successfully controlled at different cell densities.
Lux pR
Promoter lux pR is activated by LuxR in concert with HSL (homoserine lactone). Two molecules of LuxR protein form a complex with two molecules of the signalling compound HSL. This complex binds to a palindromic site on the promoter, increasing the rate of transcription. This promoter is used in our “killer protein producer” to regulate the expression of the killer protein ccdB. iGEM-Team XMU-China has successfully strengthened the expression of lux pR by mutating its DNA sequence. Three mutants, IR-3, IR-5, IR-3/5, were obtained by site-directed mutagenesis using 3-step PCR method.
IR-GFP
IR-GFP is a series of report devices designed for testing the performance of lux R promoters before and after mutagenesis. These four IR-GFP report devices have been transformed into E.coli string BL21 separately. These devices produced greenish tint visible by naked eyes when induced by IPTG. We measured and compared their florescent intensities at steady state. As the only difference between the four devices is Lux R promoter, the efficiency of the four Lux R promoters could be defined.
Protocols
1.Isolation of Plasmid
Using the Procedure for GenEluteTM Plasmid Miniprep Kit
•Collect 1-5 mL bacterium fluid in 1.5 mL centrifuge tube，and centrifuge fluid at 12000 r/min
•Resuspend cells. Discard the supernatant and completely resuspend the bacterial pellet with 250 µl of the Resuspension Solution
•Lyse cells. Lyse the resuspended cells by adding 250 µl of the Lysis Solution
•Neutralize. Precipitate the cell debris by adding 350 µl of the Neutralization/Binding Solution, and centrifuge fluid at 12000r/min
•Load cleared lysate. Transfer the supernatant from step 4 to the spin column. Centrifuge at 12000r/min for 1 minute, and then discard the filtrate
•Optional wash. Add 500 µl of the Optional Wash Solution to the column. Centrifuge at 12000 r/min for 1 minute. Discard the filtrate
•Wash column. Add 500 µl of the diluted Wash Solution to the column. Centrifuge at 12000r/min for 1 minute.
•Elute DNA. Transfer the column to a new collection tube. Add 50~100 µl of Eluent Solution to the column. Centrifuge  at 12000 r/min for 1 minute. The DNA is now present in the filtrate and is ready for immediate use or storage at -20℃
2.Reaction system of restriction endonuclease
File:XMU China 7.jpg
System1、2、3 and 4 are used for Standard BioBrick Assembly .System 5 and 6 are used for Restriction analysis. Digestion of sample: at least 500 ng DNA / 10 µL volume. Digest for 4 h at 37 °C, afterwards inactivated by adding 10x loading buffer and standing for 10 min at room temperature.
Standard BioBrick Assembly
•Digestion of insert: 2 μg~5 μg DNA / 100 µL volume, 10x H buffer, EcoRI, SpeI. Digestion and inactivation. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel, try to avoid staining or exposure to ultraviolet light of the insert.
•Digestion of vector: 2 μg~5 μg DNA / 100 µL volume, 10 x M buffer, EcoRI, XbaI. Digestion and inactivation. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel, try to avoid staining or exposure to ultraviolet light of the insert.
Suffix Insertion
•Digestion of insert: 2 μg~5 μg DNA / 100 µL volume, 10x M buffer, XbaI, PstI. Digestion and inactivation. Clean up the insert.
•Digestion of vector : 2 μg~5 μg DNA / 100 µL volume, 10x H buffer, SpeI, PstI. Digestion and inactivation. Clean up the vector.
Ligation
•After digestion and clean-up, the next step is ligation. Overnight ligation at 16°C. Table 2 is the system of ligation.
Transformation
•Preparation of competent E.coli cells
•Add 10 µL plasmid to 100 µl competent cells in centrifuge tube
•Store tube on ice for 20-30 minutes
•Water bath for 90s at 42℃
•Put the tube on ice for 1-2min
•Add 790 µL LB，and cultivation for 1h at 37 ℃，then plate on selective LB-Medium.
Restriction analysis
•Pick one colony with a sterile tip and cultivation in 20ml LB for overnight at 37 ℃
•Isolation of Plasmid
•Digest BioBrick，the system of Restriction analysis refer to table1
•Gel electrophoresis：add 2 µL loading buffer to digestion mixture. An agarose concentration is 1 %.
Determining fluorescence intensity
•Add IPTG when A600 0.6~0.8.
•Cool the culture 10 minutes on ice.
•Centrifuge at 6000rpm. Wash it with pre-cooled PBS buffer.
•Use fluorescence spectrophotometer tomeasure the fluorescenceof GFP:
•Before measuring, dilute the bacteria with PBS buffer so that it can be within the measuring #range. Set excitation wavelength 491 nm, emission wavelength 511 nm.
•Transfer the measured bacteria in a new centrifuge tube and measure the OD of the bacteria.
Cell growth
•100µL suspension was inoculated from a Glycerin tube into 20ml fresh LB and incubated overnight at 37℃ and 250 r.p.m.
•100µL suspension was inoculated again from step1 into 50ml fresh LB and incubated at 37℃ and 250r.p.m
•IPTG was added when A600≈0.6-0.8
•1 ml suspension was taken on every sample taken time. 3 samples were taken in each time.
•Diluted each sample to 10-6(Sometimes 10-5),and then plate on selective LB-Medium.
•After 12h, count the number of CFU on the plate on different time point and then draw the cell growth curve.
File:XMU China 10.jpg
Site Directed Mutagenesis
File:XMU China 11.jpg
File:XMU China 12.jpg
•Digest the template plasmid by adding 1 µL of DpnI and incubate for 1-2 h
•Transform 10 µL of t PCR product into competent E. coli cells
•Screen the transformants using restiction digest and sequencing
Wetlab journal
Week 1(3rd Apr.—9th Apr.)
Aim:
By replacing the promoter of the existing BioBrick BBa_F2621 with Placo-1, the expression of the downstream sequence can be controlled by adding IPTG.
Performance:
Constructing the new BioBrick BBa_K658000 (The figure below and sequence analysis can indicate it is correctly constructed.)
Testing the expression of BBa_K658000 by adding IPTG  (We thought the part didn’t exist in the registry and we had constructed a new part. But, afterwards, we found it(BBa_F2622) did exist! )
Fig.1
Week 2(10th Apr.—16th Apr.)
Aim:
By combining the BioBrick BBa_R0011, BBa_F1610 and BBa_K65800, we can get a new BioBrick which is part of the bacteria population-control device we have planned to design.
Performance:
Constructing the new BioBrick IR(BBa_K658012)
Week 3(17th Apr.—23rd Apr.)
Aim:
By adding the killer gene to the Biobrick IR, we can finally get the bacteria population-control device H(BBa_K658003).
Performance:
Constructing the BioBrick H. The figure shown below and sequence analysis can indicate that it is correctly constructed.
Fig.2
Week 4(24th Apr.—30th Apr.)
Aim:
We plan to construct a new part in order to test how the concentration of AHL can affect the expression of the downstream sequence under the control of lux pR. Performance:
Successfully construct the new part BBa_K658022 Testing on the performance of the part using different concentrations of AHL as inducer.The result is shown as follows:
Fig.3
Week 5-8(1st May—28th May)
Aim:
In order to investigate the performance of the device(iccdB) under different conditions, we plan to construct several devices with RBS of different efficiency to see how RBS can affect the functioning of the device.
Performance：
Week 5-7(1st May—21st May)
Constructing the population-control device with RBS0.07
Fig.4
S1:  single endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT
S2:  double endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT
S3: single endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxR-TT-lux pR
S4: double endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxR-TT-lux pR
S5: single endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-lux pR
S6: double endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-lux pR
S7: single endonuclease digestion of pSB1AK3-lacZɑ-ccdB-TT
S8: double endonuclease digestion of pSB1AK3-lacZɑ-ccdB-TT
S9: single endonuclease digestion of pSB1A2-RBS0.07-lacZɑ-ccdB-TT
S10: double endonuclease digestion of pSB1A2-RBS0.07-lacZɑ-ccdB-TT
S11: single endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-luxpR-RBS0.07-lacZɑ-ccdB-TT
S12: double endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-luxpR-RBS0.07-lacZɑ-ccdB-TT
Constructing the population-control device with RBS1.0
Fig.5
S1: single endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT;
S2: double endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT ;
S3: single endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxR-TT-lux pR
S4: double endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxR-TT-lux pR
S5: single endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-lux pR
S6: double endonuclease digestion Of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-lux pR
S7: single endonuclease digestion of pSB1A2-RBS0.6-lacZɑ-ccdB
S8: double endonuclease digestion of pSB1A2-RBS0.6-lacZɑ-ccdB
S9: single endonuclease digestion Of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-luxpR-RBS0.6-lacZɑ-ccdB
S10: double endonuclease digestion Of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-luxpR-RBS0.6-lacZɑ-ccdB
S11: single endonuclease digestion Of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-luxpR-RBS0.6-lacZɑ-ccdB
S12: double endonuclease digestion Of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-luxpR-RBS0.6-lacZɑ-ccdB
Constructing the population-control device with RBS0.3
Fig.6
S1: single endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT
S2: double endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT
S3: single endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxR-TT-lux pR
S4: double endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxR-TT-lux pR
S5: single endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-lux pR
S6: double endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-lux pR
S7: single endonuclease digestion of pSB1A2-RBS0.3-lacZɑ-ccdB
S8: double endonuclease digestion of pSB1A2-RBS0.3-lacZɑ-ccdB
S9: single endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-luxpR-RBS0.3-lacZɑ-ccdB
S10: double endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-luxpR-RBS0.3-lacZɑ-ccdB
S11: single endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-luxpR-RBS0.3-lacZɑ-ccdB
S12: double endonuclease digestion of pSB1A2-PlacO-1-RBS1.0-luxI-TT-PlacO-1-RBS1.0-luxR-TT-luxpR-RBS0.3-lacZɑ-ccdB
Week 8(22nd May—28th May)
Conducting the experiment on the cell growth of the two devices with RBS0.07.
Fig.7
Conducting the experiment on the viable cell density at steady state of the four devices with RBS0.07, RBS0.3, RBS0.6 and RBS1.0
Fig.8
Week 9-17(29th May—30th Jul.)
Aim:
The promoter lux pR can also affect the functioning of the device. So we plan to mutate the promoter lux pR to see how the mutation in lux pR can affect functioning of device. Before testing on the population-control device, we try to construct another device to test how the mutation of the promoter can affect the expression of the downstream gene. In the following experiment, we plan to use the gene of gfp to show the expression.
Performance:
Week 9(29th May—4th Jun.)
Constructing the device IR-gfp (BBa_K658016)
Week 10(5th Jun.—11th Jun.)
Experiment suspended because of exams.
Week 11(12th Jun.—18th Jun.)
Mutation of lux pR being carried out
Week 12(19th Jun. —25th Jun.)
Experiment suspended becauce of an internship in a factory away from Xiamen.
Week 13(26th Jun.—2nd Jul.)
Continuing the work of last week and getting three mutants Proving the three mutants wrong which is shown as follows:
Fig.9
Week 14(3rd Jul.--9th Jul.)
By restriction analysis, the restored IR-gfp identified to be broken
Fig.10
Reconstructing IR-gfp
Fig.11
Week 15-16(10th Jul.--23rd Jul.)
Successfully getting the mutant at site 3, site 5 and site 3/5 which can be verified by sequence analysis
Fig.12
Fig.13
Fig.14
Fig.15
Week 17(24th Jul.—30th Jul.)
Conducting the experiment of determining the fluorescence curve of the three devices with mutations respectively at site 3, 5, 3/5
Fig.16
Fig.17
Week 18-21(31st Jul.—27th Aug.)
Aim:
We plan to mutate the lux pR to see how the mutation in lux pR can affect functioning of the population-control device.
Performance:
Week 16(31st Jul. —6th Aug.) Activation of H for the following experiment
Fig.18
Fig.19
Successfully getting the mutant H3
The intended 5-site mutation turning out to be mutated at 5/15 and 5/20
Week 17-18(7th Aug.—20th Aug.)
Successfully getting the mutant at site 5 and 3/5
Week 19(21st Aug. —27th Aug.)
Conducting the experiment of the bacteria quantity of the population-control device with mutations respectively at site 3, 5, 3/5
Fig.20
Week 20-22(28th Aug.—17th Sep.)
Aim:
In order to further study whether the population-control device has an impact on the metabolic level, we plan to add the gene gfp to the population-control device to examine it.
Performance:
Week 20(28th Aug.—3rd Sep.)
Constructing the BioBrick H62M19(BBa_K658020)
Constructing 62M19(BBa_K658021)
Fig.21
Fig.22
Week 21(4th Sep.—10th Sep.)
Conducting the experiment of determining the fluorescence curves of H62M19 and 62M19
Reviewing the experimental process of the week and sorting out mistakes we have made
Week 22(11th Sep.—17th Sep.)
Repeating and optimizing the work done last week to get better results.
Fig.23
Fig.24
Week 23(18th Sep.—24th Sep.)
Changing the backbone of the plasmids according to the requirement
Retrieved from " http://2011.igem.org/Team:XMU-China/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
