<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IIT_Delhi_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IIT Delhi/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM IIT Delhi</TITLE></P><DIV id="globalWrapperdivi" style="margin-top: -6.5%;z-index: 1;background-color: white;"><DIV class="navaman"><DIV class="dropdownsakks"><DIV class="borderYtoX"><A href="javascript:void(0)"> Notebook </A></DIV><DIV class="dropdownsakks-content" style="margin-top: 66px;"><A href="https://2018.igem.org/Team:IIT_Delhi/Notebook">Lab Records</A><A href="https://igem.org/2018_Judging_Form?team=IIT_Delhi">Medal Criteria</A></DIV></DIV><DIV class="dropdownsakks"><DIV class="borderYtoX"><A href="javascript:void(0)"> Team </A></DIV><DIV class="dropdownsakks-content" style="margin-top: 66px;"><A href="https://2018.igem.org/Team:IIT_Delhi/Team">Members</A><A href="https://2018.igem.org/Team:IIT_Delhi/Attributions">Attributions</A></DIV></DIV><DIV class="borderYtoX"><A href="https://2018.igem.org/Team:IIT_Delhi/Collaborations"> Collaboration</A></DIV><DIV class="dropdownsakks"><DIV class="borderYtoX"><A href="javascript:void(0)"> Human Practices </A></DIV><DIV class="dropdownsakks-content" style="margin-top: 66px;"><A href="https://2018.igem.org/Team:IIT_Delhi/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:IIT_Delhi/Safety">Safety</A><A href="https://2018.igem.org/Team:IIT_Delhi/Public_Engagement">Public Engagement</A></DIV></DIV><DIV class="borderYtoX"><A href="https://2018.igem.org/Team:IIT_Delhi/Model">Modelling</A></DIV><DIV class="dropdownsakks"><DIV class="borderYtoX"><A href="javascript:void(0)"> Parts </A></DIV><DIV class="dropdownsakks-content" style="margin-top: 66px;"><A href="https://2018.igem.org/Team:IIT_Delhi/Basic_Part">Basic Parts</A><A href="https://2018.igem.org/Team:IIT_Delhi/Composite_Part">Composite Parts</A></DIV></DIV><DIV class="dropdownsakks"><DIV class="borderYtoX"><A href="javascript:void(0)">Results</A></DIV><DIV class="dropdownsakks-content" style="margin-top: 66px;"><A href="https://2018.igem.org/Team:IIT_Delhi/Demonstrate">Demonstrate</A><A href="https://2018.igem.org/Team:IIT_Delhi/InterLab">Interlab</A></DIV></DIV><DIV class="dropdownsakks"><DIV class="borderYtoX"><A href="javascript:void(0)"> Project </A></DIV><DIV class="dropdownsakks-content" style="margin-top: 66px;"><A href="https://2018.igem.org/Team:IIT_Delhi/Description">Overview</A><A href="https://2018.igem.org/Team:IIT_Delhi/Design">Design</A></DIV></DIV></DIV><DIV class="overlayved" id="myNav"><DIV class="overlayved-content"><BUTTON class="overlayved-btn2" style="padding-bottom: 15px;padding-top: 15px;align-self: center;font-size: 30px;font-family: raleway;border-color: white;border-style: none; border-image: none;background-color: white;border-width: 0px;cursor: pointer; color: #818181;" onclick="closeNav()">Close Navigation</BUTTON><BUTTON class="overlayved-btn" style="padding-bottom: 15px;" onclick="myFunction1()">Project
           </BUTTON><DIV class="dropdownsakks-container" id="mydropdownsakks1" style="font-style: italic; "><A href="https://2018.igem.org/Team:IIT_Delhi/Description" style="font-size: 15px; padding-bottom: 5px; padding: 10px; padding-top: 0px;">Overview</A><A href="https://2018.igem.org/Team:IIT_Delhi/Design" style="font-size: 15px;padding: 10px; padding-top: 5px;
                       padding-bottom: 0px;">Design</A></DIV><BUTTON class="overlayved-btn" onclick="myFunction2()" style="padding-bottom: 15px;">Parts
       </BUTTON><DIV class="dropdownsakks-container" id="mydropdownsakks2" style="font-style: italic;"><A href="https://2018.igem.org/Team:IIT_Delhi/Basic_Part" style="font-size: 15px;font-size: 15px; padding-bottom: 5px; padding: 10px; padding-top: 3px;">Basic Parts</A><A href="https://2018.igem.org/Team:IIT_Delhi/Composite_Part" style="font-size: 15px;font-size: 15px; padding: 10px; padding-bottom: 5px; padding-top: 5px;">Composite Parts</A></DIV><BUTTON class="overlayved-btn" onclick="myFunction6()" style="padding-bottom: 15px;">Results
       </BUTTON><DIV class="dropdownsakks-container" id="mydropdownsakks6" style="font-style: italic;"><A href="https://2018.igem.org/Team:IIT_Delhi/Demonstrate" style="font-size: 15px;font-size: 15px; padding-bottom: 5px; padding: 10px; padding-top: 3px;">Demonstrate</A><A href="https://2018.igem.org/Team:IIT_Delhi/InterLab" style="font-size: 15px;font-size: 15px; padding: 10px; padding-bottom: 5px; padding-top: 5px;">Interlab</A></DIV><A href="https://2018.igem.org/Team:IIT_Delhi/Model" style="padding-bottom: 15px;padding-top: 15px;">Modelling</A><BUTTON class="overlayved-btn" onclick="myFunction3()" style="padding-bottom: 15px;">Human Practices
       </BUTTON><DIV class="dropdownsakks-container" id="mydropdownsakks3" style="font-style: italic;"><A href="https://2018.igem.org/Team:IIT_Delhi/Human_Practices" style="font-size: 15px;font-size: 15px; padding-bottom: 5px; padding: 10px; padding-top: 0px; ">Human Practices</A><A href="https://2018.igem.org/Team:IIT_Delhi/Safety" style="font-size: 15px;font-size: 15px; padding-bottom: 5px; padding: 10px; padding-top: 5px;">Safety</A><A href="https://2018.igem.org/Team:IIT_Delhi/Public_Engagement" style="font-size: 15px;padding: 10px; padding-top: 5px;
                       padding-bottom: 0px;">Public Engagement</A></DIV><A href="https://2018.igem.org/Team:IIT_Delhi/Collaborations" style="padding-bottom: 15px;padding-top: 15px;">Collaboration</A><BUTTON class="overlayved-btn" onclick="myFunction4()" style="padding-bottom: 15px;padding-top: 15px;">Team
     </BUTTON><DIV class="dropdownsakks-container" id="mydropdownsakks4" style="font-style: italic;"><A href="https://2018.igem.org/Team:IIT_Delhi/Team" style="font-size: 15px; font-size: 15px; padding-bottom: 5px; padding: 10px; padding-top: 0px;padding-bottom: 5px;">Members</A><A href="https://2018.igem.org/Team:IIT_Delhi/Attributions" style="font-size: 15px;padding: 10px; padding-top: 5px;
                       padding-bottom: 5px;">Attributions</A></DIV><BUTTON class="overlayved-btn" onclick="myFunction5()" style="padding-bottom: 15px;">Notebook
      </BUTTON><DIV class="dropdownsakks-container" id="mydropdownsakks5" style="font-style: italic;"><A href="https://2018.igem.org/Team:IIT_Delhi/Notebook" style="font-size: 15px;font-size: 15px; padding-bottom: 5px; padding: 10px; padding-top: 0px; ">Lab Records</A><A href="https://igem.org/2018_Judging_Form?team=IIT_Delhi" style="font-size: 15px;font-size: 15px; padding-bottom: 5px; padding: 10px; padding-top: 5px;">Medal Criteria</A></DIV></DIV></DIV><HEADER class="major" style="
    text-align: center;
    font-size: 60%;
    background: white;
    margin: 0 auto;
    background-color: white;
    width: 65%;
    font-weight: normal;
"><H2 class="h2font">Overview</H2><H2 id="pfont" style="
    font-weight: normal;
">


A Mathematical model captures the essential dynamics of the system in the form of mathematical equations and helps to study and analyze the biological system before stepping into lab work. All of the chemical reactions in the system can essentially be written in the form of differential equations that capture the biological processes in the cell. These equations can then be simulated and the dynamics can be analyzed, in order to understand how a particular network is going to behave inside the cell. 

The two molecular processes that are central to the functioning of a cell are transcription and translation. The cell consists of DNA, which contains all the genetic information of the cell. It contains information for synthesis of various proteins required for normal functioning of the cell. The process of transcription leads to creation of mRNA from DNA which contains the information for protein synthesis. This mRNA is then translated into protein with the help of ribosomes and tRNA. 

Thus, the entire set of reactions happening inside a cell leading to the expression of a gene can be broken down into the following –

Here, the major processes occurring are as follows - 
</H2><H2 id="pfont2"><U id="pfont2">mRNA</U></H2><H2 class="centered" id="pfont2" style="
    list-style-type: decimal;
    text-align: left;
    font-weight: normal;
"><LI>mRNA is being produced from the plasmid DNA that has been introduced into the cell via transformation, by the process of transcription.

</LI><LI>The mRNA produced is also being degraded, because it has a certain half life (just like radioactive elements decay, all chemicals have a half life, and so do DNA and RNA!). 
</LI></H2><H2 id="pfont2"><U id="pfont2">Protein</U></H2><H2 class="centered" id="pfont2" style="
    font-weight: normal;
    text-align: left;
    list-style-type: decimal;
"><LEFT><LI>Protein is produced from the mRNA transcript by the process of translation.
</LI><LI>Protein is also degraded since it has a half life, similar to the mRNA.
</LI></LEFT></H2><H2 id="pfont2" style="
    line-height: 1.5em;
    font-weight: normal;
">What needs to be noted before we start to write a model for this is that this is a very simplistic model that makes use of several assumptions and simplifications. This is because biological systems are extremely complex, and at a single instant of time, there are several hundred reactions happening. Thus, we need to simplify and lump certain intermediate reactions, in order to have some quantitative estimate of how our system will behave. 

Some of the assumptions made here are - </H2><H2 class="centered" id="pfont2" style="
    font-weight: normal;
    text-align: left;
    list-style-type: decimal;
"><LEFT><LI>mRNA is made directly from DNA, and all the other components and processes in between, such as pulling of RNA polymerase (RNAp) by the TATA box in the promoter, binding of RNAp to the promoter and initiation of transcription are sufficiently fast, so that the parameters can be lumped and variables (such as RNAp and Promoter) can be ignored.
</LI><LI>Degradation of molecules such as mRNA and protein is spontaneous, and is not triggered or accelerated by certain components (such as ssrA). 
</LI><LI>The rates of production and degradation are constant.
</LI><LI>mRNA is not degraded, damaged, or consumed in any way during translation or production (transcription).
</LI><LI>Total DNA inside a cell is constant.
</LI><LI>Mass action kinetics is valid for the reactions occurring above.
</LI></LEFT></H2></HEADER><HEADER class="major" style="
    text-align: center;
    font-size: 60%;
    background: white;
    margin: 0 auto;
    background-color: white;
    width: 65%;
"><H2 class="h2font">Model for Unregulated Gene Expression</H2><P>                                                                                                                                                                                                            </P><H2 id="pfont" style="
    background: white;
    font-weight: normal;
">A model for this simple system shown above can then be written, keeping the assumptions in mind. To start with, we can consider the two variables that are of importance to us in determining the level of gene expression. These are the mRNA and protein levels (since the DNA levels in a cell are assumed to be constant, they are not of interest). 

Therefore, let us write the differential equation for mRNA first - 

mRNA is produced from DNA, and degraded spontaneously. Therefore, at any instant of time, the rate of change of mRNA can be written as - 

Note, that here, we have not written the reaction where mRNA is being converted to protein,  since mRNA is not actually being consumed there or being produced. 1 molecule of mRNA simply produces 1 molecule of protein (assumption). 

Further, it has to be noted that the [DNA] and [mRNA] terms appear in the equation since in writing the model, we assume that mass action kinetics are valid, ie, the rate of the reaction is equal to the rate constant times the concentration of the reactant, raised to a power equal to the number of molecules of the reactant. 

Now, we know that the DNA concentration remains constant and does not change over time. Therefore, the [DNA] term can be included in the constant itself, to give 
Now, the dynamics of the protein can be similarly written as 

And that is it! We’ve just written down our first model, for a gene being expressed from a constitutive promoter. Now that we have our model, we can simulate these and find out the dynamics. 

Simulation basically means solving the differential equations to get the variation of the component (mRNA, protein) with time. This can be done by hand for the equations above. However, as models get more complex, implicit equations appear, which are much more difficult to solve by hand. Thus, it is essential to get the hang of modelling software such as MATLAB or R, which solve differential equations and simulate the model for a specified period of time. 

Thus, we write down the model on MATLAB here, and simulate it for a time period of 200 time units. The values of the constants used for alpha, gamma etc and the MATLAB code for the same can be found on the github library link given below. The plot obtained is as follows - 

Changing the parameters for production and degradation rates can give different kinds of graphs, and can be explored by simply changing the values of alpha, gamma, K etc in the model and simulating the same. However, as we can see here, the mRNA and protein levels both rise to a certain fixed value. This is known as the steady state value. 

However, we can make a further simplification in this model. Generally, the mRNA dynamics are faster than the protein dynamics. This means that mRNA levels approach their steady state value faster than proteins do. Therefore, we can say make the assumption and further simplification that before the protein dynamics start to come into play, the date of change of mRNA is zero. This is known as the “quasi steady state assumption”.

Therefore at steady state, 


Thus, 
Now, we can replace the value of [mRNA] in equation (2) with the value given above, to get - 

We can now try to simulate and plot the graph for the protein levels, and compare the time series of the two models - 

Therefore, we can see that by making the assumption that mRNA is already at steady state at the start of time, the protein levels begin to rise faster than the earlier model. However, the steady state value for protein remains the same. This is because we have only simplified the model by changing the time scale and assuming that at the given time scale, mRNA dynamics are at steady state. We have not changed the steady state per se. 




</H2></HEADER><HEADER class="major" style="
    text-align: center;
    font-size: 60%;
    background: white;
    margin: 0 auto;
    background-color: white;
    width: 65%;
"><H2 class="h2font">Model for Regulated Gene Expression</H2><P>                                                                                                                                                                                                            </P><H2 id="pfont" style="
    background: white;
    font-weight: normal;
">

Regulation of gene expression involves changing the expression of protein or RNA produced by a particular gene. Various mechanisms exist, for doing the same allowing for control at various stages of the expression of the gene. For instance, if the control/regulation is such that it does not allow transcription to happen, it is termed as transcriptional control. Similarly, translational, post translational and several other layers of control exist. 

The simplest and most commonly employed mode of regulation is the transcriptional control by repressor proteins. These are protein molecules that can bind to specific “operator” sites in the promoter region, and stop the promoter to recruit RNA polymerase successfully, thereby inhibiting transcription. Common examples of such systems are LacI, TetR and cI, which can inhibit transcription from the pLac, pTet and pCI promoter respectively. This mode of control is also commonly referred to as repression, and should not be confused with inhibition, which is a separate control mechanism.

Here, let us try to model what regulated gene expression looks like, by looking at a typical example of transcriptional activation. Consider the following case - 

We have a protein X, that can exist in two states, the native (inactive) state X, and an active form X*. The molecule X* can bind to the promoter (say P), and promote transcription of the gene by helping the promoter to recruit RNA polymerase.

	Lets look at another case, where we have transcriptional repression -

Here we are given 2 proteins, A and B. The protein A is produced and degraded, and is a transcriptional repressor for the gene B. A has its own production and degradation rates, described by alpha and gamma, and B also has its own production and degradation rates, given by beta and gamma respectively. Further, DA and Do represent the two states that the DNA region of the promoter PB can have. DA represents the state where A is bound to the operator, and Do represents the state where A is unbound.

The system can be represented by a set of reactions as follows –

Based on these reactions, we can write the mass action model for the system. This can be represented by the following differential equations –

Further, we have a 5th equation in the model, which is based on the conservation of DNA. Since all of the DNA of the promoter can either be bound by transcription factor A (DA state) or be unbound (Do state), therefore, the total DNA (DT), at any time, can be represented as –

Thus, the model of the system, based on mass action kinetics and conservation relations can be represented by

Note that in this model, we have taken the rate of change of Do and DA as well, which are DNA molecules. This is because here the DNA concentration also changes because the DNA switches states.

Now, solving this model and simulating, we get the following results - 

From the above results, we can see that the time taken for achieving steady state for all the variables (A, Do, DA and B) is more or less similar, and takes about 4-5 hours. This goes against the intuition that the binding and unbinding happens faster, as compared to the production of A and B, which should take a larger time. We see that this does happen, when we run the system of equations for α = 100 nM/hr (results not shown). Thus, the time scale separations become more prominent as the value of α increases (time scale separation was further more prominent for α = 500 nM/hr).

Further, upon varying the values of k1 and k2 by 100 fold, we see the following –

In all of these plots, data1, data2 and data3 represent k1 = 0.01, 1 and 100 respectively. We see that on decreasing the value of k1, the effect on the steady state values is not significant. On the other hand, increasing k1 by 100 fold changes the steady state values, and brings down the level of B ultimately produced at steady state. This is because if k1 is high, that means that more A binds to the promoter of B, repressing it. Therefore, a lower steady state level of B is observed.

</H2><H2 class="h2font">Model for Unregulated Gene Expression</H2><P>                                                                                                                                                                                                            </P><H2 id="pfont" style="
    background: white;
    font-weight: normal;
">A model for this simple system shown above can then be written, keeping the assumptions in mind. To start with, we can consider the two variables that are of importance to us in determining the level of gene expression. These are the mRNA and protein levels (since the DNA levels in a cell are assumed to be constant, they are not of interest). 

Therefore, let us write the differential equation for mRNA first - 

mRNA is produced from DNA, and degraded spontaneously. Therefore, at any instant of time, the rate of change of mRNA can be written as - 

Note, that here, we have not written the reaction where mRNA is being converted to protein,  since mRNA is not actually being consumed there or being produced. 1 molecule of mRNA simply produces 1 molecule of protein (assumption). 

Further, it has to be noted that the [DNA] and [mRNA] terms appear in the equation since in writing the model, we assume that mass action kinetics are valid, ie, the rate of the reaction is equal to the rate constant times the concentration of the reactant, raised to a power equal to the number of molecules of the reactant. 

Now, we know that the DNA concentration remains constant and does not change over time. Therefore, the [DNA] term can be included in the constant itself, to give 
Now, the dynamics of the protein can be similarly written as 

And that is it! We’ve just written down our first model, for a gene being expressed from a constitutive promoter. Now that we have our model, we can simulate these and find out the dynamics. 

Simulation basically means solving the differential equations to get the variation of the component (mRNA, protein) with time. This can be done by hand for the equations above. However, as models get more complex, implicit equations appear, which are much more difficult to solve by hand. Thus, it is essential to get the hang of modelling software such as MATLAB or R, which solve differential equations and simulate the model for a specified period of time. 

Thus, we write down the model on MATLAB here, and simulate it for a time period of 200 time units. The values of the constants used for alpha, gamma etc and the MATLAB code for the same can be found on the github library link given below. The plot obtained is as follows - 

Changing the parameters for production and degradation rates can give different kinds of graphs, and can be explored by simply changing the values of alpha, gamma, K etc in the model and simulating the same. However, as we can see here, the mRNA and protein levels both rise to a certain fixed value. This is known as the steady state value. 

However, we can make a further simplification in this model. Generally, the mRNA dynamics are faster than the protein dynamics. This means that mRNA levels approach their steady state value faster than proteins do. Therefore, we can say make the assumption and further simplification that before the protein dynamics start to come into play, the date of change of mRNA is zero. This is known as the “quasi steady state assumption”.

Therefore at steady state, 


Thus, 
Now, we can replace the value of [mRNA] in equation (2) with the value given above, to get - 

We can now try to simulate and plot the graph for the protein levels, and compare the time series of the two models - 

Therefore, we can see that by making the assumption that mRNA is already at steady state at the start of time, the protein levels begin to rise faster than the earlier model. However, the steady state value for protein remains the same. This is because we have only simplified the model by changing the time scale and assuming that at the given time scale, mRNA dynamics are at steady state. We have not changed the steady state per se. 




</H2></HEADER><HEADER class="major" style="
    text-align: center;
    font-size: 60%;
    background: white;
    margin: 0 auto;
    background-color: white;
    width: 65%;
"><H2 class="h2font">Model for Recombinase Based Incoherent Feed Forwards Loop</H2><P>                                                                                                                                                                                                            </P><H2 id="pfont" style="
    background: white;
    font-weight: normal;
">

A pulse in an incoherent feedforward loop can be described as follows, when U is ON the active part of the transcription factor of protein U which is U* will bind to gene Y independent of gene X thereby activating Y that leads to the
production of protein Y(direct pathway). But transcription factor of protein X will repress gene Y, leading to a decrease in the concentration of protein Y, generating a pulse in the process. When U is OFF, the active part of the transcription factor of gene U which is U* will not bind to gene Y independent of gene X and hence there is no production of protein Y.

Here we present you with the two designs of RIFFL and their respective models,

In the case of the first design, as shown below,

We get the following model,
For which the response comes out as following, after tuning the parameters in the model,
Similarly, for our second design as given below,

We get the following model,
For which the response comes out as following, after tuning the parameters in the model,</H2></HEADER><DIV style="background-color: white"><DIV style="background-color: #555;color:black;font-size: 35px;text-align: center;font-family: sans-serif;"><H4 style="color: white">Contact us</H4></DIV><DIV style="text-align: center;margin-bottom: -136px;"><H4 style="color:black;font-size: 50px;"><U class="fx" style="cursor: pointer;text-align: center">Address</U></H4><P style="text-align: center;margin-bottom: 16px">Undergraduate Laboratory
Department of Biotechnology and Biochemical Engineering, IIT Delhi</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>