"
Page
Discussion
View source
History
teams
Log in
 
Team:Cornell/Protocol
From 2011.igem.org
Results | Protocol | Notebook | Parts Submitted
Contents
1 Molecular Cloning Protocols
1.1 PCR Reaction
1.2 Agarose Gel Electrophoresis
1.3 Gel Purification of DNA (Qiagen QIAquick Gel Extraction Kit)
1.4 DNA Quantification using NanoDrop Spectrophotometry
1.5 Digestion Reaction
1.6 Dephosphorylation of 5' Ends of Vector Backbone
1.7 PCR Clean Up of DNA (Qiagen QIAquick PCR Purification Kit)
1.8 Ligation Reaction
1.9 Desalting of Ligation Reaction Product
1.10 Transformation via Electroporation
1.11 PCR Deletion (Site-Directed Mutagenesis) Reaction
1.12 PCR Deletion (Site-Directed Mutagenesis) Thermocycler Protocol
1.13 Preparing a DNA Sample for Sequencing
2 Microfluidics Protocols
2.1 Making an SU-8 Master of Your Design
2.2 Making PDMS
2.3 Sealing Device
2.4 Streptavidin Coating and Fluorescent Testing
2.5 Cell Lysis
2.6 Filtration
Molecular Cloning Protocols
PCR Reaction
Note: Keep everything on ice and add all volumes in a PCR tube.
37.5μL ddH2O
5.0μL 10x buffer
2.5μL dNTPs
1.0μL MgSO4
1.0μL forward primer
1.0μL reverse primer
1.0μL template
1.0μL DNA polymerase
50.0μL Total
Based on primers, set an appropriate annealing temperature
Agarose Gel Electrophoresis
Prepare a 1% weight-to-volume agarose gel and add SYBR dye or ethidium bromide to stain DNA
Place the gel in the apparatus rig with the wells facing the negative end (black-colored)
Fill the rig with 1x TBE buffer
Load 2µL of 1kb ladder
Add 2µL of 6x loading dye to each PCR reaction tube. Load 20µL in wells
Run at 120V
Gel Purification of DNA (Qiagen QIAquick Gel Extraction Kit)
Cut out the DNA fragment from the agarose gel with a razor blade, while minimizing the size of the gel slice
Weigh the gel slice and add 3 volumes of Buffer QG to every 1 volume of gel (100mg = 100µL)
Dissolve the gel slice using a 60°C heat block
Apply the dissolved gel to the QIAquick column and centrifuge at 13,000rpm for 1 minute
Discard the flow-through and repeat Step 4 until all sample has passed through the column
Add 500µL of Buffer QG to the QIAquick column and centrifuge at 13,000rpm for 1 minute
Wash the column with 750µL of Buffer PE and centrifuge at 13,000rpm for 1 minute
Discard the flow-through and centrifuge at 13,000rpm for 1 minute to remove residual EtOH
Transfer the QIAquick column to a new Eppendorf
Add 35µL elution buffer to the center of the column and wait at least 2 minutes
Centrifuge at 13,000rpm for 1 minute
DNA Quantification using NanoDrop Spectrophotometry
Select Nucleic Acids measurement
Initialize the NanoDrop spectrophotometer with 2µL of autoclaved H2O and wipe off
Blank (calibrate) the NanoDrop spectrophotometer with 2µL of the same elution buffer used during DNA purification and wipe off
Measure 1.5µL of DNA sample and record the concentration in ng/µL
Digestion Reaction
Note: Keep everything on ice
? µL ddH2O (? = whatever volume needed to bring the total volume up to 50µL)
5µL 10x NEBuffer
? µL DNA sample (? = whatever volume corresponds with 1µg)
0.5µL 100x BSA
1µL first restriction enzyme
1µL second restriction enzyme
50µL Total
Note: Consult www.neb.com to determine the buffer compatibility of the restriction enzymes used
Incubate the digestion reaction tube in a 37°C water bath for 3 hours
Dephosphorylation of 5' Ends of Vector Backbone
Add 1µL of Calf Intestinal Alkaline Phosphatase (CIAP) to the digested vector backbone
Incubate at 50°C for 5 minutes
Inactivate CIAP by heating at 85°C for 15 minutes
Proceed to PCR clean up the sample
PCR Clean Up of DNA (Qiagen QIAquick PCR Purification Kit)
Add 5 volumes of Buffer PB to 1 volume of PCR sample
ex: Add 250µL Buffer PB to 50µL PCR sample
Apply this mixture to a QIAquick column and centrifuge at 13,000rpm for 1 minute
Discard flow-through and repeat Step 2 until all sample has passed through the column
Wash column with 750µL Buffer PE and centrifuge at 13,000rpm for 1 minute
Discard flow-through and centrifuge at 13,000rpm for 1 minute to remove residual EtOH
Transfer QIAquick column to new Eppendorf
Apply 50µL elution buffer to center of the column and wait at least 2 minutes
Centrifuge at 13,000rpm for 1 minute
Ligation Reaction
Note: Keep everything on ice
50-100ng vector backbone
3:1 molar ratio of insert:vector
- X ng insert = (3 * Y ng vector * A bp insert) ÷ (B bp vector)
- ? µL insert = X ng insert ÷ insert concentration (ng/µL)
? µL autoclaved H2O (? = whatever volume needed to bring the total volume to 20µL)
2µL 10x T4 DNA ligase buffer
1µL T4 DNA ligase
20µL Total
Incubate in 16°C water bath overnight
Desalting of Ligation Reaction Product
Fill a petri dish with nanopure water
Place the desalting membrane on the water surface with the shiny side facing up
Add 7µL ligation reaction product onto the membrane
Wait 15 minutes
Transformation via Electroporation
During the 15 minute wait of desalting, thaw electrocompetent bacterial cells on ice and cool the electroporation cuvette on ice
Pipet up the desalted ligation mixture and add to thawed bacteria
Transfer bacterial cell mixture to cuvette and keep on ice
Pulse the cuvette using the electroporator at "E. coli: 1mm and 1.8kV" settings
Add 900µL SOB to the cuvette, pipet mix, and transfer entire volume to the original Eppendorf containing the frozen bacteria
Shake the transformation product at 37°C for 1.5 hours
Plate the cells on an agar plate treated with the appropriate antibiotic
Incubate the plate overnight at 37°C
PCR Deletion (Site-Directed Mutagenesis) Reaction
Note: Keep everything on ice and add all volumes in a PCR tube.
? µL ddH2O (? = whatever volume needed to bring the total volume up to 50µL)
5.0μL 10x PfuUltra buffer
1.0μL dNTPs
? uL forward primer = 125ng fwd primer ÷ fwd primer concentration (ng/µL)
? uL reverse primer = 125ng rvs primer ÷ rvs primer concentration (ng/µL)
? µL dsDNA= 20ng insert ÷ insert concentration (ng/µL)
1.0μL PfuUltra high-fidelity DNA polymerase
50.0μL Total
Volumes of diluted primer based on calculations for our ng/µL concentrations
PCR Deletion (Site-Directed Mutagenesis) Thermocycler Protocol
95°C  for  2min
95°C  for  30sec  (18 times)
55°C  for  30sec
72°C  for  1 min/kb
1min/kb corresponds to: 3.20min (RFP), 3.50min (VioA), 5.40min (VioB), 3.00min (VioE)
Preparing a DNA Sample for Sequencing
- 1µL primer (8 pmol -- ex: 8µL 100mM stock primer + 92µL ddH2O)
- at least 1µg DNA
- fill up to 18µL total volume with ddH2O
Microfluidics Protocols
Protocols "Making an SU-8 Master of Your Design," "Making PDMS," and "Sealing Device" provided by Dr. Archer (Cornell University, Biomedical Engineering Department), from BME3010 lab class.
Making an SU-8 Master of Your Design
Materials
A pretreated silicon wafer that has been cleaned with acetone and isopropyl alcohol (IPA) and dehydrated at 200oC for 10-20 minutes.
Hot plate – make sure it is set to 65oC (80C for VWR hot plates)
Aluminum foil tray, wafer tweezers, 2 Pyrex bowls and 2 lids
Coating
Carefully center your wafer on the spinner chuck.  It may be difficult to tell if it’s properly positioned due to the white splatter guard.
Pour approximately 5ml of SU-8(50) into a small plastic beaker. Pour as much as possible (~ 4ml) carefully in the CENTER of the wafer so as not to entrap air.
Place the lid on the Spinner.  Check the display to see if Program 001 is selected.  Press the green Start button on the left of the spinner.  The green light should turn on.
Program 1 yields a coating thickness of 100 mm:
Spread Cycle:  500rpm at 100rpm/sec for 5s
Spin Cycle:  1200rpm at 300rpm/sec for 30 s.
When the green light turns off, remove the spinner lid.  Carefully lift your wafer off the chuck with your tweezers.
Soft Bake
Place it on the 65oC hot plate for 10 mins. After 10 mins increase the temp to 95oC (105C for VWR hot plates) with the wafer still on the hotplate and leave at this temp for 30 minutes.
While you are still baking your wafer
Collect your mask.
Clean off glass square with acetone, IPA, and then DI water.  Dry with air gun – once clean only hold at edges and use only clean room paper towels
Exposure
After 30 minutes at 95oC, let wafer cool on foil for 1 minute. Then take it over to the second clean room.  PUT ON UV GLASSES.
Expose your wafer to UV light with your mask in the contact aligner.
Brief Operating Instructions for the Contact Aligner:
Check that vacuum pump is on in the LER
Turn on Contact Aligner
Press MASK – this lifts up Mask holder to place wafer
Load wafer, long flat side forward (facing away from you), and turn SUB ON to apply vacuum to hold wafer
Press MASK again – to lower mask holder
Place mask printed (shiny side) down and bank it against the pins. Place glass cover on top.
Press MASK-ON to activate vacuum. If you hear the vacuum, you have not achieved a good seal.  Check the mask.
Raise the wafer to touch mask:  Depress small round black button on left of chuck, while pressing this button, turn the gap micrometer Counter Clockwise. Once contact is made the gap dial will slip.
Turn on the CONT. switch on the right panel.
Set exposure time to 65 seconds (preset)
Turn on the lamp power (white switch underneath the unit) and Start button
CYCLE – the microscope unit will position itself over the mask.
CYCLE – UV light will automatically expose over your sample.
CYCLE – UV source will move to home position.
MASK – Mask holder will lift
SUB-ON – wafer will be released from vacuum
Remove your wafer and dial the gap micrometer clockwise to increase distance to mask.
MASK to put down mask holder
Press PULL OFF to release Mask vacuum.
Take off your mask
Post Expose Bake
Perform a post bake of your wafer at 65oC (3rd hotplate) for 1 minute then 95oC for 10 mins to cross link the SU-8.  Allow to cool for 1 minute.
Develop
Place your exposed mask in the first Pyrex dish; fill the dish with SU-8 developer until the mask is covered and cover dish with the glass lid.  Swish around for 8 mins. You should observe the removal of the SU-8 and see your design features.
After 8 mins place your developed wafer in your second Pyrex container with fresh SU-8 developer, cover and swish for 2 mins.
At this point some of your group members can leave the clean room to start making the PDMS in the mail lab area.
Pick up wafer, again holding it along the edge so you do not scratch the surface, with tweezers.  Examine your wafer.  Are their any undeveloped regions?  If yes, place back into the developer solution and swish. The developing time depends on the complexity of the design. More complicated designs take longer to develop.
If no hold it over the glass container with your tweezers and rinse with SU-8 developer, then with IPA and finally dry thoroughly with the air gun under the hood.  A good drying technique is to aim and hold the gun directly in the center of the wafer.  Be careful, gun can be at a high pressure resulting in you dropping your wafer and having to start from the beginning again.
Place your clean wafer into your labeled case and take it outside to the PDMS station.
Those remaining in the clean room must clean up!  All solutions must go into the assigned waste container.  Rinse glass containers with soap and water in the sink.  Set them to dry on clean room paper towels. Remove undeveloped SU-8 50 on your tweezers by wiping it with developer.
Making PDMS
This section is performed in the BME Lab outside the clean room. Safety glasses, lab coats and gloves are required.
Weigh out 5g of hardener (small bottle) in a plastic cup, next add 50 g of base (large tub) to make the PDMS.  A 1:10 ratio (by weight) of hardener: base is the recommended ratio that should be used. Typically, 55g of PDMS is sufficient to cover the bottom of the standard Petri plate that you will use.
Beat the base/hardener mixture with a plastic fork (like beating an egg) vigorously until completely mixed and aerated (at least 5 minutes), the mixture should be white with lots of air bubbles.  To be safe, beat the PDMS until your lab group comes out with the wafer.
Secure your master wafer onto your Petri dish with some silicone glue and place in the 60C oven until PDMS is well aerated.
Pour your PDMS mixture in the center of your wafer for an even coat.  Place your dish in the vacuum chamber to remove air bubbles.
Turn on the vacuum pump and watch your Petri dish.
When the bubble mixture looks like it will overflow, stop the pump.
Close the inlet/outlet by turning the black valve. Unhook the clear vacuum tubing.
Open the inlet/outlet and let the air out.
Restart the vacuum pump and repeat these steps (~3-4x) until all bubbles are gone.
This should take you ~15 minutes.
Make sure your dishes and lids are labeled.
bake at 60°C overnight to cure the PDMS.
The top (as it sits in the Petri dish) of your PDMS micromixer device is called the “viewing side” while the bottom (the side with the channel indentations) is termed the “channel side.”  The Channel Side Must Be Kept Clean At ALL Times!!
Sealing Device
Clean glass slide with soap, acetone, IPA, then with water.  Dry with air gun (located in on first bench).
Using the Xacto knife/razor cut out around your device with enough room around the inlets and outlet to make a good seal
Make sure the channel side is even, with no irregularities caused by the PDMS molding around the tape or wafer edges.
Place your PDMS device in the clean square Petri dish – channel side down.
Using the purple needles, punch holes in your inlets and outlets. Firmly push down on the needle and gently pull out of the PDMS.  Use the long yellow needles to push out the PDMS plug that should be now stuck in the barrel of the purple needle.
Make Sure the PDMS Plug has been Removed. If there is no plug in the barrel, it means your channel has not been punched through fully. Use the yellow needle to push the plug out of the device.
Plasma Treatment:
To get the PDMS to stick to either glass or silicon, you must treat the two surfaces with oxygen plasma to create reactive groups for bonding.
Place your device with the CHANNEL SIDE FACING UP and the binding surface – a clean glass slide into the plasma cleaner/sterilizer. Make sure that at least one other group is ready to use the machine.
Close the pressure valve (black knob) on the door.  Turn on the vacuum pump for approximately 5 minutes to evacuate the chamber of the sterilizer.
After 5 minutes, turn on the power switch (the setting dial should be on “high”).  This turns on the RF coil and should ignite the plasma in about 30 sec.
When the plasma ignites (you can tell by the purple glow you seen in the chamber window), start timing for 1 minute.
After 1 minute, turn off the power switch, and then the vacuum switch and slowly open the valve on the door to allow air to enter the chamber.
THIS STEP IS TIME SENSITIVE:  when you have taken your samples, move quickly out of the way for the next group!
When the pressure has normalized, remove your samples from the chamber and IMMEDIATELY invert the CHANNEL SIDE of your device onto the glass/silicon surface that was face up in the chamber.
Gently press down on the device to ensure contact between the two surfaces.  You should be able to initially see edge effects between portions that are in contact and those that are not.
If you realize that a part of your channel is uneven – inhibiting proper contact, Quickly use the Xacto knife to remove uneven edges.
Bake for at least 15 min. at 95°C on a hot plate to promote bonding between the surfaces.
After 15-25 minutes, take your device off the hot plate, and allow to cool for a minute.
Insert tubing in inlets and outlet.
Streptavidin Coating and Fluorescent Testing
*All Needles and Syringes must be disposed of in the Sharps Container*
Cut tubing and insert into device at the inflow and outflow ports.
Using a syringe, fill the device with the following chemicals and incubate for the corresponding times. Remember to use different syringes for different chemicals. When inserting the chemicals, gently push down on the syringe with you thumb.
45 min in 4% (by volume) MPTMS in ethanol
20 min in 1mM GMBS
45 min in 25ng/mL NeutrAvidin in PBS
Fill device with DI water and store in 4 degree fridge.
If you are adding ATTO520 ( http://www.sigmaaldrich.com/etc/medialib/docs/Sigma/Datasheet/6/77810dat.Par.0001.File.tmp/77810dat.pdf ), fill device with the solution for 20min at 5ul/min. Use a Harvard PHD 2000 syringe pump and cover the entire setup with aluminum foil so that the fluorescent probe is exposed to as little light as possible.
Flush the device with a syringe filled with air, then take pictures.
Store in DI water in 4 degree fridge in the dark.
Protocol is a standard chemistry procedure from Gleghorn et al: http://www.ncbi.nlm.nih.gov/pubmed/20024046
Reference was generously provided by Dr. Kirby (Cornell University, Mechanical and Aerospace Engineering)
Cell Lysis
Spin cells down at 13000rpm for 10 min at 19 degrees celsius
Remove supernatant and measure the mass of the pellet
Add 2.5ml of Bugbuster Master for every 1 gram of cells
Pipet up and down until the cells are well mixed, then shake for 20min
Spin solution down again at 13000rpm for 10 min at 19 degrees celsius.
Collect supernatant for experimental use
Filtration
For use with PD-10 Desalting Columns from GE Healthcare
Remove the top cap and pour off the column storage solution
Remove the top filter using forceps
Cut the sealed end of the column at notch
Put the PD-10 Desalting column into a 50ml collection tube by suing the column adapter
Fill up the column with 5ml of PBS and allow the PBS to enter the packed bed completely
Repeat the above step 5 times
Spin down at 1000rpm for 2min
Put in lysate from cell lysis, spin down at 1000rpm for 2min
Retrieved from " http://2011.igem.org/Team:Cornell/Protocol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
