"
Page
Discussion
View source
History
teams
Log in
 
Team:Cornell/Week 13
From 2011.igem.org
Results | Protocol | Notebook | Parts Submitted
Week 1 | Week 2 - 5 | Week 6 | Week 7 | Week 8 | Week 9 | Week 10 | Week 11 | Week 12 | Week 13 | Week 14 | Week 15 | Week 16 | Week 17 | Week 18 | Week 19 | Week 20 | Week 21 |
August 28th - September 3rd
Sunday, August 28
Monday, August 29
Tuesday, August 30
Afternoon - Evening lab work done by: Claire Paduano, Youjin Cho, Maneesh Gupta, James Mathew, Charlie Chung
Digestion Reactions for BioBrick Assembly
AviTagged RFP Insert
34.6µL ddH2O
7.9µL AviTagged RFP DNA (1 µg)
5µL 10x NEBuffer 3
0.5µL 100x BSA
1µL EcoRI
1µL PstI
50µL Total
AviTagged VioA Insert
28.3µL ddH2O
14.2µL AviTagged VioA DNA (1µg)
5µL 10x NEBuffer 3
0.5µL 100x BSA
1µL EcoRI
1µL PstI
50µL Total
AviTagged VioB Insert
26.7µL ddH2O
15.8µL AviTagged VioB DNA (1µg)
5µL 10x NEBuffer 3
0.5µL 100x BSA
1µL EcoRI
1µL PstI
50µL Total
Avi-Tagged VioE Insert
29.8µL ddH2O
11.7µL AviTagged VioE DNA (1µg)
5µL 10x NEBuffer 3
0.5µL 100x BSA
1µL EcoRI
1µL PstI
50µL Total
pSB1C3 Vector Backbone
40µL pSB1C3 DNA
5µL 10x NEBuffer 3
2.5µL ddH2O
0.5µL 100x BSA
1µL EcoRI
1µL PstI
50µL Total
Dephosphorylation of pSB1C3 5' Ends via CIAP Treatment
- Please see the Protocol section for the procedure followed
Gel Purification of Digestion Reaction Products via Electrophoresis
- Please see the Protocol section for the procedure followed
Gel Extraction of DNA
- Please see the Protocol section for the procedure followed
- VioA and VioB genes migrated closely with their "native" pZE12 backbone, so initially both insert gene and backbone DNA bands were cut out
- After separating the two, a labeling mishap prevented the proper isolation of just the insert gene
- Thus, pZE12 backbone also underwent purification and ligation
NanoDrop Spectrophometry for Purified DNA Quantification
- RFP = 16.8ng/µL
- VioA (gene?) = 33.7ng/µL
- VioA (pZE12?) = 17.6ng/µL
- VioB (gene?) = 27.8ng/µL
- VioB (pZE12?) = 18.6ng/µL
- VioE = 8.6ng/µL
- pSB1C3 backbone = 16.4ng/µL
Ligation Reactions for BioBrick Inserts with iGEM Backbone
- Volumes are calculated as described in the Protocol section or, equivalently, with the Ligation Reaction Calculator spreadsheet
RFP = 678bp
VioA = 1257bp
VioB = 2997bp
VioE = 576bp
pSB1C3 = 2070bp
AviTagged RFP + pSB1C3
6.1µL pSB1C3 backbone
5.8µL AviTagged RFP DNA
5.1µL ddH2O
2µL 10x T4 DNA ligase buffer
1µL T4 DNA ligase
20µL Total
AviTagged VioA (gene?) + pSB1C3
6.1µL pSB1C3 backbone
5.5µL ddH2O
5.4µL AviTagged VioA DNA (gene?)
2µL 10x T4 DNA ligase buffer
1µL T4 DNA ligase
20µL Total
AviTagged VioA (backbone?) + pSB1C3
Note: VioA (backbone?) treated as VioA gene insert in terms of #bp in Ligation Reaction Calculator
10.4µL AviTagged VioA DNA (backbone?)
6.1µL pSB1C3 backbone
2µL 10x T4 DNA ligase buffer
0.6µL ddH2O
1µL T4 DNA ligase buffer
20µL Total
AviTagged VioB (gene?) + pSB1C3
12.2µL AviTagged VioB DNA (gene?)
4.8µL pSB1C3 backbone
2µL 10x T4 DNA ligase buffer
1µL T4 DNA ligase
20µL Total
AviTagged VioB (backbone?) + pSB1C3
Note: VioB (backbone?) treated as VioB gene insert in terms of #bp in Ligation Reaction Calculator
13.5µL AviTagged VioB DNA (backbone?)
3.5µL pSB1C3 backbone
2µL 10x T4 DNA ligase buffer
1µL T4 DNA ligase
20µL Total
AviTagged VioE + pSB1C3
9.7µL AviTagged VioE DNA
6.1µL pSB1C3 backbone
2µL 10x T4 DNA ligase buffer
1.2µL ddH2O
1µL T4 DNA ligase
20µL Total
Overnight benchtop incubation
Wednesday, August 31
Afternoon lab work done by: Youjin Cho
Desalted the ligated samples from Tuesday and transformed them into MD37 electrocompetent cells using electroporation.
After an hour of incubation, they were plated onto plate containing chloramphenicol.
Afternoon/Evening Lab work done by: Claire Paduano and Maneesh Gupta
Looked at ATTO590 coated chip from Aug 27th under fluorescent microscope: channels still fluorescent
Photo to be posted soon
Streptavidin Coating and Fluorescent Probes
1) 45 min in 4% (by volume) MPTMS in ethanol
2) 20 min in 1mM GMBS
3) 45 min in 25ng/mL NeutrAvidin in PBS
Streptavidin coating protocol from Gleghorn et al: http://www.ncbi.nlm.nih.gov/pubmed/20024046
4) 20 min in fluorescent probe
Incubated one chip with ATTO520, one in ATTO590
Flow Experiments
Details and photos to be posted soon
ATTO520 gives signal under both Texas Red and GFPA filters
ATTO590 gives signal under only Texas Red
- In future flow experimental design, keep in mind that we cannot differentiate between ATTO590 and 520 signal under Texas Red filter
Thursday, September 1
Morning lab work done by: Claire Paduano and Nancy Li
Looked at ATTO590 coated chip from Aug 27th and chips from Aug31st under fluorescent microscope: channels still fluorescent
Coated three chips with streptavidin
Streptavidin coating
Protocol from Gleghorn et al: http://www.ncbi.nlm.nih.gov/pubmed/20024046
1) 45 min in 4% (by volume) MPTMS in ethanol
2) 20 min in 1mM GMBS
3) 45 min in 25ng/mL NeutrAvidin in PBS
Evening lab work done by: Maneesh Gupta
Did continuous flow experiments to test resilience of biotin-avitin binding
Used same exposure (83.33) in all images taken
Friday, September 2
Afternoon lab work done by: Charlie Chung
Miniprep DNA purification of cultures containing pSB1C3 vector backbone with (RFP + AviTag), (VioA + AviTag), (VioB + AviTag), and (VioE + AviTag) inserts -- in preparation for iGEM BioBrick submission
Saturday, September 3
Afternoon lab work done by: Jim Mathew, Charlie Chung, Nancy Li
Quantification of Purified DNA Samples via NanoDrop Spectrophotometry
- All 260/280nm ratios were near 1.8, indicting pure DNA
- All inserts listed below are ligated with the AviTagged pSB1C3 vector backbone
RFP 1 -- 344.0ng/µL
RFP 2 -- 256.4ng/µL
RFP 3 -- 230.4ng/µL
VioA 1-1 -- 205.9ng/µL
VioA 1-2 -- 494.3ng/µL
VioA 1-3 -- 370.8ng/µL
VioA 2-1 -- 892.5ng/µL
VioA 2-2 -- 203.8ng/µL
VioA 2-3 -- 206.0ng/µL
VioB 1-1 -- 503.9ng/µL
VioB 1-2 -- 186.1ng/µL
VioB 1-3 -- 398.8ng/µL
VioB 2-1 -- 868.9ng/µL
VioB 2-2 -- 145.6ng/µL
VioB 2-3 -- 174.6ng/µL
VioE 1 -- 266.2ng/µL
VioE 2 -- 552.9ng/µL
VioE 3 -- 400.7ng/µL
DpnI Digestion of PCR Reaction Product
- PCR with specially designed primers to delete the iGEM restriction enzyme cut sites, which may have been forming too much distance between the ribosome binding site and the start codon of our gene
- first half of the forward primer consists of the base pairs flanking the deletion region to the left, second half of the forward primer consists of the base pairs flanking the deletion region to the right
- when the primer anneals to the template, it is complementary to the nucleotides on either side of the deletion region and pinches them in, forcing the deletion region to loop out without a complementary strand
Transformation via Electroporation
Note: Either the DH5α or MC4100 strain will have the degradation system of unmethylated DNA knocked out
- (RFP + AviTag) into DH5α and MC4100
- (VioA + AviTag) into DH5α and MC4100
- (VioB + AviTag) into DH5α and MC4100
- (VioE + AviTag) into DH5α and MC4100
Plating of Transformed Bacteria
- Ampicillin-treated plates are incubating at 37°C in Weill Hall
Retrieved from " http://2011.igem.org/Team:Cornell/Week_13 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
