<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cornell_Week_10"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cornell/Week_10">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cornell/Week_10&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/Week_10&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/Week_10&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cornell/Week_10" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cornell/Week 10</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV align="center"><B><H3><FONT size="3"><P><A href="/Team:Cornell/Results" title="Team:Cornell/Results"> Results</A>  |
<A href="/Team:Cornell/Protocol" title="Team:Cornell/Protocol"> Protocol</A>  |
<A href="/Team:Cornell/Week_1" title="Team:Cornell/Week 1"> Notebook</A> |
<A href="/Team:Cornell/Parts" title="Team:Cornell/Parts"> Parts Submitted</A></P></FONT></H3></B></DIV><DIV align="center"><B><P><A href="/Team:Cornell/Week_1" title="Team:Cornell/Week 1"> Week 1</A>  |
<A href="/Team:Cornell/Week_2" title="Team:Cornell/Week 2"> Week 2 - 5</A> |
<A href="/Team:Cornell/Week_6" title="Team:Cornell/Week 6"> Week 6</A> |
<A href="/Team:Cornell/Week_7" title="Team:Cornell/Week 7"> Week 7</A> |
<A href="/Team:Cornell/Week_8" title="Team:Cornell/Week 8"> Week 8</A> |
<A href="/Team:Cornell/Week_9" title="Team:Cornell/Week 9"> Week 9</A> |
<STRONG class="selflink"> Week 10</STRONG> |
<A href="/Team:Cornell/Week_11" title="Team:Cornell/Week 11"> Week 11</A> |
<A href="/Team:Cornell/Week_12" title="Team:Cornell/Week 12"> Week 12</A> |
<A href="/Team:Cornell/Week_13" title="Team:Cornell/Week 13"> Week 13</A> |
<A href="/Team:Cornell/Week_14" title="Team:Cornell/Week 14"> Week 14</A> |
<A href="/Team:Cornell/Week_15" title="Team:Cornell/Week 15"> Week 15</A> |
<A href="/Team:Cornell/Week_16" title="Team:Cornell/Week 16"> Week 16</A> |
<A href="/Team:Cornell/Week_17" title="Team:Cornell/Week 17"> Week 17</A> |
<A href="/Team:Cornell/Week_18" title="Team:Cornell/Week 18"> Week 18</A> |
<A href="/Team:Cornell/Week_19" title="Team:Cornell/Week 19"> Week 19</A> |
<A href="/Team:Cornell/Week_20" title="Team:Cornell/Week 20"> Week 20</A> |
<A href="/Team:Cornell/Week_21" title="Team:Cornell/Week 21"> Week 21</A> |
</P></B></DIV><P><FONT size="5"><I> August 7th - August 13th </I></FONT></P><H2><SPAN class="mw-headline" id="Sunday.2C_August_7">Sunday, August 7</SPAN></H2><P>Morning lab work done by: Youjin Cho, Claire Paduano
</P><DL><DD><B>Objective</B></DD><DD>Prepare VioE in AviTagged pZE12 vector backbone for VioA and VioB gene inserts
</DD></DL><DL><DD><B>Digestion Setup</B></DD><DD>23.65μL H2O
</DD><DD>18.6μL VioE in AviTagged pZE12 vector backbone (1μg)
</DD><DD>5μL 10x NEBuffer 2
</DD><DD>0.5μL 100x BSA
</DD><DD>1.25μL KpnI
</DD><DD><U>1μL HindIII</U></DD><DD>50μL Total
</DD></DL><DL><DD>KpnI has only 75% efficiency in Buffer 2, so added 0.25μL more KpnI to digestion mixture
</DD><DD>Incubate in 37°C water bath for 2 hours
</DD></DL><P>Afternoon lab work done by: Jim Mathew, Charlie Chung
</P><DL><DD><B>Objective</B></DD><DD>Ligate VioA and VioB genes onto their own pZE12 vector backbone, in place of the VioE that is now digested out
</DD></DL><DL><DD><B>Ligation Reaction Setup</B></DD><DD><UL><LI><B>VioA + AviTagged pZE12 vector backbone</B></LI></UL></DD><DD>11.4μL pZE12 backbone (cut with KpnI &amp; HindIII; former VioE gene digested out)
</DD><DD>3.3μL VioA gene insert (cut with KpnI &amp; HindIII)
</DD><DD>2.3μL H2O
</DD><DD>2μL 10x T4 DNA ligase buffer
</DD><DD><U>1μL T4 DNA ligase</U></DD><DD>20μL Total
</DD></DL><UL><LI><B>VioB + AviTagged pZE12 vector backbone</B></LI></UL><DL><DD>9μL VioB gene insert (cut with KpnI &amp; HindIII)
</DD><DD>8μL pZE12 backbone (cut with KpnI &amp; HindIII; former VioE gene digested out)
</DD><DD>2μL 10x T4 DNA ligase buffer
</DD><DD><U>1μL T4 DNA ligase</U></DD><DD>20μL Total
</DD></DL><UL><LI><B>Control Ligation for VioA and VioB Gene Inserts</B></LI></UL><DL><DD>11.4μL pZE12 backbone (cut with KpnI &amp; HindIII; former VioE gene digested out)
</DD><DD>5.6μL H2O (gene insert volumes go toward water volume)
</DD><DD>2μL 10x T4 DNA ligase buffer
</DD><DD><U>1μL T4 DNA ligase</U></DD><DD>20μL Total
</DD></DL><DL><DD>Incubate all ligation reaction constructs overnight in 16°C water bath
</DD></DL><P>Evening lab work done by: Charlie Chung
</P><DL><DD><B>Objective</B><UL><LI>Because previous DH5α colonies transformed with (VioE + AviTagged backbone) are very small, in addition to one failed 5mL culture and one discarded Miniprep sample, incubated plate for four more hours
</LI><LI>Picked three new colonies. Labeled as &quot;VioE + AviTag + BB #4, 5, 6&quot;
</LI><LI>5mL LB cultures with 5μL ampicillin incubated overnight in 37°C shaker of Room 304
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Monday.2C_August_8">Monday, August 8</SPAN></H2><P>Morning lab work done by: Youjin Cho, James Mathew
</P><DL><DD><B>Objective</B></DD><DD><UL><LI>Transform the overnight ligation of VioA/VioB + AviTag + pZE12
</LI><LI>Send the Miniprepped samples (GFP, RFP, VioE) from Saturday in for sequencing
</LI><LI>Miniprep the additional VioE samples that were grown overnight
</LI></UL></DD></DL><DL><DD><B>Transformation</B></DD><DD><UL><LI>Desalted the ligation reaction samples (VioA/VioB + AviTag + pZE12) on a membrane for 15 minutes
</LI><LI>After adding the samples into DH5α cell lines, electroporated them and put into shaker for 45 minutes
</LI><LI>The samples were plated on the plate with ampicillin
</LI></UL></DD></DL><DL><DD><B>Sequencing</B></DD><DD><UL><LI>Using the reverse primer dilution, the samples were sent in for sequencing
</LI></UL></DD><DD>GFP + AviTag + pZE12 - 1
</DD><DD>GFP + AviTag + pZE12 - 2
</DD><DD>GFP + AviTag + pZE12 - 3
</DD><DD>RFP + AviTag + pZE12 - 1
</DD><DD>RFP + AviTag + pZE12 - 2
</DD><DD>RFP + AviTag + pZE12 - 3
</DD><DD>VioE + AviTag + pZE12 - 2
</DD></DL><DL><DD><B>Miniprepping the VioE samples</B></DD><DD><UL><LI>The samples were Miniprepped using standard Qiagen Miniprep protocol
</LI></UL></DD><DD>VioE + AviTag + pZE12 - 4
</DD><DD>VioE + AviTag + pZE12 - 5
</DD><DD>VioE + AviTag + pZE12 - 6
</DD></DL><P>Evening lab work done by: Charlie Chung
</P><DL><DD><UL><LI>Checked to see if colonies were ready to pick from transformed plates of (VioA + AviTagged pZE12 backbone) and (VioB + AviTagged pZE12 backbone)
</LI></UL></DD><DD><UL><LI>VioA plate is showing early signs of a good number of colonies. Too small for picking, however
</LI><LI>VioB and Control plates look pretty empty (but just may not have grown up enough to tell)
</LI></UL></DD></DL><UL><LI>Will be best if colonies were picked Tuesday (tomorrow) morning. Please find the 3 plates still in the 37°C incubator
</LI></UL><H2><SPAN class="mw-headline" id="Tuesday.2C_August_9">Tuesday, August 9</SPAN></H2><P>Evening lab work done by: Youjin Cho
</P><DL><DD><UL><LI>Picked colonies from transformed plates of (VioA + AviTagged pZE12 backbone) and (VioB + AviTagged pZE12 backbone).
</LI></UL></DD><DD>VioA 1,2,3
</DD><DD>VioB 1,2,3
</DD></DL><UL><LI>Put them into the 37°C shaker around 5:30pm
</LI></UL><H2><SPAN class="mw-headline" id="Wednesday.2C_August_10">Wednesday, August 10</SPAN></H2><P>Morning lab work done by: Youjin Cho, Alyssa Henning
</P><DL><DD><B>Objective</B></DD><DD><UL><LI>Miniprep the samples of VioA/VioB + AviTag + pZE12 backbone and measure their concentrations
</LI><LI>Submit the Miniprepped samples for sequencing
</LI><LI>Since the sequencing results showed that GFP did not work, run a PCR reaction for more GFP
</LI></UL></DD></DL><DL><DD><B>DNA Purification of VioA, VioB Samples</B></DD><DD><UL><LI>The samples were Miniprepped using standard Qiagen Miniprep protocol
</LI></UL></DD><DD>VioA + AviTag + pZE12 - 1 = 67.6 ng/µL
</DD><DD>VioA + AviTag + pZE12 - 2 = 70.6 ng/µL
</DD><DD>VioA + AviTag + pZE12 - 3 = 71.4 ng/µL
</DD><DD>VioB + AviTag + pZE12 - 1 = 66.0 ng/µL
</DD><DD>VioB + AviTag + pZE12 - 2 = 70.6 ng/µL
</DD><DD>VioB + AviTag + pZE12 - 3 = 75.9 ng/µL
</DD></DL><DL><DD><B>Sequencing</B><UL><LI>The Miniprepped samples were prepared for sequencing using the diluted reverse pZE12 primer
</LI></UL></DD><DD><U>Order Number</U>: 10254703
</DD><DD>VioA-1
</DD><DD>VioA-2
</DD><DD>VioA-3
</DD></DL><DL><DD>VioB-1
</DD><DD>VioB-2
</DD><DD>VioB-3
</DD></DL><DL><DD><B>PCR for GFP</B><UL><LI>Since the result from the sequencing showed that GFP did not work, ran a PCR reaction using the original template of GFP in pZE12
</LI><LI>Ran two reactions and labeled them 1,2 GFP iGEM 8/10
</LI></UL></DD><DD> 1 GFP = the sample from May 5 of GFP on pZE12 with concentration of 50.6 ng/µL
</DD><DD> 2 GFP = the sample missing the date with concentration of 20.2 ng/µL
</DD></DL><UL><LI>The PCR was ran with melting temperature of 54.2°C for 1 minute
</LI></UL><H2><SPAN class="mw-headline" id="Thursday.2C_August_11">Thursday, August 11</SPAN></H2><P>Morning lab work at Olin Hall done by: Youjin Cho
</P><DL><DD><B>Objective</B></DD><DD><UL><LI>Purify the GFP that was PCRed off the GFP with pZE12 backbone
</LI><LI>Digest 1µg of the DNA of GFP and AviTag with pZE12 backbone
</LI></UL></DD></DL><DL><DD><B>PCR Purification</B></DD><DD><UL><LI>The GFP PCR samples were purified using the Qiagen Kit
</LI><LI>Using NanoDrop the two samples of GFP were quantified:
</LI></UL></DD><DD>GFP-1 = 206.4ng/ul
</DD><DD>GFP-2 = 159.2ng/ul
</DD></DL><DL><DD><B>Digestion Setup</B></DD><DD><B>GFP-1</B></DD><DD>37.2μL H2O
</DD><DD>4.84μL GFP sample-1 (1μg)
</DD><DD>5μL 10x NEBuffer 4
</DD><DD>1μL KpnI-HF (HF = high fidelity)
</DD><DD><U>1μL SphI-HF</U></DD><DD>50μL Total
</DD></DL><DL><DD><B>GFP-2</B></DD><DD>35.7μL H2O
</DD><DD>6.28μL AviTagged pZE12 vector backbone (1μg)
</DD><DD>5μL 10x NEBuffer 4
</DD><DD>1μL KpnI-HF (HF = high fidelity)
</DD><DD><U>1μL SphI-HF</U></DD><DD>50μL Total
</DD></DL><DL><DD><B>AviTagged pZE12 backbone</B></DD><DD>32.4μL H2O
</DD><DD>9.59μL AviTagged pZE12 vector backbone (1μg)
</DD><DD>5μL 10x NEBuffer 4
</DD><DD>1μL KpnI-HF (HF = high fidelity)
</DD><DD><U>1μL SphI-HF</U></DD><DD>50μL Total
</DD></DL><DL><DD>Incubate in 37°C water bath for 2 hours
</DD></DL><DL><DD><B>Dephosphorylation of 5' Ends of Vector Backbone</B></DD><DD><UL><LI>Add 1μL of Calf Intestine Alkaline Phosphatase (CIAP) to digested vector backbone in order to prevent self-ligation without gene insert included
</LI><LI>Incubate at 50°C for 5 minutes
</LI><LI>After adding 6x loading buffer, run samples through agarose gel electrophoresis
</LI></UL></DD></DL><DL><DD><B>Gel Extraction and Purification of Digested Backbone</B></DD><DD><UL><LI>Followed standard Qiagen Gel Extraction protocol for samples from digestion reaction
</LI><LI>NanoDrop spectrophotometry on the duplicate samples reported:
</LI></UL></DD><DD>GFP-1 = 10.3 ng/μl
</DD><DD>GFP-2 = 7.5  ng/μl
</DD><DD>AviTag + backbone = 11.6ng/μl
</DD></DL><P>Weill Hall lab work done by: Charlie Chung, Claire Paduano, Bill Jo, Nick Kramer, Youjin Cho
</P><DL><DD><B>Objective</B></DD><DD><UL><LI>Coat microfluidic device with steptavidin
</LI></UL></DD></DL><DL><DD><B>Diluting Chemicals</B></DD><DD><UL><LI>4% MPTMS in ethanol (MPTMS = 3-mercaptopropyl trimethoxysilane)
</LI></UL></DD><DD>4mL MPTMS
</DD><DD>96mL EtOH
</DD></DL><UL><LI>1mM GMBS in ethanol
</LI></UL><DL><DD>0.014gGMBS (280.2g/mol)(GMBS = N-g-maleimidobutyryloxy succinimide ester)
</DD><DD>50mL EtOH
</DD></DL><UL><LI>25ng/mL NeutrAvidin in PBS - serial dilutions
</LI></UL><DL><DD>Dilute 0.25mg in 10mL PBS = 0.025mg/mL
</DD><DD>Take 1mL of first dilution and bring to 10mL = 2.5µg/mL
</DD><DD>Take 1mL of second dilution and bring to 100mL = 25ng/mL
</DD></DL><DL><DD><B>Streptavidin Coating</B></DD><DD><UL><LI>Protocol from Gleghorn et al: <A href="http://www.ncbi.nlm.nih.gov/pubmed/20024046" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/20024046</A></LI></UL></DD><DD>(1) 45 min in 4% (by volume) MPTMS in ethanol
</DD><DD>(2) 20 min in 1mM GMBS
</DD><DD>(3) 45 min in 25ng/mL NeutrAvidin in PBS
</DD></DL><P>Evening lab work done by: Charlie Chung
</P><DL><DD><UL><LI>Started three additional cultures of (RFP + AviTagged pZE12 backbone) in 3mL LB with 3μL ampicillin
</LI></UL></DD><DD><UL><LI>Picked Colonies #4, 5, and 6 from (RFP + AviTag + pZE12) plate
</LI></UL></DD><DD>- Because sequencing results are positive, assuming all colonies are also successful clones
</DD></DL><UL><LI>Incubating overnight in 37°C shaker of Room 304
</LI></UL><DL><DD>To be used for tomorrow (Fri, 8/12) when lysing bacteria and releasing RFP-containing lysate into microfluidic channel for observation under fluorescent microscope
<UL><LI>Started three additional cultures of (AviTagged pZE12 backbone) in 3mL LB with 3μL ampicillin
</LI></UL></DD><DD><UL><LI>Picked Colonies #4, 5, and 6 from (AviTag + pZE12) plate
</LI></UL></DD><DD>- Because sequencing results are positive, assuming all colonies are also successful clones
</DD></DL><UL><LI>Incubating overnight in 37°C shaker of Room 304
</LI></UL><H2><SPAN class="mw-headline" id="Friday.2C_August_12">Friday, August 12</SPAN></H2><DL><DD><B>Team Meeting</B></DD><DD><UL><LI>Checked the sequencing for VioA and VioB -- VioA worked, but VioB didn't
</LI></UL></DD><DD>- Need to PCR off the VioB gene and repeat ligation with AviTagged backbone
</DD></DL><UL><LI>Need to figure out the flow rate that we will use for the microfluidics device
</LI><LI>Ask Dr. Archer about the protocol for taking picture/video with the microscope
</LI></UL><DL><DD><B>Lab Work for This Week</B></DD><DD><I>Microfluidics</I></DD><DD><OL><LI>Need to induce the RFP cultures overnight (Friday evening)
</LI><LI>Lyse the cells and run through the device to see if works (Saturday afternoon)
</LI></OL></DD></DL><DL><DD><I>Lab Work</I></DD><DD><OL><LI>Transform the ligation of GFP + AviTag + pZE12 backbone and incubate overnight (Friday evening)
</LI><LI>Store the plates away in 4°C and pick colonies from them late at night (Saturday)
</LI><LI>Miniprep the GFP samples (Sunday afternoon)
</LI><LI>Submit (GFP + AviTag + pZE12) for sequencing
</LI><LI>Set up PCR reaction for VioB gene and repeat the ligation steps (next week)
</LI></OL></DD></DL><P>Afternoon lab work done by: Youjin Cho, Charlie Chung
</P><DL><DD><B>Objective</B><UL><LI>Prepare a subculture of bacteria with (RFP + AviTagged pZE12) plasmid for production of RFP
</LI><LI>Transformation of (GFP + AviTagged pZE12 backbone) into DH5α electrocompetent bacteria and plated with 1:1000 ampicillin.
</LI><LI>Prepare freezer stocks of DH5α bacteria transformed with (RFP + AviTagged pZE12) and AviTagged pZE12
</LI><LI>Miniprep remaining volume of (RFP + AviTagged pZE12) and AviTagged pZE12 cultures
</LI></UL></DD><DD><B>Preparing a Subculture</B><UL><LI>Set up two cuvettes for preliminary optical density (OD) reading: (1) control (2) RFP sample from Thursday evening
</LI></UL></DD><DD>- Purpose of preliminary OD reading is to determine how much RFP sample you need to add to a new 25mL culture
</DD><DD>- Use a 1:10 dilution of sample to minimize error when running the spectrophotometer
</DD><DD><UL><LI>Control: 1000µL LB
</LI><LI>RFP Sample: 900µL LB + 100µL (RFP + AviTagged pZE12) Colony #5
</LI></UL></DD></DL><UL><LI>RFP Sample OD = 0.283 (treat as [bacteria with RFP]), which translates to actual OD of 2.83 in Thursday's 5mL culture tube (after undoing the 1:10 dilution)
</LI></UL><UL><LI>Use dilution equation to determine how much RFP bacteria culture is needed for the 25mL culture
</LI></UL><DL><DD>(2.83)(? µL) = (desired beginning [RFP bacteria] = 0.05)(25mL = 25000µL)
</DD><DD>? = 441.7µL Thursday's RFP bacteria culture to 25mL LB + 25µL ampicillin
</DD></DL><UL><LI>Incubate new 25mL RFP bacteria subculture in 37°C shaker for ~2 hours and 45 minutes
</LI><LI>At end of incubation time, check OD. Target OD = 0.6-0.8, which means ready for induction of RFP production via IPTG
</LI><LI>Induce 25mL RFP bacteria culture with 25µL 1M IPTG for desired 1mM addition (completed at 7:25pm)
</LI><LI>Incubate induced 25mL culture flask on room temperature shaker
</LI></UL><DL><DD><B>Preparation of Freezer Stock</B><UL><LI>1:1 volume ratio of 30% glycerol and bacteria culture media
</LI></UL></DD><DD>- Added 500µL 30% glycerol and 500µL (RFP + AviTagged pZE12) into three cryogenic storage tubes
</DD><DD>- Added 500µL 30% glycerol and 500µL AviTagged pZE12 into three cryogenic storage tubes
</DD><DD>- Stored in the -80°C freezer of Room 303
</DD></DL><DL><DD><B>Miniprep</B><UL><LI>Followed standard Qiagen Miniprep protocol
</LI><LI>NanoDrop spectrophotometry to determine DNA concentrations
</LI></UL></DD><DD>AviTagged pZE12 backbone Sample 1 = 102.2ng/µL
</DD><DD>AviTagged pZE12 backbone Sample 2 = 135.2ng/µL
</DD><DD>(RFP + AviTagged pZE12 backbone) Sample 1 = 127ng/µL
</DD><DD>(RFP + AviTagged pZE12 backbone) Sample 2 = 97.3ng/µL
</DD></DL><H2><SPAN class="mw-headline" id="Saturday.2C_August_13">Saturday, August 13</SPAN></H2><P>Afternoon lab work done by: Youjin Cho, Charlie Chung
</P><DL><DD><B>Objective</B></DD><DD><UL><LI>Lyse the 25mL culture of IPTG-induced (RFP + AviTagged backbone) bacteria
</LI><LI>Collect RFP protein
</LI></UL></DD></DL><DL><DD><B>Procedure</B></DD><DD><OL><LI>Bring BugBuster Protein Extraction lysis buffer (EMD4Biosciences of Merck) to room temperature
</LI><LI>Transfer 25mL culture to 50mL conical that is previously tared (later, we will need to measure mass of bacterial pellet). Centrifuge at 4500rpm for 25 minutes
</LI><LI>Pour out supernatant. Measure mass of bacterial pellet. 0.4g * (5mL BugBuster per g of cells) = 2mL BugBuster
</LI><LI>Rock for 20 minutes
</LI><LI>Centrifuge at maximum rpm for 20 minutes at 4°C
</LI><LI>Collect supernatant (RFP will be soluble here)
</LI></OL></DD></DL><P>Evening lab work done by: Charlie Chung
</P><DL><DD><UL><LI>Attempted to pick and culture colonies of (GFP + AviTagged backbone). However, couldn't find keys to Room 303. Thus, I was <B>unable</B> to pick and begin 3mL cultures
</LI><LI>Sorry for creating more work, but please complete this task Sunday morning
</LI></UL></DD></DL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Cornell/Week_10">http://2011.igem.org/Team:Cornell/Week_10</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cornell/Week_10" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cornell/Week_10" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cornell/Week_10&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cornell/Week_10&amp;oldid=132675" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>