<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cornell_Description"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cornell/Description">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cornell/Description&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/Description&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/Description&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cornell/Description" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cornell/Description</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV align="center"><B><H3><FONT size="3"><P><STRONG class="selflink"> Project Description</STRONG> |
<A href="/Team:Cornell/Future" title="Team:Cornell/Future"> Future Directions</A> |
<A href="/Team:Cornell/Business" title="Team:Cornell/Business"> Business Development</A> |
<A href="/Team:Cornell/Outreach" title="Team:Cornell/Outreach"> Outreach/HP</A> |
<A href="/Team:Cornell/Safety" title="Team:Cornell/Safety"> Safety</A></P></FONT></H3></B></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#The_BioFactory"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">The BioFactory</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Violacein_Pathway"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Violacein Pathway</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#How_To_Make_Our_Enzymes_Stick_to_the_BioFactory:"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">How To Make Our Enzymes Stick to the BioFactory:</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#The_BioFactory_Microfluidic_System"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">The BioFactory Microfluidic System</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Light-Induced_Lysis"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Light-Induced Lysis</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#DNA_Assembly_Methods"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">DNA Assembly Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Site-Directed_Mutagenesis"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Site-Directed Mutagenesis</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Annealed_Primer_Insertion"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Annealed Primer Insertion</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-9"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><FONT face="Comic Sans MS"><DIV style="text-align: left;"><H1><SPAN class="mw-headline" id="The_BioFactory">The BioFactory</SPAN></H1><P>Cornell’s BioFactory aims to develop a simple and efficient method for the construction of enzyme-immobilized surfaces capable of multi-step chemical reactions.  
</P><P>As more chemical production techniques begin to utilize enzymatic reactions, genetic engineers must consider ways to resolve competing side reactions and the toxic accumulation of intermediates, reduce purification costs, and correctly express non-native enzymes or proteins in bacteria. In some cases, such challenges could be more easily resolved by simply extracting the molecular metabolic mechanisms and produce the target compound in a cell-free environment. We believe a cell-free system for biosynthesis can resolve these issues and still use the power of bacteria to help build devices capable of producing complex organic compounds.  
</P><P>Over the summer, Cornell's iGEM team engineered strains of <I>E. coli</I> to produce modified enzymes from the biosynthetic pathway of Violacein which were immobilized on the surface of microfluidic devices and capable of converting an initial feed of substrate into prodeoxyviolacein, a direct intermediate of the violacein pathway. The microfluidic chips were designed, built and tested in the lab using Cornell’s modern clean room and nanofabrication facilities. We additionally designed and began construction of a light-induced apoptosis system capable of lysing bacterial cultures, producing the necessary enzymes without the use of expensive reagents or extensive protocols.
</P><H1><SPAN class="mw-headline" id="Violacein_Pathway">Violacein Pathway</SPAN></H1><P>We chose to use the violacein pathway fully characterized by Balibar et. al.<SUP>3</SUP> as our model enzyme-mediated-reaction. The violacein pathway involves five enzymes (VioA, VioB, VioC, VioD, and VioE) in the conversion of L-tryptophan into violacein, a purple chromophore. This pathway was an especially attractive candidate for our project not only because it has been thoroughly characterized in <I>E. coli</I>, but also because relatively few enzymes are needed to convert a cheap, common substrate into a visualizable product. Furthermore, <B>only three of the five enzymes (VioA, VioB, and VioE) are required to produce a colored product. L-tryptophan is converted into prodeoxyviolacein, a green pigment</B> (Figure 3A in Balibar et al.<SUP>3</SUP>). Thus, we chose to biotinylate only VioA, VioB, and VioE to provide proof-of-concept that enzymes bound to our microfluidic devices may be used to facilitate enzyme-mediated reactions.
</P><H1><SPAN class="mw-headline" id="How_To_Make_Our_Enzymes_Stick_to_the_BioFactory:">How To Make Our Enzymes Stick to the BioFactory:</SPAN></H1><DL><DD><UL><LI><B>Have the Bacteria Do the Work</B></LI></UL></DD></DL><DL><DD>We made use of the biotin-streptavidin binding interaction in fixing our biotinylated enzymes to the PDMS channel surfaces. <I>E. coli</I> has a natural mechanism encoded by the <I>birA</I> gene to produce biotin and biotinylate proteins. Thus, by appending the AviTag peptide to enzymes of interest and adding biotin to the bacterial culture for enhanced biotinylation of the AviTag, we functionalized our enzymes for biotin-NeutrAvidin binding to the microfluidic device. We chose the biotin-avidin interaction over the competing nickel-histidine binding method commonly used for protein purification, as the avidin-biotin binding interaction lasts longer.
</DD></DL><DL><DD><UL><LI><B>Chemical Treatment to Surface of BioFactory Devices</B></LI></UL></DD></DL><DL><DD>Using standard chemical procedures from Gleghorn et al.<SUP>1</SUP> provided by Dr. Kirby (Cornell University, Mechanical and Aerospace Engineering), NeutrAvidin was bonded to the surface of PDMS channels. NeutrAvidin is a deglycosylated version of avidin and is interchangeable with streptavidin. It has a similar tetramer structure, which binds with biotin in four regions. NeutrAvidin can bind greater than 12 µg of biotin per mg protein and has a dissociation constant K<SUB>d</SUB> = 10-15 M.
</DD></DL><DL><DD><B>When biotinylated enzymes come into contact with the NeutrAvidin in the devices, they bind in a ligand interaction capable of withstanding the continuous flow rates of solutions against the surfaces.</B></DD></DL><H1><SPAN class="mw-headline" id="The_BioFactory_Microfluidic_System">The BioFactory Microfluidic System</SPAN></H1><DIV><P style="margin:0in;font-family:Comic Sans MS;font-size:10.0pt"><SPAN style="font-weight:bold">Overview:</SPAN></P><P style="margin:0in;font-family:Comic Sans MS;font-size:10.0pt"> </P><P style="margin:0in;font-family:Comic Sans MS;font-size:10.0pt">In order to develop a
		scalable synthetic pathway the team required a means to apply enzymes</P><P style="margin:0in;font-family:Comic Sans MS;font-size:10.0pt">to proteins extracted
		from lysis.</P><P style="margin:0in;font-family:Comic Sans MS;font-size:10.0pt"> </P><P style="margin:0in;font-family:Comic Sans MS;font-size:10.0pt">This means would have
		to:</P><P style="margin:0in;font-family:Comic Sans MS;font-size:10.0pt"> </P><OL type="a" style="margin-left: 0.375in; direction: ltr; unicode-bidi: embed; margin-top: 0in; margin-bottom: 0in; font-family: Comic Sans MS; font-size: 10pt; font-style: normal; "><LI value="1" style="margin-top:0;margin-bottom:0;vertical-align:middle"><SPAN style="font-style: normal; font-family: Comic Sans MS; font-size: 10pt; ">Provide large enzyme exposure
		     area within a small volume.</SPAN></LI><LI style="margin-top:0;margin-bottom:0;vertical-align:middle"><SPAN style="font-family:Comic Sans MS;font-size:10.0pt">Allow for scalability for
		     industrial application.</SPAN></LI><LI style="margin-top:0;margin-bottom:0;vertical-align:middle"><SPAN style="font-family:Comic Sans MS;font-size:10.0pt">Be able to accommodate a system
		     of separate enzymes as components of a chosen biosynthetic pathway.</SPAN></LI><LI style="margin-top:0;margin-bottom:0;vertical-align:middle"><SPAN style="font-family:Comic Sans MSfont-size:10.0pt">Be relatively simple and
		     inexpensive</SPAN></LI></OL><P style="margin:0in;font-family:Comic Sans MS;font-size:10.0pt"> </P><P style="margin:0in;font-family:Comic Sans MS;font-size:10.0pt">After a series of
		brainstorming and development sessions during the spring of 2011, the team</P><P style="margin:0in;font-family:Comic Sans MS;font-size:10.0pt">decided to move
		forward with the creation of a microfluidic device.</P><P style="margin:0in;font-family:Comic Sans MS;font-size:10.0pt"> </P><P style="margin:0in;font-family:Comic Sans MS;font-size:10.0pt"><SPAN style="font-weight:bold">Final Chip Design (7/3/2011 - 7/6/2011)</SPAN></P><P style="margin:0in;margin-left:.375in;font-family:Comic Sans MS;font-size:10.0pt">Summary:</P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt">Optimized
		chip size to fit 4 chips on a 4'' diameter wafer. </P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt">After
		discussion, changed channel dimensions back to narrow safe limits of printing
		machine: 200 microns wide, 200 micron spacing. </P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt">Channel
		margins changed<SPAN> </SPAN>to limit of safe margin
		for wafer size.(1cm from edge)</P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt">Inlet/outlet
		positioning was changed to prevent channel damage when punching inlet/outlet
		holes.</P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt">Final
		Design was approved for printing. </P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt">Solidworks
		Sketch converted to DXF in Solidworks. Edited in Modo. Converted to GDS using
		LinkCAD and printed to mask.</P><P style="margin:0in;margin-left:.375in;font-family:Comic Sans MS;font-size:10.0pt">Basis
		for Revision: </P><OL type="a" style="margin-left: 0.75in; direction: ltr; unicode-bidi: embed; margin-top: 0in; margin-bottom: 0in; font-family: Comic Sans MS; font-size: 10pt; font-style: normal; "><LI value="1" style="margin-top:0;margin-bottom:0;vertical-align:middle"><SPAN style="font-style: normal; font-family: Comic Sans MS; font-size: 10pt; ">Channels dimensions needed to
		     balance space-savings and safe-print.</SPAN></LI><LI style="margin-top:0;margin-bottom:0;vertical-align:middle"><SPAN style="font-family:Comic Sans MS;font-size:10.0pt">Team wanted to print as many
		     chips in as little space as possible.</SPAN></LI><LI style="margin-top:0;margin-bottom:0;vertical-align:middle"><SPAN style="font-family:Comic Sans MS;font-size:10.0pt">Team aimed to minimize damage
		     when punching inlet/outlet holes.</SPAN></LI></OL><P style="margin:0in;margin-left:.375in;font-family:Comic Sans MS;font-size:10.0pt">Features:</P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt">Material:
		</P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt;
		color:black"><SPAN style="background:white">We chose to create our microfluidic
		chip out of Polydimethylsiloxane</SPAN></P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt;
		color:black"><SPAN style="background:white">(PDMS) because of its extensive use
		in biologically related</SPAN></P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt;
		color:black"><SPAN style="background:white">microfluidic research. PDMS is an
		optically clear, nontoxic,</SPAN></P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt;
		color:black"><SPAN style="background:white">nonflammable and inert material
		that is ideal for working with</SPAN></P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt;
		color:black"><SPAN style="background:white">biological material. It is also a
		cost efficient material that can</SPAN></P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt"><SPAN style="color:black;background:white">easily be fabricated into a microfluidic
		device.</SPAN></P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt"> </P><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt">Inlet/Outlet</P><OL type="a" style="margin-left: 1.125in; direction: ltr; unicode-bidi: embed; margin-top: 0in; margin-bottom: 0in; font-family: Comic Sans MS; font-size: 10pt; font-style: normal; "><LI value="1" style="margin-top:0;margin-bottom:0;vertical-align:middle"><SPAN style="font-style: normal; font-family: Comic Sans MS; font-size: 10pt; ">Changed inlet/outlet position
		     to reduce wasted space and reduce loss of chips</SPAN></LI></OL><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt">Channel</P><P style="margin:0in;margin-left:1.125in;font-family:Comic Sans MS;font-size:10.0pt">Dimensions</P><OL type="a" style="margin-left: 1.5in; direction: ltr; unicode-bidi: embed; margin-top: 0in; margin-bottom: 0in; font-family: Comic Sans MS; font-size: 10pt; font-style: normal; "><LI value="1" style="margin-top:0;margin-bottom:0;vertical-align:middle"><SPAN style="font-style: normal; font-family: Comic Sans MS; font-size: 10pt; ">Channel width changed to 200
		     microns.</SPAN></LI><LI style="margin-top:0;margin-bottom:0;vertical-align:middle"><SPAN style="font-family:Comic Sans MS;font-size:10.0pt">Channel spacing changed to 200
		     microns.</SPAN></LI><LI style="margin-top:0;margin-bottom:0;vertical-align:middle"><SPAN style="font-family:Comic Sans MS;font-size:10.0pt">Channel depth changed to 100
		     microns.</SPAN></LI></OL><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt">Overall
		Dimensions</P><OL type="a" style="margin-left: 1.125in; direction: ltr; unicode-bidi: embed; margin-top: 0in; margin-bottom: 0in; font-family: Comic Sans MS; font-size: 10pt; font-style: normal; "><LI value="1" style="margin-top:0;margin-bottom:0;vertical-align:middle"><SPAN style="font-style: normal; font-family: Comic Sans MS; font-size: 10pt; ">Reduced chip print dimensions
		     to 22.05mm by 25.00mm. Fits 4 safely on a 4'' diameter wafer. (101.6 mm
		     diameter)</SPAN></LI></OL><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt">Mask</P><OL type="a" style="margin-left: 1.125in; direction: ltr; unicode-bidi: embed; margin-top: 0in; margin-bottom: 0in; font-family: Comic Sans MS; font-size: 10pt; font-style: normal; "><LI value="1" style="margin-top:0;margin-bottom:0;vertical-align:middle"><SPAN style="font-style: normal; font-family: Comic Sans MS; font-size: 10pt; ">Developed. Specifications
		     drawing in Images below.</SPAN></LI></OL><P style="margin:0in;margin-left:.75in;font-family:Comic Sans MS;font-size:10.0pt">DXF
		to GDS File:</P><P style="margin:0in;margin-left:.375in;font-family:&quot;Comic Sans MS&quot;;font-size:
		10.0pt"> </P><TABLE style="margin-left:auto;	margin-right:auto;width:93%;padding-top:10px;"><TBODY><TR><TD><DIV><H1><SPAN class="mw-headline" id="Light-Induced_Lysis">Light-Induced Lysis</SPAN></H1><P>
We would like to develop a genetic switch that is sensitive to specific wavelengths of visible light and use this gene expression system to lyse bacterial cells solely with light. The genetic light sensor is based on Chris Voigt’s &quot;Multichromatic Control of Gene Expression in Escherichia coli,&quot; <SUP>4</SUP> which uses visible green light at 532 nm to induce specific gene expression. The system is composed of a light-activated surface protein, which autophosphorylates an intermediate chromophore and a reporter protein that binds to a specific promoter and is activated by the chromophore.
</P><P>The genes to be expressed downstream of the promoter make up a lysis cassette derived from the lambda phage and developed by Prof. Young at Texas A&amp;M University. This lysis system is very useful because the incubation period after gene expression is on the order of 50 minutes, and the actual lysis occurs within one minute thereafter. Our hope is to lyse bacterial cultures within a known timeframe and with specificity to green light.

</P><H1><SPAN class="mw-headline" id="DNA_Assembly_Methods">DNA Assembly Methods</SPAN></H1><H2><SPAN class="mw-headline" id="Site-Directed_Mutagenesis">Site-Directed Mutagenesis</SPAN></H2><P>The Cornell iGEM team initially encountered problems expressing RFP and the VioA, B, and E enzymes. A possible reason for this issue may be excessive nucleotide spacing in between the ribosome binding site and the start codon in each of the protein-coding DNA sequences. More specifically, our constructs had thirty base pairs of separation, consisting of mainly the iGEM prefix and other restriction sites that we introduced for insertion into our pZE12 vector backbone. In contrast, greater gene expression may require shorter separation of eight to twelve base pairs.
</P><P>QuikChange II site-directed mutagenesis is a method that can delete stretches of adjacent nucleotides. The team used this PCR-mediated technique to omit the iGEM prefix and the restriction sites necessary for insertion into vector solely to achieve protein expression and characterize the microfluidics chips.
</P><P>In general, QuikChange II site-directed mutagenesis requires a vector of double-stranded DNA that contains an insert of interest. Two oligonucleotide primers are designed to have the desired mutation, which in this case is a deletion. The forward primer can be thought of as two internal parts. The first half is made of nucleotide sequence from the plasmid that lies to the left of the deletion site, while the second half reflects the vector sequence that is to the right of the deletion site. The reverse primer is simply the reverse complement of the forward primer. The purpose of the primers' divided mirroring of the vector nucleotide sequence is to make the deletion site loop out by means of matching base pairs flanking that site. Lastly, primer constructs should meet certain guidelines.
</P><P>A high-fidelity DNA polymerase, such as <I>PfuUltra</I>, is necessary because it will be synthesizing the entire length of the vector backbone, while omitting the staggered nicks of the deletion site. Thus, the goal is to minimize the occurrences of a mutation during extension of the primers.
</P><P>A complete PCR product of the vector, remade without the deletion site, does not contain methylated DNA. In contrast, parental forms of the vector and partially mutated backbones will exhibit the dam methylation that is common of DNA isolated from <I>E. coli</I>. Because DpnI recognizes the sequence (5' – GmATC – 3'), a digest with this restriction enzyme will select for only the deletion mutant vectors.
</P><H2><SPAN class="mw-headline" id="Annealed_Primer_Insertion">Annealed Primer Insertion</SPAN></H2><P>While there are multiple ways of preparing a gene for insertion into a vector, annealed oligonucleotides are especially convenient for introducing fragments such as promoters or polylinkers. Primer dimers bypass the need for digestion and PCR amplification of the insert. The specific construction of the primers makes for a simple insertion technique.
</P><P>First, the two complementary oligos are staggered in design so that there are 5' and 3' overhangs representing the restriction site sequences of the digested vector backbone. Second, the primers must be phosphorylated using T4 polynucleotide kinase in order to be ligated into a CIAP-treated backbone.
</P><P>The Cornell iGEM team used two sets of annealed primers to introduce the AviTag sequence, a stop codon, and the iGEM suffix into the pZE12 vector backbone. The first set of primers consists of a  top single-stranded primer with the SphI restriction site as a 5' overhang and about half of the AviTag DNA sequence. The bottom half in this first set is the complementary strand. The second set of primers consists of a top single-stranded primer with the rest of the AviTag, a stop codon, the iGEM suffix, and the ClaI restriction site as a 3' overhang. For each primer set, the top and bottom primers are annealed to form the gene insert.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><DL><DD>1. Gleghorn, J. P. et al. (2010). Capture of circulating tumor cells from whole blood of prostate cancer patients using geometrically enhanced differential immunocapture (GEDI) and a prostate-specific antibody. Lab Chip, 10, 27-29. doi:10.1039/B917959C  
</DD><DD>2. Avidin and streptavidin conjugates. (n.d.). Retrieved from <A href="http://www.invitrogen.com/site/us/en/home/References/Molecular-Probes-The-Handbook/Antibodies-Avidins-Lectins-and-Related-Products/Avidin-Streptavidin-NeutrAvidin-and-CaptAvidin-Biotin-Binding-Proteins-and-Affinity-Matrices.html" class="external free" rel="nofollow">http://www.invitrogen.com/site/us/en/home/References/Molecular-Probes-The-Handbook/Antibodies-Avidins-Lectins-and-Related-Products/Avidin-Streptavidin-NeutrAvidin-and-CaptAvidin-Biotin-Binding-Proteins-and-Affinity-Matrices.html</A></DD><DD>3. Balibar, C. J. and Walsh, C. T. (2006). In Vitro Biosynthesis of Violacein from L-Tryptophan by the Enzymes VioA−E from Chromobacterium Violaceum. Biochemistry, 45(51), 15444-15457. doi:10.1021/bi061998z
</DD><DD>4. Tabor, J. J., Levskaya, A., &amp; Voigt, C. A. (2010). Multichromatic Control of Gene Expression in Escherichia coli. Journal of Molecular Biology, 405(2),315-324. doi:10.1016/j.jmb.2010.10.038</DD></DL></DIV></TD></TR></TBODY></TABLE></DIV></DIV></FONT><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Cornell/Description">http://2011.igem.org/Team:Cornell/Description</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cornell/Description" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cornell/Description" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cornell/Description&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cornell/Description&amp;oldid=262817" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>