<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UESTC-China_day-note skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UESTC-China/day-note</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:UESTC-China/Daynote - 2017.igem.org</TITLE></P><DIV class="head-v3"><DIV class="nav-up"><DIV class="nav-inner"><DIV class="nav-v3"><UL><LI class="" _t_nav="Achievement"><H2><A href="https://2017.igem.org/Team:UESTC-China/achievement">ACHIEVEMENT</A></H2></LI><LI class="" _t_nav="Project"><H2><A href="javascript:void(0);">PROJECT ∨</A></H2></LI><LI class="" _t_nav="Modeling"><H2><A href="https://2017.igem.org/Team:UESTC-China/Model">MODELING</A></H2></LI><LI class="" _t_nav="Attribution"><H2><A href="https://2017.igem.org/Team:UESTC-China/Attributions">ATTRIBUTIONS</A></H2></LI><LI class="" _t_nav="HumanPractice"><H2><A href="javascript:void(0);">HUMAN PRACTICE ∨</A></H2></LI><LI class="" _t_nav="Team"><H2><A href="javascript:void(0);">TEAM ∨</A></H2></LI><LI class="" _t_nav="Notebook"><H2><A href="javascript:void(0);">NOTEBOOK ∨</A></H2></LI></UL></DIV></DIV></DIV><DIV class="nav-down"><DIV id="Project" class="nav-down-menu" style="display: none;" _t_nav="Project"><DIV class="nav-down-inner"><DL style="margin-right: 670px;"><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/part">Part</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/Demonstrate">Demonstrate</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/design">Design</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/description">Introduction</A></DD></DL></DIV></DIV><DIV id="HumanPractice" class="nav-down-menu" style="display: none;" _t_nav="HumanPractice"><DIV class="nav-down-inner"><DL style="margin-right: 250px;"><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/Engagement">Engagement</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/research">Supporting Research</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/our-story">Our Story</A></DD></DL></DIV></DIV><DIV id="Team" class="nav-down-menu" style="display: none;" _t_nav="Team"><DIV class="nav-down-inner"><DL style="margin-right: 115px;"><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/Collaborations">Collaborations</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/team-introduce">Team Introduce</A></DD></DL></DIV></DIV><DIV id="Notebook" class="nav-down-menu" style="display: none;" _t_nav="Notebook"><DIV class="nav-down-inner"><DL style="margin-right: 36px;"><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/InterLab">Interlab</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/safety">Safety</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/protocal">Protocal</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/Plant">Plant</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/day-note">Day Note</A></DD></DL></DIV></DIV></DIV></DIV><DIV class="container" id="new-style"><DIV class="row"><NAV class="col-lg-2" id="myScrollspy"><UL class="nav nav-pills nav-stacked" data-spy="affix" data-offset-top="474"><LI class="active"><A href="#In-April">In April</A></LI><LI><A href="#In-May">In May</A></LI><LI><A href="#In-June">In June</A></LI><LI><A href="#In-July">In July</A></LI><LI><A href="#In-August">In August</A></LI><LI><A href="#In-September">In September</A></LI><LI><A href="#In-October">In October</A></LI><LI><A href="#In-November">In November</A></LI></UL></NAV><DIV class="col-lg-10"><DIV id="In-April"><H2>In April</H2><DIV class="content-out"><DIV class="content-in"><DIV class="content-item"><P>The school competition of iGEM in University of Electronic Science and Technology of China was held. After series of individual races and team competitions, excellent people were finally chosen into iGEM team of 2017: UESTC-China. Then we started to search for some useful and interesting projects. After four sessions of the group discussion and papers reading, we decided to choose TCP degradation by plants as our project. Then we started to carry out primary design of the project and then synthesized some key gene.</P></DIV></DIV></DIV></DIV><DIV id="In-May"><H2>In May</H2><P> Follow the primary design, we mainly did Molecular cloning experiments this month. </P><DIV class="content-out"><DIV class="content-in"><DIV class="content-item"><H3>5.1-5.7</H3><P>We had constructed some elements for the final single gene plasmid, which mainly includegd pGE1.1-DhaA31, pGE1.2-HheC, pGE1.3-EchA and pSE01-Nptll. Some parameters were as follow.</P><P>In addition, we had sowed tobacco seeds on the Medium for several times.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>pGE1.1-DhaA31</TD><TD>M13-F</TD><TD>\</TD><TD>1040bp</TD></TR><TR><TD>pGE1.2-HheC</TD><TD>M13-F</TD><TD>M13-R</TD><TD>923bp</TD></TR><TR><TD>pGE1.3-EchA</TD><TD>M13-F</TD><TD>M13-R</TD><TD>1043bp</TD></TR><TR><TD>pSE01-Nptll</TD><TD>M13-F</TD><TD>M13-R</TD><TD>1033bp</TD></TR></TBODY></TABLE><DIV class="pic-text" style="width: 80%;"><P>pGE1.1-DhaA31,pGE1.2-Hhec Colony PCR</P></DIV><DIV class="pic-text" style="width: 40%;"><P>pGE1.3-EchA Colony PCR</P></DIV></DIV><DIV class="content-item"><H3>5.8-5.14</H3><P>This week, three single gene vectors were constructed successfully by forward elements. In addition to that, we also completed other elements construction due to the need of Multi-gene vector. Some parameters are as follow. At the end of the week, we transferred tobacco seedlings from previous seeds sowing to other mediums for amplification and follow-up infection.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>PiGEM2017-001</TD><TD>TC013-R</TD><TD>ZY065-RB</TD><TD>953bp</TD></TR><TR><TD>PiGEM2017-002</TD><TD>TC013-R</TD><TD>ZY065-RB</TD><TD>953bp</TD></TR><TR><TD>PiGEM2017-003</TD><TD>TC013-R</TD><TD>ZY065-RB</TD><TD>953bp</TD></TR><TR><TD>pGE02-HheC</TD><TD>M13-F</TD><TD>M13-R</TD><TD>991bp</TD></TR><TR><TD>pGE03-EchA</TD><TD>M13-F</TD><TD>M13-R</TD><TD>1108bp</TD></TR><TR><TD>pSE03-NptII</TD><TD>M13-F</TD><TD>M13-R</TD><TD>1033bp</TD></TR></TBODY></TABLE><DIV class="pic-text" style="width: 90%;"><P>PiGEM2017-001, PiGEM2017-002, PiGEM2017-003 Colony PCR</P></DIV><DIV class="pic-text" style="width: 45%;"><P>pGE02-HheC colony PCR</P></DIV><DIV class="pic-text" style="width: 45%;"><P>pGE03-EchA colony PCR</P></DIV><DIV class="pic-text" style="width: 45%;"><P>pSE03-NptII colony PCR</P></DIV></DIV><DIV class="content-item"><H3>5.15-5.21</H3><P>It’s suspected that there may be stable expression of epoxide hydrolases in the plant. To verify this suppose, this week, we designed and constructed a two genes vector system piGEM2016-005 excluding EchA. What’s more, we carried out Agrobacterium transformation using prior plasmids PiGEM2017-001, PiGEM2017-002, PiGEM2017-003. Also, Agrobacterium’s positive defection is done.</P><DIV class="pic-text" style="width: 50%;"><P>pSE02-Nptll colony PCR</P></DIV><DIV class="pic-text" style="width: 30%;"><P>piGEM2017-005 colony PCR</P></DIV></DIV><DIV class="content-item"><H3>5.22-5.31</H3><P>This week, based on prior completed plasmid piGEM2017-005, Agrobacterium transformation was carried out. What’s more, the first Agrobacterium infecting tobacco carrying PiGEM2017-001, PiGEM2017-002, PiGEM2017-003 was carried out.</P></DIV></DIV></DIV></DIV><DIV id="In-June"><H2>In June</H2><DIV class="content-out"><DIV class="content-in"><DIV class="content-item"><H3>6.1-6.10</H3><P>The basic plasmids have been completed. This week we did Agrobacterium infection of tobacco including piGEM2017-001, piGEM2017-002, piGEM2017-003, piGEM2017-004, piGEM2017-005. Besides that, we started to try to optimize our plasmids design.</P></DIV><DIV class="content-item"><H3>6.11-6.20</H3><P>By papers reading, rough plasmids design had been made sure. And the target sequence was sent to company for synthesizing. Furthermore, Agrobacterium infection of tobacco had been done.</P></DIV><DIV class="content-item"><H3>6.21-6.30</H3><P> The leaves infected by Agrobacterium had appeared small bud. We transferred the bud to rooting medium.</P><P>Except, we did the standard curve of chloride ions</P></DIV></DIV></DIV></DIV><DIV id="In-July"><H2>In July</H2><DIV class="content-out"><DIV class="content-in"><DIV class="content-item"><H3>7.6</H3><OL><LI>Restriction Enzyme Digest for PYK10 fragment. There is a restriction site of PstI on PYK10, so we can get two part of it.
										</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>pPE-PYK10</TD><TD>KpnⅠ</TD><TD>PstⅠ</TD><TD>iGEM2017-SG05</TD><TD>998bp、478bp</TD></TR></TBODY></TABLE><LI>Gel Electrophoresis.
										</LI><LI>Extraction of DNA from Agarose Gel.</LI><LI>Restriction Enzyme Digest.
										</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>templation</TH></TR></THEAD><TBODY><TR><TD>Backbone of pZHY998</TD><TD>KpnⅠ</TD><TD>PstⅠ</TD><TD>pZHY998</TD></TR></TBODY></TABLE><LI>Use AXYGEN enzymatic reaction kit to do DNA cleanup.</LI><LI>Mix the digested fragment with backbone of pZHY998, then do ligation with T4 ligation enzyme to make pPE-PYK10.</LI><LI>Transform the ligation product into E.coli DH5α, Amp-plate.</LI></OL></DIV><DIV class="content-item"><H3>7.7</H3><OL><LI>Positive detection of plate pPE-PYK10. After Gel Electrophoresis, there is only one obvious target band we expect, 1609bp.
											</LI><LI>Shake bacteria fluid transformed with pPE-PYK10 on the shaking table 37℃, 180r, 18h.</LI></OL></DIV><DIV class="content-item"><H3>7.8</H3><OL><LI>Conservation for the bacteria fluid of pPE-PYK10, then use AXYGEN kit to do extraction. But the concentration is low.</LI><LI>Restriction Enzyme Digest for verification .
										</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>pPE-PYK10</TD><TD>KpnⅠ</TD><TD>PstⅠ</TD><TD>2716bp、998bp、478bp</TD></TR></TBODY></TABLE><LI>Gel Electrophoresis, and get the target band we expect.
										</LI><LI>Because the extracted pPE-PYK10 plasmid concentration is too low, shake preservative bacteria of pPE-PYK10 on the shaking table 37℃, 180r, 18h again.</LI></OL></DIV><DIV class="content-item"><H3>7.9</H3><OL><LI>Conservation for the bacteria fluid of pPE-PYK10, then use AXYGEN kit to do extraction. (successfully sequenced)</LI><LI>Golden Gate Cloning for piGEM2017-010.</LI><LI>Using Agrobacterium tumefaciens EHA105 transformed vector（PiGEM001、002、003、004、005、006）into Tobacco using the leaf-dipping method and then put the leaves into MSB-Co medium (co-cultivate)</LI></OL></DIV><DIV class="content-item"><H3>7.10</H3><OL><LI>Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</LI><LI>Extract the genome of Rice. But the OD600 is just 9.9.</LI></OL></DIV><DIV class="content-item"><H3>7.11</H3><OL><LI>1 ) Positive detection of plate piGEM2017-010. After Gel Electrophoresis, there showed clear band corresponding to the expected sequence size, 953bp.</LI><P>2 ) fasfsafasfsdafds</P><LI>1 ) Polymerase Chain Reaction.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>pSE04-NptII</TD><TD>iGEMLBL033-F</TD><TD>iGEMLBL022-R</TD><TD>pSE03-NptII</TD><TD>923bp</TD></TR><TR><TD>pGE01-eYFP</TD><TD>iGEMXDM003-F</TD><TD>iGEMXDM003-R</TD><TD>piGEM2017-006</TD><TD>815bp</TD></TR><TR><TD>pGE01AE-CKX3</TD><TD>iGEMLBL027-F</TD><TD>iGEMLBL027-R</TD><TD>pGE01A-CKX3</TD><TD>1728bp</TD></TR><TR><TD>pGE02AE-CKX3</TD><TD>iGEMXDM004-F</TD><TD>iGEMXDM004-R</TD><TD>iGEM2017-SG04</TD><TD>1737bp</TD></TR><TR><TD>pGE02-CKX3</TD><TD>iGEMLBL029-F</TD><TD>iGEMLBL029-R</TD><TD>pGE01A-CKX3</TD><TD>1625bp</TD></TR><TR><TD>pGE04-CKX3</TD><TD>iGEMLBL032-F</TD><TD>iGEMLBL032-R</TD><TD>pGE01A-CKX3</TD><TD>1623bp</TD></TR><TR><TD>pGE01-CKX3</TD><TD>iGEMXDM001-F</TD><TD>iGEMXDM001-R</TD><TD>iGEM2017-SG04</TD><TD>1609bp</TD></TR><TR><TD>piGEM2017-011</TD><TD>iGEMXDM002-F</TD><TD>iGEMXDM002-R</TD><TD>iGEM2017-SG05</TD><TD>1488bp</TD></TR><TR><TD>pGE03-CKX3</TD><TD>iGEMLBL030-F</TD><TD>iGEMLBL030-R</TD><TD>pGE01A-CKX3</TD><TD>1618bp</TD></TR></TBODY></TABLE><P>2 ) Gel Electrophoresis of PCR-Amplified product，and get the target band we expect.</P><P>3 ) Extraction of DNA from Agarose Gel.</P><LI>Using Agrobacterium tumefaciens EHA105 transformed vector（PiGEM001、002、003、004、005、006）into Tobacco using the leaf-dipping method and then put the leaves into MSB-Co medium (co-cultivate). The materials that were co-cultivate（7.9）were placed on the Agrobacterium-eliminating MSB-S medium</LI></OL></DIV><DIV class="content-item"><H3>7.12</H3><OL><LI>1 ) Restriction Enzyme Digest.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pSE04-NptⅡ-cut</TD><TD>KpnⅠ</TD><TD>PstⅠ</TD><TD>pSE04-NptⅡ</TD></TR><TR><TD>pGE01-eYFP-cut</TD><TD>KpnⅠ</TD><TD>PstⅠ</TD><TD>pGE01-eYFP</TD></TR><TR><TD>pGE01AE-CKX3-cut</TD><TD>KpnⅠ</TD><TD>PstⅠ</TD><TD>pGE01AE-CKX3</TD></TR><TR><TD>pGE01-CKX3-cut</TD><TD>KpnⅠ</TD><TD>PstⅠ</TD><TD>pGE01-CKX3</TD></TR><TR><TD>Backbone of pZHY998</TD><TD>KpnⅠ</TD><TD>PstⅠ</TD><TD>pZHY998</TD></TR></TBODY></TABLE><P>2 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P><P>3 ) Mix the four digested fragment with backbone of pZHY998, then do ligation with T4 ligation enzyme.</P><P>4 ) Transform the ligation product into E.coli DH5α, Amp-plate.</P><LI>1 ) Conservation for the bacteria fluid of piGEM2017-010 that was shook yesterday, then use AXYGEN kit to do extraction. (successfully sequenced)</LI><P>2 ) Restriction Enzyme Digest for verification .</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-010</TD><TD>ScaⅠ</TD><TD>6258bp、3386bp、407bp</TD></TR></TBODY></TABLE><P>3 ) Gel Electrophoresis, and get the target band we expect.</P></OL></DIV><DIV class="content-item"><H3>7.13</H3><OL><LI>1 ) Positive detection of plate pSE04-NptⅡ（1033bp）、pGE01-eYFP（935bp）、pGE01AE-CKX3（1730bp）、pGE01-CKX3（1844bp）. After Gel Electrophoresis, only pSE04-NptⅡand pGE01AE-CKX3 have target bands we expect.</LI><P>2 ) Shake bacteria fluid transformed with pSE04-NptⅡ and that with pGE01AE-CKX3 on the shaking table 37℃, 180r, 18h.</P><LI> He materials that were co-cultivate（7.11）were placed on the Agrobacterium-eliminating MSB-S medium</LI><LI> DhaA31 assay by the method of chloride ions in Petunia. Weight the Petunia leaves of piGEM2017-001-1 0.2237g, piGEM2017-002-2 0.0692g, wild-type 0.1163g, Grind with 200μl Tris-SO4 (200Mm, pH 8.0). Result shows no products were detected.</LI></OL></DIV><DIV class="content-item"><H3>7.14</H3><OL><LI>Conservation for the bacteria fluid of pSE04-NptⅡ&amp; pGE01AE-CKX3-1 &amp; pGE01AE-CKX3-1, then use AXYGEN kit to do extraction. (unsuccessfully sequenced)</LI><LI>1 ) Polymerase Chain Reaction.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>pGE01-CKX3</TD><TD>iGEMXDM001-F</TD><TD>iGEMXDM001-R</TD><TD>iGEM2017-SG04</TD><TD>1609bp</TD></TR><TR><TD>pGE01.1A-DhaA31</TD><TD>iGEMXDM005-F</TD><TD>iGEMXDM005-R</TD><TD>pGE01.1- DhaA31</TD><TD>936bp</TD></TR><TR><TD>pGE01.2A-HheC</TD><TD>iGEMXDM006-F</TD><TD>iGEMXDM006-R</TD><TD>pGE01.2-HheC</TD><TD>826bp</TD></TR><TR><TD>pGE01.3A-EchA</TD><TD>iGEMXDM007-F</TD><TD>iGEMXDM007-R</TD><TD>pGE01.3-EchA</TD><TD>943bp</TD></TR></TBODY></TABLE><P>2 ) Gel Electrophoresis of PCR-Amplified product，and get the target band we expect.</P><P>3 ) Extraction of DNA from Agarose Gel.</P><P>4 ) Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>templation</TH></TR></THEAD><TBODY><TR><TD>CKX3 fragment-cut</TD><TD>KpnⅠ</TD><TD>PstⅠ</TD><TD>CKX3 fragment</TD></TR></TBODY></TABLE><P>5 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P><LI>1 ) Golden Gate Cloning for piGEM2017-010.</LI><P>2 ) Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P></OL></DIV><DIV class="content-item"><H3>7.15</H3><OL><LI>1 ) Mix the CKX3 fragment &amp; the eYFP fragment with backbone of pZHY998, then do ligation with T4 ligation enzyme to make pGE01-CKX3 &amp; pGE01-eYFP.</LI><P>2 ) Transform the ligation product into E.coli DH5α, Amp-plate.</P><LI>1 ) Positive detection of plate piGEM2017-008（1100bp）. After Gel Electrophoresis, get target bands we expect.</LI><P>2 ) Shake bacteria fluid transformed with piGEM2017-008 on the shaking table 37℃, 180r, 18h.</P></OL></DIV><DIV class="content-item"><H3>7.16</H3><OL><LI>1 ) Positive detection of plate pGE01-eYFP（935bp）、pGE01-CKX3-1（1730bp）、pGE01-CKX3-2（1730bp）. After Gel Electrophoresis, get target bands we expect.</LI><P>2 ) Shake bacteria fluid transformed with piGEM2017-008 &amp; pGE01-CKX3 on the shaking table 37℃, 180r, 18h.</P><LI>Conservation for the bacteria fluid of piGEM2017-008 that was shook yesterday, then use AXYGEN kit to do extraction.</LI><LI>Using Agrobacterium tumefaciens EHA105 transformed vector（PiGEM001、002、003、004、005、006）into Tobacco using the leaf-dipping method and then put the leaves into MSB-Co medium (co-cultivate).</LI></OL></DIV><DIV class="content-item"><H3>7.17</H3><OL><LI>1 ) Restriction Enzyme Digest.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pGE02-cut</TD><TD>EcoRⅠ</TD><TD>HindⅢ</TD><TD>pGE02</TD></TR><TR><TD>pGE03-cut</TD><TD>EcoRⅠ</TD><TD>HindⅢ</TD><TD>pGE03</TD></TR><TR><TD>pGE04-cut</TD><TD>EcoRⅠ</TD><TD>HindⅢ</TD><TD>pGE04</TD></TR></TBODY></TABLE><P>2 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P><LI>1 ) Conservation for the bacteria fluid of piGEM2017-008 &amp; pGE01-CKX3-1 &amp; pGE03-CKX3-2 that was shook yesterday, then </LI><P>2 ) Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>templation</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-008-cut</TD><TD>EcoNⅠ</TD><TD>BglⅡ</TD><TD>piGEM2017-008</TD></TR><TR><TD>piGEM2017-008-cut</TD><TD>EcoNⅠ</TD><TD>AvrⅡ</TD><TD>piGEM2017-008</TD></TR></TBODY></TABLE><P>3 ) Gel Electrophoresis, but don’t get the target band we expect (1: 7196bp &amp; 3451bp; 2: 7178bp &amp; 3419bp).</P></OL></DIV><DIV class="content-item"><H3>7.18</H3><OL><LI>1 ) Positive detection of plate pGE01AE-CKX3、pSE04-NPtⅡ. After Gel Electrophoresis, get target bands we expect.</LI><P>2 ) Shake bacteria fluid transformed with pGE01AE-CKX3 on the shaking table 37℃, 180r, 18h.</P><LI>1 ) Golden Gate Cloning for piGEM2017-024.</LI><P>2 ) Transform the ligation product into E.coli DH5α, Kan-plate.</P><LI>Transform the pSE04-NPtⅡ ligation product into E.coli DH5α, Amp-plate.</LI><LI>The materials that were co-cultivate（7.16）were placed on the Agrobacterium-eliminating MSB-S medium</LI><LI>The method of gas chromatography was measured because our column can detect the metabolites of the TCP degradation pathway.</LI></OL></DIV><DIV class="content-item"><H3>7.19</H3><OL><LI>1 ) Positive detection of plate piGEM2017-024、pSE04-NPtⅡ. After Gel Electrophoresis, get target bands we expect.</LI><P>2 ) Shake bacteria fluid transformed with pSE04-NPtⅡ on the shaking table 37℃, 180r, 18h.</P><LI>Conservation for the bacteria fluid of pGE01AE-CKX3 that was shook yesterday, then use AXYGEN kit to do extraction.</LI></OL></DIV><DIV class="content-item"><H3>7.20</H3><OL><LI>1 ) Golden Gate Cloning for piGEM2017-007 &amp; piGEM2017-009.</LI><P>2 ) Transform the ligation product into E.coli DH5α, Kan-plate.</P><LI>Conservation for the bacteria fluid of pSE04-NPtⅡ that was shook yesterday, then use AXYGEN kit to do extraction.</LI><LI>Using Agrobacterium tumefaciens EHA105 transformed vector（PiGEM001、002、003、004、005、006）into Tobacco using the leaf-dipping method and then put the leaves into MSB-Co medium (co-cultivate).</LI><LI>DhaA31 assay by gas chromatography with tobacco. Extract the leaves enzymes of piGEM2017-001 and wild type, add the same amount of rough enzymes into the reaction mixture. No DCP was detected in the end.</LI></OL></DIV><DIV class="content-item"><H3>7.21</H3><OL><LI>1 ) Positive detection of platepSE04-NPtⅡ、piGEM2017-009. After Gel Electrophoresis, get target bands we expect.</LI><P>2 ) Shake bacteria fluid transformed with piGEM2017-009 &amp; pSE04-NPtⅡ on the shaking table 37℃, 180r, 18h.</P><LI>Conservation for the bacteria fluid of AV3 that was shook yesterday, then use AXYGEN kit to do extraction.</LI><LI>Transplant the bud from MSB-S medium to MSB-R medium</LI></OL></DIV><DIV class="content-item"><H3>7.22</H3><OL><LI>Conservation for the bacteria fluid of pSE04-NPtⅡ &amp; piGEM2017-009 &amp; pSB143-chl that was shook yesterday, then use AXYGEN kit to do extraction.</LI><LI>Extract the genome of 4 wide Nicotiana tabacum (NT) and 4 NT transformed piGEM2017-004 and piGEM2017-006, respectively. Carry out PCR of all samples to detect 35S promoter. After Gel Electrophoresis, only samples of piGEM2017-004 appeared brands(842bp).</LI><LI>The materials that were co-cultivate（7.20）were placed on the Agrobacterium-eliminating MSB-S medium</LI></OL></DIV><DIV class="content-item"><H3>7.23</H3><OL><LI>1 ) Restriction Enzyme Digest.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>templation</TH></TR></THEAD><TBODY><TR><TD>Backbone of pGE01AE-CKX3</TD><TD>BamHⅠ</TD><TD>EcoRⅠ</TD><TD>pGE01AE-CKX3</TD></TR></TBODY></TABLE><P>2 ) Gel Electrophoresis of Restriction Enzyme Digest product，and get the target band we expect(2852bp).</P><P>3 ) Extraction of backbone of pGE01AE-CKX3 from Agarose Gel</P><P>4 ) Mix AO-S DhaA31 fragment &amp; AO-S HheC fragment &amp; AO-S EchA fragment with backbone of pGE01AE-CKX3, then do ligation with Gibson enzyme to make pGE01.1A-DhaA31 &amp; pGE01.2A-Hhec &amp; pGE01.3A-EchA.</P><P>5 ) Transform the ligation product into E.coli DH5α, Amp-plate.</P><LI>1 ) Restriction Enzyme Digest.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>piGEM2017-009-cut</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>templation</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-009-cut</TD><TD>ScaⅠ</TD><TD>ScaⅠ</TD><TD>piGEM2017-009</TD></TR></TBODY></TABLE><P>2 ) Gel Electrophoresis, and get the target band we expect.</P><LI>Conservation for the bacteria fluid of AV3 that was shook yesterday, then use AXYGEN kit to do extraction.</LI></OL></DIV><DIV class="content-item"><H3>7.24</H3><OL><LI>1 )Positive detection of plate pGE01.1A-DhaA31 、 pGE01.2A-Hhec 、 pGE01.3A-EchA、piGEM2017-024 、 piGEM2017-007. After Gel Electrophoresis, get target bands we expect.</LI><P>2 ) Shake bacteria fluid transformed with pGE01.1A-DhaA31 &amp; pGE01.2A-Hhec &amp; pGE01.3A-EchA &amp; piGEM2017-007 on the shaking table 37℃, 180r, 18h.</P></OL></DIV><DIV class="content-item"><H3>7.25</H3><OL><LI>1 ) Restriction Enzyme Digest.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pGE02-cut</TD><TD>EcoRⅠ</TD><TD>HindⅢ</TD><TD>pGE02</TD></TR><TR><TD>pGE03-cut</TD><TD>EcoRⅠ</TD><TD>HindⅢ</TD><TD>pGE03</TD></TR><TR><TD>pGE04-cut</TD><TD>EcoRⅠ</TD><TD>HindⅢ</TD><TD>pGE04</TD></TR></TBODY></TABLE><P>2 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P><LI>1 ) Polymerase Chain Reaction.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>pSE04</TD><TD>ZY312-F5</TD><TD>ZY312-R</TD><TD>pZHY979</TD><TD>135bp</TD></TR><TR><TD>pSE04-NPtⅡ</TD><TD>iGEMLBL022-F</TD><TD>iGEMLBL022-R</TD><TD>pZHY980</TD><TD>855bp</TD></TR></TBODY></TABLE><P>2 ) Gel Electrophoresis, and get the target band we expect.</P><P>3 ) Extraction of fragments from Agarose Gel.</P><P>4 ) Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pZHY998-cut</TD><TD>KpnⅠ</TD><TD>PstⅠ</TD><TD>pZHY998</TD></TR></TBODY></TABLE><P>5 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P><P>6 ) Mix F2A fragment with backbone of pZHY998, then do ligation with T4 ligation enzyme to make pSE04.</P><P>7 ) Transform the ligation product into E.coli DH5α, Amp-plate.</P><LI>Conservation for the bacteria fluid of pGE01.1A-DhaA31 &amp; pGE01.2A-Hhec &amp; pGE01.3A-EchA &amp; piGEM2017-007 &amp; piGEM2017-010 that was shook yesterday, then use AXYGEN kit to do extraction.</LI><LI>1 ) Mix AO-S CKX3 fragment &amp; CKX3 fragment with backbone of pGE02 &amp; pGE03 &amp; pGE04, then do ligation with Gibson enzyme to make pGE02AE-CKX3 &amp; pGE02-CKX3 &amp; pGE03-CKX3 &amp; pGE04-CKX3.</LI><P>2 ) Transform the ligation product into E.coli DH5α, Amp-plate or Kan-plate.</P><LI>1 ) Restriction Enzyme Digest.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-007-cut</TD><TD>ScaⅠ</TD><TD>ScaⅠ</TD><TD>piGEM2017-007</TD></TR></TBODY></TABLE><P>2 ) Gel Electrophoresis, and get the target band we expect(6258bp &amp; 3686bp &amp; 595bp).</P><LI>Extract the genome of new wide NT and NT transformed piGEM2017-004，piGEM2017-006 as well as NT transformed piGEM2017-005. After Gel Electrophoresis, wide samples had no brands and transgenic NT had purpose brands marked 842bp.</LI></OL></DIV><DIV class="content-item"><H3>7.26</H3><OL><LI>1 ) Mix the F2A fragment with backbone of pZHY998, then do ligation with T4 ligation enzyme to make pSE04.</LI><P>2 ) Transform the ligation product into E.coli DH5α, Amp-plate.</P><LI>1 ) Positive detection of plate pGE02AE-CKX3 &amp; pGE02-CKX3 &amp; pGE03-CKX3 &amp; pGE04-CKX3 &amp; piGEM2017-024. After Gel Electrophoresis, get target bands of pGE02AE-CKX3 &amp; pGE02-CKX3 &amp; pGE04-CKX3 we expect.</LI><P>2 ) Shake bacteria fluid transformed with pGE02AE-CKX3 &amp; pGE02-CKX3 &amp; pGE04-CKX3 on the shaking table 37℃, 180r, 18h.</P><LI>The plasmids（PiGEM008、009） were introduced into Agrobacterium tumefaciens EHA105 by freeze-thaw method Transformation</LI></OL></DIV><DIV class="content-item"><H3>7.27</H3><OL><LI>1 ) Positive detection of plate pSE04 &amp; pGE01.2A-HheC. After Gel Electrophoresis, get target bands of pGE01.2A-HheC we expect.</LI><P>2 ) Shake bacteria fluid transformed with pGE01.2A-HheC &amp; pGE02AE-CKX3 on the shaking table 37℃, 180r, 18h.</P><LI>1 ) Golden Gate Cloning for piGEM2017-024.</LI><P>2 ) Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P><LI>Conservation for the bacteria fluid of pGE02AE-CKX3 &amp;pGE02-CKX3 &amp;pGE04-CKX3 that was shook yesterday, then use AXYGEN kit to do extraction.</LI></OL></DIV><DIV class="content-item"><H3>7.28</H3><OL><LI>1 ) Positive detection of plate piGEM2017-024 、 pGE01.3A-EchA. After Gel Electrophoresis, get target bands we expect.</LI><P>2 ) Shake bacteria fluid transformed with piGEM2017-024 &amp; pGE01.3A-EchA on the shaking table 37℃, 180r, 18h.</P><LI>Conservation for the bacteria fluid of pGE02AE-CKX3 &amp; pGE01.2A-HheC that was shook yesterday, then use AXYGEN kit to do extraction.</LI><LI>Transplant the bud from MSB-S medium to MSB-R medium</LI></OL></DIV><DIV class="content-item"><H3>7.29</H3><OL><LI>Conservation for the bacteria fluid of piGEM2017-024 &amp; pGE01.3A-EchA that were shook yesterday, then use AXYGEN kit to do extraction.</LI><LI>Extract the genome of new NT transformed piGEM2017-002 to 006 except piGEM2017-003. After Gel Electrophoresis, we get 4 strains of NT-004, 3 strains of NT-005 and 3 strains of NT-006 at last which are all positive.</LI><LI>Using Agrobacterium tumefaciens EHA105 transformed vector（PiGEM001、002、003、004、005、006、008、009）into Tobacco using the leaf-dipping method and then put the leaves into MSB-Co medium (co-cultivate) Transformation of Tobacco. co-cultivate</LI><LI>To explore which time could show the enzymatic activity apparently, piGEM2017-001 was chosen to do experiments. The samples were withdrawn from the reaction mixture every hour and we found that the conversion of TCP to DCP reached maximum in seven hours, so 7h were selected to show the activity of all three enzymes.</LI><P class="mid"> Figure. Time courses of concentration of production 2,3-DCP with piGEM2017-001.</P><P class="mid">Reactions were performed using different lines of piGEM2017-001.</P></OL></DIV><DIV class="content-item"><H3>7.30</H3><OL><LI>1 ) Polymerase Chain Reaction.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>pSE04</TD><TD>ZY312-F5</TD><TD>ZY312-R</TD><TD>pZHY979</TD><TD>135bp</TD></TR></TBODY></TABLE><P>2 ) Gel Electrophoresis, and get the target band we expect.</P><P>3 ) Extraction of fragments from Agarose Gel.</P><P>4 ) Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pZHY998-cut</TD><TD>KpnⅠ</TD><TD>PstⅠ</TD><TD>pZHY998</TD></TR><TR><TD>F2A</TD><TD>KpnⅠ</TD><TD>PstⅠ</TD><TD>pSE04 fragment</TD></TR></TBODY></TABLE><P>5 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P><P>6 ) Mix F2A fragment with backbone of pZHY998, then do ligation with T4 ligation enzyme to make pSE04.</P><P>7 ) Transform the ligation product into E.coli DH5α, Amp-plate.</P><LI>Transplant the bud from MSB-S medium to MSB-R medium</LI></OL></DIV><DIV class="content-item"><H3>7.31</H3><OL><LI>1 ) Polymerase Chain Reaction.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>AOS-HheC</TD><TD>XDM004-F</TD><TD>LBL023-R</TD><TD>pGE01.2A-HheC</TD><TD>921bp</TD></TR></TBODY></TABLE><P>2 ) Gel Electrophoresis, and get the target band we expect.</P><P>3 ) Extraction of fragments from Agarose Gel.</P><P>4 ) Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pGE02-cut</TD><TD>EcoRⅠ</TD><TD>HindⅢ</TD><TD>pGE02</TD></TR></TBODY></TABLE><P>5 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P><P>6 ) Mix AOS-HheC with backbone of pGE02, then do ligation with Gibson ligation enzyme to make pGE02A-HheC.</P><P>7 ) Transform the ligation product into E.coli DH5α, Amp-plate.</P><LI>The materials that were co-cultivate（7.29）were placed on the Agrobacterium-eliminating MSB-S medium</LI></OL></DIV></DIV></DIV></DIV><DIV id="In-August"><H2>In August</H2><DIV class="content-out"><DIV class="content-in"><DIV class="content-item"><H3>8.1</H3><OL><LI>1 ) Golden Gate Cloning for piGEM2017-024.</LI><P>2 ) Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P><LI>1 ) Restriction Enzyme Digest.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pGE04-cut</TD><TD>EcoRⅠ</TD><TD>HindⅢ</TD><TD>pGE04</TD></TR></TBODY></TABLE><P>2 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P><P>3 ) Mix Gus fragment with backbone of pGE04, then do ligation with Gibson ligation enzyme to make pGE04-Gus.</P><P>4 ) Transform the ligation product into E.coli DH5α, Amp-plate.</P><LI>1 ) Positive detection of plate pGE02A-HheC &amp; pSE04. After Gel Electrophoresis, get target bands we expect.</LI><P>2 ) Shake bacteria fluid transformed with pGE02A-HheC &amp; pSE04 on the shaking table 37℃, 180r, 18h.</P></OL></DIV><DIV class="content-item"><H3>8.2</H3><OL><LI>1 ) Positive detection of plate pGE04-Gus、pSE04、piGEM2017-024. After Gel Electrophoresis, get target bands we expect.</LI><P>2 ) Shake bacteria fluid transformed with pGE04-Gus &amp; pSE04 &amp; pGE01.3A-EchA on the shaking table 37℃, 180r, 18h.</P><LI>1 ) Mix F2A fragment with backbone of pZHY998, then do ligation with T4 ligation enzyme to make pSE04.</LI><P>2 ) Transform the ligation product into E.coli DH5α, Amp-plate.</P></OL></DIV><DIV class="content-item"><H3>8.3</H3><OL><LI>Conservation for the bacteria fluid of pGE04-Gus &amp; pSE04 &amp; pGE01.3A-EchA &amp; pGE02A-HheC that was shook yesterday, then use AXYGEN kit to do extraction.</LI><LI>1 ) Positive detection of plate pSE04、piGEM2017-024. After Gel Electrophoresis, get target bands we expect.</LI><P>2 ) Shake bacteria fluid transformed with pGE04-Gus &amp; pSE04 &amp; pGE01.3A-EchA on the shaking table 37℃, 180r, 18h.</P><LI>The plasmids（PiGEM014、015、016） were introduced into Agrobacterium tumefaciens EHA105 by freeze-thaw method Transformation</LI></OL></DIV><DIV class="content-item"><H3>8.4</H3><OL><LI>1 ) Polymerase Chain Reaction.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>AOS-EchA</TD><TD>LBL031-F</TD><TD>LBL025-R</TD><TD>pGE01.3A-EchA</TD><TD>1041bp</TD></TR></TBODY></TABLE><P>2 ) Gel Electrophoresis, and get the target band we expect.</P><P>3 ) Extraction of fragments from Agarose Gel.</P><P>4 ) Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pGE03-cut</TD><TD>EcoRⅠ</TD><TD>HindⅢ</TD></TR></TBODY></TABLE><P>5 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P><P>6 ) Mix AOS-EchA with backbone of pGE03, then do ligation with Gibson ligation enzyme to make pGE03A-EchA.</P><P>7 ) Transform the ligation product into E.coli DH5α, Amp-plate.</P><LI>Transplant the bud from MSB-S medium to MSB-R medium</LI></OL></DIV><DIV class="content-item"><H3>8.5</H3><OL><LI>Positive detection of plate pGE03A-EchA. After Gel Electrophoresis, didn’t get target bands we expect.</LI><LI>1 ) Mix AOS-EchA with backbone of pGE03, then do ligation with Gibson ligation enzyme to make pGE03A-EchA.</LI><P>2 ) Transform the ligation product into E.coli DH5α, Amp-plate.</P><LI>Conservation for the bacteria fluid of pSE04 &amp; pGE01.3A-EchA that was shook, then use AXYGEN kit to do extraction.</LI></OL></DIV><DIV class="content-item"><H3>8.7</H3><OL><LI>1 ) Polymerase Chain Reaction.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>AOS-EchA</TD><TD>LBL031-F</TD><TD>LBL025-R</TD><TD>pGE01.3A-EchA</TD><TD>1041bp</TD></TR></TBODY></TABLE><P>2 ) Gel Electrophoresis, and get the target band we expect.</P><P>3 ) Extraction of fragments from Agarose Gel.</P><P>4 ) Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pZHZ079-cut</TD><TD>BglⅡ</TD><TD>NcoⅠ</TD><TD>pZHZ079</TD></TR></TBODY></TABLE><P>5 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P><P>6 ) Mix AOS-EchA with backbone of pZHZ079, then do ligation with Gibson ligation enzyme to make pGE03A-EchA.</P><P>7 ) Transform the ligation product into E.coli DH5α, Amp-plate.</P></OL></DIV><DIV class="content-item"><H3>8.8</H3><OL><LI>1 ) Polymerase Chain Reaction.</LI><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>Gus fragment</TD><TD>LBL034-F</TD><TD>LBL034-R</TD><TD>pGE04-Gus</TD><TD>2081bp</TD></TR></TBODY></TABLE><P>2 ) Gel Electrophoresis, and get the target band we expect.</P><P>3 ) Extraction of fragments from Agarose Gel.</P><LI>1 ) Positive detection of plate pGE03A-EchA. After Gel Electrophoresis, get target bands we expect.</LI><P>2 ) Shake bacteria fluid transformed with pGE03A-EchA on the shaking table 37℃, 180r, 18h.</P></OL></DIV><DIV class="content-item"><H3>8.9</H3><OL><P>1.1 Golden Gate Cloning for piGEM2017-021&amp; piGEM2017-025.</P><P>1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P><P>2.1 Mix Gus fragment with backbone of pGE04, then do ligation with Gibson ligation enzyme to make pGE04-Gus.</P><P>2.2 Transform the ligation product into E.coli DH5α, Amp-plate.</P></OL></DIV><DIV class="content-item"><H3>8.10</H3><P>1.1 Conservation for the bacteria fluid of piGEM2017-021&amp; piGEM2017-024&amp;pGE03A-EchA that was shook yesterday, then use AXYGEN kit to do extraction.</P><P>2.1 Positive detection of plate piGEM2017-021&amp; piGEM2017-024&amp; piGEM2017-025&amp; pGE04-Gus. After Gel Electrophoresis, get target bands we expect.</P><P>2.2 Shake bacteria fluid transformed with pGE04-Gus on the shaking table 37℃, 180r, 18h.</P></DIV><DIV class="content-item"><H3>8.11</H3><P>1.1 Golden Gate Cloning for piGEM2017-021&amp;piGEM2017-024 &amp;piGEM2017-025.</P><P>1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P><P>2.1 Polymerase Chain Reaction.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>AOS-EchA</TD><TD>LBL031-F</TD><TD>LBL025-R</TD><TD>pGE01.3A-EchA</TD><TD>1041bp</TD></TR></TBODY></TABLE><P>2.2 Gel Electrophoresis, and get the target band we expect.</P><P>2.3 Extraction of fragments from Agarose Gel.</P><P>2.4 Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pZHZ079-cut</TD><TD>BglⅡ</TD><TD>NcoⅠ</TD><TD>pZHZ079</TD></TR></TBODY></TABLE><P>2.5 Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P><P>2.6 Mix AOS-EchA with backbone of pZHZ079, then do ligation with Gibson ligation enzyme to make pGE03A-EchA.</P><P>2.7 Transform the ligation product into E.coli DH5α, Amp-plate.</P><P>3.1 Conservation for the bacteria fluid of pGE04-Gus that was shook yesterday, then use AXYGEN kit to do extraction.</P><P>4.1 Using Agrobacterium tumefaciens EHA105 transformed vector（PiGEM008、009）into Tobacco using the leaf-dipping method and then put the leaves into MSB-Co medium (co-cultivate)</P></DIV><DIV class="content-item"><H3>8.12</H3><P>1.1 Positive detection of plate piGEM2017-021&amp; piGEM2017-024&amp; piGEM2017-025. After Gel Electrophoresis, didn’t get target bands we expect.</P></DIV><DIV class="content-item"><H3>8.13</H3><P>1.1 Positive detection of plate pGE03A-EchA. After Gel Electrophoresis, get target bands we expect.</P><P>1.2 Shake bacteria fluid transformed with pGE03A-EchA on the shaking table 37℃, 180r, 18h.</P><P>2.1 The materials that were co-cultivate（8.11）were placed on the Agrobacterium-eliminating MSB-S medium</P></DIV><DIV class="content-item"><H3>8.14</H3><P>1.1 Golden Gate Cloning for piGEM2017-026&amp;piGEM2017-027.</P><P>1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P><P>2.1 Conservation for the bacteria fluid of pGE03A-EchA that was shook yesterday, then use AXYGEN kit to do extraction.</P></DIV><DIV class="content-item"><H3>8.15</H3><P>1.1 Golden Gate Cloning for piGEM2017-021 to piGEM2017-027.</P><P>1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P><P>2.1 PCR for all positive plants to detect each purpose gene.</P><P>2.2 Gel Electrophoresis</P><P>3.1 DhaA31 assay by gas chromatography with piGEM2017-001-4. The detection method was improved. Grind fresh leaves of equal weight in Liquid nitrogen. The reaction mixture is the supernatant of leaves (6.5mL). The result of GC shows our enzyme worked in the transgene tobacco. That’s cool!</P></DIV><DIV class="content-item"><H3>8.16</H3><P>1.1 Positive detection of plate piGEM2017-026 &amp; piGEM2017-027. After Gel Electrophoresis, get target bands of 027 we expect.</P><P>1.2 Shake bacteria fluid transformed with piGEM2017-027 on the shaking table 37℃, 180r, 18h.</P></DIV><DIV class="content-item"><H3>8.18</H3><P>1.1 Golden Gate Cloning for piGEM2017-021 to piGEM2017-027.</P><P>1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P><P>2.1 Shake bacteria fluid transformed with piGEM2017-027 on the shaking table 37℃, 180r, 18h.</P></DIV><DIV class="content-item"><H3>8.19</H3><P>1.1 Positive detection of plate piGEM2017-021、piGEM2017-022、piGEM2017-024. After Gel Electrophoresis, didn’t get target bands we expect.</P><P>2.1 Conservation for the bacteria fluid of piGEM2017-027 that was shook yesterday, then use AXYGEN kit to do extraction.</P></DIV><DIV class="content-item"><H3>8.20</H3><P>1.1 Golden Gate Cloning for piGEM2017-023 to piGEM2017-027.</P><P>1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P><P>2.1 The plasmids（PiGEM014、015） were introduced into Agrobacterium tumefaciens EHA105 by freeze-thaw method Transformation</P><P>3.1 HheC assay by the same method with piGEM2017-002, but no ECH was detected.</P></DIV><DIV class="content-item"><H3>8.22</H3><P>1.1 Positive detection of plate piGEM2017-023 to piGEM2017-027. After Gel Electrophoresis, get target bands we expect.</P><P>1.2 Shake bacteria fluid transformed with piGEM2017-023 to piGEM2017-027 on the shaking table 37℃, 180r, 18h.</P><P>2.1 Golden Gate Cloning for piGEM2017-021 &amp; piGEM2017-022.</P><P>2.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P><P>3.1 The materials that were co-cultivate（8.20）were placed on the Agrobacterium-eliminating MSB-S medium</P></DIV><DIV class="content-item"><H3>8.23</H3><P>1.1 Conservation for the bacteria fluid of piGEM2017-024 &amp; piGEM2017-026 &amp; piGEM2017-027 that was shook yesterday, then use AXYGEN kit to do extraction.</P><P>1.2 Restriction Enzyme Digest for verification .</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-024</TD><TD>EcoRⅠ</TD><TD>BamHⅠ</TD><TD>1344bp、2683bp、7607bp</TD></TR><TR><TD>piGEM2017-026</TD><TD>SacⅡ</TD><TD>EcoRⅠ</TD><TD>1449bp、3630bp、8950bp</TD></TR><TR><TD>piGEM2017-027</TD><TD>SacⅡ</TD><TD>EcoRⅠ</TD><TD>1449bp、3630bp、8956bp</TD></TR></TBODY></TABLE><P>1.3 Gel Electrophoresis, and get the target band we expect.</P><P>1.4 piGEM2017-026 was successfully sequenced.</P></DIV><DIV class="content-item"><H3>8.24</H3><P>1.1 Positive detection of plate piGEM2017-021 &amp; piGEM2017-022. After Gel Electrophoresis, get target bands of 027 we expect.</P><P>1.2 Shake bacteria fluid transformed with piGEM2017-021 &amp; piGEM2017-022 on the shaking table 37℃, 180r, 18h.</P><P>2.1 Conservation for the bacteria fluid of piGEM2017-025 that was shook, then use AXYGEN kit to do extraction.</P><P>2.2 Restriction Enzyme Digest for verification.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-025</TD><TD>SacⅡ</TD><TD>EcoRⅠ</TD><TD>1344bp、3179bp、8156bp</TD></TR></TBODY></TABLE><P>2.3 Gel Electrophoresis, and didn’t get the target band we expect.</P><P>3.1 Using Agrobacterium tumefaciens EHA105 transformed vector（PiGEM015、016）into Tobacco using the leaf-dipping method and then put the leaves into MSB-Co medium (co-cultivate)</P></DIV><DIV class="content-item"><H3>8.25</H3><P>1.1 Golden Gate Cloning for piGEM2017-022 &amp; piGEM2017-025&amp; piGEM2017-027.</P><P>1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P><P>2.1 Conservation for the bacteria fluid of piGEM2017-021 &amp; piGEM2017-023&amp; piGEM2017-024 &amp; piGEM2017-027 that was shook, then use AXYGEN kit to do extraction.</P><P>2.2 Restriction Enzyme Digest for verification.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-021</TD><TD>SacⅡ</TD><TD>EcoRⅠ</TD><TD>1344bp、3179bp、8752bp</TD></TR><TR><TD>piGEM2017-023</TD><TD>SacⅡ</TD><TD>EcoRⅠ</TD><TD>1446bp、3179bp、8360bp</TD></TR><TR><TD>piGEM2017-024</TD><TD>EcoRⅠ</TD><TD>BamHⅠ</TD><TD>1344bp、2683bp、7607bp</TD></TR><TR><TD>piGEM2017-027</TD><TD>SacⅡ</TD><TD>EcoRⅠ</TD><TD>1449bp、3630bp、8956bp</TD></TR></TBODY></TABLE><P>2.3 Gel Electrophoresis, and get the target band we expect.</P><P>3.1 The plasmids（PiGEM025,026,027） were introduced into Agrobacterium tumefaciens EHA105 by freeze-thaw method Transformation</P></DIV><DIV class="content-item"><H3>8.26</H3><P>1.1 Positive detection of plate piGEM2017-022 &amp; piGEM2017-023 &amp; piGEM2017-025 &amp; piGEM2017-027. After Gel Electrophoresis, get target bands we expect.</P><P>1.2 Shake bacteria fluid transformed with piGEM2017-022 &amp; piGEM2017-023 &amp; piGEM2017-025 &amp; piGEM2017-027 on the shaking table 37℃, 180r, 18h.</P><P>2.1 Conservation for the bacteria fluid of pGE04-CKX3 &amp; pGE04-Gus that were shook, then use AXYGEN kit to do extraction.</P><P>3.1 The plasmids（PiGEM025,026,027） were introduced into Agrobacterium tumefaciens EHA105 by freeze-thaw method Transformation</P></DIV><DIV class="content-item"><H3>8.27</H3><P>1.1 Conservation for the bacteria fluid of piGEM2017-022 &amp; piGEM2017-023 &amp; piGEM2017-025 &amp; piGEM2017-027 that was shook, then use AXYGEN kit to do extraction.</P><P>1.2 Restriction Enzyme Digest for verification.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-022</TD><TD>SacⅡ</TD><TD>EcoRⅠ</TD><TD>1446bp、3179bp、8956bp</TD></TR><TR><TD>piGEM2017-023</TD><TD>SacⅡ</TD><TD>EcoRⅠ</TD><TD>1446bp、3179bp、8360bp</TD></TR><TR><TD>piGEM2017-025</TD><TD>SacⅡ</TD><TD>EcoRⅠ</TD><TD>1344bp、3179bp、8156bp</TD></TR><TR><TD>piGEM2017-027</TD><TD>SacⅡ</TD><TD>EcoRⅠ</TD><TD>1449bp、3630bp、8956bp</TD></TR></TBODY></TABLE><P>1.3 Gel Electrophoresis, and get the target band of 022 and 025 we expect.</P><P>2.1 Golden Gate Cloning for piGEM2017-022 &amp; piGEM2017-023.</P><P>2.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P></DIV><DIV class="content-item"><H3>8.28</H3><P>1.1 Polymerase Chain Reaction</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-P01F</TD><TD>iGEMpart001-F</TD><TD>iGEMpart001-F</TD><TD>pGE01.1A-DhaA31</TD><TD>1030bp</TD></TR><TR><TD>piGEM2017-P02-1</TD><TD>iGEMpart002-F</TD><TD>iGEMpart002-RM</TD><TD>pGE01.2A-HheC</TD><TD>305bp</TD></TR><TR><TD>piGEM2017-P02-2</TD><TD>iGEMpart002-FM</TD><TD>iGEMpart002-R</TD><TD>pGE01.2A-HheC</TD><TD>550bp</TD></TR><TR><TD>piGEM2017-P03-1</TD><TD>iGEMpart003-F</TD><TD>iGEMpart003-RM1</TD><TD>pGE01.3A-EchA</TD><TD>304bp</TD></TR><TR><TD>piGEM2017-P03-2</TD><TD>iGEMpart003-FM1</TD><TD>iGEMpart003-RM2</TD><TD>pGE01.3A-EchA</TD><TD>556bp</TD></TR><TR><TD>piGEM2017-P03-3</TD><TD>iGEMpart003-FM2</TD><TD>iGEMpart003-R</TD><TD>pGE01.3A-EchA</TD><TD>159bp</TD></TR><TR><TD>piGEM2017-P04F</TD><TD>iGEMpart004-F</TD><TD>iGEMpart004-R</TD><TD>pGE04-CKX3</TD><TD>1618bp</TD></TR></TBODY></TABLE><P>1.2 Gel Electrophoresis, and get the target band we expect.</P><P>1.3 Extraction of fragments from Agarose Gel.</P><P>1.4 Fusion PCR</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-P02F</TD><TD>iGEMpart002-F</TD><TD>iGEMpart002-R</TD><TD>piGEM2017-P02-1piGEM2017-P02-2</TD><TD>811bp</TD></TR><TR><TD>piGEM2017-P03F</TD><TD>iGEMpart003-F</TD><TD>iGEMpart003-R</TD><TD>piGEM2017-P03-1piGEM2017-P03-2piGEM2017-P03-3</TD><TD>931bp</TD></TR></TBODY></TABLE><P>1.5 Gel Electrophoresis, and get the target band we expect</P><P>1.6 Extraction of fragments from Agarose Gel.</P><P>1.7 Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pSB1C3-cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>pSB1C3</TD></TR><TR><TD>piGEM2017-P01F-cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>piGEM2017-P01F</TD></TR><TR><TD>piGEM2017-P02F-cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>piGEM2017-P02F</TD></TR><TR><TD>piGEM2017-P03F-cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>piGEM2017-P03F</TD></TR><TR><TD>piGEM2017-P04F-cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>piGEM2017-P04F</TD></TR></TBODY></TABLE><P>1.8 Mix the 4 digested fragment with backbone of pSB1C3, then do ligation with T4 ligation enzyme to make piGEM2017-P01, piGEM2017-P02, piGEM2017-P03 and piGEM2017-P04.</P><P>1.9 Transform the ligation product into E.coli DH5α, Chl-plate.</P></DIV><DIV class="content-item"><H3>8.29</H3><P>1.1 Golden Gate Cloning for piGEM2017-022 &amp; piGEM2017-023.</P><P>1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P><P>2.1 Positive detection of plate piGEM2017-P01, piGEM2017-P02, piGEM2017-P03 and piGEM2017-P04. After Gel Electrophoresis, get target bands we expect.</P><P>2.2 Shake bacteria fluid transformed with piGEM2017-P01, piGEM2017-P02, piGEM2017-P03 and piGEM2017-P04 on the shaking table 37℃, 180r, 18h.</P></DIV><DIV class="content-item"><H3>8.30</H3><P>1.1 HheC assay and no production.</P></DIV><DIV class="content-item"><H3>8.31</H3><P>1.1 Positive detection of plate piGEM2017-022 &amp; piGEM2017-023. After Gel Electrophoresis, get target bands we expect.</P><P>1.2 Shake bacteria fluid transformed with piGEM2017-022 &amp; piGEM2017-023 on the shaking table 37℃, 180r, 18h.</P><P>2.1 Conservation for the bacteria fluid of piGEM2017-P01, piGEM2017-P02, piGEM2017-P03 and piGEM2017-P04 that were shook, then use AXYGEN kit to do extraction. P01, P02 and P04 were successfully sequenced.</P></DIV></DIV></DIV></DIV><DIV id="In-September"><H2>In September</H2><DIV class="content-out"><DIV class="content-in"><DIV class="content-item"><H3>9.2</H3><P>1.1 Golden Gate Cloning for piGEM2017-022 &amp; piGEM2017-023.</P><P>1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P><P>2.1 Conservation for the bacteria fluid of piGEM2017-022 that was shook, then use AXYGEN kit to do extraction. But the restriction enzyme digestion was wrong.</P></DIV><DIV class="content-item"><H3>9.3</H3><P>1.1 Positive detection of plate piGEM2017-022 &amp; piGEM2017-023. After Gel Electrophoresis, get target bands we expect.</P><P>1.2 Shake bacteria fluid transformed with piGEM2017-022 &amp; piGEM2017-023 on the shaking table 37℃, 180r, 18h.</P></DIV><DIV class="content-item"><H3>9.5 </H3><P>1.1 EchA activity assayed by the same method with piGEM2017-003. Result showed EchA could work successfully in transgenic tobacco.</P></DIV><DIV class="content-item"><H3>9.7</H3><P>1.1 Polymerase Chain Reaction.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>AOS-EchA</TD><TD>LBL031-F</TD><TD>LBL025-R</TD><TD>pGE01.3A-EchA</TD><TD>1041bp</TD></TR></TBODY></TABLE><P>1.2 Gel Electrophoresis, and get the target band we expect.</P><P>1.3 Extraction of fragments from Agarose Gel.</P></DIV><DIV class="content-item"><H3>9.8</H3><P>1.1 Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pZHZ079-cut</TD><TD>BglⅡ</TD><TD>NcoⅠ</TD><TD>pZHZ079</TD></TR></TBODY></TABLE><P>1.3 Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P></DIV><DIV class="content-item"><H3>9.9</H3><P>1.1 Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pZHZ079-cut</TD><TD>BglⅡ</TD><TD>NcoⅠ</TD><TD>pZHZ079</TD></TR></TBODY></TABLE><P>1.3 Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P></DIV><DIV class="content-item"><H3>9.9</H3><P>1.1 Mix AOS-EchA with backbone of pZHZ079, then do ligation with Gibson ligation enzyme to make pGE03A-EchA.</P><P>1.2 Transform the ligation product into E.coli DH5α, Amp-plate.</P></DIV><DIV class="content-item"><H3>9.10</H3><P>1.1 Positive detection of plate pGE03A-EchA. After Gel Electrophoresis, get target bands we expect.</P><P>1.2 Shake bacteria fluid transformed with pGE03A-EchA on the shaking table 37℃, 180r, 18h.</P><P>2.1 Mix AOS-EchA with backbone of pZHZ079, then do ligation with Gibson ligation enzyme to make pGE03A-EchA.</P><P>2.2 Transform the ligation product into E.coli DH5α, Amp-plate.</P><P>3.1 Repetition of DhaA31 activity assay with piGEM2017-001-3, piGEM2017-001-4 and piGEM2017-001-10</P></DIV><DIV class="content-item"><H3>9.11</H3><P>1.1 Conservation for the bacteria fluid of pGE03A-EchA that was shook, then use AXYGEN kit to do extraction. (successfully sequenced)</P></DIV><DIV class="content-item"><H3>9.12</H3><P>1.1 Golden Gate Cloning for piGEM2017-022 &amp; piGEM2017-023 &amp; piGEM2017-026 &amp; piGEM2017-027.</P><P>1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P></DIV><DIV class="content-item"><H3>9.13</H3><P>1.1 Positive detection of plate piGEM2017-022 &amp; piGEM2017-027. After Gel Electrophoresis, get target bands we expect.</P><P>1.2 Shake bacteria fluid transformed with piGEM2017-022 &amp; piGEM2017-027 on the shaking table 37℃, 180r, 18h.</P></DIV><DIV class="content-item"><H3>9.14</H3><P>1.1 Positive detection of plate piGEM2017-023 &amp; piGEM2017-026. After Gel Electrophoresis, get target bands we expect.</P><P>1.2 Shake bacteria fluid transformed with piGEM2017-023 &amp; piGEM2017-026 on the shaking table 37℃, 180r, 18h.</P></DIV><DIV class="content-item"><H3>9.15</H3><P>1.1 Conservation for the bacteria fluid of piGEM2017-026 &amp; piGEM2017-027 that was shook, then use AXYGEN kit to do extraction. The concentration of piGEM2017-026 was too low. </P><P>1.2 Restriction Enzyme Digest for verification .</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-027</TD><TD>SalⅠ</TD><TD>EcoRⅠ</TD><TD>1446bp、2500bp、10086bp</TD></TR></TBODY></TABLE><P>1.3 Gel Electrophoresis, and get the target band we expect.</P><P>1.4 piGEM2017-027 was successfully sequenced. </P><P>2.1 Shake bacteria fluid transformed with piGEM2017-026 on the shaking table 37℃, 180r, 18h.</P></DIV><DIV class="content-item"><H3>9.16</H3><P>1.1 Positive detection of plate piGEM2017-022. After Gel Electrophoresis, get target bands we expect.</P><P>1.2 Shake bacteria fluid transformed with piGEM2017-022 on the shaking table 37℃, 180r, 18h.</P><P>2.1 Polymerase Chain Reaction</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-P05-1</TD><TD>iGEMpart005-F</TD><TD>iGEMpart005-RM1</TD><TD>piGEM2017-007</TD><TD>509bp</TD></TR><TR><TD>piGEM2017-P05-2</TD><TD>iGEMpart005-FM1</TD><TD>iGEMpart005-RM2</TD><TD>piGEM2017-007</TD><TD>697bp</TD></TR><TR><TD>piGEM2017-P05-3</TD><TD>iGEMpart005-FM2</TD><TD>iGEMpart005-R</TD><TD>piGEM2017-007</TD><TD>384bp</TD></TR></TBODY></TABLE><P>2.2 Gel Electrophoresis, and get the target band we expect.</P><P>2.3 Extraction of fragments from Agarose Gel.</P><P>3.1 Conservation for the bacteria fluid of piGEM2017-022 &amp; piGEM2017-023 &amp; piGEM2017-026 &amp; piGEM2017-027 that was shook, then use AXYGEN kit to do extraction. The concentration of piGEM2017-023 and piGEM2017-026 was too low.</P><P>3.2 Shake bacteria fluid transformed piGEM2017-023 and piGEM2017-026 on the shaking table 37℃, 180r, 18h.</P></DIV><DIV class="content-item"><H3>9.17</H3><P>1.1 Fusion PCR</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-P03F</TD><TD>iGEMpart005-F</TD><TD>iGEMpart005-R</TD><TD>piGEM2017-P05-1piGEM2017-P05-2piGEM2017-P05-3</TD><TD>1500bp</TD></TR></TBODY></TABLE><P>1.2 Gel Electrophoresis, and get the target band we expect</P><P>1.3 Extraction of fragments from Agarose Gel.</P><P>2.1 Restriction Enzyme Digest for verification .</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-022</TD><TD>SalⅠ</TD><TD>EcoRⅠ</TD><TD>1446bp、2500bp、9635bp</TD></TR><TR><TD>piGEM2017-023</TD><TD>SalⅠ</TD><TD>EcoRⅠ</TD><TD>1446bp、2307bp、9232bp</TD></TR><TR><TD>piGEM2017-026</TD><TD>SalⅠ</TD><TD>EcoRⅠ</TD><TD>1446bp、2307bp、9683bp</TD></TR></TBODY></TABLE><P>2.2 Gel Electrophoresis, and get the target band of 022 and 026 we expect.</P></DIV><DIV class="content-item"><H3>9.18</H3><P>1.1 Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pSB1C3-cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>pSB1C3</TD></TR><TR><TD>piGEM2017-P05F-cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>piGEM2017-P05F</TD></TR></TBODY></TABLE><P>1.2 Mix the digested fragment with backbone of pSB1C3, then do ligation with T4 ligation enzyme to make piGEM2017-P05.</P><P>1.3 Transform the ligation product into E.coli DH5α, Chl-plate.</P><P>2.1 Positive detection of plate piGEM2017-023. After Gel Electrophoresis, get target bands we expect.</P><P>2.2 Shake bacteria fluid transformed with piGEM2017-023 on the shaking table 37℃, 180r, 18h.</P><P>3.1 Conservation for the bacteria fluid of piGEM2017-022 &amp; piGEM2017-023 &amp; piGEM2017-026 that were shook, then use AXYGEN kit to do extraction. </P><P>4.1 Shake bacteria fluid transformed piGEM2017-023 on the shaking table 37℃, 180r, 18h.</P><P>5.1 Repetition of EchA activity assay with piGEM2017-003-1, piGEM2017-003-5 and piGEM2017-003-8.</P></DIV><DIV class="content-item"><H3>9.20</H3><P>1.1 Positive detection of plate piGEM2017-P05. After Gel Electrophoresis, didn’t get target bands we expect.</P></DIV><DIV class="content-item"><H3>9.21</H3><P>1.1 Mix the digested fragment of piGEM2017-P05F-cut with backbone of pSB1C3, then do ligation with T4 ligation enzyme to make piGEM2017-P05.</P><P>1.2 Transform the ligation product into E.coli DH5α, Chl-plate.</P><P>2.1 Conservation for the bacteria fluid of piGEM2017-022 &amp; piGEM2017-023 &amp; piGEM2017-026 that were shook, then use AXYGEN kit to do extraction. But they were mistaken.</P><P>2.2 Shake bacteria fluid transformed piGEM2017-023 on the shaking table 37℃, 180r, 18h.</P><P>3.1 Golden Gate Cloning for piGEM2017-023 &amp; piGEM2017-026.</P><P>3.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.</P></DIV><DIV class="content-item"><H3>9.22</H3><P>1.1 Positive detection of plate piGEM2017-P05. After Gel Electrophoresis, get target bands we expect.</P><P>1.2 Shake bacteria fluid transformed piGEM2017-P05 on the shaking table 37℃, 180r, 18h.</P><P>2.1 Conservation for the bacteria fluid of piGEM2017-023 that was shook, then use AXYGEN kit to do extraction. </P></DIV><DIV class="content-item"><H3>9.24</H3><P>1.1 Positive detection of plate piGEM2017-023 &amp; piGEM2017-026. After Gel Electrophoresis, get target bands we expect.</P><P>1.2 Shake bacteria fluid transformed piGEM2017-023 &amp; piGEM2017-026 on the shaking table 37℃, 180r, 18h.</P><P>2.1 Conservation for the bacteria fluid of piGEM2017-P05 that was shook, then use AXYGEN kit to do extraction. (successfully sequenced)</P></DIV><DIV class="content-item"><H3>9.26</H3><P>1.1 Conservation for the bacteria fluid of piGEM2017-023 &amp; piGEM2017-026 that were shook, then use AXYGEN kit to do extraction. (successfully sequenced)</P><P>1.2 Restriction Enzyme Digest for verification.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-023</TD><TD>BamHⅠ</TD><TD>EcoRⅠ</TD><TD>1446bp、2291bp、9148bp</TD></TR><TR><TD>piGEM2017-026</TD><TD>BamHⅠ</TD><TD>EcoRⅠ</TD><TD>1446bp、2291bp、9699bp</TD></TR></TBODY></TABLE><P>1.3 Gel Electrophoresis, and get the target band we expect.</P></DIV><DIV class="content-item"><H3>9.28</H3><P>1.1 Add the HheC produced by E.coli into the supernatant of tobacco leaves and Tris-SO4 buffer to compare the enzyme activity. Results proved HheC can’t be degrade by protease in tobacco.</P></DIV><DIV class="content-item"><H3>9.30</H3><P>1.1 Polymerase Chain Reaction</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017mutant-001-1</TD><TD>mutant-F</TD><TD>P84A-R</TD><TD>HheC-WT</TD><TD>355bp</TD></TR><TR><TD>piGEM2017mutant-001-2</TD><TD>P84A-F</TD><TD>mutant-R</TD><TD>HheC-WT</TD><TD>584bp</TD></TR><TR><TD>piGEM2017mutant-002-1</TD><TD>F12Q-F</TD><TD>F186L-R</TD><TD>HheC-WT</TD><TD>580bp</TD></TR><TR><TD>piGEM2017mutant-002-2</TD><TD>F186L-F</TD><TD>mutant-R</TD><TD>HheC-WT</TD><TD>282bp</TD></TR><TR><TD>piGEM2017mutant-003-1</TD><TD>mutant-F</TD><TD>P84A-R</TD><TD>HheC-W249P</TD><TD>355bp</TD></TR><TR><TD>piGEM2017mutant-003-2</TD><TD>P84A-F</TD><TD>mutant-R</TD><TD>HheC-W249P</TD><TD>584bp</TD></TR><TR><TD>piGEM2017mutant-004F</TD><TD>F12Q-F</TD><TD>mutant-R</TD><TD>HheC-W249P</TD><TD>825bp</TD></TR><TR><TD>piGEM2017mutant-005-1</TD><TD>mutant-F</TD><TD>F186L-R</TD><TD>HheC-W249P</TD><TD>F186L-R</TD></TR><TR><TD>piGEM2017mutant-005-2</TD><TD>F186L-F</TD><TD>mutant-R</TD><TD>HheC-W249P</TD><TD>282bp</TD></TR><TR><TD>piGEM2017mutant-006-1</TD><TD>F12Q-F</TD><TD>F186L-R</TD><TD>HheC-W249P</TD><TD>580bp</TD></TR><TR><TD>piGEM2017mutant-006-2</TD><TD>F186L-F</TD><TD>mutant-R</TD><TD>HheC-W249P</TD><TD>282bp</TD></TR></TBODY></TABLE><P>1.2 Gel Electrophoresis, and get the target band we expect.</P><P>1.3 Extraction of fragments from Agarose Gel.</P></DIV></DIV></DIV></DIV><DIV id="In-October"><H2>In October</H2><DIV class="content-out"><DIV class="content-in"><DIV class="content-item"><H3>10.1</H3><P>1.1 Fusion PCR</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017mutant-001F</TD><TD>mutant-F</TD><TD>mutant-R</TD><TD>piGEM2017mutant-001-1piGEM2017mutant-001-2</TD><TD>903bp</TD></TR><TR><TD>piGEM2017mutant-002F</TD><TD>F12Q-F</TD><TD>mutant-R</TD><TD>piGEM2017mutant-002-1piGEM2017mutant-002-2</TD><TD>825bp</TD></TR><TR><TD>piGEM2017mutant-003F</TD><TD>mutant-F</TD><TD>mutant-R</TD><TD>piGEM2017mutant-003-1piGEM2017mutant-003-2</TD><TD>903bp</TD></TR><TR><TD>piGEM2017mutant-005F</TD><TD>mutant-F</TD><TD>mutant-R</TD><TD>piGEM2017mutant-005-1piGEM2017mutant-005-2</TD><TD>903bp</TD></TR><TR><TD>piGEM2017mutant-006F</TD><TD>F12Q-F</TD><TD>mutant-R</TD><TD>piGEM2017mutant-006-1piGEM2017mutant-006-2</TD><TD>825bp</TD></TR></TBODY></TABLE><P>1.2 Gel Electrophoresis, and get the target band we expect</P><P>1.3 Extraction of fragments from Agarose Gel.</P><P>1.4 Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>HheC-WT -cut</TD><TD>PstⅠ</TD><TD>BamHⅠ/ NcoⅠ</TD><TD>HheC-WT</TD></TR><TR><TD>piGEM2017mutant-001F-cut</TD><TD>PstⅠ</TD><TD>BamHⅠ</TD><TD>piGEM2017mutant-001F</TD></TR><TR><TD>piGEM2017mutant-002F-cut</TD><TD>PstⅠ</TD><TD>NcoⅠ</TD><TD>piGEM2017mutant-002F</TD></TR><TR><TD>piGEM2017mutant-003F-cut</TD><TD>PstⅠ</TD><TD>BamHⅠ</TD><TD>piGEM2017mutant-003F</TD></TR><TR><TD>piGEM2017mutant-0041F-cut</TD><TD>PstⅠ</TD><TD>NcoⅠ</TD><TD>piGEM2017mutant-004F</TD></TR><TR><TD>piGEM2017mutant-005F-cut</TD><TD>PstⅠ</TD><TD>BamHⅠ</TD><TD>piGEM2017mutant-005F</TD></TR><TR><TD>piGEM2017mutant-006F-cut</TD><TD>PstⅠ</TD><TD>NcoⅠ</TD><TD>piGEM2017mutant-006F</TD></TR></TBODY></TABLE><P>1.5 Use AXYGEN enzymatic reaction kit to do DNA cleanup..</P><P>2.1 Polymerase Chain Reaction</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>S1.0-1</TD><TD>mutant-F</TD><TD>84-R</TD><TD>HheC-WT</TD><TD>355bp</TD></TR><TR><TD>S1.0-2</TD><TD>84-F</TD><TD>mutant-R</TD><TD>HheC-WT</TD><TD>584bp</TD></TR><TR><TD>S2.0-1</TD><TD>12-F</TD><TD>186-R</TD><TD>HheC-WT</TD><TD>580bp</TD></TR><TR><TD>S2.0-2</TD><TD>186-F</TD><TD>mutant-R</TD><TD>HheC-WT</TD><TD>282bp</TD></TR><TR><TD>S3.0-1</TD><TD>mutant-F</TD><TD>186-R</TD><TD>HheC-W249P</TD><TD>658bp</TD></TR><TR><TD>S3.0-2</TD><TD>F186L-F</TD><TD>mutant-R</TD><TD>HheC-W249P</TD><TD>282bp</TD></TR></TBODY></TABLE><P>2.2 Gel Electrophoresis, and get the target band we expect.</P><P>2.3 Extraction of fragments from Agarose Gel.</P></DIV><DIV class="content-item"><H3>10.2</H3><P>1.1 Mix the digested fragment of piGEM2017mutant-001F-cut &amp; piGEM2017mutant-002F-cut &amp; piGEM2017mutant-003F-cut &amp; piGEM2017mutant-004F-cut &amp; piGEM2017mutant-005F-cut &amp; piGEM2017mutant-006F-cut with backbone of HheC-WT -cut, then do ligation with T4 ligation enzyme to make piGEM2017-P05.</P><P>1.2 Transform the ligation product into E.coli DH5α, Amp-plate.</P><P>2.1 Fusion PCR</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>S1.0F</TD><TD>mutant-F</TD><TD>mutant-R</TD><TD>S1.0-1 S1.0-2</TD><TD>903bp</TD></TR><TR><TD>S2.0F</TD><TD>12-F</TD><TD>mutant-R</TD><TD>S2.0-1 S2.0-2</TD><TD>825bp</TD></TR><TR><TD>S3.0F</TD><TD>mutant-F</TD><TD>mutant-R</TD><TD>S3.0-1 S3.0-2</TD><TD>903bp</TD></TR></TBODY></TABLE><P>2.2 Gel Electrophoresis, and get the target band we expect</P><P>2.3 Extraction of fragments from Agarose Gel.</P></DIV><DIV class="content-item"><H3>10.3</H3><P>1.1 Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>HheC-W249P -cut</TD><TD>PstⅠ</TD><TD>BamHⅠ/ NcoⅠ</TD><TD>HheC-W249P</TD></TR><TR><TD>S1.0F -cut</TD><TD>PstⅠ</TD><TD>BamHⅠ</TD><TD>S1.0F</TD></TR><TR><TD>S2.0F -cut</TD><TD>PstⅠ</TD><TD>NcoⅠ</TD><TD>S2.0F</TD></TR><TR><TD>S3.0F -cut</TD><TD>PstⅠ</TD><TD>BamHⅠ</TD><TD>S3.0F</TD></TR></TBODY></TABLE><P>1.2 Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P><P>1.3 Mix the digested fragment with backbone of HheC-W249P, then do ligation with T4 ligation enzyme to make S1.0 &amp; S2.0 &amp; S3.0.</P><P>1.4 Transform the ligation product into E.coli MC1061, Amp-plate.</P><P>2.1 Positive detection of plate piGEM2017mutant-001 &amp; piGEM2017mutant-002 &amp; piGEM2017mutant-003 &amp; piGEM2017mutant-004 &amp; piGEM2017mutant-005 &amp; piGEM2017mutant-006. After Gel Electrophoresis, get target bands we expect.</P><P>2.2 Shake bacteria fluid transformed piGEM2017mutant-001 to piGEM2017mutant-006 on the shaking table 37℃, 180r, 18h.</P></DIV><DIV class="content-item"><H3>10.5</H3><P>1.1 Conservation for the bacteria fluid of piGEM2017mutant-001 to piGEM2017mutant-006 that were shook, then use AXYGEN kit to do extraction. (successfully sequenced)</P></DIV><DIV class="content-item"><H3>10.6</H3><P>1.1 Transform the ligation product of S1.0 &amp; S2.0 &amp; S3.0 into E.coli MC1061, Amp-plate.</P></DIV><DIV class="content-item"><H3>10.7</H3><P>1.1 Positive detection of plate S1.0 &amp; S2.0 &amp; S3.0. After Gel Electrophoresis, get target bands we expect.</P><P>1.2 Shake bacteria fluid transformed S1.0 &amp; S2.0 &amp; S3.0 on the shaking table 37℃, 180r, 18h.</P></DIV><DIV class="content-item"><H3>10.10</H3><P>1.1 Mix the digested fragment of S1.0F –cut &amp; S2.0F –cut &amp; S3.0F -cut with backbone of HheC-W249P, then do ligation with T4 ligation enzyme to make S1.0 &amp; S2.0 &amp; S3.0.</P><P>1.2 Transform the ligation product into E.coli MC1061, Amp-plate.</P><P>2.1 Detect the multienzyme conversion of TCP in piGEM2017-004 with HheC produced by recombinant E.coli.</P></DIV><DIV class="content-item"><H3>10.11</H3><P>1.1 Polymerase Chain Reaction</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-P06F</TD><TD>iGEMpart006-F</TD><TD>iGEMpart006-R</TD><TD>piGEM2017mutant-001</TD><TD>814bp</TD></TR><TR><TD>piGEM2017-P07F</TD><TD>iGEMpart006-F</TD><TD>iGEMpart006-R</TD><TD>piGEM2017mutant-002</TD><TD>814bp</TD></TR><TR><TD>piGEM2017-P08F</TD><TD>iGEMpart006-F</TD><TD>iGEMpart008-R</TD><TD>piGEM2017mutant-003</TD><TD>814bp</TD></TR><TR><TD>piGEM2017-P09F</TD><TD>iGEMpart006-F</TD><TD>iGEMpart008-R</TD><TD>piGEM2017mutant-004</TD><TD>814bp</TD></TR><TR><TD>piGEM2017-P10F</TD><TD>iGEMpart006-F</TD><TD>iGEMpart008-R</TD><TD>piGEM2017mutant-005</TD><TD>814bp</TD></TR><TR><TD>piGEM2017-P11F</TD><TD>iGEMpart006-F</TD><TD>iGEMpart008-R</TD><TD>piGEM2017mutant-006</TD><TD>814bp</TD></TR><TR><TD>piGEM2017-P12F</TD><TD>iGEMpart006-F</TD><TD>iGEMpart006-R</TD><TD>piGEM2017mutant-001</TD><TD>814bp</TD></TR></TBODY></TABLE><P>1.2 Gel Electrophoresis, and get the target band we expect.</P><P>1.3 Extraction of fragments from Agarose Gel.</P></DIV><DIV class="content-item"><H3>10.12</H3><P>1.1 Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>pSB1C3 -cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>pSB1C3</TD></TR><TR><TD>piGEM2017-P06F-cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>piGEM2017-P06F</TD></TR><TR><TD>piGEM2017-P07F-cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>piGEM2017-P07F</TD></TR><TR><TD>piGEM2017-P08F-cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>piGEM2017-P08F</TD></TR><TR><TD>piGEM2017-P09F-cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>piGEM2017-P09F</TD></TR><TR><TD>piGEM2017-P10F-cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>piGEM2017-P10F</TD></TR><TR><TD>piGEM2017-P11F-cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>piGEM2017-P11F</TD></TR><TR><TD>piGEM2017-P12F-cut</TD><TD>PstⅠ</TD><TD>EcoRⅠ</TD><TD>piGEM2017-P12F</TD></TR></TBODY></TABLE><P>1.2 Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P></DIV><DIV class="content-item"><H3>10.13</H3><P>1.1 Mix the digested fragment with backbone of pSB1C3, then do ligation with T4 ligation enzyme to make piGEM2017-P06 &amp; piGEM2017-P07 &amp; piGEM2017-P08 &amp; piGEM2017-P09 &amp; piGEM2017-P10 &amp; piGEM2017-P11 &amp; piGEM2017-P12.</P><P>1.2 Transform the ligation product into E.coli DH5α, Chl-plate.</P></DIV><DIV class="content-item"><H3>10.14</H3><P>1.1 Positive detection of plate piGEM2017-P06 &amp; piGEM2017-P07 &amp; piGEM2017-P08 &amp; piGEM2017-P09 &amp; piGEM2017-P10 &amp; piGEM2017-P11 &amp; piGEM2017-P12. After Gel Electrophoresis, get target bands we expect.</P><P>1.2 Shake bacteria fluid transformed piGEM2017-P06 &amp; piGEM2017-P07 &amp; piGEM2017-P08 &amp; piGEM2017-P09 &amp; piGEM2017-P10 &amp; piGEM2017-P11 &amp; piGEM2017-P12 on the shaking table 37℃, 180r, 18h.</P></DIV><DIV class="content-item"><H3>10.16</H3><P>1.1 Conservation for the bacteria fluid of piGEM2017-P06 &amp; piGEM2017-P07 &amp; piGEM2017-P08 &amp; piGEM2017-P09 &amp; piGEM2017-P10 &amp; piGEM2017-P11 &amp; piGEM2017-P12 that were shook, then use AXYGEN kit to do extraction. (successfully sequenced)</P></DIV><DIV class="content-item"><H3>10.18</H3><P>1.1 Polymerase Chain Reaction</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017mutant-007*-1</TD><TD>mutant-F</TD><TD>DhaA135-R</TD><TD>DhaA31</TD><TD>518bp</TD></TR><TR><TD>piGEM2017mutant-007*-2</TD><TD>DhaA135-F</TD><TD>DhaA245-R</TD><TD>DhaA31</TD><TD>367bp</TD></TR><TR><TD>piGEM2017mutant-007*-3</TD><TD>DhaA245-F</TD><TD>mutant-R</TD><TD>DhaA31</TD><TD>246bp</TD></TR></TBODY></TABLE><P>1.2 Gel Electrophoresis, and get the target band we expect.</P><P>1.3 Extraction of fragments from Agarose Gel.</P><P>1.4 Fusion PCR</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017mutant-007*F</TD><TD>mutant-F</TD><TD>mutant-R</TD><TD>piGEM2017mutant-007*-1piGEM2017mutant-007*-2piGEM2017mutant-007*-3</TD><TD>1055bp</TD></TR></TBODY></TABLE><P>1.5 Gel Electrophoresis, and get the target band we expect</P><P>1.6 Extraction of fragments from Agarose Gel.</P></DIV><DIV class="content-item"><H3>10.20</H3><P>1.1 Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>DhaA31 -cut</TD><TD>PstⅠ</TD><TD>BamHⅠ</TD><TD>DhaA31</TD></TR><TR><TD>piGEM2017mutant-007*F -cut</TD><TD>PstⅠ</TD><TD>BamHⅠ</TD><TD>piGEM2017mutant-007*F</TD></TR></TBODY></TABLE><P>1.2 Gel Electrophoresis, and get the target band we expect</P><P>1.3 Extraction of fragments from Agarose Gel.</P><P>1.4 Mix the digested fragment with backbone of DhaA31 then do ligation with T4 ligation enzyme to make piGEM2017mutant-007*.</P><P>1.5 Transform the ligation product into E.coli DH5α, Amp-plate.</P></DIV><DIV class="content-item"><H3>10.21</H3><P>1.1 Positive detection of plate piGEM2017mutant-007*. After Gel Electrophoresis, get target bands we expect.</P><P>1.2 Shake bacteria fluid transformed piGEM2017mutant-007* on the shaking table 37℃, 180r, 18h.</P><P>1.3 We measured the standard curve of glycerol by the method of GC with a DB-5 column (Supelco, Bellefonate, PA, USA).</P></DIV><DIV class="content-item"><H3>10.22</H3><P>1.1 Conservation for the bacteria fluid of piGEM2017mutant-007* that was shook, then use AXYGEN kit to do extraction.</P><P>2.1 Polymerase Chain Reaction</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017mutant-007-1</TD><TD>mutant-F</TD><TD>DhaA176-R</TD><TD>piGEM2017mutant-007*</TD><TD>640bp</TD></TR><TR><TD>piGEM2017mutant-007-2</TD><TD>DhaA176-F</TD><TD>DhaA273-R</TD><TD>piGEM2017mutant-007*</TD><TD>322bp</TD></TR><TR><TD>piGEM2017mutant-007-3</TD><TD>DhaA273-F</TD><TD>mutant-R</TD><TD>piGEM2017mutant-007*</TD><TD>158bp</TD></TR></TBODY></TABLE><P>2.2 Gel Electrophoresis, and get the target band we expect.</P><P>2.3 Extraction of fragments from Agarose Gel.</P><P>2.4 Glycerol was detected in our reaction mixture, and result shows the wild type tobacco couldn’t convert 1,2,3-TCP into GLY, but our “super tobacco”did.</P></DIV><DIV class="content-item"><H3>10.23</H3><P>1.1 Fusion PCR</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Primer-F</TH><TH>Primer-R</TH><TH>Templation</TH><TH>Size</TH></TR></THEAD><TBODY><TR><TD>piGEM2017mutant-007F</TD><TD>mutant-F</TD><TD>mutant-R</TD><TD>piGEM2017mutant-007-1piGEM2017mutant-007-2piGEM2017mutant-007-3</TD><TD>1055bp</TD></TR></TBODY></TABLE><P>1.2 Gel Electrophoresis, and get the target band we expect</P><P>1.3 Extraction of fragments from Agarose Gel.</P><P>1.4 Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Designation</TH><TH>Enzyme-A</TH><TH>Enzyme-B</TH><TH>Templation</TH></TR></THEAD><TBODY><TR><TD>piGEM2017mutant-007F -cut</TD><TD>PstⅠ</TD><TD>BamHⅠ</TD><TD>piGEM2017mutant-007F</TD></TR></TBODY></TABLE><P>1.5 Use AXYGEN enzymatic reaction kit to do DNA cleanup.</P><P>1.6 Mix the digested fragment with backbone of DhaA31 then do ligation with T4 ligation enzyme to make piGEM2017mutant-007.</P><P>1.7 Transform the ligation product into E.coli DH5α and MC1061, Amp-plate.</P></DIV><DIV class="content-item"><H3>10.24</H3><P>1.1 Restriction Enzyme Digest.</P><TABLE border="" cellspacing="" cellpadding="" class="form-hover"><THEAD><TR><TH>Plasmids</TH><TH>Group 1</TH><TH>Group 2</TH></TR></THEAD><TBODY><TR><TD>piGEM2017-001</TD><TD>PstⅠ、BamHⅠ</TD><TD>ScaⅠ</TD></TR><TR><TD>piGEM2017-002</TD><TD>SacⅡ、BamHⅠ</TD><TD>SacⅡ、PstⅠ</TD></TR><TR><TD>piGEM2017-003</TD><TD>EcoNⅠ、BamHⅠ</TD><TD>ScaⅠ</TD></TR><TR><TD>piGEM2017-004</TD><TD>EcoNⅠ、BamHⅠ</TD><TD>PstⅠ、BamHⅠ</TD></TR><TR><TD>piGEM2017-005</TD><TD>EcoNⅠ、BamHⅠ</TD><TD>ScaⅠ</TD></TR><TR><TD>piGEM2017-021</TD><TD>PstⅠ</TD><TD>EcoRⅠ、BamHⅠ</TD></TR><TR><TD>piGEM2017-022</TD><TD>PstⅠ</TD><TD>EcoRⅠ、BamHⅠ</TD></TR><TR><TD>piGEM2017-023</TD><TD>PstⅠ</TD><TD>EcoRⅠ、BamHⅠ</TD></TR><TR><TD>piGEM2017-024</TD><TD>PstⅠ</TD><TD>EcoRⅠ、BamHⅠ</TD></TR><TR><TD>piGEM2017-025</TD><TD>PstⅠ</TD><TD>EcoRⅠ、BamHⅠ</TD></TR><TR><TD>piGEM2017-026</TD><TD>PstⅠ</TD><TD>EcoRⅠ、BamHⅠ</TD></TR><TR><TD>piGEM2017-027</TD><TD>PstⅠ</TD><TD>EcoRⅠ、BamHⅠ</TD></TR></TBODY></TABLE><P>1.2 Gel Electrophoresis</P><P class="mid">a. piGEM2017-001 b. piGEM2017-002 c. piGEM2017-003 d. piGEM2017-004</P><P class="mid">e. piGEM2017-005 f. piGEM2017-021 g. piGEM2017-022 h. piGEM2017-023</P><P class="mid">i. piGEM2017-024 j. piGEM2017-025 k. piGEM2017-026 l. piGEM2017-027</P></DIV><DIV class="content-item"><H3>10.25</H3><P>1.1 Detect the multienzyme conversion of TCP in piGEM2017-005 with HheC produced by recombinant E.coli.</P></DIV><DIV class="content-item"><H3>10.25-11.1</H3><P>During this time, we mainly resort out the experimental ideas and organize experimental results. Moreover, we improved the wiki various sections, practiced oral English, prepared presentation, made PPT, and performed the rehearsals for many times. We deeply appreciated the joy of team life. We did everything possible to prepare for the final plea, and believed that we would achieve satisfactory grades.</P></DIV></DIV></DIV></DIV><DIV id="In-November"><H2>In November</H2><DIV class="content-out"><DIV class="content-in"><DIV class="content-item"><H3>11.7</H3><P><B>We set out for Boston! Fighting!</B></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>