Team:UESTC-China/day-note
Team:UESTC-China/Daynote - 2017.igem.org
ACHIEVEMENT
PROJECT ∨
MODELING
ATTRIBUTIONS
HUMAN PRACTICE ∨
TEAM ∨
NOTEBOOK ∨
Part
Demonstrate
Design
Introduction
Engagement
Supporting Research
Our Story
Collaborations
Team Introduce
Interlab
Safety
Protocal
Plant
Day Note
In April
In May
In June
In July
In August
In September
In October
In November
In April
The school competition of iGEM in University of Electronic Science and Technology of China was held. After series of individual races and team competitions, excellent people were finally chosen into iGEM team of 2017: UESTC-China. Then we started to search for some useful and interesting projects. After four sessions of the group discussion and papers reading, we decided to choose TCP degradation by plants as our project. Then we started to carry out primary design of the project and then synthesized some key gene.
In May
Follow the primary design, we mainly did Molecular cloning experiments this month.
5.1-5.7
We had constructed some elements for the final single gene plasmid, which mainly includegd pGE1.1-DhaA31, pGE1.2-HheC, pGE1.3-EchA and pSE01-Nptll. Some parameters were as follow.
In addition, we had sowed tobacco seeds on the Medium for several times.
Designation
Primer-F
Primer-R
Size
pGE1.1-DhaA31
M13-F
\
1040bp
pGE1.2-HheC
M13-F
M13-R
923bp
pGE1.3-EchA
M13-F
M13-R
1043bp
pSE01-Nptll
M13-F
M13-R
1033bp
pGE1.1-DhaA31,pGE1.2-Hhec Colony PCR
pGE1.3-EchA Colony PCR
5.8-5.14
This week, three single gene vectors were constructed successfully by forward elements. In addition to that, we also completed other elements construction due to the need of Multi-gene vector. Some parameters are as follow. At the end of the week, we transferred tobacco seedlings from previous seeds sowing to other mediums for amplification and follow-up infection.
Designation
Primer-F
Primer-R
Size
PiGEM2017-001
TC013-R
ZY065-RB
953bp
PiGEM2017-002
TC013-R
ZY065-RB
953bp
PiGEM2017-003
TC013-R
ZY065-RB
953bp
pGE02-HheC
M13-F
M13-R
991bp
pGE03-EchA
M13-F
M13-R
1108bp
pSE03-NptII
M13-F
M13-R
1033bp
PiGEM2017-001, PiGEM2017-002, PiGEM2017-003 Colony PCR
pGE02-HheC colony PCR
pGE03-EchA colony PCR
pSE03-NptII colony PCR
5.15-5.21
It’s suspected that there may be stable expression of epoxide hydrolases in the plant. To verify this suppose, this week, we designed and constructed a two genes vector system piGEM2016-005 excluding EchA. What’s more, we carried out Agrobacterium transformation using prior plasmids PiGEM2017-001, PiGEM2017-002, PiGEM2017-003. Also, Agrobacterium’s positive defection is done.
pSE02-Nptll colony PCR
piGEM2017-005 colony PCR
5.22-5.31
This week, based on prior completed plasmid piGEM2017-005, Agrobacterium transformation was carried out. What’s more, the first Agrobacterium infecting tobacco carrying PiGEM2017-001, PiGEM2017-002, PiGEM2017-003 was carried out.
In June
6.1-6.10
The basic plasmids have been completed. This week we did Agrobacterium infection of tobacco including piGEM2017-001, piGEM2017-002, piGEM2017-003, piGEM2017-004, piGEM2017-005. Besides that, we started to try to optimize our plasmids design.
6.11-6.20
By papers reading, rough plasmids design had been made sure. And the target sequence was sent to company for synthesizing. Furthermore, Agrobacterium infection of tobacco had been done.
6.21-6.30
The leaves infected by Agrobacterium had appeared small bud. We transferred the bud to rooting medium.
Except, we did the standard curve of chloride ions
In July
7.6
Restriction Enzyme Digest for PYK10 fragment. There is a restriction site of PstI on PYK10, so we can get two part of it.
Designation
Enzyme-A
Enzyme-B
Templation
Size
pPE-PYK10
KpnⅠ
PstⅠ
iGEM2017-SG05
998bp、478bp
Gel Electrophoresis.
Extraction of DNA from Agarose Gel.
Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
templation
Backbone of pZHY998
KpnⅠ
PstⅠ
pZHY998
Use AXYGEN enzymatic reaction kit to do DNA cleanup.
Mix the digested fragment with backbone of pZHY998, then do ligation with T4 ligation enzyme to make pPE-PYK10.
Transform the ligation product into E.coli DH5α, Amp-plate.
7.7
Positive detection of plate pPE-PYK10. After Gel Electrophoresis, there is only one obvious target band we expect, 1609bp.
Shake bacteria fluid transformed with pPE-PYK10 on the shaking table 37℃, 180r, 18h.
7.8
Conservation for the bacteria fluid of pPE-PYK10, then use AXYGEN kit to do extraction. But the concentration is low.
Restriction Enzyme Digest for verification .
Designation
Enzyme-A
Enzyme-B
Size
pPE-PYK10
KpnⅠ
PstⅠ
2716bp、998bp、478bp
Gel Electrophoresis, and get the target band we expect.
Because the extracted pPE-PYK10 plasmid concentration is too low, shake preservative bacteria of pPE-PYK10 on the shaking table 37℃, 180r, 18h again.
7.9
Conservation for the bacteria fluid of pPE-PYK10, then use AXYGEN kit to do extraction. (successfully sequenced)
Golden Gate Cloning for piGEM2017-010.
Using Agrobacterium tumefaciens EHA105 transformed vector（PiGEM001、002、003、004、005、006）into Tobacco using the leaf-dipping method and then put the leaves into MSB-Co medium (co-cultivate)
7.10
Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
Extract the genome of Rice. But the OD600 is just 9.9.
7.11
1 ) Positive detection of plate piGEM2017-010. After Gel Electrophoresis, there showed clear band corresponding to the expected sequence size, 953bp.
2 ) fasfsafasfsdafds
1 ) Polymerase Chain Reaction.
Designation
Primer-F
Primer-R
Templation
Size
pSE04-NptII
iGEMLBL033-F
iGEMLBL022-R
pSE03-NptII
923bp
pGE01-eYFP
iGEMXDM003-F
iGEMXDM003-R
piGEM2017-006
815bp
pGE01AE-CKX3
iGEMLBL027-F
iGEMLBL027-R
pGE01A-CKX3
1728bp
pGE02AE-CKX3
iGEMXDM004-F
iGEMXDM004-R
iGEM2017-SG04
1737bp
pGE02-CKX3
iGEMLBL029-F
iGEMLBL029-R
pGE01A-CKX3
1625bp
pGE04-CKX3
iGEMLBL032-F
iGEMLBL032-R
pGE01A-CKX3
1623bp
pGE01-CKX3
iGEMXDM001-F
iGEMXDM001-R
iGEM2017-SG04
1609bp
piGEM2017-011
iGEMXDM002-F
iGEMXDM002-R
iGEM2017-SG05
1488bp
pGE03-CKX3
iGEMLBL030-F
iGEMLBL030-R
pGE01A-CKX3
1618bp
2 ) Gel Electrophoresis of PCR-Amplified product，and get the target band we expect.
3 ) Extraction of DNA from Agarose Gel.
Using Agrobacterium tumefaciens EHA105 transformed vector（PiGEM001、002、003、004、005、006）into Tobacco using the leaf-dipping method and then put the leaves into MSB-Co medium (co-cultivate). The materials that were co-cultivate（7.9）were placed on the Agrobacterium-eliminating MSB-S medium
7.12
1 ) Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pSE04-NptⅡ-cut
KpnⅠ
PstⅠ
pSE04-NptⅡ
pGE01-eYFP-cut
KpnⅠ
PstⅠ
pGE01-eYFP
pGE01AE-CKX3-cut
KpnⅠ
PstⅠ
pGE01AE-CKX3
pGE01-CKX3-cut
KpnⅠ
PstⅠ
pGE01-CKX3
Backbone of pZHY998
KpnⅠ
PstⅠ
pZHY998
2 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.
3 ) Mix the four digested fragment with backbone of pZHY998, then do ligation with T4 ligation enzyme.
4 ) Transform the ligation product into E.coli DH5α, Amp-plate.
1 ) Conservation for the bacteria fluid of piGEM2017-010 that was shook yesterday, then use AXYGEN kit to do extraction. (successfully sequenced)
2 ) Restriction Enzyme Digest for verification .
Designation
Enzyme
Size
piGEM2017-010
ScaⅠ
6258bp、3386bp、407bp
3 ) Gel Electrophoresis, and get the target band we expect.
7.13
1 ) Positive detection of plate pSE04-NptⅡ（1033bp）、pGE01-eYFP（935bp）、pGE01AE-CKX3（1730bp）、pGE01-CKX3（1844bp）. After Gel Electrophoresis, only pSE04-NptⅡand pGE01AE-CKX3 have target bands we expect.
2 ) Shake bacteria fluid transformed with pSE04-NptⅡ and that with pGE01AE-CKX3 on the shaking table 37℃, 180r, 18h.
He materials that were co-cultivate（7.11）were placed on the Agrobacterium-eliminating MSB-S medium
DhaA31 assay by the method of chloride ions in Petunia. Weight the Petunia leaves of piGEM2017-001-1 0.2237g, piGEM2017-002-2 0.0692g, wild-type 0.1163g, Grind with 200μl Tris-SO4 (200Mm, pH 8.0). Result shows no products were detected.
7.14
Conservation for the bacteria fluid of pSE04-NptⅡ& pGE01AE-CKX3-1 & pGE01AE-CKX3-1, then use AXYGEN kit to do extraction. (unsuccessfully sequenced)
1 ) Polymerase Chain Reaction.
Designation
Primer-F
Primer-R
Templation
Size
pGE01-CKX3
iGEMXDM001-F
iGEMXDM001-R
iGEM2017-SG04
1609bp
pGE01.1A-DhaA31
iGEMXDM005-F
iGEMXDM005-R
pGE01.1- DhaA31
936bp
pGE01.2A-HheC
iGEMXDM006-F
iGEMXDM006-R
pGE01.2-HheC
826bp
pGE01.3A-EchA
iGEMXDM007-F
iGEMXDM007-R
pGE01.3-EchA
943bp
2 ) Gel Electrophoresis of PCR-Amplified product，and get the target band we expect.
3 ) Extraction of DNA from Agarose Gel.
4 ) Restriction Enzyme Digest.
designation
Enzyme-A
Enzyme-B
templation
CKX3 fragment-cut
KpnⅠ
PstⅠ
CKX3 fragment
5 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.
1 ) Golden Gate Cloning for piGEM2017-010.
2 ) Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
7.15
1 ) Mix the CKX3 fragment & the eYFP fragment with backbone of pZHY998, then do ligation with T4 ligation enzyme to make pGE01-CKX3 & pGE01-eYFP.
2 ) Transform the ligation product into E.coli DH5α, Amp-plate.
1 ) Positive detection of plate piGEM2017-008（1100bp）. After Gel Electrophoresis, get target bands we expect.
2 ) Shake bacteria fluid transformed with piGEM2017-008 on the shaking table 37℃, 180r, 18h.
7.16
1 ) Positive detection of plate pGE01-eYFP（935bp）、pGE01-CKX3-1（1730bp）、pGE01-CKX3-2（1730bp）. After Gel Electrophoresis, get target bands we expect.
2 ) Shake bacteria fluid transformed with piGEM2017-008 & pGE01-CKX3 on the shaking table 37℃, 180r, 18h.
Conservation for the bacteria fluid of piGEM2017-008 that was shook yesterday, then use AXYGEN kit to do extraction.
Using Agrobacterium tumefaciens EHA105 transformed vector（PiGEM001、002、003、004、005、006）into Tobacco using the leaf-dipping method and then put the leaves into MSB-Co medium (co-cultivate).
7.17
1 ) Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pGE02-cut
EcoRⅠ
HindⅢ
pGE02
pGE03-cut
EcoRⅠ
HindⅢ
pGE03
pGE04-cut
EcoRⅠ
HindⅢ
pGE04
2 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.
1 ) Conservation for the bacteria fluid of piGEM2017-008 & pGE01-CKX3-1 & pGE03-CKX3-2 that was shook yesterday, then
2 ) Restriction Enzyme Digest.
designation
Enzyme-A
Enzyme-B
templation
piGEM2017-008-cut
EcoNⅠ
BglⅡ
piGEM2017-008
piGEM2017-008-cut
EcoNⅠ
AvrⅡ
piGEM2017-008
3 ) Gel Electrophoresis, but don’t get the target band we expect (1: 7196bp & 3451bp; 2: 7178bp & 3419bp).
7.18
1 ) Positive detection of plate pGE01AE-CKX3、pSE04-NPtⅡ. After Gel Electrophoresis, get target bands we expect.
2 ) Shake bacteria fluid transformed with pGE01AE-CKX3 on the shaking table 37℃, 180r, 18h.
1 ) Golden Gate Cloning for piGEM2017-024.
2 ) Transform the ligation product into E.coli DH5α, Kan-plate.
Transform the pSE04-NPtⅡ ligation product into E.coli DH5α, Amp-plate.
The materials that were co-cultivate（7.16）were placed on the Agrobacterium-eliminating MSB-S medium
The method of gas chromatography was measured because our column can detect the metabolites of the TCP degradation pathway.
7.19
1 ) Positive detection of plate piGEM2017-024、pSE04-NPtⅡ. After Gel Electrophoresis, get target bands we expect.
2 ) Shake bacteria fluid transformed with pSE04-NPtⅡ on the shaking table 37℃, 180r, 18h.
Conservation for the bacteria fluid of pGE01AE-CKX3 that was shook yesterday, then use AXYGEN kit to do extraction.
7.20
1 ) Golden Gate Cloning for piGEM2017-007 & piGEM2017-009.
2 ) Transform the ligation product into E.coli DH5α, Kan-plate.
Conservation for the bacteria fluid of pSE04-NPtⅡ that was shook yesterday, then use AXYGEN kit to do extraction.
Using Agrobacterium tumefaciens EHA105 transformed vector（PiGEM001、002、003、004、005、006）into Tobacco using the leaf-dipping method and then put the leaves into MSB-Co medium (co-cultivate).
DhaA31 assay by gas chromatography with tobacco. Extract the leaves enzymes of piGEM2017-001 and wild type, add the same amount of rough enzymes into the reaction mixture. No DCP was detected in the end.
7.21
1 ) Positive detection of platepSE04-NPtⅡ、piGEM2017-009. After Gel Electrophoresis, get target bands we expect.
2 ) Shake bacteria fluid transformed with piGEM2017-009 & pSE04-NPtⅡ on the shaking table 37℃, 180r, 18h.
Conservation for the bacteria fluid of AV3 that was shook yesterday, then use AXYGEN kit to do extraction.
Transplant the bud from MSB-S medium to MSB-R medium
7.22
Conservation for the bacteria fluid of pSE04-NPtⅡ & piGEM2017-009 & pSB143-chl that was shook yesterday, then use AXYGEN kit to do extraction.
Extract the genome of 4 wide Nicotiana tabacum (NT) and 4 NT transformed piGEM2017-004 and piGEM2017-006, respectively. Carry out PCR of all samples to detect 35S promoter. After Gel Electrophoresis, only samples of piGEM2017-004 appeared brands(842bp).
The materials that were co-cultivate（7.20）were placed on the Agrobacterium-eliminating MSB-S medium
7.23
1 ) Restriction Enzyme Digest.
designation
Enzyme-A
Enzyme-B
templation
Backbone of pGE01AE-CKX3
BamHⅠ
EcoRⅠ
pGE01AE-CKX3
2 ) Gel Electrophoresis of Restriction Enzyme Digest product，and get the target band we expect(2852bp).
3 ) Extraction of backbone of pGE01AE-CKX3 from Agarose Gel
4 ) Mix AO-S DhaA31 fragment & AO-S HheC fragment & AO-S EchA fragment with backbone of pGE01AE-CKX3, then do ligation with Gibson enzyme to make pGE01.1A-DhaA31 & pGE01.2A-Hhec & pGE01.3A-EchA.
5 ) Transform the ligation product into E.coli DH5α, Amp-plate.
1 ) Restriction Enzyme Digest.
piGEM2017-009-cut
Enzyme-A
Enzyme-B
templation
piGEM2017-009-cut
ScaⅠ
ScaⅠ
piGEM2017-009
2 ) Gel Electrophoresis, and get the target band we expect.
Conservation for the bacteria fluid of AV3 that was shook yesterday, then use AXYGEN kit to do extraction.
7.24
1 )Positive detection of plate pGE01.1A-DhaA31 、 pGE01.2A-Hhec 、 pGE01.3A-EchA、piGEM2017-024 、 piGEM2017-007. After Gel Electrophoresis, get target bands we expect.
2 ) Shake bacteria fluid transformed with pGE01.1A-DhaA31 & pGE01.2A-Hhec & pGE01.3A-EchA & piGEM2017-007 on the shaking table 37℃, 180r, 18h.
7.25
1 ) Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pGE02-cut
EcoRⅠ
HindⅢ
pGE02
pGE03-cut
EcoRⅠ
HindⅢ
pGE03
pGE04-cut
EcoRⅠ
HindⅢ
pGE04
2 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.
1 ) Polymerase Chain Reaction.
Designation
Primer-F
Primer-R
Templation
Size
pSE04
ZY312-F5
ZY312-R
pZHY979
135bp
pSE04-NPtⅡ
iGEMLBL022-F
iGEMLBL022-R
pZHY980
855bp
2 ) Gel Electrophoresis, and get the target band we expect.
3 ) Extraction of fragments from Agarose Gel.
4 ) Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pZHY998-cut
KpnⅠ
PstⅠ
pZHY998
5 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.
6 ) Mix F2A fragment with backbone of pZHY998, then do ligation with T4 ligation enzyme to make pSE04.
7 ) Transform the ligation product into E.coli DH5α, Amp-plate.
Conservation for the bacteria fluid of pGE01.1A-DhaA31 & pGE01.2A-Hhec & pGE01.3A-EchA & piGEM2017-007 & piGEM2017-010 that was shook yesterday, then use AXYGEN kit to do extraction.
1 ) Mix AO-S CKX3 fragment & CKX3 fragment with backbone of pGE02 & pGE03 & pGE04, then do ligation with Gibson enzyme to make pGE02AE-CKX3 & pGE02-CKX3 & pGE03-CKX3 & pGE04-CKX3.
2 ) Transform the ligation product into E.coli DH5α, Amp-plate or Kan-plate.
1 ) Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
piGEM2017-007-cut
ScaⅠ
ScaⅠ
piGEM2017-007
2 ) Gel Electrophoresis, and get the target band we expect(6258bp & 3686bp & 595bp).
Extract the genome of new wide NT and NT transformed piGEM2017-004，piGEM2017-006 as well as NT transformed piGEM2017-005. After Gel Electrophoresis, wide samples had no brands and transgenic NT had purpose brands marked 842bp.
7.26
1 ) Mix the F2A fragment with backbone of pZHY998, then do ligation with T4 ligation enzyme to make pSE04.
2 ) Transform the ligation product into E.coli DH5α, Amp-plate.
1 ) Positive detection of plate pGE02AE-CKX3 & pGE02-CKX3 & pGE03-CKX3 & pGE04-CKX3 & piGEM2017-024. After Gel Electrophoresis, get target bands of pGE02AE-CKX3 & pGE02-CKX3 & pGE04-CKX3 we expect.
2 ) Shake bacteria fluid transformed with pGE02AE-CKX3 & pGE02-CKX3 & pGE04-CKX3 on the shaking table 37℃, 180r, 18h.
The plasmids（PiGEM008、009） were introduced into Agrobacterium tumefaciens EHA105 by freeze-thaw method Transformation
7.27
1 ) Positive detection of plate pSE04 & pGE01.2A-HheC. After Gel Electrophoresis, get target bands of pGE01.2A-HheC we expect.
2 ) Shake bacteria fluid transformed with pGE01.2A-HheC & pGE02AE-CKX3 on the shaking table 37℃, 180r, 18h.
1 ) Golden Gate Cloning for piGEM2017-024.
2 ) Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
Conservation for the bacteria fluid of pGE02AE-CKX3 &pGE02-CKX3 &pGE04-CKX3 that was shook yesterday, then use AXYGEN kit to do extraction.
7.28
1 ) Positive detection of plate piGEM2017-024 、 pGE01.3A-EchA. After Gel Electrophoresis, get target bands we expect.
2 ) Shake bacteria fluid transformed with piGEM2017-024 & pGE01.3A-EchA on the shaking table 37℃, 180r, 18h.
Conservation for the bacteria fluid of pGE02AE-CKX3 & pGE01.2A-HheC that was shook yesterday, then use AXYGEN kit to do extraction.
Transplant the bud from MSB-S medium to MSB-R medium
7.29
Conservation for the bacteria fluid of piGEM2017-024 & pGE01.3A-EchA that were shook yesterday, then use AXYGEN kit to do extraction.
Extract the genome of new NT transformed piGEM2017-002 to 006 except piGEM2017-003. After Gel Electrophoresis, we get 4 strains of NT-004, 3 strains of NT-005 and 3 strains of NT-006 at last which are all positive.
Using Agrobacterium tumefaciens EHA105 transformed vector（PiGEM001、002、003、004、005、006、008、009）into Tobacco using the leaf-dipping method and then put the leaves into MSB-Co medium (co-cultivate) Transformation of Tobacco. co-cultivate
To explore which time could show the enzymatic activity apparently, piGEM2017-001 was chosen to do experiments. The samples were withdrawn from the reaction mixture every hour and we found that the conversion of TCP to DCP reached maximum in seven hours, so 7h were selected to show the activity of all three enzymes.
Figure. Time courses of concentration of production 2,3-DCP with piGEM2017-001.
Reactions were performed using different lines of piGEM2017-001.
7.30
1 ) Polymerase Chain Reaction.
Designation
Primer-F
Primer-R
Templation
Size
pSE04
ZY312-F5
ZY312-R
pZHY979
135bp
2 ) Gel Electrophoresis, and get the target band we expect.
3 ) Extraction of fragments from Agarose Gel.
4 ) Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pZHY998-cut
KpnⅠ
PstⅠ
pZHY998
F2A
KpnⅠ
PstⅠ
pSE04 fragment
5 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.
6 ) Mix F2A fragment with backbone of pZHY998, then do ligation with T4 ligation enzyme to make pSE04.
7 ) Transform the ligation product into E.coli DH5α, Amp-plate.
Transplant the bud from MSB-S medium to MSB-R medium
7.31
1 ) Polymerase Chain Reaction.
Designation
Primer-F
Primer-R
Templation
Size
AOS-HheC
XDM004-F
LBL023-R
pGE01.2A-HheC
921bp
2 ) Gel Electrophoresis, and get the target band we expect.
3 ) Extraction of fragments from Agarose Gel.
4 ) Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pGE02-cut
EcoRⅠ
HindⅢ
pGE02
5 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.
6 ) Mix AOS-HheC with backbone of pGE02, then do ligation with Gibson ligation enzyme to make pGE02A-HheC.
7 ) Transform the ligation product into E.coli DH5α, Amp-plate.
The materials that were co-cultivate（7.29）were placed on the Agrobacterium-eliminating MSB-S medium
In August
8.1
1 ) Golden Gate Cloning for piGEM2017-024.
2 ) Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
1 ) Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pGE04-cut
EcoRⅠ
HindⅢ
pGE04
2 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.
3 ) Mix Gus fragment with backbone of pGE04, then do ligation with Gibson ligation enzyme to make pGE04-Gus.
4 ) Transform the ligation product into E.coli DH5α, Amp-plate.
1 ) Positive detection of plate pGE02A-HheC & pSE04. After Gel Electrophoresis, get target bands we expect.
2 ) Shake bacteria fluid transformed with pGE02A-HheC & pSE04 on the shaking table 37℃, 180r, 18h.
8.2
1 ) Positive detection of plate pGE04-Gus、pSE04、piGEM2017-024. After Gel Electrophoresis, get target bands we expect.
2 ) Shake bacteria fluid transformed with pGE04-Gus & pSE04 & pGE01.3A-EchA on the shaking table 37℃, 180r, 18h.
1 ) Mix F2A fragment with backbone of pZHY998, then do ligation with T4 ligation enzyme to make pSE04.
2 ) Transform the ligation product into E.coli DH5α, Amp-plate.
8.3
Conservation for the bacteria fluid of pGE04-Gus & pSE04 & pGE01.3A-EchA & pGE02A-HheC that was shook yesterday, then use AXYGEN kit to do extraction.
1 ) Positive detection of plate pSE04、piGEM2017-024. After Gel Electrophoresis, get target bands we expect.
2 ) Shake bacteria fluid transformed with pGE04-Gus & pSE04 & pGE01.3A-EchA on the shaking table 37℃, 180r, 18h.
The plasmids（PiGEM014、015、016） were introduced into Agrobacterium tumefaciens EHA105 by freeze-thaw method Transformation
8.4
1 ) Polymerase Chain Reaction.
Designation
Primer-F
Primer-R
Templation
Size
AOS-EchA
LBL031-F
LBL025-R
pGE01.3A-EchA
1041bp
2 ) Gel Electrophoresis, and get the target band we expect.
3 ) Extraction of fragments from Agarose Gel.
4 ) Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pGE03-cut
EcoRⅠ
HindⅢ
5 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.
6 ) Mix AOS-EchA with backbone of pGE03, then do ligation with Gibson ligation enzyme to make pGE03A-EchA.
7 ) Transform the ligation product into E.coli DH5α, Amp-plate.
Transplant the bud from MSB-S medium to MSB-R medium
8.5
Positive detection of plate pGE03A-EchA. After Gel Electrophoresis, didn’t get target bands we expect.
1 ) Mix AOS-EchA with backbone of pGE03, then do ligation with Gibson ligation enzyme to make pGE03A-EchA.
2 ) Transform the ligation product into E.coli DH5α, Amp-plate.
Conservation for the bacteria fluid of pSE04 & pGE01.3A-EchA that was shook, then use AXYGEN kit to do extraction.
8.7
1 ) Polymerase Chain Reaction.
Designation
Primer-F
Primer-R
Templation
Size
AOS-EchA
LBL031-F
LBL025-R
pGE01.3A-EchA
1041bp
2 ) Gel Electrophoresis, and get the target band we expect.
3 ) Extraction of fragments from Agarose Gel.
4 ) Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pZHZ079-cut
BglⅡ
NcoⅠ
pZHZ079
5 ) Use AXYGEN enzymatic reaction kit to do DNA cleanup.
6 ) Mix AOS-EchA with backbone of pZHZ079, then do ligation with Gibson ligation enzyme to make pGE03A-EchA.
7 ) Transform the ligation product into E.coli DH5α, Amp-plate.
8.8
1 ) Polymerase Chain Reaction.
Designation
Primer-F
Primer-R
Templation
Size
Gus fragment
LBL034-F
LBL034-R
pGE04-Gus
2081bp
2 ) Gel Electrophoresis, and get the target band we expect.
3 ) Extraction of fragments from Agarose Gel.
1 ) Positive detection of plate pGE03A-EchA. After Gel Electrophoresis, get target bands we expect.
2 ) Shake bacteria fluid transformed with pGE03A-EchA on the shaking table 37℃, 180r, 18h.
8.9
1.1 Golden Gate Cloning for piGEM2017-021& piGEM2017-025.
1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
2.1 Mix Gus fragment with backbone of pGE04, then do ligation with Gibson ligation enzyme to make pGE04-Gus.
2.2 Transform the ligation product into E.coli DH5α, Amp-plate.
8.10
1.1 Conservation for the bacteria fluid of piGEM2017-021& piGEM2017-024&pGE03A-EchA that was shook yesterday, then use AXYGEN kit to do extraction.
2.1 Positive detection of plate piGEM2017-021& piGEM2017-024& piGEM2017-025& pGE04-Gus. After Gel Electrophoresis, get target bands we expect.
2.2 Shake bacteria fluid transformed with pGE04-Gus on the shaking table 37℃, 180r, 18h.
8.11
1.1 Golden Gate Cloning for piGEM2017-021&piGEM2017-024 &piGEM2017-025.
1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
2.1 Polymerase Chain Reaction.
Designation
Primer-F
Primer-R
Templation
Size
AOS-EchA
LBL031-F
LBL025-R
pGE01.3A-EchA
1041bp
2.2 Gel Electrophoresis, and get the target band we expect.
2.3 Extraction of fragments from Agarose Gel.
2.4 Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pZHZ079-cut
BglⅡ
NcoⅠ
pZHZ079
2.5 Use AXYGEN enzymatic reaction kit to do DNA cleanup.
2.6 Mix AOS-EchA with backbone of pZHZ079, then do ligation with Gibson ligation enzyme to make pGE03A-EchA.
2.7 Transform the ligation product into E.coli DH5α, Amp-plate.
3.1 Conservation for the bacteria fluid of pGE04-Gus that was shook yesterday, then use AXYGEN kit to do extraction.
4.1 Using Agrobacterium tumefaciens EHA105 transformed vector（PiGEM008、009）into Tobacco using the leaf-dipping method and then put the leaves into MSB-Co medium (co-cultivate)
8.12
1.1 Positive detection of plate piGEM2017-021& piGEM2017-024& piGEM2017-025. After Gel Electrophoresis, didn’t get target bands we expect.
8.13
1.1 Positive detection of plate pGE03A-EchA. After Gel Electrophoresis, get target bands we expect.
1.2 Shake bacteria fluid transformed with pGE03A-EchA on the shaking table 37℃, 180r, 18h.
2.1 The materials that were co-cultivate（8.11）were placed on the Agrobacterium-eliminating MSB-S medium
8.14
1.1 Golden Gate Cloning for piGEM2017-026&piGEM2017-027.
1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
2.1 Conservation for the bacteria fluid of pGE03A-EchA that was shook yesterday, then use AXYGEN kit to do extraction.
8.15
1.1 Golden Gate Cloning for piGEM2017-021 to piGEM2017-027.
1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
2.1 PCR for all positive plants to detect each purpose gene.
2.2 Gel Electrophoresis
3.1 DhaA31 assay by gas chromatography with piGEM2017-001-4. The detection method was improved. Grind fresh leaves of equal weight in Liquid nitrogen. The reaction mixture is the supernatant of leaves (6.5mL). The result of GC shows our enzyme worked in the transgene tobacco. That’s cool!
8.16
1.1 Positive detection of plate piGEM2017-026 & piGEM2017-027. After Gel Electrophoresis, get target bands of 027 we expect.
1.2 Shake bacteria fluid transformed with piGEM2017-027 on the shaking table 37℃, 180r, 18h.
8.18
1.1 Golden Gate Cloning for piGEM2017-021 to piGEM2017-027.
1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
2.1 Shake bacteria fluid transformed with piGEM2017-027 on the shaking table 37℃, 180r, 18h.
8.19
1.1 Positive detection of plate piGEM2017-021、piGEM2017-022、piGEM2017-024. After Gel Electrophoresis, didn’t get target bands we expect.
2.1 Conservation for the bacteria fluid of piGEM2017-027 that was shook yesterday, then use AXYGEN kit to do extraction.
8.20
1.1 Golden Gate Cloning for piGEM2017-023 to piGEM2017-027.
1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
2.1 The plasmids（PiGEM014、015） were introduced into Agrobacterium tumefaciens EHA105 by freeze-thaw method Transformation
3.1 HheC assay by the same method with piGEM2017-002, but no ECH was detected.
8.22
1.1 Positive detection of plate piGEM2017-023 to piGEM2017-027. After Gel Electrophoresis, get target bands we expect.
1.2 Shake bacteria fluid transformed with piGEM2017-023 to piGEM2017-027 on the shaking table 37℃, 180r, 18h.
2.1 Golden Gate Cloning for piGEM2017-021 & piGEM2017-022.
2.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
3.1 The materials that were co-cultivate（8.20）were placed on the Agrobacterium-eliminating MSB-S medium
8.23
1.1 Conservation for the bacteria fluid of piGEM2017-024 & piGEM2017-026 & piGEM2017-027 that was shook yesterday, then use AXYGEN kit to do extraction.
1.2 Restriction Enzyme Digest for verification .
Designation
Enzyme-A
Enzyme-B
Size
piGEM2017-024
EcoRⅠ
BamHⅠ
1344bp、2683bp、7607bp
piGEM2017-026
SacⅡ
EcoRⅠ
1449bp、3630bp、8950bp
piGEM2017-027
SacⅡ
EcoRⅠ
1449bp、3630bp、8956bp
1.3 Gel Electrophoresis, and get the target band we expect.
1.4 piGEM2017-026 was successfully sequenced.
8.24
1.1 Positive detection of plate piGEM2017-021 & piGEM2017-022. After Gel Electrophoresis, get target bands of 027 we expect.
1.2 Shake bacteria fluid transformed with piGEM2017-021 & piGEM2017-022 on the shaking table 37℃, 180r, 18h.
2.1 Conservation for the bacteria fluid of piGEM2017-025 that was shook, then use AXYGEN kit to do extraction.
2.2 Restriction Enzyme Digest for verification.
Designation
Enzyme-A
Enzyme-B
Size
piGEM2017-025
SacⅡ
EcoRⅠ
1344bp、3179bp、8156bp
2.3 Gel Electrophoresis, and didn’t get the target band we expect.
3.1 Using Agrobacterium tumefaciens EHA105 transformed vector（PiGEM015、016）into Tobacco using the leaf-dipping method and then put the leaves into MSB-Co medium (co-cultivate)
8.25
1.1 Golden Gate Cloning for piGEM2017-022 & piGEM2017-025& piGEM2017-027.
1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
2.1 Conservation for the bacteria fluid of piGEM2017-021 & piGEM2017-023& piGEM2017-024 & piGEM2017-027 that was shook, then use AXYGEN kit to do extraction.
2.2 Restriction Enzyme Digest for verification.
Designation
Enzyme-A
Enzyme-B
Size
piGEM2017-021
SacⅡ
EcoRⅠ
1344bp、3179bp、8752bp
piGEM2017-023
SacⅡ
EcoRⅠ
1446bp、3179bp、8360bp
piGEM2017-024
EcoRⅠ
BamHⅠ
1344bp、2683bp、7607bp
piGEM2017-027
SacⅡ
EcoRⅠ
1449bp、3630bp、8956bp
2.3 Gel Electrophoresis, and get the target band we expect.
3.1 The plasmids（PiGEM025,026,027） were introduced into Agrobacterium tumefaciens EHA105 by freeze-thaw method Transformation
8.26
1.1 Positive detection of plate piGEM2017-022 & piGEM2017-023 & piGEM2017-025 & piGEM2017-027. After Gel Electrophoresis, get target bands we expect.
1.2 Shake bacteria fluid transformed with piGEM2017-022 & piGEM2017-023 & piGEM2017-025 & piGEM2017-027 on the shaking table 37℃, 180r, 18h.
2.1 Conservation for the bacteria fluid of pGE04-CKX3 & pGE04-Gus that were shook, then use AXYGEN kit to do extraction.
3.1 The plasmids（PiGEM025,026,027） were introduced into Agrobacterium tumefaciens EHA105 by freeze-thaw method Transformation
8.27
1.1 Conservation for the bacteria fluid of piGEM2017-022 & piGEM2017-023 & piGEM2017-025 & piGEM2017-027 that was shook, then use AXYGEN kit to do extraction.
1.2 Restriction Enzyme Digest for verification.
Designation
Enzyme-A
Enzyme-B
Size
piGEM2017-022
SacⅡ
EcoRⅠ
1446bp、3179bp、8956bp
piGEM2017-023
SacⅡ
EcoRⅠ
1446bp、3179bp、8360bp
piGEM2017-025
SacⅡ
EcoRⅠ
1344bp、3179bp、8156bp
piGEM2017-027
SacⅡ
EcoRⅠ
1449bp、3630bp、8956bp
1.3 Gel Electrophoresis, and get the target band of 022 and 025 we expect.
2.1 Golden Gate Cloning for piGEM2017-022 & piGEM2017-023.
2.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
8.28
1.1 Polymerase Chain Reaction
Designation
Primer-F
Primer-R
Templation
Size
piGEM2017-P01F
iGEMpart001-F
iGEMpart001-F
pGE01.1A-DhaA31
1030bp
piGEM2017-P02-1
iGEMpart002-F
iGEMpart002-RM
pGE01.2A-HheC
305bp
piGEM2017-P02-2
iGEMpart002-FM
iGEMpart002-R
pGE01.2A-HheC
550bp
piGEM2017-P03-1
iGEMpart003-F
iGEMpart003-RM1
pGE01.3A-EchA
304bp
piGEM2017-P03-2
iGEMpart003-FM1
iGEMpart003-RM2
pGE01.3A-EchA
556bp
piGEM2017-P03-3
iGEMpart003-FM2
iGEMpart003-R
pGE01.3A-EchA
159bp
piGEM2017-P04F
iGEMpart004-F
iGEMpart004-R
pGE04-CKX3
1618bp
1.2 Gel Electrophoresis, and get the target band we expect.
1.3 Extraction of fragments from Agarose Gel.
1.4 Fusion PCR
Designation
Primer-F
Primer-R
Templation
Size
piGEM2017-P02F
iGEMpart002-F
iGEMpart002-R
piGEM2017-P02-1
piGEM2017-P02-2
811bp
piGEM2017-P03F
iGEMpart003-F
iGEMpart003-R
piGEM2017-P03-1
piGEM2017-P03-2
piGEM2017-P03-3
931bp
1.5 Gel Electrophoresis, and get the target band we expect
1.6 Extraction of fragments from Agarose Gel.
1.7 Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pSB1C3-cut
PstⅠ
EcoRⅠ
pSB1C3
piGEM2017-P01F-cut
PstⅠ
EcoRⅠ
piGEM2017-P01F
piGEM2017-P02F-cut
PstⅠ
EcoRⅠ
piGEM2017-P02F
piGEM2017-P03F-cut
PstⅠ
EcoRⅠ
piGEM2017-P03F
piGEM2017-P04F-cut
PstⅠ
EcoRⅠ
piGEM2017-P04F
1.8 Mix the 4 digested fragment with backbone of pSB1C3, then do ligation with T4 ligation enzyme to make piGEM2017-P01, piGEM2017-P02, piGEM2017-P03 and piGEM2017-P04.
1.9 Transform the ligation product into E.coli DH5α, Chl-plate.
8.29
1.1 Golden Gate Cloning for piGEM2017-022 & piGEM2017-023.
1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
2.1 Positive detection of plate piGEM2017-P01, piGEM2017-P02, piGEM2017-P03 and piGEM2017-P04. After Gel Electrophoresis, get target bands we expect.
2.2 Shake bacteria fluid transformed with piGEM2017-P01, piGEM2017-P02, piGEM2017-P03 and piGEM2017-P04 on the shaking table 37℃, 180r, 18h.
8.30
1.1 HheC assay and no production.
8.31
1.1 Positive detection of plate piGEM2017-022 & piGEM2017-023. After Gel Electrophoresis, get target bands we expect.
1.2 Shake bacteria fluid transformed with piGEM2017-022 & piGEM2017-023 on the shaking table 37℃, 180r, 18h.
2.1 Conservation for the bacteria fluid of piGEM2017-P01, piGEM2017-P02, piGEM2017-P03 and piGEM2017-P04 that were shook, then use AXYGEN kit to do extraction. P01, P02 and P04 were successfully sequenced.
In September
9.2
1.1 Golden Gate Cloning for piGEM2017-022 & piGEM2017-023.
1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
2.1 Conservation for the bacteria fluid of piGEM2017-022 that was shook, then use AXYGEN kit to do extraction. But the restriction enzyme digestion was wrong.
9.3
1.1 Positive detection of plate piGEM2017-022 & piGEM2017-023. After Gel Electrophoresis, get target bands we expect.
1.2 Shake bacteria fluid transformed with piGEM2017-022 & piGEM2017-023 on the shaking table 37℃, 180r, 18h.
9.5
1.1 EchA activity assayed by the same method with piGEM2017-003. Result showed EchA could work successfully in transgenic tobacco.
9.7
1.1 Polymerase Chain Reaction.
Designation
Primer-F
Primer-R
Templation
Size
AOS-EchA
LBL031-F
LBL025-R
pGE01.3A-EchA
1041bp
1.2 Gel Electrophoresis, and get the target band we expect.
1.3 Extraction of fragments from Agarose Gel.
9.8
1.1 Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pZHZ079-cut
BglⅡ
NcoⅠ
pZHZ079
1.3 Use AXYGEN enzymatic reaction kit to do DNA cleanup.
9.9
1.1 Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pZHZ079-cut
BglⅡ
NcoⅠ
pZHZ079
1.3 Use AXYGEN enzymatic reaction kit to do DNA cleanup.
9.9
1.1 Mix AOS-EchA with backbone of pZHZ079, then do ligation with Gibson ligation enzyme to make pGE03A-EchA.
1.2 Transform the ligation product into E.coli DH5α, Amp-plate.
9.10
1.1 Positive detection of plate pGE03A-EchA. After Gel Electrophoresis, get target bands we expect.
1.2 Shake bacteria fluid transformed with pGE03A-EchA on the shaking table 37℃, 180r, 18h.
2.1 Mix AOS-EchA with backbone of pZHZ079, then do ligation with Gibson ligation enzyme to make pGE03A-EchA.
2.2 Transform the ligation product into E.coli DH5α, Amp-plate.
3.1 Repetition of DhaA31 activity assay with piGEM2017-001-3, piGEM2017-001-4 and piGEM2017-001-10
9.11
1.1 Conservation for the bacteria fluid of pGE03A-EchA that was shook, then use AXYGEN kit to do extraction. (successfully sequenced)
9.12
1.1 Golden Gate Cloning for piGEM2017-022 & piGEM2017-023 & piGEM2017-026 & piGEM2017-027.
1.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
9.13
1.1 Positive detection of plate piGEM2017-022 & piGEM2017-027. After Gel Electrophoresis, get target bands we expect.
1.2 Shake bacteria fluid transformed with piGEM2017-022 & piGEM2017-027 on the shaking table 37℃, 180r, 18h.
9.14
1.1 Positive detection of plate piGEM2017-023 & piGEM2017-026. After Gel Electrophoresis, get target bands we expect.
1.2 Shake bacteria fluid transformed with piGEM2017-023 & piGEM2017-026 on the shaking table 37℃, 180r, 18h.
9.15
1.1 Conservation for the bacteria fluid of piGEM2017-026 & piGEM2017-027 that was shook, then use AXYGEN kit to do extraction. The concentration of piGEM2017-026 was too low.
1.2 Restriction Enzyme Digest for verification .
Designation
Enzyme-A
Enzyme-B
Size
piGEM2017-027
SalⅠ
EcoRⅠ
1446bp、2500bp、10086bp
1.3 Gel Electrophoresis, and get the target band we expect.
1.4 piGEM2017-027 was successfully sequenced.
2.1 Shake bacteria fluid transformed with piGEM2017-026 on the shaking table 37℃, 180r, 18h.
9.16
1.1 Positive detection of plate piGEM2017-022. After Gel Electrophoresis, get target bands we expect.
1.2 Shake bacteria fluid transformed with piGEM2017-022 on the shaking table 37℃, 180r, 18h.
2.1 Polymerase Chain Reaction
Designation
Primer-F
Primer-R
Templation
Size
piGEM2017-P05-1
iGEMpart005-F
iGEMpart005-RM1
piGEM2017-007
509bp
piGEM2017-P05-2
iGEMpart005-FM1
iGEMpart005-RM2
piGEM2017-007
697bp
piGEM2017-P05-3
iGEMpart005-FM2
iGEMpart005-R
piGEM2017-007
384bp
2.2 Gel Electrophoresis, and get the target band we expect.
2.3 Extraction of fragments from Agarose Gel.
3.1 Conservation for the bacteria fluid of piGEM2017-022 & piGEM2017-023 & piGEM2017-026 & piGEM2017-027 that was shook, then use AXYGEN kit to do extraction. The concentration of piGEM2017-023 and piGEM2017-026 was too low.
3.2 Shake bacteria fluid transformed piGEM2017-023 and piGEM2017-026 on the shaking table 37℃, 180r, 18h.
9.17
1.1 Fusion PCR
Designation
Primer-F
Primer-R
Templation
Size
piGEM2017-P03F
iGEMpart005-F
iGEMpart005-R
piGEM2017-P05-1
piGEM2017-P05-2
piGEM2017-P05-3
1500bp
1.2 Gel Electrophoresis, and get the target band we expect
1.3 Extraction of fragments from Agarose Gel.
2.1 Restriction Enzyme Digest for verification .
Designation
Enzyme-A
Enzyme-B
Size
piGEM2017-022
SalⅠ
EcoRⅠ
1446bp、2500bp、9635bp
piGEM2017-023
SalⅠ
EcoRⅠ
1446bp、2307bp、9232bp
piGEM2017-026
SalⅠ
EcoRⅠ
1446bp、2307bp、9683bp
2.2 Gel Electrophoresis, and get the target band of 022 and 026 we expect.
9.18
1.1 Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pSB1C3-cut
PstⅠ
EcoRⅠ
pSB1C3
piGEM2017-P05F-cut
PstⅠ
EcoRⅠ
piGEM2017-P05F
1.2 Mix the digested fragment with backbone of pSB1C3, then do ligation with T4 ligation enzyme to make piGEM2017-P05.
1.3 Transform the ligation product into E.coli DH5α, Chl-plate.
2.1 Positive detection of plate piGEM2017-023. After Gel Electrophoresis, get target bands we expect.
2.2 Shake bacteria fluid transformed with piGEM2017-023 on the shaking table 37℃, 180r, 18h.
3.1 Conservation for the bacteria fluid of piGEM2017-022 & piGEM2017-023 & piGEM2017-026 that were shook, then use AXYGEN kit to do extraction.
4.1 Shake bacteria fluid transformed piGEM2017-023 on the shaking table 37℃, 180r, 18h.
5.1 Repetition of EchA activity assay with piGEM2017-003-1, piGEM2017-003-5 and piGEM2017-003-8.
9.20
1.1 Positive detection of plate piGEM2017-P05. After Gel Electrophoresis, didn’t get target bands we expect.
9.21
1.1 Mix the digested fragment of piGEM2017-P05F-cut with backbone of pSB1C3, then do ligation with T4 ligation enzyme to make piGEM2017-P05.
1.2 Transform the ligation product into E.coli DH5α, Chl-plate.
2.1 Conservation for the bacteria fluid of piGEM2017-022 & piGEM2017-023 & piGEM2017-026 that were shook, then use AXYGEN kit to do extraction. But they were mistaken.
2.2 Shake bacteria fluid transformed piGEM2017-023 on the shaking table 37℃, 180r, 18h.
3.1 Golden Gate Cloning for piGEM2017-023 & piGEM2017-026.
3.2 Transform the Golden Gate Cloning product into E.coli DH5α, Kan-plate.
9.22
1.1 Positive detection of plate piGEM2017-P05. After Gel Electrophoresis, get target bands we expect.
1.2 Shake bacteria fluid transformed piGEM2017-P05 on the shaking table 37℃, 180r, 18h.
2.1 Conservation for the bacteria fluid of piGEM2017-023 that was shook, then use AXYGEN kit to do extraction.
9.24
1.1 Positive detection of plate piGEM2017-023 & piGEM2017-026. After Gel Electrophoresis, get target bands we expect.
1.2 Shake bacteria fluid transformed piGEM2017-023 & piGEM2017-026 on the shaking table 37℃, 180r, 18h.
2.1 Conservation for the bacteria fluid of piGEM2017-P05 that was shook, then use AXYGEN kit to do extraction. (successfully sequenced)
9.26
1.1 Conservation for the bacteria fluid of piGEM2017-023 & piGEM2017-026 that were shook, then use AXYGEN kit to do extraction. (successfully sequenced)
1.2 Restriction Enzyme Digest for verification.
Designation
Enzyme-A
Enzyme-B
Size
piGEM2017-023
BamHⅠ
EcoRⅠ
1446bp、2291bp、9148bp
piGEM2017-026
BamHⅠ
EcoRⅠ
1446bp、2291bp、9699bp
1.3 Gel Electrophoresis, and get the target band we expect.
9.28
1.1 Add the HheC produced by E.coli into the supernatant of tobacco leaves and Tris-SO4 buffer to compare the enzyme activity. Results proved HheC can’t be degrade by protease in tobacco.
9.30
1.1 Polymerase Chain Reaction
Designation
Primer-F
Primer-R
Templation
Size
piGEM2017mutant-001-1
mutant-F
P84A-R
HheC-WT
355bp
piGEM2017mutant-001-2
P84A-F
mutant-R
HheC-WT
584bp
piGEM2017mutant-002-1
F12Q-F
F186L-R
HheC-WT
580bp
piGEM2017mutant-002-2
F186L-F
mutant-R
HheC-WT
282bp
piGEM2017mutant-003-1
mutant-F
P84A-R
HheC-W249P
355bp
piGEM2017mutant-003-2
P84A-F
mutant-R
HheC-W249P
584bp
piGEM2017mutant-004F
F12Q-F
mutant-R
HheC-W249P
825bp
piGEM2017mutant-005-1
mutant-F
F186L-R
HheC-W249P
F186L-R
piGEM2017mutant-005-2
F186L-F
mutant-R
HheC-W249P
282bp
piGEM2017mutant-006-1
F12Q-F
F186L-R
HheC-W249P
580bp
piGEM2017mutant-006-2
F186L-F
mutant-R
HheC-W249P
282bp
1.2 Gel Electrophoresis, and get the target band we expect.
1.3 Extraction of fragments from Agarose Gel.
In October
10.1
1.1 Fusion PCR
Designation
Primer-F
Primer-R
Templation
Size
piGEM2017mutant-001F
mutant-F
mutant-R
piGEM2017mutant-001-1
piGEM2017mutant-001-2
903bp
piGEM2017mutant-002F
F12Q-F
mutant-R
piGEM2017mutant-002-1
piGEM2017mutant-002-2
825bp
piGEM2017mutant-003F
mutant-F
mutant-R
piGEM2017mutant-003-1
piGEM2017mutant-003-2
903bp
piGEM2017mutant-005F
mutant-F
mutant-R
piGEM2017mutant-005-1
piGEM2017mutant-005-2
903bp
piGEM2017mutant-006F
F12Q-F
mutant-R
piGEM2017mutant-006-1
piGEM2017mutant-006-2
825bp
1.2 Gel Electrophoresis, and get the target band we expect
1.3 Extraction of fragments from Agarose Gel.
1.4 Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
HheC-WT -cut
PstⅠ
BamHⅠ/ NcoⅠ
HheC-WT
piGEM2017mutant-001F-cut
PstⅠ
BamHⅠ
piGEM2017mutant-001F
piGEM2017mutant-002F-cut
PstⅠ
NcoⅠ
piGEM2017mutant-002F
piGEM2017mutant-003F-cut
PstⅠ
BamHⅠ
piGEM2017mutant-003F
piGEM2017mutant-0041F-cut
PstⅠ
NcoⅠ
piGEM2017mutant-004F
piGEM2017mutant-005F-cut
PstⅠ
BamHⅠ
piGEM2017mutant-005F
piGEM2017mutant-006F-cut
PstⅠ
NcoⅠ
piGEM2017mutant-006F
1.5 Use AXYGEN enzymatic reaction kit to do DNA cleanup..
2.1 Polymerase Chain Reaction
Designation
Primer-F
Primer-R
Templation
Size
S1.0-1
mutant-F
84-R
HheC-WT
355bp
S1.0-2
84-F
mutant-R
HheC-WT
584bp
S2.0-1
12-F
186-R
HheC-WT
580bp
S2.0-2
186-F
mutant-R
HheC-WT
282bp
S3.0-1
mutant-F
186-R
HheC-W249P
658bp
S3.0-2
F186L-F
mutant-R
HheC-W249P
282bp
2.2 Gel Electrophoresis, and get the target band we expect.
2.3 Extraction of fragments from Agarose Gel.
10.2
1.1 Mix the digested fragment of piGEM2017mutant-001F-cut & piGEM2017mutant-002F-cut & piGEM2017mutant-003F-cut & piGEM2017mutant-004F-cut & piGEM2017mutant-005F-cut & piGEM2017mutant-006F-cut with backbone of HheC-WT -cut, then do ligation with T4 ligation enzyme to make piGEM2017-P05.
1.2 Transform the ligation product into E.coli DH5α, Amp-plate.
2.1 Fusion PCR
Designation
Primer-F
Primer-R
Templation
Size
S1.0F
mutant-F
mutant-R
S1.0-1 S1.0-2
903bp
S2.0F
12-F
mutant-R
S2.0-1 S2.0-2
825bp
S3.0F
mutant-F
mutant-R
S3.0-1 S3.0-2
903bp
2.2 Gel Electrophoresis, and get the target band we expect
2.3 Extraction of fragments from Agarose Gel.
10.3
1.1 Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
HheC-W249P -cut
PstⅠ
BamHⅠ/ NcoⅠ
HheC-W249P
S1.0F -cut
PstⅠ
BamHⅠ
S1.0F
S2.0F -cut
PstⅠ
NcoⅠ
S2.0F
S3.0F -cut
PstⅠ
BamHⅠ
S3.0F
1.2 Use AXYGEN enzymatic reaction kit to do DNA cleanup.
1.3 Mix the digested fragment with backbone of HheC-W249P, then do ligation with T4 ligation enzyme to make S1.0 & S2.0 & S3.0.
1.4 Transform the ligation product into E.coli MC1061, Amp-plate.
2.1 Positive detection of plate piGEM2017mutant-001 & piGEM2017mutant-002 & piGEM2017mutant-003 & piGEM2017mutant-004 & piGEM2017mutant-005 & piGEM2017mutant-006. After Gel Electrophoresis, get target bands we expect.
2.2 Shake bacteria fluid transformed piGEM2017mutant-001 to piGEM2017mutant-006 on the shaking table 37℃, 180r, 18h.
10.5
1.1 Conservation for the bacteria fluid of piGEM2017mutant-001 to piGEM2017mutant-006 that were shook, then use AXYGEN kit to do extraction. (successfully sequenced)
10.6
1.1 Transform the ligation product of S1.0 & S2.0 & S3.0 into E.coli MC1061, Amp-plate.
10.7
1.1 Positive detection of plate S1.0 & S2.0 & S3.0. After Gel Electrophoresis, get target bands we expect.
1.2 Shake bacteria fluid transformed S1.0 & S2.0 & S3.0 on the shaking table 37℃, 180r, 18h.
10.10
1.1 Mix the digested fragment of S1.0F –cut & S2.0F –cut & S3.0F -cut with backbone of HheC-W249P, then do ligation with T4 ligation enzyme to make S1.0 & S2.0 & S3.0.
1.2 Transform the ligation product into E.coli MC1061, Amp-plate.
2.1 Detect the multienzyme conversion of TCP in piGEM2017-004 with HheC produced by recombinant E.coli.
10.11
1.1 Polymerase Chain Reaction
Designation
Primer-F
Primer-R
Templation
Size
piGEM2017-P06F
iGEMpart006-F
iGEMpart006-R
piGEM2017mutant-001
814bp
piGEM2017-P07F
iGEMpart006-F
iGEMpart006-R
piGEM2017mutant-002
814bp
piGEM2017-P08F
iGEMpart006-F
iGEMpart008-R
piGEM2017mutant-003
814bp
piGEM2017-P09F
iGEMpart006-F
iGEMpart008-R
piGEM2017mutant-004
814bp
piGEM2017-P10F
iGEMpart006-F
iGEMpart008-R
piGEM2017mutant-005
814bp
piGEM2017-P11F
iGEMpart006-F
iGEMpart008-R
piGEM2017mutant-006
814bp
piGEM2017-P12F
iGEMpart006-F
iGEMpart006-R
piGEM2017mutant-001
814bp
1.2 Gel Electrophoresis, and get the target band we expect.
1.3 Extraction of fragments from Agarose Gel.
10.12
1.1 Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
pSB1C3 -cut
PstⅠ
EcoRⅠ
pSB1C3
piGEM2017-P06F-cut
PstⅠ
EcoRⅠ
piGEM2017-P06F
piGEM2017-P07F-cut
PstⅠ
EcoRⅠ
piGEM2017-P07F
piGEM2017-P08F-cut
PstⅠ
EcoRⅠ
piGEM2017-P08F
piGEM2017-P09F-cut
PstⅠ
EcoRⅠ
piGEM2017-P09F
piGEM2017-P10F-cut
PstⅠ
EcoRⅠ
piGEM2017-P10F
piGEM2017-P11F-cut
PstⅠ
EcoRⅠ
piGEM2017-P11F
piGEM2017-P12F-cut
PstⅠ
EcoRⅠ
piGEM2017-P12F
1.2 Use AXYGEN enzymatic reaction kit to do DNA cleanup.
10.13
1.1 Mix the digested fragment with backbone of pSB1C3, then do ligation with T4 ligation enzyme to make piGEM2017-P06 & piGEM2017-P07 & piGEM2017-P08 & piGEM2017-P09 & piGEM2017-P10 & piGEM2017-P11 & piGEM2017-P12.
1.2 Transform the ligation product into E.coli DH5α, Chl-plate.
10.14
1.1 Positive detection of plate piGEM2017-P06 & piGEM2017-P07 & piGEM2017-P08 & piGEM2017-P09 & piGEM2017-P10 & piGEM2017-P11 & piGEM2017-P12. After Gel Electrophoresis, get target bands we expect.
1.2 Shake bacteria fluid transformed piGEM2017-P06 & piGEM2017-P07 & piGEM2017-P08 & piGEM2017-P09 & piGEM2017-P10 & piGEM2017-P11 & piGEM2017-P12 on the shaking table 37℃, 180r, 18h.
10.16
1.1 Conservation for the bacteria fluid of piGEM2017-P06 & piGEM2017-P07 & piGEM2017-P08 & piGEM2017-P09 & piGEM2017-P10 & piGEM2017-P11 & piGEM2017-P12 that were shook, then use AXYGEN kit to do extraction. (successfully sequenced)
10.18
1.1 Polymerase Chain Reaction
Designation
Primer-F
Primer-R
Templation
Size
piGEM2017mutant-007*-1
mutant-F
DhaA135-R
DhaA31
518bp
piGEM2017mutant-007*-2
DhaA135-F
DhaA245-R
DhaA31
367bp
piGEM2017mutant-007*-3
DhaA245-F
mutant-R
DhaA31
246bp
1.2 Gel Electrophoresis, and get the target band we expect.
1.3 Extraction of fragments from Agarose Gel.
1.4 Fusion PCR
Designation
Primer-F
Primer-R
Templation
Size
piGEM2017mutant-007*F
mutant-F
mutant-R
piGEM2017mutant-007*-1
piGEM2017mutant-007*-2
piGEM2017mutant-007*-3
1055bp
1.5 Gel Electrophoresis, and get the target band we expect
1.6 Extraction of fragments from Agarose Gel.
10.20
1.1 Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
DhaA31 -cut
PstⅠ
BamHⅠ
DhaA31
piGEM2017mutant-007*F -cut
PstⅠ
BamHⅠ
piGEM2017mutant-007*F
1.2 Gel Electrophoresis, and get the target band we expect
1.3 Extraction of fragments from Agarose Gel.
1.4 Mix the digested fragment with backbone of DhaA31 then do ligation with T4 ligation enzyme to make piGEM2017mutant-007*.
1.5 Transform the ligation product into E.coli DH5α, Amp-plate.
10.21
1.1 Positive detection of plate piGEM2017mutant-007*. After Gel Electrophoresis, get target bands we expect.
1.2 Shake bacteria fluid transformed piGEM2017mutant-007* on the shaking table 37℃, 180r, 18h.
1.3 We measured the standard curve of glycerol by the method of GC with a DB-5 column (Supelco, Bellefonate, PA, USA).
10.22
1.1 Conservation for the bacteria fluid of piGEM2017mutant-007* that was shook, then use AXYGEN kit to do extraction.
2.1 Polymerase Chain Reaction
Designation
Primer-F
Primer-R
Templation
Size
piGEM2017mutant-007-1
mutant-F
DhaA176-R
piGEM2017mutant-007*
640bp
piGEM2017mutant-007-2
DhaA176-F
DhaA273-R
piGEM2017mutant-007*
322bp
piGEM2017mutant-007-3
DhaA273-F
mutant-R
piGEM2017mutant-007*
158bp
2.2 Gel Electrophoresis, and get the target band we expect.
2.3 Extraction of fragments from Agarose Gel.
2.4 Glycerol was detected in our reaction mixture, and result shows the wild type tobacco couldn’t convert 1,2,3-TCP into GLY, but our “super tobacco”did.
10.23
1.1 Fusion PCR
Designation
Primer-F
Primer-R
Templation
Size
piGEM2017mutant-007F
mutant-F
mutant-R
piGEM2017mutant-007-1
piGEM2017mutant-007-2
piGEM2017mutant-007-3
1055bp
1.2 Gel Electrophoresis, and get the target band we expect
1.3 Extraction of fragments from Agarose Gel.
1.4 Restriction Enzyme Digest.
Designation
Enzyme-A
Enzyme-B
Templation
piGEM2017mutant-007F -cut
PstⅠ
BamHⅠ
piGEM2017mutant-007F
1.5 Use AXYGEN enzymatic reaction kit to do DNA cleanup.
1.6 Mix the digested fragment with backbone of DhaA31 then do ligation with T4 ligation enzyme to make piGEM2017mutant-007.
1.7 Transform the ligation product into E.coli DH5α and MC1061, Amp-plate.
10.24
1.1 Restriction Enzyme Digest.
Plasmids
Group 1
Group 2
piGEM2017-001
PstⅠ、BamHⅠ
ScaⅠ
piGEM2017-002
SacⅡ、BamHⅠ
SacⅡ、PstⅠ
piGEM2017-003
EcoNⅠ、BamHⅠ
ScaⅠ
piGEM2017-004
EcoNⅠ、BamHⅠ
PstⅠ、BamHⅠ
piGEM2017-005
EcoNⅠ、BamHⅠ
ScaⅠ
piGEM2017-021
PstⅠ
EcoRⅠ、BamHⅠ
piGEM2017-022
PstⅠ
EcoRⅠ、BamHⅠ
piGEM2017-023
PstⅠ
EcoRⅠ、BamHⅠ
piGEM2017-024
PstⅠ
EcoRⅠ、BamHⅠ
piGEM2017-025
PstⅠ
EcoRⅠ、BamHⅠ
piGEM2017-026
PstⅠ
EcoRⅠ、BamHⅠ
piGEM2017-027
PstⅠ
EcoRⅠ、BamHⅠ
1.2 Gel Electrophoresis
a. piGEM2017-001 b. piGEM2017-002 c. piGEM2017-003 d. piGEM2017-004
e. piGEM2017-005 f. piGEM2017-021 g. piGEM2017-022 h. piGEM2017-023
i. piGEM2017-024 j. piGEM2017-025 k. piGEM2017-026 l. piGEM2017-027
10.25
1.1 Detect the multienzyme conversion of TCP in piGEM2017-005 with HheC produced by recombinant E.coli.
10.25-11.1
During this time, we mainly resort out the experimental ideas and organize experimental results. Moreover, we improved the wiki various sections, practiced oral English, prepared presentation, made PPT, and performed the rehearsals for many times. We deeply appreciated the joy of team life. We did everything possible to prepare for the final plea, and believed that we would achieve satisfactory grades.
In November
11.7
We set out for Boston! Fighting!
