<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UESTC-China_result skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UESTC-China/result</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:UESTC-China/Result</TITLE></P><DIV class="head-v3"><DIV class="nav-up"><DIV class="nav-inner"><DIV class="nav-v3"><UL><LI class="" _t_nav="Achievement"><H2><A href="https://2017.igem.org/Team:UESTC-China/achievement">Achievement</A></H2></LI><LI class="" _t_nav="Project"><H2><A href="javascript:void(0);">Project ∨</A></H2></LI><LI class="Modeling" _t_nav="Modeling"><H2><A href="https://2017.igem.org/Team:UESTC-China/modeling">Modeling ∨</A></H2></LI><LI class="" _t_nav="Attribution"><H2><A href="https://2017.igem.org/Team:UESTC-China/attribution">Attribution</A></H2></LI><LI class="" _t_nav="HumanPractice"><H2><A href="javascript:void(0);">Human Practice ∨</A></H2></LI><LI class="" _t_nav="Team"><H2><A href="javascript:void(0);">Team ∨</A></H2></LI><LI class="" _t_nav="Notebook"><H2><A href="javascript:void(0);">Notebook ∨</A></H2></LI></UL></DIV></DIV></DIV><DIV class="nav-down"><DIV id="Project" class="nav-down-menu" style="display: none;" _t_nav="Project"><DIV class="nav-down-inner"><DL style="margin-right: 702px;"><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/part">Part</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/result">Result</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/design">Design</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/description">Description</A></DD></DL></DIV></DIV><DIV id="Modeling" class="nav-down-menu" style="display: none;" _t_nav="Modeling"><DIV class="nav-down-inner"><DL style="margin-right: 565px;"><DD><A href="https://2017.igem.org/Team:UESTC-China/overview">Overview</A></DD></DL><DL><DD><A href="https://2017.igem.org/Team:UESTC-China/model-set">Model Set</A></DD></DL><DL><DD><A href="https://2017.igem.org/Team:UESTC-China/docking-simulation">Docking Simulation</A></DD></DL><DL><DD><A href="https://2017.igem.org/Team:UESTC-China/virtual-mutagenesis">Virtual Mutagenesis</A></DD></DL><DL><DD><A href="https://2017.igem.org/Team:UESTC-China/experiment-demonstrate">Experiment Demonstrate</A></DD></DL></DIV></DIV><DIV id="HumanPractice" class="nav-down-menu" style="display: none;" _t_nav="HumanPractice"><DIV class="nav-down-inner"><DL style="margin-right: 270px;"><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/hp-collaboration">Collaboration</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/engagement">Engagement</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/research">Research</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/our-story">Our Story</A></DD></DL></DIV></DIV><DIV id="Team" class="nav-down-menu" style="display: none;" _t_nav="Team"><DIV class="nav-down-inner"><DL style="margin-right: 128px;"><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/collaboration">Collaboration</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/team-introduce">Team Introduce</A></DD></DL></DIV></DIV><DIV id="Notebook" class="nav-down-menu" style="display: none;" _t_nav="Notebook"><DIV class="nav-down-inner"><DL style="margin-right: 37px;"><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/interlab">Interlab</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/safety">Safety</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/protocal">Protocal</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/day-note">Day Note</A></DD></DL></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><NAV class="col-lg-2" id="myScrollspy"><UL class="nav nav-pills nav-stacked" data-spy="affix" data-offset-top="400"><LI class="active"><A href="#Plasmid-Construct">Plasmid Construct</A></LI><LI><A href="#Tobacco" style="font-size: 14px;">Tobacco Transformation</A></LI><LI><A href="#Detection-of-Plants">Detection of Plants</A></LI><LI><A href="#GUS-staining">GUS staining</A></LI><LI><A href="#Gas-chromatography">Gas chromatography</A></LI><LI><A href="#气相色谱">气相色谱</A></LI><LI><A href="#水培及土培">水培及土培</A></LI><LI><A href="#Work-Going-On">Work Going On</A></LI></UL></NAV><DIV class="col-lg-10"><DIV id="Plasmid-Construct"><H2>1.Plasmid Construct</H2><P class="subHead"><B> Vector construction for the expression of DhaA31, HheC-W249P, EchA in tobacco</B></P><P>In order to express the three enzymes (DhaA31 isolated from <I>Rhodococcus rhodochrous</I> , HheC-W249P and EchA isolated from <I>Agrobacterium radiobacter</I> AD1) in tobacco, we firstly did the codon-optimized of the three enzymes in dicotyledonous plants. And then the genes of DhaA31, HheC-W249P and EchA were commercially synthesized. Finally, with the application of 2A peptide and Golden Gate strategy, we successfully introduced DhaA31, HheC-W249P and EchA into the plant expression vector piGEM2017-backbone which contained a kanamycin resistance gene (NptII) (Fig 1). The expression of these three enzymes were driven by the common constitutive promoter pCaMV35s in plants.</P><P class="mid">Figure 1. The introduction of piGEM2016-001 to piGEM2016-007</P><P>Before DNA sequencing, those vectors were verified by restriction enzyme digestion. After electrophoresis analysis, the samples which contained all desired bands were selected and sent for sequencing. The sequencing results showed that all the above constructed vectors were successful (Fig 2).</P><P class="mid">Figure 2. The image of agarose gel electrophoresis by double enzyme digestion. </P><UL><LI>(a)piGEM2017-001 (Line1,enzyme digested by PstI+BamHI; Line2,enzyme digested by ScaI) </LI><LI>(b)piGEM2017-002 (Line1,enzyme digested by PstI+BamHI; Line2, enzyme digested by ScaI) </LI><LI>(c) piGEM2017-003 (Line1,enzyme digested by EcoNI+BamHI; Line2,enzyme digested by ScaI) </LI><LI>(d) piGEM2017-004 (Line 1, enzyme digested by EcoRI+BamHI; Line 2, enzyme digested by PstI+BamHI)</LI><LI>(e) piGEM2017-005 (Line 1, enzyme digested by EcoRI+BamHI; Line 2, enzyme digested by ScaI) </LI><LI>M: DNA marker</LI></UL><P class="subHead"><B> Vector construction for the root expression of DhaA31, HheC-W249P, EchA in tobacco</B></P><P>Considering that TCP mainly existed in the soil and groundwater, we chose the plant root specific expression promoter pYK10 from Arabidopsis thaliana to replace the constitutive promoter pCaMV35s in order to obtain better degradation efficiency of TCP by enriching these three enzymes at the root of the plants. </P><P>At the same time, in order to strengthen the root growth of tobacco, a gene that coded the expression of the cytokinin, CKX3, was also introduced into piGEM2017-backbone after the codon optimization of dicotyledonous plants (Fig 3).</P><P class="mid">Figure 3. The introduction of piGEM2017-021 and piGEM2017-024</P><P>Before DNA sequencing, those vectors were verified by restriction enzyme digestion. After electrophoresis analysis, the samples which contained all desired bands were selected and sent for sequencing. The sequencing results showed that all the above constructed vectors were successful (Fig 4).</P><P class="mid">Figure 4. The image of agarose gel electrophoresis by double enzyme digestion. </P><P class="mid">(a) piGEM2017-021 (b) piGEM2017-024 </P><P class="mid">(M: DNA marker; Line 1, enzyme digested by PstI; Line 2, enzyme digested by EcoRI +BamHI)</P><P class="subHead"><B> Vector construction for the extracellular expression of DhaA31, HheC-W249P, EchA in tobacco</B></P><P>In order to trpansport the three enzymes out of tobacco to play their functions, we chose AO-S, a plant cell wall localization peptide, to help achieve this goal. Besides, AO-S can also stabilize the expression of the proteins. Based on this, the vectors which can fulfil the fusion expression of AO-S and the three enzymes were constructed. </P><P>Finally, in order to intuitively confirm whether the multi-gene plant expression vectors we designed can play their roles, GUS reporter gene was also introduced into the piGEM2017-backbone (Fig 5).</P><P class="mid">Figure 5. The introduction of piGEM2017-022 023 025 026 and 027</P><P>Before DNA sequencing, those vectors were verified by restriction enzyme digestion. After electrophoresis analysis, the samples which contained all desired bands were selected and sent for sequencing. The sequencing results showed that all the above constructed vectors were successful (Fig 6). </P><P class="mid">Figure 6. The image of agarose gel electrophoresis by double enzyme digestion. </P><P class="mid">a) piGEM2017-022 (b) piGEM2017-023 (c) piGEM2017-025 (d) piGEM2017-026 (e) piGEM2017-027 </P><P class="mid">(M: DNA marker; Line 1, enzyme digested by PstI; Line 2, enzyme digested by EcoRI +BamHI)</P></DIV><DIV id="Tobacco"><H2>2.Tobacco Transformation</H2><P>In order to introduce foreign genes into tobacco and achieve the stable expression of the exogenous proteins in tobacco, <I>Agrobacterium</I> mediated transformation which was a common transgenic method for dicotyledonous plants was used. </P><P>We firstly transferred our plasmids into <I>Agrobacterium</I> EHA105 and then the T-DNA fragment which contained the target genes of the <I>Agrobacterium</I> EHA105 was integrated into the chromosome of tobacco through the <I>Agrobacterium</I> mediated transformation.</P><P>For obtaining the transgenic tobacco, the whole steps can be divided into the following eight parts. Seed germination, pre-culture, infection, co-culture, resistance screening, calli induction, root culture, transplanting. The cycle of obtaining a transgenic tobacco was about ten weeks (Fig 7).</P><P class="mid">Figure 7. The steps for obtaining transgenic tobacco</P></DIV><DIV id="Detection-of-Plants"><H2>3.PCR and RT - PCR Detection of Transgenic Plants</H2><P class="subHead"><B> PCR detection of transgenic plants</B></P><P>After <I>Agrobacterium</I> mediated transformation, we successfully obtained the callis which have the resistance of kanamycin from the leaves of tobacco. The callis began to sprout in about 3 weeks. Another 3 weeks later, we obtained several transgenic tobacco. In order to verify whether the transgenic tobacco was false positive or not, we extracted the genomic DNA of T0 generation tobacco and designed pairs of primers to do PCR amplification of DhaA31, HheC-W249P, EchA, CKX3. Compared with wild type, we successfully amplified the target bands in T0 transgenic tobacco, which indicated that our multi-gene expression system worked successfully and we obtained the positive transgenic tobacco (Fig 8).</P><P class="mid">Figure 8. The image of agarose gel electrophoresis by PCR detection</P><P class="mid">a.piGEM2017-001 for Dha31A b.piGEM2017-002 for HheC-W249P c.piGEM2017-003 for EchA</P><P class="mid">d.piGEM2017-004 for Dha31A e.piGEM2017-004 for EchA f.piGEM2017-005 for Dha31A g.piGEM2017-005 for HheC</P><P class="mid">(M: DL2000 DNA marker; Line 1, positive control (plasmid)，Line 2, negative control (wild-type tobacco)</P><P class="subHead"><B> RT-PCR detection of transgenic plants</B></P><P>In order to confirm whether DhaA31, HheC-W249P, EchA and CKX3 can be normally transcribed in tobacco, and determine the transcription level of these genes in tobacco, we extracted the RNA from the leaves and roots of the transgenic tobacco. And then RT- PCR was used to detect. Actin, a common used endogenous gene of the tobacco was selected as internal reference. Compared with the wild type, the cDNA of transgenic tobacco amplified the corresponding band of target genes, indicating that the above four exogenous genes were successfully transcribed in tobacco. At the same time, the cDNA of genes expressed by pYK10 promoter amplified a clearly band in root tissue than in leaves or other tissues, indicating that pYK10 promoter successfully drove DhaA31, HheC-W249P, EchA and CKX3 these four exogenous genes expressed in the root of tobacco (Fig 9).</P><P class="mid">Figure 9. The image of agarose gel electrophoresis by RT-PCR detection</P><P class="mid">a.piGEM2017-001 for Dha31A    b.piGEM2017-002 for HheC-W249P c.piGEM2017-003 for EchA</P><P class="mid" style="font-size: 14px !important;">(M: DL2000 DNA marker; Line 1, positive control (plasmid)，Line 2, negative control (wild-type tobacco, Tubulin was an internal control in tobacco )</P></DIV><DIV id="GUS-staining"><H2>4.GUS staining</H2><P>β-D - glucuronidase (encoding GUS gene) was an acidic hydrolase isolated from the E. coli K-12 strain that catalyzed the hydrolysis of many beta-glucoside esters. In the histochemical staining with X-gluc, the plants containing the GUS gene will appear blue. Based on this, transgenic tobacco (piGEM2017-02X) which had GUS gene and wild type tobacco were identified after the histochemical staining with X-gluc for 24h at the same time. Compared with wild type tocacco, the transgenic tobacco was dyed blue, proved that the multi-gene expression system in tobacco worked as we expected again. Besides, the expression of GUS drove by pYK10 observed dark blue in roots while in leaves and other tissues was significantly weaker compared with pCaMV35s after the histochemical staining with X-gluc for 24h. It was proved that pYK10 could achieve the root-specific expression of genes (Fig 10).</P></DIV><DIV id="Gas-chromatography"><H2>5.Enzyme activity assay by gas chromatography</H2><P>In order to detect that the transgenic tobacco can work properly, the concentration of the reaction products was measured by gas chromatography (GC) as described in our protocol. Gas chromatograph Network GC System (Fuli 9750, China) equipped with a capillary column ZB-FFAP 30 m x 0.25 mm x 0.25 µm (Phenomenex, USA) was used to the quantitative analyses of all metabolites of the TCP degradation pathway except for GLY. The concentration of GLY was determined by GC (Fuli 9790) equipped with a DB-5 column (L60 m×i.d. 0.25 mm and df 0.25μm thin coating film) (Supelco, Bellefonate, PA, USA).</P><P class="subHead"><B> Standard curves of reaction products</B></P><P>For quantitative assay, standard curves of 1,2,3-TCP, 2,3-DCP, ECH, CPD, GDL, GLY ranging from 0-5 mM in 200 mM of Tris-SO4 buffer(pH 8.5) was measured (Fig 11).</P><P class="mid">Figure 11. Standard curves of reaction products</P><P class="subHead"><B> Enzyme assay in vitro</B></P><P>For piGEM2017-001 to piGEM2017-005, all the enzymes that introduced into tobacco were driven by the common constitutive promoter pCaMV35S, so we could extract enzymes from leaves to detect their activities in vitro.</P><P>Leaves of a weight equal to 2 g were collected in the mortar and rinsed thoroughly in liquid nitrogen, added 6 mL buffer (200mM Tris-SO4, pH 8.5), rinsed thoroughly. The extraction was centrifuged at 12,000×g for 30 min at 4°C. The same amount of supernatant (6.5mL) were stored in gastight glass vials for the determination of enzymatic activity. The activity of three enzymes were assayed at 37 °C with 5 mM corresponding substrates. The samples (0.25 mL) withdrawn from the reaction mixture at selected time were analyzed by gas chromatography (GC) as described.</P><P class="subsubHead mid">1）Selection of sampling time</P><P>To explore which time could show the enzymatic activity apparently, piGEM2017-001 was chosen to do experiments. The samples were withdrawn from the reaction mixture every hour and we found that the conversion of TCP to DCP reached maximum in seven hours, so 0h ,4h ,7h were selected to show the activity of all three enzymes (Fig 12).</P><P class="mid" style="font-size: 13px !important;">Figure 12. Time courses of concentration of production 2,3-DCP with piGEM2017-001. Reactions were performed using different lines of piGEM2017-001.</P><P class="subsubHead mid">2）Enzyme assay of DhaA31</P><P>Haloalkane dehalogenase (DhaA31) can degrade 1,2,3-TCP to 2,3-DCP. The activity of DhaA31 could be detected with 5mM 1,2,3-TCP using the method described in protocol. As showed in Figure 13, piGEM2017-001 could convert 1,2,3-TCP to 2,3-DCP while wild-type tobacco couldn’t which proved that DhaA31 could work in tobacco successfully.</P><P class="mid">Figure 13. Time courses of concentration of production 2,3-DCP with piGEM2017-001.</P><P class="mid">Each data represents the mean value + standard deviation from three independent experiments.</P><P class="subsubHead mid">3）Activity assay of HheC-W249P</P><P>Haloalcohol dehalogenase (HheC) can degrade 2,3-DCP and CPD to ECH and GDL, respectively. So we detected its activity with 2,3-DCP and CPD in piGEM2017-002, respectively. While no ECH or GDL were detected at the selected time in wild-type or piGEM2017-002 (data were not shown).</P><P>We guess there were two possible reasons: 1. there was a kind of protease which could degrade haloalcohol dehalogenase in tobacco. 2. HheC is a tetramer, it can not appear to be present as tetramers because of the low expression, and therefore can not play its own function. For the first suspect, HheC (which was produced by recombinant E.coli) was added into supernatant of wild-type tobacco and Tris-SO4 buffer. As shown in figure3, comparing with Tris-SO4, the concentration of ECH in supernatant of tobacco was similar. It showed that HheC couldn’t be degraded by protease in tobacco. For the second respect, we will choose a stronger promoter to express HheC or use a mutant of HheC which has a higher enzyme activity. But time is limited, one possible solution was to add HheC produced by E.coli into supernatant to detect the whole pathway of TCP degradation (Fig 14).</P><P class="subsubHead mid">4）Activity assay of EchA</P><P>Epoxide hydrolase (EchA) can convert ECH and GDL to CPD, GLY, respectively，so we could detect the activity of EchA with 5mM ECH in its supernatant. Concentration of CPD at selected time was shown in Figure 15. We could easily know that EchA can work in tobacco successfully. Wildtype tobacco could also degrade ECH to CPD because of endogenous epoxide hydrolase, the same to what described in another paper (Characterization of a tobacco epoxide hydrolase gene induced during the resistance response to TMV).</P><P class="mid">Figure 15. Time courses of concentration of CPD with piGEM2017-003.</P><P class="mid">Each data represents the mean value + standard deviation from three independent experiments.</P><P class="subsubHead mid">5）Multi-enzymes degradation of TCP</P><P>DhaA31, HheC and EchA catalyze consecutive reactions converting toxic TCP to harmless glycerol. To test whether multienzyme conversion of TCP could work in tobacco successfully, we detected its with 5mM TCP in its supernatant. While because of HheC couldn’t work successfully, only 2,3-DCP of all intermediates was detected. We added HheC (produced by transgenic E.coli) in the reaction mixture. The conversion of TCP to GLY after 7 h reached 84% and conversion of all intermediates reached 89% of the theoretical maximum as shown in Figure 16.</P><P class="subsubHead mid">6） Multi-enzymes without EchA degradation of TCP</P><P>As described above, tobacco contains endogenous epoxide hydrolase so as shown in Figure 17, the efficiency of converting TCP to GLY by piGEM2017-005 was detected and the concentration of GLY reached 89% compared with piGEM2017-004.</P></DIV><DIV id="气相色谱"><H2>6.气相色谱</H2><P>为了证明三种酶在植物体内能正常表达，我们利用气相色谱法检测了三种酶的酶活。色谱柱ZB-FFAP 30 m x 0.25 mm x 0.25 Gm (Phenomenex, USA)能够检测到此三酶链式反应中5种反应物，因此在酶活检测中，能够得到底物随时间减少以及产物随时间增加的折线图，从而证明转基因植株正常工作。</P><P>我们首先在35S启动子的三种单基因植株中分别检测了三种酶的酶活，得到的产物生成曲线如下，这证明了三种单基因植株都能正常工作（？？？），为后来三基因植株及根部表达和其他优化的正常工作打好了基础。（转化效率的问题）</P><P>接下来我们进行双基因和三基因植株的酶活检测，在研磨液中加入底物TCP，通过气相色谱检测中间产物的变化，来确定转基因植株正常工作，得到的曲线如下。（待完善）</P><P>对于根部特异性启动子，由于只在根部表达相关酶，叶片研磨的方法不再适用，所以我们采用水培的方法。在含有一定量TCP的培养液中培养转基因植株，定期从培养液中取样，用气相色谱进行检测，得到了中间产物随时间变化的曲线，如下图。</P><P>最后，对于三酶降解TCP的终产物甘油，我们使用色谱柱DB-5 column (L60 m×i.d. 0.25 mm and df0.25 _m thin coating film) (Supelco,Bellefonate, PA, USA).进行专门的检测。对35S启动的三基因植株，我们使用叶片研磨的方法，最终在反应体系中检测到了甘油的生成，得到甘油随时间变化的曲线如图。</P><P>（注：每个株系各有三组重复，进行统计分析）</P></DIV><DIV id="水培及土培"><H2>7.水培及土培</H2></DIV><DIV id="Work-Going-On"><H2>8.Work Going On</H2></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>