<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UESTC-China_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UESTC-China/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:UESTC-China/Demonstrate - 2017.igem.org</TITLE></P><DIV class="head-v3"><DIV class="nav-up"><DIV class="nav-inner"><DIV class="nav-v3"><UL><LI class="" _t_nav="Achievement"><H2><A href="https://2017.igem.org/Team:UESTC-China/achievement">ACHIEVEMENT</A></H2></LI><LI class="" _t_nav="Project"><H2><A href="javascript:void(0);">PROJECT ∨</A></H2></LI><LI class="" _t_nav="Modeling"><H2><A href="https://2017.igem.org/Team:UESTC-China/Model">MODELING</A></H2></LI><LI class="" _t_nav="Attribution"><H2><A href="https://2017.igem.org/Team:UESTC-China/Attributions">ATTRIBUTIONS</A></H2></LI><LI class="" _t_nav="HumanPractice"><H2><A href="javascript:void(0);">HUMAN PRACTICE ∨</A></H2></LI><LI class="" _t_nav="Team"><H2><A href="javascript:void(0);">TEAM ∨</A></H2></LI><LI class="" _t_nav="Notebook"><H2><A href="javascript:void(0);">NOTEBOOK ∨</A></H2></LI></UL></DIV></DIV></DIV><DIV class="nav-down"><DIV id="Project" class="nav-down-menu" style="display: none;" _t_nav="Project"><DIV class="nav-down-inner"><DL style="margin-right: 670px;"><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/part">Part</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/Demonstrate">Demonstrate</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/design">Design</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/description">Introduction</A></DD></DL></DIV></DIV><DIV id="HumanPractice" class="nav-down-menu" style="display: none;" _t_nav="HumanPractice"><DIV class="nav-down-inner"><DL style="margin-right: 250px;"><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/Engagement">Engagement</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/research">Supporting Research</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/our-story">Our Story</A></DD></DL></DIV></DIV><DIV id="Team" class="nav-down-menu" style="display: none;" _t_nav="Team"><DIV class="nav-down-inner"><DL style="margin-right: 115px;"><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/Collaborations">Collaborations</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/team-introduce">Team Introduce</A></DD></DL></DIV></DIV><DIV id="Notebook" class="nav-down-menu" style="display: none;" _t_nav="Notebook"><DIV class="nav-down-inner"><DL style="margin-right: 36px;"><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/InterLab">Interlab</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/safety">Safety</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/protocal">Protocal</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/Plant">Plant</A></DD></DL><DL><DD><A class="link" href="https://2017.igem.org/Team:UESTC-China/day-note">Day Note</A></DD></DL></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><NAV class="col-lg-2" id="myScrollspy"><UL class="nav nav-pills nav-stacked" data-spy="affix" data-offset-top="474"><LI class="active"><A href="#Pathway" style="font-size: 14px;">Pathway construction</A></LI><LI><A href="#Tobacco" style="font-size: 13.5px;">Tobacco Transformation</A></LI><LI><A href="#Positive" style="font-size: 14px;">Positive validation</A></LI><LI><A href="#Construct" style="font-size: 14px;">Construct validation</A></LI><LI><A href="#function" style="font-size: 14px;">PYK10 and CKX3 </A></LI><LI><A href="#Enzyme" style="font-size: 13px;">Activity determination</A></LI><LI><A href="#Activity" style="font-size: 12px;">Detection of hydroponics</A></LI><LI><A href="#Work">Work going on</A></LI></UL></NAV><DIV class="col-lg-10" id="new-style"><DIV id="Pathway"><H2>Pathway construction for degrading 1,2,3-TCP </H2><P class="sub-title"> 1. Degrading pathway construction based on the expression of DhaA31, HheC-W249P &amp; EchA in tobacco </P><P>In order to express the three enzymes (DhaA31 isolated from <EM>Rhodococcus rhodochrous</EM>, HheC-W249P and EchA isolated from <EM>Agrobacterium radiobacter</EM> AD1) in tobacco, we firstly did the codon-optimized of the three enzymes in dicotyledonous plants. And then the genes of DhaA31, HheC-W249P and EchA were commercially synthesized. Finally, with the application of 2A peptide and Golden Gate strategy, we successfully introduced DhaA31, HheC-W249P and EchA into the plant expression vector piGEM2017-backbone which contained the resistance of kanamycin (NptII) (Fig. 1). The expression of these three enzymes were driven by the common constitutive promoter CaMV35s in plants.</P><P class="mid">Figure 1. The introduction of piGEM2016-001 to piGEM2016-005</P><P>Before DNA sequencing, those vectors were verified by restriction enzyme digestion. After electrophoresis analysis, the samples which contained all desired bands were selected and sent for sequencing. The sequencing results showed that all the above constructed vectors were successful (Fig. 2).</P><P class="mid">Figure 2. The image of agarose gel electrophoresis by double enzyme digestion. </P><P class="mid">(a)piGEM2017-001 (Line1,enzyme digested by PstI+BamHI; Line2,enzyme digested by ScaI) </P><P class="mid">(b)piGEM2017-002 (Line1,enzyme digested by PstI+BamHI; Line2, enzyme digested by ScaI) </P><P class="mid">(c) piGEM2017-003 (Line1,enzyme digested by EcoNI+BamHI; Line2,enzyme digested by ScaI) </P><P class="mid">(d) piGEM2017-004 (Line 1, enzyme digested by EcoRI+BamHI; Line 2, enzyme digested by PstI+BamHI)</P><P class="mid">(e) piGEM2017-005 (Line 1, enzyme digested by EcoRI+BamHI; Line 2, enzyme digested by ScaI) </P><P class="mid">M: DNA marker</P><P class="sub-title">2. Degrading pathway construction based on the root expression of DhaA31, HheC-W249P &amp; EchA in tobacco </P><P>Considering that 1,2,3-TCP mainly existed in the soil and groundwater, we chose the plant root specific expression promoter PYK10 from <EM>Arabidopsis thaliana</EM> to replace the constitutive promoter CaMV35s in order to obtain better degradation efficiency of 1,2,3-TCP by enriching these three enzymes at the root of the plants. </P><P>At the same time, in order to strengthen the root growth of tobacco, a gene that coded the expression of the cytokinin, CKX3, was also introduced into piGEM2017-backbone after the codon optimization of dicotyledonous plants (Fig. 3).</P><P class="mid">Figure 3. The introduction of piGEM2017-008,piGEM2017-009, piGEM2017-021 and piGEM2017-024</P><P>Before DNA sequencing, those vectors were verified by restriction enzyme digestion. After electrophoresis analysis, the samples which contained all desired bands were selected and sent for sequencing. The sequencing results showed that all the above constructed vectors were successful (Fig. 4). </P><P class="mid">Figure 4. The image of agarose gel electrophoresis by double enzyme digestion. </P><P class="mid"> (a) piGEM2017-008(Line 1, enzyme digested by EcoNI+BamHI; Line 2, enzyme digested by BgIⅡ +PstI) </P><P class="mid">(b) piGEM2017-009( Line 1, enzyme digested by SacⅡ+AvrⅡ; Line 2, enzyme digested by EcoNI +BgIⅡ ) </P><P class="mid">(c) piGEM2017-021( Line 1, enzyme digested by PstI; Line 2, enzyme digested by EcoRI +BamHI)</P><P class="mid"> (d) piGEM2017-024( Line 1, enzyme digested by PstI; Line 2, enzyme digested by EcoRI +BamHI)</P><P class="mid">M: DNA marker</P><P class="sub-title">3. Degrading pathway construction based on the extracellular expression of DhaA31, HheC-W249P &amp; EchA in tobacco </P><P>In order to transport the three enzymes out of tobacco to play their functions, we chose AO-S, a plant cell wall localization peptide, to help achieve this goal. Besides, AO-S can also stabilize the expression of the proteins. Based on these, the vectors which can fulfil the fusion expression of AO-S together with the three enzymes were constructed. </P><P>Finally, in order to intuitively confirm whether the multi-gene plant expression vectors we designed can play their roles, GUS reporter gene was also introduced into the piGEM2017-backbone (Fig. 5).</P><P class="mid">Figure 5. The introduction of piGEM2017-022 023 025 026 and 027</P><P>Before DNA sequencing, those vectors were verified by restriction enzyme digestion. After electrophoresis analysis, the samples which contained all desired bands were selected and sent for sequencing. The sequencing results showed that all the above constructed vectors were successful (Fig. 6). </P><P class="mid">Figure 6. The image of agarose gel electrophoresis by double enzyme digestion. </P><P class="mid">(a) piGEM2017-022 (b) piGEM2017-023 (c) piGEM2017-025 (d) piGEM2017-026 (e) piGEM2017-027</P><P class="mid">(M: DNA marker; Line 1, enzyme digested by PstI; Line 2, enzyme digested by EcoRI +BamHI)</P></DIV><DIV id="Tobacco"><H2>Tobacco Transformation</H2><P>In order to introduce foreign genes into tobacco and achieve the stable expression of the exogenous proteins in tobacco, Agrobacterium mediated transformation which was a common transgenic method for dicotyledonous plants was used. </P><P>We firstly transferred our plasmids into <EM>Agrobacterium</EM> EHA105 and then the T-DNA fragment which contained the target genes of the <EM>Agrobacterium</EM> EHA105 was integrated into the chromosome of tobacco through the <EM>Agrobacterium</EM> mediated transformation.</P><P>For obtaining the transgenic tobacco, the whole steps can be divided into the following eight parts. Seed germination, pre-culture, infection, co-culture, resistance screening, calli induction, root culture, transplanting. The cycle of obtaining a transgenic tobacco was about ten weeks (Fig. 7).</P><P class="mid">Figure 7. The steps for obtaining transgenic tobacco</P></DIV><DIV id="Positive"><H2>Positive validation of transgenic plants</H2><P>After <EM>Agrobacterium</EM> mediated transformation, we successfully obtained the callis which have the resistance of kanamycin from the leaves of tobacco. The callis began to sprout in about 3 weeks. Another 3 weeks later, we obtained several transgenic tobacco. In order to verify whether the transgenic tobacco was false positive or not, we extracted the genomic DNA of T0 generation tobacco and designed pairs of primers to do PCR amplification of DhaA31, HheC-W249P&amp; EchA. Compared with wild type, we successfully amplified the target bands in T0 transgenic tobacco, which indicated that our multi-gene expression system worked successfully and we obtained the positive transgenic tobacco (Fig. 8). </P><P class="mid">Figure 8. The image of agarose gel electrophoresis by PCR detection</P><P class="mid">a.piGEM2017-001 b.piGEM2017-002 </P><P class="mid">c.piGEM2017-003 d.piGEM2017-004</P><P class="mid">e. piGEM2017-004 f.piGEM2017-005 </P><P class="mid"> ( PC, positive control (plasmid)，WT, negative control (wild-type tobacco))</P><P>In order to confirm whether DhaA31, HheC-W249P, EchA and CKX3 can be normally transcribed in tobacco, and determine the transcription level of these genes in tobacco, we extracted the RNA from the leaves and roots of the transgenic tobacco. And then RT- PCR was used to detect. Tubulin, a common used endogenous gene of the tobacco was selected as internal reference. Compared with the wild type, the cDNA of transgenic tobacco amplified the corresponding band of target genes, indicating that the above four exogenous genes were successfully transcribed in tobacco. At the same time, the cDNA of genes expressed by PYK10 promoter amplified a clearly band in root tissue than in leaves or other tissues, indicating that PYK10 promoter could successfully drive CKX3 gene expressed in the root of tobacco (Fig. 9). </P><P class="mid">Figure 9. The image of agarose gel electrophoresis by RT-PCR detection</P><P class="mid">a.piGEM2017-001 b.piGEM2017-002 c.piGEM2017-003</P><P class="mid">d.piGEM2017-004 e.piGEM2017-005 f.piGEM2017-008</P><P class="mid">(PC, positive control (plasmid)，WT, negative control (wild-type tobacco), WT-L(leaves of </P><P class="mid">wild-type tobacco),WT-R(roots of wild-type tobacco),Tubulin was an internal control in tobacco)</P></DIV><DIV id="Construct"><H2>Construct validation of multi-gene expression system </H2><P>β-D - glucuronidase (encoding GUS gene) was an acidic hydrolase isolated from the <EM>E. coli</EM> K-12 strain that catalyzed the hydrolysis of many beta-glucoside esters. In the histochemical staining with X-gluc, the plants containing the GUS gene will appear blue. Based on this, transgenic tobacco (piGEM2017-026) which had GUS gene and wild type tobacco were identified after the histochemical staining with X-gluc for 24h at the same time. Compared with wild type tocacco, the transgenic tobacco was dyed blue, indicating that the multi-gene expression system in tobacco worked as we expected again. (Fig. 10).</P></DIV><P class="mid">Figure 10. GUS staining of the roots of transgenic tobacco. a. wild-type tobacco  b.piGEM2017-026</P><DIV id="function"><H2>Function validation of PYK10 promoter and CKX3 gene </H2><P>Overexpression of cytokinin oxidase (CKX3) will decrease the expression of cytokinin which could promote roots growth in plants. However, too much expression of cytokinin oxidase will inhibit the shoots growth. Considering that the cytokinin status can be limited to a particular organ or tissue, we chose to express CKX3 by the root specific expression promoter PYK10 to avoid the detrimental consequences of cytokinin deficiency in shoots [1]. </P><P>We selected two transgenic tobacco (piGEM2017-008 and piGEM2017-009) together with the wild-type in the same growth condition and cultured them with the same nutrition. </P><P>Twenty days later, we found that the growth of piGEM2017-009 (35S::CKX3) was weaker than piGEM2017-008 (PYK10::CKX3) and wild-type. Besides, the roots of piGEM2017-008 was significantly stronger than wild-type and the growth of transgenic tobacco was not affected, indicating that CKX3 gene and PYK10 promoter played their own functions (Fig. 11).</P><P class="mid">Figure 11. Growth of PYK10:CKX3, 35S::CKX3 transgenic tobacco plants and wild-type plant</P></DIV><DIV id="Enzyme"><H2>Enzymatic activity determination of transgenic tobacco</H2><P>In order to detect that the transgenic tobacco can work properly, the concentration of the reaction products was measured by gas chromatography (GC) as described in our protocol. Gas chromatograph Network GC System (Fuli 9750, China) equipped with a capillary column ZB-FFAP 30 m x 0.25 mm x 0.25 µm (Phenomenex, USA) was used to the quantitative analyses of all metabolites of the 1,2,3-TCP degradation pathway except for GLY. The concentration of GLY was determined by GC (Fuli 9790) equipped with a DB-5 column (L60 m×i.d. 0.25 mm and df 0.25μm thin coating film) (Supelco, Bellefonate, PA, USA).</P><P class="sub-title">1. Measure the standard curves of substrates and products</P><P>For quantitative assay, standard curves of 1,2,3-TCP, 2,3-DCP, ECH, CPD, GDL, GLY ranging from 0-5 mM in 200 mM of Tris-SO<SUB>4</SUB> buffer(pH=8.5) were measured (Fig. 12).</P><P class="mid">Figure 12. Standard curves of substrates and products</P><P class="sub-title">2. Enzyme activity determined in supernatant of leaves </P><P>For transgenic tobacco carrying piGEM2017-001 to piGEM2017-005, all the enzymes that introduced into tobacco were driven by the common constitutive promoter pCaMV35S, so we could extract enzymes from leaves to detect their activities in vitro. The optimum conditions for our three-enzyme system are pH 8.5 and temperature 37 °C according to literature data [2-4].</P><P class="sub-sub-title">2.1 Activity determination of DhaA31 in transgenic tobacco carrying piGEM2017-001</P><P>Haloalkane dehalogenase (DhaA31) could degrade 1,2,3-TCP to 2,3-DCP. The activity of DhaA31 could be detected with 5mM 1,2,3-TCP using the method described in protocol. As shown in Fig. 13, transgenic tobacco carrying piGEM2017-001 could convert 1,2,3-TCP to 2,3-DCP while wild-type tobacco couldn’t, which proved that DhaA31 could work in tobacco successfully.</P><P style="text-indent: 0px !important;font-size: 14px !important;">Figure 13. The concentration of 2,3-DCP after 7 hours. The activity of DhaA31 was determined in 6.5 mL supernatant of 2 g (FW) leaves of tobacco carrying piGEM2017-001 with 5 mM 1,2,3-TCP. 2 g leaves (FW) were rinsed thoroughly with 6 mL 200mM Tris-SO<SUB>4</SUB> buffer (pH 8.5), centrifuged the mixture and 6.5mL supernatant was obtained. 001-3, 001-4, 001-10 represent three different lines of transgenic tobacco carrying piGEM2017-001. Each data represents the mean value ± standard deviation from three independent experiments.</P><P class="sub-sub-title">2.2 Activity determination of HheC-W249P in transgenic tobacco carrying piGEM2017-002 </P><P> Haloalcohol dehalogenase (HheC-W249P)  could degrade 2,3-DCP and CPD to ECH and GDL, respectively. So we detected its activity with 2,3-DCP and CPD as substrates with transgenic tobacco carrying piGEM2017-002, respectively. But no ECH or GDL were detected at the selected time in wild-type tobacco or tobacco carrying piGEM2017-002 (data not shown), indicating that no active HheC-W249P was expressed. This could be either due to the fact that there was no HheC-W249P expression or the expressed enzyme was less stable in plant. Among these three enzymes, HheC-W249P is the only enzyme that is active as a tetramer, while the other two enzymes are monomer. This could also contribute to the fact that HheC-W249P activity was not detected in transgenic tobacco carrying piGEM2017-002. Due to the limitation of time we could not further optimize the expression system of HheC-W249P. </P><P>Instead, we tested the activity of HheC by mimic the environment of tobacco in vitro. For this, HheC produced by recombinant <EM>E.coli</EM> was added into supernatant of the wild-type tobacco rinsed in Tris-SO<SUB>4</SUB> buffer. As shown in Fig. 14, the concentration of ECH in supernatant of wild-type tobacco and Tris-SO<SUB>4</SUB> buffer was similar, indicating that components existed in tobacco did not show a significant effect on the activity of HheC. In the following experiments, HheC produced by <EM>E.coli</EM> was added in the whole 1,2,3-TCP degradation pathway. </P><P style="text-indent: 0px !important;font-size: 14px !important;">Figure 14. The concentration of ECH after 7 hours. Tris-SO<SUB>4</SUB>: 6.5mL 200mM Tris-SO<SUB>4</SUB> buffer (pH 8.5) with 5mM 2,3-DCP added 20 μg purified HheC was used as control. Supernatant: 6.5mL supernatant of wild-type tobacco with 5mM 2,3-DCP added same amount HheC. 2 g (FW) leaves of wild-type tobacco were rinsed thoroughly with 6 mL 200mM Tris-SO<SUB>4</SUB> buffer (pH 8.5), centrifuged the mixture and 6.5mL supernatant was obtained. Each data represents the mean value ± standard deviation from three independent experiments.</P><P class="sub-sub-title">2.3 Activity determination of EchA in transgenic tobacco carrying piGEM2017-003</P><P>Epoxide hydrolase (EchA) could convert ECH and GDL to CPD, GLY, respectively. So we could detect the activity of EchA with 5mM ECH. The production of CPD after 7 hours was shown in Fig. 15. The result showed that EchA could work in tobacco. Wild-type tobacco could also degrade ECH to CPD because of endogenous epoxide hydrolase. The existence of endogenous epoxide hydrolase was reported in another paper [5]. </P><P style="text-indent: 0px !important;font-size: 14px !important;">Figure 15. The concentration of CPD after 7 hours. The activity of EchA was determined in 6.5 mL supernatant of 2 g (FW) leaves of tobacco carrying piGEM2017-003 with 5 mM ECH. 2 g (FW) leaves were rinsed thoroughly with 6 mL 200mM Tris-SO<SUB>4</SUB> buffer (pH 8.5), centrifuged the mixture and 6.5mL supernatant was obtained. 003-1, 003-5, 003-8 represent three different lines of transgenic tobacco carrying piGEM2017-003. Each data represents the mean value ± standard deviation from three independent experiments. </P><P class="sub-sub-title">2.4 Work validation of multi-enzyme system in transgenic tobacco carrying piGEM2017-004, piGEM2017-005, respectively </P><P>DhaA, HheC and EchA catalyzed five-step reactions converting toxic TCP to harmless glycerol. To test whether this multi-enzyme conversion system could work in tobacco successfully or not, we detected the time consumption curve of 6 mM 1,2,3-TCP using grinded leaves of transgenic tobacco carrying piGEM2017-004. As described above, tobacco contains endogenous epoxide hydrolase, so we also detected the efficiency of converting TCP to GLY by transgenic tobacco carrying piGEM2017-005 at the same time. HheC produced by E.coli was added in the reaction mixture. The concentrations of 1,2,3-TCP, GLY and other intermediates were monitored as shown in Fig. 16. Glycerol produced gradually with the consumption of 1,2,3-TCP. No glycerol was detected in supernatant of wild-type tobacco at every selected time. The degradation efficiency of tobacco carrying piGEM2017-005 was similar with that of piGEM2017-004. After 30 h, less than 1 mM of TCP was left, indicating that our system worked well in tobacco.</P><P style="text-indent: 0px !important;font-size: 14px !important;">Figure 16. Time courses of conversion of 1,2,3-TCP with 10 mL supernatant extracted from leaves of transgenic tobacco carrying piGEM2017-004 (a) and piGEM2017-005 (b) with 6 mM 1,2,3-TCP. 4 g (FW) leaves were rinsed thoroughly with 8 mL 200mM Tris-SO<SUB>4</SUB> buffer (pH 8.5), centrifuged the mixture and 10 mL supernatant was obtained. 200 μg HheC (produced by recombinant <EM>E.coli</EM>) was added in both reaction mixtures. </P><P class="sub-sub-title">2.5 Glycerol measurement using gas chromatography–mass spectrometry(GC-MS)</P><P>The glycerol production was monitored by periodically taking samples from the supernatant of tobacco carrying piGEM2017-004 and piGEM2017-005 as described in 2.4. GC-MS was used to detect the concentration of glycerol using the method reported in another paper [6]. Glycerol and internal standard 1,2,3-butanetriol were silanized by TMSIM reagent in the per-column derivatization reaction and converted to tris(trimethylsilyl) glycerol ether and 1,2,3-tris(trimethylsilyl) butanetriol ether, respectively. Fig.17 is chromatograms of the supernatant samples, the peak of glycerol after the pre-column derivatization was observed from samples of tobacco carrying piGEM2017-004 after 0h and 20h. Most importantly, the peak of glycerol after the pre-column derivatization increased within 20 hours while glycerol after the pre-column derivatization was not found in sample of wild-type tobacco. </P><P class="mid">Figure 17. Chromatogram of glycerol and internal 1,2,3-butanetriol derivatives in supernatant of transgenic tobacco carrying piGEM2017-004 and wild-type tobacco after the pre-column derivatization. Samples were taken from reaction mixture at 0h, 20h.</P><P class="mid">Figure 18. Mass spectrums of tris(trimethylsilyl) glycerol ether and 1,2,3-tris(trimethylsilyl) butanetriol ether</P></DIV><DIV id="Activity"><H2>Activity determination of multi-enzyme system in hydroponic solutions</H2><P>To investigate whether our ‘super tobacco’ can work or not when placed in the environment, we used the method of hydroponics. We made a gastight device as shown in Fig.19.  Plants were transplanted in liquid nutrient solutions containing 5mM substrate. Gas chromatography was used to detect the products. For tobacco carrying piGEM2017-001 and piGEM2017-003, samples were taken from medium on the third day. As shown in Fig.18, DhaA31 and EchA could work successfully, indicating that our transgenic plants could work properly in the hydroponic environment. </P><P>For tobacco carrying piGEM2017-004, we put the immobilized cell-free extracts containing HheC into medium. We didn't find glycerol in samples taken from medium on the sixth day.  On the eighth day, considering the GLY might be involved in plant metabolism, we also took 0.9g fresh tissue and rinsed thoroughly with 1.8mL 200mM Tris-SO4 buffer (pH 8.5).  As shown in Fig. 21, tobacco carrying piGEM2017-004 contained more glycerol than wild-type tobacco, which was caused by the conversion of 1,2,3-TCP. </P><P class="mid">Figure 19. Hydroponic device designed by our team members</P><P class="mid">Figure 20. The concentration of 2,3-DCP (a) and CPD (b) after 3 days. Samples were taken from medium.</P><P class="mid">Figure 21. The concentration of GLY taken from plant tissue</P></DIV><DIV id="Work"><H2>Work going on</H2><P>1. We will detect the efficiency of degrading 1,2,3-TCP in transgenic tobacco carrying three enzymes together with AO-S peptide and cytokinin oxidase (CKX3) gene driven by root specific expression promoter PYK10. </P><P>2. We will use a stable mutant of HheC instead of HheC-W249P to optimize our degrading system. </P><P>3. To further prove that our super tobacco can degrade 1,2,3-TCP in the natural environment, we will detect its efficiency of degrading 1,2,3-TCP planted in soil. Healthy wild-type and transgenic tobacco were transplanted into flower plot in our laboratory (Fig 22). Later we will make a gastight device to plant it and mix 1,2,3-TCP with soil in our laboratory. </P><P class="mid">Figure 22. Soil cultivation in our laboratory</P><H2>References</H2><OL><LI>Werner, T., et al., Root-specific reduction of cytokinin causes enhanced root growth, drought tolerance, and leaf mineral enrichment in Arabidopsis and tobacco. Plant Cell, 2010. 22(12): p. 3905-20.</LI><LI>Kulakova, A.N., M.J. Larkin, and L.A. Kulakov, The plasmid-located haloalkane dehalogenase gene from Rhodococcus rhodochrous NCIMB 13064. Microbiology, 1997. 143 ( Pt 1): p. 109-15.</LI><LI>van Hylckama Vlieg, J.E., et al., Halohydrin dehalogenases are structurally and mechanistically related to short-chain dehydrogenases/reductases. J Bacteriol, 2001. 183(17): p. 5058-66.</LI><LI>Jacobs, M.H., et al., Characterization of the epoxide hydrolase from an epichlorohydrin-degrading Pseudomonas sp. Eur J Biochem, 1991. 202(3): p. 1217-22.</LI><LI>Guo, A., J. Durner, and D.F. Klessig, Characterization of a tobacco epoxide hydrolase gene induced during the resistance response to TMV. Plant J, 1998. 15(5): p. 647-56.</LI><LI>Shen, Y. and Z. Xu, An improved GC-MS method in determining glycerol in different types of biological samples. J Chromatogr B Analyt Technol Biomed Life Sci, 2013. 930: p. 36-40.</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>