"
Page
Discussion
View source
History
teams
Log in
 
Team:Groningen/Navigation/SilkAssemblyShop
From 2013.igem.org
Home
Team
Team
Official profile
Acknowledgements
Attributions
Gallery
Spider silk
Overview
Background
Project
Introduction
Backbone
Silk Assembly Shop
Stay Warm, Stay Close
The Future
Wet Lab
Experiments
Protocols
Modeling
Heat Motility
Codon optimization
Labjournal
Submitted Parts
Outreach
Night of
Art & Science
Indian summer
Summerschool
Symposia
Achievements
Medal fulfillment
Collaboration
Safety
Sponsors
Sponsors
How to donate
Contact
Silk Assembly Shop
Bacillus subtilis silk genes collection derived from Argiope aurantia MaSp2
One of the challenges that the project includes is the production of spider silk. In order to achieve this goal iGEM Groningen designed for you what we call “The Bacillus subtilis silk assembly shop”.
The 'shop' idea follows those conceptual steps:
Design your own silk protein
Browse the iGEM Groningen silk subunits collection:
Spider silk
subunit
Biobrick id
Construct Features
RBS
Strep-tag
Stop codon
E1
BBa_K1085000
25px
25px
BBa_K1085001
25px
E2
BBa_K1085002
BBa_K1085003
25px
Tail
BBa_K1085008
25px
BBa_K1085009
25px
25px
Table 1: The following silk subunits are listed: E1 whose sequence is codon-optimized to prevent the formation of secondary RNA structures in close proximity to the RBS, whereas E2 is more focused in reducing the presence of restriction sites and Tail whose sequence in order to match the translation rate between Bacillus subtilis and Argiope aurantia.
Select the subunits you need
Assemble them together
At this point the silk product is ready to be produced ( See production backbone ) and employed for the desired purpose.
The Groningen team enriched the iGEM database adding several new Bacillus subtilis BioBricks containing spider silk gene subunits. This provides the iGEM community with a new bunch of parts; which, in the future, will help anyone in assembling its own silk protein and customize it with different features.
It is the first time that silk-coding BioBricks suitable for Bacillus subtilis are introduced into the database. Therefore this work marks the starting point for further studies about silk within the Bacillus subtilis chassis inside the iGEM framework.
The advantages shown by our collection are mainly three which can be summarized as follows:
Accessible design The iGEM database provides a lot of basic parts (such as RBS and terminator), but often achieving the desired goal turns out to be really tedious and time consuming, since the creation of a working construct needs to go through a lot of steps. iGEM Groningen took into account this problematic aspect while designing the silk collection, therefore all the silk subunits (each of them coding for the same final protein product) have been created in different versions (see Table 1). Such a design enables the end-user to quickly reach the desired silk protein construct without bothering too much about the molecular issues. This kind of approach helps to make synthetic biology and specifically working with silk protein production in Bacilus subtilis more accessible.
Compatibility with BioBrick standards We are aware that the BioBrick assembly standard is the foundation of the iGEM method, allowing all different laboratories to build on each others work. That is why the silk subunits are compatible with the following standard:
BBF RFC 10
BBF RFC 12
BBF RFC 21
BBF RFC 23
BBF RFC 25
BBF RFC 1000
Dealing with repetitiveness One of the issue while working with the silk protein is the high repetitiveness of the coding sequence. The problem is that the molecular techniques, such as PCR, are less effective on this type of DNA template. The subunit design provides us with a solution, since the molecular steps do not depend anymore on the silk coding sequence itself, but on the universal flanking regions.
Strep-tag
The Strep-tag is there for two reasons:
mainly as a binding component to Streptavidin ( See coating mechanism )
as well as a way of detecting the protein production
Assembly shop in action
Using the silk assembly shop idea the Groningen team developed its own personal silk gene version ( BBa_K1085018 ), which is composed by three subunits: E1, E2 and Tail (Figure 1).
Figure 1: This is the molecular schematic representation of BBa_K1085018 where the molecular features (ribosome binding site, strep-tag and stop codon) are shown.
The final construct contains all the features necessary in order to start the translation process, moreover the final protein shows the Strep-tag which can be used in order to detect the presence of the protein.
This proves that all the pieces can be assembled into any desired construct.
Secretion signal peptide accessory
Signal peptides sequences are introduced in the subunit design for the aim of, not only producing silk protein, but also secreting it. This has been done by embedding signal peptide particle (SRP) sequences, fished from the Bacillus subtilis genome, in front of the silk subunits with the purpose of directing the protein into the Sec secretion pathway (Figure 2).
Figure 2: The schematic representation (adapted from van Wely et al. 2001) shows how the signal recognition particle (SRP) helps in directing the silk protein towards the outer membrane compartment of Bacillus subtilis.  The Sec secretion machinery exports the silk protein. Proteases cleaves the SRP from the secreted protein.
A selection of SRP sequences is made and those have been cloned in front of the E1 subunit in order to directs the produced silk protein towards the Sec pathway track.
The following signal sequences are fused together with the E1 silk subunit:
SRP source
Biobrick id
Construct Features
RBS
Strep-tag
EstA-E1
BBa_K1085035
25px
25px
FliZ-E1
BBa_K1085029
25px
MotB-E1
BBa_K1085033
25px
Table 2: The signal recognition particle (SRP) sequences which were cloned in front of the E1 silk subunit are listed together with the features shown by the construct.
iGEM 2013 Groningen
Retrieved from " http://2013.igem.org/Team:Groningen/Navigation/SilkAssemblyShop "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
