<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Groningen_Navigation_Heatmotility"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Groningen/Navigation/Heatmotility">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Groningen/Navigation/Heatmotility&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/Navigation/Heatmotility&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/Navigation/Heatmotility&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Groningen/Navigation/Heatmotility" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Groningen/Navigation/Heatmotility</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="colmask threecol"><SPAN class="navigation"><NAV><UL class="accordion"><LI id="one" class="files"><A href="https://2013.igem.org/Team:Groningen">Home</A></LI><LI id="two" class="files" class="navigation"><A href="#two">Team</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Team#two">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1085" target="_blank">Official profile</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Acknowledgements#two">Acknowledgements</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Attributions#two">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/photos#two">Gallery</A></LI></UL><LI id="three" class="files" class="navigation"><A href="#three">Spider silk</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Silk#three">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Background#three">Background</A></LI><LI id="four" class="sign" class="navigation"><A href="#four">Project</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Project#four">Introduction</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Construct#four">Backbone</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/SilkAssemblyShop#four">Silk Assembly Shop</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Motility#four">Stay Warm, Stay Close</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/SilkSecretion#four">The Future</A></LI><LI id="five" class="sign" class="navigation"><A href="#five" target="_self">Wet Lab</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/experiments#five" target="_self">Experiments</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Protocols#five" target="_self">Protocols</A></LI><LI id="six" class="sign" class="navigation"><A href="#six" target="_top">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Heatmotility#six">Heat Motility</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Codonoptimization#six">Codon optimization</A></LI><LI id="seven" class="sign"><A href="https://2013.igem.org/Team:Groningen/Navigation/Notebook">Labjournal</A></LI><LI id="eight" class="sign"><A href="https://2013.igem.org/Team:Groningen/Navigation/Parts">Submitted Parts</A></LI><LI id="nine" class="sign" class="navigation"><A href="#nine" target="_top">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/noas#nine">Night of Art &amp; Science</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Indiansummer#nine">Indian summer</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/summerschool#nine">Summerschool</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Symposia#nine">Symposia</A></LI><LI id="ten" class="sign" class="navigation"><A href="#ten" target="_top">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Medals#ten">Medal fulfillment</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/PartsCollaboration#ten">Collaboration</A></LI><LI id="eleven" class="sign"><A href="https://2013.igem.org/Team:Groningen/Navigation/Safety">Safety</A></LI><LI id="twelve" class="sign" class="navigation"><A href="#twelve">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Sponsors#twelve">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Groningen/Navigation/Donations#twelve">How to donate</A></LI><LI id="eleven" class="sign"><A>Contact</A></LI></NAV></SPAN><DIV class="mainContent"><H1>Heat Motility</H1><P>The goal for our heat motility system is to obtain higher concentrations of silk in close proximity to the implant. This is achieved by immobilizing the silk producing bacteria once they are within some distance to the implant, which is enabled by integrating the standard chemotaxis system with the DesK membrane fluidity sensor system. The result is that <I>Bacillus subtilis</I> stops moving when it is 37°C, and swims when it is 25°C. In our model we simulate the behavior of these systems and, if necessary, implement biologically plausible modifications to make it feasible. </P><H2>The chemotaxis system of <I>B. subtilis</I></H2><P>We first setup a model for the standard chemotaxis system. This system enables bacteria to seek out areas of higher concentrations of nutrients by modifying the direction of flagella rotations as a response to changes in concentrations of attractants and repellents. An increase in the concentration of attractants results in more counter clockwise (CCW) rotations, and an increase in the concentration of repellents in more clockwise (CW) rotations, which corresponds to an increased chance of swimming straight and tumbling respectively. This effect is complemented by the systems essential ability to adapt to any concentration of attractant or repellent. Given a homogenous environment, the system will therefore always maintain the same swimming to tumbling ratios. </P><P>At the molecular level the flagella behavior is controlled directly by the concentration of phosphorylated CheY (YP): more YP results in more CCW rotations (more swimming) and less YP in more CW rotations (more tumbling). YP is controlled by phosphorylated CheA concentrations (AP), its natural decay rate, and by CheC_CheD complex concentrations (CD), which act as YP phosphorylases. Because the natural and CD induced YP decay rates are unknown, we combined them by making the k11 rate coefficient a function of CD with the natural YP decay rate as its offset.</P><DIV align="center"><TABLE id="layout" width="20%"><TBODY><TR><TD><FONT size="1"><DIV align="center">Chemical reaction scheme 1.</DIV></FONT></TD></TR></TBODY></TABLE></DIV><P>In order to obtain proper swimming behavior, we require an initial increase and a delayed decrease in YP. The initial YP increase is due to an in increase in AP (which donate their phosphor groups to CheY) as a response to increased attractant concentrations. The decrease in YP is realized by two negative feedback systems (the CheV and the methylation subsystem), which decrease CheA phosphorylation rates, and by one negative feedforward system (the CheC-CheD subsystem), which increases the YP decay rate.</P><DIV align="center"><TABLE id="layout" width="40%"><TBODY><TR><TD><FONT size="1">Figure 1. The chemotaxis system of <I>B.subtilis</I>. The green arrows indicate a stimulatory function, and the red arrows an inhibitory function.</FONT></TD></TR></TBODY></TABLE></DIV><H3>CheA phosphorylation</H3><P>Inside the cell CheW, CheA and CheV form complexes with the inner membrane receptor protein. When attractants bind to the receptor, CheA undergoes a change in conformation that enables it to be phosphorylated.</P><P>Following the example of David C. and John Ross [1], we assume the binding and dissociation of attractants with their receptors is fast, and that it can be described with the equilibrium equation (1), where KD is the dissociation constant for the aspartate receptor (which is focus of this model). The concentration of unbound receptors is then given by equation (2).</P><P>Due to high concentrations of ATP in the cell, we assume similar kinetics for the CheA phosphorylation rate. It then follows that the positive increase in AP levels is proportional to bound receptor (Ta) and unbound receptor (T0) levels, and can be described by equation (3), where ATR is a constant.</P><H3>The CheV subsystem</H3><P>The first adaption system of <I>B.subtilis</I>, the CheV subsystem, contributes to a delayed decrease in YP through a decrease in AP. This is realized by phosphorylated CheV (VP), which obtains its phosphor group from AP, after which it may disrupt the bond between the attractant-bound receptor and CheA, hereby inhibiting CheA phosphorylation [2].</P><P>Because AP levels are proportional to attractant levels we neglect the AP in our reactions (chem reaction 2) and take the rate coefficient k1 as a linear function of attractant levels. This disrupts the feedback loop between AP and VP, and makes the optimization later easier (see 'Parameter optimization' section). </P><DIV align="center"><TABLE id="layout" width="20%"><TBODY><TR><TD><FONT size="1"><DIV align="center">Chemical reaction scheme 2</DIV></FONT></TD></TR></TBODY></TABLE></DIV><P>The negative decrease in AP due to VP is approximately 22% of the positive increase in AP described in equation (3), and is incorporated into the AP formula by subtraction in equation (4).</P><H3>The methylation subsystem</H3><P>The second adaption system of <I>B.subtilis</I> is the methylation system, which regulates the CheA phosphorylation rate through phosphorylated CheB (BP) and CheR. After obtaining its phosphor group from AP, BP can shuffle methyl groups between different receptor residues. It is thus proposed that the methylation of certain receptor residues can activate the receptor, whilst the methylation of others can deactivate it [2].</P><P>In order to provide negative AP feedback, BP would then shuffle methyl groups on the activation residues to the deactivation residues, whilst CheR shuffles methyl groups from the deactivation residues to the activation residues. We describe this system with the chemical reactions given below, where T<I>M0</I> represents the methyl-deactivated receptors and T<I>M1</I> the methyl-activated receptors. </P><DIV align="center"><TABLE id="layout" width="25%"><TBODY><TR><TD><FONT size="1"><DIV align="center">Chemical reaction scheme 3.</DIV></FONT></TD></TR></TBODY></TABLE></DIV><P>Rate coefficients k3 and k5 are again linear functions of attractant concentrations, which enables us to eliminate the feedback loop involving AP, BP, and T<I>M1</I>, and is again justified by the fact that AP is proportional to attractant levels. The negative AP feedback involving T<I>M1</I> is incorporated into equation (4) simply by subtracting T<I>M1</I> from AP, and is given by equation (5). The total amount of negative AP feedback is approximately 35% of the positive increase. 
</P><H3>The CheC-CheD subsystem</H3><P>The third adaption system of <I>B.subtilis </I> involves the CheC and CheD chemotaxis proteins. As of yet there appears to be no consensus on the exact behavior of this system. Christopher V. Rao. et al (2008) propose two viable options; a negative feedback and a negative feedforward system. </P><P>In the negative feedback system YP binds to CheC and CheD to form a complex. In doing so, it may pull receptor-bound CheD from the receptor where it functions as a stimulatory protein with respect to CheA phosphorylation. In this system, YP is therefore decreased through AP by means of removing CheD from the receptor </P><P>In the negative feedforward system CheD is also bound to the receptor, but it no longer has its stimulatory function. Instead, it is released from the receptor upon the binding of attractant, after which it may form a complex with CheC where it functions as a YP phosphatase. </P><P>Contrary to Christopher et al (2008), we propose that the negative feedforward system is the more likely case since the negative feedback system includes a time extensive feedback loop involving AP and YP through CheC, CheD, and the receptors. In our test simulations (not presented here), we observed adaption times of up to 10 minutes, whilst wild type <I>B.subtilis</I> adaption times are approximately one minute [4]. </P><P>The chemical reactions for this negative feedforward system can be seen below, where T<I>D</I> is the concentration of receptors with CheD bound, T_ the concentration of receptors with no CheD bound, and C CheC. We furthermore model the receptors change in conformation, and the following CheD release, by taking k7 as a non-linear function of attractant levels with an offset equal to its CheD release rate with no attractant present.</P><DIV align="center"><TABLE id="layout" width="18%"><TBODY><TR><TD><FONT size="1"><DIV align="center">Chemical reaction scheme 4.</DIV></FONT></TD></TR></TBODY></TABLE></DIV><H3>Parameter optimization</H3><P>Although the rate coefficients and molecular concentrations of the chemotaxis system in <I>E.coli</I> are relatively well documented, for <I>B.subtilis</I> they are mostly unknown. Where possible, we therefore adopt values from the <I>E.coli</I> chemotaxis system, for which Li and Hazelbauer (2008) have measured the following molecular concentrations: CheA+AP=5.3µM, CheB+BP=0.28µM, CheY+YP=9.7µM, and CheR=0.16µM.</P><P>We now assume that CheA and CheV are saturated with respect to the formation of receptor complexes with CheW and the inner membrane receptor protein. We further assume that the conserved molecular concentrations of CheD are equivalent to CheA, so that we can state the following: CheV+VP = CheA+AP = T<I>a</I>+T<I>0</I> = T<I>D</I>+T_ = T<I>M1</I>+T<I>M0</I> = CD+T<I>D</I>+D = 5.3µM</P><P>To find appropriate values for the rate coefficients we applied the simulated annealing algorithm. This is a heuristic-driven &amp; probabilistic optimization algorithm that can be used to find optimal solutions for complex problems. In our case, we require the algorithm to find the set of parameters that enable exact adaption of YP concentrations. </P><P>Before we can apply the algorithm, we must define an error function. The error function determines what will be optimized, as it is the result of this function that the algorithm will strive to minimize. Because we are interested in the steady state value of YP, we require all molecular values to be in equilibrium before any further assessment can be established. The core of the error function for some set of parameters is therefore defined as the distance of each molecular concentration from its equilibrium value. </P><P>We first applied the algorithm to the CheV subsystem for attractant levels 0 and 5.8 (saturation). The error function was defined as the distance of each molecular concentration from its equilibrium value, along with the requirement that the total amount of negative AP feedback was 20 to 40% of the total attractant-induced AP increase. The same procedure was applied to the methylation subsystem, and, finally, to the system as a whole, where the previously optimized parameters were of course not further optimized. The error function was also modified to ensure that the YP equilibrium was constant across all levels of attractant ranging from 0 to 5.8. </P><H3>Exact adaption</H3><P>We used matlabs ode45 function to simulate the differential equations for different levels of attractant. The simulations show how YP reactes to the addition and removal of different levels of attractant, which correspond to <I>B.subtilis</I> swimming and tumbling along increasing and decreasing gradients of attractant respectively. The tumbling along the decreasing gradients ensures that the bacteria will not swim deeply into undesirable areas, but will instead tumble at the edge until adaption has taken place, and it resumes its standard run and tumble behavior. Because it is still on the edge of the more desirable region it will have a higher chance of falling back in, after which YP will again increase so that <I>B.subtilis</I> can swim deeper into the nutrient rich area. </P><DIV align="center"><TABLE id="layout" width="80%"><TBODY><TR><TD><FONT size="1">Figure 2. Left: YP adaption to the addition and removal of 2µM attractant at t=10 and t=70 respectively. Right: YP adaption to different concentrations of attractant.</FONT></TD></TR></TBODY></TABLE></DIV><H2>Motility model</H2><P>Before we can simulate the swimming behaviour of <I>B.subtilis</I> we need to relate YP to flagella rotations, for which we use the formula provided by Kuo and Koshland (1989), seen in equation (6), where h is a constant we use to fit our steady state YP value to data. P(CCW) can then be calculated by 1-P(CW). </P><P>We know that in normal steady state conditions the chance of flagella spinning CCW is 0.65, so that P(CCW) = 0.65 and P(CW)=0.35 [1]. We also know that the mean swimming period is ~1.02 seconds and the mean tumbling period 0.165 seconds [5], so that for every 100 ms the chance of swimming given a swimming state is 0.903%, and the chance of tumbling given a tumbling state is 0.6061, which is to say: P(S|S)= 0.903 and P(T|T)= 0.6061. We also know that when CCW=0, that P(S|S)=1 and P(T|T)=0, and that when CCW=1, that P(S|S)=0 and P(T|T)=1. We can now use these points to fit two curves (equations 7 and 8), which can provide the threshold for switching from a tumbling state to a swimming state, and vica versa. The curves and equations are given below as a function of P(CCW). </P><DIV align="center"><TABLE id="layout" width="50%"><TBODY><TR><TD><FONT size="1">Figure 3. Top: the probability of continuing a swim every 100 ms as a function of flagella CCW rotations. Bottom: the probability of continuing a tumble every 100 ms as a function of flagella CCW rotations. </FONT></TD></TR></TBODY></TABLE></DIV><P>In order to test whether our model demonstrates proper chemotaxis behavior, we simulate <I>B.subtilis</I> swimming in a small container with attractants. In the first simulation the dimensions of the container are approximately 250 by 250 micrometers, and the simulation is speeded up to 4x real time. This simulation shows how <I>B.subtilis</I> prefers to swim along increasing gradients of attractant, and tumble along decreasing gradients of attractant. In the second simulation the container is approximately 500 by 500 micrometers, and it is speeded up to 8x real time. Here the bacteria require more time to find the attractant, but when they do they get 'trapped' inside the regions with higher attractant concentrations.</P><DIV align="left"><TABLE id="layout" width="100%"><TBODY><TR><TD colspan="2"><FONT size="1">Simulations 1 and 2. Left; chemotaxis behaviour of <I>B.subtilis</I> in a ~250 by ~250 micrometer container in 4x real time. Right; chemotaxis behaviour of <I>B.subtilis</I> in a ~500 by ~500 micrometer container in 8x real time. </FONT></TD></TR></TBODY></TABLE></DIV><H3>Modifications</H3><P>After observing the swimming behavior of <I>B.subtilis</I> in homogenous and non-homogenous environments, we realized that too much time would be required for <I>Bacillus</I> to swim from the cold to the warm region: in a homogenous environment the swimming behavior is too random, and in a non-homogenous environment it may get trapped in regions with higher concentrations of attractant, and otherwise its progress will get stalled by decreasing gradients of attractants.</P><P>A solution is to knockout one of the adaption systems to increase YP levels [7]. This then increase P(CCW), and hence the chance of maintaining a swimming state. We simulate a CheC knockout, and plot the traversed path of 10 bacteria with and without the knockout (figure 4). </P><DIV align="center"><TABLE id="layout" width="80%"><TBODY><TR><TD><FONT size="1">Figure 4. Left: traversed pathway of 10 <I>B.subtilis</I> bacteria in an environment with no attractant. Right: traversed pathway of 10 <I>B&gt;subtilis</I> bacteria with a CheC knockout in an environment with no attractant.</FONT></TD></TR></TBODY></TABLE></DIV><P>The animation below shows how <I>B.subtilis</I><I>cheC</I> null mutants move around in an environment with attractants. It no longer cares if its swimming up a gradient of increasing attractants, or down a gradient of decreasing attractants.</P><DIV align="center"><TABLE id="layout" width="100%"><TBODY><TR><TD><FONT size="1">Simulation 3. The chemotaxis behaviour of a cheC <I>B.subtilis</I> null mutant in a ~250 by ~250 micrometer container in 4x real time.</FONT></TD></TR></TBODY></TABLE></DIV><H2>The effects of temperature</H2><P>The temperature range in which <I>B.subtilis</I> will be operating in is 25°C to 37°C. We therefore require our model to be valid within this range. To test this, we use a general rule of thumb, which states that rate coefficients approximately double with every 10°C increase. We use equation (9), which is valid for temperatures ranging from 25°C to 47°C, and where k is the rate coefficient in question, T the temperature and k' the new rate coefficient. </P><P>In the literature, YP adaption has been observed for temperatures of up to 42°C [6]. Adaption for 2 µM attractant for temperatures of 25°C, 31°C, and 37°C are plotted below (figure 5).

</P><DIV align="center"><TABLE id="layout" width="50%"><TBODY><TR><TD><FONT size="1">Figure 5. The effect of temperature on YP adaption for 2 µM attractant.</FONT></TD></TR></TBODY></TABLE></DIV><H3>The DesK system </H3><P>We are now ready to couple our modified chemotaxis system to the modified membrane fluidity sensor system DesK (read more about the DesK system <A href="https://2013.igem.org/Team:Groningen/Navigation/Motility#four">here</A>). In this system CheY is placed under the control of the des promoter so that CheY production is maximized at 25°C and minimized at 37°C. We thus hope for smooth swimming in the cold regions, and excessive tumbling in the warm regions.</P><P>Perhaps the most essential parameter for this process is the des promoter strength, which is, unfortunately, unknown. What we do know is that the number of PoPs at 25°C is 10 to 15 times higher than at 37°C, and that the decrease in temperature follows an exponential curve [8, 9]. We therefore assume that the promoter strength at 37°C is small enough to cause excessive tumbling, and that at 25°C CheY+YP is 10-15 times larger than at 37°C. The conserved molecular concentrations of CheY+YP are scaled with the temperature and can be seen below (figure 6). </P><DIV align="center"><TABLE id="layout" width="50%"><TBODY><TR><TD><FONT size="1">Figure 6. Conserved CheY+YP concentrations as a function of temperature.</FONT></TD></TR></TBODY></TABLE></DIV><P>We now simulate the swimming behavior of 50 bacteria with modified chemotaxis and DesK systems. The container is approximately 500 by 500 micrometers and the temperature increases linearly with the y-axis from 25°C to 37°C.</P><P> The first simulation is 4x real time, and the bacteria start close to the 25°C region. We observe that the bacteria cant get past the ~30-32°C region due to excessive tumbling, and that the bacteria will therefore never reach the 37°C region. </P><P>The second simulation is 8x real time, and the bacteria now start in or close to the 37°C region. We see that our original concept is indeed valid, and that the bacteria should never escape the 37°C region due to excessive tumbling. </P><P>The third simulation is also 8x real time, but the initial positions of the bacteria are now random. We see that the system has very little effect on the concentration of bacteria throughout the container since the end distribution is also random.</P><DIV align="center"><TABLE id="layout" width="100%"><TBODY><TR><TD colspan="3"><FONT size="1">Simulations 4, 5, and 6. Left: simulation of 50 <I>B.subtilis</I> units in a container of approximately ~500 by ~500 micrometers with a linear temperature gradient ranging from 25°C to 37°C, and with the initial position of the bacteria located near the 25°C area. Middel: simulation of 50 <I>B.subtilis</I> units in a container of approximately ~500 by ~500 micrometers with a linear temperature gradient ranging from 25°C to 37°C, and with the initial position of the bacteria located near the 37°C area. Right: simulation of 50 <I>B.subtilis</I> units in a container of approximately ~500 by ~500 micrometers with a linear temperature gradient ranging from 25°C to 37°C, and where the initial position of the bacteria is random. </FONT></TD></TR></TBODY></TABLE></DIV><H3>Conclusion </H3>
We conclude that this system could potentially be used to keep bacteria away from warm regions. It is therefore not an optimal solution for obtaining a higher concentration of bacteria in close proximity to the implant (which is located in the 37°C region). To make this system somewhat feasible for our concept, we would have to carefully inject the bacteria directly into the 37°C region.<H1>Parameters</H1><DIV align="center"><TABLE id="normal" width="40%"><DIV align="center"><TBODY><TR><TH><DIV align="center">
Protein concentrations
</DIV></TH><TH>
Value
</TH><TH>
Reference
</TH></TR><TR><TD>
Y+YP
</TD><TD>
9.7µM
</TD><TD>
[3]
</TD></TR><TR><TD>
B+BP
</TD><TD>
0.28µM
</TD><TD>
[3]
</TD></TR><TR><TD>
CheR
</TD><TD>
0.16
</TD><TD>
[3]
</TD></TR><TR><TD>
A+AP
</TD><TD>
5.3µM
</TD><TD>
[3]
</TD></TR><TR><TD>
T<I>a</I>+T<I>0</I></TD><TD>
5.3µM
</TD><TD>
-
</TD></TR><TR><TD>
V+VP
</TD><TD>
5.3µM
</TD><TD>
-
</TD></TR><TR><TD>
CheD+CD+T<I>D</I></TD><TD>
5.3µM</TD><TD>
-
</TD></TR><TR><TD>
T<I>D</I>+T_
</TD><TD>
5.3µM
</TD><TD>
-
</TD></TR><TR><TD>
T<I>M0</I>+<I>TM1</I></TD><TD>
5.3µM
</TD><TD>
-
</TD></TR><TR><TD>
CheC+CD
</TD><TD>
3.9686µM
</TD><TD>
Optimized value
</TD></TR></TBODY></DIV></TABLE><TABLE id="normal" width="40%"><TBODY><TR><TH>
Rate coefficients
</TH><TH>
Value
</TH></TR><TR><TD>
k2
</TD><TD>
2.79937
</TD></TR><TR><TD>
k4
</TD><TD>
3.21494
</TD></TR><TR><TD>
k6
</TD><TD>
0.74553
</TD></TR><TR><TD>
k8
</TD><TD>
0.09981
</TD></TR><TR><TD>
k9
</TD><TD>
0.04779
</TD></TR><TR><TD>
k10
</TD><TD>
0.11346
</TD></TR><TR><TD>
k12
</TD><TD>
1.39741
</TD></TR></TBODY></TABLE><TABLE id="normal" width="40%"><TBODY><TR><TH>
Constants
</TH><TH>
Value
</TH></TR><TR><TD>
k11_basal
</TD><TD>
0.08506
</TD></TR><TR><TD>
k11_scale
</TD><TD>
3.976
</TD></TR><TR><TD>
ATR
</TD><TD>
44.9309
</TD></TR><TR><TD>
KD
</TD><TD>
0.5
</TD></TR><TR><TD>
AP<I>low</I></TD><TD>
0.4949
</TD></TR><TR><TD>
h
</TD><TD>
0.9678
</TD></TR></TBODY></TABLE><TABLE id="normal" width="60%"><TBODY><TR><TH>
Dependent rate coefficients
</TH><TH>
Function
</TH></TR><TR><TD>
k1
</TD></TR><TR><TD>
k3
</TD></TR><TR><TD>
k5
</TD></TR><TR><TD>
k7
</TD></TR><TR><TD>
k11
</TD></TR></TBODY></TABLE><H1>Ordinary differential equations</H1><H1>Matlab code</H1><DIV align="left"><P>The code below contains the general chemotaxis model</P><DIV id="para1" style="display:none"><PRE class="prettyprint">

%% Function to be called by ode with:
% T=5.3;
% a=2;      %attractant level
% [t1, Y] = ode45(@chemotaxis,[0 180], [0.02036112434, T-0.02036112434, 0.0746125881, 0.28-0.0746125881, 5.268149451, T-5.268149451, 4.33528337, T-4.33528337, 0.1271039686, 4.437396032, 0.8376126615, 8.798676004, (9.7-8.798676004), a]);
% plot(t1, Y(:,13))     % plots YP
%% The y matrix corresponds to the following moleculs 
% y(1) = phosphorylated CheV
% y(2) = CheV
% y(3) = phosphorylated CheB
% y(4) = CheB
% y(5) = methyl-deactivated receptors
% y(6) = methyl-activated receptors
% y(7) = receptors with CheD bound
% y(8) = receptors with no CheD bound
% y(9) = CheC-CheD complexes
% y(10) = CheC
% y(11) = CheD
% y(12) = CheY
% y(13) = phosphorylated CheY
%% ode function
function [y]=chemotaxis(t, c)
y=zeros(14, 1);
y(14)=0;
a=c(14);

ATR=44.9309240600887;
T=5.3;
Ta=(T*a)/(T+0.5);
T0=T-Ta;
R=0.16;

a_high=4.36411705466097;
a_low=0.032607681869315;
a_range=a_high-a_low;
AP_low=0.494939026965836;
AP_high=4.55483185920483;
AP_range=(AP_high-AP_low);

% functions
k1=0.0107958905579173 + 0.098830729088527195456443788034135*a;
k3=1.18688540121874 + 0.11843941511337412617533886830266*a;
k5=0.00966284204027138 + 0.10765089965159632008963846037032*a;
k7=-0.3981345018*a / (-6.11424626 + a) + 55180.480108647040*a / (-4164.059738069920 + a) + 13.200240970105565*a + 0.01860300677358011;

% VP
k2=2.79937406917165;

% TM1
k4=3.21494947776107;
k6=0.745536753581247;

% CD
k8=0.0998062263334537;
k9=0.0477873627865919;
k10=0.113456184494577;
k11_scale=3.97597580910604;
k12=1.39741492483201;

% positive AP increase
positive = ((Ta * a_range)/T) + ((T0*a_range)/(T*ATR));

y(1)=c(2)*k1 - c(1)*k2;
y(2)=c(1)*k2 - c(2)*k1;
 
y(3)=c(4)*k3 - c(3)*k4 - c(3)*c(5)*k5; 
y(4)=c(3)*k4 + c(3)*c(5)*k5 - c(4)*k3;
y(5)=c(6)*R*k6 - c(3)*c(5)*k5; 
y(6)=c(3)*c(5)*k5 - c(6)*R*k6;

y(7)=c(8)*c(11)*k8 - c(7)*k7;
y(8)=c(7)*k7 - c(8)*c(11)*k8;
y(9)=c(10)*c(11)*k9 - c(9)*k12;
y(10)=c(9)*k12 - c(10)*c(11)*k9;
y(11)=c(9)*k12 + c(7)*k7 - c(8)*c(11)*k8 - c(11)*c(10)*k9;

y(12)=-c(12)*(AP_low + ((AP_range)*(positive/a_range)) - (c(1)+c(6)))*k10 + c(13)*(k11_scale*c(9)+0.0850589797402037);
y(13)=c(12)*(AP_low + ((AP_range)*(positive/a_range)) - (c(1)+c(6)))*k10 - c(13)*(k11_scale*c(9)+0.0850589797402037);
</PRE></DIV><P>The code below contains the general chemotaxis model where the temperature can be controlled</P><DIV id="para2" style="display:none"><PRE class="prettyprint">

%% Function to be called by ode with:
% T=5.3;
% a=2;      %attractant level
% Temp=37;     %temperature 
% [t1, Y] = ode45(@chemotaxis_temp,[0 180], [0.02036112434, T-0.02036112434, 0.0746125881, 0.28-0.0746125881, 5.268149451, T-5.268149451, 4.33528337, T-4.33528337, 0.1271039686, 4.437396032, 0.8376126615, 8.798676004, (9.7-8.798676004), a, Temp]);
% plot(t1, Y(:,13))     % plots YP
%% The y matrix corresponds to the following moleculs 
% y(1) = phosphorylated CheV
% y(2) = CheV
% y(3) = phosphorylated CheB
% y(4) = CheB
% y(5) = methyl-deactivated receptors
% y(6) = methyl-activated receptors
% y(7) = receptors with CheD bound
% y(8) = receptors with no CheD bound
% y(9) = CheC-CheD complexes
% y(10) = CheC
% y(11) = CheD
% y(12) = CheY
% y(13) = phosphorylated CheY
%% ode function

function [y]=chemotaxis_temp(t, c)
y=zeros(15, 1);
y(14)=0;
y(15)=0;

a=c(14);
Temp=c(15);

ATR=44.9309240600887;
T=5.3;
Ta=(T*a)/(T+0.5);
T0=T-Ta;
R=0.16;

a_high=4.36411705466097;
a_low=0.032607681869315;
a_range=a_high-a_low;
AP_low=0.494939026965836;
AP_high=4.55483185920483;
AP_range=(AP_high-AP_low);

% functions
k1=0.0107958905579173 + 0.098830729088527195456443788034135*a;
k1=0.0025*k1*Temp^2 - 0.11*k1*Temp + 1.6475*k1;
k3=1.18688540121874 + 0.11843941511337412617533886830266*a;
k3=0.0025*k3*Temp^2 - 0.11*k3*Temp + 1.6475*k3;
k5=0.00966284204027138 + 0.10765089965159632008963846037032*a;
k5=0.0025*k5*Temp^2 - 0.11*k5*Temp + 1.6475*k5;
k7=-0.3981345018*a / (-6.11424626 + a) + 55180.480108647040*a / (-4164.059738069920 + a) + 13.200240970105565*a + 0.01870300677358011;
k7=0.0025*k7*Temp^2 - 0.11*k7*Temp + 1.6475*k7;

% VP
k2=0.0025*2.79937406917165*Temp^2 - 0.11*2.79937406917165*Temp + 1.6475*2.79937406917165;

% TM1
k4=0.0025*3.21494947776107*Temp^2 - 0.11*3.21494947776107*Temp + 1.6475*3.21494947776107;
k6=0.0025*0.745536753581247*Temp^2 - 0.11*0.745536753581247*Temp + 1.6475*0.745536753581247;

% CD
k8=0.0025*0.0998062263334537*Temp^2 - 0.11*0.0998062263334537*Temp + 1.6475*0.0998062263334537;
k9=0.0025*0.0477873627865919*Temp^2 - 0.11*0.0477873627865919*Temp + 1.6475*0.0477873627865919;
k10=0.0025*0.113456184494577*Temp^2 - 0.11*0.113456184494577*Temp + 1.6475*0.113456184494577;

k11_scale=0.0025*3.97597580910604*Temp^2 - 0.11*3.97597580910604*Temp + 1.6475*3.97597580910604;
k11_basal=0.0025*0.0850589797402037*Temp^2 - 0.11*0.0850589797402037*Temp + 1.6475*0.0850589797402037;
k12=0.0025*1.39741492483201*Temp^2 - 0.11*1.39741492483201*Temp + 1.6475*1.39741492483201;

positive = ((Ta * a_range)/T) + ((T0*a_range)/(T*ATR));

y(1)=c(2)*k1 - c(1)*k2;
y(2)=c(1)*k2 - c(2)*k1;
 
y(3)=c(4)*k3 - c(3)*k4 - c(3)*c(5)*k5; 
y(4)=c(3)*k4 + c(3)*c(5)*k5 - c(4)*k3;
y(5)=c(6)*R*k6 - c(3)*c(5)*k5; 
y(6)=c(3)*c(5)*k5 - c(6)*R*k6;

y(7)=c(8)*c(11)*k8 - c(7)*k7;
y(8)=c(7)*k7 - c(8)*c(11)*k8;
y(9)=c(10)*c(11)*k9 - c(9)*k12;
y(10)=c(9)*k12 - c(10)*c(11)*k9;
y(11)=c(9)*k12 + c(7)*k7 - c(8)*c(11)*k8 - c(11)*c(10)*k9;

y(12)=-c(12)*(AP_low + ((AP_range)*(positive/a_range)) - (c(1)+c(6)))*k10 + c(13)*(k11_scale*c(9)+k11_basal);
y(13)=c(12)*(AP_low + ((AP_range)*(positive/a_range)) - (c(1)+c(6)))*k10 - c(13)*(k11_scale*c(9)+k11_basal);


</PRE></DIV><H2>References</H2><FONT size="1,5"><P>[1] David C. Hauri and John Ross. (1995). A Model of Excitation and Adaptation in Bacterial Chemotaxis. Biophysical Journal Volume 68, pp708-722.</P><P>[2] Christopher V. Rao, George D. Glekas2 and George W. Ordal. (2008). Elsevier Ltd.</P><P>[3] Li, M. and G. L. Hazelbauer, “Cellular Stoichiometry of the Components of the
Chemotaxis Signaling Complex,” J. Biol., 186, 3687–3694 (2004).</P><P>[4] Victor Sourjik and Howard C. Berg. (2001). Receptor sensitivity in bacterial chemotaxis. PNAS, vol 99. no 1. pp 123-127.</P><P>[5] Berg, H. C., and D. Brown. 1972.Chemotaxis in Escherichia coli analyzed by three dimensional tracking. Nature. 239:500-504.</P><P>[6] Ordal GW, Gibson KJ. (1977). Chemotaxis toward amino acids by Bacillus subtilis. J Bacteriol. vol 129(1):151-5.</P><P>[7] Rosario, M.L., Kirby, J., Bochar, D., and Ordal, G.W. (1995) Chemotactic methylation and behavior of Bacillus subtilis :role of two unique proteins, CheC and CheC. Biochemistry 34: 3823-3831. </P><P>[8] Thermal Regulation of Membrane Lipid Fluidity by a Two-Component System in Bacillus subtilis* Received for publication, January 3, 2011, and in revised form, January 31, 2011 BIOCHEMISTRY AND MOLECULAR BIOLOGY EDUCATION Vol. 39, No. 5, pp. 362–366, 2011</P><P>[9] Larisa E. Cybulski, Daniela Albanesi, Maria C. Mansilla, Silvia Altabe, Pablo S. Aguillar, &amp; Diego de Mendoza. (2002). Mechanism of membrane fluidity optimization: isothermal control of the Bacillus subtilis acyl-lipid desaturase. Molecular microbiology. Vol 45(5). pp; 1379-1388</P></FONT></DIV></DIV><DIV id="footer">
  iGEM 2013 Groningen



<DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Groningen/Navigation/Heatmotility">http://2013.igem.org/Team:Groningen/Navigation/Heatmotility</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Groningen/Navigation/Heatmotility" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Groningen/Navigation/Heatmotility" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Groningen/Navigation/Heatmotility&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Groningen/Navigation/Heatmotility&amp;oldid=112985" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>