Team:SJTU-BioX-Shanghai/Protocol
Protocol
Contents
1 Transformation of cyanobacteria
1.1 BG11 medium
1.1.1 The content of mother solution
1.1.2 Liquid medium (for 1L)
1.1.3 Solid medium (for 1L)
1.2 Natural Transformation
1.3 Transformation by electroporation
2 Prove transformation success
3 PCR
3.1 Primer Design
3.1.1 Several principles should be followed in primer designing
3.2 PCR Volume:
3.3 PCR Reaction Mixture:
3.4 PCR Conditions:
3.4.1 General PCR (50μl volume):
3.4.1.1 Amplification procedure:
3.4.2 Overlapping PCR(50μl volume):
3.4.2.1 Amplification procedure:
3.4.3 Bacteria liquid PCR(20μl volume):
3.4.3.1 Amplification procedure:
3.5 Problem Solving:
4 Agarose Gel Electrophoresis Analyses
5 Purification & Gel Extraction
6 LB (agar) medium preparation
7 Restriction Enzyme Reaction & Ligation
7.1 Choosing Enzyme and Cleavage Site:
7.2 Recommended Reaction Component:
7.3 Reaction Protocol:
8 Transformation in Escherichia coli
8.1 Preparing Competent Cell:
8.2 Transformation:
8.3 Blue-white Screen:
8.3.1 Protocol:
8.4 Inoculation:
9 Plasmid Extraction
10 Immunohistochemistry
Transformation of cyanobacteria
BG11 medium
The content of mother solution
Mother liquors
substance
content(g/L)
For 100mL （g）
1
K2HPO4·3H2O
40
4
2
MgSO4·7H2O
75
7.5
3
Ferric ammonium citrate
6
0.6
Citrate acid
6
0.6
4
CaCl2·2H2O
36
3.6
5
EDTA-Na2
1
0.1
6
H3BO3
2.86
0.286
MnCl2·4H2O
1.81
0.181
ZnSO4·7H2O
0.222
0.0222
NaMoO4·5H2O
0.39
0.039
CuSO4·5H2O
0.079
0.0079
Co(NO3)2·6H2O
0.04
0.004
Liquid medium (for 1L)
1ml of each mother liquors, besides, add
NaNO3
1.5g
Na2CO3
0.02g
Adjust pH to 7.5-7.55, 121℃ autoclaved for 20mins.
Solid medium (for 1L)
1ml of each mother liquors, besides, add
NaNO3
1.5g
Na2CO3
0.02g
Na2S2O3
3g
TES
2.22
Agar
1.5% Which means 15g
Adjust pH to 7.5-7.55,121℃ autoclaved for 20mins. Prepare solid plate before use. Be sure the antibiotic is active and in appropriate concentration.
Natural Transformation
Time
Steps
Day 1 at noon
Set the culture，the initial OD730 is 0.1
Day 2 in the evening
After 1.5 days，the OD730 would approach to about 0.5（even though 0.3-0.7 is ok）.
Centrifuge at RT*3000g*10min，remove the supernatant，add 1ml liquid medium，measure the od730 by nanodrop, then calculate the volume of cyanobacteria to adjust the ultimate od730 to 2.5.
Calculate the volume of DNA（5-10ug，usually take 8ug）
Mix the DNA, cyanobacteria and liquid medium in a 15 ml centrifuge tube to make the total volume is 500ul and the od730 is about 2.5, set a negative control (without DNA).
Overnight and then shake slightly by hand for about 4 times.（every 2-3 hours）
Day 3 at afternoon
Take 200ul culture, spread it on solid plate with suitable    antibiotic, put the plate into illumination incubator at 30℃.
Then wait for several days (depends on the gene)
If the solid plate with DNA have colony while the negative plate is clean, then we may conclude the natural transformation is successful, but we need three experiments for further determination.
1
Pick up the colony，spread it in a solid plate (with antibiotic) in parallel, put it in illumination incubator to store the colony. (Not all the colony can grow because this can work as segregation.)
2
Pick up the colony
Set a 500ul liquid culture, then add it into an 100ml flask for    further segregation.
Spread it on solid plate in parallel to store the    cyanobacteria.
Set a 50ul liquid culture to do colony PCR
Colony PCR：put the 50ul liquid culture into liquid N2, then into 80℃ water, repeat these steps for another 2 times, take 0.5ul to do colony PCR.
2*Taq enzyme mix
5ul
Primer
0.5ul*2
Template（the liquid culture）
0.5ul
ddH2O
3.5ul
PCR program：
95℃
3-4min
95℃
30s
35 times
60℃（ADS, the temperature depends on your primer）
30s
72℃（for taq enzyme）
2min（depends on the length of your gene）
72℃
5min
4℃
Forever
Run gel to    determination.
Transformation by electroporation
Day 1 at noon
Set the culture.
Prepare the electric switch and electric tubes, wash the tubes    with ddH2O for several times, then wash with 75% ethanol, put them    in a beaker, covering with 75% ethanol.
Day 3 in the evening or at the afternoon
Take out the    electric tubes from ethanol, put them into 60℃ oven for at least    2 hours to dry the tube, sterilize them by UV before    electrotransformation.
Centrifuge at 8000rpm*5min (the OD730 is about 0.4), dilute the    cell with sterilized ddH2O (40,30,30,20,20ml separately) wash it    for 5 times. (suggest 7 times at first time)
Dilute the cyanobacteria with 200ul sterilized ddH2O, mix it    with DNA (at least 500ng) to make the total volume to 50ul，add    them into electric tube, tunk for several times so that the liquid    can reach to the bottom.
Put the electric tube in electric switch correctly, choose Ec1    program, press “pulse”, wait for a moment, check the voltage    and time. (The best condition is 12kV & 5ms, but we take 18kV & 4-6ms as well.) Add 1ml liquid BG11 culture immediately, mix    and add back to 50ml BG11 culture, put it in shaking illumination    incubator at 30℃*130rpm for 1day.
Day 4 in the evening
Collect the cyanobacteria at 8000rpm*5min, dilute the cell with    400ul liquid culture, mixed with 5ml soft agar BG11 medium, spread    on solid medium (antibiotic added), put the plate into    illumination incubator at 30 ℃.
Wait for several days（depends on your gene）, then test your colony just like natural transformation.
Prove transformation success
Three experiments to prove the success of transformation.
1
Pick up the colony，spread it in a solid plate (with antibiotic) in parallel, put it in illumination incubator to store the colony. (Not all the colony can grow because this can work as segregation.)
2
Pick up the colony
Set a 500ul liquid culture, then add it into an 100ml flask for    further segregation.
Spread it on solid plate in parallel to store the    cyanobacteria.
Set a 50ul liquid culture to do colony PCR
Colony PCR：put the 50ul liquid culture into liquid N2, then into 80℃ water, repeat these steps for another 2 times, take 0.5ul to do colony PCR.
2*Taq enzyme mix
5ul
Primer
0.5ul*2
Template（the liquid culture）
0.5ul
ddH2O
3.5ul
PCR program：
95℃
3-4min
95℃
30s
35 times
60℃（ADS, the temperature depends on your primer）
30s
72℃（for taq enzyme）
2min（depends on the length of your gene）
72℃
5min
4℃
Forever
Run gel to    determination.
PCR
Primer Design
Several principles should be followed in primer designing
1. The length of primers should be limited between 20~60bp. For overlapping PCR, the overlapping domain should contains at least 15bp. 2. GC% should be at around 40%~60%. 3. Before synthesis, blast the primer using software like Vector NTI or other methods to check the specificity. The primers are synthesized by GeneWise. After the arriving of the primers, perform a quick spin and then dissolve the DNA powder using ddwater in the given volume.
PCR Volume:
If the PCR product will be used for testing, we suggest that the volume should be under 20 μl. If the PCR product will be used for any downstream operation (eg. Purification, gel extraction, restriction enzyme digestion), higher volume such as 50μl should be used. And for every sample, there should be at least two parallels. So, mix the total mixture in a 1.5ml centrifuge tube beforehand and then separate the desired volume mixture into each PCR tube is recommended.
PCR Reaction Mixture:
We used 2×PrimeSTAR Mix from TAKARA and 2×Taq Master Mix from LifeFeng for PCR reaction, which have already contained buffer, magnesium, dNTP and polymerase. The amount of primer should be at around 10μM and the total amount of template should be under 200ng as the protocols require (less template is needed if use genomic DNA or plasmid DNA as template, see specifically in the protocols of the products). Additionally, the bacteria liquid template must not be too much or else small fragments of bacteria may cause a jam while performing gel analysis. And pay attention to the overlapping PCR: the amount of the two templates should be equal and the primer is added later (see specifically at the example “Overlapping PCR”).
PCR Conditions:
Denaturing conditions-94℃～98℃ for 5~15 seconds. For bacteria liquid PCR, longer denaturing time (10~15minutes) is required for completely cell splitting. Annealing temperature-the annealing temperature is usually 3~5℃ lower than the melting temperatures of primers. If the melting temperatures of two primers differ greatly (for example, >5℃), try gradient PCR to find out the appropriate annealing temperature. Here are examples of the three kinds of PCR that we used in the experiments:
General PCR (50μl volume):
PCR reaction mixture
Forward primer
1μl (10μM)
Reverse primer
1μl (10μM)
Template
<200ng
2×PrimeSTAR Mix
25μl
ddwater
Fill up to 50μl
Amplification procedure:
98℃ (pre-denaturation)
3min
98℃(denaturation)
10s
×30~35 cycle
Annealing temperature
5s
72℃(extension)
5s/kb
72℃
3min
12℃
∞
Overlapping PCR(50μl volume):
Overlapping PCR is a commonly used method to link two fragments together. The two templates contain complementary sequence and can perform as primers and templates for each other. In this way the two fragments are linked together in the first 5~8 cycle. Then add the primers for the whole segment amplification and we can acquire the target sequence of a considerable amount.
PCR reaction mixture
Forward primer
1μl (10μM)
Reverse primer
1μl (10μM)
Template1
<200ng totally
Template2
2×PrimeSTAR Mix
25μl
ddwater
Fill up to 50μl
Amplification procedure:
98℃ (pre-denaturation)
3min
98℃(denaturation)
10s
×5~8 cycle
Annealing temperature
5s
72℃(extension)
5s/kb
72℃
3min
12℃
∞
(add primers)
98℃ (pre-denaturation)
3min
98℃(denaturation)
10s
×30~35 cycle
Annealing temperature
5s
72℃(extension)
5s/kb
72℃
3min
12℃
∞
Bacteria liquid PCR(20μl volume):
Bacteria liquid PCR or colony PCR is used for detecting whether the colony contains the desired plasmid or not. It can lessen the workload when extracting the plasmid, however, the case of false positives is unavoidable (decreasing cycle number may help to reduce the false positive, but we didn’t do any test to prove it). Variety kinds of examinations should be performed before sequencing. You can see that specifically in “Restriction Enzyme Reaction” and “Blue-white Screen”.
PCR reaction mixture
Forward primer
1μl (10μM)
Reverse primer
1μl (10μM)
Template(bacteria liquid)
1μl
2×Taq Mix
10μl
ddwater
Fill up to 20 μl
Amplification procedure:
95℃ (pre-denaturation)
10~15min
94℃(denaturation)
10~30s
×20~25 cycle
Annealing temperature
10~30s
72℃(extension)
1min/kb
72℃
3min
12℃
∞
Problem Solving:
1. Low concentration: increase the cycle number/decrease the annealing temperature. 2. Unspecific amplification: increase the annealing temperature/decrease the cycle number/use less template/increase the cycle number of the first round in overlapping PCR/perform gel extraction to get the desired fragment.
Agarose Gel Electrophoresis Analyses
1. Weight the appropriate amount of agarose powder. In our experiments，we used 1.0% gel for DNA fragments that between 200bp~7000bp, 2.0% gel for fragments that between 100bp~200bp, and 2.5% gel for fragments that under 100bp. Furthermore, if the gel is used for gel extraction or other operation requiring completely separation of different fragments, higher concentration should be used. 2. Dissolve the agarose powder in TBE solution buffer by heating it in microwave for about 5 minutes until no solid material can be seen. 3. Add EB solution (5μl/100ml) after the agarose solution cooling down. 4. Place the comb into the gel mold. Apply the agarose solution to the mold and wait for 30mins~1h to get the solid agarose gel. Attention: being exposed under light for too long can cause the degradation of EB. So put the ready-made gels into a box with TBE solution buffer as storage. 5. Mix the DNA sample with 6× loading buffer or 10× loading buffer (to denature the enzymes) or use reaction buffer that already contained loading buffer. 6. Load the DNA sample to the gel. Set voltage and time to 130~160V and 30~15mins. 7. Check the result and capture the figure in UV box.
Purification & Gel Extraction
PCR product and restriction enzyme digestion product that will be used for downstream operation should be purified or extracted from gel for better quality. We used Omega E.Z.N.A. Cycle-Pure Kit(http://omegabiotek.com/store/product/e-z-n-a-cycle-pure-kit-2/) and Omega Gel Extraction Kit(http://omegabiotek.com/store/product/e-z-n-a-gel-extraction-kit/) to perform purification or gel extraction and the given protocol was followed. Here are a few additional notes: 1. After applying the solution that contains DNA sample to the column and centrifuging, reload the sample to the column and centrifuge for a second time in order to make sure that all the DNA samples are bound on the column. 2. Use water instead of elution buffer to elute, for that TE buffer may cause negative effect on enzyme reaction. Heat the ddwater to 50~60℃ in water bath to ensure that the pH of the ddwater is around 7.6. But if the DNA sample will be stored for a very long time, using elution buffer is better.
LB (agar) medium preparation
We used Sangon biotech LB broth medium(25.0g/L) or Sangon biotech LB broth agar(40.0g/L), which already contains yeast extraction, tryptone, sodium chloride (and agar) so that we only need to dissolve the powder in proportion to prepare LB (agar) medium. For LB medium using for bacteria inoculation, split the medium into 5ml and transfer the 5ml medium respectively into 15ml tubes before autoclaving. Autoclave under 121℃ for 30 minutes. As for LB agar medium, after the temperature of the medium cooling down until it doesn’t hurt, add appropriate volume of antibiotic into the medium. Transfer the medium into 90mm plate afterwards. Approximately 15ml of LB agar medium is applied to each 90mm plate. Here are the antibiotics and the corresponding concentration that we used in our experiments:
Antibiotic
Storage Concentration
Working Concentration
Stringent Plasmid
Relaxed Plasmid
Ampicillin(Amp)
50mg/ml
                      50μg/ml
Chloramphenicol(Cm)
34mg/ml
34μg/ml
170μg/ml
Restriction Enzyme Reaction & Ligation
Choosing Enzyme and Cleavage Site:
1. Use enzymes that produce sticky end instead of those produce blunt end. 2. Double digestion is better than single digestion because the latter can cause a self-ligation problem. The two cleavage sites should not be too close to each other for that it may be too crowded for two kinds of enzymes to bind onto the DNA together. We faced this problem in our experiments. The fact is, though the two cleavage sites are linked, double digestion is still the most efficient way. So use double digestion as possible. 3. If denaturation is needed, choose two kinds of enzyme that have the same denaturation protocol as possible. 4. Do not use isocaudarner to digest plasmid. Instead, it’s convenient to link fragments together in this way.
Recommended Reaction Component:
We used restriction endonuclease and T4 ligase from Thermo Scientific.
Digestion reaction
Plasmid DNA
PCR product
Genomic DNA
10×buffer
2μl
2μl
5μl
DNA
1μg
0.2μg
5μg
FastDigest enzyme
1μl
1μl
5μl
Water, nuclease-free
Fill up to 20μl
Fill up to 30μl
Fill up to 50μl
Total volume
20μl
30μl
50μl
Ligation
10× buffer
1μl
Vector DNA
50~200ng
Insert DNA
5 times the mass of vector DNA
Ligase
1Weiss U
ddwater
Fill up to 10μl
Total volume
20μl
* The digestion reaction volume can be scaled up. However, we recommend that the total volume should be up to 50μl to ensure the higher efficiency. And for the concentration of the ligation product, the volume of the ligation reaction should be up to 20μl. The insert DNA can be 1~10 times the mass of the vector DNA, but 5 times the mass is the best.
Reaction Protocol:
1. Mix the reaction components completely and gently, except the restriction endonuclease or ligase. Put the enzymes on the ice. Heat the water bath to 37℃ beforehand. 2. Add the enzymes and mix gently by pipetting. 3. Put the reaction mixture into the water bath and react for 20~60mins (different enzymes may need different heating time). As for T4 ligase, put the mixture under 4℃ for 24h or under room temperature for 4h. 4. Use Thermodynamic denaturation or purification to make the sample be capable for downstream operations. Additionally, load the 10× loading buffer directly into the sample can also denature the enzymes.
Transformation in Escherichia coli
Preparing Competent Cell:
1. Suck up 1μl bacterial liquid DH5α or pick a colony from the plate and leave the bacteria into 5ml LB medium. Culture overnight in shaker. With temperature of 37℃ and rate of 220rpm. 2. Transfer 500μl bacterial liquid into an Erlenmeyer flask containing 100ml LB medium. Cult ure until the liquids’ OD600 is between 0.4~0.5. 3. Take the bacterial liquid out from the shaker. Put it on ice for 20mins. 4. Discard the supernatant. Resuspend the cell with 10ml CaCl2. Put the liquid on ice for 20mins. 5. Centrifuge again for 5mins at 8000rpm. 6. Discard the supernatant and resuspend the cell with 1700μl CaCl2 and 300μl glycerol. 7. Split the competent cell into 100μl. Store under -70℃.
Transformation:
1. Get the competent cell out from -80℃ refrigerator and put it immediately on ice. 2. Add 1μl plasmid or 5μl recombination product into 100μl competent cell. Mix completely and gently. 3. Put the mixture on ice for 30mins. 4. Heat shock: put the mixture in 42℃ water bath for 90s. 5. Get the mixture out from the water bath and put it immediately on ice for 2mins. 6. Recovery: add 500μl LB medium, culture for 1~2h in shaker. Of temperature 37℃ and rate 220rpm. 7. Spread the bacterial liquid onto the plate. Culture under 37℃ overnight.
Blue-white Screen:
The ligation may not be efficient enough so that we need screening method to improve accuracy. Most vector has the regularity sequence and the sequence encoding first 146 amino acids of lacZ, which can encode β-galactosidase. And β-galactosidase can make colonies blue with chromogenic substrate X-gal and inducer IPTG. Inside the sequence there is a MCS. If the target sequence inserts successfully into the MCS, and is long enough to destroy the ORF of lacZ, the colonies will remain white. Therefore, we can easily distinguish the colonies which do not get the right plasmid transformed.  Certainly, there are several situations that may lead to false positive or wrong judgement. For example, non-target sequence may insert into the vector, or, X-gal may not color enough after 12 hours’ cultivation.
Protocol:
1. Spread 40μl 20mg/ml X-gal (final concentration 2%) and 7μl 200mg/ml IPTG (final concentration 20%) on the plate. 2. Spread the bacterial liquid after X-gal and IPTG dry fully. 3. Culture the plate for 12h under 37℃ 4. If the colonies do not color obviously, put the plate into the 4℃ refrigerator for several hours. 5. Pick the white colony and inoculate.
Inoculation:
1. Prepare 15ml test tube, add 5ml LB medium and 5μl Amp (final concentration 50μg/ml) into the tube. The volume of the medium should not exceed 1/3 volume of the tube, or else the bacteria may be deprived of oxygen.
2. Pick the selected colony by the tip and leave the tip directly into the tube. Be careful that the selected colony should be white and round, without many satellite colonies around. Make sure that the picked colony is a single colony.
3. Shake overnight. With temperature of 37℃ and rate of 220rpm.
Plasmid Extraction
We used Biomiga Plasmid Mini-prep Kit (http://www.biomiga.com/index.php?cPath=35_61&xtt=2) and Tiangen Endo-free Maxi Plasmid Kit (http://www.tiangen.com/?productShow/t1/1/id/39.html) to operate plasmid extraction. The protocols are followed and here are a few additional tips that we performed: 1. Resuspend bacteria with buffer containing RNase by pipetting instead of vortex or other sharp methods, which may cause negative effect on enzymes. 2. After applying the solution that contains DNA only to the column and centrifuging, reload the sample to the column and centrifuge for a second time in order to bind all the DNA sample on the column. 3. Use water instead of elution buffer to elute, for that TE buffer can cause negative effect on enzyme reaction. Heat the ddwater to 50~60℃ in water bath to ensure the pH of the ddwater before use. But if the plasmid will be stored for a very long period of time, using elution buffer is better.
Immunohistochemistry
To examine whether the HR expressed successfully onto the cell membrane, we designed a special immunohistochemical protocol for cyanobacteria. For that the C terminal of the HR sequence we designed contains a His-tag, we used anti-His mAb as first antibody and goat anti-mouse Alexa Fluor 488(green fluorescence) as second antibody. And if a loop of green fluorescence can be seen under the fluorescence microscope, we will ensure that the HR expressed successfully onto the cell membrane:
Prepare 1ml bacteria liquid (including samples and controls), which has been cultured to OD730=0.6 and induced 12~16h.
Centrifuge at 8000rpm for 5min.
Resuspend the bacteria with first antibody and PBS, with proportion of 1:50~1:100.
Incubate at 4℃ overnight or at 37℃ for 2h.
Wash 5 times with 1ml PBS. Put the mixture at RT for 5min and then centrifuge at 8000rpm for 4min.
Resuspend the bacteria with second antibody and PBS, with proportion of 1:200~1:1000.
Repeat step 5.
Resuspend the bacteria with 20μl PBS.
Prepare the slide. Add 5μl of the final bacteria liquid.
Observe under the fluorescence microscope.
Bio-X Institutes
Shanghai Jiao Tong University
800, Dongchuan Road 200240, Shanghai, China
kariny888@sjtu.edu.cn
