"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Wash U/French/Biological Parts
From 2009.igem.org
Inicio
Equipe
Etudiants
Faculte
Conseillers
Ecole
Project
Resume
Analyse
Resultats
Conclusion
References
Biological Parts
Parties
Caracterisation
Modelisation
Protocole
Procedures
Recettes
Bloc-Notes
Securite
Securite
Ethique
Avenir
Photos
Diaporama
Planification
Laboratoire
Gens
Bacteries
Sponsors
Commentaires
Parties
Component
Description
Part/Accession #
Base Pairs
Plasmid
Resistance
Well
RBS
Ribosomal Binding Site
BBa_B0034
12
pSB1A2
Ampicillin
plate 1, 2M
Red Light Sensor
description
BBa_I15010
2,238
pSB2K3
Kanamycin
N/A
OmpR (E. coli)
description
NP_417864.1
720
pSB1T3
Tetracycline
N/A
Terminator
Stops Transcription
BBa_B0015
129
pSB1AK3
Ampicillin
and Kanamycin
plate 1, 23L
OmpR + Terminator
description
sequence
916
pany-amp
Ampicillin
synthesized
OmpC promoter
description
BBa_R0082
108
pSB1A2
Ampicillin
plate 1, 16K
puc B/A
description
sequence
375
pSB1A3
Ampicillin
N/A
puc B
description
YP_353390
156
?
?
N/A
puc A
description
YP_353391
165
?
?
N/A
OmpC promoter+BA
description
sequence
539
pany-kana
Kanamycin
synthesized
Light Response System
description
BBa_M30109
4,333
?
Ampicillin
N/A
TetR repressible
description
BBa_J13002
74
pSB1A2
Ampicillin
plate 1, 13B
Green Flourescent Protein
Marker for successful transformation
BBa_E0240
976
pSB1A2
Ampicillin
plate 1, 12M
Haut de Page
Caracterisation
Haut de Page
Modelisation
pucBA Expression Model Diagram
pucBA Model Equations
pucBA Model Reactions
Test Simulation Output
Modeling the Gene Regulatory Network
Our group seeks to assess the optimality of the synthetic system that modulates pucB/A gene expression and LH2 complex assembly in Rhodobacter sphaeroides.  Here we employ a mathematical model of this system to generate predictions about the behavior of the active system in response to light input. Features of the system that the model may help investigate include the time scale of response to light signals, the robustness of the system in response to fluctuations in light intensity, and the translation between changes in gene expression and the absorbance spectrum of the engineered cells.
Though the context of the model can extend back to the transcription of PrrA/B genes involved in integration oxygen and light signals, a preliminary testing model was developed using assumptions of certain initial conditions to isolate the light signal's effect.  Since Cph8 and OmpR are located on the same transcript downstream of the puc promoter region, it was assumed that their associated protein and mRNA had already reached steady state concentrations.  Moreover, the concentrations of the factors were assumed to be equal at this state.  The model whose diagram was constructed in the Simbiology Toolbox distributed by MathWorks details key reactions leading to the translation of the pucB/A genes.  The reaction rate equation used for the phosphorylation of OmpR as a consequence of the light signal reaching Cph8 bound to OmpR is capture in a modified form of Michaelis-Menten kinetics.  A logic function that corresponds to light ON/OFF (1/0) multiplies the maximum reaction rate in the numerator.  Thus, the model assumes that no phosphorylation occurs by this mechanism in the absence of light.  The OmpC promoter binding equation was based on the Hill Equation for a Repressor (1).
Component characterization steps and literature searches are underway in order to obtain quantitative parameters for the reaction rates.  In order to simulate behavior of the system, putative values were included that exaggerate true concentrations and time scales.  OmpR was given an initial concentration normalized to one, and all other components were assumed insignificant initially to this value. An ideal light pulse was introduced at an instant and removed thirty simulation seconds later. From this rudimentary simulation it can be drawn that the nonlinearities of the phosphorylation and transcription factor binding kinetics effectively smooth the sharp light input.  By design, the light switch ON yields phosphorylation of OmpR and repression of the pucB/A genes which would give rise to LH2.  Conversely, when left OFF, the concentration of pucB/A recovers and increases until the steady state determined by its translation and degradation rates.
References
1.  Alon, Uri. Introduction to systems biology and the design principles of biological networks. Boca Raton, FL: Chapman & Hall, 2006.
2.  Bower, James M. Computational Modeling of Genetic and Biochemical Networks (Computational Molecular Biology). New York: M.I.T. PRESS, 2001.
3. System modeling in cellular biology from concepts to nuts and bolts. Cambridge, MA: MIT P, 2006.
Haut de Page
Retrieved from " http://2009.igem.org/Team:Wash_U/French/Biological_Parts "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
