<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Wash_U_Biological_Parts"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Wash_U/Biological_Parts">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Wash_U/Biological_Parts&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Wash_U/Biological_Parts&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Wash_U/Biological_Parts&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Wash_U/Biological_Parts" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Wash U/Biological Parts</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><UL class="cssMenu cssMenum"><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U"><B>Home</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Team"><SPAN><B>Team</B></SPAN></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Team#Students"><B>Students</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Team#Faculty"><B>Faculty</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Team#Advisors"><B>Advisors</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Team#University"><B>University</B></A></LI></UL><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Project"><SPAN><B>Project</B></SPAN></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Project#Introduction"><B>Introduction</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Project#Organism"><B>Organism</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Project#Regulation"><B>Regulation</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Project#Results"><B>Results</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Project#Analysis"><B>Analysis</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Project#Conclusion"><B>Conclusion</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Project#References"><B>References</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Biological_Parts"><SPAN><B>Biological Parts</B></SPAN></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Biological_Parts#Parts"><B>Parts</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Biological_Parts#Characterization"><B>Characterization</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Biological_Parts#Modeling"><B>Modeling</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Protocol"><SPAN><B>Protocol</B></SPAN></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Protocol#Procedures"><B>Procedures</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Protocol#Recipes"><B>Recipes</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Notebook"><SPAN><B>Notebook</B></SPAN></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Safety"><SPAN><B>Safety</B></SPAN></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Safety#Safety"><B>Safety</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Safety#Sharing"><B>Sharing</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Pictures"><SPAN><B>Pictures</B></SPAN></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Pictures#Slideshow"><B>Slideshow</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Pictures#Planning"><B>Planning</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Pictures#Lab"><B>Lab</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Pictures#People"><B>People</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Pictures#Bacteria"><B>Bacteria</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Credits"><SPAN><B>Credits</B></SPAN></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Credits#Sponsors"><B>Sponsors</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Credits#Acknowledgements"><B>Acknowledgements</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Credits#Links"><B>Links</B></A></LI><LI class=" cssMenui"><A class="  cssMenui" href="https://2009.igem.org/Team:Wash_U/Comments"><SPAN><B>Comments</B></SPAN></A></LI></P><P>
The Registry of Standard Biological Parts is a library of DNA sequences combined with online characterization resources.  The Registry has created a standard protocol for making segments of DNA compatible with all other segments of DNA, regardless of order or size.  BioBrick is the term given to such segments of DNA, a term alluding to the fact that any number of bricks may be combined in any order to produce complex, unique systems.  This is accomplished by standardizing the restriction enzymes used to surround BioBricks, as well as the plasmids used to transform them.  For a graphical representation of of the process, please click <A href="http://ginkgobioworks.com/support/BioBrick_Assembly_Manual.jpg" class="external text" rel="nofollow">here</A>.  A powerful online database provides information and characterization of all of the BioBricks in the Registry and uses the wiki format (the same one used in wikipedia) which encourages others to edit content directly on the page.  Below are list of parts that were used/created for this project. 
</P><DIV style="text-align: left;"><FONT size="4" style="color:black"><H2><SPAN class="mw-headline" id="Parts"><B>Parts</B></SPAN></H2><CENTER><P><FONT size="2">
Parts used to characterize and build our final project
</FONT></P><TABLE cellpadding="9" cellspacing="0" style="background-color:#FFEFD5;" border="1"><TBODY><TR><TD><B>Component</B></TD><TD><B>Part/Accession #</B></TD><TD><B>Base Pairs</B></TD><TD><B>Plasmid</B></TD><TD><B>Resistance</B></TD><TD><B>Well</B></TD></TR><TR><TD>RBS-34
</TD><TD><A href="http://partsregistry.org/Part:BBa_B0034" class="external text" rel="nofollow">BBa_B0034</A></TD><TD>12
</TD><TD>pSB1A2
</TD><TD>Ampicillin
</TD><TD>plate 1, 2M
</TD></TR><TR><TD>Cph8
</TD><TD><A href="http://partsregistry.org/Part:BBa_I15010" class="external text" rel="nofollow">BBa_I15010</A></TD><TD>2,238
</TD><TD>pSB2K3
</TD><TD>Kanamycin
</TD><TD>N/A
</TD></TR><TR><TD>RFP
</TD><TD><A href="http://partsregistry.org/Part:BBa_J04051" class="external text" rel="nofollow">BBa_J04051</A></TD><TD>720
</TD><TD>N/A
</TD><TD>N/A
</TD><TD>N/A
</TD></TR><TR><TD>OmpR (E. coli)
</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K098011" class="external text" rel="nofollow">BBa_K098011</A></TD><TD>720
</TD><TD>pSB1T3
</TD><TD>Tetracycline
</TD><TD>N/A
</TD></TR><TR><TD>OmpR (R. sphaeroides)
</TD><TD><A href="http://partsregistry.org/Part:BBa_K227010" class="external text" rel="nofollow">BBa_K227010</A></TD><TD>720
</TD><TD>New</TD><TD>New
</TD><TD>New
</TD></TR><TR><TD>Terminator
</TD><TD><A href="http://partsregistry.org/Part:BBa_B0015" class="external text" rel="nofollow">BBa_B0015</A></TD><TD>129
</TD><TD>pSB1AK3
</TD><TD>Ampicillin and Kanamycin
</TD><TD>plate 1, 23L
</TD></TR><TR><TD>RBS +OmpR(sph) + Terminator
<P>includes prefix and suffix
</P></TD><TD><A href="/wiki/images/1/17/OmpR_%2B_terminator.txt" class="internal" title="OmpR + terminator.txt">sequence</A><P><A href="http://partsregistry.org/Part:BBa_K227011" class="external text" rel="nofollow">BBa_K227011</A></P></TD><TD>875/916
</TD><TD>pSB1k3
</TD><TD>Kanamycin
</TD><TD>synthesized
</TD></TR><TR><TD>OmpC promoter
</TD><TD><A href="http://partsregistry.org/Part:BBa_R0082" class="external text" rel="nofollow">BBa_R0082</A></TD><TD>108
</TD><TD>pSB1A2
</TD><TD>Ampicillin
</TD><TD>plate 1, 16K
</TD></TR><TR><TD>puc promoter
</TD><TD><A href="http://partsregistry.org/Part:BBa_K227007" class="external text" rel="nofollow">BBa_K227007</A></TD><TD>651
</TD><TD>pSB1k3
</TD><TD>Kanamycin
</TD><TD>New
</TD></TR><TR><TD>puc BA
</TD><TD><A href="http://partsregistry.org/Part:BBa_K227006" class="external text" rel="nofollow">BBa_K227006</A></TD><TD>336
</TD><TD>pSB1k3
</TD><TD>Kanamycin
</TD><TD>New
</TD></TR><TR><TD>puc B
</TD><TD><A href="http://partsregistry.org/Part:BBa_K227005" class="external text" rel="nofollow">BBa_K227005</A></TD><TD>156
</TD><TD>pSB1k3
</TD><TD>Kanamycin
</TD><TD>New
</TD></TR><TR><TD>puc A
</TD><TD><A href="http://partsregistry.org/Part:BBa_K227004" class="external text" rel="nofollow">BBa_K227004</A></TD><TD>165
</TD><TD>pSB1k3
</TD><TD>Kanamycin
</TD><TD>New
</TD></TR><TR><TD>OmpC promoter+pucBA
</TD><TD><A href="/wiki/images/2/23/OmpC_promoter_%2B_puc_BA.txt" class="internal" title="OmpC promoter + puc BA.txt">sequence</A><P><A href="http://partsregistry.org/Part:BBa_K227008" class="external text" rel="nofollow">BBa_K227008</A></P></TD><TD>492/539
</TD><TD>pSB1k3
</TD><TD>Kanamycin
</TD><TD>synthesized
</TD></TR><TR><TD>Green Fluorescent Protein
</TD><TD><A href="http://partsregistry.org/Part:BBa_E0240" class="external text" rel="nofollow">BBa_E0240</A></TD><TD>876
</TD><TD>pSB1A2
</TD><TD>Ampicillin
</TD><TD>plate 1, 12M
</TD></TR></TBODY></TABLE><P>
Plasmids used to create and characterize our project
</P><TABLE cellpadding="9" cellspacing="0" style="background-color:#FFEFD5;" border="1"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>Base Pairs</B></TD><TD><B>Resistance</B></TD><TD><B>Copy Number</B></TD></TR><TR><TD><A href="http://partsregistry.org/Part:pSB1A2" class="external text" rel="nofollow">pSB1A2</A></TD><TD>2,079
</TD><TD>Ampicillin
</TD><TD>high
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:pSB1K3" class="external text" rel="nofollow">pSB1K3</A></TD><TD>2,206
</TD><TD>Kanamycin
</TD><TD>high
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A></TD><TD>2,157
</TD><TD>Ampicillin
</TD><TD>high
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:pSB2K3" class="external text" rel="nofollow">pSB2K3</A></TD><TD>4,425
</TD><TD>Kanamycin
</TD><TD>variable
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:pSB1T3" class="external text" rel="nofollow">pSB1T3</A></TD><TD>2,463
</TD><TD>Tetracycline
</TD><TD>high
</TD></TR><TR><TD>pRKCBC3
</TD><TD>~11.5kb
</TD><TD>Tetracycline
</TD><TD>4
</TD></TR></TBODY></TABLE><FONT size="2"><P>Parts submitted to the Registry of Standard Biological Parts
</P><TABLE cellpadding="9" cellspacing="0" style="background-color:#FFEFD5;" border="1"><TBODY><TR><TD><B>Part/Accession #Component</B></TD><TD><B>Component</B></TD><TD><B>Type</B></TD><TD><B>Base Pairs</B></TD><TD><B>Plasmid</B></TD><TD><B>Resistance</B></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_I15010" class="external text" rel="nofollow">BBa_I15010</A></TD><TD>Cph8 (planned resubmission)
</TD><TD>Coding
</TD><TD>2,238
</TD><TD>plasmid
</TD><TD>Resistance
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K227004" class="external text" rel="nofollow">BBa_K227004</A></TD><TD>puc A
</TD><TD>Coding
</TD><TD>165
</TD><TD>pSB1K3
</TD><TD>Kanamycin
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K227005" class="external text" rel="nofollow">BBa_K227005</A></TD><TD>puc B
</TD><TD>Coding
</TD><TD>156
</TD><TD>pSB1K3
</TD><TD>Kanamycin
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K227006" class="external text" rel="nofollow">BBa_K227006</A></TD><TD>puc BA
</TD><TD>Coding
</TD><TD>336
</TD><TD>pSB1K3
</TD><TD>Kanamycin
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K227007" class="external text" rel="nofollow">BBa_K227007</A></TD><TD>puc promoter
</TD><TD>Regulatory
</TD><TD>651
</TD><TD>pSB1K3
</TD><TD>Kanamycin
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K227008" class="external text" rel="nofollow">BBa_K227008</A></TD><TD>ompC+PucBA (synthesized)
</TD><TD>Composite
</TD><TD>492
</TD><TD>pSB1AT3
</TD><TD>Ampicilin,Tetracycline
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K227009" class="external text" rel="nofollow">BBa_K227009</A></TD><TD>PucPromotor+GFP
</TD><TD>Composite
</TD><TD>1377
</TD><TD>pSB1A2
</TD><TD>Ampicilin
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K227011" class="external text" rel="nofollow">BBa_K227011</A></TD><TD>RBS34+OmpR(sph)+Term (synthesized)
</TD><TD>Composite
</TD><TD>875
</TD><TD>pSB1K3
</TD><TD>Kanamycin
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K227012" class="external text" rel="nofollow">BBa_K227012</A></TD><TD>RBS34+OmpR(sph)+Term+OmpC+PucB/A
</TD><TD>Composite
</TD><TD>1375
</TD><TD>pSB1K3
</TD><TD>Kanamycin
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K227013" class="external text" rel="nofollow">BBa_K227013</A></TD><TD>ompCpro + GFP
</TD><TD>Composite
</TD><TD>992
</TD><TD>pSB1A2
</TD><TD>Ampicilin
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K227014" class="external text" rel="nofollow">BBa_K227014</A></TD><TD>pucpro+pucBA
</TD><TD>Composite
</TD><TD>1035
</TD><TD>pSB1K3
</TD><TD>Kanamycin
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K227015" class="external text" rel="nofollow">BBa_K227015</A></TD><TD>RBS34+OmpR(sph)+Term+OmpC+GFP
</TD><TD>Composite
</TD><TD>1875
</TD><TD>pSB1A2
</TD><TD>Ampicilin
</TD></TR></TBODY></TABLE><DIV style="text-align: left;"><P><A href="https://2009.igem.org/Team:Wash_U/Biological_Parts" class="external text" rel="nofollow">Back To Top</A></P><H2><SPAN class="mw-headline" id="Characterization"><B>Characterization</B></SPAN></H2><P><FONT size="2"><B>pucB/A</B>
pucB/A are the genes that code for the beta and alpha subunits of the light harvesting complex LH2.  Here, we measured the expression level of pucB/A from the puc promoter in the genome vs. on a low copy plasmid, pRKCBC3.  This data allows us to utilize pucB/A as a reporter gene for expression from promoters, in complement to its role in cellular growth. <B>Method</B>
Relative LH2 expression in R. sphaeroides 2.4.1, R. sphaeroides DBCÎ© and R. sphaeroides DBCÎ© pRKCBC3 grown anaerobically in the dark</FONT></P><P>Growth conditions:
-Cultures grown in the dark at 34Â° C, shaking at 160 rpm
	- R. sphaeroides 2.4.1
		-15 ml culture in polystyrene tube
		-15ml M22 was inoculated with 1ml inoculant with OD600nm = 0.270
		- OD600nm (Volume) = 0.270
		-Culture was allowed to grow 19hrs
	- R. sphaeroides DBCÎ©
		-15 ml culture in polystyrene tube
		-15ml M22 was inoculated with 865ul inoculant with OD600nm = 0.312
		- OD600nm (Volume) = 0.270
		-Culture was allowed to grow 19hrs
	- R. sphaeroides DBCÎ© pRKCBC3
-10 ml culture in polystyrene tube
-10 ml M22 tet 5ug/ml was inoculated with loop of R. sphaeroides DBCÎ© pRKCBC3 and capped with rubber stopper
-Culture was allowed to grow for 3 days so that density was high enough to get a good reading
</P><P>Analysis:
Cultures were removed from the incubtor and placed on ice to slow changes in cellular composition. 1 ml was extracted from each culture and a UV-vis absorption spectra of the culture was taken from 300-950 nm.  
The optical density of the cultures at 600nm was used to normalize the absorption spectrum by division by this value.  Background subtraction of spectrophotometer data was performed in Origin 6.1 Software.  A ten-point baseline was created by a &quot;positive peak&quot; algorithm then modified to approximate the scattering curve that falls as the inverse fourth power of wavelength.  <B>Results</B><B>Conclusion</B>
This data indicates that pucB/A can be utilized as a reporter gene as the LH2 absorption bands at 800 and 850 nm are not present in the LH2 deficient mutant DBComega.  Furthermore, changes in the expression conditions of pucB/A are reflected on the absorption spectrum.  In this case, expression is higher when the genes are expressed from the puc promoter within genomic DNA than on plasmid pRKCBC3, despite the fact that pRKCBC3 is maintained at 4-5 copies in a cell.  </P><P><B>Puc Promoter</B></P> The puc promoterpromotes transcription of the LH2 pucB/A genes naturally in <I>Rhodobacter sphaeroides</I>.  It is important that we are able to compare the transcription rate of the puc promoter in the natural system vs. our mutant system so that we can determine exactly how much efficiency is gained by adding a red light sensor.  The absorption spectra of a DBCOmega mutant (LH2 deficient) transformed with pRKCBC3 containing the puc promoter and pucB/A genes will allow us to characterize the puc promoter under high and low oxygen conditions.  
<P>More absorption of light at the LH2 spectra peaks normalized to culture OD corresponds with more transcription and vis versa.
<B>Method</B>
Cultures were grown in the dark at 34Â° C, shaking at 160 rpm.  The anaerobic test condition was established by inoculating a 10 ml culture tube with 10 ml M22 tet 5ug/ml with a loop of R. sphaeroides DBCÎ© pRKCBC3 and capped with a rubber stopper.  The aerobic test  condition was established by innoculating a 10 ml culture tube with 5ml M22 tet 5ug/ml with a loop of R. sphaeroides DBCÎ© pRKCBC3 and covered with vented cap- the vented cap and relatively low culture volume left significant headroom in the culture for oxygen exchange. Oxygen tension was not able to be quantitatively measured due to the nature of the experiment
For measurement of pucB/A expression from the puc promoter: 
Cultures were removed from the incubtor and placed on ice to slow changes in cellular composition. 1 ml was extracted from each culture and a UV-vis absorption spectra of the culture was taken from 300-950 nm.  
The optical density of the cultures at 600nm was used to normalize the absorption spectrum by division by this value.  Background subtraction of spectrophotometer data was performed in Origin 6.1 Software.  A ten-point baseline was created by a &quot;positive peak&quot; algorithm then modified to approximate the scattering curve that falls as the inverse fourth power of wavelength.  <B>Results</B><B>Conclusion</B>
This data matches the literature for expression from the puc promoter at different oxygen tensions and as such confirms the assumptions that we have made in modeling our system. 
See: Braatsch et al. 2002
</P><H2><SPAN class="mw-headline" id="Future_Characterization"><B>Future Characterization</B></SPAN></H2> The next step in our characterization of our synthetic red light response system is to analyze changes in phosphorylation of ompR in <I>Rhodobacter sphaeroides</I>.  In our final system, we only want puc genes to be transcribed and expressed via halting the autophosphorylation of ompR, not simply the puc promoter as it naturally occurs.  By placing the ompR coding region downstream of the red light sensor and upstream of a terminator our modified system controls expression of the puc genes by the red light sensor.  It should be impossible for the puc promoter to directly cause the transcription of puc genes due to the terminator, but instead, transcription of the puc genes must be activated via decreasing the presence of phosphorylated ompR.  The end target of ompR transcription is the ompC promoter, located directly upstream of the puc genes.  Placing GFP on the ompC transcript will show how often the promoter is transcribed and how often ompR is phoshorylated.
 Part three of our characterization measures the effectiveness of the red light sensor in downregulating the phosphorylation of ompR.  This setup is identical to that of part two except we have introduced the red light sensor.  Now, the rate of ompR autophosphorylation will be halted by binding to a domain on the light-activated EnvZ kinase analogue.  GFP is still attached to the end product, the ompC promoter.  By comparing the fluorescence of GFP in this scenario compared with the second scenario the decrease in rate of phosphorylation should be apparent due to the activity of the red light sensor.
 This is the final construct that will be our actual functioning model in <I>Rhodobacter sphaeroides</I>.  This finished product will be compared to the wild type over various intensities of light and cell culture densities can be compared to see which strain, the wild type or the modified strain with the red light sensor was more efficient in harvesting light with varying intensities.
<P><FONT size="2"><A href="https://2009.igem.org/Team:Wash_U/Biological_Parts" class="external text" rel="nofollow">Back To Top</A></FONT></P><H2><SPAN class="mw-headline" id="Modeling"><B>Modeling</B></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">pucBA Expression Model Diagram</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">pucBA Model Equations</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">pucBA Model Reactions</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Test Simulation Output</DIV></DIV></DIV><P><B>Modeling the Gene Regulatory Network</B></P><DL><DD>Our group seeks to assess the optimality of the synthetic system that modulates pucB/A gene expression and LH2 complex assembly in <I>Rhodobacter sphaeroides</I>.  Here we employ a mathematical model of this system to generate predictions about the behavior of the active system in response to light input. Features of the system that the model may help investigate include the time scale of response to light signals, the robustness of the system in response to fluctuations in light intensity, and the translation between changes in gene expression and the absorbance spectrum of the engineered cells.
</DD></DL><DL><DD>Though the context of the model can extend back to the transcription of PrrA/B genes involved in integration oxygen and light signals, a preliminary testing model was developed using assumptions of certain initial conditions to isolate the light signal's effect.  Since Cph8 and OmpR are located on the same transcript downstream of the puc promoter region, it was assumed that their associated protein and mRNA had already reached steady state concentrations, and the phosphorylation reaction had already reached steady state.  Moreover, the concentrations of the factors were assumed to be equal at this state.  The model whose diagram was constructed in the Simbiology Toolbox distributed by MathWorks details key reactions leading to the translation of the pucB/A genes.  The reaction rate equation used for the lack of phosphorylation of OmpR when the light signal reaches Cph8 bound to OmpR is captured in a modified form of Michaelis-Menten kinetics.  A logic function that corresponds to light ON/OFF (1/0) multiplies the maximum reaction rate in the numerator of the phosphorylation equation.  Thus, the model assumes that no phosphorylation occurs by this mechanism in the presence of light.  The OmpC promoter binding equation was based on the Hill Equation for an Activator(1).  
</DD></DL><DL><DD>Component characterization steps and literature searches are underway in order to obtain quantitative parameters for the reaction rates.  In order to simulate behavior of the system, putative values were included that exaggerate true concentrations and time scales.  OmpR was given an initial concentration normalized to one, and all other components were assumed insignificant initially to this value. An ideal light pulse was introduced at an instant and removed thirty simulation seconds later. From this rudimentary simulation it can be drawn that the nonlinearities of the phosphorylation and transcription factor binding kinetics effectively smooth the sharp light input.  By design, the light switch ON yields no phosphorylation of OmpR and repression of the pucB/A genes which would give rise to LH2.  Conversely, when left OFF, the concentration of pucB/A recovers and increases until the steady state determined by its translation and degradation rates.
</DD></DL><P><B>References</B></P><P>1.  Alon, Uri. <U>Introduction to systems biology and the design principles of biological networks</U>. Boca Raton, FL: Chapman &amp; Hall, 2006.

2.  Bower, James M. <U>Computational Modeling of Genetic and Biochemical Networks (Computational Molecular Biology).</U> New York: M.I.T. PRESS, 2001.

3. <U>System modeling in cellular biology from concepts to nuts and bolts.</U> Cambridge, MA: MIT P, 2006.
</P><H2><SPAN class="mw-headline" id="Simulating_A_Bioreactor"><B>Simulating A Bioreactor</B></SPAN></H2><P>Nonlinear least squares estimation of wild type saturation curve </P><P><B>Problem:</B>
In a typical reactor, cells at the surface absorb more than enough light to saturate their 
photosynthetic apparatus, transmitting less energy to deeper layers. Cells operating past the saturation point 
waste incident photons by non-photochemical quenching and possibly undergo photodamage.  For wild type cells, the 
âsaturation curveâ is assumed to be approximately the same for all cells in all layers, regardless of their 
incident light intensity.  This means that layers of cells on the exterior of the reactor nearest a light source 
receive an overabundance of photons and in turn block the interior layers from receiving enough light.  In an 
optimal reactor, all layers would operate near their respective saturation points to maximize the photosynthetic 
channeling of incident light energy.
</P><P>The total saturation curve for wild type R. Sphaeroides was fit with a nonlinear least squares regression of the 
form in the corresponding equation.  Data points were generated from calculating absorbance as the negative logarithm of the ratio of the absolute irradiance detected on the next layer to incident absolute irradiance on a layer of cells.  A logistic form was chosen to account for the diminishing returns to absorption of further photons past a threshold operating capacity of the photosynthetic apparatus. (1)
</P><P>Exponential response curve for mutant LH2 saturation coefficients </P><P><B>Simulating our Mutant's advantage in a Bioreactor</B>
For our mutant cells, the LH2 saturation curve for each layer scales as a function of light intensity.  This 
predicted behavior in the mutant is due to negative regulation of LH2 complex production as incident light 
intensity increases.  The scalar of the magnitude of the saturation curve was altered according to a predicted 
exponential curve of LH2 production in repsone to changes in incident light. It was assumed that the system could 
vary expression levels such that at high light intensities, the saturation curve is scaled to 25% of that of the 
wild type.  At low light intensities, LH2 production was assumed to have the potential to be up-regulated to 150% 
of wild type expression levels.  
The advantage this mutant would confer stems from the adaptive nature of the saturation curve heights.  Cells 
receiving the most light on the outside of the bioreactor saturate at low absorption levels.  This allows more 
light to transmit to further layers, which have elevated saturation curves due to lower incident light.
</P><P><B>Model Assumptions</B></P><P>- Light intensity at next layer is given by transmittance from previous layer (assumes no backscattering).
- Total energy funneled to photosynthetic pathways is estimated as the sum of light absorbed by each layer.  This 
generalizes to the optical density measurement of cell culture density. 
- The constant wild type saturation curve inherently includes both LH2 and LH1 contributions to absorbance.  The 
mutant's variable saturation curve only accounts for LH2 absorbance, since this is the only complex under the 
light-sensing regulation.  To account for the component of absorbance provided by LH1, the proportion of total 
optical density due strictly to LH1 was investigated by comparing the growth of wild type and LH2-knockout 
(dbcOmega) cultures.  It is evident that by day three the proportion of Optical Density accounted for by LH1 
absorption converges to a value near 0.2. In other words, at the phase the layers of cells have grown in the model, 
20% of total optical density can be attributed to LH1.  To account for this, the absorption in the mutant was 
divided by the factor (1-0.2) = 0.8.  Then, the total optical density of the mutant cultures reflects total 
absorption by both LH1 and LH2. 
Relative growth of DBComega to WT in Flask 2 at OD600 </P><P>- The model was revised upon gathering optical density data from the five layers of the bioreactor setup. (See 
Results Figure 2a.) In the wild type, the optical density of the first flask of cells was much lower than 
predicted, a phenomenon that was attributed to photodamage of the cells due to exposure to a large quantity of 
light past the saturation point of the LH2 complexes.  In the optical density data for the flasks, both the 
dbcOmega knockout and the wild type logistically grew to an absorbance value of 1. This gave reason to put a hard 
limit of 1 on the first flask's potential optical density.  Any light left over from this cutoff was transmitted to 
the next layer, as evidenced by in the increased growth of the second wild type flask in the Optical Density data.
- The response curve for the coefficient of saturation for the mutant due to changes in light intensity was modeled 
as an inverse exponential form.  In other words, the system reacts to increasing light intensity by exponentially 
tapering the coefficient of saturation.</P><DIV style="text-align: center;"><P>Wild Type vs. Mutant Photobioreactor Simulation </P><P>The sequence of figures above shows the behavior of the static wild type saturation curve and the mutant variable saturation curve in response to incident light intensity.  The dotted lines indicate the point of incident intensity and the associated absorbance with that point.  The center stacked bar graph shows the accumulation of optical density given by the absorbance readings from the saturation curve.  Recall that the mutant curve includes only LH2 adaptation, while the curve on the left is the total of LH1 and LH2 contributions to absorbance.  Throughout the entire five layers of the bioreactor, it is evident from the simulation that the mutant achieved higher overall growth as indicated by OD.  Also, the mean OD for the mutant exceeds that of wild type, and there is less variation in the levels about the mean (indicated by standard deviation).
</P><DIV style="text-align: left;"><P><B>References</B></P><OL><LI>Steven, Spilatro R. &quot;Photosynthesis Investigation Study.&quot; Photosynthesis Investigation Study. Department of Biology, Marietta College, Marietta, Ohio, 1998. Web. 21 Oct. 2009. &lt;<A href="http://www.marietta.edu/~spilatrs/biol103/photolab/photosyn.html" class="external free" rel="nofollow">http://www.marietta.edu/~spilatrs/biol103/photolab/photosyn.html</A>&gt;.
</LI></OL><DIV style="text-align: left;"><P><A href="https://2009.igem.org/Team:Wash_U/Biological_Parts" class="external text" rel="nofollow">Back To Top</A></P></DIV></DIV></DIV></DIV></FONT></CENTER></FONT></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Wash_U/Biological_Parts">http://2009.igem.org/Team:Wash_U/Biological_Parts</A>&quot;</DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Wash_U/Biological_Parts" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Wash_U/Biological_Parts" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Wash_U/Biological_Parts&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Wash_U/Biological_Parts&amp;oldid=169186" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>