Team:Aalto-Helsinki/Results
Toggle navigation
Project
Modeling
Modeling
Propane pathway
Kinetics of propane pathway
CAR-Activation
Synergy model
Modeling micelle
Cellulose pathway
Laboratory
Laboratory
Lab Book
Protocols
Safety
Practice Project
Results
Results
Continuous production
Submitted parts
Achievements
Future
Practices
Practices
Outreach
Combining modeling and experimentation in iGEMe
Community
Community
Collaboration
Collaboration Platform
Collab Seeker
InterLab Study
InterLab Book
People
Team
Attributions
Journal
Project
Modeling
Models of propane pathway
Kinetics of propane pathway
Model of CAR activation
Synergy model
Micelle model
About cellulose pathway
Laboratory
Lab Book
Protocols
Safety
Practice project
Results
Continuous production
Submitted parts
Achievements
Future
Practices
Combining modeling and experimentation in iGEM
Outreach
Community
Collaboration
Collaboration Platform
Collab seeker
Interlab study
InterLabBook
People
Team
Attributions
Journal
Overview
Propane pathway
Continuous production
Cellulose degradation
Amphiphilic protein
Fusable GFP
Combining modeling and experimental work in iGEM
Software
Future
To the top
Results
Overview
Extensively modeled a microbial pathway for propane production and used this information to improve experimental design
Studied the idea of using synthetic amphiphilic micelle-forming proteins as molecular scaffolds to place enzymes in close proximity to each other and modeled both micelle formation and the effect of enzyme proximity on reaction pathways when competing enzymes are present
Submitted three BioBricks to the registry: one containing enzymes of the propane pathway, one encoding a N-terminally fusable GFP and one unlikely containing the amphiphilic protein
First report ever of successful continuous microbial propane production
Studied the relationship of modeling and experimentation in iGEM teams, as well as the educational background of iGEM participants
Built and ideated software to help iGEM teams better collaborate and communicate
Propane pathway
Background
Microbially produced propane holds enormous promise as a potential replacement of portable fossil fuels, but the propane yields with current biological pathways are low. Our pathway starts from acetyl-CoA and produces propane through 6 intermediates. To help concentrate engineering efforts on its critical parts, better quantitative understanding of the pathway is required. Our goals were to build a mathematical model of the pathway to better understand it and create and test BioBricks of the propane pathway to help future teams and researchers to continue improving it.
Outcome
Figure 1: Illustrative figure of the bottleneck results of our pathway.On the left,situation before switching FadB2 to Hbd, on the right situation after the switch.
Built a model of the pathway based on known kinetic properties of the enzymes
Identified major bottlenecks of the propane pathway using our model
Improved our experimental plans according to the modeling results by changing one enzyme to a better homolog and expressing the rate-limiting enzyme from the highest copy number backbone used
Found that propane output was sensitive to NADPH/NADH, suggesting their efficient regeneration might be a limiting factor
Designed two BioBricks containing all ten genes required for the propane pathway to produce propane in E. coli
Chose three intercompatible backbones to ensure that the propane pathway could be integrated to the same cell with cellulose hydrolysis (which required the third backbone)
Assembled Propane 1 (CAR) constructs.
Assembled Propane 2 insert.
Used NEBuilder (Gibson) Assembly to construct the BioBricks
Submitted a BioBrick (Propane 1) containing three crucial enzymes of the propane pathway and inserted it in pSB6A1 backbone for usage in propane production
After trying both ELIC and OE-PCR once and Gibson assembly twice, we managed to assemble Propane 2 with Gibson assembly and detect the right size insert with colony PCR. However, as we ran out of time, we were unable to successfully propagate the correct plasmid, despite multiple tries. As we didn’t manage to assemble the whole pathway, we couldn’t try out whether the pathway was functional.
Our chemostat.
Continuous production
Background
Industrial biopropane production would most likely occur as a continuously operated process, as it is economically more feasible in large scale production. To take the first step towards industrial scale production we wanted to try continuous production of propane using a E. coli strain provided to us by Pauli Kallio from the University of Turku.
Outcome
Successful batch production to prepare analytical equipment for continuous production
First report ever worldwide of successful microbial production of propane in continuous production
145 hour continuous production experiment in a 500 ml chemostat
Propane yield up to 22.7 µg/L in reactor gas phase
Cellulose degradation
Cellulose insert.
Background
Well over 200 million tonnes of cellulosic waste is left unused each year in the European Union alone. To elevate the microbially produced propane to a 2nd generation biofuel and avoid interfering with food production, we wanted to incorporate a plasmid for cellulose hydrolysis into the same bacteria that produces the propane. The plasmid contains three genes encoding the enzymes that hydrolyse cellulose polymers into glucose.
Outcome
Looked into modeling cellulose breakdown , but found that there was not enough information to model the breakdown sufficiently well to get any practical benefit from the model.
After trying Gibson assembly and ELIC twice, we were able to produce the Cellulose insert containing the genes encoding three enzymes required for cellulose hydrolysis with OE-PCR. However, due to its low concentration, we were unable to transfer it to a backbone and propagate it.
Amphiphilic protein
Background
Amphiphilic proteins are synthetic proteins consisting of a hydrophilic and a hydrophobic domain that have been shown to spontaneously form micellar and vesicular structures. We were interested whether these structures could be used as scaffolds to have subsequent enzymes of the propane pathway in close proximity. We wanted to model whether fusing enzymes to the proteins would disrupt micelle formation and whether our idea could enhance propane output.
Outcome
Figure 2: 2d simplification of the micelle structure
Built a stochastic synergy model in Python, simulating enzyme function in cases where two subsequent enzymes stay in close proximity to each other as opposed to moving around independently in a cell
The synergy model predicts a 200-400 % increase in product output if enzymes stay in close proximity to each other
Constructed a geometrical micelle model based on the sizes and structures of the micelle-forming proteins, indicating that it is indeed possible for micellar structures to form even as enzymes are fused to them
Submitted a BioBrick unlikely encoding the amphiphilic protein to the registry
Due to time restraints, we were unable to experimentally validate the idea by fusing either two subsequent enzymes of the propane pathway or components of the violacein pathway to the amphiphilic proteins
N-terminally fusable GFP
Background
To validate our amphiphilic brick, we needed a GFP that could be fused to the amino-terminal end of the protein. There was no such brick in the distribution kit. We also thought there was no such brick in the registry, but later came to realize we were wrong. We wanted to create a GFP BioBrick that could be fused to the N-terminal end of any protein using BioBrick methods.
Outcome
Submitted a BioBrick encoding GFP with an extra nucleotide prior to the suffix, ensuring that it can be fused to the N-terminal end of a protein using BioBrick enzyme assembly methods while maintaining the reading frame
Collaborated with team HS Slovenia to validate the brick
Figure 3: Upper row: E. coli expressing GFP fused to the N-terminal end of our amphiphilic protein. Bottom row: control. On the left, a light microscope picture and in the middle a fluorescent microscope picture of the same cells (excitation at 488 nm, detection 493-598 nm). On the right the two pictures to the left merged, showing that GFP is expressed in transformed cells, but not in control cells.
Combining modeling and experimental work in iGEM
Background
Mathematical modeling is a key component of synthetic biology and also played a central role in our project. Collaboration between modelers and biologists can however be challenging, something we also noticed in our project. We wanted to study how iGEM teams tackle these challenges and are able to integrate modeling to their experimentation.
Figure 4: Educational background of questionnaire respondents, 2014 iGEM participants and professional synthetic biology researchers categorized. "Mathematical" includes mathematics, computer science and physics.
Outcome
Created a questionnaire for iGEM teams on collaboration between modeling and experimentation, and studied 2014 teams and professional synthetic biology groups to find out what educational backgrounds team members are coming from
Biggest issues in collaboration between modelers and experimentalists are: lack of knowledge of the other field, lack of common terminology and differences in ways of thinking
Both modelers and biologists need to understand the basics of the other field to be able to effectively collaborate.
Having experimentalists and modelers work close together is beneficial. One approach generally found successful is to have some biologists get involved in modeling to help ensure models are useful for the project and connected to reality. Modelers can too take some time to get familiar with the wetlab in the beginning of the project
Regular team meetings for presenting and discussing progress and issues of every field take time, but ensure all team members stay informed and can voice their insights.
Students with a mathematical background are underrepresented in iGEM teams as compared to professional synthetic biology groups.
iGEM teams have relatively many biotechnology students, who often stand seem to act as mediators between the modeling and experimentation.
Software: HumHub report and Collab Seeker
Background
Finding new collaboration partners in iGEM is not easy, as finding information about different teams' projects is time-consuming and often difficult. On the other hand, communication with other teams as well as internal team communication can be difficult due to a multitude of platforms used, often cluttered with non-iGEM content. We wanted to do something to these issues to make iGEM even better.
Outcome
Worked with Stockholm iGEM team on HumHub, a collaboration platform for iGEM teams, and collaboratively wrote a report on it with them
Created a questionnaire on how teams found their collaboration partners and how they’re keeping in touch with them
Found that 22 out of the 23 teams that answered wished for better means to find collaboration partners with
Built Collab Seeker , a lightweight collaboration search tool, which helps find relevant collaboration partners using keywords and provides their contact information
Future
To read our thoughts on future prospects and on how to carry on from where we left, please see our Future page .
Submitted BioBricks and Achievements
To read about the BioBricks we submitted, see our Submitted Parts page . To see how our achievements match with the medal criteria, see our Checklist .
