Team:Aalto-Helsinki/Future
Toggle navigation
Project
Modeling
Modeling
Propane pathway
Kinetics of propane pathway
CAR-Activation
Synergy model
Modeling micelle
Cellulose pathway
Laboratory
Laboratory
Lab Book
Protocols
Safety
Practice Project
Results
Results
Continuous production
Submitted parts
Achievements
Future
Practices
Practices
Outreach
Combining modeling and experimentation in iGEMe
Community
Community
Collaboration
Collaboration Platform
Collab Seeker
InterLab Study
InterLab Book
People
Team
Attributions
Journal
Project
Modeling
Models of propane pathway
Kinetics of propane pathway
Model of CAR activation
Synergy model
Micelle model
About cellulose pathway
Laboratory
Lab Book
Protocols
Safety
Practice project
Results
Continuous production
Submitted parts
Achievements
Future
Practices
Combining modeling and experimentation in iGEM
Outreach
Community
Collaboration
Collaboration Platform
Collab seeker
Interlab study
InterLabBook
People
Team
Attributions
Journal
Propane pathway
Searching homologs for CAR
Propane out of sunlight, water and thin air
Improving safety
To the top
To the Parent Page
Future
Propane pathway
There is still plenty of room for improvement in the propane pathway. The identified bottlenecks, enzymes CAR and ADO could be studied further to find out whether there are more efficient enzymes with the same function available in the nature. If not, it might be worth the effort to try and engineer the existing CAR and ADO to be more efficient, as has already been once done for ADO . The idea of having different enzymes of the pathway close together, by fusion to each other or by using different kinds of scaffolds, including our amphiphilic proteins, could also be studied further.
In out pathway model, we also identified the propane output to be sensitive to NADPH/NADH concentration. Therefore, it might be that NADPH/NADH is a limiting factor, if its generation is insufficient. This is something worth studying, and if NADPH/NADH regeneration is indeed verified to be a bottleneck, then it could be studied whether this regeneration could somehow be enhanced.
Even though it was not possible within our timeframe, one could try knocking out more endogenous aldehyde reductases and alcohol dehydrogenases that compete with ADO for butyraldehyde. This approach has been tried by Pauli Kallio and his associates with success: knocking out two endogenous aldehyde reductases Ahr and YqhD resulted in significant improvement in propane output.
Our work focused on a pathway based on fatty acid synthesis, resulting in production of butyraldehyde, which is further turned to propane by ADO. This is not the only possible solution, as there are many other pathways that could be potentially used to generate butyraldehyde.
Searching homologs for CAR
As our kinetic model identified, CAR (Carboxylic acid reductase from Mycobacterium marinum) is the most significant bottleneck of the propane pathway after FadB2 and ADO. How can we solve this bottleneck? One method is to search for homologs, which have the same ancestor gene and possibly also have the same function as CAR, but a better performance on the kinetic level. We are focusing our efforts to find homologs, because enzyme analogs, ie. enzymes which have same functions but are evolutionarily from different species, are more difficult to identify as our methods cannot recognize different protein sequences that have the same functionality.
To search for CAR homologs, we created phylogenetic trees using protein sequences, because we are searching for proteins with a slightly altered structure (and therefore possibly better kinetic properties) and amino acid mutations can affect this. Conversely, nucleic acid mutations might have no effect on the amino acid sequence and therefore the protein structure, as most amino acids have multiple codons encoding them. We created two phylogenetic trees using two different methods: UPGMA (Unweighted Pair Group Method with Arithmetic Mean) and Bayesian MCMC (Markov chain Monte Carlo). UPGMA is faster but more crude than Bayesian MCMC. An UPGMA tree was done with Geneious v8.1.7 and for a Bayesian MCMC tree used BEAST v1.8.2 , BEAUti v1.8.2 [1] , Tracer v1.6 , TreeAnnotator v1.8.2 ja FigTree v1.4.2. Needed multiple protein alignment was done with MUSCLE (with 16 iterations) in Geneious.
We used diverse resources to find potential CAR homologs, which have the same function. InterPro had an entry for our CAR and we used the site’s “Similar protein” -link to look for similar proteins, which are likely to be homologs. The proteins which have AMP-dependent synthetase/ligase, acyl carrier protein-like and thioester reductase-like domains were chosen, because CAR has them. We searched through UniProt database using keywords, such as “short fatty acid coa ligase” and “Carboxylic acid reductase”. The proteins, which have similar GO-classes and a comparable description as CAR, were picked. Many similar proteins were found by protein BLAST (Blosum62). BLAST results with E value of 0 and over 80 % identity with our CAR sequence were chosen. From the protein sequence of CAR, Blastp recognized several related superfamilies, ie. protein families which are similar on the sequence level. They are adenylate forming domain Class I, phosphopantetheine attachment site and Rossmann-fold NAD(P)(+)-binding proteins, the descriptions of which match with the domain descriptions of the CAR InterPro entry.
Figure 1: The UPGMA phylogenetic tree and sources for the proteins. The node labels describe how evolutionarily distant the proteins are: the higher a value is the more distant they are. The search method tells how we found the proteins and the ID its respective identification code. The enzyme from the human is picked to be an outsider group for calculations. The human enzyme catalyzes the opposite reaction as our CAR, oxidation of butyraldehyde to butyric acid. Underlined enzyme is our CAR. The used substitution model for UPGMA was Jukes-Cantor. Click on the image to enlarge it.
Figure 2: The Bayesian MCMC phylogenetic tree. The node labels are posterior values, which describe how accurate a branch point is. Because almost all values are over 0.9, we can believe its accuracy, depending on initial values. Used Blosum62 as substitution model. ESS (effective sample size) was 6404.1033 for posterior. Click on the image to enlarge it.
Our phylogenetic trees are shown in Figure 1 and Figure 2. They are very similar to each other, even though they were calculated with different algorithms. One major difference is that the oxidoreductase from Sciscionella marina is considered to be evolutionarily closer to our CAR than the enzyme from Rhodococcus wratislaviensis in the Bayesian MCMC tree, but the enzymes closer to CAR are grouped alike in the both trees. In both figures the branch length to the thioester reductase-like protein of Cryptosporangium arvum is quite big. Therefore, we should consider the enzymes, which are closer to our CAR from the enzyme of Cryptosporangium arvum, as potential homologs which can replace CAR in the propane pathway as they have higher chance of having the same function. Little kinetic research has been done on the enzymes we found: BRENDA, an enzyme database, only had kinetic values for a CAR homolog from Nocardia iowensis, so we really cannot tell if their performance is better or worse. As CAR is one of the rate-limiting steps in the pathway, it would seem reasonable to screen its homologs for more efficient alternatives. If a homolog with better kinetic values than CAR is found, it can be used to improve the propane yield of our pathway.
Propane out of sunlight, water and thin air
One significant benefit of the pathway is that it can operate in the presence of ​oxygen. This is required to incorporate the pathway in oxygenic, photosynthetic organisms like cyanobacteria. Cyanobacterial propane production could have a tremendous effect on the way energy is produced and consumed in the society. Fuel production would essentially require only sunlight, water and CO\(_2\), and would thus be completely renewable.
Improving safety
As propane is a highly flammable liquid, large-scale microbial production could pose a fire and/or explosion hazard. The production would most likely happen in closed containers with nothing to ignite the gas, easing the problem. However, leaks are always possible: the propane or the bacteria themselves could leak from the microbial container or pipelines to enclosed spaces where ignition is possible. By replacing air, propane could also cause a suffocation danger. To help avoid these problems, it would be beneficial if the propane could be detected. However, propane itself cannot be seen and it has no odour, making detection difficult. Gas molecules with an odour (e.g. ethyl mercaptan) could be added to the purified product and production container. However, this does not allow us to detect the gas if the bacteria leak to produce propane in an enclosed space where no such safety measures are taken.
To help detect microbial propane production, it might be thus reasonable to have the propane-producing bacteria also produce a certain scent when propane production is active. This could be achieved by for instance incorporating a banana smell generator device to the same bacterium which is producing propane. Another, perhaps even better option would be to modify the bacteria so that they need to be given certain nutrients not widely available in the environment to survive.
