<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aalto-Helsinki_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aalto-Helsinki/Project</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="site-wrapper"><DIV class="site-wrapper-inner"><NAV class="navbar navbar-default visible-xs visible-sm" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON class="navbar-toggle collapsed" type="button" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" role="navigation"><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Project">Project</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Modeling </A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Modeling_propane">Propane pathway</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Kinetics">Kinetics of propane pathway</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Car-activation">CAR-Activation</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Modeling_synergy">Synergy model</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Modeling_micelle">Modeling micelle</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Modeling_cellulose">Cellulose pathway</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Laboratory </A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Lab">Laboratory</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/LabBook">Lab Book</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Experiments">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Practice_project">Practice Project</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Results </A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Design">Continuous production</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Parts">Submitted parts</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Medals">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Future">Future</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Questionnaire">Combining modeling and experimentation in iGEMe</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Community </A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Community">Community</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/humhub">Collaboration Platform</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Software">Collab Seeker</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/InterLab">InterLab Study</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/InterLabBook">InterLab Book</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">People </A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Journal">Journal</A></LI></DIV></DIV></NAV><DIV class="cover-container hidden-sm hidden-xs " style="margin-top:2%;"><NAV id="primary_nav_wrap"><UL><LI><A id="projectnav" href="https://2015.igem.org/Team:Aalto-Helsinki/Project">Project</A></LI><LI><A id="modelingnav" href="https://2015.igem.org/Team:Aalto-Helsinki/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Modeling_propane">Models of propane pathway</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Kinetics">Kinetics of propane pathway</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Car-activation">Model of CAR activation</A></LI></UL><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Modeling_synergy">Synergy model</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Modeling_micelle">Micelle model</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Modeling_cellulose">About cellulose pathway</A></LI><LI><A id="labnav" href="https://2015.igem.org/Team:Aalto-Helsinki/Lab">Laboratory</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/LabBook">Lab Book</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Experiments">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Practice_project">Practice project</A></LI><LI><A id="resultsnav" href="https://2015.igem.org/Team:Aalto-Helsinki/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Design">Continuous production</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Parts">Submitted parts</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Medals">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Future">Future</A></LI><LI><A id="practicesnav" href="https://2015.igem.org/Team:Aalto-Helsinki/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Questionnaire">Combining modeling and experimentation in iGEM</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Outreach">Outreach</A></LI><LI><A id="communitynav" href="https://2015.igem.org/Team:Aalto-Helsinki/Community">Community</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/humhub">Collaboration Platform</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Software">Collab seeker</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/InterLab">Interlab study</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/InterLabBook">InterLabBook</A></LI><LI><A id="peoplenav" href="https://2015.igem.org/Team:Aalto-Helsinki/Team">People</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Aalto-Helsinki/Attributions">Attributions</A></LI><LI><A id="journalnav" href="https://2015.igem.org/Team:Aalto-Helsinki/Journal">Journal</A></LI></NAV></DIV><UL id="sidenav" class="nav nav-stacked bottom"><LI><A href="#" data-scroll="background"><H3>Background</H3></A></LI><LI><A href="#" data-scroll="propane"><H3>Propane</H3></A></LI><LI><A href="#" data-scroll="cellulose"><H3>Cellulose</H3></A></LI><LI><A href="#" data-scroll="micelle"><H3>Micelle</H3></A></LI><LI><A href="#" data-scroll="constructs"><H3>Constructs</H3></A></LI><LI><A href="#" data-scroll="cont"><H3>Continuousproduction</H3></A></LI><LI><A href="#"><H3 style="border-top:solid;">To the top</H3></A></LI></UL><DIV class="projectbg"><DIV class="inner-container"><H1>Our Project</H1><SECTION id="background" class="active" data-anchor="background"><H2>Background</H2><P>Climate change is argued to be one of the greatest challenges faced by mankind. The current climate change is mainly caused by us humans as we have been using the Earth’s precious fossil fuel stocks without returning the emitted gases into the natural carbon cycle. According to <A href="http://www.ipcc.ch/publications_and_data/ar4/wg1/en/ch10s10-7.html#10-7-1" target="_blank">the Intergovernmental Panel on Climate Change</A> (IPCC), even if we could stop all the emissions right now, the Earth’s average temperature would rise 0.6°C. This means we must act now. To fight climate change we have come up with a solution which would tackle the emissions made by the road transportation. These emissions make up a <A href="http://www.oecd.org/env/39762914.pdf" target="_blank">considerable 11%</A> of the world’s greenhouse gas emissions.</P><P>While power generation accounts for about a quarter of the world’s greenhouse gas emissions, 11% might seem like a small number. What makes emissions made by transportation significant, though, is that there are currently no good alternatives for gasoline. Electric cars are emerging, but they still have quite a way to go before reaching the price-range suitable for middle class. Propane is already widely used as a <A href="http://www.iea.org/publications/globalevoutlook_2013.pdf" target="_blank">replacement of gasoline</A>. In South Korea, <A href="http://www.auto-gas.net/uploads/Autogas%20Incentive%20Policies%202014.pdf" target="_blank">2.2 million</A> vehicles run on propane and in Turkey, 37% of passenger cars use it. On a large scale though, <A href="http://www.auto-gas.net/uploads/Autogas%20Incentive%20Policies%202014.pdf" target="_blank">only 1.2%</A> of vehicles worldwide are suitable to run on propane. This may seem discouraging, but in reality converting a gasoline motor into a propane one is quite simple and inexpensive. DIY converter kits are sold online for less than $500. In Canada, depending on the vehicle type, conversion done by a third party costs somewhere around <A href="http://www.autopropane.com/html/faqs.html" target="_blank">2500-6500 dollars</A>.</P><P>On top of its use as a transportation fuel, propane is also a popular cooking fuel in developing countries. It’s more commonly used in urban areas, as access to propane can be difficult in rural areas. Nevertheless, according to the International Energy Association (IEA), about 20% of household in the rural areas of Botswana use propane as their source of energy. In the urban areas the usage is around 60%. Expensive infrastructure is in place for example in Brazil, where <A href="https://www.iea.org/publications/freepublications/publication/cooking.pdf" target="_blank" style="padding:0">98% of households</A> have access to propane due to government funded efforts. One major goal of promoting propane is to replace currently used biomass fuels, including wood. Wood consumption is often unsustainable and threatens the local ecosystems. Additionally, the traditional fuels (biomass and coal) produce high emissions of carbon monoxide, hydrocarbons and particulate matter. IEA suggests that these impurities are responsible for <A href="https://www.iea.org/publications/freepublications/publication/cooking.pdf" target="_blank" style="padding:0">more premature deaths</A> than malaria in developing countries.</P><P>Propane can be considered <A href="http://pubs.acs.org/doi/abs/10.1021/ef5006379" target="_blank">a clean fuel</A>, as it emits less CO\(_2\) than gasoline or ethanol and, when compared with traditional biomasses, has significantly smaller emission factors based on both mass and energy delivery. Thus, propane is a viable option for replacing traditional biomasses as cooking fuel, although currently the price is not competitive without government subsidies.</P><DIV class="row"><FIGURE style="margin: 10px;"><FIGCAPTION style="font-size:12px;"><B>Figure 1.</B><A href="http://pubs.acs.org/doi/abs/10.1021/ef5006379" target="_blank">Comparison of various fuels</A> by delivered-energy-based CO and particle matter emissions (EF, g/MJ). The y axis is shown in the log scale.</FIGCAPTION></FIGURE></DIV><P>However, the propane we are currently using is produced as a side product of the petrochemical industry. This essentially means that propane is a fossil fuel, emitting stored CO\(_2\) into the atmosphere without a way of returning to the natural carbon cycle. This contributes not only to climate change but also ocean acidification. This endangers <A href="http://www.sciencedirect.com/science/article/pii/S1877343512000620" target="_blank">the livelihoods </A>of hundreds of millions of people directly or indirectly dependent on the marine ecosystems. </P><P>We want to make the use of propane as a fuel sustainable. We want to design an <I>Eschericia coli</I> capable of producing propane from cellulose. </P></SECTION><SECTION id="propane" data-anchor="propane"><H2>E. coli Producing Propane</H2><FIGURE style="float:right;margin-left:20px;"><FIGCAPTION><CENTER><B>Figure 2.</B> Propane production pathway</CENTER></FIGCAPTION></FIGURE><P>Our propane pathway is based on the research done by <A href="http://www.nature.com/ncomms/2014/140902/ncomms5731/full/ncomms5731.html" target="_blank">Dr. Pauli Kallio <I>et al.</I></A> and <A href="http://www.biotechnologyforbiofuels.com/content/8/1/61" target="_blank">Menon <I>et al.</I></A>. Right after we decided our topic we got in touch with Dr. Pauli Kallio from the University of Turku. He was very excited about our project and eager to help. We were able to get Kallio’s groups plasmid maps from him and decided to use these as our starting material. As they had already tested this pathway, we could be sure that their genes were functional in <I>E. coli</I>.</P><P>Our chassis organism, <I>Escherichia coli</I> BL21 (DE3), was chosen because it’s a strain that produces the T7 promoter when induced with IPTG. This is the strain that was used in Kallio’s research and was available at our lab. In addition to the regular BL21 (DE3), Dr. Kallio was kind to send us a BL21(DE3) with YjgB and YqhD knocked out when we realized that producing some knock outs ourselves would be too expensive and time consuming. YjgB and YghD are <I>E. coli</I>’s endogenous genes which produce butyraldehyde consuming enzymes. This means that they compete with our pathway’s final enzyme, ADO, which uses butyraldehyde as its substrate.</P><FIGURE style="float:left;margin-right:20px;"><FIGCAPTION><CENTER><B>Figure 3.</B> Propane Plasmid 1</CENTER></FIGCAPTION></FIGURE><P>We had been warned about the pathway’s vulnerabilities: it consisted of 10 different enzymes and in earlier research it had been built with a minimum of three different plasmids. The stress for the bacteria was high. We thought that one way of reducing this stress was to use just two plasmids for the propane pathway. 
We then moved forward to design our complete constructs. We used Kallio’s group’s constructs as a basis, and arranged the genes similarly. The arrangement of our first plasmid is the same as the original one. It starts with a T7 promoter and an additional lac operator. The T7 polymerase is IPTG inducible in our BL21 (DE3) strain, but we wanted to make sure there were no leaks in our system before the actual induction. That’s why the promoters adjacent to the genes are induced by IPTG. Following the promoter, we have the YciA, Sfp and CAR each with their own RBS and a T7 terminator. We chose to use the same RBS's and terminator sequences as Kallio's group, as they had already tested the system. Our construct's terminator sequence is used in common cloning vectors (such as pET and pDF). Our second plasmid includes the same promoter, RBS’s, and terminator, but the genes AtoB, Hbd, Crt and Ter, which is a similar construct as the original one made by Kallio's group. We decided to add another promoter to the plasmid to create two operon systems. We did this to ensure that all the genes would be transcribed by the polymerase. The last genes in our second construct, ADO, PetF and Fpr function under the same promoter and induction systems as all the other genes.</P><P>Instead of Hbd, our Propane Plasmid 2 originally had FadB2 as the second gene. Hbd was originally used in Kallio's constructs, but FadB2 was used in an earlier experiment. Due to an error, FadB2 was built into our original construct instead of Hbd. According to literature, FadB2 had been shown to function in <I>E. coli</I>, but after the <A href="https://2015.igem.org/Team:Aalto-Helsinki/Kinetics">pathway's bottlenecks</A> were modeled, it became clear that the enzyme's kinetics properties were a serious problem. We then changed this gene to Hbd in our constructs, which functioned much better according to our model.</P><FIGURE style="float:right;margin-left:20px;"><FIGCAPTION><CENTER><B>Figure 4. </B>Propane Plasmid 2</CENTER></FIGCAPTION></FIGURE><P>We first planned on creating a BioBrick from each of our genes and assembling the plasmids with the three antibiotics assembly method, but soon realized it would take too much time. We were then introduced to the Gibson Assembly system, and decided to give it a try. This is also when we decided to provide whole plasmids that would allow easy production of propane, rather than creating a separate brick of each of our genes.</P><P>So, we divided our whole plasmids into pieces of about 2000 bp each and included 30 bp overlaps into our gBlocks-to-be. This is when we also realized that synthesis isn’t as simple as it sounds: we had to optimize more than half of our sequences to be fit for gBlock synthesis. We will then proceed with <A href="https://www.neb.com/products/e2621-nebuilder-hifi-dna-assembly-master-mix" target="_blank" style="padding:0">NEBuilder kit</A>, <A href="http://nar.oxfordjournals.org/content/32/2/e19.full" target="_blank">overlapping PCR</A> (OE-PCR) and/or <A href="http://link.springer.com/article/10.1007%2Fs12033-014-9817-2#page-1" target="_blank">Exonuclease and Ligation-Independent Cloning</A> (ELIC) to combine our constructs. We used multiple methods because of the time constraint and the fact that we didn't know which one would work the best. Due to a construct design error our assembly pieces contained the BioBrick prefix in the very first part, but the last part did not include the suffix. The suffix was added by PCR after synthesis and the prefix &amp; suffix areas functioned as the homologous overlap areas between our brick and the backbone.</P><P>In the assembly phase, we also had to take into account the plasmids we were to use. As our whole system would require three plasmids altogether if we were to add the cellulose hydrolysing enzymes to the same bacteria, we had to be careful about the plasmids’ compatibility groups. Our plasmids needed different antibiotic resistances and intercompatible origins of replication. After our constructs were successfully assembled they were sent for sequencing to check that everything worked as expected. We then transformed them into competent <I>E. coli</I> BL21(DE3) ΔyjgB ΔyqhD strain with chemical transformation and screened the transformants with double-antibiotic plates. These cells should be able to produce propane. All we would need to do is induce the production with IPTG and identify the propane by gas chromatography.</P></SECTION><SECTION id="cellulose" data-anchor="cellulose"><H2 id="cellulose">Degrading Cellulose</H2><P>To further develop the idea of using propane as a biofuel, feedstock alternatives were sought as glucose itself interferes with food production. At first, we planned to use cyanobacteria as a host organism to produce carbohydrates to the propane pathway but with the limited amount of time available the project would become too challenging to be accomplished. However, non-edible carbon sources like cellulose had already been investigated by previous iGEM teams and cellulose hydrolysis by bacteria wasn’t connected to the propane pathway yet. Therefore, we decided to design a system that would utilize cellulosic waste.This helped us avoid interfering with food production and was easily integrated to our original plan of propane production.</P><P>Because we wanted to incorporate cellulose degradation and propane production into the same strain, all genes expressing the cellulose degrading cellulases needed to be in the same backbone. Two plasmids already contained propane pathway genes and the approximated limit of the insert size in the plasmid is about 10 kb. The number of different degrading enzymes was limited to three cellulases in order to limit the plasmid size and the stress of the organism. To convert cellulose polymers into glucose units, the proper hydrolysis requires enzymes endoglucanase, exoglucanase and β-glucosidase. Endoglucanases hydrolyze covalent bonds from the middle of polysaccharide and exoglucanases cleave oligosaccharides from the ends of the chain. β-glucosidases convert oligosaccharides into glucose which are utilized for metabolic reactions.</P><P>Usually, fungal strains <I>Trichoderma reesei</I> and <I>Saccharomyces cerevisiae</I> produce two different types of endoglucanases and exoglucanases belonging to different enzyme families. Due to our enzyme limit, only Type I cellulases (CenA, accession M15823 and Cex, accession M15824) from <I>Cellulomonas firmi</I> were chosen to be expressed. We found only a couple of Biobricks which could be used for our project from the iGEM registry but those parts had complicated status. Fortunately, the sequences were available on the registry for parts <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K392006">BBa_K392006</A> and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K392007">BBa_K392007</A> and Distribution Kit 2015 contained the gene responsible for β-glucosidase <A href="http://parts.igem.org/Part:BBa_K118028">BBa_K118028</A> from <I>Cytophaga hutchinsonii</I>. Therefore, we decided to synthesize endoglucanase and exoglucanase expressing genes utilizing the sequences previously mentioned and take β-glucosidase from the kit. These three sequences will be amplified with PCR and connected with the homology ends of genes created by primers. Our plan was to make the connections with Gibson Assembly method. The cellulase genes are positioned downstream to the same T7 promoter induced by IPTG.</P><FIGURE style="float:right"><FIGCAPTION><CENTER><B>Figure 5.</B> Cellulose Plasmid</CENTER></FIGCAPTION></FIGURE><P><I>E.coli</I> strains do not naturally contain any secretion systems for endoglucanases and exoglucanases so if the bacteria has previously been researched for cellulose degradation, usually the enzymes just overflow from cytoplasm to environment when enough proteins are produced. However, BL21 strain which we were using naturally contained β-glucosidase gene in the genome but because the expression levels and enzyme activities were poorly documented in databases, we decided to use the same gene <A href="https://2010.igem.org/Team:Osaka" target="_blank">Osaka 2010 team</A> was using from Distribution Kit. For cenA, cex, and β-glucosidase secretion, pelB-secretion tag sequence <A href="http://parts.igem.org/Part:BBa_J32015">BBa_J32015</A> made by 2010 Duke team was synthesized prior to the coding sequences of the cellulases which would then move into cell’s periplasmic space.</P><P>Cellulase producing capacity would have been first investigated with carboxymethyl cellulose (CMC) plates which are labeled with Congo Red assay. Congo red dye is the sodium salt of 3,3'-([1,1'-biphenyl]-4,4'-diyl)bis(4-aminonaphthalene-1-sulfonic acid) which has a strong affinity to cellulose fibers. If the cellulose polymers are digested by enzymes on the plate, the spot will be changed into colorless and a halo will appear around the bacterial colony. However, the method will not tell whether glucose is produced or not.</P><P>For glucose analysis, 3,5-dinitrosalicylic acid would have been utilized. It is a compound which reacts with reductive sugars like glucose forming 3-amino-5-nitrosalicylic acid. The product absorbs light with the wavelength of 540 nm. Three different controls are needed when analyzing the produced glucose concentration because the cultivation liquid with CMC already contains sugar: one without any strain, one with <I>E.coli</I> without cellulase producing capacity and one with cellulose hydrolyzing. Furthermore, liquid chromatography could also be utilized if we find proper equipment.</P></SECTION><SECTION id="micelle" data-anchor="micelle"><H2>Micelle Fusions Enhancing the Production</H2><P>Based on the previous <A href="http://www.nature.com/ncomms/2014/140902/ncomms5731/full/ncomms5731.html" target="_blank">studies</A> about this <A href="http://www.biotechnologyforbiofuels.com/content/8/1/61">pathway</A>, we knew the propane yields weren’t very high. We thought about trying to enhance the system by searching for homologs for the enzymes, but thought this would be too time-consuming and also not very innovative. We then ran into a research article by <A href="http://www.nature.com/nmat/journal/v14/n1/full/nmat4118.html" target="_blank">Huber <I>et al.</I></A>. The group had designed a synthetic amphiphilic protein that spontaneously formed membrane-like structures inside the cell. These proteins were designed quite like membrane lipids: there is a hydrophilic and a hydrophobic end. According to the energy minimum principle, the proteins’ hydrophilic ends will face the liquid phase of the cell and the hydrophobic ends will pack together. This way the proteins will be able to form either a double layered vesicle (similar to the double lipid layer) or a micelle.</P><P>To both enhance the knowledge of these amphiphilic proteins and to gain better yield of propane, we thought of fusing enzymes to these proteins. This would bring the enzymes close together and possibly enhance their productivity. Because our system is so big, we needed to make some compromises - we didn’t think it was possible to attach all 10 of our pathway’s enzymes into these amphiphilic proteins, but decided to do it to two of the last enzymes: CAR and ADO. We chose these enzymes for two reasons, the first one being more significant. The product of CAR (and the substrate of ADO) is butyraldehyde, which is toxic to the cell. Therefore, if too much butyraldehyde is built up in the cell, our cells would die. Also, because of the toxicity, cells have many endogenous enzymes that consume butyraldehyde, thus reducing the amount of available butyraldehyde to be converted into propane by ADO. Our second reason has to do with our models suggesting that ADO is a bottleneck in our system. Therefore we would like to try to keep its substrate concentration high in the close proximity of ADO, so that it can function as efficiently as possible.</P><P>Our modelers built a model with Python to test our hypothesis about the close proximity of our enzymes, and it resulted in 200-400% increase in the final propane yield. Check out our <A href="https://2015.igem.org/Team:Aalto-Helsinki/Modeling_synergy"> Synergy Model</A> page to read more about this.

</P><P>In the situation where CAR and ADO are fused with the amphiphilic proteins, we expect the amphiphilic proteins to form micelles rather than membranes, as the enzymes will most probably repulse each other. This is why we are calling the system amphiphilic micelles, but in reality it doesn’t matter to us whether the formations are micelles or membranes because the enzymes will nevertheless be closer together in both formations.</P><P>We wanted to test our hypotheses with something we could easily detect. We found the Violacein pathway, which could by the use of three enzymes produce a detectable green color. We believe that if the violacein green color production could be enhanced by fusing these enzymes with the amphiphilic proteins, we could have an idea of whether the propane production could as well be enhanced in a similar way. There are obviously differences between these systems, but it would give us a rough estimate of whether these types of constructs would be possible to build in the first place.</P></SECTION><SECTION id="constructs" data-anchor="constructs"><H2 id="constructs">Constructs</H2><P><B>Assembly Methods</B></P><P>Both two of our propane plasmids and our cellulose plasmid were constructed with homologous pairing. Our main approach was <A href="https://www.neb.com/products/e2621-nebuilder-hifi-dna-assembly-master-mix" target="_blank">Gibson Assembly</A> which is based on &gt; 25 bp homologous regions between each piece that we want to combine. The Gibson Assembly master mix includes the enzymes exonuclease, DNA polymerase and ligase. The method functions so that the exonuclease starts degrading the parts 5’ ends and reveals the homologous regions as single stranded DNA. The exonuclease then inactivates because of the reaction temperature, and homologous pairing takes place. After this DNA polymerase will fill in any gaps that were produced by the exonuclease, and ligase ligates the pieces together. Figure 6 below describes the system when one insert is combined with a backbone. With Gibson, you can simultaneously combine up to at least 6 fragments. Our Propane Plasmid 1 and Cellulose Plasmid needed a four fragment assembly and Propane Plasmid 2 a five fragment assembly. The principle is the same regardless of how many pieces you wish to combine.</P><DIV style="margin-right:auto;margin-left:auto;max-width:70%"><FIGURE style="margin-bottom:3%;margin-top:3%"><FIGCAPTION><CENTER><B>Figure 6.</B> Gibson Assembly Method             </CENTER></FIGCAPTION></FIGURE></DIV><P>Our backup plans for Gibson Assembly were <A href="http://link.springer.com/article/10.1007%2Fs12033-014-9817-2#page-1" target="_blank">ELIC</A> and <A href="http://nar.oxfordjournals.org/content/32/2/e19.full" target="_blank">OE-PCR</A>. Of these, ELIC is a method that’s only been published in March 2015 and is thus poorly understood. Basically linear pieces of DNA, which contain homology to one another are transformed into a cell, and the cell’s endogenous homologous recombination system assembles these into a plasmid. OE-PCR also takes advantage of the homologous regions, but uses these as primers. The PCR program follows the regular guidelines, but  the annealing temperature is based on the homologous region. As the strands of DNA detach from each other, the homologous area functions as a primer that allows the polymerization of  the whole construct.</P><P><B>Backbones</B></P><FIGURE style="float:right; margin-left:3%;margin-top:-3%;"><FIGCAPTION><CENTER><B>Figure 7.</B> gBlock Assembly taking advantage of homologous regions between the blocks</CENTER></FIGCAPTION></FIGURE><P>As we ordered our constructs as gBlocks, they needed to be divided into up to 2000 bp pieces. We added 30 bp overlaps to each of our piece to enable the Gibson Assembly for our pieces. Due to a design mistake though, we didn’t design a 30 bp overlap between our Brick and the backbone: we thought we would be able to construct the bricks as linear DNA and ligate them into the backbone with the help of restriction enzymes. We did however have the prefix on all of our first brick-to-be pieces and had added the suffix with PCR. We hoped that the homologous area created by the prefix and suffix (about 15 bp) would be enough to attach our construct to the BioBrick backbone <A href="http://parts.igem.org/Part:pSB1C3" target="_blank">pSB1C3</A>. See figure 7 for gBlock assembly.</P><FIGURE style="float:left;margin-right:20px;"><FIGCAPTION><CENTER><B>Figure 8.</B> Adding homologous regions between the insert and backbone</CENTER></FIGCAPTION></FIGURE><P>The situation became a bit more complicated when it came to attaching the Propane Plasmid 2 and the Cellulose Plasmid into the commercial pACYCDuet-1 and pCDFDuet-1 vectors. As these were cloning vectors, they already contained the T7 promoter, RBS and terminator. Better yet, since they are Duet plasmids, they contain all of these for two different genes. To avoid the risk of our plasmid producing an unwanted product from another vector promoter, we restricted the second expression system out completely. As our Propane Plasmid 2 construct was based on Kallio’s plasmid maps, which were also built into Novagen’s vectors, our T7 and the first RBS were the same as in the vectors. Thus, after our construct is in the <A href="http://parts.igem.org/Part:pSB1C3" http:="">biobrick backbone</A>, we will use new primers which will remove the prefix of our constructs 5’ end and add a homologous region to the vector in the 3’ end (31 bp). With the Cellulose Plasmid, the T7 and RBS sequences differed from Novagen’s vectors, so we needed to design a forward primer as well which created a homologous region for the vector in our constructs 5’ end. Figure 8 on top left describes the primer design to transfer the Cellulose Brick into a commercial backbone and figure 9 below the transfer of a propane plasmid to a commercial backbone.</P><FIGURE style="float:left;margin-right:20px;"><FIGCAPTION><CENTER><B>Figure 9.</B> Removing the second expression system from Duet plasmids. (The pink areas represent homology)</CENTER></FIGCAPTION></FIGURE><P><B>Amphiphilic constructs</B></P><P>To add the amphiphilic proteins, we used PCR to remove the terminator and the suffix from the end of our Propane Plasmid 1 and Propane Plasmid 2 Bricks. Our primers added a region to the 3’ end of our Propane bricks to create homology with the amphiphilic protein and its linker. The amphiphilic protein was synthesized including one linker, and for CAR we added a second one with a primer. A suffix had been added to the 3’ end of the synthesized amphiphilic protein with PCR primers.</P><P>These same methods were supposed to be used to create the Violacein construct with and without amphiphilic fusions, but due to the time constraints, we weren’t able to assemble any of our Violacein Plasmids.</P></SECTION><SECTION id="cont" data-anchor="cont"><H2 id="cont" style="clear:both">Continuous Production</H2><P>The project is based on research done by <A href="http://www.nature.com/ncomms/2014/140902/ncomms5731/full/ncomms5731.html" target="_blank">Pauli Kallio <I>et al.</I></A> from the University of Turku. In the article, it was proposed that the propane producing capacity could be also tested with continuous production where the cells are grown until the steady state is achieved. Furthermore, when scaling up to industrial processes, reactor fermentation with continuous state would be commercially the best solution. Flowing product gas would be easily gathered and not disturb microbes' population sizes needed for this scale.</P><P>The steady state will be maintained with the regular feed of fresh media into a chemostat while the outline will continuously remove the same amount of media with products. Thus, the growth rate can be readily controlled by changing the diluting speed of media. Kallio's team were kind to send us their <I>E. coli</I> strain BL21 (DE3 ΔyjgB ΔyqhD, pET-TPC4 + pCDF-cAD + pACYC-Fdx-Fpr) which produces propane with the reaction pathway utilizing the intermediates of fatty acid biosynthesis and enzymes like FASII and Tes4. Onwards from the butyrate, the reaction pathway’s enzymes are similar to our reactions'. The strain has already been cultivated on a bench scale using an Erlenmeyer flask.</P><FIGURE style="float:right"><FIGCAPTION><CENTER><B>Figure 10.</B>Chemostat reactor  with feed- and outline</CENTER></FIGCAPTION></FIGURE><P>Because the strain BL21 (DE3 ΔyjgB ΔyqhD, pET-TPC4 + pCDF-cAD + pACYC-Fdx-Fpr) produces propane only by inducing with IPTG instead of constitutive production, certain limitations are met when designing the chemostat conditions. Cultivating microbes with a maximum capacity would consume too much IPTG which is quite a valuable reagent, as a diluting rate needs to be high. Therefore, instead of exploring the maximum yield by adding abundantly IPTG, the chemostat will be used for investigating how the environmental conditions affect the product formation. The most simple way to do this is to wait until the steady state is achieved without any induction and then add IPTG. To see how change of process conditions affects the cell density, OD600 is measured all the time.</P><P style="margin-bottom:0;padding-bottom:10%">The overall media volume of chemostat will be 500 ml due to the limit of the IPTG amount. The steady state is approximated to take 10 hours but it may vary radically. Diluting rate is kept to 0,072 1/h and glucose feed 20 g/L. When OD600 reaches its balance, IPTG is added to the media to a concentration of 2,0 mM. Induction will take another 10 hours so the overall process time will be probably few days. Gas products will be measured with GC-MS analysator which will be connected with the outlet stream of the chemostat. Because the reactor size is limited (2 L), only few sensors can be attached to the same unit. We will measure the cell density, pH, dissolved oxygen and composition of outstream gas.</P></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>