<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCLA_Notebook_Protein_Cages_24_June_2015 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCLA/Notebook/Protein Cages/24 June 2015</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM UCLA</TITLE></P><DIV id="top_menu"><UL id="menu"><LI><A href="/Team:UCLA">Home</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Team</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Meet the Team</A></LI><LI><A href="/Team:UCLA/Attributions">Attributions</A></LI><LI><A href="/Team:UCLA/Collaborations">Collaborations</A></LI></UL><LI><A href="/Team:UCLA/Project">Projects</A></LI><LI><A href="/Team:UCLA/Project/Programming Spider Silk">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Project/Functionalizing Silk">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Project/Honeybee Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Project/Protein Expression and Processing">Materials Processing</A></LI><LI><A href="/Team:UCLA/Project/Protein Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Practices">Human Practices</A></LI><LI><A href="/Team:UCLA/Project/Interlab Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Parts">Parts</A></LI><LI><A href="/Team:UCLA/Parts">Team Parts</A></LI><LI><A href="/Team:UCLA/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:UCLA/Composite_Parts">Composite Parts</A></LI><LI><A href="/Team:UCLA/Part_Collection">Part Collection</A></LI><LI><A href="/Team:UCLA/Notebook/Weekly Notes">Notebook</A></LI><LI><A href="/Team:UCLA/Notebook/Spider_Silk_Genetics">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Recombinant_Expression">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Honeybee_Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Materials">Materials Processing</A></LI><LI><A href="/Team:UCLA/Notebook/Protein_Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Notebook/Interlab_Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Safety">Safety</A></LI><LI><A href="https://igem.org/Judging_Form.cgi?id=1763">Judging</A></LI></DIV><P>Phillip's notes
</P><P>Introduction:  Today will be focused on designing the complete constructs to be ordered.  The sequence from Yen will be used.  They will be designed with the aid of Fasih, in order to utilize Benchling efficiently.  The best candidates to be ordered will also be selected, and if time permits, design of primers for the remaining ones will begin.
</P><P>-Attempt without Fasih
The sequence from Yen was translated using ExPASy to
</P><P>MPFITVGQENSTSIDLYYEDHGTGTPVVLIHGFPLSGHSWERQSAALLDAGARVITYDRR
GFGQSSQPTTGYDYDTFAADLNTVLETLDLQDAVLVGFSMGTGEVARYVSSYGTARIAAV
AFLASLEPFLLKTDDNPDGAAPQEFFDGIVAAVKADRYAFYTGFFNDFYNLDENLGTRIS
EEAVRNSWNTAASGGFFAAAAAPTTWYTDFRADIPRIDVPALILHGTGDRTLPIENTARV
FHKALPSAEYVEVEGAPHGLLWTHAEEVNTALLAFLAKAQEAQKQKLLTEVETYVLSIIP
SGPLKAEIAQRLEDVFAGKNTDLEVLMEWLKTRPILSPLTKGILGFVFTLTVPSERGLQR
RRFVQNALNGNGDPNNMDKAVKLYRKLKREITFHGAKEISLSYSAGALASCMGLIYNRMG
AVTTEVAFGLVCATCEQIADSQHRSHRQLEHHHHHH-
</P><P>Where – signifies stop.
</P><P>This was then backtranslated in Gene Designer to avoid EcoRI, NotI, XbaI, SpeI, and PstI.
</P><P>These were the results.  With an added TGA for stop.  Note the C-terminal HisTag!
</P><P>&gt;PCquad DNA sequence
ATGCCGTTTATTACCGTGGGCCAAGAAAATTCTACGTCTATTGATTTGTATTATGAGGACCATGGTACCGGTACGCCGGTGGTTCTCATCCACGGCTTTCCGCTATCAGGACATTCTTGGGAGCGTCAGAGCGCTGCGCTTTTAGATGCCGGTGCTCGTGTAATAACGTACGATAGACGCGGTTTTGGCCAGAGCTCTCAGCCAACGACTGGATACGATTATGACACCTTCGCCGCCGATTTAAATACTGTTCTGGAAACCCTGGATCTTCAGGATGCGGTCTTAGTTGGTTTTAGTATGGGCACAGGTGAAGTTGCCCGCTACGTCAGTTCTTATGGCACTGCTCGTATTGCAGCGGTAGCTTTTCTGGCTAGCTTAGAACCCTTTCTATTAAAAACCGATGATAATCCGGATGGGGCGGCTCCACAGGAGTTTTTCGACGGAATTGTGGCCGCTGTGAAAGCCGATAGATATGCTTTCTATACTGGGTTTTTCAATGACTTCTATAATTTAGATGAAAACCTGGGAACACGCATCTCGGAAGAGGCTGTACGGAACTCATGGAATACTGCGGCATCTGGCGGATTCTTTGCTGCCGCAGCCGCGCCGACCACTTGGTATACAGATTTTCGTGCGGACATTCCTAGAATTGATGTACCTGCCCTGATTCTGCACGGTACGGGCGACCGTACCTTACCGATTGAGAACACTGCCCGCGTCTTTCATAAAGCTCTTCCGTCCGCTGAGTACGTAGAGGTTGAAGGGGCACCTCATGGTCTGTTATGGACTCATGCTGAAGAAGTCAACACAGCCCTGCTTGCTTTCCTTGCGAAGGCTCAAGAAGCCCAAAAGCAGAAATTACTGACCGAAGTGGAAACTTATGTACTATCCATCATACCGTCTGGTCCCCTTAAAGCAGAAATCGCACAACGTCTGGAGGATGTCTTCGCGGGCAAAAATACAGACCTGGAGGTGCTGATGGAATGGCTTAAGACCCGACCGATTCTGTCACCGTTAACGAAGGGTATCCTAGGATTCGTTTTTACCCTGACGGTGCCCAGTGAGCGCGGACTGCAACGAAGAAGATTCGTCCAAAACGCATTAAACGGGAATGGTGACCCAAACAATATGGACAAAGCGGTGAAGCTGTATCGAAAACTTAAGCGCGAAATAACATTCCATGGGGCCAAAGAAATTAGCCTGAGCTATTCCGCTGGGGCGCTTGCTTCTTGTATGGGTTTAATATATAACCGAATGGGTGCGGTCACCACGGAAGTCGCGTTCGGATTAGTATGCGCGACATGCGAGCAGATTGCAGACTCTCAGCATAGAAGCCATCGCCAGTTAGAACATCATCACCATCATCATTGA
</P><P>&gt;PCquad amino acid sequence
MPFITVGQENSTSIDLYYEDHGTGTPVVLIHGFPLSGHSWERQSAALLDAGARVITYDRRGFGQSSQPTTGYDYDTFAADLNTVLETLDLQDAVLVGFSMGTGEVARYVSSYGTARIAAVAFLASLEPFLLKTDDNPDGAAPQEFFDGIVAAVKADRYAFYTGFFNDFYNLDENLGTRISEEAVRNSWNTAASGGFFAAAAAPTTWYTDFRADIPRIDVPALILHGTGDRTLPIENTARVFHKALPSAEYVEVEGAPHGLLWTHAEEVNTALLAFLAKAQEAQKQKLLTEVETYVLSIIPSGPLKAEIAQRLEDVFAGKNTDLEVLMEWLKTRPILSPLTKGILGFVFTLTVPSERGLQRRRFVQNALNGNGDPNNMDKAVKLYRKLKREITFHGAKEISLSYSAGALASCMGLIYNRMGAVTTEVAFGLVCATCEQIADSQHRSHRQLEHHHHHH
The sequence was then annotated on Benchling.  Using PDB ID: 1bro and 1aa7 (bromoperoxidase and M1 protein), the linker was identified.  
</P><P>Next, the full insertion sequence was also generated on Gene Designer.
&gt;Mutant Cage Full Insertion
GGCCTGGTTCCCCGCGGTTCAGGC
&gt;Full Insertion
GLVPRGSG
This allowed easy insertion of the sequences.  The full sequence was inserted where needed after removing redundant bases.
</P><P>For the prefix and suffix, the sequence was obtained from this site.  The first prefix sequence was used.  
<A rel="nofollow" class="external free" href="http://parts.igem.org/Help:BioBrick_Prefix_and_Suffix">http://parts.igem.org/Help:BioBrick_Prefix_and_Suffix</A>
prefix-NdeI-gene-XhoI-Histag-suffix
</P><P>Concluding remarks:
Before the rest of the sites are revisited, a meeting will be set up with David and/or Fasih to check our design rationale.
</P><P>Intro: today we realized that our work the previous day (June 23) was not useful. We had used Dr. Yates' original cage sequence instead of the optimized sequence, which would yield poorer results. Thus, the same procedure of using Benchling to design new cages with the inserted thrombin recognition sites was performed with the new cage sequence above. 
</P><P>Procedure:
</P><P>1. The cage sequence optimized by Dr. Yates was copied into a benchling file.
</P><P>2. The thrombin cleavage site was inserted at the area of interest of each site.
</P><P>3. The resultant sequence was then codon optimized. Integrated DNA Technologies Codon Optimizer, set to Escherichia coli K12, was used. <A rel="nofollow" class="external free" href="https://www.idtdna.com/CodonOpt">https://www.idtdna.com/CodonOpt</A></P><P>4. The optimized sequence was then cross checked with benchling’s digest function for EcoRI, PstI, SpeI, and XbaI restriction sites. 
</P><P>5. Any residues containing the restriction site were exchanged with different codons to preserve the reading frame and remove undesired digestion.
</P><P>Tyler Lee
</P><P>
1. Recreated Mutant Sequences(9-13) using PCquad sequence from Yeates(fully optimized)
</P><PRE>      -PCquad sequence was copied onto Benchling and the mutation was inserted onto this sequence
      -Then this sequence was codon optimized using the Gene Designer tool 
      -Then the sequence was checked for containing any iGEM restriction enzymes(Pst,EcoRI, XbaI, SpeI,NotI) and were removed by replacing nucleotides to avoid unwanted digestion. 
</PRE><P>
Nithin D
</P></DIV></DIV></DIV></DIV></BODY></HTML>