Team:UCLA/Notebook/Protein Cages/23 June 2015
iGEM UCLA
Home
Team
Meet the Team
Attributions
Collaborations
Projects
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Human Practices
Interlab Study
Parts
Team Parts
Basic Parts
Composite Parts
Part Collection
Notebook
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Interlab Study
Safety
Judging
Today I designed sites 5-8 using benchling.
Procedure:
1. The wt cage sequence was copied into a benchling file.
2. The thrombin cleavage site was inserted at the area of interest of each site.
3. The resultant sequence was then codon optimized. Integrated DNA Technologies Codon Optimizer, set to Escherichia coli K12, was used. https://www.idtdna.com/CodonOpt
4. The optimized sequence was then cross checked with benchling’s digest function for EcoRI, PstI, SpeI, and XbaI restriction sites.
5. Any residues containing the restriction site were exchanged with different codons to preserve the reading frame and remove undesired digestion.
Tyler Lee -- Wtleeiv 18:27, 23 June 2015 (CDT)
1.Created Mutant Sequences (9-13) using WT cage sequence from Yeates paper(not fully optimized)
- WT cage sequence was copied onto Benchling and the mutation was inserted onto this sequence    - Then this sequence was codon optimized using the IDT Codon Optimizer tool     - Then the sequence was checked for containing any iGEM restriction enzymes(Pst,EcoRI, XbaI, SpeI,NotI) and were removed                            replacing nucleotides to avoid unwanted digestion.
2.Helped shoot video for Spark Video.
-Nithin D.
Phillip's notes:
Introduction:  For today, the goal is to become familiarized with Benchling usage, and the general idea behind creating the constructs.  First, the Yeate’s construct will be designed based off of Dr. Yeate’s papers in the methods.  From there, the mutants will be created on Benchling.
1.  Identifying important notes about the protein cage: The sequence from Yen Ting, Dr. Yeate’s former graduate student, was obtained from email with Yeates (June 4, 2015).  See supplemental material for this paper for plasmid preparation, protein overexpression, and purification. From the paper,
Plasmid preparations
"A codon-optimized DNA sequence of the fusion protein was generated by the DNAworks server.1 A DNA fragment encoding the fusion protein was assembled by recursive PCR. The PCR product was inserted into a pET22b vector through NdeI and XhoI cutting sites. A C-terminal His-tag on the vector was included to facilitate protein purification by metal affinity purification. Mutations were introduced by QuikChange (Agilent). Cloning was done in E. coli strain XL1Blue. The resulting plasmids were verified by DNA sequencing.”
“The sequence for PCquad is attached below. This gene was inserted into the pET22b vector between NdeI and XhoI sites. Sequence was optimized by DNAworks.
ATGCCGTTCATCACCGTTGGTCAGGAAAATTCTACCTCTATCGACCTGTA  CTACGAGGACCACGGTACTGGTACTCCGGTTGTTCTGATCCACGGTTTCC  CGCTGTCTGGCCACTCTTGGGAACGCCAAAGCGCCGCTCTGCTGGACGCA  GGTGCTCGTGTTATCACCTACGACCGTCGTGGTTTCGGTCAGTCTTCCCA  GCCGACGACCGGTTACGACTACGACACCTTTGCGGCGGACCTGAACACTG  TTCTGGAAACCCTCGACCTCCAGGACGCGGTCCTGGTTGGTTTCTCTATG  GGTACCGGCGAAGTGGCGCGCTACGTCTCTTCTTACGGCACGGCGCGTAT  CGCGGCGGTTGCGTTCCTCGCTTCCCTGGAACCTTTCCTGCTGAAGACCG  ACGATAACCCGGACGGTGCTGCGCCGCAGGAATTCTTCGACGGTATCGTG  GCGGCCGTTAAAGCCGACCGTTACGCGTTTTACACCGGCTTTTTCAACGA  CTTCTACAACCTGGACGAAAACCTGGGTACTCGTATCTCTGAAGAAGCGG  TTCGTAACAGCTGGAACACGGCTGCGTCTGGCGGTTTCTTCGCGGCTGCC  GCTGCCCCGACCACCTGGTACACCGATTTTCGTGCAGACATCCCGCGCAT  CGACGTTCCGGCGCTGATCCTGCACGGCACTGGTGATCGTACGCTGCCGA  TCGAAAATACTGCGCGTGTTTTCCATAAAGCTCTGCCGTCTGCGGAATAC  GTCGAGGTGGAGGGTGCTCCGCACGGCCTGCTCTGGACCCACGCGGAAGA  AGTTAACACCGCGCTGCTCGCGTTTCTCGCCAAGGCGCAGGAAGCGCAGA  AACAGAAACTCCTGACGGAAGTCGAAACCTACGTTCTGTCTATCATCCCG  TCTGGTCCGCTGAAGGCGGAAATCGCGCAGCGTCTCGAAGATGTTTTTGC  GGGTAAAAACACCGACCTGGAAGTTCTCATGGAATGGCTCAAAACCCGTC  CGATCCTGTCTCCGCTCACTAAAGGTATCCTGGGCTTCGTTTTCACCCTG  ACCGTACCGAGCGAACGTGGTCTGCAACGTCGCCGTTTCGTTCAGAACGC  GCTGAACGGTAATGGTGACCCGAACAACATGGACAAGGCGGTGAAACTCT  ACCGTAAACTGAAACGTGAGATCACCTTCCATGGTGCGAAAGAAATCTCT  CTGTCTTACTCTGCGGGTGCGCTGGCGTCTTGCATGGGTCTGATTTACAA  CCGTATGGGTGCGGTTACCACCGAAGTCGCGTTCGGCCTGGTTTGCGCGA  CCTGCGAACAGATCGCGGACTCTCAACACCGTTCTCACCGTCAGCTCGAG  CACCACCACCACCACCACTGA
Using BLAST, a sequence alignment was ran using Yen’s sequence and the one Anuved used.  The results showed no significant sequence similiarity. Upon looking into the sequences, it was found that our original gBLOCK was designed from the PDB file, which was from the paper 2.  The most recent, which may be the one from Yen, is this paper 1.
2.  It may be possible that the mutants need to be redesigned with the most up to date sequence from Yen’s email.  For now, the main components of the plasmid will be assembled for Fasih to assist in design tomorrow.
For iGEM:  RFC10 as a standard. http://parts.igem.org/Help:Standards/Assembly/RFC10 RE sites to avoid:  EcoRI, NotI, XbaI, SpeI, PstI.   Add Prefix and Suffix.  Use one designed for protein coding regions.
1. Lai, Y.-T. & Al, E. Structure and Flexibility of Nanoscale Protein Cages Designed by Symmetric Self-Assembly. J Am Chem Soc 135, 7738–7743 (2013).
2. Lai, Y.-T., Cascio, D. & Yeates, T. O. Structure of a 16-nm Cage Designed by Using Protein Oligomers. Science (80-. ). 336, 1129–1129 (2012).
