<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCL_Software skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCL/Software</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>'

</P><NAV class="mainmenu"><DIV id="menu" class="menu"><DIV class="menu-items"><LI><A href="https://2015.igem.org/Team:UCL/Description">Project</A></LI><LI><A href="#">Lab work</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Effectors">●  Psychobiotics</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Sensors">●  Mood Detection</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Safety">●  Built-in Biosafety</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Design">● Gut-on-a-Chip</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Fermentation">● Scaling Up</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="#">Process</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Interlabstudy">●  InterLab Study</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Collaborations">●  Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Notebook">●  Notebook</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Protocols">●  Protocols</A></LI><LI><A href="#">Outcomes</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Software">● Software</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Bactoman">● Bactoman</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Parts">● Parts</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Achievements">●  Achievements</A></LI><LI><A href="#">People</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Team">● Team</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Sponsors">● Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Attributions">● Attributions</A></LI></DIV></DIV></NAV><DIV id="hidden-menu"><UL id="mobile-list"><LI><A href="https://2015.igem.org/Team:UCL">Home</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Effectors">Psychobiotics</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Sensors">Mood Detection</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Design">Gut-on-a-Chip</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Fermentation">Scaling Up</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Bactoman">Bactoman</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Interlabstudy">InterLab Study</A></LI></UL></DIV></DIV></DIV><DIV id="header2"><DIV id="wrapper"><DIV class="bigtitle" style="color:white"><DIV class="titlecircle"><SPAN class="title2">Software</SPAN><P style="margin-top: 5%; line-height: 1.7;font-family:Raleway;">
              We have developed <B>two incredible tools</B> to help not only our project but all of iGEM! <SPAN class="arrow"><A data-scroll="" data-options="{ &quot;easing&quot;: &quot;linear&quot; }" href="#submenu">▼</A></SPAN></P></DIV></DIV><DIV id="submenu"><A data-scroll="" data-options="{ &quot;easing&quot;: &quot;linear&quot; }" href="#mark" class="buttonblack">Mind-the-Mark</A><A data-scroll="" data-options="{ &quot;easing&quot;: &quot;linear&quot; }" href="#growth" class="buttonblack">Mind-the-Growth</A></DIV><DIV id="members"><DIV class="memberwrapper"><H2>Mind-the-Mark</H2><DIV style="width:90%;padding:5%;border:1px solid grey;"><H3 style="margin-top:40px;">The Tool</H3><P style="margin:0 0 0 20px;">By taking advantage of Google's easy to use extension system for the Chrome Browser, we devised a powerful
      and lightweight extension that transforms the iGEM wiki editor on whatever page you choose to edit; be it in the
      future in 2020 or now in 2015. The extension is easily downloaded and then drag-and-dropped into the Chrome browser,
      to install, or installed with a click from the chrome store. Then its working!</P><TABLE style="width:100%; margin:20px auto; z-index:999;"><TBODY><TR><TD><DIV class="bigCenterText">Before</DIV></TD><TD><DIV class="bigCenterText">After</DIV></TD></TR></TBODY></TABLE><H3>Description</H3><P>
      The Mind-the-Mark tool takes the bland and boring wiki input box and transforms it into a proper code editor with several
      interactive and assistive features. Having talked to several iGEM teams this year, and years past, one of the 
      issues they lamented over was the issue of entering code into the wiki. The bland and simple text input on each 
      page made developing interactive and pretty wikis a real hassle. Due to this many teams developed their code
      externally and then transfered it in, costing them time and effort. We have changed that.
    </P><H3>Features</H3><UL><LI>Syntax Markup for HTML, XML, CSS, Javascript</LI><LI>Customized Layout and Design</LI><LI>Tag Highlighting and linking</LI><LI>Line Count</LI><LI>Ctrl-F for Finding, replacing, and searching with RegExp</LI><LI>Open Source and Fully Changeable</LI><LI>Much More!</LI></UL><P>For a more indepth update history and singular repository see the following <A href="https://github.com/ucligem2015/Mind-The-Mark" style="color:blue;"> link</A> beside the one below</P><DIV style="font-weight:bold;font-size:20px;width:20%;margin:30px auto;text-alignment:center;">Download the Tool!</DIV><DIV style="width:100%;display:inline-flex;margin-bottom:20px;"><DIV style="width:50%;text-align:center;font-size:20px;"><DIV>GitHub</DIV></DIV><DIV style="width:50%;text-align:center;font-size:20px;"><DIV>Chrome Store</DIV></DIV></DIV></DIV><H3>Construction</H3><P>The spirit of the tool is to make editing and working on the iGEM wiki easier and more interactive for all.
      It was with this spirit that we built the tool. Although the code is part of chrome extension, it is really
      a lightweight javascript tool.</P><H3>What is a Chrome Extension?</H3><P>A chrome extension at its base is a <B>.json</B> file called the &quot;Manifest&quot; file that contains the file
      reference's and basic information about the tool. When used this tells the browser to load a piece of javascript or other files 
      on to specified pages. These then interact with the DOM (Document Object Model) of the page.</P><H3>Why use a Chrome Extension?</H3><P>Chrome is one of the most modern and widely used browser on the internet and has an easy to use and develop
      extension system, on top of a very rich API. When loading files that interact with the DOM chrome extensions
      also make sure that there is no interaction with any of the loaded scripts of libraries from the page, protecting
      the user. To top it all off, the chrome extensions are very easy to install and distribute.</P><H3>How did we Implement it?</H3><P>Our tool uses the commonly implemented &quot;Content_Scripts&quot; method for loading our extension. This allows
      chrome to load specific js and css files whenever a URL, defined by a certain patern, is loaded. In our case
      we load both js and css whenever an iGEM edit page is loaded. The javascipt that is loaded
      is the <B>background.js</B> file which adds a script refecence to the page, loads the script, and then deletes it.
      In this instance the script in question is the <B>script.js</B> file. The extension also has the ability to 
      access other js and css file which are referenced in the Manifest file. The reason the background.js file is used
      to then load the script file is that it allows for a cleaner separation of files as well as a faster loading solution
      when the scipt is large.</P><P>The script.js file contains the source code and several add-on pieces of code from a js library called CodeMirror.
      This provides the majority of functions that alter the page. It also contains the final function that runs when the 
      Document OnLoad call is made. This final function finds the textarea tag by its ID and makes the call to CodeMirror
      to change the textarea into the new snazzy code editor. CodeMirror then alters the shape of the divs in the tag
      and applies the css. As well as defining and instigating a few other features, this performs the majority of the 
      work</P><P>By packing the majority of the js into one file it reduces the overall space required and time neaded to load.
      That being said it is well commented and laid out so that it is easily editable in the future. The same runs
      true for the css.</P><H3>Future Work</H3><P>There are several features we would like to implement in the future to make this the best tool, and the only
      as far as we know, for editing the iGEM wiki. Javascipt, CSS, and HTML are all currently supported but providing 
      syntax for SCSS and other CSS variants, as well as PHP would complete the language mark up types. Providing 
      code suggestion as well as auto complete would make this a tool to rival the best code editors out there.</P><P>Another project we have begun work on is developing a larger tool coaltes our tool with other team's
      to create a complete IDE for developing iGEM wikis.</P><P>This tool will and can make a massive difference to teams current and future by making the process of creating 
      a wiki infinitely more easy and enjoyable to do!</P><H2>Mind-the-Growth</H2><DIV style="width:90%;padding:5%;border:1px solid grey;"><H3>The Tool</H3><P>When we first discussed adding a probiotic bacteria into the human gut one of the questions 
      that come up was what effect would it have on the human’s microbiome.  
      In order to understand and gain some insight into the functioning of the 
      human microbiome we decided to model it! </P><P>Many of the current models that model the human microbiome are descriptive, and use
      mathematical models and other logical operations to create a system that mimics the 
      gut loosely. 

<SPAN class="content"><SPAN class="tooltip"><A href="http://www.sciencedirect.com/science/article/pii/S0022283614001788" target="_blank"><B>[Link]</B></A>Bucci v. and Xavier J. Towards Predictive Models of the Human Gut Microbiome, Journal of Molecular Biology Volume 426, Issue 23, 25 November 2014, Pages 3907–3916
</SPAN><B>[1]</B></SPAN>

		One of the challenges is that although there exist complex models of single cell
      dynamics, that use operations like mean-field partial differential equations, the
      computational power needed to expand any discrete models to a population is very high. 
	</P><P>A potential solution to this is using metabolic analysis as a descriptive tool 
      for modelling the activity of the populations. What we have done is built a lightweight
      framework in both C++ and in C# to enable researchers to expand and build upon this
      concept in the future. Not only does this tool render cell population graphics in a physics 
      engine but it also has been used to examine the effect that consuming probiotics and prebiotics 
      (inulin in this case) has on the gut microbiome. </P><P>For a more indepth update history and singular repository see the following <A href="https://github.com/ucligem2015/Mind-The-Growth" style="color:blue;"> link</A> beside the one below</P><DIV style="font-weight:bold;font-size:20px;width:20%;margin:30px auto;text-alignment:center;">Download the Tool!</DIV><DIV style="width:100%;display:inline-flex;margin-bottom:20px;"><DIV style="width:100%;text-align:center;font-size:20px;"><DIV>GitHub</DIV></DIV></DIV></DIV><H3>Construction</H3><P>
Cellular automata are discrete models used across a variety of disciplines. The model consists of a grid of cells that are assigned one of several possible states which can be thought of as the identity of the cell. Each of these possible states has a set of rules which determine if and how the state of the that cell will change as time progresses. The model operates on discrete time, so all the states at t+1 are determined by how the grid looks at time t.
  </P><P>
The purpose of the creation of this particular model was to determine the optimum way in which a combination of probiotics and prebiotics can be used to change an initial microbiome profile to a pre-determined one.
  </P><P>
This model is a simplification that offers a framework to which more variables can be added, as well as experimental data.
  </P><P>
The states 1-5 represent different five different bacterial phyla, with state 6 representing all other phyla.
  </P><P>
The bacteria were randomly generated on a grid of cells according to a pre-determined proportion using a random number generator. As time progressed the state of each cell on the grid changed according to a set of rules that took into account the following variables:
  </P><UL><LI>
The states of the eight cells that surround it.</LI><LI>
  Durability and strength constants that were defined for each of the phyla
  </LI><LI>
  A cooperation/competition constant that take into account the proportion of the overall population that belongs to the same phyla
  </LI><LI>
  A constant to take into account each phylum’s response to the prebiotic (inulin)
  </LI></UL><P>
Each of these factors is set to zero or one in the source code, so as to eliminate its influence on the model. Each user can then set the constants as he or she sees fit.
  </P><P>
The system is disrupted periodically by the addition of a dose of Prokao containing inulin and bacteria of different phyla, whose values can also be adjusted as needed.
  </P><H3>Future Work</H3><P>
Given the amount of variables involved we would need to incorporate machine learning, such as a genetic algorithm, in order to come up with a composition of Prokao that’s even close to the ideal. </P><P>Quorum sensing could be incorporated by having particular actions at a distance between cells, and have those effects diminish with the distance between them.</P><P>More complex functions based on particular limiting nutrients could be used to define the nature of cooperation and competition between different states.</P><P>Experimental data could be interpreted in the context of the model and used to make the simulation more accurate. Current theoretical frameworks could also be incorporated in future versions of this model.</P><P>More possible states could be represented, in this case going down to the genus or even species level. With overarching characteristics that defining a phyla and particular characteristics to each genus and so on. Non-bacterial actors such as archaea, fungi and parasites could also be considered.</P><P>The “other” state could be more accurately modelled by having each of it’s cells behave in a different way and thus eliminating any competition or cooperation between them.</P><P>Empty space can be considered as one of the possible states, thereby making it possible to model initial colonisation.</P><P>Limiting factors which increase periodically such as carbon sources or specific nutrients which are essential for the growth of particular bacteria.</P><P>Cellular products could be produced, consumed and accumulated leading to different effects.
  </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>