<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCLA_Notebook_Protein_Cages_17_July_2015 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCLA/Notebook/Protein Cages/17 July 2015</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM UCLA</TITLE></P><DIV id="top_menu"><UL id="menu"><LI><A href="/Team:UCLA">Home</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Team</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Meet the Team</A></LI><LI><A href="/Team:UCLA/Attributions">Attributions</A></LI><LI><A href="/Team:UCLA/Collaborations">Collaborations</A></LI></UL><LI><A href="/Team:UCLA/Project">Projects</A></LI><LI><A href="/Team:UCLA/Project/Programming Spider Silk">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Project/Functionalizing Silk">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Project/Honeybee Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Project/Protein Expression and Processing">Materials Processing</A></LI><LI><A href="/Team:UCLA/Project/Protein Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Practices">Human Practices</A></LI><LI><A href="/Team:UCLA/Project/Interlab Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Parts">Parts</A></LI><LI><A href="/Team:UCLA/Parts">Team Parts</A></LI><LI><A href="/Team:UCLA/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:UCLA/Composite_Parts">Composite Parts</A></LI><LI><A href="/Team:UCLA/Part_Collection">Part Collection</A></LI><LI><A href="/Team:UCLA/Notebook/Weekly Notes">Notebook</A></LI><LI><A href="/Team:UCLA/Notebook/Spider_Silk_Genetics">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Recombinant_Expression">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Honeybee_Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Materials">Materials Processing</A></LI><LI><A href="/Team:UCLA/Notebook/Protein_Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Notebook/Interlab_Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Safety">Safety</A></LI><LI><A href="https://igem.org/Judging_Form.cgi?id=1763">Judging</A></LI></DIV><P>Phillip's notes
</P><P>Introduction:  Yesterday, Nithin performed a PCR amplication of our PCquad optimized for iGEM 2.0 with the primers to add the his-tag-stop, designed by Tyler.  The reaction was also done on a gradient of temperatures, in order to determine an optimal annealing temperature.  Upon visualization of the gel, it appears that there is a much stronger band around 250bp, though the predicted size around 1.4kb was visible.  
</P><P>An attempt was made at determining what the non-specific band was.  Since a self-dimer/hetero-dimer was unlikely, due the size, the non-specific band was speculated to be somewhere in the middle of the gene. Using clustal omega, a sequence alignment was done using the following:
1.	 PCquad Optimized for iGEM 2.0
2.	PCquad primer iGEM prefix
3.	PCquad primer iGEM suffix #1
It was shown that only the ends, as intended, annealed to the primers.  The source of the non-specific band is currently undetermined.
</P><P>
Procedures:
</P><P>PCR amplification of construct attempt #2
Melting temperature for the reaction was determined by the NEB Tm calculator, and the part of our primer that annealed to the template.  
Nithin prepared the reaction samples for 62C, 66C, and 70C.  0.5ng was loaded instead of 0.25ng.
</P><P>Protocol for Q5 from NEB:
</P><P>1.	Please note that protocols with Q5® High-Fidelity DNA Polymerase may differ from protocols with other polymerases. Conditions recommended below should be used for optimal performance.
</P><P>
Reaction Setup:
</P><P>We recommend assembling all reaction components on ice and quickly transferring the reactions to a thermocycler preheated to the denaturation temperature (98°C). All components should be mixed prior to use. Q5 High-Fidelity DNA Polymerase may be diluted in 1X Q5 Reaction Buffer just prior to use in order to reduce pipetting errors.
</P><P>Component	25 µl Reaction	50 µl Reaction	Final Concentration
</P><P>5X Q5 
</P><P>Reaction Buffer	5 µl	10 µl	1X
</P><P>10 mM dNTPs	0.5 µl	1 µl	200 µM
</P><P>10 µM Forward Primer	1.25 µl	2.5 µl	0.5 µM
</P><P>10 µM Reverse Primer	1.25 µl	2.5 µl	0.5 µM
</P><P>Template DNA	variable	variable	&lt; 1,000 ng
</P><P>Q5 High-Fidelity DNA Polymerase	0.25 µl	0.5 µl	0.02 U/µl
</P><P>5X Q5 High GC Enhancer (optional)	(5 µl)	(10 µl)	(1X)
</P><P>Nuclease-Free Water	to 25 µl	to 50 µl	
</P><P>
2.	Notes: Gently mix the reaction. Collect all liquid to the bottom of the tube by a quick spin if necessary. Overlay the sample with mineral oil if using a PCR machine without a heated lid. 
</P><P>
Transfer PCR tubes to a PCR machine and begin thermocycling. 
</P><P>Thermocycling Conditions for a Routine PCR:
</P><P>
STEP	TEMP	TIME
</P><P>Initial Denaturation	98°C	30 seconds
</P><P>25–35 Cycles	98°C	5–10 seconds
	*50–72°C	10–30 seconds
	72°C	20–30 seconds/kb
</P><P>
Final Extension	72°C	2 minutes
</P><P>Hold	4–10°C	
</P><P>
3.	*Use of the NEB Tm Calculator is highly recommended.
</P><P>
4.	General Guidelines:
</P><P>Template:
Use of high quality, purified DNA templates greatly enhances the success of PCR. Recommended amounts of DNA template for a 50 µl reaction are as follows:
</P><P>DNA	AMOUNT
</P><P>DNA Genomic	1 ng–1 µg
Plasmid or Viral	1 pg–1 ng
</P><P>
5.	Primers:
</P><P>Oligonucleotide primers are generally 20–40 nucleotides in length and ideally have a GC content of 40–60%. Computer programs such as Primer3 can be used to design or analyze primers. The best results are typically seen when using each primer at a final concentration of 0.5 µM in the reaction. 
</P><P>6.	Mg++ and additives:
Mg++ concentration of 2.0 mM is optimal for most PCR products generated with Q5 High-Fidelity DNA Polymerase. When used at a final concentration of 1X, the Q5 Reaction Buffer provides the optimal Mg++concentration. 
</P><P>Amplification of some difficult targets, like GC-rich sequences, may be improved by the addition of 1X Q5 High GC Enhancer. The Q5 High GC Enhancer is not a buffer and should not be used alone. It should be added only to reactions with the Q5 Reaction Buffer when other conditions have failed. 
</P><P>7.	Deoxynucleotides: 
The final concentration of dNTPs is typically 200 μM of each deoxynucleotide. Q5 High-Fidelity DNA Polymerase cannot incorporate dUTP and is not recommended for use with uracil-containing primers or templates. 
</P><P>8.	Q5 High-Fidelity DNA Polymerase concentration:
We generally recommend using Q5 High-Fidelity DNA Polymerase at a final concentration of 20 units/ml (1.0 unit/50 μl reaction). However, the optimal concentration of Q5 High-Fidelity DNA Polymerase may vary from 10–40 units/ml (0.5–2 units/50 μl reaction) depending on amplicon length and difficulty. Do not exceed 2 units/50 μl reaction, especially for amplicons longer than 5 kb.
</P><P>9.	Buffers:
The 5X Q5 Reaction Buffer provided with the enzyme is recommended as the first-choice buffer for robust, high-fidelity amplification. For difficult amplicons, such as GC-rich templates or those with secondary structure, the addition of the Q5 High GC Enhancer can improve reaction performance. The 5X Q5 Reaction Buffer is detergent-free and contains 2.0 mM MgCl2 at the final (1X) concentration. 
</P><P>10.	Denaturation:
An initial denaturation of 30 seconds at 98°C is sufficient for most amplicons from pure DNA templates. Longer denaturation times can be used (up to 3 minutes) for templates that require it.
</P><P>During thermocycling, the denaturation step should be kept to a minimum. Typically, a 5–10 second denaturation at 98°C is recommended for most templates. 
</P><P>11.	Annealing:
Optimal annealing temperatures for Q5 High-Fidelity DNA Polymerase tend to be higher than for other PCR polymerases. The NEB Tm Calculator should be used to determine the annealing temperature when using this enzyme. Typically, use a 10–30 second annealing step at 3°C above the Tm of the lower Tm primer. A temperature gradient can also be used to optimize the annealing temperature for each primer pair.
</P><P>For high Tm primer pairs, two-step cycling without a separate annealing step can be used (see note 12). 
12.	Extension:
The recommended extension temperature is 72°C. Extension times are generally 20–30 seconds per kb for complex, genomic samples, but can be reduced to 10 seconds per kb for simple templates (plasmid,E. coli, etc.) or complex templates &lt; 1 kb. Extension time can be increased to 40 seconds per kb for cDNA or long, complex templates, if necessary.
</P><P>A final extension of 2 minutes at 72°C is recommended.
</P><P>13.	Cycle number:
Generally, 25–35 cycles yield sufficient product.  For genomic amplicons, 30-35 cycles are recommended.
</P><P>14.	2-step PCR:
When primers with annealing temperatures ≥ 72°C are used, a 2-step thermocycling protocol (combining annealing and extension into one step) is possible. 
</P><P>15.	Amplification of long products:
When amplifying products &gt; 6 kb, it is often helpful to increase the extension time to 40–50 seconds/kb. 
</P><P>16.	PCR product:
The PCR products generated using Q5 High-Fidelity DNA Polymerase have blunt ends. If cloning is the next step, then blunt-end cloning is recommended. If T/A-cloning is preferred, the DNA should be purified prior to A-addition, as Q5 High-Fidelity DNA Polymerase will degrade any overhangs generated.
</P><P>
Results:
</P><P><A href="/wiki/images/f/fa/2015-07-17_PCR_PCquad_amplification_62C%2C_66C%2C_70C_.JPG" class="internal" title="2015-07-17 PCR PCquad amplification 62C, 66C, 70C .JPG">Media:2015-07-17 PCR PCquad amplification 62C, 66C, 70C .JPG</A></P><P>Lanes 1, 2, and 3 are our construct + primers at 62C, 66C, and 70C, respectively.
</P><P>It appears as though the desired band at ~1.4kb is still rather weak.
</P><P>
Conclusions:  Further attempts at optimizing the desired band should be done before moving on from this step.  On monday, if reagents come in, protein expression will be continued, so Tyler and Nithin will take over the cloning.
</P></DIV></DIV></DIV></DIV></BODY></HTML>