<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCLA_Notebook_Protein_Cages_9_July_2015 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCLA/Notebook/Protein Cages/9 July 2015</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM UCLA</TITLE></P><DIV id="top_menu"><UL id="menu"><LI><A href="/Team:UCLA">Home</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Team</A></LI><LI><A href="/Team:UCLA/Team/Meet_the_Team">Meet the Team</A></LI><LI><A href="/Team:UCLA/Attributions">Attributions</A></LI><LI><A href="/Team:UCLA/Collaborations">Collaborations</A></LI></UL><LI><A href="/Team:UCLA/Project">Projects</A></LI><LI><A href="/Team:UCLA/Project/Programming Spider Silk">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Project/Functionalizing Silk">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Project/Honeybee Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Project/Protein Expression and Processing">Materials Processing</A></LI><LI><A href="/Team:UCLA/Project/Protein Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Practices">Human Practices</A></LI><LI><A href="/Team:UCLA/Project/Interlab Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Parts">Parts</A></LI><LI><A href="/Team:UCLA/Parts">Team Parts</A></LI><LI><A href="/Team:UCLA/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:UCLA/Composite_Parts">Composite Parts</A></LI><LI><A href="/Team:UCLA/Part_Collection">Part Collection</A></LI><LI><A href="/Team:UCLA/Notebook/Weekly Notes">Notebook</A></LI><LI><A href="/Team:UCLA/Notebook/Spider_Silk_Genetics">Programming Spider Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Recombinant_Expression">Functionalizing Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Honeybee_Silk">Honeybee Silk</A></LI><LI><A href="/Team:UCLA/Notebook/Materials">Materials Processing</A></LI><LI><A href="/Team:UCLA/Notebook/Protein_Cages">Protein Cages</A></LI><LI><A href="/Team:UCLA/Notebook/Interlab_Study">Interlab Study</A></LI><LI><A href="/Team:UCLA/Safety">Safety</A></LI><LI><A href="https://igem.org/Judging_Form.cgi?id=1763">Judging</A></LI></DIV><P>
Phillip's notes:
</P><P>
Introduction:  Today ampicillin plates will be made in order to have them available for protein expression.  In addition, the PCquad optimized will be designed in order to be expressed using PSB1C3 in E. coli DE3.
</P><P>A schematic of the construct is the following:
</P><P>Prefix-T7 promoter-lac operator-RBS-NdeI-gene-XhoI-histag-stop-suffix
</P><P>The file made on benchling was named “PCquad Optimized for iGEM 3.0.”
</P><P>The T7 promoter was taken from the pET22b vector file.  A C-terminal histag and stop were created.
From the iGEM website, the master RBS sequence was found to be the following.
</P><P>Master RBS:  TCTAGAG-AAAGANNNGANNN-ACTAG-ATG(start)
The sequence from <A rel="nofollow" class="external free" href="http://parts.igem.org/cgi/partsdb/puttext.cgi">http://parts.igem.org/cgi/partsdb/puttext.cgi</A> was used.
</P><P>Notes on making agar plates:
</P><P>
LB media
Component	Amount per liter
</P><PRE>Tryptone	 10 g
Yeast extract	  5 g
NaCl	 10 g
</PRE><P>For preparation of 1 liter of LB medium, add 10 g NaCl, 10 g tryptone, and 5 g yeast extract to 950 ml distilled or deionized water, and shake or stir until dissolved. Adjust the pH to 7.0 with 5 M NaOH. Adjust the volume of the solution to 1 liter with distilled or deionized water. Decant into smaller aliquots and sterilize by autoclaving (see Sterilizing media).
</P><P>Tip: It is advisable to autoclave liquid medium in several small bottles rather than in one large vessel to avoid possible contamination of an entire batch. After autoclaving, do not use medium for 24 hours to ensure that it is properly sterilized and free of contaminating microorganisms.
</P><P>Tip: Antibiotics should be added to liquid medium immediately prior to use from stock antibiotic solutions that have been filter-sterilized, distributed into aliquots, and stored in the dark at –20°C (see Antibiotics).
</P><P>
Sterilizing media
</P><P>Sterilize liquid or solid media by autoclaving, using a pressure and time period suitable for the type of medium, bottle size, and autoclave type.
</P><P>Tip: Fill bottles only 3/4 full with medium and loosen the caps before autoclaving to avoid hot medium boiling over. Tighten caps once the media is cool (&lt;40°C) to keep it completely sterile.
</P><P>Tip: Antibiotics and nutrients such as amino acids are inactivated by the high temperatures of an autoclave. They should be sterilized by filtration through a filter unit with a pore size of 0.2 µm, and added to the cooled, autoclaved medium from properly stored stock solutions.
</P><P>
Solid media
</P><P>E. coli strains can generally be streaked and stored on LB plates containing 1.5% agar and the appropriate antibiotic(s).
Preparation: Prepare LB medium according to the composition given in Liquid media. Just before autoclaving, add 15 g agar per liter and mix. After autoclaving, swirl the medium gently to distribute the melted agar evenly throughout the solution. Take care that the hot liquid does not boil over when swirled.
</P><P>Tip: Cool autoclaved agar medium to below 50°C (when you can hold it comfortably) before adding heat-sensitive antibiotics and nutrients. Mix thoroughly to obtain an even concentration throughout the medium before pouring.
</P><P>Tip: Pour plates in a laminar-flow hood or, if no hood is available, on a cleaned bench surface next to a Bunsen. Use 30–35 ml medium per standard 90 mm petri dish (~30 plates per liter of medium).
</P><P>After pouring plates, any air bubbles may be removed by passing the flame of a Bunsen burner briefly over the surface. Do not linger with the flame as this may destroy antibiotics in sections of the plates.
</P><P>
Dry plates either directly after solidification or just before use by removing the lids and standing the plates in a laminar-flow hood for 1 hour. Alternatively, if you do not have access to a hood, plates can be dried with the covers slightly open in a 37°C incubator for 30 min, or left upside down with lids on at room temperature for 2–3 days.
</P><P>Tip: Store plates inverted at 4°C in a dark room or wrapped in aluminum foil to preserve light-sensitive antibiotics. Do not store for longer than 3 months as antibiotics may degrade.
</P><P>
Antibiotics
</P><P>Bacterial strains carrying plasmids or genes with antibiotic selection markers should always be cultured in liquid or on solid medium containing the selective agent. Lack of antibiotic selection can lead to loss of the plasmid carrying the genetic marker and potentially to selection of faster-growing mutants!
</P><P>Tip: Prepare stock solutions of antibiotics separately from batches of liquid or solid media, sterilize by filtration, aliquot, and store in the dark at –20°C. Recommended stock and working concentrations for commonly used antibiotics are shown in the table Concentrations of commonly used antibiotics.
</P><P>Tip: Before adding antibiotics to freshly autoclaved medium, ensure that the medium has cooled to below 50°C.
Ampicillin (sodium salt)	 50 mg/ml in water	 –20°C	 100 µg/ml (1/500)
</P><PRE>Chloramphenicol	 30 mg/ml in ethanol	 –20°C	 170 µg/ml (1/200)
</PRE><P>Upon going to Kosuri lab, it was found that ampicillin plates were already available.  Fasih mentioned that they may be a few months old.
</P><P>Conclusions:
If access is available, the Yeates strain will be streaked on Sunday evening to allow for protein expression throughout next week.  A meeting with Dr. Kosuri should be made in order to outline the experimental approach.
</P><P>Intro: Today Nithin, Phil, and I got certified to work in Sri's lab. Amp plates and other materials are in the iGEM cabinet in the cold room. I designed primers to express the cage within the iGEM vector to minimize transformations. Researched protein expression stuff. 
</P><P>iGEM Expression Primers:
</P><P>prefix
</P><P>handle-iGEM prefix-T7 promoter-RBS-spacer-gene
</P><P>suffix
</P><P>gene-his tag-double stop codons-iGEM suffix-handle
</P><P>The primers had -25 kJ/mol homodimer association and other pretty bad secondary structures. However, the sites of interest were the restriction sites from the iGEM suffixes etc, so there isn't any redesign we can do. Oh well...
</P><P>Protein Expression Literature:
</P><P><A rel="nofollow" class="external free" href="http://onlinelibrary.wiley.com/doi/10.1002/0471140864.ps0524s61/full">http://onlinelibrary.wiley.com/doi/10.1002/0471140864.ps0524s61/full</A></P><P>Generic overview, nice schematic and some useful info but little detail.
</P><P><A rel="nofollow" class="external free" href="http://www.nature.com/nprot/journal/v2/n2/full/nprot.2007.50.html">http://www.nature.com/nprot/journal/v2/n2/full/nprot.2007.50.html</A></P><P>more detailed protocol
</P><P>Tyler Lee --<A href="/wiki/index.php?title=User:Wtleeiv&amp;action=edit&amp;redlink=1" class="new" title="User:Wtleeiv (page does not exist)">Wtleeiv</A> 20:02, 9 July 2015 (CDT)
</P></DIV></DIV></DIV></DIV></BODY></HTML>