<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCL_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCL/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>'

</P><NAV class="mainmenu"><DIV id="menu" class="menu"><DIV class="menu-items"><LI><A href="https://2015.igem.org/Team:UCL/Description">Project</A></LI><LI><A href="#">Lab work</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Effectors">●  Psychobiotics</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Sensors">●  Mood Detection</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Safety">●  Built-in Biosafety</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Design">● Gut-on-a-Chip</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Fermentation">● Scaling Up</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="#">Process</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Interlabstudy">●  InterLab Study</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Collaborations">●  Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Notebook">●  Notebook</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Protocols">●  Protocols</A></LI><LI><A href="#">Outcomes</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Software">● Software</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Bactoman">● Bactoman</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Parts">● Parts</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Achievements">●  Achievements</A></LI><LI><A href="#">People</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Team">● Team</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Sponsors">● Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Attributions">● Attributions</A></LI></DIV></DIV></NAV><DIV id="hidden-menu"><UL id="mobile-list"><LI><A href="https://2015.igem.org/Team:UCL">Home</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Effectors">Psychobiotics</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Sensors">Mood Detection</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Design">Gut-on-a-Chip</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Fermentation">Scaling Up</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Bactoman">Bactoman</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:UCL/Interlabstudy">InterLab Study</A></LI></UL></DIV></DIV></DIV><DIV id="header2"><DIV id="wrapper"><DIV id="menu2"><P class="title">Notebook</P><UL class="cos"><LI id="item1">Week 1</LI><LI id="item2">Week 2</LI><LI id="item3">Week 3</LI><LI id="item4">Week 4</LI><LI id="item5">Week 5</LI><LI id="item6">Week 6</LI><LI id="item7">Week 7</LI><LI id="item8">Week 8</LI><LI id="item9">Week 9</LI><LI id="item10">Week 10</LI><LI id="item11">Week 11</LI><LI id="item12">Week 12</LI></UL></DIV><DIV id="menu3"><P class="title">Notebook</P><UL class="cos"><LI id="item1"><A href="#week1">Week 1</A></LI><LI id="item2"><A href="#week2">Week 2</A></LI><LI id="item3"><A href="#week3">Week 3</A></LI><LI id="item4"><A href="#week4">Week 4</A></LI><LI id="item5"><A href="#week5">Week 5</A></LI><LI id="item6"><A href="#week6">Week 6</A></LI><LI id="item7"><A href="#week7">Week 7</A></LI><LI id="item8"><A href="#week8">Week 8</A></LI><LI id="item9"><A href="#week9">Week 9</A></LI><LI id="item10"><A href="#week10">Week 10</A></LI><LI id="item9"><A href="#week9">Week 9</A></LI><LI id="item10"><A href="#week10">Week 10</A></LI><LI id="item11"><A href="#week11">Week 11</A></LI><LI id="item12"><A href="#week12">Week 12</A></LI></UL></DIV><DIV id="notebookcontent"><DIV class="about-item1 hide"><DIV class="notebookweek" id="week1"><H2>Week 1 (15<SUP>th</SUP> June – 21<SUP>st</SUP> June)</H2><DIV class="day"><H3>Bootcamp</H3><P>This is how it begins. The iGEM bootcamp at UCL is intended to prepare us for the roles we are going to play this summer. We get to talk to some of the founding members of iGEM and meet two other iGEM teams from London, from the Biohackspace in Hackney and Birkbeck College.</P></DIV><DIV class="day"><H3>Monday 15<SUP>th</SUP></H3><P>We met at 09:30 in the lab. It is the first time we could all meet each other after our exams. The team members from the Biohackspace and Birkbeck were there as well. After the briefing on lab safety and other things we started with a <STRONG><I>Spe</I>I</STRONG> and <STRONG><I>Pst</I>I</STRONG> digestion using the backbones from the InterlabStudy 2015:
<B><A id="description" href="http://parts.igem.org/Part:BBa_J23101" target="_blank">J23101</A>, 
<A href="http://parts.igem.org/Part:BBa_J23106" target="_blank">J23106</A>, 
<A href="http://parts.igem.org/Part:BBa_J23117" target="_blank">J23117</A></B>. As an insert we used an RFP containing vector from the iGEM 2014 distribution. We confirmed the digestions through an agarose gel electrophoresis and went for a break in the sun. Afterwards we performed the ligation of the insert and the backbone.</P><P>In the afternoon we had a skype meeting with Randy Rettberg from the iGEM foundation and learned something about the spirit and history of iGEM.</P></DIV><DIV class="day"><H3>Tuesday 16<SUP>th</SUP></H3><P>We split up into two groups to transform bacteria with the ligation reaction from yesterday. One group was using electroporation and the other chemical transformation. We incubated the bacteria overnight.</P><P>For the afternoon we split up into different groups preparing us for the individual roles that we will take in the team.</P><H4>DIYbio</H4><P>We discussed about the principles of DIY biology and the barriers preventing people from participating in biology. We were inroduced to the concept of the microcontroller Arduino.</P><H4>Software and Automation</H4><P>We talked to <A href="http://www.synthace.com/">Synthace</A> founding member Rob Stanley. Automating lab work cannot start early enough! We later talked to computational biologists about software methods for modelling biological systems.</P><H4>Extralab</H4></DIV><DIV class="day"><H3>Wednesday 17<SUP>th</SUP></H3><P>This morning we discovered that the RFP insert did not really fluoresce. At least not in red. We picked colonies to grow overnight in order to check whether something went wrong in the ligation or if the biobrick was not working. This could be done on Thursday through an agarose gel of the recombinant DNA.</P><P>In the afternoon we went back into our groups from yesterday.</P><H4>DIYbio</H4><P>We headed for the Biohackspace in Hackney and started doing manual. We 3D printed the parts and soldered everything together to make a spectrophotometer. Unfortunately, it didn't quite work. But being in the Hackspace was definitely worthwhile.</P><H4>Software &amp; Automation</H4><P>We explored how the iGem wiki works which was more than this one sentence indicates.</P><H4>Extra Lab</H4></DIV><DIV class="day"><H3>Thursday 18<SUP>th</SUP></H3><P>We made a mini prep of our bacterial cultures and digested the final plasmid again, this time with <STRONG><I>Spe</I>I</STRONG> and <STRONG><I>Xba</I>I</STRONG> to check the insert. We then did the anticipated gel-electrophoresis. The result was that the ligation worked as we got the expected bands on the gel. Our conclusion to why the fluorescence did not work is that something was wrong with the biobrick...</P><H4>DIY bio</H4><P>We went to the Hackspace again and finished the sensor for the photometer and measured
                                </P><H4>Interlab study meeting</H4><P>We had a skype meeting with the head of InterLab Study, Jacob Beal, and had a chat about this year's interlab study.</P></DIV><DIV class="day"><H3>Friday 19<SUP>th</SUP></H3><H4>Mini Jamboree!</H4><P>We have organised a mini jamboree in collaboration with Birkbeck iGEM and Biohackspace iGEM teams. We have invited people from the London synthetic biology community as well as the UCL Academy iGEM team to attend the jamboree. Each team presented what they did during the bootcamp.</P></DIV></DIV></DIV><DIV class="about-item2 hide"><DIV class="notebookweek" id="week2"><H2>Week 2 (22<SUP>nd</SUP> June – 28<SUP>th</SUP> June)</H2><DIV class="day"><H3>Monday 22<SUP>nd</SUP></H3></DIV><DIV class="day"><H3>Tuesday 23<SUP>rd</SUP></H3></DIV><DIV class="day"><H3>Wednesday 24<SUP>th</SUP></H3><H4> Hackathon part 1 </H4><P>The beginning of our actual project. Before today we knew that our project would be about the mind gut axis which means trying to tackle mental health problems with engineered probiotics. Today we made an actual list of effectors and promoters that we want to use in our biobricks. It was brainstorming mayhem, but we cut down our list to about ten candidates from each category for which we are going to collect more information for tomorrow.</P></DIV><DIV class="day"><H3>Thursday 25<SUP>th</SUP></H3><H4>Hackathon part 2 </H4><P>We finished our list today and had a lengthy vote on the g Blocks to order for our biobricks. Our main criterium for now is the likeliness for the gene products to yield useful data. Here is our list for the genes and promoters that we are starting to work on next week:</P><UL><LI>Effectors</LI><UL><LI>TPH1 gene with an adrenaline-sensitive promoter,
<A href="http://parts.igem.org/Part:BBa_K554001" target="_blank">BBa_K554001</A>. TPH1 converts tryptophan to a precursor of serotonin.</LI><LI>TPH1 gene with nitric oxide sensitive promoter, <A href="http://parts.igem.org/Part:BBa_K381001" target="_blank"> BBa_K381001 </A></LI><LI>naked sequence for GAD, which codes for the enzyme glutamate decarboxylase which produces GABA.</LI><LI>naked sequence for KAT (kynurenine aminotransferase), an important enzyme in serotonin metabolism</LI><LI>naked sequence for Cholin Acetyltransferase, which makes Acetylcholin, an important neurotransmiter</LI></UL><LI>Promoters</LI><UL><LI>biobrick of the adrenaline sensitive promoter <A href="http://parts.igem.org/Part:BBa_K554001" target="_blank">BBa_K554001</A></LI><LI>biobrick of an osmotic stress sensitive promoter <A href="http://parts.igem.org/Part:BBa_R0082" target="_blank">BBa_R0082</A></LI><LI>biobrick of the nitric oxide sensitive promoter <A href="http://parts.igem.org/Part:BBa_K381001" target="_blank"> BBa_K381001 </A></LI><LI>biobrick of a pH sensitive promoter <A href="http://parts.igem.org/Part:BBa_K318512" target="_blank"> BBa_K318512</A></LI></UL><LI>Constructs</LI><UL><LI>An estrogen induced construct that is not yet fully characterized. We are going to transfect HeLa cells to model a pathway in mammalian cells which is central to our project as it involves bacteria interfering with humans</LI><LI>An anti-sense Tryptophanase construct. We want to use this to control the tryptophanase expression which would allow us (or a cell) to regulate serotonin expression.</LI></UL></UL></DIV><DIV class="day"><H3>Friday 26<SUP>th</SUP></H3></DIV></DIV></DIV><DIV class="about-item3 hide"><DIV class="notebookweek" id="week3"><H2>Week 3 (29<SUP>th</SUP> June – 5<SUP>th</SUP> July)</H2><DIV class="day"><H3>Monday 29<SUP>th</SUP></H3><P>This week we finally managed to start the proper lab work! In the morning prepared agar plates with ampicillin/chloramphenicol. Then we carried out <A href="https://2015.igem.org/Team:UCL/Protocols#transformation">transformations</A> of four promoters from the 2014 distribution that we plan to use for characterization of our parts:</P><UL><LI>BBa_K554001 FlhDC promoter)</LI><LI>BBa_R0082 (OmpR promoter)</LI><LI>BBa_K144300 (blue light activated system) we got from our friends from Aalto-Helsinki team</LI><LI>BBa_K864400 (PTac promoter)</LI>+ DH5 alpha competent cells control</UL><P>In the meantime some of us were cleaning and organizing our new lab to make it ours and pretty! We will post a picture once it's ready :) We also tried to autoclave our tipette pits but they didn't quite like the autoclave!</P></DIV><DIV class="day"><H3>Tuesday 30<SUP>th</SUP></H3><P>In the morning, we have checked the plates and obtained some pretty colonies!
                                 We have inoculated 4 colonies per plate to be incubated overnight at 37C </P><P>In the afternoon, we planned the assembly of our BBa_J23100-BBa_B0034-TPH1-BBa_B0015. We have digested the gBlock1 with EcoRI and gBlock2 with PstI. Then we have performed the <A href="https://2015.igem.org/Team:UCL/Protocols">Gibson assembly </A> of digested gBlock 1, gBlock 2, and linearized pSB1C plasmid, followed by the <A href="https://2015.igem.org/Team:UCL/Protocols">transformation </A>. </P></DIV><DIV class="day"><H3>Wednesday 1<SUP>st</SUP></H3><H4>Bifidobacter</H4><P>There was only one colony on our plates from the Gibson assembly! We suspect that this is because the overlap between the linearized pSB1C3 backbone and our synthetic constructs was not long enough. We plan to try the Gibson assembly of two gBlocks alone tomorrow, followed by ligation to the plasmid backbone. We will see if this works better!</P><P>We have also purified the DNA from the transformants of four promoters from Monday. Some parts from the kit were missing, so we spent 4 hours on making a miniprep! This is bad!</P><H4>Interlab Study</H4><P>In the afternoon, we have <A href="https://2015.igem.org/Team:UCL/Protocols#transformation">transformed</A> the BBa_I13504 part from the 2014 distribution which we plan to use for our InterLab study.</P><H4>Entrepreneurship</H4><P>We are becoming entrepreneurs! Our vision is to commercialize our probiotics in an easy and accessible product to make sure anyone can benefit from our awesome engineered bacteria.</P><P>Our product of choice is chocolate bars! They will deliver the needed probiotics in your gut whilst enjoying a delicious and high quality product.</P></DIV><DIV class="day"><H3>Thursday 2<SUP>nd</SUP></H3><H4>Bifidobacter</H4><P>This morning we have repeated the <A href="https://2015.igem.org/Team:UCL/Experiments#gibson">Gibson assembly</A> of TPH1 gBlocks. This time we have decided to just assemble the two gBlocks using the assembly kit. Then we digested them with EcoR1 and Pst1, performed PCR clean-up using Wizard SV gel and PCR clean-up system, and ligated the assembled construct into the linearized pSB1C3. We finished the day by doing the <A href="https://2015.igem.org/Team:UCL/Experiments#transformation">transformation</A> of DH5alpha cells.</P><H4>Interlab Study</H4><P>One colony was picked from the transformation of the 2014 BBa_I13504 part and we made an overnight culture.</P><H4>Wiki Design</H4><P>We had a first talk about the design of our wiki and came to the conclusion to play around and figure out the perfect design through trial, error and further discussions. But fortunately, the wiki is starting to take shape.</P><H4>Entrepreneurship</H4><P>We are finalizing the details of our product and business model, today we had a very intense brainstorming session, but we can proudly say that we have set the base for our new brand.</P></DIV><DIV class="day"><H3>Friday 3<SUP>rd</SUP></H3><H4>Bifidobacter</H4><P>Bad news: Our colonies did not grow again
                                Good news: We finally run the diagnostic gel of our promoters and they all worked perfectly
                                </P><H4>Modelling</H4><P>We made our first model for a pathway with COPASI</P><H4>Lab</H4><H4>Entrepreneurship</H4><P>Our business model is finished, now it’s time to design a pitch and start attracting investors.</P><P>Miniprep</P><P>Diagnostic digest</P></DIV></DIV></DIV><DIV class="about-item4 hide"><DIV class="notebookweek" id="week4"><H2>Week 4 (6<SUP>th</SUP> July – 12<SUP>th</SUP> July)</H2><DIV class="day"><H3>Monday 6<SUP>th</SUP></H3><H4>Bifidobacter</H4><P>As the Gibson Assemly refused to work for the second time, we had adapted a new strategy. We want to subclone our TPH1 biobrick that we already have in PSB1C3 into the <A href="http://parts.igem.org/Part:BBa_K314103">BBa_K314103</A>, an IPTG-inducible expression casette. Luckily, we already have a prep of that part from our last year's team!
                                Today, we tried to run the PSB1C3-TPH1 on a gel, extract it, and ligate it to the backbone containing the expression casette. Unfortunately, it turned out that the issues we had with the freezer last week killed our restriction enzymes and we did not obtain an expected band on the gel to extract!</P></DIV><DIV class="day"><H3>Tuesday 7<SUP>th</SUP></H3><H4>Bifidobacter</H4><P>We are not giving up. Today we have executed the Monday's plan and transformed our cells with what we hope is a TPH1 ligated into IPTG-inducible cassete. Let's hope for some colonies tomorrow!</P></DIV><DIV class="day"><H3>Wednesday 8<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P>had a look at their TPH1 plates. Nothing has grown, again! We are deciding to change our strategy from gel extraction to the 2A assembly! We will subclone the TPH1 into the PSB1A3 backbone today and attempt transformation again. If we succeed with that, we will subclone it into the expression cassette on CAM-resistant backbone in the beginning of next week.</P><H4>Improvement of Biobrick</H4><P>After the RFP cultures did not grow on the Ampicilin medium we made made a new culture on chloramphenicol because we thought it might be a mistake in the registry. However, they did not grow on that plate either. A liquid control without antibiotics worked well, so at least we know that the cells (initially) are alive. We think that the the transformation did not work. Maybe there is a problem with the DNA too. We used the liquid cultures of the GFP we made on Tuesday to make minipreps and to use them with the promoters from the interlab-study.</P><H4>High School Collaboration</H4><P>The collaboration we have with UCL-academy is taking shape. We introduced them to the basics of wiki-design, modelling and let them help us with the ligation and subsequent transformation of the GFP constructs we made this morning.</P><H4>Wiki</H4><P>Little changes to the wiki. We managed to get a rough project description onto the wiki.</P></DIV><DIV class="day"><H3>Thursday 9<SUP>th</SUP></H3><H4>Bifidobacter</H4><P>Some pretty colonies!!! Let's inoculate the overnights!</P></DIV><DIV class="day"><H3>Friday 10<SUP>th</SUP></H3></DIV><DIV class="day"><H3>Weekend 12<SUP>th</SUP>-13<SUP>th</SUP></H3></DIV></DIV></DIV><DIV class="about-item5 hide"><DIV class="notebookweek" id="week5"><H2>Week 5 (13<SUP>th</SUP> July – 19<SUP>th</SUP> July)</H2><DIV class="day"><H3>Monday 13<SUP>th</SUP></H3><P>We started the week with the meeting with <A href="http://www.ismb.lon.ac.uk/vitor_pinheiro.html" target="_blank">Vitor Pinheiro</A>, who gave us some great tips on our cloning schedule and strategies as well as on directed evolution systems that we could use. During the meeting we also came up with an excited idea of using synthetic amino acids as precursors for neurotransmitter synthesis!</P><H4>Bifidobacter</H4><P>There wasn't much time for lab work left but we managed to do some preps and set the overnight digestions</P><H4>Interlab Study</H4><P>We inoculated cultures with the GFP gene in liquid cultures for the Interlab Study. The cultures were left to grow over night. We also prepared plates with kanamycin which we are going to use for 3A assembly of our promoters and effectors. We made a miniprep of another GFP culture from last week</P></DIV><DIV class="day"><H3>Tuesday 14<SUP>th</SUP></H3><H4>Interlab Study</H4><P>We used the cultures we inoculated yesterday for making a miniprep. Just to be safe we had made two cultures and after figuring out the quirks of the nanodrop it turned out that one had a DNA concentration more than twice that of the other. We digested the minipreps and the promoters from the Interlab Study with Xba1, Spe1 and Pst1 to ligate them together in the next step. We checked if the digestion of GFP (Interlab Study) worked but we could not see any bands so we'll try again tomorrow with an Ethidium Bromide gel and use more DNA. We started the ligation of promoters to GFP anyway, transformed and let these cultures grow overnight.</P><H4>Bifidobacter</H4><P>We have checked the concentrations of preps from yesterday and they were really low! The diagnostic digests confirmed that there was no DNA in our preps. In the mean time we decided to do the gel extraction od Ptac and GFP we got from our other team members. We digested the Ptac with S and P and GFP with X and P and run it on the gel. The Ptac was there, but GFP wasn't.
                                </P></DIV><DIV class="day"><H3>Wednesday 15<SUP>th</SUP></H3><H4>Interlab Study</H4><P>We ran the same digestion as yesterday but this time with lots of DNA and a ton of Ethidium Bromide and it worked!!!
                                 We could see the bands we expected in one of our samples (second well from the ladder). The other one seemed not to be digested. Now we are going to stick only to the good one. We transformed cells with the ligation from yesterday and let them grow overnight.</P><H4>Mind the Gut: Promoters</H4><P>We also started working on the promoters of our actual project Gad-A+RBS, Gad-A and PyeAr digested them and ligated them into a backbone with Cam resistance.</P><H4>Mind the Gut: Effectors</H4><P>Today we have decide to clone our TPH1 biobrick into the expression cassette using three different approaches: 3A assembly, gel extraction, and amplified insert assembly. We were not able to get hold of the kanamycin backbone we needed for subcloning the expression cassette, but we succeeded with two other approaches and managed to get to the stage of PCR/gel extraction clean ups.</P></DIV><DIV class="day"><H3>Thursday 16<SUP>th</SUP></H3><H4>Public Engagement</H4><P>We have contacted many people involved in research, mental health charities and artists who could be interested in our project. Today we got a lot of feedback. We want to use their input to start off our public engagement part of the project. It is starting to take shape...</P><H4>Interlab Study</H4><P>The cultures for the interlab study grew (yeah!) and we inoculated them to liquid cultures and let them grow over night.</P><H4>Mind the Gut: Promoters</H4><P>We transformed cells with our promoters (Gad-A+RBS, Gad-A and PyeAr) in chloramphenicol resistance.</P><H4>Mind the Gut: Effectors</H4><P>We have done ligation and transformation of both gel extracted- and PCR amplified- TPH1 with PSB1C3 carrying expression cassette. In order to maximize the chances of success, we have carried each step according to long protocols, including dephosphorylation of backbone and heat inactivation of both restriction enzymes and T4 ligase. Because of that, we have only managed to finish with the transformation by the end of the day!</P></DIV><DIV class="day"><H3>Friday 17<SUP>th</SUP></H3><H4>Mind the Gut: Promoters</H4><P>The cultures we transformed with our own promoters grew and we now need to check whether some have actually taken up the promoters or whether they just self-ligated. So we miniprepped them and wished them a nice weekend!</P><H4>Mind the Gut: Effectors</H4><P>We have repeated the mini preps and diagnostic digestions of TPH1 in PSB1A3, but no success again! :( What happened to our inserts?</P><H4>Entrepreneurship</H4><P>We had an end of week talk and shared some ideas about the entrepreneurship part of our project finding out more about regulations to consider in the retail of probiotics some of which seem a bit strange to us, so far...</P></DIV></DIV></DIV><DIV class="about-item6 hide"><DIV class="notebookweek" id="week6"><H2>Week 6 (20<SUP>th</SUP> July – 26<SUP>th</SUP> July)</H2><DIV class="day"><H3>Monday 20<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P>We have picked colonies of our TPH1 in the expression casette and transformed some new parts: BBa_J04450 on PSB1K3 from the distribution that we will need for amplification of kanamycin-resistant backbone, and our antiTNA silencing RNAs that arrived from IDT!</P><H4>Human practices</H4><P>This afternoon we went to the Dragon cafe, a creative space hosted by the <A href="http://mentalfightclub.com/" target="_blank">Mental fight club</A>, and met with some amazing people to discuss how we could present our project to the public, particularly to those who have experienced psychological problems or are interested about them.</P></DIV><DIV class="day"><H3>Tuesday 21<SUP>st</SUP></H3><H4>Mind the Gut: Effectors</H4><P>We got some pretty red colonies of the BBa_J04450, but no colonies for antiTNA :( We will retry with overnight ligation at 16C to increase ligation efficiency. We have also overrun the gel confirming out TPH1exp biobrick...what an unlucky day</P></DIV><DIV class="day"><H4>Mind the Gut: Effectors</H4><H3>Wednesday 22<SUP>nd</SUP></H3><P>Finally!!!! We have our second biobrick ❤❤❤ : the composite part consisting of BBa_K413103, BBa_B0034, TPH1 and BBa_B0015:
                                We have also got a confirmation for successful transformation of BBa_J04450 from the registry:
                                </P></DIV><DIV class="day"><H3>Thursday 23<SUP>rd</SUP></H3><P>Unfortunately, last time our ligation of GFP and the promoters from the Interlabstudy did not seem to have worked. So we started again with a ligation and we ran a gel to see if the digestions worked. In the first lane you can see the backbone and the GFP insert. The second digestion should have shown the same result but did not (for some reason) and the other three wells show us the backbone we opened for the ligation. Let's hope that we get the ligation working this time.</P></DIV><DIV class="day"><H3>Friday 24<SUP>th</SUP></H3></DIV></DIV></DIV><DIV class="about-item7 hide"><DIV class="notebookweek" id="week7"><H2>Week 7 (27<SUP>th</SUP> July– 2<SUP>nd</SUP> August)</H2><DIV class="day"><H3>Monday 27<SUP>th</SUP></H3></DIV><DIV class="day"><H3>Tuesday 28<SUP>th</SUP> July</H3><H4>Mind the Gut: Effectors</H4><P>We have been doing so much cloning that we have run out of backbones! Today our primers for backbone amplification arrived and we have successfully PCR-amplified all three backbones so that we can do lots and lots of cloning!</P><H4>Interlabstudy</H4><P>We used some of our successful cultures to make minipreps and to digest to see if we got what we expect (GFP in backbone). We picked several colonies for each promoter and used GFP from cultures from the 13th and from the 22nd of July.</P></DIV><DIV class="day"><H3>Wednesday 29<SUP>th</SUP> July</H3><H4>Interlabstudy</H4><P>We ran the gel with our 19(!) digestions from yesterday and got very pretty results. So, our ligation using 2A assembly (finally) worked. Unfortunately, we did not make any glycerol stocks of our cultures which means we had to transform our our minipreps into new competent cells again and hope for the best, losing one day for the colonies to grow. But we have the feeling that we are getting there...</P><H4>Mind the Gut promoters</H4><P>Yesterday we got new backbones with Kanamycin and Chloramphenicol resistance. So now we can finally clone the promoters Gad, Gad+RBS and PyeaR into these backbones and start assembling them with effectors. Today we started this process by digesting the backbones, a small but important step which we forgot to do last time,and the promoters with <I>EcoR</I>I and <I>Spe</I>I.</P><H4>Mind the Gut: Effectors</H4><P>We have cloned and transformed 6 IDT gBlocks today: antiTNA1, antiTNA2, KAT, GAD, FldHC-TPH1 and Pyear-TPH1</P></DIV><DIV class="day"><H3>Thursday 30<SUP>th</SUP> July</H3><H4>Interlabstudy</H4><P>The colonies of the promoters grew well, we got colonies on all three plates and they all showed green fluorescence! Hurray! So we picked three colonies from each plate to make glycerol stocks which we are going to use for the final step which is making liquid cultures that we are going to use for measurements. Hopefully, we can finish the Interlabstudy next week.</P><H4>Mind the Gut: Effectors</H4><P>Our transformations did not work. We PCRed even more PSB1C3 so that we can repeat everything all over again.</P><H4>Human Practices</H4></DIV><DIV class="day"><H3>Friday 31<SUP>sth</SUP> July</H3></DIV></DIV></DIV><DIV class="about-item8 hide"><DIV class="notebookweek" id="week8"><H2>Week 8 (3<SUP>rd</SUP> August – 9th<SUP>th</SUP> August)</H2><DIV class="day"><H3>Monday 3<SUP>rd</SUP></H3><H4>Mind the Gut: Effectors</H4><P>We have run PCR of some of the IDT genes to make sure we don't run out of them. It worked :)</P></DIV><DIV class="day"><H3>Tuesday 4<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P>We have carefully repeated digestion, ligation, and transformation of all the 6 genes today. Hopefully we will get some colonies tomorrow!</P></DIV><DIV class="day"><H3>Wednesday 5<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P>Much to our disappointment, we had no colony growth for the effector transformed. We decided to incubate the plates longer.</P><P>Meanwhile we also made mini-preps of GAD and FldHC-TPH1 with thermoscientific kit. We also ran a diagnostic digestion gel but only one band was visible, which we believe was only our plasmid of approximately 2kb. As we had only use 2ul of DNA and nano drop had shown considerably lower concentration of DNA, we decided the repeat the diagnostic digestion the next day.</P></DIV><DIV class="day"><H3>Thursday 6<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P>We re-ran the 1% agarose gel for GAD and FldHC-TPH1with 6ul of DNA but to our dismayal, we did not see two bands on the gel, rendering our experiment inconclusive.</P><P>In the afternoon, we did PCR cleanup of GAD,KAT,FidHC-TPH1, Pyear-TPH1, pSB1C3(3)and pSB1C3(4) USING PROMEGA PCR cleanup kit. We had yields ranging from 30-50 ng/ul.</P><P>For transformation of these effectors into our plasmid pSB1C3, initially we carried out digestion of 5ul of GAD and Pyear-TPH1 and 6.7ul of KAT,FldHC-TPH1 with XbaI and Pst. We also double digested our plasmid with the same enzymes. The digested samples were all incubated for half an hour and heat shocked at 80 degrees for 20 minutes. Ligation carried out at 1:5 molar ratio.</P><P>The antisenseTNA1,2 from last Tuesday finally had some colonies growing in them. The CHAT and TPH1 were both assembled from g blocks using Gibson assembly and been plated at different concentrations overnight. Only two colonies in 100 ul and numerous in 200 ul of CHAT was found whereas TPH1 only had colonies in the concentrated 500 ul plate. We picked the colonies and inoculated them overnight.</P><P>Meanwhile, we obtained aliquot of competent Nissle, we also received a plated Nissle colonies as well as inoculations in Falcon which was grown overnight.</P></DIV><DIV class="day"><H3>Friday 7<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P>Mini preps of CHAT, TPH1 and antiTNAs were prepared.</P><P>Unfortunately, nissle plate showed no growth.</P></DIV></DIV></DIV><DIV class="about-item9 hide"><DIV class="notebookweek" id="week9"><H2>Week 9(10<SUP>th</SUP> Aug – 16<SUP>th</SUP> Aug)</H2><DIV class="day"><H3>Monday 10<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P>Repeated the digestion of CHAT and TPH1 and ran the gel for digested and undigested CHAT and TPH1. The DNA fingerprint showed only CHAT had been digested.</P><P>PCR cleanup of the amplified antiTNA 1 and 2 from last Tuesday was carried out. Nano drop showed nucleic acid concentrations from 20-35 ng/ul.</P><P>Purified antiTNA 1 and 2 along with pSBIC3 backbone were digested using restriction enzymes XbaI and Pst.1:5 and 1:3 molar ratio ligation of backbone to antiTNA was done and transformed to DH5α competent cells.</P><P>In the afternoon, we repeated the digestion of CHAT and TPH1 again but with single digestion. We ran the gel with linear pSB1C3, CHAT and TPH1. It confirmed ligation of CHAT however, linear plasmids for TPH1 were shorter than pSB1C3, hence unsuccessful for TPH1.</P></DIV><DIV class="day"><H3>Tuesday 11<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P> We prepared four mini preps each of antiTNA1 and antiTNA2. Nano drop confirmed the concentration ranging from (65-147)ng/ul. We digested 4 ul of antiTNA from each mini prep with XbaI and PstI. Unfortunately, as the voltage was too high 120 mV, DNA ran out of this 1% gel so we decided to repeat the experiment at lower voltage 100mV in 2% agarose gel. Meanwhile we digested pTAC with S and P and left the digestion overnight. </P><P> No transformed nissle colonies in the plates.We decided to repeat transformation next week </P></DIV><DIV class="day"><H3>Thursday 13<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P> We purified CHAT PCR and digested pTac however had very low DNA yield for pTac. Hence, we carried out ligation 1:3 ligation of pTAC and CHAT.</P><P>We prepared fresh mini preps of the antiTNA1 and 2 from the previous inoculations as the old mini preps were suspected of contamination. </P></DIV><DIV class="day"><H3>Friday 14<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P>We ran the gel for antiTNAs but only the plasmid backbone was visible. This led us to think either our digestion had not work or because the antiTNAs are only 75bp and 100bp respectively, they were possibly invisible in the gel. </P><P>The ligated pTac:Chat was transformed into competent cells and left in sterile desk at room temperature in a dark environment. </P></DIV></DIV></DIV><DIV class="about-item10 hide"><DIV class="notebookweek" id="week10"><H2>Week 10 (17<SUP>th</SUP>Aug – 23<SUP>rd</SUP> Aug)</H2><DIV class="day"><H3>Monday 17<SUP>st</SUP></H3><H4>Mind the Gut: Effectors</H4><P>We transformed competent Nissle with TPH1 expression cassette plasmid for submission for our art project with talented bio-artmaker <B>Anna Dumitru</B></P></DIV><DIV class="day"><H3>Tuesday 18<SUP>nd</SUP></H3><H4>Mind the Gut: Effectors</H4><P> We had transformed Nissle with different time periods for heat shock and successfully obtained colonies. As from our experiment, heat shock for 180 seconds works better than for 40 seconds.</P><P> We cloned our effectors KAT and GAD in both Ptac and pSB1C3 vectors but antiTNAs only in pSB1C3. We digested, dephosphorylated and ligated the constructs into the designated vectors.</P></DIV><DIV class="day"><H3>Wednesday 19<SUP>rd</SUP></H3><H4>Mind the Gut: Effectors</H4><P> We transformed the effectors with concentrated sample and left them for overnight incubation.</P></DIV><DIV class="day"><H3>Thursday 20<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P>Our patience and hard work had paid off. We finally had some colonies from the transformation. KAT pSB1C3 was disappointing without any colonies and antiTNA1 had merely one colony </P><P> We inoculated the picked colonies in LB media. We picked 4 colonies where possible and left them in shaking incubator for 16 hours. </P></DIV><DIV class="day"><H3>Friday 21<SUP>st</SUP></H3><H4>Mind the Gut: Effectors</H4><P> We made mini preps and obtained the DNA concentration from nano drop. </P><P> We used 100ng of DNA for diagnostic digestion of each effector. We then ran 2% agarose gel for all the antiTNAs and 1% for the rest.</P><P> The gel confirmed the ligation had worked for Pyear TPH and GAD into pSB1C3 and  showed slightly unclear results for pTAC-KAT. </P><P>AntiTNAs were sadly not visible yet again. However, there was a potential band for antiTNA2 and pSB1C3 plasmid.</P><P> We decided to repeat the gel for pTAC-KAT, and antiTNA2 again as they were hopeful but not conclusive.

</P></DIV></DIV></DIV><DIV class="about-item11 hide"><DIV class="notebookweek" id="week11"><H2>Week 11 (24<SUP>th</SUP> Aug – 30<SUP>th</SUP> August)</H2><DIV class="day"><H3>Monday 24<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P> We prepared two separate gels for pTAC-KAT and antiTNA and ran the gel.</P><P>The results from the gel were still inconclusive. Only one of the three KAT pSB1C3 digest had double band. The second band had double bands however, the plasmid band was slightly lower as compared to the other bands of same plasmid.</P></DIV><DIV class="day"><H3>Tuesday 25<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P>We started our work with TPH1 characterisation. We prepared two inoculations of seed culture in LAB media and chrolamphenicol until the growth reached to the optical density of 0.5-0.6. We induced one of the culture with 1mM IPTG and left them overnight in shaking incubator. </P><P> Meanwhile, we also prepared 25mM Tris-2 mM EDTA lysis buffer at pH 8.5.</P></DIV><DIV class="day"><H3>Wednesday 26<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P>We centrifuged the whole culture and resuspended with 2ml of lysis buffer.</P><P> We lysed the cells by sonication (10 cycles of 10s with 15s break in between) and measured the concentration of TPH1 protein using Bradford assay. Meanwhile, 100mM MES buffer at pH 7 was prepared for making tryptophan solution.</P><P>We then made different concentration of TPH1 in 120 ul of tryptophan-Mes solution. </P><P> We also prepared solutions with different concentration of 5HTP as a control for the experiment. </P><P> Finally we plated the first seven columns of 96 well plates in increasing order of TPH1 concentration and the rest of the columns with increasing 5HTP concentrations which we expected would remain constant.</P><P> We expected the fluorescence to increase with increasing TPH1 concentration however, we got mixed result. We had performed the assay with excitation at 320 wavelength and emission at 420 nm. However, the paper we had based our assay on advised execution at 300nm and to measure emission at 330 nm, so we decided to repeat the process with these changes. </P></DIV><DIV class="day"><H3>Thursday 27<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P> Prepared the induced and uninduced sample as on Tuesday.</P></DIV><DIV class="day"><H3>Friday 28<SUP>th</SUP></H3><H4>Mind the Gut: Effectors</H4><P> We centrifuged and lysed our cells by sonication. </P><P> We performed the assay according to the protocol with only changing the excitation to 300nm and emission at 330 nm in fluorescense plate counter. We had good data, as the fluoresense increased with increasing concentration of TPH1 and remained constant for control sample. Yayyy! </P></DIV></DIV></DIV><DIV class="about-item12 hide"><DIV class="notebookweek" id="week12"><H2>Week 12</H2>(31<SUP>st</SUP> Aug– 6<SUP>th</SUP> Sep)<DIV class="day"><H3>Monday 31<SUP>st</SUP></H3><P style="color:red"> BANK HOLIDAY </P></DIV><DIV class="day"><H3>Tuesday 1<SUP>st</SUP></H3><H4>Mind the Gut: Effectors</H4><P> We cloned KAT AND GAD into pTAC and Pyear-TPH1 into pSBIC3. We then transformed them into competent cells and left them to grow overnight.</P></DIV><DIV class="day"><H3>Wednesday 3<SUP>rd</SUP></H3><H4>Mind the Gut: Effectors</H4><P> No colonies in any plate. We left them for longer to incubate as some cultures had previously shown colonies on second day of incubation. </P></DIV><H3>Thursday 4<SUP>th</SUP></H3></DIV><DIV class="day"><H3>Friday 5<SUP>th</SUP></H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>