Team:UCLA/Notebook/Protein Cages/24 June 2015
iGEM UCLA
Home
Team
Meet the Team
Attributions
Collaborations
Projects
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Human Practices
Interlab Study
Parts
Team Parts
Basic Parts
Composite Parts
Part Collection
Notebook
Programming Spider Silk
Functionalizing Silk
Honeybee Silk
Materials Processing
Protein Cages
Interlab Study
Safety
Judging
Phillip's notes
Introduction:  Today will be focused on designing the complete constructs to be ordered.  The sequence from Yen will be used.  They will be designed with the aid of Fasih, in order to utilize Benchling efficiently.  The best candidates to be ordered will also be selected, and if time permits, design of primers for the remaining ones will begin.
-Attempt without Fasih The sequence from Yen was translated using ExPASy to
MPFITVGQENSTSIDLYYEDHGTGTPVVLIHGFPLSGHSWERQSAALLDAGARVITYDRR GFGQSSQPTTGYDYDTFAADLNTVLETLDLQDAVLVGFSMGTGEVARYVSSYGTARIAAV AFLASLEPFLLKTDDNPDGAAPQEFFDGIVAAVKADRYAFYTGFFNDFYNLDENLGTRIS EEAVRNSWNTAASGGFFAAAAAPTTWYTDFRADIPRIDVPALILHGTGDRTLPIENTARV FHKALPSAEYVEVEGAPHGLLWTHAEEVNTALLAFLAKAQEAQKQKLLTEVETYVLSIIP SGPLKAEIAQRLEDVFAGKNTDLEVLMEWLKTRPILSPLTKGILGFVFTLTVPSERGLQR RRFVQNALNGNGDPNNMDKAVKLYRKLKREITFHGAKEISLSYSAGALASCMGLIYNRMG AVTTEVAFGLVCATCEQIADSQHRSHRQLEHHHHHH-
Where – signifies stop.
This was then backtranslated in Gene Designer to avoid EcoRI, NotI, XbaI, SpeI, and PstI.
These were the results.  With an added TGA for stop.  Note the C-terminal HisTag!
>PCquad DNA sequence ATGCCGTTTATTACCGTGGGCCAAGAAAATTCTACGTCTATTGATTTGTATTATGAGGACCATGGTACCGGTACGCCGGTGGTTCTCATCCACGGCTTTCCGCTATCAGGACATTCTTGGGAGCGTCAGAGCGCTGCGCTTTTAGATGCCGGTGCTCGTGTAATAACGTACGATAGACGCGGTTTTGGCCAGAGCTCTCAGCCAACGACTGGATACGATTATGACACCTTCGCCGCCGATTTAAATACTGTTCTGGAAACCCTGGATCTTCAGGATGCGGTCTTAGTTGGTTTTAGTATGGGCACAGGTGAAGTTGCCCGCTACGTCAGTTCTTATGGCACTGCTCGTATTGCAGCGGTAGCTTTTCTGGCTAGCTTAGAACCCTTTCTATTAAAAACCGATGATAATCCGGATGGGGCGGCTCCACAGGAGTTTTTCGACGGAATTGTGGCCGCTGTGAAAGCCGATAGATATGCTTTCTATACTGGGTTTTTCAATGACTTCTATAATTTAGATGAAAACCTGGGAACACGCATCTCGGAAGAGGCTGTACGGAACTCATGGAATACTGCGGCATCTGGCGGATTCTTTGCTGCCGCAGCCGCGCCGACCACTTGGTATACAGATTTTCGTGCGGACATTCCTAGAATTGATGTACCTGCCCTGATTCTGCACGGTACGGGCGACCGTACCTTACCGATTGAGAACACTGCCCGCGTCTTTCATAAAGCTCTTCCGTCCGCTGAGTACGTAGAGGTTGAAGGGGCACCTCATGGTCTGTTATGGACTCATGCTGAAGAAGTCAACACAGCCCTGCTTGCTTTCCTTGCGAAGGCTCAAGAAGCCCAAAAGCAGAAATTACTGACCGAAGTGGAAACTTATGTACTATCCATCATACCGTCTGGTCCCCTTAAAGCAGAAATCGCACAACGTCTGGAGGATGTCTTCGCGGGCAAAAATACAGACCTGGAGGTGCTGATGGAATGGCTTAAGACCCGACCGATTCTGTCACCGTTAACGAAGGGTATCCTAGGATTCGTTTTTACCCTGACGGTGCCCAGTGAGCGCGGACTGCAACGAAGAAGATTCGTCCAAAACGCATTAAACGGGAATGGTGACCCAAACAATATGGACAAAGCGGTGAAGCTGTATCGAAAACTTAAGCGCGAAATAACATTCCATGGGGCCAAAGAAATTAGCCTGAGCTATTCCGCTGGGGCGCTTGCTTCTTGTATGGGTTTAATATATAACCGAATGGGTGCGGTCACCACGGAAGTCGCGTTCGGATTAGTATGCGCGACATGCGAGCAGATTGCAGACTCTCAGCATAGAAGCCATCGCCAGTTAGAACATCATCACCATCATCATTGA
>PCquad amino acid sequence MPFITVGQENSTSIDLYYEDHGTGTPVVLIHGFPLSGHSWERQSAALLDAGARVITYDRRGFGQSSQPTTGYDYDTFAADLNTVLETLDLQDAVLVGFSMGTGEVARYVSSYGTARIAAVAFLASLEPFLLKTDDNPDGAAPQEFFDGIVAAVKADRYAFYTGFFNDFYNLDENLGTRISEEAVRNSWNTAASGGFFAAAAAPTTWYTDFRADIPRIDVPALILHGTGDRTLPIENTARVFHKALPSAEYVEVEGAPHGLLWTHAEEVNTALLAFLAKAQEAQKQKLLTEVETYVLSIIPSGPLKAEIAQRLEDVFAGKNTDLEVLMEWLKTRPILSPLTKGILGFVFTLTVPSERGLQRRRFVQNALNGNGDPNNMDKAVKLYRKLKREITFHGAKEISLSYSAGALASCMGLIYNRMGAVTTEVAFGLVCATCEQIADSQHRSHRQLEHHHHHH The sequence was then annotated on Benchling.  Using PDB ID: 1bro and 1aa7 (bromoperoxidase and M1 protein), the linker was identified.
Next, the full insertion sequence was also generated on Gene Designer. >Mutant Cage Full Insertion GGCCTGGTTCCCCGCGGTTCAGGC >Full Insertion GLVPRGSG This allowed easy insertion of the sequences.  The full sequence was inserted where needed after removing redundant bases.
For the prefix and suffix, the sequence was obtained from this site.  The first prefix sequence was used. http://parts.igem.org/Help:BioBrick_Prefix_and_Suffix prefix-NdeI-gene-XhoI-Histag-suffix
Concluding remarks: Before the rest of the sites are revisited, a meeting will be set up with David and/or Fasih to check our design rationale.
Intro: today we realized that our work the previous day (June 23) was not useful. We had used Dr. Yates' original cage sequence instead of the optimized sequence, which would yield poorer results. Thus, the same procedure of using Benchling to design new cages with the inserted thrombin recognition sites was performed with the new cage sequence above.
Procedure:
1. The cage sequence optimized by Dr. Yates was copied into a benchling file.
2. The thrombin cleavage site was inserted at the area of interest of each site.
3. The resultant sequence was then codon optimized. Integrated DNA Technologies Codon Optimizer, set to Escherichia coli K12, was used. https://www.idtdna.com/CodonOpt
4. The optimized sequence was then cross checked with benchling’s digest function for EcoRI, PstI, SpeI, and XbaI restriction sites.
5. Any residues containing the restriction site were exchanged with different codons to preserve the reading frame and remove undesired digestion.
Tyler Lee
1. Recreated Mutant Sequences(9-13) using PCquad sequence from Yeates(fully optimized)
-PCquad sequence was copied onto Benchling and the mutation was inserted onto this sequence       -Then this sequence was codon optimized using the Gene Designer tool        -Then the sequence was checked for containing any iGEM restriction enzymes(Pst,EcoRI, XbaI, SpeI,NotI) and were removed by replacing nucleotides to avoid unwanted digestion.
Nithin D
