Team:BIT-China/Project/Transduction
BIT-CHINA
Project
Description
Receptor Expression
Pathway optimization
Signal Output
Result
Modeling
Overview
Docking
GPCR pathway
Laboratory
Material
Notebook
Protocol
Equipment
Safety
Achievement
Parts
Medal
HP
HP-Silver
integrated＆Gold
Engagement
Collaboration
Team
Member
Attribution
Sponsor
Pathway optimization
gene knockout
In order to amplify the intensity of the signal, we need to knock out some genes in yeast, including ste2, sst2 and far1 which exist in the endogenous mating pheromone response pathways in yeast and respond to the mating signal and transduction of downstream signal.
Gene
Function
ste2
The GPCR of yeast pheromone receptor is Ste2 protein, which is an important factor during yeasts mating.
sst2
Sst2 protein is a phosphatase catalyzing GTP and producing GDP which will inhibit the release of Gγ、β subunits. Gene knockout of sst2 can improve the pheromone GPSTP.
far1
α factor is an active factor of the yeast pheromone GPSTP, which can cause the yeast cell cycle arresting in the G1 phase through Far1 protein, resulting in that cell growth is inhibited.
Gene knockout can be achieved by homologous recombination.
homologous recombination
By analyzing ste2, sst2 and far1 gene sequences from the NCBI, we ensured the appropriate homologous area in the upstream and the downstream of these genes in saccharomyces cerevisiae genome. In order to knock out these genes effectively, we selected different lengths of homologous arms, 50bp, 200bp and 500bp. According to our experiment result, we found that 200bp and 500bp homologous arms worked more efficiently.
Fig.1 Homologous combination knock-out strategy
We designed 3 pairs of primers whose templates were the upstream homologous arm, the marker and the downstream homologous arm respectively. The markers were Histone synthesis gene, Uracil synthesis gene and Tryptophan synthesis gene, abbreviated as his, ura and trp. Initially, we got 3 fragments by PCR and they had their own overlapping areas with corresponding to each homologous arms.
Secondly, the recombination fragment obtained from OE-PCR was transformed to the CEN.PK2-1C which has been already knocked out the endogenous genes ura, his and trp. Additionally, the colony was chosen by the relevant nutritional deficiency medium.
verification
To verify whether the gene was actually knocked out and to avoid false positive colonies, we designed primer1, 2, 3 and 4 for each gene, as shown in Fig.2. The primer 1 and primer 4 were designed according to the yeast genome. The primer 2 was designed according to the selection maker and primer 3 was to the gene which would be knocked out.
Fig.2 Verification strategy
We got positive result using the primer 1 and 2, meanwhile the negative result would be detected by using primer 1 and 3. Then we sequenced the PCR product using primer 1 and 4 to further confirm the realness of the results.
TOP
gene knockout
homologous recombination
verification
Contact
Institute of Biotransformation and synthetic biosystem
School of life science
Beijing Institute of Technology
100081, Beijing
Email: lichun@bit.edu.cn
Copyright  © 2017 BIT-China
