<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIT-China_Project_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIT-China/Project/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>BIT-CHINA</TITLE></P><DIV class="screen-hidden"><NAV class="navbar navbar-default navbar-fixed-top"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project</A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Project/Description"><SPAN>Description</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Project/Sense"><SPAN>Receptor Expression</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Project/Transduction"><SPAN>Pathway optimization</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Project/Detection"><SPAN>Signal Output</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Project/Results"><SPAN>Result</SPAN></A></LI></UL><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Modeling</A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Model"><SPAN>Overview</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Model/Docking"><SPAN>Docking</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Model/GPCRpathway"><SPAN>GPCR pathway</SPAN></A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Laboratory</A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Material"><SPAN>Material</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Notebook"><SPAN>Notebook</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Protocol"><SPAN>Protocol</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Equipment"><SPAN>Equipment</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Safety"><SPAN>Safety</SPAN></A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Achievement
              </A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Parts"><SPAN>Parts</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Medal"><SPAN>Medal</SPAN></A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">HP</A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/HP/Silver"><SPAN>HP-Silver</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/HP/Gold_Integrated"><SPAN>integrated＆Gold</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Engagement"><SPAN>Engagement</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Collaborations"><SPAN>Collaboration</SPAN></A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team
              </A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Member"><SPAN>Member</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Attributions"><SPAN>Attribution</SPAN></A></LI><LI class="main-nav-link"><A class="dropdown-a" href="https://2017.igem.org/Team:BIT-China/Sponsor"><SPAN>Sponsor</SPAN></A></LI></DIV></DIV></NAV></DIV><SECTION class="content_container" id="mytop"><H2 class="title-h2">Result</H2><P class="my-content-p"> The strategy for the function testing of engineered <I>CEN.PK2-1C</I> stains <SPAN class="my-oblique">△far1</SPAN> +<SPAN class="my-oblique">△sst2</SPAN>,<SPAN class="my-oblique">△sst</SPAN> and <SPAN class="my-oblique">△far1</SPAN>+<SPAN class="my-oblique">△sst2</SPAN>+<SPAN class="my-oblique"><I>△ste2</I></SPAN> were divided into the following parts; </P><P class="my-content-p">(1). Test of the inhibitive effects on the growth of engineered <I>CEN.PK2-1C</I> whose cellular period can be stopped at the G1 stage by  α factor.</P><P class="my-content-p">(2). Expression and identification of Pfus+RFP+cyc1t+PRS42K in engineered <I>CEN.PK2-1C</I> strains <SPAN class="my-oblique">△far1</SPAN>+<SPAN class="my-oblique">△sst2</SPAN>, <SPAN class="my-oblique">△far1</SPAN>+<SPAN class="my-oblique">△sst2</SPAN>+<SPAN class="my-oblique"><I>△ste2</I></SPAN>.</P><DIV class="section-upline cd-section" id="Zone"><H3 class="title-h3">Zone of inhibition</H3><H4 class="title-h4">Plan and method</H4><P class="my-content-p">The growth of <I>CEN.PK2-1C</I> can be inhibited by α factor, we want to  test the function of <SPAN class="my-oblique">△sst2</SPAN>, <SPAN class="my-oblique">△far1</SPAN>+<SPAN class="my-oblique">△sst2</SPAN> and <SPAN class="my-oblique">△far1</SPAN>+<SPAN class="my-oblique">△sst2</SPAN>+<SPAN class="my-oblique"><I>△ste2</I></SPAN> by comparing with <I>CEN.PK2-1C</I> .</P><P class="my-content-p">(1) Explore the influence of the growth of each strains cultured with different concentrations of α factor in the solid plate medium.</P><P class="my-content-p">(2) Observe the growth situation of these strains and compare the differences among them. Analyze the functions of these strains cultured with different concentrations of α factor in the solid plate medium.</P><H4 class="title-h4">Results and discussion</H4><P class="my-content-p">(1) The strain <I>CEN.PK2-1C</I> stops growing at the phase G1 due to the effects of the α factor and zone of inhibition appears in the solid plate medium.</P><DIV class="my-content-box"><SPAN>Fig.1 inhibition zone of <I>CEN.PK2-1C</I> under different concentrations of α factor.</SPAN></DIV><P class="my-content-p">(2) The strain <I>△sst2</I> is more sensitive to α factor. Meanwhile the larger zone of inhibition will appear when the concentration of α factor increased.</P><DIV class="my-content-box2"><SPAN>Fig.2 inhibition zone of <I>△sst2</I> under different concentrations of α factor.</SPAN></DIV><P class="my-content-p">(3)The <I>far1</I> of <SPAN class="my-oblique">△far1</SPAN>+<SPAN class="my-oblique">△sst2</SPAN>, <SPAN class="my-oblique">△far1</SPAN>+<SPAN class="my-oblique">△sst2</SPAN>+<SPAN class="my-oblique"><I>△ste2</I></SPAN> is knocked out successfully, and these strains will still keep growing under the effects acted by α factor and Zone of inhibition will not be shown either.</P><DIV class="my-content-box2"><SPAN>Fig.3 inhibition zone of <I>△far1+△sst2</I> and <I>△sst2+△far1+△ste2</I> under different concentrations of α factor.</SPAN></DIV><P class="my-content-p">Conclusion： Knock out the gene <I>sst2</I> can eliminate the inhibition of the signal so that the strain △<I>sst2</I> can amplify the signal when cultured with α factor. From the results above, the strain <I>△far1+△sst2</I> didn’t stay at the phase G1, because the gene <I>far1</I> knockout eliminates the growth arrest. Gene <I>ste2</I> knockout makes the strain don’t response to the α factor.</P></DIV><DIV class="section-upline cd-section" id="curve"><H3 class="title-h3">Growth curve</H3><P class="my-content-p">Research the impacts acted by the various concentrations of α factor to the growth tendency of these strains in the liquid medium.</P><P class="my-content-p"> The research of the impacts acted by various concentration of α factor to the growth of Cen.PK2-1C,  <SPAN class="my-oblique"><I>△sst2</I></SPAN> of <I>CEN.PK2-1C</I>.</P><H4 class="title-h4">Plan and method</H4><P class="my-content-p">The growth tendency of engineered CEN.PK2-1C strains were determined by the initial concentrations of α factor from 0µM to15 µM. We planned to cultivate every engineered strain with these concentrations of α factor and obtain the value of OD600 of these strains at wavelengths of 600nm during cultivation measured by UV-Vis spectrophotometer which was calibrated by the YPD solution without strains.</P><H4 class="title-h4">Results and discussion</H4><P class="my-content-p">(1) With the concentration of α factor increasing, the growth of <I>CEN.PK2-1C</I> was inhibited more obviously.</P><DIV class="my-content-box"><SPAN>Fig.4 growth curve of <I>CEN.PK2-1C</I> under different concentrations of α factor.</SPAN></DIV><P class="my-content-p">(2)the growth curve of <SPAN class="my-oblique">△sst2</SPAN>  is lower than <I>CEN.PK2-1C</I>. It was strongly inhibited by α factor.</P><DIV class="my-content-box"><SPAN>Fig.5 growth curve of <I>△sst2</I> under different concentrations of α factor.</SPAN></DIV><P class="my-content-p">Conclusion：From the results above, we could see that in the α factor concentration range, the strain would be inhibited by α factor. With the increase of the concentrations of the α factor, the inhibitory effect to the growth of the strains enhanced. Obviously, knocking out the gene sst2 makes the strain respond to α factor more sensitively.</P></DIV><DIV class="section-upline cd-section" id="Fluorescence"><H3 class="title-h3">Fluorescence</H3><P class="my-content-p">Test the expression of Pfus+<I>mRFP</I>+cyc1t+pRS42K in engineered <I>CEN.PK2-1C</I> strains, <SPAN class="my-oblique">△far1</SPAN>+<SPAN class="my-oblique">△sst2</SPAN>, <SPAN class="my-oblique">△far1</SPAN>+<SPAN class="my-oblique">△sst2</SPAN>+<SPAN class="my-oblique"><I>△ste2</I></SPAN>, <SPAN class="my-oblique">△sst2</SPAN>.</P><P class="my-content-p">We research the impacts acted by various concentrations of α factor to the amount of expression of mRFP as well as the growth of engineered <I>CEN.PK2-1C</I> strains <SPAN class="my-oblique"><I>△far1</I></SPAN>+<SPAN class="my-oblique">△sst2、△far1</SPAN>+<SPAN class="my-oblique">△sst2</SPAN>+<SPAN class="my-oblique"><I>△ste2</I>，△sst2</SPAN> containing the detection circuit.</P><H4 class="title-h4">Plan and method</H4><P class="my-content-p">We set the concentrations of α factor as 0umol/L，1 umol/L， 2.5 umol/L， 5 umol/L，10 umol/L，15 umol/L. We sampled 5ml culture liquid of each of them at 0、4、8、10、12、14、16、18、20、22、24、26、28、30h respectively after shake culture. Afterwards, the red fluorescence intensity was measured in microplate reader (584/607) which was calibrated by pure liquid medium. We set red fluorescence intensity/OD<SUB>600</SUB> as Y-axis and time as X- axis for the coordinate system.</P><H4 class="title-h4">Results and discussion</H4><DIV class="my-content-box"><SPAN>Fig.6 fuorescense curve of <I>CEN.PK2-1C</I> under different concentrations of α factor.</SPAN></DIV><DIV class="my-content-box"><SPAN>Fig.7 fuorescense curve of <I>Δsst2+Δfar1</I> under different concentrations of α factor.</SPAN></DIV><DIV class="my-content-box"><SPAN>Fig.8 fuorescense curve of <I>Δsst2+Δfar1+Δste2</I> under different concentrations of α factor.</SPAN></DIV><P class="my-content-p">For the strains <I>△far1+△sst2</I> and <I>CEN.PK2-1C</I>, the red fluorescence intensity increased with the concentrations of α factor increasing, and almost none of the red fluorescence of the strain <I>CEN.PK2-1C</I> was detected. Moreover, the red fluorescence intensity of <I>△far1+△sst2</I> is stronger than that of <I>△far1+△sst2+△ste2</I> and <I>CEN.PK2-1C</I>.</P><P class="my-content-p">Conclusion：According to the results above, the expression intensity of mRFP was induced increasingly by the increasing concentration of α factor. Because of the genes <I>sst2</I>, <I>far1</I> knockout the strains <I>△far1+△sst2+△ste2</I> and <I>△far1+△sst2</I> could amplify the signal which was induced by the inducer. The stain <I>△far1+△sst2+△ste2</I> didn’t respond to α factor due to the gene <I>ste2</I> knockout.</P></DIV><DIV class="section-upline cd-section" id="Sensing"><H3 class="title-h3">Sensing </H3><P class="my-content-p">The expression of human sweet taste receptor in yeast</P><P class="my-content-p">In the sensing part, the G protein coupled receptor (GPCR) T1R2 and T1R3 were heterologously expressed in <I>Saccharomyces Cerevisiae</I> due to its ability to regulate a wide range of cellular processes such as the senses of taste and smell. Moreover, we constructed a mRFP-based assay to achieve proper detection of sweetness responses.</P><P class="my-content-p">So we had to achieve two goals:</P><LI class="my-content-li2">1. To express T1R2 and T1R3 in <I>Saccharomyces Cerevisiae</I>.</LI><LI class="my-content-li2">2. Construct a response signal circuit to confirm that our receptors can detect the sweeteners.</LI><H4 class="title-h4">Results:</H4><LI class="my-content-li2">1. Synthesized the gene of human receptors T1R2 and T1R3</LI><LI class="my-content-li2">2. Expressed the heterologous GPCR in <I>Saccharomyces Cerevisiae</I>.</LI><LI class="my-content-li2">3. The engineered yeast with T1R2 and T1R3 can sense the sweeteners.</LI><H4 class="title-h4">Details of the results:</H4><H4 class="title-h5">1. The gene synthesis of T1R2 and T1R3 </H4><P class="my-content-p">GPCR sequences are extracted from NCBI’s NR database. The sweet taste receptor T1R2-T1R3 (ID: 83756 and 80834) responds to diverse stimuli associated with the human sense of sweet taste, such as sucrose, d-tryptophan and aspartame. In reliable expression in <I>Saccharomyces Cerevisiaei</I>, we finished the codon optimization and reverse transcription of protein sequences into DNA sequences on the website http://54.235.254.95/gd/. The sequences of our receptors were established into the parts BBa_K2368003 and BBa_K2368004.</P><P class="my-content-p">We synthesized the human receptor genes T1R2 and T1R3 using building block which based on overlapping-extension PCR. The PCR result was showed as Fig.9.  </P><DIV class="my-content-box"><SPAN>Fig. 9 The PCR result of synthetic genes T1R2 (2520 bp) and T1R3 (1989 bp).</SPAN></DIV><P class="my-content-p">We used the synthetic genes T1R2, T1R3 to construct the gene circuits for receptors expression. The galactose-induced promoter,  P<SUB>Gal1</SUB>/P<SUB>Gal10</SUB>was used to express the T1R2,T1R3. We cloned the circuits (BBa_2368007 and BBa_2368008) into the shuttle vector pESC-Leu in <I>E.coli</I> Top 10. </P><DIV class="my-content-box"><SPAN>Fig. 10  The gene circuits of T1R2 and T1R3 expression cassettes (BBa_2368007 and BBa_2368008).</SPAN></DIV><DIV class="my-content-box"><SPAN>Fig. 11  The PCR result of T1R2 (3448 bp) and T1R3 (2814 bp) expression cassettes.</SPAN></DIV><H4 class="title-h5">2. 2.Display the location of the receptor by immunofluorescence</H4><P class="my-content-p">In order to confirm that T1R2, T1R3 can be located on the membrane of <I>Saccharomyces Cerevisiae</I>, we fused antigen tags MYC and 6xHIS at the N-terminals of T1R2, T1R3 respectively. The gene circuits for displaying the location of human receptors were constructed as Fig.11. </P><DIV class="my-content-box"><SPAN>Fig. 12 The gene circuit of T1R2 and T1R3 expression cassettes with His and MYC tags </SPAN></DIV><P class="my-content-p">The plasmid, pESC-Leu-T1R2-MYC-T1R3-HIS, was transformed into engineered yeast <I>CEN.PK2-1C</I> (sst2, ste2, far1).   After transformation, the expression of T1R2,T1R3 was induced by adding 2% galactose. T1R2 and T1R3 were confirmed to e located on the membrane of yeast by immunofluorescence. <A href="https://2017.igem.org/Team:BIT-China/Protocol">(More details is showed in our protocols）</A>. </P><DIV class="my-img-box"><SPAN>Fig. 13 The immunofluorescence results about the T1R2 and T1R3 linked with MYC, HIS respectively.The scale bars represent 10µm.</SPAN></DIV><P class="my-content-p">As picture showing, the red fluorescence could be observed around the cells by confocal microscopy. Based on this stained result, we can confirm that T1R2 and T1R3 was expressed and located on the membrane successfully.</P><H4 class="title-h5">3. Function test of T1R2 and T1R3 by adding sweeteners</H4><P class="my-content-p">Although we had simulated the function of T1R2 and T1R3 in modeling, there was still a problem whether T1R2, T1R3 in yeast cells can sense the sweeteners. In order to prove the function of this receptor in yeast cell, another plasmid with our reporter device, pRS42K-mRFP-<I>CYC1t</I>, was transformed into <I>CEN.PK2-1C</I>, More details about this part of work will be displayed in host website <A href="https://2017.igem.org/Team:BIT-China/Project/Transduction">transduction</A> and <A href="https://2017.igem.org/Team:BIT-China/Project/Detection">detection</A>.  </P><P class="my-content-p">The sense device, BBa_2368007 and BBa_2368008, was expressed successfully in the host yeast. Once the receptor can respond the presence of sweeteners, the <I>mRFP</I> will be expressed as output. (Fig. 13)</P><DIV class="my-content-box"><SPAN>Fig. 14  A simplified model of the αpheromone pathway with T1R2/T1R3</SPAN></DIV><P class="my-content-p">Following the above, we separately added a list of sweeteners like fructose, sucrose, glucose into the culture to detect the red fluorescence intensity based on whether its regulatory interaction with sweet taste receptor T1R2-T1R3. </P><P class="my-content-p">So we selected two sweeteners, fructose and sucrose, for the first testing. </P><P class="my-content-p">Firstly, we selected a single cell clone of <I>CEN.PK2-1C</I> with the circuits mentioned previously and cultured this clone in SD medium with 2% glucose and corresponding concentration of antibiotic G418 at 30℃ overnight. </P><P class="my-content-p">After harvesting cells, we centrifuged the cells and added new SD medium with 2% galactose and antibiotic G418 for inducing the expression of T1R2 and T1R3. Incubated at 30℃ for 12 hours.  </P><P class="my-content-p">Then added sweeteners into culture according to Table 1</P><DIV class="my-img-box" style="justify-content: flex-start;"><TABLE class="table-co"><CAPTION>Table 1.  The sweeteners added for detecting function of the system </CAPTION><THEAD><TR><TH style="width: 50%">Sweetener and concentration (w %)</TH><TH>Description</TH></TR></THEAD><TBODY><TR><TD>2%  Fructose</TD><TD>Experimental group</TD></TR><TR><TD>2%  Sucrose</TD><TD>Experimental group</TD></TR><TR><TD>0%</TD><TD>Control group</TD></TR></TBODY></TABLE></DIV><P class="my-content-p">After adding sweeteners, we used plate reader to measure the fluorescence intensity and OD<SUB>600</SUB>, calculated the value of fluorescence intensity/ OD<SUB>600</SUB> as the  mean fluorescence intensity at the single cell level. The mean fluorescence intensity is showed in Fig.15.  </P><DIV class="my-content-box"><SPAN>Fig. 15 The mean fluorescence intensity of experimental group and control group after 12, 16 hours</SPAN></DIV><P class="my-content-p">As the picture shows, the fructose group and sucrose group established obvious increase compared to control group from 12h to 16h. </P><P class="my-content-p">The red fluorescence intensity at the single cell level all increased, but the sucrose group and fructose group increased faster than control group, which meant adding sweeteners indeed accelerated the fluorescence protein production in yeast cells in our system.</P><P class="my-content-p">In other words, our system can be sensitive to sweeteners. Besides, the red fluorescence intensity increasing rate of fructose group was higher than sucrose group’s, which means that the sweetness of fructose is higher than sucrose’s. This result also conforms to the result measured through people tastes.  </P><P class="my-content-p">In order to confirm this result, we repeated this experiment three times. 
The all results showed the similar tendency. These results demonstrate that the system can work as our expectation.</P><P class="my-content-p">Though the fluorescence intensity is still at a low level, the distinction between each group can be observed obviously. The reason may be that natural signal pathway in yeast can’t adapt well with the human sweet taste receptor. The improvements of this work are continuing. (Fig. 16)</P><DIV class="my-content-box"><SPAN>Fig. 16 The schematic diagram of G<SUB>α</SUB> mutation  </SPAN></DIV><H4 class="title-h4">Summary </H4><LI class="my-content-li2">1. We had the optimized codon and synthesized human receptor gene T1R2,T1R3 successfully by using OE-PCR.</LI><LI class="my-content-li2">2.The heterologous receptor was expressed and located on the cell membrane successfully in yeast, which has been confirmed by immunofluorescence.</LI><LI class="my-content-li2">3.Our system can sense sucrose, fructose as our expectation.  </LI>

&gt;

      <DIV class="my-content-box"><SPAN>Fig. 17 The red fluorescence intensity of different sweeteners</SPAN></DIV><P class="my-content-p">To confirm whether our yeast can respond to different sweeteners, we selected fructose, sucrose and glycyrrhizic acid as sweetener candidates. The result showed that the red fluorescence intensity sucrose induced was the lowest among these three sweeteners and it was the highest induced by glycyrrhizic acid. It suggested that our engineered yeast system could respond to different sweeteners indeed. More importantly, we found that the fluorescence intensity is positively correlated with sweetness.</P><DIV class="my-content-box"><SPAN>Fig. 18 The red fluorescence intensity of different concentrations of glycyrrhizic acid</SPAN></DIV><P class="my-content-p">We also found that fluorescence intensity induced by 1 g/L of glycyrrhizic acid and 3 g/L of glycyrrhizic acid was greatly increased than the control group. This result showed that when the concentration of the glycyrrhizic acid reached 1 g/L the fluorescence intensity was higher than 3 g/L of glycyrrhizic acid, which meant that 3 g/L of glycyrrhizic acid might exceed the detection limit of the system. The result also indicates if sweetness is beyond the detection limit, the result may not be accurate. Just like that, beyond a certain limit, the taste of sweet and the concentration of sweeteners are not positively correlated for human beings. </P></DIV></SECTION><NAV id="cd-vertical-nav"><UL><LI><A href="#Zone" data-number="1"><SPAN class="cd-label">Zone of inhibition</SPAN></A></LI><LI><A href="#curve" data-number="2"><SPAN class="cd-label">Growth curve</SPAN></A></LI><LI><A href="#Fluorescence" data-number="3"><SPAN class="cd-label">Fluorescence</SPAN></A></LI><LI><A href="#Sensing" data-number="4"><SPAN class="cd-label">Sensing</SPAN></A></LI></UL></NAV><A class="my-backTop" href="#mytop"><SPAN>TOP</SPAN></A><DIV class="footer-all"><DIV class="footer-right"><H5 class="title-h4">Contact</H5><P class="my-content-p3">Institute of Biotransformation and synthetic biosystem</P><P class="my-content-p3">School of life science </P><P class="my-content-p3">Beijing Institute of Technology</P><P class="my-content-p3">100081, Beijing</P><P class="my-content-p3">Email: lichun@bit.edu.cn</P></DIV></DIV><DIV id="footer-info" class="inner"><P class="my-content-p3" style="color: #777; text-align: center !important;">Copyright  © 2017 BIT-China</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>