<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_lab-protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/lab-protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/lab-protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/lab-protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/lab-protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/lab-protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/lab-protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>Slide Down Box Menu with jQuery and CSS3</TITLE></P><DIV id="a1"><DIV class="content"><UL id="sdt_menu" class="sdt_menu"><LI><A href="https://2011.igem.org/Team:NYMU-Taipei"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Home</SPAN><SPAN class="sdt_descr">Get to HOME</SPAN></SPAN></A></LI><LI><A href="https://2011.igem.org/Team:NYMU-Taipei/concept-on-interface-and-biomedicine"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Idea</SPAN><SPAN class="sdt_descr">Our Idea</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2011.igem.org/Team:NYMU-Taipei/concept-on-interface-and-biomedicine">Concept</A><A href="https://2011.igem.org/Team:NYMU-Taipei/brainstorming-(tagcloud)">Brainstorming</A><A href="https://2011.igem.org/Team:NYMU-Taipei/background-review">Background Review</A></DIV></LI><LI><A href="https://2011.igem.org/Team:NYMU-Taipei/optomagnetic-design"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Design/
							Experiment</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2011.igem.org/Team:NYMU-Taipei/optomagnetic-design">Optomagnetic Design</A><A href="https://2011.igem.org/Team:NYMU-Taipei/modelling-protein-structure-champ-design">Protein Modelling</A><A href="https://2011.igem.org/Team:NYMU-Taipei/immunological-solution">NeuroSymbiosis</A><A href="https://2011.igem.org/Team:NYMU-Taipei/chassis">Chassis</A><A href="https://2011.igem.org/Team:NYMU-Taipei/synthoprime">SynthoPrime</A><A href="https://2011.igem.org/Team:NYMU-Taipei/lab-protocols">Lab Protocol</A></DIV></LI><LI><A href="https://2011.igem.org/Team:NYMU-Taipei/results/optomagnetic-design1"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Results</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2011.igem.org/Team:NYMU-Taipei/results/optomagnetic-design1">Optomagnetic Bacteria</A><A href="https://2011.igem.org/Team:NYMU-Taipei/results/immunological-solution1">NeuroSymbiosis</A><A href="https://2011.igem.org/Team:NYMU-Taipei/results/chassis1">Chassis</A><A href="https://2011.igem.org/Team:NYMU-Taipei/results/parts">Parts</A><A href="https://2011.igem.org/Team:NYMU-Taipei/results/international-collaboration(bloc-magneto-worldwide)">International Collaboration</A></DIV></LI><LI><A href="https://2011.igem.org/Team:NYMU-Taipei/synthetic-neurobiology"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Human Practice &amp; Safety</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2011.igem.org/Team:NYMU-Taipei/synthetic-neurobiology">Synthetic Neurobiology</A><A href="https://2011.igem.org/Team:NYMU-Taipei/human-practices">Human Practice</A><A href="https://2011.igem.org/Team:NYMU-Taipei/evaluation---safety-assessment">Evaluation &amp;
							Safety Assessment</A></DIV></LI><LI><A href="https://2011.igem.org/Team:NYMU-Taipei/the-school"><SPAN class="sdt_wrap"><SPAN class="sdt_link">About Us</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2011.igem.org/Team:NYMU-Taipei/the-school">The School</A><A href="https://2011.igem.org/Team:NYMU-Taipei/the-team">The Team</A><A href="https://2011.igem.org/Team:NYMU-Taipei/china-meetup">China Meetup</A><A href="https://2011.igem.org/Team:NYMU-Taipei/acknowledgement">Acknowledgement</A></DIV></LI></UL></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="General"><FONT size="5"><FONT color="crimson"><B>General</B></FONT></FONT></SPAN></H2><P><B><FONT size="4"><FONT color="green">High-Speed Plasmid Mini Kit Protocol</FONT></FONT></B></P><P><FONT size="3"><FONT color="darkviolet">Geneaid High-Speed Plasmid Mini Kit</FONT></FONT></P><P><FONT size="3"><FONT color="deeppink">
Add provided RNase A to the PD1 Buffer and store at 4ºC; if precipitates have formed in the PD2 Buffer, warm the buffer in a 37ºC water bath, followed by gentle shaking to dissolve. Add absolute ethanol (see the bottle label for volume) to the Wash Buffer prior to initial use.
</FONT></FONT></P><OL><LI>Harvesting: Transfer cultured bacterial cells to a 1.5 ml microcentrifuge tube. Centrifuge at 14-16,000 x g for 1 minute and discard the supernatant. Repeat this step if necessary.
</LI><LI>Re-suspension: Add 200 µl of PD1 Buffer (RNase A added) to the tube and re-suspend the cell pellet by vortex or pipetting. 
</LI><LI>Lysis: Add 200 µl of PD2 Buffer and mix gently by inverting the tube 10 times. Do not vortex to avoid shearing the genomic DNA. Let stand at room temperature for at least 2 minutes to ensure the lysate is homologous. 
</LI><LI>Neutralizatio: Add 300 µl of PD3 Buffer and mix immediately by inverting the tube 10 times. Do not vortex. Centrifuge at 14-16,000 x g for 3 minutes. 
</LI><LI>DNA Binding: Place a PD Column in a 2 ml Collection Tube. Add the supernatant from Step 4 to the PD Column and centrifuge at 14-16,000 x g for 30 seconds. Discard the flow-through and place the PD Column back in the 2 ml Collection Tube. 
</LI><LI>Wash: Add 400 µl of W1 Buffer into the PD Column. Centrifuge at 14-16,000 x g for 30 seconds. Discard the flow-through and place the PD Column back in the 2 ml Collection Tube. Add 600 µl of Wash Buffer (ethanol added) into the PD Column. Centrifuge at 14-16,000 x g for 30 seconds. Discard the flow through and place the PD Column back in the 2 ml Collection Tube. Centrifuge at 14-16,000 x g again for 3 minutes to dry the column matrix.
</LI><LI>DNA Elution: Transfer the dried PD Column to a new microcentrifuge tube. Add 50 µl of Elution Buffer or TE into the center of the column matrix. Let stand for at least 2 minutes to allow the Elution Buffer or TE to be completely absorbed. Centrifuge at 14-16,000 x g for 2 minutes to elute the DNA.
</LI></OL><P><B><FONT size="4"><FONT color="green">Restriction Enzyme Digestion</FONT></FONT></B></P><UL><LI>15μl DNA
</LI><LI>0.5μl E1
</LI><LI>0.5μl E2
</LI><LI>2μl 10X buffer 
</LI><LI>2μl H2O
</LI><LI><FONT color="blue">20 μl total volume
</FONT></LI></UL><P><B><FONT size="4"><FONT color="green">Gel Extraction</FONT></FONT></B></P><P><FONT size="3"><FONT color="darkviolet">Geneaid Gel/PCR DNA Fragments Extraction Kit</FONT></FONT></P><P><FONT size="3"><FONT color="deeppink">
Add absolute ethanol (see the bottle label for volume) to the Wash Buffer prior to initial use. 
</FONT></FONT></P><OL><LI>Gel Dissociation: Excise the agarose gel slice containing relevant DNA fragments and remove any extra agarose to minimize the size of the gel slice (TAE buffer is recommended for gel formation). Transfer up to 300 mg of the gel slice to a 1.5 ml microcentrifuge tube. Add 500 µl of DF Buffer to the sample and mix by vortex. Incubate at 55-60ºC for 10-15 minutes to ensure the gel slice has been completely dissolved. During incubation, invert the tube every 2-3 minutes. Cool the dissolved sample mixture to room temperature. 
</LI><LI>DNA Binding: Place the DF Column in a 2 ml Collection Tube. Transfer 800 µl of the sample mixture from the previous step to the DF Column. Centrifuge at 14-16,000 x g for 30 seconds. Discard the flow-through and place the DF Column back in the 2 ml Collection Tube (If the sample mixture is more than 800 µl, repeat the DNA Binding Step). 
</LI><LI>Wash: Add 400 µl of W1 Buffer into the DF Column. Centrifuge at 14-16,000 x g for 30 seconds and then discard the flow-through. Place the DF Column back in the 2 ml Collection Tube. Add 600 µl of Wash Buffer (ethanol added) into the DF Column and let stand for 1 minute. Centrifuge at 14-16,000 x g for 30 seconds and then discard the flow-through. Place the DF Column back in the 2 ml Collection Tube. Centrifuge at 14-16,000 x g for 3 minutes to dry the column matrix. 
</LI><LI>DNA Elution: Transfer the dried DF Column to a new 1.5 ml microcentrifuge tube. Add 20-50 µl of Elution Buffer or TE into the center of the column matrix. Let stand for at least 2 minutes to ensure the Elution Buffer is absorbed by the matrix. Centrifuge for 2 minutes at 14-16,000 x g to elute the purified DNA.
</LI></OL><P><B><FONT size="4"><FONT color="green">PCR Clean Up Protocol</FONT></FONT></B></P><P><FONT size="3"><FONT color="darkviolet">Geneaid Gel/PCR DNA Fragments Extraction Kit</FONT></FONT></P><P><FONT color="deeppink">Add absolute ethanol (see the bottle label for volume) to the Wash Buffer prior to initial use 
</FONT></P><OL><LI>Sample Preparation: Transfer up to 100 µl of a reaction product to a 1.5 ml microcentrifuge tube. Add 5 volumes of DF Buffer to 1 volume of the sample and mix by vortex. 
</LI><LI>DNA Binding: Place a DF Column in a 2 ml Collection Tube. Transfer the sample mixture from step 1 to the DF Column and Centrifuge at 14-16,000 x g for 30 seconds. Discard the flow-through and place the DF Column back in the 2 ml Collection Tube. 
</LI><LI>Wash: Add 600 µl of Wash Buffer (ethanol added) into the center of the DF Column and let stand for 1 minute. Centrifuge at 14-16,000 x g for 30 seconds. Discard the flow-through and place the DF Column back in the 2 ml Collection Tube. Centrifuge again for 3 minutes at 14-16,000 x g to dry the column matrix.
</LI><LI>DNA Elution: Transfer the dried DF Column to a new 1.5 ml microcentrifuge tube. Add 20-50 µl of Elution Buffer or TE into the center of the column matrix. Let stand for at least 2 minutes to ensure the Elution Buffer is completely absorbed by the matrix. Centrifuge for 2 minutes at 14-16,000 x g to elute the purified DNA.
</LI></OL><H2><SPAN class="mw-headline" id="Gene_Transfer"><FONT size="5"><FONT color="crimson"><B>Gene Transfer</B></FONT></FONT></SPAN></H2><P><B><FONT size="4"><FONT color="green">Transformation</FONT></FONT></B></P><P><FONT size="3"><FONT color="darkviolet">ECOS<SUP>TM</SUP>One-minute competent cells</FONT></FONT></P><OL><LI>Prepare 42 degree Celsius water bath, plating beads and selective plates. Thaw frozen competent cells in -80 degree Celsius with room-temp. Water bath to obtain approximately 1/3 thawed state.
</LI><LI>Add DNA (DNA in 4 degree Celsius or ice-bathed plasmids or ligation mixture), whose total volume is less than 5% of volume of competent cells and vortex for 1 second. Wait 5 minutes.
</LI><LI>Incubate in 42 degree Celsius water bath for 45 seconds. 
</LI><LI>Transfer onto chilled and dry selection plate media, then spread the competent cells.
</LI><LI>Immediately incubate plate at 37 degree Celsius for 12-16 hours.
</LI></OL><P><FONT size="4"><FONT color="green"><B>Electroporation for <I>E. coli</I></B></FONT></FONT></P><P><FONT size="3"><FONT color="darkviolet">Preparation of Electrocompetent<I> E. coli</I></FONT></FONT></P><P><FONT size="3"><FONT color="deeppink">(Bjorkman Group, Howard Hughes Medical Institute at California Institute of Technology)</FONT></FONT></P><P>Before starting procedure, prepare/chill  the following:
1 L  2XYT media (no antibiotics!), store at RT
1 L chilled autoclaved dH20, stored in 4° cold room.
100ml chilled 10% glycerol / dH20 solution, store at 4°C.
4 Corning 250ml pointed bottles (w/ orange caps) with white adapters chilled at 4°C.
(used in Beckman tabletop GPR with GH-3.7 rotor, Church 64)
Rinse bottles first with 95% EtOH, then really well with dH20. No soap         !
</P><OL><LI>Preferably, select single colony of <I>E. coli</I> from fresh LB plate for inoculating a 10 ml 2XYT overnight (O/N) starter culture. Alternatively, streak out frozen glycerol stock of bacterial cells onto LB plate, grow plate O/N, and then select single colony for starter culture. Grow 10 ml starter culture O/N in 37°C shaker (250rpm).
</LI><LI>Inoculate 1L of 2XYT media and place culture in 37° shaker. Grow cells and measure OD600 every 45min-1hr. When the OD600 equals 0.6-0.9 (log phase growth), remove the cells from the shaker and place on ice. <FONT color="red">NOTE: It very important to keep the cells at 4°C (or on ice) for the remainder of the procedure.</FONT></LI><LI>Split the 1L culture into four equal parts by pouring ~250ml of culture into each chilled 250ml Corning pointed bottle.
</LI><LI>Spin (#1) in GPR centrifuge at 4000rpm, 25min at 4°C. (if you chose to use the J6/ JS-4.2 rotor (E. Davidson Lab), use 1L bottles , fill half full, spin 4000rpm, 20min, at 4°C.)
</LI><LI>Place bottles on ice. Remove supernate immediately as cell pellet begins to lift off quickly. Gently resuspend each pellet in 200ml ice-cold dH20.
</LI><LI>Spin (#2) in GPR centrifuge at 4000rpm, 25min at 4°C.
</LI><LI>Place bottles on ice. Remove supernate. Gently resuspend each pellet in 100ml of ice-cold dH20.
</LI><LI>Spin (#3) in GPR centrifuge at 4000rpm, 25min at 4°C.
</LI><LI>Place bottles on ice. Remove supernate. Gently resuspend each pellet in 20ml ice-cold 10% glycerol. For each pair of 250ml Corning bottles, transfer both 20ml cell suspension into one chilled 50ml conical tube- therefore you should end up with two 50ml conical tubes on ice where each tube contains ~40ml of cells in 10% glycerol.
</LI><LI>Spin (#4) in GPR centrifuge at 4000rpm, 10min at 4°C.
</LI><LI>Place tubes on ice. Remove supernate. Gently resuspend each cell pellet in 1ml of ice-cold 10% glycerol. Final OD600 of resuspended cells » 200-250.
</LI><LI>With cell suspensions on ice, prepared 70l aliquots of cells in pre-chilled 1.5ml eppendorf tubes. Snap freeze tubes containing cells in liquid N2. Store frozen cells at -80°C.<FONT color="red">NOTE: liquid N2 very hazardous- use caution and don't contact N2 directly!</FONT></LI></OL><P><FONT size="3"><FONT color="darkviolet"><B>Electroporation of Electrocompetent <I>E. coli</I></B></FONT></FONT></P><P><FONT size="3"><FONT color="deeppink">Bjorkman Group, Howard Hughes Medical Institute at California Institute of Technology)</FONT></FONT></P><OL><LI>Locate Electroporator power source and cuvette holder (Bio-Rad). Set the conditions for transformation according to strain. For DH5a cells, use 25 mFD, 200 W, and 1.8 kV. The time constant (tau value) should be 3-4 msec.) For either TG2 or JE2 strains of E. coli, use 25 mFD, 200 W, and 2.5 kV (time constant = 4.6-4.8 msec).
</LI><LI>Thaw required number of frozen cell aliquots (each tube 70l = two transformations) on ice. Thaw plasmid DNA in TE/H20 on ice. Place 15ml conical tube containing 10ml of 2XYT media without antibiotics on ice.
</LI><LI>Place 3l of DNA along wall of 0.2cm cuvette. Pipet 35l of thawed electrocompetent cells onto DNA drop. Flick cuvette to settle DNA + cells mixture into bottom of cuvette.
</LI><LI>Have 1ml pipette containing 1ml of 2XYT media ready. Dry off any moisture from cuvette outside and immediately place cuvette in white plastic holder. Slide holder into position and zap cells. If you hear a high constant tone, immediately add the 1ml of 2XYT to cells! Transfer cells from cuvette into 1.5ml eppendorf tube and store on ice until
</LI><LI>The tone indicates that you have successfully electroporated your cells.  Record the time constant value. Repeat procedure for remaining samples. If you see or hear sparking coming from your cuvette of cells, then the cells are dead!  Repeat that sample again. Things that can cause sparking: excess water on cuvette outside, human skin oil on cuvette outside, too high salt conc. in DNA sample (try diluting DNA 10-fold), and poorly made electrocompetent cells.
</LI><LI>Outgrow transformed cells in eppendorf tubes by incubating the tubes in 37°C water bath for 1-1.5hrs.
</LI><LI>Hi/Lo plate transformed cells onto LB + Amp plates.  Expect ~10<SUP>8</SUP> transformants per 1µg of DNA.Hi plate: Plate 200l of outgrown transformed cell suspension. Lo plate: Spin remaining 800l of cells in microfuge for 20-30sec to pellet cells.  Gently shake open tube over sink. The volume of liquid that clings to tube is roughly 150-200l.  Resuspend cells in this volume of retained media and plate cells (using all the volume) on to LB + Amp plates.
</LI><LI>Place LB + Amp transformant plates in 37°C bacterial incubator for 16-24 hrs until colonies appear.
</LI></OL><P><FONT size="4"><FONT color="green"><B>Electroporation for AMB-1</B></FONT></FONT></P><P><FONT size="3">
After plasmid constructions were done, we need to transform them into AMB-1. Electroporation was performed with a <B>Gene Pulser</B> (Bio-Rad Laboratories, Richmond, California, USA), at a capacitance of 25 micro-F and a resistance of 200Ω, and 0.1-cm cuvettes. Electroporation allows cellular introduction of large highly charged molecules such as DNA which would never passively diffuse across the hydrophobic bilayer core. This phenomenon indicates that the mechanism is the creation of nm-scale water-filled holes in the membrane.</FONT></P><OL><LI>Harvested and washed AMB-1 with 10 mM TES buffer containing 272 mM sucrose (pH 7.5).
</LI><LI>Resuspended AMB-1 in the same buffer at 10<SUP>9</SUP> cells/ml.
</LI><LI>Transferred to 500ul of MSGM supplemented with 20 mM Mg<FONT size="1">2+</FONT> and incubated at 27°C overnight with shaking at 100 rpm.
</LI><LI>Diluted in 5 ml of MSGM containing 0.7% agar.
</LI><LI>Plated on 1% agar in MSGM (supplied with ampicillin at 5 ug/ml or kanamycin at 2.5 ug/ml) incubated under anaerobic conditions.
</LI></OL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:NYMU-Taipei/lab-protocols">http://2011.igem.org/Team:NYMU-Taipei/lab-protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/lab-protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/lab-protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/lab-protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/lab-protocols&amp;oldid=258682" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>