"
Page
Discussion
View source
History
teams
Log in
 
Team:NYMU-Taipei/synthoprime
From 2011.igem.org
Slide Down Box Menu with jQuery and CSS3
Home Get to HOME
Idea Our Idea
Concept Brainstorming Background Review
Design/        Experiment
Optomagnetic Design Protein Modelling NeuroSymbiosis Chassis SynthoPrime Lab Protocol
Results
Optomagnetic Bacteria NeuroSymbiosis Chassis Parts International Collaboration
Human Practice & Safety
Synthetic Neurobiology Human Practice Evaluation & Safety Assessment
About Us
The School The Team China Meetup Acknowledgement
Contents
1Introduction
2HOWTO: User Inputs
3HOWTO: Output Reading
4References and Feedback
Introduction
SynthoPrime – A Synthetic Biology Primer Design Tool for Genomic Parts Creation
SynthoPrime algorithm is developed to provide the access of all the available genomic parts (biological parts from sequenced genomes) for the specific needs of primer design in synthetic biology communities. It provides an automated ONE-STEP primer design service of BioBricks for iGEM. Near 1,400 bacteria and their 2,551 replicons (chromosomes and plasmids) are extracted from NCBI genome database by BioPerl package as genomic templates in our service. The jQuery Javascript library is used for interactive web user interface design to help synthetic biologists to access genomic parts from the sequenced genome templates for primer design. Our online service aims to provide a one-step primer design tool and sequenced genomes as templates to convert sequenced genomes into biological parts with quantitative measures for synthetic biology community. SynthoPrime is freely available for everyone to try out.
HOWTO: User Inputs
STEP 1: Offer your sequence to be ‘biobricked’ by direct pasting your DNA or specifying by names of organism, replicon and gene.
STEP 2: Give your design constraints for primers, including length, melting temperature, GC-content of each primer and differences of Tm between forward and reverse primers. The primer selection process is powered by primer3.
STEP 3: In addition, you can choose to relax design constraints to gain more primer candidates. Besides, you can decide to put prefix or suffix or not by ‘Build Biobrick’ checkbox and specify extra bases before prefix and suffix for good enzyme digestion!
STEP 4: Click “SEND” when everything is ready or click “RESET” to set all parameters as default.
HOWTO: Output Reading
Main output page
Additional Info.
References and Feedback
We would be more than happy to receive your comments and suggestions.
Retrieved from " http://2011.igem.org/Team:NYMU-Taipei/synthoprime "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
