<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_modelling-protein-structure-champ-design"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/modelling-protein-structure-champ-design">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/modelling-protein-structure-champ-design&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/modelling-protein-structure-champ-design&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/modelling-protein-structure-champ-design&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/modelling-protein-structure-champ-design" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/modelling-protein-structure-champ-design</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>Slide Down Box Menu with jQuery and CSS3</TITLE></P><DIV id="a1"><DIV class="content"><UL id="sdt_menu" class="sdt_menu"><LI><A href="https://2011.igem.org/Team:NYMU-Taipei"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Home</SPAN><SPAN class="sdt_descr">Get to HOME</SPAN></SPAN></A></LI><LI><A href="https://2011.igem.org/Team:NYMU-Taipei/concept-on-interface-and-biomedicine"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Idea</SPAN><SPAN class="sdt_descr">Our Idea</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2011.igem.org/Team:NYMU-Taipei/concept-on-interface-and-biomedicine">Concept</A><A href="https://2011.igem.org/Team:NYMU-Taipei/brainstorming-(tagcloud)">Brainstorming</A><A href="https://2011.igem.org/Team:NYMU-Taipei/background-review">Background Review</A></DIV></LI><LI><A href="https://2011.igem.org/Team:NYMU-Taipei/optomagnetic-design"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Design/
							Experiment</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2011.igem.org/Team:NYMU-Taipei/optomagnetic-design">Optomagnetic Design</A><A href="https://2011.igem.org/Team:NYMU-Taipei/modelling-protein-structure-champ-design">Protein Modelling</A><A href="https://2011.igem.org/Team:NYMU-Taipei/immunological-solution">NeuroSymbiosis</A><A href="https://2011.igem.org/Team:NYMU-Taipei/chassis">Chassis</A><A href="https://2011.igem.org/Team:NYMU-Taipei/synthoprime">SynthoPrime</A><A href="https://2011.igem.org/Team:NYMU-Taipei/lab-protocols">Lab Protocol</A></DIV></LI><LI><A href="https://2011.igem.org/Team:NYMU-Taipei/results/optomagnetic-design1"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Results</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2011.igem.org/Team:NYMU-Taipei/results/optomagnetic-design1">Optomagnetic Bacteria</A><A href="https://2011.igem.org/Team:NYMU-Taipei/results/immunological-solution1">NeuroSymbiosis</A><A href="https://2011.igem.org/Team:NYMU-Taipei/results/chassis1">Chassis</A><A href="https://2011.igem.org/Team:NYMU-Taipei/results/parts">Parts</A><A href="https://2011.igem.org/Team:NYMU-Taipei/results/international-collaboration(bloc-magneto-worldwide)">International Collaboration</A></DIV></LI><LI><A href="https://2011.igem.org/Team:NYMU-Taipei/synthetic-neurobiology"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Human Practice &amp; Safety</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2011.igem.org/Team:NYMU-Taipei/synthetic-neurobiology">Synthetic Neurobiology</A><A href="https://2011.igem.org/Team:NYMU-Taipei/human-practices">Human Practice</A><A href="https://2011.igem.org/Team:NYMU-Taipei/evaluation---safety-assessment">Evaluation &amp;
							Safety Assessment</A></DIV></LI><LI><A href="https://2011.igem.org/Team:NYMU-Taipei/the-school"><SPAN class="sdt_wrap"><SPAN class="sdt_link">About Us</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2011.igem.org/Team:NYMU-Taipei/the-school">The School</A><A href="https://2011.igem.org/Team:NYMU-Taipei/the-team">The Team</A><A href="https://2011.igem.org/Team:NYMU-Taipei/china-meetup">China Meetup</A><A href="https://2011.igem.org/Team:NYMU-Taipei/acknowledgement">Acknowledgement</A></DIV></LI></UL></DIV></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#What_is_the_CHAMP_Design"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">What is the CHAMP Design</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Why_We_Use_the_CHAMP_Design"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Why We Use the CHAMP Design</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#How_to_Design_the_CHAMP_Peptides"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">How to Design the CHAMP Peptides</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="What_is_the_CHAMP_Design"><B><FONT size="5"><FONT color="crimson">What is the CHAMP Design</FONT></FONT></B></SPAN></H2><P><FONT size="3">
CHAMP, the computed helical anti-membrane protein, is designed by one of the computational and genetic methods available to engineer antibody-like molecules that target the water-soluble regions of tansmembrane (TM) proteins (Hang Yin, et al., 2007).
</FONT></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:826px;"><DIV class="thumbcaption">Fig. 1a: Close-up of the predicted tightly packed interface between designed peptides and target protein. The target protein is represented by a red surface with a blue “hot spot”. Fig. 1b: The CHAMP backbone is depicted in ribbon representation with key positions designated for computational design shown in green (Hang Yin, et al., 2007)</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Why_We_Use_the_CHAMP_Design"><B><FONT size="5"><FONT color="crimson">Why We Use the CHAMP Design</FONT></FONT></B></SPAN></H2><P><FONT size="3">
The general purpose of CHAMP design is that though Transmembrane (TM) helices usually play essential roles in biological processes, companion methods to target the TM regions are lacking. The CHAMP design of TM helices that specifically recognize membrane proteins would advance the understanding of sequence-specific recognition in membranes and simultaneously would provide new approaches to modulate protein-protein interactions in membranes (Hang Yin, et al., 2007).
</FONT></P><P>Now we want to let the CHAMP design play a pivotal role in our project to modulate the protein-protein interactions in membranes. But why is the CHAMP design so important in our iGEM project this year? <FONT color="mediumblue">In this year's <A href="/wiki/index.php?title=Link_here&amp;action=edit&amp;redlink=1" class="new" title="Link here (page does not exist)"><SPAN style="color:dark violet;">Optomagnetic design</SPAN></A>, we want to use the designed peptide, CHAMP sequence, to inhibit the tight interaction between the two helices of membrane protein Mms13.</FONT> Then we can successfully use mechanical force to change the conformation of Mms13 to induce the BiFC-based BRET phenomenon. If we do not have the target peptides, CHAMP, the two helices of Mms13 would tightly &quot;stick&quot; together and we can predict the results easily by the fundamental knowledge of physics that the magnetic force applied to the bacteria would not make any change to the conformation of Mms13 protein. If the two helices of Mms13 have tight interaction all the time, the wobbling light we expect in our project derived from BiFC-based BRET would not be excited. As the consequence of lacking the modulation of CHAMP, we would not get our final wobbling fluorescence but the constant light derived from a luciferase reaction; which means that we can only “turn on” neurons with constant light while the wobbling system can excite or inhibit the neurons for both “on and off” functions.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:480px;"><DIV class="thumbcaption">Fig. 2: The figure in the left is our initial design by using BRET phenomenon to generate the light. As for the right one, we make a little change using BiFC-based BRET with CHAMP, our designed peptides to make the light wobbling and induce the on-and-off neural control system.</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="How_to_Design_the_CHAMP_Peptides"><B><FONT size="5"><FONT color="crimson">How to Design the CHAMP Peptides</FONT></FONT></B></SPAN></H2><P><FONT size="3">
We followed the design protocols in the paper- <I><B>Supporting Online Material for Computational Design of Peptides That Target Transmembrane Helices</B></I> (Hang Yin, et al., 2007) and <I><B>Understanding Membrane Proteins. How to Design Inhibitors of Transmembrane Protein–Protein Interactions</B></I> (J.S. Slusky, et al., 2009). However, because of the limited resources, we made little adjustments by using different programs, such as TMhit (Lo A., et al., 2009), ProtMod (Godzik A., et al., 2011), YASARA (Krieger E., et al., 2011), VMD (Klaus Schulten., et al., 2011), ΔG prediction server v1.0 (Hessa, T., et al., 2007) to finalize our CHAMP design. The study by <I><B>Interaction and conformational dynamics of membrane-spanning protein helices</B></I> (D Langosch, et al., 2009) and <I><B>Helix-helix interaction patterns in membrane proteins</B></I> (D Langosch, et al., 2010) gave us more knowledge about protein-protein interactions in transmembrane domains, and helped us during the CHAMP designing process. 
</FONT></P><P><FONT size="4"><FONT color="green">Membrane peptide design</FONT></FONT></P><P><FONT size="3">
Transmembrane helical peptides can bind to targets with high specificity and affinity has been designed de novo (Hang Yin, et al., 2007). The method consists of four steps:
</FONT></P><OL><LI>Selection of a helical-pair structural motif based on sequence; 
</LI><LI>Selection of a nativelike helical-pair backbone within the chosen structural motif;
</LI><LI>Threading the sequence of the targeted TM helix onto one of the two helices of the selected pair;
</LI><LI>Selection of the amino acid sequence of the designed peptide helix with a side chain repacking algorithm.
</LI></OL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:752px;"><DIV class="thumbcaption">Fig. 3:Design process for peptides that bind to transmembrane proteins. a. Superimposed helicalpair backbones of one structural motif. b. Selected backbone. c. Target structure is threaded onto one helix (front helix), positions for repacking are selected on the other helix. Repacked positions are highlighted in light gray with spheres on Cα. d. The designed helix (farther back) is repacked to have good van der Waals contacts with the target helix. Repacked positions shown in light gray with spheres on Cα. (Hang Yin, et al., 2007)</DIV></DIV></DIV></DIV><P>Our design based on these steps is shown in the following sections.
</P><P><FONT size="4"><FONT color="darkviolet">Section 1. Selection of a helical-pair structural motif based on sequence</FONT></FONT></P><P><FONT size="3"><FONT color="deeppink">A. Determining the domains of Mms13's amino acid sequence</FONT></FONT></P><P><FONT size="3">
The full sequence of Mms13 is (A.A.)
</FONT></P><PRE><I>MPFHLAPYLAKSVPGVGVLGALVGGAAALAKNVRLLKEKRITNTEAAIDTGKETVGAGLATALSAVAATAVGGGLVVSLGTALVAGVAAKYAWDRGVDLVEKELNRGKAANGASDEDILRDELA</I>.
</PRE><P><FONT size="3">
Because Mms13's structure is not 100% defined and known. We used several online prediction programs to specify the domains of Mms13 using amino acid sequences. We used <I>sosui</I> (Mitaku S., et al., 2002), <I>DAS(low cut-off)</I> (M. Cserzo, et al., 1997),  <I>DAS(high cut-off)</I> (M. Cserzo, et al., 1997), <I>TMHMM</I> (Krogh A., et al., 2001), <I>PredictProtein</I> (Laszlo Kajan, et al., 2011), <I>TMpred</I> (K. Hofmann, et al., 1993), <I>HMMTOP</I> (G.E Tusnády, et al., 2001), <I>MEMSAT-SVM</I> (Nugent, T., et al., 2009),  <I>TopPred</I> (Gunnar von Heijne, 1992). <FONT color="mediumblue">We then merged all the results using the cross-model majority vote to determine the boundaries of the two helices of Mms13.</FONT></FONT></P><P><B>Helix 1:a.a.12~29</B> and <B>Helix2:a.a.65~87</B>.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:803px;"><DIV class="thumbcaption">Fig. 4: Part of result for cross-model majority vote</DIV></DIV></DIV></DIV><P><FONT size="3"><FONT color="deeppink">B. Selection of a helical-pair structural motif based on sequence</FONT></FONT></P><P><FONT size="3"><FONT color="mediumblue">We chose the TM helix which has a G-X3-G sequence motif.</FONT> The GxxxG motifs can induce the helices' interaction since it leads to the formation of a flat helix surface and can maximizes van der Waal’s interactions and that the loss of side-chain entropy upon association is minimal for Gly. (William P Russ, et al., 2000) Moreover, the Gly residues in the motifs reduce the distance between the helix axes facilitating hydrogen bond formation between the Cα-hydrogens and the backbone carbonyl of the partner helix (A. Senes, et al., 2001). After searching along the helices of Mms13, we found the critical G-X3-G sequence motifs on helix1 of the Mms13 (See Figure 5).
</FONT></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Fig. 5: The critical G-X3-G motifs on helix1 of the Mms13</DIV></DIV></DIV><P><FONT size="4"><FONT color="darkviolet">Section 2. Selection of a nativelike helical-pair backbone within the chosen structural motif</FONT></FONT></P><P><FONT size="3">
After the selection of a helical-pair structural motif, we have to select a native-like helical-pair backbone as the templates for our CHAMP design. First, we follow the instructions to determine the template cluster. (R. F. S. Walters, et al., 2006) A library of 445 helical pairs from 31 proteins in the paper was clustered into groups based on their three-dimensional similarity. Different clusters have specific features and also show varying degrees of homogeneity. We finally choose cluster 2 as our candidates because of the antiparallel helices on Mms13 and the traits which belong to right-handed crossing angle on helix1. (For examples, the helix1 have small residues at the helix–helix interface, and they are also spaced at four-residue intervals.) The pie chart (See Figure 6) shows the fraction of the total number of pairs that fall within a given cluster and Table 1 (See Figure 7) shows some characteristics of the top 14 clusters.
</FONT></P><P>After determining the cluster, we still have to choose one final template from within the 71 candidates of cluster 2. We first eliminated the number of candidates to 27 templates through the prediction of van der Waals minimum distance of the helical pairs based on the uniformity of packing assessed by finding structures with the minimal number of inter-atomic contacts that are closer than 1.0 Å from the expected van der Waals minimum distance. (Hang Yin, et al., 2007)
</P><P>Then to get the most matched template of our CHAMP design, we used the online program-TMhit (Lo A., et al., 2009) inferring helix-helix interactions from residue contact in Mms13. <FONT color="mediumblue">We selected the best template pairs by finding the template having one helix with the most residue contacts with Mms13 and whose sequence of another helix has the most residue alignments with Mms13.</FONT> After comparison, we chose protein <B>1iwg: A437-A450 and A932-A945</B> as the template pairs for our CHAMP design. Part of the data for the comparison is shown in Figure 8).
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:692px;"><DIV class="thumbcaption">Fig. 7: Table1 with some characteristics of the clusters (R. F. S. Walters, et al., 2006)</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:232px;"><DIV class="thumbcaption">Fig. 6: The pie chart with the fraction of a given cluster (R. F. S. Walters, et al., 2006)</DIV></DIV></DIV><CENTER><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Fig. 8</DIV></DIV></DIV></CENTER><P><FONT size="4"><FONT color="darkviolet">Section 3. Threading the sequence of the targeted TM helix onto the selected pair</FONT></FONT></P><P><FONT size="3">
By using the online program-<B>ProtMod-Protein Structure Modeling-Alignment</B> (Godzik A., et al., 2011), we got the pdb file of the repacking structure, where Mms13 is aligned onto the helix of the chosen template-<B>1iwg: A437-A450</B>. 
</FONT></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig. 10a: The whole 1iwg protein. Figure 10b: The selected template pairs of 1iwg. Figures are graphed using YASARA View (Krieger E., et al., 2011)</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:312px;"><DIV class="thumbcaption">Fig. 9: The aligned Mms13 structure from ProtMod (Godzik A., et al., 2011) graphed using YASARA View (Krieger E., et al., 2011).</DIV></DIV></DIV><P>After having both materials: the aligned Mms13 and the selected template pairs (1iwg: A437-A450; A932-A945), we threaded the Mms13 aligned sequence onto the corresponding helix. The threading process was finished by matching G-X3-G motif of Mms13 to the motif on the template structure. The backbones of the complex were then minimized to remove clashes with the threaded Mms13 sequence (See Figure 11). 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Fig. 12: The minimal inter-helical distances predicted by YASARA View (Krieger E., et al., 2011)</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Fig. 11</DIV></DIV></DIV><P><FONT size="4"><FONT color="darkviolet">Section 4. Selection of the amino acid on CHAMP design with a side chain repacking algorithm</FONT></FONT></P><P><FONT size="3">
We gave a second stage repacking of the minimized structure and selected the one that showed a uniformly packed interior. We first found the proximal positions between Mms13 and the CHAMP template since the effect of the peripheral sequences are less dominant in the helical interaction. We used YASARA View (Krieger E., et al., 2011) to calculate the minimal inter-helical distances, VMD (Klaus Schulten., et al., 2011) to show the hydrophobic regions, and both to specify the proximal positions between helical pairs. The results are shown in Figures 12 and 13.
</FONT></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:392px;"><DIV class="thumbcaption">Fig. 13: The hydrophobic regions shown by VMD (Klaus Schulten., et al., 2011) indicate the peripheral regions since the membrane lipid bilayer is also non hydrophilic.</DIV></DIV></DIV></DIV><P>On CHAMP anti-mms13 helix, seven proximal positions were selected for secondary repacking based on their proximity to the mms13-threaded helix. We used the <B>YASARA Energy Minimization Server </B>(Krieger E., et al., 2009) and the <B>ΔG prediction server v1.0</B> (Hessa, T., et al., 2007) to choose the residue sequences of proximal residues. We calculated the potentials following the potential energy functions (Hang Yin, et al., 2007) shown in Figure 14.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:642px;"><DIV class="thumbcaption">Fig. 14: The Summations of the energy potential equation. (Hang Yin, et al., 2007)</DIV></DIV></DIV><P><FONT color="mediumblue"><B>YASARA Energy Minimization Server</B> (Krieger E., et al., 2009) is performed to calculate the <B>EvdW + EHbond + Eelectrostatic + Econtact </B>.</FONT> Though the program is based on predicting the potentials in the water, we can still rank the prefer residues by using the result scores as support for our decision. <FONT color="mediumblue">The other program, <B>ΔG prediction server v1.0</B> (Hessa, T., et al., 2007), in the other hand, we use it for computing the <B>Esolvation + Eself</B> potentials.</FONT> The results of <B>ΔG prediction server v1.0</B> (Hessa, T., et al., 2007) were ranked as well. By the cross-program comparison, we decided which residue is best for the corresponding proximal position.
</P><P>For each step of the energy calculation process, the identity of one of the proximal positions on the CHAMP helix was mutated. (Hang Yin, et al., 2007) By using the exhaustive enumeration and the cross-program ranking strategy, we determined whether the mutation is either accepted or rejected and set the best candidate residue for the corresponding proximal position. Only the rotamers with high probabilities of occurrence within helical backbones are considered. The data for energy calculation is showed in Figure 15 and 16.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:899px;"><DIV class="thumbcaption">Fig. 15: One example result of <B>YASARA Energy Minimization Server</B> (Krieger E., et al., 2009)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:801px;"><DIV class="thumbcaption">Fig. 16: One example result of <B>ΔG prediction server v1.0</B> (Hessa, T., et al., 2007)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Fig. 17: The second amino acid chosen by cross-ranking results.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Fig. 18: The third amino acid chosen by cross-ranking results.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:901px;"><DIV class="thumbcaption">Fig. 19: The sixth amino acid chosen by cross-ranking results.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Fig. 20: The seventh amino acid chosen by cross-ranking results.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Fig. 21: The tenth amino acid chosen by cross-ranking results.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:901px;"><DIV class="thumbcaption">Fig. 22: The eleventh amino acid chosen by cross-ranking results.</DIV></DIV></DIV></DIV><P>In the paper, when the identities of the proximal positions were determined, the membrane-exposed residues could then be fixed. <FONT color="mediumblue">The membrane-exposed positions are determined by random assignment of hydrophobic residues, with 60% probability given to Leu and 10% to Ala, Ile, Phe, and Val. (Hang Yin, et al., 2007)</FONT> Though it could be done by random selection, we still use <B>ΔG prediction server v1.0</B> (Hessa, T., et al., 2007) computing the energy to choose the CHAMP design with least energy.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:448px;"><DIV class="thumbcaption">Fig. 23: Results for choosing membrane-exposed positions.</DIV></DIV></DIV></DIV><P><FONT color="mediumblue">The main targeting sequences of the CHAMP peptide was finally set, the last step was adding the solubilizing groups at the N- and C-termini to insert the CHAMP into the lipid membrane.</FONT> We added both Lys residues to the termini of our CHAMP design. (Lys or polyethylene glycol-containing amino acids are used in the paper since the amino acids with positive charge can be driven by the negative charge of the lipid bilayer) (Hang Yin, et al., 2007). Our CHAMP design was then completed.
</P><H2><SPAN class="mw-headline" id="References"><B><FONT size="5"><FONT color="crimson">References</FONT></FONT></B></SPAN></H2><UL><LI> Hang Yin, Joanna S. Slusky, Bryan W. Berger, Robin S. Walters, Gaston Vilaire, Rustem I. Litvinov, James D. Lear, Gregory A. Caputo, Joel S. Bennett, William F. DeGrado (2007). Supporting Online Material for Computational Design of Peptides That Target Transmembrane Helices. Science 315, 1817.
</LI></UL><UL><LI>J. S. Slusky, H. Yin and W. F. DeGrado (2009). Understanding Membrane Proteins. How to Design Inhibitors of Transmembrane Protein—Protein Interactions. PROTEIN ENGINEERING Nucleic Acids and Molecular Biology 22, 315-337.
</LI></UL><UL><LI> Lo A., Chiu Y.Y., Rødland E.A., Lyu P.C., Sung T.Y., and Hsu W.L. (2009) Predicting helix-helix interactions from residue contacts in membrane proteins. Bioinformatics 25, 996-1003.
</LI></UL><UL><LI> Lukasz Jaroszewski, Zhanwen Li, Xiao-hui Cai, Christoph Weber, and Adam Godzik. (2011) FFAS server: novel features and applications. Nucleic Acids Res. 39, W38–W44.
</LI></UL><UL><LI>Krieger E, Vriend G. (2002) Models@Home: distributed computing in bioinformatics using a screensaver based approach. Bioinformatics 18(2), 315-8.
</LI></UL><UL><LI>Krieger E, Koraimann G, Vriend G. (2002) Increasing the precision of comparative models with YASARA NOVA--a self-parameterizing force field. Proteins 47(3):393-402.
</LI></UL><UL><LI>Krieger E, Nabuurs SB, Vriend G. (2003) Homology modeling. Methods Biochem Anal.44:509-23.
</LI></UL><UL><LI>Krieger E, Joo K, Lee J, Lee J, Raman S, Thompson J, Tyka M, Baker D, Karplus K. (2009) Improving physical realism, stereochemistry, and side-chain accuracy in homology modeling: Four approaches that performed well in CASP8. Protein 77 Suppl 9:114-22.
</LI></UL><UL><LI>Venselaar H, Joosten RP, Vroling B, Baakman CA, Hekkelman ML, Krieger E, Vriend G. (2010) Homology modelling and spectroscopy, a never-ending love story. Eur Biophys J. 39(4):551-63.
</LI></UL><UL><LI>Joosten RP, te Beek TA, Krieger E, Hekkelman ML, Hooft RW, Schneider R, Sander C, Vriend G. (2011) A series of PDB related databases for everyday needs. Nucleic Acids Res. 39(Database issue):D411-9.
</LI></UL><UL><LI>William Humphrey, Andrew Dalke, and Klaus Schulten. (1996) VMD – Visual Molecular Dynamics. Journal of Molecular Graphics 14:33-38.
</LI></UL><UL><LI> R. Sharma, T. S. Huang, V. I. Pavlovic, K. Schulten, A. Dalke, J. Phillips, M. Zeller, W. Humphrey, Y. Zhao, Z. Lo, and S. Chu. (1996) Speech/gesture interface to a visual computing environment for molecular biologists. In Proceedings of 13th ICPR 96, volume 3, pp. 964-968.
</LI></UL><UL><LI> Rajeev Sharma, Michael Zeller, Vladimir I. Pavlovic, Thomas S. Huang, Zion Lo, Stephen Chu, Yunxin Zhao, James C. Phillips, and Klaus Schulten. (2000) Speech/gesture interface to a visual-computing environment.  IEEE Computer Graphics and Applications 20:29-37.
</LI></UL><UL><LI> David J. Hardy, John E. Stone, and Klaus Schulten. Multilevel summation of electrostatic potentials using graphics processing units. Journal of Parallel Computing 35:164-177.
</LI></UL><UL><LI> Hessa, T., Meindl-Beinker, N., Bernsel, A., Kim, J., Sato, Y., Lerch, M., Lundin, C., Nilsson, I., White, SH. and von Heijne, G. (2007) Molecular code for transmembrane-helix recognition by the Sec61 translocon. Nature. 450, 1026-1030.
</LI></UL><UL><LI> Hessa, T., Kim, H., Lundin, C., Boekel, J., Andersson, H., Nilsson, I., White, SH. and von Heijne, G. (2005) Recognition of transmembrane helices by the endoplasmic reticulum translocon. Nature 433, 377-381.
</LI></UL><UL><LI> Dieter Langosch, Jana R. Herrmann, Stephanie Unterreitmeier and Angelika Fuchs. (2009) Structural Bioinformatics of Membrane Proteins- Helix-helix interaction patterns in membrane proteins. 
</LI></UL><UL><LI> Dieter Langosch, Isaiah T. Arkin. (2009) Interaction and conformational dynamics of membrane-spanning protein helices. Protein Science 18: 1343–1358.
</LI></UL><UL><LI> Hirokawa T., Boon-Chieng S., and Mitaku S. (1998) SOSUI: classification and secondary structure prediction system for membrane proteins. Bioinformatics, 14:378-9.
</LI></UL><UL><LI>Mitaku S., Hirokawa T. (1999) Physicochemical factors for discriminating between soluble and membrane proteins: hydrophobicity of helical segments and protein length. Protein Eng. 11.
</LI></UL><UL><LI> Mitaku S., Hirokawa T., and Tsuji T. (2002) Amphiphilicity index of polar amino acids as an aid in the characterization of amino acid preference at membrane-water interfaces. Bioinformatics, 18:608-16.
</LI></UL><UL><LI>M. Cserzo, E. Wallin, I. Simon, G. von Heijne and A. (1997) Elofsson: Prediction of transmembrane alpha-helices in procariotic membrane proteins: the Dense Alignment Surface method. Prot. Eng. vol. 10, no. 6: 673-676.
</LI></UL><UL><LI>Krogh A, Larsson B, von Heijne G, Sonnhammer EL. (2001) Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes. J Mol Biol. 305(3):567-80.
</LI></UL><UL><LI>Laszlo Kajan, Yachdav, G., Burkhard, R. (2011). High-throughput protein feature prediction in the cloud. Bioinformatics (submitted).
</LI></UL><UL><LI>K. Hofmann &amp; W. Stoffel. (1993)TMbase - A database of membrane spanning proteins segments. Biol. Chem. Hoppe-Seyler:374,166.
</LI></UL><UL><LI>G.E Tusnády and I. Simon (2001) The HMMTOP transmembrane topology prediction server. Bioinformatics 17:849-850.
</LI></UL><UL><LI> Nugent, T. &amp; Jones, D.T. (2009) Transmembrane protein topology prediction using support vector machines. BMC Bioinformatics. 10:159. Epub.
</LI></UL><UL><LI>Jones DT. (2007) Improving the accuracy of transmembrane protein topology prediction using evolutionary information. Bioinformatics. 23: 538-544.
</LI></UL><UL><LI>Jones DT, Taylor WR, Thornton JM. (1994) A Model Recognition Approach to the Prediction of All-Helical Membrane Protein Structure and Topology. Biochem. 33: 3038-3049.
</LI></UL><UL><LI> Gunnar von Heijne. (1992) Membrane Protein Structure Prediction, Hydrophobicity Analysis and the Positive-inside Rule. J. Mol. Biol. 225:487-494.
</LI></UL><UL><LI> William P Russ, Donald M Engelman. (2000) The GxxxG motif: A framework for transmembrane helix-helix association. Journal of Molecular Biology 296, Issue 3: 911-919
</LI><LI> Alessandro Senes, Iban Ubarretxena-Belandia, and Donald M. Engelman. (2001) The Cα—H⋅⋅⋅O hydrogen bond: A determinant of stability and specificity in transmembrane helix interactions. Proc Natl Acad Sci USA.31; 98(16): 9056–9061.
</LI></UL><UL><LI> R. F. S. Walters and W. F. DeGrado. (2006) Helix-packing motifs in membrane proteins. PNAS vol. 103 no. 37:13658-13663.
</LI></UL><UL><LI> Krieger E, Joo K, Lee J, Lee J, Raman S, Thompson J, Tyka M, Baker D, Karplus K Proteins. (2009) Improving physical realism, stereochemistry, and side-chain accuracy in homology modeling: Four approaches that performed well in CASP8. Proteins.77 Suppl 9:114-22.
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:NYMU-Taipei/modelling-protein-structure-champ-design">http://2011.igem.org/Team:NYMU-Taipei/modelling-protein-structure-champ-design</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/modelling-protein-structure-champ-design" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/modelling-protein-structure-champ-design" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/modelling-protein-structure-champ-design&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/modelling-protein-structure-champ-design&amp;oldid=258633" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>