<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UIUC_Illinois_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UIUC Illinois/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:UIUC_Illinois">HOME</A></LI><LI><A href="https://2015.igem.org/Team:UIUC_Illinois/Team">TEAM</A></LI><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2015.igem.org/Team:UIUC_Illinois/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:UIUC_Illinois/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:UIUC_Illinois/Results">Results</A></LI></UL><LI><A href="#">PARTS
            					</A></LI><LI><A href="https://2015.igem.org/Team:UIUC_Illinois/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:UIUC_Illinois/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:UIUC_Illinois/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:UIUC_Illinois/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:UIUC_Illinois/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:UIUC_Illinois/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:UIUC_Illinois/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:UIUC_Illinois/Safety">SAFETY</A></LI></DIV><DIV id="contentContainer"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Project_Description"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Project Description</SPAN></A></LI><LI class="toclevel-1"><A href="#Project_motivation"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project motivation</SPAN></A></LI><LI class="toclevel-1"><A href="#Defining_the_Problem"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Defining the Problem</SPAN></A></LI><LI class="toclevel-1"><A href="#Epigenetic_switches"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Epigenetic switches</SPAN></A></LI><LI class="toclevel-1"><A href="#Recombinase"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Recombinase</SPAN></A></LI><LI class="toclevel-1"><A href="#Introduction:"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Introduction:</SPAN></A></LI><LI class="toclevel-1"><A href="#How_It_Works:_Overview"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">How It Works: Overview</SPAN></A></LI><LI class="toclevel-1"><A href="#How_It_Works:_In_Depth"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">How It Works: In Depth</SPAN></A></LI><LI class="toclevel-1"><A href="#References:"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">References:</SPAN></A></LI></UL></DIV><H2><SPAN class="mw-headline" id="Project_Description">Project Description</SPAN></H2><P>Current biosensors are limited in the sense that they produce digital outputs (either 0 or 1 output) in the presence of an inducer. In order to create a more useful biosensor, analog devices with wider ranges of outputs are necessary. In contrast to digital sensors, an analog device can measure an inducer across a spectrum; for example, an analog device could be used to pinpoint the concentration of iron in groundwater, whereas digital devices could only register whether the concentration exceeded a pre-defined threshold amount.</P><P>Our device, the bacterial tape recorder, will generate such analog outputs by converting chemical inputs into DNA based memory. Using the SCRIBE (Synthetic Cellular Recorders Integrating Biological Events) system developed by Timothy Lu’s lab at MIT, we aim to standardize a device that can characterize the intensity of and duration of events and store them for later retrieval. The SCRIBE system works by integrating plasmid DNA, later becoming specifically ssDNA into genomic DNA upon stimulation. By population analysis of cells that express the analog marker, we will use this system to monitor hazardous environmental factors such as heavy metals, which are known to be detrimental to groundwater in urban areas and developing countries.</P><P>We aim to introduce a degree of modularity that will allow future synthetic biologists to choose what inputs are the stimulus for analog recording.</P><H2><SPAN class="mw-headline" id="Project_motivation">Project motivation</SPAN></H2><P>When brainstorming project ideas, we went over what typically motivates an iGEMmer: Saving the world, making food available to all, fixing the environment, making really cool looking plates with an array of different chromoproteins. But we realized a central theme in all of our projects, there was no concrete way of telling whether or not a cell carried the memory of being manipulated to perform such tasks. An advisor suggested that we look into a new analog way to store cellular memory: SCRIBE. We didn’t really contemplate any other project after that, all we could think about is how we could manifest all of our project ideas using SCRIBE. After researching different storage capabilities using synbio, we realized that SCRIBE truly did have merit, and could possibly be a tool that our iGEM progeny could use!</P><H2><SPAN class="mw-headline" id="Defining_the_Problem">Defining the Problem</SPAN></H2><P>We are all comfortable with how modern memory works using computers. Memory is stored as binary bits: 0’s and 1’s. We call this digital memory, and biologists can implement synthetic digital memory to store bits of information. Two main methods are epigenetic-based devices and recombinase-based devices. Both incorporate digital information, however we aim to introduce an analog set up for “tape-recording” memories.</P><H2><SPAN class="mw-headline" id="Epigenetic_switches">Epigenetic switches</SPAN></H2><P>Epigenetic switches typically record binary bits through expression levels - 0 referring to no expression and 1 referring to expression. Epigenetic memory devices have a few problems however. They require orthogonal transcription factors, and they can lose their state due to fluctuations in the environment or because the cell dies</P><H2><SPAN class="mw-headline" id="Recombinase">Recombinase</SPAN></H2><P>Recombinase-based switches are different than epigenetic switches in that they are able to record data directly into the DNA of the cell. Binary bits are represented by the orientation of large stretches of DNA. However, recombinase-based switches fail to utilize the full recording power of DNA, because they require tedious engineering of these recombination sites in advance, and hundreds of basepairs are used to record a single bit. We see that the recording capacity of these devices is exhausted within a few hours. Furthermore, the scalability of these devices is limited by the number of orthogonal recombinases that exist as well. Enter SCRIBE In an attempt to address the current limitations in current biological systems, Dr. Fahim Farzadfard and Dr. Timothy Lu from MIT developed SCRIBE: an inducible process for creating and inserting single-stranded DNA into a host’s genome.</P><H2><SPAN class="mw-headline" id="Introduction:">Introduction:</SPAN></H2><P>The idea of introducing robust and practical cellular memory systems is an attractive one in the synthetic biology community due to the large storage capacity of DNA, along with its ability to maintain recorded states over long periods of time. The ability to record a cell's encountered states lends itself to sophisticated synthetic biology applications in which a cell can autonomously make complex decisions based on past events, accurately and precisely record environmental conditions for later usage by itself or researchers, and many more.[1]</P><P>Unfortunately, contemporary memory devices that autonomously function in vivo are heavily lacking in their storage capacity, along with intrinsic barriers which hinder scalability. These devices require orthogonal transcription factors or recombinases, and are either unable to maintain their epigenetic, digital state in a noisy cellular environment, or require complex engineering to allow writing to occur directly into DNA. Furthermore, digital memory is inherently limited by its ability to only record the presence or absence of an input, without recording how much input existed, or for how long it was there for.</P><P>To escape these issues, Fahim Farzadfard and Timothy K. Lu developed SCRIBE (Synthetic Cellular Recorders Integrating Biological Events), which is a system that can record both the magnitude and long-term temporal behavior of inputs over a range of values - not just digital signals.</P><H2><SPAN class="mw-headline" id="How_It_Works:_Overview">How It Works: Overview</SPAN></H2><P>In the 1980s, a retroelement called a retron was discovered in various species of gram-negative bacterium. This retron was able to produce msDNA, multicopy single-stranded DNA. It was found that certain regions of the msDNA could be modified, introducing the ability to specifify the ssDNA produced in the cell. This was then placed under the control of various promoters and coexpressed with Beta recombinase, which allowed for, given that the ssDNA had the appropriate homology requirements, recombination into the genome of E. coli. Given that there are 10E-4 recombination events per generation, only a portion of the population will undergo the recombination event. A linear relationship between concentration of inducer and recombination frequency is observed when the data is plotted on a log-log graph. Therefore, by observing the number of recombinants (through functional assays or sequencing), one is able to deduce the amount of inducer that the E. coli population was exposed to.</P><P>&lt;img src=&quot;&quot; alt=&quot;Smiley face&quot; height=&quot;300&quot; width=&quot;200&quot;&gt;&gt;</P><H2><SPAN class="mw-headline" id="How_It_Works:_In_Depth">How It Works: In Depth</SPAN></H2><P>In the 1980s, a research group discovered a retroelement (called a retron) in a species of gram-negative bacterium (Myxococcus xanthus), which was able to produce a single-stranded, linear DNA fragment [3]. Retrons are distinct DNA sequences that code for a reverse transcriptase which is able to synthesize a peculiar DNA-RNA hybrid molecule called msDNA, in which the 5' end of the DNA forms a 2'-5' phosphodiester bond with an internal guanosine residue of the RNA [4].</P><P>There are three components for a single transcript of this retron cassette: a reverse transcriptase, and two RNA portions, msr and msd, which are the primer and template, respectively, for the reverse transcriptase, which are flanked by inverted repeats. When this gets transcribed, base pairing within the transcript leads to the formation of a secondary structure. Reverse transcriptase will then recognize this secondary structure, and use the internal guanosine residue mentioned earlier as a priming site to reverse transcribe the msd sequence. The reverse transcriptases lack an RNase H domain, therefore, cellular RNase H assists in the production of the mature form of msDNA [5].</P><P>It was later found that there were strict structural requirements of the transcript for the successful production of msDNA [6][7]. As such, certain sequences cannot be replaced without the complete loss of function. However, it was found that only the lower portion of the msd stem (the first 21 bases on the 5' end and the last 23 bases on the 3' end of msd) is essential for successful reverse transcription, and that the rest of the stem and loop could be replaced by a sequence of interest.</P><P>This cassette, with a modified msd region, can be put under the control of a promoter, which allows for the production of ssDNA to be related to the presence of a defined input. Furthermore, this can be coexpressed with Beta recombinase, which has been shown to efficiently and precisely introduce ssDNA into sites of the genome that have a minimum of 20 base pairs of complete homology [8]. Beta recombinase works by helping these single strands integrate into the genome at the replication fork during DNA replication. The efficiency of the recombinase is at a relatively medium level, with 10E-4 recombination events occurring per generation. Therefore, the recording of an event is contained within the portion of cells that undergo the recombination event, which can be found through functional assays designed to show the presence of mutations, or sequencing.</P><H2><SPAN class="mw-headline" id="References:">References:</SPAN></H2><P>[1] Farzadfard, F., and T. K. Lu. &quot;Genomically Encoded Analog Memory with Precise in Vivo DNA Writing in Living Cell Populations.&quot; Science 346.6211 (2014): 1256272</P><P>[2] T. S. Gardner, C. R. Cantor, J. J. Collins, Construction of a genetic toggle switch in Escherichia coli. Nature 403, 339–342 (2000). doi: 10.1038/35002131; pmid: 10659857</P><P>[3] Yee T, Furuichi T, Inouye S, Inouye M. &quot;Multicopy single-stranded DNA isolated from a gram-negative bacterium, Myxococcus xanthus.&quot; Cell. 1984 Aug;38(1):203-9.</P><P>[4] Lampson BC1, Inouye M, Inouye S. &quot;Retrons, msDNA, and the bacterial genome.&quot; Cytogenet Genome Res. 2005;110(1-4):491-9.</P><P>[5] Shimamoto T1, Shimada M, Inouye M, Inouye S. &quot;The role of ribonuclease H in multicopy single-stranded DNA synthesis in retron-Ec73 and retron-Ec107 of Escherichia coli.&quot; J Bacteriol. 1995 Jan;177(1):264-7.</P><P>[6] Hsu MY, Inouye S, Inouye M. &quot;Structural requirements of the RNA precursor for the biosynthesis of the branched RNA-linked multicopy single-stranded DNA of Myxococcus xanthus.&quot; J Biol Chem. 1989 Apr 15;264(11):6214-9.</P><P>[7] Shimamoto T1, Hsu MY, Inouye S, Inouye M. &quot;Reverse transcriptases from bacterial retrons require specific secondary structures at the 5'-end of the template for the cDNA priming reaction.&quot; J Biol Chem. 1993 Feb 5;268(4):2684-92.</P><P>[8] V M Watt, C J Ingles, M S Urdea, and W J Rutter, &quot;Homology requirements for recombination in Escherichia coli&quot; PNAS July 1, 1985 vol. 82 no. 14 4768-4772</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>