<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Czech_Republic_Project_Location_tags skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Czech Republic/Project/Location tags</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="panel"><DIV class="centered-list"><DIV id="panel-ul-switch"><DIV id="panel-ul-button"><UL id="panel-ul"><LI><A href="/Team:Czech_Republic/Achievements">Achievements</A></LI><LI><A href="/Team:Czech_Republic/People">People</A></LI><LI><A href="/Team:Czech_Republic/Project">Project</A></LI><LI><A href="/Team:Czech_Republic/Software">Software</A></LI><LI><A href="/Team:Czech_Republic/Practices">Human Practices</A></LI><LI><A href="/Team:Czech_Republic/Parts">Parts</A></LI><LI><A href="/Team:Czech_Republic/Interlab_study">Interlab</A></LI><LI><A href="/Team:Czech_Republic/Notebook">Notebook</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="main-div"><HEADER><H1>Location Tags</H1></HEADER><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Abstract"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Key_Achievements"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Key Achievements</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Introduction"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Agglutinins"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Agglutinins</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Yeast_Display"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Yeast Display</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Design"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Traditional_Yeast_Display"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Traditional Yeast Display</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Choice_of_Markers"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">Choice of Markers</SPAN></A></LI><LI class="toclevel-4 tocsection-9"><A href="#Biotin"><SPAN class="tocnumber">4.1.1.1</SPAN><SPAN class="toctext">Biotin</SPAN></A></LI><LI class="toclevel-4 tocsection-10"><A href="#EpCAM"><SPAN class="tocnumber">4.1.1.2</SPAN><SPAN class="toctext">EpCAM</SPAN></A></LI><LI class="toclevel-4 tocsection-11"><A href="#c-Myc"><SPAN class="tocnumber">4.1.1.3</SPAN><SPAN class="toctext">c-Myc</SPAN></A></LI><LI class="toclevel-4 tocsection-12"><A href="#HuA"><SPAN class="tocnumber">4.1.1.4</SPAN><SPAN class="toctext">HuA</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Modified_Yeast_Display"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Modified Yeast Display</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Expression_Cassette"><SPAN class="tocnumber">4.2.1</SPAN><SPAN class="toctext">Expression Cassette</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Ag.CE.B11p_version"><SPAN class="tocnumber">4.2.2</SPAN><SPAN class="toctext">AgÎ±1p version</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Construction"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Construction</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Construction_of_the_Traditional_Yeast_Display_Plasmids"><SPAN class="tocnumber">4.3.1</SPAN><SPAN class="toctext">Construction of the Traditional Yeast Display Plasmids</SPAN></A></LI><LI class="toclevel-4 tocsection-18"><A href="#EpCAM_2"><SPAN class="tocnumber">4.3.1.1</SPAN><SPAN class="toctext">EpCAM</SPAN></A></LI><LI class="toclevel-4 tocsection-19"><A href="#Anti-c-Myc_.289E10.29"><SPAN class="tocnumber">4.3.1.2</SPAN><SPAN class="toctext">Anti-c-Myc (9E10)</SPAN></A></LI><LI class="toclevel-4 tocsection-20"><A href="#Anti-EpCAM"><SPAN class="tocnumber">4.3.1.3</SPAN><SPAN class="toctext">Anti-EpCAM</SPAN></A></LI><LI class="toclevel-4 tocsection-21"><A href="#Anti-HuA"><SPAN class="tocnumber">4.3.1.4</SPAN><SPAN class="toctext">Anti-HuA</SPAN></A></LI><LI class="toclevel-4 tocsection-22"><A href="#Streptavidin"><SPAN class="tocnumber">4.3.1.5</SPAN><SPAN class="toctext">Streptavidin</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Construction_of_the_Expression_Cassette"><SPAN class="tocnumber">4.3.2</SPAN><SPAN class="toctext">Construction of the Expression Cassette</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Materials_and_Methods"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Strains"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Strains</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Materials"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Methods"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-1 tocsection-28"><A href="#Results"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Validation_of_displayed_proteins_by_immunofluorescent_labeling"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Validation of displayed proteins by immunofluorescent labeling</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Verification_of_interactions_between_EpCAM_and_Anti-EpCAM_scFv"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Verification of interactions between EpCAM and Anti-EpCAM scFv</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Yeast-induced_blood_agglutination"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Yeast-induced blood agglutination</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#The_master_experiment"><SPAN class="tocnumber">6.4</SPAN><SPAN class="toctext">The master experiment</SPAN></A></LI><LI class="toclevel-1 tocsection-33"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><DIV class="info-box"><DIV class="left"><H2><SPAN class="mw-headline" id="Abstract">Abstract</SPAN></H2><P>To guarantee the ability of our system to recognize, bind, and respond to the presence of a tumour cell, we selected five antibody fragments to display on our yeast cells using the yeast display technology and monitor the cells' binding. Their expression was detected and monitored on a microfluidic chip and was also visualized with the use of antibody staining and fluorescence microscopy. 
</P></DIV><DIV class="right"><H2><SPAN class="mw-headline" id="Key_Achievements">Key Achievements</SPAN></H2><UL><LI> Expressed streptavidin, EpCAM, Anti-EpCAM scFv, c-Myc scFv and anti-HuA scFv on the surface of our yeast cells</LI><LI> Monitored the dynamic binding of the displayed fragments of antibodies and corresponding markers</LI><LI> Demonstrated compatibility of our yeast system with blood and functionality of the location tags in blood plasma</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H1><P>To enable our modified <I>Saccharomyces cerevisiae</I> cells to bind extracellular markers, we expressed single-chain variable fragments of antibodies (scFvs) on their surface. These proteins are fused with the adhesion subunit of a-agglutinin, which is a protein naturally used by <I>S. cerevisiae</I> for aggregation during mating. Binding of the antibody to its specific marker enables full development of a signal transduction dependent on the pheromone signaling pathway. We proposed a modified yeast display technique to enable our cells to present the antibody fragments on their surface at all times. We were able to detect and measure the binding of several oncogenic, epithelial, and blood markers.
</P><H2><SPAN class="mw-headline" id="Agglutinins">Agglutinins</SPAN></H2><P>Alpha-agglutinin and a-agglutinin are cell wall glycoproteins of <I>S. cerevisiae</I> that possess complementary surfaces and mediate cell-to-cell adhesion during mating <SUP id="ref_Lipke_and_Kurjan.2C_1992_back" class="ref-back"><A href="#ref_Lipke_and_Kurjan.2C_1992">[Lipke and Kurjan, 1992]</A></SUP>. A-agglutinin consists of two subunits, of which Aga1p is anchored to the membrane by a glycosylphosphatidylinositol anchor (GPI) and to which the second subunit, Aga2p, is bound by two disulphide bonds. Alpha-agglutinin is composed of only one subunit and is connected to a modified GPI anchor. They are both highly glycosylated and constitutively expressed in low levels <SUP id="ref_Zhao_et_al.2C_2001_back" class="ref-back"><A href="#ref_Zhao_et_al.2C_2001">[Zhao et al, 2001]</A></SUP>.
</P><H2><SPAN class="mw-headline" id="Yeast_Display">Yeast Display</SPAN></H2><P>The Yeast Surface Display was first described in 1997 by E. Border and K. Wittrup as a method for engineering cell surface and secretion proteins. It provides a great advantage of the possibility of post-translational modifications in a eukaryotic organism - yeast. One of the most widespread uses of Yeast Display is screening polypeptide libraries for high-affinity antibodies<SUP id="ref_Boder_and_Wittrup.2C_1997_back" class="ref-back"><A href="#ref_Boder_and_Wittrup.2C_1997">[Boder and Wittrup, 1997]</A></SUP>.
</P><H1><SPAN class="mw-headline" id="Design">Design</SPAN></H1><H2><SPAN class="mw-headline" id="Traditional_Yeast_Display">Traditional Yeast Display</SPAN></H2><P>The gene for the protein of interest to be displayed on a cell's surface is fused in frame to the AGA2 gene on a yeast display plasmid. The second part of the display, AGA1 gene is integrated into a chromosome in a special <I>S. cerevisiae</I> strain EBY100. Both of the constructs are originally under a GAL1 promoter, thus requiring galactose to be present in the medium for the system to function. <SUP id="ref_Baird_et_al.2C_2012_back" class="ref-back"><A href="#ref_Baird_et_al.2C_2012">[Baird et al, 2012]</A></SUP>.
</P><P>We obtained the original yeast display system from professor Sheldon Park, which we are thankful for. This particular yeast display plasmid used the BamHI and NheI restriction sites for insertion of scFv sequences and possessed a hemagglutinin tag upstream of the scFv sequence insertion site and a FLAG epitope at the downstream end.
</P><H3><SPAN class="mw-headline" id="Choice_of_Markers">Choice of Markers</SPAN></H3><P>To develop and test our system we chose to express on the surface of our <I>S. cerevisiae</I> cells the following markers.
</P><H4><SPAN class="mw-headline" id="Biotin">Biotin</SPAN></H4><P>Biotin was our first choice of a marker because of its extremely strong bond with streptavidin. It could give us not only great results but also a benchmark affinity value to compare the other antibodiesâ affinities to.
</P><H4><SPAN class="mw-headline" id="EpCAM">EpCAM</SPAN></H4><P>EpCAM, the transmembrane epithelial cell adhesion molecule, functions in cell signaling, proliferation, differentiation, and is present in many carcinomas. We selected it because of its significance in tumor growth promotion and its ability to bind Anti-EpCAM, which we also expressed on <I>S. cerevisiae</I> using yeast display.
</P><H4><SPAN class="mw-headline" id="c-Myc">c-Myc</SPAN></H4><P>C-Myc is a transcription factor that regulates cell growth, apoptosis, and cellular differentiation. Its levels are often upregulated in different cancer types, where it plays a role in gene duplication and Ras interaction.
</P><H4><SPAN class="mw-headline" id="HuA">HuA</SPAN></H4><P>HuA stands for human antigen A of red blood cells. The decision to enable the expression of an antibody against antigen A on yeast surface allows us to test our system directly in blood and prove that the positive agglutination reaction is specific and clearly visible.
</P><H2><SPAN class="mw-headline" id="Modified_Yeast_Display">Modified Yeast Display</SPAN></H2><P>Furthermore, we made several modifications to the yeast display method in order for it to better suit our needs. The goal was to simplify it and reduce the amount of new DNA that needs to be introduced into a yeast cell. The first of two alternatives was based on creating a general cassette containing a promoter, a terminator, and necessary cloning sites. The second alternative meant attaching the specific antibody with a terminal epitope tag directly to AgÎ±1 protein. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:482px;"><DIV class="thumbcaption">Modified plasmid using AgÎ±1p fusion for protein display.</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Expression_Cassette">Expression Cassette</SPAN></H3><P>We wanted to build five variations of the pRSII 413 plasmid based on which final cassette will act as the insert. The design of our cassette included using the constitutive pTv3 promoter, and a terminator sequence. One of the five final cassettes would contain the sequence coding for the Aga1 protein, and four of the cassette inserts would contain sequences coding for Aga2 protein fused with a single chain antibody of choice.
By carefully designed consequent PCR reactions and restriction digests we aimed to obtain in vitro the 1) promoter, 2) terminator, 3) promoter and terminator, 4) promoter, ORF and terminator. Subsequent restriction digests and ligations would lead to a complete expression plasmid containing our desired inserts.
</P><H3><SPAN class="mw-headline" id="Ag.CE.B11p_version">AgÎ±1p version</SPAN></H3><P>By fusing the antibody fragment directly to the AgÎ±1 protein, we are able to eliminate the complications that arise because of the translation of several proteins in a row. The courage to pursue this alternative stemmed from the positive results obtained by J. Wasilenko and colleagues in their work of displaying the hemagglutinin of avian influenza virus on yeast cells using alpha-agglutinin. We aimed to incorporate into only one plasmid the sequence for an antibody fused with the C-terminal end of Agalpha1 (alpha-agglutinin) and a secretion signal. All of it to be controlled by one promoter <SUP id="ref_Wasilenko_et_al.2C_2010_back" class="ref-back"><A href="#ref_Wasilenko_et_al.2C_2010">[Wasilenko et al, 2010]</A></SUP>. This alternative greatly appealed to us thanks to its elegance and simplicity but proved to be quite time-consuming to prepare. Therefore, we only finished the design of the system and assigned the idea for future experiments.
</P><H2><SPAN class="mw-headline" id="Construction">Construction</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Validation of the insert sequences coding for surface antibodies cloned into individual yeast display plasmids.</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Construction_of_the_Traditional_Yeast_Display_Plasmids">Construction of the Traditional Yeast Display Plasmids</SPAN></H3><P>We used modified pCT302 plasmid (<A rel="nofollow" class="external free" href="https://www.addgene.org/41845/">https://www.addgene.org/41845/</A>) and inserts described below for the construction of our final yeast display plasmids. Strain EBY100 was used for yeast display experiments. Expression of the protein display on the yeast surface can be readily induced by addition of galactose to the growing medium.
</P><P>More information about sequences used in yeast display experiments:
</P><H4><SPAN class="mw-headline" id="EpCAM_2">EpCAM</SPAN></H4><P>EpCAM protein consists of 314 amino acids. We used the sequence encoding only the extracellular part of the EpCAM protein (729 bps long). We also added restriction sites for BamHI, NheI (yeast display) and SpeI, EcoRI (EpCAM production) endonucleases with PCRs.
</P><PRE>ATGCAAGAAGAATGTGTATGTGAAAATTATAAACTAGCTGTCAACTGTTTTGTAAATAACAATAGACAGTGTCAATGTACTTCCGTTGGCGCTCAAAACACAGTTATCTGTTCTAAACTAGCCGCGAAGTGTCTTGTTATGAAGGCAGAGATGAATGGCTCCAAACTGGGTAGGCGTGCCAAGCCTGAGGGTGCCCTACAGAACAATGATGGGCTTTACGATCCAGATTGTGATGAATCCGGTTTATTCAAAGCCAAGCAGTGTAATGGTACTTCCATGTGCTGGTGTGTAAACACCGCTGGAGTGAGGAGAACCGATAAAGACACGGAAATCACCTGTAGCGAAAGAGTCAGGACTTACTGGATCATTATCGAACTGAAGCACAAAGCTAGGGAGAAGCCTTATGACAGCAAAAGTTTGAGGACTGCCTTACAGAAAGAAATCACAACTAGGTATCAATTAGACCCGAAATTTATAACGAGTATCCTTTACGAGAATAACGTCATTACCATTGATCTAGTGCAAAATTCTTCCCAGAAGACTCAAAATGACGTTGATATTGCAGATGTTGCCTATTACTTTGAAAAAGATGTGAAAGGAGAGAGTTTGTTCCACTCAAAAAAGATGGATCTGACAGTTAACGGGGAACAGTTAGATCTTGATCCGGGTCAAACGTTAATATATTATGTTGACGAAAAAGCACCAGAATTTAGTATGCAGGGATTGAAATAA</PRE><P>The sequence was extracted from the Uniprot database and further optimized for expression in yeast (<A rel="nofollow" class="external free" href="http://www.uniprot.org/uniprot/P16422">http://www.uniprot.org/uniprot/P16422</A>).
</P><H4><SPAN class="mw-headline" id="Anti-c-Myc_.289E10.29">Anti-c-Myc (9E10)</SPAN></H4><P>This sequence encodes part of the 9E10 Anti-cMyc antibody (scFv) and is 780 bps long. It consists of sequences encoding light and heavy chains of the antibody. These sequences are connected by a Yol linker containing a Yol epitope. Restriction sites recognized by BamHI and NheI were added for easy cloning into the yeast display plasmid.
</P><PRE>ATGCAGGTGCAGCTaCAGGAGTCTGGGGGAGACTTAGTGAAGCCTGGAGGGTCCCTGAAACTCTCCTGTGCAGCCTCTGGATTCACTTTCAGTCACTATGGCATGTCTTGGGTTCGCCAGACTCCAGACAAGAGGCTGGAGTGGGTCGCAACCATTGGTAGTCGTGGTACTTACACCCACTATCCAGACAGTGTGAAGGGACGATTCACCATCTCCAGAGACAATGACAAGAACGCCCTGTACCTGCAAATGAACAGTCTGAAGTCTGAAGACACAGCCATGTATTACTGTGCAAGAAGAAGTGAATTTTATTACTACGGTAATACCTACTATTACTCTGCTATGGACTACTGGGGTCAAGGAGCCTCAGTCACCGTCTCCTCAGCCAAAACAACACCCAAGCTTGAAGAAGGTGAATTTTCAGAAGCACGCGTAGATATCGTTCTCACTCAATCTCCAGCTTTCTTGGCTGTATCTCTAGGACAGAGGGCCACCATCTCCTGtAGAGCCAGCGAAAGTGTTGATAATTATGGCTTTAGTTTTATGAACTGGTTCCAACAGAAACCAGGACAGCCACCCAAACTCCTCATCTATGCTATATCCAACCGAggttctGGGGTCCCTGCCAGGTTTAGTGGCAGTGGGTCTGGGACAGACTTCAGCCTCAACATCCATCCTGTAGAGGAGGATGATCCTGCAATGTATTTCTGTCAGCAAACTAAGGAGGTTCCGTGGACGTTCGGTGGAGGCACCAAGCTGGAAATCAAACGGGCTGTTGCTTAA</PRE><P>The sequence was extracted from the article &quot;Primary structure and functional scFv antibody expression of an antibody against the human protooncogene c-myc.&quot; and further optimized for expression in yeast (<A rel="nofollow" class="external free" href="http://www.ncbi.nlm.nih.gov/pubmed/9219032">http://www.ncbi.nlm.nih.gov/pubmed/9219032</A>).
</P><H4><SPAN class="mw-headline" id="Anti-EpCAM">Anti-EpCAM</SPAN></H4><P>This sequence encodes part of the C215 Anti-EpCAM antibody (scFv) and is 756 bps long. It consists of sequences encoding light and heavy chains of the antibody. These sequences are connected through a G4S linker. Restriction sites recognized by BamHI and NheI were added for easy cloning into the yeast display plasmid.
</P><PRE>ATGCAGGTCAAGCTtCAGCAGTCAGGGGCTGAACTGGTGAGGCCTGGGGCTTCAGTGAAGCTGTCCTGCAAGGCTTCTGGCTACACCTTCACCAACTACTGGATAAACTGGGTGAAGCAGAGGCCTGGACAAGGCCTTGAGTGGATCGGAAATATTTATCCTTCTTATATTTATACTAACTACAATCAAGAGTTCAAGGACAAGGTCACATTGACTGTAGACGAATCCTCCAGCACAGCCTACATGCAGCTCAGCAGCCCGACATCTGAGGACTCTGCGGTCTATTACTGTACAAGATCCCCTTATGGTTACGACGAGTATGGTCTGGACTACTGGGGCCAAGGCACCACGGTCACCGTCTCCTCAGGTGGAGGCGGTTCAGGCGGAGGTGGCTCTGGCGGTGGCGGATCGGACATCGAGCTCACTCAGTCTCCATCCTCCCTGACTGTGACAGCAGGAGAGAAGGTCACTATGAACTGCAAGTCCAGTCAGAGTCTGTTAAACAGTAGAAATCAAAAGAACTACTTGACCTGGTACCAGCAGAAACCAGGGCAGCCTCCTAAACTGTTGATATACTGGGCATCCACTAGGGAATCTGGGGTCCCTGATCGCTTCACAGGCAGTGGATCTGGAACAGATTTCACTCTCACCATCAGCAGTGTGCAGGCTGAAGACCTGGCAGTTTATTACTGTCAGAATGATTATGTTTATCCGCTCACGTTCGGTGCTGGGACCAAGCTGGAAATAAAACGGTAA</PRE><P>The sequence was extracted from the European Nucleotide Archive database and further optimized for expression in yeast (<A rel="nofollow" class="external free" href="http://www.ebi.ac.uk/ena/data/view/AJ564232">http://www.ebi.ac.uk/ena/data/view/AJ564232</A>).
</P><H4><SPAN class="mw-headline" id="Anti-HuA">Anti-HuA</SPAN></H4><P>This sequence encodes part of the AC-1001 Anti-HuA (Human antigen A) antibody (scFv) and is 735 bps long. It consists of sequences encoding light and heavy chains of the antibody. These sequences are connected by a G4S linker. Restriction sites recognized by BamHI and NheI were added for easy cloning into the yeast display plasmid.
</P><PRE>ATGCAAGTCCATTTAGTGGAGTCAGGTGGTGGCGTAGTTCAACCTGGAAGATCATTGACACTTTCCTGTGCTGCGTCCGGTTTTACCTTCAATTCTTATACGTTCCACTGGGTAAGACAAACCCCAGGTAAAGGTCTTGAATGGGTTGCAGTTTTAGCATACGACGGGAGTTACCAGCATTATGCAGATTCCGTGAAAGGAAGATTTACAATTTCCAGAGATAACAGCAAAAATACTTTATACTTGCAAATGAATAGTCTGAGACTTGAAGACACAGCGGTATACTTCTGTGCAAGGGGACAGACTACAGTTACTAAAATTGATGAGGACTATTGGGGGCAGGGTACATTGGTAATTGTCTCCTCCGGTGGTGGCGGTAGTGGAGGCGGTGGCTCCGGGGGAGGGGGCTCTGAGATCGTCCTTACCCAGTCTCCAGCAACGCTTAGTTTGTCTCCTGGAGAAAGAGCTACACTTTCTTGCAGAGCAAGCCAGTCTGTCTCTTCTTACTTAGCTTGGTATCAGCAAAAACCAGGTCAGGCTCCAAGGTTGCTGATATATGATGCATCTAACAGGGCGACTGGTATTCCTGCCAGGTTTTCCGGCTCCGGTTCTGGCACTGACTTCACACTTACTATTTCATCTCTAGAACCAGAAGATTTTGCCGTTTATTACTGCCAGCAGAGATCTAATTGGCCTAGGTCTTTTGGTCAGGGTACGAAGGTCGAGATCAAAAGGTAA</PRE><P>The sequence was extracted from the article &quot;Human and mouse monoclonal antibodies to blood group A substance, which are nearly identical immunochemically, use radically different primary sequences.&quot; and further optimized for expression in yeast (<A rel="nofollow" class="external free" href="http://www.ncbi.nlm.nih.gov/pubmed/7759488">http://www.ncbi.nlm.nih.gov/pubmed/7759488</A>).
</P><H4><SPAN class="mw-headline" id="Streptavidin">Streptavidin</SPAN></H4><P>The gene encoding streptavidin protein is 381 bps long. Sequences containing priming sites for M13 forward and reverse primers were added at the ends of the synthesized streptavidin so that streptavidin gene could be amplified with these commonly used primers. Cutting sites for BamHI and NheI restriction endonucleases were also added for easy cloning into the yeast display plasmid.
</P><PRE>ATGGCTGAAGCTGGTATCACCGGCACCTGGTACAACCAGCTGGGTTCCACCTTCATCGTTACCGCTGGTGCTGACGGTGCTCTGACCGGTACCTACGAATCCGCTGTTGGTAACGCTGAAAGCCGCTACGTTCTGACCGGTCGTTACGACTCCGCTCCGGCTACCGACGGTTCCGGAACCGCTCTGGGTTGGACCGTTGCTTGGAAAAACAACTACCGTAACGCTCACTCCGCTACCACCTGGTCTGGCCAGTACGTTGGTGGTGCTGAAGCTCGTATCAACACCCAGTGGTTGTTGACCTCCGGCACCACCGAAGCCAACGCGTGGAAATCCACCCTGGTTGGTCACGACACCTTCACCAAAGTTAAACCGTCCGCTGCTTCTTAA</PRE><P>Sequence was extracted from the iGEM parts database and further optimized for expression in yeast (<A rel="nofollow" class="external free" href="http://parts.igem.org/Part:BBa_K283010">http://parts.igem.org/Part:BBa_K283010</A>).
</P><H3><SPAN class="mw-headline" id="Construction_of_the_Expression_Cassette">Construction of the Expression Cassette</SPAN></H3><P>Sequence of the Aga1 protein was amplified from yeast genome using Q5 polymerase and sequences of the pTv3 promoter and CYC terminator was amplified from plasmids provided by Module 1 (working on synthetic haploids). The primers we used are disclosed in the table below.
</P><TABLE class="wikitable"><TBODY><TR><TH scope="col"> Name
</TH><TH scope="col"> Sequence
</TH></TR><TR><TH scope="row"> pTv3 down
</TH><TD> TTAACAGAATTCGGATTCTCACAATCCTGTCGGT
</TD></TR><TR><TH scope="row"> pTv3    up
</TH><TD> ATAATTCTCGAGAAGCTTGGACTTCCCACCGCCTTC
</TD></TR><TR><TH scope="row"> CYC TT down
</TH><TD> ACAAAAGTCGACTCTAGACGAGCGTCCCAAAACCT
</TD></TR><TR><TH scope="row"> CYC TT    up
</TH><TD> ATATTAGAATTCATTATTAAATGGATCCTCATGTAATTAGTTATGTCACGCT
</TD></TR><TR><TH scope="row"> Aga1 down
</TH><TD> ATATATGGATCCTTAACTGAAAATTACATTGCAAGCA
</TD></TR><TR><TH scope="row"> Aga1 up
</TH><TD> ATATTAGAATTCACCACCATGACATTATCTTTCGCTCATTTTAC
</TD></TR></TBODY></TABLE><P>The amplification of Aga1 and assembly of the promoter and the terminator in one sequence was successful as proved by gel electrophoresis and subsequent gel purification. However, even after many repeated tries to clone the premature cassette into the pRS413 and pRS415 plasmids, our efforts remained futile. We tried using different ligases, amount of electroporated cells, and even a different cloning vector. As a result, we turned our focus solely on testing and improving the first of the three approaches - the original yeast display system.
</P><H1><SPAN class="mw-headline" id="Materials_and_Methods">Materials and Methods</SPAN></H1><H2><SPAN class="mw-headline" id="Strains">Strains</SPAN></H2><P><I>E. coli</I></P><OL><LI> DH5Î±</LI></OL><P><I>S. cerevisiae</I></P><OL><LI> EBY100</LI><LI> BY4741</LI><LI> 7284 MATa</LI></OL><H2><SPAN class="mw-headline" id="Materials">Materials</SPAN></H2><OL><LI> LB-M agar plates with ampicillin</LI><LI> 1.5 ml eppendorf tubes</LI><LI> 0.5 ml PCR tubes</LI><LI> 50 ml centrifuge tubes</LI><LI> NucleoSpin Plasmid DNA, RNA, and protein purification Kit</LI><LI> NucleoSpin Gel and PCR Clean-up Kit</LI><LI> LB-M medium with chloramphenicol</LI><LI> YPD medium</LI><LI> SD-min medium + CAA</LI><LI> NaOH agarose gel and buffer</LI><LI> TAE agarose gel and buffer</LI></OL><H2><SPAN class="mw-headline" id="Methods">Methods</SPAN></H2><OL><LI> Transformation</LI><LI> Minipreparation of plasmid DNA</LI><LI> Restriction digest</LI><LI> Ligation</LI><LI> NucleoSpin gel clean-up</LI><LI> NucleoSpin plasmid DNA purification</LI><LI> Induction of protein display</LI><LI> Fluorescent immunostaining</LI></OL><P>Detailed description of all methods can be found on our <A href="/Team:Czech_Republic/Protocols" title="Team:Czech Republic/Protocols"> Protocols</A> page.
</P><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><H2><SPAN class="mw-headline" id="Validation_of_displayed_proteins_by_immunofluorescent_labeling">Validation of displayed proteins by immunofluorescent labeling</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Validation of location tags display by fluorescence microscopy.</DIV></DIV></DIV><P>Successful surface display of our proteins was confirmed by immunofluorescent staining. Yeast display plasmid enables labeling of the fused protein by anti-HA and anti-FLAG antibodies either fused directly to a fluorophore or subsequently tagged by a corresponding fluorophore-marked secondary antibody. We have stained our modified yeast strains with both anti-HA and anti-FLAG antibodies. Presence of the fused proteins on the surface can be distinguished by a visible &quot;glowing corona&quot;. Pictures were made by merging phase field images and images showing the fluorescent signal. Pictures showed that the majority of the yeast cells successfully displayed the proteins, there were however some cells which displayed the protein in lower amounts or did not display any Aga2-fused proteins at all.
</P><H2><SPAN class="mw-headline" id="Verification_of_interactions_between_EpCAM_and_Anti-EpCAM_scFv">Verification of interactions between EpCAM and Anti-EpCAM scFv</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Cell aggregation tested on glass slides.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Cell aggregation tests evaluated by light microscopy.</DIV></DIV></DIV><P>Interaction of the cancer marker EpCAM and Anti-EpCAM scFv was harder to prove. Yeast display system should be induced in 20Â°C, as yeast cells might have difficulties with assembly of some foreign proteins in higher temperatures. Lower temperatures should slow down the rate of protein assembly and secure correct folding of the expressed proteins. However, induction in these conditions led to clumping of the yeast cells expressing both the Anti-EpCAM scFv and the EpCAM protein. We therefore decided to examine the binding strength under various conditions.
</P><P>Samples extracted from both cultures were vortexed and incubated for a brief period of time. The difference between sizes and frequency of cell aggregates in the negative control (only EpCAM or Anti-EpCAM cells) and positive controls (mixed EpCAM and Anti-EpCAM displaying cells) was visible at first glance. Inspection of the cell aggregates under the microscope revealed enormous differences between these two samples. Occurrence of this phenomenon could be explained by the affinity of anti-EpCAM antibody on the surface of the first type of cells for displayed EpCAM proteins on the surface of the second type of cells. As conditions which accompanied this experiment were not optimal, these results cannot be upscaled to full-scale conclusions. They nevertheless indicate some difference in the ability of marker and its receptor to bind each other.
</P><P>This experiment should simulate the real situation in IOD-mediated CTC detection. Individual yeast cells displaying EpCAM protein represented circulating tumor cells, whereas Anti-EpCAM displaying cells represented the first degree of detection.
</P><H2><SPAN class="mw-headline" id="Yeast-induced_blood_agglutination">Yeast-induced blood agglutination</SPAN></H2><P>One of the biggest achievements of this module was successful yeast-mediated induction of blood agglutination. Yeast cells (transformed EBY100 cells) used in this experiment were displaying scFv against the Human Antigen A (HuA). Yeast cells that served as a negative control were the exactly same cells (EBY100), but without anti-HuA display plasmid. Donor of the blood had A+ blood group. Our modified yeast cells could therefore interact with the antigen A on the surface of the red blood cells. The experiment proved capability of the modified yeast cells to agglutinate blood. This reaction should be at the very end of the signal transduction pathway and formation of blood clumps in a blood sample should indicate the presence of the particular cancer cells.
</P><DIV class="left" style="width: auto"><VIDEO width="932" height="524" poster="/wiki/images/9/97/Czech_Republic_OnChipAgglutinationTitle.jpg" controls=""><SOURCE src="/wiki/images/7/75/Czech_Republic_agglutination.mp4" type="video/mp4"><SOURCE src="/wiki/images/c/c2/Czech_Republic_agglutination.ogg" type="video/ogg"><SOURCE src="/wiki/images/a/ac/Czech_Republic_agglutination.webm" type="video/webm">
Your browser does not support the video tag.
</SOURCE></SOURCE></SOURCE></VIDEO></DIV><H2><SPAN class="mw-headline" id="The_master_experiment">The master experiment</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:382px;"><DIV class="thumbcaption">Module 3 - Master experiment results show modest activation in mixed cultures with induced location tag display. Two repeated experiments were performed and compared to pure cultures. Signaling pathway activity in mixed cultures was greater than the linear combination of activities measured in the pure cultures.</DIV></DIV></DIV><P>Influence of the relative position of the yeast cells to each other on the pheromone pathway signal transduction was examined in our last experiment. Two different yeast strains were used. The first type displayed Anti-EpCAM scFvs on its surface and also produced wild-type mating pheromones after the induction with copper sulfate solution. The second type displayed EpCAM proteins on its surface and also had a reporter plasmid that could react to concentrations of wild-type mating pheromone in the environment by GFP production. In other words, fluorescence was dependent on the concentration of wild-type mating pheromone in the proximity of the reporter yeast cell.
</P><P>This experiment confirmed that the fluorescence of the reporter type yeast cells is indeed dependent on the activation with a mating pheromone. It also suggested that clumping dependent on the expression of complementary proteins helps this activation. Yeast cells with a reciprocal affinity for each other should stay close, and thus the frequency and intensity of the signal should increase. We observed this exact mechanism that was dependent on the display of complementary proteins on the yeast cell surface. Yeast cells that were unable to bind each other showed a decreased intensity of fluorescence.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><OL><LI><CITE id="ref_Zhao_et_al.2C_2001" class="ref"><A href="#ref_Zhao_et_al.2C_2001_back">â</A> Zhao, H.,Shen, Z.-M., Kahn, P. C. and Lipke, P. N. (2001). Interaction of Î±-Agglutinin and a-Agglutinin, Saccharomyces cerevisiae Sexual Cell Adhesion Molecules. Journal of Bacteriology, 183(9), 2874â2880. doi:10.1128/JB.183.9.2874-2880.2001</CITE></LI><LI><CITE id="ref_Lipke_and_Kurjan.2C_1992" class="ref"><A href="#ref_Lipke_and_Kurjan.2C_1992_back">â</A> Lipke, P. N. and Kurjan, J. (1992). Sexual agglutination in budding yeasts: structure, function, and regulation of adhesion glycoproteins. Microbiology and Molecular Biology Reviews, 56(1), 180-194.</CITE></LI><LI><CITE id="ref_Boder_and_Wittrup.2C_1997" class="ref"><A href="#ref_Boder_and_Wittrup.2C_1997_back">â</A> Boder, E. T. and Wittrup, K. D. (1997). Yeast surface display for screening combinatorial polypeptide libraries. Nature Biotechnology, 15, 553-557. doi:10.1038/nbt0697-553</CITE></LI><LI><CITE id="ref_Baird_et_al.2C_2012" class="ref"><A href="#ref_Baird_et_al.2C_2012_back">â</A> Baird, C. et al. (2012). Yeast Display scFv Antibody Library Userâs Manual. Richland, WA: Pacific Northwest National Laboratory.</CITE></LI><LI><CITE id="ref_Wasilenko_et_al.2C_2010" class="ref"><A href="#ref_Wasilenko_et_al.2C_2010_back">â</A> Wasilenko, J. L. et al. (2010). Cell Surface Display of Highly Pathogenic Avian Influenza Virus Hemagglutinin on the Surface of Pichia Pastoris Cells Using alpha-Agglutinin for Production of Oral Vaccines. Biotechnology Progress, 26(2), 542-547. doi:10.1002/btpr.343.</CITE></LI></OL></DIV><DIV id="footer"><DIV id="footer-container"><DIV id="footer-left"><H1>links out</H1><A target="_blank" href="http://www.ccy.zcu.cz">Georgiev Lab</A><A target="_blank" href="http://www.kky.zcu.cz/en">Department of Cybernetics</A><A target="_blank" href="http://www.ntis.zcu.cz/en/">NTIS Research centre</A><A target="_blank" href="https://www.fav.zcu.cz/en/">Faculty of Applied Sciences</A><A target="_blank" href="https://www.zcu.cz/en/">University of West Bohemia</A></DIV><DIV id="footer-right"><H1>contact</H1><ADDRESS><P>Faculty of Applied Sciences</P><P>Technicka 8</P><P>Pilsen 306 14</P><P>Czech Republic</P><P>georgiev@kky.zcu.cz</P></ADDRESS></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>