Team:Czech Republic/Project/Orthogonal signals and receptors
Achievements
People
Project
Software
Human Practices
Parts
Interlab
Notebook
Orthogonal signals and receptors
Contents
1 Abstract
2 Key Achievements
3 Intro / Background
4 Design
4.1 Concept
4.1.1 Receptors
4.1.2 Signals
5 Materials and methods
5.1 Used strains
5.2 Used material / chemicals
5.3 Used methods
5.4 Used software
5.5 Construction
5.5.1 Receptors
5.5.1.1 Used primers
5.5.1.2 Used sequences
5.5.2 Signals
5.5.2.1 Used sequences
5.5.2.2 Used primers
5.5.2.3 Pheromones
5.6 Validation
5.6.1 Receptors
5.6.2 Signals
6 Results
6.1 Test of orthogonality (growth arrest approach)
6.2 Test of orthogonality (fluorescence approach)
6.3 Final constructs
6.3.1 Receptors
6.3.2 Pheromones
7 References
Abstract
In order to achieve more complex behaviour of our IODs, each IOD needs to be able to communicate with others. Since the pheromone response pathway was used for signal transduction, we prepared several synthetic signals and receptors that couple to this pathway and allow our IODs to communicate without a cross-talk.
Key Achievements
Constructed a set of yeast plasmids with different mating pheromones and their receptors and contributed to the Registry with 6 BioBricks
Verified the correct coupling of the receptors to the yeast pheromone mating pathway
Verified the correct expression and secretion of the different pheromones
Showed the orthogonality of the used receptors and pheromones
Intro / Background
By sensing mating pheromones, S. cerevisiae cells can find their mating partners. We decided to exploit this mechanism for a communication between our IODs.
As was reported in [Janiak2005] , it is possible to express chimeric STE2 receptors and couple them to the existing mating pathway. However, since each strain has differences in the G-protein that interacts with the receptor, this coupling is not very efficient. Fortunately, as reported in [Lin2011] , this efficiency can be increased by a change of the STE2 C-terminus.
We decided to test five STE2 receptors from different yeast strains. One of them with two different C-termini to see the effects of this change on the signal transduction. We constructed a plasmid expressing the corresponding pheromone for each of these receptors.
Design
Concept
Receptors
The alpha-factor receptor (STE2) is produced and displayed by MATa cells. It is a 431aa-long seven transmembrane-domain-containing (TM in the figure) GPCR . Since the sequence is known, it can be easily obtained by a PCR from the genome of our Saccharomyces cerevisiae strain. For higher translation in yeast, we added consensus Kozak sequence before the start codon. Also, restriction sites were added to the ends of the fragment to allow for easy cloning.
Other STE2 receptors (STE2 receptors from different yeast strains) were obtained via gBlocks® order. In order to achieve effective coupling to the pheromone mating pathway in the Saccharomyces cerevisiae strain and proper expression, we performed several tasks.
Transmembrane domain prediction from ELM
First, after the initial protein BLAST search, codon optimization was done using an online tool from IDT . During this process, all BioBrick restriction sites were removed from the DNA sequence.
Second, for each of the STE2 receptor, its C-terminus was replaced by the C-terminus from Candida albicans or Saccharomyces cerevisiae in order to improve coupling to the pheromone mating pathway [Lin2011] . The effect of each replacement on the protein folding was checked using the ELM resource . Using this tool, we have verified that the changes in the C-termini have no effect on the transmembrane domains.
Finally, the DNA sequences were optimized according to the recommendations from the IDT ordering form to speed up the synthesis process.
Signals
Alpha-factor mating pheromone is a short peptide (typically 13aa long) secreted by alpha cells. For our project, we needed to produce a variety of these pheromones and secrete them from the cells.  Although there are some secretion tags present in the registry ( BBa_K416003 , BBa_K792002 ), there are no parts encoding pheromones available. Also, since we found evidence that the complete wild-type signal tag is needed for the proper secretion of the pheromone [Caplan1991] , we decided to obtain the whole sequence directly from the MF(ALPHA)1 locus.
MF(ALPHA)1 gene codes for four mature alpha-factors within a putative precursor of 165 amino acids. This sequence begins with a signal tag for secretion and a segment with three glycosylation sites (89aa). The second segment contains four mature alpha-factors, each preceded by spacer peptides, which contain proteolytic processing signals [Kurjan1982] .
Since we wanted the pheromone peptides to be easily changeable by Site-directed mutagenesis (SDM), we decided to preserve the signal tag with only one copy of the pheromone:
This would create a unique site for the SDM. The final production level of the pheromone is supposed to be practically the same since the used plasmid is present in the cell in several copies (2-5). Also, for higher expression in the cells, we added consensus Kozak sequence before the start codon. Restriction sites were added to the ends of the sequence for easy future cloning.
Materials and methods
Used strains
E. coli
DH5α
S. cerevisiae
7283
7284
Δbar1
6193
Used material / chemicals
LB-M agar plates with ampicillin and chloramphenicol
1.5 ml eppendorf tubes
0.5 ml PCR tubes
50 ml centrifuge conical base and rim tubes
NucleoSpin Plasmid DNA, RNA, and protein purification Kit
NucleoSpin Gel and PCR Clean-up Kit
YPD medium, SD-min medium and minimal medium with galactosidase
NaOH agarose gel and buffer
Synthetic alpha-factor
CuSO4
Used methods
Transformation
Miniprep
Restriction digest
Ligation
NucleoSpin Gel Clean-up
NucleoSpin Plasmid DNA purification
Flow cytometry
All used protocols can be found at the Protocols page.
Used software
CFlow Plus
Microsoft Excel
MATLAB®
Construction
Receptors
Receptors were constructed by a restriction/ligation using pRSII416 vector. Wild-type Ste2 from Saccharomyces cerevisiae was obtained by a PCR from the genome (the sequence can be found as YFL026W ), pGAL1 promoter was obtained from the Registry (part BBa_J63006 ) and amplified with primers with different restriction sites. Other STE2 receptors were ordered as gBlocks.
First, the plasmid pGAL1_SC_WT was constructed (details can be found in our Notebook ):
Then, the Ste2 gene from Saccharomyces cerevisiae was replaced by the different Ste2 sequences. After this cloning, we had plasmids with 6 different Ste2 receptors under the pGAL1 promoter.
In parallel, pGAL1 promoters were replaced with constitutive pADH1 promoters on all plasmids in order to simplify future experiments. The receptor sequences and their promoters were further transferred into pRSII415 plasmids (with LEU marker instead of URA). This made up for 18 different receptor plasmids in total.
Used primers
Name
Sequence
ig2-ste2p-HindIII-F
tcaatgaagcttTACAAAATGTCTGATGCGGCTCCT
ig2-ste2p-BamHI-R
tcagatggatccTCATAAATTATTATTATCTTCAGTCCAGAACT
ig2-pGAL1-XhoI-F
tacagcctcgagCGGATTAGAAGCCGCCGAG
ig2-pGAL1-HindIII-R
tcacgtaagcttCTCCTTGACGTTAAAGTATAGAGGT
Used sequences
pGAL1 amplified from BBa_J63006
tacagcctcgaCGGATTAGAAGCCGCCGAGCGGGTGACAGCCCTCCGAAGGAAGACTCTCCTCCGTGCGTCCTCGTCTTCACCGGTCGCGTTCCTGAAACGCAGATGTGCCTCGCGCCGCACTGCTCCGAACAATAAAGATTCTACAATACTAGCTTTTATGGTTATGAAGAGGAAAAATTGGCAGTAACCTGGCCCCACAAACCTTCAAATGAACGAATCAAATTAACAACCATAGGATGATAATGCGATTAGTTTTTTAGCCTTATTTCTGGGGTAATTAATCAGCGAAGCGATGATTTTTGATCTATTAACAGATATATAAATGCAAAAACTGCATAACCACTTTAACTAATACTTTCAACATTTTCGGTTTGTATTACTTCTTATTCAAATGTAATAAAAGTATCAACAAAAAATTGTTAATATACCTCTATACTTTAACGTCAAGGAGaagcttacgtga
pADH1 amplified from the genome
ctcgagCAACTTCTTTTCTTTTTTTTTCTTTTCTCTCTCCCCCGTTGTTGTCTCACCATATCCGCAATGACAAAAAAATGATGGAAGACACTAAAGGAAAAAATTAACGACAAAGACAGCACCAACAGATGTCGTTGTTCCAGAGCTGATGAGGGGTATCTCGAAGCACACGAAACTTTTTCCTTCCTTCATTCACGCACACTACTCTCTAATGAGCAACGGTATACGGCCTTCCTTCCAGTTACTTGAATTTGAAATAAAAAAAAGTTTGCTGTCTTGCTATCAAGTATAAATAGACCTGCAATTATTAATCTTTTGTTTCCTCGTCATTGTTCTCGTTCCCTTTCTTCCTTGTTTCTTTTTCTGCACAATATTTCAAGCTATACCAAGCATACAATCAACTaagctt
Saccharomyces cerevisiae (SC) STE2 with its (SC) C-terminus
tcaatgaagcttTACAAAATGTCTGATGCGGCTCCTTCATTGAGCAATCTATTTTATGATCCAACGTATAATCCTGGTCAAAGCACCATTAACTACACTTCCATATATGGGAATGGATCTACCATCACTTTCGATGAGTTGCAAGGTTTAGTTAACAGTACTGTTACTCAGGCCATTATGTTTGGTGTCAGATGTGGTGCAGCTGCTTTGACTTTGATTGTCATGTGGATGACATCGAGAAGCAGAAAAACGCCGATTTTCATTATCAACCAAGTTTCATTGTTTTTAATCATTTTGCATTCTGCACTCTATTTTAAATATTTACTGTCTAATTACTCTTCAGTGACTTACGCTCTCACCGGATTTCCTCAGTTCATCAGTAGAGGTGACGTTCATGTTTATGGTGCTACAAATATAATTCAAGTCCTTCTTGTGGCTTCTATTGAGACTTCACTGGTGTTTCAGATAAAAGTTATTTTCACAGGCGACAACTTCAAAAGGATAGGTTTGATGCTGACGTCGATATCTTTCACTTTAGGGATTGCTACAGTTACCATGTATTTTGTAAGCGCTGTTAAAGGTATGATTGTGACTTATAATGATGTTAGTGCCACCCAAGATAAATACTTCAATGCATCCACAATTTTACTTGCATCCTCAATAAACTTTATGTCATTTGTCCTGGTAGTTAAATTGATTTTAGCTATTAGATCAAGAAGATTCCTTGGTCTCAAGCAGTTCGATAGTTTCCATATTTTACTCATAATGTCATGTCAATCTTTGTTGGTTCCATCGATAATATTCATCCTCGCATACAGTTTGAAACCAAACCAGGGAACAGATGTCTTGACTACTGTTGCAACATTACTTGCTGTATTGTCTTTACCATTATCATCAATGTGGGCCACGGCTGCTAATAATGCATCCAAAACAAACACAATTACTTCAGACTTTACAACATCCACAGATAGGTTTTATCCAGGCACGCTGTCTAGCTTTCAAACTGATAGTATCAACAACGATGCTAAAAGCAGTCTCAGAAGTAGATTATATGACCTATATCCTAGAAGGAAGGAAACAACATCGGATAAACATTCGGAAAGAACTTTTGTTTCTGAGACTGCAGATGATATAGAGAAAAATCAGTTTTATCAGTTGCCCACACCTACGAGTTCAAAAAATACTAGGATAGGACCGTTTGCTGATGCAAGTTACAAAGAGGGAGAAGTTGAACCCGTCGACATGTACACTCCCGATACGGCAGCTGATGAGGAAGCCAGAAAGTTCTGGACTGAAGATAATAATAATTTATGAggatccatctga
Candida albicans (CA) STE2 with its (CA) C-terminus (gBlock)
gtaaaacgacggccagtctcgagaagcttTACAAAATGAACATAAATTCTACATTCATTCCAGACAAGCCAGGGGATATCATTATTAGTTATAGTATACCGGGATTGGACCAGCCCATTCAAATACCATTCCATAGTTTAGATTCCTTTCAAACAGACCAAGCTAAGATCGCTTTAGTTATGGGTATCACAATAGGATCTTGTTCTATGACACTTATCTTTCTAATCTCAATCATGTATAAAACGAATAAGTTGACAAACTTGAAGTTGAAACTTAAGTTAAAGTACATCTTGCAATGGATCAACCAAAAGATCTTCACGAAGAAGAGAAATGATAACAAACAGCAACAACAACAGCAACAGCAACAGATTGAGAGTTCTAGTTATAACAATACTACAACGACTACTTCTGGCTCCTATAAACTTTTCCTATTCTATTTAAATTCCCTTATCCTTTTGATTGGGATTATCAGATCAGGTTGTTACCTAAATTACAATTTGGGCCCATTGAATAGCCTGTCATTTGTTTTCACTGGTTGGTATGATGGTAGTAGCTTTATTTCTAGCGATGTTACGAACGGTTTCAAATGCATTCTATATGCCTTGGTAGAAATATCTTTAGGTTTTCAAGTTTACGTAATGTTTAAAACAAGCAACTTGAAAATATGGGGTATAATGGCCAGTTTGTTGAGTATAGGCCTAGGCCTGATCGTGGTCGCGTTTCAAATCAATCTAACGATCTTGTCTCATATAAGGTTCTCTAGGGCTATCTCTACGAATAGATCCGAAGAAGAGAGCTCATTGTCTTCTGATAGCGTGGGTTATGTAATAAATAGCATTTGGATGGATCTGCCAACGATTTTATTTTCCATCAGCATAAATATCATGACAATCTTACTAATTGGTAAGCTAATCATCGCAATTAGAACAAGGAGGTACCTGGGTTTGAAACAGTTCGACTCATTTCACATCTTGTTGATTGGCTTTAGTCAAACGTTAATAATCCCGTCCATCATATTAGTTGTCCACTATTTCTATTTGTCTCAGAACAAGGATAGCTTACTTCAGCAAATTTCCTTATTGTTAATCATATTGATGTTACCGTTATCTTCATTATGGGCACAAACAGCAAACAACACCCATAATATCAACTCTTCTCCCTCACTGTCTTTTATTTCAAGGCATCATTCTTTCGACAGTTCCAGATCCGGAGGTAGTAACACTATTGTGTCCAACGGTGGTAGTAATGGTGGTGGCGGTGGTGGTGGCAACTTCCCAGTAAGTGGTATAGACGCGCAGTTACCGCCTGACATTGAGAAGATCTTACATGAAGATAATAACTATAAACTGCTTAACAGCAATAACGAGTCAGTTAACGACGGCGATATCATTATTAATGACGAGGGGATGATCACTAAGCAGATCACGATTAAAAGAGTTTAAgtcgacggatccactagtggtcatagctgtttcctg
Candida parapsilosis (CP) STE2 with CA C-terminus (gBlock)
gtaaaacgacggccagtctcgagaagcttTACAAAATGAATAAGATAGTCTCAAAACTTTCTTCTAGTGACGTTATAGTCACAGTAACCATTCCAAACGAAGAAGACGGTACTTATGAAGTACCATTCTATGCTATTGATAATTACCATTACAGTCGTATGGAGAACGCAGTGGTGTTGGGTGCCACGATTGGTGCTTGTTCTATGTTGTTAATCATGTTAATCGGGATTCTATTCAAGAATTTTCAAAGGTTAAGGAAATCTCTATTGTTCAACATAAACTTCGCAATTTTATTGATGCTAATTCTGAGATCTGCGTGTTATATTAACTATCTAATGAATAATCTGAGCTCTATATCATTCTTTTTCACCGGTATTTTTGACGATGAGTCATTCATGAGCTCCGATGCCGCAAATGCATTCAAAGTGATCTTAGTAGCATTAATTGAAGTTTCTTTAACCTACCAAATATACGTAATGTTTAAGACCCCCATGTTGAAGTCTTGGGGTATCTTTGCTAGTGTCTTAGCAGGTGTTTTAGGACTTGCAACGTTGGCAACACAAATATATACGACTGTAATGTCTCATGTAAATTTTGTGAACGGCACAACAGGTTCACCTTCACAGGTAACAAGTGCATGGATGGATATGCCAACAATCTTATTTTCTGTTTCTATTAACGTTTTATCTATGTTTTTGGTATGTAAGTTGGGTTTGGCCATTAGAACTAGAAGGTACTTAGGTTTAAAGCAATTCGATGCCTTCCATATTTTATTCATTATGTCAACTCAGACAATGATTATTCCTTCCATAATTTTGTTCGTCCATTACTTCGACCAGAATGATTCACAGACAACCTTGGTTAACATCAGTTTATTGTTAGTGGTCATTAGTTTGCCGTTAAGCTCTTTATGGGCACAAACAGCAAACAACACCCATAATATCAACTCTTCTCCCTCACTGTCTTTTATTTCAAGGCATCATTCTTTCGACAGTTCCAGATCCGGAGGTAGTAACACTATTGTGTCCAACGGTGGTAGTAATGGTGGTGGCGGTGGTGGTGGCAACTTCCCAGTAAGTGGTATAGACGCGCAGTTACCGCCTGACATTGAGAAGATCTTACATGAAGATAATAACTATAAACTGCTTAACAGCAATAACGAGTCAGTTAACGACGGCGATATCATTATTAATGACGAGGGGATGATCACTAAGCAGATCACGATTAAAAGAGTTTAAgtcgacggatccactagtggtcatagctgtttcctg
Candida parapsilosis (CP-SC) STE2 with SC C-terminus (gBlock)
gtaaaacgacggccagtctcgagaagcttTACAAAATGAATAAGATAGTCTCAAAACTTTCTTCTAGTGACGTTATAGTCACAGTAACCATTCCAAACGAAGAAGACGGTACTTATGAAGTACCATTCTATGCTATTGATAATTACCATTACAGTCGTATGGAGAACGCAGTGGTGTTGGGTGCCACGATTGGTGCTTGTTCTATGTTGTTAATCATGTTAATCGGGATTCTATTCAAGAATTTTCAAAGGTTAAGGAAATCTCTATTGTTCAACATAAACTTCGCAATTTTATTGATGCTAATTCTGAGATCTGCGTGTTATATTAACTATCTAATGAATAATCTGAGCTCTATATCATTCTTTTTCACCGGTATTTTTGACGATGAGTCATTCATGAGCTCCGATGCCGCAAATGCATTCAAAGTGATCTTAGTAGCATTAATTGAAGTTTCTTTAACCTACCAAATATACGTAATGTTTAAGACCCCCATGTTGAAGTCTTGGGGTATCTTTGCTAGTGTCTTAGCAGGTGTTTTAGGACTTGCAACGTTGGCAACACAAATATATACGACTGTAATGTCTCATGTAAATTTTGTGAACGGCACAACAGGTTCACCTTCACAGGTAACAAGTGCATGGATGGATATGCCAACAATCTTATTTTCTGTTTCTATTAACGTTTTATCTATGTTTTTGGTATGTAAGTTGGGTTTGGCCATTAGAACTAGAAGGTACTTAGGTTTAAAGCAATTCGATGCCTTCCATATTTTATTCATTATGTCAACTCAGACAATGATTATTCCTTCCATAATTTTGTTCGTCCATTACTTCGACCAGAATGATTCACAGACAACCTTGGTTAACATCAGTTTATTGTTAGTGGTCATTAGTTTGCCGTTAAGCTCTTTATGGGCCACGGCTGCTAATAATGCATCCAAAACAAACACAATTACTTCAGACTTTACAACATCCACAGATAGGTTTTATCCAGGCACGCTGTCTAGCTTTCAAACTGATAGTATCAACAACGATGCTAAAAGCAGTCTCAGAAGTAGATTATATGACCTATATCCTAGAAGGAAGGAAACAACATCGGATAAACATTCGGAAAGAACTTTTGTTTCTGAGACTGCAGATGATATAGAGAAAAATCAGTTTTATCAGTTGCCCACACCTACGAGTTCAAAAAATACTAGGATAGGACCGTTTGCTGATGCAAGTTACAAAGAGGGAGAAGTTGAACCCGTTGACATGTACACTCCCGATACGGCAGCTGATGAGGAAGCCAGAAAGTTCTGGACTGAGGATAATAATAATTTATGAgtcgacggatccactagtggtcatagctgtttcctg
Candida tropicalis (CT) STE2 with CA C-terminus (gBlock)
gtaaaacgacggccagtctcgagaagcttTACAAAATGGACATTAATAACACGATTCAATCTTCCGGCGATATAATCATAACATATACTATTCCAGGTATAGAAGAACCTTTTGAACTACCCTTTGAAGTCTTGAACCATTTTCAATCTGAGCAGTCCAAAAACTGTTTGGTTATGGGAGTTATGATAGGTTCTTGCTCAGTTTTGCTTATCTTCCTGGTAGGGATCTTATTCAAGACTAATAAGTTCTCCACCATAGGTAAGAGTAAGAACTTGTCAAAGAACTTTCTTTTCTATTTGAATTGCTTGATAACATTCATAGGTATTATCAGAGCTGCGTGCTTTTCCAACTATCTTTTAGGCCCATTAAATTCCGCCAGTTTTGCCTTTACAGGCTGGTATAATGGTGAATCTTATGCTAGTTCTGAAGCTGCTAACGGTTTCCGTGTTATCTTGTTCGCATTAATTGAGACTTCAATGGTTTTTCAGGTTTTTGTAATGTTCAGGGGTGCTGGCATGAAAAAACTGGCATATTCCGTGACTATACTGTGTACTGCTTTGGCTTTAGTGGTTGTGGGTTTCCAAATTAATTCAGCTGTTCTTTCTCACAGAAGGTTTGTCAACACCGTTAATGAAATTGGTGACACTGGCTTGTCTTCTATTTGGCTTGATCTACCGACCATACTATTTAGCGTATCTGTCAATTTAATGTCTGTACTTTTGATTGGCAAGTTGATTATGGCAATAAAAACCAGAAGGTATTTGGGTCTAAAACAATTTGACAGCTTCCATGTTCTACTGATTTGCAGCACACAAACCCTTTTGGTGCCTAGTTTGATATTGTTCGTACATTACTTCCTATTTTTTAGAAATGCAAATGTGATGCTGATCAATATTTCCATTTTACTTATTGTTTTGATGCTGCCTTTTTCAAGTCTGTGGGCACAAACAGCAAACAACACCCATAATATCAACTCTTCTCCCTCACTGTCTTTTATTTCAAGGCATCATTCTTTCGACAGTTCCAGATCCGGAGGTAGTAACACTATTGTGTCCAACGGTGGTAGTAATGGTGGTGGCGGTGGTGGTGGCAACTTCCCAGTAAGTGGTATAGACGCGCAGTTACCGCCTGACATTGAGAAGATCTTACATGAAGATAATAACTATAAACTGCTTAACAGCAATAACGAGTCAGTTAACGACGGCGATATCATTATTAATGACGAGGGGATGATCACTAAGCAGATCACGATTAAAAGAGTTTAAgtcgacggatccactagtggtcatagctgtttcctg
Lodderomyces elongisporus (LE) STE2 with SC C-terminus (gBlock)
gtaaaacgacggccagtctcgagaagcttTACAAAATGGATGAAGCAATAAATGCCAATCTGGTCTCCGGGGATATCATTGTTTCATTCAATATACCTGGATTACCAGAGCCTGTTCAAGTGCCATTTTCAGAGTTCGACTCCTTTCATAAGGATCAATTGATTGGAGTTATTATATTAGGCGTCACTATCGGTGCTTGCTCTTTGTTGTTAATACTATTGCTTGGAATGTTGTATAAGTCACGTGAAAAGTATTGGAAGAGCTTATTGTTTATGTTGAATGTTTGTATACTTGCCGCGACAATCTTACGTTCTGGTTGCTTTTTGGATTATTATTTGTCTGATCTAGCTTCTATCAGTTATACCTTTACCGGTGTTTATAACGGCACAAGTTTTGCAAGTAGTGATGCAGCCAATGTTTTCAAGACTATTATGTTTGCACTAATAGAGACGAGTTTGACCTTTCAAGTTTATGTTATGTTCCAAGGCACCACATGGAAGAACTGGGGACATGCAGTAACCGCATTGTCCGGTTTGTTGTCAGTCGCGTCAGTAGCATTCCAGATATATACTACTATATTAAGTCACAATAACTTCAACGCAACAATTTCAGGGACTGGTACTTTAACATCTGGGGTGTGGATGGACCTGCCAACTTTACTATTTGCTGCATCAATCAATTTTATGACTATCCTACTGCTATTCAAACTTGGCATGGCTATCAGACAAAGGAGATACTTAGGTTTGAAACAGTTTGATGGCTTTCACATCCTTTTTATTATGTTTACGCAGACCTTATTTATTCCCTCTATCTTATTGGTGATCCATTATTTTTATCAAGCTATGTCTGGTCCATTCATCATAAACATGGCCCTGTTCCTTGTGGTAGCCTTCTTGCCATTGTCATCATTGTGGGCCACGGCTGCTAATAATGCATCCAAAACAAACACAATTACTTCAGACTTTACAACATCCACAGATAGGTTTTATCCAGGCACGCTGTCTAGCTTTCAAACTGATAGTATCAACAACGATGCTAAAAGCAGTCTCAGAAGTAGATTATATGACCTATATCCTAGAAGGAAGGAAACAACATCGGATAAACATTCGGAAAGAACTTTTGTTTCTGAGACTGCAGATGATATAGAGAAAAATCAGTTTTATCAGTTGCCCACACCTACGAGTTCAAAAAATACTAGGATAGGACCGTTTGCTGATGCAAGTTACAAAGAGGGAGAAGTTGAACCCGTTGACATGTACACTCCCGATACGGCAGCTGATGAGGAAGCCAGAAAGTTCTGGACTGAGGATAATAATAATTTATGAgtcgacggatccactagtggtcatagctgtttcctg
Signals
Plasmids with pheromones were constructed by a restriction/ligation using pRSII416 vector. Wild-type alpha-factor from Saccharomyces cerevisiae was obtained by a PCR from the genome (MF(ALPHA)1 locus - YPL187W ).
We could not design the reverse primer for the whole sequence (secretion tag + alpha factor) because of the repeating pheromone sequences. To solve this issue, we firstly amplified the secretion tag only. Using a Band-stab PCR , we added the actual pheromone, stop codon and a restriction site (as a part of the new reverse primer). The pCUP1 promoter was obtained from our laboratory depository.
First, the plasmid pCUP1_SC_alpha was constructed (details can be found in our Notebook ):
Subsequently, plasmids with other pheromones were constructed using site-directed mutagenesis.
Used sequences
pCUP1
ctcgagAGATCCCATTACCGACATTTGGGCGCTATACGTGCATATGTTCATGTATGTATCTGTATTTAAAACACTTTTGTATTATTTTTCCTCATATATGTGTATAGGTTTATACGGATGATTTAATTATTACTTCACCACCCTTTATTTCAGGCTGATATCTTAGCCTTGTTACTAGTTAGAAAAAGACATTTTTGCTGTCAGTCACTGTCAAGAGATTCTTTTGCTGGCATTTCTTCTAGAAGCAAAAAGAGCGATGCGTCTTTTCCGCTGAACCGTTCCAGCAAAAAAGACTACCAACGCAATATGGATTGTCAGAATCATATAAAAGAGAAGCAAATAACTCCTTGTCTTGTATCAATTGCATTATAATATCTTCTTGTTAGTGCAATATCATATAGAAGTCATCGAAATAGATATTAAGAAAAACAAACTGTAACGAATTGGGATCCgtcgac
Secretion tag + alpha factor
gtcgacTACAAAATGAGATTTCCTTCAATTTTTACTGCAGTTTTATTCGCAGCATCCTCCGCATTAGCTGCTCCAGTCAACACTACAACAGAAGATGAAACGGCACAAATTCCGGCTGAAGCTGTCATCGGTTACTTAGATTTAGAAGGGGATTTCGATGTTGCTGTTTTGCCATTTTCCAACAGCACAAATAACGGGTTATTGTTTATAAATACTACTATTGCCAGCATTGCTGCTAAAGAAGAAGGGGTATCTTTGGATAAAAGAGAGGCTGAAGCTTGGCATTGGTTGCAACTAAAACCTGGCCAACCAATGTACTAAgaattc
Used primers
Name
Sequence
ig2-AlphaP-CA-F
ggatatttcgaacctggttaagaattcGGAGCTCCAGCTTTTGTTCCC
ig2-AlphaP-CA-R
aaaatttgttaaacggaagccAGCTTCAGCCTCTCTTTTATCCAAAG
ig2-AlphaP-CP-F
ggttactatgaaccacagtaagaattcGGAGCTCCAGCTTTTGTTCCC
ig2-AlphaP-CP-R
gtaggttgtccaatgaggcttAGCTTCAGCCTCTCTTTTATCCAAAG
ig2-AlphaP-CT-F
atatggatggttttctccaaactaagaattcGGAGCTCCAGCTTTTGTTCCC
ig2-AlphaP-CT-R
ctggttaatctgaatttgaatttggcAGCTTCAGCCTCTCTTTTATCCAAAG
ig2-AlphaP-LE-F
ggaaggtttagtcctgtttaagaattcGGAGCTCCAGCTTTTGTTCCC
ig2-AlphaP-LE-R
atatcttgtccacatccaaccAGCTTCAGCCTCTCTTTTATCCAAAG
Pheromones
Name
Sequence
Saccharomyces cerevisiae
TGGCATTGGTTGCAACTAAAACCTGGCCAACCAATGTAC
Candida albicans
GGCTTCCGTTTAACAAATTTTGGATATTTCGAACCTGGT
Candida parapsilosis
AAGCCTCATTGGACAACCTACGGTTACTATGAACCACAG
Candida tropicalis
GCCAAATTCAAATTCAGATTAACCAGATATGGATGGTTTTCTCCAAAC
Lodderomyces elongisporus
GGTTGGATGTGGACAAGATATGGAAGGTTTAGTCCTGTT
Validation
During the construction, all intermediate constructs and final plasmids were immediately verified by a restriction digest and/or sequencing (details can be accessed in our Notebook ).
Function was validated by the following experiments.
Receptors
Wild-type MATa cells (7284) and alpha cells with pRSII416_pGAL1_STE2 were grown in minimal media with galactosidase overnight. At time t=0, cultures were diluted and different concentrations of the synthetic alpha pheromone were added (0, 50, 100, 150 and 200 μM). In the ideal case, alpha cells with introduced STE2 receptor should exhibit similar behavior as WT MATa cells - growth arrest in the presence of the pheromone.
Reaction of the WT MATa cells to different concentrations of the pheromone:
Reaction of the alpha cells with introduced STE2 receptor to different concentrations of the pheromone:
As can be seen, the alpha cells producing STE2 receptor for the alpha factor showed a similar behavior to MATa cells that possess this receptor naturally.
Signals
Wild-type MATa cells (7284) , cells with pRSII416_pGAL1_STE2, cells with pCUP1_SC_Alpha and wild-type alpha cells (7283) were grown overnight in SD min.
In the morning, cultures 7283, 7284 and pRSII416_pGAL1_STE2 were diluted in the same medium and incubated for 6 hours. Culture pCUP1_SC_Alpha was centrifuged (3000 g) and resuspended in SG-min (minimal medium with galactose) supplemented with 0.1 μM CuSO4.
Alpha factors for induction were obtained by centrifugation of 7283 (aF83) and pCUP1_SC_Alpha (aF) cultures at 3000 g and by removing the cell pellet. Obtained supernatant was combined with MATa and pRSII416_pGAL1_STE2 cells (20μL culture + 80μL media).
Firstly, reaction of the MATa cells in the medium with WT alpha factor and the alpha factor produced by our MATa cells transformed with pCUP1_SC_Alpha was determined:
The MATa cells without any pheromone present exhibited normal growth after 1 hour from the transfer to the new medium. Cells with a WT pheromone present were arrested in their growth as expected. The same reaction was observed for cells in the medium with pheromones produced by the MATa cells with our pCUP1_SC_Alpha plasmid, even though on a smaller scale. This could be explained by a lower production of the pheromone from the partially induced pCUP1 promoter.
Then we tested reaction of the alpha cells transformed with the pRSII416_pGAL1_STE2 plasmid in both media:
Cells in media without any pheromone exhibited normal growth after 1 hour from the transfer to the new medium. Cells in media with WT pheromone exhibited growth arrest as expected. Cells in media with pheromones produced by MATa cells thanks to our pCUP1_SC_Alpha plasmid showed only a partial arrest, which corresponds to the lower production of the pheromone (as mentioned above).
Also, MATa cells transformed with the pCUP1_SC_Alpha were examined under the microscope:
Summary: Coupling of the introduced STE2 receptor to the pheromone mating pathway, and successful production and secretion of the introduced mating pheromone to MATa cells were demonstrated.
Results
Test of orthogonality (growth arrest approach)
To test the orthogonality of the two most promising receptors (SC and CP-SC), we conducted an assay using growth arrest as the indicator of an STE2 receptor activation.
Both receptors in MATa cells (7284) and the corresponding pheromones in MATa cells (Δbar1) were incubated for 60 h in SD-min media, supplemented with proper amino acids.
After incubation, cultures with pheromones were centrifuged (3000 g) and transferred into SG-min media supplemented with AAs and 100 μM CuSO4 in order to induce the pheromone production. After 4 h, their OD was measured and cultures were centrifuged (4000 g). The supernatants with produced pheromones were diluted to the same concentration (according to the OD measured) and kept at RT during the plate preparation.
At the same time, cultures with receptors were centrifuged (3000 g) and transferred into SG-min media (300 μL). Each well thus contained 20 μL of the receptor culture and 80 μL of the media with a single pheromone.
Measurements were done every ten minutes, shaking was set to medium and the temperature was 30°C.
The measured responses to both pheromones are shown here:
As can be seen, both receptors were activated only by their corresponding pheromones. This demonstrates the orthogonality of these two receptors.
Summary: We have experimentally verified the orthogonality of the two of our receptors.
Test of orthogonality (fluorescence approach)
Samples used for the fluorescence assay. Δbar1 cells with different STE2 receptors were grown in Erlenmeyer flasks, Δbar1 cells with different pheromones were grown in falcon tubes.
Since the orthogonality tests using the growth arrest allowed us to verify the orthogonality of only two receptors, we decided to continue these assays using fluorescence as the output.
We transformed MATa cells (Δbar1) with the pRSII415_pADH1_Receptor plasmids (to produce and display different receptors on the cell) together with the pFUS1_GFP plasmid (to detect the activation of MAPK by an increase of fluorescence).
Receptor cells and the cells producing corresponding pheromones (MATa cells (Δbar1) that we used in the growth arrest assay) were incubated for 60 h in SD-min media supplemented with proper amino acids.
After the incubation, pheromone producing cells were induced with 100 μM CuSO4 to start the pheromone production. After 2 h, these cultures were centrifuged at 3000 g for 5 min and the supernatant was combined in eppendorf tubes with the receptor cells (100 μL of the receptor cells, 250 μL of the supernatant). After 2 h of incubation in 30°C, all cultures were measured using flow cytometry.
The following table presents relative fluorescence of the receptors induced by different pheromones (the higher the value, the higher was the activation).
[RFU]
SC alpha
CA alpha
CP alpha
CT alpha
LE alpha
SC R
45622
28037
26164
29616
26978
CA R
28468
128690
39478
14365
49222
CP R
33231
21858
61210
18784
13884
CT R
17141
19921
19769
54829
16639
LE R
6849
13800
12884
11659
45527
As expected, these data showed that the activation of each receptor is the highest when induced with its own pheromone. The differences in activation levels are significant in the way that they allow us to use these receptors and signals orthogonally. Also, the majority of the results for the CA receptor are in accordance with [Alby2011] .
Summary: We have characterized the response of our STE2 receptors to different mating pheromones and showed that these receptors and signals can be used orthogonally.
Final constructs
Our module has developed many plasmids throughout the summer. The main contribution is the creation of a library of different STE2 receptors and their corresponding pheromones that can be functionally expressed in Saccharomyces cerevisiae. All plasmids were verified using restriction digest, gel electrophoresis, and DNA sequencing.
To recapitulate, we present here a complete list of plasmids.
Receptors
Saccharomyces cerevisiae with its natural C-terminus
Candida albicans with its natural C-terminus
Candida parapsilosis with C-terminus from Candida albicans
Candida parapsilosis with C-terminus from Saccharomyces cerevisiae
Candida tropicalis with C-terminus from Candida albicans
Lodderomyces elongisporus with C-terminus from Saccharomyces cerevisiae
All the receptors above were put under the control of the inducible pGAL1 and constitutive pADH1 promoter. Receptors under the control of pADH1 were also put into the pRSII415 vector (to change the selectable marker from URA to LEU).
In total, 18 plasmids were constructed.
Pheromones
Plasmids with a secretion tag and pheromones from these species were constructed:
Saccharomyces cerevisiae
Candida albicans
Candida parapsilosis
Candida parapsilosis
Candida tropicalis
Lodderomyces elongisporus
All pheromones were put under the control of the inducible pCUP1 promoter and pGAL1 promoter. Pheromones with pCUP1 promoter were also put into the pRSII415 vector (to change the selectable marker from URA to LEU).
In total, 15 plasmids were constructed.
References
↑ Lin, C.-H., Choi, a., & Bennett, R. J. (2011). Defining pheromone-receptor signaling in Candida albicans and related asexual Candida species. Molecular Biology of the Cell, 22(24), 4918–4930. doi:10.1091/mbc.E11-09-0749
↑ Caplan, S., Green, R., Rocco, J., & Kurjan, J. (1991). Glycosylation and structure of the yeast MF alpha 1 alpha-factor precursor is important for efficient transport through the secretory pathway. Journal of Bacteriology, 173, 627–635. doi:10.1039/c1mb05175j
↑ Kurjan J, Herskowitz I. (1982) Structure of a yeast pheromone gene (MF alpha): a putative alpha-factor precursor contains four tandem copies of mature alpha-factor. Cell. 1982 Oct;30(3):933-43. doi:10.1016/0092-8674(82)90298-7
↑ Caplan, S., Green, R., Rocco, J., & Kurjan, J. (1991). Glycosylation and structure of the yeast MF alpha 1 alpha-factor precursor is important for efficient transport through the secretory pathway. Journal of Bacteriology, 173, 627–635. doi:10.1039/c1mb05175j
↑ Janiak, A. M., Sargsyan, H., Russo, J., Naider, F., Hauser, M., & Becker, J. M. (2005). Functional expression of the Candida albicans alpha-factor receptor in Saccharomyces cerevisiae. Fungal Genetics and Biology, 42(2005), 328–338. doi:10.1016/j.fgb.2005.01.006
↑ Alby, K., & Bennett, R. J. (2011). Interspecies pheromone signaling promotes biofilm formation and same-sex mating in Candida albicans. Proceedings of the National Academy of Sciences of the United States of America, 108(6), 2510–2515. doi:10.1073/pnas.1017234108
links out
Georgiev Lab
Department of Cybernetics NTIS Research centre Faculty of Applied Sciences University of West Bohemia
contact
Faculty of Applied Sciences
Technicka 8
Pilsen 306 14
Czech Republic
georgiev@kky.zcu.cz
