<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Czech_Republic_Project_Orthogonal_signals_and_receptors skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Czech Republic/Project/Orthogonal signals and receptors</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="panel"><DIV class="centered-list"><DIV id="panel-ul-switch"><DIV id="panel-ul-button"><UL id="panel-ul"><LI><A href="/Team:Czech_Republic/Achievements">Achievements</A></LI><LI><A href="/Team:Czech_Republic/People">People</A></LI><LI><A href="/Team:Czech_Republic/Project">Project</A></LI><LI><A href="/Team:Czech_Republic/Software">Software</A></LI><LI><A href="/Team:Czech_Republic/Practices">Human Practices</A></LI><LI><A href="/Team:Czech_Republic/Parts">Parts</A></LI><LI><A href="/Team:Czech_Republic/Interlab_study">Interlab</A></LI><LI><A href="/Team:Czech_Republic/Notebook">Notebook</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="main-div"><HEADER><H1>Orthogonal signals and receptors</H1></HEADER><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Abstract"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Key_Achievements"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Key Achievements</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Intro_.2F_Background"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Intro / Background</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Design"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Concept"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Concept</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Receptors"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">Receptors</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Signals"><SPAN class="tocnumber">4.1.2</SPAN><SPAN class="toctext">Signals</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#Materials_and_methods"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Materials and methods</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Used_strains"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Used strains</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Used_material_.2F_chemicals"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Used material / chemicals</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Used_methods"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Used methods</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Used_software"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Used software</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Construction"><SPAN class="tocnumber">5.5</SPAN><SPAN class="toctext">Construction</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Receptors_2"><SPAN class="tocnumber">5.5.1</SPAN><SPAN class="toctext">Receptors</SPAN></A></LI><LI class="toclevel-4 tocsection-15"><A href="#Used_primers"><SPAN class="tocnumber">5.5.1.1</SPAN><SPAN class="toctext">Used primers</SPAN></A></LI><LI class="toclevel-4 tocsection-16"><A href="#Used_sequences"><SPAN class="tocnumber">5.5.1.2</SPAN><SPAN class="toctext">Used sequences</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Signals_2"><SPAN class="tocnumber">5.5.2</SPAN><SPAN class="toctext">Signals</SPAN></A></LI><LI class="toclevel-4 tocsection-18"><A href="#Used_sequences_2"><SPAN class="tocnumber">5.5.2.1</SPAN><SPAN class="toctext">Used sequences</SPAN></A></LI><LI class="toclevel-4 tocsection-19"><A href="#Used_primers_2"><SPAN class="tocnumber">5.5.2.2</SPAN><SPAN class="toctext">Used primers</SPAN></A></LI><LI class="toclevel-4 tocsection-20"><A href="#Pheromones"><SPAN class="tocnumber">5.5.2.3</SPAN><SPAN class="toctext">Pheromones</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Validation"><SPAN class="tocnumber">5.6</SPAN><SPAN class="toctext">Validation</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Receptors_3"><SPAN class="tocnumber">5.6.1</SPAN><SPAN class="toctext">Receptors</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Signals_3"><SPAN class="tocnumber">5.6.2</SPAN><SPAN class="toctext">Signals</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Results"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Test_of_orthogonality_.28growth_arrest_approach.29"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Test of orthogonality (growth arrest approach)</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Test_of_orthogonality_.28fluorescence_approach.29"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Test of orthogonality (fluorescence approach)</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Final_constructs"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Final constructs</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Receptors_4"><SPAN class="tocnumber">6.3.1</SPAN><SPAN class="toctext">Receptors</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#Pheromones_2"><SPAN class="tocnumber">6.3.2</SPAN><SPAN class="toctext">Pheromones</SPAN></A></LI><LI class="toclevel-1 tocsection-30"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><DIV class="info-box"><DIV class="left"><H2><SPAN class="mw-headline" id="Abstract">Abstract</SPAN></H2><P>In order to achieve more complex behaviour of our IODs, each IOD needs to be able to communicate with others. Since the pheromone response pathway was used for signal transduction, we prepared several synthetic signals and receptors that couple to this pathway and allow our IODs to communicate without a cross-talk.
</P></DIV><DIV class="right"><H2><SPAN class="mw-headline" id="Key_Achievements">Key Achievements</SPAN></H2><UL><LI> Constructed a set of yeast plasmids with different mating pheromones and their receptors and contributed to the Registry with 6 BioBricks</LI><LI> Verified the correct coupling of the receptors to the yeast pheromone mating pathway</LI><LI> Verified the correct expression and secretion of the different pheromones</LI><LI> Showed the orthogonality of the used receptors and pheromones</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Intro_.2F_Background">Intro / Background</SPAN></H1><P>By sensing mating pheromones, <I>S. cerevisiae</I> cells can find their mating partners. We decided to exploit this mechanism for a communication between our IODs.
</P><P>As was reported in <SUP id="ref_Janiak2005_back" class="ref-back"><A href="#ref_Janiak2005">[Janiak2005]</A></SUP>, it is possible to express chimeric STE2 receptors and couple them to the existing mating pathway. However, since each strain has differences in the G-protein that interacts with the receptor, this coupling is not very efficient. Fortunately, as reported in <SUP id="ref_Lin2011_back" class="ref-back"><A href="#ref_Lin2011">[Lin2011]</A></SUP>, this efficiency can be increased by a change of the STE2 C-terminus.
</P><P>We decided to test five STE2 receptors from different yeast strains. One of them with two different C-termini to see the effects of this change on the signal transduction. We constructed a plasmid expressing the corresponding pheromone for each of these receptors.
</P><H1><SPAN class="mw-headline" id="Design">Design</SPAN></H1><H2><SPAN class="mw-headline" id="Concept">Concept</SPAN></H2><H3><SPAN class="mw-headline" id="Receptors">Receptors</SPAN></H3><P>The alpha-factor receptor (STE2) is produced and displayed by MATa cells. It is a 431aa-long seven transmembrane-domain-containing (TM in the figure) <ABBR title="G-Protein Coupled Receptor">GPCR</ABBR>. Since the sequence is known, it can be easily obtained by a PCR from the genome of our <I>Saccharomyces cerevisiae</I> strain. For higher translation in yeast, we added consensus Kozak sequence before the start codon. Also, restriction sites were added to the ends of the fragment to allow for easy cloning.
</P><P>
Other STE2 receptors (STE2 receptors from different yeast strains) were obtained via gBlocks<SUP>®</SUP> order. In order to achieve effective coupling to the pheromone mating pathway in the <I>Saccharomyces cerevisiae</I> strain and proper expression, we performed several tasks.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Transmembrane domain prediction from ELM</DIV></DIV></DIV><P>First, after the initial protein <A rel="nofollow" class="external text" href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</A> search, codon optimization was done using an <A rel="nofollow" class="external text" href="https://www.idtdna.com/CodonOpt">online tool</A> from <ABBR title="Integrated DNA Technologies">IDT</ABBR>. During this process, all BioBrick restriction sites were removed from the DNA sequence.
</P><P>Second, for each of the STE2 receptor, its C-terminus was replaced by the C-terminus from <I>Candida albicans</I> or <I>Saccharomyces cerevisiae</I> in order to improve coupling to the pheromone mating pathway <SUP id="ref_Lin2011_back" class="ref-back"><A href="#ref_Lin2011">[Lin2011]</A></SUP>. The effect of each replacement on the protein folding was checked using the <A rel="nofollow" class="external text" href="http://elm.eu.org/">ELM resource</A>. Using this tool, we have verified that the changes in the C-termini have no effect on the transmembrane domains.
</P><P>Finally, the DNA sequences were optimized according to the recommendations from the IDT ordering form to speed up the synthesis process.
</P><H3><SPAN class="mw-headline" id="Signals">Signals</SPAN></H3><P>Alpha-factor mating pheromone is a short peptide (typically 13aa long) secreted by alpha cells. For our project, we needed to produce a variety of these pheromones and secrete them from the cells. 
Although there are some secretion tags present in the registry (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K416003">BBa_K416003</A>, <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K792002">BBa_K792002</A>), there are no parts encoding pheromones available. Also, since we found evidence that the complete wild-type signal tag is needed for the proper secretion of the pheromone <SUP id="ref_Caplan1991_back" class="ref-back"><A href="#ref_Caplan1991">[Caplan1991]</A></SUP>, we decided to obtain the whole sequence directly from the MF(ALPHA)1 locus.
</P><P>MF(ALPHA)1 gene codes for four mature alpha-factors within a putative precursor of 165 amino acids. This sequence begins with a signal tag for secretion and a segment with three glycosylation sites (89aa). The second segment contains four mature alpha-factors, each preceded by spacer peptides, which contain proteolytic processing signals <SUP id="ref_Kurjan1982_back" class="ref-back"><A href="#ref_Kurjan1982">[Kurjan1982]</A></SUP>.
</P><P>
Since we wanted the pheromone peptides to be easily changeable by Site-directed mutagenesis (SDM), we decided to preserve the signal tag with only one copy of the pheromone:
</P><P>
This would create a unique site for the SDM. The final production level of the pheromone is supposed to be practically the same since the used plasmid is present in the cell in several copies (2-5). Also, for higher expression in the cells, we added consensus Kozak sequence before the start codon. Restriction sites were added to the ends of the sequence for easy future cloning.
</P><H1><SPAN class="mw-headline" id="Materials_and_methods">Materials and methods</SPAN></H1><H2><SPAN class="mw-headline" id="Used_strains">Used strains</SPAN></H2><DL><DT>E. coli</DT><DD><OL><LI> DH5α</LI></OL></DD></DL><DL><DT>S. cerevisiae</DT><DD><OL><LI> 7283</LI><LI> 7284</LI><LI> Δbar1</LI><LI> 6193</LI></OL></DD></DL><H2><SPAN class="mw-headline" id="Used_material_.2F_chemicals">Used material / chemicals</SPAN></H2><OL><LI> LB-M agar plates with ampicillin and chloramphenicol</LI><LI> 1.5 ml eppendorf tubes</LI><LI> 0.5 ml PCR tubes</LI><LI> 50 ml centrifuge conical base and rim tubes</LI><LI> NucleoSpin Plasmid DNA, RNA, and protein purification Kit</LI><LI> NucleoSpin Gel and PCR Clean-up Kit</LI><LI> YPD medium, SD-min medium and minimal medium with galactosidase</LI><LI> NaOH agarose gel and buffer</LI><LI> Synthetic alpha-factor</LI><LI> CuSO<SUB>4</SUB></LI></OL><H2><SPAN class="mw-headline" id="Used_methods">Used methods</SPAN></H2><OL><LI> Transformation</LI><LI> Miniprep</LI><LI> Restriction digest</LI><LI> Ligation</LI><LI> NucleoSpin Gel Clean-up</LI><LI> NucleoSpin Plasmid DNA purification</LI><LI> Flow cytometry</LI></OL><P>All used protocols can be found at the <A href="/Team:Czech_Republic/Protocols" title="Team:Czech Republic/Protocols"> Protocols</A> page.
</P><H2><SPAN class="mw-headline" id="Used_software">Used software</SPAN></H2><OL><LI> CFlow Plus</LI><LI> Microsoft Excel</LI><LI> MATLAB<SUP>®</SUP></LI></OL><H2><SPAN class="mw-headline" id="Construction">Construction</SPAN></H2><H3><SPAN class="mw-headline" id="Receptors_2">Receptors</SPAN></H3><P>Receptors were constructed by a restriction/ligation using pRSII416 vector. Wild-type Ste2 from <I>Saccharomyces cerevisiae</I> was obtained by a PCR from the genome (the sequence can be found as <A rel="nofollow" class="external text" href="http://www.yeastgenome.org/locus/S000001868/overview">YFL026W</A>), pGAL1 promoter was obtained from the Registry (part <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J63006">BBa_J63006</A>) and amplified with primers with different restriction sites. Other STE2 receptors were ordered as gBlocks.
</P><P>First, the plasmid pGAL1_SC_WT was constructed (details can be found in our <A href="/Team:Czech_Republic/Notebook" title="Team:Czech Republic/Notebook">Notebook</A>):
</P><P>
Then, the Ste2 gene from <I>Saccharomyces cerevisiae</I> was replaced by the different Ste2 sequences. After this cloning, we had plasmids with 6 different Ste2 receptors under the pGAL1 promoter.
</P><P>In parallel, pGAL1 promoters were replaced with constitutive pADH1 promoters on all plasmids in order to simplify future experiments. The receptor sequences and their promoters were further transferred into pRSII415 plasmids (with LEU marker instead of URA). This made up for 18 different receptor plasmids in total.
</P><H4><SPAN class="mw-headline" id="Used_primers">Used primers</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH scope="col"> Name
</TH><TH scope="col"> Sequence
</TH></TR><TR><TH scope="row"> ig2-ste2p-HindIII-F
</TH><TD> tcaatgaagcttTACAAAATGTCTGATGCGGCTCCT
</TD></TR><TR><TH scope="row"> ig2-ste2p-BamHI-R
</TH><TD> tcagatggatccTCATAAATTATTATTATCTTCAGTCCAGAACT
</TD></TR><TR><TH scope="row"> ig2-pGAL1-XhoI-F
</TH><TD> tacagcctcgagCGGATTAGAAGCCGCCGAG
</TD></TR><TR><TH scope="row"> ig2-pGAL1-HindIII-R
</TH><TD> tcacgtaagcttCTCCTTGACGTTAAAGTATAGAGGT
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Used_sequences">Used sequences</SPAN></H4><P>pGAL1 amplified from <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J63006">BBa_J63006</A></P><PRE>tacagcctcgaCGGATTAGAAGCCGCCGAGCGGGTGACAGCCCTCCGAAGGAAGACTCTCCTCCGTGCGTCCTCGTCTTCACCGGTCGCGTTCCTGAAACGCAGATGTGCCTCGCGCCGCACTGCTCCGAACAATAAAGATTCTACAATACTAGCTTTTATGGTTATGAAGAGGAAAAATTGGCAGTAACCTGGCCCCACAAACCTTCAAATGAACGAATCAAATTAACAACCATAGGATGATAATGCGATTAGTTTTTTAGCCTTATTTCTGGGGTAATTAATCAGCGAAGCGATGATTTTTGATCTATTAACAGATATATAAATGCAAAAACTGCATAACCACTTTAACTAATACTTTCAACATTTTCGGTTTGTATTACTTCTTATTCAAATGTAATAAAAGTATCAACAAAAAATTGTTAATATACCTCTATACTTTAACGTCAAGGAGaagcttacgtga</PRE><P>pADH1 amplified from the genome
</P><PRE>ctcgagCAACTTCTTTTCTTTTTTTTTCTTTTCTCTCTCCCCCGTTGTTGTCTCACCATATCCGCAATGACAAAAAAATGATGGAAGACACTAAAGGAAAAAATTAACGACAAAGACAGCACCAACAGATGTCGTTGTTCCAGAGCTGATGAGGGGTATCTCGAAGCACACGAAACTTTTTCCTTCCTTCATTCACGCACACTACTCTCTAATGAGCAACGGTATACGGCCTTCCTTCCAGTTACTTGAATTTGAAATAAAAAAAAGTTTGCTGTCTTGCTATCAAGTATAAATAGACCTGCAATTATTAATCTTTTGTTTCCTCGTCATTGTTCTCGTTCCCTTTCTTCCTTGTTTCTTTTTCTGCACAATATTTCAAGCTATACCAAGCATACAATCAACTaagctt</PRE><P><I>Saccharomyces cerevisiae</I> (SC) STE2 with its (SC) C-terminus
</P><PRE>tcaatgaagcttTACAAAATGTCTGATGCGGCTCCTTCATTGAGCAATCTATTTTATGATCCAACGTATAATCCTGGTCAAAGCACCATTAACTACACTTCCATATATGGGAATGGATCTACCATCACTTTCGATGAGTTGCAAGGTTTAGTTAACAGTACTGTTACTCAGGCCATTATGTTTGGTGTCAGATGTGGTGCAGCTGCTTTGACTTTGATTGTCATGTGGATGACATCGAGAAGCAGAAAAACGCCGATTTTCATTATCAACCAAGTTTCATTGTTTTTAATCATTTTGCATTCTGCACTCTATTTTAAATATTTACTGTCTAATTACTCTTCAGTGACTTACGCTCTCACCGGATTTCCTCAGTTCATCAGTAGAGGTGACGTTCATGTTTATGGTGCTACAAATATAATTCAAGTCCTTCTTGTGGCTTCTATTGAGACTTCACTGGTGTTTCAGATAAAAGTTATTTTCACAGGCGACAACTTCAAAAGGATAGGTTTGATGCTGACGTCGATATCTTTCACTTTAGGGATTGCTACAGTTACCATGTATTTTGTAAGCGCTGTTAAAGGTATGATTGTGACTTATAATGATGTTAGTGCCACCCAAGATAAATACTTCAATGCATCCACAATTTTACTTGCATCCTCAATAAACTTTATGTCATTTGTCCTGGTAGTTAAATTGATTTTAGCTATTAGATCAAGAAGATTCCTTGGTCTCAAGCAGTTCGATAGTTTCCATATTTTACTCATAATGTCATGTCAATCTTTGTTGGTTCCATCGATAATATTCATCCTCGCATACAGTTTGAAACCAAACCAGGGAACAGATGTCTTGACTACTGTTGCAACATTACTTGCTGTATTGTCTTTACCATTATCATCAATGTGGGCCACGGCTGCTAATAATGCATCCAAAACAAACACAATTACTTCAGACTTTACAACATCCACAGATAGGTTTTATCCAGGCACGCTGTCTAGCTTTCAAACTGATAGTATCAACAACGATGCTAAAAGCAGTCTCAGAAGTAGATTATATGACCTATATCCTAGAAGGAAGGAAACAACATCGGATAAACATTCGGAAAGAACTTTTGTTTCTGAGACTGCAGATGATATAGAGAAAAATCAGTTTTATCAGTTGCCCACACCTACGAGTTCAAAAAATACTAGGATAGGACCGTTTGCTGATGCAAGTTACAAAGAGGGAGAAGTTGAACCCGTCGACATGTACACTCCCGATACGGCAGCTGATGAGGAAGCCAGAAAGTTCTGGACTGAAGATAATAATAATTTATGAggatccatctga</PRE><P><I>Candida albicans</I> (CA) STE2 with its (CA) C-terminus (gBlock)
</P><PRE>gtaaaacgacggccagtctcgagaagcttTACAAAATGAACATAAATTCTACATTCATTCCAGACAAGCCAGGGGATATCATTATTAGTTATAGTATACCGGGATTGGACCAGCCCATTCAAATACCATTCCATAGTTTAGATTCCTTTCAAACAGACCAAGCTAAGATCGCTTTAGTTATGGGTATCACAATAGGATCTTGTTCTATGACACTTATCTTTCTAATCTCAATCATGTATAAAACGAATAAGTTGACAAACTTGAAGTTGAAACTTAAGTTAAAGTACATCTTGCAATGGATCAACCAAAAGATCTTCACGAAGAAGAGAAATGATAACAAACAGCAACAACAACAGCAACAGCAACAGATTGAGAGTTCTAGTTATAACAATACTACAACGACTACTTCTGGCTCCTATAAACTTTTCCTATTCTATTTAAATTCCCTTATCCTTTTGATTGGGATTATCAGATCAGGTTGTTACCTAAATTACAATTTGGGCCCATTGAATAGCCTGTCATTTGTTTTCACTGGTTGGTATGATGGTAGTAGCTTTATTTCTAGCGATGTTACGAACGGTTTCAAATGCATTCTATATGCCTTGGTAGAAATATCTTTAGGTTTTCAAGTTTACGTAATGTTTAAAACAAGCAACTTGAAAATATGGGGTATAATGGCCAGTTTGTTGAGTATAGGCCTAGGCCTGATCGTGGTCGCGTTTCAAATCAATCTAACGATCTTGTCTCATATAAGGTTCTCTAGGGCTATCTCTACGAATAGATCCGAAGAAGAGAGCTCATTGTCTTCTGATAGCGTGGGTTATGTAATAAATAGCATTTGGATGGATCTGCCAACGATTTTATTTTCCATCAGCATAAATATCATGACAATCTTACTAATTGGTAAGCTAATCATCGCAATTAGAACAAGGAGGTACCTGGGTTTGAAACAGTTCGACTCATTTCACATCTTGTTGATTGGCTTTAGTCAAACGTTAATAATCCCGTCCATCATATTAGTTGTCCACTATTTCTATTTGTCTCAGAACAAGGATAGCTTACTTCAGCAAATTTCCTTATTGTTAATCATATTGATGTTACCGTTATCTTCATTATGGGCACAAACAGCAAACAACACCCATAATATCAACTCTTCTCCCTCACTGTCTTTTATTTCAAGGCATCATTCTTTCGACAGTTCCAGATCCGGAGGTAGTAACACTATTGTGTCCAACGGTGGTAGTAATGGTGGTGGCGGTGGTGGTGGCAACTTCCCAGTAAGTGGTATAGACGCGCAGTTACCGCCTGACATTGAGAAGATCTTACATGAAGATAATAACTATAAACTGCTTAACAGCAATAACGAGTCAGTTAACGACGGCGATATCATTATTAATGACGAGGGGATGATCACTAAGCAGATCACGATTAAAAGAGTTTAAgtcgacggatccactagtggtcatagctgtttcctg</PRE><P><I>Candida parapsilosis</I> (CP) STE2 with CA C-terminus (gBlock)
</P><PRE>gtaaaacgacggccagtctcgagaagcttTACAAAATGAATAAGATAGTCTCAAAACTTTCTTCTAGTGACGTTATAGTCACAGTAACCATTCCAAACGAAGAAGACGGTACTTATGAAGTACCATTCTATGCTATTGATAATTACCATTACAGTCGTATGGAGAACGCAGTGGTGTTGGGTGCCACGATTGGTGCTTGTTCTATGTTGTTAATCATGTTAATCGGGATTCTATTCAAGAATTTTCAAAGGTTAAGGAAATCTCTATTGTTCAACATAAACTTCGCAATTTTATTGATGCTAATTCTGAGATCTGCGTGTTATATTAACTATCTAATGAATAATCTGAGCTCTATATCATTCTTTTTCACCGGTATTTTTGACGATGAGTCATTCATGAGCTCCGATGCCGCAAATGCATTCAAAGTGATCTTAGTAGCATTAATTGAAGTTTCTTTAACCTACCAAATATACGTAATGTTTAAGACCCCCATGTTGAAGTCTTGGGGTATCTTTGCTAGTGTCTTAGCAGGTGTTTTAGGACTTGCAACGTTGGCAACACAAATATATACGACTGTAATGTCTCATGTAAATTTTGTGAACGGCACAACAGGTTCACCTTCACAGGTAACAAGTGCATGGATGGATATGCCAACAATCTTATTTTCTGTTTCTATTAACGTTTTATCTATGTTTTTGGTATGTAAGTTGGGTTTGGCCATTAGAACTAGAAGGTACTTAGGTTTAAAGCAATTCGATGCCTTCCATATTTTATTCATTATGTCAACTCAGACAATGATTATTCCTTCCATAATTTTGTTCGTCCATTACTTCGACCAGAATGATTCACAGACAACCTTGGTTAACATCAGTTTATTGTTAGTGGTCATTAGTTTGCCGTTAAGCTCTTTATGGGCACAAACAGCAAACAACACCCATAATATCAACTCTTCTCCCTCACTGTCTTTTATTTCAAGGCATCATTCTTTCGACAGTTCCAGATCCGGAGGTAGTAACACTATTGTGTCCAACGGTGGTAGTAATGGTGGTGGCGGTGGTGGTGGCAACTTCCCAGTAAGTGGTATAGACGCGCAGTTACCGCCTGACATTGAGAAGATCTTACATGAAGATAATAACTATAAACTGCTTAACAGCAATAACGAGTCAGTTAACGACGGCGATATCATTATTAATGACGAGGGGATGATCACTAAGCAGATCACGATTAAAAGAGTTTAAgtcgacggatccactagtggtcatagctgtttcctg</PRE><P><I>Candida parapsilosis</I> (CP-SC) STE2 with SC C-terminus (gBlock)
</P><PRE>gtaaaacgacggccagtctcgagaagcttTACAAAATGAATAAGATAGTCTCAAAACTTTCTTCTAGTGACGTTATAGTCACAGTAACCATTCCAAACGAAGAAGACGGTACTTATGAAGTACCATTCTATGCTATTGATAATTACCATTACAGTCGTATGGAGAACGCAGTGGTGTTGGGTGCCACGATTGGTGCTTGTTCTATGTTGTTAATCATGTTAATCGGGATTCTATTCAAGAATTTTCAAAGGTTAAGGAAATCTCTATTGTTCAACATAAACTTCGCAATTTTATTGATGCTAATTCTGAGATCTGCGTGTTATATTAACTATCTAATGAATAATCTGAGCTCTATATCATTCTTTTTCACCGGTATTTTTGACGATGAGTCATTCATGAGCTCCGATGCCGCAAATGCATTCAAAGTGATCTTAGTAGCATTAATTGAAGTTTCTTTAACCTACCAAATATACGTAATGTTTAAGACCCCCATGTTGAAGTCTTGGGGTATCTTTGCTAGTGTCTTAGCAGGTGTTTTAGGACTTGCAACGTTGGCAACACAAATATATACGACTGTAATGTCTCATGTAAATTTTGTGAACGGCACAACAGGTTCACCTTCACAGGTAACAAGTGCATGGATGGATATGCCAACAATCTTATTTTCTGTTTCTATTAACGTTTTATCTATGTTTTTGGTATGTAAGTTGGGTTTGGCCATTAGAACTAGAAGGTACTTAGGTTTAAAGCAATTCGATGCCTTCCATATTTTATTCATTATGTCAACTCAGACAATGATTATTCCTTCCATAATTTTGTTCGTCCATTACTTCGACCAGAATGATTCACAGACAACCTTGGTTAACATCAGTTTATTGTTAGTGGTCATTAGTTTGCCGTTAAGCTCTTTATGGGCCACGGCTGCTAATAATGCATCCAAAACAAACACAATTACTTCAGACTTTACAACATCCACAGATAGGTTTTATCCAGGCACGCTGTCTAGCTTTCAAACTGATAGTATCAACAACGATGCTAAAAGCAGTCTCAGAAGTAGATTATATGACCTATATCCTAGAAGGAAGGAAACAACATCGGATAAACATTCGGAAAGAACTTTTGTTTCTGAGACTGCAGATGATATAGAGAAAAATCAGTTTTATCAGTTGCCCACACCTACGAGTTCAAAAAATACTAGGATAGGACCGTTTGCTGATGCAAGTTACAAAGAGGGAGAAGTTGAACCCGTTGACATGTACACTCCCGATACGGCAGCTGATGAGGAAGCCAGAAAGTTCTGGACTGAGGATAATAATAATTTATGAgtcgacggatccactagtggtcatagctgtttcctg</PRE><P><I>Candida tropicalis</I> (CT) STE2 with CA C-terminus (gBlock)
</P><PRE>gtaaaacgacggccagtctcgagaagcttTACAAAATGGACATTAATAACACGATTCAATCTTCCGGCGATATAATCATAACATATACTATTCCAGGTATAGAAGAACCTTTTGAACTACCCTTTGAAGTCTTGAACCATTTTCAATCTGAGCAGTCCAAAAACTGTTTGGTTATGGGAGTTATGATAGGTTCTTGCTCAGTTTTGCTTATCTTCCTGGTAGGGATCTTATTCAAGACTAATAAGTTCTCCACCATAGGTAAGAGTAAGAACTTGTCAAAGAACTTTCTTTTCTATTTGAATTGCTTGATAACATTCATAGGTATTATCAGAGCTGCGTGCTTTTCCAACTATCTTTTAGGCCCATTAAATTCCGCCAGTTTTGCCTTTACAGGCTGGTATAATGGTGAATCTTATGCTAGTTCTGAAGCTGCTAACGGTTTCCGTGTTATCTTGTTCGCATTAATTGAGACTTCAATGGTTTTTCAGGTTTTTGTAATGTTCAGGGGTGCTGGCATGAAAAAACTGGCATATTCCGTGACTATACTGTGTACTGCTTTGGCTTTAGTGGTTGTGGGTTTCCAAATTAATTCAGCTGTTCTTTCTCACAGAAGGTTTGTCAACACCGTTAATGAAATTGGTGACACTGGCTTGTCTTCTATTTGGCTTGATCTACCGACCATACTATTTAGCGTATCTGTCAATTTAATGTCTGTACTTTTGATTGGCAAGTTGATTATGGCAATAAAAACCAGAAGGTATTTGGGTCTAAAACAATTTGACAGCTTCCATGTTCTACTGATTTGCAGCACACAAACCCTTTTGGTGCCTAGTTTGATATTGTTCGTACATTACTTCCTATTTTTTAGAAATGCAAATGTGATGCTGATCAATATTTCCATTTTACTTATTGTTTTGATGCTGCCTTTTTCAAGTCTGTGGGCACAAACAGCAAACAACACCCATAATATCAACTCTTCTCCCTCACTGTCTTTTATTTCAAGGCATCATTCTTTCGACAGTTCCAGATCCGGAGGTAGTAACACTATTGTGTCCAACGGTGGTAGTAATGGTGGTGGCGGTGGTGGTGGCAACTTCCCAGTAAGTGGTATAGACGCGCAGTTACCGCCTGACATTGAGAAGATCTTACATGAAGATAATAACTATAAACTGCTTAACAGCAATAACGAGTCAGTTAACGACGGCGATATCATTATTAATGACGAGGGGATGATCACTAAGCAGATCACGATTAAAAGAGTTTAAgtcgacggatccactagtggtcatagctgtttcctg</PRE><P><I>Lodderomyces elongisporus</I> (LE) STE2 with SC C-terminus (gBlock)
</P><PRE>gtaaaacgacggccagtctcgagaagcttTACAAAATGGATGAAGCAATAAATGCCAATCTGGTCTCCGGGGATATCATTGTTTCATTCAATATACCTGGATTACCAGAGCCTGTTCAAGTGCCATTTTCAGAGTTCGACTCCTTTCATAAGGATCAATTGATTGGAGTTATTATATTAGGCGTCACTATCGGTGCTTGCTCTTTGTTGTTAATACTATTGCTTGGAATGTTGTATAAGTCACGTGAAAAGTATTGGAAGAGCTTATTGTTTATGTTGAATGTTTGTATACTTGCCGCGACAATCTTACGTTCTGGTTGCTTTTTGGATTATTATTTGTCTGATCTAGCTTCTATCAGTTATACCTTTACCGGTGTTTATAACGGCACAAGTTTTGCAAGTAGTGATGCAGCCAATGTTTTCAAGACTATTATGTTTGCACTAATAGAGACGAGTTTGACCTTTCAAGTTTATGTTATGTTCCAAGGCACCACATGGAAGAACTGGGGACATGCAGTAACCGCATTGTCCGGTTTGTTGTCAGTCGCGTCAGTAGCATTCCAGATATATACTACTATATTAAGTCACAATAACTTCAACGCAACAATTTCAGGGACTGGTACTTTAACATCTGGGGTGTGGATGGACCTGCCAACTTTACTATTTGCTGCATCAATCAATTTTATGACTATCCTACTGCTATTCAAACTTGGCATGGCTATCAGACAAAGGAGATACTTAGGTTTGAAACAGTTTGATGGCTTTCACATCCTTTTTATTATGTTTACGCAGACCTTATTTATTCCCTCTATCTTATTGGTGATCCATTATTTTTATCAAGCTATGTCTGGTCCATTCATCATAAACATGGCCCTGTTCCTTGTGGTAGCCTTCTTGCCATTGTCATCATTGTGGGCCACGGCTGCTAATAATGCATCCAAAACAAACACAATTACTTCAGACTTTACAACATCCACAGATAGGTTTTATCCAGGCACGCTGTCTAGCTTTCAAACTGATAGTATCAACAACGATGCTAAAAGCAGTCTCAGAAGTAGATTATATGACCTATATCCTAGAAGGAAGGAAACAACATCGGATAAACATTCGGAAAGAACTTTTGTTTCTGAGACTGCAGATGATATAGAGAAAAATCAGTTTTATCAGTTGCCCACACCTACGAGTTCAAAAAATACTAGGATAGGACCGTTTGCTGATGCAAGTTACAAAGAGGGAGAAGTTGAACCCGTTGACATGTACACTCCCGATACGGCAGCTGATGAGGAAGCCAGAAAGTTCTGGACTGAGGATAATAATAATTTATGAgtcgacggatccactagtggtcatagctgtttcctg</PRE><H3><SPAN class="mw-headline" id="Signals_2">Signals</SPAN></H3><P>Plasmids with pheromones were constructed by a restriction/ligation using pRSII416 vector. Wild-type alpha-factor from <I>Saccharomyces cerevisiae</I> was obtained by a PCR from the genome (MF(ALPHA)1 locus - <A rel="nofollow" class="external text" href="http://www.yeastgenome.org/locus/mf%28alpha%291/overview">YPL187W</A>).
</P><P>We could not design the reverse primer for the whole sequence (secretion tag + alpha factor) because of the repeating pheromone sequences. To solve this issue, we firstly amplified the secretion tag only. Using a <A href="/Team:Czech_Republic/Protocols#Band-stab_PCR" title="Team:Czech Republic/Protocols">Band-stab PCR</A>, we added the actual pheromone, stop codon and a restriction site (as a part of the new reverse primer). The pCUP1 promoter was obtained from our laboratory depository.
</P><P>First, the plasmid pCUP1_SC_alpha was constructed (details can be found in our <A href="/Team:Czech_Republic/Notebook" title="Team:Czech Republic/Notebook">Notebook</A>): 
</P><P>Subsequently, plasmids with other pheromones were constructed using site-directed mutagenesis.
</P><H4><SPAN class="mw-headline" id="Used_sequences_2">Used sequences</SPAN></H4><P>pCUP1
</P><PRE>ctcgagAGATCCCATTACCGACATTTGGGCGCTATACGTGCATATGTTCATGTATGTATCTGTATTTAAAACACTTTTGTATTATTTTTCCTCATATATGTGTATAGGTTTATACGGATGATTTAATTATTACTTCACCACCCTTTATTTCAGGCTGATATCTTAGCCTTGTTACTAGTTAGAAAAAGACATTTTTGCTGTCAGTCACTGTCAAGAGATTCTTTTGCTGGCATTTCTTCTAGAAGCAAAAAGAGCGATGCGTCTTTTCCGCTGAACCGTTCCAGCAAAAAAGACTACCAACGCAATATGGATTGTCAGAATCATATAAAAGAGAAGCAAATAACTCCTTGTCTTGTATCAATTGCATTATAATATCTTCTTGTTAGTGCAATATCATATAGAAGTCATCGAAATAGATATTAAGAAAAACAAACTGTAACGAATTGGGATCCgtcgac</PRE><P>Secretion tag + alpha factor
</P><PRE>gtcgacTACAAAATGAGATTTCCTTCAATTTTTACTGCAGTTTTATTCGCAGCATCCTCCGCATTAGCTGCTCCAGTCAACACTACAACAGAAGATGAAACGGCACAAATTCCGGCTGAAGCTGTCATCGGTTACTTAGATTTAGAAGGGGATTTCGATGTTGCTGTTTTGCCATTTTCCAACAGCACAAATAACGGGTTATTGTTTATAAATACTACTATTGCCAGCATTGCTGCTAAAGAAGAAGGGGTATCTTTGGATAAAAGAGAGGCTGAAGCTTGGCATTGGTTGCAACTAAAACCTGGCCAACCAATGTACTAAgaattc</PRE><H4><SPAN class="mw-headline" id="Used_primers_2">Used primers</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH scope="col"> Name
</TH><TH scope="col"> Sequence
</TH></TR><TR><TH scope="row"> ig2-AlphaP-CA-F
</TH><TD> ggatatttcgaacctggttaagaattcGGAGCTCCAGCTTTTGTTCCC
</TD></TR><TR><TH scope="row"> ig2-AlphaP-CA-R
</TH><TD> aaaatttgttaaacggaagccAGCTTCAGCCTCTCTTTTATCCAAAG
</TD></TR><TR><TH scope="row"> ig2-AlphaP-CP-F
</TH><TD> ggttactatgaaccacagtaagaattcGGAGCTCCAGCTTTTGTTCCC
</TD></TR><TR><TH scope="row"> ig2-AlphaP-CP-R
</TH><TD> gtaggttgtccaatgaggcttAGCTTCAGCCTCTCTTTTATCCAAAG
</TD></TR><TR><TH scope="row"> ig2-AlphaP-CT-F
</TH><TD> atatggatggttttctccaaactaagaattcGGAGCTCCAGCTTTTGTTCCC
</TD></TR><TR><TH scope="row"> ig2-AlphaP-CT-R
</TH><TD> ctggttaatctgaatttgaatttggcAGCTTCAGCCTCTCTTTTATCCAAAG
</TD></TR><TR><TH scope="row"> ig2-AlphaP-LE-F
</TH><TD> ggaaggtttagtcctgtttaagaattcGGAGCTCCAGCTTTTGTTCCC
</TD></TR><TR><TH scope="row"> ig2-AlphaP-LE-R
</TH><TD> atatcttgtccacatccaaccAGCTTCAGCCTCTCTTTTATCCAAAG
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Pheromones">Pheromones</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH scope="col"> Name
</TH><TH scope="col"> Sequence
</TH></TR><TR><TH scope="row"><I>Saccharomyces cerevisiae</I></TH><TD> TGGCATTGGTTGCAACTAAAACCTGGCCAACCAATGTAC
</TD></TR><TR><TH scope="row"><I>Candida albicans</I></TH><TD> GGCTTCCGTTTAACAAATTTTGGATATTTCGAACCTGGT
</TD></TR><TR><TH scope="row"><I>Candida parapsilosis</I></TH><TD> AAGCCTCATTGGACAACCTACGGTTACTATGAACCACAG
</TD></TR><TR><TH scope="row"><I>Candida tropicalis</I></TH><TD> GCCAAATTCAAATTCAGATTAACCAGATATGGATGGTTTTCTCCAAAC
</TD></TR><TR><TH scope="row"><I>Lodderomyces elongisporus</I></TH><TD> GGTTGGATGTGGACAAGATATGGAAGGTTTAGTCCTGTT
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Validation">Validation</SPAN></H2><P>During the construction, all intermediate constructs and final plasmids were immediately verified by a restriction digest and/or sequencing (details can be accessed in our <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Czech_Republic/Notebook">Notebook</A>).
</P><P>Function was validated by the following experiments.
</P><H3><SPAN class="mw-headline" id="Receptors_3">Receptors</SPAN></H3><P>Wild-type MATa cells (7284) and alpha cells with pRSII416_pGAL1_STE2 were grown in minimal media with galactosidase overnight. At time t=0, cultures were diluted and different concentrations of the synthetic alpha pheromone were added (0, 50, 100, 150 and 200 μM). In the ideal case, alpha cells with introduced STE2 receptor should exhibit similar behavior as WT MATa cells - growth arrest in the presence of the pheromone.
</P><P>Reaction of the WT MATa cells to different concentrations of the pheromone:
</P><P>Reaction of the alpha cells with introduced STE2 receptor to different concentrations of the pheromone:
</P><P>As can be seen, the alpha cells producing STE2 receptor for the alpha factor showed a similar behavior to MATa cells that possess this receptor naturally.
</P><H3><SPAN class="mw-headline" id="Signals_3">Signals</SPAN></H3><P>Wild-type MATa cells (7284) , cells with pRSII416_pGAL1_STE2, cells with pCUP1_SC_Alpha and wild-type alpha cells (7283) were grown overnight in SD min.
</P><P>In the morning, cultures 7283, 7284 and pRSII416_pGAL1_STE2 were diluted in the same medium and incubated for 6 hours. Culture pCUP1_SC_Alpha was centrifuged (3000 g) and resuspended in SG-min (minimal medium with galactose) supplemented with 0.1 μM CuSO4.
</P><P>Alpha factors for induction were obtained by centrifugation of 7283 (aF83) and pCUP1_SC_Alpha (aF) cultures at 3000 g and by removing the cell pellet. Obtained supernatant was combined with MATa and pRSII416_pGAL1_STE2 cells (20μL culture + 80μL media).
</P><P>Firstly, reaction of the MATa cells in the medium with WT alpha factor and the alpha factor produced by our MATa cells transformed with pCUP1_SC_Alpha was determined:
</P><P>The MATa cells without any pheromone present exhibited normal growth after 1 hour from the transfer to the new medium. Cells with a WT pheromone present were arrested in their growth as expected. The same reaction was observed for cells in the medium with pheromones produced by the MATa cells with our pCUP1_SC_Alpha plasmid, even though on a smaller scale. This could be explained by a lower production of the pheromone from the partially induced pCUP1 promoter. 
</P><P>Then we tested reaction of the alpha cells transformed with the pRSII416_pGAL1_STE2 plasmid in both media:
</P><P>Cells in media without any pheromone exhibited normal growth after 1 hour from the transfer to the new medium. Cells in media with WT pheromone exhibited growth arrest as expected. Cells in media with pheromones produced by MATa cells thanks to our pCUP1_SC_Alpha plasmid showed only a partial arrest, which corresponds to the lower production of the pheromone (as mentioned above).
</P><P>Also, MATa cells transformed with the pCUP1_SC_Alpha were examined under the microscope:
</P><P><B>Summary: Coupling of the introduced STE2 receptor to the pheromone mating pathway, and successful production and secretion of the introduced mating pheromone to MATa cells were demonstrated.</B></P><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><H2><SPAN class="mw-headline" id="Test_of_orthogonality_.28growth_arrest_approach.29">Test of orthogonality (growth arrest approach)</SPAN></H2><P>To test the orthogonality of the two most promising receptors (SC and CP-SC), we conducted an assay using growth arrest as the indicator of an STE2 receptor activation. 
</P><P>Both receptors in MATa cells (7284) and the corresponding pheromones in MATa cells (Δbar1) were incubated for 60 h in SD-min media, supplemented with proper amino acids.
</P><P>After incubation, cultures with pheromones were centrifuged (3000 g) and transferred into SG-min media supplemented with AAs and 100 μM CuSO4 in order to induce the pheromone production. After 4 h, their OD was measured and cultures were centrifuged (4000 g). The supernatants with produced pheromones were diluted to the same concentration (according to the OD measured) and kept at RT during the plate preparation.
</P><P>At the same time, cultures with receptors were centrifuged (3000 g) and transferred into SG-min media (300 μL). Each well thus contained 20 μL of the receptor culture and 80 μL of the media with a single pheromone.
</P><P>Measurements were done every ten minutes, shaking was set to medium and the temperature was 30°C.
</P><P>The measured responses to both pheromones are shown here:
</P><P>As can be seen, both receptors were activated only by their corresponding pheromones. This demonstrates the orthogonality of these two receptors.
</P><P><B>Summary: We have experimentally verified the orthogonality of the two of our receptors.</B></P><H2><SPAN class="mw-headline" id="Test_of_orthogonality_.28fluorescence_approach.29">Test of orthogonality (fluorescence approach)</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Samples used for the fluorescence assay. Δbar1 cells with different STE2 receptors were grown in Erlenmeyer flasks, Δbar1 cells with different pheromones were grown in falcon tubes.</DIV></DIV></DIV><P>Since the orthogonality tests using the growth arrest allowed us to verify the orthogonality of only two receptors, we decided to continue these assays using fluorescence as the output.
</P><P>We transformed MATa cells (Δbar1) with the pRSII415_pADH1_Receptor plasmids (to produce and display different receptors on the cell) together with the pFUS1_GFP plasmid (to detect the activation of MAPK by an increase of fluorescence).
</P><P>Receptor cells and the cells producing corresponding pheromones (MATa cells (Δbar1) that we used in the growth arrest assay) were incubated for 60 h in SD-min media supplemented with proper amino acids.
</P><P>After the incubation, pheromone producing cells were induced with 100 μM CuSO4 to start the pheromone production. After 2 h, these cultures were centrifuged at 3000 g for 5 min and the supernatant was combined in eppendorf tubes with the receptor cells (100 μL of the receptor cells, 250 μL of the supernatant). After 2 h of incubation in 30°C, all cultures were measured using flow cytometry.
</P><P>The following table presents relative fluorescence of the receptors induced by different pheromones (the higher the value, the higher was the activation).
</P><TABLE class="ig2fluor wikitable"><TBODY><TR><TD>[RFU]</TD><TH>SC alpha</TH><TH>CA alpha</TH><TH>CP alpha</TH><TH>CT alpha</TH><TH>LE alpha</TH></TR><TR><TH>SC R</TH><TD style="background: #F8696B;">45622</TD><TD style="background: #FCEEF1;">28037</TD><TD style="background: #FCFCFF;">26164</TD><TD style="background: #FCE2E5;">29616</TD><TD style="background: #FCF6F9;">26978</TD></TR><TR><TH>CA R</TH><TD style="background: #FCEAED;">28468</TD><TD style="background: #F8696B;">128690</TD><TD style="background: #FCDCDF;">39478</TD><TD style="background: #FCFCFF;">14365</TD><TD style="background:#FBD0D2;">49222</TD></TR><TR><TH>CP R</TH><TD style="background:#FBC0C3;">33231</TD><TD style="background:#FCE4E7;">21858</TD><TD style="background:#F8696B;">61210</TD><TD style="background:#FCEDF0;">18784</TD><TD style="background:#FCFCFF;">13884</TD></TR><TR><TH>CT R</TH><TD style="background:#FCFBFE;">17141</TD><TD style="background:#FCF0F3;">19921</TD><TD style="background:#FCF0F3;">19769</TD><TD style="background:#F8696B;">54829</TD><TD style="background:#FCFCFF;">16639</TD></TR><TR><TH>LE R</TH><TD style="background:#FCFCFF;">6849</TD><TD style="background:#FCE2E5;">13800</TD><TD style="background:#FCE6E8;">12884</TD><TD style="background:#FCEAED;">11659</TD><TD style="background:#F8696B;">45527</TD></TR></TBODY></TABLE><P>As expected, these data showed that the activation of each receptor is the highest when induced with its own pheromone. The differences in activation levels are significant in the way that they allow us to use these receptors and signals orthogonally. Also, the majority of the results for the CA receptor are in accordance with <SUP id="ref_Alby2011_back" class="ref-back"><A href="#ref_Alby2011">[Alby2011]</A></SUP>.
</P><P><B>Summary: We have characterized the response of our STE2 receptors to different mating pheromones and showed that these receptors and signals can be used orthogonally.</B></P><H2><SPAN class="mw-headline" id="Final_constructs">Final constructs</SPAN></H2><P>Our module has developed many plasmids throughout the summer. The main contribution is the creation of a library of different STE2 receptors and their corresponding pheromones that can be functionally expressed in <I>Saccharomyces cerevisiae</I>. All plasmids were verified using restriction digest, gel electrophoresis, and DNA sequencing.
</P><P>To recapitulate, we present here a complete list of plasmids.
</P><H3><SPAN class="mw-headline" id="Receptors_4">Receptors</SPAN></H3><UL><LI><I>Saccharomyces cerevisiae</I> with its natural C-terminus</LI><LI><I>Candida albicans</I> with its natural C-terminus</LI><LI><I>Candida parapsilosis</I> with C-terminus from <I>Candida albicans</I></LI><LI><I>Candida parapsilosis</I> with C-terminus from <I>Saccharomyces cerevisiae</I></LI><LI><I>Candida tropicalis</I> with C-terminus from <I>Candida albicans</I></LI><LI><I>Lodderomyces elongisporus</I> with C-terminus from <I>Saccharomyces cerevisiae</I></LI></UL><P>All the receptors above were put under the control of the inducible pGAL1 and constitutive pADH1 promoter. Receptors under the control of pADH1 were also put into the pRSII415 vector (to change the selectable marker from URA to LEU).
</P><P><B>In total, 18 plasmids were constructed.</B></P><H3><SPAN class="mw-headline" id="Pheromones_2">Pheromones</SPAN></H3><P>Plasmids with a secretion tag and pheromones from these species were constructed:
</P><UL><LI><I>Saccharomyces cerevisiae</I></LI><LI><I>Candida albicans</I></LI><LI><I>Candida parapsilosis</I></LI><LI><I>Candida parapsilosis</I></LI><LI><I>Candida tropicalis</I></LI><LI><I>Lodderomyces elongisporus</I></LI></UL><P>All pheromones were put under the control of the inducible pCUP1 promoter and pGAL1 promoter. Pheromones with pCUP1 promoter were also put into the pRSII415 vector (to change the selectable marker from URA to LEU).
</P><P><B>In total, 15 plasmids were constructed.</B></P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><OL><LI><CITE id="ref_Lin2011" class="ref"><A href="#ref_Lin2011_back">↑</A> Lin, C.-H., Choi, a., &amp; Bennett, R. J. (2011). Defining pheromone-receptor signaling in Candida albicans and related asexual Candida species. Molecular Biology of the Cell, 22(24), 4918–4930. doi:10.1091/mbc.E11-09-0749</CITE></LI><LI><CITE id="ref_Caplan1991" class="ref"><A href="#ref_Caplan1991_back">↑</A> Caplan, S., Green, R., Rocco, J., &amp; Kurjan, J. (1991). Glycosylation and structure of the yeast MF alpha 1 alpha-factor precursor is important for efficient transport through the secretory pathway. Journal of Bacteriology, 173, 627–635. doi:10.1039/c1mb05175j</CITE></LI><LI><CITE id="ref_Kurjan1982" class="ref"><A href="#ref_Kurjan1982_back">↑</A> Kurjan J, Herskowitz I. (1982) Structure of a yeast pheromone gene (MF alpha): a putative alpha-factor precursor contains four tandem copies of mature alpha-factor. Cell. 1982 Oct;30(3):933-43. doi:10.1016/0092-8674(82)90298-7</CITE></LI><LI><CITE id="ref_Caplan1991" class="ref"><A href="#ref_Caplan1991_back">↑</A> Caplan, S., Green, R., Rocco, J., &amp; Kurjan, J. (1991). Glycosylation and structure of the yeast MF alpha 1 alpha-factor precursor is important for efficient transport through the secretory pathway. Journal of Bacteriology, 173, 627–635. doi:10.1039/c1mb05175j</CITE></LI><LI><CITE id="ref_Janiak2005" class="ref"><A href="#ref_Janiak2005_back">↑</A> Janiak, A. M., Sargsyan, H., Russo, J., Naider, F., Hauser, M., &amp; Becker, J. M. (2005). Functional expression of the Candida albicans alpha-factor receptor in Saccharomyces cerevisiae. Fungal Genetics and Biology, 42(2005), 328–338. doi:10.1016/j.fgb.2005.01.006</CITE></LI><LI><CITE id="ref_Alby2011" class="ref"><A href="#ref_Alby2011_back">↑</A> Alby, K., &amp; Bennett, R. J. (2011). Interspecies pheromone signaling promotes biofilm formation and same-sex mating in Candida albicans. Proceedings of the National Academy of Sciences of the United States of America, 108(6), 2510–2515. doi:10.1073/pnas.1017234108</CITE></LI></OL></DIV><DIV id="footer"><DIV id="footer-container"><DIV id="footer-left"><H1>links out</H1><A target="_blank" href="http://www.ccy.zcu.cz">Georgiev Lab</A><A target="_blank" href="http://www.kky.zcu.cz/en">Department of Cybernetics</A><A target="_blank" href="http://www.ntis.zcu.cz/en/">NTIS Research centre</A><A target="_blank" href="https://www.fav.zcu.cz/en/">Faculty of Applied Sciences</A><A target="_blank" href="https://www.zcu.cz/en/">University of West Bohemia</A></DIV><DIV id="footer-right"><H1>contact</H1><ADDRESS><P>Faculty of Applied Sciences</P><P>Technicka 8</P><P>Pilsen 306 14</P><P>Czech Republic</P><P>georgiev@kky.zcu.cz</P></ADDRESS></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>