Team:Czech Republic/Protocols
Achievements
People
Project
Software
Human Practices
Parts
Interlab
Notebook
Protocols
Contents
1 Purification
2 Transformation (bacteria)
3 Transformation (yeast)
4 Gel electrophoresis
4.1 TBE stock 10X (1000ml)
4.2 NaOH stock 0,1M (200mL)
4.3 Electrophoresis gel (100ml) 1%
5 Miniprep
6 Restriction digest
7 DNA Ligation
8 PCR Methods
8.1 Preparation of Primers
8.2 One Taq PCR protocol (25 ul reaction)
8.3 Q5 PCR protocol
8.4 Adjustement of thermo cycler
8.5 Band-stab PCR
9 Site-directed mutagenesis
10 Gibson
10.1 5X Isothermal Reaction Buffer (3mL)
10.2 1.33X Gibson Master Mix (375 uL)
11 Genome extraction
12 Immunostaining
13 Soft lithography - PDMS molding
14 Soft lithography - Bonding PDMS-Glass
14.1 Preparation of the substrates
14.1.1 Glass slide
14.1.2 PDMS
14.2 Air plasma treatment
14.3 Permanent irreversible bonding
Purification
Nucleospin Gel and PCR clean-up
1.5ml eppendorf tube
Protocols:
Mix 1 volume of sample with 2 volumes of NTI buffer (for example 100ul of sample with 200ul of NTI buffer) or for each 100mg of agarose gel add 200ul Buffer NTI and incubate the gel-sample for 5 – 10min at 50C and vortex until the gel slice is completely dissolved.
Transfer at max 700ul of sample into the spin column and centrifuge 30s at 11,000g (for sample from gel is maybe necessary to repeat this and next step).
Discard the  flow-through.
Add 700uL of NT3 buffer, centrifuge 1min at 11,000g.
Discard the flow-through.
Centrifuge empty column 3min at 11,000g.
Discard the flow-through and place the spin column into eppendorf tube.
Put the tube in shaker pre-heated on 70 C and incubate for 3min at 70 C.
Instil 15uL of Elution buffer into the center of spin column  filter.
Incubate for 3min at room temperature or 70 C if the DNA fragments is longer than 1000 bp.
Centrifuge 1min at 11,000g.
Repeat again steps 9-11 .
Transformation (bacteria)
Electrocompetent cells
Plasmid DNA (5pg - 100ng)
Box with ice
Electroporation cuvette
1.5mL eppendorf tube
LB-agar plate with appropriate antibiotic
950μL SOC at room-temperature
Protocol:
Put electroporation cuvette, eppendorf tube, pipette tips in a freezer.
Chill DNA samples in ice.
Place LB-agar plate in 37° C incubator.
Thaw electrocompetent cells on ice in a frige for 40min.
Prepare 950μL SOC into 1000μL pipette
Pipet 1-2μL DNA sample (diluted to an appropriate amount) and add to electrocompetent cells. Swirl tip around gently in cell to 11mix DNA with cells.
If using iGEM DNA distribution kit, punch a hole through the foil cover into the corresponding well, hydrate DNA (add 10μL of ultrapure dH20 to the well, pipette up and down a few times and let sit for 5min to make sure the dried DNA is fully resuspended).
Place cells back on ice to ensure they remain cold.
Transfer cell-DNA mixture to cuvette. Try not to handle cuvette base too much so that it stays cold.
Make sure the electrodes on the cuvette are dry.
Place the cuvette into electroporator and pulse the cells with shock.
Immediately add SOC, pipet up and down to make sure all cells are in SOC.
Transfer SOC-cell mixture to chilled eppendorf tube and put it on ice for 2min to permit the cells to recover.
Transfer eppendorf tube to 37°C shaking incubator for 1-1.5hr to per- mit expression of antibiotic resistance gene.
Plate 200μL onto prewarmed LB-agar plate with appropriate antibi-otic.
Incubate overnight at 37°C.
Transformation (yeast)
overnight culture OD = 0.4 - 0.6
salmon sperm (50ul per reaction)
DMSO
YPD
10X TE
1ml 0.5M EDTA
5ml Tris-HCl pH 7.5
fill up to 50ml
autoclave
LiOAc mix
5ml 1M LioAc (5.1g LiOAc in 50ml - autoclaved)
5ml 10X TE
fill up to 50ml
filter sterizile
PEG mix
heat 25ml dd water in microwave
20g PEG
5ml 10X TE
5ml 1M LiOAc
fill up to 50ml
filter sterilie (0.45mm filter) or autoclave - avoid evaporation
Protocol:
Heat the shaker on 100°C.
Spind down 500ul of yeast culture 5min at 3000g.
Boil salmon sperm (50ul per reaction) for 5min. Put it on ice after that.
Remove supernatatant. Resuspend pellets in 500ul 10X TE buffer. Centrifuge 3min at 3000g.
Remove supernatatant. Resuspend pellets in 500ul LiOAc mix. Centrifuge 3min at 3000g.
Remove supernatatant. Resuspend pellets in 200 LiOAc mix. Add:
50ul salmon sperm
1-5ng DNA in 10-50ul
700ul PEG mix
Vortex properly and incubate 30min at room temperature.
Add 48ul DMSO. Do not vortex, invert few times.
Heat shock 15min at 42°C.
Spin down 1min at 3000g.
Remove supernatant, resuspend in 1000ul NF water. Spin down 1min at 3000g.
Remove supernatant. Resuspend in 200ul YPD.
Spread on appropriate selection plate.
Gel electrophoresis
TBE stock 10X (1000ml)
108g Trisbase
55g Boric acid
9.3g EDTA (Ethylenediaminetetraacetic acid)
dH20
Add the ingredients into 900ml of dH20, mix it properly and fill to 1000ml.
Store at room temperature.
NaOH stock 0,1M (200mL)
0.8g NaOH
Boric acid
dH20
Add the ingredients into 200ml of dH20.
Set pH on 8,5 by Boric acid.
Store at room temperature.
Electrophoresis gel (100ml) 1%
Agarose
1X TBE or 0,1M NaOH
Ethidium bromide
Protocol:
Add 0,7g agarose to 100ml 1X TBE or 10mM NaOH.
Heat in microwave for 1 min at high power.
Mix it and put it again into microwave for 1 min at high power.
Make sure you have nitril gloves.
Let it chill down for 6min and add 4 uL of ethidium bromide.
Pour the balanced plate, don't forget to put the rake on.
After the gel congeal, cover it with 1X TBE or 10mM NaOH.
For making small electrophoresis use half the amounts.
Miniprep
Miniprep NucleoSpin Plasmid
2x 1.5ml eppendorf tube
Protocol:
Overnight culture: Centrifuge 5ml at 11,000g for 30s, remove the liquid, add 250 uL A1 buffer, vortex properly and transfer it to the eppendorf tube.
Petri dish: Add 250 L A1 buffer to the eppendorf tube, scrape off and add a culture from one partition, vortex properly.
Add 250 uL A2 buffer (blue), gently invert 8 times, incubate for 5min at room temperature.
Add 300 uL A3 buffer, gently invert to the change of colour (from blue to white).
Centrifuge 7min at 11,000g.
Transfer max 750 uL of the supernatant into 2ml spin column.
Centrifuge 1min at 11,000g.
Discard the flow-through and add 600 uL A4 buffer, centrifuge 1min at 11,000g.
Discard flow-through and centrifuge 3min at 11,000g.
Put the spin column into new eppendorf tube.
Instil 25 uL DNA free water into the center of spin column filter, incubate for 3min at room temperature.
Centrifuge for 1min at 11,000g.
Repeat again steps 10-12.
Restriction digest
For a restriction digest of 500ng DNA, you need
Restriction enzymes
Corresponding NEB buffer
100X BSA
XμL DNA (500ng)
(42.5-X)μL dH2O
0.6mL tube
Protocol:
Add appropriate amount od dH2O to the tube
Vortex NEB buffer and add 5μL
Vortex BSA and add 0.5μL (no need to add BSA when using NEB CutSmart buffer)
Vortex DNA and add appropriate amount to the tube
Vortex each enzyme and add 1μL to the tube
Incubate at 37°C for 30min, then heat-inactivate at 80°C for 20min
Store at 4°C
DNA Ligation
Insert mass in ng = \(3\frac{Insert\,length\,in\,bp}{Vector\,length\,in\,bp}Vector\,mass\,in\,ng\)
10X T4 ligase buffer
0.5uL T4 ligase
Purified insert and vector (~50ng vector) DNA
(8.5 - vector and insert)uL ultrapure dH20
0.6mL tube
Protocol:
Add appropriate amount of ultrapure water to sterile 0.6mL tube.
Vortex 10X T4 ligase buffer and add 1uL to the tube.
Add appropriate amount of vortexed insert and vector DNA to the tube.
Vortex T4 ligase and add 0.5uL to the tube.
Place the tube in thermal cycler and run ligation protocol (60min incubation at 16°C, 10min at 65°C to denaturate the ligase)
Store at -20°C
PCR Methods
Preparation of Primers
Centrifuge a primer in the original tube.
For 1 nmol of primer add 10 ul of DNA free H2O into original tube and vortex properly.
Into new eppendorf tube add 24 ul of DNA free H2O, 3 ul of forward primer and 3 ul of reverse primer – work concentration
One Taq PCR protocol (25 ul reaction)
Into PCR tube add 17,875ul of dH2O
Add 5ul 5X One Taq Standard reaction buffer, 0,5ul dNTPs, 0,5ul of mix primers – work concentration, 1ul template DNA – work concentration 1 ng/ul and 0,125ul One Taq DNA polymerase.
Q5 PCR protocol
Into PCR tube add 17 ul DNA free H2O.
Add 5 ul 5X Q5 reaction buffer, 0,5 ul dNTPs, 1,25 ul of mix primers – work concentration, 1 ul template DNA – work concentration 1 ng/ul and 0,25 ul Q5 High-Fidelity DNA polymerase.
Adjustement of thermo cycler
Denaturation – 10s, 98°C
Annaeling - 30s, temperature depends on the primers
Extension – time depends on the length of PCR construct - 30s for 1000bp, 68° for Taq / 72° for Q5
Band-stab PCR
Excellent technique when you want to amplify specific band from a gel. [ More info.. ]
Prepare a complete 50μL PCR reaction without DNA template
Take a sterile pipette tip and stab the desired band of interest 2-3 times
Swirl the tip in the tube with prepared PCR reaction
Run the PCR as usual
Site-directed mutagenesis
Perform a PCR (25μL) with primers designed by the NEBaseChanger
Verify the PCR by a gel electrophoresis
Proceed with a Kinase, Ligase & DpnI (KLD) Treatment
D treatment - add 1μL of the DpnI enzyme and 3μL of CutSmart buffer to the PCR reaction and incubate for 10min at 37°C. Heat-inactivate for 20min at 80°C.
KL treatment - Into a new tube, add 12μL of H2O, 4μL of the PCR mix after the DpnI treatment, 2μL of the 10X T4 ligase reaction buffer, 1μL of PNK T4 kinase and 1μL of T4 ligase. Incubate at RT for 20mins and transform 5μL to E. coli cells.
Gibson
5X Isothermal Reaction Buffer (3mL)
0.75g 25% PEG-8000
1.5ml 1M Tris-HCl (pH 7.5)
75 uL 2M MgCl2
150 uL 1M DTT
30 uL 100mM dATP
30 uL 100mM dTTP
30 uL 100mM dCTP
30 uL 100mM dGTP
300 uL 50mM NAD
Aliquot 3 x 1mL and store at  20 C. Note: You can use 30 L of 100mM dNTP instead of dNTPs individually.
1.33X Gibson Master Mix (375 uL)
100 uL 5X Isothermal Reaction Buffer
50 uL (40U/ uL) Taq DNA ligase
2 uL (10U/ uL) T5 exonuclease
6.25 uL (2U/ uL) Phusion DNA polymerase
216.75 L udH20
Add DNA to the Master Mix in volumetric ratio 1:3 (between 10 and 100ng of each DNA fragment)
Incubate for 1 hour at 50 C
Genome extraction
200ul of overnight yeast culcure (OD=0.4)
100ul of 200mM LioAc, 1% SDS
20ul 1M LiOAc
5ul SDS 20%
75ul NF water
300ul of ethanol 96-100%
300ul of ethanol 70%
NF water
Protocol:
Heat shaker on 70°C.
spin ddown 100-200 ul of yeast culture 3min at 5000g.
Suspend cells in 100ul of 200mM LioAc, 1% SDS.
Incubate for 5min at 70°C.
Add 300ul of ethanol 96-100%, vortex, spin down 3min at 5000g.
Remove supernatant. Add 300ul of ethanol 70%, vortex, spin down 3min at 5000g.
Remove supernatant. DIssolve pellet in 100ul NF water, spin down for 15sec at 15000g.
Use 1ul of supernatant for PCR.
Immunostaining
Cells displaying choosen protein
PBS (Phosphate Buffer Saline)
Fixing solution (3% solution of paraformaldehyde in PBS)
Blocking solution (0.25% gelatin in PBS)
Primary antibody
Secondary antibody
Protocol:
Wash the cells 2x with PBS solution.
Aspirate the liquid and add fixing solution (3% paraformaldehyde in PBS).
Pellet the cells and aspirate the liquid.
Wash the cells 2x with blocking solution (0.25% gelatin in PBS)
Incubate the cells in blocking solution for 60 minutes.
Pellet the cells, aspirate the blocking solution and add the solution of primary antibody in blocking solution (used concentrations of primary antibodies will differ, generally 500x dilutions of primary antibodies work well).
Incubate the cells with primary antibodies overnight.
Pellet the cells and wash them 3x with blocking solution.
Pellet the cells, aspirate the blocking solution and add the solution of secondary antibody in blocking solution (used concentrations of secondary antibodies will differ, generally 1000x dilutions of secondary antibodies work well).
Incubate the cells with secondary antibody for 45 minutes.
Pellet the cells, aspirate the blocking solution with secondary antibodies.
Wash the cells 3x with blocking solution.
Transfer the cells on microscopy slide and cover with microscopy coverslips.
Soft lithography - PDMS molding
Mix 40g of PDMS with 4g of curing agent
Centrifuge the mixture at 3250 RPM for 3 minutes to remove the bubbles introduced during the mixing
Clean silicon master with air gun
Wrap an aluminium foil around the edges of the silicon master to create a container
Pour the PDMS mixture over the silicon master
Cure the PDMS in an oven at 80°C for 2 hours
Leave the PDMS to cool down
Detach the PDMS carefully from the silicon master
Soft lithography - Bonding PDMS-Glass
Preparation of the substrates
Glass slide
Rinse the glass slide with acetone, isopropanol, and deionized water
Dry the glass slide with air gun
Dehydrate the glass slide on a hot plate at 120°C for 30 minutes
PDMS
Slice the PDMS to individual PDMS replicas
Drill holes for microfluidic inlets and outlets
Clean the PDMS using a scotch tape
Air plasma treatment
Place the glass slide and the PDMS replicas into a plasma cleaner, contact surfaces facing upwards
Exhaust the atmosphere with a vacuum pump and wait until the pressure drops to 500 mTorr
Activate the plasma for 2.5 minutes at Hi power
Stop the plasma and open the valve to stabilize the pressure, continue immediately with the bonding phase
Permanent irreversible bonding
Place the PDMS replica on the glass slide
Place the bonded device in an oven at 80°C for 60 minutes.
Seal the inlets and outlets with a scotch tape until use
links out
Georgiev Lab
Department of Cybernetics NTIS Research centre Faculty of Applied Sciences University of West Bohemia
contact
Faculty of Applied Sciences
Technicka 8
Pilsen 306 14
Czech Republic
georgiev@kky.zcu.cz
