<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Peking_Secretion skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Peking/Secretion</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Recovery</TITLE></P><DIV id="navigation" class="navbar navbar-fixed-top"><DIV class="navbar-inner "><DIV class="container no-padding"><DIV class="nav-collapse collapse"><UL class="nav"><LI class="menu-1"><A class="colapse-menu1" href="https://2016.igem.org/Team:Peking">Home</A></LI><LI class="dropdown menu-2"><A class="dropdown-toggle" data-toggle="dropdown" href="#"> Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Peking/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Peking/Basic_Part">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Peking/Collaborations">Collaborations</A></LI></UL><LI class="dropdown menu-3"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project</A></LI><LI><A href="https://2016.igem.org/Team:Peking/Description">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Peking/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Peking/Crosslinking">Crosslinking</A></LI><LI><A href="https://2016.igem.org/Team:Peking/Uranyl-adsorption">Uranyl adsorption</A></LI><LI><A href="https://2016.igem.org/Team:Peking/Clearance">Clearance</A></LI><LI><A href="https://2016.igem.org/Team:Peking/Secretion">Secretion</A></LI><LI><A href="https://2016.igem.org/Team:Peking/Demonstrate">Final Performance</A></LI><LI class="dropdown menu-4"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Peking/Model/GelPoint"> Model of Gel Point </A></LI><LI><A href="https://2016.igem.org/Team:Peking/Model/MassDistribution"> Model of Mass Distribution</A></LI><LI><A href="https://2016.igem.org/Team:Peking/Software">Software</A></LI><LI class="dropdown menu-5"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Practices</A></LI><LI><A href="https://2016.igem.org/Team:Peking/HP/Gold">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Peking/HP/311">Field research</A></LI><LI><A href="https://2016.igem.org/Team:Peking/HP/questionnaire">Questionnaire</A></LI><LI><A href="https://2016.igem.org/Team:Peking/HP/consulting">Consulting</A></LI><LI><A href="https://2016.igem.org/Team:Peking/HP/otherHP">Education &amp; Other</A></LI><LI class="menu-6"><A class="colapse-menu1" href="https://2016.igem.org/Team:Peking/Safety">Safety</A></LI><LI class="dropdown menu-7"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Lab</A></LI><LI><A class="" href="https://2016.igem.org/Team:Peking/Team">Team</A></LI><LI><A class="" href="https://2016.igem.org/Team:Peking/Attributions">Attribution</A></LI><LI><A class="" href="https://2016.igem.org/Team:Peking/Notebook">Notebook</A></LI><LI class="menu-8"><A class="colapse-menu1" href="https://2016.igem.org/Team:Peking/Interlab">Interlab</A></LI></DIV></DIV></DIV></DIV><DIV id="page-title"><DIV class="row"><DIV class="twelve columns centered text-center"><H1>Secretion</H1></DIV></DIV></DIV><DIV id="page-content" class="row page"><DIV id="primary" class="twelve columns"><SECTION><DIV class="row"><DIV class="three columns"><DIV id="page-wrap"><DIV id="sidebar" style="color:#000000"><H4><A href="javascript:void(0);" onclick="naver('Background')">Background</A></H4><UL><LI><A href="javascript:void(0);" onclick="naver('BE')"><I>E. coli</I></A></LI><LI><A href="javascript:void(0);" onclick="naver('BB')"><I>B. subtilis</I></A></LI></UL><H4><A href="javascript:void(0);" onclick="naver('Design')">Design</A></H4><UL><LI><A href="javascript:void(0);" onclick="naver('Library')">Signal Peptide Library</A></LI><LI><A href="javascript:void(0);" onclick="naver('Assays')">Secretion Assays</A></LI><LI><A href="javascript:void(0);" onclick="naver('Construcion')">Construction</A></LI></UL><H4><A href="javascript:void(0);" onclick="naver('Results')">Results</A></H4><UL><LI><A href="javascript:void(0);" onclick="naver('RE')"><I>E. coli</I></A></LI><LI><A href="javascript:void(0);" onclick="naver('RB')"><I>B. subtilis</I></A></LI><LI><A href="javascript:void(0);" onclick="naver('Quantitative')">Quantitative Measurements</A></LI></UL><H4><A href="javascript:void(0);" onclick="naver('references')">References</A></H4></DIV></DIV></DIV><DIV class="nine columns"><P class="lead add-bottom" style="color:#5E5656">The cell lysis step needed for the extraction of intracellular protein polymer network could be quite tedious, so we wondered whether we could make the bacteria produce the protein polymer network autonomously, that is, secrete the recombinant proteins continuously without lysis. To do so, we constructed a signal peptide library and carried out a series of experiments to find an optimal secretion strategy.</P><DIV class="texttitle">Background</DIV><DIV class="col-md-8"><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading1"><A role="button" data-toggle="collapse" href="#collapse1" aria-expanded="false" aria-controls="collapse1"><I>E. coli</I></A></DIV><DIV id="collapse1" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="heading1"><DIV class="panel-body"><P class="lead add-bottom" style="color:#5E5656"><I>E. coli</I>, the classical workhorse organism of biotechnology, is a mature and versatile host for the production of various proteins. Due to the high obtainable expression levels and the ease of genetic manipulation, <I>E. coli</I> was chosen as the first secretion vector.</P><P>For <I>E. coli</I>, we chose pET28a with its intrinsic T7 promoter and RBS as the vector backbone.</P><H3 class="classic-title"> Basic Construction </H3><P>Applying PCR and the Golden Gate Assembly method, we first cloned the proteins of interest into pET28a. Subsequently, we used the resulting expression vectors to transform <I>E. coli</I> TOP10, which is an ideal cloning strain. Following sequence confirmation and vector expansion in TOP10, the vectors were transferred into <I>E. coli</I> BL21 for expression, which promised satisfactory secretion levels.</P><H3 class="classic-title"> Improved Construction </H3><P>Although <I>E. coli</I> is desirable for achieving maximal protein production within the cytoplasm, targeting the proteins to extracellular compartments may be difficult in this host, since <I>E. coli</I> has an outer membrane.</P><P>To avoid this obstacle and increase the secretion efficiency, we adopted a gene named kil. Kil is a bacteriocin release protein (BRP) which leads to a permeabilization of the outer membrane and could thus be used for the secretion of proteins of interest into the culture medium<SUP>1</SUP>.</P><P>However, high-level induction of kil results in cell lysis and death. To solve this problem, we selected a medium-strength promoter, J23117, from the iGEM PARTS library and fused it to the 5’ end of the kil coding sequence. We also included a lac operator between J23117 and kil, which resulted in a simple inducible synthetic kil expression gene. Consequently, kil was only expressed in the presence of IPTG, which ensured the survival of the bacteria under normal conditions.</P><FIGURE><FIGCAPTION style="text-align:center;">
                                                        Fig. 1. Schematic representation of a simple inducible kil expression cassette.
                                                    </FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading1"><A role="button" data-toggle="collapse" href="#collapse2" aria-expanded="false" aria-controls="collapse2"><I>B. subtilis</I></A></DIV><DIV id="collapse2" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="heading1"><DIV class="panel-body"><P class="lead add-bottom" style="color:#5E5656"><I>B. subtilis</I> is a Gram-positive bacterium and as such does not have an outer membrane, which enables it to secrete proteins directly into the medium. As a result, <I>B. subtilis</I> may offer higher product yields for secreted proteins, which makes it an attractive alternative to <I>E. coli</I>.</P><H3 class="classic-title"> Basic Construction </H3><P>We chose the shuttle plasmid pBES as vector backbone. PBES could be manipulated easily and multiplies quickly in <I>E. coli</I>, so we conducted all molecular cloning procedures in <I>E. coli</I>. Once successfully constructed, the plasmids were extracted and used to transform <I>B. subtilis</I> for subsequent expression.</P><H3 class="classic-title">  Improved Construction </H3><P>To increase the expression and secretion levels, we examined two promoters - PaprE and P43. These two promoters are widely used for heterologous protein production in <I>B. subtilis</I><SUP>2</SUP>.  By fusing either PaprE or P43 to the coding sequence, we hoped that <I>B. subtilis</I> would achieve a high secretion efficiency.</P><FIGURE><FIGCAPTION style="text-align:center;">
                                                        Fig. 2. Schematic representation of expression cassettes used in <I>B. subtilis</I></FIGCAPTION></FIGURE><P>The target proteins included 3A-SUP (triple SpyTag-SUP), 3A-mSA (triple SpyTag-mSA), 3A-RFP (triple SpyTag-SUP), 3B (triple SpyCatcher) and 2B (double SpyCatcher). Since a number of important attributes of these fusion proteins remained uncharacterized, and especially their exact folding state in the cytoplasm, we were unable to rationally choose or design a signal peptide in each individual case. We were, however, able to build a signal peptide library to simply screen for the best candidates. The ultimate goal was to select the most appropriate ones from a large number of available signal peptides. However, due to time constraints, we only chose a comparatively limited number of signal peptides to test for their potential to mediate the secretion of the target proteins.</P></DIV></DIV></DIV></DIV></DIV><DIV class="texttitle anchor">Design</DIV><DIV class="col-md-8"><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading1"><A role="button" data-toggle="collapse" href="#collapse3" aria-expanded="false" aria-controls="collapse3">  Construction of Signal Peptide Library </A></DIV><DIV id="collapse3" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="heading1"><DIV class="panel-body"><P>Based on previous studies on their secretion performance, 4 different signal peptides, derived from LTIIb, PhoA, PelB and OmpA, were chosen for <I>E. coli</I> and 7 signal peptides, derived from Epr, Bpr, LipA, YjfA, SacB, NprE and ImdA, were chosen for <I>B. subtilis</I><SUP>3</SUP>.</P><P>To test the overall system, we first chose OmpA and ImdA, from <I>E. coli</I> and <I>B. subtilis</I> respectively, to determine whether well-established signal peptides could guide the target proteins out of the cytoplasm. This was done in order to exclude that emergent features of the complex protein folding of our multi-module monomers somehow generally inhibited secretion. Applying the Golden Gate Assembly method, we cloned these two signal peptides into the pSB1C3 vector backbone, amplified the resulting construct by PCR, and constructed the corresponding expression vectors by ligating the signal peptides along with the other expression cassette elements into either pET28a or pBES.</P><P>After we had succeeded in detecting the target proteins in culture supernatants, we kept on constructing vectors with other signal peptides. To accomplish this task, we first amplified the backbones with the signal peptides we had already constructed, and subsequently combined them with other signal peptides (Fig. 3.).</P><P>Note that essentially all constructs contained modules such as 3A, His-tag and FlAsH-tag, we omitted them in the name of these constructs. From example, the construct OmpA-Histag-3A-SUP-FlAsHtag was simplified as OmpA-SUP. The same is true for other constructs.</P><FIGURE><FIGCAPTION>
                                                        Fig. 3. Schematic diagram of the design and construction procedure used for secretion vectors
                                                    </FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading1"><A role="button" data-toggle="collapse" href="#collapse4" aria-expanded="false" aria-controls="collapse4">  Secretion Assays </A></DIV><DIV id="collapse4" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="heading1"><DIV class="panel-body"><P class="lead add-bottom" style="color:#5E5656">Each of the proteins of interest has its specific biochemical properties, such as SUP’s affinity to uranyl ions or SpyTag’s irreversible binding to SpyCatcher. Unfortunately, none of these intrinsic properties could be used to directly assay for protein secretion. Although SDS-PAGE could be used as a universal method to separate proteins with different molecular weights, it is not specific enough to verify the results of secretion experiments. Thus, in order to accurately and efficiently assay for protein secretion, we modified the proteins of interest with two effective molecular labels — a His-tag and a FlAsH-tag.</P><H3 class="classic-title"> His-tag </H3><P>The His-tag is an artificial amino acid motif that contains at least six histidine residues and is widely used for recombinant protein purification because of its affinity for resin-immobilized bivalent nickel or cobalt ions<SUP>3</SUP>.</P><H3 class="classic-title">FlAsH-tag</H3><P>A FlAsH-tag is a tetracysteine-motif-tag (-FLNCCPGCCMEP-) which binds with high affinity and specificity to the biarsenical dye FlAsH-EDT2 and thereby forms a fluorescent complex<SUP>4</SUP>.</P><FIGURE><FIGCAPTION>
                                                        Fig. 4. Schematic representation of the FlAsH-based genetic screen for protein secretion<SUP>4</SUP>.
                                                    </FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading1"><A role="button" data-toggle="collapse" href="#collapse5" aria-expanded="false" aria-controls="collapse5">  Construction </A></DIV><DIV id="collapse5" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="heading1"><DIV class="panel-body"><P class="lead add-bottom" style="color:#5E5656">We added the His-tag sequence between the signal peptide and the coding sequence of the protein of interest, so that it would be exposed at the N-terminus of the secreted protein after the signal peptide was removed during the Sec-pathway secretion process. A His-tag-reactive antibody was commercially available, enabling us to conduct further assays using western-blot analysis.</P><P class="lead add-bottom" style="color:#5E5656">We additionally fused the FlAsH-tag to the C-terminus of the protein of interest. Because the outer membrane of <I>E. coli</I> is naturally impermeable to FlAsH-EDT2<SUP>4</SUP>, undesired off-target labeling of intracellular proteins is avoided, rendering the assay highly selective for secreted proteins.</P><FIGURE><FIGCAPTION style="text-align:center;">
                                                        Fig. 5. Schematic representation of the final construct used for the assay for protein secretion
                                                    </FIGCAPTION></FIGURE><P>A scheme illustrating all the components used in the library design is shown in Fig. 6.</P><FIGURE><FIGCAPTION>
                                                        Fig. 6. Blowup schematic of all the elements used in the combinatorial approach for the construction of protein secretion signal libraries.
                                                    </FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV></DIV><DIV class="texttitle">Results</DIV><P class="lead add-bottom" style="color:#5E5656">We used pET28a and pBES as the backbones to express recombinant proteins in <I>E. coli</I> and <I>B. subtilis</I>, respectively. To simplify the cloning procedures, we first introduced eleven different signal peptides individually into the cloning vector pSB1C3 for further use. Based on DNA sequences from work reported by others, we codon-optimized and ordered synthetic genes encoding 3A-SUP, 3B, 3A-mSA and 3A-mRFP for expression in <I>B. subtilis</I>, while the genes for expression in <I>E. coli</I> were directly obtained as kind gifts from other labs. After this, we constructed vectors encoding all combinations of the four recombinant proteins with the total of eleven signal peptides used either in <I>E. coli</I> or <I>B. subtilis</I>. All the recombinant proteins were fused to a His-tag as well as a FlAsH-tag for the subsequent secretion assays.</P><DIV class="col-md-8"><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading1"><A role="button" data-toggle="collapse" href="#collapse6" aria-expanded="false" aria-controls="collapse6"><I>E. coli</I></A></DIV><DIV id="collapse6" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="heading1"><DIV class="panel-body"><P>We expressed the recombinant proteins in <I>E. coli</I> strain BL21. The bacteria were cultured in M9 minimum medium for 14 hours. We subsequently separated the cytoplasmic and periplasmic components from the culture supernatants and examined them for the presence of recombinant proteins using western blot analysis. An overview of the results is shown in Table. 1. </P><FIGURE><FIGCAPTION style="text-align:center;">
                                                        Table. 1. Secretion efficiency of recombinant proteins with different signal peptides
                                                    </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="text-align:center;">
                                                       Fig.7. Western blot results of recombinant proteins  (A)Secretion of spycatcher(3B).     (B)(D)Secretion of 3A-SUP.  (C)Secretion of 3A-mSA.  (D)Secretion of 3A-mRFP.  Control groups are proteins without signal peptides.  M: Medium components  C: Cytoplasmic components  P: Periplasmic components  IC: Insoluble cytoplasmic components  SC: Soluble cytoplasmic components.
                                                    </FIGCAPTION></FIGURE><P>The results show that all four signal peptides possess the ability to mediate secretion, but the efficiency varies. For some combinations, such as LTIIb-mSA, the secreted portion was larger than the portion remaining in the cytoplasm, while for others, such as PelB-3B, the opposite was true. This shows that the interaction between signal peptides and the proteins of interest significantly affects the secretion efficiency, as expected. Moreover, the total expression levels of the recombinant proteins are also different from one signal peptide to another. For example, proteins with the signal derived from OmpA on their N-terminus consistently showed higher expression levels than their respective combinations with any of the other three signal peptides. The reason for this may be due to the interaction between the RBS from the vector and the coding sequences of the signal peptides. It’s worth mentioning that cytoplasmic precursors of the proteins always have bigger molecular weights than their respective secretion products in the culture supernatants. This is because the signal peptides were removed during the secretion process.</P><P>It should be noted that overall efficiency decreased considerably when the proteins of interest were co-expressed with the Kil protein, most likely because Kil directly reduced the viability of the host cells. Therefore, the strength of the promoter controlling kil expression must be finely tuned to optimize the production and release of secretory proteins. Additional experiments are needed to supplement these findings.</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading1"><A role="button" data-toggle="collapse" href="#collapse7" aria-expanded="false" aria-controls="collapse7"><I>B. subtilis</I></A></DIV><DIV id="collapse7" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="heading1"><DIV class="panel-body"><P>We expressed the combined fusion proteins in <I>B. subtilis</I> strain RiK1285, which lacks the proteases AprE and NprE, that might otherwise degrade secreted proteins of interest. We first examined the secretion efficiency of constructs containing either 3A-SUP or 2B, together with one of the signal peptides derived from YjfA, NprE, LipA or SacB. The bacteria were cultured in M9 medium at 37°C for 14 hours before examination.</P><P>Unfortunately, none of our target proteins were detectable in the culture supernatants from M9 medium. We subsequently changed the culture medium to 2xYT and prolonged the culture time to 48 hours. After SDS-PAGE analysis, the protein band patterns of strains carrying 3A-SUP and 2B constructs were similar, so we concluded that the recombinant proteins were not expressed at all. Further western blot analysis confirmed this assumption. We concluded that there might be a cryptic problem related to the general <I>B. subtilis</I> plasmid design, and we consequently decided to stop the examination of other constructs for <I>B. subtilis</I>.</P><P>We instead attempted to at least express the recombinant proteins in the cytoplasm, and exchanged the PaprE promoter for P43, which is more efficient [2]. Following the same procedures as described above, we still could not detect protein expression through either SDS-PAGE or western blot analysis (data not shown).</P><P>There are two possible reasons accounting for this failure. Firstly, the proteins might have failed to fold properly, rendering them insoluble. Secondly, as mentioned above, the proteins may also have not been expressed at all. Since the two promoters we used are known to be strong promoters in this host according to earlier studies, the problem may be due to RBS design or an as of yet unknown problem with our <I>B. subtilis</I> strains and protocols.</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="heading1"><A role="button" data-toggle="collapse" href="#collapse8" aria-expanded="false" aria-controls="collapse8">Quantitative Measurements of Secretion Efficiency</A></DIV><DIV id="collapse8" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="heading1"><DIV class="panel-body"><P>We intended to quantitatively measure the secretion of recombinant proteins using the FlAsH-tetracysteine assay. The FlAsH-EDT2 molecule only shows strong fluorescence when bound to a tetracysteine motif, so that the intensity of fluorescence in the samples could be expected to be directly proportional to the amount of recombinant protein bearing a tetracysteine-tag.</P><P>Therefore, we purified 3A-SUP tagged with the tetracysteine motif, and directly used it to calibrate the fluorescence intensity of the FlAsH-EDT2 probe based on target protein concentration. The fluorescence intensity did change with the magnitude of protein concentration, as expected. However, we did not find a good linear relationship between the target protein concentration and probe fluorescence (Table. 2.). Hence, only qualitative results could be obtained using this assay.</P><FIGURE><FIGCAPTION style="text-align:center;">
                                                        Table. 2. Fluorescence data of the FlAsH-tetracysteine assay
                                                    </FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV></DIV><DIV class="references"><H3>References:</H3><P>[1] Usama, B. et al., Increasing the secretion ability of the kil gene for recombinant proteins in Escherichia coli by using a strong stationary-phase promoter. <I>Biotechnical Letter</I> ,<B>29</B>, 1893–1901 (2007).</P><P>[2] Kim et al., Comparison of PaprE , PamyE and P43 promoter strength for β-galactosidase and staphylokinase expression in Bacillus subtilis. <I>Biotechnology and bioprocess engineering 13</I>,<B> 313-318</B> (2008).</P><P>[3] Hengen, P.N., Purification of His-Tag fusion proteins from Escherichia coli.<I> Trends in Biochemical Sciences</I>,<B> 20(7)</B>, 285-286 (1995).</P><P>[4] Haitjema, C.H., et al., Universal Genetic Assay for Engineering Extracellular Protein Expression. <I>ACS Synthetic Biology</I>, <B>3(2), 74-82</B> (2014).</P></DIV></DIV></DIV></SECTION></DIV></DIV><FOOTER id="page-footer"><DIV class="row"><DIV class="twelve columns"><UL class="copyright"><LI><A href="2016.igem.org/Team:Peking">Home</A>   <A href="mailto:pkuigem2016@126.com">Contact</A></LI><SPAN>©2016 PEKING IGEM. All Rights Reserved.</SPAN><LI><A href="http://getbootstrap.com/2.3.2/">Based on Bootstrap</A></LI></UL></DIV><DIV id="go-top" style="display: block;"><A title="Back to Top" href="#">Go To Top</A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>