<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_York_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:York/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="cam-container"><DIV class="navbar navbar-inverse navbar-fixed-top navbar-pad"><DIV class="container"><DIV class="navbar-header"><A class="navbar-brand" href="//2017.igem.org/Team:York">YORK</A></DIV><DIV class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI><A href="//2017.igem.org/Team:York">HOME</A></LI><LI><A href="////2017.igem.org/Team:York/Description">PROJECT</A></LI><LI><A href="//2017.igem.org/Team:York/Description">DESCRIPTION</A></LI><LI><A href="//2017.igem.org/Team:York/Design">DESIGN</A></LI><LI><A href="//2017.igem.org/Team:York/Applied_Design">APPLIED DESIGN</A></LI><LI><A href="//2017.igem.org/Team:York/Results">RESULTS</A></LI><LI><A href="//2017.igem.org/Team:York/Background">BACKGROUND</A></LI><LI><A href="//2017.igem.org/Team:York/Downloads">DOWNLOADS</A></LI><LI><A href="//2017.igem.org/Team:York/Contribution">CONTRIBUTION</A></LI></UL></DIV><LI><A href="https://2017.igem.org/Team:York/Hardware">DIHM</A></LI><LI><A href="https://2017.igem.org/Team:York/Hardware">HARDWARE</A></LI><LI><A href="https://2017.igem.org/Team:York/Software">SOFTWARE</A></LI><LI><A href="https://2017.igem.org/Team:York/Measurement">MEASUREMENT</A></LI></DIV><LI><A href="https://2017.igem.org/Team:York/Human_Practices">COMMUNITY</A></LI><LI><A href="https://2017.igem.org/Team:York/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2017.igem.org/Team:York/HP/Silver">HP SILVER</A></LI><LI><A href="https://2017.igem.org/Team:York/HP/Gold_Integrated">HP GOLD INTEGRATED</A></LI><LI><A href="https://2017.igem.org/Team:York/Collaborations">COLLABORATIONS</A></LI></DIV><LI><A href="https://2017.igem.org/Team:York/Achievements">OUR LAB</A></LI><LI><A href="https://2017.igem.org/Team:York/Notebook">NOTEBOOK</A></LI><LI><A href="https://2017.igem.org/Team:York/Experiments">PROTOCOLS</A></LI><LI><A href="https://2017.igem.org/Team:York/Safety">SAFETY</A></LI><LI><A href="https://2017.igem.org/Team:York/Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2017.igem.org/Team:York/Achievements">ACHIEVEMENTS</A></LI></DIV><LI><A href="https://2017.igem.org/Team:York/Team">ABOUT US</A></LI><LI><A href="https://2017.igem.org/Team:York/Team">MEET THE TEAM</A></LI><LI><A href="https://2017.igem.org/Team:York/Attributions">ATTRIBUTION</A></LI><LI><A href="https://2017.igem.org/Team:York/Sponsors">SPONSORS</A></LI></DIV></DIV></DIV></DIV></DIV><DIV><HEADER class="intro-header" style="background: url(//2017.igem.org/wiki/images/2/27/TeamYork-TemporaryBackground.png); background-size: 100% 100%; background-position:0px 20px;"><DIV class="container"><DIV class="intro-message text-center container" style="margin: 0px 0px;"><H1 style="-webkit-text-stroke: 2px black; color:#fff;">Results</H1></DIV></DIV></HEADER></DIV><DIV class="cam-container" style="background-color: #ddd; color: #000;"><SECTION style="padding-top: 10px; padding-bottom: 20px; margin-top: 0px;"><DIV class="container" style="background-color: rgb(250,250,250); color: #000000; border-style: solid; border-color: #000000; border-radius: 25px;"><DIV class="text-center"><H1>Our Results</H1></DIV><DIV class="text-left" style="margin: 0px;"><P style="font-size: 18px;">Detailed, below, are the results we obtained during the various experimental stages of this project. Accompanying each display of data is a small paragraph of conclusive remarks which aims to outline the importance or significance of the results. You can get straight to a section by following these links:</P><UL style="list-style: none; text-align: center;"><LI><A style="color: #000;" href="//2017.igem.org/Team:York/Results#Modifying-Chlamydomonas">Modifying <EM>Chlamydomonas reinhardtii</EM></A></LI><LI><A style="color: #000;" href="//2017.igem.org/Team:York/Results#Growth-Tests">Growth Tests</A></LI><LI><A style="color: #000;" href="//2017.igem.org/Team:York/Results#Constructs">Constructs</A></LI><LI><A style="color: #000;" href="//2017.igem.org/Team:York/Results#Sizes">Size and Shape Analysis</A></LI><LI><A style="color: #000;" href="//2017.igem.org/Team:York/Results#Cell-Counts">Cell Density Analysis</A></LI><LI><A style="color: #000;" href="//2017.igem.org/Team:York/Results#Motility">Motility Observations</A></LI><LI><A style="color: #000;" href="//2017.igem.org/Team:York/Results#Conclusions">Conclusions</A></LI></UL></DIV></DIV></SECTION></DIV><DIV class="cam-container" style="background-color: #dddddd; color: #444444;"><SECTION id="Modifying-Chlamydomonas" style="padding-top: 10px; padding-bottom: 20px; margin-top: 0px;"><DIV class="container" style="background-color: rgba(224,245,222,0.9); color: rgb(11,33,10); border-style: solid; border-color: rgb(11,33,10); border-radius: 25px;"><DIV class="text-center"><H1>Modifying <EM>C. reinhardtii</EM></H1></DIV><DIV class="text-left" style="margin: 0px;"><P style="font-size: 18px;">We attempted to introduce MEX-1, a maltose exporter gene normally found in the plastid membrane, into the cell membrane of <EM>C. reinhardtii</EM>.</P><P style="font-size: 18px;">Unfortunately, the results were bleak.</P><P style="font-size: 18px;">The GC-content of <EM>C. reinhardtii</EM>’s genome is around 60% [1]. This is a likely cause of the modification’s failure, since high GC content is linked with difficulty in amplifying through PCR [2].</P><P style="font-size: 18px;">Further, despite our best attempts at performing the transformation, we simply did not have enough time to get a successful result. <EM>C. reinhardtii</EM> has a relatively slow growth rate (doubling around once every 8 hours at 25°C [3]) which, coupled with the waiting time for sequencing results, meant that, over the short period in which we could use laboratories in our Department of Biology, we only had time for two attempts at modification. Both were unsuccessful and, thence, we had to abandon the idea that <EM>C. reinhardtii</EM> could export maltose as a carbon source for LW06.</P></DIV></DIV></SECTION></DIV><DIV class="cam-container" style="background-color: #dddddd; color: #444444;"><SECTION id="Growth-Tests" style="padding-top: 10px; padding-bottom: 20px; margin-top: 0px;"><DIV class="container" style="background-color: rgba(224,245,222,0.9); color: rgb(11,33,10); border-style: solid; border-color: rgb(11,33,10); border-radius: 25px;"><DIV class="text-center"><H1>Will it grow, if...?</H1></DIV><DIV class="text-left" style="margin: 0px;"><P style="font-size: 18px;">Concurrently with the above, we tested the growth of our organisms under different circumstances and on different media to see what ramifications the effects would have for our proposed co-culture.</P><P style="text-align: center; font-size: 20px;"><STRONG>LW06 on Maltose</STRONG></P><CENTER><CENTER style="width: 90%;"><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 1:</STRONG> Maltose test plate 1 box plot. Plate 1 has serious anomalies in the blanks and controls when compared to plates 2 and 3. RV on y axis is the regression values. Results from this experiment where discounted due to problems with the control and blank samples.</LI></UL></CENTER></CENTER></DIV><DIV class="col-md-6"><CENTER style="border-style: solid; border-color: rgb(11,33,10); border-radius: 25px; color: #000; background-color: #fff;"><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 2:</STRONG> Maltose test plate 2 box plot. Missing data points in the blanks and controls are due to OD data not being high enough to be logged by R studio software. RV on y axis is the regression values.</LI></UL></CENTER></DIV><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 3:</STRONG> Maltose test plate 3 box plot. RV on y axis is the regression values.</LI></DIV><DIV class="col-md-6"><CENTER style="border-style: solid; border-color: rgb(11,33,10); border-radius: 25px; color: #000; background-color: #fff;"><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 4:</STRONG> Maltose test plate 1 column graph, mean growth rates on maltose gradient in 96 well plates. Media maltose (1-8) represents columns 5–12 (TP.Maltose) wells. Given previously described issues with controls data from this experiment is not to be used for analysis.</LI></UL></CENTER></DIV><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 5:</STRONG> Maltose test plate 2 column graphs, mean growth rates on maltose gradient in 96 well plates. Media maltose (1-8) represents columns 5–12 (TP.Maltose) wells.</LI></SECTION></DIV><DIV class="col-md-6"><CENTER style="border-style: solid; border-color: rgb(11,33,10); border-radius: 25px; color: #000; background-color: #fff;"><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 6:</STRONG> Maltose plate 3 column graph, mean growth rates on maltose gradient in 96 well plates. Media maltose (1-8) represents columns 5–12 (TP.Maltose) wells.</LI></UL></CENTER></DIV><P style="font-size: 18px;"><STRONG>Results:</STRONG> Consistency is shown for high growth in column 5 (1g/L) but remains more or less level throughout the increasing maltose concentrations. Such consistency indicates LW06 is somewhat poor at maltose metabolism but is able to survive on it. Mean growth rates are small, though this is potentially explained by a highly limiting culture volume. However, data from the plate reader indicates clear exponential and plateau events during growth. In conclusion, growth under maltose conditions is possible. Conditioning for maltose consumption may be required to improve final culturing with <EM>C. reinhardtii</EM> when it is exporting maltose, in future co-cultures. The problems with the first experimental run, contamination in the blanks and controls, were most likely due to condensation on the interior lid or displacement of culture material in the controls and blank sections of the plate during agitation. As a result of this, data from this experiment is to be discounted for analysis and should be considered when proceeding with experiments in the future as to avoid issues of contamination.</P><P style="text-align: center; font-size: 20px;"><STRONG>LW06 on Home Blend Medium</STRONG></P><CENTER><CENTER style="width: 90%;"><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 7:</STRONG> Home blend (Hblend) test plate 1. Regression values (y axis) show that TP-Maltose performs worse than TP-Glucose. Hblend-Glucose greatly out performs Hblend-Maltose, but is comparable to TP-Maltose.</LI></UL></CENTER><DIV class="col-md-6"><CENTER style="border-style: solid; border-color: rgb(11,33,10); border-radius: 25px; color: #000; background-color: #fff;"><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 8:</STRONG> Hblend test plate 1. Mean growth rates here show that the growth rate for Hblend-Glucose is much greater than TP or Hblend-Maltose. TP-Maltose outperforms our Home blend recipe.</LI></UL></CENTER></DIV><LI>A T-test was conducted for TP.Maltose and Hblend.Maltose. T-test (Degrees of Freedom = 48, p = 8.82 x 10<SUP>-10</SUP>). This indicates significant difference between media types.</LI><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 9:</STRONG> Hblend test plate 2. Regression values (y axis), as in figure 7, show that TP and Hblend-Maltose are out performed by Hblend-Glucose.</LI></CENTER><DIV class="col-md-6"><CENTER style="border-style: solid; border-color: rgb(11,33,10); border-radius: 25px; color: #000; background-color: #fff;"><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 10:</STRONG> Hblend test plate 2. Mean growth rates here show Hblend-Glucose out performs TP or Hblend-Maltose with no difference between the latter pair in terms of growth.</LI></UL></CENTER></DIV><LI>A T-test was conducted for the second plate. T-test (D.o.F. = 48, p = 0.058255). For the second plate, no statistical significance between media types was indicated.</LI><P style="font-size: 18px;"><STRONG>Results:</STRONG> Overall, Hblend performs unremarkably when compared to TP media. Despite having much higher micronutrients, Hblend does not perform as well as we hypothesised. Some data collected indicates that a longer run time may have had an effect, but it is unlikely that this would lead to major improvements over the TP media. The problem seems to be the uptake or metabolism of maltose. Conditioning our organisms to grow on maltose may improve growth rates in a co-culture. However, such a limitation may work to the advantage of the co-culture. A key problem to consider was the greatly divergent growth rates between <EM>C. reinhardtii</EM> and the LW06 <EM>E. coli</EM> strain. This reduced growth with maltose as the carbon source may provide a sufficient rate limiter on the growth of LW06 to avoid complete culture saturation with the <EM>E. coli</EM> in future co-cultures.</P><P style="text-align: center; font-size: 20px;"><STRONG><EM>C. reinhardtii</EM> Ethanol Tolerance</STRONG></P><CENTER><CENTER style="width: 90%;"><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 11:</STRONG> Cell counts for <EM>C. reinhardtii</EM> via multiple well plate readers. Readings taken ~24hours manually.</LI></UL></CENTER><DIV class="col-md-6"><CENTER style="border-style: solid; border-color: rgb(11,33,10); border-radius: 25px; color: #000; background-color: #fff;"><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 12:</STRONG> Cell counts for <EM>C. reinhardtii</EM> via multiple well plate readers. Readings taken ~24hours manually.</LI></UL></CENTER></DIV></CENTER><P style="font-size: 18px;"><STRONG>Results:</STRONG><EM>C. reinhardtii</EM> is able to tolerate an increase of ethanol to at the most 3.2% (as a portion of media). Given rates of increase it seems that it could tolerate much higher concentrations. This is encouraging, as ethanol resistance of the <EM>C. reinhardtii</EM> would likely be sufficient to survive upper limits of the LW06 strain’s ethanol production. Interestingly, <EM>C. reinhardtii</EM> experiences better growth at higher concentrations of ethanol, there is not a clear reason this is the case. Addition of ethanol to the media may have made growth conditions more amenable to <EM>C. reinhardtii</EM>. There is some chance that the ethanol may have evaporated which, while possible, is unlikely due to the sealing of the plate. Overall, it seems that <EM>C. reinhardtii</EM> would be able to survive the proposed amount of ethanol that could be produced by the LW06.</P><DIV class="cam-container" style="background-color: #dddddd; color: #444444;"><SECTION id="Chlamy-Coli" style="padding-top: 10px; padding-bottom: 20px; margin-top: 0px;"><DIV class="container" style="background-color: rgba(224,245,222,0.9); color: rgb(11,33,10); border-style: solid; border-color: rgb(11,33,10); border-radius: 25px;"><DIV class="text-center"><H1>Co-culture of <EM>C. reinhardtii</EM> and <EM>E. coli</EM> LW06</H1></DIV><DIV class="text-left" style="margin: 0px;"><P style="font-size: 18px;">There's a large growth disparity between the two organisms [3][4], so we also performed an experiment to check if LW06 would vastly outperform <EM>C. reinhardtii</EM> on all types of media.</P><P style="text-align: center; font-size: 20px;"><STRONG>Growth in a Co-culture</STRONG></P><CENTER><CENTER style="width: 90%;"><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 13:</STRONG> Cell counts for LW06 in each culture via spread plating at 10<SUP>4</SUP> dilutions.</LI></UL></CENTER></CENTER></DIV><DIV class="col-md-6"><CENTER style="border-style: solid; border-color: rgb(11,33,10); border-radius: 25px; color: #000; background-color: #fff;"><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 14:</STRONG> Cell counts for <EM>C. reinhardtii</EM> via manual haemocytometer microscope cell counting at a 10<SUP>4</SUP> dilution.</LI></UL></CENTER></DIV><LI style="text-align: center; font-size: 14px;"><STRONG>Figure 15:</STRONG> Overlaid cell counts for the co-culture. LW06 represented by blue points and <EM>C. reinhardtii</EM> by orange.</LI></DIV><P style="font-size: 18px;"><STRONG>Results:</STRONG> Overall LW06 outperforms <EM>C. reinhardtii</EM> in all but one of the cultures (Tube 4). It is encouraging to see LW06 perform well on TP.Maltose and TAP.Maltose, although the latter may have been due to the presence of a second carbon source in the form of acetate. It is odd that in every one of the experiments tube that were inoculated with 100 μl of 0.5-1.0 OD LW06 that the cell counts are considerably lower. It seems that the initial starting cell density may have inhibited growth but it is not clear why this effect would continue when dispersed in the co-culture. Alternatively, an unknown error may have occurred at some point during inoculation. From the results it is clear that, if it was some unknown or unnoticed error, it was made consistently in every one of the co-cultures with 100 μL inoculation. No relevant conclusions can be drawn from this. However, considerations such as the growth limiting factor of the TP.Maltose may work in our interest as to avoid culture saturation with LW06.</P><P style="font-size: 18px;">An obvious error, in hindsight, is the issue of the 10 μL inoculation. It is likely that a much higher volume added to the culture would have improved the results. This would have provided a high enough cell density, should growth have not be great enough during co-culturing with the modified <EM>C. reinhardtii</EM> in TP media. Overall co-cultures with these two organisms are possible, but higher OD/volume of inoculant may be needed.</P></SECTION></DIV><DIV class="cam-container" style="background-color: #ddd; color: #000;"><SECTION id="Constructs" style="padding-top: 10px; padding-bottom: 20px; margin-top: 0px;"><DIV class="container" style="background-color: rgba(224,245,222,0.9); color: rgb(11,33,10); border-style: solid; border-color: rgb(11,33,10); border-radius: 25px;"><DIV class="text-center"><H1>Constructs</H1></DIV><DIV class="text-left" style="margin: 0px;"><P style="font-size: 18px;">We constructed several plasmids to attempt the introduction of MEX-1 into the outer membrane of <EM>C. reinhardtii</EM>. The figures, below, show maps of these plasmids and our evidence that we correctly made the constructs.</P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true" style="color: rgb(11,33,10);"><H3 style="text-align: center;"><STRONG>Plasmid Maps</STRONG></H3><DIV class="panel panel-default" style="background-color: #ddd; border-style: solid; border-color: #444; border-radius: 25px;"><DIV class="panel-heading" role="tab" id="Map1" style="background-color: #ddd;"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#Map1-collapse" aria-expanded="false" aria-controls="Map1-collapse"><DIV><DIV class="col-md-11" style="color: #000;"><STRONG>Figure 16:</STRONG> Backbone pLM005</DIV></DIV></A></H4></DIV></DIV><DIV class="panel panel-default" style="background-color: #ddd; border-style: solid; border-color: #444; border-radius: 25px;"><DIV class="panel-heading" role="tab" id="Map3" style="background-color: #ddd;"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#Map2-collapse" aria-expanded="false" aria-controls="Map2-collapse"><DIV><DIV class="col-md-11" style="color: #000;"><STRONG>Figure 17:</STRONG> pLM005 + MEX-1</DIV></DIV></A></H4></DIV></DIV><DIV class="panel panel-default" style="background-color: #ddd; border-style: solid; border-color: #444; border-radius: 25px;"><DIV class="panel-heading" role="tab" id="Map3" style="background-color: #ddd;"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#Map3-collapse" aria-expanded="false" aria-controls="Map3-collapse"><DIV><DIV class="col-md-11" style="color: #000;"><STRONG>Figure 18:</STRONG> pLM005 + MEX-1 + Signal Peptide</DIV></DIV></A></H4></DIV></DIV><P style="font-size: 18px;"><STRONG>Results:</STRONG> We used plasmid pLM005 as our vector (see figure 16), which is designed specifically for the expression of eukaryotic ORFs and incorporates a YFP fusion at the C terminal end of the recombinant protein, so we could detect localisation of our edited MEX-1 to the outer membrane.Cutting of the plasmid at the two HpaI sites removes a small fragment of the plasmid and provides the insertion site for the recombinant gene. Once amplified, the vector is cut again at the eco-RV sites (not shown) to produce a cassette for transformation directly into the <EM>C. reinhardtii</EM> genome.We successfully cloned and inserted the MEX-1 gene into the pLM005 vector (see figure 17, gel electrophoresis and fluorescent microscopy images below). Our strategy for relocation of MEX-1 (which is a chloroplast membrane protein [5]) was to remove the signal peptide of MEX-1 by PCR cloning and assemble the truncated MEX-1 gene with the signal peptide sequence of known outer membrane protein ACA3 (see figure 18). We were able to successfully assemble this construct (see gel electrophoresis), however we did not manage to successfully transform it into <EM>C. reinhardtii</EM>.
</P><H3 style="text-align: center;"><STRONG>BioBrick</STRONG></H3><DIV class="panel panel-default" style="background-color: #ddd; border-style: solid; border-color: #444; border-radius: 25px;"><DIV class="panel-heading" role="tab" id="Map5" style="background-color: #ddd;"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#Map5-collapse" aria-expanded="false" aria-controls="Map5-collapse"><DIV><DIV class="col-md-11" style="color: #000;"><STRONG>Figure 19:</STRONG> BioBrick</DIV></DIV></A></H4></DIV></DIV><DIV class="panel panel-default" style="background-color: #ddd; border-style: solid; border-color: #444; border-radius: 25px;"><DIV class="panel-heading" role="tab" id="Map6" style="background-color: #ddd;"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#Map6-collapse" aria-expanded="false" aria-controls="Map6-collapse"><DIV><DIV class="col-md-11" style="color: #000;"><STRONG>Figure 20:</STRONG> BioBrick Fragments</DIV></DIV></A></H4></DIV></DIV><DIV class="panel panel-default" style="background-color: #ddd; border-style: solid; border-color: #444; border-radius: 25px;"><DIV class="panel-heading" role="tab" id="Map7" style="background-color: #ddd;"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#Map7-collapse" aria-expanded="false" aria-controls="Map7-collapse"><DIV><DIV class="col-md-11" style="color: #000;"><STRONG>Figure 21:</STRONG> BioBrick + MEX-1 Insert</DIV></DIV></A></H4></DIV></DIV><P style="font-size: 18px;"><STRONG>Results:</STRONG> We designed a biobrick based on the pLM005 plasmid which we used for our project (described above). It uses the silver RFC[23] prefix and suffix for insertion of a eukaryotic ORF into that site where it is tagged at the C terminal end by a YFP and expression is driven by the strong, constitutive PsaD promoter (see figure 19). We used the complimentary 1000bp gblock offer from IDT to synthesise fragments of our biobrick with 18bp overlaps for Gibson assembly. However, this large construct required 7 fragments (see figure 20) and we were unsuccessful in our attempts to assemble.</P><H3 style="text-align: center;"><STRONG>Evidence</STRONG></H3><DIV class="panel panel-default" style="background-color: #ddd; border-style: solid; border-color: #444; border-radius: 25px;"><DIV class="panel-heading" role="tab" id="Map4" style="background-color: #ddd;"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#Map4-collapse" aria-expanded="false" aria-controls="Map4-collapse"><DIV><DIV class="col-md-11" style="color: #000;"><STRONG>Figure 22:</STRONG> Gel Electrophoresis</DIV></DIV></A></H4></DIV><DIV id="Map4-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="Map4"><DIV class="panel-body" style="margin: 15px 15px 15px 15px;"><DIV class="row"><DIV class="col-md-6" style="background-color: #fff; border-style: solid; border-radius: 25px; border-color: #000;"><CENTER><UL style="list-style: none; font-size: 14px;"><LI style="text-align: left;"><STRONG>A:</STRONG> Gel electrophoresis of PCR cloned genes from plasmid library of <EM>C. reinhardtii</EM> genome. ACA3 TMD is the region of the ACA3 gene 5' of the first transmembrane domain, ACA3 SP is the region of the ACA3 gene predicted to code the signal peptide.</LI></UL></CENTER></DIV><DIV class="col-md-6" style="background-color: #fff; border-style: solid; border-radius: 25px; border-color: #000;"><CENTER><UL style="list-style: none; font-size: 14px;"><LI style="text-align: left;"><STRONG>B:</STRONG> Gel electrophoresis of PstI digested MEX-1 plasmids after Hifi assembly. Colonies 1,3,4,6 &amp;10 appear to contain successfully assembled constructs.</LI></UL></CENTER></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default" style="background-color: #ddd; border-style: solid; border-color: #444; border-radius: 25px;"><DIV class="panel-heading" role="tab" id="Evidence" style="background-color: #ddd;"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#Evidence-collapse" aria-expanded="false" aria-controls="Evidence-collapse"><DIV><DIV class="col-md-11" style="color: #000;"><STRONG>Figure 23:</STRONG> Fluorescence Microscopy</DIV></DIV></A></H4></DIV><DIV id="Evidence-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="Evidence"><DIV class="panel-body"><DIV class="row"><DIV class="col-md-6"><UL style="list-style: none;"><LI><STRONG>A:</STRONG> Wild-type <EM>Chlamydomonas reinhardtii</EM> under a fluorescence microscope before filtering for the Venus fluorescence wavelength.</LI></UL></DIV><DIV class="col-md-6"><UL style="list-style: none;"><LI><STRONG>B:</STRONG> The wild-type under fluorescence microscope with a filter for the Venus fluorescence wavelength.</LI></UL></DIV></DIV><DIV class="row"><DIV class="col-md-6"><UL style="list-style: none;"><LI><STRONG>C:</STRONG><EM>C. reinhardtii</EM> after our insertion of the Venus tag, before wavelength filtering.</LI></UL></DIV><DIV class="col-md-6"><UL style="list-style: none;"><LI><STRONG>D:</STRONG><EM>C. reinhardtii</EM> after our insertion of the Venus tag, after wavelength filtering.</LI></UL></DIV></DIV></DIV></DIV></DIV><P style="font-size: 18px;"><STRONG>Results:</STRONG> Figure 22 shows our gel electrophoresis data, confirming that we correctly assembled our construct. Figure 23 shows that we were able to insert our Venus control into <EM>C. reinhardtii</EM>. Unfortunately, the same was not true of our MEX-1 construct.</P></DIV></DIV></DIV></SECTION></DIV><DIV class="cam-container" style="background-color: #ddd; color: #000;"><SECTION id="Sizes" style="padding-top: 10px; padding-bottom: 20px; margin-top: 0px;"><DIV class="container" style="background-color: rgba(223,230,239,0.9); color: #162844; border-style: solid; border-color: #162844; border-radius: 25px;"><DIV class="text-center"><H1>Size and Shape Analysis</H1></DIV><DIV class="text-left" style="margin: 0px;"><P style="font-size: 18px;">Though we had some trouble with alignment of lenses, we achieved several results wherein observing the shape of an organism in the focal plane was facilitated. We present the results below. Organisms are outlined in red boxes.</P><P style="text-align: center; font-size: 20px;"><STRONG>Confirming Shapes and Sizes of Organisms</STRONG></P><CENTER><CENTER style="width: 90%;"><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 24:</STRONG><EM>E. coli</EM> strain BW25113. Its size is (6 x 4) pixels. This corresponds to (1.34 x 0.90) µm.</LI></UL></CENTER></CENTER></DIV><DIV class="col-md-6"><CENTER><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 25:</STRONG><EM>E. coli</EM> strain BW25113. Its size is (9 x 6) pixels. This corresponds to (2.02 x 1.34) µm.</LI></UL></CENTER></DIV><LI style="text-align: center; font-size: 14px;"><STRONG>Figure 26:</STRONG> This sample contained wildtype <EM>E. coli</EM> and <EM>C. reinhardtii</EM>. It was provided by Dr. Laurence Wilson, captured using his own DIHM. The hologram was produced using our software and the size of <EM>E. coli</EM> was (6 x 4) pixels, which equates to (1.34 x 0.90) µm.</LI></DIV><DIV class="col-md-6"><CENTER><UL style="list-style: none;"><LI style="text-align: center; font-size: 14px;"><STRONG>Figure 27:</STRONG><EM>Chlamydomonas reinhardtii</EM>. The sizes were as follows. A: (9 x 10) pixels, (10.1 x 11.2) µm. B: (9 x 9) pixels, (10.1 x 10.1) µm. C: (10 x 9) pixels, (11.2 x 10.1) µm. D: (10 x 9) pixels, (11.2 x 10.1) µm.</LI></UL></CENTER></DIV><P style="font-size: 18px;"><STRONG>Results:</STRONG> These results show that our software is capable of producing holograms within which the shapes of organisms can be distinguished. In figures 24 to 26, the <EM>E. coli</EM> can be seen to be rod shaped, approximately 2 µm in one direction and 1 µm in the other, as expected [6]. In the figure 27, <EM>C. reinhardtii</EM> is approximately circular with diameter around 10 µm, also as expected [3]. These are promising results which show the potential for the technique to be used to distinguish between organisms within a co-culture, especially when there is a size discrepancy. This is exemplified in figure 26, most of all. Figures 24, 25 and 27 are also evidence that the quality of images taken with our hardware is sufficient for shape detection. As seen <A style="color: #0000ff;" href="//2017.igem.org/Team:York/HP/Silver#International">here</A>, we also tested the ability to find the size and shape of organisms with <EM>Saccharomyces cerevisiae</EM>, which is of an intermediate size when compared with <EM>E. coli</EM> and <EM>C. reinhardtii</EM>. The <EM>S. cerevisiae</EM> was kindly provided by the iGEM Aachen 2017 team.</P></SECTION></DIV><DIV class="cam-container" style="background-color: #ddd; color: #000;"><SECTION id="Cell-Counts" style="padding-top: 10px; padding-bottom: 20px; margin-top: 0px;"><DIV class="container" style="background-color: rgba(223,230,239,0.9); color: #162844; border-style: solid; border-color: #162844; border-radius: 25px;"><DIV class="text-center"><H1>Cell Density Analysis</H1></DIV><DIV class="text-left" style="margin: 0px;"><P style="font-size: 18px;">We used our analysis software to count the cells in manually-selected focal planes. All of the below images are from data taken with our own hardware and software. We then calculated the volume of the hologram in each case (accounting for magnification, step size and number of steps) and used both to find the number of cells per millilitre. We used the formula: Cells/mL = (D x C x 10<SUP>-6</SUP>) / (512<SUP>2</SUP> x N x ((1.12/M)x10<SUP>-6</SUP>)<SUP>3</SUP> ); wherein C is the counted number of blobs from the analysis software, D is the dilution number compared with the corroborated sample (e.g. D = 100 for a 1 part in 100 dilution, D = 1 for an undiluted sample), M is the magnification and N is the number of steps between the focal plane being analysed and the closest other plane with focused cells. In most cases N was 150, indicating the whole hologram had only one focused section. We corroborated our values' relevance by comparing to either an optical density reading or a reading from a ThermoFisher Countess II FL Automated Cell Counter.</P><P style="text-align: center; font-size: 20px;"><STRONG>Confirming Cell Counts</STRONG></P><CENTER><CENTER style="width: 90%;"><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 28:</STRONG><EM>C. reinhardtii</EM>: C = 13, D = 10, M = 1, N = 150. The Countess II FL gave a reading of (5.19 x 10<SUP>6</SUP>) cells/mL. Our calculated value was (2.4 x 10<SUP>6</SUP>) cells/mL.</LI></UL></CENTER></CENTER></DIV><DIV class="col-md-4"><CENTER><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 29:</STRONG><EM>C. reinhardtii</EM>: C = 113, D = 1, M = 5, N = 150. The Countess II FL gave a reading of (2.03 x 10<SUP>8</SUP>) cells/mL. Our calculated value was (2.6 x 10<SUP>8</SUP>) cells/mL.</LI></UL></CENTER></DIV><DIV class="col-md-4"><CENTER><UL style="list-style: none;"><LI style="text-align: left; font-size: 14px;"><STRONG>Figure 30:</STRONG> TAP medium only: C = 10, D = 1, M = 1, N = 150. There are no microorganisms present, the diffraction patterns here are due to dust on the slide. The Countess II FL gave a reading of (1.47 x 10<SUP>4</SUP>) cells/mL. Our calculated value was (1.8 x 10<SUP>5</SUP>) cells/mL.</LI></UL></CENTER></DIV><LI style="text-align: center; font-size: 14px;"><STRONG>Figure 31:</STRONG><EM>E. coli</EM>: C = 10, D = 100, M = 5, N = 150. The foci were slightly misshapen (but still countable) due to misalignment of the lens during setup. The OD reading returned (4 x 10<SUP>9</SUP>) cells/mL. Our calculated value was (2.3 x 10<SUP>9</SUP>) cells/mL.</LI></DIV><DIV class="col-md-6"><CENTER><UL style="list-style: none;"><LI style="text-align: center; font-size: 14px;"><STRONG>Figure 32:</STRONG><EM>Galdieria sulphuraria</EM>: C = 248, D = 1, M = 1, N = 150. The Countess II FL reading returned (5.3 x 10<SUP>6</SUP>) cells/mL. Our calculated value was (4.5 x 10<SUP>6</SUP>) cells/mL.</LI></UL></CENTER></DIV><P style="font-size: 18px;"><STRONG>Results:</STRONG><STRONG>Figure 28</STRONG> shows a well focused, not-magnified image of <EM>C. reinhardtii</EM>. It is clear from the image that several cells that were present within the area of the image were not detected. However, many, if not all, of them did not pass through the focus of the constructed hologram. Further, the cell count we obtained was only a factor of 2 different to that given by an lab-grade automated cell counter. Considering the minimally developed analysis software, this is a promising achievement that could likely be improved with time and better algorithms.</P><P style="font-size: 18px;"><STRONG>Figure 29</STRONG> shows a 5x magnified image of <EM>C. reinhardtii</EM>. The diffraction patterns are misshapen (we expect circles, like in figure 28, but larger). This is due to the lens being slightly misaligned, which points to a weakness of the hardware's design. Perfect alignment is difficult to achieve when the levelness of component platforms must be adjusted by hand. It can be seen, from the figure, that some of the detected blobs are in erroneous positions and that some cells remain undetected. Upon counting manually, it seems that these two errors roughly cancel each other out. We conclude from this figure that the design of the hardware and software must be improved before industrial use is realisable, though this (and our other data) serve as proof of concept.</P><P style="font-size: 18px;"><STRONG>Figure 30</STRONG> shows a not-magnified image of a slide whose contents is solely TAP medium. The diffraction patterns that are apparent are most likely caused by dust or imperfections in the slide. The count we obtained for this hologram was an order of magnitude higher than the equivalent result from the automated cell counter, but was an order of magnitude (or more) lower than that for other not-magnified holograms that we took, even the least populous. The conditions used for detection in this image were the same as for figure 28, showing that our uncertainty in a measurement like this is likely around 10%. This is twice that of the automated cell counter we used [7]. This is a positive result, as it implies that, as the analysis techniques we have used are developed in future, we can approach the accuracy of widely used lab equipment.</P><P style="font-size: 18px;"><STRONG>Figure 31</STRONG> shows <EM>E. coli</EM> LW06 at 5x magnification. The cells' diffraction patterns were misshapen for the same reasons as those presented, above, for figure 29. Regardless, the count achieved was around a factor of 2 less than the value obtained through optical density analysis (which was performed at 600 nm wavelength). Yet again, as evidence that our concept is useful, this result is promising.</P><P style="font-size: 18px;"><STRONG>Figure 32</STRONG> shows <EM>Galdieria sulphuraria</EM> at 1x magnification. This was a densely populated sample, though the software still performed well. The blue outline denotes that the cell was detected by looking at the gradient image. The percentage difference between the Countess II FL reading and our own value was 15%. This equates to a difference in count of around 7. This serves as perhaps our best evidence that, when all goes according to plan with our current hardware and software, it functions as an at least somewhat viable alternative to other cell counting methods.</P><P style="text-align: center; font-size: 20px;"><STRONG>Confirming Cell Counts in a Co-Sample</STRONG></P><CENTER><CENTER style="width: 90%;"><UL style="list-style: none;"><LI style="text-align: center; font-size: 14px;"><STRONG>Figure 33:</STRONG><EM>C. reinhardtii</EM> and <EM>G. sulphuraria</EM>: C = 21, D = 1, M = 1, N = 150. The Countess II FL reading returned (6.28 x 10<SUP>5</SUP>) cells/mL for the combined total of the organisms. Our calculated value was (3.8 x 10<SUP>5</SUP>) cells/mL.</LI></UL></CENTER></CENTER></SECTION></DIV><DIV class="col-md-6"><CENTER><UL style="list-style: none;"><LI style="text-align: center; font-size: 14px;"><STRONG>Figure 34:</STRONG><EM>C. reinhardtii</EM> and <EM>G. sulphuraria</EM>: C = 9, D = 1, M = 1, N = 150. The Countess II FL was only capable of reading the total cell count, so we have no corroboratory datum for this test. Our count for the larger species was (1.6 x 10<SUP>5</SUP>) cells/mL.</LI></UL></CENTER></DIV><P style="font-size: 18px;"><STRONG>Results:</STRONG> Figures 33 and 34 show a co-sample of <EM>C. reinhardtii</EM> and <EM>G. sulphuraria</EM>. It was made by combining a sample of a set volume for each of the organisms. The <EM>C. reinhardtii</EM> sample had (4.17 x 10<SUP>5</SUP>) cells/mL and the <EM>G. sulphuraria</EM> had (3.02 x 10<SUP>5</SUP>) cells/mL. We suppose, therefore, that <EM>Galdieria</EM> made up 42.0% of the resultant co-sample's organisms by number. Our count from figure 34 is 42.1% of the count in figure 33. Figure 33's count was found by considering the entire range of documented radii for both organisms [7][8]. Figure 34's count was the upper half of that range, within which <EM>G. sulphuraria</EM> has been documented to reach during its lifecycle [8]. We conclude that it is most likely that we have found the ratio between two organisms of slightly different size in a co-sample. This, we present as evidence that our microscope and software are a useful tool for the determination of cell density in samples containing multiple visibly distinguishable organisms.</P><P style="font-size: 18px;">In short, the above data evidences the promising potential of our software and hardware combination with respect to use as a non-intrusive and fast method of monitoring the growth of cells in monocultures and in co-cultures. All of the image processing and calculations as outlined above can be done, in minutes, on the same Raspberry Pi that is used to take the images.</P><DIV class="cam-container" style="background-color: #ddd; color: #000;"><SECTION id="Motility" style="padding-top: 10px; padding-bottom: 20px; margin-top: 0px;"><DIV class="container" style="background-color: rgba(223,230,239,0.9); color: #162844; border-style: solid; border-color: #162844; border-radius: 25px;"><DIV class="text-center"><H1>Motility Observations</H1></DIV><DIV class="text-left" style="margin: 0px;"><P style="font-size: 18px;">Below we present evidence that our microscope is capable of showing the movement of organisms. Unfortunately, <EM>C. reinhardtii</EM> was the only organism we captured moving autonomously. We also tried to modify <EM>E. coli</EM> to increase its motility (see the <A style="color: #0000ff;" href="//2017.igem.org/Team:York/Description">Description</A> page), though this was unsuccessful. We borrowed another strain (DB5) that moves a lot, too. When we tried to look at it using our microscope, we couldn't see any moving cells, despite being able to see them under a light microscope. We did capture footage of <EM>E. coli</EM> moving due to the flow of liquid medium, however. It is presented below, also.</P><P style="text-align: center; font-size: 20px;"><EM>Chlamydomonas reinhardtii</EM> Moving in Samples</P><CENTER style="width: 98%;"><DIV class="row"><DIV class="col-md-6"><UL style="list-style: none;"><LI style="color: #000;">Figure 35: A GIF of a <EM>C. reinhardtii</EM> cell moving in a sample at 5x magnification.</LI></UL></DIV><DIV class="col-md-6"><UL style="list-style: none;"><LI style="color: #000;">Figure 36: A video showing a <EM>C. reinhardtii</EM> cell moving in a sample at 5x magnification.</LI></UL></DIV></DIV></CENTER><P style="font-size: 18px;"><STRONG>Results:</STRONG> It is apparent that the cells, above, are moving autonomously when compared with the motion due to flow as seen for <EM>E. coli</EM> below. The movement of <EM>C. reinhardtii</EM> is considerably more erratic and does not follow a single line/smooth path.</P><P style="text-align: center; font-size: 20px;"><EM>E. coli</EM> DB5 Drifting with Flow of Medium</P><CENTER style="width: 98%;"><DIV class="row"><DIV class="col-md-6"><UL style="list-style: none;"><LI style="color: #000;">Figure 37: A video showing <EM>E. coli</EM> cells moving, from the top-right in a down/left direction, in a sample at 5x magnification.</LI></UL></DIV></DIV></CENTER><P style="font-size: 18px;"><STRONG>Results:</STRONG> Though the moving diffraction patterns are faint, it cn be seen that the movement of each cell in the above video is identical, smooth and in a single direction. Hence, we assert that the motion is drift, rather than autonomous transit.</P></DIV></DIV></SECTION></DIV><DIV class="cam-container" style="background-color: #ddd; color: #000;"><SECTION id="Conclusions" style="padding-top: 10px; padding-bottom: 20px; margin-top: 0px;"><DIV class="container" style="background-color: rgb(250,250,250); color: #000000; border-style: solid; border-color: #000000; border-radius: 25px;"><DIV class="text-center"><H1>Conclusions</H1></DIV><DIV class="text-left" style="margin: 0px;"><P style="font-size: 18px;">In summary, we can draw several conclusions. We have had some successes and have seen some areas that seem promising but require more work.</P><P style="font-size: 18px;">The main shortcomings:</P><UL style="list-style: none;"><LI>Creating a closed analysis system using a milli-fluidic chamber.</LI><LI><EM>We came to realise that we required a thinner analysis chamber. Some evidence can be found <A style="color: #0000ff;" href="//2017.igem.org/Team:York/Design#Milli-fluidic-Problem">here</A>.</EM></LI><LI>Inserting MEX-1 into the outer membrane of <EM>Chlamydomonas reinhardtii</EM>.</LI><LI><EM>Our transformation of </EM>C. reinhardtii<EM> failed. As mentioned <A style="color: #0000ff;" href="//2017.igem.org/Team:York/Results#Modifying-Chlamydomonas">here</A>.</EM></LI><LI>Quantifying the motility of organisms using our DIHM setup.</LI><LI><EM>We could see </EM>C. reinhardtii<EM> moving on various occasions, but failed to observe the motility of </EM>E. coli<EM> DB5, a motile strain. As discussed <A style="color: #0000ff;" href="//2017.igem.org/Team:York/Results#Motility">here</A>.</EM></LI><LI>Using a UI to control the DIHM from a third party computer.</LI><LI><EM>We had a great design, there was a compatibility bug that we could not fix in the time we had. As seen <A style="color: #0000ff;" href="//2017.igem.org/Team:York/Design#UI-Problem">here</A>.</EM></LI></UL><P style="font-size: 18px;">The main successes:</P><UL style="list-style: none;"><LI>We created inexpensive and functioning upright DIHM hardware and open source holography software.</LI><LI><EM>The hardware can be seen <A style="color: #0000ff;" href="//2017.igem.org/Team:York/Hardware">here</A>. The software is discussed <A style="color: #0000ff;" href="//2017.igem.org/Team:York/Software">here</A>.</EM></LI><LI>We created blob detection software which is capable of finding cells of a given size in a focused image.</LI><LI><EM>The software is mentioned in more detail <A style="color: #0000ff;" href="//2017.igem.org/Team:York/Software#Blob-Detection">here</A>.</EM></LI><LI>We successfully observed the shapes of organisms using our DIHM and holography software.</LI><LI><EM>This can be seen <A style="color: #0000ff;" href="//2017.igem.org/Team:York/Results#Sizes">here</A>.</EM></LI><LI>We calculated the cell density of samples consistently to a fair degree of accuracy, using our DIHM and holography software.</LI><LI><EM>As shown <A style="color: #0000ff;" href="//2017.igem.org/Team:York/Results#Cell-Counts">here</A>.</EM></LI><LI>We confirmed that <EM>C. reinhardtii</EM> and <EM>E. coli</EM> can grow together in a co-culture.</LI><LI><EM>This is evidenced <A style="color: #0000ff;" href="//2017.igem.org/Team:York/Results#Growth-Tests">here</A>.</EM></LI><LI>We successfully constructed our MEX-1 plasmid.</LI><LI><EM>Though we didn't manage to make </EM>C. reinhardtii<EM> accept it, we present proof that we correctly constructed it <A style="color: #0000ff;" href="//2017.igem.org/Team:York/Results#Constructs">here</A>.</EM></LI></UL><P style="font-size: 18px;">Hence, our overarching conclusion is that digital holography is not only low-cost, easily adapted and useful for analysing co-cultures but also holds great potential in terms of future applications. We mention one such future application on our <A style="color: #0000ff;" href="//2017.igem.org/Team:York/Applied_Design#Future-Application">Applied Design</A> page.</P></DIV></DIV></SECTION></DIV><DIV class="cam-container" style="background-color: #ddd; color: #000;"><SECTION style="padding-top: 10px; padding-bottom: 20px; margin-top: 0px;"><DIV class="container" style="background-color: rgba(223,230,239,0.9); color: #162844; border-style: solid; border-color: #162844; border-radius: 25px;"><DIV class="text-center"><H1>References</H1></DIV><DIV class="text-left" style="margin: 0px;"><UL style="list-style: none; font-size: 18px;"><LI>[1]: S. Merchant, et al. The Chlamydomonas Genome Reveals the Evolution of Key Animal and Plant Functions, Science</LI><LI>[2]: Y. Benita, et al. Regionalized GC content of template DNA as a predictor of PCR success, Nucleic Acids Research</LI><LI>[3]: E. H. Harris, The Chlamydomonas Sourcebook, 2nd Edition</LI><LI>[4]: A. G. Marr, Growth rate of Escherichia coli, Microbiological Reviews</LI><LI>[5]: T. Niittylä, A Previously Unknown Maltose Transporter Essential for Starch Degradation in Leaves, Science</LI><LI>[6]: J. H. Cummings and G. T. Macfarlane, Role of intestinal bacteria in nutrient metabolism, Journal of parenteral and enteral nutrition</LI><LI>[7]: Countess II FL Automated Cell Counter, ThermoFisher Scientific, as available on 31st October 2017 at <A style="color: #0000ff;" href="//www.thermofisher.com/order/catalog/product/AMQAF1000">www.thermofisher.com/order/catalog/product/AMQAF1000</A></LI><LI>[8]: P. Albertano, C. Ciniglia, G. Pinto &amp; A. Pollio, The taxonomic position of Cyanidium, Cyanidioschyzon and Galdieria: an update, Hydrobiologia</LI></UL></DIV></DIV></SECTION></DIV><SECTION id="#footer-sec"><FOOTER><DIV class="cam-container" style="background-color: #000; padding-top: 5px; margin: 0px 0px -10px 0px;"><DIV class="container"><DIV class="footer row text-center"><DIV class="col-md-4"><UL><LI><A href="#">ABOUT US</A></LI><SPAN class="text-muted"><LI>We are an overgraduate team representing the University of York.</LI><LI>Our Quicker Way to Analyse Co-Cultures (<A href="#" style="color:#999;">QWACC</A>) is our entry into the Hardware track for iGEM 2017.</LI></SPAN></UL></DIV><DIV class="col-md-4"><UL><LI><A href="#">LOCATION</A></LI><SPAN class="text-muted"><LI>Department of Biology</LI><LI><A href="//www.york.ac.uk" style="color:#999;">University of York</A></LI><LI>Heslington</LI><LI>North Yorkshire</LI><LI>YO10 5DD</LI></SPAN></UL></DIV><DIV class="col-md-4"><UL><LI><A href="#">CONTACT US</A></LI><SPAN class="text-muted"><LI>Email: <A href="mailto:igemyork@gmail.com" style="color:#999;">igemyork@gmail.com</A></LI><LI>You can also reach us through our social media via the links below.</LI></SPAN></UL></DIV></DIV></DIV></DIV></FOOTER></SECTION></BODY></HTML>