"
Page
Discussion
View source
History
teams
Log in
 
Team:NJU-QIBEBT/wetlab/Protocol
From 2014.igem.org
HOME
PROJECT
Overview
Backgroud
Production module
Pumping out module
Indicator module
ACHIEVEMENT
Parts test results
Modeling
Judging Criteria
WET LAB
Protocol
Notebook
Parts
Cooperation
HUMAN PRACTICE
ETHICS SAFETY
TEAM
Member
Attribution
ACKNOWLEDGEMENT
Protocol
Preparation for the medium
Function: The medium which is the most commonly used for the culture of E. coli and other bacteria in the microbiological experiments. LB medium was usually added antibiotic to screen the bacteria.
LB medium formula:
Step:
1. Add all the materials into 950ml deionized water and shake the container to make sure solute dissolve completely.
2. Add deionized water into the container until the volume reaches 1L.
3. Cover the lid with aluminum foil and unscrew it.
4. Sterilize 20 minutes at 121℃.
5. Add chloramphenicol into the solution before using.
Agarose gel electrophoresis
Function: Agarose gel electrophoresis is a method of gel electrophoresis used in biochemistry, molecular biology and clinical chemistry to separate a mixed population of DNA or proteins in a matrix of agarose..
Step:
The gel is prepared by dissolving 1g agarose powder in an appropriate buffer, usually 1× TAE.
Use the microwave to heat the gel until boiling.
Cool the gel until 70℃ before proceeding, then add EB. Transfer the processed gel into cast. The comb should be setting during the gel solidification. Then remove the comb from the device.
Add DNA sample, the gel will be taken to the gel box and run at 110 V for approximately 30 minutes.
Recover and purify
Function: Recover and purify the target bands
Step:
1. According to the required number of the collected DNA strips, weigh out the approximate EP tubes then record the weight.
2. After the DNA electrophoresis, using clean blade to cutting the origin to collect the specific DNA strips under long-wave UV light. Weigh out again and record the weight.
3. Calculate the weight of the gel. Then use the 3:1 ratio to add WA solution. Incubate into 69℃ water bath until the gel dissolved completely. (If the length of the DNA fragments less than 4kb, use the ration 1.5:1 to add isopropanol as well)
4. Transfer the solution into spin column, centrifuge for 1 minute at 12000rpm and drain the waste liquid.
5. Add 700μl of WB solution, then centrifuge for 1 minute at 12000rpm and drain the waste liquid.
6. Add 500μl of WB solution, then centrifuge for 1 minute at 12000rpm and drain the waste liquid.
7. Centrifuge for 2 minutes at 12000rpm and drain the waste liquid.
8. Place the spin column into a new EP tube, add 15μl of 65℃ ddH2O.
9. Centrifuge for 1 minutes at 12000rpm, keeping the solution at -20℃.
Transformation of plasmid into competent bacteria
Function: To verify if target gene express in E. coli and the function of it.
Steps:
1. Add 5μl plasmid to 50μl of competent cell suspension, gently mixing the tube then contenting on ice for 30 minutes.
2. Incubate the EP tube into 42℃ water bath for 90s.
3. By finishing the last step, moving the EP tube into the ice bath immediately for 2 minutes.
4. Use the inoculating loop to load bacteria liquid then streak on the LB plate which contains chloramphenicol.
Plasmid Extraction
Function: To extract plasmids from E.coli.
Step:
In this part, we use GeneKlean Plasmid Nimiprep Kit to extract plasmid from E.coli. Kit contents are as follows.
1. Pellet 1.5mL of overnight culture by centrifugation at 8800g for 15 sec. Drain the liquid completely.
2. Add 100uL of Solution I to the pellet, mix gently and keep on ice for 2 min.
3. Add 200uL of Solution II to the mixture, mix gently by inverting the tube 4-6 times and then keep at room temperature (RT) for 1 min.
4. Add 350uL of Solution III, and mix gently. Incubate at RT for 1 min.
5. Spin at 11000g for 5 min.
6. Place spin column into a 2mL collection tube. Transfer supernatant (step 1.5) to the column. Let it stand for 2 min. Spin at 6200g for 1 min.
7. Discard the flow-through in the tube. Add 500uL of Wash Solution to the column, and spin at 7100g for 1 min.
8. Repeat wash procedure in step 1.7.
PCR
Function: To amplify a few copies of DNA across several orders of magnitude and check the length of DNA sequences between two designed primers.
Protocol for colony PCR (40uL)
RNA Extraction
Function: To extract RNA from E.coli.
Step:
1. 1-2 g fresh material, grinded with pestle and mortar in liquid nitrogen, then transfer to a 1.5 ml tube
2. Add 1 ml Trizol reagent, mix thoroughly
3. Cool on ice for 5 min
4. Spin at 12,000g, 4 ℃, for 10 minutes
5. Remove the supernatants to new tubes, add 400ul phenol/chloroform (1:1V), vortex mix for 15 seconds
6. Incubate the mix on ice for 5 minutes; two layers should form
7. Centrifuge at 12,000g at 4℃ 10 minutes
8. Carefully transfer the top (aqueous) layer into the new tubes. Be VERY careful at this step not to pipet over any of the interphase. If necessary, only take 70-80% of the estimated aqueous layer.
9. Add 1:1V of isopropanol to each tube, mix by inversion
10. Centrifuge at 12,000g for 10min at 4℃
11. Discard the supernatant very carefully. Depending on the tube, the pellet may be very loose. Wash the pellet with 1ml 75% ethanol centrifuge at 7500g for 5 min.
12. Remove all of the residual ethanol with a pipet. Air dry 4 min.
13. Redissolve RNA in 30µl DEPC (Diethyl Pyocarboanate ) treated H2O
14. Store at -80℃
RT-PCR
Function: To detect RNA expression levels.
Step:
According to the manufacturer’s instruction, use tubes to compound these reagent that listed below. All procedures were carried out on ice.
The volume of RNA and DEPC water are calculated according to the concentration of RNA solution
Set the PCR as requested below. Then, start to reverse-transcribe the target RNA 
Note: qRT-PCR was carried out using a Taqman miRNA PCR kit (Applied Biosystems, Foster City, CA, USA) according to the manufacturer’s instructions. Briefly, 1μl (approximate 1μg/μl ) of total RNA was reverse-transcribed to cDNA using AMV reverse transcriptase (TaKaRa, Dalian, China) and the stem-loop RT primers (Applied Biosystems). Real-time PCR was performed using SYBR (Applied Biosystems) on the Applied Biosystems 7300 Sequence Detection System (Applied Biosystems). All reactions, including the no-template controls, were run in triplicate. After the reactions, the CT values were determined using the fixed threshold settings. 
Absolute Quantification: To calculate the absolute expression levels of the target siRNAs, a series of synthetic siRNA oligonucleotides (dissolved in water) of known concentrations (from 1 fM to 105 fM) were also reverse-transcribed and amplified. The absolute amount of each siRNA was then calculated by referring to the standard curve.
Relative Quantification: To determine the relatively increasing or decreasing level of the target mRNA, U6 and β-actin were also reverse-transcribed and amplified, used for qRT-PCR analysis. The change of target mRNA was then determined by the comparison between U6 and mRNA, or between β-actin and mRNA.
qPCR
Function: To amplify and simultaneously detect or quantify a targeted DNA molecule.
Step:
1. According to the manufacturer’s instruction, use tubes to compound these reagent that listed below. All procedures were carried out on ice.
2. Set the PCR as requested below. Then, start to reverse-transcribe the target RNA 
Recombinant E. coli fermentation
Recombinant E.coli is fermented on M9 minimal medium（6 g/L Na2HPO4，3 g/L KH2PO4，1 g/L NH4Cl，0.5 g/L NaCl，1 mM MgSO4，2%glucose):
Pick recombinant E.coli single colonies into liquid LB medium, shaking overnight to saturation under 37℃.
Transfer 1％ bacteria into liquid M9 medium, shaking 4-5 hours under 37℃ to OD600 around 0.6.
Add inducer to induce the expression of target gene.
The extraction and derivatization of total lipid in E.coli
Extract total lipid inside E.coli using chloroform/methanol extraction method:
Take 200ml fermentation liquor, collect the precipitation of bacteria after inducement through centrifugation. Resuspend with 3ml ddH2O. Add 5ml extraction agent(CHCl3:CH3OH=2:1，v/v),extracting. Vortex for 3 min.
Centrifuge for 5 min, 1000 rpm. Collect the organic phase in the substratum.
Dry the organic phase with N2 in the pressure blowing concentrator. The remaining yellow liquid is the total lipid in the cell.
Add 100µL C20 fatty acid solution resolved in chloroform into the dried total lipid. Dry it with N2 in the pressure blowing concentrator.
Add 2ml methyl agent（H2SO4:CH3OH=1:100，v/v）,reacting for 1 hour under 70℃ in water bath.
Extract the fatty acid methyl ester using 2ml n-hexane. Test 1ml of the upper solution using gas phase or quality detection.
GC-MS determining the component of fatty acid
Use the Thermo Fisher ITQ 1100 GC-MS to determine the component of fatty acid sample:
Chromatographic column:HP-5ms packed column（30 m，0. 25 mm，0. 25μm, packed with 5%Diphenyl polysiloxane copolymer）;
Injection volume:1 μl；
Split ratio:30:1；
Vaporizing chamber temperature:300℃
Flow rate of carrier gas(high purity nitrogen):1 ml/min；
Temperature programing: initial temperature: 100℃, keep 2 min, rise to 280℃ at the rate of 10 °C/min, keep 3 min;
Ion source temperature: 290 ℃
EI detector: 70eV
GC determining the content of fatty acid
Use the Varian GC-450 system to determine the ratio of different fatty acid in fatty acid methyl ester:
Chromatographic column:HP-5 packed column（30 m，0. 25 mm，0. 25μm）;
Injection volume:1 μl；
Split ratio:10:1；
Vaporizing chamber temperature:300℃
Flow rate of carrier gas(high purity nitrogen):1 ml/min；
Temperature programing: initial temperature: 100℃, keep 2 min, rise to 250℃ at the rate of 10 °C/min, keep 3 min;
Ion source temperature: 290 ℃
FID detector temperature: 250℃
Nanjing University
Address：No.22 Hankou Rd.,Gulou District, Nanjing, Jiangsu 210093,P.R.China.
TEL:(+86)15261874993
Email:NJU_QIBEBT@outlook.com
Retrieved from " http://2014.igem.org/Team:NJU-QIBEBT/wetlab/Protocol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
