Team:London Biohackspace/protocols/egassembly
Project
DIY-Brewkit
Kit Product Design
BioBricks
Lab Book
Collaborations
About Us
Policy & Practices
Human Practice
Events
People
Sponsors
Lab Book
Protocols
SLiCE Ex vivo DNA assembly
Introduction
We developed a DNA assembly system purely based on parts homology, which only uses E. coli lysate to carry out the reaction. Our work builds on the previous research on lysate based assembly methods, in the particular SLiCE and Ex vivo . The concept is very similar to that of a Gibson method: the parts to be assembled contain an overlapping homology region, which allows homologous recombination to occur. While the Gibson assembly utilises an expensive piece of kit, containing a 3' to 5' exonuclease, a DNA polymerase to fill the gaps and a ligase to seal the nick. The Ex vivo, as we like to call it "E.G., or E. coli gratiae" only uses E. coli lysate to carry out this reaction. The lysate in fact does contain all the cellular machinery necessary to recognise a homology and to repair DNA. This process if facilitated when the lysate contains three lambda proteins, which can be easily expressed in the strains used to produce it. In addition to normal lysate, this system was tested using a lysate of cells expressing lamda proteins. These are the same protein that allow Lambda Red Recombineering Knock-Outs, i.e. Gam, Exo and Beta, which respectively protect linear DNA from RecBCD nuclease activity, cleave DNA 3' to 5' and promote annealing of complementary single strands.
We decided to test the efficacy of SLiCE for the assembly of parts only based on a short flanking homology. This homology is roughly equivalent to that of the biobrick prefix and suffix. This means that a two part assembly, of the insert - such as a gblock - into the standard pSB1C3 vector would only require one Seamless Ligation step. This avoids the standard (and costly!) digestion/ligation steps that usually are required for the biobrick assembly.
We tested this approach by ligating the standard J04450 RFP generator to pSB1C3 using 22bp and 21bp of homology (biobrick prefix and suffix :) ) and successfully achieved pink, ligated colonies.
Materials and methods
SLiCE & SPLiCE reactions we used
Standard SLiCE reaction:
- 50–200 ng linear vector
- Appropriate amount of insert DNA in a 1 : 1 to 10 : 1 molar ratio of insert to vector,
- 1 ul 10X SLiCE buffer (500mMTris–HCl (pH 7.5 at 25?C), 100mM MgCl2, 10mM ATP, 10mM DTT)
60 minutes at 37°C
Quick Reaction with Quick T4 Ligase Buffer (50ul reaction):
- 50–200 ng linear vector
- Appropriate amount of insert DNA in a 3:1 molar ratio of insert to vector                             - 25ul of 2X T4 ligase buffer (66mM Tris-HCL. 10mM MgCl2, 1mM Dithiothreitol, 1mM ATP, 7.5% Polyethylene glycol (PEG6000), pH 7.6 @ 25°C)
30 minutes at 37°C
Quick SPLiCE (10ul or 50ul reaction):                              - 50-200ng linear vector
- Appropriate amount of insert DNA in a 3:1 molar ratio of insert to vector
- 1ul of E. coli lysate (see below for preparation)
- 1ul of T4 ligase buffer (1x = 50mM Tris-HCl, 10mM MgCl2, 1mM ATP, 10mM DTT, pH 7.5@25°C)
- 1ul PEG 8000 (3% final concentration)
10 minutes at 37°C
Preparing the lysate (lambda protein express)
- Transform cells with plasmid pKD46 (or use strain with genomic copy e.g. EcNR2)
- Pick colony and grow O/N colture
- Dilute 1:100 in 50ml (for 5ml of lysate, 1ml is 1000 reactions)
- Grow to OD 1.0 (tips for DIY OD measurement here )
- Spin 50ml for 10 minutes at 400g and resuspend in 5ml of Tris-NaCl 10mM, transfer in eppendorf tubes
- Spin the 1ml aliquots 8000g for 1 min and remove supernatant
- [We then flash froze the pellets and continued the reaction a month later, we suggest not to wait a month and skip to the next step]
- Resuspend the cells in 100ul of lysis buffer for 10 minutes: we used B-Per by thermo fisher - any equivalent will do.                             - Spin 15000g for 5 minutes
- Carefully removed the supernatant and mix with 100ul of sterile glycerol to a final volume of 50% glycerol
- It should maintain efficiency after months in -20 according to the original papers
- We call it Ex-vivoase/Smashcoli/Splicease, your call :)
Note that this is the protocol we used for the first test and it was done in a UCL lab, this is easily optimisable and we are working on making it more hackspace friendly.
Amplifying the parts
Parts used
For all our reactions we used the following parts
- Insert BBa_J04450 amplified from the 2014 distribution part, originally in pSB1K3 backbone (50ng/ul)
Amplified with the following primers FWD: 5' GTTCTTCGAATTCGCGGCCGCTTCTAGAGAAAGAGGAGAAATACTAGATGGCTTCC 3' REV: 5' GACTGCAGCGGCCGCTACTAGTA 3'
- Linear pSB1C3 backbone amplified from BBa_K554001 including the biobrick prefix and suffix (100ng/ul)
Amplified with the following primer FWD: 5'TACTAGTAGCGGCCGCTGCAGTCCG3' REV: 5'CTCTAGAAGCGGCCGCGAATTCCAG3'
Results
For our first attempt we used the Quick 50ul reaction from above and a higher concentration of insert (10:1), electroporated using the second protocol (see electroporation protocol) using electrocompetent cells kindly donated by the Birkbeck iGEM team. The reaction was a success and we had pink colonies on every plate!
. For a first dry run we call it a success :)
We obtained 1 colony in the plate containig 50ul of recovered cells after electroporation, 3 colonies on the 100ul and 11 colonies in the 350ul. We used this protocol to successfully make a new part BBa_K1845000 which we submitted to the registry.
Following this initial protocol we tweaked the reaction protocols to optimise 10ul reactions and test the use of PEG to speed up the reaction. We used the "Quick SPLiCE" reaction protocol, comparing +/- PEG8000. PEG 8000 is a crowding agent that by reducing the amount of free water in the system (used for the solvation shell of PEG) increases the effective concentration of the reaction components. We used 3ul of the SPLiCE/SLiCE reaction for a total of 30ng of vector DNA (100ng in the 10ul reaction) and plated all the cells (500ul from 50ul of competent cells resuspended in 450ul of 2xTY). Control had water in place of PEG. A picture of the colonies and the colony count is displayed below.
Red colonies
White colonies
+PEG
750
18
+PEG
590
30
-PEG
390
2
-PEG
280
11
Discussion
We are excited of having developed this new assembly method into the framework of the biobrick assembly. All the parts formed are compatible with RFC10 and contain the correct restriction sites. This method only utilises 21 bases of homology, allowing to assemble DNA parts already containing the biorick prefix and suffix, hence without changing the design or established standard. The parts created are ultimately equivalent to a Gibson reaction, with the advantage of utilising E. coli cell lysate instead of an expensive highly purified reaction mixture. We believe that this method will be of great utility for the DIYbio community, and we are now testing it in a variety of conditions. This will make sure to obtain cross-reproducibility and an easy to use method - exponentially cuttind down the cost of DNA assembly for an amateur and professional audience.
We successfully assembled a biobrick part using the SLiCE method and submitted it to the iGEM registry. We optimised this reaction to speed up the ligation using PEG8000, cutting down the reaction time to a 10 minute one pot reaction. We called this assembly "SPLiCE" which also stands for "Seamless PEG-mediated Ligation Cell Extract" - with the "P" also conveniently standing for "Presto!"
Work is currently under development for the optimisation of the reaction conditions to obatin an efficient one-pot-one-biobrick assembly of linear fragments, such as gblocks and linear pSB1C3. We aim to additionally cut down the costs and simplify the preparation of the reaction, allowing a shared assembly method based on
Read more
About us
The DIY-Brewkit
BioBricks
Lab Book
Human Practice
Events
Our team
Our sponsors
e-mail: hello@biohackspace.org
London Biological Laboratories
447 Hackney Road
London
E2 9DY
