<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NUS_Singapore_Notebook_Group1 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NUS Singapore/Notebook/Group1</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Interactive Points | Codrops</TITLE></P><DIV id="st-container" class="st-container"><DIV class="st-pusher"><HEADER id="ha-header" class="ha-header ha-header-large"><DIV class="ha-header-perspective"><DIV class="ha-header-bottom" style="text-align: center;"><A href="https://2016.igem.org/Team:NUS_Singapore" style="color: black; font-weight: lighter;">iGEM 2016 ¬†¬†¬†¬† NUS SINGAPORE</A></DIV></DIV></HEADER><NAV class="st-menu st-effect-1 list-group panel" id="menu-1" style="overflow: auto; z-index: 100; background-color: #E59500;"><SPAN><H4 style=" left: 75px; top: 25px; position: absolute; font-size: 17px; color: white;">NUS_SINGAPORE</H4></SPAN><UL><LI><A class="icon icon-home" id="menu-item-1" href="https://2016.igem.org/Team:NUS_Singapore">Home</A></LI><LI><A class="icon icon-project" onclick="growDiv('2')" id="menu-item-2" href="https://2016.igem.org/Team:NUS_Singapore/Project">Project
								</A><DIV class="grow" id="grow-2"><DIV class="measuringWrapper"><A href="https://2016.igem.org/Team:NUS_Singapore/Project/Wet_Lab" class="list-group-item" style="color: rgb(179, 179, 179);">Overview</A><A href="https://2016.igem.org/Team:NUS_Singapore/Model" class="list-group-item" style="color: rgb(179, 179, 179);">Modelling</A></DIV></DIV></LI><LI><A class="icon icon-notebook" id="menu-item-9" onclick="growDiv('9')" href="https://2016.igem.org/Team:NUS_Singapore/Notebook">RIOT Sensor
								</A><DIV class="grow" id="grow-9"><DIV class="measuringWrapper"><A href="https://2016.igem.org/Team:NUS_Singapore/Description" class="list-group-item" style="color: rgb(179, 179, 179);">Description</A><A href="https://2016.igem.org/Team:NUS_Singapore/Proof" class="list-group-item" style="color: rgb(179, 179, 179);">Proof of Concept</A><A href="https://2016.igem.org/Team:NUS_Singapore/Demonstrate" class="list-group-item" style="color: rgb(179, 179, 179);">Demonstration</A></DIV></DIV></LI><LI><A class="icon icon-notebook" id="menu-item-5" href="https://2016.igem.org/Team:NUS_Singapore/RIOTResponder">RIOT Responder</A></LI><LI><A class="icon icon-notebook" id="menu-item-5" href="https://2016.igem.org/Team:NUS_Singapore/RIOTInvader">RIOT Invader</A></LI><LI><A class="icon icon-practice" onclick="growDiv('3')" id="menu-item-3" href="#">Human Practices
								</A><DIV class="grow" id="grow-3"><DIV class="measuringWrapper"><A href="https://2016.igem.org/Team:NUS_Singapore/HP/Silver" class="list-group-item" style="color: rgb(179, 179, 179);">Survey</A><A href="https://2016.igem.org/Team:NUS_Singapore/HP/Gold" class="list-group-item" style="color: rgb(179, 179, 179);">Business Plan</A><A href="https://2016.igem.org/Team:NUS_Singapore/Integrated_Practices" class="list-group-item" style="color: rgb(179, 179, 179);">Integration</A></DIV></DIV></LI><LI><A class="icon icon-collaboration" onclick="growDiv('4')" id="menu-item-4" href="#">Collaborations
       </A><DIV class="grow" id="grow-4"><DIV class="measuringWrapper"><A href="https://2016.igem.org/Team:NUS_Singapore/Collaborations#MELBOURNE" class="list-group-item" style="color: rgb(179, 179, 179);">MELBOURNE</A><A href="https://2016.igem.org/Team:NUS_Singapore/Collaborations#HONG_KONG_HKUST" class="list-group-item" style="color: rgb(179, 179, 179);">HONG_KONG_HKUST</A><A href="https://2016.igem.org/Team:NUS_Singapore/Collaborations#HONG_KONG_HKU" class="list-group-item" style="color: rgb(179, 179, 179);">HONG_KONG_HKU</A></DIV></DIV></LI><LI><A class="icon icon-notebook" id="menu-item-5" href="https://2016.igem.org/Team:NUS_Singapore/Parts">Parts Registry</A></LI><LI><A class="icon icon-safety" id="menu-item-5" href="https://2016.igem.org/Team:NUS_Singapore/Safety">Safety</A></LI><LI><A class="icon icon-team" id="menu-item-6" href="https://2016.igem.org/Team:NUS_Singapore/Team">Team</A></LI><LI><A class="icon icon-notebook" id="menu-item-7" onclick="growDiv('7')" href="https://2016.igem.org/Team:NUS_Singapore/Notebook">Notebook
								</A><DIV class="grow" id="grow-7"><DIV class="measuringWrapper"><A href="https://2016.igem.org/Team:NUS_Singapore/Notebook/Group1" class="list-group-item" style="color: rgb(179, 179, 179);">I. RIOT Sensor</A><A href="https://2016.igem.org/Team:NUS_Singapore/Notebook/Group2" class="list-group-item" style="color: rgb(179, 179, 179);">II. RIOT Responder</A><A href="https://2016.igem.org/Team:NUS_Singapore/Notebook/Group3" class="list-group-item" style="color: rgb(179, 179, 179);">III. RIOT Invader</A></DIV></DIV></LI><LI><A class="icon icon-acknowledgement" id="menu-item-8" href="https://2016.igem.org/Team:NUS_Singapore/Attributions">Attributions</A></LI><LI><A class="icon icon-cup" id="menu-item-8" href="https://2016.igem.org/Team:NUS_Singapore/Medals">Medals</A></LI></UL></NAV><DIV class="demos"><A class="current-demo" href="index.html">Default Demo</A><A href="index2.html">Perspective Demo (Webkit only)</A></DIV><H2 class="ss-subtitle">Wet Lab Notebook: RIOT SENSOR</H2><DIV id="ss-links" class="ss-links" style="z-index: 99;"><A href="#st-container">Jun</A><A href="#july">Jul</A><A href="#august">Aug</A><A href="#september">Sep</A><A href="#october">Oct</A></DIV><DIV id="ss-container" class="ss-container" style="background-color: white;"><DIV class="ss-row"><DIV class="ss-left"><H2 id="may" class="">Jun</H2></DIV><DIV class="ss-right"><H2>2016</H2></DIV></DIV><DIV class="ss-row ss-large"><DIV class="ss-left"><DIV class="log-content ss-circle-deco"><P>Each RIOT sensor consists of a promoter with RBS, followed by a coding sequence and a terminator.</P><P>The promoters with strong RBS (BBa_B0034) consists of:
                    <OL><LI>p70: obtained from team ETH_Zurich. It is their favourite performer in terms of sensitivity to lactate (BBa_K822000).</LI><LI>p62: a wild-type lldPRD operon promoter from E. coli (BBa_K822000)</LI><LI>J23100: a high expression promoter in a high constitutive cassette (BBa_K314100)</LI></OL></P><P>We obtained the coding sequence of ALR from the Biobrick plasmid (BBa_K1172901). The coding sequence of sfGFP and the sequence of lambda t0 terminator were received from  team ETH_Zurich.</P><P>Each individual components including the promoter, the coding sequence (ALR) and the terminator is isolated and amplified by PCR. After obtaining all the individual components, we ligated them by PCR overlap or 3A Assembly (Fig 1). A second stop codon was added at the end of ALR sequence to enhance the efficiency of the translational termination.</P></DIV><DIV class="log-content ss-circle-deco" style="margin-top: 20px;"><P style="text-align: center;">Fig1. General cloning strategy for RIOT sensors</P></DIV></DIV><DIV class="ss-right"><H3><SPAN>1 June, 2016</SPAN><A>Milestone 1: Create a general strategy for synthesizing RIOT sensors</A></H3></DIV></DIV><DIV class="ss-row"><DIV class="ss-left"><H2 id="july" class="offset">July</H2></DIV><DIV class="ss-right offset"><H2>2016</H2></DIV></DIV><DIV class="ss-row ss-large"><DIV class="ss-left"><H3><SPAN>11 July, 2016</SPAN><A>Milestone 2: Cloning of promoter p70 with Alanine Racemase (ALR)</A></H3></DIV><DIV class="ss-right ha-waypoint" data-animate-down="ha-header-rotate" data-animate-up="ha-header-rotateBack"><DIV class="log-content ss-circle-deco"><P>The p70 fragment with overhang sequence of ALR (Fig 2) was synthesized using PCR with Pfu polymerase and primers ALR 21 and ALR 34. The annealing temperature was 57.9oC and the extension time was 1 min.</P><P style="text-align: center;">Fig2. Introduction of overhangs bases on the p70 lactate sensitive promoter by PCR.</P><P>The expected size of p70-Alanine Racemase (ALR) overhang sequence is 196 base pairs (bp). Lanes 1 marks DNA ladder. Lanes 4 and 6, which are replicates of the same reaction, are DNA product of the expected PCR size of 70-ALR overhang band.</P><P>PCR amplification of ALR gene (Fig 3) was performed with Q5 polymerase and ALR 35 (forward primer) and ALR 24 (Reverse primer)</P><DIV style="float: left; display: inline-block; padding: 20px 0px 20px 0px;"><SPAN style="float: right; width: 55%;"><P>Fig 3. PCR amplification of ALR gene from biobrick plasmid (BBa_K1172901). The expected size of ALR is 1080 bps. Lane 1 marks the DNA ladder. Lanes 2-3 shows a PCR product size slightly above the 1000 bps band  and they are just replicates of the same reaction.</P></SPAN></DIV><P>Stitching of p70 promoter to ALR was performed by PCR overlapped with Q5 polymerase and ALR 21 (forward primer) and ALR 24 (Reverse primer) as illustrated in the general cloning strategy (Fig 1).</P><DIV style="float: left; display: inline-block; padding: 20px 0px 20px 0px;"><SPAN style="float: right; width: 55%;"><P>Fig 4. Stitching of the p70 promoter ligated with ALR (p70-ALR) by PCR. The expected size is 1276 bps. Lanes 1 marks DNA ladder. Lanes 3-5 are PCR products of 70-ALR fragment from replicate PCR reactions.</P></SPAN></DIV></DIV></DIV></DIV><DIV class="ss-row ss-large"><DIV class="ss-left"><DIV class="log-content ss-circle-deco"><P>t0 lambda terminator was amplified by PCR using Pfu polymerase, along with primers ALR 25 and ALR 26.</P><DIV style="float: left; display: inline-block; padding: 20px 0px 20px 0px;"><SPAN style="float: right; width: 55%;"><P>Fig 5. PCR amplification of t0 lambda terminator. After this reaction, terminator fragment was lengthen using ALR 32 as reverse primer which contained an overhang of BB suffix sequence. The expected size of the terminator is 136 bps. Lanes 1 marks DNA ladder. Lanes 2-6 showed the expected size PCR products of the terminator and are replicate PCR reactions.</P></SPAN></DIV></DIV></DIV><DIV class="ss-right"><H3><SPAN>18 July, 2016</SPAN><A>Milestone 3: Extraction of Terminator gene segment</A></H3></DIV></DIV><DIV class="ss-row ss-large"><DIV class="ss-left"><H3><SPAN>19 July, 2016</SPAN><A>Milestone 4: Synthesis of Construct 1 (p70-34-ALR-Terminator)</A></H3></DIV><DIV class="ss-right"><DIV class="log-content ss-circle-deco"><OL><LI>PCR overlap to synthesize construct 1 using Q5 polymerase, along with primers ALR 21 and ALR 38</LI><LI>Gel extraction, construct 1 and plasmid vector, pSB1C3 were digested by restriction enzymes (XbaI and PstI-HF) for 3 hours at 37 oC, followed by heat inactivation at 80oC for 20 minutes.</LI><LI>PCR clean up for construct 1 and obtained 17.9 ng/ùúál.</LI><LI>Gel extraction for the cut vector and obtained 13.8 ng/ùúál.</LI><LI>Insert and vector were  ligated overnight  at 16oC. The ratio of insert:vector was 3:1.</LI><LI>The ligation mix (5 ùúál) was used to transform 50 ùúál of competent bacteria (E. coli DH5).</LI></OL><DIV style="float: left; display: inline-block; padding: 20px 0px 20px 0px;"><SPAN style="float: right; width: 55%;"><P>Fig 6. Stitching of p70-ALR (milestone 2) and t0 lambda terminator (milestone 3) by PCR. The expected size is 1422 bps. Lanes 1 marks DNA ladder. Lane 2 is PCR product of p70-34-ALR-Terminator</P></SPAN></DIV></DIV></DIV></DIV><DIV class="ss-row ss-large"><DIV class="ss-left"><DIV class="log-content ss-circle-deco"><DIV style="float: left; display: inline-block; padding: 20px 0px 20px 0px;"><SPAN style="float: right; width: 55%;"><P>Fig 7. Synthesis of construct 2, p62-34-ALR-Terminator. Construct 2 was synthesized by PCR overlap like construct 1. After cloning, a selected bacterial colony was selected for miniprep to obtain purified plasmids. Then, purified plasmids were digested by restriction enzymes to confirm the size of inserted construct 2. Lane 1 marks the DNA ladder. Lane 2 is digested products by restriction enzymes XbaI and PstI-HF. The top band is the pSB1C3 vector with the expected size of 2052 bps. The below band is the construct 2 insert which is 1382 bps. Lane 3 is digested product by restriction enzyme PstI-HF. This reaction generated linear plasmids with the expected size of 3427 bps which was the result of addition of construct 2 insert into the pSB1C3 vector </P></SPAN></DIV><DIV style="float: left; display: inline-block; padding: 20px 0px 20px 0px;"><SPAN style="float: right; width: 55%;"><P>Fig 8. Synthesis of construct 3, J23100-34-ALR-Terminator. The promoter region (BBa_K314100) was ligated to Alanine Racemase sequence (ALR, BBa_K1172901) by 3A assembly to generate J23100-ALR fragment. Then, J23100-ALR was ligated to and t0 lambda terminator by PCR overlap. The expected size is 1747 base pairs (bp). Lanes 1 marks DNA ladder. Lanes 2-7, which are replicates of the same reaction, show PCR products of J23100-34-ALR-Terminator.</P></SPAN></DIV></DIV></DIV><DIV class="ss-right"><H3><SPAN>31 July, 2016</SPAN><A>Milestone 5: Synthesis of constructs 2 (p62-34-ALR-Terminator) and 3 (J23100-34-ALR-Terminator) for characterization purposes</A></H3></DIV></DIV><DIV class="ss-row"><DIV class="ss-left"><H2 id="august" class="offset">Aug</H2></DIV><DIV class="ss-right offset"><H2>2016</H2></DIV></DIV><DIV class="ss-row ss-large"><DIV class="ss-left"><DIV class="log-content ss-circle-deco"><OL><LI>Measure OD of each overnight cultures of RIOT sensor 1, 2 and 3 after 10x dilution: add 900 ml of LB into a cuvette and blank, then add 1 ml of overnight culture, mix and take measurement. Record the corrected OD</LI><LI>Prepare 5 ml of M9 (with glucose added) in a 15 ml tube for each constructs</LI><LI>Calculate the volume of stock culture need to be added into 5 ml of M9 to obtain the starting OD of 0.1</LI><LI>Aliquot the required volume of stock culture and centrifuge at 6000 rpm for 1 min. Resuspend with M9.</LI><LI>Add 2 ml of D-Alanine (50 mg/ml) into 3 wells of a 96-well plate for each constructs as a triplicate.</LI><LI>Add 2 ml of each lactate concentration, 1 M, 10-1 M and 10-2 M into 3 wells for each constructs. The final working lactate concentrations will be 10-2 M, 10-3 M and 10-4 M.</LI><LI>Dispense 200 ml of each constructs into each well with triplicate.</LI><LI>Dispense 200 ml of M9 into 3 wells and 200 ml of water into 3 wells for blanking.</LI><LI>Place the 96-well plate into the plate reader for 6-8 hours at 37oC.</LI><LI>Measure OD600 at 15 minutes interval.</LI></OL><P>All sensors seemed to have similar growth rate regardless of the lactate concentration. This means they are very leaky and required modifications before integrating into the whole RIOT system.</P></DIV></DIV><DIV class="ss-right"><H3><SPAN>25 August, 2016</SPAN><A>Milestone 6: Characterization of RIOT Sensors 1, 2 and 3</A></H3></DIV></DIV><DIV class="ss-row ss-large"><DIV class="ss-left"><H3><SPAN>15 August, 2016</SPAN><A>Milestone 7: Synthesis of p62, p70 and J23100 constructs with varying RBS strengths for further characterization</A></H3></DIV><DIV class="ss-right"><DIV class="log-content ss-circle-deco"><P>Upon characterization, it appeared that the promoters with strong ribosomal binding site (RBS, BBa_B0034) exhibited high leakiness. Therefore, we designed this series of constructs by replacing the strong RBS by the medium (BBa_B0032: 30% strength of B0034) and weak RBS (BBa_B0033: 1% strength of B0034). Each promoter fragment p70, p62 and J23100 (without RBS) were isolated and amplified by PCR from the construct 1, 2 and 3 respectively. Each promoter fragment (without RBS) was ligated to the ALR-Terminator fragment by PCR overlap using primers having the overhang sequence of either medium or weak RBS.</P><P>Fig 9. Synthesis of constructs 4-9.
                                <UL><LI>Construct 4: p70-32-ALR-Terminator</LI><LI>Construct 5: p62-32-ALR-Terminator</LI><LI>Construct 6: J23100-32-ALR-Terminator</LI><LI>Construct 7: p70-33-ALR-Terminator</LI><LI>Construct 8: p62-33-ALR-Terminator</LI><LI>Construct 9: J23100-33-ALR-Terminator</LI></UL></P><OL><LI>PCR amplification of construct 4, 5 and 8. Their expected size is 1404 base pairs (bp). Lane 1 marks the DNA ladder. Lane 2-5, 6-8 and 10-13, which are replicates of the 3 reactions, show PCR products of construct 4, 5 and 8</LI><LI>PCR amplification of construct 6, 7 and 9. The expected size of construct 7 is 1404 bps. The expected size of construct 6 and 9 is 1768 bps. Lane 1 marks the DNA ladder. Lane 2-5, 6-8 and 10-13, which are replicates of the 3 reactions, show PCR products of construct 6, 7 and 9.</LI></OL></DIV></DIV></DIV><DIV class="ss-row"><DIV class="ss-left"><H2 id="september" class="offset">Sep</H2></DIV><DIV class="ss-right offset"><H2>2016</H2></DIV></DIV><DIV class="ss-row ss-large"><DIV class="ss-left"><H3><SPAN>19 September, 2016</SPAN><A>Milestone 8:  Characterization of RIOT Sensors 4-9</A></H3></DIV><DIV class="ss-right"><DIV class="log-content ss-circle-deco"><P>We performed the characterization process same as what we did with RIOT sensors 1-3 in milestone 6.</P><UL><LI>RIOT sensors 4-6 did not show much improvement compared to sensors 1-3. They are also leaky and hence, not suitable to be integrated into the whole system.</LI><LI>RIOT sensors 7-9 showed encouraging results since they were more sensitive to small changes in lactate concentration.</LI></UL></DIV></DIV></DIV><DIV class="ss-row ss-large"><DIV class="ss-left"><DIV class="log-content ss-circle-deco"><P>After evaluating the sensitivity to lactate of each combination of promoter and ribosomal binding site (RBS), we chose the weak RBS to synthesize RIOT sensor 10 and 11 which contain sfGFP in place of Alanine Racemase to make them become a potential diagnostic device to detect pathogenic cells.</P><P style="clear: both;">Fig 10. Synthesis of constructs 10 and 11.</P><OL><LI>Construct 10: p70-33-sfGFP-Terminator. The expected size is 1056 base pairs (bp). Lane 1(7) marks the DNA ladder. Lane 4-7(1-4) are replicates of the same PCR reaction.</LI><LI>Construct 11 : p62-33-sfGPP-Terminator. The expected size is 1056 bps.  Lane 7(1) marks the DNA ladder. Lane 1-4(4-7) are replicates of the same PCR reaction.</LI></OL></DIV></DIV><DIV class="ss-right"><H3><SPAN>14 September, 2016</SPAN><A>Milestone 9: Synthesis of construct 10 (p70-33-sfGFP-Terminator) and 11 (p62-33-sfGFP-Terminator)</A></H3></DIV></DIV><DIV class="ss-row"><DIV class="ss-left"><H2 id="october" class="offset">Oct</H2></DIV><DIV class="ss-right offset"><H2>2016</H2></DIV></DIV><DIV class="ss-row ss-large"><DIV class="ss-left"><H3><SPAN>1 October, 2016</SPAN><A>Milestone 10: Characterization of RIOT Sensors 10 and 11</A></H3></DIV><DIV class="ss-right"><DIV class="log-content ss-circle-deco"><P>Culture mammalian cell lines HeLa and HepG2</P><OL><LI>Revive cells from frozen vials and subculturing  80% confluent</LI><LI>Enumerate cell density by a hemocytometer. Seed 106 cells into each well.</LI><LI>Resuspend with phenol free DMEM and incubate at 37 oC for 6 hours, then remove media and wash with 1X PBS.</LI><LI>Add 500 Œºl of new media (DMEM which is free of phenol and serum) and incubate at at 37oC for 18 hours.</LI><LI>Use a pipette to remove the cell supernatant separately and centrifuge at 250 rcf for 3 min to remove cells that may have left in the supernatant.</LI><LI>Cell-free supernatant is used in GFP microscop.</LI></OL><P>GFP Microscopy</P><OL><LI>Set up an overnight culture of bacteria in a culture tube, with 5mL of LB, plus the appropriate antibiotics. Incubate  in a shaking incubator at  37oC overnight.</LI><LI>Measure the OD600 of the resuspended bacteria culture. Aliquot appropriate volume so that starting OD for each tube is  0.4, shake in the 37oC incubator for approximately 30 minutes.</LI><LI>Set up five fresh and labelled 1.5Œºl microcentrifuge tubes. To each tube, add 20Œºl of each lactate concentration (0 M, 10-3 M, 10-2 M)  and 180Œºl of the diluted bacteria sample to make up 200Œºl of reaction mix. To the remaining two tubes, add 40Œºl of each cell supernatant (Hela and HepG2) and 10Œºl of the diluted bacteria sample to make 160Œºl of reaction mix, with 5x dilution of the samples.</LI><LI>Incubate in a shaking  incubator at  37oC incubator for 3 to 4 hours.</LI><LI>Add 5Œºl of each sample onto the appropriately labelled polylysine slide and cover with the slip and capture images using NIS microscope.</LI></OL><P>We found that different lactate concentration in the media or HeLa/HepG2‚Äôs supernatant induce different level of GFP expression. RIOT sensor 10 appeared to be the most sensitive sensor to lactate.</P></DIV></DIV></DIV><DIV class="ss-row ss-large"><DIV class="ss-left"><DIV class="log-content ss-circle-deco" style="margin-top: 20px;"><P>Materials: harvest supernatants of HeLa and HepG2 as shown in Milestone 10 and overnight culture of bacteria</P><P>Centrifuge bacterial cell culture at 5000 rpm for 10 minutes and remove supernatant for the assay</P><P>Each sample was 100x diluted. 1 Œºl of diluted supernatant of HeLa, HepG2 and 5 Œºl of diluted bacterial supernatant grown in LB  and the cell media, DMEM were added into the buffer and enzyme mix separately for each well in the 96-well plate. DMEM which is free of phenol red and serum serves as a negative control.</P><P>Carry out the assay as instructed in the protocol of the lactate assay kit from Sigma-Aldrich (catalog number: MAK064)</P></DIV></DIV><DIV class="ss-right"><H3><SPAN>13 Oct, 2016</SPAN><A>Milestone 11: Measure lactate concentration in supernatant of HeLa, HepG2 and bacteria</A></H3></DIV></DIV></DIV></DIV><SECTION class="content content--related"><UL class="copyright"><LI>¬© NUS_Singapore iGEM 2016</LI></UL></SECTION><A href="#" class="go-top">Go To Top</A></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>