"
Page
Discussion
View source
History
teams
Log in
Â 
Team:WashU/Week11
From 2012.igem.org
Home
Projects
Saffron in a Kan
YLC
Modeling
Notebook
Weekly Log
Protocols
Team
Students
Advisors
Official Team Profile
Safety
Attributions
Monday, August 6
Colony PCR of U construct using four new colonies and the same positive Z control has once again resulted in no successful colonies. However, the PCR has reaffirmed that the control works, which is a good sign.
We also did a PCR of the color constructs in order to form our BioPaint set. As these were not succesful, however, we will attempt them anew in the future. In the picture of the gel below, the lanes all contain color constructs. The one potentially successful PCR in the middle was obtained from the wrong DNA.
Since our T plasmid and Z construct ligations have not been going well, we started over by preparing new cultures of Z and T, running them on a gel, and then gel purifying the results.
We began 5 ml cultures for the good Z construct with TOPO and for one of the U ligation colonies. We will use the Z culture for a glycerol stock, and digest the U culture to see if our ligations actually succeded.
Tuesday, August 7
Today, we digested the U ligation construct by cutting it, in two tubes, with AgeI-HI and NcoI. A double cutter, the AgeI-HI would excise out the insert, whereas NcoI cuts once. The results, shown below, indicate that our U ligation is incorrect.
We also PCR'd up the Z construct, U construct, and Z+TOPO constructs using KlenTaq.
Wednesday, August 8
This morning, we ran the three PCRs from yesterday on a gel. The Z and U constructs were successfully PCR'd, but the Z+T construct did not succeed and we only saw primer dimers in that well of the gel. We took the Z and U constructs, digested them with EcoRI (E) and PstI (P), and ran them on a gel. We also digested the chloramphenicol plasmid that we plan to use for submission and ran it on the same gel. Afterwards, we gel purified the three pieces of DNA. We finished off by ligating together the Z construct and C plasmid in one tube and the Z construct and C plasmid in another tube. Tomorrow, we will transform E. coli with our ligation results.
Finally, we started cultures of our Z ligation, control, and a zeaxanthin culture to use tomorrow to test whether our genes are working properly.
Thursday, August 9
We transformed E. coli GC5 cells with the ligation results from yesterday and plated the cells.
In addition, we created a standard curve for crocin and calculated the extinction coefficient from the data. The picture below shows different absorbances of crocin at various concentrations, and then, using a trendline, we calculated the extinction coefficient of crocin. Since the slope is equal to path length multiplied by extinction coefficient, taking 7629 and dividing it by the path length, 0.5 cm, gives an extinction coefficient of 14538 M-1 cm-1.
We started a culture of Z-construct in T plasmid, with the zeaxanthin-producing gene.
Digested PSL2131, PSL2134, and RFP constructs. Cut with X and P. Ran on gel and gel purified. Ligated RFP constructs into plasmids while maintaining homology regions. Transformed GC5s with ligation results. Plated on Kan resistant plates.
We also ran an in vitro assay of crocin production, by adding zeaxanthin at different volumes to induced cultures of zeaxanthin-producing E. coli. However, as we believe our cultures are only producing small volumes of crocin, we were unable to see any difference in the tubes. While this assay may have failed, we will try other techniques to view crocin production.
Friday, August 10
We redid the ligations of Z, U and the C plasmid again.
This morning, we picked a colony from the U ligation plate from yesterday. After growing it up in culture, we miniprepped it. We digested it on a gel and found out that we have the correct ligation. Thus, we will sequence and submit our U construct in submission plasmid Psb1c3.
Back to Weekly Log
Retrieved from " http://2012.igem.org/Team:WashU/Week11 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
