<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Composite_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Composite Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/b/bb/T--Bielefeld-CeBiTec--title-img-parts-basic.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Composite Parts
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><DIV class="article"><H2>Short Summary</H2>
					For the application of our best composite part, we decided to nominate our reporter signal enhancing system <A href="http://parts.igem.org/Part:BBa_K2201373">BBa_K2201373</A>.
					This part contains a T3 RNA Polymerase with an inverted mRFP under T3 RNA polymerase control for the enhancing of reporter signals. It is an improved reporter and a genetic circuit that could report even weak expression levels. This part was designed based on the model of an amplifier in electrical engineering to intensify an existing input signal and could be used in a broad range of synthetic biology applications. We used this part for our selection system for the incorporation of non-canonical amino acids and demonstrate the advantages of our system in comparison with standard reporter in an integrated modelling and wet lab characterizations.
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Usage and Biology</H2><DIV class="article">
						The T3 RNA polymerase is highly specific to T3 promoters and orthogonal to the T7 RNA polymerase. Therefore, it can be widely utilized in synthetic biology, even in expression strains which encode for a T7 RNA polymerase like <I>E. coli</I> BL21. This orthogonality is due to the promoter specifity of the different polymerases (Davis <I>et al., 1971</I>; McGraw <I>et al.</I>, 1985).
						We applied this system to enhance a reporter in genetic circuits to make even weak gene expression visible.
						 
						At the moment, fluorescent proteins with an emission wavelength within the visible spectra are used to report expression of the gene of interest. Therefore, the CDS of the fluorescent protein is placed downstream of the CDS of the target protein without a terminator or promoter in between. The expression level of the target protein is nearly the same as the expression of the fluorescent protein. The fluorescence of the reporter protein indicates if the gene of interest was translated. However, this system is limited to strong expression, which generate a sufficienly strong fluorescence signal. 

						Therefore it is impractical for several applications which involve only weak expression. For our project, we needed a reliable and sensitive reporter to detect the expression of the gene of interest on a selection plasmid. A low expression of the target gene is essential for the selection system. No fluorescence was detectable, when the CDS of mRFP was placed downstream of the gene of interest. Through the function of the gene of interest, we knew it was expressed. To address this reporter challenge we built a genetic circuit following the model of an amplifier used in electrical engineering.

						Basic amplifiers were previously submitted to the Registry of biological parts e. g. by <A href="https://2009.igem.org/Team:Cambridge">iGEM Cambridge 2009</A>. They build a simple circuit using an activator, which increased the transcription of a reporter under control of a second promoter. To explain their system they used the term &quot;Polymerases per second&quot; (PoPs). This unit defined as the flow of RNA polymerase molecules over a promoter region per second. The system developed by Cambridge 2009 (Figure 1) could increase the number of PoPs. </DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 1:</B> Signal strenthening system of <A href="https://2009.igem.org/Team:Cambridge/Project/Amplification">iGEM Cambridge 2009</A>.</P></DIV><DIV class="article">
						To improve this system we decided to use an activator, which translates the reporter DNA and does only induce the translation of the polymerase which then translates the gene of interest. Amplification was achieved by an RNA polymerase, which transcribes the reporter DNA, multiple times and enhances the signal significantly.</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Functional Parameters</H2><DIV class="article">
						To design a genetic circuit that amplifies a reporter signal we decided to use an RNA polymerase, orthogonal to the native <I>E. coli</I> RNA polymerase. Therefore, the T3 RNA polymerase previously characterized by <A href="https://2010.igem.org/Team:Peking">iGEM Peking 2010</A> was deployed. The T3 RNA polymerase is highly specific to T3 promoters and orthogonal to the <I>E. coli </I>DNA polymerases and even to the T7 RNA polymerase. We decided not to use the T7 RNA polymerase, because it is already part of expression strains like BL21 and would therefore prevent the application in such strains. The construction of our designed composite part comprising a standard mRFP reporter is shown in Figure 2.
					</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 2:</B> Construction of the standard mRFP reporter and the genetic circuit for the amplification of mRFP expression. In construct 1 the CDS fot the mRFP transcript is downstream the CDS of the gene of interest. In construct 2 the CDS of the mRFP is downstream a T3 promoter and the CDS coding for the T3 RNA polymerase is downstream the CDS of the gene of interest.</P></DIV><DIV class="article">
						Figure 2 shows two reporter constructs for gene expression quantification. The first one is the standard reporter for the gene expression of the target gene, using mRFP as reporter gene. If the gene of interest is expressed, the mRFP is expressed at nearly the same level. The mRFP fluorescence reveals the gene expression of the gene of interest. This system is suitable for but also limited to high expression rates. Low, expression of the target  gene is associated with a low expression of mRFP and thus not visible. 

						The second system is our genetic circuit consisting of a CDS of the target gene upstream of the CDS for a T3 DNA polymerase encoding sequence. Therefore, the expression of the reporter gene is nearly on the same level as the expression of the target gene. The expressed T3 RNA polymerase transcribes the mRFP under the control of the T3 promoter.
						To demonstrate the advantages of our improved construct, we modelled the amount of mRFP transcript for both constructs.
						If we assume the expression of the gene of interest is low and only one <I>E. coli</I> RNA polymerase with a chain elongation rate of 50 nucleotides per second translates the both products, construct 1 produces 1 mRFP transcript in 16 seconds. Construct 2 expresses 1 T3 RNA polymerase transcript every 52 seconds. After translation (with an average  translation rate of 20 amino acids per second ~42 sec), these polymerases transcribe the mRFP transcript with a chain elongation rate of 170 nucleotides per second. Therefore, every T3 RNA polymerase generates one mRFP transcript every 4.7 seconds. The resulting amount of mRFP transcripts  is shown in Figure 3. The script for our modelling can be found <A href="https://static.igem.org/mediawiki/parts/8/8a/T--Bielefeld-CeBiTec--model-composite-2.txt">here</A>.
					</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 3:</B>Modeling on the amount of mRFP transcript transcribed through the two different models. In model 1 mRFP CDS lies is of the CDS of the gene of interest. In model 2 the CDS of the mRFP is downstream a T3 promoter and the CDS coding for the T3 RNA polymerase is downstream the CDS of the gene of interest.</P></DIV><DIV class="article">
						The number of mRFP transcripts is a lot higher with construct 2 (our reporter constuct) even though model leaves out the fact that one transcript of the T3 RNA polymerase could be translated several times and enhance the signal even more. Despite the simplicity of this model, it shows the advantages of using the T3 RNA polymerase for signal enhancing. 
					</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 4:</B> Two Smear of two clones containing only mRFP under an uninduced T7 promoter (-) and containing the mRFP enhancing system under the same promotor (+), after 12 h of incubation at 37 °C.</P></DIV><DIV class="article">
						To demonstrate the signal enhancing system we cloned either the part <A href="http://parts.igem.org/Part:BBa_E1010">E1010</A> or our signal enhancing system (<A href="http://parts.igem.org/Part:BBa_K2201373">BBa_K2201373</A>) downstream of the part <A href="http://parts.igem.org/Part:BBa_K2201900">BBa_K2201900</A>. This plasmid was characterized in cells containing our positive selection plasmid. A smear of the transformants is shown in Figure 4. The uninduced T7-promoter  promoter leads only to a basal transcription level. Despite this weak transcription, the mRFP in the cells containing the signal enhancing system is clearly visible, while cells without the signal enhancing system remain colorlesse.
					</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 5:</B> Picture of a negative selection round. The clone still containing the positive selection plasmid, thus the mRFP enhancing system, is red.</P></DIV><DIV class="article">
						For the selection of tRNA/aminoacylsynthetase to incorporate non-canonical amino acids, we needed to check if clones still contain the positive selection plasmid in the negative selection round. Therefore, we decided to incorporate the mRFP signal enhancing system downstream of the CDS of our positive selection plasmid <A href="http://parts.igem.org/Part:BBa_K2201900">BBa_K2201900</A>. If the cells contain this plasmid in the negative selection, they should be visible as red clones. In contrast, the clones containing the negative selection plasmid should be colorless. A picture of one round of the negative selection (Figure 5) shows that one clone is red, thus demonstrating the function of the system. The results of our selection system can be found <A href="">here</A>.
						</DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Applications</H2><ARTICLE>
An important criteria for a composite part is the possibility to use this part in further iGEM projects. In addition to our application for this part, there are a lot of potential applications for a reporter signal enhancement. Our system provides a reliable and especially sensitive reporter, thus applications which require these could be improved by using our system. Some possible applications are shown in Figure 6. 
</ARTICLE><DIV class="figure large"><P class="figure subtitle"><B>Figure 6:</B> Potential applications for the reporter signal enhancing system.</P></DIV><ARTICLE>
The strength of the reporter signal enhancing system lies in its strong and specific signal despite of a very weak initial gene expression. This enables usage in the analysis of weakly transcribed genes in metabolic pathway and flux analyses as well as a variety of diagnostic application which are dependent on strong, specific signals. Furthermore use of the reporter signal enhancer system in the analysis of expression <I>in vitro</I> and <I>in vivo</I> is feasible.
</ARTICLE></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Composite Parts</H2><P class="table-headline"><B>Table 1: All composite parts of iGEM Bielefeld-CeBiTec 2017.</B> For a detailed description of the parts, please refer to the <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/Overview">results</A> or corresponding sites in the <A href="http://parts.igem.org/Main_Page">Registry of Parts</A>.</P><TABLE class="parts"><THEAD><TR><TH style="width: 150px;" class="header">Name</TH><TH style="width: 120px" colspan="" class="header">Type</TH><TH style="width: auto;" colspan="" class="header">Description</TH><TH style="width: 150px" colspan="" class="header">Designer</TH><TH style="width: 80px" colspan="" class="header">Length</TH></TR></THEAD><TBODY><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201373">BBa_K2201373</A></TD><TD>Composite</TD><TD>T3 Polymerase with inverted mRFP under T3 promoter control for signal enhancing</TD><TD>Svenja Vinke</TD><TD>3411</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201201">BBa_K2201201</A></TD><TD>Composite</TD><TD>Pyrrosyl tRNA/aminoacyl-synthetase for the incorporation of propargyllysine</TD><TD>Svenja Vinke</TD><TD>1997</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201202">BBa_K2201202</A></TD><TD>Composite</TD><TD>Tyrosyl tRNA/aminoacyl-synthetase for the incorporation of p-acetophenylalanine</TD><TD>Svenja Vinke</TD><TD>1558</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201203">BBa_K2201203</A></TD><TD>Composite</TD><TD>Tyrosyl tRNA/aminoacyl-synthetase for the incorporation of p-acetophenylalanine in response to CUA</TD><TD>Svenja Vinke</TD><TD>1558</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201204">BBa_K2201204</A></TD><TD>Composite</TD><TD>Tyrosyl tRNA/aminoacylsynthetase for the incorporation of L-(7-hydroxycoumarin-4-yl)ethylglycine</TD><TD>Svenja Vinke</TD><TD>1558</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201343">BBa_K2201343</A></TD><TD>Composite</TD><TD>Fusion protein of CFP and YFP with an amber codon in the linker under T7 promoter control</TD><TD>Svenja Vinke</TD><TD>1483</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201024">BBa_K2201024</A></TD><TD>Composite</TD><TD>Left flanking sequence + Terminator + repressing lacO</TD><TD>Lennard Karsten</TD><TD>1248</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201025">BBa_K2201025</A></TD><TD>Composite</TD><TD>Terminator + right flanking sequence </TD><TD>Lennard Karsten</TD><TD>1248</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201028">BBa_K2201028</A></TD><TD>Composite</TD><TD>Repair tempplate for HDR-mediated integration of cas9 into E. coli BL21 (DE3)</TD><TD>Lennard Karsten</TD><TD>3487</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201032">BBa_K2201032</A></TD><TD>Composite</TD><TD>sgRNA MAX target + mRFP-sacB fusion protein</TD><TD>Lennard Karsten</TD><TD>3487</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201040">BBa_K220140</A></TD><TD>Composite</TD><TD>sgRNA R-target mutA</TD><TD>Markus Haak</TD><TD>231</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201041">BBa_K2201041</A></TD><TD>Composite</TD><TD>sgRNA mutG</TD><TD>Markus Haak</TD><TD>231</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201042">BBa_K2201042</A></TD><TD>Composite</TD><TD>sgRNA R-target mut&amp;#916</TD><TD>Markus Haak</TD><TD>231</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201043">BBa_K2201043</A></TD><TD>Composite</TD><TD>sgRNA R-target mutT</TD><TD>Markus Haak</TD><TD>231</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201044">BBa_K2201044</A></TD><TD>Composite</TD><TD>sgRNA R-target mutC</TD><TD>Markus Haak</TD><TD>231</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201045">BBa_K2201045</A></TD><TD>Composite</TD><TD>sgRNA MAX-target mutA</TD><TD>Markus Haak</TD><TD>256</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201046">BBa_K2201046</A></TD><TD>Composite</TD><TD>sgRNA MAX-target mutG</TD><TD>Markus Haak</TD><TD>256</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201047">BBa_K2201047</A></TD><TD>Composite</TD><TD>sgRNA MAX-target mut&amp;#916</TD><TD>Markus Haak</TD><TD>256</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201048">BBa_K2201048</A></TD><TD>Composite</TD><TD>sgRNA MAX-target mutT</TD><TD>Markus Haak</TD><TD>256</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201049">BBa_K2201049</A></TD><TD>Composite</TD><TD>sgRNA MAX-target mutC</TD><TD>Markus Haak</TD><TD>256</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201070">BBa_K2201070</A></TD><TD>Composite</TD><TD>sgRNAs R-target mutA,G</TD><TD>Markus Haak</TD><TD>470</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201071">BBa_K2201071</A></TD><TD>Composite</TD><TD>sgRNAs R-target mutT,C</TD><TD>Markus Haak</TD><TD>470</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201072">BBa_K2201072</A></TD><TD>Composite</TD><TD>sgRNAs R-target A,G,&amp;#916</TD><TD>Markus Haak</TD><TD>709</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201073">BBa_K2201073</A></TD><TD>Composite</TD><TD>sgRNAs MAX-target mutA,G,&amp;#916,T,C</TD><TD>Markus Haak</TD><TD>1187</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201074">BBa_K2201074</A></TD><TD>Composite</TD><TD>sgRNAs MAX-target mutA,G</TD><TD>Markus Haak</TD><TD>520</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201075">BBa_K2201075</A></TD><TD>Composite</TD><TD>sgRNAs MAX-target mutT,C</TD><TD>Markus Haak</TD><TD>520</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201076">BBa_K2201076</A></TD><TD>Composite</TD><TD>sgRNAs MAX-target mutA,G,&amp;#916</TD><TD>Markus Haak</TD><TD>784</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201077">BBa_K2201077</A></TD><TD>Composite</TD><TD>sgRNAs MAX-target mutA,G,&amp;#916,T,C</TD><TD>Markus Haak</TD><TD>1312</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201208">BBa_K2201208</A></TD><TD>Composite</TD><TD>Cysteinyl-lysinyl-tRNA Synthetase/tRNA pair</TD><TD>Daniel Bergen</TD><TD>1860</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201243">BBa_K2201243</A></TD><TD>Composite</TD><TD>Fusion protein of CFP and YFP with an amber codon in the linker</TD><TD>Svenja Vinke</TD><TD>1431</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201250">BBa_K2201250</A></TD><TD>Composite</TD><TD>elastine consensus for PRe-RDL</TD><TD>Daniel Bergen</TD><TD>52</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201251">BBa_K2201251</A></TD><TD>Composite</TD><TD>silk consensus for PRe-RDL</TD><TD>Daniel Bergen</TD><TD>55</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201320">BBa_K2201320</A></TD><TD>Composite</TD><TD>Functional GFP-streptavidin fusion protein with medium gly-gly-ser linker and T7-promoter and RBS</TD><TD>Yannic Kerkhoff</TD><TD>1169</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201321">BBa_K2201321</A></TD><TD>Composite</TD><TD>NM-domain of Sup35 with an amber codon at position 21 under T7 promoter control</TD><TD>Svenja Vinke</TD><TD>837</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201342">BBa_K2201342</A></TD><TD>Composite</TD><TD>Fusion protein of RFP and GFP with an amber codon in the linker under T7 promoter control</TD><TD>Svenja Vinke</TD><TD>1489</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201360">BBa_K2201360</A></TD><TD>Composite</TD><TD>Our part BBa_K2201260</TD><TD>Maximilian Edich</TD><TD>7649</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201361">BBa_K2201361</A></TD><TD>Composite</TD><TD>Our part BBa_K2201261 after the Carboxysome and T7 promoter</TD><TD>Maximilian Edich</TD><TD>6946</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201362">BBa_K2201362</A></TD><TD>Composite</TD><TD>Our part BBa_K2201262 after the carboxysome and T7 promoter</TD><TD>Maximilian Edich</TD><TD>6946</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201363">BBa_K2201363</A></TD><TD>Composite</TD><TD>Our part BBa_K2201263 after the carboxysome and T7 promoter</TD><TD>Maximilian Edich</TD><TD>6946</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201364">BBa_K2201364</A></TD><TD>Composite</TD><TD>Our part BBa_K2201264</TD><TD>Maximilian Edich</TD><TD>6946</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201365">BBa_K2201365</A></TD><TD>Composite</TD><TD>Our part BBa_K2201265 after the Carboxysome and the T7 promoter</TD><TD>Maximilian Edich</TD><TD>6946</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201366">BBa_K2201366</A></TD><TD>Composite</TD><TD>Our part BBa_K2201266 after the Carboxysome and the T7 promoter</TD><TD>Maximilian Edich</TD><TD>6946</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201367">BBa_K2201367</A></TD><TD>Composite</TD><TD>Our part BBa_K2201267 after the Carboxysome and the T7 promoter</TD><TD>Maximilian Edich</TD><TD>6955</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201368">BBa_K2201368</A></TD><TD>Composite</TD><TD>Our part BBa_K2201268 after the Carboxysome and the T7 promoter</TD><TD>Maximilian Edich</TD><TD>6937</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201404">BBa_K2201404</A></TD><TD>Composite</TD><TD>Reverse Sequence of I746909</TD><TD>Olga Schmidt</TD><TD>924</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201900">BBa_K2201900</A></TD><TD>Composite</TD><TD>Positive selection plasmid</TD><TD>Svenja Vinke</TD><TD>1158</TD></TR><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201901">BBa_K2201901</A></TD><TD>Composite</TD><TD>Negative selection plasmid</TD><TD>Svenja Vinke</TD><TD>681</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>References</H2><ARTICLE><B>Davis, R. W. &amp; Hyman, R. W </B>(1971)J. Mol. Biol <B>62</B>, 287-301. <B>McGraw, N. J., Bailey, J. N., Cleaves, G. R., Dembinski, D. R., Gocke, C. R., Joliffe, L. K., MacWright, R. S.&amp;McAllister, W. T.</B> (1985) Nucleic Acids Res. <B>13</B>, 6753–6766.
						<PARTINFO>BBa_K2201373 parameters</PARTINFO></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>