<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_toolbox_fusing skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/toolbox/fusing</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/c/c8/T--Bielefeld-CeBiTec--title-img-fusing.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Fusing
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Short summary</H3><ARTICLE>
		Fusing proteins is normally limited to the C‑ or N‑terminus of a protein. 
		The incorporation of non-canonical amino acids that could be fused to each 
		other or to surfaces enables several additional applications. Examples are the 
		immobilization of proteins as well as improved stability of protein polymer networks. 
		Furthermore, it leads to enhanced efficiency of pathways by combining enzymes of 
		one pathway or for any other system where colocalization is beneficial.
		
		As proof of concept, we work on enhanced stability of a protein polymer. These networks 
		can be applied for different applications like modern biomaterials in medicine and industry 
		(Rnjak-Kovacina <I>et al.</I>, 2011). The amino acids <I>N</I><SUP>ε</SUP>‑L‑cysteinyl‑L‑lysine (CL) 
		and <I>N</I><SUP>γ</SUP>‑2‑cyanobenzothiazol‑6‑yl‑L‑asparagine (CBT‑asparagine) 
		comprise key parts of this tool. Both amino acids can bind specifically to each other resulting in the formation 
		of a covalent bond between their side chains. We plan to use this covalent bond to increase the stability of 
		silk elastin like proteins (SELPs). The strengthened polymer network would be a perfect material to produce 
		biological wound bindings which are very thin and would be able to interact with the natural tissue matrix 
		(Boateng <I>et al.</I>, 2008).
		</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Terminus independent fusion proteins</H3><ARTICLE>
				While terminus dependent binding systems for proteins are already in use, there are only a 
				few systems for terminus independent binding systems. We want to expand the number of those 
				systems. Our aim is to incorporate two non-canonical amino acids, which are able to build a 
				specific bond to each other. According to the synthesis of D-luciferin for the firefly luciferase 
				of <I>Photinus pyralis</I>, we decided to use the specific binding of 1,2‑aminothiols 
				and the cyano group of cyanobenzothiazole (CBT). Figure 1 shows the biosynthesis of luciferin 
				and the mechanism of the binding reaction of 1,2‑aminothiol and CBT.
			</ARTICLE><DIV class="figure large"><P class="figure subtitle"><B>Figure 1: Reaction of the 1,2‑aminothiol of cysteine and 
					CBT to luciferin (Liang <I>et al.</I>, 2010).</B></P></DIV><ARTICLE>
				By synthesis of amino acids with side chains containing CBT and a 1,2‑aminothiol, polypeptides 
				binding to each other should be produced. These amino acids are CL and CBT‑asparagine whose binding 
				mechanism is shown in figure 2.
			</ARTICLE><DIV class="figure large"><P class="figure subtitle"><B>Figure 2: Specific binding reaction of CL and CBT-asparagine.</B></P></DIV><H3><I>N</I><SUP>ε</SUP>-L-cysteinyl-L-lysine</H3><ARTICLE>
				We synthesized CL and CBT‑asparagine in our lab. Additionally we are providing the community with a validated <A href="https://static.igem.org/mediawiki/2017/a/a6/T--Bielefeld-CeBiTec--protocol_CBT-asparagine.pdf">protocol for the synthesis of CBT‑asparagine</A>.
			</ARTICLE><DIV class="contentline"><DIV class="half left"><ARTICLE>
						CL is an amino acid consisting of cysteine and lysine. The cysteine was coupled to the side 
						chain of lysine so that CL contains a free 1,2-aminothiol group 
						(Nguyen <I>et al.</I>, 2011). This is an important characteristic for the 
						specific binding between CL and CBT‑asparagine. 
						<UL><LI>Name: 				<I>N</I><SUP>ε</SUP>-L-cysteinyl-L-lysine</LI><LI>Short: 				CL
							</LI><LI>Molecular Weight: 	249.33 g mol<SUP>-1</SUP></LI><LI>Storage:			-20 – 4 °C</LI><LI>Function:			Terminus independent binding system
						</LI></UL></ARTICLE></DIV><DIV class="half right"><DIV class="figure medium"><P class="figure subtitle"><B>Figure 3: Structure of CL.</B></P></DIV></DIV><H3><I>N</I><SUP>γ</SUP>-2-cyanobenzothiazol-6-yl-L-asparagine</H3><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 4: Structure of CBT‑asparagine.</B></P></DIV></DIV><DIV class="half right"><ARTICLE>
						CBT‑asparagine is a completely novel amino acid, which we are synthesizing on our own. The synthesis is based on coupling the amino group of 6-Amino-CBT to the carboxyl 
						group of the side chain of L-asparagine. The cyano group of the CBT enables the specific binding of the CBT‑asparagine to 1,2-aminothiols.
						<UL><LI>Name: 				<I>N</I><SUP>γ</SUP>‑2‑cyanobenzothiazol‑6‑yl‑L‑asparagine</LI><LI>Short: 				CBT‑asparagine
							</LI><LI>Molecular Weight: 	290.30 g mol<SUP>-1</SUP></LI><LI>Storage:			-20 – 4 °C</LI><LI>Function:			Terminus independent binding system
						</LI></UL></ARTICLE></DIV></DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Silk Elastin like Proteins</H3><ARTICLE>
			This specific binding can improve the stability of SELPs. These are linear polypeptides with repeats of silk and elastin consensus 
			sequences. They show broad applications in medicine, tissue engineering and industry 
			(Rnjak-Kovacina <I>et al.</I>, 2011). The silk consensus sequence is GAGAGS and the elastin consensus sequence is 
			VPAVG. The consensus sequences can interact with each other and are able to form non-covalent hydrogen bonds. This results in a 
			polymer network based on hydrogen bonds with a β-sheet structure. Figure 5 shows the schematic structure of a SELP polymer network.
		</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 5: Schematic structure of a SELP polymer network.</B>Silk consensus sequences (GAGAGS) are shown in green, 
				elastin consensus sequences (VPAVG) in red and the blue lines indicate the hydrogen bonds of the consensus sequences.</P></DIV><ARTICLE>
			According to the work of Collins et al. (2013), we decided to use a sequence with nine repeats of five repeats of the silk consensus 
			sequence and nine repeats of the elastin consensus sequence (see figure 6).
		</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 6: Schematic sequence of the SELP (Collins <I>et al.</I>, 2013).</B>Silk consensus sequences (S)
				are shown in green and elastin consensus sequences (E) in red.</P></DIV><ARTICLE>
			By incorporating CL and CBT‑asparagine between the silk and the elastin repeats, we receive a strengthened polymer network 
			with covalent bonds (Figure 7).
		</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 7: Schematic structure of a SELP polymer network including CL and CBT-asparagine.</B>CL and CBT-asparagine (purple) are introduced between the silk (green) and elastin (red) repeats.</P></DIV><H3>Recursive Directional Ligation by Plasmid Reconstruction (PRe-RDL)</H3><ARTICLE>
				The gene sequence for these SELPs has a high GC content and contains a high number of repeats leading to issues during synthesis. 
				Therefore, PRe-RDL can be applied to address this challenge. PRe-RDL uses three restriction sites of a parent plasmid which contains the 
				gene of interest (goi) and the subsequent ligation of fragments of two different restricted parent plasmids. The first step involves 
				the restriction of a parent plasmid at the 3'-end of the goi and in the backbone. The second step is the restriction of a parent plasmid 
				at the 5'-end of the goi and at the same position of the backbone as in the first step. The final step is the ligation of both generated 
				fragments containing the goi. The result is a plasmid with two copies of the goi (Figure 5). 
			</ARTICLE><DIV class="figure large"><P class="figure subtitle"><B>Figure 8: Scheme of the PRe-RDL according to McDaniel <I>et al.</I>, 2010 and applied to pSB1C3 containing one elastin consensus sequence.</B> The Pre-RDL
					consists of 3 steps, two different digestions (step 1 and 2) and one ligation (step 3).</P></DIV><DIV class="article">
			Applied to pSB1C3 containing the consensus sequences of monomers and the spacers between the BioBrick prefix and suffix shown in figure 9, it is possible to use the PRe-RDL in 
			combination with the restriction enzymes, <I>Acu</I>I, <I>Bse</I>RI, and <I>Bsp</I>EI, to build repetitive sequences of monomers. 
			</DIV><DIV class="figure large"><P class="figure subtitle"><B>Figure 9: Design of <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201250">BBa_K2201250</A> and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201251">BBa_K2201251</A>.</B></P></DIV><DIV class="article">
			Step 1 serves to create a fragment containing one part of the chloramphenicol resistance (CmR) and the whole monomer consensus sequence. To do so, the plasmid has to be digested 
			with <I>Acu</I>I and <I>Bsp</I>EI. The other fragments which are resulting from this step contain the other part of the CmR and each one part of the origin of replication (ori). 
			This decreases the chance of religation of the three fragments originating from this step. For step 2, it is necessary to use <I>Bse</I>RI and <I>Bsp</I>EI. The result of this 
			digestion is one fragment containing one part of the CmR and the spacer between the prefix and the monomer consensus sequence and one fragment with the other part of the CmR, 
			the ori, and the monomer consensus sequence. By ligation of both fragments of step 1 and 2 containing the monomer consensus sequence (step 3) pSB1C3 with two repeats of the 
			monomer consensus sequence results. By repeating these steps it is possible to create repetitive sequences consisting of monomeric consensus sequences. During the PRe-RDL it is also possible
			to add codons like the amber codon using primers. After the PRe-RDL it is important to add a start and a stop codon.
			</DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>References</H3><ARTICLE><B>Boateng, J., Matthews, K.H., Stevens, H.N.E., and Eccleston, G.M.</B> (2008). Wound Healing Dressings and Drug Delivery Systems: A Review. J. Pharm. Sci. <B>97</B>.
<B>Collins, T., Azevedo-silva, J., Costa, A., Branca, F., Machado, R., and Casal, M.</B> (2013). Batch production of a silk-elastin-like protein in E . coli BL21 ( DE3 ): key parameters for optimisation. Microb. Cell Fact. <B>12</B>: 1–16.
<B>Liang, G., Ren, H., and Rao, J.</B> (2010). A biocompatible condensation reaction for controlled assembly of nanostructures in living cells. Nat. Chem. 2: 54–60.
<B>McDaniel, J.R., Mackay, J.A., Quiroz, F.G., and Chilkoti, A.</B> (2010). Recursive Directional Ligation by Plasmid Reconstruction allows Rapid and Seamless Cloning of Oligomeric Genes. <B>11</B>: 944–952.
<B>Nguyen, D.P., Elliott, T., Holt, M., Muir, T.W., and Chin, J.W.</B> (2011). Genetically Encoded 1,2-Aminothiols Facilitate Rapid and Site-Specific Protein Labeling via a Bio-orthogonal Cyanobenzothiazole Condensation. J. Am. Chem. Soc. <B>133</B>: 11418–11421.
<B>Rnjak-Kovacina, J., Daamen, W.F., Pierna, M., Rodríguez-Cabello, J.C., and Weiss, A.S.</B> (2011). Elastin Biopolymers. Compr. Biomater.: 329–346.

				
			</ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>