<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Measurement</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/7/77/T--Bielefeld-CeBiTec--title-img-pipettes.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Measurement
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Short summary</H2><ARTICLE>
Characterizing parts in accordance with the design, build, and test cycle is one of the essential aspects of iGEM. Our <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">toolbox</A> for the incorporation of non-canonical amino acids provides innovative new methods for the characterization of all protein encoding parts, therefore offering advanced approaches for improved part characterization. The five different tools allow the translational incorporation of non-canonical amino acids, with additional functional groups which can be used to study proteins <I>in vivo</I> and <I>in vitro</I>. With the help of non-canonical amino acids, the subcellular localization of a protein can be investigated. Measurement of intramolecular distances, protein immobilization, light regulation and modification enables sophisticated characterization of parts.
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Localization <I>in vivo</I> with fluorescent amino acids</H2><ARTICLE>
The labeling of a protein <I>in vivo</I> is a useful tool that allows the investigation of a protein in its native environment. As a label for our target protein we use the fluorescent amino acid L-(7-hydroxycoumarin-4-yl) ethylglycine (CouAA) that is incorporated by an orthogonal t-RNA/aminoacyl synthethase pair (tRNA/aaRS) at a defined position. We provide this tRNA/aaRS <A target="blank" href="http://parts.igem.org/Part:BBa_K2201373">(BBa_K2201204)</A> to the iGEM community. In the iGEM competition protein localization <I>in vivo</I> can be performed by labeling the target protein with a fluorescent protein like green fluorescent protein (GFP) or red fluorescent protein (RFP). This labeling is done by a translational fusion of the CDS from the fluorescent protein C- or N terminal with a short linker to the CDS from the target protein. But the labeling is limited to the C- or N terminus and due to its size GFP (29 kDa (Charbon et al., 2011)) could be bigger than the target protein and be a hindrance. Both could cause a significant change of the structure of the target protein or a loss of function, especially if the protein is part of an assembly in a larger complex or oligomer (Charbon <I>et al</I>., 2011, Wang <I>et al</I>., 2006). 
The usage of a genetically encoded fluorescent amino acid would circumvent these problems and deliver a tool to study protein localization and function <I>in vivo</I> and <I>in vitro</I>. An orthogonal t RNA/aminoacyl tRNA synthetase pair allows the incorporation of amino acids in response to the amber stop codon (TAG) selectively at a defined position in the protein (Charbon <I>et al</I>., 2011).
The fluorescent amino acid L-(7-hydroxycoumarin-4-yl) (CouAA) ethylglycine is relatively small, has a high fluorescence quantum yield and relatively large Stoke's shift. It is also solvent polar and pH-sensitive so it can indicate pH-changes in the cell (Wang <I>et al</I>., 2006). The amino acid is suitable for <I>in vivo</I> and <I>in vitro</I> localization, and in contrary to fluorescent proteins even for localization in SDS PAGES. For a detailed description of our labeling tool please refer to the <A target="blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">labeling</A> page of the <A target="blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">toolkit</A>. 
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Analyzing of intermolecular distances in proteins with non-canonical amino acids</H2><ARTICLE>
The structure of proteins could be detected through protein crystallography. However, there are a lot of problems when it comes to highly flexible proteins or proteins which change their structure under different conditions. To analyze these proteins and the changes in their conformation we want to establish a tool that allows detecting changes in protein conformation. For the detection of these changes two amino acids are incorporated at specific positions in the protein. These amino acids could then be labeled with chromophores, enabling the measurement of the proteins distances with Foerster Resonance Energy Transfer (FRET)(Lembke, 2011, Kim <I>et al</I>., 2013). 
The first step is the incorporation of the non-canonical amino acids. In proteins naturally containing no cysteins (cysteines are the only canonical amino acids that could be labeled specific) or in which the exchanges of cysteines do not influence the structure only one ncAA and one cysteine at specific points need to be incorporated to be labeled. In proteins that contain cysteine, two ncAAs need to be incorporated for the labeling (Kim <I>et al</I>., 2013). 
Non-canonical amino acids could be incorporated by orthogonal tRNA/aaRS synthetases in response to the amber stop codon. However, this allows only the incorporation of one noncanonical amino acid. To incorporate the second amino acid, another orthogonal amino acid could be used for the incorporation in response to a rarely used leucine codon. For structural analysis the amino acids are specific labeled with chromophores. This labeling is possible due to the functional groups of the amino acids which could form a covalent bond to the fluorophores in a chemical reaction. After the protein is labeled, the fluorescence of the chromophores could be measured to draw conclusions on the distance of the ncAA from each other (Brustad <I>et al</I>., 2008, Kim <I>et al</I>., 2013).
To incorporate the ncAAs, we provide three different tRNA/aminoacyl-synthetases which incorporate in response to the amber or the less used leucine codon. For more details please refer to the <A target="blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">labeling page</A>.
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Immobilization and fusing of proteins with non-canonical amino acids</H2><ARTICLE>
Fusing proteins is normally limited to the C  or N terminus of a protein. The incorporation of non-canonical amino acids that could be fused to each other or to surfaces enables several additional applications. While terminus dependent binding systems for proteins are already in use, there are only a few systems for terminus independent binding systems. We want to expand the number of those systems. Our aim is to incorporate two non-canonical amino acids, which are able to build a specific bond to each other. According to the synthesis of luciferin for the firefly luciferase of <I>Photinus pyralis</I>, we decided to use the specific binding of 1,2 aminothiols and the cyano group of cyanobenzothiazole (CBT). By synthesis of amino acids with side chains containing CBT and a 1,2 aminothiol, polypeptides binding to each other should be produced. These amino acids are CL and CBT Asp. 
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Protein regulation with photoswitching and photolysis amino acids</H2><ARTICLE>
The metabolic flux of a given enzymatic pathway can be controlled by photoswitching thus resulting in an improved understanding of the biological functions of a given enzyme <I>in vivo</I> or in a controlled yield of intermediate substrates. Investigations of biological functions of proteins and enzymes <I> in vivo</I> are limited based on the inability to specifically control their functions. Usage of photolabile protection groups allows temporal and spatial control of chemical and biological processes (Bose <I>et al</I>., 2006; Wang <I>et al</I>., 2009; Klán <I>et al</I>., 2013). Two photolabile approaches can be chosen: (i) photocaging and (ii) photoswitching (Brieke <I>et al</I>., 2012). 

(i)
Photocaging: a “protection” group facilitates or inhibits the normal function of a given protein, but after cleavage of the chemical moiety the protein of interest is de/activated. This process is irreversible.
(ii)
Photoswitching: the chemical moiety used can be switched between “ON/OFF” stages. This process is reversible. For both approaches the incorporation of a chemical moiety into a permissive site of the protein of interest is accomplished through amber suppressor tRNA (Bose <I>et al</I>., 2006).
The advantage of light as the trigger for the cleaving and conformational change lies in its highly controllable, selective and inexpensive application. In contrast to chemical substrates used for induction of a reaction, light does not leave residues which themselves can influence the test environment. Furthermore, many already established techniques can be adapted to apply the specific wavelength and irradiation time for any possible non-canonical amino acid (Brieke <I>et al</I>., 2012). 
We decided to use a photoisoerisable amino acid because of the reversible reaction. To demonstrate this tool we incorporated the photoisomerisable amino acid p-azobenzene in the enzyme CrtI to regulate the lycopene pathway. We showed that the activity of CrtI could be regulated only by light irradiation.
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>References</H2><ARTICLE><B>Bose, M., Groff, D., Xie, J., Brustad, E., and Schultz, P.G. </B>(2006). The Incorporation of a Photoisomerizable Amino Acid into Proteins in E. coli. J. Am. Chem. Soc. 128:<B> 388–389</B>.<B>Brieke, C., Rohrbach, F., Gottschalk, A., Mayer, G., and Heckel, A. </B>(2012). Light-Controlled Tools. Angew. Chem. Int. Ed. 51: 8446–8476.<B>Brustad, E. M., Lembke, E. A., Schultz, P. G., Dentz, A. A.</B>(2008). A General and Efficient Method for the Site-Specific Dual-Labeling of Proteins for Single Molecule Fluorescence Resonance Energy Transfer. American Chemical Society. 130:<B> 17664-17665</B>.<B>Charbon, G., Brustad, E., Scott, K.A., Wang, J., Lobner-Oelson, A. Schultz, P. G., Jacobs-Wagner, C., Chapman, E.</B>(2011). Subcellular Protein Localization by Using a Genetically Encoded Fluorescent Amino Acid. ChemBioChem. 12:<B>1818-1821</B>.<B>Charbon, G., Wang, J., Brustad, E., Schultz, P. G., Horwiich, A. L., Jacobs-Wagner, C., Chapman, E.</B>(2011). Localization of GroEL determined by in vivo incorporation of a fluorescent amino acid. Bioorg Med Chem Lett. 21(20): <B>6067-6070</B>.<B>Kim, J., Seo, M., Lee, S., Cho, K., Yang, A., Woo, K., Kim, H., Park, H.</B>(2012). Simple and Efficient Strategy for Site-Specific Dual Labeling of Proteins for Single-Molecule Fluorescence Resonance Energy Transfer Analysis. Analytical Chemistry.85: <B>1468-1474</B>. <B>Klán, P., Šolomek, T., Bochet, C.G., Blanc, A., Givens, R., Rubina, M., Popik, V., Kostikov, A., and Wirz, J. </B>(2013). Photoremovable Protecting Groups in Chemistry and Biology: Reaction Mechanisms and Efficacy. Chem. Rev. 113: 119–191.<B>Wang, J., Xie, J., Schultz, P. G.</B>(2006). A Genetically Encoded Fluorescent Amino Acid. American Chemical Society.128:<B>8738-8739</B>
Wang, Q., Parrish, A.R., and Wang, L. (2009). Expanding the Genetic Code for Biological Studies. Chem. Biol. 16: <B>323–336.</B></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>