Team:Bielefeld-CeBiTec/Notebook/Labjournal
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Labjournal
2017-04-03   -   2017-04-09
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: electroporation transformation of BBa_K1416000
Procedure:
standard electroporation protocol
used electro competend DH5alpha from NEB
streaked out on LB-plates with Cam
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: electroporation transformation of BBa_KJ36848
Procedure:
standard electroporation protocol
used electro competend DH5alpha from NEB
streaked out on LB-plates with Cam
multiplication of plasmids containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: preculture of transformed BBa_K1416000
Procedure:
standard procedure
1,25µl Cam in 5mL LB
2017-04-10   -   2017-04-16
getting plasmids containing the Biobrick for further experiments
Investigators: Yannic Kerkhoff
Superior experiment: plasmid isolation of preculture of BBa_K1416000
Procedure:
standard protocol
517,9 ng/µL
multiplication of plasmids containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: preculture of transformed BBa_J36848
Procedure:
standard procedure
1,25µl Cam in 5mL LB
getting plasmids containing the Biobrick for further experiments
Investigators: Yannic Kerkhoff
Superior experiment: plasmid isolation of preculture of BBa_J36848
Procedure:
standard protocol
231,8 ng/µL
2017-05-29   -   2017-06-04
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: heatshock transformation of BBa_E0400
Procedure:
standard heatshock protocol
used own produced chemo competend DH5alpha
streaked out on LB-plates with Amp
2017-06-05   -   2017-06-11
multiplication of plasmids containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: preculture of transformed BBa_E0400
Procedure:
standard procedure
5,0µl Amp in 5mL LB
getting plasmids containing the Biobrick for further experiments
Investigators: Yannic Kerkhoff
Superior experiment: plasmid isolation of preculture of BBa_E0400
Procedure:
standard protocol
88,4 ng/µL
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: heatshock transformation of BBa_K525998
Procedure:
standard heatshock protocol
used own produced chemo competend DH5alpha
streaked out on LB-plates with Cam
multiplication of plasmids containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: preculture of transformed BBa_K525998
Procedure:
standard procedure
1,25µl Cam in 5mL LB
getting plasmids containing the Biobrick for further experiments
Investigators: Yannic Kerkhoff
Superior experiment: plasmid isolation of preculture of BBa_K525998
Procedure:
standard protocol
140,9 ng/µL
2017-06-12   -   2017-06-18
Assemble the fragments to get the Part BBa_K2201220
Investigators: Yannic Kerkhoff
Superior experiment: Gibson assembly of F1,F3,F5
Procedure:
standard Gibson Assembly protocol
Cellgrowth with the barnaseplasmid
Investigators: Olga Schmidt, Denise Kerkhoff, Christina Drake
Superior experiment: Overnightculture of barnase
Procedure:
3 ml LB-medium and 0,75  µl chloramphenicol
inkubation overnight 37°C, 140rpm
To get and purified the plasmid
Investigators: Christina Drake
Superior experiment: Plasmidisolation of barnase-plasmid
Procedure:
protocol from Macherley-Nagel: Plasmid DNA purification, protocol 5: NucleoSpin
concentration measurement by nanoprop
sequenzingorder
Duplicate the parts
Investigators: Christina Drake, Denise Kerkhoff
Superior experiment: Transformation via Distribution of the parts BBA_J04450, BBa_I746909, BBa_K80800 and BBa_psB1K5
Procedure:
transformation via electroporation
plated out: BBa_J04450 on tetracycline, BBa_I74909 and BBa_K808000 on chloramphenicol, BBa_psB1K5 on kanamycin
Duplicate the part
Investigators: Christina Drake, Denise Kerkhoff
Superior experiment: Transformation of BBa_psB6A1
Procedure:
transformation via electroporation
plated out on amphenicol
Cellgrowth with the plasmids
Investigators: Denise Kerkhoff, Christina Drake
Superior experiment: Overnightculture of BBA_J04450, BBa_I746909 and BBa_K808000
Procedure:
3 ml LB-medium and at BBA_J04450 3µl tetracycline and atBBa_I746909 and BBa_K808000 0,75  µl chloramphenicol
inkubation overnight 37°C, 140rpm
To get and purified the plasmid
Investigators: Christina Drake
Superior experiment: Plasmidisolation of BBA_J04450, BBa_I746909 and BBa_K80800-plasmids
Procedure:
protocol from Macherley-Nagel: Plasmid DNA purification, protocol 5: NucleoSpin
concentration measurement by nanoprop:
BBA_J04450 26ng/µl and 25,7ng/µl
BBa_I746909 90,2ng/µl
104,3 ng/µl -BBa_K808000 118,3 ng/µl
and 146,6 ng/µl
sequenzingorder
2017-06-19   -   2017-06-25
Cellgrowth with the plasmid
Investigators: Denise Kerkhoff
Superior experiment: Overnightculture of BBA_psB3K5
Procedure:
3 ml LB-medium and  2,5 µl kanamycin
inkubation overnight 37°C, 140rpm
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: heatshock transformation of assembled fragments F5,F2,F4
Procedure:
standard heatshock protocol
used own produced chemo competend DH5alpha
streaked out on LB-plates with Cam
multiplication, verification and purification of the needed fragment F2: GFPLinker2
Investigators: Yannic Kerkhoff
Superior experiment: Q5-PCR, gel and PCR-cleanUp of F2
Procedure:
standard Q5-PCR protocol
template: BBa_E0400
primer fwd: 17gk
primer rev: 17gn
annealing temperature: 58°C
extension time: 50 seconds
standard PCR-cleanUp protocol
125,8 ng/µL
multiplication, verification and purification of the needed fragment F4: StrepLinker2
Investigators: Yannic Kerkhoff
Superior experiment: Q5-PCR, gel and PCR-cleanUp of F4
Procedure:
standard Q5-PCR protocol
template: BBa_KJ36848
primer fwd: 17gp
primer rev: 17gq
annealing temperature: 61°C
extension time: 30 seconds
standard PCR-cleanUp protocol
127,0 ng/µL
multiplication, verification and purification of the needed fragment F5: linearized pSB1C3
Investigators: Yannic Kerkhoff
Superior experiment: Q5-PCR, gel and PCR-cleanUp of F5
Procedure:
standard Q5-PCR protocol
template: K1416000
primer fwd: 17gj
primer rev: 17gl
annealing temperature: 59°C
extension time: 120 seconds
standard PCR-cleanUp protocol
258,4 ng/µL
Assemble the fragments to get the Part BBa_K2201221
Investigators: Yannic Kerkhoff
Superior experiment: Gibson assembly of F2,F4,F5
Procedure:
standard Gibson Assembly protocol
Screening for positive transformations containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: GoTaq PCR of 7 clones of BBa_K2201221
Procedure:
standard GoTaq-protocol
template: colonie pcr clones of GA_K2201221
primer fwd: Präfix_fwd
primer rev: Suffix_rev
annealing temperature: 56°C
extension time: 40 seconds
positive clone 2
multiplication of plasmids containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: preculture of the positive clone 2 of BBa_K2201221
Procedure:
standard procedure
1,25µl Cam in 5mL LB
getting plasmids containing the Biobrick for further experiments
Investigators: Yannic Kerkhoff
Superior experiment: plasmid isolation of preculture of BBa_K2201200 clone 20
Procedure:
standard protocol
240,6 ng/µL
getting plasmids containing the Biobrick for further experiments
Investigators: Yannic Kerkhoff
Superior experiment: plasmid isolation of preculture of BBa_K2201221 clone 2
Procedure:
standard protocol
208,8 ng/µL
2017-06-26   -   2017-07-02
Check the sequence
Investigators: Christina Drake
Superior experiment: Sequenzingorder barnase
Procedure:
concentration measurement by nanoprop: 177 ng/µl
result: sequence was not barnase
Plasmidisolation and gibson assembly of pSB1C3-PlacUV5_PtNTT2 c30
Investigators: Camilla Maerz
Superior experiment: Construction of pSB1C3-PlacUV5_PtNTT2
Procedure:
Plasmid DNA purification kit
Nucleo spin plasmid protocol
Gibson Assembly protocol
transformation via heat shock protocol in E. coli DH5alpha
Colony PCR of T7RNAP from E. coli KRX
Investigators: Olga Schmidt
Superior experiment: Construction of the positive selection plasmid
Procedure:
PCR with Grimson Taq polymerase (per reaction, 25 µL):
5 µL 5x Buffer
2.5 µL dNTPs (2 mM)
0.5 µL 3'-Primer (10 mM)
0.5 µL 5'-Primer (10 mM)
0.125 µL Grimson Taq polymerase
1 picked bacterial colony as template
ad 25 µL H20
Primer annealing: 64 °C
Elongation: 150 s
Agarose gel electrophoresis (1%)
100 V, 30 min
Isolate T7RNAP from the chromosome of KRX-e.coli
Investigators: Christina Drake
Superior experiment: PCR of T7RNAP from KRX-e.coli
Procedure:
Q5 High-Fidelity PCR
annealingtemparatur 64°C, elogationtime 1 min
1%-agarose- TAE -gel
1kb Marker Geneuler and 5 µl sample with 1 µl 6*Loading Dye
Plated out Barnase from strain collection
Investigators: Christina Drake
Superior experiment: Plated out Barnase from strain collection
Procedure:
solved DNA in 100 µl SOC-medium
plated out on chloramphenicolplate
overnightincubation at 37°C
Colony PCR of pK18mobsacB-del_codA
Investigators: Camilla Maerz
Superior experiment: Construction of pK18mobsacB_codA_del
Procedure:
Go Tag (Promega) protocol
Primer: VR, VF2
Primer annealing: 56 °C
Elongation: 140 s
PCR of pSB1C3
Investigators: Olga Schmidt
Superior experiment: General
Procedure:
PCR
Q5 High-fidelily (NEB) protocol
Denaturation: 64 °C
Elongation: 60 s
Agarose gel electrophoresis (1%)
100 V, 30 min
Gibson assembly of T7RNAP in pSB1C3
Investigators: Olga Schmidt
Superior experiment: Construction of the selection plasmid
Procedure:
Gibson Assembly protocol
transformation via heat shock protocol in E. coli BL21
Colony PCR of pSB1C3_T7RNAP c1-8
Investigators: Olga Schmidt
Superior experiment: Construction of the selection plasmid
Procedure:
PCR
Q5 High-fidelily (NEB) protocol
Denaturation: 64 °C
Elongation: 80 s
Agarose gel electrophoresis (1%)
100 V, 30 min
Colony PCR of barnase (26.06.2017) c1&2
Investigators: Olga Schmidt
Superior experiment: Construction of selection plasmid
Procedure:
PCR
Go Tag (Promega) protocol
Danaturation: 58 °C
Elongation: 35 s
Agarose gel electrophoresis (1%)
100 V, 30 min
2017-07-03   -   2017-07-09
Check the correct barnasegen
Investigators: Christina Drake
Superior experiment: PCR of barnase
Procedure:
Go-Taq-protocol
annealingtemparatur 58°C, elogationtime 30 s
1%-agarose- TAE -gel
100bp Marker Geneuler and 5 µl sample with 1 µl 6*Loading Dye
result: PCR-product could not be barnase
2017-07-10   -   2017-07-16
Biobrick Assembly of 2B1
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
BioBrick Assembly
RuBisCoTAG mRFP as upstream part, T7-Terminator as downstream part, pSB1A3 as destination plasmid
transformation via heat shock and DH5α Cells
used 5µl from the ligation
plated out on Amp plates and let them grow over night
Check the sequence
Investigators: Christina Drake
Superior experiment: Sequenzingorder barnase
Procedure:
result: sequence was not barnase
>16.07.2017
tRNA
Overnightculture of TyrRS-Heidelberg-plasmid
CDR
Cellgrowth with the TyrRS-Heidelberg-plasmid
3 ml LB-medium and 0,75  µl chloramphenicol
inkubation overnight 37°C, 140rpm
Transformation of T7-Promotor
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
transformation via electroporation of cells with T7-Promotor
Plasmidisolation of TyrRS-Heidelberg plasmid
Investigators: Olga Schmidt
Superior experiment: Construction of selection plasmid
Procedure:
Plasmid DNA purification kit (Macherey-Nagel)
Nucleo spin plasmid protocol
Preculture of colonies with T7-Promotor
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
used 5µl LB media and kanamycin
2017-07-17   -   2017-07-23
Isolate the tRNA
Investigators: Christina Drake
Superior experiment: PCR of TyrRS-Heidelberg-plasmid
Procedure:
plasmidisolation with MN protocol 5
Q5 High-Fidelity PCR
annealingtemparatur 53°C, elogationtime 20s
Gibbsonassembly with 5µl backbone and 0.5 µl insert
transformation via heat shock
overnight incubation at 37°C
Isolation of T7-Promotor
Investigators: Laura Schlueter
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
Nucleospin Plasmidisolation protocol
got four products with the concentrations 50.1 ng/µl, 16.3 ng/µl, 24,2 ng/µl and 2.1 ng/µl
Colony PCR with 2B1 Colonies
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
GoTaq® G2 PCR
Biobrickassembly of 2B2 to 2B8, B1 and A1
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
BioBrick Assembly
digest and ligate RuBisCo parts as upstreampart and T7-Terminator/Promotor as downstreampart and pSB1A3 as destination plasmid
ligate TAG2 and T7-Terminator to 2B2
ligate TAG111 and T7-Terminator to 2B3
ligate TAG474 and T7-Terminator to 2B4
ligate TAG2+TAG111 and T7-Terminator to 2B5
ligate TAG2+TAG474 and T7-Terminator to 2B6
ligate TAG111+TAG474 and T7-Terminator to 2B7
ligate TAG2+TAG111+TAG474 and T7-Terminator to 2B8
ligate RuBisCo mRFP and T7-Terminator to B1
ligate Carboxysome and T7-Promotor to A1
preculture of positive 2B1 colony
Transformation of test devices
Investigators: Camilla Maerz
Superior experiment: Interlab study
Procedure:
dilute test devives (distribution plate 6 & 7) in 10 µL ddH20
Transformation
1 µL plasmid from each plate
transformation via heat shock protocol in E. coli XL1-blue
Transformation of 2B2-2B8, B1 and A1
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
transformation via heat shock
Plasmidisolation of 2B1
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
Nucleospin Plasmidisolation protocol
got 47 ng/µl and 41 ng/µl
Reverse the insert
Investigators: Christina Drake
Superior experiment: PCR of T7GFP
Procedure:
Q5 High-Fidelity PCR
Insert: annealingtemparatur 60°C,
Backbone:elogationtime 30s: annealingtemparatur 60°C, elogationtime 1 min
1%-agarose- TAE -gel
result: PCR didn`t work
Reverse the insert
Investigators: Christina Drake
Superior experiment: PCR of T7GFP
Procedure:
1%-agarose- TAE -gel
result: PCR did work
Gibson Assembly
transformation via heat shock
PCR of BBa_I746909 and pSB1C3
Investigators: Olga Schmidt
Superior experiment: Integration of CDS of BBa_I746909 in pSB1C3
Procedure:
PCR of I746909
Q5 High-fidelily (NEB) protocol
Primer: 17lk & 17lj
Denaturation: 60 °C
Elongation: 40 s
PCR of pSB1C3
Q5 High-fidelily (NEB) protocol
Primer: 17ll & 17li
Denaturation: 60 °C
Elongation: 70 s
Agarose gel electrophoresis (1%)
100 V, 30 min
Preparation of over-night culture
Investigators: Camilla Maerz
Superior experiment: Interlab study
Procedure:
Over-night culture of negative control, positive control, test device 1-6
LB-media + Cm were inoculated
Cultures were incubated over night (37°C, 200 rpm)
2017-07-24   -   2017-07-30
Plasmidisolation of pSB1C3_I746909
Investigators: Olga Schmidt
Superior experiment: Integration of CDS of BBa_I746909 in pSB1C3
Procedure:
Plasmid DNA purification kit (Macherey-Nagel)
Nucleo spin plasmid protocol
Repeat the degestion and transformation from the 18th and 19th july
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
BioBrick Assembly
transformation via heat shock
Cellgrowth with the plasmid
Investigators: Christina Drake
Superior experiment: Overnightculture of tRNA-psB1C3
Procedure:
3 ml LB-medium and 0,75  µl chloramphenicol
inkubation overnight 37°C, 140rpm
Cloning of the ori (pSB6A1) in pSB1C3
Investigators: Laura Schlueter
Superior experiment: aaRS Plasmid (without aaRS)
Procedure:
Gibson Assembly Master Mix
Backbone: 3.28 µL     Ori 1.0: 1.73 µL
Backbone: 0.32 µL     Ori 1.1: 4.68 µL
Backbone: 0.90 µL     Ori 3.0: 4.10 µL
Trafo: heatshock
not successfull because of the wrong overlapps (try again)
Colony PCR of the Retrafo of pRS (because of mixed culture)
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Sequencing was not succesfull (maybe mixed colony), so I did a retrafo with the aim of a successful sequencing (incorporation of the aaRS in pSB1C3)
pRS 3: 1-5
pRS 4: 1-5
pRS 5: 1-5
pRS 7: 1-5
no positive result (no band at 1000 bp)
Amplification of pSB1C3 for the cloning of the ori in pSB1C3
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
PCR with Q5 High-Fidelity Polymerase Master Mix, Primer jb,jn, 65°C
gelelectrophoresis, 1% Agarose
band  2000-2500 bp (supposed to be around 2200 bp)
Clean up from the gel
pSB1C3(nur ori): 22.2 ng/µL
Amplification of pSB1C3 fragments for a 4 part Gibson (see page 4 in the paper-labbook)
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
PCR, Q5 Master Mix
Primer jj, jn for fragment 2, 65°C
Primer jb, ht for fragment 4, 65°C
gelelectrophoresis, 1% Agarose
band at 300 (fragment 2, supposed to be around 326 bp)
band at 1200 (fragment 4, supposed to be around 1233 bp)
Clean up from the gel
fragment 2: 18.5 ng/µL  (overlapp(aaRS)_pSB1C3_overlapp(ori))
fragment 4: 19.1 ng/µL  (overlapp(ori)_pSB1C3_overlapp(aaRS))
Colony PCR of 2B2-2B8, B1 and A1
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
GoTaq® G2 PCR
most have negative results -> create new destination plasmid pSB1A3
Colony PCR of retrafo and more colonies of the pRS
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
GoTaq Master Mix, Primer vr,vf
2 strong bands at 400 bp and 600 bp, small bands of pRS3 6 and pRS4 9 at around 1200-1500 bp
bands of pRS31, pRS32, pRS34, pRS36, pRS39 at around 1200-1500 bp
GoTaq Master Mix, Primer hq, jk ( Gibson Assembly primer) with the colonies pRS31, pRS32, pRS34, pRS36, pRS39
pRS31, pRS32, pRS39 : strong bands at around 1200 bp
the two positive colony PCRs (vf-vr, hq-jk) for these probes, successful integration of the Tyr-aaRS in pSB1C3 seems to be successfull
pRS31, pRS32, pRS39 send to sequencing
pRS31, pRS32, pRS39 plasmid isolation
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Macherey-Nagel purification kit
pRS31: 324.1 ng/µL
pRS32: 231.8 ng/µL
pRS39: 225.5 ng/µL
send tosequencing
pRS31, pRS39 have the pSB1C3 with the Tyr-aaRS incorporated (but on position 32, the Arginine (cgt) should be changed to an Leucine (ctg), but it is not. On position 290 there really is a missing Leu, on position 268 the Asparagin is changed to an Arginine)
2017-07-31   -   2017-08-06
construct a biobrick
Investigators: Christina Drake
Superior experiment: Aquacloning of tRNA and backbone
Procedure:
transformation via heatschock
plasmid isolation of the positive colonies of the retrafo pRS4 4 (wrong!) and  pRS3 12
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Macherey-Nagel purification kit with each 3 mL culture
pRS4 4:  44   ng/µL
pRS3 12: 68.3 ng/µL
not send to sequencing jet
4 part Gibson and Trafo
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Gibson Assembly Master Mix with 5 µl template
pSB1C3 fragment (2):       18.5 ng/µL  around  350 bp   2.2 µL
pSB1C3 fragment (4):       19.1 ng/µL  around 1300 bp   0.8 µL
oK1.1 (pMB1 from pSB6A1):   5.8 ng/µL  around 1300 bp   1.8 µL
Tyr-aaRS:                  68.5 ng/µL  around 1000 bp   0.2 µL
Trafo via heatshock
Gibson of the ori in pSB1C3 (pori) and Trafo
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Gibson Assembly Master Mix with 5 µL template
pSB1C3(nur ori): 22.2  ng/µL   around 2100 bp   1.57 µL
ok1.0:            5.80 ng/µL   around 1258 bp   3.43 µL
Trafo via heatshock
Transfer positive colonies
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
picked and transfered positiv colonies of 2B2, 2B6, 2B7, 2B8 and A1 onto a new plate
incubate over night
2B2 and 2B8 are contaminated with negativ colonies
construct a biobrick
Investigators: Christina Drake
Superior experiment: Aquacloning of tRNA and backbone
Procedure:
transformation via heatschock
construct a biobrick
Investigators: Christina Drake
Superior experiment: Gibbsonassembly of barnase and backbone
Procedure:
transformation via heatschock
Preculture of 2B6, 2B7 and A1
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
used 3µl LB media
Cellgrowth with the plasmid
Investigators: Christina Drake
Superior experiment: Overnightculture of tRNA-psB1C3 and barnase-psB1C3
Procedure:
3 ml LB-medium and 0,75  µl chloramphenicol
inkubation overnight 37°C, 140rpm
Plasmidisolation
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
Nucleospin Plasmidisolation protocol
got 67.3 ng/µl of 2B6
got 32.7 ng/µl of 2B7
got 51.6 ng/µl of A1
To get and purified the plasmid
Investigators: Christina Drake
Superior experiment: Plasmidisolation of barnase-plasmid and tRNA-plasmids
Procedure:
protocol from Macherley-Nagel: Plasmid DNA purification, protocol 5: NucleoSpin
concentration measurement by nanoprop
sequenzingorder
Redo digestion of the destination plasmid
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
analyze the destination plasmid digestion via gelelectrophoresis
pSB1A3 digestion was not effective
redo the digestion and analyze it on the gel
new dgestion was very effective
2017-08-07   -   2017-08-13
Amplification of the backbone (pSB1C3) and insert (Tyr-aaRS)
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
PCR Q5 Polymerase (no Master Mix)
for 50 µL reaction
10   µL Reaction Buffer
5   µL 2mM dNTPs
1   µL Template
0,5 µL High Fidelity DNA Polymerase
10.0 µL High GC Enhancer
18.5 µL ddest H2O
5.0 µL Primer
pSB1C3 (from RuBisCo, ncAA team): ht, jk  65°C
aaRS (from Heidelberg) : hq, jk 65°C (mistake! should be 58°C, redone identically with 58°C)
digestion of the PCR products with Dpn1: 5 µL CutSmart Reaktionsbuffer and 1 µL Dpn1 per 50 µL PCR product, on 37°C over night)
agarose-gelelectrophhoresis, 1% (bands of the right size)
No further use, because results of the sequencing which show that the integration of the incorporation of the Tyr-aaRS in pSB1C3 was already successful (30.06.2017)
Ligation of 2B2-2B5 and B1
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
Ligation protocol from the BioBrick Assembly
Variation: incubated 1h at RT instead of 10min
characterisation of T7GFP revers
Investigators: Christina Drake
Superior experiment: preparation of KRX-competent cells
Procedure:
heatschocktransformation of T7GFP and T7GFP revers in KRX
construct a biobrick for the selectionplasmid
Investigators: Christina Drake
Superior experiment: PCR of T7RNA-Polymerase
Procedure:
1%-agarose- TAE -gel
result: PCR did work
Gibson Assembly
transformation via heat shock
Amplification of the backbone (pRS31) and insert (ori from pSB6A1, oK1.0) for Gibson assembly
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
PCR Q5 Polymerase (protocoll of 07.08.2017)
pRS31: primer jb, jn  65°C
oK1:   primer jl, jc  65°C
digestion with Dpn1 (protocoll of 07.08.2017)
agarose-gelelectrophhoresis, 1%
band around 1200 bp (oK1.1) / 2500 bp (pRS31), positive
Gibson Assembly and Trafo of the pRS31 and oK1.0
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
agarose-gelelectrophhoresis, 1%
purification from the gel with the Macherey-Nagel purification kit
oK1.1: 22.5 ng/µL
pRS32:  9.5 ng/µL
Gibson Assembly Master Mix with:
4.1 µL of oK1.1: around 2480 bp  9.5 ng/µL
0.9 µL of pRS32: around 1250 bp 22.5 ng/µL
Trafo via heatshock
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: heatshock transformation of assembled fragments F13 and NPA-RS gene synthesis
Procedure:
standard heatshock protocol
used own produced chemo competend DH5alpha
streaked out on LB-plates with Cam
multiplication, verification and purification of the needed fragment F13: linearized ONBY-Part with removal of the ONBY-RS
Investigators: Yannic Kerkhoff
Superior experiment: Q5-PCR, gel and PCR-cleanUp of F13
Procedure:
standard Q5-PCR protocol
template: K1416000
primer fwd: 17hl
primer rev: 17mv
annealing temperature: 63°C
extension time: 90 seconds
standard PCR-cleanUp protocol
80,6 ng/µL
Assemble the fragments to get the Part BBa_K2201200
Investigators: Yannic Kerkhoff
Superior experiment: Gibson assembly of F13 and NPA-RS gene synthesis
Procedure:
standard Gibson Assembly protocol
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: heatshock transformation of assembled fragments F5,F1,F3
Procedure:
standard heatshock protocol
used own produced chemo competend DH5alpha
streaked out on LB-plates with Cam
multiplication, verification and purification of the needed fragment F1: GFPLinker1
Investigators: Yannic Kerkhoff
Superior experiment: Q5-PCR, gel and PCR-cleanUp of F1
Procedure:
standard Q5-PCR protocol
template: BBa_E0400
primer fwd: 17gk
primer rev: 17gm
annealing temperature: 58°C
extension time: 50 seconds
standard PCR-cleanUp protocol
15,9 ng/µL
multiplication, verification and purification of the needed fragment F3: StrepLinker1
Investigators: Yannic Kerkhoff
Superior experiment: Q5-PCR, gel and PCR-cleanUp of F3
Procedure:
standard Q5-PCR protocol
template: BBa_KJ36848
primer fwd: 17go
primer rev: 17gq
annealing temperature: 61°C
extension time: 30 seconds
standard PCR-cleanUp protocol
17,9 ng/µL
Biobrickassembly
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
BioBrick Assembly
repeat digestion of T7-Terminator downstrampart
ligate over night
Transformation of new plasmids
Investigators: Maximilian Edich
Superior experiment: Biobrick Construction with RuBisCo Parts
Procedure:
transformation via heat shock
multiplication, verification and purification of the needed fragment F15: linear GLS1
Investigators: Yannic Kerkhoff
Superior experiment: Q5-PCR, gel and PCR-cleanUp of F15
Procedure:
standard Q5-PCR protocol
template: P2: GLS2
primer fwd: 17go
primer rev: 17gm
annealing temperature: 60°C
extension time: 190 seconds
standard PCR-cleanUp protocol
36,9 ng/µL
Primer annealing preparation
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system - Primer annealing test
Procedure:
Both Primers (17jh; 17ji) were resuspended in 1x Composite Buffer
Dilutions: 100 µM, 10 µM, 5 µM, 2.5 µM, 0.5 µM
OD 260 were measured to determine the real concentration
2017-08-14   -   2017-08-20
Screening for positive transformations containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: GoTaq PCR of 4 clones of BBa_K2201220
Procedure:
standard GoTaq-protocol
template: colonie pcr clones of GA_K2201220
primer fwd: Präfix_fwd
primer rev: Suffix_rev
annealing temperature: 56°C
extension time: 40 seconds
positive clone 3
multiplication of plasmids containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: preculture of the positive clone 3 of BBa_K2201220
Procedure:
standard procedure
1,25µl Cam in 5mL LB
Primer annealing of 17jh & 17ji
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system - Primer annealing test
Procedure:
Protocol for annealing oligonucleotides (sigmaaldrich)
Eqimolar Primers were mixed (50 µL)
Thermal profile (thermocycler)
95 °C, 2 min
Cool to 25 °C over 45 min
Cool to 4 °C for temporary storage
Integration of barnase in pSB1C3
Investigators: Olga Schmidt
Superior experiment: Construction of selection plasmid
Procedure:
Restriction Digest of pSB1C3_I749606
5 µL 10x Universal buffer
5.54 µL plasmid
1µL XbaI
1µL PstI
filled up to final volume 50 µL with ddH20
Incubation: 30 min, 37 °C
Heatinactivation: 20 min, 80 °C
Agarose gel electrophoresis (1%)
100 V, 30 min
PCR and DNA purification kit (Macherey-Nagel)
Nucleo spin PCR and Monarch® PCR & DNA Clean-up protocol
Gibson Assembly protocol
transformation via heat shock protocol in E. coli DH5alpha
transformation via electroporation protocol in E. coli DH5alpha
PCR of T7RNAP_BL
Investigators: Olga Schmidt
Superior experiment: Construction of the selection plasmid
Procedure:
PCR
Go Tag (Promega) protocol
Danaturation: 56 °C
Elongation: 180 s
Agarose gel electrophoresis (1%)
100 V, 30 min
getting plasmids containing the Biobrick for further experiments
Investigators: Yannic Kerkhoff
Superior experiment: plasmid isolation of preculture of BBa_K2201220 clone 3
Procedure:
standard protocol
108,9 ng/µL
Combination of two biobricks to get the composite part BBa_K2201320 for protein expression
Investigators: Yannic Kerkhoff
Superior experiment: Biobrick Assembly of BBa_K608006 and BBa_K2201220
Procedure:
Standard BioBrick Assembly protocol
Backbone: BBa_J04450, 7,7µL
upstream: BBa_K608006, 3,6µL
downstram: BBa_K2201220, 5,0µL
Ligation time was 12 h at 8 °C
Colony and plasmid PCR of T7RNAP_BL c1-4
Investigators: Olga Schmidt
Superior experiment: Construction of the selection plasmid
Procedure:
PCR
Go Tag (Promega) protocol
Primer: VR & VF
Danaturation: 56 °C
Elongation: 180 s
Agarose gel electrophoresis (1%)
100 V, 30 min
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: heatshock transformation of the Biobrick Assembly product BBa_K2201320
Procedure:
standard heatshock protocol
used own produced chemo competend BL21(DE3)
streaked out on LB-plates with Cam
Colony PCR of pSB1C3_barnase (gblock) c1-70
Investigators: Olga Schmidt
Superior experiment: Construction of selection plasmid
Procedure:
PCR
Go Tag (Promega) protocol
Primer: Vr & VF2
Danaturation: 56.5 °C
Elongation: 40 s
Agarose gel electrophoresis (1%)
100 V, 30 min
Screening for positive transformations containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: GoTaq PCR of 19 clones of BBa_K2201320
Procedure:
standard GoTaq-protocol
template: colonie pcr clones of GA_K2201320
primer fwd: T7-eva_fwd
primer rev: Suffix_rev
annealing temperature: 51°C
extension time: 80 seconds
positive clone 19
multiplication of plasmids containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: preculture of the positive clone 19 of BBa_K2201320
Procedure:
standard procedure
1,25µl Cam in 5mL LB
Combination of two biobricks to get the composite part BBa_K2201321 for protein expression
Investigators: Yannic Kerkhoff
Superior experiment: Biobrick Assembly of BBa_K608006 and BBa_K2201221
Procedure:
Standard BioBrick Assembly protocol
Backbone: BBa_J04450, 2,0µL
upstream: BBa_K608006, 3,6µL
downstram: BBa_K2201221, 2,4µL
Ligation time was 12 h at 8°C
Primer annealing of 17jl & 17jm
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system - Primer annealing test
Procedure:
Both Primers were resuspended in ddH20 to a final concentration of 100 µM
HEPES annealing standard protocol
Aqua annealing standard protocol
Final volume: 20 µL
Agarose gel electrophoresis (2%)
60 V, 50 min
multiplication of plasmids containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: preculture of BL21(DE3) BBa_K2201320
Procedure:
standard procedure
1,25µl Cam in 10mL LB
12 h at 37°C
expression of the fusion protein
Investigators: Yannic Kerkhoff
Superior experiment: preculture of BL21(DE3) BBa_K2201320 with IPTG
Procedure:
standard procedure
1,25µl Cam in 10mL LB
12 h at 37°C
induced with IPTG 0.8mM
getting plasmids containing the Biobrick for further experiments
Investigators: Yannic Kerkhoff
Superior experiment: plasmid isolation of preculture of BBa_K2201320 clone 19
Procedure:
standard protocol
454,4 ng/µL
multiplication of plasmids containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: preculture of BL21(DE3) BBa_K2201320
Procedure:
standard procedure
1,25µl Cam in 10mL LB
12 h at 37°C
expression of the fusion protein
Investigators: Yannic Kerkhoff
Superior experiment: preculture of BL21(DE3) BBa_K2201320 with IPTG
Procedure:
standard procedure
1,25µl Cam in 10mL LB
12 h at 37°C
Ligation of pSB1C3 and barnase (gblock)
Investigators: Olga Schmidt
Superior experiment: Construction of selection plasmid
Procedure:
Ligation reaction
7 µL vector
3 µL gblock
2 µL T4 DNA ligase buffer
1 µL T4 DNA ligase
17 µL ddH20
Incubation: RT, 1 h
Colony PCR of pSB1C3_barnaseBL c1-60
Investigators: Olga Schmidt
Superior experiment: Construction of selection plasmid
Procedure:
PCR
Q5 High-fidelily (NEB) protocol
Denaturation: 56.5 °C
Elongation: 45 s
Agarose gel electrophoresis (1%)
100 V, 30 min
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: heatshock transformation of the Biobrick Assembly product BBa_K2201321
Procedure:
standard heatshock protocol
used own produced chemo competend BL21(DE3)
streaked out on LB-plates with Cam
- standard cultivation protocoll
Investigators: Yannic Kerkhoff
Superior experiment: Cultivation of BL21(DE3) with BBa_K2201320
Procedure:
2x 100mL LB-media with 25µL Cam in 1000mL flasks
innoculation with 3.3 mL preculture of OD 3 to get a starting OD of 0.1
incubation for four hours to a OD of 0.6
addition of 500µL 0.5mM IPTG each
incubation for 8 hours at 37°C and 24 hours at 18°C
harveting of the cells
Native PAGE of annealed primers (17jl & 17jm)
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system - Primer annealing test
Procedure:
Native DNA PAGE standard protocol
tested samples:
HEPES annealing (17jl & 17jm): 100 µM, 10 µM, 5 µM, 2.5 µM, 1 µM, 0.5 µM
Aqua annealing (17jl & 17jm): 100 µM, 10 µM, 5 µM, 2.5 µM, 1 µM, 0.5 µM
Variation: ~40 mA
2017-08-21   -   2017-08-27
Native PAGE of annealed primers (17jl & 17jm)
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system - Primer annealing test
Procedure:
Native DNA PAGE standard protocol
tested samples:
HEPES annealing (17jl & 17jm): 100 µM, 10 µM, 5 µM, 2.5 µM, 1 µM, 0.5 µM
Aqua annealing (17jl & 17jm): 100 µM, 10 µM, 5 µM, 2.5 µM, 1 µM, 0.5 µM
Variation: 100 V, 1 h
Plasmidisolation of pSB1C3_barnaseB c5, 22, 30, 31, 32, 45
Investigators: Olga Schmidt
Superior experiment: Construction of selection plasmid
Procedure:
Plasmid DNA purification kit (Zymo PURE)
Nucleo spin plasmid protocol
PCR of KanR & pSB1C3
Investigators: Olga Schmidt
Superior experiment: Construction of selection plasmid
Procedure:
PCR
Q5 High-fidelily (NEB) protocol
Primer: 17nx & 17ny (Kan)
Primer: 17od & 17oe (pSB1C3)
Denaturation: 54 °C & 60 °C
Elongation: 30 s & 75 s
Restriction Digest
5 µL 10x CutSmart buffer
1 µg plasmid
1 µL DpnI
filled up to final volume 50 µL with ddH20
Incubation: ON, 37 °C
Inactivation: 20 min, 80 °C
multiplication of plasmids containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: preculture of the positive clone 3 of BBa_K2201200
Procedure:
standard procedure
1,25µl Cam in 5mL LB
Restriction digest of KanR fragment
Investigators: Olga Schmidt
Superior experiment: Construction of selection plasmid
Procedure:
Restriction Digest
5 µL NEB2.1 buffer
2 µL DNA
1 µL XbaI
1 µL PstI
filled up to final volume 50 µL with ddH20
Incubation: 60 min, 37 °C
Heatinactivation: 20 min, 80 °C
Aim of the Experiment
Investigators: Yannic Kerkhoff
Superior experiment: GoTaq PCR of 32 clones of BBa_K2201200
Procedure:
Screening for positive transformations containing the Biobrick
standard GoTaq-protocol
template: colonie pcr clones of GA_K2201200
primer fwd: 17ns
primer rev: 17nt
annealing temperature: 56°C
extension time: 40 seconds
positive clone 20
Native PAGE of annealed primers (17jl & 17jm)
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system - Primer annealing test
Procedure:
Native DNA PAGE standard protocol
tested samples:
HEPES annealing (17jl & 17jm): 100 µM, 10 µM, 5 µM, 2.5 µM, 1 µM, 0.5 µM
Aqua annealing (17jl & 17jm): 100 µM, 10 µM, 5 µM, 2.5 µM, 1 µM, 0.5 µM
Variation: 60 V, 2 h
multiplication of plasmids containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: preculture of BL21(DE3) BBa_K2201321
Procedure:
standard procedure
1,25µl Cam in 5mL LB
getting plasmids containing the Biobrick for further experiments
Investigators: Yannic Kerkhoff
Superior experiment: plasmid isolation of preculture of BBa_K2201321 clone 3
Procedure:
standard protocol
319,1 ng/µL
Native Page of annealed primers (17jl & 17jm; 17jh & 17ji)
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system - Primer annealing test
Procedure:
Native DNA PAGE standard protocol
tested samples:
HEPES annealing (17jl & 17jm): 0.5 µM
Aqua annealing (17jl & 17jm): 0.5 µM
Composite Buffer annealing (17jh & 17 jm): 0.5 µM
all Primers (ssDNA): 1 µM
2017-08-28   -   2017-09-03
Native PAGE of annealed primers (HEPES annealing & Composite Buffer annealing) of the same concentration to test if all annealings have the same quality
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system - Primer annealing test
Procedure:
all samples were diluted to a final concentration of 0.5 µM
Native DNA PAGE standard protocol
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: heatshock transformation of the Biobrick Assembly product CP3
Procedure:
standard heatshock protocol
used own produced chemo competend BL21(DE3)
streaked out on LB-plates with Cam
Restriction digest of control annealings (mutC)
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system
Procedure:
Restriction Digest (MnlI) of aqua annealing sample: mutC (0.5 µM)
2 µL MnlI
5 µL CutSmart
25 µL mutC dsDNA
18 µL nuclease-free H2O
Incubation: 37 °C, 1 h
Inactivation: 20 min., 65 °C
Native DNA PAGE standard protocol
Tested samples:
digested mutC sample
native mutC sample
ssDNA primers (17jl & 17jm)
GoTaq PCR of 14 clones of BBa_K2201322
Investigators: Yannic Kerkhoff
Superior experiment: Screening for positive transformations containing the Biobrick
Procedure:
standard GoTaq-protocol
template: colonie pcr clones of BBA_K2201322
primer fwd: Präfix_fwd
primer rev: Suffix_rev
annealing temperature: 56°C
extension time: 40 seconds
positive: clone 3, 4, 5
Restriction digest of aqua & HEPES annealing
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system
Procedure:
Restriction Digest (MnlI) of aqua & HEPES annealing sample: mutC (0.5 µM)
2 µL MnlI
5 µL CutSmart
25 µL mutC dsDNA
18 µL nuclease-free H2O
Incubation: 37 °C, 1 h
Inactivation: 20 min., 65 °C
Native DNA PAGE standard protocol
Tested samples:
digested mutC sample (aqua & HEPES annealing)
native mutC sample
ssDNA primers (17jl & 17jm)
Aqua annealing of mutA, mutT, mutG and mutC
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system
Procedure:
Aqua annealing standard protocol (final concentration: 0.5 µM)
mutA: 17jf & 17jg
mutT: 17jj & 17jk
mutG: 17jh & 17ji
mutC: 17jl & 17jm
final volume of each sample: 50 µL
2017-09-04   -   2017-09-10
Restriction digest of control annealings (mutA, mutT, mutG, mutC) and aqua annealing of oligo2
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system
Procedure:
Oligo2 annealing
Aqua annealing standard protocol (final concentration: 0.5 µM)
Restricted samples:
MnlI -> mutC, oligo2
SapI -> mutG, oligo2
BsaI -> mutT, oligo2
EciI -> mutA, oligo2
Restriction Digest :
1 µL Enzyme
5 µL CutSmart
25 µL mutC dsDNA
19 µL nuclease-free H2O
Incubation: 37 °C, 1 h
Inactivation: 20 min., 65 °C
Native DNA PAGE standard protocol
SDS-Page and western blot of BBa_K2201321
Investigators: Yannic Kerkhoff
Superior experiment: SDS-Page and western blot of BBa_K2201321
Procedure:
standard SDS protocoll
standard Western blot protocoll
integration of the mRFP (BBa_J04450) in the aaRS plasmid as optical controle for the integration of an randomerized dsDNA
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
amplification of the mRFP (Phusion Master Mix), Primer vg, vh, annealing temperature: 60°C
amplification of the library backbone(Phusion Master Mix), Primer 17ve, 17vf, Annealing temperature: 61°C
Gibson Assembly with Gibson Master Mix + mRFP (3.1 µL) + library backbone (1.9 µL)
transformation in DH5a via heatshock
Generating the randomerized dsDNA for the library
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
ssDNA Annealing protocoll with the primers 17vi (1 µL), 17vj (1,4 µL)
agarose gelelectrophoresis, 3 %, positive result: bands of 120 bp
Generating the library
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Gibson Assembly Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the optical controle
transformation in DH5a via heatshock
2017-09-11   -   2017-09-17
Annealing, restriction digest and native PAGE of all samples
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system
Procedure:
Aqua annealing standard protocol (final concentration: 0.5 µM)
mutA: 17jf & 17jg
mutT: 17jj & 17jk
mutG: 17jh & 17ji
mutC: 17jl & 17jm
oligo2: olgio2_sense & oligo2_antisense
final volume of each sample: 50 µL
Restricted samples:
MnlI -> mutC, oligo2
SapI -> mutG, oligo2
BsaI -> mutT, oligo2
EciI -> mutA, oligo2
Restriction Digest :
1 µL Enzyme
5 µL CutSmart
25 µL mutC dsDNA
19 µL nuclease-free H2O
Incubation: 37 °C, 1 h
Inactivation: 20 min., 65 °C
Native DNA PAGE standard protocol
Generating the library
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Gibson Assembly Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the optical controle (8 x)
transformation in DH5a via heatshock (8 x)
Generating the library
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Gibson Assembly Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (16 x)
transformation in DH5a via heatshock (16 x)
Generating the library
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Gibson Assembly Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (25x)
transformation in DH5a via heatshock (25 x)
Generating the library
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Gibson Assembly Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (25x)
transformation in DH5a via heatshock (25 x)
Generating the library
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Gibson Assembly Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (25x)
transformation in DH5a via heatshock (25 x)
Annealing, restriction digest and native PAGE of all samples
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system
Procedure:
Aqua annealing standard protocol (final concentration: 0.5 µM)
mutA: 17jf & 17jg
mutT: 17jj & 17jk
mutG: 17jh & 17ji
mutC: 17jl & 17jm
oligo2: olgio2_sense & oligo2_antisense
final volume of each sample: 2x50 µL
Restricted samples:
MnlI -> mutC
SapI -> mutG, oligo2
BsaI -> mutT
EciI -> mutA
Restriction Digest :
1 µL Enzyme
5 µL CutSmart
25 µL mutC dsDNA
19 µL nuclease-free H2O
Incubation: 37 °C, 1 h
Inactivation: 20 min., 65 °C
Native DNA PAGE standard protocol
2017-09-18   -   2017-09-24
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: heatshock transformation of the EutS-part from CU Boulder
Procedure:
standard heatshock protocol
used own produced chemo competend DH5alpha
streaked out on LB-plates with Amp
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: heatshock transformation of the FusionRedTag-part from CU Boulder
Procedure:
standard heatshock protocol
used own produced chemo competend DH5alpha
streaked out on LB-plates with Kan
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: heatshock cotransformation of the EutS-part and FusionRedTag-part from CU Boulder
Procedure:
standard heatshock protocol
used own produced chemo competend DH5alpha
streaked out on LB-plates with AmpKan
Agarose gel electrophoresis of digested mutA, mutT, mutG & mutC
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system
Procedure:
Agarose gel electrophoresis (2%)
60 V, 40 min
Annealing, restriction digest and native PAGE of all samples
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system
Procedure:
Aqua annealing standard protocol (final concentration: 0.5 µM)
mutA: 17jf & 17jg
mutT: 17jj & 17jk
mutG: 17jh & 17ji
mutC: 17jl & 17jm
oligo2: olgio2_sense & oligo2_antisense
final volume of each sample: 2x50 µL
Restricted samples:
MnlI -> mutC
SapI -> mutG, oligo2
BsaI -> mutT
EciI -> mutA
Restriction Digest :
1 µL Enzyme
5 µL CutSmart
25 µL mutC dsDNA
19 µL nuclease-free H2O
Incubation: 37 °C, 1 h
Inactivation: 20 min., 65 °C
Native DNA PAGE standard protocol
Combination of two biobricks to get the part BBa_K2201200 in a tetracycline low copy plasmid
Investigators: Yannic Kerkhoff
Superior experiment: Biobrick Assembly of BBa_K2201200 in psB3T5
Procedure:
Standard BioBrick Assembly protocol
Backbone: pSB3T5, 2,0µL
upstream: BBa_K2201200, 3,6µL
Ligation time was 12 h at 8°C
cultivation
glycerine stocks of K2201200, K2201220, K2201221, K2201320, K2201321
Investigators: Yannic Kerkhoff
Superior experiment: glycerine stocks of K2201200, K2201220, K2201221, K2201320, K2201321
Procedure:
glycerine stock standard protocol
Screening for positive transformations containing the Biobrick
Investigators: Yannic Kerkhoff
Superior experiment: GoTaq PCR of 30 clones of K2201200 in pSB3T5
Procedure:
standard GoTaq-protocol
template: colonie pcr clones of K2201200 in pSB3T5
primer fwd: Präfix_fwd
primer rev: Suffix_rev
annealing temperature: 56°C
extension time: 40 seconds
positive clones 1, 3, 6
Preculture
Colony PCR of pSB3K5_mRFP_link_ccdB and pSB1C3_PtNTT2(31-575)-GFP
Investigators: Camilla Maerz
Superior experiment: Construction of pSB1C3-PlacUV5_PtNTT2
Procedure:
PCR
Go Tag (Promega) protocol
Primer: VR, VF2
Danaturation: 56 °C
Elongation: 90 s & 180 s
Agarose gel electrophoresis (1%)
100 V, 30 min
2017-09-25   -   2017-10-01
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: heatshock cotransformation of K2201200 in pSB3T5 and K2201321
Procedure:
standard heatshock protocol
used own produced chemo competend BL21(DE3)
streaked out on LB-plates with TetCam
getting plasmids containing the Biobrick for further experiments
Investigators: Yannic Kerkhoff
Superior experiment: plasmid isolation of preculture of BBa_K2201207
Procedure:
standard protocol
75,6 ng/µL
Biobrick Assembly
Aqua annealing of mutA-1, mutG-1, mutC-1, mutG-1 & oligo1
Investigators: Camilla Maerz
Superior experiment: M.A.X restriction system
Procedure:
Aqua annealing standard protocol (final concentration: 0.5 µM)
mutA-1: 17iv & 17iw
mutT-1: 17jb & 17jc
mutG-1: 17ix & 17iy
mutC-1: 17jd & 17jc
oligo1: olgio1_sense & oligo1_antisense
final volume: 50 µL
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: heatshock cotransformation of K2201240 and K2201207
Procedure:
standard heatshock protocol
used own produced chemo competend BL21(DE3)
streaked out on LB-plates with AmpCam
getting positive clones for further work
Investigators: Yannic Kerkhoff
Superior experiment: heatshock cotransformation of K2201241 and K2201207
Procedure:
standard heatshock protocol
used own produced chemo competend BL21(DE3)
streaked out on LB-plates with AmpCam
Q5-PRC + Gel + cleanUp
2017-10-02   -   2017-10-08
UBP-PCR with TiTaq
Investigators: Camilla Maerz
Superior experiment: PCR with UBP
Procedure:
PCR
PCR-UBP standard protocol (TiTaq)
samples:
Ligation of: mutA, mutT, mutG, mutC and oligo 2 (1 ng/µL)
Primer: 17vt & 17vu
Primer annealing: 56 °C
Elongation: 20 s
PCR and DNA purification kit (Macherey-Nagel)
Nucleo spin PCR and Monarch® PCR & DNA Clean-up protocol
Restriction Digest
1 µL Enzyme
5 µL CutSmart
25 µL mutC dsDNA
19 µL nuclease-free H2O
Incubation: 37 °C, 1 h
Inactivation: 20 min., 65 °C
Restricted samples:
MnlI -> mutC
SapI -> mutG
BsaI -> mutT
EciI -> mutA
EciI, BsaI, SapI, MnlI -> oligo2
Variation: oligo2 was digested with 0.5 µL of each enzyme
Agarose gel electrophoresis (2 %)
100 V, 20 min
Generating the library
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Gibson Assembly Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (25x)
transformation in DH5a via heatshock (25 x)
Kanymycin with amber stop codon
Investigators: Laura Schlueter
Superior experiment: positiv selection plasmid
Procedure:
amplification (~850 bp) of the kanamycin with primers 17jo, 17jp (no second MET codon) 71°C
amplification (~850 bp) of the kanamycin with primers 17jr, 17jp (two amber stop codon) 71°C
amplification of the K608007 backbone with the primers 17hb, 17hd  57°C
Generating the library
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Gibson Assembly Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (25x)
transformation in DH5a via heatshock (25 x)
UBP-PCR with GoTaq G2 and TiTaq
Investigators: Camilla Maerz
Superior experiment: PCR with UBP
Procedure:
PCR with TiTaq:
PCR-UBP standard protocol (TiTaq)
samples:
Ligation of: mutA, mutT, mutG, mutC and oligo 2 (5 ng/µL)
Primer: 17 vt & 17 vu
Primer annealing: 56 °C
Elongation: 20 s
PCR with TiTaq:
PCR-UBP standard protocol (TiTaq)
samples:
Ligation of: mutA, mutT, mutG and mutC (5 ng/µL)
Primer: VR & VF2
Primer annealing: 56 °C
Elongation: 20 s
PCR with GoTaq:
PCR-UBP standard protocol (GoTaq)
samples:
Ligation of: mutA, mutT, mutG, mutC and oligo2 (5 ng/µL)
Primer: 17 vt & 17 vu
Primer annealing: 56 °C
Elongation: 20 s
cloning of the aaRS in pSB3C5 (low copy)
Investigators: Laura Schlueter
Superior experiment: aaRS growth experiment
Procedure:
restriction of pSB3C5 and aaRS+tRNA (NPA, AcF, Prk) with EcoRI and PstI following the restriction protocol
extraction from the gel (Macherey-Nagel Kit)
ligation of pSB3C5 and aaRS+tRNA (NPA, AcF, Prk) with T4 Ligase (NEB)
Trafo via heatshock
sceening of the kanamycin (03.10.2017)
Investigators: Laura Schlueter
Superior experiment: positiv selection plasmid
Procedure:
Colony PCR (GoTaq, Primer VF2, VR, 56.5 °C)
Restriction digest of ligated oligo2
Investigators: Camilla Maerz
Superior experiment: PCR with UBP
Procedure:
Restriction Digest
5 µL CutSmart
30 µL ligated oligo2 DNA
2 µL EcoRV
filled up to final volume 50 µL with ddH20
Incubation: 120 min, 37 °C
Inactivation: 20 min, 65 °C
Gradient PCR of mutT
Investigators: Camilla Maerz
Superior experiment: PCR with UBP
Procedure:
PCR
PCR-UBP standard protocol (GoTaq G2)
sample: mutT (5 ng/µL)
Primer: 17 vt & 17 vu
Primer annealing: 50 - 56 °C
Elongation: 20 s
Agarose gel electrophoresis (2%)
100 V, 30 min
PCR of mutA, mutT, mutG & mutC with UBP and oligo2 without UBP
Investigators: Camilla Maerz
Superior experiment: PCR with UBP
Procedure:
PCR with TiTaq:
PCR-UBP standard protocol (TiTaq)
samples:
Ligation of: mutA, mutT, mutG and mutC (5 ng/µL), oligo2 (25 ng/µL)
Primer: VR & VF2
Primer annealing: 56 °C
Elongation: 20 s
Variation:
oligo2 without UBP, instead 1 µL H20 was added
0.5 µL isoG & 0.5 µL isoCMe were added to mutA, mutT, mutG and mutC
Generating the library
Investigators: Laura Schlueter
Superior experiment: aaRS Plasmid
Procedure:
plasmid isolation of the Tyr-RS library-mix
we stamped out the not randomized, negative, clones, easy to identify by the red colour because of the mRFP
washed from the cultivation plates with LB-media
restriction of the Library with EcoRI and PstI
proof of the restriction via agarose-gelelectrophoresis, 1 %
positve result: bands on ~2000 bp (backbone) and ~1000 bp (Tyr-RS with NNK)
Generating the library
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Gibson Assembly Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (25x)
transformation in DH5a via heatshock (25 x)
2017-10-09   -   2017-10-15
PCR with switched primers: VF2 & 17vu, VR & 17vt
Investigators: Camilla Maerz
Superior experiment: PCR with UBP
Procedure:
PCR with TiTaq:
PCR-UBP standard protocol (TiTaq)
samples:
Ligation of: mutA, mutT, mutG and mutC (5 ng/µL), oligo2 (25 ng/µL)
Primer: VF2 & 17vu, VR & 17vt
Primer annealing: 56 °C
Elongation: 20 s
PCR with GoTaq:
PCR-UBP standard protocol (GoTaq)
samples:
Ligation of: mutA, mutT, mutG and mutC (5 ng/µL), oligo2 (25 ng/µL)
Primer: VF2 & 17vu, VR & 17vt
Primer annealing: 56 °C
Elongation: 20 s
Agarose gel electrophoresis (2%)
100 V, 30 min
Generating the library
Investigators: Laura Schlueter
Superior experiment: aaRS plasmid
Procedure:
Gibson Assembly Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (25x)
transformation in DH5a via heatshock (25 x)
PCR with higher DNA template concentrations and gradient PCR of mutT
Investigators: Camilla Maerz
Superior experiment: PCR with UBP
Procedure:
PCR with TiTaq:
PCR-UBP standard protocol (TiTaq)
samples:
Ligation of: mutT (5 ng/µL) and oligo2 (50 ng/µL)
Primer: VF2 & 17vu
Primer annealing: 56 °C
Elongation: 20 s
PCR and DNA purification kit (Macherey-Nagel)
Nucleo spin PCR and Monarch® PCR & DNA Clean-up protocol
Restriction Digest
0.5 µL Enzyme
2.3 µL CutSmart
20 µL dsDNA
Incubation: 37 °C, 1 h
Inactivation: 20 min., 65 °C
Restricted samples:
BsaI -> mutT
EciI, BsaI, SapI, MnlI -> oligo2
Variation: oligo2 was digested with 0.5 µL of each enzyme
Agarose gel electrophoresis (2 %)
100 V, 20 min
Gradient PCR with TiTaq:
PCR-UBP standard protocol (TiTaq)
samples:
Ligation of: mutT (5 ng/µL)
Primer: VF2 & 17vu
Primer annealing: 50 - 58 °C
Elongation: 20 s
Agarose gel electrophoresis (2%)
100 V, 30 min
PCR of mutA, mutT, mutG, mutC and oligo2
Investigators: Camilla Maerz
Superior experiment: PCR with UBP
Procedure:
PCR with GoTaq:
PCR-UBP standard protocol (GoTaq)
samples:
Ligation of: mutA, mutT, mutG, mutC (5 ng/µL) and oligo2 (25 ng/µL)
Primer: VF2 & 17vu
Primer annealing: 56 °C
Elongation: 20 s
PCR and DNA purification kit (Macherey-Nagel)
Nucleo spin PCR and Monarch® PCR & DNA Clean-up protocol
Restriction Digest
1 µL Enzyme
2.3 µL CutSmart
20 µL dsDNA
Incubation: 37 °C, 12 h
Inactivation: 20 min., 65 °C
Restricted samples:
MnlI -> mutC
SapI -> mutG
BsaI -> mutT
EciI -> mutA
EciI, BsaI, SapI, MnlI -> oligo2
Agarose gel electrophoresis (2 %)
100 V, 30 min
positive selection with the library
Investigators: Laura Schlueter
Superior experiment: pos selection
Procedure:
using cultivation plates with kan (0.5 mM), Cm (0.25 mM), Tet (0.5 mM)
no growth
Variation of restriction digest
Investigators: Camilla Maerz
Superior experiment: PCR with UBP
Procedure:
Restriction Digest
3 µL Enzyme
2.5 µL CutSmart
20 µL dsDNA
Incubation: 37 °C, 1 h
Inactivation: 20 min., 65 °C
Restricted samples:
MnlI -> mutC
SapI -> mutG
BsaI -> mutT
EciI -> mutA
EciI, BsaI, SapI, MnlI -> oligo2
Agarose gel electrophoresis (2 %)
100 V, 30 min
positive selection with the library
Investigators: Laura Schlueter
Superior experiment: pos selection
Procedure:
Transformation (heatshock) of the library plasmid mix in competent cells containing the positive selction plasmid
using cultivation plates with kan (0.3 mM), Cm (0.25 mM), Tet (0.5 mM)
no growth
positive selection with the library
Investigators: Laura Schlueter
Superior experiment: pos selection
Procedure:
Transformation (heatshock) of the library plasmid mix in competent cells containing the positive selction plasmid
extended regeneration time: adding 0,5 mM IPTG after 1 h in SOC media
using cultivation plates with kan (0.3 mM), Cm (0.25 mM), Tet (0.5 mM), IPTG (0,5 mM)
no growth
growth experiments with the aaRS
Investigators: Laura Schlueter
Superior experiment: pos selection
Procedure:
comparing AcF-TAG, AcF-Leu, Cou-AS, Prk, NPA (on low and high copy plasmid) pSB1C3, pSB3T5
cultivation:
LB media , Cm (0.25 mM) / Tet (0.5 mM) and 1 mM of the ncAA
12 microtiter well plate, 600 rpm, 37°C
2017-10-16   -   2017-10-22
PCR with TiTaq and Q5-polymerase
Investigators: Camilla Maerz
Superior experiment: PCR with UBP
Procedure:
PCR with TiTaq:
PCR-UBP standard protocol (TiTaq)
samples:
Ligation of: mutA, mutT, mutG & mutC (5 ng/µL) and oligo2 (50 ng/µL)
Primer: VF2 & 17vu
Primer annealing: 56 °C
Elongation: 20 s
PCR and DNA purification kit (Macherey-Nagel)
Nucleo spin PCR and Monarch® PCR & DNA Clean-up protocol
PCR with Q5 High-Fidelity Polymerase -polymerase (per reaction, 25 µL):
5 µL Reaction buffer
1.25 µL dNTPs (2 mM)
1.25 µL 3'-Primer (10 mM)
1.25 µL 5'-Primer (10 mM)
1 µL DNA template
0.25 µL Q5 Hifi-Polymerase
14 µL H20
mutA, mutT, mutG, mutC: + 1 µL H20
oligo2: + 0.5 µL isoCme, + 0.5 µL isoG
Primer: VF2 & 17vu
Primer annealing: 56 °C
Elongation: 10 s
PCR and DNA purification kit (Macherey-Nagel)
Nucleo spin PCR and Monarch® PCR & DNA Clean-up protocol
Restriction Digest
1 µL Enzyme
20 µL dsDNA
2.3 µL CutSmart Buffer
Incubation: 37 °C, 15 h (BsaI & MnlI), 2 h (SapI & EciI)
Inactivation: 65 °C, 20 min
Restricted samples:
MnlI -> mutC
SapI -> mutG
BsaI -> mutT
EciI -> mutA
EciI, BsaI, SapI, MnlI -> oligo2
PCR with different polymerases
Investigators: Camilla Maerz
Superior experiment: PCR with UBP
Procedure:
samples:
Ligation of: mutA, mutT, mutG & mutC (5 ng/µL) and oligo2 (50 ng/µL)
PCR with Allin Hifi DNA Polymerase (per reaction, 25 µL):
5 µL ReactionBuffer
0.25 µL Hifi DNA Polymerase
2.5 µL 3'-Primer (10 mM)
2.5 µL 5'-Primer (10 mM)
1 µL DNA template
12.75 µL H20
mutA, mutT, mutG, mutC: + 1 µL H20
oligo2: + 0.5 µL isoCme, + 0.5 µL isoG
Primer: VF2 & 17vu
Primer annealing: 56 °C
Elongation: 10 s
PCR and DNA purification kit (Macherey-Nagel)
Nucleo spin PCR and Monarch® PCR & DNA Clean-up protocol
PCR with innuDRY Standard PCR MasterMix (per reaction, 20 µL):
10 µL innduDRY Mastermix
2 µL 3'-Primer (10 mM)
2 µL 5'-Primer (10 mM)
1 µL DNA template
5 µL H20
mutA, mutT, mutG, mutC: + 1 µL H20
oligo2: + 0.5 µL isoCme, + 0.5 µL isoG
Primer: VF2 & 17vu
Primer annealing: 54 °C
Elongation: 30 s
PCR and DNA purification kit (Macherey-Nagel)
Nucleo spin PCR and Monarch® PCR & DNA Clean-up protocol
PCR with BioMaster-HS Taq PCR Color (per reaction, 25 µL):
12.5 µL Mastermix
0.7 µL 3'-Primer (10 mM)
0.7 µL 5'-Primer (10 mM)
1 µL DNA template
10.1 µL H20
mutA, mutT, mutG, mutC: + 1 µL H20
oligo2: + 0.5 µL isoCme, + 0.5 µL isoG
Primer: VF2 & 17vu
Primer annealing: 51 °C
Elongation: 20 s
PCR and DNA purification kit (Macherey-Nagel)
Nucleo spin PCR and Monarch® PCR & DNA Clean-up protocol
PCR with Fire Polymerase (per reaction, 20 µL):
4 µL Mastermix
0.5 µL 3'-Primer (10 mM)
0.5 µL 5'-Primer (10 mM)
1 µL DNA template
14 µL H20
mutA, mutT, mutG, mutC: + 1 µL H20
oligo2: + 0.5 µL isoCme, + 0.5 µL isoG
Primer: VF2 & 17vu
Primer annealing: 56 °C
Elongation: 20 s
PCR and DNA purification kit (Macherey-Nagel)
Nucleo spin PCR and Monarch® PCR & DNA Clean-up protocol
PCR with Phusion polymerase (per reaction, 25 µL):
12.5 µL Mastermix
1.25 µL 3'-Primer (10 mM)
1.25 µL 5'-Primer (10 mM)
1 µL DNA template
9 µL H20
mutA, mutT, mutG, mutC: + 1 µL H20
oligo2: + 0.5 µL isoCme, + 0.5 µL isoG
Primer: VF2 & 17vu
Primer annealing: 56 °C
Elongation: 20 s
PCR and DNA purification kit (Macherey-Nagel)
Nucleo spin PCR and Monarch® PCR & DNA Clean-up protocol
Restriction Digest
1 µL Enzyme
20 µL dsDNA
2.3 µL CutSmart Buffer
Incubation: 37 °C, 15 h (BsaI & MnlI), 2 h (SapI & EciI)
Inactivation: 65 °C, 20 min
Restricted samples:
MnlI -> mutC
SapI -> mutG
BsaI -> mutT
EciI -> mutA
EciI, BsaI, SapI, MnlI -> oligo2
PCR with different polymerases
Investigators: Camilla Maerz
Superior experiment: PCR with UBP
Procedure:
Agarose gel electrophoresis with all samples (17.10)
Agarose gel electrophoresis (2%)
100 V, 40 min
