Team:Bielefeld-CeBiTec/Project/toolbox
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Toolbox Overview
Our toolbox for advanced protein design consists of eight different tRNA/aminacyl-synthetases (aaRS), each capable of loading a specific non-canonical amino acid (ncAA) to the amber tRNA or the less used leucine tRNA to incorporate these ncAAs into all types of target proteins. The aaRSs are provided as parts in the Registry of Standard Biological Parts, so that further iGEM teams will have the benefit of using our toolbox to improve their projects. The properties we integrated in our toolbox are Analyzing , Photoswitching , Labeling , Photolysis and Fusing (Figure 1).
Analyzing ( BBa_K2201201 , BBa_K2201202 ): Two different ncAAs are incorporated which are labeled with fluorophores in a chemically reaction. With the help of Foerster Resonance Energy Transfer (FRET), the distance between the amino acids could be measured, to get information about the conformation.
Photoswitching ( BBa_K2201207 ): A photoisomerisable amino acid could be incorporated which changes its conformation when it is irradiated with light of different wavelengths. This conformation change can be used to inhibit the functionality of the target protein, so that reactions can be switched on and off on the protein level using only light exposure.
Labeling ( BBa_K2201204 ): A fluorescent amino acid could be used to label the target protein in vivo. The advantages compared to fluorescent proteins lay in the smaller size of the fluorescent amino acid.
Photolysis ( BBa_K2201200 ): Photolysis amino acids break the peptide backbone at the position they were incorporated. This cleaving could be used to activate or deactivate proteins.
Fusing ( BBa_K2201208 , BBa_K2201302 ): Two ncAAs which form a specific covalent bond between each other can be used to fuse proteins together or immobilize the target protein, independent of the C- or N-termini.
Our toolbox can be used by transforming the BioBrick of the desired synthetase along with a target protein containing the amber codon. If the co‑transformation was successful, the amber codon will no longer be read as a stop signal in the protein synthesis but as a codon to incorporate the specific ncAA and it will enable the usage of their special properties.
We hope that our toolbox will be used frequently by future iGEM-teams and help them at their projects. We also wish that they will add new synthetases for other ncAAs and thus expand the possibilities of further applications in protein design.
Two Different Ways on how to Incorporate ncAAs
Generally, the incorporation in response to the amber stop codon (TAG) and the less used leucine codon (CTA) limits the number of different non-canonical amino acids to two. The incorporation through these codons also inhibits the growth of culture, as described in the translation mechanism . An additional base in the DNA would circumvent these problems and offers 64 new different codons to incorporate different ncAAs as shown in Figure 2. Both, the incorporation through codons containing an unnatural base and the incorporation through a less used existing codon are ways to incorporate ncAAs.
Figure 2: Expanded code sun. Representation of the code sun after the incorporation of one new unnatural base to expand the genetic code and create new blank codons (purple) that can be used to evolve aaRS to enable ncAAs for advanced protein design.
