<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_toolbox_photolysis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/toolbox/photolysis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/c/c0/T--Bielefeld-CeBiTec--gfpplate.jpeg);"><DIV id="title-bg"><DIV id="title-text">
						Photolysis
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Short Summary  </H2><DIV class="article">
                   The non-canonical amino acid (ncAA) 2-nitrophenylalanine (2-NPA) has the special property to induce a <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">cleavage</A> of the peptide backbone when irradiated with light of a wavelength of &gt; 300 nm. To demonstrate the usage of 2-NPA, we designed a fusion protein (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201321">BBa_K2201321</A>) consisting of the green fluorescent protein (GFP) and streptavidin connected by a glycine-glycine-serine-linker containing 2-NPA (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201200">BBa_K2201200</A>). The streptavidin compound will form a stable and highly specific non-covalent bond to biotin, so that the fusion protein can be easily immobilized on any biotinylated surface. The GFP is used as a fluorescence tag so that the fusion protein can be easily identified through its fluorescent properties. The immobilized fusion protein can then be irradiated with light by our <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">LED panel</A> to induce the cleavage of the peptide backbone and elute the target protein
                </DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Photolysis of Peptide Chains </H2><ARTICLE>
                    This system is an alternative procedure of site-specific protein cleavage compared to a cleavage by proteases or chemicals. A big advantage of this light-induced cleavage is, that it can be used almost universally on any purpose. It prevents disadvantages like an undesired cleavage of the target protein by undetected cleavage sides or a denaturation of the protein through chemical reagents. The system has a wide range of possible applications, such as inactivating proteins by cleaving them, activation by cleaving an inactive pre-protein and releasing an active form, or a combination with other methods as demonstrated in our light-induced elution.
                </ARTICLE><H4> Explanation of the ncAA </H4><ARTICLE>
                    The ncAA 2-NPA is able to photochemically cleave the polypeptide backbone by a cinnoline forming reaction based on the photochemistry of (2-nitrophenyl)ethane derivatives which are used as photochemical caging groups. Peters <I>et al.</I> (2009) report that upon photolysis, the nitrobenzyl group rearranges to the α-hydroxy-substituted nitrosophenyl group. The nitroso group then undergoes an additional reaction with the N-terminal amide group to generate the cyclic azo product. Subsequent hydrolysis of the activated carbonyl group results in the terminal cinnoline and carboxylate products (Peters <I>et al.</I>, 2009).
                </ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 1a: Reaction scheme of the proposed mechanism of photocleavage reaction by Peters <I>et al.</I>, 2009</B>. Excited by UV-light, the 2-NPA performes a self cyclization and thus cleaves the peptide chain. The residues contain a terminal cinnoline group and carboxy group.
                </P></DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 1b: Animation of the proposed mechanism of photocleavage reaction by Peters <I>et al.</I>, 2009.</B> The amino acids (blue) are linked to each other (black). The 2-NPA (purple) converts to its cinnoline form (cyclization) when irradiated with UV-light and thus induces the cleavage.
                </P></DIV><H4> Characteristics of the ncAA </H4><DIV class="contentline"><DIV class="half left"><ARTICLE><UL><LI> Name: 		2-Nitro-L-phenylalanine
                        </LI><LI> Short:		2-NPA
                        </LI><LI> CAS:		19883-75-1
                        </LI><LI> MW:		210.19
                        </LI><LI> Storage:           2-8 °C
                        </LI><LI> Source:		<A target="_blank" href="http://www.apolloscientific.co.uk/display_item.php?id=40307">apolloscientific</A></LI><LI> Prize:		5 g - 298.12 $
                        </LI><LI> Function:          induces a cleavage of the peptide backbone when radiated with ʎ&gt;300 nm
                    </LI></UL></ARTICLE></DIV><DIV class="half right"><DIV class="figure medium"><P class="figure subtitle"><B>Figure 2: Structure of 2-nitrophenylalanine.</B></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Theoretical Basis </H2><H4> Light-Induced Elution</H4><ARTICLE>
                    As an application, we wanted to use 2-NPA in a new purification system for recombinant proteins, similar to affinity chromatography and inspired by a paper form Peters <I>et al.</I>, 2009. They cleaved a short model peptide containing 2-NPA to show the ability of the ncAA to induce a cleavage of the protein backbone after irradiation with light with a wavelength of 367 nm. We thought about this model peptide as a linker between the target and the binding protein to establish a new light-induced elution system (Figure 3).
                </ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 3:</B> Overview of the light induced elution process with our fusion protein containing 2-NPA in the protein purification column.
                </P></DIV><ARTICLE>
                    We use a recombinant fusion protein with streptavidin as binding unit and a medium gly-gly-ser-linker with 2-NPA connecting the streptavidin with a target protein. As a proof of concept, we used GFP as the target protein because of its easy optical detection.
                    We expect that the fusion protein in unfiltered cell lysate will bind strong and specifically to the <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">purification column</A> with biotinylated glass slides, so that the other proteins and cell fragments can be easily washed away. We then want to irradiate the slides with light of 485 nm wave length to detect the GFP and prove the binding efficiency of the streptavidin and the functionality of the selected linker. Afterwards, we want to irradiate the column with UV-light of 367 nm wave length to induce the photocleavage of the 2-NPA. In the following elution step the GFP will be eluted while other proteins that were bound unspecific to the biotinylated surface should not be effected by the irradiation and retain on the column. The elution of the GFP can then also be detected as well as the fluorescence of the eluate. 
                    After using the purification column it should be easily regenerated by simply washing it with SDS-solution. The SDS will denatur the streptavidin with the linker and the other proteins bound to the column so that they will lose their binding affinity to the biotin and be washed off the glass slides. The biotin itself should not be influenced by the SDS-solution so that the glass slides will still be usable for many purification steps.
                    To implement all this, we started the development of a purification column, containing the requierd biotinylated surfaces and an <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">LED panel</A> that is able to radiate the required UV-light with a wave length of 367 nm.
                </ARTICLE><H4 id="GFP"> Green Fluorescent Protein: GFP </H4><ARTICLE>
                    GFP is a fluorescent protein isolated from the jellyfish <I>Aequorea victoria</I>. It is one of the most widely studied and used proteins in biochemistry and cell biology. It has been established as a marker for gene expression and protein targeting in living cells and organisms. It is 238 amino acids long with a molecular weight of 26.5 kDa. Inside the protein a self-assembled fluorophore is located which leads to its fluorescence properties. The wild type Aequorea protein has a major excitation peak at 395 nm and a minor peak at 475 nm. In aqueous solution (pH= 7.0), excitation at 395 nm leads to an emission peak at 508 nm, whereas excitation at 475 nm causes a maximum emission at 503 nm (visible green light)	(Tsien, 1998). In our project, eGFP is used as a fusion protein to enable detection by exciting with a wave length of 485 nm.
                </ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 4: Three perspectives of the photoproduct of the tetrameric wild type <I>Aequorea victoria</I> green fluorescent protein (GFP) from rcsb.org</B></P></DIV><ARTICLE>
                    Besides GFP, which emits a visible green light, there are other versions of fluorescent proteins like the red fluorescent protein (RFP), able to emit all types of colors. They are mostly used to label more than one protein of interest and compare their localization and quantity <I>in vivo</I>.
                </ARTICLE><DIV class="figure small"><P class="figure subtitle"><B>Figure 5: Seven versions of fluorescent proteins that emit light in different colors when exited with a specific wave length <A target="_blank" href="http://pdb101.rcsb.org/motm/174">rcsb.org</A>.</B></P></DIV><H4> Streptavidin</H4><ARTICLE>
                    Streptavidin is a tetrameric protein with a molecular weight of 15 kDa for each of the four subunits. It was isolated from the actinobacterium <I>Streptomyces avidinii</I> and is homologous to avidin. Both proteins can bind up to four molecules of biotin and their derivatives with high affinity resulting in a high dissociation constant. This leads to its widespread use in diagnostic assays and protein tags that require formation of an irreversible and specific linkage between biological macromolecules (Lichty <I>et al.</I>, 2005). 
                </ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 6: Three perspectives of the structure of a wild type streptavidin tetramer complex with biotin from <A target="_blank" href="https://www.rcsb.org/pdb/explore/explore.do?structureId=3RY1">rcsb.org.</A></B></P></DIV><ARTICLE>
                   In our project, streptavidin is used as a tag enable immobilization of the target protein on a biotinylated surface.
                </ARTICLE><H4 id="Fusionprotein"> Fusion Proteins</H4><ARTICLE>
                    Fusion proteins have been developed as a class of novel biomolecules with multi-functional properties. By successfully genetically fusing two or more proteins together, the product will have the desired properties from each component. The successful construction of a fusion protein requires the component proteins and suitable linkers. They were inspired by naturally-occurring multi-domain proteins with different subunits that are covalently linked together (Weber <I>et al.</I>, 1989). Fusion proteins can contain the whole native protein sequences or just parts of it dependent on the desired function and size of the resulting product protein. 
                </ARTICLE><DIV class="figure small"><P class="figure subtitle"><B>Figure 7: Fusion protein of EGFP and Cytochrome b562 from rcsb.org.</B></P></DIV><ARTICLE>
                    In the design process of recombinant fusion proteins, linkers are very important as they can increase the detectability, the stability, bioactivity, and expression yield of the fusion protein (Chen <I>et al.</I>., 2013). Additionally, a direct fusion of functional domains without a linker can lead to misfolding and other undesirable properties. Linker sequences can be received from natural multi-domain proteins or be designed depending on the desired characteristics. They are mainly classified in three different groups: flexible linkers (Figure 8 A), rigid linkers (Figure 8 B), and cleavable linkers (Figure 8 C). Furthermore they can be classified as small (approximately five amino acids), medium (approximately ten) or large linkers (approximately 20 to 28 amino acids) (Weber  <I>et al.</I>, 1989).
                </ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 8: Three groups of protein linkers.</B> A: flexible, B: rigid, C: cleavable. 
                </P></DIV><ARTICLE>
                    In our project, we used a designed medium-sized gly-gly-ser-linker with a length of eleven amino acids. The small amino acids will allow a good expose of the 2-NPA to the UV-light while the glycines mediate the flexibility and the serines improve the solubility of the fusion protein and prevent a hydrophobic collapse. 
                </ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> References </H2><B>Lichty, J.J., Malecki, J.L., Agnew, H.D., Michelson-horowitz, D.J., and Tan, S.</B> (2005). Comparison of affinity tags for protein purification. <B>41</B>: 98–105.<B>Patricia C. Weber, D. H. Ohlendorf, J. J. Wendoloski and  F. R. Salemme</B> (1989). Structural Origins of High-Affinity Biotin Binding to Streptavidin. Science. <B>243</B>:  85-88.<B>Peters, F.B., Brock, A., Wang, J., and Schultz, P.G.</B> (2009). Photocleavage of the Polypeptide Backbone by 2-NitrophenylalaninePeters. Chem. Biol.<B> 16</B>: 148–152.<B>Roger Y. Tsien </B>(1998). The Green Fluorescent Protein. Annu. Rev. Biochem. 1998. <B>67</B>:509–44.<B>Xiaoying Chen, Jennica Zaro, and Wei-Chiang Shen</B> (2013). Fusion Protein Linkers: Property, Design and Functionality. Adv Drug Deliv Rev. <B>65(10)</B>: 1357–1369.
        </DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>