<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_unnatural_base_pair_uptake_and_biosynthesis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/unnatural base pair/uptake and biosynthesis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/4/4b/T--Bielefeld-CeBiTec--title-img-biosynthesis.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Uptake and Biosynthesis of iso-C<SUP>m</SUP>TP and iso-GTP
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Short Summary</H3><ARTICLE>
A crucial aspect in the development of semisynthetic organisms is the supply with unnatural nucleoside triphosphates Two main strategies exist to provide <I>Escherichia coli</I> with unnatural nucleoside triphosphates. The first strategy is based on the uptake of nucleoside triphosphates from the media, which requires the expression of a heterologous transporter, facilitating the uptake across the inner membrane. The second strategy is based on the <I>in vivo</I> biosynthesis of the desired nucleoside triphosphates, which can be achieved by integration of heterologous or synthetic pathways. For our project, we pursuit both paths, analyzing uptake of unnatural nucleoside triphosphates by different variants of a nucleotide transporter and investigating the biosynthesis of isoguanosine using a pathway from the plant <I>Croton tiglium</I>.  
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Strategies to Supply the Cell with iso-C<SUP>m</SUP>TP and iso-GTP  </H3><ARTICLE>
		The first essential step to ensure that unnatural base pairs are retained over a long period of time is to provide the cell with the unnatural bases required for the replication of the DNA containing the unnatural base pair. There are two different strategies for it. 
The first one is to supplement the media with the (deoxy-)nucleoside triphosphates of the desired unnatural bases. To allow uptake of the nucleoside triphosphates, the cell needs to possess a transporter that can facilitate a sufficient flux into the cell. While this strategy seems to be the easiest, it also has some disadvantages. First, <I>E. coli</I> does not possess nucleoside triphosphate transporters suitable for the transport of unnatural nucleoside triphosphates. Given that nucleoside triphosphates such as ATP are absent in the extracellular environment, such transporters have not evolved in <I>E. coli</I>. Another reason for the absence of nucleoside triphosphate transporters from the cell membrane is that the existence of such transporters could potentially put the intracellular ATP pool at risk. Therefore, a heterologous transporter has to be introduced. Secondly, chemically synthesized nucleotides are very expensive, so feeding them to the cells would make bioprocesses uneconomically for scale up. On the other hand, this strategy allows the testing of highly unique unnatural bases if uptake can be ensured. 

The second strategy is based on the <I>in vivo</I> biosynthesis of the desired unnatural bases. This strategy would allow scaling up of processes without making them uneconomical and would eliminate the need for a heterologous transport system. But the biosynthesis of fully synthetic bases is, if possible at all, extremely challenging. Furthermore, to allow the incorporation of those bases into the DNA, phosphorylation of the nucleosides has to be ensured which is facilitated by in part highly specific kinases. Therefore, this strategy represents an optimal system which as of now is extremely challenging to implement.

		</ARTICLE><DIV class="figure seventy"><P class="figure subtitle"><B>Figure 1: Two strategies for making iso-C<SUP>m</SUP>TP and iso-GTP available to the cell.</B> The first strategy (left) is based on a heterologous transporter which can facilitate the transport of iso-C<SUP>m</SUP>TP and iso-GTP. The media is supplemented with the unnatural (deoxy)nucleoside triphosphates, which are then transported into the cell and incorporated into the DNA. The second strategy (right) is more complex and is based on the <I>de novo</I> synthesis of the unnatural (deoxy)nucleoside triphosphates. Therefore, existing pathways from natural sources or newly designed pathways have to be introduced to the cell. The nucleoside triphosphates are then incorporated into the DNA. </P></DIV><H3> The Nucleotide Transporter from <I>Phaeodactylum tricornutum</I><I>Pt</I>NTT2</H3><ARTICLE>
Given the structural similarity to the natural bases, iso-dC<SUP>m</SUP> and iso-dG might be taken up through the existing nucleoside transporters in sufficient amounts. But phosphorylation of the nucleosides remains a challenge. Therefore, the introduction of a heterologous nucleoside triphosphate transporter into the desired strain represents a promising alternative for the direct uptake of iso-dC<SUP>m</SUP>TP and iso-dGTP from the media. 
<DIV class="contentline"><DIV class="half left">
			 The algae <I>Phaeodactylum tricornutum</I>, a diatom of the genus <I>Phaedactylum</I>, features six putative nucleotide transporters (NTTs). Two isoforms of these NTTs have been characterized by Ast and coworkers and it was shown that both isoforms facilitate transport across the plastid membrane. While isoform 1 (NTT1) acts as a proton-dependent adenine nucleotide importer, NTT2 facilitates the counter exchange of (deoxy-)nucleoside triphosphates (Ast <I>et al</I>., 2009). 
			</DIV><DIV class="half right"><DIV class="figure large"><P class="figure subtitle"><B>Figure 2: Uptake of α<SUP>32</SUP>-labeled nucleotides by the two isoforms <I>Pt</I>NTT1 and <I>Pt</I>NTT2 when expressed in <I>E. coli</I> (Ast <I>et al</I>., 2009)</B> The uptake of α<SUP>32</SUP>-labeled nucleotides was measured in <I>E. coli</I>. Isoform 1 (A) was shown to transport adenosine mono-, di- and triphosphates, while isoform 2 (B) shows a specificity for nucleoside triphosphates. </P></DIV></DIV></DIV><DIV class="contentline"><DIV class="half right">
			Ast and colleagues demonstrated the subcellular localization by tagging the transporters with GFP. Figure 3 shows the subcellular localization of the two isoforms <I>Pt</I>NTT1 and <I>Pt</I>NTT2. The fluorescence signal can only be detected within the cell around the plastid and not within the cell membranes. The isoform 2 of the nucleotide transporter was shown to be an unspecific (deoxy-)nucleoside transporter, facilitating the uptake of CTP, GTP, dCTP, ATP, UTP, dGTP, dATP and TTP when expressed in <I>E. coli</I>, as shown in Figure 2 (Ast <I>et al</I>., 2009). Uptake was measured using α<SUP>32</SUP>-labeled nucleotides. While isoform 1 (Figure 2A) can transport adenosine mono-, di- and triphosphates, isoform 2 (Figure 2B) shows a specificity for nucleoside triphosphates. The fact that <I>Pt</I>NTT2 can accept a broad range of different nucleotides makes the transporter interesting for the transport of unnatural nucleotides, as long as they are provided as nucleoside triphosphates. 
			</DIV><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 3: Subcellular localization of <I>Pt</I>NTT1 and <I>Pt</I>NTT2 in 
<I>Phaeodactylum tricornutum</I> (Ast <I>et al</I>., 2009).</B><I>Pt</I>NTT1 and <I>Pt</I>NTT2 were fused to GFP to study the subcellular localization. </P></DIV></DIV></DIV>
Zhang <I>et al</I>. 2017 integrated <I>Pt</I>NTT2 into the chromosome of <I>E. coli</I> BL21(DE3) under control of the lacUV5 promoter. To demonstrate its feasibility for the uptake of nucleotides, uptake of [α-<SUP>32</SUP>P]-dATP was measured. The native sequence of <I>Pt</I>NTT2 features an N-terminal signal sequence directing the subcellular localization to the plastid membrane. In <I>E. coli</I>, this signal sequence is likely to be retained, leading to a growth defect in cells expressing the native <I>Pt</I>NTT2 transporter. Therefore, a truncated version of <I>Pt</I>NTT2, <I>Pt</I>NTT2(66-575), was used. The chromosomally integrated, truncated, and codon optimized <I>Pt</I>NTT2 (65-575) under control of PlacUV5 was shown to be an optimal compromise between efficient uptake and the growth limitation resulting from expression of the heterologous protein (Zhang <I>et al</I>., 2017).

	<DIV class="figure eighty"><P class="figure subtitle"><B>Figure 4: Uptake of α<SUP>32</SUP>labeled ATP by the different versions of <I>Pt</I>NTT2 (Zhang <I>et al</I>., 2017).</B> The expression of <I>Pt</I>NTT2 was investigated in different strains, under control of different promotors, and plasmid-bound as well as integrated into the chromosome. In their final design, Zhang and colleagues integrated <I>Pt</I>NTT2 chromosomally in <I>E.coli</I> BL21(DE3) under control of the lacUV5 promoter</P></DIV>

For our experiments, we decided to use <I>Pt</I>NTT2 to facilitate the transport of unnatural nucleoside triphosphates into the cell. Therefore, our plan is to construct several versions of <I>Pt</I>NTT2 under control of the lacUV5 promoter, including the native sequence <I>Pt</I>NTT2(1-575), two truncated versions <I>Pt</I>NTT2(31-575) and <I>Pt</I>NTT2(66-575), as well as two versions with heterologous signal peptides, <I>Pt</I>NTT2-Tat and <I>Pt</I>NTT2-pelB. While <I>Pt</I>NTT2-Tat features a signal peptide targeting the twin-arginine translocation pathway, <I>Pt</I>NTT2-pelB features the signal peptide of the pectate lyase B of <I>Erwinia carotovora</I>, which targets the general secretion pathway. By characterizing all of these versions we hope to identify the best suited version for our purpose, the efficient uptake of unnatural nucleoside triphosphates. 
		

<H2> Sec- and Tat Secretion Pathways </H2><I>Pt</I>NTT2 comes from an eukaryote and is integrated into the plastid membrane in its native algael host. Therefore, expression of <I>Pt</I>NTT2 in <I>E. coli</I> might not lead to efficient integration into the inner membrane. Therefore, we wanted to test different signal peptides to find the one best suited for efficient integration of <I>Pt</I>NTT2 into the inner membrane. 
In <I>E. coli</I> , proteins can be secreted via two main pathways, namely the general secretion (Sec) and the twin-arginine translocation (Tat) pathway . Proteins which are intended to be secreted are targeted by signal peptides which are located at the N-terminus of a pre-protein and share one common tripartite structure: a positively charged N-terminal domain (N-domain) followed by a nonpolar hydrophobic domain (H-domain) and the more polar C-domain (De Keyzer <I>et al.</I>, 2003; von Heijne, 1998). Signal peptides vary in length, but are usually between 18 and 30 amino acids long. During the translocation process, the signal peptide is cleaved from the pre-protein by a signal peptide peptidase, which recognizes a cleavage site within the C-domain of the signal peptide (De Keyzer <I>et al.</I>, 2003). Signal peptides targeting the Sec pathway do not feature a common consensus sequence or motif (Caspers <I>et al.</I>, 2010). However, the binding of the secretion specific chaperone SecB or the signal recognition particle SRP  (which represents the first step of the Sec pathway) to the signal peptide is influenced by the hydrophobicity of the H-domain (De Keyzer <I>et al.</I>, 2003). Signal peptides targeting the Tat pathway are usually around 30 amino acids long (Berks <I>et al.</I>, 2014). On the contrary to Sec pathway targeting signal peptides, Tat pathway targeting signal peptides do feature a consensus sequence. The crucial element for Tat recognition is a twin-arginine motif (RR) located at the N/H-domain junction. In bacteria, the RR is usually followed by phenylalanine, leucine and lysine at the RR+2 position with especially the phenylalanine being of high importance (Cline, 2015).

<H3> Sec Pathway </H3>
In bacteria, most proteins are secreted via the conserved and essential Sec pathway in an unfolded state (Schneewind and Missiakas, 2014; Caspers and Freudl, 2008). The key component of the Sec pathway is the translocase which is located in the plasma membrane and composed of the SecYEG heterotrimeric protein as well as the SecA ATPase. SecYEG consists of three components, SecY, SecE and SecG. SecY and SecE form the core of the transmembrane channel, while SecG is an additional integral subunit that seems to be not essential for the translocation process (Schneewind and Missiakas, 2014; de Keyzer <I>et al.</I>, 2003; Caspers and Freudl, 2008). SecA serves as the molecular motor of the translocation process by consecutively binding and hydrolyzing ATP and hence initializing the translocation. The proton motive force (PMF) helps translocating the protein once the process is initialized by SecA (Caspers and Freudl, 2008). SecYEG can associate with another heterotrimeric complex involved in the translocation process consisting of SecD, SecF, and YajC. The exact role of this additional heterotrimeric complex is yet unknown but seems to optimize secretion   (de Keyzer <I>et al.</I>, 2003). It was proposed that SecDFYajC might be involved in the proton motive force dependent steps of the translocation process (Denks <I>et al.</I>, 2014). Proteins are targeted in two different ways, being mediated by the chaperone SecB, or the signal recognition particle (SRP) . SecB and SRP both bind the nascent pre-protein but seem to bind pre-proteins of different lengths. Proteins that are to be secreted are usually targeted via the SecB mechanism (de Keyzer <I>et al.</I>, 2003). No matter which mechanism is involved in the targeting process, the proteins eventually interact with the translocase. Gradual translocation of the pre-protein is catalyzed by the translocase, during which the pre-protein is further processed by the signal peptide peptidase (SPase). The SPase is embedded in the periplasmic membrane and removes the signal peptide of the pre-protein (Denks <I>et al.</I>, 2014; de Keyzer <I>et al.</I>, 2003).

<DIV class="figure fifty"><P class="figure subtitle"><B>Figure 5: Schematic overview over the Sec-pathway and the involved components.(de Keyzer <I>et al.</I>, 2003). </B> Pre-proteins are either targeted by SRP or SecB and translocated by the translocase. The translocase consists of SecYEG, forming the transmembrane channel, and SecA, the ATP-driven molecular motor of the translocation. The signal peptide is eventually removed by the SPase (here Lep) .</P></DIV><H3> Tat Pathway </H3>

Opposite to the Sec pathway, which secretes proteins while they are not fully folded, proteins secreted via the twin-arginine translocase (Tat) pathway are fully folded (Cline, 2015). The Tat pathway is an active transport system with the transmembrane proton motive force being the driving force of the translocation (Palmer and Berks, 2012). Three proteins from only two structural families form the machinery required for Tat mediated secretion (Berks <I>et al.</I>, 2014). The first family called TatA is described as a single transmembrane helix with a short N-terminal transmembrane domain and an unstructured C-tail (Berks <I>et al.</I>, 2014; Cline, 2015). TatB belongs to the TatA family but was shown to be abdicable for the basic functionality. The second structural family involved in the Tat pathway is TatC, consisting of six transmembrane helices with the N- and C-termini being exposed to the cytoplasm. The structure of TatC was determined for <I>Aquifex aeolicus</I> and is shown together with the structure of TatA from <I>E. coli</I> in Figure 6 (Berks <I>et al.</I>, 2014; Cline, 2015). A fourth component, TatE provides an overlapping function to TatA, hence making TatE dispensable (Kikuchi <I>et al.</I>, 2009).

<DIV class="figure fifty"><P class="figure subtitle"><B>Figure 6: Structures of TatA from <I>E. coli</I> and TatC from <I>Aquifex aeolicus</I></B> While TatA forms a single transmembrane helix, TatC consists of six transmembrane helices (Berks <I>et al.</I>, 2014).</P></DIV>

TatB and TatC form the receptor complex, which is composed of up to eight TatBC units (Cline, 2015). The signal peptide is bound by this receptor complex, which triggers the PMF dependent assembly of TatA to the translocation pore (Palmer and Berks, 2012; Cline, 2015). The whole complex is referred to as translocase. After the assembly, the substrate is translocated and the signal peptide is cleaved by a signal peptide peptidase as shown in Figure 7. Many details of the Tat pathway are not known yet or uncertain, including how TatA disassembles (Cline, 2015).

<DIV class="figure fifty"><P class="figure subtitle"><B>Figure 7: Steps of the Tat translocation system</B> After the signal peptide is bound by TatBC TatA assembles and forms the translocation pore. Using the transmembrane proton motive force, the substrate is translocated and the signal peptide cleaved by a signal peptidase (Palmer and Berks 2012) </P></DIV></ARTICLE></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Croton Tiglium </H3><H4> Origin of <I>Croton Tiglium</I></H4><ARTICLE><I>Croton tiglium (L.)</I>, also commonly known as the „Croton oil plant“ is a plant of the family of Euphorbiaceae that was first described in 1753 by Carl Linnaeus.  Its origin lays in the south-eastern asian countries such as India or Thailand, where it can grow up to 7m tall.  In nature, it occurs in many habitats ranging from 300-1500m height as well as from shrublands to forests.

The plant we used was kindly provided by the <A href="https://www.uni-marburg.de/botgart">botanic garden of the Philipps University in Marburg</A>. Seeds of this plant were originally given to the botanical garden Marburg in 1986. They were then provided by the botanical garden Giessen.
					</ARTICLE><H4> Usages of <I>Croton Tiglium</I></H4><ARTICLE>
Besides the fact that all components of croton tiglium are poisonous, it is widely used as an herb of the traditional chinese medicine since 2000 years. There, it found its usage as an oil to treat skin diseases and  intoxications and as a laxative. However, many of these former usages were abandoned becuase of the high toxity. Further, the plant was used against cancerous diseases due to its anti-tumoric effects that were proven in 1994 (Kim <I>et al</I>).
					</ARTICLE><H4> Secondary Metabolism of <I>C. Tiglium</I></H4><ARTICLE>
In difference to animals, plants usually have a second metabolism that does not interact with any chemicals needed for primary functions like growth or development. However, the seondary metabolism is still connected to the primary metabolism as it uses some of the substances produces within. The products that emerge from the second metabolism offer a wide range of possible usages for humanity due to their great diversity. However, the pants mostly use them as protection against herbivores and pathogens. Some metabolites are toxic whereas others do attract parasites and predators as a protection against the herbivores.
Besides from many other secondary metabolites, croton tiglium produces iso-guanosine. 
In 1932, iso-guanosine was first isolated from croton tiglum (Cherbuliez et al 1932). Today, it also commercially available as crotonoside. Iso-guanosine was object of many reaearches and is found to have a lot of effects onto biological processes (see Lowry and Brown, 1952; Huang et al 1973; Vasu and David ,1985 for reference) including antitumorous effects (Kim et al 1942)

For us, one of the main aspects that is interesting on iso-guanosine is its use as an unnatural base within organisms. 
					</ARTICLE></DIV></DIV><DIV class="container"><DIV class="contentbox"><DIV class="content"><H3><I>De novo</I> Synthesis of Purine Bases</H3><DIV class="figure eighty"><P class="figure subtitle"><B>Figure 1: <I>De novo</I> Synthesis of Pyrimidine Bases</B> . </P></DIV>
		Purine bases are produced <I>de novo</I> directly on the ribose (Berg et al., 2012). The synthesis starts with the replacement of the pyrophosphate of PRPP with an amino group, yielding phosphoribosylamine (PRA). This reaction is catalyzed by amidophosphoribosyltransferase (ATase) and also uses the ammonia from a glutamine side-chain as the donor of the amino group. The conversion of PRPP   to PRA is a committing  step in the purine biosynthesis. The synthesis of the purine ring involves nine additional steps with the first six reactions being relatively similar. In every reaction, an oxygen atom which is bound to a carbon atom is activated by phosphorylation and a subsequent substitution by ammonia or an amino-group, which act as a nucleophile agent. These subsequent reactions lead to the formation of inosinate (IMP), which acts as a key intermediate in the purine synthesis. Inosinate is converted into either AMP or GMP. AMP is synthesized by a substitution of the C-6 carbonyl oxygen with an amino group by adenylosuccinate synthase (ASS). In this reaction, GTP instead of ATP is used as a donor of the phosphoryl group. The conversion of IMP to GMP is catalyzed by the GMP synthase and starts with the oxidation of IMP to xanthylate (XMP) and the subsequent addition of an amino group. In a second step, XMP is converted into GMP, a reaction that requires ATP as a donor for an AMP group. GMP and AMP are again phosphorylated to GTP and ATP by specific kinases. 
		<H3>Conversion of Ribonucleosid Diphosphates to Deoxyribonucleotides </H3><DIV class="figure eighty"><P class="figure subtitle"><B>Figure 1: <I>De novo</I> Synthesis of Purine Bases</B></P></DIV><ARTICLE>Deoxyribonucleotides are synthesized from ribonucleotides by substitution of the 2´-hydroxyl group of the ribose by a hydrogen. The reaction is catalyzed by the enzyme ribonucleotide reductase, which is strongly conserved in all living organisms (Berg et al., 2012).  In <I>E. coli</I>, two main types of ribonucleotide reductases exist. Ribonucleoside-triphosphate reductases can convert ribonucleoside-triphosphates into deoxyribonucleoside-triphosphates, while ribonucleoside-diphosphate reductases convert ribonucleoside-diphosphates to deoxyribonucleoside-diphosphates (Kanehisa and Goto, 2000).
		</ARTICLE><H3>Salvage Pathways</H3><ARTICLE>
		Both purine and pyrimidine bases can be recycled and converted into the corresponding nucleotides through salvage pathways. Adenine can be recycled through conversion into AMP, a reaction that is catalyzed by the adenine phosphoribosyltransferase and requires PRPP . AMP can then be subsequently converted into ATP or dATP as described above. Hypoxanthine guanine phosphoribosyltransferase (HGPRT) catalyzes the recycling of guanosine, a reaction that also requires PRPP as a donor for a phosphate. HGPRT also catalyzes the conversion of hypoxanthine to IMP which again is a precursor of GMP and AMP. The recycling of thymine involves two steps: in the first step, thymine is converted to thymidine by the thymidine phosphorylase. In a second step, thymidine is converted to TMP by thymidine kinase. Cytosine can be recycled by conversion to uracil, a reaction that is catalyzed by cytosine deaminase. Following the conversion to UTP, CTP is produced by CTP synthase. The recycling of bases saves intracellular energy, since the <I>de novo</I> synthesis requires large amounts of ATP. Therefore, the recycling of bases through salvage pathways is usually favored by cells.
		</ARTICLE><H3>Adaptions of the purine pathway used in <I>C. Tiglium</I> for the production of iso-Guanosine<ARTICLE>
The ordinary form of Guanosine is produced within the purine pathway. Thus, it is likely that the production of iso-guanosine  is realised inbetween these reactions. 
</ARTICLE></H3><H2> References </H2><ARTICLE><B>Ast, M., Gruber, A., Schmitz-Esser, S., Neuhaus, H.E., Kroth, P.G., Horn, M., and Haferkamp, I.</B> (2009). Diatom plastids depend on nucleotide import from the cytosol. Proc. Natl. Acad. Sci. U. S. A. <B>106</B>: 3621–3626.
</ARTICLE><ARTICLE><B>Berks, B.C., Lea, S.M., and Stansfeld, P.J. </B>(2014). Structural biology of Tat protein transport. Curr. Opin. Struct. Biol. <B>27</B>: 32–7.
</ARTICLE><ARTICLE><B>Caspers, M., Brockmeier, U., Degering, C., Eggert, T., and Freudl, R.</B> (2010). Improvement of Sec-dependent secretion of a heterologous model protein in Bacillus subtilis by saturation mutagenesis of the N-domain of the AmyE signal peptide. Appl. Microbiol. Biotechnol. <B>86</B>: 1877–1885.
</ARTICLE><ARTICLE><B>Caspers, M. and Freudl, R. </B>(2008). Corynebacterium glutamicum possesses two secA homologous genes that are essential for viability. Arch. Microbiol. <B>189</B>: 605–610.
</ARTICLE><ARTICLE><B>Cline, K. </B>(2015). Mechanistic Aspects of Folded Protein Transport by the Twin Arginine Translocase (Tat). J. Biol. Chem. <B>290</B>: 16530–16538.
</ARTICLE><ARTICLE><B>Denks, K., Vogt, A., Sachelaru, I., Petriman, N., Kudva, R., and Koch, H.</B> (2014). The Sec translocon mediated protein transport in prokaryotes and eukaryotes. Mol. Membr. Biol. <B>31</B>: 58–84.
</ARTICLE><ARTICLE><B>von Heijne, G. </B>(1998). Life and death of a signal peptide. Nature <B>396</B>: 111, 113.
</ARTICLE><ARTICLE><B>de Keyzer, J., van der Does, C., and Driessen, A.J.M. </B>(2003). The bacterial translocase: a dynamic protein channel complex. Cell. Mol. Life Sci. <B>60</B>: 2034–2052.
</ARTICLE><ARTICLE><B>Kikuchi, Y., Itaya, H., Date, M., Matsui, K., and Wu, L.F. </B>(2009). TatABC overexpression improves Corynebacterium glutamicum Tat-dependent protein secretion. Appl. Environ. Microbiol. <B>75</B>: 603–607.
</ARTICLE><ARTICLE><B>Palmer, T. and Berks, B.C. </B>(2012). The twin-arginine translocation (Tat) protein export pathway. Nat. Rev. Microbiol. <B>10</B>: 483–496.
</ARTICLE><ARTICLE><B>Schneewind, O. and Missiakas, D. </B>(2014). Sec-secretion and sortase-mediated anchoring of proteins in Gram-positive bacteria. Biochim. Biophys. Acta - Mol. Cell Res. <B>1843</B>: 1687–1697.
</ARTICLE><ARTICLE><B>Zhang, Y., Lamb, B.M., Feldman, A.W., Zhou, A.X., Lavergne, T., Li, L., and Romesberg, F.E.</B> (2017). A semisynthetic organism engineered for the stable expansion of the genetic alphabet. Proc. Natl. Acad. Sci. <B>114</B>: 1317–1322.
</ARTICLE><ARTICLE><B>Berg, J.M., Tymoczko, J.L., and Stryer, L. </B>(2012). Biochemistry 7th Edition. (Springer-Verlag: Berlin Heidelberg).<B>Kanehisa, M. and Goto, S.</B> (2000). Kyoto Encyclopedia of Genes and Genomes. Nucleic Acids Res. <B>28</B>: 27–30.
</ARTICLE><ARTICLE><B>http://www.biologyreference.com/Re-Se/Secondary-Metabolites-in-Plants.html</B><B>Isolation of Isoguanosine from Croton tiglium and Its Antitumor Activity. Jung Han Kim, Sang Jun Lee, Young Bok Han, Jung Jo Moon, Jong Bae Kim;  Arch. Pharm. Res. Vol. 17, No.2 , pp. 115-118, 1994</B><B>https://de.wikipedia.org/w/index.php?title=Croton%C3%B6l&amp;oldid=165692497 </B><B>http://www.nationaalherbarium.nl/ThaiEuph/ThCspecies/ThCrotonT.htm </B><B>http://en.hortipedia.com/wiki/Croton_tiglium </B><B>Cherbuliez, E. and Bernhard, K., Croton seed(1)crotonoside. Helv. Chim. Act&amp;, 15, 464, 978-982 (1932).</B><B>Huang, M., Shimizu, H. and Daly, J. W., Accumulation of cyclic adenosine monophosphate in incubated slices of brain tissue. J. Med. Chem., 15, 462-468 (1972) </B><B>Lowry, B. A. and Brown, G. B., The utilization of purine nucleosides for nucleic acid synthesis in the rat. J. BioL Chem., 197, 591 (1952). </B>


Vasu, N. and David, A. Y., A New Synthesis of Isogua-
nosine. J. Org. Chem., 50, 406-408 (1985).

</ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>