<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Notebook_Labjournal skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Notebook/Labjournal</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/9/9a/T--Bielefeld-CeBiTec--title-img-labjournal.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Labjournal
					</DIV></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-04-03   -   2017-04-09 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> electroporation transformation of BBa_K1416000</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard electroporation protocol</LI><UL><LI> used electro competend DH5alpha from NEB</LI><LI> streaked out on LB-plates with Cam</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> electroporation transformation of BBa_KJ36848</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard electroporation protocol</LI><UL><LI> used electro competend DH5alpha from NEB</LI><LI> streaked out on LB-plates with Cam</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication of plasmids containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> preculture of transformed BBa_K1416000</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard procedure</LI><UL><LI>  1,25µl Cam in 5mL LB</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-04-10   -   2017-04-16 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting plasmids containing the Biobrick for further experiments </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> plasmid isolation of preculture of BBa_K1416000</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard protocol</LI><UL><LI>  517,9 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication of plasmids containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> preculture of transformed BBa_J36848</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard procedure</LI><UL><LI>  1,25µl Cam in 5mL LB</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting plasmids containing the Biobrick for further experiments </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> plasmid isolation of preculture of BBa_J36848</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard protocol</LI><UL><LI>  231,8 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-05-29   -   2017-06-04 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> heatshock transformation of BBa_E0400</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard heatshock protocol</LI><UL><LI> used own produced chemo competend DH5alpha</LI><LI> streaked out on LB-plates with Amp</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-06-05   -   2017-06-11 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication of plasmids containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> preculture of transformed BBa_E0400</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard procedure</LI><UL><LI>  5,0µl Amp in 5mL LB</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting plasmids containing the Biobrick for further experiments </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> plasmid isolation of preculture of BBa_E0400</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard protocol</LI><UL><LI>  88,4 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> heatshock transformation of BBa_K525998</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard heatshock protocol</LI><UL><LI> used own produced chemo competend DH5alpha</LI><LI> streaked out on LB-plates with Cam</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication of plasmids containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> preculture of transformed BBa_K525998</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard procedure</LI><UL><LI>  1,25µl Cam in 5mL LB</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting plasmids containing the Biobrick for further experiments </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> plasmid isolation of preculture of BBa_K525998</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard protocol</LI><UL><LI>  140,9 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-06-12   -   2017-06-18 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Assemble the fragments to get the Part BBa_K2201220 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Gibson assembly of F1,F3,F5</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard <A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> protocol</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Cellgrowth with the barnaseplasmid </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt, Denise Kerkhoff, Christina Drake</P><P><B>Superior experiment:</B> Overnightculture of barnase</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> 3 ml <A target="blank" href="https://static.igem.org/mediawiki/2017/a/a7/T--Bielefeld-CeBiTec--protocol_LB.pdf">LB-medium</A> and 0,75  µl chloramphenicol</LI><LI> inkubation overnight 37°C, 140rpm</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> To get and purified the plasmid </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> Plasmidisolation of barnase-plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  protocol from Macherley-Nagel: Plasmid DNA purification, protocol 5: NucleoSpin</LI><LI>  concentration measurement by nanoprop</LI><LI>  sequenzingorder</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Duplicate the parts </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake, Denise Kerkhoff</P><P><B>Superior experiment:</B> Transformation via Distribution of the parts BBA_J04450, BBa_I746909, BBa_K80800 and BBa_psB1K5</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/ab/T--Bielefeld-CeBiTec--protocol_electrotrafo.pdf">transformation via electroporation</A></LI><LI>  plated out: BBa_J04450 on tetracycline, BBa_I74909 and BBa_K808000 on chloramphenicol, BBa_psB1K5 on kanamycin</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Duplicate the part </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake, Denise Kerkhoff</P><P><B>Superior experiment:</B> Transformation of BBa_psB6A1</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/ab/T--Bielefeld-CeBiTec--protocol_electrotrafo.pdf">transformation via electroporation</A></LI><LI> plated out on amphenicol</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Cellgrowth with the plasmids </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Denise Kerkhoff, Christina Drake</P><P><B>Superior experiment:</B> Overnightculture of BBA_J04450, BBa_I746909 and BBa_K808000</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> 3 ml <A target="blank" href="https://static.igem.org/mediawiki/2017/a/a7/T--Bielefeld-CeBiTec--protocol_LB.pdf">LB-medium</A> and at BBA_J04450 3µl tetracycline and atBBa_I746909 and BBa_K808000 0,75  µl chloramphenicol</LI><LI>  inkubation overnight 37°C, 140rpm</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> To get and purified the plasmid </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> Plasmidisolation of BBA_J04450, BBa_I746909 and BBa_K80800-plasmids</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  protocol from Macherley-Nagel: Plasmid DNA purification, protocol 5: NucleoSpin</LI><LI>  concentration measurement by nanoprop:</LI><UL><LI> BBA_J04450 26ng/µl and 25,7ng/µl</LI><LI> BBa_I746909 90,2ng/µl</LI><LI> 104,3 ng/µl -BBa_K808000 118,3 ng/µl</LI></UL><LI> and 146,6 ng/µl</LI><LI>  sequenzingorder</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-06-19   -   2017-06-25 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Cellgrowth with the plasmid </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Denise Kerkhoff</P><P><B>Superior experiment:</B> Overnightculture of BBA_psB3K5</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> 3 ml <A target="blank" href="https://static.igem.org/mediawiki/2017/a/a7/T--Bielefeld-CeBiTec--protocol_LB.pdf">LB-medium</A> and  2,5 µl kanamycin</LI><LI>  inkubation overnight 37°C, 140rpm</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> heatshock transformation of assembled fragments F5,F2,F4</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard heatshock protocol</LI><UL><LI> used own produced chemo competend DH5alpha</LI><LI> streaked out on LB-plates with Cam</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication, verification and purification of the needed fragment F2: GFPLinker2 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Q5-PCR, gel and PCR-cleanUp of F2</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard Q5-PCR protocol</LI><UL><LI>  template: BBa_E0400</LI><LI>  primer fwd: 17gk</LI><LI>  primer rev: 17gn</LI><LI>  annealing temperature: 58°C</LI><LI>  extension time: 50 seconds</LI></UL><LI>  standard PCR-cleanUp protocol</LI><UL><LI>  125,8 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication, verification and purification of the needed fragment F4: StrepLinker2 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Q5-PCR, gel and PCR-cleanUp of F4</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard Q5-PCR protocol</LI><UL><LI>  template: BBa_KJ36848</LI><LI>  primer fwd: 17gp</LI><LI>  primer rev: 17gq</LI><LI>  annealing temperature: 61°C</LI><LI>  extension time: 30 seconds</LI></UL><LI>  standard PCR-cleanUp protocol</LI><UL><LI>  127,0 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication, verification and purification of the needed fragment F5: linearized pSB1C3 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Q5-PCR, gel and PCR-cleanUp of F5</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard Q5-PCR protocol</LI><UL><LI>  template: K1416000</LI><LI>  primer fwd: 17gj</LI><LI>  primer rev: 17gl</LI><LI>  annealing temperature: 59°C</LI><LI>  extension time: 120 seconds</LI></UL><LI>  standard PCR-cleanUp protocol</LI><UL><LI>  258,4 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Assemble the fragments to get the Part BBa_K2201221 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Gibson assembly of F2,F4,F5</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard <A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> protocol</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Screening for positive transformations containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> GoTaq PCR of 7 clones of BBa_K2201221</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard GoTaq-protocol</LI><UL><LI>  template: colonie pcr clones of GA_K2201221</LI><LI>  primer fwd: Präfix_fwd</LI><LI>  primer rev: Suffix_rev</LI><LI>  annealing temperature: 56°C</LI><LI>  extension time: 40 seconds</LI></UL><LI>  positive clone 2</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication of plasmids containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> preculture of the positive clone 2 of BBa_K2201221</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard procedure</LI><UL><LI>  1,25µl Cam in 5mL LB</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting plasmids containing the Biobrick for further experiments </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> plasmid isolation of preculture of BBa_K2201200 clone 20</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard protocol</LI><UL><LI>  240,6 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting plasmids containing the Biobrick for further experiments </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> plasmid isolation of preculture of BBa_K2201221 clone 2</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard protocol</LI><UL><LI>  208,8 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-06-26   -   2017-07-02 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Check the sequence </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> Sequenzingorder barnase</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> concentration measurement by nanoprop: 177 ng/µl</LI><LI> result: sequence was not barnase</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Plasmidisolation and gibson assembly of pSB1C3-PlacUV5_PtNTT2 c30 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> Construction of pSB1C3-PlacUV5_PtNTT2</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Plasmid DNA purification kit</LI><UL><LI> Nucleo spin plasmid protocol</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> protocol</LI><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/a0/T--Bielefeld-CeBiTec--protocol_heatshock.pdf">transformation via heat shock</A> protocol in <I>E. coli</I> DH5<I>alpha</I></LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Colony PCR of T7RNAP from <I>E. coli</I> KRX </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Construction of the positive selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR with Grimson Taq polymerase (per reaction, 25 µL):</LI><UL><LI> 5 µL 5x Buffer</LI><LI> 2.5 µL dNTPs (2 mM)</LI><LI> 0.5 µL 3'-Primer (10 mM)</LI><LI> 0.5 µL 5'-Primer (10 mM)</LI><LI> 0.125 µL Grimson Taq polymerase</LI><LI> 1 picked bacterial colony as template</LI><LI> ad 25 µL H20</LI></UL><LI> Primer annealing: 64 °C</LI><LI> Elongation: 150 s</LI><LI> Agarose gel electrophoresis (1%)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Isolate T7RNAP from the chromosome of KRX-e.coli </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> PCR of T7RNAP from KRX-e.coli</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/8/88/T--Bielefeld-CeBiTec--protocol_PCR_Q5_HF_Polymerase.pdf">Q5 High-Fidelity PCR</A></LI><UL><LI> annealingtemparatur 64°C, elogationtime 1 min</LI></UL><LI> 1%-agarose-<A target="blank" href="https://static.igem.org/mediawiki/2017/d/d4/T--Bielefeld-CeBiTec--protocol_TAE_buffer.pdf">TAE</A>-gel</LI><UL><LI> 1kb Marker Geneuler and 5 µl sample with 1 µl 6*Loading Dye</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Plated out Barnase from strain collection </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> Plated out Barnase from strain collection</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> solved DNA in 100 µl <A target="blank" href="https://static.igem.org/mediawiki/2017/f/fe/T--Bielefeld-CeBiTec--protocol_SOC.pdf">SOC-medium</A></LI><LI> plated out on chloramphenicolplate</LI><LI> overnightincubation at 37°C</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Colony PCR of pK18mobsacB-del_codA </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> Construction of pK18mobsacB_codA_del</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Go Tag (Promega) protocol</LI><UL><LI> Primer: VR, VF2</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 140 s</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> PCR of pSB1C3 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> General</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR</LI><UL><LI> Q5 High-fidelily (NEB) protocol</LI><LI> Denaturation: 64 °C</LI><LI> Elongation: 60 s</LI></UL><LI> Agarose gel electrophoresis (1%)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Gibson assembly of T7RNAP in pSB1C3 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Construction of the selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> protocol</LI><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/a0/T--Bielefeld-CeBiTec--protocol_heatshock.pdf">transformation via heat shock</A> protocol in <I>E. coli</I> BL21</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Colony PCR of pSB1C3_T7RNAP c1-8 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Construction of the selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR</LI><UL><LI> Q5 High-fidelily (NEB) protocol</LI><LI> Denaturation: 64 °C</LI><LI> Elongation: 80 s</LI></UL><LI> Agarose gel electrophoresis (1%)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Colony PCR of barnase (26.06.2017) c1&amp;2 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Construction of selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR</LI><UL><LI> Go Tag (Promega) protocol</LI><LI> Danaturation: 58 °C</LI><LI> Elongation: 35 s</LI></UL><LI> Agarose gel electrophoresis (1%)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-07-03   -   2017-07-09 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Check the correct barnasegen </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> PCR of barnase</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Go-Taq-protocol</LI><UL><LI> annealingtemparatur 58°C, elogationtime 30 s</LI></UL><LI> 1%-agarose-<A target="blank" href="https://static.igem.org/mediawiki/2017/d/d4/T--Bielefeld-CeBiTec--protocol_TAE_buffer.pdf">TAE</A>-gel</LI><UL><LI> 100bp Marker Geneuler and 5 µl sample with 1 µl 6*Loading Dye</LI><LI> result: PCR-product could not be barnase</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-07-10   -   2017-07-16 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Biobrick Assembly of 2B1 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/8/84/T--Bielefeld-CeBiTec--protocol_Standard_Biobrick.pdf">BioBrick Assembly</A></LI><UL><LI> RuBisCoTAG mRFP as upstream part, T7-Terminator as downstream part, pSB1A3 as destination plasmid</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/a0/T--Bielefeld-CeBiTec--protocol_heatshock.pdf">transformation via heat shock</A> and DH5α Cells</LI><UL><LI> used 5µl from the ligation</LI></UL><LI> plated out on Amp plates and let them grow over night</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Check the sequence </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> Sequenzingorder barnase</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> result: sequence was not barnase</LI></UL><LI> &gt;16.07.2017</LI><LI> tRNA</LI><LI> Overnightculture of TyrRS-Heidelberg-plasmid</LI><LI> CDR</LI><LI> Cellgrowth with the TyrRS-Heidelberg-plasmid</LI><UL><LI> 3 ml <A target="blank" href="https://static.igem.org/mediawiki/2017/a/a7/T--Bielefeld-CeBiTec--protocol_LB.pdf">LB-medium</A> and 0,75  µl chloramphenicol</LI><LI>  inkubation overnight 37°C, 140rpm</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Transformation of T7-Promotor </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/ab/T--Bielefeld-CeBiTec--protocol_electrotrafo.pdf">transformation via electroporation</A> of cells with T7-Promotor</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Plasmidisolation of TyrRS-Heidelberg plasmid </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Construction of selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Plasmid DNA purification kit (Macherey-Nagel)</LI><UL><LI> Nucleo spin plasmid protocol</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Preculture of colonies with T7-Promotor </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> used 5µl <A target="blank" href="https://static.igem.org/mediawiki/2017/a/a7/T--Bielefeld-CeBiTec--protocol_LB.pdf">LB media</A> and kanamycin</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-07-17   -   2017-07-23 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Isolate the tRNA </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> PCR of TyrRS-Heidelberg-plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> plasmidisolation with MN protocol 5</LI><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/8/88/T--Bielefeld-CeBiTec--protocol_PCR_Q5_HF_Polymerase.pdf">Q5 High-Fidelity PCR</A></LI><UL><LI> annealingtemparatur 53°C, elogationtime 20s</LI></UL><LI> Gibbsonassembly with 5µl backbone and 0.5 µl insert</LI><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/a0/T--Bielefeld-CeBiTec--protocol_heatshock.pdf">transformation via heat shock</A></LI><LI> overnight incubation at 37°C</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Isolation of T7-Promotor </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Nucleospin Plasmidisolation protocol</LI><UL><LI> got four products with the concentrations 50.1 ng/µl, 16.3 ng/µl, 24,2 ng/µl and 2.1 ng/µl</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Colony PCR with 2B1 Colonies </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/c/cc/T--Bielefeld-CeBiTec--protocol_colonyPCR_GoTaq_G2.pdf">GoTaq® G2 PCR</A></LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Biobrickassembly of 2B2 to 2B8, B1 and A1 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/8/84/T--Bielefeld-CeBiTec--protocol_Standard_Biobrick.pdf">BioBrick Assembly</A></LI><UL><LI> digest and ligate RuBisCo parts as upstreampart and T7-Terminator/Promotor as downstreampart and pSB1A3 as destination plasmid</LI><UL><LI> ligate TAG2 and T7-Terminator to 2B2</LI><LI> ligate TAG111 and T7-Terminator to 2B3</LI><LI> ligate TAG474 and T7-Terminator to 2B4</LI><LI> ligate TAG2+TAG111 and T7-Terminator to 2B5</LI><LI> ligate TAG2+TAG474 and T7-Terminator to 2B6</LI><LI> ligate TAG111+TAG474 and T7-Terminator to 2B7</LI><LI> ligate TAG2+TAG111+TAG474 and T7-Terminator to 2B8</LI><LI> ligate RuBisCo mRFP and T7-Terminator to B1</LI><LI> ligate Carboxysome and T7-Promotor to A1</LI></UL><LI> preculture of positive 2B1 colony</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Transformation of test devices </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> Interlab study</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> dilute test devives (distribution plate 6 &amp; 7) in 10 µL ddH20</LI><LI> Transformation</LI><UL><LI> 1 µL plasmid from each plate</LI><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/a0/T--Bielefeld-CeBiTec--protocol_heatshock.pdf">transformation via heat shock</A> protocol in <I>E. coli</I> XL1-blue</LI></UL></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Transformation of 2B2-2B8, B1 and A1 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/a0/T--Bielefeld-CeBiTec--protocol_heatshock.pdf">transformation via heat shock</A></LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Plasmidisolation of 2B1 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Nucleospin Plasmidisolation protocol</LI><UL><LI> got 47 ng/µl and 41 ng/µl</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Reverse the insert </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> PCR of T7GFP</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/8/88/T--Bielefeld-CeBiTec--protocol_PCR_Q5_HF_Polymerase.pdf">Q5 High-Fidelity PCR</A></LI><UL><LI> Insert: annealingtemparatur 60°C,</LI><LI> Backbone:elogationtime 30s: annealingtemparatur 60°C, elogationtime 1 min</LI><LI> 1%-agarose-<A target="blank" href="https://static.igem.org/mediawiki/2017/d/d4/T--Bielefeld-CeBiTec--protocol_TAE_buffer.pdf">TAE</A>-gel</LI></UL><LI>  result: PCR didn`t work</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Reverse the insert </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> PCR of T7GFP</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> 1%-agarose-<A target="blank" href="https://static.igem.org/mediawiki/2017/d/d4/T--Bielefeld-CeBiTec--protocol_TAE_buffer.pdf">TAE</A>-gel</LI><LI>  result: PCR did work</LI><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A></LI><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/a0/T--Bielefeld-CeBiTec--protocol_heatshock.pdf">transformation via heat shock</A></LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> PCR of BBa_I746909 and pSB1C3 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Integration of CDS of BBa_I746909 in pSB1C3</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR of I746909</LI><UL><LI> Q5 High-fidelily (NEB) protocol</LI><LI> Primer: 17lk &amp; 17lj</LI><LI> Denaturation: 60 °C</LI><LI> Elongation: 40 s</LI></UL><LI> PCR of pSB1C3</LI><UL><LI> Q5 High-fidelily (NEB) protocol</LI><LI> Primer: 17ll &amp; 17li</LI><LI> Denaturation: 60 °C</LI><LI> Elongation: 70 s</LI></UL><LI> Agarose gel electrophoresis (1%)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Preparation of over-night culture </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> Interlab study</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Over-night culture of negative control, positive control, test device 1-6</LI><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/a7/T--Bielefeld-CeBiTec--protocol_LB.pdf">LB-media</A> + Cm were inoculated</LI><LI> Cultures were incubated over night (37°C, 200 rpm)</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-07-24   -   2017-07-30 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Plasmidisolation of pSB1C3_I746909 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Integration of CDS of BBa_I746909 in pSB1C3</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Plasmid DNA purification kit (Macherey-Nagel)</LI><UL><LI> Nucleo spin plasmid protocol</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Repeat the degestion and transformation from the 18th and 19th july </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/8/84/T--Bielefeld-CeBiTec--protocol_Standard_Biobrick.pdf">BioBrick Assembly</A></LI><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/a0/T--Bielefeld-CeBiTec--protocol_heatshock.pdf">transformation via heat shock</A></LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Cellgrowth with the plasmid </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> Overnightculture of tRNA-psB1C3</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> 3 ml <A target="blank" href="https://static.igem.org/mediawiki/2017/a/a7/T--Bielefeld-CeBiTec--protocol_LB.pdf">LB-medium</A> and 0,75  µl chloramphenicol</LI><LI>  inkubation overnight 37°C, 140rpm</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Cloning of the ori (pSB6A1) in pSB1C3 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS Plasmid (without aaRS)</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> Master Mix</LI><UL><LI> Backbone: 3.28 µL     Ori 1.0: 1.73 µL</LI><LI> Backbone: 0.32 µL     Ori 1.1: 4.68 µL</LI><LI> Backbone: 0.90 µL     Ori 3.0: 4.10 µL</LI></UL><LI> Trafo: heatshock</LI><LI> not successfull because of the wrong overlapps (try again)</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Colony PCR of the Retrafo of pRS (because of mixed culture) </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Sequencing was not succesfull (maybe mixed colony), so I did a retrafo with the aim of a successful sequencing (incorporation of the aaRS in pSB1C3)</LI><UL><LI> pRS 3: 1-5</LI><LI> pRS 4: 1-5</LI><LI> pRS 5: 1-5</LI><LI> pRS 7: 1-5</LI><LI> no positive result (no band at 1000 bp)</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Amplification of pSB1C3 for the cloning of the ori in pSB1C3 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/8/88/T--Bielefeld-CeBiTec--protocol_PCR_Q5_HF_Polymerase.pdf">PCR with Q5 High-Fidelity Polymerase</A> Master Mix, Primer jb,jn, 65°C</LI><LI> gelelectrophoresis, 1% Agarose</LI><LI> band  2000-2500 bp (supposed to be around 2200 bp)</LI><LI> Clean up from the gel</LI><UL><LI> pSB1C3(nur ori): 22.2 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Amplification of pSB1C3 fragments for a 4 part Gibson (see page 4 in the paper-labbook) </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR, Q5 Master Mix</LI><UL><LI> Primer jj, jn for fragment 2, 65°C</LI><LI> Primer jb, ht for fragment 4, 65°C</LI></UL><LI> gelelectrophoresis, 1% Agarose</LI><UL><LI> band at 300 (fragment 2, supposed to be around 326 bp)</LI><LI> band at 1200 (fragment 4, supposed to be around 1233 bp)</LI></UL><LI> Clean up from the gel</LI><UL><LI> fragment 2: 18.5 ng/µL  (overlapp(aaRS)_pSB1C3_overlapp(ori))</LI><LI> fragment 4: 19.1 ng/µL  (overlapp(ori)_pSB1C3_overlapp(aaRS))</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Colony PCR of 2B2-2B8, B1 and A1 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/c/cc/T--Bielefeld-CeBiTec--protocol_colonyPCR_GoTaq_G2.pdf">GoTaq® G2 PCR</A></LI><LI> most have negative results -&gt; create new destination plasmid pSB1A3</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Colony PCR of retrafo and more colonies of the pRS </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> GoTaq Master Mix, Primer vr,vf</LI><UL><LI> 2 strong bands at 400 bp and 600 bp, small bands of pRS3 6 and pRS4 9 at around 1200-1500 bp</LI><LI> bands of pRS31, pRS32, pRS34, pRS36, pRS39 at around 1200-1500 bp</LI></UL><LI> GoTaq Master Mix, Primer hq, jk (<A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> primer) with the colonies pRS31, pRS32, pRS34, pRS36, pRS39</LI><UL><LI> pRS31, pRS32, pRS39 : strong bands at around 1200 bp</LI></UL><LI> the two positive colony PCRs (vf-vr, hq-jk) for these probes, successful integration of the Tyr-aaRS in pSB1C3 seems to be successfull</LI><LI> pRS31, pRS32, pRS39 send to sequencing</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> pRS31, pRS32, pRS39 plasmid isolation </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Macherey-Nagel purification kit</LI><UL><LI> pRS31: 324.1 ng/µL</LI><LI> pRS32: 231.8 ng/µL</LI><LI> pRS39: 225.5 ng/µL</LI></UL><LI> send tosequencing</LI><LI> pRS31, pRS39 have the pSB1C3 with the Tyr-aaRS incorporated (but on position 32, the Arginine (cgt) should be changed to an Leucine (ctg), but it is not. On position 290 there really is a missing Leu, on position 268 the Asparagin is changed to an Arginine)</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-07-31   -   2017-08-06 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> construct a biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> Aquacloning of tRNA and backbone</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> transformation via heatschock</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> plasmid isolation of the positive colonies of the retrafo pRS4 4 (wrong!) and  pRS3 12 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Macherey-Nagel purification kit with each 3 mL culture</LI><UL><LI> pRS4 4:  44   ng/µL</LI><LI> pRS3 12: 68.3 ng/µL</LI><LI> not send to sequencing jet</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> 4 part Gibson and Trafo </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> Master Mix with 5 µl template</LI><UL><LI> pSB1C3 fragment (2):       18.5 ng/µL  around  350 bp   2.2 µL</LI><LI> pSB1C3 fragment (4):       19.1 ng/µL  around 1300 bp   0.8 µL</LI><LI> oK1.1 (pMB1 from pSB6A1):   5.8 ng/µL  around 1300 bp   1.8 µL</LI><LI> Tyr-aaRS:                  68.5 ng/µL  around 1000 bp   0.2 µL</LI></UL><LI> Trafo via heatshock</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Gibson of the ori in pSB1C3 (pori) and Trafo </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> Master Mix with 5 µL template</LI><UL><LI> pSB1C3(nur ori): 22.2  ng/µL   around 2100 bp   1.57 µL</LI><LI> ok1.0:            5.80 ng/µL   around 1258 bp   3.43 µL</LI></UL><LI> Trafo via heatshock</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Transfer positive colonies </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> picked and transfered positiv colonies of 2B2, 2B6, 2B7, 2B8 and A1 onto a new plate</LI><LI> incubate over night</LI><LI> 2B2 and 2B8 are contaminated with negativ colonies</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> construct a biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> Aquacloning of tRNA and backbone</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> transformation via heatschock</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> construct a biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> Gibbsonassembly of barnase and backbone</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> transformation via heatschock</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Preculture of 2B6, 2B7 and A1 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> used 3µl <A target="blank" href="https://static.igem.org/mediawiki/2017/a/a7/T--Bielefeld-CeBiTec--protocol_LB.pdf">LB media</A></LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Cellgrowth with the plasmid </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> Overnightculture of tRNA-psB1C3 and barnase-psB1C3</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> 3 ml <A target="blank" href="https://static.igem.org/mediawiki/2017/a/a7/T--Bielefeld-CeBiTec--protocol_LB.pdf">LB-medium</A> and 0,75  µl chloramphenicol</LI><LI>  inkubation overnight 37°C, 140rpm</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Plasmidisolation </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Nucleospin Plasmidisolation protocol</LI><UL><LI> got 67.3 ng/µl of 2B6</LI><LI> got 32.7 ng/µl of 2B7</LI><LI> got 51.6 ng/µl of A1</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> To get and purified the plasmid </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> Plasmidisolation of barnase-plasmid and tRNA-plasmids</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  protocol from Macherley-Nagel: Plasmid DNA purification, protocol 5: NucleoSpin</LI><LI>  concentration measurement by nanoprop</LI><LI>  sequenzingorder</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Redo digestion of the destination plasmid </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> analyze the destination plasmid digestion via gelelectrophoresis</LI><UL><LI> pSB1A3 digestion was not effective</LI></UL><LI> redo the digestion and analyze it on the gel</LI><UL><LI> new dgestion was very effective</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-08-07   -   2017-08-13 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Amplification of the backbone (pSB1C3) and insert (Tyr-aaRS) </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR Q5 Polymerase (no Master Mix)</LI><UL><LI> for 50 µL reaction</LI><UL><LI> 10   µL Reaction Buffer</LI><LI>  5   µL 2mM dNTPs</LI><LI>  1   µL Template</LI><LI>  0,5 µL High Fidelity DNA Polymerase</LI><LI> 10.0 µL High GC Enhancer</LI><LI> 18.5 µL ddest H2O</LI><LI>  5.0 µL Primer</LI><UL><LI> pSB1C3 (from RuBisCo, ncAA team): ht, jk  65°C</LI><LI> aaRS (from Heidelberg) : hq, jk 65°C (mistake! should be 58°C, redone identically with 58°C)</LI></UL><LI> digestion of the PCR products with Dpn1: 5 µL CutSmart Reaktionsbuffer and 1 µL Dpn1 per 50 µL PCR product, on 37°C over night)</LI></UL><LI> agarose-gelelectrophhoresis, 1% (bands of the right size)</LI></UL><LI> No further use, because results of the sequencing which show that the integration of the incorporation of the Tyr-aaRS in pSB1C3 was already successful (30.06.2017)</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Ligation of 2B2-2B5 and B1 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Ligation protocol from the <A target="blank" href="https://static.igem.org/mediawiki/2017/8/84/T--Bielefeld-CeBiTec--protocol_Standard_Biobrick.pdf">BioBrick Assembly</A></LI><UL><LI> Variation: incubated 1h at RT instead of 10min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> characterisation of T7GFP revers </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> preparation of KRX-competent cells</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> heatschocktransformation of T7GFP and T7GFP revers in KRX</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> construct a biobrick for the selectionplasmid </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Christina Drake</P><P><B>Superior experiment:</B> PCR of T7RNA-Polymerase</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> 1%-agarose-<A target="blank" href="https://static.igem.org/mediawiki/2017/d/d4/T--Bielefeld-CeBiTec--protocol_TAE_buffer.pdf">TAE</A>-gel</LI><LI>  result: PCR did work</LI><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A></LI><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/a0/T--Bielefeld-CeBiTec--protocol_heatshock.pdf">transformation via heat shock</A></LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Amplification of the backbone (pRS31) and insert (ori from pSB6A1, oK1.0) for Gibson assembly </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR Q5 Polymerase (protocoll of 07.08.2017)</LI><UL><LI> pRS31: primer jb, jn  65°C</LI><LI> oK1:   primer jl, jc  65°C</LI></UL><LI> digestion with Dpn1 (protocoll of 07.08.2017)</LI><LI> agarose-gelelectrophhoresis, 1%</LI><UL><LI> band around 1200 bp (oK1.1) / 2500 bp (pRS31), positive</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Gibson Assembly and Trafo of the pRS31 and oK1.0 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> agarose-gelelectrophhoresis, 1%</LI><LI> purification from the gel with the Macherey-Nagel purification kit</LI><UL><LI> oK1.1: 22.5 ng/µL</LI><LI> pRS32:  9.5 ng/µL</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> Master Mix with:</LI><UL><LI> 4.1 µL of oK1.1: around 2480 bp  9.5 ng/µL</LI><LI> 0.9 µL of pRS32: around 1250 bp 22.5 ng/µL</LI></UL><LI> Trafo via heatshock</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> heatshock transformation of assembled fragments F13 and NPA-RS gene synthesis</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard heatshock protocol</LI><UL><LI> used own produced chemo competend DH5alpha</LI><LI> streaked out on LB-plates with Cam</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication, verification and purification of the needed fragment F13: linearized ONBY-Part with removal of the ONBY-RS </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Q5-PCR, gel and PCR-cleanUp of F13</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard Q5-PCR protocol</LI><UL><LI>  template: K1416000</LI><LI>  primer fwd: 17hl</LI><LI>  primer rev: 17mv</LI><LI>  annealing temperature: 63°C</LI><LI>  extension time: 90 seconds</LI></UL><LI>  standard PCR-cleanUp protocol</LI><UL><LI>  80,6 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Assemble the fragments to get the Part BBa_K2201200 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Gibson assembly of F13 and NPA-RS gene synthesis</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard <A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> protocol</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> heatshock transformation of assembled fragments F5,F1,F3</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard heatshock protocol</LI><UL><LI> used own produced chemo competend DH5alpha</LI><LI> streaked out on LB-plates with Cam</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication, verification and purification of the needed fragment F1: GFPLinker1 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Q5-PCR, gel and PCR-cleanUp of F1</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard Q5-PCR protocol</LI><UL><LI>  template: BBa_E0400</LI><LI>  primer fwd: 17gk</LI><LI>  primer rev: 17gm</LI><LI>  annealing temperature: 58°C</LI><LI>  extension time: 50 seconds</LI></UL><LI>  standard PCR-cleanUp protocol</LI><UL><LI>  15,9 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication, verification and purification of the needed fragment F3: StrepLinker1 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Q5-PCR, gel and PCR-cleanUp of F3</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard Q5-PCR protocol</LI><UL><LI>  template: BBa_KJ36848</LI><LI>  primer fwd: 17go</LI><LI>  primer rev: 17gq</LI><LI>  annealing temperature: 61°C</LI><LI>  extension time: 30 seconds</LI></UL><LI>  standard PCR-cleanUp protocol</LI><UL><LI>  17,9 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Biobrickassembly </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/8/84/T--Bielefeld-CeBiTec--protocol_Standard_Biobrick.pdf">BioBrick Assembly</A></LI><LI> repeat digestion of T7-Terminator downstrampart</LI><LI> ligate over night</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Transformation of new plasmids </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Maximilian Edich</P><P><B>Superior experiment:</B> Biobrick Construction with RuBisCo Parts</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/a0/T--Bielefeld-CeBiTec--protocol_heatshock.pdf">transformation via heat shock</A></LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication, verification and purification of the needed fragment F15: linear GLS1 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Q5-PCR, gel and PCR-cleanUp of F15</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard Q5-PCR protocol</LI><UL><LI>  template: P2: GLS2</LI><LI>  primer fwd: 17go</LI><LI>  primer rev: 17gm</LI><LI>  annealing temperature: 60°C</LI><LI>  extension time: 190 seconds</LI></UL><LI>  standard PCR-cleanUp protocol</LI><UL><LI>  36,9 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3>Primer annealing preparation </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system - Primer annealing test</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Both Primers (17jh; 17ji) were resuspended in 1x Composite Buffer</LI><LI> Dilutions: 100 µM, 10 µM, 5 µM, 2.5 µM, 0.5 µM</LI><LI> OD 260 were measured to determine the real concentration</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-08-14   -   2017-08-20 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Screening for positive transformations containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> GoTaq PCR of 4 clones of BBa_K2201220</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard GoTaq-protocol</LI><UL><LI>  template: colonie pcr clones of GA_K2201220</LI><LI>  primer fwd: Präfix_fwd</LI><LI>  primer rev: Suffix_rev</LI><LI>  annealing temperature: 56°C</LI><LI>  extension time: 40 seconds</LI></UL><LI>  positive clone 3</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication of plasmids containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> preculture of the positive clone 3 of BBa_K2201220</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard procedure</LI><UL><LI>  1,25µl Cam in 5mL LB</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Primer annealing of 17jh &amp; 17ji </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system - Primer annealing test</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Protocol for annealing oligonucleotides (sigmaaldrich)</LI><UL><LI> Eqimolar Primers were mixed (50 µL)</LI><LI> Thermal profile (thermocycler)</LI><UL><LI> 95 °C, 2 min</LI><LI> Cool to 25 °C over 45 min</LI><LI> Cool to 4 °C for temporary storage</LI></UL></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Integration of barnase in pSB1C3 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Construction of selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A> of pSB1C3_I749606</LI><UL><LI> 5 µL 10x Universal buffer</LI><LI> 5.54 µL plasmid</LI><LI> 1µL <I>Xba</I>I</LI><LI> 1µL <I>Pst</I>I</LI><LI> filled up to final volume 50 µL with ddH20</LI><LI> Incubation: 30 min, 37 °C</LI><LI> Heatinactivation: 20 min, 80 °C</LI></UL><LI> Agarose gel electrophoresis (1%)</LI><UL><LI> 100 V, 30 min</LI></UL><LI> PCR and DNA purification kit (Macherey-Nagel)</LI><UL><LI> Nucleo spin PCR and <A target="blank" href="https://static.igem.org/mediawiki/2017/0/0e/T--Bielefeld-CeBiTec--protocol_Monarch_PCR_and_DNA_clean_up_kit_NEB.pdf">Monarch® PCR &amp; DNA Clean-up</A> protocol</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> protocol</LI><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/a0/T--Bielefeld-CeBiTec--protocol_heatshock.pdf">transformation via heat shock</A> protocol in <I>E. coli</I> DH5<I>alpha</I></LI><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/ab/T--Bielefeld-CeBiTec--protocol_electrotrafo.pdf">transformation via electroporation</A> protocol in <I>E. coli</I> DH5<I>alpha</I></LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> PCR of T7RNAP_BL </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Construction of the selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR</LI><UL><LI> Go Tag (Promega) protocol</LI><LI> Danaturation: 56 °C</LI><LI> Elongation: 180 s</LI></UL><LI> Agarose gel electrophoresis (1%)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting plasmids containing the Biobrick for further experiments </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> plasmid isolation of preculture of BBa_K2201220 clone 3</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard protocol</LI><UL><LI>  108,9 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Combination of two biobricks to get the composite part BBa_K2201320 for protein expression </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Biobrick Assembly of BBa_K608006 and BBa_K2201220</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/8/84/T--Bielefeld-CeBiTec--protocol_Standard_Biobrick.pdf">Standard BioBrick Assembly</A> protocol</LI><UL><LI>  Backbone: BBa_J04450, 7,7µL</LI><LI>  upstream: BBa_K608006, 3,6µL</LI><LI>  downstram: BBa_K2201220, 5,0µL</LI><LI>  Ligation time was 12 h at 8 °C</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Colony and plasmid PCR of T7RNAP_BL c1-4 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Construction of the selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR</LI><UL><LI> Go Tag (Promega) protocol</LI><LI> Primer: VR &amp; VF</LI><LI> Danaturation: 56 °C</LI><LI> Elongation: 180 s</LI></UL><LI> Agarose gel electrophoresis (1%)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> heatshock transformation of the Biobrick Assembly product BBa_K2201320</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard heatshock protocol</LI><UL><LI> used own produced chemo competend BL21(DE3)</LI><LI> streaked out on LB-plates with Cam</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Colony PCR of pSB1C3_barnase (gblock) c1-70 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Construction of selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR</LI><UL><LI> Go Tag (Promega) protocol</LI><LI> Primer: Vr &amp; VF2</LI><LI> Danaturation: 56.5 °C</LI><LI> Elongation: 40 s</LI></UL><LI> Agarose gel electrophoresis (1%)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Screening for positive transformations containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> GoTaq PCR of 19 clones of BBa_K2201320</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard GoTaq-protocol</LI><UL><LI>  template: colonie pcr clones of GA_K2201320</LI><LI>  primer fwd: T7-eva_fwd</LI><LI>  primer rev: Suffix_rev</LI><LI>  annealing temperature: 51°C</LI><LI>  extension time: 80 seconds</LI></UL><LI>  positive clone 19</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication of plasmids containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> preculture of the positive clone 19 of BBa_K2201320</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard procedure</LI><UL><LI>  1,25µl Cam in 5mL LB</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Combination of two biobricks to get the composite part BBa_K2201321 for protein expression </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Biobrick Assembly of BBa_K608006 and BBa_K2201221</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/8/84/T--Bielefeld-CeBiTec--protocol_Standard_Biobrick.pdf">Standard BioBrick Assembly</A> protocol</LI><UL><LI>  Backbone: BBa_J04450, 2,0µL</LI><LI>  upstream: BBa_K608006, 3,6µL</LI><LI>  downstram: BBa_K2201221, 2,4µL</LI><LI>  Ligation time was 12 h at 8°C</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Primer annealing of 17jl &amp; 17jm </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system - Primer annealing test</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Both Primers were resuspended in ddH20 to a final concentration of 100 µM</LI><LI> HEPES annealing standard protocol</LI><LI> Aqua annealing standard protocol</LI><UL><LI> Final volume: 20 µL</LI></UL><LI> Agarose gel electrophoresis (2%)</LI><UL><LI> 60 V, 50 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication of plasmids containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> preculture of BL21(DE3) BBa_K2201320</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard procedure</LI><UL><LI>  1,25µl Cam in 10mL LB</LI><LI>  12 h at 37°C</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> expression of the fusion protein </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> preculture of BL21(DE3) BBa_K2201320 with IPTG</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard procedure</LI><UL><LI>  1,25µl Cam in 10mL LB</LI><LI>  12 h at 37°C</LI><LI>  induced with IPTG 0.8mM</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting plasmids containing the Biobrick for further experiments </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> plasmid isolation of preculture of BBa_K2201320 clone 19</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard protocol</LI><UL><LI>  454,4 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication of plasmids containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> preculture of BL21(DE3) BBa_K2201320</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard procedure</LI><UL><LI>  1,25µl Cam in 10mL LB</LI><LI>  12 h at 37°C</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> expression of the fusion protein </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> preculture of BL21(DE3) BBa_K2201320 with IPTG</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard procedure</LI><UL><LI>  1,25µl Cam in 10mL LB</LI><LI>  12 h at 37°C</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Ligation of pSB1C3 and barnase (gblock) </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Construction of selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Ligation reaction</LI><UL><LI> 7 µL vector</LI><LI> 3 µL gblock</LI><LI> 2 µL T4 DNA ligase buffer</LI><LI> 1 µL T4 DNA ligase</LI><LI> 17 µL ddH20</LI></UL><LI> Incubation: RT, 1 h</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Colony PCR of pSB1C3_barnaseBL c1-60 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Construction of selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR</LI><UL><LI> Q5 High-fidelily (NEB) protocol</LI><LI> Denaturation: 56.5 °C</LI><LI> Elongation: 45 s</LI></UL><LI> Agarose gel electrophoresis (1%)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> heatshock transformation of the Biobrick Assembly product BBa_K2201321</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard heatshock protocol</LI><UL><LI> used own produced chemo competend BL21(DE3)</LI><LI> streaked out on LB-plates with Cam</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> - standard cultivation protocoll </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Cultivation of BL21(DE3) with BBa_K2201320</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  2x 100mL <A target="blank" href="https://static.igem.org/mediawiki/2017/a/a7/T--Bielefeld-CeBiTec--protocol_LB.pdf">LB-media</A> with 25µL Cam in 1000mL flasks</LI><UL><LI>  innoculation with 3.3 mL preculture of OD 3 to get a starting OD of 0.1</LI><LI>  incubation for four hours to a OD of 0.6</LI><LI>  addition of 500µL 0.5mM IPTG each</LI><LI>  incubation for 8 hours at 37°C and 24 hours at 18°C</LI><LI>  harveting of the cells</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Native PAGE of annealed primers (17jl &amp; 17jm) </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system - Primer annealing test</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/e/e6/T--Bielefeld-CeBiTec--protocol_native_PAGE.pdf">Native DNA PAGE</A> standard protocol</LI><UL><LI> tested samples:</LI><UL><LI> HEPES annealing (17jl &amp; 17jm): 100 µM, 10 µM, 5 µM, 2.5 µM, 1 µM, 0.5 µM</LI><LI> Aqua annealing (17jl &amp; 17jm): 100 µM, 10 µM, 5 µM, 2.5 µM, 1 µM, 0.5 µM</LI></UL><LI> Variation: ~40 mA</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-08-21   -   2017-08-27 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Native PAGE of annealed primers (17jl &amp; 17jm) </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system - Primer annealing test</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/e/e6/T--Bielefeld-CeBiTec--protocol_native_PAGE.pdf">Native DNA PAGE</A> standard protocol</LI><UL><LI> tested samples:</LI><UL><LI> HEPES annealing (17jl &amp; 17jm): 100 µM, 10 µM, 5 µM, 2.5 µM, 1 µM, 0.5 µM</LI><LI> Aqua annealing (17jl &amp; 17jm): 100 µM, 10 µM, 5 µM, 2.5 µM, 1 µM, 0.5 µM</LI></UL><LI> Variation: 100 V, 1 h</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Plasmidisolation of pSB1C3_barnaseB c5, 22, 30, 31, 32, 45 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Construction of selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Plasmid DNA purification kit (Zymo PURE)</LI><UL><LI> Nucleo spin plasmid protocol</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> PCR of KanR &amp; pSB1C3 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Construction of selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR</LI><UL><LI> Q5 High-fidelily (NEB) protocol</LI><LI> Primer: 17nx &amp; 17ny (Kan)</LI><LI> Primer: 17od &amp; 17oe (pSB1C3)</LI><LI> Denaturation: 54 °C &amp; 60 °C</LI><LI> Elongation: 30 s &amp; 75 s</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A></LI><UL><LI> 5 µL 10x CutSmart buffer</LI><LI> 1 µg plasmid</LI><LI> 1 µL <I>Dpn</I>I</LI><LI> filled up to final volume 50 µL with ddH20</LI><LI> Incubation: ON, 37 °C</LI><LI> Inactivation: 20 min, 80 °C</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication of plasmids containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> preculture of the positive clone 3 of BBa_K2201200</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard procedure</LI><UL><LI>  1,25µl Cam in 5mL LB</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Restriction digest of KanR fragment </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Olga Schmidt</P><P><B>Superior experiment:</B> Construction of selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A></LI><UL><LI> 5 µL NEB2.1 buffer</LI><LI> 2 µL DNA</LI><LI> 1 µL <I>Xba</I>I</LI><LI> 1 µL <I>Pst</I>I</LI><LI> filled up to final volume 50 µL with ddH20</LI><LI> Incubation: 60 min, 37 °C</LI><LI> Heatinactivation: 20 min, 80 °C</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Aim of the Experiment </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> GoTaq PCR of 32 clones of BBa_K2201200</P><P><B>Procedure:</B></P><ARTICLE><LI> Screening for positive transformations containing the Biobrick</LI><UL><LI>  standard GoTaq-protocol</LI><UL><LI>  template: colonie pcr clones of GA_K2201200</LI><LI>  primer fwd: 17ns</LI><LI>  primer rev: 17nt</LI><LI>  annealing temperature: 56°C</LI><LI>  extension time: 40 seconds</LI></UL><LI>  positive clone 20</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Native PAGE of annealed primers (17jl &amp; 17jm) </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system - Primer annealing test</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/e/e6/T--Bielefeld-CeBiTec--protocol_native_PAGE.pdf">Native DNA PAGE</A> standard protocol</LI><UL><LI> tested samples:</LI><UL><LI> HEPES annealing (17jl &amp; 17jm): 100 µM, 10 µM, 5 µM, 2.5 µM, 1 µM, 0.5 µM</LI><LI> Aqua annealing (17jl &amp; 17jm): 100 µM, 10 µM, 5 µM, 2.5 µM, 1 µM, 0.5 µM</LI></UL><LI> Variation: 60 V, 2 h</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> multiplication of plasmids containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> preculture of BL21(DE3) BBa_K2201321</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard procedure</LI><UL><LI>  1,25µl Cam in 5mL LB</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting plasmids containing the Biobrick for further experiments </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> plasmid isolation of preculture of BBa_K2201321 clone 3</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard protocol</LI><UL><LI>  319,1 ng/µL</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Native Page of annealed primers (17jl &amp; 17jm; 17jh &amp; 17ji) </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system - Primer annealing test</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/e/e6/T--Bielefeld-CeBiTec--protocol_native_PAGE.pdf">Native DNA PAGE</A> standard protocol</LI><UL><LI> tested samples:</LI><UL><LI> HEPES annealing (17jl &amp; 17jm): 0.5 µM</LI><LI> Aqua annealing (17jl &amp; 17jm): 0.5 µM</LI><LI> Composite Buffer annealing (17jh &amp; 17 jm): 0.5 µM</LI><LI> all Primers (ssDNA): 1 µM</LI></UL></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-08-28   -   2017-09-03 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Native PAGE of annealed primers (HEPES annealing &amp; Composite Buffer annealing) of the same concentration to test if all annealings have the same quality </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system - Primer annealing test</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> all samples were diluted to a final concentration of 0.5 µM</LI><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/e/e6/T--Bielefeld-CeBiTec--protocol_native_PAGE.pdf">Native DNA PAGE</A> standard protocol</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> heatshock transformation of the Biobrick Assembly product CP3</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard heatshock protocol</LI><UL><LI> used own produced chemo competend BL21(DE3)</LI><LI> streaked out on LB-plates with Cam</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Restriction digest of control annealings (mutC) </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A> (<I>Mnl</I>I) of aqua annealing sample: mutC (0.5 µM)</LI><UL><LI> 2 µL <I>Mnl</I>I</LI><LI> 5 µL CutSmart</LI><LI> 25 µL mutC dsDNA</LI><LI> 18 µL nuclease-free H2O</LI><LI> Incubation: 37 °C, 1 h</LI><LI> Inactivation: 20 min., 65 °C</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/e/e6/T--Bielefeld-CeBiTec--protocol_native_PAGE.pdf">Native DNA PAGE</A> standard protocol</LI><UL><LI> Tested samples:</LI><UL><LI> digested mutC sample</LI><LI> native mutC sample</LI><LI> ssDNA primers (17jl &amp; 17jm)</LI></UL></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> GoTaq PCR of 14 clones of BBa_K2201322 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Screening for positive transformations containing the Biobrick</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard GoTaq-protocol</LI><UL><LI>  template: colonie pcr clones of BBA_K2201322</LI><LI>  primer fwd: Präfix_fwd</LI><LI>  primer rev: Suffix_rev</LI><LI>  annealing temperature: 56°C</LI><LI>  extension time: 40 seconds</LI></UL><LI>  positive: clone 3, 4, 5</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Restriction digest of aqua &amp; HEPES annealing </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A> (<I>Mnl</I>I) of aqua &amp; HEPES annealing sample: mutC (0.5 µM)</LI><UL><LI> 2 µL <I>Mnl</I>I</LI><LI> 5 µL CutSmart</LI><LI> 25 µL mutC dsDNA</LI><LI> 18 µL nuclease-free H2O</LI><LI> Incubation: 37 °C, 1 h</LI><LI> Inactivation: 20 min., 65 °C</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/e/e6/T--Bielefeld-CeBiTec--protocol_native_PAGE.pdf">Native DNA PAGE</A> standard protocol</LI><UL><LI> Tested samples:</LI><UL><LI> digested mutC sample (aqua &amp; HEPES annealing)</LI><LI> native mutC sample</LI><LI> ssDNA primers (17jl &amp; 17jm)</LI></UL></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Aqua annealing of mutA, mutT, mutG and mutC </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Aqua annealing standard protocol (final concentration: 0.5 µM)</LI><UL><LI> mutA: 17jf &amp; 17jg</LI><LI> mutT: 17jj &amp; 17jk</LI><LI> mutG: 17jh &amp; 17ji</LI><LI> mutC: 17jl &amp; 17jm</LI></UL><LI> final volume of each sample: 50 µL</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-09-04   -   2017-09-10 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Restriction digest of control annealings (mutA, mutT, mutG, mutC) and aqua annealing of oligo2 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Oligo2 annealing</LI><UL><LI> Aqua annealing standard protocol (final concentration: 0.5 µM)</LI></UL><LI> Restricted samples:</LI><UL><LI><I>Mnl</I>I -&gt; mutC, oligo2</LI><LI><I>Sap</I>I -&gt; mutG, oligo2</LI><LI><I>Bsa</I>I -&gt; mutT, oligo2</LI><LI><I>Eci</I>I -&gt; mutA, oligo2</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A>:</LI><UL><LI> 1 µL Enzyme</LI><LI> 5 µL CutSmart</LI><LI> 25 µL mutC dsDNA</LI><LI> 19 µL nuclease-free H2O</LI><LI> Incubation: 37 °C, 1 h</LI><LI> Inactivation: 20 min., 65 °C</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/e/e6/T--Bielefeld-CeBiTec--protocol_native_PAGE.pdf">Native DNA PAGE</A> standard protocol</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> SDS-Page and western blot of BBa_K2201321 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> SDS-Page and western blot of BBa_K2201321</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard SDS protocoll</LI><LI>  standard Western blot protocoll</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> integration of the mRFP (BBa_J04450) in the aaRS plasmid as optical controle for the integration of an randomerized dsDNA </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> amplification of the mRFP (Phusion Master Mix), Primer vg, vh, annealing temperature: 60°C</LI><LI> amplification of the library backbone(Phusion Master Mix), Primer 17ve, 17vf, Annealing temperature: 61°C</LI><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> with Gibson Master Mix + mRFP (3.1 µL) + library backbone (1.9 µL)</LI><LI> transformation in DH5a via heatshock</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Generating the randomerized dsDNA for the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/d/d1/T--Bielefeld-CeBiTec--protocol_ssDNA_annealing.pdf">ssDNA Annealing</A> protocoll with the primers 17vi (1 µL), 17vj (1,4 µL)</LI><LI> agarose gelelectrophoresis, 3 %, positive result: bands of 120 bp</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Generating the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the optical controle</LI><LI> transformation in DH5a via heatshock</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-09-11   -   2017-09-17 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Annealing, restriction digest and native PAGE of all samples </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Aqua annealing standard protocol (final concentration: 0.5 µM)</LI><UL><LI> mutA: 17jf &amp; 17jg</LI><LI> mutT: 17jj &amp; 17jk</LI><LI> mutG: 17jh &amp; 17ji</LI><LI> mutC: 17jl &amp; 17jm</LI><LI> oligo2: olgio2_sense &amp; oligo2_antisense</LI></UL><LI> final volume of each sample: 50 µL</LI><LI> Restricted samples:</LI><UL><LI><I>Mnl</I>I -&gt; mutC, oligo2</LI><LI><I>Sap</I>I -&gt; mutG, oligo2</LI><LI><I>Bsa</I>I -&gt; mutT, oligo2</LI><LI><I>Eci</I>I -&gt; mutA, oligo2</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A>:</LI><UL><LI> 1 µL Enzyme</LI><LI> 5 µL CutSmart</LI><LI> 25 µL mutC dsDNA</LI><LI> 19 µL nuclease-free H2O</LI><LI> Incubation: 37 °C, 1 h</LI><LI> Inactivation: 20 min., 65 °C</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/e/e6/T--Bielefeld-CeBiTec--protocol_native_PAGE.pdf">Native DNA PAGE</A> standard protocol</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Generating the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the optical controle (8 x)</LI><LI> transformation in DH5a via heatshock (8 x)</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Generating the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (16 x)</LI><LI> transformation in DH5a via heatshock (16 x)</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Generating the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (25x)</LI><LI> transformation in DH5a via heatshock (25 x)</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Generating the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (25x)</LI><LI> transformation in DH5a via heatshock (25 x)</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Generating the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (25x)</LI><LI> transformation in DH5a via heatshock (25 x)</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Annealing, restriction digest and native PAGE of all samples </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Aqua annealing standard protocol (final concentration: 0.5 µM)</LI><UL><LI> mutA: 17jf &amp; 17jg</LI><LI> mutT: 17jj &amp; 17jk</LI><LI> mutG: 17jh &amp; 17ji</LI><LI> mutC: 17jl &amp; 17jm</LI><LI> oligo2: olgio2_sense &amp; oligo2_antisense</LI></UL><LI> final volume of each sample: 2x50 µL</LI><LI> Restricted samples:</LI><UL><LI><I>Mnl</I>I -&gt; mutC</LI><LI><I>Sap</I>I -&gt; mutG, oligo2</LI><LI><I>Bsa</I>I -&gt; mutT</LI><LI><I>Eci</I>I -&gt; mutA</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A>:</LI><UL><LI> 1 µL Enzyme</LI><LI> 5 µL CutSmart</LI><LI> 25 µL mutC dsDNA</LI><LI> 19 µL nuclease-free H2O</LI><LI> Incubation: 37 °C, 1 h</LI><LI> Inactivation: 20 min., 65 °C</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/e/e6/T--Bielefeld-CeBiTec--protocol_native_PAGE.pdf">Native DNA PAGE</A> standard protocol</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-09-18   -   2017-09-24 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> heatshock transformation of the EutS-part from CU Boulder</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard heatshock protocol</LI><UL><LI> used own produced chemo competend DH5alpha</LI><LI> streaked out on LB-plates with Amp</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> heatshock transformation of the FusionRedTag-part from CU Boulder</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard heatshock protocol</LI><UL><LI> used own produced chemo competend DH5alpha</LI><LI> streaked out on LB-plates with Kan</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> heatshock cotransformation of the EutS-part and FusionRedTag-part from CU Boulder</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard heatshock protocol</LI><UL><LI> used own produced chemo competend DH5alpha</LI><LI> streaked out on LB-plates with AmpKan</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Agarose gel electrophoresis of digested mutA, mutT, mutG &amp; mutC </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Agarose gel electrophoresis (2%)</LI><UL><LI> 60 V, 40 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Annealing, restriction digest and native PAGE of all samples </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Aqua annealing standard protocol (final concentration: 0.5 µM)</LI><UL><LI> mutA: 17jf &amp; 17jg</LI><LI> mutT: 17jj &amp; 17jk</LI><LI> mutG: 17jh &amp; 17ji</LI><LI> mutC: 17jl &amp; 17jm</LI><LI> oligo2: olgio2_sense &amp; oligo2_antisense</LI></UL><LI> final volume of each sample: 2x50 µL</LI><LI> Restricted samples:</LI><UL><LI><I>Mnl</I>I -&gt; mutC</LI><LI><I>Sap</I>I -&gt; mutG, oligo2</LI><LI><I>Bsa</I>I -&gt; mutT</LI><LI><I>Eci</I>I -&gt; mutA</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A>:</LI><UL><LI> 1 µL Enzyme</LI><LI> 5 µL CutSmart</LI><LI> 25 µL mutC dsDNA</LI><LI> 19 µL nuclease-free H2O</LI><LI> Incubation: 37 °C, 1 h</LI><LI> Inactivation: 20 min., 65 °C</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/e/e6/T--Bielefeld-CeBiTec--protocol_native_PAGE.pdf">Native DNA PAGE</A> standard protocol</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Combination of two biobricks to get the part BBa_K2201200 in a tetracycline low copy plasmid </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> Biobrick Assembly of BBa_K2201200 in psB3T5</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/8/84/T--Bielefeld-CeBiTec--protocol_Standard_Biobrick.pdf">Standard BioBrick Assembly</A> protocol</LI><UL><LI>  Backbone: pSB3T5, 2,0µL</LI><LI>  upstream: BBa_K2201200, 3,6µL</LI><LI>  Ligation time was 12 h at 8°C</LI><LI> cultivation</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> glycerine stocks of K2201200, K2201220, K2201221, K2201320, K2201321 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> glycerine stocks of K2201200, K2201220, K2201221, K2201320, K2201321</P><P><B>Procedure:</B></P><ARTICLE><LI> glycerine stock standard protocol</LI></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Screening for positive transformations containing the Biobrick </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> GoTaq PCR of 30 clones of K2201200 in pSB3T5</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard GoTaq-protocol</LI><UL><LI>  template: colonie pcr clones of K2201200 in pSB3T5</LI><LI>  primer fwd: Präfix_fwd</LI><LI>  primer rev: Suffix_rev</LI><LI>  annealing temperature: 56°C</LI><LI>  extension time: 40 seconds</LI></UL><LI>  positive clones 1, 3, 6</LI><LI> Preculture</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Colony PCR of pSB3K5_mRFP_link_ccdB and pSB1C3_PtNTT2(31-575)-GFP </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> Construction of pSB1C3-PlacUV5_PtNTT2</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR</LI><UL><LI> Go Tag (Promega) protocol</LI><LI> Primer: VR, VF2</LI><LI> Danaturation: 56 °C</LI><LI> Elongation: 90 s &amp; 180 s</LI></UL><LI> Agarose gel electrophoresis (1%)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-09-25   -   2017-10-01 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> heatshock cotransformation of K2201200 in pSB3T5 and K2201321</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard heatshock protocol</LI><UL><LI> used own produced chemo competend BL21(DE3)</LI><LI> streaked out on LB-plates with TetCam</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting plasmids containing the Biobrick for further experiments </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> plasmid isolation of preculture of BBa_K2201207</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  standard protocol</LI><UL><LI>  75,6 ng/µL</LI><LI> Biobrick Assembly</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Aqua annealing of mutA-1, mutG-1, mutC-1, mutG-1 &amp; oligo1 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> M.A.X restriction system</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Aqua annealing standard protocol (final concentration: 0.5 µM)</LI><UL><LI> mutA-1: 17iv &amp; 17iw</LI><LI> mutT-1: 17jb &amp; 17jc</LI><LI> mutG-1: 17ix &amp; 17iy</LI><LI> mutC-1: 17jd &amp; 17jc</LI><LI> oligo1: olgio1_sense &amp; oligo1_antisense</LI></UL><LI> final volume: 50 µL</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> heatshock cotransformation of K2201240 and K2201207</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard heatshock protocol</LI><UL><LI> used own produced chemo competend BL21(DE3)</LI><LI> streaked out on LB-plates with AmpCam</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> getting positive clones for further work </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Yannic Kerkhoff</P><P><B>Superior experiment:</B> heatshock cotransformation of K2201241 and K2201207</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> standard heatshock protocol</LI><UL><LI> used own produced chemo competend BL21(DE3)</LI><LI> streaked out on LB-plates with AmpCam</LI><LI> Q5-PRC + Gel + cleanUp</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-10-02   -   2017-10-08 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> UBP-PCR with TiTaq </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> PCR with UBP</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR</LI><UL><LI> PCR-UBP standard protocol (TiTaq)</LI><LI> samples:</LI><UL><LI> Ligation of: mutA, mutT, mutG, mutC and oligo 2 (1 ng/µL)</LI></UL><LI> Primer: 17vt &amp; 17vu</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 20 s</LI></UL><LI> PCR and DNA purification kit (Macherey-Nagel)</LI><UL><LI> Nucleo spin PCR and <A target="blank" href="https://static.igem.org/mediawiki/2017/0/0e/T--Bielefeld-CeBiTec--protocol_Monarch_PCR_and_DNA_clean_up_kit_NEB.pdf">Monarch® PCR &amp; DNA Clean-up</A> protocol</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A></LI><UL><LI> 1 µL Enzyme</LI><LI> 5 µL CutSmart</LI><LI> 25 µL mutC dsDNA</LI><LI> 19 µL nuclease-free H2O</LI><LI> Incubation: 37 °C, 1 h</LI><LI> Inactivation: 20 min., 65 °C</LI></UL><LI> Restricted samples:</LI><UL><LI><I>Mnl</I>I -&gt; mutC</LI><LI><I>Sap</I>I -&gt; mutG</LI><LI><I>Bsa</I>I -&gt; mutT</LI><LI><I>Eci</I>I -&gt; mutA</LI><LI><I>Eci</I>I, <I>Bsa</I>I, <I>Sap</I>I, <I>Mnl</I>I -&gt; oligo2</LI><LI> Variation: oligo2 was digested with 0.5 µL of each enzyme</LI></UL><LI> Agarose gel electrophoresis (2 %)</LI><UL><LI> 100 V, 20 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Generating the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (25x)</LI><LI> transformation in DH5a via heatshock (25 x)</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Kanymycin with amber stop codon </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> positiv selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> amplification (~850 bp) of the kanamycin with primers 17jo, 17jp (no second MET codon) 71°C</LI><LI> amplification (~850 bp) of the kanamycin with primers 17jr, 17jp (two amber stop codon) 71°C</LI></UL><LI> amplification of the K608007 backbone with the primers 17hb, 17hd  57°C</LI></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Generating the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (25x)</LI><LI> transformation in DH5a via heatshock (25 x)</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> UBP-PCR with GoTaq G2 and TiTaq </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> PCR with UBP</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR with TiTaq:</LI><UL><LI> PCR-UBP standard protocol (TiTaq)</LI><LI> samples:</LI><UL><LI> Ligation of: mutA, mutT, mutG, mutC and oligo 2 (5 ng/µL)</LI></UL><LI> Primer: 17 vt &amp; 17 vu</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 20 s</LI></UL><LI> PCR with TiTaq:</LI><UL><LI> PCR-UBP standard protocol (TiTaq)</LI><LI> samples:</LI><UL><LI> Ligation of: mutA, mutT, mutG and mutC (5 ng/µL)</LI></UL><LI> Primer: VR &amp; VF2</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 20 s</LI></UL><LI> PCR with GoTaq:</LI><UL><LI> PCR-UBP standard protocol (GoTaq)</LI><LI> samples:</LI><UL><LI> Ligation of: mutA, mutT, mutG, mutC and oligo2 (5 ng/µL)</LI></UL><LI> Primer: 17 vt &amp; 17 vu</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 20 s</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> cloning of the aaRS in pSB3C5 (low copy) </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS growth experiment</P><P><B>Procedure:</B></P><ARTICLE><UL><LI>  restriction of pSB3C5 and aaRS+tRNA (NPA, AcF, Prk) with EcoRI and PstI following the restriction protocol</LI><LI>  extraction from the gel (Macherey-Nagel Kit)</LI><LI>  ligation of pSB3C5 and aaRS+tRNA (NPA, AcF, Prk) with T4 Ligase (NEB)</LI><LI>  Trafo via heatshock</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> sceening of the kanamycin (03.10.2017) </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> positiv selection plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Colony PCR (GoTaq, Primer VF2, VR, 56.5 °C)</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Restriction digest of ligated oligo2 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> PCR with UBP</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A></LI><UL><LI> 5 µL CutSmart</LI><LI> 30 µL ligated oligo2 DNA</LI><LI> 2 µL <I>Eco</I>RV</LI><LI> filled up to final volume 50 µL with ddH20</LI><LI> Incubation: 120 min, 37 °C</LI><LI> Inactivation: 20 min, 65 °C</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Gradient PCR of mutT </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> PCR with UBP</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR</LI><LI> PCR-UBP standard protocol (GoTaq G2)</LI><UL><LI> sample: mutT (5 ng/µL)</LI><LI> Primer: 17 vt &amp; 17 vu</LI><LI> Primer annealing: 50 - 56 °C</LI><LI> Elongation: 20 s</LI></UL><LI> Agarose gel electrophoresis (2%)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> PCR of mutA, mutT, mutG &amp; mutC with UBP and oligo2 without UBP </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> PCR with UBP</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR with TiTaq:</LI><UL><LI> PCR-UBP standard protocol (TiTaq)</LI><LI> samples:</LI><UL><LI> Ligation of: mutA, mutT, mutG and mutC (5 ng/µL), oligo2 (25 ng/µL)</LI></UL><LI> Primer: VR &amp; VF2</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 20 s</LI><LI> Variation:</LI><UL><LI> oligo2 without UBP, instead 1 µL H20 was added</LI><LI> 0.5 µL isoG &amp; 0.5 µL isoCMe were added to mutA, mutT, mutG and mutC</LI></UL></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Generating the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS Plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> plasmid isolation of the Tyr-RS library-mix</LI><UL><LI> we stamped out the not randomized, negative, clones, easy to identify by the red colour because of the mRFP</LI><LI>  washed from the cultivation plates with <A target="blank" href="https://static.igem.org/mediawiki/2017/a/a7/T--Bielefeld-CeBiTec--protocol_LB.pdf">LB-media</A></LI></UL><LI> restriction of the Library with EcoRI and PstI</LI><LI> proof of the restriction via agarose-gelelectrophoresis, 1 %</LI><UL><LI> positve result: bands on ~2000 bp (backbone) and ~1000 bp (Tyr-RS with NNK)</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Generating the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (25x)</LI><LI> transformation in DH5a via heatshock (25 x)</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-10-09   -   2017-10-15 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> PCR with switched primers: VF2 &amp; 17vu, VR &amp; 17vt </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> PCR with UBP</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR with TiTaq:</LI><UL><LI> PCR-UBP standard protocol (TiTaq)</LI><LI> samples:</LI><UL><LI> Ligation of: mutA, mutT, mutG and mutC (5 ng/µL), oligo2 (25 ng/µL)</LI></UL><LI> Primer: VF2 &amp; 17vu, VR &amp; 17vt</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 20 s</LI></UL><LI> PCR with GoTaq:</LI><UL><LI> PCR-UBP standard protocol (GoTaq)</LI><LI> samples:</LI><UL><LI> Ligation of: mutA, mutT, mutG and mutC (5 ng/µL), oligo2 (25 ng/µL)</LI></UL><LI> Primer: VF2 &amp; 17vu, VR &amp; 17vt</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 20 s</LI></UL><LI> Agarose gel electrophoresis (2%)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Generating the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> aaRS plasmid</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/5/55/T--Bielefeld-CeBiTec--protocol_gibson_assembly.pdf">Gibson Assembly</A> Master Mix + dsDNA (3.5 µl) + library Plasmid (0.5 µL) with the opotical controle (25x)</LI><LI> transformation in DH5a via heatshock (25 x)</LI></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> PCR with higher DNA template concentrations and gradient PCR of mutT </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> PCR with UBP</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR with TiTaq:</LI><UL><LI> PCR-UBP standard protocol (TiTaq)</LI><LI> samples:</LI><UL><LI> Ligation of: mutT (5 ng/µL) and oligo2 (50 ng/µL)</LI></UL><LI> Primer: VF2 &amp; 17vu</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 20 s</LI></UL><LI> PCR and DNA purification kit (Macherey-Nagel)</LI><UL><LI> Nucleo spin PCR and <A target="blank" href="https://static.igem.org/mediawiki/2017/0/0e/T--Bielefeld-CeBiTec--protocol_Monarch_PCR_and_DNA_clean_up_kit_NEB.pdf">Monarch® PCR &amp; DNA Clean-up</A> protocol</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A></LI><UL><LI> 0.5 µL Enzyme</LI><LI> 2.3 µL CutSmart</LI><LI> 20 µL dsDNA</LI><LI> Incubation: 37 °C, 1 h</LI><LI> Inactivation: 20 min., 65 °C</LI></UL><LI> Restricted samples:</LI><UL><LI><I>Bsa</I>I -&gt; mutT</LI><LI><I>Eci</I>I, <I>Bsa</I>I, <I>Sap</I>I, <I>Mnl</I>I -&gt; oligo2</LI><LI> Variation: oligo2 was digested with 0.5 µL of each enzyme</LI></UL><LI> Agarose gel electrophoresis (2 %)</LI><UL><LI> 100 V, 20 min</LI></UL><LI> Gradient PCR with TiTaq:</LI><UL><LI> PCR-UBP standard protocol (TiTaq)</LI><LI> samples:</LI><UL><LI> Ligation of: mutT (5 ng/µL)</LI></UL><LI> Primer: VF2 &amp; 17vu</LI><LI> Primer annealing: 50 - 58 °C</LI><LI> Elongation: 20 s</LI></UL><LI> Agarose gel electrophoresis (2%)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> PCR of mutA, mutT, mutG, mutC and oligo2 </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> PCR with UBP</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR with GoTaq:</LI><UL><LI> PCR-UBP standard protocol (GoTaq)</LI><LI> samples:</LI><UL><LI> Ligation of: mutA, mutT, mutG, mutC (5 ng/µL) and oligo2 (25 ng/µL)</LI></UL><LI> Primer: VF2 &amp; 17vu</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 20 s</LI></UL><LI> PCR and DNA purification kit (Macherey-Nagel)</LI><UL><LI> Nucleo spin PCR and <A target="blank" href="https://static.igem.org/mediawiki/2017/0/0e/T--Bielefeld-CeBiTec--protocol_Monarch_PCR_and_DNA_clean_up_kit_NEB.pdf">Monarch® PCR &amp; DNA Clean-up</A> protocol</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A></LI><UL><LI> 1 µL Enzyme</LI><LI> 2.3 µL CutSmart</LI><LI> 20 µL dsDNA</LI><LI> Incubation: 37 °C, 12 h</LI><LI> Inactivation: 20 min., 65 °C</LI></UL><LI> Restricted samples:</LI><UL><LI><I>Mnl</I>I -&gt; mutC</LI><LI><I>Sap</I>I -&gt; mutG</LI><LI><I>Bsa</I>I -&gt; mutT</LI><LI><I>Eci</I>I -&gt; mutA</LI><LI><I>Eci</I>I, <I>Bsa</I>I, <I>Sap</I>I, <I>Mnl</I>I -&gt; oligo2</LI></UL><LI> Agarose gel electrophoresis (2 %)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> positive selection with the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> pos selection</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> using cultivation plates with kan (0.5 mM), Cm (0.25 mM), Tet (0.5 mM)</LI><UL><LI> no growth</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> Variation of restriction digest </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> PCR with UBP</P><P><B>Procedure:</B></P><ARTICLE><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A></LI><UL><LI> 3 µL Enzyme</LI><LI> 2.5 µL CutSmart</LI><LI> 20 µL dsDNA</LI><LI> Incubation: 37 °C, 1 h</LI><LI> Inactivation: 20 min., 65 °C</LI></UL><LI> Restricted samples:</LI><UL><LI><I>Mnl</I>I -&gt; mutC</LI><LI><I>Sap</I>I -&gt; mutG</LI><LI><I>Bsa</I>I -&gt; mutT</LI><LI><I>Eci</I>I -&gt; mutA</LI><LI><I>Eci</I>I, <I>Bsa</I>I, <I>Sap</I>I, <I>Mnl</I>I -&gt; oligo2</LI></UL><LI> Agarose gel electrophoresis (2 %)</LI><UL><LI> 100 V, 30 min</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> positive selection with the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> pos selection</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Transformation (heatshock) of the library plasmid mix in competent cells containing the positive selction plasmid</LI><LI> using cultivation plates with kan (0.3 mM), Cm (0.25 mM), Tet (0.5 mM)</LI><UL><LI> no growth</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> positive selection with the library </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> pos selection</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Transformation (heatshock) of the library plasmid mix in competent cells containing the positive selction plasmid</LI><LI> extended regeneration time: adding 0,5 mM IPTG after 1 h in <A target="blank" href="https://static.igem.org/mediawiki/2017/f/fe/T--Bielefeld-CeBiTec--protocol_SOC.pdf">SOC media</A></LI><LI> using cultivation plates with kan (0.3 mM), Cm (0.25 mM), Tet (0.5 mM), IPTG (0,5 mM)</LI><UL><LI> no growth</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> growth experiments with the aaRS </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Laura Schlueter</P><P><B>Superior experiment:</B> pos selection</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> comparing AcF-TAG, AcF-Leu, Cou-AS, Prk, NPA (on low and high copy plasmid) pSB1C3, pSB3T5</LI><UL><LI> cultivation:</LI><UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/a/a7/T--Bielefeld-CeBiTec--protocol_LB.pdf">LB media</A>, Cm (0.25 mM) / Tet (0.5 mM) and 1 mM of the ncAA</LI><LI> 12 microtiter well plate, 600 rpm, 37°C</LI></UL></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-date"><H3> 2017-10-16   -   2017-10-22 </H3></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> PCR with TiTaq and Q5-polymerase </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> PCR with UBP</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> PCR with TiTaq:</LI><UL><LI> PCR-UBP standard protocol (TiTaq)</LI><LI> samples:</LI><UL><LI> Ligation of: mutA, mutT, mutG &amp; mutC (5 ng/µL) and oligo2 (50 ng/µL)</LI></UL><LI> Primer: VF2 &amp; 17vu</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 20 s</LI></UL><LI> PCR and DNA purification kit (Macherey-Nagel)</LI><UL><LI> Nucleo spin PCR and <A target="blank" href="https://static.igem.org/mediawiki/2017/0/0e/T--Bielefeld-CeBiTec--protocol_Monarch_PCR_and_DNA_clean_up_kit_NEB.pdf">Monarch® PCR &amp; DNA Clean-up</A> protocol</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/8/88/T--Bielefeld-CeBiTec--protocol_PCR_Q5_HF_Polymerase.pdf">PCR with Q5 High-Fidelity Polymerase</A>-polymerase (per reaction, 25 µL):</LI><UL><LI> 5 µL Reaction buffer</LI><LI> 1.25 µL dNTPs (2 mM)</LI><LI> 1.25 µL 3'-Primer (10 mM)</LI><LI> 1.25 µL 5'-Primer (10 mM)</LI><LI> 1 µL DNA template</LI><LI> 0.25 µL Q5 Hifi-Polymerase</LI><LI> 14 µL H20</LI><LI> mutA, mutT, mutG, mutC: + 1 µL H20</LI><LI> oligo2: + 0.5 µL isoCme, + 0.5 µL isoG</LI><LI> Primer: VF2 &amp; 17vu</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 10 s</LI></UL><LI> PCR and DNA purification kit (Macherey-Nagel)</LI><UL><LI> Nucleo spin PCR and <A target="blank" href="https://static.igem.org/mediawiki/2017/0/0e/T--Bielefeld-CeBiTec--protocol_Monarch_PCR_and_DNA_clean_up_kit_NEB.pdf">Monarch® PCR &amp; DNA Clean-up</A> protocol</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A></LI><UL><LI> 1 µL Enzyme</LI><LI> 20 µL dsDNA</LI><LI> 2.3 µL CutSmart Buffer</LI><LI> Incubation: 37 °C, 15 h (<I>Bsa</I>I &amp;<I>Mnl</I>I), 2 h (<I>Sap</I>I &amp;<I>Eci</I>I)</LI><LI> Inactivation: 65 °C, 20 min</LI></UL><LI> Restricted samples:</LI><UL><LI><I>Mnl</I>I -&gt; mutC</LI><LI><I>Sap</I>I -&gt; mutG</LI><LI><I>Bsa</I>I -&gt; mutT</LI><LI><I>Eci</I>I -&gt; mutA</LI><LI><I>Eci</I>I, <I>Bsa</I>I, <I>Sap</I>I, <I>Mnl</I>I -&gt; oligo2</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> PCR with different polymerases </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> PCR with UBP</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> samples:</LI><UL><LI> Ligation of: mutA, mutT, mutG &amp; mutC (5 ng/µL) and oligo2 (50 ng/µL)</LI></UL><LI> PCR with Allin Hifi DNA Polymerase (per reaction, 25 µL):</LI><UL><LI> 5 µL ReactionBuffer</LI><LI> 0.25 µL Hifi DNA Polymerase</LI><LI> 2.5 µL 3'-Primer (10 mM)</LI><LI> 2.5 µL 5'-Primer (10 mM)</LI><LI> 1 µL DNA template</LI><LI> 12.75 µL H20</LI><LI> mutA, mutT, mutG, mutC: + 1 µL H20</LI><LI> oligo2: + 0.5 µL isoCme, + 0.5 µL isoG</LI><LI> Primer: VF2 &amp; 17vu</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 10 s</LI></UL><LI> PCR and DNA purification kit (Macherey-Nagel)</LI><UL><LI> Nucleo spin PCR and <A target="blank" href="https://static.igem.org/mediawiki/2017/0/0e/T--Bielefeld-CeBiTec--protocol_Monarch_PCR_and_DNA_clean_up_kit_NEB.pdf">Monarch® PCR &amp; DNA Clean-up</A> protocol</LI></UL><LI> PCR with innuDRY Standard PCR MasterMix (per reaction, 20 µL):</LI><UL><LI> 10 µL innduDRY Mastermix</LI><LI> 2 µL 3'-Primer (10 mM)</LI><LI> 2 µL 5'-Primer (10 mM)</LI><LI> 1 µL DNA template</LI><LI> 5 µL H20</LI><LI> mutA, mutT, mutG, mutC: + 1 µL H20</LI><LI> oligo2: + 0.5 µL isoCme, + 0.5 µL isoG</LI><LI> Primer: VF2 &amp; 17vu</LI><LI> Primer annealing: 54 °C</LI><LI> Elongation: 30 s</LI></UL><LI> PCR and DNA purification kit (Macherey-Nagel)</LI><UL><LI> Nucleo spin PCR and <A target="blank" href="https://static.igem.org/mediawiki/2017/0/0e/T--Bielefeld-CeBiTec--protocol_Monarch_PCR_and_DNA_clean_up_kit_NEB.pdf">Monarch® PCR &amp; DNA Clean-up</A> protocol</LI></UL><LI> PCR with BioMaster-HS Taq PCR Color (per reaction, 25 µL):</LI><UL><LI> 12.5 µL Mastermix</LI><LI> 0.7 µL 3'-Primer (10 mM)</LI><LI> 0.7 µL 5'-Primer (10 mM)</LI><LI> 1 µL DNA template</LI><LI> 10.1 µL H20</LI><LI> mutA, mutT, mutG, mutC: + 1 µL H20</LI><LI> oligo2: + 0.5 µL isoCme, + 0.5 µL isoG</LI><LI> Primer: VF2 &amp; 17vu</LI><LI> Primer annealing: 51 °C</LI><LI> Elongation: 20 s</LI></UL><LI> PCR and DNA purification kit (Macherey-Nagel)</LI><UL><LI> Nucleo spin PCR and <A target="blank" href="https://static.igem.org/mediawiki/2017/0/0e/T--Bielefeld-CeBiTec--protocol_Monarch_PCR_and_DNA_clean_up_kit_NEB.pdf">Monarch® PCR &amp; DNA Clean-up</A> protocol</LI></UL><LI> PCR with Fire Polymerase (per reaction, 20 µL):</LI><UL><LI> 4 µL Mastermix</LI><LI> 0.5 µL 3'-Primer (10 mM)</LI><LI> 0.5 µL 5'-Primer (10 mM)</LI><LI> 1 µL DNA template</LI><LI> 14 µL H20</LI><LI> mutA, mutT, mutG, mutC: + 1 µL H20</LI><LI> oligo2: + 0.5 µL isoCme, + 0.5 µL isoG</LI><LI> Primer: VF2 &amp; 17vu</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 20 s</LI></UL><LI> PCR and DNA purification kit (Macherey-Nagel)</LI><UL><LI> Nucleo spin PCR and <A target="blank" href="https://static.igem.org/mediawiki/2017/0/0e/T--Bielefeld-CeBiTec--protocol_Monarch_PCR_and_DNA_clean_up_kit_NEB.pdf">Monarch® PCR &amp; DNA Clean-up</A> protocol</LI></UL><LI> PCR with Phusion polymerase (per reaction, 25 µL):</LI><UL><LI> 12.5 µL Mastermix</LI><LI> 1.25 µL 3'-Primer (10 mM)</LI><LI> 1.25 µL 5'-Primer (10 mM)</LI><LI> 1 µL DNA template</LI><LI> 9 µL H20</LI><LI> mutA, mutT, mutG, mutC: + 1 µL H20</LI><LI> oligo2: + 0.5 µL isoCme, + 0.5 µL isoG</LI><LI> Primer: VF2 &amp; 17vu</LI><LI> Primer annealing: 56 °C</LI><LI> Elongation: 20 s</LI></UL><LI> PCR and DNA purification kit (Macherey-Nagel)</LI><UL><LI> Nucleo spin PCR and <A target="blank" href="https://static.igem.org/mediawiki/2017/0/0e/T--Bielefeld-CeBiTec--protocol_Monarch_PCR_and_DNA_clean_up_kit_NEB.pdf">Monarch® PCR &amp; DNA Clean-up</A> protocol</LI></UL><LI><A target="blank" href="https://static.igem.org/mediawiki/2017/2/23/T--Bielefeld-CeBiTec--protocol_Restriction_Digest.pdf">Restriction Digest</A></LI><UL><LI> 1 µL Enzyme</LI><LI> 20 µL dsDNA</LI><LI> 2.3 µL CutSmart Buffer</LI><LI> Incubation: 37 °C, 15 h (<I>Bsa</I>I &amp;<I>Mnl</I>I), 2 h (<I>Sap</I>I &amp;<I>Eci</I>I)</LI><LI> Inactivation: 65 °C, 20 min</LI></UL><LI> Restricted samples:</LI><UL><LI><I>Mnl</I>I -&gt; mutC</LI><LI><I>Sap</I>I -&gt; mutG</LI><LI><I>Bsa</I>I -&gt; mutT</LI><LI><I>Eci</I>I -&gt; mutA</LI><LI><I>Eci</I>I, <I>Bsa</I>I, <I>Sap</I>I, <I>Mnl</I>I -&gt; oligo2</LI></UL></UL></ARTICLE></DIV></DIV><DIV class="labnotebox"><DIV class="labnote-title"><H3> PCR with different polymerases </H3></DIV><DIV class="labnote-content"><P><B>Investigators:</B> Camilla Maerz</P><P><B>Superior experiment:</B> PCR with UBP</P><P><B>Procedure:</B></P><ARTICLE><UL><LI> Agarose gel electrophoresis with all samples (17.10)</LI><LI> Agarose gel electrophoresis (2%)</LI><UL><LI> 100 V, 40 min</LI></UL></UL></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>