Team:Bielefeld-CeBiTec/Demonstrate
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Demonstrate your Work
We worked in many different scientific fields to find suitable ways for the translational incorporate of non-canonical amino acids into proteins. Repurposing existing codons or incorporating new bases are two possible ways. We realized both ways to expand the genetic code of Escherichia coli.
The repurposing of a codon for the incorporation of a non-canonical amino acid (ncAA) is possible using the rarely used amber stop codon UAG or other rarely used codons like the leucine codon CUA. To incorporate a non-canonical amino acid using these codons, an orthogonal tRNA/aminoacyl-tRNA synthetase ( tRNA/aaRS ) pair is necessary, which can charge the ncAA to the tRNA. We designed and synthetized the novel ncAA Nγ‑2‑cyanobenzothiazol‑6‑yl‑L‑asparagine ( CBT-asparagine ). This ncAA has the chemical ability of perform a highly specific covalent binding reaction, which we wanted to incorporate into our target protein. Therefore, we created a library of aaRS with random mutagenized amino acid binding sites and a selection system to select for the aaRS that specifically incorporates the ncAA. In parallel to the libary and selection based approach, we modeled the aaRS which could incorporate our new amino acid CBT-asparagine. We demonstrated that both ways are suitable for the evolution of aaRS.
Although incorporation of ncAAs through the amber codon works, there are challenges associated with this approach. The repurposing of codons leads to the decrease of the growth rate of E. coli and it is only feasible to incorporate up to two different ncAAs. Therefore, we took a new way to incorporate ncAAs. The incorporation of an unnatural base pair into the DNA generates 64 new codons. Our first challenge was the uptake of the unnatural base from the media, because E.coli has no nucleoside triphosphate transporter and is not able to synthetize the bases itself. We cloned a nucleoside triphosphate transporter that enables the uptake of both bases from the media. Furthermore, we analyzed the transcriptome  of the plant Croton tiglium, which produces the unnatural base isoG. The transcriptome revealed an enzyme for the biosynthesis, which was  cloned and characterized for the biosynthesis of isoG in E. coli. To detect the unnatural base we developed two orthogonal systems. A restriction experiment based on the software tool M.A.X . and an adaption of the Oxford Nanopore sequencing , which were combined into one software suite .
To demonstrate the possibilities offered by the incorporation of ncAAs, we developed a toolbox containing five different tools. We chose seven different ncAAs for these five tools and demonstrated interesting applications for them. These ncAAs can be used for various approaches in basic research, medicine and manufacturing. Furthermore, with our submitted parts , every iGEM team can incorporate these ncAAs into their target proteins.
Regarding our project, two of the ncAAs that are part of our toolbox perform an autocatalytic reaction upon irradiation with ultraviolet light. Therefore, we decided to build our own LED panel that allows us to perform experiments with these non‑canonical amino acids under reproducible irradiation conditions.
Achievements
We established two orthogonal methods for the detection of unnatural base pairs in a target sequence: an Oxford Nanopore sequencing application and an enzyme based detection method
Development of a software suite for these orthogonal methods
Integration and characterization of  the nucleotide transporter PtNTT2 from P.tricornutum in E.coli for the uptake of  unnatural nucleoside triphosphates
Confirmation that certain Taq DNA polymerases can efficiently incorporate unnatural nucleotides
Construction of a toolbox consisting of five aminoacyl-tRNA synthetases for incorporation of non-canonical amino acids
Colocalization of the RuBisCo and and subcellular compartment (carboxysome) using a fluorescent amino acid
Development of a photoswitchable lycopene pathway
Design, chemical synthesis and proof of functionality of a novel, fully synthetic amino acid based on cyanonitrobenzothiazol and asparagine
Modeling more than ten new aaRS sequences
Library development with several hundred thousand sequences for selecting aminoacyl-tRNA synthetases
Construction of positive and negative selection plasmids for the evolution of new synthetases for non-canonical amino acids
Improvement of an aminoacyl-tRNA synthetase test-system by introducing a FRET-system and development of a ranking system
Construction of an LED panel for irradiating 96-well microtiter plates, which can be used to manipulate non-canonical amino acids and for other applications
Development of an Android App to control the LED panel with your smartphone via Bluetooth
Writing a biosafety report entitled “Auxotrophy to Xeno-DNA: A Comprehensive Exploration of Combinatorial Mechanisms for a High-Fidelity Biosafety System”
Writing the ChImp Report on “Chances and Implications of an Expanded Genetic Code”
