<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_unnatural_base_pair_development_of_new_methods skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/unnatural base pair/development of new methods</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/4/4a/T--Bielefeld-CeBiTec--title-img-nanopore.jpg);"><DIV id="title-bg"><DIV id="title-text">
                        Development of New Methods
                    </DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Short summary</H3><DIV class="article">
                    For our experiments, we developed two new methods which are potentially applicable for the analysis of experiments with most unnatural bases: The restriction enzyme based Mutational Analysis Xpolorer (M.A.X) and Oxford Nanopore sequencing of unnatural bases with the help of our software iCG. Both are comparably cost efficient methods that have the major advantage of enabling the direct analysis of mutational events in addition to the mere detection of unnatural bases. In our experiments, we showed that both methods are applicable for the analysis of experiments with the unnatural base pait between isoG and isoC<SUP>m</SUP>. Additionally, we analyzed the replication efficiency of this unnatural base pair with the help of both methods, showing that they produce similar results by orthogonal approaches.
                </DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Background: Detection of Unnatural Bases in DNA</H3><DIV class="article">
                    When working with unnatural bases, one of the major challenges is the detection of unnatural base pairs (UBPs) in the DNA. For the analysis of UBP retention <I>in vivo</I>, <I>in vitro</I> replication and PCR experiments, it is mandatory to have a reliable method for UBP detection. In most cases, scientists working with unnatural bases have to develop their own methods specifically suitable for the detection of the very unnatural bases they are working with. Unfortunately, these methods often come along with unfavorable circumstances.
                </DIV><DIV class="article">
                    One method commonly applied is the usage of molecular beacons, as described by Johnson <I>et al.</I>, 2004 for the detection if isoG and isoC<SUP>m</SUP> in PCR experiments. This is a quite circumstantial and expensive method, as for every ssDNA sample, an individual fluorescence labeled, specific oligonucleotide containing the unnatural nucleotide complementary to the one which should be investigated is needed. Additionally, the influence of the analyzed unnatural bases on the annealing temperature has to be investigated previously, to prevent unspecific hybridization from influencing the analysis results. A different method used primarily for UBP retention analysis is the biotin shift assay (Zhang <I>et al.</I>, 2017). In this approach, a DNA sample is used as a template in a PCR reaction with biotin-labeled nucleotide triphosphates of one of the unnatural bases. Through specific interaction with streptavidine after amplification, molecules containing these labeled nucleotides are shifted upwards in comparison to DNA molecules of the same length in subsequent PAGE analysis. The use of specially labeled variants of the analyzed unnatural bases is generally not recommended, as even small structural changes might have a great impact on the interaction with other biomolecules and could lead to falsified experiment results. In addition, the necessity of a PCR amplification for UBP detection is disadvantageous too, as the proportion of DNA molecules with sustained and lost UBPs might be greatly impacted, particularly - but not exclusively - due to influences of the unnatural nucleoside triphosphates on the PCR reaction.
                </DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Analysis of DNA containing isoG and isoC<SUP>m</SUP> with the Mutation Analysis Xplorer</H3><H4>Primer annealing</H4><ARTICLE>
                    To test the best annealing efficiency, we applied three different annealing methods.
            <UL><LI><A href=" https://static.igem.org/mediawiki/2017/d/d1/T--Bielefeld-CeBiTec--protocol_ssDNA_annealing.pdf "> HEPES annealing </A></LI><LI><A href=" https://static.igem.org/mediawiki/2017/d/d1/T--Bielefeld-CeBiTec--protocol_ssDNA_annealing.pdf "> 1x Composite Buffer annealing </A></LI><LI><A href=" https://static.igem.org/mediawiki/2017/d/d1/T--Bielefeld-CeBiTec--protocol_ssDNA_annealing.pdf "> Aqua annealing </A></LI></UL>

            We designed five ssDNA pairs with different bases at position 40. Each of the natural bases A, T, G, and C will lead to a recognition sequence of one of the restriction enzymes <I>Eci</I>I, <I>Bsa</I>I, <I>Sap</I>I and <I>Mnl</I>I. We postulate that non of these restriction enzymes will recognize their respective recognition sequences if the basepair between isoG and isoC<SUP>m</SUP> is present at this position. 
            For good annealing efficiency, it is necessary to add the two oligo strands together in equal molar amounts. The concentration can be calculated by the OD<SUB>260</SUB> value, while an  OD<SUB>260</SUB> of 1 equals 33 µg ml<SUP>-1</SUP><A href=" https://nebiocalculator.neb.com/#!/od260"> (NEB calculator, September 2017) </A> and the molecular mass of each oligo.  
                
        <DIV class="equation-line"><P class="equation-nr">(1)</P></DIV><DIV class="figure large"><P class="figure subtitle"><B>Figure 1:</B>Sequenzes of M.A.X targets mutA, mutT, mutG ad mutC with relevant restriction sites as well as the sequence of UBP_target.</P></DIV>All reactions showed a nearly complete alignment. We prosecuted further experiments with the aqua annealing in order to avoid affecting subsequent digestion reactions by influencing the buffer conditions.Subsequently, we tested different amounts of annealed DNA, varying from 50 µmol  L<SUP>-1</SUP> to 0.25 µmol L<SUP>-1</SUP>, which still shows visible bands on the gel. 
After first results, a final annealing concentration of 0.5 µmol L<SUP>-1</SUP> seems to be a good choice in terms of visibility and low DNA quantity for complete digestion. For the following annealing reactions, 1 µmol L<SUP>-1</SUP> of ssDNA was used to get 0.5 µL  L<SUP>-1</SUP> of annealed dsDNA. 
</ARTICLE><H4>Restriction digest</H4><ARTICLE>The DNA strands were designed such that the partial restriction sites of four different restriction enzymes are located at the same position. In case of a mutation, we can validate to which base the unnatural base mutated without sequencing it. To test the practicability and quality of the restriction system, we performed several test restriction digests. To ensure the digestion is complete, we calculated the amount of DNA which is digested per unit of enzyme in 1 hour at 37 °C. 1 unit is defined as the amount of restriction enzyme needed to digest 1 µg of lambda DNA. The lambda DNA consists of 48,502 bp <A href=" https://www.neb.com/products/n3011-lambda-dna#Product%20Information"> (NEB) </A>, which equals 1.99 ∙ 10<SUP>10</SUP> molecules per µL. Depending on the sum of recognition sites of each enzyme, we calculated the cuts per hour of each enzyme.
            <P class="table-headline"><B>Table 1:</B> Calculation of restrictions per hour (1 unit) of the M.A.X enzymes.</P><TABLE><THEAD><TR><TH style="width: auto">Enzyme</TH><TH style="width: auto">unit per µL</TH><TH style="width: auto">restriction sites in lambda DNA</TH><TH style="width: auto">restriction per hour (1 unit)</TH></TR></THEAD><TBODY><TR><TD align="center">mutG</TD><TD align="center"><I>Sap</I>I</TD><TD align="center">10</TD><TD align="center">10</TD><TD align="center">1.99∙10<SUP>12</SUP></TD></TR><TR><TD align="center">mutC</TD><TD align="center"><I>Mnl</I>I</TD><TD align="center">5</TD><TD align="center">262</TD><TD align="center">2.35∙10<SUP>13</SUP></TD></TR><TR><TD align="center">mutT</TD><TD align="center"><I>Bsa</I>I</TD><TD align="center">10</TD><TD align="center">2</TD><TD align="center">3.6∙10<SUP>11</SUP></TD></TR><TR><TD align="center">mutA</TD><TD align="center"><I>Eci</I>I</TD><TD align="center">2</TD><TD align="center">29</TD><TD align="center">1.44∙10<SUP>12</SUP></TD></TR></TBODY></TABLE>In an annealing reaction with 0.5 µmol L<SUP>-1</SUP> DNA in a total reaction volume of 50 µL, we have 3.011 ∙ 10<SUP>8</SUP> molecules per µL, each contain one or two restriction site. Theoretically, more than 1 mL of the annealing DNA should be digested by 1 unit of restriction enzyme per hour. After 1 hour of incubation and following heat inactivation, all samples were resolves by electrophoresis using a 12 % bisacrlyamid <A target="_blank" href="https://static.igem.org/mediawiki/2017/e/e6/T--Bielefeld-CeBiTec--protocol_native_PAGE.pdf">Native DNA PAGE</A>.
        <DIV class="contentline"><DIV class="half left"><DIV class="figure medium"><P class="figure subtitle"><B>Figure 2:</B> Native DNA PAGE of annealed mutA oligos. Samples: UBP_target ssDNA, UBP_target annealing, restricted UBP_target (<I>Eci</I>I), restricted mutA (<I>Eci</I>I), ssDNA oligo, annealed mutA.</P></DIV></DIV><DIV class="half right"><DIV class="figure medium"><P class="figure subtitle"><B>Figure 3:</B> Native DNA PAGE of annealed oligos. Samples: annealed mutT, ssDNA oligo, restricted mutT (<I>Bsa</I>I), restricted UBP_target (<I>Sap</I>I), UBP_target annealing, UBP_target ssDNA.</P></DIV></DIV></DIV><DIV class="contentline"><DIV class="half left"><DIV class="figure medium"><P class="figure subtitle"><B>Figure 4:</B> Native DNA PAGE of annealed mutG oligos. Samples: UBP_target ssDNA, UBP_target annealing, restricted UBP_target (<I>Sap</I>I) restricted mutG (<I>Sap</I>I), ssDNA oligo, annealed mutG.</P></DIV></DIV><DIV class="half right"><DIV class="figure medium"><P class="figure subtitle"><B>Figure 5:</B> Native DNA PAGE of annealed mutC oligos. Samples: UBP_target ssDNA, UBP_target annealing, restricted UBP_target (<I>Mnl</I>I) restricted mutC (<I>Mnl</I>I), ssDNA oligo, annealed mutC.</P></DIV></DIV></DIV>Figures 2 - 5 show the expected band pattern, although the digest of the M.A.X targets is not complete. Later experiments revealed that a longer incubation time is necessary.
        The M.A.X targets are digested after 40 bp, while the UBP_target which contains the unnatuarl bases is still of full length because the recognition site of the restriction enzyme is interrupted by the UBP.
        In Figure 5, the UBP_target shows a restricted band pattern because of the second <I>Mnl</I>I restriction site just 4 bases next to the first one. It is hard to see, but the UBP_target fragment is a little bit longer
        than the restriction fragment of mutC.The UBP_target annealings are not unexpected digested, indicating that the UBP prevents sequence recognition of the tested restriction enzymes. This proofs that the M.A.X restriction system is a 
        good detection system for UBP retention and mutation event analysis in selected DNA sequences.  
        </ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> PCR with UBPs </H3><ARTICLE>Based on Sismour <I>et al</I>. (2005) and Johnson <I>et al.</I> (2004), we designed a novel protocol for PCR with the unnatural base pair isoG and isoC<SUP>m</SUP>. We first started to reproduce positive results with <A target="_blank" href="https://static.igem.org/mediawiki/2017/1/14/T--Bielefeld-CeBiTec--CMZ-PCRwithUBP.pdf">Titanium Taq (TiTaq) polymerase</A>. While Johnson <I>et al.</I> presented an efficiency of 96 % ± 3 %, Sismour <I>et al.</I> showed a reduced fidelity using the Klenow fragment of TiTaq polymerase. Without thymidine analogues, the fidelity per round PCR decreases rapidly to less than 60 % after 20 rounds of PCR. For endpoint determination, we performed PCR reactions with 30 rounds to find out if there is any polymerase activity with template DNA containing the unnatural bases isoG and isoC<SUP>m</SUP>. 
         The PCR templates were prepared by ligating each of the annealed 80 bp M.A.X targets mutA, mutT, mutG and mutC into pSB1C3_RuBisCo <A href=" http://parts.igem.org/Part:BBa_K1465202"> (BBa_K1465202) </A>. For this purpose, the plasmid backbone was linearized by digestion with <I>Bmt</I>I and <I>Xba</I>I. For complementary sticky ends, the annealed oligos were digested with <I>Bmt</I>I and <I>Spe</I>I. After ligation, subsequent digestion with <I>Xba</I>I, lambda exonuclease and exonuclease I  was performed to reduce the amount of unintended DNA template. 
        <DIV class="figure medium"><P class="figure subtitle"><B>Figure 6:  Plasmidcard of pSB1C3_RuBisCo which is used as backbone <A target="_blank" href="http://parts.igem.org/Part:BBa_K1465202">(BBa_K1465202)</A> for M.A.X targets and UBP_target during PCR.</B> It has a chloramphenicol resistance and a length of 2517 bp. Each construct contains a different M.A.X target insert (mutA, muT, mutG, mutC) or UBP_target and is 351 bp long. </P></DIV>To increase the possibility of the insertion of the unnatural bases, we used 100 µM dNTPs and 200 µM isoG and 200 µM isoC<SUP>m</SUP> for each reaction. After variations of template concentrations from 1 ng µL<SUP>-1</SUP> to 50 ng µL<SUP>-1</SUP>, the best concentrations to acquire high-quality bands were 1 ng µL<SUP>-1</SUP> for the M.A.X targets and 25 ng µL<SUP>-1</SUP> for the UBP_target template. <DIV class="figure medium"><P class="figure subtitle"><B>Figure 7: PCR with Titanium Taq polymerase of pSB1C3_RuBisCo </B> with the inserts mutA, mutT, mutG, mutC (5 ng µL<SUP>-1</SUP>) and UBP_target (25 ng µL<SUP>-1</SUP>). The expected fragment is 351 bp long. </P></DIV>To quantify the efficiency of the incorporation of isoG and isoC<SUP>m</SUP>, all PCR products were tested/restricted with the M.A.X system. In order to achieve complete digestion, different incubation times from 1 h to 15 h were tested. The best results with <I>Bsa</I>I and <I>Mnl</I>I were achieved with an incubation of 15 h overnight. For the less stable enzymes <I>Eci</I>I and <I>Sap</I>I, a 2 h digestion with an addition of further enzyme after 1 h turned out to be optimal. Nevertheless, <I>Eci</I>I and <I>Sap</I>I could not digest the complete sample even if the concentrations are lowered. Therefore, we expected undigested bands in the M.A.X targets mutA and mutG for the whole experiment. After the first successful PCR, we tested if the presence of isoG and isoC<SUP>m</SUP> has any influence on the efficiency of the polymerase. So, we added both unnatural bases to every PCR reaction with the M.A.X targets as template to see if the intensity of the bands decreases.<DIV class="figure large"><P class="figure subtitle"><B>Figure 8: PCRs</B> with Titanium Taq (<B>A</B>), Go Taq G2 (<B>B</B>), Allin HiFi DNA Polymerase (<B>C</B>), innuDRY polymerase (<B>D</B>),  BioMaster-HS Taq PCR polymerase (<B>E</B>), FirePol DNA polymerase (<B>F</B>), Phusion DNA polymerase (<B>G</B>) and Q5 DNA polymerase (<B>H</B>). The template is pSB1C3_RuBisCo with the inserts mutA, mutT, mutG, mutC (5 ng µL<SUP>-1</SUP>) and UBP_target (25 ng µL<SUP>-1</SUP>) after the restriction digest with <I>Eci</I>I (mutA) and <I>Sap</I>I (mutG) for 2 h and <I>Bsa</I>I (mutT) and <I>Mnl</I>I (mutC) for 15 h.</P></DIV>The native fragments of the PCRs are as expected 351 bp long as can be seen in Figure 8. We always used the same template concentration and the same primers to ensure the comparability between the DNA polymerases. As can be seen in Figure 8, all Taq based polymerases are able to incorporate the unnatural base pair in the DNA.The best results were apparently achieved with the Go Taq G2 DNA polymerase (Figure 8 <B>B</B>). All lanes with the UBPs show clear bands and no mutations to T or A. The M.A.X restriction digest showed there are some mutations to C or the antisense strand mutated to G and was paired with a C, which caused the completed recognition site of <I>Mnl</I>I.Every PCR product of the UBP_target fragment was digested because of the second <I>Mnl</I>I restriction site, but differences can be shown between the M.A.X target mutC and the UBP_target PCR product because of the distance between both restriction sites.The digested PCR products of the BioMaster-HS Taq PCR polymerase and the Allin HiFi DNA Polymerase show more mutations to A than the other polymerases. But also, the TiTaq polymerase seems to miss incorporate the A instead of isoG. Moreover, the Phusion DNA polymerase proceeds to miss the incorporated G while the Q5 DNA polymerase does not show any bands containing the UBPs. Both polymerases have a proofreading function in contrast to the other polymerases.<P class="table-headline"><B>Table 2:</B> Used polymerases with specific modifications during the PCRs with isoG and isoC<SUP>m</SUP>.</P><TABLE><THEAD><TR><TH style="width: auto">Position in Figure 8</TH><TH style="width: auto">DNA polymerase</TH><TH style="width: auto">Distributor</TH><TH style="width: auto">Modification</TH><TH style="width: auto">Incorporation of UBP?</TH></TR></THEAD><TBODY><TR><TD align="center">A</TD><TD align="left"><A target="_blank" href="http://www.clontech.com/DE/Products/PCR/High_Yield_PCR/Titanium_Taq_DNA_Polymerase?sitex=10024:22372:US">Titanium Taq</A></TD><TD align="left">Clontech</TD><TD align="left">Lacks 5'-exonuclease activity</TD><TD align="center">yes</TD></TR><TR><TD align="center">B</TD><TD align="left"><A target="_blank" href="https://www.promega.de/products/pcr/endpoint-pcr/gotaq-g2-dna-polymerase/?catNum=M7841">GoTaq G2</A></TD><TD align="left">Promega</TD><TD align="left">Has 5’-3’ exonuclease activity</TD><TD align="center">yes</TD></TR><TR><TD align="center">C</TD><TD align="left"><A target="_blank" href="https://www.highqu.com/allintmhifi-dna-polymerase-2-u-l.html">Allin HiFi</A></TD><TD align="left">highQu</TD><TD align="left">Derived from <I>Pfu</I> polymerase with several mutations and proof reading function.</TD><TD align="center">yes</TD></TR><TR><TD align="center">D</TD><TD align="left"><A target="_blank" href="https://www.analytik-jena.de/fileadmin/content/import/imported_dam/br_Product_Guide_Reagents_Plastic_WEB_geschuetzt.pdf">innuDRY </A></TD><TD align="left">Analytik Jena</TD><TD align="left">Specific hot-start Taq DNA polymerase.</TD><TD align="center">yes</TD></TR><TR><TD align="center">E</TD><TD align="left"><A target="_blank" href=" http://biolabmix.ru/en/products/klassicheskaja_pcr/biomaster_hs-taq_pcr-color__2_/"> BioMaster-HS Taq </A></TD><TD align="left">Biolabmix</TD><TD align="left">Hot-Start <I>Taq</I> DNA polymerase</TD><TD align="center">yes</TD></TR><TR><TD align="center">F</TD><TD align="left"><A target="_blank" href=" https://www.sbd.ee/EN/products/pcr/firepol "> FirePol </A></TD><TD align="left">Solis Biodyne</TD><TD align="left">Has a 5’-3’ polymerization-dependent exonuclease replacement, but lacks 3’-5’ exonuclease activity</TD><TD align="center">yes</TD></TR><TR><TD align="center">G</TD><TD align="left"><A target="_blank" href="https://www.neb.com/products/m0531-phusion-high-fidelity-pcr-master-mix-with-hf-buffer#Product%20Information">Phusion</A></TD><TD align="left">NEB</TD><TD align="left">Derived <I>Pyrococcus</I> enzyme fused with a processivity-enhancing domain. It possesses 5’-3’ polymerase activity and 3’-5’ exonuclease.</TD><TD align="center">yes</TD></TR><TR><TD align="center">H</TD><TD align="left"><A target="_blank" href="https://www.neb.com/products/m0491-q5-high-fidelity-dna-polymerase#Product%20Information">Q5</A></TD><TD align="left">NEB</TD><TD align="left">Fused to the processivity-enhancing Sso7d DNA binding domain with an error rate ~280-fold lower than of <I>Taq</I> DNA polymerase</TD><TD align="center">No</TD></TR></TBODY></TABLE>We always used the same template concentration and the same primers to ensure the comparability between the DNA polymerases. As can be seen in the figures above, all Taq based polymerases are able to incorporate the unnatural base pair in the DNA. The best results were apparently achieved with the Go Taq G2 DNA polymerase. All lanes with the UBPs show clear bands and no mutations to T or A. The M.A.X restriction digest showed there are some mutations to C. The PCR product of the UBP_target fragment was digested because of the second <I>Mnl</I>I restriction site, but one can see a difference between the M.A.X target mutC and the UBP_target PCR product. The digested PCR products of the BioMaster-HS Taq PCR polymerase and the Allin HiFi DNA Polymerase show more mutations to A than the other polymerases. But also the TiTaq polymerase seems to miss incorporate the A instead of isoG. Moreover, the Phusion DNA polymerase proceeds to miss the incorporated G while the Q5 DNA polymerase does not show any bands containing the UBPs. Both polymerases have a proofreading function in contrast to the other polymerases.
        In the final analysis, the faster and the stronger the poof reading function of a polymerase is, the worse is the incorporation of the UBPs.
        The M.A.X system seems to be a good method for the first review of the efficiency of the polymerases. One can see if there is any incorporation of UBPs, so that sequencing is worthwhile. Minor deviations are not detectable by gel electrophoresis. It is also difficult to make a clear statement about the proportion of correctly or incorrectly incorporated unnatural bases, because the digested fragments seem to be less intensive than intact sequences.       
        </ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Isoguanine &amp; 5-methyl isocytosine in Nanopore Sequencing</H3><H4>Nanopore Sequencing</H4><DIV class="article"><A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Oxford Nanopore Technology's (ONT)</A> sequencing technology offers a great potential as a tool for the detection of unnatural bases in DNA. In ONT sequencing, protein nanopores are distributed inside a synthetic membrane of high electrical resistance. When applying an electrical field across this membrane, an ionic current passes through each nanopore which is being measured and recorded. If a biomolecule, such as proteins, RNA or DNA are located inside the Nanopore, the ionic current is influenced. These characteristic changes can be measured and used to identify which molecule is passing through the nanopore. (Feng <I>et al.</I>, 2015) This way, an algorithm called the &quot;basecaller&quot; is able to predict the nucleotide sequence of a single stranded DNA or RNA molecule based on the raw data that is recorded when it is pulled through the nanopore. Since the commercial availability of the portable sequencer MinION in 2015 (Check Hayden, 2015), strong improvements have been made in terms of increasing the bascalling accuracy. Even though the error rate is still high compared to other sequencing techniques, the advantage of having long reads of several kilobases is often preferential regarding sequencing of DNA containing repetitive sequences or mobile genetic elements like transposable elements (Debladis <I>et al.</I>, 2017) More recently, efforts have been made towards the analysis of epigenetic information based on the identification of modified bases in nucleic acids with nanopore sequencing. For example, methylated cytosine was shown to be distinguishable from unmodified cytosine by training a hidden Markov model (Simpson <I>et al.</I>, 2017).
                        </DIV><DIV class="article">
                            Compared to other sequencing technologies, nanopore sequencing offers several advantages regarding the detection of unnatural bases. Most importantly, no PCR amplification of the DNA sample is needed in the process of library preparation. This way, no information gets lost prior to sequencing due to potentially lower PCR amplification fidelity of the unnatural base pair. Another big advantage is that no additional chemistry is needed in the process of sequencing. Other sequencing technologies such as 454, Sanger, Illumina and PacBio are based on polymerases that synthesize a DNA strand complement to a template being sequenced. When a specially labeled nucleotide is incorporated, a detectable signal is emitted. This is problematic regarding sequencing of DNA containing unnatural bases, as additional labeled nucleotides would be needed for a continuous strand synthesis and to produce a unique signal for the unnatural bases. Considering the development costs for this new chemistry, the necessary process adaptations and increased complexity of data analysis, the sequencing of orthogonal unnatural bases is unlikely to be feasible with these technologies. In contrast, nanopore sequencing omits the necessity for additional chemistry and it is unlikely that sequencing will be interrupted by unnatural bases passing through the nanopore. On top of that, Nanopore sequencing was shown to be applicable for direct sequencing of RNA, without prior transcription into cDNA (Garalde <I>et al.</I>, 2016). Therefore, it promises to be suitable for transcription studies involving unnatural bases too.
                        </DIV><DIV class="article">
                            We aimed to examine if Oxford Nanopore sequencing is suitable for sequencing DNA containing unnatural bases. Therefore, we sequenced different DNA samples containing either the unnatural bases isoG and isoC<SUP>m</SUP> or any natural bases in the same sequence context to see if the output signal differs significantly between these groups. The data processing and evaluation was performed with the help of our own <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software#iCG">software iCG</A>, that we developed specifically for analyzing Nanopore sequencing data of DNA containing unnatural bases. Our aim is to create a linear discriminant analysis model that is able to discriminate between isoG/isoC<SUP>m</SUP> and natural bases in the given neighboring sequence context of two bases upstream and two bases downstream of the position of interest. For a detailed description of how the software works, please refer to our <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A> page.
                        </DIV><H4>Reference Sample Preparation &amp; Sequencing</H4><DIV class="article">
                            In order to examine if the unnatural bases isoG and isoC<SUP>m</SUP> are differentiable from the natural bases through nanopore sequencing, five different DNA samples were prepared that differed only at a single sequence position, containing either an unnatural base or one of the four natural bases at this position of interest. For our experiments, we started by sequencing isoC<SUP>m</SUP> in the sequence context <FONT face="courier new">5'‑AG\iC<SUP>m</SUP>\CC‑3'</FONT> and, on the reverse strand, isoG in the sequence context <FONT face="courier new">5'‑GG\iG\CT‑3'</FONT>. For this purpose, we constructed five reference DNA samples with the following sequences:
                        </DIV><DIV class="figure seventy"><P class="figure subtitle"><B>Figure 9: Annealed oligonucleotides used for reference sample preparation.</B> 
                                Sequences at the position of interest of DNA samples used as references for nanopore sequencing.
                            </P></DIV><DIV class="article">
                            Each reference DNA sample was prepared starting from a pair of complementary synthetic oligonucleotides. The oligos containing isoguanine or 5‑methyl isocytosine were synthesized by <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Biolegio</A>, including subsequent purification through polyacrylamid gel electrophoresis (PAGE). Mass spectrometry and ultra performance liquid chromatography data (Figure 1) provided by Biolegio indicate that the concentrations of unmodified side products are below detection limits. In general, the manufacturer specifies the purity of PAGE purified oligonuleotides containing modified bases to be greater than 95 %. The oligonucleotides containing exclusively natural bases were ordered from <A target="_blank" href="http://www.metabion.com/">metabion</A> and were purified by desalting. 
                        </DIV><DIV class="article">                         The existence of the unnatural bases in the oligos from Biolegio and the correct sequence of the oligos containing natural bases exclusively was further confirmed by analysis with our <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software#MAX">Mutational Analysis Xplorer (M.A.X.)</A>. This orthogonal analysis approach reveals that the purity and sequence identity of the oligos was very high.
                        </DIV><DIV class="figure large"><P class="figure subtitle"><B>Figure 10: UPLC and MS data from oligos containing isoguanine or 5‑methyl isocytosine.</B> 
                                Oligonucleotides containing isoguanine or 5‑methyl isocytosine were synthesized and analysed by Biolegio by ultra performance liquid chromatography (UPLC) and mass spectrometry (MS). Shown above are the results from the UPLC (above) and MS (below) analysis for each of the complementary oligos containing either isoguanine (A) or 5‑methyl isocytosine (B).
                            </P></DIV><DIV class="article">
                            For each DNA sample, a complementary pair of oligonucleotides was <A target="_blank" href="https://static.igem.org/mediawiki/2017/d/d1/T--Bielefeld-CeBiTec--protocol_ssDNA_annealing.pdf">annealed</A> and ligated into a plasmid backbone (<A target="_blank" href="http://parts.igem.org/Part:BBa_K1465202">BBa_K1465202</A>) previously linearized by <I>Xba</I>I and <I>Bmt</I>I. For this purpose, a leading <I>Bmt</I>I and a tailing <I>Spe</I>I recognition site were included into the oligonucleotide sequences. After ligation, re-ligated backbone was linearized by digestion with <I>Xba</I>I. After consecutive digestion of double and single stranded linear DNA fragments with lambda exonuclease and <I>E. coli</I> exonuclease I, the DNA samples were linearized through <I>Eco</I>RV digestion and purified for sequencing library preparation. An individual library was prepared for each DNA sample, according to the 1D Library Protocol for SQK-LSK108, starting from the end repair step.
                        </DIV><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 11: Library preparation for Oxford Nanopore sequencing.</B> Purification of DNA containing the unnatural base pair, after the adapter ligation step of library preparation for Oxford Nanopore sequencing.
                                    </P></DIV></DIV><DIV class="half right"><DIV class="figure large"><P class="figure subtitle"><B>Figure 12: MinIon sequencer with R9.4 flowcell.</B> The MinIon sequencer that we used in our experiments, together with a R9.4 flowcell.
                                    </P></DIV></DIV></DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 13: Status of the pore grid during sequencing.</B> While sequencing, the software MinKnow gives online feedback about the pores in the flowcell. 
                            </P></DIV><DIV class="article">
                            Sequencing was performed with a new FLO-MIN106 R9.4 flowcell on a MinION sequencer with the MinKNOW software version 1.7.14.1. Each DNA sample was sequenced individually, with intermediate washing steps with the Flow Cell Wash Kit EXP-WSH002. For every sample, a minimum of 260000 reads was generated. Basecalling was performed locally with ONT albacore version 1.2.4. .
                        </DIV><H4>Data Analysis with our Software Package iCG</H4><DIV class="article">
                            After basecalling wih albacore, the Nanopore data was analyzed with our own software iCG. The scripts that were created in this process were combined to a powerful software tool named <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software#iCG">iCG</A> that can potentially be used for the analysis of DNA containing arbitrary unnatural bases with Nanopore Sequencing.
                        </DIV><DIV class="article">
                            In the first step, the reads were filtered by iCG filter in order to identify reads that contain the region of interest and have a high basecalling quality. Regarding the parameters minimum length, maximum length and minimum mean Phred qscore, the default argument settings of iCG filter were used for filtering. Of the remaining reads, only those containing the neighboring sequence context of 15 bases upstream and downstream of the POI were selected, without considering the close sequence context (blur region) of 3 ±1 bases around the POI, where influences of the unnatural bases may lead to unpredictable behavior of the basecaller. The matching reads were tolerated to contain a maximum of 2 mismatches, including indels. The maximum deviation in length was set to 1 base and reads containing the region of interest multiple times were rejected. Additionally, the selected reads were further filtered for a minimum mean quality score of 14 in this restricted sequence context and sorted by their stand orientation. For further information about, please read more about iCG on our <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software#iCG">Software</A> page.
                        </DIV><DIV class="figure large"><P class="figure subtitle"><B>Figure 14: Normalized signal traces of analyzed DNA samples.</B>
                            Overlayed, normalized signal traces of DNA samples containing either isoG/isoC<SUP>m</SUP> or any natural base at the position of interest in the analyzed sequence context. The reads displayed in these plots were selected from their respective sequencing runs by using iCG filter, using the same filter settings for all DNA samples. To remove contaminating reads from previous sequencing runs, a quantile of 0.7 of the most deviating reads was removed previous to plotting with the help of <I>iCG model</I>.
                            </P></DIV><DIV class="article">
                            Afterwards, <I>iCG model</I> was used to create linear discriminant models based on the filtered groups of template reads gathered by iCG filter. Different setting for the amount of removed, deviating reads were tested. Figure 5 shows plots of the Region of interest for both the forward and the reverse strand and all five template groups, with a quantile of 0.7 removed reads. For both strand orientations, there is a distinct difference in the mean, normalized signal trace detectable comparing the sequences containing an unnatural base with those containing a natural base at the position of interest. 
                        </DIV><DIV class="article">
                            Looking at the plot of the sample containing isoC<SUP>m</SUP>, the signal seems to be much noisier than any other sample analyzed. Based on the quality control after synthesis by UPLC and MS and our orthogonal analysis with the M.A.X. system, it is highly unlikely that this result is based on impurities of the oligos prior to sample preparation. Keeping in mind that the displayed data is the product of several steps of data processing, including event generation during sequencing, basecalling, alignment to the reference sequence and normalization, one possible explanation for this result are errors in this data processing pipeline as a consequence of a strong signal deviations to the expected signals of natural bases. For example, if the basecaller misinterprets the signal in the sequence context of isoC<SUP>m</SUP>, the error is likely to propagate and cause problems during sequence alignment and time-dependent normalization. Alternatively but less likely, the signal noise could be caused by influences of the sample preparation on isoC<SUP>m</SUP>. It has been shown that deoxy-isocytidine and deoxy-isocytidine triphosphate show a tendency for deamination, resulting in their respective uridine analogues (Switzer <I>et al.</I>, 1993). Considering the alkaline reaction conditions of the exonuclease reactions performed during sample preparation, the isoC<SUP>m</SUP> bases in the analyzed sample could be partly deaminated to methylated uracil, potentially influencing the signal in Nanopore sequencing.
                        </DIV><H4>Linear Discriminant Analysis</H4><DIV class="article">
                            Based on the data presented in Figure 14, a cluster analysis based on linear discriminant analysis was conducted using <I>iCG model</I>. Figure 15 shows dot-plots for the forward and reverse models, presenting the linear discriminants of the reads each respective model was created with. The direct comparison of both models reveals that the model created upon the data of the reverse strand seems to perform better in terms of classification of the sequencing reads. Except for the groups containing A and G at the position of interest, which slightly overlap with each other, all other groups are well separated from each other. On the other hand, the linear discriminant analysis of the data of the forward strand was unable to properly separate the reads containing A, G and iC<SUP>m</SUP> from each other, mainly due to widely scattered reads of the iC<SUP>m</SUP> group. Both results coincide with the visual assessment of the signal traces in Figure 14.
                        </DIV><DIV class="figure large"><P class="figure subtitle"><B>Figure 15: Dot-plots of the linear discriminant models of the forward and reverse strand.</B> Dot-plots of the linear discriminants of the reads used for the creation of the statistical models for base prediction at the position of interest in the forward and reverse strand. The data used for the linear discriminant analysis was previously filtered by removing 70 % of reads from each group, based on their deviation from the groups median signal in the neighboring sequence context of the position of interest.
                            </P></DIV><DIV class="article">
                            Since a statistical model should not be tested with the very data it was created with, we prepared a new set of DNA samples to properly evaluate the performance of both models concerning the prediction of bases at the position of interest in their respective sequence context. For this purpose, we modified the RuBisCO plasmid that was used for the first sample preparation by cloning five different sequences downstream RuBisCO with <A target="_blank" href="https://static.igem.org/mediawiki/2017/8/84/T--Bielefeld-CeBiTec--protocol_Standard_Biobrick.pdf">Standard BioBrick assembly</A> (part numbers , , , , ). Each of these plasmids contains a 25 nt sequence that is unique, while the remaining plasmid sequence is the same. These unique sequences can be used for identification assignment of sequencing reads comparable to the Nanopore barcoding approach. Starting with these five plasmids, we prepared new DNA samples according to the same procedure explained above. After ligation, all five samples were pooled in approximately equimolar proportion and further prepared for sequencing. After sequencing and basecalling of this pooled sample, the reads were assigned to their respective group by using iCG filter with the &quot;--barcode&quot; argument and each plasmid's unique sequence. After filtering, 50 reads of every group were randomly selected in order to be used for evaluating the performance of the linear discriminant models with iCG predict. The results of this evaluation are summarized in Figure 16.
                        </DIV><DIV class="figure large"><P class="figure subtitle"><B>Figure 16: Evaluation of the linear discriminant analysis models.</B> 
                            Evaluation results of the linear discriminant analysis models for the forward and reverse strand. (A) Linear discriminants of the test data colored in accordance with their respective base prediction. (B) Distribution of predicted bases. Based on the assumption that every read in the test data set was correctly assigned with the barcoding approach, equal portions of 20 % for each base would be ideal, corresponding to 50 reads per test data group. (C) Fidelity of base prediction, revealing which base predictions were made for the reads of each group individually.
                            </P></DIV><DIV class="article">
                            The results in Figure 16 indicate that the linear discriminant model for the reverse strand orientation is performing better than the model for the sense strand. The base prediction fidelity is especially high for reads containing an adenine, a cytosine or an isoguanine at the position of interest. Due to the hydrolysis of isoC<SUP>m</SUP> to T and the tautomerisation of isoG, leading to mispairing with T, the most common mutation that leading to a loss of the unnatural base pair between isoG and isoC<SUP>m</SUP> is the mutation from isoG to A (Bande <I>et al.</I>, 2015). Considering the fidelity of base prediction for both A and isoG with the reverse strand model, we conclude that this linear discriminant analysis model is well suited for the discrimination between isoG and all natural bases in the sequence context <FONT face="courier new">5'-ggNct-3'</FONT>. Therefore, we could show that the <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software#iCG">software package iCG</A> is applicable for the analysis of experiments with unnatural bases.
                        </DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Orthogonal Analysis with M.A.X and iCG</H3><DIV class="article">
                    In order to test the orthogonality of both methods we developed for the analysis of DNA containing unnatural bases, we performed an further PCR reaction with a template containing isoG and isoC<SUP>m</SUP> and analyzed the amplified DNA with both M.A.X. and iCG. The experimental setup was identical to those of our previous PCR experiments, except for the choice of primers and the reaction volume. A GoTaq PCR reaction with the template containing the unnatural bases was prepared with a total reaction volume of 250 μL. As primers, the standard iGEM sequencing primers VR and VF2 were used in the reaction.
                </DIV><DIV class="figure large"><P class="figure subtitle"><B>Figure 17: PCR of DNA containing isoG/isoC<SUP>m</SUP> with GoTaq analyzed with iCG and M.A.X.</B> A DNA template containing an unnatural base pair between isoG and isoC<SUP>m</SUP> was analyzed in an orthogonal approach with M.A.X. and iCG. (A) After PCR reaction and PCR cleanup, four separate restriction reactions were performed with the addition of either <I>Eci</I>I, <I>Bsa</I>I, <I>Sap</I>I or no restriction enzyme. In the mentioned order, the amplified DNA contains a recognition sequence for these enzymes in case of a mutation of isoG to A, T and G. The reaction without restriction enzyme was used to control if a fragment of the expected size of 761 bp was amplified. (B) After PCR cleanup, the sample was sequenced with Oxford Nanopore sequencing and subsequently analyzed with the software suite iCG. A dot-plot showing the linear discriminants of each filtered read after base prediction is shown, as well as the numerical result of the base prediction and a plot showing the normalized signal traces at the position of interest.
                        </P></DIV><DIV class="article">
                    The results of both analysis methods presented in Figure 17 reveal that the fidelity UBP replication is not perfect for this experimental setup. In the gel electrophoresis after the digestion with the enzymes of the M.A.X system, a distinct band is visible for the <I>Eci</I>I digest, indicating that mutations of isoG to A were occurring during the PCR reaction. The analysis with Nanopore sequencing and iCG reveals a similar result, showing that approximately 54 % of the analyzed reads contained isoG at the position of interest, while 39 % were predicted to have an A at this position. Therefore, both methods reveal that the mutation of isoG to A is the most frequent mutational event leading to a loss of the unnatural base pair between isoG and isoC<SUP>m</SUP>, which is in accordance to our expectations and respective results in literature (Switzer <I>et al.</I>, 1993, Johnson <I>et al.</I>, 2004). Interestingly, the replication fidelity seems to be considerably lower compared to our first PCR experiments with the GoTaq polymerase. Possibly, the reason for this difference is due to the longer storage time of the DNA template at 4 ℃. It was shown that d-isoCTP is quite unstable even when stored at -20 ℃ (Switzer <I>et al.</I>, 1993), possibly due to hydrolysis of isoC. Therefore, we postulate that the reduced PCR fidelity might be caused by a a reduced template integrity.
                </DIV><DIV class="article">
                    The results in Figure 17 indicate that the analysis with Nanopore sequencing and iCG as well as the analysis with M.A.X. are orthogonal methods producing the same analysis results. Arguably, it can be supposed that the results produced by Nanopore sequencing are more precise, allowing the detection of uncommon mutation events that are not detectable with M.A.X. Nevertheless, the analysis with restriction digests is much faster and cost efficient.
                </DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>References</H3><DIV class="article"><B>Bande, O., Abu El Asrar, R., Braddick, D., Dumbre, S., Pezo, V., Schepers, G., Pinheiro, V.B., Lescrinier, E., Holliger, P., Marlière, P., and Herdewijn, P.</B> (2015). Isoguanine and 5-Methyl-Isocytosine Bases, In Vitro and In Vivo. Chem. - A Eur. J. <B>21</B>: 5009–5022.
                    </DIV><DIV class="article"><B>Check Hayden, E.</B> (2015). Pint-sized DNA sequencer impresses first users. Nature <B>521</B>: 15–16.
                    </DIV><DIV class="article"><B>Debladis, E., Llauro, C., Carpentier, M.-C., Mirouze, M., and Panaud, O.</B> (2017). Detection of active transposable elements in Arabidopsis thaliana using Oxford Nanopore Sequencing technology. BMC Genomics <B>18</B>: 537.
                    </DIV><DIV class="article"><B>Feng, Y., Zhang, Y., Ying, C., Wang, D., and Du, C.</B> (2015). Nanopore-based Fourth-generation DNA Sequencing Technology. Genomics. Proteomics Bioinformatics <B>13</B>: 4–16.
                    </DIV><DIV class="article"><B>Garalde, D.R. et al.</B> (2016). Highly parallel direct RNA sequencing on an array of nanopores. bioRxiv: 68809.
                    </DIV><DIV class="article"><B>Johnson, S.C., Sherrill, C.B., Marshall, D.J., Moser, M.J., and Prudent, J.R.</B> (2004). A third base pair for the polymerase chain reaction: inserting isoC and isoG. Nucleic Acids Res. <B>32</B>: 1937–41.
                    </DIV><DIV class="article"><B>Simpson, J.T., Workman, R.E., Zuzarte, P.C., David, M., Dursi, L.J., and Timp, W.</B> (2017). Detecting DNA cytosine methylation using nanopore sequencing. Nat Meth <B>14</B>: 407–410.
                    </DIV><DIV class="article"><B>Sismour, A.M. and Benner, S.A. </B>(2005). The use of thymidine analogs to improve the replication of an extra DNA base pair: a synthetic biological system. Nucleic Acids Res. <B>33</B>: 5640–6.</DIV><DIV class="article"><B>Switzer, C.Y., Moroney, S.E., and Benner, S.A.</B> (1993). Enzymic recognition of the base pair between isocytidine and isoguanosine. Biochemistry <B>32</B>: 10489–10496.
                    </DIV><DIV class="article"><B>Zhang, Y., Lamb, B.M., Feldman, A.W., Zhou, A.X., Lavergne, T., Li, L., and Romesberg, F.E.</B> (2017). A semisynthetic organism engineered for the stable expansion of the genetic alphabet. Proc. Natl. Acad. Sci. <B>114</B>: 1317–1322.
                    </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>