<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Improve skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Improve</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/0/01/T--Bielefeld-CeBiTec--title-img-preculture2.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Part Improvement
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Introduction</H2><DIV class="article">
Besides the integration of unnatural bases into the genetic code of <I>E.coli</I> and the usage of non-canonical amino acids (ncAA) for advanced protein design, the creation and selection of new aminoacyl-tRNA synthetases (aaRS) is a fundamental part of our project. To determine the most effective and specific aaRS for the incorporation of the desired ncAAs, an evaluation system of the synthetases is essential. Literature review revealed a synthetase test system “pFRY” (a Reporter Plasmid for measuring ncAA Incorporation, <A target="_blank" href="http://parts.igem.org/Part:BBa_K1416004">BBa_K1416004</A>) of the iGEM-Team <A target="_blank" href="https://2014.igem.org/Team:Austin_Texas/kit">Austin, Texas, 2014</A> which was also used by the team <A target="_blank" href="https://2016.igem.org/Team:Aachen/Description#characterization">Aachen 2016</A> (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2020040">BBa_K2020040</A>). 
The pFRY plasmid consists of an mRFP CDS which is connected to an sfGFP  CDS by a linker sequence containing an amber stop codon. The expression of the plasmid results either in red fluorescence, or - if the ncAA is incorporated at the amber stop codon within the linker site - in both: red and green fluorescence. By comparison of the fluorescence levels it is possible to determine the incorporation efficiency of the generated synthetase variants. </DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 1: Comparison of the RFP-GFP (BBa_K2020040) and the CFP-YFP (K22012343) system.</B> A) shows the expression of the RFP unit of the fusion protein when not cotransformed with an aaRS. B) shows the expression of the complete RFP-GFP fusion protein when cotransformed with an aaRS. C) shows the expression of the CFP unit of the improved part when not cotransformed with an aaRS. D) shows the expression of the complete CFP-YFP fusion protein of the improved part when cotransformed with an aaRS.</P></DIV><DIV class="article">
We liked this elegant solution which shows synthetase activity as well as synthetase efficiency. Our investigations revealed that there were some issues with the choice of RFP and GFP for the system, e.g. a high overlap of the absorption spectra and a low emission level. This can be improved by substitution with CFP and YFP (Figure 1). We constructed and characterized parts for this FRET system (<A target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4215735/">Ma <I>et al.</I>, 2014</A>) which leads to a more accurate distinction between the partial (CFP) and the whole (CFP-YFP) expressed fusion protein.
	</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Short Summary</H2><ARTICLE>
We designed the CFP-YFP test system (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201343">BBa_K2201343</A>) just like the RFP-GFP system and only changed the coding sequences of both of the fluorescent proteins. 
First we compared the CFP and the RFP domain of the test system by transforming them into BL21(DE3) solely and performing absorption and emission measurements. We detected that the absorption and the emission maxima of the RFP are equal to the information of Texas 2014. Nevertheless, they are only 20 nm apart from each other, which lead to the first measurement problems. Furthermore we detected a second absorption maximum at 505 nm, not mentioned by Texas 2014. This excitation wave length was more suitable for RFP than 585 nm, but on the other hand very close to the absorption maximum of GFP (485 nm). As a result, heavy excitement and emission of RFP were observed as a byproduct of the GFP analysis. The absorption and emission maxima of CFP are ~ 45 nm apart and thereby very precisely differentiable. Nevertheless, we could detect that the emission signal of the CFP is approximately five times higher than that of RFP, if both of them are excited at their absorption maxima under comparable conditions. We could replicate these results <I>in vitro</I> and <I>in vivo</I>.
We demonstrated that our FRET system is functional. Excitation of the YFP-CFP fusion protein at wavelength specific for YFP excitation (475 nm) or CFP excitation (430 nm) both leads to strong YFP fluorescence signals. This means that the energy of the CFP emission is transferred to the YFP and thus the FRET system is working. Also there was still a clear CFP signal present at 475 nm which means that only the whole fusion protein of CFP and YFP forms the FRET system, and that solely CFP, where no ncAA was incorporated in the linker, still is present and detectable alongside with the FRET system. 
By comparing the emission spectra of our improved test system when cotransformed with different aaRS, the efficiency of the aaRS was analyzed. Our results imply that a comparison of different aaRS is possible, just by comparison of the relative emission at 475 nm, representing the CFP amount, and at 525 nm, representing the fusion protein amount of the cotransformants when cultivated with and without the specific ncAA.
Therefore, we developed a ranking system for the evaluation of synthetases based on initial work by Texas 2014. This new scoring system is suitable for our improved FRET system due to the integration of a negative and a positive score.
	</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>RFP and CFP Activity of the Systems <I>in vitro</I></H2><ARTICLE>
First we characterized the first translated units of the aaRS-test systems by transforming them solely into <I>E.coli</I> BL21(DE3) without any aaRS. This way only the RFP of the Texas part and the CFP of our improved part were expressed. 
We cultivated each transformant in 50 mL of LB-media in a 500 mL cultivation flask for 16 hours. Protein expression was induced with IPTG after 6 hours of cultivation at 37 °C and 150 rpm. We harvested the cells and lysated the cell pellet with the Ribolyzer and centrifuged the cell debris. Afterwards, we measured the fluorescence in 200 µL of the clear lysate. 
Figure 1 shows the results of the absorption measurement of the RFP from wavelengths of 475 nm to 750 nm. We detected two absorption maxima at 505 nm and 590 nm. The absorption maximum at 590 nm was used by Texas to excite the RFP and measure its emission on 605 nm. Here we faced the first challenge. When exciting and measuring at the determined absorption and emission maxima, the received amount of the irradiated light was high, thus the measurement of the RFP-signal was impossible. To solve this problem we decreased the excitation wavelength by 5 nm to 585 nm and increased the emission wavelength by 5 nm to 615 nm. This way, the noise was completely removed and we could proceed the measurements. 
To tackle this issue we decided to continue the measurements at the absorption maximum of 505 nm. This also leads to an excitement of the RFP and an emission maximum at 610 nm, but prevents the possibility of measuring the irradiated light.
	</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 1: Relative absorption and emission of RFP.</B> The highest value equals one. The maximal absorption peaks at ~505 nm (grey) and at ~590 nm is another local maximum. The emission maximum lays at ~ 610 nm (red).</P></DIV><ARTICLE>
Next, we started the characterization of the CFP-unit of our improved part as a component of the aaRS-test system. Here we determined just one clear absorption maximum at 430 nm and one clear emission maximum at 475 nm (Figure 2). The two maxima were 45 nm apart, which allowed an easy excitement and emission process without any noise or measurement of the irradiating light.
	</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 2: Relative absorption and emission of CFP.</B> The highest value equals one. The maximal absorption lays at ~430 nm (grey line). The emission maximum lays at ~475 nm (cyan line).</P></DIV><ARTICLE>
We then compared the intensity of the emitted light of the two different fluorescent protein units to determine if there were any differences. To do so we took care that the OD<SUB>600</SUB> of the samples was similar and normalized the emitted light after excitation to the protein amount in the cell lysate. To our surprise, the signal we got from the CFP was about six times higher than of the highest RFP-signal (Figure 3). 
	</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 3: Emission normalized to the amount of protein in the cell lysate of the two samples.</B> The emisson of CFP after excitation at 430 nm is shown in cyan. RFP emission when excited at 505 nm is shown in red squares, when excited at 585 nm is shown in red triangles. The lower red course is caused by the emission of RFP when excited at 505 nm and the higher red course is caused by RFP when excited with 585 nm.</P></DIV><ARTICLE>
We repeated this experiment with two biological replicates to make sure there was no mistake in the cultivation, induction, harvesting or sample preparation process. We measured three technical replicates of the cell lysate of the two biological replicates at their emission maximum (Figure 4). To our surprise the signal of our CFP was still approximately six times higher than the higher RFP signal, when excited at 585 nm. 
	</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 4: Maximal emission of CFP (cyan) and RFP (red) normalized to the amount of protein in the cell lysate.</B> The samples were excited at the specific absorption maximum (CFP: 430, RFP: 505, 585).</P></DIV><ARTICLE>
To make sure there was no systematical mistake in our process we repeated the experiment, now with six biological replicates each, and only did an <I>in vivo</I> measurement, hoping to reduce potential sources of error in the harvesting or sample preparation process.
	</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>RFP and CFP Activity of the Systems <I>in vivo</I></H2><ARTICLE>
We cultivated the replicates of the transformants in a 12-well microtiter plate (Figure 5), with a volume of 1 mL LB-media in each well. The cells were cultivated for 24 hours at 30 °C with 400 rpm. From each well 200 µL of the culture were transferred to a 96-well microtiter plate and measured in the Tecan Reader. The six replicates of the CFP-YFP part showed a yellowish to greenish color and the RFP-GFP part a reddish color. This implicates that both transformants expressed the fusion protein properly. Before we did the measurements we determined the OD<SUB>600</SUB> of each of the replicates. The values of all the twelve replicates were very close to each other (2.22 ± 0.06). Still, we normalized the emission signal to the corresponding OD<SUB>600</SUB>.
	</ARTICLE><DIV class="figure small"><P class="figure subtitle"><B>Figure 5: Six biological replicates of <I>E.coli</I> expressing CFP-YFP or RFP-GFP.</B> The CFP-YFP part BBa_K2201343 (top, greenish) and the RFP-GFP part BBa_K2020040 (bottom, reddish). The cells were cultivated in 1 mL LB-media for 24 hours at 30 °C with 400 rpm.The colors indicate that all cultures started expressing the desired fusion protein.</P></DIV><ARTICLE>
The result of the <I>in vivo</I> characterization confirmed the outcome of the <I>in vitro</I> measurements. Again, the CFP-signal was six times higher than the RFP-signal (Figure 6). We cannot name the exact reason for this result, be it a higher fluorescence yield of CFP, a higher expression or faster folding time of CFP. This reasons could be further investigated with e.g. purified protein experiments, western blot. Nevertheless our experiment shows that the CFP-unit of our improved aaRS-test system itself is more suitable for an application than the RFP-unit of the Texas part. 
	</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 6: Maximal emission of CFP (cyan) and RFP (red) normalized to the OD<SUB>600</SUB> of the six replicates.</B> The samples were excited at the specific absorption maximum (CFP: 430, RFP: 505, 585).</P></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>GFP and YFP/FRET Activity of the Systems <I>in vivo</I></H2><DIV class="article">
After the characterization of the CFP- and the RFP-units of the aaRS-test systems, we started the cotransformations of the test systems with some of our aaRS. Namely the CouAA-RS (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201204">BBa_K2201204</A>), the Prk-RS (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201201">BBa_K2201201</A>) and the 2-NPA-RS (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201200">BBa_K2201200</A>). At first the cultivations of the cotransformants were done without their specific ncAA, to investigate the influence of the cotransformation with different aaRS separately.
We compared the variation of the emission signals of the test systems when cotransformed with the CouAA-RS to verify the production of the complete fusion proteins. This is possible due to the limited specificity and fidelity of artificially selected and evolved synthetases. They will always couple native amino acids to the amber tRNA and thus a proportion of the whole fusion protein will always be synthesized.
Figure 7 shows the emission spectrum of a culture of the cotransformants described above. When excited at the absorption maximum of GFP, at 485 nm, we can now measure a GFP-signal at 525 nm, which was not present when no CouAA-RS was present in the cells. Even when the GFP-signal was clear to see, and an expression of the whole RFP-GFP fusion protein was thereby confirmed, we also see a very high emission of RFP. This was caused by the broad overlap in the absorption spectrum of GFP and RFP. The RFP and GFP present in the cell will compete for the light. Therefore, excitation of this protein in the sample will lead to a weaker GFP-signal than would be expected, if no RFP would be present. 
	</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 7: Relative emission spectrum (excited at 485 nm, gray) of the RFP-GFP system.</B> Cotransformed with the CouAA-RS (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201204">BBa_K2201204</A>), cultivated without CouAA. Maximal emission of the GFP at 525 nm (green line) and maximal emission of RFP at 610 nm (red line).</P></DIV><ARTICLE>
In the cotransformants of the CFP-YFP system and the CouAA-RS we could confirm our desired FRET-system.
First, we excited the sample with light of 475 nm, which corresponds to the emission maximum of CFP and a reasonable absorption for YFP. The measured emission signal of the CFP-YFP system had a maximum at 525 nm (Figure 8). This matches with the expected YFP emission maximum. There was strong evidence that through the cotransformation, a specific amount of the whole CFP-YFP fusion protein was expressed in the cells. We then excited the sample at 430 nm, the absorption maximum of CFP. If there were no FRET system present in the sample we would expect a CFP signal with an emission maximum at 475 nm. The measured emission spectrum still had its maximum near 525 nm. We confirmed the FRET system by measuring the emission of light at the YFP emission maximum when excited at the CFP absorption maximum. This can only be done, if the energy of the excited CFP is transferred to the YFP before the emission process.
	</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 8: Relative emission spectrum (CFP signal: excited at 430 nm; YFP signal: excited at 475 nm) of the CFP-YFP system.</B> Cotransformed with the CouAA-RS (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201204">BBa_K2201204</A>). Maximal absorption of CFP at 430 nm (gray line). Maximal emission of CFP at 475 nm (blue). Maximal emission of YFP at 525 nm (yellow).</P></DIV><ARTICLE>
After we confirmed that our FRET system works, we wanted to determine if there are any substantial differences in the normalized emission spectra of the aaRS-test system when cotransformed with different aaRS. This is essential for designing a ranking system and to compare the properties of the different synthetases among each other.
	</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Comparison of Different aaRS Based on the CFP-YFP FRET-System </H2><ARTICLE>
Figure 9 shows the emission spectra of the CFP-YFP system when cotransfomred with three different aaRS cultivated without their specific ncAA. They differ from each other in form and location of the emission maximum. The Prk-RS has its emission maximum at 520 nm, which is very close to the YFP-emission maximum. This indicates that the Prk-RS is relatively broad range enzymes, and couples native amino acids on the amber tRNA, which leads to a high amount of the complete CFP-YFP fusion protein. The emission spectrum when cotransformed with the CouAA-RS also has its maximum near 525 nm. It is also a bit shifted to the lower wavelengths and has increased values near 475 nm, which indicates that more solely CFP-units are present. The 2-NPA-RS leads to an emission spectrum where even a little peak at 475 nm is visible, which indicates that this aaRS produces the a lower amount of the whole fusion protein. We thereby showed, that different aaRS cotransformed with our improved test system lead to distinguishable emission spectra. The ratio of CFP-signal (emission at 475 nm) to YFP-signal (emission at 525 nm) gives information about the ratio of solely CFP-units to whole CFP-YFP FRET systems when excited at the CFP absorption maximum of 430 nm. This information is crucial to estimate how often amino acids are coupled to the amber tRNA by the investigated synthetase.
	</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 9: Relative emission spectrum (exited at 430 nm) of the CFP-YFP system (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201343">BBa_K2201343</A>).</B> Cotransformed with the CouAA-RS (BBa_K2201204), Prk-RS (BBa_K2201201) and the 2-NPA-RS (BBa_K2201200). All cultivated without their specific non-canonical amino acid. Maximal emission of CFP at 475 nm (blue). Maximal emission of YFP / FRET at 525 nm (yellow).</P></DIV><ARTICLE>
The last aspect we have to confirmed, if there are any shifts in the emission spectra of the cotransformants, if cultivated with and without the specific ncAA of the cotransformed aaRS. This test was done with the Prk-RS and the 2-NPA-RS and the results are shown in Figure 10.
The emission spectra when cotransformed with the Prk-RS showed a clear and a substantial shift, when cultivated with and without Prk, respectively (Figure 10, left). Without Prk there is a bulge in the emission peak at 475 nm, due to the presence of solely CFP-units, where the synthesis of the CFP-YFP fusion protein stopped at the amber codon in the linker. The maximum of the emission spectrum is shifted towards the maximal YFP emission of 525 nm but not located there. If cultivated with Prk the bulge at 475 nm is very small and the emission maximum is at 525 nm. This means that by supplementing the specific ncAA to the cultivation media, the Prk-RS couples more amino acids to the amber tRNA. This leads to a higher expression of the whole fusion protein and indicates that the Prk-RS is a specific and efficient aaRS.
	</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 10: Emission spectrum of three biological replicates.</B> Each of cotransformants of the improved synthetase-test system (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201343">BBa_K2201343</A>) with the Prk-RS (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201201">BBa_K2201201</A>) left and the 2-NPA-RS (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201200">BBa_K2201200</A>) right. Three replicates were cultivated with their specific ncAA and three without it to compare the resulting shift in the emission spectrum.</P></DIV><ARTICLE>
In contrast, the emission spectra of our test system cotransformed with the 2-NPA-RS shows no strong shift when cultivated with or without 2-NPA (Figure 10, right). This indicates that there is not substantial more CFP-YFP fusion protein synthesized when the ncAA is supplemented to the media.  This does not mean that the synthetase does not incorporate 2-NPA at all, but the ratio of 2-NPA to native amino acids coupled to the amber tRNA cannot be described precisely. This means that the 2-NPA-RS is an inefficient aaRS.
By comparing the emission spectra of our improved test system when cotransformed with different aaRS, the efficiency of the aaRS can be determinated. Recording whole emission spestres is interesting but a bit time consuming if many aaRS should be testes for their quality. The results this far imply that a comparison of different aaRS is possible, just by comparison of the relative emission at 475 nm, representing the CFP amount, and at 525 nm, representing the fusion protein amount of the cotransformants when cultivated with and without the specific ncAA.
	</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Negative and Positive Selectivity and Ranking System for Synthetases</H2><ARTICLE>
Similar to Texas 2014, we defined values that rank the synthetases among each other. 
The negative score is the quotient of the CFP-signal to the YFP-signal when the aaRS is cultivated without the specific ncAA. Negative score = (emission at 475 m)/(emission at 525 nm). When there is no supplemented ncAA, a very specific synthetase will not or seldom couple native amino acids to the amber tRNA. This would lead to a very high amount of CFP-units compared to the whole CFP-YFP fusion protein, resulting in a strong CFP-signal and a weak YFP-signal. The higher the negative score, the higher is the specificity of the aaRS. The maximal negative score should be around 2.0, which would correspond to solely CFP expression.
The positive score is the quotient of the YFP-signal to the CFP-signal when the aaRS is cultivated with the specific ncAA. Positive score = (emission at 525)/(emission at 475). When there is the specific ncAA is supplemented to the media, an efficient synthetase should couple the ncAA to the amber tRNA. This would lead to a very high amount of whole fusion protein compared to the solely CFP-units, resulting in a strong YFP signal including a strong contribution by the FRET-signal. The higher the positive score, the higher is the efficiency of the aaRS. The maximal positive score cannot be estimated yet, but based on our tests values between four and five seem to be plausible.
An advantage of our ranking system is, that the mean of the positive and the negative score can be used to merge the two values. This enables us to assign one specific mean score to one specific synthetase and thus to arrange tested synthetases after their quality.
The resulting scores of the tested Prk-RS and 2-NPA-RS are shown in Figure 11. 
	</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 11: Scores resulting from the synthetase-test system.</B> The negative score results from the emission quotient CFP(475 nm)/YFP(525 nm) when cultivated without the specific ncAA. The positive score results from the emission quotient YFP(525 nm)/CFP(475 nm) when cultivated with the specific ncAA. The mean rank allows the combination of the negative and the positive score to compare the efficiency of synthetases among each other.</P></DIV><ARTICLE>
The Prk-RS has a medium to low negative score of 0.66±0.11, which means that without Prk supplemented to the media, native amino acids are coupled to the amber tRNA frequently. The positive score of 3.34±0.02 is high in comparison, thus it means a substantial increase in the expression of the whole fusion protein. The mean score is 2.02±0.07, which is, as far as we can assess, appropriate for further work.
The 2-NPA-RS has also a medium to low negative score of 0.73±0.07, which means it is a bit more specific than the Prk-RS. The positive score of 1.60±0.06 is very low, as it is not half as high as the positive score of the Prk-RS. This leads to a mean score of just 1.68±0.07. </ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> References </H2><B>Linlin Ma, Fan Yang, and Jie Zheng</B> (2014). Application of fluorescence resonance energy transfer in protein
studies. J Mol Struct. <B>1077</B>: 87–100.</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>