Team:Bielefeld-CeBiTec/Results/toolbox
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Toolbox Overview
Short Summary
Our aim was to provide an innovative toolbox to incorporate different non-canonical amino acids (ncAAs) during translation to expand the possibilities in protein design. We chose seven different ncAAs for five tools and demonstrated interesting applications for them. These ncAAs can be used for various approaches in basic research, medicine and manufacturing. Furthermore, with our submitted parts, every iGEM team can incorporate these seven ncAAs into their target proteins. A short description of every tool is listed below.
Analyzing
The structural analysis of a protein can be used to study distances between non-canonical amino acids (ncAAs) with Foerster Resonance Energy Transfer (FRET). This provides measuring distances between specific incorporated amino acids in the target protein. To incorporate the ncAAs we provide three different tRNA/aminoacyl-synthetases ( BBa_K2201201 , BBa_K2201202 , BBa_K2201203 ) which incorporate in response to the amber or the less used leucine codon. To demonstrate this tool we developed a prion detection assay.
Photoswitching
We designed the concept of a light influenced lycopene production as basis for the establishment of a photo switchable enzyme activity. We introduced an amber-codon in the gene for crtI in the lycopene pathway and thus stopped the lycopene production. We then showed that the lycopene production can be retained and controlled, by cotransformation with AzoF-RS ( BBa_K2201207 ) and switching the conformation of AzoF with light.
Labeling
The labeling of a protein in vivo is a useful tool that allows the investigation of a protein in its native environment. As a label for our target protein we use the fluorescent amino acid L-(7-hydroxycoumarin-4-yl)-ethylglycine (CouAA) that is incorporated by an orthogonal t-RNA/aminoacyl synthethase pair ( BBa_K2201204 ) at a defined position. We provide this tRNA/aaRS to the iGEM community and demonstrate this tool with the colocalization of the artificial cell compartment carboxysome and the ribulose 1,5-bisphosphate carboxylase oxygenase.
Photolysis
We designed an aminoacyl-tRNA-synthetase ( BBa_K2201200 ) for the incorporation of 2-nitrophenylalanine (2-NPA) based on an experiment from Liu et al. (2009) and cloned it into pSB1C3 to send it to the parts registry and pSB3T5 for further characterization. We also designed a fusion protein of GFP and streptavidin with a linker containing an amber codon and showed that only the GFP unit is expressed under normal conditions. The fusion protein is produced if cotransformed with the 2-NPA-RS and we can detect the via SDS-Page and western blot.
Fusing
We explored the possibilities of a new method for highly specific protein terminus independent fusing based on the binding of 1,2‑aminothiols and the cyano group of cyanobenzothiazole (CBT). Therefore, we designed and synthetized a new amino acid by coupling the amino group of 6‑amino‑2‑cyanobenzothiazole and the carboxyl group of the side chain of aspartic acid. The result was Nγ‑2‑cyanobenzothiazol‑6‑yl‑L‑asparagine (CBT‑Asp). As a counterpart, we used Nε‑L‑cysteinyl‑L‑lysine (CL), which contains a 1,2‑aminothiol group. This way, we created a pair of non‑canonical amino acids residues can bind covalently to each other under physiological conditions. We cloned the aaRS for CL ( BBa_K2201208 ) in pSB1C3 and used our modeling to predict 12 possible sequences of an aaRS able to incorporate the new amino acid CBT‑Asp ( BBa_K2201302 ).
References
Peters, F.B., Brock, A., Wang, J., and Schultz, P.G. (2009). Photocleavage of the Polypeptide Backbone by 2-NitrophenylalaninePeters. Chem. Biol. 16: 148–152.
