Team:Bielefeld-CeBiTec/Part Collection
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Part Collection
Short Summary
Our aim was to expand the possibilities in protein design and with our part collection we were able to realize this plan. With the parts listed below we enable every iGEM team to incorporate any non-canonical amino acid (ncAA). These ncAAs provide additional chemical functions, that canonical amino acids lack. Our toolbox contains five different tRNA/aminoacyl-synthetases (aaRS) that could incorporate different ncAAs. All these ncAAs have chemical abilities that enable to design proteins with new functions. This applications are only a proof of concept. With our selection system, every iGEM Team could evolve their own aaRS for their amino acids and expand the possibilities for protein design themselves.
Part Collection for the Incorperation of Non-canonical Amino Acids
Currently, protein design is limited to the chemical functions of the 20 to 22 canonical amino acids. Our part collection allows to add novel amino acids with other chemical abilities than those of the canonical, to expand the possibilities of advanced protein design. It contains five different tRNA/aminoacyl-synthetases (tRNA/aaRS) to incorporate non-canonical amino acids (ncAAs) during translation. These aaRS are parts of our toolbox and provide the translational incorporation of four different ncAAs as new functionality to the iGEM community. All amino acids add additional functional groups, thus functions to enable advanced protein design. The amino acids that can be incorporated with the aaRS  in our toolkit are:
2-nitrophenylalanine (2-NPA), which cleaves the backbone when irradiated by light of a certain wavelength. Therefore, it could be used for inactivation and activation of proteins.
Propargyllysine (PrK), which provides a propargyl group. Propargyl groups form a highly specific bond to azides in click chemistry reaction. Thus, this amino acid could be used for specific, terminus independent labeling.
p-acetophenylalanine (AcF), which provides a ketone group. Ketones form a highly specific covalent bond to hydroxylamines. Like PrK, this amino acid could be used for specific, terminus independent labeling.
7-hydoxy-L-coumaryl-ethylglcine (CouAA), a fluorescent amino acid, which could be used for in vivo and in vitro localization and labeling.
In addition to parts for the translational incorporation of hese ncAAs, our part collection also provides parts for a selection system to evolve novel tRNA synthetases. Moreover, instructions for the generation of a library of plasmids, that contain an aaRS encoding sequence with randmly mutated amino acids in the appropriate tRNA-binding sites. Additionally, a positive and negative selection system enables every iGEM team to expand the possibilities of protein design.  The library and the positive selection plasmid ( BBa_K2201900 in pSB3T5) are cotransformed to select sells, which survive due to a synthetase from the library that incorporates an any amino acid in response to the amber codon. Thus, all clones that survive the positive selection contain an aaRS ceapable of incorporating the target ncAA or any endogenous amino acid. To select only the clones that incorporate only the target ncAA, an additional round of negative selection is necessary.
The negative selection plasmid ( BBa_K2201901 in pSB3T5) is used to select for high specificity of enzymes. During this negative selection, the target ncAA is not supplemented in the media. The negative selection plasmid encodes the barnase, a cell toxic protein. Since the coding sequence contains two amber codons at permissive sites, the gene product is only generated if the ncAA is present or if the aaRS is unspecifically integrated a canonical amino acid.
To determine the most effective and specific aaRS for the incorporation of the desired ncAAs, an evaluation system of the synthetasesis crucial. Therefore, we optimized the synthetase test system “pFRY” initially developed by iGEM Texas 2014 leading to our new screening part BBa_K2201343 . Our screening system contains the CDS of the cyan fluorescent protein and yellow fluorescent protein connected with a linker containing an amber stop codon. This part is placed placed under control of a T7-promotor to ensure high product amounts on demand. The amount and ratio of both fluorescent proteins could indicate how efficient and specific the investigated aaRS is working. Thoroughly characterization of all parts (Table 1) can be found on the respective part pages.
Our constructed parts are listed below. For the complete characterization please refer to the linked part collection pages.
Figure 1: Plasmid map of BBa_K2201200
Plasmid map of the 2-NPA-aaRS (BBa_K2201200).
BBa_K2201200 2-NPA-aaRS in response to UAG
Evolved tyrosine tRNA/aminoacyl-synthetase for the incorporation of 2-NPA in response to the amber stop codon (CUA).
Figure 2: Plasmid map of BBa_K2201201
Plasmid map of the PrK-aaRS (BBa_K2201201).
BBa_K2201201 PrK-aaRS in response to UAG
Evolved Pyrrolysyl tRNA/aminoacyl-synthetase for the incorporation of PrK in response to the amber stop codon (UAG).
Figure 3: Plasmid map of BBa_K2201202
Plasmid map of the AcF-aaRS in response to the amber stop codon.
BBa_K2201202 AcF-aaRS in response to UAG
Evolved tyrosine tRNA/aminoacyl-synthetase for the incorporation of AcF-aaRS in response to the amber stop codon (UAG).
Figure 4: Plasmid map of BBa_K2201203 Plasmid map of AcF-aaRS in response to the less used leucine codon CUA.
BBa_K2201203 AcF-aaRS in response to CUA Evolved tyrosine tRNA/aminoacyl-synthetase in response to the less used leucine codon in E. coli (CUA).
Figure 5: Plasmid map of BBa_K2201204
Plasmid map of the CouAA-aaRS in response to the amber stop codon UAG.
BBa_K2204 CouAA-aaRS in response to UAG
Evolved tyrosine tRNA/aminoacyl-synthetase for the incorporation of the fluorescent amino acid CouAA in response to the amber stop codon (UAG).
Figure 6: Plasmid map of BBa_K2201900
Plasmid map of the positive selection plasmid.
BBa_K2201900 Positive selection plasmid for the incorporation of ncAAs
The positive selection plasmid contains a tRNA and a kanamycin resistance with two amber codons. Cotransformed with the library of tRNA/aminoacyl-synthetase with random mutated binding sites, on kanamycin containing plates only the clones survive that could charge any amino acid to the tRNA in response to the UAG codon.
Figure 7: Plasmid map of BBa_K2201901
Plasmid map of the negative selection plasmid for the selection of aaRS that specifically inocporate ncAAs.
Ba_K2201901 Negative selection plasmid against the incorporation of endogenous amino acids
The negative selection plasmid contains an tRNA with the anticodon for the amber codon and a barnase containing amber codons at permissive sites. In the negative selection, the target amino acid is not supplemented to the media. If the cotransformend clones from the positive selection charge endogenous amino acids to the tRNA, the cells die. This provides a selection method for high specific aaRS.
Figure 8: Plasmid map of BBa_K2201343
Plasmid map of the screening plasmid for the incorporation of ncAAs.
BBa_K2201343 Screening system for the incorporation of ncAA Our screening system contains the CDS of cyan fluorescent protein and yellow fluorescent protein connected with a linker containing the amber stop codon, under control of a T7-promotor. The amount and relation of the fluorescent proteins could indicate how efficient and specific the cotransformed aaRS is.
