<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_translational_system_translation_mechanism skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/translational system/translation mechanism</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/7/77/T--Bielefeld-CeBiTec--title-img-pipettes.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Translation Mechanism
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Short Summary </H2><DIV class="article">

In order to generate our <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">toolbox</A>, we used mutated variants of two different aminoacyl tRNA/synthetases complex pair. We proved that the incorporation of non-canonical amino acids through the amber codon poses as a major metabolic burden for the cells. In addition, we demonstrated that the aminoacyl tRNA/synthetases we used in our toolkit are very different in specificity, leading to inefficient incorporation of non canonical amino acids (ncAA). Therefore, we decides to generate  our own aminoacyl tRNA/synthetase variants and enabling so the most efficient selection of aminoacyl tRNA/synthetase candidates.

                </DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Comparison of Different Aminoacyl tRNA/Synthetases </H2><DIV class="article">

Generating our Toolbox, we used variants of two different aminoacyl tRNA/synthetases (aminoacyl tRNA/RS), namely <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">tyrosyl and pyrrosylyl</A>. While apllying our toolbox, we recognized that the <I>Escherichia coli</I> cells containing different aminoacyl tRNA/RS plasmids had different characteristics. We could further confirm this assumption by performing growth experiments with <I>E.coli</I> that prior to the cultivation were transformed with our different aminoacyl tRNA/synthetases. 
For this purpose, we transformed chosen aminoacyl tRNA/synthetases and two control vectors, <A href="http://parts.igem.org/Part:pSB1C3">pSB1C3</A> and <A href="http://parts.igem.org/Part:pSB3T5">pSB3T5</A>, into <I>E. coli</I> BL21 (DE3). We performed the cultivation in 1 mL LB-media with the matching non canonical amino acid (ncAA) in 12‑well microtiter plates by 600 rpm at 37 °C. We used two biological replicates, each with three technical replicates. The optical density of each cultivation was measured with NanoDrop at 600 nm. The comparison of the growth rates of each cultivation for the cells containing the aminoacyl tRNA/RS in high copy plasmid pSB1C3, as depicted in <B>Figure 1</B>, showed significant  differences between different the cultures.
Additionally, the growth experiments show that the incorporation of ncAAs through the amber codon cause a major metabolic burden for the cells, resulting in a growth rate decreased by 0.25 and 0.5 %, as compared to control cultures. Only the <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Propargyllysine (PrK)</A>‑tRNA/synthetase showed growth of up to 17 % higher than the growth of the control cultures. This can be explained by the metabolic pressure, exerted by the pSB1C3, due to coding the mRFP. In addition, the highly specific incorporation of the ncAA by the PrK (<B>Figure 3</B>) also supports faster growth of the culture, containing the PrK aminoacyl tRNA/synthetase. This is due to the similarity of the used ncAA Prk-couple to the wild type amino acid pyrrolysyl, requiring only one point mutation of the aminoacyl synthetase (aaRS). This could be demonstrated with our CFP‑YFP system(<A href="http://parts.igem.org/Part:BBa_K2201343">BBa_K2201343</A>)for the efficiency of the incorporation of ncAA. 


                </DIV><ARTICLE><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 1: Optical density of the cultivation of <I>E. coli (DE3) </I> coding five different aminoacyl tRNA/synthetases. </B> Cultivation with the matching non canonical amino acid in 12er-mikrotiter well plates at 600 rpm . The growth rates arr based on the measurements of the optical density measured at the 600 nm with the NanoDrop.</P></DIV></DIV><DIV class="half right"><DIV class="figure large"><P class="figure subtitle"><B>Figure 2: Optical density of the cultivation of <I>E. coli (DE3) </I> coding two different aminoacyl tRNA/synthetases. </B> Cultivation with the matching non canonical amino acid in 12er-mikrotiter well plates at 600 rpm . 
			The growth rates arr based on the measurements of the optical density measured at the 600 nm with the NanoDrop.</P></DIV></DIV></DIV></ARTICLE><ARTICLE>
When investigating the influence of plasmidcopy number of the transformed plasmid on the metabolic burden, the low copy plasmids turned out to be the best choice for the growth, when expressing aminoacyl tRNA/RS to incorporate ncAA. The tendency of a better growth when coded on the low copy plasmid, depicted in <B>Figure 1</B> and <B>2</B>, is caused by the different antibiotic resistance. The comparison of the low (<B>Figure 2</B>) and high copy (<B>Figure 1</B>) plasmid backbone, did not lead to the same conclusion, due to different antibiotic resistances. In context of a better while transforming the construct with a low copy plasmids (Wang <I> et. al.</I>, 2001, Wang <I> et. al. </I>, 2000), the chloramphenicol resistance appeared to be the better choice for the expression of aminoacyl tRNA/RS.

Due to the fact that the incorporation of an ncAA through the amber codon implies a major metabolic pressure for the organism, we also investigated the influence of the re-coding of the leucine codon. For this purpose, we used variants of the <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">p-Acetylphenylalanine (AcF)</A>‑tRNA/synthetase pair, adapted to the amber codon (AcF‑TAG), and to the leucine codon (AcF‑Leu).
When comparing the growth of the different variants, growth rate of the AcF-TAG variant is evidently lower. When coding the ncAA through amber codon, the low specificity of the AcF‑TAG synthetase induces an extension of the translated proteins, which results in the lower growth. In contrary, the incorporation of ncAA through the leucine codon is much more specific and therefore causes less disadvantage for the cell growth.


<DIV class="figure medium"><P class="figure subtitle"><B>Figure 3: Optical density of the cultivation of AcF‑tRNA/synthetase variants adapted to the amber codon and the leucine codon.  </B> The AcF-tRNA/synthetases are integrated in pSB1C3 and are cultivated in <I>E.coli</I> BL21 (DE3) with the matching non canonical amino acid.</P></DIV></ARTICLE></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Efficiency in Incorporation of Non Canonical Amino Acids</H2><ARTICLE>
The incorporation of a non canonical amino acid or an endogenous amino acid through an amber codon, can be tested in various ways. A safe method to prove the incorporation is the detection by the mass spectrometry, which is a very accurate, but also a complex and an expensive method. A much simpler method to detect the incorporation is the Measurement Kit, introduced by iGEM team Austen-Texas 2014 and applied by iGEM team Aachen 2016. 
The Measurement Kit is based on a reporter plasmid combined with a screening plasmid that contains the tRNA and corresponding amino acyl‑synthetase (aaRS). The screening plasmid and the reporter plasmid have to be cotransformed.
The reporter plasmid is constructed with a mRFP domain. This domain is connected through a linker sequence, containing a recoded amber stop codon, with a sfGFP domain.

Unfortunately the specificity of an orthogonal tRNA aminoacyl-synthetase pair and the amber stop codon is not optimal. Endogenous amino acids are also incorporated, so the screening system is not completely, towards detection of the ncAA. In addition to this problem, some orthogonal tRNA synthetases posess a low efficiency, thus fewer ncAA are incorporated. Although the screening system enables to compare the efficiency, a low sfGFP signal in some cases can be difficult to detect.
For this  construct, the incorporation of ncAA is detectable throughout the fluorescence of the mRFP or sfGFP. If the ncAA is incorporated within the linker sequence, a red and green fluorescence is detectable. If the ncAA is not incorporated, mRFP is expressed, but not sfGFP, so only red fluorescence is detectable.

For our toolkit, we applied six different ncAA and therefore, six different tRNA‑aminoacyl synthetases. We characterized the efficiency of each synthetase in accordancce to the incorporation efficiency of the matching ncAA.

<DIV class="figure medium"><P class="figure subtitle"><B>Figure 4:Measurement of the incorporation efficiency of non canonical amino acids.</B> Scores resulting from the <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">synthetase-test system</A>. The negative score results from the emission quotient CFP(475 nm)/YFP(525 nm) when cultivated without the specific ncAA. The positive score results from the emission quotient YFP(525 nm)/CFP(475 nm) when cultivated with the specific ncAA. The mean rank allows the combination of the negative and the positive score to compare the efficiency of synthetases among each other.</P></DIV>	
As shown in Figure 4, the efficiency of the different synthetases to incorporate their specific ncAA vary significantly, and thus not all are not completely specific. This lead to the conclusion, that the applied aaRS also incorporates endogenous amino acids. In addition to the lacking specificity, the characterization of various tRNA/aaRS requires much effort, making it inapplicable for a higher throughput.

To receive a fully specific aaRS, a selection system should be used to select the most specific amino acyl synthetase from  a library containing numerous different aaRS variants. After the selection for the specificity of the aaRS, it can be analyzed for their efficiency with our improved Measurement Kit.
With our improved system, we generated <B>27,672 different sequence variants,</B> containing <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">tyrosyl tRNA‑synthetase library</A>, based on the vector pSB1C3.

</ARTICLE></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="contentbox"><DIV class="content"><H2> References </H2><B>Wang, Magliery TJ, Liu DR, Shcultz PG.</B> (2000). A new functional suppressor tRNA/aminoacyl-tRNA synthetase pair for the in vivo incorporation of unnatural amino acids into proteins. <B>J. Am. Chem. Soc.122,5010-5011</B><B>Wankg L, Brock A, Herberich B, Schultz PG.</B>(2001). Expanding the Genetic Code of Escherichia coli. <B>sciencemag</B></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>