<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_unnatural_base_pair_biosynthesis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/unnatural base pair/biosynthesis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/b/b4/T--Bielefeld-CeBiTec--gcms.jpeg);"><DIV id="title-bg"><DIV id="title-text">
						Biosynthesis
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Short Summary </H3><DIV class="article">
The plant <I>Croton tiglium</I> is of great importance to our project due to its ability to produce isoguanosine. In order to utilize the production pathways of this novel DNA component, we aimed at rebuilding this biosynthesis pathway of <I>C. tiglium</I> in <I>E.coli</I>. Therefore, we collected samples from different tissues of this plant. First, we isolated RNA from all these plant tissues. One normalized library as well as five tissue specific libraries were constructed for RNA-Seq analysis. A trinity assembly revealed unique enzymes of the purine metabolism in <I>C. tiglium</I>. The DNA of the enzymes of interest was extracted from the library or ordered via Gene synthesis for expression in <I>E. coli</I> to purify these enzymes via <A href="https://www.neb.com/-/media/catalog/datacards-or-manuals/manuale6901.pdf ">IMPACT® Kit(New England Biolabs)</A>. Functionality of the produced enzymes was confirmed in iso-GMP and isoguanosine formation assays. For further validation of the reaction products, we applied high performance liquid chromatographic (HPLC) analysis followed by  mass spectrometry (MS) analysis. Parts encoding these enzymes were submitted to facilitate the use of these novel DNA components in the  iGEM community.
</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Insights from Trinity Assembly </H3><H4> RNA extraction </H4><DIV class="article">
Croton tiglium samples were kindly provided by the <A href="https://www.uni-marburg.de/botgart">botanic garden of the Philipps University in Marburg</A>. Total RNA was extracted from frozen tissue samples of young leaves, stem, inflorescence, seeds and roots. Mortar and pestle were used to grind the material in liquid nitrogen. <A href="http://www.sigmaaldrich.com/catalog/product/sigma/strn50?lang=de&amp;region=DE&amp;gclid=EAIaIQobChMI7ff3iqCe1wIVHsayCh3sww4YEAAYAiAAEgKfXvD_BwE">Spectrum Plant Total RNA Kit</A> was used for the RNA extracting according to the suppliers’ instructions. mRNAs were enriched based on their polyA tail via oligo-dT beads. <A href="http://www.selectscience.net/products/dropsense16---micro-volume-spectrophotometer/?prodID=171883">DropSense16 (trinean)</A> was used for quality control.
</DIV><H4> Library Preparation </H4><DIV class="article">
Enriched mRNA samples from different tissues were used for the construction of a normalized library ( <A href="http://www.vertis-biotech.com/Home">vertis biotechnology ag</A> ). In parallel, tissue specific samples were submitted to fragmentation prior to reverse transcription into cDNA via <A href="https://www.neb.com/products/m0368-protoscript-ii-reverse-transcriptase#Product%20Information">ProtoScriptII (NEB)</A> based on suppliers’ recommendations. The <A href="https://support.illumina.com/downloads/truseq_stranded_mrna_sample_preparation_guide_15031047.html">Illumina TruSeq Stranded mRNA Sample Preparation Guide</A>  was used for the generation of five tissue specific libraries with an average insert size of 400 bp. Those libraries represent young leaves, stem, inflorescence, seed, and root. 
</DIV><H4> Sequencing </H4><ARTICLE>
Sequencing of the normalized library was performed on two lanes of an <A href="https://support.illumina.com/sequencing/sequencing_instruments/hiseq_1500.html">Illumina HiSeq1500</A> generating about 47.4 million 2x250 nt paired-end reads. Sequencing of the tissue specific libraries was performed on a HiSeq1500 generating between 20 and 44 million 2x75 nt paired-end reads per tissue-specific library (Table 1).
<P class="figure subtitle"><B>Table 1: Sequenced tissue-specfic libraries </B></P><TABLE style="margin: auto"><THEAD><TR><TH style="width: 100%;" class="header">Tissue</TH><TH style="width: 100%;" class="header">Number of sequenced fragments</TH><TH style="width: 100%;" class="header">Library ID</TH></TR></THEAD><TBODY><TR><TD>Young leaves</TD><TD>24,704,122</TD><TD>strRNA_MHA1</TD></TR></TBODY><TBODY><TR><TD>Stem </TD><TD>20,032,1422</TD><TD>strRNA_MHA2</TD></TR></TBODY><TBODY><TR><TD>Inflorescence</TD><TD>22,804,752</TD><TD>strRNA_MHA3</TD></TR></TBODY><TBODY><TR><TD>Seed</TD><TD>43,447,889</TD><TD>strRNA_MHA4</TD></TR></TBODY><TBODY><TR><TD>Root</TD><TD>28,007,089</TD><TD>strRNA_MHA5</TD></TR></TBODY></TABLE></ARTICLE><H4> Data Processing </H4><ARTICLE><A href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">FastQC</A> was applied to check the quality of all sequencing data. Low quality regions and adapter fragments were removed from the reads via trimmomatic 0.36 {Bolger, 2014}. Removal of adapters was performed based on all known illumina adapter sequences with the options 2:30:10. A sliding window of the length 4 was used to clip reads once the average PHRED score dropped below 15. Reads below the length cutoff of 100 nt were discarded.  Pairs with only one surviving read were dropped after trimming. About 96 % of all paired-end reads survived this process.
</ARTICLE><H4> Transcriptome assembly </H4><ARTICLE>
Trinity v2.4.0 {Haas, 2014} was applied with default parameters for the de novo transcriptome assembly based on all 2x250 nt paired-end reads of the normalized library. The initial assembly was followed by the quality assessment and processing steps as recommended by {Haas, 2014}. Assembly completeness was investigated by computing assemblies for subsets of the data as well as through remapping of the reads to the assembly.
</ARTICLE><H4> Expression quantification </H4><DIV class="article">
Reads from tissue specific data sets were mapped to the initial transcriptome assembly via STAR {Dobin, 2013} with 90 % length and 95 % identity. A customized python script was developed to generate a matching annotation file for the assembly. featureCounts {Liao, 2014} was applied to quantify the expression of all sequences in the assembly. Since most transcripts are represented by multiple contigs representing different splice variants, we decided to include multi-mapped reads. Dedicated python scripts were used for further investigation of the expression data as well as for the generation of expression heatmaps via the <A href="http://seaborn.pydata.org/">seaborn module</A>. VENN diagram generation was performed at <A href="http://bioinformatics.psb.ugent.be/webtools/Venn/">Bioinformatics.psb</A>.
</DIV><H4> Results </H4><DIV class="article">
In total, 45.5 million 2x250 nt paired-end reads were assembled into the 431.8 Mbp transcriptome comprising 388,181 contigs. The high continuity of the assembled contigs can be described by the E90N50 of 2,246 bp and the E90N90 of 452 bp. The completeness check indicated a sufficient amount of sequencing data were generated.
<A href="http://busco.ezlab.org/">BUSCO analysis</A> revealed the presence of 94 % complete BUSCOs. In addition, 3 % are present in fragmented form and only 3 % are missing in this transcriptome assembly.
</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Identification of Candidate Genes </H3><DIV class="article">
Known sequences for specific enzymes from related species were subjected to a <A href=" http://www.uniprot.org/ ">uniprot</A>, we used <A href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</A> search against the transcriptome assembly. We identified several putative pathways for the isoG biosynthesis based on existing databases like <A href=" http://www.uniprot.org/ ">uniprot</A>, we used <A href=" http://www.genome.jp/kegg/">KEGG</A>.
 

Firstly, there is the guanosine monophosphate synthase (GMPS), an enzyme from the class of ligases that form carbon-nitrogen-bonds with glutamine as an amido-N-donor acceptors (see <A href=" http://www.genome.jp/dbget-bin/www_bget?ec:6.3.5.2">KEGG</A> for more information). It is also known as ‘Guanosine monophosphate synthetase’. GMPS is needed for the amination of XMP (xanthosine monophosphate) to create GMP and possibly iso-GMP in the case of <I>C. tiglium</I>. Besides, GMPS can be found in many organisms apart from <I>Croton tiglium</I>, including Homo sapiens and <I>E. coli</I>. The transcriptome assembly contained two sequences that displayed a strong similarity to known GPMS encoding sequences. These sequences encode peptides of 314 amino acids and molecular mass of approximately 59.46 kDa. GMPS is a promising candidate, since it may not only be able to catalyze the reaction of XMP to GMP but also to iso-GMP. 


Another interesting enzyme from the purine metabolism is the Inosine monophosphate-dehydrogenase (IMPDH) that matched three sequences in the transcriptome assembly. IMPDH is an enzyme from the class of the oxydoreductases, which are acting on CH-OH groups of donors with NAD+ or NADP+ as acceptors <A href="http://www.genome.jp/dbget-bin/www_bget?ec:1.1.1.205">(see KEGG)</A>. The different forms of IMPDH encoded by sequences in the transcriptome assembly have a molecular mass of 53-58 kDa and amino acid lengths between 500 to 550. In the purine metabolism, IMPDH is the catalyst of the synthesis of XMP out of inosine monophosphate (IMP). Therefore,  it could enable the biosynthesis of an isoform of XMP that might then even be a substrate for the production of iso-GMP. 


Furthermore, the cytidine deaminase (CDA) seemed to be of immense potential. The CDA, which belongs to the class of hydrolases acting on carbon-nitrogen bonds different from peptide bonds <A href=" http://www.genome.jp/dbget-bin/www_bget?ec:3.5.4.5">(see KEGG)</A> is usually applied to deaminate cytidine to uridine. However, there is also the possibility of the reverse reaction catalyzed by CDA. A reaction from xanthosine to iso-GMP might be possible. The best matching sequence for CDA in the transcriptome assembly encodes 535 amino acids . The putative gene product has a molecular mass of 33.95 kDa. 

Aside from these enzymes, the adenylosuccinate synthetase (ADSS) could be an interesting candidate. The ADSS belongs to the class of ligases, which are forming carbon-nitrogen bonds  <A href=" http://www.genome.jp/dbget-bin/www_bget?ec:6.3.4.4">(see KEGG)</A>. Only one matching sequence was identified in  the transcriptome assembly. The encoded gene product has a molecular weight of 53.32 kDa and a size of 489 amino acids. In <I>C. tiglium</I>, it is expected to catalyze the reaction of IMP to adenylosuccinate that will then be further processed into AMP. 

Finally, we identified the enzyme xanthine dehydrogenase(XDH) as promising candidate. The XDH converts xanthine into urate that will be further processed afterwards. XDH is an enzyme from the class of oxidoreductases that is acting on CH or CH2 groups with NAD+ or NADH+ as an acceptor <A href=" http://www.genome.jp/dbget-bin/www_bget?ec:1.17.1.4">(see KEGG)</A>, and could even be matched with six sequences of the trinity assembly. The encoded gene products are expected to have  a molecular mass of 64.12 kDa and a size of 587 amino acids. 

</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Extraction of Enzyme DNA out of the cDNA Library </H3><ARTICLE>
After we had identified all interesting sequences from the purine pathway, we had to extract them from the cDNA of the tissue samples. Thus, we designed primers for all candidate sequences that were identified in the transcriptome assembly of <I>C. tiglium</I>. In total, we had 13 pairs of primers that were used in separate PCRs with cDNA from all different tissue samples. We could extract at least one possible cds for each protein of interest from these PCRs. Unfortunately, we were only able to amplify one  transcript isoforms of  the GMPS from the transcriptome assembly. Therefore, we  ordered a gene synthesis of the other transcript isoform. In parallel to the cloning of native sequences, we ordered codon-optimized coding sequences for all candidate genes.  BioBricks encoding the most promising enzymes were submitted ( See parts <A href=" http://parts.igem.org/Part:BBa_K2201060"> BBa_K220160</A> , <A href=" http://parts.igem.org/Part:BBa_K2201061">BBa_K220161</A>) and CDA( Part <A href=" http://parts.igem.org/Part:BBa_K2201062">BBa_K220162</A>).

</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Protein purification </H3><DIV class="article">
A <A href="https://static.igem.org/mediawiki/2017/0/0e/T--Bielefeld-CeBiTec--ImpactProtocol.pdf">modification</A> of the <A href="https://www.neb.com/-/media/catalog/datacards-or-manuals/manuale6901.pdf ">NEB Impact® Kit</A> was used for the purification of heterologous expression in <I>E. coli</I>. The protein purification via the Impact system works with the usage of an intein tag. Impact is short for “Intein Mediated Purification with Affinity Chitin-binding Tag”. In short, the coding sequence of the candidate gene is linked with an intein tag that enables the protein’s purification. As a vector, we used pTXB1 that is responsible for a c-terminal fusion of the intein tag. The expression strain ER2566 with the plasmid pRARE-2 was used. This vector is used to compensate for a bad codon usage as it encodes some rare tRNAs. 
</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Estimation of the Protein Concentration </H3><DIV class="article">
Concentrations of purified proteins were estimated based on a <A href="https://static.igem.org/mediawiki/2017/6/6f/T--Bielefeld-CeBiTec--RotiNanoquant.pdf">modification</A> of the Bradford Assay (Bradford, M., 1976) <A href="https://www.carlroth.com/downloads/ba/de/K/BA_K880_DE.pdf ">Roti®-Nanoquant</A> by Carl Roth. Depending on the protein, we reached concentrations from 1.35 up to 5.31 grams per liter (Table 2). 
<P class="figure subtitle"><B>Table 2: Concentrations of the proteins, estimated with Roti® Nanoquant. </B> Replicants were created of the most important enzymes. </P><TABLE style="margin: auto"><THEAD><TR><TH style="width: 100%;" class="header">Protein name</TH><TH style="width: 100%;" class="header">Concentration in mg/mL</TH></TR></THEAD><TBODY><TR><TD>GMPS iso-form 1</TD><TD>4.1775 2.15071.4610</TD></TR></TBODY><TBODY><TR><TD>GMPS iso-form 2 </TD><TD>1.3497</TD></TR></TBODY><TBODY><TR><TD>IMPDH form 1</TD><TD>4.4007</TD></TR></TBODY><TBODY><TR><TD>IMPDH form 2</TD><TD>4.1763</TD></TR></TBODY><TBODY><TR><TD>ADSS</TD><TD>4.2616</TD></TR></TBODY><TBODY><TR><TD>XDH</TD><TD>4.13021.9349</TD></TR></TBODY><TBODY><TR><TD>CDA</TD><TD>5,30921,80541,3881</TD></TR></TBODY></TABLE>


Afterwards, we confirmed protein identity via SDS-PAGE and MALDI-TOF-MS. 

</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Investigation of enzyme activity </H3><DIV class="article">
After we knew that all proteins had been extracted properly, our next step was to test their functionality. Therefore, we used the plate-reader “Tecan infinite® 200” and the program “Tecan i-control, 1.10.4.0”. For all enzyme reactions, we used room temperature to meet the physiological conditions of these plant enzymes. Values were plotted to show the absorption of the main substrate before and after the addition of the enzyme or water, respectively (Figures 2-3 as well as 5-6, also see Final Discussion). Also all of them showed only strong activity within the first hour, we performed over-night enzyme activity assays to reach the final end point. To verify the reaction product, we used the HPLC (high performance liquid chromatography) <A href="https://at.vwr.com/store/product/10032120/hplc-system-chromaster">”LaChrom Ultra”</A> in combination with the <A href="https://www.bruker.com/de/products/mass-spectrometry-and-separations/lc-ms/o-tof/microtof-focus-ii/overview.html">MicroToFQ mass spectrometer</A>. The combination of these separation systems allowed us to separate the substances of the reaction mixtures, analyze their molecular weight and compare them with standards. For our purposes, we used parameters for the MicroTofQ like in (Ruwe et al., 2017) with a measurement in negative mode were the masses would be measured subtracting the mass of an H atom. However, since we wanted to differentiate between different forms of substances with the same mass, we had to try additional measurement methods for the HPLC. Eventually, we used the “Zip-pHILIC” column with a length of 150 mm and a diameter of 2.1 mm from Merck. For the mobile phase, we used ammoniumbicarbonat (pH 9.3) and acetonitril in a ratio of 27 % to 73 %. This was used in isocratic mode with a flow-through of 0.2ml/min. The injection volume was set to 2 µL of the reaction mixture from the corresponding enzyme assay. The separations took place at 40 °C. Since our main goal was to produce iso-GMP or iso-Guanosine using the purified enzymes of <I>Croton tiglium</I>, we focused on the main promising candidate enzymes: both iso-forms of GMPS (<A href=" http://parts.igem.org/Part:BBa_K2201060"> BBa_K220160</A> and <A href=" http://parts.igem.org/Part:BBa_K2201061">BBa_K220161</A>) and CDA( Part <A href=" http://parts.igem.org/Part:BBa_K2201062">BBa_K220162</A>).
</DIV><H4> CDA </H4><DIV class="article">
First, we set up an enzyme activity assay for CDA with cytidine to ensure its activity following the protocol by Robert M. Cohen and Richard Wolfenden from 1971 that stated that the disappearance of cytidine can be measured in relation to the decrease of absorption at 282  nm. Therefore, we set up the following reaction mixture containing 50 mM TRIS-HCl buffer (pH 7.5) and 0.167 mM cytidine as a substrate. 
We used six replicates with 196 µL of the mixture and measured it for about 20 min (measurement all 30 sec) with the Tecan infinite® 200. We then paused the measurement program to add 4 µL (6 µg) of the previously extracted CDA or 4 µL of water to three samples each. Then, we immediately continued the measurement for about an hour. 
As it can be seen in Figure 2, the absorption of cytidine at 282 nm began to continuously decrease after the addition of the cytidine deaminase, whereas the absorption remained more or less constant when only water was added. With these results, the activity of our extracted cytidine deaminase could be proven. 
</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure (2): Enzyme activity assay for the reaction of the cytidine deaminase with cytidine.</B>The reaction took place at room temperatue. Three biological replicates were used each. After the addition of water, the absorbance at 282 nm stayed the same whereas it decreased after the addition of the CDA. </P></DIV><DIV class="article">
After confirming general activity of CDA, we set up a possible reaction with xanthosine instead of cytidine, all other components being the same. However, since there was no real literature on this reaction, we first had to figure out the absorption rate at which xanthosine can be measured. This was done using a general spectrum analysis of different mixtures, three samples each:  </DIV><OL><LI>without xanthosine, without CDA</LI><LI>with xanthosine, without CDA</LI><LI>without xanthosine , with CDA</LI><LI>with xanthosine, with CDA</LI></OL><ARTICLE>
These mixtures were then compared to estimate the absorption rate for xanthosine.
</ARTICLE><OL type="A"><LI>1+3: difference between a reaction mixture with and without CDA</LI><LI>1+2: difference between a reaction mixture with and without xanthosine</LI><LI>2+4: difference between no reaction and a possible reaction</LI></OL><ARTICLE>
We hereby could figure out the absorption rate at which xanthosine can be measured (B) as well as ensure that the peak was independent from the CDA (A). Further on, we could identify the absorbance of CDA at about 254-260 nm (A and C). (Figure 1))
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure (1): Results of the analysis of the absorbance of xanthosine at different nanometers. </B> AAll measurements made with the Tecan infinite® 200 at room temperature. The difference between a mixture with and without xanthosine (red) can clearly be made up at about 282 nm.</P></DIV><DIV class="article">
Afterwards we set up new activity assays, using 196 µL of the reaction mixture in six of the well plate’s holes. After measuring the absorbance at 282 nm, we added 4 µL of either water or the enzyme (6 µg) to three biological replicates each, continuing the (previous) measurements for about an hour. 


The reaction of the cytidine deaminase with xanthosine showed diverse results (Figure 3). Here, also a slight decrease of the xanthosine concentration could be seen, which, however, was not significant. 
<DIV class="figure medium"><P class="figure subtitle"><B>Figure (3): Enzyme activity assay for the reaction of the cytidine deaminase with xanthosine as a substrate.</B>The reaction was set up at room temperature, using three biological replicates each. After adding CDA to the reaction mixture, a slight decrease in the absorbance at 282 nm was visible. However, as there is also a very small decrease for the addition of water, no significant difference was observed.</P></DIV><DIV class="article">		
The HPLC-MicroTofQ Measurements could only make up the xanthosine and various other substances. However, there were no significant masses and peaks for guanosine or iso-guanosine. (Figure 4) 
</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure (4): HPLC-MicroTofQ measurement for the products of the reaction of CDA with xanthosine. </B>Measurement at 40 °C. Even if many different masses could be detected, none of these could be matched to guanosine or iso-guanosine. For these, a peak should be at about 282 g/mol.</P></DIV><DIV class="article">
So, with only a slight decrease of the absorbance and no detectable products in the HPLC, it seems reliable that there is only a very small amount of xanthosine converted to isoguanosine, since the reaction is not specific to the CDA and thus rare. However, supplementary tests and experiments with different reaction mixtures would be needed to further analyze it. 
</DIV><H4> GMPS </H4><DIV class="article">
We set up the reaction mixture of the two isoforms of the GMPS following a protocol for the enzyme activity assay by Abbott, J., Newell, J., Lightcap, C. et al.(2006). We also regarded the original paper from 1985 that stated the absorbance at 290 nm for the given amount of XMP within the mixture. For that, we set up the following reaction mixture: 
</DIV><UL><LI>60 mM HEPES</LI><LI>5mM ATP</LI><LI>0.2mM XMP</LI><LI>20mM MgCL2</LI><LI>200mM NH4CL</LI><LI>0.1mM DTT</LI><LI>0.8mM EDTA</LI><LI>Filled up with ddH<SUB>2</SUB>O</LI></UL><DIV class="article">
Due to their instability, XMP and ATP were always added freshly. After the samples were set up, we measured them with the Tecan infinite® 200 reader for about 20 minutes at an absorbance of 290 nm. Afterwards, 4 µL of either water or 4 µL (6 µg) of the isoforms of the GMPS (isoform1: <A href=" http://parts.igem.org/Part:BBa_K2201060"> BBa_K220160</A> and isoform 2: <A href=" http://parts.igem.org/Part:BBa_K2201061">BBa_K220161</A>) were each added to three samples. The measurement was continued for approximately an hour. The activity assays of isoforms 1 and 2 both proved that the GMPS enzymes are working correctly, reducing the amount of XMP in the reaction mixture significantly. Therefore, the absorption at 290 nm decreased a lot after adding the enzyme to the solution of isoform 1 of GMPS, whereas the initial decrease was weaker for the codon-optimized isoform 2. However, both decreased the amount of XMP about the same within the hour in which their reaction was measured. Thus, it can be said that both, isoform 1 and isoform 2 are working as expected (See Figure 5 and Figure 6 for comparison)
</DIV><DIV class="contentline"><DIV class="half left"><DIV class="figure hundred"><P class="figure subtitle"><B>Figure (5): Enzyme activity assay of iso-form 2 of the guanosine monophosphate synthetases.</B>The reaction was set up at room temperature using three biological replicates. A significant decrease in the absorption at 290 nm can be made up after the addition of the synthetized GMPS whereas the negative control with water stays at the same absorption. </P></DIV></DIV><DIV class="half right"><DIV class="figure hundred"><P class="figure subtitle"><B>Figure (6):Enzyme activity assay of iso-form1 of the guanosine monophosphate synthetases.</B>Three biological replicates were used. The reaction was set up at room temperature. A significant decrease in the absorption at 290 nm can be made up after the addition of the GMPS whereas the negative control with water stays at the same absorption.  </P></DIV></DIV></DIV><DIV class="article">
As described earlier, it took some time to figure out the right requirements for the HPLC-MicroTofQ measurements, since iso-GMP and GMP have the exact same mass and are thus only separable by their structure. However, with the method chosen in the end, it was possible to identify analytes that seem to represent iso-GMP. Therefore, at first, the general substances within the reaction mix had to be figured out to ensure that only those representing GMP/iso-GMP will be included in the analyses. The general analysis of all substances included showed significant values for all the interesting substrates and products that should be within the reaction mix, including AMP, ADP and ATP, some remaining traces of XMP and of course GMP/iso-GMP (Figures 7 and 8). 
</DIV><DIV class="contentline"><DIV class="half left"><DIV class="figure hundred"><P class="figure subtitle"><B>Figure (7): HPLC-MicroTofQ measurement for the substances within the reaction mixture of the fully extracted GMPS.  </B>Reaction conditions as described earlier. Next to the substrates, ATP and XMP, also resulting substances like AMP and GMP can be found.</P></DIV></DIV><DIV class="half right"><DIV class="figure hundred"><P class="figure subtitle"><B>Figure (8): HPLC-MicroTofQ measurement for the substances within the reaction mxture of the GMPS with the synthetized sequence.  </B>Measurement at 40 °C. Next to the substrates, ATP and XMP, also resulting substances like AMP and GMP can be found.
</P></DIV></DIV><DIV class="article">
We then compared the resulting form of GMP with a GMP-standard (10^-5 diluted solution) and the exact measurements of the HPLC. For both, isoform 2 and isoform 1 of GMPS the peaks of the substance’s flow-through found at the molecular mass of GMP and iso-GMP (approximately 363.22 g/mol, in the graph at approximately 362 g/mol because of the missing H due to the measurement method) were significantly shifted to the right compared to the standard. Thus, the form of GMP that is created with the enzyme reactions of the two isoforms of GMPS and the gene synthesis has to be another form of GMP, most likely iso-GMP. (Figure 9) 
</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure (9): HPLC-MicroTofQ measurement comparing the GMP standard and the reaction products’ flow-through. </B> In red the product of isoform 2 of GMPS. In blue, the one found for isoform 1 of GMPS, in green the standard. Even though the standard as well as the mixtures contained compounds that have the same molecular mass, they show different behaviors on the HPLC. The ordinary GMP was significantly faster than the one generated in the enzyme reactions. Thus, the form of GMP that results from the reactions is likely to be iso-GMP.  </P></DIV><ARTICLE>
In conclusion, we did not only figure out the synthesis pathways in <I>Croton tiglium</I> but could even recreate a part of it, showing that the enzymes expressed in <I>Croton tiglium</I> are more likely to generate a different form of GMP (presumably iso-GMP).
</ARTICLE></DIV></DIV><DIV class="contentbox"><H2> References </H2><ARTICLE><B>Abbott, J., Newell, J., Lightcap, C. <I>et al.</I></B>(2006) The Effects of Removing the GAT Domain from E.coli GMP Synthetase. <B>25</B> 384<B>Cohen, R.M., Wolfenden, R.</B> (1971). Cytidine Deaminase from <I>E.coli</I> – Purificarion Properties and Inhibiton by the potential transition state analog 3,4,5,6-tetrahydrouridine. The Journal of Biological Chemistry. <B>25</B><B>Zalkin, H </B>(1971), GMP sythesis
</ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>