<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_toolbox_analysing skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/toolbox/analysing</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/0/0c/T--Bielefeld-CeBiTec--title-img-analyzing.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Analyzing
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Short Summary</H2><ARTICLE>
Our analysis tool gives the opportunity to measure intermolecular distances with the help of Foerster resonance energy transfer <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing#FRET">(FRET)</A> in the target protein. We provide three tRNA/aminoacyl-synthetases (aaRS) for the incorporation of non-canonical amino acids (ncAAs) with additional functional groups compared to the canonical amino acids. The functional groups could form a covalent bond to a fluorophore in chemical reactions. The fluorescence of the fluorophores indicates how far the distance to each other is. With this tool, target proteins could be labeled at specific positions to find out if the folding of the protein changes, give information about the protein structure or if two different proteins interact. 
To demonstrate this tool, we labeled the yeast prion <A target="blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing#Sup35">Sup35</A> at specific positions. During translation of Sup35, two different orthogonal tRNA/aaRS incorporate two different ncAAs which can be labeled specific with fluorophores. With this test protein, applications like a quick prion detection assay are possible. 
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Evolved Synthetases for the Incorporation of Propargyllysine and p-Acetophenylalanine</H2><ARTICLE>
For our toolkit, we decided to use the non-canonical amino acids p-acetophenylalanine (AcF) with a ketone group and propargyllysine (PrK) with a propargyl group. Propargyl groups can be used to form a covalent bond to azide groups in a click-chemistry reaction and ketone groups could form a covalent bond to hydrazide groups. Our aim was to provide two orthogonal tRNA/aminoacyl-synthetases (aaRS) which could incorporate these amino acids through the amber codon. Furthermore, we want to provide an aaRS which incorporates AcF through the less used leucine codon (CTA). This way, both amino acids could be incorporated simultaneously and at a specific position during translation. 
We received plasmids from the <A target="blank" href="https://www.embl.de/research/units/scb/lemke/">Lemke group</A> from <A target="blank" href="https://www.embl.de/index.php">EMBL</A> in Heidelberg containing an evolved tyrosyl tRNA/ synthtase pair (tRNA/TyrRS) from <I>Methanococcus jannaschii</I> for the incorporation of AcF and an evolved pyrrolysyl synthetase from <I>Methanosarcina mazei </I> for the incorporation of PrK, both in response to the amber codon. We used Gibson Assembly to clone the tRNA/aaRS and the tRNA from these plasmids into pSB1C3 and replaced cutting sites for <I>Eco</I>RI and <I>Spe</I>I with site directed mutagenesis to provide these synthetases for the iGEM community. Furthermore, we changed the anticodon in the tRNA of the TyrRS tRNA to the anticodon for the less used leucine codon, so the new aaRS incorporates AcF in response to the codon CTA. An alignment of both evolved synthetases with the wildtypes is shown in Figure 1 (PrK-aaRS) and Figure 3(AcF-aaRS).
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 1:</B>Alignment of the amino acid sequence of PrK-aaRS and the wildtype <I>Methanosarzia mazei</I> Pyl-RS.</P></DIV><ARTICLE>
The alignment of the wildtype PylRS from <I>Methanosarzia mazei</I> and the evolved synthetase for the incorporations of PrK shows only one amino acid exchange at position 124. Although this is the aaRS with the least amino acid exchanges of our toolkit, it turned out to be the most specific. We proved the incorporation of PrK through this synthetase with our screening system for the incorporation of ncAAs. The results from our screening system, which compares the incorporation efficiency and specificity, are shown in Figure 2. For more details of our screening method, please refer to the <A target="blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">part improvement page</A>.
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 2:</B> Comparison of the incorporation rate of PrK and native amino acids through the evolved PrK-aaRS. The negative score results from the emission quotient CFP(475 nm)/YFP(525 nm) when cultivated without the specific ncAA. The positive score results from the emission quotient YFP(525 nm)/CFP(475 nm) when cultivated with the specific ncAA. The mean rank allows the combination of the negative and the positive score to compare the efficiency of synthetases among each other.</P></DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 3:</B> Alignment of the amino acid sequence of the evolved AcF-aaRS and the wildtype TyrRS from <I>methancoccus jannaschii</I>.</P></DIV><ARTICLE>
Due to recommendation of our expert <A target="blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated#Aramburu">Iker Valle Aramburu</A>, we decided to use only one ncAA to label our test protein and label the second fluorophore through maleimide labeling to a cysteine. This labeling strategy could only be used for proteins that are cysteine-free or in which all cysteines could be replaced. Because this is only possible in a few applications we decided to provide a second evolved aaRS for the incorporation of AcF in response to the rarely used leucine codon CUA. The sequence of the synthetase is similar to the aaRS described by Kim 2012 to incorporate AcF in response to CUA . 
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Construction of the Expression Plasmid for the Sup35 Test Protein</H2><ARTICLE>
For our test protein, we decided to order a gene synthesis of the NMregion of <A target="blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing#Sup35">Sup35</A>. The NM region is 250 amino acids long and responsible for the prion forming function of Sup35. The test protein contains an amber codon at position 21 and one cysteine codon at position 121. 
According to our expert Iker Valle Aramburu, the incorporation of two noncanonical amino acids lowers the yield. He recommended us to use only one non-canonical amino acid and one cysteine, which could be labeled in a maleimide coupling reaction.  Mukhopadayay <I>et al. </I>2006 showed that mutants of Sup35 containing no cysteines are still able to form prions. So, we decided to order a gene synthesis of sup35 containing no cysteines. 
For our experiments we cloned the gene synthesis of the NM region of Sup35 into pSB1C3 and used site-directed mutagenizes to construct the part <A target="blank" href="http://parts.igem.org/Part:BBa_K2201231">BBa_K2201231</A> containing a His<SUB>6</SUB>tag, one amber codon at position 21 and a cysteine codon at position 121. The control <A target="blank" href="http://parts.igem.org/Part:BBa_K2201232">BBa_K2201232</A> contains cysteine codons at positions 21 and 121. In addition, we constructed these three parts also with a T7-promoter and RBS (<A target="blank" href="http://parts.igem.org/Part:BBa_B0034">B0034</A>) (<A target="blank" href="http://parts.igem.org/Part:BBa_K2201331">BBa_K2201331</A> and <A target="blank" href="http://parts.igem.org/Part:BBa_K2201332">BBa_K2201332</A>), for the inducible expression of the Sup35 variants.
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Expression and Analysis of the Sup35 Test Protein</H2><DIV class="article">
The expression plasmid of Sup35 containing one amber stop codon at position 21 <A target="blank" href="http://parts.igem.org/Part:BBa_K2201331">(BBa_K2201331)</A> was co-transformed with the evolved PylRS synthetase for the incorporation of PrK <A target="blank" href="http://parts.igem.org/Part:BBa_K2201201">(BBa_K2201201)</A>.  The cells containing both plasmids were cultivated like described in expression of recombinant proteins, without PrK in the medium and with 1 mM PrK in the medium. Sup35 was purified through <A target="blank" href="http://www.mn-net.com/Products/Proteinpurification/PurificationofHistagproteins/ProtinoNiNTAAgarose/tabid/11434/language/en-US/Default.aspx">Ni-NTA chromatography</A>. As positive control Sup35 without stop codons was expressed and purified the same way. All three samples were analyzed by SDS-PAGE shown in Figure 4.
</DIV><DIV class="figure medium"><B>Figure 4: SDS-PAGE of the eluates of the three cultivations to check the incorporation of the ncAA PrK.</B> The positive control (PC: <A target="blank" href="http://parts.igem.org/Part:BBa_K2201332">BBa_K2201332</A>) left and <A target="blank" href="http://parts.igem.org/Part:BBa_K2201231">BBa_K2201331</A> without PrK (-PrK) in the media in the middle and with 1 mM PrK (+PrK) in the media on the right side. Information to the marker can be found <A target="blank" href="https://www.thermofisher.com/order/catalog/product/26616">here</A>.</DIV><DIV class="article">
The SDS-PAGE confirmed the results from our screening system. The positive control showed a high expression of the recombinant Sup35 and both samples with <A target="blank" href="http://parts.igem.org/Part:BBa_K2201331">BBa_K2201331</A> showed a band on the same height (~28 kDa) as the positive control. This proved that the PrK-aaRS incorporates endogenous amino acids if PrK is not supplemented in the media. But if PrK is present, the expression of Sup35 seems to be increasedr. The affinity of PrK-aaRS to PrK seems to be stronger than to the endogenous amino acids, so the test protein is expressed with the non-canonical amino acid as planned.
</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Labeling of the Sup35 Test Protein</H2><DIV class="article">
The labeling of the test protein was performed in a two-step reaction. The first step was the click chemistry reaction to label the PrK with Cyanin3-azide and the second step was the maleimide labeling of the cysteine with Cy5 maleimide. The click chemistry reaction of 50 nmol test protein in 100 µL reaction volume was performed in 1x PBS buffer containing 200 µM copper sulfate and 200µM TCEP to prevent aggregation of Sup35. To start the reaction 100 nmol of Cy3-azide and 200 µM freshly prepared sodium ascorbate were added and after that, the reaction was incubated for 2 h at room temperature. After the incubation, the Cy3-labeled test protein was washed with maleimide labeling buffer (2x PBS and 8 M guanidine hydrochlorid) and concentrated down to 50 µL by <A target="blank" href="http://www.sigmaaldrich.com/catalog/product/aldrich/z677108?lang=de&amp;region=DE">centrifugal filters</A>. To start the maleimide labeling reaction, 100 nmol of Cy5-aleimide were added to the test protein and incubated for 2 h at room temperature. After this step the excess dye should be washed away by <A target="blank" href="http://www.sigmaaldrich.com/catalog/product/aldrich/z677108?lang=de&amp;region=DE">centrifugal filters</A> or dialyses and the labeling was confirmed by a fluorescence scan of a SDS-PAGE containing the labeled sample and a negative control. The SDS-PAGE of our test protein before and after the labeling reaction is shown in Figure 5.
Furthermore, we wanted to detect the labeling efficiency through UV-VIS spectralphotometer, but the results indicate that a high excess of free Cy5 remains in the sample and the background was too high to perform FRET-measurements. To investigate the labeling efficiency and to get a fluorescence signal the labeling reaction and especially the last washing step needs to be improved to generate suitable test protein for the prion detection assay.
</DIV><DIV class="figure large"><B>Figure 5:</B>SDS-PAGE and fluorescence scan for the detection of Cyanin3 (Cy3) and Cyanin5 (Cy5) of the SDS-PAGE of the unlabeled and labeled Sup35. On the left site the SDS-PAGE was stained with Coomassie after the fluorescence scan.</DIV><ARTICLE>
The fluorescence scan proves the successful labeling of Sup35 with Cy3 and Cy5. However, in the SDS page a second higher band appears, that seemed to be labeled too. One explanation would be that Sup35 forms aggregates in the labeling reaction. If the test protein is changed to its prionic form it would not be suitable for a prion detection assay. To investigate if Sup35 forms aggregates after the labeling reaction we wanted to check, if there are any aggregates through atomic force microscopy (AFM). If there are any prion forming ability, the proteins should form big aggregates like reported by <A target="blank" href="https://2015.igem.org/Team:Kent">iGEM Kent 2015</A>, who wanted to form nanowires out of Sup35. The AFM images were generated in the Experimental Biophysics &amp; Applied Nanosciences research group from Bielefeld university, Department of Physics with a MultiMode® AFM (Bruker) in tapping mode. The images from the unlabeled and labeled Sup35 are shown in Figure 6.
</ARTICLE><DIV class="figure large"><B>Figure 6: </B>Atomic force microscopy (AFM) images of the unlabeled Sup35 (left) and the labeled Sup35 (right). The AFM images were generated in the Experimental Biophysics &amp; Applied Nanosciences research group from Bielefeld university, Department of Physics with an MultiMode® AFM (Bruker) in tapping mode.</DIV><ARTICLE>
The AFM pictures of Sup35 (Figure 6) show no form of aggregated or denaturized proteins, thus our labeling tool is suitable for the specific labeling of protein without changing their conformation. For the further development of the assay, the labeling and washing method needs to be improved regarding the remaining free dye in the samples after the labeling reactions to get an evaluable FRET signal.
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> References </H2><B>Kim, J., Seo, M., Lee, S., Cho, K., Yang, A., Woo, K., Kim, H., Park, H.</B>(2012). Simple and Efficient Strategy for Site-Specific Dual Labeling of Proteins for Single-Molecule Fluorescence Resonance Energy Transfer Analysis. Analytical Chemistry.<B>85</B>: 1468-1474.
<B>Mukhopadhyay, S., Krishnan, R., Lembke, E. A., Lindquist, S., Deniz, A. A.</B>(2007)A natively unfolded yeast prion monomer adopts
an ensemble of collapsed and rapidly
fluctuating structures.PNAS.<B>104(8)</B>:2649-2654.</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>