Team:Bielefeld-CeBiTec/Achievements
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Achievements
We established two orthogonal methods for the detection of unnatural base pairs in a target sequence: an Oxford Nanopore sequencing application and an enzyme based detection method
Development of a software suite for these orthogonal methods
Integration and characterization of  the nucleotide transporter PtNTT2 from P.tricornutum in E.coli for the uptake of  unnatural nucleoside triphosphates
Confirmation that certain Taq DNA polymerases can efficiently incorporate unnatural nucleotides
Construction of a toolbox consisting of five aminoacyl-tRNA synthetases for incorporation of non-canonical amino acids
Colocalization of the RuBisCo and and subcellular compartment (carboxysome) using a fluorescent amino acid
Development of a photoswitchable lycopene pathway
Design, chemical synthesis and proof of functionality of a novel, fully synthetic amino acid based on cyanonitrobenzothiazol and asparagine
Modeling more than ten new aaRS sequences
Library development with several hundred thousand sequences for selecting aminoacyl-tRNA synthetases
Construction of positive and negative selection plasmids for the evolution of new synthetases for non-canonical amino acids
Improvement of an aminoacyl-tRNA synthetase test-system by introducing a FRET-system and development of a ranking system
Construction of an LED panel for irradiating 96-well microtiter plates, which can be used to manipulate non-canonical amino acids and for other applications
Development of an Android App to control the LED panel with your smartphone via Bluetooth
Writing a biosafety report entitled “Auxotrophy to Xeno-DNA: A Comprehensive Exploration of Combinatorial Mechanisms for a High-Fidelity Biosafety System”
Writing the ChImp Report on “Chances and Implications of an Expanded Genetic Code”
We Applied for the Following Special Prizes
Best Integrated Human Practices
Early contacts with experts of various fields motivated us to host a workshop on “Expanding the Genetic Code” to facilitate discussions on current issues related to our topic. We incorporated these insights by submitting a comprehensive scientific review concerning biosafety issues in iGEM and the science in general. In addition, we created a report on Chances and Implications of an Expanded Genetic Code.
Best Education and Public Engagement
Throughout our project, we explained our topic to the public and the scientific community on multiple occasions and engaged in fruitful discussions. Furthermore, we raised awareness for the chances of synthetic biology by cooperating with many initiatives and programs. This includes the “6th CeBiTec Pupil a number ofs Academy”, “Teutolab Biotechnology”, the street science festival “GENIALE” and through collaboration with the biotechnological student’s initiative btS e.V.
Best Measurement
Our toolbox for the incorporation of non-canonical amino acids provides innovative new methods for the characterization of all protein encoding parts. With the help of non-canonical amino acids, the subcellular localization of a protein can be investigated. Measurement of intramolecular distances, protein immobilization light regulations and modification enables sophisticated characterizations of parts.
Best Modeling
Via structural prediction combined with recent scientific knowledge, we generated optimized synthetases, which are intended to recognize synthetic amino acids. This is the foundation of a promising and powerful alternative to synthetase selection in the lab.
Best Applied Design
Inspired by the highly specific condensation reaction of D-luciferin from the firefly Photinus pyralis, we came up with the idea to synthesize our own new amino acid, combining organic chemistry, computational modeling, and molecular biology. We designed the novel synthetic amino acid Nγ cyanobenzothiazolyl L asparagine (CBT-asparagine), its cyano group undergoes a condensation reaction with the 1,2-aminothiol group of Nε L cysteinyl L lysine (CL).
Best Basic Part
For our project, we characterized the structure, kinetics, dynamics and subcellular localization of the nucleotide transportter PtNTT2 and different PtNTT2 variants. Therefore, we applied bioinformatic prediction tools, confocal laser scanning microscopy, SDS-PAGE, Western Blot, MALDI-TOF as well as HPLC-MS. In the future, advanced endosymbiotic systems or novel biosafety mechanisms can be developed based on this part.
Best Composite Part
This part ( BBa_K2201373 ) contains a T3 RNA Polymerase with an inverted mRFP under T3 RNA polymerase control for the enhancing of reporter signals. It is an improved reporter and a genetic circuit that could report even weak expression levels. It was designed based on the model of an amplifier in electrical engineering to intensify an existing input signal and could be used in a broad range of synthetic biology applications.
Best Part Collection
Our part collection expands the possibilities for advanced protein design, utilizing novel amino acids with diverse chemical abilities. We provide six different aminoacyl-tRNA synthetases for the translational incorporation of non-canonical amino acids to the iGEM community.
Best Software Tool
Our comprehensive software suite is composed of two connected modules for the analysis of unnatural base pairs in a specified target sequence: M.A.X and iCG. We postulate that our suite is also applicable to the study of DNA modifications and epigenetics.
Best Hardware
We designed and constructed a multifunctional LED panel in a 96-well microtiter plate format through multiple rounds of optimization. This device enables the extensive irradiation of samples with a high resolution of light of different wavelengths and intensities. Using our self-written Android application, complex illumination protocols can be programmed and send to the device via Bluetooth.
Integrated Human Practices
Throughout our project we got advice and support by over thirty experts that influenced our work profoundly. We organized a conference about the expansion of the genetic code and wrote a report about the chances and implications of an expanded genetic code. Find all human practices we integrated into our project here .
Improve a previous part
We improved the validation system for aminoacyl-tRNA synthetases for ncAAs from Austin Texas 2014 (BBa_K1416004) and Aachen 2016 ( BBa_K2020040 ) with a FRET system ( BBa_K2201343 ) and used it in our project. You can find our part improvement site here .
Model your project
Using ROSETTA, we successfully modeled several protein sequences for our synthetases to incorporate the new synthetic non-canonical amino acid CBT-asparagine and 2-nitrophenylalanine. Find the results of our modeling here .
Demonstrate your work
We were able to integrate and detect the unnatural bases in the DNA sequences by software modification of Nanopore Sequencing. We also expanded the genetic code by providing a toolbox of some evolved synthetases. These are able to incorporate non-canonical amino acids into any kind of protein and proof their functionality. Find a summary of our achievements here .
Validated Part
In our project we used and created various parts. We want to highlight our nucleotide triphosphate transporter part BBa_K2201004 which functioned just as expected and has been well characterized and validated by us.
Collaboration
Throughout our project we collaborated with many iGEM teams to improve our work and help each other. Especially our part exchange with the CU Boulder team and the mentoring of the new iGEM team UNIFI was very important for us. Find the summary of our collaborations here .
Human Practices
During the last months we had a lot of interactions with the public trying to convey all aspects of our work and project. For example we worked with students at the GENIALE or at our institute’s yearly pupil’s academy and organized a literature workshop about synthetic biology. We had appearances at the radio, created a little biotechnology quiz show for a student initiative and wrote a biosafety report. All human practices of the silver category are collected here .
Register and attend
We registered for the competition in March and are proud to be able to attend the Giant Jamboree. We are very excited!
Deliverables
We have done all the required deliverables as you can see in our team wiki, our project attribution, our team poster, our presentation on the 12th of november, our safety and judging forms, our part page and our submitted samples.
Attribution
We created an attribution site where we listed and thanked all the people who supported us with advice and material throughout our project. If not mentioned specifically we have performed the work ourselves and the results are due to our own work.
Characterization
We characterized various parts and successfully participated at the Interlab Measurement Study .
