Team:Bielefeld-CeBiTec/Project/toolbox/labeling
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Labeling
Short Summary
As part of the toolbox , the labeling of a protein in vivo is a useful tool that allows the  investigation of a protein in its native environment. As a label for our target protein we  used the fluorescent amino acid L‑(7‑hydroxycoumarin‑4‑yl) ethylglycine (CouAA) that is   incorporated by an orthogonal t‑RNA/aminoacyl‑synthethase pair at a defined position.
To demonstrate this tool, we want to verify that the ribulose 1,5‑bisphosphat carboxylase oxygenase (RuBisCO) is located within the carboxysome , an artificial compartment surrounded   by proteins and used by the iGEM Team CeBiTec 2014 to increase the activity of the RuBisCO. The carboxysome has already been tagged with a
green fluorescent protein (GFP)
and we   want to co-localizate the RuBisCO labeled  with an genetically encoded fluorescent amino   acid L‑(7‑hydroxycoumarin‑4‑yl) ethylglycine and in comparison labeled with
red fluorescent protein (RFP)
.
Labeling of a Protein in vivo
Protein localization in vivo can be performed by labeling the target protein with a  fluorescent protein like green fluorescent protein (GFP) or red fluorescent protein (RFP) .  The labeling is done by a translational fusion of the CDS from the fluorescent protein C-  or N‑terminal with a short linker to the CDS of the target protein. But the labeling is  limited to the C- or N‑terminus and due to its size, GFP (29 kDa)) might negatively impact the function of the target protein or hinder protein-protein interactions if the protein is part of  a larger complex or oligomer (Charbon et al., 2011, Wang et al., 2006).
The usage of a genetically encoded fluorescent amino acid would circumvent these problems  and deliver a tool to study protein localization and function in vivo and in vitro. An   orthogonal t‑RNA/aminoacyl‑tRNA synthetase pair allows the incorporation of amino acids using the amber stop codon (TAG) selectively at a defined position in the protein   (Charbon et al., 2011).
L‑(7‑hydroxycoumarin‑4‑yl) ethylglycine (CouAA)
The fluorescent amino acid L‑(7‑hydroxycoumarin‑4‑yl) (CouAA) ethylglycine is     relatively small, has a high fluorescence quantum yield and relatively large     Stoke's shift. It is also soluble in water and pH-sensitive so it can indicate     pH-changes in the cell (Wang et al., 2006). The translational incorporation of CouAA with an    aaRS was shown by Schultz et al. 2006 and Charbon et al. 2011 into different proteins.
Name:   L‑(7‑hydroxycoumarin‑4‑yl) ethylglycine
Short:  CouAA
CAS:  905442‑42‑4
MW:  263.25
Storage:           -20 °C
Source: Bachem
Prize:  1g - £590.00
Function:          Fluorescent amino acid
Figure 1: Structure of CouAA
Structure of the fluorescent amino acid L‑(7‑hydroxycoumarin‑4‑yl) ethylglycine (CouAA).
The amino acid is suitable for in vivo and in vitro localization, even for     localization in SDS‑PAGES. The extinction and emission spectrum of CouAA is shown in Figure 2:
Figure 2: Fluorescence spectrum of CouAA
Adsorption and fluorescence spectrum of L‑(7‑hydroxycoumarin‑4‑yl) ethylglycine. (Wang et al., 2006).
The relative fluorescence signal of CouAA decreases over time when irradiated with the  excitation wavelength. This effect is called photobleaching and occurs more or less   in all fluorophores. This photobleaching effect is shown in Figure 3 for the labeling of   the bacterial tubulin FtsZ with CouAA (Charbon et al., 2011).
Figure 3: Photobleaching of CouAA
The in vivo dynamic properties of FtsZ10CouAA.  The graph represents the data corrected  for photobleaching due to image acquisition for unbleached (green) and bleached (blue) regions; the red line represents the theoretical recovery curve fit. FtsZ10CouAA (The labeled protein) half-time recovery is 12(+-5) s (mean ±standard deviation); 11.6 s in the example shown. (Charbon et al., 2011).
Colocalization of the Ribulose 1,5‑Bisphosphate Carboxylase Oxygenase and the Carboxysome
Ribulose 1,5‑bisphosphate carboxylase oxygenase (RuBisCO)
The protein we want to label with CouAA  is  ribulose 1,5‑bisphosphate carboxylase oxygenase   (RuBisCO). RuBisCO catalyzes the incorporation of inorganic CO2 to ribulose 1,5‑bisphosphate  to form two 3‑phosphoglycerate molecules. The catalyzed reaction is shown in Figure 4.  (Jordan et al., 1981).
Figure 4: RuBisCO reaction
Reaction catalyzed by ribulose 1,5-bisphosphat carboxylase oxygenase (RuBisCO). Ribulose 1,5‑bisphosphate is converted in two molecules 3‑phophoglycerate.
Due to its numerous side reactions, for example the oxygenase activity resulting in the  production of 2‑phosphoglycolate when O2 is present, RuBisCO is a very inefficient catalyst.   CO2 and O2 are competitive substrates in the two reactions and only the production of   3‑phosphoglycerate leads to CO2 fixation. (Andersson 2008, Jordan et al., 1981). To create an environment with a higher local CO2 concentration, the iGEM team Bielefeld CeBiTec 2014 created an artificial compartment the carboxysome . We want to show where the RuBisCO is located in the cell, inside the carboxysome or in the cytoplasm.
References
Andersson, I.(2008). Catalysis and regulation in Rubisco. Journal of Experimental Botany. 51(7): 1555-1568.
Charbon, G., Brustad, E., Scott, K.A., Wang, J., Lobner-Oelson, A. Schultz, P. G., Jacobs-Wagner, C., Chapman, E.(2011). Subcellular Protein Localization by Using a Genetically Encoded Fluorescent Amino Acid. ChemBioChem. 12:1818-1821.
Charbon, G., Wang, J., Brustad, E., Schultz, P. G., Horwiich, A. L., Jacobs-Wagner, C., Chapman, E.(2011). Localization of GroEL determined by in vivo incorporation of a fluorescent amino acid. Bioorg Med Chem Lett. 21(20)6067-6070.
Jordan, D. B., Ogren, W. L.(1981). Species variation in the specifity of ribulose bisphosphate carboxylase/oxygenase. Nature.291: 513-515.
Wang, J., Xie, J., Schultz, P. G.(2006). A Genetically Encoded Fluorescent Amino Acid. American Chemical Society.128:8738-8739
