Team:Bielefeld-CeBiTec/Results/translational system
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Translational System Overview
The incorporation of a non‑canonical amino acid (ncAA) requires a tRNA/aminoacyl‑tRNA synthetase (tRNA/aaRS) pair which is able to accept and bind the ncAA (to charge the tRNA with the ncAA). Therefor aour aim is to generate a library of ncAAs with different binding sites. The aaRS, based on the wild type Methanococcus jannashii tyrosyl‑tRNA synthetase , is orthogonal to tRNA/aaRS from E.coli and suitable to incorporate novel ncAAs .
For the purpose of generating our toolbox , we used mutated variants of two different aminoacyl tRNA/synthetases (aaRS), tyrosyl-tRNA synthetase and pyrrosylyl-tRNA/synthetase . We proved that the incorporation of non-canonical amino acids (ncaa)through the amber codon poses as major metabolic pressure for the cells and therefore implying that the expansion of the genetic code is an efficient approach.  In addition to that, we demonstrated that the different aaRS we used in our toolbox are very different in specificity, and therefore some are not very efficient in incorporating those. This led us to generate our own tyrosyl tRNA‑synthetase (TyrRS) library . The TyrRS, based on the wild type Methanococcus jannashii tyrosyl tRNA synthetase , is orthogonal to tRNA/aaRSs from E.coli and suitable to incorporate novel ncAA. Our library consists of more than 150,000 library plasmids, among them 27,672 different TyrRS variants with variable binding sites.  The TyrRS library is the basis for a combination of positive and negative selection cycles , where the different TyrRS variants are selected for specificity incorporating non canonical amino acids. In this manner, an optimal adapted TyrRS variant is obtained. We successfully performed both positive and negative selection cycles, with L‑(7‑hydroxycoumarin‑4‑yl)ethylglycine (CouAA) and 2-nitrophenylalanine (2-NPA). As a result we received TyrRS variants with potentially more specifity for the incorporation of the certain ncAA.
