<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_unnatural_base_pair_preservation_system skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/unnatural base pair/preservation system</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/5/5b/T--Bielefeld-CeBiTec--nanoporesequenzer.jpeg);"><DIV id="title-bg"><DIV id="title-text">
						Retention and Preservation System
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Short Summary</H3><DIV class="article">
				We started by validating the basic requirements for the <I>in vivo</I> experiments with unnatural base pairs. A two-plasmid system for retention of unnatural base pairs was developed and characterized. In our <I>in vivo</I> experiment, we were able to detect unnatural base pairs in the specified target sequence using our adapted <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods#ONSseq">Oxford Nanopore Sequencing</A> after 48 hours. Furhter experiments need to be conducted for quantification of unnatural base retention efficiency.
			
		</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Pretests</H3><ARTICLE>
           Due to the high costs of our unnatural base pair (UBP) we had to carefully plan the experiments involving these bases.
		   Therefore, we carried out cultivation experiments of <I>Escherichia coli</I> BL21(DE3) in micro well plates, because performing 
		   experiments in a microscale would significantly reduce the cost of the experiments. To cultivate at the ideal growth 
		   conditions for our purposes, we performed pretests. In our ex-periments, we tested for the ideal culture volume, surface size and 
		   shaking frequency.
			The <I>E. coli</I> strain BL21(DE3) was transformed with the plasmid pSB3C5. We use psB3C5, which is a low copy Plasmid, just like our Plasmid 
			containing the UBP to have a comparison between our cultivations.  LB<SUB>Cm25</SUB> plates were incubated overnight at 37 °C. 
			Three colonies were picked and incubated in 150 mL LB media supplemented with LB<SUB>Cm25</SUB> overnight at 37 °C. 
			Multiwell-plates with 12, 24 and 48 wells were inoculated to an OD<SUB>600</SUB> of 0.1 with a total of three biological replicates. 
			Three different cultivation volumes were used to observe the cell growth in the different well plates with the different volumes. 

<B>Table 1: Used well plates and the different cultivation volumes in the particular plate.</B> 
Three replicates per plate and volume were recorded at an rpm of 350.
<TABLE><THEAD><TR><TH><A target="_blank" href="https://online-shop.eppendorf.de/DE-de/Cell-Culture-and-Imaging-Consumables-110320/Cell-Culture-Consumables-110321/Eppendorf-Cell-Culture-Plates-PF-68136.html">12 Well Eppendorf 0030721012</A></TH><TH><A target="_blank" href="https://online-shop.eppendorf.de/DE-de/Cell-Culture-and-Imaging-Consumables-110320/Cell-Culture-Consumables-110321/Eppendorf-Cell-Culture-Plates-PF-68136.html">24 Well Eppendorf 0030722019</A></TH><TH><A target="_blank" href="https://online-shop.eppendorf.de/DE-de/Cell-Culture-and-Imaging-Consumables-110320/Cell-Culture-Consumables-110321/Eppendorf-Cell-Culture-Plates-PF-68136.html">48 Well Eppendorf 0030723015</A></TH></TR></THEAD><TBODY><TR><TD> 1 mL</TD><TD> 1 mL</TD><TD> 0.5 mL</TD></TR><TR><TD> 2 mL</TD><TD> 2 mL</TD><TD> 0.75 mL</TD></TR><TR><TD> 3 mL</TD><TD> 3 mL</TD><TD> 1 mL</TD></TR></TBODY></TABLE><ARTICLE>
  The cultivation was carried out in the VWR – Incubation Microplate Shaker and the OD<SUB>600</SUB> measured via NanoDrop ND-1000 Spectrophotometer. 
  Three technical replicates were measured for each sample.   
        </ARTICLE><DIV class="figure large"><P class="figure subtitle"><B>Figure 1: </B> 
													A: Average OD<SUB>600</SUB> of the three biological replicates of the cultivation volumes of 1 mL (red), 2 mL (blue) and 3 mL (yellow) in the 12 well plate over the cultivation period. Three technical replicates were measured. 
													B: Average OD<SUB>600</SUB> of the three biological replicates of the cultivation volumes of 1 mL (red), 2 mL (blue) and 3 mL (yellow) in the 24 well plate over the cultivation period. Three technical replicates were measured.
													C: Average OD<SUB>600</SUB> of the three biological replicates of the cultivation volumes of 0.5 mL (red), 0.75 mL (blue) and 1mL (yellow) in the 48 well plate over the cultivation period. Three technical replicates were measured.</P></DIV><ARTICLE>
        Figure 1 shows that cultivations in all multiwell plates are possible.  The lowest OD<SUB>600</SUB> is consistently achieved
		by the highest volume with values at 2.27, 2.037 and 1.29 using the 12 well plate, 24 well plate and 48 well plate, respectively.
		The highest OD<SUB>600</SUB> is associated with the lowest volume with values of 2.63, 2.42 and 1.89 using the 12 well plate,
		24 well plate and 48 well plate, respectively. 
		Irrespective of volume, the highest OD<SUB>600</SUB> values were reached using the 12 well plate. Specifically, 
		the OD<SUB>600</SUB> value of 2.27 using 3 mL in the 12 well plate is still higher than the OD<SUB>600</SUB> value of 
		2.03 using 1 mL in the 24 well plate. 
		After determining the best plate and volume to perform cultivations with, we investigated the influence of the rpm by 
		cultivating three biological replicates at 500, 600, and 700 rpm in the 12 well plate in 1 mL cultivation volume.
    
        </ARTICLE><DIV class="figure large"><P class="figure subtitle"><B>Figure 2: </B> Growth curves of the three replicates of <I>E.coli</I> BL21 (DE3) in
			LB<SUB>CM25</SUB> at 350, 500, 600, and 700 rpm, respectively. The DNA concentration was measured under the optimal condition 
			of 600 rpm and 1 mL volume.</P></DIV><ARTICLE>
            Cultivation at 600 rpm yields the highest OD<SUB>600</SUB> with a value of about 4.96 after 6.5 h. 
			If the rpm is further increased, the cells grow badly due to an out of phase movement of the media. 
			We finally concluded that the perfect cultivation conditions are reached in a 12 well plate with a total volume 
			of 1 mL and shaking with 600 rpm.
		For analysis of the retention and preservation of the unnatural base pair, the isolation of a sufficient amount of 
		plasmid DNA from the culture is important. In order to evaluate the plasmid concentration during the cultivation, 
		we cultivated at 600 rpm on a 12 well plate in 1 mL. We inoculated the wells with a starting OD<SUB>600</SUB> of 0.1 and 
		conducted a plasmid isolation using 1 mL for each measurement. The highest DNA concentration was reached in the exponential
		phase at a OD<SUB>600</SUB> of 2.75 with about 2.5 µg/µl plasmid isolated from 1 mL culture.

        </ARTICLE></ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Cytosine Deaminase <I>cod</I>A</H3><ARTICLE>
            
Deletion of the <I>cod</I>A gene could be useful for our project due to its ability [[to transform isoG and isoC]] into uracile. 
In this case the concentration of the UBPs decreases and our plasmid carrying the UBPs could not be replicated properly. 
We designed three compatible constructs for the potential deletion of <I>cod</I>A following the protocol of Jiang <I>et al.</I>, 
2015 using the CRISPR/Cas9 technology. . 



At first, we designed  our construct carrying our synthesized single guide RNA (sgRNA), which is essential for the targeting specificity. 
 Because of the presence of the sgRNA, Cas9 could carry out a sequence-specific, double-stranded break.
 In the case of a repair via homology directed repair with our designed repair template, <I>cod</I>A gets deleted. 
 Just like Jian <I>et al.</I>, we chose <A target="_blank" href="https://www.addgene.org/62226/">pTarget</A>
 as our vector. Therefore, we annealed the sgRNA oligos, cloned them into 
 the pTarget backbone via golden gate assembly and selected positive clones via blue-white screening. 



The PCR-construct responsible for the homologous recombination, and therefore the <I>cod</I>A deletion, was generated by two PCRs. In a first step, 700 bp upstream and downstream of <I>cod</I>A were amplified. These two fragments were then inserted into the backbone using Gibson Assembly. The homologous recombination of the neighboring sequences of <I>cod</I>A with the plasmid bound flanks leads to a deletion of this gene. 


<DIV class="figure medium"><P class="figure subtitle"><B>Figure 3: </B> Design of the flanking sequences upstream and downstream of the <I>cod</I>A gene for the potential deletion of <I>cod</I>A.</P></DIV><ARTICLE>
          
		As the CRISPR/Cas9 system plasmid, we conceptualized pCas, also used by Jiang <I>et al.</I>, containing the lambda-Red system for a
		higher recombineering efficiency. For the <I>cod</I>A gene deletion, you need to generate <I>E. coli</I> BL21 DE electrocompetent cells harboring pCas, 
		and Arabinose (10 mM) has to be added for the induction of the lambda-Red system. For the homology directed repair, one needs 
		to transform pTarget and the PCR fragment into the competent <I>E. coli</I> cell harboring the pCas plasmid.
		Then, one needs to verify positive transformants by colony PCR and DNA sequencing. 
			
			
			


To cure the cells from pTarget, one can inoculate a positive colony at 30 °C for 8 to 16 hours in media containing kanamycin 
(50 mg L<SUP>-1</SUP>) and IPTG (0.5 mM). For the curing of pCas, the cell can be grown overnight at 37 °C non-selectively.
The challenging part of this is the biological meaning of <I>cod</I>A because of its important role in the purine synthesis pathway. 
To counteract this, it is advised to supplement with uracil (Mahan <I>et al.</I>, 2004). 
  
        </ARTICLE></ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Design of a Plasmid for the Retention of Unnatural Base Pairs </H3><ARTICLE>
		Selection and screening of the transformants was not possible with our experimental design since the retention experiment had to be started immediately after transformation. Therefore, we designed a plasmid which would kill or inhibit growth of the wrong colonies. Our initial design focused on the use of <I>ccd</I>B, but given that <I>ccd</I>B cannot be submitted to the parts registry, we decided to use levansucrase of <I>Bacillus subtilis</I>, encoded by <I>sac</I>B instead. Therefore, a <I>mRFP</I>-<I>sac</I>B fusion construct was designed (<A href="partsregistry.org/Part:BBa_K2201017">BBa_K2201017</A>). <A href="partsregistry.org/Part:BBa_J23100">BBa_J23100</A> was used as a promoter and <A href="partsregistry.org/Part:BBa_B0034">BBa_B0034</A> was used as a strong RBS, followed by <A href="partsregistry.org/Part:BBa_E1010">BBa_E1010</A>. Between <I>mRFP</I> and <I>sac</I>B, a linker consisting of four alanines was used. As a backbone, pSB3C5 was used. Given that this plasmid was designed to eventually carry an unnatural base pair, a low-copy plasmid was the better choice for a higher stability of the unnatural base pair.
		The idea behind this construct is the following: In the initial plasmid, mRFP and <I>sac</I>B are in-frame, meaning that cells turn red when incubated in standard media such as LB, but die or grow weakly when cultivated in media supplemented with sucrose. This is due to the toxic effect of levansucrase, which unfolds when cells are cultivated in sucrose supplemented media. 
		
		<DIV class="figure eighty"><P class="figure subtitle"><B>Figure 4: Fusionprotein of <I>mRFP</I> and <I>sac</I>B. Shown is the linker between the two sequences.</B> A flexible 4xAla linker was used between <I>mRFP</I> and <I>sac</I>B. </P></DIV>
		
		
		The linker between <I>mRFP</I> and <I>sac</I>B represents the insertion site for the fragment containing the unnatural base pair. This insertion, if successful, leads to a frameshift of the downstream located <I>sac</I>B, leading to loss of function. Therefore, cells can only survive in sucrose supplemented media if the fragment containing the unnatural base pair was successfully incorporated. This provides an easy screening method, since cells that turn red and survive in sucrose supplemented media most likely inserted the fragment containing the unnatural base pair into the plasmid. Having the coding sequences of both proteins in one frame has one big advantage: Given that <I>sac</I>B is prone to loss-of-function mutations, contaminations can be easily distinguished by color. Furthermore, cells with mutations in the promotor region would also survive, since they would not express <I>sac</I>B, but not turn red. Therefore, this two-layered system allows to visually check if a culture either is contaminated, has a mutation in the promotor region, or if the correct plasmid is present. This aspect is visualized in Figure 5. 
		
		<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 5: Method for selection of the correct plasmids in liquid cultures. </B> If the assembly was successful, cells turn red and survive in sucrose supplemented media. The cells die if the assembly was not successful, since <I>sac</I>B can be expressed.</P></DIV>
		To test the effect of sucrose on growth of the cells, a cultivation was performed of <I>E. coli</I> BL21(DE3) harboring <A href="partsregistry.org/Part:BBa_K2201017">BBa_K2201017</A>. The cultivations were carried out in a 12 well plate in 1 mL of LB medium supplemented with different concentrations of sucrose. Three biological replicates were cultivated for each condition and three technical replicates taken for each measurement point. 

		
		<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 6: Cultivation of <I>E. coli</I> BL21(DE3) in LB media supplemented with different concentrations of sucrose. </B> Three biological replicates were cultivated and three technical replicates measured for each point in time. A clear difference in growth can be observed, with higher sucrose concentrations leading to weaker growth. The mRFP-SacB fusion protein does not lead to cell death, but shows bacteriostatic properties. Therefore, cells that successfully integrated the fragment containing the unnatural base pair should grow much better in the selective media and therefore overgrow cells harboring religated plasmid backbones. </P></DIV>
		
		The cultivations shown in Figure 6 clearly show that sucrose has a significant effect on growth of strains carrying <A href="partsregistry.org/Part:BBa_K2201017">BBa_K2201017</A>. A higher sucrose concentration leads to weaker growth, with strains cultivated in 20 % sucrose reaching only ~33 % of the OD600 of the reference strain cultivated in sucrose free media. The reference reached a final optical density of 3.233 ± 0.148, strains cultivated in media supplemented with 10 % sucrose 1.527 ± 0.055 and strains cultivated in 20 % sucrose reached a final optical density of 0.44 ± 0.05. 
		Figure 7 shows a comparison of strains cultivated in media without sucrose and media supplemented with 10 % sucrose. A preculture of <I>E. coli</I> BL21(DE3) harboring <A href="partsregistry.org/Part:BBa_K2201017">BBa_K2201017</A> was prepared and cultivated overnight at 37 °C. 30 µl of this precultures were used and dropped onto LB agar plates either without or supplemented with 10 % sucrose. 50 µl of the same preculture were used to inoculate 3 mL of LB medium without sucrose and 3 mL of LB media supplemented with 10 % sucrose.
		
		<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 7: <I>E. coli</I> BL21(DE3) <A href="partsregistry.org/Part:BBa_K2201017">BBa_K2201017</A> cultivated on plates and in liquid cultures with and without 10 % sucrose. </B> All cultures were cultivated for 24 hours at 37 °C. <B>A)</B> Shows a drop on a LB agar plate supplemented with 10 % sucrose. <B>B)</B> Shows a drop on a LB agar plate without sucrose. A clear difference can be observed between the two drops. In presence of sucrose, <I>E. coli</I> BL21(DE3) <A href="partsregistry.org/Part:BBa_K2201017">BBa_K2201017</A> grew much weaker and did not turn red. Without sucrose, the cells grew much better and turned red. <B>C)</B> The same difference in growth and color could be observed in liquid cultures.</P></DIV></ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Further Pretests</H3><ARTICLE>
		The antibiotics kanamycin and chloramphenicol were not added to the recovery media to achieve better growth conditions for the cells right after the heat shock transformation. We investigated the growth conditions for <I>E. coli</I> BL21(DE3) concerning the concentration of chloramphenicol. A lower concentration of chloramphenicol would decrease the stress of the cells during their cultivation since there is already a huge metabolic stress when using a two-plasmid system. So, we grew the native <I>E. coli</I> BL21(DE3) in liquid LB media supplemented with different chloramphenicol concentrations as a pretest shown in Figure 8.

		<DIV class="figure eighty"><P class="figure subtitle"><B>Figure (8): <I>E. coli</I> BL21(DE3) cultivated in liquid LB media using different Cm concentrations.</B> Precultures with LB media supplemented with chloramphenicol concentrations c= 0, 1, 2, 3, 4, 5, 10, 15, 20, 25 μg μL<SUP>-1</SUP> were inoculated with native <I>E. coli</I> BL21(DE3). All cultures were cultivated for 24 h at 37 °C. Growth was only visible for cultures with c(Cm)&lt;2 μg μL<SUP>-1</SUP>.</P></DIV>
		
		This experiment showed that not any native <I>E. coli</I> BL21(DE3) cell without a chloramphenicol resistance gene is able to grow above c(Cm)= 1 μg μL<SUP>-1</SUP>. Based on this experiment we chose c(Cm)= 3 μg μL<SUP>-1</SUP> to exclude any growth from <I>E. coli</I> BL21(DE3) during our cultivation for the UBP retention.
		Another pretest should investigate the influence of the unnatural nucleotidetriphosphates isoGTP and isoC<SUP>m</SUP>TP on the growth the chemically competent cells <I>E. coli</I> BL21(DE3) containing the pSB1K3 high-copy plasmid <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201027">BBa_K2201027</A>. Therefore we inoculated these cells in 2xYT supplemented with 50 μg μL<SUP>-1</SUP> kanamycin and different concentrations c= 5, 10, 50, 100 μM of each unnatural nucleotidetriphosphate. The different concentrations of unnatural nucleotidetriphosphates did not show any influence on the growth of the cells.
		
		<DIV class="figure seventy"><P class="figure subtitle"><B>Figure (9): Growth test of chemically competent cells <I>E. coli</I> BL21(DE3) containing <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201027">BBa_K2201027</A> with different concentrations of isoGTP and isoC<SUP>m</SUP>.</B> The plasmid <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201027">BBa_K2201027</A> is coding for the nucleotide transporter <A href=" https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">PtNTT2</A> which ensures the uptake of isoGTP and isoC<SUP>m</SUP> from the surrounding media into the cell. The cells were incubated in 500 μL 2xYT media at 37 °C for 1 h. For incubation at 37 °C and 600 rpm in 12-well plates in the VWR – Incubation Microplate Shaker 2xYT media was added to 1 mL with 50 μg μL<SUP>-1</SUP> kanamycin and c= 5, 10, 50, 100 μM of each isoGTP and isoC<SUP>m</SUP>TP final concentration. Three technical replicates of the OD<SUB>600</SUB> were measured every hour via the NanoDrop ND-1000 Spectrophotometer.</P></DIV></ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> The Two-Plasmid System for UBP Retention </H3><ARTICLE>
		The retention system was engineered to preserve the unnatural base pair (UBP) isoG-isoC<SUP>m</SUP><I>in vivo</I>. This intention requires the uptake of unnatural nucleotides from the media and a selection pressure on the plasmids carrying the UBP. We developed a two-plasmid system for the retention. As described above, the first high-copy plasmid (pSB1K3) <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201027">BBa_K2201027</A> contains the truncated version of <I>PtNTT2</I> for the uptake of isoGTP and isoC<SUP>m</SUP>TP and <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system"><I>cas9</I> for the digestion of all plasmids that have lost the UBP <I>in vivo</I></A>. The Cas9 is guided by five sgRNAs (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201077">BBa_K2201077</A>) that bind every possible point mutation that leads to the loss of the UBP. Those sgRNAs are part of the second plasmid. This plasmid <A href=" http://parts.igem.org/Part:BBa_K2201032">BBa_K2201032</A> is a composite part of <A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201077">BBa_K2201077</A> and <A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201017">BBa_K2201017</A>. We decided this plasmid to be a low-copy plasmid (pSB3C5), because a high-copy version could possibly cause a greater mutation frequency. The second plasmid also contains a different antibiotic resistance than the first plasmid for a selection of bacteria carrying both plasmids. The UBP was part of the oligo <A href="">UBP_target</A> and was assembled with the linearized backbone of <A href=" http://parts.igem.org/Part:BBa_K2201032">BBa_K2201032</A>. Primers (<A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">17tx and 17we</A>)  for the linearization via PCR were used so that the Gibson Assembly with the oligo <A href="">UBP_target</A> leads to a frameshift within the <I>sacB</I> coding sequence of the mRFP-sacB fusionprotein. Hence, a successful Gibson Assembly enables the transformed organism to grow in the presence of sucrose later on.
		
		</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3><I>In vivo</I> Experiment for UBP Retention</H3><ARTICLE>
		The Romesberg lab (Zhang <I>et al.</I>, 2017) was able to proof retention of their hydrophobic UBP in <I>E. coli</I> BL21(DE3) using the nucleotide transporter PtNTT2 and Cas9. Therefore we transformed <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201027">BBa_K2201027</A> into <I>E. coli</I> BL21(DE3) via heat shock with the goal to produce chemically competent cells containing the retention plasmid afterwards. Those chemically competent cells were used for a second heat shock transformation of the second plasmid <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201032">BBa_K2201032</A> containing our UBP isoG-isoC<SUP>m</SUP>. 10 μL of the above described Gibson Assembly was transformed into chemically competent cells. After the heat shock, the cells were recovered in liquid recovery media (2xYT media supplemented with 50 mM K<SUB>2</SUB>HPO<SUB>4</SUB>, 0.5 mM IPTG, 100 μM isoC<SUP>m</SUP>TP, and 100 μM isoGTP) and shaked at 200 rpm and 37 °C for 1 h. Growth was performed in liquid growth media (2xYT media supplemented with 50 mM K<SUB>2</SUB>HPO<SUB>4</SUB>, 0.5 mM IPTG, 100 μM isoC<SUP>m</SUP>TP, 100 μM isoGTP, 3 μg μL<SUP>-1</SUP> chloramphenicol, and 50 μg μL<SUP>-1</SUP> kanamycin) while shaking at 600 rpm and 37 °C for 48 h. The described experimental set-up is shown in Figure 10.
		K<SUB>2</SUB>HPO<SUB>4</SUB> is needed in the recovery and growth media as a competitive inhibitor for phosphatases. This prevents the dephosphorylation of unnatural nucleotide triphosphates. The IPTG in the media induces the expression of <I>cas9</I> that is negatively regulated by our designed <I>lac</I> operon (<A href=" http://parts.igem.org/Part:BBa_K2201020">lacO_tight1</A>) optimized for tight repression. This needs to be added right from the beginning after the transformation to start the retention system during recovery. For the same reason the isoGTP and isoC<SUP>m</SUP>TP were added to the recovery and growth media.

Before analyzing the efficiency of the UBP retention, the plasmids need to be isolated from the cultivation samples. But this leads to the isolation of both plasmids, the high-copy (pSB1K3) and low-copy (pSB3C5) plasmid. For the detection of the UBP, which lays on the pSB3C5 plasmid, the pSB1K3 plasmid needs to be digested by restriction enzymes. This can be achieved by the restriction enzymes <I>Pvu</I>I, <I>Nsi</I>I und <I>Bam</I>HI that are solely targeting the pSB1K3 plasmid <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201027">BBa_K2201027</A>. Afterwards, the remaining pSB3C5 plasmid can be prepared for the Oxford Nanopore Sequencing method in order to detect the UBP.
		
		<DIV class="figure eighty"><P class="figure subtitle"><B>Figure (10): Experimental flowchart for UBP retention <I>in vivo</I>.</B>The pSB3C5 plasmid <A href=" http://parts.igem.org/Part:BBa_K2201032">BBa_K2201032</A> containing five sgRNAs and <I>mRFP-sacB</I> was linearized with the primers <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">17tx and 17we</A> and assembled with <A href="">UBP_target</A> via Gibson Assembly. 10 μL of the Gibson Assembly was transformed into chemically competent <I>E. coli</I> BL21(DE3) cells containing the pSB1K3-plasmid <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201027">BBa_K2201027</A>. After the heat shock, the cells were recovered in 850 μL liquid recovery media (2xYT media supplemented with 50 mM K<SUB>2</SUB>HPO<SUB>4</SUB>, 0.5 mM IPTG, 100 μM isoC<SUP>m</SUP>TP, and 100 μM isoGTP) and shaked at 200 rpm and 37 °C in a 12-well plate in the VWR – Incubation Microplate Shaker for 1 h. Then, the recovery media was filled up with liquid growth media (2xYT media added with 50 mM K<SUB>2</SUB>HPO<SUB>4</SUB>, 0.5 mM IPTG, 100 μM isoC<SUP>m</SUP>TP, 100 μM isoGTP, 3 μg μL<SUP>-1</SUP> chloramphenicol, and 50 μg μL<SUP>-1</SUP> kanamycin for final concentrations) up to 1 mL and shaked at 600 rpm and 37 °C in a 12-well plate in the VWR – Incubation Microplate Shaker for 48 h. Plasmid isolations were performed for single 1 mL cultures. The high-copy plasmid pSB1K3 <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201027">BBa_K2201027</A> was digested by the restriction enzymes <I>Pvu</I>I, <I>Nsi</I>I und <I>Bam</I>HI. The remaining low-copy plasmid pSB3C5 containing the UBP was then prepared for the Oxford Nanopore Sequencing.</P></DIV>
		
		With the help of Oxford Nanopore Sequencing we were able to detect the UBP after the <I>in vivo</I> cultivation. Therefore we assume that the retention system is functional. Further replicates would need to be done for a statistical aproved analysis. This experimental seems to be a very promising set-up for the UBP retention <I>in vivo</I>.
		
		<DIV class="figure seventy"><P class="figure subtitle"><B>Figure (11): Oxford Nanopore Sequencing of <A href=" http://parts.igem.org/Part:BBa_K2201032">BBa_K2201032</A> containing the UBP after <I>in vivo</I> cultivation and retention.</B>The flow cell lays can be seen in the front. On the laptop screen: Green areas on the grid show the quantity of nanopores sequencing a DNA molecule at a moment.
			</P></DIV></ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> References </H3><ARTICLE><B>Zhang, Y., Lamb, B.M., Feldman, A.W., Zhou, A.X., Lavergne, T., Li, L., and Romesberg, F.E.</B> (2017). A semisynthetic organism engineered for the stable expansion of the genetic alphabet. Proc. Natl. Acad. Sci. <B>114</B>: 1317–1322.
</ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>