Team:Bielefeld-CeBiTec/Results/unnatural base pair
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Unnatural Base Pair Overview
Uptake and Biosynthesis
For the uptake of supplemented unnatural bases we used the nucleotide transporter PtNTT2. This unspecific nucleotide transporter is originated from the alga Phaeodactylum tricornutum, where it is located in the plastid membrane. Additional signal peptides guide PtNTT2 to be integrated within the plasma membrane of E. coli. This ensures the uptake of isoGTP and isoCmTP from the surrounding medium into the cells.
Instead of feeding the cells with unnatural bases by supplementing them to medium, the biosynthesis of unnatural bases would be a huge step towards a fully autonomous semi-synthetic organism. The Botanical Garden of Marburg University kindly provided us  some cuttings of the plant Croton tiglium, which is known to produce isoG as one of the unnatural bases we used. After an RNA extraction and RNA-sequencing library preparation, we identified some potential enzyme candidates based on the transcriptome, which could be responsible for the production of isoG.
Retention System
We designed a two-plasmid system for the retention of our unnatural base pair (UBP). The first plasmid contains the nucleotide transporter PtNTT2, and cas9 whereas the second plasmid contains the UBP and five sgRNAs. PtNTT2 ensures the uptake of unnatural nucleotides from the medium. Cas9 is needed to cleave all plasmids that lost the UBP on the second plasmid. There are five possible sequences after a mutation of the UBP. The UBP could either be substituted by one of the four natural bases or completely deleted. There is a sgRNA for all five cases that binds the sequence recruiting Cas9 to cut the plasmid. Therefore, the selection marker on the plasmids that carry the UBP leads to its retention.
Development of New Methods
The use of UBPs requires new protocols for standard lab methods. We had to adapt the PCR, and the transformation of plasmids with UBPs. Another challenge was the validation of the presence of the UBP. Therefore we created the software tool M.A.X. that identifies a set of restriction enzymes based on the input DNA sequence. In a simple in vitro test the predicted restriction enzymes can then be used to digest the DNA and a following gel electrophorese will indicate the presence or absence of the UBP according to M.A.X.
