<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_unnatural_base_pair_preservation_system skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/unnatural base pair/preservation system</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/5/5b/T--Bielefeld-CeBiTec--nanoporesequenzer.jpeg);"><DIV id="title-bg"><DIV id="title-text">
						Retention and Preservation System
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Short Summary</H3><ARTICLE>
The retention and preservation of unnatural nucleoside triphosphates and unnatural bases in the DNA requires a cellular retention and preservation system acting on multiple cellular levels. On one hand, degradation of the unnatural nucleoside triphosphates by enzymes like cytosine deaminase must be prohibited for efficient integration into the DNA. On the other hand, once the unnatural bases are incorporated into the DNA, loss or change of the unnatural bases have to be prevented. We want to achieve this by adapting CRISPR/Cas9 to detect and prevent mutations of the unnatural bases.
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Preservation system using Cas9 </H3><ARTICLE>
			Due to tautomerisation of isoG and hydrolysis of isoC<SUP>m</SUP> and the resulting loss of the unnatural base pair (UBP), there is a need for a system, to preserve the UBP on the plasmid. In 2017, Zhang <I>et al</I>. successfully deployed a CRISPR (<I>clustered regularly interspaced short palindromic repeat</I>)-Cas9 system for retention of a UBP. We adapted this conservation system to our UBP and thus used CRISPR/Cas9 to eliminate all plasmid DNA that had lost the UBP. 
			
The nuclease Cas9 is part of the adaptive immune system of <I>Streptococcus pyogenes</I>, where it induces double strand breaks in the genomic DNA. This enzyme is recruited   by a CRISPR RNA (crRNA). A crRNA consists of direct repeats interspaced by variable sequences called protospacer. Those protospacers are derived from foreign DNA and encode the Cas9 guiding sequence (guide RNA). An auxiliary transactivating crRNA (tracrRNA) helps processing the precursor crRNA array into an active crRNA that contains the 20 nucleotide guide RNA. The guide RNA binds to the complementary genomic DNA sequence via Watson‑Crick base pairing. For this binding, the genomic DNA sequence needs to be located upstream of a CRISPR type II specific 5’NGG protospacer adjacent motif (PAM). Synthetically chimeric single stranded guide RNA (sgRNA) was designed by combining crRNA and tracrRNA. In the sgRNA, only the 20 nucleotide guiding sequence needs to be exchanged for targeting any genomic sequence followed by a PAM sequence (Ran <I>et al</I>., 2013 a, b). The resulting double strand break introduced by Cas9 leads to exonucleolytic degradation of the DNA in prokaryotic cells (Simmon and Lederberg, 1972).

In our case we envision a retention system, where Cas9 cleaves Plasmids at sites where the UBP is absent. This works by using a sgRNA complementary to the DNA sequence without the UBP. In plasmids with the UBP present, the mismatch between isoG/isoC<SUP>m</SUP> and sgRNA greatly decreases Cas9 activity (Zhang <I>et al.</I>, 2017). In the event of UBP loss, the sgRNA now binds perfectly to the mutated site and restores Cas9 activity which leads to degradation of the mutated plasmid. Consequently, this leads to a retention of the UBP in the plasmids.
<DIV class="figure large"><P class="figure subtitle"><B>Figure 1: UBP conservation system using Cas9.</B> sgRNAs are targeted against every DNA sequence emerging from UBP loss on a plasmid. A: Loss of the UBP leads to a point mutation. Now a sgRNA can bind to the DNA target sequence. Cas9 is recruited and cleaves the plasmid, which is followed by its degradation. B: Plasmids that contain a UBP in the DNA target sequence lead to a mismatch with every sgRNA. Cas9 does not cleave the plasmid, leading to retention of the UBP.</P></DIV>
Because the iGEM competition is based on plasmids that can be submitted to the parts registry we refrained from creating a novel strain for the UBP retention. Consequently, our retention system consists of two plasmids. The first plasmid (BBa_K2201010) contains the nucleotide transporter gene <I>PtNTT2</I> and the <I>cas9</I> on a pSB1K3 backbone. This plasmid was transformed into <I>E. coli</I> BL21(DE3). After producing chemically competent cells, another transformation was done with the second plasmid. The second plasmid carries the UBP and the sgRNAs for recruiting Cas9 on a pSB3C5 plasmid. Since two large plasmids produce a great metabolic stress for an organism, a more sufficient UBP retention could be achieved with a newly created <I>E. coli</I> strain, that has the genes <I>PtNTT2</I> and <I>cas9</I> in its genomic DNA. We provide the parts that are required to create a repair template for the knock in strategy using a CRISPR/Cas9 system. These parts were designed to perform a genomic integration into <I>E. coli</I> BL21(DE3). The genomic knock in can be done according to the protocol by Cobb <I>et al.</I>, 2014 using the pCRISPomyces plasmid system. A sgRNA was designed searching for a reverse sequence with the constraint N(16)R(4)NGG (Cobb <I>et al.</I>, 2004) inside the coding sequence of the <I>arsB</I> gene (Zhang <I>et al.</I>, 2017) of the <I>E. coli</I> genome. <I>arsB</I> is coding for an arsenic efflux pump membrane protein. As a result, 5‘ TATTGTTCATAATAGAAGAG 3‘ turned out to be the suggested guiding sequence with the highest on target activity score with being unique within the complete genome. The required repair template is designed with 1 kb long flanking sequences and two terminators BBa_B0015 flanking the gene of interest for example <I>cas9</I> (Figure 2). To be able to strictly regulate the expression of <I>cas9</I> we rationally designed an optimized IPTG inducible promoter <I>P<SUB>lacO‑tight1</SUB></I>.
		
		<DIV class="figure large"><P class="figure subtitle"><B>Figure 2: The repair template (BBa_K2201028) for genomic integration of <I>cas9</I> into <I>E. coli</I> BL21(DE3).</B> The 1 kb left flanking sequence (LFS: BBa_K2201021) and right flanking sequence (RFS: K2201022) are taken from the genome of <I>E. coli</I> BL21(DE3). Inside the genome <I>LFS</I> and <I>RFS</I> are directly flanking the coding sequence of <I>arsB</I>. The strong terminators from BBa_B0015 were used to avoid basal expression and to stop transcription right after <I>cas9</I>. There are composite parts the consist of <I>LFS</I> + terminator + <I>P<SUB>lacO‑tight1</SUB></I> (BBa_K2201024) and terminator + <I>RFS</I> (BBa_K2201025) that can be assembled with any coding sequence of interest to create a repair template. The coding sequence of <I>cas9</I> is taken from the pCRISPomyces plasmid system by Cobb <I>et al</I>., 2014 and is originated from <I>S. pyogenes</I>. It is negatively regulated by the IPTG‑inducible promoter <I>P<SUB>lacO‑tight1</SUB></I>.</P></DIV>
This composite BioBrick from Figure 2 needs to be restricted with <I>Not</I>I to separate the linear repair template from the pSB1C3 backbone. Together with the target plasmid containing the sgRNA and the pCRISPomyces plasmid containing the <I>cas9</I> all three elements were co‑transfected into <I>E. coli</I> BL21(DE3). The genomic integration was verified by sequencing.
		
		</ARTICLE><H3> Optimization of the negatively regulating promoter <I>P<SUB>lac</SUB></I></H3><ARTICLE>
		
		We designed a tightly repressed <I>lac</I> operon called <I>P<SUB>lacO‑tight1</SUB></I> to achieve a low basal transcription rate. The <I>lac</I> operon was extensively researched starting from the midst of last century. This showed the role of the <I>lac</I> repressor and the inducible nature of the <I>lac</I> promoter, which results in 1000-fold increased expression in its activated state (Müller <I>et al.</I>, 1996). The wild type <I>lac</I> operon consists of the genes <I>lacZ</I>, <I>lacY</I> and <I>lacA</I> that are transcribed from the lac promoter <I>P<SUB>lac</SUB></I> into a polycistronic mRNA (Figure 4). These genes code for the proteins β‑galactosidase, Lac permease and Lac transacetylase (Oehler <I>et al</I>., 1994).
		
		<DIV class="figure large"><P class="figure subtitle"><B>Figure 3: DNA map of the wild type <I>lac</I> operon and its transcriptional and translational products (Oehler <I>et al</I>., 1994).</B></P></DIV>
		
		The transcription is constitutively activated by the CAP protein. <I>lacI</I> codes for the tetrameric Lac repressor and is expressed by the promoter <I>P<SUB>i</SUB></I>. The Lac repressor can bind to the <I>lac</I> operators O1, O2 or O3. O3 is located 92 bp upstream of O1 and O2 401 bp downstream of O1. The inter‑operator distances are counted from the center of O1 to the center of the distal operator. By binding simultaniously to O1 and O2 or to O1 and O3, the Lac repressor forms a DNA loop that negatively controlls the expression of <I>P<SUB>lac</SUB></I> (Oehler <I>et al</I>., 1994). 
		
			<DIV class="figure large"><P class="figure subtitle"><B>Figure 4: The wild type <I>lac</I> operators and its regulatory structures.</B>The tetrameric Lac repressor can bind either the <I>lac</I> operators O1 and O3 or O1 and O2 to form a DNA loop. The DNA loops efficiently inhibit the transcription by the CAP protein (Oehler <I>et al</I>., 1994).</P></DIV>
		
		In 1994, Oehler <I>et al</I>. showed that an inactivated O2 in its natural position does not decrease the repression by low amounts of tetrameric Lac repressor. Based on this result, we designed our <I>P<SUB>lacO‑tight1</SUB></I> without the <I>lac</I> operator O2. It was also shown that two weak operators result in a tighter repression than a single strong operator. This can be explained by the thermodynamic concept that a second operator increases the local concentration of the Lac repressor for the neighboring operator. As a consequence there is a higher probability of occupation for two operators by the Lac repressor leading to a tighter repression (Oehler <I>et al</I>., 1996). In 1983, Sadler <I>et al</I>. proposed a ideal <I>lac</I> operator O<SUB>id</SUB> that binds the Lac repressor 10‑fold tighter than the natural strong <I>lac</I> operator O1. O<SUB>id</SUB> is an inverted repeat of the left half of O1 (Figure 5). 
		
		<DIV class="figure medium"><P class="figure subtitle"><B>Figure 5: The <I>lac</I> operator O<SUB>id</SUB>.</B>The inversion is indicated by the arrow for the perfectly symmetric <I>lac</I> operator O<SUB>id</SUB>. It is the inverted repeat of the left half of O1 (Sadler <I>et al</I>. 1983).</P></DIV>
		
		The operator‑DNA‑operator complex requires energy for the bending process to form a DNA loop. Additional energy for a torsion is required when the two <I>lac</I> operators lay on opposite sites of the helical DNA surface. Therefore, the DNA loop formation is energetically favoured for <I>lac</I> operators in phase (Müller <I>et al</I>., 1996). In 1996, Mueller <I>et al</I>. investigated the strength of repression for an inter‑operator distance of O<SUB>id</SUB> and O1 from 57.5 bp up to 1493.5 bp. The repression values were compared to the repression by a single O1 at its natural position. A shorter spacing than 57.5 bp could not be examined due to the  35 box of the promoter. Phase dependency for the repression was observed for a spacing around 200 bp. That leads to the observation of periodically maxima for repression values (Figure 6).
		
		<DIV class="figure medium"><P class="figure subtitle"><B>Figure 6: Repression values dependent on inter‑operator distances between O1 and O<SUB>id</SUB>.</B>The repression values refer to the repression of the chromosomal <I>lacZ</I> gene under the control of O1 at its natural position and O<SUB>id</SUB> at the indicated position. With 50 tetrameric Lac repressors per cell the repression value is calculated by the specific activity of β‑galactosidase in absence of active Lac repressor divided by the specific activity of β‑galactosidase in the presence of active Lac repressor. The dashed line shows the repression value for a single natural O1 operator (Mueller <I>et al</I>., 1996).</P></DIV>
		
		The distance of 70.5 bp showed the strongest repression value which is 50‑fold higher than the natural repression. Repression drops sharply to 15‑fold at a 150.5 bp spacing and to threefold at around 600 bp. All inter‑operator distances beyond 600 bp kept a twofold increased repression value (Müller <I>et al</I>., 1996).

		According to these results, our <I>P<SUB>lacO‑tight1</SUB></I> consists of the auxiliary operator O<SUB>id</SUB> with a 70.5 bp spacing to O1 at its natural position. The residuray sequence like the P<SUB><I>lac</I></SUB> was kept as in the natural <I>lac</I> operon taken from <I>E. coli</I> BL21(DE3) (Figure 7).

<DIV class="figure large"><P class="figure subtitle"><B>Figure 7: Tight <I>lac</I> operon <I>P<SUB>lacO‑tight1</SUB></I>.</B>The figure with its annotations was created with the software Geneious 10.0.8.</P></DIV></ARTICLE></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Deletion of <I>codA</I></H3><ARTICLE>
To retain the unnatural base pair and keep a sufficient level of unnatural nucleoside triphosphate in the cell, the degradation of the unnatural nucleoside triphosphates must be minimized. In <I>E. coli</I>, the gene <I>codA</I> codes for the cytosine deaminase, an enzyme of the pyrimidine metabolism. The cytosine deaminase catalyzes the reaction of cytosine to uracil. Furthermore, it can catalyze the deamination of isoguanosione and isocytosine. Isoguanosine is formed during oxidative stress by reaction of the radical oxygen species (ROS) •OH with adenine. Other products from reactions of adenine with ROS include 8-oxoadenine and 6-N-hydroxyaminopurine. 

<DIV class="figure fourty"><P class="figure subtitle"><B>Figure 8: Reactions catalyzed by the cytosine deaminase.</B> A) The conversion of cytosine to uracil is a normal reaction step within the pyrimidine metabolism. B) Isocytosine is converted into uracil by the cytosine deaminase. C) Isoguanine is formed during oxidative stress. Cytosine deaminase catalyzes the reaction to the non-mutagenic xanthine.</P></DIV>

In <I>E. coli</I>, the <I>codA</I> gene is part of the <I>codBA</I> operon. <I>CodB</I> codes for the cytosine permease,
 while <I>codA</I> encodes cytosine deaminase.The cytosine deaminase hydrolyses cytosine to ammonia and uracil by hydrolytic deamination. 
 This reaction poses the only way how cytosine can the metabolized in <I>E.coli</I>(Danielsen <I>et al.</I>, 1992). When isoguanosine and isocytosine are provided to the cell,
 they are converted to uracil in the wildtype. Therefore, it is necessary to knock out or delete the codA gene to enable the stable
 availability of isoguanosine and isocytosine. 

<DIV class="figure large"><P class="figure subtitle"><B>Figure 9: Arrangement of the <I>codBA</I> operon. </B><I>codA</I> is located downstream of <I>codB</I> and 1.3 kb in size. Both genes overlap by 11 bases. <I>codB</I> codes for cytosine permease, while <I>codA</I> codes for cytosine deaminase.</P></DIV><I>codA</I> is 1281 bp in size, resulting in a protein composed of 427 amino acids after translation. 
The protein CodA is located in the cytosol and has an atomic mass of 47.5 kD.

<DIV class="figure large"><P class="figure subtitle"><B>Figure 10: Crystal structure of cytosine deaminase from <I>Escherichia coli</I> complexed with zinc and phosphono-cytosine. </B> The structure was determined by X-Ray crystallography with a resolution of 1.71 Å (Hall et al., 2011).</P></DIV></ARTICLE></DIV></DIV><DIV class="container"><DIV class="contentbox"><DIV class="content"><H2> References </H2><ARTICLE><DIV class="article"><B>Cobb, R.E., Wang, Y., and Zhao, H.</B> (2015). High-efficiency multiplex genome editing of Streptomyces species using an engineered CRISPR/Cas system. ACS Synth. Biol. <B>4</B>: 723–8.
 </DIV><DIV class="article"><B>Danielsen, S., Kilstrup, M., Barilla, K., Jochimsen, B., and Neuhard, J.</B> (1992). Characterization of the Escherichia coli codBA operon encoding cytosine permease and cytosine deaminase. Mol. Microbiol. <B>6</B>: 1335–1344.
 </DIV><DIV class="article"><B>Hall, R.S., Fedorov, A.A., Xu, C., Fedorov, E. V., Almo, S.C., and Raushel, F.M.</B> (2011). Three-Dimensional Structure and Catalytic Mechanism of Cytosine Deaminase. Biochemistry <B>50</B>: 5077–5085
 </DIV><DIV class="article"><B>Müller, J., Oehler, S., and Müller-Hill, B.</B> (1996). Repression of lac Promoter as a Function of Distance, Phase and Quality of an Auxiliary lac Operator. J. Mol. Biol. <B>257</B>: 21–29.
</DIV><DIV class="article"><B>Oehler, S., Amouyal, M., Kolkhof, P., von Wilcken-Bergmann, B., and Müller-Hill, B.</B> (1994). Quality and position of the three lac operators of E. coli define efficiency of repression. EMBO J. <B>13</B>: 3348–3355.
</DIV><DIV class="article"><B>Ran, F.A., Hsu, P.D., Lin, C., Gootenberg, J.S., Konermann, S., Trevino, A.E., Scott, D. a, Inoue, A., Matoba, S., Zhang, Y., and Zhang, F.</B> (2013). Double nicking by RNA-guided CRISPR Cas9 for enhanced genome editing specificity. Cell <B>154</B>: 1380–9.
</DIV><DIV class="article"><B>Ran, F.A., Hsu, P.D., Wright, J., Agarwala, V., Scott, D.A., and Zhang, F.</B> (2013). Genome engineering using the CRISPR-Cas9 system. Nat. Protoc. <B>8</B>: 2281–2308.
</DIV><DIV class="article"><B>Sadler, J.R., Sasmor, H., and Betz, J.L.</B> (1983). A perfectly symmetric lac operator binds the lac repressor very tightly. Proc. Natl. Acad. Sci. U. S. A. <B>80</B>: 6785–9.
</DIV><DIV class="article"><B>Simmon, V.F. and Lederberg, S.</B> (1972). Degradation of bacteriophage lambda deoxyribonucleic acid after restriction by Escherichia coli K-12. J. Bacteriol. <B>112</B>: 161–9.
</DIV><DIV class="article"><B>Zhang, Y., Lamb, B.M., Feldman, A.W., Zhou, A.X., Lavergne, T., Li, L., and Romesberg, F.E.</B> (2017). A semisynthetic organism engineered for the stable expansion of the genetic alphabet. Proc. Natl. Acad. Sci. <B>114</B>: 1317–1322.
</DIV></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>