<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_unnatural_base_pair_uptake skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/unnatural base pair/uptake</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/8/88/T--Bielefeld-CeBiTec--title-img-uptake.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Uptake
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Short Summary</H3><DIV class="article">
Uptake of unnatural nucleoside triphosphates is of <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">major importance</A> for the development of a semisynthetic organism. The nucleotide transporter <I>Pt</I>NTT2 from the algae <I>Phaeodactylum tricornutum</I> was successfully expressed in <I>Escherichia coli</I> BL21(DE3) and comprehensively characterized. We designed and tested five different variants of the transporter and we demonstrated that certain variants of the transporter can facilitate uptake of <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">iso-dC<SUP>m</SUP>TP and iso-dGTP</A> from the media. 
</DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Computational Analysis of <I>Pt</I>NTT2 </H3><ARTICLE>
		Zhang <I><I>et al.</I></I> used an N-terminal truncated version of <I>Pt</I>NTT2, lacking the first 65 amino acids, since they observed some kind of toxicity resulting from the native N-terminal sequence (Zhang <I>et al.</I>, 2017). For our project, we analyzed the amino acid sequence of <I>Pt</I>NTT2 using the prediction software Phobius (Käll <I>et al.</I>, 2007). Using Phobius, we analyzed the signal peptide and the transmembrane topology of <I>Pt</I>NTT2. The analysis revealed that the native signal peptide is formed by amino acids 1-30, which means that Zhang <I>et al.</I> removed more than the predicted native signal peptide for their experiment. The results of the prediction are shown in Figure 1. Analysis of the transmembrane topology of the transporter, which is integrated into the plastid membrane in its native algal cell, shows iterative non-cytoplasmatic, transmembrane and cytoplasmic regions. The topology might indicate, that the transporter will be integrated into the inner membrane when expressed in <I>E. coli</I>. 

<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 1: Results of the analysis of <I>Pt</I>NTT2 using Phobius.</B> The 30 first amino acids are clearly recognized as a signal peptide. Ten transmembrane domains are predicted.</P></DIV></ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Plasmid Design</H3><ARTICLE>
		For the analysis and characterization of <I>Pt</I>NTT2, a total of eleven plasmids were designed and cloned based on initial research and the computational analysis. The coding sequence of <I>Pt</I>NTT2 was codon optimized using the IDT Codon Optimization Tool and ordered as two gBlocks. Using overlap extension PCR, the two gBlocks were put together and inserted into pSB1C3 using Gibson Assembly. The truncated versions of the transporter as well as the versions with new signal peptides were constructed using primers and Gibson Assembly. 
		
		<DIV class="figure sixty"><P class="figure subtitle"><B>Figure 2: Schematic overview of the design of the different transporter variants.</B> The lacUV5 promotor was used together with a strong RBS (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>) for all parts. All variants except for pSB1C3-PtNTT2 were also tagged with GFP (<A href="http://parts.igem.org/Part:BBa_E0040">BBa_E0040</A>). cMyc was used as a linker (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201181">BBa_K2201181</A>).</P></DIV>
		
		All plasmids except pSB1C3-PtNTT2 were tagged with the same cMyc-GFP construct for imaging and Western Blots. cMyc was used as a linker to allow the targeting of the fusion protein with two separate antibodies, an anti-GFP antibody and the anti-cMyc antibody. The GFP part was taken from <A href="http://parts.igem.org/Part:BBa_E0040">BBa_E0040</A>, while the cMyc fragment was taken from <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201181">BBa_K2201181</A>. The fusion proteins were assembled using Gibson Assembly. A schematic overview of the design of the different plasmids is shown in Figure 2.		

		
		<P class="figure subtitle"><B>Table 1: Designed and cloned plasmids for the analysis and characterization of <I>Pt</I>NTT2. </B></P><TABLE style="margin: auto"><THEAD><TR><TH style="width: 100%;" class="header">Plasmid Name</TH><TH style="width: 100%;" class="header">BioBrick Number</TH><TH style="width: 100%;" class="header">Characteristics</TH></TR></THEAD><TBODY><TR><TD>pSB1C3-PtNTT2</TD><TD><B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201004">BBa_K2201004</A></B></TD><TD>Only the cds</TD></TR><TR><TD>pSB1C3-PlacUV5-PtNTT2</TD><TD><B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201000">BBa_K2201000</A></B></TD><TD>cds with lacUV5 promotor and strong RBS (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>) </TD></TR><TR><TD>pSB1C3-PlacUV5-PtNTT2(66-575) </TD><TD><B><A href="&lt;b&gt;http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201001">BBa_K2201001</A></B></TD><TD>cds with lacUV5 promotor and a strong RBS (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>)</TD></TR></TBODY><TBODY><TR><TD>pSB1C3-PlacUV5-PtNTT2(31-575)</TD><TD><B><A href="&lt;b&gt;http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201005">BBa_K2201005</A></B></TD><TD>cds with lacUV5 promotor and a strong RBS (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>), truncated version lacking the first 30 amino acids</TD></TR></TBODY><TBODY><TR><TD>pSB1C3-PlacUV5-pelB-SP-PtNTT2 </TD><TD><B><A href="&lt;b&gt;http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201006">BBa_K2201006</A></B></TD><TD>cds with lacUV5 promotor and a strong RBS (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>), native signal peptide replaced with the pelB signal peptide</TD></TR></TBODY><TBODY><TR><TD>pSB1C3-PlacUV5-TAT-SP-PtNTT2</TD><TD><B><A href="&lt;b&gt;http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201007">BBa_K2201007</A></B></TD><TD>cds with lacUV5 promotor and a strong RBS (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>), native signal peptide replaced with a TAT signal peptide</TD></TR></TBODY><TBODY><TR><TD>pSB1C3-PlacUV5-PtNTT2-GFP</TD><TD><B><A href="&lt;b&gt;http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201002">BBa_K2201002</A></B></TD><TD>Fusion protein of BBa_ K2201000 with GFP (<A href="http://parts.igem.org/Part:BBa_E0040">BBa_E0040</A>), Myc epitope tag as linker (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201181">BBa_K2201181</A>)</TD></TR></TBODY><TBODY><TR><TD>pSB1C3-PlacUV5-PtNTT2(66-575)-GFP</TD><TD><B><A href="&lt;b&gt;http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201003">BBa_K2201003</A></B></TD><TD>Fusion protein of BBa_ K2201001 with GFP (<A href="http://parts.igem.org/Part:BBa_E0040">BBa_E0040</A>), Myc epitope tag as linker (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201181">BBa_K2201181</A>)</TD></TR></TBODY><TBODY><TR><TD>pSB1C3-PlacUV5-PtNTT2(31-575)-GFP</TD><TD><B><A href="&lt;b&gt;http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201011">BBa_K2201011</A></B></TD><TD>Fusion protein of BBa_K2201005 with GFP (<A href="http://parts.igem.org/Part:BBa_E0040">BBa_E0040</A>), Myc epitope tag as linker (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201181">BBa_K2201181</A>)</TD></TR></TBODY><TBODY><TR><TD>pSB1C3-PlacUV5-pelB-SP-PtNTT2-GFP</TD><TD><B><A href="&lt;b&gt;http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201012">BBa_K2201012</A></B></TD><TD>Fusion protein of BBa_K2201006 with GFP (<A href="http://parts.igem.org/Part:BBa_E0040">BBa_E0040</A>), Myc epitope tag as linker (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201181">BBa_K2201181</A>)</TD></TR></TBODY><TBODY><TR><TD>pSB1C3-PlacUV5-TAT-SP-PtNTT2-GFP</TD><TD><B><A href="&lt;b&gt;http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201013">BBa_K2201013</A></B></TD><TD>Fusion protein of BBa_K2201007 with GFP (<A href="http://parts.igem.org/Part:BBa_E0040">BBa_E0040</A>), Myc epitope tag as linker (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201181">BBa_K2201181</A>)</TD></TR></TBODY></TABLE></ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Cultivations of the Different <I>Pt</I>NTT2 Variants </H3><ARTICLE>Given that Zhang <I><I>et al.</I></I>, 2017 reported some kind of toxicity resulting from the N-terminal sequence of <I>Pt</I>NTT2, we first investigated if we could also observe the same toxicity associated with the N-terminal sequence. We also tested whether our own versions of the transporter results in better growth and reduced toxicity compared to the native transporter. Therefore, after cloning the plasmids in <I>E. coli</I> DH5α and verifying the correct assembly via sequencing, all plasmids were transformed into <I>E. coli</I> BL21(DE3). The presence of the correct plasmids was again verified by colony PCR.
		The first cultivations were carried out in shake flasks in LB media. A total cultivation volume of 50 mL was used. The cultures were incubated at 37 °C and 180 rpm in 500 mL shake flasks. All cultures were inoculated with an OD<SUB>600</SUB> of 0.01. OD<SUB>600</SUB> was measured every hour during lag and stationary phase and every 30 minutes during the exponential phase. The optical density was measured using an Eppendorf Photometer and standard cuvettes. <I>E. coli</I> BL21(DE3) without a plasmid and <I>E. coli</I> BL21(DE3) harboring pSB1C3-PtNTT2 were used as negative controls. Two biological replicates of each strain were tested and three technical replicates were measured for each timepoint. 


<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 3: Shake flask cultivation of all <I>Pt</I>NTT2 variants. </B><I>E. coli</I> BL21(DE3) and <I>E. coli</I> BL21(DE3) pSB1C3-PtNTT2 (<B>BBa_K2201004</B>), not expressing <I>Pt</I>NTT2, were used as negative controls. Two biological replicates of each strain were cultivated and three technical replicates taken for each measurement. A clear difference in the growth rates can be observed, with <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 (<B>BBa_K2201000</B>) and <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 (<B>BBa_K2201007</B>) showing the weakest growth. Both strains also show the longest lag phase, which is nearly twice as long as the lag phase of <I>E. coli</I> BL21(DE3). <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) (<B>BBa_K2201001</B>) and <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-pelB-SP-PtNTT2 (<B>BBa_K2201006</B>) show the best growth of all <I>Pt</I>NTT2 variants, reaching the highest OD<SUB>600</SUB>. </P></DIV><I>E. coli</I> BL21(DE3) without any plasmid shows the best growth, with the highest specific growth rate and the highest final OD<SUB>600</SUB> of 5.178 ± 0.046. The second negative control, <I>E. coli</I> BL21(DE3) harboring pSB1C3-PtNTT2 (<B>BBa_K2201004</B>) reached the second highest OD<SUB>600</SUB> with 4.638 ± 0.029. Of the functional <I>Pt</I>NTT2 variants, strains harboring pSB1C3-PlacUV5-PtNTT2(66-575) (<B>BBa_K2201001</B>) and pSB1C3-PlacUV5-pelB-SP-PtNTT2 (<B>BBa_K2201006</B>) reached the highest ODs with 4.397 ± 0.062 and 4.171 ± 0.051, respectively. <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(31-575) (<B>BBa_K2201005</B>) showed similar growth to the two previous strains during the lag phase and early exponential phase, but reaching a lower OD<SUB>600</SUB> of 3.802 ± 0.135. <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 (<B>BBa_K2201000</B>) and <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 (<B>BBa_K2201007</B>) grew substantially weaker compared to all previous strains, with a lag phase nearly four hours long and final ODs of 2.499 ± 0.134 and 2.735 ± 0.150, respectively. The difference in growth between the two negative controls can be explained by the metabolic burden caused by plasmid replication and expression of the chloramphenicol resistance. Therefore, <I>E. coli</I> BL21(DE3) pSB1C3-PtNTT2 (<B>BBa_K2201004</B>) is the more accurate control, since all samples contain the same plasmid backbone and were also grown in LB media supplemented with chloramphenicol. If not indicated otherwise, pSB1C3-PtNTT2 (<B>BBa_K2201004</B>) was used as the main reference. The difference in growth between strains expressing the native, full length transporter and the truncated version <I>Pt</I>NTT2(66-575), observed by Zhang <I>et al.</I> 2017, could also be shown. This negative effect might be associated with the native signal peptide of <I>Pt</I>NTT2, which <I>E. coli</I> might not be able to process correctly. Another explanation for the weak growth could be that the native transporter variant has a higher activity compared to the other variants. If the activity is too high, this might lead to a toxic effect and to the observed weak growth. Given that all transporter variants were expressed with the same promotor, this difference in growth cannot be explained by differences in the expression.


<P class="figure subtitle"><B>Table 2: Final OD<SUB>600</SUB> of all cultures.</B> The highest OD<SUB>600</SUB> was reached by the wildtype <I>E. coli</I> BL21(DE3), the lowest by <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2.</P><TABLE style="margin: auto"><THEAD><TR><TH style="width: 100%;" class="header">Strain</TH><TH style="width: 100%;" class="header">Final OD<SUB>600</SUB> [-]</TH></TR></THEAD><TBODY><TR><TD><I>E. coli</I> BL21(DE3) </TD><TD>5.178 ± 0.046</TD></TR></TBODY><TBODY><TR><TD><I>E. coli</I> BL21(DE3) pSB1C3-PtNTT2 </TD><TD>4.638 ± 0.029</TD></TR></TBODY><TBODY><TR><TD><I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 </TD><TD>2.499 ± 0.134</TD></TR></TBODY><TBODY><TR><TD><I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) </TD><TD>4.397 ± 0.062</TD></TR></TBODY><TBODY><TR><TD><I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(31-575) </TD><TD>3.802 ± 0.135</TD></TR></TBODY><TBODY><TR><TD><I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-pelB-SP-PtNTT2 </TD><TD>4.171 ± 0.051</TD></TR></TBODY><TBODY><TR><TD><I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 </TD><TD>2.735 ± 0.150</TD></TR></TBODY></TABLE>
To determine the maximum specific growth rate (µ<SUB>max</SUB>), the natural logarithm of the OD<SUB>600</SUB> values was plotted against the cultivation time. The slope of the linear regression through the exponential phase gives µ<SUB>max</SUB>. The graphical determination of µ<SUB>max</SUB> for the shake flask cultivation is shown in Figure 4 	
	
	<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 4: Graphical determination of µ<SUB>max</SUB>. </B> The highest specific growth rate was determined for each culture by plotting the natural logarithm of OD<SUB>600</SUB> against the cultivation time. The slope of the linear regression through the exponential phase gives µ<SUB>max</SUB>.  </P></DIV>
	The highest specific growth rate can be observed for the negative controls <I>E. coli</I> BL21(DE3) and <I>E. coli</I> BL21(DE3) pSB1C3-PtNTT2 with values of 1.201 ± 0.070 h<SUP>-1</SUP> and 1.212 ± 0.029 h<SUP>-1</SUP>, respectively. The lowest specific growth rate could be observed for <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 with a value of 0.946 ± 0.030 h<SUP>-1</SUP>. Based on the specific growth rate, the minimal doubling time was calculated using the following equation: 
<DIV class="equation-line"><P class="equation-nr">(1)</P></DIV>
	With <I>t<SUB>d</SUB></I> being the doubling time in hours and <I>µ</I>the specific growth rate in <I>h<SUP>-1</SUP></I>. The maximum specific growth rates and minimal doubling times are show in Table 3 for all cultures. 
	
	<P class="figure subtitle"><B>Table 3: Maximum specific growth rates and minimum doubling times for all cultures. </B></P><TABLE style="margin: auto"><THEAD><TR><TH style="width: 60%;" class="header">Strain</TH><TH style="width: 20%;" class="header">µ<SUB>max</SUB> [h<SUP>-1</SUP>]</TH><TH style="width: 20%;" class="header">t<SUB>d</SUB> [h]</TH></TR></THEAD><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) </TD><TD>1.201 ± 0.070</TD><TD>0.577 ± 0.058</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PtNTT2 </TD><TD>1.212 ± 0.029</TD><TD>0.572 ± 0.024</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 </TD><TD>0.978 ± 0.033</TD><TD>0.709 ± 0.034</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) </TD><TD>1.194 ± 0.026</TD><TD>0.581 ± 0.022</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(31-575) </TD><TD>1.143 ± 0.045</TD><TD>0.606 ± 0.039</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-pelB-SP-PtNTT2 </TD><TD>1.189 ± 0.028</TD><TD>0.583 ± 0.024</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 </TD><TD>0.946 ± 0.030</TD><TD>0.733 ± 0.032</TD></TR></TBODY></TABLE>
					These results clearly show that expression of <I>Pt</I>NTT2 leads to a reduced final cell density and slower growth. Furthermore, the different variants of <I>Pt</I>NTT2 differ highly, indicating that some variants of <I>Pt</I>NTT2 negatively affect the growth rate and final cell density.
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Microcultivations of the Different <I>Pt</I>NTT2 Variants </H3><ARTICLE>
	Due to the high price of unnatural nucleoside triphosphates, performing experiments such as cultivations in a small volume is desirable. Therefore, we validated whether cultivations in micro well plates can achieved the same results as cultivations performed in shake flasks and <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">optimized the method</A>. The optimal conditions for cultivations in a small volume were determined as follows: 12 well plates with a cultivation volume of 1 mL, 37 °C and 600 rpm. To analyze the growth of the different <I>Pt</I>NTT2 variants in a micro cultivation, the shake flask cultivation was repeated in a 12 well plate. All cultures were inoculated with an OD<SUB>600</SUB> of 0.1. Three biological replicates of each strain were tested and three technical replicates were measured at each time point. The results are shown in Figure 5.
	
	<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 5: Microcultivation of all <I>Pt</I>NTT2 variants</B><I>E. coli</I> BL21(DE3) and <I>E. coli</I> BL21(DE3) pSB1C3-PtNTT2 (BBa_K2201004) were again used as negative controls. The same growth pattern as in the shake flask cultivation can be observed, with <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-pelB-SP-PtNTT2, <I>E. coli</I>BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) reaching the highest ODs, followed by <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(31-575), <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 and <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2.</P></DIV>
	
	The results of the micro cultivations show the same growth pattern observed in the shake flask cultivations. <I>E. coli</I> BL21(DE3) and <I>E. coli</I> BL21(DE3) pSB1C3-PtNTT2 reached the highest ODs with 5.487 ± 0.038 and 4.337 ± 0.01, respectively. <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-pelB-SP-PtNTT2 reached a final OD<SUB>600</SUB> of 4.110 ± 0.005, followed by <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) with a final OD<SUB>600</SUB> of 4.035 ± 0.051. <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(31-575) reached the third highest OD<SUB>600</SUB> of all<I>Pt</I>NTT2 variants with a final OD<SUB>600</SUB> of 3.865 ± 0.008. The lowest final ODs were again reached by <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 and <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2, reaching a final OD<SUB>600</SUB> of 2.28 ± 0.337 and 1.623 ± 0.481. 
	

Therefore, we were able to demonstrate that the results of the shake flask cultivations can be transferred to a smaller format, such as a micro cultivation in 1 mL. The reproduction of the results in a 50 times smaller volume is important for further experiments. 
	
	
	<P class="figure subtitle"><B>Table 4: Final OD<SUB>600</SUB> of all cultures.</B> The highest OD<SUB>600</SUB> was reached by the wildtype <I>E. coli</I> BL21(DE3) with 5,487 ± 0.038, the lowest by <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 with 1.623 ± 0.481.</P><TABLE style="margin: auto"><THEAD><TR><TH style="width: 100%;" class="header">Strain</TH><TH style="width: 100%;" class="header">Final OD<SUB>600</SUB> [-]</TH></TR></THEAD><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) </TD><TD>5.487 ± 0.038</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PtNTT2 </TD><TD>4.337 ± 0.010</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 </TD><TD>1.623 ± 0.481</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) </TD><TD>4.035 ± 0.051</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(31-575) </TD><TD>3.865 ± 0.008</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-pelB-SP-PtNTT2 </TD><TD>4.110 ± 0.005</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 </TD><TD>2.280 ± 0.337</TD></TR></TBODY></TABLE>
					Like for the shake flask cultivation, µ<SUB>max</SUB> was determined graphically (Figure 6). Bases on the obtained values, the minimum doubling time was calculated. The results are summarized in Table 5.
	
	<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 6: Graphical determination of the maximum specific growth rate µ<SUB>max</SUB> for the microcultivations. </B></P></DIV>
	
All cultures showed extended doubling times when compared to the shake flask cultivations. This is not surprising, since the oxygen transfer is much better in shake flasks compared to well plates. Nonetheless, similar final OD<SUB>600</SUB> values were reached in the micro cultivations. Furthermore, the same differences in growth between the different strains can be observed, with <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 and <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 growing the weakest.

		<P class="figure subtitle"><B>Table 5: Maximum specific growth rate and minimum doubling time for all cultures cultivated in 12 well plates.  </B></P><TABLE style="margin: auto"><THEAD><TR><TH style="width: 60%;" class="header">Strain</TH><TH style="width: 20%;" class="header">µ<SUB>max</SUB> [h<SUP>-1</SUP>]</TH><TH style="width: 20%;" class="header">t<SUB>d</SUB> [h]</TH></TR></THEAD><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) </TD><TD>1.059 ± 0.143</TD><TD>0.655 ± 0.135</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PtNTT2 </TD><TD>1.016 ± 0.133</TD><TD>0.682 ± 0.131</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 </TD><TD>0.829 ± 0.071</TD><TD>0.836 ± 0.086</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) </TD><TD>1.023 ± 0.105</TD><TD>0.678 ± 0.103</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(31-575) </TD><TD>1.021 ± 0.096</TD><TD>0.679 ± 0.094</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-pelB-SP-PtNTT2 </TD><TD>1.047 ± 0.097</TD><TD>0.662 ± 0.093</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 </TD><TD>0.924 ± 0.113</TD><TD>0.750 ± 0.122</TD></TR></TBODY></TABLE>
 					
	To investigate the effect of smaller well plates, a cultivation of two of our strains was performed by the <A href="https://2017.igem.org/Team:UNIFI">iGEM team UNIFI</A> from Florence, Italy. The team cultivated <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 and <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) in a 96 well plate. LB media was used as a reference. The cultivation was performed at 37 °C and 130 rpm in 3  mL of LB media. Three biological replicates were cultivated and measured at each time point. The results are shown in Figure 7. 
	
	<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 7: Microcultivation in a 96 well plate performed by <A href="https://2017.igem.org/Team:UNIFI">iGEM team UNIFI</A> from Florence, Italy.</B><I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 and <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) were cultivated in a total volume of 3 mL at 37 °C and 130 rpm. The growth difference between the two strains observed in previous cultivations could also be observed in this experiment carried out by the team from Florence. <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 reached a final OD<SUB>600</SUB> of 0.329 ± 0.037 while <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) reached a final OD<SUB>600</SUB> of 0.664 ± 0.033. </P></DIV>
		
		The results obtained by iGEM team UNIFI validate our results, since the same difference in growth can be observed between the truncated version <I>Pt</I>NTT2(66-575) and <I>Pt</I>NTT2. Also in a 96 well plate, <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 reached only 49.5 % of the final cell density of <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) (40.2 % for the cultivation carried out in 12 well plates). After 27 hours, the final OD<SUB>600</SUB> of the strain carrying the full-length version of <I>Pt</I>NTT2 was 0.329 ± 0.037, while the final OD<SUB>600</SUB> of the strain expressing the truncated version <I>Pt</I>NTT2(66-575) was 0.664 ± 0.033. These results clearly indicate that it is highly beneficial for higher cell densities and better growth to cultivate in larger well plates. Furthermore, shaking at higher frequencies can improve the growth, as show in the <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">validation of microcultivations</A>. 
		The graphical determination of the maximum specific growth rates is shown in Figure 8.
		
		<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 8: Graphical determination of the maximum specific growth rates for the cultivations carried out in 96 well plates by the <A href="https://2017.igem.org/Team:UNIFI">iGEM team UNIFI</A>.</B></P></DIV>
		
		Graphical determination of the specific growth rates and the minimum doubling times revealed substantially weaker growth of the cultures in 96 well plates compared to 12 well plates or shake flasks. <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) reached the highest specific growth rate with 0,11 ± 0,002 h-1. <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 showed much weaker growth with a maximum specific growth rate of 0,041 ± 0,004 h-1. The maximum specific growth rates and minimum doubling times are summarized in Table 6. 
	
			<P class="figure subtitle"><B>Table 6: Maximum specific growth rate and minimum doubling time for all cultures cultivated in 12 well plates.  </B></P><TABLE style="margin: auto"><THEAD><TR><TH style="width: 60%;" class="header">Strain</TH><TH style="width: 20%;" class="header">µ<SUB>max</SUB> [h<SUP>-1</SUP>]</TH><TH style="width: 20%;" class="header">t<SUB>d</SUB> [h]</TH></TR></THEAD><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) </TD><TD>0.042 ± 0.004</TD><TD>16.504 ± 0.095</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PtNTT2 </TD><TD>0.110 ± 0.002</TD><TD>6.301 ± 0.018</TD></TR></TBODY></TABLE>
	
	Compared to the shake flask cultivations and the micro cultivations in 12 well plates, far lower maximum specific growth rates and much higher minimum doubling times were reached.
	
	</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Verification of the Function of <I>Pt</I>NTT2</H3><ARTICLE>
		To verify that <I>Pt</I>NTT2 fulfills its desired function, a simple experiment was designed. Usually, the function of nucleotide transporters is determined using radioactively labeled nucleotides. Given that we did not want to work with radioactively labeled nucleotides and that not every lab and iGEM team has access to suitable equipment, we focused on verifying the function using a less dangerous and better accesible method. 

Under phosphate starvation, <I>E. coli</I> secretes phosphatases to utilize the phosphates from nucleotides as phosphate source. On the other hand, cells expressing <I>Pt</I>NTT2 and integrating it into the inner membrane should be able to take up nucleoside triphosphates directly. By taking up nucleoside triphosphates directly from the media, the cells can directly take up three phosphates and the nucleobase. Given that the uptake of NTPs by <I>Pt</I>NTT2 is facilitated by counter exchange of ATP, ATP is exported and consequently converted to AMP by extracellular phosphatases. 
<DIV class="figure large"><P class="figure subtitle"><B>Figure 9: Proposed function of <I>Pt</I>NTT2.</B> . A) ATP is exported in presence of the unnatural nucleotides iso-dC<SUP>m</SUP>TP and iso-dGTP, leading to a constant loss of ATP, negatively influencing growth. B) If the media is supplemented with ATP in slightly higher concentrations than the intracellular concentration, ATP is likely taken up in exchange for ATP, ADP as well as other NTPs. A beneficial effect of expression of the transporter on the growth of the cells is achieved due to a small net uptake of ATP.C) In case of much higher extracellular concentrations compared to the intracellular concentration of ATP, ATP will be taken up efficiently in exchange for NTPs, ADP and AMP. This would lead to a net uptake of ATP, but a net loss of NTPs, leading to reduced growth.</P></DIV>
		The experiment we designed consists of two parts and is based on the disability of <I>E. coli</I> to take up nucleoside triphosphates from the media. The first part consists of cultivations performed in MOPS minimal media which is supplemented with either 1.32 mM K<SUB>2</SUB>HPO<SUB>4</SUB> or 1 mM ATP. Calculation of the Odds-ratio as shown in equation (A) allows to evaluate how beneficial expression of <I>Pt</I>NTT2 is for the cell if extracellular ATP or unnatural base pairs represent the sole phosphate source. The second part of the experiment consists of liquid chromatography – mass spectrometry (LC-MS) measurements for the quantification of AMP in the supernatant. Combined, these methods provide a way to investigate the function of <I>Pt</I>NTT2 without the application of radioactively labeled nucleotides. Furthermore, these experiments might also serve as a way for future iGEM teams to easily characterize the function of membrane proteins. 
		
		Figure 9 shows the proposed function of <I>Pt</I>NTT2. In presence of the unnatural nucleotides iso-dC<SUP>m</SUP>TP and iso-dGTP, ATP is exported. Therefore, uptake of iso-dC<SUP>m</SUP>TP and iso-dGTP leads to a constant loss of ATP, negatively influencing growth. If the media is supplemented with ATP in slightly higher concentrations than the intracellular concentration, ATP is likely taken up in exchange for ATP, ADP as well as other NTPs. This would lead to a small net uptake of ATP, and therefore to a beneficial effect of expression of the transporter on the growth of the cells. In case of much higher extracellular concentrations compared to the intracellular concentration of ATP, ATP will be taken up efficiently in exchange for NTPs, ADP and AMP. This would lead to a net uptake of ATP, but a net loss of NTPs, leading to reduced growth. 
			
		For the first part of the experiment, two sets of cultivations were carried out in parallel. All transporter variants as well as two negative controls, <I>E. coli</I> BL21(DE3) and <I>E. coli</I> BL21(DE3) pSB1C3-PtNTT2, were cultivated in MOPS minimal media containing either 1,32 mM K<SUB>2</SUB>HPO<SUB>4</SUB> or 1 mM ATP as sole phosphate source. Three biological replicates of each strain were cultivated in 1 mL of media in a 12 well plate at 37 °C and 600 rpm. For each measurement point, three technical replicates were measured. Figure 10 shows the growth curves of the cultivations carried out with 1,32 mM of K<SUB>2</SUB>HPO<SUB>4</SUB> as the sole phosphate source.
					<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 10: Cultivation of all transporter variants in MOPS media with K<SUB>2</SUB>HPO<SUB>4</SUB> acting as the sole phosphate source.</B> The cultivation was carried out in 12 well plates and three biological replicates were cultivated of each strain. For measurement of the optical density at 600 nm, three technical replicates were taken. </P></DIV>
			The final OD<SUB>600</SUB> values varied widely, with <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-pelB-SP-PtNTT2 reaching the highest OD<SUB>600</SUB> of 3.907 ± 0.018. The lowest OD<SUB>600</SUB> was reached by <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 with a value of 1.537 ± 0.045. All final optical densities at 600 nm are shown in Table 7. 
			
			The cultivations were performed in parallel in MOPS media supplemented with 1 mM ATP as sole phosphate source. Again, three biological replicates of each strain were cultivated and three technical replicates measured for each time point. The growth curves are shown in Figure 11. 	
				
				<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 11: Cultivation of all strains in MOPS media with 1 mM ATP acting as the sole phosphate source.</B> Three biological replicates were cultivated and three technical replicates measured for each time point.</P></DIV>
					
					In ATP supplemented media, the wildtype strain reached the highest OD<SUB>600</SUB> with 4.967 ± 0.143. Of the transporter variants, <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-pelB-SP-PtNTT2 again reached the highest OD<SUB>600</SUB> and <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 the lowest. The results for both cultivations are summarized in Table 7. 
					
				<P class="figure subtitle"><B>Table 7: Final OD<SUB>600</SUB> values for all cultivations carried out in MOPS media with 1,32 mM K<SUB>2</SUB>HPO<SUB>4</SUB>. </B></P><TABLE style="margin:auto"><THEAD><TR><TH style="width: 60%;" class="header">Strain</TH><TH style="width: 20%;" class="header">Final OD<SUB>600, K<SUB>2</SUB>HPO<SUB>4</SUB></SUB> [-]</TH><TH style="width: 20%;" class="header">Final OD<SUB>600, ATP</SUB> [-]</TH></TR></THEAD><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) </TD><TD>2.923 ± 0.028</TD><TD>4.967 ± 0.143</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PtNTT2 </TD><TD>3.507 ± 0.048</TD><TD>3.673 ± 0.091</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 </TD><TD>1.537 ± 0.045</TD><TD>3.033 ± 0.028</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) </TD><TD>3.560 ± 0.011</TD><TD>3.347 ± 0.032</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(31-575) </TD><TD>3.797 ± 0.065</TD><TD>3.580 ± 0.006</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-pelB-SP-PtNTT2 </TD><TD>3.907 ± 0.018</TD><TD>3.710 ± 0.177</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 </TD><TD>3.307 ± 0.029</TD><TD>2.177 ± 0.007</TD></TR></TBODY></TABLE>

			The maximum specific growth rate was graphically determined for all cultures as shown in Figure 12. 

			<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 12: Graphical determination of the maximum specific growth rates for all cultures cultivated in MOPS media with 1.32 mM K<SUB>2</SUB>HPO<SUB>4</SUB>.</B></P></DIV>
						Based on the maximum specific growth rates, the minimum doubling time was calculated. The results are shown in Table 8. Once again, the strains expressing the native transporter and the transporter with a TAT signal peptide showed the weakest growth. The best growth characteristics were achieved by the strains expressing the truncated versions <I>Pt</I>NTT2(66-575) and <I>Pt</I>NTT2(31-575), and <I>Pt</I>NTT2 with a pelB  signal peptide. 				
				
				<P class="figure subtitle"><B>Table 8: Maximum specific growth rates and minimal doubling times of the cultivations in MOPS media with 1.32 mM K<SUB>2</SUB>HPO<SUB>4</SUB>.</B></P><TABLE style="margin:auto"><THEAD><TR><TH style="width: 60%;" class="header">Strain</TH><TH style="width: 20%;" class="header">µ<SUB>max</SUB> [h<SUP>-1</SUP>]</TH><TH style="width: 20%;" class="header">t<SUB>d</SUB> [h]</TH></TR></THEAD><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) </TD><TD>0.444 ± 0.053</TD><TD>1.561 ± 0.199</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PtNTT2 </TD><TD>0.499 ± 0.050</TD><TD>1.389 ± 0.100</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 </TD><TD>0.385 ± 0.044</TD><TD>1.800 ± 0.114</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) </TD><TD>0,568 ± 0.057</TD><TD>1.220 ± 0.100</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(31-575) </TD><TD>0.532 ± 0.022</TD><TD>1.303 ± 0.041</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-pelB-SP-PtNTT2 </TD><TD>0.549 ± 0.017</TD><TD>1.263 ± 0.031</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 </TD><TD>0.463 ± 0.028</TD><TD>1.497 ± 0.060</TD></TR></TBODY></TABLE>	
			The graphical determination of the maximum specific growth rates of the cultures cultivated in ATP supplemented media is shown in Figure 13. 	
				
				<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 13: Graphical determination of the maximum specific growth rates of all cultivations performed in MOPS media and 1 mM ATP. </B></P></DIV>
			The determined values for µ<SUB>max</SUB> and the minimal doubling times are shown in Table 9. 		

				<P class="figure subtitle"><B>Table 9: Maximum specific growth rates and minimal doubling times of the cultivations in MOPS media with 1 mM ATP.</B></P><TABLE style="margin:auto"><THEAD><TR><TH style="width: 60%;" class="header">Strain</TH><TH style="width: 20%;" class="header">µ<SUB>max</SUB> [h<SUP>-1</SUP>]</TH><TH style="width: 20%;" class="header">t<SUB>d</SUB> [h]</TH></TR></THEAD><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) </TD><TD>0.673 ± 0.012</TD><TD>1.030 ± 0.018</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PtNTT2 </TD><TD>0.600 ± 0.021</TD><TD>1.155 ± 0.035</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 </TD><TD>0.463 ± 0.035</TD><TD>1.497 ± 0.076</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) </TD><TD>0.644 ± 0.069</TD><TD>1.076 ± 0.107</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(31-575) </TD><TD>0.428 ± 0.091</TD><TD>1.620 ± 0.213</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-pelB-SP-PtNTT2 </TD><TD>0.518 ± 0.043</TD><TD>1.338 ± 0.083</TD></TR></TBODY><TBODY><TR><TD><I><I>E. coli</I></I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 </TD><TD>0.334 ± 0.047</TD><TD>2.075 ± 0.141</TD></TR></TBODY></TABLE>
			Based on the results of the two cultivations, the relative beneficial effect (RBE) of <I>Pt</I>NTT2 was calculated using Equation 2. 
			<DIV class="equation-line"><P class="equation-nr">2</P></DIV>
			
			For each measurement point <I>n</I>, the optical density at 600 nm of the cultivation of a transporter variant in ATP supplemented media (<I>OD<SUB>AT</SUB></I>) was divided by the optical density of the reference <I>E. coli</I> BL21(DE3) pSB1C3-PtNTT2 in ATP supplemented media (<I>OD<SUB>AR</SUB></I>). The same was done for the cultures in K<SUB>2</SUB>HPO<SUB>4</SUB> supplemented media(<I>OD<SUB>KT</SUB></I> and <I>OD<SUB>KR</SUB></I>). The value for ATP was then divided by the value for K<SUB>2</SUB>HPO<SUB>4</SUB>. The sum of the quotient for all measurement points <I>n</I> was then divided by <I>n</I> to obtain the final value for the relative beneficial effect of <I>Pt</I>NTT2 shown in Figure 14. The final error was calculated through error propagation of the standard error of each measured optical density.
		
			<DIV class="figure seventy"><P class="figure subtitle"><B>Figure 14: Relative beneficial effect of the different <I>Pt</I>NTT2 variants.</B> As expected, the native transporter variant shows the highest positive effect since it most likely also exhibits the highest activity. Surprisingly, the two truncated versions show a higher effect than the versions with a pelB and TAT signal peptide.</P></DIV>
			
			Of all transporter variants, the native <I>Pt</I>NTT2 showed the highest beneficial effect with a value of 1.360 ± 0.161. The second highest value was reached by <I>Pt</I>NTT2(31-575) with 1.294 ± 0.107. <I>Pt</I>NTT2(66-575) reached a value of 1.166 ± 0.105, pelB-SP-PtNTT2 one of 1.073 ± 0.114 and TAT-SP-PtNTT2 one of 0.732 ± 0.116. 	
			
			This data suggests that the expression of different <I>Pt</I>NTT2 variants, especially of the native <I>Pt</I>NTT2, is beneficial for the cell when cultivated in MOPS minimal media supplemented with ATP as the sole phosphate source. Given that the reference strain does not express <I>Pt</I>NTT2, the expression of <I>Pt</I>NTT2 must have a beneficial effect for the cells since they grow better compared to the reference in ATP when compared to the reference in K<SUB>2</SUB>HPO<SUB>4</SUB>. Therefore, the beneficial effect is larger than the metabolic burden associated with recombinant protein expression. Consequently, the transporter exhibits a function beneficial to the cell in ATP supplemented media, meaning it can facilitate the direct uptake of ATP from the media. Therefore, the proposed activity of <I>Pt</I>NTT2 in media supplemented with low concentrations of ATP could be verified. 	
			
			Consequently, the same experiment was conducted with MOPS minimal media supplemented with 10 mM ATP. The relative beneficial effects of the experiment are summarized in Figure 15.
			
			<DIV class="figure sixty"><P class="figure subtitle"><B>Figure 15: Relative beneficial effect of the different transporter variants when cultivated in MOPS minimal media supplemented with 10 mM ATP.</B> No substantial beneficial effect could be observed for any of the transporter variants. The highest beneficial effects were reached by <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2 (+17.2 % ± 7.2 %) and <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(31-575) (+14.0 % ± 4.7 %).</P></DIV>
			The highest beneficial effect was reached by TAT-SP-PtNTT2 with a value of +17.2 % ± 7.2 %, followed by <I>Pt</I>NTT2(31-575) with +14.0 % ± 4.7 %. All other strains did not show a substantial beneficial effect. These results suggest, that the activity of the different transporter variants is concentration dependent and that expression of <I>Pt</I>NTT2 in media supplemented with higher concentrations of ATP equalizes the effects of uptake and export of NTPs. Therefore, no substantial beneficial effect could be observed, but also no negative effect. 
			
			The experiment was repeated once again with MOPS media supplemented with 100 µM of iso-dC<SUP>m</SUP>TP and iso-dGTP. Substantial differences were observed for all transporter variants. <I>Pt</I>NTT2(66-575) reached the highest beneficial effect with +38 % ± 10 %. All other transporter variants reached showed a negative effect compared to the reference. This confirms that ATP is exported in presence of the unnatural nucleotides, leading to a net loss of ATP and inhibition of growth. Therefore, it can be concluded that the native transporter variant <I>Pt</I>NTT2 has the highest activity towards iso-dC<SUP>m</SUP>TP and iso-dGTP, followed by <I>Pt</I>NTT2(31-575).
			
			<DIV class="figure sixty"><P class="figure subtitle"><B>Figure 16: Relative beneficial effect of the best <I>Pt</I>NTT2 variants cultivated in MOPS media supplemented with 100 µM of iso-dC<SUP>m</SUP>TP and iso-dGTP each.</B> Substantial differences can be observed for all transporter variants, with <I>Pt</I>NTT2(66-575) reaching the highest beneficial effect +38 % ± 10 %. All other transporter variants reached showed a negative effect compared to the reference, which means that ATP is exported in exchange for iso-dC<SUP>m</SUP>TP and iso-dGTP, leading to a net loss of ATP.</P></DIV>
			
			The supernatants of the cultures cultivated in MOPS media supplemented with 1 mM and 10 mM ATP were further analyzed with LC-MS. Therefore, the cultures were centrifuged, and the supernatant analyzed with LC-MS. AMP, ADP and ATP were measured, but only AMP was substantially above the detection limit in nearly all samples. AMP was quantified based on the standard curve shown in Figure 18. ADP and ATP could not be quantified. The results are shown in Figure 17. 
			
			<DIV class="figure large"><P class="figure subtitle"><B>Figure 17: : Results of the LC-MS analysis of the supernatants of the cultures cultivated in MOPS media supplemented with 1 mM and 10 mM ATP.</B> Measured AMP concentrations were standardized to the corresponding final optical densities.</P></DIV>
			AMP could be detected in all samples except for the supernatants of the strains expressing pelB-SP-PtNTT2 and TAT-SP-PtNTT2 cultivated in MOPS media supplemented with 10 mM ATP. 
			
			<DIV class="figure sixty"><P class="figure subtitle"><B>Figure 18: Standard curve for AMP using 10 mM, 1 mM, 0.1 mM, 0.01 mM and 0.001 mM of AMP. </B> The standard curve was used to quantify AMP in the supernatant of the cultivations carried out in ATP supplemented MOPS media. </P></DIV>
			
			The samples from the cultivation performed in MOPS media supplemented with 1 mM ATP, the highest concentration was reached by <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575) with 0.387 mM, followed by <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2 0.294 mM. Of the samples of the cultivation carried out in MOPS media supplemented with 10 mM, the highest AMP concentration was measured for the native transporter variant with a value of 4.201 mM. The second highest AMP concentration was measured for <I>Pt</I>NTT2(31-575) with 3.886 mM of AMP, followed by <I>Pt</I>NTT2(66-575) with a concentration of 2.910 mM. 
			These measurements verify that ATP, ADP or AMP are exported in exchange for ATP, which in case of ATP and ADP are consequently converted into AMP. Given that the measurements could only be performed once, not conclusion regarding the significance of the measured concentrations can be drawn. 


Concluding, we were able to verify the function of <I>Pt</I>NTT2 without using radioactively labeled nucleotides. The native transporter variant shows the highest activity in all experiments. Uptake of the unnatural nucleotides was verified by determination of the RBE of the different transporter variants when cultivated in MOPS media supplemented with the unnatural nucleotides. Since ATP is constantly exported in exchange for iso-dC<SUP>m</SUP>TP and iso-dGTP, leading to substantial negative influence on the growth of the cultures. 
		</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Subcellular Localization of <I>Pt</I>NTT2</H3><ARTICLE>
			To investigate the subcellular localization of the different <I>Pt</I>NTT2 variants, GFP fusion proteins were constructed as listed in Table 1. Each <I>Pt</I>NTT2 variant was tagged with a cMyc epitope tag as a linker and GFP (BBa_E0040). The cells were cultivated and prepared for microscopy as described in the <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">methods section</A>. Microscopy was performed using the Zeiss LSM 700. The pictures shown in Figure 19 were taken at 100x magnification and clearly show the subcellular localization of the different <I>Pt</I>NTT2 variants.
		
		<DIV class="figure large"><P class="figure subtitle"><B>Figure 19: Confocal laser scanning microscopy of the different <I>Pt</I>NTT2 variants fused to GFP (BBa_E0040). </B> The pictures were taken with 100x magnification and show from <B>A</B> to <B>E</B>: <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2, <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(66-575), <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-PtNTT2(31-575), <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-pelB-SP-PtNTT2 and <I>E. coli</I> BL21(DE3) pSB1C3-PlacUV5-TAT-SP-PtNTT2.</P></DIV>
		All <I>Pt</I>NTT2 variants can be localized within the membrane, since the fluorescence signals of the GFP tag are concentrated there. Judging from the pictures, the variant with the pelB signal peptide is weakly integrated into the inner membrane, which would explain the good growth characteristics and the weak functionality of this variant. The TAT-signal peptide variant, which showed bad growth characteristics and no substantial function, seems to be located within the cell envelope nonetheless. Given that peptides containing the TAT signal peptide are usually folded within the cytoplasm and then secreted via the twin-arginine translocation pathway, this could be a possible explanation for the non-existent function of the TAT variant. All other variants that also showed a detectable function are successfully integrated into the membrane. The variants without a signal peptide are also located within the membrane, which is not surprising, given that the integration of membrane proteins does not necessarily require a signal peptide (Facey and Kuhn, 2004) . <I>Pt</I>NTT2(66-575) seems to be better integrated into the membrane than any other variant. The native <I>Pt</I>NTT2, and <I>Pt</I>NTT2(31-575) are integrated in a similar scale, but a little bit weaker compared to <I>Pt</I>NTT2(66-575). These results are consistent with the results of the verification of the function of <I>Pt</I>NTT2, since all variants that showed a relative beneficial effect and also the highest AMP concentrations in the supernatant do integrate the transporter into the membrane. For the TAT signal peptide variant, it is most likely that the transporter is integrated incorrectly, leading to a correct subcellular localization while lacking the correct function.
		</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Isolation of <I>Pt</I>NTT2 from the Inner Membrane </H3><ARTICLE>
		For further analysis of <I>Pt</I>NTT2 the isolation of the transporter from the cell was attempted. This task proved to be challenging, given that <I>Pt</I>NTT2 comprises ten transmembrane domains and is therefore highly hydrophobic. For that reason, several different methods and procedures were tested to achieve the best results. 
		The first two attempts were based on the isolation of the periplasmic protein fraction through cold osmotic shock and fast cell lysis followed by analysis via SDS PAGE. Unsurprisingly, both methods did not lead to the desired results, since no bands for <I>Pt</I>NTT2 could be observed. Figure 20 shows the SDS PAGE of the samples prepared using the fast cell lysis for SDS PAGE. 
		
		<DIV class="figure sixty"><P class="figure subtitle"><B>Figure 20: SDS-PAGE of the GFP-fusion constructs of <I>Pt</I>NTT2</B> The cells were prepared using the fast cell lysis for SDS PAGE protocol. <I>E. coli</I> BL21(DE3) and <I>E. coli</I> BL21(DE3) pSB1C3-PtNTT2 were used as negative controls. Unsurprisingly, no thick band can be observed around 90.3 kDa, which would be the size of <I>Pt</I>NTT2-cMyc-GFP. No bands can be observed for the other <I>Pt</I>NTT2 variants. </P></DIV>


		
		A second gel was used for a Western Blot using an anti-GFP antibody. The result is shown in Figure 21.

		<DIV class="figure sixty"><P class="figure subtitle"><B>Figure 21: Western Blot of the samples prepared using the fast cell lysis for SDS PAGE protocol. </B> An anti-GFP antibody was used for the detection of <I>Pt</I>NTT2-cMyc-GFP variants. E . coli BL21(DE3) and <I>E. coli</I> BL21(DE3) pSB1C3-PtNTT2 were used as negative controls. Much unspecific binding of the anti-GFP antibody could be observed, which is not surprising given that the entire proteome of the cells was analyzed. Thick band can be observed for <I>Pt</I>NTT2-cMyc-GFP, <I>Pt</I>NTT2(66-575)-cMyc-GFP, <I>Pt</I>NTT2(31-575)-cMyc-GFP and <I>Pt</I>NTT2(pelB)-cMyc-GFP around 35 kDa. This indicates that only the cMyc-GFP linker was detected and that the linker might be cleaved of from <I>Pt</I>NTT2 due to the high difference in hydrophobicity. </P></DIV>

		In a second approach, the cells were lysed using a nitrogen cooled Precellys Homogenizer. After three cycles, cells were centrifuged at 4 °C and the membrane fraction was isolated as described in the <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">methods section</A>. After boiling the membrane fraction, the samples were loaded onto two SDS PAGEs, one of which was consequently used for a Western Blot, if the fusion proteins were used. For the Western Blot, two different antibodies were tested, an anti-GFP antibody and an anti-cMyc antibody. Figure 22 shows the SDS-PAGE after isolation of the membrane fraction. Again, no clear difference between the negative controls and the samples could be observed around 90 kDa. 
		
		<DIV class="figure sixty"><P class="figure subtitle"><B>Figure 22: SDS PAGE performed with the isolated membrane fractions.</B> The cMyc-GFP fusion proteins were used, which should be visible around ~90 kDa, differing slightly based on the <I>Pt</I>NTT2 variant used. No bands could be observed around 90 kDa, which was subsequently proofed by performing a Western Blot. </P></DIV>
		
		The Western Blot (figure 23) performed with the same samples verified these results, since only bands around ~28 kDa could be observed for all samples except for <I>Pt</I>NTT2(TAT)-cMyc-GFP and not for the negative controls. These bands can again be associated with the cMyc-GFP tag, providing further evidence that the tag can be cleaved of from the transporter. Again, no band could be observed for <I>Pt</I>NTT2-cMyc-GFP, indicating that <I>Pt</I>NTT2 with a TAT signal peptide is weakly integrated into the membrane, which would explain the lacking function of this variant. The weak growth characteristics of the strain producing <I>Pt</I>NTT2 with a TAT signal peptide would therefore most likely be caused by accumulation of the protein within the cell. 
		
		<DIV class="figure sixty"><P class="figure subtitle"><B>Figure 23: Western Blot of the isolated membrane fractions of the strains expressing the cMyc-GFP fusion proteins.</B> Thick bands can be observed around 28 kDa for all samples except for <I>Pt</I>NTT2-cMyc-GFP with a TAT signal peptide. The negative controls do not show the same band, but some unspecific binding of the anti-GFP antibody could be observed. Compared to the previous Western Blot, unspecific binding was substantially reduced. These results indicate, that the linker is most likely separated from the transporter either during the isolation process or already within the cell. This would be no surprise, given that the transporter is highly hydrophobic while the linker is hydrophilic. </P></DIV>
		
		To investigate if the transporter can be visualized on a gel using an anti-cMyc antibody, the same experiment was repeated, and the Western Blot performed using an anti-cMyc antibody. In case that the cMyc epitope tag remains attached to the transporter during possible cleavage events, it should be possible to visualize the transporter on a gel using the cMyc antibody.
		
		<DIV class="figure sixty"><P class="figure subtitle"><B>Figure 24: Western Blot of the isolated membrane fraction using an anti-cMyc antibody. </B> Again, fragments can be observed around 35 kDa for all samples except for <I>Pt</I>NTT2-cMyc-GFP with a TAT signal peptide. No bands can be observed for the full construct, but a very weak band can be seen between 55 and 70 kDa for the fusion protein of the native transporter. </P></DIV>
		
		Once again, bands could be observed for all transporter variants except for the TAT signal peptide variants around 35 kDa. One very weak can be observed for the fusion protein of the native transporter between 55 and 70 kDa. 
All these results show how difficult the isolation of integral membrane proteins can be. A number of reasons can be brought forward to explain why isolation of <I>Pt</I>NTT2 from the membrane proved to be so challenging. First of all, <I>Pt</I>NTT2 has 10 transmembrane domains, making it a highly hydrophobic protein. To find a correct detergent to solubilize the protein from the isolated membrane fraction also proves challenging. The European Molecular Biology Laboratory (EMBL) lists dozens of possible detergents on their site (European Molecular Biology Laboratory - Protein Purification). To find a best suited detergent is also a challenge. Another reason why the visualization of membrane proteins on SDS PAGEs and Western Blots is challenging is that membrane proteins tend to aggregate when boiled and that they often exhibit “gel shifting” (Rath <I>et al.</I>, 2009). For that reason, another SDS PAGE was run using the isolated membrane fractions of the <I>Pt</I>NTT2 constructs without cMyc-GFP tag, but this time the samples were loaded onto the gel without previous boiling. 


		<DIV class="figure sixty"><P class="figure subtitle"><B>Figure 25: SDS PAGE of the isolated membrane fraction without previous boiling</B> No thick bands can be observed around 70 kDa. Slightly above 100 kDa, bands can be observed for all <I>Pt</I>NTT2 variants but not for the negative controls. But given that the samples ran quite different on the gel compared to the boiled samples, no definite conclusion can be drawn from this gel. </P></DIV>
		
		
		Based on these findings it can be concluded that <I>Pt</I>NTT2 is located in the inner membrane of <I>E. coli</I> and that isolation of the entire protein remains challenging. Since it could be shown that the tags are present in the inner membrane fraction, it is very likely that the entire construct <I>Pt</I>NTT2-cMyc-GFP could simply not be solubilized enough. In all Western Blots, the tag of <I>Pt</I>NTT2-cMyc-GFP with a TAT signal peptide was far weaker compared to the other samples, indicating that <I>Pt</I>NTT2 with a TAT signal peptide is weakly integrated into the membrane. This would also concur with our previous results and would explain the lacking function of this construct. 
		</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> References</H3><ARTICLE><B>EMBL Protein Purification - Extraction and Clarification - Solubilisation of Membrane Proteins.</B> https://www.embl.de/pepcore/pepcore_services/protein_purification/extraction_clarification/solubilisation_membrane_proteins/index.html, Date Accessed: 1st of November, 2017
		</ARTICLE><ARTICLE><B>Facey, S.J. and Kuhn, A. </B>(2004). Membrane integration of <I>E. coli</I> model membrane proteins. Biochim. Biophys. Acta - Mol. Cell Res. 1694: 55–66.
		</ARTICLE><ARTICLE><B>Käll, L., Krogh, A., and Sonnhammer, E.L.L. </B> (2007). Advantages of combined transmembrane topology and signal peptide prediction-the Phobius web server. Nucleic Acids Res. 35: 429–432.
		</ARTICLE><ARTICLE><B>Rath, A., Glibowicka, M., Nadeau, V.G., Chen, G., and Deber, C.M. </B>(2009). Detergent binding explains anomalous SDS-PAGE migration of membrane proteins. Proc. Natl. Acad. Sci. <B>106</B>: 1760–1765.
		</ARTICLE><ARTICLE><B>Zhang, Y., Lamb, B.M., Feldman, A.W., Zhou, A.X., Lavergne, T., Li, L., and Romesberg, F.E.</B> (2017). A semisynthetic organism engineered for the stable expansion of the genetic alphabet. Proc. Natl. Acad. Sci. 114: 1317–1322.
		</ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>