Team:Bielefeld-CeBiTec/Project/unnatural base pair
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Unnatural Base Pair Overview
The expansion of the genetic code is not solely confined to the incorporation of non-canonical amino acids into peptides,  it also means the incorporation of unnatural base pairs (UBPs) into the DNA. The introduction of UBPs into the DNA results in a semisynthetic organism which has been a long-desired  achievement in the field of synthetic biology. The research on UBPs bore several unnatural bases with different chemical properties.   For our project we decided to use isoguanine (isoG) and 5-methyl-isocytosine (isoCm) to expand the genetic code.  The establishment of novel   protocols for standard lab methods like   PCR and transformation were important in order to work with UBPs. Also, detection methods of UBPs in the DNA were required.
We were able to establish Oxford Nanopore Sequencing for   isoG and isoCm and developed a software to handle the data of unnatural bases.   We also developed the software M.A.X. that  suggests a set of restriction enzymes based on the neighboring DNA sequence of the UBP. With these restriction enzymes,  a simple approach for retention or loss of the UBP in vitro is available. Furthermore,   we developed a two-plasmid-system for UBP retention in vivo.  These plasmids encode for the nucleotide transporter PtNTT2 ,   Cas9 and a set of sgRNAs. PtNTT2, taken from the algae Phaeodactylum tricornutum,   ensures the uptake of the unnatural triphosphates from the media. Plasmids that lose the UBP get cut by Cas9, because the sgRNAs bind to   every DNA sequences that carries a mutation instead of the UBP. This results in the preservation of the UBP in vivo.       We also made a step towards an autonomous semisynthetic organism.  From literature, we identified a natural source of the "unnatural" base isoG, the plant Croton tiglium .   We conducted a transcriptome-based investigation to identify potential enzyme candidates responsible for the biosynthesis of isoG,   which we then heterologously expressed in E. coli and characterized in vitro.
