<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_translational_system_library_and_selection skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/translational system/library and selection</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/2/2e/T--Bielefeld-CeBiTec--title-img-colonies.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Library and Selection
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Short Summary </H2><DIV class="article">
		The incorporation of a non-canonical amino acid (ncAA) into the nascent polypeptide chain requires a <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">tRNA/aminoacyl-tRNA synthetase (tRNA/aaRS)</A> pair which is highly specific for each amino acid. Mechanistically the aaRS has to recognize, accept, and bind the amino acid and transfer it in the second step onto the tRNA.
		For our ncAA we want to develop a tRNA/aaRS pair which is specific for the ncAA as well as orthogonal to the canonical amino acid tRNA/aaRS pairs.Therefore a library of mutated<A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">orthogonal tRNA/aaRS</A> pairs is generated, based on the commonly used tyrosyl- and pyrrolysyl- tRNA/synthetases. For the adaption of the codon recognition and amino acid binding, the candidate aaRS undergoes numerous rounds of positive and negative selection. The selection results in a synthetase which can be expressed efficiently in <I>Escherichia coli</I> and is able to reliable incorporate specifically our ncAA.
		</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Tyrosyl- and Pyrrolysyl-tRNA Synthetases </H2><H4> Tyrosyl-tRNA Synthetase </H4><DIV class="article">
		There are several non-canonical amino acids which have been incorporated into peptides and proteins in <I> E. coli </I>. Among others, it was possible to incorporate ncAA with heavy atoms, keto and alkaline side chains and photo crosslinking capability(Zhang<I>   et al.,</I>2005).
For the binding of the ncAA, an adaption of the synthetase is necessary. This is done by introducing mutations in several regions of the associated  synthetase, leading to a conformational change of the binding pocket and thus in the binding of a new amino acid. 
The tyrosine synthetase is the first orthogonal <I> E. coli </I> tRNA/aaRS pair generated from archaea and the best known so far for the incorporation of ncAAs  (Wang<I>   et al,.</I>2001). This tyrosine synthetase (TyrRS) has a small anticodon loop binding domain (Steer<I>   et al,.</I>1999). Therefore, the change of the recognition in order to change the anticodon loop of its cognate tRNA to CUA is not as difficult as with a larger anticodon loop binding domain (Steer<I>   et al.,</I>1999). The binding site is located deep inside a small pocket of TyrRS (Tian<I>   et al.,</I>2004). Due to this it is assumed that already very small conformational changes result in a larger or smaller binding pocket with an altered binding interaction with the ligand. Therefore, in this protein, only few changes in the coding sequence should be necessary to incorporated ncAA. (Zhang<I>   et al.,</I>2005).
In detail, the TyrRS binding pocket is highly hydrophilic, preventing the binding of phenylalanine, whose structure is similar to the one of tyrosine (Goldgur<I>   et al.,</I>1997). Changing the characteristic of the binding pocket to become more hydrophobic, could favor the binding of phenylalanine derivates (Goldgur<I>   et al.,</I>1997). In the middle of the negative charged binding pocket (Zhang<I>   et al.,</I>2005), there is an aspartic acid on position 158. The entrance of the binding pocket is positioned between two glutamic acids at positions 36 and 172. Upon tRNA binding, the aspartic amino acid Asp158 forms two hydrogen bonds with the glycine at position 34 (Zhang<I>   et al.,</I>2005).

		</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 1: Tyrosine-binding site in apo <I> M. jannaschii TyrRS</I></B> The electrostatic distribution around the tyrosine-binding site is shown. Positive potential is blue (10 mV), neutral potential (0 mV) is white, and negative potential (−10 mV) is red (Zhang<I> et al.,</I>2005).</P></DIV><DIV class="article">
		When the ligand tyrosine is bound, hydrogen bonds to Tyr32, Asp158, Glu36, Gln173, Tyr151 and Gln155 are formed, resulting in slight movements of the side chains within the tyrosine-binding pocket (Zhang<I>   et al.</I>2005). Beside the binding pocket, other domains are also affected by the binding of tyrosine. For example, the loop 73-83, positioned at the entrance of the binding pocket, provides a hydrophobic lid over the binding pocket in reaction to the conformational changes. This is supposed to have the effect of separating the activated tyrosine from water during the catalytic reaction (Zhang<I>   et al.,</I>2005).
		</DIV><H4> Pyrrolysyl-tRNA Synthetase </H4><DIV class="article">
		
		In nature, one orthogonal aminoacyl-tRNA synthetase pair incorporates the 22<SUP>nd</SUP> proteinogenic amino acid pyrrolysine (Pyl). Interestingly, Pyl is assigned to the amber stop codon (UAG). Thus, it is incorporated naturally through an endogenous nonsense codon suppression system(Burke <I>et al.</I>, 1998; Paul <I>et al.</I>, 2000). It was discovered in the archaeal order of <I>Methanosarcinales</I> (Srinivasan <I>et al.</I>, 2002). Due to this specific property, the aminoacyl-tRNA synthetase/tRNA<SUP>Pyl</SUP> pair can be used to expand the genetic code. The pyrrolysyl synthetase (PylRS) is a class II synthetase with a highly conserved class II catalytic core at the C-terminus of around 270 amino acids with a Rossmann fold for ATP-binding. PylRS forms an obligate dimer with an active site in each subunit (Kavran <I>et al.,</I>, 2007; Nozawa <I>et al.</I>, 2009). Interestingly, the N-terminus of the synthetase shows no significant similarity to any known protein domains and its length is variable. The N-terminus leads to aggregation of full-length PylRS due to its high insoluble properties (Yanagisawa <I>et al.</I>, 2006; Jiang and Krzycki, 2012). Therefore a truncated version containing the catalytic core of the synthetase is used (residues 185-454) (Kavran <I>et al.</I>, 2007; Yanagisawa <I>et al.</I>, 2008). In <I>Methanosarcina mazei</I> the catalytic core of PylRS has a deep hydrophobic pocket where A302, L305, Y306, L309, N346, C348 and W417 form a large cavity for the binding of the Pyl side chain ((4R,5R)-4-methyl-pyrroline-5-carboxylate) (Kavran <I>et al.</I>, 2007; Yanagisawa <I>et al.</I>, 2008a). These hydrophobic interactions are relatively non-specific. Given that PylRS does not have an editing domain for hydrolysation of misacylated tRNAs, this synthetase shows high substrate side chain promiscuity (Wan <I>et al.</I>, 2014). On top of that, Pyl is quite large in comparison to the other endogenous canonical amino acids, thus there was no selection pressure for PylRS to stringently recognize Pyl(Wan <I>et al.</I>, 2014). Therefore structurally similar and dissimilar non-canonical amino acids were shown to be incorporated using PylRS (Mukai <I>et al.</I>, 2008; Polycarpo <I>et al.</I>, 2006; Yanagisawa <I>et al.</I>, 2008b). The residues Y384 and N346 in the active site of PylRS form hydrogen bond interactions with the α-amine of Pyl (Yanagisawa <I>et al.</I>, 2008a; Kavran <I>et al.</I>, 2007). But PylRS also has a high tolerance for α-hydroxy acids (Kobayashi <I>et al.</I>, 2009; Li <I>et al.</I>, 2012). 

		</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 2: Pyrrolisine-binding site of wild type <I>Methanosarcina mazei</I> PylRS. </B> Structural composition of the active site of PylRS with activated Pyl (Pyl-AMP) as a substrate (Guo <I>et al.</I>, 2014). </P></DIV><DIV class="article">
		Most interestingly, PylRS does not show a direct synthetase-anticodon interaction as other synthetases do. Even a mutation at the anticodon of tRNA<SUP>Pyl</SUP> does not influence aminoacylation activity of  PylRS (Ambrogelly <I>et al.</I>, 2007; Yanagisawa <I>et al.</I>, 2008a). This characteristic can be used to incorporate non-canonical amino acids using unnatural base pairs. The only drawback using PylRS is its low translational efficiency and catalytic activity compared to other synthetases (Guo <I>et al.</I>, 2014). Because in nature PylRS is rarely and selectively expressed, a high catalytic and translational efficiency is not needed (Srinivasan <I>et al.</I>, 2002). Every change in the native PylRS does lower the catalytic and translational activity (Guo <I>et al.</I>, 2014).
		</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Generating the Library </H2><DIV class="article">
		As shown, it is possible to tune tRNA synthetases into incorporating ncAA. We want to achieve this adaptation process by creating a library of tRNA synthetases and obtain the tRNA synthetase accepting our ncAA by a selection mechanism.
		
To pass as many mutated synthetases as possible through the cycle of the two selections for the adaption of the specific ncAA, a large library of different synthetase versions has to be created. The large library of random variants of sequences can be generated by side specific mutagenesis using randomized primers with the NNK scheme to avoid the incorporation of all stop codons, but one (Sieber <I> et al.,</I>2015). N codes for the bases A,C,G,T and K codes for the bases G and T. This leads to the use of 32 codons, canonical amino acid as well as one stop is encoded through the NNK scheme (Yuval<I>   et al.</I>,2011). By comparison to other randomization, the NNK scheme has a relatively low ratio between the most common and rarest coded amino acid (3/32 vs. 1/32) (Yuval<I>   et al.,</I>2011).
By the randomization of more than one position, numerous more sequence variants are possible. If one position is randomized, 130 variants of the sequences are needed to attain a 0.99 probability of discovering the best variant (Yuval<I>   et al.,</I>2011). At the same time, for the randomization of three positions, there are statistically 102,478 variants needed to discover the best variant. In this context, the probability of full codon coverage changes from 0.82 % for one randomizes position to 3.25 • 10<SUP>-36</SUP> % for three randomized position (Yuval<I>   et al.,</I>2011). Regarding the fact that not the full coverage is essential, but rather the discovery of the best variant, the NNK scheme is a solid method for the generation of a synthetase library for the selection of the best ncAA binding site.
 
		</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Selection </H2><ARTICLE>
		For the translational incorporation of ncAAs, a corresponding tRNA/aminoacyl-tRNA synthetase (tRNA/aaRS) pair is essential. The tRNA/aaRS pair must recognize and deliver the novel amino acid in response to a unique codon that does not encode any of the 20 canonical amino acids. Therefore, a modification of the tRNA for the recognition of the amber-codon and a modification of the synthetase for the specific binding of a new amino acid is necessary.
To ensure specificity and orthogonality, a tRNA/aaRS pair from a distinct microbial organism is used to generate tRNA and aaRS libraries which are used to evolve a suitable tRNA/aaRS pair in iterative negative and positive selection steps (Wang<I>   et al.,</I>2006). Therefore the tRNA and the aaRS each have to be separately mutated and selected.
 
		</ARTICLE><H4> Modification of the aaRS </H4><ARTICLE>
		In the negative selection step, the synthetase (aaRS) library is tested for orthogonality of the exogenous tRNAs towards the endogenous translation system. The expression of a     <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201406">   barnase    </A> gene is prevented by an amber stop codon in its coding region. If the exogenous synthetase binds an canonical amino acid, the amber stop codon preventing the expression of barnase is suppressed, which leads to cells death. Therefore, only cells with orthogonal or non-functional synthetases survive (Wang<I>   et al.</I>2006, Liu<I>   et al.,</I>2010).
The orthogonal aaRS are then subjected to positive selection, in which the tRNA/aaRS pairs are tested for their functionality. The expression of an antibiotic resistance gene is prevented by an amber stop codon. Only if the exogenous aaRS incorporates any amino acid, the amber stop codon is suppressed and the cell attains antibiotic resistance. Only cells which own a functional aaRS survive. (Wang<I>   et al.</I>2006, Liu<I>   et al.,</I>2010). By adding a ncAA to the medium, aaRS that specifically use this ncAA to charge their tRNA can be selected.
				
		</ARTICLE><DIV class="figure large"><P class="figure subtitle"><B>Figure 3: Adaption of an orthogonal aminoacyl-synthetase through positive and negative selection cycles.</B> The positive-selection plasmid (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201900">BBa_K2201900</A> in pSB3T5)contains a kanamycin resistance with amber stop codons. After cotransformation with the library plasmid only cells which contain a synthetase which incorporates any amino acid in response to the amber codon survive. All surviving clones contain an aaRS that incorporates the target ncAA or any endogenous amino acid. The negative selection plasmid (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201901">BBa_K2201901</A> in pSB3T5) is used for the selection for the specificity of the clones. The target ncAA is not supplemented in the media. When incorporating endogeneous amino acids, the barnase is expressed through an amber codon and the cell dies. </P></DIV><H4> Modification of the tRNA </H4><ARTICLE>
		The selection of a proper variant of the tRNA follows a similar scheme like the adaption by selection of the aaRS. Regarding the unnatural base pairs (UBP), the recognition of the unnatural bases through the tRNA for the incorporation of the non-canonical amino acids requires significant modifications of the tRNA. The recognition of the amber stop codon has already been achieved in other laboratories (Wang<I>   et al.,</I>2006, Liu<I>   et al.,</I>2010). Therefore, a selection cycle of negative and positive selections like to the one of the aaRS, mentioned above, was performed. The main difference is the exchange of the tRNA by the aaRS on the plasmids used for the selection (Wang<I>   et al.,</I>2006, Liu<I>   et al.,</I>2010). The outcome is a tRNA, only able to recognize the amber codon and at the same time recognizes its orthogonal synthetase.

The combination of the tRNA and aaRS modification results in an orthogonal tRNA/aaRS pair capable of specifically incorporating a certain ncAA if the amber codon is present.
		</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> References </H2><ARTICLE><B>Yuval Nov</B>, (2011). When Second Best Is Good Enough: Another Probabilistic Look at Saturation Mutagenesis.<B> Applied and Environmental Microbiology 099-2240/12/12.00.</B><B>Zhang Y, Wang L, Schultz PG, Wilson IA.</B> (2005). Crystal structures of apo wild-type M. jannaschiityrosyl-tRNA synthetase (TyrRS) and an engineeredTyrRS specific for O-methyl-L-tyrosine. <B> Protein Sci 14(5): 1340–1349 </B><B>Wang L, Xie J, Schultz PG. </B> (2006). Expanding the Genetic Code. <B> Annu. Rev. Biophys. Biomol. Struct. 2006;35:225–49. </B><B>Liu CC, Schultz PG.</B> (2010). Adding New Chemistries to the Genetic Code. <B> Annu. Rev. Biochem. 2010;79:413–44.</B><B>Ambrogelly, A., Gundllapalli, S., Herring, S., Polycarpo, C., Frauer, C., and Söll, D. </B> (2007). Pyrrolysine is not hardwired for cotranslational insertion at UAG codons. Proc. Natl. Acad. Sci. U. S. A. 104: 3141–3146.<B>Burke, S.A., Lo, S.L., and Krzycki, J.A. </B> (1998). Clustered Genes Encoding the Methyltransferases of Methanogenesis from Monomethylamine. J. Bacteriol. 180: 3432–3440. <B>Guo, L.-T., Wang, Y.-S., Nakamura, A., Eiler, D., Kavran, J.M., Wong, M., Kiessling, L.L., Steitz, T.A., O’Donoghue, P., and Söll, D. </B> (2014). Polyspecific pyrrolysyl-tRNA synthetases from directed evolution. Proc. Natl. Acad. Sci. U. S. A. 111: 16724–16729. <B>Jiang, R. and Krzycki, J.A. </B> (2012). PylSn and the Homologous N-terminal Domain of Pyrrolysyl-tRNA Synthetase Bind the tRNA That Is Essential for the Genetic Encoding of Pyrrolysine. J. Biol. Chem. 287: 32738–32746. <B>Kavran, J.M., Gundllapalli, S., O’Donoghue, P., Englert, M., Söll, D., and Steitz, T.A. </B> (2007). Structure of pyrrolysyl-tRNA synthetase, an archaeal enzyme for genetic code innovation. Proc. Natl. Acad. Sci. U. S. A. 104: 11268–11273. <B>Kobayashi, T., Yanagisawa, T., Sakamoto, K., and Yokoyama, S. </B> (2009). Recognition of non-alpha-amino substrates by pyrrolysyl-tRNA synthetase. J. Mol. Biol. 385: 1352–1360. <B>Li, Y.-M., Yang, M.-Y., Huang, Y.-C., Li, Y.-T., Chen, P.R., and Liu, L. </B> (2012). Ligation of Expressed Protein α-Hydrazides via Genetic Incorporation of an α-Hydroxy Acid. ACS Chem. Biol. 7: 1015–1022. <B>Mukai, T., Kobayashi, T., Hino, N., Yanagisawa, T., Sakamoto, K., and Yokoyama, S. </B> (2008). Adding l-lysine derivatives to the genetic code of mammalian cells with engineered pyrrolysyl-tRNA synthetases. Biochem. Biophys. Res. Commun. 371: 818–822. <B>Nozawa, K., O’Donoghue, P., Gundllapalli, S., Araiso, Y., Ishitani, R., Umehara, T., Söll, D., and Nureki, O. </B> (2009). Pyrrolysyl-tRNA synthetase-tRNA(Pyl) structure reveals the molecular basis of orthogonality. Nature 457: 1163–1167. <B>Paul, L., Ferguson, D.J., and Krzycki, J.A. </B> (2000). The Trimethylamine Methyltransferase Gene and Multiple Dimethylamine Methyltransferase Genes of Methanosarcina barkeri Contain In-Frame and Read-Through Amber Codons. J. Bacteriol. 182: 2520–2529. <B>Polycarpo, C.R., Herring, S., Bérubé, A., Wood, J.L., Söll, D., and Ambrogelly, A. </B> (2006). Pyrrolysine analogues as substrates for pyrrolysyl-tRNA synthetase. FEBS Lett. 580: 6695–6700. <B>Srinivasan, G., James, C.M., and Krzycki, J.A. </B> (2002). Pyrrolysine Encoded by UAG in Archaea: Charging of a UAG-Decoding Specialized tRNA. Science 296: 1459–1462. <B>Wan, W., Tharp, J.M., and Liu, W.R. </B> (2014). Pyrrolysyl-tRNA Synthetase: an ordinary enzyme but an outstanding genetic code expansion tool. Biochim. Biophys. Acta 1844: 1059–1070. <B>Yanagisawa, T., Ishii, R., Fukunaga, R., Kobayashi, T., Sakamoto, K., and Yokoyama, S. </B> (2008a). Crystallographic studies on multiple conformational states of active-site loops in pyrrolysyl-tRNA synthetase. J. Mol. Biol. 378: 634–652. <B>Yanagisawa, T., Ishii, R., Fukunaga, R., Kobayashi, T., Sakamoto, K., and Yokoyama, S. </B> (2008b). Multistep engineering of pyrrolysyl-tRNA synthetase to genetically encode N(epsilon)-(o-azidobenzyloxycarbonyl) lysine for site-specific protein modification. Chem. Biol. 15: 1187–1197. <B>Yanagisawa, T., Ishii, R., Fukunaga, R., Nureki, O., and Yokoyama, S. </B> (2006). Crystallization and preliminary X-ray crystallographic analysis of the catalytic domain of pyrrolysyl-tRNA synthetase from the methanogenic archaeon Methanosarcina mazei. Acta Crystallograph. Sect. F Struct. Biol. Cryst. Commun. 62: 1031–1033. <B>Sieber, Hare, Hofmann, Trepel </B>, (2015), Biomathematical Description of Synthetic Peptide Libraries, <B> PLoS One. 2015; 10(6): e0129200. </B></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>