Team:Bielefeld-CeBiTec/Project/Overview
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Aim of the Project
In the following we want to describe the intention and aims of the project, the results overview can be found here .
Currently, protein design is limited to 20 to 22 canonical amino acids. Since the chemical abilities of these canonical amino acids are restricted, a lot of interesting functions are not applicable in protein design. We want to change that by enabling the incorporation of non-canonical amino acids with a broad range of additional abilities . There is only one problem: The DNA encodes proteins using the four nucleobases adenine, thymine, cytosine and guanosine in base triplets. Due to the fact that all possible combinations of these natural triplets already encode for one of the canonical amino acid or a translational stop, we need to repurpose one of the existing codons or incorporate a new base to create new codons in E. coli.
We address this challenge in two different ways. The repurposing of a codon for the incorporation of a non-canonical amino acid (ncAA) is possible using the rarely used amber stop codon UAG or other rarely used codons like the leucine codon CUA. For this approach, an orthogonal tRNA-aminacyl synthetase (aaRS) is required, which can charge the ncAA to the tRNA. We want to design a library of aaRS and a suitable selection system to select the aaRS which incorporate the amino acids of interest. To demonstrate this tool, we want to synthetize a novel amino acid, which could build a specific covalent bond to another ncAA. Another approach to find the matching aaRS for this ncAA will be modeling the required aaRS structure.
Despite the fact that the incorporation of ncAAs through the amber codon works, various problems remain. The repurposing of codons leads to the decrease of the growth of E. coli and it is only possible to incorporate two ncAAs. We decided to try a second method to incorporate ncAAs by the incorporation of an unnatural base pair into the DNA that encodes for 64 new codons. As E. coli has no nucleoside triphosphate transporter and is not able to synthetize the bases itself, our first challenge is the uptake of the unnatural base from the cultivation media or the synthesis inside the cell, respectively. Our aim is to find a suitable transporter for the uptake and or a biosynthesis pathway .  A promising source for the biosynthesis pathway is the plant Croton tiglium, which produces isoG. We want to sequence the C. tiglium transcriptome to identify  different enzyme variants which could catalyze the reaction to the unnatural base. Furthermore, we want to develop a detection system for the detection of the unnatural base.
To demonstrate a small slice of the possibilities offered by the incorporation of ncAAs, we want to develop an innovative toolbox containing different aaRSs to incorporate different ncAAs and demonstrated interesting applications. These ncAAs are suitable for various approaches in basic research, medicine and manufacturing. Furthermore, with our submitted parts , every iGEM team should be enabled incorporate these ncAAs into their target proteins and expand the possibilities of their protein design.
