<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_toolbox_photoswitching skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/d/d4/T--Bielefeld-CeBiTec--Ledpanelfinal.jpeg);"><DIV id="title-bg"><DIV id="title-text">
						Photoswitching
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Short Summary </H2><DIV class="article">
To showcase the possibility of enzyme activity regulation on protein level, we designed a <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">photoswitching</A> experiment in which we controlled the lycopene production of an <I>E. coli</I> strain. This was achieved by incorporation of the non-canonical amino acid (ncAA) phenylalanine-4'-azobenzene (AzoF) into pytoene desaturase, encoded by <I>crtI</I>. The lycopene production can be completely terminated by introduction of amber codons into <I>crtI</I>. The enzyme activity can be partially recovered by cotransformation with an aminoacyl-tRNA synthetase (aaRS, <A href="http://parts.igem.org/Part:BBa_K2201207">BBa_K2201207</A>). Even without supplementation of the media with the desired ncAA, this will lead to some enzyme activity recovery. In addition, we showed that we are able to switch the conformation of AzoF from a mixed state to <I>trans</I> and <I>cis</I> with our <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">LED panel</A> and that the amino acids are stable in their specific conformation over several hours. When cultivated with AzoF in <I>cis</I>- or <I>trans</I>-conformation we detected a significant difference in the lycopene production. Therefore, we proved that <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">photoswitching</A> of enzyme activity on protein level  can be achieved using our system. 
</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Design of the AzoF-RS </H2><ARTICLE>
The AzoF-RS (<B><A href="http://parts.igem.org/Part:BBa_K2201207">BBa_K2201207</A></B>) was based on a part exchange with <A href="https://2017.igem.org/Team:CU-Boulder">CU Boulder 2017</A>. They got it from the Schultz lab, which performed a selection experiment on the <I>M. jannaschii</I> TyrRS to evolve a new aaRS capable of incorporating the photoisomerizable phenylalanine-4‘-azobenzene (AzoF). Figure 1 shows a sequence alignment of the protein sequences of the <I>M. jannaschii</I> TyrRS and the AzoF-RS after the selection process.
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 1: Sequence alignment of the <I>M. jannaschii</I> TyrRS and the AzoF-RS of the Schultz lab.</B> The alignment shows six differences in the protein sequences, at position partizipating in the binding process.  </P></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Two Amber-<I>crtI</I>-Variants </H2><ARTICLE>
We created two variants in which the <I>crtI</I> gene in the <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">lycopene pathway</A> has an amber-codon incorporated; one at the position 318 and the other at position 353. We cultivated <I>E.coli</I> BL21(DE3) transformed with  the two amber-variants and a functional CrtI for 24 hours at 37°C and centrifuged the culture. The pellet of the strain with the functional CrtI showed a visible orange color, typical for lycopene (Figure 2). The two amber-variants showed no color due to the absence of lycopene caused by the non-functional CrtI in the <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">lycopene pathway</A>.
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 2: Cell pellets of the three <I>crtI-variants</I> vortexed in 400 µL acetone.</B>  Functional CrtI-variant (left), amber-codon at position 318 (middle) and amber-codon at position 353 (right).</P></DIV><ARTICLE>
We then extracted the lycopene from the pellet to quantify the amount of lycopene produced by the three cultures. For that, we resuspended the pellet in 400 µL acetone and vortexed it to solve the lycopene. After adding 400 µL water, we performed an absorbance measurement. First, we generated an absorbance spectrum to identify the best wavelength for the quantification (Figure 3).
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 3: Absorption spectrum of the positive lycopene sample from 400 to 550 nm normalized with the measurement of a 1:1 acetone water sample.</B> It shows the typical absorption spectrum of lycopene.</P></DIV><ARTICLE>
Figure 3 shows that the absorbance maximum of the lycopene is 476 nm, which correlates with the data found in the literature (Chemat-Djenni <I>et al.</I>, 2013). We then quantified the lycopene production with one biological and three technical replicates (Figure 4). It verifies the implications of Figure 2 that the cells containing the amber-variants are not able to produce lycopene. That makes them suitable for the incorporation process of AzoF, such that an increase of the lycopene production is to be expected after cotransformation with the AzoF-RS and feeding with AzoF.
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 4: Absorbance at 476 nm of the extracted lycopene of the samples.</B> The functional <I>crtI</I> (left: LP), the <I>crtI</I> with an amber codon at position 318 (middle: TAG318) and with an amber codon at position 353 (right: TAG353). The absorbance at 476 nm was normalized using a 1:1 aceton water solution.</P></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Basic Lycopene Production of the Cotransformants </H2><DIV class="article">
The cotransformants, now containing the <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">lycopene pathway</A> with one of the three <I>crtI</I> variants and the AzoF-RS, were cultivated in a 6-wellplate in LB-media at 37 °C and 400 rpm. To measure the basic lycopene production when native amino acids are unspecifically incorporated at the amber-codons, no AzoF was added to the media. After 16 hours of cultivation, 15 mL of the culture were harvested and the lycopene was extracted with acetone (<A href="https://static.igem.org/mediawiki/2017/8/8c/T--Bielefeld-CeBiTec--YKE_lycopene_protocol.pdf">lycopene extraction protocol</A>).
</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 5: Absorption spectrum of the extracted lycopene of the three samples.</B> LP is the lycopene producing strain with an intact <I>crtI</I>, TAG318 has the amber-codon at position 318 in <I>crtI</I> and TAG353 has an amber-codon at position 353 in <I>crtI</I>.</P></DIV><ARTICLE>
Figure 5 shows the measured absorption spectrum of lycopene for all samples. All of the variants produce lycopene, even the ones with the amber codon. This means, that the AzoF-RS unspecifically incorporates native amino acids when no AzoF is supplemented and therefore regenerates the CrtI function partially. 
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 6: Mean and standard deviation of the absorption spectrum of the three samples from 400 to 550 nm.</B></P></DIV><ARTICLE>
Figure 6 shows that the lycopene production of the two amber variants is less than in the functional lycopene producer.  Also, a difference in the lycopene production of the two variants can be detected. While the cotransformant TAG353 shows about half of the lycopene productivity, the variant TAG318 shows less than a third of the productivity of the lycopene producing strain (LP). This implies that the two different positions in <I>crtI</I> have different effect on the binding activity of the active site.
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Irradiation, Switching and Stability of AzoF</H2><ARTICLE>
To make sure that we are able to switch the conformation of AzoF and that the <I>cis</I> and <I>trans</I> conformations are stable over the cultivation time, we irradiated a sample of LB-media containing 1 mM of AzoF with our <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">LED panel</A>. One sample was irradiated with 367 nm for 40 minutes with 100 % brightness which causes AzoF to change to its <I>cis</I>-conformation. The conformation of AzoF can be detected through its absorption spectrum (Figure 7).
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 7: Absorption spectrum of AzoF in LB media after irradiation with light of 367 nm wavelength.</B> The black line shows the typical absorption of AzoF in the <I>trans</I>-conformation while the other lines show the absorption spectrum in the <I>cis</I>-conformation. The spectrum was measured directly after the irradiation, then after 2, 4, 17 and 20 hours. The sample was incubated at 30°C.</P></DIV><DIV class="article">
Figure 7 shows that we were able to switch the conformation of AzoF in LB-media with our <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">LED panel</A>. Furthermore, the less stable <I>cis</I>-conformation, which is induced by the UV-light of 367 nm, was stable for over 20 hours at cultivation conditions. This led us to the conclusion that we could start a cultivation of the three <I>crtI</I> variants with the two different AzoF conformations and that any detectable difference in the lycopene production would be caused by the <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">photoswitching</A> event of the amino acids.
</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Influence of Photoswitching on the Lycopene Production</H2><ARTICLE>
To investigate the influence of <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">photoswitching</A> on the lycopene production, we cultivated three biological replicates of the three variants (LP, TAG318, TAG353), one for each of the AzoF conformations for 24 hours in a 6-wellplate at 37 °C and 400 rpm. The media was supplemented with 1 mM of AzoF and then split in two charges. Both were irradiated for 40 minutes and 100 % brightness, one with 367 nm and the other with 465 nm to photoswitch the amino acids. After the cultivation, we measured the OD<SUB>600</SUB> of each sample (Figure 8). The growth was not influenced in a noticeable way by the different AzoF variants since the error bars overlap each other.
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 8: OD<SUB>600</SUB> of three biological and three technical replicates of two <I>crtI</I> variants after cultivation.</B></P></DIV><ARTICLE>
We then extracted the lycopene from the cell pellet and quantified the lycopene (Figure 9). It can be seen that the TAG353 variant with the <I>trans</I>-AzoF has the highest lycopene production, followed by the TAG353 with the <I>cis</I>-AzoF and TAG318 with the <I>trans</I>-AzoF. The TAG318 variant with the <I>cis</I>-AzoF shows the lowest lycopene amount.
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 9: Absorption spectrum of the four samples of the <I>crtI</I> variants.</B> Cultivated with AzoF supplemented to the media, photoswitched to <I>cis</I>- or <I>trans</I>-conformation.</P></DIV><ARTICLE>
The absorption at 476 nm was measured and normalized to the OD<SUB>600</SUB> of the samples. The relative lycopene production of each <I>crtI</I> and AzoF variant is shown in Figure 10 compared to the unmodified lycopene producer, measured previously.
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 10: Absorption at 476 nm (indicator for lycopene) normalized to the OD<SUB>600</SUB> (indication for the cell density) to calculate the relative lycopene production of each <I>crtI</I> variant cultivated with AzoF in <I>cis</I>- and <I>trans</I>-conformation.</B></P></DIV><ARTICLE>
Figure 10 shows the effect on the lycopene production based on the incorporation of photoswitched AzoF. The <I>trans</I>-conformation seems to favor the binding activity of the active site, while the <I>cis</I>-conformation seems to reduce the binding activity. The highest difference in the lycopene production is present at the TAG353 variant. Here the sample shows a lycopene production similar to the unmodified lycopene producer when cultivated with <I>trans</I>-AzoF while the productivity is reduced to nearly a third when cultivated with <I>cis</I>-AzoF. The AzoF-variants do not seem to influence the lycopene production when no amber-codon is present in <I>crtI</I>. Concluding, we provided strong evidence that that the observed difference in lycopene production in the three variants is caused by the incorporation and <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">photoswitching</A> of AzoF.
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> References </H2><ARTICLE><B>Chemat-Djenni, Z., Ferhat, M. A., Tomao, V., Chemat, F.</B> (2013). Carotenoid Extraction from Tomato Using a Green Solvent Resulting from Orange Processing Waste. JEOP <B>13(2)</B>: 139-147.
</ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>