Team:Bielefeld-CeBiTec/Model
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Modeling
Organization of our modeling projects
On this page, we describe our main modeling project, which was integral for our whole project.       However, besides this complex modeling, we also conducted and applied several straight-forward stochastic and statistical models       to support and guide numerous steps of our laboratory work. Some of these modeling projects are briefly described in this box; however,      we recommend reading the linked pages for further information.
Discriminant function model for the ICG prediction:
We conducted a discriminant function analysis for the classification of nucleotides in Oxford Nanopore sequencing reads    at a specific position. This model is part of our iCG software module and enabled the successful detection of unnatural bases.
Calculation of the required library size for the selection system:
We applied combinatorics and statistics to calculate the optimal library size for the tRNA synthetase selection process. This was a trade off between putting lots of efforts into constructing a very huge library and missing diversity in a too small library.     Therefore, we predicted the optimal library size. Experimental validation of this prediction was done via MiSeq analysis of the diversity     of a subset of this library.
Strength prediction for a transcription signal amplification system (BBa_K2201373) :
We modeled and visually compared the mRFP production over time for a normal mRFP reporter system and compared it to our enhanced signaling. .    Validity was done as part of the positive selection process for an adapted tRNA synthetase.
Short Summary
As our project explores possibilities of an expanded genetic code via unnatural bases and non-canonical amino acids,     we set out to complement and improve our lab work via modeling of novel amino acyl tRNA synthetases (aaRS) for a non-canonical amino     acids, which were synthetized in our lab. In order to incorporate non-canonical amino acids into proteins via the translational     process, the aaRS has to attach the amino acid to the respective tRNA. Thus, we designed aaRS sequences which were adjusted     to link our own non-canonical amino acid to a fitting tRNA. Candidates were evaluated and selected, based on a ROSETTA score.    Most promising sequences were ordered via gene synthesis for the experimental validation.     Figure 1 provides a rough overview of our modeling project. Table 1 below summarizes the realization in practice.
Figure 1: Modeling Project Overview
A stylized overview of our modeling project,    containing both in silico and in vivo components.
Table 1: Steps of our modeling project Our modeling project consists of seven main steps, combining in silico and in vivo components.
Step
Software/Method
Meaning
1. Ligand preparation
Manually via Avogadro
Due to the novelty of our amino acid, no information on the ligand is available in databases.        Therefore, manual generation of a conformer ensemble, containing        for example all energetically useful arrangements of atoms within the molecule, was required.
2. Scaffold categorization
ROSETTA protocol
The scaffold describes the rough layout of the synthetase. We downloaded the scaffold 1J1U , the aaRS of Methalonococcus janischii as a template, and then relaxed its structure to improve the outcome of the ROSETTA algorithm.
3. Set simulation constrains
Manually via ROSETTA
Constrains with regards to possible mutations of the synthetase ensure that the generated sequences fit to the amino acid.        For example, we constrained the distance between certain atoms and their angle to a range optimal for hydrogen bonds.
4. Enzyme matching
ROSETTA protocol
ROSETTA combines information about the ligand and constrains to find possible hydrogen bonding partners and propose the shape        of the scaffold within the set constraints.
5. Enzyme design
ROSETTA protocol
An algorithm uses the information from the previous step and information on the ligand to simulate the mutation process and        generate sequences for optimized scaffolds with corresponding scores as measures of fit.
6. Evaluate results in silico
Manually
Based on the score values, we ordered the synthesis of the most promising sequences.
7. Evaluate results in vivo
Manually
The synthetases are validated in the lab with the corresponding ncAA via a positive-negative selection system.
Introduction
Overview
Figure 2: Tyrosyl-tRNA-synthetase
3D-structure based on "1J1U" from PDB edited with pymol
As part of our iGEM project, we faced the challenge of adapting the tRNA synthetase      to non-canonical amino acids. For this purpose, we modeled possible candidates for synthetases as an      alternative to carrying out a positive-negative selection according to (Liu et al., 2007) in the laboratory.      Due to the rapid development in the field of protein and molecular structure analysis, there has been an      increase in the availability of molecular 3D structure data. These data are organized in publicly available      databases which provide a foundation for the modeling and simulation of chemical-biological processes in bioinformatics.      As our ncAA has been synthetized in our lab, no such comprehensive information is available, yet.      However, information of similarly structured amino acids can potentially serve as a basis for our modeling.      Specifically, we focused on simulation to      design an aaRS for the new ncAA CBT-Asparagine .
Method
We used the open-source software "ROSETTA" for the main part of our modeling project, which was introduced at the University of Washington by David Baker in 1997        (Simon et al.,1997), initially in the context of protein structure prediction. ROSETTA has grown through the addition of numerous modules and is currently widely used in research. In our application, we focus on the Rosetta module called the "Rosetta Enzyme Design Protocol"
ROSETTA Enzyme Design
Overview
Since the non-canonical amino acid synthesized in the laboratory is completely novel, there is no corresponding tRNA synthetase which can load the tRNA, yet. For this reason, we use the enzyme design protocol to design the binding pocket in a way that allows it to form an effective and specific enzyme. The protocol consists of two main steps: matching and designing (Richter et al., 2011)      The enzyme design algorithm is briefly summarized in Figure 3.
Figure 3: Flowchart Enzym Design Protocol.
Matching Step
Figure 4: overview of constraints
all possible constraints, which can be set, with dashed lines indicating hydrogenbonds, normal lines indicating covalent bonds
The meaning of the matching step is to match the amino acids which constrains to the ligand, following specific constrains which ensure that the result is sensible and feasible. For this, ROSETTA analyzes the structural formula of the non- canonical amino acid and offers the possible hydrogen binding partners.
Matching step inputs
For the matching step, the following input-files are needed:
a “.params”-file specifying information about the ligand
a “.pdb”-file providing a rough scaffold layout
a “.cst”-file to define the bindings between ligand and scaffold
a “.pos”-file to define the positions of the amino acids in the scaffold
a “.flags” file to control all inputs. These files are necessary, as they describe the ligand and backbone and specify the parameters of the algorithm
To read about each file in further detail, please click the technical detail button below: SHOW TECHNICAL DETAILS
“.params”-file: A conformer ensemble has to be generated using information about the ligand, as the non-canonical amino acids are not generally available in databases like PDB , making it necessary to build them manually      using tools like pymol , Avogadro or Chemdraw . Using these tools,      files can be saved in the desired format. The ligand needs to be specified in the “.sdf”, “.mol” or “.mol2” file format. Such a      file can be obtained automatically by converting the relevant information from a “.pdb” file, if available. This conversion process usually also involves augmenting the data with hydrogen atoms      in case they are missing from the “.pdb” file. Alternatively, the ligand can be designed using "Simplified Molecular Input Line Entry Specification" (SMILES) or manually using tools such as Avogadro , as we did. In the next step, the ligand file is used      to create a conformer ensemble that is in turn used to create a Rosetta parameter (“.params”) file. In addition to the specific names of all atoms present in the ligand, this parameter file also       stores all bonds between the individual atoms, including the binding angles and binding distances. Rosetta cannot generate the conformer ensemble by itself, so an additional tool is needed.        Different tools are capable of creating the conformer ensemble automatically, but it is best to manually define constraints for the chi1, chi2 and backbone psi torsion angles that define the orientation of the ligand in the binding pocket. For this, we know of three tools: The first is OpenEye Omega , but the full license is very costly and the academic free version is hard to obtain.        The second tool is Accelrys Discovery Studio , but Accerlys does not provide a free license. The third tool is TINKER , which is free, but poorly documented and depends on a specific keyfile,        which requires a high amount of chemical expertise to generate. Conformers might also be generated without constrains, for which different tools are available, in our case, we used ConFlex .        Conformers need to be stored in one file (“.sdf”, “.mol”, or “.mol2”).
“.pdb”-file: The input-file for the scaffold, in our case the tRNA synthetase, can be downloaded in PDB format from Protein Data Bank (PDB). It is then necessary to delete the natural ligand from the PDB-file,      as we need to incorporate our own aaRS. Additionally, it is advised to relax the  preferably, the structure should be relaxed in order to allow for flexibility with regards to the simulation outcomes.      For further details, see the ROSETTA Relaxing documentation.
“.cst”-file: The .cst-file defines the potential hydrogen bonds between the ligand and the amino acid. For example, the code block  characterized by the tags “CST::BEGIN” and “CST::END”, specifies the orientation or      catalytic function of the enzyme. More specifically, the first record of the block begins with “TEMPLATE::ATOM_MAP”, followed by either “atom_name” or “atom_type”, depending on whether a specific residue or a specific type of residue is provided. In the latter case, it is not important to choose specific atoms. Instead, a catalytic residue of the amino acid such as “OH” or “Nhis” is specified. The next lines of the TEMPLATE::ATOM_MAP      record define the residues using one-letter or three-letter-codes that are prefixed by “residue1” or “residue3”, respectively.      The second record, beginning with the tag “CONSTRAINT”, contains all relevant distance, angle and torsion constraints for the matching. Each constraint is described with five parameters.      In the case of the distance constraint, the first parameter describes the optimal distance “x0” between the chosen residues, the second parameter describes the tolerance “xtol”,      the third parameter defines the strength “k” and the fourth parameter specifies the type of bond (1 for a covalent bond, 0 otherwise). If the modulus of the difference between the actual distance “x”      and the specified optimal distance is smaller than the tolerance, then the penality score is zero. Otherwise, the constraint consists of the term
k*(|x-x0|-xtol)
to the penality score. For the angle and torsion constraints, the description is similar.      If necessary, additional hydrogen bonds to other atoms of the ligand are specified in terms of additional blocks, using the tag “VARIABLE::CST”.      Finally, most of the blocks described above can be optionally followed by an “ALGORITHM_INFO” record that stores details of the matching algorithm by parameter values.      We refer to the Rosetta documentation for further details.
”.pos”-file: The “.pos” file contains the allowed locations in the scaffold for the chosen catalytic residues in each constraint block of the “.cst” file.
Matching step outputs
Figure 5: example of an output-pdb-file from the matching step
CBT-Asparagine in purple, amino acid in green, created in pymol
The output generated in the matching step is the layout of the scaffold as well as one or more states of the amino acid which enable interaction with the ligand. This information is stored as a “.pdb” file and becomes part of the input for the design step.
Our results for this step
We used the “1J1U” -scaffold from PDB for our matching step.         The “1J1U.pdb”-file contains the tyrosyl-tRNA-synthetase, which is labeld under “Chain A”, the orthogonol tRNA under “Chain B”         and the natural ligand tyrosyl. For our project, we deleted the natural ligand and “Chain B”, because it was not necessary to         change their structure or sequence and it was a way to save compute time.        We designed the ligands manually via Avogadro based on a default matching algorithm for both        amino acids, thus creating useful .cst-files.
Design Step
The design step applies an algorithm such that the binding pocket and the near environment are mutated and the remaining scaffold is         repacked. Additionally, a badness-of-fit score is generated which indicates how well the mutation fits the amino acid. For every file from the          matching step, a model with a score and a “.pdb-file” was generated, specifying where the sequence can be located. Additionally, the ".pdb-file"         makes visual analysis of the 3D-structure possible. Notably, the amino acid structure can be extracted separately.        The following section describes the structure of the design step. Further details on each step can be obtaind by showing the Technical Details Section. 1. Optimizing the catalytic interactions For the first alternative, the file can be generated either by the Rosetta standard or a manually created .”res”- file. For more details, we refer to the Rosetta documentation . For the latter alternative, residues are automatically categorized by their location of the Cα;lpha;. SHOW TECHNICAL DETAILS
Residues are catagorized as follows:
residues that have their Cα within a distance cut1 angstroms of any ligand heavyatom will be set to designable
res that have Cα within a distance cut2 of any ligand heavyatom and the Cβ closer to that ligand atom than the Calpha will be set to designable. cut2 has to be larger than cut1
res that have Cα within a certain distance cut3 of any ligand heavyatom will be set to repackable. cut3 has to be larger than cut2
res that have Cα within a distance cut4 of any ligand heavy atom and the Cβ closer to that ligand atom will be set to repackable. cut4 has to be larger than cut3
all residues not in any of the above 4 groups are kept static.
2. Cycles of sequence design and minimazation within constrains To optimize the structure we applied an iterative optimization algorithm. This algorithm mutates all residues from the backbone, which are not part         of the catalytic center, to alanine, and a small energy function refraction will place the ligand in an optimal position to the backbone. SHOW TECHNICAL DETAILS
For this approach, bb_min and chi_min allow for backbone flexibility and the rotation of the torsions. An alternative for this minimization step is the         Monte Carlo rigid body ligand sampling. For further information on this method, we refer to the ROSETTA documentation .
Design step inputs
The following input files are relevant for the design procedure:
“.pdb”-file generated in the matching step
“.cst”-file for the ligand
“.params”-file for the ligand and the scaffold
“.flags” to coordinate the inputs
For further information on these files, please refer to step 2 above.
Figure 6: example of CBT-Asparagine in the binding pocket
CBT-asparagine in purple, scaffold in green, created in pymol
Design step outputs
The output for the design step is a “.pdb”-file containing the mutated scaffold and a “.score”-file.           For every .pdb-file, a line in the score-file is generated, so it is easy to evaluate the given structure.           The first score in the file is the total score of the model. After that, the number of hydrogen bonds            in the protein as a whole and in the constraints is listed, followed by the number of dismissed solutions in the            catalytic residues as well in the whole protein and in the constraints.            See the technical details below for a full overview of the output information SHOW TECHNICAL DETAILS
total_score: energy (excluding the constraint energy)
fa_rep: full atom repulsive energy
hbond_sc: hbond sidechain energy
all_cst: all constraint energy
tot_pstat_pm: pack statistics, 0-1, 1 = fully packed
total_nlpstat_pm: pack statistics withouth the ligand present
tot_burunsat_pm: buried unsatisfied polar residues, higher = more buried unsat polars (just a count)
tot_hbond_pm: total number of hbonds
tot_NLconst_pm: total number of non-local contacts ( two residues form a nonlocal         contact if they are farther than 8 residues apart in sequence but interact with a Rosetta score of lower than -1.0 )
Results
Results in silico
We choose our synthetases based on a good total score and a good ligand score. We checked the corresponding PDB-files, and rated the ligand and the binding pocket         as satisfying, so that the ligand assumedly does not collide with residues in the near environment.        The total scores for CBT are not as good as the scores for NPA. However, the ligand scores are acceptable in both cases. A visual evaluation confirms that the ligand         fits into the binding pocket.
Our results for this step
We used this algorithm to simulate the evolution of the tyrosyl-tRNA with the amino acids Nitrophenylalanine(NPA) and Nγ‑2‑cyanobenzothiazol‑6‑yl‑L‑asparagine (CBT-asparagine).
NPA simulation:
The .cst-file contained two blocks for the nitrogroup of NPA. Since there are two oxygen-atoms in the nitrogroup,         we defined two atom nametags. As several possibilities are useful, we defined two possible constraint partners          for the hydrogen bonds. The first is asparagine (N) or glutamine (Q) and the second is glycine (G). We set the           possible distance to 2.8 Å, as it is the optimal distance for hydrogenbonds, and a tolerance level of 0.5 Å.            We set the angles to 120° with a tolerance of 40°, as recommended by Florian Richter during our discussion in cologne.            The torsion angles were set to 180° with a tolerance of 180° and a penalty of 0, such that the torsion angles can rotate            completely freely.(Richter, unpublished data)
CBT-Asparagine simulation:
CBT-Asparagine can build hydrogen bonds in two ways. The first is a weak hydrogen bond on the         sulfur atom and the other possibility is a normal hydrogen bond on the nitrogen (N2)         after the Cγ. We wrote three cst-files, one for a possible bond with sulfur, one for a         possible bond with nitrogen, and one for both bonds. As possible corresponding amino acids, we chose serine,          threonine, tyrosine, asparagine, glutamine, and glycine.
SHOW TECHNICAL DETAILS
It is recommended to write a “.flags”-file, because there are several input parameters to be defined,         but it is also possible to define them via console user interface. For the categorization of the scaffold, we chose the automatic determination and set the following cuts, like the Baker-lab commonly used: cut1: 6 Å, cut2: 8 Å, cut3: 10 Å and cut4: 12 &#8491
We used this algrithm to simulate the evolution of the tyrosyl-tRNA with the amino acids Nitrophenylalanine and CBT-ASP     We obtained 13 synthetase sequences for CBT-ASP, and 43 sequences for NPA, which fit well into the binding site according to the ROSETTA score.     The sequences for the best synthetases for NPA is available here .
Table 2: ROSETTA Enzyme Design Protocol Results
ROSETTA scores of the best modeled synthetases for CBT-Asparagine and NPA.
Sequence Number
Total Score
Ligand Score
ncAA
15
124.88
-3.77
NPA
19
23.55
-3.93
NPA
31
-3.40
-2.47
NPA
32
-1.57
-3.82
NPA
40
11.67
-4.33
NPA
41
11.55
-2.98
NPA
43
66.36
-5.05
NPA
2
38.01
-6.56
CBT-Asparagine
4
58.45
-4.37
CBT-Asparagine
5
109.13
-4.25
CBT-Asparagine
Table 2: ROSETTA Enzyme Design Protocol Results
ROSETTA scores of the best modeled synthetases for CBT-Asparagine and NPA.
Position
Synthetase Number
Original Amino Acid
Mutation Amino Acid
30
5
Serine
Asparagine
32
5
Tyrosine
Threonine
34
2, 4
Glycine
Alanine
36
2
Glutamine acid
Isoleucine
61
5
Asparagine acid
Arginine
63
5
Isoleucine
Alanine
65
4
Leucine
Glycine
65
5
Leucine
Threonine
68
4
Asparagine acid
Alanine
69
4
Leucine
Alanine
70
2
Histidine
Asparagine acid
70
4
Histidine
Glycine
72
4
Tyrosine
Glutamine acid
73
2
Leucine
Alanine
73
4
Leucine
Methionine
74
2
Asparagine
Asparagine acid
76
2
Lysine
Serine
79
4
Leucine
Arginine
101
5
Lysine
Glutamine acid
103
5
Valine
Triptophane
104
5
Tyrosine
Valine
105
4, 5
Glycine
Serine
107
5
Glutamine acid
Lysine
108
4
Phenylalanine
Lysine
108
5
Phenylalanine
Arginine
109
4, 5
Glutamine
Alanine
114
4
Tyrosine
Alanine
115
4
Threonine
Triptophane
118
4
Valine
Serine
134
2
Methionine
Asparagine
137
2
Isoleucine
Alanine
139
2
Arginine
Serine
147
2, 4
Alanine
Serine
148
2
Glutamine
Lysine
149
2
Valine
Threonine
150
2, 4
Isoleucine
Leucine
151
2
Tyrosine
Serine
152
2
Proline
Threonine
153
2
Isoleucine
Leucine
153
4
Isoleucine
Threonine
154
2
Methionine
Asparagine
154
4, 5
Methionine
Serine
155
2
Glutamine
Glycine
155
4, 5
Glutamine
Alanine
156
4
Valine
Alanine
157
4
Asparagine
Alanine
158
4, 5
Asparagine
Tyrosine
159
4
Isoleucine
Glycine
161
4
Tyrosine
Glutamine acid
161
5
Tyrosine
Asparagine acid
162
4
Leucine
Methionine
162
5
Leucine
Alanine
164
5
Valine
Alanine
172
2
Glutamine acid
Lysine
173
2
Glutamine
Serine
176
2
Isoleucine
Serine
204-210
2, 4, 5
varying
-
307
2
-
Alanine
307
4, 5
-
Tyrosine
in vivo validation of predicted tRNA synthetase structures
In order to test the functionality and specificity of our modeled aaRS, we selected the most promising 11 amino acid sequences and ordered seven synthetase sequences of NPA via IDT, and four     synthetase sequences of CBT-Asparagine by courtesy of Genscript, where we had previously won a grant of 500€.      Since the DNA synthesis by GenScript and IDT were delayed by several weeks, we could not perform the best practice characterization of these parts.     Finally, we received only three of the ordered syntheses in sufficient quality for further experiments. All of them encode predicted CBT-tRNA-synthetases.     We subjected the sequences to a positive selection as initial characterization. Plasmids encoding the predicted best candidates were cotransformed with our     positive selection plasmid into E. coli(BL21 DE3). Due to the IPTG induced promoter, we tested different IPTG concentrations, including 0 mM, 5 mM,     10 mM, and 15 mM. In addition, we tried different concentration of the antibiotics: kan15, cm15/kan15, and cm15/kan15/tet5. The numbers of resulting colonies is presented in figure 10. Our in vivo results show that our in silico designed enzymes kept their native function and are      able to integrate amino acids through an amber codon matching tRNA. Since these results only indicate the acceptance and transfer of the non-canonical     amino acid, additional experiment are required to demonstrate a high specificity of these enzymes. We offer the predicted sequences to the community for further characterization via the parts-reg ( BBa_K2201300 , BBa_K2201301 , BBa_K2201302 ).
Figure 7: positive selection of BBa_K2201300
positive selection on kanamycin
Figure 8: positive selection of BBa_K2201301
positive selection on kanamycin
Figure 9: positive selection of BBa_K2201302
positive selection on kanamycin
Figure 10: bargraph of numbers of colonies
different IPTG concentrations
References
All our files can be obtained here. Liu, W., Brock, A., Chen, S., Chen, S., Schultz, P. G. ,(2007). Genetic incorporation of unnatural amino acids into proteins in mammalian cells. Nature methods, 4(3), 239-244.
Richter, F., Leaver-Fay, A., Khare, S. D., Bjelic, S., Baker, D. (2011). De novo enzyme design using Rosetta3. PloS one, 6(5): e19230.
Simons, K. T., Kooperberg, C., Huang, E., Baker, D. (1997). Assembly of protein tertiary structures from fragments with similar local sequences using simulated annealing and Bayesian scoring functions. Journal of molecular biology, 268(1), 209-225.
