Team:Bielefeld-CeBiTec/InterLab
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Interlab Study
Reproducibility of results in different laboratories is crucial to facilitate science. We participated in the interlab study        to support the iGEM community with more data for interlab comparison of measurement results.
Protocol
We applied the protocol for plate reader that was provided by the iGEM HQ.
This year, in addition to the positive and negative control, six other plasmids containing the GFP gene, different ribosome binding sites and varying Anderson       promoters were transformed into Escherichia coli. The following devices were tested:
positive control (BBa_I20270): GFP under control of a weak Anderson promoter
negative control (BBa_R0040): TetR repressible promoter
TD 1 (BBa_J364000):  J23101 + I13504
TD 2 (BBa_J364001): J23106 + I13504
TD 3 (BBa_J364002): J23117 + I13504
TD 4 (BBa_J364003): J23101.BCD2.E004.B0015
TD 5 (BBa_J364004): J23106.BCD2.E004.B0015
TD 6 (BBa_J364005): J23117.BCD2.E004.B0015
In the beginning, there were complications to transform the plasmids directly from the distribution plates.       Therefore, we transformed them to E. coli XL-1 Blue using 1 µL plasmid each from kit plate 6 and 7. The colonies       were picked and incubated overnight in 5 mL LB and 0.25 µL mL-1 chloramphenicol. After the plasmid isolation,       all parts were sequenced. Plasmids with correct sequences were then retransformed in E. coli DH5-α.
For measurement two cultures of each plate were picked and grew over night in 5 mL LB media with 0.25 µL mL-1 chloramphenicol.
The bacteria cultures were diluted to OD600 of 0.02 in a total volume of 24 mL LB with chloramphenicol in 250 mL shake flasks      and incubated without light for 6 hours at 37 °C and 220 rpm. 500 µL samples were taken after 0, 2, 4 and 6 hours of incubation       and stored on ice until the measurement. The temperature during the measurement was at least 21.6 °C .
Material
Incubator: We used the New BrunswickTN Innova®43/43R at 37 °C and 220 rpm. We covered the glass front with paper to make sure the samples  grew in darkness.
Cuvette measurement: For the first OD600 measurement of the overnight cultures we used an Eppendorf 6131 BioPhotometer.
Plate reader: Our plate reader model was the Tecan Infinite M200 (Roche). The excitation wavelength was set to 488 nm and emission       wavelength was detected at 510 nm [1] .
96 well plate: We used the “NuncTM MicroWellTM 96-Well PolymerBase Black” by Thermo Fisher Scientific for our measurement with the plate reader.
E. coli strains
E. coli DH5-α
E. coli XL-1 Blue
Results
After 6 hours of cultivation and storage on ice, the optical density of all cultures with the different plasmids was measured.
Figure 1: Development of the OD600 of the tested devices over the cultivation time of six hours. The measured samples were test devices (td) 1-6, the positive control (pos) and negative control (neg).
The OD600 values increased during the six hours of cultivation, except for test device 1, where both colonies did not persist the dilution (Figure 1).
Figure 2: Fluorescence of the tested devices signal over the cultivation time of six hours. The measured samples were test devices (td) 1-6, the positive control (pos) and negative control (neg).
The mean fluorescence does not correspond to the expected results. Proportional to the increasing OD600 values, we expected all fluorescence intensities to rise.      But none of the cultures showed an exponential increase. Although test device 1 did not survive the cultivation, it had the strongest fluorescence during the first four hours.
Figure 3: Comparison of the mean fluorescence of the tested devices with N = 18. The measured samples were test devices (td) 1-6, the positive control (pos) and negative control (neg).
As can be seen in figure 3 the mean fluorescence of the positive control is the strongest during the whole cultivation. In contrast to our expectation the negative control shows      a stronger mean fluorescence than all test devices except test device 1.
In the final analysis, all measured values in general did not fulfil the expectation. The maximal OD600 values did not reach the 0.7 and although all cells (except test device 1)       did grow steadily, the fluorescence curve is not uniform.
Due to theses unexpected results, we repeated the whole process to validate our findings. Moreover, all plasmids were checked by Sanger sequences to exclude the influence of sequence variants.      No sequence errors were detected in the plasmids.
Since we are measuring different fluorescence signals on a routine basis, we are certain that the hardware in our lab is fine.
