Team:Bielefeld-CeBiTec/Project/toolbox/fusing
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Fusing
Short summary
Fusing proteins is normally limited to the C‑ or N‑terminus of a protein.    The incorporation of non-canonical amino acids that could be fused to each    other or to surfaces enables several additional applications. Examples are the    immobilization of proteins as well as improved stability of protein polymer networks.    Furthermore, it leads to enhanced efficiency of pathways by combining enzymes of    one pathway or for any other system where colocalization is beneficial.
As proof of concept, we work on enhanced stability of a protein polymer. These networks    can be applied for different applications like modern biomaterials in medicine and industry    (Rnjak-Kovacina et al., 2011). The amino acids Nε‑L‑cysteinyl‑L‑lysine (CL)    and Nγ‑2‑cyanobenzothiazol‑6‑yl‑L‑asparagine (CBT‑asparagine)    comprise key parts of this tool. Both amino acids can bind specifically to each other resulting in the formation    of a covalent bond between their side chains. We plan to use this covalent bond to increase the stability of    silk elastin like proteins (SELPs). The strengthened polymer network would be a perfect material to produce    biological wound bindings which are very thin and would be able to interact with the natural tissue matrix    (Boateng et al., 2008).
Terminus independent fusion proteins
While terminus dependent binding systems for proteins are already in use, there are only a      few systems for terminus independent binding systems. We want to expand the number of those      systems. Our aim is to incorporate two non-canonical amino acids, which are able to build a      specific bond to each other. According to the synthesis of D-luciferin for the firefly luciferase      of Photinus pyralis, we decided to use the specific binding of 1,2‑aminothiols      and the cyano group of cyanobenzothiazole (CBT). Figure 1 shows the biosynthesis of luciferin      and the mechanism of the binding reaction of 1,2‑aminothiol and CBT.
Figure 1: Reaction of the 1,2‑aminothiol of cysteine and       CBT to luciferin (Liang et al., 2010).
By synthesis of amino acids with side chains containing CBT and a 1,2‑aminothiol, polypeptides      binding to each other should be produced. These amino acids are CL and CBT‑asparagine whose binding      mechanism is shown in figure 2.
Figure 2: Specific binding reaction of CL and CBT-asparagine.
Nε-L-cysteinyl-L-lysine
We synthesized CL and CBT‑asparagine in our lab. Additionally we are providing the community with a validated protocol for the synthesis of CBT‑asparagine .
CL is an amino acid consisting of cysteine and lysine. The cysteine was coupled to the side        chain of lysine so that CL contains a free 1,2-aminothiol group        (Nguyen et al., 2011). This is an important characteristic for the        specific binding between CL and CBT‑asparagine.
Name: Nε-L-cysteinyl-L-lysine
Short:     CL
Molecular Weight:  249.33 g mol-1
Storage:   -20 – 4 °C
Function:   Terminus independent binding system
Figure 3: Structure of CL.
Nγ-2-cyanobenzothiazol-6-yl-L-asparagine
Figure 4: Structure of CBT‑asparagine.
CBT‑asparagine is a completely novel amino acid, which we are synthesizing on our own. The synthesis is based on coupling the amino group of 6-Amino-CBT to the carboxyl        group of the side chain of L-asparagine. The cyano group of the CBT enables the specific binding of the CBT‑asparagine to 1,2-aminothiols.
Name: Nγ‑2‑cyanobenzothiazol‑6‑yl‑L‑asparagine
Short:     CBT‑asparagine
Molecular Weight:  290.30 g mol-1
Storage:   -20 – 4 °C
Function:   Terminus independent binding system
Silk Elastin like Proteins
This specific binding can improve the stability of SELPs. These are linear polypeptides with repeats of silk and elastin consensus     sequences. They show broad applications in medicine, tissue engineering and industry     (Rnjak-Kovacina et al., 2011). The silk consensus sequence is GAGAGS and the elastin consensus sequence is     VPAVG. The consensus sequences can interact with each other and are able to form non-covalent hydrogen bonds. This results in a     polymer network based on hydrogen bonds with a β-sheet structure. Figure 5 shows the schematic structure of a SELP polymer network.
Figure 5: Schematic structure of a SELP polymer network.
Silk consensus sequences (GAGAGS) are shown in green,      elastin consensus sequences (VPAVG) in red and the blue lines indicate the hydrogen bonds of the consensus sequences.
According to the work of Collins et al. (2013), we decided to use a sequence with nine repeats of five repeats of the silk consensus     sequence and nine repeats of the elastin consensus sequence (see figure 6).
Figure 6: Schematic sequence of the SELP (Collins et al., 2013).
Silk consensus sequences (S)     are shown in green and elastin consensus sequences (E) in red.
By incorporating CL and CBT‑asparagine between the silk and the elastin repeats, we receive a strengthened polymer network     with covalent bonds (Figure 7).
Figure 7: Schematic structure of a SELP polymer network including CL and CBT-asparagine.
CL and CBT-asparagine (purple) are introduced between the silk (green) and elastin (red) repeats.
Recursive Directional Ligation by Plasmid Reconstruction (PRe-RDL)
The gene sequence for these SELPs has a high GC content and contains a high number of repeats leading to issues during synthesis.      Therefore, PRe-RDL can be applied to address this challenge. PRe-RDL uses three restriction sites of a parent plasmid which contains the      gene of interest (goi) and the subsequent ligation of fragments of two different restricted parent plasmids. The first step involves      the restriction of a parent plasmid at the 3'-end of the goi and in the backbone. The second step is the restriction of a parent plasmid      at the 5'-end of the goi and at the same position of the backbone as in the first step. The final step is the ligation of both generated      fragments containing the goi. The result is a plasmid with two copies of the goi (Figure 5).
Figure 8: Scheme of the PRe-RDL according to McDaniel et al., 2010 and applied to pSB1C3 containing one elastin consensus sequence. The Pre-RDL      consists of 3 steps, two different digestions (step 1 and 2) and one ligation (step 3).
Applied to pSB1C3 containing the consensus sequences of monomers and the spacers between the BioBrick prefix and suffix shown in figure 9, it is possible to use the PRe-RDL in     combination with the restriction enzymes, AcuI, BseRI, and BspEI, to build repetitive sequences of monomers.
Figure 9: Design of BBa_K2201250 and BBa_K2201251 .
Step 1 serves to create a fragment containing one part of the chloramphenicol resistance (CmR) and the whole monomer consensus sequence. To do so, the plasmid has to be digested     with AcuI and BspEI. The other fragments which are resulting from this step contain the other part of the CmR and each one part of the origin of replication (ori).     This decreases the chance of religation of the three fragments originating from this step. For step 2, it is necessary to use BseRI and BspEI. The result of this     digestion is one fragment containing one part of the CmR and the spacer between the prefix and the monomer consensus sequence and one fragment with the other part of the CmR,     the ori, and the monomer consensus sequence. By ligation of both fragments of step 1 and 2 containing the monomer consensus sequence (step 3) pSB1C3 with two repeats of the     monomer consensus sequence results. By repeating these steps it is possible to create repetitive sequences consisting of monomeric consensus sequences. During the PRe-RDL it is also possible    to add codons like the amber codon using primers. After the PRe-RDL it is important to add a start and a stop codon.
References
Boateng, J., Matthews, K.H., Stevens, H.N.E., and Eccleston, G.M. (2008). Wound Healing Dressings and Drug Delivery Systems: A Review. J. Pharm. Sci. 97.
Collins, T., Azevedo-silva, J., Costa, A., Branca, F., Machado, R., and Casal, M. (2013). Batch production of a silk-elastin-like protein in E . coli BL21 ( DE3 ): key parameters for optimisation. Microb. Cell Fact. 12: 1–16.
Liang, G., Ren, H., and Rao, J. (2010). A biocompatible condensation reaction for controlled assembly of nanostructures in living cells. Nat. Chem. 2: 54–60.
McDaniel, J.R., Mackay, J.A., Quiroz, F.G., and Chilkoti, A. (2010). Recursive Directional Ligation by Plasmid Reconstruction allows Rapid and Seamless Cloning of Oligomeric Genes. 11: 944–952.
Nguyen, D.P., Elliott, T., Holt, M., Muir, T.W., and Chin, J.W. (2011). Genetically Encoded 1,2-Aminothiols Facilitate Rapid and Site-Specific Protein Labeling via a Bio-orthogonal Cyanobenzothiazole Condensation. J. Am. Chem. Soc. 133: 11418–11421.
Rnjak-Kovacina, J., Daamen, W.F., Pierna, M., Rodríguez-Cabello, J.C., and Weiss, A.S. (2011). Elastin Biopolymers. Compr. Biomater.: 329–346.
