<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_unnatural_base_pair_unnatural_base_pairs skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/unnatural base pair/unnatural base pairs</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/f/f7/T--Bielefeld-CeBiTec--pipettieren.jpeg);"><DIV id="title-bg"><DIV id="title-text">
						Unnatural Base Pairs
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Short Summary</H2><DIV class="article">
The idea to expand the DNA by the incorporation of an unnatural base pair (UBP) was already born in 1962. Since then, much effort has been done to engineer UBPs that function as an orthogonal system to create a semi-synthetic DNA (xenogenic DNA or XNA). Besides hydrogen bonding, researchers also investigated UBPs with different chemical properties. Usage of an UBP creates several challenges like the adaption of the whole transcriptional and translational machinery. When dealing with a semisynthetic organism, additional tasks arise, e.g. the biosynthesis of the &quot;base&quot; as well as the synthesis of the corresponding nucleosides and nucleotides. The <I>de novo</I> synthesis as well as the salvage pathway of nucleotides is a very complex metabolism, which includes a lot of different enzymes. To obtain a fully autonomous semi-synthetic organism, the easiest path is the incorporation of UBPs that are similar to canonical nucleotides using hydrogen bonding. This brings up isoguanosine (isoG) and isocytosine (isoC) with conceivable biosynthesis pathways.
</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Background to the Unnatural Base Pairs (UBPs)</H3><DIV class="contentline"><DIV class="third double"><DIV class="figure large"><P class="figure subtitle"><B>Figure 1: Unnatural bases.</B></P></DIV></DIV><ARTICLE>All amino acids are encoded by codons, which are defined by three base pairs. This information is encoded in the genome of an organism and since the origin of life every natural genome has consisted of the two-base-pair genetic alphabet dA-dT (adenine-thymine) and dG-dC (cytosine-guanine). There are strong efforts to replace a canonical base pair or expand the genetic code by a third unnatural base pair (UBP) (Martinot and Benner, 2004; Jiang and Seela, 2010; Kwok, 2012; Zhang <I>et al.</I>, 2017; Yamashige <I>et al.</I>, 2012; Seela <I>et al.</I>, 2005; Switzer <I>et al.</I>, 1989; Yang <I>et al.</I>, 2011).
So far, the modification of sugars and phosphates for nucleotides with important applications have been explored. 
First experiments with unnatural bases extended the nucleotide alphabet by replacing thymine with 5-chlorouracil in <I>E. coli</I> 
over a period of 25 weeks (Dunn and Smith, 1957; Marlière <I>et al.</I>, 2011). But for an UBP, two modified nucleobases are needed. 
A. Rich discussed the extension of the DNA by two additional bases already in 1962 (Rich, 1962). An additional UBP can be interesting
 for physiochemical properties if the nucleobases can be site-specifically derivatized with linkers for chemical groups. Furthermore,
 the availability of an UBP <I>in vivo</I> would be a milestone in the field of synthetic biology. This would mean the creation of a 
 semi-synthetic organism with altered storage capabilities for genetic information that leads to new and useful functions and
 applications (Malyshev and Romesberg, 2015).

		</ARTICLE><H4>UBPs with hydrogen bonding</H4><ARTICLE>
		As stated above, utilizing an UBP creates several challenges. First approaches focused on orthogonal pairing and realizing of <I>in vitro</I> replication.
		For this purpose, UBPs with complementary hydrogen bonding were explored. The labs of Rapport and Benner independently 
		investigated the UBP disoG-disoC, which is constitutional isomer of dG-dC. Main problems concerning this UBP are deaminiation 
		and tautomerization that lead to mispairing with natural bases, predominantly dT/U. Those problems resulted in further derivates 
		of disoG-disoC, like the latest UBP dZ (6 6-amino-5-nitro-3-(1‘-β-D-2‘-deoxyribofuranosyl)-2(1H)-pyridone) / dP 
		(2-amino-8-(1‘-β-D-2‘-deoxyribofuranosyl)-imidazol[1,2-α]-1,3,5-trizan-4(H)-one) from the Benner lab that showed
		high-fidelity amplification by PCR (Yang <I>et al.</I>, 2010). A Taq DNA polymerase was modified to accept the new ATCGPZ-DNA,
		resulting in a retention rate of 98.9% (Laos<I> et al.</I>, 2014; Chen <I>et al.</I>, 2011). The six-nucleotide genetic alphabet 
		gives rise to DNA with a B-form as well as an A-form, with the major groves being 1 Å wider than the natural GC pair 
		(Georgiadis <I>et al.</I>, 2015). Also transcription as well as reverse transcription and even translation was successfully
		performed <I>in vitro</I> (Bain <I>et al.</I>, 1992; Leal <I>et al.</I>, 2015). Another UBP based on complementary 
		hydrogen bonding is ds-dy, which are analogs to purine and pyridine developed by Hirao in 2000. <I>In vitro</I> transcription 
		and translation was achieved using this UBP but the derivate dz with lower mispairing rates were insufficiently recognized 
		by DNA and RNA polymerases as a triphosphate (Hirao <I>et al.</I>, 2002; Hirao<I> et al.</I>, 2004).
		</ARTICLE><H4>Other UBPs</H4><ARTICLE>
		Besides hydrogen bonding, further research directed towards UBPs with metal-depending pairing,
		hydrophobic forces and ring stacking forces has been done (Malyshev and Romesberg, 2015). d5SICS – dMMO2 and d5SICS-dNaM 
		are two promising candidates using hydrophobic interactions, which allowed transcription (Seo <I>et al.</I>, 2009).
		The first demonstration in <I>E. coli</I> was based on two plasmids, one encoding the nucleoside triphosphate transporter
		for dNaM and d5SICS and the other encoding a gene sequence using the extended genetic code (Malyshev <I>et al.</I>, 2014).
		Uptake of the synthetic bases as well as a stable plasmid replication over 24 generations was demonstrated (Malyshev <I>et al.</I>, 2014). 
		In 2017, the Romesberg group presented a new version of their semi-synthetic organism. The most important advances were an optimized 
		transporter with improved uptake of unnatural triphosphates and better retention of XNA with dNaM-dTPT3. Furthermore,
		they used a CRISPR-Cas system to eliminate plasmids that lost the XNA (Zhang <I>et al.</I>, 2017).
		</ARTICLE><H4>Our approach</H4><ARTICLE>
		The challenging part about using XNA is the need for synthetic or evolved proteins that
		allow for replication, transcription, and packaging of the XNA (Schmidt, 2010). For our approach to
		expand the genetic code, we decided on the UBP disoG-disoC<SUP>m</SUP> (5-methyl-isocytosine). The 5-methyl 
		derivative shows more stability towards hydrolysis than isoC (Tor and Dervan, 1993). The disoC<SUP>m</SUP>-disoGTP system
		also has an improved behavior concerning the <I>in vitro</I> transcription with T7 RNA polymerase.
		The presence of the 5-methyl group possibly results in a better contact between the template and the polymerase (Tor and Dervan, 1993).
Another aspect is the similarity of the unnatural bases isoG und isoC<SUP>m</SUP> to the natural bases guanine and
 cytosine while being an orthogonal system at the same time. Due to the structural similarity, there is better 
 chance for compatibility with interacting enzymes. In 1992 the Benner lab showed that the <I>in vitro</I> translation 
 of mRNA containing disoC worked with a non-standard tRNA containing the purine complementary disoG inside the anticodon 
 (Bain <I>et al.</I>, 1992). Their cell free experiments showed a high specifity for the incorporation of a non-canonical
 amino acid by the ribosome using this unnatural base. With these stereoisomer of the natural bases it is more likely to 
 achieve an optimized replication, transcription or translation with less adaption of the correspondent enzymes than with 
 hydrophobic UBPs. On top of that, the hydrophobic UBPs are very expensive, because of their complex synthesis. 
 Looking forward to create an autonomous synthetic organism it seems to be impossible to create a biosynthetic pathway
 for unnatural bases that differ a lot from natural bases. Whereas isoG is already known to be metabolic substance of 
 the plant <I>Croton tiglium</I>. Revealing this metabolic pathway can make it usable for any synthetic organism and
 therefore stepping forward towards a fully autonomous synthetic organism.

		</ARTICLE></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> References</H2><ARTICLE><B>Hirao, I., Kimoto, M., and Yamashige, R.</B> (2012). Natural versus Artificial Creation of Base Pairs in DNA: Origin of Nucleobases from the Perspectives of Unnatural Base Pair Studies. Acc. Chem. Res. <B>45</B>: 2055–2065.
<B>Jiang D, Seela F. </B>Oligonucleotide Duplexes and Multistrand Assemblies with 8-Aza-2′-deoxyisoguanosine: A Fluorescent isoG d Shape Mimic Expanding the Genetic Alphabet and Forming Ionophores. J Am Chem Soc. 2010;<B>132</B>:4016–24. doi:10.1021/ja910020n.
<B>Kwok R.</B> Chemical biology: DNA’s new alphabet. Nature. 2012;<B>491</B>:516–8. doi:10.1038/491516a.
<B>Zhang Y, Lamb BM, Feldman AW, Zhou AX, Lavergne T, Li L, <I>et al.</I></B> A semisynthetic organism engineered for the stable expansion of the genetic alphabet. Proc Natl Acad Sci. 2017;<B>114</B>:1317–22.
<B>Yamashige R, Kimoto M, Takezawa Y, Sato A, Mitsui T, Yokoyama S, <I>et al.</I></B> Highly specific unnatural base pair systems as a third base pair for PCR amplification. Nucleic Acids Res. 2012;<B>40</B>:2793–806.
<B>Seela F, Peng X, Li H.</B> Base-pairing, tautomerism, and mismatch discrimination of 7-halogenated 7-deaza-2́-deoxyisoguanosine: Oligonucleotide duplexes with parallel and antiparallel chain orientation. J Am Chem Soc. 2005;<B>127</B>:7739–51. doi:10.1021/ja0425785.
<B>Switzer C, Moroney SE, Benner SA.</B> Enzymatic incorporation of a new base pair into DNA and RNA. J Am Chem Soc. 1989;<B>111</B>:8322–3. doi:10.1021/ja00203a067.
<B>Yang Z, Chen F, Alvarado JB, Benner SA.</B> Amplification, mutation, and sequencing of a six-letter synthetic genetic system. J Am Chem Soc. 2011;<B>133</B>:15105–12. doi:10.1021/ja204910n.
<B>Dunn DB, Smith JD.</B> Effects of 5-halogenated uracils on the growth of <I>Escherichia coli</I> and their incorporation into deoxyribonucleic acids. Biochem J. 1957;<B>67</B>:494–506. doi:10.1042/bj0670494.
<B>Marlière P, Patrouix J, Döring V, Herdewijn P, Tricot S, Cruveiller S, <I>et al.</I></B> Chemical evolution of a bacterium’s genome. Angew Chemie - Int Ed. 2011;<B>50</B>:7109–14.
<B>Rich, A.</B> (1962). On the problems of evolution and biochemical information transfer. Horizons Biochem.: 103–126.
<B>Malyshev, D.A. and Romesberg, F.E.</B> (2015). The expanded genetic alphabet. Angew. Chem. Int. Ed. Engl. <B>54</B>: 11930–44.
<B>Ma, R., Yang, Z., Huang, L., Zhu, X., Kai, L., Cai, J., Wang, X., and Xu, Z. </B>(2010). Construction of an efficient Escherichia coli cell-free system for in vitro expression of several kinds of proteins. Eng. Life Sci. <B>10</B>: 333–338.
<B>Laos, R., Thomson, J.M., and Benner, S.A.</B> (2014). DNA polymerases engineered by directed evolution to incorporate non-standard nucleotides. Front. Microbiol. <B>5</B>: 1–14.
<B>Chen, F., Yang, Z., Yan, M., Alvarado, J.B., Wang, G., and Benner, S.A. </B>(2011). Recognition of an expanded genetic alphabet by type-II restriction endonucleases and their application to analyze polymerase fidelity. Nucleic Acids Res. <B>39</B>: 3949–3961.
<B>Georgiadis, M.M., Singh, I., Kellett, W.F., Hoshika, S., Benner, S.A., and Richards, N.G.J.</B>(2015). Structural Basis for a Six Nucleotide Genetic Alphabet. J. Am. Chem. Soc.<B> 137</B>: 6947–6955.
<B>Bain, J.D., Switzer, C., Chamberlin, R., and Bennert, S.A.</B> (1992). Ribosome-mediated incorporation of a non-standard amino acid into a peptide through expansion of the genetic code. Nature <B>356</B>: 537–539.
<B>Leal, N. a, Kim, H., Hoshika, S., Kim, M., Carrigan, M. a, and Benner, S. A.</B>(2015). Transcription, Reverse Transcription, and Analysis of RNA Containing Arti fi cial Genetic Components. ACS Nano <B>4</B>: 407.
<B>Seo, Y.J., Matsuda, S., and Romesberg, F.E.</B> (2009). Transcription of an Expanded Genetic Alphabet. J. Am. Chem. Soc. <B>131</B>: 5046–5047.
<B>Malyshev, D.A., Dhami, K., Lavergne, T., Chen, T., Dai, N., Foster, J.M., Corrêa, I.R., and Romesberg, F.E.</B> (2014). A semi-synthetic organism with an expanded genetic alphabet. Nature <B>509</B>: 385–8.
<B>Zhang, Y., Lamb, B.M., Feldman, A.W., Zhou, A.X., Lavergne, T., Li, L., and Romesberg, F.E.</B> (2017). A semisynthetic organism engineered for the stable expansion of the genetic alphabet. Proc. Natl. Acad. Sci. <B>114</B>: 1317–1322.
<B>Schmidt, M.</B> (2010). Xenobiology: A new form of life as the ultimate biosafety tool. BioEssays <B>32</B>: 322–331.
<B>Tor, Y. and Dervan, P. </B>(1993). Site-specific enzymic incorporation of an unnatural base, N6-(6-aminohexyl) isoguanosine, into RNA. J. Am. Chem. Soc. <B>115</B>: 4461–4467.

		
		</ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>