Team:Bielefeld-CeBiTec/Project/toolbox/analysing
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Analyzing
Short Summary
As part of our toolbox , structural analysis of a protein could be used to study    distances between non-canonical amino acids with Foerster Resonance Energy Transfer    (FRET) . This provides measuring distances within specific incorporated amino acids in   the target protein to gain insight into protein folding or structural changes under    different conditions.
To demonstrate this tool, we developed a prion detection assay. We used the yeast    prion Sup35 as a model protein and incorporate two non‑canonical amino acids    ( p-acetophenylalanine and propargyllysine ). After purification, the recombinant    produced Sup35 could be labeled with two different fluorophores ( Cyanin 3 and Cyanin 5 ).   The emission spectra of the fluorophores depend on their distance.    When this test protein gets in contact with prions, the prions conformational changes    result in the change of the fluorophores spectra. Therefore, the test prion could be    used to detect prions in medical samples.
Structural Analysis with Non-canonical Amino Acids
The structure of proteins could be detected through protein crystallography.     However, there are some problems when it comes to highly flexible proteins, or    proteins which change their structure within different conditions. To analyze these   proteins and the changes in their conformation we want to establish a tool, that allows   to detect changes in protein conformation.  For the detection of those changes, two    amino acids are incorporated at specific positions into the protein. These amino acids    could then be labeled with chromophores, enabling the measurement of the proteins    distances with Foerster Resonance Energy Transfer (FRET) (Lembke, 2011, Kim et al., 2013).
The first step is the incorporation of the non-canonical amino acids. In proteins which    contain no cysteines naturally (cysteines are the only canonical amino acids, that   could be labeled specific) or in which the exchanges of cysteines does not influence    the structure, only one ncAA and one cysteine at specific points need to be incorporated   to be labeled. In proteins that contain cysteines, two ncAAs need to be incorporated for    the labeling (Kim et al., 2013).
Non-canonical amino acids could be incorporated by orthogonal tRNA/aaRS pairs using the amber stop codon. However, this allows only the incorporation of one    non-canonical amino acid. To incorporate the second amino acid, another orthogonal    amino acid has to be used for the incorporation. Another codon that could be repurposed is the rarely used leucine codon CUA. With the use of this and the amber codon, two different ncAAs could be used. For structural analysis, the amino acids are specific labeled with chromophores. This labeling (shown in figure 1)    is possible due to the functional groups of the amino acids, which could    form a covalent bond to the fluorophores in a chemical reaction.    After the protein is labeled, the fluorescence of the chromophores could be measured to    draw conclusions on the distance of the ncAAs from each other (Brustad et al., 2008, Kim et al., 2013).
Figure 1: Target protein labeled with fluorophores.
The ncAAs AcF and PrK are incorporated in the target protein. After bi-orthgonal chemical conjugation the ncAAs are coupled with the fluorescent dyes cyanin 3 (Cy3) and cyanin 5 (Cy5).
Propargyllysine (PrK)
One amino acid which provides a functional group different to the canonical amino    acids is propargyllysine. The propargyl group of PrK could form a covalent bond to    acidic groups in a click chemistry reaction. PrK is commercially available or could    be synthesized chemically in two steps starting with Boc‑I‑Lys‑OH (Kim et al., 2013).     The click chemistry reaction is performed at neutral pH, native buffers and    temperatures of 4° C to 37 °C. However, for the click‑chemistry reaction copper    is required which is toxic for living cells, so PrK could not be used for in    vivo labeling (Kim et al., 2013).
Name:   Propargyllysine
Short:  PrK
CAS:  1428330-91-9
MW:  228.25 g mol-1
Storage:           4 °C
Source: Sichem
Prize:  1g - £300.00
Function:          Propargyl group for click-chemistry reaction
Figure 2: Structure of PrK
Propargyllysine (PrK).
p‑Acetylphenylalanine (AcF)
Another amino acid with an additional functional group to the canonical amino acids    is p‑acetylphenylalanine. The ketone group of AcF is able to build a covalent bond to a    hydroxylamine coupled dye in a hydrazide reaction. This reaction is carried out at low     pH‑values which causes problems with certain proteins (Kim et al., 2013).
Name:   p‑Acetylphenylalanine
Short:  AcF
CAS:  122555-04-8
MW:  207,23 g mol-1
Storage:           -20 °C
Source: abcr
Prize:  1g - £509.00
Function:          Ketone group for hydrazide reaction
Figure 3: Structure of AcF
p-Acetylphenylalanine (AcF).
Foerster Resonance Energy Transfer (FRET)
Foerster Resonance Energy Transfer or Fluorescence Energy Transfer, short FRET,    describes an energy transfer between two chromophores. During this process the donor    chromophore is excited by light of a certain wavelength and transfers the energy to the acceptor chromophore if they    are within a certain distance to each other. In biochemistry, FRET is mostly used as    measurement tool with the help of fluorescent dyes. Using FRET, the measuring of   distances from 1 to 10 nm is possible (Brustad et al., 2006). The FRET process is shown in Figure 4.
Figure 4: Animations of a FRET fluorophore pair
Animation of the distance dependent energy transfer of two fluorophores.
The FRET efficiency (E) is used to estimate the intramolecular distance of molecules.
E=[1+(r/R0)6)]-1
E  FRET efficiency
r  intermolecular distance
R0 Foerster distance for a given dye pair
Cyanin 3 and Cyanin 5
Cyanin 3 (Cy3) in combination with Cyanin 5 (Cy5) is a chromophore pair which is    suitable for FRET measurements. Cy3 operates as the donor dye and Cy5 as the acceptor   dye. The extinction and emission spectra of both chromophores is shown in Figure 5 (Kim et al., 2013).
Figure 5: Spectra of the fluorophore pair
Extinction and emission spectra of Cy3 and Cy5.
For the labeling of the noncanonical amino acids with these chromophores a functional   group is required to build a covalent bond in chemical conjugation. PrK could be    labeled with Cy3‑azide and AcF with Cy5‑hydrazide. Both chromophores are relatively advantageously priced   in comparison to other chromophore pairs and commercially available from Lumiprobe .
Prion Detection Assay
Prions
Prions are proteins that could infect other proteins to change their conformation.  This is often causing a loss of function and aggregation of these proteins. Prions are   the cause for diseases like transmissible spongiform encephalopathies (TSEs),   neurodegenerative disorders that effect humans and animals (Wickner et al., 2015).
Sup35
Sup35 is a yeast translation termination factor from Saccharomyces cerevisiae.  The prion form of Sup35 is known to form amyloids consisting of beta-sheet rich protein  aggregates with beta-strands perpendicular to the long axis of the filament. The domain   responsible for the conformational change is the NM region. This region of the protein   contains two different sections. The N‑section (amino acids 1‑124) forms the major part   of the amyloid core that that directs the protein into the prion form. The M-section   (amino acids 124‑250) is highly charged and provides the solubility to the native form   of Sup35. In the prion form the M region changes its conformation to a beta‑sheet rich   conformation, while the N‑section stays nearly unchanged in its conformation (Mukhopadhyay et al., 2007; Wickner et al., 2015).
References
Brustad, E. M., Lembke, E. A., Schultz, P. G., Dentz, A. A.(2008). A General and Efficient Method for the Site-Specific Dual-Labeling of Proteins for Single Molecule Fluorescence Resonance Energy Transfer. American Chemical Society. 130: 17664-17665..
Kim, J., Seo, M., Lee, S., Cho, K., Yang, A., Woo, K., Kim, H., Park, H.(2012). Simple and Efficient Strategy for Site-Specific Dual Labeling of Proteins for Single-Molecule Fluorescence Resonance Energy Transfer Analysis. Analytical Chemistry.85: 1468-1474.
Lembke, E. a.(2011). Site-Specific Labeling of Proteins for Single-Molecule FRET Measurements Using Genetically Encoded Ketone Functionalities. Bioconjugation Proocols: Strategies and Methods in Molecular Biology. 751: 3-15.
Mukhopadhyay, S., Krishnan, R., Lembke, E. A., Lindquist, S., Deniz, A. A.(2007)A natively unfolded yeast prion monomer adopts an ensemble of collapsed and rapidly fluctuating structures.PNAS.104(8):2649-2654.
Wickner, R. B., Shewmaker, F. P., Bateman, D. A., Edskes, H. K., Gorkovsky, A., Dayani, Y., Bezsonov, E. E.82015) Yeast Prions: Structure, Biology, and Prion-Handling Systems. Microbiology and Molecular Reviews. 79(1):1-17.
