Team:Bielefeld-CeBiTec/Composite Part
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Composite Parts
Short Summary
For the application of our best composite part, we decided to nominate our reporter signal enhancing system BBa_K2201373 .      This part contains a T3 RNA Polymerase with an inverted mRFP under T3 RNA polymerase control for the enhancing of reporter signals. It is an improved reporter and a genetic circuit that could report even weak expression levels. This part was designed based on the model of an amplifier in electrical engineering to intensify an existing input signal and could be used in a broad range of synthetic biology applications. We used this part for our selection system for the incorporation of non-canonical amino acids and demonstrate the advantages of our system in comparison with standard reporter in an integrated modelling and wet lab characterizations.
Usage and Biology
The T3 RNA polymerase is highly specific to T3 promoters and orthogonal to the T7 RNA polymerase. Therefore, it can be widely utilized in synthetic biology, even in expression strains which encode for a T7 RNA polymerase like E. coli BL21. This orthogonality is due to the promoter specifity of the different polymerases (Davis et al., 1971; McGraw et al., 1985).       We applied this system to enhance a reporter in genetic circuits to make even weak gene expression visible.
At the moment, fluorescent proteins with an emission wavelength within the visible spectra are used to report expression of the gene of interest. Therefore, the CDS of the fluorescent protein is placed downstream of the CDS of the target protein without a terminator or promoter in between. The expression level of the target protein is nearly the same as the expression of the fluorescent protein. The fluorescence of the reporter protein indicates if the gene of interest was translated. However, this system is limited to strong expression, which generate a sufficienly strong fluorescence signal.
Therefore it is impractical for several applications which involve only weak expression. For our project, we needed a reliable and sensitive reporter to detect the expression of the gene of interest on a selection plasmid. A low expression of the target gene is essential for the selection system. No fluorescence was detectable, when the CDS of mRFP was placed downstream of the gene of interest. Through the function of the gene of interest, we knew it was expressed. To address this reporter challenge we built a genetic circuit following the model of an amplifier used in electrical engineering.
Basic amplifiers were previously submitted to the Registry of biological parts e. g. by iGEM Cambridge 2009 . They build a simple circuit using an activator, which increased the transcription of a reporter under control of a second promoter. To explain their system they used the term "Polymerases per second" (PoPs). This unit defined as the flow of RNA polymerase molecules over a promoter region per second. The system developed by Cambridge 2009 (Figure 1) could increase the number of PoPs.
Figure 1: Signal strenthening system of iGEM Cambridge 2009 .
To improve this system we decided to use an activator, which translates the reporter DNA and does only induce the translation of the polymerase which then translates the gene of interest. Amplification was achieved by an RNA polymerase, which transcribes the reporter DNA, multiple times and enhances the signal significantly.
Functional Parameters
To design a genetic circuit that amplifies a reporter signal we decided to use an RNA polymerase, orthogonal to the native E. coli RNA polymerase. Therefore, the T3 RNA polymerase previously characterized by iGEM Peking 2010 was deployed. The T3 RNA polymerase is highly specific to T3 promoters and orthogonal to the E. coli DNA polymerases and even to the T7 RNA polymerase. We decided not to use the T7 RNA polymerase, because it is already part of expression strains like BL21 and would therefore prevent the application in such strains. The construction of our designed composite part comprising a standard mRFP reporter is shown in Figure 2.
Figure 2: Construction of the standard mRFP reporter and the genetic circuit for the amplification of mRFP expression. In construct 1 the CDS fot the mRFP transcript is downstream the CDS of the gene of interest. In construct 2 the CDS of the mRFP is downstream a T3 promoter and the CDS coding for the T3 RNA polymerase is downstream the CDS of the gene of interest.
Figure 2 shows two reporter constructs for gene expression quantification. The first one is the standard reporter for the gene expression of the target gene, using mRFP as reporter gene. If the gene of interest is expressed, the mRFP is expressed at nearly the same level. The mRFP fluorescence reveals the gene expression of the gene of interest. This system is suitable for but also limited to high expression rates. Low, expression of the target  gene is associated with a low expression of mRFP and thus not visible.
The second system is our genetic circuit consisting of a CDS of the target gene upstream of the CDS for a T3 DNA polymerase encoding sequence. Therefore, the expression of the reporter gene is nearly on the same level as the expression of the target gene. The expressed T3 RNA polymerase transcribes the mRFP under the control of the T3 promoter.       To demonstrate the advantages of our improved construct, we modelled the amount of mRFP transcript for both constructs.       If we assume the expression of the gene of interest is low and only one E. coli RNA polymerase with a chain elongation rate of 50 nucleotides per second translates the both products, construct 1 produces 1 mRFP transcript in 16 seconds. Construct 2 expresses 1 T3 RNA polymerase transcript every 52 seconds. After translation (with an average  translation rate of 20 amino acids per second ~42 sec), these polymerases transcribe the mRFP transcript with a chain elongation rate of 170 nucleotides per second. Therefore, every T3 RNA polymerase generates one mRFP transcript every 4.7 seconds. The resulting amount of mRFP transcripts  is shown in Figure 3. The script for our modelling can be found here .
Figure 3:Modeling on the amount of mRFP transcript transcribed through the two different models. In model 1 mRFP CDS lies is of the CDS of the gene of interest. In model 2 the CDS of the mRFP is downstream a T3 promoter and the CDS coding for the T3 RNA polymerase is downstream the CDS of the gene of interest.
The number of mRFP transcripts is a lot higher with construct 2 (our reporter constuct) even though model leaves out the fact that one transcript of the T3 RNA polymerase could be translated several times and enhance the signal even more. Despite the simplicity of this model, it shows the advantages of using the T3 RNA polymerase for signal enhancing.
Figure 4: Two Smear of two clones containing only mRFP under an uninduced T7 promoter (-) and containing the mRFP enhancing system under the same promotor (+), after 12 h of incubation at 37 °C.
To demonstrate the signal enhancing system we cloned either the part E1010 or our signal enhancing system ( BBa_K2201373 ) downstream of the part BBa_K2201900 . This plasmid was characterized in cells containing our positive selection plasmid. A smear of the transformants is shown in Figure 4. The uninduced T7-promoter  promoter leads only to a basal transcription level. Despite this weak transcription, the mRFP in the cells containing the signal enhancing system is clearly visible, while cells without the signal enhancing system remain colorlesse.
Figure 5: Picture of a negative selection round. The clone still containing the positive selection plasmid, thus the mRFP enhancing system, is red.
For the selection of tRNA/aminoacylsynthetase to incorporate non-canonical amino acids, we needed to check if clones still contain the positive selection plasmid in the negative selection round. Therefore, we decided to incorporate the mRFP signal enhancing system downstream of the CDS of our positive selection plasmid BBa_K2201900 . If the cells contain this plasmid in the negative selection, they should be visible as red clones. In contrast, the clones containing the negative selection plasmid should be colorless. A picture of one round of the negative selection (Figure 5) shows that one clone is red, thus demonstrating the function of the system. The results of our selection system can be found here .
Applications
An important criteria for a composite part is the possibility to use this part in further iGEM projects. In addition to our application for this part, there are a lot of potential applications for a reporter signal enhancement. Our system provides a reliable and especially sensitive reporter, thus applications which require these could be improved by using our system. Some possible applications are shown in Figure 6.
Figure 6: Potential applications for the reporter signal enhancing system.
The strength of the reporter signal enhancing system lies in its strong and specific signal despite of a very weak initial gene expression. This enables usage in the analysis of weakly transcribed genes in metabolic pathway and flux analyses as well as a variety of diagnostic application which are dependent on strong, specific signals. Furthermore use of the reporter signal enhancer system in the analysis of expression in vitro and in vivo is feasible.
Composite Parts
Table 1: All composite parts of iGEM Bielefeld-CeBiTec 2017. For a detailed description of the parts, please refer to the results or corresponding sites in the Registry of Parts .
Name
Type
Description
Designer
Length
BBa_K2201373
Composite
T3 Polymerase with inverted mRFP under T3 promoter control for signal enhancing
Svenja Vinke
3411
BBa_K2201201
Composite
Pyrrosyl tRNA/aminoacyl-synthetase for the incorporation of propargyllysine
Svenja Vinke
1997
BBa_K2201202
Composite
Tyrosyl tRNA/aminoacyl-synthetase for the incorporation of p-acetophenylalanine
Svenja Vinke
1558
BBa_K2201203
Composite
Tyrosyl tRNA/aminoacyl-synthetase for the incorporation of p-acetophenylalanine in response to CUA
Svenja Vinke
1558
BBa_K2201204
Composite
Tyrosyl tRNA/aminoacylsynthetase for the incorporation of L-(7-hydroxycoumarin-4-yl)ethylglycine
Svenja Vinke
1558
BBa_K2201343
Composite
Fusion protein of CFP and YFP with an amber codon in the linker under T7 promoter control
Svenja Vinke
1483
BBa_K2201024
Composite
Left flanking sequence + Terminator + repressing lacO
Lennard Karsten
1248
BBa_K2201025
Composite
Terminator + right flanking sequence
Lennard Karsten
1248
BBa_K2201028
Composite
Repair tempplate for HDR-mediated integration of cas9 into E. coli BL21 (DE3)
Lennard Karsten
3487
BBa_K2201032
Composite
sgRNA MAX target + mRFP-sacB fusion protein
Lennard Karsten
3487
BBa_K220140
Composite
sgRNA R-target mutA
Markus Haak
231
BBa_K2201041
Composite
sgRNA mutG
Markus Haak
231
BBa_K2201042
Composite
sgRNA R-target mut&#916
Markus Haak
231
BBa_K2201043
Composite
sgRNA R-target mutT
Markus Haak
231
BBa_K2201044
Composite
sgRNA R-target mutC
Markus Haak
231
BBa_K2201045
Composite
sgRNA MAX-target mutA
Markus Haak
256
BBa_K2201046
Composite
sgRNA MAX-target mutG
Markus Haak
256
BBa_K2201047
Composite
sgRNA MAX-target mut&#916
Markus Haak
256
BBa_K2201048
Composite
sgRNA MAX-target mutT
Markus Haak
256
BBa_K2201049
Composite
sgRNA MAX-target mutC
Markus Haak
256
BBa_K2201070
Composite
sgRNAs R-target mutA,G
Markus Haak
470
BBa_K2201071
Composite
sgRNAs R-target mutT,C
Markus Haak
470
BBa_K2201072
Composite
sgRNAs R-target A,G,&#916
Markus Haak
709
BBa_K2201073
Composite
sgRNAs MAX-target mutA,G,&#916,T,C
Markus Haak
1187
BBa_K2201074
Composite
sgRNAs MAX-target mutA,G
Markus Haak
520
BBa_K2201075
Composite
sgRNAs MAX-target mutT,C
Markus Haak
520
BBa_K2201076
Composite
sgRNAs MAX-target mutA,G,&#916
Markus Haak
784
BBa_K2201077
Composite
sgRNAs MAX-target mutA,G,&#916,T,C
Markus Haak
1312
BBa_K2201208
Composite
Cysteinyl-lysinyl-tRNA Synthetase/tRNA pair
Daniel Bergen
1860
BBa_K2201243
Composite
Fusion protein of CFP and YFP with an amber codon in the linker
Svenja Vinke
1431
BBa_K2201250
Composite
elastine consensus for PRe-RDL
Daniel Bergen
52
BBa_K2201251
Composite
silk consensus for PRe-RDL
Daniel Bergen
55
BBa_K2201320
Composite
Functional GFP-streptavidin fusion protein with medium gly-gly-ser linker and T7-promoter and RBS
Yannic Kerkhoff
1169
BBa_K2201321
Composite
NM-domain of Sup35 with an amber codon at position 21 under T7 promoter control
Svenja Vinke
837
BBa_K2201342
Composite
Fusion protein of RFP and GFP with an amber codon in the linker under T7 promoter control
Svenja Vinke
1489
BBa_K2201360
Composite
Our part BBa_K2201260
Maximilian Edich
7649
BBa_K2201361
Composite
Our part BBa_K2201261 after the Carboxysome and T7 promoter
Maximilian Edich
6946
BBa_K2201362
Composite
Our part BBa_K2201262 after the carboxysome and T7 promoter
Maximilian Edich
6946
BBa_K2201363
Composite
Our part BBa_K2201263 after the carboxysome and T7 promoter
Maximilian Edich
6946
BBa_K2201364
Composite
Our part BBa_K2201264
Maximilian Edich
6946
BBa_K2201365
Composite
Our part BBa_K2201265 after the Carboxysome and the T7 promoter
Maximilian Edich
6946
BBa_K2201366
Composite
Our part BBa_K2201266 after the Carboxysome and the T7 promoter
Maximilian Edich
6946
BBa_K2201367
Composite
Our part BBa_K2201267 after the Carboxysome and the T7 promoter
Maximilian Edich
6955
BBa_K2201368
Composite
Our part BBa_K2201268 after the Carboxysome and the T7 promoter
Maximilian Edich
6937
BBa_K2201404
Composite
Reverse Sequence of I746909
Olga Schmidt
924
BBa_K2201900
Composite
Positive selection plasmid
Svenja Vinke
1158
BBa_K2201901
Composite
Negative selection plasmid
Svenja Vinke
681
References
Davis, R. W. & Hyman, R. W (1971)J. Mol. Biol 62, 287-301.
McGraw, N. J., Bailey, J. N., Cleaves, G. R., Dembinski, D. R., Gocke, C. R., Joliffe, L. K., MacWright, R. S.&McAllister, W. T. (1985) Nucleic Acids Res. 13, 6753–6766.
BBa_K2201373 parameters
