Team:Bielefeld-CeBiTec/Results/toolbox/labeling
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Labeling
Short Summary
For the labeling of a target protein in vivo a fluorescent amino acid should be incorporated through the amber stop codon. Our aim is to provide a tRNA/aminacyl-synthetase (tRNA/aaRS) to the iGEM community to easily incorporate the fluorescent amino acid L‑(7‑hydroxycoumarin‑4‑yl) ethylglycin (CouAA) during translation in response to the amber stop codon.
To demonstrate the advantages of this tool, we colocalized the ribulose 1,5‑bisphosphat carboxylase oxygenase (RuBisCo) from Halobacillus neaplitanus and the carboxysome in E. coli cells. The carboxysome was labeled with the green fluorescent protein (GFP) and the RuBisCo was labeled with the fluorescent amino acid at different positions. We showed that the RuBisCO is located inside the carboxysome, while our control was located in the whole cytoplasm.
Evolved Tyrosine tRNA/aminoacyl-synthetase (TyrRS) for the Incorporation of the Fluorescent Amino Acid L‑(7‑hydroxycoumarin‑4‑yl) ethylglycin (CouAA)
For the construction of the synthetase, we decided to use the Tyrosine tRNA/aminoacyl-synthetase from Methanococcus jannaschii with mutations at the following eight positions: Tyr32Glu, Leu65His, Ala67Gly, His70Gly, Phe108Tyr, Gln109His, Asp158Gly, and Leu162Gly, as described by Schultz, 2006. An alignment of the resulting synthetase and the wildtype is shown in Figure 1. We ordered the synthetase as gene synthesis and cloned it via Gibson assembly in pSB1C3. For the experiments, the CDS for the synthetase needs to be in a low copy plasmid. Therefore, we choose to insert it into pSB3T5 with a bio brick assembly.
Figure 1:Amino acid sequences of the wildtype Methanococcus jannashii TyrRS and the evolved CouAA aaRS . This alignment of the amino acid sequences of the wildtype Methanococcus jannashii TyrRS and the evolved CouAA aaRS with Clustal Omega .
To test whether the synthetase really incorporates CouAA, we transformed the synthetase in pSB3T5 and our composite part BBa_ K2201331 in pSB1C3 as test protein in E. coli BL21 DE3. The part K2201231 contains the CDS for the protein Sup35 with an amber stop codon at position 21 under control of a T7-promoter. If the synthetase works, it will incorporate CouAA in response to the amber codon at position 21. We expressed the test protein, which contains a His6-tag, and purified it through NiNTA chromatography. After the purification, the absorbance and emission spectra of the purified protein were measured via UV/VIS-spectralphotometry. The spectra of the purified protein and the spectra of CouAA in LB-media are shown in Figure 2.
Figure 2: Absorption and emission spectra of CouAA. Absorption and emission spectra of CouAA in LB (left) and incorporated into the protein Sup35 (right).
The two spectra shown in Figure 2 look very similar to the spectra recorded by Wang 2006. Little variances in the absorbance spectra could be caused by different media or the elution buffer of the eluate. A comparison of the fluorescence of Sup35 containing CouAA and a negative control containing cysteine at the same position are shown in Figure 3.
Figure 3: Normalized fluorescence intensity of Sup35 containing CouAA. Normalized fluorescence intensity on the absorbance at 280nm (indicator for the protein amount)  of purified Sup35 containing a cysteine at position 21 (-CouAA) and containing the fluorescent amino acid CouAA at position 21 (+CouAA).
The normalized fluorescence intensity of Sup35 with incorporated CouAA is significantly higher than the fluorescence of the Sup35 with cysteine instead of CouAA. This proves that our evolved aaRS incorporates CouAA in response to the amber stop codon.
To prove the labeling in vivo, the cotransformants with BBa_K2201204 in pSB3T5 and BBa_K2201331 in pSB1C3 were cultivated in LB containing 1 mM CouAA. After 8 h of cultivation (as described in the expression of recombinant proteins), 1 mL of the culture was washed with 1xPBS and diluted 1:10. To immobilize the cell for fluorescence microscopy, 100 µL of the cells were mixed with 100 µL 2 % agarose. The fluorescence microscopy was performed with a confocal laser scanning microscope using a DAPI filter to detect the fluorescence signal of CouAA.
Figure 4: Confocal laser scanning microscopy of Sup35 containing CouAA at amino acid position 21. The fluorescence of CouAA was visible through a DAPI filter.
The fluorescence microscopy in Figure 4 shows a constant fluorescence signal from CouAA in Sup35 in the whole cell indicating that Sup35 is localized in the whole cytoplasm.
All in all, the labeling tool for localization in vivo and in vitro worked fine. In consequence we were able to provide a functional aaRS for the incorporation of a strong fluorescence amino acid to the iGEM community.
Construction of RuBisCO mutants containing the amber stop codon
To incorporate the fluorescent amino acid at specific sites, we first had to choose the position and then use a site-directed mutagenesis to incorporate the amber stop codon at this position. For labeling issues, the fluorescent amino acid should not influence the protein folding or the activity. Furthermore, according to our expert Prof. Dr. Nedilijko Budisa, the yield of the protein could be influenced by the position the fluorescent amino acid is incorporated at. Therefore, we decided to try different positions at which the non-canonical amino acid is incorporated, all at permissive sites of the RuBisCo.
The permissive sites were detected by the alignment with Clustal Omega of different analogical RuBisCo from Thioalkalivibrio sulfidiphilus, Thioalkalivibrio denitrificans, Thiothrix nivea, Thermothiobacillus tepidarius, Acidobacillus caldus and Acidiferrobacter thiooxydans. If amino acids at the same position are heterologous, the amino acid seems to be unimportant for the functioning and folding of the protein. These permissive sites of the enzyme are suitable for the incorporation of the noncanonical fluorescent amino acid.
Figure 5: Multiple amino acid sequence alignment of analogical small subunit RuBisCo variants. The amino acid sequences of the RuBisCo variants from Thioalkalivibrio sulfidiphilus(WP_012639733.1), Thioalkalivibrio denitrificans(WP_058575622.1), Thiothrix nivea(WP_012823800.1), Thermothiobacillus tepidarius(WP_066099403), Acidobacillus caldus(WP_002708379.1) and Acidiferrobacter thiooxydans(WP_045467882.1) were aligned using Clustal Omega .
Due to the alignment, we decided to incorporate the fluorescent amino acid at position 2 and position 111 of the small subunit, as well as, at position 474 of the large subunit. In addition, we decided to try different combinations of these positions to find out if it's possible to intensify the fluorescence signal.  Thus, we constructed the following seven RuBisCO mutants:
BBa_K2201261 with a TAG at amino acid position 2 of the small subunit
BBa_K2201262 with a TAG at amino acid position 111 of the small subunit
< BBa_K2201263 with a TAG at amino acid position 474 of the large subunit
BBa_K2201264 with a TAG at amino acid position 2 and 111 of the small subunit
BBa_K2201265 with a TAG at amino acid position 2 of the small subunit and amino acid position 474 of the large subunit
BBa_K2201266 with a TAG at amino acid position 111 of the small subunit and amino acid position 474 of the large subunit
BBa_K2201267 with a TAG at amino acid position 2 and 111 of the small subunit and amino acid position 474 of the large subunit
To compare the new tool using the noncanonical amino acid to the labeling with fluorescence proteins, we designed BBa_K2201260 containing a fusion of the CDS for the small subunit of the RuBisCO and the CDS for mRFP connected with a short linker.
Colocalization of the RuBisCO and the carboxysome
To colocalize the RuBisCO in vivo, a plasmid containing the CDS a carboxysome-GFP fusion protein and one of the RuBisCO variants containing the amber stop codon were cotransformed with BBa_K2201204 , containing the CouAA-aaRS and tRNA in pSB3T5. The seven different variants and the negative control were prepared like described previously for the CLSM images. The images of the negative control and the variant containing one CouAA at amino acid position 2 of the small subunit are shown in figure 6. The construct BBa_K2201360 containing the fusion protein of mRFP and RuBisCO showed no visible fluorescence. Maybe the linker was not suitable, so the mRFP was not able to fold correctly.
Figure 6: CLSM images of the carboxysome and the RuBisCo containing no fluorescent amino acid (BBa_K2201368) and containing CouAA at position 2 of the small subunit (BBa_K2201361). For both samples the left picture shows the image generated through the DAPI, the middle the image through the GFP filter and the right picture an overlay of the light microscopy image the both filters.
Unlike Sup35 (figure 4), the RuBisCO is not localized in the whole cytoplasm. It seems like most of the RuBisCo is localized in the carboxysome and only a small amount is localized over the whole cell. The fluorescence image of BBa_K2201362 (stop codon at position 111 of the small subunit) shows the same localization of the RuBisCO inside the carboxysome. Furthermore, there is no visible difference in the fluorescence intensity between BBa_K2201361 and BBa_K2201362 (figure 6 and 7). In contrast, BBa_K2201363 (CouAA at position 474 of the large subunit) shows no visible fluorescence (figure 7). This indicates that the CouAA is incorporation position has a strong influence on the fluorescence signal, like assumed by our expert Prof. Nediljko Budisa.
Figure 7:CLSM images of the carboxysome and the RuBisCo containing CouAA at position 111 of the small subunit (BBa_K2201362) and at position 474 of the large subunit (BBa_K2201363), respectively. For both samples the left picture shows the image generated through the DAPI, the middle the image through the GFP filter and the left picture an overlay of the light microscopy image and both filters.
Furthermore, we wanted to investigate if the incorporation of more than one fluorescent amino acid strengthens the fluorescence signal. Therefore, we prepared the four RuBisCO variants containing more than one amber stop codon like described previously for fluorescence microscopy. The resulting images are shown in figure 8.
Figure 8: CLSM images of the carboxysome and the RuBisCo containing more than one CouAA at different positions. For all samples the left picture shows the image generated through the DAPI, the middle the image through the GFP filter and the left picture an overlay of the light microscopy image and the  filters. BBa_K2201364 contains an amber codon at position 2 and 11 of the small subunit. BBa_K2201365 contains an amber codon at position 2 of the small subunit and 474 of the large subunit. BBa_K2201366 contains an amber codon at position 111 of the small and 474 of the large subunit. BBa_K2201367 contains an amber codon at position 2 and 111 of the small and 474 of the large subunit.
Compared to the fluorescence of the samples containing only one amber codon (BBa_K2201361, BBa_K2201362 and BBa_K2201363), as well as those, containing all three amber codons, (BBa_K2201367) the fluorescence of the samples the two samples BBa_K2201264 and BBa_K2201265, containing two amber codons seemed to be the strongest. In these the samples containing two amber codons, and thus two CouAAs, the localization of the RuBisCO inside the carboxysome is visible. The addition of another amber codon could not increase the fluorescence intensity. This could be a result of a lower yield or the relatively low fluorescence of CouAA incorporated at position 474 of the large subunit.
All in all, the position of the CouAA and the number of the incorporated amino acids influence the fluorescence intensity and thus the quality of the localization in vivo, but there is no general optimum in the number and position of the ncAAs. For the RuBisCo the incorporation of two CouAA resulted in the best fluorescence microscopy pictures. We proved that our tool is suitable for the localization in vivo and we were able to localize the RuBisCo inside the carboxysome.
Table 1: Fluorescence visible through the DAPI and the GFP filter. Visible fluorescence and intensity through the GFP and DAPI filter for the carboxysome, tagged with GFP, and RuBisCO containing the fluorescent amino acid CouAA at different positions.
Construct name
TAG 2
TAG 111
TAG 474
CouAA fluorescence
GFP fluorescence
BBa_K2201368
+
BBa_K2201361
X
+
+
BBa_K2201362
X
+
+
BBa_K2201363
X
+
BBa_K2201364
X
X
++
+
BBa_K2201365
X
X
++
+
BBa_K2201366
X
X
+
+
BBa_K2201367
X
X
X
+
+
References
Wang, J., Xie, J., Schultz, P. G.(2006). A Genetically Encoded Fluorescent Amino Acid. American Chemical Society.128:8738-8739
