Team:Bielefeld-CeBiTec/Project/unnatural base pair/challenges
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Challenges
Short Summary
Expanding the genetic code with unnatural base pairs leads to a variety of challenges. Given that all standard lab methods are designed for a four-letter DNA, methods such as PCR, cloning and sequencing have to be adapted. Additionally, bioinformatic software tools such as BLAST do not work with a six-letter DNA or an expanded amino acid repertoire. Therefore, to validate our work, the development of new methods was essential for our project.
Biological Challenges
All organisms share the same four-letter DNA, composed of the two pyrimidine bases cytosine and thymine and the two purine bases adenine and guanin. This composition of the DNA is the result of billions of years of evolution. The entire cellular machinery is designed to work with or to be compatible with this four-letter DNA composition. Therefore, the addition of two new bases to the DNA proves very challenging from a biological point of view. Firstly, the initial integration of unnatural bases into the DNA is based on the synthesis of modified DNA, which is by now available from almost all DNA synthesis companies. The big challenges arise when it comes to the retention and preservation of these unnatural bases. Since the cell detects them as “foreign”, ergo as mutations, a system has to be put in place to prevent the correction of these bases. Furthermore, the chemical stability of the unnatural base pairs has to be considered. The acceptance of the unnatural bases also poses big challenges, since the entire transcription and translation machineries have to be adapted to accept the unnatural base pairs. This can be achieved by directed evolution of the ribosomes and the tRNA/aminoacyl-tRNA synthetases.
Technical Challenges
Given that most bioinformatic tools are designed to work with only four-letter DNA and the 20 canonical amino acids, expanding the genetic code leads to certain challenges in using these tools. Tools such as BLAST are not designed to work with an expanded base and amino acid repertoire. Therefore, these tools need to be adapted to be used with unnatural base pairs. Other problems that have to be considered are the limited possibilities for sequencing unnatural base pairs, a limited range of compatible polymerases and a restricted set of cloning methods. Sequencing of the unnatural base pairs can be achieved using Oxford Nanopore Technology´s sequencing technology. Since Oxford Nanopore Sequencing does not require additional sequencing chemistry, this technology is predestined to be used for the sequencing of unnatural base pairs. An even simpler solution to this problem is our Mutational Analysis Xplorer (M.A.X) , an enzyme based system to analyze if an unnatural base pair is present in a specified sequence or if it is mutated to a natural base. The limited set of cloning methods results mainly from the low fidelity of most polymerases regarding the incorporation of unnatural base pairs. Therefore, if possible, PCR free cloning methods should be used.
