<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/9/9c/T--Bielefeld-CeBiTec--title-img-modeling.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Modeling
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="borderbox"><DIV class="content"><H3> Organization of our modeling projects </H3><DIV class="article">
				On this page, we describe our main modeling project, which was integral for our whole project. 

				However, besides this complex modeling, we also conducted and applied several straight-forward stochastic and statistical models 

				to support and guide numerous steps of our laboratory work. Some of these modeling projects are briefly described in this box; however, 
				we recommend reading the linked pages for further information. </DIV><H4>Discriminant function model for the ICG prediction: </H4><DIV class="article">
			We conducted a discriminant function analysis for the classification of nucleotides in Oxford Nanopore sequencing reads
			at a specific position. This model is part of our <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software#iCG">iCG</A> software module and enabled the successful detection of unnatural bases.
		</DIV><H4>Calculation of the required library size for the selection system: </H4><DIV class="article">
			We applied combinatorics and statistics to calculate the optimal library size for the <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">tRNA synthetase selection process.</A> 
			This was a trade off between putting lots of efforts into constructing a very huge library and missing diversity in a too small library. 
			Therefore, we predicted the optimal library size. Experimental validation of this prediction was done via MiSeq analysis of the diversity 
			of a subset of this library. 
		</DIV><H4>Strength prediction for a transcription signal amplification system <A target="_blank" href="http://parts.igem.org/Part:BBa_K2201373">(BBa_K2201373)</A>: </H4><DIV class="article">
			We modeled and visually compared the mRFP production over time for a normal mRFP reporter system and compared it to our <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">enhanced signaling.</A>.
			Validity was done as part of the positive selection process for an adapted tRNA synthetase. 
		</DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Short Summary </H3><DIV class="article">
			As our project explores possibilities of an expanded genetic code via unnatural bases and non-canonical amino acids, 
			we set out to complement and improve our lab work via modeling of novel amino acyl tRNA synthetases (aaRS) for a non-canonical amino 
			acids, which were synthetized in our lab. In order to incorporate non-canonical amino acids into proteins via the translational 
			process, the aaRS has to attach the amino acid to the respective tRNA. Thus, we designed aaRS sequences which were adjusted 
			to link our own non-canonical amino acid to a fitting tRNA. Candidates were evaluated and selected, based on a ROSETTA score.
			Most promising sequences were ordered via gene synthesis for the experimental validation. 
			Figure 1 provides a rough overview of our modeling project. Table 1 below summarizes the realization in practice.
		</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 1: Modeling Project Overview</B> A stylized overview of our modeling project,
			containing both <I>in silico</I> and <I>in vivo</I> components.</P></DIV><P class="table-headline"><B>Table 1: Steps of our modeling project</B> Our modeling project consists of seven main steps, combining <I>in silico</I> and <I>in vivo</I> components.</P><TABLE style="margin-bottom: 0px;"><THEAD><TR><TH style="width: auto" class="header">Step</TH><TH style="width: auto" class="header">Software/Method</TH><TH style="width: auto" class="header">Meaning</TH></TR></THEAD><TBODY><TR><TD>1. Ligand preparation</TD><TD>Manually via <A href="https://avogadro.cc/">Avogadro</A></TD><TD>Due to the novelty of our amino acid, no information on the ligand is available in databases. 
						Therefore, manual generation of a conformer ensemble, containing 
						for example all energetically useful arrangements of atoms within the molecule, was required.</TD></TR><TR><TD>2. Scaffold categorization</TD><TD>ROSETTA protocol</TD><TD>The scaffold describes the rough layout of the synthetase. We downloaded the scaffold 
						<A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1j1u">1J1U </A>, the aaRS of <I> Methalonococcus janischii </I> 
						as a template, and then relaxed its structure to improve the outcome of the ROSETTA algorithm.</TD></TR><TR><TD>3. Set simulation constrains</TD><TD>Manually via ROSETTA</TD><TD>Constrains with regards to possible mutations of the synthetase ensure that the generated sequences fit to the amino acid. 
						For example, we constrained the distance between certain atoms and their angle to a range optimal for hydrogen bonds.</TD></TR><TR><TD>4. <A href="https://www.rosettacommons.org/manuals/archive/rosetta3.5_user_guide/d7/dfc/match.html">Enzyme matching</A></TD><TD>ROSETTA protocol</TD><TD>ROSETTA combines information about the ligand and constrains to find possible hydrogen bonding partners and propose the shape 
						of the scaffold within the set constraints.</TD></TR><TR><TD>5. <A href="https://www.rosettacommons.org/manuals/archive/rosetta3.5_user_guide/d6/dbc/enzyme_design.html">Enzyme design</A></TD><TD>ROSETTA protocol</TD><TD>An algorithm uses the information from the previous step and information on the ligand to simulate the mutation process and 
						generate sequences for optimized scaffolds with corresponding scores as measures of fit.</TD></TR><TR><TD>6. Evaluate results <I>in silico</I></TD><TD>Manually</TD><TD>Based on the score values, we ordered the synthesis of the most promising sequences. </TD></TR><TR><TD>7. Evaluate results <I>in vivo</I></TD><TD>Manually</TD><TD>The synthetases are validated in the lab with the corresponding ncAA via a 
						<A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">positive-negative selection system.</A></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>Introduction</H3><H4>Overview</H4><DIV class="contentline"><DIV class="third"><DIV class="figure large"><P class="figure subtitle"><B>Figure 2: Tyrosyl-tRNA-synthetase </B> 3D-structure based on &quot;1J1U&quot; from PDB edited with pymol</P></DIV></DIV><DIV class="third double"><DIV class="article">
						As part of our iGEM project, we faced the challenge of adapting the tRNA synthetase 
				to non-canonical amino acids. For this purpose, we modeled possible candidates for synthetases as an 
				alternative to carrying out a positive-negative selection according to (Liu <I>et al.</I>, 2007) in the laboratory.

				Due to the rapid development in the field of protein and molecular structure analysis, there has been an 
				increase in the availability of molecular 3D structure data. These data are organized in publicly available 
				databases which provide a foundation for the modeling and simulation of chemical-biological processes in bioinformatics. 
				As our ncAA has been synthetized in our lab, no such comprehensive information is available, yet. 
				However, information of similarly structured amino acids can potentially serve as a basis for our modeling.

				Specifically, we focused on simulation to 
				design an aaRS for the new ncAA <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing#CBT">CBT-Asparagine</A>.
				</DIV></DIV></DIV><H4>Method</H4><DIV class="article">
						We used the open-source software <A href="https://www.rosettacommons.org/docs/latest/getting_started/Getting-Started">&quot;ROSETTA&quot;</A> 
						for the main part of our modeling project, which was introduced at the University of Washington by David Baker in 1997 
						(Simon <I>et al.</I>,1997), initially in the context of protein structure prediction. ROSETTA has grown through the addition of numerous modules and is currently widely used in research. In our application, we focus on the Rosetta module called the &quot;Rosetta Enzyme Design Protocol&quot;
					 </DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3>ROSETTA Enzyme Design</H3><H4>Overview </H4><DIV class="article">
			Since the non-canonical amino acid synthesized in the laboratory is completely novel, there is no corresponding tRNA synthetase which can load the tRNA, yet. For this reason, we use the enzyme design protocol to design the binding pocket in a way that allows it to form an effective and specific enzyme. The protocol consists of two main steps: matching and designing (Richter <I>et al.</I>, 2011)
					The enzyme design algorithm is briefly summarized in Figure 3.
		</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 3: Flowchart Enzym Design Protocol.</B></P></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H4> Matching Step</H4><DIV class="contentline"><DIV class="third"><DIV class="figure large"><P class="figure subtitle"><B>Figure 4: overview of constraints</B> all possible constraints, which can be set, with dashed lines indicating hydrogenbonds, normal lines indicating covalent bonds</P></DIV></DIV><DIV class="third double"><DIV class="article">
					The meaning of the matching step is to match the amino acids which constrains to the ligand, following specific constrains which ensure that the result is sensible and feasible. For this, ROSETTA analyzes the structural formula of the non- canonical amino acid and offers the possible hydrogen binding partners.
				</DIV><H4>Matching step inputs</H4><DIV class="article">
					For the matching step, the following input-files are needed: 
					<UL><LI>a “.params”-file specifying information about the ligand</LI><LI>a “.pdb”-file providing a rough scaffold layout</LI><LI>a “.cst”-file to define the bindings between ligand and scaffold</LI><LI>a “.pos”-file to define the positions of the amino acids in the scaffold </LI><LI>a “.flags” file to control all inputs. These files are necessary, as they describe the ligand and backbone and specify the parameters of the algorithm </LI></UL> 
					To read about each file in further detail, please click the technical detail button below: <A class="hidden-expand">SHOW TECHNICAL DETAILS</A></DIV><ARTICLE class="hidden-block"><UL><LI>“.params”-file: 
				A conformer ensemble has to be generated using information about the ligand, as the non-canonical amino acids are not generally available in databases like <A href="" http:="">PDB</A>, making it necessary to build them manually 
				using tools like <A href="https://pymol.org/2/">pymol</A>, <A href="https://avogadro.cc/">Avogadro</A> or <A href="http://www.cambridgesoft.com/software/overview.aspx">Chemdraw</A>. Using these tools,
				 files can be saved in the desired format. The ligand needs to be specified in the “.sdf”, “.mol” or “.mol2” file format. Such a 
				file can be obtained automatically by converting the relevant information from a “.pdb” file, if available. This conversion process usually also involves augmenting the data with hydrogen atoms 
				in case they are missing from the “.pdb” file. Alternatively, the ligand can be designed using &quot;Simplified Molecular Input Line Entry Specification&quot; <A href="http://daylight.com/smiles/">(SMILES)</A> or manually using tools such as <A href="https://avogadro.cc/">Avogadro</A>, as we did. In the next step, the ligand file is used
				 to create a conformer ensemble that is in turn used to create a Rosetta parameter (“.params”) file. In addition to the specific names of all atoms present in the ligand, this parameter file also
				  stores all bonds between the individual atoms, including the binding angles and binding distances. Rosetta cannot generate the conformer ensemble by itself, so an additional tool is needed. 
				  Different tools are capable of creating the conformer ensemble automatically, but it is best to manually define constraints for the chi1, chi2 and backbone psi torsion angles that define the 
				  orientation of the ligand in the binding pocket. For this, we know of three tools: The first is <A href="https://www.eyesopen.com/omega">OpenEye Omega</A>, but the full license is very costly and the academic free version is hard to obtain. 
				  The second tool is <A href="http://accelrys.com/">Accelrys Discovery Studio</A>, but Accerlys does not provide a free license. The third tool is <A href="https://dasher.wustl.edu/tinker/">TINKER</A>, which is free, but poorly documented and depends on a specific keyfile, 
				  which requires a high amount of chemical expertise to generate. Conformers might also be generated without constrains, for which different tools are available, in our case, we used <A href="http://www.conflex.net/">ConFlex</A>. 
				  Conformers need to be stored in one file (“.sdf”, “.mol”, or “.mol2”).
				</LI><LI> “.pdb”-file: 
				The input-file for the scaffold, in our case the tRNA synthetase, can be downloaded in PDB format from Protein Data Bank (PDB). It is then necessary to delete the natural ligand from the PDB-file, 
				as we need to incorporate our own aaRS. Additionally, it is advised to relax the  preferably, the structure should be relaxed in order to allow for flexibility with regards to the simulation outcomes. 
				For further details, see the <A href="https://www.rosettacommons.org/docs/latest/application_documentation/structure_prediction/relax.">ROSETTA Relaxing documentation.</A></LI><LI><A href="https://www.rosettacommons.org/manuals/archive/rosetta3.5_user_guide/d5/dd4/match_cstfile_format.html">“.cst”-file:</A>
				The .cst-file defines the potential hydrogen bonds between the ligand and the amino acid. For example, the code block  characterized by the tags “CST::BEGIN” and “CST::END”, specifies the orientation or 
				catalytic function of the enzyme. 
				More specifically, the first record of the block begins with “TEMPLATE::ATOM_MAP”, followed by either “atom_name” or “atom_type”, depending on whether a specific residue or a specific type of residue 
				is provided. In the latter case, it is not important to choose specific atoms. Instead, a catalytic residue of the amino acid such as “OH” or “Nhis” is specified. The next lines of the TEMPLATE::ATOM_MAP 
				record define the residues using one-letter or three-letter-codes that are prefixed by “residue1” or “residue3”, respectively. 
				The second record, beginning with the tag “CONSTRAINT”, contains all relevant distance, angle and torsion constraints for the matching. Each constraint is described with five parameters. 
				In the case of the distance constraint, the first parameter describes the optimal distance “x0” between the chosen residues, the second parameter describes the tolerance “xtol”, 
				the third parameter defines the strength “k” and the fourth parameter specifies the type of bond (1 for a covalent bond, 0 otherwise). If the modulus of the difference between the actual distance “x”
				 and the specified optimal distance is smaller than the tolerance, then the penality score is zero. Otherwise, the constraint consists of the term <P>k*(|x-x<SUB>0</SUB>|-x<SUB>tol</SUB>)</P>
				to the penality score. For the angle and torsion constraints, the description is similar. 
				If necessary, additional hydrogen bonds to other atoms of the ligand are specified in terms of additional blocks, using the tag “VARIABLE::CST”. 
				Finally, most of the blocks described above can be optionally followed by an “ALGORITHM_INFO” record that stores details of the matching algorithm by parameter values. 
				We refer to the Rosetta documentation for further details.
				</LI><LI>”.pos”-file: 
				The “.pos” file contains the allowed locations in the scaffold for the chosen catalytic residues in each constraint block of the “.cst” file.
				</LI></UL></ARTICLE></DIV></DIV><H4>Matching step outputs</H4><DIV class="contentline"><DIV class="third"><DIV class="figure large"><P class="figure subtitle"><B>Figure 5: example of an output-pdb-file from the matching step</B> CBT-Asparagine in purple, amino acid in green, created in pymol</P></DIV></DIV><DIV class="third double"><DIV class="article">
						The output generated in the matching step is the layout of the scaffold as well as one or more states of the amino acid which enable interaction with the ligand. This information is stored as a “.pdb” file and becomes part of the input for the design step. </DIV><H4>Our results for this step </H4><DIV class="article">
							We used the <A href="http://www.rcsb.org/pdb/explore.do?structureId=1j1u">“1J1U”</A>-scaffold from PDB for our matching step. 
							The “1J1U.pdb”-file contains the tyrosyl-tRNA-synthetase, which is labeld under “Chain A”, the orthogonol tRNA under “Chain B” 
							and the natural ligand tyrosyl. For our project, we deleted the natural ligand and “Chain B”, because it was not necessary to 
							change their structure or sequence and it was a way to save compute time.
							We designed the ligands manually via <A href="https://avogadro.cc/">Avogadro</A> based on a default matching algorithm for both
							amino acids, thus creating useful .cst-files. 
					</DIV></DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H4> Design Step</H4><DIV class="article">
							The design step applies an algorithm such that the binding pocket and the near environment are mutated and the remaining scaffold is
							 repacked. Additionally, a badness-of-fit score is generated which indicates how well the mutation fits the amino acid. For every file from the 
							 matching step, a model with a score and a “.pdb-file” was generated, specifying where the sequence can be located. Additionally, the &quot;.pdb-file&quot;
							 makes visual analysis of the 3D-structure possible. Notably, the amino acid structure can be extracted separately.
							The following section describes the structure of the design step. Further details on each step can be obtaind by showing the Technical Details Section. 
							1. Optimizing the catalytic interactions 
							For the first alternative, the file can be generated either by the Rosetta standard or a manually created .”res”- file. For more details, we refer to the 
							<A href="https://www.rosettacommons.org/manuals/archive/rosetta3.5_user_guide/d1/d97/resfiles.html">Rosetta documentation</A>.  
							For the latter alternative, residues are automatically categorized by their location of the Cα;lpha;.
							<A class="hidden-expand">SHOW TECHNICAL DETAILS</A></DIV><ARTICLE class="hidden-block">
							Residues are catagorized as follows: <UL><LI> residues that have their Cα within a distance cut1 angstroms of any ligand heavyatom will be set to designable 
							  </LI><LI> res that have Cα within a distance cut2 of any ligand heavyatom and the Cβ closer to that ligand atom than the Calpha will be set to designable. cut2 has to be larger than cut1
							  </LI><LI> res that have Cα within a certain distance cut3 of any ligand heavyatom will be set to repackable. cut3 has to be larger than cut2
							  </LI><LI>res that have Cα within a distance cut4 of any ligand heavy atom and the Cβ closer to that ligand atom will be set to repackable. cut4 has to be larger than cut3
							  </LI><LI> all residues not in any of the above 4 groups are kept static.
							</LI></UL></ARTICLE><ARTICLE>
							2. Cycles of sequence design and minimazation within constrains 
							To optimize the structure we applied an iterative optimization algorithm. This algorithm mutates all residues from the backbone, which are not part 
							of the catalytic center, to alanine, and a small energy function refraction will place the ligand in an optimal position to the backbone. <A class="hidden-expand">SHOW TECHNICAL DETAILS</A></ARTICLE><ARTICLE class="hidden-block">
							For this approach, bb_min and chi_min allow for backbone flexibility and the rotation of the torsions. An alternative for this minimization step is the
							 Monte Carlo rigid body ligand sampling. For further information on this method, we refer to the <A href="https://www.rosettacommons.org/manuals/archive/rosetta3.5_user_guide/d6/dbc/enzyme_design.html">ROSETTA documentation</A>.
							 </ARTICLE><H4>Design step inputs </H4>
							The following input files are relevant for the design procedure: 
							 <UL><LI>“.pdb”-file generated in the matching step</LI><LI>“.cst”-file for the ligand</LI><LI>“.params”-file for the ligand and the scaffold </LI><LI>“.flags” to coordinate the inputs</LI></UL> 
							For further information on these files, please refer to step 2 above. <DIV class="contentline"><DIV class="third"><DIV class="figure large"><P class="figure subtitle"><B>Figure 6: example of CBT-Asparagine in the binding pocket </B> CBT-asparagine in purple, scaffold in green, created in pymol</P></DIV></DIV><DIV class="third double"><H4>Design step outputs</H4><ARTICLE>
										The output for the design step is a “.pdb”-file containing the mutated scaffold and a “.score”-file.
										For every .pdb-file, a line in the score-file is generated, so it is easy to evaluate the given structure.
										The first score in the file is the total score of the model. After that, the number of hydrogen bonds 
										in the protein as a whole and in the constraints is listed, followed by the number of dismissed solutions in the 
										catalytic residues as well in the whole protein and in the constraints. 
										See the technical details below for a full overview of the output information <A class="hidden-expand">SHOW TECHNICAL DETAILS</A></ARTICLE><ARTICLE class="hidden-block"><UL><LI>total_score: energy (excluding the constraint energy)
								</LI><LI>fa_rep: full atom repulsive energy
								</LI><LI>hbond_sc: hbond sidechain energy
								</LI><LI>all_cst: all constraint energy
								</LI><LI>tot_pstat_pm: pack statistics, 0-1, 1 = fully packed
								</LI><LI>total_nlpstat_pm: pack statistics withouth the ligand present
								</LI><LI>tot_burunsat_pm: buried unsatisfied polar residues, higher = more buried unsat polars (just a count) 
								</LI><LI>tot_hbond_pm: total number of hbonds 
								</LI><LI>tot_NLconst_pm: total number of non-local contacts ( two residues form a nonlocal
								contact if they are farther than 8 residues apart in sequence but interact with a Rosetta score of lower than -1.0 )
								</LI></UL></ARTICLE></DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Results </H3><H4> Results <I>in silico</I></H4><DIV class="article"><P>We choose our synthetases based on a good total score and a good ligand score. We checked the corresponding PDB-files, and rated the ligand and the binding pocket
							 as satisfying, so that the ligand assumedly does not collide with residues in the near environment.
							The total scores for CBT are not as good as the scores for NPA. However, the ligand scores are acceptable in both cases. A visual evaluation confirms that the ligand
							 fits into the binding pocket. </P><P><B>Our results for this step </B></P><P>We used this algorithm to simulate the evolution of the tyrosyl-tRNA with the amino acids Nitrophenylalanine(NPA) and N<SUP>γ</SUP>‑2‑cyanobenzothiazol‑6‑yl‑L‑asparagine (CBT-asparagine). </P><P><B>NPA simulation: </B></P><P>The .cst-file contained two blocks for the nitrogroup of NPA. Since there are two oxygen-atoms in the nitrogroup,
							 we defined two atom nametags. As several possibilities are useful, we defined two possible constraint partners
							  for the hydrogen bonds. The first is asparagine (N) or glutamine (Q) and the second is glycine (G). We set the
							   possible distance to 2.8 Å, as it is the optimal distance for hydrogenbonds, and a tolerance level of 0.5 Å. 
							   We set the angles to 120° with a tolerance of 40°, as recommended by Florian Richter during our discussion in cologne. 
							   The torsion angles were set to 180° with a tolerance of 180° and a penalty of 0, such that the torsion angles can rotate 
							   completely freely.(Richter, unpublished data) </P><P><B>CBT-Asparagine simulation:</B></P><ARTICLE><P>CBT-Asparagine can build hydrogen bonds in two ways. The first is a weak hydrogen bond on the 
							sulfur atom and the other possibility is a normal hydrogen bond on the nitrogen (N<SUB>2</SUB>) 
							after the Cγ. We wrote three cst-files, one for a possible bond with sulfur, one for a
							 possible bond with nitrogen, and one for both bonds. As possible corresponding amino acids, we chose serine, 
							 threonine, tyrosine, asparagine, glutamine, and glycine. </P><A class="hidden-expand">SHOW TECHNICAL DETAILS</A></ARTICLE><ARTICLE class="hidden-block">
							It is recommended to write a “.flags”-file, because there are several input parameters to be defined, 
							but it is also possible to define them via console user interface. 
							For the categorization of the scaffold, we chose the automatic determination and set the following cuts, like the Baker-lab commonly used: cut1: 6 Å, cut2: 8 Å, cut3: 10 Å and cut4: 12 &amp;#8491 </ARTICLE>
				
				We used this algrithm to simulate the evolution of the tyrosyl-tRNA with the amino acids Nitrophenylalanine and CBT-ASP
				We obtained 13 synthetase sequences for CBT-ASP, and 43 sequences for NPA, which fit well into the binding site according to the ROSETTA score.
				The sequences for the best synthetases for NPA is available
				<A target="_blank" href="https://static.igem.org/mediawiki/2017/1/12/T--Bielefeld-CeBiTec--DKE_NPAseq.pdf">here</A>.
			</DIV><P class="table-headline"><B>Table 2: ROSETTA Enzyme Design Protocol Results</B>ROSETTA scores of the best modeled synthetases for CBT-Asparagine and NPA.</P><TABLE><THEAD><TR><TH style="width: auto" class="header">Sequence Number</TH><TH style="width: auto" class="header">Total Score</TH><TH style="width: auto" class="header">Ligand Score</TH><TH style="width: auto" class="header">ncAA</TH></TR></THEAD><TBODY><TR><TD>15</TD><TD>124.88</TD><TD>-3.77</TD><TD>NPA</TD></TR><TR><TD>19</TD><TD>23.55</TD><TD>-3.93</TD><TD>NPA</TD></TR><TR><TD>31</TD><TD>-3.40</TD><TD>-2.47</TD><TD>NPA</TD></TR><TR><TD>32</TD><TD>-1.57</TD><TD>-3.82 </TD><TD>NPA</TD></TR><TR><TD>40</TD><TD>11.67</TD><TD>-4.33</TD><TD>NPA</TD></TR><TR><TD>41</TD><TD>11.55</TD><TD>-2.98</TD><TD>NPA</TD></TR><TR><TD>43</TD><TD>66.36</TD><TD>-5.05</TD><TD>NPA</TD></TR><TR><TD>2</TD><TD>38.01</TD><TD>-6.56</TD><TD>CBT-Asparagine</TD></TR><TR><TD>4</TD><TD>58.45</TD><TD>-4.37</TD><TD>CBT-Asparagine</TD></TR><TR><TD>5</TD><TD>109.13</TD><TD>-4.25</TD><TD>CBT-Asparagine</TD></TR></TBODY></TABLE><P class="table-headline"><B>Table 2: ROSETTA Enzyme Design Protocol Results</B>ROSETTA scores of the best modeled synthetases for CBT-Asparagine and NPA.</P><TABLE><THEAD><TR><TH style="width: auto" class="header">Position</TH><TH style="width: auto" class="header">Synthetase Number</TH><TH style="width: auto" class="header">Original Amino Acid</TH><TH style="width: auto" class="header">Mutation Amino Acid</TH></TR></THEAD><TBODY><TR><TD>30</TD><TD>5</TD><TD>Serine</TD><TD>Asparagine</TD></TR><TR><TD>32</TD><TD>5</TD><TD>Tyrosine</TD><TD>Threonine</TD></TR><TR><TD>34</TD><TD>2, 4</TD><TD>Glycine</TD><TD>Alanine</TD></TR><TR><TD>36</TD><TD>2</TD><TD>Glutamine acid</TD><TD>Isoleucine</TD></TR><TR><TD>61</TD><TD>5</TD><TD>Asparagine acid</TD><TD>Arginine</TD></TR><TR><TD>63</TD><TD>5</TD><TD>Isoleucine</TD><TD>Alanine</TD></TR><TR><TD>65</TD><TD>4</TD><TD>Leucine</TD><TD>Glycine</TD></TR><TR><TD>65</TD><TD>5</TD><TD>Leucine</TD><TD>Threonine</TD></TR><TR><TD>68</TD><TD>4</TD><TD>Asparagine acid</TD><TD>Alanine</TD></TR><TR><TD>69</TD><TD>4</TD><TD>Leucine</TD><TD>Alanine</TD></TR><TR><TD>70</TD><TD>2</TD><TD>Histidine</TD><TD>Asparagine acid</TD></TR><TR><TD>70</TD><TD>4</TD><TD>Histidine</TD><TD>Glycine</TD></TR><TR><TD>72</TD><TD>4</TD><TD>Tyrosine</TD><TD>Glutamine acid</TD></TR><TR><TD>73</TD><TD>2</TD><TD>Leucine</TD><TD>Alanine</TD></TR><TR><TD>73</TD><TD>4</TD><TD>Leucine</TD><TD>Methionine</TD></TR><TR><TD>74</TD><TD>2</TD><TD>Asparagine</TD><TD>Asparagine acid</TD></TR><TR><TD>76</TD><TD>2</TD><TD>Lysine</TD><TD>Serine</TD></TR><TR><TD>79</TD><TD>4</TD><TD>Leucine</TD><TD>Arginine</TD></TR><TR><TD>101</TD><TD>5</TD><TD>Lysine</TD><TD>Glutamine acid</TD></TR><TR><TD>103</TD><TD>5</TD><TD>Valine</TD><TD>Triptophane</TD></TR><TR><TD>104</TD><TD>5</TD><TD>Tyrosine</TD><TD>Valine</TD></TR><TR><TD>105</TD><TD>4, 5</TD><TD>Glycine</TD><TD>Serine</TD></TR><TR><TD>107</TD><TD>5</TD><TD>Glutamine acid</TD><TD>Lysine</TD></TR><TR><TD>108</TD><TD>4</TD><TD>Phenylalanine</TD><TD>Lysine</TD></TR><TR><TD>108</TD><TD>5</TD><TD>Phenylalanine</TD><TD>Arginine</TD></TR><TR><TD>109</TD><TD>4, 5</TD><TD>Glutamine</TD><TD>Alanine</TD></TR><TR><TD>114</TD><TD>4</TD><TD>Tyrosine</TD><TD>Alanine</TD></TR><TR><TD>115</TD><TD>4</TD><TD>Threonine</TD><TD>Triptophane</TD></TR><TR><TD>118</TD><TD>4</TD><TD>Valine</TD><TD>Serine</TD></TR><TR><TD>134</TD><TD>2</TD><TD>Methionine</TD><TD>Asparagine</TD></TR><TR><TD>137</TD><TD>2</TD><TD>Isoleucine</TD><TD>Alanine</TD></TR><TR><TD>139</TD><TD>2</TD><TD>Arginine</TD><TD>Serine</TD></TR><TR><TD>147</TD><TD>2, 4</TD><TD>Alanine</TD><TD>Serine</TD></TR><TR><TD>148</TD><TD>2</TD><TD>Glutamine</TD><TD>Lysine</TD></TR><TR><TD>149</TD><TD>2</TD><TD>Valine</TD><TD>Threonine</TD></TR><TR><TD>150</TD><TD>2, 4</TD><TD>Isoleucine</TD><TD>Leucine</TD></TR><TR><TD>151</TD><TD>2</TD><TD>Tyrosine</TD><TD>Serine</TD></TR><TR><TD>152</TD><TD>2</TD><TD>Proline</TD><TD>Threonine</TD></TR><TR><TD>153</TD><TD>2</TD><TD>Isoleucine</TD><TD>Leucine</TD></TR><TR><TD>153</TD><TD>4</TD><TD>Isoleucine</TD><TD>Threonine</TD></TR><TR><TD>154</TD><TD>2</TD><TD>Methionine</TD><TD>Asparagine</TD></TR><TR><TD>154</TD><TD>4, 5</TD><TD>Methionine</TD><TD>Serine</TD></TR><TR><TD>155</TD><TD>2</TD><TD>Glutamine</TD><TD>Glycine</TD></TR><TR><TD>155</TD><TD>4, 5</TD><TD>Glutamine</TD><TD>Alanine</TD></TR><TR><TD>156</TD><TD>4</TD><TD>Valine</TD><TD>Alanine</TD></TR><TR><TD>157</TD><TD>4</TD><TD>Asparagine</TD><TD>Alanine</TD></TR><TR><TD>158</TD><TD>4, 5</TD><TD>Asparagine</TD><TD>Tyrosine</TD></TR><TR><TD>159</TD><TD>4</TD><TD>Isoleucine</TD><TD>Glycine</TD></TR><TR><TD>161</TD><TD>4</TD><TD>Tyrosine</TD><TD>Glutamine acid</TD></TR><TR><TD>161</TD><TD>5</TD><TD>Tyrosine</TD><TD>Asparagine acid</TD></TR><TR><TD>162</TD><TD>4</TD><TD>Leucine</TD><TD>Methionine</TD></TR><TR><TD>162</TD><TD>5</TD><TD>Leucine</TD><TD>Alanine</TD></TR><TR><TD>164</TD><TD>5</TD><TD>Valine</TD><TD>Alanine</TD></TR><TR><TD>172</TD><TD>2</TD><TD>Glutamine acid</TD><TD>Lysine</TD></TR><TR><TD>173</TD><TD>2</TD><TD>Glutamine</TD><TD>Serine</TD></TR><TR><TD>176</TD><TD>2</TD><TD>Isoleucine</TD><TD>Serine</TD></TR><TR><TD>204-210</TD><TD>2, 4, 5</TD><TD>varying</TD><TD>-</TD></TR><TR><TD>307</TD><TD>2</TD><TD>-</TD><TD>Alanine</TD></TR><TR><TD>307</TD><TD>4, 5</TD><TD>-</TD><TD>Tyrosine</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H4><I>in vivo</I> validation of predicted tRNA synthetase structures </H4><DIV class="article">
				In order to test the functionality and specificity of our modeled aaRS, we selected the most promising 11 amino acid sequences and ordered seven synthetase sequences of NPA via IDT, and four
				synthetase sequences of CBT-Asparagine by courtesy of Genscript, where we had previously won a grant of 500€. 
				Since the DNA synthesis by GenScript and IDT were delayed by several weeks, we could not perform the best practice characterization of these parts.
				Finally, we received only three of the ordered syntheses in sufficient quality for further experiments. All of them encode predicted CBT-tRNA-synthetases.
				We subjected the sequences to a positive selection as initial characterization. Plasmids encoding the predicted best candidates were cotransformed with our
				positive selection plasmid into <I>E. coli(BL21 DE3)</I>. Due to the IPTG induced promoter, we tested different IPTG concentrations, including 0 mM, 5 mM,
				10 mM, and 15 mM. In addition, we tried different concentration of the antibiotics: kan15, cm15/kan15, and cm15/kan15/tet5. The numbers of resulting colonies is presented in figure 10. Our <I>in vivo</I> results show that our <I>in silico</I> designed enzymes kept their native function and are 
				able to integrate amino acids through an amber codon matching tRNA. Since these results only indicate the acceptance and transfer of the non-canonical
				amino acid, additional experiment are required to demonstrate a high specificity of these enzymes. 
				We offer the predicted sequences to the community for further characterization via the parts-reg (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201300">BBa_K2201300</A>, <A target="_blank" href="http://parts.igem.org/Part:BBa_K2201301">BBa_K2201301</A>, <A target="_blank" href="http://parts.igem.org/Part:BBa_K2201302">BBa_K2201302</A>).
					
			</DIV><DIV class="contentline"><DIV class="third"><DIV class="figure large"><P class="positive selection with synthetase(BBa_K2201300)"><B>Figure 7: positive selection of BBa_K2201300</B> positive selection on kanamycin</P></DIV></DIV><DIV class="third"><DIV class="figure large"><P class="positive selection with synthetase(BBa_K2201301)"><B>Figure 8: positive selection of BBa_K2201301</B>  positive selection on kanamycin</P></DIV></DIV><DIV class="third"><DIV class="figure large"><P class="positive selection with synthetase(BBa_K2201302)"><B>Figure 9: positive selection of BBa_K2201302</B>  positive selection on kanamycin</P></DIV></DIV></DIV><DIV class="figure large"><P class="figure subtitle"><B>Figure 10: bargraph of numbers of colonies </B> different IPTG concentrations</P></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> References </H3><DIV class="article">
				All our files can be obtained <A href="https://2017.igem.org/File:Igemfolder1.zip">here.</A><B>Liu, W., Brock, A., Chen, S., Chen, S., Schultz, P. G. </B>,(2007). Genetic incorporation of unnatural amino acids into proteins in mammalian cells. Nature methods,<B> 4(3)</B>, 239-244.<B>Richter, F., Leaver-Fay, A., Khare, S. D., Bjelic, S., Baker, D. </B>(2011). De novo enzyme design using Rosetta3. PloS one,<B> 6(5)</B>: e19230.<B>Simons, K. T., Kooperberg, C., Huang, E., Baker, D.</B> (1997). Assembly of protein tertiary structures from fragments with similar local sequences using simulated annealing and Bayesian scoring functions. Journal of molecular biology, <B>268(1)</B>, 209-225.</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>