<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_translational_system_translation_mechanism skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/translational system/translation mechanism</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/7/77/T--Bielefeld-CeBiTec--title-img-pipettes.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Translational Mechanism
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Short Summary</H2><DIV class="article">
The incorporation of non-canonical amino acids (ncAAs) is only possible if the translational mechanisms are adapted. One possible way to incorporate the ncAAs is the repurposing of codons like the amber stop or rarely used leucine codons. Another way would be the incorporation of an unnatural base into the DNA to get new codons. For both, an orthogonal tRNA/aminoacyl-synthetase is necessary, which could charge the ncAA to the tRNA. 
</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Adaption of the Translational Mechanism to Incorporate Non-canonical Amino Acids </H2><ARTICLE>
		The genetic code, consisting of the four bases adenine, guanine, thymidine, and cytosine, provides the framework for the synthesis of peptides and proteins, and thus builds the foundation of all living organisms. Three bases of DNA in protein coding sequences determine an amino acid, and multiple amino acids in turn form a polypeptide. Every possible base triplet codes for an amino acid or a translation stop, such that there are no unoccupied triplets. In order to incorporate  a non-canonical amino acid  into a polypeptide, it is necessary that a certain base triplet is dedicated to encoding for this amino acid. As there are no unoccupied triplets available, an existing codon needs to be reassigned. The amber stop codon (UAG/TAG) has been a prominent candidate, as it is one of the three codons signaling translation stop and occurs relatively rarely (roughly 8% of all stop codons in <I>Escherichia coli</I> K-12 genome. An amber suppressor system, consisting of a mutated tRNA and its corresponding tRNA synthetase, is then used to replace the stop-function of the codon with the incorporation of the novel amino acid. More specifically, the tRNA is adapted to suppress the stop-function. In synthetic biology, this is commonly achieved by exchanging a nucleotide at the anticodon part, in a manner that the amber stop codon is read as a codon for an amino acid.  Such systems are common in research (Liu <I>et. al.</I> 2010, Anaelle <I>et. al.</I> 2015, Santoro <I>et. al.</I> 2002). Especially the studies by Liu and Schultz (2010) served as a guideline for our own ncAA incorporation system, as they provide us with comprehensive information on the suppressor system.

By the incorporation of the <A href="!https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs&quot;">unnatural base pairs (UBP)</A>, the translational system enables the recognition of the new bases and the interaction of a matching tRNA/synthetase pair. This is the starting point for the engineering of an orthogonal tRNA/synthetase pair for the translational incorporation of  certain <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">non-canonical amino acids (ncAA)</A>.
In the process of protein biosynthesis, the DNA is first transcribed into mRNA and in the next step into amino acid polypeptides.
Therefore, the ribosome binds on a certain position for the mRNA and supports the binding of the interaction for the mRNA and the matching amino acyl tRNA. 
The ribosome contains three binding sites. The tRNA first binds in the A site and the peptidyl tRNA is bound in the P-site. The polypeptide is then transferred from the peptidyl tRNA in the P-site to the aminoacyl-tRNA in the A-site. In the process of translocation, the ribosome moves on to the next codon on the mRNA and the tRNA is aminoacylated by the tRNA synthetase (aaRS). The tRNA leaves the complex of mRNA and ribosome through the E-site.
Thus, the polypeptide chain is extended in numerous of these processes and the polypeptide is formed. 
This complex mechanism had to be adapted due to the use of UBPs.




		</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> tRNA/Aminoacyl-Synthetase  </H2><ARTICLE>
		Synthetases form a family of enzymes which attach amino acids to the tRNA in a two-step reaction. Therefore, each tRNA synthetase (aaRS) aminoacylates all the tRNAs in an isoaccepting group, representing a particular amino acid. ‘Isoaccepting’ implies that each amino acid can be transported to the ribosome by more than one tRNA.
The tRNA recognition is based on particular nucleotides, labelled as identity set. This identity set is often concentrated in the acceptor stem and anticodon loop region of the tRNA. However, the anticodon sequence of the tRNA is not necessary for a specific tRNA synthetase recognition due to identity determinants which vary in their importance. 


The first step of the two-step reaction is the formation of an aminoacylated intermediate by the reaction of the amino acid and ATP. This is possible through the Rossmann fold, a domain consiting of a six-stranded parallel β-sheet with connecting helices. The Rossmann fold acts analog to the active site of hydrogenases and is responsible for binding ATP, the amino acid, and the 3’-terminus of the tRNA.
		</ARTICLE><ARTICLE>
			The second step is the nucleophilic attack of the carbonyl group of the anhydride from either the 2’-OH or the 3’-OH group on the 3’-terminus of the tRNA. This leads to the delivery of the amino acid to the tRNA.

			The synthetase can either recognize directly a base specific functional group or the enzyme  binds directly to the nonspecific portions of the tRNA. Both readouts function within the context of mutual induced fit, so conformational changes in both interaction partners occur while forming a catalytic complex.
			The recognition requires determinants in the tRNA which are often located in the acceptor stem and the anticodon loop, but the identity signal can be located in the tertiary core region as well (Krebs<I>   et al.</I>, 2014).

			aaRS can be categorized into two classes, based on mutually exclusive sets of sequence motifs and properties of structural domains. The synthetases of these two classes are functionally different. While the class I synthetases aminoacylate tRNAs at the 2’-OH position, the class II synthetases aminoacylate tRNAs at the 3’-OH position. Nevertheless, the position of the initial aminoacylation is related to the binding orientation of the tRNA to the enzyme.
		</ARTICLE><DIV class="contentline"><DIV class="half left">
				The class I synthetases are monomeric synthetases and feature all structurally similar active site Rossmann fold domains in the region of the N-terminus. Beside this region, there are no significant structural or sequence similarities among the class I enzymes.
				There is an acceptor binding site inserted into the Rossmann-fold domain at a common location which binds the single stranded terminal end of the tRNA, while its C-terminal domain binds in the minor groove of the L-shaped tRNA and the anticodon arm. That is the point for the discrimination among the different tRNAs. The binding requires the formation of a hairpin structure of the single stranded 3’-terminus with the amino acid and the ATP in the active site (Chang<I>   et al.</I>, 2010, Krebs<I>   et al.</I>, 2014).
			</DIV><DIV class="half right"><P class="figure subtitle"><B>Figure 1: Crystal structure of a class I tRNA/aminoacyl synthetase. </B> The tRNA is shown in red and the protein in blue (Krebs<I>   et al.</I>, 2014).</P></DIV></DIV><ARTICLE>
			The class II synthetases build a quaternary structure, which in general consists of dimers. They own a structurally conserved active side domain, located towards the C-terminal end of the synthetase. This domain of comprises antiparallel β-sheed folds, flanked by α-helices, which is dissimilar to the Rossmann-fold (see class I synthetase) (Wang<I>   et al.</I>, 2000). In this domain, there are three conserved short sequences. One of these sequences is important for the multimerization and the other two are essential for the enzymatic activity. The class II synthetases bind to the major groove side of the tRNA acceptor stem, whereby unlike the class I synthetases no hairpin structure of the single stranded 3’-terminus into the active site is required (Chang<I>   et al.</I>, 2010, Krebs<I>   et al.</I>, 2014).

		</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> An Orthogonal tRNA/Aminoacyl-Synthetase Pair </H2><ARTICLE>
		The incorporation of non-canonical amino acids (ncAA) requires a tRNA/synthetase (tRNA/aaRS) pair which can accept and bind the ncAA. The tRNA/aaRS pair should be highly specific for the incorporation of the ncAA and most importantly must not cross-react with endogenous amino acids (Ryu <I>   et al.</I>, 2006).
Therefore, orthogonal tRNA/aaRS pairs are applied, which do not cross-react with the endogenous components of the translational system. The cognate tRNA is not recognized by the endogenous synthetase and the cognate synthetase does not incorporate the endogenous amino acids due to differences in tRNA identity elements, especially in the acceptor stem (Kwok<I>   et al.</I>, 1980). At the same time, the orthogonal tRNA/synthetase pair has a good translational function (Wang<I>   et al.</I>, 2001, Chang<I>   et al.</I>, 2010).

Since the tRNA recognition by the aaRS can be domain or species specific (Kwok<I>   et al.</I>, 1980),  a heterologous aaRS/tRNA pair from a different organism is used. A possible source for an orthogonal tRNA/synthetase pair for the application  in bacterial cells are eukaryotes. However, the adaption of these tRNA/aaRS pair which aminoacylates in <I>E. coli</I> is very difficult (Liu<I>   et al.</I>, 1999). Evolved synthetases from archaea can be expressed efficiently in <I>E. coli</I> (Wang<I>   et al.</I>, 2001, Wang<I>   et al.</I>, 2000) and at the same time are more similar to eukaryotic organisms than to the prokaryotic host (Kwok<I>   et al.</I>, 1980, Zhang<I>   et al.</I>, 2005). The reason are significant differences in the acceptor stem and anticodon binding recognition domain between prokaryotic and archaea tRNA/aaRS (Wang<I>   et al.</I>, 2000).

One possible orthogonal tRNA/aaRS pair is the tyrosine tRNA/aaRS pair of <I>Methanococcus jannaschii</I>, where the first base pair of the acceptor stem CG as a tRNA<SUP>Tyr</SUP>identity element differs from those of <I>E. coli</I> tRNA<SUP>Tyr</SUP> containing a GC (Wang<I>   et al.</I>, 2001).
The orthogonal tRNA/aaRS pair is incorporated as a response of the amber stop codon, which does not incorporate any of the 20 canonical amino acids. We use the amber stop codon (UAG), because this codon is rare in <I>E.coli</I> and incorporation of random amino acids has been shown before.

		</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> References</H2><ARTICLE><B>Zhang Y, Wang L, Schultz PG, Wilson IA. </B>(2005). Crystal structures of apo wild-type <I>M. jannaschii tyrosyl-tRNA synthetase (TyrRS)</I> and an engineered TyrRS specific for O-methyl-L-tyrosine. <B>Protein Sci 14(5): 1340–1349</B><B>Wang L, Xie J, Schultz PG. </B> (2006). Expanding the Genetic Code. <B>Annu. Rev. Biophys. Biomol. Struct. 2006;35:225–49. </B><B>Liu CC, Schultz PG.</B> (2010). Adding New Chemistries to the Genetic Code. Annu. Rev. Biochem. 2010;79:413–44.<B>Kwok Y, Wong JT. </B> (1980). Evolutionary relationship between Halobacterium cutirubrum and eukaryotes determined by use of aminoacyl-tRNA synthetases as phylogenetic probes. Canadian Journal of Biochemistry. <B>58(3): 213-218</B><B>Krebs J.</B>(2014) Lewin's GENES XI
<B>Nov Y. </B> 2011. When Second Best Is Good Enough: Another Probabilistic Look at Saturation Mutagenesis,<B> Applied and Environmental Microbiology 0099-2240/12/12.00 </B><B>Anaëlle, Lerche , Spicer, Davis. </B>(2015), Designing logical codon reassignment–expanding the chemistry in biology. <B> Chemical Science 6.1 (2015): 50-69</B><B>Stephen W. Santoro, Wang, Herberich,  King, Schultz. </B> (2002) An efficient system for the evolution of aminoacyl-tRNA synthetase specificity.&quot; <B> Nature biotechnology 20.10 </B></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>