Team:Bielefeld-CeBiTec/Results/Overview
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
During our project we worked in many different scientific fields to find suitable ways to incorporate non-canonical amino acids into proteins. Thus, we needed to repurpose one of the existing codons and incorporated two new bases to create new codons in E. coli and we realized both ways.
The repurposing of a codon for the incorporation of a non-canonical amino acid (ncAA) is possible using the rarely used amber stop codon UAG or other rarely used codons like the leucine codon CUA. To incorporate a non-canonical amino acid using these codons, an orthogonal tRNA/aminoacyl-tRNA synthetase ( tRNA/aaRS ) pair is necessary, which can charge the ncAA to the tRNA. We designed and synthetized the novel ncAA Nγ‑2‑cyanobenzothiazol‑6‑yl‑L‑asparagine ( CBT-asparagine ). This ncAA has the chemical ability of perform a highly specific covalent binding reaction, which we wanted to incorporate it into our target protein. Therefore, we created a library of aaRS with random mutagenized amino acid binding sites and a selection system to select for the aaRS that speciffally incorporates the ncAA. Next to the selection based approach, we modeled the aaRS which could incorporate our new amino acid CBT-asparagine. We showed that both ways are suitable for the evolution of aaRS.
Although incorporation of ncAAs through the amber codon works, there are a lot of problems with this approach. The repurposing of codons leads to the decrease of the growth of E. coli and it is only possible to incorporate two ncAAs. We decided to use a new way to incorporate ncAAs, the incorporation of an unnatural base pair into the DNA that encodes for 64 new codons. Our first challenge was the uptake of the unnatural base from the media, because E.coli has no nucleoside triphosphate transporter and is not able to synthetize the bases itself. We cloned a nucleoside triphosphate transporter that enables the uptake of both bases from the media.  Furthermore, by transcriptome sequencing of the plant Croton tiglium that produces isoG (also called crotonoside) as a defense mechanism, we found a suitable enzyme for the biosynthesis of isoG in E. coli. To detect the unnatural base we developed two orthogonal systems. A restriction experiment based on the software tool M.A.X. and an adaption of the Oxford Nanopore sequencing, which make up one software suite .
To demonstrate the possibilities offered by the incorporation of ncAAs, we developed a toolbox containing five different tools. We chose seven different ncAAs for these five tools and demonstrated interesting applications for them. These ncAAs can be used for various approaches in basic research, medicine and manufacturing. Furthermore, with our submitted parts , every iGEM team can incorporate these ncAAs into their target proteins.
