<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_toolbox_fusing skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/toolbox/fusing</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/c/c8/T--Bielefeld-CeBiTec--title-img-fusing.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Fusing
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Short summary </H3><DIV class="article">
		Protein fusion is usually limited by the C‑ or N‑terminus of a protein. The incorporation of non-canonical amino acids that could be fused to each other or to surfaces enables several additional 
		applications. This tool facilitates immobilization of proteins and improved stability of protein polymer networks. Furthermore, non-canonical amino acids could lead to enhanced efficiency of pathways by combining enzymes 
		of one pathway or for any other system where colocalization is beneficial. With the synthesis of <I>N</I><SUP>ε</SUP>-L-cysteinyl-L-lysine and the novel amino acid <I>N</I><SUP>γ</SUP>-cyanobenzothiazolyl-L-asparagine, 
		we provide the first steps for a new way to fuse proteins with each other and to enhance the stability of protein polymer networks. First attempts show that both amino acids are able to bind specificly to each other 
		under physiological conditions. Additionally, the methods of the synthesis of <I>N</I><SUP>γ</SUP>-cyanobenzothiazolyl-L-asparagine can be applied to each system containing one free amino group and one free carboxy group.
		</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Synthesis of <I>N</I><SUP>ε</SUP>-L-cysteinyl-L-lysine </H3><ARTICLE>
		We used a modified version of the method of Nguyen <I>et al</I>. (2011) to produce <I>N</I><SUP>ε</SUP>‑L‑cysteinyl‑L‑lysine. 
		To ensure a specific reaction between the amino group of the side chain of lysine (see Figure 1) and the carboxyl 
		group of cysteine (see Figure 1) in a selective manner, so called protecting groups are introduced. Commonly used 
		protecting groups are tert‑butyloxycarbonyl (Boc), methyl ester and trityl (Trt). They bind reversible to the 
		corresponding functionality and can be easily removed by acids and bases after the coupling reaction happened. 
		The first step of the synthesis is a coupling reaction of <I>N</I>‑Boc‑L‑lysine‑O‑methyl ester and 
		<I>N</I>‑Boc‑L‑cysteine-S-Trt. Due to the protected functional groups, only the unprotected amino group of 
		<I>N</I>‑Boc‑L‑lysine‑O‑methyl ester and the unprotected carboxyl group of the 
		<I>N</I>‑Boc‑L‑cysteine‑S‑Trt can react with each other. The result is 
		<I>N</I>‑Boc‑L‑lysine[<I>N</I><SUP>ε</SUP>‑(<I>N</I>‑Boc‑L‑cysteine‑S‑Trt)]‑6‑methyl ester. 
		After removal of ester protection with lithium hydroxide as well as Boc and Trt with trifluoroacetic acid, <I>N</I><SUP>ε</SUP>-L-cysteinyl-L-lysine 
		trifluoroacetic acid salt is obtained. Figure 1 shows the schematic reaction.	
		</ARTICLE><DIV class="figure large"><P class="figure subtitle"><B>Figure 1: Schematic reaction of the synthesis of <I>N</I><SUP>ε</SUP>‑L‑cysteinyl‑L‑lysine 
			fluoroacetatic acid salt (<I>N</I>guyen <I>et al</I>., 2011).</B> The unprotected carboxyl group of the cysteine (red) and the 
			unprotected amino group of the lysine (green) are highlighted.</P></DIV><ARTICLE>
		We synthesized <I>N</I><SUP>ε</SUP>‑L‑cysteinyl‑L‑lysine in two batches to ensure that the method of 
		Nguyen <I>et al</I>. (2001) is successful. For the first batch, we used dry dimethylformamide (DMF) 
		as solvent for the coupling reaction as described by Nguyen <I>et al</I>. (2011). Due to low yield compared to Nguyen <I>et al</I>. 
		using DMF, we used tetrahydrofuran (THF) for the second batch.

		</ARTICLE><H3>Coupling reaction of <I>N</I>-Boc-L-lysine-O-methyl ester and <I>N</I>-Boc-L-cysteine-S-Trt</H3><ARTICLE>
		Table 1 shows the used quantity of reactants and solvents for both batches.
		</ARTICLE><P class="table-headline"><B>Table 1: List of used reactants and solvents for the coupling.</B></P><TABLE style="margin-bottom: 0px;"><THEAD><TR><TH style="width: 5%" class="header">No.</TH><TH style="width: 35%" colspan="" class="header">Reagent</TH><TH style="width: 15%" colspan="" class="header">MW / g mol<SUP>-1</SUP></TH><TH style="width: 15%" colspan="" class="header">eq.</TH><TH style="width: 15%" colspan="" class="header">weight / g</TH><TH style="width: 15%" colspan="" class="header">n / mmol</TH></TR></THEAD><TBODY><TR><TD align="center">1</TD><TD><I>N</I>-Boc-L-lysine-O-methyl ester acetate salt</TD><TD align="center">320.40</TD><TD align="center">1.00</TD><TD align="center">1.00</TD><TD align="center">3.12</TD></TR><TR><TD align="center">2</TD><TD><I>N</I>-Boc-L-cysteine-S-Trt</TD><TD align="center">463.60</TD><TD align="center">1.50</TD><TD align="center">2.14</TD><TD align="center">4.62</TD></TR><TR><TD align="center">3</TD><TD>DCC</TD><TD align="center">206.33</TD><TD align="center">1.50</TD><TD align="center">0.95</TD><TD align="center">4.62</TD></TR><TR><TD align="center">4</TD><TD>DMAP</TD><TD align="center">122.17</TD><TD align="center">0.16</TD><TD align="center">0.06</TD><TD align="center">0.49</TD></TR></TBODY></TABLE><TABLE style="margin-top: 0px;"><THEAD><TR><TH style="width: 35%" colspan="" class="header">Solvent</TH><TH style="width: 60%" colspan="" class="header">Volume &amp; mL</TH></TR></THEAD><TBODY><TR><TD align="center">5</TD><TD>DMF/THF</TD><TD align="center">10</TD></TR></TBODY></TABLE><DIV class="contentline"><DIV class="third"><DIV class="figure large"><P class="figure subtitle"><B>Figure 2: Result of the TLC analysis after the coupling reaction.</B>A: 
			<I>N</I>‑Boc‑L‑lysine‑O‑methyl ester; B: <I>N</I>‑Boc‑L‑cysteine‑S‑Trt; C: 
			<I>N</I>‑Boc‑L‑lysine‑O‑methyl ester, <I>N</I>‑Boc‑L‑cysteine‑S‑Trt 
			and the reaction mixture after the coupling reaction; D: the reaction mixture after the coupling reaction.</P></DIV></DIV><DIV class="third double"><ARTICLE>
	The thin layer chromatography (TLC) analysis of the reaction mixture shows that after the coupling reaction 
	no <I>N</I>-Boc-L-lysine-O-methyl ester was left (see Figure 2). This indicates that the <I>N</I>-Boc-L-lysine-O-methyl ester 
	completely reacted. The two spots on the top of C and D are the product <I>N</I>-Boc-L-lysine[<I>N</I><SUP>ε</SUP>-(<I>N</I>-Boc-L-cysteine-S-Trt)]-6-methyl ester (lower spot) and a byproduct of the 
	reaction (upper spot).

		</ARTICLE></DIV></DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 3: Proton nuclear magnetic resonance (<SUP>1</SUP>H NMR) spectrum for the purified reaction 
			mixture after the coupling reaction.</B> The hydrogen bonds of the protecting groups are characteristic for the estimated product 
			<I>N</I>‑Boc‑L‑lysine[<I>N</I><SUP>ε</SUP>‑(<I>N</I>‑Boc‑L‑cysteine‑S‑Trt)]‑6-methyl ester and therefore, their signals are highlighted.</P></DIV><ARTICLE>
		The <SUP>1</SUP>H NMR spectrum of the purified reaction mixture of the coupling reaction shows 
		that the hydrogen atoms of all protecting groups are present (see Figure 3). The 
		Tritylphenylmethane (Trt) at 7.2 ppm is part of the <I>N</I>-Boc-L-cysteine-S-Trt and the 
		methyl ester at 3.6 ppm is originating from the <I>N</I>‑Boc‑L‑lysine‑O‑methyl ester. The 
		tert‑Butyloxycarbonyl protecting group is part of both educts. In this reaction, no protecting groups should be split off, so this diagram shows the NMR spectrum for 
		<I>N</I>‑Boc‑L‑lysine[<I>N</I><SUP>ε</SUP>‑(<I>N</I>‑Boc‑L‑cysteine‑S‑Trt)]‑6‑methyl 
		ester.

		</ARTICLE><H3>Removing the methyl ester of the <I>N</I>-Boc-L-lysine[<I>N</I><SUP>ε</SUP>-(<I>N</I>-Boc-L-cysteine-S-Trt)]-6-methyl ester</H3><ARTICLE>
		Table 2 shows the used quantity of reactants and solvents for both batches.
		</ARTICLE><P class="table-headline"><B>Table 2: List of used reactants and solvents for the reaction to remove methyl 
		ester of the first and the second batch. </B></P><TABLE style="margin-bottom: 0px;"><THEAD><TR><TH style="width: 35.5%" class="header">First batch</TH><TH style="width: 35.5%" class="header">Second batch</TH></TR></THEAD></TABLE><TABLE style="margin-bottom: 0px;" style="margin-top: 0px;"><THEAD><TR><TH style="width: 5%" class="header">No.</TH><TH style="width: 24%" colspan="" class="header">Reagent</TH><TH style="width: 10.5%" colspan="" class="header">MW / g mol<SUP>-1</SUP></TH><TH style="width: 5%" colspan="" class="header">eq.</TH><TH style="width: 10.5%" colspan="" class="header">weight / g</TH><TH style="width: 9.5%" colspan="" class="header">n / mmol</TH><TH style="width: 10.5%" colspan="" class="header">MW / g mol<SUP>-1</SUP></TH><TH style="width: 5%" colspan="" class="header">eq.</TH><TH style="width: 10.5%" colspan="" class="header">weight / g</TH><TH style="width: 9.5%" colspan="" class="header">n / mmol</TH></TR></THEAD><TBODY><TR><TD align="center">1</TD><TD><I>N</I>-Boc-L-lysine[<I>N</I><SUP>ε</SUP>-(<I>N</I>-Boc-L-cysteine-S-Trt)]-6-methyl ester</TD><TD align="center">704.17</TD><TD align="center">1.00</TD><TD align="center">0.60</TD><TD align="center">0.85</TD><TD align="center">704.17</TD><TD align="center">1.00</TD><TD align="center">0.97</TD><TD align="center">1.42</TD></TR><TR><TD align="center">2</TD><TD>LiOH · H<SUB>2</SUB>O</TD><TD align="center">42.06</TD><TD align="center">1.50</TD><TD align="center">0.05</TD><TD align="center">1.28</TD><TD align="center">42.06</TD><TD align="center">1.50</TD><TD align="center">0.09</TD><TD align="center">2.13</TD></TR></TBODY></TABLE><TABLE style="margin-top: 0px;"><THEAD><TR><TH style="width: 24%" colspan="" class="header">Solvent</TH><TH style="width: 35.5%" colspan="" class="header">Volume / mL</TH><TH style="width: 35.5%" colspan="" class="header">Volume / mL</TH></TR></THEAD><TBODY><TR><TD align="center">3</TD><TD>THF:H<SUB>2</SUB>O (3:1)</TD><TD align="center">80</TD><TD align="center">100</TD></TR></TBODY></TABLE><DIV class="contentline"><DIV class="third"><DIV class="figure large"><P class="figure subtitle"><B>Figure 4: Result of the TLC analysis after removing the methyl ester.</B>KC2: the 
			reaction mixture after the coupling reaction; KC3: the reaction mixture after removing the methyl ester.</P></DIV></DIV><DIV class="third double"><ARTICLE>
	After removing the methyl ester, the product is more polar than before. The result is that the 
	<I>N</I>‑Boc‑L‑lysine[<I>N</I><SUP>ε</SUP>‑(<I>N</I>‑Boc‑L‑cysteine‑S‑Trt)] is not 
	soluble in the EtOAc:PE solution. The dark spot at the TLC plate for the sample KC3 is the 
	<I>N</I>‑Boc‑L‑lysine[<I>N</I><SUP>ε</SUP>‑(<I>N</I>‑Boc‑L‑cysteine‑S‑Trt)] and the 
	lighter spot is the removed methyl ester (see Figure 4).

		</ARTICLE></DIV></DIV><H3>Removing the tert-Butyloxycarbonyl protecting group (Boc) and Triphenylmethane (Trt) of the 
				<I>N</I>-Boc-L-lysine[<I>N</I><SUP>ε</SUP>-(<I>N</I>-Boc-L-cysteine-S-Trt)]</H3><ARTICLE>
		Table 3 shows the used quantity of reactants and solvents for both batches.
		</ARTICLE><P class="table-headline"><B>Table 3: List of used reactants and solvents for the reaction to remove Boc 
		and Trt of the first and the second batch. </B></P><TABLE style="margin-bottom: 0px;"><THEAD><TR><TH style="width: 35.5%" class="header">First batch</TH><TH style="width: 35.5%" class="header">Second batch</TH></TR></THEAD></TABLE><TABLE style="margin-bottom: 0px;" style="margin-top: 0px;"><THEAD><TR><TH style="width: 5%" class="header">No.</TH><TH style="width: 24%" colspan="" class="header">Reagent</TH><TH style="width: 10.5%" colspan="" class="header">MW / g mol<SUP>-1</SUP></TH><TH style="width: 5%" colspan="" class="header">eq.</TH><TH style="width: 10.5%" colspan="" class="header">weight / g</TH><TH style="width: 9.5%" colspan="" class="header">n / mmol</TH><TH style="width: 10.5%" colspan="" class="header">MW / g mol<SUP>-1</SUP></TH><TH style="width: 5%" colspan="" class="header">eq.</TH><TH style="width: 10.5%" colspan="" class="header">weight / g</TH><TH style="width: 9.5%" colspan="" class="header">n / mmol</TH></TR></THEAD><TBODY><TR><TD align="center">1</TD><TD><I>N</I>-Boc-L-lysine[<I>N</I><SUP>ε</SUP>-(<I>N</I>-Boc-L-cysteine-S-Trt)]</TD><TD align="center">691.67</TD><TD align="center">1.00</TD><TD align="center">0.68</TD><TD align="center">0.98</TD><TD align="center">7691.67</TD><TD align="center">1.00</TD><TD align="center">0.96</TD><TD align="center">1.39</TD></TR><TR><TD align="center">2</TD><TD>Triethylsilane</TD><TD align="center">116.67</TD><TD align="center">4.50</TD><TD align="center">0.52</TD><TD align="center">4.42</TD><TD align="center">116.67</TD><TD align="center">4.50</TD><TD align="center">0.73</TD><TD align="center">6.26</TD></TR></TBODY></TABLE><TABLE style="margin-top: 0px;"><THEAD><TR><TH style="width: 24%" colspan="" class="header">Solvent</TH><TH style="width: 35.5%" colspan="" class="header">Volume / mL</TH><TH style="width: 35.5%" colspan="" class="header">Volume / mL</TH></TR></THEAD><TBODY><TR><TD align="center">3</TD><TD>Trifluoroacetic acid</TD><TD align="center">30</TD><TD align="center">30</TD></TR></TBODY></TABLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 5: <SUP>1</SUP>H NMR spectrum for the purified 
			<I>N</I><SUP>ε</SUP>‑L‑cysteinyl‑L‑lysine 
			trifluoroacetatic acid salt.</B>All peaks of compounds with hydrogen atoms of 
			the <I>N</I><SUP>ε</SUP>‑L‑cysteinyl‑L‑lysine 
			are highlighted because they are characteristic for this molecule.</P></DIV><ARTICLE>
		The <SUP>1</SUP>H NMR spectrum shows that all estimated hydrogen atoms are present and that the synthesis was successful 
		(see Figure 5).
		
		In the first batch, we got 400 mg of <I>N</I><SUP>ε</SUP>‑L‑cysteinyl‑L‑lysine 
		trifluoroacetic acid salt and in the second batch we obtained 500 mg. This corresponds to 0.84 mmol in the first batch 
		and 1.05 mmol in the second batch. This equals to the half of the yield of Nguyen <I><I>et al</I></I>. (2011) with 900 mg and 1.89 mmol.

</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Synthesis of <I>N</I><SUP>γ</SUP>-cyanobenzothiazolyl-L-asparagine </H3><ARTICLE>
		The synthesis of the novel amino acid <I>N</I><SUP>γ</SUP>‑cyanobenzothiazolyl‑L‑asparagine (CBT‑asparagine) was a 
		new challenge for us. Our idea was to couple the amino group of 6‑amino‑2‑cyanobenzothiazole (ACBT) with the 
		carboxy group of the side chain of aspartic acid. Therefore, it is recommendable to use two different protective groups each bound to the 
		N‑ and C‑terminus of the aspartic acid. Figure 6 and 7 show the schematic synthesis of CBT‑asparagine.
		</ARTICLE><DIV class="figure large"><P class="figure subtitle"><B> Figure 6: Schematic coupling reaction of ACBT and Fmoc‑asparagine‑OAllyl ester giving Fmoc‑CBT‑asparagine‑OAllyl ester as product.</B> The amino group of ACBT and the carboxy group of the side chain of aspartic acid get linked under water loss.</P></DIV><DIV class="figure large"><P class="figure subtitle"><B> Figure 7: Schematic deprotection reaction of Fmoc‑CBT‑asparagine‑OAllyl ester. </B> After the coupling reaction, the deprotection reaction removes the protection groups resulting in the free amino acid or a salt of the amino acid depending of the method.</P></DIV><ARTICLE>
		Due to the high price of 6‑amino‑2‑cyanobenzothiazole (ACBT), we synthesized it on 
		ourselves with 2‑chloro‑6‑nitrobenzothiazole (Cl‑NBT) as starting material 
		following the procedure of (Hauser <I>et al</I>., 2016). Figure 8 shows the schematic synthesis of ACBT. 
		</ARTICLE><DIV class="figure large"><P class="figure subtitle"><B> Figure 8: Schematic synthesis of ACBT. The synthesis contains two steps. </B> The first step (i) is the nucleophilic substitution of the chlorine atom with a cyano group and the second step (ii) is the reduction of the nitro group to an amino group.</P></DIV><ARTICLE>
		First, we changed the chlorine atom of the Cl‑NBT for a cyano group by nucleophilic substitution 
		resulting in 6‑nitrobenzothiazole‑2‑carbonitrile (NBT‑CN). The second step of 
		the synthesis of ACBT is the reduction of the nitro group of the NCBT to an amino group using iron and 
		acetic acid. The last step is troublesome. Due to the reactivity of the iron and the acetic acid, the 
		reaction mixtures contain a large amount of impurities. These impurities can disturb the coupling reaction 
		between ACBT and aspartic acid. After several attempts to get an ACBT with sufficient purity for the coupling 
		reaction of the amino group of ACBT and the carboxy group of the side chain of aspartic acid we got following 
		results. 
		</ARTICLE><H3> Nucleophilic Substitution of the Chlorine Atom with a Cyano Group </H3><ARTICLE>
		The applied reactants and solvents for the nucleophilic substitution of the chlorine atom to the cyano 
		group are listed in table 4.
		</ARTICLE><P class="table-headline"><B>Table 4: List of used reactants and solvents for the nucleophilic substitution of the chlorine atom
		to the cyano group. </B></P><TABLE style="margin-bottom: 0px;"><THEAD><TR><TH style="width: 5%" class="header">No.</TH><TH style="width: 35%" colspan="" class="header">Reagent</TH><TH style="width: 15%" colspan="" class="header">MW / g mol<SUP>-1</SUP></TH><TH style="width: 15%" colspan="" class="header">eq.</TH><TH style="width: 15%" colspan="" class="header">weight / g</TH><TH style="width: 15%" colspan="" class="header">n / mmol</TH></TR></THEAD><TBODY><TR><TD align="center">1</TD><TD>2-Chloro-6-nitrobenzothiazole</TD><TD align="center">214.63</TD><TD align="center">1.00</TD><TD align="center">5.00</TD><TD align="center">23.29</TD></TR><TR><TD align="center">2</TD><TD>1,4-diazabicyclo[2.2.2]octane</TD><TD align="center">112.17</TD><TD align="center">0.15</TD><TD align="center">0.37</TD><TD align="center">3.33</TD></TR><TR><TD align="center">3</TD><TD>NaCN</TD><TD align="center">49.01</TD><TD align="center">1.05</TD><TD align="center">1.20</TD><TD align="center">24.48</TD></TR></TBODY></TABLE><TABLE style="margin-top: 0px;"><THEAD><TR><TH style="width: 35%" colspan="" class="header">Solvent</TH><TH style="width: 60%" colspan="" class="header">Volume &amp; mL</TH></TR></THEAD><TBODY><TR><TD align="center">4</TD><TD>MeCN</TD><TD align="center">500</TD></TR></TBODY></TABLE><DIV class="figure medium"><P class="figure subtitle"><B> Figure 9: The <SUP>13</SUP>C NMR spectrum of the product of the nucleophilic substitution.</B>All peaks over 80 ppm correspond 
			to the estimated spectrum of NBT‑CN. The peak at 77 ppm shows the carbon atom of the used solvent - deuterated chlorofom.</P></DIV><ARTICLE>
		The <SUP>13</SUP>C NMR spectrum of the pure product shows that the nucleophilic substitution was successful (see Figure 9). The peak at 77 ppm corresponds to the resonance 
		frequency of deuterated chloroform, the chosen solvent for the <SUP>13</SUP>C NMR spectrum. The resonance frequency of the carbon of the cyano group is 112 ppm. The carbon 
		atom at 141 ppm is part of the thiazole residue and the six carbon atoms at 118 ppm, 123 ppm, 126 ppm, 135 ppm, 147 ppm and 155 ppm are originated 
		from the benzene ring. The signal at 112 pm is crucial here. It shows that the chlorine atom was successfully substituted by a cyano group.
		Additionally, we analyzed the NBT‑CN by proton NMR (see Figure 10).
		</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B> Figure 10: The <SUP>1</SUP>H NMR spectrum of the product of the nucleophilic substitution.</B>All peaks over 8 ppm correspond to the estimated spectrum of NBT‑CN.</P></DIV><ARTICLE>
		The three signals over 8 ppm show the three hydrogen atoms of the aromatic ring of NBT-CN. The other signals originate from excessive water, ethyl acetate and acetonitrile.
		</ARTICLE><H3> Reduction of the nitro group to an amino group</H3><ARTICLE>
		The product of the nucleophilic substitution was used for the reduction of the nitro group to an amino group. Therefore, we used iron and acetic acid to create an excess of protons. 
		After oxidation of the iron by the oxygen of the nitro group followed by the reduction of the nitro group to an amino group, we got 1.5 g of ACBT. The proton NMR shows that the 
		nucleophilic substitution was successful (see Figure 10). 
		</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B> Figure 10: The <SUP>1</SUP>H NMR spectrum of the product of the reduction of the nitro group of NBT-CN.</B>
			Despite of the impurities, all peaks of the hydrogen atoms of ACBT are present.</P></DIV><ARTICLE>
		Compared to the proton NMR of the NBT-CN (see Figure 9), there is now an additional peak at 5.7 ppm which originates from the amino group. Additionally, 
		you can see the hydrogen atoms of several impurities with resonance frequencies under 5.7 ppm like ethyl acetate, water and byproducts of the reduction with iron and acetic acid which 
		caused problems for further experiments.
		</ARTICLE><H3> Coupling Reaction of ACBT and <I>N</I>-Fmoc-aspartic-acid-OAllyl ester</H3><ARTICLE>
		For this reaction, we used the method of Yuan <I>et al</I>. (2016). They used this method to synthesize (D‑Cys‑Lys‑CBT)<SUB>2</SUB> which in turn was used to build
		cyclic D-Luciferin nanoparticles. This method enables the coupling of ACBT with a carboxyl group. We needed several attempts to successfully couple ACBT to the carboxy group of the 
		side chain of aspartic acid. Due to the high reactivity of the previous reaction with iron and acetic acid, the crude product of the reduction of the nitro group of NBT-CN to the 
		amino group of ACBT contains different impurities. In the first attempt, we tried to use <I>N</I>‑Boc‑aspartic acid‑OBzl and the crude product as reagent. 
		The following figure shows the proton NMR of the product (Figure 11).
		</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B> Figure 11: The <SUP>1</SUP>H NMR spectrum of the product of the first attempt of the coupling reaction of ACBT and aspartic acid.</B>
			There are no peaks corresponding to the resonance frequencies of the hydrogen bonds of the ACBT. There are only signals of the hydrogen atoms of the protected aspartic acid.</P></DIV><ARTICLE>
		The peaks at 1.2 ppm, 2.0 ppm and 4.0 ppm are originating from excessive ethyl acetate. All other peaks correspond to the resonance frequencies of the 
		<I>N</I>‑Boc‑aspartic acid‑OBzl. Thus, the first attempt of the coupling reaction failed. In the second attempt we purified the crude product of the reduction reaction 
		by column chromatography and used <I>N</I>‑Fmoc‑aspartic acid‑OAllyl ester. After analyzing the product of the second coupling reaction by liquid chromatography‑mass spectroscopy 
		(LC‑MS), we got following LC-MS signal (Figure 12).
		</ARTICLE><DIV class="figure large"><P class="figure subtitle"><B> Figure 12: Mass spectrum of the product of the second attempt of the coupling reaction of ACBT and aspartic acid.</B>The peak at the position 
			553.15 shows that the coupling reaction was successful.</P></DIV><ARTICLE>
		With the mass-to-charge ratio of 553.15 Da e<SUP>-1</SUP>, we got a positive signal for the anion of 
		<I>N</I>‑Fmoc‑<I>N</I><SUP>γ</SUP>‑cyanobenzothiazolyl‑aspartic acid‑OAllyl ester which has a molecular weight of 552.61 g mol<SUP>‑1</SUP>.
		</ARTICLE><H3> Removing the Fluorenylmethyloxycarbonyl Group (Fmoc) and the Allyl Ester (OAll)</H3><ARTICLE>
		According to Goodman <I>et al</I>., (2004), we chose morpholine as base to remove the allyl ester protection group. Tetrakis(triphenylphosphine)palladium(0) 
		(Pd(PPh<SUB>3</SUB>)<SUB>4</SUB>) serves as catalyst. For the removal of the protecting groups Fmoc and allyl ester, the applied reactants and solvents are listed in table 6.
		</ARTICLE><P class="table-headline"><B>Table 6: List of used reactants and solvents for the deprotection of <I>N</I>‑Fmoc‑<I>N</I><SUP>γ</SUP>‑cyanobenzothiazolyl‑asparagine‑OAll. </B></P><TABLE style="margin-bottom: 0px;"><THEAD><TR><TH style="width: 5%" class="header">No.</TH><TH style="width: 35%" colspan="" class="header">Reagent</TH><TH style="width: 15%" colspan="" class="header">MW / g mol<SUP>-1</SUP></TH><TH style="width: 15%" colspan="" class="header">eq.</TH><TH style="width: 15%" colspan="" class="header">weight / g</TH><TH style="width: 15%" colspan="" class="header">n / mmol</TH></TR></THEAD><TBODY><TR><TD align="center">1</TD><TD>Fmoc-CBT-asparagine-OAll</TD><TD align="center">552.61</TD><TD align="center">1.00</TD><TD align="center">1.40</TD><TD align="center">2.53</TD></TR><TR><TD align="center">2</TD><TD>Tetrakis(triphenylphosphine)palladium(0)</TD><TD align="center">1,155.59</TD><TD align="center">0.10</TD><TD align="center">0.29</TD><TD align="center">0.25</TD></TR><TR><TD align="center">3</TD><TD>Morpholine</TD><TD align="center">87.10</TD><TD align="center">3.00</TD><TD align="center">0.65</TD><TD align="center">7.59</TD></TR></TBODY></TABLE><TABLE style="margin-top: 0px;"><THEAD><TR><TH style="width: 35%" colspan="" class="header">Solvent</TH><TH style="width: 60%" colspan="" class="header">Volume &amp; mL</TH></TR></THEAD><TBODY><TR><TD align="center">4</TD><TD>THF</TD><TD align="center">50</TD></TR></TBODY></TABLE><ARTICLE>
		Figure 13 to 16 show the results of the LC-MS analysis of the deprotected CBT‑asparagine. Thereby, Figure 13 and 14 represent the results of the aqueous layer 
		and Figure 15 and 16 the results of the organic layer after the extraction of the reaction mixture of the deprotection reaction.
		</ARTICLE><DIV class="figure large"><P class="figure subtitle"><B>Figure 13: Mass spectrum of the fraction with an elution time of 2.33 min of the aqueous layer after the extraction of the reaction mixture of the deprotection reaction.</B>The peak for 379.11 Da e<SUP>-1</SUP> corresponds to the morpholine salt of CBT‑asparagine which has a molecular weight of 377.42 g mol<SUP>-1</SUP>.</P></DIV><DIV class="figure large"><P class="figure subtitle"><B>Figure 14: Mass spectrum of the fraction with an elution time of 1.33 min of the aqueous layer after the extraction of the reaction mixture of the deprotection reaction.</B>The peak for 88.08 Da e<SUP>-1</SUP> corresponds to excessive morpholine which has a molecular weight of 87.12 g mol<SUP>-1</SUP>.</P></DIV><DIV class="article">
		Figure 13 shows that the deprotection reaction was successful resulting in the morpholine salt of CBT-asparagin. Due to the excess of morpholine, there is still some morpholine left (see Figure 14).
		</DIV><DIV class="figure large"><P class="figure subtitle"><B>Figure 15: Mass spectrum of the fraction with an elution time of 2.40 min of the organic layer after the extraction of the reaction mixture of the deprotection reaction.</B>The peak for 291.05 Da e<SUP>-1</SUP> corresponds to the free form of CBT‑asparagine which has a molecular weight of 290.30 g mol<SUP>-1</SUP>.</P></DIV><DIV class="figure large"><P class="figure subtitle"><B>Figure 16: Mass spectrum of the fraction with an elution time of 1.37 min of the organic layer after the extraction of the reaction mixture of the deprotection reaction.</B>Compared to Figure 14, there is no peak corresponding to the morpholine.</P></DIV><DIV class="article">
		Figure 15 shows that not only the morpholine salt was obtained, but also the free amino acid. Compared to Figure 14, Figure 16 shows that the excessive morpholine is only present in the aqueous layer. 
		Usually the acidity of morpholine with a p<I>K</I><SUB>a</SUB> of 8.36 is not high enough to remove the protecting group Fmoc. But with the 3x equivalent of morpholine to the Fmoc- and 
		OAll-protected amino acid and using Pd(PPh<SUB>3</SUB>)<SUB>4</SUB>, it is possible to remove even Fmoc. This results in the free variant or the morpholine salt of the amino acid. In this case 
		we got both the free amino acid and the morpholine salt of CBT‑asparagine. 
		</DIV><DIV class="article">
		Additionally, we analyzed both layers by <SUP>1</SUP>H NMR. The spectra are shown in Figure 17 and 18.
		</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 17: <SUP>1</SUP>H NMR spectrum of the aqueous layer after the extraction of the reaction mixture of the deprotection reaction.</B>Despite of the impurities, the estimated spectrum of the unprotected CBT-asparagine is shown.</P></DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 18: <SUP>1</SUP>H NMR spectrum of the organic layer after the extraction of the reaction mixture of the deprotection reaction.</B>Despite of the impurities, the estimated spectrum of the unprotected CBT-asparagine is shown.</P></DIV><DIV class="article">
		The NMR spectra are similar to each other. Both contain the estimated characteristics. Between 2 and 3 ppm are signals of the α carbon atom of the amino acid. The signals between 7 and 9 ppm originate from the hydrogen 
		atoms of the primary amin, the benzene ring of the CBT side chain and some Fmoc-derivatives which are left as impurities. The peak at about 11 ppm corresponds to the amin of the CBT side chain. Both NMR spectra show that
		the deprotection was successful.
		</DIV><DIV class="article">
		To test the solubility of both amino acids for further experiments, we dissolved 2.9 mg of the free form and 3.7 mg of the morpholine salt of the CBT‑asparagine each in 1 mL ammonium 
		bicarbonate with a pH of 7.4 to get a 10 mM solution of each. For a better solubility, 15 µL of 10 M NaOH were added. The following figure shows both samples after vortexing (Figure 19).
		</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 19: Comparison of the solubility of the free form and the morpholine salt of CBT‑asparagine.</B>Both materials were dissolved in 1 mL ammonium bicarbonate with 15 µL of 10 M NaOH. The free form of CBT-asparagin is less soluble in ammonium bicarbonate than the morpholine salt.</P></DIV><DIV class="article">
		Due to the neutral charge and the hydrophobic side chain of the free form of the CBT‑asparagine and compared to the morpholine salt of the CBT‑asparagine, the free form possesses 
		a lower solubility compared to the morpholine salt. Therefore, we used the morpholine salt for further experiments.
		</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Specific Binding of CL and CBT-asparagine </H3><DIV class="article">
		To test the condensation reaction between the 1,2-aminothiol group of CL and the cyano group of the CBT‑asparagine, we dissolved 4.70 mg of CL trifluoroacetic acid salt and 3.77 mg of CBT‑asparagine morpholine salt 
		in two separate tubes each with 5 mL of a 20 mM ammonium bicarbonate solution with a pH of 8
		to get a 2 mM solution of each amino acid. A few drops of 10 M NaOH were added to the solution with the CBT‑asparagine. Both solutions were mixed to get a reaction mixture with a concentration of 1 mM of each amino acid.
		For 40 minutes, every 5 minutes a sample was analyzed by LC-MS. Already after 5 minutes, we got a signal at 349 which could originate from the ligated product. After 40 minutes, following mass spectrum was obtained 
		for the ligation reaction (see Figure 20).
		</DIV><DIV class="figure large"><P class="figure subtitle"><B>Figure 20: Mass spectrum of the fraction with an elution time of 2.30 min of the ligation reaction.</B>The peak at 349 should correspond to the ligated product of CL and CBT‑asparagine morpholine salt which has a molecular weight of 611.73 g mol<SUP>-1</SUP>.</P></DIV><DIV class="article">
		Considering the exact mass of the ligated form of CL and CBT-asparagin of 522.14 Da and a 2x positive charge, we get a mass-to-charge ratio of 261.07 Da e<SUP>-1</SUP>. Adding the exact mass of the morpholine anion of 88.08 Da 
		and in consideration of the 1x positive charge, the peak for 349 Da e<SUP>-1</SUP> should correspond to the estimated product of the ligation reaction. Taking into account that there is a signal for a substance with a mass-to-charge 
		ratio of 349 Da e<SUP>-1</SUP> in the aqueous layer after the extraction of the reaction mixture of the deprotection reaction (see Figure 21) prior to the ligation reaction, it is not certain that the peak for 349 Da e<SUP>-1</SUP> seen in Figure 20 
		originates from the ligated amino acids. 
		</DIV><DIV class="figure large"><P class="figure subtitle"><B>Figure 21: Mass spectrum of the fraction with an elution time of 2.18 min of the aqueous layer after the extraction of the reaction mixture of the deprotection reaction.</B>Even before the ligation reaction, there is a peak for 349 Da e<SUP>-1</SUP>.</P></DIV><DIV class="article">
		The chromatograms of the aqueous layer after the extraction of the reaction mixture of the deprotection reaction and the reaction mixture of the ligation reaction (see Figure 22) show that there is a difference 
		between those signals.
		</DIV><DIV class="figure large"><P class="figure subtitle"><B>Figure 22: Comparison of the chromatogram of the aqueous layer after the extraction of the reaction mixture of the deprotection reaction (red) and the reaction mixture of the 
			ligation reaction (blue).</B>The peak of the substance with a mass-to-charge ratio of 349 of the red chromatogram is at 2.18 min and the one of the blue chromatogram is at 2.30 min.</P></DIV><DIV class="article">
		Due to the dilution by mixing the 2 mM solutions of both amino acids, the peaks over 8 min of the blue chromatogram are lower than the ones of the red chromatogram. On the other hand, the peak corresponding 
		to the substance with a mass-to-charge ratio of 349 Da e<SUP>-1</SUP> of the blue chromatogram is higher in relation to the peaks over 8 min compared with the ratio between the peak corresponding to the substance with a mass-to-charge
		ratio of 349 Da e<SUP>-1</SUP> and the peaks over 8 min of the red chromatogram. Additionally, the peaks of the substance with a mass-to-charge ratio of 349 Da e<SUP>-1</SUP> of the red chromatogram (elution time of 2.18 min) and of the blue 
		chromatogram are shifted to each other (elution time of 2.30 min). These aspects show that the substances with a mass-to-charge ratio of both samples should be different, thus showing that the ligation between CL
		and CBT-asparagin was successful.
		</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> References </H3><ARTICLE><B>Goodman, M. Toniolo, C., and Moroder, L.</B> (2004). Methods of Organic Chemistry: Synthesis of Peptide and Peptidomimetics 4th ed. (Thieme).
		<B>Hauser, J.R., Beard, H.A., Bayana, M.E., Jolley, K.E., Warriner, S.L., and Bon, R.S.</B> (2016). Economical and scalable synthesis of 6-amino-2-cyanobenzothiazole. Beilstein J. Org. Chem. <B>12</B>: 2019–2025.
		<B>Nguyen, D.P., Elliott, T., Holt, M., Muir, T.W., and Chin, J.W.</B> (2011). Genetically Encoded 1,2‑Aminothiols Facilitate Rapid and 
		Site‑Specific Protein Labeling via a Bio‑orthogonal Cyanobenzothiazole Condensation. J. Am. Chem. Soc. <B>133</B>: 11418–11421.
		<B>Yuan, Y., Wang, F., Tang, W., Ding, Z., Wang, L., Liang, L., Zheng, Z., Zhang, H., and Liang, G.</B> (2016). Intracellular Self-Assembly of Cyclic d -Luciferin Nanoparticles for Persistent Bioluminescence 
		Imaging of Fatty Acid Amide Hydrolase. ACS Nano <B>10</B>: 7147–7153.
		</ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>