<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_toolbox_photolysis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/toolbox/photolysis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/c/c0/T--Bielefeld-CeBiTec--gfpplate.jpeg);"><DIV id="title-bg"><DIV id="title-text">
						Photolysis
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Short Summary </H2><DIV class="article">
To prove our concept of a light-induced elution by photolysis, mediated by 2-nitrophenylalanine (2-NPA), we designed a fusion protein consisting of a streptavidin tag, a linker containing the amber codon and GFP as a target protein. We also showed that our aminoacyl-tRNA synthetase (aaRS, <A target="_blank" href="http://parts.igem.org/Part:BBa_K2201200">BBa_K2201200</A>) is functional and incorporates 2-NPA into the target protein when cotransformed. Via absorption measurements we demonstrated that our self-designed <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">LED panel</A> is able to induce the structural change of 2-NPA, needed for protein cleavage, by irradiation with UV-light <I>in vitro</I>. Due to an acceptable permeability of UV light, most of the commonly used 96-well microtiter plates are suitable for the irradiation process. While we had trouble verifying the binding activity of the streptavidin-tag, probably due to cell intern biotin-binding, we were able to proof the photolysis of the fusion protein by SDS-page and western blot.
</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Design of the 2-Nitrophenylalanine-tRNA Synthetase</H2><ARTICLE>
Based on the selection experiment of Peters <I>et al.</I> (2009), a sequence was designed to obtain a mutated tyrosine aminoacyl/tRNA-synthetase (TyrRS) from <I>Methanociccus janashii</I> for the amber-codon based translational incorporation of 2-nitrophenylalanine. The gene was subsequently ordered as a gene synthesis from IDT. 
An alignment with <A target="_blank" href="https://www.ebi.ac.uk/Tools/msa/clustalo/"> ClustalOmega</A> of the amino acid sequences of the native <I> M. jannaschii</I> tyrosyl synthetase(<I>M. jannaschii</I> TyrRS) and the used 2-nitrophenylalanine-synthetase (2-NPA-RS) is shown in Figure 1. The sequences differ in ten amino acids which, play a role in the binding process of the amino acid.
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 1: Alignment of the amino acids sequences with ClustalOmega of the <I>M. jannaschii</I> tyrosyl synthetase (<I>M. jannaschii</I> TryRS) and the 2-nitrophenylalanine synthetase (2-NPA-RS) designed by Peters <I>et al., 2009</I></B>.They differ in ten amino acids, which play a role in the binding process of the amino acid.</P></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Cloning of the NPA-RS in pSB1C3 and pSB3T5</H2><ARTICLE>
We used the amino acid sequence of the clone with the highest fidelity for 2-NPA and reverse translated it into a gene sequence, which was then codon optimized for <I>Escherichia coli</I>. Matching overhangs of 35 bp to a linearized o-(2-nitrobenzyl)-L-tyrosine-Part (<A target="_blank" href="http://parts.igem.org/Part:BBa_K1416000">BBa_K1416000</A>) were designed, so that it contains a matching promoter and terminator for the synthetase and also the tRNA required for the incorporation. The pSB1C3 backbone is a high copy plasmid. For an adequate usage of the aaRS, it is required to be on a low copy plasmid to reduce the stress caused by producing a high amount of the synthetases and the tRNA. BioBrick assembly was performed to insert the new 2-NPA-Part (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201200">BBa_K2201200</A>) into the low copy plasmid pSB3T5 (Figure 2) for further use. <A target="_blank" href="http://parts.igem.org/Part:BBa_K2201200">BBa_K2201200</A> in the low copy plasmid pSB3T5 is available on request.
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 2: Two Plasmids we created for our toolbox.</B> Left: 2-NPA-RS in the pSB1C3 high copy plasmid (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201200">BBa_K2201200</A>). Right: 2-NPA-RS in the pSB3T5 low copy plasmid (available on request).</P></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Design of the Fusion Protein</H2><DIV class="article">
Two fusion proteins were designed to verify the incorporation and functionality of 2-NPA (Figure 3). Plasmid I (containing <A target="_blank" href="http://parts.igem.org/Part:BBa_K2201320">BBa_K2201320</A>) encodes for a green fluorescent protein (GFP)-streptavidin fusion protein connected by a gly-gly-ser-linker (Figure 3A). Plasmid II (containing <A target="_blank" href="http://parts.igem.org/Part:BBa_K2201321">BBa_K2201321</A>) is homologous to plasmid I but has an amber codon in the middle of the linker at position six. If plasmid II is transformed in <I>E.coli</I> BL21(DE3):
(1)	Only the GFP-unit will be expressed (Figure 3B). 
(2)	If cotransformed with an aaRS for a non-canonical amino acid but without feeding the specific ncAA, the aaRS will incorporate other amino acids profoundly phenylalanine in the linker (Figure 3C). 
(3)	If cotransformed and with the 2-NPA in the culture media, the fusion protein will be expressed with 2-NPA in the linker (Figure 3D). The fusion protein can then be irradiated by light of a wavelength of &lt; 367nm.
(4)	This irradiation induces the cleavage of the fusion protein to its GFP-unit (Figure 3E) and the streptavidin-unit (Figure 3F).
</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 3: Design of two plasmids for fusion proteins.</B> I) Plasmid (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201320">BBa_K2201320</A>) for reference protein of GFP (green) a linker (purple) and streptavidin (yellow) (A). II) Plasmid (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201321">BBa_K2201321</A>) for the application protein with Amber-codon (black star) in the linker for three different protein variants after expression. 1: Solely expression leads to GFP-unit and linker to the Amber-codon (B). 2: Cotransformed with a 2-NPA-RS (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201200">BBa_K2201200</A>) without 2-NPA leads to a fusion protein with an unspecific amino acid (presumably phenylalanine, red star) in the linker (C). 3: Cotransformed with 2-NPA-RS and 2-NPA leads to the functional fusion protein with 2-NPA (purple star) in the linker (D). 4: Irradiation of protein D leads to a cleavage of the fusion protein in the GFP unit (E) and the streptavidin unit (F).</P></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Proof of Incorporation of Amino Acids at the Amber-Codon when Cotransformed with the 2-NPA-RS </H2><ARTICLE>
To proof the expression of the 2-NPA-RS (BBa_K2201200), we transformed it, the ONBY-Part (<A target="_blank" href="http://parts.igem.org/Part:BBa_K1416000">BBa_K1416000</A>), and a plasmid only containing the coding sequence of the fusion protein in <I>E.coli</I> BL21(DE3). The three transformants were cultivated in LB-media for 16 hours at 37 °C and 150 rpm. The cultures were harvested and the pellets lysed after the standard protocol of cell lysis in lysis buffer. Afterwards 15 µL of the samples were transferred to an SDS-Page (Figure 4). Two bands were present at approximately 35 kDA at the 2-NPA and the ONBY sample which correlate with the expected masses of the synthetases. 
</ARTICLE><DIV class="figure small"><P class="figure subtitle"><B>Figure 4: SDS-Page of the expressed 2-NPA-RS (left) from <A target="_blank" href="http://parts.igem.org/Part:BBa_K2201200">BBa_K2201200</A> with ONBY-RS from <A target="_blank" href="http://parts.igem.org/Part:BBa_K1416000">BBa_K1416000</A> as positive control (middle) and the basic protein expression of BL21(DE3) as negative control (right).</B></P></DIV><DIV class="article">
To proof the functionality of the 2-NPA-RS BBa_K2201200) the part was cotransformed with plasmid II (containing <A target="_blank" href="http://parts.igem.org/Part:BBa_K2201321">BBa_K2201321</A>) in <I>E.coli</I> BL21(DE3). Also plasmid I (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201320">BBa_K2201320</A>) and plasmid II were also transformed separately. After cultivation and cell lysis as mentioned above (in which one culture of the cotransformants was cultivated with 1 mM of 2-NPA and another  with 0 mM 2-NPA) the samples were transferred on an SDS-Page and a western-blot with anti-GFP-antibodies was performed (Figure 5).
</DIV><DIV class="figure medium"><P class="figure subtitle"><B>Figure 5: Western blot with GFP-antibodies of the four different fusion proteins variants (Figure 3) as proof of the functionality of the 2-NPA-RS.</B> The band marked with * in lane A is weak because of degradation of the fusion protein while the storage. The bands at approximately 45,0 kDa mark the mass of the whole fusion protein (~ 40,9 kDa), the bands at approximately 25,0 kDa mark the GFP-unit (~ 27,0 kDa) of the fusion protein.</P></DIV><ARTICLE>
Figure 5 shows that the complete fusion protein of plasmid I was expressed (Figure 5A). The GFP unit of plasmid II is solely expressed, if not cotransformed with a matching aaRS (Figure 5B) and the fusion protein of plasmid II was fully expressed when cotransformed with the 2-NPA-RS and cultivated with (Figure 5D) and without (Figure 5C) 2-NPA in the media.
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Permeability of Microwellplates by Irradiation of 367nm</H2><ARTICLE>
To proof which 96-well microtiter plates are best for the irradiation with our <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">LED panel</A> we validated three different plates. A black and a transparent 96-well plate from Nunc and a transparent well plate from Greiner (Figure 6).
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 6: Three microwell plates tested for their suitability for the irradiation with the <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">LED panel</A>.</B> Left: Black Nunc plate. Middle: Transparent Nunc plate. Right: Transparent Greiner plate.</P></DIV><ARTICLE>
The absorption of the empty plates was measured in a Tecan Infinite M200 (Roche) at wavelengths from 300 to 450 nm (Figure 7, left). The measured absorption values were converted to the amount of light that passes the plates in percentage (Figure 7, right). Figure 7 shows that all plates are suited for the application with the <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">LED panel</A>. At 367 nm 80 % of the light permits both transparent plates and 87 % of the light permit the black Nunc plate. A small fraction of light will always be absorbed but the amount of light that permits the plates is expected to be high enough for photolysis and the photoswitching experiments.
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 7: Results of the irradiation test of the three microwell plate.</B></P></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Change of Structure of 2-Nitrophenylalanine due to UV-Light</H2><ARTICLE>
To test if the structure of 2-NPA can be changed by irradiation with our <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">LED panel</A>, an <I>in vitro</I> test was conducted. LB media supplemented with 1 mM 2-NPA was used for absorption measurements over 4 hours while irradiated with UV-light in a 96-well microtiter plate. A high absorption in the UV spectrum (&lt; 300 nm) was observed, as expected because 2-NPA absorbs the UV-light to start the self-cyclization reaction (Figure 8). Over time a constant increase in the absorption spectrum at approximately 340 nm could be noticed, which indicates the emergence of a chemical component due to the irradiation process. We are confident that this compound is the 2-NPA in its cinnoline form, as proposed by Peters <I>et al.</I>, which leads to a shift in the absorption spectrum.
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 8: Changes in the absorption spectrum of 2-NPA in LB media while irradiated at 367 nm for 240 minutes.</B> The emerging peak at ~ 340 nm indicates the change in the structure of 2-NPA from its native form, to the self-cyclized form.</P></DIV><ARTICLE>
Therefore, we were able to confirm that the <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">LED panel</A> is suitable for the induction of the photolysis of 2-NPA. Furthermore, to the best of our knowledge, we conducted the most detailed documentation of the changes in the absorption spectrum of 2-NPA during the self-cyclization process to its cinnoline form.
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Cleavage of the Fusion Protein</H2><ARTICLE>
To prove that photolysis of 2-NPA is also possible when incorporated into a protein, three transformants of <I>E.coli</I> BL21(DE3) were cultivated. The first one contains the part <A target="_blank" href="http://parts.igem.org/Part:BBa_K2201320">BBa_K2201320</A> which expresses the whole fusion protein of GFP and streptavidin. The second one contains the part <A target="_blank" href="http://parts.igem.org/Part:BBa_K2201321">BBa_K2201321</A> and produces the GFP-unit of the fusion protein due to the amber stop codon in the linker. The third one is a cotransformant of <A target="_blank" href="http://parts.igem.org/Part:BBa_K2201200">BBa_K2201200</A> (2-NPA-RS) and <A target="_blank" href="http://parts.igem.org/Part:BBa_K2201321">BBa_K2201321</A> and therefore should produce the whole fusion protein with 2-NPA in the linker. The cells were cultivated in 50 mL LB-media (for the cotransformants supplemented with 1 mM 2-NPA) in 500 mL shaking flask for 16 hours at 37 °C and 150 rpm. The cells were harvested and lysed after the cell lyses protocol for SDS-Page. Two samples of the cell lysate of the cotransformants were transferred to the black Nunc plate and irradiated with the <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">LED panel</A> at 367 nm and 100 % brightness. After irradiation the samples were transferred to a SDS-Page (Figure 9).
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 9: SDS-Page of four different samples.</B> The whole fusion protein as positive control 1 (lane 1), GFP-unit as positive control 2 (lane 4), and two samples of the irradiated fusion protein containing 2-NPA after 1 hour (lane 2) and 5 hours (lane 3) of irradiation with UV-light. Highlighted in dark green are the bands of the whole fusion protein, in purple the bands of the 2-NPA-RS, in light green the GFP-unit of the fusion protein and in yellow the bands of the cleaved streptavidin-tag.</P></DIV><ARTICLE>
Figure 9 shows the bands of the whole fusion protein at approximately 40.9 kDa for the positive control 1 and the two samples. Furthermore, the bands of the GFP-units can be seen at ~ 27 kDa and the bands of the 2-NPA-RS at ~ 35 kDa. At the bottom of the SDS-page the bands of the cleaved streptavidin-tags of the two samples can be observed. Unfortunately, there was an uneven running front which led to a shift of the samples compared to the marker. Therefore, an additional western blot was performed (Figure 10) with specific anti-GFP and anti-strep antibodies to verify that the bands seen in Figure 9 are the expected peptides.
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 10: Western Blot of a SDS-Page similar to Figure 9.</B> Anti-GFP antibodies were used to determine the bands of the SDS-Page. The blot proves that the low bands in Figure 9 are indeed cleaved streptavidin-Tags.</P></DIV><ARTICLE>
The western blot confirmed our expectations. The bands of the whole fusion protein, of the GFP-units and the cleaved streptavidin-tags are clear to see. The GFP-bands are very thick compared to the others, which cannot be explained by the photolysis itself. It seems like the 2-NPA-RS is not very effective in loading 2-NPA to the amber tRNA, which leads to a high amount of incomplete expression of the fusion protein. The low efficiency of our 2-NPA-RS was confirmed by our synthetase-test system (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2201343">BBa_K2201343</A>) (Figure 11). Therefore, we started modeling and selecting a new aaRS, able to incorporate 2-NPA to provide a better synthetase to the iGEM community.
</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 11: Scores resulting from the synthetase-test system.</B> The negative rank results from the emission quotient CFP(475 nm)/YFP(525 nm) when cultivated without the specific ncAA. The positive rank results from the emission quotient YFP(525 nm)/CFP(475 nm) when cultivated with the specific ncAA. The mean rank allows the combination of the negative and the positive rank to compare the efficiency of synthetases among each other.</P></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> References </H2><B>Peters, F.B., Brock, A., Wang, J., and Schultz, P.G.</B> (2009). Photocleavage of the Polypeptide Backbone by 2-NitrophenylalaninePeters. Chem. Biol.<B> 16</B>: 148–152.
        </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>