Team:Bielefeld-CeBiTec
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Achievements
The following list sums up all of our achievements and refers to pages of this wiki containing detailed information. Next to this we recorded the original presentation of our project in Boston at the Giant Jamboree, which sums up and explains the project in less than 20 minutes:
Awards
Best Foundational Advance Project
Best New Basic Part
Nominations
Best Software
Best Hardware
Best New Composite Part
Best Part Collection
We established two orthogonal methods for the detection of unnatural base pairs in a target sequence: an Oxford Nanopore sequencing application and an enzyme based detection method
Development of a software suite for these orthogonal methods
Integration and characterization of  the nucleotide transporter PtNTT2 from P.tricornutum in E.coli for the uptake of  unnatural nucleoside triphosphates
Confirmation that certain Taq DNA polymerases can efficiently incorporate unnatural nucleotides
Construction of a toolbox consisting of five aminoacyl-tRNA synthetases for incorporation of non-canonical amino acids
Colocalization of the RuBisCo and and subcellular compartment (carboxysome) using a fluorescent amino acid
Development of a photoswitchable lycopene pathway
Design, chemical synthesis and proof of functionality of a novel, fully synthetic amino acid based on cyanonitrobenzothiazol and asparagine
Modeling more than ten new aaRS sequences
Library development with several hundred thousand sequences for selecting aminoacyl-tRNA synthetases
Construction of positive and negative selection plasmids for the evolution of new synthetases for non-canonical amino acids
Improvement of an aminoacyl-tRNA synthetase test-system by introducing a FRET-system and development of a ranking system
Construction of an LED panel for irradiating 96-well microtiter plates, which can be used to manipulate non-canonical amino acids and for other applications
Development of an Android App to control the LED panel with your smartphone via Bluetooth
Writing a biosafety report entitled “Auxotrophy to Xeno-DNA: A Comprehensive Exploration of Combinatorial Mechanisms for a High-Fidelity Biosafety System”
Writing the ChImp Report on “Chances and Implications of an Expanded Genetic Code”
