<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_HP_Gold_Integrated skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/HP/Gold Integrated</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/4/44/T--Bielefeld-CeBiTec--title-img-presentation.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Integrated Human Practice
					</DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><ARTICLE>
Due to the innovative nature of our project to expand the genetic code by adding new bases, our team faced novel challenges. 
One major issue was that this kind of project is far-off topic in our home university. Thus, we were grateful to cooperate with more 
than twenty scientists from all over the word. This was a great benefit for our work. Besides gaining important insights into important best 
practices, we kept in contact to leading experts to evaluate if our project can improve basic research methods. The feedback from 
the scientists and partners improved our skills and influenced our work and the planning of our experiments in a great way. 
The figure below displays the most influential experts we contacted throughout our work. Below the figure, information on who they are and how they helped us are listed.

</ARTICLE><DIV class="contentline"><DIV class="half right"><ARTICLE><B>Dr. Mario F. Feldman</B> is an associate professor of molecular microbiology at the Washington University in St. Louis, USA While deciding on a project topic, he helped us with research with regards to the glycosylation in <I>E. coli</I><B>Dr. Floyd E. Romesberg</B> is professor of chemistry and head of the Romesberg Lab at the Scripps Research Institute in California, USA. During the beginning of our project, he gave us motivating advice on our project in general.
<B>Dr. Nediljko Budisa</B> is a Professor at the TU Berlin, Germany, at the institute for chemistry. He provided us with valuable information on the aaRS evolution process and gave us access to his lab for one week to work on our project.
<B>Dr. Vitor Pinheiro</B> is a lecturer in synthetic biology and at the ISMB in London, UK, and leader of the Pinheiro Lab. With his expertise, he gave us initial advice regarding our project as a whole.
<B>Dr. Piet Herdewijn</B> is a professor at the faculty of pharmaceutical sciences at the KU Leuven, Belgium. With his expertise, he gave us initial advice regarding our project as a whole.
<B>Iker Valle Aramburu</B> is a predoctoral fellow at the EMBL Heidelberg, Germany. He gave us valuable initial information on labeling non-canonical amino acids.
<B>Dr.-Ing. Risto Kõiva</B> is the administrative head of the Bielefeld Excellence Cluster &quot;Cognitive Interaction Technology&quot; (CITEC) workshops. He answered our questions with regards to the hardware design and provided resources for building our hardware, especially with regards to materials and the workshop.
<B>Thomas Greiber</B> is working at the Bundesamt für Naturschutz (Federal Office of Nature Protection) in Bonn, Germany. He gave a talk on the Nagoya Protocol, which was relevant for our work.

</ARTICLE></DIV></DIV><ARTICLE><B>Dr. Norbert Sewald</B> is Professor at the faculty of chemistry and head of the department of organic chemistry III at Bielefeld University. He and his team provided us with necessary resources to synthetize our own non-canonical amino acid, including working materials, expertise, and venues.
<B>Dr. Marcel Frese</B> is currently working at the department of organic chemistry III at Bielefeld University. As part of Dr. Norbert Sewalds team, he provided us with necessary resources to synthetize our own non-canonical amino acid, including working materials, expertise, and venues.
<B>Dr. Sandip Jadhav</B> is currently working at the department of organic chemistry III at Bielefeld University. As part of Dr. Norbert Sewalds team, he provided us with necessary resources to synthesize our own non-canonical amino acid, including working materials, expertise, and venues.
<B>Dr. Martin Smith</B> is a computational biologist and head of genomic technologies at the Garvan Institute of Medical Research in Sydney, Australia. His research revolves around biological mechanisms that control how genes are activated and repressed. Dr. Martin Smith is also an expert in Oxford Nanopore Sequencing and gave us very helpful information for sequencing of very low amounts of DNA and single cell sequencing using Oxford Nanopore Sequencing.
<B>Prof. R. Alta Charo</B> is a professor of law and bioethics at the University of Wisconsin Law School, USA. She gave us valuabel opinions on chances and implications of an expanded genetic code from an ethical and philosophical perspective.
<B>Dr. Melanie Schwarz</B> is an account manager at Biolegio. She helped us with regards to oligonucleotides containing unnatural bases and gave us information on annealing and quality control of our work.
<B>Dr. Florian Richter</B> is currently working at Bayer in Cologne, Germany, and greatly helped us getting started with the ROSETTA software for our modeling project.
<B>Raul Machado</B> is an Assistant Professor at the University of Minho, Portugal, where he focusses on genetically engineered protein-based materials. He gave us advice on finding a method to build silk elastin like proteins by recursive directional ligation (pre RDL).
<B>Prof. Dr. Dirk Lütkemeyer</B> is the General Manager of BIBITEC GmbH. He gave us helpful advice concerning the construction and further development of the purification column eluX.
<B>Dr. Benjamin Müller</B> is the CEO of Biofidus AG, a bioanalytical company located in Bielefeld, Germany. During our meeting we discussed the pros and cons of the light-induced elution method. Furthermore, Dr. Benjamin Müller helped us with several questions about analytics of biomolecules.
<B>Prof. Dr. Thomas Noll</B> is cofounder of Xell AG and professor for cell culture technology at Bielefeld University. Together with Ole Weigelt, he advised us on the possible commercialization of our light-induced elution method and provided some insight into which companies might be interested in such a technology.
<B>Ole Weigelt</B> is tax consultant and lawyer at Weigelt Miersbach Uhlemeyer joint venture partner. Together with Prof. Thomas Noll, he advised us on the possible commercialization of our light-induced elution method and provided some insight into which companies might be interested in such a technology.
</ARTICLE></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Overview </H3><ARTICLE>
Our project involves a great amount of multidisciplinarity and touches topics of chemistry
 (<A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing#CBT">synthesis of our own
 designed amino acid</A>), electrical engineering (design and construction of 
 our <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware#LED">LED-Panel</A>), 
 synthetic biology, biosafety issues (<A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver#biosafety">Biosafety 
 report</A>) and awareness of consequences of our own research
 (<A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated#chimp">ChImp report</A>).
Some of the scientists we contacted participated in our 
workshop ‘<A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated#conference">Expanding 
of the Genetic Code</A>’. There, we received a lot of feedback for our project and obtained valuable information, but also
 learned about ethical concerns regarding our approach to incorporate new bases.  <A target="_blank" href="https://de.wikipedia.org/wiki/Nediljko_Budisa">Prof.
 Dr. Budisa</A> from TU Berlin even allowed us to work in his lab to optimize the selection of the amino acyl synthetase. 
 We incorporated all insights by working with the general public and numerous experts in ethics and prepared a report about ‘Chances and Implications of an expanded genetic code’, where we draft guidelines for the handling of unnatural bases. Working with a foreign plant gave us the chance to learn about the laws and guidelines of the <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated#nagoya">Nagoya protocol</A>.
Beside our main focus on expanding the genetic code, working with the general public has been an important part of our project.
 In practice, we organized and held <A target="_blank" href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">various events</A>. At the 6th 
 CeBiTec students’ academy and the Teutolab, we taught the basic methods of synthetic biology and bioinformatics to people of all ages. 
 At the GENIALE, a science event with booths all over the city, we did exciting experiments with visitors and showed them how to conduct
 experiment with every-day materials. 
The industrial usage can be an important factor for the design of applications. We had the chance to cooperate with MERCK, Biofidus AG, BIBITEC GmbH and an attorney for patent law, where we presented our project and got important feedback with regards to our approach on the unnatural bases and our applications.
The following sections highlight our interactions with experts and how they influenced our project. They not only shaped our project, 
but also broadened our horizon on how to communicate and interact with the public about our future research in our coming careers. 

</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> ChImp Report </H3><ARTICLE>
While synthetic biology offers solutions to many current issues, it has also been met with skepticism with regards to its responsible 
use and implications resulting from its research. To assess chances and implications of an expanded genetic code in particular, we interviewed 
experts from the scientific, philosophical, and religious fields. Furthermore, the public was asked for their opinions regarding this topics in a survey.
 On the basis of our research, we developed practical guidelines for research projects concerning synthetic biology, some of which we ourselves
 applied during our iGEM project. Evaluating all perspectives, we developed flexible and easily applicable multi-faceted guidelines for research 
 concerning synthetic biology, such as the incorporation of expert committees relevant for the project, as well as possible dual use concerns.
 The complete and comprehensive Report on Chances and Implications of an Expanded Genetic Code (ChImp Report) can be 
 accessed <A target="_blank" href="https://static.igem.org/mediawiki/2017/1/18/T--Bielefeld-CeBiTec--CMZ-ChImp.pdf">here</A>.
					</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Conference on Expanding the Genetic Code </H3><ARTICLE>
From the beginning, we knew that our project is multi-facetted, and that we would need the insight and expertise of experienced scientists. Therefore, we organized our own open workshop, “EGC: Expanding the Genetic Code”, on August 4th 2017 to obtain some advice and tips for our work. We did not just intend to broaden our horizon regarding the possibilities and challenges of our project, but also to get feedback and help from external experts in synthetic biology. <DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 1:</B> Excerpt of the program for our workshop EGC: Expanding the Genetic Code.</P></DIV></DIV><DIV class="half right">While the CeBiTec has an excellent reputation as a center for genome research and biotechnology, there is no research related specifically to the expansion of the genetic code. Scientists from the CeBiTec and from Bielefeld University were invited to our workshop. Since there is just basic research on this topic in Bielefeld, the staff benefited from our conference, and gave us very positive feedback regarding the workshop. Moreover, we invited interested students to increase the popularity of iGEM and the knowledge about synthetic biology on the campus of Bielefeld University. 
</DIV></DIV>

After the official part of the workshop, we had an intense exchange with two of our invited speakers. <A target="_blank" href="https://www.embl.de/research/units/scb/lemke/members/index.php?s_personId=CP-60016219">Iker Valle Aramburu (EMBL Heidelberg, Germany)</A> received a degree in Biochemistry at the University Basque Country, Spain and his MSc in Cell Biology at Heidelberg University. His field of study includes Physical Chemistry, Chemical Kinetics, and Biochemistry. His presentation dealt with “the plasticity of FG-Nucleoporins and NTR interactions”. Due to his experience with small molecular FRET, Mr. Aramburu gave us pointers for our analyzing tool.
 <A target="_blank" href="https://en.wikipedia.org/wiki/Nediljko_Budisa">Nediljiko “Ned” Budisa</A> is a biochemist and full professor of bio catalysis at the Technical University of Berlin, Germany. He is not just an expert regarding chemical synthetical biology (xenobiology), but also on genetic code engineering, making him an excellent adviser for our project. Ned talked about “the potentials of non-canonical amino acids in synthetic biology  and possible alternative life”. His talk gave a broad overview about the field. He put emphasis on the fact that the genetic code is degenerated and for incorporating non-canonical amino acids, an extension of the genetic code is needed. This extension could be achieved by evolution and selection of tRNA/amino acyl-tRNA synthetase pairs. This pair needs to be chosen depending on the chemical properties and conformation of the given non-canonical amino acids to ensure maximal substrate enzyme binding and incorporation efficiency.  

Through the following discussions regarding our project, Ned’s insight into our project was very important. Given that we want to incorporate non-canonical amino acids into proteins and peptides, he made us aware that we need to look into the formation and capacity of the binding pocket of the synthetases we want to evolve and select for the non-canonical amino acids. He also indicated possible problems for <I>in vivo</I> incorporation of non-canonical amino acids, mainly the elongation factor EF-Tu and the ribosomes. On the topic of unnatural base pairs and incorporation of non-canonical amino acids, Ned gave us the advice to consider using the synthetase for pyrrolysin. This specific synthetase does not recognize the anticodon loop on its corresponding tRNA. Thus, the changing of the codon and anticodon would possibly not interfere with ribosomal incorporation of a given amino acid using unnatural base pairs. 
We are very happy that Ned invited us to his laboratory in Berlin. Our team member Olga went to Berlin to work with experts on this research field and to use a sophisticated selection library for the generation of a novel tRNA synthetase. <DIV class="figure large"><P class="figure subtitle"><B>Figure 2:</B> Guest speakers and part of the iGEM team “Bielefeld-CeBiTec 2017”: (left to right, back) Prof. Dr. Nediljko Budisa, Iker Valler Aramburu, Yannic Kerkhoff, Saskia Dymek, Christopher Whitford, (left to right, front) Markus Haak, Michelle Liebers, Olga Schmidt, Denise Kerkhoff</P></DIV></ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Research in the Budisa Lab, Berlin </H3><ARTICLE>
At the end of September, our team member Olga went to Berlin to work within the Budisa Lab to learn more about distinct methodological approaches needed for the project, mainly to practice the handling of the selection of the aminoacyl-tRNA synthetases, at the invitation of Professor Nediljiko “Ned” Budisa during our Workshop “Expanding the Genetic Code”. </ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 3: </B> From left to right: Huan Sun, Professor Nediljko Budisa, Olga Schmidt and Fabian Schildhauer.</P></DIV><ARTICLE>
On the first day, the theoretical background of evolution and selection of the amino acyl-tRNA synthetases was discussed. This step was necessary because many labs evolve the synthetases according to the non-canonical amino acids they want to utilize, and various selection plasmids and protocols are used for selection processes. 
Based on the first positive selection round, problems that may occur in handling the selection process as a whole were explained. The problems compromise for example low transformation rates or low incorporation efficiency of the non-canonical amino acid provided. These particular problems are tried to be eradicated by preparing a fresh batch of competent cells for each selection step and by lowering the concentration of the antibiotic which corresponds to the resistance gene with the blank codon while prolonging the incubation time, respectively. 
We decided to study the selection process using two non-canonical amino acids we also use in our toolbox, namely 2-Nitro-L-phenylalanine (2-NPA) and Nε-L-cysteinyl-L-lysin (CL). We were lucky to use existing tyrosyl and pyrrolysyl synthetase libraries provided by the lab, respectively. We were made aware that the selection may not be as efficient as it could be utilizing those libraries, because the synthetases were evolved to bind the distinct non-canonical amino acids we are working with. </ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 4: Olga at work in Berlin. </B></P></DIV><ARTICLE>
During the handling of each first positive selection step, Olga was supervised by Huan Sun (pylRS) and Fabian Schildhauer (tyrRS).  </ARTICLE><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 5: Plates with colonies for first positive selection step for 2-NPA. </B> Because 2-NPA is photo cleavable, the plates could not be scanned as usual. (A) and (B) show the plates with added non-canonical amino acid. (C) shows the negative control plate without 2-NPA. Individual colonies could not be counted due to dense growth of the cells.</P></DIV></DIV><DIV class="half right"><DIV class="figure large"><P class="figure subtitle"><B>Figure 6: Plates with colonies for first positive selection step for CL. </B> Plates were scanned. (A) and (B) show the plates with added non-canonical amino acid. (C) shows the negative control plate without CL. Individual colonies could not be counted due to dense growth of the cells.</P></DIV></DIV></DIV><ARTICLE>
One negative selection round was also performed for CL by Huan Sun (Figure 7). Though not all of the needed three positive and two negative selection rounds could be performed, the preliminary results seem to be promising. To make sure that the non-canonical amino acid was incorporated through amber codon suppression, Mass Spectrometry needs to be performed. </ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 7: Plate for first negative selection for CL. </B> Plate was scanned. Colonies are difficult to see due to the plate being scanned.</P></DIV><ARTICLE>
We thank the whole Budisa Lab, Professor Budisa, Dr. Tobias Baumann, Huan Sun and Fabian Schildhauer for their help, interesting discussions and useful advice also after Olgas stay in Berlin. </ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Merck </H3><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 8: </B> iGEM Bielefeld-CeBiTec 2017 team members Olga Schmidt and Markus Haak presenting our project in front of the Merck scientists. </P></DIV></DIV><DIV class="half right"><ARTICLE><A target="_blank" href="https://www.merckgroup.com/de">Merck</A> is a leading company for products in the pharmaceutical and chemical sectors. Their expertise consists of three mayor sectors: Healthcare, Life Science and Performance Materials. This wide variety of subject areas ensures Merck's status as a global player. Their close collaboration with scientists ensures the high-quality of their products and know-how. Merck has supported the iGEM Bielefed-CeBiTec teams for seven years. We are grateful for their continuous scientific support and expertise.  
Back in August, we visited Merck to present our final project idea and the first results. Our visit was organized by Prof. Herget, who has been a great supporter of the iGEM-Teams Bielefeld for several years. 

        </ARTICLE></DIV></DIV><ARTICLE>

The following discussion dealt particularly with possible <I>in vitro</I> and <I>in vivo</I> approaches and the ethical concerns regarding our project. They confirmed our idea to add <I>in vitro</I> experiments as an additional control for our project. 
The moral concerns have also been addressed. Therefore, we designed a survey and asked for second opinions in a variety of meetings with ethicists. This eventuated in our report “ChImp - Chances and Implication of an Expanded Genetic Code”, from which we derive guidelines and recommendations for work awareness in science.
Visiting Merck is valuable in more than just one way: we did not just gain experience in holding a presentation in front of experienced researchers, but also received great advice for the future development of our project. 
We are excited to visit Merck again in 2018 to present our final results and the progress we have made.



</ARTICLE><DIV class="figure large"><P class="figure subtitle"><B>Figure 9: </B> iGEM Bielefeld 2017 team members Olga Schmidt and Markus Haak with representatives from the iGEM Bielefeld 2016 team and Merck.</P></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2 id="EluXExperts"> Discussion with Cell Product Purification Experts </H2><ARTICLE>
                    Feedback and advice from experts actually working in the fields of concern are valuable contributions to a successful iGEM project. To get advice on how to improve our <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">purification column EluX</A>, we met with two experts: Prof. Dirk Lütkemeyer, General Manager of BIBITEC GmbH, and Dr. Benjamin Müller, CEO of Biofidus AG for Analytical Services. BIBITEC GmbH was founded in 2001 and specializes in the production of recombinant proteins and monoclonal antibodies. Biofidus AG is a bioanalytical company offering services with a wide variety of bioanalytical methods. They are specialized on spectroscopic, chromatographic or mass spectrometric assays focused on characterization of proteins as well as small molecules. We demonstrated and explained our prototypes and elution technique. We also prepared specific questions, such as ‘What do you think about our elution technique compared to common techniques?’ and ‘Which problems and limitations could our column encounter in real world applications?’. As described in the following sections, we benefited greatly from these exchanges and made several improvements to our hardware to address the predicted challenges. Therefore, we are very thankful for their opinions and great advices.                 
                </ARTICLE><H3> Meeting with Prof. Dr. Dirk Lütkemeyer, General Manager of BIBITEC GmbH  </H3><ARTICLE>
		              We met Prof. Dirk Lütkemeyer at the 24th of August at the Center for Biotechnology. We presented our purification column and explained our light-induced elution technique. The presentation was followed by a fruitful discussion about possible challenges in real world applications, like using this purification technique to purify recombinant proteins produced by our local fermentation technologies working group.                 
                </ARTICLE><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 10: Meeting with Prof. Lütkemeyer. </B>Prof. Dirk Lütkemeyer (BIBITEC GmbH) and team member Yannic during a meeting at the Center for Biotechnology. A detailed discussion provided us with useful advices and opinions about our elution technique and purification column.</P></DIV></DIV><DIV class="half right"><ARTICLE>
			Prof. Lütkemeyer stated that our technique of the light-induced elution is very interesting and a promising alternative to proteases-cleaving sites. However, we need to validate the system profoundly. Especially interesting are the minimal amounts of light and radiation time needed for reliable back bone cleavage. He also said that our system could be very useful for companies related to the production of biomolecules like Evonik Industries AG and GE Healthcare. Nevertheless, we should improve our purification column prototype design to reduce dead space, increase potential yield and enhance the scalability. Finally, he recommended focusing on the validation of our elution system, since this is a highly innovative and promising aspect.
		</ARTICLE></DIV></DIV><H3>Meeting with Dr. Benjamin Müller, CEO of Biofidus AG </H3><ARTICLE>
We met Dr. Benjamin Müller (CEO of Biofidus AG) on the 6th of September at his office (Figure 2). We showed him our peptide purification column and discussed the possibilities and conceivable use cases of our light-induced elution technique. He commented on the advantage of our purification method, namely that the one step reaction is simple and easy to understand and thus attractive for possible users.
</ARTICLE><DIV class="contentline"><DIV class="half left"><ARTICLE>
He also pointed out some aspects we will have to check if a real use of this elution technique becomes current. He stated that we need to investigate if the light that is used for the elution will not harm the target protein and if there are any restrictions to the light-spectra, intensity, or irradiation time. We should also validate the stability of the amino acid itself, namely if it only reacts after irradiation, or possibly in the storage process. This would lead to a noticeable loss of the target protein. Furthermore, he stated that it would be useful to create a kit for the light-induced elution so that all users can apply this technique to their desired process and make the usage as easy as possible. 
</ARTICLE></DIV><DIV class="half right"><DIV class="figure large"><P class="figure subtitle"><B>Figure 11: Meeting with Dr. Benjamin Müller </B>Meeting of Yannic Kerkhoff with Dr. Benjamin Müller at his office on the 6th of September to discuss the light-induced elution technique.</P></DIV></DIV></DIV><H3> Meeting with Prof. Dr. Thomas Noll and Ole Weigelt </H3><ARTICLE>
To discuss possible marketing and commercialization strategies for our EluX technology, we were invited to a meeting by Prof. Thomas Noll, co-founder of the local biotech Xell AG, and Ole Weigelt, tax consultant and lawyer at Weigelt Miersbach Uhlemeyer Partnerschaftsgesellschaft. The meeting took place on the 7th of September at Ole Weigelt’s office. 
</ARTICLE><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 12: Meeting with Prof. Thomas Noll and Ole Weigelt  </B>Group picture after a beneficial meeting about possible marketing and commercialization strategies for our EluX technology. From left to right: Yannic Kerkhoff, Prof. Thomas Noll (co-founder of Xell AG), Ole Weigelt (co-founder of Xell AG, lawyer and tax consultant) and Christopher Whitford. </P></DIV></DIV><DIV class="half right"><ARTICLE>
Prof. Noll and Mr. Weigelt both stated that our idea has a high innovation potential and could be used as a new purification method. As examples they named the very fragile factor VIII and proteins A and G due to their expensive column material. Since the iGEM idea is based on making your research open source, sharing your results with the whole community, Prof. Noll and Mr. Weigelt said that an intense scientific analysis of the method is more adequate as commercializing it. Both encouraged us to pursue our idea properly and to continue the work even after the competition either by ourselves or by handing over our results to other students of our university, who could analyze this technique in degree theses.</ARTICLE></DIV></DIV><ARTICLE>
Both experts pointed out how multifaceted our concept is, as it includes basic molecular biology, cell development, photometry, linker design, cleavage kinetics and media optimization. Eventually, Mr. Weigelt and Prof. Noll advised us in testing the range of wave lengths and light intensities suitable for the cleavage of the target protein without harming either the target protein or the affinity tag. We thus started some investigations into how the UV-light could potentially harm our proteins.
	</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Meeting with Dr. Florian Richter </H3><ARTICLE>
For our modeling part of the project, we decided to use the powerful ROSETTA software. One of the protocols necessary for our work, the Enzyme Design Protocol, was developed by Dr. Florian Richter from Bayer AG. As the protocol is very complex, we asked Dr. Richter to give us an introduction to the protocol and discuss possibilities to model the evolution of a tRNA synthetase to a non-canonical amino acid. Hence, we joined Dr. Richter on the 28<SUP>th</SUP> of July in Cologne. Besides providing some technical details, he gave us many valuable tipps on what to consider, theoretical challenges, and required time. Specifically, he advised us on relying not just on algorithms, but also using our chemical expertise to sort through sequences of possible synthetase candidates. He also recommended ordering the most promising candidates via gene synthesis and stated that creating a library is very useful, but on the other hand also very time-consuming. Lastly, he clearly stated that ROSETTA only provided theoretical results, while in reality, yields might be very different. He then went on to demonstrate the ROSETTA protocol and its outputs to give us an impression on how to make use of the input lines. In conclusion, his encouragement and expertise greatly helped us to theorize and plan our modeling project in such a way that we were able to model the evolution of synthetases to non-canonical amino acids.
					</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Talk on single molecule real time sequencing </H3><DIV class="contentline"><DIV class="half left"><ARTICLE>
One of the problems we had to face when dealing with unnatural bases was how to detect them in sequence analysis. 
However, there were many different sequencing methods that we had to consider. One of these is the single molecule real time sequencing 
that for instance is offered by pacific biosciences. For this purpose, we invited representatives of the company to introduce the method as well
 as data analysis patterns to us. The 5<SUP>th</SUP> of September, 2017, at 5pm,  Mr. Cong Luan Nguyen as well as Mr. David Stucki 
 gave a talk on “Single Molecule Real Time Sequencing with the Sequel System and an outlook in data analysis”. This talk was attended by
 our team as well as many interested colleagues. It was concluded with interesting discussion 
 featuring different aspects of the technology. The talk highlighted why this system is good to be used with unnatural base pairs 
 but also showed advantages of a general usage at the CeBiTec.   
Therefore, all in all, we gained some more insights into sequencing with the new technology and possible usages for our project.
				</ARTICLE></DIV><DIV class="half right"><DIV class="figure large"><P class="figure subtitle"><B>Figure 13: </B> Members of the team talking to the invited experts prior to the talk. Interested members of the CeBiTec and our team within the audience.</P></DIV></DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H3> Nagoya-Protocol </H3><ARTICLE>The “Nagoya Protocol on Access to Genetic Resources and the Fair and Equitable Sharing of Benefits Arising from their Utilization (ABS) to the Convention on Biological Diversity” entered into force on October 12th, 2014 in Nagoya. It is an international environmental agreement to implement the objectives of the 1993 UN Convention on biological diversity: <A href="https://www.cbd.int/abs/about/">“the fair and equitable sharing of benefit arising out of the utilization of genetic resources”</A>. Especially developing countries which often have a manifold biodiversity are affected by bio piracy. The Nagoya Protocol intends to ensure that these countries at least profit financially or non-financially by the resulting research and products of their genetic resources. In addition to accessing genetic resources, it covers traditional knowledge concerning these resources. Contracting countries are bound to take action to guarantee informed consent prior to any course of action, as well as impartial benefit-sharing, respecting local laws as well as customary use and exchange. Today, <A href="https://absch.cbd.int/">100 countries signed the Nagoya Protocol</A>. To succeed, every partner nation should establish ABS National Focal Points, competent national authorities, legislative administrative, national databases, and checkpoints for information.Germany signed the Nagoya protocol on July 20th, 2016. Our donor of the isoG-metabolism-pathway is <I>Croton tiglium</I>, which is a herb in traditional Asian medicine. Its original occurrence is in the Asian region. To make sure we do not infringe upon the Nagoya Protocol regulations, we clarified the origin of our plant as we got it from the botanical garden of the Phillips University Marburg. <I>Croton tiglium</I> is originally from an undocumented old stock of the botanical garden Giessen. In 1986, the botanical garden Marburg received seeds from this stock. The date on which the EU regulation entered into force was October 2014. All resources collected before this date are not subject to reporting requirements of the Nagoya Protocol in Germany. Thomas Greiber (Federal Agency for Nature Conservation, Head of Division Div I 1.4 “Competent National Authority for the Nagoya Protocol”) confirmed “[..]Therefore, the respective areas of application are not touched by this EU-regulation, and the mentioned regulations are not relevant.” Therefore, we are allowed to work with the plant and publish information and results about gene sequences and metabolism pathways without violating applicable law. </ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>