<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Part_Collection skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Part Collection</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/b/b8/T--Bielefeld-CeBiTec--title-img-preculture.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Part Collection
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Short Summary</H2><ARTICLE>
Our aim was to expand the possibilities in protein design and with our part collection we were able to realize this plan. With the parts listed below we enable every iGEM team to incorporate any non-canonical amino acid (ncAA). These ncAAs provide additional chemical functions, that canonical amino acids lack. Our toolbox contains five different tRNA/aminoacyl-synthetases (aaRS) that could incorporate different ncAAs. All these ncAAs have chemical abilities that enable to design proteins with new functions. This applications are only a proof of concept. With our selection system, every iGEM Team could evolve their own aaRS for their amino acids and expand the possibilities for protein design themselves.
</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2>Part Collection for the Incorperation of Non-canonical Amino Acids</H2><ARTICLE>
Currently, protein design is limited to the chemical functions of the 20 to 22 canonical amino acids. Our part collection allows to add novel amino acids with other chemical abilities than those of the canonical, to expand the possibilities of advanced protein design. It contains five different tRNA/aminoacyl-synthetases (tRNA/aaRS) to incorporate non-canonical amino acids (ncAAs) during translation. These aaRS are parts of our toolbox and provide the translational incorporation of four different ncAAs as new functionality to the iGEM community. All amino acids add additional functional groups, thus functions to enable advanced protein design. The amino acids that can be incorporated with the aaRS  in our toolkit are:
</ARTICLE><UL><LI><B>2-nitrophenylalanine (2-NPA)</B>, which cleaves the backbone when irradiated by light of a certain wavelength. Therefore, it could be used for inactivation and activation of proteins.</LI><LI><B>Propargyllysine (PrK)</B>, which provides a propargyl group. Propargyl groups form a highly specific bond to azides in click chemistry reaction. Thus, this amino acid could be used for specific, terminus independent labeling.</LI><LI><B>p-acetophenylalanine (AcF)</B>, which provides a ketone group. Ketones form a highly specific covalent bond to hydroxylamines. Like PrK, this amino acid could be used for specific, terminus independent labeling.</LI><LI><B>7-hydoxy-L-coumaryl-ethylglcine (CouAA)</B>, a fluorescent amino acid, which could be used for <I>in vivo</I> and <I>in vitro</I> localization and labeling.</LI></UL><DIV class="article">
In addition to parts for the translational incorporation of hese ncAAs, our part collection also provides parts for a selection system to evolve novel tRNA synthetases. Moreover, instructions for the generation of a library of plasmids, that contain an aaRS encoding sequence with randmly mutated amino acids in the appropriate tRNA-binding sites. Additionally, a positive and negative selection system enables every iGEM team to expand the possibilities of protein design. 
The library and the positive selection plasmid (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201900">BBa_K2201900</A> in pSB3T5) are cotransformed to select sells, which survive due to a synthetase from the library that incorporates an any amino acid in response to the amber codon. Thus, all clones that survive the positive selection contain an aaRS ceapable of incorporating the target ncAA or any endogenous amino acid. To select only the clones that incorporate only the target ncAA, an additional round of negative selection is necessary.
The negative selection plasmid (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201901">BBa_K2201901</A> in pSB3T5) is used to select for high specificity of enzymes. During this negative selection, the target ncAA is not supplemented in the media. The negative selection plasmid encodes the barnase, a cell toxic protein. Since the coding sequence contains two amber codons at permissive sites, the gene product is only generated if the ncAA is present or if the aaRS is unspecifically integrated a canonical amino acid. 
To determine the most effective and specific aaRS for the incorporation of the desired ncAAs, an evaluation system of the synthetasesis crucial. Therefore, we optimized the synthetase test system “pFRY” initially developed by iGEM Texas 2014 leading to our new screening part <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201343">BBa_K2201343</A>. Our screening system contains the CDS of the cyan fluorescent protein and yellow fluorescent protein connected with a linker containing an amber stop codon. This part is placed placed under control of a T7-promotor to ensure high product amounts on demand. The amount and ratio of both fluorescent proteins could indicate how efficient and specific the investigated aaRS is working. Thoroughly characterization of all parts (Table 1) can be found on the respective part pages.
Our constructed parts are listed below. For the complete characterization please refer to the linked part collection pages.
</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 1: Plasmid map of BBa_K2201200</B> Plasmid map of the 2-NPA-aaRS (BBa_K2201200).</P></DIV></DIV><DIV class="half right"><B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201200">BBa_K2201200</A> 2-NPA-aaRS in response to UAG</B>
Evolved tyrosine tRNA/aminoacyl-synthetase for the incorporation of 2-NPA in response to the amber stop codon (CUA).
			
</DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 2: Plasmid map of BBa_K2201201</B> Plasmid map of the PrK-aaRS (BBa_K2201201).</P></DIV></DIV><DIV class="half right"><B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201201">BBa_K2201201</A> PrK-aaRS in response to UAG</B>
Evolved Pyrrolysyl tRNA/aminoacyl-synthetase for the incorporation of PrK in response to the amber stop codon (UAG).
</DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 3: Plasmid map of BBa_K2201202</B> Plasmid map of the AcF-aaRS in response to the amber stop codon.</P></DIV></DIV><DIV class="half right"><B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201202">BBa_K2201202</A> AcF-aaRS in response to UAG</B>
Evolved tyrosine tRNA/aminoacyl-synthetase for the incorporation of AcF-aaRS in response to the amber stop codon (UAG).
</DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 4: Plasmid map of BBa_K2201203</B> Plasmid map of AcF-aaRS in response to the less used leucine codon CUA.</P></DIV></DIV><DIV class="half right"><B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201203">BBa_K2201203</A> AcF-aaRS in response to CUA</B>
Evolved tyrosine tRNA/aminoacyl-synthetase in response to the less used leucine codon in <I>E. coli</I> (CUA).
</DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 5: Plasmid map of BBa_K2201204</B> Plasmid map of the CouAA-aaRS in response to the amber stop codon UAG.</P></DIV></DIV><DIV class="half right"><B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201204">BBa_K2204</A> CouAA-aaRS in response to UAG</B>
Evolved tyrosine tRNA/aminoacyl-synthetase for the incorporation of the fluorescent amino acid CouAA in response to the amber stop codon (UAG).			
</DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 6: Plasmid map of BBa_K2201900</B>Plasmid map of the positive selection plasmid. </P></DIV></DIV><DIV class="half right"><B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201900">BBa_K2201900</A> Positive selection plasmid for the incorporation of ncAAs</B>
The positive selection plasmid contains a tRNA and a kanamycin resistance with two amber codons. Cotransformed with the library of tRNA/aminoacyl-synthetase with random mutated binding sites, on kanamycin containing plates only the clones survive that could charge any amino acid to the tRNA in response to the UAG codon.
</DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 7: Plasmid map of BBa_K2201901</B> Plasmid map of the negative selection plasmid for the selection of aaRS that specifically inocporate ncAAs.</P></DIV></DIV><DIV class="half right"><B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201901">Ba_K2201901</A> Negative selection plasmid against the incorporation of endogenous amino acids</B>
The negative selection plasmid contains an tRNA with the anticodon for the amber codon and a barnase containing amber codons at permissive sites. In the negative selection, the target amino acid is not supplemented to the media. If the cotransformend clones from the positive selection charge endogenous amino acids to the tRNA, the cells die. This provides a selection method for high specific aaRS.
				
</DIV></DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><DIV class="contentline"><DIV class="half left"><DIV class="figure large"><P class="figure subtitle"><B>Figure 8: Plasmid map of BBa_K2201343</B> Plasmid map of the screening plasmid for the incorporation of ncAAs.</P></DIV></DIV><DIV class="half right"><B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2201343">BBa_K2201343</A> Screening system for the incorporation of ncAA</B>	
Our screening system contains the CDS of cyan fluorescent protein and yellow fluorescent protein connected with a linker containing the amber stop codon, under control of a T7-promotor. The amount and relation of the fluorescent proteins could indicate how efficient and specific the cotransformed aaRS is.			
</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>