<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_toolbox_analysing skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/toolbox/analysing</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar left"><UL class="navbar-paragraph left"><LI class="navbar-paragraph left"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Expanding the Genetic Code</A></LI></UL><UL class="navbar" id="navbar-left"><LI class="navbar item-left" id="hp"><A>Human Practice ▼</A></LI><LI class="navbar item-left" id="specials"><A>Additional ▼</A></LI><LI class="navbar item-left" id="results"><A>Results ▼</A></LI><LI class="navbar item-left" id="project"><A>Theory ▼</A></LI></UL></DIV><DIV class="navbar right"><UL class="navbar-paragraph right"><LI class="navbar-paragraph right"><A class="no-link" href="https://2017.igem.org/Team:Bielefeld-CeBiTec">Bielefeld-CeBiTec 2017</A></LI></UL><UL class="navbar" id="navbar-right"><LI class="navbar item-right" id="collaborations"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI><LI class="navbar item-right" id="team"><A>Team ▼</A></LI><LI class="navbar item-right" id="partners"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="navbar item-right" id="notebook"><A>Notebook ▼</A></LI></UL></DIV><DIV class="sub-navbar"><UL class="sub-navbar-list" id="navbar-project"><LI class="sub-navbar-item" id="project-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/Overview">Aim of the Project</A></LI><LI class="sub-navbar-item" id="project-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="project-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="project-toolbox"><A>Toolbox ▼</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-ubp"><LI class="sub-navbar-item small" id="project-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="project-ubp-ubp"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/unnatural_base_pairs">Unnatural Base Pairs</A></LI><LI class="sub-navbar-item small" id="project-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/uptake_and_biosynthesis">Uptake and Biosynthesis</A></LI><LI class="sub-navbar-item small" id="project-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="project-ubp-challenges"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/unnatural_base_pair/challenges">Challenges </A></LI></UL><UL class="sub-navbar-list" id="navbar-project-translation"><LI class="sub-navbar-item small" id="project-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="project-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="project-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-project-toolbox"><LI class="sub-navbar-item small" id="project-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="project-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="project-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="project-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="project-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results"><LI class="sub-navbar-item" id="results-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Demonstrate">Results Overview</A></LI><LI class="sub-navbar-item" id="results-ubp"><A>Unnatural Base Pair ▼</A></LI><LI class="sub-navbar-item" id="results-translation"><A>Translational System ▼</A></LI><LI class="sub-navbar-item" id="results-toolbox"><A>Toolbox ▼</A></LI><LI class="sub-navbar-item" id="results-parts"><A>Parts ▼</A></LI><LI class="sub-navbar-item" id="results-achievements"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-ubp"><LI class="sub-navbar-item small" id="results-ubp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair">Overview</A></LI><LI class="sub-navbar-item small" id="results-ubp-uptake"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/uptake">Uptake</A></LI><LI class="sub-navbar-item small" id="results-ubp-biosynthesis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/biosynthesis">Biosynthesis</A></LI><LI class="sub-navbar-item small" id="results-ubp-preservation"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/preservation_system">Retention System</A></LI><LI class="sub-navbar-item small" id="results-ubp-new_methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/unnatural_base_pair/development_of_new_methods">Development of New Methods </A></LI></UL><UL class="sub-navbar-list" id="navbar-results-translation"><LI class="sub-navbar-item small" id="results-translation-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system">Overview</A></LI><LI class="sub-navbar-item small" id="results-translation-mechanism"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/translation_mechanism">Translation Mechanism</A></LI><LI class="sub-navbar-item small" id="results-translation-library"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/translational_system/library_and_selection">Library and Selection</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-toolbox"><LI class="sub-navbar-item small" id="results-toolbox-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox">Overview</A></LI><LI class="sub-navbar-item small" id="results-toolbox-analysing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/analysing">Analyzing</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photoswitching"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photoswitching">Photoswitching</A></LI><LI class="sub-navbar-item small" id="results-toolbox-labeling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/labeling">Labeling</A></LI><LI class="sub-navbar-item small" id="results-toolbox-photolysis"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/photolysis">Photolysis</A></LI><LI class="sub-navbar-item small" id="results-toolbox-fusing"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Results/toolbox/fusing">Fusing</A></LI></UL><UL class="sub-navbar-list" id="navbar-results-parts"><LI class="sub-navbar-item small" id="results-parts-parts"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-basic"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Basic_Part">Basic Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-composite"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Composite_Part">Composite Parts</A></LI><LI class="sub-navbar-item small" id="results-parts-collection"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Part_Collection">Part Collection</A></LI><LI class="sub-navbar-item small" id="results-parts-improve"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Improve">Improve</A></LI></UL><UL class="sub-navbar-list" id="navbar-specials"><LI class="sub-navbar-item" id="hardware"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Hardware">Hardware</A></LI><LI class="sub-navbar-item" id="software"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Software">Software</A></LI><LI class="sub-navbar-item" id="modelling"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Model">Modeling</A></LI><LI class="sub-navbar-item" id="measurement"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Measurement">Measurement</A></LI><LI class="sub-navbar-item" id="applied-design"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Applied_Design">Applied Design</A></LI><LI class="sub-navbar-item" id="results-ils"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/InterLab">InterLab Study</A></LI></UL><UL class="sub-navbar-list" id="navbar-hp"><LI class="sub-navbar-item" id="hp-overview"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP">Overview</A></LI><LI class="sub-navbar-item" id="hp-gold"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Gold_Integrated">Gold</A></LI><LI class="sub-navbar-item" id="hp-silver"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/HP/Silver">Silver</A></LI><LI class="sub-navbar-item" id="hp-pe"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Engagement">Public Engagement</A></LI></UL><UL class="sub-navbar-list" id="navbar-team"><LI class="sub-navbar-item" id="team-members"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team">Members</A></LI><LI class="sub-navbar-item" id="team-advisors"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Advisors">Advisors</A></LI><LI class="sub-navbar-item" id="team-bielefeld"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld">Bielefeld</A></LI><LI class="sub-navbar-item" id="team-cebitec"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/CeBiTec">CeBiTec</A></LI><LI class="sub-navbar-item" id="team-uni"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Team/Bielefeld_University">Bielefeld University</A></LI><LI class="sub-navbar-item" id="team-attributions"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI></UL><UL class="sub-navbar-list" id="navbar-notebook"><LI class="sub-navbar-item" id="notebook-methods"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Methods">Methods</A></LI><LI class="sub-navbar-item" id="notebook-oligonucleotides"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Oligonucleotides">Oligonucleotides</A></LI><LI class="sub-navbar-item" id="notebook-labjournal"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/Labjournal">Labjournal</A></LI><LI class="sub-navbar-item" id="notebook-lng"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Notebook/LabnotesGenerator">Labnotes Generator</A></LI><LI class="sub-navbar-item" id="notebook-safety"><A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Safety">Safety</A></LI></UL></DIV><DIV class="container"><DIV id="title" style="background-image: url(https://static.igem.org/mediawiki/2017/0/0c/T--Bielefeld-CeBiTec--title-img-analyzing.jpg);"><DIV id="title-bg"><DIV id="title-text">
						Analyzing
					</DIV></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Short Summary </H2><ARTICLE>
		As part of our <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox"> toolbox</A>, structural analysis of a protein could be used to study 
		distances between non-canonical amino acids with <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing#FRET">Foerster Resonance Energy Transfer 
		(FRET)</A>. This provides measuring distances within specific incorporated amino acids in
		the target protein to gain insight into protein folding or structural changes under 
		different conditions. 
		
		To demonstrate this tool, we developed a <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing#prion">prion</A> detection assay. We used the yeast 
		prion <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing#Sup35">Sup35</A> as a model protein and incorporate two non‑canonical amino acids 
		(<A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing#AcF">p-acetophenylalanine</A> and <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing#PrK">propargyllysine</A>). After purification, the recombinant 
		produced Sup35 could be labeled with two different fluorophores (<A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing#Cy3Cy5">Cyanin 3 and Cyanin 5</A>).
		The emission spectra of the fluorophores depend on their distance. 
		When this test protein gets in contact with prions, the prions conformational changes 
		result in the change of the fluorophores spectra. Therefore, the test prion could be 
		used to detect prions in medical samples.
		</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Structural Analysis with Non-canonical Amino Acids </H2><ARTICLE>
		The structure of proteins could be detected through protein crystallography.  
		However, there are some problems when it comes to highly flexible proteins, or 
		proteins which change their structure within different conditions. To analyze these
		proteins and the changes in their conformation we want to establish a tool, that allows
		to detect changes in protein conformation.  For the detection of those changes, two 
		amino acids are incorporated at specific positions into the protein. These amino acids 
		could then be labeled with chromophores, enabling the measurement of the proteins 
		distances with <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/toolbox/analysing#FRET">Foerster Resonance Energy Transfer (FRET)</A>(Lembke, 2011, Kim <I>et al.</I>, 2013).
		
		The first step is the incorporation of the non-canonical amino acids. In proteins which
		 contain no cysteines naturally (cysteines are the only canonical amino acids, that
		could be labeled specific) or in which the exchanges of cysteines does not influence 
		the structure, only one ncAA and one cysteine at specific points need to be incorporated
		to be labeled. In proteins that contain cysteines, two ncAAs need to be incorporated for 
		the labeling (Kim <I>et al.</I>, 2013). 
		
		Non-canonical amino acids could be incorporated by <A href="https://2017.igem.org/Team:Bielefeld-CeBiTec/Project/translational_system/translational_mechanism">orthogonal tRNA/aaRS pairs</A> using the amber stop codon. However, this allows only the incorporation of one 
		non-canonical amino acid. To incorporate the second amino acid, another orthogonal 
		amino acid has to be used for the incorporation. Another codon that could be repurposed is the rarely used leucine codon CUA. With the use of this and the amber codon, two different ncAAs could be used. For structural analysis, the amino acids are specific labeled with chromophores. This labeling (shown in figure 1)
		 is possible due to the functional groups of the amino acids, which could 
		form a covalent bond to the fluorophores in a chemical reaction. 
		After the protein is labeled, the fluorescence of the chromophores could be measured to 
		draw conclusions on the distance of the ncAAs from each other (Brustad <I>et al.</I>, 2008, Kim <I>et al.</I>, 2013).
		</ARTICLE><DIV class="figure large"><P class="figure subtitle"><B>Figure 1: Target protein labeled with fluorophores. </B> The ncAAs AcF and PrK are incorporated in the target protein. After bi-orthgonal chemical conjugation the ncAAs are coupled with the fluorescent dyes cyanin 3 (Cy3) and cyanin 5 (Cy5). </P></DIV><H4> Propargyllysine (PrK)</H4><ARTICLE>
			One amino acid which provides a functional group different to the canonical amino
			acids is propargyllysine. The propargyl group of PrK could form a covalent bond to
			acidic groups in a click chemistry reaction. PrK is commercially available or could
			be synthesized chemically in two steps starting with Boc‑I‑Lys‑OH (Kim <I>et al.</I>, 2013). 
			The click chemistry reaction is performed at neutral pH, native buffers and
			temperatures of 4° C to 37 °C. However, for the click‑chemistry reaction copper
			is required which is toxic for living cells, so PrK could not be used for <I> in
			vivo </I> labeling (Kim <I>et al.</I>, 2013).
			</ARTICLE><DIV class="contentline"><DIV class="half left"><UL><LI> Name: 		Propargyllysine
                        </LI><LI> Short:		PrK
                        </LI><LI> CAS:		1428330-91-9
                        </LI><LI> MW:		228.25 g mol<SUP>-1</SUP></LI><LI> Storage:           4 °C
                        </LI><LI> Source:		<A href="https://www.sichem.de/de/sc-8002.html">Sichem</A></LI><LI> Prize:		1g - £300.00
                        </LI><LI> Function:          Propargyl group for click-chemistry reaction
                    </LI></UL></DIV><DIV class="half right"><DIV class="figure medium"><P class="figure subtitle"><B>Figure 2: Structure of PrK</B> Propargyllysine (PrK).</P></DIV></DIV></DIV><H4> p‑Acetylphenylalanine (AcF) </H4><ARTICLE>
Another amino acid with an additional functional group to the canonical amino acids
			is p‑acetylphenylalanine. The ketone group of AcF is able to build a covalent bond to a
			hydroxylamine coupled dye in a hydrazide reaction. This reaction is carried out at low 
			pH‑values which causes problems with certain proteins (Kim <I>et al.</I>, 2013).
</ARTICLE><DIV class="contentline"><DIV class="half left"><UL><LI> Name: 		p‑Acetylphenylalanine
                        </LI><LI> Short:		AcF
                        </LI><LI> CAS:		122555-04-8
                        </LI><LI> MW:		207,23 g mol<SUP>-1</SUP></LI><LI> Storage:           -20 °C
                        </LI><LI> Source:		<A href="https://www.abcr.de/shop/en/catalog/product/view/sku/AB390573">abcr</A></LI><LI> Prize:		1g - £509.00
                        </LI><LI> Function:          Ketone group for hydrazide reaction
                    </LI></UL></DIV><DIV class="half right"><DIV class="figure medium"><P class="figure subtitle"><B>Figure 3: Structure of AcF</B> p-Acetylphenylalanine (AcF).</P></DIV></DIV></DIV><H4> Foerster Resonance Energy Transfer (FRET) </H4><ARTICLE>
		Foerster Resonance Energy Transfer or Fluorescence Energy Transfer, short FRET, 
		describes an energy transfer between two chromophores. During this process the donor 
		chromophore is excited by light of a certain wavelength and transfers the energy to the acceptor chromophore if they 
		are within a certain distance to each other. In biochemistry, FRET is mostly used as 
		measurement tool with the help of fluorescent dyes. Using FRET, the measuring of
		distances from 1 to 10 nm is possible (Brustad <I>et al.</I>, 2006). The FRET process is shown in Figure 4.	
		</ARTICLE><DIV class="figure large"><P class="figure subtitle"><B>Figure 4: Animations of a FRET fluorophore pair </B> Animation of the distance dependent energy transfer of two fluorophores.</P></DIV><ARTICLE>
		The FRET efficiency (E) is used to estimate the intramolecular distance of molecules.
		
		E=[1+(r/R<SUB>0</SUB>)<SUP>6</SUP>)]<SUP>-1</SUP></ARTICLE><ARTICLE>
		E  FRET efficiency 
		r  intermolecular distance 
		R<SUB>0</SUB> Foerster distance for a given dye pair
		</ARTICLE><H4> Cyanin 3 and Cyanin 5 </H4><ARTICLE>
		Cyanin 3 (Cy3) in combination with Cyanin 5 (Cy5) is a chromophore pair which is 
		suitable for FRET measurements. Cy3 operates as the donor dye and Cy5 as the acceptor
		dye. The extinction and emission spectra of both chromophores is shown in Figure 5 (Kim <I>et al.</I>, 2013).
		</ARTICLE><DIV class="figure medium"><P class="figure subtitle"><B>Figure 5: </B> Spectra of the fluorophore pair  Extinction and emission spectra of Cy3 and Cy5. </P></DIV><ARTICLE>
		For the labeling of the noncanonical amino acids with these chromophores a functional
		group is required to build a covalent bond in chemical conjugation. PrK could be 
		labeled with Cy3‑azide and AcF with Cy5‑hydrazide. Both chromophores are relatively advantageously priced
		in comparison to other chromophore pairs and commercially available from <A href="https://de.lumiprobe.com/"> Lumiprobe</A>.
		</ARTICLE></DIV></DIV><DIV class="contentbox"><DIV class="content"><H2> Prion Detection Assay
	
</H2><H4> Prions </H4><ARTICLE>
	Prions are proteins that could infect other proteins to change their conformation.
	This is often causing a loss of function and aggregation of these proteins. Prions are 
	the cause for diseases like transmissible spongiform encephalopathies (TSEs), 
	neurodegenerative disorders that effect humans and animals (Wickner <I>et al.</I>, 2015). 
	</ARTICLE><H4> Sup35 </H4><ARTICLE>
	Sup35 is a yeast translation termination factor from <I> Saccharomyces cerevisiae</I>.
	The prion form of Sup35 is known to form amyloids consisting of beta-sheet rich protein
	aggregates with beta-strands perpendicular to the long axis of the filament. The domain 
	responsible for the conformational change is the NM region. This region of the protein 
	contains two different sections. The N‑section (amino acids 1‑124) forms the major part 
	of the amyloid core that that directs the protein into the prion form. The M-section 
	(amino acids 124‑250) is highly charged and provides the solubility to the native form 
	of Sup35. In the prion form the M region changes its conformation to a beta‑sheet rich 
	conformation, while the N‑section stays nearly unchanged in its conformation (Mukhopadhyay <I>et al.</I>, 2007; Wickner <I>et al.</I>, 2015).
	</ARTICLE></DIV></DIV><DIV class="container"><DIV class="contentbox"><DIV class="content"><H2> References </H2><B>Brustad, E. M., Lembke, E. A., Schultz, P. G., Dentz, A. A.</B>(2008). A General and Efficient Method for the Site-Specific Dual-Labeling of Proteins
for Single Molecule Fluorescence Resonance Energy Transfer. American Chemical Society. <B>130</B>: 17664-17665..<B>Kim, J., Seo, M., Lee, S., Cho, K., Yang, A., Woo, K., Kim, H., Park, H.</B>(2012). Simple and Efficient Strategy for Site-Specific Dual Labeling of
Proteins for Single-Molecule Fluorescence Resonance Energy
Transfer Analysis. Analytical Chemistry.<B>85</B>: 1468-1474. <B>Lembke, E. a.</B>(2011). Site-Specific Labeling of Proteins for Single-Molecule
FRET Measurements Using Genetically Encoded
Ketone Functionalities. Bioconjugation Proocols: Strategies and Methods in Molecular Biology. <B>751</B>: 3-15.<B>Mukhopadhyay, S., Krishnan, R., Lembke, E. A., Lindquist, S., Deniz, A. A.</B>(2007)A natively unfolded yeast prion monomer adopts
an ensemble of collapsed and rapidly
fluctuating structures.PNAS.<B>104(8)</B>:2649-2654.<B>Wickner, R. B., Shewmaker, F. P., Bateman, D. A., Edskes, H. K., Gorkovsky, A., Dayani, Y., Bezsonov, E. E.</B>82015) Yeast Prions: Structure, Biology, and Prion-Handling Systems. Microbiology and Molecular Reviews. <B>79(1)</B>:1-17.</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>