Team:Bielefeld-CeBiTec/Results/unnatural base pair/preservation system
Expanding the Genetic Code
Human Practice ▼
Additional ▼
Results ▼
Theory ▼
Bielefeld-CeBiTec 2017
Collaborations
Team ▼
Partners
Notebook ▼
Aim of the Project
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Overview
Unnatural Base Pairs
Uptake and Biosynthesis
Retention System
Challenges
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Results Overview
Unnatural Base Pair ▼
Translational System ▼
Toolbox ▼
Parts ▼
Achievements
Overview
Uptake
Biosynthesis
Retention System
Development of New Methods
Overview
Translation Mechanism
Library and Selection
Overview
Analyzing
Photoswitching
Labeling
Photolysis
Fusing
Parts
Basic Parts
Composite Parts
Part Collection
Improve
Hardware
Software
Modeling
Measurement
Applied Design
InterLab Study
Overview
Gold
Silver
Public Engagement
Members
Advisors
Bielefeld
CeBiTec
Bielefeld University
Attributions
Methods
Oligonucleotides
Labjournal
Labnotes Generator
Safety
Retention and Preservation System
Short Summary
We started by validating the basic requirements for the in vivo experiments with unnatural base pairs. A two-plasmid system for retention of unnatural base pairs was developed and characterized. In our in vivo experiment, we were able to detect unnatural base pairs in the specified target sequence using our adapted Oxford Nanopore Sequencing after 48 hours. Furhter experiments need to be conducted for quantification of unnatural base retention efficiency.
Pretests
Due to the high costs of our unnatural base pair (UBP) we had to carefully plan the experiments involving these bases.      Therefore, we carried out cultivation experiments of Escherichia coli BL21(DE3) in micro well plates, because performing       experiments in a microscale would significantly reduce the cost of the experiments. To cultivate at the ideal growth       conditions for our purposes, we performed pretests. In our ex-periments, we tested for the ideal culture volume, surface size and       shaking frequency.    The E. coli strain BL21(DE3) was transformed with the plasmid pSB3C5. We use psB3C5, which is a low copy Plasmid, just like our Plasmid     containing the UBP to have a comparison between our cultivations.  LBCm25 plates were incubated overnight at 37 °C.     Three colonies were picked and incubated in 150 mL LB media supplemented with LBCm25 overnight at 37 °C.     Multiwell-plates with 12, 24 and 48 wells were inoculated to an OD600 of 0.1 with a total of three biological replicates.     Three different cultivation volumes were used to observe the cell growth in the different well plates with the different volumes.
Table 1: Used well plates and the different cultivation volumes in the particular plate. Three replicates per plate and volume were recorded at an rpm of 350.
12 Well Eppendorf 0030721012
24 Well Eppendorf 0030722019
48 Well Eppendorf 0030723015
1 mL
1 mL
0.5 mL
2 mL
2 mL
0.75 mL
3 mL
3 mL
1 mL
The cultivation was carried out in the VWR – Incubation Microplate Shaker and the OD600 measured via NanoDrop ND-1000 Spectrophotometer.    Three technical replicates were measured for each sample.
Figure 1: A: Average OD600 of the three biological replicates of the cultivation volumes of 1 mL (red), 2 mL (blue) and 3 mL (yellow) in the 12 well plate over the cultivation period. Three technical replicates were measured.               B: Average OD600 of the three biological replicates of the cultivation volumes of 1 mL (red), 2 mL (blue) and 3 mL (yellow) in the 24 well plate over the cultivation period. Three technical replicates were measured.              C: Average OD600 of the three biological replicates of the cultivation volumes of 0.5 mL (red), 0.75 mL (blue) and 1mL (yellow) in the 48 well plate over the cultivation period. Three technical replicates were measured.
Figure 1 shows that cultivations in all multiwell plates are possible.  The lowest OD600 is consistently achieved   by the highest volume with values at 2.27, 2.037 and 1.29 using the 12 well plate, 24 well plate and 48 well plate, respectively.   The highest OD600 is associated with the lowest volume with values of 2.63, 2.42 and 1.89 using the 12 well plate,   24 well plate and 48 well plate, respectively.    Irrespective of volume, the highest OD600 values were reached using the 12 well plate. Specifically,    the OD600 value of 2.27 using 3 mL in the 12 well plate is still higher than the OD600 value of    2.03 using 1 mL in the 24 well plate.    After determining the best plate and volume to perform cultivations with, we investigated the influence of the rpm by    cultivating three biological replicates at 500, 600, and 700 rpm in the 12 well plate in 1 mL cultivation volume.
Figure 2: Growth curves of the three replicates of E.coli BL21 (DE3) in    LBCM25 at 350, 500, 600, and 700 rpm, respectively. The DNA concentration was measured under the optimal condition     of 600 rpm and 1 mL volume.
Cultivation at 600 rpm yields the highest OD600 with a value of about 4.96 after 6.5 h.     If the rpm is further increased, the cells grow badly due to an out of phase movement of the media.     We finally concluded that the perfect cultivation conditions are reached in a 12 well plate with a total volume     of 1 mL and shaking with 600 rpm.   For analysis of the retention and preservation of the unnatural base pair, the isolation of a sufficient amount of    plasmid DNA from the culture is important. In order to evaluate the plasmid concentration during the cultivation,    we cultivated at 600 rpm on a 12 well plate in 1 mL. We inoculated the wells with a starting OD600 of 0.1 and    conducted a plasmid isolation using 1 mL for each measurement. The highest DNA concentration was reached in the exponential   phase at a OD600 of 2.75 with about 2.5 µg/µl plasmid isolated from 1 mL culture.
Cytosine Deaminase codA
Deletion of the codA gene could be useful for our project due to its ability [[to transform isoG and isoC]] into uracile.  In this case the concentration of the UBPs decreases and our plasmid carrying the UBPs could not be replicated properly.  We designed three compatible constructs for the potential deletion of codA following the protocol of Jiang et al.,  2015 using the CRISPR/Cas9 technology. .
At first, we designed  our construct carrying our synthesized single guide RNA (sgRNA), which is essential for the targeting specificity.   Because of the presence of the sgRNA, Cas9 could carry out a sequence-specific, double-stranded break.  In the case of a repair via homology directed repair with our designed repair template, codA gets deleted.   Just like Jian et al., we chose pTarget as our vector. Therefore, we annealed the sgRNA oligos, cloned them into   the pTarget backbone via golden gate assembly and selected positive clones via blue-white screening.
The PCR-construct responsible for the homologous recombination, and therefore the codA deletion, was generated by two PCRs. In a first step, 700 bp upstream and downstream of codA were amplified. These two fragments were then inserted into the backbone using Gibson Assembly. The homologous recombination of the neighboring sequences of codA with the plasmid bound flanks leads to a deletion of this gene.
Figure 3: Design of the flanking sequences upstream and downstream of the codA gene for the potential deletion of codA.
As the CRISPR/Cas9 system plasmid, we conceptualized pCas, also used by Jiang et al., containing the lambda-Red system for a   higher recombineering efficiency. For the codA gene deletion, you need to generate E. coli BL21 DE electrocompetent cells harboring pCas,    and Arabinose (10 mM) has to be added for the induction of the lambda-Red system. For the homology directed repair, one needs    to transform pTarget and the PCR fragment into the competent E. coli cell harboring the pCas plasmid.   Then, one needs to verify positive transformants by colony PCR and DNA sequencing.
To cure the cells from pTarget, one can inoculate a positive colony at 30 °C for 8 to 16 hours in media containing kanamycin  (50 mg L-1) and IPTG (0.5 mM). For the curing of pCas, the cell can be grown overnight at 37 °C non-selectively. The challenging part of this is the biological meaning of codA because of its important role in the purine synthesis pathway.  To counteract this, it is advised to supplement with uracil (Mahan et al., 2004).
Design of a Plasmid for the Retention of Unnatural Base Pairs
Selection and screening of the transformants was not possible with our experimental design since the retention experiment had to be started immediately after transformation. Therefore, we designed a plasmid which would kill or inhibit growth of the wrong colonies. Our initial design focused on the use of ccdB, but given that ccdB cannot be submitted to the parts registry, we decided to use levansucrase of Bacillus subtilis, encoded by sacB instead. Therefore, a mRFP-sacB fusion construct was designed ( BBa_K2201017 ). BBa_J23100 was used as a promoter and BBa_B0034 was used as a strong RBS, followed by BBa_E1010 . Between mRFP and sacB, a linker consisting of four alanines was used. As a backbone, pSB3C5 was used. Given that this plasmid was designed to eventually carry an unnatural base pair, a low-copy plasmid was the better choice for a higher stability of the unnatural base pair.   The idea behind this construct is the following: In the initial plasmid, mRFP and sacB are in-frame, meaning that cells turn red when incubated in standard media such as LB, but die or grow weakly when cultivated in media supplemented with sucrose. This is due to the toxic effect of levansucrase, which unfolds when cells are cultivated in sucrose supplemented media.
Figure 4: Fusionprotein of mRFP and sacB. Shown is the linker between the two sequences. A flexible 4xAla linker was used between mRFP and sacB.
The linker between mRFP and sacB represents the insertion site for the fragment containing the unnatural base pair. This insertion, if successful, leads to a frameshift of the downstream located sacB, leading to loss of function. Therefore, cells can only survive in sucrose supplemented media if the fragment containing the unnatural base pair was successfully incorporated. This provides an easy screening method, since cells that turn red and survive in sucrose supplemented media most likely inserted the fragment containing the unnatural base pair into the plasmid. Having the coding sequences of both proteins in one frame has one big advantage: Given that sacB is prone to loss-of-function mutations, contaminations can be easily distinguished by color. Furthermore, cells with mutations in the promotor region would also survive, since they would not express sacB, but not turn red. Therefore, this two-layered system allows to visually check if a culture either is contaminated, has a mutation in the promotor region, or if the correct plasmid is present. This aspect is visualized in Figure 5.
Figure 5: Method for selection of the correct plasmids in liquid cultures. If the assembly was successful, cells turn red and survive in sucrose supplemented media. The cells die if the assembly was not successful, since sacB can be expressed.
To test the effect of sucrose on growth of the cells, a cultivation was performed of E. coli BL21(DE3) harboring BBa_K2201017 . The cultivations were carried out in a 12 well plate in 1 mL of LB medium supplemented with different concentrations of sucrose. Three biological replicates were cultivated for each condition and three technical replicates taken for each measurement point.
Figure 6: Cultivation of E. coli BL21(DE3) in LB media supplemented with different concentrations of sucrose. Three biological replicates were cultivated and three technical replicates measured for each point in time. A clear difference in growth can be observed, with higher sucrose concentrations leading to weaker growth. The mRFP-SacB fusion protein does not lead to cell death, but shows bacteriostatic properties. Therefore, cells that successfully integrated the fragment containing the unnatural base pair should grow much better in the selective media and therefore overgrow cells harboring religated plasmid backbones.
The cultivations shown in Figure 6 clearly show that sucrose has a significant effect on growth of strains carrying BBa_K2201017 . A higher sucrose concentration leads to weaker growth, with strains cultivated in 20 % sucrose reaching only ~33 % of the OD600 of the reference strain cultivated in sucrose free media. The reference reached a final optical density of 3.233 ± 0.148, strains cultivated in media supplemented with 10 % sucrose 1.527 ± 0.055 and strains cultivated in 20 % sucrose reached a final optical density of 0.44 ± 0.05.    Figure 7 shows a comparison of strains cultivated in media without sucrose and media supplemented with 10 % sucrose. A preculture of E. coli BL21(DE3) harboring BBa_K2201017 was prepared and cultivated overnight at 37 °C. 30 µl of this precultures were used and dropped onto LB agar plates either without or supplemented with 10 % sucrose. 50 µl of the same preculture were used to inoculate 3 mL of LB medium without sucrose and 3 mL of LB media supplemented with 10 % sucrose.
Figure 7: E. coli BL21(DE3) BBa_K2201017 cultivated on plates and in liquid cultures with and without 10 % sucrose.
All cultures were cultivated for 24 hours at 37 °C. A) Shows a drop on a LB agar plate supplemented with 10 % sucrose. B) Shows a drop on a LB agar plate without sucrose. A clear difference can be observed between the two drops. In presence of sucrose, E. coli BL21(DE3) BBa_K2201017 grew much weaker and did not turn red. Without sucrose, the cells grew much better and turned red. C) The same difference in growth and color could be observed in liquid cultures.
Further Pretests
The antibiotics kanamycin and chloramphenicol were not added to the recovery media to achieve better growth conditions for the cells right after the heat shock transformation. We investigated the growth conditions for E. coli BL21(DE3) concerning the concentration of chloramphenicol. A lower concentration of chloramphenicol would decrease the stress of the cells during their cultivation since there is already a huge metabolic stress when using a two-plasmid system. So, we grew the native E. coli BL21(DE3) in liquid LB media supplemented with different chloramphenicol concentrations as a pretest shown in Figure 8.
Figure (8): E. coli BL21(DE3) cultivated in liquid LB media using different Cm concentrations.
Precultures with LB media supplemented with chloramphenicol concentrations c= 0, 1, 2, 3, 4, 5, 10, 15, 20, 25 μg μL-1 were inoculated with native E. coli BL21(DE3). All cultures were cultivated for 24 h at 37 °C. Growth was only visible for cultures with c(Cm)<2 μg μL-1.
This experiment showed that not any native E. coli BL21(DE3) cell without a chloramphenicol resistance gene is able to grow above c(Cm)= 1 μg μL-1. Based on this experiment we chose c(Cm)= 3 μg μL-1 to exclude any growth from E. coli BL21(DE3) during our cultivation for the UBP retention.
Another pretest should investigate the influence of the unnatural nucleotidetriphosphates isoGTP and isoCmTP on the growth the chemically competent cells E. coli BL21(DE3) containing the pSB1K3 high-copy plasmid BBa_K2201027 . Therefore we inoculated these cells in 2xYT supplemented with 50 μg μL-1 kanamycin and different concentrations c= 5, 10, 50, 100 μM of each unnatural nucleotidetriphosphate. The different concentrations of unnatural nucleotidetriphosphates did not show any influence on the growth of the cells.
Figure (9): Growth test of chemically competent cells E. coli BL21(DE3) containing BBa_K2201027 with different concentrations of isoGTP and isoCm.
The plasmid BBa_K2201027 is coding for the nucleotide transporter PtNTT2 which ensures the uptake of isoGTP and isoCm from the surrounding media into the cell. The cells were incubated in 500 μL 2xYT media at 37 °C for 1 h. For incubation at 37 °C and 600 rpm in 12-well plates in the VWR – Incubation Microplate Shaker 2xYT media was added to 1 mL with 50 μg μL-1 kanamycin and c= 5, 10, 50, 100 μM of each isoGTP and isoCmTP final concentration. Three technical replicates of the OD600 were measured every hour via the NanoDrop ND-1000 Spectrophotometer.
The Two-Plasmid System for UBP Retention
The retention system was engineered to preserve the unnatural base pair (UBP) isoG-isoCm in vivo. This intention requires the uptake of unnatural nucleotides from the media and a selection pressure on the plasmids carrying the UBP. We developed a two-plasmid system for the retention. As described above, the first high-copy plasmid (pSB1K3) BBa_K2201027 contains the truncated version of PtNTT2 for the uptake of isoGTP and isoCmTP and cas9 for the digestion of all plasmids that have lost the UBP in vivo . The Cas9 is guided by five sgRNAs ( BBa_K2201077 ) that bind every possible point mutation that leads to the loss of the UBP. Those sgRNAs are part of the second plasmid. This plasmid BBa_K2201032 is a composite part of BBa_K2201077 and BBa_K2201017 . We decided this plasmid to be a low-copy plasmid (pSB3C5), because a high-copy version could possibly cause a greater mutation frequency. The second plasmid also contains a different antibiotic resistance than the first plasmid for a selection of bacteria carrying both plasmids. The UBP was part of the oligo UBP_target and was assembled with the linearized backbone of BBa_K2201032 . Primers ( 17tx and 17we )  for the linearization via PCR were used so that the Gibson Assembly with the oligo UBP_target leads to a frameshift within the sacB coding sequence of the mRFP-sacB fusionprotein. Hence, a successful Gibson Assembly enables the transformed organism to grow in the presence of sucrose later on.
In vivo Experiment for UBP Retention
The Romesberg lab (Zhang et al., 2017) was able to proof retention of their hydrophobic UBP in E. coli BL21(DE3) using the nucleotide transporter PtNTT2 and Cas9. Therefore we transformed BBa_K2201027 into E. coli BL21(DE3) via heat shock with the goal to produce chemically competent cells containing the retention plasmid afterwards. Those chemically competent cells were used for a second heat shock transformation of the second plasmid BBa_K2201032 containing our UBP isoG-isoCm. 10 μL of the above described Gibson Assembly was transformed into chemically competent cells. After the heat shock, the cells were recovered in liquid recovery media (2xYT media supplemented with 50 mM K2HPO4, 0.5 mM IPTG, 100 μM isoCmTP, and 100 μM isoGTP) and shaked at 200 rpm and 37 °C for 1 h. Growth was performed in liquid growth media (2xYT media supplemented with 50 mM K2HPO4, 0.5 mM IPTG, 100 μM isoCmTP, 100 μM isoGTP, 3 μg μL-1 chloramphenicol, and 50 μg μL-1 kanamycin) while shaking at 600 rpm and 37 °C for 48 h. The described experimental set-up is shown in Figure 10.
K2HPO4 is needed in the recovery and growth media as a competitive inhibitor for phosphatases. This prevents the dephosphorylation of unnatural nucleotide triphosphates. The IPTG in the media induces the expression of cas9 that is negatively regulated by our designed lac operon ( lacO_tight1 ) optimized for tight repression. This needs to be added right from the beginning after the transformation to start the retention system during recovery. For the same reason the isoGTP and isoCmTP were added to the recovery and growth media.
Before analyzing the efficiency of the UBP retention, the plasmids need to be isolated from the cultivation samples. But this leads to the isolation of both plasmids, the high-copy (pSB1K3) and low-copy (pSB3C5) plasmid. For the detection of the UBP, which lays on the pSB3C5 plasmid, the pSB1K3 plasmid needs to be digested by restriction enzymes. This can be achieved by the restriction enzymes PvuI, NsiI und BamHI that are solely targeting the pSB1K3 plasmid BBa_K2201027 . Afterwards, the remaining pSB3C5 plasmid can be prepared for the Oxford Nanopore Sequencing method in order to detect the UBP.
Figure (10): Experimental flowchart for UBP retention in vivo.
The pSB3C5 plasmid BBa_K2201032 containing five sgRNAs and mRFP-sacB was linearized with the primers 17tx and 17we and assembled with UBP_target via Gibson Assembly. 10 μL of the Gibson Assembly was transformed into chemically competent E. coli BL21(DE3) cells containing the pSB1K3-plasmid BBa_K2201027 . After the heat shock, the cells were recovered in 850 μL liquid recovery media (2xYT media supplemented with 50 mM K2HPO4, 0.5 mM IPTG, 100 μM isoCmTP, and 100 μM isoGTP) and shaked at 200 rpm and 37 °C in a 12-well plate in the VWR – Incubation Microplate Shaker for 1 h. Then, the recovery media was filled up with liquid growth media (2xYT media added with 50 mM K2HPO4, 0.5 mM IPTG, 100 μM isoCmTP, 100 μM isoGTP, 3 μg μL-1 chloramphenicol, and 50 μg μL-1 kanamycin for final concentrations) up to 1 mL and shaked at 600 rpm and 37 °C in a 12-well plate in the VWR – Incubation Microplate Shaker for 48 h. Plasmid isolations were performed for single 1 mL cultures. The high-copy plasmid pSB1K3 BBa_K2201027 was digested by the restriction enzymes PvuI, NsiI und BamHI. The remaining low-copy plasmid pSB3C5 containing the UBP was then prepared for the Oxford Nanopore Sequencing.
With the help of Oxford Nanopore Sequencing we were able to detect the UBP after the in vivo cultivation. Therefore we assume that the retention system is functional. Further replicates would need to be done for a statistical aproved analysis. This experimental seems to be a very promising set-up for the UBP retention in vivo.
Figure (11): Oxford Nanopore Sequencing of BBa_K2201032 containing the UBP after in vivo cultivation and retention.
The flow cell lays can be seen in the front. On the laptop screen: Green areas on the grid show the quantity of nanopores sequencing a DNA molecule at a moment.
References
Zhang, Y., Lamb, B.M., Feldman, A.W., Zhou, A.X., Lavergne, T., Li, L., and Romesberg, F.E. (2017). A semisynthetic organism engineered for the stable expansion of the genetic alphabet. Proc. Natl. Acad. Sci. 114: 1317–1322.
