<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NEAU-Harbin_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NEAU-Harbin/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NEAU-Harbin/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NEAU-Harbin/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NEAU-Harbin/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NEAU-Harbin/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NEAU-Harbin/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="Layer2"><TABLE width="100%" height="29" border="0" cellpadding="0" cellspacing="0"><TBODY><TR><TD width="44%"><DIV align="center"><STRONG><A href="tema_studenes.html">Students</A></STRONG></DIV></TD><TD width="56%"><DIV align="center"><STRONG><A href="tema_instructors.html">Instructors</A></STRONG></DIV></TD></TR></TBODY></TABLE></DIV><DIV id="Layer3"><TABLE width="100%" height="23" border="0" cellpadding="0" cellspacing="0"><TBODY><TR><TD width="41%"><DIV align="center"><STRONG>Background</STRONG></DIV></TD><TD width="27%"><DIV align="center"><STRONG>AMT</STRONG></DIV></TD><TD width="32%"><DIV align="center"><STRONG>Design</STRONG></DIV></TD></TR></TBODY></TABLE></DIV><DIV id="Layer4"><TABLE width="100%" height="31" border="0" cellpadding="0" cellspacing="0"><TBODY><TR><TD><DIV align="center"><STRONG>Safety</STRONG></DIV></TD><TD><DIV align="center"><STRONG>Meetups</STRONG></DIV></TD><TD><DIV align="center"><STRONG>Visit</STRONG></DIV></TD></TR></TBODY></TABLE></DIV><DIV id="Layer5"><TABLE width="100%" height="23" border="0" cellpadding="0" cellspacing="0"><TBODY><TR><TD><DIV align="center">1</DIV></TD><TD><DIV align="center">2</DIV></TD><TD><DIV align="center">3</DIV></TD></TR></TBODY></TABLE></DIV><DIV id="Layer6"><TABLE width="100%" height="31" border="0" cellpadding="0" cellspacing="0"><TBODY><TR><TD><DIV align="center"><STRONG>background</STRONG></DIV></TD><TD><DIV align="center"><STRONG>design</STRONG></DIV></TD><TD><DIV align="center"><STRONG>fusion protein</STRONG></DIV></TD><TD><DIV align="center"><STRONG>Carotenoids</STRONG></DIV></TD></TR></TBODY></TABLE></DIV><TABLE width="1000" height="33" border="0" align="center" cellpadding="0" cellspacing="1" bgcolor="#E7E7E7"><TBODY><TR><TD bgcolor="#F2F2F2"> </TD></TR></TBODY></TABLE><TABLE width="1000" height="305" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#FFFFFF"><TBODY><TR><TD><P>Week1
      We learnt and planedthe preliminary  experiment, andprepared some of the materials needed during the experiment,  such as LB medium. In this week, we got familiar with the operation of some  basic equipment and our future experiment. In the meantime, we received four  strains of E.coliwhich contain pSZH, pAN7-1, pSZH-G and pSZH with GlaA3, and carried  rejuvenation and preservation out for them. We got the overnight suspension  culture and extracted the plasmid from these four kinds of E.coli.</P><P>Week 2
        We received the parts library from iGEM HQ  , dissolved the plasmids which containing genes of Crtw-rbs, Crtz-rbs, CrtY,  CrtB, CrtI-rbs, CrtB-rbs, CP, cjBlue, and RFP with ddH2O and removed them out  to transform them into DH5α competent  cells.</P><P>Week3
        We attempted to eliminate the Not I restriction  site. We used Not I restriction enzyme to digest the plasmid pSZH (37℃overnight ),  and recovered by using centrifugal column method . The linear DNA recovered by  us was reinforced and extended at 72℃in the PCR  instrument for 5min . We recovered the plasmid again ,but unfortunately , we  failed because of the low percent recovery . The fragment is too large and the  recovery kit we used has a limited scope , which may lead to the low efficiency  of recovery .But the good news is that the transformation of 9 genes last week  was successful and the 9 kinds of plasmids containing the genes were extracted  .Since the pigment metabolism genes only can be used later , firstly, we  digested the 3 kinds of fluorescent protein genes ( CP, cjBlue , RFP ) by Xba I  and Not I (We added the Xba I into the solution firstly to react 2h and then we  added Not I to react another 2h). Agarose Gel Electrophoresis showed that the  digest reaction was complete .</P><P>Week 4
        We get a plasmid containing the HPH and  GlaA and primer (including Linker) of GlaA3, HPH, and ultraviolet excitated  GFP, cjBlue, RFP, BFP, using PrimeStar catalyze PCR, and the product was electrophoresed  in agarose -TAE gel, tapping and recycle the target segments. Unfortunately ,  because of the error of design , we didn't get the right fragment . Then we use  Taq to add “A” to the recycling product in PCR machine (72℃for 1h), and  recycle the product in liquid (spin column method) and mix them with T vector ,  linking at 4℃ overnight . The product was transformed to DH5α competent cell and  vaccinated on Amp+ LB plate culture overnight . In addition , we tried to knockout  the NotI restriction site. We changed the recovery kit, but the effect is still  not good .</P><P>Week 5
        Luckily , strains with the T vector  containing the 5 genes transformed last week grew well . We inoculated the 5  kinds of single colony to liquid Amp+ LB medium, shaking cultured overnight at  37℃ 200rpm and extracted the plasmids of them, which was successful .  We redesigned primers , and amplificated cjBlue by PCR . Unfortunately , we  failed again . We consulted instructors , redesigned primers .</P><P> </P><P>Week 6
        We got primers of cjBlue ,and successfully  amplificated the cjBlue segment . We linked it to T vector and transformed it  to DH5α competent cell. Single colonies were inoculated to liquid LB medium and  plasmids were extracted. This week , we knockouted the NotI restriction site  again by a recycling kit with DB . The strips in agarose-TAE gel electrophoresishad  long tails that is called Comet strip, which indicated that the vector was  degraded . But there was a clear strip in the position , so we tried to operate  the blunt-end ligation by using T4 ligase (We divided the reaction system into  4 parts . Part I had 1μL plasmids.Part II had 2μL plasmids.Part III had 4μL  plasmids.)</P><P>Week 7
        The pSZH plasmid was transformed into DH5α.  Luckily , it grew well on Km+ solid LB medium , and we got many colonies . We  choose 10 colonies , to grow them in liquid Km+ LB medium , and extracted their  plasmid . We were surprised to find that all the plasmids were about 3000bp. We  suspected that the plasmids were degraded but the segment of Km+ resistance  were kept with the losing of other segments. In a word, the knockout of NotI  was failed again . Besides this, we operated double digestion of N-HPH (NcoI  and EcoRI), F-cjBlue (EcoRI and BamHI) , GlaA3 (BglIIand XbaI) , pAN7-1 (NcoI and  BamHI) , CP (ApaI and HindIII) and pSZH-G (ApaI and HindIII) ?in 20 μL systems . Agarose-TAE gel electrophoresis  showed that, the digestion was thorough and effective except CP and pSZH-G. So  we digested N-HPH , cj-Blue, GlaA3 and pAN7-1 in 100μL reaction systems, and  recycled them in agarose-TAE gel. The recycling effect of pAN7-1 is low. We  tried to link pAN7-1, F-cjBlue and N-HPH . We transformed the product into  DH5α, and spreaded on solid Amp+ LB medium .</P><P>Week 8
        Growth of the E.coli transformed with  pAN7-1 vector is not well. We analyzed the reason and find that the recycling  effect of pAN7-1 was very low which makes the connection is difficult. This  week , we digested N-HPH (NcoI and EcoRI), F-cjBlue (EcoRI and BamHI) , GlaA3  (BglIIand XbaI) , pAN7-1 (NcoI and BamHI) , CP (ApaI and HindIII) and pSZH-G (ApaI  and HindIII) again , and recycled them in agarose-TAE gel. Unfortunately , the  recycling effect was not good too. For the knockout of NotI restriction site in  pSZH , we used ethanol precipitation recycling instead of spin column method,  which got a good effect. Then blunt ends of the product recovered by us had  been connected together for 3 days.</P><P>Week 9
        This week we have a temporary break and  analyze errors occurred before.</P><P>Week 10
        We extracted the pSZH plasmid whose blunt  ends have been connected in Week 8 , and used ApaI and NotI to operate double  digestion identification . At the same time, we restreaked rejuvenation all  strains used before , and extracted their plasmid standing by.We digested N-HPH  (NcoI, EcoRI), F-cjBlue (EcoRI and BamHI), ultraviolet excited GFP (EcoRI,  BamHI), GlaA3 (BglII, XbaI), and pAN7-1 (NcoI, BamHI) , each in two 100μL system,  and recycled them in Agarose-TAE gel. Because of the discussion about the reason  of inefficient recycling last week , we improved the operation of gel tapping  and recycling , which makes the efficiency was greatly improved except the  strip of pAN7-1 vector was still dark . We mixed N-HPH, F-cjBlue, pAN7-1 and  N-HPH, GFP, pAN7-1, and added T4 ligase into the reaction system and let them  react for 48h.</P><P>Week 11
        The two ligation product systems in last  week were transformed into DH5α and inoculated on each two Amp+ solid LB  mediums to be cultured overnight . There were two single colonies on the  two? mediums of F-cjBlue , but there was  nothing on the one of GFP. We used primers of cjBlue to operate PCR for the 4  colonies , and the result indicated that they were all positive . Then we  inoculated them into Amp+ liquid LB medium and cultured in 37℃for 12h. Two  bottles of broth are green and there was no cjBlue product in another two  bottles. We extracted plasmids from them but failed in the green broth . We  used EcoRI and BamHI to digest them , showing positive and the size of fragment  is same with cjBlue , which indicated that N-HPH and cjBlue are connected to  pAN7-1 successfully . At the same time , we inoculated pSZH-G into Km+ liquid  LB medium and CP into Amp+ liquid LB medium . Plasmids were extracted , and  digested by ApaI and HindIII in 100μL reaction systems.  We recycled the CP product from Agarose-TAE gel and pSZH-G from liquid.Because  of inefficient extraction of pSZH-G , the concentration of recycling product is  not enough , but we recycled CP successfully whose strip in Agarose-TAE gel electrophoresis  is bright . So we tried to connect them (T4 ligase, 48h) . This week we tried  to knockout NotI again, using alcohol precipitation recycling pSZH digested .  Because of operational problems , we failed again .</P><P>Week 12
        We transformed the pSZH-G and CP into DH5α,  and inoculated on Km+ solid LB medium , which grew well . Then we selected 20  colonies to operate PCR , they were all positive . A bottle of broth was purple  and we suspected that there was T vector mixed in it. Other colonies were  cultured and plasmids were extracted and restriction identification was  negative . So this time was failed again . But we find that chloramphenicol can  improve the copy number of large vector pSZH-G, so we tried to optimize the  bacterial culture solution , i.e. , inoculate noon and add chloramphenicol into  it (final concentration is 60μg / mL), and extract plasmids next day . After  many lessons learned , this method is excellent for the extraction of large  vectors. In addition , we also PCR the crtE, crtB, crtI, crtY, crtZ five genes.  Restriction enzyme digestion showed that the four except crtE all cut out of  the target strip . We recycled them and add Adenine deoxynucleotide to its end for  the connection with T vector and transformed into DH5α . However , crtE had 3  strips which was dark . So we operated PCR again , and repeated the operation  again . At the same time , this week we made 8 tubes of 100μL reaction systems  for finding reason of the failure of NotI knockout . Four of them were recycled  by using spin column method and another four used alcohol precipitation  recycling method . We compared with the two methods , result indicated that alcohol  precipitation recycling method had a higher efficiency than another one under  the circumstance that all operation are right . We used PCR instrument to fill  the cohesive ends of recycling product (72 ℃, 5min), and  recovered by using ethanol precipitation method .</P><P>Week 13
        Plasmids of pSZH-G , CP and RFP were  extracted and digested again ( RFP : ApaI and ClaI ) . We found that pSZH-G and  CP were digested thoroughly , but there were not target strip shown in electrophoresis  . We analyzed the reason and found that methylation makes ClaI can not digest  its restriction sites. So we transformed RFP(in the T vector) into Trans110  competent cell, and got the strain containing RFP genes that was not  methylated. We extracted plasmids from it and digested , which was more  efficient . CP and RFP were recycled from Agarose-TAE gel and pSZH-G was  recycled from liquid. Unfortunately , only CP got a nice recycling. The failure  of RFP was because of operation errors, and pSZH-G was because of its large  size which can cause a huge loss in spin columns recycling method. We recycled  the pSZH whose cohesive ends were filled last week , and failed .</P><P>Week 14
        We inoculated RFP strains and extracted the  plasmid which was digested and recycled . We got a good efficiency in recycling  this time . At the same time , we digested and recycled pSZH-G , and increased  the size of reaction system , which makes the plasmid recycled more than before  . We operated ligation (T4 , 48h), and transformed product into DH5α competent  cell which was cultured on solid Km+ LB medium (37℃) . For the  knockout of NotI in pSZH-xynB , we designed an adapter which is 14bp to connect  cohesive ends and avoid the embarrassment of low recycling efficiency . We  mixed the two parts of adapter and connected them under 70℃in PCR  instrument and cooled down at room temperature. Finally , product was cooled  down in ice-bath for 5 min and took 3μL and 5μL for connection at 16℃for 2 days. This  week we digested pSZH-G(ApaI, HindIII) . Then we connected CP and pSZH-G and  transformed into DH5α which was cultured on Km+ solid LB medium. We got 22  colonies totally , and selected 15 colonies randomly to operate XbaI and XhoI  digestion identification . The strips cut down was around 4500bp, which had a  large difference with 3000bp which is the theoretical value. </P><P>Week 15
        We obtained the correct pSZH-G plasmid and  transformed it into DH5α competent cells to coating on Km+ solid LB medium .  Single colonies were inoculatedto liquid Km+ LB medium and plasmid from them  were extracted . Digested them as usual and divided them into two 100μL reaction  systems . One was recycled by using glass milk method and another one used  opinion column method . The result indicated that , glass milk method has a  higher efficiency in the recycling of large vector . We connected product with  CP recycled before (pSZH-G , 5μL. CP , 3μL. T4 ligase) for 48h , and  transformed into DH5α competent cells and coat on Km+ solid LB medium to  culture overnight and got 14 colonies . We operated colony PCR , only 1 colony  was positive and we suspected that electrophoresis roadsof other colonies were  jammed . We cultured the positive colony to extract its plasmid and operated  digesting identification (XbaI, HindIII) . Six colonies were suspected positive  . We chose No.1and No.13 whose target strips were bright to culture in Km+  liquid LB medium and extracted plasmids . We finished the improvement of pAN7-1  and connected it to pSZh-xynB digested by XhoI and NheI . Unfortunately , there  were many restriction sites on it, and cohesive ends formed by NheI and XbaI  can be linked together , which can delete the restriction site . So we cannot  use them to identify the plasmid. We PCRed plasmid and digested it by KpnI for  identification , which indicated that the plasmid was positive.</P><P>Week 16
        We digested the plasmid extracted last week  by XbaI and XhoI, there many strips in Agarose-TAE gel electrophoresis. We  asked many people in the lab , and finally found that XbaI was polluted by  other restriction enzyme . After many tests, we produced Agrobacterium  competent cells to transform No.13 and No.1 plasmids into them and coated the  mixture on six Rif+ solid YEB mediums(30℃).</P><P>Week 17
        Agrobacterium on six YEB plates grew well.  We PCRed their colonies (CPs, CPas) and they were all positive , which means  plasmids are all transformed into them . So we inoculated one colony into  liquid Rif+ YEB medium, shaking cultured and took some broth to mix with  Aspergilus Niger and co-cultured 3 days on solid PDA mediums(added AS) which  covered by glass papers .</P><P>Week 18
        We moved the glass papers covering on PDA  plates which were used in the co-culturing to PDA plates containing cefotaxime  and hygromycin and cultured 1 day. We removed glass papers from PDA plates and  put them in 30 ℃ incubator for culturing.</P><P> </P></TD></TR></TBODY></TABLE><TABLE width="1000" height="44" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#FFFFFF"><TBODY><TR><TD><DIV align="center">CopyRight © 2014</DIV></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:NEAU-Harbin/Notebook">http://2014.igem.org/Team:NEAU-Harbin/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NEAU-Harbin/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NEAU-Harbin/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NEAU-Harbin/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NEAU-Harbin/Notebook&amp;oldid=137472" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>