"
Page
Discussion
View source
History
teams
Log in
 
Team:NEAU-Harbin/result.html
From 2014.igem.org
Sponsor
RESULTS
Results
1.Expression vector construction
To prove that homologous recombination occurred, we constructed pSZHG-amilCP and pSZH-cjBlue first (Figure 1) and did the enzyme digestion identification (Figure 2) .Then we transformed the plasmids pSZHG-amilCP and pSZH-cjBlue into A. niger by agrobacterium-mediated transformation. We extracted genome of A. niger and did PCR identification (Figure 3 and Figure 4 ). This figure clearly shows that the amilCP gene has replaced the GLA gene and cjBlue has been ligated with the HPH.
Figure1   Plasmids pSZHG-amilCP and pSZH-cjBlue. We constructed them to measure the efficiency of the chromoproteins expediently.
Now we have already finished the construction of the plasmids pSZHG-amilCP and pSZH-cjBlue. To confirm that chromoprotein genes have already been ligated into the target sites we did the enzyme digestion identification.
Figure 2 The results of enzyme digestion identification.
The plasmid pSZHG-amilCP was digested by XhoI and XbaI, pSZH-cjBlue were digested by KpnI because there are two KpnI sites in both sides of cjBlue.The result shows that the pSZHG-amilCP and pSZH-cjBlue’s electrophoresis strips are larger than that of original pSZHG.
2.Transformation of A. niger
To confirm that the chromoprotein genes are indeed imported into the genome,we did twice PCR identification.
(1) After the transformation into the A. niger, we extracted genome DNA of the rencombinants and did the PCR identification to see whether the T-DNA fragments have been imported in the genome.The primers’ sequence is partly identical to that of the two homologous arms. In order to see the expression level of the two chromoprotein genes,we transformed pSZHG-amilCP and pSZH-cjBlue respectively.
Figure 3   genome PCR identification of amilCP and cjBlue
Figure-4   The schematic diagram of transformants’PCR identification
The figure-3 clearly shows that pSZHG-amilCP and pSZH-cjBlue has been imported in the A. niger.
(2) In order to confirm the T-DNA fragment have been integrated into the GlaA site of the genome,we did the PCR which could pick out the transformants that have experienced the homologous recombination.
Figure-5   The schematic diagram of homologous recombination’s PCR identification
The target stripe of pSZHG-amilCP is 1700bp and the The target stripe of pSZHG-cjBlue is 2400bp.If the vectors haven’t been integrated into the GlaA site,we won’t get the target stripe.From this graph we can draw the conclusion that the homologous recombination has indeed occurred.
Figure-6   homologous recombination’s PCR identification of amilCP and cjBlue. M( marker);1( Negative Control,water);2( Negative Control,original strain);3(PCR result of amilCP );4(PCR result of cjBlue).
Figure-7   homologous recombination’s PCR identification of amilCP and cjBlue in contrast with water and original strain.1(H2O) 2(Original strain) 3(pSZH) 4-10 (amilCP) 11(pSZH-cjBlue) 12-20 (cjBlue)
3.Expression of chromoproteins in A. niger
We transformed the amilCP and cjBlue respectively into the A. niger and observe the expression level using the confocal microscopy.
Figure-8 The hypha of pSZHG-cjBlue showed under the visual light (confocal microscopy)
Figure-9 The hypha of pSZHG-amilCP showed under the visual light (confocal microscopy)
From figure 8 and figure 9 we can draw the conclusion that the amilCP and cjBlue have the low expression in A. niger,but the cyan and blue color can still be screened.
4. Expression of eGFP in A. niger
Beside 2 kinds of chromoprotein, we also constructed pSZHG-eGFP and transform the plasmid into the A. niger.
Figure-10 The vector using eGFP as the seletive marker
GFP is a kind of fluorescence protein which can emit green fluorescence in E.coli cells. To facilitate GFP’s expression in eukaryotic systems,we designed the GFP’s sequence to get maximum expression. As was expected, it works very well in A. niger . It is in stark contrast with origin strains.
Figure 11     The hypha showed under the visual light and UV (fluorescence microscope)
Figure 12 The hypha showed under the visual light and UV (confocal microscopy)
The figure 11 and figure 12 declared that eGFP indeed works in the A. niger. So the codon optimization is an effective method to improve the expression level.
5.Codon optimization of amilCP and cjBlue
We designed and synthesized the gene amilCP and cjBlue’s sequences of which codons have been optimized to suit for A. niger(http://www.jcat.de/),so that we can improve the expression level. But this work is still in progressing.
Figure 13  The CAI-Value map of amilCP.The left figure is the sequence after adaptation;the right figure is the sequence before adaption.The CAI-Value of amilCP before codon opimization is 0.265,while the one after codon opimization is 0.987.
Figure 14  The CAI-Value of cjBlue.The left figure is the sequence after adaptation;the right figure is the sequence before adaption.The CAI-Value of cjBlue before codon opimization is 0.202,while the one after codon opimization is 0.985.
Retrieved from " http://2014.igem.org/Team:NEAU-Harbin/result.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
