<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NAU-CHINA_Improve skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NAU-CHINA/Improve</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Template:2018_NAU-CHINA</TITLE></P><TITLE>header</TITLE><DIV class="menu"><UL><LI><A href="#">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Team_members">Team members</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Attributions">Attributions</A></LI></UL><LI><A href="#">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Application_prospects">Application Prospects</A></LI><LI><A href="#">RESULTS</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Basic_Part">Basic Part</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Improve">Improved Parts</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Part_Collection">Part Collection</A></LI><LI><A href="#">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Model Details">Model Illustration</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Auxiliary Understanding">Auxiliary Understanding</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Software">iGEMCloud</A></LI><LI><A href="#">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/IntegratedHP">Integrated HP</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Safety">SAFETY</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Notebook">NOTEBOOK</A></LI></DIV><TITLE>Improved Parts</TITLE><DIV class="topLine"><P class="top-title"> PARTS</P><P class="sec-title"> Improved parts</P></DIV><DIV class="main-content"><DIV class="textblock"><H1>Our Improved Part</H1><DIV class="section"><H2><A href="http://parts.igem.org/Part:BBa_K2557001">BBa_K2557001</A></H2><P>The <A href="https://2017.igem.org/Team:Peking">Peking iGEM 2017</A> team identified the properties of Bxb1 gp35<A href="http://parts.igem.org/Part:BBa_K2243012">(BBa_K2243012)</A>in E. coli. We obtained the sequence of this part and requested the plasmid from them. Since our chassis organism is mammalian cell, we optimized the codon and then we identified BBa_K2557001 in detail in HEK 293T cells. Although Bxb1 works well in HEK 293T cells, we did not repeat the activity reported by Peking iGEM 2017 team in E. coli.</P></DIV><DIV class="section"><H2>Characterization</H2><FIGURE><FIGCAPTION class="_table">Fig.2 Function verification of reversal efficiency and threshold characteristics of different recombinase in HEK 293 T Cells.(A)Fluorescence microscope observation of HEK 293T undergone different experimental treatments.(B)The statistical chart of average fluorescence intensity of cells shows that the cells with Bxb1 recombinase have a higher fluorescence intensity than those with TP901 recombinase under the same promoter strength and recombinase concentration. However, if the concentration of recombinase is low, there is no significant difference in fluorescence intensity.(C)The statistics of the proportion of fluorescent cells show that the proportion of fluorescent cells has a sudden jump discontinuity between low concentration and high concentration of Bxb1 and TP901 recombinases. Similar results were obtained in all three repetitions.</FIGCAPTION></FIGURE><P>The results of image B show that the reverse efficiency of Bxb1 recombinase is higher than TP901 recombinase under the same promoter strength and recombinase concentration. However, if the concentration of recombinase is low, there is no significant difference in fluorescence intensity. The results of the image C show that Bxb1 and TP901 recombinases have a threshold property. So, the proportion of fluorescent cells have a jump discontinuity between low concentration and high concentration of recombinase.</P><H2>Characterization in original part</H2><P>The Peking iGEM 2017 team identified the properties of Bxb1 gp35(BBa_K2243012)in E. coli. We obtained the sequence of this part and requested the plasmid from them. Since our chassis organism is mammalian cell, we optimized the codon and then we identified BBa_K2557001 in detail in HEK 293T cells. Although Bxb1 works well in HEK 293T cells, we did not repeat the activity reported by Peking iGEM 2017 team in E. coli.</P><P><B>Pronuclear verification of recombinase</B></P><P>Two plasmids with different resistances and origins of replication were used for function verification of the recombinase. One of them is a reporter gene plasmid, which uses the constitutive promoter J23119. The recombination site is located on both sides of the promoter: one side is sfGFP, and the other is mRFP. The other plasmid is a recombinase expression plasmid using PBAD, an inducible promoter, which is induced by arabinose. When the two plasmids were co-transfected into E. coli, the reporter plasmid expressed sfGFP, a kind of green fluorescent protein; when the inducer arabinose was added, the recombinant enzyme was expressed, the promoter was inverted, and the mRFP , a kind of red fluorescent protein, was expressed.</P><P>Due to the use of two different resistant plasmids, kana and chloramphenicol, we used a plate containing two resistances of kana and chloramphenicol for screening, grew more colonies, and randomly selected 9 singles. After the colonies, we made colony PCR (Fig. 1) and the results showed that both plasmids were transferred.</P><FIGURE><FIGCAPTION class="_table">Fig. 3 A total of 9 single colonies were verified. Lane 1-9, recombinase expression plasmid validation; line 10, DL2000 DNA Marker; line 11-19, reporter gene expression plasmid validation;line 20, DL2000 DNA Marker.</FIGCAPTION></FIGURE><P>The verified E. coli was separately placed in a 1.5 ml centrifuge tube containing antibiotic-containing LB medium, and the culture was grown at 37° C and 200 RPM for 6 hours, and then the culture was aliquoted into two portions, one of which was added with an inducer (10 mM Arabinose). Two cultures were grown for 12 hours at 37°C and 200 RPM, and the mixture was incubated for 1 hour at room temperature prior to testing. Both lasers are used to excite both sfGFP and mRFP.</P><FIGURE><FIGCAPTION class="_table">Fig. 4 Two repetitions were selected and the results showed no obvious green fluorescence.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION class="_table">Fig. 5 Two replicates were selected after addition of the inducer and the results showed no obvious red fluorescence.</FIGCAPTION></FIGURE><P>The result shows no obvious fluorescence. We changed some conditions, such as lowering the temperature, adjusting the rotation speed, adjusting the time, etc. But we still did not get the expected results. We consulted the teacher and the teacher replied that there might be weak fluorescence but our instrument couldn't detect it.</P></DIV><H2>Other Improved parts</H2><P>We also improved other parts in our genetic circuit, such as BBa_K2557000 and BBa_K2557007.</P><P>BBa_K2557000 is improved from BBa_K2446055, which is the synNotch coding part designed by Fudan iGEM 2017 . BBa_K2557007 is the coding sequence of TP901 recombinase, designed by Peking iGEM 2017(BBa_K2243000).</P><P>This year's Improvement parts standard stipulates that comparative experiments must be conducted, but the two parts, BBa_K2557000 and BBa_K2557007, we did not characterize their original part, so we did not use them as an improvement part.</P><P><A href="http://parts.igem.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2018&amp;group=NAU-CHINA">Click here to view Part List</A></P></DIV></DIV></DIV><TITLE>footer</TITLE><DIV id="footer"><DIV class="footer-col" id="Acknowledgements"><SPAN class="footer-title">Acknowledgements</SPAN><DIV class="footer-row"><P>Nanjing Agricultural University</P></DIV><DIV class="footer-row"><P>College of Life Sciences of Nanjing Agricultural University</P></DIV><DIV class="footer-row"><P>Nanjing University</P></DIV><DIV class="footer-row"><P>Chinese Academy of Sciences</P></DIV></DIV><DIV class="footer-col" id="Useful_Links"><SPAN class="footer-title">Useful Links</SPAN><P><A href="https://2018.igem.org/">Main Page</A></P><P><A href="https://igem.org/Special:SpecialPages">Special Pages</A></P></DIV><DIV class="footer-col" id="ContactUs"><SPAN class="footer-title">Contact Us</SPAN><DIV class="footer-row"><P>Facebook</P></DIV><DIV class="footer-row"><P>Twitter</P></DIV></DIV><DIV class="footer-col" id="About Us"><SPAN class="footer-title">About Us</SPAN><P><SPAN style="font-size:1.1vw">NAU-CHINA 2018</SPAN> from 
                Nanjing Agricultural University 
                NO.1 Weigang, Xuanwu District
                Nanjing, CHINA
                210095
            </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>