<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NAU-CHINA_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NAU-CHINA/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Template:2018_NAU-CHINA</TITLE></P><TITLE>header</TITLE><DIV class="menu"><UL><LI><A href="#">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Team_members">Team members</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Attributions">Attributions</A></LI></UL><LI><A href="#">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Application_prospects">Application Prospects</A></LI><LI><A href="#">RESULTS</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Basic_Part">Basic Part</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Improve">Improved Parts</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Part_Collection">Part Collection</A></LI><LI><A href="#">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Model Details">Model Illustration</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Auxiliary Understanding">Auxiliary Understanding</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Software">iGEMCloud</A></LI><LI><A href="#">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/IntegratedHP">Integrated HP</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Safety">SAFETY</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Notebook">NOTEBOOK</A></LI></DIV><TITLE>InterLab</TITLE><DIV class="topLine"><P class="top-title">PROJECT</P><P class="sec-title">Design</P></DIV><DIV class="main-content"><DIV class="textblock"><H1>Introduction</H1><P>MOSFET (metal-oxide-semiconductor field-effect transistor) is an essential component in both analog and digital circuits, serving as analog switches or micro-processors. Inspired by this idea, we built genetic circuit &quot;MOSFETs&quot; in animal T cells, which is Monitoring and Operating System Founded on Engineered T cells. We hope our system can serve as a sensitive bioswitch to real-time monitor the extracellular concentration of a certain antigen, and determine whether to activate the expression of a downstream protein according to the preset threshold. As we expect, it should make no response to low concentration, but have quite high sensitivity near the threshold. In order to achieve our goal, we introduced synNotch, TetR-TetO, recombinase and Recombination Directionality Factors (RDF)in our system.Our MOSFET can be divided into 4 devices, namely, signal detection, signal processing, signal output and system reset.</P></DIV><DIV class="textblock"><H1>Signal Detection</H1><P><B>SynNotch, an engineered transmembrane receptor, bridges intra- and extra-cellular information.</B></P><P>Synthetic Notch (SynNotch)[1]consists of three parts: the synthetic extracellular recognition domain (SynECD, e.g.scFv), the core transmembrane domain of wild Notch receptor[2], and the synthetic intracellular transcriptional domain (SynICyi5yD, e.g.SynTF). When the SynECD binds to its target surface antigen, induced cleavages take place on the core transmembrane domain of SynNotch, releasing the SynICD. The SynICD would be transported into the nucleus and activate the transcription of its corresponding gene.</P><P><B>SynNotch is an ideal platform for customized antigen sensing behavior.</B></P><P>SynNotch provides us an exciting platform because its SynECD and SynICD are both customizable. SynECD can be designed based on currently available scFvs for different tumors .SynICD will trigger customized output after SynECD recognition.</P><FIGURE><FIGCAPTION><DIV>Fig.1  The logic of SynNotch </DIV>
                    The triggering of SynNotch pathway has 4 processes: antigen binding, cleavage, translocating and promoting transcription. SynNotch is a customizable platform for cell sensing and response.
                </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Fig.2  Here, we replace the extracellular domain with anti-EGFP, which can recognize cell surface-expressed EGFP, and the incellular domain with TEV protease, which can cleave specific-designed TetR and activate the expression of recombinase.</FIGCAPTION></FIGURE></DIV><DIV class="textblock"><H1>Signal Processing</H1><DIV class="section"><P>We want to convert the extracellular analog signal into an intracellular digital signal, using Tet-Off/Tet-On device. This part contains tetO(Tetracycline operator), TEV protease (Tobacco Etch Virus nuclear-inclusion-a endopeptidase) and TetR(Tetracycline repressor protein).</P><P>
                    TetO is a operon which can be repressed by TetR.
                    TEV protease (Tobacco Etch Virus nuclear-inclusion-a endopeptidase), a highly sequence-specific cysteine protease from Tobacco Etch Virus (TEV) [4], is currently widely used in synthetic biology.
                    TetR is a repressor from E. coli that blocks downstream expression when it binds to the TetO [3].In our project, we use specifically-designed TetR with TEV protease cleavage site which has been reformed by 2017 Oxford University iGEM project.
                </P><P>To our knowledge, the Tet control system is the perfect combination of prokaryotic and eukaryotic gene expression regulation systems.</P><FIGURE><FIGCAPTION>Fig.3  TetR binds to the DNA operon and inhibits the production of export proteins. However, TetR has a cleavage site for Tobacco Etch Virus (TEV) protease. When it is cut by TEV, the suppression will be alleviated and the downstream gene could express.</FIGCAPTION></FIGURE><P>Initially，We thought that we can implement the filtering function using these devices. But, when we verificated system by the mathematical model, we found that there was a small amount of continuous tetO-downstream promoter leakage, when there was no extracellular signal stimulation. The leakage was 1% of the normal expression (fig.4), but this still need to be improved. To solve the problem, we introduced a recombinase into the system.</P><FIGURE><FIGCAPTION><DIV>Fig.4  Expression of GFP in different TEV concentration levelswithout using recombinase</DIV> 
                        (As for each chart in this part, we choose the quantity of GFP as the vertical axis, and 0.01 hour as the unit of the horizontal axis. Different subplots correspond to different levels of intracellular protein concentration). It can be seen that under low signal levels, the effectors have inevitable leakage, which is not allowed by us, so the rec system have tobe used to ensure the rigor of the system.</FIGCAPTION></FIGURE><P>Recombinases , especially a subset called serine integrases and excisionases[5], are enzymes that can flip specific fragments of DNA. Recombinase can directionally catalyze sensitive DNA exchange reactions between targeted short (30–40 nucleotides) sequence sites that are specific to each recombinase[6]. They have been proved to be able to stably modify DNA sequences, which is the biological basis of our MOSFET construct.   </P><FIGURE><FIGCAPTION><DIV>Fig. 5 Large serine integrases reliably and irreversibly flip unique fragments of DNA</DIV> 
                        DNA cleavage and re-ligation occur at the central crossover region at a pair of recombinase recognition sites (attB and attP), which allows the sequence to be flipped between recognition sites. After recombination, the original attB and attP sequences become reconstructed sequences attL and attR. The resulting attL and attR sequences cannot be recognized and bound by integrases alone, so the state after integration is stable.</FIGCAPTION></FIGURE><H1>Signal output</H1><P>After mathematic model verification, it was found that the system achieved equal outputs with different thresholds and jump by using different promoters and recombinases. These features mean that it can function as quick-response switches. The Model team fed back to the Wet team, hoping to make more diverse switches by combining different promoters and recombinases.</P><FIGURE><FIGCAPTION><DIV>Fig. 6 We re-selected the tetO-downstream promoters</DIV>
                        UbC, EF1α, and miniCMV were selected and matched with different recombinases, including TP901, Bxb1 and PhiC31, by adjusting the parameters of each part and the strength of the promoter. Our whole system has formed nine threshold choices.</FIGCAPTION></FIGURE><P>Theoretically, by selecting different promoters and recombinases, we can actually control whether the recombinase can successfully complete the reverse of its downstream sequence under a certain antigen concentration. In other words, we can preset the threshold antigen concentration according to our practical applications.</P></DIV></DIV><DIV class="textblock"><H1>System reset</H1><DIV class="section"><P>To realize a resettable and more accurate 0/1 switch, we introduce recombination directionality factor (RDF) to our system. When the external signal disappears or falls below the threshold, our MOSFET is able to reset to restore the initial state.</P><P>In the signal processing part of our system, we have mentioned recombinases, which promote recombination between two different DNA sites, attP and attB, and in turn form recombinant attL and attR sites. The ‘reverse’ reaction requires another phage-encoded protein called RDF. RDF activates attL×attR recombination and inhibits attP×attB recombination. Recombinases can be fused to their corresponding RDFs to create single proteins that catalyse efficient attL×attR recombination in vivo and in vitro[7].</P><FIGURE><FIGCAPTION><DIV>Fig. 7 RDF identifies the attL×attR site and flips it back to the original attP×attB state, which closes the device</DIV></FIGCAPTION></FIGURE><P>Immediately, we started a new round of mathematical model verification. When the recombinase and RDF existed together, our system would be unstable. While the recombinase brought the system to the “0” state, the RDF would interfere with its behavior and make it into “1” state. (Fig.8). In order to enhance the stability of the system, we introduced the RDF-inhibitor part. RDF-inhibitor can interact with RDF and make it lose its flipping ability.In our scenario, RDF inhibitors can be siRNAs[8].</P><FIGURE><FIGCAPTION><DIV>Fig. 8 Switching of downstream genes without RDF-inhibitor noise reduction</DIV>
                        X axis represents whether the downstream gene is active. We can see that switches of downstream gene are controlled by signal concentration. Only when concentration level is more than 5, the switch will be on. However, because the expression of RDF is not restricted, it will reset the downstream genes directly, which will close the switch. Then the gene will be open by the effect of rec again, which will cause periodical shock in the system. Because of the shock of effectors’ gene, the expression of GFP is discontinuous. When the system needs to be turned on, oscillations may occur, which may damage the DNA.
                    </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Fig.9  T-cells constitutively express tetR and the promoter is inhibited. When the cells sense the signal in the environment, TEV will bind to tetR, so the recombinase and the inhibitor could be expressed. </FIGCAPTION></FIGURE><P>①When the TEV protease concentration reaches the threshold, the inhibition of the downstream gene by the TetR could be released. Then, the recombinase recognizes the attB and attP sites, reverses the sequence between attB and attP, and generates new attL and attR sites. The recombinase-RDF and RFP protein can be expressed after the reversal. But the recombinase-RDF loses function by binding with the inhibitor produced upstream. The RFP concentration continues to rise.</P><P>②When the external signal weakens and the intracellular TEV protein concentration falls below the threshold, the TetR re-block the expression of recombinase and the inhibitor. Then recombinase-RDF recognizes the attL and attR sites and shuts down the expression of RFP protein. </P><P>Based on the above design , we call it the resettable exact &quot;0/1&quot; switch</P></DIV></DIV><DIV class="textblock"><H1>Reference</H1><P>[1] Morsut, L. et al. Engineering Customized Cell Sensing and Response Behaviors Using Synthetic Notch Receptors. Cell164, 780–791 (2016).</P><P>[2] Bray, S. J. Notch signalling in context. Nat. Rev. Mol. Cell Biol.17, 722–735 (2016).</P><P>[3] Ramos, J. L. et al. The TetR Family of Transcriptional Repressors The TetR Family of Transcriptional Repressors. Microbiol. Mol. Biol. Rev.69, 326–356 (2005).</P><P>[4] Phan, J. et al. Structural basis for the substrate specificity of tobacco etch virus protease. J. Biol. Chem.277, 50564–50572 (2002).</P><P>[5] Stark WM. 2014. The serine recombinases. Microbiol Spectrum 2(6):MDNA3-0046-2014.</P><P>[6] Nern, A., Pfeiffer, B. D., Svoboda, K. &amp; Rubin, G. M. Multiple new site-specific recombinases for use in manipulating animal genomes. Proc. Natl. Acad. Sci. 108, 14198–14203 (2011).</P><P>[7] Olorunniji, F. J. et al. Control of serine integrase recombination directionality by fusion with the directionality factor. Nucleic Acids Res. 45, 8635–8645 (2017).</P><P>[8] Joshi, B. H. &amp;Pachchigar, K. P. SiRNA: Novel therapeutics from functional genomics. Biotechnol. Genet. Eng. Rev. 30, 1–30 (2014).</P></DIV></DIV><TITLE>footer</TITLE><DIV id="footer"><DIV class="footer-col" id="Acknowledgements"><SPAN class="footer-title">Acknowledgements</SPAN><DIV class="footer-row"><P>Nanjing Agricultural University</P></DIV><DIV class="footer-row"><P>College of Life Sciences of Nanjing Agricultural University</P></DIV><DIV class="footer-row"><P>Nanjing University</P></DIV><DIV class="footer-row"><P>Chinese Academy of Sciences</P></DIV></DIV><DIV class="footer-col" id="Useful_Links"><SPAN class="footer-title">Useful Links</SPAN><P><A href="https://2018.igem.org/">Main Page</A></P><P><A href="https://igem.org/Special:SpecialPages">Special Pages</A></P></DIV><DIV class="footer-col" id="ContactUs"><SPAN class="footer-title">Contact Us</SPAN><DIV class="footer-row"><P>Facebook</P></DIV><DIV class="footer-row"><P>Twitter</P></DIV></DIV><DIV class="footer-col" id="About Us"><SPAN class="footer-title">About Us</SPAN><P><SPAN style="font-size:1.1vw">NAU-CHINA 2018</SPAN> from 
                Nanjing Agricultural University 
                NO.1 Weigang, Xuanwu District
                Nanjing, CHINA
                210095
            </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>