<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NAU-CHINA_Model_Details skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NAU-CHINA/Model Details</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Template:2018_NAU-CHINA</TITLE></P><TITLE>header</TITLE><DIV class="menu"><UL><LI><A href="#">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Team_members">Team members</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Attributions">Attributions</A></LI></UL><LI><A href="#">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Application_prospects">Application Prospects</A></LI><LI><A href="#">RESULTS</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Basic_Part">Basic Part</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Improve">Improved Parts</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Part_Collection">Part Collection</A></LI><LI><A href="#">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Model Details">Model Illustration</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Auxiliary Understanding">Auxiliary Understanding</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Software">iGEMCloud</A></LI><LI><A href="#">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/IntegratedHP">Integrated HP</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Safety">SAFETY</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Notebook">NOTEBOOK</A></LI></DIV><TITLE>Safety</TITLE><DIV class="topLine"><P class="top-title">Model</P><P class="sec-title">Model Illustration</P></DIV><DIV class="main-content"><DIV class="textblock"><P><A href="#Part1 Extracellular signal converting into intracellular signal">Part1 Extracellular signal converting into intracellular signal</A></P><P><A href="#Part2 Internal signal regulating recombinase production">Part2 Internal signal regulating recombinase production</A></P><P><A href="#Part3 Recombinase turns on downstream effect expression">Part3 Recombinase turns on downstream effect expression</A></P><P><A href="#Part4 Significance of the cell culture environment">Part4 Significance of the cell culture environment</A></P><P><A href="#Part5 Recombinase filtering and de-shocking">Part5 Recombinase filtering and de-shocking</A></P><P><A href="#Part6: Robustness of the model">Part6 Robustness of the model</A></P><H1 id="Part1 Extracellular signal converting into intracellular signal">Part1 Extracellular signal converting into intracellular signal</H1><P>Tips: Skip part1 if the gene circuit is directly regulated by intracellular signal.</P><P>SynNotch can convert an extracellular signal  into an intracellular signal . Here, we take synNotch-TEV as an example. We use the number of TEV as the intracellular signal after conversion. TEV is an enzyme which is able to release the repressed promoter.</P><P>In this section, 4 reactions are considered:</P><P>$$G_{synNotch-TEV}\xrightarrow[\quad\quad]{C11}{synNotch-TEV}\ \ \ \ \ \ \ \ \ \ \ (1.1)$$ </P><P>$$synNotch-TEV+Ts\xrightarrow[\quad\quad]{C12}U+TEV\ \ \ \ \ \ \ \ \ \ \ (1.2)$$</P><P>$$synNotch-TEV\xrightarrow[\quad\quad]{C13}\emptyset\ \ \ \ \ \ \ \ \ \ \ (1.3)$$</P><P>$$TEV\xrightarrow[\quad\quad]{C14}\emptyset\ \ \ \ \ \ \ \ \ \ \ (1.4)$$</P><P>The specific contents are as follows:</P><P>$$G_{synNotch-TEV}\xrightarrow[\quad\quad]{C11}{synNotch-TEV}\ \ \ \ \ \ \ \ \ \ \ (1.1)$$</P><P>(1.1) represents the expression of synNotch which contains the TEV. Although the actual expression progress of the protein involves many chemical reactions from transcription to modification, these intermediate reactions satisfy the following conditions:</P><P>a)The factors affecting these reactions in the system are stable, such as the amount of amino acids, nucleotides, and helicases;</P><P>b)The gene is regulated by a constitutive promoter and its expression is not regulated by other products of the system;</P><P>c)The product of an intermediate reaction has no effects on the system functions.</P><P>
                We call these three conditions the post-integration conditions [1], and call these reactions the post-integration reaction.
                In the same way, the expression and degradation of other proteins involved in our model all suit the conditions of the refinement, so they can be considered as one single chemical reaction.
            </P><P>Obviously (1.1) is a post-integration reaction, and the reaction rate is a constant, for the controlling conditions are unchanged.</P><P>$$synNotch-TEV+Ts\xrightarrow[\quad\quad]{C12}U+TEV\ \ \ \ \ \ \ \ \ \ \ (1.2)$$</P><P>(1.2) represents that the TEV protein is released by the intracellular domain of the sporadic synNotch when the synNotch-TEV protein was stimulated by an external Target signal. The rate of the reaction is calculated by the law of mass action which is in proportion to the number of receptors on the cell membrane and the number of external target signals.</P><P>$$synNotch-TEV\xrightarrow[\quad\quad]{C13}\emptyset\ \ \ \ \ \ \ \ \ \ \ (1.3)$$</P><P>(1.3) represents the degradation of synNotch membrane proteins linked to TEV proteins The degradation rate is directly proportional to its protein content.</P><P>$$TEV\xrightarrow[\quad\quad]{C14}\emptyset\ \ \ \ \ \ \ \ \ \ \ (1.4)$$</P><P>(1.4) represents the degradation of TEV protein.</P><P>The degradation rate is directly proportional to its protein content.</P><P>The figures below show the result of simulation using the Gillespie algorithm.</P><FIGURE><FIGCAPTION><B>Fig. 1.1. </B>It shows the number of synNotch-TEV protein changes at different time. We can see that the synNotch-TEV of a single cell will be stable near 20,000 in the absence of a signal (Ts = 0).Because the express rate of synNotch-TEV is certain while the degradation rate of it is directly proportional to its protein content, with the rise of the amount of protein, its express rate will be equal to its degradation rate, making the amount of synNotch-TEV protein keeps stable near 20000.</FIGCAPTION></FIGURE><P>It is concluded that the initial value of synNotch-TEV is stable at about 20,000 before the Target signal is received, which is used as the initial condition for subsequent translation.</P><FIGURE><FIGCAPTION><B>Fig. 1.2. </B>Relationship between the amount of synNotch-TEV and Ts number level in stable cell lines.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Fig. 1.3. </B> Relationship between the amount of TEV and Ts number level in stable cell lines.</FIGCAPTION><FIGCAPTION><B>Fig. 1.2</B>&amp;<B>Fig. 1.3</B> When stimulated by Target signal, synNotch-TEV is also consumed by the reaction of releasing TEV. Because there are many synNotch-TEV proteins on the membrane, the rate of consumption is higher than the generation and the amount of synNotch-TEV will decrease, which causes the rate of degradation and TEV release to become slower. When the rate of consumption equals to that of generation, the amount of synNotch and the number of intracellular TEV will become stable.</FIGCAPTION></FIGURE><P><B>Conclusion: </B>When a stabilized cell contacts different levels of extracellular signals, synNotch membrane protein will release different amounts of TEV. The amount of TEV protein is in proportion of the strength of target signals.</P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1 id="Part2 Internal signal regulating recombinase production">Part2 Internal signal regulating recombinase production</H1><P>
                In this part, intracellular proteins can degrade or destroy the repressors, reducing their number. Different levels of protein have different effects on the relieving repression. Then, using Hill equation, we can calculate the expression rate of the recombinase based on the repressor protein number. At the same time, we use the error-reducing reaction to solve the problem of the extra recombinase. Here, we choose TEV as intracellular protein, and TetR as repressor protein.
            </P><P>In this section, 12 reactions are considered:</P><P>$$G_{TetR}\xrightarrow[\quad\quad]{C21}TetR\ \ \ \ \ \ \ \ \ \ \ (2.1)$$</P><P>$$TetR\xrightarrow[\quad\quad]{C22}\emptyset\ \ \ \ \ \ \ \ \ \ \ (2.2)$$</P><P>$$2TetR+tetO\xrightarrow[\quad\quad]{C23}TetOR\ \ \ \ \ \ \ \ \ \ \ (2.3)$$</P><P>$$TEV+TetOR\xrightarrow[\quad\quad]{C24}tetO+TEV+U\ \ \ \ \ \ \ \ \ \ \ (2.4)$$</P><P>$$TetOR\xrightarrow[\quad\quad]{C25}tetO+U\ \ \ \ \ \ \ \ \ \ \ (2.5)$$</P><P>$$G_{rec}\xrightarrow[\quad\quad\quad\quad\quad\quad]{C31-TetOR}rec\ \ \ \ \ \ \ \ \ \ \ (2.6)$$</P><P>$$rec\xrightarrow[\quad\quad]{C33}\emptyset\ \ \ \ \ \ \ \ \ \ \ (2.7)$$</P><P>$$G_{RDF-inhibitor}\xrightarrow[\quad\quad\quad\quad\quad\quad]{C32-TetOR}RDF-inhibitor\ \ \ \ \ \ \ \ \ \ \ (2.8)$$</P><P>$$RDF-inhibitor\xrightarrow[\quad\quad]{C34}\emptyset\ \ \ \ \ \ \ \ \ \ \ (2.9)$$</P><P>$$G_{rec-inhibitor}\xrightarrow[\quad\quad]{C35}rec-inhibitor\ \ \ \ \ \ \ \ \ \ \ (2.10)$$</P><P>$$rec-inhibitor\xrightarrow[\quad\quad]{C36}\emptyset\ \ \ \ \ \ \ \ \ \ \ (2.11)$$</P><P>$$rec+rec-inhibitor\xrightarrow[\quad\quad]{C37}\emptyset\ \ \ \ \ \ \ \ \ \ \ (2.12)$$</P><P>The specific content is as follows. (As for each chart in this part, we choose the quantity of substance as the vertical axis, and 0.01 hour as the unit of the horizontal axis. Different subplots correspond to different levels of intracellular protein number).</P><P>$$G_{TetR}\xrightarrow[\quad\quad]{C21}TetR\ \ \ \ \ \ \ \ \ \ \ (2.1)$$</P><P>(2.1) indicates the expression of the TetR protein. Through the analysis of (1.1) in part1, it is easy to conclude that (2.1) is a post-integration reaction, and the rate of the reaction is constant.</P><P>$$TetR\xrightarrow[\quad\quad]{C22}\emptyset\ \ \ \ \ \ \ \ \ \ \ (2.2)$$</P><P>(2.2) indicates the degradation of TetR protein. The degradation rate is directly proportional to its protein content.</P><P>$$2TetR+tetO\xrightarrow[\quad\quad]{C23}TetOR\ \ \ \ \ \ \ \ \ \ \ (2.3)$$</P><P>(2.3) represents the generation of the binary complex TetOR, and the reaction rate is calculated according to the law of mass action in the model.</P><P>$$TEV+TetOR\xrightarrow[\quad\quad]{C24}tetO+TEV+U\ \ \ \ \ \ \ \ \ \ \ (2.4)$$</P><P>(2.4) indicates that the TEV is a separate TetOR binary complex. This reaction rate is calculated according to the law of mass action.</P><P>$$TetOR\xrightarrow[\quad\quad]{C25}tetO+U\ \ \ \ \ \ \ \ \ \ \ (2.5)$$</P><P>(2.5) indicates that a small amount of TetOR is self-separate. The reaction rate is calculated according to the law of mass action.</P><P>The figure below shows the calculated results.</P><FIGURE><FIGCAPTION><B>Fig. 2.1. </B> After the cells undergo the above reaction, the corresponding number of the repressor is changed at the corresponding signal number (second signal conversion). We can see, when the signal strength is higher than the 9level, TetR is seldom detected. Therefore, we do not need to simulate higher signal number.</FIGCAPTION></FIGURE><P>Using the above calculated amount of repressor, the Hill equation is used to determine the expression rate of the recombinase.</P><P>$$G_{rec}\xrightarrow[\quad\quad\quad\quad\quad\quad]{C31-TetOR}rec\ \ \ \ \ \ \ \ \ \ \ (2.6)$$</P><P>(2.6) indicates the generation of rec (rec is the abbreviation of recombinase) , which is also a post-integration reaction, but the expression rate is regulated by the repressor TetR. The reaction rate in the model is calculated using the Hill equation, and the rate is ((N/k) ^n+1) ^-1. Accordingly, N is the number of TetR, and K is the half-maximum of the suppression effect.</P><P>$$rec\xrightarrow[\quad\quad]{C33}\emptyset\ \ \ \ \ \ \ \ \ \ \ (2.7)$$</P><P>(2.7) indicates degradation of rec. The degradation rate is directly proportional to rec number.</P><FIGURE><FIGCAPTION><B>Fig. 2.2. </B> Different subplots simulate the different number of the repressor calculated from the intracellular signal producing the corresponding expression of rec over time, which is calculated by the Hill equation. When this figure was generated, the response of the minor rec-inhibitor elimination of rec leakage (2.10) - (2.12) was taken into account, making the simulation effect even better. The simulation results without adding this series of noise reduction reactions were shown in <B>Fig. 5.1. </B></FIGCAPTION></FIGURE><P>$$G_{RDF-inhibitor}\xrightarrow[\quad\quad\quad\quad\quad\quad]{C32-TetOR}RDF-inhibitor\ \ \ \ \ \ \ \ \ \ \ (2.8)$$</P><P>(2.8) indicates the generation of RDF. Like rec, it is also a post-integration reaction, whose rate is regulated by the repressor TetR. The reaction rate in the model is calculated by the Hill equation, and the rate is ((N/k) ^n+1) ^-1. Similarly, N is the number of TetR, and K is the half-maximum of suppression effect.</P><P>$$RDF-inhibitor\xrightarrow[\quad\quad]{C34}\emptyset\ \ \ \ \ \ \ \ \ \ \ (2.9)$$</P><P>Degradation of RDF-inhibitor is represented by (2.9). The degradation rate is directly proportional to the number of RDF-inhibitor. The roles and meanings of this reaction will be discussed in the next part.</P><P>$$G_{rec-inhibitor}\xrightarrow[\quad\quad]{C35}rec-inhibitor\ \ \ \ \ \ \ \ \ \ \ (2.10)$$</P><P>(2.10) indicates the generation of the rec-inhibitor, which is also a post-integration reaction, and the expression rate is constant.</P><P>$$rec-inhibitor\xrightarrow[\quad\quad]{C36}\emptyset\ \ \ \ \ \ \ \ \ \ \ (2.11)$$</P><P>(2.11) indicates the degradation of the rec-inhibitor. The degradation rate is directly proportional to rec-inhibitor number.</P><P>$$rec+rec-inhibitor\xrightarrow[\quad\quad]{C37}\emptyset\ \ \ \ \ \ \ \ \ \ \ (2.12)$$</P><P> (2.12) indicates that the rec-inhibitor combined with the leaked rec. It is used to filter rec. The reaction rate was calculated according to the law of mass action and enables the low expression of rec to be combined to ensure the stability of the system.</P><FIGURE><FIGCAPTION><B>Fig. 2.3. </B>  Different sub-figures correspond to different concentrations of TEV, corresponding to different concentrations of TetR. At different concentrations of TetR, the rec concentrations are different, which affect the rec-inhibitor number. At extremely low permeation, rec-inhibitor can almost completely eliminate the rec leakage from the TetR system. </FIGCAPTION></FIGURE><P>It is <B>concluded</B> that internal signal TEV regulates recombinase production. According to (2.4), different concentrations of TEV correspond to different concentrations of TetR. By (2.6), the concentrations of TetR affect rec concentrations. Through the simulation of (2.12), a small amount of rec-inhibitor can almost completely eliminate the rec leakage from the TetR system. This pathway can significantly improves the system stability.</P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1 id="Part3 Recombinase turns on downstream effect expression">Part3 Recombinase turns on downstream effect expression</H1><P>
                In this part, we use the above calculated amount of the rec expression at different signal levels. Using rec and RDF, we can switch the state and calculate the expression of the downstream gene. In this section, 5 reactions are considered:
            </P><P>$$rec+\overline{G}_{downstream}\xrightarrow[\quad\quad]{C41}rec+G_{downstream}\ \ \ \ \ \ \ \ \ \ \ (3.1)$$</P><P>$$RDF+G_{dowmstream}\xrightarrow[\quad\quad]{C61}RDF+\overline{G}_{downstream}\ \ \ \ \ \ \ \ \ \ \ (3.2)$$</P><P>$$G_{RDF}\xrightarrow[\quad\quad]{C}RDF\ \ \ \ \ \ \ \ \ \ \ (3.3)$$</P><P>$$RDF-inhibitor+RDF\xrightarrow[\quad\quad]{C34}\emptyset\ \ \ \ \ \ \ \ \ \ \ \ (3.4)$$</P><P>$$G_{gfp}\xrightarrow[\quad\quad]{C}GFP\ \ \ \ \ \ \ \ \ \ \ (3.5)$$</P><P>$$rec+\overline{G}_{downstream}\xrightarrow[\quad\quad]{C41}rec+G_{downstream}\ \ \ \ \ \ \ \ \ \ \ (3.1)$$</P><P>(3.1) represents that recombinase specifically recognizes the DNA sequence between the two sites and reverses it, so that the downstream gene expression is reversed to an open state. (Downstream genes are GFP and RDF Gene)</P><P>$$RDF+G_{dowmstream}\xrightarrow[\quad\quad]{C61}RDF+\overline{G}_{downstream}\ \ \ \ \ \ \ \ \ \ \ (3.2)$$</P><P>(3.2) indicates that the reverse recombination factor specifically recognizes the two sites after the recombinase worked on the DNA, and makes the reversal DNA sequence between the sites restored to the shape.</P><P>$$G_{RDF}\xrightarrow[\quad\quad]{c}RDF\ \ \ \ \ \ \ \ \ \ \ (3.3)$$</P><P>(3.3) indicates the expression of the reverse recombination factor, which exists only when the downstream gene is turned into an open state. The expression of the RDF process is not regulated, and also is a post-integration reaction, so the expression rate is constant.</P><P>$$RDF-inhibitor+RDF\xrightarrow[\quad\quad]{C34}\emptyset\ \ \ \ \ \ \ \ \ \ \ \ (3.4)$$</P><P>(3.4) indicates that the RDF-inhibitor is combined with the intracellular RDF when the system requests downstream gene opening. The reaction rate is calculated according to the law of mass action. It makes our system stable in the open state. When it is not requested to open, the increase of RDF is not limited, so that it can function to shut down the system and realize real-time feedback. </P><FIGURE><FIGCAPTION><B>Fig. 3.1. </B> Relationship between the amount of RDF-inhibitor and TEV number level in stable state.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Fig. 3.2. </B>  Relationship between the amount of RDF and TEV number level in stable state.</FIGCAPTION></FIGURE><P>In this simulation, the off state is the initial state. When the TEV signal number level is less than 5, there is not enough rec to turn on the downstream gene,and RDF is not expressed; when the TEV signal number level is higher than 5, rec and RDF-inhibitor begin to express simultaneously after de-repression. Then，RDF and GFP are simultaneously expressed after the rec turns on the downstream gene. The combination of RDF-inhibitor and RDF ensures that there is not enough RDF in the cell to shut down the downstream gene, which makes the system stable.</P><P>$$G_{gfp}\xrightarrow[\quad\quad]{c}GFP\ \ \ \ \ \ \ \ \ \ \ (3.5)$$</P><P>The formula (3.5) indicates the production of the effectors. The downstream gene can be expressed when it is reversed to open state. Here, GFP is taken as an example, and the expression is constant after the rectification reaction.</P><P>The figure below shows the opening of the downstream gene. 0 means closed; 1 means open.</P><P>(a):</P><P>(b):</P><FIGURE><FIGCAPTION><B>Fig. 3.2. </B>  (a) Relationship between TEV number level and the state of the downstream gene in stable state. (b)Expression of GFP in different TEV number levels，
                    Each subplot in (b) is corresponding to the corresponding subplot in (a).
                </FIGCAPTION></FIGURE><P>The results meet our expectation: The system doesn’t express GFP in low TEV number level. It begins to express GFP when the concentration level reaches the threshold.</P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1 id="Part4 Significance of the cell culture environment">Part4 Significance of the cell culture environment</H1><P>If the normal number of the signal molecules can be added into the culture medium of cells, it will improve the system stability by 10%.</P><DIV class="section"><H2>First stimulation effect</H2><FIGURE><FIGCAPTION><B>Fig. 4.1. </B> Steady change of cell contact signal, and the amount of synNotch</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Fig. 4.2. </B>Changes in the amount of intracellular TEV in stable cells exposed to different gradient signals</FIGCAPTION></FIGURE></DIV><DIV class="section"><H2>Accompanying stimulation effect</H2><FIGURE><FIGCAPTION><B>Fig. 4.3. </B> Changes in the amount of synNotch in steady-state cell contact with different gradient signals</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Fig. 4.4. </B> Changes in the amount of TEV in steady-state cell contact with different gradient signals</FIGCAPTION></FIGURE><P>It can be seen from the calculation results that the cells need to be stimulated by adding signal molecules of normal number in the culture medium, which can improve the stability of the system by 10%. In the non-irritated culture, the stable amount of synNotch is the maximum amount that cell membrane can carry.  In the stimulus culture environment, the stable value of the synNotch is the residual amount of the synNotch at the corresponding signal  level. Therefore, in the  required environment, the amount of change will not be changed a lot, and the released TEV will not have shock.</P></DIV></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1 id="Part5 Recombinase filtering and de-shocking">Part5 Recombinase filtering and de-shocking</H1><DIV class="section"><H2>Rec-inhibitor noise reduction</H2><FIGURE><FIGCAPTION><B>Fig. 5.1. </B>  Amount of rec expression in the case of no rec-inhibitor noise reduction</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Fig. 5.2. </B> Changes in the amount of TEV in steady-state cell contact with different gradient signals</FIGCAPTION></FIGURE><P>By comparing the noise reduction, the expression discrimination of the switch signal (rec) can be significantly improved.</P><FIGURE><FIGCAPTION><B>Fig. 5.3. </B>  Switching of downstream genes without rec-inhibitor</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Fig. 5.4. </B>  Switching of downstream genes under rec-inhibitor noise reduction</FIGCAPTION></FIGURE><P>In general, rec-inhibitor noise reduction doesn’t have significant effect on the total on-off threshold, so our experiment group doesn’t add this reaction. However, for the accuracy of the system we believe that it is worthwhile to reduce noise when time and cost allow.</P></DIV><DIV class="section"><H2>RDF-inhibitor shock remover</H2><P>
                    If we do not add RDF-inhibitor, and rely solely on rec and RDF to competitively control downstream genes, we will find that our system may be in a state of constant shock.
                    The following are the simulated results:
                </P><FIGURE><FIGCAPTION><B>Fig. 5.5. </B> Switching of downstream genes without RDF-inhibitor noise reduction. X axis represents whether the downstream gene is active.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Fig. 5.6. </B>  No RDF-inhibitor noise reduction downstream gene: GFP release</FIGCAPTION></FIGURE><P>We can see that the switching of downstream gene are controlled by signal number. Only when number level is higher than 5, the switch will be open. However, because the expression of RDF is not restricted, it will reset the downstream genes directly, which will turn off the switch. Then the gene will be turned on by the effect of rec again, which will cause periodical shock in the system.</P><P>When we simulate the reaction, we assume that the open gene is not the same matter as the closed gene. Those two matters converse to each other by reaction (3.1), and (3.2). Because of the shock of effectors’ gene, the expression of GFP is discontinuous.</P><P>When the system needs to be turned on, oscillations may occur, which may damage the DNA.</P></DIV></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1 id="Part6: Robustness of the model">Part6 Robustness of the model</H1><P>
                After adding a lot of disturbances to the previously determined parameters, the calculation results are as follows:
            </P><FIGURE><FIGCAPTION><B>Fig. 6.1. </B> The switch diagram after adding a lot of disturbances</FIGCAPTION></FIGURE><P>The stability of the algorithm and our path is verified by adding a lot of perturbations on the parameters.</P><P><A href="https://static.igem.org/mediawiki/2018/c/cd/T--NAU-China--modelinterpretation.pdf" target="_blank">Full Text Access</A></P></DIV></DIV></DIV><TITLE>footer</TITLE><DIV id="footer"><DIV class="footer-col" id="Acknowledgements"><SPAN class="footer-title">Acknowledgements</SPAN><DIV class="footer-row"><P>Nanjing Agricultural University</P></DIV><DIV class="footer-row"><P>College of Life Sciences of Nanjing Agricultural University</P></DIV><DIV class="footer-row"><P>Nanjing University</P></DIV><DIV class="footer-row"><P>Chinese Academy of Sciences</P></DIV></DIV><DIV class="footer-col" id="Useful_Links"><SPAN class="footer-title">Useful Links</SPAN><P><A href="https://2018.igem.org/">Main Page</A></P><P><A href="https://igem.org/Special:SpecialPages">Special Pages</A></P></DIV><DIV class="footer-col" id="ContactUs"><SPAN class="footer-title">Contact Us</SPAN><DIV class="footer-row"><P>Facebook</P></DIV><DIV class="footer-row"><P>Twitter</P></DIV></DIV><DIV class="footer-col" id="About Us"><SPAN class="footer-title">About Us</SPAN><P><SPAN style="font-size:1.1vw">NAU-CHINA 2018</SPAN> from 
                Nanjing Agricultural University 
                NO.1 Weigang, Xuanwu District
                Nanjing, CHINA
                210095
            </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>