<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NAU-CHINA_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NAU-CHINA/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Template:2018_NAU-CHINA</TITLE></P><TITLE>header</TITLE><DIV class="menu"><UL><LI><A href="#">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Team_members">Team members</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Attributions">Attributions</A></LI></UL><LI><A href="#">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Application_prospects">Application Prospects</A></LI><LI><A href="#">RESULTS</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Basic_Part">Basic Part</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Improve">Improved Parts</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Part_Collection">Part Collection</A></LI><LI><A href="#">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Model Details">Model Illustration</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Auxiliary Understanding">Auxiliary Understanding</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Software">iGEMCloud</A></LI><LI><A href="#">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/IntegratedHP">Integrated HP</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Safety">SAFETY</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Notebook">NOTEBOOK</A></LI></DIV><TITLE>Demonstrate</TITLE><DIV class="topLine"><P class="top-title">RESULTS</P><P class="sec-title">Demonstrate</P></DIV><DIV class="main-content"><DIV class="textblock"><H1>Overview</H1><P>
                Due to the complex circuit design of our subject, the numerous combination of promoters and gene elements lead to the effect on the whole system in case of the malfunction of any parts, making it difficult for us to locate the malfunction. Therefore, we decided to adopt the method of debugging the program usually employed by computer programmers.
            </P><P>1)Verify the function of each part.</P><P>2)Combine the parts into two large modules of upstream and downstream circuits to verify the function. </P><P>3)Assemble the upstream and downstream circuits to verify the function of the whole system.</P><P> However, due to time constraints, we cannot complete such detailed and complete functional verification of various combination designs in just a few months. We have completed the functional verification for most parts and some of the upstream and downstream circuits, but time does not allow us to combine the upstream and downstream circuits for final functional verification. This is undoubtedly regretful, but we have provided concrete ideas for future experiments to help us complete the improvement of the subject. We also put these future experiments on our Wiki.</P></DIV><DIV class="textblock"><H1>Demonstrate</H1><DIV class="section"><H2 style="font-size:30px !important">Upstream circuit</H2><P>The upstream circuit mainly designs a signal path to enable cells to receive specific external signals and activate the downstream core circuit to realize the threshold function.  Therefore, the upstream circuit can be replaced considering different situations. Here, we provide an upstream circuit design as a reference and other researchers can design their own upstream path to their own needs.</P></DIV><DIV class="section"><H2>Customizing the signal path of cells in response to external signals</H2><P>There is a wide type of extracellular signals. Cells receive extracellular signals and respond to the signal molecules accordingly. We hope to customize a kind of receptor so that it can recognize the signal molecules and regulate downstream gene expression [1]. We choose synNotch as an ideal receptor.</P><P>Similar to some signal molecules, take Epidermal Growth Factor Receptor in our realistic system as an example, the GFP is also protein but more stable and has no impact on the system. As for visibility and operability, cell surface-expressed GFP as a model of extracellular signal molecule is a better choice. Therefore, we want to replace the excellular domain of synNotch with Lag16, a kind of antigen of GFP. Similar parts  have been used in previous project (<A href="https://2017.igem.org/Team:Fudan ">iGEM 2017 Fudan</A>). We received the plasmids with the gene of cell surface-expressed GFP and synNotch from iGEM 2018 Fudan team. But the intracellular domain of synNotch is tTA, a kind of activation factor. Since synNotch was applied to the transformation of cells, the intracellular domain has been replaced by transcription activator factors such as GAL - VP64 and tTA [2]. However, promoters are not completely non-expressed until they are activated, and they often have background expression. Moreover, we hope to make some changes to the intracellular domain of synNotch, trying to replace the intracellular domain with non-traditional transcriptional activator factor to broaden the selection and application of synNotch intracellular domain. So we construct the part of Anti-GFP-mnotch-TEV protease-NLS(<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2557000 ">BBa_K2557000</A>)</P><P>We found that the previous team <A href="https://2017.igem.org/Team:Oxford/ ">iGEM 2017 Oxford</A> modified TetR by replacing the domain between TetR DNA binding domain and regulatory core domain with TEV enzyme cleavage site, so that TetR with TEV cleavage site (<A href="http://parts.igem.org/Part:BBa_K2557050 ">BBa_K2557050</A>) will be destroyed in the presence of TEV, losing the function of repressing promoter after tetO(<A href="http://parts.igem.org/Part:BBa_K2557038 ">BBa_K2557038</A>) sequence and opening up the expression of downstream genes.</P><P>According to the idea of iGEM 2017 Oxford, we replaced the intracellular domain of synNotch with TEV and repeated the function verification of synNotch to explore whether replacing intracellular domain with TEV will affect the function of synNotch.  </P><FIGURE><FIGCAPTION class="_table">Fig.1. Anti-GFP-mnotch-TEV protease-NLS can be located to the membrane</FIGCAPTION><FIGCAPTION class="_table"> Transfect HEK 293T with plasmid containing anti-GFP-mnotch-TEV protease-NLS. Mix cells with GFP, and incubate for 30 minutes. Use PBS to wash away free GFP.</FIGCAPTION><FIGCAPTION class="_table">(A) A Brief Expression of the plasmid containing anti-GFP-mnotch-TEV protease-NLS   </FIGCAPTION><FIGCAPTION class="_table">(B)Schematic diagram of the experiment shown in Fig.1  </FIGCAPTION><FIGCAPTION class="_table">(C) Fluorescence microscope observation of the cells cross-linked with GFP. The results show that synNotch can be located to the membrane.  </FIGCAPTION><FIGCAPTION class="_table">(D) Blank control (without transfection).</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION class="_table">Fig.2. Assay of the synNotch-TEV and FLAG-tagged TEV concentration affected by cell surface-expressed GFP.   </FIGCAPTION><FIGCAPTION class="_table">Co-culture the 293T cells expressing GFP on the cell surface with the cells transferred with anti-GFP-mnotch-TEV protease-NLS for 1h to extract protein for western bolt detection.</FIGCAPTION><FIGCAPTION class="_table">    (A) Anti-GFP-mnotch-TEV protease-NLS affected by cell surface-expressed GFP can release its intracellular domain.</FIGCAPTION><FIGCAPTION class="_table">(B) Fluorescence microscope observation of the cells  transfected with plasmids containing the gene of cell surface-expressed GFP. </FIGCAPTION><FIGCAPTION class="_table">(C) Image results developed in Western blot shows that anti-GFP-mnotch-TEV protease-NLS affected by surface-expressed GFP can be resolved into FLAG-TEV and V5-mNotch.  </FIGCAPTION></FIGURE><FIGCAPTION class="_table">(D) Gray scale analysis of western blot image shows the relative level of the Flag tagged anti-GFP-mnotch-TEV protease-NLS affected by cell surface-expressed GFP.   </FIGCAPTION><FIGCAPTION class="_table">Data are mean ±S.E. (n=3).   </FIGCAPTION><FIGCAPTION class="_table">**, p &lt; 0.01;    </FIGCAPTION><FIGCAPTION class="_table">N.S., no significance.   </FIGCAPTION><P>The above two experiments show that the modified synNotch can be located on the surface of cell membrane normally and release intracellular domain after receiving external signals. The replacement of intracellular domain with TEV has no effect on the function of synNotch. </P></DIV><DIV class="section"><H2>Eukaryotic verification of TEV activation transcription system based on modified TetR</H2><P>As mentioned earlier, inducible promoters using transcription activator factors that cannot inhibit transcription often have some leakage due to background expression.</P><P>  However, our system hopes to realize the absolute function of 0/1 switch, and the background expression is what we do not expect. Therefore, we need to find a transcription activation system with very low background expression.</P><P> Coincidentally, the previous team iGEM 2017 Oxford was making a similar attempt. They have designed TEV activation transcription system based on the modified TetR. Although they have not fully proved that the system can work effectively due to time constraints, we believe that their theoretical basis for designing the system is reasonable. Therefore, we attempt to verify their system with eukaryotic cells.   </P><FIGURE><FIGCAPTION class="_table">Fig.3. Inhibition of TetR on different strength promoters with tetO sequence </FIGCAPTION><FIGCAPTION class="_table">(A) A schematic diagram of the composition and interaction of the two plasmids transferred into the cell in the above-mentioned experiment</FIGCAPTION><FIGCAPTION class="_table">(B) Fluorescence microscope observation of HEK 293T transfected with plasmids containing different strength promoters with tetO sequence.</FIGCAPTION><FIGCAPTION class="_table">(C) Fluorescence microscope observation of HEK 293T transfected with plasmids containing different strength promoters with tetO sequence and TetR.</FIGCAPTION></FIGURE><P>The result shows that TetR can effectively repress the expression of green fluorescent protein in the promoters with tetO sequence.  </P><FIGURE><FIGCAPTION class="_table">Fig4. Under the effect of surface-expressed GFP, TEV released as the the intracellular domain of synNotch relieves the inhibition of TetR on the promoter with tetO sequence.</FIGCAPTION></FIGURE><P>Stably transfer Jurkat T cells with the modified TetR gene to construct a stably transferred cell line. Then transfer plasmids containing anti-GFP-mnotch-TEV protease-NLS and tetO-miniCMV-EGFP(<A href="http://parts.igem.org/Part:BBa_K2557028 ">BBa_K2557028</A>) genes into the aforementioned stably transferred cell. Co-culture the 293T cells expressing GFP on the cell surface with these Jurkat T cells for 4 h when 293T cells were deposited at the bottom of the culture medium and separated from suspended Jurkat T cells. </P><P>(A) Experimental schematic diagram for verifying TEV suppressing TetR Inhibition </P><P>(B) Fluorescence microscope observation of the stably transfferred cell line stably transferred with TetR gene.</P><P>(C) Transfer the aforementioned stably transfferred cell line with anti-GFP-mnotch-TEV protease-NLS and tetO-miniCMV-EGFP genes. Fluorescence microscope observation of the cells.</P><P>(D)Fluorescence microscope observation of the Jurkat T cells in image (B) co-cultured with 293T cells expressing cell surface-expressed GFP for 4 h.</P><P>Through fluorescence microscopy, we could observe that the suspended T cells emit green fluorescence, which is clearly distinguished from the weaker green fluorescence of 293T cells expressing surface-expressed GFP deposited at the bottom of the culture medium. The results show that TEV can relieve the inhibition of TetR on the promoter in 293T cells. It means that we have successfully verified the function of TEV - activated transcription system based on the modified TetR in eukaryotic cells and the results also confirm preliminarily that our upstream circuit can work normally. However, we have to admit that due to we chosed GFP as our reporter gene, it is difficult to distinguish it from cell surface-expressed GFP. Our verification experiment is not intuitive. If we need to prove the function of TEV suppressing the inhibition of TetR strongly, further optimized experiments are still needed.</P></DIV><DIV class="section"><H2 style="font-size:30px !important">Downstream circuit</H2><P> The downstream pathway is the core circuit for us to realize the threshold function. According to literature [3], they have verified the inversion function of the three recombinases in prokaryotic cells and proved the threshold function of the recombinases, i.e. the recombinases do not have the inversion function at low concentration. Only when the concentration of recombinase reaches a certain threshold, can the recombinases work normally.    </P><P> According to the same document, we designed our pathway in eukaryotic cells, expecting to realize threshold switching in eukaryotic cells. For this reason, we try to test the inversion function of recombinases and the threshold characteristics of the combination of three different recombinases ( Bxb1, TP901, PhiC31 ) and three promoters with different intensities ( miniCMV, EF1 - α, Ubc ) in eukaryotic cells.</P><P>We also verify the function of RDF [4] to demonstrate our 0/1 switch resettable in HEK 293T cells. </P></DIV><DIV class="section"><H2>Pronuclear verification of Bxb1 recombinase plasmid given by Peking University</H2><P>Before the eukaryotic verification of the recombinases, we performed prokaryotic verification of Bxb1 recombinase.</P><P>Two plasmids with different resistances and origins of replication were used for function verification of the Bxb1 recombinase. One of them is a reporter gene plasmid, which uses the constitutive promoter J23119. The recombination site is located on both sides of the promoter: one side is sfGFP, and the other is mRFP. The other plasmid is a recombinase expression plasmid using PBAD, an inducible promoter, which is induced by arabinose. When the two plasmids were co-transfected into E. coli, the reporter plasmid expressed sfGFP, a kind of green fluorescent protein; when the inducer arabinose was added, the recombinant enzyme was expressed, the promoter was inverted, and the mRFP , a kind of red fluorescent protein, was expressed.</P><P>Due to the use of two different resistant plasmids, kana and chloramphenicol, we used a plate containing two resistances of kana and chloramphenicol for screening, grew more colonies, and randomly selected 9 singles. After the colonies, we made colony PCR (Fig. 5) and the results showed that both plasmids were transferred.</P><FIGURE><FIGCAPTION class="_table">Fig. 5.  A total of 9 single colonies were verified. Lane 1-9, recombinase expression plasmid validation; line 10, DL2000 DNA Marker; line 11-19, reporter gene expression plasmid validation;line 20, DL2000 DNA Marker.</FIGCAPTION></FIGURE><P>The verified E. coli was separately placed in a 1.5 ml centrifuge tube containing antibiotic-containing LB medium, and the culture was grown at 37° C and 200 RPM for 6 hours, and then the culture was aliquoted into two portions, one of which was added with an inducer (10 mM Arabinose). Two cultures were grown for 12 hours at 37°C and 200 RPM, and the mixture was incubated for 1 hour at room temperature prior to testing. Both lasers are used to excite both sfGFP and mRFP. </P><FIGURE><FIGCAPTION class="_table">Fig. 6. After adding inducers to induce the production of Bxb1 recombinase, no expected red fluorescence signal representing the ability of recombinase to reverse was detected.</FIGCAPTION><FIGCAPTION class="_table">(A)Schematic design of functional verification experiment of Bxb1 recombinase in Prokaryotic Cells</FIGCAPTION><FIGCAPTION class="_table">(B)Two repetitions were selected and the results showed no obvious green fluorescence</FIGCAPTION><FIGCAPTION class="_table">(C)Two replicates were selected after addition of the inducer and the results showed no obvious red fluorescence</FIGCAPTION></FIGURE><P>The result shows no obvious fluorescence. We changed some conditions, such as lowering the temperature, adjusting the rotation speed, adjusting the time, etc. But we still did not get the expected results. We consulted the teacher and the teacher replied that there might be weak fluorescence but our instrument couldn't detect it.</P><P>Although the prokaryotic function verification of the Bxb1 recombinase plasmid given by Peking University failed, unexpectedly, we successfully verified   the function of the Bxb1 recombinase optimized by codons function in eukaryotic cells. We have not yet found out the reason for the failure. But we decided to shelve our doubts for the time being and continue other experiments.</P></DIV><DIV class="section"><H2>Functional verification of three kinds of recombinases in HEK 293T cell</H2><FIGURE><FIGCAPTION class="_table">Fig.7. All three recombinases can effectively reverse the sequence between recognition sites, and exhibit different reverse efficiency due to different promoter strength and recombinase types. </FIGCAPTION><FIGCAPTION class="_table">(A) Schematic diagram of composition and reversal of different recombinase and promoter combinations</FIGCAPTION><FIGCAPTION class="_table">(B) The image under fluorescence microscope for 293T cells, transfected with plasmids containing the recombinase recognition sites (the first column picture) or transfected with plasmids containing corresponding combination of promoters and recombinase genes (other column pictures) together, are shown.</FIGCAPTION></FIGURE><P> The results show that the recombinases can recognize the sites and reverse the sequence between sites in HEK 293T. </P></DIV><DIV class="section"><H2>Functional verification of reversal efficiency and threshold characteristics of different recombinases in HEK 293T Cells</H2><FIGURE><FIGCAPTION class="_table">  Fig.8. Recombinase has different intensity reversal efficiency and threshold</FIGCAPTION><FIGCAPTION class="_table"> HEK 293T cells were co-transfected with six different amounts of plasmids containing recombinase genes (tetO-miniCMV-Bxb1(<A href="http://parts.igem.org/Part:BBa_K2557010 ">BBa_K2557010</A>)and tetO-miniCMV-TP901(<A href="http://parts.igem.org/Part:BBa_K2557016 ">BBa_K2557016</A>)) , and fixed numbers of plasmids containing corresponding recombinase recognition sites. After 36 hours of plasmid co-transfection, the proportion of fluorescent cells and the average fluorescence intensity of cells were detected by flow cytometry. Transfection of different amounts of plasmids containing recombinase genes into cells indicates that cells can produce recombinase at different concentrations. The experiment was repeated three times.  </FIGCAPTION><FIGCAPTION class="_table">
                        (A) Fluorescence microscope observation of HEK 293T  undergone different experimental treatments
                        (B) The statistical chart of average fluorescence intensity of cells shows that the cells with Bxb1 recombinase have a higher fluorescence intensity than those with TP901 recombinase under the same promoter strength and recombinase concentration. However, if the concentration of recombinase is low, there is no significant difference in fluorescence intensity.
                        (C) The statistics of the proportion of fluorescent cells show that the proportion of fluorescent cells has a sudden jump discontinuity between low concentration and high concentration of Bxb1 and TP901 recombinases. Similar results were obtained in all three repetitions.
                    </FIGCAPTION></FIGURE><P>The results of image B show that the reverse efficiency of Bxb1 recombinase is higher than TP901 recombinase under the same promoter strength and recombinase concentration. However, if the concentration of recombinase is low, there is no significant difference in fluorescence intensity. The results of the image C show that Bxb1 and TP901 recombinases have a threshold property. So, the proportion of fluorescent cells have a jump discontinuity between low concentration and high concentration of recombinase.</P></DIV><DIV class="section"><H2>Functional verification of Ubc-Bxb1 recombinase-RDF in HEK 293T cell</H2><FIGCAPTION class="_table"> Fig.9. Bxb1 recombinase-RDF can recognize and reverse the sequence between recognition sites </FIGCAPTION><FIGCAPTION class="_table">(A)Schematic diagram of RDF function verification experiment</FIGCAPTION><FIGCAPTION class="_table">(B) The image under fluorescence microscope for 293T cells, transfected with plasmids containing the recombinase-RDF recognition sites (left panel) or transfected with plasmids containing corresponding recombinase-RDF gene together (right panel), are shown.</FIGCAPTION><P> The results show that the recombinase-RDFs can recognise the sites and reverse the sequence between sites in HEK 293T. </P></DIV></DIV><DIV class="textblock"><H1>Conclusion</H1><P>We verified the functions of most parts and most upstream and downstream paths step by step. We verified the function of synNotch, the inhibition of TetR after modification, the reversal function and threshold characteristics of some recombinases and promoter combinations. However, due to the time constraints, we are unable to complete verification of TEV and the combinations of some recombinases and promoters. Moreover, the combination of upstream and downstream circuits needs to be verified by experiments. We will carry out supplementary experiments in the future to carry out a complete experimental verification of our subject.</P></DIV><DIV class="textblock"><H1>Future experiments</H1><P> In a short period of one year, it is not easy to fully realize such a complex idea. Therefore, we have envisaged the next series of experiments to further realize our project idea, combining the idea of continuous feedback between modeling and wet lab to ensure the best system.</P><DIV class="section"><H2>1. Optimized functional verification of TEV suppressing TetR Inhibition</H2><P>As mentioned earlier, since the reporter gene selected GFP, our experimental results are not intuitive. We will replace the reporter gene with RFP to solve this problem.</P></DIV><DIV class="section"><H2>2. Verification of the combinations of remaining recombinases and promoters</H2><P>We plan to continue the experiment of remaining combinations that have not yet been verified in order to verify the function and threshold characteristics of these combinations of recombinases and compare the inversion efficiency of recombinases. </P></DIV><DIV class="section"><H2>3. Construction of a fully functional stable cell line combining upstream and downstream circuits</H2><P> We plan to finally construct our parts on two plasmids.  Stable cell lines with complete functions were constructed through Puro and BSD screening and their concentration threshold functions will be verified by using agarose beads with different amounts of GFP adsorbed. We intend to apply it to real life.</P></DIV><DIV class="section"><H2>4. Upgrade our system</H2><P> The above mentioned is only a condensed version of our ultimate system which includes inhibitor and more efficient RDF. We hope to upgrade the condensed version to the final version, which also requires the search for appropriate inhabitor and more efficient RDF. We look forward to the day when our final version will come into being.</P></DIV></DIV><DIV class="textblock"><H1>Reference</H1><P> [1] Circuits, C. A. et al. Precision Tumor Recognition by T Cells With Article Precision Tumor Recognition by T Cells With Combinatorial Antigen-Sensing Circuits. 1–10 (2016).</P><P>[2] Morsut, L., Roybal, K. T., Xiong, X., Gordley, R. M. &amp; Coyle, S. M. Engineering customized cell sensing and response behaviors using synthetic notch receptors. Cell 780–791 (2016). doi:10.1016/j.cell.2016.01.012</P><P>[3] Rubens, J. R., Selvaggio, G. &amp; Lu, T. K. Synthetic mixed-signal computation in living cells. Nat. Commun. 7, 1–10 (2016).</P><P>[4] Rutherford, K. &amp; Van Duyne, G. D. The ins and outs of serine integrase site-specific recombination. Curr. Opin. Struct. Biol. 24, 125–131 (2014).</P></DIV></DIV><TITLE>footer</TITLE><DIV id="footer"><DIV class="footer-col" id="Acknowledgements"><SPAN class="footer-title">Acknowledgements</SPAN><DIV class="footer-row"><P>Nanjing Agricultural University</P></DIV><DIV class="footer-row"><P>College of Life Sciences of Nanjing Agricultural University</P></DIV><DIV class="footer-row"><P>Nanjing University</P></DIV><DIV class="footer-row"><P>Chinese Academy of Sciences</P></DIV></DIV><DIV class="footer-col" id="Useful_Links"><SPAN class="footer-title">Useful Links</SPAN><P><A href="https://2018.igem.org/">Main Page</A></P><P><A href="https://igem.org/Special:SpecialPages">Special Pages</A></P></DIV><DIV class="footer-col" id="ContactUs"><SPAN class="footer-title">Contact Us</SPAN><DIV class="footer-row"><P>Facebook</P></DIV><DIV class="footer-row"><P>Twitter</P></DIV></DIV><DIV class="footer-col" id="About Us"><SPAN class="footer-title">About Us</SPAN><P><SPAN style="font-size:1.1vw">NAU-CHINA 2018</SPAN> from 
                Nanjing Agricultural University 
                NO.1 Weigang, Xuanwu District
                Nanjing, CHINA
                210095
            </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>