<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NAU-CHINA_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NAU-CHINA/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Template:2018_NAU-CHINA</TITLE></P><TITLE>header</TITLE><DIV class="menu"><UL><LI><A href="#">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Team_members">Team members</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Attributions">Attributions</A></LI></UL><LI><A href="#">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Application_prospects">Application Prospects</A></LI><LI><A href="#">RESULTS</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Basic_Part">Basic Part</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Composite_Part">Composite Part</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Improve">Improved Parts</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Part_Collection">Part Collection</A></LI><LI><A href="#">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Model Details">Model Illustration</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Auxiliary Understanding">Auxiliary Understanding</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Software">iGEMCloud</A></LI><LI><A href="#">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/IntegratedHP">Integrated HP</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Safety">SAFETY</A></LI><LI><A href="https://2018.igem.org/Team:NAU-CHINA/Notebook">NOTEBOOK</A></LI></DIV><TITLE>Protocol</TITLE><DIV class="topLine"><P class="top-title">PROJECT</P><P class="sec-title">Protocols</P></DIV><DIV class="main-content"><DIV class="textblock"><H1>Transient transfection of Jurkat T cells</H1><P>
             1. Culturing of cells before nucleofection. Cells should be nucleofected after reaching 70-85% confluency, higher cell densities may cause lower nucleofection efficiencies, mycoplasma contamination will negatively influence experiments results.

2. Combine the cells of interest, DNA and the appropriate cell-type specific Nucleofection Solution (Here Buffer C), and transfer to an amaxa certified cuvette (invitrogen cuvette). 1X10<SUP>6</SUP>-5-10<SUP>6</SUP> cells, 1-5 μg plasmid DNA in 1-5 μL H<SUB>2</SUB>O or TE, and 100 μL room temperature Buffer are used in this step. The quality and the concentration of DNA used for nucleofection plays a central role for the efficiency of gene transfer. We strongly recommend the use of high quality products for plasmid purification like Qiagen endofree plasmid kit. The purified DNA should be resuspended in deionised water or TE buffer with a concentration between 1-5 μg/μL. The ration of A260;A280 should be at least 1.8 for nucleofection.

3. Insert the cuvette into the Nucleofector TM and choose the cell-type specific program. Press the start button &quot;X&quot;. If the program is unknown, please following the recommend above. To avoid damage to the cells remove the sample from the cuvette immediately after the program has finished (display showing &quot;OK&quot;).

4. Rinse the cuvette with culture medium and transfer the cells into the culture dish.

5. Incubate at 37 °C until use.</P><TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center">Chemical</TH><TH align="center">Stock Conc. </TH><TH align="center">Final Conc. </TH><TH align="center">40 mL </TH><TH align="center"> 200 mL</TH></TR></THEAD><TBODY><TR><TD align="center">Buffer B </TD></TR><TR><TD align="center"> NaCl   </TD><TD align="center">  5 M</TD><TD align="center"> 137 mM</TD><TD align="center">  1.1 mL</TD><TD align="center"> 5.5 mL</TD></TR><TR><TD align="center">  KCl       </TD><TD align="center">  1 M</TD><TD align="center"> 5 mM</TD><TD align="center"> 0.2 mL</TD><TD align="center">  1 mL</TD></TR><TR><TD align="center">  Na<SUB>2</SUB>HPO<SUB>4</SUB></TD><TD align="center"> 0.5 M </TD><TD align="center"> 0.7 mM</TD><TD align="center">  56 μL</TD><TD align="center">  280 μL</TD></TR><TR><TD align="center">  Glucose              </TD><TD align="center">  1 M</TD><TD align="center">   6 mM</TD><TD align="center"> 240 μL </TD><TD align="center">  1.2 mL</TD></TR><TR><TD align="center">  HEPES(pH7.0)             </TD><TD align="center">  0.1 M</TD><TD align="center"> 20 mM</TD><TD align="center">  8 mL</TD><TD align="center">  40 mL </TD></TR><TR><TD align="center">   dH<SUB>2</SUB>O</TD><TD align="center">  30.4 mL   </TD><TD align="center"> 152 mL </TD></TR></TBODY></TABLE><TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center">Chemical</TH><TH align="center">Stock Conc. </TH><TH align="center" align="center">Final Conc. </TH><TH align="center">40 mL </TH><TH align="center"> 200 mL</TH></TR></THEAD><TBODY><TR><TD align="center"> Buffer C </TD></TR><TR><TD align="center"> KCl          </TD><TD align="center">  1 M</TD><TD align="center"> 120 mM</TD><TD align="center">  4.8 mL</TD><TD align="center"> 24 mL</TD></TR><TR><TD align="center">  CaCl<SUB>2</SUB></TD><TD align="center">  1 M</TD><TD align="center"> 0.15 mM</TD><TD align="center"> 6 μL</TD><TD align="center">  30 μL</TD></TR><TR><TD align="center">  Na<SUB>2</SUB>HPO<SUB>4</SUB></TD><TD align="center"> 0.5 M </TD><TD align="center"> 10 mM</TD><TD align="center">  0.8 mL</TD><TD align="center">  4 mL</TD></TR><TR><TD align="center">  Glucose          </TD><TD align="center">  1 M</TD><TD align="center">   6 mM</TD><TD align="center"> 240 μL </TD><TD align="center">  1.2 mL</TD></TR><TR><TD align="center">  HEPES(pH7.6)       </TD><TD align="center">  1 M</TD><TD align="center"> 25 mM</TD><TD align="center">  1 mL</TD><TD align="center">  5 mL  </TD></TR><TR><TD align="center">   EGTA        </TD><TD align="center"> 0.1 M </TD><TD align="center"> 2 mM</TD><TD align="center">  0.8 mL</TD><TD align="center"> 4 mL </TD></TR><TR><TD align="center">   MgCl<SUB>2</SUB></TD><TD align="center">  1 M</TD><TD align="center"> 5 mM</TD><TD align="center">  0.2 mL</TD><TD align="center"> 1 mL </TD></TR><TR><TD align="center">   dH<SUB>2</SUB>O</TD><TD align="center">    32.1 mL  </TD><TD align="center">  160.8 mL</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1>Lipofectamine® 2000 DNA Transfection Reagent Protocol</H1><P>1、Seed cells to be 70–90% confluent at transfection. 
2、Dilute four amounts of Lipofectamine® Reagent in Opti-MEM® Medium Opti-MEM® Medium.
3、Dilute DNA in Opti-MEM® Medium.
4、Add diluted DNA to diluted Lipofectamine® 2000 Reagent (1:1 ratio).
5、Incubate for 20 minutes at room temperature. 
5、Add DNA-lipid complex to cells.
6、Incubate cells for 1–3 days at 37 °C. Then analyze transfected cells.</P><TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center">Culture Vessel </TH><TH align="center">Surface Area per well (cm<SUP>2</SUP>) </TH><TH align="center">Opti-MEM Media </TH><TH align="center">DNA (μg) </TH><TH align="center">Lipo 2000 (μL)  </TH></TR></THEAD><TBODY><TR><TD align="center"> 12-well </TD><TD align="center"> 4 </TD><TD align="center"> 50 μL *2*12 </TD><TD align="center"> 1.0 </TD><TD align="center"> 3.75</TD></TR><TR><TD align="center"> 6-well </TD><TD align="center"> 10</TD><TD align="center"> 100 μL*2*6</TD><TD align="center"> 2.0</TD><TD align="center"> 7.5</TD></TR><TR><TD align="center"> 60-mm </TD><TD align="center"> 20</TD><TD align="center"> 500 μL *2*1</TD><TD align="center"> 4.0</TD><TD align="center"> 24</TD></TR><TR><TD align="center"> 10-cm </TD><TD align="center"> 60</TD><TD align="center"> 1 mL*2*1 </TD><TD align="center"> 8.0</TD><TD align="center"> 40</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1>Calcium Phosphate Transfection</H1><P><TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center"> 2X HBS   </TH><TH align="center">  For 1 pter</TH></TR></THEAD><TBODY><TR><TD align="center"> HEPES   </TD><TD align="center">   10 g  </TD></TR><TR><TD align="center"> KCI    </TD><TD align="center">  0.74 g</TD></TR><TR><TD align="center">  Glucose  </TD><TD align="center">   2.4 g </TD></TR><TR><TD align="center"> NaCl   </TD><TD align="center"> 16.3 g </TD></TR><TR><TD align="center"> Na<SUB>2</SUB>PO<SUB>4</SUB></TD><TD align="center"> 0.214 g</TD></TR></TBODY></TABLE></P><P>
Dissolve, in 950 mL H<SUB>2</SUB>O. Adjust pH to exactly 7.5 with 1 M HCL, bring up to 1 L with ddH<SUB>2</SUB>O. Filter using 0.22 micron filter. Aliquot in 50 mL Falcon tubes. Store at -20 ℃.

  2 M CaCl<SUB>2</SUB>: Filters sterilize. Store in 1 mL aliquots at -20 ℃. Stable at RT.</P><P>Transfection</P><P>1. Twenty-tour hours prior to transfection, inoculate 1-2×10<SUP>6</SUP> cells/10 cm plate in 10 mL DMEM/F12 media + 10% BCS supplemented with L-glutamine and Pen/Strep. At the time of transfection, cells should be about 60-70% confluent.
2. The following day (20-24 hours later), transfect cells. Up to 15 to 20 μg DNA can be used for one 10-cm plate. If more than one plasmid is used for transfecting the same plate, equalize DNA amounts among different plates by adding vector plasmid DNA to some plates, so that the total amount of plasmids used for each plate is the same. For transfection.
a. Prepare the following mix for 10 cm plate:
61 μL 2 M CaCl<SUB>2</SUB>
10 μg DNA total
Add ddH<SUB>2</SUB>O to 500 μL 
Then vortex to mix.
b. Slowly (drop-wise) add 0.5 mL of 2x HBS while typing the mixture prepared in (a).
c. Drop total 1 mL of the final mix around the plate, then mix gently. Place the plate back in the incubator.
3. Twelve to 16 hours after transfection, gently aspirate the medium and add 10 mL of pre-warmed fresh medium to the plates. Try not to disturb the fine DNA-CaPO<SUB>4</SUB> precipitates on the bottom of the plate.
4. The following day, harvest the cells and extract protein as needed. </P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1>RT-qPCR</H1><DIV class="section"><H2>Extraction of RNA</H2><P>1. Suck and discard DMEM culture solution. Wash cells with 500 μL PBS, suck and discard PBS.  Add 300 μL trypsin to digest the cells.  Add the same amount of 10 % DMEM culture solution to stop the digestion. Transfer the cell suspension to a 1.5 mL EP tube and centrifuged at 1000 rpm/min for 5 min using a centrifuge. Suck and discard the supernatant , and add 600 μL PBS to resuspend the cells. Add 500 μL trizol to 170 μL cell suspension, mix well, transfer the mixture to 1.5 mL EP tube, and let stand at room temperature for 10 min.
 2. Add 1 / 5 volume of chloroform, shake violently, mix well, and let stand at room temperature for 10 min.
 3. Pre-cool the centrifuge to 4 ℃ and centrifuge at 12000 g and 4 ℃ for 20 min.
 4. Transfer 250 μL of supernatant to a new EP tube and add isopropanol of equal volume. Let stand at room temperature for 10 min and precipitate at - 20 ℃ for 1 h.
 5. Centrifuge at 12000 g and 4 ℃ for 20 min, discard supernatant, and retain precipitate.
 6. Add 1 mL of 75 % ethanol prepared with DPEC water to the precipitate, wash the precipitate, centrifuge at 12000 g and 4 ℃ for 15 min.
 7. Discard the supernatant, keep the precipitate, and dry the EP tube upside down on the table for 30 min.
 8. Add 10 μL LDPEC water to each tube, measure the concentration and store at - 80 ℃ for later use.</P></DIV><DIV class="section"><H2>RT-PCR</H2><P> 1. Use takara reverse transcription kit to configure the reverse transcription system according to the following system.</P><TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center">  Reagent</TH><TH align="center"> Volume </TH></TR></THEAD><TBODY><TR><TD align="center"> AMV  </TD><TD align="center"> 0.5 μL</TD></TR><TR><TD align="center"> 5×Buffer  </TD><TD align="center">  2 μL</TD></TR><TR><TD align="center">  dNTP </TD><TD align="center">   1 μL </TD></TR><TR><TD align="center"> DPEC Water  </TD><TD align="center">  3.75 μL </TD></TR><TR><TD align="center"> OligodT  </TD><TD align="center"> 0.5 μL </TD></TR><TR><TD align="center"> RRI </TD><TD align="center">  0.25 μL</TD></TR><TR><TD align="center"> The extracted RNA template  </TD><TD align="center">  2 μL</TD></TR></TBODY></TABLE><P>2. Set a program of 16 ℃ ( 10 min ), 42 ℃ ( 60 min ), 85 ℃ ( 5 min ) and 4 ℃ in the PCR instrument, and put the configured system into the PCR instrument for reaction.</P></DIV><DIV class="section"><H2>qPCR</H2><P>1. The qPCR system was configured in a 96 - well PCR plate using Takara kit according to the following system.</P><TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center">Reagent </TH><TH align="center"> Volume</TH></TR></THEAD><TBODY><TR><TD align="center"> 10×Buffer </TD><TD align="center"> 2 μL</TD></TR><TR><TD align="center"> dNTP  </TD><TD align="center"> 0.4 μL</TD></TR><TR><TD align="center">  MgCl<SUB>2</SUB></TD><TD align="center">  1.2 μL </TD></TR><TR><TD align="center"> rTaq </TD><TD align="center"> 0.3 μL </TD></TR><TR><TD align="center"> Evagreen </TD><TD align="center"> 1 μL </TD></TR><TR><TD align="center"> ddH<SUB>2</SUB>O  </TD><TD align="center"> 13.5 μL</TD></TR><TR><TD align="center">  Forward primer</TD><TD align="center"> 0.3 μL </TD></TR><TR><TD align="center"> Reverse primer </TD><TD align="center"> 0.3 μL </TD></TR><TR><TD align="center">  cDNA template </TD><TD align="center"> 1 μL </TD></TR></TBODY></TABLE><P>2. Set up and run the following programs in QPCR:
 Stage1 Reps:1   95 ℃ 5 min
 Stage2 Reps:40  95 ℃ 15 s
 55 ℃ 30 s
 72 ℃ 30 s
 Stage3 Reps:1 95 ℃ 15 s
 60 ℃ 1 min
 95 ℃ 15 s
 60 ℃ 15 s
 3. Export data.</P></DIV></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1>Western Blot</H1><DIV class="section"><H2>Preparation of lysate from cell culture</H2><P>
1. Suck and discard DMEM culture solution. Wash cells with 500ul PBS, suck and discard PBS.  Add 300 μL trypsin to digest the cells.  Add the same amount of 10 % DMEM culture solution to stop the digestion. Transfer the cell suspension to a 1.5 mL EP tube and centrifuged at 1000 rpm/min for 5 min using a centrifuge. Suck and discard the supernatant , and add 600 μL PBS to resuspend the cells. Use 380 μL of them for WB.
2. Drain the PBS, then add 30 μL ice-cold RIPA (0.5 mL per 107 cells/100 mM dish/150 cm<SUP>2</SUP> flask; 0.2 mL per 5x10<SUP>6</SUP> cells/60 mM dish/75 cm<SUP>2</SUP> flask; 0.05 mL per well/6-well dish).
3. Maintain constant agitation for 30 minutes at 4 °C. 
4. Centrifuge in a microcentrifuge at 4 °C, 20 minutes at 12,000 rpm.
5. Gently remove the tubes from the centrifuge and place on ice, aspirate the supernatant and place in a fresh tube kept on ice, and discard the pellet.
6. Add loading buffer (5% SDS) to supernatant by volume ratio of 1:4 (50 mL/200 mL) then boil at 100 ℃ for around 5-10 minutes. </P></DIV><DIV class="section"><H2>Running the gel</H2><P>1. Wash glass plates and spacers.
2. Place rubber seal in bottom of gel kit.
3. Put the spacers in between the two pieces of glass, making sure they are level.
4. Add the clamps and plugs.
5. Dispense the separation gel. Shake well immediately after adding TEMED to fill the gel. When filling the gel, 5 mL of gel can be sucked out along the glass by the gun, and the gel surface can be raised to a height of 1 cm under the rubber comb. Then add a layer of ethanol to the gel, and the gelation after liquid sealing is faster. 
6. placed at room temperature for 30 min (20 min in summer) to be solidified, when there is a line of refraction between ethanol and gel, the gel has been condensed. When the gel is fully solidified, the upper layer of ethanol can be poured off and the ethanol is blotted dry with absorbent paper.
7. Dispense 5% concentrated gel. Immediately after adding TEMED, the mixture can be filled. Fill the remaining space with the concentrated gel and insert the comb into the concentrate. When filling the gel, the gel should also flow down along the glass plate to avoid bubbles in the gel. Keep the comb level when inserting the comb. Since the volume shrinks and shrinks when the gel solidifies, the loading volume of the sample hole is reduced, so the gel is often applied on both sides during the solidification process of the concentrated gel. After the gel has solidified, pinch the sides of the comb and pull them straight up.
8. Make up running buffer (1 in 10 dilution of stock). Need around a liter.
9. Remove comb and place gel in opposite side of apparatus.
10. load samples and molecular weight markers. 20-30 μg of total protein from cell lysate.
11. Put on grey plastic top (make sure rubber seal is in place).
12. Add the plugs.
13. Slowly add running buffer.
14. Put on front plastic cover and run at 70 V for around 30 minutes. After the sample reaches the concentrated gel, run at 120v for 1 hour.
Note: The gel percentage required is dependent on the size of your protein of interest:
</P><TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center">Protein size </TH><TH align="center">Gel percentage </TH></TR></THEAD><TBODY><TR><TD align="center"> 4–40 kDa </TD><TD align="center">   20% </TD></TR><TR><TD align="center"> 12–45 kDa</TD><TD align="center"> 15%</TD></TR><TR><TD align="center"> 10–70 kDa </TD><TD align="center"> 12.5%</TD></TR><TR><TD align="center"> 15–100 kDa</TD><TD align="center"> 10%</TD></TR><TR><TD align="center"> 25–100 kDa</TD><TD align="center"> 8%</TD></TR></TBODY></TABLE></DIV><DIV class="section"><H2>Transferring the protein from the gel to the membrane</H2><P>1. Need 6 pieces of filter paper (11 x 14.5 cm) and 1 piece of Hybond-P PVDF membrane of the same size.
2. Hydrate Hybond-P by placing in methanol for 3 minutes and in transfer buffer for 10 min.
3. Remove gel from electrophoresis apparatus and cut to size.
4. Place gel in transfer buffer for 10 min.
5. Assemble the transfer cassette as follows (black side down): fiber pad (pre-wet in transfer buffer), filter paper (x3, pre-wet), gel (no bubbles), Hybond-P PVDF membrane (non bubbles), filter paper (x3, pre-wet) and fiber pad (pre-wet).
6. Place cassette in transfer cell (black on black).
7. Add transfer buffer to the electrophoresis tank.
8. Connect to powerpac and run at constant amps (300 mA) for 90 minutes, keep the temperature at 4 ℃.
9. Remove membrance and, if necessary, keep hydrated in TBS in the cold room until needed.</P></DIV><DIV class="section"><H2>Blocking and Antibody staining and Development</H2><P>1. Block the membrane for 1 h at room temperature or overnight at 4 °C using blocking buffer(5% skim milk powder).
2. Incubate the membrane with appropriate dilutions of primary antibody in blocking buffer. We recommend overnight incubation at 4 °C.
3. Wash the membrane in three washes of TBST, 5 min each.
4. Incubate the membrane with the recommended dilution of conjugated secondary antibody in blocking buffer at room temperature for 1 h.
5. Wash the membrane in three washes of TBST, 5 min each.
6. For signal development, follow the kit manufaculturer’s recommendations. Remove excess reagent and cover the membrane in transparent plastic wrap.
7. Acquire image using darkroom development techniques for chemiluminescence, or normal image scanning methods for colorimetric detection.</P></DIV></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1>Package the virus (For stable transfection)</H1><P>1. Add three virus packed plasmids and target plasmids to a 6 cm plate according to the transfection method of Lipo2000.
2. After 6 hours, replace the cell culture medium.
3. After 48 hours, the cell supernatant in the whole plate was recycled into a 15 mL centrifuge tube (cell fragments may be cracked in the supernatant).
4. Centrifuge for 10 minutes at 12,000 rpm.
5. Recycling supernatant, it can be applied directly to infect cells, or be saved in - 20 ℃.</P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1>Verify upstream pathway</H1><P>1. Infected jurkat T cells with the same amount of packaged viruses containing plasmids A, B and C.
2. The cells were screened for 7 days with antibiotics BSD (5 μg/mL) and PURO (1 μg/mL).
3. Continue to culture the cells when the cells no longer decrease.
4. Transfected the plasmid expressing GFP on the membrane into 293T cells according to the method of Lipo2000.
5. After 48 hours, remove the culture medium of 293T, and add Jurkat T cells with cell ratio 1:1 to incubate for 1 hour.
6. Recycle the incubated Jurkat T cells into one hole of the new 6-well plate, and add the unincubated Jurkat T cells in the same amount to another hole.
7. The fluorescent color of the two groups of cells is observed with a fluorescence microscope.</P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1>PCR(using Phanta Max Super-Fidelity DNA Polymerase PCR amplification system)</H1><TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center">Reagent  </TH><TH align="center">Volume/μL</TH></TR></THEAD><TBODY><TR><TD align="center"> ddH<SUB>2</SUB>O   </TD><TD align="center">     Up to 50 </TD></TR><TR><TD align="center"> 2× Phanta Max Buffer </TD><TD align="center">  25</TD></TR><TR><TD align="center"> dNTP Mix(10 mM each)</TD><TD align="center"> 1</TD></TR><TR><TD align="center"> Primer F(10 μM)</TD><TD align="center"> 2</TD></TR><TR><TD align="center"> Primer R(10 μM)</TD><TD align="center"> 2</TD></TR><TR><TD align="center"> Phanta Max Super-Fidelity DNA Polymerase </TD><TD align="center"> 1</TD></TR><TR><TD align="center"> template DNA </TD><TD align="center"> X</TD></TR></TBODY></TABLE><P>Thermocycling conditions for a Routine PCR:</P><TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center">Step </TH><TH align="center">Temperature (℃) </TH><TH align="center">Time </TH><TH align="center"> cycles </TH></TR></THEAD><TBODY><TR><TD align="center"> Initial denaturation  </TD><TD align="center"> 95</TD><TD align="center">  30 seconds/3 min </TD><TD align="center"> 1</TD></TR><TR><TD align="center"> Denaturation  </TD><TD align="center"> 95</TD><TD align="center">  15 seconds</TD><TD align="center" rowspan="3"> 25-35</TD></TR><TR><TD align="center"> Annealing     </TD><TD align="center">  56-72</TD><TD align="center"> 15 seconds</TD></TR><TR><TD align="center"> Extension    </TD><TD align="center"> 72</TD><TD align="center"> 30-60 sec/kb</TD></TR><TR><TD align="center"> Final extension      </TD><TD align="center"> 72</TD><TD align="center">  2-5 minutes</TD><TD align="center"> 1</TD></TR><TR><TD align="center"> Hold        </TD><TD align="center"> 4</TD><TD align="center"> Indefinitely </TD><TD align="center"> 1</TD></TR></TBODY></TABLE><P>Methods:
1.  Prepare the reaction system in a PCR tube on ice, thaw the components and mix well. After use, put them back in -20 ℃.
2.  Gently centrifuge to collect the liquid at the bottom of the tube.
3.  Transfer the PCR tube to the PCR machine, set the parameters and start the thermal cycle.</P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1>Agarose Gel Electrophoresis</H1><P>Methods(for a 1% Gel):

1.  Place the gel tray in the appropriate position in the gel cartridge and place the comb in the correct position.
2.  Measure 0.5 g agarose , put it in a 250 mL Erlenmeyer flask, add 50 mL 1 x TAE buffer and mix, then put the Erlenmeyer flask in the oven and heat to boil until the agarose is completely dissolved. 
3.  Add 5 μL GelRed to the solution.
4.  Pour the solution into the gel casting tray.
5.  After the gel cools to solid, pull out the comb.
6.  Place the gel in the electrophoresis chamber with enough TAE buffer.
7.  Add 10×loading buffer to the sample and mix, then transfer the mixture to the well on the gel with a pipette.
8.  Power on,run at 120 V for half an hour.</P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1>Gel Extraction</H1><P>According to the E.Z.N.A. Gel Extraction kit:
1.Perform agarose gel/ethidium bromide electrophoresis to fractionate DNA fragments. Any type or grade of agarose may be used. However, it is strongly recommended that fresh TAE buffer or TBE buffer be used as running buffer. Do not reuse running buffer as its pH will increase and reduce yields.
2.When adequate separation of bands has occurred, carefully excise the DNA fragment of interest using a wide, clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose. 
3.Determine the appropriate volume of the gel slice by weighing it in a clean 1.5 mL microcentrifuge tube. Assuming a density of 1 g/mL, the volume of gel is derived as follows: a gel slice of mass 0.3 g will have a volume of 0.3 mL. 
4.  Add 1 volume Binding Buffer (XP2). 
5.Incubate at 50-60 °C for 7 minutes or until the gel has completely melted. Vortex or shake the tube every 2-3 minutes. 
6.Insert a HiBind® DNA Mini Column in a 2 mL Collection Tube. 
7.Add no more than 700 μL DNA/agarose solution from Step 5 to the HiBind® DNA Mini Column. Centrifuge at 10,000 x g for 1 minute at room temperature. Discard the ltrate and reuse collection tube. 
8.Repeat Steps 7 until all of the sample has been transferred to the column. 
9.  Add 300 μL Binding Buffer (XP2). Centrifuge at maximum speed (≥13,000 x g) for 1 minute at room temperature. Discard the ltrate and reuse collection tube. 
10.Add 700 μL SPW Wash Buffer. Centrifuge at maximum speed for 1 minute at room temperature. Discard the ltrate and reuse collection tube. 
11.Centrifuge the empty HiBind® DNA Mini Column for 2 minutes at maximum speed to dry the column matrix. Transfer the HiBind® DNA Mini Column to a clean 1.5 mL microcentrifuge tube. 
12.Add 50 μL deionized water directly to the center of the column membrane. Centrifuge at maximum speed for 1 minute. 
13.Store DNA at -20 °C.</P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1>The double enzyme digestion system</H1><TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center">Components (50 μL) </TH><TH align="center">Volume/μL</TH></TR></THEAD><TBODY><TR><TD align="center"> 10 x buffer  </TD><TD align="center">    5</TD></TR><TR><TD align="center"> Enzyme I </TD><TD align="center">  1</TD></TR><TR><TD align="center"> Enzyme II </TD><TD align="center"> 1</TD></TR><TR><TD align="center"> DNA</TD><TD align="center"> 1-2 μg</TD></TR><TR><TD align="center"> ddH<SUB>2</SUB>O</TD><TD align="center"> Add to 50 μL </TD></TR></TBODY></TABLE><P>Put the system into a 37 °C water bath for half an hour.</P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1>Ligation</H1><TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center"> Components (10 μL)</TH><TH align="center">  Volume/μL</TH></TR></THEAD><TBODY><TR><TD align="center"> T4 DNA ligase </TD><TD align="center"> 1</TD></TR><TR><TD align="center"> 10 × T4 DNA Ligase Buffer </TD><TD align="center"> 1</TD></TR><TR><TD align="center">  Plasmid Skeleton</TD><TD align="center" rowspan="2"> molar ratio of Vector: plasmid  is 1:3</TD></TR><TR><TD align="center"> Insert Gene </TD></TR><TR><TD align="center"> Sterile water</TD><TD align="center">  Up to 10 μL</TD></TR></TBODY></TABLE><P>Overnight at 16 ℃.</P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1>Chemical Transformation</H1><P>1. Take competent cells (E.coli DH5α) from -80 °C refrigerator and put it on ice. （Set negative control by using chemically competent E.coli cells without plasmids） 
2. When the competent cells dissolve (about 10min), add 10 μL DNA ligation product or 2 μL plasmid per tube, Place the mixture on ice for 30 minutes. 
3. Heat shock at 42 °C for exactly 90 seconds. 
4. Put the 1.5 mL tubes back on ice for 3-5 minutes. 
5. Add 500 μL LB fluid medium without antibiotics into the 1.5 mL tubes and then culture in the shaker incubator at 37 °C for an hour. 
6. Extract 100-200 μL bacteria liquid, spread it on LB medium with relevant antibiotic. 
7. Place plates upside down and incubate at 37 °C overnight. </P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1> Colony PCR</H1><P>Using 2×Taq master Mix/2×Hieff PCR Master Mix:<TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center">Reagent </TH><TH align="center">Volume/μL </TH></TR></THEAD><TBODY><TR><TD align="center">  2× Taq master Mix/2×Hieff PCR Master Mix</TD><TD align="center"> 10</TD></TR><TR><TD align="center"> ddH<SUB>2</SUB>O  </TD><TD align="center"> 7</TD></TR><TR><TD align="center"> Primer F(10 uM) </TD><TD align="center"> 1 </TD></TR><TR><TD align="center"> Primer R(10 uM) </TD><TD align="center"> 1</TD></TR><TR><TD align="center"> E. coli colony  </TD><TD align="center"> 1 </TD></TR></TBODY></TABLE></P><P><B>Thermocycling conditions for a Routine PCR:</B></P><TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center">Step </TH><TH align="center">Temperature (℃) </TH><TH align="center">Time </TH><TH align="center"> cycles </TH></TR></THEAD><TBODY><TR><TD align="center"> Initial denaturation  </TD><TD align="center"> 94</TD><TD align="center">  5 min </TD><TD align="center"> 1</TD></TR><TR><TD align="center"> Denaturation  </TD><TD align="center"> 94</TD><TD align="center">  30 seconds</TD><TD align="center" rowspan="3"> 25-35</TD></TR><TR><TD align="center"> Annealing     </TD><TD align="center">  50-60</TD><TD align="center"> 30 seconds</TD></TR><TR><TD align="center"> Extension    </TD><TD align="center"> 72</TD><TD align="center"> 30 sec/kb</TD></TR><TR><TD align="center"> Final extension      </TD><TD align="center"> 72</TD><TD align="center">  1 minute</TD><TD align="center"> 1</TD></TR><TR><TD align="center"> Hold        </TD><TD align="center"> 4</TD><TD align="center"> Indefinitely </TD><TD align="center"> 1</TD></TR></TBODY></TABLE><P>If loading on a gel, don’t need to add loading buffer to the mixture because Taq master Mix/Hieff PCR Master Mix contains loading dye.</P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1> Plasmid extraction</H1><P><B>According to the Plasmid Extraction Mini kit</B></P><P>1. Take 1-5 mL bacterial solution into a centrifuge tube, centrifuge at 12,000 rpm for 1 min and remove supernatant. 
2. Add 250 μL SolutionⅠ in centrifuge tube, using the pipet or vortex oscillator to suspend the cells. 
3. Add 250 μL Solution II in centrifuge tube and gently flip upside down for 6-8 times to make sure the germ is full cracked. 
4.Add 350 μL Solution III, gently flip upside down for 6-8 times to mix until white, flocculent precipitate appears and centrifuge at 12,000 rpm for 10 minutes. 
5. Add the supernatant to the adsorption column in step 5, centrifuge at 12,000 rpm for l minute, discard the filtrate and reuse collection tube. 
6.Add 700 μL Wash solution to the adsorption column, centrifuge at 12,000 rpm for l minute, discard the filtrate and reuse collection tube. 
7.Add 500 μL Wash solution to the adsorption column, centrifuge at 12,000 rpm for l minute, discard the filtrate and reuse collection tube. 
8.Centrifuge the empty adsorption column at 12,000 rpm for 2 minute to dry the column matrix. (Residual ethanol may impact downstream application) 
9.Transfer the adsorption column into a clean 1.5 mL centrifuge tube, add 50 -100 μL Elution buffer to the center of the column membrane, let sit at room temperature for 2 minutes and centrifuge at 12,000 rpm for l minute, collect the plasmid solution in the centrifuge tube. 
10. Store the plasmid at -20 °C.</P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1>LB medium(For 100 mL)</H1><TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center">Component</TH><TH align="center"> Mass/g </TH></TR></THEAD><TBODY><TR><TD align="center"> Tryptone </TD><TD align="center"> 1</TD></TR><TR><TD align="center">  Yeast Extract </TD><TD align="center"> 0.5</TD></TR><TR><TD align="center"> NaCl  </TD><TD align="center"> 1 </TD></TR><TR><TD align="center">  Sterile water  </TD><TD align="center"> to 100 mL</TD></TR></TBODY></TABLE><P>
Autoclave  at 121 °C for 20 min.
(1.5 g agar should be added before autoclaving to make solid LB)</P></DIV></DIV><DIV class="main-content"><DIV class="textblock"><H1>CloneExpress</H1><P>According to the CloneExpress Multis One Step Cloning Kit:
1).Preparation for the linearized cloning vectors；
Select appropriate cloning sites, and linearize the cloning vector.The cloning vectors can be linearized by restriction digesting with endonuclease or by reverse PCR amplification. 
2).Design of PCR primers of the insertions；
The principle for the design of ClonExpress® MultiS primers is: introduce homologous sequences (15 bp～20 bp)into 5’ end of primers, aiming to making the ends of amplified insertions and linearized cloning vector identical to the ends of their neighbours which is required for recombination reaction. 
3).PCR amplification of the insertions；
Insertions can be amplified by any polymerase (Taq DNA polymerase or other high-Fidelity polymerases),but to prevent mutations introduced during PCR, high-Fidelity polymerases is highly recommended.
4).Recombination reaction；
Set up the following reaction on ice. Spin briefly to bring the sample to the bottom before reacting.</P><TABLE style="width: 75%; margin: 0px 14% 20px 14%;"><THEAD><TR><TH align="center">Component</TH><TH align="center"> Mass/g </TH></TR></THEAD><TBODY><TR><TD align="center">  ddH<SUB>2</SUB>O  </TD><TD align="center"> Up to 20 μL</TD></TR><TR><TD align="center">  5×CE MultiS Buffer  </TD><TD align="center"> 4 μL</TD></TR><TR><TD align="center"> Linearized cloning vector </TD><TD align="center">  x ng</TD></TR><TR><TD align="center">  PCR products of insertions</TD><TD align="center">  y ng</TD></TR><TR><TD align="center"> Exnase® MultiS </TD><TD align="center">  2 μL</TD></TR></TBODY></TABLE><P>5).Transformation and plating；
6).Selection of the positive colony . </P></DIV></DIV><TITLE>footer</TITLE><DIV id="footer"><DIV class="footer-col" id="Acknowledgements"><SPAN class="footer-title">Acknowledgements</SPAN><DIV class="footer-row"><P>Nanjing Agricultural University</P></DIV><DIV class="footer-row"><P>College of Life Sciences of Nanjing Agricultural University</P></DIV><DIV class="footer-row"><P>Nanjing University</P></DIV><DIV class="footer-row"><P>Chinese Academy of Sciences</P></DIV></DIV><DIV class="footer-col" id="Useful_Links"><SPAN class="footer-title">Useful Links</SPAN><P><A href="https://2018.igem.org/">Main Page</A></P><P><A href="https://igem.org/Special:SpecialPages">Special Pages</A></P></DIV><DIV class="footer-col" id="ContactUs"><SPAN class="footer-title">Contact Us</SPAN><DIV class="footer-row"><P>Facebook</P></DIV><DIV class="footer-row"><P>Twitter</P></DIV></DIV><DIV class="footer-col" id="About Us"><SPAN class="footer-title">About Us</SPAN><P><SPAN style="font-size:1.1vw">NAU-CHINA 2018</SPAN> from 
                Nanjing Agricultural University 
                NO.1 Weigang, Xuanwu District
                Nanjing, CHINA
                210095
            </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>