<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tuebingen_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tuebingen/Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>Project</P><LI><A href="https://2018.igem.org/Team:Tuebingen/Team"><P>Team</P></A></LI><LI><A href="https://2018.igem.org/Team:Tuebingen/Labwork"><P>Labwork</P></A></LI><LI><A href="https://2018.igem.org/Team:Tuebingen/Bioinformatics"><P>Bioinformatics</P></A></LI><LI><A href="https://2018.igem.org/Team:Tuebingen/Human_Practices"><P>Human Practices</P></A></LI><LI><A href="https://2018.igem.org/Team:Tuebingen/Collaborations"><P>Collaborations</P></A></LI><LI><A href="https://2018.igem.org/Team:Tuebingen/Demonstrate"><P>Results</P></A></LI><DIV id="mainContainer"><DIV id="mainHead" class="section"><H1><SPAN id="includedInitial">C</SPAN>ollaborations</H1><DIV id="meaningfulQuote">None of us, including me, ever do great things. But we can all do small things, with great love, and together we can do something wonderful.<SPAN class="quoteAttribution">- Mother Teresa</SPAN></DIV></DIV><DIV id="Introduction" class="section"><DIV class="sectionHeading"><DIV class="sectionHeadingText">Introduction</DIV></DIV><DIV class="singleContent">
One of the silver medal requirements is to include a team collaboration in an significantly way. This is because on iGEM, you have the unique opportunity to work with like-minded people all around the world on a project. You can influence the work of others in many ways, starting with meet ups to discuss problems and ideas and ending with helping them to complete an important task in their project. Over the year, we had different collaborations listed below.  
</DIV></DIV><DIV id="Carroll iGEM" class="section"><DIV class="sectionHeading"><DIV class="sectionHeadingText">Carroll iGEM</DIV></DIV><DIV class="multiContent"><DIV class="half">
In July, the Carroll iGEM Team introduced us to an opportunity to collaborate with them on Instagram. We made a picture with Mike the Microbe who visited us in our Chemistry Lab. Mike is a cartoon friendly bacteria and iGEM teams all over the world hosted him in their labs. We were very happy to had him in our lab.
</DIV><DIV class="half"><FIGURE><FIGCAPTION>Mike in our Chemistry Lab</FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV id="Meet-Up" class="section"><DIV class="sectionHeading"><DIV class="sectionHeadingText">Meet-Up</DIV></DIV><DIV class="singleContent"><STRONG>German Meet Up</STRONG>
During the German Meet Up in Marburg we talked with many German iGEM teams about their projects, comparing different approaches and meeting nice people from all around Germany.
</DIV><DIV class="singleContent"><STRONG>European Meet Up</STRONG>
The European Meet Up in Munich helped us to improve our presentation skills, exchanging experiences associated with the progress of our work and listening to intriguing talks. 
</DIV><DIV class="singleContent"><STRONG>Oslo</STRONG>
In May, we had the opportunity to visit iGEM Oslo, since one of their members is an exchange student from Tübingen. We had very nice talks and an excellent workshop with the communications which lead to the development of plenty of good ideas, like the Ract! Festival or our public debate. 
</DIV><DIV class="singleContent"><STRONG>Bielefeld</STRONG>
We visited iGEM Bielefeld during our attendance of the Hanovermesse, enjoying a tour of their laboratory and exchanging ideas. 
</DIV><DIV class="singleContent"><STRONG>HyperpodX</STRONG>
The HyperpodX Team Oldenburg gave us insight in a very different approach of a big project. 
</DIV><DIV class="singleContent">
For more information, you can visit our
<A href="https://2018.igem.org/Team:Tuebingen/Human_Practices">Human Practices</A> site
</DIV></DIV><DIV id="BioInfo" class="section"><DIV class="sectionHeading"><DIV class="sectionHeadingText">BioInfo</DIV></DIV><DIV id="Introduction" class="subSection"><H2>Introduction</H2><DIV class="singleContent">
In order to test our deimmunization tool BERT in the context of a meaningful application, we collaborated with the <A href="https://2018.igem.org/Team:Paris_Bettencourt">iGEM Team Paris-Bettencourt</A>. Their project focused on the usage of antimicrobial peptides fused together with self-assembling scaffolding proteins as a potential  alternative to classical antibiotic treatment. 
Since biologicals (biopharmaceuticals) often have the problem of being targeted by the immune system through the prescence of specifc anti-drug anitbodies (ADA), we performed our deimmunizaton workflow BERT on a construct of the iGEM team Paris-Bettencourt. A low immunogenicity would be crucial for the application of AMP/scaffolding protein constructs, since in this case binding of ADAs  to the protein would most likely severely impair the self-assembly properties to large complexes. 
Moreover, this was also a &quot;blind test&quot; for our deimmunization tool, since we applied the steps on a protein which was completely unknown to us. Had we solely focused on our own botulinum toxin shuttle with the aim of optimizing the steps of the workflow, there would have been the risk of losing the general applicability of the workflow on arbitrary proteins.
A further important aspect of this collaboration, was the validation of the structural integrity of the fusion protein we used in our deimmunization workflow. In this external validation we investigated the influence of the AMP on the tertiary structure of the scaffolding protein using MD simulations. Since we have access to the required hardware to run prolonged MD simulation, we were able to share our expertise on this subject and our computational capabilities with the iGEM Team Paris-Bettencourt. Furthermore since both our teams were using MD simulations to model the structure of the proteins in our projects, this collaboration allowed us to exchange knowledge about this topic and help us in the almost never ending process of troubleshooting while simulating complex biological problems.
For the application of our deimmunization workflow BERT on an AMP/scaffold protein, Paris-Bettencourt provided us with the sequences of their proteins. We focused our approach on a ovispirin/l-fucose Mutarotase fusion protein, for multiple reasons. Firstly, this protein has very prominent clusters of epitopes for the three most prevalent MHC class II alleles in central Europe (DRB1*03:01, DRB1*07:01, DRB1*15:01), which allows for very effective reduction in the number of immune epitopes. Secondly, we were able to find many closely and distantly related proteins, leading to an excellent identification of highly conversed residues by our tool, which are in turn avoided in the mutagenesis process. Lastly this fusion protein is of smaller size, which allowed us to include longer MD simulations in our tight schedule for simulation on our hardware.
</DIV></DIV><DIV id="StructuralAnalysis" class="subSection"><H2>Structural Analysis</H2><DIV class="singleContent">
In the initial step of the deimmunization workflow, we required a stable protein structure to be able to  evaluate the influence of possible point mutations. For this purpose, we performed two 50 ns MD simulations, the first using the isolated l-fucose Mutarotase (PDB ID: 2WCV) domain and the second the l-fucose isomerase/Ovispirin (PDB ID: 1HU5) fusion protein. 
To simulate the proteins of our collaboration partner, we used the MD simulation software GROMACS, as it is a widely used tool with an excellent performance. For the force field we chose AMBER99SB, due to the fact that it is generally applicable and well established.
</DIV><DIV class="multiContent"><DIV class="half"><FIGURE><FIGCAPTION>Figure 2.1 Structure of l-fucose Mutarotase after a MD simulation of 50 ns</FIGCAPTION></FIGURE></DIV><DIV class="half"><FIGURE><FIGCAPTION>Figure 2.2 RMSD plot of the l-fucose mutarotase: The RMSD has been calculated on the coordinates of the backbone atoms</FIGCAPTION></FIGURE></DIV></DIV><DIV class="multiContent"><DIV class="half"><FIGURE><FIGCAPTION>Figure 3.1 Structure of l-fucose mutarotase-ovispirin fusion protein after a MD simulation of 50 ns. The ovispirin AMP is the helical-like terminal domain indicated in red in picuture.</FIGCAPTION></FIGURE></DIV><DIV class="half"><FIGURE><FIGCAPTION>Figure 3.2: RMSD plot of the l-fucose mutarotase-ovisprin fusion protein: The RMSD has been calculated on the coordinates of the backbone atoms</FIGCAPTION></FIGURE></DIV></DIV><DIV class="singleContent"><FIGURE><FIGCAPTION>Figure 4: RMSD plot of the isomerase-domain of the L-fucose mutarotase-ovisprin fusion protein:The RMSD has been calculated on the coordinates of the backbone atoms of the isomerase domain</FIGCAPTION></FIGURE></DIV><DIV class="singleContent">
The simulation of the FucU isomerase protein for 50 ns without AMP shows a RMSD of about 0.25 nm which indicates some minor conformational changes in the beginning of the simualtion, stabilizing in the last 20 ns of the simulation (Figure 2.1/2.2).
The simulation of the FucU isomerase together with the AMP ovispirin as subunit shows more drastic changes to the tertiary structure of the fusion protein. The RMSD value reaches a value of 0-7-0.8 nm in the time span of 50 ns. This can be explained by the fact, that we manually concatenated the isomerase protein with the crystal structure of the ovispirin protein, which in this initial fusion state is sub-optimally folded. The ovispirin domain undergoes some conformational changes, since it is now connected to the larger isomerase domain Figure 3.1/3.2).
We also analyzed the RMSD value over the time of the simulation only for the isomerase domain, since the iGEM Team Paris-Bettencourt was particularity interested in the influence on this domain while being fused to the ovispirin. The plot indicates some minor changes, but the conformation remains stable overall, without the ovispirin domain interfering heavily with the isomerase domain (Figure 4).
</DIV></DIV><DIV id="EpitopeRemoval" class="subSection"><H2>Epitope Revomal</H2><DIV class="singleContent">
The second step of our workflow was the identification of the T-cell MHC class II immune epitopes for the three most prevalent alleles in central Europe and their removal by specific amino acid substitutions, which do not alter the tertiary structure of the protein.
In our deimmunization workflow, we focused on removing the immune epitopes within the isomerase domain, since there are barely any known homologous of the ovispirin protein. This lack of related protein drastically decreases the accuracy of our deimmunization tool and chances are high to get a structurally unstable and nonfunctional protein. That’s why the following epitope related data are for the isomerase domain only.
Initially, we identified a total of 126 immune epitopes for the three alleles mentioned above in the isomerase domain. Between residue 100-130, the isomerase domain has a cluster of amino acids which are part of many different epitopes. This makes the isomerase domain a good candidate for a deimmunization, since the substitution of single amino acids in this cluster, can destroy a large number of epitopes.
Our algorithm calculated the following three amino acid substitution, as a compromise between the maximum structural integrity and the minimal immunogenicity of the protein (Table 1).
These three substitutions reduce the number of immune epitopes to 85, which is a reduction by ~33%.
</DIV><DIV class="singleContent"><TABLE><CAPTION>Table 1 Computed amino acid point mutations in the L-fucose mutarotase protein to reduce the immunogenicity.</CAPTION><TBODY><TR><TH>Mutation</TH><TH>Number of epitopes after mutation</TH><TH>Predicted ddG value [kcal/mol]</TH></TR><TR><TD>L 49 C</TD><TD>113</TD><TD>-1.518</TD></TR><TR><TD>I 103 D</TD><TD>95</TD><TD>-2.009</TD></TR><TR><TD>K 116 D</TD><TD>85</TD><TD>0.032</TD></TR></TBODY></TABLE></DIV><DIV class="singleContent">
The changed immune epitope count of serveral of the individual amino acids in Figure 5.1 and Figure 5.2 shows, that especially the clustered regions with many epitopes were edited by our algorithm. This can be explained by the fact that a single mutation in this area has probably the biggest impact on the immunogenicity.
</DIV><DIV class="multiContent"><DIV class="half"><FIGURE><FIGCAPTION>Figure 5.1 T-Cell immune epitopes before the deimmunization workflow shown as combined bar-/lineplot. For each amino acid position, the number of MHC class II epitopes  is plotted. Note that there is a prominent cluster of amino acid between residue 100-130 which are part of many immune epitopes.</FIGCAPTION></FIGURE></DIV><DIV class="half"><FIGURE><FIGCAPTION>Figure 5.2 T-Cell immune epitopes after the deimmunization workflow shown as combined bar-/lineplot. For each amino acid position, the number of MHC class II epitopes  is plotted.</FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV id="StructuralVerification" class="subSection"><H2>Structural Verification</H2><DIV class="singleContent">
To validate whether the change of amino acids at these three positions still results in a stable protein, we did another MD simulation of 25 ns of the mutated protein with and without the fusion domain of the structures of previous MD simulations.
Both RMSD plots show a RMDS of &lt; 0.15 nm, which is a very stable conformation regarding the introduction of three point mutations (Figure 6.1/6.2). All three amino acids have their side chains oriented outwards relative to the protein core (Figure 7) and do not interfere with any present secondary structures of the protein or any other amino acids. This also suggests that our algorithm made a good choice regarding the substituted residues. 
</DIV><DIV class="multiContent"><DIV class="half"><FIGURE><FIGCAPTION>Figure 6.1: RMSD plot of the mutated L-fucose mutarotase protein: The RMSD has been calculated on the coordinates of the backbone atoms.</FIGCAPTION></FIGURE></DIV><DIV class="half"><FIGURE><FIGCAPTION>Figure 6.2 Figure 6.1: RMSD plot of the mutated L-fucose mutarotase-ovisprin fusion protein: The RMSD has been calculated on the coordinates of the backbone atoms of the isomerase domain.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="multiContent"><DIV class="half"><FIGURE><FIGCAPTION>Figure 7: Structure of the L-fucose mutarotase-ovispirin fusion protein after the substitution of the three amino acids and a 25 ns MD simulation. In the image the three amino acids are labeled.</FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV id="Discussion" class="subSection"><H2>Discussion</H2><DIV class="singleContent">
All in all this collaboration allowed us to test our workflow BERT in a meaningful iGEM related context and we were able to provide the iGEM-Team Paris-Bettencourt with an epitope reduced sequence and a corresponding stable protein structure. Moreover, we could help Paris-Bettencourt to verify the structural integrity of their isomerase protein when fused to the ovispirin AMP. Besides the modeling, we were happy to get the opportunity to discuss some problems we had with or MD simulations with another team that is also using MD simulation to model parts of their project.
</DIV></DIV></DIV></DIV><DIV id="sideBar" class="initiallyCollapsed"><DIV id="excludedInitial" class="collapsible initiallyCollapsed">C</DIV><UL id="sideBarNavigation" class="collapsible"><LI><A onclick="linkClicked()" href="#Introduction">Introduction</A></LI><LI><A onclick="linkClicked()" href="#Carroll iGEM">Carroll iGEM</A></LI><LI><A onclick="linkClicked()" href="#Meet-Up">Meet-Up</A></LI><LI><A onclick="linkClicked()" href="#BioInfo">BioInfo</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>