Team:GO Paris-Saclay/labnotebook-ler
03/07
PCR of LER
Arthur
The plasmid pKK-Ler was taken from Hervé’s database. A PCR Phusion was performed to amplify Ler.
Primer: GO2018-7 and GO2018-8
Gel 1
A PCR clean up was performed afterward and the concentration was 53, 74 ng/µL
04/07
Digestion pSB1C3-TetO and Ler
Arthur, William Britany
Component
Volume (μL)
pSB1C3 – TetO
10
SpeI
1
PstI
1
NEB buffer 2.1
2
H2O
6
Component
Volume (μL)
RBS-Ler
10
XbaI
1
PstI
1
NEB buffer 2.1
2
H2O
6
Fragment
Enzymes
pSB1C3-TetO
SpeI/PstI
RBS-Ler
XbaI/PstI
The digestion mix was then incubated at 37°C for 2 h.
DNA purification (PCR clean up)
Component
Concentration (ng/μL)
pSB1C3 – TetO
13, 74
RBS-Ler
2, 82
5/07
Ligation pSB1C3-TetO-Ler and pSB1C3-Ler
Yueying, William
We are aiming for a molar ratio insert:vector of around 1:3
Insert
Vector
T4 buffer x10
T4 ligase
Ligation 1
Ler :
0, 6 kb /C= 13, 74 ng/uL
→ 1, 5 μL
pSB1C3-TetO :
3 kb/C= 7 ng/μL
→ 5 μL
1
1
Ligation 2
Ler :
0, 6 kb/ C= 13, 74 ng/uL
→ 1, 5 μL
pSB1C3:
3 kb/ C= 7 ng/μL
→ 5 μL
1
1
For a total of 10 uL each.  Then incubate the tubes at room temperature for 2 h.
Transformation of pSB1C3-TetO-Ler and pSB1C3-Ler
Britany, Clémence
DH5α cells were transformed with the 2 ligations, following standard transformation protocol by heat shock. A negative control (cells without any plasmid) was also done.
Results: No colonies on the negative control. There were 4 colonies for the pSB1C3-TetO-Ler, about 10 for the pSB1C3-Ler
Component
Volume (μL)
pGEMTeasy
10
HincII
1
Tango buffer
2
H2O
7
Digestion pGEMTeasy
Kenn, Julie M
Fragment
Enzymes
pGEMTeasy
HincII
The digestion mix was then incubated at 37°C for 2 h.
pGEMTeasy purification (PCR clean up)
Component
Concentration (ng/μL)
pGEMTeasy
23, 5 ng/μL
Kenn, Julie M
Ligation pGEMT-ler et pGEMT-CPG2
Kenn, Julie M
Insert
Vector
T4 buffer x10
T4 ligase
Ligation
Ler:
0, 6 kb /C= 23, 1 ng/uL
→ 1 μL
pGEMT easy:
3 kb/C= 53,74 ng/μL
→ 3 μL
1
1
The mix was left in the fridge overnight
06/07
Colony PCR check
Céline, Arthur, William
Colony PCRs were performed with, verification primers VR and VF2 which flank the biobrick insertion site. They amplify the insert from both parts of the vector pSB1C3.
Gel 2
Expected size of the fragments: 800 bp
Results: For the pSB1C3-TetO-Ler there were only empty vectors and for the pSB1C3-Ler the clones have the expected size.
06/07
Transformation pGEMT-LER
Yueying
DH5α cells were transformed with the ligation, following standard transformation protocol by heat shock. A negative control (cells without any plasmid) was also done).  Cell culture was spread on two LB IPTG X-gal Ampicillin plates
08/07
Results: There are plenty of clones for the ligations pGEMT-easy-LER but most of them are blue (which means closed vector). 5 white clones for each were put in LB 4mL Ampicillin, X-gal, IPTG overnight at 37°C, for extraction by the kit (and to be sent for sequencing if they contain the insert).
09/07
Plasmid Extractions of pGEMt-Easy LER and pSB1C3-LER
Kenn William
All the extractions were realised with the extraction kit (Nucleospin plasmid).
 We also noticed that the pGEMt easy Ler 5 were slightly blue during the extraction.
Nanodrop Results:
Component
Concentration (ng/μL)
A 280/260
pSB1C3 Ler 1
14
1,87
pSB1C3 Ler 2
20
1,82
pGEMt easy Ler 1
40
1.82
pGEMt easy Ler 2
61
1.76
pGEMt easy Ler 3
31
1.79
pGEMt easy Ler 4
34
1.81
pGEMt easy Ler 5
41
1.85
pGEMT easy ler-1 and 2 and pSB1C3 ler-1 and 2 were sent for sequencing
Digestion of the extracted plasmid
Kenn
Component
Volume (μL)
pGEMTeasy-Ler
4
EcoRI
0,5
FD buffer
1
H2O
4, 5
Component
Volume (μL)
pSB1C3-Ler
4
NotI
0,5
FD buffer
1
H2O
4, 5
 
 
Gel 3
Expected size: pGEMTeasy-ler:3,5 kb/ pGEMTeasy empty:3 kb
Result: Since the plasmid has the expected size, the insert, we expect that the clones sent for sequencing contain the insert. Sequencing will confirm whether the whole sequence is as expected.
Sequencing analysis
Concerning the two clones: pSB1C3-ler 1 and 2, the sequences are correct for both, but there is an extra sequence between the last nucleotide of the RBS and the first nucleotide of the ATG:
CATATGTAGATGTGTCCTAATTTGATAGATAAACGTTATCTCACATAATTTATATCATTTGATTAATTGTTGTCCTTCCTGATAAGGATAAGGTCGCTAATAGCTTAAAATATTAAAGC
This sequence corresponds to the native RBS (and more), it comes from the original DNA of the protein. The plasmid we used as a matrix was not correctly sequenced. Most likely, it would work, but since its complicated we will start again from the gBlock we received (the one with the RBS).
Concerning the two clones pGEMt-easy- ler 1 and 2: idem, the sequence is correct, but there is this extra piece of DNA. Still, it is reassuring the confirm that it has been cloned in pGEMt-easy. However, the surrounding plasmid does not seem correct. Anyway, we will not be using it since it has this extra DNA.
12/07
Ligation gBlock RBS LER in pGEMT easy
William
We ligated the gBlock received containing prefix-rbs-ler-suffix, in pGEMT easy, as a second source of ler sequence.
Insert
Vector
T4 buffer x10
T4 ligase
Ligation
Ler:
0, 6 kb /C= 10 ng/uL
→ 2 μL
pGEMT easy:
3 kb/C= 53,74 ng/μL
→ 6 μL
1
1
Transformation gBlock RBS LER in pGEMT easy
William
Heat shock transformation of DH5alpha with gBlock RBS LER in pGEMTeasy, following standard procedure.
Plated on LB Xgal IPTG Amp plates made the same day. The inadvertent presence of blue loading dye in the ligation mix was reported.
Overnight culture: colony a bit small, put back 3 hours then to the fridge for blue-white screen improvement.
16/07
Liquid cultures of pGEMT easy RBS ler
Kenn
Liquid culture of white clones of pGEMT easy RBS ler was launched.
17/07
Extraction pGEM-t-easy / RBS Ler
Clémence et Britany
Extraction were performed from 3 mL of the 16/07 culture (clones 1 to 8). Elution were done in 50 μL buffer AE.
Digestion of pGEM-t-easy / RBS Ler
Clémence et Britany
Component
Volume (μL)
pGEMTeasy-Ler
3
NotI
1
FD buffer
1
H2O
5
24/07
Gel verification
Yueying
The digestion was a success for the plasmids. We putted all the pGEMt-easy/RBS-ler digestion on a gel in order to purify the 500 pb signal stripe.
Gel purification of pGEMTeasy/RBS-Ler
Yueying
Component
Concentration (ng/μL)
RBS-Ler
3,5-5
Digestion of pSB1C3 TetO and RBS-Ler
Component
Volume (μL)
Component
Volume (μL)
pSB1C3-TetO
10
RBS-Ler
10
Pst1
1
Pst1
1
Spe1
1
Xba1
1
NEB buffer 2.1
2
NEB buffer 2.1
2
H2O
6
H2O
6
Fragment
Enzymes
pSB1C3-TetO
SpeI/PstI
RBS-Ler
XbaI/PstI
Gel verification
William
Gel
30/07
Ligation of pSB1C3 TetO and RBS-Ler
Julie M, William
Insert
Vector
T4 buffer x10
T4 ligase
H2O
Ligation
Ler:
0, 6 kb
→ 3 μL
pSB1C3-TetO:
3 kb
→ 1 μL
1
1
4
Transformation of pSB1C3-TetO/RBS-Ler in DH5alpha
We followed the standard procedure
Results: There were plenty of clones
31/07
