Team:GO Paris-Saclay/labnotebook-lee5
Cloning the Promotor of lee5 from the vector pKK-lee5-gfp in to the vector pGEMt-easy (first try, without the colonies, 25/06-27/06).
25/06-26/06 by Celine
Amplification of the promotor of lee5 fragment by PCR (see Dream taq protocol).
PCR Results : obtention of DNA fragment with expected size.
well 1: Gene ruler DNA ladder
well 2: lee5 promotor PCR product with standard buffer
well 3: lee5 promotor PCR product with green buffer
The lee5 promotor PCR products were putted on gel. The DNA matrix used is pKK-Lee5-GFP, the enzyme used is polymerase DreamTaq, the primers used are GO2018-5 and GO2018-6 and the expected size of PCR products is 534 bps.
 
26/06 by Clémence
Digestion of DNA matrix with Dpn1 (see digestion protocol) and purification of lee5 PCR products (see PCR clean up protocol) .
DNA concentration (Nanodrop) : 239,092 ng/µL.
Ligation lee5 in plasmid pGEM-T Easy with T4 ligase (see T4 ligation protocol).
Heat-shock transformation of 2 µL of the ligation product in 50 µL DH5alpha plated on LB IPTG X-gal Ampicillin plates, left overnight at 37°C (see Heat-shock transformation protocol).
27/06 by Kenn
Transforming results: 10 colonies.  
Cloning the Plee5 in to the vector pSB1C3 (second try 28/06-/06).
28/06 by Kenn
Digestion of LEE5 PCR products(from 26/06) and pSB1C3-rfp by XbaI and SpeI (see digestion protocol).
Sequence of LEE5 PCR products: promotor of LEE5  XbaI site (top) and SpeI site (bottom) 
After digestion, LEE5 PCR products are purified (see PCR clean up protocol).
Results of measurement by Nanodrop: DNA concentration of digested Lee5-gfp  (Nanodrop) : 25 ng/µl
After digestion, pSB1C3-rfp is purified on gel (see purification on gel protocol).
well 1: Gene ruler DNA ladder
well 3: Digested pSB1C3-RFP
The digested pSB1C3-rfp were purified on gel. The band at 2kb (corresponding to pSB1C3 opened without rfp) was cut from the gel under UV and purified. The band at 1kb corresponds to the rfp fragment, and the band at 3 kb to the undigested plasmid.
DNA concentration of digested pSB1C3 (Nanodrop):   14 ng/µl
29/06/18 by Kenn
Ligation lee5 into pSB1C3 with T4 ligase (see T4 ligation protocol).
Digestion of pSB1C3-RBS-GFP by XbaI.
Ligation of LEE5 promotor in pSB1C3-RBS-GFP with T4 ligase.
02/07/18 by Britany and Julie R.
Transformation of ligation pSB1C3-LEE5 and pSB1C3-LEE5-gfp in DH5alpha (see Heat-shock transformation protocol).
Verification Digestion of pGEMT easy-LEE5 by digestion of EcoRI and SpeI (EcoRI to check the presence of the insert, and SpeI to know the orientation).
Digestion  using EcoRI
Expected Results: 500 et 3 000 pb
Results: Clones 1, 2, 6, 7, 8, 9 are good
Digestion using SpeI
Expected results : 500 et 3 000 pb
Results: Clones 6, 9 are good
According to the previous results, we decided to keep clones 6 and 9 (white-blue) for the future experimentation. These clones could be sent for sequencing.
03/07/18 by Kenn and Julie M.
Transforming results:
pSB1C3-LEE5-gfp : 2 clones 
pSB1C3-LEE5: a few doze clones
Colony PCR to check for the presence of Lee5 in pSB1C3-LEE5-gfp.
well 1 and 13: Gene ruler DNA ladder
well 2 to 3: lee5–gfp PCR product
well 5 to 13: lee5 promotor PCR product
The lee5 promotor PCR products were putted on gel. The DNA matrix used is the colonies, the enzyme used is polymerase DreamTaq, the primers used are VF2 and GO2018-6 and the expected size of PCR product is 300 bp for closed pSB1C3, for  800 pb for pSB1C3-LEE5,  1,2 kb for closed pSB1C3-gfp and 1,7 for pSB1C3-LEE5.
We have made a mistake of the concentration of primer that we have used.
04/07/18 by Céline
Colony PCR to check for the presence of Lee5 in pSB1C3-LEE5-gfp (seconde try).
well 1 and 13: Gene ruler DNA ladder
well 2 to 3: lee5–gfp PCR product
well 5 to 13: lee5 promotor PCR product
The lee5 promotor PCR products were putted on gel. The DNA matrix used is the colonies, the enzyme used is polymerase DreamTaq, the primers used are VF2 and VR and the expected size of PCR product is 300 bp for closed pSB1C3, for 800 pb for pSB1C3-LEE5,  1,2 kb for closed pSB1C3-gfp and 1,7 for pSB1C3-LEE5-gfp.
Conclusion all the Plasmids are closed, this experiment has to be repeated.
Hypothesis: Maybe the Lee5 digestion did not worked perfectly
Digestion of LEE5 PCR products(from 26/06) with XbaI, SpeI.
05/07/18 by Yueying, William, Britany and Clémence
Ligation of LEE5 promotor in pSB1C3-RBS-GFP and into pSB1C3 with T4 ligase.
Transformation of ligation pSB1C3-LEE5 and pSB1C3-LEE5-gfp in DH5alpha (see Heat-shock transformation protocol).
06/07/18 by Julie, Arthur
Plasmid DNA purification of pGEMt-Easy-LEE5p
Using [NucleoSpin Machery-Nagel] :
pGEMT easy, pGEMT easy-Lee5 P6, pGEMT easy-Lee5 P9 according to the kit.
pGEMT easy: 103, 889 ng/uL    /260/280: 1, 88
pGEMT easy-Lee5P 6: 50, 422 ng /uL   /260/280: 1, 88
pGEMT easy-Lee5P 9: 90, 478 ng/uL  /260/280: 1, 85
LEE5P 6 and LEE5P 9 were sent for sequencing (order COL18-0506, with oligo SP6)
Sequencing results :
1pb wa different between the silico version of pGEMt easy Lee5p and the real one (pGEM t easy Lee5p 6), and the number 9 clone is completely identical to the expected sequence.
For the next step we will keep and use the clone number 9.
06/07/18 by Céline, Arthur and William
Transforming results:
no growth of any colonies for  pSB1C3-LEE5.
More than 20 colonies for pSB1C3-LEE5-gfp
Colony PCR to check for the presence of Lee5 in pSB1C3-LEE5-gfp.
well 1 and 16: Gene ruler DNA ladder
well 2 to 12: lee5–gfp PCR product
The lee5 promotor PCR products were putted on gel. The DNA matrix used is the colonies, the enzyme used is polymerase DreamTaq, the primers used are VF2 and VR and the expected size of PCR product is 1,2 kb for closed pSB1C3-gfp and 1,7 for pSB1C3-LEE5-gfp.
Results: the clone 4 and the clone 8 are good.
08/07/18 by Céline
However, for pSB1C3-LEE5-gfp, the orientation must be checked, since it could be inserted in the wrong orientation.
A new PCR on colony was made for this verification.
 
well 1, 13, 14 and 27: Gene ruler DNA ladder
well 2 to 12 and 15 to 26: lee5–gfp PCR product
The lee5 promotor PCR products were putted on gel. The DNA matrix used is the colonies, the enzyme used is polymerase DreamTaq, the primers used are VF2 and GO2018-6 and the expected size of PCR product is 660bp only if the orientation is correct.
There are suspicious faint bands nearly everywhere. We extract clone 4 , 8, 13 and 22 to confirm.
09/07 by Xavier, Kenn and  William
Plasmid Extractions of pSB1C3-LEE5-gfp clone 4 and 8.
Digestion of lee5 pGEM t easy clone 9 and PCR clean up.
We purified by Electrophoresis (25 min 100V)   and use the GEL & PCR CLEAN UP kit:
We used 108µL of NTI buffer.
Result: Dna concentration: 14,837 ng/µL                 with 280/260= 1,51
09/07 by kenn
Extraction of PSB1C3- Lee5-GFP clone 13and 22
Digestion of PSB1C3- Lee5-GFP clone 13and 22  by NdeI and XBaI
well 1: Gene ruler DNA ladder
well 2 to 5: clone 4, 8, 13, 22
We can clearly see in the wells with clone 4 and clone 8 one big signal strip at 3500pb and another one at less than 500pb . This indicates that The insertion of Lee5 in the pSB1C3 RBS GFP plasmid was made in the wrong direction.
In the well were the clone 13 extract was added, we can clearly see 2 signal strips again, but this time at 3000pb and 900pb. This indicate us that the insertion was made in the right direction.
For the Well with the number 22, we cannot conclude because there is a big signal at 3500pb and 3000pb, may be the lee5 insertion is in two direction.
From these results we decided to send 20ul of the pSB1C3-Lee5-GFP number 13  for sequencing.
Since new clones were appear on petri dish,  we did anther Colony PCR to find more candidate of Lee5 in pSB1C3-LEE5-gfp.
well 1 and 15: Gene ruler DNA ladder
well 2 to 14 : lee5–gfp PCR product
The lee5 promotor PCR products were putted on gel. The DNA matrix used is the colonies, the enzyme used is polymerase DreamTaq, the primers used are VF2 and GO2018-6 and the expected size of PCR product is 800bp.
Resultat: the clone 3 is good.
13/07/18 by Guillaume William
Plasmid Extractions of pSB1C3-LEE5-gfp clone 3.
DNA concentration (Nanodrop) : 16 ng/µL.
Verification digest
pSB1C3-lee5-gfp clone3 was digest by XbaI, NdeI
well 1 : Gene ruler DNA ladder
well 2 : lee5–gfp PCR product  
Only clone of lee5 (clone 3) was sent for sequencing because it has de good size (4kb).
To have more candidate, we have redone the ligation (lee5 in pSB1C3-gfp).
16/07/18 by  William
Transformation of ligation of  pSB1C3-LEE5-gfp in DH5alpha (see Heat-shock transformation protocol).
17/07/18 by  William
Transforming results:
More than 20 colonies for pSB1C3-LEE5-gfp
18/07/18 by  William and kenn
Colony PCR to check for the presence of Lee5 in pSB1C3-LEE5-gfp.
well 1 and 14: Gene ruler DNA ladder
well 2 to 13 : lee5–gfp PCR product
The lee5 promotor PCR products were putted on gel. The DNA matrix used is the colonies, the enzyme used is polymerase DreamTaq, the primers used are VF2 and GO2018-6 and the expected size of PCR product is 800bp.
Result : the clone 8 and 10 are good.
Extraction of pSB1C3-LEE5-gfp clone 8 and 10
DNA
pSB1C3-LEE5-gfp clone 8
5,576 ng/ul
pSB1C3-LEE5-gfp clone 10
5,897 ng/ul
30/07/18 by Yueying
PCR of LEE5-gfp from pKK-lee5-gfp.
well 1 and 4 : Gene ruler DNA ladder
well 2 and 3:  lee5-gfp PCR product
The lee5 promotor PCR products were putted on gel. The DNA matrix used is pKK-Lee5-GFP, the enzyme used is polymerase phusion, the primers used are GO2018-29and GO2018-30 and the expected size of PCR products is 1,3 kb.
02/08/18 by Yueying
Digestion PCR product of lee5-gfp with EcoRI and Pst1.
Ligation of lee5-gfp in pSB1C3 with T4 ligase.
Transformation of the ligation product in TH5alpha and K12 hns- E.coli.
03/08/18 by Yueying
Result of transformation:
There’re more than 20 clones for TH5alpha.
There is only one clone appear K12 hns- E.coli.
pSB1C3-lee5-gfp in MG1655Z1 hns-
(1 clone)
pSB1C3-lee5-gfp in MG1655Z1 hns-
pSB1C3 in MG1655Z1 hns-
pKK-lee5-gfp in  MG1655Z1 hns- (positif control)
Since in K12 hns- E.coli, we can quickly find the clone with the good cloning because of d’expression of gfp. We did anther transformation to obtain more clone.
Transformation of the ligation product in K12 hns- E.coli.
04/08/18 by Yueying
Result of transformation:
There’re more than 20 clones in K12 hns- E.coli.
06/08 by Yueying
Colony PCR to check for the presence of Lee5-gfp in pSB1C3-LEE5-gfp.
well 1 and 12 : Gene ruler DNA ladder
well 2 to 11: PCR product from K12 hns- E.coli harboring pSB1C3-LEE5-gfp.
well 13 : PCR product from DH5alpha harboring pSB1C3-LEE5-gfp.
well 14 : PCR product from plasmid pSB1C3-rfp(positive control)
The PCR products were putted on gel. The DNA matrix used is the colonies or plasmid pSB1C3-rfp, the enzyme used is polymerase DreamTaq, the primers used are VF2 and VR and the expected size of PCR product is 1,5kb for pSB1C3-LEE5-gfp and 1,2kb for  pSB1C3-rfp.
Result: all the clones have the expected size, we chose the clone 1 for the correction of forbidden sites.
Extraction of pSB1C3-LEE5-gfp.
07/08 by Yueying
PCR for the correction of forbidden sites of pSB1C3-LEE5-gfp.
well 4 and 7 : Gene ruler DNA ladder
well 1 to 3: PCR product
well 13 : PCR product from DH5alpha harboring pSB1C3-LEE5-gfp.
well 14 : PCR product from plasmid pSB1C3-rfp(positive control)
The PCR products were putted on gel. The DNA matrix used is the colonies or plasmid pSB1C3-rfp, the enzyme used is polymerase DreamTaq, the primers used are VF2 and VR and the expected size of PCR product is 1,5kb for pSB1C3-LEE5-gfp and 1,2kb for  pSB1C3-rfp.
08/08/2018 by yueying
Gibson assembly for pSB1C3-LEE5-gfp and transformation of the ligation product in K12 hns- E.coli and TH5alpha strain.
09/08/2018 by yueying
Result of transformation:
There’re more than 20 clones fluorescence appear in K12 hns- E.coli and TH5alpha strain.
We choose 12 clones from K12 hns- and 2 from TH5alpha strain for the verificaiton by digestion.
10/08/2018 by yueying
Plamid extraction of 12 clones.
Digestion of 12 candidat of pSB1C3-LEE5-gfp with correction and 2 candidat of pSB1C3-LEE5-gfp without correction and pSB1C3-rfp (positive control for the effectuation of digestion)  by Xbal and Spe1.
 
well 9 : Gene ruler DNA ladder
well 1 to 8 and 10 to 15 : plasmid pSB1C3-LEE5-gfp from  E.coli K12 hns-  with correction and digestion
well 16 : plasmid pSB1C3-LEE5-gfp  from DH5alpha E.coli with correction and digestion
well 17   and 18 : plasmid pSB1C3-LEE5-gfp  without  correction with  digestion ( control)
well 19 : plasmid pSB1C3-rfp  with  digestion  (positive control)
well 20 : plasmid pSB1C3-LEE5-gfp  without  correction without  digestion  (negative  control)
The digestion products were putted on gel. The expected size for digestion  product of plasmid pSB1C3-LEE5-gfp with correction is  1,5kb and 2,5 kb.
Result : Only clone 3 , 7 and 12 are not correct, others are correct.
14/08 by yueying
We send 5 of those clones correct for the sequencing.
17/08: by yueying
Result sequencing: There are 4 of 5 clones sequenced which are correct.
we choose the clone 1 and we redid a transformation in in K12 hns- E.coli and K12 WT for the frist carracterisation.
18/08 by yueying
Result of transformation:
pKK-lee5-gfp in  K12 hns-
pSB1C3-lee5-gfp-correction in K12 hns-
pKK-lee5-gfp in  K12 WT
pSB1C3-lee5-gfp-correction in K12 WT
24/08 by Stephanie and yueying
Flow cytometry analyse for the biobrick caracterisation.
28/08 by Stephanie and yueying
Flow cytometry analyse the biobrick caracterisation.
