Team:TJUSLS China/Notebook
MENU ▤
Menu
TJUSLS
project
Description
Design
Experiments
Proof of Concept
Demonstrate
Notebook
Results
PARTS
Basic Parts
Composite Parts
Part Collection
SAFETY
ATTRIBUTIONS
HUMAN PRACTICES
Silver
Gold
Integrated Practices
Engagement
MODELING
TEAM
Team
Collaborations
Notebook
April
4
Structure：
Structure confirmation of PET-21b(+) -PETase, using PCR and enzyme digestion method.
Mutation:
We choose the mutation points of the PETase in accordance to the papers we have read which have something to do with the proteins which has the maximum similarity with PETase in amino acid sequence.
The modeling team was set up in this week! We met each other and made some communication with others even though we came from different colleges. The head of our team made the leader of this IGEM-team and our modeling team member to knew each other, which made we knew IGEM a lot.
May
1
Structure：
PET-21b(+)-PETase sequence confirmed by sequencing. Trial expression successful.
Mutation:
Structure confirmation of 3 mutant vectors, PET-21b(+)-PETase(M,D,J), using PCR and enzyme digestion method.
In order to know modeled content, we reviewed the papers about PET enzyme and math model to knew the mechanism of degration of PET in the environment of PET enzyme. What’s more, we stress some question about chemistry and biology waiting for the interflowing with experimental group.
2
Structure:
Cultivation of PETase in E.coli ;Collection of E.coli ;Ultrasonication;Centrifugation;Ni-sepharose purification;Cation exchange column;Ultrafiltration;Gel filtration chromatography (twice);Protein purified.insufficient to crystalize, due to lack of experience.
Mutation:
Three mutant vectors sequence confirmed by sequencing. Trial expression successful.
We asked so many questions to the experimental group, we discussed the information we didn’t understand on papers in detail.
3
Structure：
Cultivation of PETase in E.coli ;Collection of E.coli ;Ultrasonication;Centrifugation;Ni-sepharose purification;Cation exchange column;Ultrafiltration;Gel filtration chromatography (twice);Protein purified.The purified protein still insufficient to crystalize.
The remaining part of last week we still felt confused about was determined by browsing other relevant data and papers. So we have more understanding with the modeling about this part.
4
Preparation for school exams. Experiments paused.
We proposed several possible modeling direction and assigned a different direction to the individual in order to make everybody gather information respectively.
June
1
Preparation for school exams. Experiments paused.
We collected all the information and data this week for knowing each other’s work better.
2
Yeast:
Successful genome extraction of Pichia Pastoris GS115.
For a more detailed understanding of related experiments in the experimental group with PET digested in the environment of PET enzyme, we strengthened the exchanges with experimental group to know the experimental framework and the purpose of experiments.
3
Yeast:
PCR amplification of the anchoring protein sequence GCW21, GCW51 and GCW61, using the genome of GS115 as template.
According to the experimental information, the protein structure modeling was excluded because of the need of related professional. So the direction of modeling was focused on the optimization of the expressing condition of fungal and bacterial, and the reaction kinetics of PET digesting.
4
Preparing for school exams. Experiments paused.
We browsed many papers about math optimization and reaction kinetics according to the modeling direction respectively this week.
July
1
Preparing for school exams. Experiments paused.
In the reaction, the PET is a solid phase, which cause the reaction a heterogeneous reaction, therefore, we couldn’t use the Michaelis-Menten equation to express the reaction kinetics. In order to break the problem, we made so many discussing and reading.
2
Structure:
Cultivation of PETase in E.coli ;Collection of E.coli ;Ultrasonication;Centrifugation;Ni-sepharose purification;Cation exchange column;Ultrafiltration;Gel filtration chromatography (twice);Protein purified.Grow crystal
Mutation:
Cultivation of PETase in E.coli ;Collection of E.coli ;Ultrasonication;Centrifugation;Ni-sepharose purification;Change the buffer of PETase;Cultivation of PETase with PET;Use Nanodrop to detect the concentration of product
E.coli：
Overlap PCR of three sequences, which is the signal peptide, PETase, and the anchoring motif, aiming to fuse these three respective sequences together, but failed.
Yeast:
Overnight culturing of E.coli DH5α™ containing vectors pPIC9-inJanus/sJanus;                 Extraction of vectors pPIC9-inJanus/sJanus;
Overnight culturing and extraction of E.coli DH5α™ containing vectors  PET22b-PETase;                 PCR amplification of the anchoring protein sequence GCW21、GCW51、GCW61 and PETase sequence,which taked overlap area, using primer the extracted plasmids as template.;                 Fusing PETase and 3 types of anchoring protein sequence respectively, using overlap PCR method.                 Construction and transformation of vectors pPIC9-PETase-GCW21/51/61, the transformated E.coli DH5α™ was cultured overnight.
It was assumed that the PET digestion would start after the adsorption of PET enzyme on the PET surface. And we simplified the adsorption process as a Langmuir adsorption process, so it is in line with the Langmuir adsorption equation.
3
Structure:
Cultivation of PETase in E.coli ;Collection of E.coli ;Ultrasonication;Centrifugation;Ni-sepharose purification;Cation exchange column;Ultrafiltration;Gel filtration chromatography (twice);Grow crystal
Mutation:
Cultivation of PETase in E.coli ;Collection of E.coli ;Ultrasonication;Centrifugation;Ni-sepharose purification;Change the buffer of PETase;Cultivation of PETase with PET;Use Nanodrop to detect the concentration of product
E.coli:
After changing the overlap PCR conditions, 4 expression vectors were successfully constructed.
Yeast:
PCR amplification of inJanus and sJanus sequence, using the extracted plasmids as template.
Fusing the two sequences and GCW 61 respectively, using overlap pcr.                 Extraction of vector pPICZαA.
Construction and transformation of vectors pPICZαA-inJanus/sJanus-GCW61, the transformated E.coli DH5α™ was cultured overnight                 Linearization of vector ppic9-PETase-GCW51, using restriction enzyme Stu I(Eco 1437)
Preparing Pichia Pastoris cells for electroporation.
Linearization of vectors ppic9-PETase-GCW61/21
transformation of vector ppic9-PETase-GCW51 to Pichia Pastoris, using electroporation.
Construction confirmation of vector ppiczaA-inJanus/sJanus-GCW61, using pcr method.
We understanded a small amount of by-products in the reaction process which gave the reaction kinetics simulation greater challenges. But it was found that the final amount of by-products was fewer than the main products, so we put the point on the main reaction and ignored the by-products in the simulation.
4
Stucture:
Send the crystal to Shanghai to do the X-ray diffraction and collect the data to obtain the real structure of PETase.
Mutation:
Cultivation of PETase in E.coli ;Collection of E.coli ;Ultrasonication;Centrifugation;Ni-sepharose purification;Change the buffer of PETase;Cultivation of PETase with PET;Use Nanodrop to detect the concentration of product;Reverse-phase high performance liquid chromatography (HPLC) analysis. E.coli:                 Vector transformation of vectors into BL21. Trial expressed at various temperature, IPTG concentration, and expression time.
Yeast:
Screening for HIS+ ppic9-PETase-GCW51 recombinants on MD plates. Confirming insert of vector using PCR analysis.                 Induced expression of protein PETase-GCW51 on the confirmed ppic9-PETase-GCW51 recombinants.
ppiczaA-sJanus/inJanus-GCW61 and ppic9-PETase-GCW21/51/61 co-transformation into Pichia Pastoris GS115.
According to many guess last week we made, we finally derived a preliminary reaction kinetics, but we hadn’t binded it with the experimental results.
August
1
Mutation:
Repeat the experiments above and cooperate with modeling group to perform a curve fitting by using the data we have gained and work out a maximum activity condition, then we will test it.
E.coli:
The successfully expressed recombinants were made to react with PET. After HPLC analysis, we confirmed that these recombinants behave as whole cell biocatalysts  which can hydrolyze PET into monomers, and have similar enzymatic character as purified PETase.
Yeast:
Screening for positive ppic9-PETase-GCW21/51/61 recombinants, using PCR method.
Screening for positive ppic9-PETase-GCW51/21 and ppiczaA-inJanus/sJanus-GCW61 co-transformed recombinants, using PCR method.
The ppic9-PETase-GCW21/51/61 recombinants were cultured in BMM for induced expression.
The successfully expressed ppic9-PETase-GCW21/51/61 recombinants were made to react with PET. After HPLC analysis, we confirmed that these recombinants behave as whole cell biocatalysts  which can hydrolyze PET into monomers, and have similar enzymatic character as purified PETase.
This week we modified the model a little to fit the real reaction better, and we made some editing of documents. At the same time, we collected the experimental data to be applied to the estimated reaction kinetic constants.
2
Mutation:
Repeat the experiments above and cooperate with modeling group to perform a curve fitting by using the data we have gained and work out a maximum activity condition, then we will test it.
E.coli:
Repetition of enzymatic reaction, in order to prove repeatability and gain credibility of previous results.
Yeast:
ppic9-PETase-GCW21/51/61 plus ppiczaA-inJanus/sJanus-GCW61 recombinants were cultured in BMM for induced expression.
ppic9-PETase-GCW21/51/61 plus ppiczaA-inJanus/sJanus-GCW61 were detected to have enzymatic activity when reacting with PET, using HPLC analysis.
Considering we needed to optimize the complex reaction conditions, we turned to the related literature about multivariate data fitting.
3
Yeast:
Repeating the experiments above and cooperate with modeling group to perform a curve fitting by using the data we have gained and work out a maximum activity condition, then we will test it.
We chosen the “multivariate nonlinear regression analysis model”, for simulating PET hydrolysis reaction.
4
One week off.
In order to got more available data, we decided to use the Matlab GUI module to write a program witch was used to calculate the amount of the product.
September
1
Yeast:
Repeating enzymatic reaction of all recombinants, in order to prove repeatability and gain credibility of previous results.
Due to breakdown of HPLC machine, experiments were postponed.
Wrote the program and debugged.
2
Structure:
Cultivation of Mutate M in E.coli ;Collection of E.coli ;Ultrasonication;Centrifugation;Ni-sepharose purification;Cation exchange column;Ultrafiltration;Gel filtration chromatography (twice)
Yeast:
Western blotting and immunofluorescent photography of all yeast recombinant cells.
Completed the program, and processed the data.
3
E.coli:
Western blotting and immunofluorescent photography of all E.coli recombinant cells.
Yeast:
Western blotting and immunofluorescent photography of all yeast recombinant cells.
Collated our documents.
4
Mutation:
Western blotting.
Collated our documents.
