<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TJUSLS_China_PROJECT_PROTOCOLS skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TJUSLS China/PROJECT/PROTOCOLS</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Home</TITLE></P><DIV class="cbp-spmenu-push" id="body"><DIV class="header_bg"><DIV class="wrap"><DIV class="header"><DIV class="h_menu4"><A class="toggleMenu" href="">Menu</A><UL id="top_ul_my" class="nav"><LI class="active"><A class="active" href="https://2016.igem.org/Team:TJUSLS_China/home">TJUSLS</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Project" class="root">project</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/PROJECT/PROTOCOLS">protocols</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Notebook">Notebook</A></LI></UL><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Parts">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Modeling">MODELING</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Safety">SAFETY</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Team">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:TJUSLS_China/Collaborations">Collaborations</A></LI></DIV></DIV></DIV></DIV><DIV class="header-banner"><DIV class="top-header"><DIV class="container"><DIV class="logo"><A href="index.html">TJUSLS</A></DIV><DIV class="top-nav"><NAV class="cbp-spmenu cbp-spmenu-vertical cbp-spmenu-right" id="cbp-spmenu-s2"><H3>Attribution</H3><A href="https://2016.igem.org/wiki/index.php?title=Team:TJUSLS_China" class="active">home</A><A href="#part1">part1</A><A href="#part2">part2</A><A href="#part3">part3</A><A href="#part4">part4</A><A href="#part5">part5</A></NAV></DIV></DIV></DIV></DIV><DIV class="content"><DIV class="copyrights">Collect from <A href="http://www.cssmoban.com/">aaaa</A></DIV><DIV class="event"><DIV class="container"><H3>Protocols</H3><DIV class="event-grids"><DIV style="background-color: #FFF; text-align: left"><P style="font-size: 25px; font-family: aroline Golino; color: #808080;"><B>Molecular Cloning</B>
                            1.Take PETase &amp;MHET standardization for instance<B>Transfer</B>1.	Mix the PETase Plasmid (pET 21-b, Amp+, 1μL 100ng/μL) with the E.coli DH5α competent cells in the EP tubes. Incubate them on ice for 20-30 minutes.
                            * Plasmid pET21-b plasmid containing the Amp-resistant gene of PETase was selected.
                            2.	Heat shock the cells by immersion in a pre-heated water bath at 42ºC for 90 seconds without shake. Transport the tubes in ice for 3-5 minutes.
                            3.	Add 550 μL of LB media to each transformation. Incubate the cells at 37ºC, 220RPM for 30 minutes while the tubes are shaking to express the Amp resistant gene.
                            4.	Amp-resistant plates were preheated in a 37℃ incubator.
                            5.	Put the tube centrifugal 2 min, 3000 RPM. Remove 500μl the supernatant.
                            6.	Plate 100 µl of the transformation onto the dishes with LB agar(AMP), and spread.
                            7.	Resuspend E. coli pellets. Incubate the plates at 37ºC for 12-25 hours, making sure the agar side of the plate is up.

                            <B>Pick Monoclonal colonies and transferred to ampicillin-resistant LB medium for subculture
                            </B>In the clean bench, a single colony was picked up from the medium with a pipette tip, and introduced into a test tube containing 5 mL of LB broth and 5 μL Amp, covered with a stopper, shaken at 220 rpm at 37 ° C Bed culture 12 ~ 16h.

                            <B>Extract the amplified plasmid from the cultured Escherichia coli</B>1.	Add the E. coli solution into the EP tube
                            2.	Re-suspend pelleted bacterial cells in Buffer P1 BL (kept at 4 °C) and transfer to a micro-centrifuge tube.
                            3.	Add Buffer P2 and gently invert the tube 4–6 times to mix.
                            4.	Add 350μl Buffer N3 and invert the tube immediately and gently 4–6 times.
                            5.	Centrifuge for 10 min at 12,000 rpm in a micro-centrifuge.
                            6.	Regenerate column CP3 while centrifugation. Add 500μl Buffer BL. Centrifuge for 1 min at 12,000 rpm after static for 2min. Discard the flow-through.
                            7.	Add supernatant from the EP tube to the column and put it into collection canals. Add 500μl Buffer BL and centrifuging for 1min at 12000rpm. Discard the flow-through.  Spin for 60 seconds produces good results.
                            8.	Adding 600μl Buffer PW and centrifuging for 60s after static for 2min. Discard the flow-through.
                            9.	Repeat step 8.
             10.	Centrifuging for 2min at 12000rpm to shake off the rest of the Buffer PW.
                            11.	Place the column in a new EP tube and the opening was allowed to stand for 5 minutes, so that the ethanol in the PW can be sufficiently volatilized.
                            12.	Add 50μL 75℃ of sterile distilled water at 75 ° C dropwise to the middle of the adsorbed film. Static for 2min. Discard the flow-through. Centrifuging for 2min at 12000rpm.

                            <B>Examine the concentration of the extracted plasmid</B>Add 1 drop of sterile distilled water at the light hole of nano Drop and erase it after 90s for two times. Add 1 drop of the solution in EP tube at the light hole of nano Drop with pipette and erase it after measure.

                            PCR：
                            <B>Plasmid Digestion：</B><B>PCR product Digestion：</B><B>Agarose Gel Electrophoresis(Plasmid &amp; PCR product)	：</B>1. Prepare sufficient lx TAE to fill the electrophoresis tank and to cast the gel.
                            2. Prepare a solution of agarose in electrophoresis buffer at a concentration of 1%: Add 0.9g powdered agarose to 90ml of TAE in an Erlenmeyer flask.
                            3. Heat the slurry in a boiling-water bath or a microwave oven until the agarose dissolves.
                            4. Use insulated gloves or tongs to transfer the flask/bottle into a water bath at 55°C. When the molten gel has cooled, add ethidium bromide to a final concentration of 0.5 μg/ml. Mix the gel solution thoroughly by gentle swirling.
                            5. While the agarose solution is cooling, choose an appropriate comb for forming the sample slots in the gel. Position the comb 0.5-1.0 mm above the plate so that a complete well is formed when the agarose is added to the mold.
                            6. Pour the warm agarose solution into the mold.
                            7. Allow the gel to set completely (30-45 minutes at room temperature), then carefully remove the comb. Pour off the electrophoresis buffer and carefully remove the tape Mount the gel in the electrophoresis tank.
                            8. Add just enough electrophoresis buffer to cover the gel to a depth of ~1 mm.
                            9. Mix the samples of DNA with 10 μl green buffer
                            10. Slowly load the sample mixture into the slots of the submerged gel using a disposable
                            micropipette, an automatic micropipettor. Load size standards into slots on both the right and left sides of the gel.
                            11. Close the lid of the gel tank and attach the electrical leads so that the DNA will migrate
                            toward the positive anode (red lead). Apply a voltage of 1-5 V/cm (measured as the distance between the positive and negative electrodes). If the leads have been attached correctly, bubbles should be generated at the anode and cathode (due to electrolysis), and within a few minutes, the bromophenol blue should migrate from the wells into the body of the gel. Run the gel until the bromophenol blue and xylene cyanol FF have migrated an appropriate distance through the gel.
                            12. When the DNA samples or dyes have migrated a sufficient distance through the gel, turn off the electric current and remove the leads and lid from the gel tank.

                            <B>Plastic recycling</B>1.	Column regeneration: add 500μl balance liquid to adsorption column CA2 12000 rpm centrifugal 1 minutes, pour out the filtrate and put it back to the collecting canals.
                            2.	Cut the DNA band from the gal, weighing in the clean centrifuge tube and weight.
                            3.	Add equal volume solution PN (such as 0.1 g gal, to join 100μl PN) in rubber block, then put in 50 ℃ water bath place, turning up and down continuously in order to ensure fully dissolved. Cooling to ambient temperature.
                            4.	Add the solution into a adsorption column CA2 under the ambient temperature for 2 minutes. 12000 rpm centrifugal 30-60 seconds, pour out the filtrate, placed at ambient temperature for a few minutes to ewnsure fully getting rid of the alcohol. Put the adsorption column CA2 into the collection canals.
                            5.	Drift lotion to column to join 600 μl PW, 12000 rpm centrifugal 1 minutes, pour out the filtrate.
                            6.	Repeat step 5.
                            7.	12000 rpm centrifugal for 2 minutes, pour out the filtrate, placed at ambient temperature for a few minutes to ensure fully getting rid of the filtrate.
                            8.	Put CA2 in clean centrifuge tube, add ddH2O and wait for 10 minutes at ambient temperature, 12000 RPM, centrifugal 2 min. Collect the DNA solution.
                            Saved the DNA concentration at -20 degrees Celsius to prevent DNA cleavage.


                            <B>Cloning PCR Products into pSB1C3 Vectors：</B><B>Transformation：</B>1. Add DNA 10μl to each tube. Mix the contents of the tubes by swirling gently. Store the tubes on ice for 30 minutes.
                            2. Transfer the tubes to a rack placed in a preheated 42°C circulating water bath. Store the tubes in the rack for exactly 90 seconds. Do not shake the tubes.
                            3. Rapidly transfer the tubes to an ice bath. Allow the cells to chill for 5 minutes.
                            4. Add 600 μl of LB medium to each tube. Incubate the cultures for 45 minutes to allow the bacteria to recover and to express the antibiotic resistance marker encoded by the plasmid.
                            5. Centrifugal 3500rpm 2min
                            6. Transfer 100 μl of transformed competent cells onto agar LB medium containing chloramphenicol.
                            7. Store the plates at room temperature until the liquid has been absorbed.
                            8. Invert the plates and incubate at 37°C. Transformed colonies should appear in 12-16 hours.

                            <B>Inoculated in tubes	：</B>
                            Six 5 ml LB broths were added with 5 μl of chloramphenicol. A single colony was picked from the overnight culture plates and inserted into LB liquid medium containing antibiotics. Tubes were placed in a 37⁰C shaker and incubated for 7 h.

                            <B>Plasmid extraction (pSB1C3–PETase&amp;pSB1C3-MHET)	：</B>1.	Column regeneration: add 500 μl balance liquid to adsorption column CP3 12000 rpm centrifugal 1 minutes, pour out the filtrate and put it back to the collecting canals.
                            2.	Add 1-5 ml colony into the centrifuge tube, centrifuge for 1 minute at 12,000 rpm (~13,400 xg). Pipetting out the supernatants.
                            3.	Resuspend pelleted bacterial cells in 500 µl Buffer P1 BL. No cell clumps should be visible after resuspension of the pellet.
                            4.	Add 250 μl Buffer P2 and gently invert the tube 6–8 times to mix for the lysis reaction.
                            5.	Add 350 μl Buffer P3 and invert the tube immediately and gently 6–8 times. The solution should become cloudy. Centrifuge for 10 min at 12,000 rpm (~13,400 xg). A white pellet will form. Add the supernatants to the CS filter column by pipetting. Be careful not pipet out the white pellet.
                            6.	Centrifuge for 2 minutes at 12,000 rpm (~13,400 xg). Transfer the solution in the collecting tube into adsorption column CP3. If there is some remainder showing the supernatants from step 5 have too many impurity, the period of centrifuge should be extend. If there is a little sediment in collecting tube, only pipet out the supernatants.
                            7.	Centrifuge for 30-60 seconds at 12,000 rpm (~13,400 xg). Discard the flow-through. Put CP3 into collecting tube.
                            8.	Add 500 μl PD in CP3, centrifuge for 30-60 seconds at 12,000 rpm (~13,400 xg). Discard the flow-through. Put CP3 into collecting tube.
                            9.	Add 600 μl PW in CP3, (please check if there have  alcohol already.) centrifuge for 30-60 seconds at 12,000 rpm (~13,400 xg ). Discard the flow-through. Put CP3 into collecting tube.
                            10.	Repeat step 9.
                            11.	Put CP3 into collecting tube, and centrifuge for an additional 2 min to remove residual wash buffer. Open the CP3 in ambient temperature for minutes in order to dry out the reminder wash buffer.
                            Place the CP3 column in a clean centrifuge tube. To elute DNA, add 50-100 μlddH2O to the center of adsorption film, keep it in ambient temperature for 2 minutes, then centrifuge for 2 min.

                            <B>Double enzyme digestion and PCR</B><B>Sequencing</B><B>2.Take Flag-PETase-linker-GCW51 fusion protein standardization for instance</B>We use overlap PCR to get all the fusion protein. The overlap extension polymerase chain reaction (or OE-PCR) is a variant of PCR. It is also referred to as Splicing by overlap extension / Splicing by overhang extension (SOE) PCR. To splice two DNA molecules, special primers are used at the ends that are to be joined, because of which a linker is added into BFP and hydrophobic protein. When PCR, to splice two DNA molecules, special primers are used at the ends that are to be joined. Once both DNA molecules are extended in such a manner, the overlapping complementary sequences introduced will serve as primers and the two sequences will be fused.

                            We can use this technique to combine A fragment and B fragment. In first cycle we use a couple of primers of A and the same to B to PCR. By this, we design a same length of base sequence. In second cycle, due to the homologous sequence, when adding the beginning primers and the final primers, A and B can be primers to each other. They extend to a combined fragment.


                            This image shows how OE-PCR might be utilized to splice two DNA sequences (red and blue). The arrows represent the 3' ends
                            <B>Overlap PCR：</B><B>Protein Expression in E.coli</B><B>Pre-expression</B>1.	Put 100uL Glycerin bacteria liquid to 5mL LB liquid medium. Put in shaker 220rpm, 37℃,16-24h. Take 200uL sample and prepare it.
                            2.	In each tube, add 4uL IPTG to induce. Put in shaker for 4h. Then take 200uL of sample and prepare it.
                            Samples prepare for electrophoresis: 12000rpm, 2min centrifuge. Then pour out supernatant. Use 20uL ddH2O to re-suspended sediment and 4uL 6X protein buffer. Then heated at 100℃ for 20min. Stored at -4℃. Centrifuge 12000rpm, 2min before electrophoresis.
                            3.	Electrophoresis.

                            <B>Cultivate in 1L LB medium</B>1.	Pour one tube to one bottle. Put in shaker 220rpm for 16-24h. Must add chloramphenicol
                            2.	Add 600uL IPTG to each bottle. And also put in the same condition in last step for 4 h
                            3.	Pour out solution to a tube. 3500 rpm, centrifuge for 20min.
                            4.	Abandon supernatant, add 15mL McAc 0. Use pipette repeatedly blowing and sucking to re-suspend the sediment.
                            5.	Wash the bacteria broken machine with ethyl alcohol and McAc 0 in order. And break the bacteria for three times. We can see the solution getting clearer.
                            6.	Transfer to another tube.18000 rpm, 40min, centrifuge.
                            7.	Mixed with medium. And put on protein rotation for 1h.
                            8.	Pour out into a Ni-NTA column to filtration
                            9.	Add 100mL 10, 100mL 20, 100mL 30 and 50mL 50 McAc in order 20mL each time to elute column for wiping of the bulk proteins.
                            10.	Add 20mL 100, 20mL 200, 20mL500 McAc in order 20mL each time to elute column to wash down the aimed proteins.
                            11.	SDA-PAGE to ensure if we get the right proteins.
                            Note:Every time of washing, take samples in penetration and media, before and after inducing, and sediment in last centrifuge.

                            1.	Add as follows(separation gel):
                            2.	Pour all solution to hole between two broads.
                            3.	Add some ddH2O or n-butyl alcohol to make upper limb straight.
                            4.	Wait for a moment until gel solidifies. Then pour out ddH2O and use Filter paper to blot it.
                            5.	Add as follows (stacking gel):
                            6.	Put in comb and wait till gel solidifies about 20min.
                            7.	Take out comb cautiously and put the whole broad in running machine.
                            8.	Add samples and maker. Set voltage 180V for 20min and start.

                            <B>Analytical structure</B><B>Take the PETase crystallization for instance</B><B>A large number of cultured protein expression bacteria</B>1.	Extract 5μL of the bacterial strain from the bacterial strain retained in the test expression, and add 5μL Amp to a test tube containing 5 mL of LB culture solution.
                            2.	Incubate the plates at 37 ° C overnight at 220 rpm in a shaker
                            3.	Pour the bacteria in the test tube into a 2L Erlenmeyer flask containing 1 L of LB medium, and add 500μL Amp.
                            4.	37 ℃, 220rpm in the shaker culture 4 ~ 5h, on behalf of bacteria growth to logarithmic phase (OD value 0.6-0.8) 16 ℃, 220rpm temperature drop 1h (temperature dropped to 16 degrees), and then add 500μL IPTG induced E. coli (4) Producing the protein of interest.
                            5.	16 ℃, 220rpm overnight culture 12h ~ 15h.
                            6.	Collection: The bacteria were centrifuged at 4000rpm for 16min at 4000rpm and the supernatant was discarded. The precipitated Escherichia coli was transferred to a 50mL centrifuge tube and stored at -30 ℃ (total of 6 vials)

                            <B>Purification of PETase</B>1.	Disrupt the bacteria by high-pressure sterilizers, and the broken bacteria were ultrasonographed for 15 minutes (4S ultrasound, 6S interval) to remove the nucleic acids carried by the protein of interest. Followed by equilibration in a high speed centrifuge tube, 18000 rpm for 20 minutes, separating the supernatant containing the protein and the precipitate containing the cellular debris.
                            2.	Nickel column：The supernatant was applied to the His-Accept nickel column. After washing unbound proteins with the lysis buffer (50mM Tris-HCl, pH 7.5, 300mM NaCl, 20mM imidazole), the bound proteins were eluted with elution buffer (50mM Tris-HCl, pH 7.5, 300mM NaCl, 250mM imidazole)
                            3.	A / B Formulation A: pH 5.0 50mM Sodium Acetate 50mM NaCl B: pH 5.0 50mM Sodium Acetate 1 M NaCl
                            4.	Cation exchange column: The sample eluted from the nickel column was added to a 10kd concentrating tube. The solution was concentrated at 3500 rpm to a solution of cation exchange column A, diluted ten times with A solution to 5mL. And then separated and purified by Hitrap SP HP 5mL column to obtain a peak shape. According to the shape of the peak were collected in the collection tube sample run glue.
                            5.	Molecular Sieve: Samples obtained from the S-column were changed (75% solution A + 15% solution B) and concentrated to 500μL. Using AKTA system, the superdex75 molecular sieves with appropriate separation range but higher resolution were used to purify the target protein. The two peaks were close to each other and the size was close to that of the target protein. Therefore, the corresponding protein samples were collected. Concentrated and re-separated through molecular sieves, respectively, to get two more single peak shape. Respectively, and the peak position before the two overlapping sample preparation, running glue for validation
                            6.	Crystallization: Take the corresponding position of the sample, measured with NanoDrop its concentration. (The former peak is called Sample A, the latter peak is called Sample B), and the final concentration is 1.61mg/ml for sample A and 3.96mg/ml for Sample B. According to the residual amount of protein, we used the conditions to do both local screening of both samples.

                            <B> Mutation</B><B>Take one mutation of PETase for instance</B><B>Primer design</B><B>PCR</B><B>Amplification products were detected by nucleic acid electrophoresis</B>1.	Mix agarose M and TAE solutions in an Erlenmeyer flask in an amount of 0.3g: 30 mL
                            2.	Put the conical flask into the microwave for heating until boiling, shake it out, and then heat to boiling until the solution is clear so far.
                            3.	Rinse the outer wall of the conical flask with water, and lower the temperature of the solution to 40 to 50 ° C (temperature was sufficient), then add 1.5μL of EB.
                            4.	Pour the solution into the mold, insert a comb, wait 20min, until the gel after solidification pulled out the comb.
                            5.	Place the gel in an electrophoresis apparatus, and add appropriate amount of TAE solution to the electrophoresis apparatus so that the surface of the gel was slightly out of the gel surface.
                            6.	Mix 10μL each of the nucleic acid solutions in H and L with 1 μL of Green buffer. Transferred to the second and third wells of the gel using a pipette.
                            7.	Add 10μL of the 8k Marker to the first sample well of the gel.
                            8.	Running glue 24min at 180V.
                            9.	Remove the agar and placed under UV light for observation.

                            <B>Digestion of PCR products (removal of unmodified template strand)</B>Add 1μL DMT enzyme and PCR products, mix, 37 ℃ water bath heating 1h.

                            <B>Protein expression in Pichia Pastoris and the verification experiment</B><B>1.Take the expression of ppic9-Flag-PETase-linker-GCW51 in yeast for instance</B><B>Preparation of whole cell catalyst</B>1.	Preparation of Pichia pastoris electro - transformation competent
                            Pick single colonies from fresh Pichia pastoris GS115 (His-) plates and transfer to 5 mL YPD culture medium tube at 30℃, 250 rpm overnight. The next day, take 0.1-0.5 ml of the overnight culture from the test culture tube and inoculate into a 2 L shake flask containing 500 ml of fresh YPD medium and grown overnight to OD600 of approximately 1.3-1.5. The cells were harvested by centrifugation at 1500rmp for 5 min at 4℃, followed by the addition of 20 mL of HEPES buffer at pH 8.0 in 100 mL YPD of 100 mM YPD, then followed by 2.5 mL of freshly prepared 1M DTT), and incubated on a shaker for 15 minutes at 30℃; The cells were removed from the shaker and incubated in ice bath for half an hour. Cells were harvested by centrifugation at 1500 rpm for 5 min at 4℃, then been resuspended in 250 ml of pre-chilled sterile water; centrifuged as above, then resuspend in 20 ml of pre-chilled 1M sorbitol; centrifuged as above, then resuspend in 1 ml of pre-chilled 1M sorbitol to a final volume of about 1.5 ml; centrifuged as above, then resuspend in 500 μL of cold ultrapure water and placed at -80℃ until use
                            2.	Transformation of Pichia pastoris by electroporation
                            Mix 80μL of the above cells with 5-20μg of linearized DNA, transfer to a pre-chilled 0.2 cm electroporation beaker and place on ice for 5 min. Immediately add 1 mL of pre-cooled 1M sorbitol to the conversion cup after the 1.5 kV electric shock. Transfer the contents to a sterile centrifuge tube and resuscitate in a 30℃ incubator for 1 hour and divide into 200-600μL aliquots and apply to an MD plate. Cultures were grown in a 30℃ incubator until cloning (approximately 4-6 days) and screen the Mut+/Muts phenotype.
                            3.	Screening of Pichia pastoris positive transformants
                            Use a sterile toothpick picking Fifty His+ transformants from the MD plates and placed on a new MD plate. Standard serial number and incubated in a constant temperature incubator at 30℃.
                            4.	Picking yeast single colonies on the MD plate and cultured overnight
                            1). Take 5ml YPD liquid medium in the clean bench.
                            2). Insert Pichia pastoris containing a fusion gene of Flag-PETase-linker-GCW51 on a single colony-grown MD plate into YPD liquid medium.
                            3). Place the tubes in a 30℃ shaker and incubated for 14-16 h
                            5.	Extraction of Pichia pastoris genome
                            1). The Pichia pastoris cells (no more than 5 × 107 cells) were centrifuged at 12,000 rpm (~ 13,400 × g) for 1 minute, and the supernatant was removed as much as possible. The cells were collected by centrifugation several times into one centrifuged tube.
                            2). Add 500μl of the cell suspension to the centrifuge tube with the cell pellet. Suspend the bacterial cell pellet thoroughly using a pipetter or vortex oscillator, incubate at 37℃ for 45 minutes to 1 hour, and invert the mixture every 5-10 minutes. Centrifuge at 12,000 rpm (~ 13,400 × g) for 2 minutes and aspirate the supernatant as much as possible.
                            Note: The above 5 × 107 yeast cells Lyticase dosage, according to the yeast strain and the number of yeast cells, the incubation time should be adjusted.
                            3). Add 250μl of buffer A to the cell pellet, and shake until the cells were completely suspended. If RNA removal is required, add 6 μl of RNaseA (10 mg/ml), shake for 15 seconds, and allow to stand for 5 minutes at room temperature.
                            4). Add the 10 L protease K solution to the tube and invert the mixture.
                            5). Add 250μl of Buffer B, shake for 10 seconds, and leave at 70℃ for 10 minutes. Centrifuge at 12,000 rpm (~ 13,400 × g) for 30 seconds to 1 minute. Remove the supernatant into a fresh tube and discard the precipitate. (If the supernatant still has some turbidity, this is normal.)
                            Note: 1.Adding buffer B may produce white precipitate, but will not affect the follow-up experiments.2. Centrifugal sedimentation is not complete digestion of cell debris and denatured protein, such as precipitation is not a normal phenomenon.
                            6). Add 250μl of absolute ethanol, fully shake mixing 15 seconds, this time may be flocculent precipitate, instantaneous centrifugal to remove the water droplets inside the cover.
                            7). Add the solution and flocculent precipitate from the previous step to an adsorption column (the adsorption column was placed in a collection tube), centrifuged at 12,000 rpm (~ 13,400 × g) for 30 seconds, the waste was drained, and the column was placed in a collection tube.
                            8). Add 500μl of Buffer C to the column and centrifuged at 12,000 rpm (~ 13,400 × g) for 30 seconds. The waste was discarded and the column was placed in a collection tube.
                            9). Add 700μl of rinse solution W2 (check whether ethanol has been added before use) and centrifuge at 12,000 rpm (~ 13,400 × g) for 30 seconds. Discard the waste and place the column in the collection tube.
                            10). Add 500μl of rinse solution W2 to the adsorption column and centrifuged at 12,000 rpm (~ 13,400 × g) for 30 seconds. Discard the waste.
                            11). Place the adsorption column in a collection tube and centrifuged at 12,000 rpm (~ 13,400 × g) for 2 minutes then discard the waste. The adsorption column was left at room temperature for several minutes to thoroughly dry the residual rinse in the adsorbent material.
                            Note: The purpose of this step is to remove the residual rinse from the adsorption column. Residual ethanol in the rinse solution can affect subsequent enzymatic reactions (digestion, PCR, etc.).
                            12. Transfer the column red to a clean centrifuge tube. Suspend 50μl of elution buffer TE in the middle of the adsorbed membrane. The solution was collected by centrifugation at 12,000 rpm (~ 13,400 × g) for 2 to 5 minutes at room temperature in a centrifuge tube.
                            Note: To increase the yield of genomic DNA, the centrifuged solution can be added to the column for 2 min at room temperature and centrifuged at 12,000 rpm (~ 13,400 × g) for 2 minutes. The volume of elution buffer should not be less than 50 μl, the volume is too small affect the recovery efficiency. The pH of the eluent has a significant effect on elution efficiency. If the pH of the water is less than 7.0, the elution efficiency will be reduced. The DNA product should be kept in the pH range of 7.0-8.5, and the pH value should be within the range of 7.0-8.5. 20 ℃, to prevent DNA degradation.
                            6.	PCR to detect whether the target gene was inserted into the genome

                            Denaturation 94 ℃ 5min; denaturation 94 ℃ 1min; annealing temperature: 60 ℃, 45s; extension 72 ℃ 5.5min; fully extended 72 ℃ 10min, circulation 31 times.

                            7.	The transformants were further identified using primers specific for the Flag-PETasse-linker-GCW511 gene
                            Denaturation 95 ℃ 5min; denaturation 95 ℃ 30s; annealing temperature: a series of gradient, 30s; extension 72 ℃ 4min; fully extended 72 ℃ 10min, cycle 34 times.

                            8.	The transformants were identified by phenotypic identification and PCR identification, frozen glycerol tubes, and transferred to prepare the expression of the target protein for analysis.
                            9.	Induction of recombinant yeast expression
                            1). Select one of the positive transformants of PET-GCW21, PET-GCW51, and PET-GCW61, and place 30uL of the stored glycerol bacteria in 5 mL YPD tube culture medium, culturing at 30℃/280rpm until OD600 = 2-6.
                            2). Transfer to a shake flask filled with 10 ml of MG, BMG or BMGY medium and incubated at 30℃/280rpm until OD600 = 2-6 (~ 16-18 h).
                            3). The cells were centrifuged at 3000 g for 5 min at room temperature to collect the cells (or placed at room temperature to prevent the process of centrifugation to increase the probability of infection); 10ml BMM medium resuspended to OD600 = 1.0; placed in 200 mL shake flasks, sealed with a sealing membrane, placed on a shaker at 30℃/280rpm, and grown for 5 clones per clone.
                            4). Add 100% methanol to the culture medium to a final concentration of 1.0% every 24 h. The samples were taken at a time point of 10 ml, that is, 1 bottle at the time point: 24, 48, 72, 96 and 120 hours.

                            <B>2.Western Blot</B>【Materials】
                            ⒈ Apparatus:
                            Apparatus of SDS-PAGE, Electroblotting Apparatus, Power supply, PVDF membrane （Millipore Immobion-P #IPVH 000 10）, Whatman 3MM paper, Additional Tools: Forceps, sponge pad, scissor, gloves, small plastic or glass container, Shallow tray.
                            ⒉ Reagents:
                            ⑴ 10x buffer (660mL): Add 484.8g Tris , 2307.2g Glycine in ddH2O to final volume 660mL
                            ⑵ 4x transfer buffer (5280mL):660mL 10x buffer,3.3mL 20%SDS,4620mL ddH2O
                            ⑶ 1x transfer buffer (1 L): Add 200 ml Methanol, 250 ml 4x transfer buffer in ddH2O to final volume 1L
                            ⑷PBS buffer: Add 8g NaCl(137mM),0.2g KCl (2.7mM),1.44g Na2HPO4(10mM),0.24g KH2PO4(2mM)to 1L ddH2O and adjust pH to 7.4 with HCl.
                            ⑸ PBST buffer: 1L TBS buffer add 0.5ml Tween 20 (0.05%).
                            ⑹ First antibody
                            ⑺ Second antibody
                            ⑻ 5% Blocking buffer (10ml): Add 0.5g Non-fat milk powde in PBST buffer to final volume 10ml, keep at 4°C to prevent bacterial contamination.
                            ⑼ Developing reagent: volume ratio :A reagent:B reagent=1:1.
                            ⑽ Staining buffer: Add 1g amido black 18B (0.1%), 250ml isopropanol (25%) and 100 ml acetic acid (10%) to distilled water with final volume 1L.
                            ⑾ Destaining buffer: Add 350ml isopropanol (35%) and 2 ml acetic acid(2%) to distilled water with final volume 1L.

                            【Procedure】
                            ⒈. Separation of Protein
                            Run an electrophoretic separation of known antigenic proteins. The method of separation decided by the characters of target protein, but for sufficiently transferring, the most common method is SDS-PAGE.
                            After separation, remove upper side of sample wells with a razor blade. Notching bottom right-hand corner of gel for orientation and put gel in transfer buffer until ready to use.
                            ⒉. Electrotransfer
                            ⑴ Preparation of membrane
                            Cut a piece of PVDF membrane (Millipore Immobion-P #IPVH 000 10) according to the size of gel. Incubate in methanol for about 1 min on a rocker at room temp. Remove methanol and equilibrate membrane in 1x transfer buffer until ready to use.
                            ⑵ Arrange gel-membrane sandwich
                            In a shallow tray, open the transfer cassette. Put a well-soaked sponge pad on the black piece of the transfer cassette and a wetted 3MM paper on the sponge pad. Place the gel on the paper and arrange well so that all air bubbles are removed. Lay the PVDF membrane on the top of gel and remove any air bubbles. Place a wetted sheet of 3MM paper over the PVDF membrane and remove the bubble. Covered with the second well-soaked pad. Close the sandwich with the white piece of the cassette. Mount the sandwich in the transfer tank; put the black sides near the black side of the device. Fill the buffer tank with the transfer buffer.

                            ⑶ Electrotransfer:
                            Attach the electrodes. Set the power supply to 100V (constant voltage) for 1h at 4° C.
                            ⒊. Immunodetection
                            ⑴ Membrane staining
                            Disconnect transfer apparatus, remove transfer cassette, and peel 3MM paper from membrane. Remove the membrane to a small container. Add 10 ml PBST buffer and wash for short time. Cut out one stripe with 5mm width and put in another clean container. Stain this stripe in staining buffer for 1 min. Destain for 30 min in destaining buffer to check whether protein has been transferred from gel to membrane or not.
                            ⑵ Membrane blocking and washing
                            For other part of membrane, pour off PBST buffer. Add 5% blocking buffer，rock gently for at least 1 h. Pour off 5% blocking buffer and rinse briefly with PBST buffer three times, 5 minutes for per time.
                            ⑶ First antibody
                            Pour off PBST buffer. Add first antibody at appropriate dilution in 5 ml 0.5% blocking buffer. Rock gently for at least 1 h; pour off first antibody solution from membrane and wash twice for 10 minutes with PBST buffer.
                             ⑷ Second antibody
                            Pour off PBST buffer. Add second antibody at appropriate dilution in 5 ml 0.5% blocking buffer. Rock gently for 30min, pour off second antibody solution from membrane and wash twice for 10 minutes with TTBS buffer.
                            ⑸ Detection
                            Pour off PBST buffer from membrane and add developing reagent.Image with an imaging equipment.

                            <B> 3. Immunofluorescence detection</B>1.	Take 200uL induced culture medium at 4℃, 12000rpm, centrifugation 1min. Retain the bacteria.
                            2.	Resuspend the recombinant yeast cells in PBS buffer, centrifuged at 12000rpm for 1 min at 4℃, and the supernatant was discarded. Retain the bacteria.
                            3.	The cells were resuspended in 200μL of 3mg/mL BSA in PBS buffer for 1h at room temperature. The cells were mixed up and down at 4℃, 12000rpm and centrifuged for 1min. Retain the bacteria.
                            4.	The cells were resuspended in 200uL of PBS buffer containing 3 mg / mL BSA containing 1μL of primary antibody. Treat the cells overnight at 4℃, and the cells were allowed to mix up and down. The next day at 4 ℃, 12000rpm, centrifugation 1min. Retain the bacteria.
                            5.	Recombinant yeast cells were resuspended in PBS buffer, treated at room temperature for 10 min, centrifuged at 12000rpm for 1 min at 4 ℃. Retain the bacteria.
                            6.	Repeat step 5 twice.
                            7.	The cells were resuspended in 200uL of PBS buffer containing 1μL of secondary antibody and incubated for 1 h at room temperature. The cells were mixed up and down at 4℃ and 12000 rpm for 1 min. Retain the bacteria.
                            8.	The recombinant yeast cells were resuspended in PBS buffer for 5 min at room temperature. The cells were centrifuged at 12000rpm for 1 min at 4 ℃. The supernatant was discarded and retain the bacteria.
                            9.	Repeat step 8.
                            10.	Resuspended in 200uL of PBS buffer, each taking 10uL on clean static slide production.

                            <B>4.Detection of Enzyme Activity by HPLC</B>1.	Test the OD value of the solution
                            2.	Preparation of samples (set up an experimental group and a control group)
                            a.	Take appropriate amount of bacteria and the EP tube, 3000g, 4℃ centrifugal 5min, discard the supernatant.
                            b.	Add the appropriate amount of glycine sodium hydroxide buffer (pH = 9.0) resuspend the bacteria, centrifugate and discard the supernatant.
                            c.	Repeat the step b.
                            d.	Set the different bacterial gradient, and the same amount of bacteria per day test control.
                            e.	Add 100μl of glycine-sodium hydroxide buffer suspension.
                            3.	Add PET film in the experimental group in advance, while in the control group add no PET film (added on ice).
                            4.	The reaction was incubated in a thermostated shaker at 40 ° C, 18 h, 220 rpm.
                            5.	Frozen bacteria: Wash the remaining bacteria twice with PBS, 20% glycerol resuspended, frozen for store.
                            6.	After completion of the reaction, centrifugation was performed at 5000 g for 10 min at 4℃.
                            7.	Remove the supernatant (&gt; 50μl, do not aspirate the precipitate) and add an equal volume of stop solution.
                            8.	85℃ inactivated 10min (if not immediately do the HPLC, it can be freeze at -20 ℃)
                            9.	Centrifuge at 12,000 rpm for 10 min and take 100μl with new EP tubes.
                            10.	The prepared phosphate / monobasic sodium phosphate buffer (pH = 2.5) and methanol were treated with ultrasound for 30 min。
                            11.	Load 100μl


                        </P></DIV></DIV></DIV></DIV></DIV><DIV class="footer-section"><DIV class="container"><DIV class="footer-top"><P>Copyright © 2015.Company name All rights reserved.More Templates <A href="http://www.cssmoban.com/" target="_blank" title="模板之家">aaa</A> - Collect from <A href="http://www.cssmoban.com/" title="网页模板" target="_blank">aaa</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>