"
Page
Discussion
View source
History
teams
Log in
 
Team:Warsaw/Stage2/Background
From 2010.igem.org
Example Tabs
Main
Project
Stage 1
Stage 2
Stage 3
Wet Lab
Software
Human Practices
Safety
Team
Team
Attribution
Sponsors
Acknowledgemets
Idea
Background
Promoters Measured
RBSes Measured
ExpressIt! Vectors
Modeling
Idea
Background
Design
Results
Idea
Background and Design
Results
Gallery of microphotographs
Idea
Background
Design
Results
Modeling
Parts
Labbook
Licensing of biological parts
Syntetic biology in Poland
Synthetic BioLectures gallery
iGEM survey
About Biobrick Manager
See Biobrick Manager
MinC
Natural role:
MinC is the component of MinCDE system. Together with nucleoid occlusion it ensures that cell division will occur in the middle of a cell. Three proteins – MinC, MinD and MinE play different roles in preventing formation of division complex too close to cell poles [1]. MinC is directly responsible for stopping the early stage of division by inhibiting the polymerization of FtsZ protein monomers into a structure known as the Z-ring [8].
FtsZ is a tubuline homologue capable of creating chains, lariats and rings independently of other cellular factors [3]. The stability and decay of these structures is dependent on GTPase activity of  FtsZ proteins [7]. At the beginning of cell division FtsZ monomers form the Z-ring to which other proteins responsible for the division are recruited. The presence of N-terminal domain of MinC prevents additional Z-rings from being established [9].
MinCDE system and nucleoid occlusion[10].
In bacterial cells MinC is recruited to membrane by the second protein of the system – MinD. This process involves C-terminal domain of MinC, which is also responsible for its oligomerisation. Recruitment to membrane is necessary for the inhibiting effect to occur at the physiological concentration of protein [4]. Finally the third protein - MinE - is responsible for keeping the MinCD complex from acting in the midcell region, thus enabling the formation of Z-ring in the proper location [5].    It was shown that mutants in either minC or minD divide at cell pole much more often than wild type, resulting in creation of nucleoid-free minicells. This mutations, however are not lethal because enough of the cells in population divide properly to sustain growth [6].
Natural occurrence:
Although MinCDE is mostly analysed in E.coli, it’s elements were found in many different bacteria groups including Proteobacteria, Deinococcus-Thermus and Firmicutes phyla [1]. In B. subtilis the MinE protein is replaced by DivIVa which locates itself at the cell poles and with help of MinJ binds MinCD complex, decreasing it’s concentration at midcell [2].
Application in synthetic biology:
When expressed at high level MinC is capable of preventing bacterial cell from dividing. As a result, of this the cell becomes filamentus. This effect requires only high level of MinC, not MinD. It has been proven that overexpression of C-terminal domain of MinC alone is sufficient to inhibit cell division [9]. Recent findings suggest also, that MinC originated from E.coli is able to function properly in B.subtilis and inhibit its division [11].
References:
1. “Themes and variations in prokaryotic cell division”, William Margolin, FEMS Microbiology Reviews 24 (2000) 531-548
2. “The MinCDJ System in Bacillus subtilis Prevents Minicell Formation by Promoting Divisome Disassembly”, Suey van Baarle and Marc Bramkami, PLoS One. 5 (2010)
3. “The bacterial cell division protein FtsZ assembles into cytoplasmic ring in fission yeast”, Ramanujam Srinivasan et al. Genes Dev. 22 (2008) 1741-1746
4. “The Switch I and II Regions of MinD Are Required for Binding and Activating MinC”, Huaijin Zhou and Joe Lutkenhaus, J Bacteriol. 186 (2004) 1546–1555.
5. “The MinE ring required for proper placement of the division site is a mobile structure that changes its cellular location during the Escherichia coli division cycle.”, Fu X et al. Proc. Natl. Acad. Sci. U S A 98. (2001)
6. “FtsZ ring cluster in min and partition mutants: Role of both the Min system and the nucleoid in regulation FtsZ ring location”, Yu et al. Mol. Microbiol. 32 (1999) 315-326
7. “FtsZ from Divergent Foreign Bacteria Can Function for Cell Division in Escherichia coli”, Masaki Osawa and Harold P. Erickson, J. Bacteriol. 188 (2006) 7132-7140
8. “FtsZ, a tubulin homologue in prokaryote division”, Harold P. Erickson.Trends. Cell Biol. 7 (1997) 362-367
9. “Analysis of MinC Reveals Two Independent Domains Involved in Interaction with MinD and FtsZ”, Zonglin Hu and Joe Lutkenhaus, J. Bacteriol. 182 (2000) 3965-3971
10. "Synchronization of Chromosome Dynamics and Cell Division in Bacteria", Martin Thanbichler, doi: 10.1101/cshperspect.a000331(2009)
11. "Expression of Escherichia coli Min system in Bacillus subtilis and its effect on cell division" Nad'a Pavlendová, Katarína Muchová, Imrich Barák FEMS Microbiology Letters, 302 (2010) 58-68
Retrieved from " http://2010.igem.org/Team:Warsaw/Stage2/Background "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
