"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Warsaw/Stage1/PromMeas
From 2010.igem.org
Example Tabs
Main
Project
Stage 1
Stage 2
Stage 3
Wet Lab
Software
Human Practices
Safety
Team
Team
Attribution
Sponsors
Acknowledgemets
Idea
Background
Promoters Measured
RBSes Measured
ExpressIt! Vectors
Modeling
Idea
Background
Design
Results
Idea
Background and Design
Results
Gallery of microphotographs
Idea
Background
Design
Results
Modeling
Parts
Labbook
Licensing of biological parts
Syntetic biology in Poland
Synthetic BioLectures gallery
iGEM survey
About Biobrick Manager
See Biobrick Manager
Promoter measurement
Standardization
All measurements were done according to RFC 58 standard . Draft version of RFC document can be found here .
Experimental setup
Measured parts
We have measured J23100 and I719005 encoding pT7 .
J23100 with our reference RBS has RNA synthesis rate of 13,1pg RNA/minute/ug substrate DNA. It equals 2,80 microPoPS
pT7 with our reference RBS has RNA synthesis rate of 41,8pg RNA/minute/ug substrate DNA. It equals 8,92 microPoPS
Measurement constructs
We have used B0034 as our reference RBS and E0040 GFP as a reporter:
<partinfo>BBa_K299009 DeepComponents</partinfo>
<partinfo>BBa_K299024 DeepComponents</partinfo>
Results
Measured promoter strengths are as follows:
pT7 41,8pg RNA/minute/ug substrate DNA
J23100 13,1pg RNA/minute/ug substrate DNA
It means that from 1 ug of the DNA containing pT7 regulated reporter DNA polymerase made around 41 pg of RNA in a one minute. Is it a good unit for promoter activity? What happens when we input into our cell-free system DNA with the same BioBrick, but in a different vector? The amount of DNA changes, but it doesn't change the number of available promoters. Therefore we have to change those units to something reasonable - like number of RNA molecules made from 1 DNA molecule in a unit of time. Sounds like PoPs (polymerase per second) per DNA copy
Converting measured values into PoPs per molecule of substrate DNA
<partinfo>BBa_K299009 DeepComponents</partinfo> K299009 is 918 bp long
<partinfo>BBa_K299024 DeepComponents</partinfo> K299024 is 906 bp long
<partinfo>pSB1A2 DeepComponents</partinfo> Vector backbone has 2079 bp
Example calcultaions for BBa_K299024:
1) Approximated molecular weight of dsDNA (circular plasmid) = # nucleotides x 607.4 gram/mole.
BBa_K299024 in pSB1A2 is 2985 bp long.
Approximated molecular weight of  K299024 in pSB1A2 is 1813089  gram/mole.
in 0.5 ug there is 2,76E-013 moles of BBa_K299024 in pSB1A2 DNA (divide 0.5 ug by construct's molar mass)
2) Promoter is not transcribed therefore we assume that only the RBS sequence and gene sequence are transcribed. In both constructs RBS BBa_B0034 was used and the same GFP mut3b with B0015 terminator therefore  the same RNA is produced from two measured promoters.
Approximated molecular weight of ssRNA (mRNA transcript is  = # nucleotides x 320.5 + 159.0 (takes into account the 5'triphosphate) mRNA is single stranded and 1 Mole of single stranded RNA is 283160,5g.
3) Measured strength of BBa_K299024 is 4,18E-011 g RNA/min/ug DNA, that's 2,46E-018 mol RNA/sec/ug DNA (divide by molar mass of mRNA to convert from g to moles and then divide by 60 to convert from minutes to seconds)
4) Strength of BBa_K299024 expressed in PoPS is 8,92E-006 (divide the result from point 3) by amount of moles of DNA construct in 0.5 ug)
Same calculations for BBa_K299009 will give 2,80E-006 PoPS
Where did we get the experimental values from?
We expressed known amount if DNA in a cell-free expression system and measured RNA content over time using RTq-PCR. Simply we put the DNA into the tube with the cell-free expression system and collected samples of the mixture every 15 minutes. Then we did the reverse transcription to be able to perform a q-PCR that gives information on the amount of starting material.
Figure 1 shows you the dynamic performance of J23100 and T7 promoters - the amount of RNA versus time. Figure 2 show amplification curves that we got after RTq-PCR.
Fig 1. Dynamic performance of J23100 and pT7.
Fig 2. The amplification curves that were obtained after RTq-PCR seperately for J23100 and T7 containing construct. Different colours indicate that samples were collected at different times: 15, 30. 45, etc. minutes after DNA addition. Dark blue is amplification curve for sample after 15 minutes and light blue is after 90, as you can see there is an increase in the RNA content.
Methodology
All measurements were conducted in cell-free system which allowed us simple and precise determination of GFP encoding mRNA. The amount of mRNA was determined using quantitative reverse transcriptase realtime PCR (qRT-PCR). We have done absolute quantification using cDNA standard curve to convert delta-Ct units to RNA concentration.
We have used following protocol:
All reagents and substrates were RNase free. Experiments were conducted in RNase-free environment.
0.5 ug of DNA encoding tested construct was added to 50 ul of cell-free expression master mix containing 350 units of human placental RNase inhibitor.
Samples were incubated at 37oC with shaking at 800 RPM
Every 15 minutes 5ul of reaction mixture was collected. Reaction was stopped by freezing at -20. Samples were kept frozen until reverse transcription.
Subsamples were being collected untill reaction have reached steady state (typically 120 minutes).
After obtaining all RNA samples DNAse treatment was performed as follows: 5 ul of sample was supplemented with 1ul of 10x DNAse buffer, 3 ul of RNase-free water and 1ul of RNase-free DNase I from Fermentas
Samples were incubated at 37oC for 30 minutes. After that time 1ul of EDTA was added to each sample.
DNase was inactivated by heating in 65oC for 10 minutes.
DNase treated RNA samples were divided in two. One half was used as a substrate for reverse transcription. The other halves were mixed together and used in -RT control reaction.
Reverse transcription was performed using Maxima First Strand cDNA synthesis kit form Fermentas using manufacturer's instructions. Gene specific primer GFPqPCRr (TCGAAAGGGCAGATTGTG) was used.
cDNA was diluted 50x and 1ul was used for qRT-PCR reaction. SYBR/ROX qPCR HotStart 2x Master Mix for Fermentas was used to perform the reaction with the following primers: GFPqPCRf (GATGACGGGAACTACAAGAC) and GFPqPCRr (TCGAAAGGGCAGATTGTG). ABI 7500 qPCR system was used. PCR program: 95oC for 10 minutes followed with 40 cycles of 95oC for 15s, 55oC for 30 s, 72oC for 40s. Reaction specificity was confirmed using melting curve analysis.
Retrieved from " http://2010.igem.org/Team:Warsaw/Stage1/PromMeas "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
