"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Warsaw/Stage1/Background
From 2010.igem.org
Example Tabs
Main
Project
Stage 1
Stage 2
Stage 3
Wet Lab
Software
Human Practices
Safety
Team
Team
Attribution
Sponsors
Acknowledgemets
Idea
Background
Promoters Measured
RBSes Measured
ExpressIt! Vectors
Modeling
Idea
Background
Design
Results
Idea
Background and Design
Results
Gallery of microphotographs
Idea
Background
Design
Results
Modeling
Parts
Labbook
Licensing of biological parts
Syntetic biology in Poland
Synthetic BioLectures gallery
iGEM survey
About Biobrick Manager
See Biobrick Manager
Why iGEM Warsaw team spends summer measuring RBS parts and how you can predict RBS strength.
Gene expression control freaks
During the brainstorming period of our project we came up with many interesting ideas and we tried to find a way to squeeze all those ideas into one E. coli cell. It lead us to a general problem: how many different genes we can insert into our Coli at one time and how precisely regulate their dosage? The more genes we put inside our Coli the more important regulation is. Simply because resources needed to produce proteins are limited. We decided to use different RBS sequences to control gene expression.
We knew from the registry that B0033 is weak and for strong expression we should use B0034, but also we have learned that those RBSes were measured in one setup with standard promoter and GFP reporter gene. We faced two general questions:
1) Are those 5 measured RBSes in the registry enough? Does it make sense to to use any other RBSes?
We went downstairs, ordered lots of coffee, switched on the laptops and asked google scholar for help.  We came across a Nature paper about standardization of synthetic biological parts and devices [1]. The parts' evolutionary reliability measurement was particularly interesting for us. Paper suggest that after 74 culture doublings the measured part mutated and wasn't functional anymore. System failure resulted from deletion between repeated DNA sequences due to recombination. In that specific system it occurred because terminator B0015 was used 2 times.
This scenario would be probably  repeated if we used the same RBS parts. However the experiment was performed in the recA+ strain - in other words a strain that is capable of homologous recombination. 2009 Warsaw team used B0032 many times to obtain moderate gene expression, because Top10 a (recA-) strain was used there were no recombination events.
CONCLUSIONS: In the light of these evidence we wanted to refine our design and avoid repeated sequences to make our system portable. This is a problem if you want to use RBSes of similar strength, because there is seldom more than one characterized.
2) RBSes were measured with GFP reporter. Are these RBS measurements universal.
Would RBS behave differently if we put our favorite gene instead of GFP?
A first hint on how RBS strength is regulated comes from 1990 Van Duin et al. paper [2]. They mutated RBS of a phage coat protein gene and came to the conclusion that the stronger the RBS secondary structure is, the weaker the gene expression is. This relationship is further explored in later papers [ 3 , 4 ]. In our eyes it crushed the idea of RBS measurement. Why? The secondary structure that is formed by RBS is influenced by the downstream sequence - the beginning of the gene used. Therefore RBS measured with GFP might behave differently in other genetic setups.
We read a bit more and decided to investigate it computationally. Curious to find out if RBS parts have a chance to behave similarly with different genes? See our modeling .
Some helpful papers:
[1]Refinement and standardization of synthetic biological parts and devices, Barry Canton1, Anna Labno, Drew Endy, Nature Biotechnology 2008.
[2]Secondary structure of the ribosome binding site determines translational efficiency: a quantitative analysis, M.H.de Smit and J.van Duin, PNAS 1990
[3]Stabilised secondary structure at a ribosomal binding site enhances translational repression in E. coli. Brunel C, Romby P, Sacerdot C, de Smit M, Graffe M, Dondon J, van Duin J, Ehresmann B, Ehresmann C, Springer M. J Mol Biol. 1995
[4]Quantitative correlation between mRNA secondary structure around the region downstream of the initiation codon and translational  efficiency in Escherichia coli.Seo SW, Yang J, Jung GY. Biotechnology and Bioengineering. 2009
Retrieved from " http://2010.igem.org/Team:Warsaw/Stage1/Background "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
