<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Waterloo_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Waterloo/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Waterloo/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Waterloo/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Waterloo/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Waterloo/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Waterloo/Project</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="titleHeader"><TABLE cellspacing="0px" cellpadding="0px"><TBODY><TR><TD id="mid" width="975px" height="26px" class="titleLink">  <A href="https://2009.igem.org/Team:Waterloo">home</A> |
            <A href="https://2009.igem.org/Team:Waterloo/Team">team</A> |
            <A href="https://2009.igem.org/Team:Waterloo/Project">project</A> |
            <A href="https://2009.igem.org/Team:Waterloo/Parts">parts submitted to the registry</A> |
            <A href="https://2009.igem.org/Team:Waterloo/Modeling">modeling</A> |
            <A href="https://2009.igem.org/Team:Waterloo/Notebook">notebook</A> |
            <A href="https://2009.igem.org/Team:Waterloo/Human_Practices">human practices</A></TD></TR></TBODY></TABLE></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Abstract:_Chromobricks:_A_Platform_for_Chromosome_Engineering_with_BioBricks"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Abstract: Chromobricks: A Platform for Chromosome Engineering with BioBricks</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Project_Details"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project Details</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Background"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#The_.CE.A6C31_Integrase"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">The ΦC31 Integrase</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Our_BBa_Compatible_Cassette_Exchange_System"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Our BBa Compatible Cassette Exchange System</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#The_Goal_of_on_Chromosome_Assembly_of_Interchangeable_Parts"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">The Goal of on Chromosome Assembly of Interchangeable Parts</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Selective_Chromosome_Degradation"><SPAN class="tocnumber">2.1.4</SPAN><SPAN class="toctext">Selective Chromosome Degradation</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#The_Experiments"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">The Experiments</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Integrase-Mediated_Cassette_Exchange_.28IMCE.29"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Integrase-Mediated Cassette Exchange (IMCE)</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#BBa_Donor_Plasmid"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">BBa Donor Plasmid</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Landing_Pad_Strain"><SPAN class="tocnumber">3.1.2</SPAN><SPAN class="toctext">Landing Pad Strain</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Integrase_expression_plasmid"><SPAN class="tocnumber">3.1.3</SPAN><SPAN class="toctext">Integrase expression plasmid</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Non-cross-reactive_att_sites"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Non-cross-reactive att sites</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Integrase_Mechanism"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">Integrase Mechanism</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Design_of_new_att_sites"><SPAN class="tocnumber">3.2.2</SPAN><SPAN class="toctext">Design of new att sites</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Design_of_experiment_to_characterize_new_att_sites"><SPAN class="tocnumber">3.2.3</SPAN><SPAN class="toctext">Design of experiment to characterize new att sites</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Selective_Chromosome_Degradation_2"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Selective Chromosome Degradation</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Landing_Pad_Strain_2"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Landing Pad Strain</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#BBa_Donor_Plasmid_2"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">BBa Donor Plasmid</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Integrase_Expression_Plasmid_2"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Integrase Expression Plasmid</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Abstract:_Chromobricks:_A_Platform_for_Chromosome_Engineering_with_BioBricks"><B> Abstract: </B>Chromobricks: A Platform for Chromosome Engineering with BioBricks </SPAN></H2><P>The aim of our project is to develop a fully-featured platform for chromosome engineering, allowing the in vivo assembly of a synthetic chromosome from interchangeable parts, followed by selective degradation of the native chromosome. We have designed a proof-of-concept for chromosome-building that will use the site-specific integrase of phage ΦC31 to integrate a BioBrick into a defined locus of the <I>E. coli</I> genome. Six pairs of integrase-targeted <I>att</I> sites have been designed to be non-cross-reactive in order to support repeatable cassette-exchange reactions for chromosome building. We have also written software to model the integrase-mediated rearrangement of DNA molecules containing <I>att</I> sites, to aid the design of more elaborate chromosome-building systems. To selectively degrade the native chromosome we designed a nuclease-based, inducible genome-degradation system. In its simplest form, our system can be used to integrate biological devices into a chromosome in situations requiring stable copy number and selection-free maintenance.
</P><H2><SPAN class="mw-headline" id="Project_Details">Project Details</SPAN></H2><H4><SPAN class="mw-headline" id="Background">Background</SPAN></H4><P>Chromosome Engineering provides many benefits over plasmid based engineering, including precise copy number and stable maintenance without selection. Imprecise copy number plasmid-based systems can add unnecessary noise to synthetic biology systems, limiting the potential complexity of these systems. However, a plasmid can easily be isolated, engineered with restriction enzymes and reintroduced into a host strain. Chromosomes cannot be engineered in this way. 
</P><P>Traditionally, methods to engineer chromosomes in bacteria have been based on homologous recombination, which is an inefficient process. Other methods have taken advantage of recombinant lysogenic phage, such as the <I>E. coli</I> phage λ. These phage integrate their genome into the bacterial chromosome via a site-specific recombination. Genes that have been added to the small phage genome will be carried along. The mechanisms of phage genome integration have been studied and there are some well characterized systems. These systems can be separated from the phage itself. They can be used to engineer DNA molecules <I>in vitro</I> or <I>in vivo</I> without actually infecting the bacteria with a phage (Groth &amp; Calos, 2004).
</P><P>The phage integration system usually involves an integrase/recombinase enzyme, the phage attachment site (<I>attP</I>), the bacterial attachment site (<I>attB</I>), and for the reverse reaction an excisionase enzyme. The integrase enzyme catalyzes a stand exchange between  <I>attP</I> and <I>attB</I> sites on the DNA yielding <I>attL</I> and <I>attR</I> sites which are composites of <I>attP</I> and <I>attB</I> sites. If there is excisionase and integrase present the reaction will go in the reverse direction. The well known λ integration system is the basis of Invitrogen's Gateway® technology. Gateway® technology uses the activity of the λ integrase and excisionase to carry out cassette exchange reactions, where sections of two molecules each flanked by respective <I>att</I> sites are exchanged. The system is used to engineer plasmids as an alternative to traditional cloning using restriction enzymes. For our project, the goal of engineering chromosomes without conflicting with the widely adopted Gateway® system led us to choose the integrase from the <I>Streptomyces</I> phage C31 (ΦC31).
</P><H5><SPAN class="mw-headline" id="The_.CE.A6C31_Integrase">The ΦC31 Integrase</SPAN></H5><P>The ΦC31 integrase catalyzes a unidirectional strand exchange between a 39 bp <I>attP</I> site and a 34 bp <I>attB</I> site (Groth et al., 2000). The enzyme works by making a synapse between <I>attP</I> and <I>attB</I>, making double strand breaks producing a 2 bp sticky overhang, exchanging the strands, and re-ligating them in the recombinant configuration (<I>attL</I> and <I>attR</I>).
</P><H5><SPAN class="mw-headline" id="Our_BBa_Compatible_Cassette_Exchange_System">Our BBa Compatible Cassette Exchange System</SPAN></H5><P>Due to the lack of any BBa compatible chromosome engineering tools we decided to build a cassette exchange system based on the ΦC31 integrase. The system can be broken down into three components: the landing pad strain (LPS), the helper plasmid, and the BBa donor vector.
</P><P>The protocol to integrate a BioBrick into the chromosome of the LPS involves a conjugation of a section of the donor plasmid via bi-parental mating into the LPS which is also expressing ΦC31 integrase. Then the LPS is selected  with naladixic acid, and the loss of the landing pad markers is selected with sucrose. The colonies would then be screened for kanamycin sensitivity. The LPS contains a Landing Pad cassette in the melA locus. This cassette consists of <I>sacB</I>,<I>rfp</I>,and <I>nptII</I>(kanamycin resistance) genes flanked by ΦC31 <I>attP</I> sites. After cassette exchange these markers are lost. The BioBrick will replace them.
</P><H5><SPAN class="mw-headline" id="The_Goal_of_on_Chromosome_Assembly_of_Interchangeable_Parts">The Goal of on Chromosome Assembly of Interchangeable Parts</SPAN></H5><P>With the design of non-cross reactive <I>att</I> sites it would be possible to regenerate the landing pad after a successful integration. This would be done by having a second non-cross-reactive pair of <I>att</I> sites downstream of our integrated BioBrick (<I>attP2 attP2</I>). A landing pad regeneration plasmid with attB2 sites flanking a landing pad would be introduced and a ΦC31 integrase reaction would integrate the landing pad. Another <I>attP1 attB1</I> reaction could be carried out to integrate another BioBrick. The problem with an approach like this is that <I>attL</I> sites would build up. We decided to write software to try and solve this problem. The software is further discussed in the modelling section.
</P><P>We have designed six different <I>attB</I> sites and six different <I>attP</I> sites, giving us six non-cross-reactive pairs. This is discussed further below.
</P><H5><SPAN class="mw-headline" id="Selective_Chromosome_Degradation">Selective Chromosome Degradation</SPAN></H5><P>There are examples in the literature of whole genomes being reconstructed in a locus of an existing chromosome. One of these examples is assembly of the <I>Synechocystis</I> PCC6803 genome in the
<I>Bacillus subtilis</I> 168 genome (Itaya et al., 2005). This was done through a tedious homologous recombination based approach. 
</P><P>We figured that in the future improved chromosome engineering systems would be used to assemble synthetic genomes in host cells. After that it would be possible to circularize the synthetic genome and degrade the native genome. This would essentially change the species of the cell.
</P><P>As a proof of concept we have designed an inducible nuclease construct that will express a restriction endonuclease (PmeI) and an exonuclease (T7 gene 6 exonuclease). As long as the synthetic chromosome does not contain a PmeI site it will not be degraded.
</P><H2><SPAN class="mw-headline" id="The_Experiments"> The Experiments </SPAN></H2><H3><SPAN class="mw-headline" id="Integrase-Mediated_Cassette_Exchange_.28IMCE.29">Integrase-Mediated Cassette Exchange (IMCE)</SPAN></H3><P>The goal of this aspect of the project was to create a system capable of integrating a BioBrick into the chromosome of <I>E. coli</I> at a defined locus. The steps required for integration, baring initial cloning of the BioBrick, would be carried out <I>in vivo</I>.
</P><P>The basic idea behind integrase-mediated cassette exchange (IMCE) is pictured below.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:667px;"><DIV class="thumbcaption">Integrase Mediated Cassette Exchange (IMCE): The reactants are shown on the left and the products, produced by the integrase catalyzed recombination reaction, are shown on the right</DIV></DIV></DIV></DIV><P>We designed our system so that a plasmid carrying a BioBrick would by transferred into the Landing Pad Strain (LPS) via conjugation in such a way that it would be lacking an origin of replication. The integration reaction would take place causing the markers to be lost due to their transfer to a non-replicating molecule. In the case of our system the markers are <I>nptII</I> (kanaymycin resistance), <I>sacB</I> (sucrose sensitivity), and <I>rfp</I>. That way integrants can be selected on media containing sucrose. The design also calls for the landing pad strain to be nalidixic acid resistant so that it can be selected for after conjugation. Therefore integrants can be selected for in one step on a nalidixic acid-sucrose plate.
</P><H5><SPAN class="mw-headline" id="BBa_Donor_Plasmid">BBa Donor Plasmid</SPAN></H5><P>The donor plasmid is the intermediate vector in which the target BioBrick is first cloned into so that it can transferred into the LPS. This requires the prefix and suffix to be flanked by <I>attB</I> sites which are embedded in origin of transfer (<I>oriT</I>) sites. The two <I>oriT</I> sites ensure that in a conjugation either the BioBrick flanked by <I>attB</I> sites is transferred, or the backbone without <I>attB</I> is transferred. Only the former can support cassette exchange. Only cells in which cassette exchange has occurred will grow on media suplemented with sucrose and naladixic acid, because  <I>sacB</I> is contained within the LPS, which is naladixic acid resistant. This design solves the problem of finding a vector that will not replicate in <I>E. coli</I>. It allows for easy manipulation of the donor plasmid in regular <I>E. coli</I> strains through the usual cloning methods. 
</P><P>The plasmid backbone will contain an <I>rfp</I> BioBrick (<A href="http://partsregistry.org/Part:BBa_K093010" class="external text" rel="nofollow">BBa_K093010</A>), making it easy to replace <I>rfp</I> with a BioBrick of your choice. White colonies contain the part that is to be integrated into the chromosome red colonies still contain <I>rfp</I>. At this point the donor plasmid is complete and resembles the left plasmid shown in the Cassette Exchange figure above. This construct can now be transformed into the LPS and with a ΦC31 integrase expressing plasmid available, the BioBrick of interest and the landing pad markers will be exchanged.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption">The plasmid will initially contain an <I>rfp</I> BioBrick which is replaced with the target BioBrick</DIV></DIV></DIV></DIV><H5><SPAN class="mw-headline" id="Landing_Pad_Strain">Landing Pad Strain</SPAN></H5><P>The landing pad strain provides a chromosomal substrate for IMCE. This substrate is the landing pad cassette (LPS). The LPS consists of markers flanked by <I>attP</I>’s that is integrated, via homologous recombination, into the chromosome of the host strain, a naladixic acid resistant isolate of <I>E.coli K12 MM294A</I>. (The host strain must have RecA activity for homologous recombination to occur). The landing pad contains the screening and selection markers: <I>rfp</I> (red fluourescent protein), <I>nptII</I> (kanamycin resistance), and <I>sacB</I> (sucrose sensitivity). In the cell, these markers produce easily detectable phenotypes that allow us to select for the occurrence of cassette exchange, and confirm those occurrences with a simple screen.
</P><P>The entire landing pad construct is built on a suicide vector, which we called pFB9009, containing a streptomycin resistance gene on its backbone. Integration via homologous recombination is required for kanamycin resistance. Integrants can be selected on kanamycin. Double cross-over integrants, containing only the landing pad, can be found by screening kanamycin resistant colonies for streptomycin sensitivity.
</P><H5><SPAN class="mw-headline" id="Integrase_expression_plasmid">Integrase expression plasmid</SPAN></H5><P>The integrase expression plasmid would contain a ΦC31 integrase generator BioBrick (<A href="http://partsregistry.org/Part:BBa_K093017" class="external text" rel="nofollow">BBa_K093017</A>). The BioBrick would be cloned into the chloramphenicol resistant backbone pSB3C5.
</P><P>Primers were designed to amplify the ΦC31 integrase ORF, while adding prefix and suffix sequences, and making a synonymous mutation to abolish an <I>EcoRI</I> site contained within the ORF.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption">Integrase is expressed by a constitutive promoter and is placed on a low copy number plasmid</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Non-cross-reactive_att_sites">Non-cross-reactive <I>att</I> sites</SPAN></H3><P>We set out to design and test 6 different <I>att</I> site pairs that would be non-cross-reactive, or “orthogonal”. This would allow for increased specificity when inserting DNA pieces into the chromosome. It would also allow for repeatable insertion of DNA pieces into the chromosome as long as a build up of <I>attL</I> sites was not an issue.
</P><P>First we discuss the mechanism of the integrase; then we move on to the design and testing of our orthogonal <I>att</I> sites.
</P><H5><SPAN class="mw-headline" id="Integrase_Mechanism">Integrase Mechanism</SPAN></H5><P>The ΦC31 integrase catalyzes a unidirectional strand exchange between a 39 bp <I>attP</I> site and a 34 bp <I>attB</I> site. The enzyme works by making a synapse between <I>attP</I> and <I>attB</I>, making double strand breaks producing a 2 bp sticky overhang, exchanging the strands, and re-ligating them in the recombinant configuration (<I>attL</I> and <I>attR</I>). This reaction is not reversible with only the integrase present. The <I>att</I> sites can be mutated to produce different overhangs. A mutant <I>attB</I> with different a different overhang, or core dinucleotide, has been shown, in a related system, not to recombine with <I>attP</I> (Ghosh et al., 2003). Instead of a recombination activity the integrase showed a topoisomerase activity, reducing supercoiling on a plasmid, suggesting the double stranded breaks were made but the DNA would not re-ligate in the recombinant configuration (Ghosh et al., 2003). When identical mutations are made to the overhangs the recombination activity is restored (Ghosh et al., 2003), implying that making orthogonal <I>att</I> sites (non cross-reactive site pairs) for the ΦC31 integrase is possible.
</P><P>The sequence of the overhangs also has implications in the directionality of the recombination, which can lead to different end results of recombination (e.g., excision or inversion) (Ghosh et al., 2003). The palindromic overhangs will lead to loss of directional specificity. Given the 16 possible core dinucleotide sequences six of them should be completely non-cross reactive while maintaining direction specificity. 
</P><P>The integrase mechanism is shown below. The integrase protein forms a dimer about each <I>att</I> site. When the <I>att</I> sites are brought together, presumably through the two dimers recruiting each other, a tetramer is formed. When the tetramer is formed between an <I>attB</I> and an <I>attP</I> the small N-terminal catalytic domain is activated. A double strand is made. For each protein a covalent linkage between a serine residue and its respective 5′ phosphate is made. The complex rotates 180° and re-ligates the DNA in a recombinant configuration.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">the role of the core dinucleotide is shown. If the <I>att</I> sites had synapsed anti-parallel instead of parallel TT would have to pair with TT in order to form recombinant products. This is not possible. Therefore only synapses that align in the parallel orientation generate recombinant products.</DIV></DIV></DIV></DIV><H5><SPAN class="mw-headline" id="Design_of_new_att_sites">Design of new <I>att</I> sites</SPAN></H5><P>The above figure shows that the completion of the integrase mediated recombination reaction relies on the successful base-pairing of the two base-pair 3′ overhangs between <I>attB</I> and <I>attP</I> halves. If there is a mutation that disrupts this base-pairing the integrase complex will continue to rotate and the products will be reformed. As mentioned earlier, the base-pairing of the 3′ overhangs produced by the core dinucleotides also contains which orientation in which the <I>att</I> sites can recombine. If the core dinucelotide is palindromic directional specificity is lost (Ghosh et al., 2003). This also implies reactivity between <I>att</I> sites with reverse compliment core dinucleotides.
</P><P>To make orthogonal <I>att</I> sites for ΦC31 integrase, <I>att</I> sites with different core dinucleotides (3′ overhangs) were designed. We avoided palindromic core dinucleotides (GC, TA, etc.) and reverse compliment core dinucleotides (TT with respect to AA). Of the sixteen possible dinucleotides four are palindromic. Twelve are not palindromic, and will confer directionally specific recombination. Those twelve consist of 6 pairs of sense and reverse compliment dinucleotides. That leaves six mutually exclusive dinucleotides that should display no cross-reactivity at all. We chose TT, GG, TG, AC, GA, and CT dinucleotides. 
</P><P>The <I>att</I> sites are shown below with the core dinucleotide bolded. The only difference between orthogonal <I>att</I> sites is the core dinucleotide.
</P><P>attB34 TT:  5′-GGTGCCAGGGCGTGCCC<B>TT</B>GGGCTCCCCGGGCGCG- 3′
</P><P>attP39 TT: 5′-CCCCAACTGGGGTAACCT<B>TT</B>GAGTTCTCTCAGTTGGGGG- 3′
</P><H5><SPAN class="mw-headline" id="Design_of_experiment_to_characterize_new_att_sites">Design of experiment to characterize new <I>att</I> sites</SPAN></H5><P>To test the functionality and quantify the efficiency of recombination between six different <I>att</I> site pairs an <I>in vivo</I> recombination experiment was designed.
</P><P>Complimentary oligonucleotides were used to synthesize the <I>att</I> sites, and to serve as linkers for ligation into different vectors. 
</P><OL><LI> The <I>attP</I> sites were to be cloned into a very narrow host range vector that only replicates in select strains of <I>E. coli</I> SM10 (λ pir lysogens). This vector carries streptomycin resistance. 
</LI><LI> The <I>attB</I> sites were cloned into pSB1A2, which carries ampicillin resistance. This vector replicates in most <I>E. coli</I> strains.
</LI><LI> The <I>attB</I> plasmid was to be transformed into <I>E. coli</I> DH5α carrying the integrase expressing plasmid.
</LI><LI> The <I>attP</I> plasmid was to be conjugated from <I>E.coli</I> SM10 into the above DH5α (<I>attB</I> and integrase plasmid carrying) strain.
</LI><LI> DH5α is nalidixic acid resistant SM10 is not. The streptomycin resistance carrying <I>attP</I> plasmid does not replicate in DH5α.
</LI><LI> The cells from the conjugation mixture would be plated on media containing naladixic acid and streptomycin. The only way a streptomycin resistance can be maintained is thorough integrase mediated recombination. 
</LI><LI> The number of colonies relative to a positive control (conjugation into DH5α λ pir) would provide efficiency of recombination numbers.
</LI></OL><P>Conjugation was chosen instead of transformation because of conjugation's consistent efficiency between experiments. Transformation efficiencies vary widely between experiments.
</P><H3><SPAN class="mw-headline" id="Selective_Chromosome_Degradation_2">Selective Chromosome Degradation</SPAN></H3><P>The aim of parts project is to engineer a genome-free, cell-based expression system capable of producing a desired protein or activating a pathway in response to an environmental signal. Genome degradation is achieved using the combined activity of a restriction endonuclease to fragment the genome and an exonuclease to hasten degradation. The gene for the protein of interest will be located in a plasmid lacking recognition sites for the endonuclease, allowing it to remain intact after genome degradation. The plasmid genes will be expressed using the remaining cell resources until they expire. The primary application of this design would be an in situ compound production and delivery system for agricultural, industrial or therapeutic use to continue for a period of time.
</P><P>The construct to be integrated into the host chromosome has been completed without the sense promoter. Thus, to date the construct is as follows: RBS - T7 Gene6 endonuclease - RBS - PmeI exonuclease - Plac - TT - PlacI - CI - TT
In construction, it was essential to clone the sense promoter in last to ensure no expression of the nucleases occurred before the system was induced.
</P><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><H4><SPAN class="mw-headline" id="Landing_Pad_Strain_2">Landing Pad Strain</SPAN></H4><P>The landing pad is complete in the suicide vector, pFB9009, and is currently in the progress of being introduced into the <I>E. coli</I> K12 MM294A host strain. Further work includes screening for successful recombinants. Double recombinants are expected to be resistant to kanamycin, while being sensitive to streptomycin and sucrose. The disruption of the <I>melA</I> gene, detectable by the lack of α-galactosidase activity and the inability to grow on melibiose, indicates a correctly positioned landing pad in the chromosome. Then the strain will be plated on nalidixic acid supplemented media. Spontaneous naladixic acid resistant colonies will be issolated. Mutations, causing resistance to nalidixic acid are relatively frequent, and issolation of these mutants is commonplace.
</P><H4><SPAN class="mw-headline" id="BBa_Donor_Plasmid_2">BBa Donor Plasmid</SPAN></H4><P>The donor plasmid assembly began with the addition of the left <I>oriT</I> and <I>attB</I> sites in one fragment. The assembly was verified by sequencing. The next step was the addition of the right <I>oriT</I> and inverted <I>attB</I> sites. The final assembly could not be verified since the sequencing revealed an insertion of <I>E. coli</I> DH5α genomic DNA between the cloning site and the suffix. Future strategies include the selection of a new vector backbone and cloning the <I>oriT</I> and <I>attB</I> sites using a different set of enzymes to avoid the possible integration of host genomic fragments.
</P><H4><SPAN class="mw-headline" id="Integrase_Expression_Plasmid_2">Integrase Expression Plasmid</SPAN></H4><P>The expression plasmid using the J23119 constitutive promoter was isolated and sequencing results revealed randomly situated insertions deletions of one or two bp as well as a few point mutations in other colonies. The <I>EcoRI</I> site was modified in a very low proportion of clones but they also had frameshift mutations and nonsynonymous mutations. The integrase ORF was next cloned into a series of expression vectors with different constitutive promoter strengths, ranging from 16% to 55%, <A href="http://partsregistry.org/Part:BBa_J23118" class="external text" rel="nofollow">BBa_J23118</A>, to note effects of different metabolic loads. The new set of plasmids were also rendered unusable due to similar mutations as the plasmid using J23119.
The next step will be either to synthesize the gene by PCR using overlapping oligonucleotides or to clone the integrase ORF from an existing plasmid with verified integrase activity to create a non-BioBrick plasmid for the purpose of testing the donor plasmid and landing pad for cassette exchange.
</P><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P><B>Ghosh, P., Kim, A. I. &amp; Hatfull, G. F. (2003).</B> The orientation of mycobacteriophage Bxb1 integration is solely dependent on the central dinucleotide of attP and attB. <I>Mol Cell</I> 12, 1101-1111.
</P><P><B>Groth, A. C., Olivares, E. C., Thyagarajan, B. &amp; Calos, M. P. (2000).</B> A phage integrase directs efficient site-specific integration in human cells. <I>Proc Natl Acad Sci</I> U S A 97, 5995-6000.
</P><P><B>Groth, A. C. &amp; Calos, M. P. (2004).</B> Phage integrases: biology and applications. <I>J Mol Biol</I> 335, 667-678.
</P><P><B>Itaya, M., Tsuge, K., Koizumi, M. &amp; Fujita, K. (2005)</B>. Combining two genomes in one cell: Stable cloning of the <I>Synechocystis</I> PCC6803 genome in the <I>Bacillus subtilis</I> 168 genome. <I>Proceedings of the National Academy of Sciences</I> 102, 15971-15976.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Waterloo/Project">http://2009.igem.org/Team:Waterloo/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Waterloo/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Waterloo/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Waterloo/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Waterloo/Project&amp;oldid=167782" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>