<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_WashU_Notebook_July2011"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:WashU/Notebook/July2011">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:WashU/Notebook/July2011&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:WashU/Notebook/July2011&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:WashU/Notebook/July2011&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:WashU/Notebook/July2011" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:WashU/Notebook/July2011</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="topmenu"><UL><LI><A href="https://2011.igem.org/Team:WashU">Home</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Team#Team_Members">Team Members</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Team#Attributions">Attributions</A></LI></UL><LI><A href="https://2011.igem.org/Team:WashU/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Project#Abstract">Abstract</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Project#Project_Description">Project Description</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Project#Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Modeling">Modeling</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Modeling#Experimental_Plan">Experimental Plan</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Parts">Parts</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Notebook">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Notebook/MolBio">Molecular Biology Group</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Notebook/PCR">PCR Group</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Notebook/Transformation">Transformation Group</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Notebook/Assay">Assay Group</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Notebook">By Date</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Sponsors">Sponsors</A></LI></DIV><H2><SPAN class="mw-headline" id="July_1"> July 1 </SPAN></H2><P>Master mix composition (6 wells worth)
</P><UL><LI>15.0uL 10x buffer
</LI><LI>7.5 uL dNTP
</LI><LI>99 uL dH20
</LI><LI>1.5 uL Accutaq LA DNA polyermerase 
</LI></UL><P>= 20.5 ul/well 
</P><UL><LI>1.5 uL forward primer 
</LI><LI>1.5 uL reverse primer 
</LI></UL><UL><LI>10ng Kan DNA (1mL using our 2x diluted sample)
</LI><LI>1.5ul of Nat ligation product
</LI></UL><P>2nd attempt at PCR on the Nat ligation product: 
</P><UL><LI>Nat was not detected on a gel but may be because a very small amount of product was made. 
</LI><LI>Well 1,2,3 are PCR mix with Nat primers and 1.5uL of Nat ligation product per well
</LI><LI>Well 4 is positive control with Kan primers and Kan DNA
</LI><LI>Well 5 just has PCR mix with the Nat primers.
</LI></UL><P>Gel Team:
</P><UL><LI>Did a restriction digest on nat cassette using AgeI
</LI><LI>set up ligation reaction and let run over weekend
</LI></UL><H2><SPAN class="mw-headline" id="July_5"> July 5 </SPAN></H2><P>Ran PCR to amplify 100x diluted sample of URA3
positive control contained mastermix, LEU2 primers and 100x diluted LEU2 DNA.
negative control contained mastermix, 100x diluted URA3
</P><P>Gel Results: 
</P><P>Note: this PCR reaction failed, including the positive control
</P><P>PCR Protocol for Plasmids Review
</P><UL><LI>2.5uL 10x buffer
</LI><LI>1.25 uL dNTP
</LI><LI>16.5 uL dH20
</LI><LI>0.25 uL Accutaq LA DNA polyermerase 
</LI></UL><UL><LI>1.5 uL forward primer 
</LI><LI>1.5 uL reverse primer 
</LI><LI>1.0 ul diluted DNA
</LI></UL><P>Gel Team
</P><UL><LI>ran a gel of the ligation reaction from 7/1
</LI><LI><A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/5.jpg?attachauth=ANoY7crrAsrSGJwv64GdKkjnlUdwaJwjJ5ITE7xw6sSCy0hz7hG6LyjA86yasxYGizutyTwWiCUv5QJ6lqCcqW6gP51apVBUVQgthnd7WcJdCGVk8nIr5pORwUoG3xLK9g6jI8q1Ll5AzT3BokRYgLUvcplzJubLrYbktjQhbJgotzlKQQo31Tlgq3pfF6knRVKpiZlIsBvv&amp;attredirects=0" class="external autonumber" rel="nofollow">[1]</A></LI></UL><P>Assay Preparation:
</P><UL><LI>Performed yeast growth curve on BC177 and BC178 to see how long it took for each strain to reach mid-log phase from a single colony on a YPD plate.
</LI></UL><H2><SPAN class="mw-headline" id="July_6">July 6</SPAN></H2><P>Gel team:
</P><UL><LI>ran a gel of Leu 2 cassettes
</LI><LI><A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/6.jpg?attachauth=ANoY7cqCB9plxMPLpY-9Suw09cpHJxxcwNTfuw2ObI2NL9Z9ZzM7U3isUf5GDCNdxgR_P64EtKfKP_EJUK9kLoO_yYS4a2nAEMvsHCInCtR_4z87D6PrWrPMsIQYhtI8SB7ZsgURr-Fj7mcoRe1WXEg1_gNgy9FYDYYLhUPHVhf6P83ZKhR7gIEnk_4vidF5ApG_rACRbmB-&amp;attredirects=0" class="external autonumber" rel="nofollow">[2]</A></LI></UL><P>Assay Preparation:
</P><UL><LI>Prepared samples of BC177 and BC178 from overnight cultures. Pelleted via centrifuging and lysed with glassed beads. The lysed yeast extract was analyzed in the spectrophotometer.
</LI></UL><H2><SPAN class="mw-headline" id="July_7"> July 7 </SPAN></H2><P>Note: the 10x dilution of URA3 was the most effective, positive with LEU2 failed.
Reran PCR reaction to amplify different concentrations of URA3.
For this reaction we used a 2x, 5x, 10x, and 15x diluted samples of URA3 - ran each sample twice
2x: 1.5ul DNA, 1.5ul dH20
5x: 1ul DNA, 4ul dH20
10x: 1ul DNA, 9ul dH20
15x: 1ul DNA, 14ul dH20
positive control contained mastermix, LEU2 primers, and 20x diluted LEU2 DNA
1st negative control contained the mastermix and just LEU2 primers
2nd negative control negative contained the mastermix and just URA3 2x diluted DNA
</P><P>Gel Results:
</P><P>PCR Protocol for Plasmids Review
</P><UL><LI>2.5uL 10x buffer
</LI><LI>1.25 uL dNTP
</LI><LI>16.5 uL dH20
</LI><LI>0.25 uL Accutaq LA DNA polyermerase 
</LI></UL><UL><LI>1.5 uL forward primer 
</LI><LI>1.5 uL reverse primer 
</LI><LI>1.0 ul diluted DNA
</LI></UL><P>Transformation:
</P><UL><LI>We are still waiting for Leu2 dropout medium to come in.
</LI><LI>In the meantime, we have designed an experiment to confirm that we have the pRS425 plasmid. We will cut the plasmid with specific restriction enzymes and run the digest on a gel to measure the resulting lengths.
</LI><LI>Of the restriction enzymes we have, we find that BamHI, XbaI, PstI, and XhoI cut the plasmid once, and EcoRI cuts the plasmid twice. We will use EcoRI and compare the lengths of the two fragments. We will also cut with BamHI and run that in a separate lane to confirm the overall plasmid length.
</LI></UL><P>Assay Preparation:
</P><UL><LI>Prepared samples of beta-carotene stock solution in sterile water (8,16,32 microliter). We tried extracting beta-carotene from the water solution using hexane. Our results indicated very low yield from this extraction. This may be attributed to beta-carotene might have degraded due to prolonged exposure to light.
</LI></UL><H2><SPAN class="mw-headline" id="July_8"> July 8 </SPAN></H2><P>Transformation:
</P><UL><LI>We digested the DNA and then ran the results on a gel.
</LI><LI>Into a dilution microcentrifuge tube, we used 2 uL DNA (concentration 235.4 ng/uL) and 248 uL ddH20.
</LI><LI>BamHI tube: 2 uL diluted DNA, 2 uL NEBuffer, 2 uL diluted BSA (10x), 1 uL BamHI enzyme, 13 uL ddH20
</LI><LI>EcoRI tube: 2 uL diluted DNA, 2 ul SH buffer, 1 uL EcoRI enzyme, 15 uL reaction
</LI><LI>Incubated both tubes in a 37 C water bath for one hour for digestion
</LI><LI>Ran the solutions on a gel
</LI><LI>5 uL loading dye into three tubes: BamHI, EcoRI, control (uncut plasmid DNA)
</LI><LI>Lane 1: 5 uL ladder
</LI><LI>Lanes 2 and 3: 5 uL each BamHI
</LI><LI>Lanes 4 and 5: 5 uL each EcoRI
</LI><LI>Lane 6: 5 uL control
</LI><LI>Ran it for one hour at 132 volts
</LI></UL><P>Gel Team:
did a restriction digest of nat cassette with AgeI
</P><P>Assay Preparation:
</P><UL><LI>Prepared 3 5mL samples of lysed and saponified yeast cells from BC177 and BC178. 
</LI><LI>Yeast samples came from overnight cultures. 
</LI><LI>We followed extraction procedure from “High Level Production of Beta-Carotene in Saccharomyces cerevisiae…” Verwaal et. al. (2007). 
</LI><LI>This procedure used transformed yeast, we used untransformed BC177 and BC178 cells. 
</LI><LI>We did not add any beta-carotene. 
</LI></UL><LI>Our spectrophotometer analysis showed that the hexane layer contained minute amounts of lysed yeast cell product. These products did not interfere in the 453 nm range (where beta-carotene’s peak is).
</LI><H2><SPAN class="mw-headline" id="July_11"> July 11 </SPAN></H2><P>PCR group:
Reran five wells of PCR on URA3 - used 10x dilution URA3 DNA in our wells. 
Positive control contained 2ul of 20x diluted LEU2, LEU2 primers, and mastermix.
Negative control contained 1ul 10x dilution URA3 and mastermix --&gt; check for Plasmid DNA
</P><P>PCR Protocol for Plasmids Review
</P><UL><LI>2.5uL 10x buffer
</LI><LI>1.25 uL dNTP
</LI><LI>16.5 uL dH20
</LI><LI>0.25 uL Accutaq LA DNA polyermerase 
</LI><LI>20.5 ul/well
</LI></UL><UL><LI>1.5 uL forward primer 
</LI><LI>1.5 uL reverse primer 
</LI><LI>1.0 ul diluted DNA
</LI></UL><P>Gel Team:
the three empty wells on the right are our failed restriction digest
</P><P>Transformation:
</P><UL><LI>Set up yeast overnight for BC178
</LI><LI>Ran gel again with lower concentration enzyme; it did not work again
</LI></UL><P>Assay Preparation:
</P><UL><LI>Prepared 3 5mL samples of lysed and saponified yeast cells from BC177 and BC178.
</LI><LI>Yeast samples came from overnight cultures. We followed extraction procedure from “High Level Production of Beta-Carotene in Saccharomyces cerevisiae…” Verwaal et. al. (2007). 
</LI><LI>This procedure used transformed yeast, we instead added known concentrations of beta-carotene (8,16,32 microliter for each strain). 
</LI></UL><LI>Our results indicated that the beta-carotene was not evenly distributed in the hexane layer. 
</LI><LI>We modified the procedure so that we can get even distribution of beta-carotene in the hexane layer by pipetting the hexane layer up and down.
</LI><UL><LI>E. coli transformation of Leu2 and Ura3 and plated it with Leu2-deficient drop-out media we made
</LI></UL><H2><SPAN class="mw-headline" id="July_12"> July 12 </SPAN></H2><P>Transformation:
</P><UL><LI>Plates from yesterday
</LI><LI>Yeast transformation for BC178; we accidentally added 1 M LiAC instead of 100 uM, so we might have to redo it
</LI><LI>Set up E. coli overnight
</LI></UL><P>Gel Team:
</P><UL><LI>did a restriction digest on nat cassette using AgeI
</LI><LI>ran digest on gel and excised the two bands for later gel extraction
</LI></UL><P>Assay Preparation:
</P><UL><LI>Repeated procedure from 7/11/11 procedure with modifications. 
</LI><LI>Using hexane reagent as a blank. 
</LI></UL><LI>Much better results; absorbance almost doubled perfectly when beta-carotene concentrations were doubled.
</LI><H2><SPAN class="mw-headline" id="July_13"> July 13 </SPAN></H2><P>PCR:
</P><P>Colony PCR:
</P><P>Protocol:
</P><UL><LI>Transfer a yeast colony to a solution of 0.2% SDS. The SDS page was made by adding 0.02g SDS to 10mL of water.
</LI><LI>Vortex for 15 sec
</LI><LI>Heat in PCR machine for 4 min at 90 degrees Celsius.2
</LI><LI>Microcentrifuge for 1 min.
</LI><LI>Pipet out the supernatent and store at -20 degrees Celsius. 
</LI></UL><P>PCR mix for 50uL reaction:
</P><UL><LI>5uL 10x PCR buffer
</LI><LI>1.5 uL 50mM MgCl2 (we used 3uL because we 25mM MgCl2)
</LI><LI>1uL of 10mM dNTPs
</LI><LI>2uL of 25% TritonX-100
</LI><LI>0.3 uL Taq polymersase
</LI><LI>31.7 uL water
</LI></UL><UL><LI>Add 43 uL of PCR mix to each PCR tube
</LI><LI>Then add 3 uL of both the forward and reverse primers
</LI><LI>Add 1uL of the DNA from yeast
</LI></UL><P>
We made 2 samples with Leu2 primers, 2 samples with Uras3 primers, a positive control with Uras3 primers and DNA, and a negative control with just yeast DNA.
Strain 827 was used
</P><P>Assay Preparation:
</P><UL><LI>Experimented with different blanks
</LI><LI>Prepared a blank by adding only yeast cells, lysing them, and adding hexane. We took an aliquot of the hexane layer from the cell lysate tube and used it as a blank. 
</LI><LI>Absorbances for this procedure were much lower. 
</LI></UL><LI>Concluded that blanks have a small effect on the absorbance
</LI><LI>Performed this procedure on both strains of yeast
</LI><LI>However the blank was made from only one
</LI><LI>The tubes from which we got our two strains of yeast were grown to slightly different optical densities. This made one strain of yeast to have a higher concentration of yeast than the other. 
</LI><LI>This may explain why we got negative absorbances in the 550-750nm range
</LI><LI>Same procedure was tried without using pyrogallol in methanol and KOH. This might have also contributed to the low absorbances
</LI><LI>We inferred from the data that pyrogallol and KOH were necessary reagents in the procedure
</LI><H2><SPAN class="mw-headline" id="July_14"> July 14 </SPAN></H2><P>PCR:
</P><P>Colony PCR:
</P><P>Protocol:
</P><UL><LI>Transfer a yeast colony to a solution of 0.2% SDS. The SDS page was made by adding 0.02g SDS to 10mL of water.
</LI><LI>Vortex for 15 sec
</LI><LI>Heat in PCR machine for 4 min at 90 degrees Celsius.2
</LI><LI>Microcentrifuge for 1 min.
</LI><LI>Pipet out the supernatent and store at -20 degrees Celsius. 
</LI></UL><P>PCR mix for 50uL reaction:
</P><UL><LI>5uL 10x PCR buffer
</LI><LI>1.5 uL 50mM MgCl2 (we used 3uL because we 25mM MgCl2)
</LI><LI>1uL of 10mM dNTPs
</LI><LI>2uL of 25% TritonX-100. The 25% TritonX-100 was made by adding 6ul of TritonX-100 to 18ul of H20
</LI><LI>0.3 uL Taq polymersase
</LI><LI>31.7 uL water
</LI></UL><UL><LI>Add 43 uL of PCR mix to each PCR tube
</LI><LI>Then add 3 uL of both the forward and reverse primers
</LI><LI>Add 1uL of the DNA from yeast
</LI></UL><P>
We made 1 samples with Leu2 primers, 2 samples with Uras3 primers and a positive control with Uras3 primers and DNA
Strain 825 was used
</P><P>Gel Team:
</P><UL><LI>ran a gel of the colony PCR
</LI></UL><P>Gel Team:
</P><UL><LI>ran a gel of the colony PCR
</LI><LI><DIV class="thumb tnone"><DIV class="thumbinner" style="width:132px;"><A href="/File:Colony_PCR_7-13.gif" title="File:Colony PCR 7-13.gif">File:Colony PCR 7-13.gif</A></DIV></DIV></LI></UL><LI>did a gel extraction with grad student Bert Berla
</LI><LI>ran a ligation reaction overnight
</LI><P>Assay Preparation:
</P><UL><LI>Tested for the effect of methanol in the procedure
</LI><LI>Extracted beta-carotene using previous procedure using pyrogallol dissolved in methanol and in water
</LI><LI>Used KOH and used hexane as a blank
</LI></UL><LI>Concluded that methanol along with hexane helps extract the beta-carotene from the cell lysate products. In the samples with water instead of methanol, the spectra had much lower absorbances and had different shaped slopes in the 550-750nm range
</LI><H2><SPAN class="mw-headline" id="July_15"> July 15 </SPAN></H2><P>Ordered Biobrick primers and all 4 cassette primers without homology in order to run colony PCR
</P><P>Gel Team:
</P><UL><LI>ran a gel with PCR product from 7/13 and our ligation reaction from 7/14
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/14.jpg?attachauth=ANoY7cosx46T8ZSWGL2MZxuaA5kIny3T5og9EAmTXuyj7VV0bpDDrkyGJH2BLHeNXoY5ZBJ324GXSgelqV6Gnjg87LjFIqd3jtGjhbMvjGdukimu-Zvqq07S12zu8Mhz_e7XXlfmaNF13Ye9Kwrtq3Pi7hFUahe4khsHy7wnWRa-zF1jJ22JW5d9PMzYP1zUpm-OOqirtl2f&amp;attredirects=0" class="external autonumber" rel="nofollow">[3]</A>]
</LI></UL><H2><SPAN class="mw-headline" id="July_18"> July 18 </SPAN></H2><P>Ran a PCR for the genes: CrtI, CrtE, CcD1, CcYB --&gt; ran 3 PCR tubes for each gene
</P><UL><LI>1 positive control: Nat Genes + primers
</LI><LI>4 negative controls, 1 corresponding to each gene, contains only the mastermix soln and F/R primers 
</LI></UL><P>Recipe per sample for running plasmid DNA:
</P><UL><LI>2.5uL 10x buffer
</LI><LI>1.25 uL dNTP
</LI><LI>1.5 uL forward primer
</LI><LI>1.5 uL reverse primer
</LI><LI>16.5 uL dH20
</LI><LI>0.25 uL Accutaq LA DNA polyermerase
</LI></UL><P>
Gel Results:
</P><P>
Transformation:
</P><UL><LI>We had made colonies of BC178, but we had gotten contamination in them, so we made new plates, this time with ampicillin, in hopes of getting rid of any E. coli that may have been in our test tubes. Also, we filter-sterilized our lithium acetate (just in case) and our PEG 3350.
</LI><LI>We set up new overnights to start over.
</LI></UL><H2><SPAN class="mw-headline" id="July_20"> July 20 </SPAN></H2><P>PCR Team:
Ran a PCR for the cassettes: URA3, LEU2, KAN, NAT --&gt; ran 2 41ul PCR tubes for each gene
</P><UL><LI>4 25ul negative controls, 1 corresponding to each gene, contains only the mastermix soln and F/R primers
</LI></UL><P>Recipe per sample for running plasmid DNA (20.5ul reactions):
</P><UL><LI>2.5uL 10x buffer
</LI><LI>1.25 uL dNTP
</LI><LI>1.5 uL forward primer
</LI><LI>1.5 uL reverse primer
</LI><LI>16.5 uL dH20
</LI><LI>0.25 uL Accutaq LA DNA polyermerase 
</LI></UL><P>Gel Results:
</P><P>Designed primers to amplify Homology, Gene, Cassette, Homology
</P><UL><LI><B>crtYB-URa3</B></LI></UL><LI>Forward primer	GAAGAATATACTAAAAAATGAGCAGGCAAGATA	33	54.78
</LI><LI>Reverse primer	TATGAATGTCAGTAAGTATGTATACGAACAGTAT	34	54.58
</LI><UL><LI>BioBrick:
</LI></UL><LI>gaattcgcggccgcttctagag GAA GAA TAT ACT AAA AAA TGA GCA GGC AAG ATA
</LI><LI>ctgcagcggccgcractagta T ATG AAT GTC AGT AAG TAT GTA TAC GAA CAG TAT
</LI><UL><LI><B>crtI-kanMX4</B></LI></UL><LI>Forward primer	TTTTCCAATAGGTGGTTAGCAATCGTC	        27	55.97
</LI><LI>Reverse primer	AAATTCATAATAGAAACGACACGAAATTACAAA	33	54.14
</LI><UL><LI>Biobrick:
</LI></UL><LI>Gaattcgcggccgcttctagag TTT TCC AAT AGG TGG TTA GCA ATC GTC TTA 
</LI><LI>Ctgcagcggccgcractagta AAA TTC ATA ATA GAA ACG ACA CGA AAT TAC AAA A
</LI><UL><LI><B>CrtE-natMX4	</B></LI></UL><LI>Forward primer	TCTCCGAGCAGAAGGAAGAAC	                21	53.62
</LI><LI>Reverse primer	CTAAACTCACAAATTAGAGCTTCAATTTAATTATAT	36	52.82
</LI><UL><LI>Biobricks:
</LI></UL><LI>gaattcgcggccgcttctagag TCT CCG AGC AGA AGG AAG AAC
</LI><LI>ctgcagcggccgcractagta CTA AAC TCA CAA ATT AGA GCT TCA ATT TAA TTA TAT 
</LI><UL><LI><B>CCD1-leu2</B></LI></UL><LI>Forward primer	AGTTACTCACTAATGACTAACGAAAAGGTCTG	32	56.97
</LI><LI>Reverse primer	GGTTGAGCATTACGTATGATATGTCCATGT	        30	57.38
</LI><UL><LI>Biobricks:
</LI></UL><LI>gaattcgcggccgcttctagag AGTTACTCACTAATGACTAACGAAAAGGTCTG
</LI><LI>ctgcagcggccgcractagta  GGT TGA GCA TTA CGT ATG ATA TGT CCA TGT
</LI><P>Transformation:
</P><UL><LI>Made more YPD
</LI><LI>E. coli transformations using Nat and Kan
</LI><LI>Set up yeast overnights to try again. The contaminated plates must have yeast in them that could somehow grow in Leu-deficient medium. We are plating again to verify.
</LI></UL><P>Assay Preparation:
</P><UL><LI>Designed an experiment to specifically gauge the effects of different blanks. 
</LI><LI>Used pure hexane reagent, mixture of hexane and methanol, and hexane layer from cell lysate. 
</LI></UL><LI>Tested these different blanks on 3 samples with the same concentration of beta-carotene.
</LI><LI>In the middle of the procedure, the cell lysate blank was spilled. We could not finish the experiment.
</LI><H2><SPAN class="mw-headline" id="7.2F21.2F11">7/21/11</SPAN></H2><P>Assay Preparation:
</P><UL><LI>Re-did the experiment from yesterday on the blanks
</LI><LI>Using the results from this experiment we used the appropriate blank for another experiment to gauge our extraction efficiency
</LI><LI>Prepared four samples of each concentration of beta-carotene (2,4,8,16,32,64 µl of stock). 
</LI><LI>Determined out extraction efficiency
</LI></UL><P>Gel Team:
</P><UL><LI>ran a ligation reaction of nat fragments
</LI><LI>ran gel of ligation products: <A href="https://mail.google.com/mail/u/0/?ui=2&amp;ik=461b5e4071&amp;view=att&amp;th=1314e0f0d22ae774&amp;attid=0.1&amp;disp=inline&amp;realattid=f_gqe31z8l0&amp;zw" class="external text" rel="nofollow">ligation gel</A></LI></UL><H2><SPAN class="mw-headline" id="July_22">July 22</SPAN></H2><P>PCR team: Realized the reverse primer design from July 20 was wrong. The redesigned primers are as follows:
</P><UL><LI><B>crtYB-URa3</B></LI></UL><LI>Reverse primer	AGTATCATACTGTTCGTATACATACTTACTGACA	
</LI><UL><LI>BioBrick:
</LI></UL><LI>Reverse biobrick: ctgcagcggccgcractagta AGTATCATACTGTTCGTATACATACTTACTGACA
</LI><UL><LI><B>crtI-kanMX4</B></LI></UL><LI>Reverse primer	ATGAACATATTCCATTTTGTAATTTCGTGTC
</LI><UL><LI>Biobrick:
</LI></UL><LI>Reverse biobrick primer: ctgcagcggccgcractagta ATGAACATATTCCATTTTGTAATTTCGTGTC
</LI><UL><LI><B>CrtE-natMX4	</B></LI></UL><LI>Reverse primer	GGGTAATAACTGATATAATTAAATTGAAGCTCTAAT 
</LI><UL><LI>Biobrick:
</LI></UL><LI>ctgcagcggccgcractagta GGGTAATAACTGATATAATTAAATTGAAGCTCTAAT  
</LI><UL><LI><B>CCD1-leu2</B></LI></UL><LI>Forward primer	AGTTACTCACTAATGACTAACGAAAAGGTCTG
</LI><LI>Reverse primer	ATATTTAATTATTGTACATGGACATATCATACGTAA
</LI><UL><LI>Biobricks:
</LI></UL><LI>gaattcgcggccgcttctagag AGTTACTCACTAATGACTAACGAAAAGGTCTG
</LI><LI>ctgcagcggccgcractagta  ATATTTAATTATTGTACATGGACATATCATACGTAA
</LI><P>Assay Preparation:
</P><UL><LI>Redid calibration curve of beta-carotene in hexane
</LI><LI>Found that much of the beta-carotene has degraded
</LI><LI>Concluded that low efficiency in extraction is partly attributed to the deration
</LI></UL><H2><SPAN class="mw-headline" id="July_25">July 25</SPAN></H2><P>PCR:
</P><P>Nanodrop the minipreped Nat and Kan cassette
</P><UL><LI>Nat 1: 88.9 ng/uL
</LI><LI>Nat 2: 124.8 ng/uL
</LI><LI>Nat 3: 99.2 ng/uL
</LI><LI>Nat 4: 77.5 ng/uL
</LI><LI>Kan 1: 86.9 ng/uL
</LI><LI>Kan 2: 83.9 ng/uL
</LI><LI>Kan 3: 94.6 ng/uL
</LI><LI>Kan 4: 90.2 ng/uL
</LI></UL><P>Assay Preparation:
</P><UL><LI>Made a .1 mM beta-ionone stock solution in hexane through serial dilution
</LI><LI>Took spectrophotometric readings of samples of 1 ml of hexane with varying amounts (2, 4, 8, 16, 32, 64, 128, 256, 512 µl) of stock solution to make a calibration curve for beta-ionone in hexane
</LI></UL><H2><SPAN class="mw-headline" id="July_26">July 26</SPAN></H2><P>Gel Team:
</P><UL><LI>ran a ligation of nat fragments
</LI><LI>ran ligation product on gel: <A href="https://mail.google.com/mail/u/0/?ui=2&amp;ik=461b5e4071&amp;view=att&amp;th=131675217293ad05&amp;attid=0.1&amp;disp=inline&amp;realattid=f_gql3dz510&amp;zw" class="external text" rel="nofollow">ligation</A></LI><LI>although not clear from image, ligation was success
</LI></UL><H2><SPAN class="mw-headline" id="July_27">July 27</SPAN></H2><P>Transformation:
</P><UL><LI>We tried to determine the source of contamination on our plates. We plated our six transformation reagents on a YPD, Leu-dropout plate; plated ddH20, 100 mM LiAc, 1 M LiAc, DNA, PEG, YPD.
</LI><LI>Also, to prevent future contamination, we autoclaved some flasks and test tubes.
</LI></UL><P>Assay Preparation:
</P><UL><LI>Repeated yesterday’s procedure, except using a newly made bottle of beta-ionone stock, as beta-ionone readily degrades in light and air
</LI></UL><H2><SPAN class="mw-headline" id="July_28">July 28</SPAN></H2><P>Transformation:
</P><UL><LI>There was yeast growing on our 100 mM LiAc streak. We made a new stock by diluting 4 mL 1 M LiAc with 36 mL ddH20 and filter-sterilized it.
</LI><LI>Still waiting on salmon-sperm DNA.
</LI></UL><P>Assay Preparation:
</P><UL><LI>Tested the extraction efficiency beta ionone in hexane from yeast extract by spectrophotometer. 
</LI><LI>32µl of 0.1mM beta ionone solution was added to yeast cells suspended in 1ml of water and subsequently lysed with glass beads and extracted with 2ml of hexane
</LI><LI> Measured absorbance on spectrophotometer and used absorbance at λmax (280nm) along with extinction coefficient obtained from calibration curve made on 7/26/11
</LI></UL><H2><SPAN class="mw-headline" id="7.2F29.2F11">7/29/11</SPAN></H2><P>Assay Preparation:
</P><UL><LI>Tested the limit of detection for beta ionone in yeast using varying amounts of beta ionone stock solution (2, 4, 8 µl) and following procedure from 7/28
</LI><LI>Unfortunately, the spectrophotometer was unavailable. 
</LI><LI>Samples were parafilmed and put in -20C freezer for weekend storage
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:WashU/Notebook/July2011">http://2011.igem.org/Team:WashU/Notebook/July2011</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:WashU/Notebook/July2011" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:WashU/Notebook/July2011" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:WashU/Notebook/July2011&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:WashU/Notebook/July2011&amp;oldid=188598" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>