<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_WashU_Notebook_June"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:WashU/Notebook/June">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:WashU/Notebook/June&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:WashU/Notebook/June&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:WashU/Notebook/June&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:WashU/Notebook/June" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:WashU/Notebook/June</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="topmenu"><UL><LI><A href="https://2011.igem.org/Team:WashU">Home</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Team#Team_Members">Team Members</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Team#Attributions">Attributions</A></LI></UL><LI><A href="https://2011.igem.org/Team:WashU/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Project#Abstract">Abstract</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Project#Project_Description">Project Description</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Project#Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Modeling">Modeling</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Modeling#Experimental_Plan">Experimental Plan</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Parts">Parts</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Notebook">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Notebook/MolBio">Molecular Biology Group</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Notebook/PCR">PCR Group</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Notebook/Transformation">Transformation Group</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Notebook/Assay">Assay Group</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Notebook">By Date</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:WashU/Sponsors">Sponsors</A></LI></DIV><H3><SPAN class="mw-headline" id="June_1">June 1</SPAN></H3><UL><LI>Lesson from Bert Berla on how to design primers
</LI><LI>Codon Optimized the four genes using tool at: <A href="http://www.encorbio.com/protocols/Codon.htm" class="external free" rel="nofollow">http://www.encorbio.com/protocols/Codon.htm</A></LI><LI>Double checked gene and restriction sites to make sure enzyme does not cut in the middle of the gene
</LI></UL><LI>Design Primers for cassettes
</LI><LI>Ura3, Leu2, KanMX4, and NatMX4
</LI><H3><SPAN class="mw-headline" id="June_2">June 2</SPAN></H3><UL><LI>Made two liters of LB solution
</LI><LI>25g of LB Broth per Liter (10g Tryptone, 5g Yeast Extract, and 10g NaCl)
</LI><LI>Autoclaved solutions: 30 minute sterilization.
</LI><LI>Refrigerated after cooling.
</LI></UL><H3><SPAN class="mw-headline" id="June_3">June 3</SPAN></H3><UL><LI>Made YPD
</LI></UL><OL><LI>      In 800ml water in 1 L bottle dissolve 10g of BactoYeast extract 
</LI><LI>     Dissolve 20g of BactoPeptone in the above solution
</LI><LI>      In 200ml water in 500ml bottle dissolve 20g Dextrose 
</LI><LI>      Autoclave both, combine after autoclaving. (30 minute sterilization)
</LI></OL><UL><LI>Made 1000X Ampicillin Stock
</LI><LI>Added 2g Ampicillin to 20ml water.
</LI><LI>Added NaOH until dissolved.
</LI><LI>Sterile filtered.
</LI><LI>Frozen in -20C Freezer
</LI></UL><UL><LI>Made cultures of E. coli strains with KanMx4 and Natmx4 inserts.
</LI><LI>In culture tubes, added 5ml LB and 5ul 1000X Ampicillin Stock.
</LI><LI>Transferred one colony per strain from plates to culture tubes.
</LI><LI>Incubate at 30 degrees Celsius shaking at 225rpm overnight.
</LI></UL><UL><LI>Made cultures of Yeast strains BC177 and BC178
</LI><LI>Added 5ml YPD to two culture tubes.
</LI><LI>Transferred one colony per strain from plates to culture tubes.
</LI><LI>Incubate at 30 degrees Celsius shaking at 225rpm overnight.
</LI></UL><H3><SPAN class="mw-headline" id="June_4">June 4</SPAN></H3><UL><LI>Making freezer stocks of yeast cultures BC177 and BC178
</LI><LI>Add 1.25mL glycerol to yeast culture
</LI><LI>Freeze yeast culture in -80 degree freezer
</LI></UL><UL><LI>Make E.coli mini-preps for plasmids containing KanMX4 and NatMX4 (Sigma-Aldrich Mini-Prep Kit)
</LI><LI>Transferred 4 ml of each culture to microcentrifuge tubes.
</LI><LI>Centrifuge for one minute at 12000xg
</LI><LI>Pour out supernatant
</LI><LI>add 200uL of resuspension solution with RNase
</LI><LI>add 200uL of lysis buffer
</LI><LI>add 350uL of neutralization buffer
</LI><LI>Centrifuge for 10 minutes at 12000xg
</LI><LI>Insert a Miniprep Binding Column into microcentrifuge tubes and add 500uL of Column Preparation Solution to each miniprep column. 
</LI><LI>Centrifuge at 12000xg for 1 minute. Discard the flow-through
</LI><LI>Take the lysis of the cultures resulting from the 10 minute centrifuge and pipet it into the binding column
</LI><LI>Centrifuge at 12000xg for 1 minute. Discard the flow-through
</LI><LI>Add 500uL of Wash Solution 1 to the column and centrifuge at 12000xg for one minute. Discard the flow-through.
</LI><LI>Repeat the previous step
</LI><LI>Add 750uL of Wash Solution 2 (with ethanol) to the column and centrifuge at 12000xg for one minute. Discard the flow-through. 
</LI><LI>Centrifuge at 12000xg for one minute.
</LI><LI>Transfer the column to a fresh collection tube and add 100uL of Elution Solution to the column
</LI><LI>Centrifuge at 12000xg for one minute.
</LI><LI>Store the eluate at -20 degrees C
</LI></UL><UL><LI>This procedure was done separately for the plasmids that contain KanMX4 and NatMX4.
</LI></UL><H3><SPAN class="mw-headline" id="June_5">June 5</SPAN></H3><P>Read absorbances for all four solutions that contain plasmids with our cassettes: Ura3, Leu3, KanMX4, and NatMX4.
</P><P>Data was obtained using a nano-drop spectrophotometer. Absorbance was observed between 250nm and 280nm.
</P><P>DNA Concentrations were as follows:
</P><UL><LI>Ura3: Very high absorbance reading, must be diluted 1/10 in order to get accurate concentration reading.
</LI><LI>Leu2: Very high absorbance reading, must be diluted 1/10 in order to get accurate concentration reading.
</LI><LI>KanMX4: 30.5 ng/uL
</LI><LI>NatMX4: Very low absorbance reading, eluent was discarded.
</LI><LI>Last year's NatMX4 solution: Very high absorbance reading, must be diluted 1/10 in order to get accurate concentration reading.
</LI></UL><UL><LI>Made new culture of NatMX4
</LI><LI>5ml of TB + 5uL of 1000x Ampicillin
</LI><LI>Incubated overnight at 31 degrees Celsius.
</LI></UL><H3><SPAN class="mw-headline" id="June_6">June 6</SPAN></H3><UL><LI>LEU2 group
</LI><LI>Make dilution for nanodrop
</LI><LI>Make solution of 5uL LEU2, 45uL DI water
</LI><LI>Concentration: 42.0 ng/uL
</LI></UL><UL><LI>URA3 group
</LI><LI>Make dilution for nanodrop
</LI><LI>Make solution of 5uL URA3, 45uL DI water
</LI><LI>Concentration: 35.9 ng/uL
</LI></UL><UL><LI>KanMX4 group
</LI><LI>30.5 ng/uL (from previous day)
</LI></UL><UL><LI>NatMX4 group
</LI><LI>Performed mini-prep on previous day's culture
</LI><LI>200uL eluent with concentration of 28.7 ng/uL
</LI><LI>Made a 1/10 dilution of last year's NatMX4 stock. (DNA Concentration: 23.3 ng/ul)
</LI></UL><H3><SPAN class="mw-headline" id="June_7">June 7</SPAN></H3><UL><LI>Ordered PCR materials and electrophoresis materials from Sigma-Aldrich.
</LI><LI>Designed primers to add homology to left side of cassette and an AvrII site to right side of cassette.
</LI><LI>Primer designs are:
</LI></UL><P>Leu2
</P><UL><LI>Forward: TTT CCT AGG CCA AAC TGG AAC AAC ACT CAA CCC
</LI></UL><UL><LI>Reverse: ATA TTT AAT TAT TGT ACA TGG ACA TAT CAT ACG TAA TGC TCA ACC TAA TTT CGT GTC GTT TCT ATT ATG AAT TTC ATT TAT
</LI></UL><P>Ura3
</P><UL><LI>Forward: TTT CCT AGG ACC ACA GCT TTT CAA TTC AAT TCA TCA TTT
</LI></UL><UL><LI>Reverse: AGT ATC ATA CTG TTC GTA TAC ATA CTT ACT GAC ATT CAT AAC CGC ATA GGG TAA TAA CTG ATA TAA TTA AAT TG
</LI></UL><P>KanMX4
</P><UL><LI>Forward: TTT CCT AGG AGC TTG CCT CGT CCC CGC C 
</LI></UL><UL><LI>Reverse:ATG AAC ATA TTC CAT TTT GTA ATT TCG TGT CGT TTC TAT TAT GAA TTT TCG ACA CTG GAT GGC GGC GTT 
</LI></UL><P>NatMX4
</P><UL><LI>Forward: TTT CCT AGG AGC TTG CCT CGT CCC CGC C
</LI></UL><UL><LI>Reverse: GGG TAA TAA CTG ATA TAA TTA AAT TGA AGC TCT AAT TTG TGA GTT TAG TCG ACA CTG GAT GGC GGC GTT
</LI></UL><H3><SPAN class="mw-headline" id="June_8">June 8</SPAN></H3><UL><LI>Split into four technique groups
</LI></UL><UL><LI>PCR group
</LI><LI>Practice PCR
</LI><LI>Mastermix:
</LI><LI>10x buffer:10uL
</LI><LI>betaine: 25 uL
</LI><LI>dNTP: 2.0uL
</LI><LI>Primer 1: 2.5 uL
</LI><LI>Primer 2: 2.5 uL
</LI><LI>Taq polymerase: 5 uL
</LI><LI>dH20: 28.0 uL
</LI></UL><LI>Sample 1:
</LI><LI>3uL MgCl2
</LI><LI>1uL DNA template
</LI><LI>16 uL master mix
</LI><LI>Sample 2:
</LI><LI>4uL MgCl2
</LI><LI>1uL DNA template
</LI><LI>15 uL master mix
</LI><LI>Sample 3:
</LI><LI>5uL MgCl2
</LI><LI>1uL DNA template
</LI><LI>14 uL master mix
</LI><LI>Sample 4 (Negative Control):
</LI><LI>4uL MgCl2
</LI><LI>15 uL master mix
</LI><P>94 degrees: 4min
</P><P>34 Cycles:
</P><UL><LI>94 degrees: 30 sec
</LI><LI>58 degrees: 30 sec
</LI><LI>72 degrees: 2 min (2kb product)
</LI></UL><P>72 degrees: 5min
</P><P>4 degrees: hold
</P><P>Gel Team:
</P><UL><LI>We made a gel to use tomorrow:
</LI><LI>1. Added 500 mL 1xTAE and 5g Agarose
</LI><LI>2. Microwaved until dissolved completely
</LI><LI>3. Allowed to cool until able to handle and added 9 mL ethidium bromide
</LI><LI>4. Poured into gel mold and allowed to cool overnight
</LI></UL><H3><SPAN class="mw-headline" id="June_9">June 9</SPAN></H3><P>Gel Team:
</P><UL><LI>We ran the DNA the PCR  team amplified yesterday
</LI><LI>filled the electrophoresis chamber with 1xTAE
</LI><LI>Added 2 mL loading buffer to each sample
</LI><LI>put ladder and 4 samples into gel
</LI><LI>ran electrophoresis for ~1h
</LI></UL><LI>We imaged the gel
</LI><LI>used UV imaging machine to take picture below
</LI><UL><LI>The desired band was not visible. We determined the problem was we were testing the wrong template DNA
</LI><LI>We made a new batch of gels using 400mL of water and 3.5g of agarose
</LI></UL><P>PCR team:
</P><UL><LI>We repeated PCR with new template DNA and freshly ordered supplies
</LI></UL><P>Recipe per sample for running plasmid DNA:
</P><UL><LI>2.5uL 10x buffer
</LI><LI>1.25 uL dNTP
</LI><LI>1.5 uL forward primer 44
</LI><LI>1.5 uL reverse primer 43
</LI><LI>16.5 uL dH20
</LI><LI>0.25 uL Accutaq LA DNA polyermerase
</LI></UL><P>Mastermix:
</P><UL><LI>12.5uL 10x buffer
</LI><LI>6.25 uL dNTP
</LI><LI>7.5 uL forward primer 44
</LI><LI>7.5 uL reverse primer 43
</LI><LI>82.5 uL dH20
</LI><LI>1.25 uL LA polyermerase
</LI></UL><P>Sample 1, 2, 3:
</P><UL><LI>1.5 uL DNA of plasmid pTCP2031v
</LI><LI>23.5 uL Mastermix
</LI></UL><P>Sample 4: 
</P><UL><LI>23.5 uL plasmid Mastermix
</LI></UL><P>Recipe per sample for running genomic DNA:
</P><UL><LI>2.5uL 10x buffer
</LI><LI>1.25 uL dNTP
</LI><LI>0.5 uL DMSO
</LI><LI>0.5 uL forward primer 42
</LI><LI>0.5 uL reverse primer 43
</LI><LI>17.0 uL dH20
</LI><LI>0.25 uL Accutaq LA DNA polyermerase
</LI></UL><P>Mastermix:
</P><UL><LI>12.5uL 10x buffer
</LI><LI>6.25 uL dNTP
</LI><LI>2.5 uL DMSO
</LI><LI>2.5 uL forward primer 42
</LI><LI>2.5 uL reverse primer 43
</LI><LI>85.0 uL dH20
</LI><LI>1.25 uL Accutaq LA DNA polyermerase
</LI></UL><P>Sample 5, 6, 7:
</P><UL><LI>1.5 uL vector DNA
</LI><LI>23.5 uL Mastermix
</LI></UL><P>Sample 8:
23.5 uL genomic Mastermix 
</P><P>
Run same temperatures as before with 34 cycles
</P><H3><SPAN class="mw-headline" id="June_10">June 10</SPAN></H3><UL><LI>Ran PCR products on a 1% agarose gel.
</LI><LI>Used UV imaging machine to produce picture below
</LI></UL><P>All six expected bands were visible.
</P><P>Performed gel extraction on all six bands using the Sigma Kit
</P><UL><LI>excised the bands from the gel.  The genomic band was too big so we cut it in half
</LI><LI>vector:.37g
</LI><LI>genomic1: .20g
</LI><LI>genomic2: .21g
</LI></UL><LI>Placed gels in separate collection tubes and added 3 gel volumes of Gel Solubilization Soln.  Allowed gels to dissolve
</LI><LI>Prepared 3 binding columns by adding 500uL of Column prep soln and centrifuging for 1 min.  Discarded eluate
</LI><LI>Added 1 gel volume isopropanol to each collection tube and mixed thoroughly.
</LI><LI>Loaded the gel solution into the binding columns in 700 uL portions.  centrifuged for 1 min and discarded eluate
</LI><LI>Added 700uL of Wash Soln to each column, centrifuged 1 min, and discarded eluate
</LI><LI>Centrifuged columns again for 1 min and discarded eluate
</LI><LI>Transfer each binding column to a fresh collection tube.  Added 50uL of Elution Solution and incubate 1 min
</LI><LI>centrifuge 1 min.  The elution contains the DNA
</LI><H3><SPAN class="mw-headline" id="June_13">June 13</SPAN></H3><UL><LI>Performed PCR product clean-up using a Sigma Kit.
</LI></UL><P>gel team:
</P><UL><LI>we did PCR cleanup
</LI><LI>Added .5 mL of the column prep soln to 2 miniprep column and centrifuged at 12000g for 1 min.  Discard eluate.
</LI><LI>Added 65 uL of Binding soln and 15 uL of each PCR reaction to separate columns and centrifuged for 1 min.  Discard eluate
</LI><LI>Add .5ml of Wash soln to each column and centrifuge 1 min.  Discard eluate.
</LI><LI>Centrifuge additional 2 min.  Discard eluate and place binding column into new collection tube
</LI><LI>Add 50uL elution soln and allow to incubate 1 min
</LI><LI>centrifuge 1 min.  The PCR product is in the eluate
</LI></UL><P>Nano-dropped four samples:
</P><UL><LI>Gel extraction vector DNA
</LI><LI>No DNA present; one large peak at 230nm signifying the presence of protein.
</LI></UL><LI>Gel extraction genomic DNA
</LI><LI>No DNA present; one large peak at 230nm signifying the presence of protein.
</LI><LI>PCR product clean-up vector DNA
</LI><LI>50uL of solution at 36.9ng/uL
</LI><LI>PCR product clean-up genomic DNA
</LI><LI>50uL of solution at 18.4ng/ul
</LI><H3><SPAN class="mw-headline" id="June_14">June 14</SPAN></H3><P>Transformation:
</P><UL><LI>Made plates with LB agar
</LI><LI>Added ~300 mL H2O to graduated cylinder
</LI><LI>Added 12.5 g LB broth, 7.5 g Bacto Agar, and aforementioned water to Erlenmeyer flask
</LI><LI>Stirred on hot plate to boiling
</LI><LI>Added water until total volume was 500 mL
</LI><LI>Autoclaved
</LI><LI>Returned to stir plate for 15 minutes to cool
</LI><LI>Added 0.05 g ampicillin (for antibiotic resistance marker) to flask
</LI><LI>Poured into 20 plates
</LI><LI>3 plates discarded because the agar had solidified by the time they were poured
</LI></UL><P>Gel:
</P><UL><LI>Did a restriction digest
</LI><LI>5 uL 10X Buffer (#2 NEB)
</LI><LI>30 uL DNA from PCR cleanup of vector DNA
</LI><LI>15 uL water
</LI><LI>1 uL 1 ku EcoRI
</LI><LI>incubated at 37 C overnight
</LI></UL><H3><SPAN class="mw-headline" id="June_15">June 15</SPAN></H3><P>Assay Preparation:
</P><UL><LI>Make 500ml YPD 
</LI><LI>Autoclaved YPD and poured into 16 petri dishes
</LI></UL><H3><SPAN class="mw-headline" id="June_16">June 16</SPAN></H3><P>Biobrick Transformation in E. coli:
</P><UL><LI>GC5 E. coli cells and SOC medium arrived
</LI><LI>Began transformation of E. coli with Leu2 and Ura3 biobricks
</LI><LI>Thawed 50 uL cells in ice bath
</LI><LI>Added 10 uL sterile ddH2O into biobrick wells 11A and 11C in plate 3
</LI><LI>Waited 5 minutes for DNA to resuspend
</LI><LI>Transferred the 10 uL to microcentrifuge tubes
</LI><LI>Added 20 uL cells to Ura3 tube, 20 uL cells to Leu2 tube, 10 uL cells to control tube
</LI><LI>Added 2 uL biobrick DNA to respective tubes
</LI><LI>Waited 30 minutes for cells to transform on ice
</LI><LI>Heat shocked in water bath at 37 C for 45 seconds
</LI><LI>Placed cells on ice for 2 minutes
</LI><LI>Added 200 uL SOC medium into each tube (100 uL into control tube)
</LI><LI>Put tubes in shaker at 37 C at 220 rpm for one hour
</LI><LI>Spread onto plates with LB agar (made June 14)
</LI><LI>Using a total of 6 plates, spread 50 uL onto one plate and the rest (172 uL) onto the other, for each of the three tubes
</LI></UL><P>Gel:
</P><UL><LI>Ran digestion on a gel
</LI><LI><A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/practicedigestion1.jpg?attachauth=ANoY7cquV-GJKXb5F7NuriILJFJCR9cRbp9rQ2TkQFZXLnd2VJpDnlsjL3N6WEeZJl433RAWTTEW0khd7UlTjny0fmMu8TFrzSVEAwrhhkAv8WnImW4WLCYejOwCyMXk_zwUl_WTiPslc6FXmQUZUcf4NB9hBwoGUQ2iwtrIHBSViUPt_toji1wtPeJtNi0Ecz58crXpK0q_WaXodGxiyd3PIlU9tS-nuQ%3D%3D&amp;attredirects=0" class="external autonumber" rel="nofollow">[1]</A></LI></UL><LI>Ran a new digestion
</LI><LI>30 uL DNA from PCR cleanup of genomic DNA
</LI><LI>incubated at 37 C overnight
</LI><P>Assay Preparation:
</P><UL><LI>Streak yeast strains, BC177 and BC178, each onto a separate petri dish with YPD
</LI></UL><H3><SPAN class="mw-headline" id="June_17">June 17</SPAN></H3><P>Tranformation:
</P><UL><LI>Due to the presence of ampicillin on the plates, we expected no colonies on the control plates, but colonies should grow on the Leu2 and Ura3 plates because they should have antibiotic resistance against ampicillin, if the transformation succeeded. Colonies grew on all plates, even the negative controls; we assume it must have been caused by deficient or faulty amipicillin. Because of this, we cannot determine whether the transformation proceeded correctly or not.
</LI></UL><UL><LI>We are performing a test of our amicillin stock as well as our new ampicillin stock.
</LI><LI>Three culture tubes each with 4 mL of LB. One tube is a positive control (No AMP), one has 4 uL of our old AMP stock, and one has 4 uL of our new AMP stock.
</LI><LI>Cells that do not contain a plasmid are placed in each culture tube.
</LI><LI>Began shaking at 240rpm at 37 degrees Celsius at 10:30 am.
</LI><LI>At 2:30 pm (4 hour incubation) the OD of the positive control was around .250 and both AMP cultures were still around zero.
</LI><LI>CONCLUSION: Ampicillin stocks are viable. Perhaps the reason for the lawns on the negative control transformation plates were due to Ampicillin being added while the media was still too hot.
</LI></UL><UL><LI>We will be making new plates with new ampicillin, following the same procedures as described before:
</LI><LI>Made plates with LB agar
</LI><LI>Added ~300 mL H2O to graduated cylinder
</LI><LI>Added 12.5 g LB broth, 7.5 g Bacto Agar, and aforementioned water to Erlenmeyer flask
</LI><LI>Stirred on hot plate to boiling
</LI><LI>Added water until total volume was 500 mL
</LI><LI>Autoclaved
</LI><LI>Waited for the solution to cool
</LI><LI>Added 500 uL ampicillin to flask
</LI><LI>Poured into 12 plates
</LI></UL><P>Gel
</P><UL><LI>ran practice digest 2 on a gel
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/practicedigest2.jpg?attachauth=ANoY7cq1tu_pKp53zds_PBcljnP8nRAkic24sQrPxYUaLn77uff_bO6IRiJyGlt0KMHgqbbN8KpeO2G8JAMIHI7vXzVR4ocZEb5Jp9BdueRhMkEahq9vZ3-NoR_HtcajHu_mYJxatvHhCsG5rMQY1CRn8a50rj5eAesbJh_EpN_z2caEdpiBusoxTmxxwmn5mPnzPs1sinqBclA15StaF1wdLgwrCq3e8A%3D%3D&amp;attredirects=0" class="external autonumber" rel="nofollow">[2]</A>]
</LI></UL><P><B>PCR Team</B></P><UL><LI>We ran PCR on genomic DNA using two different protocols (Sigma Aldrich's &amp; Bert's) to create more product for the gel team to practice on. 
</LI></UL><P>Total samples: 8 
</P><P><B>Sigma's Protocol:</B></P><P>Recipe per sample for running genomic DNA:
</P><UL><LI>2.5uL 10x buffer
</LI><LI>1.25 uL dNTP
</LI><LI>0.5 uL DMSO
</LI><LI>0.5 uL forward primer 42
</LI><LI>0.5 uL reverse primer 43
</LI><LI>17.0 uL dH20
</LI><LI>0.25 uL Accutaq LA DNA polyermerase
</LI></UL><P>Mastermix:
</P><UL><LI>12.5uL 10x buffer
</LI><LI>6.25 uL dNTP
</LI><LI>2.5 uL DMSO
</LI><LI>2.5 uL forward primer 42
</LI><LI>2.5 uL reverse primer 43
</LI><LI>85.0 uL dH20
</LI><LI>1.25 uL Accutaq LA DNA polymerase
</LI></UL><P><B>Sample 1, 2, 3 (not marked):</B></P><UL><LI>1.5 uL genomic DNA
</LI><LI>23.5 uL Mastermix
</LI></UL><P><B>Sample 4(negative sign in circle):</B>
23.5 uL Mastermix</P><P><B>Bert's Protocol (w/adjustments by May):</B></P><P>Recipe per sample for running genomic DNA:
</P><UL><LI>2 uL 10x Buffer
</LI><LI>0.4 uL dNTP
</LI><LI>0.5 uL forward primer 42
</LI><LI>0.5 uL reverse primer 43
</LI><LI>10.35 uL dH2O
</LI><LI>0.25 uL Accutaq LA DNA polymerase
</LI><LI>5 uL betaine
</LI></UL><P>Mastermix:
</P><UL><LI>10 uL 10x Buffer
</LI><LI>2 uL dNTP
</LI><LI>2.5 uL forward primer 42
</LI><LI>2.5 uL reverse primer 43
</LI><LI>51.75 uL dH2O
</LI><LI>1.25 uL Accutaq LA DNA polymerase
</LI><LI>25 uL betaine
</LI></UL><P><B>Sample 5,6,7 (marked with a dot):</B></P><UL><LI>1 uL genomic DNA
</LI><LI>19 uL Mastermix
</LI></UL><P><B>Sample 8(negative sign in circle):</B>
19 uL genomic Mastermix</P><P>Anneal at 58 degrees with 34 cycles
</P><P>Assay Preparation:
</P><UL><LI>Check growth of yeast strains on petri dishes streaked yesterday.
</LI><LI>Concluded colonies on plates were too close together
</LI><LI>Re-streaked two new petri dishes of BC177 and BC178 yeast strains using colonies from yesterdayâs petri dishes.
</LI></UL><H3><SPAN class="mw-headline" id="June_18">June 18</SPAN></H3><UL><LI>Check growth of yeast strains on petri dishes
</LI><LI>Concluded colonies on plates were too close together for plate with strain BC177.
</LI><LI>Re-streaked one new petri dish for BC177
</LI></UL><H3><SPAN class="mw-headline" id="June_20">June 20</SPAN></H3><UL><LI>Check growth of yeast strains on petri dishes
</LI><LI>Concluded colonies on our new stock plates were isolated enough for streaking
</LI></UL><LI>Isolated one colony from BC177 and BC178 yeast strains and cultured them in test tubes with 5 mL of YPD solution
</LI><P>Tranformation again:
</P><UL><LI>Thawed 50 uL cells in ice bath
</LI><LI>Added 20 uL cells to Ura3 tube, 20 uL cells to Leu2 tube, 10 uL cells to control tube
</LI><LI>Added 2 uL biobrick DNA to respective tubes
</LI><LI>Waited 30 minutes for cells to transform on ice
</LI><LI>Heat shocked in water bath at 37 C for 45 seconds
</LI><LI>Placed cells on ice for 2 minutes
</LI><LI>Added 200 uL SOC medium into each tube (100 uL into control tube)
</LI><LI>Put tubes in shaker at 37 C at 220 rpm for one hour
</LI><LI>Spread onto plates with LB agar (made June 17)
</LI><LI>Using a total of 6 plates, spread 50 uL onto one plate and the rest (172 uL) onto the other, for each of the three tubes
</LI></UL><P>Gel Team:
</P><UL><LI>Did PCR cleanup on PCR product from 6/17
</LI><LI>ran a gel with PCR product from 6/17
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/practicedigest3.jpg?attachauth=ANoY7crYKGk455-j0ajqKMfhEZfXMMjQpKg_q1kGq_ZpJ9q5mPmT9rXcQ0h2koQ1zuGMR4cJuwfB03rGJgWRbVkQnOw6SJ6h8nxcEfF72TPEo3MaNLVgFkinLZJGFFdKChUZoSLLZehw2iEC5gi3Ls8R5k_vHsnNuSAFIa53rnIJ1Ohm-eYuWaY-T-JA0l9Ljdal0blVbEr8kZZwAR_x4k8fvXfJTsQkzg%3D%3D&amp;attredirects=0" class="external autonumber" rel="nofollow">[3]</A>]
</LI></UL><LI>ran a digestion of PCR product from 6/17 with .3 uL EcoRI
</LI><LI>ran a gel of the digestion
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/2011.jpg?attachauth=ANoY7cr13xIbSU1cWXo9W2txQEk02rOKOh2pxQdMbdQ5x69yNmPsNBsZ9BExtAIbR4Tc9D7eLvcOw1eIbYoULs7vE5izCZcV-lRvRtbvsN26x_s0EXuL4Yv4DvBGtyHj2hNfxLatpn1U_CQEiCa_mKC7VwcFK5ol3NsJXsqRdwj-0q-EeN1SA6DEx2sIcKBs205o6YE30Stv2lpjtLVbf_VhCTN0IbZYdQ%3D%3D&amp;attredirects=0" class="external autonumber" rel="nofollow">[4]</A>]
</LI><LI>Did gel extraction of PCR product from 6/17
</LI><P>
PCR Team:
</P><P>Ran PCR on the four cassettes (Leu2, Ura3, KanMX4, and NatMX4) with the newly arrived primers. This is amplify the cassettes, add homology to the insertion site, and add a restriction site.
Sigma's Protocol from June 17 was used.
Leu2 and Ura3 were annealed at 50 degrees C while NatMX4 and KanMx4 were annealed at 55 degrees C.
</P><H3><SPAN class="mw-headline" id="June_21">June 21</SPAN></H3><P>Transformation:
</P><UL><LI>The transformation procedure performed yesterday worked. The Leu2 and the Ura3 plates all had some colonies on them, while the control plates had none.
</LI><LI>Today, we made cultures of the Leu2 and Ura3 colonies.
</LI><LI>Added 5 mL LB and 5 uL ampicillin to two tubes each.
</LI><LI>Using sterile technique, picked off a colony from one plate and added it to one tube. Did same for other.
</LI><LI>Incubated overnight on shaker at 30 C.
</LI></UL><P>PCR:
We used the Sigma protocol with an annealing temperature of 58 degrees C on the sample plasmid DNA provided to us by Burt. This was in order for the gel team to practice ligation after cutting with an restriction site.
</P><P>Gel:
</P><UL><LI>ran gels of PCR product from 6/20
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/CassettePCR1.jpg?attachauth=ANoY7cqqbXYOGRtlg8GeEqvmrhAiivnxjdEhc9Z_4F_-_yqZXC77asW7TPjmUBo8UW8BV0XGOUhzf_EX7XdxYDju49DPOUSwwYwnmnCNsNYr7cfrPk7KOjIrBMEoLrNtBB43ls_ay_4TRqjfaJ8MoZXf7_pJl2HC0B31vrgHiSZw8KA9IvDqbvbycTCcuTweaMZeC5gILg2IQHY0p66Ip5Shnp05ZaxykA%3D%3D&amp;attredirects=0" class="external autonumber" rel="nofollow">[5]</A>]
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/CassettePCR2.jpg?attachauth=ANoY7co0WgCs4VNcjqmGdqK6I-LYdOzVk58QCsgl3cEVBcbLVJnGMVOZP4YWVH6C1UlTCImop3TPmmkeP7uuxSX3LgKrGCidtIe7spODQvKkjHDy-2cg0Swca_F9O4zGMzaJ-sPyL3WrmaLWeqCtdKPo_ZVUY30hugKNObOug6qeAhRn1rZGsJSfri4yHVbTxAEMQSyeWfMp4XILLbLNOFvMP5XzlLmo7A%3D%3D&amp;attredirects=0" class="external autonumber" rel="nofollow">[6]</A>]
</LI></UL><LI>made new gels
</LI><LI>350 mL TAE and 3 g agarose
</LI><LI>ran gels of PCR product from 6/21
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/practicegel4.jpg?attachauth=ANoY7cqWrQR283tB0AfD093BqtqdYD1wl1Y5BK1IGGhIGm_cST9h-5_TU2zuCjX9q9MUIBaSFkZmdrtTzGvbQlWYVlbv5dfiVHs2z-wcUi5koAvGLZw9TBtEsxEoKlopsYIE98Mv81nZSpB0WdrjLa7LkG8jLDJ59dMvyD9EQBSKtAtcF3pvGH8BvUMy7GXvIgZew32deAn9MO5iwhLAcHiu2w8NKPGA0Q%3D%3D&amp;attredirects=0" class="external autonumber" rel="nofollow">[7]</A>]
</LI><H3><SPAN class="mw-headline" id="June_22">June 22</SPAN></H3><P>Transformation:
</P><UL><LI>Today, we used a plasmid miniprep kit to extract the plasmids from the cultured cells
</LI><LI>For each tube, we extracted 4000 uL (4 mL) cell solution and added 1000 uL each into an individual microcentrifuge tube (for a total of 8 tubes)
</LI><LI>Centrifuged at 12,000 g for 1 minute
</LI><LI>Discarded supernatant
</LI><LI>Added 50 uL resuspension solution containing RNAse A into each of the eight tubes
</LI><LI>Vortexed to resuspend
</LI><LI>Combined the eight tubes into two new microcentrifuge tubes (for Leu2 and Ura3)
</LI><LI>Added 200 uL lysis buffer into both tubes and inverted to mix
</LI><LI>Added 350 uL neutralization/binding buffer and inverted to mix
</LI><LI>Centrifuged at 12,000 g for 10 minutes
</LI><LI>Inserted a binding column into a microcentrifuge tube and added 500 uL to the column
</LI><LI>Centrifuged new tubes 12,000 g for 1 minute; discarded flow-through
</LI><LI>Transferred supernatant (lysate) into columns
</LI><LI>Centrifuged 1 minute and discarded flow-through
</LI><LI>Added 500 uL wash solution to column; centrifuged again and discarded flow-through
</LI><LI>Added 750 uL second wash solution to column; centrifuged again and discarded flow-through
</LI><LI>Centrifuged again and discarded flow-through to remove ethanol
</LI><LI>Transferred to new collection tube
</LI><LI>Added 100 uL elution solution and centrifuged again
</LI></UL><LI>We measured the DNA concentration of the resulting eluate using a Nanodrop.
</LI><LI>Leu2: 329.8 ng/uL
</LI><LI>Ura3: 84.1 ng/uL
</LI><P>PCR:
</P><P>We repeated the PCR done on June 20th on the cassettes. The only difference is we added a sample with just plasmid DNA and no primers in order to determine if the top band on the gel resulting from the June 20th PCR was the plasmid DNA.
</P><H3><SPAN class="mw-headline" id="June_23">June 23</SPAN></H3><P>Transformation:
</P><UL><LI>The Leu2 concentration was very high. To be sure, we measured the concentrations of both again.
</LI><LI>Leu2: 323.9 ng/uL
</LI><LI>260/280: 1.89
</LI><LI>260/230: 2.33
</LI></UL><LI>Ura3: 83.4 ng/uL
</LI><LI>260/280: 1.99
</LI><LI>260/230: 2.23
</LI><LI>We also began practicing transformation of yeast using two strains that Kim gave us and the pRS425 plasmid. To transform the yeast, we would have to extract the plasmid from the E. coli colonies we were given using a miniprep kit. We cultured the three plates.
</LI><LI>For each plate, we chose one colony and added it to a glass tube of solution. For the two yeast strains, we used 5 mL YPD and 5 uL ampicillin. For the E. coli, the tube contained 5 mL LB only.
</LI><LI>The three glass tubes went into a 30 C shaker overnight.
</LI><P>Assay:
</P><UL><LI>Practiced using Olis spectrophotometer and computer program for future experiments involve spectrophotometrically assaying for the intermediates and products of the beta carotene pathway.
</LI><LI>Repaired stock beta carotene solution and 1/128 stock beta carotene solution, both in acetone.
</LI><LI>Took preliminary spectra of beta carotene solutions, starting from the stock solution, successively diluting by twofold each time.
</LI></UL><P>Gel:ran a gel of the PCR product from 6/22
</P><UL><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/CassettePCR3.jpg?attachauth=ANoY7crp70_aAQ7OtlyvaNMULlloD_yIVAch1nzniadBTYFIbCUrBnisRRQTe44fP7IKG1LU1meNeqXfDaXwLIezXfR43jcqVUWJgRkbDree1EffQHWUC6w5Ki17fzorRELPsmoij4m69lKGlfK21Bau-I1pISYjLxEnhyAKkc9ra8AyBkSCxRUB7HQ7jjhPTpu114IcA1e0hKfBEMyQUOt6fbNsilzWgA%3D%3D&amp;attredirects=0" class="external autonumber" rel="nofollow">[8]</A>]
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/CassettePCR4.jpg?attachauth=ANoY7crzcRw7kBeAYUgMKwi8NEt82e__oOwYzE0YXv47-cab02BITRwnFzjXN_4L1_c2sAXEJ2Voshw676knCd_YzG4LAWiWeOuT2l8ac59W12xxQQvwoktjMc7Wp6z4yoZeLu7OezHsB9BYW4BYmsScaOzLhnYocCJEQidX5ypfM_6i7pc6DEFDaci1_Vdma4gzYgi8lixjH228HmFjxeUSBxWl_0lgig%3D%3D&amp;attredirects=0" class="external autonumber" rel="nofollow">[9]</A>]
</LI></UL><H3><SPAN class="mw-headline" id="June_24">June 24</SPAN></H3><P>Transformation:
</P><UL><LI>Due to a misunderstanding, our yeast cultures were accidentally dumped, so we could only work with the E. coli with pRS425 plasmid today.
</LI><LI>Using the miniprep procedures previously described, we miniprepped one tube of the plasmid.
</LI></UL><LI>Nanodrop results:
</LI><LI>Concentration: 235.4 ng/uL
</LI><LI>260/280: 1.88
</LI><LI>260/230: 2.30
</LI><P>
PCR:
Repeated the PCR from June 22 because we realized the concentration of DNA was too high. For this sample 10 ng of DNA were loaded for Leu2, Ura3, Kan, and Nat.
</P><P>Assay:
</P><UL><LI>Experimentally obtained the absorption spectrum of yeast solution in YPD and beta carotene dissolved in acetone at varying concentrations. Started with solutions of each and successively diluted solutions twofold until absorption peaks are no longer characteristic of the solution.
</LI></UL><P>Gel:
</P><UL><LI>ran a gel of the PCR product from 6/23
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/24.jpg?attachauth=ANoY7cohseT3S-aYFaDVpS5hUHRGltQ6Fdn-KH-Hb8alXnzmUg405ZVKkmXRO1uqTlZyZZLXKJdiURYt1NNPS6wGptFUZZxCzZzXQfDB1TrSbwtaE1Eckl3iz7Ppbt1Z_KuNEsFFVWOlRKYjdppYiil8RM1w5NxduMYXxVmS4DEXf5EQWChVZhYgMFq56Qrc03rJVC_mSBSZ&amp;attredirects=0" class="external autonumber" rel="nofollow">[10]</A>]
</LI></UL><LI>Did a restriction digest on kan and nat cassettes
</LI><LI>ran a gel with leu2 and ura3 plasmids transformed from biobricks and restriction digest product
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/practicedigest4.jpg?attachauth=ANoY7cqwfxQeYEoqWhtK7qaNFDFR4_JgJfSh550ue7-9ORZQmIIx5nFidsNtfZKBADFbBvM6Aw-Az59U6K9D_g3b9QM4ij3PACk0YA7Y546fksQG2-BazbsRQplvF9ctieLZ_cFglW7PXie4dZT5UL0FGHFf8-LRdjEL_uPlUE_a5WYo7E8EbJTXF_ilWl0pRgH-Q82JXQ9iq8UsJIWDjReWPCTmpBfMdQ%3D%3D&amp;attredirects=0" class="external autonumber" rel="nofollow">[11]</A>]
</LI><LI>picked up gel extraction kits from Cohen lab
</LI><LI>Used spin column gel extraction kit to extract restriction digest products from gel
</LI><H3><SPAN class="mw-headline" id="June_27">June 27</SPAN></H3><P>Transformation:
</P><UL><LI>Made YPD plates with ampicillin
</LI><LI>1.65 g Bacto Agar into 200 mL YPD; autoclaved
</LI><LI>250 uL ampicillin
</LI><LI>Poured into plates (had to throw out a few and ended up with 4 plates)
</LI></UL><LI>We could not perform the transformation procedure today because we were required to leave the lab early.
</LI><LI>Put the cells into 50 mL YPD (from cultures overnights made yesterday)
</LI><LI>Added 2 mL cell solution into 5 mL more YPD and put in the 30 C shaker overnight
</LI><P>Gel:
</P><UL><LI>ran a ligation reaction overnight of the restriction digest products from 6/24
</LI></UL><H3><SPAN class="mw-headline" id="June_28">June 28</SPAN></H3><P>Transformation:
</P><UL><LI>We cannot use the plates made yesterday because ampicillin is not a marker for yeast; threw them out and planned to plate the yeast with Leu2, the real yeast marker for the plasmid pRS425
</LI><LI>Yeast transformation today
</LI><LI>Transferred 60 uL cells to 50 mL YPD
</LI><LI>Left the cells in the 30 C shaker for approximately 6 hours (checked the OD several times, but it was too low each time; the OD had to be between 0.6-1.2)
</LI><LI>Spun the cells at 3000 G for 5 minutes
</LI><LI>Discarded the supernatant
</LI><LI>Added 25 mL sterile water to each of the two tubes (825, 827)
</LI><LI>Spun again at 3000 G for 5 minutes
</LI><LI>Discarded supernatant
</LI><LI>Resuspended in 1 mL 100 uM lithium acetate to break open cells
</LI><LI>Put the cells into two microcentrifuge tubes each (one control and one transformation; four microcentrifuge tubes total)
</LI><LI>Put in the 30 C shaker for 15 minutes
</LI><LI>Found salmon sperm DNA and put it into a thermocycler, heating 25 uL of it to 98 C and the flash-cooling to 4 C
</LI><LI>Centrifuged the cell solutions for 30 seconds at 3000 G
</LI><LI>Pipetted out supernatant
</LI><LI>Added 240 uL of PEG 3350 to each tube
</LI><LI>Added 5 uL of salmon sperm DNA to each tube
</LI><LI>Added 34 uL sterile water to control tubes and 34 uL DNA to transform tubes
</LI><LI>Added 36 uL 1 M lithium acetate to each tube
</LI><LI>Vortexed
</LI><LI>Left on 30 C shaker for 30 minutes
</LI><LI>Left in 42 C water bath for 40 minutes
</LI><LI>Spun tubes at 1000 G for 30 seconds
</LI><LI>Pipetted out supernatant
</LI><LI>Added 800 uL YPD to each tube and resuspended by pipetting
</LI><LI>Left overnight at room temperature with a book on top to prevent tube caps from popping off
</LI></UL><P>Gel:
</P><UL><LI>ran a gel of PCR product from 6/24
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/CassettePCR66-27-2011.jpg?attachauth=ANoY7covbU51kmVgY28Z-b0hU00ykbLMYlTEZHatG33QPF4iPvIChYu0ieH97yLkfN5PTBxqKmJF5xHCWzutqSK7FGK8qj6BqYm9qOL5z7FTEGLjEG8P32NW7XeweWpLYF2kjUdf-NXNLLjc1CBk2CWbO4drrBUQNUEq2aEDsMCo7EjWsuxlY3q1_qaluRLL97ruWVT5CJIq4ZFXkNgPGFzHl7cteIiMGlqCi7xTQHtPyqPtH7Zhf94%3D&amp;attredirects=0" class="external autonumber" rel="nofollow">[12]</A>]
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/CassettePCR76-27-2011.jpg?attachauth=ANoY7coNW0jhbzG2I8ZbPg9PRT9GVFXBHd8ZdRwIVUmssdKORzPXW_aHxcw8oc6_xPSG8ar5faGEco8_VqWo1ad6Sj999imuhVZgiaS8xyjAJpai5Q8s9RWtuCOjREwHMe9NORQS7xFmIS2CBg0c2GC-F67F4vEqWqLMVfywCXKYL-EkAfMne-XTSFe2z9WsT48piqpHPWlJY198kYAhvyC0N2J4pE6c2_SxE3Vk2-50NdRfI7Buswc%3D&amp;attredirects=0" class="external autonumber" rel="nofollow">[13]</A>]
</LI></UL><LI>ran a gel of our ligation from 6/27
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/practiceligation1.jpg?attachauth=ANoY7cqnDhncwPaHrLoyouAca8p_S9mnMvYohAATaeuKRAbQA0UoWLEtQieXFFXDMk9896sNdntrQFUwJLw_OdaVn7w2yhxg3i-UDZZlcXHGonPdkrvABnCikENBu0A6qBl18TIEtR_dHlTJSflI3Ikywth7SSYIv7TNPxG7uFQXhAU_svEZcySCPRK92btGWgpktOX60n3qvsdgjbXxXkpJkRSo3_QvXw%3D%3D&amp;attredirects=0" class="external autonumber" rel="nofollow">[14]</A>]
</LI><LI>did a new restriction digest on nat casette using AgeI
</LI><LI>ran a gel of restriction digest
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/28.jpg?attachauth=ANoY7crPjoCsafhMlC2Xc47CuljJr1kkDXioWVrppI-f6y4xYAYxhwd9u0JV5HWTK6L1yLCqQ9mN_jXi1DAfnm3953SVC5UIXoOd_jDs25jAZbtWDHVwzDEpcqvTp4j98h5H-sQMpQFTScABn_qSEVXMBqMgqUiuSgaHTvWotpkAQ7cHh2Kxw7tnv7avQmbcz3GPLkWxQwmY&amp;attredirects=0" class="external autonumber" rel="nofollow">[15]</A>]
</LI><LI>Used spin column gel extraction to extract the DNA from our restriction digest gel
</LI><H3><SPAN class="mw-headline" id="June_29">June 29</SPAN></H3><P>PCR Protocol for Plasmids Review
</P><UL><LI>2.5uL 10x buffer
</LI><LI>1.25 uL dNTP
</LI><LI>16.5 uL dH20
</LI><LI>0.25 uL Accutaq LA DNA polyermerase 
</LI></UL><UL><LI>1.5 uL forward primer 
</LI><LI>1.5 uL reverse primer 
</LI><LI>10ng DNA (1uL if using a diluted sample)
</LI></UL><P>PCR on the Nat ligation product: 
</P><UL><LI>Nat was not detected on a gel but may be because a very small amount of product was made. 
</LI><LI>Well 1,2,3 are PCR mix with Nat primers and 3uL of Nat ligation product per well
</LI><LI>Well 4 is positive control with Kan primers and Kan DNA
</LI><LI>Well 5 just has 3uL of Nat ligation product and no primers.
</LI></UL><H3><SPAN class="mw-headline" id="June_30">June 30</SPAN></H3><P>Diluted gene primers
</P><P>crtYB F: 10uM, 576uL added
</P><P>crtE F: 10uM, 193uL added
</P><P>crtI F: 10uM, 1.028mL added
</P><P>
CCD1 F: 10uM, 1.614mL added
</P><P>Gene R: 100uM, 1.337mL added
</P><P>There is also a working solution of 10uM of gene R.
</P><P>Gel:
</P><UL><LI>Ran a gel of PCR product from 6/29
</LI><LI>[<A href="https://3626779877665202800-a-1802744773732722657-s-sites.googlegroups.com/site/washuigem2011/files/ligation6-30.jpg?attachauth=ANoY7cptLPUdoTvxKBIF0viM0O6B7en1T6Jq5LZYxwiZAZnfjwFWrfTHwpIF5NCBYUi-32Jh_sogj87WyU_Sfqj5LkS4YKvi1TUKs-6Y70fmiE03orrEEQClig9cUsnVI__RE_AlOehtXCUzvbmMHqPRbvUVPw8QZVUGLjZ8m0P3w7dW_SV2xHklwthqZOstm4uTrOXsrAmjTOwjIddR95wNhkadpF0boA%3D%3D&amp;attredirects=0" class="external autonumber" rel="nofollow">[16]</A>]
</LI></UL><LI>Did a restriction digest of nat casette using AgeI
</LI><LI>ran digest product on gel
</LI><LI>used PCR cleanup kit on part of digest product and used in a ligation reaction, ran reaction overnight
</LI><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:WashU/Notebook/June">http://2011.igem.org/Team:WashU/Notebook/June</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:WashU/Notebook/June" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:WashU/Notebook/June" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:WashU/Notebook/June&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:WashU/Notebook/June&amp;oldid=47966" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>