<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Harvard_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Harvard/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-container"><DIV class="brand"><A rel="nofollow" rel="noreferrer" href="https://2017.igem.org/Team:Harvard">Harvard OPTI POLY</A></DIV><NAV><UL class="nav-list"><LI><A rel="nofollow" rel="noreferrer" href="https://2017.igem.org/Team:Harvard">Home</A></LI><LI><A rel="nofollow" rel="noreferrer" href="#!">Team</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/Medal_Criteria">Medal Criteria</A></LI></UL><LI><A rel="nofollow" rel="noreferrer" href="#!">Project</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/Bioreactor">Bioreactor</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/Microfluidic">Microfluidic</A></LI><LI><A rel="nofollow" rel="noreferrer" href="#!">Lab</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/InterLab">InterLab</A></LI><LI><A rel="nofollow" rel="noreferrer" href="https://2017.igem.org/Team:Harvard/Basic_Parts">Parts</A></LI><LI><A rel="nofollow" rel="noreferrer" href="#!">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/Human_Practices">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Harvard/Engagement">Public Engagement</A></LI></NAV></DIV><DIV class="column full_size description" style="margin-top: 5%"><H1>Lab Notebook</H1><DIV class="panel-group"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse1"><B>Month 1: August Overview</B></A></H4></DIV><DIV id="collapse1" class="panel-collapse collapse"><DIV class="panel-body"><H3>Wetlab</H3>
		Got familiar with the protocols we would be using. 
		<H3>Model</H3>
		Started to work on the <A href="https://2017.igem.org/Team:Harvard/Model">mathematical model</A> for strain engineering parameters. 
        <H3>Bioreactor</H3>
		CAD Model and first 3D printed prototypes for Bioreactor
		<H3>Microfluidics</H3>
		Set foundational parameters for the device.
		</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse2">Week 1: 8/7 - 8/12</A></H4></DIV><DIV id="collapse2" class="panel-collapse collapse"><DIV class="panel-body"><H3>Wetlab</H3>
		In order to screen the best assay for our intended project, we tried multiple protocols to measure the quantity of produced curli, the desired polymer to optimize the production of. <CENTER><B>Growing curli producing <I>E. coli</I> on plates with congo red:*</B></CENTER><CENTER><B>Isolating curli by vacuum filtration:*</B></CENTER><CENTER><B>Running a congo red pull down assay:*</B></CENTER>

		From running the protocols in parallel we decided that the pull down assay served as the most consistent proxy for curli production. It was difficult to quantify the &quot;redness&quot; of the cultures on the red plates, and we did not notice significant difference between controls. For the vacuum filtration, it required large volumes of culture and small quantities of the yield product. 
		
		*Detailed procedure found on our <A href="https://2017.igem.org/Team:Harvard/Protocols">protocol page</A>.

		<H3>Bioreactor</H3>
		Created CAD model of bioreactor culture chamber and lid.
		
		</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse3">Week 2: 8/14 - 8/18</A></H4></DIV><DIV id="collapse3" class="panel-collapse collapse"><DIV class="panel-body"><H3>Wetlab</H3>
		To ensure consistency of results, we ran through multiple iterations of congo red pull down assays with multiple aliquots of the same liquid culture. 
		In the beginning we had different absorbance readings for the same liquid cultures, suggesting procedural error.
		<H3>Interlab</H3>
		Data was collected for the <A href="https://2017.igem.org/Team:Harvard/InterLab">interlab study</A>.
		<H3>Model</H3>
		Started to develop the mathematical model to see which gene we wanted to edit in the wild type strain of <I>E. coli</I>. This involved reading many papers on measurements of relevant parameters, such as secretion rate, rates of translation, transcription, etc. 
		<H3>Bioreactor</H3>
		Created CAD model for bioreactor impeller module.
		<H3>Microfluidics</H3>
		Mill prototype designs on polycarbonate pieces using CNC micro-milling machine
		<CENTER><B>First prototype of microfluidic design</B></CENTER></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse4">Week 3: 8/21 - 8/25</A></H4></DIV><DIV id="collapse4" class="panel-collapse collapse"><DIV class="panel-body"><H3>Wetlab</H3>
		We continued to repeat the pull down assay. Throughout the week, we focused on troubleshooting and minimizing performance errors. 
		<CENTER><B>Inconsistencies among triplicates</B></CENTER><CENTER><B>Consistent samples at the end of the week</B></CENTER><H3>Bioreactor</H3>
		Fabricated initial 3D printing prototypes and applied waterproofing sealant.
		<H3>Microfluidics</H3>
		Characterized optical sensors and design circuits.
		<CENTER><B>Waveforms generated by the optical sensor</B></CENTER></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse5">Week 4: 8/28 - 9/1</A></H4></DIV><DIV id="collapse5" class="panel-collapse collapse"><DIV class="panel-body"><H3>Bioreactor</H3>
		Created CAD model for motor mount.
		Performed water retention tests.

		<H3>Microfluidics</H3>
		Tested prototypes of mixing mechanisms using food coloring.
		<CENTER><B>Waveforms generated by the optical sensor</B></CENTER></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse6"><B>Month 2: September Overview</B></A></H4></DIV><DIV id="collapse6" class="panel-collapse collapse"><DIV class="panel-body"><H3>Wetlab</H3>
		Focused on generating the RBS library for strain engineering. Once we received the DNA parts, we used selection markers to appropriately transform, clone, and sequence the optimized strain.
        <H3>Model</H3>
		Continuous progress on the model development. 
	    <H3>Microfluidics</H3>
		Proof of concept for the microfluidic device.
		</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse7">Week 5: 9/4 - 9/8</A></H4></DIV><DIV id="collapse7" class="panel-collapse collapse"><DIV class="panel-body"><H3>Wetlab</H3>
		Designed an RBS library for csgG using the <A href="https://salislab.net/software/">Salis Lab RBS Library Calculator,</A>as well as the appropriate PCR primers to construct the library and ordered these sequences from IDT.
		<H3>Microfluidics</H3><UL><LI>Suspended <I>E. coli</I> strain in pbs and measured optical density using cuvettes and nanodrop machine.</LI><OL><LI>Spun down the liquid culture at max speed for 4 minutes.</LI><LI>Removed the liquid media.</LI><LI>Added Pbs and diluted the culture into different concentrations.</LI></OL><LI>Vapor polished surface of the milled polycarbonate: heat up methylene chloride in a bunsen flask and place the milled area over the vapor. This was done inside a fume hood.</LI><LI>Tested OD readings from vapor polished surface</LI></UL><CENTER><B>Setup for sensor calibration</B></CENTER></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse8">Week 6: 9/11 - 9/15</A></H4></DIV><DIV id="collapse8" class="panel-collapse collapse"><DIV class="panel-body"><H3>Wetlab</H3><H4>Day 1</H4>
		Our DNA parts ordered the previous week arrived and we conducted PCRs with the primers we designed and a plasmid containing the sequences for csgA, csgB, csgC, csgE, csgF, and csgG obtained from the Joshi Lab to modify the RBS sequence in front of csgG. 
		<H4>Day 2</H4>
		After verifying our PCR products with a gel, we used Gibson Assembly to put our cloned parts in an expression vector with kanamycin resistance. We then transformed our newly formed plasmids into competent cells using heat shock and plated them on agar plates with kanamycin. 
		<CENTER><B>PCR banding patterns indicated successful Gibson Assembly</B></CENTER><H4>Day 3</H4>
		After leaving our plates in an incubator overnight, we imaged the plates FluorChem E and ran an image analysis script on the images to determine the brightest colonies, which correspond to the colonies with highest curli production. We then picked the 2 brightest colonies on each plate, as well as 2 other randomly chosen colonies, and cultured them in 5 mL falcon tubes with liquid LB and kanamycin. 
		<CENTER><B>Example of imaged plates: Brightest colonies related to higher curli production</B></CENTER></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse9">Week 7: 9/18 - 9/22</A></H4></DIV><DIV id="collapse9" class="panel-collapse collapse"><DIV class="panel-body"><H3>Wetlab</H3>
		We ran a congo red pulldown assay on the cultures from the previous week to quantitatively measure the amount of curli produced. Then, we miniprepped the cell cultures to send out our parts for sequencing.
		<H3>Microfluidics</H3>
		Milled new designs of microfluidic device.
		</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse10">Week 8: 9/25 - 9/29</A></H4></DIV><DIV id="collapse10" class="panel-collapse collapse"><DIV class="panel-body"><H3>Microfluidics</H3>
		Suspend cultures in Pbs for another round of OD calibration.
		Vapor polished surface of the milled polycarbonate by heating up methylene chloride in a bunsen flask and placing the milled area over the vapor. This was done inside a fume hood. 

		</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse11"><B>Month 3: October Overview</B></A></H4></DIV><DIV id="collapse11" class="panel-collapse collapse"><DIV class="panel-body"><H3>Wetlab</H3>
		We prepared parts for submission to the parts registry, as well as sequence verified our constructs. 
		Bioreactor
		Cultured <I>E. coli</I> cultures in the bioreactor.
		<H3>Microfluidics</H3>
		Continuity of proofs of concept for the microfluidic device. Strong focus on optical sensor data analysis.
		</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse12">Week 9: 10/2 - 10/6</A></H4></DIV><DIV id="collapse12" class="panel-collapse collapse"><DIV class="panel-body"><H3>Wetlab</H3>
		We ran the congo red pulldown assay again for our constructs. 
		We sequenced our constructs, and analyzed the sequence in context of our model and other parameters. Refer to our <A href="https://2017.igem.org/Team:Harvard/Results">results</A> for a detailed analysis.
		<H3>Interlab</H3>
		We prepared our interlab data for submission. We also collected additional data for the <A href="https://2017.igem.org/Team:Harvard/InterLab">interlab study</A> and ran the congo red pulldown assay again.
		<H3>Bioreactor</H3>
		Cultured cells in the bioreactor, but observed minimal growth.
		<H3>Microfluidics</H3>
		Cultured PQN4 pBAD-CsgBACEFG (curli-producing positive control) in Kan (+) liquid media to prepare for experiments with surface treatment of microfluidic device. 
		Ensured OD calibration by pumping the cell cultures into microfluidic chambers via inlets of the device.
		Prepared Pbs-suspended cell cuture with dilution from OD600 0.1 to OD600 1.
		Used syringe pump to pump the liquid cultures in.
		<CENTER><B>Image credit to Boston U Harvardware team</B></CENTER></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse13">Week 10: 10/9 - 10/13</A></H4></DIV><DIV id="collapse13" class="panel-collapse collapse"><DIV class="panel-body"><H3>Wetlab</H3>
		We ran the congo red assay for a third time. We also sent more of our constructs to be sequenced since not all of the parts were able to be sequenced the first time. 
		<CENTER><B>Final data refer to <A href="https://2017.igem.org/Team:Harvard/Results" width="50%" height="50%"> results</A> for more details</B></CENTER></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse14">Week 11: 10/16 - 10/20</A></H4></DIV><DIV id="collapse14" class="panel-collapse collapse"><DIV class="panel-body"><H3>Wetlab</H3>
             Miniprepped DNA. </DIV><H3>Bioreactor</H3><CENTER><B>Final Cad Model</B></CENTER></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse15">Week 12: 10/23 - 10/27</A></H4></DIV><DIV id="collapse15" class="panel-collapse collapse"><DIV class="panel-body"><H3>Wetlab</H3>
		We conducted PCR on each of our miniprepped parts and cloned them into the pSB1C3 backbone for sample submission according to the <A href="http://parts.igem.org/DNA_Submission"> submission guidelines</A>.
		</DIV></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="social-media"><DIV class="text"> HARVARD IGEM 2017 </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>