<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tsinghua_project_outline_m2"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tsinghua/project/outline/m2">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tsinghua/project/outline/m2&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tsinghua/project/outline/m2&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tsinghua/project/outline/m2&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tsinghua/project/outline/m2" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tsinghua/project/outline/m2</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="topbar"><UL id="topbarul"><LI id="nav-home"><A href="/Team:Tsinghua/home">Home</A></LI><LI id="nav-team"><A href="/Team:Tsinghua/team">Team</A></LI><LI id="nav-project"><A href="/Team:Tsinghua/project">Project</A></LI><LI id="nav-experiment"><A href="/Team:Tsinghua/experiments">Experiment</A></LI><LI id="nav-parts"><A href="/Team:Tsinghua/parts">Parts</A></LI><LI id="nav-support"><A href="/Team:Tsinghua/support">Support</A></LI><LI id="nav-human"><A href="/Team:Tsinghua/HumanPractice">Human Practice</A></LI></UL></DIV><DIV id="content_box"><DIV id="right_bar"><DIV class="left_gadget"><DIV class="countdown"><DIV id="counttitle">
Until Jamboree</DIV><DIV class="countback"><P class="countstatic" id="pday"> days</P><P class="countstatic" id="phour"> hours</P><P class="countstatic" id="pmin"> minutes</P><P class="countstatic" id="psec"> seconds</P></DIV></DIV></DIV><DIV class="left_gadget"><H2>Follow us on</H2></DIV><DIV class="left_gadget"><H2>Visitor Locations</H2></DIV><DIV class="left_gadget"><H2>Join the conversation</H2></DIV></DIV><DIV id="main_content"><H2><SPAN class="mw-headline" id="Module_II">Module II</SPAN></H2><DIV class="content_block"><H3><SPAN class="mw-headline" id="Strategy_1_Bacterial_Based_Microarray">Strategy 1 <B>Bacterial Based Microarray</B></SPAN></H3><H4><SPAN class="mw-headline" id="Outline_of_screening_desired_antibody_by_bacterial_based_microarray">Outline of screening desired antibody by bacterial based microarray</SPAN></H4><P>Microarray technique is a kind of pretty mature biomedical technology applied to diverse areas from pharmacology research to clinical use, such as genetics diagnose. Due to the advantage that large amount of various substances can be screened in microarray, microarray can be used for selection of desired antibodies with specific affinity in our project. However, current existing microarray technology rely on specific DNA or protein attached substrate and our project lean on bacteria expressing specific surface protein used for screening for desired antibody. Therefore, traditional protocols and materials used in microarray will be adjusted in our project. <B>We believe that our adjustment will improve the efficiency of antibody screening and help our project achieve its ultimate goal.</B></P><H4><SPAN class="mw-headline" id="Specific_description_of_the_details_of_the_experiments">Specific description of the details of the experiments</SPAN></H4><P>In this part, we want to transform two batches of E coli with vectors carrying specific genes expressing various kinds of antibody achieved by the recombination methods in our project and specific antigen or some other protein used to select antibody from the aforementioned mix of antibodies. By fusion antibody and antigen gene with the genes of membrane integral displaying protein called OmpA, we can manage to display our protein to the surface of the bacteria. Therefore, the selection will process through interaction between antibody and antigen at the surface of the bacteria. Then, by linking the gene of display protein to the gene of one kind of protein called cellulose binding domain which can bind to cellulose, we are able to anchor the bacteria expressing CBD to the surface of the specific microarray coated with cellulose substrate. The whole process is illustrated in the following figure.
</P><H4><SPAN class="mw-headline" id="Information_concerning_the_specific_proteins">Information concerning the specific proteins</SPAN></H4><H5><SPAN class="mw-headline" id="Cellulose_binding_domain">Cellulose binding domain</SPAN></H5><P>The protein used in this part comes from CipC gene found in Clostridium cellulolyticum. CipC gene encodes a protein called cohesion which participate in degradation of cellulose for the benefits of the bacteria Clostridium cellulolyticum. The protein encoded in this gene contains a specific domain called cellulose binding domain which bind to cellulose with pretty high binding affinity and thus facilitate the degradation of the cellulose. The structure of CBD has been resolved shown in the figure below. Our purpose is to utilize the ability of CBD to bind to cellulose and express CBD in E coli and thus force it to bind to microarray coated with cellulose substrate.
</P><H5><SPAN class="mw-headline" id="Displaying_protein_OmpA">Displaying protein OmpA</SPAN></H5><P>At present, two kinds of membrane protein are known, that is, alpha helix protein from cytoplasmic membrane and beta-barrel protein in bacteria outer-membrane, exemplified by porin protein. There are two membrane layers in gram-negative bacteria, separately called inner membrane and outer membrane. OmpA is located in outer membrane and belongs to beta-barrel category. This protein is responsible for normal physiological functions in gene regulation in E coli. Because OmpA is located in outer membrane, intensive investigation has been conducted in OmpA because OmpA plays an important role in signal transduction and much work has focused on the mechanism how OmpA was transported from cytosol out of the membrane. Because of the clarified function of the protein, some bioengineers managed to utilize its function to display protein to achieve certain purposes. Therefore, we try to take advantage of the properties of OmpA to display protein, that is, to display CBD, antibody and antigen. The structure of OmpA has been determined, illustrated in the following figure.
</P><P>Previous work has demonstrated that 46-159 amino acid is adequate to anchor OmpA to membrane and from the topology figure, we can find out that the fragment of AA 46-159 comprise the transmembrane part of the protein, as the red box indicates. Besides, to ensure the transport of OmpA from cytosol to cell membrane, we need to add a signal peptide to the N terminal of OmpA. Previous work found that one kind of lipoprotein in E coli contains a signal peptide consisting of nine amino acids, which ensures the successful transport of the protein outside the membrane. Mining through the parts provided by iGEM, we found one part which provides exactly the same sequence we wants. This part contains N terminal signal peptide and the sequence AA 46-159 and the linker region downstream of OmpA which ensures the folding of the attached domain, such as CBD. The engineered OmpA is illustrated as following.
</P><H5><SPAN class="mw-headline" id="The_selection_of_antibodies">The selection of antibodies</SPAN></H5><P>Due to commercialization of antibody, we have no access to the cDNA of these antibodies. Therefore, we managed to synthesize the cDNA.
</P><H5><SPAN class="mw-headline" id="Design_of_microarray">Design of microarray</SPAN></H5><P>Microarray is a solid substrate based 2 dimensinoal array, which can be utilized to detect large sum of bio-substance. After decade of developing, various kinds of microarrays have been invented, such as DNA microarray, protein microarray, tissue microarray and so on. Different microarrays can be used to detect different components based on different substrates. Our project belongs to cellular microarray, aiming to select different kinds of antibodies based on the interaction between different proteins displayed on bacteria membrane.
This part of our project aims to attach CBD-expressing bacteria to the surface of microarray coated with cellulose. Weizmannn Institute has developed zephyrin-based microarray, in which the microarray plate is dotted with cellulose. Thus, this innovation provides potential for application in our project.
</P><H3><SPAN class="mw-headline" id="Strategy_2_ToxR-based_Transmembrane_Signaling_Pathway_Method">Strategy 2 <B>ToxR-based Transmembrane Signaling Pathway Method</B></SPAN></H3><H4><SPAN class="mw-headline" id="Propose_of_This_Step">Propose of This Step</SPAN></H4><P>In order to select a specific antibody from a large number of antibodies, we studied the formation of mammalian antibodies. In the process of antibody production in mammals, mediated Annexin antibodies-mediated antibody activation play an important role. <B>This section hopes to use a membrane receptor of E. coli to simulate this process, to achieve the purpose of screening.</B></P><H4><SPAN class="mw-headline" id="Principle_and_Background">Principle and Background</SPAN></H4><P>To achieve this, a ToxR-based two-hybrid system was introduced into E.coli cells. The Vibrio cholerae transcriptional regulator ToxR is anchored in the cytoplasmic membrane by a single transmembrane segment, its C-terminal domain facing the periplasm. Most of its N-terminal cytoplasmic domain shares sequence similarity with the winged helix–turn–helix (wHTH) motif of OmpR-like transcriptional regulators. The ToxR protein of Vibrio cholerae regulates the expression of several virulence factors that play important roles in the pathogenesis of cholera.
</P><P>ToxR stimulates transcription from the cholera toxin gene promoter ctx by direct binding to DNA element ITITTGAT present in different isolates of Vcholerae in three to eight tandemly repeated copies upstream of the ctxAB structural genes.
</P><P>Transcription activation is thought to be initiated by environmental stimuli which cause the periplasmic ToxR domain to form a homodimer. This, in turn, tethers together the two cytoplasmic ToxR domains, which can now bind to the control region of the ctx promoter. A second membraneassociated protein, ToxS (Mr 19 000), is required for maximal activation of the ctx promoter; most plausibly it stabilizes the ToxR dimer by direct contact.
</P><P><B>Figure Shows the Original structure of ToxR system in Vibrio cholerae</B></P><P>Fusion protein was made between the inner- and trans-membrane parts of ToxR and of the recombinant antibody.
</P><P>When the specific antigens recognized by antibodies, antibodies with the same antigen recognize site will be close to each other through the antigen，inner-membrane part of “Antibody-ToxR” Protein dimerize, ctx promoter active, Resistance gene starts to be transcribed.
</P><P>Use “Anti-Histag Antibody” and “Protein with His-tag” as “antibody” and “antigen” to verify the effect of this system.
</P><P>Antibody: ToxR + Anti-Histag Antibody scFv Light Chain
</P><P>Plus:     ToxR + Anti-Histag Antibody scFv Heavy Chain
</P><P>Antigen: His tag + Protein A + Histag
</P><H3><SPAN class="mw-headline" id="Protocol">Protocol</SPAN></H3><P>We need to achieve the construction of the selection vector under the control of ctx promoter, then induce to the bacteria.
Measure the expression of the two genes and measure the distribution in membrane.
Use the exogenous protein carried His-tag as the antigen.
Treat with antibiotics.
Compare the expression level between the experiment group and the control, then define the effect of our system indirectly.
</P><P>Reference：A ToxR-based two-hybrid system for the detection of periplasmic and cytoplasmic protein–protein interactions in Escherichia coli: minimal requirements for specific DNA binding and transcriptional activation
</P><H3><SPAN class="mw-headline" id="Strategy_3">Strategy 3</SPAN></H3><P>Cooperation with <A href="https://2010.igem.org/Team:Macquarie_Australia" target="blank"><FONT face="Comic Sans MS" size="4">Macquarie Australia</FONT></A></P><P>  <A href="#top"> TOP</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Tsinghua/project/outline/m2">http://2010.igem.org/Team:Tsinghua/project/outline/m2</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tsinghua/project/outline/m2" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tsinghua/project/outline/m2" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tsinghua/project/outline/m2&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tsinghua/project/outline/m2&amp;oldid=189137" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>