"
Page
Discussion
View source
History
teams
Log in
 
Team:Tsinghua/experiments/results
From 2010.igem.org
Home
Team
Project
Experiment
Parts
Support
Human Practice
Until Jamboree
days
hours
minutes
seconds
Follow us on
Visitor Locations
Join the conversation
Results
Landing Pad Construction
  
Left：The landing pad (LP) vector backbone and tetracycline gene with landing pad on its side, with size of 1k and 1.4k
Right：extraction of LP vector, about 2.4k.
Landing Pad Insertion
Colony PCR: Upper primer is in the E.coli genome, 500bp before att site, while the downer primer is in the landing pad. The positive result comes out only when the recombination occurs, shown as the figure. The last lane acts as a negative control, using the untransformed E.coli
Helper Plasmid Insertion
We transformed the helper plasmid (HP) into E.coli, and extracted the vector, about 6k.
Donor Plasmid（2） Construction
  
Here we use a multiple ligation method to construct our donor plasmid (containing two genes)
Left：the size of every part (vector backbone, chloromycetin , and kanamycin): 1k，1.6k，1.1k
Right：We transformed the ligation product and extracted the DP，and first superhelix is about 2.3k
Donor Plasmid（4） Construction
  
PCR: eGFP, mCherry, Kan and Chlr four genes separately from PIB-eGFP, PBS34, PKD13 and PKD3 plasmids.
PCR to check pUC19+eGFP and pUC19+ChlrPCR. All 12 samples are positive.
Double digest PE1 and mCherry with SalI and BamHI, double digest PC1
Using sequencing primers to PCR PEM2+Kan. Lane 1 and 5 are positive.
Use sequencing primers to PCR PEMK1+Chlr. Lane 3-5 are positive.
DP Insertion and Recombination Induction
Single Gene Recombination (Kan)
PCR checking：Upper primer is in the E.coli genome, 500bp before att site, while the downer primer is in the landing pad. The positive result comes out only when the recombination occurs, shown as the figure. The left six lane are belong to a group of Negative Control. Kanamycin is about 700bp, while the recombination rate is 100%
Two Genes Recombination (Kan and Chlr)
PCR checking: method as before. Close to the marker, we choose an untransformed E.coli as the negative control. The following eight lanes are randomly picked experimental group, half of which show as we expected. These colonies are picked from the plate with Kan resistance.
PCR checking: method as before. Close to the marker, we choose an untransformed E.coli as the negative control. The following eight lanes are randomly picked experimental group, three of which show as we expected. These colonies are picked from the plate with Chlr resistance.
  
Here we picked colonies from Kan resistance plate and Chlr resistance plate in the same time. Left figure shows the Kan positive results in the 1,2,4 lanes, while right figure shows the Chlr positive results in the 3,4,7,8 lanes.
Calculation of Recombination Rate
Recombination rate= (Positive Colony Numbers in Kan Plate + Positive Colony Numbers in Chlr Plate)/  (Positive Colony Numbers in Kan Plate + Positive Colony Numbers in Chlr Plate + Colony Numbers in Tet Plate)
We have counted the total number of colonies in the Kan plate, the number is 213, and its effective rate is 0.75; The total number of colonies in the Chlr plate, the number is 602, and its effective rate is 0.68; Colonies form Kan and Chlr are coated in Tet resistance plate, the number is 451 and 417.
So our Recombination Rate= (213*0.75+602*0.68) / (213*0.75+602*0.68+451+417) = 39.6%
CBD presentation system
PCR check: we ligated CBD coding gene and OmpA coding gene to the targeted vector, then transformed to E.coli and extracted the vector. All these four vectors show the positive results, about 1k.
SDS-PAGE Check: CBD-OmpA band is circled in the figure.
Retrieved from " http://2010.igem.org/Team:Tsinghua/experiments/results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
