<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Queens-Canada_contributing"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Queens-Canada/contributing">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Queens-Canada/contributing&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Queens-Canada/contributing&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Queens-Canada/contributing&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Queens-Canada/contributing" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Queens-Canada/contributing</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="bodyb"><H1><SPAN class="mw-headline" id="Contributing">Contributing</SPAN></H1><DIV class="asideR"><P>A simple worm construct:</P><TABLE id="magic_table"><TBODY><TR><TD align="center">Promoter Brick</TD><TD align="center">Protein-Coding Brick</TD><TD align="center">3' UTR Brick</TD></TR></TBODY></TABLE></DIV><P>We expect that future worm BioBricks will be submitted to the Parts Registry just like every other standard biological part, and use the conventional ligation standards such as Standard 10, which we utilized for our project. In the future, this will make sense to extend so that it fully supports protein fusions through the use of <A href="/Team:Queens-Canada/transcripts" title="Team:Queens-Canada/transcripts">introns</A> to conceal scars by placing intron splice sequences at the starting and finishing parts of fusion-compatible bricks, or by using special primers to insert the intron sequences during PCR to provide more control over which ends of the bricks are concealed. For now, however, this article will concentrate on the format we’ve put into practice.
</P><DIV class="section"><H2>Promoter Format</H2><P>The majority of <I>C. elegans</I> promoters used in research don’t draw a distinction between the cis-acting recognition sites used during the RNA transcription process and the upstream region of the transcript used during polypeptide translation, instead combining the two and labelling that the promoter. <A href="http://worfdb.dfci.harvard.edu/promoteromedb/" class="external text" rel="nofollow">Promoterome</A>, for example, goes as far as including the start codon itself in its definition of the promoter. While this is simplistic for synthetic biology’s purposes, for the time being we have largely chosen to keep this format, although to cope with the scarring caused by ligation, we do not include the start codon in the promoter, and so typically remove the three final nucleotides of the promoter sequence.</P></DIV><DIV class="section"><H2>Coding Brick Format</H2><P>The coding brick, in our format, extends from the start of a <A href="/Team:Queens-Canada/promoter" title="Team:Queens-Canada/promoter">synthetic Kozak sequence</A>, past the start codon, through the coding DNA itself, and ends with the stop codon. It may contain introns. The Kozak sequence we typically use are the nucleotides <B>GCCGCCACC</B> placed immediately before the start codon. This improves translational efficiency, and can be altered to provide the additional flexibility of controlling expression from within the protein-coding component of a complete gene. <I>Note</I>: under certain circumstances in <I>C. elegans</I>, the UGA codon does not stop translation, but instead appends the rare amino acid selenocysteine. While this has not impeded us so far, it may be preferable to use UAA or UAG instead to be safe.</P></DIV><DIV class="section"><H2>3' UTR Format</H2><P>These are taken verbatim from available databases and laboratories after checking for cut sites. Mutagenesis of these sites typically has few ramifications. See the <A href="/Team:Queens-Canada/rnai" title="Team:Queens-Canada/rnai">article on 3' UTRs</A> for more information on why.</P></DIV><DIV class="section"><H2>Future Project Ideas</H2><P>We realize that the idea of working with an organism other than the familiar E. coli chassis is probably an challenging prospect for a lot of teams, especially when it comes to the design phase. We came up with a lot of ideas that only seemed apparent after we had done a great deal of research on the worm; many of them are foundational advances that are analogous to things that can be done currently in bacteria, and would be crucial to making <I>C. elegans</I> a fully-viable system. We've put the most lucid bunch of these ideas here, with the hope that it would help jump-start other teams and make their work less daunting.
</P><H3><SPAN class="mw-headline" id=".22Backward_compatibility.22_with_E._coli">&quot;Backward compatibility&quot; with E. coli</SPAN></H3><UL><LI><B>Exosymbiosis</B>. Attaching the bacteria to the outside of the worm (using lectins?) on a surface-disrupted mutant. This may require bacteria that don't emit the usual nutritive signal. The <A href="https://2007.igem.org/Berkeley_UC" class="external text" rel="nofollow">BactoBlood</A> team went to some length to try and perfect such a bacterium, and since we're not talking about putting things in people, their work may be applicable here. An extension of this might be found in enabling the bacteria to attach or detach from the worms under certain circumstances, creating a bacteria distribution and relocation mechanism. Such a project would need to look out for the potential of covering the amphid or mouth, which some bacteria do naturally as a means of killing worms.
</LI><LI><B>Enterosymbiosis</B>. There are cases in nature where nematodes and bacteria have learned to work more or less together, in which the bacteria reside in the worm's gut. Studying <I>Xenorhabdus nematophilia</I>, <I>Photorhabdus luminescens</I>, and the <I>Heterorhabditidae</I> worms and trying to create something similar between <I>E. coli</I> and <I>C. elegans</I>; also look into <I>bus</I> mutants.
</LI></UL><H3><SPAN class="mw-headline" id="Messaging_in_the_worm">Messaging in the worm</SPAN></H3><UL><LI><B>Inter-worm signalling</B>. The <I>C. elegans</I> excretory gland normally produces some pheromones that are used for communication between worms; so do anus and vulva. Could we engineer these organs to produce and excrete some new molecule? The potential secondary applications are extensive. We haven't been able to find much literature on these topics, but the prospect is alluring.
</LI><LI><B>Hormones</B>. The <A href="/Team:Queens-Canada/pseudocoelom" title="Team:Queens-Canada/pseudocoelom">pseudocoelom</A> bathes most tissues, and it's believed that the worm has a number of hormones that it uses to communicate between tissues. It might be possible to transmit stuff in here of our own devising. As of yet, we don't have a strategy for receiving molecules from the pseudocoelom by a cell, but a literature search helped us find a tag that works to introduce molecules into the pseudocoelom. You can find it in our <A href="/Team:Queens-Canada/pseudocoelom" title="Team:Queens-Canada/pseudocoelom">pseudocoelom article</A>. We have not personally tested this technique.
</LI></UL><H3><SPAN class="mw-headline" id="Genetics">Genetics</SPAN></H3><UL><LI><B>Self-censorship</B>. <I>C. elegans</I> supports <A href="/Team:Queens-Canada/rnai" title="Team:Queens-Canada/rnai">RNAi</A>, a technique that leverages machinery which enables complementary RNA molecules to destroy each other. RNAi is generally used to knock out genes by feeding the worm bacteria that produce the complementary RNAs, or soaking the worm in a large volume of these RNAs. Since integrating new DNA into the <I>C. elegans</I> chromosomes is difficult and time-consuming, it may make more sense to give the worm a plasmid that knocks out the mRNAs autonomically. To restore functionality of a protein knocked out in this way, especially a mutated version of such a protein, consider employing heavy use of synonyms (alternative codons for the same amino acid). See our article on RNAi.
</LI><LI><B>Variable UTRs</B>. We've only provided a single 3' UTR in our kit (a synthetic modification of unc-54's 3' UTR). However, it's known that UTRs play a role in tissue specificity and expression strength by binding various factors. A better understanding of these UTRs, or even just a larger variety of them, stands to be a very useful project.
</LI><LI> Modular promoters. Heidelberg <A href="https://2009.igem.org/Team:Heidelberg" class="external text" rel="nofollow">stole the show in 2009</A> by creating a detailed set of promoter elements that could be arranged to produce predictable and precise variations in strength for HeLa cells. It would be very cool if something like this could be done for worms, especially if this could also include tissue specificity. The potential for doing something similar for 3' UTRs is also alluring.
</LI></UL><H3><SPAN class="mw-headline" id="Biosynthesis">Biosynthesis</SPAN></H3><UL><LI> The <A href="/Team:Queens-Canada/digestive" title="Team:Queens-Canada/digestive"><B>hindgut</B></A> (intestine) of <I>C. elegans</I> can be targeted with expression patterns that place emphasis on the front and back regions. In combination with the intestinal export tags used for secreting proteases and other digestive enzymes, this provides an environment of approximately pH 5 in which multi-step biosynthesis could be possible.
</LI></UL><H3><SPAN class="mw-headline" id="Fundamental_Tools">Fundamental Tools</SPAN></H3><UL><LI> One thing we were hoping to make this summer but never got around to was a <B>multi-species plasmid</B>: a new BioBrick backbone vector that would express a fluorescent protein in both <I>C. elegans</I> and <I>E. coli</I>. This would make it possible to eliminate the coinjection marker from the microinjection process, and guarantee that the DNA was functional in the worm if an injection were successful. In order to be grown up in <I>E. coli</I>, however, it would need to reuse the coding sequence for both eukaryotic and prokaryotic translation in order to save space. Implementing this might require the crafty use of <A href="/Team:Queens-Canada/transcripts" title="Team:Queens-Canada/transcripts">introns</A>.
</LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Queens-Canada/contributing">http://2010.igem.org/Team:Queens-Canada/contributing</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Queens-Canada/contributing" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Queens-Canada/contributing" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Queens-Canada/contributing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Queens-Canada/contributing&amp;oldid=207681" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>