"
Page
Discussion
View source
History
teams
Log in
 
Team:Queens-Canada/rnai
From 2010.igem.org
RNA Interference and the 3' UTR
RNAi
RNA interference is a laboratory technique wherein a natural gene-silencing mechanism is exploited to suppress undesired gene activity, by targeting messenger RNAs with their complementary sequences. It was discovered in C. elegans, and has been used in many other organisms since.
Like DNA, RNA can form a stable duplex, although it is much more delicate in vitro as a result of the abundant ribonucleases (RNases) produced by higher animals. This double-stranded RNA (dsRNA) is used as the storage format for the genomes of Group III viruses, and in the laboratory is typically formed using viral RNA polymerase, either in vitro like a single-step PCR reaction, or in vivo using engineered E. coli, which are then fed to the worm. The double-stranded RNA enters the worm’s tissues via the SID-2 channels located in the intestine, where it is then detected by a type of RNase III, called Dicer.
Dicer is an endonuclease that cuts dsRNA into 20–25 bp-long fragments, with 2 nt overhangs. These fragments, called small interfering RNAs (siRNAs), are responsible for the job of actually censoring messenger RNAs: they can fit through another channel called SID-1, which allows the RNA interference effect to spread throughout the rest of the worm’s body, except neurons, and bind to RDE-4 to create the RNA-induced silencing complex (RISC), which catches messenger RNAs that match the siRNA, and then cuts them up. In addition, the siRNA is amplified directly by an RNA-dependent RNA polymerase.
The RNAi mechanism most likely evolved to stop dsRNA viruses, and plays an important role in restraining the activity of cancer-fighting interferon proteins, as well as transposons (genetic elements that use transcriptases and endonucleases to move their sequences around the genome.) However, it is also used natively as a gene-silencing mechanism, and a gene-moderating mechanism; there are 175 known miRNAs (MicroRNAs) in C. elegans, which in some cases may be more sensitive and not induce full suppression due to imperfect base-pairing. miRNAs often target sequences in the 3' untranslated region of messenger RNAs rather than within the CDS (coding sequence), as these may more readily be selected to contain a distinct sequence without affecting protein structure or function.
Protocols for working with RNAi can be found can be found at Cold Spring Harbor Protocols .
RNAi offers synthetic biologists the opportunity to selectively silence gene expression, and may in the future allow for finer tuning of these effects (i.e. dampening expression to varying degrees as opposed to abruptly silencing it).  RNAi is a versatile technique in that its effects can be induced by ingestion of RNAi (through feeding E. coli that have been engineered to express it, or through direct exposure) or by expression of RNAi by the worm.  The latter offers especially interesting opportunities, as the expression can be put under an inducible promoter, which allows a gene to be silenced as a result of an external factor or input.  For example, a temperature-regulated promoter could be used to induce silencing of a gene.
More on RNAi
This paper outlines a method for the inducible expression (in this case by temperature) of RNAi that results in the inhibition of mechanosensation by a worm.
Here is a fantastic overview of the major RNAi techniques used by researchers that study gene expression.  Many of these techniques could be useful tools to a synthetic biologist working with C. elegans.
As mentioned above, the neurons in the N2 strain which we use as a standard chassis lack SID-1 (the receptor that allows siRNAs to proliferate from one cell to another) rendering them invulnerable to the effects of ingested RNAi. This paper demonstrates that neurons can be made susceptible to RNAi by engineering them to express SID-1.
The 3' UTR
In eukaryotes, there is typically a large amount of RNA in a messenger RNA that is not part of the coding sequence itself. The portion that precedes the start codon is called the 5' untranslated region, and the portion that follows it is called the 3' untranslated region. Both of these are generally hundreds of basepairs in length, which often inflates the apparent size of C. elegans promoters, as worm biologists typically measure up to the start codon, rather than drawing a distinction between the transcription and translation steps.
The 3' UTR, however, extends in the opposite direction, after the stop codon, and provides binding sites for moderating proteins and miRNAs (see above ), as well as stability and protection against RNA-degrading proteins, due to polyadenylation. (This is contrary to bacteria, where poly(A) sequences promote degradation for coding RNAs instead of preventing it.) 3' UTRs can greatly extend the lifespan of a messenger RNA transcript in the cell, as well as select for tissue and growth phase specificity. It is only in combination with 3' UTRs that promoters are able to target individual cells with the incredible precision necessary for the minute detail of the nematode’s structures.
The most important element of the 3' UTR is the polyadenylation signal, which also serves as the effective transcriptional terminator, even though the stop site of transcription may vary by up to 50 bases. The most common polyadenylation signal is AAUAAA, which is recognized in the transcript by a protein called CstF. The poly(A) sequence produced by the AAUAAA sequence is 250 nt long, which is sufficient to last several hours under ideal circumstances, and gradually shortens (variations on this signal yield shorter tails.)  In addition to limiting degradation, poly(A) sequences are involved in nuclear export and translational regulation.
For the original WormWorks chassis set, only the 3' UTR from the gene unc-54 is included. This 3' UTR is the one most frequently used in C. elegans research when constructing transgenes, and gives good, constitutive expression in all tissues, allowing selection to be done by promoters only. However, this also limits the precision with which we can target and differentiate between worm tissues. Future work in C. elegans synthetic biology may benefit greatly from the ability to design and assemble 3' UTRs with predictable, flexible, quantified characteristics. Work towards this may benefit from the knowledge gathered at UTRome and UTRdb .
Return to the Guide Hub
Retrieved from " http://2010.igem.org/Team:Queens-Canada/rnai "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
