<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Modeling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Modeling</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>内页</TITLE></P><DIV class="nbv"><DIV class="nbvmain"><UL id="nbv_all"><LI id="b1"><A href="https://2015.igem.org/Team:Tianjin">HOME</A></LI><LI id="b2"><A class="note" href="https://2015.igem.org/Team:Tianjin/Project">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Project?1">DESCRIPTION</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Project?2">RESULTS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Project?3">DESIGN</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Project?4">PROTOCOLS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Project?5">FUTURE WORK</A></LI></UL><LI id="b3"><A href="https://2015.igem.org/Team:Tianjin/Parts">PARTS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Parts?1">TEAM PARTS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Parts?2">BASIC PARTS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Parts?3">COMPOSITE PARTS</A></LI><LI class="current" id="b4"><A href="https://2015.igem.org/Team:Tianjin/Modeling">MODELING</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Modeling?1">OVERVIEW</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Modeling?2">ADSORPTION</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Modeling?3">PET DEGRADATION</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Modeling?4">PROTEIN EXTRACTION</A></LI><LI id="b5"><A href="https://2015.igem.org/Team:Tianjin/Collaborations">COLLABORATIONS</A></LI><LI id="b6"><A href="https://2015.igem.org/Team:Tianjin/Safety">SAFETY</A></LI><LI id="b7"><A href="https://2015.igem.org/Team:Tianjin/Practices">HUMAN PRACTICE</A></LI><LI id="b8"><A href="https://2015.igem.org/Team:Tianjin/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Notebook?1">TIMELINE</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Notebook?2">LAB WORK</A></LI><LI id="b9"><A href="https://2015.igem.org/Team:Tianjin/Team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Team?1">ABOUT THE TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Team?2">MEMBERS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Team?3">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Team?4">ACKNOWLEDGEMENTS</A></LI></DIV></DIV><DIV class="main-page"><DIV class="right" id="right1"><DIV class="content" id="content1"><DIV id="content-in"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 25px; font-family: arial,helvetica,sans-serif;"><STRONG>Overview</STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">This ye</SPAN><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">ar, Tianjin-iGEM makes a good combination of experiment design and modeling and solves all kinds of problems we meet.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">What’s the best condition of the two-phase extraction experiment? Do not need compare experiments results in different conditions any more. Using only few data, we can calculate the exactly best condition through computer programs and give great help to our experiments.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">How hydrophobin self-assembles on the surface? You really do not have to use electron microscope. See how we propose a physical model using mathematical method and how the model fits well with the experiment data.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">Why the application of hydrophobin accelerates the cutinase catalyzed PET hydrolysis? The classic model won’t work. Novel kinetic models are made and it is amazing that we make it visible using computer software.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">Mathematical model and computer stimulation give us powerful method to predict, improve and direct our experiments.</SPAN></P></DIV><DIV id="content-in"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:25px;font-family:'Arial',sans-serif">Adsorption</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">We decided to create a model of Janus adsorption for two main reasons:</SPAN></P><P style="margin-left: 32px; line-height: 1.2em;"><SPAN style="font-size:8px;font-family:Wingdings"><SPAN style="font:9px 'Times New Roman'">  </SPAN></SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Describing the self-assembly process of Janus on the solid substrate or air/water interface</SPAN></P><P style="margin-left: 32px; line-height: 1.2em;"><SPAN style="font-size:8px;font-family:Wingdings"><SPAN style="font:9px 'Times New Roman'">  </SPAN></SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif">To support and complement enzyme kinetic modeling as the basic part of heterogeneous kinetics</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Model Formation</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">As literature review, we know that Class I Janus are able to self-assemble at an air/water or water/oil interface, forming a robust, electron-microscopically identifiable monolayer characterized by a rodlet structure. Yu <EM>et al.</EM> studied the formation of rodlets of the inJanus from <EM>Grifola frondosa</EM>. They used a Langmuir trough and found that rodlets formed at the air-water interface during compression of a surface film of inJanus. de Vocht <EM>et al.</EM> repeated the experiment to have an image of rodlet structure after drying an aqueous solution with scanning force microscope. As for Class II hydrophobins, HFBI, HFBII and HFBIII from <EM>T. reesei</EM> have been shown to form films with a self-assembled hexagonally ordered structure (Paananen <EM>et al</EM>. 2003; Kisko <EM>et al</EM>. 2005; Kisko <EM>et al</EM>. 2007). Multilayer Langmuir-Blodgett films, HFBI and HFBII proteins formed hexagonal crystallites (Kisko <EM>et al</EM>. 2005.) Self-assembled hexagonally ordered films of HFBIII on an air/water interface and on a silicon substrate were also studied (Kaisa Kisko <EM>et al.</EM> 2007)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">As evidence given above, the pattern of two classes Janus are totally different.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Class I Janus</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">A key difference between class I and class II Janus is that only class I Janus form amyloid-like rodlets. Ingrid Macindoe <EM>et al</EM>. compared three types of EAS: wild-type EAS, EAS</SPAN><SUB>Δ<SPAN style="font-size:19px;font-family:'Arial',sans-serif">15</SPAN></SUB><SPAN style="font-size:19px;font-family:'Arial',sans-serif">(a truncated version of EAS that lacks 15 residues from the disordered Cys3–Cys4 loop) and F72G EAS<SUB>Δ15</SUB>(F72 mutate to G) by nuclear magnetic resonance(NMR) spectroscopy. They also carried out molecular docking simulations using the data-driven docking program HADDOCK. Different sets of intermolecular hydrogen bond restraints between neighboring monomers over residues S71-I75 in both parallel and anti-parallel arrangements were tested. The width of the rodlet generated in this way is approximately 65 Å, which agrees very well with the measured width of approximately 61 Å for EAS<SUB>Δ15</SUB> rodlets(Ann H Kwan,2008)(Figure 1)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><STRONG><SPAN style="font-size:16px;font-family:'Arial',sans-serif">Figure 1. </SPAN></STRONG><SPAN style="font-size:16px;font-family:'Arial',sans-serif">Ribbon diagram of EAS<SUB>Δ15 </SUB>hexamer in gray showing packing of side chains in the β-spine core and location of charged residues. Side chains of F72, L73, and I74 are shown as purple sticks, N67 as green sticks and T66 and T68 as orange sticks. Red and blue spheres denote positions of Cγ and Cε atoms of Asp an Lys side chains, respectively. Horizontal line illustrates that the majority of charges are located on the “bottom” half of the structure (corresponding to the hydrophilic face).</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">So, as the data obtained from MD simulation above, the diameter of monomer is 27 Å. </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><STRONG><SPAN style="font-size:16px;font-family:'Arial',sans-serif">Figure 2.</SPAN></STRONG><SPAN style="font-size:16px;font-family:'Arial',sans-serif"> Model of the rodlet unit by overview. Two circles simplify hydrophobins as regular spheres. The connection between dimer is the β-barrel unit.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">To figure out the stoichiometric relationship between Janus and surface site, we calculate area of Janus and a unit respectively. We assumed the size of a Janus is far larger than a single surface bound site.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">If regarding dimer as a unit (Figure 2), the area of two Janus(S<SUB>H</SUB>)= </SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif">1145 Å<SUP>2</SUP>, and the area of <A><SPAN style="color:windowtext;text-underline:none">rectangle</SPAN></A> unit(S<SUB>A</SUB>) = </SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif">1755 Å<SUP>2</SUP>.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">So, the elementary reaction equation is simplified as, </SPAN></P><TABLE align="center" cellpadding="0" cellspacing="0" width="340"><TBODY><TR class="firstRow" style=";height:18px"><TD style="border: 1px solid windowtext; padding: 0px 7px;" height="18" valign="top" width="88"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Parameters</SPAN></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="18" valign="top" width="252"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Meaning</SPAN></P></TD></TR><TR style=";height:18px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="18" valign="top" width="88"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">H</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="18" valign="top" width="252"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Hydrophobin(Janus)</SPAN></P></TD></TR><TR style=";height:18px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="18" valign="top" width="88"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">A<SUB>s</SUB></SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="18" valign="top" width="252"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Active   site on the solid surface</SPAN></P></TD></TR><TR style=";height:18px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="18" valign="top" width="88"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">k<SUB>1</SUB></SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="18" valign="top" width="252"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Associated   constant</SPAN></P></TD></TR><TR style=";height:19px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" align="center" height="19" valign="middle" width="88"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">k-<SUB>1</SUB></SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="19" valign="top" width="252"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Dissociated   constant</SPAN></P></TD></TR><TR style=";height:26px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="26" valign="top" width="88"><P style="text-align: center; text-indent: 0px; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">θ</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="26" valign="top" width="252"><P style="text-align: left; text-indent: 0px; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">fractional   occupancy of the adsorption sites</SPAN></P></TD></TR><TR style=";height:24px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="24" valign="top" width="88"><P style="text-align: center; text-indent: 0px; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">τ/τ<SPAN style=""><SUP>∞</SUP></SPAN></SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="24" valign="top" width="252"><P style="text-align: left; text-indent: 0px; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Surface   concentration of adsorption/ Maximal surface concentration</SPAN></P></TD></TR></TBODY></TABLE><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif"> </SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Hypothesis</SPAN>:
</P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">(1) Each site can hold at most one molecule of Janus(mono-layer coverage only)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">(2) The surface containing the adsorbing sites is perfectly flat plane with no corrugations (assume the surface is homogeneous)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">(3) The adsorption process is reversible </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">(4) There is an H-bond interaction in the β-barrel forcing to form rodlets. </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">When adsorption rate equals to desorption rate,</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;">,</P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">simplify it and solve</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif">θ</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif">, ,  where </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The adsorption isotherm equation is,</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Class II Janus</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The high resolution of the AFM images from both the air-facing side and the water-facing side enabled comparison of the repeating structures in the AFM images with the dimensions of an HFBI monomer.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><STRONG><SPAN style="font-size:16px;font-family:'Arial',sans-serif">Figure 3. </SPAN></STRONG><SPAN style="font-size:16px;font-family: 'Arial',sans-serif">Model of the structure of HFBI film. The film viewed from the hydrophobic side (A)  and from the hydrophilic side (B). (A) and (B) are processed AFM images showing the average repeating structures in a single crystalline area. In the middle panel and expanding to both sides is a model of the monomer arrangement in the film(Géza R. Szilvay,2007)</SPAN><SPAN style="font-size:16px;font-family:'Arial',sans-serif">The model in figure suggests that three monomers forming a trimer are arranged to form a hexagonal pattern.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><STRONG><SPAN style="font-size:16px;font-family:'Arial',sans-serif">Figure 4. </SPAN></STRONG><SPAN style="font-size:16px;font-family:'Arial',sans-serif">Simplifying model of the structure unit of class II hydrophobin film. Full line in the circle is valid hydrophobin in one unit and the hexagon is a cell of crystal monolayer film.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:16px;font-family:'Arial',sans-serif">To figure out the stoichiometric relationship between hydrophobin and surface site ,we regarded hexagonal cell as a unit. (Figure 4)The area of six effective hydrophobins  (S<SUB>H</SUB>)= </SPAN><SPAN style="font-size:16px;font-family:'Arial',sans-serif">, and the area of hexagonal cell (S<SUB>A</SUB>) = </SPAN><SPAN style="font-size:16px;font-family:'Arial',sans-serif">R.(R is the radius of ideal hydrophobin sphere)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The elementary reaction is as follow,</SPAN></P><TABLE align="center" cellpadding="0" cellspacing="0" width="340"><TBODY><TR class="firstRow" style=";height:18px"><TD style="border: 1px solid windowtext; padding: 0px 7px;" height="18" valign="top" width="83"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Parameters</SPAN></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="18" valign="top" width="257"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Meaning</SPAN></P></TD></TR><TR style=";height:18px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="18" valign="top" width="83"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">H</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="18" valign="top" width="257"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Hydrophobin(Janus)</SPAN></P></TD></TR><TR style=";height:18px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="18" valign="top" width="83"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">A<SUB>s</SUB></SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="18" valign="top" width="257"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Active   site on the solid surface</SPAN></P></TD></TR><TR style=";height:18px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="18" valign="top" width="83"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">k<SUB>1</SUB></SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" align="center" height="18" valign="middle" width="257"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Associated   constant</SPAN></P></TD></TR><TR style=";height:19px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="19" valign="top" width="83"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">k-<SUB>1</SUB></SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="19" valign="top" width="257"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Dissociated   constant</SPAN></P></TD></TR><TR style=";height:26px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="26" valign="top" width="83"><P style="text-align: center; text-indent: 0px; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">θ</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="26" valign="top" width="257"><P style="text-align: left; text-indent: 0px; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">fractional   occupancy of the adsorption sites</SPAN></P></TD></TR><TR style=";height:24px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="24" valign="top" width="83"><P style="text-align: center; text-indent: 0px; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">τ/τ<SPAN style="color:#333333;"><SUP>∞</SUP></SPAN></SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="24" valign="top" width="257"><P style="text-align: left; text-indent: 0px; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Surface   concentration of adsorption/ Maximal surface concentration</SPAN></P></TD></TR></TBODY></TABLE><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Hypothesis</SPAN><SPAN style="font-size:19px;font-family:宋体">：</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">(1) Each site can hold at most one molecule of Janus (mono-layer coverage only)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">(2) The surface containing the adsorbing sites is perfectly flat plane with no corrugations (assume the surface is homogeneous)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">(3) The adsorption process is reversible </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">(4) There is an interaction between adjacent Janus forming hexagonal crystalline structure.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">When adsorption rate equals to desorption rate,</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">simplify it an</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif">d solve θ</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif"> ,  </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The adsorption isotherm equation is,</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Parameter Finding</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">We turned to the literature to find parameters for our model, given in the Table below. In the past few years, M. Linder el at. were doing well in the field of adsorption of Janus. We looked for parameter values that had been measured in the work M. Linder el at. did, but the substrates they used were alkylated gold surface, salinized surface, polystyrene surface, etc. while what we use is PET(polyethylene terephthalate), which has different surface property. Therefore, we estimated parameters according to the data in their assays on fusion protein. Such aggregating parameters is somewhat uncertain endeavor; thereby predicting a range of the parameters and7 preparing it with the practical experiments. An explanation for how we arrived at each parameter is given in the table.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">ADSORPTION PARAMETERS</SPAN></STRONG></P><TABLE align="center" cellpadding="0" cellspacing="0"><TBODY><TR class="firstRow" style=";height:27px"><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; background: rgb(179, 179, 179) none repeat scroll 0% 0%; padding: 0px 7px;" height="27" width="155"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Parameter</SPAN></STRONG></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; background: rgb(179, 179, 179) none repeat scroll 0% 0%; padding: 0px 7px;" height="27" width="141"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Value</SPAN></STRONG></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; background: rgb(179, 179, 179) none repeat scroll 0% 0%; padding: 0px 7px;" height="27" width="124"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Description</SPAN></STRONG></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; background: rgb(179, 179, 179) none repeat scroll 0% 0%; padding: 0px 7px;" height="27" width="137"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Source/Rationale</SPAN></STRONG></P></TD></TR><TR><TD rowspan="2" style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" width="5"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">1</SPAN></STRONG></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px; word-break: break-all;" width="155"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">τ∞,</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif">HFBI-EGIc</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="141"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">0.28μmol•m-2(1.33μg•   cm-2)</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="124"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Maximal surface   concentration of sJanus-EGIc fusion protein</SPAN></P></TD><TD rowspan="2" style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="137"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Determined base on   fitting data into the first-order Langmuir isotherm, as reported in [7].It is   the binding isotherm of EGIc- sJanus fusion protein to silanized glass.</SPAN></P></TD></TR><TR><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px; word-break: break-all;" width="42"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">K<SUB>d</SUB>,HFBI-EGIc</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="201"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">21.8μg•ml-1</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="124"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Desorption   equilibrium constant of sJanus -EGIc fusion protein</SPAN></P></TD></TR><TR><TD rowspan="2" style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" width="5"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">2</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px; word-break: break-all;" width="155"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">T</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif">∞,HFBI</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="141"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">0.21μg• cm-2</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="124"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Maximal surface   concentration of sJanus</SPAN></P></TD><TD rowspan="2" style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="137"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Estimated according   to the mass relationship between sJanus and fusion protein sJanus -EGIc,   which is based on the assumption that the adsorbing and desorbing ability of sJanus   keep same as fusion protein. </SPAN></P></TD></TR><TR><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px; word-break: break-all;" width="42"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">K</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif">d,HFBI</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="201"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">21.8μg•ml-1</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" align="center" valign="middle" width="124"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Desorption   equilibrium constant of sJanus</SPAN></P></TD></TR><TR style=";height:24px"><TD rowspan="2" style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="24" width="5"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">3</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px; word-break: break-all;" height="24" valign="top" width="155"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SUB><SPAN style="font-size:19px;font-family:'Arial',sans-serif"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">T</SPAN>∞</SPAN></SUB><SPAN style="font-size:19px;font-family:'Arial',sans-serif">,HGFI</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="24" valign="top" width="141"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">0.224μg• cm-2</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="24" valign="top" width="124"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Maximal surface   concentration of sJanus</SPAN></P></TD><TD rowspan="2" style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="24" valign="top" width="137"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Estimated according   to an assay[8],with series of pH resulting in different adsorbance; thereby   there is a range of desorption constant</SPAN></P></TD></TR><TR style=";height:24px"><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px; word-break: break-all;" height="24" valign="top" width="42"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">K<SUB>d</SUB>,HGFI</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px; word-break: break-all;" height="24" valign="top" width="201"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">49~280</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif">μg•ml<SUP>-1</SUP></SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="24" valign="top" width="124"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Desorption   equilibrium constant of sJanus</SPAN></P></TD></TR></TBODY></TABLE><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">As an attachment, we read almost all the assays about sJanus and inJanus adsorption in the paper published and we found diverse conditions in every assay, such as different substrates: alkylated gold, polystyrene, silanized glass, 1-hexanethiol etc. Hence, it is not easy to estimate the certain parameters when the Janus self-assemble on the PET. NO.1 parameters in the table is in the condition where the fusion protein adsorbs on the silanized quartz surface attained from the paper and we estimate the parameters when the protein is pure Janus by ignoring the conformation change during the protein fusion. Besides, theoretically it only applies to silanized surface but we initially think it applies to all hydrophobic surfaces. Further, we used another assay[8] data to estimate the parameters when inJanus adsorb on PET although in this case they used polystyrene as the substrate.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Initial Model Result</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Using these estimated parameters, we simulated our model of the adsorption process. Below are two Janus, sJanus and inJanus, have different adsorbance with different concentration of free Janus. On the left we simulated a red line and a blue line, a range of the adsorption isotherm, which corresponds to pH=4 and pH=10, respectively. On the right is sJanus adsorption isotherm. It is convinced that no significant effect of temperature is noted; hence we do not make any note about temperature on the line graph.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">In the initial model, it is indicated that two classes of Janus have similar ability of self-assembly. They both have strong hydrophobic area stuck to hydrophobic substrate.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Result in Lab</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">We set up this model to predict the quantity of absorption wild Janus and fused Janus. We have done the PET hydrolysis experiment with Janus or fused Janus existing, however, we did not have enough time to accomplish further discussion and parameter fitting which we planned to study with QCM(Quartz crystal microbalance) and SPR(Surface plasmon resonance). Despite that, we are working to make it more clear and specific in the future.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Reference</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:16px;font-family:'Arial',sans-serif;color:black">[1]Wang, X., Graveland Bikker, J., Kruif, C. &amp; Robillard, G. Oligomerization of hydrophobin SC3 in solution: From soluble state to self-assembly. Protein Sci. 13, 810–821 (2004).</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:16px;font-family:'Arial',sans-serif;color:black">[2]Sunde, M., Kwan, A., Templeton, M., Beever, R. &amp; Mackay, J. Structural analysis of hydrophobins. <EM>Micron</EM> 39, 773–84 (2007).</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:16px;font-family:'Arial',sans-serif">[3]Géza R. Szilvay. Self-assembly of hydrophobin proteins from the fungus Trichoderma reesei.(2007)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:16px;font-family:'Arial',sans-serif;color:black">[4]Kisko, K. <EM>et al.</EM> Self-assembled films of hydrophobin protein HFBIII from Trichoderma reesei. <EM>J Appl Crystallogr</EM>(2007). doi:10.1107/S0021889807001331</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:16px;font-family:'Arial',sans-serif;color:black">[5]Kwan <EM>et al.</EM> Structural basis for rodlet assembly in fungal hydrophobins. <EM>Proc. Natl. </EM>Acad. Sci. U.S.A. 103, 3621–3626 (2006).</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:16px;font-family:'Arial',sans-serif">[6]Kwan, A. H. et al. The Cys3-Cys4 loop of the hydrophobin EAS is not required for rodlet formation and surface activity. J. Mol. Biol. 382, 708–20 (2008).</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:16px;font-family:'Arial',sans-serif;color:black">[7]Linder, M., Szilvay, G., Nakari Setälä, T.Söderlund, H. &amp; Penttilä, M. Surface adhesion of fusion proteins containing the hydrophobins HFBI and HFBII from Trichoderma reesei. Protein Science 11, 2257–2266 (2002).</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:16px;font-family:'Arial',sans-serif;color:black">[8]Wang, Z. et al. Hydrophilic modification of polystyrene with hydrophobin for time-resolved immunofluorometric assay. Biosens Bioelectron 26, 1074–9 (2010).</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:16px;font-family:'Arial',sans-serif;color:black">[9]Takatsuji, Y. et al. Solid-support immobilization of a ‘swing’ fusion protein for enhanced glucose oxidase catalytic activity. Colloids Surf B Biointerfaces 112, 186–91 (2013).</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:16px;font-family:'Arial',sans-serif;color:black">[10]Takahashi,T.et al.Ionic interaction of positive amino acid residues of fungal hydrophobin RolA with acidic amino acid residues of cutinase CutL1. Molecular Microbiology 96,14-27(2015)</SPAN></P></DIV><DIV id="content-in"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;font-size:25px;font-family:'Arial',sans-serif"><STRONG>PET  Degradation

    </STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">0. Abstract</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Modeling of PET degradation consists of three parts.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">First, a heterogeneous kinetic model is built to better describe the enzyme catalyzed hydrolysis of PET by combing the Michaelis-Menten equation with the Langmuir equation. Compared with the basic Michaelis-Menten model, we take a adsorption process into account noticing the PET is generally insoluble in the system (a heterogeneous system) and the enzyme should first combine with the enzymatic site then catalyze the hydrolysis. </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Second, we propose novel models to describe the effect of the application of Janus into the hydrolysis system based on the hypothetical elementary reactions using the result of self-assembling model and the novel kinetic model in the first part. There are three different ways to apply Janus-before the addition of the cutinase, mixed with the cutinase and fused with the cutinase, so we propose three different models with similar forms so that we can make a comparison easily. The model show that fusion of cutinase and Janus has the best catalyze efficiency.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Third, to visualize the result of our modelling, we wrote a program using MATLAB which simulate the degradation process of the hydrolysis. We use Stochastic Simulation to describe the collision of molecules and set different rates to represent the adsorption percentage. The program can used in different situations by adjustment of the parameters. The program can stimulate as well as predict and instruct the experiment well.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">In conclusion, we use math, physical and program models to describe, predict and instruct our project.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Here are the details of the modeling</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">1 Kinetic model</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">1.1 classic Michaelis–Menten model</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">In biochemistry, Michaelis–Menten kinetics is one of the best-known models of enzyme kinetics. The model takes the form of an equation describing the rate of enzymatic reactions, by relating reaction rate v to [S], the concentration of a substrate S. </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The reaction are assumed as</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Where E the enzyme, S substrate, ES the enzyme-substrate complex and P the reaction product. K1 k-1 k2 are all reaction constant </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">We assume that the reaction reaches an equilibrium state soon and the ES concentration do not change in the reaction</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">If the enzyme concentration is E0 when the reaction begins</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The rate of the reaction is </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Where </SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif"> called Michaelis-Menten constant.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Biochemical reactions involving a single substrate are often assumed to follow Michaelis-Menten kinetics, without regard to the model's underlying assumptions. However, Michaelis-Menten equation predicted results are quite different from the experimental results.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">1.2 heterogeneous </SPAN></STRONG><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">kinetic model</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Why is there such a phenomenon? After a literature review we know the reason. We use slices of PET in our experiment, which is seen insoluble. So the reaction system contains two phase, the solid PET, and the solution, which is called a heterogeneous system. So the reaction process is different from what we call a homogeneous system. We make an assumption that the enzyme Hydrolytic enzymes can only catalyze surface PET. So there are two process, first absorption and then catalyze</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">where</SPAN><SPAN style="font-size:19px;font-family: 'Arial',sans-serif">is the surface of the insoluble substrate. </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The equilibrium adsorption reaction involves a solid substrate and can be expressed as</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">where As is the substrate surface area, </SPAN><SPAN style="font-size:19px;font-family: 'Arial',sans-serif"> is the fraction of substrate surface occupied by the ES complex, (1-</SPAN><SPAN style="font-size:19px;font-family: 'Arial',sans-serif">) is the “free” surface fraction, and E0 is the initial enzyme concentration. </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The theta parameter is derived:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">where K= k1/k-1 is the adsorption equilibrium constant. </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">For the hydrolysis reaction, the proposed model gives the following rate equation:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Whose linear form is </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">It can be used to easily derive adsorption (K) and hydrolysis rate (k<SUB>2</SUB>) constants from the slope and intercept of the straight line fit to the experimental rate results.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">In fact, there is another model <SUP>[1]</SUP> .It was assumed that the adsorption of the enzyme onto the surface of the substrate obeyed a Freundlich isotherm of the form</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Where K is the Freundlich adsorption constant. For adsorption of enzyme from a solution phase onto a two-dimensional phase, n was predicted to be 2/3. Because the rate of the degradation reaction is directly proportional to [EAs] the velocity law can be written as</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Here k’ is the velocity constant.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The first model actually uses the same derivation process as Langmuir adsorption model. Inherent within this model, the following assumptions are valid specifically for the simplest case: the adsorption of a single adsorbate onto a series of equivalent sites on the surface of the solid.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">1. The surface containing the adsorbing sites is perfectly flat plane with no corrugations (assume the surface is homogeneous).</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">2. The adsorbing gas adsorbs into an immobile state.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">3. All sites are equivalent.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">4. Each site can hold at most one molecule of A (mono-layer coverage only).</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">5. There are no interactions between adsorbate molecules on adjacent sites.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The Langmuir adsorption model is widely used in all kinds of adsorption process.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The second model uses Freundlich adsorption model. As this relationship is entirely empirical, in the case where adsorption behavior can be properly fit by isotherms with a theoretical basis, it is usually appropriate to use such isotherms instead (see for example the Langmuir and BET adsorption theories). So in our experiment we choose the first model.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">1.3 parameters finding and model fitting</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">We turned to the literature to find parameters for our model, given in the Table below.</SPAN></P><TABLE align="center" height="307" cellpadding="0" cellspacing="0" width="649"><TBODY><TR class="firstRow" style=";height:27px"><TD style="border: 1px solid windowtext; background: rgb(179, 179, 179) none repeat scroll 0% 0%; padding: 0px 7px; word-break: break-all;" height="27" width="66"><P style="text-align: center; line-height: 1.5em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">enzyme</SPAN></STRONG></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; background: rgb(179, 179, 179) none repeat scroll 0% 0%; padding: 0px 7px;" height="27" width="101"><P style="text-align: center; line-height: 1.5em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Parameter</SPAN></STRONG></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; background: rgb(179, 179, 179) none repeat scroll 0% 0%; padding: 0px 7px;" height="27" width="141"><P style="text-align: center; line-height: 1.5em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Value</SPAN></STRONG></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; background: rgb(179, 179, 179) none repeat scroll 0% 0%; padding: 0px 7px;" height="27" width="124"><P style="text-align: center; line-height: 1.5em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Description</SPAN></STRONG></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; background: rgb(179, 179, 179) none repeat scroll 0% 0%; padding: 0px 7px;" height="27" width="137"><P style="text-align: center; line-height: 1.5em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Source/Rationale</SPAN></STRONG></P></TD></TR><TR style=";height:11px"><TD rowspan="3" style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="11" width="66"><P style="text-align: center; line-height: 1.5em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">FsC</SPAN></STRONG></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="11" width="101"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">[s0]</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="11" width="141"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">13miu M</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="11" width="124"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Surface concentration</SPAN></P></TD><TD rowspan="3" style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="11" width="137"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Determined   for PET hydrolysis at 40 degree<SUP>[2]</SUP></SPAN></P></TD></TR><TR style=";height:11px"><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="11" width="101"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">K</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px; word-break: break-all;" height="11" width="141"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">0.41miu M <SUP>-1</SUP></SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="11" width="124"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Adsorption   constant</SPAN></P></TD></TR><TR style=";height:18px"><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="18" width="101"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">k2</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="18" width="141"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">0.09miu   mol/cm2/h</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="18" width="124"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Reaction rate   constant</SPAN></P></TD></TR><TR><TD rowspan="3" style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" width="66"><P style="text-align: center; line-height: 1.5em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Thc-Cut1</SPAN></STRONG></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="101"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">KM</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="141"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">1.5+-0.1mM</SPAN></P></TD><TD rowspan="3" style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="124"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Michaelis-Menten   constant</SPAN></P></TD><TD rowspan="3" style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="137"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Determined   for PNPA(soluble)</SPAN></P><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">hydrolysis  <SUP>[3]</SUP></SPAN></P></TD></TR><TR><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="101"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Kcat</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="141"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">436s<SUP>-1</SUP></SPAN></P></TD></TR><TR><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="101"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">KM/Kcat</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" width="141"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">291</SPAN></P></TD></TR><TR style=";height:42px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="42" width="66"><P style="text-align: center; line-height: 1.5em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">LC</SPAN></STRONG></P></TD><TD colspan="4" style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px; word-break: break-all;" height="42" width="502"><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The amount   of PET degraded by LC-cutinase at pH 8.0 and 50°C was1.45 mg after incubation   for 24 h.<SUP>[4]</SUP></SPAN></P></TD></TR></TBODY></TABLE><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">In our experiment, we use three kinds of cutinase, unfortunately we can find few data fit for our model.(cutinase catalyzed hydrolysis of PET is a new area and there are not so many experiment carried out on it/ many experimenter just uses traditional M-M equation to fit their experiment data )</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Parameters of FsC is well-documented and can be used in our model</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Here are the result of our model for FsC</SPAN></P><P style="text-align: center; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">2 Janus applying model</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">2.1 apply Janus before the addition of the cutinase</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">First, we add hydrophobin(Janus) into the PET system, and the reaction is </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">With H the hydrophobin, As the surface of the PET</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">And the hydrophobin will self-assemble on the surface of hydrophobin</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">From <SPAN style="color:red">the self-assembling model</SPAN> we can know that different kinds of hydrophobin has different kinds of equations </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Take class II hydrophobin as an example,</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The surface concentration of hydrophobin is </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">With </SPAN><SPAN style="font-size:19px;font-family: 'Arial',sans-serif"> the max surface concentration of absorbed hydrophobin</SPAN><SPAN style="font-size:19px;font-family:宋体">，</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif">kd the dissociation constant</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Then we add cutinase into the system , the reaction can be concluded as</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">the cutinase will have some ionic interaction with hydrophobin and the reaction is stronger than that PET do<SUP>[5]</SUP></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">so we assume that most enzyme will be absorbed by hydrophobin, and then react with PET </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">in the equilibrium state, the concentration of </SPAN><SPAN style="font-size:19px;font-family: 'Arial',sans-serif"> is steady, which means</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">if the initial concentration of cutinase is E0</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">then</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Where the K=K-1+K2/K1</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">so the rate equation is </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">in fact , the ratiocination process is similar to that of Michaelis–Menten equation</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">for class one hydrophobin, the rate equation is</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">for class two hydrophobin, the rate equation is</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">2.2 hydrophobin fused with cutinase</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">First, we fuse hydrophobin with cutinase and we get (HE) .we will use X to refer to it</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Then, we add it into the system, the reaction process is concluded as follows</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Here we can see x as a new enzyme, which means it has the same function as cutinase but with different characteristics</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Unlike the first model, here we use the heterogeneous model to describe the process</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">That is because PET is a solid phase without a concentration </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Where k’=k1’/k-1’</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">2.3 parameters discussion and model fitting</SPAN></P><TABLE align="center" height="486" cellpadding="0" cellspacing="0" width="681"><TBODY><TR class="firstRow" style=";height:14px"><TD style="border: 1px solid windowtext; background: rgb(174, 170, 170) none repeat scroll 0% 0%; padding: 0px 7px;" height="14" valign="top" width="75"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">parameter</SPAN></STRONG></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; background: rgb(174, 170, 170) none repeat scroll 0% 0%; padding: 0px 7px;" height="14" valign="top" width="388"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">discussion</SPAN></STRONG></P></TD></TR><TR style=";height:80px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="80" valign="top" width="75"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">K<SUB>1</SUB>  k<SUB>-1 </SUB>K</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="80" valign="top" width="388"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">hydrophobin   self-assemble on the surface of the PET and cutinase will adsorb on the   surface more easily.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">So   k1 will be greater and k-1 will be smaller than those of non-hydrophobin   system. K of a non-hydrophobin system is 0.41miu M-1( Fsc) and K here will be   greater than that</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The   kd between CutL-1 and hydrophobin RolA is 6.5nM[5]</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif"> </SPAN></P></TD></TR><TR style=";height:77px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="77" valign="top" width="75"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">K<SUB>2</SUB>  k<SUB>2</SUB>'</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="77" valign="top" width="388"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Since   the core reaction is the same—cutinase catalyzed hydrolysis of PET , so we   think the k value doesn’t change</SPAN></P></TD></TR><TR style=";height:80px"><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" height="80" valign="top" width="75"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">K'</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" height="80" valign="top" width="388"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">It   is a main reflection of the characteristic of hydrophobin, so we think k is   the same with that of hydrophobin</SPAN></P></TD></TR></TBODY></TABLE><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px;font-family:'Arial',sans-serif">                                                                       </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">3. Visualization and programming</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">3.1 description</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">To make our result more direct. We use MATLAB to visualize it.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">The main idea is to simulate the degradation process of the hydrolysis. To simplify our model, we use a two-dimensional plane with certain area to stimulate PET surface. Besides, we look enzyme and Janus as a point since they are so small compared to the size of PET. In our program, the adsorption is thought to be collision. We use Stochastic Simulation to describe the collision of molecules and set different rates to represent the adsorption percentage. By introducing time dimension we can predict the degradation percentage with plots and pictures.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Here are some basic assumptions about our model</SPAN></P><P style="margin-left: 24px; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">1.<SPAN style="font:9px 'Times New Roman'">  </SPAN>The PET surface is treated as a 2D grid (of size n x n)</SPAN></P><P style="margin-left: 24px; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">2.<SPAN style="font:9px 'Times New Roman'">  </SPAN>Each grid may be occupied by at most one cutinase or Janus</SPAN></P><P style="margin-left: 24px; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">3.<SPAN style="font:9px 'Times New Roman'">  </SPAN>Randomly generating grids means collision of molecules, a ratio will determine how many percentage of these grids will be adsorbed</SPAN></P><P style="margin-left: 24px; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">4.<SPAN style="font:9px 'Times New Roman'">  </SPAN>If cutinase is added after Janus, there will be another ratio determine how many cutinase will be absorbed by Janus</SPAN></P><P style="margin-left: 24px; line-height: 1.5em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">5.<SPAN style="font:9px 'Times New Roman'">  </SPAN>If cutinase is fused with Janus</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">What’s more, the program allow users set different rates including the adsorption rate, the reaction rate and the lifetime of enzyme so the program will have a wider application range. The program can be used in different situations by adjustment of the parameters. The program can stimulate as well as predict and instruct the experiment well.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">3.2 code</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">Here are some basic function in our program written by Tsingua-A iGEM.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif">They have written detailed note to make our program readable.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:blue">function</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black"> dots_e=rdm(n,number)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% rdm produces binary coordinate randomly</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% (number,n),where number is the number of coordinates</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% you want and n is the n-by-n square surface</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% Calling sequence:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%        array=rdm(number)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% Define variables:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% number    --Number of the dots that is absorbed with cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% n         --N-by-n surface</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% dots_e    --The chosen random points that will react with cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">disp(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'The coordinates of degradable points on the simulated surface is as follow'</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black">)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">dots_e=randi(n,number,2);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:blue">function</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black"> combine_dots_bycut=filter_dots(number_h,number_h_cut,dots_h)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% Filter_dots is to filtate a certain array into smaller one randomly,but</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% with a certain filtration ratio.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% Calling sequence:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%        new_array=combin_cut(number)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% Define variables:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% number_h        ---The number of locuses adsorbed by hydrophobins</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% number_h_cut    ---The number of locuses adsorbed by hydrophobins and</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%                   cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% ratio_h_bind_e  ---The ratio that enzyme binds to the hydrophobin of all</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% dots_h          ---The coordinates of locuses adsorbed by hydrophobins</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% i               ---The number of locuses adsorbed by hydrophobins and</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%                   cutinase after filtration</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% temp_number     ---The number array generated randomly in the range of number_h</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">temp_number=randperm(number_h,number_h_cut);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">i=1;j=1;</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:blue">for</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black"> i=1:length(temp_number)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">   </SPAN><SPAN style="font-size: 19px;font-family:'Arial',sans-serif;color:blue">for</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black"> j=1:2</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">combine_dots_bycut(i,j)=dots_h(temp_number(i),j);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">   </SPAN><SPAN style="font-size: 19px;font-family:'Arial',sans-serif;color:blue">end</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:blue">end</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;">And here are our main program:
</P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:blue">function</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black"> start_no_h</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% Start_no_h is to guide customer to generate a random simulation</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% We simulate a n-by-n surface with which the ideal enzyme particles collide.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% This process simulates the behavior of the adsorption of the PET cuticase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% on the PET surface.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% By setting the parameters (the size of surface,the ratio of enzymes </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% collision and probabability of chemical reaction)we get the number of </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% degradable locuses and each locus's coordinates.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% Calling sequence</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%               start_no_h</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% Define variables:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% n              --The n-by-n active locus surface</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% ratio_e_pet    --The ratio that cutinase binds to locus on pet</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% number         --The number of locuses reacting with cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% dots_e         --The coordinates of points able to react with cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">n=input(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'Please input a n-by-n square surface(n=?):'</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black">);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">ratio_e_pet=input(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'Please set the ratio of locus binded by cutinase on surface:([0,1])'</SPAN><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">number=n*ratio_e_pet;</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">dots_e=rdm(n,number)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:blue">function</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black"> start_h_e</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% Start_h_e is to guide customer to generate a random simulation</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% We simulate a n-by-n surface with which the ideal hydrophobin particles</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% collide and cutinase bind hydrophobin then degrade PET.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% This process simulates the behavior of the adsorption of the hydrophobin</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% on the PET surface and cutinase interact with hydrophobin anchoring on</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% the PETsurface and degrading it.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% By setting the parameters (the size of surface,the ratio of hydrophobins </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% collision, ratio of cutinase effectively anchored on hydrophobin and </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% probabability of chemical reaction).We get the number of degradable </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% locuses and each locus's coordinates</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% Calling sequence:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%        start_h_e</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% Define variables:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% n                  --The n-by-n active locus surface</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% ratio_h_pet        --The ratio of hydrophobin adsorbing on pet</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% ratio_h_cut        --The ratio of cutinase binding on the hydrophobin when</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%                       collision occurs</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% ratio_degrade_pet  --The ratio of effective degration reaction happens when</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%                     cutinase has bound on the hydrophobin</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% number_h           --The number of locuses absorbed by hydrophobin</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% number_h_cut       --the number of locuses absorbed by hydrophobin bound</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%                     with cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% number_degrade     --The number of locuses degraded by cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% dots_e             --The coordinates of points able to react with cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% combine_dots_btcut --The coordinate of the locuses adsorbed hydrophobin</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%                       and binded cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% degrade_dots_bycut --The coordinate of the locuses degraded by cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">n=input(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'Please input a n-by-n square surface(n=?):'</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black">);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">ratio_h_pet=input(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'Please set the ratio of locus binded by hydrophobin on surface:([0,1])'</SPAN><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">ratio_h_cut=input(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'Please set the ratio of combination  hydrophobin with cutinase:([0,1])'</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black">);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">ratio_degrade_pet=input(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'Please set the ratio of locus degraded by cutinase:([0,1])'</SPAN><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">number_h=n*ratio_h_pet;</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">number_h_cut=number_h*ratio_h_cut;</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">number_degrade=number_h_cut*ratio_degrade_pet;</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">dots_h=rdm(n,number_h);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">combine_dots_bycut=filter_dots(number_h,number_h_cut,dots_h);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">degrade_dots_bycut=filter_dots(number_h_cut,number_degrade,combine_dots_bycut);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">fprintf(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'The number of locuses degraded by cutinase is %1d\n'</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black">,number_degrade);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">disp(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'The coordinates of degraded locuses is as follow'</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black">);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:blue">for</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black"> i=1:number_degrade</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">   </SPAN><SPAN style="font-size: 19px;font-family:'Arial',sans-serif;color:blue">for</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black"> j=1:2</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">        fprintf(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'%2d '</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black">,degrade_dots_bycut(i,j));</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">   </SPAN><SPAN style="font-size: 19px;font-family:'Arial',sans-serif;color:blue">end</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">    fprintf(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'\n'</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black">);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:blue">end</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:blue">function</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black"> start_h_fused_e</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% Start_h_e is to guide customer to generate a random simulation.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% We simulate a n-by-n surface to which the ideal hydrophobin particles are</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% absorbed.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% This process simulates the behavior of the absorption of the hydrophobins</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% fused with cutinase on the PET surface.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% By setting the parameters (the size of surface,the ratio of enzymes </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% collision and probabability of chemical reaction)we get the number of </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% degradable locuses and each locus's coordinates.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% Calling sequence</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%               start_h_fused_e</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% Define variables:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% n                  --The n-by-n active locus surface</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% ratio_e_pet        --The ratio that cutinase binds to locus on pet</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% ratio_degrade_pet  --The ratio that cutinase degrades the pet</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% number_degrade     --The number of locuses degraded by cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% number_h           --The number of locuses adsorbed by hydrophobins fused with</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%                       cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% dots_h_fused_e     --The coordinates of locuses adsorbed by hydrpophobin </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%                        fused with cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">% degrade_dots_bycut --The coordinates of locuses degraded by cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:forestgreen">%To prompt client to input parameters</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">n=input(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'Please input a n-by-n square surface(n=?):'</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black">); </SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:forestgreen">% Length of side of n*n square</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">ratio_h_pet=input(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'Please set the ratio of locus adsorbed by hydrophobin on surface:([0,1])'</SPAN><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">ratio_degrade_pet=input(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'Please set the ratio of locus degraded by cutinase:([0,1])'</SPAN><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">number_h=n*ratio_h_pet;</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">number_degrade=number_h*ratio_degrade_pet;</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">dots_h_fused_e=rdm(n,number_h);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">degrade_dots_bycut=filter_dots(number_h,number_degrade,dots_h_fused_e);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">fprintf(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'The number of locuses degraded by cutinase is %1d\n'</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black">,number_degrade);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">disp(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'The coordinates of degraded locuses is as follow'</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black">);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:blue">for</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black"> i=1:number_degrade</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">   </SPAN><SPAN style="font-size: 19px;font-family:'Arial',sans-serif;color:blue">for</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black"> j=1:2</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">        fprintf(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'%2d '</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black">,degrade_dots_bycut(i,j));</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">   </SPAN><SPAN style="font-size: 19px;font-family:'Arial',sans-serif;color:blue">end</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:black">    fprintf(</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:#A020F0">'\n'</SPAN><SPAN style="font-size:19px;font-family:'Arial',sans-serif;color:black">);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif;color:blue">end</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family: 'Arial',sans-serif"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size:19px;font-family:'Arial',sans-serif"> </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><STRONG><SPAN style="font-size:19px;font-family:'Arial',sans-serif">References</SPAN></STRONG></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: &quot;Arial&quot;,sans-serif; font-size: 16px;">[1] McLaren, A. D. Enzyme reactions in structurally restricted systems. IV. The digestion of insoluble substrates by hydrolytic enzymes. Enzimologia 26, 237.(1963)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: &quot;Arial&quot;,sans-serif; font-size: 16px;">[2] A ˚ sa M. Ronkvist, Wenchun Xie, Wenhua Lu, and Richard A. Gross Cutinase-Catalyzed Hydrolysis of Poly(ethylene terephthalate). Macromolecules 42, 51285138 (2009).</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: &quot;Arial&quot;,sans-serif; font-size: 16px;">[3] Acero, E. et al. Surface engineering of a cutinase from Thermobifida cellulosilytica for improved polyester hydrolysis. Biotechnol. Bioeng. 110, 2581–2590 (2013).</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: &quot;Arial&quot;,sans-serif; font-size: 16px;">[4] Sulaiman, S. et al. Isolation of a novel cutinase homolog with polyethylene terephthalate-degrading activity from leaf-branch compost by using a metagenomic approach. Appl. Environ. Microbiol. 78, 1556–62 (2011).</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: &quot;Arial&quot;,sans-serif; font-size: 16px;">[5] Toru Takahashi. et al. Ionic interaction of positive amino acid residues of fungal hydrophobin RolA with acidic amino acid residues of cutinase CutL1. Molecular Microbiology. 96(1), 14–27(2015)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: &quot;Arial&quot;,sans-serif; font-size: 16px;">[6] Figeroa, Y., Hinks, D. &amp; Montero, G. A Heterogeneous Kinetic Model for the Cutinase-Catalyzed Hydrolysis of Cyclo-tris-ethylene Terephthalate. Biotechnol. Prog. 22, 1209–1214 (2006)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: &quot;Arial&quot;,sans-serif; font-size: 16px;">[7] Wang, Z., Lienemann, M., Qiau, M. &amp; Linder, M. Mechanisms of protein adhesion on surface films of hydrophobin. Langmuir 26, 8491–6 (2010).</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: &quot;Arial&quot;,sans-serif; font-size: 16px;">[8] <SPAN style="font-family: &quot;Arial&quot;,sans-serif; font-size: 16px; color: black;">Mariastella Scandola, Maria Letizia Focarete, and Giovanna FrisoniSimple. Kinetic Model for the Heterogeneous Enzymatic Hydrolysis of Natural Poly(3-hydroxybutyrate).</SPAN><SPAN style="font-family: &quot;Arial&quot;,sans-serif; font-size: 16px; color: black;">Macromolecules31, 3846-3851(1998)</SPAN></SPAN></P></DIV><DIV id="content-in"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 25px; line-height: 150%;">Protein Extraction</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">Introduction</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">The purpose of this part of modelling is trying to found out optimize process conditions for the effective partitioning of our fusion protein and a way to proof whether or not the novel two-step extraction method is applicable to any other kinds of fusion proteins constructed by other target protein of interest and hydrophobin. Moreover, we’d like to put a mighty-possible way to find the feasible experimental condition for any other target protein.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">Part 1. Optimize process conditions modelling</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">Given the fact that our independent variables might to be concerned (Triton molecular weights and concentrations, temperature, phase volume ratios R, pH, fusion protein’s surface hydrophobicity, charge, structure, and molecular weight) do have a lot interaction between each other, we can see it that a high level of non-linearity exists in our system, complicating the study of the behavior in the aqueous two-phase extraction. What’s more, the common method of experimental design and optimization where operating conditions are optimized by changing one variable at a time, gives no guarantee for the optimal parameter determination. In order to simplify our problem, we decided to use the response surface methodology (RSM) in our model, which is based on the statistical theory. In this case, our model is less theoretical, nonetheless effective in our situation.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">Following we present you our optimization model. </SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">1.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN></SPAN></STRONG><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">Response surface methodology (RSM)</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">RSM consists of a group of mathematical and statistical techniques for seeking the optimum conditions in a system which had multi-variable and complex interactions existing between the parameters. It can be used to define the relationships between the response variables (in our project are: recovery percentage and the purity of protein) and the independent variables (in our project are: the molecular weight</SPAN><SPAN style="font-size: 19px; line-height: 150%; font-family: 宋体;">、</SPAN><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">the concentration and the Grand average of hydropathicity of fusion protein (GRAVY) and phase volume ratios R) and also generates a mathematical model that describes the process.</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">The formula below is the basic equation used in RSM, in which x<SUB>i</SUB>x<SUB>j</SUB> represents the interaction between two parameters, β is the equation coefficients. As for why we choose x<SUB>i</SUB>, x<SUB>i</SUB><SUP>2</SUP>, x<SUB>i</SUB>x<SUB>j</SUB> to regress? Here is the reason: take Taylor's formula as an instance, when the second-order can meet the accuracy requirements, than we shall set the polynomial as a second-order one, cause third-order is not necessary. The same reason, the following second-order model can meet our requirements so we do not set it as a third-order one.</SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">2.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN></SPAN></STRONG><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">Experiment data</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">To find out the statistical results of the model, we designed several experiments to gain our statistics by using the central composite rotatable design, considering four independent variables: the molecular weight</SPAN><SPAN style="font-size: 19px; line-height: 150%; font-family: 宋体;">、</SPAN><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">the concentration and the Grand average of hydropathicity of fusion protein and phase volume ratios R. The range of variables was chosen according to pre-experiment, during which we studied respective individual parameter. After the experiments were carried out, a regression analysis was carried out to extract equation coefficient in the model by using the software SPSS (Statistical Product and Service Solutions).</SPAN></SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">3.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN></SPAN></STRONG><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">Supplementary Analysis on the Model</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">Principal Component Analysis (PCA)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">In order to simplify our model, PCA is used to reduce the variables. Principal component analysis (PCA) is a statistical procedure that uses an orthogonal transformation to convert a set of observations of possibly correlated variables into a set of values of linearly uncorrelated variables called principal components. The number of principal components is less than or equal to the number of original variables. After the transformation is done, we minimize the variables and carry on the work.</SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">4.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN></SPAN></STRONG><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">Data Analysis and Model Validation</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">Due to the fact that by the time we submit the wiki, our experiments are still undergoing, so we can only process the few data we already got. Thanks to the useful software SPSS, which is a tiny but powerful tool speaking of statistical problems. We solved the basic statistical analysis by SPSS, here’s the results.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">Another thing to mention is that we define a response variable as purity ratio, which equals the purity of target protein after extraction divide by purity of target protein before extraction.</SPAN></P><TABLE align="center" cellpadding="0" cellspacing="0"><TBODY><TR class="firstRow"><TD style="border: 1px solid windowtext; padding: 0px 7px;" valign="top" width="66"><P style="line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">protein</SPAN></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" valign="top" width="94"><P style="line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">GRAVY(x<SUB>1</SUB>)</SPAN></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" valign="top" width="109"><P style="line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">Phase ratio(x<SUB>2</SUB>)</SPAN></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" valign="top" width="101"><P style="line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">Purity before</SPAN></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" valign="top" width="83"><P style="line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">Purity after</SPAN></P></TD><TD style="border-width: 1px 1px 1px medium; border-style: solid solid solid none; border-color: windowtext windowtext windowtext -moz-use-text-color; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" valign="top" width="100"><P style="line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">Purity ratio(y)</SPAN></P></TD></TR><TR><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" valign="top" width="66"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">GFP</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="94"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">-0.338</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="109"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">0.02</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="101"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">59.8</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="83"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">88.7</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="100"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">1.483277592</SPAN></P></TD></TR><TR><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" valign="top" width="66"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">GFP</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px; word-break: break-all;" valign="top" width="94"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">-0.338</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="109"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">0.05</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="101"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">94.5</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="83"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">100</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="100"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">1.058201058</SPAN></P></TD></TR><TR><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" valign="top" width="66"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">GFP</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="94"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">-0.338</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="109"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">0.05</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" align="center" valign="middle" width="101"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">86.7</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="83"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">90.1</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="100"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">1.039215686</SPAN></P></TD></TR><TR><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" valign="top" width="66"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">GFP</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="94"><P style="text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">    -0.338</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="109"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">0.05</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="101"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">26.4</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="83"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">74.6</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="100"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">2.825757576</SPAN></P></TD></TR><TR><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" valign="top" width="66"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">BFP</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="94"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">-0.303</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="109"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">0.20</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="101"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">27.5</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="83"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">69</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="100"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">2.509090909</SPAN></P></TD></TR><TR><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" valign="top" width="66"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">RFP</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="94"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">-0.563</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="109"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">0.02</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="101"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">19.5</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="83"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">78.6</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="100"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">4.030769231</SPAN></P></TD></TR><TR><TD style="border-width: medium 1px 1px; border-style: none solid solid; border-color: -moz-use-text-color windowtext windowtext; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; padding: 0px 7px;" valign="top" width="66"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">RFP</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="94"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">-0.563</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="109"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">0.05</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="101"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">19.5</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="83"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">60.9</SPAN></P></TD><TD style="border-width: medium 1px 1px medium; border-style: none solid solid none; border-color: -moz-use-text-color windowtext windowtext -moz-use-text-color; padding: 0px 7px;" valign="top" width="100"><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">3.123076923</SPAN></P></TD></TR></TBODY></TABLE><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">The experiment data</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">And , and has the confident of 73.0%</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">And here are some figures:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: center; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">These three figures illustrate the purify ratio from different point of view.</SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">5.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN></SPAN></STRONG><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">Sensitivity Analysis of the Model</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">To examine the influence on the response variables, we write the sensitivity analysis’s program, however, we do not get enough data so we did not do this part right now.</SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">6.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN></SPAN></STRONG><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">Conclusions</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">We can see that when we do the regression, the hydropathicity (x<SUB>1</SUB>) has been neglect, we may say that the target protein may be of any kind because the hydropathicity is not taken into account (Notice that here I only use the word “may”).</SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">7.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN></SPAN></STRONG><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">Strengths and Weaknesses</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">Strengths</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">Get rid of the complexity of the cases and draw a convincing conclusion despite of the complicate situation.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">The result is applied to all at a wide confidence interval and easy to get once you have limited statistics.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">Above all, the model is easy to establish and be used during research.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">Weaknesses</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">The results can only be reached after you have some basic experiments (statistics), and cannot be made if you have nothing in hand. In other word, the model can only be used to forecast by practically rather than theoretically.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">The burden of the computer increases as the data increases rapidly, in this case the model may not be suitable for occasions where there are too many variables (by saying too many, we mean more than 10 or so).</SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">8.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN></SPAN></STRONG><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">Where to improve</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">Take our project into consideration, it is still a problem that we do not have enough time to conduct so many experiments, there’s two way to solve the problem: the first is that although it is the best to conduct 31 experiments designed above, but consider there is only 15 coefficients for us to determined which gives the equations freedom, it is ok if we reduce the experiments down to 20 or so. However if you still think that is too much, then we can only solve the problem through the second method, that is we should find another way to modelling, maybe a more theoretical way that we do not have to depend the data too much.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">Part 2. Feasible experimental condition modelling</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif; color: black;">Raise questions</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">In the process of modeling, we use the regression method to find the best experimental conditions. In the gradient experiment, the experimental conditions of the three kinds of proteins were obtained, which were 60, 70 and 80, take the best effect as 100 for example. Then we used the experimental data to do RSM analysis. The experimental conditions were independent variables, the extraction effect was variable. After the analysis the regression equation was obtained, and by using the value of derivative is zero we can get the best experimental conditions when the maximum effect value is 100 points.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">And then, using the experimental data we have, what can we do if we want to use from three kinds of protein, with its separate properties (such as molecular weight and isoelectric point, hydrophilic and hydrophobic) as decision variables, to predict the acceptable conditions of any other proteins (regard the effect greater than 70 points as acceptable), whose properties were known, and how to generalize?</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif; color: black;">Analysis problem</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">The difficulty of the problem lies in, because the mechanism of extraction system is unclear, so the establishment of the mechanism model is difficult to achieve; and return to the regression which can get accurate results without knowing the mechanism well, it has always been a large sum of data to do regression analysis to predict, never have I heard small number of data can do regression to predict well. First of all, with little data may get no results (thinking that when the number of coefficients is a lot, then the data need more too to determine the coefficients’ value), not to mention the accuracy, which makes regression become meaningless. If you really have to make predictions using less data, you can consider interpolation method.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif; color: black;">Interpolation and regression</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">Both interpolation and regression are widely used for prediction and forecasting, however, there are some differences between them. Normally there is a regression function defined in terms of a finite number of unknown parameters that are estimated from the data on the basis of minimize the sum of residuals, so the regression function do not have to fit the data exactly. The effect of regression often depends to some extent on making assumptions about the process, what’s more, these assumptions are required to have a sufficient quantity of data to observe, otherwise regression can easily give misleading results. Interpolation is more used for a small number of numerical processing, through the known data points, to construct new data points or a resulting function smoothly fit within the range of a discrete set of known data points to perform the function of prediction and forecasting. In the process of interpolation, we do not need to know what the function is like, complicated or simple, whatever, we only need to do some calculation. Errors exist as well as the regression analysis, however, considering the situation where we do not have much data to use, and the resulting function is fitted by the data, we may say that interpolation is a better way compared to regression. And the same thing between interpolation and regression is that, because the dimension of our prediction function depends on the number of data points, so with small sum of data points, the dimension of our prediction function is not high, the accuracy may not be too high.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">The problem of few data has temporarily been solved by choosing the interpolation instead of the regression analysis, and the selection of the interpolation also lead to new problems. First of all, we deal with the first-order interpolation more often, but considering the characteristics of the protein data, there are three variables, and therefore we have to use multidimensional interpolation; second, there are more than one interpolation method, which is the best choice? The answer to this question is: we don’t know either. There is no conclusion or rule for interpolation method we use, we can only say that, try as much method as possible, then you may find the best answer.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif; color: black;">Problem solving</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">As we discussed before, the selection of interpolation method is varied, so we start with the most simple one, multivariate polynomial interpolation in the Lagrange type interpolation (the most simple one should be multivariate linear interpolation, but taking the accuracy into account we start from polynomial interpolation rather than linear one). Choose the selected protein’s molecular weight, isoelectric point and hydrophilic as three independent variables, and experimental conditions as dependent variables for multiple times interpolation. Method is as follows:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif; color: black;">Three-dimensional space Lagrange interpolation formula</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">Suppose a rectangular</SPAN><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">, </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif; color: black;">Given interval</SPAN><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;"> [a, b] <SPAN style="font-family: arial,helvetica,sans-serif; color: black;">of a division</SPAN>;</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif; color: black;">Given interval </SPAN><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">[c, d] <SPAN style="font-family: arial,helvetica,sans-serif; color: black;">of a division</SPAN>;</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif; color: black;">Given interval</SPAN><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;"> [s, t] <SPAN style="font-family: arial,helvetica,sans-serif; color: black;">of a division</SPAN> ;</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">Then constitute a division point set of T.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif; color: black;">Set the function</SPAN><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">, </SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">Where equals the value at each point of the division point set, and</SPAN><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;"> <SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif; color: black;">is a polynomial of n-ordered x, m-ordered y, and l-ordered z.</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">Let be</SPAN><SPAN style="color: black; font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">, then it is easy to know that:</SPAN><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">,</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">And is the Lagrange interpolation polynomial for the division point set.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">And the Lagrange interpolation basis function should meet the requirements below:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">,</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">,</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">,</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">Let be</SPAN><SPAN style="color: black; font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">, based on the Lagrange interpolation basis function of simple function we can know:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif; color: black;">So, the interpolation polynomial of</SPAN><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;"><SPAN style="font-family: arial,helvetica,sans-serif; color: black;">on the division point set</SPAN> ∆ <SPAN style="font-family: arial,helvetica,sans-serif; color: black;">is:</SPAN></SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; color: black; font-family: arial,helvetica,sans-serif;">In our project, we just have to put the experimental data into the formula, and the last work is to calculate.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; line-height: 150%; font-family: &quot;Arial&quot;,sans-serif;">Appendix</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">Here is the program we used in modelling.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; line-height: 150%; font-family: arial,helvetica,sans-serif;">No.1</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">clc</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">clear</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">load </SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">pz.txt</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">%pz.txt is the text where you store your data,put the independent variables</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">%on the left rows,and the responce variables on the right rows.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">n=4;m=2;</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">%n is the number of the independent variables </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">%m is the number of the responce variables</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">x0=pz(:,1:n);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">y0=pz(:,n+1:end);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">x=x0(:,3);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">y=x0(:,4);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">z=y0(:,1);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">[X,Y]=meshgrid(min(x):0.01:max(x),min(y):0.001:max(y));</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">Z=griddata(x,y,z,X,Y,</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">'v4'</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">);</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">figure(1)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">[c,h]=contour3(X,Y,Z);hold </SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">on</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">colorbar</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">hold </SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">off</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">meshc(X,Y,Z);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">figure(2)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">pcolor(X,Y,Z);shading </SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">interp</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">colorbar</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">figure(3)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">[c,h]=contour(X,Y,Z);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">set(h,</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">'ShowText'</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">,</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">'on'</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">);</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">colorbar</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">No.2</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: blue;">function</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;"> [ output_args ] = RSM( A0,A1,A2,A3,A4,A11,A22,A33,A44,</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: blue;">...</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">    A12,A13,A14,A23,A24,A34 )</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">x1=32;</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">x2=16;</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">[x3,x4]=meshgrid(4:0.1:8,0.05:0.01:0.45);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">f=A0+A1*x1+A2*x2+A3*x3+A4*x4+A11*x1*x1+A22*x2*x2+A33*x3.*x3+A44*x4.*x4+</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: blue;">...</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">A12*x1*x2+A13*x1*x3+A14*x1*x4+A23*x2*x3+A24*x2*x4+A34*x3.*x4;</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">Z=f;</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">figure(4)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">surf(x3,x4,Z)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">figure(5)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">[c,h]=contour(x3,x4,Z);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">set(h,</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">'ShowText'</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">,</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">'on'</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">);</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">colorbar</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: blue; font-family: arial,helvetica,sans-serif;">end</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">No.3</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: blue;">function</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;"> [ ] = maxRSM( A0,A1,A2,A3,A4,A11,A22,A33,A44,</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: blue;">...</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">    A12,A13,A14,A23,A24,A34 )</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">syms </SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">x1</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">x2</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">x3</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">x4</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">y</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">;</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">f=A0+A1*x1+A2*x2+A3*x3+A4*x4+A11*x1.*x1+A22*x2.*x2+A33*x3.*x3+A44*x4.*x4+</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: blue;">...</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">A12*x1.*x2+A13*x1.*x3+A14*x1.*x4+A23*x2.*x3+A24*x2.*x4+A34*x3.*x4;</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">fx=y-(A0+A1*x1+A2*x2+A3*x3+A4*x4+A11*x1.*x1+A22*x2.*x2+A33*x3.*x3+A44*x4.*x4+</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: blue;">...</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">   A12*x1.*x2+A13*x1.*x3+A14*x1.*x4+A23*x2.*x3+A24*x2.*x4+A34*x3.*x4);</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">eq1=diff(f,</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">'x1'</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">);</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">eq2=diff(f,</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">'x2'</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">);</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">eq3=diff(f,</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">'x3'</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">);</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">eq4=diff(f,</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">'x4'</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">);</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">[X1 X2 X3 X4]=solve(eq1,eq2,eq3,eq4);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">X1=vpa(X1,5)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">X2=vpa(X2,5)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">X3=vpa(X3,5)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">X4=vpa(X4,4)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">dx1=diff(solve(fx,x1),</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">'y'</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">);</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">dx2=diff(solve(fx,x2),</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">'y'</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">);</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">dx3=diff(solve(fx,x3),</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">'y'</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">);</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">dx4=diff(solve(fx,x4),</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">'y'</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">);</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">maxf=subs(f,{x1,x2,x3,x4},{X1,X2,X3,X4})</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">r1=vpa(maxf./X1./subs(dx1,{x1,x2,x3,x4,y},{X1,X2,X3,X4,maxf}),5)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">r2=vpa(maxf./X2./subs(dx2,{x1,x2,x3,x4,y},{X1,X2,X3,X4,maxf}),5)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">r3=vpa(maxf./X3./subs(dx3,{x1,x2,x3,x4,y},{X1,X2,X3,X4,maxf}),5)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">r4=vpa(maxf./X4./subs(dx4,{x1,x2,x3,x4,y},{X1,X2,X3,X4,maxf}),5)</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>