<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Collaborations</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>内页</TITLE></P><DIV class="nbv"><DIV class="nbvmain"><UL id="nbv_all"><LI id="b1"><A href="https://2015.igem.org/Team:Tianjin">HOME</A></LI><LI id="b2"><A class="note" href="https://2015.igem.org/Team:Tianjin/Project">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Project?1">DESCRIPTION</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Project?2">RESULTS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Project?3">DESIGN</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Project?4">PROTOCOLS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Project?5">FUTURE WORK</A></LI></UL><LI id="b3"><A href="https://2015.igem.org/Team:Tianjin/Parts">PARTS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Parts?1">TEAM PARTS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Parts?2">BASIC PARTS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Parts?3">COMPOSITE PARTS</A></LI><LI id="b4"><A href="https://2015.igem.org/Team:Tianjin/Modeling">MODELING</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Modeling?1">OVERVIEW</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Modeling?2">ADSORPTION</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Modeling?3">PET DEGRADATION</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Modeling?4">PROTEIN EXTRACTION</A></LI><LI class="current" id="b5"><A href="https://2015.igem.org/Team:Tianjin/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Collaborations?1">Tianjin for THU-A</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Collaborations?2">THU-A for Tianjin</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Collaborations?3">NKU for Tianjin</A></LI><LI id="b6"><A href="https://2015.igem.org/Team:Tianjin/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Safety?1">SAFE PROJECT DESIGN</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Safety?2">SAFE LAB WORK</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Safety?3">SAFE SHIPMENT</A></LI><LI id="b7"><A href="https://2015.igem.org/Team:Tianjin/Practices">HUMAN PRACTICE</A></LI><LI id="b8"><A href="https://2015.igem.org/Team:Tianjin/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Notebook?1">TIMELINE</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Notebook?2">LAB WORK</A></LI><LI id="b9"><A href="https://2015.igem.org/Team:Tianjin/Team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Team?1">ABOUT THE TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Team?2">MEMBERS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Team?3">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Tianjin/Team?4">ACKNOWLEDGEMENTS</A></LI></DIV></DIV><DIV class="main-page"><DIV class="right" id="right1"><DIV class="content" id="content1"><DIV id="content-in"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 25px;">Tianjin for Qsinghua-A</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">This year, we helped Tsinghua-A team in their wet-lab experiment. They want to mutate a single base in their DNA so that it will not be cut by a certain type of restriction enzyme. So we taught them how to do overlap PCR. Because we two team were not in the same city, it was hard for us to do experiment for them directly. So we taught them the principle of design and the protocol of experiment.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif;">Primer design of overlap PCR</SPAN></STRONG></SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif;">1.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN>The overlap region should be near 20bp</SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif;">2.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN>Each primer need another ~20bp for annealing (each primer is ~40bp in total)</SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif;">3.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN>The mutant point should be in the middle of the overlap region</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif;">Protocol of overlap PCR</SPAN></STRONG></SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif;">1.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN>First PCR:</SPAN></P><P style="margin-left: 9px; text-indent: 0px; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">One reaction system (50ul) contains 2ul 10uM forward primer and 2ul 10uM overlap reverse primer; the other reaction system contains 2ul*10uM reverse primer and 2ul*10uM overlap forward primer.</SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif;">2.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN>Run gel electrophoresis to test the PCR product.</SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif;">3.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN>Gel recovery.</SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif;">4.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN>Second PCR: </SPAN></P><P style="margin-left: 9px; text-indent: 0px; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">In the reaction system (50ul), there are 2ul of each product (L and R part of the target gene) from the previous step, and 2ul 10uM forward and reverse primer respectively.</SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif;">5.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN>Gel electrophoresis and recovery</SPAN></P><P style="margin-left: 9px; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif;">6.<SPAN style="font: 9px &quot;Times New Roman&quot;;">  </SPAN>Sequencing the product to test it.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif;">Results:</SPAN></STRONG></SPAN></P><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 16px; font-family: &quot;Arial&quot;,sans-serif;">Figure 1. </SPAN></STRONG><SPAN style="font-size: 16px; font-family: &quot;Arial&quot;,sans-serif;">From left to right, 1 is marker; 2 is L part of overlap; 3 is R part of overlap; 4 is another L part of overlap.</SPAN></SPAN></P><P style="text-align: center; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 16px; font-family: &quot;Arial&quot;,sans-serif;">Figure 2. </SPAN></STRONG><SPAN style="font-size: 16px; font-family: &quot;Arial&quot;,sans-serif;">From left to right, 1 is marker; 2&amp;4 is the second-step product of overlap.</SPAN></SPAN></P></DIV><DIV id="content-in"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 25px; font-family: arial,helvetica,sans-serif;"><STRONG>Tsinghua-A for Tianjin</STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">Here are some basic function in our program written by Tsingua-A iGEM.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">They have written detailed note to make our program readable.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: blue;">function</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;"> dots_e=rdm(n,number)</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% rdm produces binary coordinate randomly</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% (number,n),where number is the number of coordinates</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% you want and n is the n-by-n square surface</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">%</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% Calling sequence:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">%        array=rdm(number)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% Define variables:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% number   --Number of the dots that is absorbed with cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% n         --N-by-n surface</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% dots_e    --The chosen random points that will react with cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">disp(</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: rgb(160, 32, 240);">'The coordinates of degradable points on the simulated surface is as follow'</SPAN>)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">dots_e=randi(n,number,2);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: blue;">function</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;"> combine_dots_bycut=filter_dots(number_h,number_h_cut,dots_h)</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% Filter_dots is to filtate a certain array into smaller one randomly,but</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% with a certain filtration ratio.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% Calling sequence:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">%        new_array=combin_cut(number)</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% Define variables:</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% number_h        ---The number of locuses adsorbed by hydrophobins</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% number_h_cut    ---The number of locuses adsorbed by hydrophobins and</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">%                    cutinase</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% ratio_h_bind_e  ---The ratio that enzyme binds to the hydrophobin of all</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% dots_h          ---The coordinates of locuses adsorbed by hydrophobins</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% i               ---The number of locuses adsorbed by hydrophobins and</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">%                    cutinase after filtration</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% temp_number     ---The number array generated randomly in the range of number_h</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: forestgreen; font-family: arial,helvetica,sans-serif;">% </SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">temp_number=randperm(number_h,number_h_cut);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">i=1;j=1;</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: blue;">for</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;"> i=1:length(temp_number)</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">   </SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: blue;">for</SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;"> j=1:2</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: black; font-family: arial,helvetica,sans-serif;">combine_dots_bycut(i,j)=dots_h(temp_number(i),j);</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: black;">   </SPAN><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif; color: blue;">end</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; color: blue; font-family: arial,helvetica,sans-serif;">end</SPAN></P></DIV><DIV id="content-in"><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 25px;">Nankai for Tianjin</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 19px; font-family: &quot;Arial&quot;,sans-serif;">Expression of inJanus.</SPAN></STRONG></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">To express inJanus in <EM>pichia pastoris</EM>, Nankai chose fed-batch fementation carried out in bioreactor with 1L initial volume. When the cell fresh weight reached to 200g/L, Nankai started inducement for 96h. What’s more, there is a higher production when dissolved oxygen was maintained at 15%-25% by turning up the methanol speed.</SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">Follow is the figure shows the SDS-PAGE and western blot analysis of inJanus. And the inJanus molecule weight was 8KDa. In addition, inJanus in the broth was mainly monomer and dimer. The latter binds better with the primary antibody of Janus.</SPAN></P><P style="text-align:center"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 16px; font-family: &quot;Arial&quot;,sans-serif;">Figure 1. </SPAN></STRONG><SPAN style="font-size: 16px; font-family: &quot;Arial&quot;,sans-serif;">(a)silver staining along with SDS-PAGE and (b)western blot analysis of inJanus fermentation broth(lane 2,3), lane 1 was pure inJanus as control</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">To measure the yield of inJanus, Nankai measured the concentration of inJanus by UPLC. And Nankai used standard sample of sJanus of different concentration (0.1, 0.05, 0.025 mg/mL) to get a standard curve of concentration with adsorption area of UPLC.</SPAN></P><P style="text-align:center"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 16px; font-family: &quot;Arial&quot;,sans-serif;">Figure 2.</SPAN></STRONG><SPAN style="font-size: 16px; font-family: &quot;Arial&quot;,sans-serif;"> (a) silver staining along with SDS-PAGE of Janus fermentation broth of different induction time(lane 1-4 24,48,72,96 h).(b)inJanus production and cell fresh weight in different time.</SPAN></SPAN></P><P style="text-align:center"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 16px; font-family: &quot;Arial&quot;,sans-serif;">Figure 3. </SPAN></STRONG><SPAN style="font-size: 16px; font-family: &quot;Arial&quot;,sans-serif;">HPLC of pure injanus HPLC</SPAN></SPAN></P><P style="text-align:center"><SPAN style="font-family: arial,helvetica,sans-serif;"><STRONG><SPAN style="font-size: 16px; font-family: &quot;Arial&quot;,sans-serif;">Figure 4.</SPAN></STRONG><SPAN style="font-size: 16px; font-family: &quot;Arial&quot;,sans-serif;"> HPLC of injanus fermentation broth HPLC</SPAN></SPAN></P><P style="margin-top: 0px; margin-bottom: 16px; margin-left: 9px; text-align: left; line-height: 1.2em;"><SPAN style="font-size: 19px; font-family: arial,helvetica,sans-serif;">Form figure 3. And figure 4, we can clearly see that inJanus was expressed in<EM> pichia pastoris</EM>.</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>