Team:Tianjin/Notebook
内页
HOME
PROJECT
DESCRIPTION
RESULTS
DESIGN
PROTOCOLS
FUTURE WORK
PARTS
TEAM PARTS
BASIC PARTS
COMPOSITE PARTS
MODELING
OVERVIEW
ADSORPTION
PET DEGRADATION
PROTEIN EXTRACTION
COLLABORATIONS
Tianjin for THU-A
THU-A for Tianjin
NKU for Tianjin
SAFETY
SAFE PROJECT DESIGN
SAFE LAB WORK
SAFE SHIPMENT
HUMAN PRACTICE
NOTEBOOK
TEAM
ABOUT THE TEAM
MEMBERS
ATTRIBUTIONS
ACKNOWLEDGEMENTS
Receive standard plasmid
1. Gain standard plasmid
2. Transformation:
Result: After 16~24h overnight culturing, sJanus, sJanus-m and inJanus grow colonies about one microns in diameter, evenly distributed. But inJanus-m’ colonies are relatively rare and small. Try to going on to check out
3. Strain reservation and plasmid extraction
Results: Four tube plasmid concentration is about 100 ng/uL, suitable for next steps
4. Striction Digest
I choose Xba I and Pst enzyme
5. Electrophoresis
Results: We have strips of about 2000. We can basically ensure correct standard transformation
6, DNA gel extraction   
Results: Four tube plasmid concentration is 4.9, 6.0, 9.4, 6.2 ng/uL.
 
Prepare aimed protein DNA
1. Design primers.
2. PCR 50 uL system:
ddH2O
33uL
10Xbuffer
5uL
dNTP
5uL
Template
1uL
Enzyme
1uL
Primer
2.5+2.5u
Put in PCR system, set up annealing temperature, recycle and start PCR.
3. Electrophoresis
Results: Strips of a 200 is aimed proteins. At first time, we set,58,60,62℃,but we can’t get aimed band, just a very bright band much smaller than aimed DNA. We think its primer   dimer , then we set 50,55,60℃at second time. We can get aimed DNA from 55-60℃.
4. DAN extraction
5. Restriction enzyme digestion
6. Electrophoresis
7. DAN extraction
Use the same enzyme as one in aimed DNA.
 
Connection 
1. Connection
2. Transformation
3. Plasmid extraction The concentration is 60, 43, 70, 83 ng/mL
4. Enzyme digestion   
5. Electrophoresis
Result: I repeat 4 times. In first time there aren’t any 200 bond. I suspect it’s something wrong in enzyme digest. And then I use four pair of enzymes to cut it. There is just no 200 bond. At the fourth time, I choose to extend digestion time and smaller enzymes of XbaI and PstI. Then we have two stripes: about 200 and 2000 molecular   weight .
6. Sequencing:
Almost correct except some wrong sequence in the front of the aimed proteins.
 
17.05.15
·Transformation FsC-pSB1C3 with DH5α
19.05.15
·Inoculation of LB-media with DH5α_FsC-pSB1C3; Overnight incubation at 37°C
20.05.15
·Storage the overnight incubated bacteria at -80℃ with glycerin (final concentration is 20%)
·Miniprep of DH5α_FsC-pSB1C3 using the TIANgel Midi Purification Kit
Nanodrop measurements of the purified plasmid:
(1): 24.1 ng/μl
(2): 33.4 ng/μl
(3): 28.6 ng/μl
26.05.15
·PCR of FsC-pSB1C3 for amplification of FsC
In a sterile 0.5ml microfuge tube, mix in the following order:
ddH2O
68μL
10x Pfu buffer with MgSO4
10μL
dNTP(2.5mM)
8μL
Primer F
5μL
Primer R
5μL
Template DNA
2μL
Pfu DNA   polymerase(2.5U/μL)
2μL
Total volume
100μL
Separate into four sterile 0.5ml microfuge tubes to make 25μL as final volume.
The primer F and R showed below:
F: 5’-GAATTCGCGGCCGCTTCTAGATGCTGCCTACTTCTAACCCT-3’
R: 5’-AGAACCTCCACCGCCAGAACCTCCTCCACCAGCAGAACCACGGAC-3’
PCR gradient is 56.4°C, 57.6°C, 60.9°C, 64.4°C.
27.05.15
·Analysis of PCR products by agarose gel electrophoresis. No bands visible.
Possible reason is the template is not correct, so we extract plasmid again from DH5α.
·Inoculation of LB-media with DH5α_FsC-pSB1C3; Overnight incubation at 37°C
28.05.15
·Storage the overnight incubated bacteria at -80℃ with glycerin (final concentration is 20%)
·Miniprep of DH5α_FsC-pSB1C3 using the TIANgel Midi Purification Kit
Nanodrop measurements of the purified plasmid:
(1): 38.4 ng/μl
(2): 62.9 ng/μl
·PCR of FsC-pSB1C3 for amplification of FsC
Improve the system of PCR:
System I:
ddH2O
66μL
10x Pfu buffer with MgSO4
10μL
dNTP(2.5mM)
10μL
Primer F
5μL
Primer R
5μL
Template DNA
2μL
pfu DNA polymerase(2.5U/μL)
2μL
Total volume
100μL
System I:
ddH2O
66μL
10x Pfu buffer with MgSO4
10μL
dNTP(2.5mM)
10μL
DMSO
6μL
Primer F
5μL
Primer R
5μL
Template DNA
2μL
pfu DNA polymerase(2.5U/μL)
2μL
Total volume
100μL
 
Improve the efficiency of PCR by decreasing annealing temperature. Annealing gradient is 50.1°C, 55.4°C, 60.6°C, 64.5°C
29.05.15
·Analysis of PCR products by agarose gel electrophoresis.
Figure 1. The result of agarose gel electrophoresis. M is DNA marker (Trans2K DNA Marker). a to d are PCR products of System I, Template DNA(1), annealing gradient in an increasing order. e to h are PCR products of System I, Template DNA(2), annealing gradient in an increasing order. i to l are PCR products of System II, Template DNA(1), annealing gradient in an increasing order. m to o are PCR products of System II, Template DNA(2), annealing gradient in an increasing order.
Only the PCR product of System II, Template DNA(1), annealing temperature is 50.1°C has band at 650bp.
·PCR product purification using the TIANgel Midi Purification Kit
Nanodrop measurements of the purified PCR product: 8.7ng/μl
·PCR of PCR product for amplification of FsC
Use System II and decrease annealing temperature. Annealing gradient is 48.3°C, 49.1°C, 50.6°C, 52.4°C
·Analysis of PCR products by agarose gel electrophoresis.
Figure 2. The result of agarose gel electrophoresis. M is DNA marker (Trans2K DNA Marker). a to d are PCR products of FsC.
·PCR product purification using the TIANgel Midi Purification Kit
Nanodrop measurements of the purified PCR product: 65.5ng/μl
·PCR for amplification of four types of Jauns (sJanus, sJanus-m, inJanus, inJanus-m)
·Analysis of PCR products by agarose gel electrophoresis.
Figure 3. The result of agarose gel electrophoresis. M is DNA marker (DL2000 DNA Marker). a to d are PCR products of sJanus. e to h are PCR products of sJanus-m. i to k are PCR products of inJanus. l to o are PCR products of inJanus-m.
·PCR product purification using the TIANgel Midi Purification Kit
Nanodrop measurements of the purified PCR product:
sJanus: 30.2ng/μl
sJanus-m: 45.5ng/μl
inJanus: 58.4ng/μl
inJanus-m: 64.6ng/μl
15.06.15
·Restriction digest of pSB1C3 using XbaI and PstI
System I
ddH2O
21μl
Plasmid
3μl
Green buffer
3μl
XbaI
1.5μl
PstI
1.5μl
Total   volume
30μl
 
System II
ddH2O
14μl
Plasmid
10μl
Green buffer
3μl
XbaI
1.5μl
PstI
1.5μl
Total volume
30μl
 
Digest at 37°C for 2h.
·Analysis of restriction digest by agarose gel electrophoresis.
Figure 4. The result of agarose gel electrophoresis. M is DNA marker (Trans2K DNA Marker). a is digest product in System I. b is digest product in System II. c is control which is not digested by XbaI and PstI.
The bands are not in a right place. So we sequence FsC-pSB1C3. The result is no signal. Thus, we transform plasmid again with DH5αto get a correct plasmid.
08.07.15
·Transformation FsC-pSB1C3 with DH5α
09.07.15
·Inoculation of LB-media with DH5α_FsC-pSB1C3; Overnight incubation at 37°C
10.07.15
·Storage the overnight incubated bacteria at -80℃ with glycerin (final concentration is 20%)
·Miniprep of DH5α_FsC-pSB1C3 using the TIANgel Midi Purification Kit
Nanodrop measurements of the purified plasmid:
(1): 118.4 ng/μl
(2): 100.8 ng/μl
11.07.15
·Restriction digest of pSB1C3 using XbaI and PstI for verification.
·Analysis of restriction digest by agarose gel electrophoresis.
Figure 5. The result of agarose gel electrophoresis. M1 is DNA marker (Trans2K DNA Marker). M2 is DNA marker (Trans8K DNA Marker).a is digest product of FsC-pSB1C3.
·Overlap PCR of fragment FsC and four types Janus for amplification of fusion fragment
Annealing gradient is 58.8°C, 61.1°C, 63.0°C and 64.3°C
·Analysis of Overlap PCR products by agarose gel electrophoresis.
Figure 6. The result of agarose gel electrophoresis. M is DNA marker (Trans2K DNA Marker). a to d are Overlap PCR products of FsC-sJanus. e to h are Overlap PCR products of FsC-sJanus-m. i to l are PCR Overlap products of FsC-inJanus. m to o are Overlap PCR products of FsC-inJanus-m. All the annealing gradient is in an decreasing order.
12.07.15
·PCR product purification using the TIANgel Midi Purification Kit
Nanodrop measurements of the purified PCR product:
FsC-sJanus: 92.4ng/μl
FsC-sJanus-m: 67.4ng/μl
FsC-inJanus: 80.5ng/μl
FsC-inJanus-m: 75.3ng/μl
14.07.15
·Verification of Overlap PCR
  PCR of fusion fragment for verification fragment FsC and fragment Janus.
·Analysis of PCR products by agarose gel electrophoresis.
Figure 7. The result of agarose gel electrophoresis. M is DNA marker (Trans2K DNA Marker). a to d are PCR products of FsC which use FsC-sJanus, FsC-sJanus-m, FsC-INJANUS, FsC-inJanus-m,as template respectively. e, g, i, k are PCR products of Janus which use FsC-SJANUS, FsC-sJanus-m, FsC-inJanus, FsC-inJanus-m as template respectively. f, h, j, l are PCR products of Janus which use sJanus, sJanus-m, inJanus, inJanus-m as template respectively.
Fragment FsC and four types of Janus can amplify from fusion fragment which means Overlap PCR is successful.
16.07.15
·PCR of four types fusion fragment for amplification of expression fragment
17.07.15
·Analysis of PCR products by agarose gel electrophoresis.
Figure 8. The result of agarose gel electrophoresis. M is DNA marker (DL2000 DNA Marker). a1 to a4 are PCR products of FsC expression fragment. b1 to b4 are PCR products of FsC-sJanus expression fragment. c1 to c4 are PCR products of FsC -sJanus-m expression fragment. d1 to d4 are PCR products of FsC-inJanus expression fragment. e1 to e4 are PCR products of FsC-inJanus-m expression fragment.
·PCR product purification using the TIANgel Midi Purification Kit
Nanodrop measurements of the purified PCR product:
FsC: 84.6ng/μl
FsC-sJanus: 78.3ng/μl
FsC-sJanus-m: 43.8ng/μl
FsC-inJanus: 61.6ng/μl
FsC-inJanus-m: 48.9ng/μl
18.07.15
·Transformation plasmid pET-28a with Trans1-T1
19.07.15
·Inoculation of LB-media with Trans1-T1_ pET-28a; Overnight incubation at 37°C
·Storage the overnight incubated bacteria at -80℃ with glycerin (final concentration is 20%)
·Miniprep of Trans1-T1_ pET-28a using the TIANgel Midi Purification Kit
Nanodrop measurements of the purified plasmid:
(1): 146.7 ng/μl
(2): 139.5ng/μl
20.07.15
·Restriction digest of plasmid pET-28a, FsC expression fragment, FsC-sJanus-m expression fragment, FsC-inJanus-m expression fragment using XhoI and EcoRI
·Analysis of restriction digest by agarose gel electrophoresis.
Figure 9. The result of agarose gel electrophoresis. M1 is DNA marker (Trans8K DNA Marker). M2 is DNA marker (DL2000 DNA Marker). a is restriction digest product of pET-28a. b is restriction digest product of FsC expression fragment. c is restriction digest product of FsC-sJanus-m expression fragment. d is restriction digest product of FsC-inJanus-m expression fragment.
·Ligation of the digested genes (FsC expression fragment, FsC-sJanus-m expression fragment, FsC-inJanus-m expression fragment) into pET-28a.
·Transformation ligation products with DH5α
22.07.15
·Inoculation of LB-media with DH5α_ FsC_pET-28a, DH5α_ FsC-sJanus-m_pET-28a, DH5α_ FsC-inJanus-m_pET-28a; Overnight incubation at 37°C
·Storage the overnight incubated bacteria at -80℃ with glycerin (final concentration is 20%)
·Miniprep of DH5α_ FsC_pET-28a, DH5α_ FsC-sJanus-m_pET-28a, DH5α_ FsC-inJanus-m_pET-28a using the TIANgel Midi Purification Kit
Nanodrop measurements of the purified plasmid:
FsC_pET-28a (1): 108.4 ng/μl
FsC_pET-28a (2): 138.7ng/μl
FsC-sJanus-m_pET-28a (1): 96.8 ng/μl
FsC-sJanus-m _pET-28a (2): 114.5ng/μl
FsC-inJanus-m _pET-28a (1): 103.7 ng/μl
FsC-inJanus-m _pET-28a (2): 125.5ng/μl
·Verification of ligation product by restriction digest.
·Analysis of restriction digest by agarose gel electrophoresis.
Figure 10. The result of agarose gel electrophoresis. M1 is DNA marker (DL2000 DNA Marker). M2 is DNA marker (Trans8K DNA Marker). a1 is restriction digest product of FsC-inJanus-m _pET-28a (2). a2 is restriction digest product of FsC-inJanus-m _pET-28a (1). b1 is restriction digest product of FsC-sJanus-m _pET-28a (2). b2 is restriction digest product of FsC-sJanus-m_pET-28a (1). c1 is restriction digest product of FsC _pET-28a (2). c2 is restriction digest product of FsC _pET-28a (1).
Except FsC-inJanus-m _pET-28a (1), all the ligation products are correct through restriction digest. Sequencing DNA of these three kinds recombinant plasmid.
23.07.15
The templates of Janus have been confused through the result of sequencing DNA. sJanus-m is confused with SJANUS. inJanus is confused with inJanus-m and eventually we only get FsC-sJanus. We restart from PCR of other three types of Janus.
31.07.15
·PCR for amplification of three types of Janus (sJanus-m, inJanus, inJanus-m)
01.08.15
·Analysis of PCR products by agarose gel electrophoresis.
·PCR product purification using the TIANgel Midi Purification Kit
Nanodrop measurements of the purified PCR product:
sJanus-m: 57.5ng/μl
INJANUS: 89.4ng/μl
inJanus-m: 77.6ng/μl
·Overlap PCR of fragment FsC and four types Janus for amplification of fusion fragment
·Analysis of Overlap PCR products by agarose gel electrophoresis.
Figure 11. The result of agarose gel electrophoresis. M is DNA marker (DL2000 DNA Marker). a1 to a4 are Overlap PCR products of FsC-sJanus-m. b1 to b4 are Overlap PCR products of FsC-INJANUS. c1 to c4 are PCR Overlap products of FsC-inJanus-m.
No bands in the b1 to b4 which means the Overlap PCR of FsC-INJANUS are failed.
·Verification of Overlap PCR
  PCR of fusion fragment for verification fragment FsC and fragment Janus.
02.08.15
·Analysis of PCR products by agarose gel electrophoresis.
Figure 12. The result of agarose gel electrophoresis. M is DNA marker (Trans2K DNA Marker). a and c are PCR products of FsC which use FsC-sJanus-m, FsC-inJanus-m as template respectively. b and d are PCR products of Janus which uses FsC-sJanus-m, FsC-inJanus-m as template respectively.
Fragment FsC and two types of Janus can amplify from fusion fragment which means Overlap PCR is successful.
·PCR of two types fusion fragment for amplification of expression fragment
·Analysis of PCR products by agarose gel electrophoresis.
Figure 13. The result of agarose gel electrophoresis. M is DNA marker (DL2000 DNA Marker). a1 to a4 are PCR products of FsC -sJanus-m expression fragment. b1 to b4 are PCR products of FsC-inJanus-m expression fragment.
·PCR product purification using the TIANgel Midi Purification Kit
Nanodrop measurements of the purified PCR product:
FsC-sJanus-m:73.6ng/μl
FsC-inJanus-m: 66.4ng/μl
·Restriction digest of pET-28a, FsC-sJanus-m expression fragment, FsC-inJanus-m expression fragment using XhoI and EcoRI
·Analysis of restriction digest by agarose gel electrophoresis.
Figure 14. The result of agarose gel electrophoresis. M1 is DNA marker (Trans8K DNA Marker). M2 is DNA marker (DL2000 DNA Marker). a is restriction digest product of pET-28a. b is restriction digest product of FsC-sJanus-m expression fragment. d is restriction digest product of FsC-inJanus-m expression fragment.
·PCR product purification using the TIANgel Midi Purification Kit
·Ligation of the digested genes (FsC-sJanus-m expression fragment, FsC-inJanus-m expression fragment) into pET-28a.
·Transformation ligation products with DH5α
03.08.15
·Inoculation of LB-media with DH5α_ FsC-sJanus-m_pET-28a, DH5α_ FsC-inJanus-m_pET-28a; Overnight incubation at 37°C
04.08.15
·Storage the overnight incubated bacteria at -80℃ with glycerin (final concentration is 20%)
·Miniprep of DH5α_ FsC-sJanus-m_pET-28a, DH5α_ FsC-inJanus-m_pET-28a using the TIANgel Midi Purification Kit
Nanodrop measurements of the purified plasmid:
FsC-sJanus-m_pET-28a (1): 87.4ng/μl
FsC-sJanus-m _pET-28a (2): 95.9ng/μl
FsC-inJanus-m _pET-28a (1): 125.6ng/μl
FsC-inJanus-m _pET-28a (2): 103.4ng/μl
·Verification of ligation product by restriction digest.
·Analysis of restriction digest by agarose gel electrophoresis.
Figure 15. The result of agarose gel electrophoresis. M1 is DNA marker (DL2000 DNA Marker). M2 is DNA marker (Trans8K DNA Marker). a is restriction digest product of FsC-sJanus-m_pET-28a (1). b is restriction digest product of FsC-sJanus-m _pET-28a (2). c is restriction digest product of FsC-inJanus-m _pET-28a (1). d is restriction digest product of FsC-inJanus-m_pET-28a (2).
Two ligation products (FsC-sJanus-m_pET-28a (1), FsC-inJanus-m _pET-28a (2)) are correct through restriction digest. Sequencing DNA of these two kinds recombinant plasmid.
05.08.15
The results of sequencing DNA are correct.
07.08.15
·Transformation FsC _pET-28a, FsC-sJanus_pET-28a, FsC-sJanus-m_pET-28a with BL21.
 
We mixed 200ng of the target gene sequence-Thc_Cut1, which is synthesized by IDT Company, with 50ul ddH20, making the final concentration up to 4ng/μl. And then, we divided them into five eppendorf tubes equally. We confirmed the right of sequence of Thc_Cut1 by PCR.
We made sure that 58.8℃ is suitable for amplifying by the PCR of Thc_Cut1 which we have done. Next, we amplified the gene of Thc_Cut1, sJanus and its mutant sJanus-m by PCR. In this process, we didn’t do pre-experiment because of the experience of other groups. The temperature of PCR ranges from 55 centigrade to 65 centigrade. The following pictures of gel show that we got the target sequence successfully. (This is the standardized part.)
Today, we got the primers of inJanus and its mutant inJanus-m. In the experiment, we constructed a gradient of temperature, ranging from 55.7 centigrade to 63.0 centigrade to search for the best condition. The picture of them show that all of them is suitable, but the temperature of 56.9 is the best. And then, we make the target sequence recycled. (This is the standardized part.)
We linked the Thc_Cut1 with sJanus in the third step of overlap. The following picture shows the reagents we used and the exact amount we added in this system. The condition is the same to the process of our getting the gene of Thc_Cut1- sJanus-m. Using the method of PAGE, we found our target gene sequence and make them recycled.
Overlap system (100μl)
10×Buffer
10μl
dNTPS
10μl
Thc-_Thc_Cut1   standardized fragment
5μl
Hydrophobin standardized   fragment
5μl
Pfu enzyme
1μl
ddH2O
69μl
After the process of pre-PCR, we put the forward and reverse primer to this system, each for 4μl, and then continued the PCR as usual.
We confirmed that the final gene sequence is right by PCR. In this process, we use the target sequence we got in above steps as the template DNA. And then, we added the primer and the reverse of Thc_Cut1, sJanus and sJanus-m in the PCR system. Using method of PAGE, we can confirmed that these two gene have been linked together correctly. The following picture shows the reagents we used and the exact amount we added in this system.
PCR system（10μl）
10×Buffer
1μl
dNTPS
1μl
Forward primer
0.4μl
Reverse primer
0.4μl
Overlap fragment
0.4μl
Pfu enzyme
0.2μl
ddH2O
6.6μl
 
 
We used enzymes of XbaI and PstI to digest the product PCR and plasmid, and restored them in the -20 centigrade.
restriction enzyme digestion system (30μl)
ddH2O
10μl
ddH2o
16μl
10×Buffer
3μl
10×Buffer
3μl
Xba1
1.5μl
Xba1
1μl
Pst1
1.5μl
Pst1
1μl
DNA(plasmid)
14μl
DNA(PCR fragment)
10μl
 
We used T4 DNA ligase to connect the gene coding Thc_Cut1- sJanus with the plasmid of pSB1C3. In this experiment, the amount of product is about 5 times of plasmid.
 The following picture shows the reagents we used and the exact amount we added in the system of connection.
connection system
10×Buffer
2 μl
T4  ligase
1 μl
plasmid
X μl
fragment
5x μl
 
There are bacterial colony in the selective medium containing chloramphenicol. We selected three of them everyone, added into the tubes containing LB. And then, we incubated them in the shaker at the temperature of 37 centigrade overnight.
 When incubated overnight, we measured OD to judge whether the cells could be used to extract plasmid. We selected the suitable ones, and extracted the plasmid. And then, we used enzymes of Xba1 and Pst1 to digest some of them. The total volume of enzyme digest is 10μl. Using the method of PAGE, we know that the plasmid connected with Thc_Cut1- sJanus-m may be right. We sent our sample to company, using the technique of Solexa to check it.
We used the standardized gene as the template DNA to amplify the gene of expression. In this process, we also got the sequence of Thc_Cut1 which can be transferred and secret. The following picture of gel shows that we have got our target sequence successfully. The temperature of PCR is 60 centigrade.
We added 5μl of the bacterial containing the plasmid of pET28a(+) to 5mL of LB, which contains 5μl of ampicillin. And then, we incubated them at the shaker overnight.
 We exacted the plasmid and used enzymes of EcoR1 and Xho1 to digest it. In this process, the total volume of this system is 20μl. We also did the same to the target sequence and the total volume is 30μl. After finishing above all, we used gel electrophoresis to test and make the gene recycled.
 We connected the gene coding Thc_Cut1, Thc_Cut1- sJanus, Thc_Cut1- sJanus-m, Thc_Cut1- inJanus and Thc_Cut1- inJanus-m with plasmid of pET28a(+). The reagents and amount we used are same to above. The total volume of this system is 20μl. Finally, we transferred them into competent cells and incubated overnight.
 When incubated overnight, there are bacterial colonies in the selective medium contain ampicillin. We selected two of them and added into LB. Finally, we incubated them overnight. In this process, we can judge whether the plasmids we constructed are right by enzyme digesting.
 When incubated overnight, we measured OD to judge whether the cells could be used to extract plasmid. We selected the suitable ones, and extracted the plasmid. And then, we used enzymes of EcoR1 and Xho1 to digest some of them. The total volume of enzyme digest is 10μl. Using the method of PAGE, we know that the plasmid connected with Thc_Cut1 and Thc_Cut1- sJanus-m may be right. We sent our sample to company, using the technique of Solexa to check them.
5.27 Get the PCR products of four basic parts.
Figure 1. We got the PCR products of injanus, sjanus and sjanus-m. However, for some unknown reason, we didn’t get the PCR products of injanus-m. Although we had set temperature gradient as figure 1 showed.
7.11 overlap PCR
Figure 2.  In order to get the final parts which used to express protein. We did overlap PCR to get LC-sjanus, LC-injanus-m and LC-injanus-m. The detailed protocols can be seen as follow.
PCR system（100μl）
Buffer
10μl
dNTPS
8μl
LC   fragment
3μl
sJanus   fragment
3μl
Pfu   enzyme
2μl
ddH2O
70μl
Add two kinds of primer (2μl) after 5 cycle.
5 cycles
95℃
5min
95℃
30s
1：58.8℃
2：61.1℃
3：64.3℃
30s
72℃
2min30s
72℃
10min
4℃
Forever
   35 cycles
95℃
5min
95℃
30s
1：58.8℃
2：61.1℃
3：64.3℃
30s
72℃
2min30s
72℃
10min
4℃
Forever
Overlap PCR of LC-sjanus( used in PSB1C3)
Figure 3.  We tried to do overlap PCR of LC-sjanus used in PSB1C3 first. Then we used this as template DNA to PCR LC-sjanus used in expression.
In this step, we got very light electrophoresis strip the length of which is about 1200 bp.
7.12   PCR of LC-sjanus used in expression
Figure 4.  Failed. We didn’t get the product. The reason may be that the specificity of the primers was not high.
7.13 Restriction digest of LC-injanus, LC-injanus-m and LC-sjanus using EcoR1 and Xhol1
Figure 5.  Restriction digest of LC-injanus, LC-injanus-m and LC-sjanus using EcoR1 and Xhol1.
7.15 verify our PCR products
Figure 6.  In order to prove that the parts we got used in expression is correct, we used LC-injanus, LC-injanus-m, LC-sjanus and LC-sjanus-m as templates to pcr LC and other four basic parts: injanus, injanus-m, sjanus, sjanus-m. The result was partially successful. It is very clear in this figure that all parts have LC, but LC-injanus and LC-injanus-m do not have their other part. It may because that our PCR system 10μis too small.
7.16  verify our parts again
Figure 7.  This figure shows clearly that LC-sjanus-m, LC-injanus and LC-injanus-m are all correct.
7.17  Overlap PCR  LC-sJanus  (compared with other group adding DMSO )
 
Figure 8.  Failed. We can not see any differences.
7.18  change primers
Figure 9.  We changed primers to do overlap pcr but failed.
7.22 LC-sJanus overlap pcr
Figure 10.  We changed our pcr system from 50μ to 25μ, and changed cycle times from 5 to 10.
7.23 Colony PCR of LC-sjanus-m and LC-injanus-m after transformation.
Figure 11.  Colony PCR. Finally four lanes had light electrophoresis strip s. But after enzyme digestion showed that they were all false positive.
7.24  enzyme digestion of LC-sjanus-m and LC-injanus-m after transformation.
Figure 12.  Failed.
Verification PCR products of LC-sjanus.
Figure 13.  We verified the correction of LC-sjanus pcr products. The result showed that it was correct.
Amplification PCR of LC-sjanus
Figure 14.  We successfully amplified LC-sjanus although the electrophoresis strip s were light.
7.28 Enzyme digest LC-sjanus
Figure 15.  LC- janus after Enzyme digestion using EcoR1 and Xhol1.
 7.29 PCR of Standardized fragment
Figure 16.  We succeeded to get LC-injanus and LC-injanus-m but failed to get LC-sjanus.
enzyme digestion of LC-sjanus-m and LC-injanus-m after transformation
Figure 17.  Succeeded
8.1 enzyme digestion of LC-sjanus after transformation into PET-28a
Figure 18.  Failed. The reason may be the ratio of plasmid and LC-sjanus was not suitable. So we considered to change the ratio from 1:5 to 1:10.
8.03 Overlap PCR of LC-sjanus(Standardized fragment)
Figure 19.  It was clear that we failed.
8.04 enzyme digestion of standardized fragment using Xba1 and Pst1.
Figure 20.  Enzyme digestion of standardized fragment and PSB1C3.
enzyme digestion after transforming LC-sjanus and LC-injanus into PET-28a.
We tried many times but all failed. We can’t find the reason.
In follow figures, our parts should be about 1200bp length if we had.
8.05
Figure 21
8.06
Figure 22
8.08
Figure 23
8.16 enzyme digestion after transforming LC-sjanus-m into PSB1C3.
Figure 24. Succeeded. And the after DNA Sequencing results showed it was correct. The standardization of LC-sjanus-m was completed.
enzyme digestion after transforming LC-sjanus into PET-28a.
Figure 25. Failed
8.18  enzyme digestion after Single colony of LC-sjanus again.
Figure 26. Failed. Because of lack of fragment and failing many times, we decided to stop any work of LC-sjanus.
8.20 enzyme digestion after transforming LC-injanus and LC-injanus-m into PSB1C3.
Figure 27. succeeded. But the third lane of LC-injanus may lost the part of injanus. This situation also can be seen in other cloning group. It may due to the mispairing of fragment and plasmid or nonspecific binding when doing PCR.
Summary: We managed to complete the standardization of LC-sjanus-m, LC-injanus and LC-injanus-m except LC-sjanus. And we finally decided to give up LC-sjanus.
Protein expression
LC using N-terminal 6Xhis tag
Figure 1 and figure 2.  A is broth before inducing
B is broth after inducing
C is the Precipitation of broken bacteria
D is the liquid flowing through the Ni column
E is medium in the column
F is the MCAC 20 flowing through the Ni column
G is the medium in the column after being washed by MCAC 20
H is the MCAC 30 flowing through the Ni column
I is the medium in the column after being washed by MCAC 30
J is the MCAC 50 flowing through the Ni column
K is the medium in the column after being washed by MCAC 50
L is the MCAC 100 flowing through the Ni column
N is the medium in the column after being washed by MCAC 100
O is the MCAC 200 flowing through the Ni column
P is the medium in the column after being washed by MCAC 200
Q is the MCAC 500 flowing through the Ni column
R is the medium in the column after being washed by MCAC 500
S is the protein sample washed down by MCAC 100
T is the protein sample washed down by MCAC 200
If we successfully expressed LC, it should be about 36KDa. It is clear that LC was not expressed with N-terminal 6xhis tag, so we decided to change expression plasmid which expresses protein with N –terminal GST.
Figure 3.  A is broth before inducing
B is broth after inducing
C is supernatant after centrifugation
D is the Precipitation of broken bacteria
E is the medium in the column after being washed by 1xPBS
F is the medium in the column after being handled by Prescission Protease
G is the protein sample
If we successfully expressed LC, it should be about 60KDa for being fused with GST. And in lane D, we had an obvious stripe in a similar position, which means that LC was all expressed as inclusion body. Due to this, we decided to give up the expression of LC.
LC-sjanus-m using N-terminal 6Xhis tag
Figure 1 and figure 2.  A is broth before inducing
B is broth after inducing
C is the Precipitation of broken bacteria
D is the liquid flowing through the Ni column
E is medium in the column
F is the MCAC 20 flowing through the Ni column
G is the medium in the column after being washed by MCAC 20
H is the MCAC 30 flowing through the Ni column
I is the medium in the column after being washed by MCAC 30
J is the MCAC 50 flowing through the Ni column
K is the medium in the column after being washed by MCAC 50
L is the MCAC 100 flowing through the Ni column
N is the medium in the column after being washed by MCAC 100
O is the MCAC 200 flowing through the Ni column
P is the medium in the column after being washed by MCAC 200
Q is the MCAC 500 flowing through the Ni column
R is the medium in the column after being washed by MCAC 500
S is the protein sample washed down by MCAC 100
T is the protein sample washed down by MCAC 200
If we successfully expressed LC-sjanus-m, it should be about 44KDa. It is clear that LC-sjanus-m was not expressed with N-terminal 6xhis tag.
After we failed to express LC with GST, we stopped all expression work about LC. We hope other teams in the future can find way to manage it
Saturday.6.6.2015
PCR on PSB1C3
gene of interst：GFP
primer: Xba1-GFP、GFP-linker
Xba1-GFP: 5’-GCTCTAGATGCGTAAAGGAGAAGAACTTTTCACTGGAGTTGTC
GFP-linker: 5’-AGAACCACCACCACCAGAACCACCACCACCTTTGTATAGTTCATC
PCR system：（50μl）
Buffer
5μl
dNTPS
5μl
linker- sJanus primer
2μl
sJanus -Pst1 primer
2μl
sJanus-m pET28a(+)
1μl
Pfu enzyme
0.5μl
ddH2O
34.5μl
cycle
95℃
5min
95℃
30s
1：50℃
2：55℃
3：58℃
4：60℃
30s
72℃
2min
72℃
10min
4℃
Forever
 
Electrophoresis of the PCR products was identified, and finally PCR product was purified by gel extraction kit to prepare for digestion.
qualitative agarose gel electrophpresis：GFP on 55, 58, 60 centigrade worked well.
 
(The maximum sequence of the maker is 2000bp, and following is 1500, 1000, 750, 500, 250 and 100bp from up and down, our target sequence is about 750bp approximately.)
Electrophoresis of the PCR products was identified, and finally PCR product was purified by gel extraction kit to prepare for digestion.
Fragment concentration: 51.7 ng/μL
Date: Monday,7.13.2015
PCR on pET28a(+)
gene of interst：sJanus
primer：linker-sJanus、sJanus-Pst1
linker-sJanus:5’-GGTGGTGGTGGTTCTGGTGGTGGTGGTTCTAGCAACGGCAACGGCAA
sJanus-Pst1:5’- AACTGCAGCGGCCGCTACTAGTATCACCGACGGCGGTCTG
PCR system：（50μl）
Buffer
5μl
dNTPS
5μl
linker- sJanus primer
2μl
sJanus -Pst1 primer
2μl
sJanus pET28a(+)
1μl
Pfu enzyme
0.5μl
ddH2O
34.5μl
cycle
95℃
5min
95℃
30s
1：50℃
2：55℃
3：58℃
4：60℃
30s
72℃
2min
72℃
10min
4℃
Forever
 qualitative agarose gel electrophpresis：sJanus on all centigrades worked
(The maximum sequence of the maker is 2000bp, and following is 1500, 1000, 750, 500, 250 and 100bp from up and down, our target sequence is about 250bp approximately.)
Electrophoresis of the PCR products was identified, and finally PCR product was purified by gel extraction kit to prepare for digestion.
Fragment concentration:70.2ng/μL
Date: Monday,7.13.2015
PCR on pET28a(+)
gene of interst：sJanus-m
primer：linker- sJanus、sJanus-Pst1
linker- sJanus:5’-GGTGGTGGTGGTTCTGGTGGTGGTGGTTCTAGCAACGGCAACGGCAA
SJANUS-Pst1:5’- AACTGCAGCGGCCGCTACTAGTATCACCGACGGCGGTCTG
PCR system：（50μl）
Buffer
5μl
dNTPS
5μl
linker- sJanus primer
2μl
sJanus -Pst1 primer
2μl
sJanus-m pET28a(+)
1μl
Pfu enzyme
0.5μl
ddH2O
34.5μl
cycle
95℃
5min
95℃
30s
1：50℃
2：55℃
3：58℃
4：60℃
30s
72℃
2min
72℃
10min
4℃
Forever
qualitative agarose gel electrophpresis：sJanus-m on all centigrades worked
（The maximum of marker is 2000bp, and following is 1000, 750, 500, 250 and 100bp from up and down, our target sequence is about 250bp approximately.）
Electrophoresis of the PCR products was identified, and finally PCR product was purified by gel extraction kit to prepare for digestion.
Fragment concentration:70.6ng/μL
Date: Monday,7.13.2015
PCR on pET28a(+)
gene of interst：inJanus
primer：linker- inJanus、inJanus-Pst1
linker- inJanus: 5’- GGTGGTGGTGGTTCTGGTGGTGGTGGTTCTCAACAGTGCACCACT
INJANUS-Pst15’- GGTGGTGGTGGTTCTGGTGGTGGTGGTTCTCAACAGTGCACCACT
PCR system：（50μl）
Buffer
5μl
dNTPS
5μl
BFP_F primer
2μl
BFP_R primer
2μl
BFP pSB1C3
1μl
Pfu enzyme
0.5μl
ddH2O
34.5μl
cycle
95℃
5min
95℃
30s
1:50.1℃
2:53.4℃
3:56.4℃
4:58.5℃
30s
72℃
2min
72℃
10min
4℃
Forever
 
qualitative agarose gel electrophpresis：inJanus didn’t work well. The electrophoretic bands towing phenomenon is obvious.
(The maximum sequence of the maker is 2000bp, and following is 1500, 1000, 750, 500, 250 and 100bp from up and down, our target sequence is about 250bp approximately.)
analyse： We suspect is caused by low temperature. So we raise the temperature to conduct the PCR later.
Date: Tuesday,7.14.2015
PCR on pET28a(+)
gene of interst：inJanus
primer：linker- inJanus、inJanus-Pst1
linker- inJanus:
5’- GGTGGTGGTGGTTCTGGTGGTGGTGGTTCTCAACAGTGCACCACT
inJanus-Pst15’- GGTGGTGGTGGTTCTGGTGGTGGTGGTTCTCAACAGTGCACCACT
PCR system：（50μl）
Buffer
5μl
dNTPS
5μl
linker- inJanus
primer
2μl
inJanus
-Pst1 primer
2μl
inJanus
pET28a(+)
1μl
Pfu enzyme
0.5μl
ddH2O
34.5μl
cycle
95℃
5min
95℃
30s
1:59.4℃
2:65.4℃
3:67.0℃
30s
72℃
2min
72℃
10min
4℃
Forever
qualitative agarose gel electrophpresis：inJanus-m worked
（The maximum of marker is 2000bp, and following is 1000, 750, 500, 250 and 100bp from up and down, our target sequence is about 250bp approximately.）
Electrophoresis of the PCR products was identified, and finally PCR product was purified by gel extraction kit to prepare for digestion.
Fragment concentration:43.5ng/μL
Date: Tuesday,7.14.2015
PCR on pET28a(+)
gene of interst：inJanus-m
primer：linker-inJanus-m、inJanus-Pst1
linker-inJanus-m:5’-GGTGGTGGTGGTTCTGGTGGTGGTGGTTCTCAACAGTCTACCACTGGC
inJanus
-Pst15’- GGTGGTGGTGGTTCTGGTGGTGGTGGTTCTCAACAGTGCACCACT
PCR system：（50μl）
Buffer
5μl
dNTPS
5μl
linker-inJanus-m primer
2μl
inJanus
-Pst1 primer
2μl
inJanus-m pET28a(+)
1μl
Pfu enzyme
0.5μl
ddH2O
34.5μl
cycle
95℃
5min
95℃
30s
1:59.4℃
2:65.4℃
3:67.0℃
30s
72℃
2min
72℃
10min
4℃
Forever
qualitative agarose gel electrophpresis：inJanus-m on these three centigrades worked
 
（The maximum of marker is 2000bp, and following is 1000, 750, 500, 250 and 100bp from up and down, our target sequence is about 250bp approximately.）
Electrophoresis of the PCR products was identified, and finally PCR product was purified by gel extraction kit to prepare for digestion.
Fragment concentration:55.0ng/μL
Date: Wednesday,7.15.2015
SOE PCR
SOE PCR stands for Splicing by Overlapp Extension PCR. It is a standard overlapp extension procedure, enabling the assembly of genes wihtout performing any cloning, digesting or ligation inbetween. All you need to do ist running PCRs with specific primers. If gene A is the upstream part and gene B has to be assembled downstream of gene A, primer lo of gene A should have an overlapp of around 20 nucleotides complementary to the first 20 nucleotides of gene B. Primer up of gene B should haven a complementary overlapp of 20 nucleotides to the end of gene A.
gene of interst：GFP- sJanus
primer：Xba1-GFP、sJanus-Pst1
temperature: 53, 55, 56, and 58 centigrade.
PCR system（100μl）
Buffer
10μl
dNTPS
10μl
GFP  fragment
5μl
sJanus fragment
5μl
Pfu enzyme
1μl
ddH2O
69μl
Add two kinds of primer (1μl) after 5 cycle.
cycle
95℃
5min
95℃
30s
1：50℃
2：56℃
3：65℃
 
 
30s
72℃
2min
72℃
10min
95℃
5min
95℃
30s
1：50℃
2：56℃
3：65℃
30s
72℃
2min
72℃
10min
4℃
Forever
qualitative agarose gel electrophpresis：GFP－sJanus for SOE PCR with PSB1C3 worked
The maximum of marker is 2000bp, and following is 1000, 750, 500, 250 and 100bp from up and down, our target sequence is about 1000bp approximately.
Electrophoresis of the PCR products was identified, and finally PCR product was purified by gel extraction kit to prepare for digestion.
Fragment concentration:32.2ng/μL
Date: Wednesday,7.15.2015
SOE PCR
gene of interst：GFP-sJanus-m
primer：Xba1-GFP、sJanus -Pst1
temperature： 53, 55, 56, and 58 centigrade.
 PCR system（100μl）
Buffer
10μl
dNTPS
10μl
GFP  fragment
5μl
sJanus-m fragment
5μl
Pfu enzyme
1μl
ddH2O
69μl
Add two kinds of primer (1μl) after 5 cycle.
cycle
95℃
5min
95℃
30s
1：50℃
2：56℃
3：65℃
30s
72℃
2min
72℃
10min
95℃
5min
95℃
30s
1：50℃
2：56℃
3：65℃
30s
72℃
2min
72℃
10min
4℃
Forever
qualitative agarose gel electrophpresis：GFP－sJanus-m for SOE PCR with PSB1C3 worked
The maximum of marker is 2000bp, and following is 1000, 750, 500, 250 and 100bp from up and down, our target sequence is about 1000bp approximately.
Electrophoresis of the PCR products was identified, and finally PCR product was purified by gel extraction kit to prepare for digestion.
Fragment concentration: 30.5ng/μL
Date: Wednesday,7.15.2015
SOE PCR
gene of interst：GFP- sJanus
primer：Xba1-GFP、inJanus-Pst1
 temperature： 55, 56.3 ,57.8 and 58.9 centigrade.
PCR system（100μl）
Buffer
10μl
dNTPS
10μl
GFP  fragment
5μl
inJanus fragment
5μl
Pfu enzyme
1μl
ddH2O
69μl
Add two kinds of primer (1μl) after 5 cycle.
cycle
95℃
5min
95℃
30s
1：55℃
2：56.3℃
3：57.8℃
4：58.9℃
30s
72℃
2min
72℃
10min
95℃
5min
95℃
30s
1：50℃
2：56℃
3：65℃
30s
72℃
2min
72℃
10min
4℃
Forever
qualitative agarose gel electrophpresis：GFP－inJanus for SOE PCR with PSB1C3 worked
（The maximum of marker is 2000bp, and following is 1000, 750, 500, 250 and 100bp from up and down, our target sequence is about 1000bp approximately.）
Electrophoresis of the PCR products was identified, and finally PCR product was purified by gel extraction kit to prepare for digestion.
Fragment concentration:31.1ng/μL
Date：Wednesday,7.15.2015
SOE PCR
gene of interst：GFP-inJanus
primer：Xba1-GFP、inJanus-Pst1
 temperature： 55, 56.3 ,57.8 and 58.9 centigrade.
PCR system（100μl）
Buffer
10μl
dNTPS
10μl
GFP  fragment
5μl
inJanus-m fragment
5μl
Pfu enzyme
1μl
ddH2O
69μl
Add two kinds of primer (1μl) after 5 cycle.
cycle
95℃
5min
95℃
30s
1：55℃
2：56.3℃
3：57.8℃
4：58.9℃
30s
72℃
2min
72℃
10min
95℃
5min
95℃
30s
1：50℃
2：56℃
3：65℃
30s
72℃
2min
72℃
10min
4℃
Forever
 qualitative agarose gel electrophpresis：GFP－inJanus-m for SOE PCR with PSB1C3 worked
（The maximum of marker is 2000bp, and following is 1000, 750, 500, 250 and 100bp from up and down, our target sequence is about 1000bp approximately.）
Electrophoresis of the PCR products was identified, and finally PCR product was purified by gel extraction kit to prepare for digestion.
Fragment concentration:50.3ng/μL
 Date: Thursday,7.16.2015
restriction enzyme digestion
After qualitative agarose gel electrophpresis, all the target sequence are 1000bp approximately.
Fragment concentration: sJanus is 6.9ng/μL, sJanus-m is 15.3ng/μL, inJanus is 4.6ng/μL, inJanus-m is 7.0ng/μL.
Date: Friday,7.17.2015
We used T4 DNA ligase to connect them at the temperature of 22 centigrade about an hour.
Transformed the expression vectors we had already connected into the Escherichia coli DH5α, then incubated the bacteria for 12h (37℃).
Date:Saturday,7.18.2015
Enzymes cut each, but they did not work.
（The maximum of marker is 8000bp, and following is 8000, 5000, 3000,1500,1000and 500bp from up and down, our target sequence is about 1000bp approximately. But the bands in this figure are all about 5000bp, so we suspect that the fragments are not connected to plasmid.）
Analyse：The fragments are not connected to plasmid.
Date: Sunday,7.19.2015
We chose another DH5α and trans
formed again.
Date: Monday,7.20.2015
Enzyme cut,
Date: Tuesday,7.21.2015
We design another kind of primer and reverse which contain EcoRI and XhoI site. We have got the standard GFP-sJanus and GFP-sJanus-m fragments, and we got the target fragments used to construct the expression vector by PCR.
gene of interst：EcoR1-GFP-sJanus -Xho1、EcoR1-GFP-sJanus-m-Xho1
primer：EcoR1-GFP、sJanus-Xho1
EcoR1-GFP: 5’-CGGAATTCATGCGTAAAGGAGAAGAACTT
sJanus-Xho1: 5’-CCGCTCGAGTCAAGCACCGACGGCC
PCR system：（50μl）
Buffer
5μl
dNTPS
5μl
EcoR1-GFP primer
2μl
sJanus-Xho1 primer
2μl
Xba1-GFP-sJanus -Pst1/ Xba1-GFP-sJanus-m-Pst1
1μl
Pfu enzyme
0.5μl
ddH2O
34.5μl
cycle
95℃
5min
95℃
30s
1:55.4℃
2:56.6℃
3:57.8℃
4:60.3℃
30s
72℃
2min
72℃
10min
4℃
Forever
qualitative agarose gel electrophpresis：fragments on these four centigrades all worked well.
（The maximum of marker is 2000bp, and following is 1000, 750, 500, 250 and 100bp from up and down, our target sequence is about 1000bp approximately.）
Electrophoresis of the PCR products was identified, and finally PCR product was purified by gel extraction kit to prepare for digestion.
Fragment concentration:EcoR1-GFP-sJanus -Xho1:55.0ng/μL、EcoR1-GFP-sJanus-m-Xho1:60ng/μL
Date: Saturday,8.1.2015
We used enzymes of EcoRI site and XhoI site to digest the PCR product and plasmid. We used T4 DNA ligase to connect them at the temperature of 22 centigrade for about one hour. Then we transformed the expression vectors we had already connected into the Escherichia coli DH5α, then incubated the bacteria for 12h (37℃)
It has been confirmed that these sequences we get are right when detecting them in company.
Date:Monday,8.10.2015
Transformed the expression vectors(concentration:140.3ng/μL、121.4ng/μL) we had already built before (pET28a(+)) into the Escherichia coli BL21, then incubated the bacteria for 12h (37℃).
1.Get the gene of BFP
1.1 In order to get the fusion protein, we use overlap PCR to construct the fragment(BFP- sJanus & BFP-sJanus-m).We get the gene of BFP by the standard plasmid from iGEM. Then we use PCR to amplify it. The primer we design is as follow:
F:5’- GAATTCGCGGCCGCTTCTAGATGAGCGAACTGATCAAAGAG-3’
(EcoRI and XbaI site are contained).
R:5’- GTGGTGGTGGTCTTGGTGGTGGTGGATTCAGTTTATGACCCAGCTT-3’
(linker-GGGGSGGGGS is contained)
We get the target sequence by PCR at the temperature of 55 ,60 and 65 centigrade.
1.2 detail of the experiment
PCR system：（50μl）
Buffer
5μl
dNTPS
5μl
BFP_F primer
2μl
BFP_R primer
2μl
BFP PSB1C3
1μl
Pfu enzyme
0.5μl
ddH2O
34.5μl
 
 
95℃
5min
95℃
30s
1：55℃
2：60℃
3：65℃
 
30s
72℃
2min
72℃
10min
4℃
Forever
 
2.Get the gene of BFP- sJanus & BFP-sJanus-m
2.1We get the gene of BFP- sJanus & BFP-sJanus-m by the plasmid pET28a(+) extracting from DH5α. And then we amplify it by PCR. The primer we design is as follow:
F:5’-ACCACCAAGACCACCACCACCAAGAAGCAACGGCAACGGCAAT-3’
(linker-GGGGSGGGGS is contained)
R:5’-CTGCAGGCGGCCGCTACTAGTATCAAGCACCGACGGCGG-3’
( SpeI, NotI, and PstI site are contained)
We get the target sequence by PCR at the temperature of 55, 60 and 65 centigrade.
2.2 Details of experiment
 
 
PCR system：（50μl）
Buffer
5μl
dNTPS
5μl
P_F
2μl
P_R
2μl
sJanus PET28a
2μl
Pfu enzyme
0.5μl
ddH2O
33.5μl
 
95℃
5min
95℃
30s
1：55℃
2：60℃
3：65℃
 
30s
72℃
2min
72℃
10min
4℃
Forever
 
2.3 Results
Figure 1 The maximum of marker is 2000bp, and the centigrade of 60 is not suitable for the system because there is a track behind it. Our target is about 250bp approximately
We get the target fragment in the end. All temperature were suitable for the PCR cycle.
3.Overlap PCR
3.1 We use the technique of overlap PCR to get our target sequence. In this experiment, we add BFP and sJanus & sJanus-m which we get in two steps and the BFP’s primer we design and sJanus & sJanus-m’s reverse in the system of PCR. We get the target sequence at the temperature of 55, 60 and 65 centigrade.
3.2 Detail of the experiment
 
PCR system（100μl）
Buffer
10μl
dNTPS
10μl
BFP    fragment
5μl
sJanus fragment
5μl
Pfu enzyme
1μl
ddH2O
69μl
 
Add two kinds of primer (1μl) after 5 cycle.
 
95℃
5min
95℃
30s
1：50℃
2：56℃
3：65℃
 
30s
72℃
2min
72℃
10min
95℃
5min
95℃
30s
1：50℃
2：56℃
3：65℃
 
30s
72℃
2min
72℃
10min
4℃
Forever
 
 
3.3 Results
Figure 2
Figure 3. The maximum of marker is 2000bp, and following is 1000, 750, 500, 250 and 100bp from up and down, our target sequence is about 1000bp approximately. We failed at first attempt. But as we change the temperature we use in PCR cycle. We finally get the target fragment.
As we can see from the picture, there are two bands in every channel. One is about 1000bp and another is about 300bp. We do not know the reason of this. But after doing the confirmatory experiment, we are sure the 1000bp band is the target band we want (BFP- sJanus & BFP-sJanus M). We did not get the fragment at first attempt. We change the temperature in PCR cycle.
4. Construct the plasmid
4.1 We design another kind of primer and reverse which contain EcoRI and Hind III site. The condition to amplify it by PCR is same as above all. We choose pET28a(+) as our expression vector. And then, we use enzymes of EcoRI site and XhoI site to digest the PCR product and plasmid. We use T4 DNA ligase to connect them at the temperature of 22 centigrade. We insert the plasmid into competent DH5α. It has been confirmed that these sequences we get are right when detecting them in company.
4.2 detail of the experiment
 
Digest (60μl)    1h
ddH2O
31μl
buffer
6μl
Fd1
1.5μl
Fd2
1.5μl
DNA
20μl
 
 
T4 DNA ligase to connect（20μl）  1h
Buffer
2μl
T4 DNA ligase
0.5μl
plasmid
X
fragment
5X
 
