Team:Tianjin/Project
内页
HOME
PROJECT
DESCRIPTION
RESULTS
DESIGN
PROTOCOLS
FUTURE WORK
PARTS
TEAM PARTS
BASIC PARTS
COMPOSITE PARTS
MODELING
OVERVIEW
ADSORPTION
PET DEGRADATION
PROTEIN EXTRACTION
COLLABORATIONS
Tianjin for THU-A
THU-A for Tianjin
NKU for Tianjin
SAFETY
SAFE PROJECT DESIGN
SAFE LAB WORK
SAFE SHIPMENT
HUMAN PRACTICE
NOTEBOOK
TEAM
ABOUT THE TEAM
MEMBERS
ATTRIBUTIONS
ACKNOWLEDGEMENTS
Description
Overview
In ancient Roman myth, Janus is the god of beginnings and transitions, who is depicted as having two faces, respectively to the future and the past.
Our project is focused on another Janus - hydrophobin the protein, who faces the hydrophilicity and hydrophobicity. Because of this, a sea of new applications are created.
Firstly, we redesign the structures of two classes of hydrophobins, making expression in E.coli possible.
Secondly, we use its double-sticky-tape-like ability to make two applications. We take this advantage to fix antibodies on a high-flux tumor detection chip. Meanwhile, they are used to catch cutinases for plastic degradation. We even make them into a fusion to test if the enhancement could be better.
Thirdly, we use its amphipathicity to achieve protein separation, where they act as a special purification tag, and the system could be as simple as polymer, detergent and water. With help of this, we could even achieve recovery of cutinases.
Figure 1. The overview of our project
Feel interested about our story? So let's follow Janus and begin our journey from old to new!
 
Background
As is known to us all, for most proteins, hydrophobic residues are buried in the core of proteins stabilizing the folded conformation of the protein. However, as to our Janus - hydrophobin, one part of the surface is consisted nearly entirely of hydrophobic side chains, forming what was called “the hydrophobic patch” [1]. We could then describe the structure of Janus as a "surfactant" with one hydrophobic and one hydrophilic part.
Figure 2. The difference between most protein and hydrophobin
Janus are proteins that are produced by filamentous fungi, such as Ascomycetes and Basidiomycetes. Many different aspects of fungal development have been attributed to Janus. For example, they are thought to play a role in the formation of aerial hyphae and fruiting bodies. Because Janus could assemble at the medium–air interface and cause the surface tension to be lowered, they could allow hyphae to breach the water–air interface [2].
Figure 3. Janus could assemble at the medium–air interface and cause the surface tension to be lowered, they could allow hyphae to breach the water–air interface.
One of the most important features of Janus is that they are able to assemble spontaneously into amphipathic monolayers at hydrophobic–hydrophilic interfaces [3].What's more, the stability of assembled Janus differs and on the basis of this characteristic two classes of Janus can be distinguished [4]. And the essential differences lie in the occurrence of hydrophilic and hydrophobic amino acid residues i.e. according to their hydropathy plots [1]. Class I Janus generate very insoluble assemblies, which can only be dissolved in strong acids such as trifluoroacetic acid or formic acid, while assemblies of class II Janus can be dissolved in ethanol or sodium dodecyl sulfate or through the application of pressure or lowering of the temperature [5].
Figure 4. The principle of Janus-based aqueous two-phase system
References:
[1] M.B. Linder. Hydrophobins: proteins that self assemble at interfaces Curr Opin Colloid Interface Sci, 14 (2009), pp. 356-363
[2] Wo”sten HAB, van Wetter M-A, Lugones LG, van der Mei HC, Busscher HJ, Wessels JGH: How a fungus escapes the water to grow into the air. Curr Biol 1999, 9:85-88.
[3] Wessels, 2000. J.G.H. Wessels. Hydrophobins, unique fungal proteins. Mycologist, 14 (2000), pp. 153–159
[4] Wessels JG: Developmental regulation of fungal cell wall formation. Annu Rev Phytopathol 1994, 32:413-437.
[5] Hydrophobins: Proteins with potential. Hektor H.J., Scholtmeijer K. (2005) Current Opinion in Biotechnology, 16 (4), pp. 434-439.
 
Neo-Protein Design
Among all kinds of hydrophobins, Class I hydrophobin inJanus (we name it from its insoluble property) from Grifola frondosa and Class II sJanus (we name it from its soluble property) from Trichoderma reesei are models of Class I and Class II respectively. However, it has been reported that bacterial hosts could not be used to produce functional Class I Janus, which causes great obstacles in broader applications of them.
Figure 5. The resource of our Janus
When expressed in E.coli, inJanus usually exist as inclusion body. And there are some possible reasons for this phenomenon: the expression is so fast that protein have no enough time to fold; there are many amino acids with S element, and the environment in E.coli goes against the formation of disulfide bond; lack of necessary enzyme and cofactors [1].
Thus, we made some directed mutations of inJanus, and make its expression in E.coli possible. Meanwhile, we do the same to sJanus. Though it could be expreesed in E.coli (the difference in solubility may lead to it), we would like to research on its difference. Then we made contact angle experiments to test their properties. Ultimately, our project is about four kinds of hydrophobins- inJanus, sJanus and their respective mutants: inJanus-m and sJanus-m.
Figure 6. We make some mutation of inJanus to make its expression in E.coli possible. We also do the same to sJanus to see its difference.
References:
[1] Zefang Wang. Expression, functional application and self-assembly mechanism of hydrophobin HGF1 (2010).
 
Super Protein Chip
Background
Protein chip is an emerging technology, which can be used to track the interactions of proteins and to determine their function. Nowadays they are playing a crucial role in various fields around the world, especially in clinical medicine. According to many researches, protein chip has already become a new method of the rapid detection of tumor markers, but it is still a difficult problem to solve the combination between the matrix and the probe, due to the biomolecule inactivation caused by ordinary substrate material.
For instance, polystyrene, which has hydrophobic surface, is a commonly used substrate. When antibodies are absorbed onto it, the strong hydrophobic force between them lead to the formation of multilayer film. Among all the antibodies that are absorbed, only the top of them can maintain their activities, while the rest would go through conformational changes and become inactivated. This would result in low sensitivity and a huge waste of reagents.
Scientists have always tried to find new method of surface modification to improve substrate properties, such as three-dimensional surface modification of glass or use gold membrane to replace the ordinary substrate. However, these methods show either complex operation or high cost[1].
A very interesting characteristic of assembled Janus is the amphipathic nature of the coating. By changing the hydrophobicity of a surface the binding of various molecules and cells can be manipulated [2]. In view of this, Janus could find use in substrate modification applications.
Our Goal
Based on previous experience, we use Janus to modify protein chip substrate so as to optimize it. In the process of our experiment, Janus acts as a medium in antibody-fixing by electrostatic force [3]. Compared to the traditional methods, this fixing method not only ensures the activity of the antibody, but also improves the detection’s accuracy rate. Additionally, it can greatly reduce the cost of chip’s making [4]. The most important thing is that this method enables us to use antigens in very light concentration, which can save a lot of samples.
Our Design
In our experiment, we used two kinds of Janus-inJanus and sJanus. inJanus generate more stable assemblies compared to sJanus. We use hydrophobins from two categories on purpose of studying the differences between them in aspect of surface modification.The serum levels of tumor markers are directly related to the state of cancer. The quantitative detection of them in serum will be valuable for clinical research and early diagnosis [5]. In our experiment, we chose CEA, AFP and CA15-3 and their respectively antibodies as samples. Levels of CEA can be associated with lung cancer, and AFP - liver cancer, CA15-3 - breast cancer. In fact, there are a large number of tumor markers that associated with cancer, but the markers we chose are more representative and easier to purchase.
Figure 7. The principle of super protein chip
Based on Double Antibody Sandwich Method, we built a biosensor system which is suitable for the measurement of a wide range of biomarkers. Though the specific binding of antigen (tumor marker) and their polyclonal antibody, we can use the fluorescence microscope to detect the signal and analyze so as to get the test results, then, we can judge the type of tumor.
Figure 8. Double Antibody Sandwich Method
Our project fully takes advantage of the high-throughput of the protein microarray, the high-specificity of the Double Antibody Sandwich Method [6], the hypersensitivity of the fluorescence labeling, in order to detect tumor markers rapidly and precisely, and discover patients’ condition in time, having quite vast potential for future medicinal development.
References:
[1] Knoll W,Liley M,Piscevic D,et a1.Supramolecular architectures for the functionalization of solid SUrfaces[J].Adv Biophys,1997,34
[2] Wessels JG: Hydrophobins: proteins that change the nature of the fungal surface. Adv Microb Physiol 1997, 38:1-45.
[3] Chunwang Peng, Jie Liu, Daohui Zhao, and Jian Zhou : Adsorption of Hydrophobin on Different Self-Assembled Monolayers: The Role of the Hydrophobic Dipole and the Electric Dipole. School of Chemistry and Chemical Engineering, Guangdong Provincial Key Lab for Green Chemical Product Technology, South China University of Technology, Guangzhou, Guangdong 510640, P. R. China.
[4] Bayry J, Aimanianda V, Guijarro JI, Sunde M, Latge'JP (2012) Hydrophobins—Unique Fungal Proteins. PLoS Pathog 8(5): e1002700. doi:10.1371/journal.ppat.1002700
[5] Miao, X., Zou, S., Zhang, H., Ling, L., 2014. Sens. Actuators B 191, 396–400.
[6] Markus F. Templin, Dieter Stoll, Monika Schrenk, Petra C. Traub, Christian F. Vöhringer and Thomas O. Joos: Protein microarray technology.TRENDS in Biotechnology Vol.20 No.4 April 2002
 
Stimulated Plastic Enzymolysis
Background
Since completely synthetic plastic materials came out in 1900s, it was used more and more extensively due to its cheapness and durability. Nowadays, plastics has covered our lives in many areas. However, it is slow to degrade, which has led to serious plastic pollution.
Plastic reduction efforts have occurred in some areas in attempts to reduce plastic consumption and pollution and promote plastic recycling. Scientists have come up with many methods in order to solve white pollution, such as landfill, incineration and chemical decomposition. It is a good idea to degrade plastics by enzymes. Compared with the traditional physical and chemical methods, it costs lower power and is more environmental. However, biological method has much lower degradation efficiency. Therefore, improving the efficiency of biological plastic degradation has no time to delay.
Plastic degradation is not only of vital importance for solving environmental pollution. For example, enzymatic recycling of PET basically would break down the polymer into its building blocks ethylene glycol (EG) and terephthalic acid (TA), which have a high value and can be reused in chemical synthesis, including the production of PET. This would avoid current limitations in plastic recycling, which requires pure plastic fractions or has to fight with enrichment of contaminants [1]. At the same time, it provides a new method for plastic surface modification, making it possible for applying finishing compounds and coloring agents.
Figure 9. The reaction of plastic degradation
Among all kinds of plastics, PET (polyethylene terephthalate) plays a major role. It is the most commonly used synthetic fiber (25 million tonnes produced annually worldwide) and is forecasted to account for almost 50% of all fiber materials in 2008[2]. PET hydrolysis can be achieved by enzymes from distinct classes, such as esterases, lipases, and cutinases, the latter thereby yielding the most promising results [3]. Hence, we choose three kinds of cutinases for our project – FsC (Fusarium solani cutinase), LC (leaf-branch compost cutinase) and Thc_Cut1 (Thermobifida cellulosilytica cutinase). Especially, FsC was used and by TU_Darmstadt in 2012 iGEM, and LC by UC_Davis in 2012 iGEM.
Our Goal
1. Stimulate PET hydrolysis by FsC, LC and Thc_Cut1 with addition of four kinds of Janus.
2. Enhance enzyme activity by making fusion proteins with Janus.
3. Compare the differences between four kinds of Janus for stimulating cutinase activity.
4. Study the mechanism behind Janus's acceleration.
Our Design
In 2005, Toru Takahashi etc. found that growth of the mold fungus Aspergillus oryzae on the biodegradable polyester polybutylene succinate-coadipate (PBSA) induced not only the cutinase (CutL1) but also a hydrophobin (RolA), which adsorbed to the hydrophobic surface of PBSA, recruited CutL1, and stimulated its hydrolysis of PBSA [4]. This gives us the inspiration that Janus may stimulate enzymatic plastic degradation. As well, in 2013, Liliana Espino-Rammer etc. added two kinds of Class II hydrophobins HFB4 and HFB7 of Trichoderma into the process of PET hydrolysis by Humicola insolens cutinase, and they turned out to stimulate the enzyme activity [4].
Figure 10. Two hypotheses of the mechanism of behind Janus's stimulation of enzymatic reaction hydrolysis:(a) Janus make the PET more approachable applicable to the cutinase. (b) Janus could change the cutinase's structure.
In our project, we would try two new hydrophobins, inJanus and sJanus, and their respective mutants, to test if they have the ability to increase the activity of above-mentioned three cutinases- FsC, LC and Thc_Cut1. It is the first time to study two classes of Janus in the process of plastic degradation, and it could reveal the differences between them and provide a better hydrophobin partner. Meanwhile, will our brand new inJanus-m and sJanus-m behave well in the process? We would like to explore.
Because the mechanism behind Janus' acceleration is still unknown, we would try different ways to add Janus into plastics, and various ways could lead to huge difference of Janus's effect. The first way is simple mixing, which means mixing cutinase, Janus and PET at the same time. The second way is PET pre-incubation with Janus, which means adding Janus into plastics primarily, and then putting cutinase in the container after 24 hours. The third way is making Janus and cutinase into fusion protein, which is a novel work that could increase cutinases' activity radically.
Figure 11. Three ways of hydrophobin addition.
Furthermore, we develop comprehensive and innovative models on Janus-enhanced plastic enzymolysis. We make a good combinaton of thermodynamics and dynamics, and a novel kinetic model is built to better describe the enzyme catalyzed hydrolysis of PET by combing the Michaelis-Menten equation with the Langmuir equation. Meanwhile, we propose models to describe the effect of Janus into the hydrolysis system based on the hyphothetical elementary reactions using the result of self-asembling model and the novel kinetic model in the first part. Thirdly, to visualize the result of our modelling, we write a program using MATLAB which simulates PET degradation process. (Click here to see more about our models!)
References:
[1] Espino-Rammer L, Ribitsch D, Przylucka A, Marold A, Greimel KJ, Herrero Acero E, Guebitz GM, Kubicek CP, Druzhinina IS. 2013. Two novel class II hydrophobins from Trichoderma spp. stimulate enzymatic hydrolysis of poly(ethylene terephthalate) when expressed as fusion proteins. Appl. Environ. Microbiol. 79:4230-4238.
[2] Guebitz, G.; Cavaco-Paulo, A. Enzyme go big: surface hydrolysis and functionalisation of synthetic polymers. Trends Biotechnol. 2007, 26 (1), 32–38.
[3] Eberl A, Heumann S, Kotek R, Kaufmann F, Mitsche S, Cavaco-Paulo A, Gübitz GM. 2008. Enzymatic hydrolysis of PTT polymers and oligomers. J. Biotechnol. 135:45–51.[4] Takahashi T, Maeda H, Yoneda S, Ohtaki S, Yamagata Y, Hasegawa F, Gomi K, Nakajima T, Abe K. 2005. The fungal hydrophobin RolA recruits polyesterase and laterally moves on hydrophobic surfaces. Mol. Microbiol. 57:1780–1796.
 
Protein Extraction Kit
Background
For selective purification of proteins, affinity chromatography is one of the most efficient methods available. Often they are only suited for analytical purposes and purification of high-value products and are too expensive for large-scale products such as industrial enzymes. Furthermore, the methods are difficult to scale up [1].
Liquid-liquid extraction in an aqueous two-phase system has been applied for primary recovery of industrial bulk proteins. They are formed in mixtures between two incompatible components, e.g. PEG/dextran, polymer/salt or detergent/polymer.
Usually, aqueous two-phase systems' formation could be induced by a shift in temperature above a critical temperature (cloud point) in a detergent/water system. Non-ionic detergents, such as Triton X-114, display such temperature-sensitive phase separation, and can form cloud point extraction (CPE) systems [2].
The partitioning of a protein to one of the phases may depend on its surface charge or hydrophobicity, but the driving forces are not well understood. Obviously, the selectivity and overall efficiency of purification depend on how much the target protein differs in relevant properties as compared to the rest of proteins in the mixture. To achieve selective protein purification in aqueous two-phase system, small hydrophobic tags containing tryptophans have been fused to the target protein. However, the production of these fusion molecules is not efficient, possible reasons being problems in secretion of the tagged proteins or proteolytic degradation of the tag [1].
Our Goal
1. Achieve efficient protein extraction using Janus tag.
2. Find the best Janus tag.
3. Testify this system applicative to all kinds of proteins.
4. Develop universal BioBrick Janus tag for all users. 5. Explore the universally good conditions to all kinds of proteins.
6. Propose standard protocols for Janus-based protein extraction.
Our Design
Here comes the Janus, which could perform as a perfect protein extraction tag. The size of inJanus is about 8 kDa, and sJanus is about 7.5 kDa; both of them are not very large to affect the target protein. At the same time, Janus could promote the production of some proteins in various cells, like E4GIcore-sJanus in T. reesei [2] and GFP-sJanus in Nicotiana benthamiana plants [3].
To purify the target protein, we would make the target protein and Janus into a fusion protein, which is connected with a linker. Then, we could use aqueous two-phase system to make them separated from the bulk protein, according to the property that Janus will direct to the phase of detergent in the system of detergent/water.After back-extraction, we could get the protein out from the detergent.
Figure 12. The principle of Janus-based aqueous two-phase system
In our project, we would use four kinds of Janus to find the best one. Meanwhile, we aim to design a standard protocol to make this system applicative to almost every kind of protein, and we call it Protein Extraction Kit.  
Firstly, we use the principle of RFC 23 to design a universal Janus tag, which includes a linker able to be cut by TEV protease and a sJanus gene. Thus, if other teams want to use this system to express and purify their proteins, they could use the standard assembly to make a fusion protein. Secondly, we explore for the best detergent, the best concentration of detergent and the best buffer solution. Thirdly, we construct many fusion proteins, including three kinds of fluorescent proteins (GFP, BFP and RFP) with four kinds of Janus, which are inJanus, sJanus and their respective mutants. We also construct experiments about fusion proteins with cutinases Thc_Cut1, FsC and LC to test if this system could be used on the putification of enzymes. Furthermore, we build complete models to testify that this system is applictive to all kinds of protein and search for the universally good conditions for almost every protein.
Figure 13. The principle of Protein Extraction Kit
Of course, we also conduct experiments to detect the efficiency of this method and make sure that the target proteins have not lost their activity.
References:
[1] Linder MB, Qiao M, Laumen F, Selber K, Hyytiä T, Nakari-Setälä T, Penttilä ME (2004) Efficient purification of recombinant proteins using hydrophobins as tags in surfactant-based two-phase systems. Biochemistry 43:11873–11882
[2] Collen, A., Persson, J., Linder, M., Nakari-Setälä, T., Penttilä, M., Tjerneld, F., Sivars, U., 2002. A novel two-step extraction method with detergent/polymer systems for primary recovery of the fusion protein endoglucanase I-hydrophobin I. Biochimica et Biophysica Acta (BBA) 1569, 139–150.
[3] Joensuu JJ, Conley AJ, Lienemann M, Brandle JE, Linder MB, Menassa R. Hydrophobin fusions for high-level transient protein expression and purification in Nicotiana benthamiana. Plant Physiol 2010;152:622-33.
Results
Protein Expression
The first step of our experiments is to get the target proteins. And we have expressed what we need successfully.
 
Janus Expression
The bacteria ( E.coli BL21) were cultured in 5mL LB liquid in 37℃ for 6-8 hours. The antibiotic we used in inJanus-m and sJanus-m is kanamycin, and for sJanus is ampicillin. Then we used 500μM IPTG to induce in 37℃ for 4h.
Figure 13. Pre-expression. We successfully expressed sJanus, sJanus-m and inJanus-m.
Expression condition:
1. Mix the supernatant after centrifugation with the GST medium and incubate them in the shaker for 1~2h.
2. Add the intermixture to the column. Collect the flow-through and add it to the column again. (You can repeat the procedure for 2~3 times if necessary.)
3. Wash the column by 20ml 1xPBS for three times (more if necessary).
4. Add Prescission Protease to the column (the mass ratio of PP and your protein is 1:10). Then incubate them in the shaker overnight.
5. Collect the flow-through which is your sample. Purify your product by using UPLC.
6. Wash the column by GSH and recover the column.
Neo-Protein Design
Pre-expression:
The bacteria were cultured in 5mL LB liquid medium with kanamycin in 37℃ overnight. After taking samples, we transfer them into 1L LB medium with kanamycin.
Cultured in bottles:
After 4 hours culturing in 37℃ in bottles, we used 500μM IPTG induced in 16℃ for 8-12h. We used McAc 0 and 20 to wash off the bulk proteins, and used McAc 200 to wash off aimed proteins.
 
Figure 1. The expression of inJanus-m
We finally can use McAc 0 and 20 to wash off the bulk proteins and use McAc 200 to wash off aimed proteins.
Figure 2. The expression of sJanus-m
The concentration of IPTG, temperature and time we used in sJanus-m’s expression are same to those of inJanus-m, except that we used TB liquid culture medium rather than LB to raise bacteria to get more proteins. 
We also expressed sJanus successfully in E.coli (data not provided). And Nankai helped us to express inJanus in pichia pastoris, which you could see in detail in (COLLABORATIONS) .
Stimulated Plastic Enzymolysis
1.   About FsC
Protein pre-expression FsCUse pET-28a as expression vector and pre-express FsC, FsC-sJanus, FsC- sJanus-m in BL21. Analyze the result of SDS-PAGE to infer whether protein FsC, FsC-sJanus, FsC-sJanus-m express after induced by IPTG.
Figure 3. The results of SDS-PAGE. M is Protein marker. a is sample of FsC which is non-induced. b is sample of FsC which is induced.
We can tell the difference (in red circle) between non-induced and induced. The amount of protein after induced is larger than non-induced and molecular weight is about 25kDa which is target protein’s molecular weight. So we infer that FsC can express in E.coli BL21.
·Inoculation of LB-media with BL21_ FsC _pET-28a; Overnight incubation at 37°C
·Dump cultivated bacteria into 1L liquid LB medium. Cultivate for 4h at 37°C. Add 1M IPTG 500μL and induce at 16°C for 15h.
·Centrifugation of 1L LB medium at 4000rpm for 20min and discard supernatant.
·Crush bacteria with High Pressure Homogenizer.
·Centrifugation at 18000rpm for 30min.
·Purify protein with nickel column 
Figure 4. The result of protein BL21_FsC expression. M is Protein marker. a is sample of FsC in BL21 which is non-induced. b is sample of FsC in BL21 which is induced. c is sample of supernatant after breaking bacteria and centrifugation d is sample of sediment after breaking bacteria and centrifugation. e if sample of liquid after filtration by Ni column. f is sample of media after filtration by Ni column. g is sample of liquid after removing impurity with 50mM MCAC. h is sample of media after removing impurity with 50mM MCAC. i is sample of target protein after washing with 200mM MCAC. j is sample of media after washing with 200mM MCAC.
There is still a little protein combine with media after washing with 200mM MCAC.
Nanodrop measurements of the purified protein FsC: 0.414mg/mL
·Concentrate FsC with concentration tube and change the buffer from 200mM MCAC to 1X PBS because 200mM MCAC is noxious for enzyme. Final concentration is 2.870mg/mL
 
Inoculate of 1L liquid LB-media with C+_FsC-sJanus_pET-28a, C+_FsC-sJanus-m_pET-28a for 4h. Add 1M IPTG 500μL and induce at 16°C for 15h. Centrifuge of 1L LB medium at 4000rpm for 20min and discard supernatant. Crush bacteria with High Pressure Homogenizer and centrifuge at 18000rpm for 30min. Then purify protein with nickel column and we get the FsC-sJanus and FsC-sJanus-m.
 
Figure 5. The result of protein C+_FsC-sJanus expression. M is Protein marker. a is sample of FsC-sJanus in C+ which is non-induced. b is sample of FsC-sJanus in C+ which is induced. c is sample of sediment after breaking bacteria and centrifugation. d if sample of liquid after filtration by Ni column. e is sample of media after filtration by Ni column. f is sample of liquid after removing impurity with 50mM MCAC. g is sample of media after removing impurity with 50mM MCAC. h is sample of liquid after washing with 200mM MCAC. i is sample of media after washing with 200mM MCAC. j is sample of target protein after washing with 200mM MCAC.
Figure 6. The result of protein C+_FsC-sJanus-m expression. M is Protein marker. a is sample of FsC-sJanus-m in C+ which is non-induced. b is sample of FsC-sJanus-m in C+ which is induced. c is sample of sediment after breaking bacteria and centrifugation. d if sample of liquid after filtration by Ni column. e is sample of media after filtration by Ni column. f is sample of liquid after removing impurity with 50mM MCAC. g is sample of media after removing impurity with 50mM MCAC. h is sample of liquid after washing with 200mM MCAC. i is sample of media after washing with 200mM MCAC. j is sample of target protein after washing with 200mM MCAC.
2.   About Thc_Cut1
Protein pre-expression experimental conditions:
1.   Transferred our correct plasmid into escherichia   coli BL21 (DE3).
2.   Select bacterial colony and add into LB, incubate at 37 centigrade for 7h. Add 4μl of IPTG to induce the expression of protein.
3.   Incubate at 37 centigrade for 4h. The bacterial solution’s OD is 0.6-0.8.
Figure 7. The result of protein Thc_Cut1 pre-expression. 1: bacterial No.1 non-incubated; 1’: bacterial No.1 incubated; 2: bacterial No.2 non-incubated; 2’: bacterial No.2 incubated; 3: bacterial No.3 non-incubated; 3’: bacterial No.3 incubated
Figure 8. The result of protein Thc_Cut1-sJanus-m pre-expression. Labels are the same as before.
Figure 9. The result of protein Thc_Cut1-sJanus pre-expression. Labels are the same as before.
Finally, the protein was expressed successfully.
We added 5μl of the bacterial restored into the LB containing 5μl of ampicillin to the final concentration of 1mM. And then, we incubated them in the shaker at the temperature of 37 centigrade, working for 14-16 hours.
Experimental conditions as follow：
1. Incubate at 37 centigrade, until OD ranges from 0.6-0.8(4-5h).
2. Incubate at 4 centigrade, 220rpm, for 30mins.
3. Add 1mL IPTG to the final concentration of 1mM.
4. Incubate at 16 centigrade for 12-16h.
The purification of protein:
We used eppendorf to make bacterial deposited, working at 4000rpm for 20min. Use 15mL MCAC0 to suspend bacterial.
After resuspending, we used high pressure to break the cells.
In order to get the recombinant protein with the higher purity, the recombinant protein was purified through Ni-chelating affinity chromatography.
The results are as follow:
Figure 10. The result of protein Thc_Cut1 expression. 1: Sample non-induced; 2: Sample induced; 3: Sample of cytoplasm deposited; 4: Sample which outflowed from Ni column combined with supernatant; 5:Sample of NIi medium combined with supernatant; 6: Sample washed by MCAC20; 7: Sample resuspended by MCAC30; 8: Sample washed by MCAC30; 9: Sample resuspended by MCAC50; 10: Sample washed by MCAC50; 11: Sample resuspended by MCAC100; 12: Sample washed by MCAC100; 13: Sample resuspended by MCAC200; 14: Sample washed by MCAC200; 15: Sample resuspended by MCAC500; 16: Sample washed by MCAC500; 17: Sample resuspended by MCAC1000.
Figure 11. The result of protein Thc_Cut1-sJanus expression. 1: Sample non-induced; 2: Sample induced; 3: Sample of cytoplasm deposited; 4: Sample of supernatant; 5: Sample of NIi medium combined with supernatant; 6: Sample which outflowed from Ni column combined with supernatant; 7: Sample washed by MCAC20; 8: Sample resuspended by MCAC30; 9: Sample washed by MCAC30; 10: Sample resuspended by MCAC50; 11: Sample washed by MCAC50; 12: Sample resuspended by MCAC100
Figure 12. The result of protein Thc_Cut1-sJanus expression.13: Sample washed by MCAC100; 14: Sample resuspended by MCAC200; 15: Sample washed by MCAC200; 16: Sample resuspended by MCAC500; 17: Sample washed by MCAC500; 18: Sample resuspended by MCAC1000.
Figure 13. The result of protein Thc_Cut1-sJanus-m expression. 1: Sample non-induced; 2: Sample induced; 3: Sample of cytoplasm deposited; 4: Sample which outflowed from Ni column combined with supernatant; 5: Sample of NIi medium combined with supernatant; 6: Sample washed by MCAC20; 7: Sample resuspended by MCAC30; 8: Sample washed by MCAC30; 9: Sample resuspended by MCAC50; 10: Sample washed by MCAC50; 11: Sample resuspended by MCAC100; 12: Sample washed by MCAC100; 13: Sample resuspended by MCAC200; 14: Sample washed by MCAC200; 15: Sample resuspended by MCAC500; 16: Sample washed by MCAC500; 17: Sample resuspended by MCAC1000
Protein Extraction Kit
Besides cutinase-Janus fusion proteins, we also expressed fluorescent protein- Janus fusion for this project.
1.   About GFP
GFP-sJanus:
Figure 14. The result of protein GFP-sJanus expression. In this picture of gel, we can see that when induced by IPTG, the gene of GFP-sJanus expressed. The correspond place has been underlined by red line.
Figure 15. The result of protein BL21_GFP-sJanus expression. H is Protein marker. G is sample of liquid after filtration by Ni column. F is sample of media after removing impurity with 20mM MCAC.E is sample of liquid after removing impurity with 20mM MCAC. D is sample of media after removing impurity with 50mM MCAC. C is sample of liquid after removing impurity with 50mM MCAC. B is sample of media after removing impurity with 100mM MCAC.A is sample of liquid after removing impurity with 100mM MCAC.
Figure 16. The result of protein BL21_GFP-sJanus expression. A is Protein marker. B is sample of media after washing with 100mM MCAC. C is sample of liquid after washing with 100mM MCAC.  D is sample of media after washing with 200mM MCAC. E is sample of target protein after washing with 200mM MCAC. F is sample of media after washing with 500mM MCAC. G is sample of target protein after washing with 500mM MCAC. H is sample of target protein after washing with 1000mM MCAC.
After that we use the target protein concentrating, we got our target protein GFP-sJanus, whose concentration is 50.1mg/ml.
GFP-sJanus-m:
Figure 17. The result of protein GFP-sJanus-m expression. In this picture of gel, we can see that when induced by IPTG, the gene of GEP-sJanus-m expressed. The correspond place has been underlined by red line.
Figure 18. The result of protein BL21_GFP-sJanus-m expression. G is sample of liquid after filtration by Ni column. F is sample of media after removing impurity with 20mM MCAC.E is sample of liquid after removing impurity with 20mM MCAC. D is sample of media after removing impurity with 50mM MCAC. C is sample of liquid after removing impurity with 50mM MCAC. B is sample of media after removing impurity with 100mM MCAC.A is sample of liquid after removing impurity with 100mM MCAC.
Figure 19.The result of protein BL21_GFP-sJanus-m expression. A is Protein marker. B is sample of media after washing with 100mM MCAC. C is sample of liquid after washing with 100mM MCAC.  D is sample of media after washing with 200mM MCAC. E is sample of target protein after washing with 200mM MCAC. F is sample of media after washing with 500mM MCAC. G is sample of target protein after washing with 500mM MCAC. H is sample of target protein after washing with 1000mM MCAC.
After that we use the target protein concentrating, we got our target protein GFP-sJanus-m, whose concentration is 26.7mg/ml.
We also expressed GFP-inJanus and GFP-inJanus-m fusion protein (data not provided).
2.   About BFP
We insert the plasmid into competent C+ to make the fusion protein secreted.
Express the fusion protein:
5ml tube with C+
37℃   15h
1L culture bottle
37℃   4h
induced by IPTG
500μl  16℃ 12-16h
Collect the bacterial
Break the bacterial
Purify the fusion protein:
0mM McAc
Resuspension
10mM McAc
Wash the impurity   away
20mM McAc
Wash the impurity   away
30mM McAc
Wash the impurity   away
50mM McAc
Wash the impurity   away
200mM McAc
Elute
Figure 20. BFP- sJanus: The target protein is about 40kDa. As is shown in the picture, the protein is purified through Ni-chelating affinity chromatography.
Figure 21. BFP- sJanus-m: The target protein is about 40kDa. As is shown in the picture, the protein is purified through Ni-chelating affinity chromatography.
3. About RFP
Transforming:
Transformed the expression vectors(concentration:121.7ng/mL) we had already built before (pET-28a) into the Escherichia coli BL21, then incubated the bacteria for 12h (37℃).
Pre-expression:
Picked up the single bacteria colony on the culture dish by a tip and put it into a 5mL LB liquid medium, incubated in the shaking table for 6h (37℃), took 400μL from it for Glycerol Stocking (-80℃) and took 200μL centrifuge and the supernatant was discarded, 20μL ddH20 re-suspended and then made sample for glue leaking, added 5μL IPTG into the rest and incubated for 4h, took 200μL for centrifuging and the supernatant was discarded, 20μL ddH20 re-suspended and then made sample for glue leaking. The leaking result is as followed.
Figure 22. The result shows that after induction the bacteria expressed obviously large amount of the target protein (length:30~40)
Expression
Added 5μL Glycerin bacteria into a 5mL LB liquid medium and incubated about 15h, then added them into the 1L LB liquid medium, incubated under 37℃ until the OD was 0.6~0.8(about 4h), then cooling the atmosphere to 4℃ and incubated 0.5h, added 1mL IPTG into the flask and incubated for 16h under 16℃.
Figure 23. The expression of RFP fusion protein. The color is very obvious
Purification:
Centrifuged the mixture 4000rpm for 30min, then re-suspended the sediment by 15mL McAc. Used the ultrasonication to destroy the bacteria, then centrifuged 18000rpm for 30min and purred the supernatant by Ni-NTA, took the sample of the sediment; mixture, McAc 20, 30, 50, 100, 200, 500 which was combined with medium; misture, McAc 20, 30, 50, 100, 200 which leaked out of the Ni-NTA. Then made sample for glue leaking. The leaking result is as follow.
Figure 24.According to the result, the best removing impure concentration is 30, the best elution concentration is 200, and we got the pure protein after McAc 200 elution.
 
Super Protein Chip
We use hydrophobins (inJanus and sJanus) to conduct the experiments with three kinds of antigens (CEA, AFP and CA15-3) and their antibodies. The results were as follows.
The optimal concentration of Janus is 200μg/ml.
The optimal concentration of antigens (CEA, AFP and CA15-3) is 2.5μg/ml.
The optimal concentration of antibodies is 5μg/ml.
Figure 1. Hydrophobins sJanus show good effects of antibodies fixing. Here we use antibodies with green fluorescence tag as example. The spots with strong fluorescence represent successful binding between capture antibody (probe) and antigen (tumor marker).
Figure 2. Hydrophobins inJanus tend to show weak fluorescence signal. After compared experiments, we discovered that increasing the incubation time helps to improving detecting sensitivity.
Project Achievements
1. We introduced a new method of protein chip substrate modification by using Janus, and verified its feasibility.
2. We confirmed the optimal concentration of antigens and antibodies in the detection of tumor markers, which helps to save materials to the most extent and keep the reliability of the results at the same time.
3. The experiments were conducted with different kinds of antigens and antibodies, ensuring the universality of the tumor detection chip under a certain range.
Process of Optimization
1. In the beginning, due to our inexperience, the concentration of antigens and antibodies was too high, which leads to excessive strong fluorescence signal intensity.
At first we thought it was caused by the improperly operation of microscope, but after literature reviewing and analyzing, we tried lowering the concentration about four times and achieved good results.
Figure 3. Here is the excessive strong fluorescence signal intensity caused by high concentration.
Figure 4. Good results were achieved after lowering the concentration.
2. In mid-stage, many crystallized structures appeared in the protein spot, and hence influenced the image quality and observation results. To solve this problem, we consulted large quantities of data and information, and discovered a similar case arising when solutions of antibodies and antigens were stored for too long, leading to degeneration. So we reformulated all the solutions we need, and crystallization disappeared. From this we learned that reagents should be used right after they were ready. In this case, another possible cause was that we used TBST in the last washing step, which may also lead to crystallization. In the upper course of experiments we used water instead of TBST.
Figure 5. Crystallized structures appeared.
Figure 6. Crystallized structures disappeared after reformulating all the solutions we need.
 
Stimulated Plastic Enzymolysis
We used Cut1, FsC, Cut1-sJanus-m(fusion protein) and Janus(sJanus&inJanus) we expressed to conduct PET hydrolysis experiments. We achieved a lot and met some problems as well.
Pre-experiment
Overview
We have found out appropriate concentration of the cutinase we made and suitable pH for our enzyme Thc-Cut1, the right time to detect the product of hydrolysis of PET.
Result
We use absorbance in 600 nm to measure the turbidity of the liquid in tube. We tried different concentration of enzyme with one piece of plastic(3*3*0.25mm), decided to use 2mg/ml to conduct our experiment afterwards.
Figure 1. This curve described the OD600nm after 3h’s reaction changes through the concentration of Thc_Cut1. We can see clearly that at 2mg/ml ,the curve reaches a peak, at which concentration we will compare the hydrolysis effect.
We also conducted our system in various pH, found that the activity of Thc_Cut1 doesn’t change much in different pH, so we chose pH 7.0 as one of our experimental condition.
Figure 2.  The curve shows the absorbance at 600nm changes along with pH. The activity of enzyme doesn’t change much in various pH.
In order to find right time to detect, we detect OD600nm in several time, here’s the result.
Figure 3.  The line clearly indicates that the hydrolysis rate rises along with time.
Considering we need an appropriate detect time to measure the hydrolysis effect, we set our detect time at 3h.
Main experiment
Overview
We discovered the right temperature to conduct various way to mix up our sJanus (or inJanus) and Thc_Cut1. And we have found out the best way to increase the hydrolysis rate.
Result
We conduct our trail in different temperature, realizing 50℃ is the proper temp for the enzyme and our Janus to combine.  
Figure 4. Different colors represent different ways of mix. Thc_Cut1+Janus means enzyme had incubated with Janus for 16h before the plastic was added into the system. And Plastic+Janus means plastic had incubated with Janus before adding enzyme. And we also conduct these incubation separately in 4℃,37℃.50℃, values at 50 degree are higher than that under the other temperature.
Based on the experiment above, we compared these two different ways with simple mix, here’s the result.
Figure 5. We can conclude that pre-incubation does increase the hydrolysis rate, some ways(Plastic  pre-incubated with sJanus) even boost 200%+ react rate comparing with Thc_cutL1* in our conditions!! Another interesting phenomenon is sJanus usually works better than inJanus, and the reason behind it need our further exploration. On the whole, Our Janus works a lot!
Additional experiment
Overview
To test the universality of conclusion above, we used another cutinase FsC we expressed. We tried incubated plastic with Janus and FsC with Janus but data shows differently comparing with our expectation. In the contrary cutinase without incubation performed better. We are still working on the explanation.
Result
After incubation (PET&sJanus, PET& inJanus, FsC&sJanus, FsC&inJanus) for 16h, we added FsC or PET piece into EP tube. A group of tube without incubation were also set as control.
Figure.6 Different columns mean different ways of incubation. Control means incubating PET in the buffer system without Janus before adding  enzyme or incubating FsC in buffer system before adding Plastic.
As the picture shows, after incubation for 16h sJanus performed better than inJanus. That is in accord with conclusion we made above. Unfortunately, the control group had a higher value. The reliability of this result needs our repeat and confirmation and the explanation behind it still need our further exploration. We are working on it.
Further exploration
Overview 
We successfully build a fusion protein which attached cutinase Thc_Cut1 to our Janus (sJanus-m).
The fusion protein has much more better performance than the cutinase before.
Result 
The procedure in this part is likely to the pre-experiment, we use different concentration of fusion protein. And we compare the data to the value we conduct with Thc_Cut1 before, found that fusion protein hydrolysis more, especially in high concentration.
Figure 7.  These  curves compare fusion proteins’ performance with normal cutinase in different concentration. We can see that the fusion protein Cut1-sJanus-m perform  much better when concentration are over 1mg/ml.
We also conduct our experiments in different pH by using different Tris-HCl. Just as picture shows, the activity of our fusion protein doesn’t change a lot in different pH which is similar with Thc_Cut1.
Figure 8. These two proteins are stable when pH changes, especially when pH is between 6.0 and 9.0. And the concentration of Thc_Cut1 and Thc_Cut1-sJanus-m are 0.2mg/ml.
We drew a curve about detect time and compared it with Thc_Cut1, we can see the hydrolysis rate rose sharply after 5h,while Thc_Cut1 still in the low level. Which represent our Janus works well!
And our fusion protein does improve a lot!
Figure 9. We detect absorbance of liquid every 3h, red line refers to our fusion protein, and the blue line is former enzyme. The performance of cutinase are highly improved when fuse to the sJanus-m.
Figure 10 &11. The picture shows 5mg/ml enzyme have react with one piece of plastic（3*3*0.25mm）for 3h, before and after centrifugation. We can see clearly in the pic. After 3hs the tube of Thc_Cut1-sJanus-m had much more sediment than Thc_Cut1.
Conclusion
What we solved?
1.   We successfully expressed cutinase (Fsc, Cut1) in E.coli.
2.   We structured fusion protein connecting Cut1 with sJanus-m, and expressed it successfully in E.coli, which has extraordinary performance as our data shows.
3.   We successfully find out optimal conditions (pH, temperature, concentration) of PET hydrolysis by cutinase(Cut1,FsC). 
4.   We use different mix method to verify our Janus (sJanus&inJanus) can promote hydrolysis rate. And we confirmed the best way is incubating PET with Janus before hydrolysis.
5.   We found out after incubation, sJanus usually performed better than inJanus. 
What we unsolved?
1.   When we conducted our experiments with FsC, the data shows FsC without incubation with Janus performed better, which is contrary to our conclusions before.
2.   We structured many fusion protein DNA, and haven’t expressed them all successfully. 
Protein Extraction Kit
Background
ATPS (aqueous two-phase systems) is a novel technology to purify proteins. The mechanism of this system lies in the partition between two different phases.
Aims
Construct a brand new and standard way to purify proteins based on aqueous two-phase system.
Results
1. Confirm the strengths using aqueous two-phase systems
2. Successfully separate the target proteins from bulk protein phase based on aqueous two-phase systems at a high partition rate.
3. Construct a standard protocol to separate different kinds of proteins.
The experiment of ATPS
Pre-experiment
Process of this experiment
In the pre-experiment, the original concentration of our protein is about 50ug/ml, which volume is 200uL. We designed the pre-experiment just in order to make preparations for our next experiment. We added 5% (v/v) Berol 532 to protein solution. And then, we used shaker to make them mixed at the speed of 250r/min and 20 centigrade working about one hour. Centrifuge was used to make them separated and came into being two phases at the speed of 8000g for about 25min. In these two phases, the upper phase is rich-detergent and the lower phase is depleted-detergent phase. Because of the property of hydrophobin, fusion protein will stay in the detergent phase, and bulk protein stay in the water phase. We put the rich-detergent phase in another centrifuge cubes and added butanol which is 5 times volume of detergent. Centrifuge was used to make them separated and finally in the lower phase (water phase), we got pure target protein. The upper phase contains detergent (Berol 532), which can be recycled.
Reagents used in this experiment
Concentration(final)
Volume
Berol 532
Purity 96%
10uL (5%)
Protein solution
50ng/mL 100ng/mL 3mg/mL
200uL
 
Results of this experiment
Figure 1. In this picture of gel, the third lane is the original protein solution and the forth is the protein solution which used ATPS to purify. We can see that some bulk proteins have been removed. This fusion protein is GFP-sJanus. The first and second lane are GFP-sJanus, because we didn’t dilute the protein’s solution, the concentration of protein is so high (about 3mg/mL) that they overflowed. The next is 100ug/mL and 50ug/mL.
Figure 2. We used Bandscan5.0 to analysis the gel. We can see that other proteins have been removed all in the third and fourth lane. Meanwhile, we can see that other proteins have been removed to some degree in the fifth and sixth lane.
We calculated the efficiency of this experiment. It is 18.93% in the third and fourth lane and 46.17% in the fifth and sixth lane. The efficiency is a bit low and we analyzed the reasons. We added 10uL of Berol 532 in this system to extract the target protein. However, the total amount of protein is so large that our detergent can’t extract them all.
Formal experiment using Berol 532 as detergent and MCAC0 as buffer
Process of this experiment
In our first formal experiment, we changed some conditions. We increased the speed of shaker to 300r/min in order to make them mixed totally. Meanwhile, in the pre-experiment, we found that we didn’t need to use centrifuge to make them separated because they can divided into two phases automatically in few seconds at the room temperature(Berol 532, article says that its low solubility does not allow cloud point measurement). We used 20% (v/v) Berol to extract our fusion proteins. If less detergent used in this system, it will increase the final concentration of proteins, but it will decrease the efficiency of extraction because detergent is lacking in combing the fusion protein.
The volume of our system is about 5mL (We used original solution which got by centrifuge and high pressure after suspending by MCAC0). We added 1mL Berol 532(20% w/w) in our original solution. And then, we used shaker to make them mixed at the speed of 300r/min and 20 centigrade working about one hour. We put the tube at the room temperature and the mixture was separated into two phases in few seconds. We got the upper phase (2mL) and added water saturated butanol (5mL). Then we used centrifuge to make them separated at 4 centigrade (In this temperature, we can prevent protein from denaturation for a little longer time) at the speed of 3500rpm (8000g and 3500rpm both worked) working for 10 minutes. We got the lower phase, and it contains our target protein. We did this twice in order to increase the purity of our target proteins. We used technology of SDS-PAGE to test if this system works.
Reagents used in this experiment
Concentration(final)
Volume
Berol 532
Purity 96%
1mL (20%)
Protein solution
Not detect by machine, but we can analyze by gel.
5mL
Results of this experiment
Figure 3. In this picture of gel, the first lane is the original solution of protein. We can see that it contains many other proteins. The second lane is the lower phase (water phase), we can see that the target protein have been removed and some bulk proteins stay in the water phase. The third and fourth lane are the water phase got in the reverse extraction. We can see that the target proteins have been concentrated and some other proteins have been removed.
Figure 4. This picture was shot in ultraviolet, we can see clearly that our target protein stay in the detergent phase because of the blue fluorescence.
Figure 5. We used Bandscan5.0 to analysis the gel. We can see that target proteins have been purified by ATPS. In this process, other proteins stay in the water phase and our target protein stay in the detergent phase. When extracted by saturated butanol, target protein was concentrated in the water phase. The picture of gel shows that when used it twice, the purity will increase again
We calculated the efficiency of this experiment. It is 67.54% in the twice and fourth lane and 75.06% in the fourth and fifth lane. The efficiency of it is high to some degree, but we want get a higher separation rate and increase the purity of our target protein. In our next experiment, we used many buffer and different detergent to achieve it.
Formal experiment using Berol 532 as detergent and HAc/NaAc as buffer
The process of this experiment
In this experiment, the fusion proteins we used are GFP-sJanus and BFP-sJanus. We added 2%, 5% of Berol 532 in GFP-inJanus and 2%, 5% (v/v) of Berol 532 in BFP-sJanus. We added 200uL HAc/NaAc(pH=7.5) to 4800uL protein solution. And then, we used shaker to make them mixed at the speed of 300r/min and 20 centigrade working about one hour. We added butanol which is 5 times volume of detergent to do reverse extraction. Centrifuge was used to make them separated and finally in the lower phase (water phase), we got pure target protein.
Reagents used in this experiment
Concentration(final)
Volume
Berol 532
Purity 96%
100uL(2%) 250uL(5%)
Protein solution
(GFP-sJanus BFP-sJanus)
Not detect by machine, but we can analyze by gel.
200uL protein solution+4800uL buffer
Results of this experiment
Figure 6. In this picture of gel, the first lane is marker. The second lane is the original solution of GFP-sJanus. The third lane is the system adding 2% and the fourth is that adding 5%. We can see clearly that other proteins have been removed. Unfortunately, because of some unknown reasons, there isn’t any line about BFP-sJanus in this gel.
Figure 7. We used Bandscan5.0 to analysis the gel. The percentage of our target protein in total proteins is 11.5% before separating. It increased to 59.1% in the third lane(2% Berol 532) and 88.7% in the fourth lane(5% Berol 532). Many other proteins have been removed.
Figure 8. We can see the green florescence in the upper phase. This picture was shot in ultraviolet.
We did the second group of experiment. In this experiment, the concentration of protein solution and volume of Berol 532 changed. The total volume of protein solution is 6mL. The conditions doing this experiment is same as above.
Reagents used in this experiment
Concentration(final)
Volume
Berol 532
Purity 96%
120uL (2%) 300uL (5%)  
Protein solution
(GFP-sJanus)
GFP-sJanus   (0.794mg/mL)
100uL protein   solution+5900uL buffer
Results of this experiment
Figure 9. In this picture of gel, the first lane is original protein solution. The second and third lane are proteins separated by ATPS. The former is 2% of Berol 532; the latter is 5% of Berol 532. We can see that the system added 5% of Berol 532 get the best result.
Figure 10. We used Bandscan5.0 to analysis the gel. The percentage of our target protein in total proteins is 26.4% before separating. It increased to 74.6% in the third lane(5% Berol 532). Many other proteins have been removed. However, there isn’t any line detected in the second one.
We did the third group of experiment. In this experiment, the protein we used is RFP-sJanusm. The conditions doing this experiment is same as the second group.
Reagents used in this experiment
Concentration(final)
Volume
Berol 532
Purity 96%
120uL(2%) 300uL(5%)
Protein solution
(RFP-sJanusm)
RFP-sJanusm (0.702mg/mL)
100uL protein solution+5900uL buffer
Results of this experiment
Figure 11. In this picture of gel, the first lane is original protein solution of RFP-sJanusm. The second and third lane are proteins separated by ATPS. The former is 2% of Berol 532; the latter is 5% of Berol 532. We can see that the system added 5% of Berol 532 get the best result.
Figure 12. We used Bandscan5.0 to analysis the gel. The percentage of our target protein in total proteins is 19.5% before separating. It increased to 78.6% in the second lane(2% Berol 532) and 60.9% in the fourth lane(5% Berol 532). Many other proteins have been removed. This experiment confirmed that RFP-sJanusm does work.
We did the fourth group of experiment. In this experiment, we used GFP-sJanus, BFP-sJanus, RFP-sJanus and RFP-sJanusm to test this system. The concentration of original protein solution was detected by BCA measurement. The conditions doing this experiment is same as the second group.
Figure 13. We did many groups of experiment.
Reagents used in this experiment
Concentration(final)
Volume
Berol 532
Purity 96%
100uL(2%) 250uL(5%)
Protein solution
(GFP-sJanus, BFP-sJanus,   RFP-isJanus, RFP-sJanusm)
BFP-sJanus(1.013mg/mL)
GFP-sJanus(0.567mg/mL)
RFP-sJanus(0.583mg/mL)
RFP-sJanusm(0.475mg/mL)
50uL protein   solution+5950uL buffer
 
Results of this experiment
Figure 14. The experiment used BFP-sJanus failed but other groups succeeded. We can see that 5%(v/v)of Berol 532 is suitable for all groups. After calculating, we found that the efficiency of systems adding 5% of Berol 532 are the highest. It’s correspond to the results measured by BCA.
We did the fifth group of experiment. In this experiment, we used Cut1-sJanus to test this system. The concentration of original protein solution is 50ng/mL. The conditions doing this experiment is same as the second group.
Reagents used in this experiment
Concentration(final)
Volume
Berol 532
Purity 96%
100uL(2%) 250uL(5%)
500uL(10%)
Protein solution
50ng/mL
5mL
Results of this experiment
Figure 15. We used Bandscan5.0 to analysis the gel. We can see clearly that the protein was purified (3 and 6, 4 and 7). Other proteins were removed totally and we get the pure target protein. In this experiment, we used protein having been purified by Nickle chromatography, so the purity of our target protein is high before separating.
Formal experiment using Berol 532 as detergent and HEPES(pH=7.3) as buffer.
We changed buffer and do this experiment again. Unfortunately, they all failed.
Formal experiment using Berol 532 as detergent and Reppal PES100 as polymer.
It has been reported that polymer can increase the efficiency of extraction and the system can be divided into two phases automatically without above cloud point. We used Reppal PES100 as polymer to do this experiment. However, it doesn’t work.
Formal experiment using Triton X-100 as detergent and HAc/NaAc as buffer.
The cloud point of Triton X-100 is 64-65 centigrade. We changed detergent and did this experiment again. We put the tubes in the heat block to make system separated. However, there aren’t any florescence in the detergent phase (the lower phase).
Formal experiment using Triton X-114 as detergent and HAc/NaAc as buffer.
The cloud point of Triton X-114 is 22 centigrade. We changed detergent and did this experiment again. We made the system separated at the room temperature, and used centrifuge to get a better separation. However, there aren’t any florescence in the detergent phase (the lower phase).
Summary
We used fusion proteins which are GFP-sJanus, GFP-sJanus-m, BFP-sJanus, BFP-sJanus-m, RFP-sJanus, RFP-sJanus-m, Cut1-sJanus, Cut1-sJanus-m to conduct this experiment. The results are as follows.
1.   The optimal volume of Berol 532 is 5% (v/v) of protein solution and temperature is 20 centigrade.
2.   The process of shaking is important. Generally, the speed of 300r/min working for 1h does work.
3.   The concentration of protein solution should be low.
4.   The optimal buffer is HAc/NaAc(pH=7.5).
In this experiment, we confirmed the strengths using aqueous two-phase systems and successfully separate the target proteins from bulk protein phase based on ATPS at a high partition rate. The most important is that we constructed a standard protocol to separate different kinds of proteins.
Design
This year, we focus on solving the protein purification problem in biological lab. As we talked about in the Protein Extraction Kit page in project description, we aim to replace the lengthy and expensive chromatography with Janus-based aqueous two-phase system. Our design shows many clever points.
Above all, we take advantage of synthetic biology, using the core of iGEM- BioBrick and RFC Assembly Method , to build a universal purification tag. This tag includes a linker containing TEV protease site and sJanus, designed by the RFC23 method which is specific for fusion protein. Thus, if other teams want to use this system to express and purify their proteins, they could use the standard assembly to make a fusion protein.
Secondly, we explore for the best detergent, the best concentration of detergent and the best buffer solution. For example, we tried Berol 532 and Triton-114 as the detergent, and used 2%, 5%, 10%, 20% as the gradient of concentration. We also tried PBS, Tris-HCl and many other buffer solutions. Thirdly, we construct many fusion proteins, including three kinds of fluorescent proteins (GFP, BFP and RFP) with four kinds of Janus, which are inJanus, sJanus and their respective mutants. We also construct experiments about fusion proteins with cutinases Thc_Cut1, FsC and LC to test if this system could be used on the purification of enzymes. After back-extraction, we run SDS-PAGE and calculate the efficiency.
Then we use these data to run our model . To find the universal experiment conditions, we designed a mighty-possible way to do prediction. The first thing to be made clear is that, it is the presence of difference in protein that will affect the effect of extraction most on the first way and then we have to change our experiment conditions to achieve better results. The second thing to know is that, among all properties of protein, the molecular weight, isoelectric point, hydrophobic property is the most possible impact factors, so we build a universal model to find the feasible extraction conditions, such as what concentration we should choose, or how much volume should we add…by choosing the properties of protein as main variables, and the conditions as dependent variables, using the best condition we got from our experiment and model and the protein’s properties as interpolation point to do interpolation, through this way we can gain the formula suits all proteins as long as we know their properties.
The results of our experiment could be found in results of Protein Extraction Kit..
Figure 1. The principle of Protein Extraction Kit
Protocols
Molecular Cloning
1.   Take hydrophobin standardization for instance
Transformation:
1) Start thawing the competent cells on ice.
2) Add 50 µL of thawed competent cells into pre-chilled 2ml tube, and another 50µL into a 2ml tube, labelled for your control.
3) Add 1 - 2 µL of the re-suspended DNA to the 2ml tube. Pipet up and down a few times, gently. Make sure to keep the competent cells on ice.
4) Add 1 µL of the RFP Control to your control transformation.
5) Close tubes and incubate the cells on ice for 30 minutes.
6) Heat shock the cells by immersion in a pre-heated water bath at 42ºC for 60 seconds.
7) Incubate the cells on ice for 5 minutes.
8) Add 600 μL of LB media (make sure that the broth does not contain antibiotics and is not contaminated) to each transformation
9) Incubate the cells at 37ºC for 45min while the tubes are rotating or shaking. 
10) Label a petri dishes with LB agar and the appropriate antibiotic(s) with the part number, plasmid backbone, and antibiotic resistance. Plate 100 µl of the transformation onto the dishes, and spread. This ensures that you will be able to pick out a single colony.
11) For the control, label one petri dishes with LB agar (AMP). Plate 100 µl of the transformation onto the dishes, and spread.
12) Incubate the plates at 37ºC for 12-14 hours, making sure the agar side of the plate is up. If incubated for too long the antibiotics start to break down and un-transformed cells will begin to grow. This is especially true for ampicillin - because the resistance enzyme is excreted by the bacteria, and inactivates the antibiotic outside of the bacteria.
13) You can pick a single colony, make a glycerol stock, grow up a cell culture and  miniprep .
Strain Reservation:
In the clean bench, using Micro pipette tip pick a single colony and put it into a LB liquid medium. Put into shaking table 10-12 h. In a clean centrifuge tube, add 200uL bacterium liquid and 300uL glycerin, then put into -20℃ refrigerator.
Plasmid Extraction:
In the cell culture in a centrifuge to pellet the cells, empty the supernatant (media) into a waste collection container.
1) Re-suspend pelleted bacterial cells in 500 µl Buffer P1 BL (kept at 4 °C) and transfer to a micro-centrifuge tube. No cell clumps should be visible after resuspension of the pellet. Important: Ensure that RNase A has been added to Buffer P1.
2) Add 250 μl Buffer P2 and gently invert the tube 4–6 times to mix. Do not vortex, as this will result in shearing of genomic DNA. If necessary, continue inverting the tube until the solution becomes viscous and slightly clear. Do not allow the lysis reaction to proceed for more than 5 min.
3) Add 350 μl Buffer N3 and invert the tube immediately and gently 4–6 times. To avoid localized precipitation, mix the solution gently but thoroughly, immediately after addition of Buffer N3. The solution should become cloudy.
4) Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top micro-centrifuge. A white pellet will form. Apply the supernatants from step 4 to the QIAprep spin column by decanting or pipetting.
5) Centrifuge for 30–60 s. Discard the flow-through (Optional): Wash the QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30–60 s. Discard the flow-through.
6) Wash QIAprep spin column by adding 0.75 ml Buffer PE and centrifuging for 30–60 s. Spin for 60 seconds produces good results.
7) Discard the flow-through, and centrifuge for an additional 1 min to remove residual wash buffer.
IMPORTANT: Residual wash buffer will not be completely removed unless the flow-through is discarded before this additional centrifugation. Residual ethanol from Buffer PE may inhibit subsequent enzymatic reactions. They are right about this.
8) Place the QIAprep column in a clean 1.5 ml micro-centrifuge tube. To elute DNA, add 50 μl Buffer EB (10 mM Tris·HCl, pH 8.5) or water to the center of each QIAprep spin column, let stand for 1 min, and centrifuge for 1 min.
Restriction Digest 
60uL total add from largest to smallest
Plasmid
21uL
10Xbuffer
6uL
Enzyme
3+3uL
ddH2O
27uL
37℃in water bath 1h
AGE
1) 50 mL 1 x TAE mixed into the triangle flask.
2) In microwave melting, and pay attention to prevent the explosive boiling, liquid spilling.
3) When solid particles disappear entirely, flush with water cooling or let stand at room temperature to the proper temperature. Add 3 uL of EB.
4) Add glue board (rubber sheet put comb in advance) pay attention to prevent bubbles.
5) Cooling and let stand for about 20 min, when it becomes solid.
6) Remove comb vertically, then take out the rubber piece in the electrophoresis tank.
1) Sample: sample and maker with pipetting gun into the hole, pay attention to prevent leakage or add wrong samples.
2) Set the voltage, current and time, check the electrode. Cover, open the electrophoresis apparatus, after opening, see if the positive and negative poles have air bubbles.
3) As the sample piece ran to the bottom. Off electrophoresis. View the results under ultraviolet light.
DNA Gel Extraction   
1) Column regeneration: add 500 ul balance liquid to adsorption column CA2 12000 rpm centrifugal 1 minutes, pour out the filtrate.
2) Cut the DNA band from the gal, weighing in the clean centrifuge tube.
3) To rubber block, add equal volume solution PN (such as 0.1 g gal, to join 100 ul PN) then put in 50 ℃ water bath place, turning up and down continuously in order to ensure fully dissolved.
4) Add solution into the CA2, placed 2 min in room temperature, 12000 rpm centrifugal 1 minutes, pour out the filtrate (at room temperature, DNA binding ability is stronger, you can centrifugal concentration for many times).
5) Drift lotion to column to join ul 600 PW 12000 rpm centrifugal 1 minutes, pour out the filtrate.
6) Repeat step 5.
7) 12000 rpm centrifugal 1 minutes, pour out the filtrate, placed at room temperature for a few minutes to ensure fully getting rid of the alcohol.
8) Put CA2 in clean centrifuge tube, add 72℃ ddH2O 50 ul and wait for 10 min at room temperature, 12000 RPM, centrifugal 2 min.
9) Suck filtrate into the column, repeat 8.
10) Measure the DNA concentration and saved at -20 degrees Celsius.
Prepare Hydrophobin DNA
PCR 50 uL system:
ddH2O
33uL
10Xbuffer
5uL
dNTP
5uL
Template
1uL
Enzyme
1uL
Primer
2.5+2.5uL
Put in PCR, set up annealing temperature, recycle and start PCR.
Ligation
       
20uL 10Xbuffer
2uL
T4 ligase
1uL
Carrier+aimed DNA
17uL
Concentration ratio is 10:1-6:1
22℃ in water place for 1h
2.   Take BFP-Hydrophobin fusion protein standardization for instance
We use overlap PCR to get all the fusion protein. The overlap extension polymerase chain reaction (or OE-PCR) is a variant of  PCR . It is also referred to as Splicing by overlap extension / Splicing by overhang extension (SOE) PCR. To splice two DNA molecules, special primers are used at the ends that are to be joined, because of which a linker is added into BFP and hydrophobic protein. When PCR, to splice two DNA molecules, special primers are used at the ends that are to be joined. Once both DNA molecules are extended in such a manner, the overlapping complementary sequences introduced will serve as primers and the two sequences will be fused.
We can use this technique to combine A fragment and B fragment. In first cycle we use a couple of primers of A and the same to B to PCR. By this, we design a same length of base sequence. In second cycle, due to the homologous sequence, when adding the beginning primers and the final primers, A and B can be primers to each other. They extend to a combined fragment.
This image shows how OE-PCR might be utilized to splice two DNA sequences (red and blue). The arrows represent the 3' ends
In experiment, we use the technique of overlap PCR to get our target sequence. In this experiment, we add BFP and HFB1 which we get in two steps and the fluorescent protein’s primer we design and HFB1’s reverse in the system of PCR.
PCR system（100μl）
Buffer
10μl
dNTP
10μl
BFP fragment
5μl
HFB1 fragment
5μl
Pfu enzyme
1μl
ddH2O
69μl
Add two kinds of primer (1μl) after 5 cycles.
95℃
5min
95℃
30s
1：50℃
2：56℃
3：65℃
 
30s
72℃
2min
72℃
10min
95℃
5min
95℃
30s
1：50℃
2：56℃
3：65℃
 
30s
72℃
2min
72℃
10min
4℃
Forever
 
Protein Expression
1. Pre-expression
1) Put 100uL Glycerin bacteria liquid to 5mL LB liquid medium. Put in shaker 220rpm, 37℃,16-24h. Take 200uL sample and prepare it.
2) In each tube, add 4uL IPTG to induce. Put in shaker for 4h. Then take 200uL of sample and prepare it.
Sample prepare for electrophoresis: 12000rpm, 2min centrifuge. Then pour out supernatant. Use 20uL ddH2O to re-suspended sediment and 4uL 6X protein buffer. Then heated at 100℃ for 20min. Stored at -4℃. Centrifuge 12000rpm, 2min before electrophoresis.
3) Electrophoresis.
2. Cultivate in 1L LB medium
1) Pour one tube to one bottle. Put in shaker 220rpm for 16-24h. Must add chloramphenicol
2) Add 600uL IPTG to each bottle. And also put in the same condition in last step for 4 h
3. Pour out solution to a tube. 3500 rpm, centrifuge for 20min.
4. Abandon supernatant, add 15mL McAc 0. Use pipette repeatedly blowing and sucking to re-suspend the sediment.
5. Wash the bacteria broken machine with ethyl alcohol and McAc 0 in order. And break the bacteria for three times. We can see the solution getting clearer.
6. Transfer to another tube.18000 rpm, 40min, centrifuge.
7. Mixed with medium. And put on protein rotation for 1h.
8. Pour out into a Ni-NTA column to filtration
9. Add 100mL 10, 100mL 20, 100mL 30 and 50mL 50 McAc in order 20mL each time to elute column for wiping of the bulk proteins.
10. Add 20mL 100, 20mL 200, 20mL500 McAc in order 20mL each time to elute column to wash down the aimed proteins.
11. SDA-PAGE to ensure if we get the right proteins.
Every time of washing, take samples in penetration and media, before and after inducing, and sediment in last centrifuge.
1) Add as follows(separation gel):
2) Pour all solution to hole between two broads.
3) Add some ddH2O or n-butyl alcohol to make upper limb straight.
4) Wait for a moment until gel solidifies. Then pour out ddH2O and use Filter paper to blot it.
5) Add as follows (stacking gel):
6) Put in comb and wait till gel solidifies about 20min.
7) Take out comb cautiously and put the whole broad in running machine.
8) Add samples and maker. Set voltage 180V for 20min and start.
 
Super Protein Chip
Contact angle measurements Experimental Procedure
Preparation before the experiment: The measured sample was diluted to 100μg / ml, diluted with ultrapure water. The surface layer of mica tape stick out (remove error experiments may bring). Good marks on glass to find the location of the sample. On mica and glass point 100μl sample, and do label, overnight in a fume hood.
The next day the sample was rinsed, and blown dry with nitrogen gun. The sample is placed in contact angle measuring instrument with measurement stage, adjust the focus, so that the image on the screen clear. Spiral Injector regulator, until full of droplets appear, low it on the sample droplets. Shoot and save pictures. The contact angle can be calculated using the system comes with a 5-point calculation method.
Super Protein Chip
2.1  Janus Spotting: Two glass cover slides were prepared. On each glass substrate, 9 spots (numbered from No.1 to 9) were marked, including Janus group (No.1~7) and blank control group (No.8~9). For Janus group, inJanus (200μg/ml) and sJanus (200μg/ml) were added onto the substrates with a pipette (2μl/spot). Then the substrates were incubated for 12 hours at room temperature in fume cupboard, allowing the Janus to adhere to the substrate surface.
2.2  Capture Antibody Immobilization: For No.1~8, three kinds of capture antibodies against AFP, CEA and CA-153 were respectively added onto the spots of three glass cover slides by means of orthogonal experiment with a pipette (2μl/spot). Then the substrates were incubated for 1 hour at room temperature in fume cupboard.
NO.
Factor
Capture   Antibody（μg/ml）
Detecting   Antibody（μg/ml）
Antigen（μg/ml）
1
10
5
5
2
10
10
5
3
10
5
2.5
4
5
5
5
5
5
5
2.5
6
5
10
2.5
7
5
10
5
8
Blank
Blank
Blank
9
Blank
Blank
Blank
2.3 Blocking: For No.1~8, each spot was covered with 1% BSA and 2% sucrose (2.5μl mixture/spot) for 1 hour at room temperature in fume cupboard.
2.4  Wash: The glass substrates were soaked and rinsed with TBST (TBS with 0.05% Tween 20, pH 5.0) for several seconds in centrifuge tubes about two times and dried naturally.
2.5  Antigen Immobilization: For No.1~9, three kinds of antigens including AFP, CEA and CA-153 were respectively added onto the spots of three glass cover slides by means of orthogonal experiment with a pipette (2μl/spot). Then the substrates were incubated for 1 hour at room temperature in fume cupboard.
2.6  Wash: The glass substrates were soaked and rinsed with TBST (TBS with 0.05% Tween 20, pH 5.0) for several seconds in centrifuge tubes and dried naturally.
2.7  Fluorescently labeled Detecting Antibody Immobilization: For No.1~9, three kinds of detecting antibodies against AFP, CEA and CA-153 were respectively added onto the spots of three glass cover slides by means of orthogonal experiment with a pipette (2μl/spot). Then the substrates were incubated for 1 hour at room temperature in fume cupboard away from light.
2.8  Wash: The glass substrates were soaked and rinsed with distilled water for several seconds in centrifuge tubes and dried naturally.
2.9  Observation: The immobilized detecting antibodies were observed with inverted fluorescence microscopy.
 
Stimulated Plastic Enzymolysis
Experimental preparation
1.   Handing method of plastics
The plastics was cut into many pieces which have a same specification(3*3*0.25mm),those plastics were washed by 5mg/mL TritonX-100,100mM Na2CO3,ddH2O(3 times),each washing was set in an ultrasonic environment for 5min.Then we put those plastics which have been washed into a dryer to dry them thoroughly.
2.   Making up the buffer solution
We made up 6 kinds of pH: 3.6 (Sodium acetate -Acetic acid Buffe), 7.0 (Tris-HCl), 7.8 (Tris-HCl), 8.5 (Tris-HCl), 9.0 (Tris-HCl), 12.0 (Disodium hydrogen phosphate - sodium hydroxide buffer).
3.   Handing of proteins
We have successfully express the Cutinase Cut1 and the fusion protein Cut1-sJanus-m,using the concentration tube at a rotate speed of 3500 rpm,we changed the buffer of protein from 200mM MCAC to Tris-HCl (300mM NaCl) meanwhile we concentrate the two protein to a higher concentration.
To accurate determinate the concentration of the two proteins, we use the BCA kit and got a result as follow.
protein
Cut1
Cut1-sJanus-m
Concentration(mg/ml)
12.220
16.216
Then we put the two protein into lots of 1.5ml EP tubes,each contain 50ul protein,Quick freezing and keeping under -80℃.
Common operation
All of our experiment can be describe as three processes, the preparation of reaction system, the reaction and the detection. In order to describe easier, some common operations are explained in this part.
1.   The preparation of reaction system
All of our reaction system have a final volume of 100ul, and each system contain one piece of plastic (3*3*0.25mm). The reaction vessel are PCR tubes.
2.   Reaction
We put those PCR tubes which contain the reaction system in a table, the speed is set at 220 rpm, the temperature is set at 50℃.Each reaction system are mixed uniformity before put in table.
3.   The method of detection
As the plastic have a degradation, the turbidity of reaction system would changed which can be monitored at OD600.
Pre-experiment
To find a appropriate condition for our main experiment, we did some Pre-experiments.
1.   Decide the detecting time.
We set three detecting time: 3h, 6h, 18h. In this experiment, the concentration of the two protein are set at 2 mg/ml. The pH of buffer are 7.0.As to each protein, we set three reaction system which have the same condition except the detecting time. To study how the proteins themselves would change as the time change, we also set a control group which have no plastics in the reaction system. All kinds of reaction system can be seen as follow table.
TIME
3h
6h
18h
CUT1
CUT1-BLANK
CUT1-sJanus-m
CUT1-sJanus-m-BLANK
2.   Explore the proteins’ concentration.
We set the concentration of the proteins at 0.1mg/ml,0.5mg/ml,1.0mg/ml,1.5mg/ml,2.0mg/ml.。The pH of buffer are 7.0.The detecting time is 3h.
3.   Study how pH influence the proteins themselves and the reaction to plastics.
We set the pH of the buffers at 3.6, 7.0, 7.8, 8.5, 9.0 and 12.0. The concentration of proteins are 0.2mg/ml. The detecting time is 3h.
Main experiment
From the Pre-experiment, we confirm the reaction condition in our main experiment. The concentration of Cutinase Cut1 was set at 2mg/ml. The pH of buffer are 7.0.The detecting time is 3h.
1.   We added some sJanus and inJanus which have a final concentration of 0.1mg/ml, we also set the control groups which do not have  sJanus or inJanus as control.
2.   We mixed one piece of plastic with sJanus and inJanus,incubating for 16h,then added the Cut1,put the reaction system in table to start the reaction,the final concentration of sJanus and inJanus are 0.1mg/ml.
3.   We mixed the Cut1 with sJanus and inJanus, incubating for 16h,then added one piece of plastic,put the reaction system in table to start the reaction,,the final concentration of sJanus and inJanus are 0.1mg/ml.
Expand experiment
To study if our fusion protein Cut1-sJanus-m have a better result in promote the reaction of Cut1 and plastics.
We set the concentration of Cut1 and Cut1-sJanus-m at 0.1mg/ml,0.5mg/ml,1.0mg/ml,1.5mg/ml and 2.0mg/ml. The pH of buffer are 7.0.The detecting time is 3h.A different operation was we shake each reaction system for 1 min before we start the reaction.
Protein Extraction Kit
Process of Protein Extraction
1.   Add protein solution to a tube, and add 5% (v/v) Berol 532 into it.
2.   Use shaker to make them mixed at the speed of 300r/min, 20 centigrade, working for 1h.
3.   Put the tube at room temperature to make system separated.
4.   Put the upper phase to another tube.
5.   Add water saturated butanol whose volume is five times of Berol 532, then shake the tube for a while.
6.   Use centrifuge to make system separated at the speed of 3500rpm, working for 10min.
7.   The lower phase is the purified protein.
Future Work
Super Protein Chip
In the future, we plan to try a variety of tumor markers and to study the value of combined serum test in tumors diagnosis. We also plan to conduct specificity experiments to test the detection accuracy. Additionally, we plan to contact the most advanced and experienced laboratory in the field of protein chip in China, expanding its value on clinical application.
Stimulated Plastic Enzymolysis
Our project shows a great potential in plastic disposal field. Because our Janus is typical of environmental and harmless, there are plenty of works need to be done!
1.   Based on the data we achieved, discuss the mechanism why Janus can promote hydrolysis rate.
2.   Expressing more fusion protein (such as LC-sJanus-m) and more cutinase to verify the universality of our conclusions.
3.   Conduct more reduplicate experiments to reduce the deviation of our data.
4.   Find a way to explain the problems we mention above, weather it is accidental or not.
5.   Optimize experimental conditions, try our best to increase hydrolysis rate. In order to apply our Janus into industry someday.
6.   Comparing fusion protein using sJanus with that using inJanus, find out the difference and which one is better for PET disposal.
Protein Extraction Kit
1.     To explore how the concentration of protein influence the extraction efficiency further.
2.     Using more kinds of proteins to conduct the experiments, we want to expand the application of extraction efficiency.
3.     Given to that we only use one kind of detergent, so in the future we are going to test several kinds of detergents to make extraction more convenient and have more meanings.
 
 
Overview
Background
Neo-Protein Design
Super Protein Chip
Stimulated Plastic Enzymolysis
Protein Extraction Kit
Protein Expression
Super Protein Chip
Stimulated Plastic Enzymolysis
Protein Extraction Kit
Molecular Cloning
Protein Expression
Super Protein Chip
Stimulated Plastic Enzymolysis
Protein Extraction Kit
Super Protein Chip
Stimulated Plastic Enzymolysis
Protein Extraction Kit
