Team:Tianjin/Collaborations
内页
HOME
PROJECT
DESCRIPTION
RESULTS
DESIGN
PROTOCOLS
FUTURE WORK
PARTS
TEAM PARTS
BASIC PARTS
COMPOSITE PARTS
MODELING
OVERVIEW
ADSORPTION
PET DEGRADATION
PROTEIN EXTRACTION
COLLABORATIONS
Tianjin for THU-A
THU-A for Tianjin
NKU for Tianjin
SAFETY
SAFE PROJECT DESIGN
SAFE LAB WORK
SAFE SHIPMENT
HUMAN PRACTICE
NOTEBOOK
TIMELINE
LAB WORK
TEAM
ABOUT THE TEAM
MEMBERS
ATTRIBUTIONS
ACKNOWLEDGEMENTS
Tianjin for Qsinghua-A
This year, we helped Tsinghua-A team in their wet-lab experiment. They want to mutate a single base in their DNA so that it will not be cut by a certain type of restriction enzyme. So we taught them how to do overlap PCR. Because we two team were not in the same city, it was hard for us to do experiment for them directly. So we taught them the principle of design and the protocol of experiment.
Primer design of overlap PCR
1.   The overlap region should be near 20bp
2.   Each primer need another ~20bp for annealing (each primer is ~40bp in total)
3.   The mutant point should be in the middle of the overlap region
Protocol of overlap PCR
1.   First PCR:
One reaction system (50ul) contains 2ul 10uM forward primer and 2ul 10uM overlap reverse primer; the other reaction system contains 2ul*10uM reverse primer and 2ul*10uM overlap forward primer.
2.   Run gel electrophoresis to test the PCR product.
3.   Gel recovery.
4.   Second PCR:
In the reaction system (50ul), there are 2ul of each product (L and R part of the target gene) from the previous step, and 2ul 10uM forward and reverse primer respectively.
5.   Gel electrophoresis and recovery
6.   Sequencing the product to test it.
Results:
Figure 1. From left to right, 1 is marker; 2 is L part of overlap; 3 is R part of overlap; 4 is another L part of overlap.
Figure 2. From left to right, 1 is marker; 2&4 is the second-step product of overlap.
Tsinghua-A for Tianjin
Here are some basic function in our program written by Tsingua-A iGEM.
They have written detailed note to make our program readable.
function dots_e=rdm(n,number)
% rdm produces binary coordinate randomly
% (number,n),where number is the number of coordinates
% you want and n is the n-by-n square surface
%
% Calling sequence:
%        array=rdm(number)
% Define variables:
% number    --Number of the dots that is absorbed with cutinase
% n         --N-by-n surface
% dots_e    --The chosen random points that will react with cutinase
disp('The coordinates of degradable points on the simulated surface is as follow')
dots_e=randi(n,number,2);
function combine_dots_bycut=filter_dots(number_h,number_h_cut,dots_h)
% Filter_dots is to filtate a certain array into smaller one randomly,but
% with a certain filtration ratio.
% Calling sequence:
%        new_array=combin_cut(number)
% Define variables:
% number_h        ---The number of locuses adsorbed by hydrophobins
% number_h_cut    ---The number of locuses adsorbed by hydrophobins and
%                    cutinase
% ratio_h_bind_e  ---The ratio that enzyme binds to the hydrophobin of all
% dots_h          ---The coordinates of locuses adsorbed by hydrophobins
% i               ---The number of locuses adsorbed by hydrophobins and
%                    cutinase after filtration
% temp_number     ---The number array generated randomly in the range of number_h
%
temp_number=randperm(number_h,number_h_cut);
i=1;j=1;
for i=1:length(temp_number)
    for j=1:2
combine_dots_bycut(i,j)=dots_h(temp_number(i),j);
    end
end
Nankai for Tianjin
Expression of inJanus.
To express inJanus in pichia pastoris, Nankai chose fed-batch fementation carried out in bioreactor with 1L initial volume. When the cell fresh weight reached to 200g/L, Nankai started inducement for 96h. What’s more, there is a higher production when dissolved oxygen was maintained at 15%-25% by turning up the methanol speed.
Follow is the figure shows the SDS-PAGE and western blot analysis of inJanus. And the inJanus molecule weight was 8KDa. In addition, inJanus in the broth was mainly monomer and dimer. The latter binds better with the primary antibody of Janus.
Figure 1. (a)silver staining along with SDS-PAGE and (b)western blot analysis of inJanus fermentation broth(lane 2,3), lane 1 was pure inJanus as control
To measure the yield of inJanus, Nankai measured the concentration of inJanus by UPLC. And Nankai used standard sample of sJanus of different concentration (0.1, 0.05, 0.025 mg/mL) to get a standard curve of concentration with adsorption area of UPLC.
Figure 2. (a) silver staining along with SDS-PAGE of Janus fermentation broth of different induction time(lane 1-4 24,48,72,96 h).(b)inJanus production and cell fresh weight in different time.
Figure 3. HPLC of pure injanus HPLC
Figure 4. HPLC of injanus fermentation broth HPLC
Form figure 3. And figure 4, we can clearly see that inJanus was expressed in pichia pastoris.
