"
Page
Discussion
View source
History
teams
Log in
 
Team:TUDelft/SDP Results
From 2009.igem.org
 
 
Brainstorm Pictures Meetings Acknowledgements Calendar Contact
Home
Team Deliverables Achievements Conclusions Future work Safety Lab Notebook Protocols used Parts
Module 1: Conjugation System
Module 2: Self Destructive Plasmid
Cloning Strategy Experimental Results Conclusion Future work
Module 3: Time-Delay Device
Module Integration
Modeling
Ethics
Communication
Experimental results
The experimental results for the Self Destructive Plasmid are detailed below. In the process of constructing it, we deviated from the cloning strategy after proving that it didn't work as expected.
I-SceI restriction site
Gel electrophoresis of  in pSB1AK3 backbone. Lane 1: undigested (supercoiled) plasmid. Lane 2: BglI digested part, expected size = 3219bp. Lane 3: BglI and I-SceI restricted part, expected band lenghts are 2275 and 944bp. Expected lenghts correspond to the bands observed in the gel.
Construction of the I-SceI homing endonuclease restriction site was done by primer synthesis. Primers were annealed and ligated into a pSB1AK3 backbone (3189bp without the insert). Together with the 30bp recognition sequence (shown below) this gave a 3219 bp part. The sites where cleavage of the DNA backbone occurs are indicated with ^.
5' A G T T A C G C T A G G G A T A A^C A G G G T A A T A T A G 3'
3' T C A A T G C G A T C C C^T A T T G T C C C A T T A T A T C 5'
The resulting biobrick () was successfully sequenced and tested for functionality. Sequencing results can be found on the part page. Functionality was assessed using commercially available I-SceI restriciton enzyme. The plasmid was purified and subjected to a digestion with BglI alone and BglI plus I-SceI. The latter would theoretically result in two fragments of 2275 and 944bp.
The results in the adjecent image show that the digestion indeed results in the expected band pattern. This shows that the restriction site works as expected.
As shown in the cloning strategy, the restriction site was cloned inbetween the promoter and the RBS, resulting in  , which was expected not to influence transcription. However, when a culture containing this biobrick was induced with anhydrotetracyclin (aTc), no fluorescence was observed. This led to a modification of the cloning strategy.
A simpler part was constructed (), which only contained the restriction site in front of the aTc inducible GFP-LVA generator.
Makeup of . It consists of the I-SceI homing endonuclease restriction site and an anhydrotetracyclin inducible GFP-LVA generator.
This part was successfully sequenced and shown to work. Firstly, aTc fluorescence induction worked, as shown in the following image.
anhydrotetracyclin induction of GFP fluorescence in centrifuged culture containing . Left tube is uninduced, right tube is induced.
Gel electrophoresis of  in pSB1AK3 backbone. Lane 1: undigested (supercoiled) plasmid. Lane 2: PstI digested part, expected size = 5095bp. Lane 3: PstI and I-SceI restricted part, expected band lenghts are 3204 and 1891bp. Expected lenghts correspond to the bands observed in the gel.
Secondly, the part was digested with PstI alone and with PstI and I-SceI. The lenght of the entire biobrick was expected to be 5095bp. The second lane in the adjacent image shows that the observed length corresponds to the predicted length. The part was also digested with PstI and I-SceI, theoretically resulting in two bands of 3204 and 1891bp. The resulting bands in lane 3 of the same figure show that these also correspond well.
Thirdly, the sequence was confirmed by sequencing the part in forward and reverse direction.
I-SceI homing endonuclease
After successful construction of the restriction site with reportergene, the actual protein was added to the construct. The IPTG inducible part () made by the 2008 ETH Zurich team was used for this. However, after having it sequenced, the part appeared to be a medium constitutive T4 ligase generator (), made by the same team. To rule out a mistake by the registry, the part was requested again. A new batch was transformed and sent to us. It was grown up, isolated and restricted with EcoRI and PstI, to obtain insert length. It was exactly as long as the first batch (~1700bp), which corresponds to the expected length of  (1668bp), and is significantly longer than the expected length of 1114bp. These findings were entered into the registry as a user review of .
As a workaround, a twin of what was supposed to be  was constructed, using . The sequence of the resulting part () was confirmed, as shown in the sequence analysis section of the parts page. Thereby the sequence of  was indirectly confirmed.
Close inspection of this sequence also revealed, that the homing endonuclease had an LVA degradation tag attached to the end of its sequence. This information was not mentioned in the original I-SceI parts page (), nor in any of the constructs that contain this part. As protein tags in general can influence the folding and function of a protein and the LVA tag in specific reduces stability of a protein, this is considered quite an essential feat. This information was therefore entered into the registry at the concerned parts pages, in the form of user reviews.
Integration
With the IPTG inducible I-SceI homing endonuclease generator and the I-SceI restriction site with aTc inducible reporter ready, the two parts could be combined to yield the final Self Destructive Plasmid. This final construct is depicted in the image below.
Makeup of , the Self Destructive Plasmid. It consists of the an IPTG inducible I-SceI homing endonuclease generator and its restriction site. An anhydrotetracyclin inducible GFP-LVA generator is included to check for the destruction of the plasmid when the endonuclease is induced.
Gel electrophoresis of  in pSB1C3 backbone, isolated from 2 different colonies. The first one was a fluorescing colony (lane 1 and 2) and the second one non-fluorescing (lane 3 and 4). Lane 1 and 3: EcoRI digested part, expected length = 5100bp. Lane 2 and 4: EcoRI, PstI digested part, expected lengths = 2072 and 3028bp.
Although it seemed to be an assembly like any other, it had its complications. Several colonies were screened for the right insert. Two types appeared to be present. The first one, with colonies that were fluorescing as expected and the second with colonies that did not give fluorescence (after aTc induction). However, the insert size of the first type was around 1200bp (see lane 2 in the adjacent image), significantly smaller than the expected 3028bp.
The non-fluorescent colonies did have bands corresponding to 2072 and 3028bp, but also one corresponding to the (linear) 5100bp fragment (compare lane 3 and 4). It was assumed that the colony had a mixture of plasmids, some that could get restricted by both enzymes and some that could not. Sequencing for this part failed, possibly because of the prescence of multiple kinds of plasmid.
Gel electrophoresis of  in pSB1C3 backbone, isolated from 1 colony under various conditions. The isolated plasmids were restricted with I-SceI and PstI, with expected bands of 1891 and 3109bp. Lane 1 and 2: not induced. Lane 3,4: induced with aTc. Lane 5,6: induced with IPTG. Lane 7,8: induced with aTc and IPTG.
Ideally a plasmid would have been isolated that gave only two bands when restricted with the enzymes mentioned above. But due to time constraints this was not possible. The selected colony was incubated in liquid culture under various conditions of induction (aTc and IPTG) to see if (some of) the plasmids would get degraded by the expressed homing endonuclease.
The outcome of the experiment was disappointing. Although at least some plasmids were present that had the expected size, none of the cultures induced with aTc showed any fluorescence. Also, the restriction pattern for the cultures that were induced with IPTG (for the expression of the homing endonuclease, lanes 5-8) was identical to the non-IPTG induced cultures. This shows that these plasmids were probably not restricted by I-SceI, because the restriction site was still present. Usually linear DNA will be degraded inside a cell, so if the restriction site had been cut, one would have expected the plasmid to have degraded (at least partially).
The reason that I-SceI restriction inside the cell failed could be due to the LVA-tag attached to the homing endonuclease. Either it made the protein so unstable that it would get degraded at such a high rate that it could not restrict the plasmid. Or the tag itself prevented it from folding properly, and thereby prevented it from doing restrictions in the first place.
The reason why no fluorescence was observed after aTc induction is not well understood. Sequencing of this part only succeeded in the reverse direction (see ). It showed the presence of the GFP-LVA gene. Also the I-SceI restriction site has been shown to be present in the restriction analysis, so it is unlikely that anything between the restriction site and the GFP-LVA gene has been changed, because the observed length of the insert corresponds so well with the expected lenght.
SPONSORS
 
COLLABORATIONS
Team Groningen and Team Amsterdam
Participants in survey
 
VISITORS
Retrieved from " http://2009.igem.org/Team:TUDelft/SDP_Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
