Team:Central Calgary/notebook
Team Central Calgary
Toggle navigation
Central iGEM
Home
Project
Our Vision
Construct
Accomplishments
Future Direction
Our Team
Official Profile
Team Members
Contact Us
Policies & Practices
Safety
Ethics
Outreach
Notebook
Protocols
Lab Work
General
Media
Notebook
Protocols
Lab Procedures As adapted from the Synthetic Biology Guidebook for iGEM High School.  Everything must be sterile, and kept as aseptic as possible, throughout all procedures.
Protocol 1 - Making LB Broth
Materials: Tryptone Yeast Extract; Bacteriological NaCl (Molec Biol Grade) Distilled Water 1L Beaker Micropipette + Tips Parafilm  Procedure: Weigh out, and add the following into the 1L container. 5g Tryptone 2.5g Yeast Extract 5g NaCl Add 500ml of distilled water into the 1L container. *The glass container must remain half empty to prevent the liquid from boiling over in the autoclave.* Mix by swirling. *Some powder may not dissolve. Make sure that none gets stuck to the sides.* Cover the top with aluminum foil, loose enough so pressure can escape in the autoclave. Place/stick a strip of autoclave tape on top. Autoclave at about 15psi for 20min. Remove from autoclave and cool. The autoclaved liquid is now to be stored at  4°C in a fridge.
Protocol 1.5 - Making LB Plates
Materials: Tryptone Yeast Extract; Bacteriological Agar; Bacteriological NaCl (Molec Biol Grade) Distilled Water Sterile Antibiotic Stock Solution - If needed Autoclave + Autoclave Tape Scale 1L Glass Bottle/Erlenmeyer Flask Sterile Petri Dishes Aluminium Foil micropipette + Tips Parafilm *This protocol makes 500ml LB Agar for 20-25 plates.* *It can be scaled up or down as required.*  Procedure: Weigh out, and add the following into the 1L container. 5g Tryptone 2.5g Yeast Extract 5g NaCl 7.5g Agar Add 500ml of distilled water into the 1L container. *The glass container must remain half empty to prevent the liquid from boiling over in the autoclave.* Mix by swirling. *Some powder may not dissolve. Make sure that none gets stuck to the sides.* Cover the top with aluminum foil, loose enough so pressure can escape in the autoclave. Place/stick a strip of autoclave tape on top. Autoclave at about 15psi for 20min. Remove from autoclave and cool to 55°C. If you require antibiotic, add in by swirling into mix. Making sure that the liquid has cooled, as hot liquid will degrade antibiotic stock, and that the work area is kept as sterile as possible.
Antibiotic
Conc. of Prepared Stock
Final Conc. in LB Agar
Final Conc. in LB Agar
Volume Added to 500ml LB Agar
Ampicillin
100mg/ml
100µg/ml
1:1000
500µl
Kanamycin
50mg/ml
50µg/ml
1:1000
500µl
Chloramphenicol
50mg/ml
25µg/ml
1:2000
250µl
Label plates with date, plate number, and antibiotic used. Pour, or pipette, around 20ml LB Agar into each petri dish.This must be done aseptically. Pouring slowly, and steadily to avoid bubbles and lumps. Cover with lid, and place to cool/harden. This should take 30min - 1hr. Invert plates so agar is on top, to prevent from unnecessary moisture on the plates. Seal edges with parafilm, and store in 4°C fridge.
Protocol 2 - Using the DNA Plate - Rehydrating
Materials: DNA Kit Plate Sterile Distilled Water Micropipette + Tips  Procedure: Locate well (double, triple check, location). Draw 10µl of distilled water into pipette. Puncture foil and release water into the well. Wait for 10-15 minutes for DNA to rehydrate. Store the re-hydrated in the freezer (-20°C), leave it on the plate, or transfer into a separate tube. *As little as 1µl can be used to for a successful transformation*
Protocol 3 - Making Competent Cells
Materials: DH5alpha E. Coli Cells 50mM CaCl2 Solution (filter sterile) LB Broth Incubator and Shaker set at 37°C Ice 15ml Tubes Microcentrifuge Microcentrifuge Tubes Micropipettes + Tips Spectrophotometer (optional)  Procedure: Transfer 100µl overnight culture (E. Coli) into 10 ml of LB Broth (inside 15ml tube). Grow at 37°C with shaking for at least 3 hours. Optimally until broth is cloudy,and has an OD600 (optical density) of 0.4-0.6. Transfer 500µl into microcentrifuge tubes, and spin down all four at 7000 rpm for 2min. Discard supernatant, being very careful to not disturb the pellet. Resuspend the pellet in 500µl CaCl2 solution. Spin down again at 7000rpm for 2min and remove supernatant. Resuspend the pellet in 100µl of CaCl2 and place on ice. Cells are now ready for transformation.
Protocol 4 - Restriction Digestion
Materials: Plasmid DNA 10x Restriction Buffer Restriction Enzymes (ExoRI, SpeI, XbaI, PsI) Sterile Distilled Water Thermocycler Water Bath set at 37°C Sterile Microcentrifuge Tubes Microcentrifuge Micropipette + Tips Ice  Procedure: Thaw all solutions and buffers, flick (homogenize). *Restriction enzymes DO NOT need thawing and should be kept on ice at all times* Label a sterile tube clearly for use with the restriction enzyme. Pipette the following into the microcentrifuge tube in the order listed. 10µl Plasmid DNA (500 ng) 1µl Restriction Enzyme #1 1µl Restriction Enzyme #2 5µl 10x Restriction Buffer up to 50µl sterile distilled water Mix gently by pipetting up and down. *You can briefly spin in the centrifuge to bring contents to the bottom* Incubate at 37°C  in waterbath for 30min to an hour. Incubate (heat-shock) at 80°C for 20min. Store the restriction digest at 20°C or proceed immediately with ligation, or agarose gel electrophoresis.
Protocol 5 - Ligation
Materials: Insert Restriction Digest Mix Vector Restriction Digest Mix 10x Ligation Buffer T4 DNA Ligase Sterile Distilled Water Thermocycler Sterile Microcentrifuge Tubes Micropipette + Tips Ice  Procedure: Thaw all solutions and buffers, flick (homogenize). *Ligation enzymes DO NOT need thawing and should be kept on ice at all times* Label a sterile tube clearly for use with the ligation enzyme. Pipette the following in order (adding up to a total volume of 10µl). 3µl Sterile Distilled Water 3µl Insert Restriction Digest 2µl Vector Restriction Digest 1µl 10x Ligation Buffer Pipette up and down to mix. Briefly centrifuge to collect all liquid at the the bottom of the tube. Incubate at room temperature for about an hour. Incubate at 80°C for 20 minutes, to denature the ligase (heat-shock). Store ligation mix at 20°C in a freezer, or proceed immediately with a transformation.  *There should be a 3:1, insert DNA (the segment being added) to the vector DNA (the now linear plasmid backbone). Also, the amount of DNA in the ligation should be low, around 100ng.*  *The DNA concentration is assumed to be the same for both the insert and vector restriction digests, and it equals 25ng/µl, as set up by the previous protocol. Assuming the insert is about half the size of the vector, to get a molar ratio of 3 insert : 1 Vector, the volume ratio must 1.5 insert :1 vector.*  *Ligation buffers are usually 10 fold, (10x) more concentrated than needed. Therefore, they need to be diluted 10 times, i.e. 1µl in a total volume in 10µl.*  *A general formula for determining the volume ratio of insert to vector.* Vi / Vv = 3 x (Cv / Ci) x (Mi / Mv) Vi = Volume Insert Vv = Volume Vector Cv = DNA conc. of vector restriction digest Ci = DNA conc. of insert restriction digest Mi = Molecular weight i.e. approximate size of insert Mv = Molecular weight i.e. approximate size of vector
Protocol 6 - Transformation
Materials: 100µl Aliquots of Competent Cells (Protocol 1). DNA Ligation Mix Control Plasmid DNA LB Broth LB Agar Plates (with desired antibiotic) Water bath at 42°C Incubator at 37°C Spreader Bunsen Burner Ice Micropipettes + Tips  Protocol: Obtain fresh, or frozen, competent cells. Thaw and keep on ice. Label the tube. Pipette in 2-5µl of ligation mix into the competent cells. *The rest of the ligation mix can be stored in the freezer at -20°C* Incubate on ice for 30min. Incubate at 42°C for 60sec. Place on ice immediately for 5min. Add 200µl of LB Broth and mix gently. Incubate at 37°C for 2hrs. Warm up plates to room temperature, and label. Plate out 200µl and 20µl of transformed cells onto two separate plates, spreading the cells evenly onto the surface. Incubate cells at 37°C for 12-14hrs (overnight) or at room temp. for 24hrs. Plates with culture can be stored in fridge afterwards, and used for weeks.
Protocol - Miniprep
Preferred Kit: E.Z.N.A Plasmid DNA Mini Kit  Materials: DNA Mini Kit Micropipette + Tips 1.5ml Centrifuge Tubes.  Procedure: Grab and label 3 microcentrifuge tubes. Pipette 1.5mL (use a P1000 set at 750 ul twice) of the corresponding culture in a glass tube into the corresponding microcentrifuge tube (e.g. The pLac in the glass tube goes to the pLac in the microcentrifuge tube). Spin at max speed (151*100g) for 1 minute. Discard the supernatant . Repeat steps 2-4 to gain a bigger pellet 6 times. Add 250ul of Solution 1 to each of the microcentrifuge tubes (stored in the fridge) and resuspend the pellet. Remember to put Solution 1 back in the fridge. Add 250ul of Solution 2 to each of the microcentrifuge tubes and gently invert several times, and let it sit for 2 minutes. Add 350ul of Solution 3 to each of the microcentrifuge tubes and immediately invert. A white flocculent protein should form. Centrifuge the tubes at max speed for 10min. Grab HiBind DNA mini columns and label them accordingly. Transfer the supernatant and do not disturb the pellet of the microcentrifuge tubes to the top of the of the corresponding HiBind DNA mini columns (e.g. the supernatant of plac should be transferred to the HiBind DNA mini column labelled plac.) Centrifuge at max speed for 1 min. Discard the filtrate in the collection tube and reuse the collection tube. Pipette 500ul of HBC buffer to the top of the DNA HiBind mini column. Centrifuge at max speed for 1 min. Discard the filtrate in the collection tube and reuse the collection tube. Pipette 700 ul of DNA wash buffer to the top of the DNA HiBind mini column. Centrifuge at max speed for 1 min. Discard the filtrate in the collection tube and reuse the collection tube. Centrifuge at max speed for 2min to dry the column. Label 3 microcentrifuge tubes accordingly. Transfer the top of the DNA HiBind mini column to the corresponding microcentrifuge tube (e.g. the plac DNA HiBind mini-column would be transferred to the “plac DNA date plasmid” microcentrifuge tube). Add 30ul of elution buffer to the DNA HiBind mini column. Ensure the elution buffer lands on the middle of the the DNA filter. Centrifuge at max speed for 1 min. The DNA is now in the microcentrifuge tube. Store in the freezer.
Protocol - Making 10X TAE Buffer
Materials: EDTA Disodium Salt Distilled Water NaOH sodium hydroxide (solid or concentrate) Tris Base Glacial Acetic Acid Weigh-boats Scale Beakers Graduated Cylinders Stir Rod or Magnetic Stirrer pH Meter or Paper Clean Bottles  Procedure: *To prepare 10x TAE Buffer you must first prepare a solution of 0.5M EDTA (pH 8.0)* Weigh out 93.05g EDTA disodium salt. Add the EDTA to a beaker containing 400 ml of distilled water. Mix with stir-rod, or magnetic stirrer. Adjust the pH of the solution to 8.0 by adding NaOH. EDTA will completely dissolve at a pH of 8. Add distilled water to total 500ml and mix. Pour into a clean bottle and label “0.5M EDTA pH8”. Weigh out 48.5g Tris Base and add to a beaker containing 800 ml of distilled water. Mix to dissolve. Measure out 11.4ml of glacial acetic acid and add it to the Tris solution. Measure out  20 ml 0.5M EDTA (pH 8) and add to the Tris solution. Add distilled water to a total volume of 1L. Stir to mix. Pour into a clean bottle for long term storage, label “10x TAE”.
Protocol - Agarose Gel Electrophoresis
Materials: 10x TAE Buffer Distilled Water Agarose Syber Safe (DNA Stain) DNA Loading Buffer 6x DNA Restriction Digest DNA Ladder (Prefered = Invitrogen E-GEL 1 Kb Plus) Agarose Gel Electrophoresis Apparatus (Tank, Gel Tray, Comb, Power Supply, Etc.) Graduated Cylinder Stir Rod Erlenmeyer Flask Micropipettes + Tips Scale Microwave Tape  Procedure: Prepare 100 ml 1x TAE Buffer by adding 10 ml 10x TAE Buffer to a cylinder containing 90ml distilled water. Mix well with a clean stir rod. Pour the 100 ml of 1x TAE into a beaker or flask, allotting space for bubbles, so it does not bubble over. Weigh out 1g of agarose powder and add to the 100ml of 1x TAE. Place the beaker/flask into the microwave and set the time to 60sec carefully watching for it to start bubbling. Once bubbling open, take out of microwave, stir, place into microwave, and repeat until solids are completely dissolved. Cool gel until warm to touch. Assemble gel cassette by taping off ends, and inserting comb (thin side). Pour the agarose solution carefully, and slowly, into the gel cassette, to avoid air bubbles. Let gel cassette cool at room temperature for 30min. Wrap into saran-wrap and move into 4°C fridge and let cool for 30min, it may sit in fridge for up to 3hrs prior to use. While waiting for gel to solidify, prepare the DNA samples by mixing the following into microcentrifuge tubes. 2µl 6x Loading Buffer 4µl DNA Restriction Digest 6µl Distilled Water Very carefully pull off tape and remove comb. Place cassette into tray making sure wells are on the negative end. Fill the tank with 1x TAE Buffer covering the gel with 3-5mm of TAE. If air bubbles are trapped inside wells, carefully remove them with a pipette. Load DNA samples into wells, starting with the ladder, on both end wells. Place cover over the gel box, making sure the attached wires are black on black, and red on red. Turn on gel box, at the 15V setting, and run for 30mins. After 30mins have surpassed switch to the 75V setting and run for 40mins. At the 40min mark, switch off, remove gel and place into solution of 10ul Cybersafe and 50 ml 10x TAE. Stir, and move gel around in solution. Place on gel reader, and view outcomes. If colour is faint repeat soaking in CyberSafe solution, until desired color is available.
Lab Work
February 23, 2015: Matt, Kaiden, Alexia  Attempted to make 500 mL LB Broth as well as Agar Plates. For the broth we used: 5g Tryptone 2.5g Yeast Extract 5g NaCl For the plates we used: 5g Tryptone 2.5g Yeast Extract 5g NaCl 7.5g Agar  February 24, 2015 Alexia Made a 500 mL Agar Plate solution, but it was not autoclaved (due to unforeseen circumstances). 5g Tryptone 2.5g Yeast Extract 5g NaCl 7.5g Agar 500 ml of distilled water  Mixed in a 1L beaker.   March 2, 2015 ( lunch ) Vladimir, Jared, Zenab, Francis.  Transferred overnight culture (E.Coli) into tubes with 10 mL of LB broth. Then let it grow at 37°C for three hours.  March 2, 2015 Jared, Matt, Francis.  Ran the ‘Making Competent Cells’ procedure: Transferred 500  µL of bacterial culture into 4, 1.5 mL, microcentrifuge tubes. Spun at 7000 rpm for 2 minutes. Added 500  µL CaCl solution. Spun down again (7000 rpm for 2 minutes), supernatant removed. Resuspended in 100 µL CaCl solution.  Labeled ‘A’ - Francis’s Tubes ‘B’ - Backups - Matt’s Tubes:  Notes for future: Plasmid Backbone pSB1C3 MRFP. BBa_E1010 Plate 3 (spring distribution 2013) Well 12N The team decided that it would be best to focus on lab work and other aspects to the project, instead of visiting junior high schools. This idea was discarded.    March 5, 2015 Jared, Tatiana.  Made fresh bacterial culture of E. coli, growing at 37°C for 3 hours. 5mL of LB 100ul of the E. coli Left out at room temperature overnight.  March 6, 2015 Zenab, Jared.  Subculturing E. coli:  - 1 mL LB broth, -100 µL of E. coli cells The above were placed into a 1.5 mL microcentrifuge tubes labeled F1,F2,F3,F4 (in red).  March 9, 2015 Jared (AM). Vlad and Francis (PM).  Morning: Subcultured all tubes (F1,F2,F3,F4) into the small plastic tubes. We pipetted the following into small plastic tubes.  -1mL of LB  -100ul of E. coli cells (the F1,F2,F3,F4 strains) They are all labelled with the appropriate date on the side (3/9) and are in red.      Afternoon: Made fresh new competent cells (currently in the freezer): Transferred 500  µL of bacterial culture into 4, 1.5 mL, microcentrifuge tubes. Spun at 7000 rpm for 2 minutes. Added 500  µL CaCl solution. Spun down again (7000 rpm for 2 minutes), supernatant removed. Resuspended in 100 µL CaCl solution.   Labeled: ‘FR’ - Francis’s competent cells. ‘Z’ - Backup - Vladimir’s competent cells.  March 10,2015 Natalie, Tatiana, Jared, Jaime.  The transformation of the pLac (BBa_K822000) into E. coli.  -Take the competent cells made from yesterday that were in the freezer  -Thawed it on ice  -Placed 2ul of pLac DNA in it and flick it gently  -Placed on ice for 30 min  -Incubate at 42 degrees C for a minute  -Placed on ice for 1 minute  -add 200ul of LB and incubate at 37 degrees C for 2 hours -Plated it on a chloramphenicol plate and incubate it at 37 degrees C. Labelled pLac on the bottom with the appropriate date (3/10)   March 11,2015 Zenab, Natalie, Tatiana, Jared.  Prepared 500 mL solution for pouring agar plates. 5g Tryptone 2.5g Yeast Extract 5g NaCl 7.5g Agar  Subcultured all previous F1,F2,F3,F4 tubes, which contain E. coli.  Relabelled as F1, F2, F3, F4 in blue, with March 11, 2015 on the side. The following were mixed together in a small plastic tube to achieve this:  -1000uL of LB  -100uL of cells, E. coli.  No results from yesterday's transformation. The plate looks very empty.  March 12, 2015 (Morning) Matt, Jaime (AM).  Morning: Subcultured the F1,F2,F3,F4, the E. coli. We mixed the following together in a small plastic tube:  -1000 uL of LB  -100uL of cells, E. coli Labelled appropriately with the date on the side (3/12)  No results from March 10’s transformation. The plate looks very empty. We’re gonna try the transformation again tomorrow. The speculation is the LB was not put in during its 2 hour incubation, meaning the cells had no media to grow in, thus, killing the cells.       March 12, 2015 (Afternoon) Francis, Zenab, Jared. (PM) Afternoon: Made competent cells. Transferred 500  µL of bacterial culture into 2, 1.5 mL, microcentrifuge tubes. Spun at 7000 rpm for 2 minutes. Added 500  µL CaCl solution. Spun down again (7000 rpm for 2 minutes), supernatant removed. Resuspended in 100 µL CaCl solution.  Later.. The transformation of the pLac (BBa_K2822000), RFP (BBa_4B8TEK) occurred with the competent cells made earlier today.  -Take the competent cells and thaw them on ice -Put 2uL of the pLac (BBa_K2822000) DNA in one of the tubes with competent cells, and 2uL of the RFP (BBa_4B8TEK) in the other tube of competent cells  -Let the tubes sit on ice for 5 min  -Incubate them at 37 degrees C for 5 min  -Place on ice for 5 min  -Add 200 uL of LB to it and incubate it at 37 degrees C for 2 hours. -After the 2 hours, plate 100uL of each of the tubes to a chloramphenicol plate and label it accordingly with the date.    March 13, 2015 Jared The cells on the plates grew!  Subcultured the F1,F2,F3,F4, the E. coli, with the date on the side (3/13). The following were mixed together in a small plastic tube:  -1000uL of LB  -100uL of cells  March 16, 2015 Vlad Jared Created agar plates Plated wells with pLac and RFP Prepped for autoclaving      Overnight culture of RFP  March 17, 2015 Jared  Subcultured RFP in tube Labelled as RFP 3/17 Mini prep Labelled as RFP DNA  April 1, 2015 Jared  Subcultured - Dated it 4/1 - New T, F, and RFP  April 4, 2015 Sean Jared  - RFP on a plate - Subcultured - "T1" , "F1" - new cultures labelled with date on the side  April 10, 2015 Jaime Jared Vlad  - mini prep ---> laddered it all - Subcultured "F1, F2, F3, F4, T1, T2, T3, T4" |--> 4/10.    April 13, 2015 Jared -Subcultured new tube "RFP 4/13"  April 14, 2015 Jared  competent cells Labelled "F1 4/14 competent cells" Sitting in water bath         April 14, 2015     Continued Jaime Jared Vlad Tatiana Francis  Ran a gel and completed mini prep  April 15, 2015 Jaime Jared Francis Tatiana Vlad  Subcultured cells Labelled "cells NC" --> stands for not competent  April 17, 2015 Francis Jared  Competent cells 0.1 mL of cells into 10 mL broth **All LB plates have the antibiotic  April 17,2015 Jaime Francis Zenab Jared  - transformation of pLac 13 N ---> plated 2 plates labelled "pLac" in green  April 21, 2015 Jared  Making LB agar plates to be autoclaved. After it is autoclaved we will have to put the angiostatin in its or the angiostatin to grow. ---> The solution is in a 600 ml beaker labelled "LB agar plates solution"  April 22, 2015 Catie Sean Jared  Finished the competent cells Re suspended in CaCl  April 22, 2015   Continued Jaime Vlad Sean Jared  Transformed the angiostatin in cells. Plated it in "Amp 22/94/15" labeled as "angiostatin 4/22"--> in green It was then placed in the incubator   April 24, 2015 Jared  -Subcultured in red "NC" "4/24"   April 18,2015 Tatiana Jared Jaime  Subculture ---> 3 tubes labelled with "4/28" ---> also have "RFP" "pLac" and "angiostatin" ---> not competent cells "NC" "4/28"  April 29, 2015 Jaime Tatiana Danielle Jared  Plated angiostatin  April 29, 2015   Continued Jared Jessica Jaime Paris  Possibly mislabeled --> tubs that came from container that may have been contaminated --> mini prep "RFP" "Ang" "pLac' in green ( with date, 4/29)  May 5, 2015 Jared Matt  Ligation --> RFP pLac --> Angiostatin pLac       May 6, 2015 Matt Jaime  Subcultured - new tube of cells  May 8, 2015 Vlad Jared Francis  Transformation --> E1010 RFP  May 9, 2015 Jared Matt Francis Serena Vlad Tatiana Jessica Natalie  Workshop: mini prep, gel run---> pLac, angiostatin, RFP Subcultured  May 9, 2015 - After Canmore Presentation Problems: - Wrong RFP. Use E1010 - Include Warburg Effect - Hallmarks of cancer (deregulating cellular energetics) - Citations (APA)  CANMORE JUDGING SESSIONS: -Video, the black and white was gloomy! Include stats for more of an effect -Bring into the project:  -policy and practice  -volunteering  -someone said ___ and influenced our project by -Neoplasm = cancer -Duatuse → > public opinion -The goal is to shut off metastasis -Research/ Sites to use: -U of C microbiology -google scholar -Pink Army Initiative  CANMORE SPEAKING AND COMMUNICATION: -Audience:  -Meaning?  -”Read” your audience  -Malleable to reactions -Tell a story NEVER:  -Word for word  -Memorize  -Read -Humour  -Driving factors  -Lab problems? Q and A: -Prepare for anticipated problems  -Thank you  -Clarify question -Intro, attention getter, cancer affects all and what’s in it for the audience -Turn “YOU” into betterment of society -Be personal -Use open body language -Move with purpose -Goal statement -Summary after major points    May 11, 2015 Jared Francis  Transformation: RFP, E1010  May 12, 2015 Jared Matt  Ligation--> RFP, pLac ---> E1010 5/12  Mini prep ---> Tubes of "RFP E1010" ---> LB broth in the fridge- needs autoclaving "RE" has been linearized  May 21, 2015 Jessica Jared Jaime Vlad Sean  ---> Master mix was prepared and added to DNA ---> Centrifuge at 10000 xg for one minute while keeping the pellet intact MASTER MIX - 6 microlitres E - 6 microlitres D - 15 microlitres of 10x - 3 microlitres H2O  May 23, 2015 Jessica Jared Serena  →  Subculturing RFP, RFP pLac 1-4 →  added CHL →  100 ml of LB were added to RFP and RFP pLac 1-4 →  4 ml of CHL was added        June 3, 2015 Subcultured 10 ml →  1,2,3,4 pLac, RFP, E1010, RFP   June 4, 2015 Miniprep 1,2,3,4→  pLac, RFP, E1010  Subculture in the morning →  Tatiana and Jared  June 5, 2015 RE digest  MASTER MIX 1 x 5 = 5 Eco 1x5= 5 Pst 5-5= 25 NE Buffer 115 H2O  June 21, 2015 Push for Your Tush Run!  June 22, 2015 Ran gel  June 23,  2015 Jared Francis  Ran gel Almost out of Iain’s ladder  June 25, 2015 Jared Francis  Rylan  - Miniprep →  pLac 2 3 - Restriction enzyme →  RFP 2 3  June 26, 2015 Jared Alexia  Subcultured the individual colonies (4-12) of the pLac RFP cultures: 1 ml of LB 100 ul of cells 1ul of chloramphenicol Mixed the above in a small plastic tube.  Also mini prepped the 1, 4-12 colonies of the pLac RFP:  After the miniprep, we used the restriction enzymes on it to digest it in preparation for running a gel. The restriction enzyme procedure went as follows:  Create a mastermix of the enzymes in a plastic tube with the following amounts of enzymes, buffers and water: Eco-R1 : 1ul*10 = 10 ul Pst:  1ul*8 = 10 ul 10x NEbuffer:  5ul*10 = 50 ul Distilled water: 23ul*10 = 230 ul    Take 30ul of the master mix and mix it with 20ul of the plasmid DNA of each of the pLac RFP colonies. Label it appropriately and let it incubate at 37 degrees C for 10 minutes.   After: Incubate it at 80 degrees C for 20 minutes to deactivate the enzymes.  With the digested DNA, we created another mixture composing of the linearized DNA, loading dye and water. The following amounts were mixed together: 2ul 6 x loading buffer 4μL DNA restriction digest 6μL distilled water We did this for all of the plasmid DNA of the pLac RFP cultures, 1, 4-12.  With the dyed DNA, we then had to run a gel. To create a gel we mixed the following together: .4g of agarose 50ml of 1x TAE buffer  The above were microwaved until the liquid looked clear. After it was cooled, 4ul of DNA stain, SyberSafe, was added. This mixture was poured in the gel cassette (with tape around it),  with the comb and we waited for it to solidify.  Once the gel was solid, it was placed in the gel runner (with the tape and comb removed). Pour 1X TAE into the gel runner until the liquid level is 4mm above the gel.  The DNA was then loaded into the wells. 10ul of the DNA test samples (the RFP pLac colonies) was placed in the wells, and 2ul of the DNA ladder was used.      June 29, 2015 June 26th’s gel run may prove that the 10 and 11 colonies of the pLac RFP have our construct with the insert and the backbone. This allows us to run an endpoint assay to see if there will be a  change.   The theory of our construct is that when it is in the presence of lactic acid, it will turn red. To compare its change in colour, a negative control and a positive control will be set up. The negative control will be E. coli cells with no plasmid in it, whereas the positive control will be the LacI RFP that constantly expresses red fluorescent protein. When the pLac RFP construct is put in the presence of lactic acid, it should be red the LacI RFP.  The assay was carried out as follows:  1)WASH THE CHLORAMPHENICOL*: Centrifuge 2 of each of the pLac RFP cultures (10,11), E. coli, and RFP. Discard the supernatant and resuspend in 1000ul LB. 2)Repeat step 1 three times. 3)Pool the cells together 4) label 2 tubes of each of the cultures. Label one of the tubes “LB” and the other “lactate” 5) Pipette 16 ul of LB into the tubes labelled with LB and 16ul of lactic acid in the tubes labelled with lactate 6) Transfer 800ul of the cells into their appropriately labelled tubes. 7) incubate all cells at 37 degrees C overnight. In the morning compare how red everything is.  *The pLac does not work with chloramphenicol, which is why the culture needs the chloramphenicol to be washed.    July 5, 2015 The results of the assay thoroughly proves that we do not have our construct. Our mentor humbly recommends that we take it from the top with the new DNA from the 2015 registry kit. He argues that the DNA could have degraded entailing this failure.  A tube of cells were made to prepare for competent cells tomorrow: 10ml of LB Colony of E. coli Mix the above together, be sure to keep everything as aseptic as possible and leave it in a 37 degrees C incubator overnight.  July 6, 2015 Competent cells and transformation: Create a new tube of cells, fill it with 5mL of LB and 200ul of yesterday's overnight culture of E.  coli. Let it grow for 3 hours.  Competent cells: Create 2 tubes of the following: Take 1500 ul of the new culture and transfer it into smaller plastic tubes. Spin it down at 70000 rpm for 2min to create a pellet. Discard the supernatant and resuspend in 500ul of CaCl. Spin it down again. Resuspend in 100ul of CaCl  Transformation: Label one of the competent cells tubes as pLac and the other RFP. In the tube labelled pLac, add 2ul of pLac plasmid DNA from the registry kit. In the tube labeled RFP, add 2ul of the RFP plasmid DNA from the registry kit. Let both tubes sit on ice for 30 minutes Move the tubes to a 37 degrees C incubator for 5 minutes. Immediately place the tubes on ice for 5 minutes Add 200ul of LB to each of the tubes and let it incubate at 37 degrees C for 2 hours After the 2 hour incubation is done, plate the 100 ul of the cells in chloramphenicol plates labelled accordingly. Plate 200 ul of normal E. coli cells on the chloramphenicol plates as well as a negative control to see if the antibiotic selection works. Incubate the plates overnight  July 7, 2015 The plates grew and tomorrow these cell’s DNA will be mini prepped, ligated, and transformed. To prepare for the procedures, 3 overnight cultures were made of the pLac, RFP and E. coli without plasmid: 10 mL of LB 1 colony from the plate 7.5ul of chloramphenicol (only add if the colony has chloramphenicol resistance, the pLac and RFP colonies have the resistance)  Let the cultures incubate at 37 degrees C overnight.  July 8, 2015 Create a tube of cells for competent cells. Mix 5mL of LB and 200ul of E. coli cells together and let it grow for 3 hours. While this is happening, miniprep, restriction enzyme digest, and ligate the pLac RFP cultures.  Miniprep the pLac and RFP cultures, followed the Bio Basic kit procedure: Restriction enzyme digest of the pLac: Pipette the following together: -20ul of mini prepped pLac -1ul of EcoR1 -1ul of Spe1 -5ul 10X NE buffer -23ul of water Incubate at 37 degrees C for 10 minutes and heat inactivate it at 80 degrees C for 20 minutes   Restriction enzyme digest of the RFP: Pipette the following together: -20ul of mini prepped RFP -1ul of Xba1 -1ul of Pst1 -5ul 10X NE buffer -23ul of water Incubate at 37 degrees C for 10 minutes and heat inactivate it at 80 degrees C for 20 minutes  Ligate the pLac RFP: In a microcentrifuge tube. mix the following and incubate at room temperature for 30 minutes. -2ul of linearized psB1C3 -2ul of digested pLac DNA cut with E + S -2ul of digested RFP DNA cut with X + P -2ul of 10X T4 DNA ligase buffer -1ul of T4 DNA ligase -11ul of water Let it incubate at room temperature for 30 minutes and heat inactivate it at 80 degrees C for 20 minutes.  Create competent cells: 1)label 2 tubes “CC” 2) pipette 1.5mL of the e. coli into the tubes 3)Centrifuge it at 70,000RPM for 2 minutes 4)discard the supernatant 5)Resuspend in 500ul of calcium chloride 6)Centrifuge it again at 70,000 RPM for 2 minutes 7)Resuspend in 100ul of calcium chloride. It is ready for transformation   Transformation: 1) Pipette 2ul of the pLac RFP construct into the competent cells that were just created 2) Let it incubate on ice for 30 minutes 3) Incubate at 37 degrees C for 5 minutes 4) Immediately place on ice for 5 minutes 5) Add 200ul of LB and let it incubate for 2 hours 6)plate it on chloramphenicol plates  July 28, 2015 make LB Make Chl -- 34 mg/ml 10 ml of ethanol (95) purity 0.34 g of chl 6 cultures of isolated  from the Pla RFP 8 put in tubes -- 10 ul of LB -- 8 ul of CHL incubates at room temp  July 29, 2015 Miniprep 1-6 pLac rfp cal times RE digest: Eco. R 1 - 6 ul PSt1 - 6ul NE Buffer - 30 ul Water- 138 ul    July 30, 2015 Ran gel ------------ August 3, 2015 Create three tubes with 10mL of LB in it and 5ul chl. Label and add the following cultures in the individual tubes: -Plac (K822000) 8/3 -RFP E1010 8/3 -Ecoli negative control 8/3 Incubate at 37 degrees C for 8 hours  Create 2 more tubes with 10mL LB and 10ul amp. Label and add the following cultures in the individual tubes: -Angiostatin 8/3 -E. coli negative control 8/3 Incubate at 37 degrees C for 8 hours  Miniprep the pLac, RFP and angiostatin cultures. Follow the Biobasic miniprep procedures.  Digest the pLac with S + P by pipetting the following in a microcentrifuge tube labelled “linear pLac S + P 8/3 DNA” -10ul plasmid pLac DNA -5ul 10X NE Buffer -1 ul Spe1 -1ul Pst1 -33ul autoclaved distilled water incubate at 37 degrees C for an hour Heat inactivate at 80 degrees C 20 minutes.  Digest the RFP with X + P by pipetting the following in a microcentrifuge tube labelled “linear RFP X + P 8/3 DNA” -10ul plasmid RFP DNA -5ul 10X NE Buffer -1 ul Xba1 -1ul Pst1 -33ul autoclaved distilled water incubate at 37 degrees C for an hour Heat inactivate at 80 degrees C 20 minutes.  Digest the angiostatin with X + P by pipetting the following in a microcentrifuge tube labelled “linear angiostatin X + P 8/3 DNA” -10ul plasmid angiostatin DNA -5ul 10X NE Buffer -1 ul Xba1 -1ul Pst1 -33ul autoclaved distilled water incubate at 37 degrees C for an hour Heat inactivate at 80 degrees C 20 minutes.  Ligate the pLac and RFP together by pipetting the following in a microcentrifuge tube labelled “pLac RFP ligation mix 8/3” -2ul linear pLac cut with S + P -6ul linear RFP cut with X + P -2ul 10X T4 DNA ligase buffer -1ul T4 DNA ligase -9ul autoclaved distilled water Let it incubate at room temperature overnight, and heat inactivate at 80 degrees C tomorrow morning.  Ligate the pLac and Angiostatin together by pipetting the following in a microcentrifuge tube labelled “pLac angiostatin ligation mix 8/3” -2ul linear pLac cut with S + P -6ul linear angiostatin cut with X + P -2ul 10X T4 DNA ligase buffer -1ul T4 DNA ligase -9ul autoclaved distilled water Let it incubate at room temperature overnight, and heat inactivate at 80 degrees C tomorrow morning.  August 4, 2015 Create a 10mL tube of LB with E. coli and let it incubate at 37 degrees C for 6 hours.  Competent Cells: 1)From the 10mL tube of LB with e. coli in it, pipette 1.5mL of the culture into smaller microcentrifuge tubes labelled “CC” (abbreviated competent cells) 2)Centrifuge it at 70,000RPM for 2 minutes to collect a pellet 3)Discard the supernatant 4)Repeat steps 1-3 again to gain a bigger pellet 5)Pipette 500ul of calcium chloride into the tubes and resuspend by pipetting up and down 6)Centrifuge and discard the supernatant 7)Resuspend in 100ul of calcium chloride  Transformation: 1)Label the competent cells that were just created: “pLac angiostatin” and “pLac RFP” 2)Add 2ul of the corresponding ligation mixes to the competent cells. 3)Let it incubate on ice for 30 minutes 4)Heat shock it at 37 degrees C for 5 minutes 5)Place on ice for 5 minutes 6)Add 200ul of LB and let it incubate at 37 degrees C for 2 hours 7)Plate the cells on labelled plates and create an E. coli negative control as well. Let it incubate at 37 degrees C overnight.       August 5, 2015 Nothing grew on the plates from yesterday's transformation. Replate the transformation mix cells - 2 hours may not have been enough time for the cells to grow before plating.  Ran a gel:  From left to right: PLac S + P cut: It’s not even the right size it should be a little bigger than the backbone to the right RFP X + P: there’s a really faint line indicating that the RFP code on may be there and has separated from the psB1C3. Angiostatin X + P: the 3 bands is quite peculier. The backbone is ampicillin and the right size and the angiostatin is the right size as well. The third band may be the enzymes not cutting properly. pLac E + P: the gel was overran and we cannot be sure if the pLac dna is there.   August 18, 2015 Created tubes of pLac, RFP, and Angiostatin mixed with 10 ml of LB Created four tubes with 10 ml of LB  Labeled as:   pLac 8/18                 10 ml LB                       8 ul chl           RFP  8/18                 10 ml LB                                8 ul chl           Angiostatin 8/18       10 ml LB                      10 ul Amp           E. coli negative control    8/18  10 ml LB                       E. coli negative control    8/18   8 ul chl Transformation Spin down 1.5 ml of E. coli at 70000 RPM Remove supernatant and resuspend in 100 ul of CaCl Add 5 ul of the pLac angiostatin ligation mix Let it sit on ice for 30 minutes Heat shock it at 37 degrees C in a water bath for 5 minutes Immediately place on ice for 5 minutes Add 200 ul of LB Incubate at 37 degrees C in a water bath for 2 hours  Run Gel Prep create a gel by mixing            → 1 g Agarose            → 100 ml 1x TAE microwave it to mix it together wait until it is 60 degrees C and add 10 ul of sybersafe pour the gel in the gel tray with masking tape at the side let it cool in the fridge for 30 minutes  August 31, 2015 -measuring DNA concentrations: pLac plasmid: 63.3ng/mL pLac linear: 16.4ng/mL  RFP plasmid: 49.1 ng / mL RFP linear: 7.3 ng / mL  Ang linear: 25.2ng / mL Ange plasmid: 92 ng / mL     September 9, 2015 Create a tube of 10mL LB labelled “e. coli 9/9. 10 mL LB” and let it incubate at 37 degrees C for 6 hours.  Digest plasmid angiostatin DNA with E + P, by pipetting the following together in a microcentrifuge tube labelled “linearized angiostatin E + P”: -10ul angiostatin plasmid DNA -5ul 10X NE buffer -1ul EcoR1 -1ul Pst1 -33ul autoclaved distilled water Centrifuge it quickly to ensure all liquid falls to the bottom. Let it incubate at 37 degrees C for 2 hours. Heat inactivate at 80 degrees C for 20 minutes.  Ligate the linear angiostatin DNA cut with E + P to psB1C3. Pipette the following together in a microcentrifuge tube labelled “angiostatin, amp backbone, psB1C3 ligation mix”: -6ul linear psB1C3 cut with E + P -2ul linear angiostatin cut with E + P -2ul 10X T4 DNA ligase buffer -1ul T4 DNA ligase -9ul autoclaved distilled water Let the mixture incubate at room temperature for 4 hours. Heat inactivate at 80 degrees C for 20 minutes  Stop the pLac angiostatin ligation by heat inactivating at 80 degrees C for 20 minutes.  Competent Cells: 1)From the 10mL tube of LB with e. coli in it, pipette 1.5mL of the culture into smaller microcentrifuge tubes labelled “CC” (abbreviated competent cells) 2)Centrifuge it at 70,000RPM for 2 minutes to collect a pellet 3)Discard the supernatant 4)Repeat steps 1-3 again to gain a bigger pellet 5)Pipette 500ul of calcium chloride into the tubes and resuspend by pipetting up and down 6)Centrifuge and discard the supernatant 7)Resuspend in 100ul of calcium chloride  Transformation: 1)Label the competent cells that were just created: pLac ang, ang. psB1C3, and pLac RFP 7D 2)Add 2ul of the corresponding ligation mixes to the competent cells. 3)Let it incubate on ice for 30 minutes 4)Heat shock it at 37 degrees C for 5 minutes 5)Place on ice for 5 minutes 6)Add 200ul of LB and let it incubate at 37 degrees C for 2 hours 7)Plate the cells on labelled plates and create an E. coli negative control as well. Let it incubate at 37 degrees C overnight.   September 10, 2015 Create a tube of 10 mL LB and subculture 1000ul of an E. coli culture into it. Let it incubate at 37 degrees C for 6 hours.  Ligate the pLac and Angiostatin together by pipetting the following in a microcentrifuge tube labelled “pLac angiostatin ligation mix” -2ul linear pLac cut with S + P -6ul linear angiostatin cut with X + P -2ul 10X T4 DNA ligase buffer -1ul T4 DNA ligase -9ul autoclaved distilled water Let it incubate at room temperature for 4 hours, and heat inactivate at 80 degrees C for 20 minutes  Ligate the linear angiostatin DNA cut with E + P to psB1C3. Pipette the following together in a microcentrifuge tube labelled “angiostatin, amp backbone, psB1C3 ligation mix”: -6ul linear psB1C3 cut with E + P -2ul linear angiostatin cut with E + P -2ul 10X T4 DNA ligase buffer -1ul T4 DNA ligase -9ul autoclaved distilled water Let the mixture incubate at room temperature for 4 hours. Heat inactivate at 80 degrees C for 20 minutes   Competent Cells: 1)From the 10mL tube of LB with e. coli in it, pipette 1.5mL of the culture into smaller microcentrifuge tubes labelled “CC” (abbreviated competent cells) 2)Centrifuge it at 70,000RPM for 2 minutes to collect a pellet 3)Discard the supernatant 4)Repeat steps 1-3 again to gain a bigger pellet 5)Pipette 500ul of calcium chloride into the tubes and resuspend by pipetting up and down 6)Centrifuge and discard the supernatant 7)Resuspend in 100ul of calcium chloride   Transformation: 1)Label the competent cells that were just created: ”pLac ang”, and “ang. psB1C3” 2)Add 2ul of the corresponding ligation mixes to the competent cells. 3)Let it incubate on ice for 30 minutes 4)Heat shock it at 37 degrees C for 5 minutes 5)Place on ice for 5 minutes 6)Add 200ul of LB and let it incubate at room temperature overnight. 7)Tomorrow morning, plate the cells with a negative control.  September 15, 2015 Transform psb1c3 and pLac rfp 7D / 8D  September 16, 2015 Isolate single colonies of the psb1c3 and grow up in 10ml tubes of lb Miniprep transform and ran a gel.  Grow 10mL tubes of LB with pLac RFP 7D, pLac RFP 8D, and isolate 8 more colonies of the pLac angiostatin. Add 5ul of chloramphenicol to it too.  September 17, 2015  Create phosphate saline buffer (PBS) by mixing the following: -8g NaCl -0.2g KCl -1.44g Na2HPO4 -0.24g KH2PO4 Add 800mL of distilled water and autoclave for 20 minutes. Protocol was found here: http://cshprotocols.cshlp.org/content/2006/1/pdb.rec8247
Dilute lactic acid: 1)Label 3 tubes lactic acid 1 mol/L lactic acid, 0.1mol/L lactic acid,  and 0.01mol/L lactic acid 2) Pipette 1000ul of 1 mol./L lactic acid in the tube labelled: 1 mol/L lactic acid 3) Pipette 100 ul of the 1 mol/L lactic acid and 900 ul of distilled water in the tube labelled “0.1mol/L lactic acid” 4)Pipette 10ul of the 1 mol/L lactic acid and 990 ul of distilled water in the tube labelled “0.01mol/L lactic acid”  Assay on pLac RFP 1)Label 2 tubes pLac RFP 7D, and pLac RFP 8D, 2)From yesterday’s overnight culture, centrifuge 1.5mL of the culture at max speed for 1 minute 3)Discard the supernatant 4)repeat until there is no culture left in the 10mL tube 5)Wash the cells from chloramphenicol by resuspending in 1000uL of PBS and centrifuging it again 6)Repeat step 5 again 3 more times 7)Resuspend the cells in 400ul of PBS 8)Grab 8 more tubes and label them “pLac RFP 1mol/L”, “pLac RFP 0.1mol/L”, “pLac RFP 0.01mol/L”, and“pLac RFP water” (as a negative control) for both cultures 9)Pipette 80ul of the corresponding lactic acid concentration and water to the negative control to the tubes. 10)Pipette 700 ul of PBS into all of the tubes 11)Pipette 100 ul of the pLac RFP cultures into the tubes 12)Let it grow for 6 hours RESULTS: The promoter wasn’t induced in any of the lactic acid concentrations. Possible reasons why include: -The lactic acid we have doesn’t induce the promoter. (DL Lactic acid) -The cells were not washed from enough chloramphenicol -The E. coli strain we have (e. coli JM-109) does not permit lactic acid to enter the cytoplasm, however, other research disputes this claim
General
September 23, 2014 - First Day  Why did you join iGEM? - A few responses from our team  “I decided to join iGEM because I thought that it looked really interesting and is related to an area of science that I’m interested in and would like to learn more about!”  “Science has always interested me, but I am not sure if it is what I would like to pursue in the future. I’m joining iGEM because of how new it is- we have a bunch of different programs and such, but this is pretty new to me. I want to check it out and see if I like synthetic biology as much as I think I will like it.”  Notes - After an hour of attempting to learn grade 12 biology Basics of DNA -DNA is in the cell of the nucleus -DNA stores information that makes us, us →  controls cell function and what organelles make -Controls the cell by making proteins (coding) -Some proteins are messengers to tell other parts of the cell what to do -Cell communication -Products secreted out of the cell BACTERIA - have DNA -Do not have a nucleus; have a nucleoid region, where their DNA is found (simple!) -Bacteria can also have a plasmid DNA -This extra DNA tells us to do more than its basic DNA does -It is circular DNA and is not found in the nucleoid region  Plasmid DNA -Plasmid DNA will be cut open and our DNA sequence will be inserted -The plasmid will be sealed and made circular region -This plasmid with DNA added is called a recombinant plasmid -This plasmid will then be inserted into the bacteria (transforming) -After modifications, it needs to replicate -When the cell divides, the daughter cells will also have that plasmid  October 2, 2014 We are currently thinking of going to different junior high schools and doing a small banana experiment with the students from these schools. It would be a great way to recruit new grade ten’s (it's always hard getting them to join new things so early in high school!) and we could use survey monkey to get some statistics for our policy and practice section.  October 14, 2014  IGEM QUESTIONS: Travel dates? How long is the presentation? Is the presentation split between people? Promoter: p.Lac activates the angiostatin and is secreted into the bloodstream.  November 4, 2014 We designed a sort survey for junior high students.  November 10, 2014 Received a shaking water bath donation from Matthew’s mom’s workplace.   December 13, 2014 Components of a project: -Wetlab -Modeling -Human practices (policy and practices) -Business -Visual arts  Notes: -4 nucleotides -Directionality: 5’ to 3’ -Antiparallel - double stranded DNA -Hydrogen bonds: palindromes -A sequence which is the same as its reverse compliment  5’ - TCATGA - 3’  3’ - AGTACT - 5’ -Mirror image  → a sequence which is the same when read from right to left and left to right on the same strand  The Central Dogma -Dna replication -RNA transcription -Protein translation  The Rules of RNA: -Instead of T they use uracil (U) -Single stranded Converting DNA to RNA: -Promoter (start) | RBS (make RNA) and coding sequence | Terminator (stop)  RNA polymerase binds the promoter The polymerase reads the DNA, makes RNA copy Polymerase hits terminator and ends transcription.    RNA to protein: RNA is converted to a protein sequence using a ribosome 20 amino acids -> different codons in triplets makes different amino acids  Codes: -DNA -RNA -Amino acids / proteins DNA (gene) -> make protein (TAG, TAA, TGA)  Protein function: -Can help the structure of the cell -Transports molecules inside the cell / act to modify - Use of enzymes -Can bind / modify molecules of DNA, RNA, other proteins  Techniques: -Plasmid DNA -Vehicle for engineered DNA -Antibiotic resistance -Restriction enzyme site  Vector: - What the plasmid is before the DNA is placed in it -Origin of replication →  allows for replication -Multiple cloning site - RE sites  -Restriction Enzymes: -”Scissors” in bio -Cut at palindromes -Sticky ends - 4 bases long  Transformation: -A way to put DNA into cells -Makes plasmid -Agarose gel electrophoresis  Steps for the Project: -Transform the BB DNA -37 degrees C overnight  bacteria -Miniprep -Restriction digest (37 degrees C for 1 hour) -Run a gel (linear) -Ligation (room temperature overnight)          Transformation and select  Optical density: -Light is scattered; the less light the higher the OD -The DNA will only transform at the exponential growth phase.  Competent cells can sit on ice for 5-6 hours -Making competent cells (getting rid of proteins and sugars that make the bacteria impermeable)  -Poking small holes in the membrane  -37 degrees C would repair itself with DNA   Ligation: -Glue it all together -Sticky ends -Plasmid lets it replicate  Lactobacillus?**  -Lactic acid change in angiostatin production will be measured with RFP -Lysate - solution you make when you break the cells open  January 1, 2015 Finished letter to Calgary’s Rotary Club.
