<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stony_Brook_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stony Brook/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Home</TITLE></P><DIV class="menu-logo"><DIV class="navbar-brand"><SPAN class="navbar-caption-wrap"><A class="navbar-caption text-white display-4" href="https://2018.igem.org/Team:Stony_Brook">
				iGEM SBU</A></SPAN></DIV></DIV><DIV class="collapse navbar-collapse" id="navbarSupportedContent"><UL class="navbar-nav nav-dropdown nav-right" data-app-modern-menu="true"><LI class="nav-item"><A class="nav-link link text-white display-4" href="https://2018.igem.org/Team:Stony_Brook">
					Home</A></LI><LI class="nav-item dropdown"><A class="nav-link link text-white dropdown-toggle display-4" href="https://2018.igem.org/Team:Stony_Brook/Team" data-toggle="dropdown-submenu" aria-expanded="false">
					Team</A><DIV class="dropdown-menu"><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/Team" aria-expanded="false">Team Members</A><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/Collaborations">Collaborations</A><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/Attributions" aria-expanded="false">Attributions</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link link text-white dropdown-toggle display-4" href="#" aria-expanded="false" data-toggle="dropdown-submenu">
					Project</A><DIV class="dropdown-menu"><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/Description" aria-expanded="false">Description</A><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/Design" aria-expanded="false">Design</A><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/Experiments" aria-expanded="false">Experiments</A><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/Notebook" aria-expanded="false">Notebook</A><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/InterLab" aria-expanded="false">InterLab</A><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/Model" aria-expanded="false">Model</A><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/Results" aria-expanded="false">Results</A><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/Wiki" aria-expanded="false">Wiki</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link link text-white dropdown-toggle display-4" href="https://2018.igem.org/Team:Stony_Brook/Parts" data-toggle="dropdown-submenu" aria-expanded="false">
					Parts</A><DIV class="dropdown-menu"><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/Parts">Parts Overview</A><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/BasicParts" aria-expanded="false">Basic Parts</A></DIV></LI><LI class="nav-item"><A class="nav-link link text-white display-4" href="https://2018.igem.org/Team:Stony_Brook/Safety">Safety</A></LI><LI class="nav-item dropdown"><A class="nav-link link text-white dropdown-toggle display-4" href="https://2018.igem.org/Team:Stony_Brook/Human_Practices" data-toggle="dropdown-submenu" aria-expanded="false">Human  Practices</A><DIV class="dropdown-menu"><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/Human_Practices" aria-expanded="false">Integrated Human Practices</A><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/Public_Engagement" aria-expanded="false">Education &amp; Engagement</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link link text-white dropdown-toggle display-4" href="" data-toggle="dropdown-submenu" aria-expanded="false">Awards</A><DIV class="dropdown-menu"><A class="text-white dropdown-item display-4" href="https://2018.igem.org/Team:Stony_Brook/Plant" aria-expanded="false">Plant</A></DIV></LI></UL></DIV><DIV class="container align-left"><DIV class="media-container-column mbr-white col-md-12"><H1 class="mbr-section-title py-3 mbr-fonts-style display-1">Model</H1></DIV></DIV></DIV><SECTION class="mbr-section content4 cid-r6INV2ifaD" id="content4-pp"><DIV class="container"><DIV class="media-container-row"><DIV class="title col-12 col-md-8"><H2 class="align-center pb-3 mbr-fonts-style display-2">What is Mathematical and Theoretical Biology?</H2></DIV></DIV></DIV></SECTION><SECTION class="mbr-section article content1 cid-r6INV2G8P5" id="content1-pq"><DIV class="container"><DIV class="media-container-row"><DIV class="mbr-text col-12 col-md-8 mbr-fonts-style display-7"><P>Mathematical and theoretical biology is a branch of biology which employs theoretical analysis, mathematical models, and abstractions of the living organisms to investigate the principles that govern the structure, development, and behavior of the systems. Because of this, researchers can describe and analyze systems in a quantitative manner that allows for precise simulation, often leading to observations that might not be evident through experimental practices alone. A mathematical model can be thought of as a simplified, yet still pertinent, description of a system that utilizes mathematical concepts. Due to the often overwhelming complexities of dynamic systems in Biology, the level of mechanistic insight into a system might be little to none, leading to a ‘black-box model’. In this approach, the input parameters will be accounted for, allowing us to study the given output without understanding the full internal structure and design of the system.</P></DIV></DIV></DIV></SECTION><SECTION class="mbr-section content4 cid-r6INV312U7" id="content4-pr"><DIV class="container"><DIV class="media-container-row"><DIV class="title col-12 col-md-8"><H2 class="align-center pb-3 mbr-fonts-style display-2">What is your model about?</H2></DIV></DIV></DIV></SECTION><SECTION class="mbr-section article content1 cid-r6INV3q9Va" id="content1-ps"><DIV class="container"><DIV class="media-container-row"><DIV class="mbr-text col-12 col-md-8 mbr-fonts-style display-7">Production of <I>Synechococcus elongatus </I> PCC 7942 for applications in bioethanol production still presents important techno-economic challenges as an industrial bioprocess. Mathematical modeling of cellular metabolism in biological production usually improves process yields, though for industrial applications, the model should be as simple as possible in order to sustain model usefulness and technical feasibility [1]. A genome-scale metabolic network of chemical reactions that take place inside a living organism is primarily reconstructed from the information that is present in its genome and the literature and involves steps such as functional annotation of the genome, identification of the associated reactions and determination of their stoichiometry, assignment of localization, determination of the biomass composition, estimation of energy requirements, and definition of model constraints. This information can be integrated into a stoichiometric model of metabolism that can be used for a detailed analysis of the metabolic potential of the organism using constraint-based modeling approaches and hence is valuable in understanding its metabolic capabilities [2]. The main approach our team took was to use flux balance analysis as the basis of our model.
</DIV></DIV></DIV></SECTION><SECTION class="mbr-section content4 cid-r6INV3J6in" id="content4-pt"><DIV class="container"><DIV class="media-container-row"><DIV class="title col-12 col-md-8"><H2 class="align-center pb-3 mbr-fonts-style display-2">What is flux balance analysis?</H2></DIV></DIV></DIV></SECTION><SECTION class="mbr-section article content1 cid-r6INV48uxA" id="content1-pu"><DIV class="container"><DIV class="media-container-row"><DIV class="mbr-text col-12 col-md-8 mbr-fonts-style display-7">Flux balance analysis (FBA) is a widely used method for studying genome-scale metabolic network reconstructions. These networks serve as a roadmap for all of the known metabolic reactions in an organism and the genes that encode each enzyme. In contrast with more traditional approaches, such as utilizing a system of ordinary differential equations, flux balance analysis comparatively uses very little information in terms of the enzyme kinetic parameters and concentration of metabolites in the system. It achieves this by making two assumptions, steady state and optimality. The first assumption is that the modeled system has entered a steady state, where the metabolite concentrations no longer change, i.e. in each metabolite node the producing and consuming fluxes cancel each other out. The second assumption is that the organism has been optimized through evolution for some biological goal, such as optimal growth or conservation of resources. The steady-state assumption reduces the system to a set of linear equations, which is then solved to find a flux distribution that satisfies the steady-state condition subject to the stoichiometry constraints while maximizing the value of a pseudo-reaction (the objective function) representing the conversion of biomass precursors into biomass [3,4].
</DIV></DIV></DIV></SECTION><SECTION class="mbr-section article content1 cid-r6INV4Ss8Q" id="content1-pw"><DIV class="container"><DIV class="media-container-row"><DIV class="mbr-text col-12 col-md-8 mbr-fonts-style display-7"><STRONG>Figure 1</STRONG> Displays the formulation of an FBA problem, highlighting the keystone steps in the procedure [5].</DIV></DIV></DIV></SECTION><SECTION class="mbr-section content4 cid-r6INV5eqqg" id="content4-px"><DIV class="container"><DIV class="media-container-row"><DIV class="title col-12 col-md-8"><H2 class="align-center pb-3 mbr-fonts-style display-2">Overview of our approach and implementation:</H2></DIV></DIV></DIV></SECTION><SECTION class="mbr-section article content1 cid-r6INV5DMKy" id="content1-py"><DIV class="container"><DIV class="media-container-row"><DIV class="mbr-text col-12 col-md-8 mbr-fonts-style display-7">Here we apply FBA to model our culture of <I>Synechococcus elongatus</I> PCC 7942. The goal of our model is to predict how the cyanobacteria population along with metabolite concentrations of our culture over time and under different culture conditions. Our model aids in studying the effects of various growth conditions and genetic modification of our population, on the culture production of biofuel. We utilized a genomic reconstruction, Model: iJB785, found on the Biochemical Genetic and Genomic (BiGG) knowledge-base (or k-base) of large scale metabolic reconstructions as a basis of our model [6,7]. Once the model Genome Scale Model (GEM) was obtained, we utilized The COnstraint-Based Reconstruction and Analysis Toolbox (COBRA Toolbox) through MATLAB to further modify and adjust the model to represent the addition of reactions, adjustment of constraints of exchange reactions, and tuning of culture conditions.</DIV></DIV></DIV></SECTION><SECTION class="mbr-section content4 cid-r6INV5YHSU" id="content4-pz"><DIV class="container"><DIV class="media-container-row"><DIV class="title col-12 col-md-8"><H2 class="align-center pb-3 mbr-fonts-style display-2">FBA <I>Synechococcus elongatus</I> PCC 7942 Model Modification:</H2></DIV></DIV></DIV></SECTION><SECTION class="mbr-section article content1 cid-r6INV6o86s" id="content1-q0"><DIV class="container"><DIV class="media-container-row"><DIV class="mbr-text col-12 col-md-8 mbr-fonts-style display-7">We focused on creating an FBA simulation for <I>Synechococcus elongatus</I> PCC 7942 that would provide us with the amount of sucrose produced and exported out of the cell, along with supplementary results. This signifies the importance of our model and design question.
<DIV>The FBA simulation, when all the proper parameters are set, yields a flux of 13.5896 mmol g DW<SUP>−1</SUP> h<SUP>−1</SUP> in a 100ml culture population (Figure 2 shows the metabolic flux map for the simulation). We also wanted to use a simulation to calculate the growth curve of our cyanobacteria population in both self-shaded conditions and non self-shaded conditions (Jared Broddrick [7]) shown in Figure 2. Similar conditions in previous experiments have yielded 36.1 mg liter<SUP>−1</SUP> h<SUP>−1</SUP> [9]. Our simulated value of sucrose exportation suggests that through the optimization of exchange reactions along with introduction of cscB, and overexpression of SPP/SPS, the production of sucrose for use in biofuel manufacturing can be increased by a large means. The metabolic reconstruction map can be seen in Figure 1. The model also incorporates a script that simulates a growth curve of the modeled population that employs both a non self-shading and a self-shading approach (Figure 3). Both these model outputs can be iteratively adjusted through adjustment of input parameters to study different the effects of different growth mediums, different light conditions, and various genetic manipulations to find the conditions that promote rapid and sustained biofuel production.
</DIV></DIV></DIV></DIV></SECTION><SECTION class="mbr-section article content1 cid-r6INV77LMu" id="content1-q2"><DIV class="container"><DIV class="media-container-row"><DIV class="mbr-text col-12 col-md-8 mbr-fonts-style display-7"><STRONG>Figure 2 </STRONG>Zoomed fragment of a simulated model. Reaction nodes (rectangle) contain IDs and flux rates. Metabolite nodes (ellipse) are marked by IDs. Forward and reverse fluxes (arrows) are green and blue correspondingly, zero fluxes are grey. The thickness of an arrow is proportional to the rate of flux. Rectangles of blocked and exchange reactions are marked by red. The red rectangle draws attention to reactions of interest in the model.
</DIV></DIV></DIV></SECTION><SECTION class="mbr-section article content1 cid-r6INV8gjY8" id="content1-q4"><DIV class="container"><DIV class="media-container-row"><DIV class="mbr-text col-12 col-md-8 mbr-fonts-style display-7"><STRONG>Figure 3 </STRONG>Simulated growth curve for self-shading and no shading in-silico set up</DIV></DIV></DIV></SECTION><SECTION class="mbr-section article content1 cid-r6INV8HtUK" id="content1-q5"><DIV class="container"><DIV class="media-container-row"><DIV class="mbr-text col-12 col-md-8 mbr-fonts-style display-7">Understanding that our original model has followed our expectations of showing increased levels of sucrose production and exportation, we wanted to understand the role our cyanobacterial population would have in carbon sequestration. The Representative Concentration Pathways (RCPs) form a set of greenhouse gas concentration and emissions pathways designed to support research on impacts and potential policy responses to climate change [12,13]. We specifically looked at RCP8.5, which is a extrapolated data set that combines assumptions about high population and relatively slow income growth with modest rates of technological change and energy intensity improvements, leading in the long term to high energy demand and GHG emissions in absence of climate change policies. Compared to the total set of Representative Concentration Pathways (RCPs), RCP8.5 corresponds to the pathway with the highest greenhouse gas emissions [10].
</DIV></DIV></DIV></SECTION><SECTION class="mbr-section content4 cid-r6INV9aZrG" id="content4-q6"><DIV class="container"><DIV class="media-container-row"><DIV class="title col-12 col-md-8"><H2 class="align-center pb-3 mbr-fonts-style display-2">FBA <I>Synechococcus elongatus</I> PCC 7942 Model Output:</H2></DIV></DIV></DIV></SECTION><SECTION class="mbr-section article content1 cid-r6INV9ALZF" id="content1-q7"><DIV class="container"><DIV class="media-container-row"><DIV class="mbr-text col-12 col-md-8 mbr-fonts-style display-7"> We focused on creating an FBA simulation for <I>Synechococcus elongatus</I> PCC 7942 that would provide us with the amount of sucrose produced and exported out of the cell, along with supplementary results, that what signifies the importance of our model and design question.
<DIV>The FBA simulation, when all the proper parameters are set, yields a flux of 13.5896 mmol g DW<SUP>−1</SUP> h<SUP>−1</SUP> in a 100ml culture population. Similar conditions in previous experiments have yielded 36.1 mg liter<SUP>−1</SUP> h<SUP>−1</SUP> [9]. Our simulated value of sucrose exportation suggests that through the optimization of exchange reactions along with introduction of cscB, and overexpression of SPP/SPS, the production of sucrose for use in biofuel manufacturing can be increased by a large means. The metabolic reconstruction map can be seen in Figure 1. The model also incorporates a script that simulates a growth curve of the modeled population that employs both a non self-shading and a self-shading approach (Figure 2). Both these model outputs can be iteratively adjusted through adjustment of input parameters to study different the effects of different growth mediums, different light conditions, and various genetic manipulations to find the conditions that promote rapid and sustained biofuel production.
</DIV><DIV>Understanding that our original model has followed our expectations of showing increased levels of sucrose production and exportation, we wanted to understand the role our cyanobacterial population would have in carbon sequestration. The Representative Concentration Pathways (RCPs) form a set of greenhouse gas concentration and emissions pathways designed to support research on impacts and potential policy responses to climate change [12,13]. We specifically looked at RCP8.5, which is a extrapolated data set that combines assumptions about high population and relatively slow income growth with modest rates of technological change and energy intensity improvements, leading in the long term to high energy demand and GHG emissions in absence of climate change policies. Compared to the total set of Representative Concentration Pathways (RCPs), RCP8.5 corresponds to the pathway with the highest greenhouse gas emissions [10].
</DIV><DIV>We found the length and width of a cyanobacteria to be 2512μm and 1118μm [11], respectively, on average, which lead us to approximate the average surface area of a single face of the cyanobacteria to be 2.8084 x 10<SUP>-6</SUP> square meters, assuming a rectangular shape (as cyanobacteria are rod shaped, this is a close approximation). This would allow us to assume that all of this area of landfills in the U.S. were suitable for cyanobacteria culture, approximately 560,000 acres of area, and if we approximate the weight of a single cyanobacteria to be 2.8 x 10<SUP>-10</SUP> mg [14] and using this we can find the dry weight of the population to be 2.2595 x 10<SUP>5</SUP> mg. We can further assume that the available and suitable land in the world, there would be much more than just 560,000 acres of land suitable for cyanobacteria implementation (which is roughly 0.02% of all the land in the U.S.). There is 15.77 billion acres of habitable land on earth, and if we take 5% of all the land on earth and use it for cyanobacteria culturing, through similar calculations as shown above, we would find that the dry weight of the population would be 3.1814 x 10<SUP>8</SUP> mg. Although this seems like a substantial amount of cyanobacteria to be used in world wide carbon sequestration, if we were to find the changes in world-wide CO<SUB>2</SUB> concentration upon introduction of this population, there would be no appreciable change. This method assumes a single layer of cyanobacteria, which drastically limits the amount of cells used in this calculation. Nevertheless, this simple calculation highlights the ability for our research to have worldwide impact on clean energy and lower carbon dioxide levels in the atmosphere. We ran simulations where substantial changes in carbon dioxide atmospheric concentration can be viewed (Figure 4,5).
</DIV></DIV></DIV></DIV></SECTION><SECTION class="mbr-section article content1 cid-r6INVaubz1" id="content1-q9"><DIV class="container"><DIV class="media-container-row"><DIV class="mbr-text col-12 col-md-8 mbr-fonts-style display-7"><STRONG>Figure 4 </STRONG>  Stimulation of CO<SUB>2</SUB> atmospheric concentration using RCP dataset. Assume that cyanobacteria introduction occurs in 2020, and population growth occurs until 2120</DIV></DIV></DIV></SECTION><SECTION class="mbr-section article content1 cid-r6INVbg2iq" id="content1-qb"><DIV class="container"><DIV class="media-container-row"><DIV class="mbr-text col-12 col-md-8 mbr-fonts-style display-7"><STRONG>Figure 5</STRONG>  Stimulation of CO<SUB>2</SUB> atmospheric concentration assuming atmospheric levels of CO<SUB>2</SUB> remain constant after 2120. Assume that cyanobacteria introduction occurs in 2020, and population growth occurs until 2120. </DIV></DIV></DIV></SECTION><SECTION class="mbr-section content4 cid-r6INVbBd28" id="content4-qc"><DIV class="container"><DIV class="media-container-row"><DIV class="title col-12 col-md-8"><H2 class="align-center pb-3 mbr-fonts-style display-2">Concluding summary:
</H2></DIV></DIV></DIV></SECTION><SECTION class="mbr-section article content1 cid-r6INVc2QsZ" id="content1-qd"><DIV class="container"><DIV class="media-container-row"><DIV class="mbr-text col-12 col-md-8 mbr-fonts-style display-7">The importance of our model is two fold: It first convinces us that the optimization of sucrose exportation in <I>Synechococcus elongatus</I> PCC 7942 for biofuel production through growth media, genetic variations, and overall culture conditions, leads to significant increases in production fluxes of our objective. Our model also signifies the potential for worldwide carbon dioxide sequestration through implementation of our optimized cyanobacteria. Our work on the model reaffirms our research goal and will allow us to have a deeper understanding of our project. 
</DIV></DIV></DIV></SECTION><SECTION class="mbr-section content4 cid-r6INVcoJjh" id="content4-qe"><DIV class="container"><DIV class="media-container-row"><DIV class="title col-12 col-md-8"><H2 class="align-center pb-3 mbr-fonts-style display-2">References:</H2></DIV></DIV></DIV></SECTION><SECTION class="mbr-section article content1 cid-r6INVcQiAx" id="content1-qf"><DIV class="container"><DIV class="media-container-row"><DIV class="mbr-text col-12 col-md-8 mbr-fonts-style display-7"><P><SPAN style="font-size: 1rem;">1. Barrera-Martínez, I., et al. (2011). &quot;A simple metabolic flux balance analysis of biomass and bioethanol production in Saccharomyces cerevisiae fed-batch cultures.&quot; Biotechnology and Bioprocess Engineering 16(1): 13-22.
</SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;"> </SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;">2. Baart, G. J. E. and D. E. Martens (2012). Genome-Scale Metabolic Models: Reconstruction and Analysis. Neisseria meningitidis: Advanced Methods and Protocols. M. Christodoulides. Totowa, NJ, Humana Press: 107-126.
</SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;"> </SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;">3. MacGillivray, Michael et al. “Robust Analysis of Fluxes in Genome-Scale Metabolic Pathways.” Scientific Reports 7 (2017): 268. PMC. Web. 15 Oct. 2018.
</SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;"> </SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;">4. C. Damiani et al. “An ensemble approach to the study of the emergence of metabolic and proliferative disorders via Flux Balance Analysis.” Flux Balance Analysis: an ensemble approach
</SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;"> </SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;">5. Orth, Jeffrey D., Ines Thiele, and Bernhard Ø. Palsson. “What Is Flux Balance Analysis?” Nature biotechnology 28.3 (2010): 245–248. PMC. Web. 15 Oct. 2018.
</SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;">6.  Schellenberger, J., Park, J. O., Conrad, T. M., and Palsson, B. &amp;Oslash;., &quot;BiGG: a Biochemical Genetic and Genomic knowledgebase of large scale metabolic reconstructions&quot;, BMC Bioinformatics, 11:213, (2010).
</SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;">7. Broddrick, Jared T. et al. “Unique Attributes of Cyanobacterial Metabolism Revealed by Improved Genome-Scale Metabolic Modeling and Essential Gene Analysis.” Proceedings of the National Academy of Sciences of the United States of America 113.51 (2016): E8344–E8353. PMC. Web. 15 Oct. 2018.
</SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;"> </SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;">8.  Schellenberger, Jan et al. “Quantitative Prediction of Cellular Metabolism with Constraint-Based Models: The COBRA Toolbox v2.0.” Nature Protocols 6.9 (2011): 1290–1307. PMC. Web. 15 Oct. 2018.
</SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;">9. Rerouting Carbon Flux To Enhance Photosynthetic Productivity Daniel C. Ducat, J. Abraham Avelar-Rivas, Jeffrey C. Way, Pamela A. Silver Appl. Environ. Microbiol. Mar 2012, 78 (8) 2660-2668; DOI: 10.1128/AEM.07901-11
</SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;">10.  M. Meinshausen, S. Smith et al. (2011) &quot;The RCP GHG concentrations and their extension from 1765 to 2300&quot;, DOI 10.1007/s10584-011-0156-z, Climatic Change.
</SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;"> </SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;">11. Moronta-Barrios Félix, Espinosa Javier and Contreras Asunción (2013), Negative control of cell size in the cyanobacterium Synechococcus elongatus PCC 7942 by the essential response regulator RpaB, FEBS Letters, 587, doi: 10.1016/j.febslet.2013.01.023
</SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;"> </SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;">12.  Moss RH, Edmonds JA, Hibbard KA, Manning MR, Rose SK et al (2010) The next generation of scenarios for climate change research and assessment. Nature 463:747–756
</SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;"> </SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;">13. Van Vuuren DP, Edmonds J, Kainuma M, Riahi K, Thomson A, Hibbard K, Hurtt GC, Kram T, Krey V, Lamarque J-F, Matsui T, Meinshausen M, Nakicenovic N, Smith SJ, Rose SK (2011a) Representative concentration pathways: An overview. Climatic Change (this SI). doi: 10.1007/s10584-011-0148-z
</SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;"> </SPAN></P><P><SPAN style="color: rgb(118, 118, 118); font-size: 1rem;">14. http://bionumbers.hms.harvard.edu/files/Composition%20of%20an%20average%20E.%20coli%20Br%20cell-Neudhart.pdf
</SPAN></P></DIV></DIV></DIV></SECTION><SECTION class="cid-r6B3W0JndV" id="footer1-lf"><DIV class="container"><DIV class="media-container-row content text-white"><DIV class="col-12 col-md-3 mbr-fonts-style display-7"><H5 class="pb-3"><U>
                    2018 Stony Brook iGEM </U></H5><P class="mbr-text">The Stony Brook iGEM Team is proud to present to you their sweet and energy filled project! Made with love &lt;3 </P></DIV><DIV class="col-12 col-md-3 mbr-fonts-style display-7"><H5 class="pb-3"><U>
                    Contacts
                </U></H5><P class="mbr-text">
                    Email: igem.sbu@gmail.com                    </P></DIV><DIV class="col-12 col-md-3 mbr-fonts-style display-7"><H5 class="pb-3"><U>
                    Links
                </U></H5><P class="mbr-text"><A class="stinkylinky" href="https://2018.igem.org/Team:Stony_Brook/Team" target="_blank">Meet the Team</A><A class="stinkylinky" href="https://2018.igem.org/Team:Stony_Brook/Description" target="_blank">Description</A><A class="stinkylinky" href="https://2018.igem.org/Team:Stony_Brook/Attributions" target="_blank">Attributions</A><A class="stinkylinky" href="https://2018.igem.org/Team:Stony_Brook/Parts" target="_blank">Parts Overview</A></P></DIV></DIV></DIV></SECTION><SECTION class="engine"><A href="https://mobiri.se">Mobirise</A></SECTION><SECTION class="mbr-gallery mbr-slider-carousel cid-r6B3WpQODI" id="gallery1-lg"><DIV class="container" id="smolboi"><DIV><DIV data-app-prevent-settings="" class="mbr-slider modal fade carousel slide" tabindex="-1" data-keyboard="true" data-interval="false" id="lb-gallery1-lg"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><A class="carousel-control carousel-control-prev" role="button" data-slide="prev" href="#lb-gallery1-lg"><SPAN class="sr-only">Previous</SPAN></A><A class="carousel-control carousel-control-next" role="button" data-slide="next" href="#lb-gallery1-lg"><SPAN class="sr-only">Next</SPAN></A><A class="close" href="#" role="button" data-dismiss="modal"><SPAN class="sr-only">Close</SPAN></A></DIV></DIV></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>