Team:Stony Brook/Design
Home
iGEM SBU
Home
Team
Team Members Collaborations Attributions
Project
Description Design Experiments Notebook InterLab Model Results Wiki
Parts
Parts Overview Basic Parts
Safety
Human  Practices
Integrated Human Practices
Education & Engagement
Awards
Plant
Design
For information on the design of our BioBricks, go to the “Basic Parts” page under “Parts” in the menu.
For the plasmid backbones we inserted our constructs into, we used several plasmids developed and characterized by Dr. Susan Golden. The plasmids include pAM2991, pAM1414, and pAM1579 from Addgene.
Ribosome-Binding Site (RBS)
We decided to use the same RBS (BBa_K2596010) for every gene construct. The sequence used is “AAAGGAGGTTTAACCAA,” the end of part BBa_K390002. This includes the consensus RBS “AAAGGAGGT” from Synechocystis sp. PCC 6803 and the Shine-Delgarno core consensus sequence. Even though this RBS is from a different species than our own, S. elongatus PCC 7942, the two species are very closely related and so the RBS should function normally in our chassis. The reason for choosing this RBS over one directly from our chassis is because finding a well-characterized RBS specifically from our chassis proved difficult.
Characterization of cscB: pAM2991
pAM2991 is an IPTG-inducible vector, designed for overexpression of a gene inserted into the multiple cloning site. pAM2291 (as well as the other backbone plasmids) can be found on addgene under the Golden Lab. The smR gene confers resistance to two antibiotics: streptomycin and spectinomycin. We selected for the vector in cultures using spectinomycin in DH10-beta competent E. coli (this strain is naturally resistance to streptomycin) and both streptomycin and spectinomycin in our cyanobacteria. Additionally, we linearized this vector by cutting with both BamHI and EcoRI; by cutting with two enzymes, we minimized the chance for spontaneous re-ligation. We used this vector to characterize our sucrose symporter BioBrick, cscB. We also used this vector in an attempt to improve the enhanced yellow fluorescent protein (EYFP) BioBrick by codon optimization; unfortunately, we could not collect data on this BioBrick.
This vector is a suicide vector with neutral site II (NSII). In cyanobacteria, there are stretches of DNA in their genomes called neutral sites, which can be replaced with other DN without any detriment to the cell. Our “suicide vector” is composed of DNA that identically matches the neutral sites found in cyanobacteria. Our genes of interest and antibiotic resistance genes are flanked on both sides by the neutral site DNA (>300 bp on both sides). When our plasmid enters cells (it does this because the cells are naturally competent), the identical strands of DNA undergo a double homologous recombination event, and the DNA from the plasmid is integrated into the genome of the cyanobacteria. In the plasmid, there is no origin of replication for cyanobacteria, so this mechanism is the only way in which the exogenous DNA enters the cell. We screen colonies with antibiotics to ensure this event has taken place.
Characterization of Novel Promoters: pAM1414
pAM1414 is a promoterless luxAB reporter plasmid, designed for promoters inserted upstream of luxA to express both luxA and luxB. The luxAB bioreporter generates a light signal in the presence of decanal. The smR gene confers resistance to two antibiotics: streptomycin and spectinomycin. We selected for the vector in cultures using spectinomycin in HB101 competent E. coli and both streptomycin and spectinomycin in our cyanobacteria. We linearized this vector by digestion with BamHI and prevented re-linearization by dephosphorylation with calf intestinal phosphatase (CIP). We used this vector to characterize our novel promoters: cpc, cpc-560, idiA, and psbA2.
This vector is a suicide vector with neutral site I (NSI), and works in the same way as pAM2991.
Application of Novel Promoters: pAM1579
pAM1579 is another suicide vector lacking promoters and genes. It does have terminators, but after correspondence with McKenna Hicks, a member from the 2017 UCSC iGEM Team who worked with this vector, we realized that the terminators were oriented in the wrong direction. Therefore, we incorporated these terminators into the constructs for this part of the project. The KanR gene confers resistance to the antibiotics, kanamycin. Therefore, we selected for the vector in both DH10-beta competent E. coli and our cyanobacteria using kanamycin. We linearized this vector by cutting with both EcoRV and SalI; by cutting with two enzymes, we minimized the chance for spontaneous re-ligation. We intended to use this vector to characterize circuits pairing our novel promoters, psbA2 and idiA, with our sucrose genes, cscB and sps (sps did not get characterized in pAM2991 due to DNA synthesis issues but is referenced on the Project Description page).
This vector is a suicide vector with neutral site II (NSII), and works in the same way as pAM2991.
Gibson Assembly Overlap Design
We decided to approach construct insertion into vectors using Gibson Assembly to improve efficiency. For Gibson Assembly, we had to design overlaps between adjacent DNA sequences. Each overlap is 30 base pairs long, and was incorporated into the original DNA synthesis.
pAM2991 Overlaps
For pAM2991, because we only had to insert one construct at a time the overlaps were relatively easy to design. The 30 base pairs at each end of the linearized vector were added on to the ends of the the target gene (either cscB or EYFP), such that the promoter in the vector would be at the start of the gene.
pAM1414 Overlaps
For pAM1414, we also only had to insert one construct at a time. Therefore, the overlaps were relatively easy to design again. The 30 base pairs at each end of the linearized vector were added on to the ends of the the target promoter, such that the gene in the vector would be at the end of the promoter.
pAM1579 Overlaps 
For pAM1579, we had to insert four constructs at a time: two genes and a promoter for each gene. The order of the constructs after assembly is the cscB promoter, the cscB gene, the sps promoter, and the sps gene. For the cscB promoter, the first 30 base pairs overlap with the last 30 base pairs in the linearized pAM1579 vector. For the sps gene, the last 30 base pairs overlap with the first 30 base pairs in the linearized pAM1579 vector. For the overlaps between adjacent constructs, the same pattern was followed for each overlap: The last 15 base pairs of the first construct were added on to the beginning of the second construct, and the first 15 base pairs of the second construct were added on to the end of the first construct.
2018 Stony Brook iGEM 
The Stony Brook iGEM Team is proud to present to you their sweet and energy filled project! Made with love <3 
Contacts
Email: igem.sbu@gmail.com
Links
Meet the Team
Description
Attributions
Parts Overview
Mobirise
Previous Next Close
