<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Calgary_Attributions skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Calgary/Attributions</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Header</TITLE></P><DIV class="nav-bar"><DIV id="TopBar"><DIV class="nav" class="nav"><A href="#" class="toggle-menu">‚ò∞</A><UL class="nav-list nav-open" id="nav"><LI><A href="https://2017.igem.org/Team:Calgary">Home</A></LI><LI><A href="#" class="toggle-drop">Project</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Description">Problem/Description</A></LI><LI><A href="#" class="toggle-drop2">Synthesis </A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Synthesis">Synthesis Overview</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/BetaOxidation">Beta Oxidation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Glycolysis">Glycolysis</A></LI></UL><LI><A href="https://2017.igem.org/Team:Calgary/Secretion">Secretion</A></LI><LI><A href="#" class="toggle-drop2">Process  </A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Process">Process Overview</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/VFA_Fermentation">VFA Fermentation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/SolidLiquidSeparation">Solid-Liquid Separation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/PHB_Fermentation">PHB Fermentation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Extraction">Extraction</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Products">Products</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Model">Modelling </A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Journal">Notebook</A></LI><LI><A href="#" class="toggle-drop">Team</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Sponsors">Sponsors</A></LI><LI><A href="#" class="toggle-drop">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Human_Practices">HP Summary</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/HP/Silver">HP Silver</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/HP/Gold_Integrated">HP Gold/Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Engagement">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A href="#" class="toggle-drop">Judging</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/MedalRequirements">Medal Requirements</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/InterLab">Interlab</A></LI></DIV></DIV></DIV><DIV id="MainContent"><H1>Attributions</H1><H2>Accomplishments</H2><P>From January to April of 2017, our team got together for the first time over weekly meetings. As the majority of our team had no synthetic biology or iGEM experience, we spent a lot of this time learning about what synthetic biology is and how it works, as well as the infrastructure of iGEM. Over this period, our mentors (composed of some 2016 iGEM team members) held presentations to explain the parts registry, what human practices entails, Wiki requirements, and general iGEM advice. At this point, we also began looking into previous team's projects to get an idea of what is possible. A portion of this time was also dedicated to each team member acquiring research funding for the summer.</P><P>In mid-March, we decided on pursuing PHB production from <I>E. coli</I> to be used in wastewater systems, and the rest is history. Despite taking a large portion of April off for exams, we were able to do some literature search about our proposed methods to get a better idea of what our project will entail. In May, we began working full time, splitting our work between the lab, literature research, as well as all the other iGEM requirements such as human practices, fundraising, public engagement and Wiki design. </P><P>Over the course of the season, the team successfully designed and cloned genetic constructs into the iGEM pSB1C3 standard, characterized and submitted our parts, and <B>successfully synthesized and secreted PHB</B>! We sincerely thank all of the experts we consulted in the list below, as we would not have been able to inform the design of our project without their valuable feedback.</P><H2>General Support</H2><P><B>Dr. Mayi Arcellana-Panlilio: </B>Our fearless leader, Dr. Mayi was our primary supervisor over the summer. She guided us in every step of the project, from idea development to final result. Our team would not be where we are without her!

</P><P><B>Rachelle Varga: </B>Dr. Mayi‚Äôs right hand. Assisted with supervision, led discussions, shared protocols and taught laboratory techniques, edited content, and shared general knowledge about competition tips and general iGEM infrastructure.

</P><P><B>David Feehan: </B>Dr. Mayi‚Äôs left hand. Assisted with long hours of supervision, laboratory techniques, edited content, and shared musical talents with the team.

</P><P><B>Dr. Michael Kallos:</B> Assisted in acquiring funding for the team from the University of Calgary.

</P><P><B>Dr. Elke Monika Lohmeier-Vogel: </B>Attended weekly meetings to provide feedback on project developments and helped research the biochemical pathways of PHB production with team members.

</P><P><B>Dr. Anders Nygren: </B>Provided feedback at weekly meetings and critiqued our presentation. Assisted engineering students with process development and put team members in contact with additional professors specialized in manufacturing and polymers.

</P><P><B>Dr. Craig Jenne: </B>Attended weekly meetings and asked thought-provoking questions to ensure the team considered all aspects of safety, including the safety of the users and the surrounding environment.

</P><P><B>Dr. Fabiola Aparicio-Ting: </B>Assisted with human practices work while attending weekly meetings, and emphasized the importance of ethical and societal considerations of our project.

</P><P><B>Brad Prince: </B>Provided us with the cost analysis of PHB production, specifically, how much media is required to produce one kilogram of plastic, and how profit it could make. Offered general support on presentation and design, as well as the user interface of the 3D PHB Simulator.

</P><P><B>Dr. Christian Jacob: </B> Allowed us to work in his Swarm Design Lab to create our PHB Simulator, and provided valuable feedback about user interactive and educating the public through storytelling.

</P><P><B>Bruce Ramsay: </B>Donated approximately 250 grams of PHBcoHV and 50 grams of PHB to use in our extraction experiments. Also gave us suggestions for the demand of PHB products and their applications.

</P><P><B>Deirdre Lobb: </B>Helped ensure that all of our lab work was performed according to safety standards, and assisted in getting the whole team certified with all safety courses. Helped us order all necessary reagents. Also collaborated with our team to keep lab risk at a minimum by encouraging us to look towards alternative forms of human feces - this led us to find the NASA recipe for syn poo.

</P><P><B>Elena Fekete, Sid Goutam, Syed Jafri, David Nguyen, Nishi Patel, Nilesh Sharma, and Dan Ziemianowicz:</B> All members of the 2016 UCalgary team. They helped with the initial brainstorming of our project, general advice, and gave us need-to-know iGEM tutorials. These included how to use the parts registry, how to set up the Wiki, fundraising suggestions, and possible contacts that we may find useful.</P><H2>Synthesis &amp; Secretion Support</H2><P><B>Integrative DNA Technologies: </B>Synthesized our genetic constructs free of charge.

</P><P><B>Dr. Isabelle Barrette-Ng: </B>Assisted with troubleshooting transformation experiments, and also gave advice for proper cloning protocols.

</P><P><B>Dr. Sui-Lam Wong: </B>Donated <I>E. coli</I> BL21(DE3) to use as our chassis.

</P><P><B>Dr. Richard Moore</B> Donated competent <I>E. coli</I> DH5ùõº and his competency protocol to make our own competent cells.</P><H2>Process Support</H2><P><B>Dr. Nashaat Nassar: </B>Helped with process development in suggesting the method of chemical coagulation for extracting PHB from solution.

</P><P><B>Dr. Matthew Bamsey: </B>Put us in contact with microbiologists at the Canadian Space Agency and pointed the team to NASA‚Äôs public database to find critical assumptions and space travel information.

</P><P><B>Dr. Peter Dunfield: </B>Assisted in the formation of our project idea and provided us with contacts ad Polyferm.

</P><P><B>Christine Sharp, Marc Strous and the Solar Biocells Lab: </B> Let us use their HPLC to run experiments throughout the course of the summer, and provided suggestions for the design of our bioreactors. Also provided protocols to separate and quantify PHB via HPLC.

</P><P><B>Anirban Chakraborty and Jayne Rattray: </B> Trained us on using the HPLC and helped with setting up HPLC runs and methods for VFA and PHB quantification. Also helped us with troubleshooting HPLC runs.

</P><H2>Wiki Support</H2><B>Patrick Wu:</B> Provided us with wiki support, especially regarding how to set up templates as well as some formatting and layout troubleshooting.

<H2>Modelling Support</H2><P><B>Dr. Justin MacCallum: </B>Explained the importance of metabolomics in optimizing the synthesis of exogenous products in genetically engineered systems. Provided contacts with professors specialized in metabolomics and flux balance analysis. Informed about the ways that the Type I secretion system could be incorporated into our model in flux balance, which could improve the model's ability to simulate lab experiments.

</P><P><B>Dr. Jason de Koning: </B>Suggested the team optimize codons for expression of genes in <I>E. coli</I> and pointed students towards appropriate modelling software for kinetic modelling. Provided software that could be used for obtaining parameters, which are needed for the kinetic model.

</P><P><B>Dr. Ian Lewis: </B>Assisted the team in understanding and developing the beginnings of a model for flux balance in our PHB system. Referred to postdocs in his lab who worked on flux balance analysis. The team was told about ways flux balance could be used to inform the project and to verify our pathway design for an optimal production of PHB.

</P><H2>Human Practices Support</H2><P><B>Col. Chris Hadfield (CSA): </B>Encouraged us to consider all of the safety implications of bringing our process to space and provided us with useful contacts working in the field.

</P><P><B>Chancellor Dr. Robert Thirsk:</B> Offered feedback on the importance of considering the safety of astronauts in space and put the team in contact with Genome Alberta.

</P><P><B>Dr. Nicole Buckely (CSA):</B> Discussed the importance of considering microgravity in our experimental design and provided the team with constructive feedback about storage of our project‚Äôs materials on space crafts.

</P><P><B>Derek Thomas (Made in Space): </B>Confirmed that 3D printing has been used on the International Space Station and provided feedback concerning SLS 3D printing of our PHB product in microgravity environments.

</P><P><B>Carol Nelson (Alberta Environment and Parks): </B>Supplied the team with information about using landfill leachates as a project application. Although the team did not choose this application, she confirmed the feasibility of our idea in the future.

</P><P><B>Dr. Pascal Lee: </B>Stressed the importance of developing a document of assumptions and considerations before beginning designing our project. Also encouraged us to look into the Baseline Values and Assumptions Document so that we are using the same parameters for our process as the researchers at NASA.

</P><P><B>Marko Markicevic: </B>Provided us with detailed information about the regulation and testing performed during the wastewater treatment process, as well as the VFA concentrations. This information confirmed that using wastewater as a feedstock was feasible, but not the proper application for a pure culture of genetically engineered bacteria.

</P><P><B>Minds in Motion: </B>Gave us the platform to speak with Grades 4-6 and 7-9 campers about synthetic biology and expose them to some fun, hands-on learning activities.

</P><P><B>Pine Creek Wastewater Treatment Plant: </B>After touring the plant and consulting with their employees, we realized that a synthetic biology application into existing wastewater treatment plants was not feasible. We started looking towards alternative applications after our visit.

</P><P><B>Biology Graduate Student Association and Beakerhead: </B>Allowed us to assist in their Petri Dish Picasso event to open a discussion about genetically engineered organisms with the general public, and created some fun fluorescent artwork with us!

</P><P><B>Telus Spark: </B>Provided us with the opportunity to share the applications and advantages of bioplastics with the public at their Adults Only Night: Hack It event.

</P><H2>Team Involvement</H2><P><B>Alexandra Ivanova:</B> Worked on development of applied design through the exploration of NASA‚Äôs Baseline Values and Assumptions document and process assumptions through the Equivalent System Mass toolkit. Conducted experiments on the solid-liquid separation and PHB fermentation stages of process design. Led Minds in Motion summer camp workshops
and Contributed to human practices by developing the education material and lesson plans for the grade 5-6 and 7-9 Syn Bio workshops.


</P><P><B>Alina Kunitskaya: </B>Worked on process development, conducted experiments to determine the optimal operating temperature and duration for the VFA fermentation stage and to test PHB production under different fermentation conditions, and worked on quantification of VFA and PHB using HPLC and titrations. Also helped with flux balance analysis using MATLAB and organized meetings with experts in the space industry. 

</P><P><B>Amy Chen: </B>Part of the synthesis and modelling subgroups. Contributed to the background literature search to design genetic constructs. Helped design the genetic constructs used for synthesis. Cloned constructs into pET29b(+) vectors and, performed confirmation digest and confirmation sequencing. Synthesized and extracted PHB to be used in HPLC analysis. Ran SDS-PAGE experiments to confirm the production of protein in the transformed E. coli. Provided insight into the biological pathways to include and manipulate for mathematical modelling.

</P><P><B>Atika Ibrahim: </B>Part of synthesis and modelling subgroups. Worked on ligation of IDT geneBlocks and cloning the DNA into <I>E. coli</I> (BL21). Carried out experiments for synthesis of PHB. Looked into flux-balance analysis for synthesis of PHB and developed an initial model, which contained phaCBA genes to predict the optimal pathway for PHB production.

</P><P><B>Bilal Sher: </B>Worked on securing corporate sponsorship for the team, and helped secure donations for the Space Slam fundraising event. Researched solid-liquid separation and PHB extraction, as well as liaising with experts and sponsors.

</P><P><B>Helen Wang: </B>Part of the Synthesis group and helped to design and codon optimize constructs for synthesis and cloned constructs into pET29b(+) vectors. Took charge of iGEM team  collaborations including the first Canadian iGEM newsletter and wet-lab work with <A href="https://2017.igem.org/Team:McMasterU">iGEM McMaster University</A>. Delivered a Synthetic Biology talk at Sir Winston Churchill High School, as well as the Minds in Motion workshops.

</P><P><B>Jacob Grainger: </B>Worked on the development and implementation of the PHB secretion phasin-HlyA Biobrick. Cloned secretion parts into the appropriate plasmids, assisted with PHB production and extraction, and performed the InterLab study.

</P><P><B>Kaitlin Schaaf: </B>Helped design the secretion parts and clone them into our <I>E. coli</I>. Created a ‚Äúsuper‚Äù plasmid with both PHB-producing and PHB-secreting genes, which was used to carry out secretion assays. Conducted research to identify barriers to synthetic biology for human practices, participated in the Minds in Motion workshops, Beakerhead Petri Dish Picasso, and the Telus Spark Adults Only Night. 

</P><P><B>Lalit Bharadwaj: </B>Helped design and codon optimize the parts for the phasin-hemolysin secretion construct. Conducted SDS-PAGE experiments to identify the production of the phasin proteins. Attended various meetings for integrated human practises and public engagement activities through high school visits and the Telus Spark Adults Only Night. Also helped with the designing and ordering of the team attire. 

</P><P><B>Maliyat Aniqa Noor: </B>Researched possible extraction methods for PHB and ran lab experiments to test these methods as well as possible quantification methods. Worked on developing and testing lab protocols for measuring volatile fatty acids and PHB on a High-Performance Liquid Chromatography column.

</P><P><B>Michaela Olsakova: </B>Designed and programmed the Wiki. Created the 3D PHB Simulator program in Unreal Engine, and participated in outreach activities such as the Minds in Motion summer camp. Led the team in fundraising efforts, particularly the Space Slam fundraiser, and worked with the Human Practices team to organize tours to the wastewater treatment plants, and as well as interviews for various industry professionals.

</P><P><B>Preetha Gopalakrishnan: </B>Helped to codon optimize constructs, clone constructs into PET29B vectors, and ran experiments to characterize constructs for the synthesis subgroup. Led the team in human practices efforts: attended interviews with experts, organized outreach events (such as TELUS Spark Adults Only Night showcase and the Master‚Äôs Academy talk), and identified barriers to accessing synthetic biology. Also organized the production of the iGEM Manual video series script.

</P><P><B>Sam Wilton-Clark: </B>Designed, codon optimized and removed restriction sites from the hemolysin-phasin secretion construct. Cloned in the DNA and performed assays to measure PHB secretion. Completed the interlab study (including its affiliated wiki pages) and performed general lab duties such as creating competent cells and preparing empty plasmid backbones. Participated in the high school outreach component of human practices.

</P><P><B>Tricia Lim: </B>Helped design the synthesis constructs. Developed branding for the team and project including logos, uniforms, and promotional material. Designed graphics for the presentation, wiki, poster, and the deliverables used in workshops to engage and educate the public about synthetic biology. Designed the cover pages for the Canadian and International iGEM Newsletters and created a Sponsorship Package design with a template that is open to the public to use. Participated in a Minds in Motion Workshop during the summer.



</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>