<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Calgary_Journal skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Calgary/Journal</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Header</TITLE></P><DIV class="nav-bar"><DIV id="TopBar"><DIV class="nav" class="nav"><A href="#" class="toggle-menu">☰</A><UL class="nav-list nav-open" id="nav"><LI><A href="https://2017.igem.org/Team:Calgary">Home</A></LI><LI><A href="#" class="toggle-drop">Project</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Description">Problem/Description</A></LI><LI><A href="#" class="toggle-drop2">Synthesis </A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Synthesis">Synthesis Overview</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/BetaOxidation">Beta Oxidation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Glycolysis">Glycolysis</A></LI></UL><LI><A href="https://2017.igem.org/Team:Calgary/Secretion">Secretion</A></LI><LI><A href="#" class="toggle-drop2">Process  </A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Process">Process Overview</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/VFA_Fermentation">VFA Fermentation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/SolidLiquidSeparation">Solid-Liquid Separation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/PHB_Fermentation">PHB Fermentation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Extraction">Extraction</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Products">Products</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Model">Modelling </A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Journal">Notebook</A></LI><LI><A href="#" class="toggle-drop">Team</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Sponsors">Sponsors</A></LI><LI><A href="#" class="toggle-drop">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Human_Practices">HP Summary</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/HP/Silver">HP Silver</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/HP/Gold_Integrated">HP Gold/Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Engagement">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A href="#" class="toggle-drop">Judging</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/MedalRequirements">Medal Requirements</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/InterLab">Interlab</A></LI></DIV></DIV></DIV><DIV id="MainContent"><H1>Notebook</H1><BUTTON class="accordion"><H3> Synthesis</H3></BUTTON><DIV class="panel"><P>Detailed protocols for experiments mentioned in the journal can be found on our <A href="https://2017.igem.org/Team:Calgary/Experiments">Experiments</A> page.

</P><H3>Week 1 (May 1 - May 5, 2017): Uploaded Sequences onto Benchling</H3><P>The synthesis group focused on completing lab safety courses and laboratory training, which involved following commonly used protocols, and took inventory of our lab supplies. 

We also looked into the genes needed for synthesizing PHA in <I>E. coli </I>. We examined many organisms capable of producing PHB. 
<UL><LI><I>Pseudomonas putida</I></LI><LI><I>Pseudomonas aeruginosa</I></LI><LI><I>Aeromonas caviae</I></LI><LI><I>Ralstonia eutropha</I></LI></UL>
We also looked at different genes for the pathways (beta-oxidation and glycolysis) of interest.
After finalizing the genes, the constructs were uploaded on benchling as follows:
<UL><LI>Promoter T7, spacer sequence, RBS (B0034LINK), FadE<SUB><I>E. coli</I></SUB>, FadD<SUB><I>E. coli</I></SUB>, PhaJ4<SUB><I>P.
 putida</I></SUB>, PhaC<SUB><I>P. aeruginosa</I></SUB></LI><LI>Promoter T7, spacer sequence, RBS (B0034LINK), FadE<SUB><I>E. coli</I></SUB>,  FadD<SUB><I>E. coli</I></SUB>, PhaJ <SUB><I>A. caviae</I></SUB>, PhaC<SUB><I>P. aeruginosa</I></SUB></LI><LI>LacO operator, spacer sequence, promoter T7, spacer, RBS (B0034), FadE<SUB><I>E. coli</I></SUB>, FadD<SUB><I>E. coli</I></SUB>, PhaJ4<SUB><I>P. putida</I></SUB>, PhaC<SUB><I>P. aeruginosa</I></SUB></LI><LI>LacO operator, spacer sequence, promoter T7, spacer, RBS (B0034), FadE<SUB><I>E. coli</I></SUB>, FadD<SUB><I>E. coli</I></SUB>, PhaJ<SUB><I>A. caviae</I></SUB>, PhaC <SUB><I>P. aeruginosa</I></SUB></LI><LI>phaCAB<SUB><I>R. eutropha </I></SUB> with T7 (constitutive) on lac I</LI><LI>phaCBA<SUB><I>R. eutropha</I></SUB> with T7 (constitutive) on lac I double vector</LI><LI>Hybrid promoter with number 8</LI><LI>T7 (constitutive), spacer sequence + RBS + spacer sequence(B0034LINK) + PhaC1 ap +  phaJ4 <SUB><I>P. putida</I></SUB>+ fadE<SUB><I>E. coli</I></SUB> + fadD<SUB><I>E. coli</I></SUB> + 10 nucleotide spacer sequence + RBS (B0034) + spacer sequence + phaCAB<SUB><I>R. eutropha</I></SUB></LI></UL></P><H3>Week 2 (May 8 - May 12, 2017): Designing constructs and protocol for media</H3><P>This week, we worked on researching protocol for media to be used for bacterial growth. Our mentors and advisers raised questions concerning the containment of the bacteria. This led our group to look into an antibiotic-free selection system; a number of methods for antibiotic-free selection were brought up. We also looked into the possibility of using Bacterial Artificial Chromosome (BAC) to deliver our inserts due to concern over the large size of the insert. Ultimately, we decided that, if time allows, we will consider pursuing the antibiotic-free selection method and BAC.

The lab training we participated in this week allowed us to practice protocols for performing transformations, preparing competent cells, and performing cPCR using template DNA from colonies on a streak plate. We also practiced agarose gel electrophoresis.
</P><P>This week, the details of the plasmid design were discussed. We decided to use pET29b(+) as our vector because it has an inducible lacI and a T7 promoter. We also planned characterization experiments for PhaJC + FadE, and compare PhaCBA with the PhaCAB Biobrick from Tokyo 2012. We decided to code our parts on Benchling with the following template: junk-ENX (prefix)-RBS-cds-SNP (suffix)-kpn1 (restriction)-junk.

We finalized the three plasmid inserts with T7 promoters (<I>phaC1<SUB>P. aeruginosa</SUB>J4<SUB>P. putida</SUB></I> and <I>phaCBA<SUB>R. eutropha</SUB></I>) and double-checked the spacer sequences using BLAST. We also decided to add 6x-His tags after methionine to help with characterization in the later stages of the experiment. Restriction sites were decided on for each part so that the whole final biobrick can be ligated together (3 pieces for beta-oxidation PHA sequence and 2 pieces for glucose PHA synthesis).</P><H3>Week 3 (May 15 - May 19,2017): Codon optimization and ordering sequences </H3><P>The synthesis group optimized codons using this <A href="http://www.sci.sdsu.edu/~smaloy/MicrobialGenetics/topics/in-vitro-genetics/codon-usage.html"> tool</A>, removed a number of restriction sites, finalized our constructs, and ordered our sequences from IDT. We also researched the chemicals required for growth media and the characterization protocols we planned to perform.

We continued formulating and editing protocols for post-synthesis experiments such as chemical cell lysis, PHA extraction and purification, and Nile Red Fluorescence quantification. A table was compiled to compare the pros and cons of different procedures for processes such as PHA extraction and quantification.LINK</P><H3>Week 4 &amp; 5 (May 22 - June 2, 2017)</H3><P>This week, the entire iGEM team took a field trip to a Calgary Wastewater Plant to learn about wastewater management. This trip informed us of the applications of our project. After evaluating the economic feasibility of implementing our waste-to-plastic system for use in the municipal wastewater treatment plant, the team started to discuss other possible applications of our project (wastewater treatment, developing countries, landfills, or space). Each member of the group researched a different application of our project to present and discuss at the weekly lab meeting.

The group finalized the post-synthesis protocols that were researched last week and ordered our required chemicals for these experiments.</P><H3>Week 6 (June 5 - June 9, 2017): Restriction digest + electrophoresis</H3><P>While waiting for the rest of the ordered gblocks™ to arrive, the synthesis group assisted the efforts of the human practices and modelling groups. We practiced coding for the wiki and researched <I>E. coli</I> infections in space to evaluate its virulence and containment.</P><H3>Week 7 (June 12 - June 16, 2017): Plasmid mini prep</H3><P>We performed single digests on our Pet29b(+) vectors using restriction enzymes HindIII, Sal1, EcoRI, and KpnI to check that each enzyme worked. After the DNA was digested we ran the samples on 1% agarose gel. The gel showed bands representing linearized plasmids on the gel, which informed us that our restriction enzymes were functional. </P><DIV id="Caption"><B>Figure 1:</B> Gel electrophoresis of pET29b(+) vector digested with NotI, HindIII, SalI, and KpnI as controls. </DIV><H3>Week 8 (June 19 - June 23, 2017): Run Controls and Digest gBlocks </H3><P>We received streak plates of <I>E. coli</I> BL21(DE3) and <I>E. coli</I> DH5α containing pET29b(+) vectors from Dr. Wong’s lab. These stock plates were used to streak fresh plates of the <I>E. coli</I>. Glycerol stocks of these cells were also made to preserve them. We made overnight cultures of the <I>E. coli</I> DH5α and performed plasmid miniprep to obtain pET29b(+) vectors. When we tested the plasmids on the NanoDrop, the samples were contaminated and did not show high concentrations. We decided to re-make overnights of the <I>E. coli</I> DH5α and then perform plasmid miniprep again. This time, the NanoDrop results showed a successful miniprep of the pET29b(+) vectors. These pET29b(+) vectors were stored in the -20°C freezer. </P><H3>Week 9 (June 26 - June 30, 2017): </H3><P>We performed diagnostic testing of NotI in CutSmart<SUP>tm</SUP> buffer and HindIII in Fast Digest Buffer with RFP plasmids from the iGEM registry. Diagnostic testing was done because the manuals outlined digestion of NotI in Fast Digest Buffer and HindIII in CutSmart™. We wanted to see if one of the buffers would work for both restriction enzymes. We then ran the digests on 1% agarose gel. The results showed that the digest of NotI in CutSmart™ worked and that FastDigest Buffer did not work. This meant that we could use CutSmart™ buffer for both restriction enzymes. The resulting gel electrophoresis is shown below. </P><DIV id="Caption"><B>Figure 2:</B> Results from running plasmids containing RFP digested with NotI and HindIII.</DIV>

After we tested that the restriction enzymes are functional, we digested our PhaC and PhaBA gblocks from IDT and our pET29b(+) plasmid following the Restriction Digest protocol with the enzymes listed below and ligated the digested parts following the Ligation of DNA Inserts to Plasmid Backbone protocol.

<DIV id="Caption"><B>Table 1:</B> Restriction enzymes for the digestion of IDT inserts and pET29b(+) backbone </DIV><TABLE><TBODY><TR><TH>DNA</TH><TH>Enzymes</TH></TR><TR><TD>PhaC</TD><TD>NotI, HindIII</TD></TR><TR><TD>PhaBA</TD><TD>HindIII, KpnI</TD></TR><TR><TD>pET29b(+) for PhaC</TD><TD>NotI, HindIII</TD></TR><TR><TD>pET29b(+) for PhaBA</TD><TD>HindIII, KpnI </TD></TR></TBODY></TABLE><P>We then streaked a fresh plate of <I>E. coli</I> DH5α containing pET29b(+), made overnight cultures, and performed miniprep to get more pET vectors to use for future experiments.</P><H3>Week 10 (July 3 - July 7, 2017): Ligated gBlocks</H3><P>This week we ligated the digested PhaC and PhaBA each into pET29b(+) vectors. After ligation, we transformed our cells and incubated the plates overnight. The next day we chose four colonies from PhaC and PhaBA-transformed plates to make overnight cultures and a master plate from.  We isolated the plasmid from our overnight cultures to perform a confirmation digest of the transformed cells. We performed double or single digests of the plasmids from miniprep to check that the presence of the insert within the vector and to check the directionality of the insert. The restriction enzymes and the respective sites and expected bands on the plasmids are shown below. </P><DIV id="Caption"><B>Table 2:</B> Restriction enzymes and expected vector band sizes for digest confirmation</DIV><TABLE><TBODY><TR><TH>Transformed Plasmids</TH><TH>Enzymes</TH><TH>Expected band size</TH></TR><TR><TD>PhaC</TD><TD>For confirmation</TD><TD>HindIII, NotI</TD><TD>5.4 kb, 1.8 kb</TD></TR><TR><TD>For directionality</TD><TD>HincII</TD><TD>1.8 kb, 1.4 kb</TD></TR><TR><TD>PhaBA</TD><TD>For confirmation</TD><TD>HindIII, KpnI</TD><TD>5.3 kb, 2.1 kb</TD></TR><TR><TD>For directionality</TD><TD>HincII</TD><TD>4.4 kb, 3.0 kb</TD></TR></TBODY></TABLE><P>We ran gels of undigested and digested plasmids. When we compared the predicted Benchling digests with our results, we found that colony 1 from the transformed pET29b(+)-phaBA matched the expected bands, suggesting that the transformation was successful because the insert was in the plasmid and the direction was correct. However, the pET29b(+)-phaC transformants did not match or show corresponding bands with the digests performed on a transformed vector. The results are shown in the figure below.  </P><DIV id="Caption"><B>Figure 3:</B> Gel electrophoresis of pET29b(+)-phaC digested with HindIII and NotI (DD) and with HincII (RD) run on 1% agarose gel. </DIV><DIV id="Caption"><B>Figure 4: </B>Gel electrophoresis of pET29b(+)-phaBA digested with HindIII and KpnI (DD) and with HincII (RD) run on 1% agarose gel.</DIV><H3>Week 11 (July 10 -July 14, 2017): </H3><P>From the confirmation digest and gel electrophoresis from last week, <I>E. coli</I> with pET29b(+)-phaBA colony 1 seemed to have the correct insert. Therefore, we wanted to sequence the colony to confirm that the correct insert was in the plasmid. To do so, we designed and ordered primers for pET29b(+)-PhaBA and prepped our samples for sequencing. 

The <I>E. coli</I> transformed with pET29b(+)-phaC (colonies 1-4) did not pass the confirmation digest screening, but to double-check for the plasmids, we isolated, digested (NotI, HindIII and HincII), and ran gel electrophoresis on the plasmids from 6 more pET29b(+)-phaC transformant colonies (5-11) using with restriction enzyme digest. The results are shown below. </P><DIV id="Caption"><B>Figure 5: </B> Gel electrophoresis of pET29b(+)-phaC digested with HindIII and NotI (DD) and with HincII (RD) on 1% agarose gel.</DIV><DIV id="Caption"><B>Figure 6: </B>Gel electrophoresis results of pET29b(+)-phaC digested with HindIII and NotI (DD) and with HincII (RD) run on 1% agarose gel.</DIV><P>We planned out the rest of our experiments for the summer and set tasks to complete. We transformed our DH5α competent cells with each of our plasmid inserts. We ran digests to confirm that the transformants actually carried our plasmids. We also developed a protocol for sequential digest of our pET29b(+) vectors because the NotI and HindIII restriction sites are very close together on the plasmid. 

PhaC-J insert from IDT was digested with HindIII and SalI, ligated with pET29b(+) vectors, transformed into competent DH5α cells, plated on Kan resistant plates, and left to incubate overnight at 37°C. The culturing tubes of the transformants from the <A href="https://2017.igem.org/Team:Calgary/Experiments">Transformation protocol</A> were kept in the 4°C fridge. No colonies were observed in the plate the next day. To troubleshoot why the cells did not grow, plasmids were isolated from the inoculation tube with the transformed culture that was kept in the 4°C fridge. Confirmation digest was performed on these plasmids. The gel electrophoresis did not work. Therefore, we decided to redigest our gblock, ligate, and transform it again. 

Our fadE and fadD DNA inserts from IDT were digested with HindIII/SalI and SalI/KpnI, respectively following the Restriction Digest protocol. A salt solution was made to adjust the final salt concentrations. After heat inactivation, the digested DNA was stored at -20°C for the weekend. </P><H3>Week 12 (July 17 -July 21, 2017): </H3><P>The ligated pET29b(+)-phaC and pET29b(+)-phaCJ were transformed into competent DH5α cells and plated on Kan LB agar plates. Colonies appeared on these plates. A master plate and overnight cultures were made to screen these transformants. The overnight culture was used to isolate plasmids from. The plasmids were digested and screened for confirmation of insert and directionality. The enzymes used are shown below along with their respective restriction sites and expected band size. The screening results showed that there was no insert. </P><DIV id="Caption"><B>Table 3: </B>Restriction enzymes and expected vector band sizes for digest confirmation </DIV><TABLE><TBODY><TR><TH>Transformed Plasmids</TH><TH>Enzymes</TH><TH>Expected band size</TH></TR><TR><TD>pET29b(+)-phaC</TD><TD>For confirmation</TD><TD>HindIII, NotI</TD><TD>5.4 kb, 1.8 kb</TD></TR><TR><TD>For directionality</TD><TD>AscI, SphI</TD><TD>6.1 kb, 1.1 kb</TD></TR><TR><TD>pET29b(+)-phaCJ</TD><TD>For confirmation</TD><TD>HindIII, NotI</TD><TD>5.4 kb, 2.2 kb</TD></TR><TR><TD>For directionality</TD><TD>Xmal</TD><TD>5.0 kb, 1.5 kb, 1.1 kb</TD></TR></TBODY></TABLE><DIV id="Caption"><B>Figure 7:</B> Gel electrophoresis of pET29b(+)-phaC digested with HindIII and NotI (DD) and with AScI and SphI (RD) run on 1% agarose gel.</DIV><DIV id="Caption"><B>Figure 8:</B> Gel electrophoresis of pET29b(+)-phaCJ digested with HindIII and NotI (DD) and with XmaI (RD) run on 1% agarose gel.</DIV><P>Last week, we digested fadE and fadD inserts and stored them. This week, we ran the digested B beta-oxidation part on a low melting point (LMP) gel and recovered the part by excising it from the gel. The gel containing the B Beta oxidation insert was melted at 65°C and ligation protocol was performed. We then transformed the plasmid into competent DH5α. C Beta Oxidation was also ligated with pET29b(+) vectors and transformed into competent DH5α. The pET29b(+)-fadE and pET29b(+)-fadD transformants colonies were observed after the overnight incubation. A master plate and overnight cultures of selected colonies from each plate were made. The vectors were isolated, digested for confirmation of insert, and ran 1% agarose gel. </P><DIV id="Caption"><B>Table 4:</B> Restriction enzymes and expected vector band sizes for digest confirmation </DIV><TABLE><TBODY><TR><TH>Transformed Plasmids</TH><TH>Enzymes for Insert Confirmation</TH><TH>Expected Band Sizes</TH></TR><TR><TD>pET29b(+)-fadE</TD><TD>HindIII, SalI</TD><TD>5.4 kb, 2.5 kb</TD></TR><TR><TD>pET29b(+)-fadD</TD><TD>KpnI, SalI</TD><TD>5.3 kb, 1.8 kb</TD></TR></TBODY></TABLE><DIV id="Caption"><B>Figure 9:</B> Gel electrophoresis of pET29b(+)-fadE digested with HindIII and SalI (DD) and pET29b(+)-fadD digested with KpnI and SalI run on 1% agarose gel. </DIV><H3>Week 13 (JULY 24 - JULY 28, 2017)</H3><P>Because pETE29b(+)-phaC and pET29b(+)-phaCJ did not transform with the inserts, the gBlocks were digested, ligated, and transformed this time in pET-RFP plasmids following protocol the pETRFP Digestion protocol.   

pET29b(+)-fadE colony 2 and pET29b(+)-FadD colony 3 seemed to have worked so the plasmids were isolated by miniprep and sent in for sequencing using the designed primers. The pET29b(+)-FadD sequencing results were incorrect. However, to double-check, we prepared a master plate and overnight cultures of a few colonies from the plate of transformants. Miniprep was performed and the plasmids were digested and were run on 1% agarose gel. The results did not indicate the successful ligation of the insert. 

To keep the pET29b(+) <I>E. coli</I> DH5α fresh, a new streak plate was made. An overnight culture was made and plasmids were isolated by miniprep and stored in the -20°C freezer for future use. </P><H3>Week 14 (July 31 - Aug 4)</H3><P>This week, we digested, ligated, transformed pET29b(+)-phaC and pET29b(+)-phaCJ into competent DH5α using pET29b-RFP. However, the transformations did not work. Therefore, the phaCJ and PhaC inserts were digested, ligated, and transformed again. However, transformants did not grow. We ran control experiments with the enzymes to check their functionality. The results showed that the enzymes were functional. Ligation was performed again using quick ligase for the A beta-oxidation insert and the PhaC insert and then transformed again. However, the transformation did not work. Because the LB tube was clear we rationalized that there was no cell growth from the 1-hour incubation in the shaker after transforming. The cells in plain LB from the transformation the previous day were spun down and resuspended and plated and incubated again overnight.</P><P>
Made overnight cultures of pet29B(+)-fadE and pUCIDT-fadE. Miniprepped pET29b(+)-fadE and the other in pUCIDT-fadE. 
</P><DIV id="Caption"><B>Figure 10:</B> Gel electrophoresis of pET29b(+)-fadE digested with HindIII and SalI (DD) run on 1% agarose gel.</DIV><P>
Overnight cultures of pUCIDT-fadE were prepared for confirmation digest where it was digested with the enzymes below. </P><DIV id="Caption"><B>Table 5:</B> Digestion of pUCIDT-fadE</DIV><TABLE><TBODY><TR><TH>Tube 1</TH><TH>Tube 2</TH></TR><TR><TD><UL><LI>pUCIDT-fadE tube 1</LI><LI>0.5 uL DNA</LI><LI>1 uL HindIII-HF</LI><LI>1 uL SalI-HF</LI><LI>1 uL 10x CS buffer</LI><LI>6.5 uL ddH<SUB>2</SUB>O</LI></UL></TD><TD><LI>pUCIDT-fadE tube 2</LI><LI>4 uL DNA</LI><LI>1 uL HindIII-HF</LI><LI>1 uL SalI-HF</LI><LI>1 uL 10x CS buffer</LI><LI>3 uL ddH<SUB>2</SUB>O</LI></TD></TR></TBODY></TABLE><P>We ran the digested DNA on 1.5% agarose gel at 70V for 90 minutes and the results are shown below. </P><DIV id="Caption"><B>Figure 11:</B>  Gel electrophoresis of pUCIDT-fadE digested with HindIII and SalI (DD) run on 1.5% agarose gel.</DIV><P>pET29b(+)-fadD was digested from gblocks and ligated with digested pET29b vectors using quick ligase and transformed into DH5α. Five overnight cultures of the pET29b(+)-fadD plate from August 2nd plate were made; colonies 1 to 5 were used. Colonies 1 to 4 had no cell growth, but we performed plasmid miniprep on colony 5 and resuspended the plasmids in TE buffer. Nanodrop was performed and the concentration was 154 ng/uL. The pET29b(+)-fadD plasmid was digested with KpnI and SalI and then run on a gel at 100V for 40 minutes. The results are shown below; the ligation did not work properly. Therefore, we will be digesting this part from gBlocks once again to ligate them into the proper vectors. </P><DIV id="Caption"><B>Figure 12:</B> Gel electrophoresis of pET29b(+)-fadD digested with KpnI and SalI run on 1% agarose gel. </DIV><H3>Week 15 (AUG 8 - AUG 13)</H3><P><B>phaCJ and PhaC</B></P><P>The transformed pET29b(+)-phaCJ and PhaC plates had film of mold on the agar. Therefore, phaCJ and phaC were digested again from their gBlock. More backbone digested out of the pET29b-RFP vectors was also needed. This time, an ethanol precipitation step was used in the sequential digest of pET29b-RFP using HindIII and NotI. The two samples of digested pET29b-RFP vectors were run on a 1% agarose gel. 3uL of sample 1 and 5uL of sample 2 were loaded into the wells after mixing with loading dye and ddH2O. The samples were run at 100V for 30 minutes. No RFP insert bands were not observed on the gel. We thought that this was due to the low concentration of DNA loaded. Therefore, we ran more on 1% LMP gel; 6uL of sample 1 and 8uL of sample 2. However, there was still no band observed. The remainder volume of the two samples of digested pET29b-RFP vectors were stored in the -20°C freezer. </P><P>We thought that the reason why no bands were seen may have been due to improper digestion and decided to digest more pET29b-RFP vectors to isolate the backbone for ligations. More overnight cultures of DH5α pET29b-RFP were made. The vectors were isolated by miniprep, digested sequentially, and then run on LMP gel. Bands indicated that the digestions did not seem to work. Sam performed miniprep of more pET29b-RFP vectors for overnight cultures in three separate tubes labelled A, P, and S. However, only the tube labelled S recovered bands. The vector was digested and run on a 1% LMP gel, but the bands did not show the correct size of predicted fragments. 
We decided to wait until next week to continue experiments to isolate the pET29b backbone out of the pET29b-RFF. </P><P><B>fadE</B></P><P>fadE in pUCIDT-Kan was digested with HindIII and SalI and run on a 1.5% LMP agarose gel. We observed the expected band at 2.5kb. </P><DIV id="Caption"><B>Figure 13:</B> Gel electrophoresis of pET29b(+)-phaCJ digested with KpnI and SalI run on 1.5% LMP gel.
</DIV><P>The 2.5kb band was excised from the 1.5% LMP gel and was ligated to the pET29b vector that was digested with HindIII and SalI. The ligation mixture was used to transform 2 tubes of competent DH5a cells because there was an excess of ligation mixture. One transformation used 45uL of the ligation mixture and the other used a lower volume of 25uL. The 2 tubes of transformed competent DH5a cells were plated on LB-Kan plates and left to grow overnight at 37 degrees. Bacterial colonies were observed on the transformed plates. The 2 plates of <I>E. coli</I> transformed with B beta-oxidation were used to make master plates and overnight cultures for confirmation digest next week; 3 overnight cultures of the cells transformed with the higher volume of ligation mixture and 8 overnight of the cells transformed with the lower volume of ligation mixture.</P><P><B>fadD</B></P><P>Our ligation of fadD to the pET29b vectors did not work last week. Therefore, we digested the C beta-oxidation gBlock and the pET29b vectors with KpnI and SalI once again. We ligated the insert and the vector and transformed competent DH5a cells with the ligation mixture and plated 100uL of the mixture on an LB-Kan plate. The transformation was successful because colonies were seen on the plate. Four colonies were used from the transformation plate to make overnight cultures and a master plate. We planned to use the overnight cultures for confirmation digests for the following week. </P><H3>Week 16 (AUG 14 - AUG 18)</H3><P><B>Confirmation Digest of pET29b(+)-phaCJ, pET29b(+)-phaC, pET29b(+)-fadE, and pET29b(+)-fadD</B></P><P>The digested products from pET29b(+)-phaCJ, pET29b(+)-phaC, pET29b(+)-fadE, and pET29b(+)-fadD were run on a 1% agarose gel. </P><DIV id="Caption"><B>Table 6:</B> Restriction enzymes and expected vector band sizes for digest confirmation </DIV><TABLE><TBODY><TR><TH>Transformed Plasmids</TH><TH>Enzymes for Insert Confirmation</TH><TH>Expected Band Sizes</TH></TR><TR><TD>pET29b(+)-phaCJ</TD><TD>NotI, HindIII</TD><TD>5.4kb, 2.2kb</TD></TR><TR><TD>pET29b(+)-phaC</TD><TD>NotI, HindIII</TD><TD>5.4kb, 1.8kb</TD></TR><TR><TD>pET29b(+)-fadE</TD><TD>HindIII, SalI</TD><TD>5.4 kb, 2.5 kb</TD></TR><TR><TD>pET29b(+)-fadD</TD><TD>SalI, KpnI</TD><TD>5.3kb, 1.8kb/td&gt;
</TD></TR></TBODY></TABLE><P><B>pET29b(+)-phaCJ and pET29b(+)-phaC</B></P><P>Because the gel confirmations for the isolation of the digested pET29b backbone did not work last week, we ran the pET29b-RFP digested vectors from August 10th on a 1% LMP gel to confirm the success of the digestion. We ran the remaining digested vector samples stored in the freezer from last week loading dye; 10uL of sample 1 and 4.5 uL of sample 2. The bands were observed this time. There was one band at 5.2kb and another blurry band around the hundred bp location for the RFP insert digested out of the vector. The gel was excised to isolate the backbone at 5.2 kb. </P><P>We successfully isolated the pET29b backbone needed for ligations to our inserts. An in-gel ligation was performed with the isolated pET29b vectors and the digested A beta-oxidation and PhaC inserts. Competent DH5α cells were transformed with the ligated vectors containing phaCJ and phaC.  The transformation for pETE29b(+)-phaCJ and pET29b(+)-phaC did not work. We spun down the transformed cells in the plain LB stored in the fridge from the previous day and resuspended them in 50uL LB Kan and plated the cells. However, the plates were not placed in the incubator after plating when we checked on them the next day. There were no colonies seen on the plates. Therefore, we placed the plates in the incubator for another day. Bacterial colonies were observed on the plate when we observed them the next day. We made a master plate of the 8 of the transformed colonies for both pET29b(+)-phaCJ and pET29b(+)-phaC and made overnight cultures for restriction digest confirmation the next day. We screened 8 A beta-oxidation colonies the following day by performing miniprep on overnight cultures. </P><DIV id="Caption"><B>Figure 14:</B> Gel electrophoresis of pET29b(+)-phaCJ digested with KpnI and SalI run on 1% agarose gel. </DIV><DIV id="Caption"><B>Figure 15:</B> Gel electrophoresis of pET29b(+)-phaC digested with NotI and HindIII and pET29b(+)-fadD digested with SalI and KpnI run on 1% agarose gel.</DIV><P><B>pET29b(+)-fadE</B></P><P>We performed plasmid miniprep on the pET29b(+)-fadE overnight cultures made at the end of last week. A double digest was performed on the isolated pET29b(+)-fadE using HindIII and SalI. We also digested the pUCIDT-fadE plasmids and ran the results on a gel </P><DIV id="Caption"><B>Figure 16:</B> Gel electrophoresis of pET29b(+)-fadE digested with HindIII and SalI run on 1% agarose gel.  </DIV><P><B>pET29b(+)-fadD</B></P><P>We performed plasmid miniprep on the transformed overnight cultures made a the end of last week. The plasmids were digested using SalI and KpnI.</P><DIV id="Caption"><B>Figure 17:</B> Gel electrophoresis of pET29b(+)-phaC digested with NotI and HindIII and pET29b(+)-fadD digested with SalI and KpnI run on 1% agarose gel.</DIV><P><B>PCR of phaC gblock</B></P><P>This week, we also performed PCR on phaC gBlock to amplify the sequence to produce more phaC gblocks in case we need to digest more. The PCR did not work. We attempted PCR again with Taq polymerase instead of pfu and diluted the primers. The results showed a band the size of the template, but the gel bands appeared smeared. </P><DIV id="Caption"><B>Figure 18:</B> PCR of phaC gblock from IDT</DIV><H3>Week 17 (AUG 21 - AUG 27)</H3><P>We were having troubles cloning in PhaC into a vector. We theorized that this was due to the restriction sites being placed close to each other. We decided to try blunt ending PhaC into the vector containing PhaBA.
To prepare the samples for the blunt ending, 2 samples of pET29b were digested using HindIII; the 2 microcentrifuge tubes were labelled 1 and 2. After digestion, the enzyme mix, dNTPs and blunting buffer were added to each of the two tubes. The mixture was heat inactivated at 70 degrees for 10 minutes. DNA cleanup was performed on the mixture in the two tubes. The DNA was resuspended in the elution buffer. We performed a nanodrop on the DNA but the concentration was negative, which indicated that the DNA cleanup did not work. </P><P>The gel electrophoresis results from last week confirmed the successful transformation of DH5a with the ligated vector and inserts; pET29b(+)-phaCJ, pET29b(+)-fadE, and pET29(+)-fadD. Overnight colonies of the cells transformed with these complete ligated inserts and vectors were made from the master plate. Overnight cultures of pET29b(+)-phaC colony 2, pET29b(+)-phaCJ colony 1 and 6, and pET29b(+)-fadD were made; two sets of cultures were made for PhaC-pET29b colony 2, A beta-oxidation-pET29b colony 1, and C beta-oxidation-pET29b colony 4. Miniprep was performed on the overnight cultures. </P><P>One set of isolated plasmids were resuspended in ddH2O and sent for sequencing confirmation (PhaC-pET29b colony 2, A beta-oxidation-pET29b colony 1 and 6, and C beta-oxidation-pET29b colony 4). The other set was resuspended in TE buffer for digestion confirmation (PhaC-pET29b colony 2, A beta-oxidation-pET29b colony 1, and C beta-oxidation-pET29b colony 4) and ligation with other parts (C beta-oxidation ligation with A beta-oxidation). </P><P>XhoI and HindIII were used to digest PhaC-pET29b colony 2, A beta-oxidation-pET29b colony 1, and C beta-oxidation-pET29b colony 4. The samples were mixed with DNA loading dye and loaded into the wells of a 1% agarose gel. The gel was run at 80V for 40 minutes. </P><P>The results for PhaC did not seem to work because there were four bands observed. C beta-oxidation digested with HindIII and XhoI seemed to work but no bands were observed on the gel for C beta-oxidation digested with SalI and KpnI.</P><P>We decided to do another digestion confirmation. PhaC-pET29b colony 2 was digested with NotI and HindIII, A beta-oxidation was digested with XhoI and HindIII, and C beta-oxidation colony 4 was digested with SalI and KpnI. The digested samples were run on a 1% LMP gel together with undigested PhaC as a control. Two bands were observed on the gel.</P><P>Because the results from the digestions were not conclusive, we made more overnight cultures of A beta-oxidation colony 1 and PhaC colony 2, and C beta-oxidation colony 4 for plasmid miniprep and digestion confirmation experiments. The digested samples were run on 1% LMP. The bands seemed very smeared for the lanes with loaded DNA. We searched online for reasons why the bands appeared this way. We concluded that the DNA may have been degraded due to the continuous freeze-thaw that the samples went through. We decided to modify our experiments so that plasmid miniprep and digestion confirmation happen on the same day so that the DNA is kept in good shape. </P><P>We decided to run another digestion confirmation this time alongside undigested ligated insert+vectors and undigested pET29b as controls. No bands were observed for many of the samples.We decided to make 2 sets of overnight cultures of the PhaC-pET29b colony 2, A beta-oxidation-pET29b colony 5, and C beta-oxidation-pET29b colony 4, and B beta-oxidation colony 5. The A beta-oxidation part was successfully transformed into <I>E. coli</I> BL21(DE3). This allowed us to begin preliminary experiments to test if our bacteria are capable of PHB production while we carried on ligation experiments. We discussed some experimental conditions for our tests. </P><H3>Week 18 (AUG 28 - AUG 31)</H3><P>We labelled the tubes with numbers seen in the table. The plasmids were isolated from these plasmids and samples from the isolated plasmids were digested with different enzymes. Some samples were left undigested. </P><DIV id="Caption"><B>Table 7:</B>Content of tubes used for restriction digest confirmation </DIV><TABLE><TBODY><TR><TH> Tube</TH><TH>Content</TH></TR><TR><TD>1</TD><TD>PhaC-pET29b colony 2</TD></TR><TR><TD>2</TD><TD>PhaC-pET29b colony 2</TD></TR><TR><TD>3</TD><TD>pET29b(+)-fadD colony 4</TD></TR><TR><TD>4</TD><TD>pET29b(+)-fadD colony 4</TD></TR><TR><TD>5</TD><TD>pET29b(+)-phaCJ colony 1</TD></TR><TR><TD>6</TD><TD>pET29b(+)-phaCJ colony 1</TD></TR><TR><TD>7</TD><TD>pET29b(+)-fadE colony 5</TD></TR><TR><TD>8</TD><TD>pET29b-fadE colony 5</TD></TR></TBODY></TABLE><P>Samples of the plasmids prepared from the overnight culture were digested with enzymes and run on a gel. Undigested samples were also run alongside the digested plasmids to serve as controls. 
Samples of plasmids from overnight culture tubes 1, 6, 4, and pET29b (from freezer stock) were digested with XhoI and HindIII. Samples from tube 7 and 4 were digested with SalI and KpnI. Undigested samples from tubes 1, 6, and 4 were run. </P><DIV id="Caption"><B>Figure 19:</B> Gel electrophoresis of pET29b(+)-phaC colony 2 digested with NotI and HindIII, pET29b(+)-phaCJ colony 1 digested with, and pET29b(+)-fadD colony 4 digested with SalI and KpnI run on 1% agarose gel. </DIV><P><B>PHB-production experiments</B></P><P>Calculations on M9 minimal media were done for the PHB production experiments. M9 salts were mixed with supplementation chemicals for the inoculation of PHB-producing bacteria. Stock solutions of acetic acid, propionic acid, and butyric acid were made for the VFA mixture. We required 5g/L of VFA within our final solution. The ratio from literature was a 1:2:1 ratio for propionic, acetic, and butyric acid. The final amount of the 5g/L stock solution we made from 99% acids was 2.86 mL of acetic stock solution, 2.07mL of butyric stock, and 2.01mL of propionic stock. </P><P>We prepared 12 overnight cultures of  <I>E. coli</I> BL21(DE3) transformed A beta-oxidation for 24-hour growth. The bacteria in these tubes were added to M9 media with different chemical supplements.Detailed protocols for this inoculation can be found on our <A href="https://2017.igem.org/Team:Calgary/Experiments">Experiments</A> page.</P><P>After the flasks were inoculated with respective chemicals, they underwent 16-24 hours of inoculation following the 24-hour overnight growth of the cells. The next day, chemical extraction of the contents of flasks took place. The protocol followed was from the experiments performed by the secretion team when they were testing out past iGEM parts that were capable of producing PHB. After extraction, a white powder was collected at the bottom of the tubes.  The results are shown in the picture below.</P><DIV id="Caption"><B>Figure 20:</B> Extracted PHB from PhaJPhaC-expressing cells cultured for 24 hours and inoculated with FSPS for 16 hours. PHB was extracted using TritonX-100, sodium hypochlorite, and ethanol in a series of washes and incubations. </DIV><H3>Week 19 (Sep 4 - Sep 8)</H3><P>We made overnight cultures of A+B beta-oxidation colonies 1-6 and pET29b(+)-phaCBA colonies 1-10 from the master plate that was streaked last week. We performed plasmid miniprep on these cells the next day. phaCBA was then digested using NotI and KpnI. We only digested pET29b(+)-phaCJ FadE (A+B) beta-oxidation colonies 3 and 5 with NotI and SalI; the other colonies did not show pure samples from the nanodrop. The plasmids were digested at 37 degrees in a water bath for 1.5 hours and then heat inactivated for 20 minutes immediately after. We ran the samples on the gel. We transformed pSB1C3 into <I>E. coli</I> DH5a and made overnight cultures of the <I>E. coli</I>. </P><DIV id="Caption"><B>Figure 21:</B> Gel electrophoresis of pET29b(+)-phaC digested with NotI and HindIII and pET29b(+)-fadD digested with SalI and KpnI run on 1% agarose gel. </DIV><P>We sent pET29b(+)-phaCBA for sequencing because the gel bands confirmed the successful transformation of the part. The sequencing results showed that colony 9 was successfully ligated and cloned into <I>E. coli</I> DH5a. 
Only one band was seen for A+B beta-oxidation so we decided to repeat the procedure again. We made overnights of the A+B beta-oxidation colonies 1-14 and performed the same procedures for plasmid isolation and digestion confirmation. </P><P>We also began putting our parts into the pSB1C3 backbone for the iGEM registry. We made overnight cultures of the <I>E. coli</I> transformed with pSB1C3, pET29b(+)-phaCJ, and pET29b(+)-phaCBA. We performed miniprep on the cultures and then digested the plasmids using XbaI. We then ligated them together and transformed them into DH5a. </P><P>During these three weeks, the team was busy preparing for aGEM. We had meetings to discuss the aGEM presentation and make revisions to the presentation. 

When we got back from aGEM, we focused on cloning our completed constructs pET29b(+)-phaCBA and pet29b(+)-phaCJ into the pSB1C3 plasmid for the iGEM registry. We used XbaI and PstI to clone in our pET29b(+)-phaCBA part (BBa_) and used EcoRI and NotI to half digestion (So not all NotI sites were cut off because then EcoRI would have been cut off the insert) to clone in our pet29b(+)-phaCJ. The parts were sequence confirmed. The DNA from these sequence-confirmed parts were then transferred onto the plate following the iGEM parts submission protocol. </P><H3>Week 20-21 (Sep 11 - Oct 1)</H3><P>During these three weeks, the team was busy preparing for aGEM. We had meetings to discuss the aGEM presentation and make revisions to the presentation. 

When we got back from aGEM, we focused on cloning our completed constructs pET29b(+)-phaCBA and pet29b(+)-phaCJ into the pSB1C3 plasmid for the iGEM registry. We used XbaI and PstI to clone in our pET29b(+)-phaCBA part (BBa_) and used EcoRI and NotI to half digestion (So not all NotI sites were cut off because then EcoRI would have been cut off the insert) to clone in our pet29b(+)-phaCJ. The parts were sequence confirmed. The DNA from these sequence-confirmed parts were then transferred onto the plate following the iGEM parts submission protocol. 
</P><H3> Week 22 (Oct 2 - Oct 8)</H3><P>We decided to perform a big-batch experiment to produce PHB. We made 3 replicates of 10mL O/N for each of <I>E. coli</I> BL21(DE3) transformed with pET29b(+)-phaCBA, phaCAB (Imperial College Part), and pET29b without any insert. We measured OD600 of the O/N after 24 hours. We measured the OD using 1mL of these O/N cultures, using LB Kan to blank the spectrophotometer. These cultures were then inoculated following our <A href="https://2017.igem.org/Team:Calgary/Experiments"> synthetic poop supernatant inoculation protocol</A>. Before extraction, we weighed and labelled some falcon tubes. We measured the OD of the inoculated cultures resuspended in PSB and continued with the rest of the extraction protocol. After the extraction products dried, we measured the weight of the flask again. The results of the big batch experiment are shown below.</P><DIV id="Caption"><B>Table 8: </B> Recorded OD<SUB>600</SUB> of the overnight cultures before inoculating the media containing synthetic poop supernatant, OD<SUB>600</SUB> of cells after growing in media for ~24 hours, then centrifuged and resuspended in (1x) PBS for extraction. Initial weight of 50 ml falcon tubes and final weights of tube containing PHB in grams was recorded.</DIV><H3>Week 23 (Oct 9 - Oct 15)</H3><P>To quantify the amount of PHB made by our phaCBA construct, we made 9 overnight cultures of <I>E. coli</I> BL21(DE3) that contained the pET29b(+) phaCBA plasmid in 10mL of LB-Kan broth. As a negative control, we also made 3 overnight cultures of <I>E. coli</I> BL21(DE3) that contained the pET29b(+) vector each in 10mL of LB-Kan broth. The OD was then measured using 1mL of the overnight cultures before they were inoculated in flasks that contained the synthetic poop supernatant for 24 hours. The next day, we performed the chemical PHB extraction protocol on the cultures in the inoculation flasks. After the contents of the falcon tubes were dried, we weighed the falcon tubes and compared the weight with the initial weight of the falcon tubes. Our results are shown in the table below. </P><DIV id="Caption"><B>Table 9: </B> Recorded OD<SUB>600</SUB> of the overnight cultures before inoculating the media containing synthetic poop supernatant, OD<SUB>600</SUB> of cells after growing in media for ~24 hours, then centrifuged and re-suspended in (1x) PBS for extraction. Initial weight of 50 ml falcon tubes and final weights of tube containing PHB in grams was recorded.</DIV><DIV id="Caption"><B>Figure 22:</B> PHB extracted from <I>PhaCBA</I>-expressing cells cultured for 24 hours and inoculated with FSPS for 16 hours. PHB was extracted using TritonX-100, sodium hypochlorite, and ethanol in a series of washes and incubation. The negative control tube contained <I>E. coli</I> transformed with the pET29b(+) vector (without insert); this sample underwent the same extraction process as the <I>phaCBA</I>-expressing cells</DIV><H3>Week 24 (Oct 16 - Oct 22)</H3><P>We also began to quantify PHB produced by our <I>E. coli</I> BL21(DE3) transformed with the pET29b(+)-phaCJ. Subcultures of <I>E. coli</I> BL21(DE3) were grown in 60 mL of various types media for 24 hours. Cultures were then induced with 0.1 mM IPTG to express <I>phaC1</I> and <I>phaJ4</I>. Cultures were incubated for 24 hours and OD<SUB>600</SUB> was measured. Absorbance was adjusted by diluting cultures with LB so they were all between 0.4-0.6, and nutrients and substrates were added to the flasks to allow for PHB production. The composition of each test culture is shown below:</P><DIV id="Caption"><B>Table 10: </B>Composition of test cultures</DIV><TABLE><TBODY><TR><TH><B><I>E. coli</I> BL21(DE3) with pET29B(+)-phaCJ in Glucose (positive control)</B></TH><TH><B><I>E. coli</I> BL21(DE3) with pET29B(+) in LB Media (negative control)</B></TH></TR><TR><TD><UL><LI> 10 ml of culture in LB+Kanamycin</LI><LI> 7 ml 20% Glucose </LI><LI> 100 uL MgSO<SUB>4</SUB></LI><LI> 5 uL CaCl<SUB>2</SUB></LI><LI> 10 ml M9 salts</LI><LI> 5 uL 1M IPTG</LI><LI> 20 ml dH<SUB>2</SUB>O</LI></UL></TD><TD><UL><LI> 10 ml of culture in LB+Kanamycin</LI><LI> &quot;Syn poo&quot; fermented supernatant </LI><LI> 100 uL MgSO<SUB>4</SUB></LI><LI> 5 uL CaCl<SUB>2</SUB></LI><LI> 10 ml M9 salts</LI><LI> 5 uL 1M IPTG</LI></UL></TD></TR><TR><TH><B><I>E. coli</I> BL21(DE3) with pET29B(+)-phaCJ + Fermented &quot;Syn Poo&quot; Supernatant Containing Glucose and VFAs</B></TH><TH><B><I>E. coli</I> BL21(DE3) with pET29B(+)-phaCJ + Pure VFAs</B></TH></TR><TR><TD><UL><LI> 10 ml of culture in LB+Kanamycin</LI><LI> 10 ml Syn Poo fermented supernatant</LI><LI> 100 uL MgSO<SUB>4</SUB></LI><LI> 5 uL CaCl<SUB>2</SUB></LI><LI> 10 ml M9 salts</LI><LI> 5 uL 1M IPTG</LI><LI> 23 ml dH<SUB>2</SUB>O</LI></UL></TD><TD><UL><LI> 10 ml of culture in LB+Kanamycin</LI><LI> VFAs
               </LI><LI> 410 uL propionic acid</LI><LI> 118 uL acetic acid</LI><LI> 55 uL butyric acid</LI></UL><LI> 100 uL MgSO<SUB>4</SUB></LI><LI> 5 uL CaCl<SUB>2</SUB></LI><LI> 10 ml M9 salts</LI><LI> 5 uL 1M IPTG</LI><LI> 30 ml dH<SUB>2</SUB>O</LI></TD></TR></TBODY></TABLE><P> 
Three replicates of each growth condition were performed. Their OD<SUB>600</SUB> readings were recorded:

</P><DIV id="Caption"><B>Table 11: </B>OD<SUB>600</SUB> measurements of the cultures in various growth conditions</DIV><TABLE><TBODY><TR><TH>Condition</TH><TH> OD<SUB>600</SUB> of replicate 1</TH><TH> OD<SUB>600</SUB> of replicate 2</TH><TH> OD<SUB>600</SUB> of replicate 3</TH></TR><TR><TD><I>E. coli</I> BL21(DE3) with pET29B(+) in LB Media (negative control)</TD><TD>0.571</TD><TD>0.531</TD><TD>0.487</TD></TR><TR><TD><I>E. coli</I> BL21(DE3) with pET29B(+)-phaJC in Glucose (positive control)</TD><TD>0.190</TD><TD>0.195</TD><TD>0.139</TD></TR><TR><TD><I>E. coli</I> BL21(DE3) with pET29B(+)-phaJC + Pure VFAs</TD><TD>0.140</TD><TD>0.134</TD><TD>0.146</TD></TR><TR><TD><I>E. coli</I> BL21(DE3) with pET29B(+)-phaJC + Fermented &quot;Syn Poo&quot; Supernatant Containing Glucose and VFAs</TD><TD>0.135</TD><TD>0.107</TD><TD>0.144</TD></TR></TBODY></TABLE><P> After pelleting the cultures, the cells were resuspended in 1 x PBS. The OD<SUB>600</SUB> readings were taken:

</P><DIV id="Caption"><B>Table 12: </B>OD<SUB>600</SUB> measurements of the cultures in various growth conditions resuspended in 1xPBS</DIV><TABLE><TBODY><TR><TH>Condition</TH><TH> OD<SUB>600</SUB> of replicate 1</TH><TH> OD<SUB>600</SUB> of replicate 2</TH><TH> OD<SUB>600</SUB> of replicate 3</TH></TR><TR><TD><I>E. coli</I> BL21(DE3) with pET29B(+) in LB Media (negative control)</TD><TD>2.659</TD><TD>2.001</TD><TD>2.899</TD></TR><TR><TD><I>E. coli</I> BL21(DE3) with pET29B(+)-phaJC in Glucose (positive control)</TD><TD>1.934</TD><TD>1.887</TD><TD>1.919</TD></TR><TR><TD><I>E. coli</I> BL21(DE3) with pET29B(+)-phaJC + Pure VFAs</TD><TD>0.510</TD><TD>0.571</TD><TD>0.532</TD></TR><TR><TD><I>E. coli</I> BL21(DE3) with pET29B(+)-phaJC + Fermented &quot;Syn Poo&quot; Supernatant Containing Glucose and VFAs</TD><TD>2.533</TD><TD>2.559</TD><TD>2.349</TD></TR></TBODY></TABLE><P>PHB was produced by this trial of pET29b(+)-phaCJ after the chemical extraction.</P><DIV id="Caption"><B>Figure 23:</B> PHB extracted from <I>PhaCJ</I>-expressing cells cultured for 24 hours and inoculated with FSPS for 16 hours. PHB was extracted using TritonX-100, sodium hypochlorite, and ethanol in a series of washes and incubation. The negative control tube contained <I>E. coli</I> transformed with the pET29b(+) vector (without insert). This sample underwent the same extraction process as the <I>phaCJ</I>-expressing cells</DIV><P>The PHB extraction experiments have shown us that we are able to produce PHB with our parts. We decided to perform SDS PAGE experiments to verify that our proteins are being expressed. We performed SDS page on proteins samples that we extracted from overnight cultures of <I>E. coli</I> BL21(DE3) containing pET29b(+) vector, pET29b(+) vector with phaCBA insert, and  pET29b(+) vector with phaCJ. The results of this initial SDS-PAGE is shown in the picture below.</P><DIV id="Caption"><B>Figure 24:</B> SDS-Page of <I>E. coli</I> BL21(DE3) containing pET29b(+) vector, pET29b(+) vector with phaCBA insert, and pET29b(+) vector with phaCJ which were induced with 0.1mM, 0.25mM, and 0.5mM of IPTG. Each lane contains 15uL of soluble protein (S) or insoluble protein (I) samples with 3 μL of protein loading dye. NEB Color Prestained Protein Standard, Broad Range (11–245 kDa) was used as the protein marker. The gels were run for 50 minutes at 30mA.</DIV><P>The expected sizes of the proteins produced by our constructs are as follows:</P><UL><LI>PhaC: 65.1kDa</LI><LI>PhaB: 27.2kDa</LI><LI>PhaA: 41.4kDa</LI><LI>PhaC: 63.4 kDa</LI><LI>PhaJ: 17.7 kDa</LI><P>The results showed that PhaJ was being expressed for our pET29b(+)-phaCJ construct in the insoluble phase. After trouble-shooting our experiment, we decided that the cells were not fully lysed because our insoluble proteins samples were stringy, indicating that the cell membrane was still intact for some cells. Therefore, we decided to modify the SDS-PAGE protocol by adding a sonication step after the 15 minutes incubation of cells resuspended in STET buffer and lysozyme.</P><P>We performed another SDS-PAGE with another batch of overnight cultures containing the same vectors. We sonicated the samples five times; we sonicated for 5 seconds and cooled the sample for 5 seconds on ice each time. In addition, we also decided to resuspend the insoluble protein after the soluble portion was separated in 100uL of Tris-glycine running buffer instead of the 50uL outlined on the original SDS-PAGE protocol. The results of this SDS-PAGE experiment are shown below. </P><DIV id="Caption"><B>Figure 25:</B> SDS-Page of <I>E. coli</I> BL21(DE3) containing pET29b(+) vector, pET29b(+) vector with phaCBA insert, and pET29b(+) vector with phaCJ which were induced with 0.1mM and sonicated. Each lane contains 15uL of soluble protein (S) or insoluble protein (I) samples with 3 μL of protein loading dye. NEB Color Prestained Protein Standard, Broad Range (11–245 kDa) was used as the protein marker. The gels were run for 45 minutes at 300V.</DIV>

The gel did not turn out properly so we ran the proteins samples that were left over again. The results are shown below.

<DIV id="Caption"><B>Figure 26:</B> SDS-Page of <I>E. coli</I> BL21(DE3) containing pET29b(+) vector, pET29b(+) vector with phaCBA insert, and pET29b(+) vector with phaCJ which were induced with 0.1mM and sonicated. Each lane contains 15uL of soluble protein (S) or insoluble protein (I) samples with 3 μL of protein loading dye. NEB Color Prestained Protein Standard, Broad Range (11–245 kDa) was used as the protein marker. The gels were run for 120 minutes at 100V. </DIV><H3>Week 25 (Oct 23 - Oct 29)</H3><P>Our SDS-PAGE results from last week showed that the cells were still not completely lysed following the original protocol due to the stringy state of the protein samples during loading. The protein PhaJ was also being expressed in the insoluble proteins, likely due to the inclusion bodies from culturing at high temperatures. Therefore, this week we decided to lower the temperature that the subculture grows in 28°C instead of 37°C after IPTG induction. However, our shaker temperature remained at room temperature when we checked on the cultures. Therefore, we still stored these cells after centrifuging and removal of the liquid in the -20°C freezer. We decided to make overnight cultures again and ensured that the temperature was maintained at 28°C. We then prepared the proteins from both of the samples from this week (subcultures incubated at room temperature 28°C). We made changes to the original protocol for SDS-PAGE by resuspending the frozen cell pellets in 500μL of STET buffer to reduce bubbles when sonicating and adding a sonication step. In addition, we resuspended the insoluble proteins in a higher volume of Tris-glycine running buffer; 100μL and 200μL for room temperature and 28°C samples, respectively. The gels were run for 50 minutes at 30mA. </P><DIV id="Caption"><B>Figure 26: </B>SDS-Page of <I>E. coli</I> BL21(DE3) containing pET29b(+) vector, pET29b(+) vector with phaCBA insert, and pET29b(+) vector with phaCJ which were induced with 0.1mM, incubated at room temperature and sonicated. Each lane contains 15uL of soluble protein (S) or insoluble protein (I) samples with 3 μL of protein loading dye. NEB Color Prestained Protein Standard, Broad Range (11–245 kDa) was used as the protein marker. The gels were run for 50 minutes at 30mA.</DIV><DIV id="Caption"><B>Figure 27: </B> SDS-Page of <I>E. coli</I> BL21(DE3) containing pET29b(+) vector, pET29b(+) vector with phaCBA insert, and pET29b(+) vector with phaCJ which were induced with 0.1mM, incubated at 28°C, and sonicated. Each lane contains 15uL of soluble protein (S) or insoluble protein (I) samples with 3 μL of protein loading dye. NEB Color Prestained Protein Standard, Broad Range (11–245 kDa) was used as the protein marker. The gels were run for 50 minutes at 30mA.</DIV><P>This week, we also tried to quantify PHB made by transformed <I>E. coli</I> with Nile red staining. We made overnight cultures of our transformed <I> E. coli</I>BL21(DE3), inoculated the culture with chemical media, and used Nile Red to stain the cells. More details are on our <A href="https://2017.igem.org/Team:Calgary/Experiments">Experiments</A> page. The results of the Nile Red staining are shown below.</P><DIV id="Caption"><B>Table 13: </B> Nile Red staining results for <I> E. coli</I>BL21(DE3) read at an absorbance of 535nm using a 96 well plate reader.</DIV><DIV id="Caption"><B>Table 14: </B> Nile Red staining results for <I> E. coli</I>BL21(DE3) read at an absorbance of 605nm using a 96 well plate reader.</DIV><BUTTON class="accordion"><H3> Secretion</H3></BUTTON><DIV class="panel"><H3> Week 1 (May 1-May 5, 2017) </H3><P> We are all fully trained in laboratory safety! We each completed 6 online courses and 2 seminars on lab safety and biosafety. In the lab we practiced important protocols, which include media preparation, overnight culture inoculation, preparing chemically competent cells, and transforming chemically competent cells (see protocols <A href="https://2017.igem.org/Team:Calgary/Experiments">here</A>)</P><P>We also researched ways that we will be able to use synthetic biology to extract synthesized Polyhydroxybutyrate (PHB) from cells, without the use of traditional chemical or mechanical lysis. 
</P><H3>Week 2 (May 8-May 12, 2017)</H3><P> We practiced more important laboratory techniques: plasmid minipreparation, preparing master plates of transformed <I>E. coli</I>, and agarose gel electrophoresis (protocols are <A href="https://2017.igem.org/Team:Calgary/Experiments">here</A>).</P><P>We also narrowed down our PHB extraction method to a hemolysin type I secretion system native to <I>E. coli</I>, like what was used by <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#secretion">Team SDU-Denmark</A> in 2016. We uploaded sequences of the hemolysin secretion tag (HlyA) fused to Phasin (<A href="http://parts.igem.org/Part:BBa_K2018024">Part:BBa_K2018024</A>) and two parts of the hemolysin membrane transport protein unit (HlyB : <A href="http://parts.igem.org/Part:BBa_K2018027">Part:BBa_K2018027</A> and HlyD: <A href="http://parts.igem.org/Part:BBa_K2018029">Part:BBa_K2018029</A> ) onto Benchling and began designing our parts to be ordered for synthesis from Integrated DNA Technologies (IDT). Each part will be upregulated by control under a T7 Promoter. Modifications to the parts we uploaded to Benchling were made:
<UL><LI>FLAG tags for easier protein expression validation were added to each coding sequence</LI><LI>Restriction sites were removed from <A href="http://parts.igem.org/Part:BBa_K2018029">Part:BBa_K2018029</A> to make the part RFC10,12,21,23,25, and 1000 compatible.</LI></UL></P><H3> Week 3 (May 15-May 19, 2017) </H3><P> This week we finished editing our secretion parts before ordering them from IDT. Our complete system with Phasin-HlyA tag, HlyB, and HlyD was split into two separate gBlocks: Secretion Part 1 (SP1) and Secretion Part 2 (SP2). SP1 contains Phasin-HlyA tag + HlyD and SP2 contains HlyB. A gBlock with just the phasin-HlyA tag was ordered as well so that we could compare secretion of the endogenous <I>E. coli</I> secretion system to our system with upregulated HlyB and HlyD. Restriction sites for the HindIII-HF enzyme were added to the ends of SP1 and SP2 so they could be easily ligated together. Before ordering, further modifications were made to the original parts we uploaded onto Benchling:
<UL><LI>All stop codons were changed to TAA because it is the most effective stop codon in <I>E. coli</I>.</LI><LI> Codons were optimized for protein expression in <I>E. coli</I> using this <A href="http://www.sci.sdsu.edu/~smaloy/MicrobialGenetics/topics/in-vitro-genetics/codon-usage.html"> tool</A>. 
</LI></UL></P><H3> Week 4 (May 23- May 26, 2017) </H3><P> Jacob and Sam began working on the Interlab Study. To begin, chemically competent DH5𝛼 <I>E. coli</I> were made and stored at -80°C in glycerol stocks, PSB buffer was made and diluted several times to develop a standard absorbance curve for the study, and the plate reader was calibrated.</P><P>Actual work on the Interlab Study was not very successful because there was little to no growth on plates of the transformed DH5𝛼, even though they were incubated at 37°C for two days. </P><P>The entire iGEM team was given very informative tour of the Pine Creek Wastewater Treatment Plant in Calgary. Many questions were answered and the information we have gained will be used in deciding which application route our project will take. More information about this can be found on our <A href="https://2017.igem.org/Team:Calgary/HP/Silver">Silver Human Practices</A> page 
</P><H3> Week 5 (May 29- June 2, 2017) </H3><P> Due to the lack of growth on the Interlab Study plates last week, chemically competent DH5𝛼 <I>E. coli</I> were made again because there may have been issues with the first set of competent cells created. However, there was still no growth on the plates inoculated with transformed cells. </P><P>On a positive note, our constructs from IDT that were ordered in week 3 arrived! </P><P>Outside of the lab, lots of work was dedicated to researching four possible applications of our project (space, wastewater treatment, landfill leachate, and developing countries). More about this research can be found on our <A href="https://2017.igem.org/Team:Calgary/Applied_Design">Applied Design</A> and <A href="https://2017.igem.org/Team:Calgary/HP/Silver">Silver Human Practices</A> pages. 
 </P><H3> Week 6 (June 5- June 9, 2017) </H3><P> Again, Jacob and Sam’s work on the Interlab Study was unsuccessful. Different protocols for making chemically competent cells and transformation were used, but there was still no growth observed. This strongly suggests that there is an issue with our DH5𝛼 cells, so next week we will get new cells and try again. Due to our problematic cells, work has not yet begun with our actual secretion construct from IDT. </P><P>Lalit and Kaitlin prepared necessary supplies that will be needed on Lalit’s trip to Winston Churchill High School on Monday, June 12, 2017. During this outreach, he will discuss synthetic biology with Grade 11 students, practice gel loading, and perform strawberry DNA extraction experiments with them. Outcomes of this outreach can be found on our <A href="https://2017.igem.org/Team:Calgary/Engagement">Engagement</A> page.
 </P><H3> June 11, 2017- Geekstarter iGEM Workshop, presented by MindFuel and Alberta Innovates </H3><P> Today was an iGEM workshop hosted by Geekstarter that was attended by the University of Alberta, University of Calgary, Urban Tundra High School, and University of Lethbridge iGEM teams. There were four speakers who gave ½ hour presentations on an introduction to syn bio, mathematical modeling, wiki design, and integration of art into syn bio. Later, we had the chance to speak with each presenter individually and ask questions specifically related to our project. Also, there was a collaboration period where we got to mingle with the other teams and discuss our projects with them. Overall it was a successful day where we learned a lot from the presenters and identified possible collaborations with the other teams.
 </P><H3> Week 7 (June 12- June 16, 2017) </H3><P> On Monday, June 12 Lalit and Sam visited Winston Churchill High School to present to Grade 11 students about Synthetic Biology (which we prepared for in Week 8). </P><P>We acquired new competent <I>E. coli</I> DH5α cells from Dr. Richard Moore and transformed them for the Interlab study. Finally, transformation was successful and there was colony growth of GFP cells. This supports the theory that there was something wrong with the cells that we were previously using. Therefore, for our work throughout the summer we will continue using the DH5α cells from Dr. Moore. Jacob and Sam used the new cells to successfully complete the laboratory component of the Interlab study this week. </P><P>Also, we isolated pSB1C3 from RFP <I>E. coli</I> Top10 for use in cloning our secretion inserts in the following weeks.
 </P><H3> Week 8 (June 19- June 23, 2017) </H3><P> To prepare ourselves for work with our IDT constructs, we performed various diagnostic tests with restriction enzymes. pSB1C3-RFP and -GFP were digested SpeI and EcoRI-HF, and confirmed on a 1% agarose gel. After some troubleshooting, we obtained two distinct bands on the gel, representing the pSB1C3 backbone and RFP/GFP insert. This indicates that these enzymes are functional and can be used on our IDT parts.</P><P>Outside of the lab, Jacob and Sam completed and submitted the online portion of the Interlab Study this week. Kaitlin and Lalit attended various meetings with other members of the team. 
 </P><H3> Week 9 (June 26- June 30, 2017) </H3><P> The digested pSB1C3-RFP and -GFP from last week were run on a 3% low melting-point agarose gel and both the plasmid backbones and the RFP/GFP inserts were excised. Our T4 DNA Ligase was tested by ligating the GFP inserts to the RFP backbones, and the RFP inserts to the GFP backbones. As we expected, bacteria transformed with the GFP insert/RFP backbone grew green colonies and bacteria transformed with RFP insert/GFP backbone grew red colonies. This indicated that our T4 DNA Ligase functioned properly and can be used with our IDT parts.</P><P>Jacob and Sam tested out a new protocol for making chemically competent <I>E. coli</I> DH5α cells from Richard Moore, who is part of Dr. Dong’s lab here at the Foothills Hospital. The cells’ competency was tested with 100 ng/µL RFP. The transformed cells grew extremely well with numerous colonies and a high transformation efficiency. The new protocol can be seen on our <A href="https://2017.igem.org/Team:Calgary/Experiments">Experiments</A> page.</P><P>Our IDT parts were digested and ligated into their corresponding backbones (pET29B or pSB1C3), then transformed into chemically competent <I>E. coli</I>  DH5α cells.
 </P><DIV id="Caption"><B>Table 1:</B>  The different secretion parts that were ordered from IDT, their restriction sites that will be used for molecular cloning, and the backbone that each part will be ligated to for transformation.</DIV><TABLE><TBODY><TR><TH> Part/Insert </TH><TH> Digested with... </TH><TH> Backbone </TH></TR><TR><TD> Phasin-HlyA Tag </TD><TD> EcoRI-HF, SpeI </TD><TD> pSB1C3 </TD></TR><TR><TD> Secretion Part 1 </TD><TD> XbaI, HindIII-HF </TD><TD> pET29B </TD></TR><TR><TD> Secretion Part 2 </TD><TD> HindIII-HF, NotI-HF </TD><TD> pET29B </TD></TR></TBODY></TABLE><H3> Week 10 (July 3-July 7, 2017) </H3><P> We isolated the plasmids of <I>E. coli</I> DH5α transformants from our ligated backbones/parts from Week 9. We screened 8 phasin-HlyA colonies, 8 SP1 colonies, and 2 SP2 colonies. The isolated plasmid of each colony was digested, then ran on a 1% agarose gel for confirmation. Some of the plasmids from each colony were left undigested as a control, double-digested with the same enzymes we used last week for ligation to check for the insert size, and digested with another enzyme to check for insert directionality: 
</P><DIV id="Caption"><B>Table 2:</B> The restriction enzymes that were used for screening transformant colonies of each secretion part and the corresponding band sizes that will be visible if colonies contain the part.</DIV><TABLE><TBODY><TR><TH> Part/Insert </TH><TH> Backbone </TH><TH> Digested with… </TH><TH> Expected Band Sizes </TH></TR><TR><TD> Phasin-HlyA Tag </TD><TD> pSB1C3 </TD><TD> EcoRI-HF, SpeI </TD><TD> 2.0 kb, 889 bp </TD></TR><TR><TD> Secretion Part 1 </TD><TD> pET29B </TD><TD> XbaI, HindIII-HF </TD><TD> 5.2 kb, 2.4 kb </TD></TR><TR><TD> Secretion Part 1 </TD><TD> pET29B </TD><TD> HincII </TD><TD> 6 kb, 1.5 kb </TD></TR><TR><TD> Secretion Part 2 </TD><TD> pET29B </TD><TD> HindIII-HF, NotI-HF </TD><TD> 5.4 kb, 2.2 kb </TD></TR><TR><TD> Secretion Part 2 </TD><TD> pET29B </TD><TD> EcoRV </TD><TD> 6.1 kb, 1.5 kb </TD></TR></TBODY></TABLE><P>We determined that only colony 1 of the phasin-HlyA tag had successfully received a plasmid with our part, as seen in Figure 1 below. Our other parts were not successfully transformed into our cells, which indicates that our ligation or transformation of SP1 and SP2 into <I>E. coli</I> DH5α had failed.  
 </P><DIV id="Caption"><B>Figure 1: </B>Screening results of colony 1 of DH5α transformed with pSB1C3-Phasin-HlyA Tag. Plasmid from the colony was digested with EcoRI-HF and SpeI (DD) then run on a 1% agarose gel at 100V for 30 minutes. The molecular ladder (L) is visible on the far left and the expected band sizes, obtained from Benchling Virtual Digest, are visible on the right. Undigested plasmid (U) was used as a control and the digests from colony 1 (which had successfully received our part) are visible in lanes 3-4.</DIV><H3> Week 11 (July 10- July 14, 2017) </H3><P> Colony 1 Phasin-HlyA Tag was sequenced and confirmed that these cells do in fact contain pSB1C3-Phasin-HlyA Tag. Consequently, we miniprepped DH5α colonies containing our pSB1C3-Phasin-HlyA plasmid and transformed into <I>E.coli</I> BL21(DE3), which we will use for protein expression. We also transformed <I>E. coli</I> DH5ɑ with a PHB synthesis biobrick, PhaCAB (<A href="http://parts.igem.org/Part:BBa_K934001">Part:BBa_K934001</A>) present in the iGEM 2017 distribution kit. This was done to establish a PHB-producing cell line and we do not have to wait for the Synthesis subgroup to finish their molecular cloning before we can begin testing the secretion of PHB. </P><P>On Friday, July 14 Kaitlin and some of the other iGEM team members visited the Grades 7-9 Minds in Motion summer camp at the University of Calgary. With the children, they discussed the potential implications of genetic engineering for the future and performed a strawberry DNA extraction experiment. The plans/worksheets from this workshop can be found on our <A href="https://2017.igem.org/Team:Calgary/Engagement">Engagement</A> page. 
 </P><H3> Week 12 (July 17-July 21, 2017) </H3><P>Our SP1 gBlock was re-digested with XbaI and HindIII-HF and ligated into a linearized pET29B backbone with XbaI/HindIII sticky ends. Colonies were run on a confirmation gel and the transformation of colony 3 appeared successful, as shown in Figure 2. A sample of SP1-colony 3 was submitted for sequencing.</P><P>Rachelle developed a line of cells that contain pET29B-RFP. This was done because HindIII and NotI restriction sites in pET29B are overlapping when there is no insert, making our double digests of SP2 with NotI and HindIII-HF very difficult. Thus, RFP was inserted to separate the two restriction sites and permit our digestion. Vector pET29B-RFP can be sequentially digested with HindIII-HF then NotI-HF to remove the RFP insert and leave the sticky ends intact for SP2 to be ligated to. </P><P>On the wiki, Sam and Kaitlin coded the Experiments page. Furthermore, we took individual and team photos so that we could start completing the Team page of the wiki.
 </P><DIV id="Caption"><B>Figure 2: </B>Screening results of 3 colonies of DH5α transformed with pET29B-SP1. Plasmids from the colonies were digested with HindIII-HF and XbaI (DD) or HincII (RD) then run on a 1% agarose gel at 100V for 30 minutes. The molecular ladder (L) is visible on the far left and the expected band sizes, obtained from Benchling Virtual Digest, are visible on the right. Undigested plasmid (U) was used as a control and the digests from colony 3 (the only SP1-containing colony) are visible in lanes 11-13.</DIV><H3> Week 13 (July 24 - July 28, 2017) </H3><P>The Experiments page of the wiki was completed, and we uploaded all of the general protocols used by all subgroups so far.</P><P> Our sequencing results of SP1-colony 3 showed that we had successfully transformed pET29B-SP1 into  <I>E.coli</I> DH5ɑ. With regards to SP2, we tried multiple times to perform our sequential restriction digest of the pET-RFP backbone with HindIII-Hf and NotI-Hf, and ran into multiple roadblocks. After various troubleshooting techniques, we were finally successful upon using a <A href="https://2017.igem.org/Team:Calgary/Experiments">DNA isolation protocol</A> in between the two digestion steps. Following this, we excised the pET29B backbone from a low melting-point agarose gel, and ligated with our SP2 part that had been digested with HindIII-HF and NotI. We also managed to create our first batch of PHB using the PHB synthesis biobrick (PhaCAB: <A href="http://parts.igem.org/Part:BBa_K934001">Part:BBa_K934001</A>) that we transformed into DH5α in Week 11. The PHB was extracted with chloroform and poured into a sheet (Figure 3). These protocols can be found <A href="https://2017.igem.org/Team:Calgary/Experiments">here</A>.</P><DIV id="OneCol"><DIV id="Caption"><B>Figure 3:</B>PHB sheet in a small petri dish. The PHB was produced by <I>E.coli</I> DH5ɑ cells that had been transformed with <A href="http://parts.igem.org/Part:BBa_K934001">Part:BBa_K934001</A> (PhaCAB) and lysed with chloroform to extract the plastic.</DIV><H3> Week 14 (July 31 - August 4, 2017) </H3><P>Work on cloning in our SP2 part was continued this week, using the same DNA isolation protocol as before in between the HindIII and NotI phases of our sequential digest of the pET29B vector. Using this technique, we successfully digested, ligated and transformed pET29B-SP2 into DH5α. 11 Colonies grew and these will be screened next week. </P><P>Lalit performed an SDS-PAGE in an attempt to qualify phasin production in our cells transformed with Phasin-HlyA Tag. Results were inconclusive because there were too many other bands around the same size as phasin, and the SDS-PAGE will be repeated at a later date.</P><P>Jacob transformed DH5α with a PhaCAB biobrick from Imperial College (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1149052">Part:BBa_K1149052</A>) that he will compare to the part from Tokyo Tech (<A href="http://parts.igem.org/Part:BBa_K934001">Part:BBa_K934001</A>) that has been used in previous weeks. </P><H3> Week 15 (August 8 - August 11, 2017) </H3><P>All of the pET29B-SP2 colonies from Week 14 were digested and screened; however, none contained our insert, indicating that the ligation/transformation had failed. We re-digested our SP2 gBlock with NotI-HF and HindIII-HF and ligated it into one of the pET29B backbones prepared Week 14 and transformed into chemically competent DH5ɑ.</P><P>Jacob Made LB plates that contained Nile Red and with them, he confirmed the presence of PHB granules in the cells transformed in Weeks 13 and 14 with PhaCAB biobricks from both the Imperial College and the Tokyo Tech teams. After overnight incubation in LB media with 3% glucose, he also successfully extracted the PHB granules from these cells with sodium hypochlorite (bleach)(Figure 4). Our protocols can be seen on this <A href="https://2017.igem.org/Team:Calgary/Experiments">page.</A></P><DIV id="Caption"><B>Figure 4: </B>PHB produced by <I>E.coli</I> DH5ɑ cells that had been transformed with PhaCAB (<A href="http://parts.igem.org/Part:BBa_K934001">Part:BBa_K934001</A>) from Tokyo Tech (Tubes 1 and 5) or PhaCAB (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1149052">Part:BBa_K1149052</A>) from Imperial College (Tubes 2 and 4). <I>E.coli</I> DH5ɑ transformed with an empty pSB1C3 backbone was used as a control (Tube 3). The transformed cells were incubated overnight in LB media + 3% glucose, then the plastic was extracted from the cells with sodium hypochlorite and washed with 70% ethanol. The PHB pellets are shown before the ethanol wash on the left and after the ethanol wash on the right.</DIV><H3> Week 16 (August 14 - August 18, 2017) </H3><P>Kaitlin and Sam re-digested pET29B-RFP sequentially with NotI-HF and HindIII-HF, as was done in weeks 13-15. The digests were run on a 1% low melting-point agarose gel, then the backbones were excised from the gel. 10 replicates of the backbone were excised so that it could be used by our group and the Synthesis group. This backbone was ligated to our SP2 gBlock digested with HindIII-HF and NotI-HF. We also began a new approach of ligating SP2 directly into pET29B with SP1 insert (originally obtained in Week 12). To do this, both pET29B-SP1 and SP2 were digested with XhoI and HindIII-HF then ligated together to create pET29B-SP1-SP2. Both ligation products were transformed into chemically competent <I>E.coli</I> DH5ɑ cells.</P><H3> Week 17 (August 21 - August 25, 2017) </H3><P>There were plenty of colonies on both the pET29B-SP2 and pET29B-SP1-SP2 plates that Kaitlin and Sam had prepared last week. However, when overnight cultures and master plates of these colonies were prepared for screening, none of the preparations showed any sign of growth. It was suspected that there was a viral contamination in the LB + kanamycin broth used for overnight preparation. New LB + kanamycin was made, but nonetheless, overnight cultures and master plates of these colonies still showed no growth. From this, it was deduced that the LB + kanamycin broth was not the issue, and some other factor was preventing our transformants from growing. The same HindIII-HF/XhoI digests as in Week 16 were carried out in order to prepare a new set of pET29B-SP1-SP2 colonies that hopefully would not have the same growth issues and could be screened.</P><P>Lalit worked on another SDS-PAGE experiment to qualify phasin production in our BL21(DE3) cells transformed with pSB1C3-Phasin-HlyA Tag. BL21(DE3) transformed with empty pSB1C3 were used as controls. He sub-cultured overnights of the cells, induced them with 1mM IPTG for 4 hours, then lysed them in order to separate them into soluble and insoluble protein fractions, as well as a supernatant fraction. More details can be seen on our <A href="https://2017.igem.org/Team:Calgary/Experiments"> Experiments</A> page. He will run the purified fractions through Sigma-Aldrich anti-FLAG M2 affinity gels (because our phasin-HlyA has FLAG tags) to isolate the phasin before running the SDS-page gels.</P><P> With the Process subgroup, Jacob carried out experiments to determine if PHB could be produced with synthetic feces supernatant that had been fermented at either 23°C  or 37°C  for 3 days. After the fermentation, the supernatant was inoculated with <I>E.coli</I> DH5𝛼 that had been transformed with pSB1C3-PhaCAB from Imperial College. <I>E.coli</I> DH5ɑ transformed with an empty pSB1C3 backbone was used as a control. Then, after overnight incubation at 37°C, PHB was extracted with bleach. Supernatant fermented at 23°C yielded significantly more PHB than supernatant fermented at 37°C. See greater details about the results <A href="https://2017.igem.org/Team:Calgary/VFA_Fermentation#fermentation">here</A>.</P><H3> Week 18 (August 28 - September 1, 2017) </H3><P>Overnights and master plates of the second set of pET29B-SP1-SP2 colonies from last week did not show any signs of growth. Since two sets of pET29B-SP1-SP2 colonies and two sets of pET29B-SP2 colonies were unable to grow in overnights or on master plates it was deduced that our SP2 is somehow harmful to the <I>E.coli</I> and it would be very difficult to transform this part into our bacteria. We decided to move past cloning our SP2 part in and instead focus on carrying out assays with the pSB1C3-Phasin-HlyA Tag (cloned in Week 10)  and pET29B-SP1 (cloned in Week 12).</P><H3> Week 19 (September 5 - September 8, 2017) </H3><P>Jacob repeated the experiment from Week 17 with 3-day fermented synthetic feces supernatant. Again, results were similar, and supernatant fermented at 23°C yielded a much higher amount of PHB than supernatant fermented at 37°C. See the results <A href="https://2017.igem.org/Team:Calgary/VFA_Fermentation#fermentation">here</A>.</P><H3> Week 20 (September 11 - September 15, 2017) </H3><P> In order to carry out assays of PHB secretion, <I>E.coli</I> BL21(DE3) must first be double transformed with a plasmid containing PHB-producing genes, as well a plasmid containing our secretion genes. So, this week Kaitlin worked on double transforming cells with iGEM registry pSB1C3-PhaCAB from Imperial College and pET29B-SP1 (cloned in Week 12). Since synthesis has not yet completed ligating their PhaCBA genes together, we decided to use the part from Imperial College, which we know can successfully produce PHB. Although we carried out two double transformation attempts, neither showed any colony growth, indicating that the double transformations had failed. </P><P> Kaitlin also digested both pSB1A3-RFP and pSB1C3-Phasin-HlyA Tag with EcoRI-HF and SpeI, then ligated the products together in order to produce pSB1A3-Phasin-HlyA Tag. This is necessary because pSB1C3-Phasin-HlyA Tag would not be able to be transformed into the same cells with pSB1C3-PhaCAB, since it is not possible to double transform a cell with two of the same plasmid backbones. Digest confirmation with NotI-Hf showed that the ligation was successful and pSB1A3-Phasin-HlyA Tag that can be transformed into BL21(DE3) alongside pSB1C3-PhaCAB had been obtained.</P><H3> Week 21 (September 18 - September 22, 2017) </H3><P>We had used up most of the registry pSB1C3-PhaCAB from Imperial College and double transformations with what was left last week had failed. We decided to isolate a fresh sample of pSB1C3-PhaCAB from the cells that Jacob had been working with earlier in the summer. This new pSB1C3-PhaCAB and pET29B-SP1 was transformed into chemically competent BL21(DE3). Also, the pSB1C3-PhaCAB and pSB1A3-Phasin-HlyA Tag were transformed into chemically competent BL21(DE3). However, these transformations all failed and we decided to move forward with a new method of double transforming, in which chemically competent BL21(DE3) were transformed with pSB1C3-PhaCAB alone. Then, these cells were made competent and next week we will attempt to transform either pSB1A3-Phasin-HlyA Tag or pET29B-SP1 into the chemically competent BL21(DE3)(+pSB1C3-PhaCAB).</P><H3> Week 22 (September 25 - September 29, 2017) </H3><P> Two attempts of transforming chemically competent BL21(DE3)(+pSB1C3-PhaCAB) with either pSB1A3-phasin or pET29B-SP1 were made this week, however, both attempts failed and the cells were not successfully transformed.</P><H3> Week 23 (October 2 - October 6, 2017) </H3><P>After some research, we realized that pSB1A3 is incompatible with pSB1C3 and that transforming cells with both of these plasmids at the same time would not be possible. This led us to attempt to create a &quot;super&quot; pSB1C3 plasmid containing both our Phasin-HlyA Tag part and PhaCAB from Imperial College. Sam used XbaI and SpeI to cut Phasin-HlyA Tag out of the pSB1A3 backbone and ligate it into pSB1C3-PhaCAB that had been linearized with XbaI. This ligation was unsuccessful and will be re-attempted next week.</P><P>Kaitlin and Lalit prepared the Phasin-HlyA Tag protein fractions collected by Lalit in Week 17 for running on an SDS-Page Gel. First, Sigma-Aldrich anti-FLAG M2 affinity gel resin was prepared, as per manufacturer's instructions. Then each sample fraction was incubated with the anti-FLAG resin in order to help isolate our Phasin-HlyA Tag (which has a FLAG tag) from the large volume of other proteins present in the samples.</P><H3> Week 24 (October 10 - October 13, 2017) </H3><P>Unfortunately, the SDS-Page gel run in Week 23 did not work, even after our samples had been incubated with anti-FLAG resin. All lanes, even the controls appeared empty, therefore it is believed that the resin was not properly prepared and all proteins were eluted out of the samples.</P><P>Kaitlin re-attempted the creating of a pSB1C3 &quot;super&quot; plasmid with both PHB-secreting (Phasin-HlyA Tag) and PHB-producing genes (PhaCAB from Imperial College). She followed the same digestion as Sam in Week 23, where pSB1C3-PhaCAB was linearized with XbaI and Phasin-HlyA Tag was removed from pSB1A3 with XbaI and SpeI. However, she then ran the digested products on a 1% Low-Melting-Point agarose gel, excised the linearized backbone and Phasin-HlyA Tag insert, and ligated the two together (see protocols <A href="https://2017.igem.org/Team:Calgary/Experiments">here</A>).</P><P>A lot of work was done on the wiki, such as updating the secretion journal and writing content for various pages.</P><H3> Week 25 (October 16 - October 20, 2017) </H3><P>The &quot;super&quot; plasmid ligation product from last week was transformed into chemically competent DH5𝛼. Growth was very slow, however after a couple of days colonies were seen on the plate and 8 colonies were mini-prepped and digested with NotI-HF for confirmation.</P><H3> Week 26 (October 23 - October 27, 2017) </H3><P>The &quot;super&quot; plasmid colonies digested with NotI-HF in Week 25 were run on a 1% agarose gel at 100V for 30 minutes. Several colonies produced the expected band sizes (4.8kb and 2.0kb). Figure 5 below shows 4 of the digested colonies. Colony 6 had the crispest bands and lowest amount of faint background smearing seen at other band sizes, therefore it was chosen to be transformed into BL21(DE3) for protein expression.</P><DIV id="Caption"><B>Figure 5: </B>Screening results of 4 colonies of DH5α transformed with pSB1C3-PhaCAB-Phasin-HlyA Tag. Plasmids from these colonies were digested with NotI-HF and XbaI (D) then run on a 1% agarose gel at 100V for 30 minutes. The molecular ladder (L) is visible on the far left and the expected band sizes, obtained from Benchling Virtual Digest, are visible on the right. Undigested plasmid (U) was used as a control and the digests from colony 6 (colony used for cloning into BL21(DE3) for secretion assays) are visible in lanes 6-7.</DIV><P>Transformation of pSB1C3-PhaCAB-Phasin-HlyA Tag was successful and many colonies were obtained that can be used to carry out a test of PHB secretion.</P><H3> Week 27 (October 30 - November 3, 2017) </H3><P>Kaitlin performed a secretion assay. The asssay was carried out in triplicates, with a negative control of pSB1C3-PhaCAB. After induction with IPTG, samples were separated into intracellular PHB fractions and secreted fraction samples by differential centrifugation after either 24 hours or 48 hours of incubation. The results of this assay can be seen <A href="https://2017.igem.org/Team:Calgary/Results">here</A> and more experimental details about the assay can be found on our <A href="https://2017.igem.org/Team:Calgary/Secretion">Secretion</A> page.</P><DIV id="Caption"><B>Table 3: </B>Experimental results of secretion assay carried out from October 28-Nov 1, 2017. (-) is used to denote negative control, pSB1C3-PhaCAB and (+) is used to denote PHB-secreting strains, pSB1C3-PhaCAB-Phasin-HlyA Tag. 50 mL of LB + chloramphenicol + 3 % glucose was inoculated with (-) and (+) and induced with IPTG, then separated into secreted and intracellular fractions for quantification of PHB secretion.</DIV><P>Since SDS-Page to determine Phasin-HlyA Tag production had failed after incubation with an anti-FLAG resin (in Week 24), Lalit performed an immunoblot (Western Blot) on the protein samples with anti-FLAG Rabbit primary antibodies and goat anti-rabiit igG-HRP secondary antibodies. The HRP (Horse-Radish Peroxidase) changes color to blue if it is bound to the anti-FLAG rabbit antibodies, which thereby indicates that our Phasin-HlyA Tag with a FLAG tag is present. The Western Blot protocol can also be found on our <A href="https://2017.igem.org/Team:Calgary/Experiments">Experiments</A> page.</P><DIV id="Caption"><B>Figure 6:</B> Photograph of one of our SDS-PAGE gel electrophoresis apparatuses running with a protein standard, proteins from <I>E.coli</I> BL21(DE3) transformed with an empty pSB1C3 vector, and proteins from <I>E. coli</I> BL21(DE3) transformed with pSB1C3-phasin-HlyA Tag.</DIV></DIV></DIV><BUTTON class="accordion"><H3> Process</H3></BUTTON><DIV class="panel"><H3>Week 1: May 1 - May 5</H3><P>During the first week, we checked the inventory and organized our lab supplies. We familiarized ourselves with the lab equipment and some of the protocols we might be using during the summer such as preparing LB agar plates, chemically competent <I>E. coli</I> cells, plasmid minipreps, and running colony PCRs. We also completed the required lab safety and biosafety training. We started researching Alberta’s wastewater treatment process and conducted a preliminary literature review of volatile fatty acid (VFA) production and polyhydroxybutyrate (PHB) extraction. We met with Dr. Peter Dunfield, a professor of microbiology at the University of Calgary, to discuss a novel approach of extracting PHB from bacteria to avoid using solvents such as chloroform for PHB extraction in our final process. Consequently, we brainstormed ideas about binding PHB granules to beads after autolysis of engineered <I>E.coli</I>.</P><H3>Week 2: May 8 - May 12</H3><P>We continued practicing common lab procedures and reviewed similar projects from the past to explore avenues for improvement and innovation. As a result, we decided on a secretion based system as opposed to a lysis system. Concerns were also raised this week about the use of pure cultures with sludge in the wastewater treatment process. Genetically engineered bacteria can be out-competed if added directly to sludge while sterilizing sludge before introducing engineered bacteria would result in additional costs. Since PHB production occurs in nutrient-limited conditions, we also began to research ways to remove nutrients before inoculating sludge with the engineered bacteria. Our team started a draft outline of a proposed process from VFA production to PHB extraction and purification.</P><H3>Week 3: May 15 - May 19</H3><P>During this week, we discussed our sub-group’s role in the project and divided the work into subcategories: extraction and purification of PHB, quantification and characterization of PHB, process development/scale up. Objectives and milestones were then developed for each subcategory. Since extraction and purification of PHB step is one of the major contributors to high costs of PHB production (Jiang, Mikova, Kleerebezem, van der Wielen &amp; Cuellar, 2015), we will research some novel, feasible extraction and purification methods for the large-scale process and will test proposed methods in the lab. The process development group will also support the synthesis group by quantifying and characterizing the PHB that they produce in the lab. This week, we looked into potential protocols and the availability of resources for lab-scale quantification and characterization of PHB. When designing a PHB production process, we will also consider scalability and feasibility at large by conducting a cost analysis for the proposed large-scale PHB production process. </P><P>We also met with Dr. Christine Sharp, a postdoctoral fellow from the Energy Bioengineering Group supervised by Dr. Marc Strous. Their group is working on PHB production using alkaline soda lake biomass and designing their own bioreactors. </P><H3>Week 4: May 22 - May 26</H3><P>At the start of the week, the group met with Dr. Nashaat Nassar, an Assistant Professor of Chemical and Petroleum Engineering, to discuss PHB extraction and purification techniques. He recommended a combination of coagulation and settling to separate out PHB. We also met with Dr. Saurabh Sarma, a postdoctoral fellow in Civil Engineering, to discuss VFA production in wastewater treatment plants (WWTP). He advised us on integrating our process with the desired applications. We also discussed the composition of VFA commonly found in WWTP and parameters that impact VFA composition. Dr. Sarma suggested reaching out to Daniel Larson, a laboratory technician in Civil Engineering, to discuss VFA and PHB quantification using gas chromatography. </P><P>The entire team visited one of Calgary’s WWTP this week, where we learned about the current wastewater treatment process. After the tour, we met with representatives from ACWA (Advancing Canadian Water Assets) Research Facility at the Pine Creek Wastewater Treatment Plant, who are working on wastewater treatment research. However, they work with the output from the wastewater treatment plant, while most of our questions were about the wastewater treatment process itself. They directed us to the City of Calgary chemistry laboratory that performs analysis on intermediate samples from various stages of the process. We have contacted representatives from the City of Calgary. </P><H3>Week 5: May 29 - June 2</H3><P>This week, the entire team focused on evaluating the feasibility of the 4 proposed project applications: production of PHB on Mars from human waste, integrating PHB production in a wastewater treatment plant, integrating PHB production with leachate treatment, and integrating PHB production in developing countries. Our subgroup calculated the approximate amounts of PHB that would be expected in each scenario. </P><P>We approximated that about 40 - 90 kg of PHB can be produced on Mars per year with a crew of 6 astronauts. A crew of 6 will generate about 6 tonnes of solid organic waste over 2.5 years (Zhang, Ylikorpi &amp; Pepe, 2015). Reported COD content in feces was found to be 354 mg COD per gram of wet human waste (Rose, Parker, Jefferson &amp; Cartmell, 2015). One study looking at PHA production from food waste estimated the yield of 0.05 g of PHA per g of COD applied (Rhu, Lee, Kim &amp; Choi, 2003). Another study reported 0.11 kg of PHA produced per kg of effluent COD in a PHA production process from activated sludge (Bengtsson, Werker, Christensson &amp; Welander, 2008). The predicted PHB range was based on COD to PHA conversion. Another member of the team assumed the average COD content in human excretions to be 61.75 g/cap/day (Rose, Parker, Jefferson &amp; Cartmell, 2015), the COD to VFA ratio of 0.74  (Coats, VandeVoort, Darby &amp; Loge, 2011)and the VFA to PHB conversion of 0.38 g PHA/g VFA ,(Coats, VandeVoort, Darby &amp; Loge, 2011) which resulted in 41 kg of PHA per year per crew of 6. According to NASA, the cost of shipping supplies to space using SpaceX Dragon spacecraft is $27,000 per pound. The costs saved by producing 41 kg of PHA in space would then be about $2,440,000.</P><P>We also contacted a 3D printing company called 4G Vision Tech that uses selective laser sintering (SLS), which can be used to 3D print with PHB (Pereira et al., 2012). Howard from 4G Vision Tech approximated that the predicted amount of PHB can be used to create approximately 50 hydroponic systems and 20 general tools like wrenches, hammers, and scissors. </P><P>The integration of PHB production in leachate treatment would likely be unfeasible due to low volumes of leachate that are usually produced at landfills. In Calgary, a single landfill generates about 100,000 L of leachate per day. Although COD content in leachate is higher than in wastewater, the estimated amount of PHB produced in Calgary was about 8000 kg/year, based on COD content of 1977 mg/L in Calgary leachate (Kashef &amp; Lungue, 2016). This would result in about $40,000 in revenue, assuming a price of $5 per kg of PHB (Manufacturing and properties of PHB, 2017). Another member of the team performed similar calculations assuming 0.38 grams of PHB produced per gram of VFA, which resulted in 22,250 kg of PHB per year and a potential revenue of $111,126 per year. The cost of implementing a PHB production process will likely be magnitudes larger. Leachate treatment in China is a more promising alternative. China generates a larger amount of leachate compared to many other countries(&quot;Leachate treatment in China: Technologies and Import Opportunities&quot;, 2015). Additionally, the COD content in Hong Kong, China ranges from 15,700 to 50,000 mg/L for young landfills (&quot;Leachate treatment in China: Technologies and Import Opportunities&quot;, 2015), which is 8 to 25 times greater than in Calgary. We estimated that about 900,000 - 3,000,000 kg of PHB can be produced per year in Hong Kong depending on COD content and using PHA yield of 0.11 kg of PHA per kg of COD. PHB production from leachate was also considered for Vancouver, which generates 2,225,978 cubic meters of leachate per year (Vancouver landfill 2016 annual report, 2017). Based on our estimates, about 3,100,000 kg of PHB can be produced per year assuming the COD content of about 13,000 mg/L (Tao, Hall &amp; Masbough, 2005) </P><P>For the wastewater treatment plant, we estimated 28,100,000 kg of PHB produced per year. The Pine Creek Wastewater Treatment Plant in Calgary processes about 1 million cubic meters of waste per day. The COD content and PHA yield were assumed to be 1000 mg/L and 0.11 kg of PHA per kg of COD, respectively, for the calculations.</P><P>In developing countries, we envisioned PHB production incorporated into scaled-down wastewater treatment systems in small communities that lacked established treatment methods. Selling PHB would provide a monetary incentive to construct a wastewater treatment system, which, in turn, will reduce diseases due to poor sanitation. Additionally, we wanted to compare PHB production between genetically engineered bacteria and natural bacterial communities in sludge, which have been previously used to feasibly produce PHB. Assuming a community size of 2000 people, solid waste generation of 3.113 x 10-3 m3 /day/person (Palanivel &amp; Sulaiman, 2014), COD content of 601 mg/L [14], COD to PHB conversion of 0.11 for mixed cultures and 0.88 for pure cultures  (Rhu, Lee, Kim &amp; Choi, 2003) and a price of $5 per kg of PHB (Choi &amp; Lee, 1997), we found that using pure cultures results in additional $2,000 in revenues. However, the cost of sterilization of waste stream before inoculation with pure culture was estimated at $100,000 (Choi &amp; Lee, 1997). </P><H3>Week 6: June 5 - June 9</H3><P>This week, our team decided to pursue PHB production on Mars from human waste as the main application for our project. Our subgroup revisited the scope of work and different stages of the required process and prioritised tasks. We first researched potential extraction and purification techniques and chose the top 3 methods for further testing in the lab. At the same time, we also looked at VFA production from feces. For safety reasons, we decided to use synthetic feces in preference to real samples. We found synthetic feces recipes (Wignarajah, Litwiller, Fisher &amp; Hogan, 2006),  (Colon, Forbis-Stokes &amp; Deshusses, 2015) and purchased required materials. After preparing synthetic feces, we researched desired conditions for VFA production, separation of VFA from feces and VFA quantification methods.  </P><H3>Week 7: June 12 - June 16</H3><P>This week, we met with professors in the Chemical and Petroleum Engineering department at the Schulich School of Engineering to discuss PHB extraction and purification techniques. One professor suggested growing bacteria in a biofilm while continuously collecting PHB. With this setup, dead bacteria can be filtered out by a 0.2 µm filter and harvest containing PHB can be separated from biomass without centrifugation. However, biofilms require a large surface area, which is not ideal for the space application where compact systems are desired. Another professor suggested that our process involve the continuous production of PHB in a membrane reactor that retains the bacteria, but lets the PHB through. This continuous PHB production system will also be compact compared to biofilms. We also discussed coagulation including electrocoagulation as a potential method for extraction and purification of PHB with Dr. Nashaat Nassar.  </P><P>For lab-scale experiments, we plan to test centrifugation and coagulation methods for PHB extraction. Centrifugation method would involve two centrifugation steps: the first step to separate supernatant containing PHB from biomass and other large particles and the second step to settle the PHB granules. Since bacteria and PHB have different settling velocities, the first centrifugation step can remove biomass while leaving PHB in suspension. The second step will likely require an ultracentrifuge but will allow to settle PHB granules and remove the supernatant. Adding a coagulant such as calcium ions can aid in settling PHB granules; we will experiment with coagulation to see whether it can eliminate the need for the second centrifugation step or reduce required centrifugation speeds. </P><P>Additionally, we discussed a fermentation experiment to produce VFA from synthetic feces. The experiment will aim to answer the following questions:</P><UL><LI>Is it possible to make VFA by inoculating synthetic feces with <I>E. coli</I>?</LI><LI>How much VFA can be made per amount of input feces and what types of VFA will be made?</LI><LI>How long does it take to reach maximum VFA composition?</LI><LI>Does temperature have an impact on VFA production?</LI><LI>Does the addition of yeast to synthetic feces make a difference in VFA production? </LI></UL><P>
The proposed conditions for the first VFA experiment are summarized in Table 1. We plan to run the fermentation process for 5 days. Samples for VFA analysis will be collected on day 1 and day 5. </P><DIV id="Caption"><B>Table 1: </B> Proposed conditions for the first VFA fermentation experiment. </DIV><TABLE class="tg"><TBODY><TR><TH class="tg-yw4l">Condition</TH><TH class="tg-yw4l">Temperature</TH><TH class="tg-yw4l">Yeast present in synthetic feces</TH></TR><TR><TD class="tg-yw4l">1</TD><TD class="tg-yw4l">Room Temperature </TD><TD class="tg-yw4l">yes</TD></TR><TR><TD class="tg-yw4l">2</TD><TD class="tg-yw4l">Room Temperature</TD><TD class="tg-yw4l">no</TD></TR><TR><TD class="tg-yw4l">3</TD><TD class="tg-yw4l">37 degrees Celsius</TD><TD class="tg-yw4l">yes</TD></TR><TR><TD class="tg-yw4l">4</TD><TD class="tg-yw4l">37 degrees Celsius</TD><TD class="tg-yw4l">no</TD></TR></TBODY></TABLE><P>We plan to detect VFA using gas chromatography. The first step is to test the gas chromatography method and determine the amount of VFA present in synthetic feces. To test this, we will take 4 samples of synthetic feces and spike 3 of the sample with known concentrations of VFA. </P><H3>Week 8: June 19-23</H3><P>This week, we researched microbial composition of human feces and the role of different microorganisms in the human gut and feces. It was found that gram-positive bacteria averaged 10 ^(10.5±0.4(sd)) organisms per gram of wet feces with significant variation from host to host (Dae Lee et al., 2010). The 5 major strains found in human feces are <I>Bifidobacterium adolescentis</I>, <I>Eubacterium aerofaciens</I>, <I>Eubacterium rectale</I>, <I>Peptostreptococcus productus</I>, and <I>Ruminococcus bromii</I> (Dae Lee et al., 2010). Short-chain fatty acids such as propionate, butyrate, and acetate are produced by <I>Bifidobacteriu</I>.</P><P>We also researched fermentation conditions that result in higher VFA production. The literature reported highest VFA production when the pH is controlled at 6.8 with NaOH additions (Lifschits, Wolin &amp; Reeds, 1990). Additionally, we discussed the proposed VFA production experiment with Dan, one of our advisors. He suggested reducing the number of replicates and the number of samples we plan to take for the first experiment as our experiment might not work the first time. We also found an alternative method to measure VFA using High-Performance Liquid Chromatography (HPLC) instead of gas chromatography. We contacted Christine Sharp, who is a postdoctoral fellow in Dr. Marc Strous’ research lab, as their lab uses HPLC to measure VFA, and they were willing to let us use their HPLC to measure VFA.</P><P>Together with our team members from other subgroups, we visited Bonnybrook wastewater treatment plan to discuss wastewater analysis conducted by City of Calgary. We were particularly interested in VFA and COD monitoring in the wastewater treatment plant. Marko Markicevic, the City of Calgary contact we met with, answered our questions about COD and VFA monitoring and suggested we submit a request form obtain COD and VFA data from the City of Calgary. Marko also mentioned a titration method that might be easier for us to use to measure VFA instead of gas chromatography or HPLC. A titration method; however, will only show the total concentration of VFA (unlike the HPLC that can show concentrations of different types of VFA). </P><P>Lastly, we contacted nanoparticle manufacturers including US Research Nanomaterials Inc., Nanostructured &amp; Amorphous Materials Inc., Sigma-Aldrich, and NN-Labs to inquire about PHB particles in 20 - 60 nm range. We wanted to obtain PHB nanoparticles to mimic the PHB particles we expect our engineered bacteria will make, in order to start testing our extraction and purification methods.  Only NN-Labs might be able to produce PHB nanoparticles in this size range. </P><H3>Week 9 : June 26 - June 30</H3><P>This week, we made synthetic feces in the lab following a recipe found in the literature (Colon, A. Fobris-Stokes &amp; A. Deshusses, 2015). However, we substituted oleic acid with peanut oil, which contains about 50% oleic acid and is much cheaper. Peanut oil was also used in a different synthetic feces recipe developed by NASA [23]. We prepared synthetic poop supernatant samples with 0.05 mM, 0.5 mM and 2 mM concentrations of 1:1:1 solution of acetic acid, butyric acid and propionic acid assuming no VFA were present in the samples initially. Although the HPLC working range for VFA quantification is 0.1 - 1 mM, we wanted to test concentrations above and below the limit as we are unsure of VFA concentrations we should expect from our fermentation experiments. Lastly, we also prepared a supernatant sample without added VFA. We then ran these samples on the HPLC.</P><P>While running HPLC samples in Dr. Strous’ lab, we also spoke with Karen, who is working on producing PHB from algae. Karen can provide us with samples of their PHB. They produce approximately 2 mg of PHB from 25 mg of dry weight, obtained from 15 mL of harvest and have approximately 700 mL of harvest per day.</P><P>NN-Labs, that we contacted last week, can make PHB granules in 20 - 60 nm range, but we may face issues with getting the PHB granules out of the aqueous suspension. We contacted Dr. Nashaat Nassar, Dr. Maen Husein, &amp; Dr. Giovanniantonio Natale about re-suspension and dispersion of nanoparticles. Dr. Natale recommended literature search on chemical functionalization (Sperling &amp; Parak, 2010).</P><H3>Week 10: July 3-7 </H3><P>The results of the first HPLC run to quantify VFA were inconclusive due to background noise (which made peak integration challenging) and issues with the standard curve (which most likely occurred due to a dilution mistake during preparation). We identified peanut oil as a potential source of the background noise; for the second run, we will not add oil when preparing synthetic feces and will dilute samples with milliQ water. </P><P>This week, we also broke down the process we would need to develop for Mars into multiple stages: collection and fermentation of solid human waste to obtain VFA, separation of VFA from fermented human waste, fermentation of genetically engineered bacteria with obtained VFA to produce PHB, and PHB extraction and purification. After discussing our project with Matthew Bamsey, who is a Chief Systems Engineer at the German Aerospace Center, we decided to use Equivalent System Mass (ESM) to evaluate all the stages of our process. In ESM analysis, pressurized volume, power generation, cooling power and crew time required for a proposed system are converted to an equivalent mass number and added to the system mass. Since the cost of transporting payload is proportional to its mass, ESM is used in Advanced Life Support (ALS) studies as a cost of transportation and as a way to compare different systems (Levri et al., 2003). To help with ESM analysis, Matt Bamsey suggested we also read the Life Support Baseline Values and Assumptions Document, which was created by NASA and provides common assumptions that can be made when developing life support systems. </P><P>We read NASA’s document with Equivalent System Mass Guidelines to familiarize ourselves with ESM analysis and factors we need to consider. We also read the Baseline Values and Assumptions Document to understanding what resources would be available on Mars. Among other useful assumptions, we learned about the power sources such as solar and nuclear power that would be available on Mars and how much power each source could provide. </P><P>This week, we also run the next set of HPLC samples to quantify VFA. Firstly, we made changes to synthetic poop recipe: live yeast was replaced with yeast extract and peanut oil was removed from the recipe. Secondly, we added a mixture of acetic acid, propionic acid, and butyric acid to synthetic feces supernatant to the final concentrations that were within the HPLC working range: 0.25 mM, 0.5 mM, and 0.75 mM assuming the initial sample had no VFA. We also run an undiluted sample of synthetic feces supernatant without additional VFA added. All samples were run in triplicates, as before.</P><H3>Week 11: July 10-14</H3><P>Although the second HPLC run still had background noise, we were able to manually integrate the peaks for acetic acid, propionic acid, and butyric acid. </P><P>Results from the second run of our VFA quantification experiments showed that there were approximately 109 mmol/L acetic acid, 197.51 mmol/L propionic acid, 12.65 mmol/L butyric acid in our supernatant, which was much larger than we had expected. This meant we needed to dilute our samples before passing them through the HPLC. </P><P>We have also spent a considerable amount of time brainstorming the process flow: starting from the feces production and collection and finishing with the PHB extraction and purification. The goal of these discussions was to determine the main stages in the process and ensure all members of process development sub-group envision the process the same way. We have also discussed the milestones related to each stage of the process and created a timeline. </P><P>VFA extraction from synthetic feces protocols were created, a new synthetic feces recipe was chosen and all the required laboratory supplies were purchased to test those protocols next week. We have also found a sponsor for the PHB samples - PolyFerm Canada has kindly agreed to send us a couple of hundred grams of PHB powder to test the PHB extraction procedures and to use as a standard. </P><H3>Week 12: July 17-21</H3><P>This week, the engineering team worked on three major project components: VFA separation from synthetic feces laboratory experiments, VFA quantification titration experiments and the HPLC experiments.  </P><P>Titration is the industry employed method for continuous testing for the overall VFA concentration in the solution. It is based on the acid-base chemistry: the sample is titrated to the specified endpoints with sulfuric acid and the VFA concentration is found using a formula based on the amount of acid consumed to reach the endpoint. This method worked when the standards of 1mM acetic acid and 1mM of acetic, propionic and butyric (1:1:1) acids were tested. The titration method was then tested on synthetic poop supernatant and the total VFA concentration of about 80mM was found, which is in line with the range of VFA present in the human fees. </P><P>Four experiments were conducted to test different VFA separation methods: simple filtration, settlement, centrifugation, and pressure-filtration. It was found that pressure filtration is the most efficient way of recovering liquid from synthetic feces. However, it required quite a lot of pressure and didn’t seem to be the most feasible solution for the solid human waste on Mars. So another experiment was conducted where a staged filtration approach was used - the feces passed through a series of filters which gradually decreased in size. The recovery past 0.2-micron filter was very small - 10% of the water present in synthetic feces. However, this was also a result of losses of liquids due to transfer between containers.
</P><P>For this week's HPLC experiment, samples were diluted by three different dilution factors. The HPLC results were more reliable. The optimal dilution rates were determined from this experiment and will be used in the future.</P><H3>Week 13: July 24-28</H3><P>This week a couple of group members worked on developing a large-scale method for VFA extraction based on conducted experiments. The proposed technologies included screw dewatering system, centrifuge, worm centrifuge and self-cleaning filters.  We have also looked into the wastewater processing technologies employed on the International Space Station and the technologies proposed for Mars missions by NASA. This research generated new ideas such as multi-filtration, distillation, and torrefaction. Torrefaction can be defined as a thermochemical treatment of biomass in the absence of oxygen. The two methodologies that the team chose to explore further were the screw press followed by multi-filtration and the torrefaction processing unit. </P><P> This week, we also focused on feces collection on Mars and fermentation of feces to increase the concentration of VFA, which would then be consumed by engineered bacteria to produce PHB. In our proposed system, astronaut’s feces would be collected using a vacuum toilet into a 10L storage tank. Although a vacuum toilet requires a small amount of water, it can be recovered at the end of the process. The volume of the storage tank was selected based on NASA requirements of accommodating production of up to 150 grams of feces per event for 2 events per crew member per day and also accommodating for potential diarrhea events that can result in up to 1.5L of liquid. In the next stage of the process, feces would be fermented with naturally occurring bacteria to increase the concentration of VFA. Although higher VFA production is expected to occur at 37C, experiments will be conducted to determine whether VFA production can also occur at room temperature. </P><H3>Week 14: July 31 - August 4</H3><P>To compare the two proposed methodologies for VFA and liquids extraction for large scale, the ESM parameters were found/estimated for different systems. Since the torrefaction process is a novel solution to solid waste management the ESM parameters for the equipment are not yet developed, so it was chosen to make estimations based on the vapour compression distillation (VCD) system (because torrefaction and distillation are governed by similar principles). </P><P>Table 2 summarises the ESM parameters. The data for multi-filtration and VCD system was found in the paper by (W. Jones, W. Fisher, D. Delzeit, T. Fynn &amp; H. Kliss, 2016). The parameters for the screw dewatering systems were found while doing the market search. </P><DIV id="Caption"><B>Table 2: </B> ESM results for the proposed VFA separation technologies.</DIV><TABLE><TBODY><TR><TH>Screw dewatering system</TH><TH>Multi-filtration</TH><TH>VCD values</TH></TR><TR><TD>Cooling requirement</TD><TD>0.298</TD><TD>0.92</TD><TD>0.44</TD></TR><TR><TD>Power (kW)</TD><TD>0.298</TD><TD>0.92</TD><TD>0.44</TD></TR><TR><TD>Weight (kg)</TD><TD>179</TD><TD>232</TD><TD>378</TD></TR><TR><TD>Volume (m^3)</TD><TD>2</TD><TD>1.83</TD><TD>3.21</TD></TR><TR><TD>Spares and consumables (kg/day)</TD><TD>0.00328</TD><TD>0.3667</TD><TD>1.9188</TD></TR><TR><TD>Spares and consumables (m^3/day)</TD><TD>0.00282</TD><TD>0.00478</TD><TD>0.0121</TD></TR><TR><TD>ESM Estimation (kg)</TD><TD>1050</TD><TD>1683</TD><TD>3899</TD></TR><TR><TD><B>Total (kg)</B></TD><TD colspan="2">2733</TD><TD>3899</TD></TR></TBODY></TABLE><H3>Week 15: August 7- August 15</H3><P>After researching the torrefaction of solid human waste process, we realised that the VCD estimation for the process is not reliable due to different resource requirements, feedstock size, and temperature. A diagram (Figure 1)  of the full-scale apparatus was created with all the materials, dimensions, and parameters. Equipment set-up, materials, and energy inputs were estimated based on experimental setup found in the literature (A. Serio, E. Cosgrove &amp; A. Wojtowicz, 2016). The suggested equipment set-up assumes batch processing once in 3 days, mild pyrolysis 45 minutes in duration, the maximum temperature of the reaction vessel of 280 degrees Celsius, and 2.5kW power supply. The above-stated conditions should assure complete recovery of moisture (A. Serio, E. Cosgrove &amp; A. Wojtowicz, 2016), some additional water production (pyrolytic water), a modest reduction of the dry solid mass, and the evaporation of all the required VFA. The ESM estimates for the torrefaction processing unit are included in Table 3. </P><DIV id="Caption"><B>Table 3: </B>  Data used in ESM analysis of Torrefaction Processing Unit (TPU).</DIV><TABLE><TBODY><TR><TH>Torrefaction Processing Unit</TH></TR><TR><TD>Cooling requirement (kW)</TD><TD>2.5</TD></TR><TR><TD>Power (kW)</TD><TD>2.5</TD></TR><TR><TD>Weight (kg)</TD><TD>68.9</TD></TR><TR><TD>Volume (m^3)</TD><TD>0.0433</TD></TR><TR><TD>Spares and consumables (kg/day)</TD><TD>0</TD></TR><TR><TD>Spares and consumables (m^3/day)</TD><TD>0</TD></TR><TR><TD>ESM Estimation (kg)</TD><TD>661</TD></TR></TBODY></TABLE><P>Final comparisons for the two proposed systems were made (Table 4) and the team chose to proceed with torrefaction processing unit.</P><DIV id="Caption"><B>Table 4: </B> Summary of the proposed VFA separation methods on Mars.</DIV><TABLE><TBODY><TR><TH>Screw Press/Multi-filtration</TH><TH>Torrefaction</TH></TR><TR><TD>ESM (kg)</TD><TD>2733</TD><TD>661</TD></TR><TR><TD>Required consumables</TD><TD>Polymer</TD><TD>Argon/Nitrogen gas</TD></TR><TR><TD>Efficiency</TD><TD>99.9%</TD><TD>99%</TD></TR><TR><TD>Maintenance requirements</TD><TD>Low</TD><TD>Medium-high</TD></TR><TR><TD>Power source</TD><TD>Electrical</TD><TD>Thermal/electrical</TD></TR><TR><TD>Waste product usability</TD><TD>Unknown; long-term storage</TD><TD>Radiation shielding, building material, food production</TD></TR><TR><TD>Water recovery</TD><TD>95% (assumed)</TD><TD>120% since pyrolytic water is recovered</TD></TR><TR><TD>Liquid stream sterility</TD><TD>Sterile</TD><TD>Sterile</TD></TR></TBODY></TABLE><DIV id="Caption"><B>Figure 1:</B> Diagram of Torrefaction Processing Unit (TPU)</DIV><P>This week we also planned our experiments for PHB extraction. We plan to make a 5 g/L PHB-co-HV suspension in water to be sonicated. The granule size of PHB-co-HV sample we received from PolyFerm was too large for our purposes because we expect our secreted PHB to be in the 20-60 nm range (Rahman, Linton, Hatch, Sims &amp; Miller, 2013). To make a representative mixture from which we would extract the PHB, we plan to dilute the sonicated PHB-co-HV suspension with synthetic feces supernatant to make a 3.5 g/L mixture, which is within the range of concentrations we might expect to be secreted (Rahman, Linton, Hatch, Sims &amp; Miller, 2013)). We want to test 8 different conditions:
</P><UL><LI>PHB self-agglomeration (24 hrs, 48 hrs)
</LI><LI>Centrifugation (200 RPM, 500 RPM, 1000 RPM)
</LI><LI>PHB + Coagulant (CaCl2) + settling
</LI><LI>PHB + Coagulant (CaCl2) + centrifugation (500 RPM) 
</LI></UL><P>We plan to use 10 mM (final concentration) of CaCl2 as our coagulant because literature reported 95% of PHB granules were sedimentable by low-speed centrifugation after addition of 10 mM (Resch et al., 1998).</P><P> This week, we also tested an HPLC method for PHB quantification. The same HPLC column we use for VFA quantification can also be used for PHB quantification. In the proposed method, PHB is converted to crotonic acid by dissolving in sulfuric acid for 30 min at 95C. For the first HPLC run, we prepared crotonic acid standards of various concentrations to see whether the HPLC method will be able to detect crotonic acid. The run was extended to 60 minutes as we were unsure of the retention time of crotonic acid. The method was successful and crotonic acid picked around 20 minutes. </P><P> Together with the secretion team, we started working on synthetic feces fermentation experiments to determine the optimal temperature for VFA fermentation step and to see whether engineered bacteria can produce PHB from synthetic feces. The two temperatures used in the experiment were 37°C and 22°C, which correspond to optimal bacterial growth temperature and room temperature in a Mars habitat, respectively. For the experiment, we used Imperial and Tokyo <I>phaCAB</I> constructs and <I>E. coli</I> without PHB-producing genes as a control. </P><H3>Week 16: August 14-18</H3><P>This week we worked on creating representative PHB samples for the extraction experiments. Our bacteria would be producing PHB in the 20-60nm range, and the PHB that we received from PolyFerm is in the 0.1-1mm range. PHB was suspended in tap water to achieve 5 g/L concentration. PHB-co-HV was also suspended in tap water to achieve 5 g/L concentration. The samples were then sonicated using QSonica homogeniser. The sonication was performed on both samples at Amplitude -37 at 25-second intervals for 10 minutes. The particle size distribution was later measured on the NanoPlus HD device(Figure 2). It is clear that the finer PHB-co-HB powder was broken down better than the larger PHB particles, yet the required size range was still not achieved. More research would be done to improve the experimental procedure to achieve representative sizes. </P><DIV id="Caption"><B>Figure 2:</B>Measured PHB-co-HB particle size distribution after sonication. </DIV><H3>Week 17: August 21-25</H3><P> This week, we ran PHB extraction experiments using sonicated PHB-co-HV from last week. After sonication, the PHB-co-HV granules were in 1-1.5 um range. PHB granules were allowed to settle for 48 hours before running extraction experiments. </P><P>Since we are still validating the HPLC method for PHB quantification, which is also a time-consuming method, we quantified the effectiveness of tested PHB extraction methods by measuring the absorbance of the supernatant at 600 nm wavelengths. The PHB suspension at the start of the experiments was visibly cloudy, hence we used a wavelength within the visible region of the electromagnetic spectrum. We then used absorbance readings to quantify the amount of PHB-co-HV that settled down as a result of PHB extraction method.</P><P> The following conditions were tested during PHB extraction experiments:</P><UL><LI>Centrifugation at 1000 rpm</LI><LI>Centrifugation at 3750 rpm</LI><LI>Centrifugation at 3750 rpm with addition of calcium chloride (CaCl2)</LI></UL><P> Centrifugation was performed for 10 minutes. Each condition had 3 replicates and we calculated the average absorbance. The results of the experiment are shown in Figure 3. Based on these results, centrifugation with addition of CaCl2 was the most effective method for settling PHB-co-HV at lab scale.</P><P> For the process on Mars, chemical coagulation and centrifugation, however, is not an ideal PHB extraction method because it would either require calcium chloride to be shipped to Mars or a separate process to extract calcium chloride from the Martian soil. Therefore, we plan to test electrocoagulation next week as an alternative PHB coagulation method. PHB-co-HV particles have a zeta potential of pH 3.5 resulting in a negative charge(van Hee, Elumbaring, van der Lans &amp; Van der Wielen, 2006). PHB in suspension in water or supernatant (pH 5.3) can be coagulated by supplying positively charged ions. Literature has shown that dications (Ca 2+, and Mg 2+ ions) are effective at agglomerating PHB (Resch et al., 1998). We plan to use an iron cathode (which would release Fe 2+ ions) and a steel anode for our first electrocoagulation experiment. As a first step, we will test this electrocoagulation method using a suspension of PHB-co-HV in water. As a next step, we will use synthetic feces supernatant with no PHB as a negative control and a suspension of PHB-co-HV in synthetic feces supernatant. </P><P> PHB extraction experiments so far were carried out with microscale PHB-co-HV particles, while secreted PHB was expected to be in nanoscale. Next week, we will continue experimenting with different sonication techniques such as water bath sonicator and two-step sonications. </P><P>This week, synthesis sub-group conducted PHB production experiments in different nutrient conditions. During these experiments, no bacterial growth and plastic production was observed when PHB-producing bacteria was inoculated and fermented in water with VFA, which would be the output from torrefaction in a Mars process. This was likely due to low pH of the resulting solution and lack of required nutrients. As a result, we concluded that the torrefaction process cannot be used to extract VFA on Mars. However, it can be used to treat the by-product to recover additional water. We will revisit VFA extraction methods to find a more suitable solution.</P><TABLE style="width: 537px; margin-left: auto; margin-right: auto; height: 120px;"><TBODY><TR><TD style="width: 356px;"><P><EM><SPAN style="font-weight: 400;">Media for the PHB producing bacteria fermentation: </SPAN></EM></P></TD><TD style="width: 181px;"><P><EM><SPAN style="font-weight: 400;">PHB production results</SPAN></EM></P></TD></TR><TR><TD style="width: 356px;"><P><SPAN style="font-weight: 400;">Synthetic Poop supernatant </SPAN></P></TD><TD style="width: 181px;"><P><SPAN style="font-weight: 400;">PHB produced</SPAN></P></TD></TR><TR><TD style="width: 356px;"><P><SPAN style="font-weight: 400;">Minimal media supplemented with VFAs (representative of the torrefaction process liquid output stream) </SPAN></P></TD><TD style="width: 181px;"><P><SPAN style="font-weight: 400;">No PHB produced </SPAN></P></TD></TR></TBODY></TABLE><I>PHB fermentation update</I><P>We also looked into bioreactor designs for the PHB fermentation step. The following types of bioreactors were considered:</P><UL><LI>Stirred tank bioreactor (CSTR) with an external cell separator</LI><LI>External membrance bioreacror (EMBR)</LI><LI>Immersed membrane bioreactor (IMBR)
    </LI><LI>Flat sheet bioreactor</LI><LI>Hollow fiber bioreactor</LI></UL><H3>Week 18: August 28-September 1</H3><I>Electrocoagulation and Sonication experiments updates: </I><P>This week we tested a number of different sonication protocols including using a water bath sonicator, trying a four-step process and trying different sonication energies; the four involved removing the top phase of a previously sonicated sample after letting it settle for 48 hours and then sonicating that top phase before centrifuging it. We measured the particle size in the sample after each sonication experiment and found that the four-step process yielded the smallest particle sizes ranging from 70 nm to 400 nm. However, PHB concentration in the sample became minute at this point and therefore the relative success of different extraction methods would be difficult to determine at such small concentrations of PHB. Furthermore, we also had to mimic the concentration of PHB we would expect from the secretion system. </P><P>Therefore we decided to conduct our electrocoagulation experiments with both the sample containing microscale PHB, which would be representative of the concentrations we expected from the secretion system and the nanoscale PHB which we obtained from the four-step sonication process.</P><P>We used an electrocoagulation setup similar to the one used in (&quot;Make Water - Collaborative Water Purification&quot;, 2017) using iron and steel electrodes. We first tested whether electrocoagulation worked with PHB in a suspension of water. Therefore, we set up the separate electric cells with microscale PHB in suspension in water and nanoscale PHB in suspension in water. In both cases we found significant amounts of PHB settling within 3 hours of running the cell. We also noticed that once we stopping passing electricity through the cell, a layer of brown powder was deposited over the layer of PHB. We hypothesized this was probably iron(III) hydroxide</P><P>Next, we went on to test a 1:1 mixture of microscale PHB suspension and synthetic feces supernatant. However, this time we found that the settlement was a brown sludge with no discernable layer of PHB. This is was probably due to the high amounts of salts present in the supernatant; the iron ions released by the anode were most likely binding to other anions present within the sample as opposed to the PHB granules. We went on to wash the sludge with dilute sulphuric acid with the hopes of dissolving out the metal salts, but this was also unsuccessful in separating PHB from the sludge. Since electrocoagulation did not seem selective for PHB within our supernatant, we began searching for other methods to separate PHB. </P><I>Bioreactors Research </I><P>We have considered advantages and disadvantages of each bioreactor type. The major issue with membrane bioreactor is fouling - the process resulting in loss of performance of a membrane due to the deposition of suspended or dissolved substances on its external surfaces, at its pore openings, or within its pores. </P><P>The two runner ups for consideration were hollow fibre immersed membrane bioreactor and stirred tank bioreactor.</P><P><STRONG><EM>Hollow Fiber (HF) </EM></STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Nano filtration possibility Can provide 99.99% bacterial and bacterial debris removal, near to drinking water quality</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Models have been developed without and with aeration</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Can be considered/modified as continuous process</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Greater area --&gt; lower flux --&gt; lower energy requirement</SPAN></LI></UL><P><SPAN style="font-weight: 400;">However:</SPAN></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Fouling is a great issue dues to increased crew-time commitment</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">High bacterial waste due to fouling</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Would require spares and consumables</SPAN></LI></UL><P><STRONG>Stirred tank bioreactor + Self cleaning filter in the outflow stream:</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Appropriate for anaerobic processes</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Assures good mixing of feedstock</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Can monitor retention time</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Low energy consumption</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">No aeration required</SPAN></LI><P><SPAN style="font-weight: 400;">Advantages of using mechanical self-cleaned filter:</SPAN></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Continuous bacteria removal from the walls and recycling back into the bioreactor</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">No fouling issues</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Low power consumption</SPAN></LI></UL><P>It was decided to use the stirred tank bioreactor followed by the self-cleaning filter as the technology for bacterial fermentation, removal and recycling. </P><H3>Week 19: September 4-8</H3><P>This week we have finalized the VFA extraction process after revisiting all the different separation technologies. We have also found that some of our ESM calculations were slightly odd. The following is the summary of ESM estimations for different liquid-solid separation technologies:</P><P><TABLE><TBODY><TR><B><TD> </TD></B></TR></TBODY></TABLE></P><P><SPAN style="font-weight: 400;">Screw dewatering system</SPAN></P><P><SPAN style="font-weight: 400;">Multi-filtration</SPAN></P><P><SPAN style="font-weight: 400;">torrefaction</SPAN></P><P><SPAN style="font-weight: 400;">centrifugal separator  (lab scale)</SPAN></P><P><SPAN style="font-weight: 400;">Self cleaning filter</SPAN></P><P><SPAN style="font-weight: 400;">Power (kW)</SPAN></P><P><SPAN style="font-weight: 400;">0.298</SPAN></P><P><SPAN style="font-weight: 400;">1.84</SPAN></P><P><SPAN style="font-weight: 400;">0.88</SPAN></P><P><SPAN style="font-weight: 400;">7</SPAN></P><P><SPAN style="font-weight: 400;">2</SPAN></P><P><SPAN style="font-weight: 400;">Weight (kg)</SPAN></P><P><SPAN style="font-weight: 400;">179</SPAN></P><P><SPAN style="font-weight: 400;">232</SPAN></P><P><SPAN style="font-weight: 400;">378</SPAN></P><P><SPAN style="font-weight: 400;">5</SPAN></P><P><SPAN style="font-weight: 400;">16</SPAN></P><P><SPAN style="font-weight: 400;">Volume (m^3)</SPAN></P><P><SPAN style="font-weight: 400;">2</SPAN></P><P><SPAN style="font-weight: 400;">1.83</SPAN></P><P><SPAN style="font-weight: 400;">3.21</SPAN></P><P><SPAN style="font-weight: 400;">0.0138</SPAN></P><P><SPAN style="font-weight: 400;">0.028</SPAN></P><P><SPAN style="font-weight: 400;">CT (hours/(CT*day)</SPAN></P>     <P><SPAN style="font-weight: 400;">Size range of particles removal</SPAN></P> <P><SPAN style="font-weight: 400;">20 micron</SPAN></P> <P><SPAN style="font-weight: 400;">20 micron</SPAN></P><P><SPAN style="font-weight: 400;">10 micron</SPAN></P><P><SPAN style="font-weight: 400;">Spares and consumables mass (kg)/day</SPAN></P><P><SPAN style="font-weight: 400;">0.00846</SPAN></P><P><SPAN style="font-weight: 400;">0.3669</SPAN></P><P><SPAN style="font-weight: 400;">0</SPAN></P><P><SPAN style="font-weight: 400;">0</SPAN></P> <P><SPAN style="font-weight: 400;">Spares and consumables volume (m^3)</SPAN></P><P><SPAN style="font-weight: 400;">0.0098</SPAN></P><P><SPAN style="font-weight: 400;">0.004778</SPAN></P><P><SPAN style="font-weight: 400;">0</SPAN></P><P><SPAN style="font-weight: 400;">0</SPAN></P> <P><SPAN style="font-weight: 400;">ESM Estimation</SPAN></P><P><SPAN style="font-weight: 400;">1809.514</SPAN></P><P><SPAN style="font-weight: 400;">1559.01035</SPAN></P><P><SPAN style="font-weight: 400;">1149.525</SPAN></P><P><SPAN style="font-weight: 400;">616.9877</SPAN></P><P><SPAN style="font-weight: 400;">196.062</SPAN></P><P>Using the summary below, it was decided to use the centrifugal centrifuge followed by a self-cleaning filter for the removal of all the solid particles and sterilisation of the liquid throug filtration. </P><H3>Week 20: September 11-15</H3><P> This week we ran the PHB produced by the synthesis group and PHB standards from Sigma Aldrich on the HPLC. Following the protocol in (Karr, Waters, Emerich, 1983), we digested the PHB in 1 ml concentrated sulphuric acid at 90 degrees Celsius. We decided to run the digest for 3 hours because the PHB standards from Sigma were in large pellet form, and would probably be digested slower than the typical powder form of PHB. After the digestion was complete we cooled the samples on ice and rapidly mixed in 4 ml of 0.014N sulphuric acid. We diluted the samples by predicting the possible concentration of crotonic acid assuming an 80% conversion (Karr, Waters, Emerich, 1983) and then ran the samples on the HPLC. </P><H3>Week 21: September 18-22</H3><P> We also got back the results from the HPLC run we did last week. Unfortunately, while the PHB standards showed clear peaks for crotonic acid, our sample showed none. However, this could have been due to a number of reasons, including the fact that our digestion ran for too long. We found that crotonic acid was not stable for more than 60 minutes at 90 degrees Celsius (Karr, Waters, Emerich, 1983). Furthermore, our PHB was in powder form, whereas the PHB standards were in pellet form, it was quite possible that the digestion rate for our PHB was much faster. Next time, we plan to run higher concentrations of PHB and run the digest for only 30 minutes. </P><H3>Week 22: September 25-29</H3><P>This week we have focused on developing the safety analysis of our system. We have analysed our system and made certain adjustments to is to assure that the system is fail-safe, which means that the system's design prevents or mitigates unsafe consequences of the system's failure. That is, if and when a &quot;fail-safe&quot; system &quot;fails&quot;, it is &quot;safe&quot; or at least no less safe than when it was operating correctly. Because our system can operate in batch and continuous modes, it is possible to retain the matter inside different storage tanks throughout the process, thus allowing for longer retention while a specific component of the system is getting fixed.</P><P>We have also analysed our system and checked whether or not it follows the inherent safety guidelines well. Since the whole system was developed with the goal of minimizing and replacing the hazardous materials (Eg. chloroform) use and minimising the energy consumption, we found the process to be cohesive with most of the inherent safety guidelines.</P><P>We decided to adopt the NASA biosafety hazard levels to check whether or not using our process in Mars is possible. NASA is adapting the same biosafety hazard levels as the Center for Disease Control and Prevention (CDC) in the USA. The biohazard level 1 and 2 materials are allowed on the ISS, while levels 3 and 4 are generally prohibited. Since our <I>E. coli</I> possesses a minimal risk to humans and would not survive outside of the system it falls under biohazard level 1 and hence would be allowed in Martian expeditions. </P><P> finally we worked on identifying the potential points of system failure and have considered different arrangements of equipment in space to allow easy accessibility to the failure points by the astronauts. The points of failure are: filter after the centrifugal separator (clogging potential), bioreactor containment with dead cells, self-cleaning filter (clogging potential).</P><H3>Week 23: October 2-6</H3><P> This week, we have spent a lot of time in the lab preparing the syn poop supernatant for the synthesis group to run their experiments on. We have made two batches of syn poop, left it to ferment with non-PHB producing bacteria for 3 days and then centrifuged and filtered the supernatant. </P><P> We have continued to work on creating diagrams and writing up the Applied design page and the journal. </P><H3>Week 24: October 9-13</H3><P> This week the team was busy with writing up the missed sections from the journal and the wiki pages. We have made significant progress on uploading the first half of the journal onto our wiki, as well as in writing up the Applied design, Solid Liquid separation, and Products pages. 
</P><P> the team have also met with Genome Alberta, which successfully connected us to two professionals who used to work for Metabolix - a company that used to specialise in the PHB plastic production using plants. We held a Skype call with Kristi Snell and with Nii Patterson who has advised us on potential methods for PHB plastic characterisation in small scale:
<OL><LI> Looking at the cells under the light microscope to explore whether or not we can identify large inclusion bodies inside (PHB plastic granules). Then perform the Nile Blue staining protocol on PHB producing cells and on the control cells and look under the light microscope again to see if those inclusion bodies got colored in blue -- would mean than PHB plastic is present</LI><LI> We were advised against Nile Red staining, as it also colours the oil droplets inside the sell. Yet giving no other option Nile Red can be an alternative to Nile Blue</LI><LI> We were also advised against using HPLC for PHB characterisation since it was found insensitive. Yet it has been successful in certain literature</LI><LI> Both professionals have advised us to use Gas Chromatography for PHB characterisation, yet we would not have time to perform the experiments before the iGEM Giant Jamboree </LI></OL></P><P> We have further met with a professor in the engineering faculty, who have advised us to run our control and bacterial PHB on different chemical analytical equipment and then compare the produced peaks. This would have helped us to characterise the PHB by comparing the properties of the produced material to the properties of standard and the literature values. Yet we would not be able to generate enough plastic for those experiments in such short notice and hence can’t perform the experiments </P><H3> Week 25: October 16 - 20</H3><P> This week the team have continued working on writing and uploading the journal onto the wiki, as well as writing up the Overview, Fermentation and PHB extraction pages.</P><P> We have further coordinated with the synthesis group to adopt the Nile Red staining protocol and to look at the PHB producing cells under the microscope to move forward with the characterisation experiments </P><P> We finalized our extraction method this week and calculated the ESM for it. Literature has demonstrated the use of dissolved air flotation in the separation of PHAs with an overall yield of 86 ± 0.3% (van Hee, Elumbaring, van der Lans &amp; Van der Wielen, 2006). When water saturated with air is bubbled through media containing PHB, the tiny PHB granules float up to the top of the mixture due to particle-bubble interactions. The PHB-rich top layer can be separated from the mixture and then dried to produce relatively pure PHB powder. However, we would need to recycle the water we recover from our process back into the extraction because the compressed air is dissolved in water. We eliminated the design option using chemical coagulation because we would need to incorporate a backwashing step to remove the calcium chloride that would be attached to the PHB. We also eliminated electrocoagulation, based on the fact that it was not selective for PHB within our media. </P><TABLE><TBODY><TR><TH>Component</TH><TH>Volume (m3)</TH><TH>Power (kW)</TH><TH>Weight (kg)</TH><TH>ESM</TH></TR><TR><TD>Air Compressor</TD><TD>0.0396</TD><TD>0.3729</TD><TD>10</TD><TD>51.0157</TD></TR><TR><TD>Dissolved Air Vessel</TD><TD>0.8541</TD><TD>0</TD><TD>3.12</TD><TD>188.0327</TD></TR><TR><TD>Flotation Column</TD><TD>0.1153</TD><TD>0</TD><TD>15</TD><TD>39.9625</TD></TR><TR><TD>Drying Machine</TD><TD>0.012</TD><TD>0.249</TD><TD>13.03</TD><TD>316.3018</TD></TR><TR><TD><B>Total</B></TD><TD>76</TD><TD>0.057</TD><TD>1.82</TD><TD><B>246</B></TD></TR></TBODY></TABLE><H3> Week 25: October 24 - 27</H3><P> This week we ran PHB produced from the two different parts made by the synthesis group on the HPLC. We were able to detect peaks for crotonic acid for the samples extracted from both parts. This confirms that we did indeed produce PHB using our parts. </P><H3> Week 26: October 30 - November 3</H3><P> This week we melted commercial PHB in a steel mold of a small wrench to see if the plastic could indeed be used for that purpose. On cooling, the plastic hardened, but proved difficult to free from the mold, breaking when we tried to push it out. Perhaps a silicon mold would work better. </P><DIV id="Caption"><CENTER><B>Figure 1:</B>   A wrench molded from commercial PHB, the same type of plastic that was produced by engineered bacteria.</CENTER></DIV><BUTTON class="accordion"><H3> Modelling</H3></BUTTON><DIV class="panel"><H3> Week 1 &amp; 2: May 15 - May 26</H3><P>The modelling subgroup was involved in lab safety training and planning tasks for the  other subgroups they were part of.</P><H3> Week 3 (May 29 - June 2) </H3><P> The modelling subgroup  brainstormed a number of possible models for our project. The details of those models is given<A href="https://2017.igem.org/Team:Calgary/Model"> here</A>. To analyze the different mathematical models proposed in Week 1 &amp; 2, we used the following criteria:
<UL><LI> Usefulness to the project</LI><LI> Time required </LI><LI> Skills</LI><LI> Resources</LI></UL>

Among the models, FBA and kinetic model were most suitable for our project. The modelling subgroup deemed that flux balance analysis will help us find an optimal pathway for maximizing production of PHB in <I>E. coli</I> BL21. Furthermore, kinetic modelling will help us find loopholes in the pathway suggested by FBA. Hence, FBA and kinetic modelling will work together to improve the synthesis of PHB in <I>E. coli</I> (BL21). We contacted faculty members at the University of Calgary, who worked on mathematical modelling to discuss our plan for the summer.
</P><H3> Week 4: June 5 - June 16</H3><P>The modelling group met with Dr. MacCullum to discuss the possible mathematical modelling methods. The group was advised that flux balance analysis and kinetic model would be the best to pursue given our project's scope as it will inform our experiments and is feasible in the given time.<A href="https://opencobra.github.io/cobratoolbox/latest/">OpenCobra toolbox</A> for MATLAB was installed because it has functions for carrying out flux-balance analysis and visualizing the results.</P><H3> Week 5: June 19 - June 23 </H3><P>This week we met with a group of postdocs working in Dr. Ian Lewis’s lab. We discussed how flux balance analysis could be helpful for our project. We also discussed flux variability analysis (FVA) in comparison to flux balance analysis. We decided that after finding optimal solutions using FBA, we can look into FVA. We were given some suggestions on some objectives we could look for in our model such as optimizing bacterial growth, optimizing PHB production using glycolysis only, and optimizing PHB production using beta-oxidation pathway only. This could be done by changing the parameters of the command optimizeCbModel(‘parameter’). 
</P><H3> Week 6 &amp; 7: June 26 - July 7 </H3><P>The modeling team had a meeting to discuss the milestones for flux balance analysis and kinetic model. We decided on the different models we plan to optimize using the flux balance analysis and kinetic modelling. We plan to study a number of models. One of the models will contain pathways for beta-oxidation and PhaCJ genes that can help produce PHB from medium-chain and long-chain fatty acids. The other model will contain CBA genes and PhaCJ, which can help produce PHB from glucose, short-chain, medium-chain, and long-chain fatty acids. The kinetic model will look into reactions that are rate-limiting in these pathways. Thus, FBA and kinetic model will work together to help optimize the production of PHB in <I>E. coli</I> (BL21) and modify substrate concentrations involved in the rate-limiting steps.</P><H3> Week 8 &amp; 9: July 10 - July 21 </H3><P>FBA:
  For the flux balance analysis we found an e coli model for our strain, which is BL21 (DE3). The model was found from the BiGG database. We looked into the genes and reactions it contained and the reactions that have to be added to the model. The genes of interest that the model already contained are: 
<UL><LI>FadD</LI><LI>FadE</LI><LI>tolC</LI><LI>LacY</LI><LI>LacZ</LI></UL>
We found that we needed to add the following genes and as a result their reactions: 
<UL><LI>PhaA</LI><LI>PhaB1</LI><LI>PhaC1</LI><LI>HlyA</LI><LI>HlyB</LI></UL></P><P>We also used a plugin called Paint4Net to visualise the pathways/reactions in the model. A zoomed in section of the visual representation of FBA analysis resulting from calling the &quot;draw_by_rxn&quot; command for the coli_core_model is given in figure 1.
</P><DIV id="Caption"><B>Figure 1. A zoomed in section of the Paint4Net dray_by_rxn command on the ecoli_core_model.</B></DIV><P>Kinetic:
  After deciding to do a kinetic model. We brainstormed what questions we would try to answer with our model. 

We wanted to compare PHB production via the beta-oxidation pathway and the glycolysis pathway. Therefore we would create a model with phaJ and phaC and another with phaC, phaB, and phaA. We also wanted to compare overexpressing fadD and fadE and see which one had a higher effect on PHB yield. The following are two other systems we hope to model.
Some other questions we considered answering with our kinetic model include:
What is the rate-limiting step in the synthesis of PHB?
What is the rate-limiting step in the secretion of PHB?

With those questions in mind, we started searching for existing models for beta-oxidation and PHB synthesis. We decided to base the PHB synthesis part of our model on the one created by the 2013 Imperial team. [1] For the beta-oxidation part, since we could not find an established model we looked at the pathways for degradation of oleic acid (the main long-chain fatty acid in our media) via beta-oxidation (Ren et al., 2004).</P><H3> Week 10 &amp; 11: July 24 - Aug 4 </H3><P>FBA:
This week we looked into having a visual representation of the reactions/pathways in e coli BL21. However, the graph had too many nodes and the cobra toolbox could not visualise it. Thus, we looked into plotting a part or subsystems that included reactions of interest. This was done after optimizing the model and then calling the draw_by_rxn command. The command we used to select the specific reactions of interest such as the citric acid cycle was:
fluxReactions = Model.rxns(ismember(Model.subSystems,'Citric Acid Cycle'));
</P><DIV id="Caption"><B><DIV id="Caption"><B>Figure 2. A zoomed in section of the Paint4Net dray_by_rxn command on the ecoli_core_model.</B></DIV></B></DIV><P> Unfortunately, members of the modelling subgroup were more involved with running experiments and other lab duties. The models chosen required more research in order to be developed for our project and required a lot of time. Thus, the group decided to prioritize lab experiments. </P></DIV><BUTTON class="accordion"><H3> Human Practices</H3></BUTTON><DIV class="panel"><H3>Week 1: May 1 - May 5</H3><P>
In the planning phases of our project, our team envisioned a wastewater treatment application to our engineered bacteria; however, much of the planning work done before the start of May was focused on the synthetic biology and engineering aspects of our project. On May 2nd, our team began holding human practices meetings. These meetings, held throughout the summer, focused on both high-level policy aspects of human practices and the education and public engagement aspect of human practices. At our first human practices meeting, the human practices team brainstormed different ways to get involved with the community.
</P><P>
We made a list of bioplastic production companies to contact regarding the input they would have for our team. We sent emails to TELUS Spark, Minds in Motion, TedX, and the wastewater treatment plant in order to secure possible meetings with these organizations. 
</P><P>
We also considered different aspects of policy to look into if we had chosen the wastewater treatment application: the safety of bioplastics, the impact of our engineered bacteria on the ecosystem, and the different applications to which our bioplastic is suited (medical implants, 3D printing, etc.) We reviewed the 2016 UofC Calgary team’s human practices efforts in order to understand possible avenues which we could take for our human practices efforts this summer.
</P><H3> Week 2: May 8 - May 12</H3><P>
We continued to follow up with contacts at the Pine Creek wastewater treatment plant. On May 9th, Michaela met with Magdalena Pop (Magda) from GeekStarter to discuss logistics for a workshop which GeekStarter was planning on holding at the University of Calgary for all of the Alberta iGEM teams. The list of requirements for the workshop was provided by Magda and is listed below:

<UL><LI>Room booking (lecture theatre for expert presentations and small rooms for breakout sessions) </LI><LI>Catering (lunch, served buffet-style in the HRIC atrium) </LI><LI>Printed signs and providing directions to the different rooms during the event (needed to know the number of signs and directions of arrows needed) </LI><LI>Accommodating the final number of attendants (provided by Magda by May 23), as well as any dietary restrictions for the 50-70 planned attendees </LI><LI>Help with snacks and coffee for the afternoon (which was not provided through catering company) </LI><LI>No hotel rates for out-of-town guests could be arranged (as the number of attendees was not yet known) </LI></UL></P><H3> Week 3: May 15 - May 19</H3><P>
We looked into sponsorship opportunities with Genome Alberta, based on past teams from Calgary receiving funding from the aforementioned organization. On May 18th, Michaela scheduled a meeting with Magda for May 30th at 2:00 PM (MST) in order to review and finalize the plan for the June 11th GeekStarter workshop. Michaela also contacted <A href="http://www.ucalgary.ca/ancillaryservices/catering">Lamiley Ludderodt </A>, a caterer affiliated with the University of Calgary, regarding the catering for the workshop. 
</P><P>
Our team began to read into microbead policies, as we were aware that microbeads were being phased out of cosmetic products in Canada, and the option of biodegradable microbeads was not widely known to the public. We wondered how increased public awareness of the possibility of biodegradable microbeads would impact government action on what is seen in Canada as a serious ecological issue. A summary of our findings can be found on our human practices page, as well as under Week 6 of this journal.
</P><H3> Week 4: May 22- May 26</H3><P>
We began emailing other Canadian iGEM teams in search of collaboration opportunities. UBC’s iGEM team replied to our email and asked for our CRISPR/Cas9 protocols. McMaster also replied to our email and expressed interest in a wet lab collaboration with our team.</P><P>
We also inquired about their interest in possibly organizing a Canadian iGEM team Newsletter. Helen took on the initiative of managing the submissions, editing, and production of the newsletter, which she envisioned as an accessible platform for collaborations between Canadian teams.</P><P>
On Thursday, May 25, the entire iGEM Calgary team attended a tour of the Pine Creek wastewater treatment plant. We learned about the different stages of wastewater treatment. An image of our team at the wastewater treatment plant is shown below:
</P><DIV id="Caption"><B>Figure 1: </B> iGEM Calgary 2017 visits the Pine Creek wastewater treatment plant. Image courtesy of Lalit Bharadwaj.</DIV><P>
We also discovered that integrating our engineered bacteria into the wastewater treatment system may not be cost-effective and that the natural PHB-producing microbes would be easier to culture in the bioreactors which already exist at the wastewater treatment plant. We also considered the costs of implementing our system into the wastewater treatment infrastructure and realized that any profit from the sale of the plastic would be negligible, especially since the wastewater treatment plant was already obtaining a value-added product (biogas, for electricity generation) from its treatment process.
</P><P>
By the end of the week, our team had a clearer direction as to the possible impacts and applications of the plastic produced. Our findings are summarized below:
</P><P>
Literature and team brainstorm summary (applications of PHB):
</P><P> This project will also explore whether bioplastic production is actually more environmentally friendly than traditional plastic production. Factors to consider include carbon usage and storage, biodegradability, contribution to landfill waste, and the ecological impact of the bacteria we create. Ultimately, it is important to our team to identify a few key applications (preliminary research has suggested medical implants and microbeads) where our product would be safe, non-polluting, and effective. With regards to the bacteria, we are looking into ways to use antibiotic resistance-free selection mechanisms so that our bacteria do not pass on resistance genes to wild-type populations of bacteria. This will ensure that the water supply does not become contaminated. </P><P> Medical implants could be a good application for our product since the specific PHA product we hope to synthesize, Poly-(3-hydroxybutyrate-co-3-hydroxyvalerate (PHBV) is nontoxic and biocompatible (in mouse studies) as long as it is treated to remove inflammation-causing agents prior to tissue implantation (Williams et al., 1998). PHBV is also a good candidate for creating strong sutures which can naturally degrade (Williams et al., 1998). </P><P>Microbeads in cosmetics seem to be a great application for our product due to the need for a biodegradable alternative to traditional plastic microbeads. The biodegradability of PHA microbeads would mean that microbeads do not damage aquatic ecosystems. Our conversations with experts in the field of bioplastics have alerted us to the fact that many nations have banned all plastic microbeads regardless of their biodegradability; the policies enacted by these nations as they relate to PHA microbeads are an important future area of study.</P><H3> Week 5: May 29 - June 2</H3><P>
CRISPR and dCas9 protocols were obtained from the Childs Lab at the University of Calgary. These were then sent to the UBC iGEM team as per their request. The remaining Canadian iGEM teams were contacted to follow up on possible collaborations and the newsletter. All the Canadian teams got back to us and expressed their interest in establishing the newsletter. </P><P>On May 30th, our team began researching the different applications for our project, as the wastewater treatment plant field trip made us rethink the direction of our project. The applications we chose to look further into were:
</P><UL><LI> Wastewater treatment </LI><LI> Small-scale wastewater treatment in developing countries </LI><LI> Landfill leachate treatment </LI><LI>Outer space missions </LI></UL><P>
A few team members were chosen to look into specific aspects of each application so as to gain a holistic understanding of the feasibility of each application and thus choose the application which was the best fit for our project. The aspects of each application which our team researched are listed as questions below:
</P><OL><LI> Is there a significant demand for a solution to this problem? </LI><LI> What are the costs associated with implementing our process to solve this problem? </LI><LI> Do we have any resources or industry contacts in this field? Whom should we contact for an expert opinion if we choose this issue? </LI><LI> What is the environmental/social/political/economic impact of our engineered bacteria as a solution to this problem? </LI><LI> Is synthetic biology the best solution to solving this problem? </LI></OL><P>
We made a table comparing the above aspects of the different applications. This table is shown below:
</P><DIV id="Caption"><B>Table 1: </B>Comparing different applications of our engineered <I>E. coli</I>.</DIV><TABLE><TBODY><TR><TH>Wastewater treatment</TH><TH> Small-scale wastewater treatment in developing countries </TH><TH> Landfill leachate treatment </TH><TH> Outer space </TH></TR><TR><TH>Demand</TH><TD>Moderate</TD><TD>High</TD><TD>Low</TD><TD>High</TD></TR><TR><TH>Costs</TH><TD>High</TD><TD>Moderate</TD><TD>Moderate</TD><TD>Low</TD></TR><TR><TH>Impact (Safety, Environment, Society, Economy)</TH><TD>Low</TD><TD>High</TD><TD>Low</TD><TD>Moderate</TD></TR><TR><TH>Available Resources and Industry Contacts</TH><TD>Many</TD><TD>Some</TD><TD>Few</TD><TD>Many</TD></TR><TR><TH>Is Synthetic Biology Best?</TH><TD>Not yet</TD><TD>Not yet</TD><TD>Not yet</TD><TD><FONT color="#5ec3af">Yes</FONT></TD></TR></TBODY></TABLE><P>
On May 31st, we had a Skype meeting with the UNBC iGEM team to discuss project details. The notes taken during that video call are summarized below:
</P><P>
UNBC Project details (chassis, etc.)
</P><UL><LI>Using a Level 1 Staph strain as a proof of concept for Level 2 Staph aureus </LI><LI>Phage delivery system for spreading their plasmids in their bacterial population </LI></UL><P>
Newsletter 
</P><UL><LI>UNBC “definitely interested” in newsletter and were interested in the July issue </LI></UL><P>
Collaboration opportunities:
</P><UL><LI> Their team: mostly biomedical/biochemistry </LI><LI> May need help in wiki and engineering (mostly wiki since their project is very biomedical) </LI><LI> They said they have a very advanced analytical lab (may be able to help us with characterizing our PHB product later on) </LI><H3> Week 6: June 5th - June 9</H3><P>
Since our team decided that pursuing the avenue of wastewater treatment was no longer the best possible direction for our project to take, we decided that our microbead research, while valuable, would be too far outside the scope of our space travel application to have a meaningful connection to our project. Our human practices efforts were thus rerouted to ensuring the health of the astronauts who would be using our system on long-term space missions; however, we have summarized our preliminary research on microbeads below:
</P><H4> Annotated Bibliography: Microbeads in Canadian Public Opinion/ Policy </H4><P>McDevitt, J.P., Criddle, C.S., Morse, M., Hale, R.C., Bott, C.B., Rochman, C.M. Addressing the issue of microplastics in the wake of the Microbead-Free Waters Act: a new standard can facilitate improved policy. Environ Sci Technol. 2017 May 15. <A href="http://dx.doi.org/10.1021/acs.est.6b05812">http://dx.doi.org/10.1021/acs.est.6b05812</A>. [Epub ahead of print].</P><UL><LI>From this article, we learned that microplastics (&lt;5mm) are a major environmental concern. 4.8 - 12.7 million metric tons of plastic waste is mismanaged, and this mass of plastic is expected to increase tenfold by 2025. This source also mentioned a concern regarding the adverse impact to animals such as fish, oyster etc. from the microplastics. Over 100 species were reported with microbead accumulation. 
Reason for microbead usage is that there is no compelling incentive for corporations to stop using them. 
This article suggests the incorporation and usage of the term “Ecocylable” to refer to materials that are degradable in soil, wastewater etc. and which pass toxicity and bioaccumulation standards. </LI></UL><P>Government of Canada. (2016). Microbeads in Toiletries Regulations. 05 November 2016. Accessed from: <A href=" http://www.gazette.gc.ca/rp-pr/p1/2016/2016-11-05/html/reg2-eng.php "> http://www.gazette.gc.ca/rp-pr/p1/2016/2016-11-05/html/reg2-eng.php </A>. </P><UL><LI>From this archived government document, we learned that Canada has already passed legislation to phase out microbeads from toiletries by mid-2018. The period of review for this legislation had already lapsed by the time we began our search.</LI></UL><P> We corresponded with other iGEM teams in Canada through email. </P><H3> interlude 1: GeekStarter workshop (June 11, 2017)</H3><P>
The GeekStarter MindFuel Workshop allowed our team to consult with experts experienced with iGEM and ask for their input on our project. We got specific advice in four areas of our project during the breakout sessions: how to design a wiki, how to model our experiments in an iterative way, how to brand ourselves, and how to go about contacting experts. We also had opportunities to speak to the other Alberta synthetic biology teams, including UrbanTundra high school, the University of Lethbridge, Lethbridge High School, and the University of Alberta. 
</P><H3> Week 7: June 12 - June 16</H3><P>On Monday: Sam, Helen, Lalit, and Alina went to Sir Winston Churchill High School, a local school with IB (International Baccalaureate) Biology classes, to host a lecture about synthetic biology followed by a guided DNA extraction experiment. At our weekly team meeting on Tuesday, we made an exhaustive list of different fundraising opportunities including crowdfunding, promotional events, and corporate sponsorships. We researched different crowdfunding websites, set up a crowdfunding page and began promoting it on team social media and to our personal networks.</P><H3> Week 8: June 19 - June 23</H3><P>This week, our team focused on fundraising. We began the process of applying to various corporate sponsorships.</P><P>The policy side of our team began to look into how to lobby for a more concrete space policy. We attempted to contact the Canadian Space Agency to get more defined answers about Canada’s space strategy, as we wanted to consider how our project would work with current or future space policy.</P><P>We also contacted iGEM HQ about the safety of our project if we were to use human waste to prove our concept.</P><P>Preetha began to discuss possible collaborations with our local TELUS Spark science centre, given that we now had a solid understanding of our project direction.</P><P>Lastly, we met with a representative of Calgary Labs in order to ask about what types of data are collected at city wastewater treatment plants and to request this data for our use.</P><P>The general guidelines and submission requirements for the Canadian iGEM Newsletter were sent out to the Canadian teams. We also sent out a form for all participating teams to fill out. This form served as confirmation for participation and allowed the teams to choose which issues they would be interested in participating in.</P><H3> Week 9: June 26 - June 30</H3><P>This week, two members of the team met the president of the University of Calgary, Dr. Cannon, to discuss the project’s progress.</P><DIV id="Caption"><B>Figure 2: </B> (From left to right:) iGEM Calgary team members Alina Kunitskaya and Preetha Gopalakrishnan, University of Calgary President Dr. Elizabeth Cannon, and iGEM Calgary 2017 Supervisor Dr. Mayi Arcellana-Panlilio. Photo courtesy of the office of Dr. Elizabeth Cannon.</DIV><P>A meeting was scheduled for next Thursday (July 6th) to meet with a team member from the McMaster iGEM team (Dhanyasri Maddiboina) who is currently in Calgary for an internship. Our goal for this meeting was to gain a deeper understanding of the McMaster team’s project, share details of our project, and discuss possible collaborations between the two teams. The students who agreed to represent iGEM Calgary at this meeting were Helen, Lalit, Preetha, and Maliyat.</P><P>On Thursday (June 29th), all of our team members skyped with Canadian astronaut Chris Hadfield, and gained some invaluable advice as to how to design and market our project for use in space.</P><DIV id="Caption"><B>Figure 3: </B> The iGEM Calgary 2017 team attended a skype call with Col. Chris Hadfield. Photo courtesy of David Feehan.</DIV><P>We also spent a large portion of Friday afternoon discussing our public engagement strategy, and overlap between the “public engagement” and “integrated human practices” category. We decided to do a literature search of barriers to accessing synthetic biology and eventually create a list of policy recommendations. We also set a timeline for the production of our iGEM manual. Our goal for this is to compile a collection of resources that will help the formation of new iGEM teams, therefore increasing public involvement and engagement in synthetic biology.</P><H3> Week 10: July 4 - July 7</H3><P>This week, we had a meeting with Dhanyasri from McMaster University and we discussed possible opportunities for both dry lab and wet lab collaborations between our teams. The notes from this meeting are listed below:</P><P>The McMaster Project: a fast detector for pathogenic bacteria; antigen which uses DNAzymes to target specific protein from the bacteria. The McMaster team is using <I>E. coli</I> as proof of concept before moving onto a more pathogenic bacterium (<I>Clostridium difficile</I>).</P><P>Software improvement: if one of us generates a software, we can ask their team to test it. McMaster has a large dry lab team.</P><P>Characterization: reproducibility of parts and experimental data (protein assays, etc.) could be shown if both teams test each other's parts.</P><P>Containment: they are making a detector for <I>E. coli</I> as a proof of concept first so they can send it to us as one possible method to ensure the containment of <I>E. coli</I> on the colony on Mars. As soon as a leak occurs, the detector will detect it within 5 to 10 minutes. This allows for appropriate action to be taken in the event of a containment catastrophe.</P><P>McMaster iGEM was planning on starting a general Synthetic Biology Journal comprised of review articles and primary research happening across Canada. One possible avenue for a collaboration with them would be to write a submission in regards to Synthetic Biology projects/discoveries around the university.</P><P>The McMaster team also had some tips to contribute to our manual. Their main difficulty: getting funding and finding a PI while maintaining the project as student-led. Dhanyasri explained that most primary PIs the team spoke to wanted their project to have more relations to their research.</P><P>We also fundraised by selling tickets for a Stampede Kickoff at Knoxville’s Calgary. The kickoff was on July 6th, 2017, and we raised almost $400 CAD through this fundraising venture.</P><H3> Week 11: July 10 - July 14</H3><P>Throughout the week, our team worked on a draft of the iGEM Manual. Our goal for this is to compile a collection of resources that will help the formation of new iGEM teams. The team was divided into teams of 2 people and each group worked to complete one manual topic. These discrete topics were then compiled into a summary document by Preetha.</P><DIV id="Caption"><B>Table 2: </B> Subdivision of the iGEM Calgary 2017 team into manual topic groups </DIV><TABLE><TBODY><TR><TH>Topic</TH><TH>Students</TH></TR><TR><TD> Generating student interest </TD><TD> Preetha and Alina </TD></TR><TR><TD> What you need </TD><TD> Kaitlin </TD></TR><TR><TD>Interdisciplinary recruitment </TD><TD> Helen and Maliyat </TD></TR><TR><TD> General tips for success </TD><TD> Preetha and Jacob </TD></TR><TR><TD> Grant Applications </TD><TD> Atika </TD></TR><TR><TD> Fundraising </TD><TD> Lalit and Alex </TD></TR><TR><TD> Synthetic biology mythbusting in your community </TD><TD> Alina and Amy </TD></TR><TR><TD> Marketing your project </TD><TD> Bilal and Tricia </TD></TR><TR><TD> Wiki tips </TD><TD> Michaela </TD></TR></TBODY></TABLE><P>
On Friday: Helen, Preetha, Michaela, Alex, and Kaitlin led a Genetic Engineering workshop for 24 Grade 7 to 9 students at the Minds in Motion Summer Program. For our presentation, we talked about the importance of genetic engineering in agriculture and medicine. The students were given the opportunity to extract strawberry DNA in pairs.
</P><DIV id="Caption"><B>Figure 4: </B>iGEM Calgary 2017 team members held a workshop for grades 7-9 students at Minds in Motion Summer Camps. Photo courtesy of Minds in Motion.</DIV><H3> Week 12: July 17 - July 21</H3><P>We sent a reminder email for the Canadian teams that have yet to confirm their participation in the Canadian iGEM Newsletter to fill out our form.</P><P>We also continued the literature search detailing barriers to accessing synthetic biology.</P><H3> Week 13: July 24 - July 28</H3><P>Kaitlin made the world map illustrating access to synthetic biology for the iGEM Manual. </P><H3> Week 14: July 31 – August 4</H3><P>This week, we had our second meeting with iGEM McMaster where we finalized the wet-lab collaboration between the two teams. iGEM McMaster will be sending us their proof-of-concept <I>E. coli</I>-detecting DNAzyme to us for us to test. </P><P>We also continued the literature search detailing barriers to accessing synthetic biology.</P><H3> Week 15: August 8 – August 11</H3><P> We started working on the wiki content for the human practices section of the wiki. Preetha and Tricia began to write and design the silver and gold medal human practices pages. Preetha also filled out the form for TELUS Spark and dealt with the insurance provider (the University of Calgary) for this event.
</P><H3> Week 16: August 14 – August 18</H3><P>We hosted a second Minds in Motion workshop on August 16<SUP>th</SUP>.</P><H3> Week 17: August 21 - August 25</H3><P> iGEM Calgary team members called various businesses around the city in order to obtain silent auction items for the Space Slam fundraiser.</P><H3> Week 18: August 28 - September 1</H3><P> We polished the iGEM Manual script and spent time planning our Space Slam fundraiser. </P><H3>Week 19: September 4 - September 8</H3><P>iGEM Calgary team members prepared for the TELUS Spark Adults Only Night booth by doing a trial run of the bacteria-free biopolymer demo (explained below) and compiling a small presentation about our project.</P><H3>Week 20: September 11 - September 15</H3><P>On September 14th, some team members (Preetha, Lalit, Alex, Maliyat, Amy, Kaitlyn) ran a booth at the TELUS Spark Adults Only Night. There, we explained iGEM, synthetic biology, and a little bit about our project while guiding them through a demonstration where common kitchen items (glycerol, vinegar, water, and cornstarch) were heated and turned into a biopolymer. This outreach allowed members of the general public to use scientific instruments such as beakers and micropipettes.</P><P>Several people also had questions about synthetic biology and genetic engineering, namely GMOs. Our team was able to meaningfully discuss synthetic biology and improve the public’s understanding of the topic. </P><H3>Week 21: September 18 - September 22</H3><P> The entire iGEM Calgary 2017 team spent this week preparing for the Alberta iGEM team meetup (aGEM) hosted by MindFuel. The presenters (Amy, Sam, and Alina) worked hard on their oral presentations, while Tricia designed the PowerPoint slides. </P><H3>INTERLUDE 2: aGEM (September 23rd and 24, 2017) </H3><P>We competed at the Alberta Genetically Engineered Machines Competition and took home 1<SUP>st</SUP> place, as well as Best Presentation and Best Integrated Human Practices.</P><H3>Week 22: September 25 - September 29</H3><P>We continued the literature search detailing barriers to accessing synthetic biology for the iGEM Manual.</P><P>We also received the DNAzyme sample sent to us by <A href="https://2017.igem.org/Team:McMasterU">iGEM McMasterU</A>. To prepare for the wet-lab collaboration, we made streak plates for <I>E. coli</I> DH5α and <I>Bacillus subtilis</I> WB800 and autoclaved 200mL of 1M MgSO<SUB>4</SUB> and 1M CaCl<SUB>2</SUB>. </P><P>This week we also held our first annual iGEM Fundraiser. Space Slam was a huge success and we raised over $2000 for the team through a combination of ticket sales and silent auction items.</P><H3>Week 23: October 2 - October 6</H3><P>For the collaboration with <A href="https://2017.igem.org/Team:McMasterU">iGEM McMasterU</A>, we prepared and autoclaved the M9 liquid media and the M9 plates. The plates were not poured smoothly. As such, we re-made the M9 plates after autoclaving more plating media. </P><H3>Week 24: October 9 - October 13</H3><P>We made four overnight cultures, with two for <I>E. coli</I> DH5α and another two for <I>B. subtilis</I> WB800. After 24 hours of incubation, the OD600 absorbances of the overnight cultures were measured and diluted appropriately with M9 liquid media to attain a cell density of 10<SUP>6</SUP> cells/mL. No growth was observed on the two plates after 36 hours so we re-made four plates, with 2 for <I>E. coli</I> DH5α and 2 for <I>B. subtilis</I> WB800. Again, no growth was observed. Overnight cultures were re-made, one for <I>E. coli</I> DH5α and another for <I>B. subtilis</I>.</P><H3>Week 25: October 16 - October 20</H3><P>The DNAzyme sent by <A href="https://2017.igem.org/Team:McMasterU">iGEM McMasterU</A> was diluted to the suggested concentration of 8μM. After 48 hours of incubation, the growth on the <I>E. coli</I> DH5α and <I>B. subtilis</I> WB800 quadrants of the plates were enough to plate the DNAzyme. After incubation, a picture was taken of the wet-lab experiment results. Due to limited amounts of DNAzyme sent by <A href="https://2017.igem.org/Team:McMasterU">iGEM McMasterU</A>, no replicates or repetitions could be done.
</P><DIV id="Caption"><B>Figure 5:</B>  Final results of the <I>E. coli</I> DH5α and <I>B. subtilis</I> WB800 plates after the addition of DNAzyme. No bright fluorescence can be detected throughout the plates. The areas circled in red indicate the location in which the DNAzyme was dispensed on the quadrants containing the growth of <I>E. coli</I> DH5α (left) and <I>B. subtilis</I> WB800 (right).</DIV><H3>Week 26: October 23 - October 27</H3><P>We made two presentation posters for iGEM, with one focusing on wet-lab work and another on the iGEM Manual with the theme Access to Synthetic Biology worldwide. We presented them at the University of Calgary Cumming School of Medicine Bachelor of Health Sciences Undergraduate Research Symposium.</P><H3>Week 27: October 30 – November 1</H3><P>On October 30, 2017, Preetha visited Master’s Academy, a local high school, to share our project and give interested students more information about our iGEM experience so that they could start their own iGEM team for the 2018 competition. </P><DIV class="apa-reference"><H2> Works Cited </H2><P>Anderson, G., &amp; Yang, G. (1992). Determination of Bicarbonate and Total Volatile Acid Concentration in Anaerobic Digesters Using a Simple Titration. Water Environment Research, 64(1), 53-59</P><P>A. Serio, M., E. Cosgrove, J., &amp; A. Wojtowicz, M. (2016). Torrefaction Processing for Human Solid Waste Managment. In 46th International Conference on Environmental Systems (pp. 1-18). East Hartford: Advanced Fuel Research, Inc. </P><P>Bengtsson, S., Werker, A., Christensson, M., &amp; Welander, T. (2008). Production of polyhydroxyalkanoates by activated sludge treating a paper mill wastewater. Bioresource Technology, 99(3), 509-516</P><P>Choi, J., &amp; Lee, S. (1997). Process analysis and economic evaluation for Poly(3-hydroxybutyrate) production by fermentation. Bioprocess Engineering, 17(6), 335-342 </P><P>Coats, E., VandeVoort, K., Darby, J., &amp; Loge, f. (2011). Toward Polyhydroxyalkanoate Production Concurrent with Municipal Wastewater Treatment in a Sequencing Batch Reactor System. Journal Of Environmental Engineering, 137(1)</P><P>Colon, J., A. Fobris-Stokes, A., &amp; A. Deshusses, M. (2015). Anaerobic digestion of undiluted simulant human excreta for sanitation and energy recovery in less-developed countries. Energy And Sustainable Development, 29, 57-64</P><P>Dae Lee, B., Otorowski, H., Son, R., Pang, A., Chen, E., &amp; Araiza, D. (2010). Large Intestine - MicrobeWiki. Microbewiki.kenyon.edu. Retrieved 16 August 2017, from <A href="https://microbewiki.kenyon.edu/index.php/Large_Intestine">https://microbewiki.kenyon.edu/index.php/Large_Intestine</A></P><P>Government of Canada. (2016). Microbeads in Toiletries Regulations. 05 November 2016. Accessed from: <A href=" http://www.gazette.gc.ca/rp-pr/p1/2016/2016-11-05/html/reg2-eng.php "> http://www.gazette.gc.ca/rp-pr/p1/2016/2016-11-05/html/reg2-eng.php</A></P><P>Jiang, Y., Mikova, G., Kleerebezem, R., van der Wielen, L., &amp; Cuellar, M. (2015). Feasibility study of an alkaline-based chemical treatment for the purification of polyhydroxybutyrate produced by a mixed enriched culture. AMB Express, 5(1).<A href=" http://dx.doi.org/10.1186/s13568-015-0096-5"> http://dx.doi.org/10.1186/s13568-015-0096-5</A></P><P> Karr, D. B., Waters, J. K., &amp; Emerich, D. W. (1983). Analysis of poly-β-hydroxybutyrate in <I>Rhizobium japonicum</I> bacteroids by ion-exclusion high-pressure liquid chromatography and UV detection. Applied and environmental microbiology, 46(6), 1339-1344</P><P>Kashef, O., &amp; Lungue, L. (2016). Successes of The City of Calgary’s Leachate Treatment Pilot Plant, and Use of Treated Leachate to Build a Greener Future. Presentation</P><P>Leachate treatment in China: Technologies and Import Opportunities. (2015). Inside Industrial Infrastructure (ICORP)</P><P>Lifschits, C., Wolin, M., &amp; Reeds, P. (1990). Characterization of Carbohydrate Fermentation in Feces of Formula-Fed and Breast-Fed Infants. Paediatric Research, 27(2)</P><P> Make Water - Collaborative Water Purification. (2017). Hackaday.io. Retrieved 30 October 2017, from <A href="https://hackaday.io/project/20812-make-water-collaborative-water-purification#menu-description">https://hackaday.io/project/20812-make-water-collaborative-water-purification#menu-description&quot;</A></P><P>Manufacturing and properties of PHB. (2017)</P><P>McDevitt, J.P., Criddle, C.S., Morse, M., Hale, R.C., Bott, C.B., Rochman, C.M. Addressing the issue of microplastics in the wake of the Microbead-Free Waters Act: a new standard can facilitate improved policy. Environ Sci Technol. 2017 May 15. <A href="http://dx.doi.org/10.1021/acs.est.6b05812">http://dx.doi.org/10.1021/acs.est.6b05812</A>. [Epub ahead of print]</P><P>Palanivel, T., &amp; Sulaiman, H. (2014). Generation and Composition of Municipal Solid Waste (MSW) in Muscat, Sultanate of Oman. APCBEE Procedia</P><P>Pereira, T., Oliveira, M., Maia, I., Silva, J., Costa, M., &amp; Thiré, R. (2012). 3D Printing of Poly(3-hydroxybutyrate) Porous Structures Using Selective Laser Sintering. Macromolecular Symposia, 319(1), 64-73. <A href="http://dx.doi.org/10.1002/masy.201100237">http://dx.doi.org/10.1002/masy.201100237</A></P><P>Rahman, A., Linton, E., Hatch, A., Sims, R., &amp; Miller, C. (2013). Secretion of polyhydroxybutyrate in Escherichia coli using a synthetic biological engineering approach. Journal Of Biological Engineering, 7(1), 24. <A href="http://dx.doi.org/10.1186/1754-1611-7-24">http://dx.doi.org/10.1186/1754-1611-7-24</A></P><P>Resch, S., Gruber, K., Wanner, G., Slater, S., Dennis, D., &amp; Lubitz, W. (1998). Aqueous release and purification of poly(β-hydroxybutyrate) from Escherichia coli. Journal Of Biotechnology, 65(2-3), 173-182. <A href="http://dx.doi.org/10.1016/s0168-1656(98)00127-8">http://dx.doi.org/10.1016/s0168-1656(98)00127-8</A></P><P>Rhu, D., Lee, W., Kim, J., &amp; Choi, E. (2003). Polyhydroxyalkanoate (PHA) production from waste. Water Science &amp; Technology, 48(8), 221-228</P><P>Rose, C., Parker, A., Jefferson, B., &amp; Cartmell, E. (2015). The Characterization of Feces and Urine: A Review of the Literature to Inform Advanced Treatment Technology. Critical Reviews In Environmental Science And Technology, 45(17), 1827-1879. <A href="http://dx.doi.org/10.1080/10643389.2014.1000761">http://dx.doi.org/10.1080/10643389.2014.1000761</A></P><P>Sperling, R., &amp; Parak, W. (2010). Surface modification, functionalization and bioconjugation of colloidal inorganic nanoparticles. Philosophical Transactions Of The Royal Society A: Mathematical, Physical And Engineering Sciences, 368(1915), 1333-1383. <A href="http://dx.doi.org/10.1098/rsta.2009.0273">http://dx.doi.org/10.1098/rsta.2009.0273</A></P><P>Tao, W., Hall, K., &amp; Masbough, A. (2005). Characterization of Leachate from A Wood Waste File. Water Quality Research Journal Of Canada, 40(4)</P><P>Vancouver landfill 2016 annual report. (2017)</P><P>van Hee, P., Elumbaring, A., van der Lans, R., &amp; Van der Wielen, L. (2006). Selective recovery of polyhydroxyalkanoate inclusion bodies from fermentation broth by dissolved-air flotation. Journal Of Colloid And Interface Science, 297(2), 595-606</P><P>W. Jones, H., W. Fisher, J., D. Delzeit, L., T. Fynn, M., &amp; H. Kliss, M. (2016). Developing the Water Supply System for Travel to Mars. In 46th International Conference on Environmental Systems (pp. 1-15). Moffett Field: NASA Ames Research Center</P><P>Wignarajah, K., Litwiller, E., Fisher, J., &amp; Hogan, J. (2006). Simulated Human Feces for Testing Human Waste Processing Technologies in Space Systems. SAE Technical Paper Series. <A href="http://dx.doi.org/10.4271/2006-01-2180">http://dx.doi.org/10.4271/2006-01-2180</A></P><P>Zhang, Ylikorpi, T., &amp; Pepe, G. (2015). Biomass-based Fuel Cells for Manned Space Exploration Final Report</P></DIV></UL></DIV></UL></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>