<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Calgary_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Calgary/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Header</TITLE></P><DIV class="nav-bar"><DIV id="TopBar"><DIV class="nav" class="nav"><A href="#" class="toggle-menu">☰</A><UL class="nav-list nav-open" id="nav"><LI><A href="https://2017.igem.org/Team:Calgary">Home</A></LI><LI><A href="#" class="toggle-drop">Project</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Description">Problem/Description</A></LI><LI><A href="#" class="toggle-drop2">Synthesis </A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Synthesis">Synthesis Overview</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/BetaOxidation">Beta Oxidation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Glycolysis">Glycolysis</A></LI></UL><LI><A href="https://2017.igem.org/Team:Calgary/Secretion">Secretion</A></LI><LI><A href="#" class="toggle-drop2">Process  </A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Process">Process Overview</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/VFA_Fermentation">VFA Fermentation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/SolidLiquidSeparation">Solid-Liquid Separation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/PHB_Fermentation">PHB Fermentation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Extraction">Extraction</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Products">Products</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Model">Modelling </A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Journal">Notebook</A></LI><LI><A href="#" class="toggle-drop">Team</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Sponsors">Sponsors</A></LI><LI><A href="#" class="toggle-drop">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Human_Practices">HP Summary</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/HP/Silver">HP Silver</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/HP/Gold_Integrated">HP Gold/Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Engagement">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A href="#" class="toggle-drop">Judging</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/MedalRequirements">Medal Requirements</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/InterLab">Interlab</A></LI></DIV></DIV></DIV><DIV id="MainContent"><H1>Results</H1><BUTTON class="accordion"><H3> Synthesis Results</H3></BUTTON><DIV class="panel"><H3> Characterization of pET29B(+)-phaC1J4 </H3><A href="http://parts.igem.org/Part:BBa_K2260001">BBa_K2260001</A><P> The objective of our project was to genetically engineer <I>E. coli</I> to synthesize PHB. We designed a pET29b(+)-phaC1J4 construct to allow our engineered <I>E. coli </I> to employ the beta-oxidation pathway to break down volatile fatty acids (VFAs) and use it to synthesize PHB. We have successfully ligated and transformed this part into <I>E.coli</I>. The figure below shows the digestion confirmation of the vector pET29B(+)-phaC1J4. We have also sequence verified this part.</P><DIV id="Caption"><B>Figure 1:</B> To confirm transformation of the pET29B(+) vector containing phaC1J4 into competent <I>E. coli</I> cells, a double digest confirmation was performed using the restriction enzymes NotI and HindIII. DNA samples were run on 1% agarose gel run at 100 V for 40 minutes.</DIV><P> To confirm the production of protein, we induced gene expression in <I> E. coli</I> BL21(DE3) transformed with pET29B(+)-phaC1J4 with IPTG and performed SDS-PAGE. Our results are shown in the picture below. The SDS-PAGE results indicated that our protein PhaJ4 was being expressed. It was difficult to distinguish PhaC1 amongst the other protein bands. However, because PhaJ4 is downstream of PhaC1, it is likely that PhaC1 was being expressed as well, but further SDS-PAGE runs must be performed for confirmation. Furthermore, our PHB synthesis experiment with <I> E. coli</I> containing pET29B(+)-phaC1J4 produced PHB as shown in Figure 3. The successful synthesis of PHB indicated the functionality of our exogenous proteins.</P><DIV id="Caption"><B>Figure 2:</B> SDS-PAGE of soluble (S) and insoluble (I) proteins obtained from <I>E. coli</I> containing pET29B(+)-phaC1J4 induced with 0.1 mM of IPTG at 37°C. Cells were lysed with lysozyme and sonication to harvest proteins. The gel was run at 30 mA for 50 minutes and stained with Coomassie Blue. </DIV><P> To confirm the ability of <I>E. coli</I> transformed with our pET29B(+)-phaC1J4 construct to produce PHB, we grew O/N cultures for 24 hours and induced them with 0.1 mM of IPTG while supplementing them with chemical media mimicking fermented synthetic poop supernatant (FSPS). Cells were then left to culture for 16 hours. To collect the resulting PHB, we performed sodium hypochlorite extraction on the cultures, which can be found on the <A href="https://2017.igem.org/Team:Calgary/Experiments">Experiments</A> page.</P><DIV id="Caption"><B>Figure 3:</B> PHB extracted from <I>phaC1J4</I>-expressing cells cultured for 24 hours and supplemented with FSPS for 16 hours. PHB was extracted using TritonX-100, sodium hypochlorite, and ethanol in a series of washes and incubation. The negative control tube contained <I>E. coli</I> transformed with the pET29B(+) vector (without insert).</DIV><P>To confirm that the identity of the white powder made by the transformed <I>E. coli</I> was actually our desired product of PHB, we ran our results alongside a sample of pure industrial-grade PHB provided to use by PolyFerm by dissolving PHB in sulphuric acid, which can be detected via HPLC as crotonic acid. The analysis confirmed the identity of our PHB and its similarity to industrial-grade material because the presence of crotonic acid was indicated by the peaks in the spectrum below.

</P><DIV id="Caption"><B>Figure 4:</B> HPLC spectra of pure PHB powder from PolyFerm (left) and PHB powder produced from our phaC1J4-expressing bacteria (right) digested with crotonic acid.</DIV><P><B>Detailed results of these experiments can be found on this part's <A href="http://parts.igem.org/Part:BBa_K2260001">Registry</A> page.</B></P><H3> Characterization of pET29B(+)-phaCBA </H3><A href="http://parts.igem.org/Part:BBa_K2260000">BBa_K2260000</A><P> As part of our project, we also used a pathway that relied on acetyl-CoA, a metabolite from the breakdown of glucose and some VFAs. We designed a pET29B(+)-phaCBA construct so that the <I>E. coli </I> could convert acetyl-CoA to PHB. We have successfully ligated and transformed this part into <I>E. coli</I>. The figure below shows the digestion confirmation of the vector pET29B(+)-phaCBA. We have also sequence verified this part.</P><DIV id="Caption"><B>Figure 5:</B> To confirm transformation of the pET29B(+) vector containing phaCBA into competent <I>E. coli</I>, a double digest confirmation was performed using the restriction enzymes NotI and KpnI. DNA samples were run on 1% agarose gel.</DIV><P> We then cultured these transformed cells following the same method described above to product and extract PHB.</P><DIV id="Caption"><B>Figure 6:</B> PHB extracted from <I>phaCBA</I>-expressing cells cultured for 24 hours and inoculated with FSPS for 16 hours. PHB was extracted using TritonX-100, sodium hypochlorite, and ethanol in a series of washes and incubation. The negative control tube contained <I>E. coli</I> transformed with the pET29b(+) vector (without insert).</DIV><P>Again, To confirm that the identity of the white powder made by the transformed <I>E. coli</I> was PHB, we ran our results alongside a sample of pure industrial-grade PHB provided to use by PolyFerm by dissolving it in sulfuric acid. This leads to the production of crotonic acid peaks after HPLC analysis. The analysis confirmed the identity of our PHB, as the presence of crotonic acid was indicated by the peaks.</P><DIV id="Caption"><B>Figure 7:</B> HPLC analysis of pure PHB powders from PolyFerm (left) and PHB powders produced from <I>phaCBA</I>-expressing bacteria (right) digested with crotonic acid.</DIV><B>Detailed results can be found on this part's <A href="http://parts.igem.org/Part:BBa_K2260000">Registry</A> page.</B></DIV><BUTTON class="accordion"><H3> Secretion Results</H3></BUTTON><DIV class="panel"><H3>Characterization of pSB1C3-Phasin-HlyA Tag</H3><A href="http://parts.igem.org/Part:BBa_K2260002">BBa_K2260002</A><P>In order to secrete PHB, we engineered the fusion protein phasin-HlyA tag first into <I>E. coli</I> DH5α for plasmid propagation then into <I>E. coli</I> BL21(DE3) for IPTG-inducible protein expression. Figure 8 shows the successful digestion of the phasin-HlyA tag construct with EcoRI and SpeI ligated in the pSB1C3 backbone.</P><DIV id="Caption"><B>Figure 8: </B>Screening results of <I>E. coli</I> transformed with pSB1C3-phasin-HlyA tag. The plasmid was double digested with EcoRI and SpeI (DD) then run on a 1% agarose gel at 100 V for 30 minutes. The molecular ladder (L) is visible on the far left and the expected band sizes, obtained from Benchling Virtual Digest, are visible in the diagram on the right. Undigested plasmid (U) was used as a control and the double digests (which had successfully received our part) are visible in lanes 3 and 4.</DIV><P>To confirm the functionality of the phasin-HlyA protein, a &quot;super&quot; plasmid that contains both PHB-producing and PHB-secreting genes was made. This plasmid was created by ligating the phasin-HlyA construct into pSB1C3-phaCAB. <I>PhaCAB</I> is a PHB-producing BioBrick from <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1149052">Imperial College, 2013</A>). This plasmid was transformed into <I>E. coli</I> BL21(DE3). The successful ligation of the &quot;super&quot; plasmid was confirmed with NotI restriction digest, as shown in Figure 9.</P><DIV id="Caption"><B>Figure 9: </B>Screening results of 4 colonies of <I>E. coli</I> transformed with pSB1C3-phaCAB-Phasin-HlyA. Plasmid from the colonies were digested with NotI (D) then run on a 1% agarose gel at 100 V for 30 minuntes. The molecular ladder (L) is visible on the far left and the expected band sizes, obtained from Benchling Virtual Digest, are visible in the diagram on the right. Undigested plasmid (U) was used as a control and the digests from colony 6 are visible in lanes 6-7.</DIV><P>For a secretion assay, cultures of <I>E. coli</I> BL21(DE3) containing the &quot;super&quot; plasmid were inoculated in LB media + chloramphenicol + 3 % glucose. pSB1C3-pPhaCAB without the phasin-HlyA construct in <I>E. coli</I> BL21(DE3) was used as a control, and the experiment was run in triplicates. The bacteria were induced with IPTG, then incubated at 37°C for either 24 hours or 48 hours. After that time, the samples were separated into intracellular and secreted PHB fractions. CaCl<SUB>2</SUB> was added to promote PHB agglomeration in the secreted fraction so that it could be separated from cells via differential centrifugation. The amount of PHB collected in each fraction was measured and those results are presented below (Table 1).</P><P>The addition of CaCl<SUB>2</SUB> to the secreted fractions results in a pellet being obtained that contains both PHB and  CaCl<SUB>2</SUB>. To account for this, the mass of secreted PHB was corrected for the mass of CaCl<SUB>2</SUB> added (0.05549 g). The corrected results are presented in Table 2.</P><P>The assay showed that cells with PHB-secreting genes secreted more PHB than control cells which did not contain these genes, but this difference only became apparent after 48 hours of incubation at 37°C. At 24 hours, there was no meaningful difference between the amounts of PHB in the media of both <I>E. coli</I> variants (0.12851 g for the control and 0.12417 for the positive PHB-secreting samples). At 48 h, control cells had only 0.08051 g of PHB in the media, but the PHB-secreting cells secreted 0.17251 g. This is a 114% increase in the amount of PHB secreted by cells with Phasin-HlyA Tag compared to the control. This indicates that our part, Phasin-HlyA Tag, improves secretion of PHB after an incubation time of at least 24 hours.</P><DIV id="Caption"><B>Table 1: </B>Experimental results of secretion assay. (-) is used to denote negative control, BL21(DE3) with pSB1C3-PhaCAB and (pSB1C3-Phasin) is used to denote PHB-secreting strains, BL21(DE3) with pSB1C3-PhaCAB-Phasin-HlyA Tag.</DIV><DIV id="Caption"><B>Table 2: </B>Corrected experimental results of secretion assay. (-) is used to denote negative control, pSB1C3-PhaCAB and (pSB1C3-Phasin) is used to denote PHB-secreting strains, pSB1C3-PhaCAB-Phasin-HlyA Tag. The amount of PHB secreted was corrected to account for mass of the CaCl<SUB>2</SUB> that was added to promote PHB agglomeration.</DIV><DIV id="Caption"><B>Figure 10: </B>Experimental results of secretion assay. (-) is used to denote negative control, pSB1C3-PhaCAB and (pSB1C3-Phasin) is used to denote PHB-secreting strains, pSB1C3-PhaCAB-Phasin-HlyA Tag. There is a 114 % increase in the amount of PHB secretion by (pSB1C3-Phasin) compared to (-) when time = 48 hours (0.17251 g and 0.08051 g of secreted PHB, respectively).</DIV><P><B>These results can also be found on this part's <A href="http://parts.igem.org/Part:BBa_K2260002">Registry</A> page.</B></P></DIV><BUTTON class="accordion"><H3> Process Development Results</H3></BUTTON><DIV class="panel"><H3> Methods for VFA quantification and characterization </H3><P> As mentioned in our <A href="https://2017.igem.org/Team:Calgary/Journal">Notebook</A>, determination of the total VFA concentration in the solution was an important step in the process – knowing how to quantify total VFAs in the solution helped to prove that the fermentation of human feces with naturally-occurring bacteria increases the VFA concentration, as well as it helped to prove VFA presence in both fermented and unfermented synthetic feces. </P><P><B> Titration </B>is commonly employed by wastewater treatment plants to give a rapid estimate of the VFA concentration in sample solution. Toe measure VFA concentration, we performed Simple Titration procedures, which can be found on our <A href="https://2017.igem.org/Team:Calgary/Experiments">Experiments</A> page.  The results in Table 3 indicate that this method tends to give a slight overestimate of the total concentration; however, it can be used for quick estimations and show changes in overall VFA concentration.</P><DIV id="Caption"><B>Table 3: </B>Titration experiments results</DIV><TABLE><TBODY><TR><TH>Experimental Conditions</TH><TH>Trial 1</TH><TH>Trial 2</TH><TH>Trial 3</TH></TR><TR><TD>Actual VFA Conentration (mg/L)</TD><TD>60</TD><TD>60 </TD><TD>60 </TD></TR><TR><TD>Sample volume (mL)</TD><TD> 40</TD><TD>40 </TD><TD>40 </TD></TR><TR><TD>Acid normality</TD><TD> 0.1</TD><TD>0.1 </TD><TD>0.1 </TD></TR><TR><TD><B>Results</B></TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD> Original pH</TD><TD> 6.61</TD><TD>6.6 </TD><TD>6.61 </TD></TR><TR><TD>Volume of acid added to titrate to pH 5 (mL)</TD><TD> 0.53</TD><TD> 0.53</TD><TD> 0.536</TD></TR><TR><TD>Volume of acid added to titrate to pH 4.3 (mL)</TD><TD> 0.745</TD><TD> 0.75</TD><TD> 0.785</TD></TR><TR><TD> Volume of acid added to titrate to pH 4 (mL)</TD><TD> 0.825</TD><TD> 0.830</TD><TD> 0.858</TD></TR><TR><TD> Calculated VFA concentration (mg/L)</TD><TD> 66.1</TD><TD> 67.7</TD><TD> 74.9</TD></TR></TBODY></TABLE><P><B>HPLC</B> is another method commonly employed in laboratory setting for the VFA concentration determination. The advantage of the method is the fact that it provides the concentration of different volatile fatty acids in the solution. </P><P>Table 4 shows VFA concentrations in fermented synthetic feces supernatant as measured by the HPLC. Since the amount of VFAs in the supernatant was unknown, different dilution factors were tested to ensure the VFA concentrations are within the optimal HPLC detection range. </P><DIV id="Caption"><B>Table 4: </B> VFA concentrations in fermented synthetic feces supernatant as measured by the HPLC at different dilutions.</DIV><TABLE><TBODY><TR><TH>Sample Dilution</TH><TH>Acetate (mM)</TH><TH>Propionate (mM)</TH><TH>Butyrate (mM)</TH></TR><TR><TD>Undiluted</TD><TD>108.9</TD><TD>197.55</TD><TD>12.65</TD></TR><TR><TD>1:32</TD><TD>37.15</TD><TD>99.27</TD><TD>8.03</TD></TR><TR><TD>1:128</TD><TD>41.10</TD><TD>109.17</TD><TD>9</TD></TR><TR><TD>1:256</TD><TD>40.53</TD><TD>111.77</TD><TD>11.67</TD></TR></TBODY></TABLE><H3>VFA fermentation results</H3><P>We fermented synthetic feces to mimic the fermentation by natural gut flora that produce VFAs in the supernatant. In our process, these VFAs can later be used as a feedstock for our PHB-producing <I>E. coli</I>. First, we determined VFA production by leaving synthetic feces to ferment at 22 and 37°C. This experiment was performed before the Synthesis team had successfully created our PHB-producing parts, so we borrowed those of other teams available in the iGEM Registry (from <A href="http://parts.igem.org/wiki/index.php?title=Part%3ABBa_K1149051">Imperial College 2013</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part%3ABBa_K934001 ">Tokyo 2012</A>). The first VFA fermentation experiment showed higher VFA production at 37°C than at 22°C (Figure 11). The control condition was <I>E. coli </I> BL21(DE3) transformed with pET29B(+) vector without any inserts (non-PHB producing), &quot;Imperial condition&quot; was PHB-producing <I>E. coli</I> transformed with a PHB-producing part from <A href="http://parts.igem.org/wiki/index.php?title=Part%3ABBa_K1149051">Imperial College 2013</A>, and &quot;Tokyo condition&quot; was PHB-producing <I>E. coli</I> transformed with a PHB-producing part from <A href="http://partsregistry.org/wiki/index.php?title=Part%3ABBa_K934001 ">Tokyo 2012</A>. The reported results are an average of 3 titrations performed for each sample.</P><DIV id="Caption"><B>Figure 11: </B>VFA fermentation results for the first experiment.</DIV><P> We performed a second VFA fermentation experiment confirmed higher VFA production at 37°C than at 22°C (Figure 12). In addition to a 3-day fermentation (denoted as D3), a 5-day fermentation was also introduced during this experiment (denoted as D5). Higher VFA concentrations were observed after 5 days of fermentation. Similarly to the first experiment, the reported results are an average of 3 titrations performed for each sample.</P><DIV id="Caption"><B>Figure 12: </B>VFA fermentation results for the second experiment.</DIV><P>In subsequent experiments, the supernatant from fermented synthetic feces was collected after the VFA fermentation experiments, sterilized, and cultured with PHB-producing bacteria for 2 – 3 days at 37°C. PHB-producing bacteria cultured in the supernatant from synthetic feces fermented at 37°C had little to no growth, resulting in little to no PHB produced. 
Although the supernatant from synthetic feces fermented at 37°C had higher VFA concentration (Figure 11, Figure 12), we believe the low pH of these samples prohibited bacterial growth, resulting in no PHB production. Consequently, the synthetic feces supernatant fermented at 22°C was selected as preferred VFA fermentation temperature due to optimal pH for bacterial growth, although it had a lower VFA concentration.</P><H3>Liquid-solid separation results </H3><P>The first experiments we performed for solid-liquid separation of synthetic feces from supernatant were Gravity-Driven Sedimentation and Gravity-Driven Filtration; both protocols can be found on the <A href="https://2017.igem.org/Team:Calgary/Experiments">Experiments</A> page. Both methods rely on gravity to settle solids in a phase separate from that of the liquid. The results of these experiments are summarized in Table 5 below:

</P><DIV id="Caption"><B>Table 5: </B> Gravity driven filtration and sedimentation results</DIV><TABLE style="height: 400px;" width="528"><TBODY><TR style="height: 92px;"><TH style="height: 92px; width: 151px;"><P>Gravity driven filtration</P></TH><TH style="height: 92px; width: 138px;"><P>Weight of water present in sample (g)</P></TH><TH style="height: 92px; width: 135px;"><P>Weight of liquid recovered after 24 hours (g)</P></TH><TH style="height: 92px; width: 104px;"><P>Percent of liquid recovered (%)</P></TH></TR><TR style="height: 32px;"><TD style="height: 32px; width: 151px;"><P>Sample 1</P></TD><TD style="height: 32px; width: 138px;"><P>15</P></TD><TD style="height: 32px; width: 135px;"><P>0</P></TD><TD style="height: 32px; width: 104px;"><P>0.00</P></TD></TR><TR style="height: 32px;"><TD style="height: 32px; width: 151px;"><P>Sample 2</P></TD><TD style="height: 32px; width: 138px;"><P>40</P></TD><TD style="height: 32px; width: 135px;"><P>20.4</P></TD><TD style="height: 32px; width: 104px;"><P>51.00</P></TD></TR><TR style="height: 32px;"><TD style="height: 32px; width: 151px;"><P>Sample 3</P></TD><TD style="height: 32px; width: 138px;"><P>65</P></TD><TD style="height: 32px; width: 135px;"><P>52.5</P></TD><TD style="height: 32px; width: 104px;"><P>80.77</P></TD></TR><TR style="height: 22px;"><TD style="height: 22px; width: 151px;"> </TD><TD style="height: 22px; width: 138px;"> </TD><TD style="height: 22px; width: 135px;"> </TD><TD style="height: 22px; width: 104px;"> </TD></TR><TR style="height: 92px;"><TH style="height: 92px; width: 151px;"><P>Gravity Driven sedimentation</P></TH><TH style="height: 92px; width: 138px;"><P>Weight of water present in sample (g)</P></TH><TH style="height: 92px; width: 135px;"><P>Weight of liquid pipetted out after 24 hours (g)</P></TH><TH style="height: 92px; width: 104px;"><P>Percent of liquid recovered (%)</P></TH></TR><TR style="height: 32px;"><TD style="height: 32px; width: 151px;"><P>Sample 1</P></TD><TD style="height: 32px; width: 138px;"><P>15</P></TD><TD style="height: 32px; width: 135px;"><P>0</P></TD><TD style="height: 32px; width: 104px;"><P>0.00</P></TD></TR><TR style="height: 32px;"><TD style="height: 32px; width: 151px;"><P>Sample 2</P></TD><TD style="height: 32px; width: 138px;"><P>40</P></TD><TD style="height: 32px; width: 135px;"><P>21.4</P></TD><TD style="height: 32px; width: 104px;"><P>53.50</P></TD></TR><TR style="height: 32px;"><TD style="height: 32px; width: 151px;"><P>Sample 3</P></TD><TD style="height: 32px; width: 138px;"><P>65</P></TD><TD style="height: 32px; width: 135px;"><P>47.5</P></TD><TD style="height: 32px; width: 104px;"><P>73.08</P></TD></TR></TBODY></TABLE><P>It is clear the gravity alone did not sufficiently separate solids from liquids; therefore, the Staged Filtration <A href="https://2017.igem.org/Team:Calgary/Experiments">Experiment</A> was conducted using  25 g of synthetic feces (Recipe 2). This method relied on using a series of filters with sequentially smaller pores to separate solids from liquid in multiple steps. The original sample contained 15 g of water, yet only 10% of it was recovered, meaning that a more advanced and power intensive technology has to be considered for this stage of the process.</P><DIV id="Caption"><B>Table 6: </B> Staged filtration experiments results</DIV><TABLE><TBODY><TR><TH><P>Filtration type</P></TH><TH><P>Weight of liquid recovered (g)</P></TH><TH>Liquid lost due to transfer (g)</TH><TH><P>comments</P></TH></TR><TR><TD><P>Strainer</P></TD><TD><P>18.6</P></TD><TD><P>1.1</P></TD><TD><P>Yellow thick liquid went through. Yeast bodies we visible in the filtrate.</P></TD></TR><TR><TD><P>&quot;Paper towel&quot; filter</P></TD><TD><P>13.9</P></TD><TD><P>1.8</P></TD><TD><P>A thick creamy-yellow sludgy layer remained on the filter and could be scraped down. Yeast bodies could still be visible</P></TD></TR><TR><TD><P>Coffee filter</P></TD><TD><P>8.6</P></TD><TD><P>1.5</P></TD><TD><P>Another similar looking creamy-yellow layer was scraped down. The yeast bodies were not visible in the liquid any more</P></TD></TR><TR><TD><P>11 micron filter</P></TD><TD><P>5.8</P></TD><TD><P>1.2</P></TD><TD><P>Had to press very hard on the top of the filter to push the liquid through.</P></TD></TR><TR><TD><P>0.2 micron filter</P></TD><TD><P>1.5</P></TD><TD><P> </P></TD><TD><P>The majority of the liquid was not recovered because the filter got clogged. The recovered liquid had a brown tint, but appeared clear and transparent.</P></TD></TR></TBODY></TABLE><P> Finally, we decided to investigate the efficiency of centrifugal-based extraction methods using the Centrifugation for Solid-Liquid Separation <A href="https://2017.igem.org/Team:Calgary/Experiments">Experiment</A>. When a 50 g undiluted sample of synthetic feces (Recipe 2) was tested, the mass of water recovered was 19.6 g, while the mass of initial water present in the sample was 30 g, meaning that there was 65% recovery of water. This result indicated that centrifugal based solid-liquid separation technology would be the best fit for our application. </P><H3>PHB Extraction Results</H3><H4>Chemical Coagulation Results</H4><P>We demonstrated in the lab that adding calcium chloride and then centrifuging increased the amount of PHB removed from a suspension in water, as CaCl<SUB>2</SUB> assists in PHB agglomeration. We carried out centrifugation at various speeds and then collected absorbance data for the samples to measure relative quantity of PHB in solution. The lower the absorbance reading, the higher the amount of PHB that was removed. </P><DIV id="Caption"><B>Figure 13:</B>From left to right, PHB suspension after centrifugation at 1000 RPM, PHB suspension after centrifugation at 3750 RPM, PHB suspension after addition of 10 mM final concentration of CaCl2 and centrifugation at 3750 RPM.</DIV><P>Details concerning the methods of this experiment can be found <A href="https://2017.igem.org/Team:Calgary/Experiments">here</A>. The absorbance readings and the standard deviations of the various conditions tested are shown in the figure below: </P><DIV id="Caption"><B>Figure 14:</B> Absorbance readings and their standard deviations for the various extraction methods tested.</DIV><P>As expected, centrifugation at higher RPM resulted greater removal of PHB from solution. The addition of CaCl<SUB>2</SUB> increased PHB agglomeration, resulting in additional PHB extraction.

</P><H4>Electrocoagulation</H4><P>We then looked at the possibility of using electrocoagulation to remove PHB from solution. We tested electrocoagulation on microscale PHB particles in suspension in water, nanoscale PHB particles in suspension in water, synthetic feces supernatant, and a 1:1 mixture of microscale PHB suspension and synthetic feces supernatant. </P><DIV id="Caption"><B>Figure 15:</B> From left to right, electrocoagulation set-up using nanoscale PHB suspension in water, electrocoagulation set up using 1:1 mixture of PHB suspension, and synthetic feces supernatant.</DIV><P> From our preliminary experiments with just PHB particles in a water suspension, we were able to demonstrate that PHB particles do settle out via electrocoagulation. After we stopped running electricity through our apparatuses, we were able to observe a layer of PHB at the bottom of the container. However, after a few hours a layer of brown powder settled on top of the white layer of PHB. We hypothesized that this was probably iron(III) hydroxide, formed from the excess iron ions released by the anode that did not bind to the PHB. </P><DIV id="Caption"><B>Figure 16:</B> From left to right, PHB settling via electrocoagulation. Layers of PHB and brown powder settled after electrocoagulation.</DIV><P> From our experiments with synthetic feces supernatant, we found that a layer of brown sludge settled at the bottom each time. Even with the 1:1 mixture there was no discernable layer of PHB within the sludge. We tried washing the sludge with dilute acid to remove the metal salts that might have been present in the sludge. However, we were still unable to separate PHB from the sludge. </P><P> The electrocoagulation experiments led us to conclude that while it is possible to settle out PHB using electrocoagulation, this method was not suitable for our media which contained a number of salts that interfered with the coagulation process and caused the formation of sludge. </P><H3>High Performance Liquid Chromatography (HPLC) Analysis of PHB</H3><P>Crotonic acid is the product of PHB digestion with sulphuric acid, and itcan be detected using HPLC. During the first HPLC run with our PHB samples extracted from cell culture, we didn’t observe crotonic acid peaks as expected. This was likely due to digesting our PHB in sulphuric acid for 3 hours, which was likely too long, and resulted in degradation of crotonic acid in solution. </P><P> For our second run of HPLC trials, we tested two different sulphuric acid digestion times (20 minutes and 30 minutes) and observed peaks for crotonic acid for samples obtained from our <A href="http://parts.igem.org/Part:BBa_K2260000">phaCBA part</A> and <B>confirmed that PHB was present in the our samples</B>. We also ran HPLC trials on PHB produced using <A href="http://parts.igem.org/wiki/index.php?title=Part%3ABBa_K1149051">Imperial College's 2013 BioBrick</A> and observed crotonic acid peaks as well. </P><P>Although we were successful at detecting PHB using HPLC, quantification of PHB using HPLC would require further development. In particular, an optimal digestion time needs to be selected where all of the PHB in the sample is converted to crotonic acid <B>and</B> crotonic acid does not start to degrade. Nonetheless, we attempted to quantify our PHB samples by assuming an 80% conversion of PHB to crotonic acid, a value reported by Karr <I>et al.</I> (1983). However, conversation factors in our HPLC runs ranged from 40% to 60% for the standard PHB samples, resulting in inaccurate dilution factor predictions, and we were not able to quantify our samples. Accurate dilutions are important to ensure the sample concentration is within the HPLC detection range.</P><P> For our third HPLC run, we digested PHB samples from our beta-oxidation <A href="http://parts.igem.org/Part:BBa_K2260001">BioBrick</A> for 15 and 30 minutes and observed clear peaks for crotonic acid. Again, we based our dilution factors on 60% conversion of PHB to crotonic acid. More details about the HPLC results can be found on the <A href="http://parts.igem.org/Part:BBa_K2260001">Beta-Oxidation</A> and <A href="http://parts.igem.org/Part:BBa_K2260000">Glycolysis</A> part pages.</P><H3>Tools from PHB</H3><P>We used commerical PHB, the same type of bioplastic that was produced in the lab by genetically engineered bacteria, to make a 2.5 cm (length) by 1.1 cm (width) wrench (Figure 17). The wrench was made by melting PHB into a wrench-shaped mold and letting the plastic to cool. </P><DIV id="Caption"><B>Figure 17:</B> A wrench molded from commercial PHB, the same type of plastic that was produced by engineered bacteria.</DIV></DIV><DIV class="apa-reference"><H2>Works Cited </H2><P> Karr, D. B., Waters, J. K., &amp; Emerich, D. W. (1983). Analysis of poly-β-hydroxybutyrate in Rhizobium japonicum bacteroids by ion-exclusion high-pressure liquid chromatography and UV detection. Applied and environmental microbiology, 46(6), 1339-1344. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>