<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Calgary_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Calgary/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Header</TITLE></P><DIV class="nav-bar"><DIV id="TopBar"><DIV class="nav" class="nav"><A href="#" class="toggle-menu">☰</A><UL class="nav-list nav-open" id="nav"><LI><A href="https://2017.igem.org/Team:Calgary">Home</A></LI><LI><A href="#" class="toggle-drop">Project</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Description">Problem/Description</A></LI><LI><A href="#" class="toggle-drop2">Synthesis </A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Synthesis">Synthesis Overview</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/BetaOxidation">Beta Oxidation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Glycolysis">Glycolysis</A></LI></UL><LI><A href="https://2017.igem.org/Team:Calgary/Secretion">Secretion</A></LI><LI><A href="#" class="toggle-drop2">Process  </A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Process">Process Overview</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/VFA_Fermentation">VFA Fermentation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/SolidLiquidSeparation">Solid-Liquid Separation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/PHB_Fermentation">PHB Fermentation</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Extraction">Extraction</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Products">Products</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Model">Modelling </A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Journal">Notebook</A></LI><LI><A href="#" class="toggle-drop">Team</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Sponsors">Sponsors</A></LI><LI><A href="#" class="toggle-drop">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Human_Practices">HP Summary</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/HP/Silver">HP Silver</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/HP/Gold_Integrated">HP Gold/Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Engagement">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A href="#" class="toggle-drop">Judging</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/MedalRequirements">Medal Requirements</A></LI><LI><A href="https://2017.igem.org/Team:Calgary/InterLab">Interlab</A></LI></DIV></DIV></DIV><DIV id="MainContent"><H1>Modelling</H1><H2> Overview</H2><P> The goal of the modelling component of our project was to find optimal pathway for maximizing production of PHB in <I> E. coli</I>. Furthermore, the model will be used to simulate the reactions that occur in the bacteria after it is transformed with our <A href="https://2017.igem.org/Team:Calgary/Composite_Part">Constructs</A>. After a detailed analysis and <A href="https://2017.igem.org/Team:Calgary/HP/Gold_Integrated">discussion with experts</A>, the team decided to pursue flux balance analysis (FBA) and kinetic modelling.</P><H2> Analysis of various modelling methods </H2><P> The modelling subgroup carefully considered a number of possible modelling topics for our project. We looked into the pros and cons of each model. We also analyzed the usage of the models by previous iGEM teams and in literature. In order to decide a topic, we graded each of the models based on their impact on our project, integration with lab experiments, risks, and availability of resources. The following is a list of our analysis of each of the modelling topics considered:</P><H3>Modelling Method: Flux balance analysis (FBA) </H3><H4> Questions addressed by model</H4><P><UL><LI> Can use maximization of PHB production as an objective function and determine the optimal steady-state flux distribution, then compare to what we observe in the lab.</LI><LI> Can potentially compare PHB production between unmodified <I>E. coli</I> and <I>E. coli</I>  after genetic modifications</LI><LI> Explore the capabilities and limitations of biochemical networks</LI><LI> Analysis of metabolic network robustness (Raman <I>et al.</I>, 2009)</LI><LI> Analysis of Genome-scale metabolic models</LI></UL></P><H4> Pros</H4><P><UL><LI> Does not require kinetic parameters and can be computed fairly quickly for large networks (Orth <I>et al.</I>, 2009).</LI><LI> Proof of concept that our genetic modifications maximize PHB production from human feces</LI><LI> Can compare model predictions (maximum theoretical yield) to experimental data and potentially troubleshoot the system.</LI></UL></P><H4> Cons</H4><P><UL><LI> The accuracy of the FBA prediction largely depends on the accurate definition of the metabolic network, the various constraints and definition of biologically relevant objective functions.</LI><LI> May not always be accurate because does not account for regulatory effects (activation of enzymes by protein kinases or regulation of gene expression)  (Orth <I>et al.</I>, 2009).</LI></UL></P><H3>Modelling Method: Reaction kinetics of pathways for synthesis of PHB </H3><H4> Questions addressed by model</H4><P><UL><LI> Determine the rate limiting step in the pathway (and implement solutions to improve the rate of that step)</LI><LI> Can potentially provide insight into the impact of gene order </LI></UL></P><H4> Pros</H4><P><UL><LI> Relatively easy to translate metabolic network to mathematical terms (Schallau <I>et al.</I>, 2010) </LI><LI> Can be integrated with the project’s experimental data and used to reiterate model and lab experiments.</LI></UL></P><H4> Cons</H4><P><UL><LI> Difficult to find parameters for many reactions in the pathway.</LI><LI> When analyzing multiple pathways that interact with each other, we may have to assume constant reaction rates for simplicity.</LI></UL></P><H3>Modelling Method: Steady state mass balance to predict amount of PHB</H3><H4> Questions addressed by model</H4><P><UL><LI> Helpful for cost analysis of PHB production</LI><LI> Identify the least efficient parts of the process (in terms of conversion)</LI><LI> Can be combined with cost analysis to figure out the areas of the <A href="https://2017.igem.org/Team:Calgary/Process">Process</A>, which can be modified to improve PHB production while adding the least cost.</LI><LI>Evaluate how much PHB will be produced depending on plant capacity (also combined with cost analysis to suggest the optimal scale at which PHB production is most feasible)</LI></UL></P><H4> Pros</H4><P><UL><LI> Can be integrated with the cost analysis</LI><LI> Can be integrated with <A href="https://2017.igem.org/Team:Calgary/Process">Process</A> development.</LI></UL></P><H4> Cons</H4><P><UL><LI> Cannot be integrated with experimental data.</LI></UL></P><H3>Modelling Method: Translational kinetics to predict effect of codon optimization</H3><P>Different bacteria have different tRNAs that are abundant in their cells. When genes from other species of bacteria are expressed in <I>E. coli</I>, they may have a very inefficient translation efficiency if the available tRNA pools are very different. The impact of the difference can be calculated using tAI (tRNA adaptation index)  (Han <I>et al.</I>, 2010). The tAI for a gene is a measure of the availability of the tRNAs that serve each codon in the gene (Han <I>et al.</I>, 2010). Synonymous codon substitutions can affect translational kinetics, which subsequently affects the final protein structure and function (RegulonDB).</P><H4> Pros</H4><P><UL><LI> Determine increase in translation efficiency after codon optimization.</LI><LI> Determine effect of synonymous codon substitutions on protein folding (synthesis enzymes or membrane proteins involved in secretion).</LI><LI> Further optimize sequence to take into account secondary binding structures etc to optimize mRNA translational efficiency.</LI></UL></P><H4> Cons</H4><P><UL><LI> Cannot be integrated with experimental data.</LI><LI>Molecular Dynamics Simulations can be used to model protein folding.</LI><LI> Many unknown parameters involved.</LI></UL></P><H2> Flux Balance Analysis </H2><P> FBA is a mathematical method for simulating metabolism by calculating steady-state metabolic fluxes for large metabolic networks. The modelling subgroup deemed that flux balance analysis will help us find an optimal pathway for maximizing production of PHB in <I>E. coli </I> BL21 (DE3) after inserting our <A href="https://2017.igem.org/Team:Calgary/Composite_Part">genetic construct</A>. MATLAB has Cobra toolbox that is used for flux balance analysis and visualization of metabolic networks. Thus, the modelling subgroup installed the toolbox and searched for models that contained <I>E. coli’s</I> reactions/metabolic networks. We found that the <I>E. coli</I><A href="http://gcrg.ucsd.edu/Downloads/EcoliCore">core model</A> contained pathways and reactions of our interest. We decided to use this model as a proof of concept because it was not possible to visualize the <A href="http://bigg.ucsd.edu/models/iECD_1391"><I>E. coli</I> BL21 (DE3) model</A> and was computationally expensive due to the presence of large number of reactions (edges) and metabolites (nodes). We loaded the core model and and visualized it using <A href="http://biosystems.lv/index.php/software/paint4net">Paint4Net functions</A>. The reaction nodes are represented as rectangle, metabolite nodes as ellipses, zero rate fluxes as grey arrows, green edges represent positive-rate (forward) fluxes, and blue edges represent negative-rate (backward) fluxes.</P><CENTER><B>Figure 1.</B> Visual representation of <I>E. coli</I><A href="http://gcrg.ucsd.edu/Downloads/EcoliCore">core model</A> with optimized fluxes for the bacteria’s biomass.</CENTER><P>After reading the model containing metabolic networks, the toolbox has ‘optimizeCbModel’ function that calculates steady-state flux for the metabolic network. The objective of the function can be changed to a different metabolite. In our case, the objective was changed to PHB because we wanted to find the optimal pathway for PHB synthesis after inserting our <A href="http://parts.igem.org/Part:BBa_K2260000">construct</A>.</P><H3> Results</H3><P> The first step for FBA analysis was to insert the genes that utilize products of <A href="https://2017.igem.org/Team:Calgary/Glycolysis">glycolysis</A> pathway (<A href="http://parts.igem.org/Part:BBa_K2260000">phaC, phaB, and phaA</A>) and <A href="https://2017.igem.org/Team:Calgary/BetaOxidation">beta-oxidation</A> pathway (<A href="http://parts.igem.org/Part:BBa_K2260001">phaC1, phaJ4</A>) into the <I>E. coli</I><A href="http://gcrg.ucsd.edu/Downloads/EcoliCore">core model</A>. The team was able to add the phaCBA genes into the<A href="http://gcrg.ucsd.edu/Downloads/EcoliCore">core model</A>. The model was visualized to look for any changes in the pathway.</P><CENTER><B>Figure 2.</B> Reaction nodes for phaB and phaC. The red rectangles show that the reactions are dead ends. </CENTER><P> We noticed that our model showed that the new reactions as a result of adding phaCBA were ‘dead ends’. The result of optimizeCbModel was 0, which means there was no steady-state flux for the objective selected i.e. optimal pathway for PHB. The product (PHB) was shown as a dead-end metabolite. In order to solve this problem, we found that we needed to add exchange or transport reactions for the removal or <A href="https://2017.igem.org/Team:Calgary/Secretion">secretion of PHB</A> from the cell. This will allow the cell to <A href="https://2017.igem.org/Team:Calgary/Secretion">secrete</A> and <A href="https://2017.igem.org/Team:Calgary/Synthesis">synthesize PHB</A> at steady-state fluxes. </P><CENTER><B>Figure 3.</B> PHB shown as a dead end product after being synthesized in <I>E. coli</I>. </CENTER><H3> Future Directions</H3><P>In order to get conclusive results for finding optimal pathway(s) for the synthesis of PHB at steady-state fluxes, it is necessary to introduce a secretion system into the model. Thus, the solution will be to use <A href="https://opencobra.github.io/cobratoolbox/latest/modules/fastGapFill.html">fastGapFill</A>, which is used to find missing reactions and metabolites from the universal biochemical reaction database (e.g. Kyoto Encyclopedia of Genes and Genomes) for a given (compartmentalized) metabolic reconstruction. After addition of the transport reactions, the model will have to be validated before it can be used to inform lab experiments.</P><H2> Kinetic Modelling </H2><P>We began developing a kinetic model for PHB synthesis via beta-oxidation. We hoped to identify the rate-limiting steps in our reaction pathways and thereby identify whether upregulation of fadD and fade genes would increase the rate of reaction for the PHB-synthesis pathway via β-oxidation. We used Micheles-Menten enzyme kinetics to model the reactions of the beta-oxidation pathway. However, we ran into a number of challenges including lack of reliable data for β-oxidation of fatty acids in E.coli as well as how complex the model was becoming.</P><H3>Methods</H3><P>We used the SymBiology toolbox in MATLAB to build the model and used kinetic parameters obtained from literature or online databases.</P><DIV id="Caption"><B>Figure 4:</B> Visual Representation of beta oxidation reaction pathways</DIV><H3>Results</H3><P>Our online search from literature, databases, and pre-existing models revealed that there was not much information on the kinetics of many enzymes and pathways for PHB-synthesis via β-oxidation pathway. Therefore, we were unable to obtain informative results even when different organisms' kinetic parameters were used to substitute values for key enzyme reactions. While building the model, we also learned that the <I>fadD</I> and <I>fadE</I> genes were most active for long chain fatty acids and therefore we built our model based on using oleic acid as the substrate to test the activity of these genes for metabolism of long-chain fatty acids in human fecal waste. However, due to time restraints and lack of accurate kinetic parameters in literature and databases we decided not to pursue kinetic modelling; the information generated would not be applicable for informing our experiment given current circumstances. </P><H3> Future Directions</H3><P>In the future we hope to test our phaJC, fadE, and fadD constructs in the lab to obtain data of enzyme kinetics which we can use to adjust our model's kinetic parameters to obtain accurate results.</P><DIV class="apa-reference"><H2>Works Cited</H2><P>Raman, K., and N. Chandra. &quot;Flux Balance Analysis Of Biological Systems: Applications And Challenges&quot;. Briefings in Bioinformatics 10.4 (2009): 435-449. Web.</P><P>Orth, Jeffrey D, Ines Thiele, and Bernhard Ø Palsson. &quot;What Is Flux Balance Analysis?&quot;. Nature Biotechnology 28.3 (2010): 245-248. Web. </P><P>Schallau, K., Junker, B. “Simulating Plant Metabolic Pathways with Enzyme-Kinetic Models”. Plant Physiology Vol.152 (2010): 1763-1771. Web.</P><P>Han N. Lim1, Yeong Lee, and Razika Hussein. “Fundamental relationship between operon organization and gene expression”. Proceedings of the National Academy of Sciences of the United States Vol. 108 (2011): 10626-10631. Web.</P><P>RegulonDB version 9.0: high-level integration of gene regulation, coexpression, motif clustering and beyond. 
Gama-Castro S, Salgado H, Santos-Zavaleta A, Ledezma-Tejeida D, Muñiz-Rascado L, García-Sotelo JS, Alquicira-Hernández K, Martínez-Flores I, Pannier L, Castro-Mondragón JA, Medina-Rivera A, Solano-Lira H, Bonavides-Martínez C, Pérez-Rueda E, Alquicira-Hernández S, Porrón-Sotelo L, López-Fuentes A, Hernández-Koutoucheva A, Moral-Chávez VD, Rinaldi F, Collado-Vides J. 
Nucleic Acids Res. 2016 Jan 4;44(D1):D133-43. doi: 10.1093/nar/gkv1156. Epub 2015 Nov 2.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>