<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Marburg_PEG_Method_Outlook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Marburg/PEG Method/Outlook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Marburg"><B>iGEM Marburg 2016</B></A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Projects">Projects</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/HP/Silver">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Notebooks">Notebooks</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Achievements">Achievements</A></LI></UL></DIV><DIV class="container"><DIV class="master-heading"><H1>
            SynDustry <SMALL>Fuse. Produce. Use.</SMALL></H1></DIV></DIV><DIV class="container"><DIV class="heading"><H2><B>Fusion of microorganisms</B></H2></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-sm-4 text-center"><A href="https://2016.igem.org/Team:Marburg/PEG_Method/Results" class="button" role="button">« Results  </A></DIV><DIV class="col-sm-4 text-center"><A href="#" class="button active" role="button">Outlook</A></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
One of the most interesting questions, the quantity of bacteria that is incorporated into yeast spheroplasts, still remains unanswered. We tried flow cytometry after applying the PEG protocol but no differences between co-culture and actual endosymbiosis could be obtained. The most obvious reason for this was a conglomeration due to the adhesive properties of PEG - especially at higher molecular weights cells showed to form midsize to large aggregates. Due to the fact that subsequent dilution of PEG always goes along with cell lysis, we will try to solve this issue in another way.
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
An implemented and functional dependency could be the solution for this. The ability of the cells to live with the endosymbiont over a longer time would lead to increased stability, which would then enable us to perform several washing steps for elimination of PEG. Flow cytometry without cell aggregates would then be possible, which would lead to a direct quantification of the uptake efficiency, even though it does not allow conclusions regarding the uptake efficiency right after the application of the PEG protocol. Another approach towards better understanding of our project would be a proteomic analysis of the organism's metabolic behaviour after invasion or being invaded, respectively. Through this we could obtain valuable information about a further optimization of the PEG protocol in terms of knocking out or overexpressing genes, which have a big influence to our system. Our first approach towards this was trying to knock out the <I>atg1</I> gene in yeast, which is the essential gene for yeast autophagy. A knockout should decrease of lysis rate of <I>E. coli</I> and therefore increase the chance of successful establishment.

            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
Furthermore, it may be interesting to study the behaviour of the fused cells in terms of gene transfer. Again, this would require a stable dependency, an ensured and synchronized division and more insight into the influences of the uptake to the cell's metabolic response. If these factors are better understood an approach towards isolation and growth of endosymbiotic cells could be made. If these cells could be grown in culture over a longer time period, gene transfer is more likely to occur. Sequencing of such cells could reveal interesting details about the principles of endosymbiosis in evolutionary terms.

            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
Last, we want to try the possibility of using different organisms as the host chassis. Our system was designed to work for different yeast strains, therefore this could be tested by simply adapting the buffer conditions with respect to the new host organism’s individual requirements.

            </P></DIV></DIV></DIV><DIV class="container"><FOOTER><P class="pull-right"><A href="#">Back to top</A></P><P>© 2016 iGEM Marburg ·<A href="mailto:igemteam@synmikro.uni-marburg.de">Contact</A></P></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>