<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Marburg_Dependency skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Marburg/Dependency</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Marburg"><B>iGEM Marburg 2016</B></A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Projects">Projects</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/HP/Silver">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Notebooks">Notebooks</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Achievements">Achievements</A></LI></UL></DIV><DIV class="container"><DIV class="master-heading"><H1>
            SynDustry <SMALL>Fuse. Produce. Use.</SMALL></H1></DIV></DIV><DIV class="container"><DIV class="heading"><H2><B>Establishing a dependency among different organisms</B></H2></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-sm-4 text-center"><A href="#" class="button active" role="button">Description</A></DIV><DIV class="col-sm-4 text-center"><A href="https://2016.igem.org/Team:Marburg/Dependency/Results" class="button" role="button">  Results »</A></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
                Using different organisms for our artificial endosymbiosis results in enormous stress for the
                organism to deal with. The organisms have to survive suboptimal conditions, since they are
                perfectly adapted to an unicellular lifestyle. Our first experiments showed that <I>S. cerevisiae</I>
                lyse invading <I>E. coli</I>. Since killing the invader seems to be more feasible than living in
                symbiosis, we needed a reliable interaction to increase the fitness of the participating organisms.
                To overcome this issue we established a dependency between the symbiont and its host to guarantee
                the survival of the <I>E. coli</I> cells. 
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
                Possible dependencies could be based on well known methods in molecular biology. For example
                active antibiotic resistance or dependencies based on auxotrophic markers. Another possible
                approach, metabolic dependencies based on substrate exchange, is commonly used in co-culture
                and due to various similarities more suitable for our purpose, which is why we decided to choose
                this approach. For our project is was crucial to develop a system implementing a dependency
                based on a compound that on the one hand can be produced and secreted by <I>E. coli</I> and on
                the other hand is essential for yeast`s viability. A malonate based dependency is a promising
                approach to achieve both aims (Fig.1). 
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12"><DIV class="figure_text"><B>Figure 1. Scheme of the interaction between the hosting <I>S. cerevisisae</I> cell
                and the invading <I>E. coli</I> cell. </B> Due to a knockout of the acc1 gene of
                <I>S. cerevisiae</I> the cells are no longer able to produce malonyl-CoA which is essential
                for fatty acid production and therefore a major player for yeast`s viability. The introduction
                of the <I>matB </I> gene from <I>Rhizobium leguminosarium</I> leads to an alternative pathway
                for malonyl-CoA production based on malonate, which is delivered by the invading <I>E. coli</I>
                cell. We introduced several genes from various organisms, including <I>E. coli</I>, and
                overexpressed them to channel the flux into the beta alanine pathway towards the malonic
                acid production. To achieve this we designed an operon plasmid consisting of <I>ppc</I>
                (<I>E. coli</I>),<I> aspA</I> (<I>E. coli</I>), <I>panD</I> (<I>C. glutamicum</I>),
                <I>pa0132</I> (<I>P. aeroginosa</I>), <I>yneI</I> (<I>E. coli</I>) and the <I>mae1</I> gene
                of <I>S. pombe</I> which encodes for a permease that enables the ability to secrete the
                produced malonic acid. 
            </DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
                We created a knockout mutant yeast strain lacking the <I>acc1</I> gene, coding for an
                acetyl-CoA-carboxylase. This protein catalyzes the irreversible carboxylation of acetyl-CoA
                to produce malonyl-CoA <A class="ref" href="#ref_1">[1]</A>, which is involved in fatty acid production and  
                thus, is an
                essential player in yeast viability <A class="ref" href="#ref_2">[2]</A>. To achieve this we created a  
                knockout-construct
                consisting of the marker-gene <I>kanMX</I>, which is flanked by a 200bp fragment-homologue
                to the upstream area of the wild type <I>acc1</I> gene in <I>S. cerevisiae</I> and a 200bp
                fragment homologue to the downstream area of the wildtype <I>acc1</I> gene in <I>S. cerevisisae</I>.
                The construct was integrated into a pUC19 backbone with an ampicillin resistance marker gene.
                To overcome the absence of malonyl-CoA in our yeast strain we transformed the for <I>E. coli</I>
                codon optimized version of <I>matB</I> gene from <I>Rhizobium leguminosarium</I> bv. <I>trifolii</I>,
                which encodes for a malonyl-CoA synthetase: an enzyme that catalyzes the biosynthesis of malonyl-CoA
                from malonate <A class="ref" href="#ref_3">[3]</A>. In addition we transformed the <I>mae1</I> gene of <I>S. 
                pombe</I> , which
                encodes for a permease that allows the uptake of malonic acid <A class="ref" href="#ref_4">[4]</A>. Both the <I>mae1</I> and the
                <I>matB</I> gene were integrated into the pNK26 backbone (Fig. 2). It carries the ampicillin
                resistance marker gene for amplification in <I>E. coli</I> and the <I>trp1</I> marker gene, that
                encodes for a phosphoribosylanthranilate isomerase that catalyzes the third step in tryptophan
                biosynthesis and therefore can be used as an auxotrophic marker in yeast. Additionally, it carries
                a bidirectional promoter region with two constitutive promoters (pPGK1 and pTEF1). The gene
                <I>matB</I> was inserted under the control of the pPKG1 promoter and <I>mae1</I> was inserted
                under the control of pTEF1 promoter. With access to the malonyl-CoA synthetase we created an
                alternative pathway for malonyl-CoA production based on the uptake of external malonate.
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12"><DIV class="figure_text"><B>Figure 2. Plasmid map of the expression plasmid in <I>S. cerevisisae</I>.</B>
                The plasmid consists of a bidirectional promoter region that regulates the expression
                of the <I>mae1</I> gene and the <I>matB</I> gene. The fragments were inserted into the
                pNK26 backbone that carries an ampicillin resistance for selection in <I>E. coli</I> and
                a Trp1 cassette for selection in <I>S. cerevisiae</I>. The plasmid was assembled via Gibson
                assembly reaction.
            </DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
If malonate, which is essential for yeast viability and as it serves as fuel for the biosynthesis reaction is removed from the media it has to be delivered by the invading <I>E. coli</I> strain. To achieve this major changes in the beta-alanine route of <I>E. coli</I> had to be made to direct flux towards malonic acid production <A class="ref" href="#ref_3">[3]</A>. We designed an operon-plasmid consisting of six different genes from various organisms including genes from <I>E. coli</I>, which was overexpressed to increase the yield of produced malonic acid (Fig. 3). As backbone we used the part of the pACYC184 plasmid that consists of the <I>E. coli</I> origin p15A ori and the chloramphenicol resistance gene under its native cat promoter as the marker-gene. The operon itself was under the control of the constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23108">BBa_J23108</A> and consists of the <I>ppc</I> gene from <I>E. coli</I> that encodes for a phosphoenolpyruvate carboxylase that catalyzes the addition of bicarbonate to phosphoenolpyruvate (PEP) to form oxaloacetate <A class="ref" href="#ref_6">[6]</A>, the <I>yneI</I> gene from <I>E. coli</I> that encodes for a succinic semialdehyde dehydrogenase that should catalyze the oxidation of malonic semialdehyde to malonic acid <A class="ref" href="#ref_3">[3]</A> and the <I>aspA</I> gene from <I>E. coli</I> that encodes for an aspartate ammonia lyase that catalyzes the reaction of fumaric acid to aspartic acid <A class="ref" href="#ref_7">[7]</A>. Additionally, we integrated the <I>panD</I> gene from <I>C. glutamicum</I>, which encodes for an aspartate-α-decarboxylase that catalyzes the reaction of aspartic acid to β-alanine <A class="ref" href="#ref_3">[3]</A> and the <I>pa0132</I> gene from <I>P. aeroginosa</I> that encodes for a β-alanine pyruvate transaminase that catalyzes the reaction of  β-alanine to malonic semialdehyde <A class="ref" href="#ref_3">[3]</A>. Last but not least, we integrated the <I>mae1</I> gene from <I>S. pombe</I> that encodes for a permease for malate and other C<SUB>4</SUB> dicarboxylic acids <A class="ref" href="#ref_4">[4]</A>, that should enable the strain to segregate malonic acid <A class="ref" href="#ref_5">[5]</A>. The first three genes of the operon (<I>panD</I>, <I>aspA</I> and <I>pa0132</I>) were regulated by the RBS <A href="http://parts.igem.org/Part:BBa_J61101">BBa_J61101 </A> with a related strength of 22.7 %. The last three genes (<I>ppc</I>, <I>yneI</I> and <I>mae1</I>) were under the control of the RBS <A href="http://parts.igem.org/Part:BBa_B0032">BBa_B0032 </A> with a related strength of 33.96 %. As terminator we used the double terminator <A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A> with a forward efficiency of 0.984.
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12"><DIV class="figure_text"><B>Figure 3. Plasmid map of the Operon plasmid in <I>E. coli</I>.</B>
The plasmid consists of the BBa_B0015 terminator and the constitutive promoter BBa_J23108 that controls the expression of <I>panD</I>, encoding for an aspartate-α-decarboxylase, <I>aspA</I>, encoding for aspartase ammonia lyase and <I>pa0132</I>, encoding for a β-alanine pyruvate transaminase, regulated by the BBa_J61101 ribosome binding site. Additionally, <I>ppc</I>, encoding for a phosphoenolpyruvate carboxylase, <I>yneI</I>, encoding for a succinic semialdehyde dehydrogenase and <I>mae1</I>, encoding for a permease, regulated by the BBa_B0032 ribosome binding site. The operon was inserted into a pACYC184 backbone carrying a chloramphenicol resistance marker gene and a p15A <I>E. coli</I> origin.
            </DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
Both, the operon plasmid and the expression plasmid were assembled via Gibson assembly. <I>Ppc</I>, <I>aspA</I> and <I>yneI</I> were amplified using genomic DNA from the <I>E. coli</I> strain MG1655. <I>PanD</I>, <I>matB</I> and <I>pa0132</I> were amplified using synthesized nucleotide sequences as templates <I>Mae1</I> was amplified using genomic <I>S. pombe</I> DNA as template.
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
A different approach, to achieve the goal of reliably interacting organisms, was to set a protein-based dependency. In this scenario the yeast has an essential protein-coding gene knocked out, which then is complemented by <I>E. coli</I> expressing and secreting the protein. The targeted genes in yeast can vary from classic knockout genes such as <I>trp</I>, which can be supplemented from the medium to more unique knockouts; for example a subunit of the yeasts ribosome. The expression of the corresponding protein in <I>E. coli</I> should be regulated by an inducible promoter where the inducer has to be able to pass the yeast cell wall and membrane, such as T7 or <I>lac</I>. Additionally, this has the advantage that it can be easily controlled whether a survival of both cells is a result of the dependency or occurred for different reasons e.g. leaving out the induction.
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
For the export we chose two different approaches. The first, was to construct a fusion protein consisting out of the protein of interest and YebF fused to its N-terminal end. YebF occurs in the genome of <I>E. coli</I> and has shown to be capable of exporting fused proteins due to unknown mechanisms <A class="ref" href="#ref_8">[8]</A>. The second approach was similar, yet a little bit more complicated. We fused the 178 bp signal sequence of the Flagellin encoding gene <I>fliC</I> upstream of the coding region of the desired protein. This sequence acts as a signal for the directed transport and assembly into the flagellum. A knockout of FliC and FliD interferes with the regulation of both transport and assembly. Hence, instead of Flagellin our protein of interest is transported directly to the membrane, where it gets secreted into the medium due to defective assembly. In addition, the FliC signal sequence has shown to be cleaved during this procedure <A class="ref" href="#ref_9">[9]</A>.
            </P></DIV></DIV><DIV><H3 class="text-center">Literature</H3><OL class="literature_list"><LI id="ref_1"><A href="#ref_1" class="ref">[1]</A> RW, Brownsey, R. Zhande, and A. N. Boone. &quot;lsoforms of acetyl-CoA carboxylase: structures, regulatory properties and metabolic functions.&quot; (1997).</LI><LI id="ref_2"><A href="#ref_2" class="ref">[2]</A> Tehlivets, Oksana, Kim Scheuringer, and Sepp D. Kohlwein. &quot;Fatty acid synthesis and elongation in yeast.&quot; Biochimica et Biophysica Acta (BBA)-Molecular and Cell Biology of Lipids 1771.3 (2007): 255-270.</LI><LI id="ref_3"><A href="#ref_3" class="ref">[3]</A> Song, Chan Woo, et al. &quot;Metabolic Engineering of Escherichia coli for the Production of 3-Hydroxypropionic Acid and Malonic Acid through β-Alanine Route.&quot; ACS synthetic biology (2016).</LI><LI id="ref_4"><A href="#ref_4" class="ref">[4]</A> Grobler, Jandre, et al. &quot;The mae1 gene of Schizosaccharomyces pombe encodes a permease for malate and other C4 dicarboxylic acids.&quot; Yeast 11.15 (1995): 1485-1491.</LI><LI id="ref_5"><A href="#ref_5" class="ref">[5]</A> Chen, Wei Ning, and Kee Yang Tan. &quot;Malonate uptake and metabolism in Saccharomyces cerevisiae.&quot; Applied biochemistry and biotechnology 171.1 (2013): 44-62.</LI><LI id="ref_6"><A href="#ref_6" class="ref">[6]</A> Kai, Yasushi, Hiroyoshi Matsumura, and Katsura Izui. &quot;Phosphoenolpyruvate carboxylase: three-dimensional structure and molecular mechanisms.&quot;Archives of Biochemistry and Biophysics 414.2 (2003): 170-179.</LI><LI id="ref_7"><A href="#ref_7" class="ref">[7]</A> Song, Chan Woo, et al. &quot;Metabolic engineering of Escherichia coli for the production of 3-aminopropionic acid.&quot; Metabolic engineering 30 (2015): 121-129.</LI><LI id="ref_8"><A href="#ref_8" class="ref">[8]</A> Zhang, Guijin, Stephen Brokx, and Joel H. Weiner. &quot;Extracellular accumulation of recombinant proteins fused to the carrier protein YebF in Escherichia coli.&quot;Nature biotechnology 24.1 (2006): 100-104.</LI><LI id="ref_9"><A href="#ref_9" class="ref">[9]</A> Majander, Katariina, et al. &quot;Extracellular secretion of polypeptides using a modified Escherichia coli flagellar secretion apparatus.&quot; Nature biotechnology23.4 (2005): 475-481.</LI></OL></DIV></DIV><DIV class="container"><FOOTER><P class="pull-right"><A href="#">Back to top</A></P><P>© 2016 iGEM Marburg ·<A href="mailto:igemteam@synmikro.uni-marburg.de">Contact</A></P></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>