Team:Marburg/Dependency
iGEM Marburg 2016
Projects
Human Practices
Notebooks
Safety
Team
Achievements
SynDustry
Fuse. Produce. Use.
Establishing a dependency among different organisms
Description
Results »
Using different organisms for our artificial endosymbiosis results in enormous stress for the                 organism to deal with. The organisms have to survive suboptimal conditions, since they are                 perfectly adapted to an unicellular lifestyle. Our first experiments showed that S. cerevisiae lyse invading E. coli. Since killing the invader seems to be more feasible than living in                 symbiosis, we needed a reliable interaction to increase the fitness of the participating organisms.                 To overcome this issue we established a dependency between the symbiont and its host to guarantee                 the survival of the E. coli cells.
Possible dependencies could be based on well known methods in molecular biology. For example                 active antibiotic resistance or dependencies based on auxotrophic markers. Another possible                 approach, metabolic dependencies based on substrate exchange, is commonly used in co-culture                 and due to various similarities more suitable for our purpose, which is why we decided to choose                 this approach. For our project is was crucial to develop a system implementing a dependency                 based on a compound that on the one hand can be produced and secreted by E. coli and on                 the other hand is essential for yeast`s viability. A malonate based dependency is a promising                 approach to achieve both aims (Fig.1).
Figure 1. Scheme of the interaction between the hosting S. cerevisisae cell                 and the invading E. coli cell. Due to a knockout of the acc1 gene of S. cerevisiae the cells are no longer able to produce malonyl-CoA which is essential                 for fatty acid production and therefore a major player for yeast`s viability. The introduction                 of the matB gene from Rhizobium leguminosarium leads to an alternative pathway                 for malonyl-CoA production based on malonate, which is delivered by the invading E. coli cell. We introduced several genes from various organisms, including E. coli, and                 overexpressed them to channel the flux into the beta alanine pathway towards the malonic                 acid production. To achieve this we designed an operon plasmid consisting of ppc (E. coli), aspA (E. coli), panD (C. glutamicum), pa0132 (P. aeroginosa), yneI (E. coli) and the mae1 gene                 of S. pombe which encodes for a permease that enables the ability to secrete the                 produced malonic acid.
We created a knockout mutant yeast strain lacking the acc1 gene, coding for an                 acetyl-CoA-carboxylase. This protein catalyzes the irreversible carboxylation of acetyl-CoA                 to produce malonyl-CoA [1] , which is involved in fatty acid production and                   thus, is an                 essential player in yeast viability [2] . To achieve this we created a                   knockout-construct                 consisting of the marker-gene kanMX, which is flanked by a 200bp fragment-homologue                 to the upstream area of the wild type acc1 gene in S. cerevisiae and a 200bp                 fragment homologue to the downstream area of the wildtype acc1 gene in S. cerevisisae.                 The construct was integrated into a pUC19 backbone with an ampicillin resistance marker gene.                 To overcome the absence of malonyl-CoA in our yeast strain we transformed the for E. coli codon optimized version of matB gene from Rhizobium leguminosarium bv. trifolii,                 which encodes for a malonyl-CoA synthetase: an enzyme that catalyzes the biosynthesis of malonyl-CoA                 from malonate [3] . In addition we transformed the mae1 gene of S.                  pombe , which                 encodes for a permease that allows the uptake of malonic acid [4] . Both the mae1 and the matB gene were integrated into the pNK26 backbone (Fig. 2). It carries the ampicillin                 resistance marker gene for amplification in E. coli and the trp1 marker gene, that                 encodes for a phosphoribosylanthranilate isomerase that catalyzes the third step in tryptophan                 biosynthesis and therefore can be used as an auxotrophic marker in yeast. Additionally, it carries                 a bidirectional promoter region with two constitutive promoters (pPGK1 and pTEF1). The gene matB was inserted under the control of the pPKG1 promoter and mae1 was inserted                 under the control of pTEF1 promoter. With access to the malonyl-CoA synthetase we created an                 alternative pathway for malonyl-CoA production based on the uptake of external malonate.
Figure 2. Plasmid map of the expression plasmid in S. cerevisisae. The plasmid consists of a bidirectional promoter region that regulates the expression                 of the mae1 gene and the matB gene. The fragments were inserted into the                 pNK26 backbone that carries an ampicillin resistance for selection in E. coli and                 a Trp1 cassette for selection in S. cerevisiae. The plasmid was assembled via Gibson                 assembly reaction.
If malonate, which is essential for yeast viability and as it serves as fuel for the biosynthesis reaction is removed from the media it has to be delivered by the invading E. coli strain. To achieve this major changes in the beta-alanine route of E. coli had to be made to direct flux towards malonic acid production [3] . We designed an operon-plasmid consisting of six different genes from various organisms including genes from E. coli, which was overexpressed to increase the yield of produced malonic acid (Fig. 3). As backbone we used the part of the pACYC184 plasmid that consists of the E. coli origin p15A ori and the chloramphenicol resistance gene under its native cat promoter as the marker-gene. The operon itself was under the control of the constitutive promoter BBa_J23108 and consists of the ppc gene from E. coli that encodes for a phosphoenolpyruvate carboxylase that catalyzes the addition of bicarbonate to phosphoenolpyruvate (PEP) to form oxaloacetate [6] , the yneI gene from E. coli that encodes for a succinic semialdehyde dehydrogenase that should catalyze the oxidation of malonic semialdehyde to malonic acid [3] and the aspA gene from E. coli that encodes for an aspartate ammonia lyase that catalyzes the reaction of fumaric acid to aspartic acid [7] . Additionally, we integrated the panD gene from C. glutamicum, which encodes for an aspartate-α-decarboxylase that catalyzes the reaction of aspartic acid to β-alanine [3] and the pa0132 gene from P. aeroginosa that encodes for a β-alanine pyruvate transaminase that catalyzes the reaction of  β-alanine to malonic semialdehyde [3] . Last but not least, we integrated the mae1 gene from S. pombe that encodes for a permease for malate and other C4 dicarboxylic acids [4] , that should enable the strain to segregate malonic acid [5] . The first three genes of the operon (panD, aspA and pa0132) were regulated by the RBS BBa_J61101 with a related strength of 22.7 %. The last three genes (ppc, yneI and mae1) were under the control of the RBS BBa_B0032 with a related strength of 33.96 %. As terminator we used the double terminator BBa_B0015 with a forward efficiency of 0.984.
Figure 3. Plasmid map of the Operon plasmid in E. coli. The plasmid consists of the BBa_B0015 terminator and the constitutive promoter BBa_J23108 that controls the expression of panD, encoding for an aspartate-α-decarboxylase, aspA, encoding for aspartase ammonia lyase and pa0132, encoding for a β-alanine pyruvate transaminase, regulated by the BBa_J61101 ribosome binding site. Additionally, ppc, encoding for a phosphoenolpyruvate carboxylase, yneI, encoding for a succinic semialdehyde dehydrogenase and mae1, encoding for a permease, regulated by the BBa_B0032 ribosome binding site. The operon was inserted into a pACYC184 backbone carrying a chloramphenicol resistance marker gene and a p15A E. coli origin.
Both, the operon plasmid and the expression plasmid were assembled via Gibson assembly. Ppc, aspA and yneI were amplified using genomic DNA from the E. coli strain MG1655. PanD, matB and pa0132 were amplified using synthesized nucleotide sequences as templates Mae1 was amplified using genomic S. pombe DNA as template.
A different approach, to achieve the goal of reliably interacting organisms, was to set a protein-based dependency. In this scenario the yeast has an essential protein-coding gene knocked out, which then is complemented by E. coli expressing and secreting the protein. The targeted genes in yeast can vary from classic knockout genes such as trp, which can be supplemented from the medium to more unique knockouts; for example a subunit of the yeasts ribosome. The expression of the corresponding protein in E. coli should be regulated by an inducible promoter where the inducer has to be able to pass the yeast cell wall and membrane, such as T7 or lac. Additionally, this has the advantage that it can be easily controlled whether a survival of both cells is a result of the dependency or occurred for different reasons e.g. leaving out the induction.
For the export we chose two different approaches. The first, was to construct a fusion protein consisting out of the protein of interest and YebF fused to its N-terminal end. YebF occurs in the genome of E. coli and has shown to be capable of exporting fused proteins due to unknown mechanisms [8] . The second approach was similar, yet a little bit more complicated. We fused the 178 bp signal sequence of the Flagellin encoding gene fliC upstream of the coding region of the desired protein. This sequence acts as a signal for the directed transport and assembly into the flagellum. A knockout of FliC and FliD interferes with the regulation of both transport and assembly. Hence, instead of Flagellin our protein of interest is transported directly to the membrane, where it gets secreted into the medium due to defective assembly. In addition, the FliC signal sequence has shown to be cleaved during this procedure [9] .
Literature
[1] RW, Brownsey, R. Zhande, and A. N. Boone. "lsoforms of acetyl-CoA carboxylase: structures, regulatory properties and metabolic functions." (1997).
[2] Tehlivets, Oksana, Kim Scheuringer, and Sepp D. Kohlwein. "Fatty acid synthesis and elongation in yeast." Biochimica et Biophysica Acta (BBA)-Molecular and Cell Biology of Lipids 1771.3 (2007): 255-270.
[3] Song, Chan Woo, et al. "Metabolic Engineering of Escherichia coli for the Production of 3-Hydroxypropionic Acid and Malonic Acid through β-Alanine Route." ACS synthetic biology (2016).
[4] Grobler, Jandre, et al. "The mae1 gene of Schizosaccharomyces pombe encodes a permease for malate and other C4 dicarboxylic acids." Yeast 11.15 (1995): 1485-1491.
[5] Chen, Wei Ning, and Kee Yang Tan. "Malonate uptake and metabolism in Saccharomyces cerevisiae." Applied biochemistry and biotechnology 171.1 (2013): 44-62.
[6] Kai, Yasushi, Hiroyoshi Matsumura, and Katsura Izui. "Phosphoenolpyruvate carboxylase: three-dimensional structure and molecular mechanisms."Archives of Biochemistry and Biophysics 414.2 (2003): 170-179.
[7] Song, Chan Woo, et al. "Metabolic engineering of Escherichia coli for the production of 3-aminopropionic acid." Metabolic engineering 30 (2015): 121-129.
[8] Zhang, Guijin, Stephen Brokx, and Joel H. Weiner. "Extracellular accumulation of recombinant proteins fused to the carrier protein YebF in Escherichia coli."Nature biotechnology 24.1 (2006): 100-104.
[9] Majander, Katariina, et al. "Extracellular secretion of polypeptides using a modified Escherichia coli flagellar secretion apparatus." Nature biotechnology23.4 (2005): 475-481.
Back to top
© 2016 iGEM Marburg · Contact
