<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Marburg_Dependency_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Marburg/Dependency/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Marburg"><B>iGEM Marburg 2016</B></A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Projects">Projects</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/HP/Silver">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Notebooks">Notebooks</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Marburg/Achievements">Achievements</A></LI></UL></DIV><DIV class="container"><DIV class="master-heading"><H1>
            SynDustry <SMALL>Fuse. Produce. Use.</SMALL></H1></DIV></DIV><DIV class="container"><DIV class="heading"><H2><B>Establishing a dependency among different organisms</B></H2></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-sm-4 text-center"><A href="https://2016.igem.org/Team:Marburg/Dependency" class="button" role="button">« Description  </A></DIV><DIV class="col-sm-4 text-center"><A href="#" class="button active" role="button">Results</A></DIV><DIV class="col-sm-4 text-center"><A href="https://2016.igem.org/Team:Marburg/Dependency/Outlook" class="button" role="button">  Outlook »</A></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-sm-12 text-justify"><H3><B>Expression plasmid and ACC1 knockout</B></H3></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
We created an expression plasmid that enables <I>S. cerevisiae</I>  to take up external malonate as well as establishes a novel pathway for malonyl-CoA synthesis. To achieve this, the plasmid consists of  the integrated <I>matB</I> gene from <I>R. leguminosarium</I> bv. <I>trifolii</I> under the control of constitutive promoters on the backbone (pNK26) carrying an ampicillin resistance for selection in <I>E. coli</I> and the <I>TRP1</I> gene for selection in <I>S. cerevisiae</I>. The plasmid was assembled via Gibson assembly reaction and Yeast assembly method.
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
For amplification of the <I>matB</I> and <I>mae1</I> gene we wanted to use codon optimized, synthesized nucleotide sequences. Since we had problems amplifying the received nucleotide sequence, which delayed the progress of this project, we decided to reorder the sequence of the codon optimized <I>matB</I> and amplify the <I>mae1</I> gene using genomic DNA from <I>S. pombe</I> 2698(lov)972 h- strain. 
The amplified <I>matB</I> fragment had a total length of 1615 bp and possessed short overhangs homologous to the backbone and the promoter region for the assembly reaction, that were added by using specially designed primers (Fig.1A).
<I>Mae1</I> was amplified using genomic <I>S. pombe</I> DNA. The amplified fragment had a total length of 1398 bp with overhangs homologous to the backbone and the promoter region (Fig.1B).
The bidirectional promoter region, consisting of the constitutive promoters pPGK1 and pTEF1, had a total length of 1409 bp and the backbone carrying the marker genes had a total length of 5962 bp. For both amplifications we used pNK26 as template.
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
For assembling the fragments and creating the final expression plasmid we performed Gibson assembly as well as Yeast assembly to directly introduce the expression plasmid into <I>S. cerevisiae</I> CEN.PK113-7D. The Gibson assembly product was transformed into <I>E. coli</I> NEB Turbo. The transformation was confirmed by colony PCR and analytical restriction using the restriction enzyme XhoI (Fig.1D). Due to limitations in time, we were not able to transform the expression plasmid into <I>S. cerevisiae</I> CEN.PK113-7D. 
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
In addition to establish the novel malonyl-CoA synthesis pathway, a deletion of the <I>acc1</I> gene in <I>S. cerevisiae</I> is required to disable yeast to produce malonyl-CoA from acetyl-CoA <A class="ref" href="#ref_1">[1]</A>. We created a knockout construct consisting of the <I>his3</I> gene flanked by a 200 bp nucleotide sequence homologous to the upstream area of the <I>E. coli</I><I>acc1</I> gene and a 200 bp nucleotide sequence homologous to the downstream area of the <I>E. coli</I><I>acc1</I> gene. We were able to amplify the fragments via Gibson assembly and cloned them into a pUC19 backbone, which was subsequently transformed into <I>E. coli</I> NEB Turbo and verified performing an analytical digest using the restriction enzyme PstI.
A knockout of <I>acc1</I> could not be performed due to its lethality and lacking of the expression plasmid which could compensate for the absence of malonyl-CoA.
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12"><DIV class="figure_text"><B>Figure1. Gel pictures of amplified fragments for Gibson assembly reaction.</B> 
The fragments were amplified using Phusion high fidelity polymerase. As marker we used GeneRuler 1kb Plus. <B>[A]</B> The gel shows the amplified fragments of <I>aspA</I> (1437 bp), <I>yneI</I> (1432 bp), <I>pa0132</I> (1499 bp) and <I>matB</I>(1615 bp). <B>[B]</B> The gel shows the amplified fragments of <I>mae1</I> with overhangs homologous to the adjacent fragments in the operon plasmid (1 and 2) and the amplified fragment of the <I>mae1</I> with overhangs homologous to the promoter region and the backbone of the expression plasmid (3 and 4). <B>[C]</B> Both samples show the amplified <I>panD</I> fragment including the constitutive promoter with homologous overhangs to the backbone and the adjacent gene in the operon plasmid (600 bp). <B>[D]</B> Marked is the linearized expression plasmid after test restriction with the restriction enzyme XhoI (10198 bp).
            </DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><H3><B>Operon plasmid</B></H3></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
The operon consists of genes that will direct the flux into the beta alanine pathway in <I>E. coli</I> towards malonic acid production <A class="ref" href="#ref_2">[2]</A>. In the end, the plasmid consists of <I>panD</I> from <I>C. glutamicum</I>, <I>aspA</I> from <I>E. coli</I>, <I>ppc</I> from <I>E. coli</I>, <I>yneI</I> from <I>E. coli</I> and <I>mae1</I> from <I>S. pombe</I>. The operon was designed to be controlled by the constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23108">BBa_J23108</A>. For the terminator, we chose the double terminator <A href="”http://parts.igem.org/Part:BBa_J61101”">BBa_J61101</A>. The backbone consists of the <I>E. coli</I> origin p15A ori and the chloramphenicol resistance gene under its native cat promoter(part of pACYC184). Additionally, we integrated two different spacer sequences between adjacent genes to avoid identical overhangs for the Gibson assembly reactions. To realize the design, we had to assemble the parts via Gibson assembly reaction since the yeast assembly methods would require additional sequences in the plasmid for selection and replication. We planned to pre-assemble some parts of the final plasmid, in order to avoid Gibson assembly reactions with more than four fragments. 
We had the templates for the amplifications of <I>panD</I> and <I>pa0132</I> codon optimized and synthesized for <I>E. coli</I>. The promoter for the operon was part of the synthesized sequence for <I>panD</I>. With this we were able to amplify the first two parts of the operon via PCR. The whole fragment including the promoter, the <I>panD</I> gene, one of the two spacer sequences between adjacent genes and a small overhang for the Gibson assembly reaction had a length of 600 bp (Fig.1C).
The synthesized sequence for the codon optimized <I>pa0132</I> gene also possessed overhangs for adjacent genes for Gibson assembly. The sequence had a total length of 1499 bp (Fig.1A). The last part of the pre-assembled plasmid Gibson1 was the <I>E. coli</I> native <I>aspA</I> gene consisting of 1437 bp (Fig.1A). Those three fragments were cloned into a pUC19 backbone and transformed into <I>E. coli</I> NEB Turbo. The plasmid was verified via an analytical digest using Kpn1. Subsequently it was used as template for the amplification of the combined fragment for the promoter <I>panD</I>, <I>aspA</I> and <I>pa0132</I> with a total length of 3471 bp (Fig.2A).
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
For the second pre-assembled plasmid Gibson2 we amplified the <I>E. coli</I> native <I>ppc</I> gene using genomic DNA from <I>E. coli</I> MG1655. The fragment consisted of 2650 bp carrying the <I>ppc</I> gene as well as upstream and downstream overhangs homologous to the adjacent genes (Fig.2C). The overhangs contained one of the two different spacer sequences and were created using specially designed primers. 
The second fragment for the pre-assembled plasmid Gibson2 was amplified using genomic DNA from <I>E. coli</I> MG1655. The fragment had a length of 1432 bp and consisted of the coding area of  the <I>E. coli</I> native <I>yneI</I> gene with overhangs homologous to adjacent genes (Fig.1A). 
For the last part we used genomic DNA from <I>S. pombe</I> 2698(lov)972 h- to amplify <I>mae1</I> with a length of 1397 bp (Fig.1B). The three fragments were assembled by Gibson assembly reaction and cloned into a pUC19 backbone with an ampicillin resistance gene. 
The transformation was verified by colony PCR. It was later used for the amplification of the combined fragment of <I>ppc</I>, <I>yneI</I> and <I>mae1</I> with a total length of 5480 bp
(Fig.2B).
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12"><DIV class="figure_text"><B>Figure 2. Gel pictures of amplified fragments for Gibson assembly reaction.</B>
The fragments were amplified using Phusion high fidelity polymerase. As marker we used GeneRuler 1kb Plus. <B>[A]</B> The gel shows the amplification of the combined fragment of the promoter, <I>panD</I>, <I>aspA</I> and <I>pa0132</I> (3471 bp). <B>[B]</B> The marked area shows a faint band at the right length of the combined fragment of <I>ppc</I>, <I>yneI</I> and <I>mae1</I> (5480 bp). <B>[C]</B> Both samples show the amplified fragment of <I>ppc</I> (2650 bp).
            </DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
For the final Gibson assembly we amplified the terminator BBa_J61101 using DNA from the distribution plates and the backbone using the plasmid pACYC184 as template.
The Gibson assembly product was transformed into the <I>E. coli</I> NEB Turbo, which was plated on chloramphenicol containing media.
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
To verify the presence of the construct a 2352 bp long fragment containing the backbone, the promoter and the <I>panD</I> gene was amplified via PCR (Fig.3A) and a 8939 bp long fragment containing the backbone, the terminator BBa_J23108, the combined fragment of the second pre-assembled plasmid and <I>pa0132</I> (Fig.3B).
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12"><DIV class="figure_text"><B>Figure 3. Gel pictures of amplified fragments for Gibson assembly reaction.</B>
The fragments were amplified using Phusion high fidelity polymerase. As marker we used GeneRuler 1kb Plus. <B>[A]</B> The gel shows the amplification of the <I>panD</I> and the backbone of the final operon plasmid (2352 bp). <B>[B]</B> Marked is faint band at the right length of the estimated PCR project consisting of the backbone, the terminator BBa_J23108, the combined Gibson 2 fragment and <I>pa0132</I> (8939). The strong band with a length of 4000 bp seems to be pollution that we had in all of the test samples.
            </DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
Malonate, secreted into the supernatant, was detected by UPLC-MS. Both the mutant E. coli and WT were measured via a modified protocol from Schwander et al. <A class="ref" href="#ref_3">[3]</A>. We compared the detected malonate levels related to the optical density of the control strain NEB Turbo with the transformed strain carrying our assembled operon plasmid.
For the NEB Turbo transformed with the operon plasmid the ion count normalized to the OD600 was at 5906.52 which is double the amount of the control (Fig.4). Both samples were treated the same.
Future work should include further validation of the system's functionality in addition to these promising results.
            </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12"><DIV class="figure_text"><B>Figure 4. Graphical representation of the UPLC-MS results.</B>
It shows the compared results of malonate detection of the NEB turbo control strain and the transformed NEB Turbo carrying the operon plasmid. With the protocol we followed malonate (103 m/z) elutes at 1.57 minutes. 
            </DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><H3><B>A different approach: Protein-based dependency</B></H3></DIV></DIV><DIV class="row"><DIV class="col-sm-12 text-justify"><P>
Our initial attempt was to use a previously described system, in which the <I>E. coli</I> protein YebF is fused to the N-terminal end of the protein of interest <A class="ref" href="#ref_5">[4]</A>. After transport of the fusion protein to the membrane due to a signal sequence from YebF, it will get cleaved while the protein of interest gets exported to the extracellular space. 
We amplified <I>yebF</I> with an additional His-tag performing colony PCR from the <I>E. coli</I> genome and ligated it into a pET28b vector under the control of an IPTG-inducible promoter. Even though the construct was verified through PCR, induction with different concentrations of IPTG showed no expression of the protein on SDS-page and via Western Blot. We suspected this could be due to the unknown cleavage mechanism that is part of the YebF export system. Hence, we fused the common reporter gene ß-glucunronidase to YebF and cloned it into pET28b, respectively. Again, induction with IPTG led to no detectable expression.
The second approach on establishing a functional protein export was by utilizing the flagellum assembly machinery of <I>E. coli</I>. The flagellin encoding gene <I>fliC</I> is regulated by an 172 bp signal sequence upstream of the protein coding region, containing promoter as well as RBS, and has been shown to be suitable for protein export in an E. coli strain with knocked out FliC and FliD <A class="ref" href="#ref_6">[5]</A>. Both genes are essential for the tightly regulated flagellar assembly and a knockout should result in unregulated expression of a protein fused to the <I>fliC</I> signal sequence. Therefore, we cloned the signal sequence, which was amplified from the <I>E. coli</I> genome to a His-tagged mRFP (<A href="http://parts.igem.org/Part:BBa_J04450">BBa_J04450</A>). The fusion protein got ligated into a tet-inducible pASK vector, since strong overexpression led to interference with the existing promoter of the signal sequence. Simultaneously, we tried to knock out <I>fliC</I> and <I>fliD</I> using the <I>E. coli</I> strain AB330, which is capable of homologous recombination using an amplified kanamycin resistance with 50 bp overhangs homologous to <I>fliC</I> and <I>fliD</I>. Even though the knockout was successful and could be verified indirectly through the formation of adhesive <I>E. coli</I> cells lacking the flagellum the culture did not grow sufficiently in the remaining time to transform it with the construct.
Another attempt to show protein expression was transforming the construct into BL21 and inducing it with different concentrations of anhydrotetracycline. Again, no expression could be detected on either SDS-page or via Western Blot, which should be due to the aforementioned tight regulation of FliC expression in an <I>E. coli</I> strain with functional genomic <I>fliC</I>.
            </P></DIV></DIV><DIV class="row"><H3 class="text-center">Literature</H3><OL class="literature_list"><LI id="ref_1"><A href="#ref_1" class="ref">[1]</A> Chen, Wei Ning, and Kee Yang Tan. &quot;Malonate uptake and metabolism in Saccharomyces cerevisiae.&quot; <I>Applied biochemistry and biotechnology</I>  171.1 (2013): 44-62.</LI><LI id="ref_2"><A href="#ref_2" class="ref">[2]</A> Song, Chan Woo, et al. &quot;Metabolic Engineering of Escherichia coli for the Production of 3-Hydroxypropionic Acid and Malonic Acid through β-Alanine Route.&quot; <I>ACS synthetic biology</I> (2016).</LI><LI id="ref_3"><A href="#ref_3" class="ref">[3]</A> Schwander, T. et al. “A synthetic pathway for the fixation of carbon dioxide in vitro.” <I>Science In press</I></LI><LI id="ref_5"><A href="#ref_5" class="ref">[4]</A> Zhang, Guijin, Stephen Brokx, and Joel H. Weiner. &quot;Extracellular accumulation of recombinant proteins fused to the carrier protein YebF in Escherichia coli.&quot;Nature biotechnology 24.1 (2006): 100-104.</LI><LI id="ref_6"><A href="#ref_6" class="ref">[5]</A> Majander, Katariina, et al. &quot;Extracellular secretion of polypeptides using a modified Escherichia coli flagellar secretion apparatus.&quot; Nature biotechnology23.4 (2005): 475-481.</LI></OL></DIV></DIV><DIV class="container"><FOOTER><P class="pull-right"><A href="#">Back to top</A></P><P>© 2016 iGEM Marburg ·<A href="mailto:igemteam@synmikro.uni-marburg.de">Contact</A></P></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>