<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Manchester_Aimtest2"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Manchester/Aimtest2">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Manchester/Aimtest2&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Manchester/Aimtest2&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Manchester/Aimtest2&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Manchester/Aimtest2" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Manchester/Aimtest2</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE id="title"> page </TITLE></P><DIV class="container"><DIV id="block"><A href="https://2013.igem.org/Team:Manchester/Hometest3" id="link1">Home</A></DIV><DIV id="block1"><UL class="list"><LI><A href="https://2013.igem.org/Team:Manchester/OurTeamtest" id="link2">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1027" target="_blank">Team Profile</A></LI></UL></DIV><DIV id="block2"><UL class="list"><LI><A href="https://2013.igem.org/Team:Manchester/Projecttest" id="link3">Project</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/Aimtest2" id="link3">Project Overview</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/Notebooktest2" id="link3">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/LabBooktest" id="link3">Lab Book</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/contributiontest" id="link3">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/Safetytest" id="link3">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/medaltest" id="link3">Judging</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/Attributionstest" id="link3">Attributions</A></LI></UL></DIV><DIV id="block3"><UL class="list"><LI><A href="https://2013.igem.org/Team:Manchester/Modellingtest" id="link6">Modelling</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/parametertest" id="link6">Uncertainty Analysis</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/fabAmodeltest" id="link6">FabA Dynamics Model</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/popdynamictest" id="link6">Population Dynamics</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/collabtest" id="link6">Modelling Collaboration</A></LI></UL></DIV><DIV id="block4"><UL class="list"><LI><A href="https://2013.igem.org/Team:Manchester/HumanPracticetest" id="link4">Human Practices</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/outreachtest" id="link4">Public Outreach</A><UL class="submenu1"><A href="https://2013.igem.org/Team:Manchester/stars" id="link4">Science Stars</A></UL></LI><A href="https://2013.igem.org/Team:Manchester/openday" id="link4">Open day</A></UL><LI><A href="https://2013.igem.org/Team:Manchester/ethicstest" id="link4">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/environmenttest" id="link4">Environmental Impact</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/economytest" id="link4">Economic Impact</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/managementtest" id="link4">Impact Management</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/conclusiontest" id="link4">Conclusion</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/businesstest" id="link4">Business Plan</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/collabtest" id="link4">Modelling Collaboration</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/knoledgetest" id="link4">Knowledge Deficit Assumption</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/conferencetest" id="link4">Conferences and Discussions</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/socialtest" link4="">Social Media</A></LI></DIV><DIV id="block5"><A href="https://2013.igem.org/Team:Manchester/sponsorstest" id="link5">Sponsorship</A></DIV></DIV><DIV class="anchor"><A href="#" class="topButton">Top</A></DIV><P><TITLE> Safety </TITLE></P><DIV class="global"><DIV class="wrapper"><DIV class="title"><I>E. c(oil)i</I>; The Lean Green Fat-Producing SynBio Machine
         </DIV><DIV class="container2"><P><B><U><A id="Q1">Introduction</A></U></B></P><P>	Palm oil is a huge business. From food products such as margarine and chocolate, to cosmetics and even applications in biodiesel, palm oil is the most widely used vegetable oil in the world, and its demand is only increasing. But what most of the manufacturers of these products fail to publicly address is the massive devastation caused by ripping up rainforests in order to make room for oil palm plantations. Not only does this mass deforestation damage the planet by increasing carbon emissions and decreasing the amount of CO<SUB>2</SUB> taken up from the atmosphere, it also destroys the homes and habitats of thousands of endangered animals, including orangutans and tigers. Additionally, many rainforests are found growing atop peatlands, and so it follows that when they are ripped up or burned, colossal amounts of CO<SUB>2</SUB> are released. CO<SUB>2</SUB> is of course a greenhouse gas and so obviously this has a big impact on the extent and rate of global warming. We discuss the devastation associated with the palm oil industry extensively over on our <A href="https://2013.igem.org/Team:Manchester/environmenttest" target="_blank"> Ethics</A> pages.</P><P>	Palm oil is made up of four major components: palmitic acid (44%), stearic acid (4.5%), oleic acid (39.2%), and linoleic acid (10.1%). These four fatty acids are fractionated from crude palm oil and used in various different applications according to their degree of saturation- stearic acid is used primarily in cosmetics, where oleic acid is an important oil used in cooking, for example. The biosynthesis of fatty acids has been studied extensively in <I>E. coli</I>, which makes this the ideal chassis for our project. </P><P>	Our aim is to overexpress the pathway of naturally occurring saturated fatty acids (palmitic (C16:0) and stearic acid (C18:0)) to result in a net increase in their production, and then insert non-native genes to introduce double bonds in stearic acid, to yield oleic (C18:1, Δ9) and linoleic acid (C18:2, Δ9,12). Producing the most widely used and valuable components of palm oil in this way would render the widespread deforestation directly associated with the palm oil industry redundant, and so would halt this relentless destruction of the environment. </P></DIV><DIV class="container4"><P><B><U><A id="Q2">Why did we choose delta 9 desaturase, delta 12 desaturase, FabA?</A></U></B></P><P><B>Delta 9 and delta 12 desaturase</B></P><P>The reactions converting stearic acid to oleic acid and the conversion of oleic acid to linoleic acid are performed by delta 9 and delta 12 desaturase enzymes respectively. These enzymes are not native to <I>E. coli</I>, which meant our project would include the introduction of non-native genes in <I>E. coli</I> - presenting its own set of problems. We evaluated the various organisms that are native hosts to delta 9 and delta 12 desaturase enzymes in the hope of finding an organism as similar as possible to <I>E. coli</I> in order to minimise the risk of mis-folded products amongst other potential problems. We began looking into other prokaryotic microorganisms that would possibly contain the desaturases we were looking for. We first looked at the bacteria <I>Mycobacterium tuberculosis</I> - that had the native delta 9 and delta 12 desaturase enzymes. However, the potential implications of using a pathogenic organism such as <I>M. tuberculosis</I> meant this wasn’t a viable option. In addition to this, we discovered that there may have been some protein folding issues when <I>M. tuberculosis</I> were introduced in to <I>E.coli</I></P><P>Algae was the next organism we considered, which also has native delta 9 and delta 12 desaturase enzymes. From our initial evaluation of eukaryotic algae (which would almost certainly present mis-folded product) we were hopeful we could also find the same genes located within prokaryotic cyanobacteria. We eventually were able to find a suitable cyanobacteria, <I>Synechocystis</I> sp. PCC 6803. </P><P><B>FabA</B></P><P>FabA is one of the specific genes encoded by the ‘FAS Module’, generously donated to us for use by Prof. Koffas (of Rensselaer Polytechnic Institute, USA), for overexpression of specific genes resulting in increased stearic (and palmitic) acid output. Initially our modelling team were struggling to obtain kinetic data for FabA, and as this was unfortunately not an existing BioBrick component we decided we would create the FabA BioBrick in conjunction with our other submitted BioBricks. We then hoped to be able to use the data we would gather from FabA expression to help our modelling team by providing the kinetic parameters of this enzyme. The FabA BioBrick we submitted included His tags in the N-Terminus of the protein product to help with purification of this protein. FabA was also the only gene in the ‘FAS Module’ not present in the BioBrick library, we therefore felt this should be submitted as a useful tool for future iGEM Teams. </P><P><B>Why did we choose to knock out FadD? </B></P><P> Preliminary research showed that FadD was involved in the start of the beta-oxidation pathway, which breaks down fatty acids for energy. From this, we decided to attempt to knock-out the FadD gene in <I>E. coli</I>, replacing it with a marker for chloramphenicol resistance. Our Modelling team also explored the impact of removing enzymes that could potentially be involved in the breakdown of our products. They saw that the protein product of FadD has a negative impact on the accumulation of the longer chain fatty acids that we aim to produce, therefore supporting the experimental work. </P></DIV><DIV class="container5"><P><B><U><A id="Q3">One batch to rule them all?</A></U></B></P><P>	Initial ideas surrounding our project led to us wanting to attempt to produce all 4 of our chosen fatty acids within the same cell. We believed that this would be advantageous for a number of reasons. Firstly, having all components produced within the same batch of bacteria would potentially reduce the running costs of the project on an industrial scale, as all vats within the setting would contain the exact same bacteria. This would also decrease the likelihood of any cross-contamination occurring. Another reason for the proposed one cell approach was that the products of the naturally-occurring mechanism for fatty acid biosynthesis (palmitic and stearic acid) could be overexpressed and then used to feed the following reactions; that of delta 9 desaturase (oleic acid) and delta 12 desaturase (linoleic acid). This would lead to all four components being dependent upon the production of the others, and we hypothesised that this pathway could be tuned so as to give different ratios of each fatty acid in the fatty acid profile seen. The ability to finely tune the ratios of the fatty acids produced would open doors to specialised oils, such as a high-oleic cooking oil for instance. Moreover, having the system essentially feed itself (after the initial input of glucose) rather than using exogenous stearic and oleic acid to feed the later pathways would hopefully reduce operating costs. This would only be the case if the fatty acids being added to the media worked out more expensive than the glucose needed to maintain the reactions. Whilst an exciting concept, we didn’t feel like we had the time during our project to properly attempt this. Maybe it’s a job for a future iGEM team!</P><P>The alternative to having all components produced within one batch of cells is to have each construct expressed in a separate batch of bacteria. This is a much more simple route, as problems such as plasmid conflicts would be eliminated. In essence, different ratios of oils could still be achieved by adding each component to the others at a specified volume after being extracted from the culture. After speaking with a representative of a company who uses palm oil in their products, they expressly said that producing the components of palm oil in separate batches would be the best scenario. This is because crude palm oil, once it has been processed from the fruit, is then fractionated to give separate fatty acids according to their physical properties anyway. Therefore, logic suggests that having separate batches producing each main component of palm oil would reduce the need for further processing down the production line. </P><P>Ultimately, the team decided that producing oleic acid and linoleic acid separately would be the best route to take, and so that’s what we did!</P></DIV><DIV class="container6"><P><B><U><A id="Q4">Homogenisation vs. exogenous feeding</A></U></B></P><P>Because we had chosen to produce our fatty acids in three different production strains, it was necessary to feed our transformed <I>E. coli</I> with fatty acids to produce the product we wanted. Without this exposure, the proteins would be translated but there would be no fatty acid for them to act upon. The ‘Delta 9’ strain had to be exposed to stearic acid to produce oleic acid, and the ‘Delta 12’ strain had to be exposed to oleic acid to produce linoleic acid.</P><P>We had two main options: breaking the cells open (homogenisation) or exogenous feeding. Both options had pros and cons associated with them. Homogenisation would allow the fatty acids and the enzymes to come in to direct contact with each other very easily <I>in vitro</I>. Whilst this would appear to be a very simple way of producing the required fatty acids, there is of course the problem that the cells would be killed and therefore unable to produce any more product in the long run. This would be a major inconvenience for industry, which would want sustained production over time. Therefore feeding the bacteria exogenously would be a more viable option of long-term production. This has its own set of problems, however. The primary concern is that passive diffusion of long-chain fatty acids is very slow and therefore could be rate limiting. This said, it would cause less metabolic stress on the cells and in the long run return a higher yield per cell.</P></DIV><DIV class="container7"><P><B><U><A id="Q5">Results</A></U></B></P><P><B>Improvement of existing BioBrick</B></P>
Our biobrick BBa_K1027002 (<B>delta 12 desaturase</B>) is an improvement of the <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K925000" target="_blank">BBa_K925000</A> biobrick created by the St Andrews team in 2012. We improved this biobrick from the ground up, starting by taking the protein sequence from <I>Synechocystis</I> sp. PCC 6803. This protein sequence was reverse translated and then codon optimised for expression in <I>E. coli</I>. We believe our redesigned biobrick has two features that the original biobrick does not. Firstly, the original biobrick could not be transformed by iGEM HQ, whereas we have managed successful transformation (as shown by the plate pictures). Next, the St. Andrews biobrick was cloned from cyanobacteria, and so is not optimised for expression in <I>E. coli</I>. As St. Andrews then went on to express their delta 12 desaturase biobrick in <I>E. coli</I> it follows that a sequence optimised for expression in this chassis, as ours was, would perform better than the sequence naturally found in cyanobacteria.<P><B>Orbitrap LC-MS analysis</B></P>
Delta 9 desaturase and delta 12 desaturase enzymes were chosen because their products, when expressed in their host organism (<I>Synechocystis</I> sp. PCC 6803), convert stearic acid into <B>oleic acid</B>, and oleic acid into <B>linoleic acid</B> respectively. Therefore, we fed batches of transformed DH5-alpha with 2 different concentrations of exogenous fatty acid (0.1% and 0.5% stearic acid fed to the delta 9 desaturase batch, and 0.1% and 0.5% oleic acid fed to the delta 12 desaturase batch), left the cultures growing overnight and then harvested the cells.
To analyse the metabolites extracted from both wild-type DH5-alpha and DH5-alpha expressing our delta 9 desaturase and delta 12 desaturase enzymes, we made use of the MIB’s in-house Orbitrap Liquid Chromatography - Mass Spectrometry (LC-MS). This technique was chosen because of its high mass accuracy and sensitivity. Upon analysing the most abundant metabolites extracted from our expression strains and comparing this data with the most abundant metabolites extracted from wild-type, it is apparent that a <B>massive increase in linoleic acid</B> (incorporated in phosphatidylethanolamines, PE) has occurred. This is demonstrated in the figure above. The chromatograms produced for the delta 12 desaturase expression strains, along with further characterisation, can be found on the <A href="https://2013.igem.org/Team:Manchester/contributiontest" target="_blank">Parts</A> page.<B>Further analysis</B>
The data obtained from the LC-MS was enormous. 2721 metabolites were detected in the samples, and so we filtered this down to the 43 fatty acids and phospholipids incorporating the compounds we were interested in (oleic acid and linoleic acid). A heat map was generated (seen below) showing the abundances of these 43 phospholipids and fatty acids in the 17 samples we run on the LC-MS. From this heatmap the diversity of detected compounds and and high dynamic range can be seen. For the characterisation of our delta 12 desaturase BioBrick construct, a focus on the individual compounds of high intensity was necessary. An example of this can be seen in the bar chart in the work flow diagram below. Here you can see that, when fed with oleic acid (to a total w/v concentration of 0.1% and 0.5%), the amount of linoleic acid found within this representative phospholipid is much higher in the delta 12 desaturase expression strain than in the wild-type DH5-alpha strain. From this data we can confidently conclude that the delta 12 desaturase is converting the substrate oleic acid into the linoleic acid. Currently, the linoleic is incorporated into a phospholipid, but expression of the tesA gene (biobricked previously) would cleave the compound away to give free fatty acid.<B>A work flow of the methods we used to get from sample to data analysis can be seen below.</B></DIV><DIV class="container8"><P><B><U><A id="Q6">Future work: What’s next for Synthetic Palm Oil?</A></U></B></P><P>Whilst our iGEM Project is over, we looked at a variety of ways to build onto our project in the future.</P><P><B>Beta-Carotene Enhanced Palm Oil</B></P><P>It may be possible to optimise the palm oil produced to provide a healthier alternative to traditionally produced palm oil. One such example of this would be the introduction of beta-carotene into the palm oil component mix. 
Beta-carotene is a precursor to vitamin A. Vitamin A is essential for a variety of health aspects – from eyesight to a strengthened immune system. Vitamin A can prove toxic at high levels, which is why introducing beta-carotene would be a safe alternative. The human body only converts sufficient beta-carotene to reach optimum vitamin A levels – preventing a potentially toxic excess of vitamin A. 
This idea could potentially make use of the work of <A href="https://2013.igem.org/Team:Uppsala/betacarotene" target="_blank">Uppsala iGEM 2013</A>, who have been working on genetically engineering beta-carotene as part of their project this year. 
Implications for the taste and appearance of this optimised palm oil alternative may however make this goal hard to achieve. Many food production companies would be keen to derive the benefits of beta-carotene enhanced palm oil but would be unlikely to implement such a product if changes to appearance and taste occurred as a result of using beta-carotene enhanced palm oil.</P><P><B>Waste paper/agricultural materials as a source of Glucose</B></P><P>After attending the 2013 YSB 1.0 London conference and speaking to the <A href="https://2013hs.igem.org/Team:UCL_Academy" target="_blank">UCL Academy team</A> we were really interested in how we might find an alternative feed stock to glucose. Achieving this would dramatically decrease the running costs of any palm oil production plant, as glucose is one of the major cost factors in starting materials for production of our synthetic palm oil. The UCL Academy team worked to produce a cellulase BioBrick (due to be submitted to the registry in 2014) which would break down particular cellulose fibres in paper structure, producing glucose. We would also be interested in exploring the application of this biobrick in terms of the use of cellulose from argicultural waste, such as rice husks, as a feed stock. Not only would this reduce the cost of producing our palm oil, but would also aid in the disposal of waste.</P><P><B>Delta 12 desaturase</B></P><P>In 2012 the <A href="https://2012.igem.org/Team:St_Andrews" target="_blank">St Andrews iGEM Team</A> aimed to achieve a similar goal to our team, to produce a synthetic Omega-3 fatty acid. Their pathway included delta 12 desaturase, an enzyme we required in the conversion of oleic acid to linoleic acid. St Andrews had produced the delta 12 BioBrick and submitted this to the Registry – which provided an invaluable opportunity to use existing components from the Registry. Unfortunately when we requested this part from the iGEM Registry they were unable to send us this BioBrick as it would not transform, and after contacting the University they were also unable to provide us with this BioBrick.</P><P>As delta 12 desaturase is an essential enzyme in the production of our Palm oil substituents we therefore attempted to re-produce the delta 12 BioBrick submitted by St. Andrews with improvements such as codon optimisation for <I>E. coli</I> BL21 (DE3). </P><P><B>Creating this system in Algae</B></P><P>During our project planning we evaluated the prospect of genetically engineering algae, such as cyanobacteria in order to produce our fatty acid products. The advantages of this would include being able to use light as a source of energy, significantly reducing the expense of media required - in particular glucose. The gene sequences for delta 9 and delta 12 desaturases are also not-native in <I>E. coli</I>, so these were obtained from a cyanobacteria. Whilst the cyanobacteria is prokaryotic, there was potential risk for incorrect protein folding when expressing these genes in <I>E. Coli</I>. 
Unfortunately because of the slow growth rate of cyanobacteria we were unable to use this as a chassis for production of our fatty acids. Further, as an iGEM requirement all BioBricks must function in <I>E. coli</I> - so we would potentially have to repeat our project in two different organisms in order to satisfy this criteria. 
In the future we would like to see our system created in algae to derive the potential benefits discussed. </P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Manchester/Aimtest2">http://2013.igem.org/Team:Manchester/Aimtest2</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Manchester/Aimtest2" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Manchester/Aimtest2" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Manchester/Aimtest2&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Manchester/Aimtest2&amp;oldid=320567" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>