<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Manchester_contributiontest"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Manchester/contributiontest">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Manchester/contributiontest&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Manchester/contributiontest&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Manchester/contributiontest&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Manchester/contributiontest" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Manchester/contributiontest</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE id="title"> page </TITLE></P><DIV class="container"><DIV id="block"><A href="https://2013.igem.org/Team:Manchester/Hometest3" id="link1">Home</A></DIV><DIV id="block1"><UL class="list"><LI><A href="https://2013.igem.org/Team:Manchester/OurTeamtest" id="link2">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1027" target="_blank">Team Profile</A></LI></UL></DIV><DIV id="block2"><UL class="list"><LI><A href="https://2013.igem.org/Team:Manchester/Projecttest" id="link3">Project</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/Aimtest2" id="link3">Project Overview</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/Notebooktest2" id="link3">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/LabBooktest" id="link3">Lab Book</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/contributiontest" id="link3">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/Safetytest" id="link3">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/medaltest" id="link3">Judging</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/Attributionstest" id="link3">Attributions</A></LI></UL></DIV><DIV id="block3"><UL class="list"><LI><A href="https://2013.igem.org/Team:Manchester/Modellingtest" id="link6">Modelling</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/parametertest" id="link6">Uncertainty Analysis</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/fabAmodeltest" id="link6">FabA Dynamics Model</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/popdynamictest" id="link6">Population Dynamics</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/collabtest" id="link6">Modelling Collaboration</A></LI></UL></DIV><DIV id="block4"><UL class="list"><LI><A href="https://2013.igem.org/Team:Manchester/HumanPracticetest" id="link4">Human Practices</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/outreachtest" id="link4">Public Outreach</A><UL class="submenu1"><A href="https://2013.igem.org/Team:Manchester/stars" id="link4">Science Stars</A></UL></LI><A href="https://2013.igem.org/Team:Manchester/openday" id="link4">Open day</A></UL><LI><A href="https://2013.igem.org/Team:Manchester/ethicstest" id="link4">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/environmenttest" id="link4">Environmental Impact</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/economytest" id="link4">Economic Impact</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/managementtest" id="link4">Impact Management</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/conclusiontest" id="link4">Conclusion</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/businesstest" id="link4">Business Plan</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/collabtest" id="link4">Modelling Collaboration</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/knoledgetest" id="link4">Knowledge Deficit Assumption</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/conferencetest" id="link4">Conferences and Discussions</A></LI><LI><A href="https://2013.igem.org/Team:Manchester/socialtest" link4="">Social Media</A></LI></DIV><DIV id="block5"><A href="https://2013.igem.org/Team:Manchester/sponsorstest" id="link5">Sponsorship</A></DIV></DIV><DIV class="anchor"><A href="#" class="topButton">Top</A></DIV><P><TITLE> Safety </TITLE></P><DIV class="global"><DIV class="wrapper"><DIV class="text2">
            We submitted and sequenced three biobricks integral to our project. These can be found below:<DIV id="groupparts" style="min-height:100px;width:900px;"><DIV style="width:300px;margin:2px;padding:5px;color:gray;border:1px solid gray">Loading.....</DIV></DIV></DIV><DIV class="title"><B>This year we submitted and characterised 3 BioBricks, one of which is an improvement of <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K925000" target="_blank">BBa_K925000</A>!</B></DIV><DIV class="text3"><CENTER><I>We used an existing BioBrick component with Constitutive promoter and RBS (BBa_K608002) to express our BioBrick parts:</I></CENTER></DIV><DIV class="text1"><P><B>Successful Expression of delta 9 and delta 12 desaturase, and FabA</B></P><P>In order to characterise our biobricks, we made use of the standard parts found in the registry. By inserting the created biobricks BBa_K1027001 and BBa_K1027002 in to BBa_K608002 (a biobrick consisting of a ribosomal binding site and a constitutive promoter), we were able to create a new construct that expressed the delta 9 desaturase, delta 12 desaturase and FabA proteins. The constructs are shown above.</P><P><B>Plates</B></P><P>Another piece of evidence suggesting that our constructs were successfully expressing our delta 9 desaturase and delta 12 desaturase enzymes was the size of the colonies grown on the plates. Pictures of the plates are shown below. On the far left is a control plate, with bacteria transformed with BBa_K608002 but with no gene inserted in front of the promoter. To the right of that are our delta 12 colonies. They are much smaller than the control, and even taking 20 hours to grow to that size as opposed to the usual 16 hours. We hypothesise that because delta 12 desaturase is a membrane-bound protein that the <I>E. coli</I> does not normally express, constitutively expressing it could inhibit the bacteria and slow growth. Compare this to FabA on the far right. Looking through the literature, we found that overexpression of FabA results in no significant difference in growth size or speed relative to the wild type strain (Luo et al, 2009). Compared to the control plate, we also found that the colonies grew normally.</P><P><B>Gel Digests</B></P><P>To confirm that our desired genes were in fact within the expression construct mentioned above, we carried out test digests of our ligated plasmids. Happily, when we ran our digests on an agarose gel, we saw all of the bands we would predict from the expected fragments. The gel pictures are on the right. <B>delta 9 desaturase:</B> Cut with BamHI + EcoRV. Expected bands 1424 bp, 1262 bp, 99 bp.<B>delta 12 desaturase:</B> Cut with BamHI, XbaI, PstI. Expected bands 2127 bp, 656 bp, 430 bp.<B>FabA</B> Cut with EcoRV. Expected bands 1429 bp, 1115 bp. Cut with EcoRV, PstI. Expected bands 1153 bp, 1115 bp, 339 bp.</P><P><B>Improvement of existing BioBrick</B></P><P>Our biobrick BBa_K1027002 (<B>delta 12 desaturase</B>) is an improvement of the <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K925000" target="_blank">BBa_K925000</A> biobrick created by the St Andrews team in 2012. We improved this biobrick from the ground up, starting by taking the protein sequence from <I>Synechocystis</I> sp. PCC 6803. This protein sequence was reverse translated and then codon optimised for expression in <I>E. coli</I>. We believe our redesigned biobrick has two features that the original biobrick does not. Firstly, the original biobrick could not be transformed by iGEM HQ, whereas we have managed successful transformation (as shown by the plate pictures). Next, the St. Andrews biobrick was cloned from cyanobacteria, and so is not optimised for expression in <I>E. coli</I>. As St. Andrews then went on to express their delta 12 desaturase biobrick in <I>E. coli</I> it follows that a sequence optimised for expression in this chassis, as ours was, would perform better than the sequence naturally found in cyanobacteria.</P></DIV><DIV class="double2"><DIV class="left"><P id="footer"><B>LB-Agar (CmR) plates after 20h growth at 37°C. Left: Control plate (transformed with empty vector BBa_K608002). Right: Transformed with delta 12 desaturase (note smaller colonies)</B></P></DIV><DIV class="right"><P id="footer"><B>LB-Agar (CmR) plates after 20h growth at 37°C. Left: Control plate (transformed with empty vector BBa_K608002). Right: Transformed with FabA gene (note similar sized colonies)</B></P></DIV></DIV><DIV class="text1"><P><B>Orbitrap LC-MS analysis</B></P><P> Delta 9 desaturase and delta 12 desaturase enzymes were chosen because their products, when expressed in their host organism (<I>Synechocystis</I> sp. PCC 6803), convert stearic acid into <B>oleic acid</B>, and oleic acid into <B>linoleic acid</B> respectively. Therefore, we fed batches of transformed DH5-alpha with 2 different concentrations of exogenous fatty acid (0.1% and 0.5% stearic acid fed to the delta 9 desaturase batch, and 0.1% and 0.5% oleic acid fed to the delta 12 desaturase batch), left the cultures growing overnight and then harvested the cells.
To analyse the metabolites extracted from both wild-type DH5-alpha and DH5-alpha expressing our delta 9 desaturase and delta 12 desaturase enzymes, we made use of the MIB’s in-house Orbitrap Liquid Chromatography - Mass Spectrometry (LC-MS). This technique was chosen because of its high mass accuracy and sensitivity. Upon analysing the most abundant metabolites extracted from our expression strains and comparing this data with the most abundant metabolites extracted from wild-type, it is apparent that a massive increase in linoleic acid (incorporated in phosphatidylethanolamines, PE) has occurred. This is demonstrated in the two figures directly below. The chromatograms produced for the delta 12 desaturase expression strains are also shown below. There is a clear difference between the wild-type <I>E. coli</I> fed with exogenous substrate compared with the <I>E. coli</I> strains expressing delta 12 desaturase. It is probable that the peak appearing around 7.9 min in the delta 12 desaturase strains is due to phospholipid incorporating <B>18:2 (9Z, 12Z) - linoleic acid</B>.
</P></DIV><DIV class="double2"><DIV class="left"><P id="footer"><B>The four most abundant metabolites extracted from <B>wild-type DH5-alpha</B>. Analysed using Orbitrap LC-MS. Note prominence of short-chain fatty acids. </B></P></DIV><DIV class="right"><P id="footer"><B>The four most abundant metabolites extracted from our <B>delta 12 desaturase</B> expression strains. Analysed using Orbitrap LC-MS. Note prominence of C18:2 (9Z, 12Z) (linoleic acid). </B></P></DIV></DIV><DIV class="text4"><P id="footer"><B>Above Left: Chromatograms obtained through Orbitrap LC-MS analysis. Above Right: Workflow of how we went from samples to resultsBelow Left: Bar chart showing increase in linoleic acid when delta 12 desaturase is presentBelow Right: Heat map showing metabolite abundances </B></P><DIV class="text5"><P><B>Further analysis</B>
The data obtained from the LC-MS was enormous. 2721 metabolites were detected in the samples, and so we filtered this down to the 43 fatty acids and phospholipids incorporating the compounds we were interested in (oleic acid and linoleic acid). A heat map was generated (seen above) showing the abundances of these 43 phospholipids and fatty acids in the 17 samples we run on the LC-MS. From this heatmap the diversity of detected compounds and high dynamic range can be seen. For the characterisation of our delta 12 desaturase BioBrick construct, a focus on the individual compounds of high intensity was necessary. An example of this can be seen in the bar chart above. Here you can see that, when fed with oleic acid (to a total w/v concentration of 0.1% and 0.5%), the amount of linoleic acid found within this representative phospholipid is much higher in the delta 12 desaturase expression strain than in the wild-type DH5-alpha strain. From this data we can confidently conclude that the delta 12 desaturase is converting the substrate oleic acid into the linoleic acid. Currently, the linoleic is incorporated into a phospholipid, but expression of the tesA gene (biobricked previously) would cleave the compound away to give free fatty acid. <B>A work flow of the methods we used to get from sample to data analysis can be seen above.</B></P></DIV></DIV><DIV class="text6"><P id="footer"><B>Atomic Force Microscopy pictures. Left: WT DH5-alpha. Right: DH5-alpha transformed with delta 12 desaturase expression plasmid.</B></P><P><B>Atomic Force Microscopy</B>
Whilst overexpression of the delta 12 desaturase led to a 4 fold increase in linoleic acid production incorporated in certain phospholipids, we had hypothesised that both overexpression of a non-native membrane protein and the resulting fatty acid composition changes would possibly alter <I>E. coli</I> membrane structure and potentially interfere growth and replication.  To address this, we performed Atomic Force Microscopy (AFM) on transformed wild-type and <I>E. coli</I> transformed with our delta 12 desaturase expression construct. From this we found that there was no significant difference in the membranes between the wild-type and delta 12 desaturase expression constructs.  From this we can conclude that overexpression of the non-native membrane bound delta 12 desaturase using our constitutive promoter system does not significantly alter the membranes. Therefore on an industrial scale the expression construct could be modified, such as by the addition of a stronger promoter sequence and RBS site or potentially using inducible expression systems, thus resulting in a greater production of linoleic acid.  
</P></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Manchester/contributiontest">http://2013.igem.org/Team:Manchester/contributiontest</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Manchester/contributiontest" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Manchester/contributiontest" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Manchester/contributiontest&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Manchester/contributiontest&amp;oldid=347604" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>