<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Manchester_LabBookText"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Manchester/LabBookText">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Manchester/LabBookText&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Manchester/LabBookText&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Manchester/LabBookText&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Manchester/LabBookText" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Manchester/LabBookText</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><B>14/06/2013 - Making media</B></P><P>1. Calculate amount of powder of LB agar or LB broth mix required in accordance with manufacturer's instructions</P><P>2. Add LB powder to distilled water (we used 10 g of LB-broth or LB agar to 400ml of water</P><P>3. Autoclave the flask ( Make sure you have the autoclave tape on with label) </P><P>4. Add any antibiotic if needed once the flask has sufficiently cooled (approx 50 degrees).</P><P><B>19/06/2013 - Preparation of Chemically competent Cells</B></P><P>1. Grow the specific strain (BL21(DE3))on the plate. Plates will be in the incubator.</P><P>2. Select a colony from the plate with inoculating loop</P><P>3. The colony is dispersed in 100 ml LB broth in a conical flask.</P><P>4. The culture is left overnight- vigorous shaking at 37°C</P><P><I>Taking the OD of the grown culture</I></P><P>Want an OD of 0.1 in a culture volume of 100ml</P><P>Calculation:</P><P>V<SUB>1</SUB> x 3.2 = 100 x 0.1</P><P>V<SUB>1</SUB>= 3.1 ml</P><P>96.9 ml [LB broth] + 3.1 ml [culture] = 100 ml culture with OD of 0.1</P><P>5. Take original OD volume of the culture (at 600 nm). Use LB as a blank. N.B: You 
need an OD less than 1.5 to measure the density accurately</P><P>6. Diluted culture is transferred to sterile flask. This is placed in shaking incubator at 
37°C. After an hour, the OD is measured 15 minutes until it reaches the range of 0.4-
0.6</P><P>7. Centrifuge at 6000 xg for 10 min or 5500 xg for 15 min in 2 x 50 ml tubes.</P><P>8. Remove the supernatant from the tubes. Combine both the pellets and suspend in ice cold 0.1M CaCl<SUB>2</SUB></P><P>9. Leave the tube on ice for 30 min.</P><P>10. Spin cells at 6000 xg for 10 min or 5500 xg for 15 min and remove supernatant</P><P>11. Resuspend the pellet in 4 ml ice cold 0.1 CaCl2</P><P>12. Put on ice in cold room overnight</P><P>13. Add 1 ml sterile 100% glycerol</P><P>14. Make 50 µl aliquot and store at -80°C (after freezing with liquid nitrogen)</P><P><B>20/06/2013 - FadD Knock Out Part 1 (Transformation of Chemically Competent <I>E.Coli</I>) </B></P><P>1) Incubate 50-100µl of competent cells with either 0.5-1 µl of commercial plasmid (pKD46: (Lambda Red Recombinase Plasmid)) or the same volume of water/ligation mixture (control). ~1.2µl of plasmid to 50µl of cells</P><P>2) Leave on ice for 30 minutes</P><P>3) Heat shock: 42 ºC in water bath for 45 seconds OR 37ºC heat block for 2 mins</P><P>4) Leave on ice for 5 minutes</P><P>5) Recover cells by adding 500µl of LB broth</P><P>6) Incubate at 37 ºC for 1-2 hours</P><P>7) Plate dilutions on agar containing appropriate antibiotic</P><P><B>21/06/2013 - Continued</B></P><P> Yesterday’s transformation failed. Today we tried the experiment following the protocol of 20/06 although cells were heat shocked at 37 ºC on a heat block for 2 minutes instead of 42 ºC in a water bath for 45 seconds.</P><P>50 µg/ml ampicillin agar plates mate. 0.75g of ampicillin added to 15 ml of 50% ethanol to create 50 mg/ml stock to be kept in -20 ºC freezer. 0.4ml of this stock added to 400 ml of LB-agar:</P><P>50 mg/ml x V<SUB>1</SUB> = 50 µg/ml x 400 ml</P><P>50000 µg x V<SUB>1</SUB> = 50 µg/ml x 400ml </P><P>V<SUB>1</SUB>=0.4 ml (400 µl)</P><P><B>24/06/2013 - Growing Cells From 21/06/2013</B></P><P>1) Colony selected with inoculating loop </P><P>2) Colony added to 100 ml LB broth </P><P>3) Colony left overnight in incubator at 37 ºC </P><P><B>25/06/2013 - Electrically Competent Cells For Electroporation (<I>E.Coli</I> BL21 (DE3))</B></P><P><B>Change of method for transformation</B></P><P>The pKD46 plasmid we are using has temperature sensitive replication. Heat shocking and incubating at 37 ºC is not an appropriate method for transformation. Electroporation will be used instead. The protocol for forming electrocompetent cells is as follows:</P><P><I>Preparation of stock</I></P><P>1.  Add 1 ml of the overnight culture to the eppendorf tube.</P><P>2. Centrifuge the culture at 5000g for 10 minutes , harvest the cells.</P><P>3. Resuspend the cells in 1 ml of 10% glycerol in LB .</P><P>4. Store the cells at -80 ºC.</P><P><I>Preparation of cells</I></P><P>At an OD of 0.6:</P><P>Centrifuge 1: transfer the cultures to ice-cold centrifuge bottles. Harvest the cells by centrifugation at 2000g for 20 minutes at 4 ⁰C. Decant the supernantant and resuspend the cell pellet in 50 ml of ice-cold sterilised water.</P><P>Centrifuge 2: harvest the cells by centrifuge at 2000g for 20 minutes at 4⁰C. Decant the supernatant and resuspend the cell pellet in 20 ml ice-cold sterilised water</P><P>Centrifuge 3: harvest the cells by centrifuge at 2000g for 20 minutes at 4⁰C . Decant the supernatant and resuspend the cell pellet in 1-2 ml ice-cold 10% glycerol. 
Store the cells at -80 ⁰C.</P><P><B>26/06/2013 - Electroporation of pKD46 in E.Coli BL21 DE3 from 25/06/2013</B></P><P>3 cuvettes prepared: each contained 50 µl of electrocompetent cells and either 1 µl of pKD46 plasmid, 5 µl of pKD46 plasmid or 5 µl of sterile H<SUB>2</SUB>O. The following protocol was followed:</P><P>1) Place the cuvettes on ice and put the cells in them</P><P>2) Add the plasmid/water to the cuvettes, tapping to remove any air bubbles</P><P>3) Electroporate using electroporation machine on bacteria setting</P><P>4) Add 750 µl of super optimal broth (SOB) to each cuvette and carefully transfer to sterile 1.5 ml eppendorf tube</P><P>5) Recover the cells at 30 ºC for 2 hours </P><P>Cells were then plated on 50 µg/ml ampicillin plates (100 µl or 600 µl of culture) and LB only plates (100 µl of culture) and incubated at 30 ºC overnight</P><P><B>27/06/2013 - Transformation Results</B></P><P> The following table shows the number of colony forming units (cfu) found on each plate. The number in brackets indicates number of colonies taken to be re-plated</P><TABLE border="1" style="background-color:#FFFFFF" width="100%" cellpadding="3" cellspacing="3"><TBODY><TR><TD>Plate Load</TD><TD>LB AMP-50 100µl </TD><TD>LB AMP-50 600 µl</TD><TD>LB 100 µl</TD></TR><TR><TD><I>E. coli</I> BL21 (DE3) &amp; 80 ng pKD46</TD><TD>33 (3)</TD><TD>69 (3)</TD><TD>Lawn</TD></TR><TR><TD><I>E. coli</I> BL21 (DE3) &amp; 400 ng pKD46</TD><TD>2 (2)</TD><TD>3 (3)</TD><TD>Lawn</TD></TR><TR><TD><I>E. coli</I> BL21 (DE3) &amp; 0 ng pKD46</TD><TD>0</TD><TD>0</TD><TD>Lawn</TD></TR></TBODY></TABLE><P><B>28/06/2013 - Further selection of Transformed cells and verification</B></P><P>Successful cultures from yesterday were re-plated</P><P><B>01/07/2013 - Continued</B></P><P><B><I>WHAT IS THIS MATHS????</I></B></P><P><B>02/07/2013 - Primer ordering</B></P><P> Order for primers put in today. They were as follows:</P><P><I>H1catF</I></P><P>5’<U>CCGTACATATAGTAAACCCCAACGCTACTGCTGCTTGTGCGTAAAATCTCCACTTCTT</U><I>TACCTCTTTTTTTAGTGACC</I> 3’</P><P><I>H1catR</I></P><P>5’<U>CTCTTTAGTGGGCGTCAAAAAAAACGCCGGATTAACCGGCGTCTGACGACTGACTTAACACT</U><I>TTACGCCCCGCCCTGCC</I> 3’</P><P><I>VerF1</I></P><P>5’ GGGTCGTCCACTAATACGGC 3’</P><P><I>VerfadR</I></P><P>5’ GGTGCCGCCGGTGTATTGTTGCAG 3’</P><P><I>VercatR</I></P><P>5’ GTGTAGAAACTGCCGGAAATCG 3’</P><P><I>VerR1</I></P><P>5’ GGGTTGTGGTAATTCGCC 3’</P><P><B>04/07/2013 - Induction of Lambda Red Recombinase</B></P><P>For each batch of cells, 2 batches were prepared. Some were electrically induced, and others were not</P><P>1)  70 ml of LB broth added to 700 µl of overnight culture of induced and uninduced cells</P><P>2) Grow cultures at 30 ºC for 1 hour </P><P>3) Check and record the OD until it is in a range of 0.1-0.3</P><P>4) Add 400 µl of arabinose to 10 ml of culture from step 1 to get a final concentration of 0.2%</P><P>5) Grow the culture at 30 ºC for a further 2 hours</P><P>6) Check and record the OD value until it lies in a range 0.6-0.9</P><P><B>09/07/2013 - PCR of Chloramphenicol and Homologous regions</B></P><P><I>Creation of a dNTP stock </I></P><P>dNTPs initially come at 100 mM concentration. Working stock requires 10 mM concentration. This was made by adding 10 µl each of dATP, dGTP, dCTP and dTTP to 60 µl of water</P><P><I>Creation of a Master Mix </I></P><TABLE border="1" style="background-color:#FFFFFF" width="100%" cellpadding="3" cellspacing="3"><TBODY><TR><TD>Item</TD><TD>Volume (µl)</TD></TR><TR><TD>5x phusion buffer</TD><TD>20</TD></TR><TR><TD>10 mM dNTPs</TD><TD>2</TD></TR><TR><TD>Primer 1 (10 mM stock)</TD><TD>2</TD></TR><TR><TD>Primer 2 (10 mM stock)</TD><TD>2</TD></TR><TR><TD>DNA/Water (Water control)</TD><TD>4</TD></TR><TR><TD>Phusion polymerase</TD><TD>1</TD></TR><TR><TD>DMSO</TD><TD>5</TD></TR><TR><TD>Water</TD><TD>64</TD></TR></TBODY></TABLE><P><I>PCR Thermocyler Settings</I></P><TABLE border="1" style="background-color:#FFFFFF" width="100%" cellpadding="3" cellspacing="3"><TBODY><TR><TD>Temperature (ºC)</TD><TD>Time</TD><TD>Number of cycles</TD></TR><TR><TD>98</TD><TD>2 m</TD><TD>1</TD></TR><TR><TD>98</TD><TD>15s </TD></TR><TR><TD>55</TD><TD>30s</TD><TD>10</TD></TR><TR><TD>72</TD><TD>3.5 min</TD></TR><TR><TD>98</TD><TD>15s</TD></TR><TR><TD>55</TD><TD>30s</TD><TD>20</TD></TR><TR><TD>72</TD><TD>3.5 min + 5s per cycle</TD></TR></TBODY></TABLE><P><B>10/07/2013 - Agarose Gel Electrophoresis of PCR product from 09/07</B></P><P><I>Making The Gel</I></P><P>1%  gel we require 1 g Agarose in 100 ml TAE.
Therefore, 0.7% gel we require 0.7g agarose in 100 ml TAE</P><P>1. Microwave 1-3 min unless agarose is dissolved in TAE and rolling boil of mixture</P><P>2. Leave to cool 5 mins. Add 2 ul of Ethidium Bromide per 100 ml </P><P>3. Pour into casting plates with comb</P><P>4. Leave 10-15 mins at 4⁰C or 20 mins at room temperature</P><P>Result of PCR: FAILURE</P><P><B>INSERT PICTURE OF GEL HERE</B></P><P><B>11/07/2013 - Received FAS Module - Plating up</B></P><P>2 vials of stab culture received from Prof Mattheos Koffas of the Rensselaer Polytechnic Institute, NY, USA (2x DH5α w/pETM6-CnFatB2-fabAH<SUP>*</SUP>GI in Amp80</P><P>1 vial kept at 4 ºC and other vial plated on LB with Amp100</P><P>Left on bench for 3 hours and then placed in a 30 ºC incubator</P><P>Second plate, Amp 50µg/ml, plated w/DH5α w/pETM6-CnFatB2-fabAH<SUP>*</SUP>GI  and placed in 30 ºC incubator</P><P>Both plates removed and left on bench overnight</P><P><B>12/07/2013 - FAS plate results</B></P><P>Very little growth. Allowed to grow over the weekend</P><P><B>15/07/2013 - Stock of FAS cells for freezer</B></P><P>Successful growth found on the plates over the weekend. Stock made using the following protocol:</P><P>1) Make 10 ml of LB and 100 µg/ml of Ampicillin</P><P>2) Inoculate media with cells on plates w/FAS</P><P>3) Grow at 37 ºC for 5 hours</P><P>4) Spin cells down in centrifuge</P><P>5) Resuspend in 700 µl of LB and glycerol</P><P><B>18/07/2013: Growing up DH5α for plasmid extraction</B></P><P>1) 100 ml of LB w/100 µg/ml Ampicillin</P><P>2) Inoculate with cells</P><P>3) Leave to grow at 37 ºC overnight</P><P><B>18/07/2013: Extraction of FAS module</B></P><P>FAS module successfully extracted using plasmid purification kit</P><P><B>INSERT PICTURE OF GEL HERE</B></P><P><B>19/07/2013-24th July: New Primers</B></P><P>Found that the PCR from 9/7/2013 failed due to incorrect primers (H1catF and H2catR). New primers were ordered:</P><P><I>H1catF_2a</I></P><P>CGGCATGTATATCATTTGGGGTTGCGATGACGACGAACACGCATT</P><P><I>H1catR_2a</I></P><P>CTCTTTAGTGGGCGTCAAAAAAAACGCCGGATTAACCGGCGTCTG</P><P><B>24/7/2013 - Knockout Take II PCR of Chloramphenicol (New primers)</B></P><P>PCR ran with new primers. To be on the safe side we ran a PCR gradient in order to find the best annealing temperature. A rough estimate of the optimum T<SUB>m</SUB> was made using the T<SUB>m</SUB> calculator on the NEB site</P><P><B>25/7/2013 - Results of PCR</B></P><TABLE border="1" style="background-color:#FFFFFF" width="100%" cellpadding="3" cellspacing="3"><TBODY><TR><TD>Sample Number</TD><TD>Temperature (ºC)</TD><TD>Band?</TD></TR><TR><TD>1</TD><TD>49.8</TD><TD>No</TD></TR><TR><TD>2</TD><TD>50.2</TD><TD>No</TD></TR><TR><TD>3</TD><TD>51.4</TD><TD>Yes (with mystery product)</TD></TR><TR><TD>4</TD><TD>53.5</TD><TD>Yes</TD></TR><TR><TD>5</TD><TD>56.4</TD><TD>No</TD></TR><TR><TD>6</TD><TD>58.5</TD><TD>No</TD></TR><TR><TD>7</TD><TD>59.6</TD><TD>No</TD></TR><TR><TD>8</TD><TD>59.7</TD><TD>No</TD></TR></TBODY></TABLE><P><B>26/7/2013 - Chloramphenicol resistance gene extraction</B></P><P>Tuc01 strain of <I>E. coli</I> used for third attempt at fadD knockout</P><P>Method used was the QIAGEN method for genome extraction</P><P>Analysis using nanodrop: 480 ng/µl of DNA</P><P><B>29/7/2013 - Making chloramphenicol stock</B></P><P>Want a 34 mg/ml stock in 100% ethanol: 0.34 g in 10 ml of ethanol</P><P><I>Making a subculture of Tuc01 strain for DNA extraction:</I>&lt;p&gt;&lt;p&gt;1. Add 600 µl of Tuc01 in 5ml of Cm<SUB>10</SUB> and LB</P><P>2. Put the tube in 37 ºC shaking incubation</P><P>3. After 5 hours, transfer 1 ml from the tube to 9 ml LB Cm <SUB>10</SUB>&lt;p&gt;4. Culture overnight</P><P><B>1/8/2013 - Making FAS module media</B></P><P>Due to the high levels of fatty acid production in bacteria containing the FAS module, a new media is required to keep the cells alive:</P><P><I>1. Making the growth media (without Glucose stock)</I></P><P>10 g Yeast Extract
&lt;p&gt;4 g (NH<SUB>4</SUB>)2HPO<SUB>4</SUB> - Diammonium Phosphate
&lt;p&gt;13.5 g KH2PO4 -Mono potassium phosphate</P><P>1.7 g Citric Acid</P><P>Dissolve in approx 300ml DI water, add more if required. </P><P>Send for Auto-Clave</P><P>*Also send water for autoclave as this will be required in the later step if there is not sufficient water autoclaved. You will need approx 700 ml water max</P><P><I>2. Making the glucose stock</I></P><P>20 g glucose dissolved in approx 100ml DI water, add more if required.</P><P><I>3. Prepare the trace metal stock</I></P><P>You will eventually add 10 ml of this stock, these volumes are required to make a 100 ml stock.&lt;p&gt;&lt;p&gt;1.0 g FeSO<SUB>4</SUB>.7H<SUB>2</SUB>O</P><P>0.2 g CaCl<SUB>2</SUB></P><P>0.22 g ZnSO<SUB>4</SUB>.7H<SUB>2</SUB>O</P><P>0.05 g MnSO<SUB>4</SUB>.4H<SUB>2</SUB>O</P><P>0.1 g CuSO<SUB>4</SUB>.5H<SUB>2</SUB>O</P><P>0.01 g (NH<SUB>4</SUB>)6Mo<SUB>7</SUB>O<SUB>2</SUB>4.4H<SUB>2</SUB>O - Ammonium molybdate hydrate</P><P>0.002 g Na<SUB>2</SUB>B<SUB>4</SUB>O<SUB>7</SUB>.10H<SUB>2</SUB>O (BORAX) - Sodium Borate</P><P>4. Filter sterilisation of metal stock and glucose stock (separately)</P><P>5. Mix filtered stocks with autoclaved growth media.</P><P>6. Add NaOH to make to pH6.8</P><P>7. Make up to 1 Litre with sterilised water</P><P>Ran a gel of chloramphenicol resistance gene: successful band</P><P>INSERT PICTURE OF GEL HERE</P><P>However gel extraction failed: 5ng/ul, 280/260 of 6. Ran an overnight repeat PCR</P><P><B>2/8/2013-5/8/2013 - Chloramphenicol PCRs</B></P><P> Repeated PCR described previously another two times, both failed. Discovered incorrect PCR settings. Repeated and got successful band:</P><P>INSERT PICTURE OF GEL HERE</P><P>Performed PCR purification (QIAGEN). Resulted in 16.9ng/ul of DNA. Electroporated cells and plated. Cells failed to grow</P><P><B>8/8/2013 - Cell growth curve NEED TO DO THIS TABLE</B></P><P><B>12/8/2013 - RBS Biobrick Hydration</B></P><P>Hydrated 3 ribosomal binding site biobricks: BBa_B0034, BBa_B0034, BBa_B0032, and transformed into chemically competent NovaBlue cells. They were plated on LB-AMP plates</P><P><B>9/08/13 Primers recieved for FabA</B></P><P>FabA_Fwd_EcoRI</P><P>CTAGTAGAATTCATGGTAGATAAACGCGAATCCT</P><P>FabA_Rev_Spe1</P><P>CTAGTACTGCAGTTATTAGAAGGCAGACGTATCCTGG</P><P>FabA_Fwd_Prefix</P><P>GAATTCGCGGCCGCTTCTAGATGGTAGATAAACGCGAATCCT</P><P>FabA_Rev_Suffix</P><P>CTGCAGCGGCCGCTACTAGTATTATTAGAAGGCAGACGTATCCTG</P><P>FabA_Fwd_Prefix_N-His</P><P>GAATTCGCGGCCGCTTCTAGATGCATCATCACCACCACCATGTAGATAAACGCGAATCCT&lt;/p
&lt;p&gt;FabA_Rev_Suffix_C-His</P><P>CTGCAGCGGCCGCTACTAGTATTATTAATGGTGGTGGTGATGATGGAAGGCAGACGTATCCTGG</P><P><B>13/08/13 - Finding the failure for the FadD knockout</B></P><P>Agarose Gel for confirmation the presence of DNA in pKD46</P><P>RESULT: Success</P><P>Insert gel Picture
</P><P>Conclusion: Electrocompetent cells are no longer competent</P><P><B>FadD knockout all over again</B></P><P>Result PCR of Chloramphenicol DNA</P><P>RESULT: Obtained the desired band. Non-template control was contaminated, showing a band same the chloramphenicol gene</P><P>Insert the gel picture
</P><P><B>14/8/13- Primers received.</B></P><P>Delta9_Rev</P><P>CTGCAGCGGCCGCTACTAGTATTATTAGGCTTTGTTGGCCATCGCAGTT (49) </P><P>Delta12_Rev</P><P>CTGCAGCGGCCGCTACTAGTATTATTAAACTTTTTTCAGGGAGCCGAAG (49)</P><P>Delta9 &amp; Delta12_F</P><P>GAATTCGCGGCCGCTTCTA (19)</P><P>sSB1C3_VF2</P><P>TGCCACCTGACGTCTAAGAA (20)</P><P>sSB1C3_VR</P><P>ATTACCGCCTTTGAGTGAGC (20)</P><P>FabA_Fwd</P><P>TGGTAGATAAACGCGAATCCT (21)</P><P>FabA_Rev</P><P>TTATTAGAAGGCAGACGTATCCTGG (25)</P><P><B>PCR for FabA</B></P><P>A full scale 3 hour PCR was run after the confirmation of the product. The primers used in this PCR contained His-tags.</P><P><B>FadD knockout Continued</B></P><P>Transforming pKD46 in BL21(DE3)</P><P>1.Overnight culture at 30°C</P><P>2.Measure the OD<SUB>600</SUB>. It was in the range of 0-0-1</P><P>3.Keep in 30°C shaking incubator for 2 hrs</P><P>4.Measure the OD again and were in the range of 0.2-0.6</P><P>5.Store in 4°C</P><P>6.Follow the protocol for making the cells Electrocompetent as on 25/6/13</P><P><B>DNA precipitation of chloramphenicol gene product on 13/8/13</B></P><P>Ethanol precipitation
&lt;p&gt;1. 2 volumes absolute ethanol (ice cold) and 1/10 volumes sodium acetate (pH 5.2)</P><P>2. Keep for an hour at -80°C</P><P>3. Centrifuge at 5417 rpm for 30 minutes at 4°C</P><P>4. Wash with 70% ethanol</P><P>5. Repeat the centrifuge as step 3</P><P>6. Nanodrop was performed scaling up to 52 ng/ul</P><P><B>Restriction digest of chloramphenicol gene</B></P><P> The restriction digestion was performed using the enzyme EcoRI.</P><TABLE border="1" style="background-color:#FFFFFF" width="100%" cellpadding="3" cellspacing="3"><TBODY><TR><TD><B>Item</B></TD><TD><B>Sample</B></TD></TR><TR><TD>1</TD><TD>2</TD><TD>3</TD></TR><TR><TD>DNA</TD><TD>4µl</TD><TD>1.2µl</TD><TD>2µl</TD></TR><TR><TD>Buffer</TD><TD>1µl</TD><TD>1µl</TD><TD>1µl</TD></TR><TR><TD>EcoRI</TD><TD>0.5µl</TD><TD>0.5µll</TD><TD>0.5µl</TD></TR><TR><TD>Water</TD><TD>4.5µl</TD><TD>7.3µl</TD><TD>6.5µl</TD></TR></TBODY></TABLE><P>The samples were run on 2% gel</P><P>insert the picture of gel 
</P><P> RESULT: Success</P><P><B>15/8/13- Result of FabA PCR of 14/8/13</B></P><P>Insert gel picture 
</P><P>RESULT: Successful. Band size 600bp</P><P>FabA restriction digestion</P><P> Enzyme used Apo1</P><P>The restriction digest was performed at 50°C for an hour</P><P>RESULT: Successful. Bands size were 400 and 200 bp respectively</P><P> FadD knockout continued</P><P> Followed the protocol as on 25/08/13</P><P><B>16/9/13- Induction of lamda red recombinase and making competent cells</B></P><P>Followed the protocol as on 04/07/13</P><P><B>19/08/13-Electroporation of choramphenicol gene in BL21(DE) with lamda red.</B></P><P>Followed the protcol as on 25/08/13</P><P> RESULT: FadD knockout failed.</P><P><B>20/08/2013 - Electroporation of RBS Biobricks and RBS+ Constitutive promoter. </B>
We decided to use existing BioBrick parts for our Ribosomal Binding sites for FabA, Delta 9, Delta 12 - and also the RBS and promoter Biobrick. We intend on using the RBS+Promoter biobrick for expression of our FabA, Delta9 and Delta 12 genes in our lab work <B>Batches </B>
RBS 1 = BBa_B0034
RBS 2 = BBa_B0030 
RBS 3=  BBa_B0032 
RBS + P =  BBa_K608002</P><TABLE border="1" style="background-color:#FFFFFF" width="100%" cellpadding="3" cellspacing="3"><TBODY><TR><TD><B>Batch</B>/td&gt;</TD><TD><B>Electrode Values</B></TD></TR><TR><TD>RBS1</TD><TD>4.6</TD></TR><TR><TD>RBS1 Control (No DNA)</TD><TD>5.1</TD></TR><TR><TD>RBS2</TD><TD>5.2</TD></TR><TR><TD>RBS2 Control</TD><TD>5.2</TD></TR><TR><TD>RBS3</TD><TD>5.0</TD></TR><TR><TD>RBS3</TD><TD>5.4</TD></TR><TR><TD>RBS/Promotor</TD><TD>5.2</TD></TR><TR><TD>RBS/Promotor Control</TD><TD>5.2</TD></TR></TBODY></TABLE><P>
Cells then left to recover for 3 hours at 37 degrees Celcius and then plated on appropriate marker plates (LB+Chromamphenicol at a concentration of 10 µg/ml) and left overnight<B> 20/08/2013 - FabA PCR Products (from 15/08/2013) - Blunt End Ligation  </B>
We performed a blunt end ligation on the FabA PCR Products from 15/08/2013 using the Thermo Scientific CloneJET PCR Cloing Kit, and transformed chemically competent NovaBlue Cells at 37 degrees celsius for 2 minutes. The cells were left to recover for 3 hours at 37 degress celcius and plated on appropriate antibiotic plates (LB + Ampicillin 100µg/ml) overnight at 37 degrees celcius. </P><P><B>21/08/2013 - Growing up of FabA Clone Jet Transformed cells in media &amp; Mini Prep performed </B>
1.The CloneJet Transformed cells were selected and grown up in LB + Ampicillin for 6 hours. 
2.A miniprep was performed using Qiagen Miniprep Kit. 
3. We completed a test digestion with ApoI, EcoR1 and Pst1 - same protocol as 15/08/13 confirming the identity of FabA His tag in the N Terminus</P><P><B>27/08/2013 - Transformation of Delta9 and Delta 12 synthesised plasmids </B> 
1.The Delta 9 and Delta 12 Synthesised genes arrived, and were transformed using NEB High Efficiency DH5 alpha chemically competent cells. *37 Degress Celcius for 2 minutes* 
2. FabA Blunting reaction with CloneJet was repeated and also transformed with NEB DH5 alpha cells. *37 Degress Celcius for 2 minutes* 
3. All of the transformed cells were left to recover for 3 hours and then plated on appropriate LB plates</P><P><B> 28/08/2013 - Inoculation of LB + AMP with transformants from 27/08/2013 </B> 
1. Inoculation in preparation for MiniPrep the following day</P><P><B> 29/08/2013 - Miniprep of FabA, D9, D12 </B>
1. Miniprep on the three grown up cell colonies from 28/08/2013 was carried out. 
2. Test digestion was completed with EcoR1 and Pst1 to confirm identity.
3. Large scale overnight digestion was carried out with EcoR1 and Pst1 to obtain the DNA for Submission Vector and RBS + P biobrick ligation for experimental work. <B> 03/09/2013 - Transformation of NEB-5a with BBA_K608002. </B>
1. Cells were transformed and plated on LB + Chloramphenicol 10 microgram/ml 
2. Single colonies taken and inoculated the next day in preparation for miniprep </P><P><B> 05/09/2013 - Miniprep of Colonies in LB from 04/09/2013 and digestion</B>
1. A Miniprep was carried out using the Qiagen MiniPrep Kit 
2. A digestion using EcoR1 and Pst1 was completed on the BioBrick Vector to linearise the fragment for ligation with D9/D12 and FabA  and gel ran to confirm the correct size fragments were obtained. Result - Success. </P><P><B> 06/09/2013 - Gel Extraction of digested BBa_K608002 (RBS + P) from 05/09/2013 </B>
Products from digestion from 05/09/2013 were run on a Gel and Extracted using the Qiagen QiaQuick Gel Extraction Kit. The Elution buffer used was heated to 40 degrees celsius to improve eluted DNA concentration. </P><P><B> 09/09/2013 - Gel Extraction of digestion products from 29/08/2013 </B>
1. A gel was run with the products from the digestion on 29/08/2013 to confirm the size of the required products
2. A gel extraction was then performed to extract the D9, D12, FabA fragments using the Qiagen Qiaquick Gel Extraction kit.<B> 12/09/2013 - Ligation of D9, D12, FabA into Submission Vector </B>
The Extracted products from 09/02/2013 and digested Submission Vectors were ligated using NEB T4 DNA Ligase protocol. <B>13/09/2013 - Test digestion of Submission vector with D9, D12 </B>
Following Ligation of 09/0/2013 a test digestion was carried out as a preliminary method of confirming the constructs produced.
1. D9+SUBMISSION VECTOR was digested with BamHI and EcoRV. Result - Anticipated fragments present, Success. 
2. D12+SUBMISSION VECTOR  was Digestions with BamHI and Pvu1 - Failed digestion, this was a result of the wrong Enzyme buffer used.
3. D12+SUBMISSION VECTOR Digestion repeated  with correct restriction enzyme buffers and new restriction enzymes (BamHI, EcoR1, Pst1 )and gel repeated - Gel confirmed correct fragment size, Success. 
4. As our Submission Vectors with D9 and D12 constructs appeared to be present these plasmids were sent for sequencing with the iGEM Verification primers. </P><P><B>20/09/2013 - FabA test digestion </B>
1. NEB Enzymes EcoRV and Pst1 were used to digest the FabA and submission vector ligation from 13/09/2013 - Results proved this to be successful. 
2. FabA construct sent for sequencing </P><P><B>22/09/2013 - Ligation of Into GRBS + P (BBa_K608002) vector from 05/09/2013 </B>
The extracted delta 9, 12 and fabA products from 09/09/2013 were ligated into the RBS + P biobrick vectors for expression in the laboratory experiments. <B>23/09/2013 - Test digestion to confirm correct ligations from 22/09/2013 </B>
1. Digestions were carried out to confirm the identity of the Expression vectors with our D9/D12/FabA inserts using NEB restriction enzymes - 
(D9 - EcoRV and BamHI) -&gt; Success
(D12 - BamHI, Pst1, Xba1) -&gt; Success
(FabA - EcoRV and Pst1) -&gt; Success 
Samples sent to LC-MS for characterisation
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Manchester/LabBookText">http://2013.igem.org/Team:Manchester/LabBookText</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Manchester/LabBookText" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Manchester/LabBookText" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Manchester/LabBookText&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Manchester/LabBookText&amp;oldid=253410" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>