Team:HUST-China/Demonstrate
<!DOCTYPE html>
Team:HUST-China/Demonstrate - 2018.igem.org
Loading menubar.....
Team:HUST-China/Demonstrate
HillSide Multi purpose HTML5 Template
Home
PROJECT
Background
Description
Applied Design
Safety
Demonstrate
WETLAB
Experiments
Results
Protocols
Interlab
Notebook
Model
Modeling overview
Model of systems
Sort of three genes
Software
PARTS
Parts Overview
Composite part
Part collection
HP
Human Practices
Education&Engagement
TEAM
Team Members
Collaborations
Attributions
Demonstrate
Part1: 1.Photosynthetic microorganism system
1.1 Synechocystis sp. PCC6803
Construction of genes
Since synechocystis itself lacks a pathway for producing lactate, and as a photoautotrophic microorganism, synechocystis lacks a lactate transporter to transport lactate out of the cell[1]. Therefore, we combined the lactate dehydrogenase gene with the lactate transporter gene in one circuit to achieve lactate production and transportation. For lactate dehydrogenase gene, we chose ldhD, and ldhDc is a codon-optimized version of ldhD, ldhDnARSdR is ldhD with D176A/I177R/F178S/N180R, and ldhDARSdR is the codon-optimized version of ldhDnARSdR. These codon optimizations are aim at increasing the production of lactate. The lldP ​​protein gene is used to transport the lactate out of the cell. We successfully construct genes ldhD-lldp, ldhDC-lldp, ldhDnARSdR-lldp and ldhDARSdR-lldp.
Figure 1. PCR of ldhD-lldp expression and ldhDC-lldp expression
Figure 2. PCR of ldhDnARSdR-lldp expression and ldhDARSdR-lldp expression
1.2 Synechocystis sp. PCC6803
Verification of gene expression
Detection of yellow fluorescent protein
And all the genes were constructed into the plasmid pck306 and transformed into synechocystis. Since pck306 plasmid contain the gene of yellow fluorescent protein, so we put engineered cynobacteria under the fluorescence microscope to verify the transformation in cynobacteria.
Figure 3. Results of yellow fluorescent protein of pck306 plasmid in Synechocystis sp. PCC6803. L-Rhamose(1g/L) was added and after 36 hours, engineered bacteria showed it expressed successfully.
We could draw the conclusion that these genes were successfully constructed into engineered Synechocystis sp. PCC6803.
Affinity Chromatography
In addition to knowing transcription of the gene, we want to exhibit the expression of the protein, we insert 6Xhis-tag into lldP and did affinity chromatography to show we finally made it. The result is shown below:
Figure 4a. There were nearly 0.2g bacteria used to testify the expression of lldP. After affinity chromatography, the protein was electrophoresed through SDS-PAGE. As the red box shows, lldP can be seen . Comparing to wild type, lldP is expressed in Synechocystis sp. PCC 6803.
Because of the slow growth of Synechocystis sp. PCC 6803 and time limitation, we did not have too much bacteria, so we only use nearly 0.2g of them to show whether lldP expressed. The color is pale but still can be seen. lldP expression succeeds.
Real-Time Quantitative PCR
To illustrate our project does work. We must prove that our circuit express successfully. Therefore, we did real-time quantitative PCR to demostrate the transcription of the circuit ldhD-lldP, ldhDc-lldP, ldhDnARSdR-lldP and ldhDARSdR-lldP. The result shows as the figure below:
Figure 4b. The figure shows that ldhD-lldP, ldhDc-lldP, ldhDnARSdR-lldP, ldhDARSdR-lldP express successfully.
Lactate Detection
Finally, to testify how the whole circuit in Synechocystis sp. PCC 6803(synechocystis) works, we did lactate detection experiment. After adding L-Rhamose(1g/L) for 72h, we use Lactic Acid assay kit, provided by Nanjing Jiancheng Bioengineering, to quantify lactate concentration.
Figure 5. Lactate production of engineered Synechocystis sp. PCC6803. Comparison of WT, ldhD-lldP, ldhDc-lldP, ldhDnARSdR-lldP, ldhDARSdR-lldP. Synechocystis grew for 7 days, then L-Rhamose(1g/L) was added. After induced 72 hours, lactate concentration has shown above.
The figure indicates that after transforming our circuit in bacteria, the production and releasing of lactate increase evidently, and ldhDARSdR-lldP is the most efficient one. In summary, our engineered bacteria, Synechocystis sp. PCC 6803(synechocystis), do achieve our goal to provide lactate to Shewanella.
1.3 Rhodopseudomonas palustris
We try to use mleS (convert pyruvate to D-lactate), ldhA (convert pyruvate to D-lactate) and lldP (L-lactate permease) to make sure their strains could produce lactate. We construct genes into the shuttle plasmid pMG105 in E.coli BL21, and then detect it .
For example of pMG105-PpckaA-RBS -lldP-RBS-ldhA-TT (Figure 3).
Figure 6. shows gene circuits are successfully constructed into E.coli BL21.
Real-Time Quantitative PCR
we do Real-Time Quantitative PCR to verify the expression of gene circuits by E.coli BL21.
(A)
(B)
(C)
Figure 7.Relative expression level of targeted genes in E.coli BL21. We choose mleS(malate dehydrogenase, the conversion of malic acid to L-lactate), lldP(L-lactate permease, the lactate is transported out of the cell) and ldhA(fermentative D-lactate dehydrogenase, NAD-dependent, convert pyruvate to D-lactate) as the reference genes and pMG105 as the standard quantity. (A). The expression level of mleS. (B). The expression level of lldP. (C). The expression level of ldhA. We could find that mleS, lldP and ldhA have a higher expression compared with pMG105. It means every genes can express efficiently.
It shows the verification of gene expression is correct. Gene mleS, lldP and ldhA have a good expression on mRNA level.
Lactate Detection
We detect lactate production of gene circuit in E.coli BL21 (Figure8)
Figure 8.LB is a kind of medium to cultivate bacterial. We detect lactate content of LB and others. PMG105 has the lowest lactate content because pMG105 doesn’t have any gene to help produce lactate. It proves that all of our gene circuits have ability to produce lactate. Besides, compared with ldhA and lldP, ldhA-lldP has a better production efficiency;  compared with ldhA-lldP, lldP-ldhA has a better production efficiency; compared with ldhA-lldP, mleS-lldP has a better production efficiency; compared with other gene circuits, mleS-lldP-ldhA has a better production efficiency. The result proves our modeling is correct.
Part2: Electrogenic microorganism system
2.1 Plasmid Construction and Conjugation
Since lactate is the carbon source of Shewanella Oneidensis MR-1, overexpression of lactate dehydrogenase might help the bacteria to utilize lactate more efficiently and produce more electricity. We chose dld(encodes D-lactate dehydrogenase), lldEFG(encodes L-lactate dehydrogenase) gapA(encodes glyceraldehyde-3-phosphate dehydrogenase), mdh(encodes NAD-dependent malate dehydrogenase), pflB(encodes pyruvate formate-lyase) and fdh(encodes formate dehydrogenase) to overexpress in Shewanella Oneidensis MR-1. All the genes were constructed into the shuttle plasmid pYYDT, which were then conjugated into Shewanella Oneidensis MR-1.
Figure 9.Verification of successful conjugation. (A). pYYDT-dld and pYYDT-lldEFG. (B). pYYDT-gapA-mdh. (C). pYYDT-pflB-fdh.
We could draw the conclusion that these genes has been successfully constructed into engineered Shewanella Oneidensis MR-1.
2.2 Verification of expression
(1). RT-qPCR
RT-qPCR(Real-Time quantitative PCR) is a technique of molecular experiment. The expression of genes could be detected by measuring the number of copies of an RNA transcript of that gene in engineered cells, which means that by doing RT-qPCR, we could demonstrate that the related genes could be expressed by engineered bacteria.
Figure 10. Relative expression level of targeted genes in engineered Shewanella Oneidensis MR-1.We chose gyrB(encodes DNA gyrase B) as the reference gene and 1 as the standard quantity. (A). The expression level of pYYDT-dld. (B). The expression level of pYYDT-gapA-mdh. (C). The expression level of pYYDT-pflB-fdh.
As shown by figure 10, it is demonstrated that these genes could be expressed in engineered Shewanella Oneidensis MR-1.
(2).Electrogenesis
Improvement about the efficiency of electricity production is our final goal. We compared the Shewanella contained targeted genes with the Shewanella with no-load vector pYYDT. If the former one could produce more electricity, our construction could be proved to receive a success.
Part3: Whole design
There may be more potential interactions in the co-culture of Rhodopseudomonas palustris and Shewanella, which can greatly improve the coulombic efficiency of our MFC system.
Figure 11. The interaction between Rhodopseudomonas palustris and Shewanella.
Figure 12.The comparison of electricity production between wild type Rhodopseudomonas palustris and type Synechocystis PCC6803.
Figure 13.The comparison between Shewanella with no-load vector and Shewanella with targeted genes.
We could learn from this figure that constructed bacteria could produce more electricity, which means that the targeted genes have been expressed successfully in the cell and helped to produce more electricity.
College of Life Science & Technology
Huazhong University of Science and Technology
Add: 1037 Luoyu Road, Wuhan, Hubei, China; P.C: 430074
Copyright ©2018 Huazhong University of Science & Technology. Produced By HUST-China
