Team:HUST-China/wetlab/protocols
HillSide Multi purpose HTML5 Template
Home
PROJECT
Background
Description
Applied Design
Safety
Demonstrate
WETLAB
Experiments
Results
Protocols
Interlab
Notebook
Model
Modeling overview
Model of systems
Sort of three genes
Software
PARTS
Parts Overview
Composite part
Part collection
HP
Human Practices
Education&Engagement
TEAM
Team Members
Collaborations
Attributions
Protocols
Part1: Bacteria and culture medium
1.1 E.coli TOP10, DH5α, BL21, WM3064
1.1.1 LB (Luria-Bertani) medium
1% NaCl 1% Tryptone 0.5% Yeast Extract
1.2 Rhodopseudomonas palustris(DSM5859)
1.2.1 culture medium[1]
Component
Amount
CH3COONa
3.8g/L
NH4Cl
1.3g/L
Yeast extract
0.7g/L
Microelement
0.8Ml/L
—FeCl3·6H2O
5mg/L
—CuSO4·5H2O
0.05mg/L
—H3BO3
41mg/L
—MnCl2·4H2O
0.05mg/L
—ZnSO4·7H2O
1mg/L
—CO(NO3)2·6H2O
0.5mg/L
1.3  Synechocystis sp. PCC6803
1.3.1 BG11(Blue-Green medium)[2]
Component
Amount
Stock Solution
NaNO3
10mL/L
15.0g/100mLdH2O
K2HPO4
1mL/L
4g/1O0mLdH2O
MgSO4 ·7H2O
1mL /L
7.5g/100mLdH2O
CaCl2 ·2H2O
1mL/L
3.6g/1 00mLdH2O
Citric acid
1mL /L
0.6g/100mLdH2O
Ferric ammonium citrate
1mL/L
0.6g/100mLdH2O
EDTANa2
1mL/L
0.1g/100mldH2O
Na2CO3
1mL/L
2.0g/100mldH2O
H3BO3
2.86g/LdH2O
MnCl2·4H2O
1.86g/LdH2O
ZnSO4·7H2O
0.22g/LdH2O
Na2MoO4·2H2O
0.021g/LdH2O
CuSO4·5H2O
0.08g/L dH2O
Co(NO3)2·6H2O
0.05g/LdH2O
Adjust PH to 7.1 with 1M NaOH or HCl
1.4 Shewanella oneidensis MR-1
1.4.1 LB medium[2]
1% NaCl 1% Tryptone 0.5% Yeast Extract
1.4.2 M9 medium[3]
Component
Amount
1 M MgSO4
2mL
20% Glucose
20mL
1 M CaCl2
0.1mL
5×M9 Salt solution
200mL
Na2PO4·7H2O
12.8g
KH2PO4
3.0g
Nacl
0.5g
H2O
200mL
Add H2O up to 1000mL in total
Part2: Cloning method
2.1 PCR reaction system
Component:
Volume:
PrimeSTAR Max
12.5μl
Template DNA
1μl
Primer-F
1μl
Primer-R
1μl
ddH2O
up to 25 μl
Total:
25μl
Step：
Temperature:
Time:
Cycle:
Hot
94℃
2min
1
Denaturation
94℃
1min
25
Annealing
55℃
1min
25
Extension
72℃
1min
25
Final Extension
72℃
7min
1
2.2 Double enzyme digestion system
Component:
Volume:
Universal Buffer
5μl
Enzyme 1
1.5μl
Enzyme 2
1.5μl
ddH2O
up to 25 μl
Total:
50μl
Coditions
37℃ 30~40min
2.3 Ligation system
Component:
Volume:
T4 ligase
1μl
T4 ligase buffer (1x)
1μl
Linearized Vector
0.5μl
Insert Gene
4.5 μl
2.4 Reaction system of colony PCR
Component:
Volume:
rTaq Mix(2×)
5μl
Primer-F(10μM)
0.3μl
Primer-R(10μM)
0.3μl
ddH2O
4.4μl
Toal:
10μl
2.5 Transformation into E. coli TOP10/DH5α/BL21/WM3064
The ligation product was transformed into E. coli DH5a Strain. The strains were cultivated in LB plate medium containing antibiotic at 37℃.
2.6 Transformation into Synechocystis sp. PCC6803
1. Take 2 ml of Synechocystis solution when the O.D. is 0.8, and centrifuge for 15 minutes at 5800 rpm. 2.  Discard the supernatant and dissolve the precipitate with the new BG11 medium. 3.  Add the solution into 50 ml small conical flask with 30 μl plasmid. 4.  Add the new BG11 medium gradually to the final concentration of 5 ml. 5.  Put in light culture box. 6.  Expand the culture to 50ml at last, when the color is deep enough. 7.  Take part of them to test for protein expression.
2.7 Detection of yellow fluorescent protein
Cultivate the engineering cynobacteria induced by rhamnose for 24h and put them under the fluorescence microscope at the wave of 488nm excitation light.
1.Making electrocompetent cells for Rhodopseudomonas palustris:
a. Inoculate 10% of Rhodopseudomonas palustris DSM5859 into LB medium (2 ml) and incubate at 37 °C for 8 to 24 hours. b. Place the bacterial solution in an ice bath for 30 minutes. c. Wash with cold ddH2O   and refrigerated centrifuge for 7 to 8 times to ensure the temperature of the bacteria below 4 °C. d. Add 10% glycerol and store at -80 °C after the last centrifugation.
2.Electrotransformation for Rhodopseudomonas palustris DSM5859:
a. Mix 50 μl of electocompetent cells with 10 to 20 μl of plasmid in an ice bath for at least 1 minute. b. Move into an electric rotor (2mm pre-cooling) and use the Gene Pulser XcellTM Total System (BIO-RAD) (400 Ω, 2.5 kV, 25 μF, 12.5 kV/cm field strength). c. Add 1 ml of LB medium immediately after the electric shock. d. Mix the bacterial solution in the electric rotor with the culture medium and transfer it into the EP tube. e. Incubate for 2-6 hours at 37 °C, apply it to the kanamycin resistant culture plate. f. Culture in dark conditions.
2.8 Making competent cells of E.coli WM3064
1. Inoculate wm3064 into DAP-LB, cultivate the bacteria at 37°C, 200rpm for 6~8 hours. 2. Streak the bacteria on DAP-LB plate medium at 37°C. 3. Pick a single colony and cultivate in 10mL DAP-LB overnight. 4. Transfer 10mL bacterium solution into conical flask with 100mL DAP-LB and cultivate the bacteria. Put the conical flask in the ice when OD600 increases to 0.4~0.6. 5. Put CaCl2-glycerol solution, two 50mL sterilized centrifuge tubes and some 1.5mL EP tubes in the ice. 6. Divide the bacterium solution into to 50mL sterilized centrifuge tubes. 7. Centrifuge at 4°C, 4000rpm for 8 minutes. Discard the supernatant. 8. Add 30mL CaCl2 and resuspend. 9.Repeat 7 and 8 for 4 times. 10.Add 2mL CaCl2- glycerol solution and resuspend. Divide the bacterium solution into 1.5mL EP tubes. Store the competence at -80°C.
2.9 Transformation into E.coli WM3064 (2,6-diaminopimelic acid deficiency type)
1. Add 50μl competence wm3064 and 1μl plasmid into an EP tube. 2. Put the tube in the ice and incubate the cell for 30 minutes. 3. Take a heat shock (42°C) for 80 seconds. 4. Take an ice-bath for 3~5 minutes. 5. Add 300μl DAP-LB into the EP tube and incubate the cell at 37°C, 200rpm for 40 minutes. 6. Add 200μl solution in a plate with kanamycin and DAP. Cultivate these bacteria overnight.
2.10 Conjugation into Shewanella oneidensis MR-1
1. Inoculate Shewanella MR-1 and WM3064 (possess the targeted plasmid). 2. Add 500μl Shewanella MR-1 and WM3064 into a 1.5mL EP tube, mixing the bacterium solution. 3. Centrifuge the solution at 5000rpm for 10 minutes. Discard the supernant. 4. Resuspend the bacteria with 1000mL DAP-K+-LB. Take a 30°C water bath for 1 hour. 5. Add 200μl solution in a plate with kanamycin.
Part3: Verification of gene expression
3.1 Real-Time Quantitative PCR
1. Normalize the primer concentrations, mix gene-specific forward and reverse primer pair. Each primer (forward or reverse) concentration in the mixture is 5 pmol/μl. 2. Set up the experiment and the following PCR program on ABI Prism SDS 7000. Do not click on the dissociation protocol if you want to check the PCR result by agarose gel. Save a copy of the setup file and delete all PCR cycles (used for later dissociation curve analysis). Note the extension steps are slightly different from described in our paper. a. 50℃ 2 min, 1 cycle b. 95℃ 10 min, 1 cycle c. 95 ℃ 15 s -> 60 ℃ 30 s -> 72℃ 30 s, 40 cycles d. 72℃ 10 min, 1 cycle 3. A real-time PCR reaction mixture can be either 50 μl or 25 μl. Prepare the following mixture in each optical tube.
25 μl SYBR Green Mix (2x)
12.5 μl SYBR Green Mix (2x)
0.5 μl liver cDNA
or
0.2 μl liver cDNA
2 μl primer pair mix (5 pmol/μl each primer)
1 μl primer pair mix (5 pmol/μl each primer)
22.5 μ H2O
11.3 μl H2O
25  μl SYBR Green Mix (2x) 0.5  μl liver cDNA 2  μl primer pair mix (5 pmol/ μl each primer) 22.5  μl H2O   OR  12.5  μl SYBR Green Mix (2x) 0.2  μl liver cDNA 1 μl primer pair mix (5 pmol/μl each primer) 11.3  μl H2O                                    4. Remove the tubes from the machine, after finishing PCR. The PCR specificity is examined by 3% agarose gel using 5 μl from each reaction. 5. Put the tubes back in SDS 7000 and perform dissociation curve analysis with the saved copy of the setup file. 6. Analyze the real-time PCR result with the SDS 7000 software. Check to see if there is any bimodal dissociation curve or abnormal amplification plot.
3.2 Detection of Flag
1.Protein Extraction:
a. Take some Synechocystis in 2ml EP tube and centrifuge at 5800rpm for 15min。Discard the supernatant，add a fresh BG11 medium containing regulatory factors and centrifuge after the 20m, discard supernatant. Wash the cells with 1X PBS; b. Add 1X SDS Sample buffer (6-well plate per hole 100µl or 500µl) PS: because of the unknown expression, slightly less to increase protein concentration to lysis cells. Immediately scrape the cells off the board and transfer the extract to the micro centrifuge tube.                                 Put on ice. c. Ultrasonic crushing 10–15 seconds to complete the cell lysis as well as to cut DNA (to reduce the sample viscosity). d. Take 20µl samples and heat for 5 minutes under 95–100°c.Then cool the sample with ice. e. Add 20ul samples to SDS-PAGE (10cm X 10cm) f. Electrophoresis
2.Transmembran:
a. Remove the gel from the glass plate and abandon the lower part of the separation of glue. Make the remaining gel containing the target protein soaked in the membrane buffer to prevent the gel from solidification, and deformate. b. Wet: Prepare 2 layer of Whatman 3 #filter paper, 1 NC membrane (the size as similar as the gel size) . Put NC membrane in the water for 3 minutes, and then put into the membrane buffer for 10 minutes. (If using PVDF membrane, should soak in 100% methanol previously, otherwise it is difficult to wetting.) (Methanol can be reused.) c. Put the buffer soaked sponge, 1 layer of whatman3 #filter paper, gel, nitrocellulose membrane, 1 layer of Whatman3# filter paper, sponge in order . Ensure that there is no air bubbles between each layer. Assembly Order: Turn film clip black (negative)-Sponge-pad-Filter paper-Adhesive-film-filter paper-sponge pad-red face (positive) Remember: Get rid if the bubble between each layer with glass rod. d. Transfer film: Transfer the clips into the transfer groove and add 4 ℃ pre-cooled membrane buffer. Make the gel surface connected with the negative electrode when nitrocellulose membrane and the positive electrode connect. Transfer film current 350mA, transfer film 45MIN,PS: adhesive in the negative, the membrane close to the positive, the membrane buffer must be pre-cooled. Preferably a day ahead of preparing the buffer and put in the 4 ℃ refrigerator. Filter paper should not be larger than the film ,to prevent short circuit. e. After the film and gel are clamped and ensure that no bubbles exist between the gel/film and filter .Otherwise it will cause the film to be incomplete. Make sure to wear gloves or touch the membrane with plastic tweezers, because the protein and grease on the hand will affect the film efficiency and make the membrane dirty. In the process of film transfer, especially the high-current rapid transfer, there will usually be a lot of heat. Preferably put the transfer film slot in the ice bath for the trarsmembra.
3.Membrane closure:
a. Wash the nitrocellulose membrane with ML TBS for 5 minutes, after the transfer, at room temperature. b.Place the membrane in an ml buffer and incubate at room temperature for 1 hours. c. Wash three times with 15ml TBST, 5 minutes each time.
4.One-fight incubation:
a. Place the membrane and the first antibody(1:1000 buffer diluted)in a 10 ml 5%w/vBSA,1xTBS,0.1%Tween20 and incubate overnight at 4°c without gently shaking. b. Wash three times with 15ml TBST, 5 minutes each time. c. Put the membrane with Anti-mouse IgG, hrp-linked Antibody (#7076, by 1:2000 ratio) and Anti-biotin, hrp-linked Antibody (#7075, press 1:1000–1:3000). Incubate at room temperature for 1 hours with a gentle shaking. d. Wash three times with 15ml TBST, 5 minutes each time.
5.Protein Detection:
a. Conjugate of antibody and clean membrane with HRB in TBST for 5 minutes, three times. b. Dilute part of 2X reagent A and part of 2X reagent B to prepare 1X SignalFire™ ECL Reagent (#6883).Mix evenly. c. Incubate the substrate with the membrane for 1 minutes and pour out the excess solution (make the membrane remain moist), then wrap it in plastic and expose it under X-ray film.
3.3 Detection of 6×His
1. Weigh and add the bacteria solution, then centrifuge at 5000rpm for 10min. 2. Add lysis buffer for 10-20ml/g bacteria and shake. 3. Ultrasonic crushing, ultrasonic 2s and gap 3s by turns for 5min. 4. Centrifugate at 12500rpm for 30-40min at 4 °C. 5. Take the column firstly loaded with NiSO4, then wash it with deionized water until the effluent is colorless. The leaching-out nickel can be recycled. 6. Add lysis buffer to the column. 7. Add the centrifugation supernatant. 8. Add wash buffer. 9. Add 4ml Elution buffer and gather the liquid with the collection tube (a few drops can be abandoned at first and at last) 10. Add NiSO4, cover for a short-term preservation of the middle of liquid. If it should be long-term stored, NiSO4 needs to be washed off and then add 10% alcohol.
3.4 Lactate detection assay
1. Add 2mL bacteria into 2mL EP tube. 2. Centrifuge at 12000rpm for 1 minutes. 3. Prepare the following mixture in 10mL tubes. 4. Take 0.02mL supernatant into each experimental group. 5. After mixing, incubate at 37℃ for 10 min. 6. Add 2mL into 10 mL EP tube. 7. Detect absorbance (OD530) 8. Calculate.
Blank
Positive control
Experimental group
ddH2O (mL)
0.02
Solution I (mL)
0.02
Enzyme solution (mL)
1
1
1
chromogenic agent (mL)
0.2
0.2
0.2
Part4: Electrogenesis measurement assay
4.1 Electrogenesis
1. Add 100mL FeCl3 (9.72g/L) solution to the negative site. 2. Add lactate (110mM), LB medium and Shewanella oneidensis MR-1(with the OD600 at about 1) in different proportion. 3. Measure the output voltage once an hour.
4.2 Measurement of Electrode-Attached Biomass
1. Place the electrode in a 50 mL tube containing 5 mL NaOH (0.2 mol/L). 2. Vortex for 2 min and incubate in a water bath to lyse cells at 96 °c for 30 min 3. Test extracts by bicinchoninic acid protein assay kit (Solarbio, China) after being cooled to 25 °c.
Reference
[1] Jingjing Xing, Ruiping Wang, Zhongyu Liu, Lian Ma. Optimization of culture medium for Rhodopseudomonas palustris. Journal of Yangtze University (NATURAL SCIENCE EDITION).2017.2:42-45.
[2] https://wenku.baidu.com/view/85141ca74afe04a1b071dea1.html)
[3] https://wenku.baidu.com/view/c805f293fc4ffe473368abfe.html
College of Life Science & Technology
Huazhong University of Science and Technology
Add: 1037 Luoyu Road, Wuhan, Hubei, China; P.C: 430074
Copyright ©2018 Huazhong University of Science & Technology. Produced By HUST-China
