<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HUST-China_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HUST-China/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Team:HUST-China/Demonstrate - 2018.igem.org</TITLE></P><DIV id="globalWrapper"><DIV id="top_menu_14" class="noprint">Loading menubar.....</DIV><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HUST-China/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>HillSide Multi purpose HTML5 Template</TITLE></P><DIV id="wrapper" class="home-page"><HEADER style="margin-top: 12px"><DIV class="navbar navbar-default navbar-static-top"><DIV class="container"><DIV class="navbar-collapse collapse "><UL class="nav navbar-nav"><LI class="HillSide"><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China">Home</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle waves-effect waves-dark">PROJECT</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/project/background">Background</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Description">Description</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Applied Design">Applied Design</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Safety">Safety</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Demonstrate">Demonstrate</A></LI></UL><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle waves-effect waves-dark">WETLAB</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Experiments">Experiments</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Results">Results</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/wetlab/protocols">Protocols</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/InterLab">Interlab</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle waves-effect waves-dark">Model</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Modeling overview">Modeling overview</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/model_of_systems">Model of systems</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Sort of three genes">Sort of three genes</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Software">Software</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle waves-effect waves-dark">PARTS</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Parts Overview">Parts Overview</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Composite part">Composite part</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Part collection">Part collection</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle waves-effect waves-dark">HP</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Human Practices">Human Practices</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Education_Engagement">Education&amp;Engagement</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle waves-effect waves-dark">TEAM</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Team">Team Members</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Collaborations">Collaborations</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Attributions">Attributions</A></LI></DIV></DIV></DIV></HEADER><SECTION id="inner-headline"><DIV class="container"><DIV class="row"><DIV class="col-lg-12"><H2 class="pageTitle">Demonstrate</H2></DIV></DIV></DIV></SECTION><SECTION class="content"><DIV class="container"><DIV class="row"><DIV class="col-md-12 content-text"><DIV class="about-logo"><H3><STRONG>Part1: <SPAN class="red-content">1.Photosynthetic microorganism system</SPAN></STRONG></H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 info-blocks"><DIV class="info-blocks-in" style="background-color: #ffffff; border: 1px solid #eeeeee;border-radius:5px;"><H3>1.1 Synechocystis sp. PCC6803</H3><DIV class="col-md-12"><P>Construction of genes</P><P>Since synechocystis itself lacks a pathway for producing lactate, and as a photoautotrophic microorganism, synechocystis lacks a lactate transporter to transport lactate out of the cell[1]. Therefore, we combined the lactate dehydrogenase gene with the lactate transporter gene in one circuit to achieve lactate production and transportation. For lactate dehydrogenase gene, we chose ldhD, and ldhDc is a codon-optimized version of ldhD, ldhDnARSdR is ldhD with D176A/I177R/F178S/N180R, and ldhDARSdR is the codon-optimized version of ldhDnARSdR. These codon optimizations are aim at increasing the production of lactate. The lldP ​​protein gene is used to transport the lactate out of the cell. We successfully construct genes ldhD-lldp, ldhDC-lldp, ldhDnARSdR-lldp and ldhDARSdR-lldp. </P></DIV><DIV class="col-md-12 col-md-offset-3 "><P style="font-size: 5px;">Figure 1. <I>PCR of ldhD-lldp expression and ldhDC-lldp expression</I></P></DIV><DIV class="col-md-12 col-md-offset-3"><P style="font-size: 5px;">Figure 2. <I>PCR of ldhDnARSdR-lldp expression and ldhDARSdR-lldp expression</I></P></DIV><DIV class="col-md-12"><H3>1.2 Synechocystis sp. PCC6803</H3></DIV><DIV class="col-md-12"><P>Verification of gene expression</P><H3>Detection of yellow fluorescent protein</H3><P>And all the genes were constructed into the plasmid pck306 and transformed into synechocystis. Since pck306 plasmid contain the gene of yellow fluorescent protein, so we put engineered cynobacteria under the fluorescence microscope to verify the transformation in cynobacteria.
                         </P></DIV><DIV class="col-md-12"><P style="font-size: 5px;">Figure 3. Results of yellow fluorescent protein of pck306 plasmid in Synechocystis sp. PCC6803. L-Rhamose(1g/L) was added and after 36 hours, engineered bacteria showed it expressed successfully.</P><P>We could draw the conclusion that these genes were successfully constructed into engineered Synechocystis sp. PCC6803.</P></DIV><DIV class="col-md-12"><H3>Affinity Chromatography</H3></DIV><DIV class="col-md-12"><P>In addition to knowing transcription of the gene, we want to exhibit the expression of the protein, we insert 6Xhis-tag into lldP and did affinity chromatography to show we finally made it. The result is shown below:</P></DIV><DIV class="col-md-12"><P style="font-size: 5px;">Figure 4a. <I>There were nearly 0.2g bacteria used to testify the expression of lldP.</I> After affinity chromatography, the protein was electrophoresed through SDS-PAGE. As the red box shows, lldP can be seen . Comparing to wild type, lldP is expressed in Synechocystis sp. PCC 6803.</P><P>Because of the slow growth of Synechocystis sp. PCC 6803 and time limitation, we did not have too much bacteria, so we only use nearly 0.2g of them to show whether lldP expressed. The color is pale but still can be seen. lldP expression succeeds.</P><H3>Real-Time Quantitative PCR</H3></DIV><DIV class="col-md-12"><P>To illustrate our project does work. We must prove that our circuit express successfully. Therefore, we did real-time quantitative PCR to demostrate the transcription of the circuit ldhD-lldP, ldhDc-lldP, ldhDnARSdR-lldP and ldhDARSdR-lldP. The result shows as the figure below:</P></DIV><DIV class="col-md-8 col-md-offset-1" style="text-align: center;"><DIV class="col-md-offset-1"><P style="font-size: 5px;">Figure 4b. The figure shows that ldhD-lldP, ldhDc-lldP, ldhDnARSdR-lldP, ldhDARSdR-lldP express successfully.</P></DIV></DIV><DIV class="col-md-12"><H3><B>Lactate Detection</B></H3><P>Finally, to testify how the whole circuit in Synechocystis sp. PCC 6803(synechocystis) works, we did lactate detection experiment. After adding L-Rhamose(1g/L) for 72h, we use Lactic Acid assay kit, provided by Nanjing Jiancheng Bioengineering, to quantify lactate concentration. </P></DIV></DIV><DIV class="col-md-12"><P style="font-size: 5px;">Figure 5. <I> Lactate production of engineered Synechocystis sp. PCC6803. </I>Comparison of WT, ldhD-lldP, ldhDc-lldP, ldhDnARSdR-lldP, ldhDARSdR-lldP. Synechocystis grew for 7 days, then L-Rhamose(1g/L) was added. After induced 72 hours, lactate concentration has shown above.</P><P>The figure indicates that after transforming our circuit in bacteria, the production and releasing of lactate increase evidently, and ldhDARSdR-lldP is the most efficient one. In summary, our engineered bacteria, Synechocystis sp. PCC 6803(synechocystis), do achieve our goal to provide lactate to Shewanella. </P><H3>1.3 Rhodopseudomonas palustris</H3></DIV><DIV class="col-md-12 "><P>We try to use mleS (convert pyruvate to D-lactate), ldhA (convert pyruvate to D-lactate) and lldP (L-lactate permease) to make sure their strains could produce lactate. We construct genes into the shuttle plasmid pMG105 in E.coli BL21, and then detect it .</P></DIV><DIV class="col-md-12 col-md-offset-3 style:font-size:5px;"><P>For example of pMG105-PpckaA-RBS -lldP-RBS-ldhA-TT (Figure 3).</P></DIV><DIV class="container"><P style="font-size: 5px;">Figure 6. shows gene circuits are successfully constructed into E.coli BL21.</P></DIV><DIV class="container col-md-12"><DIV class="row"><H3>Real-Time Quantitative PCR</H3><P>we do Real-Time Quantitative PCR to verify the expression of gene circuits by E.coli BL21.</P></DIV></DIV><DIV class="col-md-4"><P>(A)</P></DIV><DIV class="col-md-4"><P>(B)</P></DIV><DIV class="col-md-4"><P>(C)</P></DIV><DIV class="col-md-12 col-md-offset0"><DIV class="container"><DIV class="row"><P style="font-size: 5px;">Figure 7.Relative expression level of targeted genes in E.coli BL21. We choose mleS(malate dehydrogenase, the conversion of malic acid to L-lactate), lldP(L-lactate permease, the lactate is transported out of the cell) and ldhA(fermentative D-lactate dehydrogenase, NAD-dependent, convert pyruvate to D-lactate) as the reference genes and pMG105 as the standard quantity. (A). The expression level of mleS. (B). The expression level of lldP. (C). The expression level of ldhA. We could find that mleS, lldP and ldhA have a higher expression compared with pMG105. It means every genes can express efficiently.</P></DIV><P>It shows the verification of gene expression is correct. Gene mleS, lldP and ldhA have a good expression on mRNA level. </P><H3>Lactate Detection</H3><P>We detect lactate production of gene circuit in E.coli BL21 (Figure8)</P></DIV><P style="font-size: 5px;">Figure 8.LB is a kind of medium to cultivate bacterial. We detect lactate content of LB and others. PMG105 has the lowest lactate content because pMG105 doesn’t have any gene to help produce lactate. It proves that all of our gene circuits have ability to produce lactate. Besides, compared with ldhA and lldP, ldhA-lldP has a better production efficiency;  compared with ldhA-lldP, lldP-ldhA has a better production efficiency; compared with ldhA-lldP, mleS-lldP has a better production efficiency; compared with other gene circuits, mleS-lldP-ldhA has a better production efficiency. The result proves our modeling is correct.</P></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-12 content-text"><DIV class="about-logo"><H3><STRONG>Part2: <SPAN class="red-content">Electrogenic microorganism system</SPAN></STRONG></H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 info-blocks"><DIV class="info-blocks-in" style="background-color: #ffffff; border: 1px solid #eeeeee;border-radius:5px;"><H3>2.1 Plasmid Construction and Conjugation </H3><P>Since lactate is the carbon source of Shewanella Oneidensis MR-1, overexpression of lactate dehydrogenase might help the bacteria to utilize lactate more efficiently and produce more electricity. We chose dld(encodes D-lactate dehydrogenase), lldEFG(encodes L-lactate dehydrogenase) gapA(encodes glyceraldehyde-3-phosphate dehydrogenase), mdh(encodes NAD-dependent malate dehydrogenase), pflB(encodes pyruvate formate-lyase) and fdh(encodes formate dehydrogenase) to overexpress in Shewanella Oneidensis MR-1.
All the genes were constructed into the shuttle plasmid pYYDT, which were then conjugated into Shewanella Oneidensis MR-1. </P><DIV class="col-md-12"><P style="font-size: 5px;">Figure 9.Verification of successful conjugation. (A). pYYDT-dld and pYYDT-lldEFG. (B). pYYDT-gapA-mdh. (C). pYYDT-pflB-fdh.</P></DIV><DIV class="col-md-12"><P>We could draw the conclusion that these genes has been successfully constructed into engineered Shewanella Oneidensis MR-1.</P></DIV><DIV class="col-md-12"><H3>2.2 Verification of expression </H3><P>(1). RT-qPCR</P><P>RT-qPCR(Real-Time quantitative PCR) is a technique of molecular experiment. The expression of genes could be detected by measuring the number of copies of an RNA transcript of that gene in engineered cells, which means that by doing RT-qPCR, we could demonstrate that the related genes could be expressed by engineered bacteria.</P></DIV></DIV><DIV class="col-md-12"><P style="font-size: 5px;">Figure 10. Relative expression level of targeted genes in engineered Shewanella Oneidensis MR-1.We chose gyrB(encodes DNA gyrase B) as the reference gene and 1 as the standard quantity. (A). The expression level of pYYDT-dld. (B). The expression level of pYYDT-gapA-mdh. (C). The expression level of pYYDT-pflB-fdh.</P><P>As shown by figure 10, it is demonstrated that these genes could be expressed in engineered Shewanella Oneidensis MR-1.</P><P>(2).Electrogenesis</P><P>Improvement about the efficiency of electricity production is our final goal. We compared the Shewanella contained targeted genes with the Shewanella with no-load vector pYYDT.
If the former one could produce more electricity, our construction could be proved to receive a success.</P></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-12 content-text"><DIV class="about-logo"><H3><STRONG>Part3: <SPAN class="red-content">Whole design</SPAN></STRONG></H3><DIV class="container"><P>There may be more potential interactions in the co-culture of Rhodopseudomonas palustris and Shewanella, which can greatly improve the coulombic efficiency of our MFC system.
</P></DIV><DIV class="col-md-12 col-md-offset-2"><P style="font-size: 5px;">Figure 11. The interaction between Rhodopseudomonas palustris and Shewanella.</P></DIV></DIV><DIV class="col-md-12 col-md-offset-2"><P style="font-size: 5px;">Figure 12.The comparison of electricity production between wild type Rhodopseudomonas palustris and type Synechocystis PCC6803.</P></DIV></DIV><DIV class="col-md-12 col-md-offset-2"><P style="font-size: 5px;">Figure 13.The comparison between Shewanella with no-load vector and Shewanella with targeted genes.</P></DIV></DIV><DIV class="container"><P>We could learn from this figure that constructed bacteria could produce more electricity, which means that the targeted genes have been expressed successfully in the cell and helped to produce more electricity.
</P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV><FOOTER><DIV class="container"><DIV class="row"><DIV class="copyright col-md-12" style="text-align: center;color: gray; font-size: 14px;"><SPAN>
                College of Life Science &amp; Technology
                Huazhong University of Science and Technology
                Add: 1037 Luoyu Road, Wuhan, Hubei, China; P.C: 430074
                Copyright ©2018 Huazhong University of Science &amp; Technology. Produced By </SPAN><A href="https://2018.igem.org/Team:HUST-China" target="_blank" style="color:white;">HUST-China</A></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>