<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HUST-China_wetlab_protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HUST-China/wetlab/protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>HillSide Multi purpose HTML5 Template</TITLE></P><DIV id="wrapper" class="home-page"><HEADER style="margin-top: 12px"><DIV class="navbar navbar-default navbar-static-top"><DIV class="container"><DIV class="navbar-collapse collapse "><UL class="nav navbar-nav"><LI class="HillSide"><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China">Home</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle waves-effect waves-dark">PROJECT</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/project/background">Background</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Description">Description</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Applied Design">Applied Design</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Safety">Safety</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Demonstrate">Demonstrate</A></LI></UL><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle waves-effect waves-dark">WETLAB</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Experiments">Experiments</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Results">Results</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/wetlab/protocols">Protocols</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/InterLab">Interlab</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle waves-effect waves-dark">Model</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Modeling overview">Modeling overview</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/model_of_systems">Model of systems</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Sort of three genes">Sort of three genes</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Software">Software</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle waves-effect waves-dark">PARTS</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Parts Overview">Parts Overview</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Composite part">Composite part</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Part collection">Part collection</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle waves-effect waves-dark">HP</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Human Practices">Human Practices</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Education_Engagement">Education&amp;Engagement</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle waves-effect waves-dark">TEAM</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Team">Team Members</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Collaborations">Collaborations</A></LI><LI><A class="waves-effect waves-dark" href="https://2018.igem.org/Team:HUST-China/Attributions">Attributions</A></LI></DIV></DIV></DIV></HEADER><SECTION id="inner-headline"><DIV class="container"><DIV class="row"><DIV class="col-lg-12"><H2 class="pageTitle">Protocols</H2></DIV></DIV></DIV></SECTION><SECTION class="content"><DIV class="container"><DIV class="row"><DIV class="col-md-12 content-text"><DIV class="about-logo"><H3><STRONG>Part1: <SPAN class="red-content">Bacteria and culture medium</SPAN></STRONG></H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 info-blocks"><DIV class="info-blocks-in" style="background-color: #ffffff; border: 1px solid #eeeeee;border-radius:5px;"><DIV class="col-md-12

"><H3><STRONG>1.1 E.coli TOP10, DH5α, BL21, WM3064</STRONG></H3><P class="red-content"><STRONG>1.1.1 LB (Luria-Bertani) medium</STRONG></P><P> 1% NaCl
                                1% Tryptone
                                0.5% Yeast Extract</P></DIV><DIV class="col-md-12

 table-responsive"><H3><STRONG>1.2 Rhodopseudomonas palustris(DSM5859)</STRONG></H3><P class="red-content"><STRONG>1.2.1 culture medium<SUP>[1]</SUP></STRONG></P><TABLE class="table table-bordered table-hover" style="text-align: center;"><TBODY><TR><TH>Component</TH><TH>Amount</TH></TR><TR><TD>CH<SUB>3</SUB>COONa</TD><TD>3.8g/L</TD></TR><TR><TD>NH<SUB>4</SUB>Cl</TD><TD>1.3g/L</TD></TR><TR><TD>Yeast extract</TD><TD>0.7g/L</TD></TR><TR><TD>Microelement</TD><TD>0.8Ml/L</TD></TR><TR><TD>—FeCl<SUB>3</SUB>·6H<SUB>2</SUB>O</TD><TD>5mg/L</TD></TR><TR><TD>—CuSO<SUB>4</SUB>·5H<SUB>2</SUB>O</TD><TD>0.05mg/L</TD></TR><TR><TD>—H<SUB>3</SUB>BO<SUB>3</SUB></TD><TD>41mg/L</TD></TR><TR><TD>—MnCl<SUB>2</SUB>·4H<SUB>2</SUB>O</TD><TD>0.05mg/L</TD></TR><TR><TD>—ZnSO<SUB>4</SUB>·7H<SUB>2</SUB>O</TD><TD>1mg/L</TD></TR><TR><TD>—CO(NO<SUB>3</SUB>)<SUB>2</SUB>·6H<SUB>2</SUB>O</TD><TD>0.5mg/L</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-12 table-responsive"><H3><STRONG>1.3  Synechocystis sp. PCC6803</STRONG></H3><P class="red-content"><STRONG>1.3.1 BG11(Blue-Green medium)<SUP>[2]</SUP></STRONG></P><TABLE class="table table-bordered table-hover" style="text-align: center;"><TBODY><TR><TH>Component</TH><TH>Amount</TH><TH>Stock Solution</TH></TR><TR><TD>NaNO3</TD><TD>10mL/L</TD><TD>15.0g/100mLdH<SUB>2</SUB>O

</TD></TR><TR><TD>K<SUB>2</SUB>HPO<SUB>4</SUB></TD><TD>1mL/L</TD><TD>4g/1O0mLdH<SUB>2</SUB>O

</TD></TR><TR><TD>MgSO<SUB>4</SUB> ·7H<SUB>2</SUB>O

</TD><TD>1mL /L</TD><TD>7.5g/100mLdH<SUB>2</SUB>O

</TD></TR><TR><TD>CaCl<SUB>2</SUB> ·2H<SUB>2</SUB>O

</TD><TD>1mL/L</TD><TD>3.6g/1 00mLdH<SUB>2</SUB>O

</TD></TR><TR><TD>Citric acid</TD><TD>1mL /L</TD><TD>0.6g/100mLdH<SUB>2</SUB>O

</TD></TR><TR><TD>Ferric ammonium citrate</TD><TD>1mL/L</TD><TD>0.6g/100mLdH<SUB>2</SUB>O

</TD></TR><TR><TD>EDTANa2</TD><TD>1mL/L</TD><TD>0.1g/100mldH<SUB>2</SUB>O

</TD></TR><TR><TD>Na<SUB>2</SUB>CO<SUB>3</SUB></TD><TD>1mL/L</TD><TD>2.0g/100mldH<SUB>2</SUB>O

</TD></TR><TR><TD>H<SUB>3</SUB>BO<SUB>3</SUB></TD><TD>2.86g/LdH<SUB>2</SUB>O</TD></TR><TR><TD>MnCl<SUB>2</SUB>·4H<SUB>2</SUB>O

</TD><TD>1.86g/LdH<SUB>2</SUB>O</TD></TR><TR><TD>ZnSO<SUB>4</SUB>·7H<SUB>2</SUB>O

</TD><TD>0.22g/LdH<SUB>2</SUB>O</TD></TR><TR><TD>Na<SUB>2</SUB>MoO<SUB>4</SUB>·2H<SUB>2</SUB>O

</TD><TD>0.021g/LdH<SUB>2</SUB>O</TD></TR><TR><TD>CuSO<SUB>4</SUB>·5H<SUB>2</SUB>O

</TD><TD>0.08g/L dH<SUB>2</SUB>O</TD></TR><TR><TD>Co(NO<SUB>3</SUB>)<SUB>2</SUB>·6H<SUB>2</SUB>O

</TD><TD>0.05g/LdH<SUB>2</SUB>O</TD></TR><P>Adjust PH to 7.1 with 1M NaOH or HCl</P></TBODY></TABLE></DIV><DIV class="col-md-12

 table-responsive"><H3><STRONG>1.4 Shewanella oneidensis MR-1</STRONG></H3><P class="red-content"><STRONG>1.4.1 LB medium<SUP>[2]</SUP></STRONG></P><P>
                                1% NaCl
                                  1% Tryptone
                                  0.5% Yeast Extract</P></DIV><DIV class="col-md-12

 table-responsive" align="center"><P class="red-content"><STRONG>1.4.2 M9 medium<SUP>[3]</SUP></STRONG></P><TABLE class="table table-bordered table-hover" style="text-align: center;"><TBODY><TR><TH>Component</TH><TH>Amount</TH></TR><TR><TD>1 M MgSO<SUB>4</SUB></TD><TD>2mL</TD></TR><TR><TD>20% Glucose</TD><TD>20mL</TD></TR><TR><TD>1 M CaCl<SUB>2</SUB></TD><TD>0.1mL</TD></TR><TR><TD>5×M9 Salt solution</TD><TD>200mL</TD></TR><TR><TD>Na<SUB>2</SUB>PO<SUB>4</SUB>·7H<SUB>2</SUB>O</TD><TD>12.8g</TD></TR><TR><TD>KH<SUB>2</SUB>PO<SUB>4</SUB></TD><TD>3.0g</TD></TR><TR><TD>Nacl</TD><TD>0.5g</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>200mL</TD></TR><P>Add H<SUB>2</SUB>O up to 1000mL in total</P></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 content-text"><DIV class="about-logo"><H3><STRONG>Part2: <SPAN class="red-content">Cloning method</SPAN></STRONG></H3></DIV></DIV></DIV><DIV class="row"><DIV class="cl-md-12 info-blocks"><DIV class="info-blocks-in" style="background-color: #ffffff; border: 1px solid #eeeeee;border-radius:5px;"><DIV class="col-md-12"><H3><STRONG>2.1 PCR reaction system</STRONG></H3></DIV><DIV class="col-md-12  table-responsive"><TABLE class="table table-bordered table-hover" style="text-align: center;"><TBODY><TR><TH>Component:</TH><TH>Volume:</TH></TR><TR><TD>PrimeSTAR Max</TD><TD>12.5μl</TD></TR><TR><TD>Template DNA</TD><TD>1μl</TD></TR><TR><TD>Primer-F</TD><TD>1μl</TD></TR><TR><TD>Primer-R</TD><TD>1μl</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>up to 25 μl</TD></TR><TR><TH>Total:</TH><TD>25μl</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-12 table-responsive"><TABLE class="table table-bordered table-hover" style="text-align: center;"><TBODY><TR><TH>Step：</TH><TH>Temperature:</TH><TH>Time:</TH><TH>Cycle:</TH></TR><TR><TD>Hot</TD><TD>94℃</TD><TD>2min</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>94℃</TD><TD>1min</TD><TD>25</TD></TR><TR><TD>Annealing</TD><TD>55℃</TD><TD>1min</TD><TD>25</TD></TR><TR><TD>Extension</TD><TD>72℃</TD><TD>1min</TD><TD>25</TD></TR><TR><TD>Final Extension</TD><TD>72℃</TD><TD>7min</TD><TD>1</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-12"><H3><STRONG>2.2 Double enzyme digestion system</STRONG></H3></DIV><DIV class="col-md-12

 table-responsive"><TABLE class="table table-bordered table-hover" style="text-align: center;"><TBODY><TR><TH>Component:</TH><TH>Volume:</TH></TR><TR><TD>Universal Buffer</TD><TD>5μl</TD></TR><TR><TD>Enzyme 1</TD><TD>1.5μl</TD></TR><TR><TD>Enzyme 2</TD><TD>1.5μl</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>up to 25 μl</TD></TR><TR><TD>Total:</TD><TD>50μl</TD></TR><TR><TD>Coditions</TD><TD>37℃ 30~40min</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-12"><H3><STRONG>2.3 Ligation system</STRONG></H3></DIV><DIV class="col-md-12

 table-responsive"><TABLE class="table table-bordered table-hover" style="text-align: center;"><TBODY><TR><TH>Component:</TH><TH>Volume:</TH></TR><TR><TD>T4 ligase</TD><TD>1μl</TD></TR><TR><TD>T4 ligase buffer (1x)</TD><TD>1μl</TD></TR><TR><TD>Linearized Vector</TD><TD>0.5μl</TD></TR><TR><TD>Insert Gene</TD><TD>4.5 μl</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-12"><H3><STRONG>2.4 Reaction system of colony PCR</STRONG></H3></DIV><DIV class="col-md-12

 table-responsive"><TABLE class="table table-bordered table-hover" style="text-align: center;"><TBODY><TR><TH>Component:</TH><TH>Volume:</TH></TR><TR><TD>rTaq Mix(2×)</TD><TD>5μl</TD></TR><TR><TD>Primer-F(10μM)</TD><TD>0.3μl</TD></TR><TR><TD>Primer-R(10μM)</TD><TD>0.3μl</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>4.4μl</TD></TR><TR><TD>Toal:</TD><TD>10μl</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-12"><H3><STRONG>2.5 Transformation into E. coli TOP10/DH5α/BL21/WM3064</STRONG></H3><P>The ligation product was transformed into E. coli DH5a Strain. The strains were cultivated in LB plate medium containing antibiotic at 37℃.</P></DIV><DIV class="col-md-12"><H3><STRONG>2.6 Transformation into Synechocystis sp. PCC6803</STRONG></H3><P> 1. Take 2 ml of Synechocystis solution when the O.D. is 0.8, and centrifuge for 15 minutes at 5800 rpm. 


                            2.  Discard the supernatant and dissolve the precipitate with the new BG11 medium. 


                            3.  Add the solution into 50 ml small conical flask with 30 μl plasmid. 


                            4.  Add the new BG11 medium gradually to the final concentration of 5 ml. 


                            5.  Put in light culture box. 


                            6.  Expand the culture to 50ml at last, when the color is deep enough. 


                            7.  Take part of them to test for protein expression.</P></DIV><DIV class="col-md-12"><H3><STRONG>2.7 Detection of yellow fluorescent protein</STRONG></H3><P> Cultivate the engineering cynobacteria induced by rhamnose for 24h and put them under the fluorescence microscope at the wave of 488nm excitation light.
                               </P></DIV><DIV class="col-md-12"><H3> 1.Making electrocompetent cells for Rhodopseudomonas palustris:</H3><P>a. Inoculate 10% of Rhodopseudomonas palustris DSM5859 into LB medium (2 ml) and incubate at 37 °C for 8 to 24 hours.


                                  b. Place the bacterial solution in an ice bath for 30 minutes.


                                  c. Wash with cold ddH<SUB>2</SUB>O

 and refrigerated centrifuge for 7 to 8 times to ensure the temperature of the bacteria below 4 °C.


                                  d. Add 10% glycerol and store at -80 °C after the last centrifugation.</P></DIV><DIV class="col-md-12"><H3> 2.Electrotransformation for Rhodopseudomonas palustris DSM5859:</H3><P>a. Mix 50 μl of electocompetent cells with 10 to 20 μl of plasmid in an ice bath for at least 1 minute.


                                  b. Move into an electric rotor (2mm pre-cooling) and use the Gene Pulser XcellTM Total System (BIO-RAD) (400 Ω, 2.5 kV, 25 μF, 12.5 kV/cm field strength).


                                  c. Add 1 ml of LB medium immediately after the electric shock.


                                  d. Mix the bacterial solution in the electric rotor with the culture medium and transfer it into the EP tube.


                                  e. Incubate for 2-6 hours at 37 °C, apply it to the kanamycin resistant culture plate.


                                  f. Culture in dark conditions.</P></DIV><DIV class="col-md-12"><H3><STRONG>2.8 Making competent cells of E.coli WM3064</STRONG></H3><P>1. Inoculate wm3064 into DAP-LB, cultivate the bacteria at 37°C, 200rpm for 6~8 hours.


                                  2. Streak the bacteria on DAP-LB plate medium at 37°C.


                                  3. Pick a single colony and cultivate in 10mL DAP-LB overnight.


                                  4. Transfer 10mL bacterium solution into conical flask with 100mL DAP-LB and cultivate the bacteria. Put the conical flask in the ice when OD600 increases to 0.4~0.6.


                                  5. Put CaCl2-glycerol solution, two 50mL sterilized centrifuge tubes and some 1.5mL EP tubes in the ice.


                                  6. Divide the bacterium solution into to 50mL sterilized centrifuge tubes. 


                                  7. Centrifuge at 4°C, 4000rpm for 8 minutes. Discard the supernatant.


                                  8. Add 30mL CaCl2 and resuspend.


                                  9.Repeat 7 and 8 for 4 times.


                                  10.Add 2mL CaCl2- glycerol solution and resuspend. Divide the bacterium solution into 1.5mL EP tubes. Store the competence at -80°C.</P></DIV><DIV class="col-md-12"><H3><STRONG>2.9 Transformation into E.coli WM3064 (2,6-diaminopimelic acid deficiency type)</STRONG></H3><P>1. Add 50μl competence wm3064 and 1μl plasmid into an EP tube.


                                  2. Put the tube in the ice and incubate the cell for 30 minutes.                             3. Take a heat shock (42°C) for 80 seconds. 


                                  4. Take an ice-bath for 3~5 minutes.


                                  5. Add 300μl DAP-LB into the EP tube and incubate the cell at 37°C, 200rpm for 40 minutes.


                                  6. Add 200μl solution in a plate with kanamycin and DAP. Cultivate these bacteria overnight.</P></DIV><DIV class="col-md-12"><H3><STRONG>2.10 Conjugation into Shewanella oneidensis MR-1</STRONG></H3><P>1. Inoculate Shewanella MR-1 and WM3064 (possess the targeted plasmid).


                                  2. Add 500μl Shewanella MR-1 and WM3064 into a 1.5mL EP tube, mixing the bacterium solution.
                                  3. Centrifuge the solution at 5000rpm for 10 minutes. Discard the supernant.


                                  4. Resuspend the bacteria with 1000mL DAP-K+-LB. Take a 30°C water bath for 1 hour.


                                  5. Add 200μl solution in a plate with kanamycin.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 content-text"><DIV class="about-logo"><H3><STRONG>Part3: <SPAN class="red-content">Verification of gene expression</SPAN></STRONG></H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 info-blocks"><DIV class="info-blocks-in" style="background-color: #ffffff; border: 1px solid #eeeeee;border-radius:5px;"><DIV class="col-md-12"><H3><STRONG>3.1 Real-Time Quantitative PCR</STRONG></H3><P>
                                1. Normalize the primer concentrations, mix gene-specific forward and reverse primer pair. Each primer (forward or reverse) concentration in the mixture is 5 pmol/μl.


                                2. Set up the experiment and the following PCR program on ABI Prism SDS 7000. Do not click on the dissociation protocol if you want to check the PCR result by agarose gel. Save a copy of the setup file and delete all PCR cycles (used for later dissociation curve analysis). Note the extension steps are slightly different from described in our paper.


                                a. 50℃ 2 min, 1 cycle


                                b. 95℃ 10 min, 1 cycle


                                c. 95 ℃ 15 s -&gt; 60 ℃ 30 s -&gt; 72℃ 30 s, 40 cycles


                                d. 72℃ 10 min, 1 cycle


                                3. A real-time PCR reaction mixture can be either 50 μl or 25 μl. Prepare the following mixture in each optical tube.</P><DIV class="col-md-12 table-responsive"><TABLE class="table table-bordered table-hover" style="text-align: center;"><TBODY><TR><TH>25 μl SYBR Green Mix (2x)</TH><TH>12.5 μl SYBR Green Mix (2x)</TH></TR><TR><TH>0.5 μl liver cDNA</TH><TH>or</TH><TH>0.2 μl liver cDNA</TH></TR><TR><TH>2 μl primer pair mix (5 pmol/μl each primer)</TH><TH>1 μl primer pair mix (5 pmol/μl each primer)</TH></TR><TR><TH>22.5 μ H<SUB>2</SUB>O

</TH><TH>11.3 μl H<SUB>2</SUB>O

</TH></TR></TBODY></TABLE></DIV><P>
                                25  μl SYBR Green Mix (2x)


                                0.5  μl liver cDNA


                                2  μl primer pair mix (5 pmol/ μl each primer)


                                22.5  μl H<SUB>2</SUB>O

 OR  12.5  μl SYBR Green Mix (2x)


                                0.2  μl liver cDNA


                                1 μl primer pair mix (5 pmol/μl each primer)


                                11.3  μl H<SUB>2</SUB>O



                                4. Remove the tubes from the machine, after finishing PCR. The PCR specificity is examined by 3% agarose gel using 5 μl from each reaction. 


                                5. Put the tubes back in SDS 7000 and perform dissociation curve analysis with the saved copy of the setup file.


                                6. Analyze the real-time PCR result with the SDS 7000 software. Check to see if there is any bimodal dissociation curve or abnormal amplification plot.</P></DIV><DIV class="col-md-12"><H3><STRONG>3.2 Detection of Flag</STRONG></H3><H3>1.Protein Extraction:</H3><P> a. Take some Synechocystis in 2ml EP tube and centrifuge at 5800rpm for 15min。Discard the supernatant，add a fresh BG11 medium containing regulatory factors and centrifuge after the 20m, discard supernatant. Wash the cells with 1X PBS;


                                b. Add 1X SDS Sample buffer (6-well plate per hole 100µl or 500µl) PS: because of the unknown expression, slightly less to increase protein concentration to lysis cells. Immediately scrape the cells off the board and transfer the extract to the micro centrifuge tube.
                                Put on ice.


                                c. Ultrasonic crushing 10–15 seconds to complete the cell lysis as well as to cut DNA (to reduce the sample viscosity). 


                                d. Take 20µl samples and heat for 5 minutes under 95–100°c.Then cool the sample with ice.


                                e. Add 20ul samples to SDS-PAGE (10cm X 10cm) 


                                f. Electrophoresis</P><H3>2.Transmembran:</H3><P> a. Remove the gel from the glass plate and abandon the lower part of the separation of glue. Make the remaining gel containing the target protein soaked in the membrane buffer to prevent the gel from solidification, and deformate.


                                b. Wet: Prepare 2 layer of Whatman 3 #filter paper, 1 NC membrane (the size as similar as the gel size) . Put NC membrane in the water for 3 minutes, and then put into the membrane buffer for 10 minutes. (If using PVDF membrane, should soak in 100% methanol previously, otherwise it is difficult to wetting.) (Methanol can be reused.)


                                c. Put the buffer soaked sponge, 1 layer of whatman3 #filter paper, gel, nitrocellulose membrane, 1 layer of Whatman3# filter paper, sponge in order . Ensure that there is no air bubbles between each layer.


                                Assembly Order: Turn film clip black (negative)-Sponge-pad-Filter paper-Adhesive-film-filter paper-sponge pad-red face (positive) Remember: Get rid if the bubble between each layer with glass rod.


                                d. Transfer film: Transfer the clips into the transfer groove and add 4 ℃ pre-cooled membrane buffer. Make the gel surface connected with the negative electrode when nitrocellulose membrane and the positive electrode connect. Transfer film current 350mA, transfer film 45MIN,PS: adhesive in the negative, the membrane close to the positive, the membrane buffer must be pre-cooled. Preferably a day ahead of preparing the buffer and put in the 4 ℃ refrigerator. Filter paper should not be larger than the film ,to prevent short circuit.


                                e. After the film and gel are clamped and ensure that no bubbles exist between the gel/film and filter .Otherwise it will cause the film to be incomplete. Make sure to wear gloves or touch the membrane with plastic tweezers, because the protein and grease on the hand will affect the film efficiency and make the membrane dirty. In the process of film transfer, especially the high-current rapid transfer, there will usually be a lot of heat. Preferably put the transfer film slot in the ice bath for the trarsmembra.</P><H3>3.Membrane closure:</H3><P> a. Wash the nitrocellulose membrane with ML TBS for 5 minutes, after the transfer, at room temperature.
                                b.Place the membrane in an ml buffer and incubate at room temperature for 1 hours.
                                c. Wash three times with 15ml TBST, 5 minutes each time.</P><H3>4.One-fight incubation:</H3><P>a. Place the membrane and the first antibody(1:1000 buffer diluted)in a 10 ml 5%w/vBSA,1xTBS,0.1%Tween20 and incubate overnight at 4°c without gently shaking.
                                b. Wash three times with 15ml TBST, 5 minutes each time. 
                                c. Put the membrane with Anti-mouse IgG, hrp-linked Antibody (#7076, by 1:2000 ratio) and Anti-biotin, hrp-linked Antibody (#7075, press 1:1000–1:3000). Incubate at room temperature for 1 hours with a gentle shaking. 
                                d. Wash three times with 15ml TBST, 5 minutes each time.</P><H3>5.Protein Detection:</H3><P>a. Conjugate of antibody and clean membrane with HRB in TBST for 5 minutes, three times. 


                                b. Dilute part of 2X reagent A and part of 2X reagent B to prepare 1X SignalFire™ ECL Reagent (#6883).Mix evenly.


                                c. Incubate the substrate with the membrane for 1 minutes and pour out the excess solution (make the membrane remain moist), then wrap it in plastic and expose it under X-ray film.</P><H3><STRONG>3.3 Detection of 6×His</STRONG></H3><P>
                                1. Weigh and add the bacteria solution, then centrifuge at 5000rpm for 10min.


                                2. Add lysis buffer for 10-20ml/g bacteria and shake.


                                3. Ultrasonic crushing, ultrasonic 2s and gap 3s by turns for 5min.


                                4. Centrifugate at 12500rpm for 30-40min at 4 °C. 


                                5. Take the column firstly loaded with NiSO4, then wash it with deionized water until the effluent is colorless. The leaching-out nickel can be recycled.


                                6. Add lysis buffer to the column.


                                7. Add the centrifugation supernatant.


                                8. Add wash buffer.


                                9. Add 4ml Elution buffer and gather the liquid with the collection tube (a few drops can be abandoned at first and at last)


                                10. Add NiSO4, cover for a short-term preservation of the middle of liquid. If it should be long-term stored, NiSO4 needs to be washed off and then add 10% alcohol.
                              </P><H3><STRONG>3.4 Lactate detection assay</STRONG></H3><P>
                              1. Add 2mL bacteria into 2mL EP tube.


                              2. Centrifuge at 12000rpm for 1 minutes. 


                              3. Prepare the following mixture in 10mL tubes. 


                              4. Take 0.02mL supernatant into each experimental group. 


                              5. After mixing, incubate at 37℃ for 10 min. 


                              6. Add 2mL into 10 mL EP tube. 


                              7. Detect absorbance (OD530)


                              8. Calculate.</P><DIV class="col-md-12

 table-responsive"><TABLE class="table table-bordered table-hover" style="text-align: center;"><TBODY><TR><TH>Blank</TH><TH>Positive control</TH><TH>Experimental group</TH></TR><TR><TH>ddH<SUB>2</SUB>O (mL)</TH><TD>0.02</TD></TR><TR><TH>Solution I (mL)</TH><TD>0.02</TD></TR><TR><TH>Enzyme solution (mL)</TH><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TH>chromogenic agent (mL)</TH><TD>0.2</TD><TD>0.2</TD><TD>0.2</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 content-text"><DIV class="about-logo"><H3><STRONG>Part4: <SPAN class="red-content"> Electrogenesis measurement assay</SPAN></STRONG></H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 info-blocks"><DIV class="info-blocks-in" style="background-color: #ffffff; border: 1px solid #eeeeee;border-radius:5px;"><H3><STRONG>4.1 Electrogenesis</STRONG></H3><P>
                               1. Add 100mL FeCl<SUB>3</SUB> (9.72g/L) solution to the negative site.


                                2. Add lactate (110mM), LB medium and Shewanella oneidensis MR-1(with the OD600 at about 1) in different proportion.


                                3. Measure the output voltage once an hour.</P><H3><STRONG>4.2 Measurement of Electrode-Attached Biomass</STRONG></H3><P>
                               1. Place the electrode in a 50 mL tube containing 5 mL NaOH (0.2 mol/L).


                                2. Vortex for 2 min and incubate in a water bath to lyse cells at 96 °c for 30 min


                                3. Test extracts by bicinchoninic acid protein assay kit (Solarbio, China) after being cooled to 25 °c.
                                </P></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="about-logo"><H4><STRONG>Reference </STRONG></H4><P style="font-size:12px;">[1] Jingjing Xing, Ruiping Wang, Zhongyu Liu, Lian Ma. Optimization of culture medium for Rhodopseudomonas palustris. Journal of Yangtze University (NATURAL SCIENCE EDITION).2017.2:42-45. </P><P style="font-size:12px;">[2] https://wenku.baidu.com/view/85141ca74afe04a1b071dea1.html)</P><P style="font-size:12px;">[3] https://wenku.baidu.com/view/c805f293fc4ffe473368abfe.html</P></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><FOOTER><DIV class="container"><DIV class="row"><DIV class="copyright col-md-12" style="text-align: center;color: gray; font-size: 14px;"><SPAN>
                College of Life Science &amp; Technology
                Huazhong University of Science and Technology
                Add: 1037 Luoyu Road, Wuhan, Hubei, China; P.C: 430074
                Copyright ©2018 Huazhong University of Science &amp; Technology. Produced By </SPAN><A href="https://2018.igem.org/Team:HUST-China" target="_blank" style="color:white;">HUST-China</A></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>