<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Nanjing-China_Model_h2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Nanjing-China/Model/h2</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:Nanjing-China - 2017.igem.org</TITLE></P><DIV class="container" align="center"><DIV class="header"><DIV id="menu"><UL><LI><A href="https://2017.igem.org/Team:Nanjing-China">HOME</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China">Introduction</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Background">Background</A></LI></UL><LI><A href="https://2017.igem.org/Team:Nanjing-China/Parts">PARTS</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Sliver">JUDGE</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Sliver">Sliver</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Gold">Gold</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Design">PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Improvement">Improvement</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/TEAM/Introduction">TEAM</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/TEAM/Introduction">Introduction</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Members">Members</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Collaborations">Collaboration</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/HP/Silver">OTHERS</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/HP/Silver">HP-Silver</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/HP/Gold_Integrated">HP-Gold</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Model">Model</A></LI></DIV></DIV><DIV class="contain"><DIV id="h2"><DIV class="word"><DIV align="left"><H2>The Signalling Cascade</H2><P>Protein  hoxB and hoxC form a Heterodimeric core(RH) they acts as an hydrogen sensor in  concert with the histidine protein kinase HoxJ . This RH-HoxJ (hoxBC-hoxJ) complex  detects the presence of hydrogen in the environment and transmits the signal to  a response regulator HoxA. The response regulator, HoxA, is active in its  non-phosphorylated form and inactivated by HoxJ-mediated phosphorylation. In  the hydrogen atmosphere, HoxA is dephosphorylated and interacts with a promotor  hoxP, and it is capable of activating downstream gfp gene transcription.</P><P><STRONG>Thus,  our key assumptions for interactions among proteins are:</STRONG>
        1.Ignoring the procession of hoxB hoxC and  hoxJ combination.
        2. the Law of Mass Action  kinetics is suitable for Hydrogen detection.
        3. the Law of  Michaelis–Menten kinetics is suitable for hoxBC-hoxJ catalyst.
        4.Because HoxA has been phosphorylated as soon as it’s translated, we assume  there is only hoxA-P before hydrogen induces dephosphorization.</P><H2>Signalling Kinetics</H2><P>In  cell signalling, molecule Hydrogen binds to their receptor sites hoxBC-hoxJ  which triggers a conformational change in the receptor. The donor bind to their  receptors with varying rates and affinities depending on the molecules  involved. 
We model this rate using the Law of Mass Action kinetics. This principle  governs that the rate of a reaction (d/dt) is proportional to the  concentrations of the substrates [2].
For a particular reaction, for example:</P><P>the rate can be written mathematically as:</P><P>But most biological reactions are  reversible, so</P><P>Applying these principles to the  interaction between hoxBC-hoxJ and Hydrogen yields:</P><DIV align="center"><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="236" valign="top"><P><STRONG>Signalling Cascade Model Parameters</STRONG></P></TD><TD width="142" valign="top"><P><STRONG>Value</STRONG></P></TD><TD width="142" valign="top"><P><STRONG>Units</STRONG></P></TD></TR><TR><TD width="236" valign="top">k<SUB>f</SUB></TD><TD width="142" valign="top"><P>Hydrogen binding constant</P></TD><TD width="142" valign="top"><P>8(est)</P></TD></TR><TR><TD width="236" valign="top">k<SUB>r</SUB></TD><TD width="142" valign="top"><P>Hydrogen dissociation</P></TD><TD width="142" valign="top"><P>0.8(est)</P></TD></TR></TBODY></TABLE></DIV><P>In cell signaling, the enzyme hoxBC-hoxJ,  combined with Hydrogen, catalyzes the reaction of hoxA –P dephosphorization, We model this  rate using the Law of Michaelis–Menten kinetics, this model involves an enzyme,  E, binding to a substrate, S, to form a complex, ES, which in turn releases a  product, P, regenerating the original enzyme. This may be represented  schematically as </P><P>where (forward rate), (reverse  rate), and (catalytic  rate) denote the rate constants, the double arrows between S (substrate) and ES  (enzyme-substrate complex) represent the fact that enzyme-substrate binding is  a reversible process, and the single forward arrow represents the formation of  P (product).
        Under certain  assumption– such as the enzyme concentration being much less than the substrate  concentration – the rate of product formation is given by</P><P>Applying these principles to the  interaction between hoxBC-hoxJ and hoxA-P yields:</P><DIV align="center"><TABLE border="1" cellspacing="0" cellpadding="0" width="0"><TBODY><TR><TD width="245" valign="top"><P><STRONG>Signalling Cascade Model Parameters</STRONG></P></TD><TD width="217" valign="top"><P><STRONG>Value</STRONG></P></TD><TD width="95" valign="top"><P><STRONG>Units</STRONG></P></TD></TR><TR><TD width="245" height="31" valign="top"><DIV align="center">k<SUB>cat</SUB></DIV></TD><TD width="217" valign="top"><P>the turnover number</P></TD><TD width="95" valign="top"><P>950（est）</P></TD></TR><TR><TD width="245" valign="top"><DIV align="center">k<SUB>m</SUB></DIV></TD><TD width="217" valign="top"><P>The substrate concentration at which the    reaction rate is half of </P></TD><TD width="95" valign="top"><P>0.015（est）</P></TD></TR></TBODY></TABLE><P> </P></DIV><H2>Transcription </H2><P>Transcription is a complex procession.  However, it is well established that initiation of transcription is the major  factor in determining overall rate of mRNA production. Transcription initiation  is a process influenced by a number of key steps including: promoter escape,  binding of the RNA Polymerase onto the promoter, isomerization, and formation  of the closed RNA Pol complex from the open complex, and your <EM>Escherichia coli</EM> strain. In this model,  we will assume that binding of the RNA polymerase is the rate determining step  in transcription initiation.</P><P><STRONG>Thus,  our key assumptions for transcription are: </STRONG> 1. For the constitutively expressed genes (hoxA, hoxB, hoxC,  hoxJ), we can assume that they are at maximal transcription.
  2. hoxP transcription rate is determined primarily by the amount of  substrate (hoxA) bound to the DNA and the strength of the promoter hoxP.  Maximum transcription rate occurs when all binding sites are filled.  3. Promoter strength (in Polymerases  per second) is a constant, and represents the rate of transcription at maximal  expression. 
  As for the reporter genes, the model is  complicated by the binding of signalling molecules, Which is contrast to the  constitutively expressed genes, we model this rate using Hill equation[3]. </P> The equation's terms are defined as follows:
<UL type="disc"><LI>-       Fraction of the receptor protein concentration that is bound to ligand.</LI><LI>[L]- Free (unbound) ligand concentration</LI><LI>-       Apparent dissociation constant derived from the law of mass action (the       equilibrium constant for dissociation), which is equal to the ratio of the       dissociation rate of the ligand-receptor complex to its association rate ().[4] </LI><LI>-       The ligand concentration producing half occupation (ligand concentration       occupying half of the binding sites). Because is       defined so that ,       this is also known as the microscopic dissociation constant. In recent       literature, this constant is sometimes referred to as  .</LI><LI>n&gt;1, <STRONG>Positively cooperative binding</STRONG>: Once one ligand molecule is       bound to the enzyme, its affinity for other ligand molecules increases.</LI><LI>n&lt;1, <STRONG>Negatively cooperative       binding</STRONG>: Once one ligand molecule is bound to the enzyme, its affinity       for other ligand molecules decreases.</LI><LI>n=1, <STRONG>Noncooperative (completely       independent) binding</STRONG>: The affinity of the enzyme for a ligand molecule       is not dependent on whether or not other ligand molecules are already bound.</LI></UL><P> So </P><DIV align="center"><TABLE border="1" cellspacing="0" cellpadding="0" width="0"><TBODY><TR><TD width="236" valign="top"><P><STRONG>Model Parameters</STRONG></P></TD><TD width="180" valign="top"><P><STRONG>Value</STRONG></P></TD><TD width="142" valign="top"><P><STRONG>Units</STRONG></P></TD></TR><TR><TD width="236" valign="top"><P>T5 Promoter strength</P></TD><TD width="180" valign="top"><P>0.012</P></TD><TD width="142" valign="top"><P>mRNA molecules/sec</P></TD></TR><TR><TD width="236" valign="top">V<SUB>max</SUB></TD><TD width="180" valign="top"><P>0.23</P></TD><TD width="142" valign="top"><P>mRNA molecules/sec</P></TD></TR><TR><TD width="236" valign="top"><P>K<SUB>d</SUB></P></TD><TD width="180" valign="top"><P>The substrate concentration at which the    reaction rate is half of </P></TD><TD width="142" valign="top"><P>0.012（est）</P></TD></TR></TBODY></TABLE></DIV><H2>Translation</H2><P>After transcription,  it must be translated into a functional chromo protein. Translation involves 3  main stages: initiation, elongation of the polypeptide, and termination. The  key molecular machinery driving this process are ribosomes[5], who's binding  and movements determine the overall rate of transcription. 
            Translation rate can be affected by 3 major processes:
            a) the binding of the ribosome to the RBS,
            b) formation of initiation complexes, 
            c) speed of the  ribosome across the mRNA transcript.
          we will assume that the movement of the  ribosome (in amino acids formed per second) is the rate determining step in the  translation of the proteins, and the average  translation rate of 17.1 amino acids/sec.[6] </P><DIV align="center"><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="138" valign="top"><P><STRONG>Product</STRONG></P></TD><TD width="138" valign="top"><P><STRONG>Length (amino    acids)</STRONG></P></TD><TD width="271" valign="top"><P><STRONG>Translation rate (1/s)</STRONG></P></TD></TR><TR><TD width="138" valign="top"><P>hoxBC-hoxJ</P></TD><TD width="138" valign="top"><P>3628</P></TD><TD width="271" valign="top"><P>0.0047</P></TD></TR><TR><TD width="138" valign="top"><P>HoxA</P></TD><TD width="138" valign="top"><P>482</P></TD><TD width="271" valign="top"><P>0.0353</P></TD></TR><TR><TD width="138" valign="top"><P>Gfp</P></TD><TD width="138" valign="top"><P>230</P></TD><TD width="271" valign="top"><P>0.0739</P></TD></TR></TBODY></TABLE></DIV><H2>Degradation and Dilution</H2>
        The concentrations of each species in the  model is affected by it’s degradation (inherent to the molecule) and dilution  (due to cell growth). While these processes are completely unrelated, they have  the same net effect: Decreasing the concentration of the species[7]. Therefore,  we represent the two rates as a combined degradation/dilution parameter in our  model. The promotor of hox gene is constitutive, so we assume only the  degradation/dilution of Gfp mRNA and Gfp.<DIV align="center"><TABLE border="1" cellspacing="0" cellpadding="0" width="0"><TBODY><TR><TD width="264" valign="top"><P><STRONG>Name</STRONG></P></TD><TD width="170" valign="top"><P><STRONG>Value</STRONG></P></TD><TD width="123" valign="top"><P><STRONG>Units</STRONG></P></TD></TR><TR><TD width="264" valign="top"><P>Gfp mRNA degradation/dilution</P></TD><TD width="170" valign="top"><P>0.0022</P></TD><TD width="123" valign="top"><P>molecules/sec</P></TD></TR><TR><TD width="264" valign="top"><P>Gfp protein    degradation/dilution</P></TD><TD width="170" valign="top"><P>0.0013（est）</P></TD><TD width="123" valign="top"><P>molecules/sec</P></TD></TR></TBODY></TABLE></DIV><H2>Cellular Equations</H2><P> Using the parameters found above, we can  construct a series of ordinary differential equations:</P><P>The  system described by these equations can be represented diagramatically by  SimBiology:</P><H2>Result</H2><P>The simulation was run with an initial ethylene  concentration of 20 mM, over a time period of 10 hr, The graphs below show the  time course concentrations of each species, with species of similar  concentrations being plotted on the separate axes.</P><P align="left"> </P><H2>Reference:</H2><OL><LI><P>[1] Tanja Burgdorf, Oliver  Lenz, Thorsten Buhrke: [NiFe]-Hydrogenases of Ralstonia eutropha H16:
             Modular Enzymes for Oxygen-Tolerant  Biological Hydrogen Oxidation. J Mol Microbiol Biotechnol 2005;10:181–196; DOI:  10.1159/000091564
             [2] Péter Érdi; János Tóth: Mathematical  Models of Chemical Reactions: Theory and Applications of Deterministic and  Stochastic Models.1989 Manchester University Press. p. 3.  ISBN 978-0-7190-2208-1.
             [3] Coval, ML: Analysis of Hill interaction  coefficients and the invalidity of the Kwon and Brown equation.1970, J. Biol.  Chem. 245 (23): 6335–6. PMID 5484812
             [4] Nelson, David L.; Cox, Michael M.  (2013). Lehninger principles of biochemistry (6th ed.). New York: W.H. Freeman.  pp. 158–162. ISBN 978-1429234146.
             [5] S Proshkin, AR Rahmouni, A Mironov, E  Nudler: Cooperation Between Translating Ribosomes and RNA Polymerase in  Transcription Elongation. Science, 2010, 328 (5977) :504
             [6] R Young, H Bremer:  Polypeptide-chain-elongation rate in Escherichia coli B/r as a function of  growth rate. Biochemical Journal, 1976 , 160 (2) :185
             [7] Selinger, D. W. :Global RNA Half-Life  Analysis In Escherichia Coli Reveals Positional Patterns Of Transcript  Degradation. Genome Research 13.2 (2003): 216-223. Web. 18 Oct. 2016.</P></LI></OL><P> </P></DIV></DIV></DIV><DIV class="footer"><DIV align="center"><DIV class="f"><DIV class="f-word"><P><STRONG><U>Address</U></STRONG></P><P>Life Science Department
              #163 Xianlin Blvd, Qixia District
              Nanjing University
              Nanjing, Jiangsu Province
              P.R. of China
              Zip: 210046
            </P><P><STRONG><U>Email</U></STRONG></P><P>nanjing_china@163.com</P><P><STRONG><U>Social media</U></STRONG></P><P>Twitter button: <A href="https://twitter.com/Guja1501">iGEM Nanjing-China@nanjing_china</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>