Team:Nanjing-China/Model/h2s
Team:Nanjing-China - 2017.igem.org
HOME
Introduction
Background
PARTS
JUDGE
Sliver
Gold
PROJECT
Design
Notebook
Results
Demonstrate
Improvement
TEAM
Introduction
Members
Attributions
Collaboration
OTHERS
HP-Silver
HP-Gold
Model
The Signalling Cascade
sdSQR  is a NAD(P)/FAD-dependent oxidoreductase, which oxidizes S2- to  zero-valent S0. SqrR, encoded by sqrR,  is a constitutively expressed suppressor protein of sqr promotor. When there is no H2S in the environment,  SqrR combines with the sqr promotor  and keeps the expression of the enzyme at a low level; while in the case where H2S  exists, it will be oxidized by the small amount of expressed enzyme, and the  product will interact with SqrR, forming a tri- or tetrasulfide cross-links  linking C41 and C107 on the same subunit of SqrR. The change in the structure  of SqrR makes it fall off from the sqr promotor and thus the expression of sqr turns on. Overall, SQR promotes the expression of itself by producing  substances that help with disinhibition[1,2].
Signalling Kinetics
In cell signaling, the enzyme sqr, catalyzes the oxidation of S2- to zero-valent S0, We model this rate using the Law of Michaelis–Menten kinetics, this model involves  an enzyme, E, binding to a substrate, S, to form a complex, ES, which in turn  releases a product, P, regenerating the original enzyme. This may be  represented schematically as
where (forward rate), (reverse  rate), and (catalytic  rate) denote the rate constants, the double arrows between S (substrate) and ES  (enzyme-substrate complex) represent the fact that enzyme-substrate binding is  a reversible process, and the single forward arrow represents the formation of  P (product).
Under certain  assumption– such as the enzyme concentration being much less than the substrate  concentration – the rate of product formation is given by[3]
Applying  these principles to the reaction:
Signalling Cascade Model Parameters
Value
Units
kcat
the turnover number
1.5X10^2（est）
km
The substrate concentration at which the    reaction rate is half of
0.015（est）
In cell  signalling, . S will interact with  SqrR, forming a tri- or tetrasulfide cross-links linking C41 and C107 on the  same subunit of SqrR. The change in the structure of SqrR makes it fall off  from the sqr promotor and thus the  expression of sqr turns on.
We model this rate using the Law of Mass Action kinetics. This principle  governs that the rate of a reaction (d/dt) is proportional to the  concentrations of the substrates.
For a particular reaction, for example:
the rate can be written mathematically as:
But most biological reactions are reversible, like this:
so
Applying these principles to the  interaction betw
een S and sqrR:
 
Model Parameters
Value
Units
Kf1
sqrR binding constant
13000（est）
Kr1
SqrR dissociation
220
 
Transcription
Transcription is a complex procession.  However, it is well established that initiation of transcription is the major  factor in determining overall rate of mRNA production. Transcription initiation  is a process influenced by a number of key steps including: promoter escape,  binding of the RNA Polymerase onto the promoter, isomerization, and formation  of the closed RNA Pol complex from the open complex, and your Escherichia coli strain. In this model,  we will assume that binding of the RNA polymerase is the rate determining step  in transcription initiation.
Thus,  our key assumptions for transcription are:
 1. For the constitutively expressed gene sqrR, we can assume that they are at maximal transcription.
2. transcription rate is determined  primarily by the disassociation of SqrR.
 3. Promoter strength (in Polymerases  per second) is a constant, and represents the rate of transcription at maximal  expression.
We  modeltranscription rate using Mass Action kinetics..
Model Parameters
Value
Units
 PsqrR strength
0.12
mRNA molecules/sec
kf2
S binding constant
970(est)
kr2
S dissociation
25（est）
Translation
After transcription,  it must be translated into a functional chromo protein. Translation involves 3  main stages: initiation, elongation of the polypeptide, and termination. The  key molecular machinery driving this process are ribosomes, who's binding and  movements determine the overall rate of transcription.
Translation rate can be affected by 3 major processes:
a) the binding of the ribosome to the RBS,
b) formation of initiation complexes,
c) speed of the  ribosome across the mRNA transcript.
we will assume that the movement of the  ribosome (in amino acids formed per second) is the rate determining step in the  translation of the proteins, and the average  translation rate of 17.1 amino acids/sec.[4]
Product
Length (amino    acids)
Translation rate    (1/s)
Sqr
427
0.040
sqrR
124
0.137
Mrfp
236
0.072
Degradation and Dilution
The concentrations of each species in the  model is affected by it’s degradation (inherent to the molecule) and dilution  (due to cell growth). While these processes are completely unrelated, they have  the same net effect: Decreasing the concentration of the species. Therefore, we  represent the two rates as a combined degradation/dilution parameter in our  model. we assume only the degradation/dilution of mrfp mRNA and mrfp.
Name
Value
Units
Mrfp mRNA degradation/dilution
0.0022
molecules/sec
Mrfp protein    degradation/dilution
0.0013（est）
molecules/sec
Cellular Equations
Using the parameters found above, we can  construct a series of ordinary differential equations:
The system described by these equations can be represented diagramatically by SimBiology:
Results
We  assume the initial sqrR protein concentration was 0.2 mM, and the initial  concentration of binding  complex was 0.1mM, and the sqr mRNA as well as mecherry rfp mRNA was 0.01mM. The  initial  concentration was 20 mM for a 10 hour period  of time.
Reference:
[1] Takayuki  Shimizu, Jiangchuan Shen, Mingxu Fang, Yixiang Zhang, Koichi Hori, Jonathan C.  Trinidad, Carl E. Bauer, David P. Giedroc, and Shinji Masuda. (2017). Sulfide-responsive  transcriptional repressor SqrR functions as a master regulator of  sulfide-dependent photosynthesis. PNAS,  114(9):2355-2360.
[2] Christoph Griesbeck, Günter Hauska and  Michael Schütz. (2000). Biological Sulfide-Oxidation: Sulfide-Quinone Reductase  (SQR), the Primary Reaction. Recent  Research Developments in Microbiology, 4:179-203.
[3] Nelson, David L.; Cox, Michael M.  (2013). Lehninger principles of biochemistry (6th ed.). New York: W.H. Freeman.  pp. 158–162. ISBN 978-1429234146.
[4]    Young,  R and H Bremer: Polypeptide-Chain-Elongation Rate In Escherichia Coli B/R As A  Function Of Growth Rate. Biochem. J. 160.2 (1976): 185-194. Web.
Address
Life Science Department
#163 Xianlin Blvd, Qixia District
Nanjing University
Nanjing, Jiangsu Province
P.R. of China
Zip: 210046
Email
nanjing_china@163.com
Social media
Twitter button:
iGEM Nanjing-China@nanjing_china
