<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Nanjing-China_Model_ch2o skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Nanjing-China/Model/ch2o</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:Nanjing-China - 2017.igem.org</TITLE></P><DIV class="container" align="center"><DIV class="header"><DIV id="menu"><UL><LI><A href="https://2017.igem.org/Team:Nanjing-China">HOME</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China">Introduction</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Background">Background</A></LI></UL><LI><A href="https://2017.igem.org/Team:Nanjing-China/Parts">PARTS</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Sliver">JUDGE</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Sliver">Sliver</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Gold">Gold</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Design">PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Improvement">Improvement</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/TEAM/Introduction">TEAM</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/TEAM/Introduction">Introduction</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Members">Members</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Collaborations">Collaboration</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/HP/Silver">OTHERS</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/HP/Silver">HP-Silver</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/HP/Gold_Integrated">HP-Gold</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Model">Model</A></LI></DIV></DIV><DIV class="contain"><DIV id="ch2o"><DIV class="word"><DIV align="left"><H2>The Signalling Cascade</H2><P>Protein FrmR acts as a regulator, it can  form DNA-protein cross-links with a special promotor P<SUB>frmAB</SUB> to  prevent activating downstream transcription of Mecherry RFP gene. Formaldehyde  is able to covalently modify protein FrmR, inhibiting its function, so when  formaldehyde exists, promotor P<SUB>frmAB</SUB> is capable of activating RFP by  the destruction of Promoter<SUB>frmAB-</SUB>- FrmR protein  cross-link. Therefore Promoter<SUB>frmAB</SUB> is activated and promote the  transcription of Mecherry rfp gene.</P><H2>Kinetics of Signalling</H2><P>In cell signaling, a ligand formaldehyde  binds to its receptor site FrmR, which triggers a conformational change in the  receptor. This leads to a change in functionality. The formaldehyde bind their  receptors at different rates and affinities according to the molecules  involved.
These binding rates are critical because  they determine the rate at which the biosensor responds to the Formaldehyde it  detects. Low rates will lead to low expression, high rates will lead to high  expression. We use the Law of Mass Action kinetics model to model these rates.  The principle determines that the reaction rate
(d / dt) is proportional to the concentration  of the substrate.</P><P>For specific reaction, for example:</P><P>The rate can be mathematically converted  into:</P><P>But most of the biological reactions are  reversible, so when we want to model the reaction speed, we must take this into  account by adding an additional term.</P><P>Applying these principles to the binding of  FrmR and Promoter frmAB yields:</P><H2>Transcription </H2>
Transcription rates may be affected by  various factors. However, it has been determined that transcription initiation  is a major factor in determining the overall rate of mRNA production.  Transcription initiation is a process that involves several key steps,  including: promoter escape (Kugel &amp; Goodrich, 1998),[2] RNA polymerase  binding to the promoter, isomerization and formation of closed RNA Pol  complexes from open complexes (Hakkinen et al. 2013).[3] In this model, we  assume that the binding of RNA polymerase is the rate-determining step of  transcription initiation and is therefore the rate determination step for the  entire transcription.<P><STRONG>Thus,  our key assumptions for transcription are: </STRONG>
            1.  The total transcription rate is mainly determined by the amount of substrate – FrmR dissociates with promoter<SUB>frmAB</SUB> and the intensity of the promoter. When all  binding sites are destroyed, the maximum transcription rate occurs.
            2.The promoter intensity (in polymerase per second) is a constant that  represents the maximum expression rate of transcription.
            3.  Thus, the total transcription rate is the function of the maximum transcription  rate and the fraction of the signal molecule bound to its binding site.
            4.  For constitutive expression of genes - FrmR, we can assume that it is  the maximum transcript of all t.</P><P><STRONG>①Finding parameters for </STRONG><STRONG>FrmR (Constitutive Transcription)</STRONG>
            In our final construct, FrmR is  placed under the control of promoterfrmR, and Mecherry rfp is placed  under the control of promoter<SUB>frmAB</SUB>. </P><P><STRONG>②Finding parameters for the Mecherryrfp  reporter protein (Induced Transcription)</STRONG>
            For our reporter genes, the model for  expression is complicated by the binding of signalling molecules. In contrast  to the constitutively expressed genes, we must assume that the transcription  rate is a function of the amount of signalling molecules that are bound. </P><P>This rate can be modelled as below:</P><DIV align="center"><P align="center">Promoter <SUB>frmAB</SUB> − FrmR +  Formaldehyde → FrmR − Formaldehyde + Promoter<SUB>frmAB</SUB></P><TABLE border="1" cellspacing="0" cellpadding="0" width="0"><TBODY><TR><TD width="256" valign="top"><P align="center"><STRONG>Signalling Cascade Model Parameters</STRONG></P></TD><TD width="178" valign="top"><P align="center"><STRONG>Value</STRONG></P></TD><TD width="96" valign="top"><P align="center"><STRONG>Units</STRONG><STRONG>[1]</STRONG></P></TD></TR><TR><TD width="256" valign="top"><P align="left">K<SUB>f2</SUB>(M <SUP>−1</SUP>s<SUP>−1</SUP> ) b </P></TD><TD width="178" valign="top"><P align="left">Formaldehyde binding constant </P></TD><TD width="96" valign="top"><P align="left">5660 </P></TD></TR><TR><TD width="256" valign="top"><P align="left">K<SUB>r2</SUB>(s<SUP>−1</SUP>) a </P></TD><TD width="178" valign="top"><P align="left">Formaldehyde dissociation </P></TD><TD width="96" valign="top"><P align="left">0.003 </P></TD></TR></TBODY></TABLE></DIV><H2>Translation</H2><P>After the formation of Mecherry rfp mRNA transcripts,  it must be translated into functional chromogenic proteins. The translation  involves three major phases: the initiation, extension and termination of the  polypeptide. The key molecular mechanism driving the process is ribosomes,  whose binding and movement determine the overall transcription rate.</P><P><STRONG>Translation rate can be affected by 3  major processes: </STRONG></P><OL><LI>The binding of the ribosome to  the RBS</LI><LI>Formation of initiation  complexes</LI><LI>Rate of the ribosome across the  mRNA transcript (Proshkin et al. 2010).[4]</LI></OL><P>In this model, we will assume that the movement of ribosomes (in  amino acids formed per second) is the rate determination step in protein  translation.</P><P><STRONG>Thus, our major assumptions for  translation are: </STRONG></P><OL><LI>The main factor influencing the translation rate is the rate of  ribosomes across the mRNA, which can be approximated by constant rates.</LI><LI>The translation rate is limited by the amount of available mRNA  transcripts that bind with ribosome.</LI><LI>Thus, the translation rate is a function of the ribosome velocity and  the mRNA transcript concentration.</LI></OL><DIV><H4>Finding parameters  for translation </H4></DIV><P>The  literature search showed an average translation rate of 17.1 amino acids per  second (Young and Bremer, 1976[5] and Proshkin et al., 2010[4]), so the  transcription factor would depend on the length of the mRNA transcript divided  by 17.1 in the table below.          </P><DIV align="center"><P> </P><TABLE border="1" cellspacing="0" cellpadding="0" width="0"><TBODY><TR><TD width="91" valign="top"><P align="center"><STRONG>Protein</STRONG></P></TD><TD width="137" valign="top"><P align="center"><STRONG>Length (amino acids)</STRONG></P></TD><TD width="139" valign="top"><P align="center"><STRONG>Translation rate (1/s)</STRONG></P></TD></TR><TR><TD width="91" valign="top"><P align="left">FrmR</P></TD><TD width="137" valign="top"><P align="left">101 </P></TD><TD width="139" valign="top"><P align="left">0.168 </P></TD></TR><TR><TD width="91" valign="top"><P align="left">Mecherry rfp </P></TD><TD width="137" valign="top"><P align="left">236 </P></TD><TD width="139" valign="top"><P align="left">0.072 </P></TD></TR></TBODY></TABLE><P> </P></DIV><P>These translation rate coefficients are  transmitted by a single mRNA molecule in the ribosome. In fact, our cells will  contain multiple copies of the same mRNA molecule. We multiply these  translation rate coefficients by the total number of mRNA molecules to get the  total translation rate per second of each DNA construct.</P><H2><STRONG>Degradation and Dilution</STRONG></H2><P>The concentration of each species in the  model is affected by its degradation (molecular inherent) and dilution (due to  cell growth). Although these processes are completely unrelated, they have the  same net effect: to reduce the concentration of species. Thus, we represent  these ratios as the combined degradation / dilution parameters in our model.
            The life of the mRNA is usually very  limited. mRNA is degraded prior to any significant dilution compared to cell  growth rate or cell dilution.
            The opposite is true for proteins. While  proteins are victims of degradation, it is not the major factor in the  reduction of a protein’s concentration. Proteins tend to far outlive the mRNA  molecules that produced them. It is often the changes in the protein’s  environment that lead to it’s decrease in concentration. As a cell grows in  size, the amount of Mecherry rfp inside it remains relatively constant, but the  concentration of Mecherry rfp decreases. 
            The opposite is true for proteins. Although  the protein is the victim of degradation, it is not a major factor in reducing  protein concentration. Proteins tend to go far beyond the mRNA molecules that  produce them. Changes in the protein environment often result in a decrease in  concentration. When the cell growth size, the internal Mecherry rfp volume  remained relatively constant, but the concentration of Mecherry rfp decreased.
          In our model, we only consider the  degradation / dilution of Mecherry rfp mRNA and Mecherry rfp . In any case, the  following sensitivity analysis shows that degradation and dilution have minimal  impact on overall biosensor performance compared to other more important  parameters.          </P><DIV align="center"><TABLE border="1" cellspacing="0" cellpadding="0" width="0"><TBODY><TR><TD width="252" valign="top"><P><STRONG>Name</STRONG></P></TD><TD width="61" valign="top"><P><STRONG>Value</STRONG></P></TD><TD width="101" valign="top"><P><STRONG>Units</STRONG></P></TD></TR><TR><TD width="252" valign="top"><P>Mecherry rfp mRNA degradation/dilution </P></TD><TD width="61" valign="top"><P>0.0022</P></TD><TD width="101" valign="top"><P>molecules/sec </P></TD></TR><TR><TD width="252" valign="top"><P>Mecherry rfp degradation/dilution </P></TD><TD width="61" valign="top"><P>0.0012</P></TD><TD width="101" valign="top"><P>Molecules/sec </P></TD></TR></TBODY></TABLE></DIV><H2>Cellular  Equations</H2><P>Using the above parameters, we can  construct a series of ordinary differential equations representing our  biosensor network.</P><P>The system described by these equations can  be represented diagramatically by SimBiology: </P><H2>Result</H2><P align="left">Our model shows the process of formaldehyde  induction after the promter<SUB>frmAB</SUB>- FrmR binding has been constructed. We  assume the initial FrmR concentration was 0.1 mM, and the initial concentration  of promter<SUB>frmAB</SUB>- FrmR binding complex was 0.1mM.The initial  formaldehyde concentration was 20 mM for a 10 hour period of time. 
		  The following figure shows the time course  concentration of each species, where similar concentrations of material are  plotted on separate axes.</P><P> </P><H2>Reference:</H2><OL><LI>Denby, K. J. et al: The  mechanism of a formaldehyde-sensing transcriptional regulator. Sci. Rep.  10.1038 (2016). web. 09 Dec. 2016.</LI><LI>Kugel J. F. and J. A. Goodrich:  Promoter Escape Limits The Rate Of RNA Polymerase II Transcription And Is  Enhanced By TFIIE, TFIIH, And ATP On Negatively Supercoiled DNA. Proceedings of  the National Academy of Sciences 95.16 (1998): 9232-9237. Web.</LI><LI>Häkkinen, Antti et al: Effects  Of Rate-Limiting Steps In Transcription Initiation On Genetic Filter  Motifs&quot;. PLoS ONE 8.8 (2013): e70439. Web.</LI><LI>Proshkin, S. et al: Cooperation  Between Translating Ribosomes And RNA Polymerase In Transcription Elongation.  Science 328.5977 (2010): 504-508. Web.</LI><LI>Young, R and H Bremer:  Polypeptide-Chain-Elongation Rate In Escherichia Coli B/R As A Function Of  Growth Rate. Biochem. J. 160.2 (1976): 185-194. Web.</LI></OL><P> </P></DIV></DIV></DIV><DIV class="footer"><DIV align="center"><DIV class="f"><DIV class="f-word"><P><STRONG><U>Address</U></STRONG></P><P>Life Science Department
              #163 Xianlin Blvd, Qixia District
              Nanjing University
              Nanjing, Jiangsu Province
              P.R. of China
              Zip: 210046
            </P><P><STRONG><U>Email</U></STRONG></P><P>nanjing_china@163.com</P><P><STRONG><U>Social media</U></STRONG></P><P>Twitter button: <A href="https://twitter.com/Guja1501">iGEM Nanjing-China@nanjing_china</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>