<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Nanjing-China_Model_h2s skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Nanjing-China/Model/h2s</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:Nanjing-China - 2017.igem.org</TITLE></P><DIV class="container" align="center"><DIV class="header"><DIV id="menu"><UL><LI><A href="https://2017.igem.org/Team:Nanjing-China">HOME</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China">Introduction</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Background">Background</A></LI></UL><LI><A href="https://2017.igem.org/Team:Nanjing-China/Parts">PARTS</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Sliver">JUDGE</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Sliver">Sliver</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Gold">Gold</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Design">PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Improvement">Improvement</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/TEAM/Introduction">TEAM</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/TEAM/Introduction">Introduction</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Members">Members</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Collaborations">Collaboration</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/HP/Silver">OTHERS</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/HP/Silver">HP-Silver</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/HP/Gold_Integrated">HP-Gold</A></LI><LI><A href="https://2017.igem.org/Team:Nanjing-China/Model">Model</A></LI></DIV></DIV><DIV class="contain"><DIV id="h2s"><DIV class="word"><DIV align="left"><H2>The Signalling Cascade</H2><P>sdSQR  is a NAD(P)/FAD-dependent oxidoreductase, which oxidizes S<SUP>2-</SUP> to  zero-valent S<SUP>0</SUP>. SqrR, encoded by <EM>sqrR</EM>,  is a constitutively expressed suppressor protein of <EM>sqr</EM> promotor. When there is no H2S in the environment,  SqrR combines with the <EM>sqr</EM> promotor  and keeps the expression of the enzyme at a low level; while in the case where H<SUB>2</SUB>S  exists, it will be oxidized by the small amount of expressed enzyme, and the  product will interact with SqrR, forming a tri- or tetrasulfide cross-links  linking C41 and C107 on the same subunit of SqrR. The change in the structure  of SqrR makes it fall off from the <EM>sqr</EM> promotor and thus the expression of <EM>sqr</EM> turns on. Overall, SQR promotes the expression of itself by producing  substances that help with disinhibition[1,2].</P><H2>Signalling Kinetics</H2><P>In cell signaling, the enzyme <EM>sqr</EM>, catalyzes the oxidation of S<SUP>2-</SUP>  to zero-valent S<SUP>0</SUP>, We model this rate using the Law of Michaelis–Menten kinetics, this model involves  an enzyme, E, binding to a substrate, S, to form a complex, ES, which in turn  releases a product, P, regenerating the original enzyme. This may be  represented schematically as </P><P>where (forward rate), (reverse  rate), and (catalytic  rate) denote the rate constants, the double arrows between S (substrate) and ES  (enzyme-substrate complex) represent the fact that enzyme-substrate binding is  a reversible process, and the single forward arrow represents the formation of  P (product).
        Under certain  assumption– such as the enzyme concentration being much less than the substrate  concentration – the rate of product formation is given by[3]</P><P>Applying  these principles to the reaction:</P><DIV align="center"><TABLE border="1" cellspacing="0" cellpadding="0" width="0"><TBODY><TR><TD width="245" valign="top"><P><STRONG>Signalling Cascade Model Parameters</STRONG></P></TD><TD width="198" valign="top"><P><STRONG>Value</STRONG></P></TD><TD width="113" valign="top"><P><STRONG>Units</STRONG></P></TD></TR><TR><TD width="245" valign="top">k<SUB>cat</SUB></TD><TD width="198" valign="top"><P>the turnover number</P></TD><TD width="113" valign="top"><P>1.5X10<SUP>^2</SUP>（est）</P></TD></TR><TR><TD width="245" valign="top">k<SUB>m</SUB></TD><TD width="198" valign="top"><P>The substrate concentration at which the    reaction rate is half of </P></TD><TD width="113" valign="top"><P>0.015（est）</P></TD></TR></TBODY></TABLE></DIV><P>In cell  signalling, . S will interact with  SqrR, forming a tri- or tetrasulfide cross-links linking C41 and C107 on the  same subunit of SqrR. The change in the structure of SqrR makes it fall off  from the <EM>sqr</EM> promotor and thus the  expression of <EM>sqr</EM> turns on. 
		  We model this rate using the Law of Mass Action kinetics. This principle  governs that the rate of a reaction (d/dt) is proportional to the  concentrations of the substrates.
		  For a particular reaction, for example:</P><P>the rate can be written mathematically as:</P><P>But most biological reactions are reversible, like this:</P><P>so</P><P>Applying these principles to the  interaction between S and sqrR:</P><DIV align="center"><P> </P><TABLE border="1" cellspacing="0" cellpadding="0" width="0"><TBODY><TR><TD width="236" valign="top"><P><STRONG>Model Parameters</STRONG></P></TD><TD width="180" valign="top"><P><STRONG>Value</STRONG></P></TD><TD width="142" valign="top"><P><STRONG>Units</STRONG></P></TD></TR><TR><TD width="236" valign="top"><P>K<SUB>f1</SUB></P></TD><TD width="180" valign="top"><P>sqrR binding constant</P></TD><TD width="142" valign="top"><P>13000（est）</P></TD></TR><TR><TD width="236" valign="top"><P>K<SUB>r1</SUB></P></TD><TD width="180" valign="top"><P>SqrR dissociation</P></TD><TD width="142" valign="top"><P>220</P></TD></TR></TBODY></TABLE><P> </P></DIV><H2>Transcription</H2><P>Transcription is a complex procession.  However, it is well established that initiation of transcription is the major  factor in determining overall rate of mRNA production. Transcription initiation  is a process influenced by a number of key steps including: promoter escape,  binding of the RNA Polymerase onto the promoter, isomerization, and formation  of the closed RNA Pol complex from the open complex, and your <EM>Escherichia coli</EM> strain. In this model,  we will assume that binding of the RNA polymerase is the rate determining step  in transcription initiation.</P><P><STRONG>Thus,  our key assumptions for transcription are: </STRONG> 1. For the constitutively expressed gene <EM>sqrR</EM>, we can assume that they are at maximal transcription.
            2. transcription rate is determined  primarily by the disassociation of SqrR. 3. Promoter strength (in Polymerases  per second) is a constant, and represents the rate of transcription at maximal  expression. 
          We  modeltranscription rate using Mass Action kinetics.. </P><DIV align="center"><TABLE border="1" cellspacing="0" cellpadding="0" width="0"><TBODY><TR><TD width="236" valign="top"><P><STRONG>Model Parameters</STRONG></P></TD><TD width="180" valign="top"><P><STRONG>Value</STRONG></P></TD><TD width="142" valign="top"><P><STRONG>Units</STRONG></P></TD></TR><TR><TD width="236" valign="top"><P> P<SUB>sqrR</SUB> strength</P></TD><TD width="180" valign="top"><P>0.12</P></TD><TD width="142" valign="top"><P>mRNA molecules/sec</P></TD></TR><TR><TD width="236" valign="top">k<SUB>f2</SUB></TD><TD width="180" valign="top"><P>S binding constant</P></TD><TD width="142" valign="top"><P>970(est)</P></TD></TR><TR><TD width="236" valign="top"><P>k<SUB>r2</SUB></P></TD><TD width="180" valign="top"><P>S dissociation</P></TD><TD width="142" valign="top"><P>25（est）</P></TD></TR></TBODY></TABLE></DIV><H2>Translation</H2><P>After transcription,  it must be translated into a functional chromo protein. Translation involves 3  main stages: initiation, elongation of the polypeptide, and termination. The  key molecular machinery driving this process are ribosomes, who's binding and  movements determine the overall rate of transcription. 
            Translation rate can be affected by 3 major processes:
            a) the binding of the ribosome to the RBS,
            b) formation of initiation complexes, 
            c) speed of the  ribosome across the mRNA transcript.
          we will assume that the movement of the  ribosome (in amino acids formed per second) is the rate determining step in the  translation of the proteins, and the average  translation rate of 17.1 amino acids/sec.[4] </P><DIV align="center"><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="138" valign="top"><P align="center"><STRONG>Product</STRONG></P></TD><TD width="211" valign="top"><P align="center"><STRONG>Length (amino    acids)</STRONG></P></TD><TD width="198" valign="top"><P align="center"><STRONG>Translation rate    (1/s)</STRONG></P></TD></TR><TR><TD width="138" valign="top"><P align="center">Sqr</P></TD><TD width="211" valign="top"><P align="center">427</P></TD><TD width="198" valign="top"><P>0.040</P></TD></TR><TR><TD width="138" valign="top"><P align="center">sqrR</P></TD><TD width="211" valign="top"><P align="center">124</P></TD><TD width="198" valign="top"><P>0.137</P></TD></TR><TR><TD width="138" valign="top"><P align="center">Mrfp</P></TD><TD width="211" valign="top"><P align="center">236</P></TD><TD width="198" valign="top"><P>0.072</P></TD></TR></TBODY></TABLE></DIV><H2>Degradation and Dilution</H2><P>The concentrations of each species in the  model is affected by it’s degradation (inherent to the molecule) and dilution  (due to cell growth). While these processes are completely unrelated, they have  the same net effect: Decreasing the concentration of the species. Therefore, we  represent the two rates as a combined degradation/dilution parameter in our  model. we assume only the degradation/dilution of mrfp mRNA and mrfp.</P><DIV align="center"><TABLE border="1" cellspacing="0" cellpadding="0" width="0"><TBODY><TR><TD width="264" valign="top"><P><STRONG>Name</STRONG></P></TD><TD width="170" valign="top"><P><STRONG>Value</STRONG></P></TD><TD width="123" valign="top"><P><STRONG>Units</STRONG></P></TD></TR><TR><TD width="264" valign="top"><P>Mrfp mRNA degradation/dilution</P></TD><TD width="170" valign="top"><P>0.0022</P></TD><TD width="123" valign="top"><P>molecules/sec</P></TD></TR><TR><TD width="264" valign="top"><P>Mrfp protein    degradation/dilution</P></TD><TD width="170" valign="top"><P>0.0013（est）</P></TD><TD width="123" valign="top"><P>molecules/sec</P></TD></TR></TBODY></TABLE></DIV><H2>Cellular Equations</H2><P>Using the parameters found above, we can  construct a series of ordinary differential equations:</P></DIV><P>The system described by these equations can be represented diagramatically by SimBiology:</P><H2>Results</H2><P>We  assume the initial sqrR protein concentration was 0.2 mM, and the initial  concentration of binding  complex was 0.1mM, and the sqr mRNA as well as mecherry rfp mRNA was 0.01mM. The  initial  concentration was 20 mM for a 10 hour period  of time.</P><H2>Reference:</H2><P>[1] Takayuki  Shimizu, Jiangchuan Shen, Mingxu Fang, Yixiang Zhang, Koichi Hori, Jonathan C.  Trinidad, Carl E. Bauer, David P. Giedroc, and Shinji Masuda. (2017). Sulfide-responsive  transcriptional repressor SqrR functions as a master regulator of  sulfide-dependent photosynthesis. <EM>PNAS</EM>,  114(9):2355-2360.
         [2] Christoph Griesbeck, Günter Hauska and  Michael Schütz. (2000). Biological Sulfide-Oxidation: Sulfide-Quinone Reductase  (SQR), the Primary Reaction. <EM>Recent  Research Developments in Microbiology</EM>, 4:179-203.
         [3] Nelson, David L.; Cox, Michael M.  (2013). Lehninger principles of biochemistry (6th ed.). New York: W.H. Freeman.  pp. 158–162. ISBN 978-1429234146.
        [4]    Young,  R and H Bremer: Polypeptide-Chain-Elongation Rate In Escherichia Coli B/R As A  Function Of Growth Rate. Biochem. J. 160.2 (1976): 185-194. Web.</P></DIV></DIV></DIV><DIV class="footer"><DIV align="center"><DIV class="f"><DIV class="f-word"><P><STRONG><U>Address</U></STRONG></P><P>Life Science Department
              #163 Xianlin Blvd, Qixia District
              Nanjing University
              Nanjing, Jiangsu Province
              P.R. of China
              Zip: 210046
            </P><P><STRONG><U>Email</U></STRONG></P><P>nanjing_china@163.com</P><P><STRONG><U>Social media</U></STRONG></P><P>Twitter button: <A href="https://twitter.com/Guja1501">iGEM Nanjing-China@nanjing_china</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>