"
Page
Discussion
View source
History
teams
Log in
Â 
Team:DTU-Denmark/pBAD SPL
From 2013.igem.org
Home
Team
Project
Description
Parts
Timeline
Notebook
pBAD SPL
Experiments
Modeling
Protocols
Hello World Pilot Project
Safety
Outreach
Overview
High School Outreach
IP and Synthetic Biology
BioBrick Workshop
Helping Other Teams
Attributions
iGEM
pBAD SPL
Contents
1 pBAD synthetic promoter library
2 Methods
2.1 Experimental procedure
2.2 Data analysis
3 Results
3.1 Summary
3.2 Details
4 Sequences
5 Example of use
6 References
pBAD synthetic promoter library
A synthetic promoter library (SPL) is a library of cells which contain a promoter that has been randomly mutated to give different expression levels. This promoter upstream of a fluorescent protein like GFP/RFP. The term was first coined in 1998 and used in Lactococcus lactis [1]. The method was adapted by 2010 DTU iGEM team to make a SPL for E. coli that enabled the modulation of constitutive gene expression with great precision. They even made a new standard for this method with the use of biobricks ( RFC 63 ).
We used the method to build a non-leaky arabinose inducible promoter as a tool for expressing lethal proteins in E. coli. The reason we did this is that many of the proteins we are working with are membrane bound or integral membrane proteins and will be lethal if expressed in high quantities.
For testing our pathways we needed to grow the transformed cells to a certain concentration, and attempting this with the genes constitutively expressed was often not yielding viable colonies.  We needed a inducible system, but the standard arabinose inducible system was way too leaky and we with this promoter we can into the same problem as before. Therefore we needed to either build or buy (if possible) an inducible system with great tightness that is able to be induced easily. We choose to build such a system.
Methods
Experimental procedure
Random promoter sequences were ordered matching the sequence CTGACGNNNNNNNNNNNNNNNNNNTAWWATNNNNA.
USER cloning to add RFP downstream of promoter.
Colonies were plated.
After inspection under UV-light the non RFP containing colonies (via visual inspection) where picked.
Plates were induced by spraying them with an 5% w/v aqueous arabinose solution.
The plates were again inspected under UV-light and this time the most red florescent cells were picked.
Colonies were grown in culture tubes and screened in parallel on the BioLector. Wells on BioLector plate were loaded with culture by transferring a toothpick from each overnight culture selected and into the wells of the plate. All wells were run in duplicate.
All duplicate colonies were run twice -- once with arabinose added at t=0, and again without arabinose.
The pBAD system BBa_K808000 was used as a reference.
This reverse primer sequence is incorporating randomized promoter sequences into the pBAD construct. The sequence is annotated with -10 and -35 consensus region. Note that I2-bindingsite is overlapping with the -35 region. Blue is the binding part of the primer, red is the USER made sticky ends. N=random, W=50% A and 50% T
Data analysis
Data was collected from the Biolector, and analyzed using a series of R scripts written by Chris Workman (unpublished).
The maturation and degradation times for mCherry were both assumed to be 40 min.
The growth rate, mu, was estimated to be 1.28 (from an average of all wells on all plates) since we expect each strain to grow at the same rate.
A time window representing exponential growth was selected (between 4 and 8 hours).
The RFP measurement for a constitutively expressed strain was used as a standard measure of growth.  This is plotted on the x-axis in the detailed plots per colony below.
Figures were plotted using R.
Results
Summary
Promoter activity when induced (with arabinose added) plotted vs basal activity (without arabinose; ie leakiness of the promoter).  The colonies that we selected all show less activity than the the constitutive promoter, and when induced, show higher activity than the constitutive promoter.  The induced activity of the arabinose reference promoter is shown as a band representing two replicates.
Details
Promoter strengths for two trials of each colony with and without arabinose.
Identifier
Sequencea
Induced Strength
Leakiness
Inducibilityb
Col. 02
CTGACGACGCCCCTCTCCGCCCCTTAAAATGTCCA
14.2
0.203
69.9
Col. 03
CTGACGTAATCATACCGCCGAAAGTATTATCATTA
16.6
0.426
39
Col. 04
CTGACGCGACAACATTGCGTCCTATAAAATGCCGA
14.2
0.345
41
Col. 05
CTGACGGCCCGCCCCCGATGCGCATAAAATACCAA
17.7
0.424
41.6
Col. 12
CTGACGCGTATCGCGAGCGGGCGTTATTATACGCA
13.6
0.414
32.9
Col. 10
CTGACGCCCCGAATCAGTAGTATTTATTATCTAGA
18
0.462
38.8
Col. 09
CTGACGCCACCCCCCCCCCCGGCGTATAATTCCCA
16.1
0.530
30.4
Col. 08
CTGACGCCCTACCGCTCGCCCCCCTATTATACCCA
13.4
0.211
63.4
Col. 13
CTGACGCCCCCCCCGTCCACCCCCTAATATCCCGA
10.9
0.096
113.6
Col. 15
CTGACGTGCGCCTGCCGTCCAAAGTAATATCCTTA
15.5
0.235
65.8
Col. 18
CTGACGCCCCGAATCAGTAGTATTTATTATCTAGA
19.8
0.524
37.8
Col. 19
CTGACGCTCGCACCGGCGACCCGATAATATCCATA
18.0
0.315
57
Col. 31
CTGACGCCCCCCCTCCCCCGACCTAAAATATCCAG
3.8
0.161
23.6
Col. 29
CTGACGCCGCTGCACCCGTCCCCCTATTATAGCAA
14.8
0.301
49.2
Col. 26
CTGACGCCGATCCTATCTCCCTATTAAAATTTTTA
13.8
0.292
47.2
Col. 22
CTGACGCCCCGAATCAGTAGTATTTATTATCTAGA
16.8
0.466
36.1
Col. 33
CTGACGCCAGCTTCCTACTTCAAATATAATCCGTA
14.4
1.01
14.3
Col. 34
CTGACGTCCCTCTTCCGCGCCCCCTAAAATACCCA
12.4
0.187
65.9
Col. 35
CTGACGCCCAACCCGACACAGCGATATAATAATGA
18.4
0.89
20.7
a The sequence of each pBAD promoter characterized. To use this simply produce a primer with the promoter sequence you wish in a tail. Replace the old promoter in your system e.g. BBa_K808000 with the new and see the change.
b Inducibility = Induced Strength divided by leakiness.
Sequences
We sequenced and aligned the promoters that are shown above.  The sequences show conservation in the -10 and -35 regions by design.  Within the -10 region, we allowed for two random weak (A or T) bases.  We do not see a strong preference for A over T or vice versa.  Within the regions where we allowed any base, we see a preference for C over the other bases.  This could be due to bias during synthesis.
Example of use
The tight inducible pBAD promoter was used in our "Hello World project" to regulate the expression of GFP SF, which was tagged with a signal peptide to direct it into the periplasm. Production and folding of GFP SF is faster than the transport system of E. coli, which leads to undesired accumulation of GFP SF in the cytoplasm. Only when using a promoter with low leakiness it is possible to translocate a significant fraction of GFP SF after its production has been switched off. Thereby we get a clear signal from the periplasm with low interference from the cytoplasm.
To incorporate the new promoter sequence, we ordered primers containing the sequence as a tail and used USER cloning to insert it into our construct.
Zoomed in picture of "Hello World" transformants, showing a clear separation of green and red fluorescence. GFP is primarily located in the periplasm while RFP is located in the cytoplasm. Fluorescence intensity measurements are taken along the cross-section indicated by the white line. Picture taken with a fluorescence microscope and subjected to background subtraction
Line profile through the cell. The profile shows both the green and red channel. It can be seen that the intensity of the red fluorescence is restricted to the cytoplasm while green fluorescence has it's peaks on the edges. A weak green signal is measured for the cytoplasmic region caused by the periplasm enveloping the cytoplasm.
See also "Hello World project" .
References
[1] Jensen, Peter Ruhdal, and Karin Hammer. "The sequence of spacers between the consensus sequences modulates the strength of prokaryotic promoters." Applied and environmental microbiology 64.1 (1998): 82-87.
Sponsors
Retrieved from " http://2013.igem.org/Team:DTU-Denmark/pBAD_SPL "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
