"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU-Denmark/Notebook/12 July 2013
From 2013.igem.org
Home
Team
Project
Description
Parts
Timeline
Notebook
pBAD SPL
Experiments
Modeling
Protocols
Hello World Pilot Project
Safety
Outreach
Overview
High School Outreach
IP and Synthetic Biology
BioBrick Workshop
Helping Other Teams
Attributions
iGEM
12 July 2013
Navigate to the Previous or the Next Entry
Contents
1 Lab 208
1.1 Main purpose
1.2 Who was in the lab
1.3 Procedure
1.3.1 Plasmid isolation of biobricks from transformants made on 10-07-2013
1.3.2 Amplification of cytochromes, AMO, HAO and Nir genes
1.3.3 Gel analysis of PCR products from Nir operon, AraBAD promoter and RFP
1.3.4 USER reaction of RFP and pZA21 (with native promoter) and transformation of E. coli cells
1.3.4.1 Colony PCR to extract the Nir operon directly with USER-primers
1.4 Conclusion
Lab 208
Main purpose
miniprep of biobrick transformants from 10-07-2013
PCR with USER primers to amplify cytochromes, AMO, HAO  and Nir genes
gel analysis of PCR products from Nir operon , AraBAD promoter and RFP
USER reaction of RFP and pZA21 (with native promoter) and transformation of E. coli cells
Gel on todays PCR-reactions.
Who was in the lab
Henrike, Julia, Kristian, Jakob, Gosia
Procedure
Plasmid isolation of biobricks from transformants made on 10-07-2013
Isolation was performed according to protocol given in the kit PowerPrep HP Plasmid Miniprep Kit
Amplification of cytochromes, AMO, HAO and Nir genes
Each PCR reaction was performed in triplicate.
Samples are named:
1, 2, 3 → cytochromes
4, 5, 6 → AMO
7, 8, 9 → HAO
10, 11, 12 → Nir
PCR reaction mix was done according to standard procedure. Primers and templates used for samples are as follows:
1, 2, 3 → cytochromes, primers 16a, 16b, template CYC isolated by colony PCR from Nitrosomonas europea
4, 5, 6 → AMO, primers 17a, 17b, template AMO isolated by colony PCR from N. europea
7, 8, 9 → HAO, primers 18a, 18b, template HAO isolated by colony PCR from N. europea
10, 11, 12 → Nir, primers 15a, 15b, template - 2uL of liquid culture of Pseudosomonas aeruginosa
PCR programs based on standard program with differences in annealing temperature and elongation time as follows:
1, 2, 3 → cytochromes - 57°C, 1:30 min
4, 5, 6 → AMO - 54°C, 3 min
7, 8, 9 → HAO - 59°C, 9 min
10, 11, 12 → Nir - 59°C, 9 min
Gel analysis of PCR products from Nir operon, AraBAD promoter and RFP
1 % Agorase Gel (Nir operon)
Wells 2 - 7 contain the products of yesterday's PCR, Wells 8 - 14 contain PCR product of an unkown day (no note)
Wells
1: ladder 100bp NEB
2: RFP
3: RFP
4: AraBAD promoter
5: AraBAD promoter
6: AraBAD promoter
7: RFP
8: Nir
9: Nir
10: Nir
11:Nir
12: Nir
13: Nir
14: Nir
15: Ladder 1 kb
USER reaction of RFP and pZA21 (with native promoter) and transformation of E. coli cells
USER reaction was performed according to standard protocol.
USER mix:
USER enzyme    1 uL
NEB 4 buffer   0,5 uL
10x BSA        0,5 uL
backbone pZA21 1uL
Sample 1 - 3 uL of USER mix + 7 uL of RFP
Sample 2 - Negative control, USER mix + 7 uL water
USER reaction was performed in PCR machine with programm: 37C for 40 min and then 25C for 30 min.
To 100uL of chemically competent E. coli cells 10 uL of USER mix after USER reaction was added.
Incubation of cells on ice for 30 minutes, heat shock at 42C for 90 sec. and 5 min on ice. Addition of SOC medium and 2 hours of incubation at 37C. Plating on plates with LB medium with Kanamycin, overnight incubation at 37C.
Colony PCR to extract the Nir operon directly with USER-primers
We had to try to extract the Nir operon with USER-primers directly because we could not get anything useful with the Nir-template that we made with the flanking non-uracil containing primers.
Conclusion
We have USER fragments for RFP and AraBAD which was verified with the first gel but Nir operon is another story. The Nir primers seems to make lots of unspecific products. We tried to do the colony PCR with the USER primers directly and this gave fragments longer than expected but not with unspecific product though. We will try and transform this fragment into E.coli and see what will happen and then we will send the construct to sequencing so that we can see whats causing the elongation of the fragment.
Navigate to the Previous or the Next Entry
Sponsors
Retrieved from " http://2013.igem.org/Team:DTU-Denmark/Notebook/12_July_2013 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
