<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_DTU-Denmark_pBAD_SPL"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:DTU-Denmark/pBAD_SPL">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:DTU-Denmark/pBAD_SPL&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark/pBAD_SPL&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark/pBAD_SPL&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:DTU-Denmark/pBAD_SPL" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:DTU-Denmark/pBAD SPL</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="navigation"><DIV class="navigation"><DIV class="centering2"><UL class="topnav" id="topnav"><LI class="home navitem1"><A href="/Team:DTU-Denmark">Home</A></LI><LI class="team navitem2"><A href="/Team:DTU-Denmark/Team">Team</A></LI><LI class="parts navitem3"><A href="/Team:DTU-Denmark/Project">Project</A></LI><LI><A href="/Team:DTU-Denmark/Project">Description</A></LI><LI><A href="/Team:DTU-Denmark/Parts">Parts</A></LI><LI><A href="/Team:DTU-Denmark/Timeline">Timeline</A></LI><LI><A href="/Team:DTU-Denmark/Notebook">Notebook</A></LI><LI><A href="/Team:DTU-Denmark/pBAD_SPL">pBAD SPL</A></LI><LI><A href="/Team:DTU-Denmark/Experiments">Experiments</A></LI><LI><A href="/Team:DTU-Denmark/Modeling">Modeling</A></LI><LI><A href="/Team:DTU-Denmark/Methods">Protocols</A></LI><LI><A href="/Team:DTU-Denmark/HelloWorld">Hello World Pilot Project</A></LI></UL><LI class="safety navitem4"><A href="/Team:DTU-Denmark/Safety">Safety</A></LI><LI class="outreach navitem7"><A href="/Team:DTU-Denmark/Human_Practices">Outreach</A></LI><LI><A href="/Team:DTU-Denmark/Human_Practices">Overview</A></LI><LI><A href="/Team:DTU-Denmark/High_School_Outreach">High School Outreach</A></LI><LI><A href="/Team:DTU-Denmark/IP_and_Synthetic_Biology">IP and Synthetic Biology</A></LI><LI><A href="/Team:DTU-Denmark/Biobrick_Workshop">BioBrick Workshop</A></LI><LI><A href="/Team:DTU-Denmark/Helping_Other_Teams">Helping Other Teams</A></LI><LI class="sponsors navitem8"><A href="/Team:DTU-Denmark/Attributions">Attributions</A></LI><LI class="igem navitem9"><A href="/">iGEM</A></LI></DIV></DIV></DIV><DIV id="innercontent"><DIV class="centering"><DIV class="whitebox article"><DIV class="overviewPage"><H1>

pBAD SPL

</H1></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#pBAD_synthetic_promoter_library"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">pBAD synthetic promoter library</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Methods"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Experimental_procedure"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Experimental procedure</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Data_analysis"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Data analysis</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Results"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Summary"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Summary</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Details"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Details</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Sequences"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Sequences</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Example_of_use"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Example of use</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="pBAD_synthetic_promoter_library"> pBAD synthetic promoter library </SPAN></H2><P>A synthetic promoter library (SPL) is a library of cells which contain a promoter that has been randomly mutated to give different expression levels. This promoter upstream of a fluorescent protein like GFP/RFP. The term was first coined in 1998 and used in <I>Lactococcus lactis</I> [1]. The method was adapted by <A href="https://2010.igem.org/Team:DTU-Denmark/SPL" class="external text" rel="nofollow">2010 DTU iGEM team</A> to make a SPL for <I>E. coli</I> that enabled the modulation of constitutive gene expression with great precision. They even made a new standard for this method with the use of biobricks (<A href="http://dspace.mit.edu/handle/1721.1/60080" class="external text" rel="nofollow">RFC 63</A>).
</P><P>We used the method to build a non-leaky arabinose inducible promoter as a tool for expressing lethal proteins in <I>E. coli</I>. The reason we did this is that many of the proteins we are working with are membrane bound or integral membrane proteins and will be lethal if expressed in high quantities. 
</P><P>For testing our pathways we needed to grow the transformed cells to a certain concentration, and attempting this with the genes constitutively expressed was often not yielding viable colonies.  We needed a inducible system, but the standard arabinose inducible system was way too leaky and we with this promoter we can into the same problem as before. Therefore we needed to either build or buy (if possible) an inducible system with great tightness that is able to be induced easily. We choose to build such a system.
</P><H2><SPAN class="mw-headline" id="Methods"> Methods </SPAN></H2><H3><SPAN class="mw-headline" id="Experimental_procedure"> Experimental procedure </SPAN></H3><OL><LI> Random promoter sequences were ordered matching the sequence CTGACGNNNNNNNNNNNNNNNNNNTAWWATNNNNA.
</LI><LI> USER cloning to add RFP downstream of promoter.
</LI><LI> Colonies were plated.
</LI><LI> After inspection under UV-light the non RFP containing colonies (via visual inspection) where picked. 
</LI><LI> Plates were induced by spraying them with an 5% w/v aqueous arabinose solution.
</LI><LI> The plates were again inspected under UV-light and this time the most red florescent cells were picked.  
</LI><LI> Colonies were grown in culture tubes and screened in parallel on the BioLector. Wells on BioLector plate were loaded with culture by transferring a toothpick from each overnight culture selected and into the wells of the plate. All wells were run in duplicate.
</LI><LI> All duplicate colonies were run twice -- once with arabinose added at t=0, and again without arabinose.  
</LI><LI> The pBAD system <A href="http://parts.igem.org/Part:BBa_K808000" class="external text" rel="nofollow">BBa_K808000</A> was used as a reference.
</LI></OL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:632px;"><DIV class="thumbcaption">This reverse primer sequence is incorporating randomized promoter sequences into the pBAD construct. The sequence is annotated with -10 and -35 consensus region. Note that I2-bindingsite is overlapping with the -35 region. Blue is the binding part of the primer, red is the USER made sticky ends. N=random, W=50% A and 50% T</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Data_analysis"> Data analysis </SPAN></H3><OL><LI> Data was collected from the Biolector, and analyzed using a series of R scripts written by Chris Workman (unpublished).  
</LI><LI> The maturation and degradation times for mCherry were both assumed to be 40 min.
</LI><LI> The growth rate, mu, was estimated to be 1.28 (from an average of all wells on all plates) since we expect each strain to grow at the same rate.
</LI><LI> A time window representing exponential growth was selected (between 4 and 8 hours).
</LI><LI> The RFP measurement for a constitutively expressed strain was used as a standard measure of growth.  This is plotted on the x-axis in the detailed plots per colony below.  
</LI><LI> Figures were plotted using R.
</LI></OL><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><H3><SPAN class="mw-headline" id="Summary"> Summary </SPAN></H3><P>Promoter activity when induced (with arabinose added) plotted vs basal activity (without arabinose; ie leakiness of the promoter).  The colonies that we selected all show less activity than the the constitutive promoter, and when induced, show higher activity than the constitutive promoter.  The induced activity of the arabinose reference promoter is shown as a band representing two replicates.
</P><H3><SPAN class="mw-headline" id="Details"> Details </SPAN></H3><P>Promoter strengths for two trials of each colony with and without arabinose.  
</P><TABLE class="sortable" style="border:3px solid black; width: 70%" align="center" cellspacing="0"><TBODY><TR><TH style="background:#AAAAAA; color:black">Identifier
</TH><TH style="background:#AAAAAA; color:black">Sequence<SUP>a</SUP></TH><TH style="background:#AAAAAA; color:black">Induced Strength
</TH><TH style="background:#AAAAAA; color:black">Leakiness
</TH><TH colspan="3" style="background:#AAAAAA; color:black">Inducibility<SUP>b</SUP></TH></TR><TR><TD align="center">Col. 02
</TD><TD><FONT face="Courier" size="+1">CTGACGACGCCCCTCTCCGCCCCTTAAAATGTCCA</FONT></TD><TD align="center">14.2
</TD><TD align="center">0.203
</TD><TD align="center">69.9
</TD></TR><TR><TD align="center">Col. 03
</TD><TD><FONT face="Courier" size="+1">CTGACGTAATCATACCGCCGAAAGTATTATCATTA</FONT></TD><TD align="center">16.6
</TD><TD align="center">0.426
</TD><TD align="center">39
</TD></TR><TR><TD align="center">Col. 04
</TD><TD><FONT face="Courier" size="+1">CTGACGCGACAACATTGCGTCCTATAAAATGCCGA</FONT></TD><TD align="center">14.2
</TD><TD align="center">0.345
</TD><TD align="center">41
</TD></TR><TR><TD align="center">Col. 05
</TD><TD><FONT face="Courier" size="+1">CTGACGGCCCGCCCCCGATGCGCATAAAATACCAA</FONT></TD><TD align="center">17.7
</TD><TD align="center">0.424
</TD><TD align="center">41.6
</TD></TR><TR><TD align="center">Col. 12
</TD><TD><FONT face="Courier" size="+1">CTGACGCGTATCGCGAGCGGGCGTTATTATACGCA</FONT></TD><TD align="center">13.6
</TD><TD align="center">0.414
</TD><TD align="center">32.9
</TD></TR><TR><TD align="center">Col. 10
</TD><TD><FONT face="Courier" size="+1">CTGACGCCCCGAATCAGTAGTATTTATTATCTAGA</FONT></TD><TD align="center">18
</TD><TD align="center">0.462
</TD><TD align="center">38.8
</TD></TR><TR><TD align="center">Col. 09
</TD><TD><FONT face="Courier" size="+1">CTGACGCCACCCCCCCCCCCGGCGTATAATTCCCA</FONT></TD><TD align="center">16.1
</TD><TD align="center">0.530
</TD><TD align="center">30.4
</TD></TR><TR><TD align="center">Col. 08
</TD><TD><FONT face="Courier" size="+1">CTGACGCCCTACCGCTCGCCCCCCTATTATACCCA</FONT></TD><TD align="center">13.4
</TD><TD align="center">0.211
</TD><TD align="center">63.4
</TD></TR><TR><TD align="center">Col. 13
</TD><TD><FONT face="Courier" size="+1">CTGACGCCCCCCCCGTCCACCCCCTAATATCCCGA</FONT></TD><TD align="center">10.9
</TD><TD align="center">0.096
</TD><TD align="center">113.6
</TD></TR><TR><TD align="center">Col. 15
</TD><TD><FONT face="Courier" size="+1">CTGACGTGCGCCTGCCGTCCAAAGTAATATCCTTA</FONT></TD><TD align="center">15.5
</TD><TD align="center">0.235
</TD><TD align="center">65.8
</TD></TR><TR><TD align="center">Col. 18
</TD><TD><FONT face="Courier" size="+1">CTGACGCCCCGAATCAGTAGTATTTATTATCTAGA</FONT></TD><TD align="center">19.8
</TD><TD align="center">0.524
</TD><TD align="center">37.8
</TD></TR><TR><TD align="center">Col. 19
</TD><TD><FONT face="Courier" size="+1">CTGACGCTCGCACCGGCGACCCGATAATATCCATA</FONT></TD><TD align="center">18.0
</TD><TD align="center">0.315
</TD><TD align="center">57
</TD></TR><TR><TD align="center">Col. 31
</TD><TD><FONT face="Courier" size="+1">CTGACGCCCCCCCTCCCCCGACCTAAAATATCCAG</FONT></TD><TD align="center">3.8
</TD><TD align="center">0.161
</TD><TD align="center">23.6
</TD></TR><TR><TD align="center">Col. 29
</TD><TD><FONT face="Courier" size="+1">CTGACGCCGCTGCACCCGTCCCCCTATTATAGCAA</FONT></TD><TD align="center">14.8
</TD><TD align="center">0.301
</TD><TD align="center">49.2
</TD></TR><TR><TD align="center">Col. 26
</TD><TD><FONT face="Courier" size="+1">CTGACGCCGATCCTATCTCCCTATTAAAATTTTTA</FONT></TD><TD align="center">13.8
</TD><TD align="center">0.292
</TD><TD align="center">47.2
</TD></TR><TR><TD align="center">Col. 22
</TD><TD><FONT face="Courier" size="+1">CTGACGCCCCGAATCAGTAGTATTTATTATCTAGA</FONT></TD><TD align="center">16.8
</TD><TD align="center">0.466
</TD><TD align="center">36.1
</TD></TR><TR><TD align="center">Col. 33
</TD><TD><FONT face="Courier" size="+1">CTGACGCCAGCTTCCTACTTCAAATATAATCCGTA</FONT></TD><TD align="center">14.4
</TD><TD align="center">1.01
</TD><TD align="center">14.3
</TD></TR><TR><TD align="center">Col. 34
</TD><TD><FONT face="Courier" size="+1">CTGACGTCCCTCTTCCGCGCCCCCTAAAATACCCA</FONT></TD><TD align="center">12.4
</TD><TD align="center">0.187
</TD><TD align="center">65.9
</TD></TR><TR><TD align="center">Col. 35
</TD><TD><FONT face="Courier" size="+1">CTGACGCCCAACCCGACACAGCGATATAATAATGA</FONT></TD><TD align="center">18.4
</TD><TD align="center">0.89
</TD><TD align="center">20.7
</TD></TR></TBODY></TABLE><P><SUP>a</SUP> The sequence of each pBAD promoter characterized. To use this simply produce a primer with the promoter sequence you wish in a tail. Replace the old promoter in your system e.g. <A href="http://parts.igem.org/Part:BBa_K808000" class="external text" rel="nofollow">BBa_K808000</A> with the new and see the change.
</P><P><SUP>b</SUP> Inducibility = Induced Strength divided by leakiness.
</P><H2><SPAN class="mw-headline" id="Sequences"> Sequences </SPAN></H2><P>We sequenced and aligned the promoters that are shown above.  The sequences show conservation in the -10 and -35 regions by design.  Within the -10 region, we allowed for two random weak (A or T) bases.  We do not see a strong preference for A over T or vice versa.  Within the regions where we allowed any base, we see a preference for C over the other bases.  This could be due to bias during synthesis.  
</P><H2><SPAN class="mw-headline" id="Example_of_use"> Example of use </SPAN></H2><P>The tight inducible pBAD promoter was used in our <A href="https://2013.igem.org/Team:DTU-Denmark/HelloWorld" class="external text" rel="nofollow">&quot;Hello World project&quot;</A> to regulate the expression of GFP SF, which was tagged with a signal peptide to direct it into the periplasm. Production and folding of GFP SF is faster than the transport system of <I>E. coli</I>, which leads to undesired accumulation of GFP SF in the cytoplasm. Only when using a promoter with low leakiness it is possible to translocate a significant fraction of GFP SF after its production has been switched off. Thereby we get a clear signal from the periplasm with low interference from the cytoplasm.
</P><P>To incorporate the new promoter sequence, we ordered primers containing the sequence as a tail and used USER cloning to insert it into our construct. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Zoomed in picture of &quot;Hello World&quot; transformants, showing a clear separation of green and red fluorescence. GFP is primarily located in the periplasm while RFP is located in the cytoplasm. Fluorescence intensity measurements are taken along the cross-section indicated by the white line. Picture taken with a fluorescence microscope and subjected to background subtraction</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:292px;"><DIV class="thumbcaption">Line profile through the cell. The profile shows both the green and red channel. It can be seen that the intensity of the red fluorescence is restricted to the cytoplasm while green fluorescence has it's peaks on the edges. A weak green signal is measured for the cytoplasmic region caused by the periplasm enveloping the cytoplasm.</DIV></DIV></DIV><P>See also <A href="https://2013.igem.org/Team:DTU-Denmark/HelloWorld" class="external text" rel="nofollow">&quot;Hello World project&quot;</A>.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>[1] Jensen, Peter Ruhdal, and Karin Hammer. &quot;The sequence of spacers between the consensus sequences modulates the strength of prokaryotic promoters.&quot; Applied and environmental microbiology 64.1 (1998): 82-87.
</P></DIV></DIV></DIV><DIV id="footer-panel"><DIV class="centering"><DIV class="footerBox"><TABLE border="0"><TBODY><TR><TD width="300px"><H4><A href="https://2013.igem.org/Team:DTU-Denmark/Attributions#Sponsors">Sponsors</A></H4></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:DTU-Denmark/pBAD_SPL">http://2013.igem.org/Team:DTU-Denmark/pBAD_SPL</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:DTU-Denmark/pBAD_SPL" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:DTU-Denmark/pBAD_SPL" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:DTU-Denmark/pBAD_SPL&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:DTU-Denmark/pBAD_SPL&amp;oldid=305279" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>