<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Penn_project_protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Penn/project/protocols/">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Penn/project/protocols/&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/project/protocols/&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/project/protocols/&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Penn/project/protocols/" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Penn/project/protocols/</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>  Protocols | Penn iGEM 2011</TITLE></P><DIV id="top"><DIV id="top-wrapper"><DIV id="top-content"><DIV id="bottom-light"><DIV class="container"><DIV id="menu" class="clearfix"><DIV id="menu-right"><DIV id="menu-content"><UL id="top-menu" class="nav"><LI id="menu-item-62" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-home menu-item-62"><A href="http://benjaminshyong.com/igem2011/">Home</A></LI><LI id="menu-item-217" class="menu-item menu-item-type-custom menu-item-object-custom current-menu-ancestor current-menu-parent menu-item-217"><A href="http://benjaminshyong.com/igem2011/project/overview/">Project</A></LI><LI id="menu-item-73" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-73"><A href="http://benjaminshyong.com/igem2011/project/overview/">Overview</A></LI><LI id="menu-item-72" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-72"><A href="http://benjaminshyong.com/igem2011/project/our-system/">Our System</A></LI><LI id="menu-item-74" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-23 current_page_item menu-item-74"><A href="http://benjaminshyong.com/igem2011/project/protocols/">Protocols</A></LI><LI id="menu-item-70" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-70"><A href="http://benjaminshyong.com/igem2011/project/future-directions/">Future Directions</A></LI></UL><LI id="menu-item-75" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-75"><A href="http://benjaminshyong.com/igem2011/results/">Results</A></LI><LI id="menu-item-270" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-270"><A href="http://benjaminshyong.com/igem2011/results/luminometer/">Luminometer Experiments</A></LI><LI id="menu-item-269" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-269"><A href="http://benjaminshyong.com/igem2011/results/ivis-imaging/">iVIS Imaging</A></LI><LI id="menu-item-268" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-268"><A href="http://benjaminshyong.com/igem2011/results/fura-2-calcium-imaging/">Fura-2 Calcium Imaging</A></LI><LI id="menu-item-67" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-67"><A href="http://benjaminshyong.com/igem2011/biobrick/">Biobrick</A></LI><LI id="menu-item-68" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-68"><A href="http://benjaminshyong.com/igem2011/notebook/">Notebook</A></LI><LI id="menu-item-76" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-76"><A href="http://benjaminshyong.com/igem2011/safety-and-society/">Human Practices</A></LI><LI id="menu-item-314" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-314"><A href="http://benjaminshyong.com/igem2011/safety-and-society/media-and-synthetic-biology/">Media and Synthetic Biology</A></LI><LI id="menu-item-313" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-313"><A href="http://benjaminshyong.com/igem2011/safety-and-society/survey-results/">Survey Results</A></LI><LI id="menu-item-78" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-78"><A href="http://benjaminshyong.com/igem2011/safety/">Safety</A></LI><LI id="menu-item-63" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-63"><A href="http://benjaminshyong.com/igem2011/about-us/">About Us</A></LI><LI id="menu-item-66" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-66"><A href="http://benjaminshyong.com/igem2011/about-us/the-team/">The Team</A></LI><LI id="menu-item-64" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-64"><A href="http://benjaminshyong.com/igem2011/about-us/advisors/">Advisors</A></LI><LI id="menu-item-237" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-237"><A href="http://benjaminshyong.com/igem2011/about-us/sponsors/">Sponsors</A></LI><LI id="menu-item-87" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-87"><A href="https://2011.igem.org/Main_Page">iGEM</A></LI></DIV></DIV></DIV><DIV id="category-name"><H1 class="category-title">Protocols</H1></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="content"><DIV id="content-shadow"><DIV class="container"><DIV id="breadcrumbs"><A href="http://benjaminshyong.com/igem2011">Home</A><SPAN class="raquo">»</SPAN>
				
									  Protocols					</DIV><DIV id="content-area" class="clearfix"><DIV id="left-area"><DIV class="entry post clearfix"><H2>Passing Cells Protocol in Sarkar Lab</H2><H4>Items Needed:</H4><P>• Media, PBS, Trypsin
• Aspirating Pipettes
• Pipet Aid
• Test Tube Racks (2)
• 10 mL pipettes (5)
• 5 mL pipette
• 2 mL pipette
• T75 Flask
• 15 mL conical tube
• Eppendorf tube
• Tweezers (used to take Eppendorf tubes out of container)</P><H4>Procedure:</H4><P>1. Warm media, PBS, trypsin in water bath to 37 C (Takes approximately 10-15 minutes)
2. While media, PBS, and trypsin is heating up, clean everything with 70% EtOH
3. Aspirate media from flask
4. Wash cells with 10 mL warm PBS. Pipette onto the bottom of the flask, then turn flask to the side and slowly move back and forth.
5. Aspirate PBS
6. Add 1.5 mL trypsin to the bottom of the flask. Move the flask back and forth, and then place it into the incubator for approximately 1 minute.
7. Remove flask from the incubator. Add 8.5 mL of media to the flask. If you are just passing cells to maintain them, the media should contain FBS and PenStrep. If you are planning to transfect, the media should be free of antibiotics.
8. Transfer 10 mL of the trypsinized cells/media from the flask to a 15 mL conical tube.
9. Spin the conical tube at 250g for 4 minutes. Make sure to balance the centrifuge with a conical tube filled with H2O (10 mL).
10. Aspirate the supernatant.
11. Resuspend the cell pellet with 10 mL media (with PenStrep if just passing, without PenStrep if planning to transfect).
12. Take approximately 1 mL of cells and put into an Eppendorf tube. Take this tube out of the TC hood and bring it to the bench with the microscope and hemocytometer.
13. Take 50 uL of cells and add to another Eppendorf tube. Take 50 uL Trypan blue and mix this with the 50 uL of cells.
14. Take 10 uL of this new mixture and pipette onto the hemocytomter.
15. Place the hemocytometer slide onto the microscope and count cells. Average the number of cells you count per each square, then double that number since the mixture was diluted 1:2 with Trypan blue. This is the number you use to calculate cell density.
16. The volume of 1 square of the hemocytometer is 10-4 mL. Take the number of cells determined above/10-4 mL. This is the cell density.
17. Calculate how much volume you need to add to the new flask/6-well-plate/etc. For a T75 flask, we’ve been plating 1×104 cells/mL in a total volume of 20 mL. Other volumes/cell densities can be found on the ATCC website. We’ve been plating 5.7×105 cells/mL per well in a 6 well plate.</P><H2>Transfection with TransPass D2</H2><H4>Procedure</H4><P>1. Plate cells (in complete growth medium containing 5-10% serum and no antibiotics/antimycotics) at an appropriate density so that they will reach 70-80% confluence at the time of transfection. *Remember that any media you add must be at 37 degrees, because HEKs will come off/die if media is added at room temperature
2. Mix x µg plasmid DNA in x µl serum-free medium. *See table below for all references to x
3. Gently mix TransPass D2 Transfection Reagent prior to pipetting. Do not vortex. Add x µl to the DNA/medium mix from step 2. Mix gently by flicking the tube.
4. Incubate at room temperature for 20-30 minutes to form the transfection complex.
5. Add the transfection complex mixture to cells. Rock the plate gently in order to evenly disperse the complex mixture. *Just tilt it back and forth a few times, and left and right
6. Return the plate to the incubator and incubate 24-72 hours before assaying.
7. Replace medium as needed to maintain healthy cells. *While the table below says 1 mL of media for every well in a 6 well plate, it’s actually better to grow the cells in 2 mL media</P><H4>Notes:</H4><P>• Volume of plating medium = how much antibiotic-free media each well should have, prior to adding transfection complex
• DNA in serum-free mixture = add this amount of DNA to this amount of serum-free medium. After incubation of 30 min, then add this to the cells+media.
• For 6 well plates, we usually use 6 uL D2 TransPass.</P><P> </P><H2>Restriction Digest</H2><H4>Quick protocol:</H4><P>Combine the following in a microfuge tube in order:
• 1µl 10x Buffer
• 6.5µl H2O
• 2ul of 0.5 ug DNA
• 0.5µl Enzyme</P><P>Incubate for 1 hour at 37°C in a waterbath. Meanwhile prepare the agarose gel.</P><P>1x BSA should likely be used in all restriction digests. Also 0.5uL of enzyme can be overkill. You want no more than 5 units per microgram of DNA. If your enzyme stock is 20U/uL, then 0.25uL is more than sufficient. This varies with enzymes, so always check the stock concentration.</P><H4>Special conditions:</H4><P>A few enzymes require special conditions. It will say in the catalogue. Some require BSA (bovine serum albumin) added into the mixture. This is usually provided free with the enzyme at 100x concentration. Some require weak detergents (eg. triton-X-100) to reduce surface tension. Some require to be incubated at temperatures other than 37°C (e.g. 50°C). If you have not heard of the enzyme before then it is worth checking for these things in the catalogue. Some manufacturers require all kinds of weird stuff when the enzyme works perfectly fine without it. If you cannot obtain some arcane ingredient then don’t panic. It will probably work perfectly well without it.</P><H2>Characterizing Gaussia using a Luminometer</H2><H4>Step 1: Make CZ Stock</H4><P>Make coelenterazine (C26H21N3O3) stock (MW = 423.48 g/mol)
Online Information – “How to Use Coelenterazine” : http://www.nanolight.com/nanofuel.htm</P><P>Choice 1: To 10ml of 100% Methanol or Ethanol add 200ul of 3N or 6N HCl mix well. Take 0.25-1ml of the acidified alcohol and weigh out up to 10mgs of CTZ/ml of alcohol. Usually 1mg/ml is used, and it should go into suspension quite rapidly. 10mgs/ml is about the most concentrated and a slight precipitate may form at this concentration. Store at –20C or better –70C. While not as good as freshly mixed CTZ, this should be good for 1-3 weeks. Always protect any CTZ from light.</P><P>Choice 2: Solubilize coelenterazine as 25mM stock solutions. Dissolve this in acidified methanol (1 microliter of concentrated HCl and 10 mL methanol). This should be stored in -80, replacing air in the tube with dry nitrogen (please ask someone how to do this).</P><P>Choice 3: Stock at 250uM (250x): dissolve 1mg coelenterazine in 9.44ml 100% ethanol.Prepare 400ul aliquots and 40ul (most useful) and 20ul aliquots in 1.5ml tubes. Dry down with speed vac and gas with N2 (nitrogen), seal, and store at -80C in a black box. – Reconstitute coelenterazine by adding 40ul 100% ethanol to a ’40ul aliquot’. This is a 250x stock. Our final concentration of coelenterazine is 1uM, but others use higher concentrations (3-10uM). I find it useful to add another 960ul water to make 1ml of a 10x stock. Keep tube wrapped in foil and on ice throughout the experiment. If you have leftover coelenterazine at the end of the day you can store it at –70oC but its activity will dwindle.</P><P>Choice 4: Dan’s Protocol (this is what we actually used)
Make CZ stock at 1mg/mL- which is 100 to 125x in strength (so dilute 1:100 when using).
Dissolve in acidified ethanol (1mL ethanol + 20 microliters of 3N HCl, where N = normal = molar for HCl).
*Extra ethanol and 3N HCl is in the iGEM box above the sink.</P><H4>Step 2: Measure Luminescence of Samples Using the Promega Glomax 20/20 Luminometer</H4><P>Technical Manual: http://www.promega.com/resources/protocols/technical-manuals/0/glomax-2020-luminometer-protocol/</P><DL><DT>1. The luminometer is located in the Chen Lab. Only use with supervision. Power switch is located in the back of the machine.</DT><DT>2. Flush with Milli-Q water</DT><DD>a. Fill two 20 mL scintillation vials with Milli-Q water and place the vials in the left and right holders</DD><DD>b. Place an empty, waste vial in the center</DD><DD>c. Select INJ FUNC and then FLUSH BOTH INJECTORS.</DD><DT>3. Priming with substrates</DT><DD>a. Remove waste vial and replace it with the metal sample holder</DD><DD>b. Fill a new 20 mL scintilaltion vial with 1 mL LAR II substrate and place in the left side. Fill an additional vial with 1 mL of Stop &amp; Glow substrate and place in the right side. Stop &amp; Glow comes as a 50X stock solution which can be frozen and thawed for use. Make a working solution by adding 20 uL of 50X to 1 mL PBS. Do not freeze the working solution.</DD><DD>c. Tape the tubing to the scintillation vial so that the tubing is touching the bottom of the vial.</DD><DD>d. Place a 1.5 mL eppendorf tube in the sample holder to collect buffer.</DD><DD>e. Select PRIME INJECTOR 1. Once priming is finished, remove the center 1.5 mL tube and label as “old” LAR II.</DD><DD>f. Repeat steps (d) and (e) for Stop &amp; Glow, selecting PRIME INJECTOR 2 and labeling as “old” Stop &amp; Glow.</DD><DD>g. There will be bubbles in the tubing, so more substrate needs to be used for priming. The substrate can now be collected and re-used because there is no further mixing with water once the initial substrate flowed through the tube. Repeat the priming steps with substrates LAR II and Stop &amp; Glow to remove bubbles.</DD><DD>h. NOTE: Since we will be using Coelenterazine substrate, and should have a large amount of CZ solution, priming can be done with a 3 mL aliquot and completed in 1 round of priming.</DD><DT>4. Measuring samples</DT><DD>a. Make sure there is a sufficient amount of substrates for the number of samples to be tested.</DD><DD>b. Select PROTOCOL → RUN USER PROTOCOL → DAN II . To see parameters, select SHOW PARAMETERS. We will make our own iGEM program with our set parameters in the future. Click OK to exit.</DD><DD>c. Fill a 1.5 mL eppendorf tube with 5 uL sample and place the tube in the center. The first sample tested should be the one predicted to have the highest luminescence, so a dilution can be made if desired. Select READ SAMPLE. The reading will not be higher than 8 or 9 digits, and is always a positive integer. If the reading is less than 3 or 4 digits, the sample has only background luminescence.</DD><DD>d. Record RLUs1, RLUs2, and ratio if desired. Measure remaining samples.</DD><DD>e. There is a kinetics option for single-injector use that enables you to measure the sample’s luminescence at different time periods during an integration. Instead of adding an integration time parameter, you input the frequency of measurements you want as well as the total number of measurements. Connect a computer to the luminometer using an RS-232 to USB Data Cable. Download the detection software from the following link. The data will be inputed into the Glomax Spreadsheet in Microsoft Excel.<P>http://www.promega.com/resources/tools/detection-instruments-software/promega-branded-instruments/glomax-20-20-luminometer/</P></DD></DL><H4>Step 3: Cleanup</H4><P>1. Prime with Milli-Q water by adding ~ 10 mL Milli-Q water to two 20 mL scintillation vials. Take out the metal sample holder and replace with a 20 mL waste scintillation vial. Select PRIME BOTH INJECTORS. This removes the salts that are in the substrates from the tubing.
2. Prime with 70% Ethanol by repeating step (a) with 70% ethanol. This removes the Coelenterazine from the tubing (not soluble in H2O).
3. Flush with Milli-Q water by selecting FLUSH BOTH INJECTORS. This removes the Ethanol from the tubing. The flushing function flows a lot more liquid through the tubing than priming does.
4. NOTE: We have 2 scintillation vials labeled and filled with Milli-Q and 2 scintillation vials labeled and filled with 70% Ethanol in the BE undergrad lab for future use</P><H2>Characterizing pRRL-CMV-PCZ-MODC</H2><H4>Experimental Condition:</H4><P>1. HEK293T Cells expressing pRRL-CMV-PCZ-MODC, Wash many times in PBS, Lyse, save lysate
2. Sample in luminometer: <B>A</B> = Gaussia supernatant
3. Injection: <B>B</B> = lysate from cells</P><TABLE cellpadding="4"><TBODY><TR><TD><B>Controls</B></TD><TD><B>Purpose</B></TD></TR><TR><TD><B>B</B> = PBS instead of pCZ lysate</TD><TD>shows Gaussia is reliable</TD></TR><TR><TD><B>B</B> = CZ stock solution instead of pCZ lysate</TD><TD>shows CZ is required for Gaussia luminescence</TD></TR><TR><TD><B>B</B> = lysate from HEK293T wild-type cells instead of pCZ lysate (Negative control – should not bioluminesce)</TD><TD>shows pCZ is required for Gaussia luminescence</TD></TR><TR><TD>Gaussia supernatant with dual luciferase kit (Dan’s program)</TD><TD>shows the Gaussia is reliable</TD></TR></TBODY></TABLE><H2>Thawing Cells</H2><P>
*volumes in this protocol are for a bullet of cells from half a P150 and into four P100s for a 1:10 dilution</P><P>1. Take the cells out from the -80 freezer
2. Shake cells in a water bath at 37 degrees C manually (just hold it with your hand. This will just thaw the cells).  Make sure to refill water bath if level is too low.
3. Spray this with tons of ethanol, dry with kimwipe, and put this in TC hood
4. Resuspend in 9mL warm media in a 15mL conical tube. This dilutes the DMSO (antifreeze substance) in the cells, which can be toxic to them once thawed. We want to get rid of DMSO immediately, so that’s why we dilute them with media. This entire process needs to be done quickly because we want to get rid of the DMSO.
5. Spin at _rcf or 1300RPM according to the large centrifuge in BE undergrad labs, for five min (this is relatively slow speed which is good for the cells)
6. Aspirate off the media (which has a bit of DMSO in it)
7. Resuspend in 10 mL media and place 0.89 mL into each of four TC plates. Then add enough media to make the plates filled with 10 mL media total (Arthur just added 10 mL to each P100 plate).
8. Add more warm media to the TC plate? and label the plate (cell type, passage number, name, date).
9. Aspirate off the unused cells- these are garbage
10. Incubate P100 plates in incubator at 37C.  Make sure to change the paper towel in the incubator every day.
11. Freeze 2 plates into cell bullets to be stored again
12. ***this entire process needs to be done with sterile technique: see below protocol for more details on that</P><H4>Plate Specifications</H4><P>P150 Surface Area: 176.65cm^2
P150 Half-Plate Surface Area: 88cm^2
P100 Surface Area: 78.5cm^2
1:10 Dilution</P><H2><U>Splitting (Passaging) Cells</U></H2><P>
1. Turn TC hood on to start flow and spray with 70% ethanol 30 min before culturing (see sterile technique protocol).
2. You want to split cells when you see there is 60-70% confluency (under the microscope, 60-70% of the plate is filled with cells)
3. Clean hands with 70% EtOH and rub them.
4. Warm media, PBS, trypsin in a water bath at 37 degrees C for 15 minutes. Be sure not to exceed 15 min.
5. Warm trypsin to room temperature (check on it periodically to not leave it out too long, because trypsin can degrade once warm).
6. Aliquot the volume of media, PBS, trypsin into small test tubes depending on how much you need. Put the rest in the refrigerators you got them from
7. Spray entire cell culture hood surface with 70% EtOH and wipe the cell culture hood
8. Wipe from back of hood, working your way up to the front
9. Wipe down pipettor
10. Wipe down 1000uL micropipettor
11. Spray PBS bottle with 70% EtOH
12. Spray CO2 Independent media with 70% EtOH
13. Spray Trypsin Aliquot with 70% EtOH
14. Basically wipe down everything you put into the hood, except plates with live cells in them and things that are already sterile, like individually wrapped sterile pippettes and sterile 6 well plates. Containers of trypsin, media, etc should definitely be wiped down and make sure you pay special attention to the edge of the cap: spray this down excessively!
15. Place unused cell culture plate (6-well) into hood
16. Place cell culture plate containing cells into hood
17. Add 2 mL of CO2 independent media to an unused plate. If you want to have just one well of cells, just add 2 mL to one well. If you want to grow cells in 2 wells, add 2 mL to 2 wells, etc.
18. Aspirate your old cell culture media from cell culture plate  (tip it to one side).
19. When adding any liquid to 293T plates with adherent cells, add it gently and slowly to the plate wall when the plate is tilted (you don’t have to be super gentle when the cells are already trypsinized)
20. Add 15 mL PBS for a 100mm dish. You can scale this with surface area for a smaller well. This is to wash off any extra proteins that are on the plate, because these can deactivate trypsin.
21. Aspirate off the PBS
22. Add 1.5 mL 0.05% trypsin for P100 (remember to scale this down with surface area).
23. Put this in the incubator for 1 min and check under microscope to make sure most of the cells are floating. If not, stick it back in incubator and check every 30s or so. If you let trypsin in for too long, it can degrade the cell.
24. Immediately deactivate trypsin with media. Add 8.5 mL for a P100 plate.
25. Put this in a conical tube and spin at 250 G’s for 4 min.
26. Aspirate the supernatant
27. Resuspend the cell pellet in 1 mL media with a 1000-microliter pipette.
28. Depending on how dilute you want your new culture, aliquot the resuspended pellet into the well (s). Ex: let’s say you want a 1:5 dilution in one plate. You’d take 200 microliters of resuspension and add it to the 2 mL already present in your one well.
29. Aspirate off the rest of your resuspended cells that you don’t want to use.
30. Label the new plate w/ date, name, dilution, cell type including what they’re transduced with (live CMV-Gaussia), passage number (abbreviated P#)
31. Shake plate on surface left-right and then front-back to evenly spread out the cells
32. Observe seeded cells and verify they are evenly seeded
33. Put seeded cell plate back in incubator
34. Throw old plate away in biohazard
35. Remove all bottles, place everything but the PBS back into the 4C refigerator
36. Spray entire cell culture hood surface
37. Wipe from back of hood, working your way up to the front
38. Make sure you change media every 2-3 days because the cells use up the nutrients, glucose, etc.</P><H2><U>Splitting Cells (Chenlab Protocol for C2C12 and HEK293) </U></H2><H4>C2C12 in a P100 Plate (100 mm):</H4><P>1. Warm media to 37 ºC (10-15 min in water bath or 45 min deep in bead bath)
2. Warm trypsin to room temperature
3. Observe cells and estimate confluence
4. Aspirate media
5. Add 15 mL PBS for a P100 Plate
6. Aspirate PBS
7. Add 1.5 mL 0.25% trypsin
8. Put plate back in incubator for 7-10 min
9. Observe cells and verify they have almost all detached
10. Neutralize trypsin with 8.5 mL media
11. Spin at 1300 RPM (record the ###xg value) for 4 min
12. Aspirate supernatent
13. Gently resuspend cell pellet in 1 mL media with P1000
14. Dilute resuspended cells with 9 mL additional media
15. Label a new P100 plate: Date Initials C2C12 PXXX CMV-Gaussia
16. Add 14 mL fresh media to new plate
17. Seed 1 mL cell suspension into new plate
18. Shake plate on surface left-right and then front-back to evenly spread out the cells
19. Observe seeded cells and verify they are evenly seeded
20. Put seeded cell plate back in incubator</P><P>Next time we will:
Count remaining cell suspension and determine seeding density (#cells/cm^2)</P><H4>Important Differences for 293T cells:</H4><P>• 293T — temperature sensitive — minimize time at room temperature and be sure to use warm media or cells will come off the plate; room temperature PBS is okay just add and aspirate without wasting time
• Dilute 0.25% trypsin to 0.05% trypsin with PBS and use 0.05% trypsin for trypsinization step
• When adding any liquid to 293T plates with adherent cells, add it gently and slowly to the plate wall when the plate is tilted (you don’t have to be super gentle when the cells are already trypsinized)</P><H2><U>General Notes on Growing Cells, What Media is, and What Materials are needed and Why</U></H2><H4>When growing cells:</H4><P>1. Usually you need carbon dioxide tanks that feed into the incubators, because CO2 turns into carbonic acid in liquid, and this makes the media more acidic. Since we don’t have these incubators, we need to use CO2 independent media instead, with our incubators that we have in the BE undergrad lab.
2. We also need to make sure the CO2 independent media is clear, because most media has phenol red in it. This absorbs fairly high (50% of peak) at 470nm, we need clear, CO2 Independent Media</P><P>http://www.google.com/url?sa=t&amp;source=web&amp;cd=1&amp;sqi=2&amp;ved=0CBYQFjAA&amp;url=http%3A%2F%2Fwww.plantgenomeoutreach.eeob.iastate.edu%2Fteachmod%2F2_measuredilute.doc&amp;rct=j&amp;q=absorbance%20of%20phenol%20red&amp;ei=AmkCTpiFGJOr0AGqxZ2ZDg&amp;usg=AFQjCNF5vjDhyOr6onMyvGhAVl2sEuXFtQ&amp;cad=rja</P><P>2. You also need to make sure the incubator is set to <B>37 degrees C</B> at all time, and that there is a <B>tray of water</B> at the bottom of the incubator. This is to keep the humidity level in the incubator.
3. Try not tip the plates too much when transporting because we don’t want spills or contamination</P><H4>Media Notes</H4><P>What Are the Ingredients of Media and What Do They Do?</P><P>• Water- A major constituent of the cytoplasm is water, so cells must be in an aqueous environment.
• HEPES or other buffering agents- Cells can only live in a relatively narrow range of pH due to protein folding and enzyme activity being very sensitive to pH. Even if we were to adjust the pH of our media before using, the waste products produced by the cell will quickly cause the pH to change and kill the cells.
• Phenol Red- Added to many media types as a rough indicator of pH. It is not necessary for culture media, and we believe it may interfere with our light emission/detection.
• NaCl, MgCl2, CaCl2, other salts- Pure water will burst cells because it is hypertonic, so we add salts to make the media isotonic with the cell. Furthermore, some of the ions in these salts are necessary as cofactors for enzymes
• <B>The above components of media will keep cells alive for a few minutes and at most an hour or so.</B>
• Glucose- Cells use this for energy. There are two types of media when it comes to glucose. High Glucose Media (4.5 g/L) is used for HEK 293T cells, they are very metabolically active and Low Glucose Media (1 g/L)
• Raw Essential Amino Acids- These are the building blocks of proteins, and only the essential amino acids are added, cells can synthesize the other, nonessentially amino acids, so we don’t have to add those in. Some amino acids are not stable when dissolved in the media, so they are added right before the media is to be used (L-Glutamine)
• Fetal Bovine Serum- A product made directly from fetal cows, which means that this component of media varies from each cow. It contains proteins that provide more nutrition to the cells, but also contains chemical signals to encourage cells to divide (mitogens).
• Penicillin/Streptomycin-Other organisms such as bacteria and fungi will like to grow in your culture and will kill your cells. Therefore, we must add antibiotics. A common formulation is Penicillin and Streptomycin (PenStrep). This does not kill bacteria, just retards their growth so although it reduces the possibility of contamination, bad technique will still lead contaminated cells!
• <B>This is what is in cell culture media, and will keep cells alive for days.</B>
• DMEM is a commonly used media, and is short for (Dulbecco’s Modified Eagle Media)</P><H4>How Do We Sterilize Our Media?</H4><P>    Autoclaving-Uses heat and pressure to kill organisms, but this may destroy some of the components of our media, such as proteins
    Ethanol-Chemically destroys cells, but this will also kill our cells!
    Sterile Filter- Uses a filter with very small pores (.22um) to physically sperate the bacteria from our media</P><H4>What Else Will We Use in Cell Culture?</H4><P>    Trypsin- A purified digestive enzyme that will digest the matrix surrounding the cells and allow us to make a fine suspension of cells for passage. Because it breaks down proteins, leaving cells in trypsin for too long will kill them. Furthermore, trypsin itself is a protein, so if left at room temperature for too long, trypsin will digest itself and the solution will lose effectiveness.
    EDTA- A metal ion chelator that will cause cells to release themselves from the protein in the matrix surrounding the cells, as well as other cells. This makes them easier to dislodge and subjects the cell to less physical trauma.
    Phosphate Buffered Saline (PBS)- Contains salts to create an isotonic environment, and is buffered. This will keep the cells alive for a few minutes, and is primarily used to wash cells.</P><H2><U>Protocol for CO2 Independent Media Production and Sterile Lab Technique</U></H2><H4>Materials:</H4><P>500ml Gibco 18045 CO2 Independent Media
50ml Gibco 10437 Fetal Bovine Serum
5ml Gibco 25030 L-glutamine 200mM 100x
5ml Gibco 15140 Pen Strep
pipette aid
10ml costar stripettes
25 ml costar stripettes</P><H4>Procedure:</H4><P>Notes about the hood: Everything should be sterilized before putting it into the hood by spraying it down with 70% ethanol solution.  This includes the gloves you are wearing.  Be especially mindful to spray around the edges of the caps of solutions because they are most likely to be contaminated.  The vent at the front of the hood sucks in air, so it both produces a barrier between the outside of the hood and the inside of the hood and it sucks in the air inside the hood, drawing it forward.  This means that the area of the hood that is most sterile is farthest back, away from the vent.  The least sterile area is the part of the hood closest to the vent.  Work as far into the hood as possible.  Don’t rest arms on the hood. Because the hood doesn’t have UV light, there is high probability of contamination. When pipetting, never let the tip of the stripette touch anything in the hood.  Also try not to touch the edge of the opening in solution bottles when pipetting solution with the stripette. Do not wave your hand over any open bottles (so close all bottles immediately after taking some liquid from them).</P><P>1. Sterilize the hood by wiping it down from back to front with 70% ethanol solution.
2. Turn on the hood and let it run for 30 minutes with the door open (*if the door is not left open at the sash level, the machine will beep).
3. Put the CO2 Independent Media, the Fetal Bovine Serum, and the L-glutamine in a 37 degree water bath until they thaw (around 15 minutes).
4. Turn on the power supply in the hood using the center button on the top right corner of the hood. Plug in pipette aid machine.
5. Remove enough peeling from the sterile stripette so that you can attach it to the pipette aid without touching the stripette with your gloves inside the hood. Adjust the stripette so that you can see the numbers clearly. Remove the packaging and throw away.
6. Open the Pen Strep and the CO2 Independent Media. Add 5ml of Pen strep to the CO2 Independent Media. When resting caps on the hood surface, place them face down because the risk of splashing the liquid in the caps by turning them upside down is greater than the risk of contamination by putting them in the hood face down.
7. Open the L-glutamine. Aspirate and stir the liquid simultaneously (aspirate by pipetting the liquid up and down) using the stripette. Add 5ml to the CO2 Independent Media.
8. Open the Fetal Bovine Serum (FBS).  Aspirate until liquid is uniform. Add 50 mL to the media solution.
9. Store CO2 Independent Media Solution in 4C refrigerator.  Media cannot be repeatedly freeze/thawed because the proteins will become denatured over time.
10. Aliquot 10 mL FBS in separate 15 mL conical tubes.  Aliquot 5 mL of Pen Strep and L-glutamine each in sterile 15 mL conical tubes.  Store all aliquots in -20C freezer until more CO2 Independent Media is needed.
11. Close up the hood: close all of the valves inside, wipe down the hood, close the door, and shut off the air ventilation shortly after.</P><P>*Note: the machine will beep if the door is closed while the ventilation is running, but we actually want to do it this way to avoid any air from getting in. This minimizes contamination during clean up.</P><H2><U>Transformation of MDS 42recA Chemically Competent Cells</U></H2><H4>Components:</H4><P>1. MDS 42recA Chemically Competent Cells
2. pUC19 Control DNA (10pg/ul)
3. SOC Medium</P><H4>Storage Conditions</H4><P>Store at -80C. Do not store in liquid nitrogen.</P><H4>Background</H4><P>Using synthetic biology methods, the Escherichia coli K-12 genome was reduced by making a series of planned, precise deletions. The multiple-deletion series (MDSTM) strains (1), with genome reduction of up to 15%, were designed by identifying non-essential genes and sequences for elimination, including recombinogenic or mobile DNA and cryptic virulence genes, while preserving robust growth and protein production. Genome reduction also led to unanticipated beneficial properties, including high electroporation efficiency and accurate propagation of recombinant genes and plasmids that are unstable in other strains. Subsequent deletions and introduction of useful alleles produce strains suitable for many molecular biology applications.</P><H4>Before you begin:</H4><P>We recommend preparing glycerol stock cultures of clones and storing at -80°C, or keeping plates at room temperature for up to 5 days.</P><P>Clean Genome strains do not have flagella and tend to aggregate and drop fairly quickly from solution. To obtain OD readings, cells should be mixed just before taking an aliquot for dilution, and dilution samples should be mixed just before taking an OD reading.</P><H4>PRE-TRANSFORMATION STEPS</H4><P>1.  Equilibrate a water bath to 42°C.
2.  Thaw the provided SOC medium and warm it to 37°C to dissolve any visible precipitate.
3.  Warm selective antibiotic plates to room temperature or 37°C.
4.  If you are transforming a plasmid with antibiotic resistance other than ampicillin, you will need at least one LB agar plate containing 100 μg/ml ampicillin (or 50 μg/ml carbenicillin) for plating the pUC19 Control DNA transformation.</P><H4>TRANSFORMATION PROCEDURE</H4><P>1.  Place required number of 17x100mm culture tubes (14 ml), or 1.5 ml Eppendorf tubes, on ice.
2.  Thaw MDSTM 42recA Chemically Competent Cells on ice.
3.  Flick competent cells tube gently 2-3 times to evenly suspend cells. Add 50 μl cells to each pre-chilled tube.
4.  Add controls and DNA samples to culture tubes.</P><H4>NOTES:</H4><P>For no DNA Control, add 1 μl Ultrapure water. For pUC19 Control DNA, add 1 μl DNA For ligation products, add 2-5 μl (2-10 ng DNA) of heat inactivated ligation reaction. Gently flick the cells/DNA mix 2-3 times.<B>Transformation efficiency of ligation products can be increased several fold by diluting the ligation reaction 5-fold with TE or water prior to adding DNA to the cells.
Unused cells may be re-frozen in dry ice/ethanol bath for 5 min before returning to the -80°C freezer. Expect re-freezing cells to decrease their transformation efficiency.</B></P><P>5.  Incubate tubes on ice for 30 min.
6.  Heat shock cells for 30 s in a 42°C water bath, without agitation.
7.  Place tubes on ice for 2 min.
8.  Add 450 μl of pre-warmed SOC medium to each tube.
9.  Incubate at 37°C with shaking at 250–275 rpm for 1 h.
10. For cells transformed with pUC19 Control DNA, spread 100 μl of the outgrowth culture onto pre-warmed LB agar plates containing 100 μg/ml ampicillin or 50 μg/ml carbenicillin.
11. For cells transformed with experimental DNA, spread up to 150 μl of the outgrowth culture onto selective plates.
12. Store the remaining cultures at 4°C for future plating if desired. Expect reduced efficiency after storage.
13. Incubate plates overnight at 37°C.</P><H4>TRANSFORMATION EFFICIENCY USING pUC19 CONTROL DNA</H4><P>1.Count cells on each control plate. Calculate the average colonies/plate for the pUC19 Control DNA plates if more than one replicate was plated (Note: the No DNA Control plate should not have any colonies).
2. Use the formula below to calculate transformation efficiency in colony forming units (cfu) per μg control DNA.</P><P>Average # colonies/ pg DNA plated * 10^6pg/ug = cfu/ug plasmid DNA</P><P><B>TROUBLESHOOTING</B></P><TABLE cellpadding="4"><TBODY><TR><TD>No colonies or low number of colonies</TD><TD>
1. Incorrect drug selection or drug concentration. Verify that LB agar plates contained the appropriate selective antibiotic. Repeat transformation or plate an aliquot of the remaining out growth culture onto new plates.
2. Incorrect incubation temperature. Be sure to incubate at 37°C. Verify the temperature setting. Incubate plates at 37° for the remainder of one day. If no/low number of colonies is still observed, repeat transformation or plate an aliquot of the remaining out growth culture onto new plates.
3. Cells were not handled correctly. Cells should handled very gently. Do not pipet vigorously or vortex. Always gently pipet or gently flick cells to resuspend. Thaw and keep cells on ice until ready to transform. Repeat transformation.</TD></TR><TR><TD>Lawn or confluent cell growth or satellite</TD><TD>
1. Incorrect drug concentration. Verify that LB agar plates contain the appropriate selective antibiotic concentration. Repeat transformation or plate an aliquot of the remaining out growth culture onto new plates.
2. Antibiotic has degraded. Plates are too old, antibiotic stock(s) have degraded, or antibiotic was added when medium was too hot. Prepare fresh antibiotic stock(s) and fresh plates. Repeat transformation or plate an aliquot of the remaining out growth culture onto new plates.         R
3. Incubated at 37°C too long. Plates should incubate for 16-18 h. Amp cells secrete β-lactamase that creates a drug-free zone in the surrounding medium, allowing small ampS colonies to grow. Carbenicillin (an ampicillin analog) appears to be less vulnerable to degradation. Repeat transformation, or plate an aliquot of remaining out growth culture, and incubate plates for 16-18 h.</TD></TR></TBODY></TABLE><H2><U>Gateway Reaction</U></H2><P><B>Storage:</B> store at -80°C in 5ul aliquots</P><P><B>To Convert fmoles to nanograms (ng)</B>
ng = (xfmol)(N)(660fg/fmol)(1ng/10^6fg) where N is the size of the DNA in base pairs, and x is the number of fmoles</P><H4>MultiSite Gateway® and MultiSite Gateway® Pro LR Reaction Mixture</H4><P>For multi-fragment (i.e. 2-, 3-, or 4-fragment recombination) reactions, use an equimolar amount of each entry clone. We recommend 10 fmol of each entry clone and 20 fmol of DEST vector per 10 μl reaction. *Change: to conserve reaction mixture, we are going to use half of the recommended quantities.  We are using 5fmol of each entry clone and 10fmol of DEST vector.</P><P>Note: If you are using MultiSite Gateway®, you can only recombine three entry clones into pDEST R4R3. If you are using MultiSite Gateway® Pro, you can recombine up to four entry clones into any pDEST vector containing attR1 and attR2 sites.</P><P>Add the following components to a 1.5-ml microcentrifuge tube at room temperature and mix. Use this reaction mixture in the Procedure on the next page.</P><P>Entry clones (5 fmoles)
Destination vector (10 fmoles)
1X TE buffer, pH 8.0
*All entry clones (two, three or four, depending on the type of reaction) must be included. The total of all entry clones combined should not exceed 4 μl.</P><H4>LR Reaction Procedure</H4><P>1.  Remove LR ClonaseTM II Plus enzyme mix from freezer and thaw on ice for about 2 minutes. Vortex the enzyme mix briefly twice (2 seconds each).
2.  To each MultiSite or MultiSite Pro LR reaction mixture, add 1 μl of LR ClonaseTM II Plus and mix well by vortexing briefly twice. Microcentrifuge briefly.
3.  Return enzyme mix to freezer immediately after use. The enzyme mix can be stored at -20°C for up to 6 months or at -80°C for long-term storage.
4.  Incubate recombination reaction at 25°C for 16 hours.
5. Add 0.5μl of the Proteinase K solution to each sample to terminate the
reaction. Vortex briefly. Incubate samples at 37°C for 10 minutes.</P><H4>Transformation</H4><P>1. Add 2 μl of the gateway reaction to 50ul of MDSTM 42recA chemically competent cells and incubate on ice for 30 minutes.
2. Heat-shock cells by incubating at 42°C for 30 seconds.
3. Immediately transfer the tubes to ice for 2 minutes.
4. Add 450 μl of S.O.C. medium and incubate at 37°C for 1 hour with shaking at 250-275 RPM.
5. For 2-fragment recombination reactions, dilute 1:10 in S.O.C. medium and plate 50 μl and 100 μl of each transformation on prewarmed LB plates containing 50-100 μg/ml antibiotic of choice, invert and incubate overnight at 37°C. For 3-fragment recombination reactions, plate 50 μl and 100 μl of each transformation as above. For 4-fragment recombination reactions, spin at 6,000 rpm, remove 180 μl of supernatant and gently resuspend the pellet in the residual media. Plate 1/4 to 1/5 of the total transformation.</P><P>*NOTE: If the transformations are successful and we are getting 50-100 or more colonies with this protocol, we can cut the quantity of competent cells in half to 25ul of competent cells and 225ul of SOC.</P><H4>Next Steps</H4><P>Refer to the manual for the destination vector you are using for guidelines and instructions to express your recombinant protein in the appropriate system.
*Change: since we are using p-DEST designed by Dan, we should talk to him if we have questions.</P><P>This protocol was adapted from http://tools.invitrogen.com/content/sfs/manuals/lr%20clonase%20ii%20plus_man.pdf
And edits were made according to Dan Cohen.</P><P>This chart was made up by Arthur to determine how much of each construct, enzyme, etc. we will need per reaction and overall in our project.</P><TABLE wifth="100%" cellpadding="5"><TBODY><TR><TH>Reagent</TH><TH>Total needed for project</TH><TH>Total needed per reaction</TH></TR><TR><TD>Clonase</TD><TD>5ul</TD><TD>1 ul</TD></TR><TR><TD>pENTR 5-CMV</TD><TD>25 fmol (53.9 ng)</TD><TD>11.78ng</TD></TR><TR><TD>pENTR/D-Topo</TD><TD>2ul</TD></TR><TR><TD>pRRL-DEST</TD><TD>50fmol</TD><TD>10fmol (57.186ng)</TD></TR><TR><TD>PCR product</TD><TD>1 ul per TOPO rxn</TD></TR><TR><TD>Salt solution for TOPO</TD><TD>1ul per TOPO</TD></TR><TR><TD>pENTR/D-TOPO-Xgene</TD><TD>25fmol</TD><TD>5fmol</TD></TR></TBODY></TABLE><H2><U>Glycerol Stocks for freezing down bacteria </U></H2><P>
1.       E. coli is stored frozen in glycerol stock
2.       Glycerol stock is made as 50% v/v (volume per volume). This means we want 50% of the volume to be water and 50% to be glycerol.
3.       Let’s say you want to make 50 mL of glycerol stock
4.       Take 25 mL of MilliQ water, which is super sterile deionized water and add to a 50 mL conical tube
5.       Since the density of glycerol is 1.261 g/cm3, and 1 cm3 = 1 mL, and D = M/V, and we want a volume of 25 mL, if you do the calculation you should get a mass of 31.525g. You need to measure our glycerol using its mass, not its volume, because glycerol is super viscous and you can’t measure exactly 25 mL.
6.       Go to Chenlab and put your conical tube + rack + water on a scale. Zero it and add glycerol until you hit 31.525 g.
7.       Now you need to sterilize this: ultrasound it for 10 minutes and then sterile-filter it through a sterile-flip tube thing.
8. To freeze bacteria, add 500 ul of bacteria culture and 500 ul glycerol and stick it in the -80 freezer
9. Whenever you make a glycerol stock, please go to this link: https://igem.chenlab.seas.upenn.edu/items and update the inventory database. Be sure to include the bacterial strain name, antibiotic resistance, vector information, date and your initials on your glycerol stock, as well as on the inventory database.</P><P>The actual protocol for storing glycerol stocks</P><P>http://www.oardc.ohio-state.edu/stockingerlab/Protocols/GlycerolStocks.pdf</P><H2><U>Preparing for Sequencing</U></H2><P><B>Facility:</B> It’s at the Genetics department in the med school, and it’s called their “DNA Sequencing Facility.” Here’s the link:</P><P>http://www.med.upenn.edu/genetics/dnaseq/</P><P>It’s located very near the Cell Center and is located at B1 in the Richards Building.</P><P><B>How to prepare samples:</B>
The PI needs to give you a form and I think submit one online.</P><P>Samples are to be submitted in PCR strip tubes with the PI’s last name labeled on the side of the first two tubes. The tops of the tubes should be labeled 1-8 or A-H (basically, numbers only or letters only). You don’t have to use/write on all 8 tubes that come in a strip.</P><P>In each tube there should be 6 microliters of 80 ng/microliter of DNA (so our DNA in a vector). This can be diluted in DNA suspension buffer/EB Buffer. DNA has to be in a plasmid.</P><P>Also in each tube there should be 3 microliters of 1.1 uM primers. That means you have to order the primers ahead of time. These should be dissolved in 10 mM Tris-HCl at pH 7.5. Each tube can only hold one type of primer maximum! So for DNA longer than 600 bp, you have to use multiple primers and therefore multiple tubes.</P><P>If there is a primer that the DNA Sequencing Facility offers for free, you don’t have to add the primer into your tube. At this website there’s a list of primers they offer:</P><P>http://www.med.upenn.edu/genetics/dnaseq/auto-desc.shtml</P><H2><U>Stab bacteria from Glycerol Stock</U></H2><DL><DT>1. Stab glycerol stock with sterile pipette tip and streak onto LB agar plates with antibiotic</DT><DT>2. Select single colonies (select colony from end of streaks so as to limit genetic variation)</DT><DT>3. Inoculate bacteria in LB broth with antibiotic overnight at 37°C in a shaker.  Do not let this step last more than 1 day.</DT><DT>4. Measure OD of bacteria for optimization purposes (optical density) using Bio-RAD SmartSpec Plus (the spectrophotometer in the BE Undergrad lab)</DT><DD>a. Select Program OD600 for Bacteria. Make sure the screen reads “Ready to read absorbance”</DD><DD>b. Put 50 uL LB broth into a BioRad trUView Cuvette and place the cuvette into the spectrophotometer with the arrow at the top of the cuvette facing yourself.  Select BLANK measurement.  This enters the absorbance of LB broth as the base level.</DD><DD>c. Fill a clean cuvette with 50 uL of your sample after pipetting up and down to mix the culture.  Place the cuvette in the machine and select READ SAMPLE. Record concentrations in cell/mL. Do this for the remaining samples.</DD><DT>5. To freeze down unused bacteria, add 500 uL of 50% glycerol to 500 uL of sample.  To make glycerol stock, see the following protocol.</DT><DT>6. Whenever you make a glycerol stock, please go to this link: https://igem.chenlab.seas.upenn.edu/items and update the inventory. Be sure to include the bacterial strain name, antibiotic resistance, vector information, date and your initials on your glycerol stock as well as on the inventory database. </DT></DL><H2><U>Using the Linearized Plasmid Backbones</U></H2><P><B>Storage:</B>
The linearized plasmid backbones (25ng/ul at 50ul) should be stored at 4C or lower. Prior to ligation the plasmid backbones need to be cut with EcoRI and PstI.</P><P><B>Digest:</B></P><DL><DT>• Enzyme Master Mix for Plasmid Backbone (25ul total, for 6 rxns)</DT><DD>• 5 ul NEB Buffer 2</DD><DD>• 0.5 ul NEB BSA</DD><DD>• 0.5 ul EcoRI-HF</DD><DD>• 0.5 ul PstI</DD><DD>• 0.5 ul DpnI</DD><DD>• 18 ul dH20</DD><DT>• Digest Plasmid Backbone</DT><DD>• Add 4 ul linearized plasmid backbone (25ng/ul for 100ng total)</DD><DD>• Add 4 ul of Enzyme Master Mix</DD><DD>• Digest 37C/30 min, heat kill 80C/20 min</DD></DL><P><B>Ligation:</B></P><P>    • Add 2ul of digested plasmid backbone (25 ng)
    • Add equimolar amount of EcoRI-HF SpeI digested fragment (&lt; 3 ul)
    • Add equimolar amount of XbaI PstI digested fragment (&lt; 3 ul)
    • Add 1 ul NEB T4 DNA ligase buffer. Note: Do not use quick ligase
    • Add 0.5 ul T4 DNA ligase
    • Add water to 10 ul
    • Ligate 16C/30 min, heat kill 80C/20 min
    • Transform with 1-2 ul of product</P><H2><U>Making Linearized Plasmid Backbones</U></H2><P><B>Bulk Production:</B>
The following is the protocol that we used to create the linearized plasmid backbones shipped with the Spring 2011 DNA Distribution. The protocol is in 96 well format, but may be scaled down to suit smaller batches.
Primers:
gccgctgcagtccggcaaaaaa,SB-prep-3P-1
atgaattccagaaatcatccttagcg,SB-prep-2Ea</P><P><B>PCR mix</B>
    • 9.6ml of PCR Supermix High Fidelity –??
    • 67 ul of primer SB-prep-2Eb
    • 67 ul of primer SB-prep-3P-1
    • 10 ul of template DNA at 10ng/ul (100ng total)
    • Aliquot 100ul per well in 96 well plate</P><P><B>PCR program</B>
1. 95C/2min
2. 95C/30s
3. 55C/30s
4. 68C/3min
5. Repeat cycle (steps 2 to 4, 37 more times)
6. 68C/10min</P><P><B>PCR cleanup</B>
Purification of 96 well plates was done through Promega Wizard SV 96 PCR Clean-Up kit and a vacuum manifold. The protocol below follows the manual, with a few changes (in bold), however please see manual for setup instructions.</P><P>1. Add equal volume of Binding Solution to PCR product (add 100ul of Binding Solution to 100ul of product)
2. Mix by pipetting, transfer all 200ul to Binding Plate, let sit for 1 min
3. Apply vacuum until samples pass through, about 30s to 1 min
4. Add 200 ul of freshly prepared 80% ethanol to Binding Plate, let sit for 1min, apply vacuum until ethanol passes through, about 20s to 1 min.
5. Repeat ethanol wash (step 4) twice more for three washes total
6. Remove Binding Plate from wash manifold, blot on kim wipes, reinstall in wash manifold
7. Apply vacuum for 4 min to fully dry Binding Plate
8. Remove Binding Plate from wash manifold, blot on kim wipes, reinstall in collection manifold
9. Add 50ul of TE buffer, let sit for 1 min, apply vacuum until eluted, about 1 min
10. Repeat 50ul elution (step 9) for a total elution of 100ul
11. Measure concentration on nanodrop, adjust to 25 ng/ul with TE</P><P><B>Quality Control for constructed linearized plasmid backbones</B>
We recommend QCing constructed linearized plasmid backbones, to test success of PCR, ligation efficiency, and background.</P><P>1. Run unpurified PCR product (1 ul) on a gel to verify the correct band and concentration and lack of side products.
2. Test concentration of purified PCR product. Note: Expected yield should be 40ng/ul or higher. Adjust to 25ng/ul with TE.
3. Run a digest and ligation test with purified PCR product to determine EcoRI and PstI cutting and ligation efficiency.</P><P><B>Digest:</B></P><DL><DT>• Digest Master Mix (10rxns)</DT><DD>• 15 ul NEB Buffer 2</DD><DD>• 1.5 ul BSA</DD><DD>• 90 ul dH20</DD><DT>• Run Digest</DT><DD>• 4 ul of plasmid backbone (approximately 100 ng)</DD><DD>• 10.5 ul of Digest Master Mix</DD><DD>• 0.5 ul either EcoRI-HF or PstI enzyme (not both!)</DD><DD>• Digest 37C/30min; 80C/20 min</DD><DD>• Proceed directly to ligation</DD></DL><P><B>Ligation</B></P><DL><DT>• Ligation Master Mix (10rxns)</DT><DD>• 20 ul T4 DNA ligase buffer</DD><DD>• 5 ul T4 DNA ligase</DD><DD>• 25 ul water</DD><DT>• Ligation Test</DT><DD>• Add 5 ul of ligation master mix to digested product</DD><DD>• Ligate 16C/30min; 80C/20 min</DD><DD>• Run all 20 ul on a gel</DD><DD>• Compare intensity of the single and double length bands. More efficient ligations will show stronger double length bands than single.</DD></DL><P><B>Transformation test</B>
    • Transform 1 ul of the diluted final product into highly competent cells
    • Control transform 10 pg of pUC19
    • Plate on the appropriate antibiotic
    • Observe few colonies. Any colonies represent background to the three antibiotic assembly process
    • Quantify the effective amount of remaining circular DNA able to transform</P><P><B>Quality control for plasmid backbone production</B><B>Gel Runs</B>
After each completed PCR run we tested the product in each well on a gel.</P><P>    • Created dilution plate (96 wells) with 19ul of dH20 in each well
    • Added 1ul of PCR Product to each well, using multi-channel pipette
    • Loaded all 20ul into two 48 lane E-gels, following our standard loading pattern
    • Loaded 20ul of NEB 2-log ladder (1:100 dilution) into all four ladder lanes
    • Ran E-gels for 24min
    • Imaged according to standard parameters</P><P><B>Gel Results</B>
pSB1C3</P><P><B>Nanodrop Tests</B>
Once we determined that product was standard across all wells through the gel run, we purified the PCR product with a Promega Wizard SV 96 Kit. Note: We found yields to be around 40ng/ul after purification at a total volume of about 70ul (Eluted with 50ul twice, but not all elution passes through)</P><P>    • After purification tested six wells on Nanodrop: 1D, 3A, 6D, 9H, 12D, 12H
    • Combined contents of all wells into 15ml tube
    • Nanodropped combined product three times and averaged
    • Adjusted to 25ng/ul with TE
    • Nanodropped adjusted product three times and averaged</P><P><B>Digest and Ligation Tests</B>
Results from the digest and ligation tests completed for the 2011 Linearized Plasmid Backbones. Tests were for pSB1C3, pSB1A3, pSB1K3, pSB1T3 and pSB1C3 from 2010 (Tom’s pSB1C3). See: Digest and Ligation Test Protocol<B>Notes:</B>
    • Cutting with PstI and ligating is more efficient then cutting with EcoRI and ligating. For the former, double band quantity appears greater than single band quantity. Whereas for the latter, the reverse is true.
    • pSB1C3 from 2011 appears to be just as efficient if not more so than the pSB1C3 from 2010 (may have degraded over time)</P><P><B>Transformation test</B>
    • Transform 1 ul of the diluted final product into highly competent cells
    • Control transform 10 pg of pUC19
    • Plate on the appropriate antibiotic
    • Observe few colonies. Any colonies represent background to the three antibiotic assembly process
    • Quantify the effective amount of remaining circular DNA able to transform</P><H2><U>Miniprep</U></H2><P>
1. Save 500 ul for glycerol stock (see protocol on “making glycerol stocks”)
2. Use QiaSPIN Miniprep protocol to isolate plasmid
3. Make sure you note the volume of culture you are using in the Miniprep.  This, along with the OD reading, will help us determine how much culture we need per level of plasmid obtained for future transfections.
4. See detailed Miniprep protocol in QiaSpin Handbook. Can also be found here: http://www.qiagen.com/hb/qiaprepminiprep
5. Measure the concentration of the plasmid using the Nanodrop (ChenLab).</P><H2><U>Plate Making </U></H2><H4>Making LB/AMP Plates</H4><P>1. Find a vessel with suitable volume (i.e. a vessel that will hold the entire volume of LB/AMP you plan to make PLUS AT LEAST an EQUAL VOLUME OF AIR to the amount of liquid in said vessel).
2. To make ONE LITER OF LB/AMP, put 35g of LB agar into a vessel with ONE LITER of MilliQ water (35g:1L water).
3. Take a clean (washed in MilliQ) stir bar and place into the vessel. Stir on stir plate for approx. 5min.
4. Take vessel (containing LB agar and stir bar) to the autoclave room and autoclave for 30 minutes on the LIQUID SETTING. NOTE: Autoclave vessel in a pan of water.
5. After autoclaving, place a thermometer into the water pan and allow to sit.
6. When temperature is 60C, place vessel with LB agar onto the stir plate and begin to stir.
7. Add 100mg of AMPICILIN to the LB agar and stir for 2 minutes.
8. QUICKLY pipet 30ml of LB/AMP into Petri dishes. One liter LB/AMP should make approx 33 plates. NOTE: SAVE THE BAG THE PLATES CAME IN FOR STORAGE OF PLATES IN THE REFRIGERATOR.
9. Allow plates to cool on bench top overnight.
The next day, MARK EACH PLATE with ONE BLACK STRIPE, place the LB/AMP plates back into their bags, and place in the refrigerator. NOTE: PLEASE WRITE: “LB/AMP PLATES” on the bag.</P><H2><U>Transformation</U></H2><P>
Transformation is the process of introducing plasmid DNA into bacterial cells which provide the transcriptional and translational apparatus necessary for gene expression.  This protocol describes the technique for inserting plasmid DNA into E. coli cells using chemical treatment and heat shock.</P><P><B>Materials:</B>
• Chemically competent DH5alpha E. coli cells (Invitrogen #18263-012)
• Plasmid DNA to be inserted
• Ice bucket (VWR midi ice pan #35754-004)
• Foam eppendorf tube holders (VWR test tube rack with small holes #35754-076)
• 1.5 ml Eppendorf tubes
• Water bath adjusted to 42C
• Timer/wristwatch
• SOC medium
• 37C incubator
• LB plates ampicillin (or other correct antibiotic)
• Glass spreader
• Bunsen burner
• Incubator set at 37C
• Rotator</P><P><B>Protocol:</B>
1. Fill an ice bucket with crushed ice and water to form a slush.
2. Thaw a 200\ul~ tube of Invitrogen library efficiency chemically competent DH5alpha cells on by inserting the tube into the slush mixture.
3. Insert several 1.5 ml eppendorf tubes into a foam holder and prechill by inserting the foam holder and tubes into the slush mixture.
4. Aliquot 10-20\ul~ minimum of the competent cells to each tube, while holding the cells on ice.
5. Remove tubes one at a time from the foam holder, and mix 1.0 \ul~ of plasmid DNA (dilute your plasmid to 100ng/uL) into the cell.  Close the tube and mix by tapping and reinsert the tube into the foam holder and ice.
6. Hold the tubes on ice for 30 minutes.
7. Remove the foam holder from ice and submerge the tube bottoms into a 42C water bath for exactly 45 seconds.
8. Place the tubes back on ice, and hold for two minutes.  Add 0.5 ml of SOC medium (room temperature) to each tube.  Incubate the tubes at 37C for one hour.
9. For each tube, add 10\ul~ and 100\ul~ of the culture to a pair of LB plates with ampicillin (or other antibiotic).
10. Sterilize a glass spreader with ethanol, and flame to remove the ethanol.  Spread the culture on each plate with the spreader to form a uniform film on the plate surface.  Rotating the plate assists in this spreading.
11. Incubate the plates upside down overnight at 37C.
12. Examine plates after approximately 12 – 16 hours to evaluate transformation efficiency.  Choose plates which have many colonies, each carefully separated from others.</P><P><B>Transformation notes:</B>
We use Invitrogen chemically competent cells for transformation. These cells have been chemically treated and carefully frozen in a state suitable for heat shock and insertion of plasmid DNA.  This treatment is tricky and not the first thing you want to try, so we strongly recommend that first attempts utilize commercially prepared competent cells. The specific cells we normally use are E. coli DH5alpha cells. These cells are available in several transformation efficiencies.  The efficiency numbers relate number of transformed cells to microgram of plasmid DNA.  Invitrogen supplies low concentration pUC19 plasmid controls which can be used to verify the transformation efficiency of the cell line.  Performing these controls initially is a good idea, although they are usually not necessary unless a problem is suspected. DH5alpha cells are available in several efficiencies, ranging from 105 to 109 transformants per microgram.  We routinely use library efficiency cells and more rarely the max and sub-cloning efficiency cells. Invitrogen has started supplying SOC medium along with the competent cells.</P><H2><U>Gel Making </U></H2><P><B>Materials</B>
• 1X TAE
• SYBR safe (10,000X stock)
• Agarose
• Microwave
• Stir plate and stir bar (optional)</P><P><B>Procedure</B>
1. Add 300mL 1X TAE to a 500 mL bottle.
2. Measure out sufficient agarose to cast either a 1% (3 g) or 1.5% (4.5 g) gel.
3. Add the agarose to the TAE buffer in the 500 mL bottle.
4. Swirl to mix.
5. Microwave bottle with loosened cap on high until the gel starts to bubble and is transparent.
6. This generally takes just over two minutes for 300 mL. If you microwave too long, the gel will bubble over causing a big mess and you will need to start over.
7. Remove from microwave and let cool by either sitting on bench top or adding stir bar and placing on stir plate.
8. The advantage of the stir plate is that, if you forget about your gel for a while, it is less likely to solidify accidentally.
9. If you are in a hurry, you can place the bottle in a beaker of room temperature water on the stir plate to speed the cooling process significantly.
10. While gel is cooling, assemble casting trays and gel combs and verify that the trays are level.
11. Once gel is cooled so that it can be touched comfortably with your gloved hand, add 30 μL SYBR Safe (10,000X concentrate).
12. Pour gel into casting trays.
13. The height of the gel will depend on how much you wish to load. Diagnostic gels can be reasonably shallow since typically 10 μL volumes are loaded. For gel purifications, the gel should be deeper to enable loading of large sample volumes.
14. Let gels sit until they are solidified.
15. Gels are solid when they are cloudy in appearance and firm to the touch.
16. Gels may be used immediately. Alternatively, gels may be individually sealed in 6 x 10 inch polyethylene bags, labelled with initials, date and percentage and stored at 4 °C.
17. It is a judgement call as to whether a gel is too old to be used. If it takes on a shrivelled appearance, don’t use it. If there is lots of condensation on the bag, only use it if your intended experiment isn’t critical.</P><P><B>Materials for Running Gel</B>
• Prepared DNA ladder
• Precast gel with the appropriate percentage and well size/numbers for your samples (see above)
• 1X TAE
• Loading dye</P><P><B>Procedure</B>
1. Take a gel from the 4°C fridge.
2. If the number of gels is getting low, cast more gels as described above.
3. Place your gel in gel box.
4. Add 1X TAE buffer to gel box such that buffer just covers the top of the gel.
5. Remove comb.
6. Load 12 μL prepared ladder
7. Typically load ladder in left-most lane and sometimes right-most lane as well depending on whether you have the space.
8. Use 2 μL loading dye per 10 μL of sample.
9. Load samples left to right.
10. The capacity of the 8 well, 1.5mm wide well is approximately 45 μL. The capacity of the 15 well, 1.5mm well is approximately 15 μL.
11. Place gel box cover on gel box such that your samples will run towards the positive, red electrode.
12. Run your gel at ~85 volts for 1 hr 20 mins. Use the timer to enable automatic shutoff of your gel.
13. If you are in a hurry the gel can be run faster at ~95 volts for less than an hour.
14. Verify that bubbles are rising from the electrodes once you start your gel to ensure your gel is running properly.</P><H2>Lentivirus Prep and Infection Protocol</H2><P><B>Strategy Overview</B>
• Make plasmid with YFG** in Gateway, transform TOP10 bacterial cells, streak plates
• Miniprep TOP10 cells to amplify enough plasmid DNA for transfection
• Transfect HEK 293T cells with DNA with YFG and lenti packaging plasmids (3 total)
• Harvest Supernatent and infect YFC**</P><P><B>Bacterial growth</B>
Grow transformed bacteria in Super LB, which is Luria Broth supplemented with 0.4% glucose. Use Carbenicillin (analog of ampicillin) in the superbroth at a final concentration of 100 µg/mL as a selection agent (1:1,000 dilution of our 100 mg/mL stock). Grow bacteria at 265 RPM at 37 ºC overnight (usually 15-20 hours, especially if directly from frozen stock).</P><P>You need 6 mL culture per miniprep, each miniprep will yield ~12 µg YFG plasmid, which is enough for only 1 plate of HEK293T transfection. So I usually do 4-6 minipreps per YFG plasmid I want to amplify.</P><P><B>Day Before transfection</B>
Plate 8e6 HEK 293T cells in Pen/Strep-Free Media!!!! a 100 mm tissue culture-grade petri dish (note: these cells are different than regular 293 cells, as they express the T- antigen which is optimal for infection/fast growth/lentivirus preparation). Each plate will give you ~8 mL unconcentrated lentivirus supernatent.</P><P><B>Lipofectamine Transfection of HEK 293T cells for Lentivirus Production</B></P><P>1. 2 polypropylene tubes per plate to be infected; 1.5 mL Opti-MEM media in each. One tube is labeled DNA, the other is labeled lipofectamine</P><P>2. Add 36 µL Invitrogen lipofectamine 2000 reagent to lipofectamine tube. Pipette gently to mix</P><P>**YFG = Your Favorite Gene. This is your gene of interest, such as EGFP, mCherry, or Luciferase.
**YFC = Your Favorite Cell type. These are the cells you ultimately want to express YFG, such as 3T3 or HDF.</P><P>3. Add plasmids to DNA tube. 2 µg MD2.G plasmid + 4 µg PSPAX2 plasmid + 6 µg YFG plasmid. Flick tube gently to mix</P><P>4. Wait 5 min (best not to wait longer than 15 min)</P><P>5. Add contents of DNA tube to lipofectamine tube. Pipette up and down. The lipofectamine will form transfectable complexes with the DNA</P><P>6. Wait exactly 15 min. The lipofectamine is self-assembling, so this timing is CRITICAL to get correct.</P><P>7. Add 5 mL regular media to the Opti-MEM media containing the lipofectamine-plasmid complexes. You should now have 8 mL. Aspirate media from the HEK 293T cells in the 100 mm dish. Add all 8 mL of the transfecting reagent to the 100 mm dish.</P><P>8. do the transfection with Lipofectamine 2000 for 12-16 hrs (overnight). Then change media to normal media, media containing Pen/Strep is good from here on out. You want to remove the residual lipofectamine and DNA complexes from the supernatent so they do not contaminate the lentivirus stocks you are making.                                   </P><P>9. Check the next day for YFG**, even if it is strictly inducible it should still be expressing in these transfected cells. The 5′ LTR on the pRRL vector is different during lentivirus production (i.e. in the context of the pRRL vector) than in the active virus. The 5′ LTR in pRRL is a chimeric promoter than is partly derived from the RSV virus (and thus is a strong, constitutive promoter).  In the active virus, both LTRs are derived from copying the 3′ LTR which has very limited activity as a promoter.  So it is not an issue of the LTR being absent in the virus (you can’t have a functional virus without it), but just different. So, if your YFG** is GFP or mCherry, your cells should be fluorescent. If they are making firefly luciferase, they should be luminescent with luciferin addition to the media (this will not affect your lentivirus stock, so definitely try it!). This allows you a good spot-check that your construct is correct and that your transfection was correct.</P><P>10. Allow lentivirus to accumulate in the supernatent for ~72 hours from the time of transfection. If you transfected on Thursday, changed media on Friday, you will harvest supernatent on Sunday afternoon. There’s a range here, generally you can harvest virus 48-72 hours after the media change (64-88 hrs post-transfection).</P><P>11. You can use this exact protocol to transfect cells and not make lentivirus if you just leave out the MD2.G and PSPAX2 plasmids. But tranfected cells lose the plasmids over several days. So you really want to make lentiviruses instead. Trust me.</P><P><B>Lentiviral Harvest</B></P><P>1.  Pipet 293 supernatant into a 15mL conical tube.  Hold the tube at a slant for at least 1 minute to collect any residual supernatant (usually 0.25-0.5mL additional material recovered this way).  Sterilize the pipet with bleach and add bleach to the p100 dishes.  I usually keep a 50mL conical tube with 5mL of bleach as a temporary respository for used pipets while I’m working in the hood.  You’ll also need the big glass dishes full of bleach (usually 50-70% diluted with water) in which you can put your contaminated tips.</P><P>2.  Centrifuge the 293 supernatant, 15 minutes, 4C, at 3800rpm.  During this centrifugation step, label cryovials into which you’ll aliquot the virus.</P><P>3.  Remove the supernatant to a fresh tube with care not to disturb any cell pellet at the bottom of the conical tube.  Aliquot and store at -80C.  Usually I make 1mL aliquots, and freeze in the plastic racks (if you drop them into a storage box before they are frozen, virus can splash up into the top of the vial, which can pose a risk to you when you open it later.</P><P>NOTE: I don’t usually do this concentration step:
If desired, you can concentrate the virus using the PEG-iT kit from SystemBio. They have a protocol (pdf) online. Briefly, you dilute the 5x PEG solution to 1x in the supernatant from step 3 and then incubate at 4C overnight, followed by a 30min, 1500g spin. You discard the supn’t, and resuspend the pellet in a minimum volume (1/10-1/100 initial volume) of PBS or DMEM. Then store at -80 in aliquots. The PEG solution is in the t.c. fridge on the bottom right shelf.</P><P>4. Wipe down the hood with 70% ethanol + 1% SDS when you are done.  Any obvious viral spills should be treated immediately with bleach.</P><P><B>Lentiviral infection of Mammalian Cells</B></P><P>Spinfection is the preferred way to do lentiviral infection. However, on large scale infections (i.e. using T-flasks instead of multi-well plates), spinfection cannot be used because T-flasks cannot be placed in the centrifuge. So, use longer infection times instead, as described below. If you are limited in viral amount, Grace has written up a protocol using HEPES-buffered media so you can do the spinfection. HEPES is only needed to keep the pH of the solution consistent if you do the spinfection (since you will be away from CO2 in the incubator to keep the pH).</P><P>NOTE: polybrene is a polycation and helps prevent charge-based repulsion of the lentivirus from the cell membrane. * polybrene (6 µg/mL, stock is at 6 mg/mL; polybrene is a polycation which decreases electrostatic repulsion of virus to the cell membrane)</P><P>1) Make stock polybrene media (include non-infection ctrls, make extra):
Add stock polybrene media to each cell culture vial</P><P>Thaw, mix, then add virus(es) using filter tips. Discard filter tips into bleach container (diluted 1:1 with water).</P><P>Mix cell culture vessel very well to evenly distribute the virus</P><P>Return plates to incubator for ~36 hours</P><P>Remove virus-containing media to bleach, wash with PBS, add fresh growth media</P><P>Cells will take ~48 hours before you can test for stable expression. Example: If you infected on Sunday with a fluorescent YFG, cells will begin to be fluorescent on Tuesday night. Change to fresh media on Wednesday morning.</P><P><B>HOW MUCH VIRUS TO USE?</B>
Rule of thumb:
1 µL per 100,000 cells for HEK cells
10x that amount if using unconcentrated supernatent (most likely case) another 100x if using hard to infect cells (mouse cells, HDF, NHLF, etc.)</P><P>Standard titration of the virus is necessary for each lentivirus preparation you make, since we have no easy way to measure viral titer or MOI (lentiviruses do not form plaques, etc.). Perhaps DLS (dynamic light scattering) would be useful here, which is typically used to measure nanoparticle concentration.</P><P>**YFG = Your Favorite Gene. This is your gene of interest, such as EGFP, mCherry, or Luciferase.
**YFC = Your Favorite Cell type. These are the cells you ultimately want to express YFG, such as 3T3 or HDF.</P><H2><U>Characterizing Experiments</U></H2><H4>Characterizing CMV-Gaussia</H4><P><B>Luminometer Experimental Setup</B></P><DL><DT>1. We have cells constitutively expressing Gaussia in solution</DT><DT>2. Injection: ____ of Gaussia supernatant (A) and ___ of Cz solution (B)</DT><DT>3. Check for luminescence</DT><DD>a. If luminescence occurs, can characterize the luminescence by adding coelenterazine solution in a gradient to find optimal substrate concentration</DD></DL><P><B>Controls</B></P><TABLE><TBODY><TR><TH>Type</TH><TH>A</TH><TH>B</TH><TH>Controls for?</TH></TR><TR><P>Negative</P><P>Gaussia supernatant</P><P>Acidified methanol solution w/o Cz</P><P>Cz is needed for luminescence</P></TR><TR><P>Negative</P><P>Supernatant from cells not expressing Gaussia</P><P>Cz</P><P>Gaussia needed for luminescence</P></TR><TR><P>Positive</P><P>Gaussia</P><P>Dual-Luciferase Kit</P><P>Check if the Gaussia is good</P></TR></TBODY></TABLE><H4>Characterizing CMV-Aeq</H4><P><B>Luminometer Experimental Setup</B>
1. Get cells to express Aeq
2. Add Cz to cell solution
3. Wash many times in PBS (w/o Ca2+)
4. Lyse cells, save the lysate
5. Injection: ___ of Lysate (A) and ___ of Ca2+ solution (B)</P><P><B>Controls</B></P><TABLE><TBODY><TR><TH>Type</TH><TH>A</TH><TH>B</TH><TH>Controls for?</TH></TR><TR><TD>Negative</TD><TD>Lysate</TD><TD>PBS</TD><TD>Calcium required</TD></TR><TR><TD>Negative</TD><TD>PBS</TD><TD>Ca2+</TD><TD>Lysate Required</TD></TR><TR><TD>Negative</TD><TD>Lysate from WT HEK293T cells</TD><TD>Ca2+</TD><TD>Aeq required</TD></TR><TR><TD>Negative</TD><TD>Lysate from HEK293T cells expressing Aeq with no Cz added</TD><TD>Ca2+</TD><TD>Cz required</TD></TR><TR><TD>Positive</TD><TD>Gaussia supernatant</TD><TD>Cz being used</TD><TD>Cz works</TD></TR><TR><TD>Positive</TD><TD>Gaussia supernatant</TD><TD>Dual luciferase kit</TD><TD>Gaussia good (controls the control)</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><DIV id="footer-top"><DIV id="footer-content"><DIV id="footer-bottom" class="clearfix"><DIV class="container"><UL id="menu-main" class="bottom-nav"><LI class="menu-item menu-item-type-custom menu-item-object-custom menu-item-home menu-item-62"><A href="http://benjaminshyong.com/igem2011/">Home</A></LI><LI class="menu-item menu-item-type-custom menu-item-object-custom current-menu-ancestor current-menu-parent menu-item-217"><A href="http://benjaminshyong.com/igem2011/project/overview/">Project</A></LI><LI class="menu-item menu-item-type-post_type menu-item-object-page menu-item-75"><A href="http://benjaminshyong.com/igem2011/results/">Results</A></LI><LI class="menu-item menu-item-type-post_type menu-item-object-page menu-item-67"><A href="http://benjaminshyong.com/igem2011/biobrick/">Biobrick</A></LI><LI class="menu-item menu-item-type-post_type menu-item-object-page menu-item-68"><A href="http://benjaminshyong.com/igem2011/notebook/">Notebook</A></LI><LI class="menu-item menu-item-type-post_type menu-item-object-page menu-item-76"><A href="http://benjaminshyong.com/igem2011/safety-and-society/">Human Practices</A></LI><LI class="menu-item menu-item-type-post_type menu-item-object-page menu-item-78"><A href="http://benjaminshyong.com/igem2011/safety/">Safety</A></LI><LI class="menu-item menu-item-type-post_type menu-item-object-page menu-item-63"><A href="http://benjaminshyong.com/igem2011/about-us/">About Us</A></LI><LI class="menu-item menu-item-type-custom menu-item-object-custom menu-item-87"><A href="https://2011.igem.org/Main_Page">iGEM</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Penn/project/protocols/">http://2011.igem.org/Team:Penn/project/protocols/</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Penn/project/protocols/" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Penn/project/protocols/" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Penn/project/protocols/&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Penn/project/protocols/&amp;oldid=203791" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>