<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Penn_notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Penn/notebook/">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Penn/notebook/&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/notebook/&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/notebook/&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Penn/notebook/" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Penn/notebook/</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>  Notebook | Penn iGEM 2011</TITLE></P><DIV id="top"><DIV id="top-wrapper"><DIV id="top-content"><DIV id="bottom-light"><DIV class="container"><DIV id="menu" class="clearfix"><DIV id="menu-right"><DIV id="menu-content"><UL id="top-menu" class="nav"><LI id="menu-item-62" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-home menu-item-62"><A href="http://benjaminshyong.com/igem2011/">Home</A></LI><LI id="menu-item-217" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-217"><A href="http://benjaminshyong.com/igem2011/project/overview/">Project</A></LI><LI id="menu-item-73" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-73"><A href="http://benjaminshyong.com/igem2011/project/overview/">Overview</A></LI><LI id="menu-item-72" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-72"><A href="http://benjaminshyong.com/igem2011/project/our-system/">Our System</A></LI><LI id="menu-item-74" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-74"><A href="http://benjaminshyong.com/igem2011/project/protocols/">Protocols</A></LI><LI id="menu-item-70" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-70"><A href="http://benjaminshyong.com/igem2011/project/future-directions/">Future Directions</A></LI></UL><LI id="menu-item-75" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-75"><A href="http://benjaminshyong.com/igem2011/results/">Results</A></LI><LI id="menu-item-270" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-270"><A href="http://benjaminshyong.com/igem2011/results/luminometer/">Luminometer Experiments</A></LI><LI id="menu-item-269" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-269"><A href="http://benjaminshyong.com/igem2011/results/ivis-imaging/">iVIS Imaging</A></LI><LI id="menu-item-268" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-268"><A href="http://benjaminshyong.com/igem2011/results/fura-2-calcium-imaging/">Fura-2 Calcium Imaging</A></LI><LI id="menu-item-67" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-67"><A href="http://benjaminshyong.com/igem2011/biobrick/">Biobrick</A></LI><LI id="menu-item-68" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-27 current_page_item menu-item-68"><A href="http://benjaminshyong.com/igem2011/notebook/">Notebook</A></LI><LI id="menu-item-76" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-76"><A href="http://benjaminshyong.com/igem2011/safety-and-society/">Human Practices</A></LI><LI id="menu-item-314" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-314"><A href="http://benjaminshyong.com/igem2011/safety-and-society/media-and-synthetic-biology/">Media and Synthetic Biology</A></LI><LI id="menu-item-313" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-313"><A href="http://benjaminshyong.com/igem2011/safety-and-society/survey-results/">Survey Results</A></LI><LI id="menu-item-78" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-78"><A href="http://benjaminshyong.com/igem2011/safety/">Safety</A></LI><LI id="menu-item-63" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-63"><A href="http://benjaminshyong.com/igem2011/about-us/">About Us</A></LI><LI id="menu-item-66" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-66"><A href="http://benjaminshyong.com/igem2011/about-us/the-team/">The Team</A></LI><LI id="menu-item-64" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-64"><A href="http://benjaminshyong.com/igem2011/about-us/advisors/">Advisors</A></LI><LI id="menu-item-237" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-237"><A href="http://benjaminshyong.com/igem2011/about-us/sponsors/">Sponsors</A></LI><LI id="menu-item-87" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-87"><A href="https://2011.igem.org/Main_Page">iGEM</A></LI></DIV></DIV></DIV><DIV id="category-name"><H1 class="category-title">Notebook</H1></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="content"><DIV id="content-shadow"><DIV class="container"><DIV id="breadcrumbs"><A href="http://benjaminshyong.com/igem2011">Home</A><SPAN class="raquo">»</SPAN>
				
									  Notebook					</DIV><DIV id="content-area" class="clearfix"><DIV id="left-area"><DIV class="entry post clearfix"><H1><STRONG><SPAN style="text-decoration: underline;">Table of Contents</SPAN></STRONG></H1><P>Click on the links below to jump to the month in question.</P><H1>May 2011</H1><P> </P><H1>June 2011</H1><P> </P><H1>July 2011</H1><P> </P><H1>August 2011</H1><P> </P><H1>September 2011</H1><H1><STRONG><SPAN style="text-decoration: underline;">May 2011</SPAN></STRONG></H1><P><A href="#Top">Back to Top</A></P><H1>5/21/11</H1><UL><LI>froze down HEK293T @-80C and C212 CMV-Gaussia at -80C for the iGEM team.
Took 2 plates of 100% confluence C2C12 CMV-Gaussia P6 and transferred into 5 bullets 500uL each (labeled on bullet= 2/5 p100 plate)</LI><LI>Took 2 plates of ~100% confluent 293T CMV-Gaussia P10 and transferred into 5 bullets 500uL each (white cap on the bullet)</LI><LI>Whenever you add trypsin, you increase passage # by one</LI><LI>P7 C2C12 CMV-Gaussia</LI><LI>P11 293T CMV-Gaussia</LI><LI>293T cells have very weak contact with the plate, so we must be cautious not to add PBS too quickly.</LI><LI>10 bullets are put into -80C freezer by Jordan Miller at 7:00pm.</LI></UL><P> </P><H1>5/23/11</H1><P>TC Scope</P><UL><LI>Rings are for phase contrast</LI><LI>Left for 10x objective</LI><LI>Mid, right for 4x and 3D gels</LI></UL><P>TE200 Epifluorescent Scope</P><UL><LI>Two light sources</LI><LI>1. Transmitted light – comes from the top of your sample</LI><LI>2. Light Box (metal halide) – comes from the bottom of your sample
</LI><LI>Leave lamp on if someone is using after you</LI><LI>Lamp only lasts 1500 hours</LI><LI>Don’t abuse reservation (see schedule)</LI></UL><P>Turn on Arc lamp if you need fluorescent light</P><UL><LI>Lower left, vertically oriented, push in to turn ON</LI><LI>LCD lights up if you turn ON</LI><LI>The dial changes transmission setting</LI><LI>More transmission = more chance of killing cells
</LI><LI>Should optimize least amount of light needed</LI></UL><P>Immersion Objective</P><UL><LI>Want to minimize exposure to light</LI><LI>Spinning disc vs. laser scanning</LI><LI>Spinning disc also uses a laser and is faster</LI></UL><P> </P><H1>5/24/11</H1><P>PEARL Training</P><UL><LI>5 minutes to warm up</LI><LI>Start new project</LI><LI>How well can IFP stay fluorescent – 30 million cells/mil</LI><LI>Push button to open</LI></UL><H1><SPAN style="text-decoration: underline;"><STRONG>June 2011</STRONG></SPAN></H1><P><A href="#Top">Back to Top</A></P><H1>6/1/11</H1><UL><LI>Made over 100 +amp plates</LI><LI>Made 2L 1x TAE</LI></UL><P> </P><H1>6/9/11</H1><UL><LI>Humanized Aequorin gene synthesis from GeneArt – ordered</LI><LI>Add this sequence in front of Aeq: CACCGGTAATCGAATTCAGCCACCATG
</LI><LI>Contains D-TOPO site, AgeI, spacer, EcoRI, and a kozak sequence.</LI></UL><LI>Total 618base pairs.</LI><LI>Experimental planning: pRRL-CMV-Gaussia + Coelenterazine- luminescence; pcDNA-CMV-ChR2-image for eYFP.</LI><H1>6/16</H1><UL><LI>Cells with ChR2 were plated on LB agar.</LI></UL><P> </P><H1>6/17/11</H1><UL><LI>phusion PCR- mutagenesis protocol for precoelenterazine (PCZ)</LI><LI>rationale: patent filed by ward et al claimed that a 1bp mutation in WT GFP chromophore region results in PCZ formation, which then forms CZ (in E. coli). We decided to use EGFP because it is more sable and expresses better in mammalian cells.</LI><LI>Designed and ordered primers to introduce the mutation.</LI><LI>We have pRRL-CMV-EGFP-mODC as a template.</LI><LI>Designed 4 primers (2 pairs) to introduce the single bp mutation.</LI><LI>Ran a gel after PCR to make sure correct sized bands formed after mutagenesis.</LI><LI>Once we clone out PCZ, we can put that gene into pENTR/D-TOPO .</LI><LI>We now have pENTR/D-TOPO-PCZ-mODC. Next step is to add a promoter source.</LI></UL><P> </P><H1>6/20/11</H1><P><SPAN style="text-decoration: underline;">CMV-ChR2 (with ampR gene) cell culture:</SPAN></P><OL start="1"><LI>Made 5 mL of 50 mg/ml ampicillin in LB broth</LI><LI>Bacterial colonies with ChR2 were inoculated in LB broth with ampicillin at 37C</LI><LI>Added 10 uL of 50 mg/ml ampicillin solution to each tube to reach a concentration of 100 ug/mL in the 5 mL tube.</LI></OL><P><SPAN style="text-decoration: underline;">eGFP-MODC site directed mutagenesis to get Pre-CZ:</SPAN></P><OL start="1"><LI>Gel electrophoreis with product of 1st PCR to make sure that bands are the right length</LI><LI>DNA Purification after first PCR (remove primers)</LI><LI>Step 2 of PCR for site-directed mutagenesis of eGFP to PreCZ</LI></OL><OL><LI>6 cycles without primers to allow two parts to anneal</LI><LI>Add primers and run 30(?) cycles to amplify gene</LI></OL><P>DNA Purification after second PCR (remove primers)</P><H1>6/21/11</H1><P><SPAN style="text-decoration: underline;">CMV-ChR2 cell culture:</SPAN></P><OL start="1"><LI>Miniprep</LI></OL><P> </P><H1>6/23/11</H1><P><SPAN style="text-decoration: underline;">1. Luminometer training and sample measurements –</SPAN></P><P>— We used Dan’s parameters already programmed in the luminometer.  Both the LAR II and Stop&amp;Glow substrates had the same parameters of 80 uL volume, a delay period of 2 seconds before measurement (allows reaction to take place), and a integration time of 10 seconds (measurement time).— Gaussia sample was supernatant from HEK293T cells.  The supernatant was undiluted and in the fridge at 4C since 5/11/11.— Media was the negative control.  It is good to have more than 1 sample of your negative control in order to get error bars.</P><TABLE width="416" border="1" cellspacing="0" cellpadding="5"><TBODY><TR><TD valign="top"><STRONG>Sample</STRONG></TD><TD valign="top"><STRONG>RLUs1</STRONG></TD><TD valign="top"><STRONG>RLUs2</STRONG></TD><TD valign="top"><STRONG>Ratio</STRONG></TD></TR><TR><TD valign="top">1 – Gaussia</TD><TD valign="top">84</TD><TD valign="top">447564736</TD><TD valign="top">0.0</TD></TR><TR><TD valign="top">2 – Media</TD><TD valign="top">76</TD><TD valign="top">416</TD><TD valign="top">0.1833</TD></TR><TR><TD valign="top">3 – Media</TD><TD valign="top">74</TD><TD valign="top">337</TD><TD valign="top">0.2221</TD></TR><TR><TD valign="top">4 – Media</TD><TD valign="top">75</TD><TD valign="top">288</TD><TD valign="top">0.2624</TD></TR></TBODY></TABLE><P>— The luminescence of Gaussia was 9 digits compared to the controls that were 3 digits (background luminescence).  Results show that the Gaussia lasted in the supernatant for longer than a month!</P><P>2. <SPAN style="text-decoration: underline;">Optical density measurements of CMV-ChR2 bacterial cultures with spectrophotometer -</SPAN></P><TABLE width="416" border="1" cellspacing="0" cellpadding="5"><TBODY><TR><TD valign="top"><STRONG>Sample (Colony)</STRONG></TD><TD valign="top"><STRONG>Concentration (cell/mL)</STRONG></TD></TR><TR><TD valign="top">1a</TD><TD valign="top">1.04 x 10<SUP>9</SUP></TD></TR><TR><TD valign="top">1b</TD><TD valign="top">1.48 x 10<SUP>9</SUP></TD></TR><TR><TD valign="top">2a</TD><TD valign="top">1.30 x 10<SUP>9</SUP></TD></TR><TR><TD valign="top">2b</TD><TD valign="top">1.30 x 10<SUP>9</SUP></TD></TR></TBODY></TABLE><P>Since 1b was the most concentrated, a 25% glycerol solution was prepared with the sample and is being stored in the BE undergrad freezer (500 uL 50% glycerol and 500 uL 1b sample).  The remaining colonies 1a, 1b, 2a, 2b were spun down with centrifuge and are in the BE undergrad -20C freezer (3 mL of each colony).</P><P> </P><H1>6/29/11</H1><UL><LI>Autoclaved eppendorf tubes and bottles</LI><LI>Called Promega Tech to figure out how to get graph readout from luminometer</LI><LI>Picked up Fetal Bovine Serum, Qualified, USDA approved, Lot #643123 from Cell Center</LI><LI>Checked on Aequorin status, told it would arrive 7/6/11</LI></UL><P> </P><H1>6/30/11</H1><UL><LI>Made sterile 1x PBS in 250mL aliquots</LI><LI>Designed and ordered ChR2 (CatCh) mutagenesis primers</LI><LI>Sterilized Tissue Culture hood by turning on UV light overnight</LI></UL><H1><SPAN style="text-decoration: underline;"><STRONG>July 2011</STRONG></SPAN></H1><P><A href="#Top">Back to Top</A></P><H1>7/5/11</H1><OL start="1"><LI>Diluted 2 L of 10 x PBS to 1 x PBS.  Autoclaved.  Sterile 1 x PBS is now in BE undergrad lab.</LI></OL><P> </P><H1>7/6/11</H1><UL><LI>Made 70% EtOH – in spray bottle</LI><LI>Defrosted FBS in 37C water bath</LI><LI>Sterile TC Hood technique training</LI><LI>New mentor is Ken Chen (CBE Undergraduate Lab Director)</LI><LI>Prepared CO2 independent media. Located in BE undergrad fridge</LI><LI>Began aliquoting FBS into 15ml conical tubes so that we don’t have to continuously thaw/refreeze it. This will minimize protein denaturation.</LI><LI>Began constructing flow chart for project</LI><LI>Dr. Sarkar signed off on chem stockroom form, allowing us to purchase supplies</LI><LI>Cleaned Tissue Culture hoods</LI></UL><P> </P><H1>7/7/11</H1><UL><LI>Aliquoted L-glutamine into 20 tubes of 5ml, Pen Strep into 20 tubes of 5ml each. Being stored in BE -20C freezer.</LI><LI>Aliquoted FBS into 40 tubes of 10ml each. Being stored in BE -20C freezer.</LI><LI>Prepared for lab transplantation. Moved water bath into room with hoods.</LI></UL><P> </P><H1>7/8/11</H1><P>Aspiration setup in hood by connecting tube to vacuum spout and flask for biohazard disposal.</P><P> </P><H1>7/11/11</H1><UL><LI>Made 15 1mL-15mL tubes of PBS to use as weights for the centrifuge.</LI><LI>Cleaned up and organized lab space.</LI><LI>Tissue culture of HEK293T cells – finished at 12:00 noon.</LI><UL><LI>Spun down original sample to remove DMSO</LI><LI>Resuspend sample with 10 mL CO2-independent media</LI><LI>Dilute sample 1:10 by taking 0.89 mL sample and adding to 10 mL media on a P100 plate.  Made four P100 plates.</LI></UL></UL><P> </P><H1>7/12/11</H1><UL><LI>CO2 Independent Media was replaced to ensure that DMSO was removed.</LI><LI>Cells were observed in microscope, confluency has decreased since plating to about 10%. (Does this mean our media does not work?)</LI></UL><P> </P><H1>7/13/11</H1><UL><LI>Made and autoclaved LB Agar</LI><LI>Poured plates with +chloramphenicol (170ug/mL)</LI></UL><P> </P><H1>7/14/11</H1><UL><LI>Transformed PCZ into DH5a cells.</LI><LI>transformed LuxBioBrick.</LI></UL><P> </P><H1>7/15/11</H1><UL><LI>6PM: inoculated pENTR/D-Topo-PCZ-mODC and pcDNA31-ChR2-eYFP in 4mL culture with appropriate antibiotic (kan for pcz, amp for chr2)</LI></UL><P> </P><H1>7/16/11</H1><UL><LI>11AM: mini-prepped pENTR/D-Topo-PCZ-mODC and pcDNA31-ChR2-eYFP. pcDNA31-ChR2-eYFP had yield of 60ng/ul, while pENTR/D-Topo-PCZ-mODC had a terrible yield of ~6ng/ul</LI><LI>To ensure that we have enough bacteria growing, we will measure OD of our LB liquid culture next time before mini-prepping.</LI><LI>5:30PM: Picked some more colonies from the 100ul plate of PCZ-mODC, and tried inoculation again with these colonies</LI></UL><P> </P><H1>7/17/11</H1><UL><LI>Measured OD of six PCZ samples</LI><LI>Mini-prepped two of those samples. PCZ yield: 105.9ng/uL</LI><LI>Next step is to gateway pENTR-PCZ into pRRL-CMV</LI></UL><P> </P><H1>7/18/11</H1><UL><LI>Made 20 LB agar (16 g powder in 500 mL Milli-Q) + 25 ug/mL chloramphenicol plates</LI><UL><LI>Added 368 uL chloramphenicol stock (34 mg/mL) to 500 mL LB agar solution to get a final concentration of 25 ug/mL</LI></UL></UL><P> </P><H1>7/19/11</H1><UL><LI>Re-transformed LuxBioBrick</LI></UL><P> </P><H1>7/20/11</H1><UL><LI>Removed plates from the fridge so that they could dry (trying to salvage the liquidy plates).</LI><LI> Learned how to use Ken Chen’s autoclave machine and wrote a protocol for the machine/LB plate making.  The autoclave machine is the small 70’s-looking machine on wheels on the far side of the BE undergrad lab.</LI><LI>Streaked a plate using a colony from the 20ul Lux plate (this was the only plate that had successful cell growth).</LI><LI>Started autoclaving 2 250ml LB agar with the intention of making chloramphenicol plates and ampicillin+arabinose plates.  The amp plates that are currently in the fridge look bad and should be checked.</LI></UL><P> </P><H1>7/21/11</H1><UL><LI>16 LB + chloramphenicol + arabinose plates were made.  184ul of chloramphenicol was added to 250ml LB agar. (The amount used in 500ml of the agar was 368ul so we divided this number by two.  See lab journal 07/18 for details).</LI><UL><LI>.1876g of arabinose was added to this 250ml LB agar. (We want approximately 5*10^3uM of arabinose to maximize light production, so 5*10^-3moles/L*.25L*150.14g/mol = .1876g arabinose)</LI></UL><LI>12 LB + ampicillin plates were made.</LI><LI>Threw away LB + ampicillin plates that were liquid/unusable.</LI><LI>2 colonies were isolated from the 20ul lux plate and inoculated in 2 snap cap tubes at 4:30pm.</LI><LI>7.35ul of 34mg/ml chloramphenicol was added to 5ml LB broth to yield an end concentration of 50ug chloramphenicol/ml solution).</LI><LI>The plate with the streaked colonies and the original 20ul lux plate are in the minifridge in one of the side doors.</LI><LI>MatLab forms were emailed; signed letter will be mailed out tomorrow.</LI></UL><P> </P><H1>7/25/11</H1><UL><LI>Gateway reaction for PCZ into pRRL-CMV</LI><LI>Aeq+pENTR/D-TOPO</LI><LI>ChR2+pENTR/D-TOPO</LI><LI>Transfect 293T with ChR2- characterize tomorrow.</LI><LI>ChR2 miniprep yields was high: 447.6ng/uL.</LI></UL><P> </P><H1>7/26/11</H1><UL><LI>Stopped gateway with Proteinase K, transformed cells.</LI><LI>Characterized ChR2 in HEK cells under the scope for EYFP fluorescence – confirmed</LI><LI>Transformed TOPO Aeq + TOPO ChR2.</LI><LI>Inoculated pRRL in 1mL of culture.</LI><LI>Created PCR protocol with help of Daphne</LI><LI>Performed PCR of aequorin and ChR2 with Daphne</LI><LI>Ran gel to confirm PCR product on aequorin and ChR2</LI><LI>Made 10 agar +kanamycin plates</LI><LI>Performed Gateway on Pre-CZ</LI><LI>Ran gel with control primers and CatCh primers</LI><LI>Prepared presentations for SAAST/West Philadelphia students</LI></UL><P> </P><H3>Lab Meeting</H3><P><SPAN style="text-decoration: underline;">Human Cell</SPAN>:</P><UL><LI>Tomorrow we can test if our transfection constructs work tomorrow and we can transfect the (?) cells with CHR2 to characterize and see if it works</LI><LI>We can also make pENTR5-CMV glycerol stocks</LI></UL><P><SPAN style="text-decoration: underline;">Bacterial Cell</SPAN>:</P><UL><LI>Lux cells that didn’t glow were not a result of the “destroyed” operon, they should be fine if we continue to grow them but under different conditions (the Cambridge iGEM team suggests that the cells be grown at 30C)</LI><LI>Can we put CHR2 in bacteria (especially because ours is human optimized)</LI><LI>Gaussia and manually inserted coelenterazine is the fastest way to show that this project is feasible.  –Arielle Clynes</LI><LI>We want to figure out what types of control handles that we have: need to figure light/time scale by characterizing gaussia and the coelenterazine that we have and what concentrations of each that we need.
</LI><LI>We haven’t done anything with the biobrick aequorin; we only have about 4ng of it.  It is worth PCRing.</LI><LI>What is our end goal with the bacteria?</LI><LI>They can be the sender cells</LI><LI>Good to compare the intensity of light given from the lux brick vs. what we are using
</LI><LI>Where the bacterial aequorin fits in: another sender cell option</LI><LI>Just try this construct to see if it expresses better</LI></UL><P><SPAN style="text-decoration: underline;">Microbiology</SPAN>:</P><UL><LI>We still need to design the primers for making iGaussia</LI></UL><P><SPAN style="text-decoration: underline;">Miscellaneous</SPAN>:</P><UL><LI>Take a look at the team wiki; update your information and photo</LI><LI>We need to make sure that our protocols are all organized in the way that we are going to carry out processes (ie: transformation, miniprep, inoculation, etc)</LI><LI>We should be making work flows for these
</LI><LI>When performing minipreps, always make sure that you tilt the snap cap tubes to allow for more aeration. This will increase the yield of your miniprep</LI><LI>We are going to begin using the bubbl.us website as a way to prioritize what we are doing.  As things are accomplished, change the color, date, and initial.  This will allow us to determine how long it takes things.  Ultimately, we would like to add in sequencing to the flow chart, we want to have protocols attached to each of the bubbles as needed, and we want to include side notes like making glycerol stocks for particular substances so that we don’t forget</LI><LI>Add bacterial sender sell as an option on the flow chart
</LI><LI>Once we get the sequence for the pRRL vector, we can begin designing primers</LI><LI>Dan might have stock primers for inserting components into the vector</LI><LI>We should be able to use the same primers for all of the things sequenced with pRRL vector because the primers rely on the backbone</LI><LI>700-800 bases can be sequenced at once (&lt;1300 means that you can just use one forward and one reverse primer)
</LI><LI>we have to go to airgas.com to buy CO2 tanks for the incubator (coming in next week)</LI><LI>a pre-freshman program wants us to talk for about 2 hours about synthetic biology to a group of 15 students who are predominantly first generation and have limited background in math/science</LI><LI>each week we should create a to-do list based on the priority constructs</LI></UL><P>CZ: is still powder.  It is in the Chen Lab. –Arielle Clynes</P><P>PCZ: it is currently in pENTR5-dTOPO. We are going to gateway it together tomorrow so that we have pRRL-PCZ.  This hasn’t been sequenced yet but we have run it through a gel and it is likely that right construct.  Make sure that you sequence this before you transfect!</P><P> </P><H1>7/27/11</H1><UL><LI>miniprepped pRRL and pENTR5-CMV. pENTR5: 231.5ng/uL, pRRL:561.6ng/uL.</LI><LI>Made glycerol stocks of pENTR5 and pRRL.</LI><LI>Inoculated ChR2- glycerol stock tomorrow</LI><LI>Miniprepped pENTR-aeq  and pENTR-chr2
</LI><LI>Nanodropped and proceeded with Gateway</LI></UL><LI>Inoculated colony of MDS cells with pRRL-CMV-PCZ in the morning into 2 mL LB + 4 uL Amp (at 50ng/ul), and re-inoculated (3 ml LB, 6 ul Amp, 3 ul morning culture)</LI><LI>Diluted 0.25% trypsin to 0.50% trypsin in sterile PBS</LI><LI>Retransformed LuxBrick</LI><LI>Plated, inoculated, and made glycerol stock of ChR2.</LI><LI>Performed PCR of aequorin</LI><LI>Received primers for CatCh and performed mutagenesis</LI><LI>Made sterile water 250mL aliquots</LI><P> </P><H1>7/28/11</H1><UL><LI>Restriction Digest. pRRL + pENTR5-CMV using EcoRV, and EcoRI.</LI><LI>Miniprepped and nanodropped pRRL-CMV-PCZ-mODC</LI><LI>Stopped gateway reaction of aequorin and ChR2 with proteinase K
</LI><LI>Transformed MDS cells with amp resistance and plated</LI></UL><P> </P><H1>7/29/11</H1><UL><LI>9AM, inoculate/streak pENTR5-CMV + pRRL</LI><LI>PCR amplified aeq+ChR2 out of vectors</LI><LI>Transfected HEK w/ pcDNA3.1-ChR2</LI><LI>Gateway PCZ into pRRL</LI><LI>put Aeq+ChR2 into pENTR-dTOPO</LI><LI>Designed primers for sequencing aequorin and ChR2 in pRRL-DEST vector</LI><LI>Inoculated one colony of MDS cells with pRRL-CMV-Aeq  and pRRL-CMV-ChR2  in the morning into 2 mL LB + 4 uL Amp (at 50ng/ul) , and re-inoculate (3 ml LB, 6 ul Amp, 3 ul morning culture)</LI></UL><P> </P><H1>7/30/11</H1><UL><LI>Miniprepped aequorin and ChR2
</LI><LI>Ran gel to confirm correct size of pRRL-CMV-Aeq and ChR2</LI></UL><LI>Transfected pRRL-CMV-Aeq and pRRL-CMV-ChR2
</LI><LI>Put PCZ-mODC into pENTR/d-Topo</LI><LI>Performed first restriction digest of pENTR/d-Topo-pcz-mODC*Didn’t work, may have been a problem with the DNA ladder which was left out at 4 degrees.  PCZ-mODC should be 846bp and pENTR/d-TOPO should be 2850bp (the pCZ-mODC band doesn’t match).  Decided to go ahead with gateway reaction anyways.</LI><UL><LI>First Gateway reaction: combined 2ul of pRRL-CMV and 2ul of pENTR/d-TOPO-pCZ-mODC in gateway reaction to yield pRRL-CMV-pCZ-mODC</LI></UL><P> </P><H1>7/31/11</H1><UL><LI>Stopped gateway reaction.  Transformed gateway product.</LI></UL><P> </P><TABLE border="1" cellspacing="0" cellpadding="5"><TBODY><TR><TD valign="top" width="197">Transformation</TD><TD valign="top" width="197">Positive Control</TD><TD valign="top" width="197">Negative Control</TD></TR><TR><TD valign="top" width="197">50ul MDS cells with 2ul of pRRL-CMV-pCZ-mODC</TD><TD valign="top" width="197">25ul MDS cells with 2ul pRRL-CMV-eGFP-mODC</TD><TD valign="top" width="197">25ul MDS cells with 2 ul water (No DNA)</TD></TR></TBODY></TABLE><H1><SPAN style="text-decoration: underline;"><STRONG>August 2011</STRONG></SPAN></H1><P><A href="#Top">Back to Top</A></P><H1>8/1/11</H1><UL><LI>Ordered ampicillin plates</LI><LI>Transferred -20 iGEM box from Chen Lab to BE Undergrad Lab</LI><LI>Performed PCR of aequorin
</LI><LI>Nanodropped and ran a gel</LI><LI>No PCZ colonies grew on any of the plates.  Extra LB + Amp (3ml and 3ul) were added to transformed cells and shook in incubator overnight.</LI></UL><P> </P><H1>8/2/11</H1><UL><LI>No PCZ colonies grew then either.  This may have been a result of MDS cells that lost competency during aliquotting and temporary storage in the -20C.</LI></UL><P> </P><P><STRONG>Lab Meeting</STRONG></P><P><STRONG>What has been accomplished in terms of the big picture?</STRONG></P><P>1.     Transfection of the channelrhodopsin worked in the pcDNA that we ordered.  We could use that, but we are trying to put it under pENTR-TOPO to allow us to have more flexibility instead of leaving it under the CMV promoter.  Dan says that it could be toxic to the cell which is why we want to have more control of transmission.  Dr. Sarkar suggests that we have a higher signal that is less stable (and cells that may live a shorter span) because our goal thus far is just proof-of-concept.  We already have a working receiver component, so use it! Images of the cells were made in the Chen Lab under TE microscope, this is the proof that transfection was successful.</P><P><STRONG>Next steps: image analysis of these pictures and ask for confocal time</STRONG></P><P>2.     CatCh mutagenesis didn’t work a couple of weeks ago.  We ran it with our outer primers, we got many unspecific bands (that were very weak).  We tried a restriction digest with the original plasmid (5’ HindIII site and 3’ <STRONG>Xbal(?)</STRONG> site). It’s possible that they gave us the wrong plasmid from addgene because our primers should’ve worked.  We plan to send it for sequencing once it’s under pENTR-dTOPO.  Though we could sequence the pcDNA directly (and then complain to addgene if the plasmid is wrong).  Using Dan’s outer primers, the lengths seem ok.  Today we ran the restriction digest and a gel, product was found between 1500-2000bp.  We tried gel extraction purification with Qiagen kit and ran a nanodrop on the product but the graph was off because the spectra is changed by the gel contamination.  It is worth moving forward with it.</P><P>3.     The point of doing a double digest is to save time, but we’re able to run one single digest, purify the DNA, and run another digest to see if the problem was that the enzymes are not compatible, to ensure that we have the correct sites in the DNA, or perhaps the site is ecluded in the supercoil.  We could try this on the pENTR/D-TOPO-pcz-mODC to figure out what’s wrong. *Make sure that we always check NEB on the doubledigest finder to determine if it’s ok to use them together.  Also check to see that the enzymes have the same optimal temperature.</P><P>4.     For the gateway of pENTR/D-TOPO-pcz-mODC and pENTR/D-TOPO-CMV: The recA cells were left out for 30 minutes during aliquotting.  We should’ve been aliquotting them in the cold room and use liquid nitrogen to flash freeze them before putting them into the -80C.  This likely explains why the gateway reaction of pENTR/D-TOPO-CMV and pENTR/D-TOPO-PCZ-mODC into pRRL didn’t work.  The cells must remain in temperatures below -50C to remain competent.  Since they were left in the -20C overnight so that we could perform the transformation over the weekend (we don’t have access to the -80), they likely lost competency.  After gatewaying and transformation, nothing grew on the positive control, the negative control, or the actual gateway reaction.</P><P>5.     We can try to salvage the MDS cells (restore their competency) by adding calcium chloride.  You do an overnight culture at 30C and then grow them up at 20-23C because growing them up slowly makes them really competent.  Essentially, you repeat the process of spinning the cells down and resuspending in calcium chloride.  Dr. Sarkar has never done it with this particular strain of cells, but he has done it with others and it has been successful.</P><P>6.     Another option would be trying to clone directly without using the gateway procedure (but this would alter a lot of what we’re doing). Dan is (possibly?) going to give us an aliquot of HB101 competent cells so that we can try it in parallel to see if it works</P><P><STRONG>Next step: gel restriction digest of PCZ, and if this is successful, continue with HB101 and MDSrecA</STRONG></P><P>7.     Someone needs to begin working on the iGaussia.  We need to remove the N-terminus secretion signal.</P><P><STRONG>Next step: find secretion signal and design/order primers to cut this part off. Is Sarkar giving us the vector for this? </STRONG></P><P>8.     We should transfect cells with the PCZ-mODC vector and then lyse the cells to access the coelenterazine, and combine this with the gaussia supernatant to see if it glows.</P><P>9.     We should characterize the channel rhodopsin</P><P>10.  If PCZ doesn’t work, but add CZ to all of the media, the project doesn’t fall apart and this way we are able to characterize/induce expression of aequorin in the sender cell.</P><P>11.  The output from the receiver cell can be anything if we’re not going to daisy chain the sending/receiving cells together.  The sender cell could have gaussia which is good because it has higher expression.</P><P>12.  *Potential problem: if PCZ doesn’t work, is our project that unique?  PCZ allows us to combine cell-cell signaling with genetic encoding, and PCZ is the genetic encoding component.</P><P>13.  Most of the people have been focusing on the mammalian constructs; but it would be good for us to continue working on the bacterial sender cell as well.  Cambridge team has been contacted about the lux biobrick and how best to transform/get the cells to glow, and the aequorin still has yet to be PCRed.</P><P><STRONG>Next step: design primers to amplify aequorin with restriction sites for putting into pET3a</STRONG></P><P>14.  It is worth it to continue moving forward with the aequorin because PCZ was originally patented in E. coli.  The rationale for using it in mammalian cells is that the mODC is what will target the proteins to degradation, but the proteases will only break down the outside, releasing the CZ from its beta-barrel (<STRONG>someone who knows more about this please add details)</STRONG></P><P>15.  We could think about getting fura or indo dyes that are calcium sensitive so that we can use these dyes to have the cells light up differently.</P><P><STRONG>Next step: buy dyes</STRONG></P><P>16.  Arielle is on the luminometer experiments and the coelenterazine resuspension.</P><P>17.  BBC documentary due date is this Friday! In our pitch, what should we say that we expect for them to see when they come?  Don’t over-advertise, just say the vision. Don’t necessarily say that it isn’t working. Things that we need to do to make this successful: update wiki, rent video equipment from van pelt and record ourselves, talk to someone in seas to see if someone could help up with webdesign. (avin is going to email joe sun)</P><P>18.  We don’t need to worry about having a UV bulb, but we should keep the hoods running 24/7.</P><P><STRONG>Next step: have Sarkar talk to Seville abut it</STRONG></P><P>Upcoming Goals:</P><P>1.     Characterize PCZ lysate in gaussia supernatant and aequorin lysate with CZ  by themselves and make sure that you can get light.</P><P> </P><H1>8/3/11</H1><UL><LI>Gateway reaction for pRRL-CMV-PCZ-mODC
</LI><LI>Second restriction digest of pENTR/d-TOPO-pCZ-mODC</LI><LI>Didn’t work again. The bands are ~3000bp, when the pCZ band should be 846bp. Note: there is a tiny band at the bottom around 1000bp.  It’s possible that the darkest band is only cut in one site, while the other two bands correspond to pENTR/d-TOPO and pCZ-mODC</LI></UL><P> </P><H1>8/4/11</H1><UL><LI>Stopped Gateway reaction for pRRL-CMV-PCZ-mODC</LI><LI>Transformed into HB101 cells, plated onto Amp plates</LI></UL><P> </P><H1>08/05/11</H1><UL><LI>Third restriction digest of pENTR/d-TOPO-pCZ-mODC</LI></UL><UL><LI>This gel was run at 200V but was left for too long. We could not determine whether or not there were two bands in the double restriction digest. So we ran another gel.</LI></UL><P> </P><H1>8/6/11</H1><UL><LI>Ran a NotI restriction enzyme digest to confirm the identity of any pRRL constructs. These are the sizes we should expect to see:
5.67 kb, vector
1.7kb, vector + CMV promoter
? kb, size of your cDNA of interest + 50bp (this could be aequorin, PCZ, or ChR).</LI></UL><UL><LI>Fourth restriction digest of pENTR/d-TOPO-pCZ-mODC</LI></UL><UL><LI>Success! The individual restriction digests yield one band between 3000bp-4000bp, and the combined bp of pENTR/d-TOPO and pCZ-mODC is 3696bp.</LI><LI>The double restriction digest yields two bands, one between 2500bp-3000bp (pENTR/d-TOPO which is 2850bp) and another around 900bp (pCZ-mODC)</LI></UL><P> </P><H1>8/8/11</H1><UL><LI>Stopped gateway reaction and transformed gateway product. The pRRL-CMV-PCZ-mODC transformation failed</LI></UL><TABLE border="1" cellspacing="0" cellpadding="5"><TBODY><TR><TD valign="top" width="197">Transformation</TD><TD valign="top" width="197">Positive Control</TD><TD valign="top" width="197">Negative Control</TD></TR><TR><TD valign="top" width="197">•50ul MDS cells (from PGFI, which were stored at -80) with 2ul of pRRL-CMV-pCZ-mODC•25ul HB101 cells with 2ul of pRRL-CMV-pCZ-mODC•Plated 50 and 100 ul of both transformations<P> </P></TD><TD valign="top" width="197">25ul of MDS with 2ul pRRL-CMV-eGFP-mODC (~1.526 ng/ul) Plated 100 ul</TD><TD valign="top" width="197">25ul MDS with 2ul water (no DNA) Plated 100 ul</TD></TR></TBODY></TABLE><UL><LI>Spun down the untransformed cells, resuspended, and plated again onto Amp plates. None of these grew.</LI><LI>Transformed the leftover Gateway into HB101 and the diluted Gateway into MDS. No colonies grew on any plates.</LI></UL><P> </P><H1>8/9/11</H1><UL><LI>Results:</LI></UL><P>1. MDS had 2 &amp; 5 colonies</P><P>2. HB101 had 81 and 69 colonies on each plate</P><P>3. Positive and negative control had nothing</P><UL><LI>Troubleshooting:
</LI><LI>Plated the rest of the transformed cells</LI><LI>Used 2ul of the leftover Gateway reaction to transform HB101</LI><LI>Took .2ul of leftover Gateway reaction, added 1.8ul of Milli-Q, and used it to transform 50ul MDS (1:10 dilution of the Gateway)</LI></UL><UL><LI>ALWAYS
</LI><LI>save a little of your inoculation before miniprepping. This way, you’ll be able to grow up more of the DNA that you have after it has been sequenced and verified</LI><LI>We should always use controls in our experiments</LI><LI>For example,  double digests, we can run one enzyme in each lane, no enzyme in one lane, and both enzymes in another</LI><LI>More importantly, when running transformations, always run positive and negative controls</LI></UL><UL><LI>CONTROLS FOR GATEWAY:</LI><LI>Use the aequorin because we know that it has worked with aequorin.  IF all of the aequorin plates work when running the gateway and transformations of PCZ and aequorin together, and the PCZ plates don’t, then we know that we have to work on the pENTR-dTOPO-PCZ-mODC</LI><LI>Plate controls (transform w/ pUC19 and water)</LI><LI> We need to start up again using a new colony containing pENTR-dTOPO-PCZ-mODC, make a glycerol stock, miniprep, etc.
</LI><LI>In addition, we can also be retransforming using the DNA that we already isolated and then growing up cells</LI></UL><LI>We had 2 different stocks of the MDS, we should call scarab genomics and complain to tell them that we followed their protocol and did it with HB101 on the side and had success with HB101 and no with the MDS.  See if we can get the money back.</LI><LI>CHARACTERIZING aequorin and CHR2 together:</LI><LI>We have to shine light, add coelenterazine, and calcium.</LI><UL><LI>HOW TO CHARACTERIZE EACH PIECE:</LI><LI>you have to use a high powered microscope, which is beyond the scope of what we are doing?</LI><LI>CHR2: use fura dye, these compounds act as chelators, they aggregate around calcium ions which increases their fluorescence (they’re only very fluorescent in agreggation with other dye molecules attached to a calcium)</LI><LI>Using a confocal microscope, you can see inside the cell.  This makes sure that you won’t assume that light emitted by the outside of the cell itslf is what you’re seeing, but that the characterization is actually accurate</LI><LI>These dyes have a background fluorescence, but you want to do ratio imaging (compare the fluorescence before pulsing with the laser to the fluorescence after hitting the cells with the laser) this method takes 3 months of training on expensive equipment</LI><LI>CHR2 alternate option: we already have characterized it as much as we can</LI><LI>Aequorin: transfect cells with aequorin, wait 24 hours, but we don’t know the maturation time of the aequorin protein (its structure? would compare well with the gaussia), and since we saw with the gaussia? that 24 hours was good, this should work</LI></UL><P>Characterizing CMV-Gaussia</P><P>Luminometer Experimental Setup</P><UL><LI>1.              We have cells constitutively expressing Gaussia in solution</LI><LI>2.              Injection: ____ of Gaussia supernatant (A) and ___ of Cz solution (B)</LI><LI>3.              Check for luminescence</LI><LI>1.                                If luminescence occurs, can characterize the luminescence by adding coelenterazine solution in a gradient to find optimal substrate concentration</LI></UL><P>Controls</P><TABLE border="1" cellspacing="0" cellpadding="5"><TBODY><TR><TD valign="top" width="84">Type</TD><TD valign="top" width="184">A</TD><TD valign="top" width="162">B</TD><TD valign="top" width="167">Controls for?</TD></TR><TR><TD valign="top" width="84">Negative</TD><TD valign="top" width="184">Gaussia supernatant</TD><TD valign="top" width="162">Acidified methanol solution w/o Cz</TD><TD valign="top" width="167">Cz is needed for luminescence</TD></TR><TR><TD valign="top" width="84">Negative</TD><TD valign="top" width="184">Supernatant from cells not expressing Gaussia</TD><TD valign="top" width="162">Cz</TD><TD valign="top" width="167">Gaussia needed for luminescence</TD></TR><TR><TD valign="top" width="84">Positive</TD><TD valign="top" width="184">Gaussia</TD><TD valign="top" width="162">Dual-Luciferase Kit</TD><TD valign="top" width="167">Check if the Gaussia is good</TD></TR></TBODY></TABLE><P>Characterizing CMV-Aeq</P><P>Luminometer Experimental Setup</P><UL><LI>1.              Get cells to express Aeq</LI><LI>2.              Add Cz to cell solution</LI><LI>3.              Wash many times in PBS (w/o Ca2+)</LI><LI>4.              Lyse cells, save the lysate</LI><LI>5.              Injection: ___ of Lysate (A) and ___ of Ca2+ solution (B)</LI></UL><P>Controls</P><TABLE border="1" cellspacing="0" cellpadding="5"><TBODY><TR><TD valign="top" width="85">Type</TD><TD valign="top" width="240">A</TD><TD valign="top" width="113">B</TD><TD valign="top" width="159">Controls for?</TD></TR><TR><TD valign="top" width="85">Negative</TD><TD valign="top" width="240">Lysate</TD><TD valign="top" width="113">PBS</TD><TD valign="top" width="159">Calcium required</TD></TR><TR><TD valign="top" width="85">Negative</TD><TD valign="top" width="240">PBS</TD><TD valign="top" width="113">Ca2+</TD><TD valign="top" width="159">Lysate Required</TD></TR><TR><TD valign="top" width="85">Negative</TD><TD valign="top" width="240">Lysate from WT HEK293T cells</TD><TD valign="top" width="113">Ca2+</TD><TD valign="top" width="159">Aeq required</TD></TR><TR><TD valign="top" width="85">Negative</TD><TD valign="top" width="240">Lysate from HEK293T cells expressing Aeq with no Cz added</TD><TD valign="top" width="113">Ca2_</TD><TD valign="top" width="159">Cz required</TD></TR><TR><TD valign="top" width="85">Positive</TD><TD valign="top" width="240">Gaussia supernatant</TD><TD valign="top" width="113">Cz being used</TD><TD valign="top" width="159">Cz works</TD></TR><TR><TD valign="top" width="85">Positive</TD><TD valign="top" width="240">Gaussia supernatant</TD><TD valign="top" width="113">Dual luciferase kit</TD><TD valign="top" width="159">Gaussia good (controls the control)</TD></TR></TBODY></TABLE><P>*If the CZ works with the aequorin, the next step will be testing the PCZ with the aequorin.</P><UL><LI>Ken Chen has a plate reader, but we don’t know if it is a fluorescent plate reader (this would be useful for the characterization of the transfections with aequorin and CHR2)</LI><LI>Shine light at 475nm and detect at what wavelength?! We want to use a plate reader that will allow us to account for timing because if we’re shining light and detecting the same wavelength that we’re shining, we won’t be finding real data?</LI><LI>We want to shine the light, turn the light off, then measure</LI><LI>What if we: add coelenterazine, some is taken up by the cell, add gaussia to the solution (and it’s a big protein that can’t get into the cell), so it will take all of the extracellular coelenterazine and use that to make light, this light will activate the CHR2 which will take up the calcium, then (some of the coelenterazine is assumed to be taken up by the cell), and aequorin will use the coelenterazine and calcium to emit another light.  All of these wavelengths are around 480nm.</LI><LI>So we should see the background light up, and hopefully a little bit later we will see the cells themselves light up</LI><LI>Things we need to think about: how long each of them light up for (there is information in the kinetics of each of these components, along with their half lives, in the email that Jordan sent us)</LI><LI>How long did the luminometer measure the gaussia: 10 seconds (that’s how long the light shines) *and it may be that the camera is not sensitive enough..if we can use a higher objective, we will be able to better detect</LI><LI>Take an image, turn the camera off, then turn it back on after everything has equilibrated.  We have to use the light to find the cells? Turn off the light for 10 minutes to let everything reequilibrate/go back to normal, then add the gaussia and begin running the camera</LI><LI>*Other idea; look at cells initially using red light so that nothing is activated/stimulated, then when actually ready to record/measure/add gaussia, use blue light (480nm).  Use a filter on the microscope light.  If this doesn’t work that well then you have a justification for using the CatCH mutagenesis</LI><LI>We need cells with CHR2 and aequorin for the transfection</LI><LI>Concern: gaussia light is so much longer lasting and stronger that it might block out the light that is emitted by the cells. We will have to play with the stoichiometry for the cotransfection to try to figure out the best way for showing the second glowing of the cells after the CHR2 have taken up the calcium.  A perfect cotransfection is not possible the first time, but we can at least try to find thresholds and work from there</LI><LI>Coelenterazine needs to be characterized TOMORROW. We need to do better with planning because Jordan is very busy and it is difficult for him to leave what he is doing to help us.</LI><LI>@2:45PM Jordan and Mike are going to work together on characterization using the microscope in Chen Lab.</LI><LI>The most accurate way to count the # of cells that you have is to image cells on a microscope, and count the # of cells in a region of the plate, and magnify this by the total area of the plate. This is better than the hemocytometer?</LI><LI>Original calculations can be used to make approximate additions of media etc. based on the confluency of the cells.</LI><LI> iGaussia is a little bit dimmer than regular gaussia, but do we know if the renilla luciferase is a lot brighter? We’re going to contact Dan to see if there is a CMV-renilla plasmid available (or see if addgene has that too)</LI><LI>some people in Chen lab are doing dual luciferase assays (we just don’t know if we need to clone it out or not)</LI></UL><P> </P><H1>8/10/11</H1><UL><LI>inoculated pRL-CMV-Renilla from Chenlab glycerol stock</LI><LI>The transformed PCZ cells did not grow.  The leftover gateway reactions that were used to transform also did not result in cell growth.</LI></UL><P> </P><H1>8/11/11</H1><UL><LI>Mini-prepped and nanodropped</LI><LI>355.7 ng/ul HB101 pRRL-CMV-aeq</LI><LI>284. 4ng/ul HB101 pRRL-CMV-aeq</LI><LI>44.6 ng/ul MDS pRRL-CMV-aeq</LI><LI>33.3 ng/ul MDS pRRL-CMV-aeq</LI><LI>97.0 ng/ul CMV-renilla</LI><LI>339.9 ng/ul pRRL-CMV-ChR2-eYFP</LI><LI>336.0 ng/ul pRRL-CMV-ChR2 -eYFP</LI></UL><P> </P><H1>8/12/11</H1><UL><LI>1 mg/ml CZ stocks were made up with Dan’s protocol (1 mL ethanol, 20 ul 3N HCl). These are in 50 ul aliquots.</LI><LI>Extra ethanol and 3N HCl is in the iGEM box above the sink.</LI></UL><P> </P><H1>8/16/11</H1><P><STRONG>LAB MEETING</STRONG></P><UL><LI>PCZ
</LI><LI>still waiting on sequencing account, use Sarkar’s account instead</LI><LI>We only have it in pENTR/d-Topo-PCZ-mODC</LI></UL><LI>Aequorin
</LI><LI>In pRRL-CMV-Aequorin</LI><LI>Transfected cells, lysed, added CZ, saw luminescence on the luminometer (yay!)</LI><LI>Cells were not seeded well (large clumps) → be more careful with tissue culture</LI><LI>Is this good enough to say we’ve characterized aequorin? (can we can submit our pRRL-cmv-aeq as a biobrick)
</LI><LI>best to quantify relative light units per mole of aequorin</LI><LI>future: buy aequorin, lyse untransfected cells, and add known amounts of aequorin (buy it) to back out the amount of aequorin (kind of like a BCA protein assay)</LI><LI>are we lower than Chenlab’s luminometer experiments?</LI><LI>compare Gaussia papers</LI><LI>need to repeat experiment for more data</LI><LI>ready to make lentivirus
</LI><LI>someone needs to contact Jordan about that</LI><LI>will it actually help in maxing out signal?</LI><UL><LI>ChR2
</LI><LI>Cotransfection with aequorin</LI><LI>added CZ, Ca and Gaussia in hopes that Gaussia would activate ChR2 (pcDNA3.1) and we would be able to see luminscence by eye under microscope. We didn’t see anything</LI><LI>May be hard to manipulate in luminometer (ChR2 is activated by white light)</LI></UL><LI>Characterize ChR2:
</LI><LI>Indo dyes with flow cytometry</LI><LI>the lab across from Sarkar’s has this machinery (Chocus) for ratiometric imaging with indo and fura dyes</LI><LI>cell permeable Indo dyes, add hydrolyzable esthers, etherases in cells cleave the esthers to prevent it from leaving the cell</LI><LI>look at Dan’s email on drugs for calcium</LI><LI>talk to flow cytometry core (calcium imaging). Penn has a great core!</LI><LI>how long does Ca stay in the cell so that we can image it?</LI><LI>will be hard to do by flow: decay is quick</LI><LI>samples will need to be in foil so they are not exposed to light</LI><LI>then you’ll shine light and it will sort cells</LI><LI>We now have this in pRRL-CMV-ChR2
</LI><LI>can be sequenced (need to prepare!!)</LI><LI>ready to make lentivirus</LI><LI>someone needs to contact Jordan about that</LI><LI>Ordered primers for Quikchange PCR (mutate into CaTcH)
</LI><LI>needs to be picked up from cell center</LI><LI>Mike knows about this</LI><UL><LI>iGaussia
</LI><LI>decided to add KDEL sequence and restriction sites</LI><LI>Ty and Mike know about this</LI><LI>bacterial aequorin</LI><LI>Somebody needs to get on this</LI><LI>sequence?</LI></UL><LI>Renilla luciferase
</LI><LI>We have it in a transient vector from Chenlab</LI><LI>Need to verify what vector it is</LI><LI>Is it worth putting into pRRL?</LI><LI>Lux
</LI><LI>Has anyone received an email from Cambridge about this? Probably better to email the advisors rather than students.. more responsive</LI><LI>cells per well? Are those wells spun down?</LI><LI>how stable is the arabinose signal?</LI><LI>Cambridge team
</LI><LI>need to do pos control, negative control, not onto arabinose plates</LI><LI>they used TOP10</LI><UL><LI><STRONG>MATERIALS</STRONG></LI><LI>Has anyone asked for these items from Invitrogen:
</LI><LI>Gateway clonase/proteinase K, dTOPO, Fura dyes, Indo dyes, 0.05% trypsin, pen-strep, cell culture media, FBS</LI></UL><LI>MDS cells
</LI><LI>Scarab genomics should be shipping another 1 mL</LI><LI>previous transformations had very low yields</LI><LI>What’s surprising is that nanodrop yields were also significantly lower</LI><LI>optimal time to incubate for miniprep?</LI><LI>HB101
</LI><LI>should we buy some? We’re using ChenLab’s at the moment</LI><LI>Bioluminescence imager/confocal
</LI><LI>Does imaging work in our wavelength (call company)</LI><LI>email Julie for donation time</LI><LI>Should we ask for time on confocal?</LI><LI>did Mike run HEK cells through the guava?</LI><P> </P><P> </P><H1>8/17/11</H1><UL><LI>Look into indo, fura, and fluo dyes to use with the Tecan plate reader: <SPAN style="text-decoration: underline;">http://www.tecan.com/platform/content/element/2265/Tecan_GENios_Pro_AppNote_Photina.pdf</SPAN>, ask Penn’s flow cytometry core for advice on how to image the ChR2 (and if flow will work if we cover everything with aluminum foil and stimulate a layer of cells with light), look into ratiometric imaging (lab across from Sarkar’s may be willing to do it for us, we only need to run ChR2 and possibly CatCh), look into ionomycin and thapsigargin</LI><LI>Contact Cambridge team (try both the students and the advisors, just CC everyone in one email) about Lux brick: How many cells did they seed per 96 well plate? Were those wells spun down? How stable is the arabinose (do you have to keep adding arabinose to maintain a stable signal, and how long does this last, how much arabinose did they add?) Were these grown on Ara plates? Is it okay to use DH5a and not TOP10. Concentration of chloramphenicol? Incubation time? Basically we need all the growth conditions because it’s not working for us.</LI><LI>Set up CO2 incubator (order missing connector to CO2 tank). We are missing a piece that has a 7 or 10 serration hose outlet with a male inlet.</LI></UL><P>Pick up primers from Cell center</P><H1>8/19/11</H1><UL><LI>Mini-prepped DNA (renilla, aeq, and chr2) for transfection.</LI><LI>Design primers for Quikchange PCR of ChR2 into CatCh</LI><LI>Design primers for iGaussia primers</LI><LI>How long does Ca stay in the cell for imaging purposes? Find a paper and send it out.</LI><LI>transfected HEKs in a 6 well plate 8PM, they looked 95% confluent at time of transfection. Followed NEB’s D2 transpass protocol. The transfections are as follows: ChR2, Aeq, Renilla, ChR2+Aeq, ChR2+Aeq, ChR2+Aeq</LI><LI>Prepared ChR2 for sequencing</LI><LI>Contacted Scarab Genomics for optimal time to incubate MDS before miniprep</LI><LI>Prepared and sent plasmids for sequencing;</LI><LI>pRRL-CMV-ChR2-eYFP</LI><LI>pENTR/d-Topo-PCZ-mODC</LI><LI>pRRL-CMV-Aequorin</LI><LI>pRL-CMV-Renilla</LI><LI>Inoculated renilla, aeq, chr2 from glycerol stocks in three 1mL cultures and let it grow for 4 hours</LI><LI>took 10uL from each 1mL culture and inoculated into a 50mL LB in a 250mL shake flask and let it grow for 17.5 hours.</LI></UL><H1>8/21/11</H1><UL><LI>transfected HEKs in a 6 well plate 8PM, they looked 95% confluent at time of transfection. Followed NEB’s D2 transpass protocol. The transfections are as follows: ChR2, Aeq, Renilla, ChR2+Aeq, ChR2+Aeq, ChR2+Aeq</LI><LI>Prepared ChR2 for sequencing</LI><LI>Contacted Scarab Genomics for optimal time to incubate MDS before miniprep</LI><LI>Prepared and sent plasmids for sequencing;</LI><LI>pRRL-CMV-ChR2-eYFP</LI><LI>pENTR/d-Topo-PCZ-mODC</LI><LI>pRRL-CMV-Aequorin</LI><LI>pRL-CMV-Renilla</LI><LI>Inoculated renilla, aeq, chr2 from glycerol stocks in three 1mL cultures and let it grow for 4 hours</LI><LI>took 10uL from each 1mL culture and inoculated into a 50mL LB in a 250mL shake flask and let it grow for 17.5 hours.</LI></UL><H1>8/22/11</H1><UL><LI>Prepared and sent plasmids for sequencing;</LI><LI>pRRL-CMV-ChR2-eYFP</LI><LI>pENTR/d-Topo-PCZ-mODC</LI><LI>pRRL-CMV-Aequorin</LI><LI>pRL-CMV-Renilla</LI><LI>Inoculated renilla, aeq, chr2 from glycerol stocks in three 1mL cultures and let it grow for 4 hours</LI><LI>took 10uL from each 1mL culture and inoculated into a 50mL LB in a 250mL shake flask and let it grow for 17.5 hours.</LI></UL><P> </P><H1>8/25/11</H1><UL><LI>Sequencing results:
</LI><LI>pRRL-CMV-ChR2-eYFP aligns well with the sequence AddGene gives for this plasmid.</LI><LI>pENTR/d-Topo-PCZ-mODC aligns with the PCZ insert. Our next step is to try the Gateway reaction again</LI></UL><UL><LI>Aequorin aligns with our insert.</LI><LI>Renilla aligns with the pRL-CMV vector from promega. The alignment is from bp 23- end of the Renilla we sequenced. Should we assume our vector is pRL then? I sequenced it with the CMV sense primer.</LI><LI><SPAN style="text-decoration: underline;">http://www.ncbi.nlm.nih.gov/nuccore/AF025843</SPAN></LI></UL><P> </P><H1>8/31/11</H1><UL><LI>Luminometer experiments:</LI><LI>add gaussia supernatant + PBS – negative control</LI><LI>add gaussia supernatant + CZ    - positive control to see if 1) gluc works, 2) CZ works, 3) luminometer works</LI><LI>add gaussia supernatant + PCZ cell pellet lysate (100mL)</LI><LI>add gaussia supernatant + PCZ supernatant after spindown</LI><LI>add gaussia supernatant + mCherry cell pellet lysate</LI><LI>add gaussia supernatant + mCherry supernatant after spindown</LI></UL><H1><SPAN style="text-decoration: underline;"><STRONG>September 2011</STRONG></SPAN></H1><P><A href="#Top">Back to Top</A></P><H1>9/13/11</H1><UL><LI>Transfected Cardiacmyocytes (Primary Cells) with Aequorin using lipofectamine 2000.</LI><LI>Next step: observe whether we can see real time calcium influx with luminescent reporter.</LI></UL><P> </P><H1>9/16/11</H1><UL><LI>Imaged cardiocmyocytes on coverslips (200,000 cells)</LI><LI>Added 250uL fresh media to 1uL CZ.</LI><LI>Used TiScope – saw no luminescencec</LI></UL><P> </P><H1>09/20/2011</H1><UL><LI>Calcium imaging.</LI><LI>Brought over 293T’s in a 6 well plate, which had been transfected with CatCh. Also had positive control cells on which we used ionophores. Used the Meaney Lab’s confocal and x-rhod1 dye. Media had 2mM Calcium.</LI><LI>1uM Ionomycin addition did not change x-rhod1 fluorescent intensity in Anthony’s neurons.</LI><LI>Also did not work on our 293T’s, but we had the problem of being unable to cleanly pipet the ionomycin into the 6 well plates because the wells we looked at with the scope became entirely blocked by the scope lens.</LI><LI>Illumination of CatCh cells with 476nm laser from confocal did not increase x-rhod1 intensity. eYFP expressing cells were identified, so transfection worked.</LI><LI>Will try again Thursday (9/22) and Friday (9/23) with larger plates, and possibly switch to FURA-2 and Avin’s laser if the confocal still doesn’t work.</LI><LI>Anthony is going to try further with x-rhod1 dye in neurons to make sure that the dye/laser parameters aren’t the problem. Will also attempt the co-culture experiment on these dates.</LI></UL><P> </P><H1>9/22/11</H1><UL><LI>Ran a gel on PCR products (humanized aequorin)</LI><LI>Gel Isolation/Purification</LI><LI>Ligation into backbone</LI><LI>Transformation into DH5a</LI></UL><P> </P><H1>9/27/11</H1><UL><LI>Fura-2 imaging</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><DIV id="footer-top"><DIV id="footer-content"><DIV id="footer-bottom" class="clearfix"><DIV class="container"><UL id="menu-main" class="bottom-nav"><LI class="menu-item menu-item-type-custom menu-item-object-custom menu-item-home menu-item-62"><A href="http://benjaminshyong.com/igem2011/">Home</A></LI><LI class="menu-item menu-item-type-custom menu-item-object-custom menu-item-217"><A href="http://benjaminshyong.com/igem2011/project/overview/">Project</A></LI><LI class="menu-item menu-item-type-post_type menu-item-object-page menu-item-75"><A href="http://benjaminshyong.com/igem2011/results/">Results</A></LI><LI class="menu-item menu-item-type-post_type menu-item-object-page menu-item-67"><A href="http://benjaminshyong.com/igem2011/biobrick/">Biobrick</A></LI><LI class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-27 current_page_item menu-item-68"><A href="http://benjaminshyong.com/igem2011/notebook/">Notebook</A></LI><LI class="menu-item menu-item-type-post_type menu-item-object-page menu-item-76"><A href="http://benjaminshyong.com/igem2011/safety-and-society/">Human Practices</A></LI><LI class="menu-item menu-item-type-post_type menu-item-object-page menu-item-78"><A href="http://benjaminshyong.com/igem2011/safety/">Safety</A></LI><LI class="menu-item menu-item-type-post_type menu-item-object-page menu-item-63"><A href="http://benjaminshyong.com/igem2011/about-us/">About Us</A></LI><LI class="menu-item menu-item-type-custom menu-item-object-custom menu-item-87"><A href="https://2011.igem.org/Main_Page">iGEM</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Penn/notebook/">http://2011.igem.org/Team:Penn/notebook/</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Penn/notebook/" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Penn/notebook/" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Penn/notebook/&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Penn/notebook/&amp;oldid=203779" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>