"
Page
Discussion
View source
History
teams
Log in
 
Team:BGU Israel/Experiments
From 2013.igem.org
BGU_Israel
Home
Environment
Enviromental Problem
Consequence Analysis
The Debate
Project
Motivation
Our Solution
Bio Bricks
Achievements
Judging Forms
Notebook
Protocols
Experiments & Results
Modelling
Stochastic Model
Modelling Results
Human Practice
Overview
Survey
Outreach
Continuity
Safety
Team
Acknowledgements
Experiments & Results Read our notebook
Week 1       18.07.2013 - 24.07.2013
18.7.2013 Fluorescence over time under different IPTG conc.
1. Overnight starters were diluted. 2. Incubation for 3.5 hours and check O.D. (O.D. came out 0.6 which was too high and therefore we diluted the vials x2 and added 10 ml LB and another 10µl CRB and put back in incubation for 22 min. 3. We inserted different conc. of IPTG in the vial and samples were taken to the 96 well plate reader. Results: No bacterial growth was observed.
19.7.2013 Fluorescence over time by different conc. of ITPG
1. 3 starters that grew overnight in the incubator: (1)  GFP, CRB100, 0.5% glucose, LB (2) GFP, CRB100, LB (3) no GFP, CRB100, LB 2. diluting the starters by:
3. The vials were put in incubation for 2 hrs until O.D. was ~ 0.35 4. We inserted IPTG in different conc. in the GFP vials and 1mM in the control vials. samples were taken into the 96 well plate. The plate was inserted into the plate reader for reading FLU and O.D. in intervals of 30 min for 18 hrs.
Week 2       25.07.2013 - 31.07.2013
Note: We started the project with a different name for each approach. Eventually we came up with "P.A.S.E", but in our notebook we continued using the old ones for comfort sake: GK means P.A.S.E 1, and TB means P.A.S.E 2.
28.7.2013 Analyzing GFP decline rate post induction
1. A starter was diluted 1:50 with 10 ml LB, 10 µl CRB X 6 vials. 2. The vials were put in incubation for 2 hours until the O.D. reached  ~0.3 3. 1 mM of IPTG was inserted in each vial and the vials were returned to the incubator for 4 hours. 4. Centrifugation for 10 min, 4000 RPM of the 6 vials. 5. Separation of the fluid from the precipitation. 6. Bacterial precipitation of each vial was mixed with 10 ml LB, 0.1 ml 50% Glucose and 10 µl CRB. 7. Samples of the vials we taken to a 96 well plate and was put in the plate reader for 9 hours with 30 min intervals of O.D. and FLU examination. Results:
An average of the 6 vials has been taken and the decline rate has been calculated by a graph of the decline period only.
Conclusions: 1. Even though the medium with IPTG has been removed and replaced, the initial 2 hours showed a continued induction of IPTG on the expression of GFP. Possible explanation is that the IPTG that was in the bacteria diffused very slowly to the medium which didn’t contain any. Therefore, it is needed to take into account when calculating a specific duration of induction, the amount of time the inducer will diffuse out of the bacteria. 2. After 2 hours there was a significant decline in FLU while the bacteria still in the Log phase. An equation of the decline rate was determined. The values of FLU/O.D. here declining because of the increase of O.D. while FLU stayed constant. This explains how GFP is diluted during bacterial growth and its FLU per cell declines by the equation listed in graph 2.
29.7.2013 IPTG induction on GFP expression over time
1. Starters were diluted by the following (amounts in ml):
2. The vials were put in incubation for 2 hours until the O.D. reached  ~0.3 3. IPTG was added by the following amounts:
4. 1 ml was taken from each vial into separate 15ml and the vials returned to the incubator. 5. Centrifugation for 10 min, 4000 RPM of the 5 samples. 6. Separation of the fluid from the precipitation. 7. Wash X2 with 0.1M PB. 8. Bacterial precipitation was mixed with 1 ml PB of each sample. 9. O.D. and FLU we read from the plate reader. 10. Steps 4 thru 9 were repeated every 30 min for 4 hours. Results:
Conclusions: 1. 0.5% glucose has shown once again that it is much stronger than the induction of IPTG on the expression of GFP and therefore there was no increase in FLU. 2. While the bacteria were in their Log phase from the beginning, a significant increase of FLU has been identified only after 3 hours. Therefore the long response time of the bacteria to IPTG indicates that in order to model the rate of protein expression as a result of induction, it is needed to take into account a 3 hour delay. 3. The initial 3 hours have not shown any difference between the different concentrations of IPTG. However, once the induction has started to kick in (after 3 hrs) a gap has been formed between them when the 2 mM IPTG had the most FLU and the 0.5 mM IPTG had the least.
29.7.2013 Transformation of pkd78 to Electro-competent cells
The Electroporation standard protocol was used with the following data:
DNA volume - 1 uL.
cuvette width - 0.2 cm.
Time constant (Tao) - 5.5 msec.
Two LB agar plates with cmp were plated with bacteria - one with 20 uL of culture, the other with 50uL. The plates were grown overnight in 300c, and on both colonies grew.
31.7.2013 PCR amplification of GFP gene with lacI promoter
3 vials were made - one of them as a control vial. A mixture was made, containing:
51 uL DDW.
15 uL KAPA HIFI buffer.
2.25 uL of dNTPs.
2.25 uL of forward primer.
2.25 uL of reverse primer.
1.5 uL of KAPA polymerase.
The mixture was divided into the 3 vials. DNA template was added to 2 of the vials (not to the control) - 0.3 uL of 102 ng/ml template solution.
The annealing temperature was set to 450C.
The vials were put in the PCR for 30 cycles. We used the protocol ”Routine high-fidelity amplification of longer DNA fragments is challenging due to the low processivity and lack of robustness of most proofreading DNA polymerases”.
Week 3       01.08.2013 - 10.08.2013
1.8.2013 Recombineering protocol
In order to try and experience with the recombination system, BL21 pkd78 were made recombination – ready and transformed with a cassette containing only GFP gene and homologous recombination sites. Changes from original protocol:
Centrifuge 10 min at 4200 g and resuspend pellet in 300 μL ice cold 10% glycerol. (final Vol.  1:100)
Transformation and selection:
Add up to 100ng of the purified PCR product cassette (1 μL) to 40 μL of competent cells. 3 min on ice.
Transfer the competent cells into an electroporation cuvette.
Electroporate with Ec2 setting. #1 cuvette-> 5.8m/sec. #2 cuvette-> 5.9 m/sec.
Plate 50 μL of culture on LB plates containing the appropriate antibiotic and/or selection agent. A serial of dilution was done, we use 104 – 108 to plate on the agar.
Cuvette #1 was add to LB agar + IPTG 1mM, incubated at 37oC over night.
Cuvette #2 was add to LB agar + IPTG 1mM+CMP 50mg/ml, incubate at 37oC over night, to confirm loss of pkd78.
2.8.2013
2 samples of BL21-Psup-Acetylizing syntethase were freezed (-80oC)
4.8.2013
Goal: replacing antibiotic resistance Components:
PkD 78
pET 15b
HiFi buffer X5
DDW
dNTP's
ampR FWD & REV primer
cmpR FWD & REV primer
kappa HiFi polymerase
DNA template
Gel Agarose
Amounts:
Control sample was prepared – containing all components except the template.
Each reaction was made twice.
PCR reaction:
Agarose gel has been prepared in order to get qualitative estimation about the components after the PCR.
The samples have background noise, thus additional agarose gel has been prepared, except now it is used to separate the pkD 78  backbone & the antibiotic resistance from the pET 15b.
5.8.2013 Colony PCR
Goal: Last week Adi transformed BL21 with a GFP cassette. We want to find colonies that have taken the cassette into their chromosome. Colony PCR The cassette is a linear DNA sequence not containing an origin of replication and no antibiotic resistance. So, we chose randomly 10 colonies from the plates in which the transformed BL21 were seeded and used their DNA as template for PCR with primers for the GFP gene.
Colony preparation – pick a colony and resuspend in 5 μl DDW PCR.
Boil for 5 min at 99oC and immediately chill on ice (PCR  program: colony_prep).
Use the 5 μl as template for PCR.
Amounts:
Control sample was prepared – containing all components except the template.
10 samples were made – one from each colony.
PCR reaction:
Agarose gel has been prepared in order to get qualitative estimation about the components after the PCR:
Apparently, none of the selected colonies have not taken the cassette into the chromosome.
5.8.2013 Extraction of pkd78 bacbone and CRB gene from gel
Goal: PCR was performed by Alex on 4/08. Today we extract the DNA from the gel for further use. Extraction of pkd78 and CRB resistance gene from agarose gel using QIAquick kit 1. Gel slice weight:
pkd78 – 151.1 μg ∼ 151.1 μl
CRB gene – 169.8 μg ∼ 169.8 μl
2. Added volume of QG Buffer:
pkd78 – 455 μl
CRB gene – 510 μl
3. Incubated for 10 min at 500c 4. Added volume of Isopropanol:
pkd78 – 151 μl
CRB gene – 170 μl
5. Centrifuge: 1 min, 13,400 rpm. Discarded flow through. 6. Added 500 μl QG Buffer. 7. Centrifuge: 1 min, 13,000 rpm. Discarded flow through. 8. Added 750 μl PE Buffer. 9. Centrifuge: 1 min, 13,000 rpm. Discarded flow through. 10. Centrifuge: 1 min, 13,000 rpm. Discarded flow through. 11. Added 35 μl DDW PCR. 12. Centrifuge: 30 min, 13,000 rpm. 13. DNA concentration:
pkd78 backbone – 36 ng/μl
CRB resistance gene – 19 ng/μl
Both placed in -200c, top shelf, pink box, 1D & 2D.
6.8.2013 Eantibiotic replacement of pkd78
Ligation of CRB gene to pKD backbone 1. The DNA fragments were digested by restriction enzymes. These following components have been inserted into 2 separate epindorphs:
conc. of CRB: 19 ng/μl         conc. of pKD: 36 ng/μl Insert DDW first in order to lose less material. 2. Vortex and short spin. 3. Incubation at 37 for 15 min. 4. 10 min. at 80 degrees for inactivation of restriction enzymes. 5. Gel preparation: 1 gr Agarose, 50 ml TAE x1.       5.1 Heat in microwave for 1 min.       5.2 Cool the solution until it is warm and make sure it stays liquidly.       5.3 Add two drops of Etidium Bromide       5.4 Insert the gel into its facility and place the 30 well comb in its spot.       5.5 Wait until the gel gets tough.       5.6 Add TAE x1 until it covers the gel. 6. Insert the DNA fragments (30 μl) into the wells. 7. Insert marker lambda pST 6 μl to another well. 8. Turn electric potential to 90. 9. After 1.5 hr the bends were indentified and were extracted from the gel with the kit into 2 epindorphs with the DNA. QIAquick gel extraction kit (50) 10. Parameters for ligation:
11. Ligation mix (μl):
12. ligation was conducted by the Ligation protocol 13. Result:
7.8.2013 Transformation of BL21 pkd78 with pUC57 cI
Electrocompetent BL21 pkd78 were transformed by electroporation with pUC57 cI (τ=5.2 ms). Transformed cells were incubated in 300c for 2 hours, plated on 2 crb & cmp LB agar plates, and grown overnight in 300c. For negative control, untransformed BL21 pkd78 were also plated on crb & cmp plates. After 2 days, colonies grew on both plates which were plated with transformed cells, and none grew on the plates plated with untransformed BL21 pkd78.
7.8.2013
The C1 plasmid arrived from hy-labs and was diluted. Final concentration - 50 ng\ul.
Transformation of puc57-cI to super compotent cells (dh5α) by heat shock.
Transformation of the pGFPuv into dh5α. This pGFPuv has undergone directed mutagenesis in order to add the stop codon.
Transformation of C.O (copper oxidase) into BL21 that contain the UAA incorporation tRNA and the acRS (acetyl lysine synthethase). The machinery was on pSUP.
8.8.2013 freeze bacteria
3 samples were frozen in -800c:  pGFPuv 39 TAG, pSUP acRS PET15b – BL21, puc57-cI dh5α.S
Week 4       11.08.2013 - 17.08.2013
11.8.2013
2 starters containing: 10 ml (LB), 10ul (CRB 100), 10ul (cmp 50), pkD78 + C1
Kanamycin cassete has arrived.
kan cassete PCR from pkD4 plasmid ( T annealing= 52oC).
pkD78-ampR assembly - digestion of PCR products with XbaI and Xhol (FD)
products were separated on agarose gel and extracted using gel extracting kit.
12.8.2013
            
            
14.8.2013
Transformation of BL21 pkd78 with kan cassette. Starters have been prepared:
BL21 pkD78 + puc57-C1 +CRB100 (10ul) + CMP50 (10ul)
BL21 puc57-C1 +CRB100 (10ul)
15.8.2013 replacing antibiotic resistance
Goal: replacing antibiotic resistance Components:
PkD 78 PCR product
CRB PCR product
HiFi buffer X5
DDW
dNTP’s
ampR FWD & REV primer
cmpR FWD & REV primer
kappa HiFi polymerase
DNA template
Gel Agarose
Amounts:
PCR reaction:
Agarose gel has been prepared in order to get qualitative estimation about the components after the PCR. NO PCR Product was detected. Transformation via Electroporation protocol:
Sample 1 – BL21 pkD78 (EC) ->  puc57-CI  (1ul)
Sample 2 – BL21(EC) -> puc57-Time Bomb (1.5ul)
Sample 3 – BL21 (EC) -> puc57-CI (1ul)
For all the samples Τ=5.56 m.sec (Electric shock) Antibiotic plates prepared:
2* CRB 100 CMP 50
5* CRB 100
3* KAN 30
16.8.2013 mutagenesis - cI to cI+his tag (second attempt)
We tried to use site directed mutagenesis on puc57 cI to create Puc57 cI+his tag. A pcr vial was made:
35 uL DDW
10 uL HIFI buffer
1.5 uL rev+fwd primer (cI +his tag)
dNTP 1.5 uL
1 uL DNA template (cI plasmid)
1uL KAPA polymerase.
The following protocol was used: Cycling Parameters
Number of cycles
The PCR product were used for electroporation (Τ=4.9 msec) The bacteria were then plated on 2 crb agar plates (20 ul, 50 ul) and incubated in 37oC.
Week 5       18.08.2013 - 24.08.2013
18.8.2013
BL21 pUC57 cI and Bl21 pUC57 TB were frozen in -800c.
19.8.2013 Mutagenesis - cI to cI+his tag
The last 2 attempts of site-directed mutagenesis failed. The PCR products of Stav's experiment (13/08/13) were ran on agarose gel. The results showed  that the product was less than 300 bp - that indicates that the PCR failed. My experiment from the 16th was conducted again with several changes - the primer's stock was made again, and the denaturation temperature of the DNA was raised to 99oC, and this time, two exact PCR vials were made.
19.8.2013 Colony PCR
Colony PCR for: (using colony PCR protocol)
BL21 pkD78 +  puc57-CI
BL21+ puc57-Time Bomb
BL21 + puc57-CI
5 starters have been prepared from the colonies on the kanMX cassette transformation plates.
22.8.2013 Starters
4 starterts were made from plates containing puc57-c1 His-tag. 2 starters were made from frozen stock of BL21 pkd78+ puc57 C1
Week 6       25.08.2013 - 31.08.2013
26.8.2013 Bacteria with cI + his-tag lysis and separation
BL-21, puc57-cI w. cI+his tag culture after an over-night incubation w. IPTG induction (1mM) - was divided into 3 vials (10 mL each). - The cultures waere centrifuged @ 4000 rpm for 15 mins - afterwards suspension was discarded. - 2.5 ml of lysis buffer were added to all the vials (sediment was resuspended). - 2.5 uL of benzoaze were added to each vial. - 5 uL of protease inhibitor were added. - 1.5 uL of 30 mg/ml lysosyme were added. - 1 ml of bag-buster was added. - all the vials were shaken for 20 mins and then centrifuged @ 4000 rpm for 25 mins. - the lysate was filtered through a 45 um-pore filter. - 3 nickel-colons were taken: - washed with 6 ml DDW. - the lysate was added, the fluids at this step were collected in vials labeled "Flow". - 12 ml of wash buffer were added, the fluids at this step were collected in vials labeled "Wash". - 4 ml of elution buffer were added, the fluids at this step were collected in vials labeled "Elution". Results The vials were used the next day in a follow up experiment.
27.8.2013 Mini Prep + enzyme restriction for TB cassette
Mini Prep + enzyme restriction for TB cassette 1. mini prep for DNA extraction were done for: - pUC57 TB x2 - pUC57 cI x2 - pUC57 GK DNA stored in -200c 2. In order to extract the TB cassette from pUC57, pUC57 TB was restricted with XbahI and BamHI. 3. The reaction didnt work (twice), probably because of the XbaI, which is probably not active. New enzyme will be ordered and we will do the reaction again. Western for cI + His tag 1. For the western blotting we filled a 10 well gel with the following:
F - flow through W - wash E - elution 1,2,3 - 3 different colonies for the transformation 2. For the comassie we filled a 10 well gel with the following:
3. After the wells finished running on the gel, they were analyzed by the protocols: Western Blotting and Comassie. Results: There was no identification of cI-histag though there was a bend at the control well which signifies that the membrane blotting worked. For conclusion, the histag-cI mutagenesis didn't work. comassie:
membrane after blotting:
control:
28.8.2013 Comassie Analysis on cI expression leak without IPTG
1. 3 vials were grown overnight:       BL21 pUC57 cI histag + IPTG 1%       BL21 pUC57 cI histag       BL21 pUC57 cI histag + glu 1% 2. The vials were processed through protein extraction by lysis protocol. 3. The extraction, lysate, was loaded on a gel and ran for 1hr. 4. The gel was stained by comassie by the comassie protocol. Results:
Conclusions: 1. The bends around 30 KDa suggest expression of cI though its not guaranteed. If so, there is a leak, but either way there is a very high rate of expression of cI which is a good thing. 2. The bend is much bigger with IPTG which shows that the promoter works fine. It is important to check the initial O.D.  because each lysate has different amounts of protein.
31.8.2013 Comassie analysis for the lisate products made on 29.8
1. The lisate products were charged on the gel by the following order:
2. After they finished running on the gel, comassie staining took place by the comassie protocol Results:
Conclusions: It was impossible to identify anything. Probably because the protein concentration in the lysate was too low.
31.8.2013 Comassie analysis on cI expression under various conditions
1. 6 vials were grown overnight       1.1 BL21 pKD78 pUC57 cI + IPTG 1%, CRB100, CMP50       1.2 BL21 pKD78 pUC57 cI + 1% glu, CRB100, CMP50       1.3 BL21 pKD78 pUC57 cI + CRB100, CMP50       1.4 BL21 pUC57 cI + IPTG 1%, CRB100       1.5 BL21 pUC57 cI + glu 1%, CRB100       1.6 BL21 pUC57 cI + CRB100 2. The vials were diluted 1:40 with the new LB in the same conditions and incubated until the O.D. reached 0.6. 3. The vials were processed through protein extraction by lysis protocol and comassie staining by comassie protocol. Results: No bands were noticed on the gel! A possible reason could be because there wasn't enough lysate in the first place because intentionally the bacteria grew to only O.D. 0.6 so there could be an analysis only on live bacteria and not dead ones but seems that it's not enough to analyze protein expression.
31.8.2013 PCR to amplify pKD78 backbone and CRB resistance
1. PCR by 4/8 protocol 2. The PCR products were loaded on Agarose gel by the following order:
Results:
The PCR didnt work at all! The primers were not good.
Week 7       01.09.2013 - 07.09.2013
7.9.2013 Restriction of the P.A.S.E 1 cassette - preparation for tranformation
A plasmid containing the P.A.S.E 1 cassette (cI repressed promoter + holin + lysozime) was prepared for restriction in order to be transformed into bacteria. A vial was made contining: 1) 13.5 uL DDW 2) 2uL FastDigest buffer 3) 2.5 uL DNA (180 ng/uL) 4) 1 uL FastDigest BamH1 enzyme 5) 1 uL FastDigest XbaI enzyme (20 uL total). The vial was kept in 37oC for 15 mins, and then in 80oC for 20 mins (enzyme deactivation). The results were run on gel. Results: This experiment was conducted twice: 1- in the first time no DNA was spotted in the gel. 2- in the second time a bend that is >5 kbp was seen - meaning that the restriction failed.
7.9.2013 SDS-PAGE and staining of cI under different induction conditions
Bacteria with puc57-cI (IPTG-Arabinose promoter) were grown in 4 conditions: 1) no IPTG no Arabinose. 2) 1 mM IPTG, no Arabinose. 3) no IPTG no Arabinose. 0.6% glucose. 4) 1 mM IPTG, 20 mM Arabinose. The bacteria cultures were lysed (following the same protocol from 29/08 in our notebook). The lysis products were then separated by SDS-PAGE and stained (following the coomassie staining protocol). Results:
Week 8       08.09.2013 - 14.09.2013
8.9.2013 PCR amplification of 3 different templates
Three PCR vials were made, each containing: - 10 uL Taq readymix buffer - 7 uL DDW - 1uL DNA template - 1 uL forward primer -1 uL reverse primer (total 20 uL each). The vials were: 1) To amplify the cI + Ara/lacI promoter - DNA template: pUC57 cI. 2) To amplify pkd78 backbone without the cmp ABR - DNA template: pkd78. 3) To amplify crb ABR - DNA template: pet15. PCR program: 1) 5 mins @ 95oC - 2) a) 20 sec @ 98oC    b) 30 sec @ 51oC    c) 2.5 min @ 72oC 3) 5 min @ 72oC The results were ran on gel by Stav on the same day: 1.5% Agarose gel was made:
As can be seen pkd78 backbone amplification did not work, while crbR gene worked. crbR gene was excised from gel and extracted using QIA quick gel extraction kit. Extracted DNA concentration: 7.5 ng/&MU;l
9.9.2013 PCR for pGFPuv
1. 4% agarose gel with lacI/AraC primers in order to purify it from mix (100bp). 2. Gel extraction by QIAGEN protocol. 3. PCR by the following amounts:
The cycling plan:   (x18 cycles)
Results: The PCR didn't work!
9.9.2013 crbR gene restriction
2 different stock of crbR gene previously amplified through PCR and puc57 GK cassette (from original stock) were used for restriction:
The mixture was incubated in 370c for 15 minutes, and then the reaction was inactivated by incubating in 800c for 20 minutes. Restriction products were ran on a gel and extracted. DNA concetrations:
crbR 1 - 40 ng/μl
crbR 2 - 10 ng/μl
GK cassette - 17.5 ng/μl
GK cassette was used for transformation and sequencing: BL21 pkd78 puc57 cI were previously induced with arabinose (for expression of recombination machinery) and IPTG (for cI expression) and washed to become electrocompetent. This culture was used for 5 transformations by electroporation with GK cassette previously restricted. IPTG was added to the SOC used after transformation to a final concentration of 1mM. Transformed cells were plated on kan LB agar plates. Results: No colonies grew. Later the sequencing results shown no match with GK cassette.
9.9.2013 Miniprep for pPGF-UV Purification
Insulation of plasmid PGF-UV from BL-21 using protocol from kit. Result:
Vails stored in -20 C.
10.9.2013
DPN1 treatment was performed on the PCR products from the 9\9\13. and electroporation was performed on the products and grown on 2 plates (50ul, 150ul)
starter has been prepared from single colony - pkD78-amp + CRB100. - checking we have succeeded to replace the antibiotic resistance.
11.9.2013
one colony was taken from the plate made on the 10/9 in order to make a starter.
2 starters of BL21, pkD78, amp50, 30oC.
2 starters of BL21, pkD78 + puc57-cI, cmp50, amp100, 30oC.
12.9.2013
mini prep of BL21 pkD78 - 82 ng/ul. preparation of compotent cells using recombineering protocol -
BL21 + pkD78
BL21 + pkD78 + puc57-cI
Week 9       15.09.2013 - 21.09.2013
15.9.2013 Gradient_PCR - pull out the promoter
Reaction mixture:
      
Primers: araC-Lac upstr araC-Lac dnstr Temp: 500c, 52.10c, 58.10c, 600c
8 PCR tubes were prepared: 4 from puc57-cI sample, 4 from TB-cassette sample
1 PCR tube was used as blank.
16.9.2013 Site-directed Mutagenesis
Mutant Strand Synthesis: using protocol Site-directed Mutagenesis The sample reaction (X2 duplicate):
10 μL of 5XKAPA HiFi Buffer
1.5 μL of dNTP mix
6.6 μL (125 ng) of Forward Primer +  (125 ng) of Reverse Primer (primer is LAC1/araC , amplification PCR product at 8.9.13)
1 μL (5-50 ng) of DNA template (template is pGFPuv from 9.9.13)
ddH2O to final concentration 50 μL
Then add 1 μL of KAPA HiFi Polymerase.
Mutagenesis program- on protocol. Products Stored in -20C.
17.9.2013 - Transformation of Mutagenesis product to DH5a by Heat shock
using super-competent DH5a cells . In a sterile tubes (X2 duplicate):
10μl of competent cells + 2μl of mutagenesis product incubate on ice for 30min.
Cells were transferred to water bath at 42C for 30sec and immediately back to ice for 2 min.
200 μl of SOC medium was added.
Cells incubated at 37C for 1hr.
Plated  X μl on CRB plates over night at 37C.  (50 μl plate X2, 100 μl plate X2)
Control: same protocol done on competent cells without mutagenesis product.
18.09.2013   - Site directed mutagenesis (cI to cI +his tag) with new primers.
Two PCR vials were made, each containing: - 33.5 μL DDW.                         - 10 μ KAPA HIFI buffer.                         - 1.5 μ of dntp's mix.                         - 1.5 μ FWD primer.                         - 1.5 μ REV primer.                         - 1 μ template DNA (puc57 cI).                         - 1 μL KAPA polymerase.                         (total of 50 μL). Same PCR program was used as the experiment from the 16.08. After PCR, 1.5 μL of DpnI was added to each vial and kept in room temperature for 1 hour (following the protocol from the 16.09 experiment - from step 2). The DNA was ran on a gel. The two samples shown clear bands, indicating an amplification occurred. Proving the mutagenesis requires more steps. The DNA was then transformed into BL-21 E.coli (using the electro-poration protocol): Time constants -5 msec in vial 1. -5.1 msec in vial 2. After incubation the bacteria were plated on crb containing agar plates. Results after an overnight incubation, no colonies were spotted.
19.9.2013 - PCR for amplification:
1. The following templates we used for PCR: 1.1 PKD4 for AMP & KAN extraction 1.2 pUC57 cI 1.3 PeT 15b cueo-His 1.4 pUC57-GK 1.5 pUC57-TB 1.6 pKD78 2. The Primers used for the PCR:
3. Reaction mixture and cycle program:
      
Polymerization time: Reactions A,B,C,E,G - 45 seconds Reactions D,F - 3 minutes 4. Results: 4.1 SFR gel:
4.2 normal gel:
5. PCR extraction Other than the GK cassette and the pKD78 b bone, all the reactions worked and they were purified. Regarding the two that didn't, a possible reason could be that because the templates are very long (2000-5000bp) there should be a much longer extension time during the PCR reaction.
20.9.2013 - PCR to make cI his tag gene a biobrick:
pUC 57 cI his tag generated by site directed mutagenesis (from 18/09 - 2 different samples) was used as a template to create a standard biobrick cassette to be inserted into psB1C3. The primers used are B his + stop F and B cI TU R. Reaction mixture and cycle program:
      
PCR products were ran on a gel:
The marked bands correspond to ~800 bp, which is about the size of cI his tag with biobrick standard prefix and suffix. The PCR products were purified using QIA quick PCR purification kit. Purified DNA concentration - 28 ng/μl. The purified cI his tag was restricted with XbaI and pstI to become ready for ligation with a plasmid backbone. Reaction mix details: DNA -11 μl Buffer -2 μl pstI -1.5 μl XbaI -1.5 μl DDW -14 μl Reaction products were ran on agarose gel:
The marked band was excised from the gel and DNA was extracted using QIA quick gel extraction kit. Extracted DNA concentration - 1.000. Extraction did not work. We will try again tomorrow. Making biobricks:
The DNA purified from yesterday's PCR products, detailed above, were restricted with XbaI and pstI to become ready for ligation with pSB1C plasmid backbone. Reaction mix details (all in μl):
The restriction products were ligated with pSB1C plasmid backbone. Ligation mix details:
Amplifying GK cassette and pkd78 backbone Since these amplifications did not work yesterday, we repeated them with a different PCR program. Each reaction was done twice. 1. The following templates we used for PCR: 1.1 pUC57-GK 1.2 pKD78 2. The Primers used for the PCR:
3. Reaction mixture and cycle program:
      
4. Results: Agarose gel showd no bands. PCR did not work again.
20.9.2013 - Site-directed Mutagenesis
1.Mutant Strand Synthesis: using protocol Site-directed Mutagenesis The sample reaction (X2 duplicate):
10 μL of 5XKAPA HiFi Buffer
1.5 μL of dNTP mix
2.5 μL (250 ng) of Reverse+ Forward Primers (primer is LAC1/araC , done by Alex at 16.9.13)
1 μL (5-50 ng) of DNA template (template is pGFPuv from 9.9.13)
35 μL ddH2O to final concentration 50 μL
Then add 1 μL of KAPA HiFi Polymerase.
Mutagenesis program- at protocol. 2. Dpn1 restriction:
1.5 μL Dpn1 was added to mutagenesis product
Incubation at 37C for 1hr.
Products stored at -20C. 3.Transformation of Mutagenesis product to DH5a by Heat shock using super-competent DH5a cells . In a sterile tubes (X2 duplicate):
10μl of competent cells + 2μl of restriction product incubate on ice for 30min.
Cells were transferred to water bath at 42C for 30sec and immediately back to ice for 2 min.
200 μl of SOC medium was added.
Cells incubated at 37C for 1hr.
Plated  Xμl on CRB plates over night at 37C.  (50μl plate X2, 100μl plate X2)
Control:  same protocol done on competent cells without mutagenesis product. Incubation over night, 37C
21.9.2013
Result of Transformation: no colonies have seen on plates. Transformation of BL21 pUC57 cI with pUC57 GK: In order to provide proof of concept for PASE1 (GK), 2 samples of electrocompetent BL21 puc57 cI were transformed with pUC57 GK by electroporation (time constants were both 5.70 ms). The SOC used contained 1 mM IPTG, and the transformed cells were plated on crb+kan plates with 30 μl 100 mM IPTG. Back Bone digestion- PSB1C3
digestion with restriction enzymes PstI and XbaI.
Running restriction product in 2% agarose gel for B.bone extraction.
QI Aquik gel extraction kit for purifying DNA from gel. (nanodrop: con. 16ng/ul)
Ligation of the purifying B.Bone product with: cI T4, TB cassette, His tag+stop codon, KanR and, AmpR. (incubation for 1hr, 33C. stop reaction in 80C)
Transformation by electroforetion to BL-21, incubation over night.
Week 10       22.09.2013 - 30.09.2013
22.9.2013 Restriction and Ligation of BioBricks to PSB1C3 Back Bone
BioBricks: KanR, cI T4, HisTag+stop codon, TB cassette, and AmpR- PCR pruducts from 18.9.
PSB1C3 Back Bone- back bone extraction product from 21.9.
Restriction: using restriction enzymes PstI and XbaI  to restricted each BioBrick for ligation with PSB1C3 Back Bone.
NanoDrop measure of the restriction products.
Ligation: using T4 ligase and T4 buffer for 16 hr, in room temperature, to maximized ligation  products.
24.9.2013 pKD78 back bone and crbR- PCR, restriction, ligation and transformation
pKD78 back bone amplification PCR, program "long fragment".
analyzed products on 1.5% agarose gel and then purifying from gel  using kit.
25.9.2013 pKD78 back bone and crbR- PCR, restriction, ligation and transformation
Restriction: using restriction enzymes XhoI and XbaI to restricted the purifying PCR products of pDK78 B.bone and crbR (from the 18.9).
Both restriction products analyzed on 1.5% agarose gel and then purifying from gel  using kit.
Ligation (over nigth,room tem.)  of the pKD78 back bone with crbR, using T4 ligase.
26.9.2013 PCR to make cI his tag gene a biobrick
pUC 57 cI his tag generated by site directed mutagenesis was used as a template to create a standard biobrick cassette to be inserted into psB1C3. The primers used are B his + stop FWD and B cI TU REV. Reaction mixture and cycle program:
    
The bands on the  gel correspond to ~800 bp, which is about the size of cI his tag with biobrick standard prefix and suffix.
The purified cI his tag (using QIA quick PCR extraction kit) Restriction and Ligation of BioBricks to PSB1C3 Back Bone 1. PCR products from 19.9 (kanR, cI TU, TB) and PCR product cI+his tag compatible (made by alex) were digested with Xbah and PST1 by the restriction protocol and by the following amounts:
2. After the restriction, the DNA was purified with the PCR purification kit. 3. pSB1C3 RFP plasmid was digested as well in the same protocol with the same restriction enzymes but using the green buffer. 4. After restriction, the DNA was loaded on agarose gel and the top bend was extracted with the QIAGEN extraction kit.
5. Ligation of the inserts with the pSB1C3 backbone:
 
 
26.9.2013 Final step of inserting the P.A.S.E. 1 system in BL21 cells
1. Max induction with IPTG to produce maximum cI repressor 1.1 BL21 with pUC57 cI  starter was grown over night. 1.2 The starter was diluted 1:50 into 3 vials with 10 ml LB and 10 µl CRB and incubated until       it reached O.D. &tilde 0.2. 1.3 IPTG was added to each vial with different conc. ( 1mM, 2mM, 5mM). 1.4 The vials were put back in the incubator for 5 hrs of induction until they reach &tilde0.6 O.D. 1.5 Petri dishes were made with LB Agar that included 0.1% CRB, 0.05% Kan, 2mM IPTG. 2. Preparation of the cells for transformation 2.1 The vials were put in ice for 20 min and than in centrifugation for 15 min at 4000 RPM. 2.2 The supernatant was discarded and the cells were resuspended with 10 ml of 10% ice cold glycerol. 2.3 Centrifuge at 4000 RPM for 15 min and the supernatant was discarded. 2.4 The vials were resuspended with 5 ml of ice cold 10% glycerol and put back in centrifuge for 15 min at 4000 RPM. 2.5 The supernatant was discarded and the cells were resuspended with 400 µl of ice cold 10% glycerol and moved to 1.5 microfuge tubes. 2.6 Centrifuge at 10,000 RPM for 15 min in small centrifuge and discard of supernatant. 2.7 Resuspend with 90 µl of ice cold 10% glycerol. 3. Transformation of cells with GK cassette 3.1 Transformation was submitted by electroporation threw the Electotransformation protocol. 3.2 200 µl of 100mM IPTG were added to the plates made earlier (1.5). 3.3 150 µl of the cells were plated on plates and put in incubation for 12 hours. 4. Results Bacteria grew on all three plates. P.A.S.E 1 system insertion is complete! Starters were made from each plate with LB, kan50 and mM IPTG.
27.9.2013 40 LB agar plates with kanamycin and 2mM IPTG were made.
Preparing BL21 pUC57 cI & pUC57 GK for self-destruct essay:
From the starter of BL21 pUC57 cI & pUC57 GK made yesterday (denoted a3), 1 ml was taken and diluted in 10 ml LB with crb, kan and 2mM IPTG. This culture was grown to OD 0.2 and plated on kan crb 2mM IPTG LB agar plates. The culture remaining was diluted – 0.1 ml of the culture was added to 9.9 ml LB with crb, kan and 2 Mm IPTG, and grown overnight.
Bio Bricks transformation – Kan , cI TU, P.A.S.E 2 , cI +His With DH5α super compotent cells by heat shock. growin in 370C incubator.
pkD78-crbR transformation with BL21 electro compotent using electroporation – growing in 300C incubator.
Mini prep to DH5α – Kan , cI TU , P.A.S.E 2. yielding –
Kan – 59.5 ng/&MU;l , 1.919 , 2.705
cI TU – 44.5 ng/&MU;l , 1.935, 2.697
P.A.S.E 2 – 75.5 ng/&MU;l, 1.936, 2.359
Freezing DH5Α – Kan, cI TU, P.A.S.E 2. Using (-80) freezing protocol.
28.9.2013 Colony PCR for cells that grew on plates after transformation with pUC57 GK
1. Colony PCR was submitted in order to insure that the BL21 that grew on the plates after transforming GK cassette to BL21 with pUC57 cI worked and they contain the complete P.A.S.E 1 system. 2. One colony from the plate was taken and grew in 10 ml LB with 2mM IPTG and kan30 overnight. 3. The plasmids were extracted from the starter with the Invitrogen miniprep kit. 4. PCR for following templates:       4.1 plasmid extraction from the starter with GK primers       4.2 pUC57 GK with GK primers for control       4.3 plasmid extraction from the starter with cI TU primers       4.4 pUC57 cI with cI TU primers for control 5. Reaction Mixture and cycle program:       5.1  For GK presence (4.1 & 4.2):
   
      5.2 For cI presence (4.3 & 4.4):
   
6. The PCR products were analyzes on agarose gel with lambada pSTl1 DNA marker 7. Results
It’s clear that there is pUC57 cI in the cells though there was no amplification of GK. However there was no amplification of the control either. Therefore there is no conclusion regarding the presence of GK in the cells. The is a need to change the PCR program and try again.
28.9.2013 Verification for pKD78 with CRB resistance plasmid in the cells
1. A starter that was made out of a single colony that grew after 24 hrs of incubation as a result of the transformation of the pKD78 + CRB ligation product to BL21 EC cells from 27.09. 2. The starter grew for 10 hrs until it reached O.D. ˜ 1.2 3. The plasmid was extracted from the cells by the mini prep kit of Invitrogen. 4. The conc. of DNA was 131 ng/ul 5. In order to verify that this was the right plasmid and that the ligation from the 25.09 worked, restriction took place by the following amounts:
30 min in 37 degrees 10 min in 80 degrees 6. The restriction product was loaded on agarose gel for 30 minutes.
Continue the journey: read about our Modelling .
With appreciation to our sponsors:
Retrieved from " http://2013.igem.org/Team:BGU_Israel/Experiments "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
