"
Page
Discussion
View source
History
teams
Log in
 
Team:BGU Israel/Solution
From 2013.igem.org
BGU_Israel
Home
Environment
Enviromental Problem
Consequence Analysis
The Debate
Project
Motivation
Our Solution
Bio Bricks
Achievements
Judging Forms
Notebook
Protocols
Experiments & Results
Modelling
Stochastic Model
Modelling Results
Human Practice
Overview
Survey
Outreach
Continuity
Safety
Team
Acknowledgements
Our Solution Programmable Autonomous Self Elimination (P.A.S.E)
Overview
P.A.S.E 1
P.A.S.E 2
Overview
The processes of bioremediation, biosensors and bio-medicine often require the release of genetically modified organisms (GMO's) to the environment. After being released, these GMO's are no longer under direct control, therefore posing potential threats such as substituting the natural bacterial population, horizontal gene transfer, and lastly, creating public concern and opposition to the field of genetic engineering.
In order to overcome these issues, we are designing a genetic circuit that limits the lifetime of a bacterial population after it is released into the environment. Our ultimate goal is to allow the end user to program a GMO population to survive in the environment until it has completed its task, after which the entire population will disappear. Crucially, our mechanism will function independently, without any external intervention.
We are approaching this goal from two angles: "P.A.S.E 1" is based on the dilution of a vital component through cell division, while "P.A.S.E 2" is based on the lifetime of an essential protein. The two methods utilize novel mechanisms such as a recombination cassette, the innovative use of an unnatural amino acid (UAA) in a biologic gate, and more. We also intend to introduce new biobricks that will be useful for a wide range of purposes, including bacterial genome incorporation parts and UAA incorporation machinery. This project will be the first modular, independent and generic bacterial control system.
We want our construct to pave the way to a safe synthetic revolution: one that will allow developers to innovate and create freely without arousing public opposition or harming the environment.
P.A.S.E 1
Objective
Creation of a modular system to program E.coli to destroy themselves after a predetermined amount of time.
Overview
Our system consists of two parts – a protective element and a lethal element. The protective element is overexpressed only in the presence of a specific inducer, provided in a controlled environment. As long as the protective element is present in the cell, it prevents expression of the lethal element. Through cell division, the protective element will be diluted, and eventually, when its concentration will fall under a critical threshold, it won’t be able protects the cell from the lethal element.  This system is autonomous – unlike a “kill-switch”, it does not require any external intervention after the bacteria has already been released to the environment. Additionally, the system is intended to be programmable – through controlling different overexpression parameters; it is possible to affect the concentration of the protective element before the bacteria are released. The higher the concentration upon release, the longer the cells would be able to live outside the lab.
Behind the Design
After having decided what our project will be, we sat down and devised a general mechanism to make it work, as described in the overview above. It seemed to us, at that point, that we have done a great deal of the planning. But as we started looking for actual parts and biobricks that we could use to make our plan a reality, we realized the real challenge was still ahead of us – the registry is so big, that there is simply too much to choose from! We knew what kind of parts we need:         1. A toxin         2. A repressor/promoter couple  to regulate the toxin expression         3. A regulation system to control the expression of the toxin regulating repressor While we had to look for the first two parts, it was rather obvious for us what we should use for the third – the famous lacI/lacO from the lac operon we all learned about in Microbiology 101. We were eager to work, and it happened so that the lab we worked in had a stock of pGFPuv – a plasmid containing GFP gene under the regulation of lacO. So, we started conducting a long series of experiments with two goals:         1. Verifying that lacI/lacO is tightly regulating the gene under its control.         2. Quantifying the different level of expression under different inducer (IPTG in this case) concentrations – in order to show our system could be potentially “programmable”. After a lot of experiments (detailed in the July section of our lab notebook) and further theoretical research we came to two conclusions – the lacI/lacO is very “leaky” and the inducer concentration has no effect on the levels of expression [4], despite what our results have shown. We learned that different levels of fluorescence achieved by different inducer concentration, are the not the result of different expression levels on the single cell basis – they are the outcome of a bimodal distribution. All cells are either “on” (induced) or “off”. The ratio between the cells which are on and the cells which are of is the reason for different fluorescence measured. We wanted to reproduce the experiments that lead to this conclusion, so we used FACS analysis on our e. coli with pGFPuv. The FACS was done on 3 different cultures: IPTG induced BL21 pGFPuv, uninduced BL21 pGFPuv, and uninduced BL21.
These results we produced show that the conclusion mentioned above is correct (x axis is fluorescence, y axis is cell count). The left graph shows that the induced culture could be divided to two sub populations – one which expressed GFP, and one that didn’t. The middle graph shows that even when uninduced, most of the cells do produce GFP, which supports the fact this regulation system is leaky. We had to return to the drawing table. After a thorough searching session we found an interesting promoter that might suit our 2 goals (tight & programmable) – lac/ara-1 promoter – part BBa_K354000. This promoter is activated by induction with IPTG and/or L - arabinose. [1] This promoter functions as a fourway switch - using different variation of inducers (IPTG/Arabinsoe/IPTG+Arabinose/no inducer) allows the user to adjust the induction strength and levels of expression. We also thought since this promoter has binding sites for two repressors instead of just one, it would be tightly regulated. We also found a toxin and the suitable repressor/promoter, as detailed below.
Detailed design
The chassis used for the system is E. coli, strain BL-21. The functional system has two parts:
1. pUC57 plasmid
cI repressor gene (the protective element) - part BBa_K327018, edited to remove the LVA tail, which was originally meant to cause rapid degradation of the protein. Since the delay of the system greatly relies on the life time of cI, the tail was removed. The expression of this gene is regulated by the araC/lacI promoter.
lac/ara-1 promoter – part BBa_K354000. This promoter is activated by induction with IPTG and (but not necessarily) L - arabinose. [1] Using different variation of inducers (IPTG/Arabinsoe/IPTG+Arabinose) allows the user to adjust the induction strength, which controls the concentration of cI repressor in the cell. Higher concentration means longer life time of the population in the uncontrolled environment. Other ways to control induction strength have not yet been characterized - but our vision is that according to the user needs, different induction parameter or different promoters with different characteristics could be used to fine tune the life time of a population.
A constituently expressed resistance for Carbenicillin (for selection).
2. A cassette which undergoes recombination into the bacterial chromosome
Coupled holin and lysozyme genes – part BBa_K112022, coding for the toxins that will eventually kill the cell. Their expression is regulated by the cI regulated promoter.
cI regulated  promoter – par BBa_R0051. cI binding results in repression of transcription. When cI concentration is low enough, holin and lyzozyme genes are expressed.
A constituently expressed resistance for kanamycin (to allow for selection).
Why recombination? In our initial design, we intended to clone the toxin genes on a plasmid. However, one of the most important requirements of our system is that all cells must contain the toxin genes. In light of this requirement, plasmids pose a problem – their stability is influenced by the nature of the host cell, the type of plasmid and/or environmental conditions. Plasmid encoded properties may confer a selective advantage on the host cell but can be an energy drain due to replication and expression [2]. In our case – not only they confer no selective advantage, they carry a selective disadvantage! So, even if one cell would lose the toxin plasmid, it would not die, quickly giving rise to a population without the self-destruct system. In order to overcome this obstacle, we chose to use a cassette which undergoes recombination into the chromosome. The cell cannot lose the toxin gene, and is assured to pass it on to all daughter cells. This ensures the stability of the gene in the entire population. Using recombination has another advantage for our system. Because the gene is located on the chromosome instead of plasmid, it has the lowest copy number possible. This theoretically makes the protective element effective at lower concentrations, which would allow for a longer life time of the population. Assembling the system requires three transformations (although ultimately the cell will contain only one plasmid, and one addition to the chromosome):
pkd78 - make the cell recombination-ready: most bacteria are not readily transformable with linear DNA. One reason E. coli is not so transformable with linear DNA, is because of the presence of intracellular exonucleases that degrade linear DNA. In order to make the cells recombination-ready, we used the phage l Red recombinase. The recombination system we used was designed by Datsenko and Wanner: pkd78 is an easily cured, low copy number plasmid containing the phage l Red recombinase, which is synthesized under the control of arabinose. [3]
puc57 cI (Bba_K1223010) – insert the protective element gene.
Toxin cassette (Bba_K1223001) – insert the lethal element genes. Before transforming the cell with the cassette, in addition to following regular protocol for making competent cells, the culture has to be induced with arabinose and IPTG. Arabinose induces the recombination system, and IPTG induces the synthesis of cI, in order to prevent synthesis of the toxin after the transformation. After the transformation the culture is grown in 37oc curing the cell from pkd78.
References:
[1] Lutz, R. & Bujard, H. (1997). “Independent and tight regulation of transcriptional units in Escherichia coli via the LacR/O, the TetR/O and AraC/I1-I2regulatory elements”. Nucleic Acids Research, (25) 1203 – 1210. [2] McLoughlin, A. J. (1994). “Plasmid stability and ecological competence in recombinant cultures”. Biotechnology Advances (12), 279–324. [3] Datsenko K. A., Wanner, B.L. (2000). “One-step inactivation of chromosomal genes in Escherichia coliK-12 using PCR products”. PNAS (97), 6640-6645. [4] J Biotechnol. 2007 Feb 1;128(2):362-75. Epub 2006 Oct 17. Cell population heterogeneity in expression of a gene-switching network with fluorescent markers of different half-lives. Portle S, Causey TB, Wolf K, Bennett GN, San KY, Mantzaris N. View Source
P.A.S.E 2
Objective
Creation of a modular system to program E.coli to destroy themselves after a predetermined amount of time.
Overview
In this system, we intentionally encode for a nonsense stop codon (TAG) mutation in one of E.coli’s essential genes – Tyrosyl tRNA synthetase (tyrS). Its location in the E.coli genome is 1,713,972 - 1,715,246 ( - ); 36.94 min; 1275 (bp) , 424 (aa). In addition, we add an inducible promoter, AraC/LacI, for the regulation of the expression of this gene. Unable to translate this gene, the bacteria will lose most of their protein synthesis capabilities, resulting first in proliferation arrest and eventually death. We then introduce into bacteria a specific machinery, which allows the suppression of the TAG stop codon, resulting in translation of the essential tyrS gene (See detailed design section). Suppression is possible only when all the machinery is in place and when a special “building block” (in the form of an unnatural amino acid), is present. Moreover, only when induced by external compounds, the required machinery and the essential protein itself will be translated. This mechanism is designed as a “logic AND gate” – the bacteria are dependent on two different inducers (Arabinose and IPTG/Lactose) regulating transcription, and a special “building block” (Unnatural Amino Acid – UAA) for the translation regulation. All three compounds are relatively rare in most natural environments. In the lab, the engineered cells are supplied with the building blocks, machinery and inducers, thus synthesizing the essential protein in large quantities. Once the bacteria are released from the controlled environment (i.e. the lab), this protein naturally degrades over time. In the absence of the building blocks and the machinery inducer the bacteria cannot synthesize more of this essential protein. Once the protein reaches a critically low concentration the cell cannot sustain itself and dies. Currently we are working on creating one nonsense mutation in our target gene, however theoretically multiple nonsense mutations can be introduced, rendering the system even safer. Similar to our first approach (P.A.S.E 1), this system is also autonomous – unlike the "kill-switch", it requires no external intervention to function after it has been released to the environment. It can also be programmable - through controlling different parameters during induction, it is possible to affect the concentration of the essential protein before the cells are released. The higher the concentration upon release, the longer the cells would survive. Additional control over the lifetime of the bacteria is possible through controlling the lifetime of the essential protein itself (N-terminus mutations), or choosing to work with different essential proteins with different lifetimes. You are welcome to watch a short schematic 3D animation depicting P.A.S.E. 2:
Detailed design
A short introduction to unnatural amino acids (UAA)
Other than a few exceptions, the genetic code is universal and encodes 20 amino acids, the building blocks of all proteins. Recently, it has become possible to incorporate man-made amino acids into proteins by using the degeneracy of the genetic code. The incorporation of unnatural amino acids (UAAs) enables the production of proteins with novel properties for new applications. The incorporation of UAAs uses the cell's natural protein synthesis machinery by introducing a pair of tRNAcua and tRNA-synthetase that do not cross-react with (i.e. are orthogonal to) endogenous natural amino acids/tRNA-synthetases/tRNAs. A general approach of creating a system that will incorporate unnatural amino acids into proteins is described by L. Wang and P. G. Schultz [1]: the system that was developed consists of a tRNA and tRNA synthetase set that is orthogonal to the other amino acids and amino-acyl-tRNA-synthethases in the cell. The orthogonality of the system is achieved by using natural tRNA and tRNA-synthethase pair from another species, the archae Methanocaldococcus jannaschii. After mutating the tRNA anticodon sequence to C-U-A to suppress the Amber stop codon (UAG), the tRNA and the synthethase were randomly mutated in chosen locations to produce a library for selection of the mutant with the desired properties. The selection of mutants was done by a series of negative and positive selection tests for orthogonally, ability to suppress the amber stop codon, and incorporation of unnatural amino acids into a protein. The gene of the target protein is site-specifically mutated to insert the amber stop codon (TAG) in the desired site for the incorporation of the UAA. The Amber stop codon was chosen since it is the least used stop codon in the E. coli’s genetic code (only 314 sites are known) and therefore is not anticipated to interfere with the synthesis of other proteins.
The chassis used for the system is E.coli, strain BL-21. The functional system has two parts:
1. UAA machinery (transformed into bacteria):
Orthogonal tRNAcua.
Pyro-lysyl tRNA synthetase, regulated by LacI promoter.
A constituently expressed resistance for Chloramphenicol (for selection).
UAA: Propargyl-L-lysine.
2. A linear DNA cassette, which undergoes recombination into the bacterial chromosome:
Homology regions were designed so that the cassette will enter the chromosome in a specific site: it replaces the first 60 base pairs of the 20 N-ternimus codons of the Tyrosyl tRNA synthetase gene. The replaced sequence is the same as the original chromosomal sequence, with one change – the sixth amino acid code has been mutated to the amber stop codon (TAG). After the recombination, only cells with the UAA machinery (tRNA & tRNA synthetase couple) and UAA (Propargyl-L-lysine) supply can translate the Tyrosine tRNA Synthetase, an essential gene.
lac/ara-1 promoter – part BBa_K354000. Upstream of the homology site and the Tyrosyl Synthetase gene, we added the lac/ara-1 promoter. This promoter is activated by induction with IPTG and (but not necessarily) L-arabinose.[2] Using different combinations of IPTG, Arabinsoe, or IPTG+Arabinose allows the user to adjust the induction strength, which controls the concentration of tyrosyl synthetase in the cell. A higher concentration means a longer lifetime of the population in the uncontrolled environment. Other ways to control induction strength have not yet been characterized - but our vision is that different induction parameters or another promoter with different characteristics could be used to fine-tune the lifetime of a population according to the needs of the end user.
A constituently expressed resistance for kanamycin (for selection).
System Assembly
Assembling the system requires three transformations (although ultimately the cell will contain only one plasmid, and one change in the chromosome):
Transformation of pkd78 - makes the cell recombination-ready: most bacteria are not readily transformable with linear DNA. One reason E.coli is not easily transformable with linear DNA, is because of the presence of intracellular exonucleases that degrade linear DNA. In order to make the cells recombination-ready, we used the phage l Red recombinase. The recombination system used was designed by Datsenko and Wanner: pkd78 is an easily cured, low copy number plasmid containing the phage l Red recombinase, which upon induction with Arabinose makes the bacteria significantly susceptible to linear DNA cassette genome incorporation. [3] This pkd78 has a different antibiotic resistance than the one used in P.A.S.E 1.
Transformation of UAA machinery (tRNAcua and tRNA synthetase) into the bacterium on a protein expression plasmid (with high copy number).
Transformation of the P.A.S.E 2 cassette (Bba_K1223002) – adds a regulatory element (lacI/araC) to the chromosomal Tyrosine synthetase gene and inserts a nonsense mutation to it (TAG stop codon). Before transforming the cell with the cassette, in addition to following regular protocol for making competent cells, the culture has to be induced with arabinose, inducing the expression of the recombination proteins.
After genome cassette incorporation, the pkd78 is removed using heat shock or incubation in high temperatures (37-42 degrees Celsius).
The bacteria are grown in a rich medium containing two antibiotics (the antibiotic resistance is expressed from the UAA machinery plasmid and from the genome cassette insert), Propargyl-L-lysine and inducers IPTG and Arabinose.
After the growth and accumulation of the essential recombinant protein, the medium is changed to a medium without antibiotics, inducers or Propargyl-L-lysine.
Lastly the bacteria undergo (-80 degrees Celsius) freeze protocol and are ready for use.
Continue the journey: read about Our Bio Bricks .
References:
[1] Wang, L. and P.G. Schultz, Expanding the genetic code. Chem. Commun. 2002(1): p. 1-11. [2] A. J. McLoughlin, Plasmid stability and ecological competence in recombinant cultures. Biotech. Advances 12(2), 279–324 (1994). [3] K. A. Datsenko, B. L. Wanner, One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proc. Natl. Acad. Sci. 97 (12), 6640-6645 (2000).
With appreciation to our sponsors:
Retrieved from " http://2013.igem.org/Team:BGU_Israel/Solution "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
