"
Page
Discussion
View source
History
teams
Log in
 
Team:Aberdeen Scotland/Original Project Abstract
From 2010.igem.org
University of Aberdeen - ayeSwitch - iGEM 2010
Home
Our Project
Our Team
iGEM at Aberdeen
Team Members
Team Advisors
iGEM at Work
Official Team Profile
Biology
General Overview
DNA Constructs
Results
Protocols
Parts Submitted to the Registry
Modelling
Equations
Nullclines and Fixed Points
Bifurcation and Stability
Stochastic Model
Parameter Space Analysis
Determination of the Hill coefficient n2
Directed Evolution
Lab Diary
Switch Characterisation
Biobrick related
Safety
Safety Concerns
Safety Solutions
 
University of Aberdeen - ayeSwitch
iGEM 2010
Bio-traffic light systems; engineering toggle switches in yeast using translational control of gene expression
Abstract
This synthetic biology project will engineer a novel gene circuit allowing yeast to express either a green, or red fluorescent protein, in a stable, mutually exclusive way, thus exhibiting toggle switch control of expression. Exposure of yeast to specific environmental signals will switch between the fluorescent proteins. In contrast to existing genetic toggle switches, employing transcriptional control to achieve regulation, this project will achieve the same control principle by engineering a translational regulation circuit in a model eukaryote. A key advantage of translational control is that it enables a more rapid regulation of protein levels since it by-passes mRNA synthesis and export. Translationally regulated toggle switches could have multiple applications, stably recording exposure to environmental triggers, or in long-term maintenance of therapeutic protein synthesis 3 .
Aims
We will implement a translational control toggle switch using two proteins, Iron Response element binding Protein (IRP), and MS2-stem loop binding protein (MS2-BP); both proteins bind RNA stem loops in a sequence-specific manner. In the toggle switch circuit, the gene for IRP will contain an MS2 RNA stem in its 5’ leader; a second gene will encode MS2-BP, with an iron response element in its 5’ leader. Each RNA-binding protein can potentially inhibit the translation of the other ( 1 , 2 ; Figure 1). The two RNA binding proteins will be fused to GFP and RFP respectively, to allow fluorescence monitoring of toggle switch position. By additionally regulating the transcription of each RNA-binding protein gene, using galactose- and copper-inducible promoters respectively, a re-set function is established, allowing red/green toggling. The project has scope for development if time allows; translational control of stop codon readthrough using suppressor tRNAs can be used to tag IRP-GFP with a destabilising peptide tag, acting as a translationally controlled toggle reset.
Mathematical modelling will be used to describe the expression of the circuit’s genes, including both transcription and translation. Initially, we will use ordinary differential equations to predict the concentration of proteins in response to signal input. Later refinement will employ a stochastic model of translation that considers the traffic of ribosomes on the mRNAs, and links to the transcription process. The presence of a protein blocking the 5’ leader of an mRNA will be modelled by reducing the probability with which ribosomes bind the mRNA. Stochastic simulations will moreover allow generalising predictions from single cells to populations of cells. The proposed model will be validated by comparing the predictions to experimental data.
Overall, the project will test the hypothesis that synthetic toggle switches can be engineered using translational control. The project will additionally establish a translational control toolkit for eukaryote bioengineering. The specific objectives and anticipated outcomes of this project are as follows;
Development of deterministic and stochastic mathematical models of translational toggle switches to guide circuit design.
Construction of RNA binding protein BioBricks, fused to GFP and RFP, in yeast expression cassettes.
Testing of RNA-binding protein/fluorescent protein expression in yeast, at the single cell and population levels.
Iterative improvement of the model using experimental data, and test of new predictions generated by the model.
References
1 Oliveira CC, et al. (1993) Nucleic Acids Res; 21:5316-22.
2 Stripecke R, et al (1994) Mol Cell Biol; 14:5898-909.
3 Haynes KA and Silver PA (2009), J. Cell Biol., 187:589-96
Retrieved from " http://2010.igem.org/Team:Aberdeen_Scotland/Original_Project_Abstract "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
