<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Aberdeen_Scotland_Evolution"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Aberdeen_Scotland/Evolution">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Aberdeen_Scotland/Evolution&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Aberdeen_Scotland/Evolution&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Aberdeen_Scotland/Evolution&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Aberdeen_Scotland/Evolution" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Aberdeen Scotland/Evolution</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P><TITLE>University of Aberdeen - ayeSwitch - iGEM 2010</TITLE></P><DIV id="pagewrapper"><DIV id="titleheader"><DIV id="mainnavigation"><DIV class="menuwidth0"><DIV class="menuwidth1"><DIV class="menuwidth2"><DIV class="menu"><UL><LI><A title="Home" href="https://2010.igem.org/Team:Aberdeen_Scotland" class="menu">Home</A></LI></UL><UL><LI><A class="menu" href="https://2010.igem.org/Team:Aberdeen_Scotland/Project_Overview">Our Project</A></LI></UL><UL><LI><A class="menu" href="https://2010.igem.org/Team:Aberdeen_Scotland/Team">Our Team</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Team">iGEM at Aberdeen</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/TeamMembers">Team Members</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Advisors">Team Advisors</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/LabPics">iGEM at Work</A></LI><LI><A href="https://igem.org/Team.cgi?year=2010">Official Team Profile</A></LI></UL><UL><LI><A class="menu" href="https://2010.igem.org/Team:Aberdeen_Scotland/General Overview">Biology</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/General Overview">General Overview</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Constructs">DNA Constructs</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Results">Results</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Protocols">Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Parts">Parts Submitted to the Registry</A></LI></UL><UL><LI><A class="menu" href="https://2010.igem.org/Team:Aberdeen_Scotland/Modeling">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Equations">Equations</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Fixed_Points">Nullclines and Fixed Points</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Bifurcation">Bifurcation and Stability</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Stochastic_Model">Stochastic Model</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Probability">Parameter Space Analysis</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Curve Fitting">Determination of the Hill coefficient n<SUB style="font-size:10px&quot;">2</SUB></A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Evolution">Directed Evolution</A></LI></UL><UL><LI><A class="menu" href="https://2010.igem.org/Team:Aberdeen_Scotland/Switch Characterisation">Lab Diary</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Switch Characterisation">Switch Characterisation</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Biobrick Related">Biobrick related</A></LI></UL><UL><LI><A class="menu" href="https://2010.igem.org/Team:Aberdeen_Scotland/Safety">Safety</A></LI><LI><A href="https://2010.igem.org/Safety">Safety Concerns</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Safety">Safety Solutions</A></LI></UL></DIV></DIV></DIV></DIV><H3 class="toptitle"> </H3></DIV><DIV id="mainpic"><DIV id="pagetitlewrapper"><H3 class="maintitle">University of Aberdeen - ayeSwitch</H3><DIV id="pagesubtitle">iGEM 2010</DIV></DIV></DIV></DIV><DIV class="wide"><H1>What is directed evolution?</H1><P>In protein engineering, the method of <A href="http://en.wikipedia.org/wiki/Directed_evolution">directed evolution</A> is used to harness the power of natural selection.  It is used to evolve certain proteins or RNA with desirable properties not found in nature.  Rather than swapping out different components of the structure, directed evolution can be used to manipulate the behaviour of these components.  Then, by screening or selection, we can obtain the desired phenotype.</P><P>There are three typical steps involved that include:</P> <P><B>1. Diversification:</B> The gene that encodes the desired protein is mutated and recombined at random to create a large library of gene mutants.  Techniques commonly used in this step involve PCR and DNA shuffling.</P> <P><B>2. Selection:</B> The library is tested for the existence of mutants possessing the property of interest using screening or selection.</P><P>    <B>a.</B> Selections automatically remove all mutants that do not function.</P><P>    <B>b.</B> Screening enables the researcher to manually detect and isolate the high-performing mutants.</P> <P><B>3. Amplifications:</B> The mutations identified int he selection or screening process are replicated several times so it allows the researchers to sequence their DNA and recognize what mutations have occurred.</P><P>For optimizing efficiency, altering more than one parameter would be preferable in order to obtain the desired protein.  Therefore, we accululate mutations over multiple generations.</P><P>In order to examine how changes in the parameters were affecting the stability of the steady states, we could do uniform bifurcation analysis e.g. determining th regions of stability.</P><P>In the experiments, we were able to use the FACS (fluorescence activated cells sorting) machine to separate the proteins with the brightest colours and thus use the strongest ones. Once we had this, we could put the proteins through the system again as initial conditions and then we would have a more distinctive switch.</P><H3>Pros</H3><P>Directed evolution can systematically perturb and distinguish components then, using these perturbed components, we can gain further insight into the functionality of the normal circuit.</P><P>If some of the pathways enhance the desired performance, then iterative rounds of random mutation of the circuit and screening for the desired properties may provide steady improvement.  This only works if the initial design is close to a better design.</P><H3>Why use maths to model directed evolution?</H3><P>There are three primary limitations that mathematical modelling can overcome:</P> <P><B>1.</B> The evolutionary space for a genetic circuit is too large to explore efficiently.</P> <P><B>2.</B> Refining our parameters is much easier than improving (or altering) the system experimentally.</P> <P><B>3.</B> Although selecting for independent properties is possible, it usually requires setting up multiple rounds of screening or selection.</P><H3>Procedure</H3> <P><B>1.</B> We first took the basic program where we integrated the system, plugging in an estimation of the parameters.
 </P><P><B>2.</B> For each parameter, it yielded the value for which we obtain the largest difference between GFP and CFP.
 </P><P><B>3.</B> We then fixed our chosen parameter and continue the program with the next parameter.
 </P><P><B>4.</B> This can be done over and over for each parameter of the model, eventually producing a clearly defined switch that will somewhat resemble our original system.
 </P><P><B>5.</B> Repeat these steps for each parameter in the system to eventually produce a clearly optimized switch that will yield the maximum difference between GFP and CFP when switching from one state to the other.
</P><H3>Results</H3><P>Fig 1. shows the behaviour of the original system over a set period of time.</P><DIV align="center"><P><B>Fig 1.</B></P></DIV><P>As shown, the system has been integrated over a long enough timespan such that it has reached equilibrium.</P><P>Once we had this, we could then do this again but change one parameter and instead insert a range of values that can be put through the system.</P><P>Then we found the value for that parameter that produced the largest difference between the two proteins.
We did this for each parameter until we had an “ideal” value for each one. Finally, we entered these values into the system and integrated it again.</P><DIV align="center"><P><B>Fig 2.</B></P></DIV><DIV align="center"><P><B>Fig 3.</B></P></DIV><P>Fig 2. shows our final result of the mRNAs and the two proteins. We can see at first that CFP was the brightest protein initially, then we added more galactose and, at around time = 50, we can clearly see the decline of CFP and the increase of GFP.  Fig 3. shows how copper and galactose are acting in the system. Thus, a clear switch in our system.  The results for each parameter were as follows;</P><DIV align="center"><TABLE><TBODY><TR><TD><DIV align="right"><B><P>Parameter</P></B></DIV></TD><TD><P><B>Optimal Valued</B></P></TD></TR><TR><TD><DIV align="right"><P>K_1</P></DIV></TD><TD><P>5000</P></TD></TR><TR><TD><DIV align="right"><P>K_2</P></DIV></TD><TD><P>10000</P></TD></TR><TR><TD><DIV align="right"><P>K_3</P></DIV></TD><TD><P>5000</P></TD></TR><TR><TD><DIV align="right"><P>K_4</P></DIV></TD><TD><P>100000</P></TD></TR><TR><TD><DIV align="right"><P>lambda_1</P></DIV></TD><TD><P>133.9940</P></TD></TR><TR><TD><DIV align="right"><P>lambda_2</P></DIV></TD><TD><P>415.9916</P></TD></TR><TR><TD><DIV align="right"><P>lambda_3</P></DIV></TD><TD><P>583.9984</P></TD></TR><TR><TD><DIV align="right"><P>lambda_4</P></DIV></TD><TD><P>417.9918</P></TD></TR><TR><TD><DIV align="right"><P>mu_1</P></DIV></TD><TD><P>0.00092593</P></TD></TR><TR><TD><DIV align="right"><P>mu_2</P></DIV></TD><TD><P>0.00024995</P></TD></TR><TR><TD><DIV align="right"><P>mu_3</P></DIV></TD><TD><P>0.0004495</P></TD></TR><TR><TD><DIV align="right"><P>mu_4</P></DIV></TD><TD><P>0.00025</P></TD></TR></TBODY></TABLE></DIV><H3>References</H3><P><SUP style="font-size:10px">[1]</SUP> HEric L. Haseltine and Frances H. Arnold (2007) Synthetic Gene Circuits: Design with Directed Evolution. Annual Review Biophys Biomol Struct. 2007;36:1-19. </P><P><SUP style="font-size:10px">[2]</SUP> Andrianantoandro, E., Basu, S., Karig, D.K., Weiss, R. Synthetic biology: New engineering rules for an emerging discipline (2006) Molecular Systems Biology, 2, art. no. msb4100073, pp. 2006.0028 Biol. Chem., Vol. 266, No.1, pp.71-75 </P><P><SUP style="font-size:10px">[3]</SUP> Alon, U. (2006) An Introduction to Systems Biology: Design Principles of Biological Circuits. </P><P><SUP style="font-size:10px">[4]</SUP> Chen, B.-S., Chang, C.-H., Lee, H.-C. Robust synthetic biology design: Stochastic game theory approach (2009) Bioinformatics, 25 (14), pp. 1822-1830. </P><P><SUP style="font-size:10px">[5]</SUP> Guntas, G; Mansell, TJ, Kim, JR, Ostermeier, M. Directed evolution of protein switches and their application to the creation of ligan-binding proteins (2005). Procedings of the National Academy of Sciences of the United States of America, 102 (32): 11224-11229 </P><P><SUP style="font-size:10px">[6]</SUP> Marchisio, MA (Marchisio, M. A.); Stelling, J (Stelling, J. Computational design of synthetic gene circuits with composable parts (2008). IOINFORMATICS, 24 (17): 1903-1910 </P><P><SUP style="font-size:10px">[7]</SUP> Johannes, TW; Zhao, HM. Johannes, TW; Zhao, HM (2006). CURRENT OPINION IN MICROBIOLOGY, 9 (3): 261-267 </P><P><SUP style="font-size:10px">[8]</SUP> Nowak MA (2006). Evolutionary Dynamics: Exploring the Equations of Life.Harvard University Press. (Excerpt, Nature review, Science review,American Scientist review, Quarterly Review of Biology article, Science Daily, R.R. Hawkins Award, 2007 Book of Distinction) </P><P><SUP style="font-size:10px">[9]</SUP><A href="http://www.bio.davidson.edu/courses/genomics/method/FACS.html">Biology @ Davidson, Davidson College Academics.</A></P><TABLE class="nav"><TBODY><TR><TD><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Probability">  Return to Determination of the the Hill Coefficient n<SUB style="font-size:10px&quot;">2</SUB></A></TD><TD align="right"><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Switch_Characterisation">Continue to the Switch Characterisation Lab Diary  </A></TD></TR></TBODY></TABLE><P align="CENTER"><A href="#top">Back to the Top</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Aberdeen_Scotland/Evolution">http://2010.igem.org/Team:Aberdeen_Scotland/Evolution</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Aberdeen_Scotland/Evolution" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Aberdeen_Scotland/Evolution" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Aberdeen_Scotland/Evolution&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Aberdeen_Scotland/Evolution&amp;oldid=198829" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>