<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Aberdeen_Scotland_Probability"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Aberdeen_Scotland/Probability">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Aberdeen_Scotland/Probability&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Aberdeen_Scotland/Probability&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Aberdeen_Scotland/Probability&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Aberdeen_Scotland/Probability" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Aberdeen Scotland/Probability</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P><TITLE>University of Aberdeen - ayeSwitch - iGEM 2010</TITLE></P><DIV id="pagewrapper"><DIV id="titleheader"><DIV id="mainnavigation"><DIV class="menuwidth0"><DIV class="menuwidth1"><DIV class="menuwidth2"><DIV class="menu"><UL><LI><A title="Home" href="https://2010.igem.org/Team:Aberdeen_Scotland" class="menu">Home</A></LI></UL><UL><LI><A class="menu" href="https://2010.igem.org/Team:Aberdeen_Scotland/Project_Overview">Our Project</A></LI></UL><UL><LI><A class="menu" href="https://2010.igem.org/Team:Aberdeen_Scotland/Team">Our Team</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Team">iGEM at Aberdeen</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/TeamMembers">Team Members</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Advisors">Team Advisors</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/LabPics">iGEM at Work</A></LI><LI><A href="https://igem.org/Team.cgi?year=2010">Official Team Profile</A></LI></UL><UL><LI><A class="menu" href="https://2010.igem.org/Team:Aberdeen_Scotland/General Overview">Biology</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/General Overview">General Overview</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Constructs">DNA Constructs</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Results">Results</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Protocols">Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Parts">Parts Submitted to the Registry</A></LI></UL><UL><LI><A class="menu" href="https://2010.igem.org/Team:Aberdeen_Scotland/Modeling">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Equations">Equations</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Fixed_Points">Nullclines and Fixed Points</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Bifurcation">Bifurcation and Stability</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Stochastic_Model">Stochastic Model</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Probability">Parameter Space Analysis</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Curve Fitting">Determination of the Hill coefficient n<SUB style="font-size:10px&quot;">2</SUB></A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Evolution">Directed Evolution</A></LI></UL><UL><LI><A class="menu" href="https://2010.igem.org/Team:Aberdeen_Scotland/Switch Characterisation">Lab Diary</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Switch Characterisation">Switch Characterisation</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Biobrick Related">Biobrick related</A></LI></UL><UL><LI><A class="menu" href="https://2010.igem.org/Team:Aberdeen_Scotland/Safety">Safety</A></LI><LI><A href="https://2010.igem.org/Safety">Safety Concerns</A></LI><LI><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Safety">Safety Solutions</A></LI></UL></DIV></DIV></DIV></DIV><H3 class="toptitle"> </H3></DIV><DIV id="mainpic"><DIV id="pagetitlewrapper"><H3 class="maintitle">University of Aberdeen - ayeSwitch</H3><DIV id="pagesubtitle">iGEM 2010</DIV></DIV></DIV></DIV><DIV class="wide"><H1>Parameter Space Analysis</H1><P>Recall that the equations of our system are as follows (see <A href="https://2010.igem.org/Team:Aberdeen_Scotland/Equations">Equations</A> page for parameter definitions):</P><P>We wrote a program which takes the equations for our system, calculates the nullclines and records whether or not bistability is achieved. Whether or not bistability is achieved depends strongly on the parameters  λ<SUB style="font-size:10px">i</SUB>, K<SUB style="font-size:10px">i</SUB>, μ<SUB style="font-size:10px">i</SUB> and n<SUB style="font-size:10px">i</SUB>.</P><P>The values for the parameters λ<SUB style="font-size:10px">i</SUB>, K<SUB style="font-size:10px">i</SUB> and μ<SUB style="font-size:10px">i</SUB>  were taken from literature<A href="#ref1"><SUP style="font-size:10px">[1]-[13]</SUP></A> but have a large uncertainty attached to them. To take this uncertainty into account we selected each parameter value randomly from a range of two orders of magnitude around the literature value for each parameter. We knew that n<SUB style="font-size:10px">2</SUB> was between 1 and 3 and n<SUB style="font-size:10px">4</SUB>=1. This is due to the number of stem loops on the two DNA strands (see Fig 1 in <A href="https://2010.igem.org/Team:Aberdeen_Scotland/Equations">Equations</A>). The strand associated with GFP has two stem loops. A protein can bind to one stem loop and this can then hinder the binding of a protein to the second stem loop (thus making n<SUB style="font-size:10px">2</SUB> less than 2 but more than 1) or encourage the binding of a protein to the second stem loop (thus making n<SUB style="font-size:10px">2</SUB> more than 2 but less than 3). The strand associated with the CFP has only one stem loop and thus n<SUB style="font-size:10px">4</SUB>=1. Please see <A href="https://2010.igem.org/Team:Aberdeen_Scotland/Curve_Fitting">Determination of the Hill coefficient n<SUB style="font-size:10px">2</SUB></A> for an explanation on how we narrowed down the value for n<SUB style="font-size:10px">2</SUB>.

We ran the program for various combinations of Hill coefficients (n<SUB style="font-size:10px">i</SUB>) between 1 and 5. Each time the program was run, the Hill coefficients remained constant but the other parameters varied. </P><P>Choosing the parameters in this way meant that each time the program was run we would sometimes achieve a combination of parameters that allowed bistability, and sometimes not. 

The program was run 100 times and the number of times bistability was achieved was output to the screen as a percentage. 

We then decided to run the program for various different parameter ranges. We cannot change individual parameters specifically, but in the lab it is possible to vary the order of magnitude of a parameter. We wanted to know, if we varied our parameters and Hill coefficients, what is the best percentage we can possibly get? These results are shown in scenarios 2-4.</P><H3>Scenario 1 – unmodified parameters </H3><P>These parameters were taken from, or derived from, the information contained within various sources of literature<A href="#ref1"><SUP style="font-size:10px">[1]-[13]</SUP></A>. The parameters were converted from concentrations to numbers of molecules.

We did not know what concentration of galactose and methionine was optimal, so we ran various tests and found the value for which the percentages were maximised. We found that the optimal percentages occurred if we have 100000 times more galactose than methionine. See figure 1a and 1b.</P><CENTER>
a)
b)<P style="font-size:10px">Figure 1. a) The percentage of parameter combinations tested that will give us bistability for various combinations of Hill coefficients. n<SUB style="font-size:5px">2</SUB> is the Hill coefficient for the GFP rate-reaction equation and n<SUB style="font-size:5px">4</SUB> is the Hill coefficient for the CFP rate-reaction equation. b) Graphical representation of 1a. </P></CENTER><P>Here we can see that if n<SUB style="font-size:10px">4</SUB>=1, then the best scenario we can hope for is that n<SUB style="font-size:10px">2</SUB>=5. In this situation, 3.29% of the parameter combinations tested will give bistability and the possibility of switching. However, with n<SUB style="font-size:10px">2</SUB> estimated to be between 1 and 3, we can say that at most 2.03% of the parameter combinations tested gave bistability. </P><H3>Scenario 2 – modified parameters 1</H3><P>Percentages are maximised if we have 100000 times more galactose than methionine. See figure 2a and 2b.</P><CENTER>
a) 
b) <P style="font-size:10px">Figure 2. a) The percentage of parameter combinations tested that will give us bistability for various combinations of Hill coefficients. b) Graphical representation of 2a.</P></CENTER><P>The best scenario is if we have n<SUB style="font-size:10px">2</SUB>=5 and n<SUB style="font-size:10px">4</SUB>=5.</P><H3>Scenario 3 – modified parameters 2</H3><P>Percentages are maximised if there is 100000 times more galactose than methionine. See figure 3a and 3b.</P><CENTER>
a) 
b) <P style="font-size:10px">Figure 3. a) The percentage of parameter combinations tested that will give us bistability for various combinations of Hill coefficients. b) Graphical representation of 3a.</P></CENTER><P>Here we find that the optimal scenario is n<SUB style="font-size:10px">2</SUB>=3 and n<SUB style="font-size:10px">4</SUB>=5. The trend which has developed in this table will be discussed at the end of scenario 4. </P><H3>Scenario 4 – modified parameters 3 </H3><P>Percentages are maximised if we have 10000000 times more galactose than methionine. See Figure 4a and 4b.</P><CENTER>
a) 
b) <P style="font-size:10px">Figure 4. a) The percentage of parameter combinations tested that will give us bistability for various combinations of Hill coefficients. b) Graphical representation of 4a.</P></CENTER><P>An interesting trend is seen in scenarios 3 and 4. The percentages increase to a maximum at n<SUB style="font-size:10px">2</SUB>=2 and n<SUB style="font-size:10px">4</SUB>=5 (n<SUB style="font-size:10px">2</SUB>=3 and n<SUB style="font-size:10px">4</SUB>=5 in scenario 3) and then decrease, quite rapidly, towards Hill coefficients of n<SUB style="font-size:10px">2</SUB>=5 and n<SUB style="font-size:10px">4</SUB>=5. In the parameter space, there is a region where bistability is possible. For the smaller Hill coefficients, the randomly chosen parameters (as described above) would allow the nullclines to have a shape such that they would cross three or more times and bistability is achieved. The higher Hill coefficients, however, for the same range of parameters, would have nullclines which were shaped so that they did not cross three or more times – in parameter space, they fall outside the region of bistability. </P><H3>Conclusions</H3><P>The ideal scenario is to change the parameters to that in scenario 4 and have Hill coefficients of n<SUB style="font-size:10px">2</SUB>=2 and n<SUB style="font-size:10px">4</SUB>=5. Here 98.17% of the parameter combinations tested gave bistability. Contrary to our initial assumptions, depending on the range of parameters, higher Hill coefficients are not always better. The parameters will dictate the shape and position of the nullclines and whether or not bistability is possible. 

It is possible to determine the optimal values of the parameters for an ideal working of the switch. However, to change the parameters in the experimental construct so much would require much more time than we had available. </P><H3>References</H3><P><SUP style="font-size:10px">[1]</SUP> Hershey, J., Sonenberg, N., and Mathews, M. (2007), Origins and Principles of Translational Control. In Translational Control (pp.1-29). Cold Spring Harbor, NY: Cold Spring Harbor Press. </P><P><SUP style="font-size:10px">[2]</SUP> Edwards, A.M., Kane, C.M., Young, R.A., and Kornberg, R.D. (1991), Two Dissociable Subunits of Yeast RNA Polymerase II Stimulate the Initiation of Transcription at a Promoter in Vitro, <I>J. Biol. Chem.</I>, Vol. 266, No.1, pp.71-75 </P><P><SUP style="font-size:10px">[3]</SUP> Fürst, P., and Hamer, D. (1989), Cooperative activation of a eukaryotic transcription factor: Interaction between Cu(I) and yeast ACE1 protein, <I>Proc. Natl. Acad. Sci. USA</I>, Vol. 86, pp. 5267-5271 </P><P><SUP style="font-size:10px">[4]</SUP> Jensen, L.T., Posewitz, M.C., Srinivasan, C., and Winge, D.R. (1998), Mapping of the DNA binding domain of the copper-responsive transcription factor Mac1 from Saccharomyces cerevisiae, <I>J. Biol. Chem.</I>, Vol. 273, No. 37, pp.23805-23811 </P><P><SUP style="font-size:10px">[5]</SUP> Rodgers, K.K., and Coleman, J.E. (1994), DNA binding and bending by the transcription factors GAL4(62*) and GAL4(149*), <I>Protein Science</I>, Vol.3, No. 4, pp. 608-619 </P><P><SUP style="font-size:10px">[6]</SUP> Garcia-Martinez, J., Aranda, A., and Perez-Ortin, J.E. (2004), Genomic run-on evaluates transcription rates for all yeast genes and identifies gene regulatory mechanisms, <I>Mol. Cell.</I>, Vol. 15, No. 2, pp.303-313 </P><P><SUP style="font-size:10px">[7]</SUP> Wang, Y., Liu, C.L., Storey, J.D., Tibshirani, R.J., Herschlag, D., and Brown, P.O. (2002), Precision and functional specificity in mRNA decay, <I>Proc. Natl. Acad. Sci. USA</I>, Vol. 99, No.9, pp.5860-5865 </P><P><SUP style="font-size:10px">[8]</SUP> Beyer, A. et al. (2004), Post-transcriptional sxpression regulation in the Yeast Saccharomyces Cerevisiae on a genomic scale, <I>Molecular and Cellular Proteomics</I>, Vol. 3, No. 11, pp.1083-1092 </P><P><SUP style="font-size:10px">[9]</SUP> Mateus, C. and Avery, S.V. (2000), Destabilized green fluorescent protein for monitoring dynamic changes in yeast gene expression with flow cytometry, <I>Yeast</I>, Vol.16, No.14, pp.1313-1323 </P><P><SUP style="font-size:10px">[10]</SUP> Carey, J., Cameron, V., de Haseth, P.L., and Uhlenbeck, O.C. (1983), Sequence-specific interaction of R17 coat protein with its ribonucleic acid binding site, <I>Biochemistry</I>, Vol.22, No.11, pp.2601-2610 </P><P><SUP style="font-size:10px">[11]</SUP> Talbot, S.J., Goodman, S., Bates, S.R., Fishwick, C.W., and Stockley, P.G. (1990), Use of synthetic oligoribonucleotides to probe RNA-protein interactions in the MS2 translational operator complex, <I>Nucleic Acids Res.</I>, Vol.18, No.12, pp.3521-3528 </P><P><SUP style="font-size:10px">[12]</SUP> Arava, Y., Wang, Y., Storey, J.D., Liu, C.L., Brown, P.O., and Herschlag, D. (2003), Genome-wide analysis of mRNA translation profiles in Saccharomyces cerevisiae, <I>Proc. Natl. Acad. Sci. USA</I>, Vol. 100, No.7, pp.3889-3894 </P><P><SUP style="font-size:10px">[13]</SUP> Chattopadhyay, S., Garcia-Mena, J., DeVito, J., Wolska, K., and Das, A. (1995), Bipartite function of a small RNA hairpin in transcription antitermination in bacteriophage lambda, <I>Proc. Natl. Acad. Sci. USA</I>, Vol.92, No.9, pp.4061-4065 </P><TABLE class="nav"><TBODY><TR><TD><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Stochastic_Model">  Return to the Stochastic Model</A></TD><TD align="right"><A href="https://2010.igem.org/Team:Aberdeen_Scotland/Curve_Fitting">Continue to Determination of the Hill Coefficient n<SUB style="font-size:10px&quot;">2</SUB>  </A></TD></TR></TBODY></TABLE><P align="CENTER"><A href="#top">Back to the Top</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Aberdeen_Scotland/Probability">http://2010.igem.org/Team:Aberdeen_Scotland/Probability</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Aberdeen_Scotland/Probability" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Aberdeen_Scotland/Probability" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Aberdeen_Scotland/Probability&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Aberdeen_Scotland/Probability&amp;oldid=195481" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>