"
Page
Discussion
View source
History
teams
Log in
 
Team:Aberdeen Scotland/Project Overview
From 2010.igem.org
University of Aberdeen - ayeSwitch - iGEM 2010
Home
Our Project
Our Team
iGEM at Aberdeen
Team Members
Team Advisors
iGEM at Work
Official Team Profile
Biology
General Overview
DNA Constructs
Results
Protocols
Parts Submitted to the Registry
Modelling
Equations
Nullclines and Fixed Points
Bifurcation and Stability
Stochastic Model
Parameter Space Analysis
Determination of the Hill coefficient n2
Directed Evolution
Lab Diary
Switch Characterisation
Biobrick related
Safety
Safety Concerns
Safety Solutions
 
University of Aberdeen - ayeSwitch
iGEM 2010
Project Overview
Introduction
For this year’s iGEM competition the Aberdeen team has worked on developing a translationally controlled toggle switch embedded in yeast. [1] Genetic toggle switches are a vital component for synthetic biology circuits , enabling functional control of biological functions. The majority of toggle switches used for iGEM are embedded in Escherichia coli and can only be controlled at the transcriptional level [2] , [3] . Our main goal was to create and model a novel gene circuit, wherein yeast cells can be switched between mutually exclusive fluorescent proteins under exposure to environmental factors.  This switching behaviour would be regulated at the translational level, an innovation over previous systems that only demonstrated transcriptional regulation [4] , [5] .The novel genetic toggle switch operated by controlling gene expression at the translational level consisted of two gene expression constructs expressing an RNA-binding protein fused to either Green (GFP) or Cyan (CFP) fluorescent protein in the presence of appropriate inducer. When co-expressed in yeast, these translational fusions would be mutually inhibitory at the translational level, thereby forming a biological, ‘Toggle Switch’ system.
The AyeSwitch
The toggle switch is shown by Fig 1 and was named the ‘AyeSwitch’. It is regulated by controlling the two constructs, GAL1p-[Npeptide-GFP] and CUP1p-[MS2-CFP], via inducible yeast promoters GAL1 or CUP1 in the presence or absence of galactose and Cu2+ ions respectively.
For example, in the presence of galactose only, GAL1 is induced and there is expression of N-peptide-GFP protein. The subsequent addition of Cu2+ then induces the transcription of mRNA coding for MS2 coat binding protein and CFP. In addition to this, the mRNA also codes for a Bbox stem loop sequence that can be bound by N-peptide.
Ideally, there is initial inhibition of MS2-CFP translation by Npeptide-GFP binding to the Bbox stem loop. Evolution of time corresponds to the ratio of MS2-CFP mRNA to N-peptide-GFP protein increasing allowing some MS2-CFP to be produced until CFP ‘switches ON’ as it gains dominance over GFP.
Additionally, N-peptide-GFP protein translation can also be inhibited by MS2-CFP via MS2 protein binding to the MS2 stem loops on the N-peptide-GFP mRNA. This may help the switching ON of CFP and also means GFP would face a similar situation if the inducer was changed from Cu2+ to galactose.
However, additional variables may come into play affecting the outcomes described above. It is likely that the concentration of each inducer present, the translational rate and binding efficiency of stem loop binding proteins to mRNA stem loop and degradation rate of proteins can also affect the outcome. Reversing the order of inducer present may also affect the outcome.
Experimental Characterisation of the AyeSwitch
The experimental work addressed these issues by initially characterising the promoters in terms of their dose response and time response using constructs GAL1-[GFP] and CUP1-[GFP]. These experiments were then extended to characterise GAL1p-[Npeptide-GFP] and CUP1p-[MS2-CFP] which discovered that CUP1p-[MS2-CFP] did not function as expected.
The experimental work diverged from this point to troubleshoot CUP1p-[MS2-CFP], investigating the translation inhibition of GAL1p-[Npeptide-GFP] by MS2 coat protein using construct MET17p - [MS2], Bio-brick construction and testing of Bio-brick E2050 mOrange.
Modelling Characterisation of the Ayeswitch
Our team proposed a novel model to describe the functioning of the Aye-switch, based on ordinary differential equations (ODEs). The proposed system of ODEs was carefully and systematically studied both analytically and computationally. A bifurcation analysis was performed and the bistability of the system was investigated with respect to large variations in the parameters of the system. The deterministic simulations were compared with stochastic ones, using the Gillespie algorithm. The parameter space of the model was thoroughly investigated, using two different approaches: Monte-Carlo and directed evolution. These two approaches are very useful for a wide range of projects in synthetic biology. The theoretical predictions led to the  proposition of optimised parameters for the Aye-switch that allow a very robust translational switch.
Troubleshooting CUP1p-[MS2-CFP]
Troubleshooting of CUP1p-[MS2-CFP] was carried out through a series of gene cassette replacement experiments testing the promoter and CFP sequences for functionality. The conclusions to these experiments suggest that the Bbox Stem loop, usually located in the 3’untranslated region but is in the 5’ untranslated region of our construct may be preventing the expression of downstream proteins. It may also be that the fusion of MS2 to CFP results in inappropriate protein folding, inhibiting expression.
Verification of Translation Inhibition as a Regulatory Mechanism
It was shown that the translational inhibition of GAL1p-[Npeptide-GFP] by MS2 coat protein was possible, confirming that translational regulation is viable. Further work if time permitted would investigate if this inhibition could work in the context of a toggle switch.
Bio-brick construction and testing
In parallel, Bio-bricks were constructed and submitted to the Registry of parts whilst testing of the Bio-brick E2050 mOrange using fluorimetry and FACS analysis lead to the conclusion that the mOrange sequence did not function within our GAL1p-[Npeptide-GFP] construct that was shown to be able to express GFP appropriately.
Attribution and Contributions
Biological circuit construction and testing
The students within the experimental section of the team were provided (by their host lab) with two yeast strains that had Gal1p-GFP and Cup1p-GFP integrated into the genome (see 'DNA constructs). They then used these constructs to analyse the properties of the CUP1 and GAL1 promoters. With some instructor oversight, the student team themselves then completely designed constructs Gal1p-(Npep-GFP) and Cup1p-(MS2-CFP), which were then synthesised by a synthetic DNA supply company. The students then tested these constructs, and further engineered them during the trouble-shooting phase of the project.
All the experimental work described on the wiki, involving characterisation, testing and re-engineering of the bio-bricks, was carried out by the student members of the team. All the construction and sequencing of the four submitted bio-bricks was also carried out by members of the student team.
Mathematical modelling of the AyeSwitch
The students within the theoretical section of the team carried out all the described modelling. Team activities were overseen by the Instructors, but all model coding and model analysis was performed by the students within the team.
References
[1] Ernesto Andrianantoandro et al. Synthetic biology: new engineering rules for an emerging discipline Molecular Systems Biology 2:2006.0028
[2] Timothy S. Gardner et al. Construction of a genetic toggle switch in Escherichia coli Nature 403, 339-342 (20 January 2000)
[3] Mariette R. Atkinson et al. Development of Genetic Circuitry Exhibiting Toggle Switch or Oscillatory Behavior in Escherichia coli Cell, Volume 113, Issue 5, 597-607, 30 May 2003
[4] Adam Platt and Richard J Reece The yeast galactose genetic switch is mediated by the formation of a Gal4p–Gal80p–Gal3p complex The EMBO Journal (1998) 17, 4086 - 4091
[5] D W Griggs and M Johnston Regulated expression of the GAL4 activator gene in yeast provides a sensitive genetic switch for glucose repression PNAS October 1, 1991 vol. 88 no. 19 8597-8601
  Return to Home Page
Continue to iGEM at Aberdeen  
Back to the Top
Retrieved from " http://2010.igem.org/Team:Aberdeen_Scotland/Project_Overview "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
