<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Nagahama_project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Nagahama_project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Nagahama_project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Nagahama_project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Nagahama_project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Nagahama_project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Nagahama project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TABLE id="menu" width="100%" cellspacing="0" height="135px"><TBODY><TR height="50px"><TD width="162px" align="center" onmouseover="this.bgColor='#FEE5AD'" onmouseout="this.bgColor='#A1DBB2'" bgcolor="#A1DBB2"><A href="https://2014.igem.org/Team:Nagahama" style="text-decoration:none;color:#1C140D">
HOME
</A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#FEE5AD'" onmouseout="this.bgColor='#A1DBB2'" bgcolor="#A1DBB2"><A href="https://2014.igem.org/Team:Nagahama_project" style="text-decoration:none;color:#1C140D">
PROJECT
</A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#FEE5AD'" onmouseout="this.bgColor='#A1DBB2'" bgcolor="#A1DBB2"><A href="https://2014.igem.org/Team:Nagahama_Parts" style="text-decoration:none;color:#1C140D">
PARTS
</A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#FEE5AD'" onmouseout="this.bgColor='#A1DBB2'" bgcolor="#A1DBB2"><A href="https://2014.igem.org/Team:Nagahama_team" style="text-decoration:none;color:#1C140D">
TEAM 
</A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#FEE5AD'" onmouseout="this.bgColor='#A1DBB2'" bgcolor="#A1DBB2"><A href="https://2014.igem.org/Team:Nagahama_Policy_&amp;_Practice" style="text-decoration:none;color:#1C140D">
Policy &amp; Practice
</A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#FEE5AD'" onmouseout="this.bgColor='#A1DBB2'" bgcolor="#A1DBB2"><A href="https://2014.igem.org/Team:Nagahama_safety" style="text-decoration:none;color:#1C140D">
SAFETY
</A></TD></TR></TBODY></TABLE></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Abstract"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Our_Project"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Our Project</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Modeling"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Modeling</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Method"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Medium"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Medium</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Trypton_broth"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">Trypton broth</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Wash_medium"><SPAN class="tocnumber">4.1.2</SPAN><SPAN class="toctext">Wash medium</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Chemotaxis_medium"><SPAN class="tocnumber">4.1.3</SPAN><SPAN class="toctext">Chemotaxis medium</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Synthesis_medium"><SPAN class="tocnumber">4.1.4</SPAN><SPAN class="toctext">Synthesis medium</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#LB_medium"><SPAN class="tocnumber">4.1.5</SPAN><SPAN class="toctext">LB medium</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#2.C3.97YT_medium"><SPAN class="tocnumber">4.1.6</SPAN><SPAN class="toctext">2×YT medium</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#M9_swarming_agar"><SPAN class="tocnumber">4.1.7</SPAN><SPAN class="toctext">M9 swarming agar</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#5_.C3.97_M9_stock_solution"><SPAN class="tocnumber">4.1.8</SPAN><SPAN class="toctext">5 × M9 stock solution</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-14"><A href="#Aspartate_synthesis"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Aspartate synthesis</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#SDS_PAGE"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#TLC_assay"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">TLC assay</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Chemotaxis_Assay"><SPAN class="tocnumber">4.5</SPAN><SPAN class="toctext">Chemotaxis Assay</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Introduction"><SPAN class="tocnumber">4.5.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Swarming_Assay"><SPAN class="tocnumber">4.5.2</SPAN><SPAN class="toctext">Swarming Assay</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Capillary_Assay"><SPAN class="tocnumber">4.5.3</SPAN><SPAN class="toctext">Capillary Assay</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#Result_.26_Discussion"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Result &amp; Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#TLC_Assay_2"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">TLC Assay</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#SDS_PAGE_2"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Swarming_Assay_2"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Swarming Assay</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Capillary_Assay_2"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Capillary Assay</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#Future_work"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Future work</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#Reference"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Reference</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Abstract">Abstract</SPAN></H1><P>Cadmium is the heavy metal which is very harmful to a creature.　The pollution of cadmium brings a serious problem for natural environments. It is very important to collect this efficiently. We tried to solve this problem with recombination <I>Escherichia coli</I> which our team made in this year. 
We decided to make two kinds of recombination <I>Escherichia coli</I>. One traps cadmium on the outer membrane. It produces fusion protein with the heavy metal binding protein and outer membrane protein and it traps cadmium. Another collects other <I>Escherichia coli</I>. It produces aspartic acid of positive chemotaxis factor to <I>Escherichia coli</I>.
</P><P>Is it necessary to distribute it to two kinds of why? We suggest the following theory for this question.
</P><P><FONT size="3">&quot;One <I>E. coli</I> has One function theory&quot;</FONT></P><P>There is the possibility that the following demerit occurs when I summarize two functions in one.The first is that the ability of the <I>Escherichia coli</I> situation decreases by producing multiple protein at a time.The second is that subsequent editing becomes difficult when we complete a  gene cassette once. The merit in case of one kind is next. 
At first it is that a function becomes very simple.We can make <I>Escherichia coli</I> which was specialized in it by a function becoming simple. The number of Escherichia coli just leads to the level of the function, analysis becomes easy.
</P><H1><SPAN class="mw-headline" id="Our_Project">Our Project</SPAN></H1><P>We make various systems by cell-cell communication. We keep one function in one <I>E. coli</I>.  This means to make simple plasmid. The following is one example. We’d like to collect cadmium in water. Therefore we use two kinds of E.coli. One catches Cadmium. The other attracts all <I>E.coli</I> by using chemoattractant. Catches <I>E.coli</I> displays metallothionein a protein combines a heavy metal. Cadmium is a kind of heavy metal. The other synthesizes aspartic acid (Asp) one kind of chemoattractant. All E.coli gather in the <I>E. coli</I> synthesizes Asp. To use these E.coli, finally cadmium will be caught.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">When we collect cadmium in the water, if use only catching <I>E. coli</I>(Blue),they go away from cadmium. Because they have negative chemotaxisin contrast with cadmium. Consequently, they don't catch cadmium very much.[fig1-A] In constract, if we use two kind of recombinant <I>Escherichia coli</I>(<I>E. coli</I>);gathering <I>E. coli</I>(Red) and catching <I>E. coli</I>(Blue), we can build the creature system which collects cadmium while sitting at cadmium by using two kinds of <I>Escherichia coli</I>.[fig1-B]   It is this theoretical merit that we can suggest two kinds of possibility that meeting condition and collection quantity of <I>Escherichia coli</I> are changed by changing the ratio of <I>Escherichia coli</I> in addition to cadmium solution.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">We constracted plasmid that synthesize aspartic acid in the presence of cadmium. Part's namber are BBa_K896008(promoter),(RBS),BBa_C0083(Protein coding sequences),(Double terminator).  BBa_K896008(zinTp) is a promoter inducing gene expression only in existence of cadmium. AspA mainly catalyzes the deamination of L-aspartic acid to produce fumarate and ammonium ion. As a result of having cultured it in the nutrient medium including the substrate, we confirmed synthesis of aspartic acid by thin layer chromatography.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">We are constracting this plasmid now. Recombination Escherichia coli having this plasmid becomes the thing with the function to trap cadmium on outer membrane if construction of this plasmid is completed. It is fusion protein with metallothionine(MT);SmtA and adventitia protein;Ag43. The genetic origin of SmtA is Synechococcus PCC. 7942.</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Modeling">Modeling</SPAN></H1><P>We consigned modeling of our project to
<A href="https://2014.igem.org/Team:UT-Tokyo" class="external text" rel="nofollow">UT-Tokyo</A>.They readily compiled with our requests. Their model is ideal.We really appreciate their jobs We wish a friendly relationship with UT-Tokyo.Thank you so much!! The detail of their jobs are lists below!!
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:642px;"><DIV class="thumbcaption">Fig.1 This is random walk of <I>Escherichia coli</I> on &quot;swarming plate&quot;.The vertical axis means arbitrary unit.The horizontal axis means to spatial axis. We culture <I>E.coli</I>on &quot;swarming plate&quot;. &quot;swarming plate&quot;, a solid culture plate containing lower concentration of agar, on that plate,<I> E. coli</I> can swim.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:642px;"><DIV class="thumbcaption">Fig.2 This is positive chemotaxis by <I>E.coli</I>.Assuming that spoted aspartic acid center of &quot;swarming plate&quot;. <I> E.coli</I> has positivechemotaxis for aspartic acid. <I>E.coli</I> gather to spoted point.</DIV></DIV></DIV></DIV>.
<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:642px;"><DIV class="thumbcaption">Fig.3 This is negativechemotaxis by <I>E.coli</I>.Assuming that spoted cadmium center of &quot;swarming plate&quot;. <I>E.coli</I> has negaitivechemotaxis for cadomium. <I>E.coli</I> run away from spoted point.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:642px;"><DIV class="thumbcaption">Fig.4 This model describes our project. Assuming cadmium was spotted on center of swarming plate (spatial axis 500). <I>E.coli</I> run away from spotted point (spatial axis 350~450 and 550~650). Then cadmium promoter of transgenic　<I>E.coli</I> operates. So aspartase produced (spatial axis 350~450 and 550~650).</DIV></DIV></DIV></DIV>.
<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Meｍberｓ of UT-Tokyo</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Method">Method</SPAN></H1><H2><SPAN class="mw-headline" id="Medium">Medium</SPAN></H2><H3><SPAN class="mw-headline" id="Trypton_broth">Trypton broth</SPAN></H3><P>Trypton                10g/L
</P><P>NaCl                   10g/L
</P><P>H2O                    1L
</P><H3><SPAN class="mw-headline" id="Wash_medium">Wash medium</SPAN></H3><P>Potassium phosphate buffer (pH 7.0), 10⁻²M
</P><P>MgSO₄, 10⁻³M
</P><P>potassium ethylenediaminetetraacetate (EDTA), 10⁻⁴M
</P><H3><SPAN class="mw-headline" id="Chemotaxis_medium">Chemotaxis medium</SPAN></H3><P>Potassium phosphate buffer (pH 7.0), 10⁻²M
</P><P>potassium EDTA, 10⁻⁴M
</P><P>L-methionine 10⁻⁶M
</P><H3><SPAN class="mw-headline" id="Synthesis_medium">Synthesis medium</SPAN></H3><P>Sodium hydrogen fumarate 	 46g/L
</P><P>Ammonium chloride 		17.8g/L
</P><P>Magnesium sulfate 7 hydrate	0.25g/L
</P><P>H₂O                                1L
</P><P>Ph8.5with sodium hydrogen
</P><H3><SPAN class="mw-headline" id="LB_medium">LB medium</SPAN></H3><P>Tryptone		10g/L
</P><P>Yeast extract 	 5g/L
</P><P>NaCl		10g/L
</P><P>(agar		15g/L)
</P><P>H₂O             1L
</P><H3><SPAN class="mw-headline" id="2.C3.97YT_medium">2×YT medium</SPAN></H3><P>Tripton		16g/L
</P><P>Yeast extract 	10g/L
</P><P>NaCl		 5g/L
</P><P>(agar		15g/L)
</P><P>H₂O             1L
</P><H3><SPAN class="mw-headline" id="M9_swarming_agar">M9 swarming agar</SPAN></H3><P>M9 salt
</P><P>1.25 % (v/v) glycerol
</P><P>0.3 % (w/v) agar
</P><P>0.1 % (w/v) CaCl₂ solution (after autoclave)
</P><P>0.1 % (w/v) MgSO₄ solution (after autoclave)
</P><P>0.03 % (w/v) thiamine (after autoclave)
</P><H3><SPAN class="mw-headline" id="5_.C3.97_M9_stock_solution">5 × M9 stock solution</SPAN></H3><P>6.4 % (w/v) Na<SUB>2</SUB>HPO<SUB>4</SUB></P><P>1.5 % (w/v) KH<SUB>2</SUB>PO<SUB>4</SUB></P><P>0.25 % (w/v) NaCl
</P><P>0.5 % (w/v) NH<SUB>4</SUB>Cl
</P><H2><SPAN class="mw-headline" id="Aspartate_synthesis">Aspartate synthesis</SPAN></H2><P>E.coli K12 transformed with CdP-R.B.S-AspA-d.Ter (BBa_K1342001) previous cultured with cadmium in LB medium (250μM) in 37℃ for 12hr at 120rpm. Adjust Cell mass (OD1.0) and therefor centrifuged 4000rpm for 20 min. Cell pellets ware activated in synthesis medium in 37℃ for 2hr at 120rpm/min.
</P><H2><SPAN class="mw-headline" id="SDS_PAGE">SDS PAGE</SPAN></H2><P>・E. coli JM109 containing CdP-R.B.S-AspA-d.Ter (BBa_K1342001) or normal E. coli was precultured at 37 ℃ for 24 hr with shaking.
</P><P>・The culture (OD600 0.6-1.0) was used for induction of protein expression by isopropylthio-β-D-galactoside 1mM and Cd2+ 100µM soln. 37℃、o hr, 0.5 hr, 2hr, 6 hr, 24hr.
</P><P>・200 μl of the induced culture was centrifuge at 13,000 rpm for 1 min at 4 ℃.
</P><P>・The supernatant was discarded and the pellet was stored at -20 ℃.
</P><P>・The stored pellet was resuspended in 1 x sample buffer (100 μL), followed by heating at 98 °C for 5 min.
</P><P>・The lysate was centrifuged at 13,000 rpm for 10 min. at 4℃.
</P><P>・The supernatant was collected into a new microcentrifuge tube.
</P><P>・20 μL of the supernatant was analyzed by SDS-PAGE.
</P><H2><SPAN class="mw-headline" id="TLC_assay">TLC assay</SPAN></H2><P>We analyzed L-aspartate by thin-layer chromatography (TLC). The equal volume of the supernatant of synthesis medium was added with 7 mg/mL 5-Dimethylamino naphthalene-1-sulfonyl Chloride (dissolved in acetone) and was incubated for more than 30 min at room temperature. Two microliters of the reactant was spotted on a TLC silica plate, and was developed in a mixture of ethyl acetate, pyridine, water, and acetic acid(162：21：11：6 v/v). It is observed by UV irradiation (365nm).
</P><H2><SPAN class="mw-headline" id="Chemotaxis_Assay">Chemotaxis Assay</SPAN></H2><H3><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H3><P>Chemotaxis of <I>E. coli</I> against aspartic acis was assayed by two methods. One was &quot;swarming assay&quot;, and the other was capillary assay. In &quot;swarming assay&quot;, we have used &quot;sawarming plate&quot;, a solid culture plate containing lower concentration of agar, on that plate,<I> E. coli</I> can swim. We have used capillary  containing aspartic acid. The capillary was set in chemotaxis medium containing motile <I>E. coli</I>. The number of <I>E. coli</I> attracted to aspartic acid in capillary was determined  by colony formation on LB plate.
</P><H3><SPAN class="mw-headline" id="Swarming_Assay">Swarming Assay</SPAN></H3><P>We assayed chemotaxis of <I>E. coli</I> against cadmium and aspartic acid on soft agar, containing M9 synthetic medium, on that plate <I>E. coli</I> can swim.</P><P>Protocol</P><P>1.<I>E. coli</I> JM109 was cultured at 30℃ for 12 hours with shaking (50 rpm).
</P><P>2.	Aliquot of the culture was spotted on the center of agar plate.
</P><P>3.	10 mM L-aspartic acid (40 μl) or 100 mM cadmium chloride (4 μl) was spotted on 25 mm distant from the center of the agar plate.
</P><P>4.	The plate was standed for 5 min at RT.
</P><P>5.	 The plate was incubated at 30℃.
</P><H3><SPAN class="mw-headline" id="Capillary_Assay">Capillary Assay</SPAN></H3><P>Object
</P><P>Assay the chemotaxis of <I>E. coli</I> against aspartic acid with capillary.
</P><P>Plotocol
</P><P>1.	Preculture <I>E.　coli</I> in tripton broth at 30 ℃ for 12  hr with shaking (50 rpm).
</P><P>2.	Check the motility of <I>E. coli</I> by microscope (× 800).
</P><P>3.	Dilute the <I>E. coli</I> culture (200 μL) with tripton broth (20 mL).
</P><P>4.	Incubate at 30 ℃ with shaking (50 rpm) until log phase (OD600 = ～ 0.2).
</P><P>5.	Aliquote (500 μL) of the culture was centrifuged at 25℃ for 10 min at 3400 x G.
</P><P>6.	Dicard the supernatant.
</P><P>7.	Resuspend the pellets gently with wash medium (50 μL).
</P><P>8.	Repeat 5~7 step.
</P><P>9.	Resuspend the pellets gently with chemotaxis medium (500 μL).
</P><P>10.	Check the motility of <I>E. coli</I> by microscope (× 800).
</P><P>11.	Aliquote (100 μL) of <I>E. coli</I> suspended in chemotaxis medium was set into chamber apparatus.
</P><P>12.	Prepare aspartic acid capillary and negative control capillary that contain 1 μL of chemotaxis medium with and without 10 mM aspartic acid, respectively.
</P><P>13.	Both capillaries (asparate capillary and negative control capillary) were set into chamber apparatus.
</P><P>14.	Incubate at 30 ℃ for 90 mim.
</P><P>15.	The chemotaxis medium in capillary was collected in 100 μL of fresh chemotaxis medium.
</P><P>16.	All chemotaxis medium containing attracted <I>E. coli</I> was plated on LB plate.
</P><P>17.	Incubate at 37 ℃ for 12 hr.
</P><P>18.	Count the number of colony.
</P><H1><SPAN class="mw-headline" id="Result_.26_Discussion">Result &amp; Discussion </SPAN></H1><H3><SPAN class="mw-headline" id="TLC_Assay_2">TLC Assay</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Analysis of aspartic acid synthesized by <I>E. coli</I> JM109/<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1342001" class="external text" rel="nofollow">BBa_K1342001</A> by thin-layer chromatography (TLC) Lane: 1, 2 no addition into culture (diluted 1:50 and 1:100 by synthesis medium, respectively) 3, additon of cadmium ion (250μM) 4, 5, 6 (diluted 1:50, 1:100 and 1:200 by synthesis medium, respectively)</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">fig.2   Estimation of aspartic acid content in spot Each intensity of spots indicating the content of aspartic acid, was estimated by Image J. .</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="SDS_PAGE_2">SDS PAGE</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Analyzed aspartase by <I>E. coli</I> JM109 and <I>E. coli</I> JM109/<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1342001" class="external text" rel="nofollow">BBa_K1342001</A> by SDS-PAGE ,37℃ for 0,6 hr Lane:1,10 Protein Marker(Precision Plus Protein Standards (BIO-RAD)) Lane:2,3 <I>E. coli</I> JM109  Lane:4,5 <I>E. coli</I> JM109/BBa_K1342001  Lane:6,7 <I>E. coli</I> JM109/BBa_K1342001 (Additon of cadmium ion 100μM) Lane:8,9 <I>E. coli</I> JM109/BBa_K1342001 (Additon of cadmium ion 100μM and IPTG 1mM) respectively. OD600=0.74 CdCl2 100μM/IPTG 1mM AspA has 1482bp. It is about 54kDa.</DIV></DIV></DIV><P>
Aspartase synthesize was not detect by CBB stain.
</P><H3><SPAN class="mw-headline" id="Swarming_Assay_2">Swarming Assay</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:414px;"><DIV class="thumbcaption">Fig.1 <I>E. coli</I> swarming to aspartic acid  L: aspartic acid (10 mM, 40 μl) R: ddH<SUB>2</SUB>O (40 μl)Incubation: 108 hours  temperature: 30 ℃ <I>E. coli</I> cultured was spotted on the center of agar plate, and chemotaxis was spotted on 25 mm distant from the center.               The swarming circle of aspartic acid (L) was larger than that of ddH<SUB>2</SUB>O (R), and the swarming area around aspartate acid (L) was more larger than that of ddH<SUB>2</SUB>O (R), suggesting that E. coli swarming to aspartate acid. This experiment indicate that <I>E. coli</I> may have positive chemotaxis for aspartic acid.</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:430px;"><DIV class="thumbcaption">Fig.2 <I>E. coli</I> swarming to cadmium  L: cadmium(100 mM, 4 μl) R: ddH<SUB>2</SUB>O (40 μl)Incubation: 110 hours  temperature: 30 ℃ <I>E. coli</I> cultured was spotted on the center of agar plate, and chemotaxis was spotted on 25 mm distant from the center.  The swarming circle of cadmium (L) was smaller than that of ddH<SUB>2</SUB>O (R), and the swarming area around cadmium (L) was more smaller than that of ddH<SUB>2</SUB>O (R), suggesting that <I>E.</I><I>coli</I> hate cadmium.This experiment indicate that <I>E. coli</I> may have negative chemotaxis for cadmium.</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Capillary_Assay_2">Capillary Assay</SPAN></H3><P>Result
</P><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig.1 Left : Aspartic acid, Right : controlThere were 13 colonies on Asp plate. One other plate has 1326 colonies.</DIV></DIV></DIV><P>Discussion
</P><P>At this method. We didn't assay aspartic acid cehmotaxis of <I>E.coli</I>. We did eleven assays.In this assay,　<I>E.coli</I> formation will appear on aspartic acid plate more than conrol plate. But at result of Fig.1,<I> E.coli</I> colonies appeared control plate more than aspartic acid plate. And all result were not authenticity.
There are some problems. One is heat of bacteria spreader. At plating moments. I may burn  'E.coli<I> to death. Next is diameter of capillary. I made capillary by myself.</I>
There were various sizes of capillary. I used their capillary. So colonies formation were ununiformity. Last is the number of times of assays. In our source journal (Reference No.7), they assayed more then us.
</P><H1><SPAN class="mw-headline" id="Future_work">Future work</SPAN></H1><P>We performed a project to collect cadmium using chemotaxis of Escherichia coli this year. But the second plasmid is still building it. We want to perform an experiment such as fig.1-B with two kinds of Escherichia coli as soon as construction is completed. In addition, the chemotactic instruction and control that we used have the applied nature to other projects very much. We want to create the new project using chemotaxis of Escherichia coli.
</P><H1><SPAN class="mw-headline" id="Reference">Reference</SPAN></H1><P>[1]<A href="http://www.ncbi.nlm.nih.gov/pubmed/12480884" class="external text" rel="nofollow">Ferianc, P, et al, Regulation of yodA encoding a novel cadmium-induced protein in Escherichia coli, Microbiology, vol.148, 3801–3811, (2002)</A>
[2]<A href="https://2012.igem.org/Team:Goettingen/Notebook/Results" class="external text" rel="nofollow">iGEM Goettingen, Homing Coli, https://2012.igem.org/Team:Goettingen/Notebook/Results, (2012)</A>
[3]<A href="http://www.ccbi.cam.ac.uk/iGEM2006/index.php/Protocols#Autoclaving_stock_solutions_or_glassware_.28sterilisation.29" class="external text" rel="nofollow">iGEM University Of Cambridge, Making swimming Agar, http://www.ccbi.cam.ac.uk/iGEM2006/index.php/Protocols#Autoclaving_stock_solutions_or_glassware_.28sterilisation.29, (2006)</A>
[4]<A href="https://2011.igem.org/Team:UT-Tokyo/Data/Method" class="external text" rel="nofollow">iGEM UT-Tokyo, L-aspartate chemotaxis assay, https://2011.igem.org/Team:UT-Tokyo/Data/Method, (2011)</A>
[5]<A href="http://ecocyc.org/" class="external text" rel="nofollow">Keseler et al, EcoCyc, http://ecocyc.org/, (2013)</A>
[6]<A href="http://www.ncbi.nlm.nih.gov/pubmed/17650676" class="external text" rel="nofollow">Chmiel, A, Selection and activation of Escherichia coli strains for L-aspartic acid biosynthesis, Polish J. Microbiol, Vol.56, 71-76, (2007)</A>
[7]<A href="http://www.ncbi.nlm.nih.gov/pubmed/4632978" class="external text" rel="nofollow">ADLER, J, A method for measuring chemotaxis and use of the method to determine optimum conditions for chemotaxis by Escherichia coli, Polish J. General Microbiol, Vol.74, 77-91, (1973)</A>
[8]<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC106306/" class="external text" rel="nofollow">Sternberg, C, et al, New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria, Polish J. Microbiol, Vol.64, 2240–2246, (1998)</A>
[9]<A href="https://2010.igem.org/Team:UPO-Sevilla/Project/Assays" class="external text" rel="nofollow">iGEM UPO-Sevilla, Bacterial crowding, https://2010.igem.org/Team:UPO-Sevilla/Project/Assays, (2010)</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Nagahama_project">http://2014.igem.org/Team:Nagahama_project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Nagahama_project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Nagahama_project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Nagahama_project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Nagahama_project&amp;oldid=386585" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>