<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UPF_CRG_Barcelona_Overexpression skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UPF CRG Barcelona/Overexpression</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Wiki</TITLE></P><DIV class="sticky-top"><NAV class="navbar has-shadow" role="navigation" aria-label="main navigation" id="navbar"><DIV class="container"><DIV class="navbar-menu" id="navbar-menu"><DIV class="navbar-grid"><A class="navbar-item is-active" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Description">Project</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/DryLab">DryLab</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Parts">Parts</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Deliverables">Deliverables</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/WetLab">WetLab</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Intro_HP">Human Practices</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Aboutteam">Team</A></DIV></DIV></DIV></NAV></DIV><MAIN><SECTION class="container"><DIV><P class="page-title">Enhancing <I>E. Coli</I> beta-oxidation</P><P>In order to obtain a system with an enhanced long chain fatty acid (LCFA) uptake and degradation we first
                  considered the overexpression of endogenous <I>E. coli</I> fatty acid (FA) degradation gene family.
               </P><P>
                  LCFAs are essential membrane components and constitute important members of signalling pathways. <I>E. coli</I>
                  can use FAs with various chain lengths as sole source of carbon and energy [1]. After their uptake, these FAs
                  can either be degraded via the beta-oxidation pathway or used as precursors for membrane phospholipid
                  biosynthesis. The degradation pathway is catalysed by the enzymes encoded by the fad regulon, which includes
                  <B>fadL</B>, <B>fadD</B>, <B>fadE</B>, <B>fadA</B> and <B>fadB</B>[2].
               </P><P><B>fadL</B> and <B>fadD</B> are responsible for LCFA uptake and activation for their previous metabolism.
                  FadL is an outer membrane homodimeric channel involved in the transport of external LCFA to the periplasm
                  [3]. To metabolize FAs, they must be activated to acyl-CoA esters. This step is performed through <B>FadD</B>,
                  an inner membrane-associated acyl-CoA synthase [4]. Finally, the complete oxidation of activated LCFA is
                  catalysed by the other three fatty-acid degradation gene family: <B>fadE</B>, <B>fadA</B> and <B>fadB</B>.
               </P><CENTER><P class="fig-caption"><B>Figure 1 | Scheme representing fatty acid intracellular uptake and degradation</B> Both
                     unsaturated and saturated fatty acids can enter inside the cell through FadL, an outer membrane channel.
                     Once in the
                     periplasm, FadD allows the transport of this fatty acids inside the cytosol using a system coupled to
                     acyl-coA ester formation. Finally, the beta-oxidation process starts under catalysis of FadE and FadH
                     enzymes, followed by FadA and FadB, both of them forming a tetrameric complex which is responsible of
                     hydration, oxidation and thiolytic cleavage.
                  </P></CENTER><P>Fatty acid degradation and biosynthesis pathways must have a very dynamic expression system according to the
                  availability of FAs in the environment to maintain cell functionality [5]. Thus, fad genes expression depends
                  directly on the presence of internal LCFA bound to acyl-CoA. 
               </P><P>FadR protein is the main transcriptional regulator of fad genes and is the key element regulating this
                  process [6, 7]. It is involved in the negative regulation of LCFA degradation; in the absence of FAs it is
                  constitutively bound to the fad genes promoter, repressing its transcription. In the presence of LCFA in the
                  medium conversion to acyl-coA esters occurs. Then, LCFA-CoA bind to <B>FadR</B>, causing its release from
                  the promoters and allowing the transcription of fad genes [8].
               </P><CENTER><P class="fig-caption"><B>Figure 2 |</B> The pfadBA promoter is constitutively repressed by FadR protein. LCFA enter
                     the cell through the outer membrane receptor fadL, then, they get bound to acety-CoA. LCFA-acetyl-CoA is able
                     to bind to FadR which allows it to be released from the promoter making the later active
                  </P></CENTER><P>FadL and FadD play a key role in the regulation of this pathway, as they are the main responsible of the
                  uptake of LCFA and its proper conversion to acyl-coA ester in the cytosol. We hypothesized that
                  overexpressing FadL and FadD, in presence of extracellular LCFA, would produce an scenario in which LCFA-CoA
                  would repress FadR, resulting in an enhanced expression of the rest of the fad genes. Thus, FadL and FadD
                  were targeted in order to develop a system with an increased LCFA degradation metabolism.
               </P><P>To do so, we expressed the sequences of either FadD or FadL downstream the TetR repressible promoter
                  (BBa_R0040). Thus, allowing to tune the protein expression levels with the addition of Anhydrotetracycline
                  (ATC).
               </P><CENTER><P class="fig-caption"><B>Figure 3 | Schematic representation of the fad gene construct. </B> Scheme representing
                     the inducible expression system of the fad genes. A double terminator (BBa_B0014), a weak rbs
                     (BBa_B0032), the inducible promoter (BBa_R0040) and the target gene were coupled in this biobrick.
                  </P></CENTER><P>Growth assays were performed in order to study the metabolic burden caused by the expression of these
                  constructs. Moreover, in order to analyse the functionality of these genetic units, palmitic acid in the
                  medium was quantified using cupric-acetate. Therefore, an increase of LCFA uptake when overexpressing these
                  two fad genes could be measured. 
     </P><P>The main approach of our project relies on an effective plasmid overexpression system. However, plasmidic
                  presence within a live therapeutic in the intestine could deem not so effective, by posing a threat to the
                  patients’ health due to the spread of antibiotic resistance to the natural gut ecosystem by plasmid
                  transmission from our probiotic. In order to solve this, we approached overexpression of the mentioned genes
                  from a different perspective; by directly modifying the <I>E. coli</I> genome as explained in the <A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Integration" alt="Integration Page">integration section</A>. 
               </P><P class="subapart2">References:</P><P class="references">[1] Iram, S. H., &amp; Cronan, J. E. (2006). The β-oxidation systems of Escherichia coli and
                  Salmonella enterica are not functionally equivalent. Journal of bacteriology, 188(2), 599-608.
               </P><P class="references">[2] Fujita, Yasutaro, Hiroshi Matsuoka, and Kazutake Hirooka. &quot;Regulation of fatty acid
                  metabolism in bacteria.&quot; Molecular microbiology 66.4 (2007): 829-839.
               </P><P class="references">[3] Lepore BW, Indic M, Pham H, Hearn EM, Patel DR, van den Berg B: Ligand-gated
                  diffusion across the bacterial outer membrane. Proc Natl Acad Sci U S A 2011, 108:10121–10126.
               </P><P class="references">[4] Weimar JD, DiRusso CC, Delio R, Black PN: Functional role of fatty acyl-coenzyme A
                  synthetase in the transmembrane movement and activation of exogenous long-chain fatty acids. Amino acid
                  residues within the ATP/AMP signature motif of Escherichia coli FadD are required for enzyme activity and
                  fatty acid transport. J Biol Chem 2002, 277:29369–29376.
               </P><P class="references">[5] Janßen, H. J., &amp; Steinbüchel, A. (2014). Fatty acid synthesis in Escherichia coli and
                  its applications towards the production of fatty acid based biofuels. Biotechnology for biofuels, 7(1), 7.
               </P><P class="references">[6] Henry MF, Cronan JE Jr: Escherichia coli transcription factor that both activates
                  fatty acid synthesis and represses fatty acid degradation. J Mol Biol 1991, 222:843–849. 
               </P><P class="references">[7] Henry MF, Cronan JE Jr: A new mechanism of transcriptional regulation: release of an
                  activator triggered by small molecule binding. Cell 1992, 70:671–679.
               </P><P class="references">[8] Feng Y, Cronan JE Jr: Crosstalk of Escherichia coli FadR with global regulators in
                  expression of fatty acid transport genes. PLoS One 2012, 7:e46275.
               </P></DIV></SECTION></MAIN><FOOTER class="footer"><DIV class="container"><DIV><P class="subapart2" style="font-weight: 800; font-size: 1.2rem">Follow us at:</P></DIV><DIV><DIV style="text-align: center !important;"><P class="subapart2" style="font-weight: 800; font-size: 1.2rem; text-align: center !important;">For Judges</P></DIV><DIV id="judges"><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Attributions">Attributions</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/InterLab">InterLab</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Improve">Improved part</A><A href="https://igem.org/2018_Judging_Form?id=2581">Judging Form</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Model">Model</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Parts">Parts</A></DIV></DIV><DIV class="footer-contact"><P class="subapart2" style="font-weight: 800; font-size: 1.2rem">Contact us:</P><P>igem [at] upf [dot] edu</P><P> PRBB Building</P><P>Doctor Aiguader 88, 08003 Barcelona (Spain)</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>