Team:UPF CRG Barcelona/Overexpression
Wiki
Project DryLab Parts Deliverables WetLab Human Practices Team
Enhancing E. Coli beta-oxidation
In order to obtain a system with an enhanced long chain fatty acid (LCFA) uptake and degradation we first                   considered the overexpression of endogenous E. coli fatty acid (FA) degradation gene family.
LCFAs are essential membrane components and constitute important members of signalling pathways. E. coli can use FAs with various chain lengths as sole source of carbon and energy [1]. After their uptake, these FAs                   can either be degraded via the beta-oxidation pathway or used as precursors for membrane phospholipid                   biosynthesis. The degradation pathway is catalysed by the enzymes encoded by the fad regulon, which includes fadL, fadD, fadE, fadA and fadB[2].
fadL and fadD are responsible for LCFA uptake and activation for their previous metabolism.                   FadL is an outer membrane homodimeric channel involved in the transport of external LCFA to the periplasm                   [3]. To metabolize FAs, they must be activated to acyl-CoA esters. This step is performed through FadD,                   an inner membrane-associated acyl-CoA synthase [4]. Finally, the complete oxidation of activated LCFA is                   catalysed by the other three fatty-acid degradation gene family: fadE, fadA and fadB.
Figure 1 | Scheme representing fatty acid intracellular uptake and degradation Both                      unsaturated and saturated fatty acids can enter inside the cell through FadL, an outer membrane channel.                      Once in the                      periplasm, FadD allows the transport of this fatty acids inside the cytosol using a system coupled to acyl-coA ester formation. Finally, the beta-oxidation process starts under catalysis of FadE and FadH                      enzymes, followed by FadA and FadB, both of them forming a tetrameric complex which is responsible of                      hydration, oxidation and thiolytic cleavage.
Fatty acid degradation and biosynthesis pathways must have a very dynamic expression system according to the                   availability of FAs in the environment to maintain cell functionality [5]. Thus, fad genes expression depends                   directly on the presence of internal LCFA bound to acyl-CoA.
FadR protein is the main transcriptional regulator of fad genes and is the key element regulating this                   process [6, 7]. It is involved in the negative regulation of LCFA degradation; in the absence of FAs it is                   constitutively bound to the fad genes promoter, repressing its transcription. In the presence of LCFA in the                   medium conversion to acyl-coA esters occurs. Then, LCFA-CoA bind to FadR, causing its release from                   the promoters and allowing the transcription of fad genes [8].
Figure 2 | The pfadBA promoter is constitutively repressed by FadR protein. LCFA enter                      the cell through the outer membrane receptor fadL, then, they get bound to acety-CoA. LCFA-acetyl-CoA is able                      to bind to FadR which allows it to be released from the promoter making the later active
FadL and FadD play a key role in the regulation of this pathway, as they are the main responsible of the                   uptake of LCFA and its proper conversion to acyl-coA ester in the cytosol. We hypothesized that                   overexpressing FadL and FadD, in presence of extracellular LCFA, would produce an scenario in which LCFA-CoA                   would repress FadR, resulting in an enhanced expression of the rest of the fad genes. Thus, FadL and FadD                   were targeted in order to develop a system with an increased LCFA degradation metabolism.
To do so, we expressed the sequences of either FadD or FadL downstream the TetR repressible promoter                   (BBa_R0040). Thus, allowing to tune the protein expression levels with the addition of Anhydrotetracycline                   (ATC).
Figure 3 | Schematic representation of the fad gene construct. Scheme representing                      the inducible expression system of the fad genes. A double terminator (BBa_B0014), a weak rbs                      (BBa_B0032), the inducible promoter (BBa_R0040) and the target gene were coupled in this biobrick.
Growth assays were performed in order to study the metabolic burden caused by the expression of these                   constructs. Moreover, in order to analyse the functionality of these genetic units, palmitic acid in the                   medium was quantified using cupric-acetate. Therefore, an increase of LCFA uptake when overexpressing these                   two fad genes could be measured.
The main approach of our project relies on an effective plasmid overexpression system. However, plasmidic                   presence within a live therapeutic in the intestine could deem not so effective, by posing a threat to the                   patients’ health due to the spread of antibiotic resistance to the natural gut ecosystem by plasmid                   transmission from our probiotic. In order to solve this, we approached overexpression of the mentioned genes                   from a different perspective; by directly modifying the E. coli genome as explained in the integration section .
References:
[1] Iram, S. H., & Cronan, J. E. (2006). The β-oxidation systems of Escherichia coli and                   Salmonella enterica are not functionally equivalent. Journal of bacteriology, 188(2), 599-608.
[2] Fujita, Yasutaro, Hiroshi Matsuoka, and Kazutake Hirooka. "Regulation of fatty acid                   metabolism in bacteria." Molecular microbiology 66.4 (2007): 829-839.
[3] Lepore BW, Indic M, Pham H, Hearn EM, Patel DR, van den Berg B: Ligand-gated                   diffusion across the bacterial outer membrane. Proc Natl Acad Sci U S A 2011, 108:10121–10126.
[4] Weimar JD, DiRusso CC, Delio R, Black PN: Functional role of fatty acyl-coenzyme A                   synthetase in the transmembrane movement and activation of exogenous long-chain fatty acids. Amino acid                   residues within the ATP/AMP signature motif of Escherichia coli FadD are required for enzyme activity and                   fatty acid transport. J Biol Chem 2002, 277:29369–29376.
[5] Janßen, H. J., & Steinbüchel, A. (2014). Fatty acid synthesis in Escherichia coli and                   its applications towards the production of fatty acid based biofuels. Biotechnology for biofuels, 7(1), 7.
[6] Henry MF, Cronan JE Jr: Escherichia coli transcription factor that both activates                   fatty acid synthesis and represses fatty acid degradation. J Mol Biol 1991, 222:843–849.
[7] Henry MF, Cronan JE Jr: A new mechanism of transcriptional regulation: release of an                   activator triggered by small molecule binding. Cell 1992, 70:671–679.
[8] Feng Y, Cronan JE Jr: Crosstalk of Escherichia coli FadR with global regulators in                   expression of fatty acid transport genes. PLoS One 2012, 7:e46275.
Follow us at:
For Judges
Attributions Collaborations Human Practices InterLab Improved part Judging Form Model Parts
Contact us:
igem [at] upf [dot] edu
PRBB Building
Doctor Aiguader 88, 08003 Barcelona (Spain)
