<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UPF_CRG_Barcelona_Biosensor skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UPF CRG Barcelona/Biosensor</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Wiki</TITLE></P><DIV class="sticky-top"><NAV class="navbar has-shadow" role="navigation" aria-label="main navigation" id="navbar"><DIV class="container"><DIV class="navbar-menu" id="navbar-menu"><DIV class="navbar-grid"><A class="navbar-item is-active" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Description">Project</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/DryLab">DryLab</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Parts">Parts</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Deliverables">Deliverables</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/WetLab">WetLab</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Intro_HP">Human Practices</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Aboutteam">Team</A></DIV></DIV></DIV></NAV></DIV><MAIN><SECTION class="container"><DIV><P class="page-title">LONG CHAIN FATTY ACID INTRACELLULAR BIOSENSOR</P><P>The main goals of our genetically modified organism are enhancing long chain fatty acids (LCFA) absorption
                  and metabolism and. This is doable by overexpressing Fad genes. However, this results in a metabolic burden
                  that affects the fitness of the cells, compromising the functionality of our probiotic, specially if it has
                  to compete with the rest of the microbiome for resources [1, 2]. For this reason, a system in which this
                  overexpression can be switched on and off depending on intracellular concentration of LCFA would be the best
                  approach for the designed proof of concept.
               </P><P>We have developed a genetic LCFA sensor device. This system not only allows a reliable characterization of
                  the fatty acid (FA) absorption of any bacterial system but can also be used to have a LCFA inducible
                  expression system.
               </P><P>The starting point of our design has been the pFadBA promoter. This is the promoter of the fadB and fadA
                  genes, and contains FadR binding sequences [3]. Thus, it is a biosensor of LCFA in the cytoplasm, as it is
                  derepressed in the presence of LCFA. Other iGEM teams have previously attempted to use it as a LCFA sensor,
                  such as NTU_Taida 2014 [4]. However, their results showed a very high baseline expression of the reporter
                  proteins coupled to the promoter. This did not allow them to see a significant rise in the signal after
                  induction with LCFA. 
               </P><P>In order to lower baseline expression, NTU_Taida 2014 team overexpressed FadR, forcing the repression of the
                  promoter. However, this approach would impair the functionality of the fad genes, as they are downregulated
                  by FadR.
               </P><P>As this interference with LCFA uptake was not compatible with our goal, we aimed to design a biosensor that
                  did not compromise bacteria fatty acid metabolism. In order to assess this goal by modulating the activation
                  threshold and diminishing the baseline expression, we have followed four different approaches:
               </P><P class="subapart1">Using different RBS</P><P>Ribosome binding sites (RBSs) are effective control elements for translation initiation. In fact, they are
                  commonly mutated to optimize genetic circuits and metabolic pathways [5]. In cases where the promoter has
                  leakage, the use of a weak RBS can help to reduce the signal. This approach does not directly tune the
                  promoter activity but the output of our biosensor. We therefore coupled the pFadAB promoter with different
                  RBS to see its contribution in each case. 
               </P><P class="subapart1">Inducible LuxR-pLux engineered device </P><P>In order to accurately quantify LCFA uptake with a genetic reporter, it is needed to obtain a significant
                  signal rise after induction with LCFA. Nevertheless, the fold change of the pfadBA promoter after induction
                  is very low and does not allow a reliable quantification. We designed a system that allowed us to externally
                  modulate genetic expression (<A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Measurement">See More</A>)
                  in an attempt to create an accurate LCFA biosensor.
                  We coupled the pfadBA promoter to the LuxR/pluxR lactone inducible system [6]. This circuit consists of the
                  LuxR, acyl-homoserine lactone receptor, under the expression of the pfadAB promoter, coupled with the lux
                  promoter expressing a reporter. Lux promoter is activated through luxR-lactone heterodimers.
               </P><P>The use of a second inducer, such as lactone, makes possible to modulate the signal in different ways: (A)
                  It enables both to increase the signal and (B) modify the fold change expression of the reporter (with the
                  appropriate lactone concentration) 
               </P><P class="
                  subapart1">Optimizing the dynamic range of the promoter</P><P>As already explained, pFadBA is a sensor with excessive leakage and a poor dynamic range. However, Zhang et
                  al. 2012 described a synthetic promoter with a higher dynamic range (pAR), which we have characterized for
                  the first time to avoid these levels of basality [7]. In short, this promoter contains an additional FadR
                  binding sequence than the natural one. In order to evaluate the responses of this promoter, we builded a
                  circuit with pAR and different RBS coupled to fluorescent reporter (BBa_E1010).
               </P><P class="subapart1">FURTHER APPROACHES: High pass filter</P><P>In order to eliminate the baseline expression of our reporter systems, we designed a genetic construct that
                  would only activate gene expression under the presence of LCFA. Our system consists of an incoherent
                  feed-forward loop which creates a high-pass filter.
               </P><P>Through an incoherent feed-forward loop, we could build a biological high-pass filter. This structure is
                  widely used in the treatment of electrical and radiofrequency signals, since only those signals with higher
                  frequency will pass. In our case, the highpass filter eliminates weak signals (leakage) by passing only the
                  strong ones; in other words, signals produced from higher LCFA concentrations.
               </P><P>To understand the effect of a high pass filter we must consider two cases:</P><OL><LI>Low activity of CI:
                     If CI is almost not expressed, it can not fully repress 343_CI and neither activate enough prm. So the
                     constitutively repression of 343_CI over prm will not allow a signal output. 
                  </LI><LI>High activity of CI:
                     If CI is strongly expressed, it can fully repress 343_CI and fully activate prm. Without the 343_CI
                     repressing nor CI activating, prm would be able to spread the signal.
                  </LI></OL><P class="subapart1">FURTHER APPROACHES: The metabolic switch </P><P>The capacity to induce or repress a gene depending on the presence of FA can not only be applied within
                  the overexpression approach. This could be oriented to change bacteria preference between glucose and LCFA.
                  To do so, we have worked with <I>in silico E. coli</I> metabolic models (<A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Model">
                  see more </A>). We have found 5 knockouts (KO) that theoretically maintain the growth that a wild type
                  has in presence of glucose and palmitic acid. As mentioned in the <A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Model">
                  Metabolic Switch Model</A>.The fact is that with only one KO more, the PA uptake multiplies for three
                  with a little decrease of the growth.
               </P><P>If we make this sixth gene expression dependent of our
                  biosensor, we can force our bacteria to absorb more PA than glucose in a rich LCFA medium. In this
                  manner, we can maintain our bacteria in absence of PA acid more time in the gut.
               </P><P>Finally, if we mix the overexpression and the metabolic switch biosensor, we would make our bacteria
                  uptake even more palmitic acid during a longer period.
               </P><P class="subapart2">References</P><P class=" references">[1] Silva, F., Queiroz, J. A., &amp;
                  Domingues, F. C. (2012). Evaluating metabolic stress
                  and plasmid stability in plasmid DNA production by Escherichia coli. Biotechnology advances, 30(3),
                  691-708.
               </P><P class="references">[2] Rozkov, A., Avignone‐Rossa, C. A., Ertl, P. F., Jones, P., O'Kennedy, R. D., Smith,
                  J. J., ... &amp; Bushell, M. E. (2004). Characterization of the metabolic burden on Escherichia coli DH1 cells
                  imposed by the presence of a plasmid containing a gene therapy sequence. Biotechnology and bioengineering,
                  88(7), 909-915.
               </P><P class="
                  references">[3] Feng Y, Cronan JE Jr: Crosstalk of Escherichia coli FadR with global regulators in
                  expression of fatty acid transport genes. PLoS One 2012, 7:e46275. 
               </P><P class="references">[4] NTU_Taida 2014 Wiki page. https://2014.igem.org/Team:NTU_Taida </P><P class="
                  references">[5] Salis, H. M., Mirsky, E. A., &amp; Voigt, C. A. (2009). Automated design of synthetic
                  ribosome binding sites to control protein expression. Nature biotechnology, 27(10), 946.
              </P><P class="references">[6] Fuqua, C., Winans, S. C., &amp; Greenberg, E. P. (1996). Census and consensus in bacterial ecosystems: the LuxR-LuxI family of quorum-sensing transcriptional regulators. Annual Reviews in Microbiology, 50(1), 727-751.</P><P class="references">[7] Zhang, F., Carothers, J. M., &amp; Keasling, J. D. (2012). Design of a dynamic sensor-regulator system for production of chemicals and fuels derived from fatty acids. Nature biotechnology, 30(4), 354.</P><P class="references">[8] Pédelacq J-D, Cabantous S, Tran T, Terwilliger TC, Waldo GS. 2006. Engineering and characterization of a superfolder green fluorescent protein. Nat Biotechnol 24:79–88. doi:10.1038/nbt1172.</P></DIV></SECTION></MAIN><FOOTER class="footer"><DIV class="container"><DIV><P class="subapart2" style="font-weight: 800; font-size: 1.2rem">Follow us at:</P></DIV><DIV><DIV style="text-align: center !important;"><P class="subapart2" style="font-weight: 800; font-size: 1.2rem; text-align: center !important;">For Judges</P></DIV><DIV id="judges"><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Attributions">Attributions</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/InterLab">InterLab</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Improve">Improved part</A><A href="https://igem.org/2018_Judging_Form?id=2581">Judging Form</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Model">Model</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Parts">Parts</A></DIV></DIV><DIV class="footer-contact"><P class="subapart2" style="font-weight: 800; font-size: 1.2rem">Contact us:</P><P>igem [at] upf [dot] edu</P><P> PRBB Building</P><P>Doctor Aiguader 88, 08003 Barcelona (Spain)</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>