<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UPF_CRG_Barcelona_Integration skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UPF CRG Barcelona/Integration</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Wiki</TITLE></P><DIV class="sticky-top"><NAV class="navbar has-shadow" role="navigation" aria-label="main navigation" id="navbar"><DIV class="container"><DIV class="navbar-menu" id="navbar-menu"><DIV class="navbar-grid"><A class="navbar-item is-active" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Description">Project</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/DryLab">DryLab</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Parts">Parts</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Deliverables">Deliverables</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/WetLab">WetLab</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Intro_HP">Human Practices</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Aboutteam">Team</A></DIV></DIV></DIV></NAV></DIV><MAIN><SECTION class="container"><DIV><P class="page-title">A LIVE BIOTHERAPEUTIC</P><P>Since GARGANTUA is planned to work as a live biotherapeutic, several facts should be considered to
          increase stability and safety of the chassis strain. Having in mind recent probiotic designed strains [1],
          FDA Guidance on Live Biotherapeutic Organisms (docket number FDA-2010-D-0500), and the concerns 
          that emerged upon our discussions with specialists, we explored which is the best engineering approach to address this
          in the specific case of our probiotic. We discussed which steps could be added to our experimental design
          to pave the way for an appropriate medical implementation of our probiotic, coming up with the following.</P><P>Firstly, we aimed to circumvent the use of antibiotic resistance genes in our strain in order to avoid
          horizontal gene transfer of synthetic circuits from our therapeutical strain into the gut ecosystem.
          Secondly, we targeted chromosomal integration of the enhanced intake properties to provide our design with
          genetic stability and robustness of expression in the gut. Thirdly, we intended to make overexpression of
          beta oxidation genes inducible, eliminating the metabolic burden of protein synthesis during the growth
          phase. This is highly important in industrial large scale culture process, where this burden could diminish
          final yield.</P><P>We thought that Multiplex Automated Genome Engineering (MAGE) [2] technology could provide a ideal
          platform for achieving the aforementioned characteristics that a safe and efficient live biotherapeutic
          should incorporate. The reason that lead us to use MAGE was the fact that intake capabilities are already
          present in the metabolism of E. coli. Therefore, simply tuning their expression in order to achieve the
          desired result is needed. Moreover, MAGE is highly scalable, and can be implemented in probiotic strains such as E.
          Coli Nissle, which would allow for direct editing of
          strains with therapeutic potential.</P><CENTER><P class="fig-caption"><B>Figure 1 | Description of MAGE.</B> Multiplex Automated Genome Engineering (MAGE) procedure allows for multiple site mutagenesis by recombination of degenerated oligonucleotides, with the possibility of targeting multiple specific sites at once. The possibility of genomic integration of our concept allows for compliance with international probiotic strain standards.
</P></CENTER><P class="subapart1">How does MAGE work?</P><P>Multiplex Automated Genome Engineering (MAGE) is a directed evolution tool for multiple genetic loci based
          on the lambda phage homologous recombination abilities for targeted mutagenesis. Exo, beta and gam phage
          lambda genes are induced under the temperature-sensible $\lambda$ cI promoter which allows for temperature
          inducible expression of recombination machinery in MAGE strains . This expression promotes the
          recombination of electroporated ssDNA oligonucleotides targeting specific sites which can introduce
          deletions, mismatches or insertions. Multiple cycles of electro competence induction, electroporation, and
          population expansion make the editing process highly scalable. Remarcably, this process can be free of
          selection markers, which are not desired in biotherapeutical strains. </P><CENTER><P class="fig-caption"><B>Figure 2 | Summary of a MAGE cycle.</B>
            Pre-grown colonies are electroporated for oligonucleotide incorporation in order to achieve the desired
            recombination. Different possibilities of screening are available in order to detect recombinant organisms
            of interest, such as allele specific PCR, or in the case of our fadR KO mutant, picking up red colonies.</P></CENTER><P class="subapart1">Genomic integration of LCFA uptake:</P><P>A simplistic albeit efficient approach consists in silencing the fadR gene, which is the main regulator of
          fad genes. Its absence should therefore raise constitutive levels of fad genes. To achieve this, we
          designed an oligo that when introduced in to the genome by the recombineering machinery would code for two
          stop codons in the beginning of the fadR coding sequence.</P><P>Considering the fact that beta oxidation genes are not solely regulated by the fadR repressor [3] we
          considered the introduction of T7 phage RNA polymerase promoter sequences upstream of fatty acid (FA)
          intake and degradation pathway genes.
          This would provide an orthogonal and inducible control of all the components in our system, which would be
          complicated to achieve merely by the use of synthetic constructs due to the large sequence size of the
          pathway. Genomic integration of the T7 polymerase is needed so as to achieve the desired levels of
          expression of the modified genes. The control of the polymerase under an inducible promoter provides the
          possibility of avoiding the metabolic burden caused by overexpression of beta oxidation genes in early
          stages of the probiotic strain production and batch culture. This would result in a more efficient
          production process.</P><P class="subapart1">Semirational Directed Evolution of uptake capabilities:</P><P>Incorporating sequence variation in to RBS of a particular pathway using MAGE can be used for optimization of expression of a particular metabolic pathway. In our case, this could be used for fine tuning the expression and consequently the efficiency of beta-oxidation. A  minimal media complemented with LCHFA as the only carbon source could be used as a selection method for optimal RBS sequences and organisms that present enhanced uptake of LCHFA.</P><P class="subapart1">Oligo design &amp; Cycling process:</P><P>The oligo of choice for silencing fadR was designed (using <A href="http://modest.biosustain.dtu.dk/">MODEST</A>)
          so as
          to introduce two STOP codons within the coding frame of the gene. Glutamic acid 13 and 14 were mutated in
          order to induce STOP codons. Using the same in silico predictor, we designed recombinant oligos of 90bp in
          length to include T7 polymerase promoter sequences as described in the CoSelection mage [4].</P><CENTER><P class="fig-caption"><B>Figure 3 | </B> MAGE deoligonucleotides target wild type sequences, and use Okazaki
            fragments during replication in order to be inserted in the genome by homology and recombination induced by
            exo, beta and gam.</P></CENTER><P class="subapart1">Screening</P><P>LacZ silencing is used as a positive control when performing MAGE cycles due to its easily visible
          phenotype. Moreover, an improved alternative method for screening fadR mutants was designed. Our fatty
          acyl-CoA biosensor is dependant on fadR repressor levels, which acts on the pFadBA promoter sequence. We
          envisioned that transforming our part BBa_K2581006 into MAGE strain EcM2.1 could allow for a efficient
          screening of fadR knock-outs, since the absence of the regulator on the promoter should lead to the
          expression of basal levels of RFP (<A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Results">See More</A>).</P><P class="subapart2">References</P><P class="references">[1] Isabella, V. M., Ha, B. N., Castillo, M. J., Lubkowicz, D. J., Rowe, S. E.,
            Millet, Y. A., ... &amp; Reeder, P. J. (2018). Development of a synthetic live bacterial therapeutic for the
            human metabolic disease phenylketonuria. Nature biotechnology.</P><P class="references">[2] Wang, H. H., Isaacs, F. J., Carr, P. A., Sun, Z. Z., Xu, G., Forest, C. R., &amp;
            Church, G. M. (2009). Programming cells by multiplex genome engineering and accelerated evolution.
            Nature, 460(7257), 894.</P><P class="references">[3] Cho, B. K., Knight, E. M., &amp; Palsson, B. Ã˜. (2006). Transcriptional regulation of
            the fad regulon genes of Escherichia coli by ArcA. Microbiology, 152(8), 2207-2219.</P><P class="references">[4] Wang, H. H., Kim, H., Cong, L., Jeong, J., Bang, D., &amp; Church, G. M. (2012).
            Genome-scale promoter engineering by coselection MAGE, 9(6). https://doi.org/10.1038/nmeth.1971</P></DIV></SECTION></MAIN><DIV class="container"><DIV><P class="subapart2" style="font-weight: 800; font-size: 1.2rem">Follow us at:</P></DIV><DIV><DIV style="text-align: center !important;"><P class="subapart2" style="font-weight: 800; font-size: 1.2rem; text-align: center !important;">For Judges</P></DIV><DIV id="judges"><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Attributions">Attributions</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/InterLab">InterLab</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Improve">Improved part</A><A href="https://igem.org/2018_Judging_Form?id=2581">Judging Form</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Model">Model</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Parts">Parts</A></DIV></DIV><DIV class="footer-contact"><P class="subapart2" style="font-weight: 800; font-size: 1.2rem">Contact us:</P><P>igem [at] upf [dot] edu</P><P> PRBB Building</P><P>Doctor Aiguader 88, 08003 Barcelona (Spain)</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>