<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UPF_CRG_Barcelona_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UPF CRG Barcelona/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Wiki</TITLE><UL id="floating-menu-results"><LI><A href="#Overexpression">Overexpression</A></LI><LI><A href="#Biosensor">Biosensor</A></LI><LI><A href="#Integration">Integration</A></LI></UL></P><DIV class="sticky-top"><NAV class="navbar has-shadow" role="navigation" aria-label="main navigation" id="navbar"><DIV class="container"><DIV class="navbar-menu" id="navbar-menu"><DIV class="navbar-grid"><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Description">Project</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/DryLab">DryLab</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Parts">Parts</A><A class="navbar-item is-active" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Deliverables">Deliverables</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/WetLab">WetLab</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Intro_HP">Human
                            Practices</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Aboutteam">Team</A></DIV></DIV></DIV></NAV></DIV><MAIN style="max-width: 1450px;"><SECTION class="container-centered"><A name="results"><P class="page-title">RESULTS</P></A><DIV class="grid-sixcolumns"><DIV class="button_grid_sixcolumns"><BUTTON class="button"><A href="#Overview">Overview</A></BUTTON></DIV><DIV class="button_grid_sixcolumns"><BUTTON class="button"><A href="#Overexpression">Overexpression</A></BUTTON></DIV><DIV class="button_grid_sixcolumns"><BUTTON class="button"><A href="#Biosensor">Biosensor</A></BUTTON></DIV><DIV class="button_grid_sixcolumns"><BUTTON class="button"><A href="#Integration">Integration</A></BUTTON></DIV><DIV class="button_grid_sixcolumns"><BUTTON class="button"><A href="#Conclusion">Conclusion</A></BUTTON></DIV></DIV><A name="Overeview"><P class="subapart1">Overview</P></A><UL style="background-color: #c6d1e5; border-radius: 15px; padding: 1.8rem; width: 80%; margin: auto;"><LI style="padding: 0.3rem">When fadL and fadD are overexpressed LCFA uptake increases resulting in
                    a higher
                    growth if LCFA are the only carbon source available.
                </LI><LI style="padding: 0.3rem">pFadBA-luxR-pLux lactone inducible LCFA biosensor can amplify the
                    signal of pfadBA promoter-controlled gene expression.</LI><LI style="padding: 0.3rem">pFadBA-cI-pRM LCFA biosensor reduces a lot the baseline expression of
                    genes directly regulated by the pfadBA promoter.</LI><LI style="padding: 0.3rem">pAR promoter significantly improves pfadBA function as an off/on switch
                    dependent on the presence of LCFA.</LI><LI style="padding: 0.3rem">We used MAGE recombineering to generate a strain missing fadR, in a
                    effort to integrate intake capabilities in to the genome.</LI><LI style="padding: 0.3rem">We applied the PfadBA based biosensor to improve screening of
                    recombineered fadR colonies.</LI></UL><A name="Overexpression"><P class="subapart1">Overexpression results</P></A><P>In order to study the functionality of our system we characterized the constructs both in palmitic
                acid (PA) and oleic acid (OA). Bacteria was grown for 3 days in M9 minimal medium with the
                respective fatty acid, to ensure that cells were using solely LCFA as a carbon source .
                Additionally, the supernatant was also collected and LCFA concentration was determined using
                cupric-acetate colorimetric (see <A href="”https://2018.igem.org/Team:UPF_CRG_Barcelona/Methods">methods </A>)
                    technique at OD715nm. The same growth assay was performed in LB to infer cell growth in enriched
                    media.</P><P>Different levels of fad genes expression were achieved through the use
                    of different strains. Genetic constructs consisting of either FadD or FadL downstream of the
                    tetR repressible promoter were transformed in Top10 (DH5-alpha) and Zn1 <I>E. coli</I> strains
                    . Top 10 constitutively synthesizes tetR, which results in a continuous expression of our
                    construct. Zn1 does not express tetR, thus meaning that our genetic constructs will not be
                    induced in this strain. This induction can be modulated by adding anhydrotetracycline (ATc) in
                    the medium. We created a cell library expressing the fadD and fadL proteins: </P><CENTER style="display: flex;justify-content: center; width: 100%"><TABLE class="fluid-table" style="align-self: center; max-width: 90%;"><THEAD><TR><TH>Cell</TH><TH>Parts Registry</TH><TH>Construct</TH><TH>Expression</TH><TH>Cellular type</TH></TR></THEAD><TBODY><TR><TD class="heading" data-label="Cell"><B>Constitutive FadD</B></TD><TD data-label="Parts Registry">BBa_K2581009 </TD><TD data-label="Construct">T14_ptet_32_FadD</TD><TD data-label="Expression">constitutive</TD><TD data-label="Cellular type">TOP10</TD></TR><TR><TD class="heading" data-label="Cell"><B>Inducible FadD</B></TD><TD data-label="Parts Registry">BBa_K2581009 </TD><TD data-label="Construct">T14_ptet_32_FadD</TD><TD data-label="Expression">tuneable</TD><TD data-label="Cellular type">ZN1</TD></TR><TR><TD class="heading" data-label="Cell"><B>Constitutive FadL</B></TD><TD data-label="Parts Registry">BBa_K2581010</TD><TD data-label="Construct">T14_ptet_32_FadL</TD><TD data-label="Expression">constitutive</TD><TD data-label="Cellular type">TOP10</TD></TR><TR><TD class="heading" data-label="Cell"><B>Inducible FadD</B></TD><TD data-label="Parts Registry">BBa_K2581010</TD><TD data-label="Construct">T14_ptet_32_FadL</TD><TD data-label="Expression">tuneable</TD><TD data-label="Cellular type">ZN1</TD></TR><TR><TD class="heading" data-label="Cell"><B>Constitutive Reporter</B></TD><TD data-label="Parts Registry">BBa_K2581010</TD><TD data-label="Construct">T14_ptet_32_RFP</TD><TD data-label="Expression">constitutive</TD><TD data-label="Cellular type">TOP10</TD></TR></TBODY></TABLE></CENTER><CENTER><P class="fig-caption">Table 1 | Summary of the parts and experimental conditions used for this
                    section</P></CENTER><P class="subapart2">FadD overexpression increases growth when either PA or OA are the only carbon
                source.
            </P><P>We analyzed the behavior of the constitutive and inducible FadD cell lines with M9 minimal media
                supplemented with either PA or OA. Constitutive reporter cell line was used as a control. OD
                measurements were performed at OD 600 nm as an indicative for cell growth for 72 hours.</P><P>Our results show that, when fadD is overexpressed, bacterial growth increases both in 0,4 mM PA (Fig
                1A) and 2 mM PA (Fig 1B) in relation to control. Moreover, non-induced FadD cells show a lower
                increase of bacterial growth, compared to the control. On the other hand, when tested with OA
                medium (Fig 1C and Fig 1D), the difference in growth is higher than in PA medium. </P><P class="fig-caption"><B>Figure 1 | Growth assay comparing induced and non-induced fadD in minimal medium
                    enriched with different concentrations of Oleic Acid (OA) and Palmitic Acid (PA).</B> Figure A
                shows OD600nm over time for induced (top10 strain) and not induced (Zn1 strain) fadD in 0,4mM PA
                concentration. Figure B shows OD600nm over time for induced (top10 strain) and non-induced (Zn1 strain)
                fadD in 2mM PA concentration. Figure C shows OD600nm over time for induced (top10 strain) and
                non-induced (Zn1 strain) fadD in 0,4mM OA concentration. Figure D shows OD600nm over time for induced
                (top10 strain) and not induced (Zn1 strain) fadD in 2mM OA concentration. Error bars represent the
                Standard Deviation of the Mean (SEM).</P><P>Moreover, growth in enriched media (LB) was studied. Expression for induced FadD cell line was tuned
                with ATc. Tuneable FadD overexpression levels entailed a big metabolic burden (Figure 2). Our results
                show that when ATc induction increases, growth rate diminishes. This way, our results suggest that even
                when PA is also available in the media, our system will prefer other carbon sources. However, when LCFA
                is the only available carbon source, the overexpression of fad genes gives a metabolic advantage
                reflected in enhanced growth (Figure 1).
            </P><CENTER><P class="fig-caption"><B>Figure 2 | Growth assay comparing induced and not induced fadD in enriched
                        medium (LB).
                    </B>
                    Top 10 cells (DH5-alpha) were grown for 16 hours in enriched medium with a PA concentration of
                    0,4mM. In order to induce fadD expression a concentration of 60ng/ml of ATc was added. Error bars
                    represent the Standard Deviation of the Mean (SEM).
                </P><P class="subapart2"> FadL overexpression increases growth when either PA or OA are used as the only
                    carbon
                    source.
                </P><P>We analyzed the behavior of the constitutive and inducible FadL cell lines with M9 minimal media
                    supplemented with either PA or OA. Constitutive reporter cell line was used as a control. OD
                    measurements were performed at OD 600nm as an indicative cell growth for 72 hours.</P><P>Here we demonstrate that overexpression of fadL entails a metabolic burden. This is shown in the
                    decrease of the OD600 when induced bacteria are grown in both 0,4mM PA or OA (Figure 3A and 3C) and
                    2mM PA or OA (Figure 3B and 3D). When comparing this metabolic burden between the two concentrations
                    it is clear that in the 2mM concentration induced bacteria grow more than in 0,4mM. Thus, this
                    results suggest that FadL entails a metabolic burden even in minimum media but is reduced when LCFA
                    concentration increases, as LCFA can be used as an energetic source. </P><P class="fig-caption"><B>Figure 3. Growth assay comparing induced and not induced fadL in minimal
                        medium enriched with different concentrations of Oleic Acid (OA) and Palmitic Acid (PA). </B>Figure
                    A shows OD600nm over time for induced (top10 strain) and not induced (Zn1 strain) fadL in 0,4mM PA
                    concentration. Figure B shows OD600nm over time for induced (top10 strain) and not induced (Zn1
                    strain) fadL in 2mM PA concentration. Figure C shows OD600nm over time for induced (top10 strain)
                    and not induced (Zn1 strain) fadL in 0,4mM OA concentration. Figure D shows OD600nm over time for
                    induced (top10 strain) and not induced (Zn1 strain) fadL in 2mM OA concentration. Error bars
                    represent the Standard Deviation of the Mean (SEM).</P><P class="subapart2">FadD overexpression increases OA uptake</P><P>We analyzed the behavior of the constitutive and inducible FadD cell lines with M9 minimal media
                    supplemented with either PA or OA. Constitutive reporter cell line was used as a control.
                    Supernatant was collected from the medium after 72 hours growing. Cupric-acetate colorimetric
                    technique was performed quantify LCFA concentration (OD715nm).</P><P>Our results showed that, when fadD was overexpressed, OA uptake nearly doubled the uptake of non
                    induced bacteria and control (p &lt;0,001) (Figure 4). However, this increase could not be observed
                    in
                    PA. A possible explanation would be the infeasibility to obtain a proper standard curve for PA
                    using cupric-acetate technique when compared to oleic acid (see methods). </P><P class="fig-caption"><B>Figure 4 | LCFA uptake in FadD overexpression. </B> Plot showing LCFA
                    uptake when comparing induced (Top10 strain) and not induced (Zn1 strain) fadD gene.
                    Constitutive reporter cell line (top10) was used as a control. Uptake was measured
                    considering the OD715nm difference between the medium at certain concentration of LCFA and
                    the medium with grown bacteria. A concentration of 0,4mM of both PA and OA was used.
                    Cupric-acetate technique was used to quantify LCFA in the medium. LCFA uptake was
                    normalized by the growth of the bacteria, measured at OD600. Error bars represent the
                    Standard Deviation of the Mean (SEM). Statistical significance of the mean was calculated
                    using a paired t-test. P value &lt; 0,05, which indicates a statistically significant
                    difference among relevant groups, is designated with an asterisk.</P><P class="subapart2">Overexpression
                    of fadL increases OA uptake.
                </P><P>Here to study the LCFA uptake the same experimental procedure was followed as described in
                    FadD. Our results show that, when fadL was induced OA uptake was more than 2-fold higher in
                    relation to non-induced and control bacteria (Figure 5). This increased uptake was only
                    statistically significant in OA when compared to control (p &lt; 0,0001) and not induced
                    bacteria (p &lt; 0,0001). PA media didn’t show significant results (Figure 5).</P><P class="fig-caption"><B>Figure 5. LCFA uptake in FadL overexpression.</B> Plot
                    showing LCFA uptake when comparing induced (Top10 strain) and not induced (Zn1
                    strain) fadL gene. Constitutive reporter cell line (Top10) was used as a control.
                    Uptake was measured considering the OD715nm difference between the medium at
                    certain concentration of LCFA and the medium with grown bacteria. A concentration
                    of 0,4mM of both PA and OA was used. Cupric-acetate technique was used to quantify
                    LCFA in the medium. LCFA uptake was normalized by the growth of the bacteria,
                    measured at OD600. Error bars represent the Standard Deviation of the Mean (SEM).
                    Statistical significance of the mean was calculated using a paired t-test. P value
                    &lt; 0,05, which indicates a statistically significant difference among relevant
                    groups, is designated with an asterisk.</P><P class="subapart2">Discussion of
                    overexpression results </P><DIV style="background-color: #c6d1e5; border-radius: 15px; padding: 1.8rem; width: 70vw;"><P>Our results demonstrate that overexpression of fadL and fadD in enriched media
                        entails a metabolic burden for the cell. However, when these genes are
                        overexpressed in a minimum media with LCFA as the unique carbon sources, the
                        metabolic burden is reduced. Furthermore, bacterial growth is even higher
                        between induced and not induced cells. We have demonstrated that induction of
                        fadL and fadD results in a LCFA uptake increase, being statistically
                        significant in OA. </P><P>Considering our results, it can be deduced that when fadL and fadD are
                        overexpressed LCFA uptake increases resulting in a higher growth. In this way,
                        we can conclude that LCFA influx is increased in our system, resulting in
                        higher Acyl-CoA concentration inside the cell. Consequently, we have
                        hypothesized that this results in an enhanced expression of the rest of fad
                        genes and therefore in an increase of the LCFA degradation rate. This increase
                        in the LCFA degradation, leads to more metabolic fuel to be used for growth
                        (only when LCFA is the only carbon source available). Therefore, overexpression
                        of fadL and fadD gives the cell a positive advantage that results in enhanced
                        growing when LCFA is the only carbon source.
                    </P></DIV><A name="Biosensor"><P class="subapart1">Biosensor Results</P></A><P>In order to characterize the functionality of the LCFA biosensors, fluorescent
                    proteins were coupled to them. Top10 (DH5-alpha) <I>E. coli</I> expressing the
                    genetic constructs were grown for 16-20 hours in LB media with different
                    concentrations of palmitic acid (PA). Fluorescence intensity and OD600 were
                    analysed at steady state. </P><CENTER><TABLE class="fluid-table" style="width: 70vw;;"><THEAD><TR><TH>Cell</TH><TH>Parts Registry</TH><TH>Construct</TH><TH>Expression</TH><TH>Cellular type</TH></TR></THEAD><TBODY><TR><TD class="heading" data-label="Cell"><B>pfadBA Reporter cell</B></TD><TD data-label="Parts Registry">BBa_K2581006</TD><TD data-label="Construct">t14_pfadBA_34_RFP</TD><TD data-label="Expression">PA inducible</TD><TD data-label="Cellular type">TOP10</TD></TR><TR><TD class="heading" data-label="Cell"><B>pfadBA-Lux Lactone inducible
                                    reporter cell</B></TD><TD data-label="Parts Registry">BBa_K2581007 </TD><TD data-label="Construct">t14_pFadBA_34_luxR_T_32_RFP</TD><TD data-label="Expression">PA and HSL inducible</TD><TD data-label="Cellular type">TOP10</TD></TR><TR><TD class="heading" data-label="Cell"><B>pfadBA-prm Reporter cell
                                    expressing CI activator</B></TD><TD data-label="Parts Registry">BBa_K2581017</TD><TD data-label="Construct">t14_pfadBA_34_CI_prm_34_RFP</TD><TD data-label="Expression">PA inducible</TD><TD data-label="Cellular type">TOP10</TD></TR><TR><TD class="heading" data-label="Cell"><B>pAR Reporter cell</B></TD><TD data-label="Parts Registry">BBa_K2581011</TD><TD data-label="Construct">t14_pAR_32_RFP</TD><TD data-label="Expression">PA inducible</TD><TD data-label="Cellular type">TOP10</TD></TR></TBODY></TABLE><P class="fig-caption">Table 2 | Summary of the parts and experimental conditions
                        used for this section </P></CENTER><P class="subapart2">Characterization of the Fatty acid acyl-CoA inducible promoter</P><P> To evaluate the function of the pfadBA promoter (BBa_K817002) we designed a
                    reporter system with RFP (BBa_K2581006). </P><P class="fig-caption"><B>Figure 6 | </B><I>E. coli</I> bacteria (DH5-alpha) expressing
                    the pFad34+RFP construct were induced with different concentrations of PA in LB
                    media. Fluorescence was analyzed once it had reached the steady state (13-15h).</P><P>Our results show that the baseline fluorescence intensity of the pfadBA34RFP
                    construct is really high. Despite this, an increase of fluorescence intensity was
                    observed for increasing PA induction. </P><P>Fluorescence saturation could not be observed in the transfer function due to the
                    fact that medium with higher concentrations than 1 mM PA generated a lot of noise
                    which made impossible to analyse fluorescence at such high concentrations. </P><P class="subapart2">Characterization of the Inducible LuxR-pLux engineered device</P><P>In order to modulate the expression of the genes under control of the pfadBA
                    promoter for different LCFA concentrations, we designed a pfadBA reporter system
                    inducible by lux-homoserine lactone (HSL). </P><P class="fig-caption"><B>
                        Figure 7 |
                        <E. coli="">
                            bacteria (DH5-alpha) expressing the pFfadBA_Lux34_plux_32_RFP construct.
                    </E.></B>
                    They were induced with different concentrations of HSL and 3 different
                    concentrations of PA (0, 0.4 and 1 mM) in LB medium. Fluorescence was analyzed once
                    it had reached the steady state (11-13h). Fluorescence intensity values were
                    normalized by the OD.
                </P><P>Our data indicates that the activation threshold of the lactone inducible pfadBA
                    construct is approximately at 5x10<SUP>-9</SUP> M of HSL. It saturates at 1x10<SUP>-7</SUP>
                    M of lactone. Significant fluorescence difference between different PA
                    concentrations is better observed at lactone concentration ranges close to
                    fluorescence saturation.</P><P class="subapart2">Characterization of the cI mediated activity</P><P>In order to reduce the baseline expression of genes under direct control of the
                    pfadBA promoter we designed two constructs that together would act as a genetic
                    high pass filter. Due to time constraints, we were only able to build the first
                    construct of the designed system (<A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Biosensor">See
                        more</A>). In order to analyse its behaviour we coupled it to a RFP reporter
                    protein. </P><P class="fig-caption"><B>Figure 8 | <I>E. coli</I> bacteria (DH5-alpha) expressing the
                        pFadBA+34+CI+ pRM+34+RFP construct were induced with different concentrations
                        of PA in LB media.</B> Fluorescence was analysed once it had reached the steady
                    state(12-14h). Fluorescence was normalised by the OD.</P><P class="subapart2">Comparative study between the Inducible LuxR-pLux, the pFadBA and
                    the cI biosensors</P><P>Our results indicate that the baseline expression of the pfadBA-CI-prm construct is
                    relatively low. A linear increase of fluorescence intensity is observed for
                    increasing concentrations of PA.</P><P class="fig-caption"><B>Figure 9 | Comparison of fluorescence intensity between
                        different pFadBA constructs.</B> Bacteria were grown in LB media with different
                    concentrations of PA (0, 0.4 and 1 mM). Bacteria expressing the pfadBA-plux
                    construct were induced with lactone 1e-7 M. Fluorescence was analysed at steady
                    state. Fluorescence intensity was normalized by the OD.</P><P class="fig-caption"><B>Figure 10 | </B>Comparison of fluorescence intensity fold
                    change (induced/non-induced) differences between different pfadBA constructs.
                    pfadBA-plux construct was additionally induced with lactone 1x10<SUP>-7</SUP>. </P><P>pfadBA-luxR-plux-RFP construct presented the highest fluorescence intensity in all
                    medium conditions and it showed similar fluorescence fold changes after induction
                    compared to pfadBA alone. On the other hand, pfadBA-CI-pRM-RFP showed little
                    fluorescence compared to the other constructs. Nevertheless, it showed a
                    significant fold change increase compared to the other two constructs. </P><P class="subapart2">Characterization of the Improved fatty acid acyl-CoA biosensor</P><P>We characterised the improved pFadBA promoter, pAR (<A href="http://parts.igem.org/Part:BBa_K2581012" target="_blank">BBa_K2581012</A>) coupling it to a RFP reporter gene. </P><P class="fig-caption"><B>Figure 11 | </B> E.coli bacteria (DH5-alpha) expressing the
                    constructs were induced with different concentrations of PA in LB media.
                    Fluorescence was analyzed once it had reached the steady state (13-15h).
                    Fluorescence intensity values were normalised by the OD.</P><P class="fig-caption"><B>Figure 12 |</B> Comparison of fluorescence intensity fold
                    change (induced/non induced) differences between pFadBA and pAR inducible
                    constructs.</P><P>After induction of the pAR promoter with different PA concentrations our results
                    show a significant fluorescence fold change increase compared to the pfadBA
                    promoter. Moreover, our data indicates that the baseline fluorescence of the pAR
                    construct is much lower than that of pfadBA. </P><P class="subapart2">Discussion of biosensor</P><DIV style="background-color: #c6d1e5; border-radius: 15px; padding: 1.8rem; width: 70vw;"><P>We have proved that the pFadBa-34-RFP biosensor can detect changes in PA concentration. However, the fluorescence ratio for induced/non-induced cultures is very low due to the fact that the baseline fluorescence intensity is non-negligible . If we expect to use our device as a LCFA quantification system, it is needed to obtain a significant difference in the output signals for different LCFA concentrations. This can be achieved either by amplifying the signal or by increasing the signal fold change between the induction with different LCFA concentrations. We achieved amplification of  the output signal by inducing the pfadBA-luxR-plux-RFP with lactone 1e-7 M. On the other hand, we were able to reduce the basality of the previous LCFA biosensors with our pfadBA-CI-pRM construct. By reducing the baseline fluorescence we observed an increased fold change of the output signals between induction with different PA concentrations. Nevertheless, with this construct the fluorescence signal deemed lower than the other constructs due to the fact that the pRM promoter is weak. We think that we could obtain a LCFA reliable quantification system by coupling the two constructs together. The hypothetical system would work as follows: the first part (pfadBA-CI-pRM-luxR) would increase the fold change concentration of luxR and the signal would be processed by the luxR-HSL inducible plux-RFP reporter system in order to amplify the signal. This system would potentially generate enough fluorescence difference between different PA induction concentrations to make it  possible to predict the concentration of PA in the medium.</P><P>On the other hand, we observed that the pAR promoter has a sharper response after PA induction compared to the pfadBA promoter. Moreover, our data suggests that it saturates at much lower PA concentrations than the pfadBA promoter and it has less leakage. Taking all this together, we can affirm that our newly characterised promoter would work much better as an off/on switch dependent on LCFA than the pfadBA promoter, therefore we have successfully improved a previous iGEM parts registry part.</P><P>It’s also important to take into account that we have used RFP as a reporter gene, which has a long expression and degradation time. Consequently, this means a delay when reporting LCFA uptake and the moment when expression stops. This is due to the fact that RFP remains some hours in the medium even if it is no longer being expressed. We suggest that a faster reporter protein, the superfolder GFP, which has a very short expression and degradation time, would be more accurate when measuring the switch ON/OFF of our system. </P></DIV><A name="Integration"><P class="subapart1">Integration Results</P></A><P>
In order to explore the genomic integration of uptake capabilities for a stable chassis that could be used as a live biotherapeutic, we tested the MAGE technology and developed a FadR Knock-Out mutant. 
</P><P class="subapart2">MAGE cycling efficiency:</P><P>Estimation of allele replacement frequency (ARF) after each cycle was performed by
                    X-gal blue-white
                    screening. Oligos encoding for two stop codons in the lacZ gene were electroporated
                    in each cycle as a
                    positive control. Recombineered colonies are deficient in the Beta-galactosidase
                    enzyme and therefore
                    appear white when plated on X-gal IPTG containing plates.
                </P><P class="fig-caption"><B>Figure 13 | ARF screening on LB+X-gal+IPTG plates after five
                        mage cycles in
                        EcM2.1 </B>strain. Cultures were plated at 10e-5 (left) and 10e-6 (right).
                </P><P>Cultures where plated after five cycles. ARF was 4.8%, estimated by colony counting
                    (Figure 13). This corresponds to
                    ~1% of colonies incorporating mutations in the lacZ allele in each cycle. Although
                    our ARF was lower than
                    reported [1], we considered it sufficient to screen for fadR mutants.
                </P><P class="subapart2">MAGE fadR KO screening:</P><P class="fig-caption"><B>Figure 14 | Agarose gel electrophoresis of FadR KO screening
                        through allele specific
                        PCR, after five MAGE cycles. </B>Numbers correspond to screened colonies. 36
                    colonies were analyzed, only
                    12 of them are shown in the picture. Top wells correspond to colonies screened with
                    wt specific primers.
                    Bottom line shows 12 same colonies amplified with specific primers for the FadR
                    mutations. Colony 11 shows
                    no amplification for the wt allele (361bp), but has the expected amplicon for FadR
                    knock-outs; 362bp.
                </P><P>After performing five cycles in EcM2.1 MAGE strain using oligos coding for two stop
                    codons in the
                    beginning of FadR ORF, We performed allele specific PCR screening. One primer pair
                    was designed to
                    specifically recognize the WT sequence, while another primer pair was designed to
                    anneal to the mutant
                    sequence. Performing a colony PCR with both primer pairs should lead to a single
                    amplicon, depending on the
                    genotype of the analysed colony, as seen in figure 14. All colonies are amplified
                    with the WT allele except
                    colony number 11, in which amplification with the mutant primer pair appears. This
                    colony was selected as
                    the bona fide mutant of the FadR repressor. A total of 36 colonies were analyzed
                    (only 12 of them shown,
                    all other 24 where WT specific). From this we can make a rough estimation of the
                    efficiency of FadR KO
                    oligo incorporation, which is 2,77%.
                </P><P class="subapart2">fadR optimized screening method:</P><P>One of our final composite and biosensor parts; BBa_K2581006 is regulated by FadR
                    repressor, which binds
                    to pfadBA in the absence of LCFA. We hypothesized that FadR KO cells transformed
                    with this construct should
                    have basal expression of RFP. Different red intensities can be seen in EcM2.1
                    transformed colonies (Figure 15). We
                    speculate that this might be due to the fact that EcM2.1 are not optimal for
                    protein expression since they
                    lack MutS machinery and thus have impaired growth. Red colonies were picked and
                    analysed by allele specific
                    PCR. Of 16 analysed colonies, 4 included the mutation (Figure 15) which corresponds
                    to ~25%. Although this
                    screening method does not bypass the need for allele specific PCR screening, it
                    significantly increased the
                    number of positive colonies in our screening, which would allow for a reduced
                    number of cycles when
                    obtaining a FadR KO. Finally, we cultured and re-plated a FadR KO colony (num 9
                    figure. 16) expressing our
                    BBa_K2581006 construct and a fadR negative colony( num. 10 Figure 16). As expected,
                    WT colony had a low basal
                    level of RFP expression and fadRKO colonies had much higher RFP expression (Figure
                    17)
                </P><P class="fig-caption"><B>Figure 15 | EcM2.1 strain cycled fadR KO. Cells were
                        transformed with
                        pFadBA_34_RFP(BBa_K2581006).</B></P><P class="fig-caption"><B>Figure 16 | Allele specific PCR screening with red colonies
                        that were transformed
                        with BBa_K2581006.</B> Top gel lanes correspond to colony screening with WT
                    specific FadR primers. 4 of
                    the 16 tested colonies show amplification for the wt allele. The bottom lanes of
                    the gel show colony
                    screening using FadR KO primers. Four colonies (8,9,12,13) tested positive for FadR
                    KO induced by MAGE.
                </P><P class="fig-caption"><B>Figure 17 | Biosensor activation comparison between a FadR KO
                        colony expressing RFP
                        and a wt EcM 2.1 biosensor transformed colony. </B></P><P class="subapart2">Discussion of
                    Integration results </P><DIV style="background-color: #c6d1e5; border-radius: 15px; padding: 1.8rem; width: 70vw;"><P> 
We generated a strain missing the fadR using MAGE technology,  thus creating a system of overexpression of all the proteins involved in LCFA beta-oxidation, which can be implemented in a biotherapeutic strain. During our screening of mutant colonies, we used the PfadBA biosensor as a way to improve  detection of colonies missing fadR expression. Levels of uptake capabilities between overexpression achieved through synthetic constructs and overexpression by the absence fadR  remains to be characterized. 
</P></DIV><P class="references">References:</P><P class="references">[1] Gallagher R, Li Z, Lewis A, Isaacs F. Rapid editing and evolution of bacterial
                    genomes
                    using libraries of synthetic DNA. Nat Protoc. 2014; 9(10): 2301-2316. Available
                    from: doi: 10.1038/nprot.2014.082.
                    </P></CENTER></SECTION></MAIN></DIV><FOOTER class="footer"><DIV class="container"><DIV><P class="subapart2" style="font-weight: 800; font-size: 1.2rem">Follow us at:</P></DIV><DIV><DIV style="text-align: center !important;"><P class="subapart2" style="font-weight: 800; font-size: 1.2rem; text-align: center !important;">For Judges</P></DIV><DIV id="judges"><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Attributions">Attributions</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/InterLab">InterLab</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Improve">Improved part</A><A href="https://igem.org/2018_Judging_Form?id=2581">Judging Form</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Model">Model</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Parts">Parts</A></DIV></DIV><DIV class="footer-contact"><P class="subapart2" style="font-weight: 800; font-size: 1.2rem">Contact us:</P><P>igem [at] upf [dot] edu</P><P> PRBB Building</P><P>Doctor Aiguader 88, 08003 Barcelona (Spain)</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>