<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UPF_CRG_Barcelona_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UPF CRG Barcelona/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Wiki</TITLE></P><DIV class="sticky-top"><NAV class="navbar has-shadow" role="navigation" aria-label="main navigation" id="navbar"><DIV class="container"><DIV class="navbar-menu" id="navbar-menu"><DIV class="navbar-grid"><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Description">Project</A><A class="navbar-item is-active" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/DryLab">DryLab</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Parts">Parts</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Deliverables">Deliverables</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/WetLab">WetLab</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/intro_HP">Human Practices</A><A style="color: rgb(242, 242, 242)" class="navbar-item" href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Aboutteam">Team</A></DIV></DIV></DIV></NAV></DIV><MAIN><SECTION class="container-centered"><DIV><P class="page-title">MODEL</P><CENTER><DIV class="grid-sixcolumns"><DIV style="grid-column: span 2"><BUTTON class="button"><A href="#fba">Flux Balance Analysis Models</A></BUTTON></DIV><DIV style="grid-column: span 2"><BUTTON class="button"><A href="#Switch">Metabolic Switch</A></BUTTON></DIV></DIV></CENTER><A name="fba"><P class="subapart1">Flux Balance Analysis Models</P></A><P>In order to simulate LCFA uptake and metabolism by <I>E. coli</I> we performed flux balance analysis (FBA),
          a mathematical approach to analyze the flow of metabolites through a metabolic network [3] (see figure 1).
          FBA models allow for the prediction of cellular growth rate under different environmental conditions or
          genetic perturbations. Moreover, it enables the analysis of the influx or production of a desired metabolite.
          For our models, we used the genome-scale metabolic network reconstruction of <I>Escherichia coli</I> K-12
          MG1655 iJO1366 [4], which contains all known metabolic reactions in our <I>E. coli</I> strain and the genes
          that encode each enzyme [5]. The final goal was to find new strategies to engineer <I>E. coli</I> for
          achieving a bacterial system capable of efficiently uptaking LCFA in human gut-like conditions.</P><A name="Simulation"><P class="subapart1">Simulation of Intestine Conditions</P></A><P class="subapart2">Aerobic vs anaerobic growth with LCFA</P><P>Firstly, we wanted to assess whether beta-oxidation of LCFA was possible under anaerobic conditions. To do
          so, growth was predicted by optimizing the biomass function under either aerobic or anaerobic conditions.
          Each condition was analyzed with palmitate as the sole carbon source or glucose as a control variable.</P><P class="fig-caption"><B>Figure 1 |</B> Simulation using flux balance analysis by maximizing flux through the biomass reaction. In aerobic conditions, oxygen maximum uptake flux was set to an arbitrarily high level, 1000 mmol gDW-1 hr-1 (millimoles per gram dry cell weight per hour, the default flux units used in the COBRA Toolbox). Whereas in anaerobic conditions, oxygen was set to have no flux.</P><P>The model predicted no growth when palmitic acid (PA) was the only carbon source in anaerobic conditions
          (Fig. 1). That means that beta oxidation cannot be functional under anaerobic conditions. On the other hand,
          in aerobic conditions the simulations predicted a lot of growth (more than with only glucose*). When glucose
          was the only carbon source <I>E. coli</I> showed little growth compared to aerobic conditions as expected.</P><P class="subapart2"> Anaerobic growth simulation with different electron acceptors </P><P>The fact that in the simulated minimal medium bacteria could not grow with only palmitate as a carbon source
          under anaerobic conditions can be explained by the lack of reductive power in the cell. This blocks the beta
          oxidation preventing the metabolism of palmitic acid.</P><P>Alternative electron acceptors molecules have been shown to allow anaerobic respiration of sugars [6]. We
          hypothesized that these molecules could be able to oxidize LCFA too. Since our final objective is to make our
          bacteria work in a human gut-like environment, we were only interested in those electron acceptor molecules
          that could be found in the gut. Fumarate, nitrate and Trimethylamine N-oxide (TMAO) have been shown to allow
          anaerobic respiration and both can be found in the gut in normal conditions [7]. Therefore, the effect of
          these three molecules on the phenotype of <I>E. coli</I> under anaerobic conditions was modelled.</P><P class="fig-caption"><B>Figure 2 |</B> Simulation using flux balance analysis by maximizing flux through the biomass reaction. Anaerobic conditions were simulated setting oxygen influx reaction flux to 0. Nitrate, TMAO and oxygen uptake fluxes were set to be unconstraint by allowing a maximum flux of 1000 mmol gDW-1 hr-1. Fumarate was set to be at a high concentration by aallowing a maximum flux of 19 mmol gDW-1 hr-1 An increasing maximum uptake flux, range 0 to 19 mmol gDW-1 hr-1, was set for either glucose or palmitate.</P><P>Growth rate was predicted under anaerobic conditions with an arbitrary high concentration of nitrate,
          fumarate or TMAO in the simulated medium and oxygen as a control group. When nitrate was present, both palmitate and glucose were able to be oxidized (fig. 2), indicating that nitrate was the final electron acceptor in the phosphorylative
          oxidation. On the other hand, neither fumarate nor TMAO were able to oxidize PA.</P><P>Gut concentrations of nitrate and fumarate are not well known due to the fact that is very difficult to
          quantify molecule concentrations <I>in vivo</I> in the intestine. However, in healthy conditions their
          concentrations are not very high [7]. For this reason, in order to model PA uptake, we set a low
          concentration of nitrate in the medium for all the models (20 mmol gDW-1 hr-1 (millimoles per gram dry cell
          weight per hour).</P><P class="subapart2">Anaerobic growth with palmitate and glucose </P><P>In the gut, PA is never found alone, as it is ingested with other foods, such as sugars or amino acids.
          Therefore, it was analyzed how the PA uptake is affected under different concentrations of glucose, the main
          sugar for excellence. To do that, a fixed concentration of palmitate was set and the biomass function was
          optimized for an increasing concentration of glucose. The nitrate available was set to be really low as we
          did before, in order to make it the limiting factor.</P><P class="fig-caption"><B>Figure 3 |</B> Simulation using flux balance analysis by maximizing flux through the biomass reaction. Nitrate influx reaction maximum flux was set to 20 mmol gDW-1 hr-1. Palmitate influx reaction maximum flux was set to 10 mmol gDW-1 hr-1. Glucose maximum influx reaction was set in a range from 0 to 19 mmol gDW-1 hr-1.</P><P>The model predicted a reduction of PA uptake (from 2.5 to 0.4 mmol gDW-1 hr-1) when glucose concentration
          was increased (fig. 3). Nevertheless, when glucose concentration overcame a certain threshold, approximately
          at 14 mmol, palmitate uptake got stabilized at 0.4 mmol (fig. 3). These results indicate that bacteria, in
          order to maximize their growth, when both PA and glucose are present and there is little reductive power,
          prioritize the oxidation of glucose before palmitate. When the reductive power is not enough to fully oxidize
          all the glucose available, glucose is partially fermented. When this happens, palmitate uptake does not
          decrease anymore for any increase in glucose concentration.</P><P class="subapart2">Finding strategies to priorize LCFA oxidation over sugars oxidation</P><P>In the light of the previous results, we hypothesized that an increase of glucose fermentation would
          increase PA uptake. In order to check the hypothesis, the production of lactate was optimized using FBA in
          order to force the organism to ferment glucose. To do so, the lower bound of the lactate exchange reaction
          was set to be different positive values.</P><P class="fig-caption"><B>Figure 4 |</B>Simulation using flux balance analysis by maximizing flux through the biomass reaction. Nitrate influx reaction maximum flux was set to 20 mmol gDW-1 hr-1. Palmitate influx reaction maximum flux was set to 10 mmol gDW-1 hr-1. Glucose maximum influx reaction was set to 10 mmol gDW^-1. Lactate production was forced by changing the lower bound of the lactate out flux reaction in a range of 0-19 mmol gDW-1 hr-1.</P><P>Our models predicted that an increase of lactate production allowed an increase of palmitate uptake (fig.
          4). PA uptake flux went from 0.99 to 2.81 mmol gDW-1 hr-1, when there was no lactate production and when
          there was 19 mmol gDW-1 hr-1 of lactate production respectively (fig. 4). Biomass was reduced from 0.93 to
          0.41 mmol gDW-1 hr-1.</P><P class="subapart2">Creating an optimal PA consuming organism under intestinal conditions</P><P>To achieve the aforementioned behavior in our bacteria, we would have to modify genes encoding for metabolic enzymes. One simplistic option would be to overexpress the lactate dehydrogenase gene. However, this system would not be very effective because the lactate dehydrogenase enzyme catalyses a reversible reaction and as soon as the lactate concentration increases it would reach a steady state stopping lactate production. Also, the overexpression would result in such a metabolic burden that would considerable reduce bacterial growth. For this reason, we decided to find genome knock outs that would cause the same effect.</P><P>We implemented an algorithm to compare the metabolic flow between the wt strain and a simulated lactate
          producer strain, with that, we were able to find possible targets as knock outs. We designed another
          algorithm to simulate the effects of the target knock outs on the biomass and PA uptake and we iterated both
          algorithms until achieving a substantial increase in PA uptake without creating a letal genome modification.
          We ended up with a list of 63 knock outs that together produced the desirable effect. However, it would be
          unfeasible to perform that many knock outs. Thus, we designed one more algorithm to refine the large list and
          rule out the redundant gene knock outs. We achieved to find a combination of 6 knock out that were able to
          increase a lot PA uptake.</P><P class="fig-caption"><B>Figure 5 |</B> Figure 5 | Comparison of the growth rate constant and PA uptake between WT <I>E.coli</I> and the 5 and 6 KO mutants. Gene KO of the 6 KO strain are: pyruvate dehydrogenase (pdh), pyruvate formate lyase (PFL), pyruvate oxidase (POX), Formyltetrahydrofolate deformylase (FTHFD), Glucose 6-phosphate dehydrogenase (G6PDH2r) and Deoxyribose-phosphate aldolase (DRPA). The 5 KO strain has the same previous KO but the pyruvate dehydrogenase (pdh) gene. </P><A name="Switch"><P class="subapart1">Metabolic Switch</P></A><P>In order to prevail bacteria for LCFA metabolism before glucose oxidation we have demonstrated here that 6 gene knockouts are needed. However, when LCFA are not present in the environment, cells with the 6 KO would have less energetic efficiency. By only reducing one specific KO (Pyruvate dehydrogenase) cells can oxidize sugars again and have the same energetic efficiency as the wt cells. For this reason, taking advantage of our LCFA biosensor we propose a metabolic switch that would work as follows:</P><P>Our <I>E. coli</I> would have the 6 gene knockouts. Without LCFA in the environment the modified bacteria would express the pyruvate dehydrogenase under the control of a repressible promoter so the bacteria would have the same fitness as the WT. However, with the presence of LCFA in the environment, the modified bacteria would sense it and repress the expression of the pyruvate dehydrogenase gene by expressing a repressor gene. This would lead to preference of LCFA oxidation over glucose that would result in an enhanced absorption compared to the non modified bacteria. </P><P class="subapart2">References</P><P class="references">[1] Espey, M. G. (2013). Role of oxygen gradients in shaping redox relationships between
          the human intestine and its microbiota. Free Radical Biology and Medicine, 55, 130-140.</P><P class="references">[2] Na, D., Kim, T. Y., &amp; Lee, S. Y. (2010). Construction and optimization of synthetic
          pathways in metabolic engineering. Current opinion in microbiology, 13(3), 363-370.</P><P class="references">[3] Orth, J. D., Thiele, I., &amp; Palsson, B. Ø. (2010). What is flux balance analysis?.
          Nature biotechnology, 28(3), 245.</P><P class="references">[4] McCloskey, D., Gangoiti, J. A., King, Z. A., Naviaux, R. K., Barshop, B. A., Palsson,
          B. O., &amp; Feist, A. M. (2014). A model‐driven quantitative metabolomics analysis of aerobic and anaerobic
          metabolism in E. coli K‐12 MG1655 that is biochemically and thermodynamically consistent. Biotechnology and
          bioengineering, 111(4), 803-815.</P><P class="references">[5] Oberhardt, M. A., Palsson, B. Ø., &amp; Papin, J. A. (2009). Applications of genome‐scale
          metabolic reconstructions. Molecular systems biology, 5(1), 320.</P><P class="references">[6] Unden, G., &amp; Bongaerts, J. (1997). Alternative respiratory pathways of Escherichia
          coli: energetics and transcriptional regulation in response to electron acceptors. Biochimica et Biophysica
          Acta (BBA)-Bioenergetics, 1320(3), 217-234.</P><P class="references">[7] Jones, S. A., Gibson, T., Maltby, R. C., Chowdhury, F. Z., Stewart, V., Cohen, P. S.,
          &amp; Conway, T. (2011). Anaerobic respiration of Escherichia coli in the mouse intestine. Infection and
          immunity, 79(10), 4218-4226.</P></DIV></SECTION></MAIN><DIV class="container"><DIV><P class="subapart2" style="font-weight: 800; font-size: 1.2rem">Follow us at:</P></DIV><DIV><DIV style="text-align: center !important;"><P class="subapart2" style="font-weight: 800; font-size: 1.2rem; text-align: center !important;">For Judges</P></DIV><DIV id="judges"><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Attributions">Attributions</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/InterLab">InterLab</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Improve">Improved part</A><A href="https://igem.org/2018_Judging_Form?id=2581">Judging Form</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Model">Model</A><A href="https://2018.igem.org/Team:UPF_CRG_Barcelona/Parts">Parts</A></DIV></DIV><DIV class="footer-contact"><P class="subapart2" style="font-weight: 800; font-size: 1.2rem">Contact us:</P><P>igem [at] upf [dot] edu</P><P> PRBB Building</P><P>Doctor Aiguader 88, 08003 Barcelona (Spain)</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>