<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NCTU_Formosa_results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NCTU_Formosa/results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NCTU_Formosa/results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NCTU_Formosa/results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NCTU_Formosa/results&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NCTU_Formosa/results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NCTU Formosa/results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="header-wrapper"><DIV id="header"><DIV id="title"><A href="https://2014.igem.org/Team:NCTU_Formosa"><P>NCTU_Formosa - Operation Debug</P></A></DIV><DIV id="igemlogo"><A href="https://2014.igem.org/"><P>iGEM 2014</P></A></DIV><DIV id="igemlogo2"><A href="http://www.nctu.edu.tw/"><P>iGEM 2014</P></A></DIV><DIV id="igemlogo3"><A href="https://www.facebook.com/pages/NCTU_Formosa-IGEM-team/267841893250331"><P>iGEM 2014</P></A></DIV><NAV id="nav" class="navwrapper"><BUTTON id="navb" class="dl-trigger">Open Menu</BUTTON></NAV></DIV><DIV class="navv"><UL class="nav"><LI><A href="https://2014.igem.org/Team:NCTU_Formosa">Home</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/project">Project</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/project#Motivation">Motivation</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/project#PBAN_.28_Pheromone_Biosynthesis_Activating_Neuropeptide_.29"> PBAN</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/project#Main_Idea">Main Idea</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/project#Biobrick_Design">Biobrick Design</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/project#Device">Device</A></LI></UL><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/results">Results</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/results#Magic_Power_of_Our_Pyramidal_Device">Magic Power of Our Pyramidal Device</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/results#PBAN_Biobricks_Tests">PBAN Biobricks Tests</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/results#Obtaining_PBAN_from_E.coli">Obtaining PBAN from <I>E.coli</I></A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/results#Insect_Tests">Insect Tests</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/modeling#Modeling_Introduction">Modeling Introduction</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/modeling#Modeling_of_PBAN">Modeling of PBAN</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/modeling#Modeling_of_Device">Modeling of Device</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/modeling#Modeling_Software">Modeling Software</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/biobricks">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/biobricks#Parts_Submitted_to_the_registry">Parts Submitted to The Registry</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/biobricks#Brief_Information">Brief Information</A></LI><LI><A style="line-height:1;" href="https://2014.igem.org/Team:NCTU_Formosa/human_practice">Human Practice</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/human_practice#Field_Trip_and_Industry_Collaboration">Field Trip and Industry Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/human_practice#Team_Exchange">Team Exchange</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/human_practice#iGEM_Advertising">iGEM Advertising</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/human_practice#Surveys">Surveys</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/team">Team</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/team#Attribution">Attribution</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/team#Members_and_Advisers">Members and Advisers</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/team#Sponsors">Sponsors</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/Notebook">Note</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/Notebook#About_The_Notes">About The Notes</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/Notebook#Material_and_Methods">Material and Methods</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/Notebook#Note">Note</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/Safety#Researcher_safety">Researcher Safety</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/Safety#Public_Safety">Public Safety</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/Safety#Lab_Instruments">Lab Instruments</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/Safety#Safety_Form">Safety Form</A></LI><LI><A href="https://2014.igem.org/Team:NCTU_Formosa/Achievement">Achievement</A></LI><UL id="back-top"><LI><A href="#top">Top</A></LI></UL></DIV></DIV><DIV id="main-wrapper"><DIV id="main"><DIV id="cover"><DIV id="coverbox"><DIV id="ctitle">Results</DIV></DIV></DIV><DIV id="changeFont"><SPAN class="changeFontText">Change the font size right here</SPAN></DIV><DIV id="content-wrapper"><DIV id="realcontent"><DIV id="crw"><DIV id="cr"><DIV class="cont"><DIV class="li"><DIV class="card"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Magic_Power_of_Our_Pyramidal_Device"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Magic Power of Our Pyramidal Device</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#PBAN_Biobricks_Tests"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">PBAN Biobricks Tests</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#PBAN_Gene_Synthesis_.28Full_Gene_Sequence_Design_Process.29"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">PBAN Gene Synthesis (Full Gene Sequence Design Process)</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#PCR_experiment_of_PBAN"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">PCR experiment of PBAN</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#PBAN_Peptide_Check_by_SDS_Protein_Electrophoresis"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">PBAN Peptide Check by SDS Protein Electrophoresis</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Blue_Light_Fluorescence_Test"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Blue Light Fluorescence Test</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#Obtaining_PBAN_from_E.coli"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Obtaining PBAN from E.coli</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Insect_Tests"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Insect Tests</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Behavior_of_Target_Insects_After_PBAN_Treatment"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Behavior of Target Insects After PBAN Treatment</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Effective_Attraction_after_PBAN_Treatment"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Effective Attraction after PBAN Treatment</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Spodoptera_Litura.27s_Preference_for_Temperature_and_Light"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Spodoptera Litura's Preference for Temperature and Light</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Magic_Power_of_Our_Pyramidal_Device">Magic Power of Our Pyramidal Device</SPAN></H3><P>  We develop a powerful and specific insect attractive device which combine the use of blue light and PBAN. The video shows that <B>our device successfully and effectively attract more insects</B>. In this video: </P><P>  1. We used our pyramidal device, biobrick part <A href="http://parts.igem.org/Part:BBa_K1415105" class="external text" rel="nofollow">BBa_K1415105: Pcons+RBS+PBAN(<I>Spodoptera litura</I>)</A> and its target insect <I>Spodoptera litura</I> inside an arcrylic chamber for this expiriment. </P><P>  2. The female moth is fed with PBAN in a small beaker covering with plastic wrap. Soon, the female flaps its wings frantically, a sign of sexual stimulation. From this point on, it starts to release pheromones. </P><P>  3. We transferred the beaker into our device then placed the device in an acrylic chamber(moth box) with approximately 150 male moths inside to conduct the test. We kept the chamber dark to ensure that blue light would be the only light source inside. We recorded the number of insects entering the device per hour for eleven hours.</P><P> Fig.1-1 shows the <B>entering times</B> of moths in different conditions. Entering times is defined as the times that moths get into the device. In this experiment, the moth might get out of our device, since we were only using a simple model device without trap, such as nets for keeping the moths. However, we can still clearly see the magic power of our pyramidal device by using blue light with PBAN treatment, which attracted more moths to get into the device than <A href="https://www.youtube.com/watch?v=IXGEvhqHs-c" class="external text" rel="nofollow">the other experiment (only blue light treatment)</A>. In addition, we can even observe a unique periodicity of attracting moths only in blue light with PBAN treatment. We consider this phenomenon is related to the physiology of the female moths' rut situation.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fig.1-1 Entering times is defined as the times that moths get into the device. The times of moths entering the device was recorded hourly for eleven hours, and can clearly see the magic power of our pyramidal device by using blue light with PBAN treatment.</DIV></DIV></DIV></DIV><P>Our experiment can be divided into two categories. </P><P>1. PBAN Biobricks Tests: gene recombination and protein expression. </P><P>2. Insect Tests: PBAN effect test, insect behavior test and device test. </P></DIV></DIV><DIV class="li"><DIV class="card"><H3><SPAN class="mw-headline" id="PBAN_Biobricks_Tests">PBAN Biobricks Tests</SPAN></H3><H4><SPAN class="mw-headline" id="PBAN_Gene_Synthesis_.28Full_Gene_Sequence_Design_Process.29">PBAN Gene Synthesis (Full Gene Sequence Design Process)</SPAN></H4><P>To capture the harmful insects causing damage in agriculture, we first found 9 different kinds of PBAN peptide of harmful insects  common in many places of the world from our long literature review. Next, we obtain the DNA sequences by reversely translating the peptide sequences of these PBANs from NCBI (ex: PBAN Spodoptera litura: <B><A href="http://www.ncbi.nlm.nih.gov/protein/AAK84160.1" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/protein/AAK84160.1</A></B> ) Finally, we <B>modified every codon</B> on the DNA sequence and <B>designed the DNA sequence</B> for <I>E.coli</I> to express a certain PBAN. </P><P>DNA Modification Process: </P><P>1. Avoid the rare codons of <I>E.coli</I>, and choose high frequency codons. 
ããã(Frequency Table Tool: <B><A href="http://www.genscript.com/cgi-bin/tools/codon_freq_table" class="external free" rel="nofollow">http://www.genscript.com/cgi-bin/tools/codon_freq_table</A></B>)</P><P>Use <A href="http://www.genscript.com/cgi-bin/tools/rare_codon_analysis" class="external text" rel="nofollow">Rare Codon Analysis Tool</A> to inspect if there is any problem to express our gene for <I>E.coli.</I></P><P>2. Avoid choosing the same codon when modifying our designed gene sequence to prevent the <I>E.coli</I> from running out of ããããããnucleotides due to repeated use. </P><P>3. Avoid the start codon ATG in the middle of the coding sequence.  </P><P>Take the PBAN of <I>Spodoptera litura</I> for example: </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption">Fig.2-1-1 The distribution of codon usage frequency along the length of your CDS to be expressed in your target host organism. Possibility of high protein expression level is correlated to the value of CAI - a CAI of 1.0 is considered to be ideal while a CAI of &gt;0.8 is rated as good for expression in the desired expression organism. GenScript's OptimumGeneTM codon optimization tool can typically improve your sequence to reach a CAI of higher than 0.8 thus better chance of high level protein expression.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:551px;"><DIV class="thumbcaption">Fig.2-1-2 The ideal percentage range of GC content is between 30% to 70%. Any peaks outside of this range will adversely affect transcription and translation efficiency.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:551px;"><DIV class="thumbcaption">Fig.2-1-3 The percentage distribution of codons in computed codon quality groups. The value of 100 is set for the codon with the highest usage frequency for a given amino acid in the desired expression organism. Codons with values lower than 30 are likely to hamper the expression efficiency.</DIV></DIV></DIV></DIV><P>5. Add iGEM standard sequence in front of and at the back of our modified DNA sequence.</P><P>6. Synthesize the modified DNA sequence of PBANs in a gene synthesis company.</P><H4><SPAN class="mw-headline" id="PCR_experiment_of_PBAN">PCR experiment of PBAN</SPAN></H4><P>After receiving the DNA sequences from the gene synthesis company, we recombined each PBAN gene to PSB1C3 backbones and conducted a PCR experiment to check the size of each of the PBANs. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">Fig.2-2-1 The PCR result of the 9 different kinds of PBAN. The DNA sequence length of PBANs are around 100-150 bp, so the PCR products should appear at 300-350 bp.<P> Below are biobrick serial numbers of PBAN abbrevation:</P> BM: BBa_K1415001ããããMB: BBa_K1415002ããããAI: BBa_K1415003 LD: BBa_K1415004ããããSL: BBa_K1415005ããããHAH: BBa_K1415006 AS: BBa_K1415007ããããSI: BBa_K1415008ãããããAA: BBa_K1415009</DIV></DIV></DIV></DIV><P>The DNA sequence length of the PBAN are around 100-150 bp. In this PCR experiment, the PBAN products size should be around 300-350 bp. Fig.2-2-1 showed the correct size of the PBAN, and proved that we <B>successful ligated the PBAN DNA sequence onto an ideal backbone</B>.</P><H4><SPAN class="mw-headline" id="PBAN_Peptide_Check_by_SDS_Protein_Electrophoresis">PBAN Peptide Check by SDS Protein Electrophoresis</SPAN></H4><P>Moreover, to verify that all 9 kinds of PBAN can be expressed by the <I>E.coli</I>, we conducted a <B>SDS protein electrophoresis experiment</B>. We first smashed the <I>E.coli</I> containing the PBAN with a sonicator and then took the supernatant divided from the bacterial pellet by centrifugation. Finally, we used the supernatant to run a SDS protein electrophoresis in a 20Â % SDS gel.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fig.2-3-1 Protein Electrophoresis of P<SUB>cons</SUB> + RBS + 5 different kinds of PBAN (control: plasmid of  P<SUB>cons</SUB>+RBS). Each peptide of PBAN is an around 30 amino acids, so we can see the band of PBANs at 2-4 kDa.<P> Below are biobrick serial numbers of PBAN abbrevation:</P> BM: BBa_K1415101ãããAA: BBa_K1415109ãããLD: BBa_K1415104ããã AS: BBa_K1415107ãããSL: BBa_K1415105ããããããããã</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fig.2-3-2 Protein Electrophoresis of P<SUB>cons</SUB> + RBS + 4 different kinds of PBAN  (control: plasmid of  P<SUB>cons</SUB>+RBS). Each peptide of PBAN is an around 30 amino acids, so we can see the band of PBANs at 2-4 kDa.<P> Below are biobrick serial numbers of PBAN abbrevation:</P> AI: BBa_K1415103ãããMB: BBa_K1415102ãããHAH: BBa_K1415106ãããSI: BBa_K1415108</DIV></DIV></DIV></DIV><P>These SDS PAGE results in Fig.2-3-1 showed that the <B>bands are at 2-4 kDa</B> for each of the PBANs, while the plasmid of P<SUB>cons</SUB>+RBS weren't there (the PBAN peptide is around 30 amino acids long). This result proves that the<B><I>E.coli</I> can produce the PBAN</B> of our interest.
</P><H4><SPAN class="mw-headline" id="Blue_Light_Fluorescence_Test">Blue Light Fluorescence Test</SPAN></H4><P>To predict the PBAN expression in <I>E.coli</I> by computer modeling, we next tested PBAN BFP biobricks. We obtained the average  expressive value of the blue fluorescence in the biobrick part (above) and also the control part of P<SUB>cons</SUB> + RBS + BFP + Ter. Therefore, we can use the average value to generate predictions of the PBAN expression in <I>E.coli</I>. 
<A href="https://2014.igem.org/Team:NCTU_Formosa/modeling#Modeling_for_biobricks" class="external text" rel="nofollow">(<I>See more details in our Modeling Page</I>).</A>
Below is the <B>blue fluorescence expression</B> curve in a long period of time. We used these data to predict the PBAN expression in <I>E.coli</I>.

</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:889px;"><DIV class="thumbcaption">Fig.2-4-1 The blue light fluorescence expression curve of <I>E.coli</I> containing P<SUB>cons</SUB> + RBS + 9 different kinds of PBAN + RBS + BFP + Ter plasmid (control: competent cells that cannot emit blue light).ãBelow are biobrick serial numbers of the PBAN abbreviations: SL: BBa_K1415205ããããBM: BBa_K1415201ããããMB: BBa_K1415202 AI: BBa_K1415203ããããLD: BBa_K1415204ããããHAH:BBa_K1415206 AS: BBa_K1415207ããããSI: BBa_K1415208ããããAA: BBa_K1415209</DIV></DIV></DIV></DIV><P>In Fig.2-4-2, we can clearly see that the blue fluorescence expressed by the <I>E.coli</I> is different from the control without BFP expressed. Also, we find that different kinds of PBAN has different expression in <I>E.coli</I>. We think because different PBAN has <B>different codons</B> on DNA sequence, <B>different PBAN peptide</B> or PBAN plasmid will be expressed or replicated in different amount.  
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Fig.2-4-2 Blue Fluorescence of P<SUB>cons</SUB> + RBS + 9 different kinds of PBAN + RBS + BFP + Ter plasmid (control: <I>E.coli</I> containg P<SUB>cons</SUB>+RBS Plasmid).ãBelow are biobrick serial numbers of the PBAN abbreviations:<P style="text-align:left;">SL: BBa_K1415205ããããBM: BBa_K1415201ããããMB: BBa_K1415202</P><P style="text-align:left;">AI: BBa_K1415203ããããLD: BBa_K1415204ããããHAH:BBa_K1415206</P><P style="text-align:left;">AS: BBa_K1415207ããããSI: BBa_K1415208ããããAA: BBa_K1415209</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="li"><DIV class="card"><H3><SPAN class="mw-headline" id="Obtaining_PBAN_from_E.coli">Obtaining PBAN from <I>E.coli</I></SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fig.3-1-1 The Process of Obtaining PBAN from <I>E.coli</I>. We first culture the <I>E.coli</I> that contains our constructed plasmid of P<SUB>cons</SUB> + RBS + One Kind of PBAN for 12 hr. Second, We smash <I>E.coli</I> with a sonicator to divide the PBAN from the cell debris pellets. Finally, we autoclave the supernatant to avoid biosafet issues.</DIV></DIV></DIV></DIV><P>In order to obtain PBAN from our <I>E.coli</I>, we first culture the <I>E.coli</I> that contains our constructed plasmid of P<SUB>cons</SUB> + RBS + One Kind of PBAN in best time condition at which <A href="https://2014.igem.org/File:PBAN_OD600_Value.jpg" class="external text" rel="nofollow">the blue light fluorescence expression</A> reaches the maximum value. Then, we centrifuged the bacterial solution to pour-off the LB broth and resuspend the <I>E.coli</I> pellet with pure water later for the next step. From this step, we can obtain a LB-free solution, which is better for our insect experiment. Then, we <B>smash</B> the <I>E.coli</I> with a sonicator and <B>dilute</B> this PBAN solution with 250 ml pure water in a serum bottle. Finally, we <B>sterlized</B> the PBAN solution to avoid biosafety issues. As we know, PBAN is a very <B>simple and short</B> peptide so it will not be degraded after the autoclave treatment. Since only a very small amount (10 pmol) of PBAN will be enough to stimulate the maximum production of pheromone, therefore, we don't have to worry that our PBAN concentration will be inadequate after diluting with 250 ml pure water. Overall, we can simply use the sterilized solution without purification, following recommended culturing conditions. </P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:292px;"><DIV class="thumbcaption">Fig.3-1-2 We culture the <I>E.coli</I> in best time condition to have the maximum expression of PBAN. Below are biobrick serial numbers of PBAN abbrevation:<SPAN style="text-align:left;">MB: BBa_K1415102HAH: BBa_K1415106SL: BBa_K1415105</SPAN></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:292px;"><DIV class="thumbcaption">Fig.3-1-4 We smash the <I>E.coli</I> with a sonicator. Below are biobrick serial numbers of PBAN abbrevation:<SPAN style="text-align:left;">MB: BBa_K1415102HAH: BBa_K1415106SL: BBa_K1415105</SPAN></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:292px;"><DIV class="thumbcaption">Fig.3-1-3 We centrifuged the bacterial solution to pour LB broth. Below are biobrick serial numbers of PBAN abbrevation:<SPAN style="text-align:left;">MB: BBa_K1415102HAH: BBa_K1415106SL: BBa_K1415105</SPAN></DIV></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:292px;"><DIV class="thumbcaption">Fig.3-1-5 We dilute this PBAN solution with 250ml pure water in serum bottle. Below are biobrick serial numbers of PBAN abbrevation:<SPAN style="text-align:center;">MB: BBa_K1415102HAH: BBa_K1415106SL: BBa_K1415105</SPAN></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:292px;"><DIV class="thumbcaption">Fig.3-1-6 We sterlized PBAN solution to avoid biosafety issues.</DIV></DIV></DIV></DIV></DIV><DIV class="li"><DIV class="card"><H3><SPAN class="mw-headline" id="Insect_Tests">Insect Tests</SPAN></H3><H4><SPAN class="mw-headline" id="Behavior_of_Target_Insects_After_PBAN_Treatment">Behavior of Target Insects After PBAN Treatment</SPAN></H4><P>
To investigate what behavior the female moth would show after sucking PBAN, we put one female moth into a beaker for observation. The beaker is divided into two parts by plastic wrap. The bottom part contains the PBAN solution we prepared, and the upper part is the space for the moth to stay(showed in Fig.4-2-3). We soaked cotton that spans the entire length of the beaker with the PBAN solution and sprinkle it with sugar. This way, the moth can suck on the PBAN without drowning in PBAN solution. After all the equipment is set, we put the female moth into the upper part of the beaker. At the time, we started filming as soon as we observed the female moth showing obvious behaviors of <B>sexual stimulation</B> such as flapping their wings.</P><P>The activeness of the 9 kinds of target insects changes with the <B>seasons</B>. Our observation was made around September to October, which is the most active time for <I>Spodoptera litura, Mamestra brassicae</I> and <I>Helicoverpa armigera Hubner</I>. Therefore, in this observation, we caught these three kinds of moth in Sunnymorning organic farm as sample moths. </P><P>
We observed that the moth could absorb the PBAN in the solution through suck, and that the PBAN could <B>stimulate</B> the moth's <B>pheromone gland</B> to produce pheromone.</P><P><B>As soon as the moth is sexually excited, it would flap its wings rapidly and move its tail slightly upward.</B></P><P>These vidoes show the behaviors of 3 different kinds of female moths after sucking their separate PBANs. Each of the moths clearly became <B>excited and all flapped their wings rapidly</B>.
</P><H4><SPAN class="mw-headline" id="Effective_Attraction_after_PBAN_Treatment">Effective Attraction after PBAN Treatment</SPAN></H4><P>After observing how PBAN treatment affects the behavior of female moths, we want to check the <B>attractive effect</B> of the moths. We expected that,  after sucking the PBAN, the female moth would not only become excited, <B>flap its wings</B>, but also actually attract male moths to <B>aggregate</B> together. We used two beakers which are the same as what we used in the former experiment. One contained PBAN solution and the other contained only sucrose solution as control. We first put one beaker at one side and the other at the opposite side in the moth box (shown in Fig.4-2-1). Then we put two female moths in each beaker and at least 100 male moths in the moth box. This time, we did a long time observation and took a picture with our camera. In Fig.4-2-1, the female moth sucked the PBAN then attracted more male moths than the one sucking the sucrose solution. Thus, Fig.4-2-1 can prove that the female moth sucked our PBAN then released a large amount of sex pheromone to attract many male moths. In addition, we conducted a simple test to compare the luring effect of female moths sucking PBAN solution with that of sucrose solution (moth's favorite food). Again, we can see the conspicuous effect. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Fig.4-2-1 Negative Control: Female moths suck sucrose solution without PBAN (number = 0). Experiment: PBAN sucking of Female moths (number = 11). In this picture, we can see that the female moths sucking PBAN solution can release much sex pheromone, and attract many male moths.</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fig.4-2-2  Negative Control: sucrose solution, Experiment: Female moth eating PBAN solution. Also, we can see the PBAN effect again from this picture.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fig.4-2-3 The container we used to contain female moths with PBAN solution, we designed it with beaker.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Spodoptera_Litura.27s_Preference_for_Temperature_and_Light"><I>Spodoptera Litura'</I>s Preference for Temperature and Light </SPAN></H4><P>As we all know, light can attract many kinds of harmful insects. 
</P><P>Temperature is another environmental factor which the farmer can not change practically. We want to use computer <B>modeling</B> to explore in depth the moths' preferences for different combinations of <B>light and temperature</B> conditions. In the future, we hope that farmers can choose the appropriate light according to temperature condition when using our device. For this purpose, we chose the average temperature range in Taiwan in a year, and most common harmful insect, <I>Spodoptera Litura</I> to conduct this test (Fig.4-3-1 below), which we used to model the moths' preferences for different combinations of light and temperature conditions with ANFIS (see detail in the  <A href="https://2014.igem.org/Team:NCTU_Formosa/modeling#Device_modeling" class="external text" rel="nofollow">modeling of device</A> page). </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Fig.4-3-1 Result of 30 repeats of <I>Spodoptera Litura'</I>s preference testing for temperature and light (there are some moths staying in the bottom in every test). In this table, we can see blue light has a consistent attractive effect <I>Spodoptera Litura</I> under different temperature condition.</DIV></DIV></DIV></DIV><P>Fig.4-3-1 shows blue light has consistent attraction to our target harmful moth, <I>Spodoptera Litura</I>, in any temperature condition. Thus, we decided to use blue LED light in our device design.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig.4-3-2 Modeling tool (CCW No.1) for this test. We use blue, green, red, yellow, and positive control: white at the same time under different temperature conditions for our device modeling testing. <I>Spodoptera Litura</I>s were packed into the central barrel. Every test was followed by continuous beating on the barrel for 5 min in order to make the moths fly.</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:NCTU_Formosa/results">http://2014.igem.org/Team:NCTU_Formosa/results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NCTU_Formosa/results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NCTU_Formosa/results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NCTU_Formosa/results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NCTU_Formosa/results&amp;oldid=401571" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>