<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Princeton_ProjectDetails"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Princeton/ProjectDetails">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Princeton/ProjectDetails&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Princeton/ProjectDetails&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Princeton/ProjectDetails&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Princeton/ProjectDetails" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Princeton/ProjectDetails</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV style="display:inline;padding:0;margin:0;position:relative;left:0;top:0"><P style="font-variant: small-caps; font-size: 36px; color: orange; font-weight: bold">PRINCETON IGEM 2008</P><TABLE style="color:orange;background-color:black;min-height:80px;padding:0;border-style:solid;border-width:0" cellpadding="0" cellspacing="0" border="5" width="90%" align="right"><TBODY><TR><TH align="center"><A href="/Team:Princeton" title="Team:Princeton">Home</A></TH><TH align="center"><A href="/Team:Princeton/Project_Overview" title="Team:Princeton/Project Overview">Project Overview</A></TH><TH align="center"><STRONG class="selflink">Project Details</STRONG></TH><TH align="center"><A href="/Team:Princeton/Experiments" title="Team:Princeton/Experiments">Experiments</A></TH><TH align="center"><A href="/Team:Princeton/Results" title="Team:Princeton/Results">Results</A></TH><TH align="center"><A href="/Team:Princeton/Notebook" title="Team:Princeton/Notebook">Notebook</A></TH></TR></TBODY></TABLE><TABLE style="color:orange;background-color:black;min-height:80px;padding:0;border-style:solid;border-width:0" cellpadding="0" cellspacing="0" border="5" width="90%" align="right"><TBODY><TR><TH align="center"><A href="/Team:Princeton/Parts" title="Team:Princeton/Parts">Parts Submitted to the Registry</A></TH><TH align="center"><A href="/Team:Princeton/Modeling" title="Team:Princeton/Modeling">Modeling</A></TH><TH align="center"><A href="/Team:Princeton/Team" title="Team:Princeton/Team">The Team</A></TH><TH align="center"><A href="/Team:Princeton/Gallery" title="Team:Princeton/Gallery">Gallery</A></TH></TR></TBODY></TABLE></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#The_Toggle_Switch"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">The Toggle Switch</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Plasmid_Designs"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Plasmid Designs</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Surface_Patterning"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Surface Patterning</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#The_Learning_Circuit"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">The Learning Circuit</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#References"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="The_Toggle_Switch">The Toggle Switch</SPAN></H2><P>‎‎
</P><P>
A toggle switch is a bi-stable system where two inputs are responsible for two different outputs - similar to an on/off switch. The system depicted on the right consists of three groups of cells: one pacemaker group (P) and two groups of neurons (B1 and B2). By nature, pacemaker cells do not need to be activated themselves, but can continuously activate B1 and B2. 
</P><P>B1 and B2 consistently inhibit each other, which requires their signals to be able to overcome the excitation of the pacemaker cells. Activation by P allows B1 and B2 to continuously inhibit B1 and B2. When Input 1 is introduced into the system, B1 is inhibited and therefore cannot, in turn, inhibit B2. Thus, we have B2 as an output. Conversely, when Input 2 is applied, B2 is inhibited and B1 is the output.
</P><P>If at any time both inputs were introduced into the system, the system would enter a race condition. In engineering theory, it is impossible to predict what the outcome of the race would be; in practice, however, because of the nuances in strength and degradation time of the two inhibitory signals, one would eventually win out over the other. For our purposes, one input will be introduced into the system so as to gain a singular output and test the functionality of our design.
</P><P>The diagram to the right shows the neurotransmitters we have chosen to control the neuronal toggle switch. In this case, the excitatory signal produced by the pacemaker cells is a result of dopamine, in combination with Dopamine Receptor 5 (D5R) located on the B1 and B2 neurons. Their activation causes the production of glycine by the B2 cells and GABA by the B1 cells. B2 cells are designed to be receptive to inhibition by the neurotransmitter produced by the B1 cells (GABA) and vice versa for B1 cells. When both groups are excited by the pacemaker cells and inhibit one another equally, the system enters into an unpredictable race state. If there is more of one inhibitory neurotransmitter in the system, the group receptive to it will be repressed and the other group will &quot;win&quot;. The key to the R/S latch and, in fact, our whole system, is that it is designed in such a way that when the input is taken away, the system will enter a &quot;hold&quot; state during which it will continue its last known state, even in the absence of new signals.
</P>‎
<P>Specific genes used to achieve the differentiation of the desired neuron types for the system:
</P><P><B>Pacemaker:</B></P><UL><LI>Cav3.1 - Allows for continuous activation in cells, or differentiation into the pacemaker type
</LI><LI>Nurr1 - Dopaminergic 
</LI></UL><P><B>B1:</B></P><UL><LI>DRD5 - Dopamine Receptor 5
</LI><LI>GLRA1 - Glycine Receptor
</LI><LI>Lbx1 - GABAergic
</LI></UL><P><B>B2:</B></P><UL><LI>DRD5 - Dopamine Receptor 5
</LI><LI>GABAR - GABA Receptor (specific gene still not found due to multiple subunits)
</LI><LI>GLYT2 and VIATT - Glycinergic
</LI></UL><H3><SPAN class="mw-headline" id="Plasmid_Designs">Plasmid Designs</SPAN></H3><P>We worked on designing and constructing several plasmids this summer. Our plasmids were on the whole designed using PCR SOEing (Splicing by Overlap Extension). The process involves PCRing the pieces of the plasmid insert separately, but adding one or more &quot;overlap&quot; regions to each insert section that consists of the end sequence of the neighboring section. 
</P><P>For example, if the insert to be put into the vector is A-B-C, then A would be PCRed with a bit of the beginning of B added onto the end using a primer overhang. B would be PCRed with a bit of the beginning of C added on (the end of A need not be added onto the beginning of B because B already has an overlap with the beginning of its sequence that was added onto the end of A). Then A could be &quot;SOEed&quot; to B by running a PCR reaction of equimolar amounts of A and B, the forward primer of A and the reverse primer of B. A and B will anneal to each other and acts as primers in addition to the oligos in the reaction. B and C could similarly be SOEed, followed by A-B and B-C. A and B and C could also be SOEed together in a single reaction, with every piece acting as primer, but there are decreasing outputs the more pieces that one attempts to SOE in a single reaction.
</P><P>A list of plasmids we have designed and worked on is <A href="/Team:Princeton/ListofPlasmidDesigns" title="Team:Princeton/ListofPlasmidDesigns"> given here</A>, each linking to a design page. Some plasmids were redesigned, and hence appear more than once, with a number tacked on the end. At the top of each page the name of the plasmid is listed, followed by the vector, with the position and names of the restriction sites used to cut, and the insert, similarly marked with restriction sites. If the enzymes do not match it they have compatible overhangs.
</P><P><B>PCR SOEing and Sequencing Primers</B></P><P>We created over 100 primers for the construction and sequencing of our plasmids. For the sequences and properties of each primer, as well as which primers were used for which plasmids, <A href="https://2008.igem.org/Media:PrimerOligos.xls" class="external text" rel="nofollow">click here</A>.
</P><H3><SPAN class="mw-headline" id="Surface_Patterning">Surface Patterning</SPAN></H3><P><B>Previous Work I - Preparation of Gold Surface for Cell Adhesion</B></P><P>M.B. Shenai et al (2003) have demonstrated a protocol for viable cell adhesion. A layer
of chrome or titanium is deposited on a glass slide, followed by the deposition of a layer
of gold by electron beam evaporation in the required pattern. Once this surface has been
cleaned of organic impurities, it is covered by a 7.5M solution of mercaptoundecanoic
acid (MUA), which adheres to the gold and has carboxyl groups on the other end. These
groups are activated (using a carbodiimide) and they attach to the amide groups in a
synthetic polymer called a dendrimer that forms the final layer on top of the gold. The
image below shows a cross-section of the surface once it is ready for the addition of
medium and cell adhesion to the dendrimer.
</P><P>The MUA is attached to the gold, while the EDC activates the carboxyl groups of the
MUA to attach to the amide groups of the dendrimer.
</P><P><B>Previous Work II - Patterning of the Gold Surface</B></P><P>It has been shown that plating neurons in defined patterns gives the emerging neural
networks properties that are directly related to those patterns.  Feinerman, Rotem, and Moses of the
Weizmann Institute demonstrated that triangular plating, for example, encourages the
propagation of asymmetric signals and directed axonal growth, thereby acting somewhat
similar to a diode (personal communication).
</P><P>The procedure for creating the surface patterns is as follows. The mask design is created on L-Edit, which is a mask-designing software. A neuronal toggle switch is designed to look like <A href="https://2008.igem.org/Image:Pattern_Princeton.jpg" class="external text" rel="nofollow">this</A>. The idea is not only to preferentially form connections based on neurotransmitters as specified through differentiation of cells, but also by breaking the symmetry of networks by introducing triangles. 
</P><P>Once the pattern is designed, it is used to form gold patterns on a glass slide. The procedure described above is then carried out to attach the thiol, attach the dendrimers, and encourage &quot;selective cell adhesion&quot;. For experimental purposes, the pattern shown, and only two triangles connected to each other, will be created.
</P><H2><SPAN class="mw-headline" id="The_Learning_Circuit">The Learning Circuit</SPAN></H2>‎‎
<P>
Our goal for the learning circuit was to control long-term potentiation and long-term depression in order to teach a self-formed neuronal network to recognize a desired input. We were also focused on training a neuronal network to recognize a specific stimulus by giving a &quot;correct&quot; response when exposed to the &quot;correct&quot; input. The diagram shown above depicts our desired placement for each step of the process, as the circuit would be coupled with the surface patterning described above.  
</P>‎‎
<P>
In this diagram, the picture on the left is of a neuronal network that will be plated with a &quot;correct&quot; input to create an excitatory signal. This would cause a cascade of potentiation throughout the network pathway and cause neuronal firing, achieving the desired output. On the right side, the network is plated with a different input inducing an inhibitory signal. That signal causes depotentiation to occur across the network, resulting in the &quot;wrong&quot; output. By building long-term potentiation through bouts of long-term depression, we can hopefully teach the network to recognize the &quot;correct&quot; input.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI> Nanotexturing Gold Surfaces with Dendrimers for Selective Cell Adhesion 
</LI></UL><P>M.B. Shenai et. al; 2003
</P><UL><LI> Feinerman, O. Rotem, A. and E. Moses. Reliable neuronal logic devices from patterned hippocampal cultures, Nature Physics 2008 Dec; 4(12):967-73
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Princeton/ProjectDetails">http://2008.igem.org/Team:Princeton/ProjectDetails</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Princeton/ProjectDetails" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Princeton/ProjectDetails" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Princeton/ProjectDetails&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Princeton/ProjectDetails&amp;oldid=106323" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>