<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCC_Ireland_Projects_SeeDNA_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCC_Ireland/Projects_SeeDNA.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCC_Ireland/Projects_SeeDNA.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCC_Ireland/Projects_SeeDNA.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCC_Ireland/Projects_SeeDNA.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCC_Ireland/Projects_SeeDNA.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCC Ireland/Projects SeeDNA.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="nav"><DIV id="nav_wrapper"><NAV><UL><LI><A style="margin-top:1px">≡</A></LI><LI><A href="#fbsection1" class="cbp-fbcurrent">SeeDNA - Overview </A></LI><LI><A href="#fbsection2">DNA diagnostics </A></LI><LI><A href="#fbsection3">Detector Design </A></LI><LI><A href="#fbsection4">HPV target </A></LI><LI><A href="#fbsection5">Sry target </A></LI><LI><A href="#fbsection6">Detector Testing and Optimization </A></LI><LI><A href="#fbsection7">Commerical </A></LI></UL><LI class="non-list"><A href="https://igem.org/Main_Page"><B>|</B></A></LI><LI class="non-list"><A href="https://2014.igem.org/Team:UCC_Ireland"><B>UCC iGEM 2014</B></A></LI><UL class="right"><LI><A href="https://2014.igem.org/Team:UCC_Ireland">Home</A></LI><LI>Projects
                            </LI><LI><A href="https://2014.igem.org/Team:UCC_Ireland/Projects_SeaDNA.html">SeaDNA</A></LI><LI><A href="https://2014.igem.org/Team:UCC_Ireland/Projects_SeeDNA.html">SeeDNA</A></LI></UL><LI><A href="https://2014.igem.org/Team:UCC_Ireland/Team">Team</A></LI><LI>Entrepreneurship
                        </LI><LI><A href="https://2014.igem.org/Team:UCC_Ireland/Entre_Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:UCC_Ireland/Entre_SeaDNA">SeaDNA</A></LI><LI><A href="https://2014.igem.org/Team:UCC_Ireland/Entre_SeeDNA">SeeDNA</A></LI><LI><A href="https://2014.igem.org/Team:UCC_Ireland/Entre_Practice_Outreach.html">Practice and Outreach</A></LI><LI>Notebook
                        </LI><LI><A href="https://2014.igem.org/Team:UCC_Ireland/Notebook_SeaDNA">SeaDNA</A></LI><LI><A href="https://2014.igem.org/Team:UCC_Ireland/Notebook_SeeDNA">SeeDNA</A></LI><LI><A href="https://igem.org/Safety/Safety_Form?team_id=1518">Safety Form</A></LI><LI><A href="https://2014.igem.org/Team:UCC_Ireland/OurLab_Main">Our Lab</A></LI></NAV></DIV></DIV><DIV id="cbp-fbscroller" class="cbp-fbscroller"><NAV><A href="#fbsection1" class="cbp-fbcurrent">Section 1</A><A href="#fbsection2">Section 2</A><A href="#fbsection3">Section 3</A><A href="#fbsection4">Section 4</A><A href="#fbsection5">Section 5</A><A href="#fbsection6">Section 6</A><A href="#fbsection7">Section 7</A></NAV><SECTION><DIV class="content"><H1>Abstract</H1><DIV class="newspaper2"><P><I>SeeDNA</I> - aims to create a system for bacterial-based DNA detection.  We have designed customizable plasmids that can be activated to detect any specific short DNA sequence.  This system potentially provides digital quantification of target DNAs – representing an alternative to digital and real-time PCR.  Interaction of the target sequence with the plasmid and subsequent transformation into <I>E. coli</I> results in bacterial cell growth and production of a visible fluorescent signal.   As proof-of-principle we have designed a HPV-detector plasmid that detects a short sequence from the human papilloma virus.  This could be a rapid and cheap diagnostic tool for pathogenic DNA, and a revolution in resource poor hospital labs in developing countries or in an agricultural or industrial setting.</P></DIV><H1>Project Overview</H1><DIV class="newspaper2"><P>DNA diagnostics is a large and expanding biomedical/biotech sector.  Traditionally DNA diagnostics employs  PCR-based methods to identify disease associated genetic variations and to detect pathogens.  Despite their success, PCR-based methods have limitations for accurate DNA quantification and are expensive, limiting their application in point-of-care, cost critical and resource poor healthcare settings.  </P><P><I>SeeDNA</I> - aims to create a system for bacterial-based DNA detection.  We have designed DNA detector system that can in principle detect any short DNA sequence.  This DNA detector system potentially provides digital quantification of target DNAs – representing an alternative to digital and real-time PCR.  In our DNA detector system, interaction of the target sequence with the plasmid and subsequent transformation into <I>E. coli</I> results in bacterial cell growth and production of a visible fluorescent signal as readouts.   As proof-of-principle we have designed a HPV-detector plasmid that detects a short sequence (HPV target) from the human papilloma virus and a Y chromosome detector (SRY target).  The results of our testing and optimization to date confirm the versatility and specificity of the system.  Improvements to the sensitivity of the <I>SeeDNA</I> technology could see it become a simple, rapid DNA diagnostic tool that could revolutionize DNA diagnostics in resource poor hospital labs in developing countries or in cost critical agricultural or industrial settings.  Our ultimate goal is to develop commercial applications of <I>SeeDNA</I> technology.</P></DIV><H1>Graphical Overview</H1><DIV class="caption"><P><B>Fig. 1 SeeDNA Graphical Overview.</B></P></DIV></DIV></SECTION><SECTION><DIV class="banner1" id="#fbsection2" style="background-image: url(https://static.igem.org/mediawiki/2014/b/b8/SeeDNA_HPV_Detected_GFP.jpg)"><DIV class="banner_content2" style="background-color: #35873A"><H1 class="banner_content1">DNA Diagnostics</H1></DIV></DIV><DIV class="content"><H1>DNA Molecular Diagnostics</H1><DIV class="newspaper2"><P>The rapid and sensitive detection of genetic mutations and pathogenic organisms is vital for improving patient care. Molecular diagnostics aid in genetic counselling, identifying suitable treatments, preventing the spread of disease, and identifying sources of infection in medical laboratory, home or field settings (Allegranzi et al, 2011). For pathogen detection the use of DNA-based methodology is derived from the premise that each species of pathogen carries unique DNA or RNA sequences that differentiate it from other organisms.</P></DIV><H1>PCR</H1><DIV class="newspaper2"><P>Numerous molecular diagnostic methods have evolved in biomedical laboratories.  Polymerase chain reaction (PCR) technology is the current gold standard for molecular diagnostic tests. PCR is an <I>in vitro</I> method of amplifying and detecting highly specific nucleotide sequences.  Developed in 1983 by Kary Mullis, PCR is now an indispensable technique used in biomedical and research labs for various applications including DNA cloning, sequencing, functional analysis of genes; hereditary disease diagnosis; forensics, paternity testing; and the detection and diagnosis of infectious diseases. The 1993 Nobel Prize in Chemistry was awarded to Mullis - along with Michael Smith - for his work on PCR.</P></DIV><DIV class="caption"><P><B>Fig. 2 PCR.</B> The method relies on thermal cycling, i.e. cycles of repeated heating and cooling of the reaction for DNA melting and replication of the DNA by polymerase enzymes. Primers (short DNA fragments) which contain sequences that are complementary to the target region along with a DNA polymerase are key components to enable specific and repeated amplification. As PCR progresses, the DNA generated is itself used as a template for replication, setting in motion a chain reaction in which the DNA template is amplified exponentially.  Figure Source: Wikipedia Commons (User: Enzoklop) 
                    </P></DIV><H1>Limitations of PCR</H1><DIV class="newspaper2"><P>The sensitivity of PCR arises from the exponential amplification of the template DNA.  However, this feature of PCR can also be problematic when it comes to accurate DNA quantification because any errors during reaction set up are also amplified exponentially.  Also avoiding contamination of samples with the product of a previous PCR reaction is a concern.  </P><P>To quantify a target DNA molecule, quantitative real-time PCR (qPCR)-is often used.  Diagnostic qPCR is applied to detect nucleic acids that are diagnostic of, for example, infectious diseases, cancer and genetic abnormalities and has greatly improved the diagnosis of infectious diseases such as new strains of flu, Sails AD (2009).  However, quantitative real-time PCR (qPCR)-based systems are often too expensive for resource-limited environments (Niemz et al, 2011). </P></DIV><H1>Digital PCR</H1><DIV class="newspaper2"><P>The latest procedure to emerge in the field is Digital PCR. In digital PCR, a single copy of the target sequence enters a compartment.  Each compartment is then allowed to amplify the target, if present, by end point PCR.  The compartments are then screened for the presence of the target. Digital PCR in essence counts the number of copies of target template DNA are present in a sample providing accurate quantification.   Digital PCR machines can cost on average $100,000, not including the specialist reagents required.  Digital PCR has yet to become a mainstream diagnostic tool in clinical labs.</P><P>Our <I>SeeDNA</I> technology may also act as a quantifying system bestowing many of the benefits of digital PCR. The number of discrete fluorescent cells or colonies on the plate would be indicative of the number of copies of the target sequence in the original sample giving a cheap, simple and informative readout.</P></DIV><H2>References</H2><DIV class="newspaper1"><UL><LI>Allegranzi, B. <I>et al.</I> Burden of endemic health-care-associated infection in developing countries: systematic review and meta-analysis. <I>Lancet</I><B>377</B>, 228–241 (2011).Sails AD (2009). &quot;Applications in Clinical Microbiology&quot;. <I>Real-Time PCR: Current Technology and Applications.</I> Caister Academic Press. ISBN 978-1-904455-39-4.</LI><LI>Niemz, A., Ferguson, T. M. &amp; Boyle, D. S. Point-of-care nucleic acid testing for infectious diseases. <I>Trends Biotechnol</I>. <B>29</B>, 240–250 (2011).</LI><LI>Baker, M. Digital PCR hits its stride <I>Nature Methods</I>. <B>9</B>, 541–544 (2012).</LI></UL></DIV></DIV></SECTION><SECTION><DIV class="banner1" id="#fbsection3" style="background-image: url(https://static.igem.org/mediawiki/2014/b/b8/SeeDNA_HPV_Detected_GFP.jpg)"><DIV class="banner_content1" style="background-color: #35873A"><H1 class="banner_content1">Detector Design</H1></DIV></DIV><DIV class="content"><H1>Detector Design</H1><DIV class="newspaper2"><P>The concept behind the <I>SeeDNA</I> technology is to design a detector plasmid such that following standard transformation into <I>E. coli</I>,  the detector plasmid will only support the formation of colonies or reporter gene expression if it has encountered a specific short linear target DNA sequence.  </P><P>We designed several detector plasmid prototypes.  All the detectors comprise a standard Biobrick cloned into the standard pSB1-C3 plasmid that following an activation step should have the properties outlined above.  Several prototypes were developed and tested.   From these prototypes a successful system for the design and preparation of plasmid-based DNA detectors was eventually devised.  </P><P>We feel that potentially this method may have commercial applications.  To keep open the option of a patent application we are therefore not disclosing the details of the detector design and preparation.  Once a patent has been secured, or if it is decided that a patent application is not merited, we would plan to make the <I>SeeDNA</I> parts (Biobricks) available through the iGEM Registry for non-commercial entities. </P><P>The <I>SeeDNA</I> detector plasmid prototype is a standard Biobrick cloned into the standard pSB1-C3 plasmid.  Following transformation into <I>E. coli</I>,  the detector plasmid will only support the formation of colonies if it has encountered a specific short linear target DNA sequence (see Figure below, part A and B).  Since bacterial growth is an exponential process, the signal amplification that can be achieved is similar to PCR.  The sensitivity of the system is thus primarily limited by the efficiency of target detection and the transformation efficiency.  </P><P>Although bacterial growth as a readout provides exponential signal amplification it has a number of disadvantages in that it is rather slow and requires plating of cells on antibiotic agar plates.  Expression of a reporter gene at high levels from a high copy number plasmid should also provide significant signal amplification.  We therefore modified our DNA detector by adding a Registry Part that encodes a green fluorescent protein reporter under the control of a constitutive promoter (BBa_K584001).  This should in theory permit quicker DNA detection by visualizing GFP fluorescence under a microscope or by fluorescence activated cell sorting (FACS).</P></DIV><DIV class="caption"><P><B>Fig. 3 Schematic depiction of the read out from the <I>SeeDNA</I> DNA Detector system.</B><B>A) </B>Bacterial growth on an agar plate containing chloramphenicol as a readout of the presence of the target DNA 
                        <B>B) </B>Absence of bacterial growth when the target is not present
                        <B>C) </B>GFP reporter gene fluorescence as an alternative readout from an alternative version of the DNA detector system. 
                    </P></DIV></DIV></SECTION><SECTION><DIV class="banner1" id="Div1" style="background-image: url(https://static.igem.org/mediawiki/2014/b/b8/SeeDNA_HPV_Detected_GFP.jpg)"><DIV class="banner_content2" style="background-color: #35873A"><H1 class="banner_content1">HPV target</H1></DIV></DIV><DIV class="content"><H1>Summary</H1><DIV class="newspaper2"><P>As proof of principle for our SeeDNA technology we aimed to create a system for bacterial-based detection of highly pathogenic strain of Human Papilloma virus (HPV).  A 55bp HaeIII restriction fragment from the genome of a (HPV16) was chosen as the target for detection.  Oligonucleotides corresponding to this target region were synthesised and a plasmid containing this target was generated in order to test our HPV detector. </P></DIV><H1>Background</H1><H2>Human Papilloma virus </H2><DIV class="newspaper2"><P>HPV is a common pathogen worldwide. The human papilloma virus (HPV) is a small double-stranded DNA virus with a genome of 8000 base pairs. It infects the epithelial cells of skin and mucosa (Cutts et al., 2007). Theses epithelial surfaces (squamous cells) include all areas covered by skin and/or mucosa such as the mouth interior, throat, tongue, tonsils, vagina, cervix, vulva, penis (the urethra - the opening), and anus. Transmission of the virus occurs when these areas come into contact with a virus, allowing it to transfer between epithelial cells. Over 170 HPV types have been identified and, each HPV strain is denoted by a number. Of all the HPV variants, HPV16 is one of the most carcinogenic &quot;high-risk&quot; HPV stains. HPV16 is implicated in oropharyngeal (throat) cancer, while it also plays a role in cervical cancer (Muñoz et al., 2003).  </P><P>It has been postulated that HPV16 infects epithelial tissues through micro-abrasions of epitheliums; here HPV16 partners with receptors located on the basement membrane such as alpha integrins and laminins. This leads to entry of the virus into basement membrane cells. HPV16 accomplishes entry via clathrin- and caveolin-mediated endocytosis (Laniosz et al., 2009) . At this point, the viral genome is transported to the nucleus where it establishes itself at a copy number between 10-200 viral genomes per cell.  HPV Type 16, causes the most incidences of cervical cancer cases worldwide, hence a target from this genome (Ref: NC_001526) was chosen.</P><P>HPV16’S L1 gene codes for the L1 protein, the major capsid protein of HPV16. By spontaneous self-assembly, five of these L1 proteins form a homopentamer called a capsomere (Heino et al., 1995).72 of these capsomeres come together to form a capsids. The capsomeres are held together by disulphide bonds and the minor capsid protein L2, which stabilise the overall capsid structure.</P></DIV><DIV class="caption"><P><B>Fig. 4 Structure of HPV 16 capsid</B></P></DIV><H1>Results</H1><H2>1. Identification of a HPV target sequence </H2><DIV class="newspaper2"><P>Most probes used to detect the virus today target the L1 gene which codes for the major capsid protein of the virus. As this region has been used as a genetic probe for the virus with success, its sequence has shown to be robust and reliable as a target. We chose to identify a target from this region also.  The target sequence chosen corresponds to a fragment that is generated from the HPV16 genome upon digestion with the 4-cutter enzyme HaeIII. This generates a 55 base pair fragment.  A number of factors had to be considered when choosing the target sequence. This included the considerations similar to when choosing a primer such as GC content, dimerization and annealing temperature.  This fragments low GC content (54.05%) mean that there is a low chance that it will form hairpin structures, making it an ideal target sequence to detect.  </P></DIV><DIV class="caption"><P><B>Fig. 5 Depiction of the sequence of the HPV target region within the L1 gene of the HPV16 virus genome.</B></P></DIV><H2>2. Choosing a Restriction Enzyme</H2><DIV class="newspaper2"><P>Our target was chosen as it was flanked by restriction sites of HaeIII. A number of restriction sites could have been chosen, however our choice was limited by factors such as the iGEM restriction sites located on the biobrick prefix and suffix.  A target had to lack these sites to create a biobrick part that could be used by other teams. </P><P>The HaeIII restriction enzyme bluntly cuts at two GGCC recognition sites within HPV16’s L1 gene.  HaeIII is not affected by DNA methylation and so the 55bp target sequence should be readily released upon HaeIII digestion of DNA prepared from a clinical sample in which HPV  infection is present. </P></DIV><DIV class="caption"><P><B>Fig. 6 Sequence of the HPV target region of the HPV16 virus as depicted in Benchling software. </B>
                       Oligonucleotides corresponding to both the top and bottom strand of the 55bp target were used to test the HPV detector system.
                    </P></DIV><H2>3.	Single Stranded HPV Target Preparation</H2><DIV class="newspaper2"><P>The sequence of our target was chosen by analysing the HPV16 genome on bioinformatics tool Benchling.  Either the top or bottom strand of this sequence was synthesized and phosphorylated. This was carried out as described in our protocol file. The single stranded target was then ready for reaction with our HPV detector plasmid.</P></DIV><H2>4.	Double Stranded HPV Target Preparation</H2><DIV class="newspaper2"><P>Both the top and bottom strands only of the 55bp target sequence were synthesized and phosphorylated. The two complimentary were then annealed together to generate a double stranded target.  This product was then reacted with the HPV detector plasmid.  This was carried out as described in our protocol file.</P></DIV><H2>5.	Cloning of HPV Target into a Plasmid</H2><DIV class="newspaper2"><P>Overlapping forward and reverse oligonucleotides containing the HPV target and flanking HaeIII recognition sites and that had overhanging XhoI and EcoRI restriction sites were synthesized.  Double stranded DNA was generated by primer extension in a standard PCR reaction.  Following digestion with XhoI and EcoRI enzymes this target DNA was cloned into the pBluescript SKII vector that had been cut with the same enzymes.  This was done to mimic the true structure of the HPV target as it would be found in a clinical sample; as a double stranded fragment of a circular piece of DNA.  Since we could not work with actual HPV genomic DNA for safety reasons, this “target plasmid” served as a surrogate for actual HPV DNA.  pBluescript SKII was chosen as a plasmid that was not closely related to the Detector plasmid pSB1-C3. </P></DIV><H2>References</H2><DIV class="newspaper1"><UL><LI>Cutts, F. T., Franceschi, S., Goldie, S., Castellsague, X., De Sanjose, S., Garnett, G., Edmunds, W. J., Claeys, P., Goldenthal, K. L., Harper, D. M. &amp; Markowitz, L. 2007. Human papillomavirus and HPV vaccines: a review. <I>Bull World Health Organ</I>, 85, 719-26.</LI><LI>Heino, P., Dillner, J. &amp; Schwartz, S. 1995. Human papillomavirus type 16 capsid proteins produced from recombinant Semliki Forest virus assemble into virus-like particles. <I>Virology</I>, 214, 349-59.</LI><LI>Laniosz, V., Dabydeen, S. A., Havens, M. A. &amp; Meneses, P. I. 2009. Human papillomavirus type 16 infection of human keratinocytes requires clathrin and caveolin-1 and is brefeldin a sensitive. <I>J Virol</I>, 83, 8221-32.</LI><LI>Muñoz, N., Bosch, F. X., De Sanjosé, S., Herrero, R., Castellsagué, X., Shah, K. V., Snijders, P. J. F. &amp; Meijer, C. J. L. M. 2003. Epidemiologic Classification of Human Papillomavirus Types Associated with Cervical Cancer. <I>New England Journal of Medicine</I>, 348, 518-527.</LI></UL></DIV></DIV></SECTION><SECTION><DIV class="banner1" id="Div2" style="background-image: url(https://static.igem.org/mediawiki/2014/b/b8/SeeDNA_HPV_Detected_GFP.jpg)"><DIV class="banner_content1" style="background-color: #35873A"><H1 class="banner_content1"><I>Sry</I> target</H1></DIV></DIV><DIV class="content"><H1>The <I>Sry</I> gene</H1><DIV class="newspaper2"><P><I>Sry</I> is a gene located on the Y chromosome.  It codes for sex-determining region Y (SRY) protein which is also known as Testis-determining factor (TDF) (Kashimada and Koopman, 2010).  SRY protein is transcription factor that is responsible for male sex determination in mammals.  </P><P>A target located on the Y chromosome was chosen to test our <I>SeeDNA</I> method because it would allow us to test both the sensitivity and specificity of the system in detecting a target sequence from genomic DNA.  The ability to detect and quantify target sequences from a genomic DNA sample would provide evidence that <I>SeeDNA</I> technology could be used for genetic testing of either inherited diseases for example.  By choosing a target located on the Y chromosome, it will be possible to test the detector with genomic DNA from a male and to use genomic DNA from a female as a negative control.  This would be a very good test of the sensitivity and specificity of the detection system.</P></DIV><H1>Results</H1><H2>1.	Identification of an Sry target sequence </H2><DIV class="newspaper2"><P>We identify a target sequence within a conserved region of the mouse Sry gene.  This 62bp region corresponds to a fragment that is generated upon digestion with the 4-cutter enzyme HaeIII. </P></DIV><H2>2.	Choosing a Restriction Enzyme</H2><DIV class="newspaper2"><P>Our target was chosen as it was flanked by restriction sites of HaeIII. A number of restriction sites could have been chosen, however our choice was limited by factors such as the iGEM restriction sites located on the biobrick prefix and suffix.  A target had to lack these sites to create a biobrick part that could be used by other teams. </P><P>The HaeIII restriction enzyme bluntly cuts at two GGCC recognition sites and is not affected by DNA methylation and so the 62bp target sequence should be readily released upon HaeIII digestion of male genomic DNA. </P></DIV><DIV class="caption"><P><B>Fig. 7 Sequence of the <I>Sry</I> target region from the mouse Sry gene located on the Y chromosome as depicted in Benchling software.  </B>
                        Oligonucleotides corresponding to both the top strand of the 62bp target were used to test the <I>Sry</I> detector system.
                    </P></DIV><H2>3.  Single Stranded <I>Sry</I> Target Preparation</H2><DIV class="newspaper2"><P>The sequence of our target was chosen by analysing the <I>Sry</I> gene on bioinformatics tool Benchling.  Either the top or bottom strand of this sequence was synthesized and phosphorylated. This was carried out as described in our protocol file.  The single stranded target was then ready for reaction with our <I>Sry</I> detector plasmid.</P></DIV><H2>4.  Cloning of Sry Target into a Plasmid</H2><DIV class="newspaper2"><P>Overlapping forward and reverse oligonucleotides containing the <I>Sry</I> target and flanking HaeIII recognition sites and that had overhanging XhoI and EcoRI restriction sites were synthesized.  Double stranded DNA was generated by primer extension in a standard PCR reaction.  Following digestion with XhoI and EcoRI enzymes this target DNA was cloned into the pBluescript SKII vector that had been cut with the same enzymes.  This was done to mimic the true structure of the <I>Sry</I> target.  This “target plasmid” was used to test the <I>Sry</I> detector. pBluescript SKII was chosen as a plasmid that was not closely related to the Detector plasmid pSB1-C3. </P></DIV><H2>References</H2><DIV class="newspaper1"><UL><LI>Kashimada, K., Koopman, P. (2010). Sry: the master switch in mammalian sex determination.  <I>Development</I>. 2010 Dec;137(23):3921-30</LI></UL></DIV></DIV></SECTION><SECTION><DIV class="banner1" id="Div3" style="background-image: url(https://static.igem.org/mediawiki/2014/b/b8/SeeDNA_HPV_Detected_GFP.jpg)"><DIV class="banner_content2" style="background-color: #35873A"><H1 class="banner_content1">Detector Testing and Optimization</H1></DIV></DIV><DIV class="content"><H1>Validation and versatility of HPV and <I>Sry</I> Detectors</H1><DIV class="newspaper2"><P>Initial testing of the detector involved experimenting our construct with the most simple targets – either single stranded or double stranded 55bp oligonucleotides corresponding to our target sequences. This sequence was chosen as described above and synthesised for testing with the detector. An excess of the target oligonucleotide was used as our key aim was to evaluate the detectors ability to bind to the target.  A number of conditions for activation of the HPV detector were initially tested.  A control was also set up which lacked the target oligo. </P></DIV><H2>Experiment 1 (HPV detector + single stranded oligonucleotide target)</H2><DIV class="newspaper1"><P>Reactions were set up as shown in Table 1.</P></DIV><DIV class="caption"><TABLE><TBODY><TR><TD>Activation Condition 1 </TD><TD>Activation Condition 2</TD><TD>Activation Condition 3</TD></TR><TR><TD>No HPV oligonucleotide</TD><TD>No HPV oligonucleotide</TD><TD>No HPV oligonucleotide</TD></TR><TR><TD>HPV target</TD><TD>HPV target</TD><TD>HPV target</TD></TR></TBODY></TABLE><P><B>Table 1. HPV detector + single stranded oligonucleotide target Experiment Reaction Set up. </B></P></DIV><DIV class="newspaper1"><P>Following one hour incubation the reactions were transformed into DH5alpha competent cells and plated on chloramphenicol plates and incubated overnight and the number of colony forming units (cfu) counted for each reaction. </P></DIV><DIV class="caption"><TABLE><TBODY><TR><TD>Condition 1</TD><TD>Condition 1  (repeated)</TD><TD>Condition 2</TD><TD>Condition 3</TD></TR><TR><TD>No oligo</TD><TD>2 cfu</TD><TD>2 cfu</TD><TD>1 cfu</TD><TD>72 cfu</TD></TR><TR><TD>ssHPV target oligo</TD><TD>1840 cfu</TD><TD>1712 cfu</TD><TD>1152 cfu</TD><TD>752 cfu</TD></TR></TBODY></TABLE><P><B>Table 2. HPV detector + single stranded oligonucleotide target Experiment Results. </B></P></DIV><DIV class="caption"><P><B>Fig. 8 Successful detection of an HPV target oligonucleotide</B>
                        Results shown were obtained using the HPV detector prepared using activation condition 1.  Picture of plates and colonies obtained after reaction of the HPV detector either in the absence (left) or presence (right) of a single stranded target HPV oligonucleotide.
                    </P></DIV><DIV class="newspaper1">
                    HPV target oligo was successfully detected under all conditions with very few colonies obtained in the absences of the oligo.  Activation condition 1 consistently gave the highest number of colonies and was used hereafter. 
                </DIV><H2>Experiment 2 (HPV detector - comparison of single and double stranded targets)</H2><DIV class="newspaper2"><P>Double stranded target sequences were prepared by annealing the top and bottom strand target oligonucleotides as described above.  This dsTarget was compared to a ssTarget corresponding to either the top or bottom strand.  Reaction volumes were scaled down compared to the previous experiment for ease of counting colonies. </P><P>Both ssHPV target oligo and dsHPV target oligos were successfully detected with no colonies obtained in the absences of oligos.</P></DIV><DIV class="caption"><TABLE><TBODY><TR><TD>Number of colonies</TD></TR><TR><TD>No oligo</TD><TD>0 cfu</TD></TR><TR><TD>ssHPV target oligo</TD><TD>200 cfu</TD></TR><TR><TD>dsHPV target oligo</TD><TD>60 cfu</TD></TR></TBODY></TABLE><P><B>Table 3. HPV detector comparison of single and double stranded targets.</B></P></DIV><H2>Experiment 3 (testing of the Sry detector)</H2><DIV class="newspaper2"><P>The <I>Sry</I> detector was prepared using activation condition 1 as for the HPV detector. To validate our second detector plasmid reactions were prepared as described above. </P><P>ssSry target oligos were successfully detected with no colonies obtained in the absences of oligos.  This validates a second detector without any alterations in preparation or reaction conditions, demonstrating the versatility and ease of use of the <I>SeeDNA</I> system.</P></DIV><DIV class="caption"><TABLE><TBODY><TR><TD>Number of colonies </TD></TR><TR><TD>No oligo </TD><TD>0 cfu</TD></TR><TR><TD>ssSry target oligo</TD><TD>125 cfu</TD></TR></TBODY></TABLE><P><B>Table 4. Results for <I>Sry</I> detector</B></P></DIV><H1>Specificity of the SeeDNA detector plasmids</H1><DIV class="newspaper2"><P>Having demonstrated that the detectors work with a pure oligonucleotide target we wanted to evaluate its ability to detect more complex targets. We wanted to examine the detectors ability to detect its target sequence when the target is inserted in a larger piece of DNA, such as a plasmid.  In other words, to test if target sequences could be detected from a mixture of DNA fragments.  This test would reveal the system’s potential ability to pick out its target from a larger DNA sample such as the entire human genome which would be present in a clinical sample.</P><P>To examine this issue the HPV target and <I>Sry</I> target sequences with their flanking HaeIII sites were cloned into a plasmid (pBluescript SK II) that was unrelated to the detector plasmid (pSB1-C3) and that had a different antibiotic resistance gene.  These “target plasmids” were then digested with HaeIII to release the 55bp target sequence along with a number of other HaeIII restriction fragments.  The parental empty pBluescript plasmid was also digested in the same manner.  Uncut “target plasmids” were prepared as a an additional  control.   </P></DIV><DIV class="caption"><P><B>Fig. 9 Preparation of HPV and Sry target plasmids.</B><B>LEFT) </B>
                        HPV and <I>Sry</I> target plasmids were digested with HaeIII and a small quantity analysed by electrophoresis on a 2% agarose gel.  The parental pBluescript plasmid was also digested.  The digest of the target plasmids is expected to yield 14 fragments, one of which is the short 55/62bp target fragment.  However the actual target fragments are too small to see clearly.    
                        <B>RIGHT) </B>
                        Undigested HPV and <I>Sry</I> target plasmids.
                    </P></DIV><DIV class="newspaper2"><P>The digested plasmid DNA along with the intact control DNAs were combined with the detector plasmids as for the oligonucleotide targets above.  </P><P>Both the HPV detector and the Sry detector were able to specifically detect their target when it was released from the target plasmid and present in a mixture of other DNA fragments.  This provides some evidence of the specificity of the <I>SeeDNA</I> detection system.</P></DIV><DIV class="caption"><TABLE><TBODY><TR><TD>Number of colonies</TD></TR><TR><TD>HaeIII digested pBluescript</TD><TD>0 cfu</TD></TR><TR><TD>HaeIII digested pBluescript+HPV target</TD><TD>82 cfu</TD></TR><TR><TD>Uncut pBluescript</TD><TD>0 cfu</TD></TR></TBODY></TABLE><P><B>Table 5a. HPV Detector</B></P></DIV><DIV class="caption"><TABLE><TBODY><TR><TD>Number of colonies</TD></TR><TR><TD>HaeIII digested pBluescript</TD><TD>0 cfu</TD></TR><TR><TD>HaeIII digested pBluescript+Sry target</TD><TD>23 cfu</TD></TR><TR><TD>Uncut pBluescript</TD><TD>0 cfu</TD></TR></TBODY></TABLE><P><B>Table 5b. <I>Sry</I> Detector</B></P></DIV><H1>Optimization of the <I>SeeDNA</I> system to improve sensitivity</H1><DIV class="newspaper2"><P>Results so far show that in principle the <I>SeeDNA</I> system is working as expected.  To be useful the system must be able to detect low amounts of target DNA.  We have begun to systematically optimize each step and parameter in our detection protocols to improve efficiency.  A big increase in efficiency can probably be achieved by using cells that are more highly competent.  We focused on other parameters initially.  </P></DIV><H2>Experiment 1 (optimization of temperature of the detection reaction)</H2><DIV class="newspaper2"><P>We varied the temperature at which the activated detector plasmid is incubated with the target DNA.  For this we used the HPV detector and dsHPV target oligonucleotides. By varying this simple parameter a significant improvement in detection efficiency was obtained.  It is hoped that the optimization of further parameters can yield even more improvements.</P></DIV><DIV class="caption"><P><B>Fig 10. Optimization of detection reaction temperature.</B>
                        Number of colonies obtained when the detection reaction was incubated at various temperatures.  Temperature 5 represents a 2 fold improvement over the starting conditions (Temp 2).  Data courtesy of Daniel James (non team member).
                    </P></DIV><H1>Testing a fluorescent read out for the <I>SeeDNA</I> system</H1><DIV class="newspaper2"><P>To test this fluorescent HPV detector plasmid we used the HaeIII digested pBluescript+HPVtarget as our target, with HaeIII digested pBluescript (empty vector) as a control.  Detection reactions were set up and transformed as usual.  One hour after transformation chloramphenicol was added to the transformation reaction and it was incubated in a 37 degree shaker.  Transformed cells were examined at various time points under a fluorescent microscope.  </P></DIV><DIV class="caption"><P><B>Fig 11. Fluorescent detection of HPV target sequence with <I>SeeDNA</I> technology.</B>
                        Cell transformed with the pSB1-C3-K584001 HPV Detector plasmid that had been mixed with either HaeIII digested pBluescript plasmid (left) HaeIII digested pBluescript+HPVtarget target plasmid (right) were imaged at the indicated  times post transformation.  DAPI staining of DNA served as an aid to focus on cells in the absence of widespread GFP fluorescence at early time points.   
                    </P></DIV><DIV class="newspaper2"><P>Within a few hours of transformation single fluorescent cells were visible in the sample to which the digested HPVtarget plasmid had been added (see Figure below), but not in the control sample.  Following an overnight incubation, bacterial growth has provided massive signal amplification with large numbers of green cells visible, but again no cells were present in the control sample.  </P><P>Identification of GFP positive cells directly after transformation should provide a very accurate, quantitative measure of the amount of target sequence in a sample.  Essentially this is a digital detection system analogous to digital PCR, but with expression of a reporter gene at high levels from a high copy number plasmid providing the signal amplification.  Fluorescent detection thus permits quicker and probably more quantitative DNA detection that can be visualized with a fluorescence microscope or by fluorescence activated cell sorting (FACS).</P></DIV></DIV></SECTION><SECTION><DIV class="banner1" id="Div4" style="background-image: url(https://static.igem.org/mediawiki/2014/b/b8/SeeDNA_HPV_Detected_GFP.jpg)"><DIV class="banner_content1" style="background-color: #35873A"><H1 class="banner_content1">Commercial</H1></DIV></DIV><DIV class="content"><H1>Towards Proof of Principle </H1><DIV class="newspaper2"><P>In this project we have devised a strategy for constructing plasmids which can detect specifically a genetic sequence and give a readout when transformed into <I>E. coli</I>.  As the method by which the plasmid was constructed was novel, full details were not disclosed to keep the possibility of patenting open.  However the results presented here for two different DNA detector plasmid demonstrate clearly that the system works.  </P><P>Plasmid samples which were incubated with the target sequence gave a large numbers of colonies when transformed (typically &gt;100 CFU per plate).  Whereas controls, which lacked target DNA, when incubated with the plasmid, gave little or no colonies when transformed.  Slightly less efficient, but still conclusive, results were achieved with a double stranded DNA target compared to single stranded oligos.  We have gone some way towards demonstrating the specificity of the system for a target sequence.  We have also begun the process of optimizing the system to increase its sensitivity.  Our initial goal is to demonstrate that <I>SeeDNA</I> could be used to specifically detect a target sequence from restriction digested genomic DNA.  If this level of sensitivity and specificity can be achieved following further testing and optimisation, it would suggests that the <I>SeeDNA</I> diagnostics could work in a clinical setting.  Its success in detecting both single stranded and double stranded targets mean it can be used to probe for many targets in molecular diagnostics.</P><P>In summary our experiments showed the potential for<I>SeeDNA</I> to be used in a variety of applications but further optimisation is required to maximise its diagnostic ability.  </P></DIV><H1>Potential Applications</H1><DIV class="newspaper1"><P><I>SeeDNA</I> is a versatile technology.  We anticipate that it could be used for the following applications</P></DIV><H2>Detection of Pathogenic DNA</H2><DIV class="newspaper2"><P>This system could be a rapid and cheap diagnostic tool for DNA sequences from pathogenic bacteria or viruses. We have designed a detector for the double stranded Human Papillomavirus (HPV) and it should be possible to detect other DNA viruses like Varicella-zoster virus (Chickenpox) and potentially RNA viruses like Human Immunodeficiency virus.  Bacteria that cause food poisoning or infection should also be detectable </P></DIV><H2>Mutation detection</H2><DIV class="newspaper2"><P>Many diseases arise by mutations in DNA that result in a faulty protein. <I>SeeDNA</I> could in principle be designed to detect the presence of specific mutations (insertions, deletions, substitutions).  A detector plasmid could be constructed which could detect known mutations or to detect only non-mutated sequences.  <I>SeeDNA</I> could be a quick screening diagnostic.</P><P>For example it could be used as the readout of a restriction fragment length polymorphisms (RFLPs) as an alternative to gel electrophoresis or other forms of chromatography.  RFLP (Restriction Fragment Length Polymorphisms) are ideal targets for this detector as binding to the target relies heavily on a target of the correct length.  As such it could be used for genetic testing of for example Cystic Fibrosis  or Sickle Cell Anaemia (SCA).  SCA is caused by a point mutation at position 6 in the β-globin chain of haemoglobin. This mutation results in glutamic acid being replaced with valine. This results in the aggregation of haemoglobin, which can result in the blockage of small blood vessels such as capillaries.  Our detector plasmids could be tailored to detect both the normal and SCA alleles, allowing for the determination of an individual’s genotype for SCA.</P></DIV><H2>Accurate DNA quantification</H2><DIV class="newspaper2"><P>Our system is similar in several ways to digital PCR, which is used to determine the amount of DNA that was in an initial sample (Hall Sedlak and Jerome, 2014).  In our system bacterial cells are the equivalent of the compartments into with a reaction is divided in digital PCR.  Since bacteria usually only take up one plasmid upon transformation SeeDNA could be used to measure how many copies of a target DNA sequence are in a sample.  Accurate DNA quantification is essential in several diagnostic tests.  One example is the detection of copy number variations (CNVs), where a variation in the number of times a particular gene sequence occurs, has to be detected. With further tuning of our detector, a colony / fluorescent cell count and DNA target input could be related to a certain degree of accuracy. Further testing will need to be carried out to evaluate how sensitive the detector is to varying concentrations of the target.</P></DIV><H1>Potential Markets</H1><DIV class="newspaper1"><P>Molecular diagnostics is a growing area of diagnostics and analysing genetic material has now become the norm for disease diagnosis and treatment. Genetic abnormalities, cancer diagnosis and detection of viral / bacterial infections all rely on these new methods of analysis as they are accurate, highly reliable and reveal more information than traditional tests.</P><P>Many molecular diagnostic methods exist, however these methods which are mostly PCR-based do have limitations as outlined below: </P><P>Limitations of current DNA Diagnostic methods</P><UL><LI>Requires highly sophisticated, expensive equipment (real time or digital  PCR machines)</LI><LI>Requires expertise to use</LI><LI>Large running costs (reagents, chemicals)</LI><LI>Complicated results analysis</LI><LI>Problems of contamination of samples with previous PCR products in the lab environment. </LI></UL><P><I>SeeDNA</I> would solve these issues... and more!</P><UL><LI>Versatility - detector plasmid can be easily changed to detect DNA sequences from any virus/bacteria.</LI><LI>No expensive equipment</LI><LI>Use of cheap reagents/chemicals (bacterial media)</LI><LI>Straightforward, easy to use</LI><LI>Cheap/affordable</LI><LI>Rapid and efficient results</LI><LI>Requires little training to use</LI></UL><P>In principle, the presence and quantification of a target in a sample could be carried out, just like quantitative or digital PCR, but with <I>SeeDNA</I>, could be done with just competent cells and a handful of cheap, commonly used reagents. This avoids costly machines and expensive chemicals. In addition, no special training is required to carry out this test; a result could be obtained with just a few simple steps. This means that <I>SeeDNA</I> could be used in areas where molecular diagnostics has not reached.</P><P>Where could we envision applications for <I>SeeDNA</I>? :</P><UL><LI><B>On the farm</B> – <I>SeeDNA</I> could be used to carry out testing of animals for infection or disease. Farmers could carry out onsite diagnostic test on herd(s) for disease, allowing for rapid treatment or isolation of infected animals to prevent spread of infection.  This could save money on vet costs as the farm owner themselves could carry out the test. This could be a solution also for rurally located farms where frequent testing cannot be carried out otherwise.</LI><LI><B>At the GP office</B> – <I>SeeDNA</I> could reduce time needed to carry out molecular tests by bringing the test right to the doctor’s office. Patients could be saved worry-time by getting an instant screening result from the GP at the time of sample collection.</LI><LI><B>In under-resourced labs in developing countries</B> – Medical laboratories, such as those in provisional hospitals run by aid workers, may lack the specialist equipment of bigger hospitals. With a low-cost, simple test such as <I>SeeDNA</I>, patients of these hospitals could still receive first-rate medical diagnostics.  By allowing for detection of pathogenic viruses/bacteria in poorer countries potentially global public health catastrophes such as the recent Ebola outbreak could be prevented.</LI></UL></DIV><H1>Commercialization</H1><DIV class="newspaper2"><P>The model of our DNA detector is currently in the design phase.  The first DNA sequence to be detected will belong to HPV (Human Papilloma Virus) 16, which causes the majority of cervical cancer cases worldwide. Screening for HPV infection, although commonplace in Ireland, is not often carried out in under-resourced countries where the infection is most prevalent. The <I>SeeDNA</I> HPV-16 detector could be a solution to these populations as it is simple, cheap and requires no specialist equipment.  The most exciting feature of <I>SeeDNA</I> is its ability to be customised. Our DNA detector could easily be modified to detect virtually any DNA sequence, making its applications in diagnosis endless.</P><P>For details of our plans to commercialize <I>SeeDNA</I> please see our Business Plan.</P></DIV><H1>Reference</H1><DIV class="newspaper1"><UL><LI>Hall Sedlak, R. &amp; Jerome, K. R. 2014. The potential advantages of digital PCR for clinical virology diagnostics. <I>Expert Rev Mol Diagn</I>, 14, 501-7.</LI></UL></DIV><H1>Appendix</H1><DIV class="newspaper1">
                    See <A href="https://2014.igem.org/Team:UCC_Ireland/Notebook_SeeDNA">Notebook</A> sections for lab book and protocols.
                </DIV></DIV></SECTION></DIV><SECTION id="bottom" class="footer"><UL class="footer"><LI class="footer"><A href="#fbsection1" class="cbp-fbcurrent">Up to Top</A></LI><LI class="footer">UCC iGEM 2014</LI><LI class="footer">uccigem2014@gmail.com</LI></UL></SECTION><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UCC_Ireland/Projects_SeeDNA.html">http://2014.igem.org/Team:UCC_Ireland/Projects_SeeDNA.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCC_Ireland/Projects_SeeDNA.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCC_Ireland/Projects_SeeDNA.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCC_Ireland/Projects_SeeDNA.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCC_Ireland/Projects_SeeDNA.html&amp;oldid=316502" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>