"
Page
Discussion
View source
History
teams
Log in
 
Team:UCC Ireland/Notebook SeeDNA
From 2014.igem.org
≡
|
UCC iGEM 2014
Home
Projects
SeaDNA
SeeDNA
Team
Entrepreneurship
Overview
SeaDNA
SeeDNA
Practice and Outreach
Notebook
SeaDNA
SeeDNA
Safety Form
Our Lab
SeeDNA - Protocols
Click on the link to download the SeeDNA protocol .
SeeDNA - Logbook
June
Week 1: 02-June-2014 to 06-June-2014
The idea of creating a molecular diagnostic using the principle of genetic recombination came about in March and on entering the lab in June, a number of ideas as to how the diagnostic might work had emerged. These were discussed and researched for the first few weeks to decide the most efficient way to approach the project considering time constraints and the cost of our work.
Week 2: 09-June-2014 to 13-June-2014
We received lab training and learnt how to perform basic molecular biology skills and practicing protocols outlined that are included in our lab design. E.g. Gel electrophoresis, running restriction digests, SDS-PAGE and DNA extraction.
Week 3: 16-June-2014 to 20-June-2014
We continued the lab training, learnt how to analyse data and present that data.
Week 4: 23-June-2014 to 27-June-2014
Our focus this week was on choosing a suitable target for our molecular diagnostic. We considered situations where a low cost DNA screening test could be beneficial. One example which could have widespread use was the detection of HPV 16, a dsDNA virus and the leading cause of cervical cancer worldwide.
Our next obstacle was choosing a region on this genome to detect. This involved analysis and cross-referencing sequences on Benchling.
When our target sequence was chosen, this oligo was synthesised and delivered.
July
Weeks 5 - 10 : 30-June-2014 to 1-August-2014
A design for our detector plasmid was chosen as it plausibly could be constructed and tested within three months and appeared theoretically well-founded. The method by which the detector was to be constructed however underwent many changes.
In July, Protocol 1 was attempted.* This involved the cloning of two variations of our target insert into our vector pSBC13 and creating a hybrid plasmid through the process of heating (to separate the strands) and cooling (to re-anneal). A number of variations of this protocol were attempted. The resulting plasmids were ran on agarose gels to assess size and transformed with the target sequence to assess success in detection. Results were generally inconclusive and no clear indication that the intended detector had been constructed was achieved.
September
Week 14: 1-September-2014 to 5-September-2014
Protocol 3 was began.* This protocol did not feature the creation of a hybrid plasmid and did not require plasmid strands to separate to create our detector.
Our detector insert had to be synthesised and cloned into our vector pSBC13. The detector was then activated by a novel process.*
Week 15: 8-September-2014 to 12-September-2014
The activated detector was incubated with the target sequence and allowed to undergo various conditions of incubation. This was then transformed and compared to a control transformation, which lacked the target sequence.
Results were positive. The experiment was repeated to confirm results.
Week 16: 15-September-2014 to 19-September-2014
A plasmid prep was carried out on a number of colonies from the initial experiment to confirm that detection of the sequence had occurred.
Once this was confirmed, the process could be optimised. Work began in repeating the detection process varying conditions such as temperature and salt concentration.
Week 17: 22-September-2014 to 27-September-2014
Optimisation work continued.
A double stranded target was constructed. The detector was assessed with the double stranded target and transformed as done with the single stranded target to compare results.
A new target, on the SRY gene on the Y chromosome, was also synthesised to test with the detector in the same way as with the HPV sequence.
Week 18: 29-September-2014 to 3-October-2014
Optimisation work continued.
The double stranded target was cloned into a new vector, pBluescript, which contained a number of HaeIII restriction sites. This plasmid was then digested with HaeIII and assessed on an agarose gel for the presence of the target fragment.
The digested fragments (containing the target sequence among other fragments) were incubated with the detector and transformed to assess detector competency when background DNA was present
Also, the detector insert was cloned into plasmid pSBC13 containing a GFP promoter. On successful detection and transformation of the target, cells were assessed for expression of GFP using a fluorescent microscope.
Week 19: 6-October-2014 to 10-October-2014
Further work was done to assess the timeframe at which GFP could be detected in cells containing the target sequence.
Further work was done to assess the timeframe at which GFP could be detected in cells containing the target sequence.
October
Week 14: 1-October-2014 to 5-October-2014
We prepared the plasmids required for Biobrick assembly for iGEM by digesting the subunits with the enzyme SpeI and Eco RI, and digesting the given plasmid with the same enzymes. These digestions were then followed by a ligation reaction and transformation to bulk up the plasmid and screen for successful colonies.
Up to Top
UCC iGEM 2014
uccigem2014@gmail.com
Retrieved from " http://2014.igem.org/Team:UCC_Ireland/Notebook_SeeDNA "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
