Team:Manchester/Notebook
PHOSPHOSTORE
Project
Overview
Description
Results
Demonstration
Medal Criteria
Achievement
‚òÜJUDGING FORM‚òÜ
Modelling
Overview
D. o. Experiment
Cont. Culture
PHO Operon
Parts
Overview
Basic Parts
Composite Parts
Improved Parts
Part Collection
Notebook
Diary
Protocols
Lab book
InterLab
Measurement
Human Practices
Human Practices
Integrated Practices
Public Engagement
Water Industry
Intellectual Property
Entrepreneurship
Team
Team
Collaborations
Attributions
MENU
HOME
TEAM
Team
Collaborations
Attributions
PROJECT
Overview
Description
Results
Demonstration
Medal Criteria
Achievement
NOTEBOOK
Protocols
Diary
Lab Book
InterLab
Measurement
PARTS
Parts
Basic Parts
Composite Parts
Improved Parts
Part Collection
HUMAN PRACTICES
Silver HP
Water Industry
Integrated and Gold
Intellectual Property
Public Engagement
Entrepreneurship
MODELLING
Overview
D. O. Experiment
Continuous Culture
PHO Operon
JUDGING FORM
üçÄ Lab Plan üçÄ
Polyphosphate Kinase (PPK) Fusion Protein
Basic aim: Assess impact localisation tag has on kinetics of PPKs
Introduction
Our project initially aimed to screen a selection of bacterial polyphosphate kinases to select one with the greatest forward rate of reaction (polyphosphate formation). Due to time constraints, we have decided to focus on a single polyphosphate kinase (PPK) that we identified in the Brenda database. We chose a PPK from Corynebacterium glutamicum, a bacteria used for industrial production of amino acids. The bacteria was discovered to accumulate high concentrations of polyphosphate (600mM, 37% cell volume), which was largely attributed to its class II Polyphosphate kinase. This PPK (encoded by NCgl2620) has a Kcat value of 74 for ATP ‚ûù Poly P under the experimental conditions used. It is also fully soluble, unlike the E. coli PPK which was encapsulated within a Pdu microcompartment by Warren et al. earlier this year. We hypothesise that solubility may enhance efficiency of encapsulation.
At this stage we will only be trialing one localisation tag (PduD1-20) and hope to characterise it using mCherry and fluorescence microscopy as an indicator for incorporation into microcompartment.
We were planning on ordering these sequences as gBlocks and simultaneously cloning them into both the submission vector and the expression vector pNIC-bsa28 in frame with a T7 promoter. Then we can over-express these fusion proteins in BL21 (DE3) cells and purify them using a nickel affinity resin column.
After expression and purification, the enzymes constructs will be characterised via Michaelis-Menten kinetics using the Promega ADP-Glo‚Ñ¢ Kinase Assay. Technical support from Promega confirmed that the kit should work on a polyphosphate kinase. Promega have kindly provided us with 400 assays to use free of charge.
Kinetics data generated from these assays will allow us to assess any impact the localisation tag or mCherry tag has on enzyme activity. Results from this will aid how we build the construct to target the enzyme into the microcompartment at a later stage.
Experimental Plan
a. Design and order the following constructs with EcoRI, XbaI prefix, SpeI, PstI and XhoI suffix (XhoI will be used to go into pNIC28):
- (N)tag_mCherry_PPK_His6
- (N)tag_PPK_mCherry_His6
- PPK_His6
b. Resuspend powder gBlocks (1000 ng) in 20 uL EB buffer (miniprep kit)
c. Clone PPK gBlocks into pNIC28 (PPKs in pNIC28) and pSB1C3 (PPKs in pSB1C3):
1. Restriction digest pSB1C3 with EcoRI and PstI,  pNIC28 with XbaI and XhoI and the PPK constructs (some for pSB1C3, some for pNIC28)
2. Perform ligation reaction 3 : 1 (~50ng vector)
3. Transform into DH5a cells
4. Overnight cultures
5. Miniprep plasmid
6. Restriction digest some of the ligated plasmid and run agarose gel to confirm insert (Table 1)
Table 1: Expected results from restriction digest of all PPK constructs
Construct
Digest with
Expected fragment size (bp)
pSB1C3_PPK_His
EcoRI + PstI
2029 + 1121
pSB1C3_PduD_mCherry_PPK_His
PstI + HindIII
2596 + 1337
pSB1C3_PduD_PPK_mCherry_His
EcoRI + HindIII
2439 + 1494
pNIC28_PPK_His
XbaI + XhoI
5192 + 1111
pNIC28_PduD_mCherry_PPK_His
XbaI + XhoI
5192 + 1894
pNIC28_PduD_PPK_mCherry_His
XbaI + XhoI
5192 + 1894
7. Send all constructs for sequencing
8. Submit pSB1C3 constructs as new BioBricks
9. Transform pNIC28 constructs into BL21(DE3) cells for expression
10. Colony PCR and agarose gel - to confirm ligated plasmid is present
11. Grow each of the confirmed transformants in TB
12. Purify protein using nickel resin
13. Measure specific activity of enzyme using ADP-Glo‚Ñ¢ Kinase Assay kit
14. Determine kinetics with respect to PO4-
15. Upload characterization data for submitted parts
Eut Microcompartment
Basic aim: Form a Eut bacterial microcompartment (BMC)
Hopefully: Optimise BMC formation
Introduction
We initially considered using a Pdu (1,2-propanediol utilisation) microcompartment to package our enzyme, as it is well characterised in literature. However, after consulting our supervisors and PIs, we have decided to use Eut (Ethanolamine utilisation) microcompartments for our experiments. This contrasts the paper which inspired our project (Liang M, et al., 2017), but follows on from the 2013 HongKong iGEM teams work (link wiki). The Eut microcompartment functions as a site where ethanolamine is cleaved into ammonia and acetaldehyde in bacteria such as Citrobacter and Salmonella. This is achieved through the action of ethanolamine-ammonia lyase, which is localised inside the microcompartment.
Localisation of proteins like the aforementioned enzyme requires the presence of a Eut signal sequence/ tags - a short sequence of 18-20 amino acids that is located at the N-terminal of the protein of interest. From literature and through consultation, we have found that EutC1-19, EutD1-20, and EutE1-20  seem to be the most well characterised of these sequences. Additionally, due to the relatedness of Eut and Pdu, Pdu signal sequences PduD1-20 and PduP1-18 may also be able to localize proteins inside Eut microcompartments.
In order to accurately control and optimise the formation of BMCs, we want to build the following circuits to express the Eut proteins S, M, N, L, K under the control of inducible promoters. The operon has been split into 3 sections: each part has been designed and is ready to order as gBlocks from IDT. (Table 2).
Table 2: Eut constructs
Part
Details
Length (bp)
lacUV5_EutS
IPTG inducing lacUV5 promoter causing EutS synthesis. N term His tag on EutS.
1983
tetA_EutMN
Tetracyclin inducing TetA promoter causing GFP-EutM and EutN synthesis. N term His tag on GFP-EutM; C term FLAG tag on EutN.
2452
araBAD_EutLK
Arabinose inducing araBAD promoter causing EutL and EutK synthesis. N term His tag on EutL; C term FLAG tag on EutK.
2740
Figure 1: Three Eut operons. (From top to bottom) lacUV5_EutS, tetA_EutMN and araBAD_EutLK
Table 3: Sizes of Eut proteins with and without tags
Eut protein
Length (a.a.)
Size (kDa)
Tags
Size with tags (kDa)
S
111
11.6
NTerm His6
12.70
M
97
9.4
NTerm His6 + GFP
40.12
N
95
10.0
CTerm FLAG
11.20
L
219
22.8
NTerm His6
23.92
K
166
17.9
CTerm FLAG
19.10
Experimental Plan
a. Order 3 Eut parts as gBlocks from IDT
b. Clone each Eut part into pSB1C3:
1. Restriction digest pSB1C3, Lac_EutS, Tet_EutMN, and ara_EutLK with EcoRI and PstI
2. Perform the 3 ligation reactions (+ 2 controls)
3. Transform into DH5a cells (from NEB)
4. Overnight cultures
5. Miniprep plasmid
6. Restriction digest some of the ligated plasmid and run agarose gel to confirm insert (Table 4)
Table 4: Expected results from restriction digest of all Eut constructs
Construct
Digest with
Expected fragment size (bp)
pSB1C3_lac_EutS
XhoI
1700 + 1405 + 892
pSB1C3_tetR_EutMN
NcoI
2655 + 1808
pSB1C3_araBad_EutLK
BamHI
3830 + 925
7. Send all constructs for sequencing (Sanger sequencing - Eurofins Genomics)
8. Submit pSB1C3 constructs as new BioBricks
9. Transform pSB1C3 Eut constructs into DH5a cells for expression
10. Colony PCR and agarose gel - to confirm ligated plasmid is present
11. Overnight cultures of each of the confirmed transformants in TB
12. Induce expression of each construct
13. Check construct expression by western blot
c. Assemble the Eut operon in pSB1C3 using BioBrick cloning sites:
1. Restriction digest pSB1C3_Lac_EutS with SpeI + PstI and pSB1C3_TetR_EutMN with XbaI + PstI
2. Run on agarose gel and gel extract band of pSB1C3_Lac_EutS at ~4000bp, and band of TetR_EutMN at ~2400bp
3. Ligation reaction = to form pSB1C3_Lac_EutS_TetR_EutMN
4. Transform into DH5a cells
5. Overnight culture
6. Miniprep
7. Restriction digest with EcoRI + PstI
8. Run agarose gel to confirm successful insert - looking for bands at 4369bp + 2029bp, and send for sequencing
9. Restriction digest pSB1C3_Lac_EutS_TetR_EutMN with SpeI + PstI and pSB1C3_araBad_EutLK with XbaI + PstI
10. Run on agarose gel and gel extract band of pSB1C3_Lac_EutS_TetR_eutMN at 6383bp, and band of araBad_eutLK at 2708bp
11. Ligation reaction = to form pSB1C3_Lac_EutS_TetR_EutMN_araBad_EutLK
12. Transform into DH5a cells
13. Colony PCR, and run agarose gel
14. Overnight culture
15. Miniprep
16. Restriction digest with NcoI to confirm successful insert - look for bands at 5348bp and 3743bp (Table 5)
17. Send for sequencing (Sanger sequencing - Eurofins Genomics)
18. Submit pSB1C3_Lac_EutS_TetR_EutMN_araBad_EutLK as BioBrick
Table 5: Expected result from restriction digests during Eut operon assembly
Construct
Digest with
Expected fragment size (bp)
pSB1C3_Lac_EutS_TetR_EutMN
EcoRI + PstI
4369 + 2029
pSB1C3_Lac_EutS_TetR_EutMN_araBad_EutLK
NcoI
5348 + 3743
pSB4A5_Lac_EutS_TetR_EutMN_araBad_EutLK
PstI + EcoRI
7062 + 3354
d. Expression of Eut operon:
1. PCR the Eut operon out of pSB1C3 (VR, VF2)
2. Restriction digest Eut operon and pSB4A5 with EcoRI + PstI
3. Ligation reaction = to form pSB4A5_LacEutS_TetEutMN_araEutLK
4. Transform into DH5a cells
5. Colony PCR and agarose gel
6. Overnight culture
7. Miniprep plasmid
8. Restriction digest with EcoRI + PstI (Table 5) and run agarose gel to confirm insert
9. Send for sequencing to ensure no mutation
10. Grow each of the confirmed transformants in TB
11. Induce pSB4A5_Lac_EutS_TetR_EutMN_araBad_EutLK
12. Confirm expression of all Eut proteins via western blot
13. Fluorescence microscopy to visualise and confirm BMC formation - this will be the empty BMC
Localisation Tag Characterisation
Basic aim: Encapsulate mCherry tagged with the signal sequences inside the Eut microcompartment
Introduction
Successful encapsulation would be justified by fluorescence microscopy using colocalisation technique, which involves the observation of the spatial overlap between two different fluorescent labels, i.e. mCherry and GFP, each having a separate emission wavelength. Microcompartments carry GFP while the localisation tags carry mCherry.
Experimental Plan
a. PCR out the RBS_PduD 1-20_mCherry constructs from existing constructs in the lab. The constructs were in a pET-DUET plasmid.
b. Cloning:
1. Cut all tag constructs with EcoRI and PstI to clone into pSB1C3 submission vector with medium strength constitutive promoter (BioBrick: BBa_J23108).
2. Do a restriction digest. Cut the promoter vector (pSB1C3 which contains the promoter BBa_J23108) with SpeI and PstI and cut the PCR product with XbaI and PstI. Use a phosphatase to cleave the 5‚Äô phosphate in the promoter vector.
c. Removal of the illegal Xba1 site from all PduD1-20 constructs ( BBa_K2213006-BBa_K2213008 ).
d. Sequence and submit all PduD 1-20 constructs with different strength anderson promoters as improved parts .
e. Confirm expression - agarose gel electrophoresis.
Selecting constitutive promoters from the registry
We have chosen medium, strong and weak constitutive promoters for expression of the tag-mcherry parts:
- Part:BBa_J23108 - strength = 0.51 - plate 4 - well 4C
- Part:BBa_J23104 - strength = 0.72 - plate 4 - well 17L
- Part:BBa_J23105 - strength = 0.24 - plate 4 - well 17N
The strong and weak promoters (Part:BBa_J23104 and Part:BBa_J23105) are in the backbone BBa_J61002 (not pSB1C3) so we will get these synthesised as primers instead to reduce the amount of cloning we have to do. This will allow us to express promoter at different levels during design of experiment.
For low and high strength promoters:
BBa_J23104 and BBa_J23105 are in the plasmid BBa_J61002 and we want them in the pSB1C3:
1. Cut at EcoRI and SpeI to cut out promoter from plasmid BBa_J61002.
2. PCR the promoter extracted.
3. Cut pSB1c3 at EcoRI and SpeI and ligate the promoter in.
4. Constructs ready to be used in Eut DoE experiments.
Eut Design of Experiment (DoE)
Basic aim: Optimise BMC formation
Modelling: Design of Experiments is used to find the optimal parameters using minimal tests by simultaneously changing multiple factors and making justified changes as the experiment is repeated. As the experiment is repeated, fundamental factors are identified and the next round of experiments hone in on these. By structuring our experiments and data in this way, we will be able to gain a deeper insight into to interactions and factors influencing our data. A general linear model can then be fitted (using JMP).
Response: Function of formation of microcompartments
Table 7: DoE Input Factors
Name
Type of factor (categorical or continuous)
Difficulty (e, m, h, vh)
Range
Variables
Temperature
con
15 - 37 ¬∞C
pH
con
4.0 (E.coli strain M23)
Optimum: 6-7 (O157:H7)
Range: 4.4-9.0(O157:H7)
E.coli BL21: 7.5-8.5 (Wang et.al, 2014)
Copy number of plasmid
cat
Tags x 5 + empty (6)
cat
e
n/a
Cell type
cat
BL21, DH5a
Strength of Anderson promoters
con
0.24, 0.51, 0.72
Time of BMC induction
con
Time of BMC harvesting
con
RPM in shaker (aeration)
con
Carbon source
cat
Iron conc
con
Phosphate conc
con
Magnesium conc
con
Carbon conc
con
Calcium conc
con
Organic solvent conc
con
Cell density
con
Secondary metabolite conc
con
ammonia/ammonium conc
con
Eut plasmid
cat
Maybe: plyss, pacyc duet, pET duet, pSB4A5
Concentration of inducers used for the  promoters in the Eut operon
con
IPTG: 0-100 uM
Tetracycline: 0-100 nM
Arabinose: 0-20 mM
(Lee et.al, 2011)
Table 8: DoE Responses
Name
Response type / range
Other information
Final OD (optical density)
Objects ranked by circularity automatically using imageJ
Reference
Software automated - mCherry/GFP ratio analysis to determine co-localization of tag in BMC
Reference
Similarity to template BMC ranked by software
Reference
‚çü Actual Lab Work Record ‚çü
Toggle mini / wide pdf size
If the browser does not support embedded PDFs: Please download the PDF to view it.
PPK
Eut
Tag
Connect with us:
Email: manchesterigem17@gmail.com
