<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Manchester_Results1 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Manchester/Results1</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="verytop"><NAV class="navbar navbar-default navbar-fixed-top visible-md visible-lg gradient bmc-bg" style=" padding-top: 20px; padding-left: 8vw; padding-right: 8vw"><DIV class="container-fluid"><DIV class="logo-center logoheader"><A class="navbar-logo" href="https://2017.igem.org/Team:Manchester"><H2 style="font-size: 1em!important; float: left; letter-spacing: 1px; padding-top: 10px!important; margin-top: 20px!important; width: 0px!important; overflow: visible!important">PHOSPHOSTORE</H2></A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin-top: 10px;"><LI class="dropdown dropdown10"><A class="dropdown-toggle active dropdowntextcenter" href="https://2017.igem.org/Team:Manchester/Description">Project</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Description1">Description</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Demonstrate">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Medals">Medal Criteria</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Achievement">Achievement</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Manchester">☆JUDGING FORM☆</A></LI></UL><LI class="dropdown dropdown10"><A class="dropdown-toggle active dropdowntextcenter" href="https://2017.igem.org/Team:Manchester/Model">Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Model/DoE">D. o. Experiment</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Model/Continuous_Culture">Cont. Culture</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Model/PSO">PHO Operon</A></LI><LI class="dropdown dropdown10"><A class="dropdown-toggle active dropdowntextcenter" href="https://2017.igem.org/Team:Manchester/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Improve">Improved Parts</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Part_Collection">Part Collection</A></LI><LI class="dropdown dropdown10"><A class="dropdown-toggle active dropdowntextcenter" href="https://2017.igem.org/Team:Manchester/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Diary">Diary</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Experiments">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Notebook">Lab book</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Measurement">Measurement</A></LI><LI class="dropdown dropdown13"><A class="dropdown-toggle active dropdowntextcenter" href="https://2017.igem.org/Team:Manchester/HP/Silver">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/HP/Silver">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/HP/Water_Industry">Water Industry</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/HP/Intellectual_Property">Intellectual Property</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Entrepreneurship">Entrepreneurship</A></LI><LI class="dropdown dropdown7"><A class="dropdown-toggle active dropdowntextcenter" href="https://2017.igem.org/Team:Manchester/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV><DIV class="igem_2017_menu_wrapper visible-sm visible-xs"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:Manchester"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:Manchester/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Manchester/Description"><DIV class="submenu_button" id="Description_page">
					Overview
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Description1"><DIV class="submenu_button" id="Description1_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstration
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Medals"><DIV class="submenu_button" id="Medals_page">
					Medal Criteria
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Achievement"><DIV class="submenu_button" id="Achievement_page">
					Achievement
				</DIV></A></DIV><DIV class="menu_button"> NOTEBOOK
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Manchester/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Protocols
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Diary"><DIV class="submenu_button" id="Diary_page">
					Diary
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Lab Book
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/InterLab"><DIV class="submenu_button" id="InterLab_page">
					InterLab
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Manchester/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Composite_Part"><DIV class="submenu_button" id="Composite_Part_page">
					Composite Parts
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Improve"><DIV class="submenu_button" id="Improve_page">
					Improved Parts
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Manchester/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Water Industry
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/HP/Water_Industry"><DIV class="submenu_button" id="Water_Industry_page">
					Integrated and Gold
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/HP/Intellectual_Property"><DIV class="submenu_button" id="Intellectual_Property_page">
					Intellectual Property
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement 
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Entrepreneurship"><DIV class="submenu_button" id="Entrepreneurship_page">
					Entrepreneurship
				</DIV></A></DIV><DIV class="menu_button"> MODELLING
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Manchester/Model"><DIV class="submenu_button" id="Model_page">
					Overview
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Model/DoE"><DIV class="submenu_button" id="DOE_page">
					D. O. Experiment
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Model/Continuous_Culture"><DIV class="submenu_button" id="CC_page">
					Continuous Culture
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Model/PSO"><DIV class="submenu_button" id="PSO_page">
					PHO Operon
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=Manchester"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><DIV class="col-md-12 sectionfirst" style="background-color: #fbf9f8!important"><H2 class="border" style="margin-top: 5vh; text-align: center">Results (1/4)</H2></DIV><DIV style="padding: 0px 5vw; background-color: #fbf9f8!important"><DIV class="clear" style="padding: 0px 12vw; border: 3px solid transparent; background-color: #fbf9f8!important"><CENTER><H4>Polyphosphate Kinase (PPK)</H4></CENTER><P><FONT color="#111"><B>Background</B></FONT></P><P><FONT color="#111">Accumulation of polyphosphate by polyphosphate kinases (PPKs) is an important mechanism of phosphate uptake for enhanced biological phosphate removal (EBPR) in water treatment plants (Mino, van Loosdrecht and Heijnen, 1998). Degradation of polyphosphate in cells limits its accumulation (Akiyama, Crooke and Kornberg, 1992).</FONT></P><P>Encapsulation of PPKs within bacterial microcompartments has been shown to be a viable approach to enhance accumulation of phosphate in <I>Escherichia coli</I> (Liang et al., 2017). This was achieved by co-expression of a recombinant <I>Citrobacter freundii</I> 1,2-propanediol utilisation (Pdu) microcompartment operon with <I>E. coli</I> PPK that had been fused with an N-terminal sequence which has been shown to direct heterologously expressed proteins towards the interior of the microcompartment.</P><P>We wanted to build on this work by identifying a polyphosphate kinase with a higher turnover rate than the E. coli PPK, and directing it towards the expressed bacterial microcompartment. We used the Braunschweig Enzyme Database (BRENDA) to compare specific activities of polyphosphate kinases and identified the class II polyphosphate kinase from <I>Corynebacterium glutamicum</I> (cgPPK) as a suitable candidate, having a 30 fold greater turnover rate than that of the <I>E. coli</I> PPK (<B>Figure 1.</B>; Lindner et al., 2007; Kornberg and Simms, 1956).</P><CENTER><P><FONT color="#111"><CENTER><B>Figure 1.</B> Comparison of specific activity between the <I>E. coli</I> class I PPK and the <I> Corynebacterium glutamicum</I> class II PPK 
</CENTER></FONT></P><P><FONT color="#111">
We used the SwissModel (Schwede, 2003) server to generate a homology model of the structure of the cgPPK enzyme based on its amino acid sequence. SwissModel identified an existing crystal structure of class II polyphosphate kinase from <I> Sinorhizobium meliloti</I> (see <B>Figure 2</B>).
</FONT></P><CENTER><DIV class="col-sm-8" style="padding: 1% 5%"><P style="font-size: 15px!important"><B>Figure 2: cgPPK2 model</B> Homology model of cgPPK generated using SwissProt (AS Rose et al., 2016; 2015)
</P><P>We believed that a fused PduD(1-20) sequence would be sufficient to direct recombinant cgPPK towards the interior of the Ethanolamine utilisation (Eut) bacterial microcompartment, owing to a hydrophobic motif that it shares with the N-terminal sequences of EutE and EutC, which are targeted to the Eut microcompartment in <I>Salmonella enterica</I> (Jakobson et al., 2015).</P><P><B>PPK Constructs</B></P><P>We designed three constructs that would allow us to study whether the PduD(1-20) sequence could localise cgPPK to the microcompartment (see <B>Figure 3</B>). Two of the constructs (<B>PduD(1-20)_mCherry_cgPPK</B> and <B>PduD(1-20)_cgPPK_mCherry</B>) contained an mCherry fusion that would allow us to determine subcellular localisation of the enzyme via microscopy. All three constructs contained a C-terminal hexa-histidine sequence that would allow purification with immobilised nickel-affinity media.
</P><P><FONT color="#111"><CENTER><B>Figure 3.</B> Architecture of the constructs based on the class II polyphosphate kinase from <I>Corynebacterium glutamicum</I> (cgPPK): PduD(1-20)_mCherry_cgPPK (<A href="http://parts.igem.org/Part:BBa_K2213005"><B>BBa_K2213005</B></A>); PduD(1-20)_cgPPK_mCherry (<A href="http://parts.igem.org/Part:BBa_K2213004"><B>BBa_K2213004</B></A>); cgPPK (<A href="http://parts.igem.org/Part:BBa_K2213003"><B>BBa_K2213003</B></A>).</CENTER></FONT></P><P>The constructs shown above were cloned into the pNIC28-bsa4 vector downstream of a T7 promoter for overexpression. The vectors were transformed into BL21 (DE3) <I>E. coli</I> cells and cultured under similar conditions to those used by Lindner <I>et al.</I> (2007) to express recombinant cgPPK: The liquid cultures were grown at 37°C and 180 rpm until the OD600 reached 0.5 – 0.6, at which point Isopropyl β-D-1-thiogalactopyranoside (IPTG) was added to a final concentration of 0.5 mM. The cultures were grown for a further 4 hours at 37°C and 180 rpm before harvest by centrifugation at 5,000 rpm for 10 minutes. The cell pellets were frozen at -20°C for later purification.</P><P><B>Purification of Constructs</B></P><P><FONT color="#111"><CENTER><B>Figure 4.</B> SDS-PAGE analysis of fractions from the Ni-NTA agarose purification of the cgPPK constructs. Ladder: Precision Plus Protein™ (Biorad). Black arrows indicate where a band with a molecular mass corresponding to that of the construct should be. 
</CENTER></FONT></P><P><FONT color="#111">
All three constructs were purified using Ni-NTA agarose (QIAgen) according to our <A href="https://2017.igem.org/Team:Manchester/Experiments#NiNTA" target="_blank"><B>Nickel-affinity purification protocol</B></A>.  
The SDS-PAGE analysis showed that the elution fraction of the PduD(1-20)_cgPPK_mCherry (<A href="http://parts.igem.org/Part:BBa_K2213004"><B>BBa_K2213004</B></A>) construct gave a band of around 30 kDa rather than the expected 66 kDa. The bright pink colour of the concentrated elution fraction suggested that the mCherry unit was intact. This suggested that the PduD(1-20)_cgPPK region of the construct was cleaved before or during the purification. It wasn’t clear from our results why this construct was unstable, but we reasoned that it was too unstable for our use, so no further attempts to express and purify it were made.

The cgPPK construct was purified to near-homogeneity (see <B>Figure 4</B>). Absorbance of the purified protein was measured using a NanoDrop 1000 UV/Vis spectrometer (Thermofisher) and using the extinction coefficient 65890 (generated by inputting the primary sequence into the ExPASy ProtParam) the final concentration of cgPPK was calculated to be 0.89 mg/ml. The presence of kinase activity of the construct was confirmed by data generated using the Promega ADP Glo™ kinase assay kit ( see <B>Figure 5</B>). 
</FONT></P><P><B>Determining the Presence of PPK Activity</B></P><CENTER><P><FONT color="#111"><CENTER><B>Figure 5.</B> Qualitative kinase activity assessment of the cgPPK (<A href="http://parts.igem.org/Part:BBa_K2213003"><B>BBa_K2213003</B></A>) and PduD(1-20)_mCherry_cgPPK (<A href="http://parts.igem.org/Part:BBa_K2213005"><B>BBa_K2213005</B></A>) constructs, assessed using the Promega ADP Glo (TM) Kinase assay
</CENTER></FONT></P><P><FONT color="#111">
The PduD(1-20)_mCherry_cgPPK (<A href="http://parts.igem.org/Part:BBa_K2213005"><B>BBa_K2213005</B></A>) co-purified with a ~70 kDa protein in roughly equimolar quantities. The co-purified protein may be the Gro EL heat shock protein (Thain et al., 1996), which may be the result of mis-folding. The PduD(1-20)_mCherry_cgPPK  construct retained a bright pink colour and kinase activity (see <B>Figure 5</B>), which suggests that both the mCherry and the cgPPK domains folded correctly. It’s possible that the putative heat shock protein bound to the N-terminal PduD(1-20) targeting peptide as a result of its hydrophobicity. 

Fluoresence microscopy imaged of 4',6-diamidino-2-phenylindole (DAPI)-stained cells expressing all five Eut microcompartment structural genes and PduD(1-20)_mCherry_cgPPK suggested that the construct is localised within the bacterial microcompartment and synthesize polyphosphate. This can be seen in our proof of concept page, <A href="https://2017.igem.org/Team:Manchester/Demonstrate" target="_blank">here</A>.
<B>Expression Optimization - Round 1</B>
PPK optimization is necessary to reduce the <A href="https://2017.igem.org/Team:Manchester/Model/Continuous_Culture">cost of the system</A>. Expression of the pNIC28-bsa4:PduD(1-20)_mCherry_cgPPK construct was optimised using a <A href="https://2017.igem.org/Team:Manchester/Model/DoE" target="_blank">Design of Experiments</A> approach. mCherry fluorescence in raw culture would be measured by a plate reader as a proxy for protein yield. JMP software from SAS was used to design an initial round of 20 experiments that would screen the following factors for their effect on protein yield: <UL>
OD600 at induction (0.2 - 0.8)
[IPTG] at induction (0.1 – 1 mM)
Growth temperature after induction (20 – 37°C)
Growth period after induction (4 – 24 hours)</UL></FONT></P><P><FONT color="#111">
The initial design took into account potential interactions between the factors, but did not include mid-points between the extremes of each factor.
All cultures were grown from a clonal glycerol stock in Luria-Bertani (LB) broth supplemented with 100 µg/ml kanamycin. The initial growth phase prior to induction was always carried out in a single flask at 37°C and 180 rpm. At the point of induction, the flask was split into smaller shake flasks which met the specified growing conditions and the resulting mCherry fluorescence was quantified (see <B>Figure 6</B>).
</FONT></P><P><FONT color="#111"><CENTER><B>Figure 6.</B> Relative mCherry fluorescence from raw liquid culture expressing the PduD(1-20)_mCherry_cgPPK (<A href="http://parts.igem.org/Part:BBa_K2213005"><B>BBa_K2213005</B></A>) construct under different conditions.
</CENTER></FONT></P><P>Using JMP, we fit a linear model to our data using standard least squares regression analysis (R²=0.99, <B>Figure 7</B>). The effects summary outlined the significant individual factors and the two-way interactions between the factors that affected yield (see <B>Figure 8<B>). </B></B></P><CENTER><P><CENTER><B>Figure 7.</B> Predictive vs. actual plot of the linear model generated from the experimental data  (R²=0.99)</CENTER></P><P><FONT color="#111"><CENTER><B>Figure 8.</B> Effect summary of input factors and their interactions on yield of PduD(1-20)_mCherry_cgPPK (<A href="http://parts.igem.org/Part:BBa_K2213005"><B>BBa_K2213005</B></A>)</CENTER></FONT></P><P><CENTER><B>Figure 9.</B> The interaction profile of the input factors, maximised for yield. OD600 at induction consistently correlated with higher yields. The profiler also suggests that optimal conditions for expression of PduD(1-20)_mCherry_cgPPK (<A href="http://parts.igem.org/Part:BBa_K2213005"><B>BBa_K2213005</B></A>) lie close to 20°C and a post-induction growth period of 24 hours or more.</CENTER></P></CENTER><P>The model suggested that OD600 at the time of induction had consistent positive correlation with yield (see <B>Figure 9</B>), so we decided that for future experiments we would fix this value to 0.8, which allowed us to look at other factors with greater resolution. The model also suggested that the optimal conditions for PduD(1-20)_mCherry_cgPPK (<A href="http://parts.igem.org/Part:BBa_K2213005"><B>BBa_K2213005</B></A>) lay outside of our initial parameters.</P><P><B>Expression Optimization - Round 2</B></P><P>We designed a second round of 20 experiments that would explore the effects of our input factors slightly closer to the predicted optimal values. The second round of experiments resulted in yields that were beyond the detection capability of the plate reader using the gain settings that were used for the first round (see <B>Figure 10</B>), so the gain had to be lowered. This meant that the results from the two rounds were no longer comparable and could not be modelled together. However, we could be sure that the yields of the second round exceeded those of the first (see <B>Figure 11</B>).</P><CENTER><P><CENTER><B>Figure 10.</B> mCherry fluorescence measurements from the 24 hour time points from round 2 of optimisation (Pink) plotted alongside the mCherry fluorescence measurements from the round one experiments, which used the same gain setting. The readings from the second round have saturated the detector, so we had no choice but to change the gain settings on the plate reader. This meant that we could no longer directly compare data from round 1 and round 2.</CENTER></P><P><CENTER><B>Figure 11.</B> mCherry fluorescence of raw culture from round 2 of expression optimisation. Fluorescence intensity for all round 2 experiments are greater than those of round 1.</CENTER></P><P>Once again, we fit a model to the data (R² =0.80, <B>Figure 12</B>) which predicted that post-induction growth period and post-induction incubation temperature had the most significant effects on mCherry fluorescence (P &lt; 0.05 in both cases); variation in IPTG concentration within the experimental range (1-10 mM) was not found to have a particularly significant effect on mCherry fluorescence (P = 0.15). <B>Figure 13</B> shows that post-induction growth period and post-induction incubation temperature are the most significant factors affecting PduD(1-20)_mCherry_cgPPK (<A href="http://parts.igem.org/Part:BBa_K2213005"><B>BBa_K2213005</B></A>) yield. <B>Figure 14</B> is a surface plot that shows how PduD(1-20)_mCherry_cgPPK (<A href="http://parts.igem.org/Part:BBa_K2213005"><B>BBa_K2213005</B></A>) yield responds to post-induction growth period and post-induction incubation temperature, and indicates a maximum yield value at 48 hours of incubation at a growth temperature between 20-24°C.</P><CENTER><P><FONT color="#111"><CENTER><B>Figure 12.</B> Actual vs. predictive plot generated using the model fitted to the second round of expression optimisation data </CENTER></FONT></P><P><FONT color="#111"><CENTER><B>Figure 13.</B> Effects summary of the second round of expression optimisation. Post induction temperature and the harvest time had the most significant effect on yield within the parameters tested
</CENTER></FONT></P><P><FONT color="#111"><CENTER><B>Figure 14.</B> Surface plot mapping the effect of post-induction temperature and post-induction incubation time on yield of PduD(1-20)_mCherry_cgPPK (<A href="http://parts.igem.org/Part:BBa_K2213005"><B>BBa_K2213005</B></A>), measured by relative fluorescence units.
</CENTER></FONT></P><P>The model predicted that within the limits of the tested parameters, optimal expression of PduD(1-20)_mCherry_cgPPK (<A href="http://parts.igem.org/Part:BBa_K2213005"><B>BBa_K2213005</B></A>) could be achieved by inducing the culture to a final concentration of 1mM IPTG when the OD600 reached 0.8, followed by a 48 hour growth period at 24°C at 180 rpm. 
</P><P><B>Post-Optimization Purification of PPK Construct</B></P><P>A culture of BL21 (DE3) cells harbouring pNIC28-bsa4:PduD(1-20)_mCherry_cgPPK was grown under the predicted optimal conditions for mCherry yield per volume of culture. On attempt to purify the recombinant protein according to our <A href="https://2017.igem.org/Team:Manchester/Experiments#NiNTA" target="_blank"><B>Nickel-affinity purification protocol</B></A>, we found that after repeated attempts to disrupt the cells by sonication, the bright pink recombinant protein was insoluble and could not be separated from the cell debris under non-denaturing conditions (see <B>Figure 15</B>).</P><CENTER><P><FONT color="#111"><CENTER><B>Figure 15.</B> SDS-PAGE analysis of the purification stages of PduD(1-20)_mCherry_cgPPK expressed under predicted optimal conditions. A large amount of the protein of the expected weight (66 kDa) appears to have been retained in the insoluble (insol) stage, with a roughly equimolar 70 kDa band that may correspond to a heat shock protein.</CENTER></FONT></P><P>From this we concluded that mCherry fluorescence in raw culture was not a suitable proxy for soluble protein yield because it did not account for protein aggregation in vivo. In retrospect, a more useful, but more labour intensive end point would have been mCherry fluorescence in the supernatant following cell disruption and removal of cell debris by centrifugation.</P></CENTER></CENTER></CENTER></CENTER></DIV></CENTER></CENTER></DIV><FOOTER class="footer-distributed"><DIV class="footer-right"><P>Connect with us:</P><P> Email: manchesterigem17@gmail.com </P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>