<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Manchester_Model_Continuous_Culture skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Manchester/Model/Continuous Culture</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="verytop"><NAV class="navbar navbar-default navbar-fixed-top visible-md visible-lg gradient bmc-bg" style=" padding-top: 20px; padding-left: 8vw; padding-right: 8vw"><DIV class="container-fluid"><DIV class="logo-center logoheader"><A class="navbar-logo" href="https://2017.igem.org/Team:Manchester"><H2 style="font-size: 1em!important; float: left; letter-spacing: 1px; padding-top: 10px!important; margin-top: 20px!important; width: 0px!important; overflow: visible!important">PHOSPHOSTORE</H2></A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin-top: 10px;"><LI class="dropdown dropdown10"><A class="dropdown-toggle active dropdowntextcenter" href="https://2017.igem.org/Team:Manchester/Description">Project</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Description1">Description</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Demonstrate">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Medals">Medal Criteria</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Achievement">Achievement</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Manchester">☆JUDGING FORM☆</A></LI></UL><LI class="dropdown dropdown10"><A class="dropdown-toggle active dropdowntextcenter" href="https://2017.igem.org/Team:Manchester/Model">Modelling</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Model/DoE">D. o. Experiment</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Model/Continuous_Culture">Cont. Culture</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Model/PSO">PHO Operon</A></LI><LI class="dropdown dropdown10"><A class="dropdown-toggle active dropdowntextcenter" href="https://2017.igem.org/Team:Manchester/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Improve">Improved Parts</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Part_Collection">Part Collection</A></LI><LI class="dropdown dropdown10"><A class="dropdown-toggle active dropdowntextcenter" href="https://2017.igem.org/Team:Manchester/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Diary">Diary</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Experiments">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Notebook">Lab book</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Measurement">Measurement</A></LI><LI class="dropdown dropdown13"><A class="dropdown-toggle active dropdowntextcenter" href="https://2017.igem.org/Team:Manchester/HP/Silver">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/HP/Silver">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/HP/Water_Industry">Water Industry</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/HP/Intellectual_Property">Intellectual Property</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Entrepreneurship">Entrepreneurship</A></LI><LI class="dropdown dropdown7"><A class="dropdown-toggle active dropdowntextcenter" href="https://2017.igem.org/Team:Manchester/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Manchester/Attributions">Attributions</A></LI></DIV></DIV></NAV></DIV><DIV class="igem_2017_menu_wrapper visible-sm visible-xs"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:Manchester"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:Manchester/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Manchester/Description"><DIV class="submenu_button" id="Description_page">
					Overview
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Description1"><DIV class="submenu_button" id="Description1_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstration
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Medals"><DIV class="submenu_button" id="Medals_page">
					Medal Criteria
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Achievement"><DIV class="submenu_button" id="Achievement_page">
					Achievement
				</DIV></A></DIV><DIV class="menu_button"> NOTEBOOK
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Manchester/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Protocols
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Diary"><DIV class="submenu_button" id="Diary_page">
					Diary
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Lab Book
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/InterLab"><DIV class="submenu_button" id="InterLab_page">
					InterLab
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Manchester/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Composite_Part"><DIV class="submenu_button" id="Composite_Part_page">
					Composite Parts
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Improve"><DIV class="submenu_button" id="Improve_page">
					Improved Parts
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Manchester/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Water Industry
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/HP/Water_Industry"><DIV class="submenu_button" id="Water_Industry_page">
					Integrated and Gold
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/HP/Intellectual_Property"><DIV class="submenu_button" id="Intellectual_Property_page">
					Intellectual Property
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement 
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Entrepreneurship"><DIV class="submenu_button" id="Entrepreneurship_page">
					Entrepreneurship
				</DIV></A></DIV><DIV class="menu_button"> MODELLING
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Manchester/Model"><DIV class="submenu_button" id="Model_page">
					Overview
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Model/DoE"><DIV class="submenu_button" id="DOE_page">
					D. O. Experiment
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Model/Continuous_Culture"><DIV class="submenu_button" id="CC_page">
					Continuous Culture
				</DIV></A><A href="https://2017.igem.org/Team:Manchester/Model/PSO"><DIV class="submenu_button" id="PSO_page">
					PHO Operon
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=Manchester"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><DIV class="col-md-12 sectionfirst" style="background-color: #fbf9f8!important"><H2 class="border" style="margin-top: 5vh; text-align: center">Continuous Culture of Bacteria</H2></DIV><DIV class="clear" style="background-color: #fbf9f8!important"><DIV class="plan" style="padding: 0px 6vw; border: 3px solid transparent"><DIV class="plan" style="padding: 0px 10vw; border: 3px solid transparent"><P><B>Achievements:</B></P><P style="margin-left: 40px">1. Predicted the output rate of a continuous culture system in different initial substrate concentrations, using computational modelling with parameters found in literature.</P><P style="margin-left: 40px">2. Estimated the production cost to produce 1 kg of bacteria, and the cost to treat 1 ton of waste water using phosphate data given by <A href="https://2017.igem.org/Team:Manchester/HP/Gold_Integrated" target="_blank">Davyhulme Treatment Works</A>.</P><P style="margin-left: 40px">3. Estimated the total cost to treat wastewater for a year, allowing us to predict the operation cost for our technology in our <A href="https://2017.igem.org/Team:Manchester/Entrepreneurship" target="_blank">business plan</A> and compare them to two existing technologies: Chemical Precipitation and Enhanced Biological Phosphate Removal.</P><P style="margin-left: 40px">4. Discussed a number of strategies to reduce cost further.</P><P style="margin-left: 40px">5. Calculated two of those cost-reduction strategies and determined the effectiveness of each strategy.</P><P style="margin-left: 40px">6. Determined the financial viability of our project and discussed alternative synthetic biology engineering strategies to improve the feasibility of our project in real life.</P><P><B>Introduction</B></P><P>We envision that our bacteria biomass would be cultured in a chemostat for <A href="https://2017.igem.org/Team:Manchester/Entrepreneurship" target="_blank">production</A>. A chemostat is a fermentation scheme, in which fresh medium is continually added whilst culture liquid containing leftover nutrients, microorganisms, and metabolic products are continuously removed at the same rate. This technique is called continuous culture and allows microbial growth to take place under steady-state conditions - growth that occurs at a constant rate and in a constant environment. Unlike a batch culture method where bacterial cells undergo the full bacterial cell cycle, a continuous culture keeps the bacteria growing on its exponential phase of the bacterial cell cycle, thus a continuous supply of bacteria can be produced.</P><P><B>Aim</B></P><P>Estimate the amount of bacteria required to clean up a given amount of phosphate, and the associated production cost to determine the profitability of our project using computational modelling.</P></DIV><DIV class="plan" style="padding: 0px 6vw; border: 3px solid transparent"><H4>Background Theory</H4><DIV class="col-lg-12 col-centered"><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P1"><H3 class="panel-title"><A role="button" class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#P1-collapse" aria-expanded="false" aria-controls="P1-collapse" style="color: #000000!important"><DIV><DIV class="col-md-11">Variables - Kinetics of Bacterial Growth</DIV></DIV></A></H3></DIV><DIV id="P1-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P1"><DIV class="panel-body"><P>The growth of bacteria in its exponential phase can be represented in the following exponential growth equation:</P>
\begin{equation} \frac{1}{x} \frac{dx}{dt} = \mu = \frac{log_e 2}{t_d} \end{equation}
<P> where: 
\(x\) is the <I>bacteria concentration</I> (dry weight mass/unit volume) at time \(t\)
\(μ\) is the <I>specific growth rate</I>
\(t_d\) is the <I>doubling time</I> (time required for the concentration of organism to double)</P><P>In 1942, Jacques Monod showed that there is a relationship between the specific growth rate and the concentration of a limiting growth substrate that can be represented in this equation:</P>
\begin{equation} \mu = \mu_{max} \bigg(\frac{s}{K_s + s}\bigg) \end{equation}
<P> where: 
\(s\) the <I>concentration of a limiting growth substrate</I>
\(μ_{max}\) is the <I>maximum growth rate</I> (growth rate when organism is placed in excess nutrients without any limiting factors)
\(K_s\) is the <I>saturation constant</I> – the value of \(s\) when: \(\frac{μ}{μ_{max}}\) = \(\frac{1}{2}\)
</P><P>A relationship between growth and utilization of substrate has also been shown by Monod by the equation:</P>

\begin{equation} \frac{dx}{dt} = −Y \frac{ds}{dt} \end{equation}

\begin{equation*} Y = \frac{\textrm{weight of bacteria formed}}{\textrm{substrate utilized}} \end{equation*}

<P>where \(Y\) is known as the <I>yield constant</I></P><P>If the values of the three growth constants: \(μ_{max}\), \(K_s\) and \(Y\) are known, equation (1) to (3) provides a complete quantitative description of the ‘growth cycle’ of a batch culture.</P></DIV></DIV></DIV></DIV></DIV><DIV class="col-lg-12 col-centered"><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P2"><H3 class="panel-title"><A role="button" class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#P2-collapse" aria-expanded="false" aria-controls="P2-collapse" style="color: #000000!important"><DIV><DIV class="col-md-11">Fundamental Equations of Continuous Culture</DIV></DIV></A></H3></DIV><DIV id="P2-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P2"><DIV class="panel-body"><P>A chemostat is a continuous flow system in which fresh growth medium is added into the vessel at a <I>steady flow-rate</I> 
 (\(F\)) and culture liquid exits at the same rate. Contents within the vessel are stirred so that the growth medium is uniformly dispersed. The rate in which nutrient is exchanged in the vessel is expressed as the <I>dilution rate</I>  (\(D\)):</P>

\begin{equation} D = \frac{\textrm{medium flow rate}}{\textrm{culture volume}} = \frac{F}{V} \end{equation}

<P> If we assume that the bacteria within the vessel stops growing and dividing, with equal stirring and continuous flow of medium, every organism will have an equal probability of leaving the vessel within a given time. The <I>wash-out rate</I> (rate in which organism initially present in the vessel will be washed out) can be expressed as:</P>

\begin{equation} - \frac{dx}{dt} = Dx \end{equation}

<P>where \(x\) is the concentration of organisms in the vessel</P><P><B>1. Changes in concentration of organism</B></P><P>In a continuous culture, bacteria are growing at a rate expressed in equation (1) and at the same time, it is being washed out at a rate expressed in equation (5). The net rate of increase is therefore:</P>

\begin{equation} \begin{split}
\textrm{increase} &amp; = \textrm{growth} - \textrm{output} \\

\frac{dx}{dt} &amp; = \mu x - Dx
\end{split} \end{equation}

<P> If:
\(μ\) &gt; \(D\), concentration of organism will increase
\(μ\) &lt; \(D\), concentration of organism will decrease
\(μ\) = \(D\), concentration of organism is constant; a ‘steady state’</P><P> Substituting (2) into \(μ\):</P>

\begin{equation} \frac{dx}{dt} = x \bigg\{\mu_{max} \bigg(\frac{s}{K_s + s}\bigg) - D\bigg\} \end{equation}
<P><B>2. Changes in substrate concentration</B></P><P>In a continuous culture, substrate enters the vessel at a concentration \(S_{in}\), consumed by the bacterial cell and exits the vessel at concentration \(S_{out}\). The net rate of change is therefore:</P>

\begin{equation*} \begin{split}
\textrm{increase} &amp; = \textrm{input} - \textrm{output} - \textrm{consumption} \\
&amp; = \textrm{input} - \textrm{output} - \frac{\textrm{growth}}{\textrm{yield constant}}
\end{split} \end{equation*}



\begin{equation} \frac{ds}{dt} = D S_{in} - D S_{out} - \frac{\mu x}{Y} \end{equation}

<P> Substituting (2) into \(μ\):</P>

\begin{equation} \frac{ds}{dt} = D (S_{in} - S_{out}) - \frac{\mu_{max} x}{Y} \bigg(\frac{s}{K_s + s}\bigg) \end{equation}

     </DIV></DIV></DIV></DIV></DIV><DIV class="col-lg-12 col-centered"><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P3"><H3 class="panel-title"><A role="button" class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#P3-collapse" aria-expanded="false" aria-controls="P3-collapse" style="color: #000000!important"><DIV><DIV class="col-md-11">Steady States</DIV></DIV></A></H3></DIV><DIV id="P3-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P3"><DIV class="panel-body"><P>From equation (7) and (9), if \(D\) and \(S_{in}\) are held constant, there is a unique solution for \(x\) and \(s\) for which \(\frac{dx}{dt}\) and \(\frac{ds}{dt}\) are 0. When \(\frac{dx}{dt}\) and \(\frac{ds}{dt}\) is 0, the system is said to be in a ‘steady state’ because the concentration of organism and substrate within the continuous culture is kept constant. The values of steady state \(x\) and \(s\), designated as \(\tilde{x}\) and \(\tilde{s}\) are expressed as:</P>

\begin{equation} \tilde{s} = K_s \bigg(\frac{D}{\mu_{max} - D}\bigg) \end{equation}


\begin{equation} \tilde{x} = Y (S_{in} - \tilde{s}) = Y \bigg\{ S_{in} - K_s \bigg(\frac{D}{\mu_{max} - D}\bigg) \bigg\} \end{equation}

<P>The steady state values depend solely on the value of \(D\) and \(S_{in}\) which can be manipulated within the chemostat. The values of growth constant \(μ_{max}\), \(K_s\) and \(Y\) are constant for a specific organism within a growth medium.</P></DIV></DIV></DIV></DIV></DIV><DIV class="col-lg-12 col-centered"><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P4"><H3 class="panel-title"><A role="button" class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#P4-collapse" aria-expanded="false" aria-controls="P4-collapse" style="color: #000000!important"><DIV><DIV class="col-md-11">Measuring Output</DIV></DIV></A></H3></DIV><DIV id="P4-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P4"><DIV class="panel-body"><P>The output rate is the quantity of bacteria produced in unit time. The total output from a continuous culture in a steady state is equal to the product of flow-rate and concentration of organism. The output per unit volume of culture is therefore \(D\tilde{x}\):</P>

\begin{equation} \textrm{Output} = D\tilde{x} = D * Y \bigg\{ S_{in} - K_s \bigg(\frac{D}{\mu_{max} - D}\bigg) \bigg\} \end{equation}

<P>We can measure the maximum output rate of organism by differentiating the above equation with respect to \(D\) and equating to zero. This will give us the dilution rate that gives the maximum output organism in unit time, \(D_{M}\):</P>

\begin{equation} D_M = \mu_{max} \Bigg\{ 1 - \sqrt{\frac{K_s}{K_s + S_{in}}} \Bigg\} \end{equation}

<P>The steady-state concentration of organisms at this dilution rate can be obtained by substituting \(D_M\) in equation (11):</P>

\begin{equation} \tilde{x}_M = Y \Bigg\{ (S_{in} + K_s) - \sqrt{K_s (S_{in} + K_s)} \Bigg\} \end{equation}

<P>The <I>maximum output rate</I>, \(D_{M}\tilde{x}_M\) is the product of the two equations:</P>

\begin{equation} D_M \tilde{x}_M = \mu_{max} * Y * S_{in} \Bigg\{ \sqrt{\frac{K_s + S_{in}}{K_s}} - \sqrt{\frac{K_s}{S_{in}}} \Bigg\}^2 \end{equation}

     </DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="plan" style="padding: 0px 6vw; border: 3px solid transparent"><DIV class="plan" style="padding: 0px 6vw; border: 3px solid transparent"><H4>Parameters</H4><TABLE class="table table-bordered table-striped"><TBODY><TR><TH>Parameter Name</TH><TH>Description</TH><TH>Value</TH><TH>Unit</TH><TH>Source</TH></TR><TR><TD>Growth Yield \(Y\) </TD><TD>A typical value of the observed growth yield of E. coli at rapid growth rates</TD><TD>0.44</TD><TD>grams dry weight / grams glucose</TD><TD>Majewski RA, Domach MM. Simple constrained-optimization view of acetate overflow in E. coli. Biotechnol Bioeng. 1990 Mar 25 35(7):732-8. p.736 left column paragraph above bottom</TD></TR><TR><TD>Saturation constant \(K_s\) </TD><TD>Monod substrate affinity constants (Ks) for E. coli growing on glucose as the only source of carbon and energy</TD><TD>0.099</TD><TD>grams / liter</TD><TD>Senn H, Lendenmann U, Snozzi M, Hamer G, Egli T. The growth of Escherichia coli in glucose-limited chemostat cultures: a re-examination of the kinetics. Biochim Biophys Acta. 1994 Dec 15 1201(3):424-36. P.425 left column top paragraph &amp; P.433 table 4</TD></TR><TR><TD>Maximum growth rate \(μ_{max}\) </TD><TD>Maximum growth rate for E. coli growing on glucose as the only source of carbon and energy</TD><TD>1.05</TD><TD>hour <SUP>-1</SUP></TD><TD>Senn H, Lendenmann U, Snozzi M, Hamer G, Egli T. The growth of Escherichia coli in glucose-limited chemostat cultures: a re-examination of the kinetics. Biochim Biophys Acta. 1994 Dec 15 1201(3):424-36. P.425 left column top paragraph &amp; P.433 table 4</TD></TR></TBODY></TABLE></DIV><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><H4>Modelling Output</H4><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><P>The output is modelled through the equation:</P>

\begin{equation*} \textrm{Output} = D\tilde{x} = D * Y \bigg\{ S_{in} - K_s \bigg(\frac{D}{\mu_{max} - D}\bigg) \bigg\} \end{equation*}
<P>There are two variables that can be changed according to the design of the chemostat: \(D\) (<I>dilution rate</I>) and \(S_{in}\) (<I>initial substrate concentration</I>).
</P><P>We modelled the output vs. dilution rate at five different initial substrate concentrations (1 g/L, 5 g/L, 10 g/L, 20 g/L and 40 g/L).</P><CENTER><FIGURE><FIGCAPTION>Figure 1: Relationship between output rate and dilution rate in different initial substrate concentration</FIGCAPTION></FIGURE></CENTER><P>We can see from the graph that for all five different concentrations of initial substrate, the output rate increase as dilution rate is increased. However, at a certain dilution rate, the output then starts to fall off dramatically. This is when the dilution rate value is higher than the specific growth rate of bacteria. At this point, the bacteria are washed out of the reactor at a faster rate than it can reproduce. Eventually, all the bacteria will be displaced out of the reactor.</P><P>The different points in the graph represent the dilution rate that gives the maximum output. As the initial substrate concentration increases, the maximum output rate is also shown to increase. In the figure below, we graphed the correlation between maximum output rate and initial substrate concentration from the values obtained from the graph above.</P><CENTER><FIGURE><FIGCAPTION>Figure 2: Correlation between maximum output and initial substrate concentration</FIGCAPTION></FIGURE></CENTER><P>We can see that the relationship between initial substrate concentration and maximum output rate is a positive linear line. There is theoretically no limit to the output rate that we can achieve, as long as we can afford the initial substrate concentration. In this case, the limitations would be financial limit, based on substrate concentration and cost (£/g).</P><P>The main assumption in this model is that all other factors for growth are in sufficient supply to ensure that glucose is kept as the limiting factor in the chemostat. In high concentrations of glucose, for example, there are other factors which may become limiting. Therefore, there would be an additional cost in supplying enough of these other factors, such as oxygen or nitrogen to ensure that it reaches the desired growth rate.</P><P>However, it is important to note from figure 1 that as initial substrate concentration increases, the graph gets steeper. The dilution rate required to get maximum output rate gets closer and closer to a critical dilution rate where output then starts to fall off. Therefore, we can see that while initial substrate concentration increases the maximum output rate at a constant rate (equal to the Growth Yield), there is also a greater risk of reaching a dilution rate which would ultimately wash out all the bacteria from the reactor.</P></DIV></DIV><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><H4>Cost Estimation</H4><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><P><B>Estimating Cost of Production</B></P><P>Our <A href="https://2017.igem.org/Team:Manchester/HP/Silver#cpi" target="_blank">interaction with John Liddell from the Centre Process of Innovation</A> told us that sources of glucose usually come from cheap media such as molasses. To determine the concentration of glucose in molasses, we searched through the United States National Nutrient Database for Standard Reference by the Department of Agriculture and found that molasses is ~12% glucose. Since molasses has a density of 1400 g/L (1 L = 1400 g), the concentration of glucose is therefore 168 g/L.</P><CENTER><FIGURE><FIGCAPTION>Figure 3. Relationship between output rate and dilution rate in 168 g/L substrate concentration</FIGCAPTION></FIGURE></CENTER><P>As seen in figure 3, at a dilution rate of 1.024 we can achieve maximum output rate of 73.94 g/L of bacteria liquid culture per hour. However, it is important to note that the dilution rate is really close to the critical dilution rate in which bacteria will be washed away from the reactor. Therefore, it is better to use a lower dilution rate to account for any variation in growth rate. We chose a dilution rate of 0.8 which will give an output rate of 59.02 g/L.</P><P>Now that we know the output rate, the next step is to determine the cost of producing a certain amount of bacteria.</P><P style="margin-left: 40px">-Molasses cost $0.07/kg</P><P style="margin-left: 40px">-The density of molasses is roughly 1.4 kg/L</P><P style="margin-left: 40px">-Therefore, 1 L of molasses will cost $0.07 x 1.4 = $0.098</P><P>To calculate the cost of 1 kg of bacteria, we’ll start by going back to the output equation:</P>

\begin{equation*} \textrm{Output} = D\tilde{x} = D * Y \bigg\{ S_{in} - K_s \bigg(\frac{D}{\mu_{max} - D}\bigg) \bigg\} \end{equation*}
<P>Since output rate is simply the <B>concentration of bacteria produced per unit time</B>, the units are:</P>

\begin{equation*} \textrm{Output} = \frac{g/L}{h} = \frac{\textrm{Amount (g)}}{\textrm{Time (h)}*\textrm{Volume (L)}} \end{equation*}

<P>The total cost of producing a 1 kg of bacteria can be calculated by taking into account several variables and multiplying it with the cost of 1 Liter of molasses:</P><P style="margin-left: 40px">1.	The time it takes to produce 1 kg. The faster we can produce 1 kg, the cheaper it will be.</P><P style="margin-left: 40px">2.	The dilution rate or turnover rate (the rate of nutrient exchange in the chemostat) which is equal to the flow rate divided by the volume of the chemostat. A higher dilution rate means a larger volume of substrate used which increases the output rate up to a certain point.</P><P style="margin-left: 40px">3.	The volume of the chemostat. The larger the volume, the more bacteria can be produced per time.</P><P>Therefore:</P>

\begin{equation*} \textrm{Cost} = \textrm{Time} * \textrm{Dilution rate} * \textrm{Volume (L)} * \frac{$0.098}{\textrm{L}} \end{equation*}
<P>To calculate the time required to produce 1 kg of bacteria, we can simply rearrange the output equation from before:</P>

\begin{equation*} \textrm{Output} = \frac{\textrm{Amount (g)}}{\textrm{Time (h)}*\textrm{Volume (L)}} \end{equation*}

\begin{equation*} \textrm{Time (h)} = \frac{\textrm{1 kg}}{\textrm{Output rate (g/L/h)}*\textrm{Volume (L)}} \end{equation*}
<P>Plugging it into the cost equation:</P>

\begin{equation*} \textrm{Cost} = \textrm{Time} * \textrm{Dilution rate} * \textrm{Volume} * $0.098 \end{equation*}

\begin{equation*} \textrm{Cost} = \frac{\textrm{1 kg}}{\textrm{Output rate}*\textrm{Volume}} * \textrm{Dilution rate} * \textrm{Volume} * $0.098 \end{equation*}
<P>Inserting our values and crossing out the same variables:</P>

\begin{equation*} \textrm{Production Cost} = \frac{\textrm{1000 g}}{\textrm{59.02 g/L/h}} * \textrm{0.8\(h^{-1}\)} * $0.098/L \end{equation*}

\begin{equation*} \textrm{} = \frac{\textrm{1000 g}* L * h}{\textrm{59.02 g}} * \frac{0.8}{h} * \frac{$0.098}{L} \end{equation*}

\begin{equation*} \textrm{} = $1.33\textrm{ for 1 kg} \end{equation*}

<P><B>Volume of Chemostat</B></P><P>The volume of the chemostat is independent to the production cost as it is cancelled in the equation above. The volume of the chemostat affects <I>how fast</I> it produces our desired product and therefore would reduce the running cost of the chemostat. We can graph the time equation to see the relationship between time and volume at an output rate of 59.02 g/L per hour.</P>

\begin{equation*} \textrm{Time (h)} = \frac{\textrm{1 kg}}{\textrm{Output rate (g/L/h)}*\textrm{Volume (L)}} \end{equation*}

<CENTER><FIGURE><FIGCAPTION>Figure 4. Relationship between time and chemostat volume at an output rate of 59.02 g/L  per hour</FIGCAPTION></FIGURE></CENTER><P>From the figure above, we can see that the time required to produce 1 kg of bacteria at an output rate of 59.02 g/L per hour decreases exponentially as the volume increases. It can be seen that we can produce 1 kg of bacteria in less than 2 hours in a chemostat volume above 10 L.</P><P><B>Estimating Cost to Clean Phosphate in Waste Water</B></P><P>Now that the basic cost of production has been calculated, the next step is to calculate the cost to sequester phosphate from a certain amount of waste water.</P><P>We were able to get some phosphate data from <A href="https://2017.igem.org/Team:Manchester/HP/Gold_Integrated" target="_blank">our visit to Davyhulme Treatment Works</A>. Since Davyhulme Treatment Works does not treat phosphate at their site, we can use the data as a case study to see how much it would cost to treat waste water that comes to the plant. The data below shows the concentration of phosphate in incoming waste water from the past 12 months:</P><CENTER><FIGURE><FIGCAPTION>Figure 5. Concentration of phosphate in incoming wastewater to Davyhulme Treatment Works in the past 12 months</FIGCAPTION></FIGURE></CENTER><P>Assuming that we treat 1 ton of waste water, equivalent to 1000 L, we would need to sequester 7323mg or ~7.3g of phosphate. To calculate the cost, we will have to find out how much bacterial preparation we need by knowing how much phosphate can be accumulated per gram of bacteria. Then, we can multiply this by the production cost that we have calculated earlier:</P>


\begin{equation*} \textrm{\(Cost_{P-Treatment}\)} = \textrm{Amount of phosphate} * \frac{\textrm{Amount of bacteria}}{\textrm{Accumulated phosphate}} * \textrm{Production cost} \end{equation*}

<P>Inserting values that we know:</P>

\begin{equation*} \textrm{\(Cost_{P-Treatment}\)} = \textrm{7.3g} * \frac{\textrm{Amount of bacteria}}{\textrm{Accumulated phosphate}} * \frac{$1.33}{\textrm{1 kg}} \end{equation*}

<P>To find out how much phosphate a bacterium can accumulate, we went to literature that we based our project on. In the paper <I>“Bacterial microcompartment-directed polyphosphate kinase promotes stable polyphosphate accumulation in E. coli</I>” (Liang et al., 2017), <I>E. coli</I> with microcompartment-direct PPK was able to accumulate (180 μg polyphosphate)/(mg protein in whole cell)  in 48 hours.</P><P>So we know that:</P>

\begin{equation*} \frac{\textrm{Amount of bacteria (\( g_{\textrm{ bacteria}}\))}}{\textrm{Accumulated phosphate (\( g_{\textrm{ phosphate}}\))}} = \frac{\textrm{180 \(μg_{\textrm{ phosphate}}\)}}{\textrm{\(mg_{\textrm{ whole cell protein}}\)}} * \textrm{...}\end{equation*}

<P>The equation is not complete, there are other variables that would have to be added, such as how many grams of protein are in a bacterium. We were able to obtain protein data from New England Biolabs which shows that a liquid culture of <I>E. coli</I> has a total protein weight of 0.15 g per liter at 10<SUP>9</SUP> cells per ml. Therefore:</P>

\begin{equation*} 0.15 \textrm{ g/L}=150 \textrm{ mg/L} \end{equation*}

\begin{equation*} \frac{\textrm{Amount of bacteria (\( g_{\textrm{ bacteria}}\))}}{\textrm{Accumulated phosphate (\( g_{\textrm{ phosphate}}\))}} = \frac{\textrm{180 \(μg_{\textrm{ phosphate}}\)}}{\textrm{\(mg_{\textrm{ whole cell protein}}\)}} * \frac{\textrm{\(150  mg_{\textrm{ whole cell protein}}\)}}{L * \textrm{\(10^{9}\) cells/ml}} * \textrm{...}\end{equation*}

<P>The next variable that is missing is the number of cells per wet weight of bacteria. A study has shown that an OD<SUB>600</SUB> of 1.0 will have a cell wet weight of 1.7 g/L (Glazyrina et al., 2010). For bacterial cell cultures, an OD<SUB>600</SUB> of 1.0 corresponds to 8 x 10<SUP>8</SUP> cells/ml. Thus:</P>

\begin{equation*} \frac{8 * \textrm{\(10^{8}\) cells/ml}}{1.7 \textrm{g/L}} \end{equation*}

<P>Adding it into the previous equation:</P>

\begin{equation*} \frac{\textrm{Amount of bacteria (\( g_{\textrm{ bacteria}}\))}}{\textrm{Accumulated phosphate (\( g_{\textrm{ phosphate}}\))}} = \frac{\textrm{180 \(μg_{\textrm{ phosphate}}\)}}{\textrm{\(mg_{\textrm{ whole cell protein}}\)}} * \frac{\textrm{\(150 mg_{\textrm{ whole cell protein}}\)}}{L * \textrm{\(10^{9}\) cells/ml}} * \frac{8 * \textrm{\(10^{8}\) cells/ml}}{1.7 \textrm{g/L}}\end{equation*}

<P>Now we have all the variables needed to cross out the units, thus we can solve it as:</P>

\begin{equation*} \frac{\textrm{\( g_{\textrm{ bacteria}}\)}}{\textrm{\( g_{\textrm{ phosphate}}\)}} = \textrm{180\(μg_{\textrm{ phosphate}}\)} * \frac{150}{\textrm{\(10^{9}\)}} * \frac{8 * \textrm{\(10^{8}\)}}{1.7 \textrm{g}}\end{equation*}

\begin{equation*} \frac{\textrm{\( g_{\textrm{ bacteria}}\)}}{\textrm{\( g_{\textrm{ phosphate}}\)}} = \frac{\textrm{12705.88  \( μg_{\textrm{ phosphate}}\)}}{\textrm{\( g_{\textrm{ bacteria}}\)}} = \frac{\textrm{0.01270588 \( g_{\textrm{ phosphate}}\)}}{\textrm{\( g_{\textrm{ bacteria}}\)}}\end{equation*}

\begin{equation*} \frac{\textrm{\( g_{\textrm{ bacteria}}\)}}{\textrm{\( g_{\textrm{ phosphate}}\)}} = \frac{1}{\frac{\textrm{0.01270588 \( g_{\textrm{ phosphate}}\)}}{\textrm{\( g_{\textrm{ bacteria}}\)}}}\end{equation*}

<P>Thus:</P>

\begin{equation*} \frac{\textrm{Amount of bacteria (\( g_{\textrm{ bacteria}}\))}}{\textrm{Accumulated phosphate (\( g_{\textrm{ phosphate}}\))}} = \frac{\textrm{\(78.7 g_{\textrm{ bacteria}}\)}}{\textrm{\( g_{\textrm{ phosphate}}\)}}\end{equation*}

<P>Therefore, we can plug it into the cost equation to determine how much it cost to treat 1 ton of wastewater:</P>

\begin{equation*} \textrm{\(Cost_{P-Treatment}\)} = \textrm{Amount of phosphate} * \frac{\textrm{Amount of bacteria}}{\textrm{Accumulated phosphate}} * \textrm{Production cost} \end{equation*}

\begin{equation*} \textrm{\(Cost_{P-Treatment}\)} = \textrm{\(7.3g_{\textrm{ phosphate}}\)} * \frac{\textrm{\(78.7 g_{\textrm{ bacteria}}\)}}{\textrm{\( g_{\textrm{ phosphate}}\)}} * \frac{$1.33}{\textrm{\(1 kg_\textrm{ bacteria}\)}} \end{equation*}

\begin{equation*} \textrm{\(Cost_{P-Treatment}\)} = $0.76 \end{equation*}

<P>Therefore, it would cost <B>$0.76</B> to produce bacteria that can accumulate 7.3g of phosphate in 1 ton of wastewater in 48 hours.</P></DIV></DIV><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><H4>Real Life Scenario</H4><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><P>In Davyhulme Treatment Works, flows of more than 30,000 litres per second are treated for 24 hours straight every single day. This is equivalent to 2 592 000 tons of water treated every single day or 946 080 000 tons of water every year (United Utilities, 2017).</P><P>We have previously calculated that it would cost $0.76 to treat 1 ton of waste water in 48 hours. To treat 946 080 000 tons of water would therefore cost <B>$722 898 120</B> or roughly <B>£550 million</B> every year. With an estimate of 1.17 million households in Greater Manchester in 2016 (New Economy Manchester, 2016), the cost of implementing this technology would be an additional £470 per household each year which is expensive. We would need substantial improvements of our system to make it competitive in the real-world market – costs would have to be reduced further before water companies would be interested in our technology. We therefore considered alternative strategies to drastically re-engineer our system to achieve a substantially better economic position.</P></DIV></DIV><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><H4>Cost-Reduction Strategy</H4><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><P>We discussed a number of other potential strategies how we can further reduce the cost and how it can potentially be achieved:</P><P><B>1. Grow bacteria directly in the waste water stream</B></P><P>If grown directly in the waste water stream, we would no longer face the need to replenish the bacteria as they would be sustaining themselves on the nutrients provided in the waste. To ensure that this can work, we would have to develop a safety mechanism that would prevent bacteria from escaping to the wild, such as kill-switches. Although various <A href="http://parts.igem.org/Biosafety" target="_blank">kill-switches</A> have been developed by numerous iGEM teams in the past, some of them are 'leaky' and does not guarantee 100% cell death. As an alternative strategy, we also thought of using a bacterial chassis that has been genetically modified to 

<A href=" https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4579030/ " target="_blank">require unnatural amino acids</A>

to produce essential proteins. While some unnatural amino acids are synthetic and expensive to produce, there are around 300 of them that can be found in nature and may be cheaper to produce.</P><P>However, we considered that growing bacteria directly in the waste water stream may not be viable after <A href="https://2017.igem.org/Team:Manchester/Notebook" target="_blank">observing in the lab</A> that microcompartment expression substantially inhibits bacterial growth and makes the cells ‘sick’. To circumvent this, we thought about implementing a phosphate starvation operon to inhibit microcompartment expression in low phosphate concentration. Through this, the bacteria would be able to grow optimally in the chemostat and would only express microcompartments in waste water (high phosphate concentrations). We modelled a <A href="https://2017.igem.org/Team:Manchester/Model/PSO" target="_blank">phosphate starvation operon </A>where we showed microcompartment expression at different phosphate levels.</P><P><B>2. Use of a different substrate</B></P><P>Although <I>Escherichia coli</I> grows best on glucose, there are other carbon sources that are much cheaper that could also be used, such as glycerol. The increase in demand for fuel and increased environmental concern have emphasized the need for renewable sources of energy and have increased the production of biodiesel. With its production capacity increased and developed in recent years, supply of its by-product – crude glycerol – has also increased and resulted in a dramatic reduction of price over the past years (Yang et al., 2012). In 2011, the price of glycerol fell to $0.04-0.11 per kg (Quispe et al., 2013). Thus, using a different substrate may potentially reduce the production cost significantly.</P><P>However, it is important to note that the maximum growth rate of <I>E. coli</I> would be lower when grown in glycerol than in glucose. This is an opportunity for synthetic biology: it would be generally beneficial to engineer <I>E. coli</I> so that it can efficiently take up glycerol, possibly through engineering genes involved in catabolite repression.</P><P><B>3. Different chassis</B></P><P>The calculations and experiments done for our project are focusing on <I>E. coli</I> as a chassis and we have observed substantially reduced growth when these cells express microcompartments. <I>E. coli</I> is chosen because it is easy to engineer and is suitable for us to work within the time constraints of iGEM. But there are other chassis that would be more suitable for real-world industrial applications of this project. These include other bacterial species and even photosynthetic organisms such as algae or cyanobacteria. Whilst the use of photosynthetic organisms presents its own challenges, they are of particular interest as their use would circumvent the need for supplying a costly carbon source.</P><P><B>4. Phosphate fertilizer</B></P><P>Another potentially important aspect of increasing the economic viability of our synthetic biology approach is the opportunity of recovering some of the cost by selling phosphate fertilizer. We have originally thought about this since the beginning, and we envisaged this as one of the main selling points of our project. Since the bacteria accumulate phosphate in their microcompartments, we can harvest them afterwards and make phosphate fertilizer out of them. We thought about putting our bacteria in a semi-permeable membrane that would only let water pass through the membrane and keep the bacteria inside. This would achieve two goals: a robust device that can prevent the bacteria from escaping into the wild and a way to concentrate the bacteria in one place to facilitate efficient phosphate harvesting.</P><P>We made two further cost calculations in the next section for two of our possible strategies: phosphate fertilizer and use of a different substrate.</P></DIV></DIV><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><H4>Phosphate Fertilizer</H4><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><P>One of the selling points of our project is the ability to harvest the phosphate and then sell it further to generate some profit, improve resource security and agricultural sustainability. This profit can be used to further reduce the operating cost of our technology and consequently the price for customers.</P><P>Assuming that we can reliably accumulate the required amount of phosphate from Davyhulme Water Treatment Facility in our bacteria, we can calculate the amount of phosphate per year that we can harvest:</P>

\begin{equation*} \frac{\textrm{7.3229  mg}}{L} * \textrm{946 080 000 000 L} = \textrm{6 928 049 kg} \end{equation*}

<P>To calculate our pricing cost, we use the market cost for Diammonium Phosphate (DAP), the most widely used phosphate fertilizer which is priced at £348-£355 per ton or an average of £352 per ton (Horne, 2016):</P>

\begin{equation*} \textrm{6 928 049 kg} * \frac{£352}{\textrm{1000 kg}} = \textrm{£2 438 673} ≈ \textrm{£2.4 million} \end{equation*}

<P>Assuming that all the phosphate is sold at this price, we can cut down the price of treating waste water from the revenue gained through selling phosphate. However, this value is small when compared to the £550 million predicted treatment cost (considering our initial design).</P></DIV></DIV><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><H4>Using Glycerol as a Substrate</H4><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><P>The calculation from the previous section shows that the price to implement the technology is still too expensive to be affordable, even if the substrate concentration could be increased. There are two other ways to reduce the cost further. One of them is to increase the amount of phosphorus that can be accumulated in the microcompartment by using a different PPK enzyme or some other means. Another way would be simply to use a cheaper alternative source of carbon.</P><P>In this section, we will calculate the cost of bacteria production in a continuous culture using glycerol as the substrate. Since the price of glycerol is at a range of $0.04-0.11 per kg, we will use its median price at $0.08 (Quispe et al., 2013). In addition, we will also assume that the growth yield (Y) of bacteria growing in glycerol is the same as when it is growing in glucose at 0.44 (we could not find the growth yield value with glycerol in the literature but we assume that this could realistically be achieved, as a result of synthetic biology engineering of the bacteria’s metabolism).</P><P>Using the production cost equation, we can plug in the values for dilution rate and maximum output rate to obtain the cost of producing 1 kg of bacteria using glycerol:</P>

\begin{equation*} \textrm{Production Cost} = \frac{\textrm{1000 g}}{\textrm{Output rate}} * \textrm{Dilution rate} * \textrm{Price of glycerol} \end{equation*}

<P>The price of glycerol will depend on the initial substrate concentration; the higher the initial substrate concentration, the more glycerol we would need:</P>

\begin{equation*} \textrm{Price of glycerol} = \textrm{Initial substrate concentration (g/L)} * $0.08/kg \end{equation*}

<P>Therefore:</P>

\begin{equation*} \textrm{Production Cost} = \frac{\textrm{1000 g}}{\textrm{Output rate}} * \textrm{Dilution rate} * \textrm{Initial substrate concentration (g/L)} * \frac{$0.08}{\textrm{1000 g}} \end{equation*}

<P>Plugging in the values that we know:</P><TABLE class="table table-bordered table-striped"><TBODY><TR><TH>Initial Substrate Concentration</TH><TH>Output rate</TH><TH>Dilution rate</TH></TR><TR><TD>168 g/L</TD><TD>59.02</TD><TD>0.8</TD></TR></TBODY></TABLE>

\begin{equation*} \textrm{Production Cost} = $0.18 \end{equation*}

<P>Using the phosphate treatment cost equation, we can plug in the production cost into the equation to obtain the cost of treating 1 ton of waste water:</P>

\begin{equation*} \textrm{\(Cost_{P-Treatment}\)} = \textrm{Amount of phosphate} * \frac{\textrm{Amount of bacteria}}{\textrm{Accumulated phosphate}} * \textrm{Production cost per kg} \end{equation*}

\begin{equation*} \textrm{\(Cost_{P-Treatment}\)} = \textrm{\(7.3g_{\textrm{ phosphate}}\)} * \frac{\textrm{\(78.7 g_{\textrm{ bacteria}}\)}}{\textrm{\( g_{\textrm{ phosphate}}\)}} * \frac{\textrm{Production cost}}{\textrm{\(1000 g_{\textrm{ bacteria}}\)}} \end{equation*}

\begin{equation*} \textrm{\(Cost_{P-Treatment}\)} = $0.103 \end{equation*}

<P>Thus, we can find the total cost to treat 946 080 000 tons of water and from there, we can divide the total cost with the number of households in Greater Manchester (1.17 million) to calculate how much the implementation of this technology will cost per household:</P><TABLE class="table table-bordered table-striped"><TBODY><TR><TH>Cost P-Treatment per 1 ton</TH><TH>Total Cost P-Treatment</TH><TH>Cost per household per year</TH></TR><TR><TD>$0.103</TD><TD>$97 446 240</TD><TD>$83.29</TD></TR></TBODY></TABLE><P>We can see that the price is significantly reduced when using glycerol in comparison to glucose. We would now reach a cost level that perhaps comes close to an economically feasible range (although still too high to be realistically competitive). However, glucose is the preferred carbon source for <I>E. coli</I> and substituting it with glycerol may result in different growth rates. <I>E. coli</I> grown on glycerol has been shown to express different levels of certain metabolic genes which may possibly lead to slower growth and metabolism. In addition, there is not enough data and studies in regards to the carbon stress response of <I>E. coli</I> grown in glycerol. Carbon stress is a phenomenon that is exhibited by <I>E. coli</I> when grown in carbon-limiting environments such as in a chemostat and is therefore an important aspect to consider for industrial applications (Martinez-Gomez et al., 2012).</P><P>Perhaps there is an opportunity to use synthetic biology to engineer a strain of <I>E. coli</I> that can utilize glycerol as effectively as glucose. With the increased production of biodiesel, glycerol as a by-product is becoming more and more appealing as an alternative carbon source due to its cheap cost. An <I>E. coli</I> that can utilize glycerol effectively would possibly reduce the production cost of many other commercial bacterial cultures in addition to ours. It would potentially reduce the production cost of various commercially produced proteins such as enzymes and antibiotics, which could lead to more affordable healthcare and advancement in research.</P></DIV></DIV><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><H4>Summary</H4><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><P>We have made a rough cost estimation on the <B>cost to treat waste water</B> using our synthetic biology microcompartments in bacteria. Our calculations are using phosphate data obtained from <A href="https://2017.igem.org/Team:Manchester/HP/Silver#davyhulme" target="_blank">our interaction with Davyhulme Treatment Works</A>. From our cost estimation, the cost to treat phosphate using our original design would be high: to remove phosphate from waste water for a year with an output rate of 59.02 g/L per hour would cost <B>£550 million</B>, which is equivalent to <B>£470 per household in Greater Manchester</B>, and this is excluding the cost to set-up and maintain the equipment. This is quite <B>expensive</B> and would not be competitive in a real-life market. From this calculation, we were able to integrate it into <A href="https://2017.igem.org/Team:Manchester/Entrepreneurship#sec3" target="_blank">our business plan and assess the operation and maintenance cost of our device</A> in comparison with two other technologies:
 Chemical Precipitation and Enhanced Biological Phosphate Removal (EBPR)</P><P>We discussed a number of <B>alternative synthetic biology engineering strategies</B> that would improve the economic viability of our project, including alternative chassis (e.g. photosynthetic algae), self-sustaining microbes feeding on wastewater with engineered kill switches, and metabolically engineered bacteria that can use much cheaper glycerol as their carbon source.</P><P>In another attempt to reduce predicted cost, we calculated how much revenue we can earn from <B>selling back the phosphate</B> that is accumulated and found that it would reduce the production cost by <B>£2.4 million</B>. Unfortunately, this is a small fraction of the entire production cost. Next, we calculated the production cost by using <B>glycerol</B> as a substrate instead of glucose. From our calculations, the cheap price of glycerol has reduced the cost per household per year <B>by 5-fold</B>. </P><P>We concluded that glycerol as a carbon source for <I>E. coli</I> would be needed for our project to be affordable and financially feasible, but the different metabolic pathway that the bacteria uses when grown in glycerol would slow down growth. Therefore, additional synthetic biology would be required to engineer a strain of <I>E. coli</I> that can utilize glycerol as effectively as glucose by modifying or improving its metabolic pathway.</P></DIV></DIV><DIV class="plan" style="padding: 0px 5vw; border: 3px solid transparent"><H4>References</H4><DIV class="plan" style="padding: 0px 10vw; border: 3px solid transparent"><P>Bailey, J. E. &amp; Ollis, D. F. (1986). <I>Biochemical Engineering Fundamentals.</I> 2nd edn. Singapore: McGraw-Hill Book Company.</P><P>Biolabs, N. (2017). <I>Protein Data | NEB.</I> [online] Neb.com. Available at: https://www.neb.com/tools-and-resources/usage-guidelines/protein-data (Accessed 8 Sep. 2017).</P><P>Genomics.agilent.com. (2017). <I>Agilent Genomics: Tools - Bio Calculators.</I> [online] Available at: http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp (Accessed 9 Sep. 2017).</P><P>Glazyrina J, Materne EM, Dreher T, Storm D, Junne S, Adams T, Greller G, Neubauer P. High cell density cultivation and recombinant protein production with Escherichia coli in a rocking-motion-type bioreactor. Microb Cell Fact. 2010 May 30 9: 42. doi: 10.1186/1475-2859-9-42. p.4 left column 5th paragraph</P><P>Herbert, D., Elsworth, R. and Telling, R. (1956). The Continuous Culture of Bacteria; a Theoretical and Experimental Study. <I>Journal of General Microbiology</I>, 14(3), pp.601-622.</P><P>Horne, S. (2017). <I>Farm fertiliser prices steady as spring peak approaches.</I> [online] Farmers Weekly. Available at: http://www.fwi.co.uk/business/farm-fertiliser-prices-steady-as-spring-peak-approaches.htm (Accessed 12 Sep. 2017).</P><P>Martinez-Gomez, K., Flores, N., Castaneda, H. M., Martinez-Batallar, G., Hernandez-Chavez, G., Ramirez, O. T., Gosset, G., Encarnacion, S. &amp; Bolivar, F. 2012. New insights into Escherichia coli metabolism: carbon scavenging, acetate metabolism and carbon recycling responses during growth on glycerol. <I>Microbial Cell Factories,</I> 11, 21.</P><P>New Economy Manchester. (2016). <I>Greater Manchester Key Facts.</I> [online] Available at: http://www.neweconomymanchester.com/media/1474/ne-key-facts-dec-15-web.pdf (Accessed 14 Sep. 2017).</P><P>Office for National Statistics. (2016). <I>Families and households in the UK: 2016.</I> [online] Available at: https://www.ons.gov.uk/peoplepopulationandcommunity/birthsdeathsandmarriages/families/ bulletins/familiesandhouseholds/2016 (Accessed 14 Sep. 2017).</P><P>Quispe, C. A. G., Coronado, C. J. R. &amp; Carvalho, J. A. 2013. Glycerol: Production, consumption, prices, characterization and new trends in combustion. <I>Renewable &amp; Sustainable Energy Reviews,</I> 27, 475-493.</P><P>Sigma-Aldrich. (2017). <I>D-(+)-Glucose G8270.</I> [online] Available at: http://www.sigmaaldrich.com/catalog/product/sigma/g8270?lang=en&amp;region=GB (Accessed 9 Sep. 2017).</P><P>United States Department of Agriculture Research Service. (2016). Full Report (All Nutrients): 19034, Molasses. [online] Available at: National Nutrient Database for Standard Reference Release 28 (Acessed 9 Oct. 2017).</P><P>United Utilities (2017). <I>Davyhulme Treatment Works.</I> [online] UnitedUtilities.com. Available at: https://www.unitedutilities.com/help-and-support/about-us/greater-manchester/davyhulme-treatment/ (Accessed 12 Sep. 2017).</P><P>Yang, F. X., Hanna, M. A. &amp; Sun, R. C. 2012. Value-added uses for crude glycerol-a byproduct of biodiesel production. <I>Biotechnology for Biofuels,</I> 5, 10.</P></DIV></DIV></DIV></DIV><FOOTER class="footer-distributed"><DIV class="footer-right"><P>Connect with us:</P><P> Email: manchesterigem17@gmail.com </P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>