Team:Botchan Lab Tokyo/Project
HOME
PROJECT
RESULTS LABORATORY NOTEBOOK MODELING
LAB
SAFETY INTERLAB
PARTS
COMPOSITE PARTS PARTS BOOK
HUMAN PRACTICE
SILVER GOLD COLLABOLATIONS
TEAM
MEMBER ATTRIBUTION
Projects Results
1.Experiment plan
1.1.Overview
This year, we tried to reproduce Last year's project at first.    Last year, We tried to synthesize amino acids from nitrogen in the air inside Escherichia coli. By constructing that pathway in E.coli, and by introducing the "bacteria products" to human intestine,we hope that it will be possible for people to "eat" nitrogen.
We divided the plan into following three specific pathways.
The First pathway is expressing nitrogenase to create ammonium from nitrogen gas.     To express nitrogenase adequately inside E.coli, we chose several nif gene groups and introduced them in E.coli.      Although we hoped that synthesized ammonium would accumulate in a cell, the ammonium seemed to have been used in a metabolic       pathway  in E.coli itself.
Therefore, as the second pathway, we used an inhibitor called MSX to inhibit the glutamate synthesis pathway - which is one of the ammonium using anabolic pathways in E.coli - to make the ammonium accumulate in a cell.
And finally, by introducing and expressing genes of amino acids dehydrogenase, synthesize amino acids from the ammonium.      But this year we targeted on glutamic acid dehydrogenase because many kinds of amino acids are synthesized from glutamic acid.
Figure1:The pathway of getting amino acids from nitrogen in the air
Figure2:The inhibition of Glutamine synthesis pathway by using MSX.
1.2.Nitrogen-fixing reaction
Figure3:nitrogenase's activity
Nitrogen is primary component of atmosphere, so there are much nitrogen around us.     But most living organisms are not able to use nitrogen directly,     because nitrogen exists as a less reactive molecular form.     Nitrogenase has an ability to convert nitrogen into highly reactive substance"ammonium". Nitrogenase is expressed by the “nif” genes. The expression of nitrogenase is a burden to E.coli, so we had to reduce the burden as much as possible. Therefore, we decided to introduce the minimal gene cluster which is necessary for expressing nitrogenase into E.coli.  We picked the minimal nif genes (nifB,H,D,K,N,X,hesA)  from the strain Paenibacillus Polymyxa ATCC 15970. And also, in addition to the genes which is essential for expressing this nitrogenase, we also introduced genes (nifV,S,U) from the strain lahnella aquatillus that have an ability to increase the activity of nitrogenase. We introduced these two nif gene groups into two vectors. The designs of the plasmids are mentioned in 1.4 .
1.3.Glutamate-synthesizing reaction
Many kinds of amino acid are synthesised from Glutamic acid.    So this year, we decided start with glutamic acid synthesis.The reaction is catalyzed by Glutamate dehydrogenase.
As shown above, ammonium is necessary to synthesize glutamic acid.We picked the glutamate dehydrogenase gene from the strain Bacillus licheniformis.
1.4.The design of our plasmid
We used two vectors to construct plasmids because nifgenes were too long. PHY plasmid contains nif-B to hesA genes. And pMW plasmid contains nifV,S,Ugenes and glutamate dehydrogenase gene(glu DH)..
Figure4:pHY plasmid(nifB~hesA)
Figure5:pMW plasmid(nifVSU+gluDH)
2.Result
2.1.Introduce nif-gene
Figure6:Reconfirmation of the last year's samples
Sample
JM109 (pHY);empty vector
JM109(pHY nif nifB,H,D,K,N,X,hesA
JM109(pMW);empty vector
JM109(pMW nifVSU
JM109(pHY+pMW);empty vectors
JM109(pHY nifB,H,D,K,N,X,hesA,pMW nifVSU)
*1~5;control
We found that transformant which we made last year had nifBHDKNXhesA genes. (sample2)But appeared that nifVSU genes were not inserted(sample3,4). Besides, it's unclear why pHY plasmid band was unshown.
So we cloned nifVSU genes from Lahnella Aquatillus.
Figure7;PCR(nifVSU genes and pMW)
1,2 pMW-Fw,Rv
3,4 nifVSU-Fw,Rv for restriction enzyme
5,6 nifVSU-Fw,Rv for infusion
Fw;Forward primer Rv;Reverse primer
We did infusion cloning and made transformant which assumed to be had nifVSU genes. Then,we did insert check by plasmid extraction from the cells.
Figure8;Insert check for pMW, nifVSU genes.
Sample
pMW(extracted from July 12Sample1)
pMW-nifVSU(extracted from July 12Sample2)
pMW(Digested by SacI)
pMW,nifVSU(Digested by SacI)
pMW(Digested by XmaI)
pMWnifVSU(Digested by XmaI)
We introduced two plasmids(pHYnifB~hesA and pMWnifVSU into one cell.
Figure9;Two vectors introducted in one sample
Sample
pMWnifVSU +pHY nifB,H,D,K,N,X,hesA(August 9)
pMW+pHY (August 9)
pHY
pHY nifB,H,D,K,N,X,hesA
pMW
pMWnifVSU
2.2.Introduce dehydrogenase gene
After that, we cloned glutamic acid dehydrogenase gene from Bacillus licheniformis and inserted it in the downstream of pMW nifVSU genes, introducing the plasmid into E.coli. JM109.
2.3.Determine the activity of nitrogenase
We investigated the activity of nitrogenase by measuring the amount of synthesized ammonium by using the indofenol method. Indofenol is a blue compound and it is formed by the ammonium using reaction. If ammonium is exsisted in the solution, the color of the liquid changes into blue. The absorbance of solution (depth of blue) increases in proportion to ammonium concentration, thus we can determine the amount of ammonium in cells by measuring the absorbance.  At first we drew the standard curve using ammonium solution.
Figure10;Standard curve(relation between the ammonium concentration and the absorbance of solution)
Next, we confirmed nitrogen fixation ability of the Paenibacillus polymyxa which has nifB,H,D,K,N,X,hesA genes. Before moving indofenol reaction, we replaced the air with nitrogen gas in the medium and incubated the samples for 24 or 48 hours. All indofenol samples were adjusted as well as these samples.
Calibration curve between ammonia concentration  and Abs. Ammonia concentration and Abs is proportional.
We confirmed Nitrogen fixation ability of Paenibacillus polymyxa.
Figure11;Comparison of ammonium amount between Paenibacillus(diazotroph) and E.coli(non-diazotroph)
This graph suggests Paenibacillus Polymyxa ATCC15970(diazotroph) produced more ammonium than that of E.coli JM109.
Indophenol sample
Sample Name
Phenol/EtOH(µl)
Sodium nitroprusside/H2O(µl)
Sodium hypochlorite(µl)
1.E.coli(pHYnifB~hesA)
40
40
100
2.E.coli(pHYnifB~hesA+pMWnifVSU)
40
40
100
3.E.coli(pHY)
40
40
100
4.E.coli(pHYnifB~hesA+pMW)
40
40
100
5.E.coli(pHY+pMW)
40
40
100
6.Paenibacillus polymyxa
40
40
100
7.Medium only
40
40
100
Figure12;The result of indofenol method(48h,72h)
Paenibacillus showed the highest ammonium accumulation. The E.coli with pHY nifB~hesA genes and the E.coli with pHY nifB~hesA, pMWnifVSU)showed high ammonium accumulation. But LB medium sample(negative control) showed high Abs too. This result suspects the contamination in the medium.
Indophenol(48 h)MSX
Sample Name
Phenol/EtOH(µl)
Sodium nitroprusside/H2O(µl)
Sodium hypochlorite(µl)
1.E.coli(pHYnifB~hesA+pMWnifVSU)
40
40
100
2.E.coli(pHY+pMW)
40
40
100
3.Paenibacillus polymyxa
40
40
100
4.Medium only
40
40
100
Figure.13;The result of indofenol method with MSX(48h,72h)
All data except Paenibacilllus showed higher ammonium accumulation than that of samples without MSX, and pHYnifB~hesApMWnifVSU showed the highest ammonium accumulation.
Indopheol(72h)
Sample Name
Phenol/EtOH(µl)
Sodium nitroprusside/H2O(µl)
Sodium hypochlorite(µl)
1.E.coli(pHYnifB~hesA)
40
40
100
2.E.coli(pHYnifB~hesA+pMWnifVSU)
40
40
100
3.E.coli(pHY)
40
40
100
4.E.coli(pHYnifB~hesA+pMW)
40
40
100
5.E.coli(pHY+pMW)
40
40
100
6.Paenibacillus polymyxa
40
40
100
7.medium only
40
40
100
Figure.14;The result of indofenol method(72h)
E.coli with empty pHY vector showed high ammonium accumulation. It is different from the result of Figure12. Some factors expect ammonium might have affected the absorbance.
Indopheol(72h)MSX
Sample Name
Phenol/EtOH(µl)
Sodium nitroprusside/H2O(µl)
Sodium hypochlorite(µl)
1.E.coli(pHYnifB~hesA+pMWnifVSU) MSX
40
40
100
2.E.coli(pHY) MSX
40
40
100
3.E.coli(pHYnifB~hesA+pMW) MSX
40
40
100
4.E.coli(pHY+pMW) MSX
40
40
100
5.Paenibacillus polymyxa MSX
40
40
100
Figure15;The result of indofenol method with MSX(72h)
All data except Paenibacilllus showed higher ammonia accumulation than that of samples without MSX, and the E.coli with pHYnifB~hesApMW showed the highest ammonium accumulation and the E.coli with pHYnifB~hesApMWnifVSU was the second. Still, the negative control samples(pHY and pHY,pMW) showed not a little absorbance.
To decrease the absorbance of negative controls, we tried reducing nitrogen source in the medium to 0.3 mM Glutamate (normally it's 2 mM Glutamate). And we excluded normal LB medium by centrifuging when we inoculate E.coli into Nitrogen deficient medium(0.3 mM Glutamate) for the indofenol measurement.
Indophenol(48h)
Sample Name
Phenol/EtOH(&microl)
Sodium nitroprusside/H2O(&microl)
Sodium hypochlorite(&microl)
1.E.coli(pHYnifB~hesA)
40
40
100
2.E.coli(pHYnifB~hesA+pMWnifVSU)
40
40
100
3.E.coli(pHY)
40
40
100
4.E.coli(pHYnifB~hesA+pMW)
40
40
100
5.E.coli(pHY+pMW)
40
40
100
6.Paenibacillus polymyxa
40
40
100
7.Medium only
40
40
Figure16;The result of indofenol method with reduced nitrogen source in the medium(48h).
In this result, the ammonium accumulation was too low to measure. Even, we couldn't see the changing of color. All the samples were transparent.
That's probably because in the few presence of nitrogen source, the ammonium produced by nitrogenase was used in an anabolism of E.coli itself.
Indophenol(48h)MSX-1
Sample Name
Phenol/EtOH(&microl)
Sodium nitroprusside/H2O(&microl)
Sodium hypochlorite(&microl)
1.E.coli(pHYnifB~hesA+pMWnifVSU) MSX
40
40
100
2.E.coli(pHY+pMW)MSX
40
40
100
Figure17;The result of indofenol method with reduced nitrogen source and with MSX.
The color of the solution changed into blue and the ammonium absorbance of the E.coli with pHYnifB~hesApMWnifVSU showed more than 2 times as high as the ammonium absorbance that of E.coli with pHY,PMW(empty two vectors).
So this result suspects that the transformant has the nitrogen fixing ability.
2.4.Determine the activity of glutamate dehydrogenase
Finally we approached the last step, glutamate synthesis. By using infusion cloning method, we succeeded in introducing Glutamate Dehydrogenase gene from Bacillus licheniformis into E.coli genome.  To measure the amount of glutamate produced by the cells, we used a glutamate assay kit called “Glutamine/Glutamate-Glo™  Assay”. Here we show the simple principle. The glutamate dehydrogenase (not from the transformant) in this kit produces NADH from glutamate, and that NADH is used to emit light by Luciferin. The fluorescence intensity of Luciferin increases in proportion to glutamate concentration, thus we can determine the amount of glutamate in the cell by measuring the fluorescence intensity.
Figure18;Reaction mechanism of glutamate assay
At first, we drew the standard curve, which shows the relationship between the concentration of glutamate and the fluorescence intensity of Luciferin. We planned to use this standard curve for measuring the quantity of glutamate in E.coli. But unfortunately, we didn’t have enough time to do the experiment.
Figure19;Standard curve of glutamate
3.Conclusion
As a conclusion, here we suggest what we could do and what we couldn’t in our experimental project this year. At first, what we’ve done is expressing nitrogenase in E.coli. Second, confirming the activity of nitrogenase. And last, we could express glutamate dehydrogenase gene in E.coli. What we couldn’t do is determining the activity of dehydrogenase gene and getting glutamate from E.coli. So we would like to do those things in the next opportunity.
Contact
Sponsor
1,Experiment plan
2,Results
3,Conclusion
