<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Botchan_Lab_Tokyo_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Botchan Lab Tokyo/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV id="header"><DIV id="menu_bar"><A href="https://2018.igem.org/Team:Botchan_Lab_Tokyo">HOME</A><DIV id="drop_down_menu"><BUTTON class="buttonbar"><SPAN style="border-bottom: 2px solid black">PROJECT</SPAN></BUTTON><DIV id="project_contents"><A href="https://2018.igem.org/Team:Botchan_Lab_Tokyo/Project">RESULTS</A><A href="https://2018.igem.org/Team:Botchan_Lab_Tokyo/Notebook">LABORATORY NOTEBOOK</A><A href="https://2018.igem.org/Team:Botchan_Lab_Tokyo/Model">MODELING</A></DIV></DIV><DIV id="drop_down_menu1"><BUTTON class="buttonbar1"><SPAN style="border-bottom: 2px solid black">LAB</SPAN></BUTTON><DIV id="lab_contents"><A href="https://2018.igem.org/Team:Botchan_Lab_Tokyo/Safety">SAFETY</A><A href="https://2018.igem.org/Team:Botchan_Lab_Tokyo/InterLab">INTERLAB</A></DIV></DIV><DIV id="drop_down_menu2"><BUTTON class="buttonbar2"><SPAN style="border-bottom: 2px solid black">PARTS</SPAN></BUTTON><DIV id="parts_contents"><A href="https://2018.igem.org/Team:Botchan_Lab_Tokyo/Parts_Composite parts">COMPOSITE PARTS</A><A href="https://2018.igem.org/Team:Botchan_Lab_Tokyo/Parts_notebook">PARTS BOOK</A></DIV></DIV><DIV id="drop_down_menu4"><BUTTON class="buttonbar4"><SPAN style="border-bottom: 2px solid black">HUMAN PRACTICE</SPAN></BUTTON><DIV id="humanpractice_contents"><A href="https://2018.igem.org/Team:Botchan_Lab_Tokyo/HP/Silver">SILVER</A><A href="https://2018.igem.org/Team:Botchan_Lab_Tokyo/Human_Practices">GOLD</A><A href="https://2018.igem.org/Team:Botchan_Lab_Tokyo/Collaborations">COLLABOLATIONS</A></DIV></DIV><DIV id="drop_down_menu5"><BUTTON class="buttonbar5"><SPAN style="border-bottom: 2px solid black">TEAM</SPAN></BUTTON><DIV id="team_contents"><A href="https://2018.igem.org/Team:Botchan_Lab_Tokyo/Team">MEMBER</A><A href="https://2018.igem.org/Team:Botchan_Lab_Tokyo/Attributions">ATTRIBUTION</A></DIV></DIV></DIV><DIV id="wrapper"><DIV id="main"><DIV id="main_content1"><H1>Projects Results</H1><H2 id="link1">1.Experiment plan</H2><H3>1.1.Overview</H3><P class="paragrah">This year, we tried to reproduce Last year's project at first.
   Last year, We tried to synthesize amino acids from nitrogen in the air inside <I>Escherichia coli</I>. By constructing that pathway in 
   <I>E.coli</I>, and by introducing the &quot;bacteria products&quot; to human intestine,we hope that it will be possible for people to &quot;eat&quot; nitrogen.</P><P class="paragraph">We divided the plan into following three specific pathways.</P><P class="paragraph"> The First pathway is expressing nitrogenase to create ammonium from nitrogen gas.
    To express nitrogenase adequately inside <I>E.coli</I>, we chose several <I>nif</I> gene groups and introduced them in <I>E.coli</I>.
     Although we hoped that synthesized ammonium would accumulate in a cell, the ammonium seemed to have been used in a metabolic 
     pathway  in <I>E.coli</I> itself.</P><P class="paragraph">Therefore, as the second pathway, we used an inhibitor called MSX to inhibit the glutamate synthesis pathway - which is one of the ammonium using anabolic pathways in <I>E.coli</I> - to make the ammonium accumulate in a cell.
</P><P class="paragraph">And finally, by introducing and expressing genes of amino acids dehydrogenase, synthesize amino acids from the ammonium.
     But this year we targeted on glutamic acid dehydrogenase because many kinds of amino acids are synthesized from glutamic acid.</P><P>Figure1:The pathway of getting amino acids from nitrogen in the air</P><P>Figure2:The inhibition of Glutamine synthesis pathway by using MSX.</P></DIV><DIV id="main_content3"><H3>1.2.Nitrogen-fixing reaction</H3><P>Figure3:nitrogenase's activity</P><P>Nitrogen is primary component of atmosphere, so there are much nitrogen around us.
    But most living organisms are not able to use nitrogen directly,
    because nitrogen exists as a less reactive molecular form.
    Nitrogenase has an ability to convert nitrogen into highly reactive substance&quot;ammonium&quot;.
Nitrogenase is expressed by the “<I>nif</I>” genes. The expression of nitrogenase is a burden to <I>E.coli</I>, so we had to reduce the burden as much as possible. Therefore, we decided to introduce the minimal gene cluster which is necessary for expressing nitrogenase into <I>E.coli</I>. 
We picked the minimal <I>nif</I> genes (<I>nifB,H,D,K,N,X,hesA</I>)  from the strain <I>Paenibacillus Polymyxa</I> ATCC 15970. And also, in addition to the genes which is essential for expressing this nitrogenase, we also introduced genes (<I>nifV,S,U</I>) from the strain <I>lahnella aquatillus</I> that have an ability to increase the activity of nitrogenase. We introduced these two <I>nif</I> gene groups into two vectors. The designs of the plasmids are mentioned in 1.4 .

    </P></DIV><DIV id="main_content4"><H3>1.3.Glutamate-synthesizing reaction</H3><P>Many kinds of amino acid are synthesised from Glutamic acid.
   So this year, we decided start with glutamic acid synthesis.The reaction is catalyzed by Glutamate dehydrogenase.
 
As shown above, ammonium is necessary to synthesize glutamic acid.We picked the glutamate dehydrogenase gene from the strain <I>Bacillus licheniformis</I>.</P></DIV><DIV id="main_content5"><H3>1.4.The design of our plasmid</H3><P>We used two vectors to construct plasmids because <I>nif</I>genes were too long. PHY plasmid contains <I>nif-B</I> to <I>hesA</I> genes. And pMW plasmid contains <I>nifV,S,U</I>genes and <I>glutamate dehydrogenase</I> gene(<I>glu DH</I>)..
</P><P>Figure4:pHY plasmid(<I>nifB~hesA</I>)</P><P>Figure5:pMW plasmid(<I>nifVSU+gluDH</I>)</P></DIV><DIV id="main_content6"><H2 id="link2">2.Result</H2><H3>2.1.Introduce nif-gene</H3><P>Figure6:Reconfirmation of the last year's samples</P><P><OL>
Sample
       <LI>JM109 (pHY);empty vector</LI><LI>JM109(pHY <I>nif</I><I>nifB,H,D,K,N,X,hesA</I></LI><LI>JM109(pMW);empty vector</LI><LI>JM109(pMW <I>nif</I>VSU</LI><LI>JM109(pHY+pMW);empty vectors</LI><LI>JM109(pHY <I>nifB,H,D,K,N,X,hesA</I>,pMW <I>nifVSU</I>)</LI>
       *1~5;control</OL></P><P>We found that transformant which we made last year had <I>nif</I><I>BHDKNXhesA</I> genes.
(sample2)But appeared that <I>nifVSU</I> genes were not inserted(sample3,4).
Besides, it's unclear why pHY plasmid band was unshown.</P><P>So we cloned <I>nifVSU</I> genes from <I>Lahnella Aquatillus</I>.</P><P>Figure7;PCR(<I>nif</I>VSU genes and pMW)

<UL><LI>1,2 pMW-Fw,Rv</LI><LI>3,4 <I>nif</I><I>VSU</I>-Fw,Rv for restriction enzyme</LI><LI>5,6 <I>nif</I><I>VSU</I>-Fw,Rv for infusion</LI>
Fw;Forward primer
Rv;Reverse primer
</UL></P><P>We did infusion cloning and made transformant which assumed to be had <I>nif</I><I>VSU</I> genes. Then,we did insert check by plasmid extraction from the cells.</P><P>Figure8;Insert check for pMW, <I>nifVSU</I> genes.</P><P>Sample
<OL><LI>pMW(extracted from July 12Sample1)</LI><LI>pMW-<I>nif</I><I>VSU</I>(extracted from July 12Sample2)</LI><LI>pMW(Digested by SacI)</LI><LI>pMW,<I>nif</I><I>VSU</I>(Digested by SacI)</LI><LI>pMW(Digested by XmaI)</LI><LI>pMW<I>nif</I><I>VSU</I>(Digested by XmaI)</LI></OL></P><P>We introduced two plasmids(pHY<I>nifB</I>~<I>hesA</I> and pMW<I>nifVSU</I> into one cell.</P><P>Figure9;Two vectors introducted in one sample</P><P>Sample</P><OL><LI>pMW<I>nif</I>VSU +pHY <I>nif</I><I>B,H,D,K,N,X</I>,<I>hes</I><I>A</I>(August 9)</LI><LI> pMW+pHY (August 9)</LI><LI> pHY</LI><LI>pHY <I>nif</I><I>B,H,D,K,N,X</I>,<I>hes</I><I>A</I></LI><LI>pMW</LI><LI>pMW<I>nif</I><I>VSU</I></LI></OL></DIV><DIV id="main_content7"><H3>2.2.Introduce dehydrogenase gene</H3><P>After that, we cloned glutamic acid dehydrogenase gene from <I>Bacillus licheniformis</I> and inserted it in the downstream of pMW <I>nifVSU</I> genes, introducing the plasmid into <I>E.coli.</I> JM109.</P></DIV><DIV id="main_content8"><H3>2.3.Determine the activity of nitrogenase</H3><P>We investigated the activity of nitrogenase by measuring the amount of synthesized ammonium by using the indofenol method. Indofenol is a blue compound and it is formed by the ammonium using reaction. If ammonium is exsisted in the solution, the color of the liquid changes into blue. The absorbance of solution (depth of blue) increases in proportion to ammonium concentration, thus we can determine the amount of ammonium in cells by measuring the absorbance. 
At first we drew the standard curve using ammonium solution. </P><P>Figure10;Standard curve(relation between the ammonium concentration and the absorbance of solution)</P><P>Next, we confirmed nitrogen fixation ability of the <I>Paenibacillus polymyxa</I> which has <I>nifB,H,D,K,N,X,hesA</I> genes. Before moving indofenol reaction, we replaced the air with nitrogen gas in the medium and incubated the samples for 24 or 48 hours. All indofenol samples were adjusted as well as these samples.</P><P>Calibration curve between ammonia concentration  and Abs.
Ammonia concentration and Abs is proportional.</P><P>We confirmed Nitrogen fixation ability of <I>Paenibacillus polymyxa</I>.</P><P>Figure11;Comparison of ammonium amount between <I>Paenibacillus</I>(diazotroph) and <I>E.coli</I>(non-diazotroph)
</P><P>This graph suggests <I>Paenibacillus Polymyxa</I> ATCC15970(diazotroph) produced more ammonium than that of <I>E.coli</I> JM109.</P><DIV class="keln_container"><SPAN class="project_exp"><H4>Indophenol sample</H4></SPAN><TABLE class="project_table" style="text-transform:none;"><TBODY><TR><TH>Sample Name</TH><TH>Phenol/EtOH(µl)</TH><TH>Sodium nitroprusside/H2O(µl)</TH><TH>Sodium hypochlorite(µl)</TH></TR><TR><TD>1.<I>E.coli</I>(pHY<I>nifB</I>~<I>hesA</I>)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>2.<I>E.coli</I>(pHY<I>nifB</I>~<I>hesA</I>+pMW<I>nifVSU</I>)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>3.<I>E.coli</I>(pHY)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>4.<I>E.coli</I>(pHY<I>nifB</I>~<I>hesA</I>+pMW)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>5.<I>E.coli</I>(pHY+pMW)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>6.<I>Paenibacillus polymyxa</I></TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>7.Medium only</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR></TBODY></TABLE></DIV><P>Figure12;The result of indofenol method(48h,72h)</P><P><I>Paenibacillus</I> showed the highest ammonium accumulation. The <I>E.coli</I> with pHY <I>nifB</I>~<I>hesA</I> genes and the <I>E.coli</I> with pHY <I>nifB</I>~<I>hesA</I>, pMW<I>nifVSU</I>)showed high ammonium accumulation. But LB medium sample(negative control) showed high Abs too. This result suspects the contamination in the medium.
</P><DIV class="keln_container"><SPAN class="project_exp"><H4>Indophenol(48 h)MSX</H4></SPAN><TABLE class="project_table"><TBODY><TR><TH>Sample Name</TH><TH>Phenol/EtOH(µl)</TH><TH>Sodium nitroprusside/H2O(µl)</TH><TH>Sodium hypochlorite(µl)</TH></TR><TR><TD>1.<I>E.coli</I>(pHY<I>nifB</I>~<I>hesA</I>+pMW<I>nifVSU</I>)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>2.<I>E.coli</I>(pHY+pMW)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>3.<I>Paenibacillus polymyxa</I></TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>4.Medium only</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR></TBODY></TABLE></DIV><P>Figure.13;The result of indofenol method with MSX(48h,72h)</P><P>All data except <I>Paenibacilllus</I> showed higher ammonium accumulation than that of samples without MSX, and pHY<I>nifB</I>~<I>hesA</I>pMW<I>nifVSU</I> showed the highest ammonium accumulation.</P><DIV class="keln_container"><SPAN class="project_exp"><H4>Indopheol(72h)</H4></SPAN><TABLE class="project_table"><TBODY><TR><TH>Sample Name</TH><TH>Phenol/EtOH(µl)</TH><TH>Sodium nitroprusside/H2O(µl)</TH><TH>Sodium hypochlorite(µl)</TH></TR><TR><TD>1.<I>E.coli</I>(pHY<I>nifB</I>~<I>hesA</I>)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>2.<I>E.coli</I>(pHY<I>nifB</I>~<I>hesA</I>+pMW<I>nifVSU</I>)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>3.<I>E.coli</I>(pHY)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>4.<I>E.coli</I>(pHY<I>nifB</I>~<I>hesA</I>+pMW)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>5.<I>E.coli</I>(pHY+pMW)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>6.<I>Paenibacillus polymyxa</I></TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>7.medium only</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR></TBODY></TABLE></DIV><P>Figure.14;The result of indofenol method(72h)</P><P><I>E.coli</I> with empty pHY vector showed high ammonium accumulation. It is different from the result of Figure12. Some factors expect ammonium might have affected the absorbance.</P><DIV class="keln_container"><SPAN class="project_exp"><H4>Indopheol(72h)MSX</H4></SPAN><TABLE class="project_table"><TBODY><TR><TH>Sample Name</TH><TH>Phenol/EtOH(µl)</TH><TH>Sodium nitroprusside/H2O(µl)</TH><TH>Sodium hypochlorite(µl)</TH></TR><TR><TD>1.<I>E.coli</I>(pHY<I>nifB</I>~<I>hesA</I>+pMW<I>nifVSU</I>) MSX</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>2.<I>E.coli</I>(pHY) MSX</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>3.<I>E.coli</I>(pHY<I>nifB</I>~<I>hesA</I>+pMW) MSX</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>4.<I>E.coli</I>(pHY+pMW) MSX</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>5.<I>Paenibacillus polymyxa</I> MSX</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR></TBODY></TABLE></DIV><P>Figure15;The result of indofenol method with MSX(72h)</P><P>All data except <I>Paenibacilllus</I> showed higher ammonia accumulation than that of samples without MSX, and the<I> E.coli</I> with pHY<I>nifB</I>~<I>hesA</I>pMW showed the highest ammonium accumulation and the <I>E.coli</I> with pHY<I>nifB</I>~<I>hesA</I>pMW<I>nifVSU</I> was the second. Still, the negative control samples(pHY and pHY,pMW) showed not a little absorbance.</P><P>To decrease the absorbance of negative controls, we tried reducing nitrogen source in the medium to 0.3 mM Glutamate (normally it's 2 mM Glutamate). And we excluded normal LB medium by centrifuging when we inoculate <I>E.coli</I> into Nitrogen deficient medium(0.3 mM Glutamate) for the indofenol measurement.</P><DIV class="keln_container"><SPAN class="project_exp"><H4>Indophenol(48h)</H4></SPAN><TABLE class="project_table"><TBODY><TR><TH>Sample Name</TH><TH>Phenol/EtOH(&amp;microl)</TH><TH>Sodium nitroprusside/H2O(&amp;microl)</TH><TH>Sodium hypochlorite(&amp;microl)</TH></TR><TR><TD>1.<I>E.coli</I>(pHY<I>nifB</I>~<I>hesA</I>)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>2.<I>E.coli</I>(pHY<I>nifB</I>~<I>hesA</I>+pMW<I>nifVSU</I>)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>3.<I>E.coli</I>(pHY)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>4.<I>E.coli</I>(pHY<I>nifB</I>~<I>hesA</I>+pMW)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>5.<I>E.coli</I>(pHY+pMW)</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>6.<I>Paenibacillus polymyxa</I></TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>7.Medium only</TD><TD>40</TD><TD>40</TD></TR></TBODY></TABLE></DIV><P>Figure16;The result of indofenol method with reduced nitrogen source in the medium(48h).</P><P>In this result, the ammonium accumulation was too low to measure. Even, we couldn't see the changing of color. All the samples were transparent.</P><P>That's probably because in the few presence of nitrogen source, the ammonium produced by nitrogenase was used in an anabolism of <I>E.coli</I> itself.

</P><DIV class="keln_container"><SPAN style="font-size:20px;"><SPAN class="project_exp"><H4>Indophenol(48h)MSX-1</H4></SPAN><TABLE class="project_table"><TBODY><TR><TH>Sample Name</TH><TH>Phenol/EtOH(&amp;microl)</TH><TH>Sodium nitroprusside/H2O(&amp;microl)</TH><TH>Sodium hypochlorite(&amp;microl)</TH></TR><TR><TD>1.<I>E.coli</I>(pHY<I>nifB</I>~<I>hesA</I>+pMW<I>nifVSU</I>) MSX</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR><TR><TD>2.<I>E.coli</I>(pHY+pMW)MSX</TD><TD>40</TD><TD>40</TD><TD>100</TD></TR></TBODY></TABLE></SPAN></DIV><P>Figure17;The result of indofenol method with reduced nitrogen source and with MSX.
</P><P>The color of the solution changed into blue and the ammonium absorbance of the <I>E.coli</I> with pHY<I>nifB</I>~<I>hesA</I>pMW<I>nifVSU</I>  showed more than 2 times as high as the ammonium absorbance that of <I>E.coli</I> with pHY,PMW(empty two vectors).
</P><P>So this result suspects that the transformant has the nitrogen fixing ability.</P><H3>2.4.Determine the activity of glutamate dehydrogenase</H3><P>Finally we approached the last step, glutamate synthesis. By using infusion cloning method, we succeeded in introducing Glutamate Dehydrogenase gene from <I>Bacillus licheniformis</I> into <I>E.coli</I> genome. 
To measure the amount of glutamate produced by the cells, we used a glutamate assay kit called “Glutamine/Glutamate-Glo™  Assay”.
Here we show the simple principle. The glutamate dehydrogenase (not from the transformant) in this kit produces NADH from glutamate, and that NADH is used to emit light by Luciferin. The fluorescence intensity of Luciferin increases in proportion to glutamate concentration, thus we can determine the amount of glutamate in the cell by measuring the fluorescence intensity. </P><P>Figure18;Reaction mechanism of glutamate assay</P><P>At first, we drew the standard curve, which shows the relationship between the concentration of glutamate and the fluorescence intensity of Luciferin. We planned to use this standard curve for measuring the quantity of glutamate in <I>E.coli</I>. But unfortunately, we didn’t have enough time to do the experiment. </P><P>Figure19;Standard curve of glutamate</P><H2 id="link3">3.Conclusion</H2><P>As a conclusion, here we suggest what we could do and what we couldn’t in our experimental project this year. At first, what we’ve done is expressing nitrogenase in <I>E.coli</I>. Second, confirming the activity of nitrogenase. And last, we could express glutamate dehydrogenase gene in <I>E.coli</I>. What we couldn’t do is determining the activity of dehydrogenase gene and getting glutamate from <I>E.coli</I>. So we would like to do those things in the next opportunity.



</P><DIV id="footer-elements"><DIV id="footer-right"><H1>Contact </H1></DIV><DIV id="footer-left"><H1>Sponsor</H1></DIV></DIV><DIV id="inpagelink"><A href="#link1">1,Experiment plan</A><A href="#link2">2,Results</A><A href="#link3">3,Conclusion</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>