Team:CityU HK/Protocol
HOME
PROJECT
OVERVIEW
EXPERIMENTS
RESULTS
BIOSAFETY
PARTS
NOTEBOOK
LAB LOG
PROTOCOL
ATTRIBUTIONS
TEAM
OUTREACH
COLLABORATION
HUMAN PRACTICE
INTERLAB
/************* sidebar style **************/
/* ****** Back to top button ******* */
Cloning
Characterization
PCR Amplification Protocol
Using TIANGEN 2× Pfu PCR MasterMix
 
PCR Reaction Mixture
 
Volume (μl) 
for 1 reaction
Master mix (μl) 
for 4.2 reactions
ddH2O
9.5
39.9
Pfu PCR MasterMix (2X)
12.5
52.5
F-primer (10 mM)
1
4.2
R-primer (10 mM)
1
4.2
Total
24
/
§   Add 1 μl of DNA template to 24 μl of master mix in each PCR tube.
Thermocycling Conditions
Step
Temperature (oC)
Time
Initial   denaturation
94
3 minutes
Denaturation
94
30 seconds
Annealing
 
55
30 seconds
Extension
(This process is   repeated for 32 cycles)
72
2 minutes*
 
Final extension
72
5 minutes
Hold
12
∞
*Depending on the size of amplicon (1,000 bp/minute) 
Purification of PCR Products Protocol
Using TIANGEN® TIANquick Midi Purification Kit (DP204)
 
[1]      Add 500 μl of Buffer BL to a CB2 spin column in a collection tube. Centrifuge for 1 minute at 12,000 rpm (~13,400 x g). Discard the flow-through.
[2]      Add 5 volumes of Buffer PB to 1 volume of nucleic acid solution in a 1.5 ml microcentrifuge tube. Mix gently.
[3]      Transfer the mixture to the spin column and incubate at room temperature for 2 minutes. Centrifuge for 1 minute at 12,000 rpm (~13,400 x g). Discard the flow-through.
[4]      Add 600 μl of Buffer PW to the spin column and centrifuge for 1 minute at 12,000 rpm (~13,400 x g). Discard the flow-through. 
[5]      Centrifuge the empty spin column for 2 min.
[6]      Place the spin column in a clean 1.5 ml microcentrifuge tube. Add 30 μl of Buffer EB or ddH2O (prewarmed at 60oC) to the center of the membrane. Incubate at room temperature for 2 minutes. Centrifuge at 12,000 rpm (~13,400 x g) for 2 minutes.
[7]      Repeat step 6 by using the flow-through (DNA) in the 1.5 ml microcentrifuge tube.
[8]      Store the DNA at 4oC or –20oC.
Plasmid Extraction Protocol
Using TIANGEN® TIANprep Mini Plasmid Kit (DP103)
 
[1]      Add 500 μl of Buffer BL to spin column CP3 with collection tube to activate the DNA-binding membrane. Centrifuge at 12,000 rpm (~13,400 x g) for 1 minute. Discard the flow-through.
[2]      Harvest a total of 3 ml of bacterial cells in a 1.5 ml microcentrifuge tube by centrifuging 1.5 ml bacterial cells at 12,000 rpm (~13,400 x g) for 1 minute and repeat.
[3]      Remove the supernatant. Resuspend the cell pellet in 250 μl Buffer P1 until no cell crumbs can be seen.
[4]      Add 250 μl of Buffer P2 and mix thoroughly by inverting the tube 6 to 8 times. (No vortexing)
[5]      Add 350 μl of Buffer P3 and immediately invert the tube 6 to 8 times. (No vortexing)
[6]      Centrifuge the tube at 12,000 rpm (~13,400 x g) for 10 minutes.
[7]      Apply the supernatant to the activated spin column and centrifuge for 1 minute at 12,000 rpm (~13,400 x g). Discard the flow-through.
[8]      Add 500 μl of Buffer PD to the spin column and centrifuge for 1 minute at 12,000 rpm (~13,400 x g). Discard the flow-through.
[9]      Add 600 μl of Buffer PW to the spin column and centrifuge for 1 minute at 12,000 rpm (~13,400 x g). Discard flow-through. Repeat.
[10]  Spin the empty spin column for 2 minutes.
[11]  Place the spin column in a clean 1.5 ml microcentrifuge tube. Add 50 μl of Buffer EB or ddH2O (prewarmed at 60oC) to the center of the membrane. Incubate at room temperature for 2 minutes. Centrifuge for 2 minutes at 12,000 rpm (~13,400 x g).
[12]  Repeat step 11 by using the flow-through (DNA) in the 1.5 ml microcentrifuge tube.
[13]  Store the DNA at 4oC or –20oC.
Purification of DNA from Gel Protocol
Using TIANGEN® TIANgel Maxi DNA Purification Kit (DP210)
 
[1]      Excise the desired DNA fragment from the agarose gel with a clean and sharp scalpel. Slice the gel piece into small pieces.
[2]      Transfer the pieces into a microcentrifuge tube.
[3]      Add 3 volumes of Buffer PN to 1 volume of gel pieces in a 1.5 ml microcentrifuge tube and mix well. Incubate the tubes in a 50oC waterbath until all gel pieces have completely dissolved.
[4]      During the incubation, add 500 μl of Buffer BL to a CA3 spin column. Centrifuge at 12,000 rpm (~13,400 x g) for 1 minute and discard the flow-through.
[5]      Apply the DNA mixture to the spin column (£ 800 μl) and incubate at room temperature for 2 minutes. Centrifuge 1 minute at 12,000 rpm (~13,400 x g). Discard the flow-through. Repeat to transfer the remaining DNA mixture if necessary.
[6]      Add 600 μl of Buffer PW to the spin column and centrifuge for 1 minute at 12,000 rpm (~13,400 x g). Discard the flow-through. Repeat.
[7]      Spin the empty spin column for 2 minutes.
[8]      Place the spin column in a clean 1.5 ml microcentrifuge tube. Add 30 μl of Buffer EB or ddH2O (warmed at 60oC) to the center of membrane. Incubate at room temperature for 2 minutes. Centrifuge for 2 minutes at 12,000 rpm (~13,400 x g).
[9]      Repeat step 8 by using the flow-through (DNA) in the 1.5 ml microcentrifuge tube.
[10]  Store the DNA at 4oC or –20oC.
Transformation Protocol
Promega®E. coli Competent Cells
 
[1]     Chill sterile 17 x 100 mm polypropylene culture tubes on ice.
[2]     Remove frozen competent cells from –70oC freezer and place them on ice until thawed.
[3]     Aliquot 50 μl of thawed competent cells to each chilled culture tube.
[4]     Add 2 to 5 μl of ligation mixture to the thawed competent cells. Mix gently by flicking the tube several times and return tubes to ice for 10 min.
[5]     Heat-shock the cells at exactly42oC for 45 seconds without shaking.
[6]     Immediately place tubes on ice for 2 minutes.
[7]     Add 900 μl of SOC medium to the cells and incubate with shaking (~ 230 rpm) at 37oC for 60 minutes.
[8]     Plate 200 μl of the cells onto LB agar plate containing the relevant antibiotic at an appropriate concentration*.
[9]     Incubate the agar plates overnight at 37oC.
Antibiotic
*Concentration (μg/ml)
Ampicillin (Amp)
100
Chloramphenicol (Cm)
34
Colony PCR (analysis) Protocol
Using TaKaRa Ex Taq® DNA Polymerase
 
Part I: Cell lysate (Template) preparation
[1]      Select well isolated colonies on LB plates for PCR amplification. Prepare a PCR tube for each colony.
[2]      Pick colony with a sterile pipette tip and resuspend colony in 5μl of sterile ddH2O.
[3]      Heat PCR tubes in the thermocycler at 98oC for 10 minutes to lyse the cells. And centrifuge the lysate at 14,000 rpm for 1 minute.
Part II: PCR
PCR Reaction Mixture
 
Volume of reaction mix (μl) for 1 reaction
Volume of reaction mix (μl) for 12 reactions
Ex Taq polymerase (5 units/μl)
0.075
0.9
Taq PCR buffer (10X)
1.5
18
dNTPs (10 mM)
1.2
14.4
VF2 primer (10 mM)
0.3
3.6
VR primer (10 mM)
0.3
3.6
ddH2O
10.625
127.5
Total
14
168
§   Mix 1 μl of DNA template (from step 3) with 14 μl of master mix in a PCR tube.
Thermocycling Conditions
Step
Temperature (oC)
Time
Initial   denaturation
98
2 minutes
Denaturation
98
30 seconds
Annealing
 
55
30 seconds
Extension
(This process is   repeated for 32 cycles)
72
 
2 minutes*
 
Final extension
72
1 minute
Hold
4
∞
*Depending on the size of amplicon (e.g. 2 minutes for 1 to 2 kb)
Genomic DNA Extraction Protocol
QIAGEN® DNeasy 96 Blood & Tissue Kit
 
[1]       Harvest cells by centrifugation at 5000 x g (7500 rpm) for 10 minutes. Discard supernatant.
[2]      Resuspend cell pellet in 180 µl Buffer ATL.
[3]      Add 20 µl proteinase K to the mixture. Mix thoroughly by vortex, and incubate in a 56°C water bath until the tissue has completely lysed. Vortex occasionally during incubation.
[4]      Add 4 µl RNase A (100 mg/ml), mix by vortexing and incubate at room temperature for 30 minutes.
[5]       Vortex for 15 seconds.
[6]       Add 200 µl of Buffer AL to the sample, and mix thoroughly using a vortex.
[7]       Add 200 µl of ethanol (96–100%), and mix again thoroughly by vortex.
[8]      Pipette the mixture from step 7 (including any precipitate) into the DNeasy Mini spin column placed in a 2 ml collection tube. Centrifuge at 6000 x g (8000 rpm) for 1 minute. Discard the flow-through and collection tube.
[9]       Place the DNeasy Mini spin column in a new 2 ml collection tube, add 500 µl of Buffer AW1, and centrifuge for 1 minute at 6000 x g (8000 rpm). Discard the flow-through and collection tube.
[10]   Repeat step 9 with centrifuge time 3 minutes at 20,000 x g (14,000 rpm) to dry the DNeasy membrane. Discard the flow-through and collection tube.
[11]   Place the DNeasy Mini spin column in a clean 1.5 ml microcentrifuge tube, and pipette 200 µl of Buffer AE directly onto the DNeasymembrane. Incubate at room temperature for 1 minute and then centrifuge for 1 min at 6000 x g (8000 rpm) to elute the DNA.
[12]    Repeat the elution in step 11 to recover more DNA.
Protein Extraction Protocol
BioVision® EZLysTM Bacterial Protein Extraction Reagent
 
[1]       Collect cells by centrifugation at 16,000g for 10 minutes in a pre-weighed centrifuge tube. Remove as much liquid as possible. Determine the net weight of the cell pellet.
[2]      Resuspend the pellet in at least 4 ml of EZLys reagent (pre-warmed to room temperature) per 1g of wet cell paste by pipetting and/or gentle vortex.
[3]      Incubate the cell suspension by shaking or slowly mixing for approximately 5 minutes.
[4]      Remove the insoluble cell debris by centrifugation at 16,000g for 20 minutes at 4oC. The supernatant containing the soluble proteins can now by analyzed or further purified by using a Ni-NTA column or stored at -20oC for future use.
SDS-PAGE Protocol
 
[1] Preparation of protein samples from bacterial cultures
A)    Transfer 100 μl of an overnight bacterial culture into 10mL LB medium and incubate at 37oC shaker for growth.
B)    Measure A600 of the bacterial culture at various time intervals.
C)    Pipette 1ml of LB medium into a cuvette as blank.
D)   Pipette 1ml of sample.
E)     Record the results.
F)     Centrifuge 500 ul of bacterial culture (OD600 = 0.8) and resuspend cell pellet in 25 uL of SDS loading buffer.
[2] Preparation of separating gel
A)     Set the casting frames (clamp two glass plates in the casting frames) on the casting stands.
B)      Prepare the gel solution as follow:
Component
Volume
H2O
3.93 ml
30% Acrylamide
3.96 ml
Buffer B
2.5 ml
10% APS
140 μl
TEMED
10 μl
 
C)      Swirl the solution gently but thoroughly.
D)     Pipette appropriate amount of separating gel solution into the gap between the glass plates.
E)      Fill in water to make the top of the separating gel be horizontal.
F)      Wait for 20-30 minutes to allow the gel to solidify.
[3] Preparation of stacking gel
A)    Discard the water.
B)    Prepare the gel solution as follows:
Component
Volume
H2O
2.975 ml
30% Acrylamide
0.67 ml
Buffer B
0.5 ml
10% APS
30 μl
TEMED
5 μl
 
C)    Pipette stacking gel into the gap until overflow.
D)   Insert the comb without trapping any air under the teeth.
E)     Wait for 20 to 30 minutes to allow the gel to solidify.
[4] Remove the comb when the gel has set.
[5] Take the glass plates out of the casting frame and set them in the cell buffer dam.
[6] Prepare the running buffer as follow:
Component
Volume
Tris base
30.28 g
Glycine
142.63 g
SDS
10 g
Milli-Q water
Up to 1 litre
 
[7] Prepare a 10-fold dilution of the running buffer.
[8] Add the running buffer into the inner chamber and keep pouring after overflow until the buffer surface reaches the required level in the outer chamber.
[9] Heat the samples in 95oC water for 5 minutes.
[10]                Load 20 μl of prepared samples into the wells.
[11]                Run the gel at 10 mA for stacking.
[12]                Switch to 20mA afterwards.
[13]                Staining:
A)     Prepare fixing solution:
Component
Volume
Methanol
250 ml
Acetic Acid
50 ml
Deionized water
Up to 500 ml
 
B)      Prepare coomassie blue staining solution:
Component
Volume
Methanol
50 ml
Acetic Acid
10 ml
Coomassie Brilliant Blue
0.05 ml
Deionized water
Up to 100 ml
 
C)      Prepare destaining solution:
Component
Volume
Methanol
100 ml
Acetic Acid
50 ml
Deionized water
Up to 500 ml
 
D)     Cut away the stacking gel.
E)      Immerse the gel in fixing solution for 10 minutes and agitate slowly on shaker.
F)      Discard the fixing solution and stain the gel with the coomassie blue staining solution for 1 hour.
G)     Discard the coomassie blue staining solution and wash the gel with distilled water to remove excessive dye.
H)     Destain the gel with destaining solution overnight.
I)        Remove the destaining solution and wash with distilled water.
Western Blot Protocol
 
[1]     Run an SDS-PAGE gel.
[2]     Prepare 1x Wet Transfer Buffer:
Component
Volume
Tris base (48   mM)
5.8 g
Glycine (39 mM)
2.9 g
Methanol
200 ml
SDS
0.375 g
Milli-Q Water
Up to 1L
 
[3]     Soften the filter paper, fiber pad and membrane with transfer buffer.
[4]     Immerse the gel into the transfer buffer for 15 minutes.
[5]     Place the fiber pad at the bottom of the gel holder cassette.
[6]     Place filter paper, gel, membrane, filter paper and fiber pad accordingly on the top.
[7]     Seal the gel holder cassette and put into the electrode module.
[8]     Place the electrode module with the blue cooling into the buffer tank.
[9]     Run the gel at 350 mA for 1-2 hours
[10]Prepare the blocking buffer
Component
Volume
10x PBS
5.8 g
Non-fat milk
2.5 g
Tween 20
0.05 ml
H2O
Up to 50 ml
 
[11]Immerse the membrane into the blocking buffer and shake gently at room temperature for 1 hour.
[12]Add 1:200 dilution primary antibody directly into the blocking buffer and incubate at room temperature for 2 hours.
[13]Rinse the membrane 3 times with 5 ml TBST for 10 minutes each time.
[14]Add 5 ml of blocking buffer to the membrane with 1:2000 secondary antibody and incubate for 2 hours.
[15]Rinse the membrane 3 times with 5 ml TBST for 15 minutes each time.
[16]Add WesternBrightTM ECL to the membrane for 2 minutes.
[17]Drain excess reagent.
ONPG Assay Protocol
 
[1]      Incubate the E. coli culture in 10 ml of LB broth overnight at 37oC with shaking.
[2]      Measure the OD600 of the overnight culture following a 10-fold dilution. 
(Dilute culture with LB broth and use LB broth as blank)
[3]       Transfer 1 ml of E. coli to a new glass test tube.
[4]       Add 1 ml of Z buffer to the tube.
[5]       Add 40 μl of chloroform and 20 μl of 0.1% SDS to the tube. Mix thoroughly.
[6]       Incubate the tube in a 37oC water bath for 10 minutes. Mix vigorously.
[7]       Add 0.4 ml of ONPG (4 mg/ml) to the tube and shake for a few times.
[8]       Incubate the tube in a 37oC water bath for 15 minutes.
[9]       Add 1 ml of 1 M Na2CO3 to the tube to stop the reaction.
[10]   Transfer 1 ml of supernatant from the tube to a cuvette.
[11]   Measure the OD420 of the supernatant. 
(Dilute with LB broth if necessary and use 0.4 ml ONPG + 2 ml Z buffer + 1 ml Na2CO3 as blank)
RNA Extraction Protocol
Using TaKaRa® MiniBEST Universal RNA Extraction Kit
 
[1]      Harvest 500 μl of bacterial cells in a 1.5 ml microcentrifuge tube by centrifugation at 12,000 rpm (~13,400 x g) for 1 minute at 4oC.
[2]      Remove the supernatant. Resuspend the cell pellet in 200 μl of freshly prepared lysozyme (0.5 mg/mL in Tris/EDTA Buffer) solution and ensure no cell clumps can be seen. Incubate at room temperature for 15 minutes.
[3]      Add 600 μl of Buffer RL and 12 μl of 50X DTT Solution. Mix thoroughly. Incubate the cell lysate at room temperature for 2 minutes.
[4]      Apply the cell lysate to a gDNA Eraser spin column with a 2 ml collection tube. Centrifuge at 12,000 rpm (~13,400 x g) for 1 minute.
[5]      Discard the gDNA Eraser Spin Column. Add an equal volume of 70% ethanol (812 μl) to the flow-through in the 2 ml collection tube. Mix thoroughly.
[6]      Transfer the mixture to an RNA spin column with 2 ml collection tube. (If the volume of the mixture is more than 600 μl, apply the mixture by dividing into several aliquots). Centrifuge at 12,000 rpm (~13,400 x g) for 1 minute. Discard the flow-through.
[7]      Add 500 μl Buffer RWA to the RNA spin column. Centrifuge at 12,000 (~13,400 x g) for 30 seconds. Discard the flow-through.
[8]      Add 600 μl Buffer RWB to the RNA spin column. Centrifuge at 12,000 (~13,400 x g) for 30 seconds. Discard the flow-through.
[9]      Prepare the DNase I mixture in a 1.5 ml microcentrifuge tube by adding 5 μl 10X DNase I Buffer, 4 μl Recombinant DNase and 41 μlRNase free H2O. Mix gently by inverting the tube.
[10]   Add 50 μl of the DNase I mixture to the center of membrane. Incubate at room temperature for 15 minutes.
[11]   Add 350 μl RWB to the RNA spin column. Centrifuge at 12,000 rpm (~13,400 x g) for 30 seconds. Discard the flow-through.
[12]   Repeat step 8.
[13]   Centrifuge the empty spin column at 12,000 rpm (~13,400 x g) for 2 minutes.
[14]   Place the RNA spin column in a new 1.5 ml RNase free collection tube. Add 30 μl of RNase-free water to the center of membrane. Incubate at room temperature for 5 minutes. Centrifuge for 2 minutes at 12,000 rpm (~13,400 x g).
[15]   Repeat step 14 by using the flow-through (RNA) in the 1.5 ml RNase free collection tube.
[16]   Store the RNA at -80oC.
Reverse-transcription PCR Protocol
Using TaKaRa PrimeScript™ RT reagent Kit with gDNA Eraser
 
Part I: Removal of genomic DNA in RNA samples
Digestion of genomic DNA
 
Volume (μl) 
for 1 reaction
Master mix (μl) 
for 8 reactions
gDNA Eraser Buffer (5X)
2.0
16
gDNA Eraser
1.0
8.0
Total
3.0
24
§   Add 1 µg of each total RNA template to 3 μl of master mix in each PCR tube
§   Add an appropriate volume of RNase-free sterile dH2O to yield a final volume of 10 μl in each PCR tube.
§   Place the tubes inside the thermocycler under 37oC for 2 minutes, followed by 85oC for 5 seconds.
Part II: Reverse-transcription reaction
Reverse-transcription reaction
 
Volume (μl) 
for 1 reaction
Master mix (μl) 
for 8 reactions
PrimeScript Buffer 2 (5X)
4.0
32
PrimeScript RT Enzyme Mix I
1.0
8.0
RT Primer Mix
1.0
8.0
RNase Free dH2O
4.0
32
Total
10.0
80.0
§   Add 10.0 μl of reaction solution from step 1 to 10.0 μl of master mix in each PCR tube.
§   Place the tubes inside the thermocycler under 37oC for 2 minutes, followed by 85oC for 5 seconds.
Real time PCR Protocol
Using Promega GoTaq® qPCR Master Mix
 
Part I: Standard Curve Construction
[1]      Prepare 4 concentrations of cDNA template by serially diluting a standard RT reaction product as follows:
Dilution
Recipe
A (1/5)
20 μl RT reaction product +   80 μl water
B (1/50)
10 μl Dilution A + 90 μl water
C (1/500)
10 μl Dilution B + 90 μl water
D (1/5000)
10 μl Dilution C + 90 μl water
 
[2]       Add 2 μl of each concentration (in triplicate) to appropriate wells on a 96-well PCR plate:
 
Triplicate
Conc.
1
2
3
A
○
○
○
B
○
○
○
C
○
○
○
D
○
○
○
 
[3]        Prepare a PCR master mix as follows:
 
Volume (μl) 
for 1 reaction
Master mix (μl) 
for 13 reactions
GoTaq® qPCR Master Mix (2X)
5.0
80
SYBP Green Supermix
0.1
1.3
10 μM Forward primer
0.18
2.34
10 μM Reverse primer
0.18
2.34
Water
2.54
33.02
Diluted cDNA
2.0
(pre-add to well)
Total
10
130
[4]       Mix a 8 μl PCR master mix with each cDNA aliquot by gently pipetting up and down several times
[5]       Seal the 96-well plate with an optical tape. Spin down all droplets by centrifugation at 3,000 rpm for 2 minutes.
Part II: Gene expression measurements
[1]       Dilute RT product 1 in 5 times as follows:
à20 μl RT reaction product + 80 μl water
If the expression of the target gene is low, prepare a less diluted sample of the RT-product for analysis.
[2]       Add 2 μl of the diluted RT product (in triplicate) to appropriate wells of a 96-well PCR plate.
[3]        Prepare a PCR master mix as follows:
 
Volume (μl) 
for 1 reaction
Master mix (μl) 
for 4 reactions
GoTaq® qPCR Master Mix (2X)
5.0
20
SYBP Green Supermix
0.1
0.4
10 μM Forward primer
0.18
0.72
10 μM Reverse primer
0.18
0.72
Water
2.54
10.16
Diluted cDNA
2.0
(pre-add to well)
Total
10
40
 
[4]       Mix 8 μl PCR master mix with each cDNA aliquot by gently pipetting up and down several times
[5]       Seal the 96-well plate with an optical tape. Spin down all droplets by centrifugation at 3,000 rpm for 2 minutes.
Characterization of “Lysis Gene” Cassette
(Overnight-culture protocol)
 
[1]     Pick a single bacterial colony from agar plate and inoculate the cells in liquid medium (of 5 or 10 ml of LB or MM containing 34 µg/ml chloramphenicol) and grow overnight at 37oC with shaking.
[2]     Measure OD600 of the culture with a spectrophotometer.
[3]     Transfer an appropriate volume of the overnight culture to 30 ml LB or MM medium (plus antibiotic) such that the final OD600 is approximately 0.05.
[4]     Prewarm 2 tubes to 37oC in a shaking water bath.
[5]     When the OD600 falls about 0.6, split the liquid culture into two aliquots; one lot represents the control and to the second lot (test), IPTG is added to a final concentration of 1 mM.
[6]     Incubate both tubes in a shaking water bath at 37oC.
[7]     OD600 and total count of both cultures were determined 10-minute intervals.
Characterization of “Lysis Gene” Cassette
(Without overnight growth on LB)
 
[1]     Pick a single bacterial colony from agar plate and inoculate the cells in liquid medium (of 20 to 25 ml of MM or LB containing 34 µg/ml of chloramphenicol).
[2]     Grow culture with shaking at 37oC until OD600reaches 0.1 to 0.2.
[3]     Label two new tubes; one represents the negative control and the second the test culture for lysis induction by IPTG.
[4]     Transfer 10 ml of the bacterial culture to each of the two tubes.
[5]     Add IPTG (to a final concentration ranging from 0.01 mM to 1 mM) to the test cultures.
[6]  Incubate cultures in a shaking water bath at 37oC. Measure OD600 and total counts of the bacterial cultures at 10-minute intervals.
Characterization of L8-UV5 Promoter (BBa_K1695000)
 
[1]     Pick a single bacterial colony from agar plate and inoculate the cells in liquid medium (of 5 or 10 ml of LB or MM containing 34 µg/ml chloramphenicol) and grow overnight at 37oC with shaking.
[2]     Measure OD600 of the culture with a spectrophotometer.
[3]     Transfer an appropriate volume of the overnight culture to LB or MM medium (plus antibiotic) such that the final OD600 is approximately 0.2.
[4]     Transfer 15 mL of the diluted culture to each of 8 tubes.
[5]     Add IPTG to a different final concentration (ranging from 0.01 mM to 10 mM) for each tube.
[6]     Incubate the tubes in a shaking water bath at 37oC.
[7]     The OD600 and fluorescence signal of all cultures were determined at 30-minute intervals.
