"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-CeBiTec/Notebook/Protocols
From 2014.igem.org
Journal
Protocols
Media
Strains & Constructs
Kits & Enzymes
Primer
Acronyms
Organisms
Additional
Home
Summary
Applications
Achievements
Bielefueli
Project
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Results
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Modeling
Parts
Data Page
Achievements
Team
Members
Subteams
Pictures & Videos
Bielefeld & University
Official Team Profile
Acknowledgements
Contact
Policy & Practices
Experts
Interviews
NRW-Day
Pupils Acedemy
How To Wiki
namu
SYNENERGENE
Applications
Vignettes
Notebook
Partners
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Partners
Protocols
Genetic engineering
Analytics
Cultivations
Genetic engineering
Blunt-End
  (Thermo Scientific)
Blunt-End
  (Thermo Scientific)
For cloning blunt-end PCR products generated by proofreading DNA polymerases, such as Pfu DNA polymerase. (If the DNA end structure of the PCR products is not specified by the supplier of the DNA polymerase, follow the Sticky-End Cloning Protocol).
For cloning of blunt-end DNA fragments generated by restriction enzyme digestion. Gel-purify the DNA fragment prior to ligation and use in a 3:1 molar ratio with pJET1.2/blunt (see Table 1).
1. Set up the ligation reaction on ice:
Component
Volume
2X Reaction Buffer
10 µl
Non-purified PCR product
or
purified PCR product/other blunt-end DNA fragment
1 µl
0.15 pmol ends
pJET1.2/blunt Cloning Vector (50 ng/μl)
1 μl (0.05 pmol ends)
Water, nuclease-free
up to 19 μl
T4 DNA Ligase
1 µl
Total volume
20 µl
Vortex briefly and centrifuge for 3-5 s.
2. Incubate the ligation mixture at room temperature (22°C) for 5 min.
Note. For PCR products >3 kb, ligation can be prolonged to 30 min. Ligation times longer than 30 min are not recommended and may decrease cloning efficiency.
3. Use the ligation mixture directly for transformation.
Note. Keep the ligation mixture at -20°C if transformation is postponed. Thaw on ice and mix carefully before transformation.
Colony PCR
Colony PCR
First alternative:
Pick one colony with a sterile tip and elute it in 100 µl ddH2O or medium
Store the colony at 4 °C while colony PCR is running
Second alternative:
Pick one colony with a sterile tip and streak cells at a marked position on a new plate
Put tip in PCR tube already containing the reaction mixture
For both alternatives continue as follows:
One reaction mix contains:
5 µl 5x buffer
1 µl MgCl2 (25 mM stock)
0.5 µl 10mM dNTPs
0.25 µl primer mix (prefix/suffix primers or sequencing primers)
17.625 µl ddH2O
0.125 µl GoTaq polymerase (Promega)
0.5 µl template
PCR program:
Cell lysis and initial denaturation: 5 min, 95 °C
30 cycles of:
10 s, 95 °C
30 s, annealing temperature
1 min / 1 kb of expected product, 72 °C
Final elongation: 5 min, 72 °C
Gel electrophoresis: check the fragment size
First (as above) alternative:
Plate the correct colony
Second (as above) alternative:
Use cells from the right positions to start liquid cultures or streak them on a new plate
DNA isolation (genomic DNA)
DNA isolation (genomic DNA)
The buffer and spin column for this DNA isolation were taken from the QIAprep® Spin Miniprep Kit
Resuspend the cells of the plate in 800 µl Buffer P1.
Transfer the suspension in small tubes with beads.
Use the ribolyser with 6200 rpm 3 x 60 seconds.
Centrifuge at top speed for 3 minutes.
Transfer 500 µl of the supernatant in a new 2 ml reaction tube.
Add 500 µl of Buffer P2 and mix thoroghly by inverting the tube 6-8 times.
Add 700 µl of Buffer N3 and mix thoroghly by inverting the tube 6-8 times.
Centrifuge at top speed for 10 minutes.
Apply the supernatant to the spin column by decanting or pipetting.
Centrifuge at top speed for 30-60 seconds and discard the flow-through.
Wash the spin column by adding 750 µl Buffer PE.
Centrifuge for 30-60 seconds and discard flow-through.
Wash again by adding 750 µl Buffer PE.
Centrifuge for 30-60 seconds and discard flow-through.
Centrifuge 2 minutes to remove residual wash buffer.
Place the column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 20 µl H
2O, incubate at room temperature for a few minutes and centrifuge at top speed for 1 minute. Repeat this step.
DNA Purification
DNA Purification by Centrifugation
DNA purification by centrifugation by Promega
First alternative:
Dissolving the Gel Slice
Following electrophoresis, excise DNA band from gel and place gel slice in a 1.5 ml                      microcentrifuge tube.
Add 10 µl Membrane Binding Solution per 10 mg of gel slice. Vortex and incubate at 50-65 °C until                     gel slice is completely dissolved.
Second alternative:
Processing PCR Amplifications
Add an equal volume of Membrane Binding Solution to the PCR amplification.
Binding of DNA
Insert SV Minicolumn into Collection Tube.
Transfer dissolved gel mixture or prepared PCR product to the Minicolumn assembly. Incubate at                      room temperature for 1 minute.
Centrifuge at 16,000 x g for 1 minute. Discard flowthrough and reinsert Minicolumn into                       Collection Tube.
Washing
Add 700 µl Membrane Wash Solution (ethanol added). Centrifuge at 16,000 x g for 1 minute. Discard                     flowthrough and reinsert Minicolumn into Collection Tube.
Repeat Step before with 500 µl Membrane Wash Solution. Centrifuge at 16,000 x g for 5 minutes.
Empty the Collection Tube and recentrifuge the column assembly for 1 minute with the                     microcentrifuge lid open (or off) to allow evaporation of any residual ethanol.
Elution
Carefully transfer Minicolumn to a clean 1.5 ml  microcentrifuge tube.
Add 15 µl of Nuclease-Free Water to the Minicolumn. Incubate at 60 °C for 5 minutes. Centrifuge at
16,000 x g for 1 minute. Repeat this step.
Discard Minicolumn and store DNA at 4 °C or -20 °C.
QIAquick Gel Extraction Kit by QIAGEN
Note: All centrifugation steps are carried out at 17,900 x g (13,000 rpm) in a conventional table-top microcentrifuge.
Excise the DNA fragment from the agarose gel with a clean, sharp scapel.
Wigh the gel slice in a colorless tube. Add 3 volumes Buffer QG to 1 volume gel (100 mg gel ~ 100 µl). The maximum amount of gel per spin column is 400 mg. For > 2% agarose gels, add 6 volumes Buffer QG.
Incubate at 50 °C for 10 min (or until the gel slice has completely dissolved). Vortex thee tube every 2-3 min to help dissolve the gel. After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture is orange or violet, add 10 µl 3 M sodium acetate, pH 5.0, and mix. The mixture turns yellow.
Add 1 volume isopropanol to the sample and mix.
Place a QIAquick spin colun in a provided 2 ml collection tube. To bind DNA, apply the sample to the QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAquick column back into the same tube. For sample volumes of >800 µl, load and spin again.
If the DNA will subsequently be used for sequencing, in vitro transcription, or microinjection, add 500 µl Buffer QG to the QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAqick column back into the same tube.
To wash, add 750 µl Buffer PE o QIAquick column and centrifuge for 1 mi. Discard flow-through and place the QIAquick column back into the same tube.
Note: If the DNA will be used for salt-sensitive applications (e.g., sequencing, blunt-ended ligation), let the column stand 2-5 min after addition of Buffer PE.
Centrifuge the QIAquick clumn in the provided 2 ml collection tube 1 min to remove residual wash buffer.
Place QIAquick column into a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 µl Buffer EB (10 mM TrisCl, pH 8.5) or water to the center of the QIAquick membrane and centrifuge the column for 1 min. For increased DNA concentration, add 30 µl Buffer EB to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min. After the addition of Buffer EB to the QIAqick membrane, increasing the incubation time up to 4 min can increase the yield of purified DNA.
If the purified DNA is to be analyzed on a gel, add 1 volume of Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel.
Generating chemocompetent cells
Generating chemocompetent cells
Material:
300 ml LB-Medium
~10 ml cooled TSS buffer
5 pre-cooled 50 ml Falcons
Protocol:
Inoculate 2x3 ml LB with bacterial stock; incubate over night at 37 °C and 200 rpm
Inoculate 1x250 ml LB with the over night cultures in 1 ml flask at 37 °C and 140 rpm
Incubate until OD600 0.4-0.6
Devide the culture into five cooled 50 ml Falcons and incubate on ice for 10 minutes
Onwards all steps at 4 °C
Centrifugate at 3000 rpm, 4 °C for 10 minutes
Discard supernatant
Pipette out remaining media
The volume of TSS to use is 10% of the culture volume that you spun down (we only used 1-2 ml). You may need to vortex gently to fully resuspend the culture, keep an eye out for small cell aggregates even after the pellet is completely off the wall.
Use 100 µl per aliquot and drop them in liquid nitrogen immediatly
Store at -80 °C
Generating electrocompetent cells
Generating electrocompetent cells
Material:
550 ml LB-Medium
1 l cooled bidest. H2O
50 ml cooled 10% glycerine
10 pre-cooled 50 ml Falcons
Protocol:
Inoculate 2x3 ml LB with bacterial stock; incubate over night at 37 °C and 200 rpm
Inoculate 2x250 ml LB with the over night cultures in 1-litre-flask at 37 °C and 140 rpm
Incubate until OD600 0.4-0.6
Cool the culture 15-30 minutes on ice
Onwards all steps at 4 °C
Divide the cultures into cooled 50 ml Falcons and centrifugate at 4000 rpm, 4 °C for 15 minutes, make  sure to slowly accelerate and deccelerate
Discard supernatant
Resuspend pellet in 5 ml cooled bidest H2O (and don't get frustrated while doing it, keep shaking gently)
Pool two suspensions each, add bidest H2O up to 50 ml and centrifugate again (see centrifugation above)
Discard supernatant
Resuspend pellet in 5 ml cooled bidest H2O
Add bidest H2O up to 50 ml and centrifugate again (see centrifugation above)
Discard supernatant
Resuspend pellet in 5 ml cooled 10% glycerine
Transfer suspensions in two 50 ml Falcons and centrifugate again (see centrifugation above)
Discard supernatant
Add volume of 10% glycerine that is approximately equal to the volume of the pellet and resuspend
Divide cells in 50 µl aliquots and freeze in liquid nitrogen immediately
Store at -80 °C
Gibson Assembly
Gibson Assembly
Modified from Gibson et al. (2009)
This assembly method is an isothermal, single-reaction method for assembling multiple overlapping DNA molecules. By coordinating the activity of a 5‘ exonuclease, a DNA polymerase and a DNA ligase two adjacent DNA fragments with complementary terminal sequence overlaps can be joined into a covalently sealed molecule, without the use of any restriction endonuclease.
Gibson-Assembly
Preparation of DNA molecules for in vitro recombination
Generate the complementary sequence overlaps by PCR using the Phusion DNA-polymerase . If necessary add 5 M Betain in the reaction mix by reducing the amount of H2O to decrease the number of false PCR products.
Identify the PCR products of interest by gel electrophoresis with known DNA standards.
Extract the PCR products from the gel by cutting out the DNA fragments and clean them up by using a commercial PCR clean-up kit .
in vitro recombination
Assembly mixture:
320 µl 5x isothermal reaction buffer
0.64 µl of 10 U ml-1 T5 exonuclease (for DNA molecules overlapping by greater than 150 bp add 3.2 µl of 10 U ml–1 T5 exonuclease)
20 µl of 2 U ml-1 Phusion DNA polymerase
160 µl of 40 U ml-1 taq DNA ligase
add ddH2O water up to a final volume of 1.2 ml
aliquote 15 µl of the reagent-enzyme mix and store it at –20 ˚C
Thaw 15 µl assembly mixture aliquot and keep it on ice until use.
Add 5 µl of the purified DNA molecules in equimolar amounts (between 10 and 100 ng of each DNA fragment).
Incubate the resulting mixture at 50 ˚C for 15 to 60 min, with 60 min being optimal.
Transformation ( via heat shock or via electroporation ) without cleaning up the assembly product.
NEB BioBrick Assembly Kit
  (NEB)
NEB BioBrick Assembly Kit
  (NEB)
Gibson-Assembly
We used the BioBrick Assembly Kit to assemble an upstream part with a downstream part into destination plasmid.
Digestion Protocoll:
Digest upstream part with EcoRI-HF and SpeI
Digest downstream part with XbaI and PstI
Digest destiantion destination plasmid with EcoRI-HF and PstI
500 ng part DNA
1 μl of each enzyme
5 μl 10x NEBuffer 2.1
to 50 μl H20
Incubate all three restriction digest reactions at 37 °C for 10 minutes and then heat inactivate at 80 °C for 20 minutes.
Dephosphorylation Protocoll
Add 1 μl of AP (Antarctic phosphatase) and 5 µl of 10x AP reaction buffer to digested destination plasmid, incubate for 1 h at 37 °C.
Ligation Protocoll
Ligate the upstream and downstream parts into the digested destination plasmid.
2 μl of each part
2 μl 10x T4 DNA Ligase Buffer
1 μl T4 DNA Ligase
11 μl H2O
Incubate at room temperature for 10 minutes and then heat inactivate at 80 °C for 20 minutes.
Transform 1-2 μl of the ligation product into 50 μl of competent E. coli cells.
PCR
PCR
Phusion (our standard) master mix (25 µl):
5 µl 5 x HF-buffer
0.5 µl 10mM dNTPs
Up to 20 ng * µl-1 template
0.25 µl polymerase
0.25-0.5 µl primer 10 µM (each one)
0.75 µl DMSO (optional)
Fill up to 25 µl with H2O
Q5 master mix (25 µl):
5 µl 5 x buffer
0.5 µl 10 mM dNTPs
Up to 50 ng * µl-1 template
0.1 µl polymerase
0.5-1 µl primer 10 µM (each one)
Fill up to 25 µl with H2O
KOD master mix (50 µl):
5 µl 10 x buffer for KOD Hot Start DNA Polymerase
3 µl 25 mM MgSO4
1 µl 10mM dNTPs
1 µl polymerase
1.5 µl primer 10 µM) (each one)
10 ng * µl-1 template
Fill up to 50 µl with H2O
Plasmid Isolation
Plasmid isolation
Wizard® Plus SV Minipreps DNA Purification System by Promega (Our standard kit)
Production of cleared lysate:
Pellet 1-10 ml of overnight culture for 5 minutes
Thoroughly resuspend pellet with 250 µl of Cell Resuspension Solution
Add 250 µl of Cell Lysis Solution to each sample; invert 4 times to mix
Add 10 µl of Alkaline Protease Solution; invert 4 times to mix. Incubate 5 minutes at room temperature
Add 350 µl of Neutralization Solution; invert 4 times to mix
Centrifuge at top speed for 10 minutes at room temperature
Binding of plasmid DNA
Insert Spin Column into Collection Tube
Decant cleared lysate into Spin Column
Centrifuge at top speed for 1 minute at room temperature. Discard flowthrough, and reinsert Column into Collection Tube.
Washing
Add 750 µl of Wash Solution (ethanol added). Centrifuge at top speed for 1 minute. Discard flowthrough and reinsert column into Collection Tube
Repeat step before with 250 µl of Wash solution
Centrifuge at top speed for 2 minutes at room temperature
Elution
Transfer Spin Column to a sterile 1.5 ml microcentrifuge tube, being careful not to transfer any of the Column Wash Solution with the Spin Column. If the Spin Column has Column Wash Solution associated with it, centrifuge again for 1 minute at top speed, then transfer the Spin Column to a new, sterile 1.5 ml microcentrifuge tube
Add 100 µl of Nuclease-Free Water to the Spin Column. Centrifuge at top speed for 1 minute at room temperature
Discard column, and store DNA at -20 °C or below
GeneJET Plasmid Miniprep Kit by Thermo Scientific
Note:
All purification steps should be carried out at room temperature.
All centrifugation should be carried out in a table-top microcentrifuge at >12000 x g
(10 000-14                     000 rpm, depending on the rotor type).
Resuspend the pelleted cells in 250 µl of the Resuspension Solution. Transfer the cell suspension to a                 microcentrifuge tube. The bacteria should be resuspended completely by vortexing or pipetting up and                down until no cell clumps remain.
Note Ensure RNase A has been added to the Resuspension Solution.
Add 250 µl of the Lysis Solution and mix thoroughly by inverting the tube 4-6 times until the solution                 becomes viscous and slightly clear.
Note Do not vortex to avoid shearing of chromosomal DNA. Do not incubate for more than 5 minutes to avoid denaturation of supercoiled plasmid DNA.
Add 350 µl of the Neutralization Solution and mix immediately and thoroughly by inverting the tube 4-6                 times.
Note It is important to mix thoroughly and gently after the addition of the Neutralization                     Solution to avoid localized  precipitation of bacterial cell debris. The neutralized bacterial                      lysate should become cloudly.
Centrifuge for 5 minutes to pellet cell debris and chromosomal DNA.
Transfer the supernatant to the supplied GeneJET spin coloumn by decanting or pipetting. Avoid                 disburbing or transferring the white  precipitate.
Centrifuge for 1 minute. Discard the flow-through and place the coloumn back into the same collection                tube.
Note Do not add bleach to the flow-through.
Add 500 µl of the Wash Solution (diluted with ethanol prior to first use) to the GeneJET spin column.                 Centrifuge for 30-60 seconds and discard the flow-through. Place the column back into the same                 collection tube.
Repeat the wash procedure (step before) using 500 µl of the Wash Solution.
Discard the flow-through and centrifuge for an additional  1 minute to remove residual Wash Solution.                This step is essential to avoid residual ethanol in plasmid preps.
Transfer the GeneJET spin column into a fresh 1.5 ml microcentrifuge tube (not included). Add 50 µl of                the Elution Buffer to the center of GeneJET spin column membrane to elute the plasmide DNA. Take care                not to contact the membrane with the pipette tip. Incubate for 2 minutes at room temperature and                 centrifuge for 2 minutes.
Note An additional elution step (optional) with Elution Buffer or water will recover                      residual DNA from the membrane and increase the overall yield by 10-20%. For elution of plasmids                     or cosmids >20 kb, prewarm Elution Buffer to 70 °C before applying to silica membrane.
Discard the column and store the purified plasmid DNA at -20 °C.
QIAprep® Spin Miniprep Kit by QIAGEN
Note: All centrifugation steps are carried out at 13,000 rpm (~17,900 x g) in a conventional able-top microcentrifuge.
Pellet 1-5 ml bacterial overnight culture by centrifugation at >8000 rpm (6800 x g) for 3 min at room temperature (15-25 °C).
Resuspend pelleted bacterial cells in 250 µl Buffer P1 and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and mix thoroughly by inverting the tube 4-6 times until the solution becomes clear. Do not allow the lysis reaction to proceed for more than 5 min. If using LyseBlue reagent, the solution will turn blue.
Add 350 µl Buffer N3 and mix immediately and thoroughly by inverting the tube 4-6 times. If using LyseBlue reagent, the solution will turn colorless.
Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge.
Apply the supernatant from step 5 to the QIprep spin column by decanting or pipetting. Centrifuge for 30-60 s and discard the flow-through, or apply vacuum to the manifold to draw the solution through the QIAprep spin column and switch off the vacuum source.
Recommended: Wash the QIAprep spin column by adding 500 µl Buffer PB. Centrifuge for 30-60 s and discard the flow-through, or apply vacuum to the manifold to draw the solution through the QIprep spin column and switch off the vacuum source.
Wash the QIAprep spin column by adding 750 µl Buffer PE. Centrifuge for 30-60s and discard the flow-through, or apply vacuum to the manifold to draw the solution through the QIAprep spin column and switch off the vacuum source. Transfer the QIAprep spin column to the collection tube.
Centrifuge for 1 min to remove residual wash buffer
Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 µl Buffer EB (10 mM Tris*Cl, pH 8.5) or water to the center of the QIAprep spin column. Let it stand for 1 min, and centrifuge for 1 min.
Standard BioBrick Assembly
Standard BioBrick Assembly
modified from silver lab
This assembly method can be used for BioBricks which are bigger than 150 bp. The BioBrick should be at least 500 bp bigger or smaller than the backbone. The BioBrick, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other used BioBrick, called vector. So you have to differentiate between a prefix and a suffix insertion.
Silver Suffix Insertion
Silver Prefix Insertion
Suffix insertion:
Digestion of insert: at least 700 ng DNA / 10 µl volume, 1 µl 10x NEBuffer 2.1, 0.5 µl XbaI , 0.5 µl PstI . Digest for 1 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert.
Digestion of vector about 700 ng DNA / 10 µl volume, 1 µl 10x NEBuffer 2.1, 0.5 µl SpeI , 0.5 µl PstI . Digest for 1 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µl AP (Antarctic phosphatase) and 1.2 µl 10 x AP reaction buffer, incubate for 1 h at 37 °C. Clean up the vector with a PCR clean-up kit .
Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µl ligation volume, 2 µl T4-Ligase-Buffer, 1 µl T4-Ligase. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
Prefix insertion:
Digestion of insert: at least 700 ng DNA / 10 µl volume, 1 µl 10x NEBuffer 2.1, 0.5 µl EcoRI , 0.5 µl SpeI . Digest for 1 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert.
Digestion of vector about 700 ng DNA / 10 µl volume, 1 µl 10 x NEBuffer 2.1, 0.5 µl EcoRI , 0.5 µl XbaI . Digest for 1h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µl AP (Antarctic phosphatase) and 1.2 µl 10 x AP reaction buffer, incubate for 1 h at 37 °C. Clean up the vector with a PCR clean-up kit .
Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µl ligation volume, 2 µl T4-Ligase-Buffer, 1 µl T4-Ligase. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
Variations:
A digestion over night is possible. If you digest over night use only 0.1 µl restriction enzyme.
It is also possible to use PCR product as insert. Digest after PCR with corresponding restriction enzymes and clean up with a PCR clean-up kit . This could lead to higher yields of insert DNA because a lot of DNA gets lost during the gel electrophoresis clean up.
Sometimes some BioBricks are hard to assemble. Then you have to clean up the vector by gel electrophoresis as well.
Transformation via electroporation
Transformation via electroporation
Thaw 50 µl electrocompetent E. coli cells on ice, dilute with icecold 50 µl glycerine (10%) if necessary
Add 0.5-5 µl plasmid to 50 µl electrocompetent cells
Store cells on ice for 1 minute
Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ω
Transfer transformation reaction to 450 µl SOC-Medium and incubate 1 h at 37 °C
Centrifuge 3 minutes at 1200 rpm and plate on selective LB-Medium
Incubate over night at 37 °C
Transformation via heat shock
Transformation via heat shock
Thaw 100 µl chemocompetent E. coli cells on ice
Add 0.5-5 µl plasmid to 100 µl chemocompetent cells
Store cells on ice for 10-30 minutes
Heat shock for 90 seconds at 42 °C
Transfer transformation reaction to 450 µl SOC-Medium and incubate 1 h at 37 °C
Centrifuge 3 minutes at 1200 rpm and plate on selective LB-Medium
Incubate over night at 37 °C
Analytics
Amicon Ultra - Protein buffer exchange and desalting
Amicon Ultra - Protein buffer exchange and desalting
Load up to 15 ml of purified enzyme on to the Amicon Ultra filter device
Place capped filter device into centrifuge rotor; counterbalance with a similar device
When using a swinging bucket rotor, spin the device at 4,000xg maximum for approximately 15-60 minutes
When using a fixed angle rotor, orient the device with the membrane panel facing up and spin at 5,000xg maximum for approximately 15-60 minutes
To recover the the concentrated solute, insert a pipettor into the bottom of the filter device and withdraw the sample using a side-to-side sweeping motion to ensure total recovery. The ultrafiltrate can be stored in the centrifuge tube.
For buffer exchange load up to 15 ml of new buffer on to the Amicon Ultra filter device and centrifuge for 15-60 minutes
Cold Osmotic Shock
Cold Osmotic Shock
Release of periplasmic protein fraction from E. coli by cold osmotic shock
Modified protocol from Neu & Heppel, 1965 :
Centrifuge E. coli cell suspension for 5 min at 14,000 g (4 °C) to collect the cells
Discard the entire supernatant
Resuspend the cells in ice-cold Cell Fractioning Buffer 1 . The resulting volume should be 1/4 of the former suspension volume
Incubate for 20 min on ice. Invert the suspension at regular intervals to counteract sedimentation
Centrifuge the cell suspension for 15 min at 14,000 g (4 °C)
Discard the entire supernatant
Resuspend the cells in ice-cold Cell Fractioning Buffer 2 . The resulting volume should be 1/4 of the former suspension volume
Incubate for 10 up to 20 min on ice under regular invertion
Centrifuge the cell suspension for 15 min at 14,000 g (4 °C)
Save the supernatant, which contains the periplasmatic proteins and membrane proteins
Fast Cell Lysis for SDS-PAGE
Fast Cell Lysis for SDS-PAGE
Measure OD600
Transfer two samples a 1,5 ml in a microcentrifuge tube
Centrifuge at top speed for 5 minutes at room temperature
Discard supernatant quantitative
Store pellet at -20 °C
Thaw pellet and resupend in Fast Cell Lysis sample buffer. (30 µl Fast Cell Lysis sample buffer. per OD600)
Heat for 5 - 10 minutes at 95 - 100 °C
Centrifuge at top speed for 15 minutes at room temperature
Transfer supernatant to a new microcentrifuge tube
Analyze samples by SDS-PAGE. Use 3 - 10 µl per sample
Gas-Chromatography (GC) - Mass Spectroscopy (MS)
Gas-Chromatography (GC) - Mass Spectroscopy (MS)
Modified from Kondo et al. (2012)
Sample preparation
Supernatant from cultivation (100 µl) was extracted with 900 µl of GC standard-grade acetone containing 0.1 % 2-butanol as an internal standard
150 µl were used for GC-MS analysis
GC-MS analysis was realized with the following settings
AS 2000 (Autosampler)
sample volume
1 µl
Trace GC 2000
column
Stabilwax® (Crossbond Carbowax - PEG)
(30 m, 0.25 mm internal diameter, 0.25 µm film thickness)
Carries Gas
Helium (1 ml / min)
Oven temperature
rate [°C]
temp [°C]
hold time [minutes]
initial
60
3.00
ramp 1
10.0
100
0.00
ramp 2
120.0
280
2.00
Injector temperature250°C Split flow20 ml / min MS transfer line250°C
Polaris Q (Ion trap MS)
Ion source
220°C
Mass range
25-100
start time
2,5 min
With these settings we generated a calibration line with the following concentrations for further analysis and quantification of our samples.
0,5 % isobutanol
0,1 % isobutanol
0,05 % isobutanol
0,01 % isobutanol
0,001 % isobutanol
His Trap FF Purification
His Trap FF Purification
Column preparation
1 h Incubation with 1 M NaOH
Wash with 10 ml Binding Buffer
Wash with 10 ml distilled water
Strip the column with 10 ml Stripping Buffer
Wash with 10 ml Binding Buffer
Wash with 10 ml distilled water
Recharge with 2.5 ml 0.1 M NiSO4
Wash with 5 ml distilled water
Wash with 5ml Binding Buffer
Perform a blank run:
Wash with 5 ml distilled water (After Ethanol storage)
Wash with 5 ml Binding Buffer
Wash with 5 ml Elution Buffer 5
Equilibrate with 10 ml Binding Buffer
Store in 20% ethanol
Sample preparation:
Add 5 ml Binding Buffer to each gram of pellet
Add 0.2 mg/ml lysozyme, 20 µg/ml DNAse, 1 mM MgCl2, 1 mM PMSF
Stir for 30 min at +4 °C up to +20 °C (depending on the protein)
Centrifuge 30 min at 4 °C
Adjust pH to 7.4 - 7-6
Apply the lysate on the column immediatly
Sample run:
Wash with 5 ml distilled water
Equilibrate with 5 ml Binding Buffer
Apply the lysate on the column
Wash with Binding Buffer until the absorbance reaches a steady baseline (10 ml)
Elute with 5 ml Elution Buffer gradient (Elution Buffer 1 – 5)
High performance liquid chromatography (HPLC)
High performance liquid chromatography (HPLC)
The HPLC was used for substrate analytics. We were able to detect glucose, xylose, succinate and fumarate in the media.
Sample preparation:
Centrifuge 1.5 ml of bacteria culture for 5 minutes at 14.000 rpm
Transfer supernatant to a fresh tube. Avoid to resuspend the cells
Add 2 µl of sodium azide (100g/ L) to each sample.
Store the samples at -20 °C untill you analyse them by HPLC
HPLC measurement:
750 µl of each sample are transferred into HPLC vials with a septum
The samples are determined in duplicates together with two sets standards á three concentrations
The concentration of the standards depends on the maximum expected carbon source concentration in                      the media
Sample run:
It was used the VA 300/7.7 NUCLEOGEL SUGAR 810 H column for sample separation
Eluent was 3.5 mM H2SO4
The column was heated at 65 °C and the injected sample volume was 20 µl
Detection was achieved by conductivity measurement
For performing enzyme assays with purified enzymes or crude cell extracts we were using following HPLC-MC system.
HPLC: LaChromUltra (Hitachi Europe, United Kingdom)
Analysis: microTOF-Q hybrid quadrupole / time of flight mass spectrometer (Bruker Daltonics, Bremen, Germany)
Ionization: Electrospray ionization (ESI)
Column: SeQuant ZICpHILLIC column (150 by 2.1 mm, Merck KGaA, Darmstadt, Germany)
Solutions: 10 mM Ammonium bicarbonate solution (eluent A), acetonitrile (eluent B)
Injection volume: 2 µl
Flow rate: 150 µl min-1
Gradient [tmin, % of eluent B]: t0: 80%, t30: 10%, t35: 10%, t40: 80%, t60: 80%
Negative ionization mode
Internal mass calibration: Formate (0.1 M), in 50% (vol/vol) isopropanol
Matrix-assisted Laser Desorption/Ionization – Time of flight (MALDI-TOF)
Matrix-assisted Laser Desorption/Ionization – Time of flight (MALDI-TOF)
Tryptic digest of gel lanes for analysis with MALDI-TOF:
Make sure to work under a fume hood
Do not work with protective gloves to prevent contamination of your sample with plasticizers
Reaction tubes have to be cleaned with 60 % (v/v) CH3CN and 0.1 % (v/v) TFA. Afterwards the solution has to be removed completely followed by evaporation of the tubes under a fume hood. Alternatively microtiter plates from Greiner® (REF 650161) can be used without washing
Cut out the protein lanes of a Coomassie-stained SDS-PAGE using a clean scalpel. Gel parts are transferred to the washed reaction tubes. If necessary cut the parts to smaller slices
Gel slices should be washed two times. Therefore add 200 µl 30 % (v/v) acetonitrile in 0,1 M ammonium hydrogen carbonate each time and shake lightly for 10 minutes. Remove supernatant and discard to special waste
Dry gel slices at least 30 minutes in a Speedvac
Rehydrate gel slices in 15 µl Trypsin-solution followed by short centrifugation
Trypsin-solution: 1 µl Trypsin + 14 µl 10 mM NH4HCO3
For this solution solubilize lyophilized Trypsin in 200 µl of provided buffer and activate Trypsin for 15 minutes at 30 °C. For further use it can be stored at -20 °C
Gel slices have to be incubated 30 minutes at room temperature, followed by incubation at 37 °C over night
Dry gel slices at least 60 minutes in a Speedvac
According to the size of the gel slice, add 5 - 20 µl 50 % (v/v) ACN / 0,1 % (v/v) TFA
Samples can be used for MALDI measurement or stored at -20 °C
Preparation and Spotting for analysis of peptides on Bruker AnchorChips:
Spot 0,5 - 1 µl of sample aliquot
Add 1 µl HCCA matrix solution to the spotted sample aliquots. Pipet up and down approximately five times to obtain a sufficient mixing. Be careful not to contact the AnchorChip. Note: Most of the sample solvent needs to be gone in order to achieve a sufficiently low water content. When the matrix solution is added to the previously spotted sample aliquot at a too high water content in the mixture, it will result in undesired crystallization of the matrix outside the anchor spot area
Dry the prepared spots at room temperature
Spot external calibrants on the adjacent calibrant spot positions. Use the calibrant stock solution (Bruker’s “Peptide Calibration Standard II”, Part number #222570), add 125 µl of 0.1 % TFA (v/v) in 30 % ACN to the vial. Vortex and sonicate the vial
Mix the calibrant stock solution in a 1:200 ratio with HCCA matrix and deposit 1 µl of the mixture onto the calibrant spots
Analyze samples in ultrafleXtreme by Bruker Daltonics
Microscopy
Microscopy
For the fluorescence microscopy of our GFP targeted carboxysome  we used two different miscroscope, a Leica DMI 6000 SD and a structured illumination microscope (DeltaVision|OMXv4.0 BLAZE, GE Healthcare). The preparation of the samples were treated identically.
Take 1 ml of the culture (you can choose different time points).
Centrifuge at top speed for 5 min.
Discard supernatant and resuspend in 600 µl 1x PSB buffer .
Prepare your object plates with 1% agarose for a fixation of the cells.
Take 5-10 µl of the washed culture for the microscopy (maybe dilute it with PBS buffer before using).
NPN-Assay
NPN-Assay
NPN membrane permeability assay
inoculate 20 ml of an overnight culture with 1 ml preculture
incubate cells at 37 °C until OD600 of 0.5
centrifugation of whole culture volume (10 min at 3000 g)
discard entire supernatant
resuspend pellet in the same volume PBS Buffer
repeat washing steps 3 to 5 twice
mix washed cells with 10 mM NPN stock solution to a final concentration of 10 μM in a UV cuvette
fluorescence measurement with Shimadzu fluorescence spectrophotometer
sample volume: 1 ml
Excitation: 355 nm
Emission Bandwidth: 380 - 500 nm
Promega NAD/NADH-Glo™ Assay
  (Promega)
Promega NAD/NADH-Glo™ Assay
  (Promega)
Basic principle of the enzymatically detection assay:
NAD/NADH-Glo™ Assay
We used the NAD/NADH-Glo™ Assay for the determination of the NAD+ and NADH levels in the cells during the cultivation in different growth phases.
Protocol:
The 1st step, before the assay can be performed is the lysis of the cells.
1 ml of bacteria culture (OD600= 0,375 ≈ 3 • 103 cells)get                       pelleted by centrifugation
Remove supernatant and add 300 µl bicarbonatebuffer + 1% DTAB                      (dodecyl(trimethyl)azanium bromide)
Mix thoroughly for cell lysis
Assay the neutralized samples using the NAD/NADH-Glo Assay by transferring 30 µl of each                     sample to the wells of a 96-well white luminometer plate and add 30µl of NAD/NADH-Glo                     Detection Reagent.
Incubate at room temperature and read luminescence after 30 to 60 minutes.
Determine NAD/NADH ratios by comparing RLU, or calculate the concentrations by comparison to a                       standard curve.
This method is used, if you want to measure the total amount of NAD+ and NADH. For the determination of the individual NAD+ and NADH concentrations follow the description below.
Individual determination of NAD+ and NADH
After the cells got mixed briefly transfer two 100µl  aliquots to each of two new tubes.(R1 and R2)
Add 100µl HCl (0.4M) to R1 for acid treatment
Heat both aliquots at 60 °C for 15 minutes
Cool at room temperature for 10 minutes
Neutralize R1 with 100 µl trizma base (0.5M). This sample now contains the oxidized form NAD.
Neutralize R2 with 200µl of a 1:1 mixture of HCl (0.5M) and trizma base (0.5M).  This sample now contains the oreduced form NADH
Both samples can be assayed as already described.
This protocol is taken from "Bioluminescent Nicotinamide Adenine Dinucleotide Detection in Bacteria" of the Promega corporation. All measurements were carried out with the GloMax® Discover Multimode-Reader .
RuBisCo activity assay
RuBisCO activity assay
Cultivation and cell lysis
Grow overnight culture of your cells containing BBa_K1465213 or the construct T7_Hneap RuBisCo in LB medium containing specific antibiotic. Grow at 37 °C
Dilute overnight culture 1:50 in LB medium containing antibiotic and grow cells at 37 °C
When the culture reaches an OD600 of 0.6-0.8, shift the temperature to 20 °C
Induce protein expression using the T7 promotor by adding rhamnose to a final concentration of 0.1 %. When using the ptac promotor, induction concentration is 0,5 mM.
Grow the culture at 20 °C overnight
Harvest cells by centrifugation at 4 °C
Discard supernatant
Add 5 ml 20 mM sodium phosphate, 500 mM NaCl buffer to each gram of cell pellet
Add 0.2 mg/ml lysozyme, 20 µg/ml DNAse, 1 mM MgCl2, 1 mM PMSF
Stir for 30 min at +4 °C
Centrifuge 30 min at 4 °C
Transfer supernatant (cell extract) to a new tube
RuBisCo activity assay
For the assay, make a reaction mixture containing:
750 µL bicine buffer (200 mM bicine solution, containing 0.4 mM EDTA and 1 mM DTT, pH 8.2)
10 µL 2 M MgCl2
450 µL H2O
40 µL 250 mM NaHCO3
200 µL 12.5 mM ribulose-1,5-bisphosphate solution
20 µL cell extract
Add NaHCO3, ribulose-1,5-bisphosphate solution and cell extract last
Incubate the reaction at 37 °C with continous purge of carbon dioxide.
Take samples in 5 min intervalls Freeze them immediatly at -20 °C
Before measuring the sample in the HPLC, centrifuge the samples through a Amicon Ultra filter device with a 10 kDa cut-off. Further informations are given in this protocol
Analyze samples in HPLC following the protocol for HPLC
Sodium dodecyl sulfate polyacrylamide gelelectrophoresis (SDS-PAGE)
Sodium dodecyl sulfate polyacrylamide gelelectrophoresis (SDS-PAGE)
Pouring the polyacrylamide gel:
For each separating gel (12 %) aliquote:
1.35 ml Bisacrylamid/Acrylamid (0.8 % , 30 %, at the ratio of 37.5:1)
0.675 ml H2O
0.675 ml 1.88 M Tris-HCl (pH 8.8)
0.675 ml 0.5 % SDS
Add 120 µl 10 % ammonium persulfate and 37,5 µl TEMED to each aliquote and mix
Pour the solution quickly into the gel casting form. Leave about 2 centimeters below the bottom of the comb for the stacking gel
Layer isopropanol on top of the gel
Leave the separating gel at room temperature for >60 minutes to polymerize
Remove isopropanol and wash the surface of the separating gel with H2O. Wait until the surface is dry
For each stacking gel (5 %) aliquote:
0.309 ml Bisacrylamid/Acrylamid (0.8 % , 30 %, at the ratio of 37.5:1)
0.726 ml H2O
0.375 ml 0.625 M Tris-HCl (pH 6.8)
0.375 ml 0.5 % SDS
Add 60 µl 10 % ammonium persulfate and 22,5 µl TEMED to each aliquote and mix
Insert comb without getting bubbles stuck underneath
Leave the gel at room temperature for >60 minutes to polymerize
For storage:
Remove sealing and store the gel wrapped in moistened paper towel at 4 °C
Preparing the sample:
Mix your protein mixture 3:1 with PBJR buffer (15 µl protein solution + 5 µl PBJR buffer )
Heat for 5 minutes at 95 °C
Running the gel:
Remove sealing, put the polymerized gel into gel box and pour SDS-PAGE running buffer into the negative and positive electrode chamber
Remove comp without destroying the gel pockets
Pipet the SDS running buffer in the gel pockets up and down for flushing the gel pockets
Pipet slowly 20 µl of the sample into the gel pockets
Make sure to include at least one lane with molecular weight standards (PageRuler Prestained Protein Ladder™ (Fa. Fermentas)) to determinate the molecular weight of the sample
Connect the power lead and run the stacking gel with 10 mA until the blue dye front enters the separating gel
Raise amperage up to 20 mA for running the separating gel
When the distance of the lowest molecular weight standard lane to the gel end is down to 0.5 cm stop the electrophoresis by turning off the power supply
Staining the polyacrylamide gel (Colloidal Coomassie Brilliant Blue staining):
After finishing the SDS-PAGE remove gel from gel casting form and transfer it into a box
Add 100 ml of the Colloidal Coomaassie Brilliant Blue staining solution to your polyacrylamid gel
Incubate the gel in the solution at room temperature until the protein bands got an intensive blue color. Shake the gel continuously during incubation
Remove the staining solution
Wash the gel with  7 % (v/v) acetic acid in H2O for decoloration
Incubate the gel in H2O (2-6 h) for bleaching the background. Shake the gel continuously during incubation.  If necessary replace the colored water with new one
Cultivations
Biolog System
Biolog System
This protocol is based on the technical guide from Biolog
Grow the strain of interest on a BUG+B (Biolog Universal Growth Medium + 5 % Sheep blood) agar plate by streaking out for single colonies
Incubate overnight at 37 °C
Pick single colonies with sterile swab
Transfer them into sterile tube containing 10 ml 1.0x IF-0a (mix 8 ml 1.2x IF-0a with 2 ml sterile water) to adjust transmittance to 42 %. This creates solution A.
Mix 144 μl Dye Mix A for Gram-negative bacteria and 1.856 ml sterile water with 8 ml 1.2x IF-0a solution. This creates solution B.
Mix 1.8 ml of solution A with 9 ml of solution B to achieve a final cell density of 85 % transmittance. This creates solution C.
Inoculate Biolog PM plate (96 wells) with 100 μl suspension C per well  incubate loaded plate for 48 h in the OmniLog PM system
Pretreatment of the Nafion® membrane
Pretreatment of the Nafion® membrane
Slightly boiling (≈ 80°C) in 3 &percent; H2O2 for 1 hour
Slightly boiling (≈ 80°C) in deionized water for 2 hours
Slightly boiling (≈ 80°C) in 0.5M H2SO4
The membrane needs to be rinsed between the boiling steps and after the final boiling step. After the pretreatment the membrane needs to be stored in deionized water all the time and must not dry.
Anaerobic cultivation of Escherichia coli
Anaerobic cultivation of Escherichia coli
Hole anaerobic work takes place in a two-hand Glove Bag .
All media and buffer has to be degased  with nitrogen (N2) gas via a sparger befor starting cultivation.
Grow preculture under aerobic condition at 37°C.
Cells were cultivated in a gas-tight 15 ml tube additional sealed with parafilm.
Cultivation volume is about 10 mL. Fill cultivation tube with 8.5 ml steril degased cultivation medium.
When preculture reaches OD600 of 0.6-0.8, take 1.5 ml into a steril tube.
Centrifugate 1 min at 5,000 rpm and discard complete supernatant.
resuspend pellet in 1.5 ml steril degased PBS buffer .
Repeat previous step but resuspend in steril, degased cultivation medium. This two steps avoid residual oxygen in the inoculum.
Inoculate culture with 1.5 ml washed and resuspended cells at OD600 of 0.1.
Tubes are filled and opened for sampling only under nitrogen atmosphere.
Tubes are incubated in a shaker at 37°C.
At the beginning of anaerobic cultivation residual oxygen from preculture would be consumed. Therefore faster growth could be observed in the first period of cultivation until cells shift to anaerobic metabolism.
There should be more frequent sampling at the beginning about 4 to 6-hour intervalls (two times) after inoculation.
Regular sampling takes place at 1-day intervalls.
Sampling volume is about 1 mL, using 500 μl for OD600 measurement and 500 μl for HPLC analysis.
After sampling process tubes are overflowed with nitrogen gas, tightly closed and sealed with parafilm again.
Tubes are further incubated in a shaker at 37°C.
Cultivation for Expression of recombinant proteins
Cultivation for Expression of recombinant proteins
This protocol is based on the technical bulletin from Promega concerning protein expression under control of the T7 promotor
Grow overnight culture of your cells with the plasmid containing the genes for the recombinant protein in LB medium containing specific antibiotic. Grow at 37 °C
Dilute overnight culture 1:50 in LB medium containing antibiotic and grow cells at 37 °C
When the culture reaches an OD600 of 0.6-0.8, shift the temperature to 20 °C
Induce protein expression using the T7 promotor by adding rhamnose to a final concentration of 0.1 %. When the ptac promotor is used, standard induction concentrations are 0,5 mM or 1 mM.
Grow the culture at 20 °C overnight
Harvest cells by centrifugation at 4 °C
Discard supernatant and freeze pellet at -20 °C
Retrieved from " http://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
