<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_Modelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-CeBiTec/Results/Modelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-CeBiTec/Results/Modelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Results/Modelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Results/Modelling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-CeBiTec/Results/Modelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-CeBiTec/Results/Modelling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV style="height:100%" id="main_box"><DIV id="flag"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Overview" style="color:#000000"><FLAG class="flag_element1" style="margin-left:70px;"><P class="flag"><FONT color="#FFFFFF">Overview</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC" style="color:#000000"><FLAG class="flag_element2"><P class="flag"><FONT color="#FFFFFF">rMFC</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation" style="color:#000000"><FLAG class="flag_element3"><P class="flag"><FONT color="#FFFFFF">CO<SUB>2</SUB> Fixation</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Isobutanol" style="color:#000000"><FLAG class="flag_element4"><P class="flag"><FONT color="#FFFFFF">Isobutanol</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Biosafety" style="color:#000000"><FLAG class="flag_element5"><P class="flag"><FONT color="#FFFFFF">Biosafety</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Modelling" style="color:#000000"><FLAG class="flag_element6"><P class="flag"><FONT color="#FFFFFF">Modeling</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Parts" style="color:#000000"><FLAG class="flag_element7"><P class="flag"><FONT color="#FFFFFF">Parts</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/DataPage" style="color:#000000"><FLAG class="flag_element8"><P class="flag"><FONT color="#FFFFFF">Data Page</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Achievements" style="color:#000000"><FLAG class="flag_element9"><P class="flag"><FONT color="#FFFFFF">Achievements</FONT></P></FLAG></A></DIV><UL id="nav"><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec"><FONT color="#FFFFFF">Home</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/ProjectSummary"><FONT color="#FFFFFF">Summary</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/Applications"><FONT color="#FFFFFF">Applications</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Achievements"><FONT color="#FFFFFF">Achievements</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Bielefueli"><FONT color="#FFFFFF">Bielefueli</FONT></A></LI></UL><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Overview"><FONT color="#FFFFFF">Project</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Overview"><FONT color="#FFFFFF">Overview</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Theory"><FONT color="#FFFFFF">rMFC</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/CO2-fixation/CarbonDioxide"><FONT color="#FFFFFF">CO<SUB>2</SUB> Fixation</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Isobutanol/Theory"><FONT color="#FFFFFF">Isobutanol</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Biosafety"><FONT color="#FFFFFF">Biosafety</FONT></A></LI><LI><ACTUAL><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Overview"><FONT color="#FFFFFF">Results</FONT></A></ACTUAL></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team"><FONT color="#FFFFFF">Team</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team/Members"><FONT color="#FFFFFF">Members</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team/Subteams"><FONT color="#FFFFFF">Subteams</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/PicturesAndVideos"><FONT color="#FFFFFF">Pictures &amp; Videos</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/BielefeldAndUniversity"><FONT color="#FFFFFF">Bielefeld &amp; University</FONT></A></LI><LI><A href="https://igem.org/Team.cgi?year=2014"><FONT color="#FFFFFF">Official Team Profile</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Acknowledgements"><FONT color="#FFFFFF">Acknowledgements</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Contact"><FONT color="#FFFFFF">Contact</FONT></A></LI><LI style="width:155px;"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice"><FONT color="#FFFFFF">Policy &amp; Practices</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Experts"><FONT color="#FFFFFF">Experts</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/RadioInterview"><FONT color="#FFFFFF">Interviews</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/NRW-day"><FONT color="#FFFFFF">NRW-Day</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/ScholarAcademy"><FONT color="#FFFFFF">Pupils Acedemy</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Wikisession"><FONT color="#FFFFFF">How To Wiki</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Namu"><FONT color="#FFFFFF">namu</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene"><FONT color="#FFFFFF">SYNENERGENE</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/Applications"><FONT color="#FFFFFF">Applications</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/TechnomoralScenario"><FONT color="#FFFFFF">Vignettes</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook"><FONT color="#FFFFFF">Notebook</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Journal"><FONT color="#FFFFFF">Journal</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols"><FONT color="#FFFFFF">Protocols</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media"><FONT color="#FFFFFF">Media</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes"><FONT color="#FFFFFF">Kits &amp; Enzymes</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs"><FONT color="#FFFFFF">Strains &amp; Constructs</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Primer"><FONT color="#FFFFFF">Primer</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Acrynom"><FONT color="#FFFFFF">Acronyms</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms"><FONT color="#FFFFFF">Organisms</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Safety"><FONT color="#FFFFFF">Additional</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners"><FONT color="#FFFFFF">Partners</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/GoldSponsor"><FONT color="#FFFFFF">Gold Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/SilverSponsor"><FONT color="#FFFFFF">Silver Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/BronzeSponsor"><FONT color="#FFFFFF">Bronze Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/Partners"><FONT color="#FFFFFF">Partners</FONT></A></LI><H1>Modeling</H1><DIV class="element"><DIV class="element" style="margin:10px; padding:10px"><DIV id="text"><H6>Summary</H6><P>
Different modeling approaches were used to identify enzymatic bottlenecks in the isobutanol production pathway and to predict the formation of the desired product in a given time. First of all we created a map of all metabolic reactions which are part of our project (Figure 1). This network not only provides a good overview, it also serves as the basic tool for further considerations. Due to the huge amount of components it does not seem feasible to create a computational model for all reactions at once. Therefore we started our modeling work by carrying out a stochiometric analysis. It revealed that 42 electrons are needed for the production of one isobutanol molecule from CO<SUB>2</SUB>. As for the isobutanol production pathway, dynamic modeling was carried out, in the course of which several bottlenecks could be identified. An increase in expression of <I>kivD</I> and <I>adhA</I> could boost the isobutanol production after four hours by 100 %. Finally we prepared the extension of the existing model to predict the effect of specific carbon dioxide fixing reactions.
</P></DIV></DIV><DIV class="element" style="margin:10px; padding:10px"><DIV id="text"><H6>Introduction</H6><P>
Mathematical modeling is crucial to understand complex biological systems (<A href="#schaber2009">Schaber et al., 2009</A>). The analysis of isolated biological components has been supplemented by a systems biology approach in over ten years (<A href="#chuang2010">Chuang et al., 2010</A>). Furthermore mathematical modeling is used to combine biological results (<A href="#kherlopian2008">Kherlopian et al., 2008</A>). Modeling is also a way to achieve results without carrying out experiments in a laboratory. The behaviour of a system can be simulated to get results which cannot be derived from simply looking at the given system (<A href="#schaber2009">Schaber et al., 2009</A>).
The most important aim of any modeling approach is the reduction of complexity. As the given biological reality is often diverse and variable, it is important to identify the major rules and principles which can describe a system.
</P></DIV></DIV><DIV class="element" style="margin:10px; padding:10px"><DIV id="text"><H6>Our aims</H6><UL><LI>Visualization of the complete metabolic network </LI><LI>Relating electron input to product formation </LI><LI>Relating electron input to carbon dioxide fixation </LI><LI>The identification of bottlenecks in the isobutanol production pathway </LI><LI>Prediction of maximal isobutanol production over the time</LI></UL></DIV></DIV></DIV><DIV class="element" style="margin:10px; padding:10px"><DIV id="text"><H6>Stoichiometric analysis</H6><P>
Stoichiometric analysis is useful to get information about the maximal output of a system. In this project the number of electrons moving into the system limits the amount of the product which could be synthetized in the cells. Therefore the stoichiometric relations of all substances involved in our complex reaction network were calculated (Figure 1). The calculation starts with the electrons, which are transported into the system by mediators. We calculated the resulting production of intracellular molecules based on our map of the metabolic network (Figure 1). The results are listed below.
</P><CENTER><DIV class="element" style="width:750px"><FONT size="2" style="text-align:left;"><B>Figure 1</B>: Complete metabolic network of reactions involved in our project. </FONT></DIV></CENTER><DIV class="element" style="width:400px; float:left"><FONT size="2"><B>Table 1:</B> This table shows the theoretical electron cost of different intracellular molecules. The electron cost is the number of electrons which are needed for the synthesis of this metabolite. The intermediates are substances which are needed for the production of the final product. Their costs are included in the final value for each substance.</FONT><TABLE width="400px" border="1" cellpadding="5" style="background-color:transparent;float:left"><TBODY><TR><TH>Substance</TH><TH>Intermediates</TH><TH>Number of electrons</TH></TR><TR><TD>FADH<SUB>2</SUB></TD><TD>-</TD><TD>2</TD></TR><TR><TD>NADH+H<SUP>+</SUP></TD><TD>-</TD><TD>2</TD></TR><TR><TD>ATP</TD><TD>-</TD><TD>1</TD></TR><TR><TD>Triosephosphate</TD><TD>9 ATP + 6 NADH</TD><TD>21</TD></TR><TR><TD>CO<SUB>2</SUB> fixation</TD><TD>-</TD><TD>7</TD></TR><TR><TD>Pyruvate</TD><TD>Triosephosphate</TD><TD>19</TD></TR><TR><TD>Isobutanol</TD><TD>2x Pyruvate</TD><TD>42</TD></TR></TBODY></TABLE></DIV><P>
The theoretical electron costs of different molecules is listed in Table 1. All calculations are based on our pathway map. According to our pathway map there are 42 electrons needed for the production of one molecule isobutanol, if CO<SUB>2</SUB> is used as sole carbon source. Our calculation does not involve the house keeping metabolism of <I>E. coli</I>, which consumes lots of energy for the survival of the cell. The true number of consumed electrons per produced isobutanol molecule is therefore much higher. The applied electric power can be converted into a number of electrons by the following equation: 1 A = 1 C * s<SUP>-1</SUP> = 6.2415065 * 10<SUP>18</SUP> electrons.
</P></DIV></DIV><DIV class="element" style="margin:10px; padding:10px"><DIV id="text"><H6> Dynamic modeling </H6><P>
After the stoichiometric analysis of the system we designed a dynamic model containing kinetic equations for the metabolite concentrations. It allows the identification of metabolic or enzymatic bottlenecks, the major aim of our modeling approach. This information might then be used in the next step to modify constructs e.g. exchange an RBS or a promotor sequence. This could help to balance the different enzyme concentrations in upcoming experiments in the laboratory, it was our goal to predict the production of isobutanol per substrate in a given time. Our model predicts the isobutanol production in a carbon dioxide fixing cell. To achieve our aims we reduced the complex system shown in Figure 1 to the version shown in Figure 2. This metabolic network was suitable for our dynamic modeling approach.
</P><CENTER><DIV class="element" style="width:750px"><FONT size="2" style="text-align:left;"><B>Figure 2</B>: Reduced metabolic network of reactions which were selected for modeling. </FONT></DIV></CENTER><P>
The modeling work on the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Isobutanol/GeneticalApproach" target="_blank">isobutanol pathway</A> is based on publications about the isobutanol production pathway <A href="#atsumi2008">(Atsumi et al., 2008</A> and <A href="#atsumi2010"> Atsumi et al., 2010)</A>. We started our work on the isobutanol production pathway by collecting the appropriate kinetic parameters. They were used for the development of a  system of differential equations. Concerning the choice of the kinetics used, we stuck to Michaelis-Menten kinetics as this was published as the best approach if reaction kinetics are not known (<A href="#breitling2008">Breitling et al., 2008</A>; <A href="#chubukov2014">Chubukov et al., 2014</A>). Additionally k<SUB>cat</SUB> and K<SUB>M</SUB> values were collected from databases like <A href="http://www.genome.jp/kegg" target="_blank">KEGG</A>, <A href="http://biocyc.org" target="_blank">biocyc</A> and <A href="http://www.brenda-enzymes.org" target="_blank">BRENDA </A>(Table 2). Missing values had to be estimated. The starting concentrations for different metabolites were also taken from the literature and from different databases (Table 3).

We decided to use only the forward reactions from pyruvate to isobutanol for different reasons. First of all, it is necessary to get the maximal product concentration, secondly the reactions are nearly irreversible due to the decarboxylation steps and thirdly there is only few information on the back reactions available.
</P><DIV class="element"><FONT size="2"><B>Table 2:</B> This table shows all enzymatic parameters which were used for our dynamic model.</FONT><TABLE width="100%" border="1" cellpadding="5" style="background-color:transparent"><TBODY><TR><TH>Enzyme</TH><TH>k<SUB>cat</SUB> [-]</TH><TH>K<SUB>M</SUB> [mM]</TH><TH>Reference</TH></TR><TR><TD>AlsS</TD><TD>121</TD><TD>13.6</TD><TD><A href="#atsumi2009">Atsumi et al., 2009</A></TD></TR><TR><TD>IlvC</TD><TD>2.2</TD><TD>0.25</TD><TD><A href="#tyagi2005">Tyagi et al., 2005</A></TD></TR><TR><TD>IlvD</TD><TD>10 (estimated)</TD><TD>1.5</TD><TD><A href="#flint1993">Flint et al., 1993</A></TD></TR><TR><TD>KivD</TD><TD>20 (estimated)</TD><TD>5 (estimated)</TD><TD><A href="#werther2008">Werther et al., 2008</A>; <A href="#gorcke2007">Gorcke et al., 2007</A></TD></TR><TR><TD>AdhA</TD><TD>6.6</TD><TD>9.1</TD><TD><A href="#atsumi2010">Atsumi et al., 2010</A></TD></TR></TBODY></TABLE></DIV><DIV class="element"><FONT size="2"><B>Table 3:</B> This table shows all metabolite concentrations which were used for our first model. The metabolite concentration was set to zero, if no published value was available.</FONT><TABLE width="100%" border="1" cellpadding="5" style="background-color:transparent"><TBODY><TR><TH>Metabolite</TH><TH>Concentration [mM]</TH><TH>Reference</TH></TR><TR><TD>Pyruvate</TD><TD>10</TD><TD><A href="#yang2000">Yang et al., 2000</A></TD></TR><TR><TD>2-Acetolactate</TD><TD>0</TD><TD>-</TD></TR><TR><TD>2,3-Dihydroxyisovalerate</TD><TD>0</TD><TD>-</TD></TR><TR><TD>2-Ketoisovalerate</TD><TD>0</TD><TD>-</TD></TR><TR><TD>Isobutyraldehyde</TD><TD>0.6 (estimated)</TD></TR><TR><TD>Isobutanol</TD><TD>variable</TD><TD>-</TD></TR></TBODY></TABLE></DIV><DIV id="text"><P>


We implemented the system of differential equations in Matlab (<A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Modelling/sourcecode" target="_blank">source code</A>). Two examples are shwon in Figure 3. The predicted changes of the metabolic concentration over the time is shown in Figures 4-7. The amount of expressed proteins could differ depending on the distance of the coding sequence downstream of the promoter. The coding sequences for the involved enzymes are located downstream of a common promoter. Therefore we decided to set the enzyme concentration for the first enzyme to 1 and decrease in steps of 0.1 (<A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Modelling/sourcecode" target="_blank">source code</A>). These different values were tried to identify appropriate concentrations for each enzyme. The results of this dynamic modeling approach could be transferred to the laboratory by using promoters of different strength.
</P></DIV><CENTER><DIV class="element" style="width:750px"><FONT size="2" style="text-align:left;"><B>Figure 3</B>: Examples of differential equations for the dynamic modeling of the isobutanol production. </FONT></DIV></CENTER><CENTER><DIV class="element" style="width:750px"><FONT size="2" style="text-align:left;"><B>Figure 4</B>: Predicted changes in metabolic concentration over time. </FONT></DIV></CENTER><CENTER><DIV class="element" style="width:750px"><FONT size="2" style="text-align:left;"><B>Figure 5</B>: Predicted changes in metabolic concentration over time. The development in the first minutes is shown by zooming into the Figure 4. </FONT></DIV></CENTER><CENTER><DIV class="element" style="width:750px"><FONT size="2" style="text-align:left;"><B>Figure 6</B>: Predicted changes in metabolic concentration in an improved system over time. The concentration of KivD and AdhA was increased by factors 3 and 4, respectively. </FONT></DIV></CENTER><CENTER><DIV class="element" style="width:750px"><FONT size="2" style="text-align:left;"><B>Figure 7</B>: Predicted changes in metabolic concentration in an improved system over time. The concentration of KivD and AdhA was increased by factor 3 and 4 respectively. The development in the first minutes is shown by zooming into the Figure 6. </FONT></DIV></CENTER><DIV id="text"><P>
Our modeling results indicated that the concentrations of two enzymes are limiting the isobutanol production <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Isobutanol/GeneticalApproach" target="_blank">KivD</A> and <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Isobutanol/GeneticalApproach" target="_blank">AdhA</A>. An experimental validation of this effect is the next logical step. This enzymatic bottleneck could be removed by overexpression of the corresponding coding sequences (<A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Isobutanol/GeneticalApproach" target="_blank"><I>kivD</I></A> and <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Isobutanol/GeneticalApproach" target="_blank"><I>adhA</I></A>). One way to achieve this is the integration of a strong promotor upstream of their coding sequences. It could be a possibility to improve our isobutanol production.
As shown in Figure 4 the isobutanol concentration reaches 2 mM after four hours. In the improved system the concentration of isobutanol after four hours is nearly doubled.
</P></DIV></DIV></DIV><DIV class="element" style="margin:10px; padding:10px"><DIV id="text"><H6>Conclusion</H6>
The complete metabolic network of our project is visualized in Figure 1. Stoichiometric calculations revealed the number of electrons which is in theory need for the production of a desired substance. There are 42 electrons required for the synthesis of one molecule isobutanol. The fixation of one carbon dioxide molecule would cost seven electrons. We were able to identify a putative enzymatic bottleneck in the isobutanol production pathway. An increased amount of KivD and AdhA could improve the isobutanol production. Additionally the production of isobutanol from pyruvate can be predicted. The next and very important step would be the validation of these predictions by experimental approaches.

</DIV></DIV><DIV class="element" style="margin:10px; padding:10px"><DIV id="text"><H6>Outlook: addition of carbon dioxide fixing reactions</H6><P>
The next step would be the addition of specific carbon fixing reactions and the pathway leading to pyruvate in the existing dynamic model. Therefore we collected k<SUB>cat</SUB> and K<SUB>M</SUB> values for nearly all relevant steps (Table 4). They can be used to formulate additional differential equations which describe these additional reactions. 
</P><DIV class="element"><FONT size="2"><B>Table 4:</B> This table shows all k<SUB>cat</SUB> and K<SUB>M</SUB> values of enzymes involved in CO<SUB>2</SUB> fixation and the pathway leading to pyruvate.</FONT><TABLE width="100%" border="1" cellpadding="5" style="background-color:transparent"><TBODY><TR><TH>Enzyme</TH><TH>k<SUB>cat</SUB> [-]</TH><TH>K<SUB>M</SUB> [mM]</TH><TH>Reference</TH></TR><TR><TD>PrkA</TD><TD>72.6</TD><TD>0.09</TD><TD><A href="#wadano1998">Wadano et al., 1998</A>,<A href="#kobayashi2003">Kobayashi et al., 2003</A></TD></TR><TR><TD>RubisCO</TD><TD>20 (estimated)</TD><TD>0.02 (estimated)</TD><TD><A href="#lan1991">Lan and Mott, 1991</A>,<A href="#sage2002">Sage, 2002</A></TD></TR><TR><TD>Pgk</TD><TD>480</TD><TD>1 (estimated)</TD><TD><A href="#fifis1978">Fifis and Scopes., 1978</A></TD></TR><TR><TD>GapA</TD><TD>-</TD><TD>0.5</TD><TD><A href="#zhao1995">Zhao et al., 1995</A></TD></TR><TR><TD>GpmA</TD><TD>490 (in <I>S.cerevisiae</I>)</TD><TD>0.15</TD><TD><A href="#fraser1999">Fraser et al., 1999</A>,<A href="#white1992">White and Fothergil-Gilmore, 1992</A></TD></TR><TR><TD>Eno</TD><TD>17600</TD><TD>0.1</TD><TD><A href="#spring1972">Spring and Wold, 1972</A>, <A href="#albe1990">Albe et al., 1990</A></TD></TR><TR><TD>PykF</TD><TD>3.2</TD><TD>0.3 (estimated)</TD><TD><A href="#oria2005">Oria-Hernandez et al., 2005</A></TD></TR></TBODY></TABLE></DIV>

Using these information the combination of two parts of our project would be possible. The carbon dioxide fixation pathway could be checked for enzymatic bottlenecks. It would be possible to predict the isobutanol production from fixed carbon dioxide.

</DIV></DIV><DIV class="element" style="margin:10px; padding:10px"><DIV id="text"><H4 class="member" style="margin-left:20px">References</H4><UL><DIV element="class"><LI id="atsumi2008"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Atsumi, Shota, Taizo Hanai, und James C. Liao. „Non-Fermentative Pathways for Synthesis of Branched-Chain Higher Alcohols as Biofuels“. <A href="http://www.nature.com/nature/journal/v451/n7174/full/nature06450.html" target="_blank">Nature</A> 451, no. 7174 (2008): 86–89.
</DIV></DIV></LI><LI id="atsumi2010"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Atsumi, Shota, Tung-Yun Wu, Eva-Maria Eckl, Sarah D. Hawkins, Thomas Buelter, und James C. Liao. „Engineering the isobutanol biosynthetic pathway in Escherichia coli by comparison of three aldehyde reductase/alcohol dehydrogenase genes“. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2802489" target="_blank">Applied Microbiology and Biotechnology</A> 85, no. 3 (2010): 651–57.
</DIV></DIV></LI><LI id="yang2000"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Yang, Y. T., G. N. Bennett, und K. Y. San. „The Effects of Feed and Intracellular Pyruvate Levels on the Redistribution of Metabolic Fluxes in Escherichia Coli“. <A href="http://bioc.rice.edu/bennetlab/references/ref40-sdarticle.pdf" target="_blank">Metabolic Engineering</A> 3, no. 2 (2001): 115–23.
</DIV></DIV></LI><LI id="wadamo1998"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Wadano, Akira, Keisuke Nishikawa, Tomohiro Hirahashi, Ryohei Satoh, und Toshio Iwaki. „Reaction Mechanism of Phosphoribulokinase from a Cyanobacterium, Synechococcus PCC7942“. <A href="http://link.springer.com/article/10.1023%2FA%3A1005979801741" target="_blank">Photosynthesis Research</A> 56, no. 1 (1998): 27–33.
</DIV></DIV></LI><LI id="kobayashi2003"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Kobayashi, Daisuke, Masahiro Tamoi, Toshio Iwaki, Shigeru Shigeoka, und Akira Wadano. „Molecular Characterization and Redox Regulation of Phosphoribulokinase from the Cyanobacterium Synechococcus Sp. PCC 7942“. <A href="http://www.ncbi.nlm.nih.gov/pubmed/12668773" target="_blank">Plant &amp; Cell Physiology</A> 44, no. 3 (2003): 269–76.
</DIV></DIV></LI><LI id="lan1991"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Lan, Yun, und Keith A. Mott. „Determination of Apparent Km Values for Ribulose 1,5-Bisphosphate Carboxylase/Oxygenase (Rubisco) Activase Using the Spectrophotometric Assay of Rubisco Activity“. <A href="http://www.plantphysiol.org/content/95/2/604.abstract" target="_blank">Plant Physiology</A> 95, Nr. 2 (1991): 604–9.
</DIV></DIV></LI><LI id="sage2002"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Sage, Rowan F. „Variation in the K(cat) of Rubisco in C(3) and C(4) Plants and Some Implications for Photosynthetic Performance at High and Low Temperature“. <A href="http://www.ncbi.nlm.nih.gov/pubmed/11886880" target="_blank">Journal of Experimental Botany</A> 53, no. 369 (2002): 609–20.
</DIV></DIV></LI><LI id="fifis1978"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Fifis, T., und R. K. Scopes. „Purification of 3-Phosphoglycerate Kinase from Diverse Sources by Affinity Elution Chromatography“. <A href="http://www.ncbi.nlm.nih.gov/pubmed/?term=fifis+1978" target="_blank">The Biochemical Journal</A> 175, no. 1 (1978): 311–19.
</DIV></DIV></LI><LI id="zhao1995"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Zhao G, Halbur T, Pankratz DC. Colonization of oropharynx and nasal cavity of CDCD pigs by a nontoxigenic strain of Pasteurella multocida type D. <A href="http://www.aasv.org/shap/abstracts/abstract.php?v3n3p113" target="_blank">J Swine Health Prod</A> 1995;3(3):113-115.
</DIV></DIV></LI><LI id="fraser1999"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Fraser, H. I., M. Kvaratskhelia, und M. F. White. „The Two Analogous Phosphoglycerate Mutases of Escherichia Coli“. <A href="http://www.ncbi.nlm.nih.gov/pubmed/10437801" target="_blank">FEBS Letters</A> 455, no. 3 (1999): 344–48.
</DIV></DIV></LI><LI id="white1992"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
White, Malcolm F., und Linda A. Fothergill-Gilmore. „Development of a Mutagenesis, Expression and Purification System for Yeast Phosphoglycerate Mutase“. <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1432-1033.1992.tb17099.x/pdf" target="_blank">European Journal of Biochemistry</A> 207, no. 2 (1992): 709–14.
</DIV></DIV></LI><LI id="spring1972"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Spring, Thomas G., und Finn Wold. „The Purification and Characterization of Escherichia Coli Enolase“. <A href="http://www.jbc.org/content/246/22/6797.long" target="_blank">Journal of Biological Chemistry</A> 246, no. 22 (1971): 6797–6802.
</DIV></DIV></LI><LI id="albe1990"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Albe, Kathy R., Margaret H. Butler, und Barbara E. Wright. „Cellular concentrations of enzymes and their substrates“. <A href="http://www.sciencedirect.com/science/article/pii/S0022519305802668" target="_blank">Journal of Theoretical Biology</A> 143, no. 2 (1990): 163–95.
</DIV></DIV></LI><LI id="oria2005"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Oria-Hernández, Jesús, Nallely Cabrera, Ruy Pérez-Montfort, und Leticia Ramírez-Silva. „Pyruvate Kinase Revisited: The Activating Effect of K+“. <A href="http://www.ncbi.nlm.nih.gov/pubmed/16147999" target="_blank">The Journal of Biological Chemistry</A> 280, no. 45 (2005): 37924–29.
</DIV></DIV></LI><LI id="breitling2008"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Breitling, Rainer, David Gilbert, Monika Heiner, und Richard Orton. „A Structured Approach for the Engineering of Biochemical Network Models, Illustrated for Signalling Pathways“. <A href="http://bib.oxfordjournals.org/content/9/5/404.short" target="_blank">Briefings in Bioinformatics</A> 9, no. 5 (2008): 404–21.
</DIV></DIV></LI><LI id="chubukov2014"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Chubukov, Victor, Luca Gerosa, Karl Kochanowski, und Uwe Sauer. „Coordination of Microbial Metabolism“. <A href="http://www.nature.com/nrmicro/journal/v12/n5/abs/nrmicro3238.html" target="_blank">Nature Reviews Microbiology</A> 12, no. 5 (Mai 2014): 327–40.
</DIV></DIV></LI><LI id="schaber2009"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Schaber, J., W. Liebermeister, und E. Klipp. „Nested Uncertainties in Biochemical Models“. <A href="http://www.ncbi.nlm.nih.gov/pubmed/19154080" target="_blank">IET Systems Biology</A> 3, no. 1 (2009): 1–9.
</DIV></DIV></LI><LI id="chuang2010"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Chuang, Han-Yu, Matan Hofree, und Trey Ideker. „A Decade of Systems Biology“. <A href="http://www.ncbi.nlm.nih.gov/pubmed/20604711" target="_blank">Annual Review of Cell and Developmental Biology</A> 26 (2010): 721–44.
</DIV></DIV></LI><LI id="kherlopian2008"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Kherlopian, Armen R., Ting Song, Qi Duan, Mathew A. Neimark, Ming J. Po, John K. Gohagan, und Andrew F. Laine. „A Review of Imaging Techniques for Systems Biology“. <A href="http://www.biomedcentral.com/1752-0509/2/74" target="_blank">BMC Systems Biology</A> 2, no. 1 (12. August 2008): 74.
</DIV></DIV></LI><LI id="tyagi2005"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Tyagi, R., Lee, Y.-T., Guddat, L.W., and Duggleby, R.G. (2005). Probing the mechanism of the bifunctional enzyme ketol-acid reductoisomerase by site-directed mutagenesis of the active site. <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1742-4658.2004.04506.x/pdf" target="_blank">FEBS J.</A> 272, 593–602.
</DIV></DIV></LI><LI id="atsumi2009"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Atsumi, S., Li, Z., and Liao, J.C. (2009). Acetolactate Synthase from Bacillus subtilis Serves as a 2-Ketoisovalerate Decarboxylase for Isobutanol Biosynthesis in Escherichia coli. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2753059" target="_blank">Appl Environ Microbiol</A> 75, 6306–6311.
</DIV></DIV></LI><LI id="flint1993"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Flint, D.H., Emptage, M.H., Finnegan, M.G., Fu, W., and Johnson, M.K. (1993). The role and properties of the iron-sulfur cluster in Escherichia coli dihydroxy-acid dehydratase. <A href="http://www.ncbi.nlm.nih.gov/pubmed/8325851" target="_blank">J. Biol. Chem.</A> 268, 14732–14742.
</DIV></DIV></LI><LI id="gorcke2007"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Gocke, D., Nguyen, C.L., Pohl, M., Stillger, T., Walter, L., and Müller, M. (2007). Branched-Chain Keto Acid Decarboxylase from Lactococcus lactis (KdcA), a Valuable Thiamine Diphosphate-Dependent Enzyme for Asymmetric CC Bond Formation. <A href="http://onlinelibrary.wiley.com/doi/10.1002/adsc.200700057/pdf" target="_blank">Adv. Synth. Catal.</A> 349, 1425–1435.
</DIV></DIV></LI><LI id="werther2008"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Werther, T., Spinka, M., Tittmann, K., Schütz, A., Golbik, R., Mrestani-Klaus, C., Hübner, G., and König, S. (2008). Amino Acids Allosterically Regulate the Thiamine Diphosphate-dependent α-Keto Acid Decarboxylase from Mycobacterium tuberculosis. <A href="http://www.jbc.org/content/283/9/5344.long" target="_blank">J. Biol. Chem.</A> 283, 5344–5354.
</DIV></DIV></LI></DIV></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Modelling">http://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Modelling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-CeBiTec/Results/Modelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-CeBiTec/Results/Modelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Results/Modelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Results/Modelling&amp;oldid=393745" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>