<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_CO2-fixation_Calvin-Cycle"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV style="height:100%" id="main_box"><DIV id="flag"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Overview" style="color:#000000"><FLAG class="flag_element1" style="margin-left:70px;"><P class="flag"><FONT color="#FFFFFF">Overview</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC" style="color:#000000"><FLAG class="flag_element2"><P class="flag"><FONT color="#FFFFFF">rMFC</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation" style="color:#000000"><FLAG class="flag_element3"><P class="flag"><FONT color="#FFFFFF">CO<SUB>2</SUB> Fixation</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Isobutanol" style="color:#000000"><FLAG class="flag_element4"><P class="flag"><FONT color="#FFFFFF">Isobutanol</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Biosafety" style="color:#000000"><FLAG class="flag_element5"><P class="flag"><FONT color="#FFFFFF">Biosafety</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Modelling" style="color:#000000"><FLAG class="flag_element6"><P class="flag"><FONT color="#FFFFFF">Modeling</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Parts" style="color:#000000"><FLAG class="flag_element7"><P class="flag"><FONT color="#FFFFFF">Parts</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/DataPage" style="color:#000000"><FLAG class="flag_element8"><P class="flag"><FONT color="#FFFFFF">Data Page</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Achievements" style="color:#000000"><FLAG class="flag_element9"><P class="flag"><FONT color="#FFFFFF">Achievements</FONT></P></FLAG></A></DIV><UL id="nav"><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec"><FONT color="#FFFFFF">Home</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/ProjectSummary"><FONT color="#FFFFFF">Summary</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/Applications"><FONT color="#FFFFFF">Applications</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Achievements"><FONT color="#FFFFFF">Achievements</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Bielefueli"><FONT color="#FFFFFF">Bielefueli</FONT></A></LI></UL><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Overview"><FONT color="#FFFFFF">Project</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Overview"><FONT color="#FFFFFF">Overview</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Theory"><FONT color="#FFFFFF">rMFC</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/CO2-fixation/CarbonDioxide"><FONT color="#FFFFFF">CO<SUB>2</SUB> Fixation</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Isobutanol/Theory"><FONT color="#FFFFFF">Isobutanol</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Biosafety"><FONT color="#FFFFFF">Biosafety</FONT></A></LI><LI><ACTUAL><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Overview"><FONT color="#FFFFFF">Results</FONT></A></ACTUAL></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team"><FONT color="#FFFFFF">Team</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team/Members"><FONT color="#FFFFFF">Members</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team/Subteams"><FONT color="#FFFFFF">Subteams</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/PicturesAndVideos"><FONT color="#FFFFFF">Pictures &amp; Videos</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/BielefeldAndUniversity"><FONT color="#FFFFFF">Bielefeld &amp; University</FONT></A></LI><LI><A href="https://igem.org/Team.cgi?year=2014"><FONT color="#FFFFFF">Official Team Profile</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Acknowledgements"><FONT color="#FFFFFF">Acknowledgements</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Contact"><FONT color="#FFFFFF">Contact</FONT></A></LI><LI style="width:155px;"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice"><FONT color="#FFFFFF">Policy &amp; Practices</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Experts"><FONT color="#FFFFFF">Experts</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/RadioInterview"><FONT color="#FFFFFF">Interviews</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/NRW-day"><FONT color="#FFFFFF">NRW-Day</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/ScholarAcademy"><FONT color="#FFFFFF">Pupils Acedemy</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Wikisession"><FONT color="#FFFFFF">How To Wiki</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Namu"><FONT color="#FFFFFF">namu</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene"><FONT color="#FFFFFF">SYNENERGENE</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/Applications"><FONT color="#FFFFFF">Applications</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/TechnomoralScenario"><FONT color="#FFFFFF">Vignettes</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook"><FONT color="#FFFFFF">Notebook</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Journal"><FONT color="#FFFFFF">Journal</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols"><FONT color="#FFFFFF">Protocols</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media"><FONT color="#FFFFFF">Media</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes"><FONT color="#FFFFFF">Kits &amp; Enzymes</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs"><FONT color="#FFFFFF">Strains &amp; Constructs</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Primer"><FONT color="#FFFFFF">Primer</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Acrynom"><FONT color="#FFFFFF">Acronyms</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms"><FONT color="#FFFFFF">Organisms</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Safety"><FONT color="#FFFFFF">Additional</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners"><FONT color="#FFFFFF">Partners</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/GoldSponsor"><FONT color="#FFFFFF">Gold Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/SilverSponsor"><FONT color="#FFFFFF">Silver Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/BronzeSponsor"><FONT color="#FFFFFF">Bronze Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/Partners"><FONT color="#FFFFFF">Partners</FONT></A></LI><H1>Module II - Carbon Dioxide (CO<SUB>2</SUB>) Fixation</H1><DIV id="main_menue" style="margin-left:135px; height:80px"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation" style="color:#000000"><DIV class="main_menueButton"><P class="buttoncenter"><FONT color="#FFFFFF">Overview</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation/RuBisCO" style="color:#000000"><DIV class="main_menueButton" style="width:130px"><P class="buttoncenter"><FONT color="#FFFFFF">RuBisCO</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Carboxysome" style="color:#000000"><DIV class="main_menueButton" style="width:130px"><P class="buttoncenter"><FONT color="#FFFFFF">Carboxysome</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle" style="color:#000000"><DIV class="main_menueButtonActual" style="width:130px"><P class="buttoncenter"><FONT color="#FFFFFF">Calvin-Cycle</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Measurement" style="color:#000000"><DIV class="main_menueButton" style="width:130px"><P class="buttoncenter"><FONT color="#FFFFFF">Measurement</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Outlook" style="color:#000000"><DIV class="main_menueButton" style="width:100px"><P class="buttoncenter"><FONT color="#FFFFFF">Outlook</FONT></P></DIV></A></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6>Introduction</H6><P>
As already mentioned in the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/CO2-fixation" target="_blank">project</A> section we decided to implement the Calvin-cycle in our project. There are different reasons for this. One the one side we searched for a method to cultivate under aerobic conditions and on the other side we searched for a cyclic system which is not given in every carbon dioxide fixation mode.
In <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms#E.coli" target="_blank"><I>E. coli</I></A> are three enzymes missing to enable the whole cycle. Our goal is to transform the missing genes from different organisms. Our main research is based on a publication by <A href="#bonacci2011">Bonacci et al., 2011</A>. The idea was to use the carboxysom as a microcompartiment for <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms#E.coli" target="_blank"><I>E. coli</I></A> from <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms#H.neapolitanus" target="_blank"><I>Halothiobacillus neapolitanus</I></A> for an efficient RuBisCO activity. The problem of handling the RuBisCO is its affinity to oxygen, which can be used instead of CO<SUB>2</SUB>. If the RuBisCO uses oxygen as a co-substrate, no CO<SUB>2</SUB> can be fixed. A carboxysom establishs a higher concentration of carbon dioxide in the microcompartiment. The substrate for the RuBisCO is provided by an enzyme called <A href="#prkA">phosphoribulokinase</A> which was derived from <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms#S.elongatus" target="_blank"><I>Synechococcus elongatus</I></A>. The last missing enzyme was taken from <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms#B.methanolicus"><I>Bacillus methanolicus</I></A> and is called <A href="#SBPase">sedoheptulose 1,7-bisphosphatase</A> (<A href="#stolzenberger2013">Stolzenberger et al., 2013</A>).
A problem for our system would be the glycolysis which we aimed to inhibit with a knock-down of the phosphofructokinase. We called this system the <A href="#gluco-switch">gluco-switch</A>.
The main goal for the characterization was to establish a new <A href="#protein-purification">enzyme purification</A> system based on an intein-tag containing a chitin binding domain. It could be used by integrating a coding sequence into the vector.
   </P></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6 id="prkA">Phosphoribulokinase</H6><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:300px;float:right"><FONT size="2" style="text-align:center;"><B>Figure 1</B>: Toxicity of phosphoribulokinase without RuBisCO</FONT></DIV>
The sequence of the phosphoribulokinase was synthesized to remove illegal restriction sites and to optimize the codon usage for <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms#E.coli" target="_blank"><I>E. coli</I></A>. We were able to transform the prkA (<A href="http://parts.igem.org/Part:BBa_K1465201" target="_blank">BBa_K1465201</A>, <A href="http://parts.igem.org/Part:BBa_K1465212" target="_blank">BBa_K1465212</A>, <A href="http://parts.igem.org/Part:BBa_K1465214" target="_blank">BBa_K1465214</A>) into <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms#E.coli" target="_blank"><I>E. coli</I></A> but without ribosom binding site. As a template for the synthesis we used the prkA of <I>Synechococcus elongatus</I>.
The toxicity of the PrkA in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms#E.coli" target="_blank"><I>E. coli</I></A> was described previously by <A href="#parikh2006">Parikh et al., 2006</A> and <A href="#bonacci2011">Bonacci et al., 2012</A>. The toxicity results through the accumulation of ribulose 1,5-bisphosphate which can not be further metabolized as shown in Figure 1.
We performed an enzyme assay to identify the functionality of the PrkA in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms#E.coli" target="_blank"><I>E. coli</I></A>. In order to investigate this we cultivated the strain and made a crude cell extract. The cell extract was incubated for 1 h at 37°C. We compared different approaches. First we incubated a wild type strain, secondly we incubated the crude cell extract and thirdly we incubated the crude cell extract with 1 mM ribulose 5-phosphate which is the substrate for the PrkA. The aim was to identify ribulose 1,5-bisphosphate with the HPLC. Because the wild type is not able to produce ribulose 1,5-bisphosphate the PrkA activity should be easily observed. We were not able to identify the product, ribulose 1,5-bisphosphate, with HPLC in all approaches.
To perform a SDS-Page we cultivated <I>E. coli</I> carrying prkA strain and induced gene expression with 1 mM IPTG (<A href="http://parts.igem.org/Part:BBa_K1465212" target="_blank">BBa_K1465212</A>, <A href="http://parts.igem.org/Part:BBa_K1465214" target="_blank">BBa_K1465214</A>). In comparison to the wild type the prkA carrying strain showed similar growth behavior. The resulting SDS-Page is shown below.<CENTER><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:350px;"><FONT size="2" style="text-align:center;"><B>Figure 2</B>: SDS-Page of ptac prkA</FONT></DIV></CENTER>
The PrkA has a molecular size of around 38 kD. There is a clear band shown between 35 kD and 40 kD. This fragment was cut out to identify the protein via MALDI-TOF analysis. With the MALDI-TOF we were able to identify three peptides of the PrkA which is sufficient.
The results of our overall approaches looked like the PrkA is expressed by <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms#E.coli" target="_blank"><I>E. coli</I></A> but does not show functionality in an <I>in vitro</I> assay. A possible problem is the light dependent activation of the PrkA. The light triggers the PSI in photosynthetic active organisms. This trigger reduces ferredoxine. In the follwing reaction thioredoxin is reduced while ferredoxin is oxidized again. The reduced thioredoxin is able to break disulfides. The PrkA activation is triggered by thioredoxin which is maybe not present sufficiently in <I>E. coli</I>. As previously described it would be possible to activate the PrkA by adding DTT in the enzyme assay (<A href="#hariharan1998">Hariharan et al., 1998</A>) which would be our next target for an enzyme assay.


              </DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6 id="SBPase">Sedoheptulose 1,7-bisphosphatase</H6><P>
We succesfully transformed the SBPase from <I>Bacillus methanolicus</I> (<A href="http://parts.igem.org/Part:BBa_K1465228" target="_blank">BBa_K1465228</A>, <A href="http://parts.igem.org/Part:BBa_K1465229" target="_blank">BBa_K1465229</A>). For the characterization of the sedoheptulose 1,7-bisphosphatase (SBPase / glpX) we did an enzyme assay with a His-Tag purification as described before (<A href="#stolzenberger2013">Stolzenberger et al., 2013</A>).
The proteins were <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#Cultivation for Expression of recombinant proteins" target="_blank">overexpressed</A> by adding 1 mM IPTG for inducing the T7 promotor(<A href="http://parts.igem.org/Part:BBa_K1465229" target="_blank">BBa_K1465229</A>). The increasing amount of protein could be verified with a <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#Sodiumdodecylsulfatepolyacrylamidegelelectrophoresis%20%28SDS-PAGE%29" target="_blank">SDS-PAGE.</A>.<CENTER><TABLE style="background-color:transparent"><TBODY><TR><TD><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:260px"><FONT size="2" style="text-align:center;"><B>Figure 3</B>: Proteinexpression of Fba</FONT></DIV></TD><TD><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:260px"><FONT size="2" style="text-align:center;"><B>Figure 4</B>: Proteinexpression of Tkt</FONT></DIV></TD><TD><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:260px"><FONT size="2" style="text-align:center;"><B>Figure 5</B>: Proteinexpression of GlpX</FONT></DIV></TD></TR></TBODY></TABLE></CENTER>

All three SDS-Pages showed a band with a clear increase in the amount of protein like described in <A href="#stolzenberger2013">Stolzenberger et al., 2013</A>. We purified the transketolase (Tkt) and the fructose bisphosphate aldolase (Fba) as well as the sedoheptulose 1,7-bisphosphatase with the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#HisTrapFFPurification" target="_blank">His-Tag mediated purification system</A>.<CENTER><TABLE style="background-color:transparent"><TBODY><TR><TD><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:260px"><FONT size="2" style="text-align:center;"><B>Figure 6</B>: Protein purification of Fba</FONT></DIV></TD><TD><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:260px"><FONT size="2" style="text-align:center;"><B>Figure 7</B>: Protein purification of Tkt</FONT></DIV></TD><TD><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:260px"><FONT size="2" style="text-align:center;"><B>Figure 8</B>: Protein purification of GlpX</FONT></DIV></TD></TR></TBODY></TABLE></CENTER>

For the purified enzymes we performed a Bradford assay.
</P><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:450px"><FONT size="2" style="text-align:center;"><B>Figure 9</B>: Bradford assay with purified enzymes (two technical and two biological replicates)</FONT></DIV>
The Bradford assay showed high concentrations of Tkt and Fba as well as a very low concentration of GlpX. After the purification we performed an enzyme assay as shown below.<TABLE style="background-color:transparent"><TBODY><TR><TD><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:350px"><FONT size="2" style="text-align:center;"><B>Figure 10</B>: Schematical view of the SBPase assay</FONT></DIV></TD></TR><TR><TH>Molecule</TH><TH>Formula</TH><TH>Molecular weight [M-H]</TH></TR><TR><TD>Fructose-6-P</TD><TD>C<SUB>6</SUB>H<SUB>15</SUB>O<SUB>9</SUB>P</TD><TD>259.021</TD></TR><TR><TD>Glyceraldehyd-3-P</TD><TD>C<SUB>3</SUB>H<SUB>7</SUB>O<SUB>6</SUB>P</TD><TD>168.99</TD></TR><TR><TD>Dihydroxyacetone-P</TD><TD>C<SUB>3</SUB>H<SUB>7</SUB>O<SUB>6</SUB>P</TD><TD>168.99</TD></TR><TR><TD>Xylulose-5-P</TD><TD>C<SUB>5</SUB>H<SUB>11</SUB>O<SUB>8</SUB>P</TD><TD>229.01</TD></TR><TR><TD>Erythrose-4-P</TD><TD>C<SUB>4</SUB>H<SUB>9</SUB>O<SUB>7</SUB>P</TD><TD>199.00</TD></TR><TR><TD>Sedoheptulose-7-P</TD><TD>C<SUB>7</SUB>H<SUB>15</SUB>O<SUB>10</SUB>P</TD><TD>289.03</TD></TR><TR><TD>Sedoheptulose-1,7-BP</TD><TD>C<SUB>7</SUB>H<SUB>16</SUB>O<SUB>15</SUB>P<SUB>2</SUB></TD><TD>368.99</TD></TR></TBODY></TABLE>
The product of the reaction, sedoheptulose 7-phosphate, could be identified via HPLC. We made different approaches for the enzyme assay to characterize all reactions.<DIV class="element" style="margin:10px; padding:10px; width:400px;float:right"><FONT size="2" style="text-align:center;"><B>Figure 11</B>: Approaches by adding one enzyme at each step. Each step shows the activity of the enzymes.</FONT></DIV>
Reaction mix:<UL><LI>20 mM Fructose 6-phosphate</LI><LI>20 mM Glyceraldehyde 3-phosphate</LI><LI>20 mM Dihydroxyacetonephosphate</LI><LI>10 µM Thiamine pyrophosphate</LI><LI>2 mM Manganese chloride</LI><LI>50 mM Tris-HCl</LI></UL>
For the first approach we added no enzyme to verify that no product is generated as shown in Figure 11. The second approach includes the transketolase which catalyzes the reaction of F6P and GAP to erythrose 4-phosphate. For the third approach fructose bisphosphate aldolase was added to convert erythrose 4-phosphate with dihydroacetonephosphate to sedoheptulose 1,7-bisphosphate. For the last approach the sedoheptulose 1,7-bisphosphatase (GlpX) was added which results in sedoheptulose 7-phosphate. All intermediates could be verified in all approaches as expected. This measurement showed the activity of the SBPase <I>in vitro</I> (Xylulose 5-phosphate is a byproduct of one of the enzymatical reactions).

<CENTER><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:900px"><FONT size="2" style="text-align:center;"><B>Figure 12</B>: Comparison of 37°C and 50°C of the <I>in vitro</I> assay (all enzymes).</FONT></DIV></CENTER>

We did a comparison between 37°C and 50°C of the <I>in vitro</I> assay to investigate the different enzymatic activities as shown in Figure 12. The transketolase and aldolase showed a higher activity at 37°C which resulted in more respectively products. The sedoheptulose 1,7-bisphosphatase activity is higher at 50°C but also shows activity at 37°C. We could identify this characteristic because of the accumulation of sedoheptulose 1,7-bisphosphate in the 37°C approach. The second approach has a lower concentration of this sedoheptulose 1,7-bisphosphate but showed a higher concentration of sedoheptulose 7-phosphate which is due to the higher activity of the SBPase at 50°C. This results suggest that it is possible to enable SBPase activity with GlpX in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms#E.coli" target="_blank"><I>E. coli</I></A>.
We decided to use glpX as a target for amplification and transformation because of the shown acitivity. The finished construct of ptac_glpX in pSB1C3 was cultivated in M9 glucose in comparison to the wild type. We performed two biological replicates and two technical replicates.
<CENTER><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:650px"><FONT size="2" style="text-align:center;"><B>Figure 13</B>: Cultivation experiment in M9 glucose (two technical and two biological replicates)</FONT></DIV></CENTER>

The cultivation shows that the modified strain has an extended lag phase. Two hours after IPTG induction of the gene expression, a decrease of growth was observed in comparison to the uninduced strain. There are two possible explanations for this behavior. On the one side IPTG acts as a toxic substance for bacteria which may result in growth decrease and on the other side the production of the protein can result in decreased growth. We exclude IPTG as a reason because earlier cultivations showed that 1 mM IPTG has no effects on the wild type growth. Furthermore the wild type strain shows an one hour reduced lag-phase in comparison to the induced strain. The induced strain also shows a higher OD. By inducing the SBPase in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms#E.coli" target="_blank"><I>E. coli</I></A> the substances for glycolysis are deflected towards other pathways. These reactions are reversible which means that the glucose of the M9 medium is not metabolized in another pathway. This time shifted use of glucose results in a higher OD in two biological replicates.
This result shows that the SBPase does not limit the growth maximum of <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms#E.coli" target="_blank"><I>E. coli</I></A>. The glucose concentration confirms these results. The wild type consumes the glucose faster than the mutant strains.

<CENTER><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:650px"><FONT size="2" style="text-align:center;"><B>Figure 14</B>: Cultivation experiment in M9 xylose</FONT></DIV></CENTER>

The cultivation was repeated with xylose. Xylose could be suitable additional carbon source if the Calvin cycle has a lower efficiency. This second cultivation shows the impact of the induction more clearly. The induced mutant strain shows a lower growth rate in comparison to the uninduced mutant and the wild type. This indicates that the lack of glucose in the medium has a large impact because necessary intermediates for the glycolysis were used by the SBPase.

              </DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6 id="gluco-switch">Gluco-Switch</H6><P>
The idea for a gluco-switch was the knock-down of the <I>pfk</I> via sRNA (<A href="http://parts.igem.org/Part:BBa_K1465225" target="_blank">BBa_K1465225</A>, <A href="http://parts.igem.org/Part:BBa_K1465226" target="_blank">BBa_K1465226</A>, <A href="http://parts.igem.org/Part:BBa_K1465227" target="_blank">BBa_K1465227</A>).
For the knock-down of the <I>pfkA</I> we used the synthesized sRNA:pfkA. Herewith we wanted to down-regulate the glycolysis to prohibit the bacteria to use sugar as a carbon source for our follow-up experiments. This should cause a reduction of the growth of our bacteria carrying the plasmid. We tried to show this with the help of a cultivation (Figure 15).

</P><DIV class="element" style="width:650px"><FONT size="2"><B>Figure 15</B>: Cultivation of sRNA_pfkA_glucose</FONT></DIV>

Neither the construct with the p<SUB>tac</SUB>(<A href="http://parts.igem.org/Part:BBa_K1465226" target="_blank">BBa_K1465226</A>) promoter nor the one with the T7 promoter (<A href="http://parts.igem.org/Part:BBa_K1465227" target="_blank">BBa_K1465227</A>) showed a difference between induced and uninduced state. As we used M9 glucose as the medium for the cultivation there should be a difference in the growth. The HPCL results did not show a hint for the functional activity of the sRNA as well because the induced culture as well as the uninduced one showed the same consumption of glucose. For the experiments we inoculated the cultures with a pre-culture out of the exponential phase. For any reason the wild type had an unusual long lag phase that we cannot explain. 
We think that it was maybe not enough to only take the sRNA of <I>pfkA</I>. Another try would be to integrate the sRNA:pfkB downstream of the sRNA:pfkA to be sure to cover the activity of both phosphofructokinases. Additionally the formation of the sRNA should be checked.



              </DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6 id="protein-purification">Protein purification</H6><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:250px;float:right"><FONT size="2" style="text-align:center;"><B>Figure 16</B>: Scheme for intein mediated purification</FONT></DIV><P>For the purpose of characterizing our BioBricks we thought of using enzyme assays to verify the functionality of different proteins. Enzyme assays depend on purified enzymes. A typical purification approach is the His-tag mediated purification system. The disadvantage of this system is that the tag remains attached to the enzyme after the purification and has to be cleaved afterwards. A further development of this system is the intein tag mediated purification (<A href="http://parts.igem.org/Part:BBa_K1465230" target="_blank">BBa_K1465230</A>).
By adding an intein tag attached to a chitin binding domain to the enzyme of interest a purification of the natural untagged protein can be achieved. The chitin binding domain binds the column at which chitin beads are stored. After adding binding buffers and washing solutions an elution with DTT allows the cleavage of the attached intein tag to the coding sequence. The enzyme is eluted from the column and can be stored in the desired buffer. The chitin binding domain and intein tag can be eluted from the column afterwards to reuse the column.

We implemented this system in the pSB1C3 backbone by combining the T7 promoter with RBS and intein tag with a chitin binding domain.</P><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:350px;margin-left:150px"><FONT size="2" style="text-align:center;"><B>Figure 17</B>: Scheme vor purification vector</FONT></DIV>
By designing Gibson Assembly primers with flanking overhangs it is possible to add a coding sequence between the first and the second part of the purification vector (add the gene specific part behind the overhang with the right orientation):<CENTER>&gt;GSP_fw
CTATAGGGAAAGAGGAGAAAT&gt;GSP_rev
CTAGTGCATCTCCCGTGATGCA
Note: The stop codon of the coding sequence has to be deleted through primer design.</CENTER>
It is perhaps possible to redesign the pSB1C3 backbone to the purification vector by including the T7 and RBS as well as the intein tag with chitin binding domain into the backbone. The restriction sites for BioBrick assembly may be placed in between both patterns. This would allow an in frame addition of the coding sequence by using BioBrick assembly.
Because of problems during the transformation of the coding sequences we were not able to characterize this BioBrick.</DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6>Summary</H6><P>
In the following subproject we were trying to transform the prkA and the glpX as well as enabling the gluco switch. It was possible to transform the prkA which could be verified by MALDI-TOF. With HPLC we were not able to show the functionality of the PrkA. Secondly we transformed the SBPase which could be verified by SDS-Page. The protein and additional proteins for an <I>in vitro</I> assay were purified via His-tag purification. The functionality could be shown via HPLC. A decrease of growth resulting from reducing the amount of available substrates for glycolysis was shown by comparative cultivation. Thirdly the gluco-switch with knock-down of the <I>pfk</I> was transformed. The expected decrease of growth could not be verified through cultivation.
</P></DIV></DIV><DIV id="main_menue" style="margin-left:135px; height:80px"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation" style="color:#000000"><DIV class="main_menueButton"><P class="buttoncenter"><FONT color="#FFFFFF">Overview</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation/RuBisCO" style="color:#000000"><DIV class="main_menueButton" style="width:130px"><P class="buttoncenter"><FONT color="#FFFFFF">RuBisCO</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Carboxysome" style="color:#000000"><DIV class="main_menueButton" style="width:130px"><P class="buttoncenter"><FONT color="#FFFFFF">Carboxysome</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle" style="color:#000000"><DIV class="main_menueButtonActual" style="width:130px"><P class="buttoncenter"><FONT color="#FFFFFF">Calvin-Cycle</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Measurement" style="color:#000000"><DIV class="main_menueButton" style="width:130px"><P class="buttoncenter"><FONT color="#FFFFFF">Measurement</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Outlook" style="color:#000000"><DIV class="main_menueButton" style="width:100px"><P class="buttoncenter"><FONT color="#FFFFFF">Outlook</FONT></P></DIV></A></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H4 class="member" style="margin-left:20px">References</H4><P><UL><LI id="stolzenberger2013"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
       Stolzenberger et al., 2013. Characterization of Fructose 1,6-Bisphosphatase and Sedoheptulose 1,7-Bisphosphate from the Facultative Ribulose Monophosphate Cycle Methylotroph <I>Bacillus methanolicus</I>. <A href="http://jb.asm.org/content/195/22/5112.long" target="_blank">Journal of Bacteriology</A>, Vol. 195, pp. 5112-5122
  </DIV></DIV></LI><LI id="bonacci2011"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
        Bonacci et al., 2011. Modularity of carbon-fixing protein organelle. <A href="http://www.pnas.org/content/109/2/478" target="_blank">PNAS</A>, vol. 109, pp. 478-483
  </DIV></DIV></LI><LI id="parikh2006"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
       Parikh et al., 2006. Directed evolution of RuBisCO hypermorphs through genetic selection in engineered <I>E. coli</I>. <A href="http://peds.oxfordjournals.org/content/19/3/113.long" target="_blank">Protein Engineering, Design &amp; Selection</A>, vol. 19, pp. 113-119
  </DIV></DIV></LI><LI id="hariharan1998"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Hariharan et al., 1998. Purification and characterization of phosphoribulokinase from the marine chromophytic alga <I>Heterosigma carterae</I>. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC35018/">Plant Physiol.</A> Vol. 117, pp. 321-329 
  </DIV></DIV></LI></UL></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle">http://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle&amp;oldid=392222" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>