"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-CeBiTec/Project/CO2-fixation/GeneticalApproach
From 2014.igem.org
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Home
Summary
Applications
Achievements
Bielefueli
Project
Results
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Modeling
Parts
Data Page
Achievements
Team
Members
Subteams
Pictures & Videos
Bielefeld & University
Official Team Profile
Acknowledgements
Contact
Policy & Practices
Experts
Interviews
NRW-Day
Pupils Acedemy
How To Wiki
namu
SYNENERGENE
Applications
Vignettes
Notebook
Journal
Protocols
Media
Kits & Enzymes
Strains & Constructs
Primer
Acronyms
Organisms
Additional
Partners
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Partners
Module II - Carbon Dioxide (CO2) Fixation
Overview
Modes of carbon fixation
Calvin cycle
Carboxysome
Genetical Approach
Results
Introduction
Our aim is to bind carbon dioxide in organic molecules using E. coli. There are many different pathways known for carbon dioxide fixation in bacteria. Most suitable for our application are the 3-hydroxypropionate bicycle and the reductive pentose phosphate cycle, also called Calvin cycle . We were looking for aerobic possibilities of carbon dioxide fixation, because of the desired energy source, because electric power should be the only energy source of our bacteria. Electric power should be the only source of energy for our bacteria. Therefore an aerobic cultivation is required. The electrons should pass through the bacterial respiratory chain to the final acceptor oxygen. Finally we decided to construct a bacterial version of the calvin cycle (Figure 1 and Figure 2) which is a well understood carbon dioxide fixation mechanism in plants (Winkel, 2004 ; Raines, 2003) . One reason for our choice was the existence of almost all essential enzymes in E. coli. Only the enzymes for three steps are missing. The implementation of the 3-hydroxypropionate bicycle would require the heterologous expression of genes for nearly all involved enzymes. All in all this would sum up to at least 12 genes. Nevertheless we developed plans for the construction of both pathways in E. coli.
Figure 1: Pathway of the D-xylose consumption in E. coli for the fixation of carbon dioxide by the RuBisCO from Halothiobacillus neapolitnaus. For this approach the substrate ribulose 1,5-bisphosphate needs to be accumulated in the cell. This is done by the PrkA from Snyechoccous elongatus.
Figure 2: Schematic representation of the Calvin cylce. The reactions shown in green can be catalyzed by enzymes that naturally exist in E. coli, while the red ones need to be expressed heterologous to enable the whole Calvin cycle in E. coli.
Sedoheptulose-1,7-bisphosphatase
The first missing enzyme of the Calvin cycle in E. coli is the sedoheptulose-1,7-bisphosphatase (SBPase). While this enzyme is common in plants only a few bacterial versions of it are known. One bacterial SBPase was recently identified and characterized ( Stolzenberger et al. in 2013 ). The origin of this sequence is the thermophilic Bacillus methanolicus , which lives at 50°C. Therefore the temperature optimum of this enzyme is also at 50°C ( Stolzenberger et al., 2013 ). We would like to use this enzyme at 37°C in E. coli. It had not escaped our notice that this temperature difference could cause a significant reduction in activity ( Peterson et al., 2007 ). To our best knowledge there are no better fitting bacterial enzymes available. Therefore we decided to clone the coding sequence of glpXC into pSB1C3. This BioBrick should then be combined with a strong promoter to enable a heterologous overexpression in E. coli. We selected the T7 ( BBa_I719005 ) and Ptac ( BBa_K731500 ) promoters for this task. The T7 is at least suitable for a simple overexpression, but could cause the building of inclusion bodies ( Tabor and Richardson, 1985 ). The resulting protein could be analyzed by SDS-PAGE and MALDI-TOF-MS. These methods are suitable to show the synthesis of GlpX. Additionally there is an enzyme assay available to validate the function of this protein ( Stolzenberger et al., 2013 ). The expression for this assay should be done under the control of the Ptac promoter.
Phosphoribulokinase
The next critical step of the Calvin cycle is the phosphorylation of ribulose-5-phosphate. This reaction is catalyzed by the phosphoribulokinase. E. coli lacks such an enzyme. Therefore a phosphoribulokinase gene (prkA) is taken from Synechococcus elongatus. This was successfully attempted before ( Parikh et al., 2006 ). The evolutionary distance between these two species is huge and the codon usage is different. Therefore we ordered the synthesis of a codong optimized sequence for E. coli( BBa_K1465201 ). The function of this enzyme could be analyzed by the toxicity of its product. Ribulose-1,5-bisphosphate is not metabolized in the wildtype of E. coli which leads to an at least inhibiting if not even toxic accumulation( Parikh et al., 2006 ). It is also possible to detect ribulose-1,5-bisphosphate via HPLC-ESI-MS. For both approaches is the expression in E. coli necessary. The coding sequence of prkA was placed under the control of the Ptac and T7 promoter respectively to achieve an overexpression ( BBa_K1465212 and pSB1A2-T7:prkA ).
RubisCO
The most important step is the actual carbon dioxide fixing reaction. It is catalyzed by ribulose-1,5-bisphosphate-carboxylase/oxygenase (RubisCO) in plants. The position of this reaction in the whole pathway is shown in figure 1.
We looked for a bacterial ezymes and selected two different RuBisCOs. One is an enzyme from the carboxysome of Halothiobacillus neapolitanus ( Bonacchi et al., 2011 ). The other one is from Synechococcus elongatus ( Long et al., 2007 ). We decided to work on both enzymes and to them regarding compatibility and efficiency.
First of all the coding sequences of the H. neapolitanis RubisCO ( Bonacchi et al., 2012 ) and of the S. elongatus RubisCO should be cloned into E. coli. The first characterization approach is an overexpression in E. coli. Therefore we designed the plasmid pSB1A2-T7:Hneap . The produced protein could be identified by SDS-PAGE separation of the protein extract and a following MALDI-TOF-MS analysis. We are interested in the difference of the protein extract between one induced and one uninduced culture.
To validate the enzyme function of the RubisCO it should be coexpress with the phosphoribulokinase. The toxicity of the PrkA should be avoided by the activity of the RubisCO, because the produced ribulose-1,5-bisphosphate is no longer a metabolic dead end. The RubisCO should use this metabolite as substrate and thereby close this pathway. We designed BBa_K1465214 (RubisCO and PrkA) for the coexpression of BBa_1465201 (PrkA) and BBa_K1465202 (RubisCO) .
The removal of ribulose-1,5-bisphosphate by the RubisCO implies another method to demonstrate the function of this heterologous expressed protein. An protein extract from the wild type should not degrade ribulose-1,5-bisphosphate as mentioned before. If our RubisCO is functional in E. coli we expect the removal of this substance. BBa_K146213 was designed for an heterologous expression of the RubisCO in E. coli.
The RubisCO from H. neapolitanus is usually located in a special microcompartiment called "carboxysome"( Bonacchi et al. in 2011 ). Therefore we decided to construct a carboxysome in E. coli.
We would like to compare the enzymatic activity inside the carboxysome and in the cytoplasm. The separation of enzymes from both compartments could be achieved by ultracentrifugation and a following enzyme assay.
Carboxysome
The carboxysome ensures an efficient carbon fixation reaction. It separates the RubisCO from the aerobic cytoplasm. The concentration of carbon dioxide is selectively increased inside the carboxysome ( Reinhold et al., 1991 ; Dou et al., 2008 ; Mangan and Brenner, 2014 ). We selected the carboxysome encoding plasmid pHnCBS1D from H. neapolitanus as template for our synthetic construct.
The gene cluster of the carboxysome contains many illegal restriction sites. Therefore we decided to order some synthesized sequences to replace special parts of pHnCBS1D. This synthesis is not just used to remove the illegal restriction sites, but also allowed an optimization of the codon usage for expression in E. coli.
The assembly should be started at the downstream end of the carboxysome operon. We would like to analyze the effect of csoS1D on the carboxysome. The construction will be done in parallel with and without csoS1D to achieve this aim. The illegal restriction sites of csoS1CAB and csoS4AB will be removed by PCR. We would like to construct pSB1C3-csoS4AB-csoS1CAB ( BBa_1465206 ) from four different pcr products in a single Gibson-assembly. In the next step csoS1D will be added from pSB1C3-csoS1D ( BBa_K1465207 ) by suffix assembly.
The next step is the addition of csoS3, which encodes the carbonic anhydrase, to the existing construct pSB1C3-csoS4AB-csoS1CAB with and without csoS1D by prefix assembly.  Afterwards is csoS2, which encodes the shell associated protein, added to the existing construct BBa_K1465208 by prefix assembly. The resulting construct should encode a complete carboxysome ( Bonnachi et al, 2012 ).
The complete carboxysome encoding plasmid could subsequently be combined with the RubisCO encoding sequence and prkA under the control of a strong and induvible promoter e.g. Ptac. Therefore we will use a prefix assembly. This step would produce a large plasmid, which encodes all proteins needed for carbon dioxide fixation in E. coli.
To demonstrate the correct carboxysome assembly on protein level csoS1A should be translational fused with gfp. The fluorescence of GFP can be used to indicate the correct folding of a protein ( Waldo et al., 1999 ; Hsu et al., 2009 ). Therefore we designed BBa_K1465222 . We would like to analyze the function of CsoS2 in the carboxysome assembly by attempting an assembly without it. The construction of BBa_K1465223 was necessary for a comparison. The fluorescence from GFP can be quantified by photometric measurements. The correct assembly of the entire carboxysome could be visualized by fluorescence microscopy.
Phosphofructokinase / Fructose-1,6-bisphosphatase
The activity of fructose-1,6-bisphosphatase (FBPase) and phosphofructokinase (PFK) forms a futile cycle. The PFK consumes ATP to phosphorylate the fructose-6-phosphate while the FBPase removes the phosphate. The regulation of both enzymes is well understood in plants ( Michelet et al., 2013 ). E. coli is strict heterotroph so there is no need to switch between oxidative and reductive pentose phosphate pathway. Usually the glycolysis is active, which should be changed. Therefore a reduction of the phosphofructokinase activity to a minimum is necessary. To avoid the complications of chromosomal modifications a knock-down approach was chosen. There are two PFK encoding genes in E. coli, which are called pfkA ( Hellinga & Evans, 1985 ) and pfkB ( Daldal, 1984 ). The PfkA accounts for 90% of the phosphofructokinase activity in E. coli ( Hellinga & Evans, 1985 ). Hence we will focus on the knock down of the PfkA expression.
We ordered the synthesis of two sequences, which are complementary to the start of the corresponding pfk transcripts. The in vivo transcription of these sequences will produce small regulatory RNAs (sRNAs), which will knock down the expression of PfkA (sRNA-pfkA and sRNA-pfkB) respectively. Both sequences include a strong terminator downstream of the complementary sequence to avoid unnecessary transcription. The designed constructs for pfkA are pSB1C3-sRNApfkA ( BBa_K1465225 ), the sequence under control of the Ptac promoter ( BBa_1465226 ) and the sequence under control of the T7 promoter ( BBa_1465227 ). These constructs could be used in growth experiments to compare their impact on the cellular metabolism. Their effects could be compared to a knockout mutant of the pfkA ( JW3887-1 ).
Additionally the overexpression of an FBPase encoding sequence would be useful. The overexpression of glpX is already part of this project. GlpX is actually has a strong FBPase activity and additionaly a weak SBPase activity ( Stolzenberger et al., 2013 ). If there is enough of this enzyme to cover the need for an SBPase there will also be enough for the FBPase function. In addition we could use the endogenous FBPase from E. coli.
Summary
If it is possible to enable the whole cycle in E. coli the cells should be able to grow with carbon dioxide as the sole carbon source. Nevertheless there is still a need for an energy source. We would like to address this issue by inventing an electroautotrophic E. coli strain. As a backup we think of feeding a pentose (e.g. xylose) to support the Calvin cycle in case the efficiency of the newly created electroautotrophic metabolism is not high enough.
To engineer an autotrophic E. coli strain we need to fullfill the following tasks:
heterologous expression of glpXC from B. methanolicus in E. coli
heterologous expression of prkA from S.elongatus in E. coli
construction of a plasmid encoding a carboxysome
production of a functional carboxysome in E. coli
heterologous expression of an RubisCO in E. coli
knock down of pfkA to avoid an futile cycle
invention of an energy feeding mechanisms (e.g. electric )
Overview
Modes of carbon fixation
Calvin cycle
Carboxysome
Genetical Approach
References
Winkel, B. S. J. „Metabolic Channeling in Plants“. Annual Review of Plant Biology 55 (2004): 85–107. doi:10.1146/annurev.arplant.55.031903.141714.
Raines, C. A. „The Calvin Cycle Revisited“. Photosynthesis Research 75, Nr. 1 (2003): 1–10. doi:10.1023/A:1022421515027.
Stolzenberger, Jessica, Steffen N. Lindner, Marcus Persicke, Trygve Brautaset, und Volker F. Wendisch. „Characterization of Fructose 1,6-Bisphosphatase and Sedoheptulose 1,7-Bisphosphatase from the Facultative Ribulose Monophosphate Cycle Methylotroph Bacillus methanolicus“. Journal of Bacteriology 195, Nr. 22 (2013): 5112–22. doi:10.1128/JB.00672-13.
Peterson, Michelle E., Roy M. Daniel, Michael J. Danson, und Robert Eisenthal. „The dependence of enzyme activity on temperature: determination and validation of parameters“. Biochemical Journal 402, Nr. Pt 2 (1. März 2007): 331–37. doi:10.1042/BJ20061143.
Bonacci, Walter, Poh K. Teng, Bruno Afonso, Henrike Niederholtmeyer, Patricia Grob, Pamela A. Silver, und David F. Savage. „Modularity of a Carbon-Fixing Protein Organelle“. Proceedings of the National Academy of Sciences 109, Nr. 2 (1. Oktober 2012): 478–83. doi:10.1073/pnas.1108557109.
Long, Benedict M., Murray R. Badger, Spencer M. Whitney, und G. Dean Price. „Analysis of Carboxysomes from Synechococcus PCC7942 Reveals Multiple Rubisco Complexes with Carboxysomal Proteins CcmM and CcaA“. Journal of Biological Chemistry 282, Nr. 40 (10. Mai 2007): 29323–35. doi:10.1074/jbc.M703896200.
Reinhold et al., 1990. A model for inorganic carbon fluxes and photosynthesis in cyanobacterial carboxysomes. Canadian Journal of Botany , vol. 69, pp. 984-988
Dou, Zhicheng, Sabine Heinhorst, Eric B. Williams, C. Daniel Murin, Jessup M. Shively, und Gordon C. Cannon. „CO2 Fixation Kinetics of Halothiobacillus Neapolitanus Mutant Carboxysomes Lacking Carbonic Anhydrase Suggest the Shell Acts as a Diffusional Barrier for CO2“. The Journal of Biological Chemistry 283, Nr. 16 (18. April 2008): 10377–84. doi:10.1074/jbc.M709285200.
Mangan et al., 2014.„Systems analysis of the CO2 concentrating mechanism in cyanobacteria“. eLife .vol. 3.
Michelet et al., 2013.Redox regulation of the Calvin-Benson cycle: something old, something new. Front Plant Sci 4.
Tabor & Richardson, 1985.A bacteriophage T7 RNA polymerase/promoter system for controlled exclusive expression of specific genes. PNAS 82, 1074–1078.
Daldal, 1984.Nucleotide sequence of gene pfkB encoding the minor phosphofructokinase of Escherichia coli K-12. Gene 28, 337–342.
Hsu et al., 2009.The folding, stability and conformational dynamics of beta-barrel fluorescent proteins. Chem. Soc. Rev. 38, 2951–2965.
Waldo et al., 1999.Rapid protein-folding assay using green fluorescent protein. Nat. Biotechnol. 17, 691–695.
Retrieved from " http://2014.igem.org/Team:Bielefeld-CeBiTec/Project/CO2-fixation/GeneticalApproach "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
