"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-CeBiTec/Results/CO2-fixation/Carboxysome
From 2014.igem.org
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Modeling
Parts
Data Page
Achievements
Home
Summary
Applications
Achievements
Bielefueli
Project
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Results
Team
Members
Subteams
Pictures & Videos
Bielefeld & University
Official Team Profile
Acknowledgements
Contact
Policy & Practices
Experts
Interviews
NRW-Day
Pupils Acedemy
How To Wiki
namu
SYNENERGENE
Applications
Vignettes
Notebook
Journal
Protocols
Media
Kits & Enzymes
Strains & Constructs
Primer
Acronyms
Organisms
Additional
Partners
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Partners
Module II - Carbon Dioxide (CO2) Fixation
Overview
RuBisCO
Carboxysome
Calvin-Cycle
Measurement
Outlook
Introduction
Naturally occuring, there are plasmids which encode different proteins of the carboxysome in bacteria. One such plasmid is pHnCBS1D which was found in Halothiobacillus neapolitanus ( Cannon and Shively, 1983 ). We used different parts of this plasmid for the production of BioBricks. These BioBricks (BBa_K1465202-BBa_K1465223) were used for the construction of a synthetic carboxysome-encoding plasmid. It consists up of the shell proteins CsoS1ABC and CsoS4AB and the shell associated protein CsoS2. Thereby, the proteins should form an anaerobic microcompartment. This bottom up approach allows verifying the essentiality of the different components. We used a translational fusion of one of the shell proteins CsoS1A and GFP ( BBa_K1465222 ) as an indicator of correct protein folding. A concentrated subcellular localisation of the gfp-fluorescence shows the positions of carboxysomes within the cells. This reporter function of GFP was identified and used for this purpose before ( Waldo et al., 1999 ; Hsu et al., 2009 ).
The expression of BBa_K1465223 (pSB1C3-T7:sap-csoS4AB-csoS1CA:gfp-csoS1B) in E. coli leads to the assembly of carboxysomes . We used a translational fusion of one shell protein (CsoS1A) with the CDS of the green fluorescent protein as an indicator of correct protein folding. The observed fluorescence in the carboxysome-expressing E. coli cells is concentrated at different points of the cell and indicates the presence of functional carboxysomes.
Shell protein folding
With our GFP fusion we wanted to analyze the protein folding of our constructed microcompartment, the carboxysome. On the one hand we had a GFP fusion with the shell proteins. On the other hand we analyzed the GFP fusion of the microcompartment with the additional shell associated protein encoded by csoS2 upstream of the shell proteins on the plasmid. The exact function of this protein is still unknown ( Yeates et al., 2008 ).
Figure 1: Fluorescence measurement of E. coli KRX wild type (negative control) and E. coli KRX carrying one of the three different plasmids pSB1C3_pTet_gfp (positive control), pSB1A2_T7_csoS4AB_csoS1CA_gfp_csoS1B and pSB1C3_T7_csoS2_csoS4AB_csoS1CA_gfp_csoS1B. The measurement has been done with the GloMax® Discover Multimode-Reader .
The fluorescence shown in Figure 1 was quantified by photometric measurements . The signal intensity of the E. coli KRX wild type (negative control) is compared to cells carrying a plasmid without csoS2 named T7_csoS14_GFP ( BBa_K1465222 ). E. coli KRX cells with pSB1C3-pTet_gfp ( BBa_I13522 ) serve as a positive control and lead to the highest fluorescence signal. The expression of csoS1A_gfp with all other known essential shell proteins ( BBa_K1465222 ) does not yield any fluorescence. This indicates that the presence of the shell associated protein is needed for the correct folding of the shell proteins and the assembly of the whole carboxysome. The combined expression of csoS2 and the coding sequences of the shell proteins csoS1CAB and csoS4AB ( BBa_K1465223 ) yields green fluorescence. From these results we deduce that the correct folding of the shell proteins is dependent on the presence of the shell associated protein CsoS2.
Carboxysome assembly
To show the successful assembly of the carboxysome we analyzed the subcellular localization of the green fluorescent protein. Pictures taken through a fluorescence microscope are shown in Figure 2.
Figure 2: Fluorescence pictures taken through a Leica DMI6000 microscope. (A) E. coli wild type KRX (negative control), (B) Expression of E. coli KRX with plasmid pSB1C3_pTet_gfp (positive control), (C) Expression of E. coli KRX with plasmid pSB1A2_T7_csoS4AB_csoS1CA_gfp_csoS1B, (D) and (E) Expression of E. coli KRX with plasmid pSB1C3_T7_csoS2_csoS4AB_csoS1CA_gfp_csoS1B.
The negative control with the wildytpe of E. coli in picture A does not yield any fluorescence just as well as the sample with the plasmid pSB1A2_T7_csoS4AB_csoS1CA_gfp_csoS1B in picture C. Regarding this we can conclude that the carboxysome cannot be folded correctly without the shell associated protein CsoS2. In picture B, the sample with the positive control, the whole cells cause fluorescence. The E. coli bacteria carrying the whole carboxysome plasmid with the GFP fusion are shown in picture D and E. As expected not the whole cells are fluorescing but only a small part of the cells. We assume that the concentrated green fluorescence results from GFP molecules which are fused to the shell proteins of a successfully assembled carboxysome.
To get a higher resolution of the carboxysomes in the cells we also used a structured illumination microscope . The pictures are shown in Figure 3.
Figure 3: Fluorescence pictures taken through a structured illumination microscope . (A) E. coli wild type KRX (negative controls), (B) Expression of  coli KRX with plasmid pSB1C3_pTet_gfp (positive control), (C) Expression of E. coli KRX with plasmid pSB1A2_T7_csoS4AB_csoS1CA_gfp_csoS1B, (D) and (E) Expression of E. coli KRX with plasmid pSB1C3_T7_csoS2_csoS4AB_csoS1CA_gfp_csoS1B. With this higher resolution we could identify the carboxysomes in the cells with the pSB1C3_T7_csoS2_csoS4AB_csoS1CA_gfp_csoS1B plasmid in one specific location of the cell (picture D and E). Most of the cells seem to have only one carboxysome but there were also some that carried two. Additionally we could detect some cells forming a filament located at the membrane of the cell (Figure 4). These filaments might be formed because of a high induction of the plasmid ( Bonacchi et al., 2012 ). As we used the strong T7 promoter this could be the reason for this occurrence.
Figure 4: Fluorescence picture of a single E. coli carrying the pSB1C3_T7_csoS2_csoS4AB_csoS1CA_gfp_csoS1B plasmid and forming a filament. Picture taken through a structured illumination microscope .
Figure 5: Fluorescence picture of a single E. coli carrying the pSB1C3_T7_csoS2_csoS4AB_csoS1CA_gfp_csoS1B plasmid. Picture taken through a structured illumination microscope .
In Figure 5 we tried to find out the size of the carboxysome in a cell. The microcompartment has a size of ~0.3 µm. This value confirms the data given in the paper Bonacchi et al., 2012 where they assumed a size of approximately ~0.136 µm (+/- 0.036).
The assembly of the carboxysome was possible without the expression of csoS1D, another coding sequence which is located on pHnCBS1D . The resulting protein is probably responsible for the pore size in the carboxysome envelope ( Kinney et al., 2011 ; Bonacchi et al., 2012 ). Nevertheless it seems that it has no essential function in protein folding.
Carboxysome purification
Purification of recombinant proteins is one essential foundation of synthetic biology. For this reason, it was our first approach to purify the carboxysome from the chemolitoautotroph model organism Halothiobacillus neapolitanus , recombinant expressed in E. coli . After establishment of the method, our constructed carboxysome should be purified in this way. The work for purifying carboxysomes was nevertheless based on the plasmid pHnBCS1D, which is coding for the H. neapolitanus carboxysome. For cultivation and recombinant expression of the carboxysome, we followed the method for cultivation described in Bonacchi et al., 2012 and the method for purification described by So et al., 2004 . Different experiments were done, containing changes in the experimental setup, which is described everytime in our labjournal .
To proof the expression of the carboxysomal proteins, we analyzed recombinant protein expression by SDS-PAGE and MALDI-TOF . The results of the SDS-PAGE are shown in Figure 6-8:
Figure 6: Proteinexpression of pHnCBS1D
induced with 0.5 mM IPTG
Figure 7: Proteinexpression of pHnCBS1D
induced with 2 mM IPTG
Figure 8: Proteinexpression of pHnCBS1D
induced with 5 mM IPTG
The SDS-PAGE shows the cellular proteins at different time points during the cultivation. We can recognize that there is 12 hours after induction a new band somewhat smaller than 55 kDa. Analysis via MALDI-TOF identified this band as the large subunit of the RuBisCO enzyme (52.6 kDa) from Halothiobacillus neapolitanus . The analysis was done via tryptic digestion in silico and alignment of the identified peptides with the in silico peptides. Four identical peptides could be found and the sequence coverage (MS) was 8.9 %. After 12 h protein expression was induced with 0.5 mM IPTG and a band with a size of probably 13 kDa increases. MALDI-TOF displayed that this is the small subunit of the RuBisCo (12.8 kDa). It was identified via in silico digest and alignment. Two peptides were found and the sequence coverage (MS) was 28.2. When IPTG was added to 2 or 5 mM, it seems that the intensity of the band of the small subunit decreases in time of cultivation. We hypothesize from this that the small subunit is degraded. Inconsistent to this hypothesis is the increase of the large subunit. Degradation of the small subunit is not described in the literature. The fact that RuBisCo composes nearly 70 % of the protein mass (by weight) in the carboxysome could explain why other proteins from the carboxysome could not be found in this SDS-PAGE.
The purification of the carboxysome is carried out via several centrifugation steps. The last step of this method is a sucrose gradient centrifugation. This method did not work at all. Unfortunately, we were not able to correct the limiting step.
Two different methods for cell lysis were tried, sonication and french press lysation. In the execution of the method it was ensured, that the mechanical stress for the cells could not destroy the microcompartments. The following centrifugation steps seemed to be successful, as the supernatants after centrifugation was analyzed by SDS-PAGE, showing the expected result. After the first centrifugation step in order to separate the cell debris, the supernatant carried the soluble proteins. After pelleting the carboxysomes, the supernatant of both centrifugation steps contained a low amount of protein.
The sample, which was layered on top of the sucrose gradient, had a slightly blue color, and it builds up a layer on the sucrose gradient, indicating that the gradient seems to be correctly formed. After ultracentrifugation there was no visible band of detectable carboxysomes. Unfortunately, we had no access to the equipment for scanning such gradients. SDS-PAGE of the fractionated gradient showed protein in every fraction, indicating that there was not enough concentrating force for concentration of the carboxysomes. A longer centrifugation period did not show a satisfactory result.
Summary and outlook
We report the construction of a synthetic plasmid encoding a functional carboxysome. Despite the fact that it has not shown the fixation of carbon dioxide in this compartment yet, we would like to emphasize the construction of a useful microcompartment. The functional carboxysome could be used for the carbon dioxide fixation. Therefore, three enzymes need to be heterologous expressed in E. coli . We already tried to combine the carboxysome encoding genes and the three required enzymes on a single plasmid. The carbonic anhydrase (csoS3) should be cloned between the shell associated protein and the shell proteins. One RubisCO ( BBa_K1465202 ) and one phosphoribulokinase ( BBa_K1465201 ) encoding sequence should be integrated upstream of the carboxysomal operon.
Different microscopy approaches showed a distinct fluorescence spot inside the cell. In contradiction the positive control showed a fluorescence throughout the whole cell. By doing this microscopy pictures we could identify fully assembled carboxysomes as well as filaments which result from the overexpression through the T7 promoter as described before.
The complete carboxysome could be used for anaerobic reactions in aerobic cultures of E. coli . One possible application beside the carbon dioxide fixation could be the fixation of atmospheric nitrogen which is essential for the growth of plants. With a size of approximately ~100-300 nm, all desired proteins that are smaller can be brought into our microcompartment for further experiments.
Overview
RuBisCO
Carboxysome
Calvin-Cycle
Measurement
Outlook
References
Cannon, Shively 1983. Characterisation of a Homogenous Preparation of Carboxysomes from Thiobacillus neapolitanus. Archive of Microbiology , vol. 134, pp. 52-59
Waldo, G. S., B. M. Standish, J. Berendzen, und T. C. Terwilliger. „Rapid Protein-Folding Assay Using Green Fluorescent Protein“. Nature Biotechnology 17, Nr. 7 (Juli 1999): 691–95. doi:10.1038/10904.
Hsu, Shang-Te Danny, Georg Blaser, und Sophie E. Jackson. „The Folding, Stability and Conformational Dynamics of Beta-Barrel Fluorescent Proteins“. Chemical Society Reviews 38, Nr. 10 (2009): 2951–65. doi:10.1039/b908170b.
Bonacci, Walter, Poh K. Teng, Bruno Afonso, Henrike Niederholtmeyer, Patricia Grob, Pamela A. Silver, und David F. Savage. „Modularity of a Carbon-Fixing Protein Organelle“. Proceedings of the National Academy of Sciences 109, Nr. 2 (1. Oktober 2012): 478–83. doi:10.1073/pnas.1108557109.
Kinney, James N., Seth D. Axen, und Cheryl A. Kerfeld. „Comparative analysis of carboxysome shell proteins“. Photosynthesis Research 109, Nr. 1–3 (September 2011): 21–32. doi:10.1007/s11120-011-9624-6.
Yeates et al., 2011. Protein-based organelles in bacteria: carboxysomes and related microcompartments. Nature Reviews Microbiology , vol. 6, pp. 681-691
Retrieved from " http://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Carboxysome "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
