"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-CeBiTec/Results/AdhA
From 2014.igem.org
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Modeling
Parts
Data Page
Achievements
Home
Summary
Applications
Achievements
Bielefueli
Project
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Results
Team
Members
Subteams
Pictures & Videos
Bielefeld & University
Official Team Profile
Acknowledgements
Contact
Policy & Practices
Experts
Interviews
NRW-Day
Pupils Acedemy
How To Wiki
namu
SYNENERGENE
Applications
Vignettes
Notebook
Journal
Protocols
Media
Kits & Enzymes
Strains & Constructs
Primer
Acronyms
Organisms
Additional
Partners
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Partners
Module III - Isobutanol production
Overview
Alcohol dehydrogenase
Isobutanol pathway
Outlook
Alcohol dehydrogenase (AdhA)
During our literature research about the isobutanol production pathway we found out, that the alcohol dehydrogenase from Lactococcus lactis (AdhA) is more efficient than the one from Escherichia coli ( Atsumi et al., 2008 , Atsumi et al., 2010 ). For that reason we wanted to increase the production of isobutanol by cloning the adhA gene behind our production pathway. The adhA from L. Lactis was not  available as a BioBrick so we designed a new part which contains the coding sequence ( BBa_K1465301 ). You can read more about our approach in the following sections.
Cloning
For the isolation of the adhA gene from L. lactis we first had to isolate the genomic DNA . Afterwards we amplified the adhA gene with the primer rev_pSB1C3_adhA and fw_adhA_pSB1C3 . Via these primer the ahdA gene was combined with the RBS BBa_B0034 . Together with the amplified backbone we performed a Gibson Assembly . We verified the correct insertion of the adhA connected with RBS in the backbone pSB1k3 via colony PCR and sequencing. Please note, that due to difficulties during the cloning process, we first did an intermediate cloning step into the pSB1K3 backbone. For that reason we performed a successful recloning in pSB1C3 which lead to the pSB1C3_adhA.This is our part BBa_K1465301 which is ready for usage.
For the characterization of our BioBrick BBa_K1465301 we needed a promoter for expression and therefore performed a BioBrick Suffix Assembly with pSB1A2_T7 ( BBa_I719005 ) and our part. We used pSB1A2_T7 because still ongoing cloning difficulties with the pSB1C3. Nevertheless, we wanted to have the new created part in pSB1C3 too, so a successful recloning in pSB1C3 was done resulting in pSB1C3_T7_adhA. The new part can now be found as the BioBrick BBa_K1465304 .
Additionally we performed a BioBrick Prefix Assembly with the pSB1C3_Ptac ( BBa_K731500 ) and our part. The successfully created part pSB1C3_Ptac_adhA can now also be found as the BioBrick BBa_K1465305 .
Expression
For the protein expression analysis of AdhA we performed a cultivation of E. coli KRX with our inducible construct pSB1A2_T7_adhA. Samples were taken like explained in the cell lysis for a SDS-PAGE Protocol . Protein expression was induced with rhamnose when the culture reached an OD600 of 0,8. The first sample was taken right before the induction. Additionally we took samples one, two, three and 20 hours after the induction. With these samples, we performed a SDS Page (Figure 1).
Figure 1: SDS page from pSB1A2_T7_adhA.
The mass of the overexpressed protein AdhA is 35,776 Da
One can see that the size of the band at a mass of ~ 38 kD increases the later the sample was taken. There is a conspicuously difference between the size of the bands at the point of induction and 20 hours later. This is an  indication for the successful overexpression of the AdhA (35,776 Da).
Analytics via MALDI-TOF identified the band at ~ 38 kD as the alcohol dehydrogenase (AdhA) from the organism L. lactis. The analysis was done via tryptic digestion and comparison to an in silico computed peptide mass pattern. Three matching peptid masses could be found and the sequence coverage (MS) was 13.3 %. The sequence coverage (MS/MS) was 4.0 %. Analyzing a second sample (the band was cut into two pieces), two identical bands were found with a sequence coverage of 8.4 % (MS) and 4.0 % (MS/MS).
These considerations lead to the conclusion that the AdhA was successfully overexpressed in E. coli KRX.
Conclusion
We successfully created three new BioBricks ( BBa_K1465301 , BBa_K1465304 and BBa_K1465305 ). We also could demonstrate the successful overexpression of the AdhA from L. lactis in E.coli via SDS Page and follow-up MALDI-TOF analysis.
This part is used in our isobutanol production pathway . An alcohol dehydrogenase is required for the conversion of isobutyraldehyde into isobutanol.
Overview
Alcohol dehydrogenase
Isobutanol pathway
Outlook
References
Atsumi S, Hanai T, Liao JC., 2008. Non-fermentative pathways for synthesis of branched-chain higher alcohols as biofuels. In: Nature 451 , 86–89.
Atsumi S, Wu TY, Eckl EM, Hawkins SD, Buelter T, Liao JC. 2010. Engineering the isobutanol biosynthetic pathway in Escherichia coli by comparison three aldehyde reductase/alcohol dehydrogenase genes. In: Appl. Microbiol. Biotechnol 85 , 651–657
Retrieved from " http://2014.igem.org/Team:Bielefeld-CeBiTec/Results/AdhA "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
