<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_rMFC_GeneticalApproach"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-CeBiTec/Project/rMFC/GeneticalApproach">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-CeBiTec/Project/rMFC/GeneticalApproach&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Project/rMFC/GeneticalApproach&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Project/rMFC/GeneticalApproach&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-CeBiTec/Project/rMFC/GeneticalApproach" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-CeBiTec/Project/rMFC/GeneticalApproach</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV style="height:100%" id="main_box"><DIV id="flag"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Overview" style="color:#000000"><FLAG class="flag_element1" style="margin-left:70px;"><P class="flag"><FONT color="#FFFFFF">Overview</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Theory" style="color:#000000"><FLAG class="flag_element2"><P class="flag"><FONT color="#FFFFFF">rMFC</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/CO2-fixation/CarbonDioxide" style="color:#000000"><FLAG class="flag_element3"><P class="flag"><FONT color="#FFFFFF">CO<SUB>2</SUB> Fixation</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Isobutanol/Theory" style="color:#000000"><FLAG class="flag_element4"><P class="flag"><FONT color="#FFFFFF">Isobutanol</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Biosafety" style="color:#000000"><FLAG class="flag_element5"><P class="flag"><FONT color="#FFFFFF">Biosafety</FONT></P></FLAG></A></DIV><UL id="nav"><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec"><FONT color="#FFFFFF">Home</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/ProjectSummary"><FONT color="#FFFFFF">Summary</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/Applications"><FONT color="#FFFFFF">Applications</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Achievements"><FONT color="#FFFFFF">Achievements</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Bielefueli"><FONT color="#FFFFFF">Bielefueli</FONT></A></LI></UL><LI><ACTUAL><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Overview"><FONT color="#FFFFFF">Project</FONT></A></ACTUAL></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Overview"><FONT color="#FFFFFF">Results</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Overview"><FONT color="#FFFFFF">Overview</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC"><FONT color="#FFFFFF">rMFC</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation"><FONT color="#FFFFFF">CO<SUB>2</SUB> Fixation</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Isobutanol"><FONT color="#FFFFFF">Isobutanol</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Biosafety"><FONT color="#FFFFFF">Biosafety</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Modelling"><FONT color="#FFFFFF">Modeling</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Parts"><FONT color="#FFFFFF">Parts</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/DataPage"><FONT color="#FFFFFF">Data Page</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Achievements"><FONT color="#FFFFFF">Achievements</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team"><FONT color="#FFFFFF">Team</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team/Members"><FONT color="#FFFFFF">Members</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team/Subteams"><FONT color="#FFFFFF">Subteams</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/PicturesAndVideos"><FONT color="#FFFFFF">Pictures &amp; Videos</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/BielefeldAndUniversity"><FONT color="#FFFFFF">Bielefeld &amp; University</FONT></A></LI><LI><A href="https://igem.org/Team.cgi?year=2014"><FONT color="#FFFFFF">Official Team Profile</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Acknowledgements"><FONT color="#FFFFFF">Acknowledgements</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Contact"><FONT color="#FFFFFF">Contact</FONT></A></LI><LI style="width:155px;"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice"><FONT color="#FFFFFF">Policy &amp; Practices</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Experts"><FONT color="#FFFFFF">Experts</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/RadioInterview"><FONT color="#FFFFFF">Interviews</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/NRW-day"><FONT color="#FFFFFF">NRW-Day</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/ScholarAcademy"><FONT color="#FFFFFF">Pupils Acedemy</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Wikisession"><FONT color="#FFFFFF">How To Wiki</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Namu"><FONT color="#FFFFFF">namu</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene"><FONT color="#FFFFFF">SYNENERGENE</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/Applications"><FONT color="#FFFFFF">Applications</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/TechnomoralScenario"><FONT color="#FFFFFF">Vignettes</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook"><FONT color="#FFFFFF">Notebook</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Journal"><FONT color="#FFFFFF">Journal</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols"><FONT color="#FFFFFF">Protocols</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media"><FONT color="#FFFFFF">Media</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes"><FONT color="#FFFFFF">Kits &amp; Enzymes</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs"><FONT color="#FFFFFF">Strains &amp; Constructs</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Primer"><FONT color="#FFFFFF">Primer</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Acrynom"><FONT color="#FFFFFF">Acronyms</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms"><FONT color="#FFFFFF">Organisms</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Safety"><FONT color="#FFFFFF">Additional</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners"><FONT color="#FFFFFF">Partners</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/GoldSponsor"><FONT color="#FFFFFF">Gold Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/SilverSponsor"><FONT color="#FFFFFF">Silver Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/BronzeSponsor"><FONT color="#FFFFFF">Bronze Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/Partners"><FONT color="#FFFFFF">Partners</FONT></A></LI><H1> Module I - Reverse Microbial Fuel Cell (rMFC) </H1><DIV id="main_menue" style="margin-left:130px; height:80px"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Theory" style="color:#000000"><DIV class="main_menueButton" style="width:100px"><P class="buttoncenter"><FONT color="#FFFFFF">Overview</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/GeneticalApproach" style="color:#000000"><DIV class="main_menueButtonActual" style="width:180px"><P class="buttoncenter"><FONT color="#FFFFFF">Genetical Approach</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/ReactorSystem" style="color:#000000"><DIV class="main_menueButton" style="width:150px"><P class="buttoncenter"><FONT color="#FFFFFF">Reactor system</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/MeasurementSystem" style="color:#000000"><DIV class="main_menueButton" style="width:200px"><P class="buttoncenter"><FONT color="#FFFFFF">Measurement System</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Mediators" style="color:#000000"><DIV class="main_menueButton" style="width:100px"><P class="buttoncenter"><FONT color="#FFFFFF">Mediators</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC" style="color:#000000"><DIV class="main_menueButton" style="width:90px; float:right;margin-right:140px"><P class="buttoncenter"><FONT color="#FFFFFF">Results</FONT></P></DIV></A></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6 id="GeneticalApproach" style="font-size:30px">Genetical approach</H6><P><TABLE align="center" cellspacing="10&quot;" style="background-color:transparent; float:right"><TBODY><TR><TD><DIV class="element" style="margin:10px; padding:10px; text-align:center; float:right"><FONT size="2" style="text-align:center;"><B>Figure 1</B>: Module 1 overview</FONT></DIV></TD></TR></TBODY></TABLE>
Our first module deals with the construction of an <I>E. coli</I> strain, which is able to accept electrons stimulating its metabolism. We considered two different electron transfer systems: <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Theory#DirectElectronTransfer">direct</A> and <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Theory#IndirectElectronTransfer">indirect electron transfer</A>.  

Direct electron transfer in bacteria is a very complex and not completely cleared up so far. Because of this we focused on the indirect electron transfer via <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Mediators">mediators</A>, which are reduced at the electrode in a <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/ReactorSystem#DesignReactorSystem">electrobiochemical reactor</A> and then reoxidized again inside bacterial cells. 
In the gram-negative bacteria <I>E. coli</I> there are two membranes with a periplasmatic space between them, which have to be overcome in order to achieve a successful electron transfer.
We worked on three different mediators: <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Mediators#NR">neutral red</A>, <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Mediators#BPB">bromphenol blue</A> and <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Mediators">Cytochromes</A>. Additionally we constructed an electrophilic <I>E. coli</I> strain, which grows in the presence of electric current and therefore shows increased metabolic activity. (<A href="#Park1999">Park <I>et al.</I>, 1999</A>)
To realize this aim, several steps and problems in electron transfer have to be resolved. The electron transfer system consists of different steps: 
First the reduced mediator has to cross the outer membrane of the <I>E. coli</I> cell. For that we are going to use the outer membrane porine OprF (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_K1172507">BBa_K1172507</A>) provided by <A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Porins">iGEM Team Bielefeld-Germany 2013</A>. 
Crossing the periplasmatic space, the mediator adsorbs at the inner membrane of the <I>E. coli</I> cell. Electrons have to be transferred into the cytoplasm, but the mediator should not enter the cell, because regeneration at the electrode is absolutely necessary. So membrane proteins are required to transfer electrons trough the inner membrane. We looked for different oxidoreductase systems located in bacterial respiration system (shown in figure 3).

<TABLE align="left" cellspacing="10&quot;" style="background-color:transparent; float:left"><TBODY><TR><TD><DIV class="element" style="margin:10px; padding:10px; text-align:left; float:left"><FONT size="2" style="text-align:left;"><B>Figure 2</B>: 
The electron flow mediated by redox active mediator 
(for example neutral red) in interaction with fumarate reductase 
in <I>E. coli</I> cell. The electrochemical reduced mediator has to cross 
outer membrane via genome integrated outer membrane porines OprF. 
The fumarate reductase in the inner bacterial membrane get electrons 
from the mediator for reduction of fumarate into succinate
in the cytoplasm. Excretion of succinate is avoid becuase of  
knocked out C4 carboxylate antiporter DcuB.</FONT></DIV></TD></TR></TBODY></TABLE>
We focused on respiratory complex II, which contains the fumarate reductase (Frd) and succinate dehydrogenase (Sdh). Last one catalyzes the oxidation of succinate into fumarate by transferring electrons to FAD<SUP>+</SUP> generating FADH<SUB>2</SUB>. (<A href="#Iverson1999">Iverson <I>et al.</I>, 1999</A>; <A href="#Richardson1999">Richardson <I>et al.</I>, 1999</A>) FADH<SUB>2</SUB> enter the electron transport chain and achieves proton translocation over the inner bacterial membrane. The proton motoric force is used by ATP synthase. Generated ATP effects an increasing metabolic activity. (<A href="#Gottschalk1986">Gottschalk <I>et al.</I>, 1986</A>)
So our aim is to generate a succinate overage in the cytoplasm by increased fumarate reductase activity. The reduced mediator functions as an electron donor for the fumarate reductase. Reduction and oxidation between fumarate and succinate creates a loop into the citric acid cycle. In fact electrons are transferred from the mediator to FAD<SUP>+</SUP> regenerating ATP via the electron transport chain. This concept was successfully shown in succinate producing <I>Actinobacillus succinogenes</I> by <A href="#Park1999">Park <I>et al.</I>, 1999</A>. 

Naturally <I>E. coli</I> cells release overproduced succinate into the medium. This occurs especially under anaerobic conditions because bacteria use fumarate as a final electron acceptor instead of oxygen. Besides this effect could be observed if high fumarate concentrations in the medium are available. The produced succinate would be transported out of the cell via the C4 carboxylate transporter dcuB. (<A href="#Janausch2001">Janausch, 2001</A>; <A href="#Unden1997">Unden <I>et al.</I>, 1997</A>) In connection to our second module we planned on working under oxygen limiting conditions, hence effective carbon dioxid fixation is possible. To achieve an effective electron uptake and prevent any succinate excretion, the C4 carboxylate antiporter DcuB has to be knocked out in our <I>E. coli</I> strain. 

In conclusion we are going to modify the metabolic pathway of fumarate by a knockout of the C4 carboxylate antiporter DcuB in <I>E. coli</I> and overexpression of different fumarate reductases. Furthermore the outer membrane porine OprF had to be integrated into the bacterial chromosome to ensure a constitutive expression of OprF and reduce the plasmid overload of bacterial cells. 
</P><DIV class="element" style="width:800px"><FONT size="2" style="text-align:left;"><B>Figure 3</B>: The electron flow in the respiratory chain. Different complexes of electron transport chain under aerobic conditions are visualized. Free energy (E<SUB>0</SUB>Â´) change from negative to positive during electron transfer from NADH to oxygen finally. </FONT></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6 id="FumarateReductase">Fumarate Reductase</H6><P>
We worked on the expression of different fumarate reductases: Fumarate reductase (Frd) from <I>Escherichia coli</I> (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1465102">BBa_K1465102</A>) and Fumarate reductase (Fum) from <I>Actinobacillus succinogenes</I>  (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1465103&quot;">BBa_K1465103</A>; <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1465104&quot;">BBa_K1465104</A> ) both under the control of the T7 promotor in different <I>E. coli</I> strains. In figure 4 our concept is visualized with neutral red as a mediator. The fumarate reductase has a key role in our first module because this enzyme makes sure that electrons are transferred from the reduced mediator into bacterial cells. This leads to an increased succinate production, which support ATP production and generation of reductive power, for example FADH<SUB>2</SUB>.
</P><DIV class="element" style="width:800px"><FONT size="2" style="text-align:left;"><B>Figure 4</B>: The electron flow mediated by redox active mediator (for example neutral red) in interaction with fumarate reductase in <I>E. coli</I> cell. The electrochemical reduced mediator has to cross outer membrane via genome integrated outer membrane porines OprF. The fumarate reductase in the inner bacterial membrane get electrons from the mediator for reduction of fumarate into succinate in the cytoplasm. Excretion of succinate is avoid becuase of knocked out C4 carboxylate antiporter DcuB.</FONT>
Fumarate reductase is part of the anaerobic fumarate respiration in <I>E. coli</I>. It catalyzes the reaction from fumarate into succinate using fumarate as a final electron acceptor in anaerobic fumarate respiration. The related enzyme in aerobic respiration is the succinate dehydrogenase, which catalyses the reaction from succinate to fumarate. 
The electrons are transferred from succinate to FAD<SUP>+</SUP> producing fumarate and FADH<SUB>2</SUB>. The succinate dehydrogenase is also a membrane enzyme and it is part of the citric acid cycle. They both belong to the respiration complex II. Naturally there is no activity of both enzymes at the same time in <I>E. coli</I> cells. (<A href="#Iverson1999">Iverson <I>et al.</I>, 1999</A>)

In our project we used the fumarate reductase in combination with an extracellular mediator as electron donor to transfer electrons into bacterial cells. The reduced mediator crosses the outer membrane of <I>E. coli</I> through the outer membrane porine OprF (<A href="http://parts.igem.org/Part:BBa_K1172507">BBa_K1172507</A>). 
Mediators adsorb at the inner membrane and transfer electrons to the fumarate reductase. After that the reduced fumarate reductase transfer electrons to fumarate producing succinate. This process has been shown for the fumarate reductase in <I>Actinobacillus succinogenes</I> by <A href="#Park1999">Park <I>et al.</I></A>. 
Succinate can serve as a substrate for the succinate dehydrogenase, which catalyzes the oxidation of succinate into fumarate again. So we want to create a loop in the citric acid cycle between fumarate and succinate generating FADH<SUB>2</SUB> as a reductive power in the cell. 
Electrons are transferred to FAD+, which generate proton translocation from the cytoplasm into the periplasmatic space. The proton motoric force leads to ATP production. 
To conclude the mediator-dependent activity of the fumarate reductase could serve as an energy source for bacterial cells. 
We plan on comparing the activity of the fumarate reductases (Frd) from <I>Escherichia coli</I> KRX and the fumarate reductase (Fum)from <I>Actinobacillus succinogenes</I> working with different mediators, for example neutral red or bromphenol blue, as electron donors.  
</DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6 id="AntiporterDcuB">C4 Carboxylate Antiporter DcuB</H6><P>
Under anaerobic conditions <I>E. coli</I> cells use different alternative electron acceptors instead of oxygen. Partially the bacteria use fumarate respiration, whereby fumarate is reduced into succinate. There are also other potential less-oxidizing substances for bacteria to release their electrons, for example anorganic compounds like nitrate (NO<SUB>3</SUB><SUP>-</SUP>) or sulfate (SO<SUB>4</SUB><SUP>2-</SUP>).(<A href="#Gottschalk1986">Gottschalk <I>et al.</I>, 1986</A>)
Fumarate respiration leads to succinate excretion through the C4 carboxylate transporter DcuB. It is an antiporter which exchanges fumarate against succinate under anaerobic conditions. Under aerobic condition there is usually no succinate release observed. In connection to the carbon dioxide fixation in our second module we planned on working under oxygen limiting conditions, hence effective carbon dioxid fixation is possible. So in case of oxygen limiting conditions, there could occured partial fumarate respiration in <I>E. coli</I>. Besides there was shown activity of DcuB antiporter in the presence of high fumarate concentrations (<A href="#Janausch2001">Janausch, 2001</A>). To achieve an effective electron uptake and prevent any succinate excretion, the C4 carboxylate antiporter DcuB has to be knocked out in our <I>E. coli</I> strain. 
  
We planned a targeted knockout of the <I>dcuB</I> gene in <I>E. coli</I> KRX using the <A href="http://www.genebridges.com/storage/Manuals_PDF/K006%20Ecoli%20Gene%20Deletion%20Kit-version2.3-2012.pdf" target="_blank">Genebridge Red/ET-System</A>. In the same step we are going to integrate the outer membrane porine OprF (<A href="http://parts.igem.org/Part:BBa_K1172507">BBa_K1172507</A>) into the bacterial chromosome under controll of a constitutive promotor (<A href="http://parts.igem.org/Part:BBa_J23104">BBa_J23104</A>). This ensure the permeability of the outer membrane and avoid a plasmid overload of the bacteria, because for our system the outer membrane porines are indispensable.</P></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6>Cytochromes</H6><P>
Besides the intention of using a mediator for indirect electron transfer we additionally focused on the electron transfer via cytochromes. Although the exact mechanism of electron transfer by cytochromes is not clarified up to now there are a few basic approaches that indicate which ones could have an important function. The basic idea of this attempt is to bring a periplasmatic cytochrome which is essential for electron uptake in <I>E. coli</I> and use it as an intermediate electron acceptor. Therefore we decided to work with a key-type cytochrome <I>c</I>, called PccH of <I>Geobacter sulfurreducens</I> because it has been shown that biofilms of this organism are capable of accepting electrons from an electrode. Furthermore it has been demonstrated that the mechanism of electron delivery and electron uptake varies in view of involved proteins. In this case transcriptome analysis by microarray revealed that the corresponding gene ( <I> GSU 3274</I>) of PccH  is far more abundant in current-consuming cells using an electrode as electron donor in order to reduce fumarate than the genes known to be essential for electron delivery. Additionally deletion of <I>GSU 3274</I> inhibits the electron uptake completely and complementation of it afterwards can fully restore electron consumption. Based on this we aim to  express PccH (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1465101">BBa_K1465101</A>) in <I>E. coli</I> to bring the electrons into the cell by reducing the membrane-bound fumarate reductase.(<A href="#Strycharz2011">Strycharz et al., 2011</A>; <A href="#Joana2013">Dantas et al., 2013</A>)  Figure 5 shows the general principle of the electron flow.                 

</P><DIV class="element" style="width:800px"><FONT size="2" style="text-align:left;"><B>Figure 5</B>: The electron flow mediated by redox active molecules.</FONT>

Therefore the periplasmatic cytochrome leaves the periplasm through the porins OprF (<A href="http://parts.igem.org/Part:BBa_K1172507">BBa_K1172507</A>) in the outer membrane, gets reduced at the cathode and enters the periplasm again.  There the fumarate reductase which is located in the inner phospholipid membrane oxidized the cytochrome again and is thereby the connective element which transfers the electrons into the cell. These supplied electrons increase the succinate production and together with the succinate dehydrogenase the metabolic activity of the cells should be enhanced.
One of the challenges going along with this approach is the correct localization of the <I>c-type</I>-cytochrome in the periplasm. It is documented that PccH has its own signal peptide for localization and therefore we initially want to use this signal peptide(<A href="#Joana2013">Dantas et al., 2013</A>). Furthermore we  consider to use <I>E. coli</I> specific signal peptides afterwards if necessary.
Another challenge is the correct folding of PccH because there are specific <I>cytochrome c maturation (ccm)</I> genes needed. Those genes also exist in <I>E. coli</I> but are normally just expressed under anaerobic conditions (<A href="#Jensen2010">Jensen et al., 2010</A>). In view of module two and three and of the characterization of PccH completely anaerobic conditions are not possible and can not be guaranteed. This is why we have to bring the <I>ccmA-H</I> genes on a plasmid in <I>E. coli</I> to express it under aerobic conditions. Furthermore we bring it under the control of a constitutive anderson promoter by an appropriate design of primer. 
The expression strength of both, PccH and <I>ccmA-H</I>, are another challenge because they have to be balanced correctly (<A href="#Jensen2010">Jensen et al., 2010</A>). Therefore we want to express <I>ccmA-H</I> constitutivley whereas the promoter of PccH should be changed to test the optimal ratio between both.    

</DIV></DIV><DIV id="main_menue" style="margin-left:130px; height:80px"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Theory" style="color:#000000"><DIV class="main_menueButton" style="width:100px"><P class="buttoncenter"><FONT color="#FFFFFF">Overview</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/GeneticalApproach" style="color:#000000"><DIV class="main_menueButtonActual" style="width:180px"><P class="buttoncenter"><FONT color="#FFFFFF">Genetical Approach</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/ReactorSystem" style="color:#000000"><DIV class="main_menueButton" style="width:150px"><P class="buttoncenter"><FONT color="#FFFFFF">Reactor system</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/MeasurementSystem" style="color:#000000"><DIV class="main_menueButton" style="width:200px"><P class="buttoncenter"><FONT color="#FFFFFF">Measurement System</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Mediators" style="color:#000000"><DIV class="main_menueButton" style="width:100px"><P class="buttoncenter"><FONT color="#FFFFFF">Mediators</FONT></P></DIV></A></DIV><DIV class="element"><DIV id="text"><H6>References</H6><P><UL><LI id="Iverson1999"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
     Iverson <I>et al.</I>, 1999. Structure of the <I>Escherichia coli</I> Fumarate Reductase Respiratory Complex. <A href="http://www.sciencemag.org/content/284/5422/1961.short" target="_blank">Science, vol. 284, pp. 1961-1966</A></DIV></DIV></LI><LI id="Janausch2001"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
     Janausch, 2001. Rekonstitution des Fumaratsensors DcuS in Liposomen und Transport von Fumarat und Succinat in Escherichia coli. <A href="http://ubm.opus.hbz-nrw.de/volltexte/2002/265/pdf/diss.pdf" target="_blank">Doctoral dissertation at Johannes Gutenberg-UniversitÃ¤t Mainz, Germany</A></DIV></DIV></LI><LI id="Gottschalk1986"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
     Gottschalk <I>et al.</I>, 1986. Bacterial Metabolism. <A href="http://onlinelibrary.wiley.com/doi/10.1002/food.19870310532/abstract" target="_blank">Springer-Verlag, New York, Berlin, Heidelberg, Tokyo </A></DIV></DIV></LI><LI id="Park1999"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
     Park <I>et al.</I>, 1999. Utilization of Electrically Reduced Neutral Red byActinobacillus succinogenes: Physiological Function of Neutral Red in Membrane-Driven Fumarate Reduction and Energy Conservation. <A href="http://onlinelibrary.wiley.com/doi/10.1002/food.19870310532/abstract" target="_blank">Journal of Bacteriology, vol. 181, pp. 2403-2410</A></DIV></DIV></LI><LI id="Richardson1999"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
     Richardson <I>et al.</I>, 1999. Bacterial respiration: a flexible process for a
changing environment. <A href="http://mic.sgmjournals.org/content/146/3/551.full" target="_blank">Microbiology, vol. 146, pp. 551-571</A></DIV></DIV></LI><LI id="Unden1997"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
     Unden <I>et al.</I>, 1997. Alternative respiratory pathways of <I>Escherichia coli</I>: energetics and
transcriptional regulation in response to electron acceptors. <A href="http://mic.sgmjournals.org/content/146/3/551.full" target="_blank">Biochimica et Biophysica Acta, vol. 1320, pp. 217-234</A></DIV></DIV></LI><LI id="Strycharz2011"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
 Stycharz, S. M., Glaven, R. H., Coppi, M. V., Gannon, S. M., Perpetua, L. A., Liu, A., Nevin, K. P. &amp;amp. Lovley, D. R. (2011):Gene expression and deletion analysis of mechanisms for electron transfer from
electrodes to <I>Geobacter sulfurreducens</I>.
In: <A href="http://www.sciencedirect.com/science/article/pii/S1567539410001167" target="_blank">Bioelectrochemistry</A> 80, pp. 142 - 150.
</DIV></DIV></LI><LI id="Joana2013"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
 Dantas, J. M., Tomaz, D. M., Morgado, L. &amp; Salgueiro, C. A.(2013): Functional charcterization of PccH, a key cytochrome for electron transfer from electrodes to the bacteium <I>Geobacter sulfurreducens</I>.
In: <A href="http://www.sciencedirect.com/science/article/pii/S0014579313005139" target="_blank">FEBS Letters</A> 587, pp. 2662 - 2668.
</DIV></DIV></LI><LI id="Jensen2010"><DIV class="element" style="margin_10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
Jensen, H. M., Albers, A. E., Malley, K. R., Londer, Y. Y., Cohen, B. E., Helms, B. A., Weigele, P., Groves, J. T., Ajo-Franklin, C. M. (2010): Engineering of a synthetic electron conduit in living cells.
In: <A href="http://www.pnas.org/content/107/45/19213.full.pdf+html" target="_blank">PNAS</A> 107, pp. 19213â19218.
</DIV></DIV></LI></UL></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/GeneticalApproach">http://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/GeneticalApproach</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-CeBiTec/Project/rMFC/GeneticalApproach" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-CeBiTec/Project/rMFC/GeneticalApproach" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Project/rMFC/GeneticalApproach&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Project/rMFC/GeneticalApproach&amp;oldid=400584" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>