"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-CeBiTec/Results/Biosafety
From 2014.igem.org
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Modeling
Parts
Data Page
Achievements
Home
Summary
Applications
Achievements
Bielefueli
Project
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Results
Team
Members
Subteams
Pictures & Videos
Bielefeld & University
Official Team Profile
Acknowledgements
Contact
Policy & Practices
Experts
Interviews
NRW-Day
Pupils Acedemy
How To Wiki
namu
SYNENERGENE
Applications
Vignettes
Notebook
Journal
Protocols
Media
Kits & Enzymes
Strains & Constructs
Primer
Acronyms
Organisms
Additional
Partners
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Partners
Biosafety - Antibiotic-free Selection
Motivation
Transformation efficiency
Transformation process
Stability
Challenges
Outlook
Motivation
An important aspect of synthetic biology is to prevent the uncontrolled interaction between the genetically modified organisms, the environment and the mankind. To prevent this interaction there are many ideas in discussion to implement different biosafety systems ( Wright et al., 2013 ) in synthetic biology. On the other hand there are several studies dealing with the interaction of genetically modified bacteria and wild types, demonstrating in most cases that genetically modified bacteria do not influence the environment. Due to the fact that genetically modified bacteria are adapted to the excellent artificial conditions of the laboratory, the wild type bacteria will usually outlast these modified strains in nature due to their better adaptation to the natural environment. Nevertheless there is always a remaining risk and no guarantee that there is really no interaction and that the release of genetically modified bacteria do not affect the equilibrium of the environment ( Myhr et al., 1999 ) ( Snow et al., 2005 ).
As discussed in the project description here, the most problematic factor in case of release is the transfer of synthetic or genetically modified DNA, particularly the transfer of antibiotic-resistances, which would be an active intervention in the environment.
Therefore, we want to establish an antibiotic-free selection system, which provides the possibility of using genetically modified organisms with a reduced remaining risk to the environment. Such an antibiotic-free selection system can be used for molecular cloning as well as to guarantee long-term plasmid stability. Finally, our studies revealed a further advantage of such a system. It seems to be an even more efficient selection in comparison to antibiotic based selection systems.
Antibiotic-free Selection - Introduction
The cell wall is essential for every living bacterium as it ensures stability and structure, protection against osmotic pressure and regulation of molecule transport. The composition of the cell wall differs among differnt bacteria species, a feature which is commonly used to classify taxonomy. The most common classification is based on the Gram-staining into Gram-negative, for example Escherichia coli and Gram-positive bacteria like Bacillus subtillis . Gram-negative bacteria are characterized by an inner plasma membrane, a thin peptidoglycan layer, periplasmatic spaces and the outer membrane (Figure 1). In contrast Gram-positive bacteria generally lack the outer membrane but consist of a thicker peptidoglycan layer.
Hence, the peptidoglycan layer is an interesting target to control bacterial cell dvision. Peptidoglycan itself is a polymer consisting of a linear chain of polysaccharides and short peptides. The polysaccharides component is composed of alternating residues of beta-(1,4) linked N-acetylglucosamine and N-acetylmuramic acid. They are cross-linked in E. coli by a tetra-peptide of L-alanine, D-glutamic acid, meso-diaminopimelic acid and finally D-alanine. The cross-linkage is thereby realized by a transpeptide-linkage of meso-diaminopimelic acid and D-alanine ( Cava et al., 2011 ).
Figure 1: Structure of the bacterial cell wall of a Gram-negative bacteria. The peptidoglycane layer consist of altering N-acetylglucosamine and N-acetylmuramic acid, which are crosslinked by a tetra-peptide. The crosslinkage is realized by D-alanine. Therefore, D-Alanin is an interesting target to control bacterial cell division.
Bacteria with a missing cross-linkage are not able to perform cell division. Cells would lysate instead because of the broken peptidoglycane layer. One possibility to prevent the crosslinkage is the use of a ß-lactam antibiotic, like penicilin. Penicillin inhibits the enzyme D-transpeptidase which is responsible for the cross linkage. In contrast to that another possibility is the prevention of the D-alanine-synthesis in the cell itself. Bacteria without D-alanine supplementation will lyse during cell division.
The synthesis of D-alanin in E. coli can be catalyzed by an alanine racemase (EC 5.1.1.1). This enzyme enables the reversible reaction from L-alanine into the enantiomer D-alanine. This reaction requires the cofactor pyridoxal-5'-phosphate (PLP) as shown in Figure 2. E. coli posseses two alanine racemases. One is encoded by alr and constitutively expressed and therefore responsible for the accumulation of D-alanine. The other one is encoded by dadX, under the control of the dad-operon and usually used in catabolism ( Walsh, 1989 ).
Figure 2: The alanine racemase Alr from E. coli catalyses the reversible reaction from L-alanine to D-alanine, whereby Pyridoxal-5'-phosphate (PLP) is an essential cofactor for this reaction.
The deletion of the constitutive alanine racemase (alr) and the catabolic alanine racemase (dadX) in the genome will lead to a strict dependence on D-alanine. A bacterium with such a double deletion is only able to grow on media with D-alanine supplementation or with a complementation of the alanine racemase on a separate plasmid like BBa_K1465401 . This approach can be used for an antibiotic-free selection system and even for molecular cloning without antibiotics.
References
Cava F, Lam H, de Pedro MA, Waldor MK (2011) Emerging knowledge of regulatory roles of d-amino acids in bacteria. Cell and Molecular Life Sciences , vol. 68, pp. 817 - 831.
Snow A., Andow D., Gepts P., Hallerman E., Power A., Tiedje J. and Wolfenbarger L. (2005) Genetically Engineered Organisms And The Environment: Current Status And Recommendations. Ecological Applications , vol. 15, pp. 377 - 404.
Myhr, Anne and Traavik, Terje (1999)  The Precautionary Principle Applied to Deliberate Release of Genetically Modified Organisms (GMOs). Microbial Ecology in Health and Disease , vol. 11, pp. 65 - 74.
Walsh, Christopher (1989) Enzymes in the D-alanine branch of bacterial cell wall peptidoglycan assembly. Journal of biological chemistry , vol. 264, pp. 2393 - 2396.
Wright O, Stan GB, Ellis T. (2013) Building-in biosafety for synthetic biology. Microbiology , vol. 159, pp. 1221 - 1235.
Retrieved from " http://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Biosafety "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
