<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Notebook_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-CeBiTec/Notebook/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-CeBiTec/Notebook/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Notebook/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Notebook/Protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-CeBiTec/Notebook/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-CeBiTec/Notebook/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV style="height:100%" id="main_box"><DIV id="flag"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Journal" style="color:#000000"><FLAG class="flag_element1" style="margin-left:70px;"><P class="flag"><FONT color="#FFFFFF">Journal</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols" style="color:#000000"><FLAG class="flag_element2"><P class="flag"><FONT color="#FFFFFF">Protocols</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media" style="color:#000000"><FLAG class="flag_element3"><P class="flag"><FONT color="#FFFFFF">Media</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs" style="color:#000000"><FLAG class="flag_element4"><P class="flag"><FONT color="#FFFFFF">Strains &amp; Constructs</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes" style="color:#000000"><FLAG class="flag_element5"><P class="flag"><FONT color="#FFFFFF">Kits &amp; Enzymes</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Primer" style="color:#000000"><FLAG class="flag_element6"><P class="flag"><FONT color="#FFFFFF">Primer</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Acrynom" style="color:#000000"><FLAG class="flag_element7"><P class="flag"><FONT color="#FFFFFF">Acronyms</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms" style="color:#000000"><FLAG class="flag_element8"><P class="flag"><FONT color="#FFFFFF">Organisms</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Safety" style="color:#000000"><FLAG class="flag_element9"><P class="flag"><FONT color="#FFFFFF">Additional</FONT></P></FLAG></A></DIV><UL id="nav"><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec"><FONT color="#FFFFFF">Home</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/ProjectSummary"><FONT color="#FFFFFF">Summary</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/Applications"><FONT color="#FFFFFF">Applications</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Achievements"><FONT color="#FFFFFF">Achievements</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Bielefueli"><FONT color="#FFFFFF">Bielefueli</FONT></A></LI></UL><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Overview"><FONT color="#FFFFFF">Project</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Overview"><FONT color="#FFFFFF">Overview</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Theory"><FONT color="#FFFFFF">rMFC</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/CO2-fixation/CarbonDioxide"><FONT color="#FFFFFF">CO<SUB>2</SUB> Fixation</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Isobutanol/Theory"><FONT color="#FFFFFF">Isobutanol</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Biosafety"><FONT color="#FFFFFF">Biosafety</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Overview"><FONT color="#FFFFFF">Results</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Overview"><FONT color="#FFFFFF">Overview</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC"><FONT color="#FFFFFF">rMFC</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation"><FONT color="#FFFFFF">CO<SUB>2</SUB> Fixation</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Isobutanol"><FONT color="#FFFFFF">Isobutanol</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Biosafety"><FONT color="#FFFFFF">Biosafety</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Modelling"><FONT color="#FFFFFF">Modeling</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Parts"><FONT color="#FFFFFF">Parts</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/DataPage"><FONT color="#FFFFFF">Data Page</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Achievements"><FONT color="#FFFFFF">Achievements</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team"><FONT color="#FFFFFF">Team</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team/Members"><FONT color="#FFFFFF">Members</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team/Subteams"><FONT color="#FFFFFF">Subteams</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/PicturesAndVideos"><FONT color="#FFFFFF">Pictures &amp; Videos</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/BielefeldAndUniversity"><FONT color="#FFFFFF">Bielefeld &amp; University</FONT></A></LI><LI><A href="https://igem.org/Team.cgi?year=2014"><FONT color="#FFFFFF">Official Team Profile</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Acknowledgements"><FONT color="#FFFFFF">Acknowledgements</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Contact"><FONT color="#FFFFFF">Contact</FONT></A></LI><LI style="width:155px;"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice"><FONT color="#FFFFFF">Policy &amp; Practices</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Experts"><FONT color="#FFFFFF">Experts</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/RadioInterview"><FONT color="#FFFFFF">Interviews</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/NRW-day"><FONT color="#FFFFFF">NRW-Day</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/ScholarAcademy"><FONT color="#FFFFFF">Pupils Acedemy</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Wikisession"><FONT color="#FFFFFF">How To Wiki</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Namu"><FONT color="#FFFFFF">namu</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene"><FONT color="#FFFFFF">SYNENERGENE</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/Applications"><FONT color="#FFFFFF">Applications</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/TechnomoralScenario"><FONT color="#FFFFFF">Vignettes</FONT></A></LI><LI><ACTUAL><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook"><FONT color="#FFFFFF">Notebook</FONT></A></ACTUAL></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners"><FONT color="#FFFFFF">Partners</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/GoldSponsor"><FONT color="#FFFFFF">Gold Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/SilverSponsor"><FONT color="#FFFFFF">Silver Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/BronzeSponsor"><FONT color="#FFFFFF">Bronze Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/Partners"><FONT color="#FFFFFF">Partners</FONT></A></LI><H1> Protocols </H1><DIV id="main_menue" style="margin-left:280px"><A href="#GeneticEngineering" style="color:#000000"><DIV class="main_menueButton" style="width:170px"><P class="buttoncenter"><FONT color="#FFFFFF">Genetic engineering</FONT></P></DIV></A><A href="#Analytics" style="color:#000000"><DIV class="main_menueButton"><P class="buttoncenter"><FONT color="#FFFFFF">Analytics</FONT></P></DIV></A><A href="#Cultivations" style="color:#000000"><DIV class="main_menueButton"><P class="buttoncenter"><FONT color="#FFFFFF">Cultivations</FONT></P></DIV></A></DIV><DIV class="element" id="GeneticEngineering" style="margin:-50px 10px 10px 10px; padding:10px; background-color:#3fb498"><DIV id="text"><CENTER>Genetic engineering</CENTER></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="BluntEnd"><DIV class="show"><A href="#BluntEnd"> Blunt-End </A> <A href="http://www.thermoscientificbio.com/molecular-cloning/clonejet-pcr-cloning-kit/" target="_blank">(Thermo Scientific)</A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>Blunt-End </H6></A> <A href="http://www.thermoscientificbio.com/molecular-cloning/clonejet-pcr-cloning-kit/" target="_blank"><FONT size="3px">(Thermo Scientific)</FONT></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><LI>For cloning blunt-end PCR products generated by proofreading DNA polymerases, such as Pfu DNA polymerase. (If the DNA end structure of the PCR products is not specified by the supplier of the DNA polymerase, follow the Sticky-End Cloning Protocol).</LI><LI>For cloning of blunt-end DNA fragments generated by restriction enzyme digestion. Gel-purify the DNA fragment prior to ligation and use in a 3:1 molar ratio with pJET1.2/blunt (see Table 1).</LI><LI>1. Set up the ligation reaction on ice:
              <TABLE style="background-color:transparent" border="1"><TBODY><TR><TH>Component</TH><TH>Volume</TH></TR><TR><TD width="80%">2X Reaction Buffer</TD><TD>10 µl</TD></TR><TR><TD>Non-purified PCR product  or  purified PCR product/other blunt-end DNA fragment</TD><TD>1 µl 0.15 pmol ends</TD></TR><TR><TD>pJET1.2/blunt Cloning Vector (50 ng/μl)</TD><TD>1 μl (0.05 pmol ends)</TD></TR><TR><TD>Water, nuclease-free</TD><TD>up to 19 μl</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1 µl</TD></TR><TR><TD>Total volume</TD><TD>20 µl</TD></TR></TBODY></TABLE>
              Vortex briefly and centrifuge for 3-5 s.</LI><LI>2. Incubate the ligation mixture at room temperature (22°C) for 5 min.<FONT size="1.9"><B>Note.</B> For PCR products &gt;3 kb, ligation can be prolonged to 30 min. Ligation times longer than 30 min are not recommended and may decrease cloning efficiency.</FONT></LI><LI>3. Use the ligation mixture directly for transformation.<FONT size="1.9"><B>Note.</B> Keep the ligation mixture at -20°C if transformation is postponed. Thaw on ice and mix carefully before transformation.</FONT></LI></UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="ColonyPCR"><DIV class="show"><A href="#ColonyPCR"> Colony PCR </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>Colony PCR </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><LI style="color:#610B0B"> First alternative:</LI><UL style="color:#610B0B"><LI> Pick one colony with a sterile tip and elute it in 100 µl ddH<SUB>2</SUB>O or medium </LI><LI> Store the colony at 4 °C while colony PCR is running</LI></UL><LI style="color:#0B610B"> Second alternative:</LI><UL style="color:#0B610B"><LI> Pick one colony with a sterile tip and streak cells at a marked position on a new plate </LI><LI> Put tip in PCR tube already containing the reaction mixture </LI></UL>
          For both alternatives continue as follows:
          <LI> One reaction mix contains: </LI><UL><LI> 5 µl 5x buffer</LI><LI> 1 µl MgCl<SUB>2</SUB> (25 mM stock) </LI><LI> 0.5 µl 10mM dNTPs </LI><LI> 0.25 µl primer mix (prefix/suffix primers or sequencing primers)</LI><LI> 17.625 µl ddH<SUB>2</SUB>O</LI><LI> 0.125 µl <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#Polymerases" target="_blank">GoTaq polymerase</A> (Promega) </LI><LI> 0.5 µl template </LI></UL><LI>PCR program: </LI><UL><LI>Cell lysis and initial denaturation: 5 min, 95 °C</LI><LI>30 cycles of:</LI><UL><LI>10 s, 95 °C </LI><LI>30 s, annealing temperature </LI><LI>1 min / 1 kb of expected product, 72 °C </LI></UL><LI>Final elongation: 5 min, 72 °C </LI></UL><LI> Gel electrophoresis: check the fragment size </LI><LI style="color:#610B0B">First (as above) alternative:
          </LI><LI> Plate the correct colony</LI></UL><LI style="color:#0B610B">Second (as above) alternative:
          </LI><LI> Use cells from the right positions to start liquid cultures or streak them on a new plate</LI></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="DNAisolation"><DIV class="show"><A href="#DNAisolation"> DNA isolation (genomic DNA)</A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>DNA isolation (genomic DNA)</H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%; type:disc" type="disc"><LI>The buffer and spin column for this DNA isolation were taken from the <A href="http://www.qiagen.com/de/products/catalog/sample-technologies/dna-sample-technologies/plasmid-dna/qiaprep-spin-miniprep-kit/" target="_blank">QIAprep® Spin Miniprep Kit</A></LI><LI>Resuspend the cells of the plate in 800 µl Buffer P1.</LI><LI>Transfer the suspension in small tubes with beads.</LI><LI>Use the ribolyser with 6200 rpm 3 x 60 seconds.</LI><LI>Centrifuge at top speed for 3 minutes.</LI><LI>Transfer 500 µl of the supernatant in a new 2 ml reaction tube.</LI><LI>Add 500 µl of Buffer P2 and mix thoroghly by inverting the tube 6-8 times.</LI><LI>Add 700 µl of Buffer N3 and mix thoroghly by inverting the tube 6-8 times.</LI><LI>Centrifuge at top speed for 10 minutes.</LI><LI>Apply the supernatant to the spin column by decanting or pipetting.</LI><LI>Centrifuge at top speed for 30-60 seconds and discard the flow-through.</LI><LI>Wash the spin column by adding 750 µl Buffer PE.</LI><LI>Centrifuge for 30-60 seconds and discard flow-through.</LI><LI>Wash again by adding 750 µl Buffer PE.</LI><LI>Centrifuge for 30-60 seconds and discard flow-through.</LI><LI>Centrifuge 2 minutes to remove residual wash buffer.</LI><LI>Place the column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 20 µl H<UB>2O, incubate at room temperature for a few minutes and centrifuge at top speed for 1 minute. Repeat this step.</UB></LI></UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="DNAPurificationbyCentrifugation"><DIV class="show"><A href="#DNAPurificationbyCentrifugation"> DNA Purification </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>DNA Purification by Centrifugation </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%; type:disc" type="disc"><LI><B>DNA purification by centrifugation by <A href="http://www.promega.de/products/dna-and-rna-purification/dna-fragment-purification/wizard-sv-gel-and-pcr-clean_up-system/" target="_blank">Promega</A></B></LI><UL><LI style="color:#610B0B">First alternative:</LI><UL style="color:#610B0B"><LI style="color:#610B0B"> Dissolving the Gel Slice </LI><UL style="color:#610B0B"><LI> Following electrophoresis, excise DNA band from gel and place gel slice in a 1.5 ml 
                    microcentrifuge tube. </LI><LI> Add 10 µl Membrane Binding Solution per 10 mg of gel slice. Vortex and incubate at 50-65 °C until
                    gel slice is completely dissolved. </LI></UL></UL><LI style="color:#0B610B">Second alternative:</LI><UL><LI style="color:#0B610B"> Processing PCR Amplifications </LI><UL style="color:#0B610B"><LI> Add an equal volume of Membrane Binding Solution to the PCR amplification. </LI></UL></UL><LI> Binding of DNA </LI><UL><LI> Insert SV Minicolumn into Collection Tube. </LI><LI> Transfer dissolved gel mixture or prepared PCR product to the Minicolumn assembly. Incubate at
                     room temperature for 1 minute. </LI><LI> Centrifuge at 16,000 x g for 1 minute. Discard flowthrough and reinsert Minicolumn into 
                     Collection Tube. </LI></UL><LI> Washing </LI><UL><LI> Add 700 µl Membrane Wash Solution (ethanol added). Centrifuge at 16,000 x g for 1 minute. Discard
                    flowthrough and reinsert Minicolumn into Collection Tube. </LI><LI> Repeat Step before with 500 µl Membrane Wash Solution. Centrifuge at 16,000 x g for 5 minutes.
               </LI><LI> Empty the Collection Tube and recentrifuge the column assembly for 1 minute with the
                    microcentrifuge lid open (or off) to allow evaporation of any residual ethanol. </LI></UL><LI> Elution </LI><UL><LI> Carefully transfer Minicolumn to a clean 1.5 ml  microcentrifuge tube. </LI><LI> Add 15 µl of Nuclease-Free Water to the Minicolumn. Incubate at 60 °C for 5 minutes. Centrifuge at
                    16,000 x g for 1 minute. Repeat this step. </LI><LI> Discard Minicolumn and store DNA at 4 °C or -20 °C. </LI></UL></UL><LI><B>QIAquick Gel Extraction Kit by <A href="http://www.qiagen.com/de/products/catalog/sample-technologies/dna-sample-technologies/dna-cleanup/qiaquick-gel-extraction-kit/" target="_blank">QIAGEN</A></B></LI><UL><LI>Note: All centrifugation steps are carried out at 17,900 x g (13,000 rpm) in a conventional table-top microcentrifuge.</LI><LI>Excise the DNA fragment from the agarose gel with a clean, sharp scapel.</LI><LI>Wigh the gel slice in a colorless tube. Add 3 volumes Buffer QG to 1 volume gel (100 mg gel ~ 100 µl). The maximum amount of gel per spin column is 400 mg. For &gt; 2% agarose gels, add 6 volumes Buffer QG.</LI><LI>Incubate at 50 °C for 10 min (or until the gel slice has completely dissolved). Vortex thee tube every 2-3 min to help dissolve the gel. After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture is orange or violet, add 10 µl 3 M sodium acetate, pH 5.0, and mix. The mixture turns yellow.</LI><LI>Add 1 volume isopropanol to the sample and mix.</LI><LI>Place a QIAquick spin colun in a provided 2 ml collection tube. To bind DNA, apply the sample to the QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAquick column back into the same tube. For sample volumes of &gt;800 µl, load and spin again.</LI><LI>If the DNA will subsequently be used for sequencing, in vitro transcription, or microinjection, add 500 µl Buffer QG to the QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAqick column back into the same tube.</LI><LI>To wash, add 750 µl Buffer PE o QIAquick column and centrifuge for 1 mi. Discard flow-through and place the QIAquick column back into the same tube.<B>Note:</B> If the DNA will be used for salt-sensitive applications (e.g., sequencing, blunt-ended ligation), let the column stand 2-5 min after addition of Buffer PE.Centrifuge the QIAquick clumn in the provided 2 ml collection tube 1 min to remove residual wash buffer.</LI><LI>Place QIAquick column into a clean 1.5 ml microcentrifuge tube.</LI><LI>To elute DNA, add 50 µl Buffer EB (10 mM TrisCl, pH 8.5) or water to the center of the QIAquick membrane and centrifuge the column for 1 min. For increased DNA concentration, add 30 µl Buffer EB to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min. After the addition of Buffer EB to the QIAqick membrane, increasing the incubation time up to 4 min can increase the yield of purified DNA.</LI><LI>If the purified DNA is to be analyzed on a gel, add 1 volume of Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel.</LI></UL></UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="Generatingchemocompetentcells"><DIV class="show"><A href="#Generatingchemocompetentcells"> Generating chemocompetent cells </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>Generating chemocompetent cells
        </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><LI><U> Material: </U></LI><UL><LI> 300 ml <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#LBmedium">LB-Medium</A></LI><LI> ~10 ml cooled TSS buffer </LI><LI> 5 pre-cooled 50 ml Falcons </LI></UL><LI><U> Protocol: </U></LI><UL><LI> Inoculate 2x3 ml LB with bacterial stock; incubate over night at 37 °C and 200 rpm </LI><LI> Inoculate 1x250 ml LB with the over night cultures in 1 ml flask at 37 °C and 140 rpm </LI><LI> Incubate until OD<SUB>600</SUB> 0.4-0.6 </LI><LI> Devide the culture into five cooled 50 ml Falcons and incubate on ice for 10 minutes
             </LI><LI> Onwards all steps at 4 °C </LI><LI> Centrifugate at 3000 rpm, 4 °C for 10 minutes</LI><LI> Discard supernatant </LI><LI> Pipette out remaining media</LI><LI> The volume of <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#TSS buffer" target="_blank">TSS</A> to use is 10% of the culture volume that you spun down (we only used 1-2 ml). You may need to vortex gently to fully resuspend the culture, keep an eye out for small cell aggregates even after the pellet is completely off the wall.</LI><LI> Use 100 µl per aliquot and drop them in liquid nitrogen immediatly </LI><LI> Store at -80 °C </LI></UL><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="Generatingelectrocompetentcells"><DIV class="show"><A href="#Generatingelectrocompetentcells"> Generating electrocompetent cells </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>Generating electrocompetent cells</H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><LI><U> Material: </U></LI><UL><LI> 550 ml <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#LBmedium">LB-Medium</A></LI><LI> 1 l cooled bidest. H<SUB>2</SUB>O </LI><LI> 50 ml cooled 10% glycerine </LI><LI> 10 pre-cooled 50 ml Falcons </LI></UL><LI><U> Protocol: </U></LI><UL><LI> Inoculate 2x3 ml LB with bacterial stock; incubate over night at 37 °C and 200 rpm </LI><LI>Inoculate 2x250 ml LB with the over night cultures in 1-litre-flask at 37 °C and 140 rpm </LI><LI> Incubate until OD<SUB>600</SUB> 0.4-0.6 </LI><LI> Cool the culture 15-30 minutes on ice </LI><LI> Onwards all steps at 4 °C </LI><LI> Divide the cultures into cooled 50 ml Falcons and centrifugate at 4000 rpm, 4 °C for 15 minutes, make  sure to slowly accelerate and deccelerate </LI><LI> Discard supernatant </LI><LI> Resuspend pellet in 5 ml cooled bidest H<SUB>2</SUB>O (and don't get frustrated while doing it, keep shaking gently) </LI><LI> Pool two suspensions each, add bidest H<SUB>2</SUB>O up to 50 ml and centrifugate again (see centrifugation above)</LI><LI> Discard supernatant </LI><LI> Resuspend pellet in 5 ml cooled bidest H<SUB>2</SUB>O</LI><LI> Add bidest H<SUB>2</SUB>O up to 50 ml and centrifugate again (see centrifugation above) </LI><LI> Discard supernatant </LI><LI> Resuspend pellet in 5 ml cooled 10% glycerine </LI><LI> Transfer suspensions in two 50 ml Falcons and centrifugate again (see centrifugation above) </LI><LI> Discard supernatant </LI><LI> Add volume of 10% glycerine that is approximately equal to the volume of the pellet and resuspend </LI><LI> Divide cells in 50 µl aliquots and freeze in liquid nitrogen immediately </LI><LI> Store at -80 °C </LI></UL><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="Gibson"><DIV class="show"><A href="#Gibson"> Gibson Assembly </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>Gibson Assembly</H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%; type:disc" type="disc"><LI>Modified from <A href="http://www.nature.com/nmeth/journal/v6/n5/full/nmeth.1318.html" target="_blank">Gibson et al. (2009)</A></LI><LI>This assembly method is an isothermal, single-reaction method for assembling multiple overlapping DNA molecules. By coordinating the activity of a 5‘ exonuclease, a DNA polymerase and a DNA ligase two adjacent DNA fragments with complementary terminal sequence overlaps can be joined into a covalently sealed molecule, without the use of any restriction endonuclease.</LI><DIV class="element" style="padding:10px; width:250px; text-align:center; margin-bottom:20px; margin-left:auto; margin-right:auto"><FONT size="1">Gibson-Assembly</FONT></DIV><LI>Preparation of DNA molecules for <I>in vitro</I> recombination</LI><UL><LI>Generate the complementary sequence overlaps by PCR using the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#Polymerases" target="_blank">Phusion DNA-polymerase</A>. If necessary add 5 M Betain in the reaction mix by reducing the amount of H<SUB>2</SUB>O to decrease the number of false PCR products.</LI><LI>Identify the PCR products of interest by gel electrophoresis with known DNA standards.</LI><LI>Extract the PCR products from the gel by cutting out the DNA fragments and clean them up by using a commercial <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#DNAPurificationbyCentrifugation" target="_blank&quot;">PCR clean-up kit</A>.</LI></UL><LI><I>in vitro</I> recombination</LI><UL><LI>Assembly mixture:</LI><UL><LI>320 µl 5x isothermal reaction buffer</LI><LI>0.64 µl of 10 U ml<SUP>-1</SUP> T5 exonuclease (for DNA molecules overlapping by greater than 150 bp add 3.2 µl of 10 U ml<SUP>–1</SUP> T5 exonuclease)</LI><LI>20 µl of 2 U ml<SUP>-1</SUP><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#Polymerases" target="_blank">Phusion DNA polymerase</A></LI><LI>160 µl of 40 U ml<SUP>-1</SUP> taq DNA ligase</LI><LI>add ddH<SUB>2</SUB>O water up to a final volume of 1.2 ml</LI><LI>aliquote 15 µl of the reagent-enzyme mix and store it at –20 ˚C</LI></UL><LI>Thaw 15 µl assembly mixture aliquot and keep it on ice until use.</LI><LI>Add 5 µl of the purified DNA molecules in equimolar amounts (between 10 and 100 ng of each DNA fragment).</LI><LI>Incubate the resulting mixture at 50 ˚C for 15 to 60 min, with 60 min being optimal.</LI><LI>Transformation (<A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#Transformationviaheatshock"> via heat shock</A> or <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#Transformationviaelectroporation" target="_blank">via electroporation</A>) without cleaning up the assembly product.</LI></UL></UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="NEBBioBrick"><DIV class="show"><A href="#NEBBioBrick"> NEB BioBrick Assembly Kit </A> <A href="http://www.neb.com/products/E0546-BioBrick-Assembly-Kit" target="_blank">(NEB)</A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>NEB BioBrick Assembly Kit</H6></A> <A href="http://www.neb.com/products/E0546-BioBrick-Assembly-Kit" target="_blank"><FONT size="3px">(NEB)</FONT></A></DIV><DIV class="content"><DIV class="element" style="padding:10px; width:350px; text-align:center; margin-bottom:20px; margin-left:auto; margin-right:auto"><FONT size="1">Gibson-Assembly</FONT></DIV><UL style="margin-left:10%; margin-right:10%; type:disc" type="disc"><LI>We used the BioBrick Assembly Kit to assemble an upstream part with a downstream part into destination plasmid.</LI><LI>Digestion Protocoll:</LI><UL><LI>Digest upstream part with <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#RestrictionEnzymes" target="_blank">EcoRI-HF</A> and <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#RestrictionEnzymes" target="_blank">SpeI</A></LI><LI>Digest downstream part with <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#RestrictionEnzymes" target="_blank">XbaI</A> and <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#RestrictionEnzymes" target="_blank">PstI</A></LI><LI>Digest destiantion destination plasmid with <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#RestrictionEnzymes" target="_blank">EcoRI-HF</A> and <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#RestrictionEnzymes" target="_blank">PstI</A></LI><UL><LI>500 ng part DNA</LI><LI>1 μl of each enzyme</LI><LI>5 μl 10x NEBuffer 2.1</LI><LI>to 50 μl H<SUB>2</SUB>0</LI></UL><LI>Incubate all three restriction digest reactions at 37 °C for 10 minutes and then heat inactivate at 80 °C for 20 minutes.</LI></UL><LI>Dephosphorylation Protocoll</LI><UL><LI>Add 1 μl of AP (Antarctic phosphatase) and 5 µl of 10x AP reaction buffer to digested destination plasmid, incubate for 1 h at 37 °C.</LI></UL><LI>Ligation Protocoll</LI><UL><LI>Ligate the upstream and downstream parts into the digested destination plasmid.</LI><UL><LI>2 μl of each part</LI><LI>2 μl 10x T4 DNA Ligase Buffer</LI><LI>1 μl T4 DNA Ligase</LI><LI>11 μl H<SUB>2</SUB>O</LI></UL><LI>Incubate at room temperature for 10 minutes and then heat inactivate at 80 °C for 20 minutes.</LI></UL><LI>Transform 1-2 μl of the ligation product into 50 μl of competent E. coli cells.</LI></UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="PCR"><DIV class="show"><A href="#PCR"> PCR </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>PCR </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><LI>Phusion (our standard) master mix (25 µl):</LI><UL><LI>5 µl 5 x HF-buffer</LI><LI>0.5 µl 10mM dNTPs</LI><LI>Up to 20 ng * µl<SUP>-1</SUP> template</LI><LI>0.25 µl polymerase</LI><LI>0.25-0.5 µl primer 10 µM (each one)</LI><LI>0.75 µl DMSO (optional)</LI><LI>Fill up to 25 µl with H<SUB>2</SUB>O</LI></UL><LI>Q5 master mix (25 µl):</LI><UL><LI>5 µl 5 x buffer</LI><LI>0.5 µl 10 mM dNTPs</LI><LI>Up to 50 ng * µl<SUP>-1</SUP> template</LI><LI>0.1 µl polymerase</LI><LI>0.5-1 µl primer 10 µM (each one)</LI><LI>Fill up to 25 µl with H<SUB>2</SUB>O</LI></UL><LI>KOD master mix (50 µl):</LI><UL><LI>5 µl 10 x buffer for KOD Hot Start DNA Polymerase</LI><LI>3 µl 25 mM MgSO4</LI><LI>1 µl 10mM dNTPs</LI><LI>1 µl polymerase</LI><LI>1.5 µl primer 10 µM) (each one)</LI><LI>10 ng * µl<SUP>-1</SUP> template
                 </LI><LI>Fill up to 50 µl with H<SUB>2</SUB>O</LI></UL></UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="PurificationPromega"><DIV class="show"><A href="#PurificationPromega">Plasmid Isolation</A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>Plasmid isolation</H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%; type:disc" type="disc"><LI><B>Wizard®<I>Plus</I> SV Minipreps DNA Purification System by <A href="http://www.promega.de/resources/protocols/technical-bulletins/0/wizard-plus-sv-minipreps-dna-purification-system-protocol/" target="_blank">Promega</A></B> (Our standard kit)</LI><UL><LI>Production of cleared lysate:</LI><UL><LI>Pellet 1-10 ml of overnight culture for 5 minutes</LI><LI>Thoroughly resuspend pellet with 250 µl of Cell Resuspension Solution</LI><LI>Add 250 µl of Cell Lysis Solution to each sample; invert 4 times to mix</LI><LI>Add 10 µl of Alkaline Protease Solution; invert 4 times to mix. Incubate 5 minutes at room temperature</LI><LI>Add 350 µl of Neutralization Solution; invert 4 times to mix</LI><LI>Centrifuge at top speed for 10 minutes at room temperature</LI></UL><LI>Binding of plasmid DNA</LI><UL><LI>Insert Spin Column into Collection Tube</LI><LI>Decant cleared lysate into Spin Column</LI><LI>Centrifuge at top speed for 1 minute at room temperature. Discard flowthrough, and reinsert Column into Collection Tube.</LI></UL><LI>Washing</LI><UL><LI>Add 750 µl of Wash Solution (ethanol added). Centrifuge at top speed for 1 minute. Discard flowthrough and reinsert column into Collection Tube</LI><LI>Repeat step before with 250 µl of Wash solution</LI><LI>Centrifuge at top speed for 2 minutes at room temperature</LI></UL><LI>Elution</LI><UL><LI>Transfer Spin Column to a sterile 1.5 ml microcentrifuge tube, being careful not to transfer any of the Column Wash Solution with the Spin Column. If the Spin Column has Column Wash Solution associated with it, centrifuge again for 1 minute at top speed, then transfer the Spin Column to a new, sterile 1.5 ml microcentrifuge tube</LI><LI>Add 100 µl of Nuclease-Free Water to the Spin Column. Centrifuge at top speed for 1 minute at room temperature</LI><LI>Discard column, and store DNA at -20 °C or below</LI></UL></UL><LI><B>GeneJET Plasmid Miniprep Kit by <A href="http://www.thermoscientificbio.com/nucleic-acid-purification/genejet-plasmid-miniprep-kit/" target="_blank">Thermo Scientific</A></B></LI><UL><LI> Note: </LI><UL><LI> All purification steps should be carried out at room temperature. </LI><LI> All centrifugation should be carried out in a table-top microcentrifuge at &gt;12000 x g (10 000-14
                    000 rpm, depending on the rotor type). </LI></UL><LI> Resuspend the pelleted cells in 250 µl of the Resuspension Solution. Transfer the cell suspension to a 
               microcentrifuge tube. The bacteria should be resuspended completely by vortexing or pipetting up and
               down until no cell clumps remain. </LI><UL><LI><U>Note</U> Ensure RNase A has been added to the Resuspension Solution. </LI></UL><LI> Add 250 µl of the Lysis Solution and mix thoroughly by inverting the tube 4-6 times until the solution 
               becomes viscous and slightly clear. </LI><UL><LI><U>Note</U> Do not vortex to avoid shearing of chromosomal DNA. Do not incubate for more than 5 
                    minutes to avoid denaturation of supercoiled plasmid DNA. </LI></UL><LI> Add 350 µl of the Neutralization Solution and mix immediately and thoroughly by inverting the tube 4-6 
               times. </LI><UL><LI><U>Note</U> It is important to mix thoroughly and gently after the addition of the Neutralization
                    Solution to avoid localized  precipitation of bacterial cell debris. The neutralized bacterial 
                    lysate should become cloudly. </LI></UL><LI> Centrifuge for 5 minutes to pellet cell debris and chromosomal DNA. </LI><LI> Transfer the supernatant to the supplied GeneJET spin coloumn by decanting or pipetting. Avoid 
               disburbing or transferring the white  precipitate. </LI><LI> Centrifuge for 1 minute. Discard the flow-through and place the coloumn back into the same collection
               tube. </LI><UL><LI><U>Note</U> Do not add bleach to the flow-through. </LI></UL><LI> Add 500 µl of the Wash Solution (diluted with ethanol prior to first use) to the GeneJET spin column. 
               Centrifuge for 30-60 seconds and discard the flow-through. Place the column back into the same 
               collection tube. </LI><LI> Repeat the wash procedure (step before) using 500 µl of the Wash Solution. </LI><LI> Discard the flow-through and centrifuge for an additional  1 minute to remove residual Wash Solution.
               This step is essential to avoid residual ethanol in plasmid preps. </LI><LI> Transfer the GeneJET spin column into a fresh 1.5 ml microcentrifuge tube (not included). Add 50 µl of
               the Elution Buffer to the center of GeneJET spin column membrane to elute the plasmide DNA. Take care
               not to contact the membrane with the pipette tip. Incubate for 2 minutes at room temperature and 
               centrifuge for 2 minutes. </LI><UL><LI><U>Note</U> An additional elution step (optional) with Elution Buffer or water will recover 
                    residual DNA from the membrane and increase the overall yield by 10-20%. For elution of plasmids
                    or cosmids &gt;20 kb, prewarm Elution Buffer to 70 °C before applying to silica membrane. </LI></UL><LI> Discard the column and store the purified plasmid DNA at -20 °C.</LI></UL><LI><B>QIAprep® Spin Miniprep Kit by <A href="http://www.qiagen.com/us/search/qiaprep-spin-miniprep-kit" target="_blank">QIAGEN</A></B></LI><UL><LI>Note: All centrifugation steps are carried out at 13,000 rpm (~17,900 x g) in a conventional able-top microcentrifuge.</LI><LI>Pellet 1-5 ml bacterial overnight culture by centrifugation at &gt;8000 rpm (6800 x g) for 3 min at room temperature (15-25 °C).</LI><LI>Resuspend pelleted bacterial cells in 250 µl Buffer P1 and transfer to a microcentrifuge tube.</LI><LI>Add 250 µl Buffer P2 and mix thoroughly by inverting the tube 4-6 times until the solution becomes clear. Do not allow the lysis reaction to proceed for more than 5 min. If using LyseBlue reagent, the solution will turn blue.</LI><LI>Add 350 µl Buffer N3 and mix immediately and thoroughly by inverting the tube 4-6 times. If using LyseBlue reagent, the solution will turn colorless.</LI><LI>Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge.</LI><LI>Apply the supernatant from step 5 to the QIprep spin column by decanting or pipetting. Centrifuge for 30-60 s and discard the flow-through, or apply vacuum to the manifold to draw the solution through the QIAprep spin column and switch off the vacuum source.</LI><LI>Recommended: Wash the QIAprep spin column by adding 500 µl Buffer PB. Centrifuge for 30-60 s and discard the flow-through, or apply vacuum to the manifold to draw the solution through the QIprep spin column and switch off the vacuum source.</LI><LI>Wash the QIAprep spin column by adding 750 µl Buffer PE. Centrifuge for 30-60s and discard the flow-through, or apply vacuum to the manifold to draw the solution through the QIAprep spin column and switch off the vacuum source. Transfer the QIAprep spin column to the collection tube.</LI><LI>Centrifuge for 1 min to remove residual wash buffer</LI><LI>Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 µl Buffer EB (10 mM Tris*Cl, pH 8.5) or water to the center of the QIAprep spin column. Let it stand for 1 min, and centrifuge for 1 min.</LI></UL><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="BioBrick"><DIV class="show"><A href="#BioBrick"> Standard BioBrick Assembly </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>Standard BioBrick Assembly</H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%; type:disc" type="disc"><LI>modified from <A href="http://openwetware.org/wiki/Silver:_BB_Strategy">silver lab</A></LI><LI>This assembly method can be used for BioBricks which are bigger than 150 bp. The BioBrick should be at least 500 bp bigger or smaller than the backbone. The BioBrick, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other used BioBrick, called vector. So you have to differentiate between a prefix and a suffix insertion.</LI><TABLE align="center" width="100%" cellspacing="10" style="background-color:transparent"><TBODY><TR><TD><DIV class="element" style="padding:10px; text-align:center"><FONT size="1">Silver Suffix Insertion</FONT></DIV></TD><TD><DIV class="element" style="padding:10px; text-align:center"><FONT size="1">Silver Prefix Insertion</FONT></DIV></TD></TR></TBODY></TABLE><LI>Suffix insertion:</LI><UL><LI>Digestion of insert: at least 700 ng DNA / 10 µl volume, 1 µl 10x NEBuffer 2.1, 0.5 µl <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#RestrictionEnzymes" target="_blank">XbaI</A>, 0.5 µl <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#RestrictionEnzymes" target="_blank">PstI</A>. Digest for 1 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert.</LI><LI>Digestion of vector about 700 ng DNA / 10 µl volume, 1 µl 10x NEBuffer 2.1, 0.5 µl <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#RestrictionEnzymes" target="_blank">SpeI</A>, 0.5 µl <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#RestrictionEnzymes" target="_blank">PstI</A>. Digest for 1 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µl AP (Antarctic phosphatase) and 1.2 µl 10 x AP reaction buffer, incubate for 1 h at 37 °C. Clean up the vector with a <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#DNAPurificationbyCentrifugation" target="_blank">PCR clean-up kit</A>.</LI><LI>Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µl ligation volume, 2 µl T4-Ligase-Buffer, 1 µl T4-Ligase. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.</LI></UL><LI>Prefix insertion:</LI><UL><LI>Digestion of insert: at least 700 ng DNA / 10 µl volume, 1 µl 10x NEBuffer 2.1, 0.5 µl <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#RestrictionEnzymes" target="_blank">EcoRI</A>, 0.5 µl <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#RestrictionEnzymes" target="_blank">SpeI</A>. Digest for 1 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert.</LI><LI>Digestion of vector about 700 ng DNA / 10 µl volume, 1 µl 10 x NEBuffer 2.1, 0.5 µl <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#RestrictionEnzymes" target="_blank">EcoRI</A>, 0.5 µl <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes#RestrictionEnzymes" target="_blank">XbaI</A>. Digest for 1h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µl AP (Antarctic phosphatase) and 1.2 µl 10 x AP reaction buffer, incubate for 1 h at 37 °C. Clean up the vector with a <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#DNAPurificationbyCentrifugation" target="_blank">PCR clean-up kit</A>.</LI><LI>Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µl ligation volume, 2 µl T4-Ligase-Buffer, 1 µl T4-Ligase. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.</LI></UL><LI>Variations:</LI><UL><LI>A digestion over night is possible. If you digest over night use only 0.1 µl restriction enzyme.</LI><LI>It is also possible to use PCR product as insert. Digest after PCR with corresponding restriction enzymes and clean up with a <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#DNAPurificationbyCentrifugation" target="_blank">PCR clean-up kit</A>. This could lead to higher yields of insert DNA because a lot of DNA gets lost during the gel electrophoresis clean up.</LI><LI>Sometimes some BioBricks are hard to assemble. Then you have to clean up the vector by gel electrophoresis as well.</LI></UL></UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="Transformationviaelectroporation"><DIV class="show"><A href="#Transformationviaelectroporation"> Transformation via electroporation </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>Transformation via electroporation </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><LI> Thaw 50 µl <A href="#Generatingelectrocompetentcells">electrocompetent <I>E. coli</I></A> cells on ice, dilute with icecold 50 µl glycerine (10%) if necessary </LI><LI> Add 0.5-5 µl plasmid to 50 µl electrocompetent cells </LI><LI> Store cells on ice for 1 minute </LI><LI> Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ω</LI><LI> Transfer transformation reaction to 450 µl <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#SOC-medium">SOC-Medium</A> and incubate 1 h at 37 °C </LI><LI> Centrifuge 3 minutes at 1200 rpm and plate on selective <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#LBmedium">LB-Medium</A></LI><LI> Incubate over night at 37 °C </LI></UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="Transformationviaheatshock"><DIV class="show"><A href="#Transformationviaheatshock"> Transformation via heat shock </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>Transformation via heat shock </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><LI> Thaw 100 µl <A href="#Generatingchemocompetentcells">chemocompetent <I>E. coli</I></A> cells on ice</LI><LI> Add 0.5-5 µl plasmid to 100 µl chemocompetent cells </LI><LI> Store cells on ice for 10-30 minutes </LI><LI> Heat shock for 90 seconds at 42 °C</LI><LI> Transfer transformation reaction to 450 µl <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#SOC-medium">SOC-Medium</A> and incubate 1 h at 37 °C </LI><LI> Centrifuge 3 minutes at 1200 rpm and plate on selective <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#LBmedium">LB-Medium</A></LI><LI> Incubate over night at 37 °C </LI></UL></DIV></DIV></DIV></DIV><DIV class="element" id="Analytics" style="margin:10px 10px 10px 10px; padding:10px; background-color:#3fb498"><DIV id="text"><CENTER>Analytics</CENTER></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="ProteinBufferExchangeAndDesalting"><DIV class="show"><A href="#ProteinBufferExchangeAndDesalting"> Amicon Ultra - Protein buffer exchange and desalting </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>Amicon Ultra - Protein buffer exchange and desalting
        </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><LI>Load up to 15 ml of purified enzyme on to the Amicon Ultra filter device</LI><LI>Place capped filter device into centrifuge rotor; counterbalance with a similar device</LI><LI>When using a swinging bucket rotor, spin the device at 4,000xg maximum for approximately 15-60 minutes</LI><LI>When using a fixed angle rotor, orient the device with the membrane panel facing up and spin at 5,000xg maximum for approximately 15-60 minutes</LI><LI>To recover the the concentrated solute, insert a pipettor into the bottom of the filter device and withdraw the sample using a side-to-side sweeping motion to ensure total recovery. The ultrafiltrate can be stored in the centrifuge tube.</LI><LI>For buffer exchange load up to 15 ml of new buffer on to the Amicon Ultra filter device and centrifuge for 15-60 minutes</LI></UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="ColdOsmoticShock"><DIV class="show"><A href="#ColdOsmoticShock"> Cold Osmotic Shock </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6> Cold Osmotic Shock
        </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc">
 
Release of periplasmic protein fraction from <I>E. coli</I> by cold osmotic shock

Modified protocol from <A href="http://www.jbc.org/content/240/9/3685.full.pdf+html?sid=4a90c176-0ec3-489f-8c82-4734274cebf5">Neu &amp; Heppel, 1965</A>:
           <UL><LI>Centrifuge <I>E. coli</I> cell suspension for 5 min at 14,000 g (4 °C) to collect the cells</LI><LI>Discard the entire supernatant</LI><LI>Resuspend the cells in ice-cold <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#FractioningBuffer">Cell Fractioning Buffer 1</A>. The resulting volume should be 1/4 of the former suspension volume</LI><LI>Incubate for 20 min on ice. Invert the suspension at regular intervals to counteract sedimentation</LI><LI>Centrifuge the cell suspension for 15 min at 14,000 g (4 °C)</LI><LI>Discard the entire supernatant</LI><LI>Resuspend the cells in ice-cold <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#FractioningBuffer">Cell Fractioning Buffer 2</A>. The resulting volume should be 1/4 of the former suspension volume</LI><LI>Incubate for 10 up to 20 min on ice under regular invertion</LI><LI>Centrifuge the cell suspension for 15 min at 14,000 g (4 °C)</LI><LI>Save the supernatant, which contains the periplasmatic proteins and membrane proteins</LI></UL><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="FastCellLysisforSDS-PAGE"><DIV class="show"><A href="#FastCellLysisforSDS-PAGE"> Fast Cell Lysis for SDS-PAGE </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6> Fast Cell Lysis for SDS-PAGE
        </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><UL><LI> Measure OD<SUB>600</SUB></LI><LI> Transfer two samples a 1,5 ml in a microcentrifuge tube </LI><LI> Centrifuge at top speed for 5 minutes at room temperature </LI><LI> Discard supernatant quantitative </LI><LI> Store pellet at -20 °C </LI><LI> Thaw pellet and resupend in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#Fast%20Cell%20Lysis%20sample%20buffer" target="_blank">Fast Cell Lysis sample buffer.</A> (30 µl <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#Fast%20Cell%20Lysis%20sample%20buffer" target="_blank">Fast Cell Lysis sample buffer.</A> per OD<SUB>600</SUB>) </LI><LI> Heat for 5 - 10 minutes at 95 - 100 °C </LI><LI> Centrifuge at top speed for 15 minutes at room temperature </LI><LI> Transfer supernatant to a new microcentrifuge tube </LI><LI> Analyze samples by <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#Sodiumdodecylsulfatepolyacrylamidegelelectrophoresis%20%28SDS-PAGE%29" target="_blank">SDS-PAGE.</A> Use 3 - 10 µl per sample </LI></UL><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="GC-MS"><DIV class="show"><A href="#GC-MS"> Gas-Chromatography (GC) - Mass Spectroscopy (MS) </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6> Gas-Chromatography (GC) - Mass Spectroscopy (MS) 
        </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><LI>Modified from <A href="http://www.sciencedirect.com/science/article/pii/S0168165612000703" target="_blank">Kondo et al. (2012)</A></LI><LI>Sample preparation</LI><UL><LI>Supernatant from cultivation (100 µl) was extracted with 900 µl of GC standard-grade acetone containing 0.1 % 2-butanol as an internal standard</LI><LI>150 µl were used for GC-MS analysis</LI></UL><LI>GC-MS analysis was realized with the following settings</LI></UL><TABLE style="background-color:transparent; margin-left:10%; margin-right:10%" type="disc"><TBODY><TR><TH colspan="2">AS 2000 (Autosampler)</TH></TR><TR><TD>sample volume</TD><TD> 1 µl</TD></TR></TBODY></TABLE><TABLE style="background-color:transparent; margin-left:10%; margin-right:10%" type="disc"><TBODY><TR><TH>Trace GC 2000</TH></TR><TR><TD>column</TD><TD>Stabilwax® (Crossbond Carbowax - PEG)</TD></TR><TR><TD>(30 m, 0.25 mm internal diameter, 0.25 µm film thickness)</TD></TR><TR><TD>Carries Gas</TD><TD>Helium (1 ml / min)</TD></TR><TR><TD>Oven temperature</TD></TR><TR><TH>rate [°C]</TH><TH>temp [°C]</TH><TH>hold time [minutes]</TH></TR><TR><TD>initial</TD><TD>60</TD><TD>3.00</TD></TR><TR><TD>ramp 1</TD><TD>10.0</TD><TD>100</TD><TD>0.00</TD></TR><TR><TD>ramp 2</TD><TD>120.0</TD><TD>280</TD><TD>2.00</TD></TR></TBODY></TABLE>Injector temperature250°CSplit flow20 ml / minMS transfer line250°C<TABLE style="background-color:transparent; margin-left:10%; margin-right:10%" type="disc"><TBODY><TR><TH colspan="2">Polaris Q (Ion trap MS)</TH></TR><TR><TD>Ion source</TD><TD>220°C</TD></TR><TR><TD>Mass range</TD><TD>25-100</TD></TR><TR><TD>start time</TD><TD>2,5 min</TD></TR></TBODY></TABLE><UL style="margin-left:10%; margin-right:10%" type="disc"><LI>With these settings we generated a calibration line with the following concentrations for further analysis and quantification of our samples.</LI><UL><LI>0,5 % isobutanol</LI><LI>0,1 % isobutanol</LI><LI>0,05 % isobutanol</LI><LI>0,01 % isobutanol</LI><LI>0,001 % isobutanol</LI></UL></UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="HisTrapFFPurification"><DIV class="show"><A href="#HisTrapFFPurification"> His Trap FF Purification </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>His Trap FF Purification
        </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><LI>Column preparation</LI><UL><LI>1 h Incubation with 1 M NaOH</LI><LI>Wash with 10 ml Binding Buffer</LI><LI>Wash with 10 ml distilled water</LI><LI>Strip the column with 10 ml Stripping Buffer</LI><LI>Wash with 10 ml Binding Buffer</LI><LI>Wash with 10 ml distilled water</LI><LI>Recharge with 2.5 ml 0.1 M NiSO<SUB>4</SUB></LI><LI>Wash with 5 ml distilled water</LI><LI>Wash with 5ml Binding Buffer</LI><LI>Perform a blank run:</LI><UL><LI>Wash with 5 ml distilled water (After Ethanol storage)</LI><LI>Wash with 5 ml Binding Buffer</LI><LI>Wash with 5 ml Elution Buffer 5</LI><LI>Equilibrate with 10 ml Binding Buffer</LI></UL><LI>Store in 20% ethanol</LI></UL><LI>Sample preparation:</LI><UL><LI>Add 5 ml Binding Buffer to each gram of pellet</LI><LI>Add 0.2 mg/ml lysozyme, 20 µg/ml DNAse, 1 mM MgCl<SUB>2</SUB>, 1 mM PMSF</LI><LI>Stir for 30 min at +4 °C up to +20 °C (depending on the protein)</LI><LI>Centrifuge 30 min at 4 °C</LI><LI>Adjust pH to 7.4 - 7-6</LI><LI>Apply the lysate on the column immediatly</LI></UL><LI>Sample run:</LI><UL><LI>Wash with 5 ml distilled water</LI><LI>Equilibrate with 5 ml Binding Buffer</LI><LI>Apply the lysate on the column</LI><LI>Wash with Binding Buffer until the absorbance reaches a steady baseline (10 ml)</LI><LI>Elute with 5 ml Elution Buffer gradient (Elution Buffer 1 – 5)</LI></UL></UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="HPLC"><DIV class="show"><A href="#HPLC">High performance liquid chromatography (HPLC)</A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>High performance liquid chromatography (HPLC) </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc">
       The HPLC was used for substrate analytics. We were able to detect glucose, xylose, succinate and fumarate in the media.  
       <LI>Sample preparation:</LI><UL><LI>Centrifuge 1.5 ml of bacteria culture for 5 minutes at 14.000 rpm</LI><LI>Transfer supernatant to a fresh tube. Avoid to resuspend the cells</LI><LI>Add 2 µl of sodium azide (100g/ L) to each sample.</LI><LI>Store the samples at -20 °C untill you analyse them by HPLC</LI></UL><LI>HPLC measurement:</LI><UL><LI>750 µl of each sample are transferred into HPLC vials with a septum</LI><LI>The samples are determined in duplicates together with two sets standards á three concentrations</LI><LI>The concentration of the standards depends on the maximum expected carbon source concentration in   
                  the media</LI></UL><LI>Sample run:</LI><UL><LI>It was used the <A href="http://www.mn-net.com/tabid/8628/default.aspx"><I>VA 300/7.7 NUCLEOGEL SUGAR 810 H</I></A> column for sample separation</LI><LI>Eluent was 3.5 mM H<SUB>2</SUB>SO<SUB>4</SUB></LI><LI>The column was heated at 65 °C and the injected sample volume was 20 µl</LI><LI>Detection was achieved by conductivity measurement</LI></UL>
        For performing enzyme assays with purified enzymes or crude cell extracts we were using following HPLC-MC system.
        <UL><LI>HPLC: LaChromUltra (Hitachi Europe, United Kingdom)</LI><LI>Analysis: microTOF-Q hybrid quadrupole / time of flight mass spectrometer (Bruker Daltonics, Bremen, Germany)</LI><LI>Ionization: Electrospray ionization (ESI)</LI><LI>Column: SeQuant ZICpHILLIC column (150 by 2.1 mm, Merck KGaA, Darmstadt, Germany)</LI><LI>Solutions: 10 mM Ammonium bicarbonate solution (eluent A), acetonitrile (eluent B)</LI><LI>Injection volume: 2 µl</LI><LI>Flow rate: 150 µl min<SUP>-1</SUP></LI><LI>Gradient [t<SUB>min</SUB>, % of eluent B]: t<SUB>0</SUB>: 80%, t<SUB>30</SUB>: 10%, t<SUB>35</SUB>: 10%, t<SUB>40</SUB>: 80%, t<SUB>60</SUB>: 80%</LI><LI>Negative ionization mode</LI><LI>Internal mass calibration: Formate (0.1 M), in 50% (vol/vol) isopropanol</LI></UL></UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="Matrix-assistedLaserDesorption/Ionization–Timeofflight (MALDI-TOF)"><DIV class="show"><A href="#Matrix-assistedLaserDesorption/Ionization–Timeofflight (MALDI-TOF)"> Matrix-assisted Laser Desorption/Ionization – Time of flight (MALDI-TOF) </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6> Matrix-assisted Laser Desorption/Ionization – Time of flight (MALDI-TOF)
        </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><LI><U> Tryptic digest of gel lanes for analysis with MALDI-TOF: </U></LI><UL><LI> Make sure to work under a fume hood </LI><LI>Do not work with protective gloves to prevent contamination of your sample with plasticizers</LI><LI>Reaction tubes have to be cleaned with 60 % (v/v) CH<SUB>3</SUB>CN and 0.1 % (v/v) TFA. Afterwards the solution has to be removed completely followed by evaporation of the tubes under a fume hood. Alternatively microtiter plates from Greiner® (REF 650161) can be used without washing</LI><LI>Cut out the protein lanes of a Coomassie-stained SDS-PAGE using a clean scalpel. Gel parts are transferred to the washed reaction tubes. If necessary cut the parts to smaller slices</LI><LI> Gel slices should be washed two times. Therefore add 200 µl 30 % (v/v) acetonitrile in 0,1 M ammonium hydrogen carbonate each time and shake lightly for 10 minutes. Remove supernatant and discard to special waste </LI><LI> Dry gel slices at least 30 minutes in a Speedvac </LI><LI> Rehydrate gel slices in 15 µl Trypsin-solution followed by short centrifugation </LI><UL><LI> Trypsin-solution: 1 µl Trypsin + 14 µl 10 mM NH<SUB>4</SUB>HCO<SUB>3</SUB></LI><UL><LI> For this solution solubilize lyophilized Trypsin in 200 µl of provided buffer and activate Trypsin for 15 minutes at 30 °C. For further use it can be stored at -20 °C </LI></UL></UL><LI> Gel slices have to be incubated 30 minutes at room temperature, followed by incubation at 37 °C over night </LI><LI> Dry gel slices at least 60 minutes in a Speedvac </LI><LI> According to the size of the gel slice, add 5 - 20 µl 50 % (v/v) ACN / 0,1 % (v/v) TFA </LI><LI> Samples can be used for MALDI measurement or stored at -20 °C </LI></UL><LI><U> Preparation and Spotting for analysis of peptides on Bruker AnchorChips: </U></LI><UL><LI> Spot 0,5 - 1 µl of sample aliquot </LI><LI> Add 1 µl HCCA matrix solution to the spotted sample aliquots. Pipet up and down approximately five times to obtain a sufficient mixing. Be careful not to contact the AnchorChip. Note: Most of the sample solvent needs to be gone in order to achieve a sufficiently low water content. When the matrix solution is added to the previously spotted sample aliquot at a too high water content in the mixture, it will result in undesired crystallization of the matrix outside the anchor spot area </LI><LI> Dry the prepared spots at room temperature </LI><LI> Spot external calibrants on the adjacent calibrant spot positions. Use the calibrant stock solution (Bruker’s “Peptide Calibration Standard II”, Part number #222570), add 125 µl of 0.1 % TFA (v/v) in 30 % ACN to the vial. Vortex and sonicate the vial
</LI><LI> Mix the calibrant stock solution in a 1:200 ratio with HCCA matrix and deposit 1 µl of the mixture onto the calibrant spots </LI><LI> Analyze samples in ultrafleXtreme by Bruker Daltonics </LI></UL><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="microscopy"><DIV class="show"><A href="#microscopy"> Microscopy </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>Microscopy
        </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc">

For the fluorescence microscopy of our GFP targeted carboxysome  we used two different miscroscope, a Leica DMI 6000 SD and a structured illumination microscope (DeltaVision|OMXv4.0 BLAZE, GE Healthcare). The preparation of the samples were treated identically.
<UL><LI>Take 1 ml of the culture (you can choose different time points).</LI><LI>Centrifuge at top speed for 5 min.</LI><LI>Discard supernatant and resuspend in 600 µl 1x <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#PBSBuffer" target="_blank">PSB buffer</A>.</LI><LI>Prepare your object plates with 1% agarose for a fixation of the cells.</LI><LI>Take 5-10 µl of the washed culture for the microscopy (maybe dilute it with PBS buffer before using).</LI></UL></UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="NPNAssay"><DIV class="show"><A href="#NPNAssay"> NPN-Assay </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>NPN-Assay
        </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc">

NPN membrane permeability assay
<UL><LI>inoculate 20 ml of an overnight culture with 1 ml preculture</LI><LI>incubate cells at 37 °C until OD<SUB>600</SUB> of 0.5</LI><LI>centrifugation of whole culture volume (10 min at 3000 g)</LI><LI>discard entire supernatant</LI><LI>resuspend pellet in the same volume <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#PBSBuffer">PBS Buffer</A></LI><LI>repeat washing steps 3 to 5 twice</LI><LI>mix washed cells with 10 mM NPN stock solution to a final concentration of 10 μM in a UV cuvette </LI><LI>fluorescence measurement with <A href="http://www.ssi.shimadzu.com/products/product.cfm?product=rf5301">Shimadzu fluorescence spectrophotometer</A></LI><UL><LI>sample volume: 1 ml</LI><LI>Excitation: 355 nm</LI><LI>Emission Bandwidth: 380 - 500 nm</LI></UL></UL><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="NAD/NADH-Glo™ Assay"><DIV class="show"><A href="#NAD/NADH-Glo™ Assay"> Promega NAD/NADH-Glo™ Assay</A> <A href="http://www.promega.de/products/cell-health-and-metabolism/metabolism-assays/nad_nadh_glo-assay/" target="_blank">(Promega)</A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>Promega NAD/NADH-Glo™ Assay</H6></A> <A href="http://www.promega.de/products/cell-health-and-metabolism/metabolism-assays/nad_nadh_glo-assay/" target="_blank"><FONT size="3px">(Promega)</FONT></A></DIV><DIV class="content">
Basic principle of the enzymatically detection assay:
<DIV class="element" style="padding:10px; width:350px; text-align:center; margin-bottom:20px; margin-left:auto; margin-right:auto"><FONT size="1">NAD/NADH-Glo™ Assay</FONT></DIV><UL style="margin-left:10%; margin-right:10%; type:disc" type="disc">
We used the NAD/NADH-Glo™ Assay for the determination of the NAD<SUP>+</SUP> and NADH levels in the cells during the cultivation in different growth phases.</UL><H8>Protocol:</H8><UL>
            The 1st step, before the assay can be performed is the lysis of the cells.
               <LI>1 ml of bacteria culture (OD<SUB>600</SUB>= 0,375 ≈ 3 • 10<SUP>3</SUP> cells)get   
                   pelleted by centrifugation</LI><LI>Remove supernatant and add 300 µl bicarbonatebuffer + 1% DTAB  
                   (dodecyl(trimethyl)azanium bromide)</LI><LI>Mix thoroughly for cell lysis </LI><LI>Assay the neutralized samples using the NAD/NADH-Glo Assay by transferring 30 µl of each 
                   sample to the wells of a 96-well white luminometer plate and add 30µl of NAD/NADH-Glo 
                   Detection Reagent. </LI><LI> Incubate at room temperature and read luminescence after 30 to 60 minutes. </LI><LI> Determine NAD/NADH ratios by comparing RLU, or calculate the concentrations by comparison to a  
                    standard curve.</LI><LI>This method is used, if you want to measure the total amount of NAD<SUP>+</SUP> and NADH. For the determination of the individual NAD<SUB>+</SUB> and NADH concentrations follow the description below. </LI></UL><H8>Individual determination of NAD<SUP>+</SUP> and NADH</H8><UL><LI>After the cells got mixed briefly transfer two 100µl  aliquots to each of two new tubes.(R1 and R2)</LI><LI>Add 100µl HCl (0.4M) to R1 for acid treatment</LI><LI>Heat both aliquots at 60 °C for 15 minutes</LI><LI>Cool at room temperature for 10 minutes</LI><LI>Neutralize R1 with 100 µl trizma base (0.5M). This sample now contains the oxidized form NAD.</LI><LI>Neutralize R2 with 200µl of a 1:1 mixture of HCl (0.5M) and trizma base (0.5M).  This sample now contains the oreduced form NADH</LI><LI>Both samples can be assayed as already described.</LI>
This protocol is taken from &quot;Bioluminescent Nicotinamide Adenine Dinucleotide Detection in Bacteria&quot; of the Promega corporation. All measurements were carried out with the <A href="http://www.promega.de/products/pm/fluorometers-luminometers-multimode-readers/glomax-discover/" target="_blank">GloMax® Discover Multimode-Reader</A>.
          </UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="RuBisCOactivityassay"><DIV class="show"><A href="#RuBisCOactivityassay"> RuBisCo activity assay </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6> RuBisCO activity assay
        </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><UL><LI> Cultivation and cell lysis </LI><UL><LI> Grow overnight culture of your cells containing <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1465213" target="_blank">BBa_K1465213</A> or the construct T7_Hneap RuBisCo in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#LBmedium" target="_blank">LB medium</A> containing specific antibiotic. Grow at 37 °C </LI><LI> Dilute overnight culture 1:50 in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#LBmedium" target="_blank">LB medium</A> containing antibiotic and grow cells at 37 °C </LI><LI> When the culture reaches an OD<SUB>600</SUB> of 0.6-0.8, shift the temperature to 20 °C </LI><LI> Induce protein expression using the T7 promotor by adding rhamnose to a final concentration of 0.1 %. When using the ptac promotor, induction concentration is 0,5 mM.</LI><LI> Grow the culture at 20 °C overnight </LI><LI> Harvest cells by centrifugation at 4 °C </LI><LI> Discard supernatant</LI><LI> Add 5 ml 20 mM sodium phosphate, 500 mM NaCl buffer to each gram of cell pellet</LI><LI>Add 0.2 mg/ml lysozyme, 20 µg/ml DNAse, 1 mM MgCl<SUB>2</SUB>, 1 mM PMSF</LI><LI>Stir for 30 min at +4 °C </LI><LI>Centrifuge 30 min at 4 °C</LI><LI> Transfer supernatant (cell extract) to a new tube </LI></UL><LI> RuBisCo activity assay </LI><UL><LI> For the assay, make a reaction mixture containing:
</LI><LI> 750 µL bicine buffer (200 mM bicine solution, containing 0.4 mM EDTA and 1 mM DTT, pH 8.2) </LI><LI> 10 µL 2 M MgCl<SUB>2</SUB></LI><LI> 450 µL H<SUB>2</SUB>O </LI><LI> 40 µL 250 mM NaHCO<SUB>3</SUB></LI><LI> 200 µL 12.5 mM ribulose-1,5-bisphosphate solution </LI><LI> 20 µL cell extract </LI><LI> Add NaHCO<SUB>3</SUB>, ribulose-1,5-bisphosphate solution and cell extract last </LI></UL><LI> Incubate the reaction at 37 °C with continous purge of carbon dioxide.</LI><LI> Take samples in 5 min intervalls  Freeze them immediatly at -20 °C </LI><LI> Before measuring the sample in the HPLC, centrifuge the samples through a Amicon Ultra filter device with a 10 kDa cut-off. Further informations are given in this <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#ProteinBufferExchangeAndDesalting" target="_blank">protocol</A></LI><LI> Analyze samples in HPLC following the protocol for <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#HPLC" target="_blank">HPLC</A></LI></UL></UL></DIV></DIV></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="Sodiumdodecylsulfatepolyacrylamidegelelectrophoresis (SDS-PAGE)"><DIV class="show"><A href="#Sodiumdodecylsulfatepolyacrylamidegelelectrophoresis (SDS-PAGE)"> Sodium dodecyl sulfate polyacrylamide gelelectrophoresis (SDS-PAGE) </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>Sodium dodecyl sulfate polyacrylamide gelelectrophoresis (SDS-PAGE)
        </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><LI><U> Pouring the polyacrylamide gel: </U></LI><UL><LI> For each separating gel (12 %) aliquote: </LI><UL><LI>1.35 ml Bisacrylamid/Acrylamid (0.8 % , 30 %, at the ratio of 37.5:1)
                 </LI><LI>0.675 ml H<SUB>2</SUB>O</LI><LI>0.675 ml 1.88 M Tris-HCl (pH 8.8)</LI><LI>0.675 ml 0.5 % SDS</LI></UL><LI> Add 120 µl 10 % ammonium persulfate and 37,5 µl TEMED to each aliquote and mix </LI><LI> Pour the solution quickly into the gel casting form. Leave about 2 centimeters below the bottom of the comb for the stacking gel </LI><LI> Layer isopropanol on top of the gel </LI><LI> Leave the separating gel at room temperature for &gt;60 minutes to polymerize </LI><LI> Remove isopropanol and wash the surface of the separating gel with H<SUB>2</SUB>O. Wait until the surface is dry</LI><LI> For each stacking gel (5 %) aliquote: </LI><UL><LI>0.309 ml Bisacrylamid/Acrylamid (0.8 % , 30 %, at the ratio of 37.5:1) </LI><LI>0.726 ml H<SUB>2</SUB>O</LI><LI>0.375 ml 0.625 M Tris-HCl (pH 6.8)</LI><LI>0.375 ml 0.5 % SDS</LI></UL><LI> Add 60 µl 10 % ammonium persulfate and 22,5 µl TEMED to each aliquote and mix </LI><LI> Insert comb without getting bubbles stuck underneath</LI><LI> Leave the gel at room temperature for &gt;60 minutes to polymerize</LI><LI> For storage:</LI><UL><LI> Remove sealing and store the gel wrapped in moistened paper towel at 4 °C </LI></UL></UL><LI><U> Preparing the sample: </U></LI><UL><LI> Mix your protein mixture 3:1 with <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#PBJR%20buffer" target="_blank">PBJR buffer</A> (15 µl protein solution + 5 µl <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#PBJR%20buffer" target="_blank">PBJR buffer</A>) </LI><LI> Heat for 5 minutes at 95 °C</LI></UL><LI><U> Running the gel: </U></LI><UL><LI> Remove sealing, put the polymerized gel into gel box and pour <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#SDS-PAGE%20running%20buffer" target="_blank">SDS-PAGE running buffer</A> into the negative and positive electrode chamber </LI><LI> Remove comp without destroying the gel pockets </LI><LI> Pipet the SDS running buffer in the gel pockets up and down for flushing the gel pockets  </LI><LI> Pipet slowly 20 µl of the sample into the gel pockets </LI><LI> Make sure to include at least one lane with molecular weight standards (PageRuler Prestained Protein Ladder™ (Fa. Fermentas)) to determinate the molecular weight of the sample </LI><LI> Connect the power lead and run the stacking gel with 10 mA until the blue dye front enters the separating gel </LI><LI> Raise amperage up to 20 mA for running the separating gel </LI><LI> When the distance of the lowest molecular weight standard lane to the gel end is down to 0.5 cm stop the electrophoresis by turning off the power supply </LI></UL><LI><U> Staining the polyacrylamide gel (Colloidal Coomassie Brilliant Blue staining): </U></LI><UL><LI> After finishing the SDS-PAGE remove gel from gel casting form and transfer it into a box </LI><LI> Add 100 ml of the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#Colloidal%20Coomassie%20Brilliant%20Blue%20staining%20solution" target="_blank">Colloidal Coomaassie Brilliant Blue staining solution</A> to your polyacrylamid gel </LI><LI> Incubate the gel in the solution at room temperature until the protein bands got an intensive blue color. Shake the gel continuously during incubation </LI><LI> Remove the staining solution </LI><LI> Wash the gel with  7 % (v/v) acetic acid in H<SUB>2</SUB>O for decoloration </LI><LI> Incubate the gel in H<SUB>2</SUB>O (2-6 h) for bleaching the background. Shake the gel continuously during incubation.  If necessary replace the colored water with new one
       </LI></UL><DIV class="element" id="Cultivations" style="margin:10px 10px 10px 10px; padding:10px; background-color:#3fb498"><DIV id="text"><CENTER>Cultivations</CENTER></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="BiologSystem"><DIV class="show"><A href="#BiologSystem"> Biolog System </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6> Biolog System
        </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc">
This protocol is based on the technical guide from <A href="http://www.biolog.com/products-static/phenotype_microbial_cells_use.php">Biolog</A><UL><LI>Grow the strain of interest on a BUG+B (Biolog Universal Growth Medium + 5 % Sheep blood) agar plate by streaking out for single colonies</LI><LI>Incubate overnight at 37 °C </LI><LI>Pick single colonies with sterile swab</LI><LI>Transfer them into sterile tube containing 10 ml 1.0x IF-0a (mix 8 ml 1.2x IF-0a with 2 ml sterile water) to adjust transmittance to 42 %. This creates solution A.</LI><LI>Mix 144 μl Dye Mix A for Gram-negative bacteria and 1.856 ml sterile water with 8 ml 1.2x IF-0a solution. This creates solution B.</LI><LI>Mix 1.8 ml of solution A with 9 ml of solution B to achieve a final cell density of 85 % transmittance. This creates solution C.</LI><LI>Inoculate Biolog PM plate (96 wells) with 100 μl suspension C per well 
incubate loaded plate for 48 h in the OmniLog PM system </LI></UL><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="Pretreatment of the Nafion® membrane"><DIV class="show"><A href="#Pretreatment of the Nafion® membrane">Pretreatment of the Nafion® membrane</A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6> Pretreatment of the Nafion® membrane
        </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><UL><LI>Slightly boiling (≈ 80°C) in 3 &amp;percent; H<SUB>2</SUB>O<SUB>2</SUB> for 1 hour</LI><LI>Slightly boiling (≈ 80°C) in deionized water for 2 hours</LI><LI>Slightly boiling (≈ 80°C) in 0.5M H<SUB>2</SUB>SO<SUB>4</SUB></LI>

The membrane needs to be rinsed between the boiling steps and after the final boiling step. After the pretreatment the membrane needs to be stored in deionized water all the time and must not dry.


</UL><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="AnaerobicCultivation"><DIV class="show"><A href="#AnaerobicCultivation">Anaerobic cultivation of <I>Escherichia coli</I></A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6> Anaerobic cultivation of <I>Escherichia coli</I></H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><UL><LI>Hole anaerobic work takes place in a two-hand <A href="http://www.sigmaaldrich.com/catalog/product/aldrich/z530220?lang=de&amp;region=DE">Glove Bag</A>.</LI><LI>All media and buffer has to be degased  with nitrogen (N<SUB>2</SUB>) gas via a sparger befor starting cultivation.</LI><LI>Grow preculture under aerobic condition at 37°C.</LI><LI>Cells were cultivated in a gas-tight 15 ml tube additional sealed with parafilm.</LI><LI>Cultivation volume is about 10 mL. Fill cultivation tube with 8.5 ml steril degased cultivation medium.</LI><LI>When preculture reaches OD<SUB>600</SUB> of 0.6-0.8, take 1.5 ml into a steril tube.
</LI><LI>Centrifugate 1 min at 5,000 rpm and discard complete supernatant.</LI><LI>resuspend pellet in 1.5 ml steril degased <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#PBSBuffer">PBS buffer</A>.</LI><LI>Repeat previous step but resuspend in steril, degased cultivation medium. This two steps avoid residual oxygen in the inoculum.</LI><LI>Inoculate culture with 1.5 ml washed and resuspended cells at OD<SUB>600</SUB> of 0.1.</LI><LI>Tubes are filled and opened for sampling only under nitrogen atmosphere.</LI><LI>Tubes are incubated in a shaker at 37°C.</LI><LI>At the beginning of anaerobic cultivation residual oxygen from preculture would be consumed. Therefore faster growth could be observed in the first period of cultivation until cells shift to anaerobic metabolism.</LI><LI>There should be more frequent sampling at the beginning about 4 to 6-hour intervalls (two times) after inoculation.</LI><LI>Regular sampling takes place at 1-day intervalls.</LI><LI>Sampling volume is about 1 mL, using 500 μl for OD<SUB>600</SUB> measurement and 500 μl for HPLC analysis.</LI><LI>After sampling process tubes are overflowed with nitrogen gas, tightly closed and sealed with parafilm again.</LI><LI>Tubes are further incubated in a shaker at 37°C.</LI></UL><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><DIV class="tab" id="Cultivation for Expression of recombinant proteins"><DIV class="show"><A href="#Cultivation for Expression of recombinant proteins"> Cultivation for Expression of recombinant proteins </A></DIV><DIV class="hide"><A style="font-size:24px" href="#"><H6>Cultivation for Expression of recombinant proteins
        </H6></A></DIV><DIV class="content"><UL style="margin-left:10%; margin-right:10%" type="disc"><UL><LI> This protocol is based on the technical bulletin from <A href="http://www.promega.de/~/media/files/resources/protocols/technical%20bulletins/101/single%20step%20competent%20cells%20protocol.pdf/" target="_blank">Promega</A> concerning protein expression under control of the T7 promotor </LI><LI> Grow overnight culture of your cells with the plasmid containing the genes for the recombinant protein in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#LBmedium" target="_blank">LB medium</A> containing specific antibiotic. Grow at 37 °C </LI><LI> Dilute overnight culture 1:50 in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#LBmedium" target="_blank">LB medium</A> containing antibiotic and grow cells at 37 °C </LI><LI> When the culture reaches an OD<SUB>600</SUB> of 0.6-0.8, shift the temperature to 20 °C </LI><LI> Induce protein expression using the T7 promotor by adding rhamnose to a final concentration of 0.1 %. When the ptac promotor is used, standard induction concentrations are 0,5 mM or 1 mM. </LI><LI> Grow the culture at 20 °C overnight </LI><LI> Harvest cells by centrifugation at 4 °C </LI><LI> Discard supernatant and freeze pellet at -20 °C </LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols">http://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-CeBiTec/Notebook/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-CeBiTec/Notebook/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Notebook/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Notebook/Protocols&amp;oldid=378156" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>