"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer
From 2014.igem.org
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Modeling
Parts
Data Page
Achievements
Home
Summary
Applications
Achievements
Bielefueli
Project
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Results
Team
Members
Subteams
Pictures & Videos
Bielefeld & University
Official Team Profile
Acknowledgements
Contact
Policy & Practices
Experts
Interviews
NRW-Day
Pupils Acedemy
How To Wiki
namu
SYNENERGENE
Applications
Vignettes
Notebook
Journal
Protocols
Media
Kits & Enzymes
Strains & Constructs
Primer
Acronyms
Organisms
Additional
Partners
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Partners
Module I - Reverse Microbial Fuel Cell (rMFC)
Overview
Electron transfer
Construction
Construction of an electrophilic E. coli strain
Our first module deals with the construction of an E. coli strain, which is able to accept electrons stimulating its metabolism. We considered two different electron transfer systems: direct and indirect electron transfer.
Direct electron transfer in bacteria is a very complex and not completely cleared up so far. Due to this we focused on the indirect electron transfer via a mediator, which is reduced at the electrode in a electrobiochemical reactor and would be reoxidized again by bacterial cells.( Park et al., 1999 ) In gram-negative bacteria E. coli there are two membranes with a periplasmatic space between them, which has to be overcome in the course of the electron transfer.
We worked on three different mediators: neutral red, bromphenol blue and cytochromes. Additionally we constructed an electropholic E. coli strain, which shows an increased metabolic activity growing with electric power, which has been proven in our H-cell reactor ( Park et al., 1999 ).
The electron transfer system consists of different steps. First of all the reduced mediator has to cross the outer membrane of the E. coli cell. For that we used the outer membrane porine OprF ( BBa_K1172507 ) provided by the iGEM Team Bielefeld-Germany 2013 . Crossing the periplasmatic space, the mediator adsorb at the inner membrane of the E. coli cell. The mediator functions as an electron donor for the over expressed fumarate reductase. In this step succinate is produced in the cytoplasm. Reduction of fumarate into succinate creates a loop into the citric acid cycle, because succinate would be reoxidized again by the succiante dehydrogenase. Succinate excretion is avoid, because of the knockout of the C4 carboxylate transporter DcuB. In the reaction of succinate dehydrogenase electrons are transferred to FAD+ generating FADH2, which enter the electron transport chain. The electron transport facilitates proton translocation over the inner bacterial membrane. The proton motoric force is used by ATP synthase. Generated ATP and reductive power in the bacterial cell leads to an increasing metabolic activity.
E. coli KRX ΔdcuB::oprF strain
We investigated the effect of the C4 carboxylate transporter DcuB knockout on E. coli KRX. Furthermore we showed the integration of the outer membrane porine OprF ( BBa_K1172507 ) into the bacterial genome by replacing the gene of E. coli C4 carboxylate antiporter DcuB. So we did knockout and insertion in a one-step process.  The successful knockout of the DcuB antiporter and simultaneous insertion of BBa_K1172507 was shown with PCR analysis, DNA sequencing and phenotypic investigation via Biolog analysis and anaerobic cultivation in M9 minimal media with fumarate supplemented. Substrates and products were analyzed by HPLC.  The electrobiochemical behavior of E. coli KRX with knocked out C4 carboxylate antiporter DcuB was tested in a H-cell reactor .
Preparation of knockout deletion cassette
We decide to knock out the C4 carboxylate transporter DcuB to prevent succinate export. Besides we inserted the outer membrane porines OprF ( BBa_K1172507 ) into the genome. Both could be realized with the Genebridge RedET-System .  For simultaneous knockout and insertion a deletion cassette had to be designed and established. We used overlap extension PCR to amplify the complete deletion cassette ( Bryksin & Matsumura, 2010 ). We used kanamycin as an antibiotic selective marker amplified from the plasmid Flp705 using the Genebridge RedET-System protocoll .  We designed the primers ( BBa_K1465107 , BBa_K1465108 , BBa_K1465109 , BBa_K1465110 ) with complementary 5´extensions for both fragments to connect them in overlap extension PCR. The amplified deletion cassette has homologous sites for recombination with the dcuB gene in the E. coli KRX genome.
Verification of knockout and deletion cassette
Figure 1:  Results of the colony PCR for
analysis of dcuB genome area
analyzed via agarose gelelectrophoresis.
E. coli KRX wildtype band
is shown on the rigth of about 2391 bp.
E.coli KRX ΔdcuB::oprF band is shown
on the left at 4046 bp.
The gene of the C4 carboxylate transporter DcuB, which exchange fumarate against succinate, has a size of 1341 bp. We amplified the genome area of the dcuB gene with primers ( dcuB_del_kon1 and dcuB_del_kon2 ), which bind 530 bp upstream and 520 bp downstream of the dcuB gene. When using the E. coli KRX wildtype as a template the resulting PCR product has a size of 2391 bp, which could be demonstrated by agarose gelelectrophoresis.  However the E. coli KRX knockout strain ( E.coli KRX ΔdcuB::oprF ) showed a 4046 bp PCR product analyzed by agarose gelelectrophoresis. Figure 1 shows the result of the agarose gelelectrophoresis with E. coli KRX wildtype and the modified strain E.coli KRX ΔdcuB::oprF .
The PCR product of E.coli KRX ΔdcuB::oprF genome amplified with primer dcuB_del_kon1 and dcuB_del_kon2 (4046 bp) and is composed of the antibiotic cassette (1637 bp), BBa_K1172507 (1359 bp) and upstream and downstream spacer elements (520 bp and 530 bp).
DNA sequencing of deletion cassette from E.coli KRX ΔdcuB::oprF showed the expected results.
NPN-Assay
The funtionality of the outer membrane porin OprF ( BBa_K1172507 ) in E.coli KRX ΔdcuB::oprF was investigated with a NPN-Uptake-Assay ( Cheng et al., 2005 ).
1-N-Phenylnaphthylamine (NPN) changes fluorescence activity between aqueous and hydrophobic milieus. There is only minor fluorescence in aqueous solution, but the transport in the hydrophobic periplasmatic space causes an increased fluorescence. So NPN fluorescence is a good indicator for membrane permeability. Expression of the outer membrane porines OprF effect an increasing membrane permeability.( Loh et al., 1984 )
Figure 2: Results of the NPN-Uptake-Assay . Comparison between Escherichia coli KRX wildtype and Escherichia coli KRX with BBa_K1172507 , BBa_K1172502 and Escherichia coli KRX with genome integrated oprF gene ( E.coli KRX ΔdcuB::oprF ).
Figure 2 shows the results of the NPN-Uptake-Assay. Higher fluorescence emission and higher membrane permeability could be observed with increasing promotor strength for oprF. The genome integrated oprF shows mid-level fluorescence emission and membrane permeability. Highest fluorescence levels could be measured with the oprF gene on a high-copy pSB1C3 plasmid under control of the T7 promotor.
Genome integrated oprF showed almost the same membrane permeability as the constitutive expressed oprF with BBa_K1172507 . There even seems to be a marginal higher expression of the genome integrated oprF in contrast to the plasmid coded oprF although the same constitutive promotor ( BBa_J23104 ) is used. This could be explained by physiological condition of the cell. Constitutive expression of the high-copy oprF causes cell stress. Protein expression could be automatically downregulated by the cells. The single-copy genome integrated oprF showed a reduced expression level, therefor cell stress is also smaller. As a consequence E. coli cells showed a more effective expression of the outer membrane porin OprF adjusted on physiological cell condition.
Phenotypic characterization with Biolog® system
Biolog® Microbial ID System is a simple and fast method for the characterization of different bacteria, yeast and fungi species. It is based on respiration in the presence of different metabolic relevant substances for example carbon sources, nitrogen sources or toxines. The system uses redox chemistry wherein a tetrazolium dye is reduced when significant respiration occurs. Tetrazolium dye change its colour, because the cells generate reductive conditions in the course of the electron transport chain. A tetrazolium cation is reduced to formazan by dehydrogenase from the respiratory complex I. This allow a statement if respiration occurs in the presence of different substances.
The E. coli KRX wildtype and E.coli KRX ΔdcuB::oprF were incubated in the Biolog® system to test respiration in the presence of fumarate. It was expected that the E.coli KRX ΔdcuB::oprF shows no activity in the presence of fumarate because  of the knockout of the C4 carboxylate transporter dcuB, which makes fumarate uptake impossible for E. coli cells.
The results of the Biolog® analysis are shown in figure 3.
Figure 3: Results of the Biolog® analysis of E.coli KRX ΔdcuB::oprF in comparison to Escherichia coli KRX wildtype. Respiratory activity is shown under influence of fumarate.
Biolog® analysis showed that there is no significant respiratory activity of E.coli KRX ΔdcuB::oprF in the presence of fumarate during the first 35 hours of incubation. Later on respiration occurs up to 50% of maximal respiratory activity of the E. coli KRX wildtype after 48 hours of incubation at 37°C.
The results in figure 3 demonstrate that the knockout of C4 carboxylate antiporter dcuB was successful. Respiration activity after 35 hours could be explained by a change in metabolism of E. coli cells. Based on cell stress in minimal medium bacteria use other ways for uptake of fumarate. It is possible that another transporter compensates the activity of the DcuB transporter, but activation and expression of alternative pathways require a space of time, espacially under limited conditions in minimal medium. This could explain the delayed respiration of E.coli KRX ΔdcuB::oprF .
Anaerobic cultivation
We cultivated E. coli KRX ΔdcuB::oprF under anaerobic conditions . We used M9 minimal medium with 50 mM glucose. Among growth characteristic we focused on metabolite analysis. Under anaerobic conditions E. coli cells use different alternative electron acceptors instead of oxygen. In part the bacteria use fumarate respiration, whereby fumarate is reduced into succinate. There are also other potential pathways for bacteria to release their electrons, for example anorganic compounds like nitrate (NO3-) or sulfate (SO42-).( Gottschalk et al., 1986 ) We analyzed succinate concentration and glucose concentration in the culture supernatant via HPLC.
We expect succinate production of the E. coli KRX wildtype, but E. coli KRX ΔdcuB::oprF should not release succinate into the media because the C4 carboxylate transporter DcuB is knocked out. As a positive control we use E. coli ΔdcuB749::kan , an E. coli strain with a reviewed knockout in dcuB gene. This strain also should not show any succinate transport into the medium.
Figure 4: Results of the anaerobic cultivation of Escherichia coli KRX wildtype in M9 minimal medium with 50 mM glucose. Glucose and succinate concentration were measured in duplicates with HPLC.
Figure 5: Results of the anaerobic cultivation of Escherichia coli ΔdcuB749::kan in M9 minimal medium with 50 mM glucose. Glucose and succinate concentration were measured in duplicates with HPLC.
Figure 6: Results of the anaerobic cultivation of Escherichia coli KRX ΔdcuB::oprF in M9 minimal medium with 50 mM glucose. Glucose and succinate concentration were measured in duplicates with HPLC.
Figure 7: Results of the anaerobic cultivation of Escherichia coli KRX ΔdcuB::oprF in M9 minimal medium with 50 mM glucose as compared to Escherichia coli KRX wildtype. Glucose and succinate concentration were measured in duplicates with HPLC.
Figure 4 shows the production of succinate by the Escherichia coli KRX wildtype under anaerobic condition as expected. Succinate is released into the medium via the C4 carboxylate transporter DcuB. Our constructed E. coli KRX ΔdcuB::oprF strain shows no succinate export under anaerobic conditions displayed in figure 6. This demonstrates a successful knockout of the dcuB gene in the E. coli KRX ΔdcuB::oprF strain. As positive control we got an E. coli strain from the KEIO collection with a verified knockout in the dcuB gene.
Figure 5 shows the result of the anaerobic cultivation and succinate concentration measurement of this E. coli strain. A very low succinate concetration in the culture supernatant could be observed. In comparison to the E. coli KRX wildtype there is a very low succinate export. Only traces of succinate could be measured attributed to transport of succinate by other transporters with reduced specifity.
Furthermore in Escherichia coli ΔdcuB749::kan the knockout is not over the complete dcuB gene. Insertion of kanamycin selection cassette was executed at position 749 of dcuB gene. So there could be a residual activity of DcuB because of residual expressed parts of the antiporter protein. However the knockout of the dcuB in E. coli KRX ΔdcuB::oprF is complete and residual activity is not possible as shown in figure 7. Comparison between the E. coli wildtype and E. coli KRX ΔdcuB::oprF is an obvious demonstration of a completely functional knockout of the C4 carboxylate antiporter DcuB.
Fumarate reductase (Frd) from Escherichia coli
Upon the expression of the fumarate reductase Frd in E. coli we analyzed the metabolic behavior under aerobic and anaerobic conditions .  Successful expression of the fumarate reductase Frd ( BBa_1465102 ) could be proven via SDS-PAGE .  Activity of the fumarate reductase displayed with HPLC analysis of fumarate consumption and succinate production in anaerobic cultivation of E. coli was shown with BBa_1465102 . Furthermore we investigated the fumarate reductase activity in different E. coli strains by phenotypic MicroArray (PM) analysis with a Biolog® system .
SDS-PAGE
The fumarate reductase could be detected in purified membrane and periplasmatic protein fractions. Proteins were fractioned by cold osmotic shock of E. coli KRX at different steps after induction of protein expression.
SDS-PAGE shows the expression of fumarate reductase in E. coli KRX  under control of the T7 promotor ( BBa_1465102 ). Fumarate reductase with a total molecular mass of 121 kD consists of four subunits. There are two large catalytic and water-soluble subunits, flavoprotein (66 kDa) and iron-sulfur protein (27 kDa). Two small membrane associated subunits (15 and 13 kDa) are not detectable in SDS-PAGE . ( Iverson et al., 1999 ).
Figure 8 shows the result of the SDS-PAGE of isolated membrane proteins via cold osmotic shock of E. coli KRX with BBa_1465102 . Membrane protein fractions at different times after induction of BBa_1465102 are displayed.
Figure 8: Results of the SDS-PAGE of purified membrane protein from Escherichia coli BBa_1465102 via cold osmotic shock . Expected size of fumarate reductase subunits A and B are 66 kDa and 27 kDA respectively. Increasing band at 66 kDa could be recognized attributable to subunit A. Increasing band between 40 and 55 kDa could be referable to dimer or trimer build-up by subunit B, C and D.
SDS-PAGE shows a significantly higher protein concentration in membrane extracts from E. coli overexpressing frd under control of T7 promoter ( BBa_1465102 ) after induction. This is caused by usually higher membrane protein concentration of cultivated cells attributable to higher optical density. Nevertheless, a strong overexpression band can be observed at the expected Frd size (subunit A) of about 66 kDa for BBa_1465102 , which can be traced back on the strong expression and overproduction of Frd. Besides an overexpression band can be recognized at a size of about 45 kDA. This could be explained by dimer formation of two or three Frd subunits, for example subunit B (27 kDa) with subunit C oder D (13 and 15 kDA). That implies that subunit B build up stable dimers with small membrane associated subunits C and D of about 40 kDa respectively 42 kDa. There seems to be no dimers of subunit A with other parts of the Frd.
Anaerobic cultivation
We cultivated E. coli KRX with BBa_1465102 under anaerobic conditions to characterize the activity of the fumarate reductase Frd ( BBa_1465102 ) under control of the T7 promotor. We used M9 minimal medium with 50 mM xylose as carbon source and 50 mM fumarate as substrate of the fumarate reductase Frd. We focused on growth characteristic and metabolite analysis. Under anaerobic conditions E. coli cells use different alternative electron acceptors instead of oxygen. In part the bacteria use fumarate respiration, whereby fumarate is reduced into succinate. There are also other potential pathways for bacteria to release their electrons, for example anorganic compounds like nitrate (NO3-) or sulfate (SO42-). As a result of this the expected level of produced succinate is very low. We analyzed fumarate and succinate concentrations in the culture supernatant via HPLC analysis .
We expect mid-level succinate production of the E. coli KRX wildtype under anaerobic conditions. Strong expression of Frd ( BBa_1465102 ) leads to an increasing succinate production and fumarate consumption.
As a control we are going to use E. coli ΔfdrA730::kan , an E. coli strain with a reviewed knockout in frdA gene. This strain does not show any fumarate reductase activity and so succinate concentraion should be on a low level. Furthermore we transform BBa_1465102 into E. coli ΔfdrA730::kan to compensate functional defect of fumarate reductase in this strain.
Unfortunately the results of the cultivartion of E. coli ΔfdrA730::kan and E. coli ΔfdrA730::kan with complementation by BBa_1465102 are not successful. HPLC analysis could not evaluated because of a mistake in the process.
Figure 9: Results of the anaerobic cultivation of Escherichia coli KRX wildtype in M9 minimal medium with 50 mM xylose and 50 mM fumarate. Induction occurred after 72 h with 0.1% rhamnose and 500 μM IPTG. Fumarate and succinate concentration were measured duplicated via HPLC.
Figure 10: Results of anaerobic cultivation of Escherichia coli KRX wildtype with strong expression of frd ( BBa_1465102 ) in M9 minimal medium with 50 mM xylose and 50 mM fumarate. Induction occurred after 72 h with 0.1% rhamnose and 500 μM IPTG. Fumarate and succinate concentration were measured  duplicated via HPLC.
Figure 11: Growth of the Escherichia coli KRX wildtype with strong expression of frd ( BBa_1465102 ) compared to Escherichia coli KRX wildtype under anaerobic condition in M9 minimal medium with 50 mM xylose and 50 mM fumarate. Induction occurred after 72 h with 0.1% rhamnose and 500 μM IPTG. Fumarate and succinate concentration were measured duplicated via HPLC.
Figure 12: Comparison of fumarate consumption and succinate production in anaerobic cultivation of the Escherichia coli KRX wildtype and Escherichia coli KRX wildtype with strong expression of frd ( BBa_1465102 ) in M9 minimal medium with 50 mM xylose and 50 mM fumarate. Induction occurred after 72 h with 0.1% rhamnose and 500 μM IPTG. Fumarate and succinate concentration were measured duplicated via HPLC.
As expected figure 9 shows the production of succinate by the Escherichia coli KRX wildtype under anaerobic condition with 50 mM xylose and 50 mM fumarate as sole carbon sources. Under anaerobic conditions fumarate respiration takes place. Growth displayed via OD600 shows rise in the first three days of anaerobic cultivation . After that cells were induced via 0.1% rhamnose and 500 μM IPTG. This leads to  a reduction in growth because of the expression load and the toxic effect of IPTG. Cultivation of the E. coli wildtype also should be induced because of better comparability. The same effect could be observed in cultivation of E. coli with BBa_1465102 , as shown in figure 10. There is also a drop in growth detectable. Comparison of both growth characteristics are shown in figure 11. It could be proven that there are no distinct differences in growth betweenE. coli KRX wildtype and E. coli KRX with BBa_1465102 . So statement about fumarate reductase activity in E. coli KRX with BBa_1465102 as measured by succinate production and fumarate consumption is possible.
Figure 12 shows comparison of succinate production and fumarate consumption between E. coli KRX wildtype and E. coli KRX with BBa_1465102 . This illustrates an increasing succinate production by E. coli KRX with BBa_1465102 after induction of T7 promotor. A higher value of succinate could be achieved after 8 days of anaerobic cultivation and also lower fumarate level was shown compared to E. coli KRX wildtype.
Phenotypic characterization with Biolog® system
Biolog® Microbial ID System is a simple and fast method for the characterization of different bacteria, yeast and fungi species. It is based on respiration in the presence of different metabolic relevant substances for example carbon sources, nitrogen sources or toxines. The system uses redox chemistry wherein a tetrazolium dye is reduced when significant respiration occurs. Tetrazolium dye change its colour, because the cells generate reductive conditions in the course of electron transport chain. Tetrazolium cation is reduced to formazan by dehydrogenase from respiratory complex I. This allow a statement if respiration occurs in the presence of different substances.
We tested the influence of fumarate on the Escherichia coli KRX wildtype and E. coli KRX with BBa_1465102 . To show activity of fumarate reductase under controll of the T7 promotor ( BBa_1465102 ), we incubated cells with fumarate in a Biolog® system for 48h. Results of the Biolog ® analysis are shown in figure 13.
E. coli KRX wildtype and E. coli KRX with BBa_1465102 show respiratory activity in the presence of fumarate. As unexpected E. coli KRX wildtype shows higher activity on fumarate as compared to BBa_1465102 . Respiration of E. coli KRX with BBa_1465102 seems to be fewer and occurs subsequently in comparison to wildtype.
Figure 13: Results of Biolog® analysis of E.coli KRX with BBa_1465102 in comparison to Escherichia coli KRX wildtype. Respiratory activity is shown under influence of fumarate.
Figure 13 shows the respiratory activity of E. coli KRX with strong expressed BBa_1465102 in comparison to the E. coli KRX wildtype. As expected the wildtype shows higher activity in presence of fumarate as the fumarate reductase expressing strain. This effect is attributed to the fumarate reductase activity.
Because the Biolog® Microbial ID System is based on the activity in the electron transport chain and reduction of tetrazolium dye, direct conclusions about fumarate consumption are not possible. But there are indirect hints for fumarate reductase activity via the strong expression of BBa_1465102 . Fumarate reductase naturally gets electrons out of electron transport chain and transfer them to fumarate generating succinate, which is transported out of the cells under anaerobic conditions. Under aerobic conditions succinate dehydrogenase oxidize succinate to fumarate again. This created loop by expression of BBa_1465102 leads to lower electron transfer activity in the respiratory chain. So there are less electrons available for reduction of the extracellular tetrazolium dye compared to the E. coli KRX wildtype.
H-cell reactor
Fumarate reductase activity could be demonstrated by cultivation of E. coli KRX BBa_K1465102 in our electrobiochemical H-cell reactor . We show that expression of fumarate reductase increases the electron transfer into E. coli cells via neutral red as a mediator. Results of these cultivations are shown under construction results .
Fumarate reductase (Fum) from Actinobacillus succinogenes
Besides the fumarate reductase (Frd) from Escherichia coli we worked on the fumarate reductase (Fum) from Actinobacillus succinogenes (Fum). As published by Park et al., 1999 the fumarate reductase from A. succinogenes shows high activity accepting electrical reduced neutral red as an electron donor and so stimulates the metabolism in a electrobiochemical reactor system. So we planed to compare these two different enzymes with regard to activity accepting neutral red or bromphenol blue as alternative electron acceptors in our electrobiochemical reactor system .
Adaptation of fum gene codon usage to E. coli KRX
The fumarate reductase Fum from A. succinogenes just as Frd from E. coli is a membrane associated enzyme, which consists of four subunits. Because there was no full genomic DNA available from A. succinogenes, we ordered the Fum gene via custom gene synthesis at gBlocks® Gene Fragments . Before ordering DNA we ensured for optimal codon usage adapted to E. coli by jcat online optimization tool . Furthermore we checked the sequence for illegal restriction sites (EcoRI, PstI, NotI, XbaI) and eliminated them if required.
Because the sequence is too large for synthesis in one part with gBlocks® system , we divided the sequence of fum into two different parts: fumA and fumBCD (also called fumB). FumA includes the coding sequence of the fumarate reductase (Fum) subunit A ( BBa_K1465103 ). However fumB consists of the fumarate reductase subunits B, C and D ( BBa_K1465104 ). The fumarate reductase subunits were designed with their own ribosome binding site respectively.
Cloning fumarate Reductase Fum
Two different subunits of the fumarate reductase Fum from A. succinogenes had to be cloned into the shipping vector pSB1C3 . Several problems occurred during the cloning of the ordered fumA and fumB into pSB1C3 via gibson assembly .
Cloning of fum was not successful for a long time. Unfortunately we could not realize the ligation of fumA and fumB creating an active fumarate reductase (Fum). So there are no characterizations of fumarate reductase (Fum) available.
Mediators
We use different mediators for electron transfer in our electrobiochemical reactor system. The electrochemical behavior of the mediators were analyzed in a H-cell reactor . We characterized the different mediators in combination with different electrode marterials to identifie the best suitable set up for cultivation of our E. coli strain in the electrobiochemical reactor system .
Overview
Electron transfer
Construction
Reference
Iverson et al., 1999. Structure of the Escherichia coli Fumarate Reductase Respiratory Complex. Science, vol. 284, pp. 1961-1966
Cheng et al., 2005. New antibiotic peptides, useful in treating or preventing a microbial or viral infections or in inactivating Gram-positive and -negative bacteria, protozoa, fungi and virus. patent  DE10360435 (A1) ― 2005-07-28
Loh et al., 1984. Use of the fluorescent-probe 1-Nphenylnaphthylamine to study the interactions of aminoglycoside antibiotics with the outer-membrane of Pseudomonas aeruginosa. Antimicrob. Agents Chemother., vol. 26, pp. 546-551
Gottschalk et al., 1986. Bacterial Metabolism. Springer-Verlag, New York, Berlin, Heidelberg, Tokyo
Park et al., 1999. Utilization of Electrically Reduced Neutral Red byActinobacillus succinogenes: Physiological Function of Neutral Red in Membrane-Driven Fumarate Reduction and Energy Conservation. Journal of Bacteriology, vol. 181, pp. 2403-2410
Retrieved from " http://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
