<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_rMFC_ElectronTransfer"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV style="height:100%" id="main_box"><DIV id="flag"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Overview" style="color:#000000"><FLAG class="flag_element1" style="margin-left:70px;"><P class="flag"><FONT color="#FFFFFF">Overview</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC" style="color:#000000"><FLAG class="flag_element2"><P class="flag"><FONT color="#FFFFFF">rMFC</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation" style="color:#000000"><FLAG class="flag_element3"><P class="flag"><FONT color="#FFFFFF">CO<SUB>2</SUB> Fixation</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Isobutanol" style="color:#000000"><FLAG class="flag_element4"><P class="flag"><FONT color="#FFFFFF">Isobutanol</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Biosafety" style="color:#000000"><FLAG class="flag_element5"><P class="flag"><FONT color="#FFFFFF">Biosafety</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Modelling" style="color:#000000"><FLAG class="flag_element6"><P class="flag"><FONT color="#FFFFFF">Modeling</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Parts" style="color:#000000"><FLAG class="flag_element7"><P class="flag"><FONT color="#FFFFFF">Parts</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/DataPage" style="color:#000000"><FLAG class="flag_element8"><P class="flag"><FONT color="#FFFFFF">Data Page</FONT></P></FLAG></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Achievements" style="color:#000000"><FLAG class="flag_element9"><P class="flag"><FONT color="#FFFFFF">Achievements</FONT></P></FLAG></A></DIV><UL id="nav"><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec"><FONT color="#FFFFFF">Home</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/ProjectSummary"><FONT color="#FFFFFF">Summary</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/Applications"><FONT color="#FFFFFF">Applications</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Achievements"><FONT color="#FFFFFF">Achievements</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Bielefueli"><FONT color="#FFFFFF">Bielefueli</FONT></A></LI></UL><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Overview"><FONT color="#FFFFFF">Project</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Overview"><FONT color="#FFFFFF">Overview</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/rMFC/Theory"><FONT color="#FFFFFF">rMFC</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/CO2-fixation/CarbonDioxide"><FONT color="#FFFFFF">CO<SUB>2</SUB> Fixation</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Isobutanol/Theory"><FONT color="#FFFFFF">Isobutanol</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Biosafety"><FONT color="#FFFFFF">Biosafety</FONT></A></LI><LI><ACTUAL><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Overview"><FONT color="#FFFFFF">Results</FONT></A></ACTUAL></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team"><FONT color="#FFFFFF">Team</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team/Members"><FONT color="#FFFFFF">Members</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Team/Subteams"><FONT color="#FFFFFF">Subteams</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/PicturesAndVideos"><FONT color="#FFFFFF">Pictures &amp; Videos</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/BielefeldAndUniversity"><FONT color="#FFFFFF">Bielefeld &amp; University</FONT></A></LI><LI><A href="https://igem.org/Team.cgi?year=2014"><FONT color="#FFFFFF">Official Team Profile</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Acknowledgements"><FONT color="#FFFFFF">Acknowledgements</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Contact"><FONT color="#FFFFFF">Contact</FONT></A></LI><LI style="width:155px;"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice"><FONT color="#FFFFFF">Policy &amp; Practices</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Experts"><FONT color="#FFFFFF">Experts</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/RadioInterview"><FONT color="#FFFFFF">Interviews</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/NRW-day"><FONT color="#FFFFFF">NRW-Day</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/ScholarAcademy"><FONT color="#FFFFFF">Pupils Acedemy</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Wikisession"><FONT color="#FFFFFF">How To Wiki</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Namu"><FONT color="#FFFFFF">namu</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene"><FONT color="#FFFFFF">SYNENERGENE</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/Applications"><FONT color="#FFFFFF">Applications</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/HumanPractice/Synenergene/TechnomoralScenario"><FONT color="#FFFFFF">Vignettes</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook"><FONT color="#FFFFFF">Notebook</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Journal"><FONT color="#FFFFFF">Journal</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols"><FONT color="#FFFFFF">Protocols</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media"><FONT color="#FFFFFF">Media</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/KitsAndEnzymes"><FONT color="#FFFFFF">Kits &amp; Enzymes</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs"><FONT color="#FFFFFF">Strains &amp; Constructs</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Primer"><FONT color="#FFFFFF">Primer</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Acrynom"><FONT color="#FFFFFF">Acronyms</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Organisms"><FONT color="#FFFFFF">Organisms</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Safety"><FONT color="#FFFFFF">Additional</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners"><FONT color="#FFFFFF">Partners</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/GoldSponsor"><FONT color="#FFFFFF">Gold Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/SilverSponsor"><FONT color="#FFFFFF">Silver Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/BronzeSponsor"><FONT color="#FFFFFF">Bronze Sponsors</FONT></A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Partners/Partners"><FONT color="#FFFFFF">Partners</FONT></A></LI><H1> Module I - Reverse Microbial Fuel Cell (rMFC) </H1><DIV id="main_menue" style="margin-left:270px; height:80px"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC" style="color:#000000"><DIV class="main_menueButton"><P class="buttoncenter"><FONT color="#FFFFFF">Overview</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer" style="color:#000000"><DIV class="main_menueButtonActual" style="width:150px"><P class="buttoncenter"><FONT color="#FFFFFF">Electron transfer</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/Construction" style="color:#000000"><DIV class="main_menueButton"><P class="buttoncenter"><FONT color="#FFFFFF">Construction</FONT></P></DIV></A></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6 style="font-size:30px">Construction of an electrophilic <I>E. coli</I> strain</H6><P>

Our first module deals with the construction of an <I>E. coli</I> strain, which is able to accept electrons stimulating its metabolism. We considered two different electron transfer systems: direct and indirect electron transfer. 
Direct electron transfer in bacteria is a very complex and not completely cleared up so far. Due to this we focused on the indirect electron transfer via a mediator, which is reduced at the electrode in a electrobiochemical reactor and would be reoxidized again by bacterial cells.(<A href="#Park1999">Park <I>et al.</I>, 1999</A>) In gram-negative bacteria <I>E. coli</I> there are two membranes with a periplasmatic space between them, which has to be overcome in the course of the electron transfer.
We worked on three different mediators: neutral red, bromphenol blue and cytochromes. Additionally we constructed an electropholic <I>E. coli</I> strain, which shows an increased metabolic activity growing with electric power, which has been proven in our <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/Construction#H-cellResults">H-cell reactor</A> (<A href="#Park1999">Park <I>et al.</I>, 1999</A>).
The electron transfer system consists of different steps. First of all the reduced mediator has to cross the outer membrane of the <I>E. coli</I> cell. For that we used the outer membrane porine OprF (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_K1172507">BBa_K1172507</A>) provided by the <A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Porins">iGEM Team Bielefeld-Germany 2013</A>. Crossing the periplasmatic space, the mediator adsorb at the inner membrane of the <I>E. coli</I> cell. The mediator functions as an electron donor for the over expressed fumarate reductase. In this step succinate is produced in the cytoplasm. Reduction of fumarate into succinate creates a loop into the citric acid cycle, because succinate would be reoxidized again by the succiante dehydrogenase. Succinate excretion is avoid, because of the knockout of the C4 carboxylate transporter DcuB. In the reaction of succinate dehydrogenase electrons are transferred to FAD<SUP>+</SUP> generating FADH<SUB>2</SUB>, which enter the electron transport chain. The electron transport facilitates proton translocation over the inner bacterial membrane. The proton motoric force is used by ATP synthase. Generated ATP and reductive power in the bacterial cell leads to an increasing metabolic activity. 
   </P></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6 id="deltaResults" style="font-size:30px"><I>E. coli</I> KRX ΔdcuB::oprF strain</H6><P>

We investigated the effect of the C4 carboxylate transporter DcuB knockout on <I>E. coli</I> KRX. Furthermore we showed the integration of the outer membrane porine OprF (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_K1172507">BBa_K1172507</A>) into the bacterial genome by replacing the gene of E. coli C4 carboxylate antiporter DcuB. So we did knockout and insertion in a one-step process. 
The successful knockout of the DcuB antiporter and simultaneous insertion of <A href="http://parts.igem.org/wiki/index.php/Part:BBa_K1172507">BBa_K1172507</A> was shown with PCR analysis, DNA sequencing and phenotypic investigation via <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#BiologSystem">Biolog analysis</A> and <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic cultivation</A> in M9 minimal media with fumarate supplemented. Substrates and products were analyzed by HPLC. 
The electrobiochemical behavior of <I>E. coli</I> KRX with knocked out C4 carboxylate antiporter DcuB was tested in a <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/Construction#H-cellResults">H-cell reactor</A>. 
   </P></DIV></DIV><DIV class="element"><DIV id="text"><H6>Preparation of knockout deletion cassette</H6><P>
We decide to knock out the C4 carboxylate transporter DcuB to prevent succinate export. Besides we inserted the outer membrane porines OprF (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_K1172507">BBa_K1172507</A>) into the genome. Both could be realized with the <A href="http://www.genebridges.com/storage/Manuals_PDF/K006%20Ecoli%20Gene%20Deletion%20Kit-version2.3-2012.pdf" target="_blank">Genebridge RedET-System</A>. 
For simultaneous knockout and insertion a deletion cassette had to be designed and established. We used overlap extension PCR to amplify the complete deletion cassette (<A href="#Bryksin&amp;Matsumura2010">Bryksin &amp; Matsumura, 2010</A>). We used kanamycin as an antibiotic selective marker amplified from the plasmid Flp705 using the <A href="http://www.genebridges.com/storage/Manuals_PDF/K006%20Ecoli%20Gene%20Deletion%20Kit-version2.3-2012.pdf" target="_blank">Genebridge RedET-System protocoll</A>. 
We designed the primers (<A href="http://parts.igem.org/Part:BBa_K1465107">BBa_K1465107</A>, <A href="http://parts.igem.org/Part:BBa_K1465108">BBa_K1465108</A>, <A href="http://parts.igem.org/Part:BBa_K1465109">BBa_K1465109</A>, <A href="http://parts.igem.org/Part:BBa_K1465110">BBa_K1465110</A>) with complementary 5´extensions for both fragments to connect them in overlap extension PCR. The amplified deletion cassette has homologous sites for recombination with the <I>dcuB</I> gene in the <I>E. coli</I> KRX genome. 


  </P></DIV></DIV><DIV class="element"><DIV id="text"><H6>Verification of knockout and deletion cassette</H6><P><TABLE align="left" cellspacing="10&quot;" style="background-color:transparent; float:left"><TBODY><TR><TD><DIV class="element" style="margin:10px; padding:10px; text-align:center; float:left"><FONT size="2" style="text-align:left;"><B>Figure 1</B>: 
Results of the colony PCR for
analysis of <I>dcuB</I> genome area 
analyzed via agarose gelelectrophoresis. <I>E. coli</I> KRX wildtype band  
is shown on the rigth of about 2391 bp. <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E.coli</I> KRX ΔdcuB::oprF</A> band is shown  
on the left at 4046 bp.</FONT></DIV></TD></TR></TBODY></TABLE>


The gene of the C4 carboxylate transporter DcuB, which exchange fumarate against succinate, has a size of 1341 bp. We amplified the genome area of the <I>dcuB</I> gene with primers (<A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Primer#dcuB_del_kon1">dcuB_del_kon1</A> and <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Primer#dcuB_del_kon2">dcuB_del_kon2</A>), which bind 530 bp upstream and 520 bp downstream of the <I>dcuB</I> gene. When using the <I>E. coli</I> KRX wildtype as a template the resulting PCR product has a size of 2391 bp, which could be demonstrated by agarose gelelectrophoresis. 
However the <I>E. coli</I> KRX knockout strain (<A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E.coli</I> KRX ΔdcuB::oprF</A>) showed a 4046 bp PCR product analyzed by agarose gelelectrophoresis. Figure 1 shows the result of the agarose gelelectrophoresis with <I>E. coli</I> KRX wildtype and the modified strain <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E.coli</I> KRX ΔdcuB::oprF</A>.
The PCR product of <I>E.coli</I> KRX ΔdcuB::oprF genome amplified with primer <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Primer#dcuB_del_kon1">dcuB_del_kon1</A> and <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Primer#dcuB_del_kon2">dcuB_del_kon2</A> (4046 bp) and is composed of the antibiotic cassette (1637 bp), <A href="http://parts.igem.org/wiki/index.php/Part:BBa_K1172507">BBa_K1172507</A> (1359 bp) and upstream and downstream spacer elements (520 bp and 530 bp). 

DNA sequencing of deletion cassette from <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E.coli</I> KRX ΔdcuB::oprF</A> showed the expected results. 
</P></DIV></DIV><DIV class="element"><DIV id="text"><H6 id="NPNResult">NPN-Assay</H6><P>
The funtionality of the outer membrane porin OprF (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_K1172507">BBa_K1172507</A>) in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E.coli</I> KRX ΔdcuB::oprF</A> was investigated with a <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#NPNAssay">NPN-Uptake-Assay</A> (<A href="#Cheng2005">Cheng <I>et al.</I>, 2005</A>). 

1-N-Phenylnaphthylamine (NPN) changes fluorescence activity between aqueous and hydrophobic milieus. There is only minor fluorescence in aqueous solution, but the transport in the hydrophobic periplasmatic space causes an increased fluorescence. So NPN fluorescence is a good indicator for membrane permeability. Expression of the outer membrane porines OprF effect an increasing membrane permeability.(<A href="#Loh1984">Loh <I>et al.</I>, 1984</A>)

</P><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:600px"><FONT size="2" style="text-align:left;"><B>Figure 2</B>: Results of the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#NPNAssay">NPN-Uptake-Assay</A>. Comparison between <I>Escherichia coli</I> KRX wildtype and <I>Escherichia coli</I> KRX with <A href="http://parts.igem.org/wiki/index.php/Part:BBa_K1172507">BBa_K1172507</A>, <A href="http://parts.igem.org/wiki/index.php/Part:BBa_K1172502">BBa_K1172502</A> and <I>Escherichia coli</I> KRX with genome integrated <I>oprF</I> gene (<A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E.coli</I> KRX ΔdcuB::oprF</A>).</FONT></DIV>

Figure 2 shows the results of the NPN-Uptake-Assay. Higher fluorescence emission and higher membrane permeability could be observed with increasing promotor strength for <I>oprF</I>. The genome integrated <I>oprF</I> shows mid-level fluorescence emission and membrane permeability. Highest fluorescence levels could be measured with the <I>oprF</I> gene on a high-copy <A href="http://parts.igem.org/Part:pSB1C3">pSB1C3 plasmid</A> under control of the T7 promotor.  
Genome integrated <I>oprF</I> showed almost the same membrane permeability as the constitutive expressed <I>oprF</I> with <A href="http://parts.igem.org/wiki/index.php/Part:BBa_K1172507">BBa_K1172507</A>. There even seems to be a marginal higher expression of the genome integrated <I>oprF</I> in contrast to the plasmid coded <I>oprF</I> although the same constitutive promotor (<A href="http://parts.igem.org/Part:BBa_J23104">BBa_J23104</A>) is used. This could be explained by physiological condition of the cell. Constitutive expression of the high-copy <I>oprF</I> causes cell stress. Protein expression could be automatically downregulated by the cells. The single-copy genome integrated <I>oprF</I> showed a reduced expression level, therefor cell stress is also smaller. As a consequence <I>E. coli</I> cells showed a more effective expression of the outer membrane porin OprF adjusted on physiological cell condition. 




</DIV></DIV><DIV class="element"><DIV id="text"><H6>Phenotypic characterization with Biolog® system</H6><P><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#BiologSystem">Biolog® Microbial ID System</A> is a simple and fast method for the characterization of different bacteria, yeast and fungi species. It is based on respiration in the presence of different metabolic relevant substances for example carbon sources, nitrogen sources or toxines. The system uses redox chemistry wherein a tetrazolium dye is reduced when significant respiration occurs. Tetrazolium dye change its colour, because the cells generate reductive conditions in the course of the electron transport chain. A tetrazolium cation is reduced to formazan by dehydrogenase from the respiratory complex I. This allow a statement if respiration occurs in the presence of different substances. 


The <I>E. coli</I> KRX wildtype and <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E.coli</I> KRX ΔdcuB::oprF</A> were incubated in the Biolog® system to test respiration in the presence of fumarate. It was expected that the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E.coli</I> KRX ΔdcuB::oprF</A> shows no activity in the presence of fumarate because  of the knockout of the C4 carboxylate transporter <I>dcuB</I>, which makes fumarate uptake impossible for <I>E. coli</I> cells. 
The results of the Biolog® analysis are shown in figure 3. 
</P><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:800px"><FONT size="2" style="text-align:left;"><B>Figure 3</B>: Results of the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#BiologSystem">Biolog® analysis</A> of <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E.coli</I> KRX ΔdcuB::oprF</A> in comparison to <I>Escherichia coli</I> KRX wildtype. Respiratory activity is shown under influence of fumarate.</FONT></DIV><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#BiologSystem">Biolog® analysis</A> showed that there is no significant respiratory activity of <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E.coli</I> KRX ΔdcuB::oprF</A> in the presence of fumarate during the first 35 hours of incubation. Later on respiration occurs up to 50% of maximal respiratory activity of the <I>E. coli</I> KRX wildtype after 48 hours of incubation at 37°C. 
The results in figure 3 demonstrate that the knockout of C4 carboxylate antiporter <I>dcuB</I> was successful. Respiration activity after 35 hours could be explained by a change in metabolism of <I>E. coli</I> cells. Based on cell stress in minimal medium bacteria use other ways for uptake of fumarate. It is possible that another transporter compensates the activity of the DcuB transporter, but activation and expression of alternative pathways require a space of time, espacially under limited conditions in minimal medium. This could explain the delayed respiration of <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E.coli</I> KRX ΔdcuB::oprF</A>.  

  </DIV></DIV><DIV class="element"><DIV id="text"><H6>Anaerobic cultivation</H6><P>
We cultivated <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E. coli</I> KRX ΔdcuB::oprF</A> under <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic conditions</A>. We used <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#M9medium">M9 minimal medium</A> with 50 mM glucose. Among growth characteristic we focused on metabolite analysis. Under anaerobic conditions <I>E. coli</I> cells use different alternative electron acceptors instead of oxygen. In part the bacteria use fumarate respiration, whereby fumarate is reduced into succinate. There are also other potential pathways for bacteria to release their electrons, for example anorganic compounds like nitrate (NO<SUB>3</SUB><SUP>-</SUP>) or sulfate (SO<SUB>4</SUB><SUP>2-</SUP>).(<A href="#Gottschalk1986">Gottschalk <I>et al.</I>, 1986</A>) We analyzed succinate concentration and glucose concentration in the culture supernatant via HPLC. 

We expect succinate production of the <I>E. coli</I> KRX wildtype, but <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E. coli</I> KRX ΔdcuB::oprF</A> should not release succinate into the media because the C4 carboxylate transporter DcuB is knocked out. As a positive control we use <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#deltadcuBKEIO"><I>E. coli</I>ΔdcuB749::kan</A>, an <I>E. coli</I> strain with a reviewed knockout in <I>dcuB</I> gene. This strain also should not show any succinate transport into the medium.
</P><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:800px"><FONT size="2" style="text-align:left;"><B>Figure 4</B>: Results of the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic cultivation</A> of <I>Escherichia coli</I> KRX wildtype in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#M9medium">M9 minimal medium</A> with 50 mM glucose. Glucose and succinate concentration were measured in duplicates with HPLC.</FONT></DIV><CENTER><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:800px"><FONT size="2" style="text-align:left;"><B>Figure 5</B>: Results of the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic cultivation</A> of <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#deltadcuBKEIO"><I>Escherichia coli</I>ΔdcuB749::kan</A> in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#M9medium">M9 minimal medium</A> with 50 mM glucose. Glucose and succinate concentration were measured in duplicates with HPLC.</FONT></DIV></CENTER><CENTER><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:800px"><FONT size="2" style="text-align:left;"><B>Figure 6</B>: Results of the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic cultivation</A> of <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>Escherichia coli</I> KRX ΔdcuB::oprF</A> in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#M9medium">M9 minimal medium</A> with 50 mM glucose. Glucose and succinate concentration were measured in duplicates with HPLC.</FONT></DIV></CENTER><CENTER><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:800px"><FONT size="2" style="text-align:left;"><B>Figure 7</B>: Results of the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic cultivation</A> of <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>Escherichia coli</I> KRX ΔdcuB::oprF</A> in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#M9medium">M9 minimal medium</A> with 50 mM glucose as compared to <I>Escherichia coli</I> KRX wildtype. Glucose and succinate concentration were measured in duplicates with HPLC.  </FONT></DIV></CENTER>
Figure 4 shows the production of succinate by the <I>Escherichia coli</I> KRX wildtype under anaerobic condition as expected. Succinate is released into the medium via the C4 carboxylate transporter DcuB. Our constructed <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E. coli</I> KRX ΔdcuB::oprF</A> strain shows no succinate export under anaerobic conditions displayed in figure 6. This demonstrates a successful knockout of the <I>dcuB</I> gene in the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E. coli</I> KRX ΔdcuB::oprF</A> strain. As positive control we got an <I>E. coli</I> strain from the <A href="http://cgsc.biology.yale.edu/Strain.php?ID=109221">KEIO collection</A> with a verified knockout in the <I>dcuB</I> gene. 
Figure 5 shows the result of the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic cultivation</A> and succinate concentration measurement of this <I>E. coli</I> strain. A very low succinate concetration in the culture supernatant could be observed. In comparison to the <I>E. coli</I> KRX wildtype there is a very low succinate export. Only traces of succinate could be measured attributed to transport of succinate by other transporters with reduced specifity. 
Furthermore in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#deltadcuBKEIO"><I>Escherichia coli</I>ΔdcuB749::kan</A> the knockout is not over the complete <I>dcuB</I> gene. Insertion of kanamycin selection cassette was executed at position 749 of <I>dcuB</I> gene. So there could be a residual activity of DcuB because of residual expressed parts of the antiporter protein. However the knockout of the <I>dcuB</I> in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E. coli</I> KRX ΔdcuB::oprF</A> is complete and residual activity is not possible as shown in figure 7. Comparison between the <I>E. coli</I> wildtype and <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#KRXdeltadcuB"><I>E. coli</I> KRX ΔdcuB::oprF</A> is an obvious demonstration of a completely functional knockout of the C4 carboxylate antiporter DcuB. 


  </DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6 style="font-size:30px">Fumarate reductase (Frd) from <I>Escherichia coli</I></H6>

Upon the expression of the fumarate reductase Frd in <I>E. coli</I> we analyzed the metabolic behavior under aerobic and <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic conditions</A>. 
Successful expression of the fumarate reductase Frd (<A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>) could be proven via <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#Sodiumdodecylsulfatepolyacrylamidegelelectrophoresis%20%28SDS-PAGE%29">SDS-PAGE</A>. 
Activity of the fumarate reductase displayed with <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#HPLC">HPLC analysis</A> of fumarate consumption and succinate production in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic cultivation</A> of <I>E. coli</I> was shown with <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>. Furthermore we investigated the fumarate reductase activity in different <I>E. coli</I> strains by phenotypic MicroArray (PM) analysis with a <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#BiologSystem">Biolog® system</A>. 
  </DIV></DIV><DIV class="element"><DIV id="text"><H6 id="SDSResults">SDS-PAGE</H6><P>
The fumarate reductase could be detected in purified membrane and periplasmatic protein fractions. Proteins were fractioned by <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#ColdOsmoticShock">cold osmotic shock</A> of <I>E. coli</I> KRX at different steps after induction of protein expression. <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#Sodiumdodecylsulfatepolyacrylamidegelelectrophoresis%20%28SDS-PAGE%29">SDS-PAGE</A> shows the expression of fumarate reductase in <I>E. coli</I> KRX  under control of the T7 promotor (<A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>). Fumarate reductase with a total molecular mass of 121 kD consists of four subunits. There are two large catalytic and water-soluble subunits, flavoprotein (66 kDa) and iron-sulfur protein (27 kDa). Two small membrane associated subunits (15 and 13 kDa) are not detectable in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#Sodiumdodecylsulfatepolyacrylamidegelelectrophoresis%20%28SDS-PAGE%29">SDS-PAGE</A>. (<A href="#Iverson1999">Iverson <I>et al.</I>, 1999</A>). 
Figure 8 shows the result of the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#Sodiumdodecylsulfatepolyacrylamidegelelectrophoresis%20%28SDS-PAGE%29">SDS-PAGE</A> of isolated membrane proteins via <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#ColdOsmoticShock">cold osmotic shock</A> of <I>E. coli</I> KRX with <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>. Membrane protein fractions at different times after induction of <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A> are displayed. 

</P><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:800px"><FONT size="2" style="text-align:left;"><B>Figure 8</B>: Results of the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#Sodiumdodecylsulfatepolyacrylamidegelelectrophoresis%20%28SDS-PAGE%29">SDS-PAGE</A> of purified membrane protein from <I>Escherichia coli</I><A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A> via <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#ColdOsmoticShock">cold osmotic shock</A>. Expected size of fumarate reductase subunits A and B are 66 kDa and 27 kDA respectively. Increasing band at 66 kDa could be recognized attributable to subunit A. Increasing band between 40 and 55 kDa could be referable to dimer or trimer build-up by subunit B, C and D. </FONT></DIV><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#Sodiumdodecylsulfatepolyacrylamidegelelectrophoresis%20%28SDS-PAGE%29">SDS-PAGE</A> shows a significantly higher protein concentration in membrane extracts from <I>E. coli</I> overexpressing <I>frd</I> under control of T7 promoter (<A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>) after induction. This is caused by usually higher membrane protein concentration of cultivated cells attributable to higher optical density. Nevertheless, a strong overexpression band can be observed at the expected Frd size (subunit A) of about 66 kDa for <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>, which can be traced back on the strong expression and overproduction of Frd. Besides an overexpression band can be recognized at a size of about 45 kDA. This could be explained by dimer formation of two or three Frd subunits, for example subunit B (27 kDa) with subunit C oder D (13 and 15 kDA). That implies that subunit B build up stable dimers with small membrane associated subunits C and D of about 40 kDa respectively 42 kDa. There seems to be no dimers of subunit A with other parts of the Frd. 
 
  </DIV></DIV><DIV class="element"><DIV id="text"><H6 id="AnaerobeFrdResults">Anaerobic cultivation</H6><P>
We cultivated <I>E. coli</I> KRX with <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A> under <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic conditions</A> to characterize the activity of the fumarate reductase Frd (<A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>) under control of the T7 promotor. We used <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#M9medium">M9 minimal medium</A> with 50 mM xylose as carbon source and 50 mM fumarate as substrate of the fumarate reductase Frd. We focused on growth characteristic and metabolite analysis. Under anaerobic conditions <I>E. coli</I> cells use different alternative electron acceptors instead of oxygen. In part the bacteria use fumarate respiration, whereby fumarate is reduced into succinate. There are also other potential pathways for bacteria to release their electrons, for example anorganic compounds like nitrate (NO<SUB>3</SUB><SUP>-</SUP>) or sulfate (SO<SUB>4</SUB><SUP>2-</SUP>). As a result of this the expected level of produced succinate is very low. We analyzed fumarate and succinate concentrations in the culture supernatant via <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#HPLC">HPLC analysis</A>. 

We expect mid-level succinate production of the <I>E. coli</I> KRX wildtype under anaerobic conditions. Strong expression of Frd (<A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>) leads to an increasing succinate production and fumarate consumption. 
As a control we are going to use <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#deltafrdKEIO"><I>E. coli</I>ΔfdrA730::kan</A>, an <I>E. coli</I> strain with a reviewed knockout in <I>frdA</I> gene. This strain does not show any fumarate reductase activity and so succinate concentraion should be on a low level. Furthermore we transform <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A> into <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#deltafrdKEIO"><I>E. coli</I>ΔfdrA730::kan</A> to compensate functional defect of fumarate reductase in this strain.
Unfortunately the results of the cultivartion of <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#deltafrdKEIO"><I>E. coli</I>ΔfdrA730::kan</A> and <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/StrainsAndConstructs#deltafrdKEIO"><I>E. coli</I>ΔfdrA730::kan</A> with complementation by <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A> are not successful. HPLC analysis could not evaluated because of a mistake in the process.
</P><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:800px"><FONT size="2" style="text-align:left;"><B>Figure 9</B>: Results of the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic cultivation</A> of <I>Escherichia coli</I> KRX wildtype in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#M9medium">M9 minimal medium</A> with 50 mM xylose and 50 mM fumarate. Induction occurred after 72 h with 0.1% rhamnose and 500 μM IPTG. Fumarate and succinate concentration were measured duplicated via HPLC.</FONT></DIV><CENTER><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:800px"><FONT size="2" style="text-align:left;"><B>Figure 10</B>: Results of <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic cultivation</A> of <I>Escherichia coli</I> KRX wildtype with strong expression of <I>frd</I> (<A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>) in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#M9medium">M9 minimal medium</A> with 50 mM xylose and 50 mM fumarate. Induction occurred after 72 h with 0.1% rhamnose and 500 μM IPTG. Fumarate and succinate concentration were measured  duplicated via HPLC.</FONT></DIV></CENTER><CENTER><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:800px"><FONT size="2" style="text-align:left;"><B>Figure 11</B>: Growth of the <I>Escherichia coli</I> KRX wildtype with strong expression of <I>frd</I> (<A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>) compared to <I>Escherichia coli</I> KRX wildtype under <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic condition</A> in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#M9medium">M9 minimal medium</A> with 50 mM xylose and 50 mM fumarate. Induction occurred after 72 h with 0.1% rhamnose and 500 μM IPTG. Fumarate and succinate concentration were measured duplicated via HPLC.</FONT></DIV></CENTER><CENTER><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:800px"><FONT size="2" style="text-align:left;"><B>Figure 12</B>: Comparison of fumarate consumption and succinate production in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic cultivation</A> of the <I>Escherichia coli</I> KRX wildtype and <I>Escherichia coli</I> KRX wildtype with strong expression of <I>frd</I> (<A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>) in <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Media#M9medium">M9 minimal medium</A> with 50 mM xylose and 50 mM fumarate. Induction occurred after 72 h with 0.1% rhamnose and 500 μM IPTG. Fumarate and succinate concentration were measured duplicated via HPLC.</FONT></DIV></CENTER>

As expected figure 9 shows the production of succinate by the <I>Escherichia coli</I> KRX wildtype under anaerobic condition with 50 mM xylose and 50 mM fumarate as sole carbon sources. Under anaerobic conditions fumarate respiration takes place. Growth displayed via OD<SUB>600</SUB> shows rise in the first three days of <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#AnaerobicCultivation">anaerobic cultivation</A>. After that cells were induced via 0.1% rhamnose and 500 μM IPTG. This leads to  a reduction in growth because of the expression load and the toxic effect of IPTG. Cultivation of the <I>E. coli</I> wildtype also should be induced because of better comparability. The same effect could be observed in cultivation of <I>E. coli</I> with <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>, as shown in figure 10. There is also a drop in growth detectable. Comparison of both growth characteristics are shown in figure 11. It could be proven that there are no distinct differences in growth between<I>E. coli</I> KRX wildtype and <I>E. coli</I> KRX with <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>. So statement about fumarate reductase activity in <I>E. coli</I> KRX with <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A> as measured by succinate production and fumarate consumption is possible.
Figure 12 shows comparison of succinate production and fumarate consumption between <I>E. coli</I> KRX wildtype and <I>E. coli</I> KRX with <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>. This illustrates an increasing succinate production by <I>E. coli</I> KRX with <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A> after induction of T7 promotor. A higher value of succinate could be achieved after 8 days of anaerobic cultivation and also lower fumarate level was shown compared to <I>E. coli</I> KRX wildtype. 



  </DIV></DIV><DIV class="element"><DIV id="text"><H6 id="BiologFrdResults">Phenotypic characterization with Biolog® system</H6><P><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#BiologSystem">Biolog® Microbial ID System</A> is a simple and fast method for the characterization of different bacteria, yeast and fungi species. It is based on respiration in the presence of different metabolic relevant substances for example carbon sources, nitrogen sources or toxines. The system uses redox chemistry wherein a tetrazolium dye is reduced when significant respiration occurs. Tetrazolium dye change its colour, because the cells generate reductive conditions in the course of electron transport chain. Tetrazolium cation is reduced to formazan by dehydrogenase from respiratory complex I. This allow a statement if respiration occurs in the presence of different substances. 

We tested the influence of fumarate on the <I>Escherichia coli</I> KRX wildtype and <I>E. coli</I> KRX with <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>. To show activity of fumarate reductase under controll of the T7 promotor (<A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>), we incubated cells with fumarate in a Biolog® system for 48h. Results of the Biolog ® analysis are shown in figure 13. 
<I>E. coli</I> KRX wildtype and <I>E. coli</I> KRX with <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A> show respiratory activity in the presence of fumarate. As unexpected <I>E. coli</I> KRX wildtype shows higher activity on fumarate as compared to <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>. Respiration of <I>E. coli</I> KRX with <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A> seems to be fewer and occurs subsequently in comparison to wildtype. 
</P><DIV class="element" style="margin:10px; padding:10px; text-align:center; width:800px"><FONT size="2" style="text-align:left;"><B>Figure 13</B>: Results of <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#BiologSystem">Biolog® analysis</A> of <I>E.coli</I> KRX with <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A> in comparison to <I>Escherichia coli</I> KRX wildtype. Respiratory activity is shown under influence of fumarate.</FONT></DIV>

Figure 13 shows the respiratory activity of <I>E. coli</I> KRX with strong expressed <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A> in comparison to the <I>E. coli</I> KRX wildtype. As expected the wildtype shows higher activity in presence of fumarate as the fumarate reductase expressing strain. This effect is attributed to the fumarate reductase activity. 
Because the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#BiologSystem">Biolog® Microbial ID System</A> is based on the activity in the electron transport chain and reduction of tetrazolium dye, direct conclusions about fumarate consumption are not possible. But there are indirect hints for fumarate reductase activity via the strong expression of <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A>. Fumarate reductase naturally gets electrons out of electron transport chain and transfer them to fumarate generating succinate, which is transported out of the cells under anaerobic conditions. Under aerobic conditions succinate dehydrogenase oxidize succinate to fumarate again. This created loop by expression of <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_1465102</A> leads to lower electron transfer activity in the respiratory chain. So there are less electrons available for reduction of the extracellular tetrazolium dye compared to the <I>E. coli</I> KRX wildtype. 



  </DIV></DIV><DIV class="element"><DIV id="text"><H6>H-cell reactor</H6><P>
Fumarate reductase activity could be demonstrated by cultivation of <I>E. coli</I> KRX <A href="http://parts.igem.org/Part:BBa_K1465102">BBa_K1465102</A> in our electrobiochemical <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/Construction#text">H-cell reactor</A>. We show that expression of fumarate reductase increases the electron transfer into <I>E. coli</I> cells via neutral red as a mediator. Results of these cultivations are shown under <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/Construction#ChronoAmpMessung">construction results</A>. 

  </P></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6 style="font-size:28px">Fumarate reductase (Fum) from <I>Actinobacillus succinogenes</I></H6><P>
Besides the fumarate reductase (Frd) from <I>Escherichia coli</I> we worked on the fumarate reductase (Fum) from <I>Actinobacillus succinogenes</I> (Fum). As published by <A href="#Park1999">Park <I>et al.</I>, 1999</A>  the fumarate reductase from <I>A. succinogenes</I> shows high activity accepting electrical reduced neutral red as an electron donor and so stimulates the metabolism in a electrobiochemical reactor system. So we planed to compare these two different enzymes with regard to activity accepting neutral red or bromphenol blue as alternative electron acceptors in our <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/Construction#H-cellresults">electrobiochemical reactor system</A>. 

</P></DIV></DIV><DIV class="element"><DIV id="text"><H6>Adaptation of <I>fum</I> gene codon usage to <I>E. coli</I> KRX</H6><P>
The fumarate reductase Fum from <I>A. succinogenes</I> just as Frd from <I>E. coli</I> is a membrane associated enzyme, which consists of four subunits. Because there was no full genomic DNA available from <I>A. succinogenes</I>, we ordered the Fum gene via custom gene synthesis at <A href="http://www.idtdna.com/pages/products/genes/gblocks-gene-fragments">gBlocks® Gene Fragments</A>. Before ordering DNA we ensured for optimal codon usage adapted to <I>E. coli</I> by <A href="http://www.jcat.de/">jcat online optimization tool</A>. Furthermore we checked the sequence for illegal restriction sites (EcoRI, PstI, NotI, XbaI) and eliminated them if required.
Because the sequence is too large for synthesis in one part with <A href="http://www.idtdna.com/pages/products/genes/gblocks-gene-fragments">gBlocks® system</A>, we divided the sequence of <I>fum</I> into two different parts: <I>fumA</I> and <I>fumBCD</I> (also called <I>fumB</I>). <I>FumA</I> includes the coding sequence of the fumarate reductase (Fum) subunit A (<A href="http://parts.igem.org/Part:BBa_K1465103">BBa_K1465103</A>). However <I>fumB</I> consists of the fumarate reductase subunits B, C and D (<A href="http://parts.igem.org/Part:BBa_K1465104">BBa_K1465104</A>). The fumarate reductase subunits were designed with their own ribosome binding site respectively. 
</P></DIV></DIV><DIV class="element"><DIV id="text"><H6>Cloning fumarate Reductase Fum</H6><P>
Two different subunits of the fumarate reductase Fum from <I>A. succinogenes</I> had to be cloned into the shipping vector <A href="http://parts.igem.org/Part:pSB1C3">pSB1C3</A>. Several problems occurred during the cloning of the ordered <I>fumA</I> and <I>fumB</I> into <A href="http://parts.igem.org/Part:pSB1C3">pSB1C3</A> via <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#Gibson">gibson assembly</A>.
Cloning of <I>fum</I> was not successful for a long time. Unfortunately we could not realize the ligation of <I>fumA</I> and <I>fumB</I> creating an active fumarate reductase (Fum). So there are no characterizations of fumarate reductase (Fum) available.
 
  </P></DIV></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6 style="font-size:30px">Mediators</H6><P>
We use different mediators for electron transfer in our electrobiochemical reactor system. The electrochemical behavior of the mediators were analyzed in a <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/Construction">H-cell reactor</A>. We characterized the different <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/Construction#CV">mediators</A> in combination with different <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/Construction#ElectrodeMaterials">electrode marterials</A> to identifie the best suitable set up for cultivation of our <I>E. coli</I> strain in the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/Construction#H-cellResults">electrobiochemical reactor system</A>.

</P></DIV></DIV><DIV id="main_menue" style="margin-left:270px; height:80px"><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC" style="color:#000000"><DIV class="main_menueButton"><P class="buttoncenter"><FONT color="#FFFFFF">Overview</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer" style="color:#000000"><DIV class="main_menueButtonActual" style="width:150px"><P class="buttoncenter"><FONT color="#FFFFFF">Electron transfer</FONT></P></DIV></A><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/Construction" style="color:#000000"><DIV class="main_menueButton"><P class="buttoncenter"><FONT color="#FFFFFF">Construction</FONT></P></DIV></A></DIV><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text"><H6 style="font-size:30px">Reference</H6><P><UL><LI id="Iverson1999"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
     Iverson <I>et al.</I>, 1999. Structure of the <I>Escherichia coli</I> Fumarate Reductase Respiratory Complex. <A href="http://www.sciencemag.org/content/284/5422/1961.short" target="_blank">Science, vol. 284, pp. 1961-1966</A></DIV></DIV></LI><LI id="Cheng2005"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
     Cheng <I>et al.</I>, 2005. New antibiotic peptides, useful in treating or preventing a microbial or viral infections or in inactivating Gram-positive and -negative bacteria, protozoa, fungi and virus. <A href="http://worldwide.espacenet.com/publicationDetails/biblio?CC=DE&amp;NR=10360435A1&amp;KC=A1&amp;FT=D" target="_blank">patent  DE10360435 (A1) ― 2005-07-28</A></DIV></DIV></LI><LI id="Loh1984"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
     Loh <I>et al.</I>, 1984. Use of the fluorescent-probe 1-Nphenylnaphthylamine to study the interactions of aminoglycoside antibiotics with the outer-membrane of Pseudomonas aeruginosa. <A href="http://aac.asm.org/content/26/4/546.short|" target="_blank">Antimicrob. Agents Chemother., vol. 26, pp. 546-551</A></DIV></DIV></LI><LI id="Gottschalk1986"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
     Gottschalk <I>et al.</I>, 1986. Bacterial Metabolism. <A href="http://onlinelibrary.wiley.com/doi/10.1002/food.19870310532/abstract" target="_blank">Springer-Verlag, New York, Berlin, Heidelberg, Tokyo</A></DIV></DIV></LI><LI id="Park1999"><DIV class="element" style="margin:10px 10px 10px 10px; padding:10px 10px 10px 10px"><DIV id="text">
     Park <I>et al.</I>, 1999. Utilization of Electrically Reduced Neutral Red byActinobacillus succinogenes: Physiological Function of Neutral Red in Membrane-Driven Fumarate Reduction and Energy Conservation. <A href="http://onlinelibrary.wiley.com/doi/10.1002/food.19870310532/abstract" target="_blank">Journal of Bacteriology, vol. 181, pp. 2403-2410</A></DIV></DIV></LI></UL></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer">http://2014.igem.org/Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-CeBiTec/Results/rMFC/ElectronTransfer&amp;oldid=403420" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>