"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-CeBiTec/Results/Modelling
From 2014.igem.org
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Modeling
Parts
Data Page
Achievements
Home
Summary
Applications
Achievements
Bielefueli
Project
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Results
Team
Members
Subteams
Pictures & Videos
Bielefeld & University
Official Team Profile
Acknowledgements
Contact
Policy & Practices
Experts
Interviews
NRW-Day
Pupils Acedemy
How To Wiki
namu
SYNENERGENE
Applications
Vignettes
Notebook
Journal
Protocols
Media
Kits & Enzymes
Strains & Constructs
Primer
Acronyms
Organisms
Additional
Partners
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Partners
Modeling
Summary
Different modeling approaches were used to identify enzymatic bottlenecks in the isobutanol production pathway and to predict the formation of the desired product in a given time. First of all we created a map of all metabolic reactions which are part of our project (Figure 1). This network not only provides a good overview, it also serves as the basic tool for further considerations. Due to the huge amount of components it does not seem feasible to create a computational model for all reactions at once. Therefore we started our modeling work by carrying out a stochiometric analysis. It revealed that 42 electrons are needed for the production of one isobutanol molecule from CO2. As for the isobutanol production pathway, dynamic modeling was carried out, in the course of which several bottlenecks could be identified. An increase in expression of kivD and adhA could boost the isobutanol production after four hours by 100 %. Finally we prepared the extension of the existing model to predict the effect of specific carbon dioxide fixing reactions.
Introduction
Mathematical modeling is crucial to understand complex biological systems ( Schaber et al., 2009 ). The analysis of isolated biological components has been supplemented by a systems biology approach in over ten years ( Chuang et al., 2010 ). Furthermore mathematical modeling is used to combine biological results ( Kherlopian et al., 2008 ). Modeling is also a way to achieve results without carrying out experiments in a laboratory. The behaviour of a system can be simulated to get results which cannot be derived from simply looking at the given system ( Schaber et al., 2009 ). The most important aim of any modeling approach is the reduction of complexity. As the given biological reality is often diverse and variable, it is important to identify the major rules and principles which can describe a system.
Our aims
Visualization of the complete metabolic network
Relating electron input to product formation
Relating electron input to carbon dioxide fixation
The identification of bottlenecks in the isobutanol production pathway
Prediction of maximal isobutanol production over the time
Stoichiometric analysis
Stoichiometric analysis is useful to get information about the maximal output of a system. In this project the number of electrons moving into the system limits the amount of the product which could be synthetized in the cells. Therefore the stoichiometric relations of all substances involved in our complex reaction network were calculated (Figure 1). The calculation starts with the electrons, which are transported into the system by mediators. We calculated the resulting production of intracellular molecules based on our map of the metabolic network (Figure 1). The results are listed below.
Figure 1: Complete metabolic network of reactions involved in our project.
Table 1: This table shows the theoretical electron cost of different intracellular molecules. The electron cost is the number of electrons which are needed for the synthesis of this metabolite. The intermediates are substances which are needed for the production of the final product. Their costs are included in the final value for each substance.
Substance
Intermediates
Number of electrons
FADH2
-
2
NADH+H+
-
2
ATP
-
1
Triosephosphate
9 ATP + 6 NADH
21
CO2 fixation
-
7
Pyruvate
Triosephosphate
19
Isobutanol
2x Pyruvate
42
The theoretical electron costs of different molecules is listed in Table 1. All calculations are based on our pathway map. According to our pathway map there are 42 electrons needed for the production of one molecule isobutanol, if CO2 is used as sole carbon source. Our calculation does not involve the house keeping metabolism of E. coli, which consumes lots of energy for the survival of the cell. The true number of consumed electrons per produced isobutanol molecule is therefore much higher. The applied electric power can be converted into a number of electrons by the following equation: 1 A = 1 C * s-1 = 6.2415065 * 1018 electrons.
Dynamic modeling
After the stoichiometric analysis of the system we designed a dynamic model containing kinetic equations for the metabolite concentrations. It allows the identification of metabolic or enzymatic bottlenecks, the major aim of our modeling approach. This information might then be used in the next step to modify constructs e.g. exchange an RBS or a promotor sequence. This could help to balance the different enzyme concentrations in upcoming experiments in the laboratory, it was our goal to predict the production of isobutanol per substrate in a given time. Our model predicts the isobutanol production in a carbon dioxide fixing cell. To achieve our aims we reduced the complex system shown in Figure 1 to the version shown in Figure 2. This metabolic network was suitable for our dynamic modeling approach.
Figure 2: Reduced metabolic network of reactions which were selected for modeling.
The modeling work on the isobutanol pathway is based on publications about the isobutanol production pathway (Atsumi et al., 2008 and Atsumi et al., 2010) . We started our work on the isobutanol production pathway by collecting the appropriate kinetic parameters. They were used for the development of a  system of differential equations. Concerning the choice of the kinetics used, we stuck to Michaelis-Menten kinetics as this was published as the best approach if reaction kinetics are not known ( Breitling et al., 2008 ; Chubukov et al., 2014 ). Additionally kcat and KM values were collected from databases like KEGG , biocyc and BRENDA (Table 2). Missing values had to be estimated. The starting concentrations for different metabolites were also taken from the literature and from different databases (Table 3).
We decided to use only the forward reactions from pyruvate to isobutanol for different reasons. First of all, it is necessary to get the maximal product concentration, secondly the reactions are nearly irreversible due to the decarboxylation steps and thirdly there is only few information on the back reactions available.
Table 2: This table shows all enzymatic parameters which were used for our dynamic model.
Enzyme
kcat [-]
KM [mM]
Reference
AlsS
121
13.6
Atsumi et al., 2009
IlvC
2.2
0.25
Tyagi et al., 2005
IlvD
10 (estimated)
1.5
Flint et al., 1993
KivD
20 (estimated)
5 (estimated)
Werther et al., 2008 ; Gorcke et al., 2007
AdhA
6.6
9.1
Atsumi et al., 2010
Table 3: This table shows all metabolite concentrations which were used for our first model. The metabolite concentration was set to zero, if no published value was available.
Metabolite
Concentration [mM]
Reference
Pyruvate
10
Yang et al., 2000
2-Acetolactate
0
-
2,3-Dihydroxyisovalerate
0
-
2-Ketoisovalerate
0
-
Isobutyraldehyde
0.6 (estimated)
Isobutanol
variable
-
We implemented the system of differential equations in Matlab ( source code ). Two examples are shwon in Figure 3. The predicted changes of the metabolic concentration over the time is shown in Figures 4-7. The amount of expressed proteins could differ depending on the distance of the coding sequence downstream of the promoter. The coding sequences for the involved enzymes are located downstream of a common promoter. Therefore we decided to set the enzyme concentration for the first enzyme to 1 and decrease in steps of 0.1 ( source code ). These different values were tried to identify appropriate concentrations for each enzyme. The results of this dynamic modeling approach could be transferred to the laboratory by using promoters of different strength.
Figure 3: Examples of differential equations for the dynamic modeling of the isobutanol production.
Figure 4: Predicted changes in metabolic concentration over time.
Figure 5: Predicted changes in metabolic concentration over time. The development in the first minutes is shown by zooming into the Figure 4.
Figure 6: Predicted changes in metabolic concentration in an improved system over time. The concentration of KivD and AdhA was increased by factors 3 and 4, respectively.
Figure 7: Predicted changes in metabolic concentration in an improved system over time. The concentration of KivD and AdhA was increased by factor 3 and 4 respectively. The development in the first minutes is shown by zooming into the Figure 6.
Our modeling results indicated that the concentrations of two enzymes are limiting the isobutanol production KivD and AdhA . An experimental validation of this effect is the next logical step. This enzymatic bottleneck could be removed by overexpression of the corresponding coding sequences ( kivD and adhA ). One way to achieve this is the integration of a strong promotor upstream of their coding sequences. It could be a possibility to improve our isobutanol production. As shown in Figure 4 the isobutanol concentration reaches 2 mM after four hours. In the improved system the concentration of isobutanol after four hours is nearly doubled.
Conclusion
The complete metabolic network of our project is visualized in Figure 1. Stoichiometric calculations revealed the number of electrons which is in theory need for the production of a desired substance. There are 42 electrons required for the synthesis of one molecule isobutanol. The fixation of one carbon dioxide molecule would cost seven electrons. We were able to identify a putative enzymatic bottleneck in the isobutanol production pathway. An increased amount of KivD and AdhA could improve the isobutanol production. Additionally the production of isobutanol from pyruvate can be predicted. The next and very important step would be the validation of these predictions by experimental approaches.
Outlook: addition of carbon dioxide fixing reactions
The next step would be the addition of specific carbon fixing reactions and the pathway leading to pyruvate in the existing dynamic model. Therefore we collected kcat and KM values for nearly all relevant steps (Table 4). They can be used to formulate additional differential equations which describe these additional reactions.
Table 4: This table shows all kcat and KM values of enzymes involved in CO2 fixation and the pathway leading to pyruvate.
Enzyme
kcat [-]
KM [mM]
Reference
PrkA
72.6
0.09
Wadano et al., 1998 , Kobayashi et al., 2003
RubisCO
20 (estimated)
0.02 (estimated)
Lan and Mott, 1991 , Sage, 2002
Pgk
480
1 (estimated)
Fifis and Scopes., 1978
GapA
-
0.5
Zhao et al., 1995
GpmA
490 (in S.cerevisiae)
0.15
Fraser et al., 1999 , White and Fothergil-Gilmore, 1992
Eno
17600
0.1
Spring and Wold, 1972 , Albe et al., 1990
PykF
3.2
0.3 (estimated)
Oria-Hernandez et al., 2005
Using these information the combination of two parts of our project would be possible. The carbon dioxide fixation pathway could be checked for enzymatic bottlenecks. It would be possible to predict the isobutanol production from fixed carbon dioxide.
References
Atsumi, Shota, Taizo Hanai, und James C. Liao. „Non-Fermentative Pathways for Synthesis of Branched-Chain Higher Alcohols as Biofuels“. Nature 451, no. 7174 (2008): 86–89.
Atsumi, Shota, Tung-Yun Wu, Eva-Maria Eckl, Sarah D. Hawkins, Thomas Buelter, und James C. Liao. „Engineering the isobutanol biosynthetic pathway in Escherichia coli by comparison of three aldehyde reductase/alcohol dehydrogenase genes“. Applied Microbiology and Biotechnology 85, no. 3 (2010): 651–57.
Yang, Y. T., G. N. Bennett, und K. Y. San. „The Effects of Feed and Intracellular Pyruvate Levels on the Redistribution of Metabolic Fluxes in Escherichia Coli“. Metabolic Engineering 3, no. 2 (2001): 115–23.
Wadano, Akira, Keisuke Nishikawa, Tomohiro Hirahashi, Ryohei Satoh, und Toshio Iwaki. „Reaction Mechanism of Phosphoribulokinase from a Cyanobacterium, Synechococcus PCC7942“. Photosynthesis Research 56, no. 1 (1998): 27–33.
Kobayashi, Daisuke, Masahiro Tamoi, Toshio Iwaki, Shigeru Shigeoka, und Akira Wadano. „Molecular Characterization and Redox Regulation of Phosphoribulokinase from the Cyanobacterium Synechococcus Sp. PCC 7942“. Plant & Cell Physiology 44, no. 3 (2003): 269–76.
Lan, Yun, und Keith A. Mott. „Determination of Apparent Km Values for Ribulose 1,5-Bisphosphate Carboxylase/Oxygenase (Rubisco) Activase Using the Spectrophotometric Assay of Rubisco Activity“. Plant Physiology 95, Nr. 2 (1991): 604–9.
Sage, Rowan F. „Variation in the K(cat) of Rubisco in C(3) and C(4) Plants and Some Implications for Photosynthetic Performance at High and Low Temperature“. Journal of Experimental Botany 53, no. 369 (2002): 609–20.
Fifis, T., und R. K. Scopes. „Purification of 3-Phosphoglycerate Kinase from Diverse Sources by Affinity Elution Chromatography“. The Biochemical Journal 175, no. 1 (1978): 311–19.
Zhao G, Halbur T, Pankratz DC. Colonization of oropharynx and nasal cavity of CDCD pigs by a nontoxigenic strain of Pasteurella multocida type D. J Swine Health Prod 1995;3(3):113-115.
Fraser, H. I., M. Kvaratskhelia, und M. F. White. „The Two Analogous Phosphoglycerate Mutases of Escherichia Coli“. FEBS Letters 455, no. 3 (1999): 344–48.
White, Malcolm F., und Linda A. Fothergill-Gilmore. „Development of a Mutagenesis, Expression and Purification System for Yeast Phosphoglycerate Mutase“. European Journal of Biochemistry 207, no. 2 (1992): 709–14.
Spring, Thomas G., und Finn Wold. „The Purification and Characterization of Escherichia Coli Enolase“. Journal of Biological Chemistry 246, no. 22 (1971): 6797–6802.
Albe, Kathy R., Margaret H. Butler, und Barbara E. Wright. „Cellular concentrations of enzymes and their substrates“. Journal of Theoretical Biology 143, no. 2 (1990): 163–95.
Oria-Hernández, Jesús, Nallely Cabrera, Ruy Pérez-Montfort, und Leticia Ramírez-Silva. „Pyruvate Kinase Revisited: The Activating Effect of K+“. The Journal of Biological Chemistry 280, no. 45 (2005): 37924–29.
Breitling, Rainer, David Gilbert, Monika Heiner, und Richard Orton. „A Structured Approach for the Engineering of Biochemical Network Models, Illustrated for Signalling Pathways“. Briefings in Bioinformatics 9, no. 5 (2008): 404–21.
Chubukov, Victor, Luca Gerosa, Karl Kochanowski, und Uwe Sauer. „Coordination of Microbial Metabolism“. Nature Reviews Microbiology 12, no. 5 (Mai 2014): 327–40.
Schaber, J., W. Liebermeister, und E. Klipp. „Nested Uncertainties in Biochemical Models“. IET Systems Biology 3, no. 1 (2009): 1–9.
Chuang, Han-Yu, Matan Hofree, und Trey Ideker. „A Decade of Systems Biology“. Annual Review of Cell and Developmental Biology 26 (2010): 721–44.
Kherlopian, Armen R., Ting Song, Qi Duan, Mathew A. Neimark, Ming J. Po, John K. Gohagan, und Andrew F. Laine. „A Review of Imaging Techniques for Systems Biology“. BMC Systems Biology 2, no. 1 (12. August 2008): 74.
Tyagi, R., Lee, Y.-T., Guddat, L.W., and Duggleby, R.G. (2005). Probing the mechanism of the bifunctional enzyme ketol-acid reductoisomerase by site-directed mutagenesis of the active site. FEBS J. 272, 593–602.
Atsumi, S., Li, Z., and Liao, J.C. (2009). Acetolactate Synthase from Bacillus subtilis Serves as a 2-Ketoisovalerate Decarboxylase for Isobutanol Biosynthesis in Escherichia coli. Appl Environ Microbiol 75, 6306–6311.
Flint, D.H., Emptage, M.H., Finnegan, M.G., Fu, W., and Johnson, M.K. (1993). The role and properties of the iron-sulfur cluster in Escherichia coli dihydroxy-acid dehydratase. J. Biol. Chem. 268, 14732–14742.
Gocke, D., Nguyen, C.L., Pohl, M., Stillger, T., Walter, L., and Müller, M. (2007). Branched-Chain Keto Acid Decarboxylase from Lactococcus lactis (KdcA), a Valuable Thiamine Diphosphate-Dependent Enzyme for Asymmetric CC Bond Formation. Adv. Synth. Catal. 349, 1425–1435.
Werther, T., Spinka, M., Tittmann, K., Schütz, A., Golbik, R., Mrestani-Klaus, C., Hübner, G., and König, S. (2008). Amino Acids Allosterically Regulate the Thiamine Diphosphate-dependent α-Keto Acid Decarboxylase from Mycobacterium tuberculosis. J. Biol. Chem. 283, 5344–5354.
Retrieved from " http://2014.igem.org/Team:Bielefeld-CeBiTec/Results/Modelling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
