"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-CeBiTec/Project/Biosafety
From 2014.igem.org
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Home
Summary
Applications
Achievements
Bielefueli
Project
Results
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Modeling
Parts
Data Page
Achievements
Team
Members
Subteams
Pictures & Videos
Bielefeld & University
Official Team Profile
Acknowledgements
Contact
Policy & Practices
Experts
Interviews
NRW-Day
Pupils Acedemy
How To Wiki
namu
SYNENERGENE
Applications
Vignettes
Notebook
Journal
Protocols
Media
Kits & Enzymes
Strains & Constructs
Primer
Acronyms
Organisms
Additional
Partners
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Partners
Biosafety - Antibiotic-free Selection
An import aspect of biosafety discussion in the field of synthetic biology is the prevention of interactions between genetically modified bacteria and natural wild type strains. This prevents the uncontrolled distribution of sequences coding for genetically modified genes in the environment. Even if these sequences would be released from a laboratory it would be very unlikely that they are integrated in a bacterial genome. Most organisms protect themselves against foreign nucleic acids for example by expressing special RNases and DNases (Bertani and Weigle, 1953; Tock and Dryden, 2005). Further the genetically modified bacterial strains are adapted to the excellent conditions of the laboratory. It is highly unlikely that the natural bacteria will outlast these modified strains in nature due to the better adaptation of the wildtype strains to their environment conditions. But there is no guarantee that there is really no interaction and that the release does not affect the equilibrium of the environment.  So the question how a novel designed sequence or even a heterologous used sequence would affect the environment, can be divided into three cases. In the first scenario it is an artificial altered sequence of DNA or even the changed amino acid sequence of a protein. By introducting naturally not used building blocks in these sequences, like xenonucleotids (XNAs) one could reduce the risk of interaction with the environment.
The second case deals with sequences that are useless for the survival of an organisms in its natural enviroment. An example  would be the RubisCO we introduced to E. coli. This enzyme is only useful to its host within high concentrations of carbon dioxide and the supply with ribulose-1,5-bisphosphate, which cannot be synthesized in most of the prokaryotes or Fumerate reductase, which also might be only useful under special conditions of an increasing electron supply and anaerobic conditions. So assumed that an intact sequence of one of these genes would integrate into the genome of a natural cell or would be plasmid based taken up, it might be lost even faster because of its higher burden to the cell.
The third case is the type of genes that probably have an greater impact to the environment, like antibiotics and other bacterial or artificial toxins that entail an advantage in selection for the bacteria. Because of this, the uptake of a corresponding sequence will have an impact of the environment. Toxin encoding sequences are for example used in kill switches, which should prevent genetically modified bacteria from surviving in environment apart from laboratory conditions. If these systems fail there is the possibility of the distribution of such sequences in the environment. Also the probability is very low the consequences would be severely. An bacterium, which is able to produce a toxin, poses a great risk to its enviroment. The consequences could be even more striking when the organism is resistant to antibiotics. It is well known that antibiotic resistances are spreading out due to various reasons (Heuer et al., 2011; Lupo et al., 2012). This increases the risk of the formation of multi resistances, which causes problems in medical facilities. Especially in these institutes where a lot of weak, sick and immune suppressed peaple are hospitilized the threat of antibiotic resistant bacteria arises.
Antibiotic resistances occur in nature in many species. The distribution of such resistances is a novel research field. Soil bacteria seem to be a reservoir of antibiotic resistance genes (Riesenfeld et al., 2004; Forsberg et al., 2012). All antibiotic resistances which are used in the laboratory were isolated from naturally occurring organisms. Nevertheless it is crucial to keep these genes from spreading out.
To address this problem we implemented an antibiotic-free selection system by the complementation of a D-alanine auxotrophic E. coli strain. We have done this to reduce the need for antibiotic resistances as selection markers and enable an antibiotic-free plasmid selection.
Further this sysem might be not only limited to E. coli since a D-alanine auxotrophy could be demonstrated also for other bacteria like Listeria monocytes ( Thompson et al., 1998 ), Corynebacterium  glutamicum ( Tauch et al., 2002 ) or Bacillus subtilis ( Ferrari et al., 1985 ) as example. It is likely that the system is feasible with any bacteria where the cross-linkage of the peptidoglycane layer is realized with D-alanine.
References
E. Ferrari, D. Henner und M. Yang (1985) Isolation of an alanine racemase gene from Bacillus subtilis and its use for plasmid maintenance in B.subtilis. Nature Biotechnology , vol. 3, pp. 1003 - 1007.
A. Tauch, S. Götker, A. Pühler, J. Kalinowski, G. Thierbach (2002) The alanine racemase gene alr is an alternative to antibiotic resistance genes in cloning systems for industrial Corynebacterium glutamicum strains. Journal of Biotechnology , vol. 99, pp. 79 - 91.
R. Thompson, H. Bouwer, D. Portnoy, F. Frankel (1998) Pathogenicity and and immunogenicity of a Listeria monocytogenes strain that requires D-alanine for growth. Infection and Immunity , vol. 66, pp. 3552 - 3561.
Lupo, A., Coyne, S. & Berendonk, T. U. Origin and Evolution of Antibiotic Resistance: The Common Mechanisms of Emergence and Spread in Water Bodies. Front Microbiol 3, (2012).
Heuer, H., Schmitt, H. & Smalla, K. Antibiotic resistance gene spread due to manure application on agricultural fields. Current Opinion in Microbiology 14, 236–243 (2011).
Forsberg, K. J. et al. The Shared Antibiotic Resistome of Soil Bacteria and Human Pathogens. Science 337, 1107–1111 (2012). 1. Riesenfeld, C. S., Goodman, R. M. & Handelsman, J. Uncultured soil bacteria are a reservoir of new antibiotic resistance genes. Environmental Microbiology 6, 981–989 (2004).
Bertani, G. & Weigle, J. J. HOST CONTROLLED VARIATION IN BACTERIAL VIRUSES. J Bacteriol 65, 113–121 (1953).
Tock, M. R. & Dryden, D. T. The biology of restriction and anti-restriction. Current Opinion in Microbiology 8, 466–472 (2005).
Retrieved from " http://2014.igem.org/Team:Bielefeld-CeBiTec/Project/Biosafety "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
