"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle
From 2014.igem.org
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Modeling
Parts
Data Page
Achievements
Home
Summary
Applications
Achievements
Bielefueli
Project
Overview
rMFC
CO2 Fixation
Isobutanol
Biosafety
Results
Team
Members
Subteams
Pictures & Videos
Bielefeld & University
Official Team Profile
Acknowledgements
Contact
Policy & Practices
Experts
Interviews
NRW-Day
Pupils Acedemy
How To Wiki
namu
SYNENERGENE
Applications
Vignettes
Notebook
Journal
Protocols
Media
Kits & Enzymes
Strains & Constructs
Primer
Acronyms
Organisms
Additional
Partners
Gold Sponsors
Silver Sponsors
Bronze Sponsors
Partners
Module II - Carbon Dioxide (CO2) Fixation
Overview
RuBisCO
Carboxysome
Calvin-Cycle
Measurement
Outlook
Introduction
As already mentioned in the project section we decided to implement the Calvin-cycle in our project. There are different reasons for this. One the one side we searched for a method to cultivate under aerobic conditions and on the other side we searched for a cyclic system which is not given in every carbon dioxide fixation mode.
In E. coli are three enzymes missing to enable the whole cycle. Our goal is to transform the missing genes from different organisms. Our main research is based on a publication by Bonacci et al., 2011 . The idea was to use the carboxysom as a microcompartiment for E. coli from Halothiobacillus neapolitanus for an efficient RuBisCO activity. The problem of handling the RuBisCO is its affinity to oxygen, which can be used instead of CO2. If the RuBisCO uses oxygen as a co-substrate, no CO2 can be fixed. A carboxysom establishs a higher concentration of carbon dioxide in the microcompartiment. The substrate for the RuBisCO is provided by an enzyme called phosphoribulokinase which was derived from Synechococcus elongatus . The last missing enzyme was taken from Bacillus methanolicus and is called sedoheptulose 1,7-bisphosphatase ( Stolzenberger et al., 2013 ).
A problem for our system would be the glycolysis which we aimed to inhibit with a knock-down of the phosphofructokinase. We called this system the gluco-switch .
The main goal for the characterization was to establish a new enzyme purification system based on an intein-tag containing a chitin binding domain. It could be used by integrating a coding sequence into the vector.
Phosphoribulokinase
Figure 1: Toxicity of phosphoribulokinase without RuBisCO
The sequence of the phosphoribulokinase was synthesized to remove illegal restriction sites and to optimize the codon usage for E. coli . We were able to transform the prkA ( BBa_K1465201 , BBa_K1465212 , BBa_K1465214 ) into E. coli but without ribosom binding site. As a template for the synthesis we used the prkA of Synechococcus elongatus.
The toxicity of the PrkA in E. coli was described previously by Parikh et al., 2006 and Bonacci et al., 2012 . The toxicity results through the accumulation of ribulose 1,5-bisphosphate which can not be further metabolized as shown in Figure 1. We performed an enzyme assay to identify the functionality of the PrkA in E. coli . In order to investigate this we cultivated the strain and made a crude cell extract. The cell extract was incubated for 1 h at 37°C. We compared different approaches. First we incubated a wild type strain, secondly we incubated the crude cell extract and thirdly we incubated the crude cell extract with 1 mM ribulose 5-phosphate which is the substrate for the PrkA. The aim was to identify ribulose 1,5-bisphosphate with the HPLC. Because the wild type is not able to produce ribulose 1,5-bisphosphate the PrkA activity should be easily observed. We were not able to identify the product, ribulose 1,5-bisphosphate, with HPLC in all approaches. To perform a SDS-Page we cultivated E. coli carrying prkA strain and induced gene expression with 1 mM IPTG ( BBa_K1465212 , BBa_K1465214 ). In comparison to the wild type the prkA carrying strain showed similar growth behavior. The resulting SDS-Page is shown below.
Figure 2: SDS-Page of ptac prkA
The PrkA has a molecular size of around 38 kD. There is a clear band shown between 35 kD and 40 kD. This fragment was cut out to identify the protein via MALDI-TOF analysis. With the MALDI-TOF we were able to identify three peptides of the PrkA which is sufficient.
The results of our overall approaches looked like the PrkA is expressed by E. coli but does not show functionality in an in vitro assay. A possible problem is the light dependent activation of the PrkA. The light triggers the PSI in photosynthetic active organisms. This trigger reduces ferredoxine. In the follwing reaction thioredoxin is reduced while ferredoxin is oxidized again. The reduced thioredoxin is able to break disulfides. The PrkA activation is triggered by thioredoxin which is maybe not present sufficiently in E. coli. As previously described it would be possible to activate the PrkA by adding DTT in the enzyme assay ( Hariharan et al., 1998 ) which would be our next target for an enzyme assay.
Sedoheptulose 1,7-bisphosphatase
We succesfully transformed the SBPase from Bacillus methanolicus ( BBa_K1465228 , BBa_K1465229 ). For the characterization of the sedoheptulose 1,7-bisphosphatase (SBPase / glpX) we did an enzyme assay with a His-Tag purification as described before ( Stolzenberger et al., 2013 ).
The proteins were overexpressed by adding 1 mM IPTG for inducing the T7 promotor( BBa_K1465229 ). The increasing amount of protein could be verified with a SDS-PAGE. .
Figure 3: Proteinexpression of Fba
Figure 4: Proteinexpression of Tkt
Figure 5: Proteinexpression of GlpX
All three SDS-Pages showed a band with a clear increase in the amount of protein like described in Stolzenberger et al., 2013 . We purified the transketolase (Tkt) and the fructose bisphosphate aldolase (Fba) as well as the sedoheptulose 1,7-bisphosphatase with the His-Tag mediated purification system .
Figure 6: Protein purification of Fba
Figure 7: Protein purification of Tkt
Figure 8: Protein purification of GlpX
For the purified enzymes we performed a Bradford assay.
Figure 9: Bradford assay with purified enzymes (two technical and two biological replicates)
The Bradford assay showed high concentrations of Tkt and Fba as well as a very low concentration of GlpX. After the purification we performed an enzyme assay as shown below.
Figure 10: Schematical view of the SBPase assay
Molecule
Formula
Molecular weight [M-H]
Fructose-6-P
C6H15O9P
259.021
Glyceraldehyd-3-P
C3H7O6P
168.99
Dihydroxyacetone-P
C3H7O6P
168.99
Xylulose-5-P
C5H11O8P
229.01
Erythrose-4-P
C4H9O7P
199.00
Sedoheptulose-7-P
C7H15O10P
289.03
Sedoheptulose-1,7-BP
C7H16O15P2
368.99
The product of the reaction, sedoheptulose 7-phosphate, could be identified via HPLC. We made different approaches for the enzyme assay to characterize all reactions.
Figure 11: Approaches by adding one enzyme at each step. Each step shows the activity of the enzymes.
Reaction mix:
20 mM Fructose 6-phosphate
20 mM Glyceraldehyde 3-phosphate
20 mM Dihydroxyacetonephosphate
10 µM Thiamine pyrophosphate
2 mM Manganese chloride
50 mM Tris-HCl
For the first approach we added no enzyme to verify that no product is generated as shown in Figure 11. The second approach includes the transketolase which catalyzes the reaction of F6P and GAP to erythrose 4-phosphate. For the third approach fructose bisphosphate aldolase was added to convert erythrose 4-phosphate with dihydroacetonephosphate to sedoheptulose 1,7-bisphosphate. For the last approach the sedoheptulose 1,7-bisphosphatase (GlpX) was added which results in sedoheptulose 7-phosphate. All intermediates could be verified in all approaches as expected. This measurement showed the activity of the SBPase in vitro (Xylulose 5-phosphate is a byproduct of one of the enzymatical reactions).
Figure 12: Comparison of 37°C and 50°C of the in vitro assay (all enzymes).
We did a comparison between 37°C and 50°C of the in vitro assay to investigate the different enzymatic activities as shown in Figure 12. The transketolase and aldolase showed a higher activity at 37°C which resulted in more respectively products. The sedoheptulose 1,7-bisphosphatase activity is higher at 50°C but also shows activity at 37°C. We could identify this characteristic because of the accumulation of sedoheptulose 1,7-bisphosphate in the 37°C approach. The second approach has a lower concentration of this sedoheptulose 1,7-bisphosphate but showed a higher concentration of sedoheptulose 7-phosphate which is due to the higher activity of the SBPase at 50°C. This results suggest that it is possible to enable SBPase activity with GlpX in E. coli .
We decided to use glpX as a target for amplification and transformation because of the shown acitivity. The finished construct of ptac_glpX in pSB1C3 was cultivated in M9 glucose in comparison to the wild type. We performed two biological replicates and two technical replicates.
Figure 13: Cultivation experiment in M9 glucose (two technical and two biological replicates)
The cultivation shows that the modified strain has an extended lag phase. Two hours after IPTG induction of the gene expression, a decrease of growth was observed in comparison to the uninduced strain. There are two possible explanations for this behavior. On the one side IPTG acts as a toxic substance for bacteria which may result in growth decrease and on the other side the production of the protein can result in decreased growth. We exclude IPTG as a reason because earlier cultivations showed that 1 mM IPTG has no effects on the wild type growth. Furthermore the wild type strain shows an one hour reduced lag-phase in comparison to the induced strain. The induced strain also shows a higher OD. By inducing the SBPase in E. coli the substances for glycolysis are deflected towards other pathways. These reactions are reversible which means that the glucose of the M9 medium is not metabolized in another pathway. This time shifted use of glucose results in a higher OD in two biological replicates.
This result shows that the SBPase does not limit the growth maximum of E. coli . The glucose concentration confirms these results. The wild type consumes the glucose faster than the mutant strains.
Figure 14: Cultivation experiment in M9 xylose
The cultivation was repeated with xylose. Xylose could be suitable additional carbon source if the Calvin cycle has a lower efficiency. This second cultivation shows the impact of the induction more clearly. The induced mutant strain shows a lower growth rate in comparison to the uninduced mutant and the wild type. This indicates that the lack of glucose in the medium has a large impact because necessary intermediates for the glycolysis were used by the SBPase.
Gluco-Switch
The idea for a gluco-switch was the knock-down of the pfk via sRNA ( BBa_K1465225 , BBa_K1465226 , BBa_K1465227 ). For the knock-down of the pfkA we used the synthesized sRNA:pfkA. Herewith we wanted to down-regulate the glycolysis to prohibit the bacteria to use sugar as a carbon source for our follow-up experiments. This should cause a reduction of the growth of our bacteria carrying the plasmid. We tried to show this with the help of a cultivation (Figure 15).
Figure 15: Cultivation of sRNA_pfkA_glucose
Neither the construct with the ptac ( BBa_K1465226 ) promoter nor the one with the T7 promoter ( BBa_K1465227 ) showed a difference between induced and uninduced state. As we used M9 glucose as the medium for the cultivation there should be a difference in the growth. The HPCL results did not show a hint for the functional activity of the sRNA as well because the induced culture as well as the uninduced one showed the same consumption of glucose. For the experiments we inoculated the cultures with a pre-culture out of the exponential phase. For any reason the wild type had an unusual long lag phase that we cannot explain.
We think that it was maybe not enough to only take the sRNA of pfkA. Another try would be to integrate the sRNA:pfkB downstream of the sRNA:pfkA to be sure to cover the activity of both phosphofructokinases. Additionally the formation of the sRNA should be checked.
Protein purification
Figure 16: Scheme for intein mediated purification
For the purpose of characterizing our BioBricks we thought of using enzyme assays to verify the functionality of different proteins. Enzyme assays depend on purified enzymes. A typical purification approach is the His-tag mediated purification system. The disadvantage of this system is that the tag remains attached to the enzyme after the purification and has to be cleaved afterwards. A further development of this system is the intein tag mediated purification ( BBa_K1465230 ).
By adding an intein tag attached to a chitin binding domain to the enzyme of interest a purification of the natural untagged protein can be achieved. The chitin binding domain binds the column at which chitin beads are stored. After adding binding buffers and washing solutions an elution with DTT allows the cleavage of the attached intein tag to the coding sequence. The enzyme is eluted from the column and can be stored in the desired buffer. The chitin binding domain and intein tag can be eluted from the column afterwards to reuse the column.
We implemented this system in the pSB1C3 backbone by combining the T7 promoter with RBS and intein tag with a chitin binding domain.
Figure 17: Scheme vor purification vector
By designing Gibson Assembly primers with flanking overhangs it is possible to add a coding sequence between the first and the second part of the purification vector (add the gene specific part behind the overhang with the right orientation):
>GSP_fw
CTATAGGGAAAGAGGAGAAAT
>GSP_rev
CTAGTGCATCTCCCGTGATGCA
Note: The stop codon of the coding sequence has to be deleted through primer design.
It is perhaps possible to redesign the pSB1C3 backbone to the purification vector by including the T7 and RBS as well as the intein tag with chitin binding domain into the backbone. The restriction sites for BioBrick assembly may be placed in between both patterns. This would allow an in frame addition of the coding sequence by using BioBrick assembly.
Because of problems during the transformation of the coding sequences we were not able to characterize this BioBrick.
Summary
In the following subproject we were trying to transform the prkA and the glpX as well as enabling the gluco switch. It was possible to transform the prkA which could be verified by MALDI-TOF. With HPLC we were not able to show the functionality of the PrkA. Secondly we transformed the SBPase which could be verified by SDS-Page. The protein and additional proteins for an in vitro assay were purified via His-tag purification. The functionality could be shown via HPLC. A decrease of growth resulting from reducing the amount of available substrates for glycolysis was shown by comparative cultivation. Thirdly the gluco-switch with knock-down of the pfk was transformed. The expected decrease of growth could not be verified through cultivation.
Overview
RuBisCO
Carboxysome
Calvin-Cycle
Measurement
Outlook
References
Stolzenberger et al., 2013. Characterization of Fructose 1,6-Bisphosphatase and Sedoheptulose 1,7-Bisphosphate from the Facultative Ribulose Monophosphate Cycle Methylotroph Bacillus methanolicus. Journal of Bacteriology , Vol. 195, pp. 5112-5122
Bonacci et al., 2011. Modularity of carbon-fixing protein organelle. PNAS , vol. 109, pp. 478-483
Parikh et al., 2006. Directed evolution of RuBisCO hypermorphs through genetic selection in engineered E. coli. Protein Engineering, Design & Selection , vol. 19, pp. 113-119
Hariharan et al., 1998. Purification and characterization of phosphoribulokinase from the marine chromophytic alga Heterosigma carterae. Plant Physiol. Vol. 117, pp. 321-329
Retrieved from " http://2014.igem.org/Team:Bielefeld-CeBiTec/Results/CO2-fixation/Calvin-Cycle "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
