Team:British Columbia/Kaempferol
PROJECT
OVERVIEW DISTRIBUTED BIOSYNTHESIS BIOSENSOR POPULATION DYNAMICS PARTS
MODEL
HUMAN PRACTICES
HUMAN PRACTICES PUBLIC ENGAGEMENT COLLABORATIONS
TEAM
MEMBERS ATTRIBUTIONS
LABORATORY
NOTEBOOK PROTOCOLS INTERLAB
ACHIEVEMENTS
With significant advances in sequencing and synthesis technologies, and our understanding of gene function and genetic regulatory networks, we have the capacity to design complex biosynthetic systems that distribute metabolic load optimally between multiple co-existing strains1.  A significant challenge in the engineering of synthetic co-cultures is the regulation of population dynamics within the culture.  In natural systems, relative fitness, growth rates and interdependencies lead to an equilibrium between members of a community1-3.  In engineered co-cultures, activity of the biosynthetic pathway of interest in not related to fitness of each individual strain.  Competition between strains, as well as varied growth characteristics, may skew ratios in the population in ways that decrease the biosynthetic efficiency of the system1.
Various methods of addressing this concern have been employed.  Engineering interdependence of strain, where auxotrophs are reliant on a metabolite produced by the other strain will maintain co-culture members4.  Engineering dependence on specific carbon sources can also minimize competition between members of a synthetic community.  A previous iGEM team ( Imperial College ) used quorum sensing to engineer communication between strains in a co-culture, enabling the maintenance of a set ratio between strains.  However, in none of these cases is the ratio related to the engineered biosynthetic process.
This project aims to engineer a co-culture in which population dynamics are regulated by the production of the pathway intermediate naringenin2,3.  A biosynthetic imbalance in strain ratios will lead to either a build-up of naringenin in the culture or a naringenin deficit, with strain 1 being unable to produce as much naringenin as could be consumed by the strain 2 component of the population.  Both of these outcomes will decrease the biosynthetic potential of the co-culture.
In order to overcome this risk, growth rates and activities of strains were coupled to the biosynthesis of naringenin within the system using the FdeR biosensor system 5.  By coupling GP2 expression directly to the FdeR biosensor, the growth rate and activity of strain 1 will decrease as naringenin builds up6.  By using the pt181 transcript-dependant transcriptional and translational regulatory mechanism, GP2 expression can be inversely coupled to the presence of naringenin, increasing the growth rate and activity of strain 27.  This results in a system where an increase in naringenin concentration leads to a decrease in the growth rate and activity of the producing strain and an increase in the growth rate and activity of the consuming strain.  An absence of the intermediate naringenin would lead to an increase in naringenin production and a decrease in naringenin consumption activity.  Co-cultures will then reach a steady state, where there is neither excess naringenin production nor excess energy and resource consumption by the strain catalyzing naringenin conversion into kaempferol, when there is not sufficient naringenin present in the system.
References
Zhang, H. and X. Wang, Modular co-culture engineering, a new approach for metabolic engineering. Metabolic Engineering, 2016. 37: p. 114-121.
Ganesan, V., et al., Heterologous biosynthesis of natural product naringenin by co-culture engineering. Synthetic and Systems Biotechnology, 2017. 2(3): p. 236-242.
Zhang, H., et al., Engineering Escherichia coli coculture systems for the production of biochemical products. Proceedings of the National Academy of Sciences, 2015. 112(27): p. 8266.
Hosoda, K., et al., Cooperative Adaptation to Establishment of a Synthetic Bacterial Mutualism. PLoS ONE, 2011. 6(2): p. e17105.
Siedler, S., et al., Novel biosensors based on flavonoid-responsive transcriptional regulators introduced into Escherichia coli. Metabolic Engineering, 2014. 21: p. 2-8.
Cámara, B., et al., T7 phage protein Gp2 inhibits the Escherichia coli RNA polymerase by antagonizing stable DNA strand separation near the transcription start site. Proceedings of the National Academy of Sciences of the United States of America, 2010. 107(5): p. 2247-2252.
Westbrook, A.M. and J.B. Lucks, Achieving large dynamic range control of gene expression with a compact RNA transcription–translation regulator. Nucleic Acids Research, 2017. 45(9): p. 5614-5624.
