<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_GeorgiaTech_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:GeorgiaTech/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:GeorgiaTech/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:GeorgiaTech/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:GeorgiaTech/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:GeorgiaTech/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:GeorgiaTech/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navigater" align="center"><UL><LI class="column" id="HOME"><A href="https://2014.igem.org/Team:GeorgiaTech" style="color:#FFE740"> Home</A></LI><LI class="column" id="PROJECT"><A href="https://2014.igem.org/Team:GeorgiaTech/Project" style="color:#FFE740"> Project</A></LI><LI style="height:50px"><A href="https://2014.igem.org/Team:GeorgiaTech/Project/FlowbackProcessing" style="color:#FFE740">Sustainable Fracking</A></LI><LI><A href="https://2014.igem.org/Team:GeorgiaTech/Project/Primers" style="color:#FFE740">Insertion Primers</A></LI><LI><A href="https://2014.igem.org/Team:GeorgiaTech/Parts" style="color:#FFE740">Parts</A></LI><LI><A href="https://2014.igem.org/Team:GeorgiaTech/Modeling" style="color:#FFE740">Collaboration</A></LI></UL><LI class="column" id="SAFETY"><A href="https://2014.igem.org/Team:GeorgiaTech/Safety" style="color:#FFE740"> Safety</A></LI><LI class="column" id="NOTEBOOK"><A href="https://2014.igem.org/Team:GeorgiaTech/Notebook" style="color:#FFE740"> Notebook</A></LI><LI><A href="https://2014.igem.org/Team:GeorgiaTech/Notebook/Protocols" style="color:#FFE740">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:GeorgiaTech/Notebook/LabNotes" style="color:#FFE740">Labnotes</A></LI><LI class="column" id="TEAM"><A href="https://2014.igem.org/Team:GeorgiaTech/Team" style="color:#FFE740"> Team </A></LI><LI><A href="https://2014.igem.org/Team:GeorgiaTech/Team/Bios" style="color:#FFE740">Members</A></LI><LI><A href="https://2014.igem.org/Team:GeorgiaTech/Attributions" style="color:#FFE740">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=GeorgiaTech" style="color:#FFE740">Official Profile</A></LI></DIV><DIV class="content"><DIV id="wrapper" width="100%"><DIV id="left" style="margin-right:2.5%; float:left; width: 40%;"><DIV id="side-nav" style="height: auto;"><UL style="list-style: none;"><LI><H4 style="font: 18px 'Bitter' serif;">Overview</H4></LI><LI style="margin-top:10px"><A href="#Frack" style="font: 18px 'Bitter' serif; color: #840000;">What the Frack?</A></LI><LI style="margin-top:10px"><A href="#Elimination" style="font: 18px 'Bitter' serif; color: #840000;">Elimination System</A></LI><LI style="margin-top:10px"><A href="#Process" style="font: 18px 'Bitter' serif; color: #840000;">Process</A></LI><LI style="margin-top:10px"><A href="#Status" style="font: 18px 'Bitter' serif; color: #840000;">Project Status</A></LI><LI style="margin-top:10px"><A href="#Modeling" style="font: 18px 'Bitter' serif; color: #840000;">Protein Modeling</A></LI><LI style="margin-top:10px"><A href="#Future" style="font: 18px 'Bitter' serif; color: #840000;">Future</A></LI><LI style="margin-top:10px"><A href="#Work" style="font: 18px 'Bitter' serif; color: #840000;">Work Attributions and Acknowledgements</A></LI><LI style="margin-top:10px"><A href="#References" style="font: 18px 'Bitter' serif; color: #840000;">References</A></LI></UL></DIV></DIV><DIV id="team" style="margin-top:1.5%; position: relative;float:left; width: 55%;"><DIV class="meterouter" style="float: left; width:100%"><P>RBS Cloning Using PCR Primers: 100%</P><P>Promoter Cloning Using PCR Primers: 100%</P><P>Creation of sMMO Gene Biobricks: 100%</P><P>Creation of sMMO Expression Vectors: 47%</P><P>Characterization of Recombinant sMMO Proteins: 0%</P></DIV></DIV></DIV><DIV id="Frack" class="c-1"><H2> What the Frack? </H2></DIV><DIV class="bio"><DIV class="biotext"><P><B>Fracking is a mining technique</B> in which a high pressure fluid is injected into a well bore in order to generate small fractures. These fractures allow previously inaccessible hydrocarbons to migrate to the well for later extraction. Because fracking can be applied to most oil and natural gas wells, it is a common technique for extending the life of these wells, and thus lowering the need for construction of new wells. While fracking activity is crucial for providing low cost energy resources for American industry, there is a difficulty in storing and treating fracking waste water in an environmentally and economically suitable fashion. </P><P><B>The waste water,</B> commonly referred to as &quot;flowback,&quot; is often collected in open air evaporation pools, where water and volatile (likely to evaporate) hydrocarbons including methane will evaporate away, reducing the volume of fluids that must be transported for treatment. Although the evaporation of water is necessary to reduce treatment costs, the loss of volatile hydrocarbons produces an environmental hazard and is also a loss of product for the drilling company, since treatment of the flowback often yields additional liquid and gas fuels. Since methanol is a much less volatile compound than methane, the enzymatic conversion of methane to methanol will both prevent the release of methane into the atmosphere and increase the liquid fuel yield from flowback treatment.</P><A href="https://2014.igem.org/Team:GeorgiaTech/Project/FlowbackProcessing" style="font: 18px 'Bitter' serif; color: #840000;">Learn more about flowback treatment and our visions for improving it</A></DIV><DIV id="Elimination" class="c-1"><H2> Elimination System </H2></DIV><DIV class="bio"><DIV class="biotext"><P><B>In order to remove the methane</B> dissolved in the flowback, our team wants to engineer a strain of bacteria that can survive in temporary flowback, storage tanks and convert the methane to methanol before the flowback is depressurized and transferred to the open air evaporation pool. Our first step in this project is to engineer E.coli that expresses soluble methane monooxygenase (sMMO), an enzyme capable of converting methane to methanol.</P><P><B>This protein is found</B> exclusively in methanotrophs, a class of bacteria which are found in energy poor environments with high methane concentrations, from which they obtain their energy. The sMMO enzyme is composed of the proteins A, B, and C. Protein A is the hydroxylase which replaces one hydrogen on a methane with one hydroxyl group; it is composed of 2 symmetric dimers of subunits alpha, beta, and gamma. Protein C oxidizes NADH to acquire 2 electrons, which are then transferred by protein B to protein A for use in the methane oxidation pathway.</P><P><B>Previous attempts to clone sMMO</B> into E. coli by other labs have successfully expressed proteins B and C, however, protein A while expressed, was non-functional, likely due to improper folding of the protein.</P></DIV></DIV><DIV id="Process" class="c-1"><H2> Process </H2></DIV><DIV class="bio"><DIV class="biotext"><P><B>The genes necessary</B> to clone sMMO into E.coli are MMOX, MMOY, MMOZ, MMOB, and MMOC. The genes MMOX, MMOY, and MMOZ express the sub units of protein A while MMOB and MMOC express proteins B and C respectively. The sequences of these genes were found in literature and optimized for cloning in E.coli.</P><P><B>In comparison to previous attempts</B> to express sMMO in E.coli, this project is novel in it's use of synthetic, E.coli codon optimized sMMO genes and it's use of a wide array of varying strength expression vectors. In order to effectively regulate the translation and transcription rates for the genes a number of <A href="https://2014.igem.org/Team:GeorgiaTech/Project/Primers" style="color: #840000;">insertion primers</A> have been designed to turn our biobrick parts into expression vectors by inserting Ribosomal binding sites (RBS) and promoters into the PSB-1C3 plasmid (effectively eliminating months of unsuccessful ligations with small regulatory sequences). It is hypothesized that finding the right expression level for each sMMO gene relative to each other and the cell as a whole could create an &quot;optimum mix&quot; of sMMO proteins that minimizes protein misfolding and inclusion body formation and maximizes functional sMMO enzyme production.</P><P><B>In order to characterize the function</B> of the sMMO complex, the proposed methods included a 2D SDS PAGE as well as conducting a colorimetric plate assay for sMMO activity. The 2D SDS PAGE would show the production of sMMO proteins in the cell (functional or not) when compared against a control E.coli strain. On the other hand, the colorimetric assay would be able to assess the function of the entire sMMO protein complex. The colorimetric assay is a straight-forward method for screening methanotrophic colonies that express sMMO activity on solid media. The sMMO activity would result in the development of a colored complex which occurs when the sMMO reacts with naphthalene, and o-dianisidine. We would expect the methanotrophic colonies that express the sMMO to turn deep purple when exposed to these compounds.</P></DIV></DIV><DIV id="Status" class="c-1"><H2> Project Status </H2></DIV><DIV class="bio"><DIV class="biotext"><P><B>The Biobricked sMMO genes</B> are currently having RBS and promoter pairs of varying strengths inserted into the plasmids, with plans to characterize their different expression levels via 2D-SDS PAGE.</P><P><B>Currently,</B> RBS's are being inserted into the sMMOX and sMMOY biobricks, and promoters are being inserted into the sMMOC Biobrick. All four RBS and promoter pair crosses have already been created using sMMOZ and sMMOB.</P><H3>Current Confirmed RBS, Promoter, sMMO Gene Biobricks
<TABLE rules="all" style="text-align:center; border:1px solid #C0C0C0; width:100%;"><TBODY><TR><TH width="25%">sMMO Gene</TH><TH>X</TH><TH>Y</TH><TH>Z</TH><TH>B</TH><TH>C</TH></TR><TR><TD>HE RBS (promoterless)  </TD><TD> √ </TD><TD> √ </TD><TD> √ 
</TD></TR><TR><TD>LE RBS (promoterless)  </TD><TD> √ </TD><TD> √ </TD><TD> √ 
</TD></TR><TR><TD>HE Promoter/HE RBS     </TD><TD> √ </TD><TD> √ </TD></TR><TR><TD>HE Promoter/LE RBS     </TD><TD> √ </TD><TD> √ </TD></TR><TR><TD>LE Promoter/HE RBS     </TD><TD> √ </TD><TD> √ </TD></TR><TR><TD>LE Promoter/LE RBS     </TD><TD> √ </TD><TD> √ </TD></TR></TBODY></TABLE></H3><H2> sMMO Protein Models </H2></DIV><DIV class="bio"><DIV class="biotext"><DIV class="models" align="center"><UL style="position: relative; left:-1%;z-index:100;text-align: center;display: inline;list-style: none;padding: 0;       margin: 0;"><LI class="column" id="A" style="z-index:100;
  font: bold 14px/20PX helvetica;
  display: inline-block;
  position: relative;
  width: 15%;
  height: auto;
  padding-right: 5px;
  margin: 0;
  background-color: transparent;
  background-repeat:no-repeat;
  text-align:center;"><P style="margin-right:5%; text-align:center;">sMMO Protein A</P></LI><LI class="column" id="B" style="z-index:100;
  font: bold 14px/20PX helvetica;
  display: inline-block;
  position: relative;
  width: 15%;
  height: auto;
  padding-right: 5px;
  margin: 0;
  background-color: transparent;
  background-repeat:no-repeat;
  text-align:center;"><P style="margin-right:5%; text-align:center;">sMMO Protein B</P></LI><LI class="column" id="C" style="z-index:100;
  font: bold 14px/20PX helvetica;
  display: inline-block;
  position: relative;
  width: 15%;
  height: auto;
  padding-right: 5px;
  margin: 0;
  background-color: transparent;
  background-repeat:no-repeat;
  text-align:center;"><P style="margin-right:5%; text-align:center;">sMMO Protein C</P></LI><LI class="column" id="AB" style="z-index:100;
  font: bold 14px/20PX helvetica;
  display: inline-block;
  position: relative;
  width: 15%;
  height: auto;
  padding-right: 5px;
  margin: 0;
  background-color: transparent;
  background-repeat:no-repeat;
  text-align:center;"><P style="margin-right:5%; text-align:center;">A and B Complex</P></LI><LI class="column" id="ABC" style="z-index:100;
  font: bold 14px/20PX helvetica;
  display: inline-block;
  position: relative;
  width: 15%;
  height: auto;
  padding-right: 5px;
  margin: 0;
  background-color: transparent;
  background-repeat:no-repeat;
  text-align:center;"><P style="margin-right:5%; text-align:center;">A, B, and C Complex</P></LI></UL></DIV><P><B>Protein modelling</B> for the project is necessary for heterologous expression of genes in E.coli that come from a distant prokaryote in evolutionary terms. Comparing structures of the same protein expressed in the two organisms can yield changes in conformation necessary for the protein to function properly in the organism for which the genes have been transplanted. In addition, the catalytic mechanisms and active sites of these proteins are important to elucidate how the overall reaction takes place as there are many proteins/subunits in this complex. If mutagenesis of the proteins may be required to make them functional, predicting the possible outcomes and choosing the best manipulations to conduct experimentally saves resources and makes success rates substantially higher.</P><P><B>Pymol</B> (Python Molecular Modelling) was used to visualize the crystal structures of proteins A, B and the A+B complex obtained from the Protein Data Bank and from ab initio homology modelling. Additionally, Modeller from the Sali lab (Marti-Renom et. al) provided algorithms for multiple template, homology modelling which was used to reconstruct protein C from the two protein fragments that have been crystallized and published on rcsb.org. Protein C will be docked to the protein A and B complex using Autodock VINA to create the 3D model of the entire soluble methane monooxygenase supercomplex.</P></DIV></DIV><DIV id="Future" class="c-1"><H2> Future </H2></DIV><DIV class="bio"><DIV class="biotext"><P><B>In the long run,</B> our project’s end goal is to create a filtration system that directly treats waste water at fracking sites before the methane is allowed to evaporate out of the water. This filtration system will incorporate E.coli biofilm with our engineered sMMO that will convert methane into less volatile methanol as the waste water passes through to the REC enclosure. The methanol then can be used for other purposes, such as fuel. Our system will reduce the amount of methane contamination in the water and atmosphere, cut current treatment costs, and increase the renewability of fracking waste water.</P></DIV></DIV><DIV id="Work" class="c-1"><H2> Work Attributions and Acknowledgements </H2></DIV><DIV class="bio"><DIV class="biotext"><H4><B>Team Project Work Attributions</B></H4><P style="text-indent:0px"><B>sMMO Project:</B> designed and carried out by all members of this year's iGEM team, independent of the Barker Lab which does not study or work with Methanotrophs or Fracking.</P><P style="text-indent:0px"><B>Insertion primers project:</B>designed and carried out by all members of this year's iGEM team, continuing this work from the Georgia Tech 2013 iGEM Team.</P><P style="text-indent:0px"><B>Design of the wiki</B> was carried out independently by our wiki czar, without outside help, but with source code obtained from other team's wikis in past years.</P><P style="text-indent:0px"><B>All images</B> were created or taken by this year's iGEM Team excluding public domain images released under a creative commons license from Publicdomainpictures.net</P><P style="text-indent:0px"><B>All writings</B> on this wiki were composed by this year's iGEM Team or taken from the 2013 Georgia Tech wiki unless otherwise cited.</P><H4><B>Support Acknowledgements</B></H4><P style="text-indent:0px"><B>For general support</B> we would like to acknowledge Dr. Thomas Barker, Dr. Mark Styczynski, Dr. Eric Gaucher, and our two graduate advisors Dwight Chambers and Haylee Bachman for assistance in brainstorming and developing our ideas with feedback.</P><P style="text-indent:0px"><B>For laboratory support</B> we would like to acknowledge Dr. Thomas Barker who has provided us with laboratory space and supplies during our iGEM experience.</P></DIV></DIV><DIV id="References" class="c-1"><H2> References </H2></DIV><DIV class="bio"><DIV class="biotext"><OL style="list-style-type:decimal"><LI><P style="text-indent:0px">Coufal, D. E., Blazyk, J. L., Whittington, D. A., Wu, W. W., Rosenzweig, A. C., &amp; Lippard, S. J. (2000). Sequencing and analysis of the Methylococcus capsulatus (Bath) soluble methane monooxygenase genes. <I>European Journal of Biochemistry</I>, 267(8), 2174-2185.</P></LI><LI><P style="text-indent:0px"> A. Marti-Renom, A. Stuart,  A. Fiser,  R.Sanchez,  F. Melo,  A. Sali. Comparative protein structure modeling of genes and genomes. Annu. Rev. Biophys. Biomol. Struct. 29, 291-325, 2000.</P></LI><LI><P style="text-indent:0px">Murrell, J. C., Gilbert, B., &amp; McDonald, I. R. (2000). Molecular biology and regulation of methane monooxygenase. Archives of microbiology, 173(5-6), 325-332.</P></LI><LI><P style="text-indent:0px">Pandey, V. C., Singh, J. S., Singh, D. P., &amp; Singh, R. P. (2014). Methanotrophs: promising bacteria for environmental remediation. <I>International Journal of Environmental Science and Technology</I>, 11(1), 241-250.</P></LI><LI><P style="text-indent:0px">West, C. A., Salmond, G. P., Dalton, H., &amp; Murrell, J. C. (1992). Functional expression in Escherichia coli of proteins B and C from soluble methane monooxygenase of Methylococcus capsulatus (Bath).<I> Journal of general microbiology</I>, 138(7), 1301-1307.</P></LI></OL></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:GeorgiaTech/Project">http://2014.igem.org/Team:GeorgiaTech/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:GeorgiaTech/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:GeorgiaTech/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:GeorgiaTech/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:GeorgiaTech/Project&amp;oldid=347803" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>