"
Page
Discussion
View source
History
teams
Log in
 
Team:GeorgiaTech/Project/Primers
From 2014.igem.org
Home
Project
Sustainable Fracking
Insertion Primers
Parts
Collaboration
Safety
Notebook
Protocols
Labnotes
Team
Members
Attributions
Official Profile
Biobrick Primers for Universal Transcriptional and Translational Regulation
Ribosomal Binding Site Insertion
Purpose
The RBS primer is a standard primer that can be used for site directed mutagenesis in all linearized backbones available from the iGEM database.
such and such and such and such and such and such and such
Background
Most BioBrick parts require a Ribosomal Binding Site to express the protein. This has proven problematic as the RBS is very small and hard to insert using 3A Assembly. We believe the creation of an oligo primer with this RBS may be easier to place the RBS in front of the protein that is needed to be expressed. An easy oligo primer that could be used in PCR extension and then placed into a vector backbone would save time and frustration as to whether the part had been inserted.
such and such
Design
The format of this primer needs the BioBrick formatted prefix with an additional strong Ribosomal binding site within it. The prefix of the standard BioBrick was analyzed and the primer was found to be able to be inserted after the XbaI site and still have the scar intact between the part and the RBS site. The RBS sites chosen were selected from the Anderson RBS library degenerate sequence and a library was created with that degenerate sequence via the Salis Lab at Penn State University, optimized for E. coli 16s.
Promoter Sequence Insertion
Purpose
The Promoter primer is a standard primer that can be used for site directed mutagenesis in all linearized backbones available from the iGEM database and the Anderson RBS sites from the Anderson RBS library.
Background
Most BioBrick parts require a Promoter to express the protein. This has proven problematic as the promoter is very small and hard to insert using 3A Assembly. We believe the creation of an oligo primer with this promoter may be easier to place the promoter in front of the RBS site that is needed to be expressed. An easy oligo primer that could be used in PCR extension and then placed into a vector backbone would save time and frustration as to whether the part had been inserted.
Design
The format of this primer needs the BioBrick formatted prefix, the promoter and part of the RBS site. The prefix of the standard BioBrick was analyzed and the primer was found to be able to be inserted after the XbaI site and be able to insert the promoter just before the RBS site. The promoters were selected from the Anderson Constitutive Promoter Library in the iGEM parts registry.
Name
Type
Description
Length
BBa_K1539021
Device
This primer is used to insert a high efficiency RBS to transcribe genes starting with ATG in E. coli.
46 bp
BBa_K1539034
Device
This primer is used to insert a low efficiency RBS to transcribe genes starting with ATG in E. coli.
46 bp
BBa_K1539055
Device
This primer is used to insert a high efficiency promoter to translate E. coli. genes. Must be used after insertion of RBS primer. Use Georgia Tech iGEM RBS primers (BBa_K1539021 or BBa_K1539034) or an RBS primer with a similar design.
58 bp
BBa_K1539089
Device
This primer is used to insert a low efficiency promoter to translate E. coli. genes. Must be used after insertion of RBS primer. Use Georgia Tech iGEM RBS primers (BBa_K1539021 or BBa_K1539034) or an RBS primer with a similar design.
58 bp
Characterization of Function
Analysis
The flow cytometry data suggests successful insertion of both an RBS and a promoter via PCR with our specialized primers. The vertical lines transecting the histograms represent the “gate” or boundaries used to determine the presence of mCherry. When comparing the mean fluorescent molecules per cell, the high efficiency (HE) RBS and promoter combination is about 2,000 molecules per cell higher than the low efficiency (LE) RBS and promoter combination. This result was expected; however the degree to which the RBS or promoter individually played in the overall expression is yet to be determined. Insertion of the RBS and promoter crosses (e.g. RH2 with PL1 and RL1 with PH2) into pSB1C3_mCherry are underway. Theoretically, the flow cytometry data will yield plots that fit in between the current data.
Initial computer predicted results for these RBS and promoter efficiencies predicted that a much greater difference should have been seen between the high efficiency RBS and promoter pair and the low efficiency RBS and promoter pair. Predicted transcription rates for the RBS’s were: RH2= 0.914 and RL1=0.007. Theoretical translation rates for promoters were: PH2=0.86 and PL1=0.06. It is not yet determined why this discrepancy exists.
Retrieved from " http://2014.igem.org/Team:GeorgiaTech/Project/Primers "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
