<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_St_Andrews_modelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:St_Andrews/modelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:St_Andrews/modelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:St_Andrews/modelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:St_Andrews/modelling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:St_Andrews/modelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:St Andrews/modelling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="container0"><DIV class="top-slider-main"><DIV class="slider-box1"><DIV class="content"><A href="https://2011.igem.org/Team:St_Andrews" class="listmain">Description</A><A href="https://2011.igem.org/Team:St_Andrews/diary" class="listmain" id="diary">Project Diary</A><A href="https://2011.igem.org/Team:St_Andrews/safety" class="listmain" id="safety">Safety Form</A></DIV></DIV><DIV class="slider-box2"><DIV class="content"><A href="https://2011.igem.org/Team:St_Andrews/switch" class="listmain" id="kill">The Kill Switch</A><A href="https://2011.igem.org/Team:St_Andrews/modelling" class="listmain" id="model">Modelling</A><A href="https://2011.igem.org/Team:St_Andrews/parts" class="listmain" id="parts">Parts Submitted</A></DIV></DIV><DIV class="slider-box3"><DIV class="content"><A href="https://2011.igem.org/Team:St_Andrews/drugdel" class="listmain">Drug Delivery</A><A href="https://2011.igem.org/Team:St_Andrews/biosafe" class="listmain">Biosafety</A><A href="https://2011.igem.org/Team:St_Andrews/conj" class="listmain">Conjugation</A></DIV></DIV><DIV class="slider-box4"><DIV class="content"><A href="https://2011.igem.org/Team:St_Andrews/review" class="listmain">&quot;Is iGEM Fair?&quot;</A><A href="https://2011.igem.org/Team:St_Andrews/debate" class="listmain">Panel Debate</A><A href="https://2011.igem.org/Team:St_Andrews/essays" class="listmain">Reflective Essays</A></DIV></DIV><DIV class="slider-box5"><DIV class="content"><A href="https://2011.igem.org/Team:St_Andrews/team" class="listmain">The Team</A><A href="https://2011.igem.org/Team:St_Andrews/sponsors" class="listmain">Sponsors</A><A href="https://2011.igem.org/Team:St_Andrews/collaboration" class="listmain">Collaboration</A></DIV></DIV></DIV></DIV><DIV id="container"><DIV class="stutitles"><H1>Modelling</H1></DIV><DIV id="mainpart"><DIV id="wlinks"><TABLE id="tb" cellspacing="10"><TBODY><TR><TD><H4><A href="#details">Details</A></H4></TD><TD><H4><A href="#creating">Creating the Model</A></H4></TD><TD><H4><A href="#listeq">List Of Equations</A></H4></TD><TD><H4><A href="#noteable">Notation</A></H4></TD><TD><H4><A href="#prog">Programs Used</A></H4></TD><TD><H4><A href="#param">Parameter List</A></H4></TD></TR><TR><TD><H4><A href="#ranges">Ranges Used</A></H4></TD><TD><H4><A href="#init">Initial Concentrations</A></H4></TD><TD><H4><A href="#res">Results</A></H4></TD><TD><H4><A href="#assump">Assumptions</A></H4></TD><TD><H4><A href="#conclu">Conclusions</A></H4></TD><TD><H4><A href="#ref">References</A></H4></TD></TR></TBODY></TABLE></DIV><DIV id="t1"><A id="details"><H2>Details of our promoter: pBAD Strong</H2></A><P class="textpart">We decided to utilise the pBAD strong promoter (<A href="http://partsregistry.org/Part:BBa_K206000">K206000</A>), which was created by the British Columbia iGEM 2009 team as a mutagenized form of the naturally occurring pBAD promoter.  The modelling was based upon and used references pertaining to the pBAD promoter, also known as the 'ara operon'. pBAD and pBAD strong are both arabinose-inducible promoters.</P><P class="textpart">In the absence and presence of arabinose, the pBAD promoter acts as a repressor and an inducer respectively.  The gene products of pBAD in <I>Escherichia coli</I> allow the cells to take up and catabolize L-arabinose, a five-carbon sugar.  pBAD’s very basic structure is depicted below:</P><P class="textpart3">Figure 1: A basic structural outline of the pBAD and, adjacent, AraC protein, in the absence of arabinose. (Schleif, 2011)</P><P class="textpart">The three enzymes that comprise the pBAD promoter cause the catabolism of the sugar arabinose as follows:</P>
araA – <B>arabinose isomerase</B> which converts arabinose to ribulose
araB – <B>ribulokinase</B> which phosphorylates ribulose
araD – <B>ribulose-5-phosphate epimerase</B> which converts ribulose-5-phosphate which can then be metabolised via the pentose phosphate pathway (Patel, GU)<P class="textpart">Adjacent to the three pBAD structural genes is the AraC regulatory gene. The dimeric (the compound comprises of two structurally similar subunits called monomers which together make a dimer) AraC protein actively represses transcription as well as the synthesis of the pBAD genes; this occurs when arabinose is not present in the environment.  The AraC protein binds to the half sites araO<SUB>2</SUB> and ara I<SUB>1</SUB> which creates a loop within the DNA, thus blocking the RNA polymerase from binding to the pC and pBAD promoters. This can be seen in Figure 2, where it’s possible to comprehend the position of the relative half-sites.</P><P class="textpart3">Figure 2: The red circles in the AraC pockets denote the inducer, L-arabinose. (Schleif, 2003)</P><P class="textpart">When arabinose is present, it binds to the AraC protein and this destabilises the AraC protein binding to the araI<SUB>1</SUB>-O<SUB>2</SUB> half-site looped complex, but stabilizes binding to the adjacent half-sites araI<SUB>1</SUB> and araI<SUB>2</SUB>, which are upstream of the pBAD promoter. This then ‘straightens’ the DNA loop, (Carra and Schleif, 1993) allowing the activation of the transcription of pBAD (Schleif, 2011).</P><A id="creating"><H2>Creating a model</H2></A><P class="textpart">The modelling was established using the theoretical equations used in St Andrews 2010 iGEM team which can be found in their <A href="https://2010.igem.org/Team:St_Andrews/project/modelling/models/ODEs">wiki</A> under the subheading “Typical ODE Elements”. Our system is a series of five ordinary differential equations which attempt to mathematically describe the very basic functions in our biology, they are as follows:</P><A id="listeq"><H2>List of Equations</H2></A><A id="noteable"><H2>Notation Table</H2></A><A id="prog"><H2>Programs Used</H2></A><P class="textpart">We considered coding our own basic iterative method but consequently decided that there may be too many complications and bugs to worry about so opted to perform our modelling in the MATLAB environment. We also opted for the same strategy that last year’s St Andrews team adopted, namely the 4th order Runge-Kutta solver. This solver is inbuilt to the MATLAB environment, and is stated as ode45.</P><A id="param"><H2>Parameters Used</H2></A><P class="textpart">The various parameters were researched throughout previous iGEM team’s findings, several journals and numerous articles. The initial relevant findings and their sources are given below:</P><P class="textpart">Referring back to the equations, it can be seen that we had difficulties in locating five of the parameters within any literature. These particular constants and constraints were attempted, via parameter sensitivity analysis, at being established. We used ranges for these elusive parameters to find optimal values.</P><A id="ranges"><H2>Parameter Ranges</H2></A><P class="textpart">These rates were estimated from a variety of sources. The optimal extracellular arabinose concentration is actually assumed to be around 1x10<SUP>-6</SUP>M, as if the extracellular arabinose concentration exceeds 1x10<SUP>-7</SUP>M then it allows the induction of the AraC promoter [8] and according to the Standard Registry of Biological Parts [1], the ‘switching point’ for arabinose in pBAD is 3.14x10<SUP>-6</SUP>M.</P><A id="init"><H2>Initial Concentrations</H2></A><A id="res"><H2>Results</H2></A><P class="textpart">There was a significant amount of output from the model as we had a considerable volume of parameters to compare against. 5 subplots were created from the model, which were in reference to the time evolution of the concentrations of intracellular arabinose, arabinose-AraC complex, AraC protein, mRNA coding and protegrin-1 respectively. For the production of the protegrin-1, mainly encased in equation (1.5), the parameters are unknown so there was a great deal of variation in the final graph produced in the output. Below are two selected outputs from the model which depict the different behaviour and the possible instability of the protegrin-1 production.</P><P class="textpart3">Figure 3: The MATLAB output from the stated initial conditions (where the y-axis is molar concentration (M) for the respective compounds or complexes for each respective equation (1.1)-(1.5) and the x-axis is time (seconds)). </P><P class="textpart3">Figure 4: The MATLAB output from the stated initial conditions (where the y-axis is molar concentration (M) for the respective compounds or complexes for each respective equation (1.1)-(1.5) and the x-axis is time (seconds)). </P><P class="textpart">These two sets of graphs use a similar set of initial conditions chosen from the ranges given above. The initial conditions that actually created each series are labelled above each respective graph array. However it can be seen that the Figure 3 has a complication when it comes to the concentration of the protegrin-1 concentration graph (the last in each of the series). These graphs depict the instability that forms because of the initial conditions. The method employed in creating the equations allows the protegrin-1 to remain positive if it starts positive. However, after the initial time lag the protegrin-1 concentration must become negative and therefore induce the instability that is shown. Also the oscillatory and almost severe peaks and troughs formed on the graph were assumed to have been contributed from the time step utilised in the odesolver inbuilt in MATLAB. The time step in MATLAB attempts to optimise the data that it collects which can also affect the output. </P><P class="textpart"> Figure 4 shows a much smoother inclination, and positive, for the protegrin-1 concentration and, at the end of the iterations, appears to be approaching a steady concentration value of 4x10<SUP>-4</SUP>. This was an expected output for the model. This comparison between just 2 selected outputs, details the variation between very small changes in the initial conditions and how they affect the behaviour of the model.</P><A id="assump"><H2>Assumptions and Complications</H2></A><P class="textpart">The modelling was based on a set of assumptions and there were a variety of complications that were neglected to simplify the model initially. Any further work would have included more of these complications to match the real biological system.</P><P class="textpart">The theory based on the system was based on a unicellular system however the concentrations used are for multicellular environments. The cell’s own pH was assumed to be similar to deionised water when referring to the degradation of arabinose inside the cell. The diffusion of arabinose within the cell was assumed to be instantaneous, moreover that once the arabinose has entered the cell it would remain intracellular. In addition to this, the diffusion of compounds within the cell to the right place was deemed as instantaneous, i.e. space in the model was neglected. There is also an equilibrium created between L-arabinose (the particular structure that induces pBAD) and L-ribulose [1], so as increased concentrations of the arabinose is added there is a shift to create L-ribulose (which is another sugar that would have been consumed by the cell). This means that the arabinose concentration may decrease slightly before the equilibrium stabilises. In our case we assumed that this shift, again, was instantaneous [1]. The degradation of arabinose was to include any consumption or use of it in other metabolic pathways in the cell.</P><P class="textpart">Also it was presumed that there were infinite supplies of ribosome and ribosome binding sites available and that the RNA polymerase would not become degraded.  A condition we placed on the model, as well as assumed, was that the maximal transcription rate is always greater than the half-maximal transcription rate as this makes logical sense.
Dissociation and association constants of the AraC protein from the half-sites were neglected at this stage as were investigating the simplest setup of the biology. This particular process is reasonably slow and would have caused a time lag in the initial stages.</P><A id="conclu"><H2>Conclusions</H2></A><P class="textpart">Based on our current results, we were not able to find conclusive data. The next stage would have been to look at the stability of the system and which rate values the model would have behaved as expected or become oscillatory.</P><P class="textpart">We have a number of theories that have been discussed as to the technicalities encountered in the model.
The first of which involved the maximal transcription rate constants’ units and the arabinose-AraC complex association constant. These were initially supposed to be in s<SUP>-1</SUP> however the k<SUB>maxprodRNA </SUB> was considered to be in Ms<SUP>-1</SUP>, the k<SUB>1/2maxprodRNA </SUB>in M and the arabinose-AraC complex in M<SUP>-1</SUP>s<SUP>-1</SUP>. In order to find the association constant, it was discussed that second order chemical kinetics be utilised and, by knowing the equilibrium constant [1], the association constant could be calculated.</P><P class="textpart">The second theory was that once we have a suggested value for the association constant, we can provide some mathematical estimation in finding the transcriptional rate constants. This is based on whether the equations (1.1)-(1.3), referred to as the AraC cycle, is much faster than the PG-1 cycle, equations (1.4) and (1.5), or even if both cycles perform at similar speeds. If we assume that both sets are of a relatively comparable speed then all terms on the right hand side of the ODEs should be of a similar magnitude and we could choose the constants to show this is true. However, if the two cycles are of differing speeds then the requirement would be to find the equilibrium of AraC cycle, by allowing the left hand side of each equation equal zero. Via this process, the steady value output for the arabinose-AraC (A<SUB>1</SUB>AraC) complex could be obtained and then entered into the AMP cycle, therefore creating two separate systems.</P><A id="ref"><H2>References:</H2></A><P class="textpart2">[1] – Schleif, R (1969), &quot;Induction of the L-arabinose operon&quot;, J.Mol.Biol, Vol 46, pg197-199. 
<A href="http://www.ncbi.nlm.nih.gov/pubmed/4902209">Link to paper.</A></P><P class="textpart2">[2] – Hendrickson, W and Schleif, R (1984), &quot;Regulation of the <I>Escherichia coli </I> L-Arabinose Operon Studied by Gel Electrophoresis DNA Binding Assay&quot;, J.Mol.Biol, Vol 174, pg 611-628.
<A href="http://www.sciencedirect.com/science/article/pii/0022283684902419">Link to paper.</A></P><P class="textpart2">[3] – Koostra, A et al (2009), &quot;Differential effects of mineral and organic acids on the kinetics
of arabinose degradation under lignocellulose pretreatment conditions&quot;, Biochemical Engineering Journal, Vol 43, pg 92-97.
<A href="https://engineering.purdue.edu/ABE/People/Papers/nathan.s.mosier.1/arabinose">Link to paper.</A></P><P class="textpart2">[4] – Davidson, C et al (2010), &quot;Integration of transcriptional inputs at promoters of the arabinose catabolic pathway&quot;, BMC Systems Biology, Vol 4, pg75.
<A href="http://www.biomedcentral.com/1752-0509/4/75">Link to paper.</A></P><P class="textpart2">[5] – N/A (2011), &quot;Registry of Standard Biological Parts - pBAD Strong Characterization&quot;, Last Updated 22 Oct 2009)
<A href="http://partsregistry.org/Part:BBa_K206000:Characterization">Link.</A></P><P class="textpart2">[6] – Alon, U (2007) “An Introduction to Systems Biology Design Principles of Biological Circuits.” London: Chapman &amp; Hall/CRC, 1st ed.
</P><P class="textpart2">[7] – Weldon, J et al (2007), &quot;Structure and Properties of a Truly Apo Form of AraC Dimerization Domain&quot;, Proteins: Structure, Function and Bioinformatics, Vol 66, pg646-654.
<A href="http://gene.bio.jhu.edu/Ourspdf/118.pdf">Link to paper.</A></P><P class="textpart2">[8] – Schleif, R (2010), &quot;AraC protein, regulation of the L-arabinose operon in <I>Escherichia coli </I>, and the light switch mechanism of AraC action&quot;, FEMS Microbiol Rev, pg1-18.
<A href="http://gene.bio.jhu.edu/Ourspdf/127.pdf">Link to paper.</A></P><P class="textpart2">Patel, Bharat. &quot;Regulation of Gene Expression in Prokaryotes.&quot; Lecture 4. Griffiths University. <A href="http://trishul.ict.griffith.edu.au/courses/3011bbs/Lecture4.pdf">Link to paper.</A></P><P class="textpart2">Schleif, R (2003), &quot;AraC Protein: A Love-Hate Relationship.&quot; BioEssays, Vol. 25, pg. 274-282. <A href="http://gene.bio.jhu.edu/Ourspdf/109.pdf">Link to paper.</A></P><P class="textpart2">Schleif, Robert.  The Johns Hopkins University. Last updated August 2011. <A href="http://gene.bio.jhu.edu/">Link to paper.</A></P><P class="textpart2">Carra, John and Schleif, R (1993) &quot;Variation of half-site organization and DNA looping by AraC protein.&quot; The EMBO Journal, Vol. 12, pg. 35-44. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC413173/pdf/emboj00073-0049.pdf">Link to paper.</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:St_Andrews/modelling">http://2011.igem.org/Team:St_Andrews/modelling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:St_Andrews/modelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:St_Andrews/modelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:St_Andrews/modelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:St_Andrews/modelling&amp;oldid=168148" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>