"
Page
Discussion
View source
History
teams
Log in
 
Team:Tuebingen/NotebookPreparations
From 2012.igem.org
Home
Team
Who we are
Where we're from
First steps
Attributions
Experience
Impressions
Acknowledgment
Project
Overview
Questions
Mechanism
Implementation
Submitted Parts
Application
Results
References
Notebook
Preparations
Reports
Protocols
Appendix
Calendar
Activities & Press
Safety
Sponsorship
Contact us
Preparations
Contents
1 Preparations
1.1 Identification of plasmids
1.2 Identification of genes
1.3 Sequence analysis and primer design
After determining our principal project idea we had to design our system prior to any work in the wet lab. Several steps were involved:
Identification of plasmids
We decided to use a shuttle vector which works in E. coli and S. cerevisiae. The advantage is that we can assemble and build our target constructs in the rapid growing E. coli. The shuttle vectors must meet the following demands:
Multiple cloning site with XbaI and SpeI restriction sites
Multiple cloning site with beta-galactosidase, so blue-white screening is possible
Ampicillin resistance
amino-acid genes for selection
Integration site for integration in the yeast genome
After consultation with Prof. Jansen, our yeast expert, we decided to use the shuttle vectors pRS313, pRS315 and pRS316.
Due to several EcoRI and PstI restriction sites in the pRS plasmids (not located around the multiple cloning site), we can only use XbaI and SpeI for assembly.
Identification of genes
Receptors: Aiming at aquatic environment we chose to use the membrane progesterone receptor (mPR) from the model organism Danio rerio (zebrafish). The second organism was Salmo salar (salmon) but we could not locate a homologous gene due to missing full genome sequence data. Following a database-wide BLAST search we selected the membrane progesterone receptor from Xenopus laevis (African clawed frog), another well-studied model organism, as our second receptor.
Signalling: The fet3 promoter targeted by the membrane receptors was already determined and proven working by J Smith et al. (2008). We could not obtain the sequence used by Smith and decided to take the upstream sequence (approx. 600bp) of the fet3 gene.
Since Pfet3 is regulated negatively by the mPR we decided on inverting our signal with an additional signaling step to have more sensitive measurement results. So the second part of the signalling system needs a repressor and its target. Additionally, knock-out strains not containing a working repressor have to be viable. We chose the mig1 (repressor) / Psuc2 (repressor target) pair and the rox1 (repressor) / Panb1 (repressor target) pair.
Reporter: The targets of our signalling system regulate our reporter. We have access to the plasmid storage of our lab. Common reporters used in this yeast-based environment are firefly luciferase and beta-galactosidase. Both are available to us and have no legal issues concerning the publishing in the PartsRegistry.
Sequence analysis and primer design
Next was the check of all sequences for unwanted restriction sites.
As noted above, due to several unwanted restriction sites (EcoRI and PstI) in the pRS vectors, we can only use XbaI and SpeI for assembly.
The firefly luciferase has an unwanted XbaI site so we had to use NheI/SpeI restriction instead. The XbaI site could be removed by gene synthesis.
Three weeks before the official lab time started, we designed our primers with Vector NTI . The programm was basically used to compute fitting primers. We then manually optimized the primer sequences by setting G and C nucleotides at the 3'-tail so that the development of hydrogen bridge bonds is increased, which leads to stronger bindings. In addition the primer sequences were modified so that the related primer pairs have preferably the same or close annealing temperatures.
Part #
Part name
forward primer
reverse primer
1
lacZ
5’-GCTAGCATGGTGCTGCGTTGG
5’-ACTAGTTTATTTTTGACACCAG
2
luciferase
5’-GCTAGCATGGAAGACGCCAAA
5’-ACTAGTTTAAAGCTTCTTTCCGCC
3
Padh1
5’-TCTAGAAAGAAATGATGGTAAA
5’-ACTAGTAGTTGATTGTATGCTT
4
Psuc2
5’-TCTAGACATACTAAGACATTTACCG
5’-ACTAGTCATATACGTTAGTGAAAA
5
Pfet3
5’-TCTAGACATTACTGCTGTAAAAAGG
5’-GCAAAAAATTAGAACTAGACTAGT
6
Panb1
5’-TCTAGATTTTTTCCTGTGTTCACC
5’-ACTAGTGTTTTAGTGTGTGAATGA
7
Tadh1
5’-TCTAGAGCGAATTTCTTATGATTT
5’-ACTAGTAGGTGTTGTCCTCTGAGG
8
rox1
5’-TCTAGAATGAATCCTAAATCCTCTACAC
5-ACTAGTAATTGTTCTTTTGAGGCG
Retrieved from " http://2012.igem.org/Team:Tuebingen/NotebookPreparations "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
