<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tuebingen_NotebookReports"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tuebingen/NotebookReports">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tuebingen/NotebookReports&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tuebingen/NotebookReports&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tuebingen/NotebookReports&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tuebingen/NotebookReports" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tuebingen/NotebookReports</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="nav"><UL><LI><A href="/Team:Tuebingen" title="Team:Tuebingen">Home</A></LI><LI><SPAN>Team</SPAN></LI><LI><A href="/Team:Tuebingen/TeamWho" title="Team:Tuebingen/TeamWho">Who we are</A></LI><LI><A href="/Team:Tuebingen/TeamWhere" title="Team:Tuebingen/TeamWhere">Where we're from</A></LI><LI><A href="/Team:Tuebingen/TeamWhat" title="Team:Tuebingen/TeamWhat">First steps</A></LI><LI><A href="/Team:Tuebingen/Attributions" title="Team:Tuebingen/Attributions">Attributions</A></LI><LI><A href="/Team:Tuebingen/ResultExp" title="Team:Tuebingen/ResultExp">Experience</A></LI><LI><A href="/Team:Tuebingen/ResultPersonal" title="Team:Tuebingen/ResultPersonal">Impressions</A></LI><LI><A href="/Team:Tuebingen/ResultAcks" title="Team:Tuebingen/ResultAcks">Acknowledgment</A></LI></UL><LI><SPAN>Project</SPAN></LI><LI><A href="/Team:Tuebingen/ProjectOverview" title="Team:Tuebingen/ProjectOverview">Overview</A></LI><LI><A href="/Team:Tuebingen/ProjectQuestions" title="Team:Tuebingen/ProjectQuestions">Questions</A></LI><LI><A href="/Team:Tuebingen/ProjectMechanism" title="Team:Tuebingen/ProjectMechanism">Mechanism</A></LI><LI><A href="/Team:Tuebingen/ProjectImplementation" title="Team:Tuebingen/ProjectImplementation">Implementation</A></LI><LI><A href="/Team:Tuebingen/Parts" title="Team:Tuebingen/Parts">Submitted Parts</A></LI><LI><A href="/Team:Tuebingen/Application" title="Team:Tuebingen/Application">Application</A></LI><LI><A href="/Team:Tuebingen/Results" title="Team:Tuebingen/Results">Results</A></LI><LI><A href="/Team:Tuebingen/References" title="Team:Tuebingen/References">References</A></LI><LI><SPAN>Notebook</SPAN></LI><LI><A href="/Team:Tuebingen/NotebookPreparations" title="Team:Tuebingen/NotebookPreparations">Preparations</A></LI><LI><STRONG class="selflink">Reports</STRONG></LI><LI><A href="/Team:Tuebingen/NotebookProtocols" title="Team:Tuebingen/NotebookProtocols">Protocols</A></LI><LI><A href="/Team:Tuebingen/NotebookAppendix" title="Team:Tuebingen/NotebookAppendix">Appendix</A></LI><LI><A href="/Team:Tuebingen/NotebookCalendar" title="Team:Tuebingen/NotebookCalendar">Calendar</A></LI><LI><A href="/Team:Tuebingen/Activities" title="Team:Tuebingen/Activities">Activities &amp; Press</A></LI><LI><A href="/Team:Tuebingen/Safety" title="Team:Tuebingen/Safety">Safety</A></LI><LI><A href="/Team:Tuebingen/Sponsorship" title="Team:Tuebingen/Sponsorship">Sponsorship</A></LI><LI><A href="/Team:Tuebingen/Contact" title="Team:Tuebingen/Contact">Contact us</A></LI></DIV><H1><SPAN class="mw-headline" id="Weekly_Reports"> Weekly Reports </SPAN></H1><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Weekly_Reports"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Weekly Reports</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Procedure"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Parts.2C_Plasmids_and_Constructs"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Parts, Plasmids and Constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Week_1_.287.2F9_-_7.2F15.29"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Week 1  (7/9 - 7/15)</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Week_2_.287.2F16_-_7.2F22.29"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Week 2  (7/16 - 7/22)</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Week_3_.287.2F23_-_7.2F29.29"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Week 3  (7/23 - 7/29)</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Week_4_.287.2F30_-_8.2F05.29"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Week 4  (7/30 - 8/05)</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Week_5_.288.2F06_-_8.2F12.29"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Week 5  (8/06 - 8/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Week_6_.288.2F13_-_8.2F19.29"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Week 6  (8/13 - 8/19)</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Week_7_.288.2F20_-_8.2F26.29"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Week 7  (8/20 - 8/26)</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Week_8_.288.2F27_-_9.2F02.29"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">Week 8  (8/27 - 9/02)</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Week_9_.289.2F03_-_9.2F09.29"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">Week 9 (9/03 - 9/09)</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Week_10_.289.2F10_-_9.2F16.29"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">Week 10 (9/10 - 9/16)</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Week_11_.289.2F17_-_9.2F23.29"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">Week 11 (9/17 - 9/23)</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Week_12_.289.2F24_-_9.2F30.29"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">Week 12 (9/24 - 9/30)</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Procedure"> Procedure </SPAN></H2><P>The following illustration gives a general overview to our approach of lab work.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">lab procedures</DIV></DIV></DIV></DIV><P>Both parts and vectors had to be worked on separately until they are available as digested fragments with XbaI/SpeI overhangs. Then parts could be ligated into the shipping plasmid pSB1C3 to submit them to the Parts Registry. To assemble our parts into the desired constructs ligation in correct order in the pRS plasmids were necessary. The final step is the transformation of all three plasmids into yeast strains.
</P><H2><SPAN class="mw-headline" id="Parts.2C_Plasmids_and_Constructs"> Parts, Plasmids and Constructs </SPAN></H2><P>To understand all referenced parts and their enumeration, here is a full listing:
</P><TABLE class="wikitable"><TBODY><TR><TH> Part # </TH><TH> Part name </TH><TH> source </TH><TH> length [bp] </TH><TH> annealing temperature </TH><TH> Registry Part
</TH></TR><TR><TD> 1 </TD><TD> lacZ </TD><TD> plasmid of AG Jansen  (University of Tuebingen) </TD><TD> 2514 </TD><TD> 48.0 °C </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K950005" class="external text" rel="nofollow">BBa_K950005</A></TD></TR><TR><TD> 2 </TD><TD> luciferase </TD><TD> plasmid of AG Jansen </TD><TD> 1650 </TD><TD> 46.4 °C </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=BBa_K950004" class="external text" rel="nofollow">BBa_K950004</A></TD></TR><TR><TD> 3 </TD><TD> Padh1 </TD><TD> plasmid of the iGEM Kit </TD><TD> 1457 </TD><TD> 48.0 °C </TD><TD><A href="http://partsregistry.org/wiki/index.php/Part:BBa_K165015" class="external text" rel="nofollow">BBa_K165015</A></TD></TR><TR><TD> 4 </TD><TD> Psuc2 </TD><TD> genomic yeast DNA </TD><TD> 711 </TD><TD> 45.6 °C </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K950003" class="external text" rel="nofollow">BBa_K950003</A></TD></TR><TR><TD> 5 </TD><TD> Pfet3 </TD><TD>  genomic yeast DNA </TD><TD> 587 </TD><TD> 47.1 °C </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K950000" class="external text" rel="nofollow">BBa_K950000</A></TD></TR><TR><TD> 6 </TD><TD> Panb1 </TD><TD>  genomic yeast DNA </TD><TD> 412 </TD><TD> 48.0 °C </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K950002" class="external text" rel="nofollow">BBa_K950002</A></TD></TR><TR><TD> 7 </TD><TD> Tadh1 </TD><TD>  genomic yeast DNA </TD><TD> 317 </TD><TD> 51.8 °C </TD></TR><TR><TD> 8 </TD><TD> rox1 </TD><TD>  genomic yeast DNA </TD><TD> 1237 </TD><TD> 49.9 °C </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K950001" class="external text" rel="nofollow">BBa_K950001</A></TD></TR><TR><TD> 9 </TD><TD> mPR <I>Danio rerio</I></TD><TD><I>Danio rerio</I>, synthesized by IDT </TD><TD> 1077 </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K950006" class="external text" rel="nofollow">BBa_K950006</A></TD></TR><TR><TD> 10 </TD><TD> mig1 </TD><TD><I>Saccharomyces cerevisiae</I>, synthesized by IDT </TD><TD> 1527 </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K950009" class="external text" rel="nofollow">BBa_K950009</A></TD></TR><TR><TD> 11 </TD><TD> mPR <I>Xenopus laevis</I></TD><TD><I>Xenopus laevis</I>, synthesized by IDT </TD><TD> 1074 </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K950007" class="external text" rel="nofollow">BBa_K950007</A></TD></TR><TR><TD> pGEM-T Easy vector </TD><TD> pGEM-T Easy Vector Kit </TD><TD> 3015 </TD></TR><TR><TD> pRS313 vector </TD><TD>  vector of AG Jansen </TD><TD> 4967 </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K950008" class="external text" rel="nofollow">BBa_K950008</A></TD></TR><TR><TD> pRS315 vector </TD><TD> vector of AG Jansen </TD><TD> 6018 </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K950010" class="external text" rel="nofollow">BBa_K950010</A></TD></TR><TR><TD> pRS316 vector </TD><TD> vector of AG Jansen </TD><TD> 4887 </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K950011" class="external text" rel="nofollow">BBa_K950011</A></TD></TR></TBODY></TABLE><P>For the full information of these parts in the Parts Registry, refer to <A href="/Team:Tuebingen/Parts" title="Team:Tuebingen/Parts">Submitted Parts</A>.
</P><H2><SPAN class="mw-headline" id="Week_1_.287.2F9_-_7.2F15.29"> Week 1  (7/9 - 7/15) </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">LB, SOB, TAE</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">TOP10 cells on plate</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">PCR results Psuc2, Padh1, luciferase, LacZ</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">PCR results Rox1, Tadh1, Pfet3, Panb1</DIV></DIV></DIV><P>Thursday the 12th of July was the first day in our laboratory.
</P><P>1. At first different substances, for example LB, SOB and TAE buffer 50x, which would be necessary for the further practice, were prepared.
</P><P>2.  To determine the optimal annealing temperature, a <A href="/Team:Tuebingen/NotebookProtocols#PCR" title="Team:Tuebingen/NotebookProtocols">gradient PCR</A> for the parts 1-8 was performed. 
For each part we made a 7 times preparation with a gradient of 45°C - 58,9°C with the following intermediate stages:
</P><TABLE class="wikitable"><TBODY><TR><TH> Position in the  thermocycler </TH><TH> gradient
</TH></TR><TR><TD> A </TD><TD> 60.0 °C
</TD></TR><TR><TD> B </TD><TD> 58.9 °C
</TD></TR><TR><TD> C </TD><TD> 57.0 °C
</TD></TR><TR><TD> D </TD><TD> 54.1 °C
</TD></TR><TR><TD> E </TD><TD> 50.5 °C
</TD></TR><TR><TD> F </TD><TD> 47.9 °C
</TD></TR><TR><TD> G </TD><TD> 46.0 °C
</TD></TR><TR><TD> H </TD><TD> 45.0 °C
</TD></TR></TBODY></TABLE><P>Doing a <A href="/Team:Tuebingen/NotebookProtocols#Gel_electrophoresis" title="Team:Tuebingen/NotebookProtocols">gel-electrophoresis</A> the PCR results were tested.
</P><P>3. Preparing chemocompetent <I>E. coli TOP 10</I> cells after <A href="/Team:Tuebingen/NotebookProtocols#Chemo-competent_cells" title="Team:Tuebingen/NotebookProtocols">Inoue protocol</A>.
</P><H2><SPAN class="mw-headline" id="Week_2_.287.2F16_-_7.2F22.29"> Week 2  (7/16 - 7/22) </SPAN></H2><P>1.  A successful <A href="/Team:Tuebingen/NotebookProtocols#PCR" title="Team:Tuebingen/NotebookProtocols">PCR</A> for the Parts 1-7 with the optimal annealing temperature was performed. It was controlled by a <A href="/Team:Tuebingen/NotebookProtocols#Gel_electrophoresis" title="Team:Tuebingen/NotebookProtocols">gel-electrophoresis</A>.
</P><P>The Parts 3-7 were cleaned with a PCR-DNA-Purification-Kit. After that the concentration of the purified parts was measured with NanoDrop.
</P><P>2. To test the competence of the chemocompetent <I>E. coli TOP 10</I> cells a <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformation</A> with pRS313 and a negative control was done. Due to the fact that the competent cells didn't work, new chemocompetent <I>E. coli TOP 10</I>  cells were prepared. The <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformation</A> of pRS313, pRS315 and pRS316 in these competent cells was successful.
</P><H2><SPAN class="mw-headline" id="Week_3_.287.2F23_-_7.2F29.29"> Week 3  (7/23 - 7/29) </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">gel electrophoresis documentation system</DIV></DIV></DIV><P>1. The first <A href="/Team:Tuebingen/NotebookProtocols#pGEM_Ligation" title="Team:Tuebingen/NotebookProtocols">ligation</A> of the parts 1-8 in pGEM with following <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformation</A> in the competent <I>E. coli TOP10</I> was done. But unfortunately only a few colonies grew on the inoculated agar-plates, which were incubated over night at 37°C. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">gel with pGEM bands and a few inserts</DIV></DIV></DIV><P>The <A href="/Team:Tuebingen/NotebookProtocols#control_digest" title="Team:Tuebingen/NotebookProtocols">restriction digest</A> was performed on the extracted plasmids of the grown colonies to control the ligation of the insert. The following gel electrophoresis showed that the ligation was not successful, because only bands of 3000bp for the pGEM vector was visible, but no bands for the insert.
</P><P>2. A new <A href="/Team:Tuebingen/NotebookProtocols#PCR" title="Team:Tuebingen/NotebookProtocols">PCR</A> of the parts 1 and 8 was executed. Using a PCR-DNA-Purification-Kit the PCR-product of part 8 was purified. The PCR-product of part 1 was purified with a preparative gel. The concentration of the final products was measured with NanoDrop.
</P><H2><SPAN class="mw-headline" id="Week_4_.287.2F30_-_8.2F05.29"> Week 4  (7/30 - 8/05) </SPAN></H2><P>1. The shipment with the synthesized parts (mPR of <I>Danio rerio</I> and mig1) arrived.
</P><P>2. A <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformation</A> of the parts mPR <I>Danio rerio</I> and mig1 was performed using the competent <I>E. coli TOP10</I> cells. Another transformation of the backbone plasmids pRS313, pRS315 and pRS316 was executed. Both were successful.
</P><P>The first attempt to isolate the plasmids was through usage of a plasmid preparation kit, but this try failed. Therefore the plasmid isolation was successfully repeated using alkaline lysis.
</P><P>3. A new <A href="/Team:Tuebingen/NotebookProtocols#PCR" title="Team:Tuebingen/NotebookProtocols">PCR</A> of the parts 1-8 with Taq/Pfu polymerase was performed applicating new yeast DNA. As an effect of the frequent freezing and defrosting the old yeast DNA was probably damaged. Therefore some earlier PCRs did not work.
</P><H2><SPAN class="mw-headline" id="Week_5_.288.2F06_-_8.2F12.29"> Week 5  (8/06 - 8/12) </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">freezer with most of our reagent</DIV></DIV></DIV><P>1. A <A href="/Team:Tuebingen/NotebookProtocols#control_digest" title="Team:Tuebingen/NotebookProtocols">small restriction digest</A> of the shuttle vectors pRS313, pRS315 and pRS316 was performed with XbaI and SpeI in order to examine the capability to linearize with the right overhangs for a ligation to take place later.
The restriction digest was executed with the parts mig1 and mPR of <I>Danio rerio</I>, too.
Due to unclean plasmids and DNA (perhaps to much salt) this step had to be repeated several times, because the restriction digests were incomplete.
</P><P>Therefore the plasmids (pRS313, pRS315, pRS316 and the parts mig1, mPR <I>Danio rerio</I>) were purified again with a Midi Prep DNA purification kit. Now the restriction digest was executed completely. We estimate that max. 30 µg DNA can be digested with our reaction.
</P><P>The purification of the parts mig1 and mPR <I>Danio rerio</I> and the pRS vectors was performed with a <A href="/Team:Tuebingen/NotebookProtocols#Genaxxon_PCR_DNA_Purification_Mini_Prep_Kit" title="Team:Tuebingen/NotebookProtocols">PCR Purification Kit</A> in order to prepare the DNA for ligation.
</P><P>2. A new <A href="/Team:Tuebingen/NotebookProtocols#PCR" title="Team:Tuebingen/NotebookProtocols">PCR</A> of the parts 1-8 was executed using Herculase in order to obtain a higher amount of PCR product. The polymerase Herculase was used due to its precision and productivity. Indeed the result of the PCR was better than with the Pfu/Taq polymerase.
</P><P>A preparative gel for PCR products 3, 4, 5, 6, 7, 8 (from PCR with Herculase) provided new template DNA for another PCR with Taq/Pfu polymerase.
</P><H2><SPAN class="mw-headline" id="Week_6_.288.2F13_-_8.2F19.29"> Week 6  (8/13 - 8/19) </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">preparative gel to isolate digested inserts</DIV></DIV></DIV><P>1. The first successful <A href="/Team:Tuebingen/NotebookProtocols#pGEM_Ligation" title="Team:Tuebingen/NotebookProtocols">ligation</A> and <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformation</A> into pGEM vector of part 4 in <I>E. coli TOP10</I> was executed. A lot of colonies grew on the agar-plate. After a <A href="/Team:Tuebingen/NotebookProtocols#control_digest" title="Team:Tuebingen/NotebookProtocols">restriction digest</A> with XbaI and SpeI and the control with a <A href="/Team:Tuebingen/NotebookProtocols#Gel_electrophoresis" title="Team:Tuebingen/NotebookProtocols">gel electrophoresis</A> the right band of 711bp was visible. The sequencing of the DNA confirmed that part 4 has the expected nucleotide sequence.
A Midi-Prep, restriction digest and preparative gel electrophoresis followed in order to prepare them for later ligation into pRS vectors.
</P><P><A href="/Team:Tuebingen/NotebookProtocols#pGEM_Ligation" title="Team:Tuebingen/NotebookProtocols">Ligation</A> of part 3, 6, 7 and 8 in pGEM vector was performed. Reaction took place over night at 4 °C. The <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformation</A> of these parts was executed into <I>E. coli TOP10</I>.
After growth over night, a mini plasmid preparation was performed. After a colony-PCR with parts 3, 6, 7, 8 failed,  a step backwards to the restriction digest for insert controlling was done. Positive samples were prepared for sequencing. The parts 3 and 8 were sequenced successfully and yielded a good sequence. The ligation of parts 6 and 7 failed, so it was decided to skip part 6, in order to use only Psuc2 as a alternative promotor for luciferase.  
</P><P>2. The synthesized receptor of <I>Xenopus laevis</I> arrived at the laboratory. It was successfully transformed in <I>E. coli TOP10</I> and purified with a Midi-Prep.
</P><H2><SPAN class="mw-headline" id="Week_7_.288.2F20_-_8.2F26.29"> Week 7  (8/20 - 8/26) </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">preparing one of many gel electrophoreses</DIV></DIV></DIV><P>1. The <A href="/Team:Tuebingen/NotebookProtocols#PCR" title="Team:Tuebingen/NotebookProtocols">PCR</A> of the parts 1 and 2 with Herculase polymerase was executed.
The PCR products were checked with an analytical gel afterwards. The PCR of part 1 failed again, so it was decided to reject part 1 and continue working only with luciferase (part 2), because only one reporter gene was needed.
</P><P>2. Because of a lack of luciferase plasmid DNA, the decision was made to <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transform</A> the remaining DNA of luciferase into <I>E. coli TOP10</I>. The transformation was successful. A Midi-Prep yielded new plasmid DNA.
</P><P>3. The receptors (mPR <I>Danio rerio</I>, mPR <I>Xenopus laevis</I>) and mig1 were initially <A href="/Team:Tuebingen/NotebookProtocols#Ligation" title="Team:Tuebingen/NotebookProtocols">ligated into pRS vectors</A> and <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformed</A> into <I>E. coli TOP10</I>. But no colonies grew on the agar-plates.
</P><P>4. A <A href="/Team:Tuebingen/NotebookProtocols#preparative_double_digest" title="Team:Tuebingen/NotebookProtocols">scaled up restriction digest</A> and preparative <A href="/Team:Tuebingen/NotebookProtocols#Gel_electrophoresis" title="Team:Tuebingen/NotebookProtocols">gel electrophoresis</A> of the parts 3 and 8 was executed to prepare them for later ligation into pRS.
</P><H2><SPAN class="mw-headline" id="Week_8_.288.2F27_-_9.2F02.29"> Week 8  (8/27 - 9/02) </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">evaluating sequences via BLAST</DIV></DIV></DIV><P>1. Part 5 was <A href="/Team:Tuebingen/NotebookProtocols#pGEM_Ligation" title="Team:Tuebingen/NotebookProtocols">ligated into pGEM vector</A> and <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformed</A> into <I>E. coli TOP10</I> afterwards. After performing a <A href="/Team:Tuebingen/NotebookProtocols#control_digest" title="Team:Tuebingen/NotebookProtocols">restriction digest</A> it was obvious that the insert did not have the correct length and therefore had to be discarded.
</P><P>2. Second <A href="/Team:Tuebingen/NotebookProtocols#Ligation" title="Team:Tuebingen/NotebookProtocols">ligation</A> of mPR <I>Danio rerio</I>, mPR <I>Xenopus laevis</I> and mig1 into pRS vectors and <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformation</A> into <I>E. coli TOP10</I> was executed. Some colonies grew on the agar-plates. Therefore a mini-prep and a <A href="/Team:Tuebingen/NotebookProtocols#control_digest" title="Team:Tuebingen/NotebookProtocols">restriction digest</A> with XbaI and SpeI with a following <A href="/Team:Tuebingen/NotebookProtocols#Gel_electrophoresis" title="Team:Tuebingen/NotebookProtocols">gel electrophoresis</A> was conducted. But there was only ligation of the insert into pGEM, not into pRS. The hypothesis was that the pGEM constructs were contamination.
</P><P>3. <A href="/Team:Tuebingen/NotebookProtocols#Ligation" title="Team:Tuebingen/NotebookProtocols">Ligation</A> of the parts 3 and 4 in pRS vectors with following transformation into <I>E. coli TOP10</I> was performed. But it was not successful.
</P><H2><SPAN class="mw-headline" id="Week_9_.289.2F03_-_9.2F09.29"> Week 9 (9/03 - 9/09) </SPAN></H2><P>1. <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">Transformations</A> of the vector pSB1C3 with the insert RFP into <I>E. coli TOP10</I>. The cells were plated on agar with different Chloramphenicol concentrations in order to determine the right concentration of the antibiotic.
</P><P><B>Concentration results:</B></P><TABLE class="wikitable"><TBODY><TR><TH> Chloramphenicol concentration </TH><TH> results (after transformation) </TH><TH> results (already selected colonies)
</TH></TR><TR><TD> 30 µg/ml </TD><TD> no growth </TD><TD> viable
</TD></TR><TR><TD> 15 µg/ml </TD><TD> no growth </TD><TD> viable
</TD></TR><TR><TD> 5 µg/ml </TD><TD> up to 30 colonies </TD><TD> viable
</TD></TR><TR><TD> 1 µg/ml </TD><TD> lawn </TD><TD> viable
</TD></TR><TR><TD> 0.1 µg/ml </TD><TD> lawn </TD><TD> viable
</TD></TR></TBODY></TABLE><P>2. Some more unsuccessful <A href="/Team:Tuebingen/NotebookProtocols#Ligation" title="Team:Tuebingen/NotebookProtocols">ligations</A> and <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformations</A> of the parts 3, 4, 8, 9, 10, 11  into pRS vectors. There grew no colonies on the agar-plates or the ligation failed.
</P><H2><SPAN class="mw-headline" id="Week_10_.289.2F10_-_9.2F16.29"> Week 10 (9/10 - 9/16) </SPAN></H2><P>1. After many <A href="/Team:Tuebingen/NotebookProtocols#Ligation" title="Team:Tuebingen/NotebookProtocols">ligations</A> and <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformations</A> of part 5 into <I>E. coli TOP10</I> without any result, it was decided to order new primers for Pfet3, to achieve annealing temperatures closer to each other.
</P><P>New primers were also ordered for Tadh1, because the old primers did not match the yeast-DNA.
</P><P>2. A <A href="/Team:Tuebingen/NotebookProtocols#QIAGEN_Plasmid_Midi_Kit" title="Team:Tuebingen/NotebookProtocols">Midi-Prep</A> of the pSB1C3 vector with a following <A href="/Team:Tuebingen/NotebookProtocols#preparative_double_digest" title="Team:Tuebingen/NotebookProtocols">restriction digest</A> was executed.
</P><P>3. After a lot of unsuccessful <A href="/Team:Tuebingen/NotebookProtocols#Ligation" title="Team:Tuebingen/NotebookProtocols">ligations</A> and <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformations</A> of part 2 in pGEM some colonies grew on the plate. Therefore a <A href="/Team:Tuebingen/NotebookProtocols#Genaxxon_Plasmid_DNA_Purification_Mini_Prep_Kit" title="Team:Tuebingen/NotebookProtocols">Mini-Prep</A> with following <A href="/Team:Tuebingen/NotebookProtocols#control_digest" title="Team:Tuebingen/NotebookProtocols">restriction digest</A> with XbaI and SpeI was performed. After a <A href="/Team:Tuebingen/NotebookProtocols#Gel_electrophoresis" title="Team:Tuebingen/NotebookProtocols">gel electrophoresis</A> the right band of 1650bp was visible.
</P><P>Part 2 was sequenced, but the primer SP6 and T7 did not match the DNA. Perhaps the insert (Part 2) was not in the pGEM vector. To achieve sequencing results custom sequencing primers were designed: The aim was to get a 150bp overlap in the center of the luciferase gene. The two sequencing results reach from the center of the gene to approximately 100bp outside of the gene.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:751px;"><DIV class="thumbcaption">custom sequencing primers</DIV></DIV></DIV></DIV><P>4. <A href="/Team:Tuebingen/NotebookProtocols#Ligation" title="Team:Tuebingen/NotebookProtocols">Ligations</A> of the parts 3, 4, 8, 9, 10, 11 in pSB1C3 and <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformations</A> into <I>E. coli</I>. A lot of colonies grew on the plates.
</P><P>5. The new primers for parts 5 and 6 arrived at the end of the week.
</P><H2><SPAN class="mw-headline" id="Week_11_.289.2F17_-_9.2F23.29"> Week 11 (9/17 - 9/23) </SPAN></H2><P>1. Some <A href="/wiki/index.php?title=NotebookProtocols&amp;action=edit&amp;redlink=1" class="new" title="NotebookProtocols (page does not exist)">Mini-Preps</A> of the parts 3, 4, 8, 9, 10 and 11 in pSB1C3 were executed with a following <A href="/Team:Tuebingen/NotebookProtocols#control_digest" title="Team:Tuebingen/NotebookProtocols">restriction digest</A>. 
But there were still problems with the Chloramphenicol concentration, because the bacterial densitity was very low in the LB medium. Therefore no bands were visible on the gel after the mini-prep.
</P><P>2. A new <A href="/Team:Tuebingen/NotebookProtocols#PCR" title="Team:Tuebingen/NotebookProtocols">gradient PCR</A> of the parts 5 and 7 was performed with the new primers in order to determine the optimal annealing temperature.
</P><TABLE class="wikitable"><TBODY><TR><TH> Position in the thermocycler </TH><TH> gradient </TH><TH> Part name
</TH></TR><TR><TD> A </TD><TD> 60.0 °C </TD></TR><TR><TD> B </TD><TD> 58.9 °C </TD><TD> Tadh1
</TD></TR><TR><TD> C </TD><TD> 57.0 °C </TD><TD> Tadh1
</TD></TR><TR><TD> D </TD><TD> 54.1 °C </TD></TR><TR><TD> E </TD><TD> 50.5 °C </TD><TD> Tadh1
</TD></TR><TR><TD> F </TD><TD> 47.9 °C </TD><TD> Pfet3
</TD></TR><TR><TD> G </TD><TD> 46.0 °C </TD><TD> Pfet3
</TD></TR><TR><TD> H </TD><TD> 45.0 °C </TD><TD> Pfet3
</TD></TR></TBODY></TABLE><P>The <A href="/Team:Tuebingen/NotebookProtocols#PCR" title="Team:Tuebingen/NotebookProtocols">PCR</A> of part 7 was successful. After the purification with the <A href="/Team:Tuebingen/NotebookProtocols#Genaxxon_Plasmid_DNA_Purification_Mini_Prep_Kit" title="Team:Tuebingen/NotebookProtocols">PCR purification kit</A> a <A href="/Team:Tuebingen/NotebookProtocols#pGEM_Ligation" title="Team:Tuebingen/NotebookProtocols">ligation into pGEM vector</A> and <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformation</A> in <I>E. coli TOP10</I> was executed. 
Due to the fact that the time was running out we decided not to continue to work with this parts.
</P><P>3. A new <A href="/Team:Tuebingen/NotebookProtocols#preparative_double_digest" title="Team:Tuebingen/NotebookProtocols">scaled-up restriction digest</A> of the parts 3, 4 and 10 was executed, because the digested stocks of these parts were empty.
</P><P>4. Some more unsuccessful <A href="/Team:Tuebingen/NotebookProtocols#Ligation" title="Team:Tuebingen/NotebookProtocols">ligations</A> and <A href="/Team:Tuebingen/NotebookProtocols#Chemotransformation" title="Team:Tuebingen/NotebookProtocols">transformations</A> of the parts 3, 4, 8, 9, 10, 11 into pRS vectors. There grew no colonies on the agar-plates or the ligation failed.
</P><P>5. The T-shirts for the Jamboree in Amsterdam were designed. Aside, a lot of work on the wiki had to be done.
</P><H2><SPAN class="mw-headline" id="Week_12_.289.2F24_-_9.2F30.29"> Week 12 (9/24 - 9/30) </SPAN></H2><P>1. Some more <A href="/Team:Tuebingen/NotebookProtocols#Genaxxon_Gel_Extraction_Mini_Prep_Kit" title="Team:Tuebingen/NotebookProtocols">mini-preps</A> of the parts 3, 4, 8, 9, 10 and 11 in pSB1C3 were executed with a following <A href="/Team:Tuebingen/NotebookProtocols#control_digest" title="Team:Tuebingen/NotebookProtocols">restriction digest</A>.
But there were still issues with the Chloramphenicol concentration, because the bacterial densitity was very low in the LB medium. Therefore no bands were visible on the gel after the mini-prep.
</P><P>2. Finally the Wiki was completetd. The final design of the poster and the presentation for the Jamboree in Amsterdam were made.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Tuebingen/NotebookReports">http://2012.igem.org/Team:Tuebingen/NotebookReports</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tuebingen/NotebookReports" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tuebingen/NotebookReports" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tuebingen/NotebookReports&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tuebingen/NotebookReports&amp;oldid=210042" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>