"
Page
Discussion
View source
History
teams
Log in
 
Team:TU Delft-Leiden/Project/Life science/curli/integration
From 2014.igem.org
Project
General Overview
A summarizing description of project ELECTRACE
Life Science
Scientific foundations and construction and characterization of BioBricks
Microfluidics
Design and use of microfluidics
Gadget
Working towards a portable biosensor
Parts
New BioBricks submitted to the Registry
Safety
Handling of GMOs and relevant chemicals
Brainstorming
Find out how our iGEM team picked the topic
Notebook
All details on the labwork
Modeling
Modeling Overview
Discussion of what we did and our results
Curli Module
Modeling of the conductive curli module
EET Module
Modeling of the Extracellular Electron Transport module
Landmine Module
Modeling of the landmine detection module
Interaction with Life Science and Microfluidics
Discussion of the interactions we had with the Life Science and Microfluidics departments of our team
Modeling Methods
Discussion of several methods we used in the modeling of the modules
Code Repository
Here you can download the Matlab scripts we made
Policy & Practice
Overview
ELECTRACE
Application, potential, advantages and issues
Case study: Landmines
ELECTRACE applied to the detection of landmines
Interviews with stakeholders
Opinions about ELECTRACE and synthetic biology
Outlook
Synbio issues and possible solutions
Outreach
Interaction with the general public
Achievements
Team & Supporters
Meet our team
Students and advisors
Attributions
Individual contributions
Acknowledgments
Individuals providing help & advice
Sponsors
Faculties, departments, companies etc. who funded our work
Collaborations
Support of other iGEM teams
Media
Public attention (newspapers, journals, interviews) of our project
Integration of Departments
Module Conductive Curli
Context
Integration of Departments
Cloning
Characterization
Described are the general why and how of the practices Modeling as well as Microfluidics with respect to experimental work performed in the Module Conductive Curli. Curli are bacterial protein chains that are formed extracellularly and are attached to bacterial cell walls. These chains play a role in biofilm formation, amongst others. This Module describes the implementation and emergence of Curli in E. coli. In addition, a link with the Module Electron Transport is made by binding conductive gold nanoparticles to the protein chains. Formation of Curli will thus improve conductivity of the medium.
Novel Modeling strategies form basis for in vivo measurements of Curli formation
Refined mathematical standards were implemented within the subdivision Modeling. Summarizing, Curli Modeling covers the generation of Curli utilizing parameters specified with respect to the genome of the single cell and implements algorithms in determination of the emergence of amyloids. The results from this level of the model form the parameters for the next level: plotting the point(s) in time in which cultures of cells operating in a defined space generate the expected quantifiable decrease in resistance. This latter level is modelled via Graph Theory, which is accurately fit to the underlying biological problem. Modeling pointed to a significant change in resistance due to Curli formation. For future prospects, it is quite feasible to generate the data necessary to compare hypotheses derived from models of Curli to actual situations. Sidetrack and useful in a more general sense is the implementation of Percolation Theory , which shows the transition in conductivity at a certain and definable point in time.
Escherichia coli natively carry genes for the production of the amyloid fibrils termed Curli. Experiments were designed centering on strain ΔCsgB. Theory states protein CsgB as the central factor in nucleation of amyloid monomers CsgA. Generating CsgA in a constitutively overexpressed sense and, at a defined point in time, transcribing specified amounts of CsgB will result in nucleation, generation of Curli and a measurable formation of biofilm. Curli monomers carry modified histidine tags in order to bind gold nanoparticles [1, 2, 3]. Experimental characterization thus consists of three complementary aspects, being assaying biofilm formation via eGFP signals constitutively expressed in relevant cells, theory on speed of nucleation and measurements of conductivity, measured via the resistance of the generated biofilm as a result of binding of gold nanoparticles.
Microfluidics are particularly useful for measurements of biofilm formation
Implementation of a system of microfluidics within this Module is based on the aspiration to image biofilm formation accurately and, eventually, couple formation of biofilm to measurements of conductivity. Aspects central in development of this type of device are, amongst others, the speed of reactions, the possibility of in vivo measurements at all points in time and options for quantification of signal of eGFP. Aim, construction and functionality of what is hereafter referred to as the mothermachine are discussed under Microfluidics: MotherMachine . This type of microfluidic system is intended for single-cell experiments, utilizing nanoscale channels coupled to a central channel for flow of media. Team iGEM 2014 TU Delft has made use of the standard model previously utilized in various studies [.].  In general terms, the device is constructed via a positive silicon waver coupled to a plastic mold. The team has constructed its negative with PDMS, generating channels. Size of channels and size of cells should be taken in consideration, the latter partly depending on growth conditions. Holes and tubes coupled to these holes, coupled to the central channel deal with flushing with medium, with respect to the Module Conductive Curli imaging of eGFP. Before application, the PDMS sample is connected to a glass slide by oxygen plasma and prepared for action by consecutive flushing with, amongst others, BSA. More information regarding the design and construction of the mothermachine can be found in the section Microfluidics: MotherMachine .
References
1. Chen et al., Synthesis and patterning of tunable multiscale materials with engineered cells. Nature Materials 13, 515–523 (2014)
2. M. M. Barnhart and M. R. Chapman, Curli Biogenesis and Function. Annu Rev Microbiol. 60, 131–147 (2006)
3. L.S. Robinson et al., Secretion of curli fibre subunits is mediated by the outer membrane-localized CsgG protein. Mol Microbiol. 59(3): 870–881. (2006)
Retrieved from " http://2014.igem.org/Team:TU_Delft-Leiden/Project/Life_science/curli/integration "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
