"
Page
Discussion
View source
History
teams
Log in
 
Team:TU Delft-Leiden/Project/Life science/EET/cloning
From 2014.igem.org
Project
General Overview
A summarizing description of project ELECTRACE
Life Science
Scientific foundations and construction and characterization of BioBricks
Microfluidics
Design and use of microfluidics
Gadget
Working towards a portable biosensor
Parts
New BioBricks submitted to the Registry
Safety
Handling of GMOs and relevant chemicals
Brainstorming
Find out how our iGEM team picked the topic
Notebook
All details on the labwork
Modeling
Modeling Overview
Discussion of what we did and our results
Curli Module
Modeling of the conductive curli module
EET Module
Modeling of the Extracellular Electron Transport module
Landmine Module
Modeling of the landmine detection module
Interaction with Life Science and Microfluidics
Discussion of the interactions we had with the Life Science and Microfluidics departments of our team
Modeling Methods
Discussion of several methods we used in the modeling of the modules
Code Repository
Here you can download the Matlab scripts we made
Policy & Practice
Overview
ELECTRACE
Application, potential, advantages and issues
Case study: Landmines
ELECTRACE applied to the detection of landmines
Interviews with stakeholders
Opinions about ELECTRACE and synthetic biology
Outlook
Synbio issues and possible solutions
Outreach
Interaction with the general public
Achievements
Team & Supporters
Meet our team
Students and advisors
Attributions
Individual contributions
Acknowledgments
Individuals providing help & advice
Sponsors
Faculties, departments, companies etc. who funded our work
Collaborations
Support of other iGEM teams
Media
Public attention (newspapers, journals, interviews) of our project
Module Electron Transport – Cloning
In the wet lab we integrated the Electron Transport pathway of Shewanella oneidensis into Escherichia coli. Here you can find information with respect to cloning of the BioBricks for the Electron Transport pathway.
Module Electron Transport
Context
Integration of Departments
Cloning
mtrCAB and mtrCAB:HIS
ccmAH
Characterization
mtrCAB and mtrCAB-HIS
Our BioBricks BBa_K1316012 and BBa_K1316017 encode the mtrCAB genes under control of a weakened T7 promoter with the lac operator (T7 lacO). In addtion, BBa_K1316017 carries a histidine (HIS) tag at the 5' end of the coding sequence which makes it easier to purify MtrB during characterization experiments. The end constructs are visualized in figure 1 and 2.
Figure 1: BBa_K1316012. Schematic overview of our BioBrick encoding the mtrCAB genes under control of a T7 lacO promoter.
Figure 2: BBa_K1316017. Schematic overview of our BioBrick encoding the mtrCAB genes and HIS tag under control of a T7 lacO promoter.
Cloning Scheme
Making a BioBrick of the mtrCAB operon is quite challenging. First of all, the coding sequence of mtrCAB contains several illegal restrictions sites. Secondly, we need to have the operon under the regulation of a weakened T7 lacO promoter[1]. To get rid of the illegal restriction sites as well as implementing the T7 lacO promoter in our BioBricks BBa_K1316012 and BBa_K1316017 , we ended up having 5 pieces of DNA to be ligated into pSB1C3. To work efficiently, we used the Golden Gate Assembly to clone these BioBricks  (see figure 3).
Figure 3. Golden Gate Assembly of the mtrCAB operon. The BioBricks BBa_K1316012 and BBa_K1316017 encode a weakened T7 lacO promoter (red rectangle) and the coding sequences of mtrC, mtrA and mtrB, indicated with grey arrows. The beam beneath the arrows visualizes the regions and relative sizes of the 5 DNA pieces that were ligated into pSB1C3.
The Golden Gate primers were designed in a way that restriction with the BsaI enzyme resulted in a EcoRI overhang at the 3' end of the DNA and a SpeI overhang at the 5' of the DNA. In this way we were able to clone the DNA fragment into pSB1C3 by using EcoRI and SpeI. To construct the mtrCAB-HIS, the reverse primer of the last DNA part was designed in a way that a HIS tag was introduced at the C-terminus of the MtrB protein. However, there was no additional stop codon included in this primer. Therefore we do not characterize this BioBrick.
For characterization the mtrCAB BioBrick was introduced into pSB3K3. This backbone has a lower copy number than pSB1C3 and is therefore more consistent with the literature [1].
ccmAH
Our BioBrick BBa_K1316011 encodes the cytochrome C maturation genes (ccm) under control of the pFAB640 promoter [1]. The Ccm proteins help to mature the MtrCAB conduit and are therefore needed to facilitate Electron Transport. The end construct is visualized figure 4.
Figure 4: BBa_K1316011. Schematic overview of the ccm genes under control of the pFAB640 promoter.
Cloning Scheme
In order obtain our ccm BioBrick we inserted the pFAB640 promoter in BBa_K917006 , a BioBrick that encodes the ccm gene cluster, constructed by the Edinburgh 2012 iGEM team. A schematic representation of the cloning strategy can be found on figure 5.
Figure 5: Cloning scheme of BBa_K1316011 BioBrick.
References
[1] C.P. Goldbeck, H.M. Jensen et al., “Tuning Promoter Strengths for Improved Synthesis and Function of Electron Conduits in Escherichia coli”, ACS Synth. Biol. 2, 150-159, 2013.
Retrieved from " http://2014.igem.org/Team:TU_Delft-Leiden/Project/Life_science/EET/cloning "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
