"
Page
Discussion
View source
History
teams
Log in
 
Team:TU Delft-Leiden/Project/Life science/curli/cloning
From 2014.igem.org
Project
General Overview
A summarizing description of project ELECTRACE
Life Science
Scientific foundations and construction and characterization of BioBricks
Microfluidics
Design and use of microfluidics
Gadget
Working towards a portable biosensor
Parts
New BioBricks submitted to the Registry
Safety
Handling of GMOs and relevant chemicals
Brainstorming
Find out how our iGEM team picked the topic
Notebook
All details on the labwork
Modeling
Modeling Overview
Discussion of what we did and our results
Curli Module
Modeling of the conductive curli module
EET Module
Modeling of the Extracellular Electron Transport module
Landmine Module
Modeling of the landmine detection module
Interaction with Life Science and Microfluidics
Discussion of the interactions we had with the Life Science and Microfluidics departments of our team
Modeling Methods
Discussion of several methods we used in the modeling of the modules
Code Repository
Here you can download the Matlab scripts we made
Policy & Practice
Overview
ELECTRACE
Application, potential, advantages and issues
Case study: Landmines
ELECTRACE applied to the detection of landmines
Interviews with stakeholders
Opinions about ELECTRACE and synthetic biology
Outlook
Synbio issues and possible solutions
Outreach
Interaction with the general public
Achievements
Team & Supporters
Meet our team
Students and advisors
Attributions
Individual contributions
Acknowledgments
Individuals providing help & advice
Sponsors
Faculties, departments, companies etc. who funded our work
Collaborations
Support of other iGEM teams
Media
Public attention (newspapers, journals, interviews) of our project
Module Conductive Curli – Cloning
In the wet lab, we made constructs containing csgA and csgB, two of the genes involved in curli formation. Here you can find information with respect to cloning of the BioBricks for the Conductive Curli pathway.
Module Conductive Curli
Context
Integration of Departments
Cloning
inducible csgB, constitutive csgA or csgA tagged with HIS
inducible csgB and csgA
constitutively expressed eGFP
Characterization
Inducible csgB, constitutive csgA/ csgA-HisTag
Final constructs
The final BioBricks are BBa_K1316013 and BBa_K1316014 (figures 1 and 2)
Figure 1: Plasmid containing the BBa_K1316013 BioBrick.
Figure 2: Plasmid containing the BBa_K1316014 BioBrick.
Cloning Scheme
For the construction of the final BioBricks BBa_K1316013 and BBa_K1316014 ,    the two curli genes used (csgA and csgB) were obtained from the iGEM BioBrick BBa_K540000 from Lyon team 2011.  In order to introduce the L-Rhamnose promoter upstream csgB and a constitutive Anderson promoter upstream csgA, the Golden Gate Assembly was used.
Golden Gate Assembly
To construct our BioBricks BBa_K1316013 and BBa_K1316014 three pieces of DNA had to be cloned into pSB1C3 . Both BioBricks encode an L-rhamnose promoter and the coding sequences of csgB and csgA or csgA-HIS. To work efficiently, we used the Golden Gate Assembly to construct our BioBricks. In this way we were able to restrict and ligate all the DNA fragments at once (see figures 3 and 4).
Figure 3. Golden Gate Assembly of BBa_K1316013 and BBa_K1316014 . The BioBricks both encode an L-rhamnose promoter (red rectangle) and the coding sequences of csgB and csgA or csgA-HIS, indicated with grey arrows. The beam beneath the arrows visualizes the regions and relative sizes of the three DNA pieces that were ligated into pSB1C3.
Therefore, three PCR fragments were generated, all of them properly designed so that, during the Golden Gate Assembly, the three fragments could only be assembled in the desired order and orientation.
Due to the small size of the constitutive Anderson promoter used ( BBa_J23110 ), the first half of the promoter was included at the 3' ending of the csgB reverse primer; and the second half of the promoter was included at the 5' ending of the csgA forward primer. As a consequence, the promoter would be properly reconstituted during the Assembly.
Note: the construct with CsgA carrying the 6xHisTag (BBa_K1316014) was made the same way as BBa_K1316013 but the reverse primer amplifying csgA was designed in order to incorporate the Tag right before the STOP codon of this gene.
Figure 4: Cloning scheme of the BBa_K1316013 and BBa_K1316014 BioBricks.
Inducible csgB and csgA
Final constructs
The final BioBrick is BBa_K1316015 (figure 5)
Figure 5: Plasmid containing the BBa_K1316015 BioBrick.
Cloning Scheme
For the construction of this BioBrick ( BBa_K1316015 ),  both csgB and csgA genes were PCRed directly from the BBa_K540000 BioBrick from Lyon team 2011 and then placed behind the L-Rhamnose promoter of the iGEM BioBrick BBa_K914003 .
Figure 6: Cloning scheme of the BBa_K1316015 BioBrick.
Constitutive eGFP
Final constructs
The final BioBrick is BBa_K1316016 (figure 7)
Figure 7: Plasmid containing the BBa_K1316016 BioBrick.
Cloning Scheme
The BioBrick ( BBa_K1316016 ) was constructed amplifying the coding region of the Enhanced Green Fluorescent Protein (eGFP) from a comercial plasmid and placing it downstream the constitutive Anderson promoter ( BBa_J23110 ) on its Original iGEM plasmid.
Figure 8: Cloning scheme of the BBa_K1316016 BioBrick.
Retrieved from " http://2014.igem.org/Team:TU_Delft-Leiden/Project/Life_science/curli/cloning "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
