<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Johns_Hopkins-Wetware_humanpractice"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Johns_Hopkins-Wetware/humanpractice">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Johns_Hopkins-Wetware/humanpractice&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins-Wetware/humanpractice&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins-Wetware/humanpractice&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Johns_Hopkins-Wetware/humanpractice" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Johns Hopkins-Wetware/humanpractice</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3></DIV></DIV></DIV><TITLE>JHU iGEM 2012</TITLE><DIV id="global_container"><DIV id="header"><UL id="navbar"><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/Team">team</A></LI><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/Project">projects</A></LI><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/Project">At a Glance</A></LI><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/etohproject">Ethanol control</A></LI><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/etohproject#modelanchor">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/lightproject">Optogenetic control</A></LI></UL><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/etohnotebook">notebooks</A></LI><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/etohnotebook">Ethanol control</A></LI><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/lightnotebook">Optogenetic control</A></LI><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/yeastgoldengate">Yeast Golden Gate</A></LI><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/Parts">Parts</A></LI><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/yeastgoldengate">RFC88</A></LI><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/humanpractice">human practice</A></LI><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/thepartscourselabmanual">Lab Manual</A></LI><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/Safety">safety</A></LI><LI><A href="https://2012.igem.org/Team:Johns_Hopkins-Wetware/requirements">Medal Fulfillment</A></LI></DIV><DIV class="content_container"><DIV class="content"><P>
							iGEM Human Practices has led to a wealth of materials available online for anyone interested in synthetic biology. There are a number of resources including lecture presentations, discussions on the ethics of synthetic biology, and lab resources such as animations or protocols. CommunityBricks has also provided a source of lesson plans and activities. However, these resources have often been used in a one time community outreach program or event. There are efforts to work on standardized set of materials for use in the classroom such as BioBuilder. 
						</P><P>
							This led us to consider how to develop a standard course which could be implemented in any biology lab to teach the basics of synthetic biology. General biology courses often focus on broad concepts and provide a general overview of many fields of biology. We decided the best way would be to produce a lab component which could be integrated into a general biology curriculum. Furthermore, this could provide an educational experience to the college student who does not plan on majoring in the STEM (Science/Technology/Engineering/Mathematics) fields. 
						</P><P>
							We have developed a module which would fit into introductory college labs. There are a number of general biology labs with a variety of labs and activities, but our module was modeled from the JHU general biology lectures and labs. Instructors can use this module and integrate a synthetic biology perspective into their molecular biology lab lessons. The module includes lectures on synthetic biology principles such as abstraction and standardization. Our goal is to introduce the field to a broader range of students, not just those planning to major in a biological science or engineering field. Students will leave with more concrete ideas about what synthetic biology is. 
			</P></DIV><DIV class="content"><FIGURE class="center_align"><FIGCAPTION>
								A flow-chart describing the logic of course design used to guide the work on our course. Published in Klymkowsky, 2011, Getting serious about science education, ASBMB Today. 
							</FIGCAPTION></FIGURE><P>
							With the assistance of advisors who had developed and assisted with the Build-a-Genome course (link to the journal article by Dymond), we began the design process. However, these previous courses have been catered to students with a background in biology or those who had some sort of previous course in molecular biology. Our aims here were difference. Hence, we looked at the literature on science education and found an interesting article from Dr. Klymkowsky who described traditional methods of assessment are not effective ways to check students conceptions and misconceptions. The synthetic biology module was developed using his course design as a guide. 
						</P><P>
							Our goals were to:
							</P><P>
							1. Encourage students to learn the language of synthetic biology.
							</P><P>
							2. Teach basic molecular biology lab techniques with synthetic biology applications.
							</P><P>
							3. Discuss current synthetic biology research.
							</P><P>
							4. Discuss ethical issues in the framework of the project.
							</P><P>
							Examining general biology lab manuals, it seemed the best way to accomplish these goals would be to have students work on a project utilizing the molecular biology lab techniques taught in class. Currently, students often practice these techniques with no goal. This provides instructors an opportunity to show the usefulness of the techniques without overwhelming the students. Doing this will also be a recruitment pitch to curious students to think about pursuing synthetic biology research.
						</P><P>
							With this idea in mind, we established a workflow in which students take a bioparts all the way from PCR amplification to cloning and sequence analysis. We brainstormed what an instructor might require to adapt part of the general biology lab to make bioparts.
						</P></DIV><DIV class="content"><FIGURE class="wrap left"><FIGCAPTION>
								Screenshot of the BioParts Database.
							</FIGCAPTION></FIGURE><P>
							JHU iGEM members had the opportunity to test the class during the summer in order to troubleshoot and optimize protocols. It also served as an opportunity to provide less experienced team members instruction and advice on basic molecular biology techniques. The team also invited a student from Baltimore Polytechnic High School to participate in the trial bioparts course. 
						</P><P>
							We asked a post-doctoral fellow from the Boeke lab to serve as an instructor for the course. The syllabus is here (insert link to pdf). Since part of the assumption in designing the course was that students would also be attending general biology lectures, our instructor provided basic lessons on molecular biology as well as presentations on molecular biology lab techniques. In addition, scientists from the Boeke Lab gave presentations explaining how they would use the parts. 
						</P><P>
							The trial run gave us a chance to modify the protocol and make adjustments to our syllabus, lab manual, and software. Our team also made 900 parts (link to parts page). Our high school student informed us at the end of the course that it was exciting to contribute to an actual research project. Furthermore, this gave him a concrete example of what synthetic biology is.
						</P></DIV><DIV class="content"><P>

							We started our iGEM adventure with human practices. For our human practices, we are implementing an introductory synthetic biology lab course with real world applications to supplement traditional biology lectures. The course is targeted at college freshman and advanced high school students. 4 full day courses represent a synthetic biology unit that can span either a month with weekly labs or less than a week of continuous experiments. The experiments are designed to be modular so that it is easy to pick up at any time and does not require a lot of time create a finished product to contribute to the synthetic parts library. Students will receive DNA primers to amplify parts out of the yeast genomic DNA with signature BsaI overhangs, subclone them into bacteria for sequencing, and finally assemble their parts for testing. The parts we worked on over the summer are listed here: <A href="https://static.igem.org/mediawiki/2012/4/43/JHUiGEMSummary_of_parts_for_Jef.xls"> Parts Spreadsheet</A></P><P><B>Week 1: June 11-15</B>
							For our human practices testing, our iGEM team will be attempting to synthesize 831 parts. The full list of all the parts we synthesized can be found <A href="images/parts.xlsx">here.</A>  Because of the very large number of parts being synthesized, a general overview will be summarized each week with an overall progress report on the parts. 
						</P><P>
							This week we distribute out laboratory supplies and begin work with genomic PCR amplification, ligations, and transformations. We received our primers from IDT for PCR amplification of yeast parts with BsaI overhangs. The primer stock concentration (100µM) are diluted to a final concentration of 5µM. Add 10µl of stock primer to 90µl of DI water giving a concentration of 10µM. Mix 20µl of each forward and reverse primers to get the final concentration of 5µM.
						</P><P>
							Preliminary trials with standard PCR(sPCR). Each member did ~6 gDNA PCR to test that the primers were working. The reactions included a positive control and a negative control. The standard gDNA PCR protocol was used with 30 second extension per kilobase. We found that 10% extra master mix should be made to ensure enough volume for reactions. For example if you have between 1-10 reactions, you would make a master mix for n+1 reactions. 
						</P><TABLE border="1"><TBODY><TR><TD>Reagent</TD><TD>Volume(µl, 1 reaction)</TD><TD>Master Mix(µl, 9 rxns)</TD></TR><TR><TD>Water</TD><TD>15.75</TD><TD>141.75</TD></TR><TR><TD>5x Herculase Buffer</TD><TD>5</TD><TD>45</TD></TR><TR><TD>dNTP(2.5mM)</TD><TD>2.5</TD><TD>22.5</TD></TR><TR><TD>Genomic DNA</TD><TD>0.5</TD><TD>4.5</TD></TR><TR><TD>Herculase polymerase (1:2 dilution)</TD><TD>0.25</TD><TD>2.25</TD></TR></TBODY></TABLE><P>
						The samples ran on 1% agarose gel with a 2-log ladder for 1 hr at 100 volts. 
						</P><P>
						From this point, the workflows of individual students separated. Any deviations made to the standard 
						procedures or lessons learned will be explained below. Some tried to 
						complete all their genomic PCR first, while others started doing transformations and ligations. 
						
						James had very large parts of varying sizes. James split his PCR reactions by size.
						First he finished all parts up to 2kb with 1 minute extension, then 3kb with 90 second extension, and finally
						large parts up to 4.5kb with 132 second extension, and a large 6.5kb part with 200 second extension. As seen
						from the gels, the larger parts are more difficult to amplify than the smaller parts and will require troubleshooting. 
					</P><FIGURE class="center"><FIGCAPTION>All of James parts up to 2kb 
				    	</FIGCAPTION></FIGURE><FIGURE class="center"><FIGCAPTION>All of James parts up to 3kb 
				    	</FIGCAPTION></FIGURE><FIGURE class="center"><FIGCAPTION>All of James parts up to 6.5kb 
				    	</FIGCAPTION></FIGURE><P> 
						Anne Marie had very small parts because all her parts were terminators. Her parts were between 180-220bp. At first it looked like her PCRs had failed since the product looked like primer dimer.						
						Therefore she adjusted to a 1.5% agarose gel and ran it with a <A href="http://www.neb.com/nebecomm/products/productN0474.asp">low molecular weight ladder.</A></P><P>
						Successful gPCR were then ligated into a blunt vector with Ccdb at the insertion site. Ccdb is usually lethal and 
						plasmids without an insertion will not grow on our antibiotic plates. This improves efficiency when screening for the clones with the correct insert. The ligation vector also has multiple cloning site at the insertion point.
						We will be using m13 when amplifying out of the vector. The plasmid is then transformed into e. coli using the 
						standard protocol. The E. coli are plated onto LB plates with Kanamycin to test for uptake of our plasmid. 
					</P><H2> Overall progress </H2><TABLE border="1"><TBODY><TR><TD>Not Started</TD><TD>Finished Genomic PCR</TD><TD>Finished Ligation + Transformation</TD><TD>Finished Colony Screening PCR</TD><TD>Submitted for Sequencing</TD></TR><TR><TD>831</TD><TD>596</TD><TD>36</TD><TD>0</TD><TD>0</TD></TR></TBODY></TABLE><P><B>Week 2: June 18-22</B> This week we continue amplifying parts, troubleshoot ones that failed, and start colony screening PCR.
					</P><P>
					Colony screening PCR selects colonies from our growth plate and attempts to amplify our insert. If the insert is correct if it matches the expected size of the gene plus 200 bases added by the m13 cloning sites. 
					At first, we were having trouble getting our colonies to amplify after they had been grown in reinnoculated in liquid cultures.
					We used a new strain of E.Coli that grew faster than our previous strain. The colonies did not amplify because there was too much starting DNA.
					We came up with 3 methods to remedy this. 
					 1) Reinnoculate the colonies and grow them overnight at 30C instead of 37C. 
					 2) Double-dip PCR. In order to save time and reduce the amount of DNA, a colony picked from an LB plate is first dipped in LB media to innoculate single colonies and then using the same pipette tip is put into the csPCR solution.
					The majority of the cells come out when it is swirled in the liquid LB. We used plastic pipette tips instead of the traditional wooden toothpicks because we found that the toothpicks would absorb our csPCR solution and reduce the reaction volume. 
					Plastic pipette tips did not have this problem and had the added benefit that the liquid taken up by osmosis wcould be expelled with a pipette. 
					 3) Instead of pipetting out cells from the overnight cultures. We would just dip into the cultures instead. The amount of cells taken in by just dipping was sufficient for csPCR. 
					 Some parts had consistently smaller bands than expected. We think this is because of impure gPCR samples. The primer dimer from the gPCR was in higher concentration than the larger parts. This difference in concentration coupled with the fact that smaller inserts are more likely to be taken up during ligation reduce the chance of screening a colony with the correct insert. 
					</P><P>
					PCR troubleshooting
					
					The common methods of troubleshooting PCR were
					 1) Lowing the annealing temperatures. 
					<FIGURE class="limitwidth center"><FIGCAPTION>
					James lowered his annealing temperature to 47C instead of the usual 55C and was able to fix many failed amplifications
					</FIGCAPTION></FIGURE></P><P> 2) Changing the primer concentration
					 Often times there was too much primer dimer and reducing the primer concentration improved yield. Other times the stock primer solutions were not properly mixed and therefore were not at the correct concentration. 
					<FIGURE class="center"><FIGCAPTION>
							Some parts worked with 0.5x concentration that did not work with 1x concentration and vice-versa
						</FIGCAPTION></FIGURE></P><P>
					Ligation and Transformation troubleshooting
					
					Successful ligations results from successful gPCR products. To change poor ligations, we would have to go back and do secondary PCRs to increase the concentration of our desired part. We would also have to adjust the concentration of our DNA fragment if there was too much DNA from gPCR. 
					We found improvements in transformation efficiency or larger parts when we incubated with the ligated plasmids and competent cells longer before heat shocking. We incubated on ice for up to 30 mins instead of the standard 5 mins. 
					We also found improvements by using cold pipette tips. We stored pipette tips for transformation in the freezer and used cold tips from the freezer to improve efficiency. 
					<TABLE border="1"><TBODY><TR><TD>Not Started</TD><TD>Finished Genomic PCR</TD><TD>Finished Ligation + Transformation</TD><TD>Finished Colony Screening PCR</TD><TD>Submitted for Sequencing</TD></TR><TR><TD>831</TD><TD>788</TD><TD>190</TD><TD>118</TD><TD>0</TD></TR></TBODY></TABLE></P><P><B>Week 3: June 25-29</B> This week we continued to fix failed parts and started to submit parts for sequencing
					
					We submitted 2 clones of each part since the parts were amplified from genomic DNA, we do not expect there to be many errors in sequencing. 
					The clones will be sequenced with Pacific Biosciences SMRT Single Molecule Real Time sequencing. 
					SMRT chip sequencing allows sequencing of up to 1,500 clones of different sequences at the same time. 
					We separated our 2 clones into 2 separate plates so that we would not have overlapping clones being sequenced on the same chip.'
					We have had success sequencing 288 clones(3 96-well plates) worth of clones previously. 
					<TABLE border="1"><TBODY><TR><TD>Not Started</TD><TD>Finished Genomic PCR</TD><TD>Finished Ligation + Transformation</TD><TD>Finished Colony Screening PCR</TD><TD>Submitted for Sequencing</TD></TR><TR><TD>831</TD><TD>793</TD><TD>751</TD><TD>743</TD><TD>206</TD></TR></TBODY></TABLE></P></DIV><DIV class="content"><P>
							The team submitted a RFC for a new standard. RFC88 or Yeast Golden Gate: Standardized Assembly of S. Cerevisiae Transcription Units describes the assembly standard which the parts made during the course conform to. RFC88 describes a new standard for the assembly of basic Saccharomyces cerevisiae transcriptional units (TUs) consisting of a promoter/5'untranslated region (UTR), open reading frame (ORF), and 3'UTR/terminator. Note that we use the term &quot;promoter&quot; here to refer to both the promoter and the 5' UTR, which we currently define as a single part. Future iterations of this standard will incorporate subdivision of currently defined parts e.g. into promoter and 5' UTR. The standard makes use of the type IIS restriction enzyme BsaI to generate standardized and user-defined &quot;signature overhangs&quot;, thus enabling directional and seamless TU assembly. RFC88 is supported by the Yeast Standardized Collection of Parts for Expression (SCoPE), a repository of subcloned and sequence verified parts compatible with this assembly standard. The Yeast SCoPE is currently populated by a large number of S. cerevisiae promoters and terminators that facilitate expression and characterization of non-native ORFs. Our vision is that the parts course will allow the academic community access to golden gate parts. A repository of these parts will be maintained at JHU
						</P><P>
							The database will be expanded and used as a tool for the parts repository. It will also allow instructors to choose parts to work on and alert the JHU staff when parts are ready for sequencing. This will also be a resource for instructors. We envision parts used for projects to have a link or description to the project the part will contribute to. This will allow students to see their participation in a scientific endeavor and also provide instructors an opportunity to discuss advanced synthetic biology projects or concepts. 
						</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Johns_Hopkins-Wetware/humanpractice">http://2012.igem.org/Team:Johns_Hopkins-Wetware/humanpractice</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Johns_Hopkins-Wetware/humanpractice" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Johns_Hopkins-Wetware/humanpractice" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Johns_Hopkins-Wetware/humanpractice&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Johns_Hopkins-Wetware/humanpractice&amp;oldid=266157" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>