"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Johns Hopkins-Wetware/etohnotebook
From 2012.igem.org
JHU iGEM 2012
team
projects
At a Glance
Ethanol control
Modeling
Optogenetic control
notebooks
Ethanol control
Optogenetic control
Yeast Golden Gate
Parts
RFC88
human practice
Lab Manual
safety
Medal Fulfillment
7-19-12 Primer dilutions and sPCR
Primer plate arrived in the mail from IDT, all wells at 100 uM concentration. Primers were designed by a Python script, double-checked with Gene Carver. Please email jwang158@jhu.edu if you are interested in either of these tools. All of the information about the promoters and plate layout was compiled into this document and shared with the team.
https://docs.google.com/spreadsheet/ccc?key=0AsvYhi1DlV5mdDktMGczZG1lMnRXeFBYcURPNkFscVE#gid=2
Anne Marie diluted these primers 1:10 by using 10uL from each well of the original IDT plate and 90uL water. This gives us 10uM of each. Then we combined the F and R of each part 1:1 using 20uL F + 20uL R, to get 40uL of each part with 5uM of each primer. Then the following PCR method was carried out:
"Standard PCR (sPCR)" protocol (25uL reaction):
H2O                           15.75 uL
Herculase Buffer (10x)     5.0 uL
dNTP mix                       2.5 uL
Herculase (1:2)              0.25 uL
genomic DNA                  0.5 uL
F&R primer mix               1.0 uL
Total: 25uL
26 PCR reactions were carried out on the 24 parts, where two parts, A3 and A4, were repeated (had extra master-mix):
[1] A1,A2,A3,A4,A5,A6,A7,A8
[2] A9,A10,A11,A12,B1,B2,B3,B4
[3] B5,B6,B7,B8,B9,B10,B11,B12
[4] A3,A4, - , - , - , - , - , -
The gels were loaded in the order above (left to right, top to bottom). All the products are of the correct size.
These parts are ready for ligation and transformation.
Summary:
24/24 EtOH promoter passed sPCR.
7-20-12 Boeke lab meeting notes, Ligations and Transformations
Morning - Boeke Lab meeting. Presented project to the lab and got a lot of useful feedback.
Promoter characterization:
instead of the two times idea, just do a range of ethanol concentrations and grow for 1hr, or however long enough so that the promoters are induced but not too long or else the initial ethanol concentration will change. This method does not require ethanol concentration measurement because the starting ethanol level is known. Use a plate reader.
Ethanol concentration can be measured using a "cheap kit". Look for ways to continuously monitor CO2, more solid information on the feasibility of that. We also might want to monitor how long the cells stay alive, cell viability etc instead of just biomass. We can consider throwing out measuring biomass altogether.
P. pastoris has a very strong methanol-inducible promoters. There have been a lot of studies on this, maybe compare these studies to the ethanol inducible system
Ligation and Transformations were carried out for all 24 promoters A1 - B12.
All ligations were carried out under the standard BioParts procedure. All 24 ligations and the first 12 transformations were done by Scott. The last 12 transformations were done by me.
Summary:
24/24 EtOH promoters passed sPCR, Ligation, Transformations
7-23-12 Colony counts, csPCR, 4mL cultures
Colony counting and screening for B1-B12 (Scott did A1-A12):
A1- 25
A2- 29
A3- 18
A4- 28
A5- 55
A6- 45
A7- 80
A8- 20
A9- 72
A10- 118
A11- 6
A12- 46
B1.) 14
B2.) 25
B3.) 36
B4.) 25
B5.) 36
B6.) 120
B7.) 92
B8.) 7
B9.) 27
B10.) 24
B11.) 3
B12.) 8
csPCR using the double-dip method. Growth plates B2 and C2 got messed up, marked on plate lid - will not use for further experiments.
Boeke Lab work:
Cell culture for mini-prep quantities were done. Take 30uL from 100uL per well from the growth plates to inoculate 4 mL of LB Kan. One clone per part, total of 24 growth tubes. Incubated in a rotating drum incubator at 5:51 PM at 37 degrees C ( there is a chance it is at 30, although the temperature is not too important).
Summary:
24/24 EtOH promoters passed sPCR, Ligation, Transformations, csPCR, cloning
7-24-12 Mini-prep, picking control promoters, one-pot Dig Lig
Cells inoculated last night grew well in general. A11 had trouble, so it will go back into the incubator to be processed with the control promoters. B8 and B7 had low growth, so 1.5mL of culture were taken, spun down, supernatant discarded, and resuspended in the standard 600uL volume (sterile H2O) for mini-prep. For Mini-prep: elution with 30uL Tris buffer was used instead of water. Results below.
To do one-pot digestion ligation (diglig), we are using GFP (already quick-changed to remove internal BsaI sites for Golden Gate assembly), the terminator for MFA2, cloned by Anne Marie, each individual promoter mini-prep product, and the RFP accepter vector pRS405, clone 2. The volumes used from each mini-prep product, as well as the detailed calculations, are here:
https://docs.google.com/spreadsheet/ccc?key=0AsvYhi1DlV5mdGIzVndDZE1pT0c5ZTQzYTBZcXFud0E#gid=0
We used the 25x cycle PCR program from the originally published Golden Gate Assembly protocol. To each mastermix, equal molar of DNA was added (to 100ng backbone), and water was used to top off each reaction to 7.5 uL (Note: originally 15uL, but we scaled it down).
A search for control promoters was performed to serve as constitutive controls during the ethanol-induced promoter characterization phase. The following promoter candidates was chosen from the list of parts synthesized by the team for the parts course:
YBR003W     low copy     COQ1     plate 2, B6
YLR028C      avg copy      ADE16    plate 6, C12
YER015W     low copy     FAA2      plate 2, G4
YPL195W     avg copy      APL5      plate 3, B1
YLR370C      avg copy      ARC18    plate 3, H3
YDL069C      avg copy      CBS1     plate 3, E7
I am picking the last 3 because they are all ready, and the people doing plate 2 and 6 were not available at the time, so I neither had knowledge of whether or not they had already synthesized the parts nor access to their plates. Inoculated 4mL LB Kan cultures with 10 uL from growth plate 3. A suggestion for the course is to add the function to be able to see other people's progress through the bioparts database.
Summary:
23/24 EtOH promoters passed sPCR, Ligation, Transformation, csPCR, cloning, mini-prep, diglig
1/24 EtOH promoters and 3/3 control promoters are at the cloning step.
7-25-12 Mini-prep of slow grower and control promoters, EcoRI digestion of A11, Transformations
Notes on GFP: excitation wavelength is 395 nm, emission wavelength is 508 nm. The excitation is somewhere in the blue-purple range.
Mini-preps were performed for: A11 EtOH promoter, B1_plate3, E7_plate3, H3_plate3, (pDS190, pDS277 - light part vectors, unrelated to the ethanol project)
A11     25.1 ng/uL
B1      229.5
E7      281.3
H3      222.6
pDS190     357.8
pDS277     217.2
A11 had 3uL removed instead of 1 uL due to multiple tests, so there is only 27uL left instead of 29uL. All 260/280 and 260/230 ratios looked good.
Because A11 had such a low concentration, there is suspicion on whether or not it cloned properly. An EcoRI digestion was carried out to determine its integrity.
EcoRI Digestion for A11, 10uL reaction
NEB Buffer 4(10x)     1.0uL
EcoRI-HF                 0.2uL
DNA                        1.5uL
H2O                        7.3uL
Negative control was done with no enzyme, but still with 1.5uL A11
Positive control was done with the vector pRS415 (50 ng/uL)
The order in which the gels were loaded is: - , + , A11. The digestion was successful, all the bands are of the right length. Negative control bands do not follow the ladder because it is circular, so it runs in different conformations and the speed is different from that of linear DNA. The length of A11 is 4022bp including 522bp promoter and 3500bp backbone, pRS415 RFP is 6400bp including 735bp RFP and 5665bp backbone.
Transformations were carried out by Anne Marie on the 23/24 parts that were grown yesterday (7-24-12). 40uL comp cells received from Patrick were mixed with 1 uL of diglig product, incubated on ice for 20 minutes, heat shocked for 45 seconds, recovered with LB medium for 60 minutes before plating on LB Carb agar plates.
Summary:
23/24 EtOH promoters passed sPCR, Ligation, Transformation, csPCR, cloning, mini-prep, diglig, Transformation
1/24 EtOH promoters and 3/3 control promoters passed cloning and have been mini-prepped.
7-26-12 Counted plates, bad acc vec, Dig Lig repeats
On average the transformation plates for 23/24 EtOH promoters had 30 colonies, all but 3 red. It turns out the accepter vector used, pRS405 RFP clone 2, was a bad clone. We repeated the diglig reactions for 23/24 EtOH promoters, A11, and 3/3 control promoters, but this time all products were pre-diluted so a constant volume can be added to each pot.
Positive control:
Promoter - 522bp
Terminator - 222bp
Gene - 690bp
Plasmid - 5900bp
A dig lig reaction was set up using half the original reaction volume published in Engler 2008, 7.5uL from the original 15uL. This is the same protocol as the first dig lig reaction.
(1) - , + , A1,A2,A3,A4,A5,A6
(2) A7,A8,A9,A10,A11,A12,B1,B2
(3) B3,B4,B5,B6,B7,B8,B9,B10
(4) B11,B12,B1_3,E7_3,H3_3
Dilution calculation notes:
m/(Vh2o + Va) = 118.9 and 199 * Va = 118.9 * (Va + Vh2o) #Not used, too complicated for MS Excel.
50ng / 199 ng/uL = .251uL
Xconc * 28 = MINconc * (28 + Vh2o)
Vh2o = (Xconc * 28 - MINconc * 28)/(MINconc)
Summary:
23/24 EtOH promoters passed sPCR, Ligation, Transformation, csPCR, cloning, mini-prep, diglig (failed), Transformation (failed), diglig
1/24 EtOH promoters and 3/3 control promoters passed cloning, mini-prepped, diglig.
Aggregate status:
24/24 EtOH promoters, 3/3 control promoters, - and + promoters are at diglig
7-27-12 Transformations
30 Transformations were performed:
- DNA control (for transformation), - Promoter control, + control, A1-B12,B1_3,E7_3,H3_3
Following the diglig protocol (Engler 2008), but all reactions halved. 2.5 uL of DNA was mixed with 50 uL competent cells. Incubation on ice for 20 minutes was followed by heat shock at 42 degrees C for 45 seconds. Cells were then recovered in 125 uL LB medium for 30 minutes at 37 degrees C. 150 uL of the resulting cells were plated.
Summary:
24/24 EtOH promoters, 3/3 control promoters, +--controls at transformations
7-30-12 HP animations and Wiki graphic design, morning meeting notes
Transformations from Friday were reported to be a success, plan to pick up the plates tomorrow.
Worked on finding animation software for Human Practices animations/lectures. Screened the following programs:
Toon Boom - Huge learning curve, costs money that we don't have
Adobe Flash - same problems as Toon Boom, and if can't convert format will lose support on all apple devices
Pencil 2D: open source, simple
Going to learn Pencil 2D for now since it has the shortest learning curve. Details here: http://www.pencil-animation.org/
Figured out how the code from James works for the wiki. Why are some chunks of the code, like the title, in the individual pages repeated? This makes changing filenames and paths extremely frustrating because a Find and Replace has to be done in every single file. This is going to be a problem because some of the filenames and code are still called "test".
Summary:
All parts are in the right characterization vectors on plates in the Boeke Lab fridge.
7-31-12 New plates csPCR and colony counts
Wiki background idea, gears surrounding the sides. Attempted to do graphics, but this is all that resulted:
This isn't exactly publishable. We are going to have to find another way.
Performed csPCR and cloning on growth plates following standard double-dip protocol, using M13 F & R primers for csPCR.
Scott did the top gel, Margo for the bottom gel. A10 is very faint, but this also occurred in the first assembly, cloning into the invitrogen TOPO blunt vector, csPCR gel reproduced below (from 7-23-12):
So the explanation for why this is the case is unclear, but we do know that this promoter characterization vector assembled correctly from the observed absence of RFP in liquid culture.
Colony counts (Margo):
A1: 544 colonies
A2: 616
A3: 536
A4: 464
A5: 520
A6: 504
A7: 736
A8: 776
A9: 864
A10: 664
A11: 520
A12: 592
Positive Control: 160
-DNA: 0
-Promoter: 18
B1_3: 312
E7_3: 360
H3_3: 320
B1: 520
B2: 872
B3: 688
B4: 448
B5: 608
B6: 488
B7: 280
B8: 144
B9: 248
B10: 344
B11: 280
B12: 336
Summary:
24/24 EtOH promoters and 3/3 control promoters have been constructed into GFP characterization vectors, confirmed by csPCR, and grown on growth plates.
8-1-12 Overnight 4mL culture, Leu2 cutter design
Picked 30uL of the first clone of every part on the growth plate (except A1, which had little growth in the first clone so the second one was used instead) to inoculate 4mL of LB+Amp for parts B4 - B12, the 3 control promoters, and controls; LB+Carb for A1-B3. The cells were left to grow overnight in a rotating drum incubator. The order used is as follows:
A1-A12,B1-B12,B1_3,E7_3,H3_3,+,-Pro
For identifying the correct lengths in yesterday's csPCR, these sizes should be observed:
promoter = mostly 522bp
GFP = 750bp
tMFA2 = 222bp
sum = 1494bp, plus approximately 250bp extra from the stuff in between M13 and the actual construct should be 1750bp
The -promoter control should be around 1230bp, and indeed the gels are of the correct sizes. The sizes were not confirmed yesterday because they were all so similar and successful that the chance of them all failing was too small to consider. Looking back on it, it probably would have been a good idea to double check the sizes on the csPCR gel the day of.
In order to transform these EtOH promoter characterization plasmids into (-leu) yeast by integration, we have to digest each vector with an enzyme that linearizes it by cutting at the leu2 site, and select the resulting colonies on -leu plates. The enzyme that is supposed to be used for this is apparently
EcoRV:
5' GAT | ATC 3'
3' CTA | TAG 5'
So the site "GATATC" was added to the original promoter finder and primer design program so that this extra site can be searched through all 24 parts. The following three promoters were found to include this site:
YPR006C_P, ICL2, A9 at 340
YER103W_P, SSA4, B8 at 112,196
YER150W_P, SPI1, B9 at 207
No EcoRV sites were found within GFP or tMFA2. The expected site was found in the leu2 gene on RFP accepter vector pRS405, but surprisingly another site was found right next to the bacterial origin of replication. A program to find another such enzyme is currently in development to see if it is possible to rescue the above three parts, but completion of the algorithm depends on knowledge of where or not there is supposed to be that second EcoRV site in pRS405.
Summary:
All parts cloned into characterization vectors are now growing overnight in 4 mL cultures.
8-2-12 Mini-prep, 4mL culture re-dos
A9 of the EtOH promoters was pink, though not as red as the negative promoter control. This means that I picked a red colony, grew it up, and without looking inoculated it. Will have to repeat this one again and mini-prep tomorrow, with another clone from the growth plate. Here's an interesting picture:
The left tube is a normal successful assembly, middle tube is A9, and the right tube is the negative promoter control. A9 was not a complete failure because it does make a nice-looking picture. Mini-prepped 3 mL worth of cells from each tube, eluted using 31 uL of tris-buffered water. 1 uL was used for determining concentration and purity of the samples. The following DNA concentrations were measured by Nano-Drop (Thermo):
Summary:
21/24 EtOH promoters and 3/3 control promoters characterization vectors are mini-prepped and ready to go. The other 3/24 EtOH promoters are growing in liquid culture and will be ready for mini-prep tomorrow.
8-3-12 Mini-prep re-dos, plate reader experiments
Mini-prep was carried out for the repeat parts grown yesterday: A4 clone 3, A9 clone 2, B9 clone 3. Used up all 4mL of culture, but when the pellets were resuspended with 550 uL LB medium, it was found that the LB medium used had been contaminated.
Eluted with 31 uL tri-buffered water. These mini-preps will have to be redone.
The plate reader was investigated. At first just RFP strains of E.Coli were put onto the plate, 200uL, in serial dilutions. All information and data are here:
https://docs.google.com/spreadsheet/ccc?key=0AsvYhi1DlV5mdGlETFktN3NHTVZKUXNoRGRXRTN3VUE#gid=0
This data confirms that the plate reader will be able to screen the ethanol promoters in yeast cells. Auto-fluorescence was also measured and determined to have little interference with the desired signal.
Summary:
21/24 EtOH promoters and 3/3 control promoters characterization vectors are mini-prepped and ready to go. The other 3/24 EtOH promoters need to be regrown at a later date.
8-4-12 Leu2 cutter design, preparation for yeast transformation
Before transforming into yeast, promoter characterization vectors need to be linearized at the Leu2 site. The competent yeast strain must include a point mutation or otherwise be deficient in the Leu biosynthetic phenotype but retain most of its genotype. Which restriction enzymes will cut the backbone only in the middle of Leu, and nowhere else? This question means searching through each promoter, GFP, tMFA, and pRS405 RFP accepter vector backbone, which means a lot of manual labor.
Thanks to computers, a Python script will do the trick.
Leu2 cutter - please email jwang158@jhu.edu for the original code
The inputs to the program is a list of enzymes that cut at Leu2 and nowhere else on pRS405, which can be easily found using ApE. The output is as follows:
Python 3.2.2 (default, Sep  4 2011, 09:51:08) [MSC v.1500 32 bit (Intel)] on win32
Type "copyright", "credits" or "license()" for more information.
>>> ================================ RESTART ================================
>>>
#===========================================#
Leu2 Cutter Finder v1.0  By Jerry Wang
#===========================================#
Finding enzymes that only cuts pRS405 at the Leu site and nowhere else on the backbone...
Done!
Searching identified sequences for these restriction sites...
YOR178C  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YER053C  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YDL174C  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [20], 'ACCTGC': [], 'GCAGGT': []}
YML070W  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YCL040W  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YOL052C-A  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YJL052W  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YHL046C  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YBR054W  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YMR170C  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YBR126C  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [226], 'ACCTGC': [], 'GCAGGT': [240]}
YCR012W  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YEL039C  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YDL021W  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YJL200C  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YER150W  conflicts:
{'CCANNNNNNNNNTGG': [110], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YFR053C  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YNL160W  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YPR006C  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [376], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YCR021C  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YER065C  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YMR251W-A  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YER103W  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
YGL062W  conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
Terminator conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
Gene conflicts:
{'CCANNNNNNNNNTGG': [], 'CTTAAG': [], 'GGTNACC': [], 'ACCTGC': [], 'GCAGGT': []}
Note: The index number "1" refers to the "first" nucleotide. It is the same numbering system used by ApE.
>>>
A blank [] indicates that the site was not found in the sequence, and any number within it can be used to see where the site is. After searching these enzymes on NEB, their cut sites and how far they cut into the Leu2 site were determined:
['XcmI', 'CCANNNNN|NNNNTGG']                       539 [cut] 568
['AflII', 'C|TTAAG']                               776 [cut] 331
['BstEII', 'G|GTNACC']        Start of Leu2  5' - 1056 [cut] 51 - 3'    End Leu2
['BspMI', 'ACCTGCNNNNNNNN|']  Start of Leu2   5' - 102 [cut] 1005 - 3'   End Leu2
['BfuAI', 'ACCTGCNNNNNNNN|']  Start of Leu2   5' - 102 [cut] 1005 - 3'   End Leu2
The best-looking candidates here are AflII and XcmI, since they cut a lot closer to the middle than the other ones, and also because most promoters do not have this site. Will need to check if the Boeke Lab has these enzymes. It probably does.
8-6-12 Contamination clean-up, more Leu2 design
The three contaminated samples from last week were determined to be unusable in the name of good science.
EtOH promoter characterization vectors re-do:
A4 clone 3
A9 clone 2
B9 clone 3
30uL from growth plate were used to inoculate 4 mL LB+Carb cultures. Grown in rotating drum overnight at 37 degrees C.
Initially, when pRS405 was first given to me by Leslie, she said to use EcoRV for linearization, but a quick search in ApE revealed that the backbone has two EcoRV sites: one in the expected place in the middle of Leu2, the other just outside of BsaI in the MCS. The latter is hypothesized to be undesireable, although evidence is still being gathered to support this claim. First, the end result is a yeast strain with your desired insert flanked by two functional Leu2 genes, which means that linearizing it at any place outside of Leu2 would be undesirable. Of course, if the digestion were to produce sticky ends, the DNA could come back together. However, this would require an extra ligation step that was not mentioned, and EcoRV leaves blunt ends, so this is not the case.
Summary:
21/24 EtOH induced promoter characterization vectors and 3/3 control promoters are ready for linearization and integrative yeast transformation.
3/14 EtOH promoter vectors are being caught-up and are growing overnight.
8-7-12 Mini-prep, a surprise gift
Cultures from yesterday were mini-prepped and nano-dropped:
ng/uL   260/280   260/230
A4c3   162.5     1.82     2.12
A9c2   382.1     1.83     2.22
B9c3   22.1     1.64     0.57
Patrick surprised me with his own strain PCY245 of Leu- yeast for my transformation. Originally, Leslie informed me that she had a plate of yph500 somewhere in her area in the cold room. I had been going through her plates for half an hour before I was rescued.
6:00 PM - Grow 2 vials of 3 mL each at 30 degrees C in the rotating drum overnight. A single colony was used for inoculation. The plan is to grow 300 mL, which is enough for 30 individual transformations.
Summary:
All parts ready for digestion and integrative yeast transformation.
8-8-12 Digestion, yeast transformation
9:30AM - Inoculate 300mL YPD with 3mL overnight culture. Grown in flask shaker at 30 degrees C.
XcmI will work for every promoter except YER150W, or B9. AflII will be used for B9 instead. This is convenient because B9 also happened to be the one with the lowest concentration (around 20 ng/uL).
Diluted all mini-prepped products to the lowest concentration of 95.4 ng/uL for all products to be used in XcmI digestion. A4,A9 re-dos were used instead of the original low-conc preps. No dilutions were made to B9 for AflII digestion.
Digestion order:
(1) A1,A2,A3,A4,A5,A6,A7,A8
(2) A9,A10,A11,A12,B1,B2,B3,B4
(3) B5,B6,B7,B8,B9,B10,B11,B12
(4) B1_3,E7_3,H3_3
B9 -> AflII at 30 uL
Rest -> XcmI at 20uL
These digestion reactions were carried out using the standard manufacturer's protocols available online through New England Biolabs.
1:30PM - Realized that the thermometer in the incubator was showing 28 degrees C, and indeed someone wrote on the sign that the incubator needs to be set higher for the thermometer to read 30. This probably explains why the OD600 after 4 hours was merely .095.
2:43PM - OD600 = .140
4:24PM - OD600 = .234
5:20PM - OD600 = .320
By my model, the cells will be ready by 700 to 800 min, or 9 - 11 o'clock!
But if 1/3 of starting inoculation is used, it takes 900 - 1050 min to reach mid-log phase. Calculating how much to inoculate:
[(0.25*(1/3))/.290]*300uL = 0.86 mL
However, this model was overturned when an OD600 was found to be .320 at 5:20PM, which the model predicted to be only .290. This means that the cells will reach mid-log phase a lot sooner than anticipated.
At an OD of .616, 250 mL of the culture was harvested, following the protocol "Yeast Transformation" from Meluh Lab, last edited 2-28-2011. The following amendments were made:
pcy245 is not temperature sensitive, so wash step was at 30 degrees C for 30 minutes.
500ng of linearized plasmid DNA was used instead of 1ug
83uL of comp cells were used instead of 100uL
716uL of 44%PEG was used instead of 630uL of 50% PEG
200uL of sterile H2O was used to resuspend cells prior to plating
All cells were plated
Summary:
All characterization vectors have been transformed into yeast. Hopefully they all work.
8-9-12 Mathematical modeling of cyp system
Modeling of ethanol-induced promoter and CYP2E1 control system. Protein kinetics data:
Taken from Hano Tuoru et al, 1998 (available: http://www.sciencedirect.com/science/article/pii/S009167499870140X ). It looks like the expression signal is just a unit step convolved with exponential decay with a time delay. Or in other words,  the transfer function looks like:
H(s) = alpha * Emax * (1/(s+alpha)) * exp(-t0*s)
Where alpha is the expression rate (it means how fast the cell gets to max expression), Emax is the highest expression level. This is easily modeled in Simulink with a simple transfer function block hooked up to a transfer function block. The output however is rate, which is not the only rate affecting overall EtOH in the cell since the yeast is constantly producing more EtOH. The EtOH out (this model is still in progress) minus the effect of the CYP2E1 system will go through an integrator before feeding the ethanol concentration information back to the controller, making the system complete. Once promoters get characterized, the constants for these equations can be figured out, and future behavior can be predicted given user-defined parameters.
Summary:
All parts are still growing in 30 degrees C after yeast transformation
8-10-12 Transformation progress check
Checked on the yeast transformation plates, all of them seem to be working. They all have a large number of colonies, too large to pick single colonies from, except for A9 which had fewer colonies; it was also the one with a different digestion.
-DNA and -Digestion controls were cloudy and showed no colonies. The cloudiness is the result of dead yeast cells, so this experiment went as expected. These cells will need to grow for a bit longer before streaking them out to be able to pick out single colonies.
Modeling work:
The closed loop transfer function was determined today. For detailed derivation, please check the paper notebook. You can email me at jwang158@jhu.edu for a copy.
Y(s) = (gamma*s^2+(alpha*gamma-alpha*E*Kp*exp(-t0*s))*s + alpha*gamma*E*Kp*exp(-t0*s))/(s*(s^3+(alpha-gamma)*s^2+(-alpha*gamma-alpha*E*Kp*exp(-t0*s))*s + alpha*gamma*E*Kp*exp(-t0*s)))
In this case r(t) is the unit step, so R(s) = 1/s , or Hcl(s) = s*Y(s). E is the maximum protein expression level, alpha is the rate of turning on protein expression, gamma is ethanol growth rate, t0 is the protein expression lag time, and Kp is the proportional control constant. These variables can be determined experimentally by the promoter characterization experiment.
Summary:
All parts have been transformed, all are showing beginning signs of colonies.
8-11-12 Streak new plates, mold troubleshooting
Today's colonies look a lot better and show obvious need for more growing space.
The four cases are shown in the picture. In the top left, a partially contaminated plate (total of 2). In the top right, a clean plate (total of 24). In the bottom left, a clean negative control (total of 2). In the bottom right, a clean and comfortable number of colonies (total of 1). These were all streaked onto new SC -Leu plates, even with a streaked negative control.
The existence of contamination is worrying, however three days ago someone else was having the same mold issue in that same incubator on LB Carb plates for E.Coli. It is likely that is the source of the contamination and not something in the yeast transformation process because both -controls worked without a speck of contamination. Hopefully this will not affect future experiments down the road. Streaking new plates should fix this problem.
Close-up of the moldy situation:
Summary:
All parts have finished yeast transformation and have been streaked out onto new colonies.
8-13-12 Grow up characterization strains, digestion ligations
Streaked plates all worked, -streak negative control worked fine. The contamination issue is gone, so the source was probably something in the incubator and not from any reagents or equipment used for the yeast transformation.
SC -Leu media recipe for 500mL:
250mL     2X SC -Leu -His
50mL     20% Dex
1.5mL     100mM His
-             top off with very clean water
Will pick one colony and grow in 10mL of SC -Leu in yellow tubes at 30 degrees C in the drum incubator. Will grow:
-Leu control, -Ura control, PCY245 (in SC -Ura), JOY001, JOY002, JOY003, JOY025, JOY026, JOY027
JOY is what I will be naming this strain of yeast. It stands for Jerry's Optimistic Yeast. JOY is looking forward to his first experiment. Obviously, it's going to work.
CYP2E1 arrived from GenScript a while ago, so we are going to golden gate assemble the promoters with CYP2E1, pRS405 RFP accepter vector, and tMFA2. They sent us 4 ug of DNA, which is plenty. CYP2E1 was diluted to 100ng/uL before starting digestion ligation, following the standard protocol, using a 7.5 uL reaction volume.The tube layout is as follows:
(1) A1,A2,A3,A4,A5,A6,A7,A8
(2) A9,A10,A11,A12,B1,B2,B3,B4
(3) B5,B6,B7,B8,B9,B10,B11,B12
(4) B1.3,E7.3,H3.3,-CONTROL
Summary:
All parts have been re-streaked on new -Leu plates, 6 parts are being grown for initial testing before testing all the others.
8-14-12 Plate reader experiments, colony picking
Grew single colonies from streaked-out plates yesterday at noon, picked up at 8:15 AM today. They smell yeasty and a little alcoholy, but no tube is visibly green, not even the strains with the 3 constitutive promoters. PCY245 grew a lot slower in -URA media compared with the ethanol induced promoter strains growing on -LEU media. This is very interesting because JOY001-027 came directly from PCY245. Both media negative controls were clean.
To prepare ethanol standards from 1 to 16% in 1% increments, 16 x 15mL tubes will be used. First fill each tube half-way with reagent grade water and chill on ice to prevent any added EtOH from evaporating. Uncertainty calculation:
0.150 mL in 15 mL makes 1%
so N% = N * 150 uL and top off to 15 mL with sterile water.
Uncertainty of H2O volume: 15mL +- 0.2 mL (just from looking at the top of the tube with the naked eye)
so 150uL / (15mL +- 0.2mL) = 0.150/15.2 ~ 0.150/14.8 = 0.9868% ~ 1.0135%
But at 16%, this error increases to a range of: 15.7895% ~ 16.2162%
Using 200 proof anhydrous EtOH (99.5% pure). Maybe 1% increment will not be achievable at higher percentages, so switch to 2% increments by around 10%. As a result, 11%,13%, and 15% standards were not made because this method of measurement is too inaccurate at this range.
Cell harvesting (prior to first plate reader experimentation):
Transfer cells (from 10mL SC-Leu overnight) into 1.5mL eppendorf tubes.
Spin @ 3000 x g for 3:00
Decant and resuspend with 25uL of SC -Leu
Add 3uL of cells to 300uL SC -Leu of N% EtOH in each well of a clear 96 well plate.
Columns 5,6 are 025 (ethanol gradient) and media blank. Ran this plate through the plate reader, taking a fluorescence (Ex:480,Em:510) and A600 reading every 15 minutes.
Only 1 in 3 ethanol induced promoters showed a signal and 0 in 3 constitutive promoters showed a signal. This is not good because theoretically they should all work. After referring to Tom Ellis's protocol for promoter characterization, it seems that colony picking might be necessary to find the right clone that responds to ethanol concentrations. Will pick new colonies from both lawn plates and streaked-out plates to screen for EGFP-positive clones.
Initial data looks promising however for the one that did work, JOY002:
Plate 1:
A1-A6 : The control strip. The first two wells are SC -Leu media blanks, second two are PCY245 control, third two are JOY002 positive control from the original overnight culture.
Rest of the plate: JOY001 - JOY015, picked 6 colonies each. First 4 from lawn plate, last 2 from streaked-out plate.
Plate 2:
A1-A6: The same control strip.
Rest of the plate: JOY016 - JOY027, same colonies picked.
All plates had a volume of 200 uL in each well. After these experiments, it was found that the standards made at the beginning of the day were not useful because they had been made with water instead of media. It would also be easier to just do the dilution every time with a multi-channel pipette than to keep track of every concentration of standard.
Summary:
6 colonies from each strain JOY001 - JOY027 have been picked to be screened for the right fluorescent insert.
8-15-12 Colony screening for functional EtOH induced promoters
Inoculate new 200uL plates with 50uL of overnight growth plates so that experimentation can be done on the old growth plates without worrying about recovering the cells after testing with the plate reader. While doing this, also resuspend the settled cells in the overnight plate. To screen colonies for EGFP signal, first take initial A600 and fluorescence readings of both plates, before monitoring over time (3 hours at 3 minute intervals, 30 degrees C, fastest continuous orbital shaking).
Add 30 uL 99.5% EtOH to 150uL remaining cells of the old growth plate (to get 16.66% final EtOH concentration) and immediately run plate reader protocol, one plate at a time. Keep the other plate on the plate shaker so that the cells don't settle.
The first plate screening results came out pretty disappointing, probably because 16% ethanol was too much and the cells were dying. Many A600 curves started high and started dropping exponentially over time.
Use instead of pure EtOH, the 32% EtOH media made by Jake to bring the 150uL of cells per well to 8% EtOH by adding 50uL 32% EtOH in SC -Leu media. Hopefully, this won't kill them. This change was made for plate 2 only.
Jake
Transformations of CYP2E1.
29 transformations: A1-A12,B1-B12,B3.3,E7.3,H3.3, - , transf. -
1.5uL DNA in 30uL comp. cells
125uL LB media
Incubate @20min. on ice
Heat shock for 45s
Recover for 30min.
Plate 150uL
Add 9.6mL ethanol to 20.4mL of SC -Leu media to get 32% ethanol (30mL total)
Scott's coming in tomorrow.
Summary:
Plate 1 (JOY001-015) failed screening due to too high of ethanol concentration. Plate 2 (JOY016-027) worked better, but still not as well as expected.
8-16-12 Continued colony screening, yeast transformation strain troubleshooting
66uL of 32% EtOH in SC -Leu into 200uL makes a final concentration of 8%. Using 65 instead because of limited multi-channel pipette accuracy. Reinoculate plates to a third generation of growth plates and use the second generation plate 1 to do experiments with 8% ethanol instead of the lethal 16%, following the same plate reader protocol.
Screening colonies at 8% EtOH, final verdicts. The parts that are getting thrown out are based on the plate reader screening 6 out of 6 consistently zero fluorescent signals. For plate 2:
X016 - All bad, lower signal than media control
017 - B1 in plate 2 (gen2), max fluorescence signal (RFU): 320 compared with 240 at control.
018 - B9, 380
019 - C5, 800
020 - C7, 2000
X021 - All bad
022 - D7, 6500
023 - E1, 4000
X024 - All bad
025 - F1, 300
026 - All bad
027 - G3, 400
8-17-12 YPH500 yeast transformation preparations, continued plate reader experiments
Inoculated 2 vials of 3 mL YPD each with single colonies of YPH500 (red color). Will use this to repeat integrative yeast transformations of the linearized ethanol induced promoters characterization vectors. The old strain (JOY0XX) came from PCY245, which is a BY strain used for the systematic deletion project, with a designer Leu deletion that makes it impossible for integrative transformation at that location.
Meanwhile, it would still be interesting to see the ethanol-inductive effects of the strongest candidates of JOY0XX, so that future experiments with the correct integration can have something to compare to. JOY024 did not have a detectable signal, so in this case it will serve as the negative control.
Spin down 1.5mL of cells & resuspend in 40uL SC -Leu. Add 5uL remaining to each well.
If using stock 32% EtOH in SC -Leu, the following volumes should be used:
87.5 (of 32% EtOH in SC-Leu) + 107.5 (SC-Leu) + 5 (cells) -> 14% final
75 (of 32% EtOH in SC-Leu) + 120 (SC-Leu) + 5 (cells) -> 12% final
62.5 (of 32% EtOH in SC-Leu) + 132.5 (SC-Leu) + 5 (cells) -> 10% final
.            .                                                                           .
.            .                                                                           .
.            .                                                                           .
0 (of 32% EtOH in SC-Leu) + 195 (SC-Leu) + 5 (cells) -> 0% final
Same plate reader protocol was used, except the total time is now 24 hours and the interval is every 10 minutes.
csPCR results for the CYP2E1 vectors.
Summary:
JOY0XX strain will be unused for a while (and eventually obsolete). A small number of candidates were analyzed for 24 hours for comparison purposes only. JOY1XX will be made with YPH500; the transformation prep was done.
8-18-12 Yeast transformation
From the 20uL (and 30uL for B9) of linearized ethanol induced promoter characterization vectors, take 2.5uL (and 4uL) of DNA per transformation.
12:30PM - Inoculate 150mL of YPD with all 6mL of overnight cultures. Prepare the transformation mastermix for 30 reactions:
PEG (44%) 240uL     [7.2mL]
1M LiOAc 36uL     [1.08mL]
HS DNA 25uL     [0.75mL]
H2O 19uL     [0.57mL]
total: 320uL    [9.6mL]
Add to this mix 50 uL harvested comp cells, which totals 1.5mL in the master mix, bringing the individual sum up to 370uL per reaction.
4:14PM - OD at around 0.4, harvest cells. Resuspend in 1/100 of starting volume, which is 1.5mL if started with 150mL. This makes enough volume for 30 reactions, 50uL of cells in each.
All 27 and -digestion control were transformed including another -DNA control.
Summary:
JOY1XX strains have been transformed, total 29 reactions.
8-20-12 Yeast transformation check
Checked on yeast plates this morning, small colonies begin to show on plates where they are supposed to show. B9 was low, can only count 3 colonies, but still in early growth right now. No red color seen yet, which is weird since these cells started out red.
8-21-12 Troubleshoot transformation
Digest pRS405 using procedure from 8/8/12 to act as + control. Using XcmI for digestion, pRS405 concentration: 199 ng/uL, so 5 uL is enough + 5.48 extra H2O.
Streaked out 6 plates from transformation as a test to see if the really small colonies are indeed the correctly integrated strains. The plates include -DNA, B9, etc. I tried to pick as diversely as possible.
Inoculated 2 x 3 mL cultures of YPH 500 in light YPD media. Also streaked old plate (from June 22) onto a clean YPD. Did 2 unlabeled -controls of YPD only. There is some question as to why the transformations are not working. Will do some digestions on the original vector with suitable controls to investigate the problem.
8-22-12 Analytical yeast transformation
A600 = 0.059 at 11:03 AM, YPH500 strain. Growing cells for a yeast transformation with new plasmid to make sure that the protocol is valid. Digested pRS405RFP (clone1), pRS405RFP (clone 3), pRS405 no RFP with both XcmI and AflII to check integrity of these vectors. Maybe this experiment will shed light on the transformation failures. Digested by 1:20 incubation at 37 degrees C followed by cooling on ice.
Run 5 uL of product + 1 uL H2O + 1 uL 6x dye on a .8% agarose gel at 140V. The thin green 20-well comb works the best because it gives such clear bands.
OD = 0.15 at 3:37 PM
OD = 0.33 at 5:15 PM
OD = 0.58 at 6:20 PM
Carry out the standard yeast transformation protocol. Add 15 uL of digestion product, 1 uL of +control and 15 uL of H2O for -DNA. 5uL of A1.
8:18PM heat shock end
8:26 ~ 8:36PM incubate with CaCl2
Spin down and take 100 uL, plate the rest.
8-24-12 Yeast transformation problem resolved
Results of the yeast transformation was quite conclusive. There was no integration going on inside the cell with either AflII or XcmI digestion.
The idea of constructing a Golden Gate to Bio Brick standard conversion acceptor vector was brought up today. This sounds like an excellent idea for high-throughput biobricking.
The following CYP2E1 golden gate assemblies needed. New colonies picked:
B1_3, A2, A4, A5, A7, A8, B7, B10
Mini-prep the rest, including B9 from GFP ethanol-induced promoter characterization vectors.
After meeting with Pam, it was determined that the strain we were using had too big of a deletion in its Leu2 domain. We will instead use a strain that has a Leu2 - 3, 112 deletion, which is a point and frameshift mutation. Before, we were using a strain that didn't have the correct homology with the ends of where XcmI and AflII were cutting. W303A could be a possible strain to use.
8-29-12 Pick colonies, high-throughput yeast transformations
Picked new clones off the CYP vectors plate to be re-grown; A7 grew well so 40 uL was used to inoculate a new tube.
Drip transformation had some plates where the drips collided, so digesting more DNA just in case the experiment turns out to be a failure, also inoculating more yeast cells for another transformation.
90 minute digestion following the standard XcmI digestion method. Added 28 uL of B9, which is what was left.
8-30-12 Mini-preps
Mini-prepped the following clones:
A5 158.5 ng/uL
A4 196.6 ng/uL
B9(GFP) 233.6 ng/uL
A8 335.3 ng/uL
A7 446.8 ng/uL
8-31-12 Yeast transformation
Missing CYP2E1 vector preps from:
A2, B7, B10, B1_3
Re-inoculate those in LB + Carb
The Best Yeast Transformation Method. Doing this with the strain BY362
PEG 3500 50% 250 uL
LiOAC 1M  36 uL
HS DNA   50 uL
cells   100 uL
That's 426 uL per tube. H2O + plasmid: 34 uL
9-1-12 Linearize CYP2E1 vectors
Diluted miniprep products of CYP2E1 vectors(with exception of A2, B7, B10, B1_3) to final concentrations of 135.64 ng/uL
Digestion w/XcmI & AflII following previous protocol (from 8-8-12). 1000 ng of DNA per digestion.
(1) A1, A3-A9
(2) A10, A11, A12, B1, B2, B3, B4, B5
(3) B6, B8, - , B11, B12, E7_3, H3_3, B9
1:30 at 37 degrees C, 0:20 at 65 degrees C, infinity at 4 degrees C.
9-3-12 Yeast transformation results
Yeast tranformations with the strain BY362 turned out to be great! All the colonies are pink. Colony counts:
A1. 748, A2. 196, A3. 372, A4. 296, A5. 184, A6. 212, A7. 208, A8. 132, A9. 412, A10. 384, A11. 436, A12. 244
B1. 396, B2. 404, B3. 556, B4. 92, B5. 536, B6. 420, B7. 104, B8. 384, B9. 2, B10. 312, B11. 84, B12. 172
B1_3. 144, E7_3. 108, H3_3. 144, 415. 12, 405. 35, 405RFP. 204, -Digestion. 0, -DNA. 0
Pick 3 colonies of each, grow overnight at 30 degrees C in a 96-well plate at 200 uL SC -Leu media per well. Streak out 2 colonies. A note about naming. BY362 -> integrative transformation -> EY001 ~ EY027. Called "Ethanol Yeast".
A2, B10 of CYP vectors didn't grow out, need to re-do csPCR. Inoculated the other 2 for mini-prep.
Integrative yeast transformation for CYP2E1 linearized vectors following the same protocol as before, same strains. First 6 tubes might have had cross contamination - did not change tips when aspirating the supernatant with the vacuum.
9-4-12 H-T plate reader promoter screening with EY001-EY027
Growth plates look good, take out 50 uL from each well (200 uL to begin with) and add to 155 uL in new plate.
Put 50 uL 32% EtOH in SC -Leu to remaining 150 uL of each well to reach a final concentration of 8% ethanol in every well. Incubate at 30 degrees C, shake at medium, and monitor OD600 as well as GFP expression once every 15 minutes for 24 hours.
9-5-12 Choose final promoter colonies
Plate reader screening final decisions. Priority was given to the colonies which showed quickest induction, then the level of induction was taken into consideration - the higher the better. The first coordinate refers to the promoter. The second coordinate refers to the location on the actual growth plate
A1- A1  ;A12- C12 ;B11- F9
A2- A6  ;B1- D3  ;B12- F12
A3- A9  ;B2- D4  ;B1.3- G1
A4- A11  ;B3- D8  ;E7.3- G5
A5- B1  ;B4- D10  ;H3.3- G8
A6- B4  ;B5- E3  ;405- G12
A7- B8  ;B6- E6  ;405RFP- H1
A8- B11  ;B7- E9  ;415- H4
A9- C1  ;B8- E12
A10- C4  ;B9- F1
A11- C9  ;B10- F4
9-6-12 Promoter characterization
Promoter characterization experiment:
- Inoculate screened and passed colony into 3 mL of SC -Leu. Grow overnight at 30 degrees C until saturated.
- Harvest 1.5 mL in ep tubes @ 3000 x g, 3-4 minutes.
- Decant & resuspend with 30 uL SC -Leu
Now follow the same procedure detailed previously. Summary: serial dilution of a 32% ethanol standard. Note: extra cells were added in E9 on this run
SC -Leu formula: 2x SC -Leu -His + 1.5 mL His + 50 mL 20% Dextrose
9-11-12 Characterization media troubleshooting
Make new media in 15 mL tubes, one for each tested concentration if ethanol. 45 uL of His per 15 mL media. Begin with 2x SC -Leu -His so that the 14% ethanol standard has the same amount of nutrients as the 0% standard. Add 150 uL ADE back to the media to each tube, on top of 15 mL. Total 8 tubes were made, 0 - 14%. Promoters characterized with this set of media were resuspended with 30 uL SC -Leu media before added to the standards media.
Plans regarding an RT-PCR experiment to prove mRNA existence of CYP2E1 were made. Primers were designed and ordered.
9-14-12 Dot plate experiment
Re-inoculate the first set of promoters to be characterized: A1-A10. 4mL SC -Leu.
Plate reader data for A11, A12, B1-B8 all looked good. Did not inoculate.
Need also inoculate: B9, B10, B11, B12, B1_3, E7_3, H3_3. And in addition to all of this, -DNA, pRS405 control
CYP drip plate experiments. Make two sets of rectangular SC -Leu Agar + 8% ethanol and SC -Leu Agar plates.
Serial dilution (x4 times) were done for the 23 CYP constructs + 415 control. These cells came from colonies streaked out plates from the original yeast transformation in the Leu2 - 3, 112 mutation-containing BY361 strain. Some of these dilutions had colonies where half were small and the other half weren't. Next time, we need to streak out better and pick SINGLE colonies. There may be some kind of a revertant here.
9-15-12 Final plate reader experiments, colony PCR
Plate reader for A1-A10 using new protocol. Using the repeat-pipettor to add 200 uL of each ethanol standard into each well.
- Spin down cells @ 3000 x g for 4 minutes.
- Resuspend with 0% media (30 uL). Add 5 uL to each well via multi-channel pipette.
Parameters for the plate reader:
- Control Temp at 30 degrees C
- Read A at 600nm
- Read fluorescence at 480nm, 510nm (Ex\Em) -Best wavelengths for EGFP
- Shake
- Repeat reads every 10 minutes for 24 hours
Colony PCR with RT-PCR primers:
Make 5 mL diluted NaOH. Multichannel into PCR strips (15 uL in each). Negative control for this experiment will be pRS415 control strain. Positive control will be purified CYP2E1 plasmid (1 ng). Don't put the positive control in NaOH. Pick a colony and mush it around in the NaOH solution. Boil cells for 5 minutes @ 95-100 degrees C. Cool down for more than 10 minutes @ 4 degrees C. Make a standard GoTaq PCR master mix, 12.5 uL reaction volume.
Before starting, diluted IDT primers to 100 uM first, then took out only a little for a 10 uM final concentration. The following gel shows that all the colonies picked had the correct insert in them.
9-20-12 CYP2E1 fermentation experiment plan
Made 12% ethanol plates for replicating the failed 8% dot plate experiment. Formula: 1.5 mL His, 50 mL 20% Dextrose, 250 mL 2x SC -Leu -His, 250 mL 4% Agar. Divide the resulting mix in two and add 37.82 mL of either water or ethanol to each half to get a final concentration of 12% by volume. The water is to make sure that the ethanol plates have the same amount of nutrients as the SC -Leu plates.
After meeting with Shige, we have decided on the optimal conditions for fermentation. 10% dextrose YPD starting media. Collect samples every hour. Start with an OD600 of 0.5 so that it will finish by 24 hours. Use 35 mL working volume in a 100 mL flask, foil cover. Incubate at 30 degrees C, 120 rpm at 40mm displacement. The strains to be used for this experiment are A4, A10, B4, and pRS414 control strain. Inoculate for each, 2 x 10 mL cultures at 30 degrees until saturated aerobically before starting fermentation.
Plate reader experiment data have been analyzed.
The distribution of these promoters in terms of the lowest % EtOH required to induce these promoters.
9-22-12 CYP2E1 fermentation begins
8:08 AM - 30 mL YPD 10% dextrose in 125 mL flask. Add in 5 mL resuspended saturated overnight cells. At first, the OD was:
A4, A10, B4, 415 --- OD: 0.697, 0.579, 0.583, 0.724
Then, media was carefully added so that every flask could have around the same starting OD. The experiment started at roughly 9 AM
A4, A10, B4, 415 --- OD: 0.559, 0.551, 0.543, 0.577
Samples were collected according to this protocol: Dilution factor for ethanol kits = 50
- Prepare collection plate. 96-well round bottom
- For the first 4 samples, add 196L water to the first 4 wells, then fill with 4 uL of sample supernatant. Seal with foil and freeze.
- For each new set of 4 samples, remove foil, add samples, and foil again before freezing. #Edit: Instead of wasting foil, we decided to use ep tubes.
- For every hour, take 100 uL out and read OD on the plate reader. Then put that sample in an ep tube, spin down at max speed for 1 min, and take only the top 4 uL and place into collection tubes (of 196uL water).
Fermentation experiment diary: [.559 .551 .543 .577; .494 .476 .485 .467; .451 .557 .523 .500; .500 .532 .536 .516; .649 .674 .712 .704; .763 .713 .768 .732; .829 .911 .773 .824; .790 .928 .774 .778; .944, .859, .973, .961; 1.027, 1.068, 1.026, 1.033; 1.083, 1.130, 1.113, 1.044; 1.146, 1.185, 1.189, 1.181; 1.173, 1.221, 1.295, 1.228; 1.160, 1.272, 1.398, 1.291; 1.265, 1.385, 1.324, 1.342; 1.353, 1.440, 1.389, 1.401; 1.407, 1.475, 1.389, 1.443; 1.452, 1.469, 1.422, 1.484; 1.398, 1.495, 1.424, 1.523; 1.462, 1.501, 1.514, 1.601; 1.454, 1.546, 1.401, 1.499; 1.538, 1.602, 1.411, 1.630; 1.574, 1.626, 1.458, 1.591; 1.606, 1.651, 1.518, 1.612; 1.606, 1.675, 1.470, 1.615;]  Time(Jake): 4:06 pm, 5:04 pm, 6:04 pm, 7:04 pm, 8:02 pm,  Scott: 9:04pm, 10:03pm, 11:00pm, 12:00am, 1:01am, 2:00am, 3:00am, 4:03am(computer restarted for windows update.Left everything default except 600wavelength), 5:01am, 6:06am, 7:03am, 8:03am, 8:57am
This is what the data looks like graphically.
9-25-12 CYP2E1 fermentation ethanol data collection
Take out time points 2 and 23 to make 4 * 2 = 8 wells of free space. Perfect size to run the standard curve. A serial dilution was carried out starting from 0.1% ethanol, 0.05, 0.025, 0.0125, 0.00625, 0.003125, 0.0015625. At a dilution ratio of 50, the lowest detection limit of this curve would be .078125%. When making the standard, special water (100 uL YPD 10% Dex + 5 mL H2O) was used to account for the fluorescence of the media. The samples were tested for ethanol content following the kit's instructions (AAT Bioquest). A linear relationship was found between log(concentration) and log(fluorescence), R-squared = 0.99136, n = 1.
The standard curve. The x axis is log(concentration) and the y axis is log(fluorescence).
The data, after calculating the concentration from the standard curve.
Retrieved from " http://2012.igem.org/Team:Johns_Hopkins-Wetware/etohnotebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
