"
Page
Discussion
View source
History
teams
Log in
 
Team:Johns Hopkins-Wetware/lightnotebook
From 2012.igem.org
JHU iGEM 2012
team
projects
At a Glance
Ethanol control
Modeling
Optogenetic control
notebooks
Ethanol control
Optogenetic control
Yeast Golden Gate
Parts
RFC88
human practice
Lab Manual
safety
Medal Fulfillment
8/14/2012
Restriction digest of acceptor vector backbones
50µL pTZ19u SmaI digest
Component
Volume (µl)
10x NEBuffer 4
5
pTZ19u (~500ng/µL)
1
100x BSA
.5
SmaI (20U/µL)
1
H20 to 50µL
42.5
1 hour digest at 25°C.
50µL SalI/BamHI digest of pRS415/416-IIs
Component
Volume (µL)
10x NEBuffer 3
5
pRS415/416 (~70ng/µL)
5
100x BSA
.5
SalI (20U/µL)
1
BamHI (20U/µL)
1
H20 to 50µL
37.5
1 hour digest at 37°C.
Gel extraction of assembly parts
Gel Extraction of: linearized pTZ19u, LUAV.LOVpep, LUAV.URA3; pRS416-IIs backbone, CTAV.pPGK1, CTAV.RFP, CTAV.mCherry, CTAV.LOVpep; pRS415-IIs backbone, NTAV.pTDH3, NTAV.ePDZb, NTAV.GFP, NTAV.RFP. Eluted in 10µL water.
Nanodropped all parts:
Component
Concentration (ng/µL)
Dilution
pTZ19u
11.8
LUAV.LOVpep
38
1:2
LUAV.URA3
11.5
pRS415-IIs
11.5
NTAV.pTDH3
48.6
1:4
NTAV.ePDZb
20.7
1:2
NTAV.GFP
37.5
1:2
NTAV.RFP
56.6
1:3
pRS416-IIs
11.4
CTAV.pPGK1
21.1
1:2
CTAV.RFP
34.9
1:2
CTAV.mCherry
37.2
1:3
CTAV.LOVpep
38.4
1:3
Gibson assembly
Combined DNA below with 15µL 1.33x Gibson assembly master mix on ice
for LUAV: 3µL pTZ19u, .5µL LOVpep, 1.5µL URA3
for NTAV: 3.5µL pRS415, .4µL pTDH3, .5µL ePDZb, .35µL GFP, .45µL RFP
for CTAV: 3.5µL pRS416, .35µL pPGK1, .4µL RFP, .4µL mCherry, .4µL LOVpep
also pTZ19u, pRS415, pRS416 only
incubated in thermal cycler at 50°C for 30min
transformed 2µL into 50µL competent cells, plated everything on LB-carb along with no DNA control
Introducing two mutations into both SWE1.R1 AND SWE1.R2
Because these two light parts (first 1200 bp in SWE1.R1 and first 900 bp in SWE1.R2) both require removal of two internal BsaI sites so as to ensure proper digestion and ligation in the Golden Gate Assembly method, we are introducing the mutations with primers in these parts, which are cloned to the 3600-bp Kanamycin vector pCR-Blunt II-TOPO. Because there are two, and not one of these sites, we can proceed with a method that is more sure of success than QuickChange: creating two overlapping fragments that each contain the mutation, chewing back with an exonuclease to expose 3' overhangs, and ligating the mutated fragments together to yield circular, mutated plasmids.
Today, we build two fragments for each light part, each of which contain both mutations.
8.14.12 notebook spread
Table 1 of the above attachment lists the PCR reactions used. Two concentrations of templates are used in order to ensure appearance of a good product. Table 2 shows the amounts of reagents used, and how I made the master mix.
Reactions 1, 2, 5, 6 had 30" extension periods, to accommodate for the short fragment size. Reactions 3, 4, 7, 8 had 5' extension periods, to accommodate for the long fragment size. I used the manual-recommended extension temperature for <10 kb fragments of 72°C. I also chose one annealing temperature of 55°C for all, because I did not expect a problem with specificity of binding (Tm's of primers were 79.6°C and 84.4°C).
Top of page
8/15/2012
Colony PCR of 8/14 gibson assembly transformations
colony counts
LUAV- 23
CTAV- 2
NTAV- 0
pTZ19u only- 0
pRS416 only- 2
pRS415 only- 0
no DNA- 0
Looks like something didn't work.
screening for LUAV using M13 FWD, T7
screening for CTAV using M13 FWD, M13 REV
Top of page
8/16/2012
Gel purification of Gibson fragments and some TEL1 parts
Running 1% TTE agarose gel with 10 µl EtBr. 10 µl of 2-log ladder were put on either side of the samples. The first 8 samples are site-directed-mutagenized fragments of SWE1.R1 and SWE.R2. The following and last 2 samples are TEL1.R1 and TEL.F2, which are very long parts which have been difficult to ligate into the pCR-Blunt II-TOPO vectors. The manual for pCR-Blunt II-TOPO recommended gel purification for large and difficult inserts.
All bands displayed the appropriate lengths.
Identity
Task
Expected  length (bp)
1
SWE1.R1 fraction
Gibson frag
~300
2
SWE1.R1 fraction
Gibson frag
~300
3
SWE1.R1 fraction
Gibson frag
~4500
4
SWE1.R1 fraction
Gibson frag
~4500
5
SWE1.R2 fraction
Gibson frag
~300
6
SWE1.R2 fraction
Gibson frag
~300
7
SWE1.R2 fraction
Gibson frag
~4200
8
SWE1.R2 fraction
Gibson frag
~4200
9
TEL1.R1
Improve blunt ligation by gel purification
~6000
10
TEL1.F2
Improve blunt ligation by gel purification
~5400
All samples gel-extracted using Gel DNA Recovery Kit.
Pairs of equivalent fragments were combined.
Top of page
8/17/2012
James
Digested more pTZ19u, pRS415/16 no IIs, gel extracted
Gibson attempt 2
Jerry
sPCR following standard 25 uL reaction for the new A1/A2 Reverse primer for amplifying ePDZ, used for GPF fusion by yeast transformation. The last primer didn't work, so hopefully the re-designed primer amplifies correctly. Annealing temp: 50 degrees C, extension time 1:25.
Ligation of Gibson parts to finalize SWE1.R1 and SWE.R2 inserts in KanR acceptor vectors
Each pair of equivalent gel purifications (1-8) (made on 8.16.12) were combined into tubes 1, 3, 5, 7.
Using a pre-made 15 µl Gibson master mix (detail mix if necessary?), the following reactions were made:
Rxn #
Vol. of ea. DNA fragment (µl)
Insert name
Vol. H2O (µl)
1
1
SWE1.R1
3
2
2
SWE1.R1
1
3
1
SWE1.R2
3
4
2
SWE1.R2
1
5
1 (only one DNA fragment added)
SWE1.R2 backbone fragment with KanR gene
4
The fragments are referred to by their assigned number as given when they were first PCR'd on 8.14.12. See 8.14.12 notebook spread.xlsx for a refresher.
Ligation 5 is a negative control. If the fragment circularizes alone, it will survive through the transformation and demonstrate background after transformation. Water was added to total 5 µl of DNA-H2O, and to total 20 µl of Gibson rxn.  The point of varying concentrations is to toggle parameters and better the results.
Top of page
8/20/2012
Re-amplification of acceptor vector parts
Repeating PCR of light system acceptor vector parts in order to repeat gibson assembly again.
Parts: LUAV.LOVpep, LUAV.URA3, CTAV.pPGK1, CTAV.RFP, CTAV.mCherry, CTAV.LOVpep, NTAV.pTDH3, NTAV.ePDZb, NTAV.GFP, NTAV.RFP
Made new primer mixes for each part. 90µL water with 5µL each 100µM forward and reverse primers (5µM each final).
50µL Herculase reaction
Component
Volume (µL, one reaction)
x11
Master Mix (µL)
5x Herculase Buffer
10
110
2.5 mM ea. dNTPS
5
55
1:2 Herculase
.5
5.5
H20 to 46µL
30.5
335.5
Total
46
506
Added 2µL of each template and 2µL of primer mix.
PCR program: 3min @ 95°C, 30x (30s @ 95°C, 30s @ 53°C, 45s @ 72°C), 3min @ 72°C, hold at 4°C
LUAV.LOVpep and CTAV.LOVpep did not amplify at all. Since both LOVpep parts failed, I suspect something went wrong with the template (pDS277).
Made new pDS277 dilution (1:100 from stock, approximately 2ng/µL). Diluted CTAV.pPGK1 PCR product 1:10 to use as template in secondary PCR since it did not amplify strongly.
PCRs of LUAV.LOVpep using old and new templates, CTAV.LOVpep using old and new templates, CTAV.pPGK1 using gDNA and PCR product as template.
Component
Volume (µL, one reaction)
x7
Master Mix (µL)
5x Herculase buffer
10
70
2.5 mM ea. dNTPs
5
35
1:2 Herculase
.5
3.5
H20 to 46µL
30.5
213.5
Total
46
322
2µL template, 2µL primer mix
PCR program: 3min @ 95°C, 30x (30s @ 95°C, 30s @ 50°C, 30s @ 72°C), 3min @ 72°C, hold at 4°C
Weak amplification for LOVpep parts, do a secondary PCR.
Diluted LOVpep PCR products 1:2 for template. 50µL Herculase reaction, gradient annealing 50/48/46°C.
Component
Volume (µL, one reaction)
x7
Master Mix (µL)
5x Herculase buffer
10
70
2.5 mM ea. dNTP
5
35
1:2 Herculase
.5
3.5
H20 to 46µL
30.5
213.5
Total
46
322
2µL of template to each, 2µL of 5µM ea. primer mix to each.
PCR program: 3min @ 95°C, 30x (30s @ 95°C, 30s @ 50/48/46°C, 30s @ 72°C), 3min @ 72°C, hold at 4°C
Good product from all reactions.
Amplification of ePDZb::KAN
Problem: new reverse primer for ePDZb::KAN has Tm ~58°C, while forward primer has Tm ~50°C. Try multiple conditions to see if we can get it to work.
25µL Herculase reactions, 3 reactions mixes at 4 PCR programs.
Reaction mixes: standard, half primer, half template
PCR programs: annealing 55/51/47°C, or touchdown: 5 cycles 55°, 5 cycles 51°C, 20 cycles 47°C
3 min @ 95°C, 30x (30s @ 95°C, 30s @ T annealing, 2min @ 72°C), 3 min @ 72°C, hold at 4°C
Got a correct length products for integration into GFP-tagged yeast strains.
Top of page
8/21/2012
Gibson assembly of acceptor vectors
Miniprep IIs-less pRS415 RFP and pRS416 RFP with zymo kit. Resuspended 1.5mL culture in 600µL water.
DNA concentrations from nanodrop:
Miniprep
dsDNA concentration (ng/µL)
415-1
238
415-2
253
416-1
321
416-2
36
Digest 415/416 minipreps with BsaI to check that RFP drops out, digest 415/416 with SalI/BamHI for acceptor vector Gibson assembly, digest pTZ19u with SmaI for Gibson assembly.
25µL BsaI digests of pRS415/416 RFP no IIs
Component
Volume (µL)
10x NEBuffer 4
2.5
100x BSA
.25
plasmid
1
BsaI-HF (20u/µl)
.5
H20 to 25
20.75
Total
25
50µL SalI/BamHI digests of pRS415/416 RFP no IIs
Component
Volume (µL)
10x NEBuffer 3
5
100x BSA
.5
plasmid
5
SalI (20U/µL)
1
BamHI (20U/µL)
1
H20 to 50
37.5
Total
50
50µL SmaI digest of pTZ19u
Component
Volume (µL)
10x NEBuffer 4
5
100x BSA
.5
plasmid
5
SmaI (20U/µL)
1
H20 to 50µL
38.5
Total
50
BsaI and SalI/BamHI digests: 1 hour at 37°C. SmaI digest: 1 hour at 25°C.
Gel extraction of acceptor vector components with Zymo kit. Eluted in 30µL water.
Verification gel of all parts for gibson assembly (2µL of each):
Gibson assembly of acceptor vectors: 15µL Gibson master mix with:
LUAV: 1.5µL pTZ19u, 1.5µL LOVpep, 1µL URA3, 1µL water
NTAV: .8µL pRS415, 1.6µL pTDH3, .8µL ePDZb, .8µL GFP, .8µL RFP, .2µL water
CTAV: .5µL pRS416, .8µL pPGK1, .8µL RFP, .8µL mCherry, 1.6µL LOVpep, .5µL water
pTZ19u backbone only: 1.5µL pTZ19u, 3.5µL water
pRS415 backbone only: .8µL pRS415, 4.2µL water
pRS416 backbone only: .5µL pRS416, 4.5µl water
30min at 50°C, transformed 2µL into 50µL competent E. coli along with .2ng pRS416 positive control and no DNA negative control.
Top of page
8/22/2012
Integrative transformation of ePDZb::KAN into GFP yeast
Inoculated 10mL YPD with 100µL overnight cultures of GFP-tagged yeast strains: TUB1, CDC14, CLF1, NET1, APC11, SPC97, KAP95, NIC96, MID2, CDC15. Grew up to OD600 ~.6 in 30°C drum. Spilled MID2 and CDC15, reinoculated 2mL in 8mL YPD, grew up to OD600 ~.5.
Yeast Transformation of ePDZb::KAN PCR products, along with 2µl .1ng/µL pRS416 transformation control and no DNA control.
Plated 50µL and 350µL on YPD, pRS416 control on SC-ura, put in 30°C incubator.
Inoculate overnight cultures of acceptor vector gibson assemblies
Colony counts:
Construct
White colonies
Red colonies
LOVpep::URA3 vector
165
0
N-Terminal Acceptor Vector
0
120
C-Terminal Acceptor Vector
11
21
pTZ19u backbone only
13
0
pRS415-IIs backbone only
0
0
pRS416-IIs backbone only
1
0
pRS416 +ctrl (.2ng)
thousands
0
No DNA
3
0
Inoculated 4 clones each of LUAV, NTAV, CTAV into 5mL LB+carb for miniprep and digestion for verification.
csPCR of SWE1.R1 and SWE1.R2; then test digest to check removal of BsaI sites
12 clones from each plate (the first two being SWE1.R1, the second two being SWE1.R2), cloned in 100 µl LB+kan and run in csPCR using standard protocol. In the 96-well growth plate, SWE1.R1 is in A1- B12, and SWE.R2 is in C1-D12.
csPCR products were run on a 150 µl 1% TTE gel with 7.5 µl EtBR. 5 µl 2-log Ladder. 5 µl of each csPCR sample. The following image was obtained:
Top of page
8/23/2012
Verification of acceptor vector constructs
Miniprep overnight cultures of LUAV, NTAV, CTAV. NTAV and CTAV clones are pink, which is good. Zymo kit, resuspended 1.5mL culture in 600µL water, eluted in 30µL water.
Nanodrop concentrations:
Miniprep
dsDNA concentration (ng/µL)
LUAV-1
459.4
LUAV-2
351.8
LUAV-3
232.0
LUAV-4
360.5
NTAV-1
364.5
NTAV-2
279.5
NTAV-3
265.0
NTAV-4
302.4
CTAV-1
243.4
CTAV-2
244.8
CTAV-3
224.8
CTAV-4
253.6
Restriction digests for verification:
25µL EcoRI/XbaI digest of LUAV
Component
Volume (µL)
10x NEBuffer 4
2.5
100x BSA
.25
plasmid (~230-450ng/µL)
1
EcoRI-HF (20U/µL)
.5
XbaI (20U/µL)
.5
H20 to 25
20.25
Total
25
25µL BsaI digest of NTAV/CTAV
Component
Volume (µL)
10x NEBuffer 4
2.5
100x BSA
.25
plasmid (~250-300ng/µL)
1
BsaI-HF (20u/µL)
.5
H20 to 25
20.75
Total
25
Both sets of digests 1 hour at 37°C.
Run gel of cut/uncut vectors:
Looks goot! 3 good LUAV clones (1,2,4), all 4 NTAV clones good, 3 good CTAV clones (1,2,3). Made 15% glycerol stocks of good clones.
Replica plated yesterday's integrations of ePDZb::KAN into GFP strains from YPD onto YPD+G418 plates.
Leslie inoculated -leu -ura yeast to transform NTAV and CTAV tomorrow, and mCherry strains for LOVpep::URA3 transformation after LOVpep::URA3 transformation tomorrow.
Introducing mutations with CHK1.F2 by producing Gibson fragments
CHK1.F2 requires two mutations before assembly. One removes the BsaI digestion site, the other to mutate a functional residue. As we did with SWE1.R1 and SWE1.R2, we can introduce the mutations without QuikChange. Instead, we amplify fragments of each vector using non-complementary forward-reverse pairs of the pertinent primers, and then Gibson digest-ligate the fragments together. This process has a higher rate of success and is a bit simpler than sequential QuikChange reactions.
The set-up of the PCR reactions is detailed in the following spreadsheet:
8.23.12 chk1mutagenesis spread
We set up four reactions: two reactions per fragment. The pairs of reactions toggled the amount of template added. This reaction does not require miniprepped DNA, so we added 10 µl of transformed E. coli in LB+kanamycin at toggled dilutions, as noted in the spreadsheet.
The thermocycler settings were set to account for these facts:
-55°C is the default annealing temp, especially when you don't expect too much non-specific binding, and when the annealing temps of the primer don't necessarily agree
-Pfu Ultra prefers 72°C
-The fragments that were 640 bp long required about 1 minute extension time. The fragments that were >4500 bp long required 4.5 minutes extension time.
Top of page
8/24/2012
PCR of LOVpep::URA3, yeast transformation of LOVpep::URA3 into mCherry strains, yeast transformation of NTAV and CTAV
Inoculated mCherry strains and BY4741 into 10mL YPD (from 100µL of saturated overnight).
Diluted LUAV.2 miniprep 1:10 for PCR template (final ~35ng/µL).
8x 50µL Phusion reaction
Component
Volume (µL, one reaction)
x8.5
Master Mix (µL)
5x Phusion buffer
10
85
2.5mM dNTP
4
34
LOVpep::URA3 FWD primer (10µM)
2
17
LOVpep::URA3 REV primer (10µM)
2
17
LUAV template (~35ng/µL)
1
8.5
Phusion polymerase
.5
4.25
H20 to 50
30.5
259.25
Total
50
425
PCR program: 30s @ 98°C, 30x (10s @ 98°C, 20s @ 55°C, 60s @ 72°C), 5min @ 72°C, hold at 4°C
Ran 5µL each reaction:
Product looks great! Will proceed with transformation into mCherry-tagged yeast.
Made plates for NTAV, CTAV transformation. Spread 75µL His, 250µL Lys, 300µL Met on SC-8 plates.
Yeast transformation of LOVpep::URA3 into mCherry strains (plated on SC-ura), NTAV (SC-leu), CTAV (SC-ura), NTAV+CTAV (plates described above), pRS416 transformation control (SC-ura), and no DNA negative controls of BY4741 on SC-ura, SC-leu, and the -leu -ura plates above.
Minipreps of light parts to prepare for QuikChange
The following parts (inserted into pCR-Blunt II-TOPO KanR vectors) required one mutation each, so they were already cultured in 5 ml LB+kan o/n at 37°C and miniprepped in preparation for QuikChange:
CHK1.F1
CHK1.R1
FAR1.F1
FAR1.F2
FAR1.R1
FAR1.R2
CDC15.R1
CDC15.R2
CDC15.R3
Gel purification of Gibson fragments of CHK1.F2 and subsequent Gibson digestion/ligation
Amplified fragments 1-4 from 8.23.12 (precursors to the Gibson reaction) were run on a gel and gel-purified. (They are in order, left-to-right on the following gel.)
1 and 2 were combined into a new 1, and 3 and 4 were combined into a new 2. The Gibson reaction that followed used a pre-made 15 µl Gibson rxn tube. This mix is intended to be combined with 5 µl of H2O/DNA. I mixed them as follows:
Fragment 1
Fragment 1 vol (µl)
Fragment 2
Fragment 2 vol (µl)
Vol. H2O (µl)
A
1
1
2
1
3
B
1
2
2
2
1
C
2
1
--
--
4
C is a control. The fragment in it contains the KanR gene, so when transformed and plated, it should give us an idea of how well A&B digested/ligated.
Top of page
8/25/2012
QuikChange PCR of some CHK1, FAR1 and CDC15 parts
To remove BsaI sites and change some functional residues, QuikChange reactions were performed on all necessary light parts. Each part only needed ONE site of mutagenesis, which is why amplification followed by Gibson digestion/ligation, as has been done previously, was not used.
See the following document for a list of the parts used, and the contents of each reaction. The DNA templates are all minipreps from 2012_8_24. Two of each part is initially set up, one primer for each in a pair. They are later combined in a process that will be described below. The negative control was set up in double because we did not have to combine two reactions, since there are no primers.
8.25.12.lightparts.quikchange
Table 1 describes the contents of each 25 µl reaction in a general manner. Table 2 details the variables of each reaction.
After each tube is set up, with the negative control totaling 50 µl, they were put through the following PCR scheme
95°C, 5'
3 cycles:
95°C, 30"
55°C, 30"
72°C, 5'
72°C, 10'
4°C, hold
Afterward, Each reaction was combined with the other reaction that contained a primer complimentary to the one it contained. This of course did not apply to the neg ctrl. Each rxn now had a total of 50 µl. Another round of PCR was carried out, identical except that the cycle was repeated 18x instead of 3x.
There were a few issues in the execution of this protocol. Firstly, the amounts of DNA template added were of such small volume that I do not trust they were pipetted correctly. If I were to do this again, I would use dilutions of the minipreps. Secondly, the master-mix came up very short: while I set up for 21 rxns, I only got a little more than 16 rxns. However, I set up new reagents correctly for 17, 18 and neg ctrl. When the QuikChange products are finally transformed, we will see if something truly went wrong.
Top of page
8/26/2012
Mutagenized CHK1.F2 transformed into E.coli
The mutagenized CHK1.F2 is the product of the amplified, gel-purified and Gibson digest-ligate product. It was last handled on 2012_8_24. The products were named A, B and C. A and B should yield theoretically identical products, but they were ligated using differing amounts of DNA. C was a control that only had one of the two necessary DNA amplicons. It contains the KanR gene, so if it somehow circularizes then that will show in the transformation. Theoretically no colonies should grow from C.
A, B and C were transformed into TOP10 competent cells using standard protocol and plated to LB+Kanamycin. The plates were put in a 30°C incubator for o/n growth.
Top of page
8/27/2012
QuikChange of parts (cont'd)
DpnI digest of all QuikChange parts (including the negative control) from 2012_8_25 by adding .5 µl DpnI to each 50 µl QuikChange rxn. The digest was incubated at 37°C for 3 hrs. A PCR clean-up (from a kit) was performed on each product following the digestion. They were then transformed to competent cells by standard protocol and grown o/n on LB+kanamycin.
Inoculating lots of yeast
ePDZb::KAN integrations into GFP tagged yeast strains did not grow. Inoculate all mCherry strains in 5mL YPD for transformation repeat tomorrow.
NTAV and CTAV acceptor vectors successfully rescued -leu and -ura yeast, respectively. The NTAV+CTAV co-transformation also rescued -leu -ura yeast. Inoculated one colony each of NTAV, CTAV, and NTAV+CTAV in 5mL YPD to check fluorescence tomorrow.
LOVpep::URA3 integrations into mCherry tagged yeast strains grew. Struck out 8 clones of each strain on SC-ura plates to colony purify.
Top of page
8/28/2012
Repeat yeast transformation of ePDZb::KAN into GFP strains
Inoculated TUB1, CDC14, CLF1, NET1, APC11, SPC97, KAP95, NIC96, MID2, CDC15 GFP yeast into 10mL YPD.
Plate based yeast transformation along with Jerry's transformations for the ethanol project. Tried dripping 10µL and ~80µL volumes, but had lots of problems dripping the cells onto the plates (lanes running together, pooling, etc.).
Check fluorescence of NTAV, CTAV, few GFP::ePDZb clones
Re-inoculated NTAV, CTAV, NTAV+CTAV, 3 ePDZb clones into SD media from overnights (100µL into 5mL), to 30°C drum.
Leslie saw GFP and mCherry fluorescence in the NTAV+CTAV double transformant, and took this picture of Mid2 ePDZb GFP (Mid2 localizes to the cell membrane, so expect a ring of GFP around the cell):
QuikChange of parts (cont'd)
QuikChanged light parts on agar plates (plated on 8.27.12) are grown up at this point. The negative control came up completely blank. 7 of 9 experimental parts had 100+ colonies. 2 had under 15 colonies each, but had enough for cloning and confirmation (which parts were they?)
4 colonies from each plate (excluding negative control) were picked to each inoculate a 5 mL liquid LB+kan culture. Incubated o/n at 37°C.
Top of page
8/29/2012
Check fluorescence of NTAV, CTAV, NTAV+CTAV, 3 ePDZb GFP strains, LOVpep integrations
Re-inoculated NTAV, CTAV, NTAV+CTAV, and ePDZb transformants into 5mL SD. Inoculated 8 LOVpep integrations into 5mL SD from colony purifications.
Verify overwriting of markers from integrative transformations
Patched Net1.4, Nic96.1, Whi3.7, Ssl1, Mid2.1, Rpn1.1, Sik1.1, Nic96.2 LOVpep-mCherry strains on a YPD+G418 plate. LOVpep::URA3 should have overwritten the Kan resistance downstream of mCherry and there should be no growth.
Spotted Mid2, Spc97, and Cdc15 ePDZb-GFP strains on SC-his plate. ePDZb::KAN should have overwritten the HIS marker downstream of GFP and there should be no growth.
Prepare to repeat ePDZb::KAN integrations into GFP strains
Inoculated GFP strains into 5mL YPD in preparation for repeating transformation tomorrow, since the plate based yeast transformation probably did not work for these integrative transformations.
QuikChange of parts (cont'd): organization and test digests
All picked/cultured clones of QC'd parts were labeled uniquely with the part name and one of the letters A, B, C and D. They were then miniprepped and test digested with BsaI using the miniprepped DNA. Each digestion contained the following:
1 µl 10X NEB4 buffer
1 µl DNA
.5 µl BsaI HF
.1 µl 100X BSA
7.4 µl H2O
A no-BsaI control reaction was prepared for each clone. Each reaction was incubated for an hour and immediately run on a 1% TTE gel with 5 µl EtBr. 1 µl of the reaction was combined with 5 µl 1.2X Loading Dye and run.
See the spreadsheet for the detailed list of the reactions, and how each clone was judged (true + vs. false +):
Reaction Spreadsheet
In the following gel image, the samples are listed on the gel in the same order as they are in the spreadsheet. 1-24 in the first row, 25-48 in the second row:
Because it turns out that CDC15.R1 was never successfully cloned into the Kan vector, the clones from its QC reaction were disregarded. Clones from other QC'd parts that passed were sent in for sequencing.
Top of page
8/30/2012
Repeat transformation of ePDZb::KAN into GFP tagged yeast
Yeast Transformation of ePDZb::KAN PCR product into cell cycle GFP tagged yeast. Spun down cells and resuspended in 50µL .1M LiOAc after LiOAc wash. Plated two volumes on YPD: 50µL and ~250µL (the remainder), along with .5ng pRS416 on SC-ura and no DNA on YPD controls.
Test KAN knockout of LOVpep::URA3 transformants
Patch out all 8 clones of each transformant onto SC-URA to remove background. Replica plate onto G418 tomorrow to verify overwriting of KAN.
Top of page
8/31/2012
Submit split genes with functional mutations for sequencing
Minipreps from 4 clones each of CHK1.C1, FAR1.N1, FAR1.N2 sent to GeneWiz for sequencing.
Replica plate ePDZb::KAN transformants, LOVpep::URA3 transformants
Replica plate ePDZb::KAN transformants from YPD onto YPD+G418. YPD plate have a lawn of cells for both volumes plated.
Replica plate LOVpep::URA3 transformants from SC-ura onto YPD+G418. Expect no growth on G418 for clones with correct integrations.
Amplify BsaI RFP, pSB1C3 backbone
BsaI RFP to be used as template for RFP in the 3 golden gate to biobrick acceptor vectors. pSB1C3 to be used as the backbone for these vectors.
Resuspended primers to 100µM, made 5µM each primer mixes.
50µL Phusion:
Component
Volume (µL, one reaction)
5x Phusion HF buffer
10
dNTPs 2.5mM ea.
4
FWD/REV primer mix (5µM ea.)
4
template DNA (~10-30ng)
1
Phusion polymerase
.5
H20 to 50µL
30.5
Total
50
PCR program: 30s @ 98°C, 30x (10s @ 98°C, 20s @55°C, 60s @ 72°C), 5 min @ 72°C, hold at 4°C.
Both parts look correct.
Inoculate split gene e. coli for minipreps
Inoculated WHI5.C, CHK1.C2 A-D, SWE1.N1 A-D, SWE1.N2 A-D, CDC15.C1, CDC15.C2, CDC15.C3, CHK1.N2, WHI3.N1, WHI3.C1, WHI3.N2, WHI3.C2 in 5mL LB+kan.
Top of page
9/01/2012
James- miniprep and test BsaI digest of split gene light parts
Miniprepped split gene plasmids with Zymo kit (resuspended 1.5mL culture in 600µL water).
10µL BsaI verification digest
Component
Volume (µL, one reaction)
x23
Master Mix (µL)
10x NEBuffer 4
1
23
100x BSA
.1
2.3
plasmid (~156-764 ng total)
2
BsaI-HF (20u/µL)
.5
11.5
H20 to 10
6.4
147.2
Total
10
184
One hour digest at 50°C in thermal cycler.
All digests look correct except maybe SWE1.N1.B (extra band), WHI3.C1 (extra band), WHI3.N2 (no cut). Choose new clones of WHI3.C1 and WHI3.N2 to test.
GBAV construction
DNA cleanup of RFP BsaI (Zymo kit, resuspended in 30µL water). 45.7ng/µL, diluted 1:3 to ~15ng/µL for PCR.
Amplification of promoter, ORF, and terminator overhang RFPs from RFP BsaI for Gibson assembly of golden gate to biobrick acceptor vectors.
50µL Phusion
Component
Volume
5x Phusion HF buffer
10
dNTPs 2.5mM ea.
4
FWD/REV primer mix (5µM ea.)
4
template (~15ng/µL)
1
Phusion polymerase
.5
H20 to 50µL
30.5
Total
50
PCR program: 30s @ 98°C, 30x (10s @ 98°C, 20s @ 55°C, 40s @ 72°C), 5min @ 72°C, hold at 4°C
Don't know what the band at ~250bp is, but it shouldn't affect the assembly after gel extraction.
Ran pSB1C3 backbone alone with 3 RFP parts on .7% agarose gel and gel purified with Zymo gel purification kit (eluted in 30µL water).
Ran 1µL of each product as verification before Gibson assembly, also nanodropped each part.
Part
Concentration (ng/µL)
pSB1C3 backbone
44.9
PRO RFP
23.1
ORF RFP
26.4
TT RFP
24.7
260/280 ratios were high for some reason.
Proceeded with Gibson assembly of vectors:
backbone only: 3µL pSB1C3, 2µ H20
PRO GBAV: 3µL pSB1C3, 3µL PRO RFP (accidentally added 1µL extra of backbone)
ORF GBAV: 2µL pSB1C3, 3µL ORF RFP
TT GBAV: 2µL pSB1C3, 3µL TT RFP
50°C 30min in thermal cycler
spread chloramphenicol on plain LB
Transformed 2µL of Gibson reaction into 50µL chemically competent e. coli.
Plated 200µL on LB+chloro, to 37°C incubator.
inoculate clones of remaining split gene parts
inoculated WHI5.N, SWE1.C1, SWE1.C2, WHI3.C1, WHI3.N2 into 5mL LB+kan, 37°C drum overnight. For miniprep tomorrow and test BsaI digest.
Top of page
9/02/2012
streak out clones of ePDZb::KAN transformations
Integrative transformation was successful; lots of colonies for each strain. Approximate transformation efficiency: 944 colonies/ (5 ng * (50 µL cells / ~300 cells total)) * (1000ng/1µg) = ~1.13x10^6 cfu/µg. Very rough calculation.
Picked 8 transformants from each strain and struck out on YPD+G418 to remove background. Will pick singles from these plates and patch out on G418 and replica plate to SC-his to verify overwriting of HIS3 with ePDZb::KAN.
miniprep and test digest of remaining split gene parts
Miniprep of WHI5.N, SWE1.C1, SWE1.C2, WHI3.C1, WHI3.N2 using Zymo kit (resuspended 3mL culture in 600µL water, eluted in 30µL water). Growth of WHI3.N2 was noticeably less than the other strains.
Concentrations from nanodrop:
Part
Concentration (ng/µL)
WHI5.N
187.3
SWE1.C1
284.3
SWE1.C2
197.8
WHI3.C1
132.7
WHI3.N2
161.6
10µl BsaI test digests to check that part drops out of vector correctly
Component
Volume (µL, one reaction)
x5.5
Master Mix (µL)
10x NEBuffer 4
1
5.5
100x BSA
.1
.55
plasmid
2
BsaI-HF (20U/µL)
.5
2.75
H20 to 10
6.4
35.2
Total
10
44
1 hour 50°C digest in thermal cycler. Ran gel of digest next to uncut vector.
All parts look like they cut correctly.
Made glycerol stocks of all good parts from today and yesterday, and also spotted them out on an LB+kan plate.
GBAV construction
Transformations from yesterdays Gibsons were largely unsuccessful. There may be a single red colony for the ORF RFP.
Spread chloramphenicol from Leslie's stock on plain LB to make new LB+chloramphenicol plates.
Repeated the Gibson assembly:
backbone only: 2µL pSB1C3, 3µL H20
PRO GBAV: 2µL pSB1C3, 3µL PRO RFP
ORF GBAV: 2µL pSB1C3, 3µL ORF RFP
TT GBAV: 2µL pSB1C3, 3µL TT RFP
50°C 30min
Transformed 2µL each reaction into 50µL chemically competent E, coli, along with ~2ng circular pSB1C3-RFP positive control and no DNA negative control. Plated 200µL on LB+chloramphenicol, to 37°C incubator.
Top of page
9/03/2012
GBAV construction continued
2 red colonies for pro-AV, 1 red colony for ORF-AV, 0 colonies for TT-AV from 9/1 transformation. Inoculated the 3 red colonies into 5 mL LB+chloramphenicol
No colonies visible yet on 9/2 transformations (~12 hours of growth).
One-pot digestion-ligation of split gene light parts
List of parts: In NTAV: WHI5.N, CHK1.N1-A, CHK1.N1-B, CHK1.N1-C, CHK1.N1-D, CHK1.N2, CDC15.N2-B, CDC15.N3-A, SWE1.N1-C, SWE1.N2, WHI3.N1, WHI3.N2, FAR1.N1-B, FAR1.N2-B. In CTAV: WHI5.C, CHK1.C1-B, CHK1.C2-A, CDC15.C2, CDC15.C3, SWE1.C1, SWE1.C2, WHI3.C1, WHI3.C2, FAR1.C1-B, FAR1.C2-A.
Transformed 2µL of dig-lig into 30µL competent e. coli, plus pRS416 and no DNA controls.
Top of page
9/04/2012
split gene dig-lig status
No colonies on any plates, including positive control. Will redo with different competent cells.
BsaI test digest of PROAV and ORFAV clones
10µL BsaI digest
RFP drops out of all acceptor vectors as expected. Made freezer stocks of these 3 clones.
Inoculated 2 more proAV clones and 3 more orfAV clones from second tranformation into 5mL LB+chloramphenicol.
verification of ePDZb integrations
Patched out the 8 clones of each transformation onto G418 for replica plating onto SC-his to confirm overwrite of HIS3 by KAN marker.
Top of page
9/05/2012
repeat transformations
Transformations for 25 dig-lig parts, plus pRS416 and no DNA controls on carbenicillin. TTAV Gibson assembly transformation, plus pSB1C3 and no DNA controls on chloramphenicol. One transformation for CYP2E1.
verification of more biobrick acceptor vector clones
Zymo miniprep and 10µL BsaI digest of ProAV-3, OrfAV-2,3,4.
RFP drops out of all clones as expected. Made freezer stocks of all 4 clones.
Top of page
9/06/2012
Inoculating cultures and mating haploid strains
Continuing with JC's Golden Gate Assemblies into NTAV and CTAV, and his TTAV's; and JW's CYP2E1 (all in BioBrick acceptor vectors);
From JC's agar growth plates, none of the CTAV's appeared to work, and are the subject of later troubleshooting.
One white colony from each NTAV plate was picked and used to innoculate 5 ml LB+carb. We need to select for vectors whose RFP gene has been replaced, so red colonies are avoided.
4 red colonies from the TTAV plate were picked and each was used to innoculate 5 ml LB+chloramphenicol.
One colony from the CYP2E1 plate was picked and used to innoculate 5 ml LB+carb
We mated haploid yeast strains as well:
Mid2 x everything
Cdc14-Net1
Rrp14-Ssf1
Rpt1-Rpn1
Cdc18-Whi3
Clf2-Mid2
Kap95-Mid2
Cdc15-Mid2
Tub1-Mid2
Top of page
9/07/2012
Minipreps of NTAV's, TTAV and CYP2E1, and selecting mated strains
All liquid o/n cultures (innoculated on 2012_9_6) of NTAV's, TTAV and CYP2E1 were miniprepped.
Digests of assembled acceptor vectors
Recently miniprepped NTAV's were double-digested with BlpI and BamHI with the following reagents:
.5 µl BlpI
.5 µl BamHI-HF
2 µl  DNA
1 µl 10X NEB4
6 µl H2O
Incubated at 37°C for 1 hr.
Miniprepped TTAV was digested with the following reagents:
.5 µl BsaI-HF
2 µl DNA
1 µl 10X NEB4
.1 µl BSA
6.4 µl H2O
Incubated at 37°C for 1 hr.
All subjects had control digests that replaced restriction enzyme with water.
See entry on 2012_9_8 for a diagnostic gel on all the digests and their controls.
Top of page
9/08/2012
Gel of 2012_9_7 NTAV dig-lig verification digests, TTAV verification digests
NTAV dig-lig results:
4 TTAV clones drop out RFP as expected. Most NTAV dig-ligs cut as expected, however some are not visible on the gel. Need to redo digest for CHK1.N2, SWE1.N1-C, SWE1.N2, WHI3.N2.
Troubleshooting CTAV construction
PCR verification using primers used for amplification of CTAV parts. Attempt to amplify pPGK1, RFP, mCherry, and LOVpep to verify individual components. Also amplify pPGK1+RFP, RFP+mCherry, and mCherry+LOVpep to confirm order of components.
25µL GoTaq Green reaction
Component
Volume (µL, one reaction)
x32
Master Mix (µL)
2x GoTaq Green Master Mix
12.5
400
10µM Forward primer
1
-
10µM Reverse primer
1
-
CTAV template (~50ng/µL)
1
-
H20 to 25
9.5
304
176µL master mix to 4 tubes, plus 8µL of CTAV template each. 23µL of each mix to each reaction, plus 1µL of each primer.
PCR program: 2min 95°C, 30x(30s@95°C, 30s@55°C, 1:50@72°C), 5min @ 72°C, hold at 4°C
Results:
Only RFP and LOVpep are present, and no parts are in order.
Top of page
9/09/2012
verification of more CTAV clones
Minprep CTAV clones 5-10 (Zymokit, resuspend 1.5mL culture in 600µL H20, elute 30µL water)
Clone
Concentration (ng/µL)
CTAV-5
236.7
CTAV-6
235.4
CTAV-7
151.3
CTAV-8
93.5
CTAV-9
105.6
CTAV-10
124.1
BsaI, BamHI/SalI test digest
BsaI:
Component
Volume
10x NEBuffer 4
1
100x BSA
.1
plasmid DNA (conc. above)
2
BsaI-HF (20U/µL)
.5
H20 to 10
6.4
Total
10
BamHI/SalI
Component
Volume
10x NEBuffer 4
1
100x BSA
.1
plasmid DNA (conc. above)
2
BamHI-HF (20U/µL)
.5
SalI-HF (20U/µL)
.5
H20 to 10
5.9
Total
10
One hour digest at 37°C.
Results:
Clones 6-10 show digestion pattern relatively close to expected, but many look slightly different from each other.
PCR verification of clones 6-10:
25µL GoTaq Green
Component
Volume (µL, one reaction)
x40
Master Mix (µL)
2x GoTaq Green Master Mix
12.5
500
FWD primer 10µM
1
-
REV primer 10µM
1
-
CTAV template (~50ng/µL)
1
-
H20 to 25
9.5
380
Total
25
880
176µL to each mix, plus 8µL CTAV template. 23µL to each PCR tube, plus 1µL of each primer. Primer pairs: promoter, RFP, mCherry, LOVpep+terminator, promoter+RFP, RFP+mCherry, mCherry+LOVpep.
PCR program: 3min 95°C, 25x(30s@95°C, 30s@55°C, 1:50@72°C), 5min@72°C, hold at 4°C
Results:
Clones 6,8,10 look correct.
More verification digests: 10µL BamHI-HF/SalI-HF, ScaI, PvuI, KpaI-HF. .5µL of each enzyme, 2µL of miniprep DNA, BSA in each reaction, 37°C for one hour.
Results:
Top of page
9/10/2012
Sporulation of diploid re-localization strains
Inoculate acceptor vectors for miniprep
Inoculated TTAV-1, TTAV-2 into 5mL LB+chloramphenicol, CTAV-6,7,8 INTO 5mL LB+carb
Top of page
9/11/2012
repeat test digests of some N-Terminal vectors
BamHI/BlpI digests of 9/8 digests that did not have enough DNA to be visible on gel: CHK1.N2, SWE1.N1-C, SWE1.N2, WHI3.N2. EcoRI/BsaI digests of CHK1.N1 to verify that BsaI site is removed.
Component
Volume (µL)
10x NEBuffer
1
100x BSA
.1
DNA+H20 to ~300ng
5
Enzyme 1 (20U/µL)
.5
Enzyme 2 (20U/µL)
.5
H20 to 10
2.9
Total
10
Digestion/Ligations into CTAV
WHI5.C, CHK1.C1-B, CHK1.C2-A, CDC15.C2, CDC15.C3, SWE1.C1, SWE1.C2, WHI3.C1, WHI3.C2, FAR1.C1-B, FAR1.C2-A.
Diluted all parts to concentration for 1µL per reaction to be equimolar to 100ng of CTAV backbone.
Component
Volume (µL)
10x T4 buffer
1.5
100x BSA
.15
BsaI-HF (20U/µL)
1
NEB T4 ligase (2000U/µL)
1
CTAV (114ng/µL)
1
H20 to 14
9.35
Total
14
Plus 1µL of each part. 37°C for one hour, 5 min 50°C, 5 min 80°C, hold at 4°C.
Transformed 2µL into 40µL competent cells.
inoculate acceptor vectors for minipreps
Inoculated ProAV-2, ORFAV-2, TTAV-2 in 5 mL LB+chlor.
Top of page
9/12/2012
Inoculate CTAV dig-lig colonies
All CTAV dig-ligs grew except for FAR1.C2-A. ~772 colonies for .1ng pRS416 control = 7.7*10^6cfu/µg, 0 colonies no DNA control. Picked one colony each dig-lig and inoculated into 5mL LB+carb
Tetrad dissection practice
Top of page
9/13/2012
miniprep of C-terminal dig-ligs
Miniprep performed on the following:
CTAV-WHI5.C
CTAV-CHK1.C1-B
CTAV-CHK1.C2-A
CTAV-CDC15.C2
CTAV-CDC15.C3
CTAV-SWE1.C1
CTAV-SWE1.C2
CTAV-WHI3.C1
CTAV-WHI3.C2
CTAV-FAR1.C1-B
Used the Zymo kit*
*Before lysing, spun down 1.5mL of the culture, tossed the supernatant, and resuspended in 600µL water
Practice tetrad dissection
Top of page
9/14/2012
check C-terminal dig-lig miniprep concentrations
test digest of C-terminal dig-ligs
Top of page
9/15/2012
gel of CTAV dig-lig test digest
yeast co-transformation of N-terminal and C-terminal split protein plasmids
Top of page
9/16/2012
Colony PCR of biobrick dig-ligs
12µL GoTaq green colony PCR of 4 clones each of ProAV-1, ProAV-2, ProAV-3, ProAV-4, ProAV-5, GFP, CYP2E1 biobricks, tMFA2 biobrick.
PCR program: 3min 95°C, 25x(30s @95°C, 30s @55°C, 1:30 @72°C), 5 min 72°C, hold at 4°C.
All inserts are correct. Inoculated first clone of each part from 96 well plate to 5mL LB-chlor at 37°C.
Top of page
9/17/2012
Miniprep biobricks
Miniprepped ProAV-1 through ProAV-5, GFP, CYP2E1, tMFA3 biobricks. (Zymo kit, resuspend 1.5mL culture, elute in 30µL water).
Miniprep
Concentration (ng/µL)
ProAV-1
67.3
ProAV-2
62.2
ProAV-3
61.9
ProAV-4
70.1
ProAV-5
185.8
GFP
105.7
CYP2E1
138.5
tMFA2
41.8
Top of page
9/18/2012
Streak out split protein transformants onto SC-leu-ura
Spread 300µL Met, 250µL Lys, 75µL His onto SC-8 plates, let dry.
All transformants had many colonies except for CHK1.1 (one colony), CDC15.3 (~5 colonies), SWE1.1 (few very small colonies). 255 colonies for .1ng pRS416 control (2.55*10^6 cfu/µg), no colonies for no DNA control.
Struck out up to 8 colonies onto SC-leu-ura for each strain.
Replica plate dissections onto SC-ura+G418
Leslie dissected 8 mating pairs: two pairs each of Cdc14 x Net1, Apc11 x Mid2, Kap95 x Mid2, Tub1 x Mid2.
Spread 175µL water, 25µL 1000x G418 onto SC-ura plates, let dry.
Replica plated from dissection plates onto SC-ura+G418.
Top of page
9/19/2012
Restreak CDC14-4 x MID2-1 onto YPD+G418
From SC-ura plates from 9/17 onto YPD+G418.
Redo replica plating of 9/18 relocalization strains
From YPD dissection plate onto SC-ura and YPD+G418 rather than SC-ura+G418. G418 doesn't select strongly in dropout media.
No YPD+G418 plates left, spread 175µL water and 25µL 1000x G418 onto YPD, let dry.
Top of page
9/20/2012
Sent biobricks for sequencing
Submitted: ProA1 through ProA5, GFP, CYP2E1, tMFA2 with VF2 and VR sequencing primers.
Top of page
9/21/2012
Blue light induction of some re-localization strains
Subcultured Apc11 x Mid2, Cdc14 x Mid2, Cdc14 x Net1, Kap95 x Mid2, Tub1 x Mid2 from overnights into 5mL SC. Grew to OD600 of ~.2, put 200µL each into two plates, one blue light and one dark. Induced in 30C without shaking for 2 hours. Spun down cultures and examined under light microscope 40x. Counted 200 cells each.
Strain
Unbudded
Small bud
Large bud
Apc11 x Mid2 dark
56
49
95
Apc11 x Mid2 blue light
46
36
118
Tub1 x Mid2 dark
57
59
84
Tub1 x Mid2 blue light
56
43
101
Kap95 x Mid2 dark
62
66
72
Kap95 x Mid2 blue light
55
52
93
Cdc14 x Net1 dark
33
58
109
Cdc14 x net1 blue light
41
55
104
Cdc14 x Mid2 dark
48
31
121
Cdc14 x Mid2 blue light
42
35
123
No arrest observed. May be due to using YPD which is dark, and lack of shaking, or too short of an induction time.
Inoculated two-plasmid strains into 5mL SC-leu-ura for overnights.
Patched out good clones of remaining relocalization strains onto YPD, and inoculated 5mL YPD for overnights.
Top of page
9/22/2012
Fluorescence microscopy of some relocalization strains
DIC, mCherry, GFP images for Cdc14 x Net1, Cdc14 x Mid2, Cdc15 x Mid2
Inoculated split protein strains into 5mL SC-leu-ura for microscopy.
Top of page
9/23/2012
Fluorescence microscopy of split protein strains
DIC, mCherry, GFP, 15s blue light excitation, mCherry, GFP
Cdc15.2, cdc15.3, chk1.1, chk1.2, far1.1, whi3.1, whi3.2, whi5
Top of page
9/24/2012
Inoculated 12 re-localization strains
Cdc14 x Mid2 4-1, 3-2, 4-1, 3-1
Cdc14 x Net1 1-1, 2-1, 1-3, 2-3
Tub1 x Mid2 2-4
Apc11 x Mid2 1-7
Kap95 x Mid2
Clf1 x Mid2 1-3
Cdc15 x Mid2 2-2
Into 5mL SC for blue light induction experiments tomorrow and re-localization microscopy.
Top of page
Retrieved from " http://2012.igem.org/Team:Johns_Hopkins-Wetware/lightnotebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
