Team:IISER-Bhopal-India/Modelling
Team Methnote
Home
Project
Problem
Idea
Approach
Modelling
Future Prospects
Human Practices
Outreach
Public Engagement
Integrated Human Practices
Collaborations
Biosafety
People
Team
Attributions
Sponsors
For Judges
Lab Logs
Notebook
Protocols
Results
Inter Lab Study
Parts
Project
Modelling
Modelling
Theory
A biological system can be visualized at various levels through modeling. A model for gene regulation pathway was built for the relevant genes and the reactions in Pichia pastoris, which represents the induced pathway of soluble Methane Monooxygenase (sMMO) along with the AOX-iLOV gene construct. A kinetic analysis was thereby done using mathematical expressions, which included differential equations, initial assignments, repeated assignments, and algebraic constraints.
The Model for Pichia pastoris.
For the kinetic analysis of the model, the law of mass action and the steady-state principle were taken into consideration. Hence, the equation for mass action kinetics and the Michaelisâ€“Menten equation were used. The temperature and pH values considered throughout the analysis were 25 degrees and 7 respectively.
sMMO sequence Transcription and Translation
The gene sequences P2A1a and P2A2a would be transcribed in the organism as shown below
Reaction 1 Â 
Â  Â  Â  Â 
The reaction rate is calculated by the formula
Â  Â 
Where,
RDNA = Rate of transcription,
ktranscription = Transcription rate constant representing the no. of DNA molecules transcribed per second (Its value depends on the type of RNA polymerase)
Â  Â 
Hence, Â 
Reaction 2
Â 
where,
RmRNA = Rate of Translation,
ktranslation = Transcription rate constant representing the no. of mRNA molecules translated per second.
Hence,
Both the mRNA molecules code for the sMMO subunits which would get assembled in the cell.
Further kinetic analysis involves steady-state kinetic analysis of sMMO enzyme. Methane binds to the enzyme, followed by NADH, which reacts to yield reduced enzyme and NAD+. The reduced sMMO-methane complex binds to O2. The complex then breaks down to release water and methanol. The values of KM and Vmax are taken from literature[1].
where,
ð‚ = Reaction rate,
P = Product,
Vmax = Maximum rate achieved by the system
S = Substrate (for sMMO enzyme)
KM = Michaelis Menten Constant
Reaction 3
Â 
Reaction 4
Â 
Reaction 5
Â 
br>
Growing Pichia pastoris in the presence of sodium formate would help in NADH regeneration[2]. The rate of NADH regeneration is taken to be the same as that of the rate of NADH reduction.
Reaction 6
Â 
iLOV gene expression kinetics: The methanol thus produced participates in iLOV gene expression through AOX promoter, and thus the mRNA will be produced. The transcription and translation rates depend on the gene length and other factors as described previously.
Reaction 7
Â 
Reaction 8
Â 
Mass action kinetics was applied for reactions 7 and 8.
Degradation Reactions: The sMMO enzyme, like, every other enzyme, gets degraded after a time period. The half life (t1/2) of a species can be used to determine its rate of degradation (kdegradation) for a first order reaction through decay kinetics.
The degradation rate of sMMO complex[3] and mRNA[4] molecules was thus determined.
Reaction 9
Â 
Reaction 10
Â 
Reaction 11
Â 
Reaction 12
Â 
The methanol is also used by the cell as a carbon source apart from being used for iLOV gene expression. The methanol uptake by Pichia pastoris was thus used to calculate the degradation rate of methanol in this model.[5]
Model simulations
The dynamic behaviour of the pathway reactions was studied by running a simulation through the software. The simulation results are given in figure 1a and 1b. The input concentrations value for P2A1, P2A2, AOX-iLOV, H+, H2O and O2 species were kept constant for a simulation. Hence, the plots of concentration against time for these species have overlapped. The details are given in description of Figure 1.
Figure 1a
Figure 1b.
Figure 1: States represent the amount of substance. The simulation was run considering 70, 135 and 200 copies of P2A1, P2A2 and AOX-iLOV plasmids in the cell. The time range of the simulation is for (a)1hr and (b) 2 days.  Straight line with slope tending to zero represents overlap of the following species: Methane, Methane- sMMO complex, NADH, NAD+, O2, H2O, iLOV FMN fluorophore, mRNA 1, mRNA 2, mRNA 3, P2A1, P2A2, AOX-iLOV and H+ concentration plots.  Upper curve represents methanol concentration, lower curve represents sMMO enzyme concentration and the straight line with negative slope represents Reduced methane- sMMO complex concentration.
Fluorescence estimation:
UV/blue light is detected via the chromophore flavin mononucleotide (FMN) located within the LOV domain, giving the protein a weak intrinsic fluorescence with a maximal emission wavelength at 495 nm. The no. of photons emitted could be calculated by multiplying the quantum yield[6] to the no. of iLOV domains produced.  A study of estimated fluorescence with changing plasmid copy number was done. Since the plasmids (pGKB and pGHYB) used are yeast episomal plasmids, multiple copies of the same are present in the cytoplasm. The plasmid does not get integrated in the genome of the organism. All the reactions are therefore, supposed to take place in the cytoplasm of the chassis organism.
Figure 2: Variation in no. of photons emitted with the changing no. of plasmid copies in the organism
Technical details:
The system model for P. pastoris was built using the SimBiology toolbox in MATLAB software. A SimBiology model is composed of a set of expressions that describe the dynamics of a biological system altogether. These set of expressions can be in the form of reactions, differential equations or discrete events. The expressions are usually written in terms of quantities such as compartments, species or parameters. These are enumerated in the model. The initial parameters assigned to a model can be empirical experimental values or reported literature values. The modeling of our chassis was done without any empirical data. The estimations made could not be validated. The parameter values provided to the software were, thus, taken solely from literature.
References:
[1] Green, J. and Dalton, H., 1986. Steady-state kinetic analysis of soluble methane mono-oxygenase from Methylococcus capsulatus (Bath). Biochemical Journal, 236(1), pp.155-162.
[2] Mehta, P.K., Ghose, T.K. and Mishra, S., 1991. Methanol biosynthesis by covalently immobilized cells of Methylosinus trichosporium: batch and continuous studies. Biotechnology and bioengineering, 37(6), pp.551-556.
[3] Sakai, Y., Koller, A., Rangell, L.K., Keller, G.A. and Subramani, S., 1998. Peroxisome degradation by microautophagy in Pichia pastoris: identification of specific steps and morphological intermediates. The Journal of cell biology, 141(3), pp.625-636.
[4] http://book.bionumbers.org/how-fast-do-rnas-and-proteins-degrade/
[5] JordÃ , J., Rojas, H.C., Carnicer, M., Wahl, A., Ferrer, P. and Albiol, J., 2014. Quantitative metabolomics and instationary 13C-metabolic flux analysis reveals impact of recombinant protein production on trehalose and energy metabolism in Pichia pastoris. Metabolites, 4(2), pp.281-299.
[6] Chapman, S., Faulkner, C., Kaiserli, E., Garcia-Mata, C., Savenkov, E.I., Roberts, A.G., Oparka, K.J. and Christie, J.M., 2008. The photoreversible fluorescent protein iLOV outperforms GFP as a reporter of plant virus infection. Proceedings of the National Academy of Sciences, 105(50), pp.20038-20043.
[7] Pavlov, M.Y. and Ehrenberg, M., 1996. Rate of Translation of Natural mRNAs in an Optimizedin VitroSystem. Archives of biochemistry and biophysics, 328(1), pp.9-16.
[8] McClure, W.R., 1980. Rate-limiting steps in RNA chain initiation. Proceedings of the National Academy of Sciences, 77(10), pp.5634-5638.
iGEM
Supported by Innovation and Incubation Cell, IISER Bhopal
