<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IISER-Bhopal-India_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IISER-Bhopal-India/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team Methnote</TITLE></P><DIV id="wrapper"><HEADER><DIV class="navbar navbar-default navbar-static-top"><DIV class="container"><DIV class="navbar-collapse collapse "><UL class="nav navbar-nav"><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" data-hover="dropdown" data-delay="0" data-close-others="false">Project </A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Problem">Problem</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Idea">Idea</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/approach">Approach</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Modelling">Modelling</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Prospects">Future Prospects</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle " data-toggle="dropdown" data-hover="dropdown" data-delay="0" data-close-others="false">Human Practices </A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Human_Practices">Outreach</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Integrated_Human_Practices">Integrated Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Biosafety">Biosafety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle " data-toggle="dropdown" data-hover="dropdown" data-delay="0" data-close-others="false">People </A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Sponsors">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/ForJudges">For Judges</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle " data-toggle="dropdown" data-hover="dropdown" data-delay="0" data-close-others="false">Lab Logs </A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/InterLab">Inter Lab Study</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Bhopal-India/Improve">Parts</A></LI></DIV></DIV></DIV></HEADER><SECTION id="inner-headline"><DIV class="container"><DIV class="row"><DIV class="col-lg-12"><UL class="breadcrumb"><LI>Lab Logs</LI><LI class="active">Results</LI></UL></DIV></DIV></DIV></SECTION><SECTION id="content"><DIV class="container"><DIV class="row"><DIV class="col-lg-12"><DIV class="column full_size"><H3>Gene Splicing by Overlap Extension PCR</H3><H4>Parts and vector confirmation:</H4><P align="justify" style="font-family:'Roboto', sans-serif; font-size:16px; font-weight:300; line-height:1.6em; color:#656565;"> BioBrick parts obtained from iGEM Repository and vectors ordered from AddGene were confirmed by their transformation into competent DH5α strain of <I>E.coli</I> and subsequent plasmid isolation from the obtained colonies.

</P><P style="text-align:center;"><LI align="center"><B>Figure 1 (a) </B> Single Digestion Confirmation of parts procured from iGEM repository - 
Plasmids encoding subunits of sMMO complex procured from iGEM repository [Bba_K139007:sMMO B_pSB1C3 (2312kB), Bba_K139008:sMMO C_pSB1C3 (3047bp), Bba_K139009:sMMO D_pSB1C3 (2312bp), Bba_K139010:sMMO X_pSB1C3 (3584bp), Bba_K139011:sMMO Y_pSB1C3 (3170bp), Bba_K139012:sMMO Z_pSB1C3 (2510bp), Bba_K139013:AOXP-RFP_pSB1C3 (3577bp)] were single-digested with EcoRI restriction enzyme for size confirmation.  Incubated at 37’C for 1hr and the digestion products were run on 1% Agarose gel.<B>(b)</B> Isolation and purification of pGKB plasmid - A Pichia pastoris expression vector pGKB (3506bp) procured from AddGene was isolated and purified by miniprep and run on 1% agarose gel for validation</LI></P><H4>PCR Amplification of parts to be cloned:</H4><P align="justify" style="font-family:'Roboto', sans-serif; font-size:16px; font-weight:300; line-height:1.6em; color:#656565;"> PCR amplification of sMMO D and sMMO X genes using appropriate primers (IDT) with overhangs containing restriction sites necessary for cloning into pGKB along with kozak sequence for protein expression in <I>Pichia</I> was done. 

Gel Electrophoresis was done for the amplified parts and DNA bands were visualized. Bands of proper sizes were observed which confirmed the size of the parts.



 </P><P style="text-align:center;"><LI align="center"><B>Figure 2</B> PCR Amplification of sMMO Subunits - Genes for sMMO D (312bp) and sMMO X (1584bp) were PCR amplified from parts procured from iGEM repository, purified and run on 1% Agarose gel for validation
</LI></P><H4>sMMO D and sMMO X cloned in pGKB Vector:
</H4><P align="justify" style="font-family:'Roboto', sans-serif; font-size:16px; font-weight:300; line-height:1.6em; color:#656565;"> Genes for MMO subunits-sMMO D and sMMO X- obtained after PCR Amplification and pGKB Vector were digested with Restriction Enzymes NgoMIV and SalI  at 37ºC  overnight. Digested DNA was purified by Gel Elution purification and ligation was setup followed by transformation into competent DH5α E.coli cells. Plasmid was isolated from the obtained colonies and confirmed by Agarose Gel Electrophoresis.

 </P><P align="justify" style="font-family:'Roboto', sans-serif; font-size:16px; font-weight:300; line-height:1.6em; color:#656565;">The cloned plasmid sMMO D_pGKB (pGKB with sMMO D gene flanked by GAP Promoter and AOX Terminator) was confirmed by double digestion with NgoMIV and SalI and sMMO X_pGKB (pGKB with sMMO X gene flanked by GAP Promoter and AOX Terminator) with SacII and EcoRI. Bands of appropriate sizes were observed after Agarose Gel Electrophoresis which confirmed the cloning of sMMO D and sMMO X in pGKB.
</P><P style="text-align:center;"><LI align="center"><B>Figure 3 (a), (b), (c)</B> Cloning of sMMO D and sMMO X parts in pGKB vector and clone confirmation by restriction digestion - Subunits of sMMO complex, sMMO D (312bp) and sMMO X(1584bp) were cloned in pGKB(3506bp). The plasmids sMMO D_pGKB and sMMO X_pGKB were isolated and run on 1% Agarose gel. The same plasmids were double digested using restriction enzymes (NgoMIV-SalI and SacI EcoRI, respectively),  by incubation at 37’C for 1 hr and the digestion products sMMO D_pGKB (RD) and sMMO X_pGKB (RD) were run on 1% Agarose gel for clone confirmation
</LI></P><H4> PCR Amplification of DNA fragments for SOEing PCR:

</H4><P align="justify" style="font-family:'Roboto', sans-serif; font-size:16px; font-weight:300; line-height:1.6em; color:#656565;">Genes for sMMO D and GAP Promoter were PCR amplified from sMMO D_pGKB recombinant plasmid with primers necessary for introducing the overlap region required for SOEing PCR. This confirmed the cloning of sMMO D in pGKB as well. Also, genes for sMMO X and AOX Terminator were PCR amplified from sMMO X_pGKB recombinant plasmid similarly. These were confirmed by observing bands of appropriate sizes after Agarose Gel Electrophoresis.
 </P><P style="text-align:center;"><LI align="center"><B>Figure 4</B> PCR Amplification of parts for SOEing PCR - sMMO D (312bp), sMMO X (1584bp), GAP Promoter (477bp) and AOX Terminator (247bp) were PCR amplified using primers containing complementary overhangs for SOEing PCR. The PCR products were verified by electrophoresis on 1% Agarose gel
</LI></P><H4> SOEing PCR for sMMO D and AOX Terminator:


</H4><P align="justify" style="font-family:'Roboto', sans-serif; font-size:16px; font-weight:300; line-height:1.6em; color:#656565;">sMMO D and AOX Terminator were joint by homologous recombination using SOEing PCR. The amplified product was visualised and verified by gel electrophoresis. 




 </P><P style="text-align:center;"><LI align="center"><B>Figure 5</B> sMMO D-AOX_T construct obtained by SOEing PCR - sMMO D subunit (312bp) and AOX terminator (247bp) were PCR amplified with complementary overhangs and finally amplified by SOEing PCR to get sMMO D-AOX_T (559bp). The SOEing PCR product was validated by electrophoresis on 1% Agarose gel
</LI></P><H4> SOEing PCR for sMMO X and GAP Promoter:



</H4><P align="justify" style="font-family:'Roboto', sans-serif; font-size:16px; font-weight:300; line-height:1.6em; color:#656565;">sMMO X and GAP Promoter were joint by homologous recombination using SOEing PCR. The amplified product was visualised and verified by gel electrophoresis.





 </P><P style="text-align:center;"><LI align="center"><B>Figure 6</B> sMMO X-GAP_P construct obtained by SOEing PCR - sMMO X subunit (1584bp) and GAP Promoter (477bp) were PCR amplified with complementary overhangs and finally amplified by SOEing PCR to get sMMO X-GAP_P(2061bp). The SOEing PCR product was validated by electrophoresis on 1% Agarose gel
</LI></P><H4>Cloning of sMMO D+AOX Terminator in pGKB Vector:

</H4><P align="justify" style="font-family:'Roboto', sans-serif; font-size:16px; font-weight:300; line-height:1.6em; color:#656565;">sMMO D+ AOX Terminator construct obtained after PCR Amplification and pGKB Vector were digested with restriction enzymes NgoMIV and KpnI at 37ºC overnight. Digested DNA was purified by Gel Elution purification and ligation was setup followed by transformation into competent DH5α E.coli cells. The plasmid was isolated from the obtained colonies and run on a gel by agarose gel electrophoresis.
The cloned plasmid sMMO D+AOX Terminator_pGKB was visualized and verified by observing a band of expected size by gel electrophoresis.
The cloned plasmid was further confirmed by double digestion of the plasmid with NgoMIV and KpnI after which bands of appropriate sizes were observed. 




 </P><P style="text-align:center;"><LI align="center"><B>Figure 7 (a), (b)</B> Cloning of sMMO D-Aox T in pGKB vector and clone confirmation by restriction digestion - SOEing PCR product sMMO D-AOX T (559bp) cloned in pGKB Plasmids (3506bp) and the resultant clone sMMO D-AOX T_pGKB (4065bp) was run on 1% Agarose gel for validation.  The clone sMMO D-AOX T_pGKB (4065bp) was double digested using restriction enzymes  NgoMIV and KpnI  for clone confirmation 
s</LI></P><H4>Cloning of sMMO X+GAPP in sMMO D+AOXT_pGKB Vector:


</H4><P align="justify" style="font-family:'Roboto', sans-serif; font-size:16px; font-weight:300; line-height:1.6em; color:#656565;">sMMO X+GAPP construct obtained after PCR Amplification and sMMO D+AOXT_pGKB Vector were digested with restriction enzymes KpnI and SalI at 37’C  overnight. Digested DNA was purified by Gel Elution purification and ligation was setup followed by transformation into competent DH5α E.coli cells. Plasmid was isolated from the obtained colonies and run on gel by agarose gel electrophoresis.
The cloned plasmid sMMO D+AOXT+sMMO X+GAPP_pGKB was visualized and verified by observing band of expected size by gel electrophoresis.
The cloned plasmid was further confirmed by double digestion of the plasmid with KpnI and SalI after which bands of appropriate sizes were observed.





 </P><P style="text-align:center;"><LI align="center"><B>Figure 8</B> Cloning of sMMO X-GAP_P in sMMO D+AOXT_pGKB vector and clone confirmation by restriction digestion with KpnI and SalI- SOEing PCR product sMMO X-GAP_P (2061bp) cloned in sMMO D+AOXT_pGKB Plasmids (4065bp) and the resultant clone sMMO D+AOXT+sMMO X+GAPP_pGKB (6126bp) was run on 1% Agarose gel for validation </LI></P><H3> Antibiotic Sensitivity Assay </H3><H4>Antibiotic Sensitivity Assay for Pichia pastoris for G418:</H4><I>Pichia pastoris</I> was grown in different concentrations of the antibiotic G418 to determine the optimum concentration of the antibiotic to be used for selection of the transformed yeast. O.D.600 of the culture was recorded at different time points with antibiotic concentrations ranging from 100μg/ml to 500μg/ml and plotted against time. The results obtained were analysed and 300μg/ml was found to be optimal for selection of the transformed yeast as the growth of the original untransformed strain was restricted in that concentration. 

Further yeast transformations were done using plates with 300μg/ml G418.
<P style="text-align:center;"><LI align="center"><B>Figure 9 </B>G418 Antibiotic Sensitivity Assay for <I> Pichia pastoris</I> - Growth of <I>Pichia pastoris</I> cells was recorded in liquid YPD media with different G418 concentration at 30'C for 24hrs using an automated plate reader. The average cell density (OD600) of three independent isolates per G418 concentrationwith error bar was plotted against different time points to obtain a growth curve suggesting optimum G418 concentration sensitive for <I>P. pastoris</I></LI></P><H3>Alternate cloning strategy</H3><H4> Cloning of P2A Insert 1 in pGKB vector: </H4><P align="justify" style="font-family:'Roboto', sans-serif; font-size:16px; font-weight:300; line-height:1.6em; color:#656565;">
Genes for MMO subunits - sMMO B, sMMO Y and sMMO D- synthesized as gBlocks from IDT with 6xHis tags tailing each subunit flanked by the Kozak sequence in the front and intermediate P2A sequences (Insert Name: P2A1)  and pGKB Vector were digested with Restriction Enzymes EcoRI and KpnI at 37ºC overnight. Digested DNA was purified by Gel Elution purification and ligation was setup followed by transformation into competent DH5α E.coli cells. Plasmid was isolated from the obtained colonies and confirmed by Agarose Gel Electrophoresis.
The cloned plasmid P2A1_pGKB (pGKB with P2A1 insert flanked by GAP Promoter and AOX Terminator) was confirmed by double digestion with EcoRI and KpnI. Bands of appropriate sizes were observed after Agarose Gel Electrophoresis which confirmed the cloning of P2A1 Insert in pGKB. 
</P><P style="text-align:center;"><LI align="center"><B>Figure 10</B> Cloning of P2A1 in pGKB vector and clone confirmation by restriction digestion with EcoR1 and KpnI- g block P2A1 (2733bp) cloned in pGKB Plasmid (3506bp) and the resultant clone P2A1_pGKB (6239bp) was run on 1% Agarose gel for validation </LI></P><H3>
Part Submission:
</H3><H4> Cloning of P2A1 and P2A2 in pSB1C3</H4><P align="justify" style="font-family:'Roboto', sans-serif; font-size:16px; font-weight:300; line-height:1.6em; color:#656565;">P2A1(sMMO C, sMMO D and sMMO Y) and P2A2(sMMO B, sMMO X and sMMO Z) parts were submitted, cloned into pSB1C3, for the silver medal criteria.
</P><P style="text-align:center;"><LI align="center"><B>Figure. 11</B> Cloning of P2A1 (a) and P2A2 (b) in pSB1C3 vector and clone confirmation by restriction digestion with EcoRI and with EcoRI and PstI- g block P2A1 (2733bp) cloned in pSB1C3 Plasmid (2Kb) and g block P2A2 (2733bp) cloned in pSB1C3 Plasmid (2Kb) was run on 1% Agarose gel for validation </LI></P></DIV></DIV></DIV></DIV></SECTION><TITLE>iGEM</TITLE><FOOTER><DIV class="container"><DIV class="row"><DIV class="col-lg-6"><DIV class="copyright"><P><A href="https://iice.iiserb.ac.in"><FONT color="gray"> Supported by Innovation and Incubation Cell, IISER Bhopal</FONT></A></P></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>