<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NYU_Abu_Dhabi_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NYU Abu Dhabi/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="app"><NAV class="navbar navbar-expand-lg fixed-top navbar-light bg-light"><DIV class="collapse navbar-collapse" id="navbarNavDropdown"><UL class="navbar-nav"><LI class="nav-item active"><A class="nav-link" href="https://2017.igem.org/Team:NYU_Abu_Dhabi">Home <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                Project
              </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Description">Our Project</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Results">Results</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Parts">Parts</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Demonstrate">Demo</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Applied_Design">Design</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Collaborations">Collaborations</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                Lab Notebook
              </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Biology">Biology</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Engineering">Engineering</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Protocols">Protocols</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                Human Practices
              </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/HP">Overview</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/HP/Silver">Human Practices</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/HP/Gold_Integrated">Integrated Human Practices</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Engagement">Engagement and Education</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/InterLab">Interlab</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                Team
              </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Team">Team</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Attributions">Attributions</A></DIV></LI></UL></DIV></NAV><DIV class="section-block"><DIV class="tab"><BUTTON class="tablinks" onclick="openTab(event, 'Key Achievements')" id="defaultOpen">Key Achievements</BUTTON><BUTTON class="tablinks" onclick="openTab(event, 'Biology')" id="defaultOpen">Biology</BUTTON><BUTTON class="tablinks" onclick="openTab(event, 'Engineering')">Engineering</BUTTON><BUTTON class="tablinks" onclick="openTab(event, 'Proof of Concept')">Proof of Concept</BUTTON><BUTTON class="tablinks" onclick="openTab(event, 'Cost Analysis')">Cost Analysis</BUTTON></DIV><DIV id="Biology" class="tabcontent"><ARTICLE><H3 class="subheading">Successfully digested, ligated, and transformed <I>rfbE</I> into <I>pSB1C3</I></H3><P class="section-content">  
The objective of this experiment was to digest and ligate the <I>rfbE</I> BioBrick  <A href="http://parts.igem.org/Part:BBa_K2495001">BBa_K2495001</A>  into the <I>pSB1C3</I> linearized backbone for DNA submission and characterization. The <I>rfbE</I> gene was obtained in the BioBrick format from IDT. A modified restriction double digest and ligation protocol was used and can be found on our  <A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Protocols">protocol</A> page. We confirmed that our gene of interest was successfully ligated into the <I>pSB1C3</I> backbone by running PCR using primers <A href="http://parts.igem.org/Part:BBa_K2495019"> BBa_K2495019</A> and <A href="http://parts.igem.org/Part:BBa_K2495020"> BBa_K2495020</A>. The products were visualized by running a 1.2% agarose gel. Our ligated part in lane 2 shows a band at approximately 2000-bp, which corresponds to the length of our amplified region. Lane 3 also shows the amplified product, as well as the ligated product (in yellow) at approximately 4500-bp.
</P><DIV id="caption"><FIGCAPTION><B>Figure 1.</B> Lanes were loaded as follows: (1) 500-bp Bio-Rad ladder; (2) Positive control of amplified DNA using PCR primers <A href="http://parts.igem.org/Part:BBa_K2495019">BBa_K2495019</A> and <A href="http://parts.igem.org/Part:BBa_K2495020">BBa_K2495020</A>; (3) Ligated product, with the amplified <I>rfbE</I> in white and the ligated product itself in yellow; and, (4) Negative control of water with PCR primers. </FIGCAPTION></DIV></ARTICLE><H3 class="subheading">Optimized the LAMP reaction to 20 minutes without lysis or centrifugation </H3><P class="section-content"> 

The optimal conditions for cell lysis were previously investigated using <I>E. coli</I> DH5α transformed with pGLO plasmid (see <A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Biology">biology lab notebook</A>). The optimal cell lysis conditions for use with LAMP were tested using <I>rfbE</I> transformed into <I>E. coli</I>. 
</P><P class="section-content"> 
To test the specificity of the LAMP reaction, we employed multiple controls. Ligated products that were purified by miniprep (Qiagen) and verified previously by PCR were used as the positive controls. Negative controls included DH5α cells and a swab of <I>E. coli</I> from beef, which was verified by gram staining. 
</P><P class="section-content"> 
 To show that we have successfully cloned <I>rfbE</I> into the <I>pSB1C3</I> backbone, and to show the specificity of our PCR primers (<A href="http://parts.igem.org/Part:BBa_K2495019">BBa_K2495019</A>, <A href="http://parts.igem.org/Part:BBa_K2495020">BBa_K2495020</A>) targeting <I>rfbE</I> that we designed using <A href="http://primerexplorer.jp">PrimerExplorer V5</A>, we ran PCR of our samples below according to the following program:

<UL><LI>Initial denaturation at 95ºC for 5 min
</LI><LI>30 cycles of denaturation at 94ºC for 30 s, annealing at 50ºC for 45 s
</LI><LI>Final extension at 72ºC for 1.5 min
</LI><LI>Hold at 4ºC
</LI></UL></P><P class="section-content"> 
LAMP was run according to the instructions of the manufacturer (OptiGene) using primers targeted at <I>rfbE</I> designed with <A href="http://primerexplorer.jp">PrimerExplorer V5</A>. The experimental conditions were:
<UL><LI><I>rfbE</I> gene fragment (IDT) - positive control
</LI><LI> Plasmid DNA of <I>rfbE</I> in DH5α - negative control
</LI><LI> Plasmid DNA of <I>rfbE</I> in DH5α treated with 10 minutes of 95˚C heat lysis - negative control
</LI><LI> Plasmid DNA of <I>rfbE</I> in DH5α
</LI><LI><I>rfbE</I> in DH5α suspended in TE buffer and treated with 10 minutes of 95˚C heat lysis
</LI><LI><I>rfbE</I> in DH5α suspended in TE buffer (without heat treatment)
</LI><LI> Plasmid DNA of beef <I>E. coli</I></LI><LI> Plasmid DNA of beef <I>E. coli</I> treated with 10 minutes of 95˚C heat lysis
</LI></UL></P><P class="section-content">
The PCR reaction showed a clear band at 1000-bp (circled in black), while LAMP showed the typical smear of amplification. Both PCR and LAMP reactions did not show any banding or smearing in the DH5α or beef <I>E. coli</I> results, indicating that both techniques are specific to our gene of interest. However, comparison of the heat versus no heat lysis variables (circled in red) show that LAMP is able to amplify our gene of interest in the absence of the heat lysis step. This indicates that the LAMP technique is much more sensitive than PCR.
</P><P class="section-content">
Due to the superior specificity and sensitivity of LAMP, we envision that it will not be necessary to include a heat lysis step in our final version of the detection process for Shiga-toxigenic bacteria.
</P><FIGURE><DIV id="caption"><FIGCAPTION><B>Figure 2.</B> Comparison of PCR and LAMP sensitivities and specificities. Lanes were loaded as follows: (1) 500-bp Bio-Rad ladder; (2) Positive control of rfbE synthesized by IDT; (3) <I>rfbE</I>; (4) <I>rfbE</I> with heat lysis; (5) <I>rfbE</I>without heat lysis; (6) DH5α; (7) DH5α with heat; (8) DH5α without heat; (9) Beef E. coli with heat; and, (10) Beef E. coli without heat. </FIGCAPTION></DIV></FIGURE><H3 class="subheading">Sensitive to 10<SUP>6</SUP> cells/mL</H3><P class="section-content"> 
To determine the limit of detection of our LAMP reactions, we ran it on serial dilutions of inoculated broth and monitored the progress using the Applied Biosystems SteponePlus Real-Time PCR System (Thermo Fisher, MA). Inoculated broth was diluted and run according to the optimized protocol described in our <A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Biology">biology lab notebook</A>. The dilutions ranged from 10<SUP>8</SUP> cells/mL to 10<SUP>1</SUP> cells/mL. Six serial dilutions of a previously miniprepped sample were used as positive controls. The negative control was TE buffer. The reaction was run on a total volume of 12.5µl. Duplicates were averaged and the fluorescence was plotted against time using Excel.


</P><P class="section-content"> 
The results corresponded with a subsequent gel on the amplified products. The band intensity of the gels were in agreement with the relative fluorescence intensities visualized during the reaction.  
</P><DIV id="caption"><FIGCAPTION><B>Figure 3.</B> Fluorescence monitoring and gel of serial dilutions of inoculated broth.. Lanes were loaded as follows: (1) 500-bp Bio-Rad ladder; (2) <I>rfbE</I>, positive control; (3) LB broth, negative control; (4) ddH<SUB>2</SUB>O, negative control; (5) 10<SUP>8</SUP> cells/mL; (6) 10<SUP>7</SUP> cells/mL; (7) 10<SUP>6</SUP> cells/mL; (8) 10<SUP>5</SUP> cells/mL; (9) 10<SUP>4</SUP> cells/mL; (10) 10<SUP>3</SUP> cells/mL; (11) 10<SUP>2</SUP> cells/mL; (12) 10<SUP>1</SUP> cells/mL.</FIGCAPTION></DIV><P class="section-content"> 
The results visualized on the gel are in great agreement with the recorded change in fluorescence intensity, both indicating that the intensity of concentrations above and including 10<SUP>6</SUP> cells/mL are significantly higher than the control when using a 12.5µl total reaction volume. While samples of lower concentrations were run, we could not draw any conclusive evidence from these as they did not differ significantly from the fluorescence intensity of the control. We have found that using a larger reaction volume (25µl) does increase our sensitivity significantly, but due to shipping difficulties we were unable to obtain enough MasterMix to continue testing with a 25 µl reaction volume. Instead, subsequent tests were tested with 12.5µl volume.
Based on these results, we determined that our limit of detection is 10<SUP>6</SUP> cells/mL.   
 </P><FIGURE><DIV id="caption"><FIGCAPTION><B>Figure 4.</B> Comparison of sensitivity between 25 µl and 12.5 µl final reaction volume. Visually,
 the gels show that the larger volume allows us to detect lower concentrations of cells.</FIGCAPTION></DIV></FIGURE><P class="section-content">
The reaction products were also visualized under blue light and UV light at 302 nm. We can clearly visualize a gradient in the fluorescence from highest to lowest concentrations of DNA. All samples are clearly more fluorescent than the negative control. </P><DIV id="caption"><FIGCAPTION><B>Figure 5.</B> Visualization of the samples under blue light and UV light at 302 nm show a clear distinction between negative and positive controls. We envision using a colorimetric dye, hydroxynaphthol blue, for easier visualization in the future.  </FIGCAPTION></DIV></DIV><DIV id="Engineering" class="tabcontent"><ARTICLE><H3 class="subheading">Constructed a PDMS chip to house the reaction</H3><P class="section-content">
Each chip design was tested with the following conditions and requirements: <OL type="1"><LI>Able to let a volume of 25µl of sample to flow into the necessary reaction chamber.</LI><LI>Mix sufficiently well with the reagents (Simulated with water and dye for experimentation).</LI><LI>Prevent any leakage of fluid either via gaps or evaporation.</LI></OL></P><P class="section-content"> 
After testing numerous chip prototypes, we designed the optimized chip design that shows both successful fluid flow and thermal performance when subjected to 65˚C heating for LAMP reactions. A 5-well design was the result which prevented any fluid loss due to leakage and optimum mixing between the sample and LAMP reagents. No valving was necessary and the entire chip is 40 mm x 25 mm, making it much smaller and thus cheaper than previous designs.
</P><P class="section-content"> 
Each well can hold up to a   of 50µl, which is more than enough to contain both liquid reagents and sample. Thus, our design is not only limited to just lyophilized reagents. Previous designs only showed successful fluid flow but were not able to prevent condensate from escaping. The current design solves this by covering the wells with a hydrophobic, transparent material (optical adhesive) that does not interfere with fluorescence results.
</P><H3 class="subheading">Designed a heating circuit</H3><P class="section-content"> 
When developing the heating method, we found that it needed to satisfy the following properties:
<OL><LI> Sustain a temperature of 65˚C for the reaction
</LI><LI> Contain feedback control to maintain the temperature
</LI><LI> Affordable
</LI></OL></P><P class="section-content"> 
Between Peltier Cooler Modules, ITO heaters and hand warmers, we chose the Peltier because we found that it could reach the desired temperature, was easy-to-use and customize, and was reasonably priced compared to the ITO heater. The Peltier can be used with 5-7V voltage input and 1-2A current input. This property was ideal, considering that most portable power banks have a DC output of 5V and 2.1A. Hence, the heater was developed such that a portable power bank can act as the power supply.  

The Peltier temperature response graph of data taken over 2 hours shows that the time it takes for the glass plate to reach 65˚C is less than 4 minutes and 30 seconds, and remains at a constant 65 ±2 ˚C throughout the entire 2 hours.
</P><P class="section-content"> 

To control the Peltier Cooler Module, we used the tmp36 temperature sensor because it is cheap and easy-to-use. Additionally, an Arduino nano was used as the microchip controller due to its simplicity, a compact size, and its ability to do a proportional-integral-derivative (PID) control using tmp36. Finally, an N-Mosfet transistor was used to act as a switch control for the Peltier heater.
</P><P class="section-content"> 
An external power supply was added using a power adapter that allows for 6V and 1.5A. The Arduino nano was powered by Laptop USB, which only allows 500mA of current.The tmp36 uses the 5V voltage as a reference to measure the temperature on the Peltier heating plate. This sensor controls the N-Mosfet transistor that works with a low-side switch setup, allowing the current to flow whenever the temperature of the Peltier registers below 65˚C.
</P></ARTICLE></DIV><DIV id="Proof of Concept" class="tabcontent"><ARTICLE><P class="section-content"> 
The prototype of our STEC-detecting device was constructed using a PDMS chip containing 5 wells of 3 mm of diameter sealed with PCR-compatible optical adhesive film. The 65ºC temperature was achieved using a Peltier Modular Cooling system with a 6 Volts, 1.5 Amperes external power supply adapter. For more information on the design of the chip and the Peltier system used, please refer to the <A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Engineering">lab notebook</A>. 
</P><P class="section-content"> 
All the wells were loaded with 5μL of Isothermal Master Mix (OptiGene), 1.25μL of 10X SYBR Green (BioRad),  0.25μL of F3 primer (<A href="http://parts.igem.org/Part:BBa_K2495014">BBa_K2495014</A>), 0.25μL of B3 primer (<A href="http://parts.igem.org/Part:BBa_K2495015">BBa_K2495015</A>), 1μL of FIP primer (<A href="http://parts.igem.org/Part:BBa_K2495016">BBa_K2495016</A>), and 1μL of BIP primer (<A href="http://parts.igem.org/Part:BBa_K2495017">BBa_K2495017</A>). All primers were procured from Sigma Aldrich.
</P><P class="section-content"> 
Our LAMP reaction on our gene of interest, <I>rfbE</I>, was confirmed twice. The left image shows our first trial, in which the wells were loaded from left to right as follows: miniprep, positive control; TE buffer, negative control; 2.5µl of 10<SUP>6</SUP> cells/mL; 5µl of 10<SUP>6</SUP> cells/mL; and, 5µl of 10<SUP>6</SUP> cells/mL. The results were visualized under UV at 302 nm and blue light.  The image on the right shows the wells under blue light after running the LAMP reaction. The positive control (miniprepped target) shows significantly stronger fluorescence than the negative control (TE buffer). The samples of inoculated broth show an intermediate degree of fluorescence, indicating that there our target gene, <I>rfbE</I> is present in the samples.</P><FIGURE><DIV id="caption"><FIGCAPTION><B>Figure 1.</B> Left: the first trial visualized under blue light and UV light at 302 nm. Right: the second trial visualized under UV light at 365 nm. </FIGCAPTION></DIV></FIGURE><P class="section-content"> 
A time lapse of both trials are provided below. 

</P><P class="section-content"> 
To demonstrate the wide applicability of our device, we decided to test its capability to detect the presence of malaria-causing pathogen Plasmodium falciparum using primers targeting the mitochondrial cytochrome oxidase subunit 1 gene, which is specific to <I>Plasmodium falciparum</I> and allows it to be distinguished from <I>Plasmodium vivax</I>.<SUP>1</SUP> The LAMP reaction was prepared according to our optimized <A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Protocols">protocol</A>. The samples were loaded to the PDMS chip as follows:
<UL><LI>3x diluted genomic DNA, Sample 1 
</LI><LI>5x diluted genomic DNA, Sample 2
</LI><LI>7x diluted genomic DNA, Sample 3
</LI><LI>TE Buffer, negative control
</LI><LI>Genomic DNA (2ng/µl), positive control
</LI></UL></P><FIGURE><DIV id="caption"><FIGCAPTION><B>Figure 2.</B> Device test on <I>Plasmodium falciparum</I> genomic DNA visualized under blue light (left) and UV at 365 nm (right). </FIGCAPTION></DIV></FIGURE><P class="section-content"> 
After the reaction proceeded, all three samples exhibited a gradient of fluorescence under UV light (365 nm) and under blue light, as expected (Figure 2). There is also a clear difference in fluorescence between the samples and the negative control, indicating that the LAMP reaction successfully amplified the malarial marker with a high specificity. This experiment demonstrates our device’s potential as a pathogen surveillance method. The impact of our device extends far beyond our original purpose of food safety. By simply using a different set of primers to target a different genomic sequence, we can utilize the same device and technique to detect the presence of other known pathogens and viruses. This diagnostic method can easily be applied to the field of disease surveillance. In the context of malaria, this device could be used to test multiple water samples to detect the presence of malaria-causing pathogen in its breeding ground. 
</P><P class="section-content"> 
These preliminary studies prove that our device works with fluorescence detection. In future trials, the use of microfluidics channels will be investigated to provide a more user-friendly method to load the samples at once. The device will also be tested with a broader range of bacterial concentrations and on different pathogens and viruses in the near future.
</P></ARTICLE></DIV><DIV id="Cost Analysis" class="tabcontent"><ARTICLE><P class="section-content"><A href="http://pubs.acs.org/doi/abs/10.1021/acs.analchem.7b02478">Chen, et al. (2017)<SUP>2</SUP></A> developed a similar device which required the use of centrifugation and heat lysis. Their device was estimated to cost $550 USD, which is considerably more expensive and requires more laboratory techniques than the method we have developed.
</P></ARTICLE></DIV></DIV><P class="section-content">
1. Modak et al. Rapid Point-of-Care Isothermal Amplification Assay for the Detection of Malaria without Nucleic Acid Purification. Infectious Diseases: Research and Treatment 2016:9 1–9 doi:10.4137/IDRT.S32162. 
2. Chen, et al. Mobile Platform for Multiplexed Detection and Differentiation of Disease-Specific Nucleic Acid Sequences, Using Microfluidic Loop-Mediated Isothermal Amplification and Smartphone Detection. Analytical Chemistry 2017 doi:10.1021/acs.analchem.7b02478.

</P></DIV><DIV class="footer"><DIV class="container-fluid"><DIV class="row"><DIV class="col-lg-4 footer-section"><P class="footer-text">
              NYU Abu Dhabi is a research university with a fully integrated liberal arts and science college. It draws students from around the world, and prepares them for the challenges and opportunities of our interconnected world. <A href="http://nyuad.nyu.edu/en/about.html">Read More</A></P></DIV><DIV class="col-lg-4 footer-section"><P class="footer-text">
              NYUAD Saadiyat Island 
              Abu Dhabi, P.O. Box 129188, U.A.E <A href="mailto:nyuad.igem@nyu.edu">Email</A><A href="https://www.facebook.com/iGEMNYUAD/" target="_blank">Facebook</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>