Team:NYU Abu Dhabi/Parts
Home (current)
Project
Our Project Results Parts Demo Design
Collaborations
Lab Notebook
Biology Engineering Protocols
Human Practices
Overview Human Practices Integrated Human Practices Engagement and Education
Interlab
Team
Team Attributions
Basic Parts
Composite Parts
Characterized Parts
rfbE gene for O157 expression (BBa_K2495000) O-antigen transporter encoding sequence for E.coli. Essential for E.coli O157 expression and can be used as a safe alternative to testing for the presence of Shiga toxin.
rfbE LAMP primers - set 1 ( BBa_K2495002 , BBa_K2495003 , BBa_K2495004 , BBa_K2495005 , BBa_K2495006 , BBa_K2495007 )
This is the first set of LAMP primers used for the amplification of rfbE. The primers were designed using PrimerExplorer.
rfbE LAMP primers - set 2 ( BBa_K2495008 , BBa_K2495009 , BBa_K2495010 , BBa_K2495011 , BBa_K2495012 , BBa_K2495013 )
This is the second set of LAMP primers used for the amplification of rfbE. The primers were designed using PrimerExplorer.
rfbE LAMP primers - set 3 ( BBa_K2495014 , BBa_K2495015 , BBa_K2495016 , BBa_K2495017 )
This is the third set of LAMP primers used for the amplification of rfbE. The primers were designed using PrimerExplorer.
rfbE PCR primers ( BBa_K2495019 , BBa_K2495020 ) This is a set of PCR primers used for the amplification of rfbE. The primers were designed using Optigene.
Device expressing the putative O-antigen transporter under control of an arabinose inducible promoter( BBa_K2495001 )
Putative O-antigen transporter encoding sequence under the control of an arabinose inducible promoter and constitutive promoter. It is used to code for the O-antigen transporter characteristic of E.coli O157:H7 strain that produces Shiga-toxin.
Experimental Validation
Throughout our project, we successfully ligated our gene, rfbE to the pSB1C3 linearized backbone and transformed it into competent DH5α E. coli cells. The part was validated by running PCR on the gene fragment using primers BBa_K2495019 and BBa_K2495020 . These primers amplified a region of approximately 1000 bp. Appropriate bands were seen in Lane 3, which indicated that the part was successfully ligated and transformed. This transformed part was used later in our LAMP assays, and the results of our part in use can be seen here .
CheB chemotaxis coding sequence (protein glutamate methylesterase) ( BBa_C0024 )
This part was designed by MIT SBC 2004 and is a protein involved in the E. coli chemotaxis pathway. 2017 Team NYU Abu Dhabi ran an in silico characterization of this protein and found that it was extremely stable as a free enzyme in an explicit solvent solution mimicking the KCl concentrations of E. coli.
In Silico Validation
The MD simulations were run using GROMACS with the CHARMM36 forcefield. We ran an energy minimization for 500 ps using the steepest descent minimization algorithm with a stopping condition of maximum force less than 1000 kJ/mol/nm. We found that the energy was minimized in this time frame as shown by the steady convergence of the potential energy (Figure 1).
The minimized structure was solvated with water >12 A away from the protein surface. Potassium and chlorine atoms were added to neutralize the system and mimic experimental conditions of approximately 30 mM concentrations. We equilibrated in two phases, first under an NVT ensemble and then under an NPT ensemble. Both were found to equilibrate within 150 ps as shown in Figures 2 and 3, respectively. These adjusted the volume to 1 bar and temperature to 300K with restraints on the solutes.
The last frame of the NPT equilibration was extracted to start NVT simulations. A short animation of the protein run for 1 ns is attached. RMSD and radius of gyration analyses were run in order to determine the stability of the protein. The RMSD plot (Figure 4) shows the RMSD of the backbone does not fluctuate dramatically over the 1 ns run, indicating that the structure is stable. The 0.05 nm difference in backbone RMSD over the whole run indicate that the structure at 1 ns is slightly different from the initial structure, due to the energy minimization.
The radius of gyration, which measures a protein’s compactness, indicates that the protein is stably folded over the 1 ns production run. The plot shows reasonable variation and, along with the RMSD plots, show that this protein is very stable in explicit solvent solution (Figure 5).
glgC16 (glgC with G336D substitution) ( BBa_K118016 )
This part was designed by Edinburgh 2008 and is a protein from E.coli JM109 with the substitution G336D. 2017 Team NYU Abu Dhabi ran an in silico experiment of part BBa_K118016 which is the glgC (ADP-glucose pyrophosphorylase) from E. coli with mutation Gly336Asp.
In Silico Validation
The properties of this mutation were compared to the wild type by running MD simulations using GROMACS with the CHARMM36 forcefield. We ran an energy minimization for 500 ps using the steepest descent minimization algorithm with a stopping condition of maximum force less than 1000 kJ/mol/nm. We found that the energy was minimized in this time frame as shown by the steady convergence of the potential energy (Figure 1).
The minimized structures were solvated with water >12 A away from the protein surface. Potassium and chlorine atoms were added to neutralize the system and mimic experimental conditions of approximately 30 mM concentrations. We equilibrated in two phases, first under an NVT ensemble and then under an NPT ensemble. Both were found to equilibrate within 150 ps as shown in Figures 2 and 3, respectively. These adjusted the volume to 1 bar and temperature to 300K with restraints on the solutes.
The last frame of the NPT equilibration was extracted to start NVT simulations. A short animation of the protein run for 1 ns is attached. RMSD and radius of gyration analyses were run in order to determine the stability of the protein. The RMSD plot (Figure 4) shows the RMSD of the wildtype and G336D backbone does not fluctuate dramatically over the 1000 ps run, indicating that the structure is stable. The backbone of the G336D is overall more stable than the wildtype in the 1000 ps run. The increased stability may facilitate the increased activity of glgC in the absence of its activator observed during experimental tests.
The radius of gyration, which measures a protein’s compactness, indicates that both proteins are stably folded over the 1 ns production run. The plot shows reasonable variation and, along with the RMSD plots, show that both proteins are very stable in explicit solvent solution (Figure 5).
Analysis of the overlaid proteins showed only slight difference after alignment (Figure 6). Closeup of the mutation site shows that the longer side chain of aspartic acid causes the displacement of a loop due to steric hindrance with Leu318 (Figure 7). This displacement translates through the rest of the protein, accounting for some of the variance in RMSD.
cjBlue Green Coding Device ( BBa_K592022 )
This part was designed by Uppsala-Sweden 2011 and is a reporter that constitutively expresses the cjBlue chromoprotein. 2017 Team NYU Abu Dhabi tested this device by observing the development of color in agar and broth cultures.
Experimental Validation
2017 Team NYU Abu Dhabi attempted to express cjBlue in DH5α using composite part BBa_K592022 . Our assembly and transformation protocol were followed in order to obtain the part. All initial, intermediate, and final products obtained throughout the 3A assembly were digested with EcoRI-HF following our digestion protocol and run on a 1.2% agarose gel to confirm that the constructs were of the right size (Figure 1).
However, no blue colonies were observed after 24 hs of incubation in agar and no blue pellet was obtained after centrifuging an overnight LB broth culture at 7000 rpm for 5 min. The reason for the lack of expression remains elusive.
NYU Abu Dhabi is a research university with a fully integrated liberal arts and science college. It draws students from around the world, and prepares them for the challenges and opportunities of our interconnected world. Read More
NYUAD Saadiyat Island Abu Dhabi, P.O. Box 129188, U.A.E Email Facebook
