<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NYU_Abu_Dhabi_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NYU Abu Dhabi/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="app"><NAV class="navbar navbar-expand-lg fixed-top navbar-light bg-light"><DIV class="collapse navbar-collapse" id="navbarNavDropdown"><UL class="navbar-nav"><LI class="nav-item active"><A class="nav-link" href="https://2017.igem.org/Team:NYU_Abu_Dhabi">Home <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                Project
              </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Description">Our Project</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Results">Results</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Parts">Parts</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Demonstrate">Demo</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Applied_Design">Design</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Collaborations">Collaborations</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                Lab Notebook
              </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Biology">Biology</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Engineering">Engineering</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Protocols">Protocols</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                Human Practices
              </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/HP">Overview</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/HP/Silver">Human Practices</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/HP/Gold_Integrated">Integrated Human Practices</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Engagement">Engagement and Education</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/InterLab">Interlab</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                Team
              </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Team">Team</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Attributions">Attributions</A></DIV></LI></UL></DIV></NAV><DIV class="section-block"><DIV class="tab"><BUTTON class="tablinks" onclick="openTab(event, 'Concern')" id="defaultOpen">Concern</BUTTON><BUTTON class="tablinks" onclick="openTab(event, 'Project Description')">Project Description</BUTTON><BUTTON class="tablinks" onclick="openTab(event, 'iGEM NYU Abu Dhabi 2016')">iGEM NYU Abu Dhabi 2016</BUTTON><BUTTON class="tablinks" onclick="openTab(event, 'Medal Requirements')" id="defaultOpen">Medal Requirements</BUTTON></DIV><DIV id="Concern" class="tabcontent"><ARTICLE><H2 class="section-header">What is STEC?</H2><P class="section-content"><I>Escherichia coli</I> is part of the Enterobacteriaceae family, which includes a great diversity of Gram-negative rods and some of the most common etiologic agents of foodborne disease. Organisms of this family contain antigenic walls that allow the distinction of several serotypes within the same species. There are three types of antigens used in the classification of serotypes: O-antigens (composed by the outer lipopolysaccharide or LPS), K-antigens (capsule-forming polysaccharides), and H-antigens (flagellar proteins). The name of the serotype is given by the variant (a number) of each type of antigen next to its letter. Not all organisms contain the three types. 
</P><P class="section-content">
Shiga toxin-producing <I>Escherichia coli</I> (STEC) is a group of enterohemorrhagic strains of <I>E. coli</I> (EHEC) that are responsible for foodborne disease. The O157:H7 serotype is the most prevalent strain of STEC. <SUP>1</SUP></P><H2 class="section-header">Why is STEC a major concern?</H2><P class="section-content">

Globally, 1 in every 10 people is affected by foodborne diseases each year. In 2015, the WHO found that 600 million individuals were affected by such diseases and 420,000 died as a result (Figure 1). 
STEC causes over 70,000 infections per year in the United States alone. A portion of these individuals will experience kidney failure after 6 days, 50% of which will require renal replacement therapy.<SUP>2</SUP>
Countries of low- and middle-income are the most affected due to unsafe practices of food production and storage (Figure 2). </P><DIV id="caption"><FIGCAPTION><CENTER><B>Figure 1.</B> WHO epidemiological data of foodborne illnesses</CENTER></FIGCAPTION></DIV><CENTER><DIV id="caption"><FIGCAPTION><CENTER><B>Figure 2. </B>WHO prevalence map of foodborne illnesses</CENTER></FIGCAPTION></DIV><H2 class="section-header">What is the mode of action?</H2><P class="section-content">


Shiga toxin is an exotoxin of the AB type. The subunit B directs binding to a glycolipid that acts as a receptor in eukaryotic cells, globotriasylceramide (Gb3). Upon binding, the Shiga toxin is internalised by endocytosis. The endocytotic vacuole is transported to the trans-Golgi network, where subunit A exits through the vacuolar membrane and enters the cytoplasm. Then, this subunit enzymatically modifies the 28S rRNA of the 60S rRNA subunit by removing an adenine base. As a result, the binding of aminoacyl tRNA to the ribosome (dependent on elongation factor 1) is prevented, effectively blocking protein synthesis.<SUP>2</SUP> Shiga toxin also induces apoptosis (Figure 3).<SUP>3</SUP></P><DIV id="caption"><FIGCAPTION><CENTER><B>Figure 3.</B> Shiga-toxin's mode of action</CENTER></FIGCAPTION></DIV><H2 class="section-header">What are the symptoms of STEC infections?</H2><P class="section-content">

As an EHEC strain, STEC are capable of delivering virulence factors to epithelial cells in the surface of the colon through a contact secretion system. These factors alter the structure of the cytoskeleton of the eukaryotic cell, resulting in the formation of a pedestal where the bacterium sits. Moreover, the virulence factors trigger the expression of intimin receptors that tightly bind to the bacterium. It is thought that the deformation of the intestinal epithelium affects its absorptive functions, thus leading to diarrhea. In addition, the secretion of Shiga toxin causes capillary thrombosis and inflammation in the colonic mucosa,
 leading to the manifestation of blood in stool. In more severe cases, the circulation of Shiga toxin can cause hemolytic uremic syndrome (HUS), a life threatening disease characterized by renal failure, low platelet count, and haemolytic anemia.<SUP>4</SUP> In the US alone, this strain causes 70,000 infections per year. Even more worrying is the fact that 6-9% of those causes resulted in HUS. 5-7% of the individuals affected by HUS did not survive (Fig. 4).<SUP>5,6</SUP></P><DIV id="caption"><FIGCAPTION><CENTER><B>Figure 4.</B>Symptoms of STEC infection</CENTER></FIGCAPTION></DIV></CENTER></ARTICLE></DIV><DIV id="Project Description" class="tabcontent"><ARTICLE><H2 class="section-header">What are we doing?</H2><P class="section-content">
              Due to the geographic location of NYUAD, which is in close proximity to countries where food safety is a daily, life-threatening concern, we were inspired to design and deliver an efficient solution to reduce the severity of this issue. We have produced a rapid, affordable and portable device that allows for the detection of Shiga toxin-producing <I>Escherichia coli</I> using loop-mediated isothermal amplification (LAMP). 
            </P><H2 class="section-header">How are we doing it?</H2><P class="section-content">

LAMP is a highly specific, efficient and rapid DNA amplification technique that uses 4-6 primers that bind to 6-8 distinct regions of target DNA. This technique was <A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Results">shown</A> to be more specific than colony PCR without the need for heat lysis or centrifugation steps. Since the NYU Abu Dhabi iGEM team cares deeply about biosafety, the selectivity of our system was tested using the <I>rfbE</I> gene, a non-toxic coding sequence required for O157-antigen synthesis. A part containing this sequence was created and transformed into DH5α in order to obtain a strain that we could use in our trials. This way, we managed to test our device without employing a toxic strain of <I>E. coli</I>, ensuring the safety of our project and protecting the environment of the United Arab Emirates. </P><DIV id="caption"><FIGCAPTION><CENTER>Source: <A href="https://www.neb.com/IsothermalAmplification">New England Biolabs</A></CENTER></FIGCAPTION></DIV><P class="section-content">

The samples are contained in a PDMS chip that contains wells for positive control, negative control, and three samples. Heating is supplied by a Peltier Modular Cooling system. The reaction temperature of 65˚C was achieved using the Peltier system supplied with a 6V, 1.5A external power supply. The reaction was visualized under UV and blue light. It is envisioned that a smartphone will suffice for capturing the output of the reaction.<SUP>5</SUP></P><H2 class="section-header">Why E.coLAMP?</H2><DIV class="media reversed media-white"><DIV class="media-body reversed">
Our device offers a novel method to aid in pathogen surveillance by allowing every individual to become citizen scientists. With our easy-to-use device, which can be tailored for the detection of any known pathogen or virus, our device allows for the better collection of statistics to track and prevent potential outbreaks. This information can be combined on a cloud-based app that can be sorted to provide actionable data and allow public health officials and scientists to better understand the emergence and spread of diseases.

          </DIV></DIV><DIV class="media media-white"><DIV class="media-body">
        After surveying potential device users hailing from various parts of the world, we discovered that there was a shortage of quick and reliable methods to detect pathogenic <I>E. coli</I>. Most available methods are time- and resource-consuming, as they require the expertise of laboratory-trained technicians. More than 70% of the survey participants indicated the need of a method to detect pathogenic <I>E. coli</I> in less than 30 minutes. In response to this, we tailored our device to complete a diagnostic reaction within 20 minutes from the the beginning of sample submission to the attainment of the fluorescence signal.
          </DIV></DIV><DIV class="media reversed media-white"><DIV class="media-body reversed">
          There is a lack of stringent government regulations regarding food safety in many parts of the world. As most food vendor stalls are not subjected to government procedures and regulations about food safety, it is our priority to protect potential consumers from the possibility of ingesting harmful diseases or toxins. Our device aims to address this issue by allowing consumers to own a cost-effective, easy-to-use handheld device that can assess the safety of a product that they wish to buy. The results of our device can be easily recorded, exported to an Excel file, and uploaded to a public database, which can be accessed by anyone.

          </DIV></DIV><DIV class="media media-white"><DIV class="media-body">
        Foodborne diseases are often detected in undercooked meat, raw milk, and raw produce. In these scenarios, these pathogens may be unknowingly and exponentially spread throughout the food chain. In the food industry, it is important to prioritize food hygiene and prevent the sale of contaminated food to potential buyers. Our device is well-suited as a surveillance method for food vendors, ranging from small-scale individual sellers at traditional markets to large-scale supermarkets selling meat and produce as well as cooked food. Its rapid and easy method allows users from all kinds of background to interpret its results and discard the contaminated food accordingly. This device offers a simple solution to easily prevent and stem outbreaks.

          </DIV></DIV><DIV class="media reversed media-white"><DIV class="media-body reversed">
          As a proof of concept, we have acquired specific amplification of the target gene <I>rfbE</I> specific to STEC O157:H7 by using four LAMP primers, rather than the usual pair of primers in PCR. In the future, the LAMP primers can be designed to target the Shiga toxin <I>stx1</I> and <I>stx2</I> gene as well, which we deliberately chose to exclude in this project due to our consideration of the iGEM safety standards and of the health standards in the United Arab Emirates. We have also shown that our device is able to detect other pathogens by testing our device for the detection of malaria by targeting the <I>Plasmodium falciparum</I> mitochondrial cytochrome oxidase subunit 1 gene. The versatility of the technique and the design of the device allows for a broad range of diagnostic uses.

          </DIV></DIV><DIV class="media media-white"><DIV class="media-body">
By using a 12.5µL total reaction volume, we have shown that our lower limit of detection is 10<SUP>6</SUP> cells/mL. Improved limit of detection can also be attained by using a 25µL reaction volume. Additionally, our device only requires a swab of the food sample to attain such sensitivity.
          </DIV></DIV><DIV class="media reversed media-white"><DIV class="media-body reversed">
Another major concern for most diagnostic methods is the high cost per use. Problems arise when a device is accurate but expensive to use or, inversely, cost-effective but not reliable. We aim to solve both problems by bringing the price of the device as low as possible while still maintaining high sensitivity and accuracy. About 44% of the survey participants indicated that they would pay between $50-70 USD for this device. By optimising the materials and size of the device, we have currently reduced the price to $50 USD, with an additional $13 USD for the biological reagents needed to run the 5 reactions contained on each chip.
          </DIV></DIV></ARTICLE></DIV><DIV id="iGEM NYU Abu Dhabi 2016" class="tabcontent"><ARTICLE><H2 class="section-header">iGEM NYU Abu Dhabi 2016 Project</H2><H3 class="subheading">Award: Silver medal </H3><P class="section-content">
 In many developing countries, people depend on reasonably priced and conveniently available street food. However, lack of action taken by governments to regulate street food vendors has led to the prevalence of severe street food-related illnesses. One of the primary microbial contaminants in street food is <I>E. coli</I> O157:H7, which acts by secreting the Shiga-like toxin (SLT). Currently, there is no detection method for SLT outside of a lab setting, thus putting the consumers of foods at risk. Our project aims to develop a device that would be used by street vendors and restaurant owners to verify the safety of their products. Through our device, we exploit the binding of Gb3 to subunit B of the Shiga toxin, and compare the migration pattern of the bound Gb3-subunit B complex to a non bound subunit B. A shift in the migration pattern on a PAGE gel will occur when Gb3 is bound, indicating the presence of the toxin in the food sample. If no shift occurs in the SLT migration pattern, this implies the absence of the toxin within the sample, and reflects the safety status of the food.<SUP>7</SUP></P><H2 class="section-header">Our improvements</H2><P class="section-content">
Shiga toxin is an exotoxin that consists of two subunits. Subunit B binds to Gb3 receptor expressed in the surface of target cells and permits the entry of subunit A, which inhibits protein synthesis.<SUP>4</SUP> 2016 Team NYU Abu Dhabi exploited the binding of Gb3 to subunit
                B to detect for the presence of STEC. Their prototype compared the migration pattern of a bound Gb3-subunit
                B complex to that of free subunit B using a PAGE gel. Their device was estimated to take 45 minutes and their
                prototyping process ran into several issues that negatively impacted the specificity, affordability, and accessibility of the product. However, based on feedback from food vendors, we discovered that very few individuals would wait for so long to obtain results. Since they also had issues with expressing their protein of interest, we decided to target a DNA sequence for our device.
            </P></ARTICLE></DIV><DIV id="Medal Requirements" class="tabcontent"><ARTICLE><DIV class="media media-white"><DIV class="media-body">✔<FONT color="#b36200">Register/attend:</FONT> Registered for iGEM and will be attending the Giant Jamboree. ✔<FONT color="#b36200"> Team wiki:</FONT> Created a wiki page and documented our project. ✔<FONT color="#b36200">Project Attribution:</FONT> Recorded and attributed the work done for our project (<A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Attributions">Read more</A>)  . ✔<FONT color="#b36200"> Team Poster:</FONT> Created a poster and are waiting to present it in the Giant Jamboree. ✔<FONT color="#b36200"> Team presentation:</FONT> Plan to present our project during the Giant Jamboree. ✔<FONT color="#b36200"> Safety Forms:</FONT> Have submitted the safety forms. ✔<FONT color="#b36200"> Judging Forms:</FONT> Have submitted the judging form. ✔<FONT color="#b36200"> Registry Part Pages:</FONT> Have created and documented the part pages (<A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Parts">Read more</A>). ✔<FONT color="#b36200"> Sample Submission:</FONT> Submitted the DNA samples of our parts (<A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Parts">Read more</A>). ✔<FONT color="#b36200"> Characterization/contribution:</FONT> Participated in the Interlab Measurement Study and improved the characterization of three existing BioBrick Parts and entered it on that part's Main Page in the Registry. </DIV></DIV><DIV class="media media-white"><DIV class="media-body">✔<FONT color="#b36200"> Validated Part:</FONT> Validated and documented <I>rfbE</I> gene for O157 expression<A href="http://parts.igem.org/Part:BBa_K2495001">(BBa_K2495001)</A> (<A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Parts">Read more</A>) ✔<FONT color="#b36200"> Collaboration: </FONT> Worked with other registered iGEM teams: participated in Team Dusseldorf-Cologne postcard campaign, organized a virtual conferences with Teams Vilnius-Lithuania and Groningen, participated in variety of other Team’s surveys. (<A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Collaborations">Read more</A>) ✔<FONT color="#b36200">  Human Practices: </FONT> We hosted the first iGEM workshop for high schools students in the UAE, published a magazine <I>Synthetic Biology 101</I>, helped and advised a student from the NYU New York campus on how to start their own iGEM team, and developed a program that allows users to facilitate information sharing regarding their detection results. (<A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/HP/Silver">Read more</A>) </DIV></DIV><DIV class="media media-white"><DIV class="media-body">✔<FONT color="#b36200">  Integrated Human Practices: </FONT> We surveyed potential consumers of different backgrounds: (1) food vendors from Pakistan and Indonesia, and (2) individuals from our international student body and modified our eco-friendly device according to the survey results. Their feedback was integral in the development of our heating method and chip design. Their genuine interest in our device led us to pitch our idea to startAD, an innovation and entrepreneurship platform for start-ups in the UAE, in order to meet the initial consumer demand. (<A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/HP/Gold_Integrated">Read more</A>)✔<FONT color="#b36200">   Improve a previous part or project: </FONT> Team NYU Abu Dhabi 2017 improved upon the Team NYU Abu Dhabi 2016's project and created a working device that detects the presence of Shiga toxin. (<A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Description">Read more</A>)✔<FONT color="#b36200">   Demonstrate your work: </FONT> We created a video that demonstrates our working project. (<A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Demonstrate">Read more</A>)</DIV></DIV></ARTICLE></DIV></DIV><P class="section-content">
1. Ryan, K. J., Ray, C. G. (2003) <I>Sherris Medical Microbiology</I> (McGraw-Hill). 
2. Borgatta, B.; Kmet-Lunaček, N.; Rello, J., <I>E. coli</I> O104:H4 outbreak and haemolytic–uraemic syndrome. <I>Medicina Intensiva (English Edition)</I> 2012, 36 (8), 576-583.
3. Tesh, V.L. (2010) Induction of apoptosis by Shiga toxins. <I>Future microbiology</I> 5:431-453. 
4. Pacheco, A. R., Sperandio, V., Shiga toxin in enterohermorrhagic <I>E. coli</I>: regulation and novel anti-virulence strategies. <I>Front. Cell. Infect. Microbiol.</I> 2012, 2 (81), 1-12. 
5. Borgatta, B.; Kmet-Lunaček, N.; Rello, J., <I>E. coli</I> O104:H4 outbreak and haemolytic–uraemic syndrome. <I>Medicina Intensiva (English Edition)</I> 2012, 36 (8), 576-583. 
6. &quot;Virulence factors of entertoxigenic <I>E. coli</I> from Epidemiology of Infectious Diseases. Available at: http://ocw.jhsph.edu. Copyright © Johns Hopkins Bloomberg School of Public Health. Creative Commons BY-NC-SA. 
7. Team NYU Abu Dhabi - 2016. Available at: https://2016.igem.org/Team:NYU-AD.</P></DIV><DIV class="footer"><DIV class="container-fluid"><DIV class="row"><DIV class="col-lg-4 footer-section"><P class="footer-text">
              NYU Abu Dhabi is a research university with a fully integrated liberal arts and science college. It draws students from around the world, and prepares them for the challenges and opportunities of our interconnected world. <A href="http://nyuad.nyu.edu/en/about.html">Read More</A></P></DIV><DIV class="col-lg-4 footer-section"><P class="footer-text">
              NYUAD Saadiyat Island 
              Abu Dhabi, P.O. Box 129188, U.A.E <A href="mailto:nyuad.igem@nyu.edu">Email</A><A href="https://www.facebook.com/iGEMNYUAD/" target="_blank">Facebook</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>