<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NYU_Abu_Dhabi_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NYU Abu Dhabi/Parts</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="app"><NAV class="navbar navbar-expand-lg fixed-top navbar-light bg-light"><DIV class="collapse navbar-collapse" id="navbarNavDropdown"><UL class="navbar-nav"><LI class="nav-item active"><A class="nav-link" href="https://2017.igem.org/Team:NYU_Abu_Dhabi">Home <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                Project
              </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Description">Our Project</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Results">Results</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Parts">Parts</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Demonstrate">Demo</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Applied_Design">Design</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Collaborations">Collaborations</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                Lab Notebook
   </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Biology">Biology</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Engineering">Engineering</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Protocols">Protocols</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                Human Practices
              </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/HP">Overview</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/HP/Silver">Human Practices</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/HP/Gold_Integrated">Integrated Human Practices</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Engagement">Engagement and Education</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/InterLab">Interlab</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                Team
              </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Team">Team</A><A class="dropdown-item" href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Attributions">Attributions</A></DIV></LI></UL></DIV></NAV><DIV class="section-block"><DIV class="tab"><BUTTON class="tablinks" onclick="openTab(event, 'Basic Parts')" id="defaultOpen">Basic Parts</BUTTON><BUTTON class="tablinks" onclick="openTab(event, 'Composite Parts')">Composite Parts</BUTTON><BUTTON class="tablinks" onclick="openTab(event, 'Characterized Parts')">Characterized Parts</BUTTON></DIV><DIV id="Basic Parts" class="tabcontent"><ARTICLE><P class="section-content"><B><I>rfbE</I> gene for O157 expression</B><A href="http://parts.igem.org/Part:BBa_K2495000">(BBa_K2495000)</A>
O-antigen transporter encoding sequence for <I>E.coli</I>. Essential for <I>E.coli</I> O157 expression and can be used as a safe alternative to testing for the presence of Shiga toxin.
	</P><P class="section-content"><B><I>rfbE</I> LAMP primers - set 1</B>  
(<A href="http://parts.igem.org/Part:BBa_K2495002">BBa_K2495002</A>, <A href="http://parts.igem.org/Part:BBa_K2495003">BBa_K2495003</A>, 
<A href="http://parts.igem.org/Part:BBa_K2495004">BBa_K2495004</A>, <A href="http://parts.igem.org/Part:BBa_K2495005">BBa_K2495005</A>, 
<A href="http://parts.igem.org/Part:BBa_K2495006">BBa_K2495006</A>, <A href="http://parts.igem.org/Part:BBa_K2495007">BBa_K2495007</A>) 
This is the first set of LAMP primers used for the amplification of <I>rfbE</I>. The primers were designed using PrimerExplorer.
	</P><P class="section-content"><B><I>rfbE</I> LAMP primers - set 2</B>  
(<A href="http://parts.igem.org/Part:BBa_K2495008">BBa_K2495008</A>, <A href="http://parts.igem.org/Part:BBa_K2495009">BBa_K2495009</A>, 
<A href="http://parts.igem.org/Part:BBa_K2495010">BBa_K2495010</A>, <A href="http://parts.igem.org/Part:BBa_K2495011">BBa_K2495011</A>, 
<A href="http://parts.igem.org/Part:BBa_K2495012">BBa_K2495012</A>, <A href="http://parts.igem.org/Part:BBa_K2495013">BBa_K2495013</A>) 
This is the second set of LAMP primers used for the amplification of <I>rfbE</I>. The primers were designed using PrimerExplorer. 
	</P><P class="section-content"><B><I>rfbE</I> LAMP primers - set 3</B>   
(<A href="http://parts.igem.org/Part:BBa_K2495014">BBa_K2495014</A>, <A href="http://parts.igem.org/Part:BBa_K2495015">BBa_K2495015</A>, 
<A href="http://parts.igem.org/Part:BBa_K2495016">BBa_K2495016</A>, <A href="http://parts.igem.org/Part:BBa_K2495017">BBa_K2495017</A>) 
This is the third set of LAMP primers used for the amplification of <I>rfbE</I>. The primers were designed using PrimerExplorer. 
	</P><P class="section-content"><B><I>rfbE</I> PCR primers</B>   (<A href="http://parts.igem.org/Part:BBa_K2495019">BBa_K2495019</A>,<A href="http://parts.igem.org/Part:BBa_K2495020">BBa_K2495020</A>) 

This is a set of PCR primers used for the amplification of <I>rfbE</I>. The primers were designed using Optigene.
      </P></ARTICLE></DIV><DIV id="Composite Parts" class="tabcontent"><ARTICLE><H3 class="subheading">Device expressing the putative O-antigen transporter under control of an arabinose inducible promoter(<A href="http://parts.igem.org/Part:BBa_K2495001">BBa_K2495001</A>) </H3><P class="section-content">

Putative O-antigen transporter encoding sequence under the control of an arabinose inducible promoter and constitutive promoter. It is used to code for the O-antigen transporter characteristic of <I>E.coli</I> O157:H7 strain that produces Shiga-toxin.
</P><P class="section-content"><B> Experimental Validation </B>
Throughout our project, we successfully ligated our gene, <I>rfbE</I> to the <I>pSB1C3</I> linearized backbone and transformed it into competent DH5α <I>E. coli</I> cells. The part was validated by running PCR on the gene fragment using primers <A href="http://parts.igem.org/Part:BBa_K2495019">BBa_K2495019</A> and <A href="http://parts.igem.org/Part:BBa_K2495020">BBa_K2495020</A>. These primers amplified a region of approximately 1000 bp. Appropriate bands were seen in Lane 3, which indicated that the part was successfully ligated and transformed. This transformed part was used later in our LAMP assays, and the results of our part in use can be seen <A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Results">here</A>.
</P></ARTICLE></DIV><DIV id="Characterized Parts" class="tabcontent"><ARTICLE><H3 class="subheading">CheB chemotaxis coding sequence (protein glutamate methylesterase) (<A href="http://parts.igem.org/Part:BBa_C0024">BBa_C0024</A>) </H3><P class="section-content">
 
This part was designed by MIT SBC 2004 and is a protein involved in the <I>E. coli</I> chemotaxis pathway. 2017 Team NYU Abu Dhabi ran an <I>in silico</I> characterization of this protein and found that it was extremely stable as a free enzyme in an explicit solvent solution mimicking the KCl concentrations of <I>E. coli</I>. 	
</P><P class="section-content"><B><I>In Silico</I> Validation</B>
The MD simulations were run using GROMACS with the CHARMM36 forcefield. We ran an energy minimization for 500 ps using the steepest descent minimization algorithm with a stopping condition of maximum force less than 1000 kJ/mol/nm. We found that the energy was minimized in this time frame as shown by the steady convergence of the potential energy (Figure 1). 
The minimized structure was solvated with water &gt;12 A away from the protein surface. Potassium and chlorine atoms were added to neutralize the system and mimic experimental conditions of approximately 30 mM concentrations. We equilibrated in two phases, first under an NVT ensemble and then under an NPT ensemble. Both were found to equilibrate within 150 ps as shown in Figures 2 and 3, respectively. These adjusted the volume to 1 bar and temperature to 300K with restraints on the solutes. 
The last frame of the NPT equilibration was extracted to start NVT simulations. A short animation of the protein run for 1 ns is attached. RMSD and radius of gyration analyses were run in order to determine the stability of the protein. The RMSD plot (Figure 4) shows the RMSD of the backbone does not fluctuate dramatically over the 1 ns run, indicating that the structure is stable. The 0.05 nm difference in backbone RMSD over the whole run indicate that the structure at 1 ns is slightly different from the initial structure, due to the energy minimization. 
The radius of gyration, which measures a protein’s compactness, indicates that the protein is stably folded over the 1 ns production run. The plot shows reasonable variation and, along with the RMSD plots, show that this protein is very stable in explicit solvent solution (Figure 5). </P><H3 class="subheading">glgC16 (glgC with G336D substitution) (<A href="http://parts.igem.org/Part:BBa_K118016">BBa_K118016</A>) </H3><P class="section-content">
This part was designed by Edinburgh 2008 and is a protein from <I>E.coli</I> JM109 with the substitution G336D. 2017 Team NYU Abu Dhabi ran an <I>in silico</I> experiment of part BBa_K118016 which is the glgC (ADP-glucose pyrophosphorylase) from <I>E. coli</I> with mutation Gly336Asp. 
</P><P class="section-content"><B><I>In Silico</I> Validation</B>


The properties of this mutation were compared to the wild type by running MD simulations using GROMACS with the CHARMM36 forcefield. We ran an energy minimization for 500 ps using the steepest descent minimization algorithm with a stopping condition of maximum force less than 1000 kJ/mol/nm. We found that the energy was minimized in this time frame as shown by the steady convergence of the potential energy (Figure 1). 
The minimized structures were solvated with water &gt;12 A away from the protein surface. Potassium and chlorine atoms were added to neutralize the system and mimic experimental conditions of approximately 30 mM concentrations. We equilibrated in two phases, first under an NVT ensemble and then under an NPT ensemble. Both were found to equilibrate within 150 ps as shown in Figures 2 and 3, respectively. These adjusted the volume to 1 bar and temperature to 300K with restraints on the solutes.
The last frame of the NPT equilibration was extracted to start NVT simulations. A short animation of the protein run for 1 ns is attached. RMSD and radius of gyration analyses were run in order to determine the stability of the protein. The RMSD plot (Figure 4) shows the RMSD of the wildtype and G336D backbone does not fluctuate dramatically over the 1000 ps run, indicating that the structure is stable. The backbone of the G336D is overall more stable than the wildtype in the 1000 ps run. The increased stability may facilitate the increased activity of glgC in the absence of its activator observed during experimental tests.
The radius of gyration, which measures a protein’s compactness, indicates that both proteins are stably folded over the 1 ns production run. The plot shows reasonable variation and, along with the RMSD plots, show that both proteins are very stable in explicit solvent solution (Figure 5).
Analysis of the overlaid proteins showed only slight difference after alignment (Figure 6). Closeup of the mutation site shows that the longer side chain of aspartic acid causes the displacement of a loop due to steric hindrance with Leu318 (Figure 7). This displacement translates through the rest of the protein, accounting for some of the variance in RMSD.</P><H3 class="subheading">cjBlue Green Coding Device (<A href="http://parts.igem.org/Part:BBa_K592022">BBa_K592022</A>)</H3><P class="section-content">
This part was designed by Uppsala-Sweden 2011 and is a reporter that constitutively expresses the cjBlue chromoprotein. 2017 Team NYU Abu Dhabi tested this device by observing the development of color in agar and broth cultures.
</P><P class="section-content"><B>Experimental Validation</B>
2017 Team NYU Abu Dhabi attempted to express cjBlue in DH5α using composite part <A href="http://parts.igem.org/Part:BBa_K592022">BBa_K592022</A>. Our assembly and transformation <A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Protocols">protocol</A> were followed in order to obtain the part. All initial, intermediate, and final products obtained throughout the 3A assembly were digested with EcoRI-HF following our digestion <A href="https://2017.igem.org/Team:NYU_Abu_Dhabi/Protocols">protocol</A> and run on a 1.2% agarose gel to confirm that the constructs were of the right size (Figure 1).
However, no blue colonies were observed after 24 hs of incubation in agar and no blue pellet was obtained after centrifuging an overnight LB broth culture at 7000 rpm for 5 min. The reason for the lack of expression remains elusive.
</P></ARTICLE></DIV></DIV><DIV class="footer"><DIV class="container-fluid"><DIV class="row"><DIV class="col-lg-4 footer-section"><P class="footer-text">
              NYU Abu Dhabi is a research university with a fully integrated liberal arts and science college. It draws students from around the world, and prepares them for the challenges and opportunities of our interconnected world. <A href="http://nyuad.nyu.edu/en/about.html">Read More</A></P></DIV><DIV class="col-lg-4 footer-section"><P class="footer-text">
              NYUAD Saadiyat Island 
              Abu Dhabi, P.O. Box 129188, U.A.E <A href="mailto:nyuad.igem@nyu.edu">Email</A><A href="https://www.facebook.com/iGEMNYUAD/" target="_blank">Facebook</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>