<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UFAM-UEA_Brazil_Project_NewPromoters skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UFAM-UEA Brazil/Project/NewPromoters</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="main-container"><DIV class="main-content" id="t1"><H1 class="h-sectionT">New Promoters Library</H1><H2 style="background:#fff;margin-bottom:0.5em; color:#000"><CENTER><B>Description</B></CENTER></H2><P id="t2">Transcription is the first step of genetic expression and thus is one of greater and important processes of life. Promoters are DNA sequences that controls the transcription initiation and mRNA production of all genes, recognized and synthetized, respectively, by the enzyme RNA polymerase. </P><P id="t2">Different regions for interaction of the promoter with RNA polymerase were described as represented in the figure 1 bellow, such as: up element, sequences of -35 and -10 hexamer, extended -10 element, discriminator. </P><P> Figure 1 – Specific sequences for interaction between promoters sequences and RNA polymerase </P><P id="t2">Together, these regions specify the RNA polymerase recognition and interaction to the promoter. Furthermore, two others elements control the transcription of certain gene or set of genes: the repressor proteins and its operators. Many steps of this process can be also highly regulated by accessory protein factors and small ligands, temperature, salt and solute concentrations, and other environmental variables.  </P><H2 style="background:#fff;margin-bottom:0.5em; color:#000"><CENTER><B>Mer Operon regulation </B></CENTER></H2><P>Molecular structure of MerR protein is constituted of 144 individuals amino acids that differ in nine type of residues. Structurally, the regulator presents itself as an homodimer with mass equivalent to 31 kDa. The C and N-terminal extremities are intimately connected in MerR role as regulator. C-terminal is the Hg binding site, mediated by three cysteine residues. Then, N-terminal is linked to the operator (MerO), between -35 and -10 regions, so the recognition site remains inaccessible to RNA polymerase and genes transcription does not begin.</P><P>MerR is able to attract RNA polymerase to the promoter region, even in absence of Hg(II). The pre-transcriptional complex merR-polymerase remains steady and not producing until Hg(II) is bound to merR’s C site, causing an allosteric change in the protein’s structure. This is propagated to the DNA strain, which unwinds and favors RNA polymerase binding to the promoter, and transcription starts. The formation of pre-transcriptional complex allows quick answer to Hg(II) presence in the medium. The regulatory mechanism of merR and activation of Mer Operon is illustrated below:</P><P><B>Figure 01:</B>  Transcription activation of Mer Operon. A) Pre-transcriptional complex, composed by merR, RNA polymerase and the others transcription factors: merR conformity does not allows RNA polymerase recognition, thus Mer Operon remains off. B)  Hg binding to terminal C of the regulator promotes conformational changes in the protein and allows RNA polymerase to starts transcription. </P><P id="t2">This unique system of genetic regulated expression sensible to the mercury absence and presence, inactivating and activating, inspired us to design new promoter sequences, merging the well documented and widely used promoter sequences with mer operator. We designed 8 new promoters, the first and only collection of MerR regulated promoters, sensible to mercury, of iGEM!</P><P id="t2">The new designed promoters were merged with Tac promoter and JK26 promoter, promoters that interact with RNA polymerase on exponential and stationary phase of bacterial growth. Check out the synthetic genetic circuits on the videos below.</P><P id="t2">We characterized its efficiency through the RFP regulated expression! Check this out in results!</P><H2 style="background:#fff;margin-bottom:0.5em; color:#000"><CENTER><B>Results</B></CENTER></H2><P>Our team (UFAM-UEA_Brazil) worked improving the Mer operon expression to increase bioremediation in E. coli through novel mer promoters sequences. For it, we primarily characterized the MerR expression under control of different promoters from Anderson Collection (BBa_J23100, BBa_J23104, BBa_J23106 e BBa_I142033) through the repression of RFP (BBa_K081014) production, in a synthetic genetic circuit represented bellow.   </P><CENTER><P><B>Figure 02:</B> MerR expression test. </P></CENTER><P>On the first experiment, we used the novel regulated promoter designed by our team BBa_K2123109 (Stationary growth phase promoter with downstream mer operator) to measure the RFP repression by MerR under control of BBa_J23100, BBa_J23104, BBa_J23106 e BBa_I142033 constitutive promoters in solid LB media, as you can see below. </P><CENTER><B><P>Figure 03:</P></B> MerR repressing RFP production in different levels cloned in E. coli DH5-alpha.</CENTER><P>You can analyse by the RFP expression and thus fluorescence intensity that the samples greater repressed by MerR was the ones under control of BBa_J23100 and BBa_I14033 constituve promoters. So, we measured the RFP expression using Chamaleon Spectrofluorometer with and without MerR repressor protein under control of these two constituve promoter. The results are presented in the graph 1 below.	</P><P><B>Graph 1:</B> RFP expression with and without MerR regulator under control of BBa_J23100 and BBa_I14033. Control bacteria is DH5-alpha without any plasmid vector. </P><P>As we can see, between BBa_J23100 and BBa_I14033, the best repressed was with BBa_J23100 constitutive promoter. So, we selected BBa_J23100 to controls MerR expression in our synthetic genetic circuits. We made also other experiments to understand the interactions between MerR and new regulated promoters designed by our team, aiming to reach a well repressor mechanism, increasing the natural MerR regulation.</P><P>To characterize the intensity of the promoters library expression, RFP (BBa_K081014) was linked to them, as reporter gene. Fluorescence tests were performed in parallel to growth curves. From these promoters, the ones who stood out were BBa_K2123006 and BBa_K2123007, but the most well regulated one, chosen to compose Mer Operon, was BBa_K2123004.</P><P>Despite not composing Mer Operon, the regulated promoter whose regulation stood out was BBa_K2123001, with overlap operator. At fluorescence tests, comparing to the control, 99% of RFP expression was inhibited, thanks to the design which overlapped not only to spacing, but the isomers -10 and -35, complicating, or almost obstructing, the interaction between RNA polymerase and promoter.</P><P>Another promoter which exceled during inhibition tests was BBa_K2123002, in which design between two main isomers, showed 92% inhibtion of RFP expression when compared to control.</P><P>To finalize this part characterization, we cloned it with the Mer Operon improved genes device (as the figure aiming to measure the amoung of mercury bioremediated under control of our MerR with very strong regulation in 7.5ppm of mercury chloride with 10 hours of growth. </P><P>We used two new regulated promoters to compare with natural one. The first test was validated the growth curve in 7.5ppm of HgCl2 in liquid LM media.  The result are shown below. </P><P>In the growth tests we used 7,5 µg/mL of mercury, comparing with the device K1355004 (used in 2014) and the new construction. At the best take, the new construction showed  4,6 times better result than the previous device. </P><P>The curve from the new construction under our bioreactor conditions had an incredible maintenance of its growth rhythm under 5µg/mL mercury concentration in a 42 hours period. 
In posterior analysis, mercury was reduced in 97% from its initial concentration, proving the efficiency of this new construction and its potential to depollute contaminated waters.</P><H2 style="background:#fff;margin-bottom:0.5em; color:#000"><CENTER><B>References</B></CENTER></H2><P>1.	LODISH, H.; BERK, A.; MATSUDAIRA, P. et al. Biologia Celular e Molecular. 5. ed. Porto Alegre: Artmed, 2005. </P><P>2.	NELSON, D. L.; COX, M. M. Lehninger Princípios de Bioquímica. 3 ed. São Paulo: Sarvier, 2002. </P><P>3.	KHORANA, H. G; AGARWALS, K. L.; BESMER, P. et al. Total Synthesis of the Structural Gene for the Precursor of a Tyrosine Suppressor Transfer RNA from Escherichia coli. The Journal Of Biological Chemistry. Vol 251. Cambrigde, MT, 1975.
</P><P>4.	DEHASETH, P. L., ZUPANCIC, M. L.; RECORD, M. T. RNA Polymerase-Promoter Interactions: the Comings and Goings of RNA Polymerase. Journal Of Bacteriology Vol. 180. Madison, Wisconsin, 1998.</P><P>5.	GREEN, M.R. e SAMBROOK, J. Molecular Cloning: A Laboratory Manual. Cold Spring Harbour Laboratory Press, 4ª Ed. Cold Spring Harbour, USA, 2012. </P><P>6.	COHEN, S. N.; CHANG, A. C. Y.; HSU, L. Non chromosomal Antibiotic Resistance in Bacteria: Genetic Transformation of Escherichia coli by R-Factor DNA. Proc. Nat. Acad. Sci. Vol. 69. USA, 1972.</P><P>7.	SHETTY, R. P.; ENDY, D.; KNIGHT, T. F. Engineering BioBrick vectors from BioBrick parts. Journal of Biological Engineering. Cambridge, MA, USA, 2008.</P><P>8.	DASH, H. R.; DAS, S. Bioremediation of mercury and the importance of bacterial mer genes International Biodeterioration &amp; Biodegradation. Orissa, India, 2012.</P><P>9.	MIKSCH, G.; BETTENWORTH, F.; FRIEHS, L et al. The sequence upstream of the -10 consensus sequence modulates the strength and induction time of stationary-phase promoters in Escherichia coli. Appl Microbiol Biotechnol Vol. 69., 2005.  
</P><P>10.	BOER, H. A.; COMSTOCK, L. J.; VASSER, M.  The tac promoter: A functional hybrid derived from the trp and lac promoters. Proc. NatL Acad. Sci Vol. 80, USA 1983.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>