<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UFAM-UEA_Brazil_Project_Phytochelatin skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UFAM-UEA Brazil/Project/Phytochelatin</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="main-container"><DIV class="main-content" id="t1"><H1 class="h-sectionT">Phytochelatin</H1><H2 style="background:#fff;margin-bottom:0.5em; color:#000"><CENTER><B>Description</B></CENTER></H2><NAV class="nav-page"><UL><LI class="page-item"><A href="#t1" style="color:#368E8C;">Description</A></LI><LI class="page-item"><A href="#t2">Results</A></LI><LI class="page-item"><A href="#t3">Notebook</A></LI></UL></NAV><P>In the environment, Hg has successive transformations which pose risks not only for microorganisms but also to macrofauna. However, it's known that some bacteria species has mercury resistance and are able of bioaccumulation, among them <I>Serratia marcescens</I>, <I>Pseudomonas putida</I>, <I>Cupriavidus metallidurans</I> and <I>Entereobacter</I>. Bacterial resistance to mercury occurs due to proteins that can act in Hg capture. Among those metal-chelating proteins, there is phytochelatin. </P><P>Phytochelatin can be found in the bioaccumulation system of plants with long enzymatic pathways and precursor molecules for their expression. That protein has as main feature the interaction with heavy metals which occurs due to the great amount of cysteine amino acid present. (BAE; MEHRA, 1997; ) </P><P>The use of natural membrane proteins is already in place and serve as a tool to anchor heterologous proteins in a system called “cell surface display”. It presents a great potential for a variety of biotech uses. By this strategy target peptides could be anchored to antibodies production, catalyzers, bioremediation and other uses. In heavy metal bioremediation, its is showed that recombinant microorganisms with modified surface, enriched with metal chelate proteins are better to cope the adsorption of metallic ions. </P><P>There are several strategies to anchor peptides in the bacterial membrane. In this project we used the most abundant protein to do so, the <I>E. coli </I>outer membrane protein (OmpA) – fused with synthetic phytochelatin to bioremediation of mercury metal, as represented in the image below. In 2000 a new series of peptides serving as heavy metal adsorbents was proposed by Bae and collaborators. The strategy consisted in the use of an analogous to a natural phytochelatin without the necessity of post-translational modifications to work without using enzymatic routes or precursor molecules to its, in other words: a gene a protein. </P><P style="font-size:0.8em; text-align: center;"><I>Phytochelatin model.</I></P><P>In the previous competition, we tested the efficiency of <A href="https://2014.igem.org/Team:UFAM_Brazil/Bioaccumulation" target="blank" style="color:#40FF00">Metal Binding Peptide</A>. In this project, we design a system for Hg bioaccumulation by a synthetic phytochelatin anchored in the membrane protein OmpA of a host bacteria. </P><P>One of the pillars of our project is to design the sequences for our genetic modified systems and for that the most challenging step is to adjust preferential codons for our chassis. Glutamic acid (E) and cysteine (C) show two different codons in <I>E. coli</I>: GAG and GAA for E; TGC and TGT for C. GAA 70% e códon GAG 30%; códon TGC 60% e códon TGT 40%.</P><P>Our synthetic phytochelatin (EC20 - Glu-Cis) optimized to E. coli has: I) 10 cysteine amino acids being 12 codons TGC and 8 codons TGT; II) 10 glutamic acid amino acids, 14 codons GAA and 6 codons GAG. As described in the image below. </P><P style="font-size:0.8em; text-align: center;"><I>Codon composition of our synthetic phytochelatin.</I></P><P>With the phytochelatin designed we decided to express it in membrane cell display. For that we incorporated the followed parts: </P><P>With the phytochelatin designed we decided to express it in membrane cell display. For that we incorporated the followed parts: </P><P>Promoter JK26 interacting with sigma factor RpoS or sigma factor S to express in the stationary phase in the cell growth. JK26 is a promoter for late phase (lag) described by Miksch et al in 2005 and has been the strongest one we tested. </P><P>Important to say in this construction, the promoter JK26 with NdeI site was calculated to - when bound to the Llp-OmpA-phytochelatin + terminator – have 9 base pairs in distance between Shine-Dalgarno and the mRNA initiation. </P><P>Lpp-OmpA (BBa_K103006) <I>E. coli</I> membrane protein is where our phytochelatin was bound; The double transcription terminator is the union of T1 from <I>E. coli</I> and TE from T7, denominated as BBa_B0015, available in the Registry. </P><P id="t2"><LABEL class="notebook-link" for="toggleis">In silico Study</LABEL></P><DIV id="is"><P>In silico from Software Snapgene: </P><P>To achieve this final construction we followed this cloning strategy: </P><P>1st Step: </P><P>JK26 promoter and Lpp-OmpA digestion with NdeI and XbaI, to linearize Lpp-OmpA (2526 bp) and release the promoter in insert (~83 bp) form. Then, they’ll be linked. </P><P>2nd Step: </P><P>Phytochelatin digestion with EcoRI + SpeI and release of insert (148 bp). The terminator was digested with EcoRI + XbaI (2526 bp). Then, these parts were linked. </P><P>3rd Step: </P><P>Digestion of first step construction, resulting the PO construction (JK26 promoter + Lpp-OmpA) digested with EcoRI + SpeI, releasing a 564 bp insert. The PT (Phytochelatin + Terminator) construction was linearized with EcoRI + XbaI, releasing insert of approximately 2317 bp. </P><P>Finally we had our bioaccumulator construction!! </P><P>If you want to know how it went in the lab access our Phytochelatin notebook.</P></DIV><H2 style="background:#fff;margin-bottom:0.5em; color:#000"><CENTER><B>Results</B></CENTER></H2><NAV class="nav-page"><UL><LI class="page-item"><A href="#t1">Description</A></LI><LI class="page-item"><A href="#t2" style="color:#368E8C;">Results</A></LI><LI class="page-item"><A href="#t3">Notebook</A></LI></UL></NAV><P>To determine our construction resistance in DH5-alpha transformed with BBa_K2123302. To make plates, the transforming clone with the BBa_K2123302 construction was selected and pre-inoculated alongside a random sample as a negative control with its respective antibiotics. After the growth, the optical density was analyzed intending to standardize the density with the control.</P><P>Solid LM medium (LB with a low concentration of NaCl) used in the plates without adding antibiotic. The pre-inoculated samples were plated and the filter paper disks were put to further add HgCl2 solution at 2000ppm.
The samples were incubated at 37ºC for 16 hours.
</P><P>The graph represented on Figure 1 shows that our construction with phytochelatin is 41% more resistant than our control!! As shown in Figure 2 the measurement from the center of the paper filter shows that our construction is capable of getting nearest to the disc than our control which supports our Bioaccumulation device.</P><P>With a better notion to which levels of mercury our cell can grow, we determined the concentration to be used in the experiment for our in-depth bioaccumulation characterization. Made with BBa_K2123302 transformed in DH5-alpha and inoculated in LM (LB with a low concentration of NaCl) liquid medium with chloramphenicol and overnight growth.
Then, an aliquot of 100μl was taken and inoculated in three Erlenmeyer flask with 50 ml of LM.The samples were incubated at 37°C at 150 rpm on a shaker, and the optical density was measured every hour until it presented 0.6abs (measured on a spectrophotometer at 600 nm wavelength). At that point, 5 ppm of HgCl2 solution was added to the sample. After 8 hours of growth, the three of them were centrifuged at 12000g for 3 minutes and the supernatant recovered (LM medium).</P><P>To measure bioaccumulated Hg, we needed to quantify the Hg in the medium after incubation/exposure time. So we collected and measured the amount of Hg in LM medium supernatant recovered. In order to do so, we used the equipment Direct Mercury Analyzer (DMA-80). We used a control estimated by the DMA-80 machine.</P><P id="t3">The graph represented on Figure 3 shows the amount of Hg in supernatant (LM medium recovered) of our DH5-alpha transforming with BBa_K2123302. In the 5 ppm Hg concentration our Phytochelatin, accumulated 69% of total Hg amount in just 08:00 hours of incubation!!!</P><H2 style="background:#fff;margin-bottom:0.5em; color:#000"><CENTER><B>Notebook</B></CENTER></H2><NAV class="nav-page"><UL><LI class="page-item"><A href="#t1">Description</A></LI><LI class="page-item"><A href="#t2">Results</A></LI><LI class="page-item"><A href="#t3" style="color:#368E8C;">Notebook</A></LI></UL></NAV><LABEL class="notebook-link" for="toggleph1">1st Week</LABEL><DIV id="ph1"><H2 style="color:#368E8C; font-size:1.5em"> 1st Day</H2><P>Preparation of electrocompetente cells of JM110 and DH5α. </P><P>Parts transformation (JK26 promoter, Lpp-OmpA BBa_K103006, Phytochelatin and Terminator BBa_B0015). </P><H2 style="color:#368E8C; font-size:1.5em"> 2nd Day</H2><P>Isolation in plate (four transforming of each plate) </P><H2 style="color:#368E8C; font-size:1.5em"> 3rd Day</H2><P>Inoculum of four transforming colonies from the parts (JK26 promoter, Lpp-OmpA BBa_K103006, Phytochelatin and Terminator BBa_B0015) isolated the previous day. </P><H2 style="color:#368E8C; font-size:1.5em"> 4th Day</H2><P>Plasmidial extraction of the inoculum with kit Illustra plasmidPrep Mini Spin GE Healthcare. </P><P style="font-size:0.8em; text-align: center;"><I> Gel extraction of parts in agarose gel 0,8%; 1, 2, 3, 4) P1 promoter; 5, 6, 7, 8) Lpp-OmpA (OmpA); 9, 10, 11, 12) Phytochelatin (PTC); 13, 14, 15, 16) Terminator BBa_B0015; Colonies 1, 2, 3, 4 of each part respectively.</I></P><P>Test digestion to confirm the sizes with EcoRI. </P><P style="font-size:0.8em; text-align: center;"><I> Digestion test of the parts in agarose gel 0,8%; 1, 3, 5, 7) P1 Promoter – Not Digested; 2, 4, 6, 8) P1 Promoter – Digestion with EcoRI.; 9, 11, 13, 15) Omp A – Not Digested.; 10, 12, 14, 16) Omp A – Digestion with EcoRI; 17, 19, 21, 23) FTQ – Not Digested.; 18, 20, 22, 24) FTQ – Digestion with EcoRI.; 25, 27, 29, 31) Terminator – Not Digested.; 26, 28, 30, 32) Terminator – Digestion with EcoRI.; Colonies 1, 2, 3, 4 of each part respectively.</I></P><P>Digestion to confirm inserts. </P><P style="font-size:0.8em; text-align: center;"><I> Digestion to confirm insert in agarose gel 1,5%; 1) P1 Promoter – Not Digested.; 2) P1 Promoter – Digested with EcoRI and PstI.; 3) Omp A – Not Digested.; 4) Omp A – Digested with EcoRI and PstI.; 5) PTC – Not Digested.; 6)PTC- Digested with EcoRI and PstI.; 7) Terminator – Not Digested.; 8) Terminator – Digested with EcoRI and PstI.;</I></P><H2 style="color:#368E8C; font-size:1.5em"> 5th Day</H2><P>Analytical digestion for parts purifying. </P><P style="font-size:0.8em; text-align: center;"><I> Extraction gel from the parts in agarose gel 0,8%; 1,2,3,4,5,6,7,8,9) PO – JK26 Promoter linked to Lpp-OMPA.</I></P><P>Purification of parts.</P><P>Ligation; promoter to Lpp-OMPA and Phytochelatin to Terminator.</P></DIV><LABEL class="notebook-link" for="toggleph2">2nd Week</LABEL><DIV id="ph2"><H2 style="color:#368E8C; font-size:1.5em"> 6th Day</H2><P>Transformation of the ligations in JM110. </P><H2 style="color:#368E8C; font-size:1.5em"> 7th Day</H2><P>Isolation in plate of all transforming cells in each linking.</P><H2 style="color:#368E8C; font-size:1.5em"> 8th Day</H2><P>Inoculum of transforming isolated from PO (Promoter JK26 and Lpp-OMPA) and PT (Phytochelatin and Terminator) linkings. </P><H2 style="color:#368E8C; font-size:1.5em"> 9th Day</H2><P>Plasmidial extraction of the inoculum with Kit Illustra plasmidPrep Mini Spin GE Healthcare. </P><P style="font-size:0.8em; text-align: center;"><I>Extraction gel from the parts in agarose gel 0,8%; 1,2,3,4,5,6,7,8,9) PO – JK26 Promoter linked to Lpp-OMPA.</I></P><P style="font-size:0.8em; text-align: center;"><I> Extraction gel from parts in agarose gel 0,8%; 1,2,3,4,5) PT -  Phytochelatin linked to Terminator.</I></P><P>Digestion test to confirm the size with EcoRI and PstI. </P><P style="font-size:0.8em; text-align: center;"><I> Digestion test to confirm insert of PO (Promoter and Lpp-OmpA) linking in agarose gel 0,8%;</I></P><P style="font-size:0.8em; text-align: center;"><I> Digestion to confirm the insert from the PT (Phytochelatin and Terminator) linking in agarose gel 0,8%.</I></P><H2 style="color:#368E8C; font-size:1.5em"> 10th Day</H2><P>Analytical digestion from parts purification</P><P style="font-size:0.8em; text-align: center;"><I> Analytical gel to parts purification in agarose gel 0,8%; 1) PO – Not Digested.; 2) PO – Digested with NotI.; 3) PO – Digested with EcoRI and SpeI.; 4)PT – Not Digested.; 5) PT- Digestion with NotI.; 6) PT – Digestion with EcoRI and XbaI.</I></P><P>Purification. </P><P style="font-size:0.8em; text-align: center;"><I>1) PO – Purified; digested with EcoRI and SpeI; 2)PT – Purified; digestion with EcoRI and XbaI.;</I></P><P>Linking</P></DIV><LABEL class="notebook-link" for="toggleph3">3rd Week</LABEL><DIV id="ph3"><H2 style="color:#368E8C; font-size:1.5em"> 11th Day</H2><P>Transformation of POFT linking in DH5α</P><H2 style="color:#368E8C; font-size:1.5em"> 12th Day</H2><P>Isolating all POFT linking transforming in plate. </P><H2 style="color:#368E8C; font-size:1.5em"> 13th Day</H2><P>Inoculum of isolated transforming from POFT ligation. </P><H2 style="color:#368E8C; font-size:1.5em"> 14th Day</H2><P>Plasmidial extraction of inoculum from POFT linking with Kit Illustra plasmidPrep Mini Spin GE Healthcare. </P><P style="font-size:0.8em; text-align: center;"><I> Extraction gel POFT ligation in agarose  gel 0,8%.</I></P><P>Digestion test to confirm size with EcoRI and PstI. </P><P style="font-size:0.8em; text-align: center;"><I>Lanes 5,6,7 and 8 digestion to confirm the insert of POFT linking (Promoter+Lpp-OmpA+Phytochelatin and Terminator ~850pb) in agarose gel 0,8%.</I></P><P>With the ending of the construction we continued with characterization</P></DIV><P>REFERENCES</P><P>1.    BIONDO, R. Engenharia Genética de Cupriavidusmetallidurans para a biorremediação de efluentes contendo metais pesados, 2008. </P><P>2.    BAE, W.; MEHRA, R.K. Metal-binding characteristics of a phytochelatins analog (GluCys)2Gly.J. Inorg. Biochem., v. 68, p. 201-210, 1997. </P><P>3.    BAE, W.; CHEN, W.; MULCHANDANI, A.; MEHRA R. Enhanced bioaccumulation of heavy metals by bacterial cells displaying synthetic phytochelatins. Biotechnol. Bioeng., v. 70, p. 518-524, 2000. </P><P>4.    BAE, W.; MEHRA, R.K.; MULCHANDANI, A.; CHEN, W. Genetic engineering of Escherichia coli for enhanced uptake and bioaccumulation of mercury. Appl. Environ. Microbiol., v. 67, n. 11, p. 5335-5338, 2001. </P><P>5.    COSTA, G. S. Aplicação De Biossensor Microbiano Bioluminescente Na Detecção De HG (II), 2010. </P><P>6.    GIOVANELLA, P.; BENTO, F.; CABRAL, L.; GIANELLO, C.; CAMARGO, F. A. O.Isolamento e seleção de microrganismos resistentes e capazes de volatilizar mercúrio, 2010. </P><P>7.    NASCIMENTO, A. M. A.; CHARTONE-SOUZA. E. Operon mer: Bacterial resistance to mercury and potential for bioremediation of contaminated environments, 2003. </P><P>8.    NEVES-PINTO, M. Bases moleculares da resistência ao mercúrio em bactérias gram-negativas da Amazônia brasileira, 2004. </P><P>9.    SAMBROOK, J.; RUSSEL, D.W. Molecular Cloning: a Laboratory Manual. 3rd ed. Cold Spring HarborLaboratory Press, New York 2001. </P><P>10. SHEILA, S. S. Estudo do gene merA em bactérias gram-negativas resistentes ao mercúrio isoladas de ecossistemas aquáticos brasileiros: contribuição para a mitigação dos riscos do mercúrio à saúde humana através da biorremediação, 2012. </P><P>11. SOUZA, J. R.; BARBOSA, A. C. Contaminação por Mercúrio e o caso da Amazônia, 2000. </P><P>12. GREEN, M.R. e SAMBROOK, J. Molecular Cloning: A Laboratory Manual. Cold Spring Harbour Laboratory Press, 4ª Ed. Cold Spring Harbour, USA, 2012. </P><P>13. SHETTY, R. P.; ENDY, D.; KNIGHT, T. F. Engineering BioBrick vectors from BioBrick parts. Journal of Biological Engineering. Cambridge, MA, USA, 2008. </P><P>14. DASH, H. R.; DAS, S. Bioremediation of mercury and the importance of bacterial mer genes International Biodeterioration &amp; Biodegradation. Orissa, India, 2012. </P><P>15. WASSERMAN, J. C.; HACON, S. S.; WASSERMAN, M. A. O Ciclo do Mercúrio no Ambiente Amazônico. Mundo &amp; Vida Vol. 2. Niterói, RJ, Brasil, 2001. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>