"
Page
Discussion
View source
History
teams
Log in
 
Team:Hong Kong-CUHK/documentation.html
From 2014.igem.org
<style type="text/css"> .fixed {
position: fixed;
} /* sidebar */
.bs-docs-sidebar {
padding-left: 20px;    margin-top: 20px;    margin-bottom: 20px;
} /* all links */
.bs-docs-sidebar .nav>li>a {
color: #999;    border-left: 2px solid transparent;    padding: 4px 20px;    font-size: 13px;    font-weight: 400;
} /* nested links */
.bs-docs-sidebar .nav .nav>li>a {
padding-top: 1px;    padding-bottom: 1px;    padding-left: 30px;    font-size: 12px;
} /* active & hover links */
.bs-docs-sidebar .nav>.active>a, .bs-docs-sidebar .nav>li>a:hover, .bs-docs-sidebar .nav>li>a:focus {
color: #563d7c;    text-decoration: none;    background-color: transparent;    border-left-color: #563d7c;
} /* all active links */
.bs-docs-sidebar .nav>.active>a, .bs-docs-sidebar .nav>.active:hover>a, .bs-docs-sidebar .nav>.active:focus>a {
font-weight: 700;
} /* nested active links */
.bs-docs-sidebar .nav .nav>.active>a, .bs-docs-sidebar .nav .nav>.active:hover>a, .bs-docs-sidebar .nav .nav>.active:focus>a {
font-weight: 500;
} /* hide inactive nested list */
.bs-docs-sidebar .nav ul.nav {
display: none;
} /* show active nested list */
.bs-docs-sidebar .nav>.active>ul.nav {
display: block;
} .subgroup.well {
text-align: justify;
} .subgroup {
background-color: rgba(255, 255, 255, 0.8);
} </style>
<nav class="col-sm-3 hidden-xs bs-docs-sidebar">
<a href="#protocol">Protocol</a>
<a href="#DNA">Bacterial DNA extraction protocol for
Azotobacter vinelandii or E.coli</a>
<a href="#miniprep">Miniprep</a>
<a href="#BL21">Preparation of chemically competent BL21
E. coli cells</a>
<a href="primer">Primer Design</a>
<a href="#pcr1">PCR - Phusion DNA polymerase NEB</a>
<a href="#pcr2">PCR - PCR - LA Taq DNA polymerase Takara</a>
<a href="#DDD">Double Digestion of DNA with 2 different restriction enzymes NEB</a>
<a href="#BL21">Preparation of chemically competent BL21 E. coli cells</a>
<a href="#notebook">Notebook</a>
<a href="#june">June</a>
<a href="#july">July</a>
<a href="#august">August</a>
<a href="#september">September</a>
<a href="#october">October</a>
</nav>
<section id="protocol" class="group">
PROTOCOL
Bacterial DNA extraction protocol for Azotobacter vinelandii or E.coli
We are using the <a href=" http://www.takara.co.kr/file/manual/pdf/9763_e.v1309Da.pdf " target="_blank">TaKaRa MiniBEST Bacteria Genomic DNA Extraction Kit</a> of Takara according to the manufacturer directions.
Miniprep
We are using the <a href=" http://eshop.intronbio.com/product/detail04.asp?pIdx=1 " target="_target">DNA-spin&#8482 Plasmid DNA Purification Kit</a> of Intron Technology according to the manufacturer directions.
Preparation of chemically competent BL21 E. coli cells
Day1
Streak Bl21 on a LB agar plate without antibiotic, grow overnight in 37&#8451 incubator
Day2
Pick a single colony and inoculate into 3ml LB broth, grow o/n in 37&#8451 shaker
Prepare & autoclave 500ml LB broth
Check if there is enough liquid nitrogen
Day3 Morning: pour the 3ml dense pre-culture into 500ml LB broth
Shake in 37C until OD600nm reach 0.8
[Melody's experience: it takes 4~5hrs]
Solution Needed
Wash Buffer I
800mM MgCl2 + 20mM CaCl2
Wash Buffer II
125mM CaCl2
Resuspend Buffer
85mM CaCl2 + 15% glycerol [filtered]
pre-cool Wash Buffer I & Wash Buffer II in ice
Pre-cool the centrifuge to 4C (with fixed angle rotor)
Check the OD600nm of the 500ml culture
Centrifuge the cells at 4000g for 5 mins, 4&#8451
Discard the supernatant
Gently resuspend the pellet in 20ml ice cold Wash Buffer I
Put the samples on ice for 10 mins
Centrifuge the cells at 4000g for 5 mins, 4&#8451
Discard the supernatant
Gently resuspend the pellet in 10ml ice cold Wash Buffer II
Put the samples on ice for 10 mins
Centrifuge the cells at 4000g for 5 mins, 4&#8451
Discard the supernatant
Resuspend cells in 20ml ice cold Resuspend Buffer
Aliquot 200ul using sterile pre-chilled eppendorf tubes
Primer Design
Primers were designed manually using <a href=" http://www.bioinformatics.org/sms2/pcr_primer_stats.html " target="_blank">PCR Primer Stats</a> of Sequence Manipulation Suite, for analyzing secondary structures and also annealing conditions.
PCR - Phusion DNA polymerase NEB
50ul reaction
<thead>
Reactives
Volume(μL)
</thead> <tbody>
DNA template
1
5x Phusion HF Buffer
10
10mM dNTPs
1.5
10uM Primer Fw
0.5
10uM Primer Rv
0.5
Phusion DNA polymerase
0.25-0.5
100% DMSO
1.5
dH2O
Up to 50
Total
50
</tbody>
Cycling condition:
Initial Denaturation (1 cycle):
98°C 30 sec
Amplification (35 cycles):
98°C 10 sec
55-72 °C 30 sec
72°C 0.25- 0.5 min/kb
Final elongation (1 cycle)
72°C 3 min
PCR - LA Taq DNA polymerase Takara
50ul reaction
<thead>
Reactives
Volume(μL)
</thead> <tbody>
DNA template
1
10x LA taq Buffer
5
10mM dNTPs
1.5
10uM Primer Fw
0.5
10uM Primer Rv
0.5
LA taq polymerase
0.25-0.5
100% DMSO
1.5
dH2O
Up to 50
Total
50
</tbody>
Cycling condition:
Initial Denaturation (1 cycle):
94°C 1 min
Amplification (30 cycles):
98°C 10 sec
55-72 °C 30 sec
68°C 0.5- 1 min/kb
Final elongation (1 cycle)
72°C 5 min
NEB Gibson assembly
We are using the Gibson Assembly® Master Mix of NEB Inc. according to the manufacturer directions.
Double Digestion of DNA with 2 different restriction enzymes NEB
30μl reaction
<thead>
Reactives
Volume(μL)
</thead> <tbody>
DNA
Up to 1 μg
CutSmart™ Buffer
3
EcoRI-HF®*
1
SpeI/PstI/XbaI*
1
dH2O
Up to 30μL
Total
30
</tbody>
Incubate at 37 °C incubator or heat bath for 0.25 to 2 hours.
*For combinations of restriction enzymes other than the above, please kindly refer to Double Digest Finder from NEB Inc. for suitable buffer condition.
DNA ligation with T4 DNA ligase NEB
<thead>
Reactives
Volume(μL)
</thead> <tbody>
T4 DNA ligase
1
Buffer 10X
2
Vector DNA
n pmol
Insert DNA
3n pmol
dH2O
Up to 20μL
Total
20
</tbody>
Incubate at room temperature for 0.25 -1 hours.
</section>
<section id="notebook" class="group">
NOTEBOOK
JUNE
Preparation of Competent Cells of dh5&#945 and BL21, aliquot the competent cells, stored in liquid nitrogen and then -80C refrigerator.
Transformation of: J23100-119 into dh5&#945
Preparation of special medium and plate for A. vinelandii
Make LB agar solution
Study the growth curve characteristics of A. vinelandii
Study the natural antibiotics resistance of A. vinelandii
Make plate with antibiotics -Ampicillin, kanamycin and chloramphenicol.
Preparation of Competent Cells of A. vinelandii
Purchase for primers for the Carbon fixation system project.
Primer dilution of the all carbon fixation system primers
Amplification by using PCR, following by run gel of A. vinelandii nifB gene.
Amplification by using PCR, following by run gel of A. vinelandii nifQ gene.
Amplification by using PCR, following by run gel of A. vinelandii FdxA gene.
Amplification by using PCR, following by run gel of Aquifex aeolicus mbhS3 gene.
Amplification by using PCR, following by run gel of Aquifex aeolicus mbhL3 gene.
Amplification by using PCR, following by run gel of E. coli SH3PDZ domain gene.
JULY
Make plate with 2% hard agar with antibiotics- chloramphenicol.
Make plate with antibiotics -Ampicillin.
Overlapping PCR of A. vinelandii nifB gene, A. vinelandii nifQ gene and A. vinelandii FdxA gene
Restriction of Overlapping PCR product A. vinelandii nifB-nifQ-FdA, ligase into double terminator plasmid and transform into dh5&#945.
Overlapping PCR of Aquifex aeolicus mbhS3 gene and mbhL3 gene.
Restriction of Overlapping PCR product Aquifex aeolicus mbhS3-L3, ligase into double terminator plasmid and transform into dh5&#945.
Make LB agar solution
Restriction of PCR product E. coli SH3PDZ domain gene, ligase into C backbone and transform into dh5&#945.
Stable genome integration of J23100 into A. vinelandii and study its characteristics
Purchase for primers for the Nitrogen- repressible T7 expression system.
Arrival of synthetic sequences of A. vinelandii nitrogenase structural gene
Try Gibson assembly of synthetic sequences of A. vinelandii nitrogenase structural gene
AUGUST
Arrival of synthetic sequences of A. vinelandii nitrogenase accessory gene.
Arrival of synthetic sequences of Aquifex aeolicus hydrogenase accessory gene.
Arrival of all primers for the Nitrogen- repressible T7 expression system
Primer dilution of the all Nitrogen- repressible T7 expression system primers
Amplification by using PCR, following by run gel of A. vinelandii nifH gene.
Amplification by using PCR, following by run gel of A. vinelandii nifK gene
Amplification by using PCR, following by run gel of A. vinelandii nifH gene with strong rbs.
Amplification by using PCR, following by run gel of A. vinelandii nifH gene with weak rbs.
Transformation of T7 RNA polymerase and T7 promoter into dh5&#945.
Amplification by using PCR, following by run gel of BL21 T7 RNA polymerase.
Overlapping PCR of A. vinelandii nifH gene with strong rbs and BL21 T7 RNA polymerase.
Overlapping PCR of A. vinelandii nifH gene with weak rbs and BL21 T7 RNA polymerase.
Amplification by using oligo PCR, following by run gel of random sequence
Transformation of ptet- mRFP-d.t. and mRFP-d.t. into dh5&#945.
Amplification by using oligo PCR, following by run gel of part of mRFP
Restriction of PCR product random sequence, ligase into ptet- mRFP-d.t. plasmid and transform into dh5&#945.
Transformation of amilcp-d.t. into dh5&#945.
Restriction of PCR product nifH, ligase into mRFP-d.t. plasmid and transform into dh5&#945.
September
Try Gibson Assembly of synthetic sequences of A. vinelandii nitrogenase accessory gene.
Try Gibson Assembly of synthetic sequences of Aquifex aeolicus hydrogenase accessory gene.
Restriction of PCR product A. vinelandii nifH gene with strong rbs and BL21 T7 RNA polymerase, ligase into double terminator plasmid and transform into dh5&#945.
Restriction of PCR product A. vinelandii nifH gene with weak rbs and BL21 T7 RNA polymerase, ligase into double terminator plasmid and transform into dh5&#945.
Restriction of biobrick A. vinelandii nifH gene with strong rbs and BL21 T7 RNA polymerase-double terminator ligase into random sequence with ptet-mRFP-d.t. and transform into dh5&#945.
Restriction of biobrick A. vinelandii nifH gene with weak rbs and BL21 T7 RNA polymerase-double terminator ligase into random sequence with ptet-mRFP-d.t. and transform into dh5&#945.
Restriction of PCR product A. vinelandii nifK gene,ligase into biobrick A. vinelandii nifH gene with strong rbs and BL21 T7 RNA polymerase-double terminator and random sequence with ptet-mRFP-d.t. and transform into dh5&#945.
Restriction of PCR product A. vinelandii nifK gene,ligase into biobrick A. vinelandii nifH gene with weak rbs and BL21 T7 RNA polymerase-double terminator and random sequence with ptet-mRFP-d.t. and transform into dh5&#945.
Restriction of PCR product A. vinelandii nifK gene, ligase into C back bone and transform into dh5&#945.
Restriction of PCR product A. vinelandii nifH gene, ligase into C back bone and transform into dh5&#945.
Restriction of PCR product part of mRFP, ligase into C back bone and transform into dh5&#945.
Restriction of BBa_K1314013, ligase into A backbone and transform into dh5&#945.
Restriction of BBa_K1314014, ligase into A backbone and transform into dh5&#945.
Restriction of PCR product part of mRFP, ligase into amilcp- d.t. and transform into dh5&#945.
Restriction of PCR product, random sequence, ligase into T7 promoter and transform into dh5 &#945.
Restriction of biobrick random sequence- T7 promoter, ligase into C backbone and transform into dh5&#945
Restriction of biobrick random sequence- T7 promoter, ligase into amilcp- d.t.-part of mRFP, and transform into dh5&#945.
Send all biobricks to sequencing
Transformation of BBa_K1314011 into BL21.
Stable genome integration of BBa_K1314013 into A. vinelandii
Stable genome integration of BBa_K1314014 into A. vinelandii
Stable genome integration of BBa_K1314011 into A. vinelandii
Blunt end ligation and transformation to make BBa_K1314015
October
1. Submission of all biobricks to the registry
</section>
<script type="text/javascript"> $(document).ready(function() {            $('body').scrollspy({                target: '.bs-docs-sidebar',                offset: 100            });        });        $(window).scroll(function(e) {            var scrollTop = $(window).scrollTop();            if (scrollTop > 250) {                $('#sidebar').addClass('fixed').offset({                    top: scrollTop                });            } else {                $('#sidebar').removeClass('fixed');            }        });
</script>
Retrieved from " http://2014.igem.org/Team:Hong_Kong-CUHK/documentation.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
