<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Hong_Kong-CUHK_modelling-1_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Hong_Kong-CUHK/modelling-1.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Hong_Kong-CUHK/modelling-1.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Hong_Kong-CUHK/modelling-1.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Hong_Kong-CUHK/modelling-1.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Hong_Kong-CUHK/modelling-1.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Hong Kong-CUHK/modelling-1.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Model_for_protein_expression_system"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Model for protein expression system</SPAN></A></LI><LI class="toclevel-2"><A href="#1..09Overview"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">1.	Overview</SPAN></A></LI><LI class="toclevel-2"><A href="#2..09Analysis_and_Discussion"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">2.	Analysis and Discussion</SPAN></A></LI><LI class="toclevel-2"><A href="#2.1.09Assumptions"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">2.1	Assumptions</SPAN></A></LI><LI class="toclevel-2"><A href="#2.2.09Discussion_of_the_two_stages"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">2.2	Discussion of the two stages</SPAN></A></LI><LI class="toclevel-3"><A href="#In_the_first_stage:"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">In the first stage:</SPAN></A></LI></UL><LI class="toclevel-2"><A href="#2.3_Constant_Search"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">2.3 Constant Search</SPAN></A></LI><LI class="toclevel-3"><A href="#Reference:"><SPAN class="tocnumber">1.5.1</SPAN><SPAN class="toctext">Reference:</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Model_for_protein_expression_system">Model for protein expression system</SPAN></H1><H2><SPAN class="mw-headline" id="1..09Overview">1.	Overview</SPAN></H2><P>Amino acids are the basic building blocks of protein. The rate of protein translation (V) increases with the concentration of amino acids (C) and bounds when the amount of amino acids is saturated.(i.e. there exists Vmax, such that V is smaller or equal to Vmax no matter how high C is.) Normally, the protein expression rates of different system can be compared by the transcription efficiency of the system only because we provide enough nitrogen sources to make V close to Vmax. However, this is not quite applicable to our protein expression system. The transcription process of our system is repressed by nitrogen sources in the cell, so we believe that it is not practical to make C reaches saturated. However, this system allows us to achieve a constant protein expression rate by controlling the duration for expressing T7 polymerase.</P><H2><SPAN class="mw-headline" id="2..09Analysis_and_Discussion">2.	Analysis and Discussion</SPAN></H2><P>Let us review our protein expression system. The schematic diagram is shown below.
&lt;img src=&quot;&quot;&gt;</P><H2><SPAN class="mw-headline" id="2.1.09Assumptions">2.1	Assumptions</SPAN></H2><P>First, we may assume that:
a. Concentration of σ-RNA polymerase is constant.
b. All proteins being expressed share the same nitrogen source.
Then, let:
[N] be the concentration of nitrogen source in cell;
[T7p] be the concentration of T7 polymerase;
[TP] be the concentration of target protein;
Knc be the rate of nitrogen consumption out of this protein expression system;
Vmax([T7p]) be the maximum rate for target protein expression;
A,B,C,D,E,F,G,H, be constant.</P><H2><SPAN class="mw-headline" id="2.2.09Discussion_of_the_two_stages">2.2	Discussion of the two stages</SPAN></H2><H3><SPAN class="mw-headline" id="In_the_first_stage:">In the first stage:</SPAN></H3><P>In this stage, we want to optimize the production of T7 polymerase by setting a relatively low concentration of nitrogen source.
We have the following equation for the rate of change of [N]:
&lt;img src=&quot;&quot;&gt;</P><P>Where in the right-hand side of the equation,
First term corresponds to the nitrogen consumption rate for T7 polymerase;
Second term corresponds to the nitrogen consumption rate for target protein;
Third term corresponds to the nitrogen consumption rate of the bacteria without the system;
A is the nitrogen needed to expression one T7 polymerase;
B is the nitrogen needed to expression one target protein;
C is the maximum speed of nitrogen consumption of unmodified bacteria;
D is the concentration needed for the rate of nitrogen consumption equals to half of maximum speed.</P><P>Now, we need to figure out the formulae of rate of change of [T7P] and [TP].
Since [TP] is not the major focus in this stage, let us ignore this temporarily.
Then, we need to predict the formula for rate of change of [T7P]. Below is the information we have:
i	As [N] tends to 0,  rate of change of [T7P]  tends to 0.
ii	When [N] is very large,   rate of change of [T7P] tends to 0.
iii	When [N] is saturated for the translation process (denoted by [Nsat]), the second derivative of [T7P] is negative.
vi	The formula is continuous.
Base on the above information, we proposed the following two formulae:</P><PRE>&lt;img src=&quot;&quot;&gt;22:58, 17 October 2014 (CDT)~ (2)&lt;img src=&quot;&quot;&gt;(3)</PRE><P>Reason for proposing (2): we assume that the repressive power of nitrogen sources grows exponentially.&lt;img src=&quot;&quot;&gt;</P><P>Graph of (2) with different values of F. (red: 0.5; blue: 0.75; green: 1)
Reason for proposing (3): we assume that the repressive power of nitrogen sources grows linearly.&lt;img src=&quot;&quot;&gt;</P><P>Graph of (3) with different values of G. ([Nsat] = 1) (red: 0.5; blue: 1; green: 2)</P><P>Now, we want to find out the optimized value for [N] to produce polymerase as fast as possible. Therefore, we want to have the maximum of (2) and (3).
For (2), we have&lt;img src=&quot;&quot;&gt;</P><P>Solving the equation, we have [N optimal] = 1/F,
For (3), we have [N optimal] = [Nsat]
If we want to set [N] = [N optimal] for a certain period, we need to find the rate of change of [N] at [N] = [N optimal]
When [N] is optimal, [N] is quite small. As we assume that the transcription efficiency of nifH promoter is very high with small [N] and [T7p] initially is very small, we have</P><PRE>&lt;img src=&quot;&quot;&gt;</PRE><P>In the second stage:
In this stage, we want to optimize the production of target protein by setting an excess nitrogen source.
We have the equation for the rate of target protein production:</P><PRE>&lt;img src=&quot;&quot;&gt;22:58, 17 October 2014 (CDT)~~(4)</PRE><P>The formula is similar to the Michaelis-Menten Equation with Vmax replaced by a function of [T7P].
When [N] is very large, we have rate of change of [N] closed to zero.
Also, rate of change of [T7P] closes to zero as the nifH promoter is highly repressed.
This implies that [T7P] is a constant
As a result, Vmax([T7P]) is a constant
From (4), when [N] is very large, rate of change of [TP] closes to Vmax([T7P]).
Hence, we have rate of target protein production is a constant.</P><H2><SPAN class="mw-headline" id="2.3_Constant_Search">2.3 Constant Search </SPAN></H2><P>The composition of T7 RNA polymerase has been investigated [1]. From the table above, the expected value of A can be found.
If the target protein chosen have been investigated, B should be able to be found in some literatures 
C, D can be found experimentally:
	i. We can grow wild type Azotobacter vinelandii and monitor the nitrogen concentration at different time points.
E, F, G can be found experimentally:
	i. We can calculate [T7P] by measuring the RNA concentration. Here, we assume that RNA concentration is always directly proportional to the [T7P].
	ii. We can modify T7 polymerase and find [T7P] with the aid of western blot.
It is not necessary to search H. We can predict   without knowing H.
</P><H3><SPAN class="mw-headline" id="Reference:">Reference: </SPAN></H3><P>[1] Moffatt, Barbara A., John J. Dunn, and F. William Studier. &quot;Nucleotide sequence of the gene for bacteriophage T7 RNA polymerase.&quot; Journal of molecular biology 173.2 (1984): 265-269.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Hong_Kong-CUHK/modelling-1.html">http://2014.igem.org/Team:Hong_Kong-CUHK/modelling-1.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Hong_Kong-CUHK/modelling-1.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Hong_Kong-CUHK/modelling-1.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Hong_Kong-CUHK/modelling-1.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Hong_Kong-CUHK/modelling-1.html&amp;oldid=401562" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>