<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SCUT-ChinaA_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SCUT-ChinaA/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><TITLE>SCUT-ChinaA</TITLE><HEADER><NAV class="navbar navbar-fixed-top navbar-custom"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle navbar-icon" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
              Team
              </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Team"> Team Members </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Attributions"> Attributions </A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle navbar-icon" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
              Project
              </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Overview"> Overview</A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Design"> Design</A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Experiments"> Experiments </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Results"> Results </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle navbar-icon" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
              Model
              </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Model"> Model </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Software"> Software Tool </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle navbar-icon" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
              Achievement
              </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Parts"> Parts </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Deliverables"> Deliverables </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/InterLab"> InterLab</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle navbar-icon" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
              Practice 
              </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Human_Practices"> Overview </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Silver"> Silver </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Gold"> Gold </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Collaborations"> Collaborations </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Safety"> Safety </A></LI></DIV></DIV></NAV></HEADER><DIV class="igem_content_wrapper"><DIV class="column full_size"><H2 style="text-align: left">Construction </H2><OL><H3>1.Design</H3><P style="text-align: justify">In our project, for obtaining various engineered strains, we applied module assembly technology like gibson assembly and overlap extension PCR to construct gene expression modules in E. coli. In the process of pathway construction, we successfully constructed pUC19-NDPS1-LS, pUC19-Catcher-NDPS1-Tag-LS, pUC19-tHMG1, pUC19-ERG8-ERG12-ERG19, pCAS1yl-ku70, pCAS1yl-pox5, respectively. The upstream and downstream regions are homologous to the target integration site and prepared for homologous recombination in further gene editing. The <EM>LEU2</EM> and <EM>URA3</EM> was used for selection in <EM>Y. lipolytica</EM> and the <EM>AmpR</EM> was used in <EM>E.coli.</EM> NotI restriction sites was added between the homology regions. These plasmids will be further linearized in the subsequent steps.
</P><H3>2.Method</H3><H7>
Gibson assembly
</H7><P style="text-align: justify">
The method can simultaneously combine up to 15 DNA fragments based on sequence identity. It requires that the DNA fragments contain ~20-40 base pair overlap with adjacent DNA fragments. These DNA fragments are mixed with a cocktail of three enzymes, along with other buffer components. The three required enzyme activities are: exonuclease, DNA polymerase, and DNA ligase. The exonuclease chews back DNA from the 5' end, thus not inhibiting polymerase activity and allowing the reaction to occur in one single process. The resulting single-stranded regions on adjacent DNA fragments can anneal. The DNA polymerase incorporates nucleotides to fill in any gaps. The DNA ligase covalently joins the DNA of adjacent segments, thereby removing any nicks in the DNA.
</P><P style="text-align: justify">
The detailed protocol is as follows:
</P><UL><LI>1.    Add overlap region to adjacent DNA fragments via PCR;
</LI><LI>2.    Combine the to-be-assembled DNA to a total volume of 5 μl;
</LI><LI>3.    On ice, add 15μl Master Mix to the DNA, mix well and briefly centrifuge;
</LI><LI>4.    Incubate at 50℃ for 1 hour;
</LI><LI>5.    Store reactions at -20℃ or proceed to transformation.
</LI></UL></OL></DIV><DIV class="column full_size"><H2 style="text-align: left">Integration</H2><OL><H3>1.Design</H3><P style="text-align: justify">After the successfully construction of various expression modules, we linearized them with NotI and then were transformed into competent Po1f cells together, using the kit, Frozen-EZ yeast transformation II. After transformation, cells were cultured on SD medium plates without uracil or leucine. The URA3 marker was integrated into the cells along with the expression cassettes, which would affect the insertion of subsequent gene expression modules. Therefore, YPD media supplemented with 1 mg/mL 5-FOA was used for URA3 marker plasmid removal. Screening for integration was accomplished using colony PCR of single colonies. Furthermore, we would further verify it by extracting the genome of the yeast, using the kit from TIANGEN Biotech (Beijing, China). At the transcriptional level, we verified the transcription of the integrated gene modules by quantitative PCR with a kit from TAKARA BIO INC. </P><H3>2.    Method</H3><H7>Transformation</H7><P style="text-align: justify">First, prepare the yeast competent cells. Grow yeast cells at 30℃ in 10ml YPD broth until mid-log phase (OD600 of 0.8-1.0). The following steps are accomplished at room temperature.</P><UL><LI>1.    Pellet the cells at 500 x g for 4 minutes and discard the supernatant.
</LI><LI>2.    Add 10 ml EZ 1 solution to wash the pellet. Repellet the cells and discard the supernatant.
</LI><LI>3.    Add 1 ml EZ 2 solution to resuspend the pellet.
</LI></UL><P style="text-align: justify">At this point, the competent cells can be used for transformations directly or stored frozen at or below -70℃ for future used. It is important to freeze the cells slowly. To accomplish this, either wrap the aliquoted cells in 2-6 layers of paper towels or place in a Styrofoam box before placing in the freezer. Do not use liquid nitrogen to snap-freeze the cells.
</P><P style="text-align: justify">With yeast competent cells, we can then carry out yeast transformation.
</P><UL><LI>1.    Mix 50 μl of competent cells with 0.2-1 μg DNA (in less than 5 μl volume); add 500 μl EZ 3 solution and mix thoroughly.
</LI><LI>2.    Incubate at 30℃ for 45 minutes. Mix vigorously by flicking with finger or vortexing (if appropriate for your DNA) 2-3 times during this incubation.
</LI><LI>3.    Spread 50-150 μl of the above transformation mixture on an appropriate plate. It is unnecessary to pellet and wash the cells before spreading.
</LI><LI>4.    Incubate the plate at 30℃ for 2-4 days to allow for growth of transformations.
</LI></UL><H7>Yeast total RNA Extraction</H7><P style="text-align: justify">Most of RNA in the cell is bound to protein and exists as nuclear protein. Therefore, RNA is separated and removed from protein when extracting RNA. The cells are placed in a buffer containing sodium dodecylsulfonate, and then water-saturated phenol and chloroform are added to it. Through vigorous shaking and centrifuging, the liquid is divided into the upper aqueous phase and the lower phenol phase. The nucleic acid is dissolved in the aqueous phase, the protein denatured by the phenol is either dissolved in the phenol phase, or forming a gel layer at the interface of the two phases. The upper aqueous phase is collected and the RNA is precipitated from the aqueous solution with ethanol.
</P><P style="text-align: justify">All necessary materials for the experiment should be prepared as described below:
</P><UL><LI>1.    0.2% DEPC-treated water: Wash and dry a wide-mouth bottle, wash it with hydrogen peroxide for 3 times, and then wash it with absolute ethanol for 3 times. After drying, add 1 L ultrapure water, then add 2 ml DEPC in a fume hood, shake overnight at 37 °C, sterilize at 121 °C for 30 min, and then store at room temperature.
</LI><LI>2.    AE buffer (pH 5.2, 50 mM NaAc, 10 mM EDTA): Weigh 0.41 g anhydrous NaAc, 0.37 g EDTA Na2, dissolve in 80 mL pure water, adjust the pH to 5.2 with glacial acetic acid, and then constant volume to 100 mL. Add 200 μl DEPC, shake overnight at 37 °C, sterilize at 121 °C for 45 min, and then store at room temperature.
</LI><LI>3.    10% (w/v) SDS: Weigh 10 g SDS, dissolve with 80 mL DEPC-treated water, and constant volume to 100 mL with DEPC-treated water, don’t need sterilization. Store at room temperature.
</LI><LI>4.    Water-saturated phenol:chloroform (1:1): While taking water-saturated phenol, the tips should be inserted into the underlying organic phase. Then mix with chloroform and store. If the upper layer still has an aqueous phase, use pipettes to remove it.
</LI><LI>5.    3M NaAc (pH 5.2): Weigh 24.6 g anhydrous NaAc, dissolve in 80 mL pure water, adjust the pH to 5.2 with glacial acetic acid, constant volume to 100 mL, add 200 μl DEPC, shake overnight at 37 °C, 121 °C Sterilize for 30 min and store at room temperature.
</LI><LI>6.    Absolute ethyl alcohol
</LI><LI>7.    75% (v/v) ethanol: Mix 1 volume of DEPC water with 3 volumes of absolute ethanol.
</LI><LI>8.    0.2% DEPC treated TAE buffer: Measure out 20 mL TAX buffer, add DEPC treated water to 1 L, shake at 37 °C overnight (ensure to fix the bottle, prevent overflow)
</LI></UL><P style="text-align: justify">The specific experimental steps are as mentioned follows:</P><UL><LI>1.    Cell collection: Place 10 mL YPD in a 50 mL centrifuge tube and inoculate Yarrowia lipolytica for 16-18 h to an OD of 5-7. Collect 10 OD cells in a 1.5 mL centrifuge tube, centrifuged at 10000 rpm for 2 min, discard the supernatant. Centrifuge briefly and remove the remaining medium with a pipette. Quickly freeze the cells in a liquid nitrogen tank and store at -80 °C refrigerator (can be stored for 2-3 days).
</LI><LI>2.    Take and place the cells sample on ice, resuspend the cells in 50 μl AE buffer and mix by pipetting.
</LI><LI>3.    Take a 1.5 mL centrifuge tube and draw 600 μL phenol: chloroform (1:1), 350 ul AE buffer, and 50 ul SDS into it, quickly mix upside down and preheat in 65 °C metal bath for 5 minutes.
</LI><LI>4.    After incubation, incubate the cells at 30 °C, 250 rpm for 3 days, measure the OD and collect 10 OD bacteria. Centrifuge the cells at 4 °C,12000 g for 2min, remove the supernatant, add the mixture in step 3 to it, quickly reverse three times to mix well, immediately put into a 65 °C metal bath, and shake it at 1000 rpm for 14 min, and ice bath for 5 min.
</LI><LI>5.    Centrifuge at 4 °C, 12000 g for 7min, softly take 300 ul supernatant into a 1.5 mL centrifuge tube containing 750 ul absolute ethanol and 30 ul 3M NaAc (pH 5.2), mix upside down, incubate at -20℃ for more than 30 min.
</LI><LI>6.    Centrifuge at 4 °C, 12000 g for 5min, remove the supernatant with a 200ul pipette (be careful not to aspirate the RNA pellet), add 1 mL 75% ethanol to wash the pellet (Flick the tube bottom by finger, upside down ten times).
</LI><LI>7.    Centrifuge at 4 °C, 12000 g for 3min, adjust the pipette to 500ul to remove the supernatant (Be careful not to aspirate the RNA pellet). Centrifuge for about 30 s and adjust the pipette to 50 ul to remove the supernatant (Be careful not to aspirate the RNA pellet). Store at room temperature for 3 min to dry, add 37.5 ul DEPC treated water to dissolve the precipitate (The precipitate on the side wall of the centrifuge tube should also be dissolved).
</LI><LI>8.    Use Nanodrop ultra-micro UV spectrophotometer to determine RNA concentration and parameters.
</LI><LI>9.    Take 1 ug for electrophoresis. 25-40min, 60-80v. The running comb, rubber tray, rubber plate and electrophoresis tank all should be soaked in disinfectant for 20 minutes and rinsed once with absolute ethanol. TAE buffer should be treated with DEPC-treated water.

</LI></UL><P style="text-align: justify">Here are some points that need special attention.</P><UL><LI>
1.    The water for preparing solution is DEPC-treated water.
</LI><LI>2.    Wear latex gloves and replace them frequently during the entire operation to avoid RNase contamination. Wear a mask and talk as little as possible.
</LI><LI>3.    Remove the lower phenol phase when using water-saturated phenol.
</LI><LI>4.    Carefully remove the upper aqueous phase and do not absorb the denaturing gel layer between the aqueous phase and the phenolic phase.
</LI></UL><H7>RNA electrophoresis</H7><P style="text-align: justify">
All necessary materials for the experiment should be prepared as described below:
</P><UL><LI>
1.    DEPC-treated water: Transfer 1 L ultrapure water with a 500 ml graduated cylinder to a 1 L wide-mouth bottle, add 2 ml DEPC in a fume hood, and shake overnight at 37 °C. Sterilize at 121 °C for 30min and store at room temperature.
</LI><LI>2.    TAE buffer: Measure 20 ml 50xTAE, add 980 ml DEPC-treated H2O, dilute to 1x, and pre-cool on ice before use.
</LI><LI>3.    The experimental electrophoresis tank, comb, rubber tray, glue tank, conical flask for heating agarose, and conical flask for measuring TAE buffer all should be soaked in absolute ethanol for 15-20 min and dried for 3 min.
</LI><LI>4.    1.2% agarose gel: Weigh 0.3g agarose powder and dissolve it in 25ml TAE buffer prepared with DEPC-treated water. Heat it in a microwave oven for one minute, then pour it into the plastic tray and insert the comb. Cool to be solidified at room temperature (15-20 min).
</LI><LI>5.    Horizontal electrophoresis tank and electrophoresis apparatus.
</LI></UL><P style="text-align: justify">The specific experimental steps are as mentioned follows:</P><UL><LI>
1.    Turn on the electrophoresis system in advance and set corresponding parameters. (80v, 25min)
</LI><LI>2.    After the agarose gel has solidified, place the electrophoresis tank in an ice bath and pour the DEPC-treated TAE buffer into the electrophoresis tank. Pull out the comb and put the rubber block with the rubber tray into the electrophoresis solution. The electrophoresis solution should pass the rubber surface.
</LI><LI>3.    Use an appropriative 10xDNA loading buffer (Haven’t be used in other experiments), add 1 ul to the PCR tube containing 200 ul axygen, add the RNA sample to the tube, mix it with a pipette, and load. The operation must be quick.
</LI><LI>4.    After running the glue, take it out immediately and take a picture with a glue meter at 320nm wavelength.
</LI></UL><P style="text-align: justify">Here are some points that need special attention.</P><UL><LI>
1.    Wearing masks and gloves during the experiment;
</LI><LI>2.    Talk as little as possible;
</LI><LI>3.    The operation must be quick.
</LI></UL></OL></DIV><DIV class="column full_size"><H2 style="text-align: left">Fermentation</H2><OL><H3>1.Design</H3><P style="text-align: justify">After repeated validation, we inoculated engineered strains into the medium for two-phase fermentation. For exploring the difference between yeast growth and product yield between different species sources of <EM>LS</EM>, we introduced two kinds of <EM>LS</EM> genes expression cassettes to compare the growth curves. We first carried out the fermentation for six consecutive days, taking samples once a day to obtain the growth curve of the yeast. At the same time, the content of limonene was determined by gas chromatography-mass spectroscopy.</P><H3>2.    Method</H3><H7>Yeast cultivation</H7><P style="text-align: justify">During the fermentation, the engineered strains are cultured with liquid YPD medium. First, all strains were inoculated into a 250 ml shake flask containing 40 ml medium. When the strains in the shake flask grow to an OD600 value of about 1.0, the next step can be performed. The formal fermentation system was carried out in a 250 ml shake flask containing 50 ml of medium and 2 ml of N-dodecane, which function as an organic extractant to extract the product. The initial OD600 of the fermentation was 0.1. The fermentation will be carried out for six days at 30 °C and 220 rpm, with taking samples once a day. All flask fermentation results represent the mean ± S.D. of three independent experiments.</P><H7>Limonene detection</H7><P style="text-align: justify">


For limonene detection, the dodecane layers were collected and myrcene was used as an internal standard.  Then the sample (1 μL) was analyzed by GC using an Agilent System 6890 gas chromatograph coupled to an Agilent 5975 quadrupole mass selective detector (EI) (Agilent Technologies, Santa Clara, CA), equipped with a HP-5 (30 m × 0.25 mm, 0.25 μm film thickness) GC column. The GC oven temperature program was set as follows: 100 °C for 1 min, a ramp of 0.5 °C/min to 102 °C, and then a rise to 240 °C in 6 min with a final holding time of 2 min. The split ratio was 30:1. Limonene and myrcene standards (purchased from Sigma-Aldrich) were used for quantification.


</P><DIV class="column full_size"><H2 style="text-align: left">Discussion</H2><P style="text-align: justify">
We have elaborated on most of the experimental methods we have used here, and we omitted some basic experimental methods, such as PCR and the transformation of E. coli. We had been very struggle in the process of conducting experiments before. To overcome these difficulties, we have carried out a lot of exploratory experiments for optimizing the protocols here, such as the source of the gene and the method of RNA extraction. We will continue to maintain this spirit of unremitting struggle, explore and seek knowledge, and continue our path to “defeating monsters and upgrading”.


</P></DIV></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>