<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SCUT-ChinaA_Overview skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SCUT-ChinaA/Overview</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><TITLE>SCUT-ChinaA</TITLE><HEADER><NAV class="navbar navbar-fixed-top navbar-custom"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle navbar-icon" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
              Team
              </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Team"> Team Members </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Attributions"> Attributions </A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle navbar-icon" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
              Project
              </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Overview"> Overview</A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Design"> Design</A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Experiments"> Experiments </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Results"> Results </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle navbar-icon" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
              Model
              </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Model"> Model </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Software"> Software Tool </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle navbar-icon" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
              Achievement
              </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Parts"> Parts </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Deliverables"> Deliverables </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/InterLab"> InterLab</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle navbar-icon" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
              Practice 
              </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Human_Practices"> Overview </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Silver"> Silver </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Gold"> Gold </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Collaborations"> Collaborations </A></LI><LI><A href="https://2018.igem.org/Team:SCUT-ChinaA/Safety"> Safety </A></LI></DIV></DIV></NAV></HEADER><DIV class="igem_content_wrapper"><DIV class="column full_size"><H2 style="text-align: left">Abstract</H2><P style="text-align: justify">Terpenoid flavor and fragrance compounds (TFFCs) show extensive application in nutraceutical, pharmaceutical and food industries that have a rapid grow market demand. The use of GRAS microorganisms to convert natural raw materials into aroma compounds that can be described as natural products, which has been considered as one of the most promising strategies. However, fermentative TFFCs produced by engineered microbes mostly only obtain intermediates or low yields of end-product currently. This study proposes a non-conditional yeast <EM>Yarrowia lipolytica</EM> as a chassis for TFFCs production, in which limonene was chosen as target product. By employing synthetic biology technology including gibson assembly, CRISPR/Cas9 and protein scaffold, we develop a high-performance enzyme self-assembling system (HESS) to rewire the pathway into limonene accumulation. Furthermore, the MVA pathway will be enhanced by overexpression of two rate-limiting enzymes (HMG1 and ERG12) to increase the production of limonene. This project will provide an alternative metabolic engineering strategy for biosynthesis of TFFCs.</P><OL><DIV class="column jiadefull_size"><FIGCAPTION><H6>Figure 1: Overview of enhancing limonene biosynthesis by a high efficiency enzyme self-assembly system.</H6></FIGCAPTION></DIV></OL></DIV><DIV class="column full_size"><H2 style="text-align: left">Background</H2><OL><H3>1. Limonene</H3><P style="text-align: justify">Due to its unique plant aroma and potential pharmaceutical properties, the terpenoid spice has attracted attention and has a rapidly growing market demand. At present, the most important source is the direct extraction. Due to the limitations of plant growth and other conditions, the product is difficult to purify, and the yield is low, which cannot meet the industrial demand. At the same time, the chemical synthesis of terpenes is difficult or has low yield and poor quality. The biosynthetic method of producing spices is recognized as a natural product and is currently the most important research direction for the breakthrough of terpenoids synthesis. However, because of various types of terpenoids, it is difficult to obtain high-yield production strains due to complex metabolic network and regulation. Therefore, the selection of a suitable chassis, the re-routing of the metabolic network, and the control of metabolism flux can increase terpenoids production.

                    </P><DIV class="column jiadefull_size"><FIGCAPTION><H6>Figure 2: Applications of limonene and limonene-derived molecules.</H6></FIGCAPTION></DIV><DIV class="column full_size"><OL><H3>2. <EM>Yarrowia lipolytica</EM></H3><P style="text-align: justify">The use of GRAS (generally regarded as safe) microorganisms to convert natural raw materials into aroma compounds that can be described as natural products, which have been considered as one of the most promising strategies. <EM>Yarrowia lipolytica</EM> is one of the most widely studied unconventional yeast, which possess the potential ability to become the excellent chassis for monoterpenes production. <EM>Y. lipolytica</EM> meets GRAS standards and its whole genome sequence has been detected. Moreover, with strong acetyl-CoA synthesis ability, <EM>Y. lipolytica</EM> can grow to high biomass yield on a wide carbon source. Its fat granules regulate lipid distribution contributing to the high-yield of monoterpenes. Various metabolic engineering tools has been highly-developed for <EM>Y. lipolytica</EM>. The production of DHA and EPA in <EM>Yarrowia lipolytica</EM> has been commercialized and its production of lycopene is the highest currently.</P><H3>3. SpyTag/SpyCatcher</H3><P style="text-align: justify">
Multi-enzyme complex systems are ubiquitous in the process of biotransformation and material metabolism as enzymes are naturally distributed in the same region thereby forming metabolic channels. SpyTag/SpyCatcher tagging system, as a potential protein scaffold, is based on the CnaB2 domain of Streptococcus pyogenes (Spy). CnaB2 contains an internal isopeptide bond between amino acid residue Lys31 and residue Asp117. CnaB2 can be split into two fragments: An N-terminal protein fragment containing Lys31, named SpyCatcher and a C-terminal peptide containing Asp117, named SpyTag. Reaction occurred in high yield simply in diverse conditions of pH, temperature, and buffer. As the isopeptide bond is covalent, the SpyTag/SpyCatcher complex forms irreversibly and has great stability. The SpyTag can be placed at the N-terminus, at the C-terminus and at internal positions of a protein. Thus, the SpyTag/SpyCatcher system has versatile potential and is suitable for generally use.
			</P><DIV class="column half_size"><FIGCAPTION><H6>Figure 2: Cartoon of SpyTag/SpyCatcher system. Reactive residues are highlighted in red.</H6></FIGCAPTION></DIV></OL></DIV><DIV class="column full_size"><H2 style="text-align: left">Design</H2><OL><H3>1. Pathway Construction</H3><P style="text-align: justify">Limonene has been synthesized in a variety of biological chassis. Literatures have shown that isopentenyl diphosphate (IPP) synthesis from glucose has a natural metabolic pathway in <EM>Yarrowia lipolytica</EM>, which also known as the mevalonate pathway (MVA pathway). To construct the limonene synthesis pathway, two gene encoding neryl diphosphate synthase 1 (NDPS1) and d-limonene synthase (LS) were optimized and heterologously expressed in <EM>Y. lipolytica</EM>. We constructed two expression cassettes for each enzyme first. Based on this, we integrated two expression cassettes together to construct a complete pathway of metabolic synthesis of limonene.</P><DIV class="column jiadefull_size"><FIGCAPTION><H6>Figure 3: the pathway design of limonene synthesis. The yellow arrow represents the rate-limiting enzymes (tHMG1 and ERG12). The orange arrow represents the enzymes (NDPS1 and LS) heterologously expressed in <EM>Y. lipolytica</EM>.</H6></FIGCAPTION></DIV><DIV class="column full_size"><OL><H3>2. Overexpression</H3><P style="text-align: justify">The 3-hydroxy-3-methylglutaryl-coenzyme A (HMG-CoA) reductase is the major rate-limiting enzyme of the MVA pathway in yeasts. The production of isoprenoid biosynthesis can be promoted with the overexpression of the catalytic domain of the HMGR, which is also known as truncated HMG-CoA reductase gene (tHMG1). Thus, we decided to integrate the tHMG1 gene into limonene-producing <EM>Y. lipolytica</EM> cells for production increasing. In addition to the tHMG1 gene, we also co-overexpressed other genes (ERG8, ERG12 and ERG19) involved in the MVA pathway, in order to further explore potential and efficient strategies for improving limonene productivity.</P><H3>3. Scaffold Application</H3><P style="text-align: justify">
Furthermore, by applying the SpyTag/SpyCatcher tagging system, we develop a high-performance enzyme self-assembling system (HESS) to rewire the pathway into limonene accumulation. As its isopeptide bond is covalent, the SpyTag/SpyCatcher complex forms irreversibly and has great stability. Here, we chose to use SpyTag/ SpyCatcher protein assembly systems to build metabolic coupling modules, rewiring the metabolic flux. Attaching Catcher to NDPS1 and Tag to LS, NDPS1 and LS would be assembled together by the spontaneous bonding reaction, thereby enhancing the metabolic flux from IPP to limonene.
			</P></OL></DIV></OL></DIV></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>