Team:SCUT-ChinaA/Design
SCUT-ChinaA
Team
Team Members
Attributions
Project
Overview
Design
Experiments
Results
Model
Model
Software Tool
Achievement
Parts
Deliverables
InterLab
Practice
Overview
Silver
Gold
Collaborations
Safety
Principle
1. Pathway
In the mevalonic acid (MVA) pathway, isopentenyl diphosphate (IPP) and its isomer dimethylallyl diphosphate (DMAPP), the two common building blocks, are the precursors of all terpenoids. Y. lipolytica possesses a native MVA pathway supplying intermediates DMAPP and IPP, but still cannot produce limonene. It has been reported that neryl diphosphate (NPP) instead of its isomer geranyl diphosphate (GPP) is the substrate of limonene in S. cerevisiae. There are also experimental proofs that NPP is an important precursor of the limonene synthesis in Y. lipolytica. NDPS1 catalyzes the conversion of DMAPP and IPP to NPP and LS catalyzes the intramolecular cyclization of NPP into limonene. So to construct the limonene synthesis pathway, NDPS1 and LS are supposed to heterologously express in yeast. Meanwhile, the enzyme 3-hydroxy-3-methylglutaryl-coenzyme A (HMG-CoA) reductase is well known as the major rate-limiting enzyme of the MVA pathway in yeasts. Overexpression of the catalytic domain of the HMGR (producing a form of the enzyme that lacks the membrane-binding region, namely truncated HMG-CoA reductase gene or tHMG1) in yeasts has been shown to boost production of isoprenoid biosynthesis. Thus, the overexpression of tHMG1 gene could increase limonene production in limonene-producing Y. lipolytica cells. Furthermore, to further optimize the flow of metabolic flux, we also look for other enzymes in the synthesis pathway using modeling and experiments. In the metabolic pathway, ERG8 encoding phosphomevalonate kinase, ERG12 encoding mevalonate kinase, and ERG19 encoding mevalonate diphosphate decarboxylase. The three enzymes pull the metabolic flux from Mevalonate and push it to IPP directly, which would causes the accumulation of IPP in cells if overexpressed. This will definitely do significant influence to the increase of limonene production. So in addition to the overexpression of the tHMG1 gene alone, we also look forward to the co-overexpression of the tHMG1 gene together with other genes involved in the MVA pathway, in order to further explore more efficient strategies for improving limonene productivity in Y. lipolytica.
Figure 1: Biosynthesis pathway and bypass of limonene production in Y. lipolytica. Blue arrows represent that the pathways were exogenously integrated in Y. lipolytica, while red arrows represent that the pathways were overexpressed. Single arrows represent the one-step conversions, while triple arrows represent multiple steps.
2. Protein Scaffold
Multi-enzyme complex systems are ubiquitous in the process of biotransformation and material metabolism in nature. In general, enzymes involved in multi-stage reactions are usually distributed in the same region, close to each other, and even combine to form macromolecular complexes, thereby forming metabolic channels. Therefore, there has been an increasing emphasis on the application of protein scaffolds in the construction of metabolic blocks. In the past ten years, several research groups have imitated natural bifunctional enzymes and multi-enzyme complexes, and developed a spatially ordered assembly method that can realize the construction of multi-enzyme complexes using proteins, nucleic acids and polymers as scaffold carriers. Among them, the most successful and typical case is the research of John E Dueber et al in 2009. To solved the problem that when introducing the mevalonate pathway of yeast cells into E. coli, the yield is reduced due to the accumulation of intermediates causing by enzyme activity level, they designed a protein scaffold, into which three key enzymes, AtoB, HMGS and HMGR are assembled by proportional optimization. Thereby, the diffusion of the intermediate is prevented and its diffusing time is reduced, increasing the flow rate of the mevalonate pathway, thereby achieving a 77-fold increase in the production of mevalonate in E. coli. In another research, Robert J. Conrado et al used a more designable DNA scaffold to maximize the pathway flux. By designing and regulating the proportion and spatial ordering of the enzymes to be assembled into scaffold and applying it, the yield of compounds such as resveratrol, 1,2-propanediol and mevalonate obtained by bio-metabolic synthesis pathway have been increased in varying degrees.
Figure 2: Employing metazoan machinery for modular control over pathway flux. This strategy achieved a 77-fold increase in the production of mevalonate in E. coli.
Figure 3: DNA scaffold-assisted assembly of metabolic pathways in E. coli. The yield of 1,2-propanediol obtained by bio-metabolic synthesis pathway is twice the initial output.
SpyTag/SpyCatcher tagging system, as a potential protein scaffold, is based on the CnaB2 domain of Streptococcus pyogenes (Spy). CnaB2 contains an internal isopeptide bond between amino acid residue Lys31 and residue Asp117. CnaB2 can be split into two fragments: An N-terminal protein fragment containing Lys31, named SpyCatcher and a C-terminal peptide containing Asp117, named SpyTag. Reaction occurred in high yield simply in diverse conditions of pH, temperature, and buffer. As the isopeptide bond is covalent, the SpyTag/SpyCatcher complex forms irreversibly and has great stability. The SpyTag can be placed at the N-terminus, at the C-terminus and at internal positions of a protein. Thus, the SpyTag/SpyCatcher system has versatile potential and is suitable for generally use. In our pathway, for the IPP accumulation caused by overexpression, applying the SpyTag/SpyCatcher system to attach NDPS1 and LS together, we developed a high-performance enzyme self-assembling system (HESS) to pull the metabolic flux to NPP and limonene instead of GPP.
3. CRISPR/Cas9
Clustered regularly interspaced short palindromic repeats (CRISPR) is an RNA guided adaptive immunological mechanism in bacteria and archaea during its long-term evolution. CRISPR functions in fighting against the invasive virus and foreign DNA. The class 2 type II DNA-targeting CRISPR/Cas9 system has been developed as a powerful genome-editing tool for its high specificity basing on the nucleic acid sequence. According to its principle, the mechanism of CRISPR/Cas9 system is divided into three distinct stages: adaptation, expression and interference. During the adaptation stage, the complex forming by Cas1 and Cas2 nuclease recognizes foreign target element and integrates it into the CRISPR array. In the following expression stage, transactivating CRISPR RNA (tracrRNA) and CRISPR RNAs (crRNAs) is processed and matured. In addition, the Cas9 gene is expressed. The dual crRNA-tracrRNA can be fused into a chimeric single-guide RNA (sgRNA), and a two-component system composed of Cas9 and its sgRNA is created. Cas9 effector nuclease assembles with sgRNA forming the crRNA-effector complex. Finally, in the stage termed interference, through the sequence-targeting binding of crRNA, the RuvC domain and HNH domain of Cas9 endonuclease cut the top and bottom strand of the invading elements at the target site. The double stranded break (DSB) generating by the cleavage would be repaired by either homologous recombination (HR) or non-homologous end joining (NHEJ).  CRISPR/Cas9 system also has a lot of new developments in Y. lipolytica. In a new research, Cory M. Schwartz et al modified the CRISPR/Cas9 system to develop a highly efficient pCRISPRyl system, and the modified system was successfully used to perform gene knockout and knockin in Y. lipolytica. Professor Sheng Yang, from Institute of Plant Physiology and Ecology of the Chinese Academy of Science, also developed the CRISPR/Cas9 gene editing system for Yarrowia lipolytica and applied it to the transformation of carotene synthesis.
Figure 4: Schematic diagram of gene knockout in Y. lipolytica.
NHEJ preforms the repair for DSB without the need for a homologous template. Ku heterodimer does the recognition and binding of the DSB during the NHEJ repairing. Ku heterodimer recruits multiple subunits, including KU70 and KU80. HR replies on homology based repair process to repair DSB with a homologous template. However, the efficiency of NHEJ to repair DSB is very high, which has been widely mentioned in previous studies. From the mechanism, obviously, the deletion of ku70/ku80 can improve the efficiency of homologous recombination and create conditions for subsequent gene editing in Y. lipolytica. Based on strains with deletion of ku70/ku80, we used the CRISPR/Cas9 system to introduce exogenous and endogenous fragments into specific sites in the yeast genome.
Figure 5: Overview of genome editing by the CRISPR system in Y. lipolytica.
Design
1. Pathway Construction
As mentioned before, to construct a complete limonene synthesis pathway, NDPS1and LS are supposed to be introduced into Y. lipolytica. Here we optimized two key synthetase genes (NDPS1 and LS) and integrated them into the the chromosome of Y. lipolytica Po1f to obtain the engineered strain Po1f/lim. The gene encoding d-limonene synthase (LS, GenBank ID: AY055214.1) from Agastache rugosa and gene encoding neryl diphosphate synthase 1 (NDPS1, GenBank ID: NM_001247704.1) from Solanum lycopersicum were codon-optimized and synthesized by Sangon Biotech. Using gibson assembly technology, the NDPS1 and LS gene were cloned into pUC19, to construct the gene expression cassettes (P-NDPS1-T, P-LS-T), respectively. Based on the plasmid pUC19-rDNAu-HUH-rDNAd (which represents the plasmid pUC19-rDNAupstream-HisG-URA3-HisG-rDNAdownstream), the expression cassettes were cloned into it to construct a co-expression module, as a donor DNA vector.
Figure 6: The co-expression module contents the expression cassettes of NDPS1 and LS. The rDNAupstream and rDNAdownstream is homologous to the rDNA target site and prepared for homologous recombination in further gene editing.
Meanwhile, in order to obtain a high efficient biological chassis for HR, we first constructed pCAS1yl-ku70 plasmid targeting ku70 gene in Y. lipolytica. And then we introduced the plasmids, pCAS1yl-ku70upstream-HUH-ku70downstream into Y. lipolytica, thereby knocking the ku70 gene out, improving the efficiency of homologous recombination repairing. Based on the ku70 deletion strain, we introduced the co-expression module with the plasmid pCAS1yl-rDNA, so the module would be integrated into the rDNA site of the chromosome by CRISPR/Cas9 system, to get a engineered strain named Po1f/lim. The same method would be applied in the subsequent experiments.
Figure 7: Schematic diagram of the plasmid pCAS1yl. The LEU2 was used for selection in Y. lipolytica and the AmpR was used in E.coli.
2. Overexpression
Enhancing the gene copy number of the enzyme on the synthetic pathway, might lead to improvements of the limonene production. For further optimizing the flow of metabolic flux, we decided to overexpress the rate limiting enzymes involved in the limonene synthesis pathway, including tHMG1, ERG8, ERG12 and ERG19. The tHMG1 gene was cloned from the chromosome of the Y. lipolytica, while the ERG8, ERG12 and ERG19 with their terminators were cloned from the genome. Moreover, to increase the application scope of overexpression modules, we assembled the expression cassettes of ERG8/12/19 together for the overexpression of other metabolic pathway. Homologous tHMG1 gene from Y. lipolytica was overexpressed under the control of GPD2 promoter in the limonene-producing strain Po1f/lim. ERG8/12/19 was fused with EXP1, TEF1 and GPD2 promotor, respectively. Then the tHMG1 expression cassettes fused with ku80upstream/downstream was transformed into Po1f/lim with the plasmid pCAS1yl-ku80, simultaneously achieve the purpose of knocking out ku80 gene and introducing tHMG1 gene. The ERG8/12/19 module was integrated into the pox5 site of the genome synchronously, obtaining engineered strain named Po1f/lim-tE.
3. Scaffold Application
The SpyTag/SpyCatcher system has a wide range of applications in enhancing the expression of metabolic pathways and rerouting pathways. We applied SpyTag/SpyCatcher tagging system to develop a high-performance enzyme self-assembling system (HESS) to pull the metabolic flux to NPP and limonene instead of GPP. Considering the size of the proteins structure, we attached SpyCatcher to NDPS1 and SpyTag to LS. On the other hand, we fused SpyCatcher to the C terminal of NDPS1 and SpyTag to the N terminal of LS for ensuring the catalytic activity of the enzymes. So, based on the co-expression module, which contents the expression cassettes of NDPS1 and LS, we developed the HESS to enhance the flow of metabolic pathway. The SpyCatcher gene was cloned into the module between the NDPS1 and XPR terminator, while the SpyTag gene was cloned into the module between the EXP promoter and LS gene. The HESS module would be integrated into the rDNA site of the chromosome in the Po1f/lim-tE to get the strain Po1f/lim-tESS for fermentation.
Figure 8: The co-expression module contents the expression cassettes of NDPS1, SpyCatcher, LS and SpyTag.
Figure 9: Schematic diagram of HESS in the metabolic pathway.
Reference
[1]. John, E.Dueber.; Gabriel, C. Wu.; G.Reza, Malmirchegini.; et al.: Synthetic protein scaffolds provide modular control over metabolic flux. Nature Biotechnology 2009, 27, 8.
[2]. Robert, J.Conrado.; Gabriel, C. Wu.; Jason, T. Boock.; et al.: DNA-guided assembly of biosynthetic pathways promotes improved catalytic efficiency. Nucleic Acids Research 2012, 40, 4, 1879-1889.
[3]. Gianluca, Veggiani.; Tomohiko, Nakamura.; Michael, D.Brenner.; et al.: Programmable polyproteams built using twin peptide superglues. Proceedings of the National Academy of Sciences of The USA 2016, 113(5), 1202-1207.
[4]. Zhang, WB.; Sun, F.; David, A.Tirrell.; et al.: Controlling macromolecular topology with genetically encoded spytag−spycatcher. Journal of the American Chemical Society 2013, 135, 13988-13997.
[5]. Esmer, Jongedijk.; Katarina, Cankar.; Markus, Buchhaupt.; et al.: Biotechnological production of limonene in microorganisms. Applied Microbiology and Biotechnology 2016, 100, 2927-2938.
[6]. Xuan, Cao.; Yu-Bei, Lv.; Jun, Chen.; et al.: Metabolic engineering of oleaginous yeast Yarrowia lipolytica for limonene overproduction. Biotechnol Biofuels 2016, 9, 214.
[7]. Jorge, Alonso-Gutierrez.; Rossana, Chan.; Tanveer, S.Batth.; et al.: Metabolic engineering of Escherichia coli for limonene and perillyl alcohol production. Metabolic Engineering 2013, 19, 33-41.
[8]. Willrodt, C.; David, C.; Cornelissen, S.; Buhler, B.; Julsing, MK.; Schmid, A: Engineering the productivity of recombinant Escherichia coli for limonene formation from glycerol in min-imal media. Biotechnology Journal 2014, 9, 1000-1012.
[9]. Kiyota, H.; Okuda, Y.; Ito, M.; et al.: Engineering of cyanobacteria for the photosynthetic production of limonene from CO2. Journal of Biotechnology 2014, 185, 1-7.
[10]. Davies, FK.; Work, VH.; Beliaev, AS.; et al.: Engineering limonene and bisabolene production in wild type and a glycogen-deficient mutant of Synechococcus sp. PCC 7002. Frontiers in Bioengineering and Biotechnology 2014, 2, 21.
[11]. Cao, X.; Lv,Y.B.; Chen, J.; et al.: etabolic engineering of oleaginous yeast Yarrowia lipolytica for limonene overproduction . Biotechnology for Biofuels 2016, 9 (1) : 214
[12]. Cory, M. Schwartz.; Murtaza, Shabbir, Hussain.; Mark, Blenner.; et al.: Synthetic rna polymerase iii promoters facilitate high-efficiency CRISPR–Cas9-mediated genome editing in Yarrowia lipolytica. ACS Synthetic Biologyl 2016, 5 (4), 356–359
[13]. Gao, SL; Han, LN; Zhu, L; et al.; One-step integration of multiple genes into the oleaginous yeast Yarrowia lipolytica. Biotechnology Letters 2014 36(12) : 2523-2528.
[14]. F.A.G.Gonçalves.; G.Colen.; J.A.Takahashi: Yarrowia lipolytica and its multiple applications in the biotechnological industry. Scientific World Journal 2014, 2014, 476207.
[15]. Yang, X.; Nambou, K.; Wei, L.; et al.: Heterologous production of α-farnesene in metabolically engineered strains of Yarrowia lipolytica. Bioresource Technology 2016, 216, 1040-8.
[16]. Ferrara, MA.; Almeida, DS.; Siani, AC.; et al.: Bioconversion of R-(+)-limonene to perillic acid by the yeast Yarrowia lipolytica. Brazilian Journal of Microbiology 2014, 44(4), 1075-80.
[17]. Jongedijk, E.; Cankar, K.; Ranzijn, J.; et al.: Capturing of themonoterpene olefin limonene produced in Saccharomyces cerevisiae. Yeast 2015, 32, 159–171
