Team:CGU Taiwan/Model
Leijuvant
HOME
ACHIEVEMENTS
PROJECT
MODELING
HUMAN PRACTICES
PEOPLE
INTERLAB
SAFETY
PARTS
TOP
Motivation
Demo
McHug
Results
Notebook
Modeling Motivation
In our 2016 iGEM project, immune and protein information searching is inevitably required. Therefore, McHug is a software platform that is created to arrange your data and search the protein infomation from several databases. We will output your data with a user-friendly interface and you can easily browse the results by submitting in a requested form. The concept of McHug software is originated from 2016 CGU iGEM group. We aim to test the potential of Leishmania to be a new vaccine adjuvant by carrying antigens directly into immune cells. The antigen peptides will be presented on MHCI or II molecules to activate T cells. Therefore, McHug is created to predict the peptides on MHC molecules and help to optimize the peptide presentation and T cell activation. Also, cloning efficiency is considered to be an important step of the experiment. We then expect that this platform and help us shorten the antigen sequence so that it can be more effective to subclone the shuttle vector. The major functions of McHug can be sorted into 3 parts:
1. Protein Structure
Protein structure can affect the possibility of being epitope. Peptides in strong structural sequences like alpha-helix have small chance to be antigenic determinant. On the other hand, sequences in loop structure tend to be recognized by the immune system, specifically by antibodies, B cells, or T cells. Here, McHug shows you the 3D structure of your protein and you can select the peptide sequence to be colored. In case you want to design an epitope sequence to generate antibodies, you can choose sequences on the protein surface in a visible way.
2. MHC Affinity Graphs
Pathogenic proteins will be chopped into peptides and presented by MHC molecules to activate T cells. Therefore, the prediction of MHC affinity in your protein sequence can help you design your experiment. McHug is generated to arrange your numerical data into an easy understanding graph. We can show your IEDB prediction result in a trend chart. Also, users can enter the affinity threshold to curtail the signal in low-affinity position. Each affinity of amino acid in the chart stands for the nonamer starting from the specific amino acid position. Users can easily choose the high-affinity sequence and optimize their experiment.
3. Modification Sites
McHug will provide you the basic information of protein modifications. Moreover, the modification sites will be shown correspondingly to the amino acid position of MHC affinity chart. With the information, users can twig the profile of the protein.
4. Conservation Level
Protein conservation level will be given after BLASTing and multiple sequence aligning the submitted protein sequence. The outcome indicates the protein sequence conservation level between homologous protein sequences in different species. The conservation data will also be shown correspondingly to the amino acid position of MHC affinity chart. Users can choose the highly conserved peptide sequence to perform their experiment. In the project of CGU iGEM 2016, highly conserved region of pathogenic antigens sequence indicates a higher common share of pathogens. This can ensure the high coverage of the vaccine. (Future Work)
Demo
This demo clip was filmed to showcase how to use McHug software and explan the function of result page. The protein ID we used in this clip was OVA protein so that you can see the result of our targeting antigen. The ultimate goal of McHug platform is to integrate several protein databases and provide the users with easy-understanding illustrations.
So far, we are able to show you protein 3D structure on the top of the interface. You can easily zoom in and zoom out to peek every part of your protein. And even select a partial peptide sequence. The peptide sequence in the protein will light up and reveal its position in the 3D structure. Moreover, MHC binding affinity and protein annotations are shown below. Amino acid positions are arranged correspondingly so that you can check all the information side by side. McHug 2016 also features the visualized interface which can transform loads of numerical data into legible charts and all basic protein information are integrated into a canvas penal at the buttom of the page.
Results
McHug
McHug was programmed by computer languages such as Php, javascript, css and html. The major sources from published databases are Uniprot, IEDB and PDB. We thank Dr. Chi-Ching Lee for guiding us programming this software platform. We would also like to thank Chang-Gung Molecular Medicine Research Center for providing us server. This work was supported by several faculties of Chang-Gung University. We are grateful for sponsors and donations from all the supporters.
Result for OVA protein
Based on our experimental design, this result was analysed under condition of mouse MHC molecules and OVA protein. As the penal shown above, there are two MHC high affinity regions on OVA protein which are amino acid 0-50th and 150-250th. However, the 0-50th sequence is also around the signal peptide sequence rendering a possibility of cutting off. Therefore, we predict that sequence 150-250th is the best part to be a high antigenicity peptide sequence. Besides, this sequence will remain its 3D structure due to enough length of the peptide which means B cell will also able to recognize this antigen-like-sequence by binding on it. In the future, we expect that this can help us shortening the sequence cloned into the shuttle vector and enhance the efficiency of transfection by designing this peptide sequence. Moreover, we will test the actual peptides binding to MHC molecules to directly elucidate this result in our in vitro test.
The 150-250th sequence on the 3D structure shows that the sequence is on the shell of the protein. In other words, the sequence has a higher possibility to be a epitope sequence to activate B cell and generate antibody against the OVA antigen. If you further peek into the sequence position, you will find that the 150-250th sequence contains turn structure which is prone to be a high antigenicity part.
Notebook
Week 1 (8/1~8/7)
8/5 Jmol: 3D sturcture    First, we need to get  profile of PDB (protein data bank). We may find a great amount of the same protein under different conditions. As long as you research any protein in any condition, you can submit the data to PDB.
Week 2 (8/8~8/14)
8/12 1. There is a simple Jsmol model code. We can change the code to meet our goals.  http://163.25.92.36/igemcgu/jsmol/jmol/jsmol/lite.htm 2.Our goals are :   Auto spin in the beginning.  We need to have buttons, such as, spin on and spin off.  Most important, link "PDB file" to 3D structuer.     3.In the first goal, we get......  http://163.25.92.36/igemcgu/jsmol/jmol/jsmol/resize_test.htm
Week 3 (8/15~8/21)
8/19 Use " iframe function" to combine each track, such as, MHC class I&II affinity tracks and 3D structure track.  Ex:
Add a new track make user can choose specific amino acids region.  We use " If ( $_POST['APP']) $APP=' ' ;                   Else $AAP="slect $_POST['APP']; color: red;"
Week 4 (8/22~8/28)
8/26 1.1. We already complete "inframe".We also separate two page, one is home page. The other is data page, such as, 3D structure and MHC binding affinity. 3.In the improvement, we want to add "3D structure information when we select amino acid region".  4. And we should find Swiss-Prot ID /PDB ID. http://www.uniprot.org/help/programmatic_access  http://www.uniprot.org/uniprot/P12345.txt    2.Next part, we want to put PDB protein informations.
Week 5 (8/29~9/4)
8/31 1.use "
…......
" to define a track and the button 2.json decode (use to catch the p12345,XML and then )     1. step 1: catch the p12345,XML   shell_exec("Wget  URL  -0  $jobId/ p12345.XML");     2. step 2: read the p12345,XML    $XML_content = file_get_contents(" $jobId/ p12345.XML"     3. step 3: change the XML to Array    $a= json_decode ( $XML_content, true)     4. step 4: print_r($a) first to check      5. step 5: echo $a["feature"]["modified_residue"][…][….];    3.go to http://www.uniprot.org/uniprot/P12345.txt and find the features.     4.xml_parse_into_struct()  can use compare with json_decodegfdgfdgg    5.Next step, use "html5 " to draw fancy picture.
Week 1 (9/5~9/11)
9/2   1.altert of word use "onclick" function to restrict user to submit correct codes or words.    3.use "internal" to show slide of pictures    4.complete the inframe of information which come from different tracks
9/9 1.put MHC class II panel  2.put xml   3.PDB viewer  4.user interface  5.3D structure :           First, select 130-139, and then select 10 , and then label 10.          Second, use php to catch pdb protein and output the amino acid symbol.  6.feature table:         Start, end, and features. (ok)  7.We need to translate these feature word to table or pictures.
Week 2 (9/12~9/18)
Week 3 (9/19~9/25)
9/20 We use "canvas" to perform our protein information. 2.use "in_array($knownList, $each[1])" to decrease the FT information overlap
Week 4 (9/26~10/2)
9/30 1. increase the sample number (eg: notch, tp53, HA, )  2.$infos=array ("helix", "domain","site"….)
Week 5 (10/3~10/10)
