<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CGU_Taiwan_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CGU Taiwan/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><HEADER class="top"><H1 class="headline">Leijuvant </H1><UL class="header-subnav"><LI><A href="https://2016.igem.org/Team:CGU_Taiwan">HOME</A></LI><LI><A href="https://2016.igem.org/Team:CGU_Taiwan/Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2016.igem.org/Team:CGU_Taiwan/Description">PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:CGU_Taiwan/Software">MODELING</A></LI><LI><A href="https://2016.igem.org/Team:CGU_Taiwan/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:CGU_Taiwan/Team">PEOPLE</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:CGU_Taiwan/Interlab">INTERLAB</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:CGU_Taiwan/Safety">SAFETY</A></LI><LI><A href="https://2016.igem.org/Team:CGU_Taiwan/Parts">PARTS</A></LI></UL></HEADER><DIV class="left" style="height:225px; border-color:#140731; border-radius:8px;border-width:2px;border-style:solid;"><A href="#anchor0"><SECTION class="left-tab" style="position:fixed;top:210px;left:35px;" id="myP" onmousedown="mouseDown()" onmouseup="mouseUp()">TOP</SECTION></A><A href="#anchor1"><SECTION class="left-tab" style="position:fixed;top:245px;left:50px;">Motivation</SECTION></A><A href="#anchor2"><SECTION class="left-tab" style="position:fixed;top:280px;left:50px;">Demo</SECTION></A><A target="_blank" href="http://163.25.92.36/igemcgu/igemwebhome.htm"><SECTION class="left-tab" style="position:fixed;top:390px;left:50px;">McHug</SECTION></A><A href="#anchor4"><SECTION class="left-tab" style="position:fixed;top:315px;left:50px;">Results</SECTION></A><A href="#anchor5"><SECTION class="left-tab" style="position:fixed;top:350px;left:50px;">Notebook</SECTION></A></DIV><DIV class="mid" style="top:405px;background-color:white;"><H1 style="font-size:30px;color:#FF8800;text-decoration:none;position:absolute;left:200px;top:70px;margin:20px;">Modeling Motivation</H1><P style="color:black;text-decoration:none;font-size:15px;position:absolute;left:200px;top:110px;margin-right:80px;margin-left:50px;margin-top:30px;text-align:justify;"> 
In our 2016 iGEM project, immune and protein information searching is inevitably required. Therefore, McHug is a software platform that is created to arrange your data and search the protein infomation from several databases. We will output your data with a user-friendly interface and you can easily browse the results by submitting in a requested form. The concept of McHug software is originated from 2016 CGU iGEM group. We aim to test the potential of Leishmania to be a new vaccine adjuvant by carrying antigens directly into immune cells. The antigen peptides will be presented on MHCI or II molecules to activate T cells. Therefore, McHug is created to predict the peptides on MHC molecules and help to optimize the peptide presentation and T cell activation. Also, cloning efficiency is considered to be an important step of the experiment. We then expect that this platform and help us shorten the antigen sequence so that it can be more effective to subclone the shuttle vector. The major functions of McHug can be sorted into 3 parts:<SPAN style="font-weight:bold;">1. Protein Structure</SPAN>
Protein structure can affect the possibility of being epitope. Peptides in strong structural sequences like alpha-helix have small chance to be antigenic determinant. On the other hand, sequences in loop structure tend to be recognized by the immune system, specifically by antibodies, B cells, or T cells. Here, McHug shows you the 3D structure of your protein and you can select the peptide sequence to be colored. In case you want to design an epitope sequence to generate antibodies, you can choose sequences on the protein surface in a visible way.<SPAN style="font-weight:bold;">2. MHC Affinity Graphs</SPAN>
Pathogenic proteins will be chopped into peptides and presented by MHC molecules to activate T cells. Therefore, the prediction of MHC affinity in your protein sequence can help you design your experiment. McHug is generated to arrange your numerical data into an easy understanding graph. We can show your IEDB prediction result in a trend chart. Also, users can enter the affinity threshold to curtail the signal in low-affinity position. Each affinity of amino acid in the chart stands for the nonamer starting from the specific amino acid position. Users can easily choose the high-affinity sequence and optimize their experiment.<SPAN style="font-weight:bold;">3. Modification Sites</SPAN>
McHug will provide you the basic information of protein modifications. Moreover, the modification sites will be shown correspondingly to the amino acid position of MHC affinity chart. With the information, users can twig the profile of the protein.<SPAN style="font-weight:bold;">4. Conservation Level</SPAN>
Protein conservation level will be given after BLASTing and multiple sequence aligning the submitted protein sequence. The outcome indicates the protein sequence conservation level between homologous protein sequences in different species. The conservation data will also be shown correspondingly to the amino acid position of MHC affinity chart. Users can choose the highly conserved peptide sequence to perform their experiment. In the project of CGU iGEM 2016, highly conserved region of pathogenic antigens sequence indicates a higher common share of pathogens. This can ensure the high coverage of the vaccine. (Future Work)</P></DIV><DIV class="mid" style="top:1220px;background-color:white;"><H1 style="font-size:30px;color:#FF8800;text-decoration:none;position:absolute;left:200px;top:50px;margin:20px;">Demo</H1><P style="color:black;text-decoration:none;font-size:15px;position:absolute;left:600px;top:100px;margin-right:80px;margin-left:50px;margin-top:30px;text-align:justify;"> 
This demo clip was filmed to showcase how to use McHug software and explan the function of result page. The protein ID we used in this clip was OVA protein so that you can see the result of our targeting antigen. The ultimate goal of McHug platform is to integrate several protein databases and provide the users with easy-understanding illustrations. 

So far, we are able to show you protein 3D structure on the top of the interface. You can easily zoom in and zoom out to peek every part of your protein. And even select a partial peptide sequence. The peptide sequence in the protein will light up and reveal its position in the 3D structure. Moreover, MHC binding affinity and protein annotations are shown below. Amino acid positions are arranged correspondingly so that you can check all the information side by side. McHug 2016 also features the visualized interface which can transform loads of numerical data into legible charts and all basic protein information are integrated into a canvas penal at the buttom of the page.
</P></DIV><DIV class="mid" style="top:1680px;background-color:white;"><H1 style="font-size:30px;color:#FF8800;text-decoration:none;position:absolute;left:200px;top:50px;margin:20px;">Results</H1><P style="color:black;text-decoration:none;font-size:15px;position:absolute;left:200px;top:100px;margin-right:80px;margin-left:50px;margin-top:30px;text-align:justify;"><BUTTON class="mchug mchug5"><A target="_blank" href="http://163.25.92.36/igemcgu/igemwebhome.htm">McHug</A></BUTTON>McHug was programmed by computer languages such as Php, javascript, css and html. The major sources from published databases are Uniprot, IEDB and PDB. We thank Dr. Chi-Ching Lee for guiding us programming this software platform. We would also like to thank Chang-Gung Molecular Medicine Research Center for providing us server. This work was supported by several faculties of Chang-Gung University. We are grateful for sponsors and donations from all the supporters.  <SPAN style="font-weight:bold;font-size:20px;">Result for OVA protein</SPAN>
Based on our experimental design, this result was analysed under condition of mouse MHC molecules and OVA protein. As the penal shown above, there are two MHC high affinity regions on OVA protein which are amino acid 0-50th and 150-250th. However, the 0-50th sequence is also around the signal peptide sequence rendering a possibility of cutting off. Therefore, we predict that sequence 150-250th is the best part to be a high antigenicity peptide sequence. Besides, this sequence will remain its 3D structure due to enough length of the peptide which means B cell will also able to recognize this antigen-like-sequence by binding on it. In the future, we expect that this can help us shortening the sequence cloned into the shuttle vector and enhance the efficiency of transfection by designing this peptide sequence. Moreover, we will test the actual peptides binding to MHC molecules to directly elucidate this result in our in vitro test.
The 150-250th sequence on the 3D structure shows that the sequence is on the shell of the protein. In other words, the sequence has a higher possibility to be a epitope sequence to activate B cell and generate antibody against the OVA antigen.
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>