Team:CGU Taiwan/Parts
HOME
ACHIEVEMENTS
PROJECT
MODELING
HUMAN PRACTICES
PEOPLE
INTERLAB
SAFETY
PARTS
Biobricks
1. Basic part:
BBa_K1955000 : pSB1C3-Hemagglutinin
BBa_K1955001 : pSB1C3-2.3intron
BBa_K1955002 : pSB1C3-3'UTR
BBa_K1955003 : pSB1C3-5'HYG
BBa_K1955004 : pSB1C3-Ova
2. Composite parts:
BBa_K1955005 : pSB1C3-5'HYG-HA-3'UTR
BBa_K1955006 : pSB1C3-5'HYG-OVA-3'UTR
BBa_K1955007 : pSB1C3-5'HYG-GFP-3'UTR
BBa_K1955008 : pSB1C3-lacI inducible Hemagglutinin
BBa_K1955010 : pSB1A2-lacI inducible GFP
(1) Insert 5’HYG ( BBa_K1955003 ), 3’UTR ( BBa_K1955002 ), HA ( BBa_K1955000 ) and OVA ( BBa_K1955004 ) gBlocks into pSB1C3 vector:
The biobrick parts, including 5’HYG, 3’UTR, HA and OVA, were synthesized directly by IDT. After receiving the synthesized parts, we used EcoRI and PstI to digest the parts and pSB1C3 backbone, then ligated and transformed the DNA samples into DH5a competent cells. According to the digestion and colony PCR results of the colony, all the parts were inserted into the pSB1C3 vector with the right length of DNA sequences, 5’HYG is 1446 bp, HA is 1700 bp, OVA is 2098 bp and 3’UTR is 774 bp.
(Fig. 1) pSB1C3-3’UTR, pSB1C3-5’HYG, pSB1C3-OVA checked by colony PCR and enzyme digestion
(A),(C) The pSB1C3-3’UTR and pSB1C3-OVA were transformed and the colonies were picked to perform colony PCR. The forward primer sequence was 5’- GAATTCGCGGCCGCTTCTAGAG-3’, which was in the prefix site. And the reverse primer sequence was 5’-CTGCAGCGGCCGCTACTAGTA-3’, which was in the suffix site. The PCR reaction was performed with Taq polymerase, and screened in 0.8％ agarose gel by electrophoresis. As the results, a 700~800 bp sequence was proliferated in pSB1C3-3’UTR, and a 2000~2500 bp sequence was proliferated from pSB1C3-OVA. (B) The pSB1C3-5’HYG was transformed and the colonies were picked and amplified in LB broth. pSB1C3-5’HYG plasmid was purified by miniprep, and digested with EcoRI and PstI for 4 hrs, then screened in 0.8％ agarose gel by electrophoresis. The results showed a 2000 bp band of pSB1C3 and the 1500 bp 5’HYG.
(Fig. 2) The basic part checked by PCR
We used pSB1C3-5’HYG, pSB1C3-3’UTR, pSB1C3-HA, pSB1C3-OVA as template, to check the length of the inserts. The PCR reaction was performed with Taq polymerase, and screened in 0.8％ agarose gel by electrophoresis.
(2) The construction of pSB1C3-HA-3’UTR and pSB1C3-OVA-3’UTR ( BBa_K1955006 ):
The pSB1C3-3’UTR was digested with EcoRI and XbaI, then the pSB1C3-HA and pSB1C3-OVA were digested with EcoRI and SpeI. After the purifying step, the pSB1C3-3’UTR was ligated with HA and OVA, then transformed after 16℃ overnight. The colony were checked with colony PCR, as the results, the HA-3’UTR would be about 2.6 kb (1774 bp +774 bp), and the OVA-3’UTR would be about 2.9 kb (2098 bp +774 bp).
pSB1C3-HA-3’UTR and pSB1C3-OVA-3’UTR checked by colony PCR
The pSB1C3-HA-3’UTR and pSB1C3-OVA-3’UTR were transformed and the colonies were picked to perform colony PCR. The PCR reaction was performed with Taq polymerase, and screened in 0.8％ agarose gel by electrophoresis. The 2600 bp HA-3’ UTR and 2900 bp OVA-3’UTR were proliferated from pSB1C3-HA-3’UTR and pSB1C3-OVA-3’UTR.
(3) The construction of pSB1C3-5’HYG-HA-3’UTR ( BBa_K1955005 ) and pSB1C3-5’HYG-OVA-3’UTR ( BBa_K1955006 ) :
The pSB1C3-HA-3’UTR and pSB1C3-OVA-3’UTR were digested with EcoRI and XbaI, while the pSB1C3-5’UTR was digested with EcoRI and SpeI. The pSB1C3-HA-3’UTR, pSB1C3-OVA-3’UTR and 5’UTR were purified by gel extraction, and ligated together. After the transformation step, we used colony PCR to check the correctness of the plasmid. The results showed that the approximately 4100 bp long 5’HYG-HA-3’UTR (1446 bp +1700 bp + 774 bp) and 4500 bp 5’HYG-HA-3’UTR (1446 bp + 2098 bp+ 774 bp) could be amplified from the plasmid, meaning that the pSB1C3-HA-3’UTR, pSB1C3-OVA-3’UTR were finished in the step. In order to transfect the plasmid into leishmania by electroporation, we amplified the plasmid in 200 ml LB broth, and purified the DNA by midiprep.
pSB1C3-HA-3’UTR, pSB1C3-OVA-3’UTR checked by colony PCR
The pSB1C3-5’HYG-HA-3’UTR and pSB1C3-5’HYG-OVA-3’UTR were transformed and the colonies were picked to perform colony PCR. The PCR reaction was performed with Taq polymerase, and screened in 0.8％ agarose gel by electrophoresis. The 4100 bp 5’HYG-HA-3’UTR and 4500 bp 5’HYG-OVA-3’UTR were amplified from pSB1C3-5’HYG-HA-3’UTR and pSB1C3-5’HYG-OVA-3’UTR.
(4) Construction of pSB1C3-2300 intron ( BBa_K1955001 ):
Since the 2300 bp intrinsic sequence contained too many CG pairs, it couldn’t be synthesized. We used point mutation to change the nucleotide in the 2300 bp sequence, therefore, the sequence would be separated into 3 parts, the first and the second part were about 400~450 bp and the third part was approximately 1500 bp in length. Through the PCR, we could have these 3 parts amplified from p6.5 plasmid. We used the PCR-after-ligation strategy, ligating the first and second part together and performed PCR to amplify the sequence. Next, ligated the part 1 +part 2 sequence with part 3, and amplify the ligated parts with PCR again. The reason why we used the PCR-after-ligation strategy was because the ligation rate of the sequence was really low. However, although the parts of 2300 intron could be proliferated by PCR, we were unable to ligate the 3 parts together. The sequencing results of the 2300 intron always lost the second part, no matter what strategy we used in the construction. So, it turned out that we couldn’t put the 2300 intrinsic region into the final construction of our shuttle vector.
All the parts of 2300 intron checked by PCR
The PCR reaction was performed with Taq polymerase, and screened in 0.8％ agarose gel by electrophoresis. Lane A to lane C were the three parts of 2300 intron, the first part was 400 bp, the second part was about 450 bp, and the third part was 1500 bp. Lane D was the ligation of part 1 + part 2, which would be approximately 800 bp. Lane E was the ligation of all three parts, which would be 2.3 kb in length. However, the second part would always be lost during the construction.
(5) Construction of pSB1C3-5’HYG-GFP-3’UTR ( BBa_K1955007 )
Since we can’t detect the HA and OVA protein by western blotting after the pSB1C3-5’HYG-HA-3’UTR and pSB1C3-5’HYG-OVA-3’UTR plasmid were transfected into leishmania. We decided to construct pSB1C3-5’HYG-GFP-3’UTR in order to prove if our leishmania shuttle vector could express the second protein or not. The GFP sequence came from BBa_E0040 in the vector pSB1A2.
The pSB1C3-3’UTR was digested with EcoRI and XbaI, The pSB1A2-GFP and pSB1C3-5’HYG were digested with EcoRI and SpeI. After the purification, the pSB1C3-3’UTR was ligated with GFP and 5’HYG successively, then transformed into DH5a. The colonies were checked by colony PCR. The right length of GFP-3’UTR should be approximately 1.5 kb (720 bp +774 bp), while the 5’HYG-GFP-3’UTR should be about 3 kb (1446 bp +720 bp +774 bp). As the result, we knew that all the colonies contained the correct plasmid after the construction. The right colony of pSB1C3-5’HYG-GFP-3’UTR was picked and amplified in 200 ml LB broth, then the plasmid DNA was purified by midiprep.
pSB1C3-GFP-3’UTR and pSB1C3-5’HYG-GFP-3’UTR check by colony PCR
The PCR was performed with Taq polymerase, and screened in 0.8％ agarose gel by electrophoresis. The GFP-3’UTR was about 1.5 kb in length, and the 5’HYG-GFP-3’UTR was about 3 kb.
(6) Restriction enzyme cutting of pSB1C3-OVA/HA and pSB1A2-GFP:
Used EcoRIHF and XbaI to cut pSB1C3-OVA/HA and pSB1A2-GFP. Due to the short gap between EX restriction cutting site, it is hard to observe double band in the double enzyme digestion.
The whole length of pSB1C3-OVA/HA is approximately 4200bp/3800bp, and pSB1A2-GFP is 3000bp. By double enzyme digestion, all plasmids can be digested into one band (linear form) on the DNA gel.
Fig. 1. Restriction enzyme cutting of pSB1C3-HA and pSB1A2-GFP.
Using EcoRIHF and XbaI with cutsmart enzyme buffer reaction for 3hr. Run a TAE gel for 45 min.
(7) Restriction enzyme cutting of pSB1C3-J04500:
Utilized EcoRIHF and SpeI to digest pSB1C3-J04500, and served as the insert of ligation. J04500 is a 220bp DNA sequence, therefore a 2100bp (pSB1C3) and 220bp band(J04500) can be observed.
Fig. 2. Restriction enzyme cutting of pSB1C3-J04500. Using EcoRIHF and SpeI with cutsmart enzyme buffer reaction for 3hr. Run a TAE gel for 45 min.
(8) Ligation and transformation of pSB1C3-J04500-HA and pSB1A2-J04500-GFP:
According to the calculation of the ligation protocol, we can ligate pSB1C3-HA and pSB1A2-GFP with J04500. After ligation, transform 5μl sample into DH5α competent cell.
Fig. 3. Transformation of the ligation samples. After ligation overnight, transform 5 μl samples into DH5α competent cell following the transformation protocol. Pictures show the colonies of pSB1C3-J04500-HA and pSB1A2-J04500-GFP onto LB plate.
(9) Colony PCR check of pSB1C3-J04500-HA and pSB1A2-J04500-GFP colonies:
With the primer for prefix and suffix, we easily picked up colonies and conducted colony PCR to check the transformation result. The colonies that had successfully insert J04500 into pSB1C3-HA, showed the band size of 220+1700bp (Fig. 4A). As for the negative control, we used pSB1C3-HA(1700 bp). The successful transformed pSB1A2-GFP inserted with J04500 showed the band size of 220+720 bp(Fig. 4B.) and used pSB1A2-GFP as our negative control.
Fig. 4. Colony PCR check of the insertion of J04500 in pSB1C3-HA and pSB1A2-GFP with Taq DNA polymerase and primer for prefix and suffix. (A) Check the insertion of J04500 into pSB1C3- HA, pSB1C3- HA were as the negative control. (B) Check the insertion of J04500 into pSB1A2-GFP, pSB1A2-GFP was as the negative control.
(10) Transform pSB1C3-J04500-HA into BL21 to express desired gene and validate the normal function of J04500 by GFP:
J04500 is a LacI inducible promoter, so it can be induced by IPTG to activate the promoter for transcription. We picked two colonies of pSB1C3-J04500-HA and pSB1A2-J04500-GFP and incubated in 4ml LB broth containing antibiotics at 37℃, 250rpm for overnight. Inoculate 4ml fresh LB supplemented with antibiotics with 20μl of overnight cultures. Shake cultures at 37℃, 250rpm for 3hr. Split cultures into two (2ml each) and add 2μl 1M IPTG to one of the tube to reach 1mM IPTG for induction. The other tube will be the uninduced control. Shake both tubes for another 3hrs at 37℃, 250rpm.
For the pSB1C3-J04500-HA, their induction can be checked by Coomassie blue or Western blot analysis. Transfer cultures from both tubes to two Eppendorf tubes and centrifuge bacteria at max speed for 3min. Discard the supernatant, and resuspend each pellet in 100μl 1x SDS sample buffer by pipetting. Fit cap-guards onto Eppendorf tubes and boil the samples at 100℃ for 10min. Run SDS-PAGE to check the expression of desired protein.
Conduct the Western blot analysis, we successfully recognize the HA protein (approximately 62.3kd) in our two colonies (JH1 and JH2) when induced by IPTG (Fig. 5A).
To check the feasibility of J04500, we also built up a construct containing J04500 and GFP. After induction of 3hrs, the induced bacteria turn fluorescent green. In contrast, the uninduced bacteria did not appear to have any color change. (Fig. 5B)
Fig. 5. Protein expression of pSB1C3-HA and pSB1A2-GFP. The day before induction, two different colonies were picked up from each construct into 4ml LB broth at 37℃, 250rpm for overnight. Inoculated 4ml fresh LB broth with 20μl of overnight cultured at 37℃, 250rpm for 3hrs. Splited into two tubes(each 2ml), and added 2μl 1M IPTG to one tube as the induced group, the other tube without IPTG was the uninduced group. IPTG induction for 3~4hrs. (A) After IPTG induction, centrifuged bacteria at max speed. Discarded the supernatant, lysed the cell with 100μl 1x SDS sample buffer. Western blotting to check the desired gene expression. In fig. 5A, His-tag HA serves as positive control to validate that the antibody can work normally. We also transform pUC-19 into BL21 and follow the induction protocol as the negative control. (B) IPTG induction of pSB1A2-J04500-GFP to verify the LacI inducible promoter. The GFP fluorescence can be observed easily by bare eye.
TOP
SITE MAP
HOME
ACHIEVEMENTS
INTERLAB
SAFETY
PARTS
PROJECT
Design
Background
Results
Proof
Notebook
HUMAN PRACTICES
Survey
Education
Communication
Entrepreneurship
MODELING
McHug
OUR TEAM
Collaborations
Attributions
Sponsors
