"
Page
Discussion
View source
History
teams
Log in
 
Team:UANL Mty-Mexico/Modeling/transport and accumulation
From 2012.igem.org
iGEM UANL 2012
Home
Team
Members
Profile
Project
Abstract
Overview
Detection
Capture
Recovery
Circuit
Modeling
Overview
Transport
Biosensor
Silica Binding
Molecular Modeling
Outreach
Human Practices
SynBio Tunnel
Notebook
Cloning strategy
Wetlab
Protocols
Parts
Safety
Overview
Attributions
Transport and accumulation
Before us, team iGEM Groningen 2009 made a model for an arsenic accumulator at the population level; that is, they set some ODEs that represent the change on the total intracellular arsenic (considering not a single cell, but the whole culture, or more exactly, the total cell volume) with respect to time. Nevertheless, as the precise value for some parameters were unavailable, specially for the ArsB effect, part of their model remains aparameterized and they perform a quasi-steady state analysis.
After considering the effect of their metallothioneins (As-binding proteins), GlpF, ArsB and ArsR, they ended with the following time derivative:
\begin{equation} \large\frac{\mathrm{d[As(III)in] } }{\mathrm{d} x} = -ArsR_{As}-MBPArsR_{As} -n_{f}\cdot fMT_{As} -k_{1} ArsB_{As} + \frac{k_{2}V_{s}GlpF_{As}}{V_{c}}  \end{equation}
Where As(III)in is the total intracellular arsenic; ArsRAs, MBPArsRAs, fMTAs, ArsBAs and GlpFAs are the arsenic bound proteins; nf is the Hill coefficient for the interaction between As and fMT; k1 and k2 are the kinetic constants for the interaction between As and ArsB and GlpF, respectively; finally, Vs/Vc represents the proportion between the total solution volume (Vs) and the total cell volume (Vc).
Core model
We built upon their model and made the following modifications, which we'll call the "core modifications" from now on:
We assume that ArsB is non functional, so that the only protein affecting As transport is GlpF.
GlpF effect is masked by the population level kinetics.
We assume that the intracellular As concentration and the GlpF effect at the population level (that is, considering total cell volume) are homogeneously distributed and should be the same as in a single cell.
The protein MBPArsR is not present in our system, so the variable MBPArsRAs is not considered for our model.
The next equation shows the application of those modifications:
\begin{equation} \large\frac{\mathrm{d[As(III)in] } }{\mathrm{d} x} = -ArsR_{As} -n_{f}\cdot fMT_{As} + (\frac{V_s}{V_c}) \cdot V_{max} \cdot \frac{As_{ex}}{K_{t}+As_{ex}}  \end{equation}
Now, let us introduce a variable called AsTOTALin, which represents the total amount of arsenic inside a cell (recall core modification number 2), whether free or bound to whatever protein. Let's also call AsFREEin the amount of free intracellular arsenic and AsBOUNDin the protein-bound As. In this way, AsTOTALin can be represented as follows: \begin{equation} \large[As_{TOTALin}] = [As_{FREEin}] + [As_{BOUNDin}] \end{equation}
In the iGEM Groningen 2009 model, Asin represents the free intracellular arsenic, as this variable has negative terms related to the binding of As to proteins; to avoid further confusions, we'll establish this equivalence as follows: AsFREEin, so that equation 2 changes to:
\begin{equation} \large[As_{FREEin}] = [As_{in(iGEM Groningen 2009)}] \end{equation}
If we further analyze the AsBOUNDin variable, considering that in our system only ArsR and a methalothionein (which we'll simply call MT) are being expressed, then equation 5 turns to be:
\begin{equation} \large\large[As_{TOTALin}] = h_{2}[ArsR|As] + h_{3}[MT|As] \end{equation}
Where h2 and h3 are the Hill coefficients for the interaction between As and ArsR and MT, respectively.  Finally, taking into consideration the work of team Cambridge 2009 and assuming that equilibrium is reached quickly, we can describe the formation kArsR|As and kMT|As as follows:
\begin{equation} \large \frac{d[ArsR|As]}{dt} = (ArsR_{TOTAL}) - (\frac{[ArsR_{TOTAL}]}{1+(\frac{[As_{FREEin}])}{k_{[ArsR|As]}})^{h_2}} - \delta_{ArsR|As}[ArsR|As]  \end{equation}
\begin{equation} \large \frac{d[ArsR_{FREE}]}{dt} = (\frac{[ArsR_{TOTAL}]}{1+(\frac{[As_{FREEin}])}{k_{[ArsR|As]}})^{h_2}} - \delta_{ArsR|As}[ArsR|As]  \end{equation}
\begin{equation} \large \frac{d[MT|As]}{dt} = (MT_{TOTAL})-(\frac{[MT_{TOTAL}]}{1+(\frac{[As_{FREEin}])}{k_{[MT|As]}})^{h_3}} - \delta_{MT|As}[MT|As]  \end{equation}
\begin{equation} \large \frac{d[MT_{FREE}]}{dt} = (\frac{[MT_{TOTAL}]}{1+(\frac{[As_{FREEin}])}{k_{[MT|As]}})^{h_3}} - \delta_{MT|As}[MT|As]  \end{equation}
In equations 5 to 7, kArsR|As and kMT|As are the kinetic constants for the interaction of arsenic with ArsR and MT, respectively, using a different nomenclature as in equations 1 and 2; here, in equations 6 and 7, the binding of two molecules is represented as "moleculeA|moleculeB". The indexes h2 and h3 are the Hill coefficients for the interaction between arsenic and ArsR and MT, respectively. The deltas are the degradation constants for the protein|As complexes. The unbound As that results from complex degradation then goes to the AsFREEin pool and is ready to bind again available ArsR or MT.
ODEs
The core modifications and equation 6 allow us to propose a set of ODEs that describe the change of the concentrations of intracellular As, ArsR|As, MT|As and the unbound protein species.
Core model ODEs
mRNAs
\begin{equation} \large \frac{d[mRNA_{ArsR}]}{dt} = \alpha _{mArsR}\cdot (pro_{ars})\cdot(\frac{k_{D1}^{h_{1}}}{k_{D1}^{h_{1}}+[ArsR]^{h_{1}}})- \delta _{mRNA_{ArsR}}[mRNA_{ArsR}] \end{equation}
\begin{equation} \large \frac{d[mRNA_{MT}]}{dt} = \alpha _{mMT}\cdot(pro_{cons})- \delta _{mRNA_{MT}}[mRNA_{MT}] \end{equation}
Proteins
\begin{equation} \large \frac{d[ArsR]}{dt} = \alpha _{pArsR}\cdot[mRNA_{ArsR}]- \delta _{ArsR}[ArsR] - [ArsR|As] \end{equation}
\begin{equation} \large \frac{d[MT]}{dt} = \alpha _{pMT}\cdot[mRNA_{MT}]- \delta _{MT}[MT] - [MT|As] \end{equation}
Proteins with arsenic
See equations 7 and 8
Arsenic
\begin{equation} \large\frac{\mathrm{d[As_{FREEin}] } }{\mathrm{dt}} = (\frac{V_s}{V_c} \cdot V_{max} \cdot \frac{As_{e}}{K_{t}+As_{e}} + h_2 \delta _{ArsR|As}[ArsR|As] + h_3 \delta _{MT|As}[MT|As]) \end{equation}
Parameters
Transcription Rates
Based on the arithmetic average of five experimental references for the mean transcription rate in E. coli, we used in our model a value of 48.18 nt/s, or 2890.8 nt/min [ Gotta, Miller Jr and French (1991), , Vogel and Jensen (1994) , Bremer and Dennis, (1996) in Neidhardt, et al. , Gene Expression Modelling ] for this parameter. Assuming that transcription for all genes occurs at this speed, and based on the supposition that 1 nM equals one molecule per cell in E. coli ( Bionumbers ), we propose the following equation to estimate the maximum transcription rate for every transcriptional unit in the model:
Maximum transcription rate = Average transcription speed (2890.8 nt/min)*Number of copies of the plasmid
The size of a transcriptional unit takes into account the CDS plus 3' and 5' untranslated regions.
Translation Rates
A similar process was followed to calculate the maximum translation rates for all the proteins in our model. Using an average translation rate of 19 aa/s, or 1140 aa/min [ Bremer and Dennis, (1996) in Neidhardt, et al. ], and assuming that all translation in our system works at the same speed, the maximum translation rate can be written as:
Maximum translation rate = Average translation rate (1400 aa/min⁡〖)/〗 [Protein size (aa)]*RBS strenght(Assumed as 1 due to lack of data)
mRNA and Protein degradation rates
The degradation rates for all mRNAs were obtained based on the half-lives (in minutes) and the cell division rates, and expressed as the sum of the actual degradation rate ln(2)/half life and the dilution rate (Ln(2)/cell duplication time, 30 minutes). When the half-lives of the mRNAs used in our system were not available in the literature, we assumed them to be the average half-life of mRNA in E. coli, 6.8 minutes, Selinger, GW, et al. (2003) .
According to Varshavsky, (1997) and Tobias et al., (1991) (8), when the N-terminal aminoacid of a protein in E. coli is K, R, L, F, Y or W, the half-life of that protein will be as short as 2 minutes; otherwise, it will be greater than 10 hours. Since none of the proteins in our system begins with any of the aminoacids listed above, and because the term Ln(2)/600 has a value very close to zero, the actual degradation rate for the protein will not be taken into consideration, except when the half-life is available in the literature. Thus, the protein degradation rates will be equal to the dilution rate (Ln(2)/cell duplication time, 30 minutes).
Cell volume and solution volume
The total cell volume was fixed in all simulations 0.0073 mL and the solution volume was 1.1 mL
Parameter table
Parameter
Description
Value
References
αmArsR
Maximal transcription rate of ArsR
3.74 nM/min
Assumptions
αmMT
Maximal transcription rate of MT
5.08 nM/min
Assumptions
αpArsR
Maximal translation rate of ArsR
9.74 nM/min
Assumptions
αpMT
Maximal translation rate of MT
6.33 nM/min
Assumptions
δmRNAArsR
Degradation rate of ArsR mRNA
2.16x10-1 min-1
Assumptions, Selinger, et al. (2003)
δmRNAMT
Degradation rate of MT mRNA
1.25x10-1 min-1
Assumptions
δArsR
Degradation rate of ArsR
2.31x10-2 min-1
Assumptions
δMT
Degradation rate of MT
2.31x10-2 min-1
Assumptions
proars
Concentration of ars promoter
aprox. 1*plasmid copy (nM)
Assumptions
procons
Concentration of constitutive promoter
aprox. 1*plasmid copy (nM)
Assumptions
KD1
Dissociation constant for the interaction of ArsR and proars
330 nM
Groningen 2009
KD1
Dissociation constant for the interaction of ArsR and As
6000 nM
Groningen 2009
KD1
Dissociation constant for the interaction of MT and As
6000 nM
Unknown; taken same value as KD2
kt
Kinetic constant for the transport of extracellular arsenic
27.21 µM
Groningen 2009
h1
Hill coefficient for the interaction between ArsR and proars
2
Groningen 2009
h2
Hill coefficient for the interaction of ArsR and As
1
Wei-Ping (1996)
h3
Hill coefficient for the interaction of MT and As
6
Ngu and Stillman (2006)
VS/VC
Relation between total solution volume(VS) and total cell volume (VC)
6.64x10-3
Groningen 2009
Vmax
Vmax for the Michaelis-Menten equation that describes As transport
3.186 µM/sL
Groningen 2009
Simulations
For the simulations of the ODEs, we built a Simulink model. First, we set the extracellular As to zero and ran a simulation with the initial conditions for all variables set to zero, as well; then, we interpolated the graphs obtained for the variables ArsR and MT (both in protein and mRNA) and determined the following initial conditions for further simulations:
mRNAs: ArsR = 2.1058 nM; MT = 24.6324 nM
Proteins: ArsR = 887.7 nM; MT = 6748.4978 nM
We also set all the numerical integrators to have lower saturation limits equal to zero, to be in concordance with the mass conservation law. Here are the results for simulations at extracellular As set to 0, 0.1, 1, 5 and 10 μM (0, 100, 1000, 5000, 10000 nM).
Figure 1.Time vs total bound arsenic
Figure 2.Extracellular arsenic, total intracellular arsenic and free intracellular arsenic vs time
The color code is the same in figures 1 and 2. In the first one, we show the simulations for the five different extracellular arsenic concentration; in the second one, we see the simulated dynamics of the total internal arsenic and the free internal arsenic.
Remember that we assume that the arsenic concentration in the total cell volume should be equally distributed; in consequence, the concentration inside a cell should be the same as the one at the total cell volume.
MT plasmid copy number effect
For the simulations presented in figures 1 and 2, the copy plasmid of each plasmid is assumed to be equal to 1; also, we assume that the concentration of one molecule in the volume of a cell should be approximately equal to 1 nM, so that "(1 nM)*(plasmid copy number)" should give a result the concentration of a promoter.
Here we show simulations varying the copy number of the plasmid containing MT (we used copy numbers 1, 5, 10 and 100) at 500 nM extracellular arsenic.
Figure 3. Total captured As vs time at different plasmid copy number for MT
Simulink model: Transport and accumulation
You can find our Simulink model for the transport and accumulation module here .
>>Transport and accumulation
Core model
ODEs
Parameters
Simulations
Copy number effect
Simulink model: Transport and accumulation
Biosensor
Silica binding
Retrieved from " http://2012.igem.org/Team:UANL_Mty-Mexico/Modeling/transport_and_accumulation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
