<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_British_Columbia_crispr skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:British Columbia/crispr</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>British_Columbia_Base</TITLE></P><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2017.igem.org/Team:British_Columbia/autoinduction">Autoinduction</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/crispr">CRISPR-Cas9</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/conjugation">Conjugation</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/plasmid_maintenance">Plasmid Maintenance</A></LI></UL><LI><A href="https://2017.igem.org/Team:British_Columbia/HP/Silver">Human Practices Silver</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/HP/Gold_Integrated">Human Practices Gold</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/Engagement">Engagement and Education</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/team_bios">Bios</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/sponsors">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/Notebook">Notebooks</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/Deliverables">Judging Deliverables</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/Parts">Parts</A></LI></DIV></DIV><DIV class="row fix" id="design"><DIV class="col-sm-10"><H1 class="section-heading">Overview</H1><P>CRISPR is a novel system that allows us to alter genes with unprecedented precision and accuracy (Barrangou, 2007). CRISPR, which stands for Clustered Regularly Interspaced Palindromic Repeats, comes in many flavours all with unique abilities to modify organism as we see ethically fit. The most commonly known CRISPR subtype is Cas9,  an endonuclease, that causes double strand DNA breaks. In essence, CRISPR is the vaccination machinery of a bacterium, containing short sequences derived from foreign genetic elements and causing cleaving and degradation of potential threats. These short sequences are transcribed into RNA, typically 20 base pairs long, that bind to the Cas9 enzyme guiding it to a complementary sequence and cleave it. The CRISPR/Cas system has been modified for gene editing, and it has become a widely used tool for both medical and biological researchers. The most widely used variant of the CRISPR systems is CRISPR/Cas9, which is performed by transfecting cell with the Cas9 and a single-guide RNA (sgRNA) (Peters, 2015).</P><P>The sgRNA is a simpler version of the CRISPR RNA, which is constituted by fusing a conserved RNA sequence and a target-complementary sequence. Normally, the target-complementary sequence will be 20 base pairs in length. For each target sequence, a protospacer adjacent motif (PAM site) is required to facilitate Cas9 protein binding to the target DNA. The PAM sequences are specific to the enzymes and are required for successful DNA cleavage; for <I>S.pyogenes</I> CRISPR/Cas9, the CRISPR variant British_Columbia uses, the PAM site is 5’ - NGG - 3’. The popularity of the CRISPR/Cas9 approach has resulted in a variety of software tools to design guide RNA, this normally take into account non specific targeting which could have major implications on your experimental approach.</P><P>The ability of CRISPR/Cas9 to edit the genome relies on the double stranded break repair pathway of its host organism. Normally eukaryotic cells are able to undergo double stranded break repair through two major pathways: Non Homologous End Joining Repair (NHEJ), and Homologous Repair (HR).  In some prokaryotes like <I>Agrobacterium</I>, HR is the dominant approach according to Zuniga-Castillo (2004). HR is only successful upon the presence of donor DNA, when this is absent the double stranded break results in lethality for the organism or plasmid curing by causing degradation of the targeted DNA.</P></DIV></DIV><DIV class="row fix" id="key_achievements"><DIV class="col-sm-10"><H1 class="section-heading">Key Achievements</H1><UL><LI>Performed the first <B>functional analysis of exogenous Cas9 expression in <I>Agrobacterium</I></B>, to our knowledge</LI><LI>Successfully <B>demonstrated the CRISPR/Cas9 activity in <I>Agrobacterium</I></B></LI><LI>Developed a functional assay of Cas9 cleavage of the Ti Plasmid</LI><LI>Implemented model generated sgRNA sequences into <I>Agrobacterium</I> to target the virulence operon</LI><LI>Performed guide assembly design to integrate multiple guide sequences into a single plasmid containing CRISPR/Cas9</LI><LI>Cloned Cas9 and sgRNA into a modified plant transformation vector - pCAMBIA-MCS1</LI></UL></DIV></DIV><DIV class="row fix" id="methods"><DIV class="col-sm-10"><H1 class="section-heading">Project Background</H1><P><I>Agrobacterium</I> strains only exhibit the ability to infect plants when it hosts the Ti plasmid, which gives it the molecular tools necessary to transform plant cells. The integration of the transfer-DNA (T-DNA) segment from the Ti plasmid into the plant genome causes a persistent and chronic infection, which will lead to the formation of a tumour, otherwise known as Crown Gall.</P><P>The machinery that is involved in the injection of this T-DNA is present as the <I>vir</I> operon within the Ti plasmid. In theory, using our CRISPR/Cas9 system to target a gene within this operon would render the <I>Agrobacterium</I> non-pathogenic, while keeping the cell viable. Since the Ti plasmid is kept at a low copy number, and it comes with a high metabolic cost without a plant host, there will be a low rate of homologous recombination. Thus, the Ti plasmid would be degraded. If recombination does occur, a contingent strategy is required. To ensure the <I>vir</I> operon has been disabled, a second CRISPR/Cas9 target can included into the system. The guides will continue to cut the targeted sites until the guide sequences no longer exists inside the cell, either neutralizing the targeted gene entirely or curing the Ti plasmid. </P><P>We decided to target the <I>virD2</I> gene, given its essential role in pathogenesis and degree of conservation in Ti plasmids. This <I>virD2</I> gene is involved in a complex that cleaves off the T-DNA from the Ti plasmid and transports it into the transfer machinery to inject it into a plant cell (Christie &amp; Gordon, 2014). Without the <I>virD2</I> gene, T-DNA cleavage from the Ti plasmid is highly unlikely, leading to the inability to transform plants.</P><P>There are two kinds of Cas9 proteins, the enzymatically active Cas9 and the enzymatically ‘dead’ Cas9 (dCas9). One of the application of dCas9 is the inhibition of gene expression, by preventing RNA polymerases from reading the gene. Therefore, dCas9 battles between being bound and unbound to its target gene, leading to background expression levels of a gene. If we wanted to eliminate all possibility of infection, we had to rely on Cas9’s endonuclease activity to cut and degrade the Ti plasmid.</P><P>The backbone of our Cas9 construct was a pCAMBIA vector, which has the ability to replicate within <I>E. coli</I> and <I>Agrobacterium</I>. This vector is commonly used for plant genome transformation, similar to a Ti plasmid surrogate. We cloned in our Cas9 gene and our sgRNA downstream to the Cas9 gene, into our modified pCAMBIA-MCS vector.  A common method used in CRISPR/Cas9 research is having the sgRNA and Cas9 coding sequences in separate plasmids. However, this did not seems optimal for the purposes of our experiments since the probability of a sgRNA and a Cas9 plasmid to be conjugated into the same cell would be lower than if both were present in the same plasmid. Naturally, all the cloning work was performed initially into <I>E. coli</I>since it allowed us to work in a better understood organism and led to higher copy numbers of our construct. </P></DIV></DIV><DIV class="row fix" id="results"><DIV class="col-sm-6"><H1 class="section-heading">Project Strategy</H1><P>Our final construct, pGUIDE, consisted of Chloramphenicol and Kanamycin resistance genes, the Cas9 gene, an sgRNA gene regulated by a tetracycline operator, and plasmid maintenance genes initially included on the plasmid backbone.
We started our cloning by using the pCAMBIA plasmid, which contained the necessary genetic components to replicate in E. coli and <I>Agrobacterium</I>. It also housed the Kanamycin resistance gene under a constitutive promoter. In our first round of cloning with our pCAMBIA plasmid, we noticed that there was a segment of DNA present in the plasmid that was not annotated. We identified this segment since our digest fragments were shifted by one kilobase pairs when running a gel. This was most likely due to the the plasmid’s use in plant transformation. To work around this unknown component, we added a Multi-Cloning Site, and sequenced confirmed its presence.</P><P>Our pCAMBIA-MCS plasmid allowed us to have greater certainty that our Cas9 would be successfully cloned in. We probed the sequence of both the MCS sequence of our pCAMBIA-MCS and pCas9-CR4 plasmid to find shared unique restriction enzyme cut sites in both plasmids. One of our goals was to keep both the selection markers in the plasmid. pCAMBIA-MCS marker is Kanamycin, while pCas9-CR4 is Chloramphenicol. We discovered that SpeI and SacII would provide us with the desired result. 
Given that our pCAMBIA-Cas9 plasmid contained two antibiotic resistant genes, we could be sure that the colonies grown in LB + Kan + Cm plates were successful clonings. To be more thorough with our results, we sequenced confirmed the presence of Cas9 in our plasmid.</P></DIV><DIV class="col-sm-4"><P><SMALL>Figure 1. Plasmid cloning plan for CRISPR project.</SMALL></P></DIV></DIV><DIV class="row"><DIV class="col-sm-10"><P>Our next goal was to clone in the sgRNA gene blocks that our modelling team generated. Prior to synthesis of sgRNA sequence, added two flanking RE cut sites to the sgRNA. To identify the optimal cut sites, we searched our pCAMBIA-Cas9 plasmid for unique cut sites downstream of our Cas9 gene. SpeI and PstI were decided us our ideal cut sites.</P><P>To address potential plasmid recombination, we decided to recruit two sgRNA in one pGUIDE plasmid. Due to synthesis limitations, we could not order a gene block containing two sgRNA. Therefore, we decided on ordering the sgRNA in separate fragments flanked with either SpeI - HindIII, HindIII - PstI, or PstI - HindIII, depending on the desired combination. After ligating the sgRNAs at the HindIII cut site, we would clone them into our pCAMBIA-Cas9 plasmid using SpeI and PstI cut sites, creating our dual pGUIDE.</P><P>For pGUIDE, both the Cas9 and sgRNA constructs are under control of tetracycline induction. The incentive for having both under control of the same inducer is to minimize the variability of sgRNA and CRISPR concentration. For an environmental application, the CRISPR/Cas9 system would be under control of an environmental inducer, however, the inducible promoters used for other functions in <I>Agrobacterium</I> were not experimentally characterized well enough to perform our CRISPR/Cas9 validation. Our team’s related work on inducible <I>Agrobacterium</I> promoters can be seen <A href="https://2017.igem.org/Team:British_Columbia/autoinduction"><U>here.</U></A></P><P>Once our pGUIDE plasmid was formed, having all the components described earlier, the plasmid was transformed into <I>Agrobacterium</I> through electroporation.</P></DIV></DIV><DIV class="row"><DIV class="col-sm-6"><P>Following our first attempt to clone in the sgRNAs to our construct, we sought to confirm the presence of sgRNAs via restriction digest and gel electrophoresis. Every digest shown above from “A1”-”Neg2” should be various sgRNA sequences cloned into our plasmid, and “pCambia+Cas9” serves as our control, since it is the plasmid we were attempting to insert with sgRNAs. </P><P><I>Expected results</I>
Digesting “pCambia+Cas9” with SacII and Pst1 results in two fragments that are both almost exactly 6000 base pairs, so we would expect to see a single thick band at 6000 bp on a gel.  This lane looks as expected, since we see a single band at 6000 bp. Each construct from “A2”-”Neg2” should be the “pCambia+Cas9” plasmid with a 150-400 insert depending on the lane, so we are expecting all of the lanes to look similar to “pCambia + Cas9” except with one band slightly above 6000. Clearly this is not what we observed on our gel, which led us to doubt the success of these clones. That being said, 4 lanes (*starred in green) appeared nearly as expected. We rationalized that the upper band in “B1” could be undigested plasmid, so this lane was also considered a success.</P></DIV><DIV class="col-sm-4"><P><SMALL>Figure 2. Gel eletrophoresis showing restriction digests with SacII and Pst1 on various pGuide constructs.</SMALL></P></DIV></DIV><DIV class="row"><DIV class="col-sm-10"><P>Given these results, we decided to proceed with experimentation using “B1”, “BC-1” and “Neg 1”, which will be explained below.</P></DIV></DIV><DIV class="row fix" id="conclusion"><DIV class="col-sm-10"><H1 class="section-heading">Guide RNA Design</H1><P>CRISPR/Cas9 targets DNA through complementary sequences between a single-guide RNA and a DNA strand. One of the main concerns surround CRISPR technologies is the potential for off-targets, which are when a sgRNA has multiple specificities. Naturally, our goal was to limit the number of off-targets one sgRNA would have. With this aim, our team developed a model based on a bioinformatic and biophysical approach (Farasat &amp; Salis, 2016) in order to find the sgRNA sequences with the fewest off target events. More information on the model can be found <A href="https://2017.igem.org/Team:British_Columbia/Model"><U>here.</U></A></P><P>Our energy based sgRNA model was used to design the sgRNA for our system. To confirm that the <I>Agrobacterium</I> strain we used contained this plasmid, we tested growth in antibiotics that <I>Agrobacterium</I> is known to have resistance to. Our <I>Agrobacterium</I> grew successfully in gentamicin and rifampicin, identifying it as a <I>Agrobacterium</I> strain containing a TiC58 variant, instead of an Ach5 Ti plasmid. With this information, we opted to use the TiC58 plasmid sequence, which we noted as the standard in the field. 
We wanted our CRISPR/Cas9 to display functionality in multiple strains with <I>virD2</I> targets. There were differences between the <I>virD2</I> on the TiC58 and the Ti Ach5, so the model was run with both variants, finding all potential guides for both plasmids. 
The gene sequences input into the model for the TiC58 plasmidtargeting can be accessed at www.ncbi.nlm.nih.gov and are as follows: <I>Agrobacterium tumefaciens</I> str. C58 linear chromosome (AE007870.2), <I>Agrobacterium tumefaciens</I> str. C58 circular chromosome (AE007869.2), <I>Agrobacterium tumefaciens</I> str. C58 plasmid Ti (AE007871.2),   <I>Agrobacterium tumefaciens</I> str. C58 plasmid At (AE007872.2), pCambia1305.1 (http://www.snapgene.com/resources/plasmid_files/plant_vectors/pCAMBIA1305.1/), pCas9-CR4 (https://www.addgene.org/62655/), and the gene input was the <I>virD2</I> region in the TiC58 plasmid. 
For targetting the Ach5 Ti plasmid, the genome files used were ASM97156v1 (https://www.ncbi.nlm.nih.gov/assembly/GCF_000971565.1), along with pCambia1305.1, pCas9-CR4, and the gene input was the <I>virD2</I> region from the Ach5 Ti plasmid.</P><P>Guides from both Ti plasmids were ranked based on their partition functions and the sequences were compared to determine guides compatible with both strands. When determining compatibility between sequences, the five nucleotides next to the PAM site were required to be homologous, as these nucleotides are most important for Cas9 binding. From the pool of homologous sgRNA obtained from both Ti plasmid sequence, cross-referenced with the <I>Agrobacterium</I> genome, we selected the three sgRNA sequences: CGATATTGGGCATAAGTGAT with a partition function of 84.902, 
AGGCCGTAAGATAGTTGTAT with a partition function of 84.489, and TCAACAAATTATCAATCAGT with a partition function of 73.445. With these guides, CRISPR/Cas9 experiments could be performed in <I>Agrobacterium</I> to target the <I>virD2</I> gene and test effectiveness of our construct.</P></DIV></DIV><DIV class="row fix" id="references"><DIV class="col-sm-10"><H1 class="section-heading">CRISPR/Cas9 Toxicity in <I>Agrobacterium tumefaciens</I></H1><H2>Introduction</H2><P>An important facet of our CRISPR/Cas9 system is the effect of the Cas9 protein in the <I>Agrobacterium</I>. Whenever there is exogenous expression of a protein, it is crucial to identify any inherent toxicity or phenotype that our protein may present in its new host. Therefore, one of our goals was to characterize any deleterious, toxic, effects that Cas9 may have on <I>Agrobacterium</I>.</P><P>Currently, we have not been able to find literature that has attempted to demonstrate Cas9’s effect on <I>Agrobacterium</I> growth. Therefore, we designed an assay to quantitatively measure the effect of Cas9 at varying levels of induction. Moreover, our Cas9’s expression is induced by anhydrotetracycline, a common effector used in eukaryotic genetic engineering. Anhydrotetracycline is a derivative of tetracycline, which is known to be a bacteriostatic effector. Therefore, we implemented a similar assay to quantify the effect of anhydrotetracycline on <I>Agrobacterium</I> growth rate.</P><H2>Methods</H2><P>Cultures of <I>Agrobacterium tumefaciens</I> GV3101 with and without pCambia-Cas9 were inoculated 24 hours before the start of the assay in 5 mL of LB media with 10 µg/mL Rifampicin and 50 µg/mL Kanamycin at 30°C. After 24 hours, cultures were diluted to 0.100 OD600, induced with anhydrotetracycline at 0 ng/mL (control), 21 ng/mL, 42 ng/mL or 84 ng/mL and incubated on a shaker at 30°C and 220 rpm. OD600 values were measured with a spectrophotometer at 0, 1, 2, 8 and 24 hours after addition of anhydrotetracycline.</P><H2>Results</H2><P>Anhydrotetracycline slowed growth regardless of pCambiaCas9 plasmid presence, but cultures with the plasmid were slowed to a greater extent. Higher doses of anhydrotetracycline showed increasing growth inhibition at all time points. From 0 hours to 8 hours, <I>Agrobacterium</I> without pCambiaCas9 and 42 ng/mL anhydrotetracycline had the least inhibition, but did not recover as well as other conditions after 24 hours.</P><CENTER><SMALL>Figure 3. 24 hour normalized growth curves of pCambia-based constructs. Anhydrotetracycline concentrations for each condition shown in legend.</SMALL></CENTER><H2>Conclusions</H2><P>The purpose of this assay was to determine whether Cas9 induction or anhydrotetracycline itself would be toxic to <I>Agrobacterium</I>. The results indicate that both of these factors slow growth, but are not conclusively cytotoxic. It was expected that cultures harboring pCambiaCas9 would grow more slowly than those without, because they must commit to synthesizing the Cas9 protein in addition to their normal metabolism. A more likely explanation for the reduced growth in conditions expressing the Cas9 protein would be unguided endonuclease cuts made more frequent by the increased concentration of the Cas9 construct. Anhydrotetracycline also slowed growth in cultures that did not have any plasmids, which shows evidence of some other inhibitory effect on cellular growth aside from metabolic burden. It follows logically that increasing concentrations of the inducer had a greater effect. Despite modest levels of growth inhibition, we know from this assay that anhydrotetracycline is a viable inducer for Cas9 in <I>Agrobacterium</I> for future experiments.</P><H2>Considerations</H2><P>This preliminary test for Cas9 toxicity occurred before we had successfully cloned our final CRISPR/Cas9 constructs with sgRNAs included, so it is important that we repeat a toxicity assay with the sgRNAs included.</P><H1><I>In vitro</I> proof of concept: CRISPR/Cas9-mediated deactivation of Ti plasmid in <I>Agrobacterium tumefaciens</I></H1><H2>Introduction</H2><P>In order to prove that our idea for treatment of A. tumefaciens is viable, we needed to demonstrate that:</P><P><OL><LI>CRISPR/Cas9 behaves as expected in <I>Agrobacterium</I>.</LI><LI>sgRNAs designed by our model were functional and effective at disrupting Ti plasmids.</LI></OL></P><P>We tackled this challenge by taking advantage of the Ti plasmid’s gentamicin resistance cassette. If a cell has the Ti plasmid it will be gentamicin resistant, and if we can disrupt the plasmid via Cas9-mediated double-stranded breaks, it will lose its gentamicin resistance. We designed this assay to show loss of gentimicin resistance by an OD-based growth curve as well as demonstrating a visual reduction via plating. </P><H2>Methods (Preparation):</H2><P>Overnight cultures of 3 biological replicates of 5 different constructs in <I>Agrobacterium</I> were inoculated two days before the assay. The constructs are as follows:</P><H2>Constructs</H2><P><UL><LI><B>pCambia</B>: The basic cloning vector used for <I>Agrobacterium</I>. Confers Kanamycin resistance, and has no other relevant features for this assay. Used as a negative control for Cas9 activity.
  </LI><LI><B>pCambia-Cas9</B>: Cas9 with an ATC inducible promoter inserted into pCambia.  No sgRNAs are provided with the Cas9 protein, so it should not cleave the the Ti plasmid when induced.
  </LI><LI><B>pCambia-Cas9-N1</B>: Derived from pCambia, this plasmid contains Cas9 with an ATC inducible promoter and sgRNA scaffold, but no guide sequence itself. This will be our most important negative control, as it is identical to the guide constructs except it lacks the 20-nucleotide guide.
  </LI><LI><B>pCambia-Cas9-B1</B>: Same as above, but has a single complete sgRNA that targets the <I>virD2</I> gene on Ti plasmids. When induced with ATC, we expect this construct to cleave the Ti plasmid.
</LI><LI><B>pCambia-Cas9-BC1</B>: Same as above but instead has 2 complete sgRNAs that target different regions of the <I>virD2</I> gene. We expect that this construct will also cleave the Ti plasmid when induced with ATC.</LI></UL></P><P>After two days of growth, each of the 15 cultures were read via plate reader to determine their OD, and split in to the following four conditions at 0.100 OD:</P><H2>Conditions</H2><P><UL><LI><B>LB + Kanamycin</B>: Base medium. All of our constructs confer Kanamycin resistance, so any cell with our plasmids will grow in this condition. This is our negative control for all conditions.</LI><LI><B>LB + Kanamycin + ATC</B>: Same as above but contains the Cas9 inducer, ATC at 42 ng/mL. Given that there is no gentamicin in this medium, we expect all cells to grow even if Cas9 is working as intended. This media condition is used as a control for ATC effect on growth.</LI><LI><B>LB + Kanamycin + Gentamicin</B>: Since the only source of gentamicin resistance is the Ti plasmid, we expect only cells with the Ti plasmid to grow. Since ATC is not present, we do not expect Cas9 to have activity in this condition.</LI><LI><B>LB + Kanamycin + Gentamicin + ATC</B>: Same as above, but ATC is included. This means that cells will need an intact Ti plasmid to grow in this media, but Cas9 will be induced to cleave the Ti plasmid. We are expecting pCambia-Cas9-Guide constructs to reduce growth in this condition due to Ti plasmid degradation.
</LI></UL></P><P>Each culture was grown in 2 mL of the appropriate media at 30°C, shaking at 220 rpm.</P><H2>Methods (OD readings):</H2><P>At t=4, t=13, and t=24 hours, 200 uL of each culture was loaded in to a 96-well plate and OD was read using a plate reader.</P><H2>Methods (Spot plating):</H2><P>Each overnight culture was diluted to 0.001 OD. 5 uL of this dilution was spotted in technical triplicates on both Kanamycin + Gentamicin plates, and Kanamycin + Gentamicin + ATC plates. All plates were allowed to settle for 30 minutes and then incubated at 30°C for two days.</P><H2>Results</H2><P>Significant growth reduction was not observed for any construct in any media condition aside from LB + Kanamycin (Kan) + Gentamicin (Gent) + Anhydrotetracycline (ATC) (Figure 4, 5). A visual representation of Kan + Gent and Kan + Gent + ATC can be seen in Figure 6. This demonstrates that our assay environment is properly controlled and fits our expected results. For the LB + Kan + Gent + ATC condition, pCambia-Cas9-N1 did not show growth reduction, while pCambia-Cas9-B1 and pCambia-Cas9-BC1 showed significantly reduced growth after 24 hours. This indicates that our guides are successfully targeting the Ti plasmid in <I>Agrobacterium</I> and that growth reduction is not caused by Cas9 induction alone.</P><CENTER><SMALL>Figure 4. 24 hour growth curves of A. tumefaciens cultures grouped by construct being tested. All cultures were grown in 2 mL of media at 30°C and shaking at 220 rpm.</SMALL></CENTER><CENTER><SMALL>Figure 5. 24 hour growth curves of A. tumefaciens cultures grouped by media condition.  All cultures were grown in 2 mL of media at 30°C and shaking at 220 rpm.</SMALL></CENTER></DIV></DIV><DIV class="row"><DIV class="col-sm-6" style="padding-top: 12%;"><H2>Conclusions</H2><P>Given our numerous controls, we can conclude that our CRISPR/Cas9 constructs disrupted the functionality of Ti plasmids in <I>Agrobacterium</I>, causing them to lose gentamicin resistance. By comparing sgRNA “N1” with sgRNA “B1” and “BC1”, we have shown that the loss of resistance entirely depends on sgRNAs targeted to the Ti plasmid with the Cas9 protein present, rather than only being due to Cas9 activity. For guide “B1”, since no growth reduction was observed when gentamicin was not present and Cas9 was induced, we have shown that disrupting the Ti plasmid does not inherently inhibit growth of <I>Agrobacterium</I>. Given that the guides used in this experiment were derived from our biophysical model of CRISPR/Cas9 activity (see modeling page), we have proven that our model can generate effective sgRNAs with little user input.
These in vitro findings are critical proofs that CRISPR/Cas9 can function as a specific, programmable and controlled method of Ti plasmid disruption in <I>Agrobacterium tumefaciens</I>, which could ultimately prevent infection of plants without affecting growth of any microbes involved.</P></DIV><DIV class="col-sm-4"><SMALL>Figure 6.  5 uL of 0.001 OD cultures of various constructs plated on selective plates in the presence and/or absence of anhydrotetracycline. Plates were incubated for two days at 30°C.</SMALL></DIV></DIV><DIV class="row"><DIV class="col-sm-10"><H1 class="section-heading">References</H1><P>Barrangou et al (2007). CRISPR Provides Acquired Resistance Against Viruses in Prokaryotes. Science 315 (5819), 1709 - 1712</P><P>Christie, P. J., &amp; Gordon, J. E. (2014). The Agrobacterium Ti Plasmids. Microbiology Spectrum, 2(6), 10.1128/microbiolspec.PLAS–0010–2013. http://doi.org/10.1128/microbiolspec.PLAS-0010-2013</P><P>Farasat I, Salis HM (2016) A Biophysical Model of CRISPR/Cas9 Activity for Rational Design of Genome Editing and Gene Regulation. PLOS Computational Biology 12(1): e1004724. 
https://doi.org/10.1371/journal.pcbi.1004724</P><P>Peters, J. M., Silvis, M. R., Zhao, D., Hawkins, J. S., Gross, C. A., &amp; Qi, L. S. (2015). Bacterial CRISPR: Accomplishments and Prospects. Current Opinion in Microbiology, 27, 121–126. http://doi.org/10.1016/j.mib.2015.08.007</P><P>Zuñiga-Castillo, J., Romero, D., &amp; Martínez-Salazar, J. M. (2004). The Recombination Genes addAB Are Not Restricted to Gram-Positive Bacteria: Genetic Analysis of the Recombination Initiation Enzymes RecF and AddAB in Rhizobium etli. Journal of Bacteriology, 186(23), 7905–7913. http://doi.org/10.1128/JB.186.23.7905-7913.2004</P></DIV></DIV><TITLE>British_Columbia_Base</TITLE></DIV></DIV></DIV></DIV></DIV></BODY></HTML>