<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_British_Columbia_plasmid_maintenance skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:British Columbia/plasmid maintenance</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>British_Columbia_Base</TITLE></P><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2017.igem.org/Team:British_Columbia/autoinduction">Autoinduction</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/crispr">CRISPR-Cas9</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/conjugation">Conjugation</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/plasmid_maintenance">Plasmid Maintenance</A></LI></UL><LI><A href="https://2017.igem.org/Team:British_Columbia/HP/Silver">Human Practices Silver</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/HP/Gold_Integrated">Human Practices Gold</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/Engagement">Engagement and Education</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/team_bios">Bios</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/sponsors">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/Notebook">Notebooks</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/Deliverables">Judging Deliverables</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:British_Columbia/Parts">Parts</A></LI></DIV></DIV><DIV class="row fix" id="overview"><DIV class="col-sm-6"><H1 class="section-heading">Overview</H1><P>
        Toxin-antitoxin systems are widespread throughout the prokaryotic world. Typically the systems are composed of two genes:
        toxin and antitoxin, where the toxin is more stable than the antitoxin but the latter is expressed to a higher level.
        (Melderen &amp; De Bast, 2009) Toxin-antitoxin systems provide a selective pressure on plasmid maintenance, as plasmid loss leads
        to the loss of supply of the antitoxin which then can no longer counteract the effect of the toxin. Consequently,
        the toxin is activated and can act on the bacterial cell killing it. (Unterholzner, Poppenberger, &amp; Rozhon, 2013)
        In the system we have chosen, <I>ietAS</I>, literature searches have revealed that the antitoxin <I>ietA</I> is similar to an AAA-ATPase
        and the toxin <I>ietS</I> is similar to subtilisin-like serine proteases. (Yamamoto et Al., 2009) Therefore the killing mechanism
        for our system is thought to be proteolysis although further characterization is needed and ongoing. </P></DIV></DIV><DIV class="row"><DIV class="col-sm-10"><P> In the aGROW system, the CRISPR-Cas9 system targeting <I>Agrobacterium tumefaciens</I> virulence factors adds metabolic
        burden to a cell, therefore the plasmid carrying Cas9 may get lost from the population. To counter this and ensure
        plasmid maintenance, we have employed a toxin-antitoxin native to <I>Agrobacterium</I> tumour inducing plasmids. The
        toxin-antitoxin system we have used is native to the pTIC58 plasmid and is essential for high level incompatibility with
        other plasmids of the repABC family as well as contributing to the general stability of the plasmid.
      </P></DIV></DIV><DIV class="row fix" id="key_achievements"><DIV class="col-sm-10"><H1 class="section-heading">Key Achievements</H1><UL><LI>Biobricked <I>ietA</I> (antitoxin), <I>ietS</I> (toxin), and <I>ietAS</I> (toxin antitoxin) </LI><LI>Subcloned <I>ietAS</I> into pCAMBIA plasmid</LI><LI><B>Characterized <I>ietAS</I> toxin antitoxin complex</B> in <I>Agrobacterium</I> to compare plasmid loss</LI></UL></DIV></DIV><DIV class="row fix" id="design"><DIV class="col-sm-10"><H1 class="section-heading">Design</H1><P>
        After an extensive literature search we identified <I>ietA</I><I>ietS</I> toxin-antitoxin system, originally identified by Yamamoto et
        Al., as our main candidate for plasmid maintenance in <I>Agrobacterium</I>. As the two regions were
        fairly large and we were not working with the wild type strain of <I>Agrobacterium</I> that carries the pTiC53 plasmid,
        we decided to synthesize them as IDT gBlocks.
         The genes are part of an operon and we kept them under the native promoter. The two fragments we ordered
        were <I>ietA</I> which consisted of the nucleotides between position 97,533 and 98859 in GenBank AE007871.2 and <I>ietS</I> which
        consisted of the nucleotides between position 98863 and 101373 in GenBank AE007871.2. In order to biobrick our fragments
        we ordered the gBlocks with the bio bricking suffix and prefix flanking the edges. We also removed internal Eco-RI,
        Xba-I, Spe-I, and Pst-I cutsites by changing interchanging codons using <A href="http://www.kazusa.or.jp/codon/cgi-bin/showcodon.cgi?species=176299"><U>this codon usage chart</U></A> for <I>Agrobacterium</I>        .
         To have a construct work in <I>Agrobacterium</I> we needed to use a plasmid that could replicate efficiently
        in <I>Agrobacterium</I>. We chose pCambia 1305.1 a standard plasmid used for <I>Agrobacterium</I> mediated plant
        transformation. For our purposes we only wanted to use the plasmid’s backbone which included: KanR, a Kanamycin resistance
        cassette, ori a high copy number pUC origin of replication for <I>E. coli</I>, bom a basis of mobility from pBR322,
        pVS1 oriV origin of replication for <I>Agrobacterium</I>, pVS1 RepA, and pVS1 StaA. We also wanted to insert a multicloning
        site (MCS) to our design which we ordered as a gBlock from IDT. The sequence for our MCS was: GCGTTCCGCGGGTTCTTAAGGTTCTCGAGGTTAAGCTTGTTAGATCTGTTGAATTCGCGGCCGCTTCTAGAGTACTAGTAGCGGCCGCTGCAGGT
TGGATCCGTTCCATGGGTTGCATGCGTTCG.</P></DIV></DIV><DIV class="row fix" id="methods"><DIV class="col-sm-10"><H1 class="section-heading">Methods</H1><P>
        The strains used were <I>Agrobacterium tumefaciens GV3101</I> and <I>Escherichia coli</I> DH5-Alpha unless otherwise
        stated. All <I>E. coli</I> cultures were grown in LB media at 37℃ unless otherwise stated. All <I>Agrobacterium</I> were grown in LB at 30℃ unless otherwise stated. All plasmid DNA extractions were performed with ABM Column-Pure
        Plasmid Mini-Prep Kit. DNA purification from gels with ABM Column-Pure DNA Gel Recovery Kit. The same protocols for
        DNA gel electrophoresis, restriction enzyme digest, DNA ligation, and chemical and electro transformation were used
        throughout unless otherwise stated and can be found in <A href="https://2017.igem.org/Team:British_Columbia/Protocols"><U>Protocols</U>.</A></P></DIV></DIV><DIV class="row fix"><DIV class="col-sm-6"><H2>Cloning <I>ietAS</I> into psb1c3</H2><P>
        Cloning <I>ietAS</I> into pSB1C3 was done in two consecutive rounds of standard restriction enzyme cloning. The <I>ietA</I> and <I>ietS</I> gBlocks
        ordered from IDT and pSB1C3 were digested with Eco-RI and Spe-I enzyme. The two pieces of pSB1C3 were separated and
        purified by gel purification. pSB1C3 was ligated with <I>ietA</I> and <I>ietS</I> in two separate ligation reaction overnight.
        The resulting ligations were then transformed into chemically competent <I>E. coli</I>. Transformants were
        picked 24 hours later and grown overnight in LB-Chloramphenicol. Overnight cultures were miniprepped and checked
        by restriction enzyme digest using Eco-RI and Pst-I enzymes, transformants identified with the right size band (1.3
        kB for <I>ietA</I> and 2.5 kB for <I>ietS</I>) were checked by Sanger sequencing using VF2 and VR primers.
         To combine the two constructs and place <I>ietA</I> upstream of <I>ietS</I>, pSB1C3 <I>ietS</I> was digested using Eco-RI And
        Xba-I enzymes and pSB1C3 <I>ietA</I> was digested using Eco-RI and Spe-I. Both digests were separated and purified by gel
        electrophoresis and gel clean up kits. The <I>ietA</I> fragment was ligated with the pSB1C3 fragment to form a the full
        <I>ietAS</I> construct. The resulting ligation was then transformed into chemically competent <I>E. coli</I>. Transformants
        were picked 24 hours later and grown overnight in LB-Chloramphenicol. Overnight cultures were miniprepped and checked
        by restriction enzyme digest using Eco-RI and Pst-I enzymes, transformants identified with the right size band (3.8
        kB for <I>ietAS</I>) were checked by Sanger sequencing using VF2 and VR primers.
      </P><H2>Cloning in the Multiclonal Site into pCAMBIA</H2><P>
        To clone in our MCS into pCambia we digested pCambia with Sac-II and Sph-I which cut the plasmid at the edge of the
        plasmid backbone, removing the left and right border regions necessary for plant transformation. We separated and
        purified this digest by gel electrophoresis. Additionally, we digested our MCS IDT gBlock with Sac-II and Sph-I enzymes.
        We ligated the two fragments overnight and proceeded to transform them into chemically competent <I>E. coli</I>.
        Transformants were picked and grown overnight in LB-Kanamycin, and then mini prepped. The minipreps were checked
        by Sanger sequencing using custom primers. This new pCambia plasmid with inserted
        MCS was used by all other teams when cloning parts to be used in <I>Agrobacterium</I>.
      </P><H2>Cloning <I>ietAS</I> into pCambia</H2><P>
        To insert <I>ietAS</I> into pCambia we subcloned <I>ietAS</I> from pSB1C3. We digested pSB1C3 <I>ietAS</I> and pCambia MCS with Eco-RI and Spe-I
        enzymes. We separated and purified our digests using gel electrophoresis. Next, we proceeded to ligate the <I>ietAS</I> and
        pCambia digests together overnight. The overnight ligations were then transformed into chemically competent <I>E. coli</I>.
        Transformants were selected the following day and grown overnight in LB-Kanamycin. Cultures were then miniprepped
        and and digested, first with Eco-RI and Pst-I enzymes, then with Hind-III to check on a gel that the inserts were
        the correct size. The minipreps were then transformed into electrocompetent <I>Agrobacterium</I> the standard electroporation
        protocol (6) which can be found in <A href="https://2017.igem.org/Team:British_Columbia/Protocols"><U>Protocols</U>.</A></P><H2>Characterization of <I>ietAS</I> in <I>Agrobacterium</I></H2><P>
        To determine whether <I>ietAS</I> was effective at maintaining plasmids in <I>Agrobacterium</I> without antibiotic pressure
        we did a multiday plating assay. 3 biological replicates of both pCambia <I>ietAS</I><I>Agrobacterium</I> and pCambia <I>Agrobacterium</I> were grown in 3 mL of LB overnight. Each day serial dilutions ranging from 10<SUP>-3</SUP> to 10<SUP>-8</SUP> were made from each replicate
        and spot plated on LB plates and LB-Kan plates. 3 uL of each culture was passaged into a fresh 3 mL of LB and
        grown to repeat the assay next day. On the first and last day, we also plated on LB-Gent and LB-Gent-Kan plates
        to see if <I>ietAS</I> contributed to the loss of the modified Ti plasmid present in <I>Agrobacterium GV3101</I>. Full details
        of this assay can be found in <A href="https://2017.igem.org/Team:British_Columbia/Protocols"><U>Protocols</U>.</A></P></DIV><DIV class="col-sm-4"><P style="font-size: smaller"><B>Figure 1</B> Cloning procedure for <I>ietAS</I> into pCambia and into pSB1C3.
      </P><P style="font-size: smaller"><B>Figure 2</B> pCambia <I>ietAS</I> digested with Hind-III, expected band sizes are 10kb(plasmid and segment <I>ietAS</I>) and
          700kb(partial segment of <I>ietA</I>) as seen in the photo.
        </P></DIV></DIV><DIV class="row fix" id="results"><DIV class="col-sm-4"><P style="font-size: smaller"><B>Figure 3</B> Plasmid stability plating assay, day 2. Top-left: pCambia without <I>ietAS</I> in <I>Agrobacterium</I> plated
          on LB-Kan agar. Bottom-left: pCambia without <I>ietAS</I> in <I>Agrobacterium</I> plated on LB agar. Top-right: pCambia
          with <I>ietAS</I> in <I>Agrobacterium</I> plated on LB-Kan agar. Bottom-right: pCambia with <I>ietAS</I> in <I>Agrobacterium</I>          plated on LB agar. Each sample is plated as two technical replicates over five dilutions based on an OD 600 reading.
        </P><P style="font-size: smaller"><B>Figure 4</B> Effect of <I>ietAS</I> on plasmid stability. CFU ration is the number of CFU's on LB-Kan agar plate divided
          by the number of CFU's on LB agar plate. See protocols section for details. Each CFU value is the average of three
          biological replicates, each biological replicate had two technical replicates and was corrected against OD600 .
          Vertical bars represent standard deviation. On day 0 for ietAS there were too many colonies to count so we were unable to quantify that results for that day. 
        </P></DIV><DIV class="col-sm-6"><H1 class="section-heading">Results</H1><H2>Cloning</H2><P>
        Cloning of <I>ietA</I>, <I>ietS</I>, and <I>ietAS</I> into pSB1C3 in E. coli was successful and confirmed by Sanger sequencing and restriction
        enzyme digest. Cloning the MCS into pCambia in E. coli was successful and confirmed by Sanger sequencing.
         Subcloning <I>ietAS</I> into pCambia was attempted several times, with the last two times working. The <I>ietAS</I> pCambia
        cloning was confirmed using restriction enzyme digest with Eco-RI and Pst-1 as well as Hind-III(Figure 2) the resulting
        digests were run on a gel and the correct bands were observed. <I>ietAS</I> pCambia miniprep was successfully transformed
        into Agrobacterium.</P><H2>Plasmid Stability</H2><P> As cloning <I>ietAS</I> into pCambia delayed our progress for 3 weeks, we were unable to fully
        characterize the efficacy of <I>ietAS</I> on the plasmid stability. We ran a plasmid stability assay for 3 days using spot
        plating to compare the amount of colony forming units on LB versus LB-Kanamycin (Figure 3). Unexpectedly we noticed
        a decrease in the ratio of CFU's on LB-Kan to CFU's on LB with the <I>ietAS</I> (Figure 4).
        Yamamoto et. Al saw at 15-fold decrease in colony forming units with the addition of <I>ietAS</I> with the
        plasmid containing the sacB gene conferring hypersensitivity to sucrose. In our plating experiment we did not see
        a significant difference between the pCambia plasmid containing <I>ietAS</I> and pCambia plasmid without <I>ietAS</I>. This could
        be due to the presence of pVS1 StaA on pCambia, a stabilizing protein which is shown to maintain plasmids containing
        it for 100 generations growing at 30℃ (Heeb et Al., 2000). In future work we could perform the same expirement knocking
        out the pVS1 staA gene in pCambia.
      </P></DIV></DIV><DIV class="row"><DIV class="col-sm-10"><H1 class="section-heading">Conclusion</H1><P>For our project we created a new composite Bio-Brick part that is shown in the literature to increase plasmid incompatibility
        with other repABC plasmids. This part is also shown in the literature to increases plasmid stability through toxin-antitoxin
        interaction. Having this part in our aGROW plasmid would add selective pressure to keep the plasmid in <I>Agrobacterium</I>        despite the negative fitness of harboring CRISPR-Cas9. Further work on characterization of the <I>ietAS</I> construct can
        be done to quantify the added level of stability that <I>ietAS</I> adds to a plasmid, this can be used in conjunction with
        literature values. This is the first toxin-antitoxin part specifically for <I>Agrobacterium</I> that has been entered
        into the Biobrick registry of standard biological parts.</P><H1 class="section-heading">References</H1>

      Heeb, S., Itoh, Y., Nishijyo, T., Schnider, U., Keel, C., Wade, J., Haas, D. (2000). Small, stable shuttle vectors based on the minimal pVS1 replicon for use in gram-negative, plant-associated bacteria. Molecular Plant-Microbe Interactions, 13(2), 232-237. doi:10.1094/MPMI.2000.13.2.232 Melderen, L. V., &amp; Bast, M. S. (2009). Bacterial Toxin–Antitoxin
      Systems: More Than Selfish Entities? PLoS Genetics, 5(3). doi:10.1371/journal.pgen.1000437 Unterholzner, S.
      J., Poppenberger, B., &amp; Rozhon, W. (2013). Toxin–antitoxin systems: Biology, identification, and application. Mobile
      Genetic Elements, 3(5), e26219. http://doi.org/10.4161/mge.26219 Yamamoto, S., Kiyokawa, K., Tanaka, K., Moriguchi,
      K., &amp; Suzuki, K. (2009). Novel Toxin-Antitoxin System Composed of Serine Protease and AAA-ATPase Homologues Determines
      the High Level of Stability and Incompatibility of the Tumor-Inducing Plasmid pTiC58. Journal of Bacteriology, 191(14),
      4656-4666. doi:10.1128/jb.00124-09</DIV></DIV><TITLE>British_Columbia_Base</TITLE></DIV></DIV></DIV></DIV></DIV></BODY></HTML>