Team:British Columbia/plasmid maintenance
British_Columbia_Base
Autoinduction
CRISPR-Cas9
Conjugation
Plasmid Maintenance
Human Practices Silver
Human Practices Gold
Engagement and Education
Bios
Attributions
Sponsors
Interlab
Protocols
Notebooks
Judging Deliverables
Collaborations
Parts
Overview
Toxin-antitoxin systems are widespread throughout the prokaryotic world. Typically the systems are composed of two genes:         toxin and antitoxin, where the toxin is more stable than the antitoxin but the latter is expressed to a higher level.         (Melderen & De Bast, 2009) Toxin-antitoxin systems provide a selective pressure on plasmid maintenance, as plasmid loss leads         to the loss of supply of the antitoxin which then can no longer counteract the effect of the toxin. Consequently,         the toxin is activated and can act on the bacterial cell killing it. (Unterholzner, Poppenberger, & Rozhon, 2013)         In the system we have chosen, ietAS, literature searches have revealed that the antitoxin ietA is similar to an AAA-ATPase         and the toxin ietS is similar to subtilisin-like serine proteases. (Yamamoto et Al., 2009) Therefore the killing mechanism         for our system is thought to be proteolysis although further characterization is needed and ongoing.
In the aGROW system, the CRISPR-Cas9 system targeting Agrobacterium tumefaciens virulence factors adds metabolic         burden to a cell, therefore the plasmid carrying Cas9 may get lost from the population. To counter this and ensure         plasmid maintenance, we have employed a toxin-antitoxin native to Agrobacterium tumour inducing plasmids. The         toxin-antitoxin system we have used is native to the pTIC58 plasmid and is essential for high level incompatibility with         other plasmids of the repABC family as well as contributing to the general stability of the plasmid.
Key Achievements
Biobricked ietA (antitoxin), ietS (toxin), and ietAS (toxin antitoxin)
Subcloned ietAS into pCAMBIA plasmid
Characterized ietAS toxin antitoxin complex in Agrobacterium to compare plasmid loss
Design
After an extensive literature search we identified ietA ietS toxin-antitoxin system, originally identified by Yamamoto et         Al., as our main candidate for plasmid maintenance in Agrobacterium. As the two regions were         fairly large and we were not working with the wild type strain of Agrobacterium that carries the pTiC53 plasmid,         we decided to synthesize them as IDT gBlocks.
The genes are part of an operon and we kept them under the native promoter. The two fragments we ordered         were ietA which consisted of the nucleotides between position 97,533 and 98859 in GenBank AE007871.2 and ietS which         consisted of the nucleotides between position 98863 and 101373 in GenBank AE007871.2. In order to biobrick our fragments         we ordered the gBlocks with the bio bricking suffix and prefix flanking the edges. We also removed internal Eco-RI,         Xba-I, Spe-I, and Pst-I cutsites by changing interchanging codons using this codon usage chart for Agrobacterium .
To have a construct work in Agrobacterium we needed to use a plasmid that could replicate efficiently         in Agrobacterium. We chose pCambia 1305.1 a standard plasmid used for Agrobacterium mediated plant         transformation. For our purposes we only wanted to use the plasmid’s backbone which included: KanR, a Kanamycin resistance         cassette, ori a high copy number pUC origin of replication for E. coli, bom a basis of mobility from pBR322,         pVS1 oriV origin of replication for Agrobacterium, pVS1 RepA, and pVS1 StaA. We also wanted to insert a multicloning         site (MCS) to our design which we ordered as a gBlock from IDT. The sequence for our MCS was: GCGTTCCGCGGGTTCTTAAGGTTCTCGAGGTTAAGCTTGTTAGATCTGTTGAATTCGCGGCCGCTTCTAGAGTACTAGTAGCGGCCGCTGCAGGT TGGATCCGTTCCATGGGTTGCATGCGTTCG.
Methods
The strains used were Agrobacterium tumefaciens GV3101 and Escherichia coli DH5-Alpha unless otherwise         stated. All E. coli cultures were grown in LB media at 37℃ unless otherwise stated. All Agrobacterium were grown in LB at 30℃ unless otherwise stated. All plasmid DNA extractions were performed with ABM Column-Pure         Plasmid Mini-Prep Kit. DNA purification from gels with ABM Column-Pure DNA Gel Recovery Kit. The same protocols for         DNA gel electrophoresis, restriction enzyme digest, DNA ligation, and chemical and electro transformation were used         throughout unless otherwise stated and can be found in Protocols.
Cloning ietAS into psb1c3
Cloning ietAS into pSB1C3 was done in two consecutive rounds of standard restriction enzyme cloning. The ietA and ietS gBlocks         ordered from IDT and pSB1C3 were digested with Eco-RI and Spe-I enzyme. The two pieces of pSB1C3 were separated and         purified by gel purification. pSB1C3 was ligated with ietA and ietS in two separate ligation reaction overnight.         The resulting ligations were then transformed into chemically competent E. coli. Transformants were         picked 24 hours later and grown overnight in LB-Chloramphenicol. Overnight cultures were miniprepped and checked         by restriction enzyme digest using Eco-RI and Pst-I enzymes, transformants identified with the right size band (1.3         kB for ietA and 2.5 kB for ietS) were checked by Sanger sequencing using VF2 and VR primers.
To combine the two constructs and place ietA upstream of ietS, pSB1C3 ietS was digested using Eco-RI And         Xba-I enzymes and pSB1C3 ietA was digested using Eco-RI and Spe-I. Both digests were separated and purified by gel         electrophoresis and gel clean up kits. The ietA fragment was ligated with the pSB1C3 fragment to form a the full ietAS construct. The resulting ligation was then transformed into chemically competent E. coli. Transformants         were picked 24 hours later and grown overnight in LB-Chloramphenicol. Overnight cultures were miniprepped and checked         by restriction enzyme digest using Eco-RI and Pst-I enzymes, transformants identified with the right size band (3.8         kB for ietAS) were checked by Sanger sequencing using VF2 and VR primers.
Cloning in the Multiclonal Site into pCAMBIA
To clone in our MCS into pCambia we digested pCambia with Sac-II and Sph-I which cut the plasmid at the edge of the         plasmid backbone, removing the left and right border regions necessary for plant transformation. We separated and         purified this digest by gel electrophoresis. Additionally, we digested our MCS IDT gBlock with Sac-II and Sph-I enzymes.         We ligated the two fragments overnight and proceeded to transform them into chemically competent E. coli.         Transformants were picked and grown overnight in LB-Kanamycin, and then mini prepped. The minipreps were checked         by Sanger sequencing using custom primers. This new pCambia plasmid with inserted         MCS was used by all other teams when cloning parts to be used in Agrobacterium.
Cloning ietAS into pCambia
To insert ietAS into pCambia we subcloned ietAS from pSB1C3. We digested pSB1C3 ietAS and pCambia MCS with Eco-RI and Spe-I         enzymes. We separated and purified our digests using gel electrophoresis. Next, we proceeded to ligate the ietAS and         pCambia digests together overnight. The overnight ligations were then transformed into chemically competent E. coli.         Transformants were selected the following day and grown overnight in LB-Kanamycin. Cultures were then miniprepped         and and digested, first with Eco-RI and Pst-I enzymes, then with Hind-III to check on a gel that the inserts were         the correct size. The minipreps were then transformed into electrocompetent Agrobacterium the standard electroporation         protocol (6) which can be found in Protocols.
Characterization of ietAS in Agrobacterium
To determine whether ietAS was effective at maintaining plasmids in Agrobacterium without antibiotic pressure         we did a multiday plating assay. 3 biological replicates of both pCambia ietAS Agrobacterium and pCambia Agrobacterium were grown in 3 mL of LB overnight. Each day serial dilutions ranging from 10-3 to 10-8 were made from each replicate         and spot plated on LB plates and LB-Kan plates. 3 uL of each culture was passaged into a fresh 3 mL of LB and         grown to repeat the assay next day. On the first and last day, we also plated on LB-Gent and LB-Gent-Kan plates         to see if ietAS contributed to the loss of the modified Ti plasmid present in Agrobacterium GV3101. Full details         of this assay can be found in Protocols.
Figure 1 Cloning procedure for ietAS into pCambia and into pSB1C3.
Figure 2 pCambia ietAS digested with Hind-III, expected band sizes are 10kb(plasmid and segment ietAS) and           700kb(partial segment of ietA) as seen in the photo.
Figure 3 Plasmid stability plating assay, day 2. Top-left: pCambia without ietAS in Agrobacterium plated           on LB-Kan agar. Bottom-left: pCambia without ietAS in Agrobacterium plated on LB agar. Top-right: pCambia with ietAS in Agrobacterium plated on LB-Kan agar. Bottom-right: pCambia with ietAS in Agrobacterium plated on LB agar. Each sample is plated as two technical replicates over five dilutions based on an OD 600 reading.
Figure 4 Effect of ietAS on plasmid stability. CFU ration is the number of CFU's on LB-Kan agar plate divided           by the number of CFU's on LB agar plate. See protocols section for details. Each CFU value is the average of three           biological replicates, each biological replicate had two technical replicates and was corrected against OD600 .           Vertical bars represent standard deviation. On day 0 for ietAS there were too many colonies to count so we were unable to quantify that results for that day.
Results
Cloning
Cloning of ietA, ietS, and ietAS into pSB1C3 in E. coli was successful and confirmed by Sanger sequencing and restriction         enzyme digest. Cloning the MCS into pCambia in E. coli was successful and confirmed by Sanger sequencing.
Subcloning ietAS into pCambia was attempted several times, with the last two times working. The ietAS pCambia         cloning was confirmed using restriction enzyme digest with Eco-RI and Pst-1 as well as Hind-III(Figure 2) the resulting         digests were run on a gel and the correct bands were observed. ietAS pCambia miniprep was successfully transformed         into Agrobacterium.
Plasmid Stability
As cloning ietAS into pCambia delayed our progress for 3 weeks, we were unable to fully         characterize the efficacy of ietAS on the plasmid stability. We ran a plasmid stability assay for 3 days using spot         plating to compare the amount of colony forming units on LB versus LB-Kanamycin (Figure 3). Unexpectedly we noticed         a decrease in the ratio of CFU's on LB-Kan to CFU's on LB with the ietAS (Figure 4).
Yamamoto et. Al saw at 15-fold decrease in colony forming units with the addition of ietAS with the         plasmid containing the sacB gene conferring hypersensitivity to sucrose. In our plating experiment we did not see         a significant difference between the pCambia plasmid containing ietAS and pCambia plasmid without ietAS. This could         be due to the presence of pVS1 StaA on pCambia, a stabilizing protein which is shown to maintain plasmids containing         it for 100 generations growing at 30℃ (Heeb et Al., 2000). In future work we could perform the same expirement knocking         out the pVS1 staA gene in pCambia.
Conclusion
For our project we created a new composite Bio-Brick part that is shown in the literature to increase plasmid incompatibility         with other repABC plasmids. This part is also shown in the literature to increases plasmid stability through toxin-antitoxin         interaction. Having this part in our aGROW plasmid would add selective pressure to keep the plasmid in Agrobacterium despite the negative fitness of harboring CRISPR-Cas9. Further work on characterization of the ietAS construct can         be done to quantify the added level of stability that ietAS adds to a plasmid, this can be used in conjunction with         literature values. This is the first toxin-antitoxin part specifically for Agrobacterium that has been entered         into the Biobrick registry of standard biological parts.
References
Heeb, S., Itoh, Y., Nishijyo, T., Schnider, U., Keel, C., Wade, J., Haas, D. (2000). Small, stable shuttle vectors based on the minimal pVS1 replicon for use in gram-negative, plant-associated bacteria. Molecular Plant-Microbe Interactions, 13(2), 232-237. doi:10.1094/MPMI.2000.13.2.232
Melderen, L. V., & Bast, M. S. (2009). Bacterial Toxin–Antitoxin       Systems: More Than Selfish Entities? PLoS Genetics, 5(3). doi:10.1371/journal.pgen.1000437
Unterholzner, S.       J., Poppenberger, B., & Rozhon, W. (2013). Toxin–antitoxin systems: Biology, identification, and application. Mobile       Genetic Elements, 3(5), e26219. http://doi.org/10.4161/mge.26219
Yamamoto, S., Kiyokawa, K., Tanaka, K., Moriguchi,       K., & Suzuki, K. (2009). Novel Toxin-Antitoxin System Composed of Serine Protease and AAA-ATPase Homologues Determines       the High Level of Stability and Incompatibility of the Tumor-Inducing Plasmid pTiC58. Journal of Bacteriology, 191(14),       4656-4666. doi:10.1128/jb.00124-09
British_Columbia_Base
