Team:Paris Bettencourt/Project/Binding
Project
Overview
Assay
Microbiology
Enzyme
Binding
Indigo
Model
Practices
About Perc
Dry Cleaners
Design
Events
Safety
Lab Safety
Product
Safety
Achievements
Results
Interlab Study
MedalRequirements
Parts
Notebook
Daily Note
Protocols
Bibliography
Team
Our Team
Attributions
Collaborations
Sponsors
@iGEM_Paris
Goals
To discover short peptide sequences with affinity for fabrics.
To characterize the affinity and specificity of these sequences.
Results
Isolated over 200 peptides, 40 each showing binding to cotton, linen, wool, polyester and silk.
Confirmed and quantified binding with ELISA
The peptides found here were passed to the enzyme group for testing on proteins.
Methods
Phage display
Motif analysis
Quantitative ELISA
Skip to results
Overview of the project
Abstract
In this sub-project we develop Fabric Binding Domains (FBDs), short peptide sequences capable of binding to fabric samples. Results from the Modeling Group suggest that the addition of an FBD with optimal affinity can concentrate an enzyme near the fabric surface and therefore increase stain-degrading enzymatic activity. Using the method of phage display, we isolated 40 short peptides with affinity for cotton, linen, wool, polyester or silk. Bioinformatic analysis identified sequence motifs and biophysical features associated with binding to each fabric, or nonspecific binding to several fabrics. Seven peptides were selected for further analysis by quantitative ELISA, to determine their exact binding constant for each of the five fabrics. The peptides discovered by the Binding Group were passed to the Enzyme Group, where they were used to target GFP and stain-degrading enzymes to wine-stained fabric samples.
Motivation and Background
Classical biochemistry studies the behavior of enzymes in well-mixed solutions with simple, mass-action kinetics. However, most chemical systems in the real world are not well mixed. Living cells in particular have a detailed spatial structure, and cells have developed strategies to maximize enzyme performance in a structured environment (Sweetlove, 2013). Many natural metabolic pathways are structured as metabolons, multi-enzyme complexes in which pathway intermediates are physically channeled from one enzyme to the next.
In recent years, synthetic biologists have created novel metabolic scaffolds to adapt this strategy to engineered metabolic pathways (Pröschel, 2015). The result has been significantly improved pathway performance: increased reaction rates, fewer side reactions, and better control of metabolic intermediates that may be unstable or toxic. Different mechanisms contribute to the improvement depending on the biochemical details of the scaffolded enzymes (Lee, 2012). However, many models emphasize the ability of enzymatic scaffolding to increase the effective concentration of enzymes and substrates by constraining them to the same physical space.
We reasoned that the activity of stain-degrading enzymes might also be enhanced by physically linking them to the fabric surface. Our initial intuitions were confirmed and refined by results from the Modeling Group. In fact, our models predict that optimal enzyme performance will be achieved at intermediate binding affinities. At low affinity, enzymes are primarily found in solution and away from the stain. But at extremely high affinity, enzymes are primarily bound to unstained fabric and also unable to reach the stain. Therefore, we sought to prepare a library of fabric binding domains, allowing us to optimize the binding affinity separately for each enzyme, fabric and application.
To identify these Fabric Binding Domains we used the strategy of phage display. In this technique, large libraries of randomized amino acid sequences are displayed by genetically fusing them to phage coat proteins oriented toward the external environment. The phage library is panned against a specific binding target by mixing them and allowing them to reach binding equilibrium. Common targets for phage display libraries include protein antigens and DNA sequences. Some of the randomized peptides displayed on the surface of the phage may become bound to the target while unbound phage are removed by wash steps. The population of bound phage can then be recovered, amplified and further enriched by additional rounds of panning. Because the randomized amino acid library is genetically encoded, individual phage clones can then be isolated and sequenced to identify high affinity peptides.
Finally we quantified the binding affinity of specific peptide sequences using colorimetric ELISA. In this technique, a specific phage clone is bound to its high-affinity target and then washed a defined number of times. A primary antibody is introduced to recognize the phage, then a secondary antibody is bound carrying Horse Radish Peroxidase, an enzyme that allows a specific colorimetric reading. By quantifying the number of phage that remain bound after each wash, we can determine the precise binding constant.
Results
Phage display yields peptides binding cotton, linen, wool, polyester and silk.
Our phage display experiments were performed with the commercial Ph.D.-7 Phage Display Peptide Library (NEB # E8100S). This library consists 100 copies each of 109 phages expressing a randomized 7-mer peptide fused to the phage N-terminal pIII coat protein. The displayed peptide is separated by a short Gly-Gly-Gly-Ser linker to minimize interactions between the displayed peptide and the phage itself.
The phage library was panned against fabric samples of cotton, linen, wool, polyester and silk. The cycle of phage binding, elution, and amplification was repeated three times for each fabric. Following enrichment, individual clones were isolated and sequenced to determine the sequence of their displayed peptide. We isolated and sequenced at least 40 phage plaques from each fabric (Table 1A).
COTTON
LINEN
SILK
WOOL
POLYESTER
ADARYKS
ADARYKS
ADARYKS
ADARYKS
ADARYKS
ADARYKS
AGHVVPR
ADARYKS
ADARYKS
ADARYKS
ADARYKS
ASPDQEK
AGHVVPR
ADARYKS
ADARYKS
ADARYKS
HDSPTAA
AQSNPKN
ADARYKS
ASSHIHH
ADARYKS
HDVMWQR
AQSNPKN
ADARYKS
FRKKRKS
ALANFEP
HWNTVVS
ASSHIHH
ADARYKS
GALAKDE
ASSHIHH
MPRLPPA
AWPYVTL
ADARYKS
GASNIWN
ASSHIHH
MPRLPPA
AWPYVTL
ADARYKS
HWNTVVS
ASSHIHH
MPRLPPA
DETCSSM
ADIRHIK
ISTTLFP
ASSHIHH
MPRLPPA
FPSPMVG
ADIRHIK
METVVSS
DPRLSPT
MPRLPPA
GKNLMNM
ADIRHIK
MPRLPPA
ELAGTTW
MPRLPPA
HDVMWQR
ASSHIHH
MPRLPPA
ERGFLLL
MPRLPPA
HWNTVVS
ASSHIHH
MPRLPPA
FSRSNNT
MPRLPPA
KNANSRE
ASSHIHH
MPRLPPA
FSRSNNT
MPRLPPA
KNANSRE
GQSVVSL
MPRLPPA
GLHYDHS
MPRLPPA
KTAMKGP
GSTSFSK
MPRLPPA
GVKSEQL
MPRLPPA
MLQGNGY
GSTSFSK
MPRLPPA
GVLRYAP
MPRLPPA
MPRLPPA
GSTSFSK
MPRLPPA
HNWMHQN
MPRLPPA
MPRLPPA
HDSPTAA
MPRLPPA
HYPPVDD
MPRLPPA
MPRLPPA
HDSPTAA
MPRLPPA
ISTTLFP
MPRLPPA
MPRLPPA
MPRLPPA
MPRLPPA
MPRLPPA
MPRLPPA
MPRLPPA
MPRLPPA
MPRLPPA
MPRLPPA
MPRLPPA
MPRLPPA
MRLSVPN
MPRLPPA
MRLSVPN
MPRLPPA
MPRLPPA
MRLSVPN
MPRLPPA
MSNTLDP
MPRLPPA
MPRLPPA
PSNRQNT
MPRLPPA
MTQQLHT
MPRLPPA
MPRLPPA
QFDHWRN
MPRLPPA
QGDYFTY
MPRLPPA
MPRLPPA
QFDHWRN
MPRLPPA
RLLQYNS
QFPPPPG
MPRLPPA
SILPVTR
MPRLPPA
SFLVTRN
QPIYRVQ
MPRLPPA
SILPVTR
MPRLPPA
SIHERAK
SILPVTR
MPRLPPA
SILPVTR
MQEMRQM
SILPVTR
SILPVXR
MPRLPPA
SILPVTR
MRLSVPN
SSHSVQR
SLLTHNM
MPRLPPA
SILPVTR
NNSVSMN
SVVMPHG
STNPTSL
MPRLPPA
SILPVTR
SILPVTR
TDHAHRY
TLINYRG
MPRLPPA
SILPVTR
SLETMSN
TDMTAPK
TNLHINP
MPRLPPA
SILPVTR
SNYHWRM
TSDATQR
WTNVFVG
MPRLPPA
TRPTDTI
TESAPTL
TTHPRWG
XPRLPPA
SILPVTR
TVHVHKT
VFQTTYK
VECINNC
XPRLPPX
SILPVTR
YMGPSKT
VPRLPPA
VTLPDPR
XPRLXPA
TSNRAPY
YMGPSKT
WHLPAQR
XPXXPPT
XPRLPPA
YMGPSKT
Table 1A Novel fabric binding domains with diverse properties.
COTTON
LINEN
SILK
WOOL
POLYESTER
FSRSNNT(2)
SILPVTR(2)
AQSNPKN(2)
MRLSVPN(2)
ADARYKS(3)
MPRLPPA(2)
AWPYVTL(2)
ASSHIHH(3)
ASSHIHH(4)
MPRLPPA(25)
KNANSRE(2)
ADARYKS(6)
MPRLPPA(20)
ADARYKS(5)
SILPVTR(2)
SILPVTR(7)
MPRLPPA(20)
Table 1B Table of all peptide sequences observed to bind more than once. The number in parenthesis indicates the number of times the respective peptide was recovered. These candidate peptides were validated for binding affinity with an ELISA.
We focused our analysis on phage sequences that appeared at least twice. Isolated peptides displayed a range of fabric specificities, sequence motifs and phyicochemical properties (Table 2). Nine peptide sequences were selected for further analysis, chosen with a preference for chemical diversity and highly enriched peptides that appeared in many clones. Five of the nine peptides were fabric-specific, meaning they were isolated from only one fabric, while the others were found to bind multiple fabrics. The peptides were designated FBD 1-9.
COTTON
LINEN
SILK
WOOL
POLYESTER
Positive Charged
MPRLPPA(2)
SILVPTR
ADARYKS(5))
SFLVTRN
MRLSVPN
GVLRYAP
MPRLPPA(25)
HWNTVVS)
SILVPTR(2)
AGHVVPR)
ADARYKS)
MPRLPPA(20)
HWNTVVS
SILVPTR(2)
AGHVVPR
ADARYKS(2)
MPRLPPA
SILVPTR(7)
ADARYKS(9)
MRLSVPN(2)
YMGPSKT(4)
MPRLPPA(20)
HWNTVVS
ADARYKS(3)
MRLSVPN
Negative Charged
ELAGTTW
ALANFEP
VECINNC
HDSPTAA)
HDSPTAA
METVVSS
TESAPTL
Hydrophilic
FSRSNNT(2)
HNWMHQN
TDHAHRY
TDMTAPK
GVKSEQL
VTLPDPR
MTQQLHT
SIHERAK
DPRLSPT
WHLPAQR
GLHYDHS
HYPPVDD
RLLQYNS
TSDATQR
QGDYFTY
TTHPRWG
SSHSVQR
ASSHIHH(4)
MSNTLDP
TLINYRG
FRKKRKS
TNLHINP
QFPPPPG
QPIYRVQ
HDVMWQR
ASPDQEK
STNPTSL
KNANSRE(2)
KTAMKGP
HDVMWQR
GKNLMNM
TSNRAPY
AQSNPKN(2)
ASSHIHH
QFDHWRN
TVHVHKT
ADIRHIK(3)
PSNRQNT
TRPTDTI
ASSHIHH(3)
GSTSFSK(3)
NNSVSMN
VFQTTYK
GALAKDE
DETCSSM
SNYHWRM
SLETMSN
MQEMRQM
ASSHIHH
Hydrophobic
ERGFLLL
ISTTLFP
SVVMPHG
WTNVFVG
SLLTHNM
AWPYVTL(2)
MLQGNGY
FPSPMVG
GQSVVSL
ISTTLFP
GASNIWN
Table 2 Table of the sequence motifs and physico chemical properties of the peptides
Validation of the peptides using ELISA
The nine selected FBD peptides were validated with ELISA. Briefly, a clonal library of phage expressing each peptide was incubated with each of five fabric samples under conditions similar to the original panning experient. Following extensive washing, the fabric was incubated with an anti-phage primary antibody, then a secondary antibody linked to HRP, the Horse Radish Peroxidaze. The HRP enzyme produces a pigmented product, allowing semi-quantitative measurement of the level of fabric-bound phage.
In figure 1, we compare the ELISA signal before and after washing for phage displaying each of the FBD peptides against each fabric. In most cases, the phage are retained after washing and produce an ELISA signal significantly above control. Peptides were found to have a range of affinities for different fabrics. Notably wool appeared to be difficult to bind, with only FBD9 staying bound after washing.
Repeated washing reduced signal, allowing a quantitative measurement of binding affinitiy. In a typical case of FBD9 binding to cotton, more than 50% signal was retained after 10 rounds of washing. Analysis of these data is ongoing, with the goal of estimating the KD for each binding interaction.
Figure 1 Fabric binding domain validation by ELISA. A. ELISA for each FBD incubated with each fabric after two rounds of washing, normalized to the level after a single round. B. Control matrix repeating the procedure of panel A without phage. C. ELISA signal determined ater multple rounds of washing. Points represent individual measurements from three replicates. Lines represent best-fit to an exponential decay model.
Methods
Selection of fabric samples for panning
Cotton was obtained from Khadi & Co (Hand Woven, 100% cotton). Wool and Linen were obtained from Dharma Trading Co. (#PWFC, #LIN21). Silk thread was obtained from Au Ver à Soie (Thread 1003, Crème color, 100% Silk). Polyester thread was obtained from Mediac (Ref 960, Color 400, 100% Polyester).
Preparation of fabric samples for panning
To remove possible factory coatings or treatments, cotton was boiled in sodium carbonate solution then washed with water until pH neutral.
The other fabrics were washed thoroughly with ethanol then water.
Detailed protocol for phage display
Panning was performed with 10 µl of Ph.D.-7 Phage from NEB # E8100S, at a titer of 1013 pfu/ml, was mixed with 16 mg washed and blocked fabric to give a concentration of 2*1011 phage. Fabric was washed vigorously with 0.1% TBST to remove unbound phage. Phage were further eluted with 0.2 M Glycine-HCl (pH 2.2) and 1 mg/ml BSA to disrupt all nonspecific binding interactions, then neutralized with 1 M Tris-HCl, pH 9.1.
The eluate was amplified by infecting a 5 ml culture of E. coli overnight, then mixed with 20% PEG/NaCl to precipitate the phages. Precipitated phages were resuspended in TBS and titered by the method of top agar to calculate the input for next round of panning. After 3 additional rounds of panning, individual phage clones were isolated with the method of top agar.
Sequencing of Phage DNA
Single plaques were picked from fresh titration plates and amplified in E. coli. Phage were precipitated with 20% PEG/NaCl, then were resuspended in Iodide buffer. Phage DNA was then precipitated by incubation with 70% ethanol. DNA was washed with ice cold 70% ethanol, air dried and later resuspended in DNAse free distilled water and quantified with a nanodrop spectrophotometer.
Binding quantification with ELISA
Single phage clones were applied to selected fabrics at a titer of  1011 virions. Experimental treatments were washed twice with TBST prior to ELISA, while control treatments were washed only once. Bound phage ws detected with an HRP-conjugated anti-M13 antibody , GE Healthcare #27-9421-01). ABTS (Sigma # A-1888) peroxidase substrate was added following the manufacturers instructions and produced a measurable green product read at 405 nm.
Attributions
This project was done mostly by Shruthi and Thomas. We would like to thank Clément Nizak and the Nizak lab for help in designing the phage display protocol.
References
Sweetlove, L. J., & Fernie, A. R. (2013). The Spatial Organization of Metabolism Within the Plant Cell. Annual Review of Plant Biology, 64(1), 723–746.
Lee, H., DeLoache, W. C., & Dueber, J. E. (2012). Spatial organization of enzymes for metabolic engineering. Metabolic Engineering, 14(3), 242–251.
Pröschel, M., Detsch, R., Boccaccini, A. R., & Sonnewald, U. (2015). Engineering of Metabolic Pathways by Artificial Enzyme Channels. Frontiers in Bioengineering and Biotechnology, 3(Pt 5), 123–13.
Assay
Microbiology
Enzyme
Binding
Indigo
Model
Centre for Research and Interdisciplinarity (CRI) Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor Paris Descartes University 24, rue du Faubourg Saint Jacques 75014 Paris, France
+33 1 44 41 25 22/25
igem2016parisbettencourt@gmail.com
2016.igem.org
