<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navmenubkg"><DIV style="margin-left:-15px" id="navmenubox"><UL id="nav"><LI><A style="margin-left:10px" href="https://2016.igem.org/Team:Paris_Bettencourt/Description">Project</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Description">Overview</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Assay">Assay</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Microbiology">Microbiology</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Enzyme">Enzyme</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Binding">Binding</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Indigo">Indigo</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Model">Model</A></LI></UL><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/About_Perc">Practices</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/About_Perc">About Perc</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Integrated_Practices">Dry Cleaners</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Design">Design</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Human_Practices">Events</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Safety">Safety </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Safety">Lab Safety</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Product_Safety" style="margin-top:-10px; margin-bottom:30px;">ProductSafety</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Achievements">Achievements</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Interlab_Study">Interlab Study</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Medal_Requirements">MedalRequirements</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI style="margin-left:10px"><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Daily Note</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Protocols">Protocols</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Bibliography">Bibliography</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Team">Team </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Team">Our Team </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Sponsors">Sponsors</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><DIV id="subheader"><DIV style="margin-top:20px; margin-bottom:20px"><DIV style="margin-top:20px; margin-bottom:40px"><DIV class="panel"><A href="https://2016.igem.org/Team:Paris_Bettencourt/Results" title="Results"><DIV id="pspanel" class="subpanel2" onmouseover="chgtrans(this)"><DIV class="titlebox">
		Results
	      </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Medal_Requirements" title="Medal Requirements"><DIV id="pspanel" class="subpanel2" onmouseover="chgtrans(this)"><DIV class="titlebox">
		Medal Requirements
	      </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Parts" title="Parts"><DIV id="pspanel" class="subpanel2" onmouseover="chgtrans(this)"><DIV class="titlebox">
		Parts
	      </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Interlab_Study" title="Interlab Study"><DIV id="pspanel" class="subpanel2" onmouseover="chgtrans(this)"><DIV class="titlebox">
		Interlab Study
	      </DIV></DIV></A></DIV><DIV id="definition"><H2>Results</H2><H2 class="red">Automated/DIY microplate on fabric-Image processing tool for stain measurement</H2><P class="”input”"> 
An automated 96 well microplate for fabric was made using a fluorescent microplate, 96 well tip holder and laser-cut cotton circles. 
<UL><LI> Spread a flat, wet cotton sheet on glass substrate of 96 well plate dimension. 

</LI><LI> 96 Wet cotton circles of 6.5mm are laser cut on glass substrate.

</LI><LI> Remove the fabric. The circles of cotton stay on the glass substrate.

</LI><LI> Place the glass substrate of cotton circles on fluorescent microplate such that all circles face the wells and align well. 

</LI><LI> Hold it tight and move the glass substrate laterally so that all the cotton circles fall into the wells.	        

</LI><LI> Put the tip plate holder on the top of the microplate and adhere it to the microplate using a glue.

</LI><LI> Cover it with a lid and the microplate is ready to use for experiments. The same can be done for other fabrics like silk, polystyrene, nylon etc.

</LI></UL>
An Image processing plugin was developed to measure the pixel intensity of each well of the scanned image of Microplate on fabric. The relative change in pixel intensity is the quantitative measure for degradation of stain.	
	         
</P><DIV style="text-align:center;"><H2 class="red">Selected Peptides from Binding Domain Group</H2><P>
    Phage display yields peptides binding cotton, linen, wool, polyester and silk. 
 
The phage library was panned against fabric samples of cotton, linen, wool, polyester and silk. The cycle of phage binding, elution, and amplification was repeated three times for each fabric. Following enrichment, individual clones were isolated and sequenced to determine the sequence of their displayed peptide. We isolated and sequenced at least 40 phage plaques from each fabric.</P></DIV><P>
 
The nine sequences out of 40 FBD peptides were selected  for validation with ELISA. 
        </P><H2 class="red">Catechol dioxydases were expressed with and without Fabric Binding Domains and their functionality was demonstrated in both cases</H2><P class="”input”"> 
Six candidate plant and bacterial enzymes were tested for expression in <I>E. coli</I>
Three out of six enzymes were functionally expressed in <I>E. coli</I> and tested for their ability to degrade their natural substrate

Feasibility of fusing proteins with FBDs was tested by fusing them to GFP and assessing their GFP ability and their ability to bind to several fabrics

The selected enzymes were fused with Fabric Binding Domains and their activity was still observed


	         
</P><DIV style="text-align:center;"><H2 class="red">Microbes that degrade indigo </H2><P class="input">We succeeded in finding 16 microbes that grow on denim, covered with M9 media. After two rounds of culture in liquid media with indigo and measurements of absorbance as a way to assess indigo degradation, we found 3 strains highly capable at degrading indigo : <I>Streptmoyces fumigatiscleroticus </I>, <I> Streptomyces coelicolor </I> and <I> Pantoea agglomerans </I>. </P><H2 class="red">Bpul laccase degrades indigo </H2><P>
Structural formulas of two substrates (indigo carmine, and indigo) of Bpul laccase are showing structural similarity. Since it has been shown that bpul degrades indigo carmin, we went to prove its ability to degrade indigo. All results shown in the figure above are made with our bpul gene expressed in the E. coli strain BL21(DE3). The protein gel (figure above B) is showing an over-expression of the bpul, after the induction with IPTG. When comparing uninduced bpul sample with the control sample (BL21(DE3)) we can notice leakiness of our promoter, which is probably due to great transcriptional strength of T7 polymerase. In the degradation experiments (figure above C and D), both the transformant BL21(DE3)-bpul, and the negative control BL21(DE3) have been induced with IPTG. The cell extracts of both strains were tested for the ability to degrade indigo carmine and indigo. 10µM acetosyringone (ACS) was added to the reaction mixture to facilitate the oxidation of the substrates. ACS is a known mediator of the laccases and it is often used and needed in the studies of the  laccase activitiy As expected, the 1mM indigo carmine solution is entirely degraded within the 13 hours of the experiment in the cell extract which contains overexpressed bpul. Additionally, we have shown that more than 60% of indigo can be degraded by bpul laccase in the 13 hours of the experiment. It is important to notice, that in longer experiments would probably lead to even more degradation since the degradation curve didn't reach the saturation.
</P><H2 class="red">Construction of the strain database by the microbiology team</H2><P> 
This summer, our group designed an experiment to find the best stain fighting enzymes that nature has to offer. 
We build a big database with 186 strains through selective and non-selective plating on different media. 
 <UL><LI>We managed to isolate species from all around the world through iGEM collaborations.</LI><LI>186 bacteria were tested for quercetin degradation.</LI><LI>A 186 bacterial database was built and characterized.</LI><LI>20 strains produced more than 50% quercitin degradation.</LI><LI>A phylogenetic tree of 174 different  bacterial species was created .</LI><LI>3 bacterias were shown able to degrade anthocyanin</LI><LI>Common candidate genes were selected through genome sequencing of 4 different bacterial strains.</LI></UL></P><DIV style="margin-top:20px; margin-bottom:20px"><DIV class="panel"><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Assay" title="Assay"><DIV id="pspanel" class="subpanel8" onmouseover="chgtrans(this)"><DIV class="titlebox">
	    Assay
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Microbiology" title="Microbiology"><DIV id="dspanel" class="subpanel8"><DIV class="titlebox">
	    Microbiology
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Enzyme" title="Enzyme"><DIV id="tcpanel" class="subpanel8"><DIV class="titlebox">
	  Enzyme
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Binding" title="Binding domains"><DIV id="thpanel" class="subpanel8"><DIV class="titlebox">
	    Binding
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Indigo" title="Indigo"><DIV id="thpanel" class="subpanel8"><DIV class="titlebox">
	    Indigo
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Model" title="Model"><DIV id="thpanel" class="subpanel8"><DIV class="titlebox">
	    Model
	  </DIV></DIV></A></DIV></DIV></DIV></DIV><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:igem2016parisbettencourt@gmail.com">igem2016parisbettencourt@gmail.com</A><CENTER><A href="https://2016.igem.org/">
    2016.igem.org
</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>