<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Notebook_Indigo skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Notebook/Indigo</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navmenubkg"><DIV style="margin-left:-15px" id="navmenubox"><UL id="nav"><LI><A style="margin-left:10px" href="https://2016.igem.org/Team:Paris_Bettencourt/Description">Project</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Description">Overview</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Assay">Assay</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Microbiology">Microbiology</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Enzyme">Enzyme</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Binding">Binding</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Indigo">Indigo</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Model">Model</A></LI></UL><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/About_Perc">Practices</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/About_Perc">About Perc</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Integrated_Practices">Dry Cleaners</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Design">Design</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Human_Practices">Events</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Safety">Safety </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Safety">Lab Safety</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Product_Safety" style="margin-top:-10px; margin-bottom:30px;">ProductSafety</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Achievements">Achievements</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Interlab_Study">Interlab Study</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Medal_Requirements">MedalRequirements</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI style="margin-left:10px"><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Daily Note</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Protocols">Protocols</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Bibliography">Bibliography</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Team">Team </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Team">Our Team </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Sponsors">Sponsors</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><DIV id="subheader"><TABLE id="ancre"><TBODY><TR><TD><A href="#ancre">Notebook</A></TD><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Protocols">Protocols</A></TD><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Bibliography">Bibliography</A></TD></TR></TBODY></TABLE></DIV><DIV id="subheader2"><TABLE><TBODY><TR><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Assay">Assay</A></TD><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Microbiology">Microbiology</A></TD><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Enzyme">Enzyme search</A></TD><TD><A href="#ancre2">Mission Indigo</A></TD><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Binding">Binding Domains</A></TD><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Anthocyanin">Anthocyanin</A></TD></TR></TBODY></TABLE></DIV><DIV id="ancre2"><DIV class="input"><H2 class="red">Week 27th June - 3rd July</H2><H3>Introduction</H3><P class="input">Inspired by the idea of removing wine stains, we have decided to give an artistic tone to our project. What if we can exploit the power of enzymes not only to digest wine stains, but also to create art? For this we are planing to find enzymes and micro-organisms which can digest indigo dye traditionally used to color denim. Our ultimate goal is to have a tool which can create beautiful patterns by bleaching out the indigo color in our favorite pair of jeans. To achieve this we are using the same strategies as the Protein and Microbiology groups. Thus, within the Mission Indigo group we will try to find enzymes and micro-organisms, especially fungi, capable of digesting indigo.</P><H3><I>Bacillus Pumillus</I> laccase (bpul)</H3><P class="input">In the enzymatic part we have decided to focus on a single enzyme which has shown the capability of digesting many different phenolic compounds. The enzyme is a spore coat laccase from Bacillus pumilus called CotA (I3RYX9), but for the sake of simplicity, and to avoid confusion with CotA protein (P07788) from Bacillus subtilis we will refer to it as bpul.</P><P class="input">bpul stands for Bacillus pumilus laccase I, and it belongs to multi-copper oxidases family (MCO). The bpul enzymes catalyze the oxidation of a range of organic substances via a four-electron reduction of oxygen to water. These enzymes are capable of digesting even more compounds in the presence of a mediators, which facilitate the reaction. This is called indirect oxidation. Therefore, in many of our experiments we will sometimes use Acetosyringone (ACS) as a mediator. In advance, laccases are already used in denim bleaching processes, but, to our knowledge, bpul hasn't been verified nor characterized on its ability to digest indigo. We are hoping to successfully bleach denim with bpul, and use it to create indigo digesting bacteria.</P><P class="input">Our main literature and a guideline for our experiments is a research conducted by Reiss et. al. The paper is entitled “Bacillus pumilus laccase: a heat stable enzyme with a wide substrate spectrum”, and in case of unclarity in these notebook please refer to this paper (http://www.biomedcentral.com/1472-6750/11/9). In their research they have demonstrated the capability of bpul to digest indigo carmine. Because of similarity of the molecular structure of indigo carmine and indigo dye (figure 1), and because of the capability of laccasses to digest indigo.</P><P class="input"><B>Figure 1.</B> Structural formulas of indigo carmine and indigo dye.</P><P class="input">Additionally, bpul laccase is available in the iGEM's Registry of Standard Biological Parts. Its also one of the best documented Standard Biological Parts. Documentation about this protein can be found in the iGEM parts registry under code: BBa_K863000. For more information check: http://parts.igem.org/Part:BBa_K863000</P><P class="input">During the first week we have been discussing and planing how to realize Mission Indigo sub-project. We have also transformed BBa_K863000 which contains His-tagged bpul into the E.coli DH5alpha strain in order to preserve the plasmid. The new strain was called DH5alpha-bpul and can be found in the glycerol stocks. </P><P class="input"><B>Protocols used for transformation: [PROTOCOL 1] [PROTOCOL 2]</B></P><H2 class="red">Week 4th - 10th July</H2><H2 class="red">Week 11th - 17th July</H2><H3>Transforming <I>E. coli</I> BL21(DE3) strain with BBa_K863000</H3><P class="input">The bpul construct from the iGEM Standard Biological Parts registry (BBa_K863000) has been transformed into the E.coli BL21(DE3) strain. The construct is under T7 promoter, and therefore, T7 polymerase is needed to transcribe bpul DNA sequence into mRNA which will then be used to produce bpul laccase. BL21(DE3) strain has the T7 polymerase gene under IPTG inducible lac promoter. We have received both the E. coli BL21 and BL21(DE3) by a kind donation from David Bikard's lab from Institute Pasteur. Both strains have the same genetic background, but unlike BL21(DE3), BL21 strain does not express the T7 polymerase.</P><P class="input">This week we started by plating the BL21 and the BL21(DE3) strains and making electro-competent cells out of them. We minipreped the bpul plasmid from previously transformed DH5alpha-bpul using the ThermoScientific GENEJET Plasmid MIniprep kit #K0503. We electroporated 2uL of plasmid and 20uL of the electro-competent cells. We plated on LB crm25, 200uL and 500uL (Our mentor, Jake told us that 99% of the cells would die due to over-expression of our protein).</P><P class="input"><B>Figure 2.</B> The fluorescence is observed in the positive control (BL21(DE3)-GFP) transformed with the same electroporation protocol.</P><P class="input">The transformation was successful with bot bpul and GFP plasmids (figure 2). We are observing confluent growth on plates. The strains were re-stricken on the new plates and glycerol stocks were made. Transformation efficiency is 1–5 x 107 cfu/μg pUC19 DNA.</P><P class="input"><B> PROTOCOLS USED:
Miniprep: The protocol used is given in the link: https://tools.thermofisher.com/content/sfs/manuals/MAN0013117_GeneJET_Plasmid_Miniprep_UG.pdf
[link1]
[link2]
[link3]</B></P><H2 class="red">Week 18th -24th July</H2><H3> Denim Experiment </H3><P class="input"> We want to find microbes that naturally digest indigo. In order to find them, we started with a simple experiment: we put pieces of denim in a M9 agar medium. We prepared M9 without glucose, so there is no other nutrient but denim. The medium was not sterilized, since our goal is to find whatever organisms that feeds on indigo. Pieces of denim were cut directly from a pair of jeans, and soaked in M9 media. They were then put in a square petri dish, later fill with M9. One piece was dip in media and put in a bottle with the lid not totally closed. At first, the plates were left open, for any microbes to contaminate it. Problem is, it dried out really fast. For the piece of denim in the bottle, it wasn’t a problem at all, and it stayed soaked. </P><H2 class="red">Week 25th -31st July</H2><H3> Species isolation </H3><P class="input">First colonies were observed on denim 4 days after the start of the experiment. 6 different strains were identified and potential candidates for degradation of indigo on denim. These colonies grew on the piece of denim that was put in the bottle, as it wasn’t dry. The plates were emptied and done again, this time having the lid almost closed to keep moisture.  Samples were harvest and inoculated on LB plates, in order to isolate them. Plates were put at 30 °C for 24 to 48 hours. They grew well except for one for which a new sample was taken. A database was created with details on them. As half of the strains were fungi, the question on how to work with fungi was raised and investigated throughout our lab work. </P><H3> Pleurotus ostreatus </H3><P class="input">During this week we investigated a potential candidate called Pleurotus Ostreatus. Indeed, a few plant pathogen mushrooms are described in literature as indigo-degrading organisms, the degrading enzyme being laccase. Pleurotus ostreatus is a mushroom that grow on decayed wood, but can also be cultured on straw containing medium or other similar media. It is actually an edible mushroom, used in cooking. We will probably work with it. </P><H2 class="red">Week 1st - 7th August</H2><H3>Producing fully copper loaded bpul</H3><P class="input"> bpul enzyme has 4 copper ions which help catalyze its reactions. In order to make sure we obtain fully copper loaded enzymes we have grown our BL21(DE3)-bpul strain in specific conditions: </P><UL class="input"><LI>Overnight culture was diluted 1:50 and grown until exponential phase</LI><LI>After reaching exponential phase 0.1 mM of IPTG was added to initiate bpul transcription</LI><LI>0.25 mM CuCl2 was added to meet the copper needs </LI><LI>Cells were incubated for 24h at 25oC.</LI><LI>They were grown 4hr with shaking and then 20h without. [reference] it increases the yield of copper loaded enzymes</LI></UL><P class="input">
Full protocol description can be found here <B>[link 6]. </B>
1L of cells BL21(DE3)-bpul was prepared alongside 1L of BL21(DE3) cells (without the bpul construct) which will be used as a positive control. 
After 24h incubation cells were collected by centrifugation, washed with PBS and stored at -20 oC. 
Later, protein extract was obtained by using B-per reagent with lysozyme and DNase-1 (protocol).
</P><H3>Preparing solutions</H3><P class="input">
Stock solutions of chemical needed to perform the indigo degradation were prepared. I have faced some troubles with dissolving non-polar indigo and ACS, but eventually I have succeeded and mixes are presented below:
</P><UL class="input"><LI>100 mM Acetosyringone (ACS) stock solution - 196 mg of ACS was dissolved in small volume of ethanol, after which the water was added until 10 mL volume</LI><LI>50 mM indigo stock in DMSO - 39.34 mg of indigo dye was disolved in 3 mL of DMSO</LI><LI>1M CuCl2 stock solution - 2.689g of CuCl2 was dissolved in 20 mL of H2O</LI><LI>0.1M Porassium phosphate buffer 7.8 pH - 1.541g of K2HPO4 and 156.2mg of KH2PO4 in 100mL of H2O</LI></UL><H3> Species isolation </H3><P class="input"> Now that we isolated a few strains, we could start species identification. A 16S colony PCR was performed, using all bacterial strains that were grown on LB. After that a gel electrophoresis was performed to check if the PCR worked and gave good results, leading to the next step in identification: DNA purification of PCR products to send them for sequencing. As there was a suspicion of primers contamination only one sample was sent.  3 new strains appeared on denim pieces during the week. They were harvest and inoculated on LB plates, just like the first ones. After all samples were taken, the piece of denim that was in the bottle was washed with water, and ethanol to make sure no fungi remained as they can easily disperse and contaminate stuff. Sadly, it seems no color removing was observed; it could either be that it is a longer process, or it needs other factors to make it more effective. </P><P><I> P. ostreatus arrived on Thursday, waiting for it to grow! </I></P><H2 class="red">Week 8th - 14th August</H2><H3>indigo degradation experiments</H3><P class="input"> 2 experiments of indigo degradation were performed, between which the protein extracts were frozen once (which might have potentially lowered the bpul efficiency). Before experiments proteins were extracted using B-per and quantified using Bradford reagent kit. First experiment has failed because of indigo precipitation, thus in a new media solution indigo was dissolved in 0.5% Tween80. Indigo degradation experiments were performed in the presence of ACS to help the degradation reaction.</P><H3>Results</H3><P class="input"><B>Figure 1.</B> Curve after staining standard BSA solutions with Bradford reagent. The function of the obtained curve was used to quantify protein extracts from the table below. Control sample had 19.903 g/L of proteins. bpul sample had 12.636 g/L of proteins
<B>Table 1.</B> Absorbance of the bpul and control sample dilutions from which we determined the protein concentration
</P><TABLE style="width:100%"><TBODY><TR><TH>Control sample 10x dilution</TH><TH>0.9074</TH></TR><TR><TH>Control sample 100x dilution</TH><TH>0.2484</TH></TR><TR><TH>Control sample 100x dilution</TH><TH>0.2543</TH></TR><TR><TH>bpul sample 10x dilution</TH><TH>0.6819</TH></TR><TR><TH>bpul sample 100x dilution</TH><TH>0.2055</TH></TR><TR><TH>bpul sample 100x dilution</TH><TH>0.20573</TH></TR></TBODY></TABLE><P class="input"><B>Figure 2.</B> The absorbance scan of different concentrations of indigo solutions. Here, stock solution of indigo dissolved in DMSO was used and mixed with water to get different concentrations. Here we can see that the peak of indigo absorbance is around 700, and these values were used when performing indigo degradation experiments. 

The first indigo degradation experiment (DATA NOT SHOWN) failed because of precipitation of indigo in the buffer.</P><P class="input"><B>Figure 3. </B> After failed experiment, we have decided to repeat it but with indigo dissolved directly in the buffer to avoid precipitation. 0.5% of Tween 80 was used to increase indigo solubility in water. The result on the figure shows saturated water-indigo solutions and x2 dilutions. We can notice that the absorbance is really low. Hopefully this will be enough to capture indigo degradation. </P><P class="input"><B>Figure 4.</B> Indigo degradation in the 1.5 mL tube. From left to right there are: blank x2, negative control x2, negative control, cell extract with bpul x2. The reaction happened overnight.</P><P class="input"><B>Figure 5. </B> Encouraged by the indigo degradation in 1.5 mL tubes we performed additional indigo degradation experiment. Only this time instead of dissolving indigo in the DMSO first, we added Tween80 to our buffer and dissolved indigo directly in the buffer. We also centrifuged the buffer to remove all the insoluble indigo granules. Also, instead of the 96 well plates, the reaction was observed in the 1.5 mL tubes and measurements were taken at a larger time step.</P><P class="input"> Even though the first experiment performed in 96-well plate failed, we can see the enzyme activity present. The last indigo degradation experiment (in1.5 mL tubes) was successful, and we can see that the reaction is taking place within the first 17h. The indigo degradation in tubes (figure 4. and 5.) shows that bpul can degrade indigo, the only problem left is quantifying it with plate reader to capture the enzyme kinetics.</P><H3> Species isolation </H3><P class="input">Throughout the last week we collected 9 samples on denim. However growing them in LB plates we could see that there was not one specie per sample, but rather two or even three different ones per colony we harvested. To go further in species isolation all different strains from the first plate were inoculated on a second plate, giving us not 9 but 16 candidate organisms. Growth rate varied between the 16 strains. After a few tries we were able to have 16 grown organisms on LB plates. Another round of 16S colony PCR was performed with 9 out of 16 strains.  Our first sequence came back, and after blasting, it seems to be an enterobacteria. </P><P class="input"> Our first experiment with denim was over, but it was not perfect: even though we had organisms growing on denim, this fabric is made of cotton tainted with indigo, so we can’t be sure that these organisms fed on indigo or on cellulose. So it’s time for our second experiment: cultivate these organisms on medium with indigo as the only carbon source. </P><H2 class="red">Week 15th - 21st August</H2><H3> Species isolation </H3><P class="input"> Looking at the results of the PCR from last week, it was obvious that the protocol we used was not suited for fungi. Indeed fungi are eukaryote hence having a 18S rRNA and not a 16S (we still tried the 16S colony PCR with fungi in case it works for some of them). We thus spent the week working on a protocol for DNA extraction and colony PCR for fungi. Luckily almost half of the species isolated from jeans are bacteria so we could carry a 16S colony PCR for them, going further in the identification of our candidates. DNA was then purified and sent for sequencing. We had a total of 11 samples from 6 different strains to work with, from which 9 were eligible for sequencing. Next step is making plates, with either indigo as the only carbon source or glucose + indigo. Each strains will be inoculated on a plate with indigo and a plate with a piece of denim. </P><H3>Determining indigo solubility</H3><P class="input">In order to follow indigo degradation with our enzyme (bpul) we had to determine the function which would connect the absorbance measurements with indigo concentration. This was done by following Beer-Lambert law. 8 concentrations ranging from 20 mg/L to 0 mg/L of indigo dissolved in potassium phosphate buffer with tween80 have been made. In total we had 4-plicates of buffer for each indigo concentrations. Each indigo concentration has been measured 3 times to confirm result, at 3 different wavelengths (660nm, 670nm and 680nm) that match the peak of indigo absorbance. In total we had 288 measurements (96 per wavelength). Measurements belonging to same concentrations and wavelength have been averaged. From the averages 3 graphs (one per wavelength) with tread-lines and functions were made. The data and the graphs are shown below.</P><P class="input"><B>Table 2</B> Example of the absorbance measurements for the 670nm wavelength</P><P class="input"><B>Figure 6.</B> The example of the absorbance-concentration plot for 670nm with its belonging mathematical formuls and tread-line.</P><P class="input"><B>Figure 7.</B> We have also tried if our bpul will degrade indigo from the jeans of our mentor. Unfortunately, this might have been too ambitious for this stage of our project. Not surprisingly, we failed.</P><H2 class="red">Week 22nd - 28th August</H2><H3><I> Pleurotus Ostreatus </I></H3> P. ostreatus finished growing in the week end, so when I came back to the lab on Monday it was fully grown and I could collect its spores. Spores, as asexual single-cell reproductive units, are good to work with, they will easily grow on plates with the good medium and it is not difficult to collect them and store them. One way to collect fungi spores is spore print, and it requires very little material: once the mushrooms are mature and ready to produce spores, you need to cut it and keep only the mushroom cap, put it on a sheet of paper (it’s best to use two papers, one white and one black, if you don’t know what color the spores are), place a drop of water on top the mushroom cap and cover with a glass container to prevent it from drying out. 24 hours later you should have spores on the paper. To conserve it just don’t forget to keep spore print in a dry location.  I didn’t have to do the spore print since the fungi had already produce spores and they felt on the bench: I cleaned the bench and kept the tissues I used which were full with spores. Once you have your spores you can store them in spore syringe. To make spore syringe, first prepare sterile water, then scrap spores from spore print (or here tissues) into it. You then just have to use a syringe, fill and empty it once or twice to have a good mix and then fill it with your spore solution. You need to keep it 2 or 3 days, for spores to hydrate. After three days, I could inoculate spores on plates. We had one experiment with denim and M9 minimal base, but to make sure our candidates degrade fungi we need one with indigo as the carbon. I therefore made some M9 media with indigo as carbon source instead of glucose, prepared plates with it, and use them for spores’ inoculation. I also made plates with M9 and denim, to try to grow the spores on it (remember p. ostreatus was first grown in a media containing straw, to collect spores and then try to grow it on denim and/or indigo). More plates will be made, with also glucose as a carbon source in case indigo as a sole carbon source is not enough. <H3> Species isolation </H3> This leads us to our other candidates, the organisms that grew on denim in our first experiment. I used M9+indigo plates again, this time to inoculate our candidate organisms on them. 12 samples out of 15 were inoculated. This week bacterial samples’ sequences came back so I could start to identify our species. Obviously not everything was perfect and another round of PCR will be needed, but a few strains have now been partially identified and we will be able to carry more experiments to further identify them. Actually for now, most of the species identified seem to degrade cellulose and not indigo, so trying to grow them on M9+indigo plates will tell a great deal about them. And, we have to not forget the fungi samples we have, that is why next week we will be focusing on DNA extraction from fungi and 18S PCR. <H2 class="red">Week 29th August - 4th September</H2><H3> Working with fungi </H3> To perform DNA extraction from fungi, we are following a protocol developed from a previous protocol from a kit for DNA extraction. This protocol uses lyticase, a complex of endonuclease and protease for cell lysis. This enzyme caused us trouble, as going through a few protocols we couldn’t find a consensus concentration, and it varied greatly between protocols. Finally, we used a 10U/mL concentration. As eukaryotes it is more complicated to extract genomic DNA from fungi than it is from bacteria, given that fungi have cell walls containing chitin, and that you need to lyse to access DNA. After performing our DNA following the protocol, we wanted to make sure that it actually worked, so I performed an absorbance measurement using the nanodrop, except most results weren’t good, measuring very little DNA. Could it be that the protocol is not suited for the kind of fungi I have? To figure that out I will do an 18S PCR (since we now have the primers for that!) next week, and if it is negative we will have to think again about our protocol. <H3> On plates </H3> After the first denim experiment, and growing samples on LB, we had to be sure : <LI> that they do grow on denim, some strains on LB plates could just be contamination </LI><LI> that they degrade indigo and not cellulose</LI> For that purpose, last week I prepared M9 plates with either indigo or a piece of denim, and streaked my samples on plates. Throughout the week microbes grew, or not, and now we can see which ones do grow on denim. For indigo plates it is trickier because I did not get a good, blue color on my plates, and I did not try M9 with glucose AND indigo… However, one of our most serious candidate grew on one of these M9 indigo plates: we identified one of our bacterial samples as <I> Streptomyces </I>, a bacteria already known for having laccases that can degrade indigo. <I> Streptomyces coelicolor </I> is used to produce high quantity of bacterial laccases resistant to many various conditions including variation of pH and presence of conventional inhibitors. <I> Streptomyces </I> grew well on denim plates and indigo plates. Few other species seem interesting too, looking at the pattern they make while growing; we thus have a few candidates to further study. <H2 class="red">Week 5th - 11th September</H2><H3> Indigo degradation </H3> We had a few encouraging results with our first experiment on denim and then growing samples on plates with indigo and/or pieces of denim. To go further, we need to assess the degradation (or not) of indigo by our strains, as well as the level of degradation and how long it takes. To do so, I prepared liquid culture of our strains made of liquid M9 and indigo. Indigo was solubilized in DMSO at a 50 mM concentration, and 700 µL were added to 500 mL M9. 5 mL media were put in 50 mL tubes, and 20 µL indigo added. The strains tested in this experiment are: bacterial strains isolated from the first experiment on LB plates, fungal strains from Sabouraud plates (isolated from the first experiment except for one which was isolated from one M9 plate with denim), a few other strains grown a second time on denim plates. For these last strains I carried out a PCR then a gel electrophoresis, and 3 PCR products were purified and sent for sequencing.  We are now measuring absorbance at 700 nm (approximately indigo’s absorbance) to assess indigo degradation. <H2 class="red"> Week 12th - 18th September </H2><H3>Indigo degradation</H3><P>This week I focused on measuring indigo degradation through absorbance. 18 strains, fungal and bacterial, were tested on liquid M9 with indigo, and absorbance was measured once a day by taking 1mL of solution, vortexed and centrifuged, and then putting 200µL in a 96-wells plate. The experiment lasted five days and data were taken for four out of five days. We first did an absorbance scan of only indigo, to know at which wavelength to measure absorbance: we finally chose 700 nm as our wavelength for indigo, which is not far from what was found before (see week   indigo solubility). Absorbance was also measured at 450 nm, lowest absorbance for indigo, to have a broad idea of cell pellets and other stuff’s absorbance. </P><P>At first, the indigo solution was centrifuged. However, some results were not coherent after centrifugation, one strain very obviously do not degrade indigo but after centrifugation absorbance was low and the solution looked very clear. Since indigo is hardly soluble in water (our indigo is made soluble with DMSO), it could alter the results, so I take measurements with centrifuged and not centrifuged solution. Finally, looking at indigo absorbance centrifuged and non-centrifuged I decided to keep “non centrifuged” measurements. (indigo, when centrifuged, aggregate in the bottom of the tube)
After five days the results were good enough. Both fungi and bacteria could degrade indigo, as expected from previous search in literature. Some identified species supposed to degrade indigo showed not so impressive results, but it could be due to the medium used: also for <I>Pleurotus ostreatus</I> spores were used, and as for <I>Streptomyces</I> the strain identified as it did not show great results but other strains later identified as <I>Streptomcyes</I> do. The results are shown in the graph below. </P><H2 class="red"> Week 19 th – 25 th September </H2><H3> Identifying candidates </H3><P> We ordered primers to perform a 18S PCR for fungi. Gel electrophoresis shows good results for 4 fungal strains which happen to be the most interesting strains as to indigo degradation, looking at the results from growing candidates in liquid M9+indigo. <I> Waiting for results of the sanger sequencing</I></P><P><STRONG> Next step: growing our 9 candidates on cotton tainted with indigo </STRONG> We started the liquid culture in M9+indigo experiment with 18 candidates. At the end of the experiment I could identify 9 out of 18 candidates that were the best at degrading indigo: I will be carrying out next experiment with these 9 strains.</P><H2 class="red">Week 26th – 30th September</H2><H3> Getting everything ready for next step ! </H3><P> As said before, next experiment involved cotton and indigo: we want to test our candidates on pure untreated cotton stained with indigo. The reason behind using cotton that we stain ourselves and not denim is previous experiments on denim were not successful. Microbes have been grown two times on pieces of denim put in M9 media without glucose, but no degradation of color has been observed in these two experiments. However by growing our samples in liquid culture with indigo, our third experiment, we could see some degradation (triplicates will be made next week with our candidates to obtain more precise data). The absence of degradation in the two first experiments could be because indigo bound to cotton is less easy to degrade, but it could also be caused by special chemicals used in the making of the denim, treated cotton, etc. That’s why we are now doing a fourth experiment using cotton stained with indigo this time.  To stain the fabric we will try to methods, since indigo can be difficult to work with because of it not being soluble in water. First, we will dissolve indigo in DMSO like we have been doing previously. Our second method is one we found online, a kind of “do it at home” method, in which we will use lye, thiourea dioxide and sodium hydroxide. This will give us two types of colored pieces of cotton. We will put them in M9 media and try our candidates on them. </P><P> We originally had 9 candidates, out of the 18 strains isolated. One of the candidate was Pleurotus Ostreatus, a saprophytic fungi already used in mycoremediation. However, it is a macroscopic fungi so not as easy to work with as other fungi, and its spores did not degrade indigo that much in liquid culture with indigo assay. So, even though it is likely to be an interesting candidate according to literature, we will not work with it any further and focus on only 8 strains, but those 8 strains we know can give us good results and will be easier to work with. </P><P><STRONG> Our 8 candidates : 3 <I> Streptomyces </I>, <I> Pantoea </I>, 3 <I> Aspergillus </I> and <I> Chaetomium globosum </I>. </STRONG></P>. 
<P> This week, in addition to testing the staining of cotton with indigo in DMSO, I also prepared plates and liquid cultures for dna extraction. Not all our strains have been identified : 4 fungi still need to be identified, so I did genomic dna extraction of this 4 fungi, performed a 18S PCR and gel electrophoresis. Also, one <I> Streptomyces </I> was sent this week for genomic sequencing. Finally, I did glycerol stocks of our 18 strains. </P><H2 class="red">Week 3rd – 7th October </H2><P> This week I focused on fungi identification, staining of cotton and doing triplicates for liquid culture for our 8 candidates.  
11 fungal strains were to be identified, so genomic dna was extracted using the same protocol as before, then 18S PCR and gel electrophoresis. Sadly, the dna extraction or PCR didn’t work. For Mission Indigo it’s not a real problem because all the interesting strains we are now focusing on are identified. 
For staining 40mg indigo powder were solubilized in 3 mL DMSO to have around 50 mM indigo solution, and it was used to stain pieces of cotton by dipping them in it. They were then let to dry, to finally be washed to sterilize them and then use them to test strains for degradation of indigo on cotton.  
We have 8 interesting strains: 4 bacteria and 4 fungi. The same experiment in liquid M9 in which indigo is added was made with these 8 microbes, and each of them was made in triplicate. From this experiment we could choose 3 strains that seem to work better than the others at indigo degradation and they will be tested on cotton stained with indigo. </P></DIV></DIV><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:igem2016parisbettencourt@gmail.com">igem2016parisbettencourt@gmail.com</A><CENTER><A href="https://2016.igem.org/">
    2016.igem.org
</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>