<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Project_Binding skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Project/Binding</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navmenubkg"><DIV style="margin-left:-15px" id="navmenubox"><UL id="nav"><LI><A style="margin-left:10px" href="https://2016.igem.org/Team:Paris_Bettencourt/Description">Project</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Description">Overview</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Assay">Assay</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Microbiology">Microbiology</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Enzyme">Enzyme</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Binding">Binding</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Indigo">Indigo</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Model">Model</A></LI></UL><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/About_Perc">Practices</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/About_Perc">About Perc</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Integrated_Practices">Dry Cleaners</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Design">Design</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Human_Practices">Events</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Safety">Safety </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Safety">Lab Safety</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Product_Safety" style="margin-top:-10px; margin-bottom:30px;">ProductSafety</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Achievements">Achievements</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Interlab_Study">Interlab Study</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Medal_Requirements">MedalRequirements</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI style="margin-left:10px"><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Daily Note</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Protocols">Protocols</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Bibliography">Bibliography</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Team">Team </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Team">Our Team </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Sponsors">Sponsors</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><DIV class="projtile1"><H2 class="red" style="text-align:center;">Goals</H2><UL><LI>
        To discover short peptide sequences with affinity for fabrics.
        </LI><LI>
        To characterize the affinity and specificity of these sequences.
        </LI></UL></DIV><DIV class="projtile3"><H2 class="red" style="text-align:center;">Results</H2><UL><LI>Isolated over 200 peptides, 40 each showing binding to cotton, linen, wool, polyester and silk.</LI><LI>Confirmed and quantified binding with ELISA</LI><LI>The peptides found here were passed to the enzyme group for testing on proteins.</LI></UL></DIV><DIV class="projtile4"><H2 class="red" style="text-align:center;">Methods</H2><UL><LI>Phage display</LI><LI>Motif analysis</LI><LI>Quantitative ELISA</LI></UL></DIV><DIV id="subheader"><DIV id="input"><DIV style="margin-top:-50px; margin-bottom:20px; margin-left:60px;"><DIV class="panel"><A href="https://2016.igem.org/Team:Paris_Bettencourt/Results" title="Results"><DIV id="thpanel" class="subpanel3"><DIV class="titlebox2">
	  Skip to results
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Description" title="Project description"><DIV id="thpanel" class="subpanel3"><DIV class="titlebox2">
	    Overview of the project
	  </DIV></DIV></A></DIV></DIV><H2 class="red">Abstract</H2><P><B>In this sub-project we develop Fabric Binding Domains (FBDs), short peptide sequences capable of binding to fabric samples. Results from the Modeling Group suggest that the addition of an FBD with optimal affinity can concentrate an enzyme near the fabric surface and therefore increase stain-degrading enzymatic activity. Using the method of phage display, we isolated 40 short peptides with affinity for cotton, linen, wool, polyester or silk. Bioinformatic analysis identified sequence motifs and biophysical features associated with binding to each fabric, or nonspecific binding to several fabrics. Seven peptides were selected for further analysis by quantitative ELISA, to determine their exact binding constant for each of the five fabrics. The peptides discovered by the Binding Group were passed to the Enzyme Group, where they were used to target GFP and stain-degrading enzymes to wine-stained fabric samples.</B></P><H2 class="red"> Motivation and Background</H2><P>
                    Classical biochemistry studies the behavior of enzymes in well-mixed solutions with simple, mass-action kinetics. However, most chemical systems in the real world are not well mixed. Living cells in particular have a detailed spatial structure, and cells have developed strategies to maximize enzyme performance in a structured environment (Sweetlove, 2013). Many natural metabolic pathways are structured as metabolons, multi-enzyme complexes in which pathway intermediates are physically channeled from one enzyme to the next.In recent years, synthetic biologists have created novel metabolic scaffolds to adapt this strategy to engineered metabolic pathways (Pröschel, 2015). The result has been significantly improved pathway performance: increased reaction rates, fewer side reactions, and better control of metabolic intermediates that may be unstable or toxic. Different mechanisms contribute to the improvement depending on the biochemical details of the scaffolded enzymes (Lee, 2012). However, many models emphasize the ability of enzymatic scaffolding to increase the effective concentration of enzymes and substrates by constraining them to the same physical space.We reasoned that the activity of stain-degrading enzymes might also be enhanced by physically linking them to the fabric surface. Our initial intuitions were confirmed and refined by results from the Modeling Group. In fact, our models predict that optimal enzyme performance will be achieved at intermediate binding affinities. At low affinity, enzymes are primarily found in solution and away from the stain. But at extremely high affinity, enzymes are primarily bound to unstained fabric and also unable to reach the stain. Therefore, we sought to prepare a library of fabric binding domains, allowing us to optimize the binding affinity separately for each enzyme, fabric and application.To identify these Fabric Binding Domains we used the strategy of phage display. In this technique, large libraries of randomized amino acid sequences are displayed by genetically fusing them to phage coat proteins oriented toward the external environment. The phage library is panned against a specific binding target by mixing them and allowing them to reach binding equilibrium. Common targets for phage display libraries include protein antigens and DNA sequences. Some of the randomized peptides displayed on the surface of the phage may become bound to the target while unbound phage are removed by wash steps. The population of bound phage can then be recovered, amplified and further enriched by additional rounds of panning. Because the randomized amino acid library is genetically encoded, individual phage clones can then be isolated and sequenced to identify high affinity peptides.Finally we quantified the binding affinity of specific peptide sequences using colorimetric ELISA. In this technique, a specific phage clone is bound to its high-affinity target and then washed a defined number of times. A primary antibody is introduced to recognize the phage, then a secondary antibody is bound carrying Horse Radish Peroxidase, an enzyme that allows a specific colorimetric reading. By quantifying the number of phage that remain bound after each wash, we can determine the precise binding constant.</P><H2 class="red">Results</H2><P>
    Phage display yields peptides binding cotton, linen, wool, polyester and silk. 
 
Our phage display experiments were performed with the commercial Ph.D.-7 Phage Display Peptide Library (NEB # E8100S). This library consists 100 copies each of 109 phages expressing a randomized 7-mer peptide fused to the phage N-terminal pIII coat protein. The displayed peptide is separated by a short Gly-Gly-Gly-Ser linker to minimize interactions between the displayed peptide and the phage itself.
 
The phage library was panned against fabric samples of cotton, linen, wool, polyester and silk. The cycle of phage binding, elution, and amplification was repeated three times for each fabric. Following enrichment, individual clones were isolated and sequenced to determine the sequence of their displayed peptide. We isolated and sequenced at least 40 phage plaques from each fabric (Table 1A).</P><DIV id="figurebox"><DIV style="text-align:center;"><TABLE style="width:90%"><TBODY><TR><TH align="center">COTTON</TH><TH align="center">LINEN</TH><TH align="center">SILK</TH><TH align="center">WOOL</TH><TH align="center">POLYESTER</TH></TR><TR><TD align="center">ADARYKS</TD><TD align="center">ADARYKS</TD><TD align="center">ADARYKS</TD><TD align="center">ADARYKS</TD><TD align="center">ADARYKS</TD></TR><TR><TD align="center">ADARYKS</TD><TD align="center">AGHVVPR</TD><TD align="center">ADARYKS</TD><TD align="center">ADARYKS</TD><TD align="center">ADARYKS</TD></TR><TR><TD align="center">ADARYKS</TD><TD align="center">ASPDQEK</TD><TD align="center">AGHVVPR</TD><TD align="center">ADARYKS</TD><TD align="center">ADARYKS</TD></TR><TR><TD align="center">ADARYKS</TD><TD align="center">HDSPTAA</TD><TD align="center">AQSNPKN</TD><TD align="center">ADARYKS</TD><TD align="center">ASSHIHH</TD></TR><TR><TD align="center">ADARYKS</TD><TD align="center">HDVMWQR</TD><TD align="center">AQSNPKN</TD><TD align="center">ADARYKS</TD><TD align="center">FRKKRKS</TD></TR><TR><TD align="center">ALANFEP</TD><TD align="center">HWNTVVS</TD><TD align="center">ASSHIHH</TD><TD align="center">ADARYKS</TD><TD align="center">GALAKDE</TD></TR><TR><TD align="center">ASSHIHH</TD><TD align="center">MPRLPPA</TD><TD align="center">AWPYVTL</TD><TD align="center">ADARYKS</TD><TD align="center">GASNIWN</TD></TR><TR><TD align="center">ASSHIHH</TD><TD align="center">MPRLPPA</TD><TD align="center">AWPYVTL</TD><TD align="center">ADARYKS</TD><TD align="center">HWNTVVS</TD></TR><TR><TD align="center">ASSHIHH</TD><TD align="center">MPRLPPA</TD><TD align="center">DETCSSM</TD><TD align="center">ADIRHIK</TD><TD align="center">ISTTLFP</TD></TR><TR><TD align="center">ASSHIHH</TD><TD align="center">MPRLPPA</TD><TD align="center">FPSPMVG</TD><TD align="center">ADIRHIK</TD><TD align="center">METVVSS</TD></TR><TR><TD align="center">DPRLSPT</TD><TD align="center">MPRLPPA</TD><TD align="center">GKNLMNM</TD><TD align="center">ADIRHIK</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">ELAGTTW</TD><TD align="center">MPRLPPA</TD><TD align="center">HDVMWQR</TD><TD align="center">ASSHIHH</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">ERGFLLL</TD><TD align="center">MPRLPPA</TD><TD align="center">HWNTVVS</TD><TD align="center">ASSHIHH</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">FSRSNNT</TD><TD align="center">MPRLPPA</TD><TD align="center">KNANSRE</TD><TD align="center">ASSHIHH</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">FSRSNNT</TD><TD align="center">MPRLPPA</TD><TD align="center">KNANSRE</TD><TD align="center">GQSVVSL</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">GLHYDHS</TD><TD align="center">MPRLPPA</TD><TD align="center">KTAMKGP</TD><TD align="center">GSTSFSK</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">GVKSEQL</TD><TD align="center">MPRLPPA</TD><TD align="center">MLQGNGY</TD><TD align="center">GSTSFSK</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">GVLRYAP</TD><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD><TD align="center">GSTSFSK</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">HNWMHQN</TD><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD><TD align="center">HDSPTAA</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">HYPPVDD</TD><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD><TD align="center">HDSPTAA</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">ISTTLFP</TD><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD><TD align="center">MRLSVPN</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">MRLSVPN</TD><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD><TD align="center">MRLSVPN</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">MSNTLDP</TD><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD><TD align="center">PSNRQNT</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">MTQQLHT</TD><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD><TD align="center">QFDHWRN</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">QGDYFTY</TD><TD align="center">MPRLPPA</TD><TD align="center">MPRLPPA</TD><TD align="center">QFDHWRN</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">RLLQYNS</TD><TD align="center">QFPPPPG</TD><TD align="center">MPRLPPA</TD><TD align="center">SILPVTR</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">SFLVTRN</TD><TD align="center">QPIYRVQ</TD><TD align="center">MPRLPPA</TD><TD align="center">SILPVTR</TD><TD align="center">MPRLPPA</TD></TR><TR><TD align="center">SIHERAK</TD><TD align="center">SILPVTR</TD><TD align="center">MPRLPPA</TD><TD align="center">SILPVTR</TD><TD align="center">MQEMRQM</TD></TR><TR><TD align="center">SILPVTR</TD><TD align="center">SILPVXR</TD><TD align="center">MPRLPPA</TD><TD align="center">SILPVTR</TD><TD align="center">MRLSVPN</TD></TR><TR><TD align="center">SSHSVQR</TD><TD align="center">SLLTHNM</TD><TD align="center">MPRLPPA</TD><TD align="center">SILPVTR</TD><TD align="center">NNSVSMN</TD></TR><TR><TD align="center">SVVMPHG</TD><TD align="center">STNPTSL</TD><TD align="center">MPRLPPA</TD><TD align="center">SILPVTR</TD><TD align="center">SILPVTR</TD></TR><TR><TD align="center">TDHAHRY</TD><TD align="center">TLINYRG</TD><TD align="center">MPRLPPA</TD><TD align="center">SILPVTR</TD><TD align="center">SLETMSN</TD></TR><TR><TD align="center">TDMTAPK</TD><TD align="center">TNLHINP</TD><TD align="center">MPRLPPA</TD><TD align="center">SILPVTR</TD><TD align="center">SNYHWRM</TD></TR><TR><TD align="center">TSDATQR</TD><TD align="center">WTNVFVG</TD><TD align="center">MPRLPPA</TD><TD align="center">TRPTDTI</TD><TD align="center">TESAPTL</TD></TR><TR><TD align="center">TTHPRWG</TD><TD align="center">XPRLPPA</TD><TD align="center">SILPVTR</TD><TD align="center">TVHVHKT</TD><TD align="center">VFQTTYK</TD></TR><TR><TD align="center">VECINNC</TD><TD align="center">XPRLPPX</TD><TD align="center">SILPVTR</TD><TD align="center">YMGPSKT</TD><TD align="center">VPRLPPA</TD></TR><TR><TD align="center">VTLPDPR</TD><TD align="center">XPRLXPA</TD><TD align="center">TSNRAPY</TD><TD align="center">YMGPSKT</TD></TR><TR><TD align="center">WHLPAQR</TD><TD align="center">XPXXPPT</TD><TD align="center">XPRLPPA</TD><TD align="center">YMGPSKT</TD></TR></TBODY></TABLE><P><B>Table 1A</B> Novel fabric binding domains with diverse properties.
</P></DIV><DIV style="text-align:center;"><TABLE style="width:90%"><TBODY><TR><TH align="center">COTTON</TH><TH align="center">LINEN</TH><TH align="center">SILK</TH><TH align="center">WOOL</TH><TH align="center">POLYESTER</TH></TR><TR><TD align="center"><FONT color="#8B1813"><B>FSRSNNT(2)</B></FONT></TD><TD align="center" rowspan="2"><FONT color="#438CEE"><B>SILPVTR(2)</B></FONT></TD><TD align="center"><FONT color="#183255"><B>AQSNPKN(2)</B></FONT></TD><TD align="center"><FONT color="#183F24"><B>MRLSVPN(2)</B></FONT></TD><TD align="center" rowspan="2"><FONT color="#138B3F"><B>ADARYKS(3)</B></FONT></TD></TR><TR><TD align="center"><FONT color="#F6403A"><B> MPRLPPA(2)</B></FONT></TD><TD align="center"><FONT color="#8C998D"><B>AWPYVTL(2)</B></FONT></TD><TD align="center"><FONT color="#FF33E3"><B>ASSHIHH(3)</B></FONT></TD></TR><TR><TD align="center"><FONT color="#FF33E3"><B>ASSHIHH(4)</B></FONT></TD><TD align="center" rowspan="3"><FONT color="#F6403A"><B> MPRLPPA(25)</B></FONT></TD><TD align="center"><FONT color="#08807A"><B>KNANSRE(2)</B></FONT></TD><TD align="center"><FONT color="#138B3F"><B>ADARYKS(6)</B></FONT></TD><TD align="center" rowspan="3"><FONT color="#F6403A"><B> MPRLPPA(20)</B></FONT></TD></TR><TR><TD align="center" rowspan="2"><FONT color="#138B3F"><B>ADARYKS(5)</B></FONT></TD><TD align="center"><FONT color="#438CEE"><B> SILPVTR(2)</B></FONT></TD><TD align="center" rowspan="2"><FONT color="#438CEE"><B> SILPVTR(7)</B></FONT></TD></TR><TR><TD align="center"><FONT color="#F6403A"><B> MPRLPPA(20)</B></FONT></TD></TR></TBODY></TABLE><P><B>Table 1B</B> Table of all peptide sequences observed to bind more than once. The number in parenthesis indicates the number of times the respective peptide was recovered. These candidate peptides were validated for binding affinity with an ELISA.
</P></DIV></DIV><P>
 
We focused our analysis on phage sequences that appeared at least twice. Isolated peptides displayed a range of fabric specificities, sequence motifs and phyicochemical properties (Table 2). Nine peptide sequences were selected for further analysis, chosen with a preference for chemical diversity and highly enriched peptides that appeared in many clones. Five of the nine peptides were fabric-specific, meaning they were isolated from only one fabric, while the others were found to bind multiple fabrics. The peptides were designated FBD 1-9.</P><DIV id="figurebox"><DIV style="text-align:center;"><TABLE style="width:90%"><TBODY><TR><TH align="center">COTTON</TH><TH align="center">LINEN</TH><TH align="center">SILK</TH><TH align="center">WOOL</TH><TH align="center">POLYESTER</TH></TR><TR><TD align="center"><B>Positive Charged</B></TD><TD align="center">MPRLPPA(2)
                                                SILVPTR
                                                ADARYKS(5))
                                                SFLVTRN
                                                MRLSVPN
                                                GVLRYAP</TD><TD align="center">MPRLPPA(25) 
                                                HWNTVVS)
                                                SILVPTR(2) 
                                                AGHVVPR)
                                                ADARYKS)</TD><TD align="center">MPRLPPA(20) 

                                                HWNTVVS

                                                SILVPTR(2) 

                                                AGHVVPR

                                                ADARYKS(2) </TD><TD align="center">MPRLPPA

                                                SILVPTR(7) 

                                                ADARYKS(9) 

                                                MRLSVPN(2) 

                                                YMGPSKT(4) </TD><TD align="center">MPRLPPA(20) 

                                                HWNTVVS                  ADARYKS(3) 

                                                MRLSVPN</TD></TR><TR><TD align="center"><B>Negative Charged</B></TD><TD align="center">ELAGTTW
                                                ALANFEP
                                                VECINNC</TD><TD align="center">HDSPTAA)</TD><TD align="center">HDSPTAA</TD><TD align="center">METVVSS

                                                TESAPTL</TD></TR><TR><TD align="center"><B>Hydrophilic</B></TD><TD align="center">FSRSNNT(2) 
                                                HNWMHQN
                                                TDHAHRY

                                                TDMTAPK

                                                GVKSEQL

                                                VTLPDPR

                                                MTQQLHT

                                                SIHERAK

                                                DPRLSPT

                                                WHLPAQR

                                                GLHYDHS

                                                HYPPVDD

                                                RLLQYNS

                                                TSDATQR

                                                QGDYFTY

                                                TTHPRWG

                                                SSHSVQR

                                                ASSHIHH(4) 

                                                MSNTLDP</TD><TD align="center">TLINYRG

                                                FRKKRKS

                                                TNLHINP

                                                QFPPPPG

                                                QPIYRVQ

                                                HDVMWQR

                                                ASPDQEK

                                                STNPTSL</TD><TD align="center">KNANSRE(2) 

                                                KTAMKGP

                                                HDVMWQR

                                                GKNLMNM

                                                TSNRAPY

                                                AQSNPKN(2) 

                                                ASSHIHH</TD><TD align="center">QFDHWRN

                                                TVHVHKT

                                                ADIRHIK(3) 

                                                PSNRQNT

                                                TRPTDTI

                                                ASSHIHH(3) 

                                                GSTSFSK(3) </TD><TD align="center">NNSVSMN

                                                VFQTTYK

                                                GALAKDE

                                                DETCSSM

                                                SNYHWRM

                                                SLETMSN          MQEMRQM

                                                ASSHIHH</TD></TR><TR><TD align="center"><B>Hydrophobic</B></TD><TD align="center">ERGFLLL

                                            ISTTLFP

                                            SVVMPHG</TD><TD align="center">WTNVFVG

                                                SLLTHNM</TD><TD align="center">AWPYVTL(2) 

                                            MLQGNGY

                                            FPSPMVG</TD><TD align="center">GQSVVSL</TD><TD align="center">ISTTLFP

                                                GASNIWN</TD></TR></TBODY></TABLE><P><B>Table 2</B> Table of the sequence motifs and physico chemical properties of the peptides
</P></DIV></DIV><H3>Validation of the peptides using ELISA </H3><P>
        The nine selected FBD peptides were validated with ELISA. Briefly, a clonal library of phage expressing each peptide was incubated with each of five fabric samples under conditions similar to the original panning experient. Following extensive washing, the fabric was incubated with an anti-phage primary antibody, then a secondary antibody linked to HRP, the Horse Radish Peroxidaze. The HRP enzyme produces a pigmented product, allowing semi-quantitative measurement of the level of fabric-bound phage.
 
        

        In figure 1, we compare the ELISA signal before and after washing for phage displaying each of the FBD peptides against each fabric. In most cases, the phage are retained after washing and produce an ELISA signal significantly above control. Peptides were found to have a range of affinities for different fabrics. Notably wool appeared to be difficult to bind, with only FBD9 staying bound after washing.
 
        Repeated washing reduced signal, allowing a quantitative measurement of binding affinitiy. In a typical case of FBD9 binding to cotton, more than 50% signal was retained after 10 rounds of washing. Analysis of these data is ongoing, with the goal of estimating the KD for each binding interaction.
        </P><DIV id="figurebox"><DIV style="text-align:center;"><P><B>Figure 1</B> Fabric binding domain validation by ELISA. <B>A.</B> ELISA for each FBD incubated with each fabric after two rounds of washing, normalized to the level after a single round. <B>B.</B> Control matrix repeating the procedure of panel A without phage. <B>C.</B> ELISA signal determined ater multple rounds of washing. Points represent individual measurements from three replicates. Lines represent best-fit to an exponential decay model.
        </P></DIV></DIV><H2 class="red">Methods</H2><H3>Selection of fabric samples for panning</H3><P>
                                              
                        Cotton was obtained from Khadi &amp; Co (Hand Woven, 100% cotton). Wool and Linen were obtained from Dharma Trading Co. (#PWFC, #LIN21). Silk thread was obtained from Au Ver à Soie (Thread 1003, Crème color, 100% Silk). Polyester thread was obtained from Mediac (Ref 960, Color 400, 100% Polyester). 
                    </P><H3>Preparation of fabric samples for panning</H3><P>
                        To remove possible factory coatings or treatments, cotton was boiled in sodium carbonate solution then washed with water until pH neutral.
                        The other fabrics were washed thoroughly with ethanol then water.</P><H3>Detailed protocol for phage display</H3><P>
                        Panning was performed with 10 µl of Ph.D.-7 Phage from NEB # E8100S, at a titer of 1013 pfu/ml, was mixed with 16 mg washed and blocked fabric to give a concentration of 2*1011 phage. Fabric was washed vigorously with 0.1% TBST to remove unbound phage. Phage were further eluted with 0.2 M Glycine-HCl (pH 2.2) and 1 mg/ml BSA to disrupt all nonspecific binding interactions, then neutralized with 1 M Tris-HCl, pH 9.1.
                        The eluate was amplified by infecting a 5 ml culture of E. coli overnight, then mixed with 20% PEG/NaCl to precipitate the phages. Precipitated phages were resuspended in TBS and titered by the method of top agar to calculate the input for next round of panning. After 3 additional rounds of panning, individual phage clones were isolated with the method of top agar.
  
                        </P><H3>Sequencing of Phage DNA</H3><P>
                        Single plaques were picked from fresh titration plates and amplified in E. coli. Phage were precipitated with 20% PEG/NaCl, then were resuspended in Iodide buffer. Phage DNA was then precipitated by incubation with 70% ethanol. DNA was washed with ice cold 70% ethanol, air dried and later resuspended in DNAse free distilled water and quantified with a nanodrop spectrophotometer.
                    
                    </P><H3>Binding quantification with ELISA</H3><P> 
                       
                        Single phage clones were applied to selected fabrics at a titer of  1011 virions. Experimental treatments were washed twice with TBST prior to ELISA, while control treatments were washed only once. Bound phage ws detected with an HRP-conjugated anti-M13 antibody , GE Healthcare #27-9421-01). ABTS (Sigma # A-1888) peroxidase substrate was added following the manufacturers instructions and produced a measurable green product read at 405 nm.
                        
                    </P><H2 class="red">Attributions</H2><P>
                    This project was done mostly by Shruthi and Thomas. We would like to thank Clément Nizak and the Nizak lab for help in designing the phage display protocol. 
                </P><H2 class="red">References</H2><UL><LI>Sweetlove, L. J., &amp; Fernie, A. R. (2013). The Spatial Organization of Metabolism Within the Plant Cell. Annual Review of Plant Biology, 64(1), 723–746.</LI><LI>Lee, H., DeLoache, W. C., &amp; Dueber, J. E. (2012). Spatial organization of enzymes for metabolic engineering. Metabolic Engineering, 14(3), 242–251.</LI><LI>Pröschel, M., Detsch, R., Boccaccini, A. R., &amp; Sonnewald, U. (2015). Engineering of Metabolic Pathways by Artificial Enzyme Channels. Frontiers in Bioengineering and Biotechnology, 3(Pt 5), 123–13.</LI></UL></DIV><DIV style="margin-top:20px; margin-bottom:20px"><DIV class="panel"><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Assay" title="Assay"><DIV id="pspanel" class="subpanel2" onmouseover="chgtrans(this)"><DIV class="titlebox">
	    Assay
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Microbiology" title="Microbiology"><DIV id="dspanel" class="subpanel2"><DIV class="titlebox">
	    Microbiology
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Enzyme" title="Enzyme"><DIV id="tcpanel" class="subpanel2"><DIV class="titlebox">
	  Enzyme
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Binding" title="Binding domains"><DIV id="thpanel" class="subpanel2"><DIV class="titlebox">
	    Binding
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Indigo" title="Indigo"><DIV id="thpanel" class="subpanel2"><DIV class="titlebox">
	    Indigo
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Model" title="Model"><DIV id="thpanel" class="subpanel2"><DIV class="titlebox">
	    Model
	  </DIV></DIV></A></DIV></DIV></DIV></DIV><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:igem2016parisbettencourt@gmail.com">igem2016parisbettencourt@gmail.com</A><CENTER><A href="https://2016.igem.org/">
    2016.igem.org
</A></CENTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>