<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Project_Indigo skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Project/Indigo</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navmenubkg"><DIV style="margin-left:-15px" id="navmenubox"><UL id="nav"><LI><A style="margin-left:10px" href="https://2016.igem.org/Team:Paris_Bettencourt/Description">Project</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Description">Overview</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Assay">Assay</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Microbiology">Microbiology</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Enzyme">Enzyme</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Binding">Binding</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Indigo">Indigo</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Model">Model</A></LI></UL><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/About_Perc">Practices</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/About_Perc">About Perc</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Integrated_Practices">Dry Cleaners</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Design">Design</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Human_Practices">Events</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Safety">Safety </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Safety">Lab Safety</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Product_Safety" style="margin-top:-10px; margin-bottom:30px;">ProductSafety</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Achievements">Achievements</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Interlab_Study">Interlab Study</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Medal_Requirements">MedalRequirements</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI style="margin-left:10px"><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Daily Note</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Protocols">Protocols</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Bibliography">Bibliography</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Team">Team </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Team">Our Team </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Sponsors">Sponsors</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><DIV class="projtile1"><H2 class="red" style="text-align:center;">Goals</H2><UL><LI> Find microorganisms that naturally degrade indigo </LI><LI> Test enzyme known for degrading indigo </LI><LI> Make patterns on jeans </LI></UL></DIV><DIV class="projtile3"><H2 class="red" style="text-align:center;">Results</H2><UL><LI>3 strains degrading indigo in liquid M9 were identified</LI><LI>bpul laccase is proven and quantified for indigo degradation</LI></UL></DIV><DIV class="projtile4"><H2 class="red" style="text-align:center;">Methods</H2><UL><LI> Cultures in minimal M9 media (with indigo), and on cotton stained with indigo </LI><LI> 16S rRNA and 18S rRNA sequencing </LI><LI> Indigo carmine and indigo degradation absorbance measurements </LI><LI> SDS-PAGE electrophoresis </LI><LI> Protein induction and extraction </LI></UL></DIV><DIV id="subheader"><DIV id="input"><DIV style="margin-top:0px; margin-bottom:20px; margin-left:70px;"><DIV class="panel"><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Indigo#ancre" title="Results"><DIV id="thpanel" class="subpanel3"><DIV class="titlebox2">
	  Skip to results
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Description" title="Project description"><DIV id="thpanel" class="subpanel3"><DIV class="titlebox2">
	    Overview of the project
	  </DIV></DIV></A></DIV></DIV><H2 class="red"> Abstract </H2><P><B>
The main focus of our project this year was enzyme design and discovery. We chose to engineer enzymes because they are a practical technology for removing stains, already known to be safe and economical. However, in our search for microbes that produce stain-removing enzymes we discovered something else: microbes are beautiful. When nutrients are added to fabric samples, bacterial and fungal colonies bloom in every size, shape and color. These microbes grow in beautiful patterns, remove pigments from fabric and leave other pigments behind. When the microbes were gone, the color and texture of the fabric was changed in ways that looked cool to us. What we describe below is not necessarily practical and only somewhat scientific. We are playing around with microbes on ordinary denim blue jeans and trying to create beauty.
</B></P><H2 class="red"> Motivation and Background </H2><H3>Blue Jeans: Structure and Function</H3><P>
Jeans are trousers made with denim, a cotton textile. Blue jeans are dyed with indigo, a plant-derived pigment that is now mostly produced synthetically. In a classic blue denim weave, the warp thread is dyed with indigo while the weft threat is left white. The resulting fabric is blue on one surface, white on the other, with a grainy texture where the fibers cross. The blue threads are stained only on the outside and stay white at the core. As a result, the fabric lightens with wear, producing a characteristic fading pattern (Figure 1).
</P><H3>Denim Washes</H3><P>
The popular appeal of faded denim has lead to the development of many industrial processes that simulate or accelerate the fading process. New denim may be chemically treated, sandblasted, rubbed with sandpaper, or tumbled with abrasive stones prior to sale. These &quot;wash&quot; processes soften the fabric, fade the indigo, and produce a distinctive pattern and appearance. Especially with the trend toward high fashion and premium jeans, designers are motivated to discover new and innovative wash processes. The global jeans market was predicted to be 56 billion USD in 2014 (Agarwal, 2009).
</P><H3>Enzymatic Biowashing</H3><P>
In last few decades, enzymes have become a popular tool in the denim industry. Enzymes are mostly used in two ways: for bleaching the indigo dye from jeans, and for treating the toxic effluents after the denim wash. Enzymatic methods are used because they require less energy, produce fewer chemical by-products, and cause less damage than chemical or abrasive washes (Couto, 2006). Some of the enzyme treatments use fungal laccases to fade denim (Campos, 2001). Laccases (EC 1.10.3.2) are copper-containing oxidases that act on a variety of substrates. Acting on indigo dye, laccases abstract one electron from each aromatic amine, leaving behind an unstable radical that quickly degrades. Whole live cells with laccase activity have been used as bioremediation agents, in particular for indigo-containing wastewater from textile factories (Conceição, 2013). This activity was also the inspiration for the Bielefeld 2012 iGEM team. 

Thus, guided by this knowledge, we decided to focus on laccase enzymes to degrade indigo, and to potentially produce beautiful patterns on the jeans when placed in the right micro-organism. More precisely, we are focusing on bpul, a laccase naturally found in the spore coat of Bacillus pumilus. There are several reasons why we have chosen this enzyme:
<OL><LI>
It already exists in the registry of Standard Biological Parts, and it is one of the most well documented BioBricks.
</LI><LI>
Apart from the registry, a good biochemical analysis and methods can be found in the paper by Reiss et. al 2011.
</LI><LI>
The laccasse comes from bacteria (Bacillus pumilus) which makes it potentially easier to express and study in E. coli than fungal lacasses.
</LI><LI>
To our knowledge, bpul hasn't been shown to degrade indigo. On the other hand Reiss et. Al 2011. have shown that it successfully degrades indigo carmine which is structurally similar to indigo dye (figure 4A).
</LI><LI>
Due to a large number of substrates laccases can potentially degrade compounds from wine making it a good basis for collaboration with the Enzyme Group.
</LI></OL></P><H2 class="red" id="ancre"> Results </H2><P> We cultured microorganisms on denim covered with minimal M9, isolated the strains and identified them. Then, we tested their capacity for degrading indigo. At the end we isolated three indigo-degrading microorganisms: 2 <I>Streptomyces</I> and 1 <I>Pantoea</I> species.
</P><H3>Isolation of Microbes on Denim Media</H3><P>After approximately one week of incubation, we were able to isolate 16 strains of different colors and shapes on denim-covered media, half of which were bacteria, half fungi. We identified the bacterial and fungal strains through 16S and 18S rRNA sequencing, respectively; however, a few fungi species we isolated are still unknown, as we did not succeed in extracting genomic DNA from these species for PCR. These 16 species are mostly <I>Aspergillus</I>, <I>Streptomyces</I> and <I>Enterobacter</I> species. All isolated strains are shown in <B>table 1</B> below. These results are coherent with the literature: indeed <I>Enterobacter</I> are commonly used in bioreactors for waste and dye cleaning, and <I>Aspergillus fumigatus</I> and some <I>Streptomyces</I> have been shown to degrade certain dyes, specifically indigo in the case of <I>Streptomyces</I>. 
</P><DIV id="figurebox"><DIV style="text-align:center;"><P><STRONG>Figure 1. Microbes grown on denim. A. Pieces of denim </STRONG> with different types of microbial growth. <STRONG> B. </STRONG> All 16 strains recovered from denim or on a plate. From top to bottom and left to right: F1, S1, F2, S2, F3, E1, S3, F4, F5, S4, F6, E2, F7, P1, P2, F8. </P></DIV></DIV><TABLE style="text-align:center; margin: 0 auto;"><THEAD><CAPTION><STRONG> Table 1 : Microbes found on denim</STRONG></CAPTION><TR><TH colspan="4"> Bacteria </TH><TH colspan="4"> Fungi </TH></TR><TR><TH> Name </TH><TH> Species </TH><TH> Name </TH><TH> Species </TH><TH> Name </TH><TH> Species </TH><TH> Name </TH><TH> Species </TH></TR></THEAD><TBODY><TR><TD> S1</TD><TD>Streptomyces albidoflavus</TD><TD> S2 </TD><TD> Streptomyces globisporus </TD><TD>F1</TD><TD>Aspergillus Fumigatus</TD><TD> F2 </TD><TD> Aspergillus Fumigatus </TD></TR><TR><TD> S3</TD><TD> Streptomyces fumigatiscleroticus </TD><TD> S4 </TD><TD> Streptomyces coelicolor</TD><TD> F4 </TD><TD> Unknown </TD><TD> F3 </TD><TD> Unknown </TD></TR><TR><TD> E1 </TD><TD> Gamma proteobacterium </TD><TD> E2 </TD><TD> Enterobacter </TD><TD> F6 </TD><TD> Chaetomium Globosum </TD><TD> F5 </TD><TD> Unknown </TD></TR><TR><TD> P1 </TD><TD> Pantoea agglomerans </TD><TD> P2 </TD><TD> Pantoea agglomerans</TD><TD> F8 </TD><TD> Unknown </TD><TD> F7 </TD><TD> Aspergillus Fumigatus </TD></TR></TBODY></TABLE><P>We then wanted to determine whether the isolated strains were capable of degrading or metabolizing indigo. As denim is made of cotton, it could be possible that the microbes use cellulose as a carbon source rather than indigo. Cellulases are already a popular enzymatic substitute used to fade blue jeans, but we wished to find microbes capable of degrading indigo itself. To do this, we cultured our strains on petri dishes containing indigo as the only carbon source. Only two of our strains were able to grow: <I>Pantoea agglomerans</I> P1 and <I>Streptomyces</I> S2 <B>figure 2</B>. However, as the indigo we used was not sufficiently visible in the plates, we were unable to observe indigo digestion due to color change of the media, and bacterial growth itself was the only evidence of &quot;indigo degradation&quot; using this method.
</P><P>As we were not satisfied with the results of the plate assay, we redid the indigo digestion assay in liquid minimal media containing indigo with all of our isolated strains <B>figure 2</B>. We observed three strains capable of indigo degradation: <I>Pantoea agglomerans</I> P1 and <I>Streptomyces coelicolor</I> S4 strains that grew on the agar-indigo plates, as well as strain <I>Streptomyces fumigatiscleroticus </I> S3. These results offered further proof that we had successfully isolated microbes capable of indigo digestion.
</P><DIV id="figurebox"><DIV style="text-align:center;"><P><STRONG>Figure 2. Degradation of indigo on solid and liquid media. A. </STRONG> Only two strains were able to grow on plates supplemented with indigo: <I> Streptomyces</I> S3 (left) and <I>Pantoea agglomerans</I> P1 (right). <STRONG> B. </STRONG>Degradation of indigo is highly visible in liquid M9. <I> Streptomyces fumigatiscleroticus</I> S3, <I> Streptomyces coelicolor </I> S4 and indigo media without microorganisms.</P></DIV></DIV><H3>Quantification of Indigo Degradation</H3><P>In order to quantify indigo degradation, we then carried out two rounds of selection in order to find the highest indigo degrading strains as determined through optical density (OD) measurements. We started with all 16 strains isolated from denim, and after several days of incubation we were able to select 8 strains for the second round: F1, F2, F6, F7, S1, S3, S4 and P1.</P><P>In the second round of selection, the 8 strains chosen from the first round were grown in the same conditions, but in triplicate, measuring the OD each day. As the deep blue color of the indigo confounds spectrophotometric reading in the first couple of days of the experiment, we only analyzed time points beginning at day 3 of the experiment. As shown in figure 2B, indigo degradation was easy to visualize, as the media changes from dark blue to nearly transparent. OD measurements for the 8 strains at the final day 7 time point are shown in <STRONG>figure 3</STRONG>.</P><P>
In the end, the three strains we identified from this experiment, <I>Streptomyces fumigatiscleroticus</I>, <I>Streptomyces coelicolor</I>, and <I>Pantoea agglomerans</I>, were coherent with the strains observed in the plate and culture tube experiments described above.</P><DIV id="figurebox"><DIV style="text-align:center;"><P><STRONG>Figure 3: Indigo absorbance measurements. A : Indigo absorbance at last day of first round experiment. </STRONG> Measured for 16 strains and  <I> P. ostreatus </I>. <STRONG> B: Indigo absorbance throughout the experiment</STRONG> The ten most interesting strains are shown. S1 and S4 were not used for further experiments and we finally kept 8 strains for the second round.
</P></DIV></DIV><H3>Microbe Wash Denim</H3><P>Knowing that our three last strains degrade indigo in liquid and more or less on plates, and keeping our goal of denim fading in mind, we needed to asses degradation of indigo on fabric. In order to do so we used cotton: as denim is made of cotton but is a very thick fabric. Pieces of cotton were stained with indigo (see more on staining with indigo in the <A href="#ancremethods">methods</A>). Then they were put in either solid media or liquid media: for plates M9 media was used, for liquid culture it was liquid M9 and LB. The three remaining strains, S3, S4 and P1, were cultured for one week in plates and liquid media with these pieces of stained cotton, and for liquid culture one tube were left without microbes as a negative control. Unfortunately, after a week we cannot see noticeable change of color on cotton pieces. However, this is not the only experiment we carried out on fabric: we also left colored threads from denim in LB with strain S4 for two months. And after two months, the result is the following: color faded on these threads. Comparison with untreated threads can be seen in <STRONG> figure 4 </STRONG>. In conclusion, the process of indigo’s removal by microbes seems to be a long but effective process. </P><DIV id="figurebox"><DIV style="text-align:center;"><P><STRONG>Figure 4. </STRONG> Indigo degradation on threads. Threads were incubated with <I> Streptomyces coelicolor </I> S4 strain in LB for two months (left). Comparison with threads from the same pair of jeans that were not put in media with microbes (right) </P></DIV></DIV><H3> Bpul Indigo Digestion </H3><P>
At the beginning of the Indigo project we have used the BioBrick Bba_K863001 from the registry of the Standard Biological Parts. Later, our own version of bpul (Bba_K863007) was made in the collaboration with the Enzymes team. Our bpul gene is placed under T7 promoter, and it is codon optimized for expression in E.coli. After the translation, a His-tag attached to the C-terminus of Bpul protein allows simple purification.

All results shown in the <STRONG>figure 5</STRONG> are made with our bpul gene expressed in the E. coli strain BL21(DE3). The protein gel (<STRONG>figure 5B</STRONG>) is showing an over-expression of the bpul, after the induction with IPTG. When comparing uninduced bpul sample with the control sample (BL21(DE3)) we can notice leakiness of our promoter, which is probably due to great transcriptional strength of T7 polymerase.
 
In the later experiments, both the transformant BL21(DE3)-bpul, and the negative control BL21(DE3) have been induced with IPTG. The cell extracts of both strains were tested for the ability to degrade indigo carmine (<STRONG>figure 5C</STRONG>) and indigo (<STRONG>figure 5D</STRONG>). Additionally, 10µM acetosyringone (ACS) was added to the reaction mixture to facilitate the oxidation of the substrates. ACS is a known mediator of the laccases and it is often used and needed in the studies of the  laccase activitiy As expected, the 1mM indigo carmine solution is entirely degraded within the 13 hours of the experiment in the cell extract which contains overexpressed bpul. Additionally, we have shown that more than 60% of indigo can be degraded by bpul laccase in the 13 hours of the experiment. It is important to notice, that in longer  experiments would probably lead to even more degradation since the degradation curve didn't reach the saturation.
</P><DIV id="figurebox"><DIV style="text-align:center;"><P><B>Figure 5.</B> (<B>A</B>) Structural formulas of two substrates (indigo carmine, and indigo) of bpul laccase are showing structural similarity. (<B>B</B>) SDS-PAGE electrophoresis gel with samples from left to right: BIO-RAD kaleidoscope protein ladder; BL21(DE3) cell extract not induced; BL21(DE3) cell extract induced with 0.5 mM IPTG; BL21(DE3)-bpul cell extract not induced; BL21(DE3)-bpul cell extract induced with 0.5mM IPTG. Electrophoresis was performed in BIO-RAD's Mini-PROTEAN Tetra cell electrophoresis gear on 10% BIO-RAD mini-PROTEAN precast TGX gel. (<B>C</B> and <B>D</B>) Indigo carmine and indigo degradation by cell extract containing bpul are shown over a period of 13h. The degradation was observed in 96-well plate at the λ=650 for indigo carmine, and λ=680 nm for indigo, at room temperature. 1 µL of cell extract was added to the reaction mixture which consisted of substrate dissolved in 0.1 mM of potassium phosphate buffer (pH 7.8), and 10 µM ACS added to the reaction mix. 
</P></DIV></DIV><H2 class="red"> Methods </H2><H3> Preparation of Denim and Indigo Media </H3><P> We used denim pieces cut out of a pair of jeans for our experiment. We put these pieces onto M9 minimal media agar supplemented with CaCL2 and MgSO4, but without a glucose carbon source. One piece was incubated in a bottle with the lid almost closed to avoid dehydration; the other pieces were put in onto square plates with minimal M9. After a few days different colonies were taken with an inoculation loop and inoculated on LB plates for growth. </P><H3> Microbial Identification by rRNA Sequencing </H3><P> To identify bacterial species, 16S sequences were amplified from isolated colonies by PCR. These samples were sequenced via Sanger sequencing (GATC Biotech), and identified through BLAST alignment (https://www.ncbi.nlm.nih.gov/BLAST/). The full protocol can be found <A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Protocols">here</A>. </P><P>Fungi were identified through 18S rRNA sequencing. For each candidate species, one colony from an LB plate was inoculated on Sabouraud media and left to grow for 3 days. Genomic DNA was extracted using a DNeasy Blood and Tissue kit and a modified protocol, and was then used for 18S rRNA PCR (protocol can be found <A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Protocols">here</A>). </P><H3> Quantification of Indigo Consumption</H3><P><U> Indigo plates </U>: we used minimal M9 media and added indigo dissolved in DMSO, until blue color was visible in the plates. Candidate strains were then inoculated on M9-indigo plates and left to grow at 30°C. </P><P><U>Liquid M9 with indigo </U>: 700µL of indigo in DMSO at 50mM was first added to 500mL Minimal M9 media. We put 5mL M9-indigo in 50 mL Falcon tubes, then put 20µL 50mM indigo in DMSO to have a deep blue color. Colonies from LB plates were used for inoculation. Tubes were left in a shaking incubator at 30°C for one week. Indigo Consumption was quantified using absorbance. Indigo strongest absorbance was determined at 680nm. Measurements were made every 24 hours for each strains for 6 days: 500µL media from previous 50mL falcon tubes were collected, vortexed, and 200µL were used in 96 well-plates. Absorbance was also measured at 450nm, for which indigo has a weak absorbance, to measure absorbance of cells and other things that can interfere with the absorbance at 680nm. Absorbance measurements were performed using a TECAN plate reader. </P><P><U> Stained cotton </U>: small pieces of cotton were stained with around 100µL 50mM indigo dissolved in DMSO (sometimes more indigo was needed to completely stain the cotton). The cotton was washed at high temperatures, then sterilized with an ethanol wash. These pieces are then placed on M9 plates and liquid culture to assess indigo consumption on cotton. </P><H3> Laccases </H3><P>High insolubility of indigo in water (990 µg/L) was causing majority of our indigo degradation experiments to fail due to either precipitation of the indigo in negative controls or undetectably low concentration of the dye. Eventually, the indigo solubility in our buffer was increased to a detectable level by adding 0.5% Tween 80.

To insure fully copper-loaded enzyme needed for our degradation experiments, cells were grown for 20h in LB with 0.1 mM IPTG, and 0.25 mM CuCl<SUB>2</SUB>, at 25<SUP>o</SUP>C without shaking (Reiss et al. 2011). Later, the cells were harvested by centrifugation and the protein extraction was performed by B-PER, protein extraction reagent from Thermo Fisher. The cell extracts were stored in 0.5mL aliquots at -80<SUP>o</SUP>C.
</P><H2 class="red"> Attributions </H2><DIV style="text-align:center;"><P>This project was done mostly by Mislav and Elisa. Special thank to Jake for sacrifying his jeans for science. </P></DIV><H2 class="red"> References </H2><UL><LI>Agarwal, S. (2009). World Denim Market Production and Consuption Report 2012, Denimsandjeans.com.</LI><LI>Couto S. R. and Toca-Herrera J. L. (2006). Lacasses in the textile industry.  Biotechnology and Molecular Biology Review, 1(4), 115-20</LI><LI>Conceição, V., Freire, F. B., &amp; Carvalho, K. Q. de. (2013). Treatment of textile effluent containing indigo blue dye by a UASB reactor coupled with pottery clay adsorption. Acta Scientiarum. Technology, 35(1), 1–6</LI><LI> Campos, R., Kandelbauer, A., Robra, K. H., Cavaco-Paulo, A., &amp; Gübitz, G. M. (2001). Indigo degradation with purified laccases from Trametes hirsuta and Sclerotium rolfsii. Journal of Biotechnology, 89(2-3), 131–139.</LI><LI> Dubé, E., Shareck, F., Hurtubise, Y. et al. Appl Microbiol Biotechnol (2008) 79: 597. Homologous cloning, expression and characterization of a laccase from <I> Streptomyces coelicolor </I> and enzymatic decolourisation of an indigo dye.</LI><LI> J Margot, C Bennati-Granier, J Maillard, P Blánquez, D.A Barry, C Holliger (2013). Bacterial <I>versus</I> fungal laccase : potential for micropollutant degradation. ABM Express ; 3 :63.</LI><LI>Reiss R., Ihssen J., Thöny-Meyer L. (2011). Bacillus pumilus laccase: a heat stable enzyme with a wide substrate spectrum. BMC Biotechnology 11:9</LI><LI> Woodhead Publishing Series in Textile: Number 164; Denim, Manufacture, Finishing and Applications; edited by R. Paul </LI></UL></DIV><DIV style="margin-top:20px; margin-bottom:20px"><DIV class="panel"><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Assay" title="Assay"><DIV id="pspanel" class="subpanel2" onmouseover="chgtrans(this)"><DIV class="titlebox">
	    Assay
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Microbiology" title="Microbiology"><DIV id="dspanel" class="subpanel2"><DIV class="titlebox">
	    Microbiology
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Enzyme" title="Enzyme"><DIV id="tcpanel" class="subpanel2"><DIV class="titlebox">
	  Enzyme
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Binding" title="Binding domains"><DIV id="thpanel" class="subpanel2"><DIV class="titlebox">
	    Binding
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Indigo" title="Indigo"><DIV id="thpanel" class="subpanel2"><DIV class="titlebox">
	    Indigo
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Model" title="Model"><DIV id="thpanel" class="subpanel2"><DIV class="titlebox">
	    Model
	  </DIV></DIV></A></DIV></DIV></DIV><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:igem2016parisbettencourt@gmail.com">igem2016parisbettencourt@gmail.com</A><CENTER><A href="https://2016.igem.org/">
    2016.igem.org
</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>