<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Project_Microbiology skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Project/Microbiology</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navmenubkg"><DIV style="margin-left:-15px" id="navmenubox"><UL id="nav"><LI><A style="margin-left:10px" href="https://2016.igem.org/Team:Paris_Bettencourt/Description">Project</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Description">Overview</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Assay">Assay</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Microbiology">Microbiology</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Enzyme">Enzyme</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Binding">Binding</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Indigo">Indigo</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Model">Model</A></LI></UL><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/About_Perc">Practices</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/About_Perc">About Perc</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Integrated_Practices">Dry Cleaners</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Design">Design</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Human_Practices">Events</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Safety">Safety </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Safety">Lab Safety</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Product_Safety" style="margin-top:-10px; margin-bottom:30px;">ProductSafety</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Achievements">Achievements</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Interlab_Study">Interlab Study</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Medal_Requirements">MedalRequirements</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI style="margin-left:10px"><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Daily Note</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Protocols">Protocols</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Bibliography">Bibliography</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Team">Team </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Team">Our Team </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Sponsors">Sponsors</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><DIV class="projtile1"><H2 class="red" style="text-align:center;">Goals</H2><UL><LI>To screen soil samples from around the world for microbes that naturally degrade anthocyanin.</LI><LI>To identify enzymes linked to the most efficient anthocyanin eaters.</LI></UL></DIV><DIV class="projtile3"><H2 class="red" style="text-align:center;">Results</H2><UL><LI>We collected species from all around the world through iGEM collaborations.
            </LI><LI>186 bacteria were tested for degradation of quercetin.
            </LI><LI>174 bacteria were identified by 16s rRNA sequencing.
            </LI><LI>4 promising species were selected for whole genome sequencing.
            </LI><LI>Candidate enzymes were identified from genome sequence, and passed to the enzyme group.</LI></UL></DIV><DIV class="projtile4"><H2 class="red" style="text-align:center;">Methods</H2><UL><LI>Microbe cultivation
        </LI><LI>Anthocyanin purification
        </LI><LI>Anthocyanin &amp; quercetin measurement
        </LI><LI>16S rRNA sequencing
        </LI><LI>Whole genome sequencing
        </LI><LI>Phylogenetic analysis
    </LI></UL></DIV><DIV id="subheader"><DIV id="input"><H2 class="red">Abstract</H2><P><B>Anthocyanins, the key pigments found in red wine, are abundant in grapes, berries, flowers and many plants. Like all naturally ocurring metabolites, they eventually biodegrade and re-enter the carbon cycle. In this project, we search nature for enzymatic pathways that can break down anthocyanins into simpler, unpigmented molecules. We reasoned that microbes living in the soil near vineyards were likely to catabolize and consume anthocyanin. Therefore we collected soil samples from 10 vinyards around France, Europe and the world, notably with the help of our fellow iGEM teams. In total we isolated 186 strains through selective and non-selective plating on different media. All of the strains were identified by 16s rRNA sequencing, then characterized for their ability to degrade quercetin, a compound structurally similar to anthocyanin. By phylogenetic analysis, we were able to connect quercetin degradation to specific bacterial phyla and genus including <I>Micrococcus</I>, <I>Pseudomonas</I>, <I>Lysinibacillus</I> and <I>Oerskovia</I>. The most effective strains were further characterized by whole genome sequencing to identify enzymes linked to natural quercetin degradation. By bioprospecting with the help of the worldwide iGEM community, we were able to find the best stain fighting enzymes that nature has to offer.</B></P><H2 class="red"> Motivation and Background</H2><H3>Bioprospecting and Bioremediation</H3><P>
Bioprospecting is the process of searching nature for genetic information that can be adapted in useful or profitable ways. In recent years, bioprospecting efforts have focused on the search for small molecule pharmaceuticals and other bioactive compounds (Müller, 2016). Bioremediation is the use of living organisms to remove environmental toxins from contaminated areas. Microbes in particular are well known for their ability to degrade organic pollutants like petroleum, pesticides and phenolic compounds. Bioremediation has always been a popular topic in iGEM, producing many notable projects with diverse organisms and applications.</P><P>Anthocyanin is not harmful, but in the context of a stain it is unwanted and so could be considered a target for bioremediation. Therefore a mechanism to degrade anthocyanin could be revealed with a classic bioremediation strategy :
<OL><LI>Select organisms from a contaminated environment, where enzymatic decontamination may have naturally evolved.</LI><LI>Isolate pure strains and measure their activity.</LI><LI>Connect the activity to specific pathways using molecular genetics.</LI></OL></P><H3>Anthocyanidin and quercetin</H3><P>
In these experiments, we use quercetin as a chemical proxy for anthocyanins. Naturally occurring anthocyanidins are chemically diverse derivatives of a a core flavylium cation. Plant sources of anthocyanidin carry a range of anthocyanidin pigments substituted at any of up to seven positions, with the relative concentrations contributing to a characteristic color.
Pure anthocyanidins, like malvidin, are expensive (120 EUR for 1 mg) and do not necessarily represent the full chemical diversity of a natural wine stain. Therefore, in this work we use quercetin, a flavonol, as a structural proxy. Quercetin is cheap (40 EUR for 10 g), stable, and can be quantified by absorbance at 315 nm. In follow up experiments, microbes are tested on real wine and on bulk anthocyanins that we extract directly from grape skins.
</P><DIV id="figurebox"><DIV style="text-align:center;"><P><B>Figure 1</B> Structure and absorbance of malvidin, the most abundant anthocyanidin in wine, and quercetin, a flavonol.</P><P>
All flavonoids are structured as two phenyl rings and a heterocyclic ring. Anthocyanin itself is structured as a chromane ring with an aromatic ring on C2. Cyanindin and malvidin comprise 90% of the anthocyanins found in nature. These chemicals differ only in their cyclic B groups, and the chromane ring is well conserved in most flavonoids. Therefore, we theorized that the chromane ring itself presented an ideal target for degradation.
 Based on these criteria, we chose the flavanol quercetin as our anthocyanin substitute. This molecule differs from anthocyanins only in the presence of a carbonyl group. Additionally, quercetin is present in wine, and contributes to its color. Thus, even in the case where enzymes are isolated that break down quercetin and not anthocyanin, the possibility exists of reducing the color or intensity of wine stains.
Finally, co-pigmentation chemical interactions occur between anthocyanin and quercetin, increasing wine color stability, mainly through π-π stacking between their phenolic cycles. Thus, it leads to the possibility that quercetin degradation could also impact anthocyanin stability.
</P></DIV></DIV><H2 class="red">Results</H2><H3>Anthocyanin Extraction and analysis</H3><P>Anthocyanins were extracted from <I>Vinis vitifera</I> fruits. It skin was separated from the rest of the fruit and macerated overnight in an ethanol solution with 1% chloridric acid. After maceration, the solution was passed through with a paper filter to eliminate solid material and evaporated at 37°C at 150 rpm. We confirmed the presence of anthocyanin with HPLC and, colour variation with pH.</P><H3>Collection of the soil samples</H3><P>Soil samples were collected from France, Spain, Croatia, Namibia and Australia. Samples from the Paris region were collected by members of our team. Other samples were sent by friends, family members, and collaborating iGEM teams. Soil samples were declared to French customs authorities with a <A href="https://static.igem.org/mediawiki/2016/b/b7/Paris_Bettencourt_Facture_Proforma.pdf">Facture Proforma</A>, printed out by the sample donor and included in the shipment.

Upon arrival, samples were washed gently with phosphate-buffered saline (PBS) solution, then left to stand, allowing large particles to settle. The resulting eluate was diluted further with PBS then used directly as a source of soil microbes.
</P><DIV id="figurebox"><TABLE border="1"><TBODY><TR><TH>Country</TH><TH>Location</TH><TH>Collector</TH></TR><TR><TD>France</TD><TD>Clos Monmartre Vineyard, Paris</TD><TD>Our Team</TD></TR><TR><TD>France</TD><TD>Cochin Port Royal, Paris</TD><TD>Our team</TD></TR><TR><TD>France</TD><TD>Vaucluse region’s vineyard</TD><TD>INSA-Lyon iGEM team</TD></TR><TR><TD>Spain</TD><TD>Barcelona</TD><TD>UPF-CRG Barcelona iGEM team</TD></TR><TR><TD>Spain</TD><TD>Utiel Requena</TD><TD>UPV Valencia iGEM team</TD></TR><TR><TD>Australia</TD><TD>Hunter Valley</TD><TD>UNSW</TD></TR><TR><TD>Australia</TD><TD>Sydney</TD><TD>Macquarie 2016 iGEM team</TD></TR><TR><TD>Namibia</TD><TD>Etosha National Park</TD><TD>Our team</TD></TR><TR><TD>Algeria</TD><TD>Alger</TD><TD>Our team</TD></TR><TR><TD>Croatia</TD><TD>Kricke</TD><TD>Our team</TD></TR><TR><TD>Israel</TD><TD>Jerusalem</TD><TD>Our team</TD></TR></TBODY></TABLE><DIV style="float:right; margin-bottom:10px;margin-top: -500px; "><P><B>Figure 2</B> Location of soil samplescollected or obtained by the team.
</P></DIV></DIV><H3>Preparation of the microbe library</H3><P>
For safety and to avoid environmental contamination, microbial isolation was performed in a fume hood in a BSL 2 facility. More safety information in provided on our safety page.

Microbes were isolated from soil samples using either selective or nonselective plating. For the selective plating, M9 agar was supplemented with either quercetin alone or quercetin with glucose to enrich for microbes with the ability to metabolize quercetin.

The resulting culture was incubated at 30 C for 48 hours, then re-streaked to eliminate potential contamination.

To maximize library diversity, we preferentially chose colonies with unique morphologies. After isolation, we performed colony PCR with universal 16s rRNA primers (see methods). Sequencing the resulting PCR products allowed us to identify the strains and position them within the greater bacterial taxonomy.

</P><H3>Quercetin degradation assay</H3><P>
Each of the 189 isolated strains was innoculated into M9 quercitin medium in triplicate and incubated at 37 C for 6 days. At the end of the period, quercitin concentration was measured by absorbance.
 

Of 186 strains tested, 50 produced quercetin levels significantly lower than controls (Figures 3 and 4). 20 strains degraded more than 50% of quercetin and 2 strains degraded more than 80%.
Both selective and nonselective plating methods were able to produce quercetin-degrading strains. 4 of the 5 strains showing the most quercetin degradation were obtained by selective plating. However, 40 of the 50 strains showing significant degradation were obtained by nonselective plating.
 </P><DIV id="figurebox"><DIV style="text-align:center;"><P><B> Figure 3</B> Quercetin degradation by 186 microbes collected from global soil samples. Single colonies were inoculated in M9 minimal medium and grown for 6 days. Quercetin was measured by absorbance. Strains to the extreme left of the figure represent the highest-degrading strains. Red bars represent strains isolated from selective media and blue from non selective media.
</P></DIV></DIV><DIV id="figurebox"><P style="text-align: left;"><B> Figure 4</B> Quercetin degradation detail. The top-performing strains included those isolated from both selective and nonselective media. Strains marked with an asterisk were selected for further investigation. Red bars represent strains isolated from selective media and blue from non selective media.
        </P></DIV><P>
Working with M9 quercetin plates can be challenging. The bright green color of the plates makes colony visualization difficult. Indeed, only fungal mycelia were visible due to a white halo resulting from quercetin degradation. However, working with liquid media allows the control of residual sample carbon source contamination through sample dilution. Additionally, we were concerned agar in plates could be used as a carbon source.
8 top-performing strains were selected for detailed valication in a time-course assay, and all were able to degrade quercitin significantly better than E. coli negative controls. We observed a 50% degradation over 6 days in the best case.</P><DIV id="figurebox"><DIV style="text-align:center;"><B> Figure 5</B> Kinetics of quercetin degradation by 8 promising strains. This experiment had two negative controls, one with no bacteria (black line) and one with non-quercetin degrading <I>E. coli</I>. <I>Pseudomonas putita</I> was included as a positive control. Four strains, shown in bold, degraded quercetin at a higher rate than our <I>P. putida</I> control.
        </DIV></DIV><H3>Phylogenetic analysis of quercetin degradation</H3><P>
Using the 16s rRNA sequences, we constructed a global phylogeny of all the assayed strains (Figure 6). Quercitin-degrading activity was concentrated among the Pseudomonas genus, where most of the strains were positive. However many diverse species were able to significantly degrade quercitin, suggesting that the phenotype may be highly variable in evolution.</P><DIV id="figurebox"><P><B> Figure 6: Phylogenetic Tree of isolated strains. </B>
We constructed a phylogenetic tree of all isolated bacterial strains. Strain taxonomic classification is indicated by the color key to the left of the figure. Strains that demonstrated high quercetin degradation are marked with an asterisk. Those strains marked with a large star were selected for whole genome sequencing to look for common anthocyanin-degrading genes.

</P></DIV><H3>Mining genomes for quercetin-degrading enzymes</H3><P>

Four of our top-degrading strains, representing diverse phylogentic lineages, were selected for whole genome sequencing. As of the wiki freeze, the genome sequences were not complete. So, as a proxy, we obtained complete genome sequences of their nearest relatives in GenBank. BLAST searches against the genomes revealed the presence of several candidate quercitin-degrading enzymes, including laccase and XylE.
 
The results of these genomic analysis were passed to the enzyme group, where these enzymes form the basis of our synthetic stain fighting product.
</P><DIV id="figurebox"><P><B> Figure 7: Genome table.</B> Genome sequences were analyzed for the presence of potential anthocyanin degrading enzymes as identified in the Enzymes Project. The genes of at least one of the six candidates were identified for each of the sequenced strains. The number of + signs indicated represent the number of gene copies in the bacterial genome.
</P></DIV><H2 class="”red”">Methods</H2><H3>Anthocyanin extraction</H3><P><I>Vinis vitifera</I> grapes were our source for anthocyanin extraction.  Grapes were peeled and the skins were collected, washed and soaked overnight in ethanol with 1% HCl. By trial and error, we determined that 2.5 mL of this solution per 1 g of skins was the best compromise between efficiency and the quantity of solvent used. The solution was filtered with Whatman paper, with the filtrate collected, and the solvent evaporated at 37°C for several hours. The dry extract was resuspended in water (10 mL for 1g of grape skin).</P><H3>Anthocyanin quantification by differential absorbance</H3><P>
Following Lee et al. (2005), we prepared one buffer at pH 1 (0.025M potassium chloride) and a second at pH 4.5 (sodium acetate, 0.4M).
100 µL of the anthocyanin solution was mixed with 900µL of each buffers and the color was allowed to develop over 20 minutes. Absorbance measurements were obtained at 510 nm and 700 nm for each solution. Anthocyanin concentration was determined as a function of the four absorbance measurements, using an established formula (Lee et al., 2005).</P><H3>Protein quantification with Bradford assays</H3><P>
A stock solution of Bovine Serum Albumin (BSA) was prepared in water at 1 mg/mL. 100 μL of standard dilutions of BSA solution were mixed with 1 mL of Bradford Reagent and mixed by vortexing. Absorbance was measured at 595 nm. Experimental samples were treated similarly and compared to the BSA standard curve to determine concentration.</P><H3>Carbohydrate quantification with Fehling Reaction</H3><P>
200µL of Fehling's A solution, 200µL of Fehling's solution B and 200µL of our carbohydrate solution into sodium acetate buffer (20µL of solution and 180 µL of buffer). The Fehling reaction is measured as the loss of absorbance at 650nm relative to a blank solution without carbohydrate. Quantification was achieved by comparison to a standard curve of glucose prepared at 1g/L to 5g/L.
</P><H3>Bacteria plating on selective and non-selective media</H3><P>1 g of soil samples were suspended in 5 mL Phosphate Buffered Saline (PBS) then left to stand allowing large particles to settle. The soil suspension was serially diluted to obtain a suitable density of microbes (typically 1:1000) then 200 µL was plated on standard Petri dishes with M9 agar with 1 g/L quercetin for selection. Non-selective plating was performed on a range of rich media including FTO agar (Curry, 1976), Mossel agar (Mossel, 1967), standard LB, standard TSA and standard M9 glucose.</P><H3>Protocol for growth assay in Quercetin M9 liquid media</H3><P>Following the protocol of Dantas <I>et al.</I> 2012, all step were performed in liquid media to control soil carbon source contamination. We suspended soil samples in 5 ml M9 with 1g/L quercetin at pH 7 in 50 ml Falcon tubes with 500µL of overnight culture of strains isolated from selective or non-selective plates.
All cultures were made in triplicate at 30°C with shaking at 150 rpm for several days. As quercetin is not soluble at pH=7, shaking important to avoid precipitation.</P><H3> Quercetin absorbance measurement </H3><P>Quercetin absorbance was measured at two time points for histogram construction: at 0 days to ensure quercetin sample concentration consistent with the controls, and at 6 days to evaluate quercetin degradation. M9-quercetin and <I>Pseudomonas putida</I>K2440 samples were included as negative and positive controls, respectively.
Prior to absorbance measurement, Quercetin was solubilized by diluting samples 10 fold in 0.5M NaOH, centrifuged to remove cell material, and further diluted 100X for measurement at 315 nm in a Tecan plate reader.</P><H3>PCR for 16s characterization, sequencing interpretation and phylogenetic tree construction.</H3><P>To identify bacterial strains, 16S rRNA sequences were amplified through colony PCR, column purified, and Sanger sequenced by GATC. The resulting sequences were submitted for BLAST comparison at ncbi.gov. Alignments were performed using the Ribosomal Database Project Aligner tool (https://rdp.cme.msu.edu/), and a phylogenetic tree was constructed using Geneious software with the following parameters: we used Neighbor-Joining tree building with Jukes Cantor as the genetic distance model, with a 93% similarity cost matrix for the alignment with free end gaps. The tree was then exported and improved using the online Tree of Life software (http://www.tolweb.org/tree/).</P><H3>PCR for genome sequencing.</H3><P>We isolated bacterial DNA using the DNeasy Blood and Tissue Kit from Qiagen. We submitted four strains to GATC for whole genome sequencing: NS.4 (<I>Lysinibacillus</I>), S.48 (<I>Stenothrophomonas maltophilia</I>), S.33 (<I>Oerskovia Paurometabola</I>), NS.33 (<I>Microccocus Luteus</I>) according to their sample preparation specifications.</P><H2 class="”red”">Attributions</H2><P>This project was done by Antoine Villa Antoine Poirot and Sébastien Gaultier. Anthocyanin data was obtained by Ibrahim Haouchine.  Thanks to our advisors Jake and Jason for all their help with the figures. We would like to thank Philippe Morand from the microbiology lab of Cochin for his advice.</P><H2 class="”red”">References</H2><UL><LI>Kanekar, P. P., Sarnaik, S. S., &amp; Kelkar, A. S. (1998). Bioremediation of phenol by alkaliphilic bacteria isolated from alkaline lake of Lonar, India. <I>Journal of applied microbiology</I>, 85(S1).</LI><LI>Dantas, G., Sommer, M. O., Oluwasegun, R. D., &amp; Church, G. M. (2008). Bacteria subsisting on antibiotics. <I>Science</I>, 320(5872), 100-103.</LI><LI>Lee, J., Durst, R. W., &amp; Wrolstad, R. E. (2005). Determination of total monomeric anthocyanin pigment content of fruit juices, beverages, natural colorants, and wines by the pH differential method: collaborative study. <I>Journal of AOAC international</I>, 88(5), 1269-1278.</LI><LI>Curry, J. C., &amp; Borovian, G. E. (1976). Selective medium for distinguishing micrococci from staphylococci in the clinical laboratory. <I>Journal of clinical microbiology</I>, 4(5), 455.</LI><LI>Pillai, B. V., &amp; Swarup, S. (2002). Elucidation of the flavonoid catabolism pathway in Pseudomonas putida PML2 by comparative metabolic profiling. <I>Applied and environmental microbiology</I>, 68(1), 143-151.</LI><LI>Herrmann, H., Janke, D., Krejsa, S., &amp; Kunze, I. (1987). Involvement of the plasmid pPGH1 in the phenol degradation of Pseudomonas putida strain H. <I>FEMS microbiology letters</I>, 43(2), 133-137.</LI></UL><DIV style="margin-top:20px; margin-bottom:20px"><DIV class="panel"><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Assay" title="Assay"><DIV id="pspanel" class="subpanel2" onmouseover="chgtrans(this)"><DIV class="titlebox">
	    Assay
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Microbiology" title="Microbiology"><DIV id="dspanel" class="subpanel2"><DIV class="titlebox">
	    Microbiology
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Enzyme" title="Enzyme"><DIV id="tcpanel" class="subpanel2"><DIV class="titlebox">
	  Enzyme
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Binding" title="Binding domains"><DIV id="thpanel" class="subpanel2"><DIV class="titlebox">
	    Binding
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Indigo" title="Indigo"><DIV id="thpanel" class="subpanel2"><DIV class="titlebox">
	    Indigo
	  </DIV></DIV></A><A href="https://2016.igem.org/Team:Paris_Bettencourt/Model" title="Model"><DIV id="thpanel" class="subpanel2"><DIV class="titlebox">
	    Model
	  </DIV></DIV></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:igem2016parisbettencourt@gmail.com">igem2016parisbettencourt@gmail.com</A><CENTER><A href="https://2016.igem.org/">
    2016.igem.org
</A></CENTER></DIV></DIV></DIV></BODY></HTML>