<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Notebook_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Notebook/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navmenubkg"><DIV style="margin-left:-15px" id="navmenubox"><UL id="nav"><LI><A style="margin-left:10px" href="https://2016.igem.org/Team:Paris_Bettencourt/Description">Project</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Description">Overview</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Assay">Assay</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Microbiology">Microbiology</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Enzyme">Enzyme</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Binding">Binding</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Indigo">Indigo</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Model">Model</A></LI></UL><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/About_Perc">Practices</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/About_Perc">About Perc</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Integrated_Practices">Dry Cleaners</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Design">Design</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Human_Practices">Events</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Safety">Safety </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Safety">Lab Safety</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Product_Safety" style="margin-top:-10px; margin-bottom:30px;">ProductSafety</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Achievements">Achievements</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Interlab_Study">Interlab Study</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Medal_Requirements">MedalRequirements</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI style="margin-left:10px"><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Daily Note</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Protocols">Protocols</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Bibliography">Bibliography</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Team">Team </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Team">Our Team </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Sponsors">Sponsors</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><P><TITLE>jQuery UI Accordion - Fill space</TITLE></P><DIV id="subheader"><TABLE><TBODY><TR><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Notebook</A></TD><TD><A href="#ancre">Protocols</A></TD><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Bibliography">Bibliography</A></TD></TR></TBODY></TABLE></DIV><DIV id="ancre"><DIV class="input"><DIV id="accordion"><H2 class="red">16s rRNA PCR protocols for bacterial identification</H2><DIV><H4> 16s rRNA PCR protocol</H4><OL><LI>
                                Pick a single colony freshly struck  into 10 μL of sterile RNAse and DNAse free water (purchased from Gifco®), in a PCR tube.
Fresh colonies (grown that day) work best, but they can also come from 4°C.
                            </LI><LI>
                             Incubate at 95°C for 10 minutes.
	2 μl can then be used directly for PCR but the solution can also be stored at 4°C for a few days.

The NEB Quick-Load® Taq 2X Master Mix was used in the following master mix (notice that the 6X loading dye is already add to the Dream Taq used in this protocol):
 
The following PCR protocol was followed : The extent temperature was chosen according to the PCR product length.
 
The following primers were purchased from IDT, stocked at -20°C at a 100 μM concentration. </LI></OL><H4>Gel Electrophoresis using SYBR safe</H4><UL><LI>
Preparation of the Gel (50mL)
</LI><LI>
Mix 0.5g of agarose with 50mL of a 0.5X TBE solution in an Erlenmeyer.</LI><LI>Heat the solution in a microwave until the liquid becomes totally transparent, avoid boiling. </LI><LI>Let it rest a few minutes so that you can manipulate the Erlenmeyer in your hand without pain. </LI><LI>Add 5μL of SYBR® Safe (10000X) and mix gently until the solution homogenizes. </LI><LI>Pour the liquid in a gel mold with the desired number of wells and wait 20-30 min (or until the gel is solidified).</LI><LI>Put the gel inside the electrophoresis machine and pour TBE 0.5X until the gel is totally submerged.</LI><LI>Preparation of the samples
</LI><LI> Pipette 5 μL of each PCR product inside a different well.</LI><LI>Pipette 5 μL of a 1kb ladder 'purchased to Thermofisher) in one of the wells (better if it flanks the PCR products when there is many of them).</LI><LI>Close the machine and launch the migration with a 100V voltage and wait approximately 20 minutes.
</LI><LI> When the migration is finished, turn off the machine, remove the lid and put the gel inside the gel visualizer.</LI>
It is here important to check two things:
<OL><LI> According to the ladder, the bands should present a weigh of 1,4Kb.</LI><LI> No band should appear in the negative control, so that no contamination during the PCR preparation could interfere with the sample's bacterial identification.</LI></OL></UL><H4>PCR purification</H4>
The PCR purifcation was performed using the QIAquick® PCR Purification Kit (Qiagen) :
<OL><LI>Add 5 volumes of binding buffer to 1 volume of PCR product in a sterile 1.5mL microcentrifuge tube, mix by pipetting up and down. </LI><LI>Transfer in a centrifugation column and centrifuge 1 min at 14000 rpm. </LI><LI>Throw the filtration product and add 750μL of washing solution (make sure ethanol was added before according Qiagen protocol). </LI><LI>Wait 5 minutes, then centrifuge 1 min at 14000 rpm. </LI><LI>Throw the filtration product and centrifuge again 1 min at 14000 rpm. </LI><LI>Put the column in a sterile 1.5mL microcentrifuge tube.</LI><LI>Add directly to the center of the column 35 μL of the elution buffer.</LI><LI>Wait 1 minute and centrifuge 1 min at 13000 rpm.</LI><LI>Discard the column and stock the solution at 4°C before quantification.
</LI></OL><H4>DNA quantification</H4>
DNA quantification was performed with the Thermo Science Nanodrop 2000 and enabled to check if our PCR purification product fit with GATC samples requirements.
Notably, the PCR product concentration needs to be between 20 and 80 ng/μl.
Per sample, two 1,5 mL microcentrifuge tubes will be necessary. In both, add 5μL of the purified DNA at the proper concentration ( For instance, a dilution with sterile water is thus possible if the concentration is higher than 80 ng/μL).
Add 5μL of the forward primer in one of the tube, and the reverse in the other tube (we used a concentration of 10 uM).<H4>Interpretation of data</H4>
The purpose is here to generate a consensus sequence for each sample, based on the alignement of the two chromatogram (AB1 file) obtained through GATC. This was performed with the Geneious De Novo Assembly tool. 
Then, the Targeted Loci BLAST tool on NCBI enables to identify the strain.
<H4>Further investigations</H4>
Whole Genomic Sequencing was performed using Illumina® Technology at GATC®, for 5 strains (4 gram negative and one gram positive).
Genomic DNA extraction was performed with the Qiagen® DNeasy Blood &amp; Tissue Kit.
The followed protocol was indicated in Qiagen® handbook. Notice that genomic extraction from gram positive bacteria needs additional lysis steps.
</DIV><H2 class="red"> Fungal genomic DNA extraction and 18S PCR </H2><DIV><H4> Genomic DNA extraction </H4><P><I> This protocol was inspired by a protocol called “Isolation of genomic DNA from fungi (culture and blood) using the QIAamp DNA Mini Kit” that can be found under the section user-developed protocol  <A href="https://www.qiagen.com/us/resources/resourcedetail?id=8ce3e789-478c-47d7-a8ed-2a00731fb085&amp;lang=en"> here </A></I></P><P> In this protocol we use lyticase, thus before doing our dna extraction we prepared lyticase aliquots:
	<OL><LI> Prepare XmL of TE. </LI><LI> Resuspend lyticase at a 10U/µL concentration. </LI><LI> Make 20µL aliquots (=200U). </LI><LI><STRONG> Store in a freezer at -20°C </STRONG> (or the lyticase will loose efficiency) </LI></OL></P><P class="input">DNA is isolated from fungi grown 48h to 72h on Sabouraud medium at 30°C, so 3 days before going through the dna extraction we inoculated our fungal strains on Sabouraud. As for genomic dna isolation:
<OL><LI>Harvest colonies using an inoculating loop and resuspend them in 1 mL of 0.9% saline solution in 1,5 mL tubes, centrifuge and collect the cell pellets (we did by pipetting the supernatant). </LI><LI> Prepare a 50mM Tris, 10mM EDTA and 28mM β-mercaptoethanol solution. Add the lyticase to a 10U/mL concentration.</LI><LI> Add 500 µL lyticase solution to the cell pellets and incubate for 30 minutes at 37°C. </LI><LI> Centrifuge at full speed for 10 min then discard the supernatant. Resuspend the pellets in 180µL ATL buffer and 20 µL Proteinase K, and incubate at 55°C in water bath for 15 minutes. </LI><LI> Add 200 µL buffer AL, mix by pulse vortexing for 15s, and incubate at 70°C for 10 min. </LI><LI> Add 200 µL ethanol (96 - 100%), mix by pulse vortexing for 15s and briefly centrifuge to remove drops. Then apply to the QIAamp Mini spin column in 2 mL collection tube and centrifuge at 6000 x g (8000 rpm) for 1 min. </LI><LI> Place the spin column in a clean 2 mL collection tube, add 500 µL buffer AW1 to spin column (carefully) and centrifuge at 8000 rpm for 1 m. </LI><LI> Place the spin column in clean 2 mL collection tube again, add 500 µL buffer AW2. Centrifuge at full speed (14 000 rpm) for 3 min. </LI><LI> Place spin columns in a new collection tube and centrifuge at full speed for 1 min. After that, place spin columns in a 1.5 mL tube, and add 50 µL distilled water. Incubate at room temperature for 1 min, then centrifuge at 8000 rpm for 1 min. </LI></OL>
The isolated genomic dna can then be used directly for 18S PCR. </P><H4> 18S PCR </H4>

The NEB Quick-Load® Taq 2X Master Mix was used in the following master mix :
 
The following protocol was followed : The extent temperature was chosen according to the PCR product length.
 
The following primers were purchased from IDT, stocked at -20°C at a 100 μM concentration. <CENTER><P style="text-align:center"> ITS1 5'-TCC TCC GCT TAT TGA TAT GC-3'</P><P style="text-align:center"> ITS4 5'-TCC GTA GGT GAA CCT GCG G-3' </P></CENTER></DIV><H2 class="red">Phage Display Protocols</H2><DIV><P>
            All protocols displayed here are identical to the ones provided in the NEB Ph.D.™-7 Phage Display Peptide Libraries Instruction Manual, except for the Panning Procedure that was adapted to our needs.
    
        </P><H4>Phage Titering</H4>
   
                The number of plaques will increase linearly with added phage only when the multiplicity of infection (MOI) is much less than 1 (i.e., cells are in considerable excess). For this reason, it is recommended that phage stocks be titered by diluting prior to infection, rather than by diluting cells infected at a high MOI. Plating at low MOI will also ensure that each plaque contains only one DNA sequence.
            
            <OL><LI>
                    Inoculate 10mL of LB with ER2738 from a plate and incubate with shaking ~3-4 hours (mid-log phase, OD600 ~ 0.5).
                </LI><LI>
                    While cells are growing, melt Top Agar in microwave and dispense 3 ml into sterile culture tubes, one per expected phage dilution. Maintain tubes at 45°C.
                </LI><LI>
                    Pre-warm, for at least one hour, one LB/IPTG/Xgal plate per expected dilu¬tion at 37°C until ready for use.
                </LI><LI>
                    Prepare 10 to 103 -fold serial dilutions of phage in LB; 1 ml final volumes are convenient. Suggested dilution ranges: for amplified phage culture supernatants, 10^8 – 10^11; for unamplified panning eluates, 10^1–10^4. Use aerosol-resistant pipette tips to prevent cross-contamination, and use a fresh pipette tip for each dilution.
                </LI><LI>
                    When the culture in Step 1 reaches mid-log phase, dispense 200 μl into microfuge tubes, one for each phage dilution.
                </LI><LI>
                    To carry out infection, add 10 μl of each phage dilution to each tube, vortex quickly, and incubate at room temperature for 1–5 minutes.
                </LI><LI>
                    Transfer the infected cells one infection at a time to culture tubes containing 45°C Top Agar. Vortex briefly and IMMEDIATELY pour culture onto a pre-warmed LB/IPTG/Xgal plate (200µM IPTG, 30µg/mL X-gal). Gently tilt and rotate plate to spread top agar evenly.
                </LI><LI>
                    Allow the plates to cool for 5 minutes, invert, and incubate overnight at 37°C.
                </LI><LI>
                    Count plaques on plates that have approximately 100 plaques. Multiply each number by the dilution factor for that plate to get phage titer in plaque forming units (pfu) per 10µL. Multiply by 100 to get the pfu/mL.
                </LI></OL><H4>Panning Procedure</H4>
    
 
                This protocol was adapted from the Surface Panning Procedure (Direct Target Coating) supplied in the NEB Ph.D.™-7 Phage Display Libraries Instruction Manual.

            <H5>DAY 0</H5><UL><LI>
                        Streak ER2738 from glycerol stock on LB+Tet plate.
                    </LI><LI>
                        Incubate overnight at 4°C the piece of fabric/thread with 1mL of blocking buffer. Alternatively, after inoculation of the titering culture incubate the piece of fabric/thread at least 1 hour at 4°C on DAY 1.
                    </LI></UL><H5>DAY 1</H5><OL><LI>
                        Inoculate 10 ml of LB+Tet medium with ER2738 in a 125mL Erlenmeyer flask. This culture will be used for titering in Step 8 and can be used in ~3.5 - 4 hours. Incubate at 37°C with vigorous shaking. Incubate the titering culture until needed.
                    </LI><LI>
                        Remove the piece of fabric/thread from the blocking solution. Wash the fabric rapidly 10 times with TBST 0.1% (TBS + 0.1% [v/v] Tween-20) by dipping the threads in a well from a six-well plate filled with ~8mL TBST per well and going back and forth through the washing solution. Alternatively beackers can be used in place of six well plate.
                    </LI><LI>
                        Dilute a 25-fold representation of the library (e.g., 2 x 10 11 phage for a library with 2 x 10 9 clones) with 250µl of TBST (therefore add 10µL of phage library). Place the piece of fabric/thread in the Eppendorf tube containing the library and rock gently for 60 minutes at room temperature.
                    </LI><LI>
                        Inoculate 20mL of LB medium in a 250mL Erlenmeyer flask just before the end of the incubation period of the fabric in the phage library solution. Incubate at 37°C with vigorous shaking. This culture will be used to amplify the eluted phages.
                    </LI><LI>
                        Remove piece of fabric/thread from the Eppendorf tube.
                    </LI><LI>
                        Wash fabric 20 times with TBST as in step 2.
                    </LI><LI>
                        Elute bound phage with 1 ml of an ap¬propriate elution buffer for the interaction being studied. A general buffer for nonspecific disruption of binding interactions is 0.2 M Glycine-HCl (pH 2.2), 1 mg/ml BSA. Rock gently for 15 minutes. Discard the piece of fabric. Neutralize supernatant with 150 μl of 1 M Tris-HCl, pH 9.1.
                    </LI><LI>
                        Titer a small amount (~2 μl) of the eluate as described in the Phage Titering protocol. Plaques from the first or second round eluate titering can be sequenced if desired.
                    </LI><LI>
                        Amplify the rest of the eluate by adding the eluate to the 20-ml ER2738 culture from Step 4 (should be early-log at this point, i.e. OD600 ~ 0.01-0.05) and incubating with vigorous shaking for 4.5 hours at 37°C.
                    </LI><EM>
                        Note: The remaining eluate can be stored overnight at 4°C at this point, if preferred, and amplified the next day. In this case, inoculate 10 ml of LB+Tet with ER2738 and incubate with shaking overnight at 37°C. The next day, dilute the overnight culture 1:100 in 20 ml of LB in a 250-ml Erlenmeyer flask <STRONG>(do not use a 50 ml conical tube)</STRONG> and add the unamplified eluate. Incubate with vigorous shaking for 4.5 hours at 37°C and proceed to Step 10.
                    </EM><LI>
                        Transfer the culture to a centrifuge tube and spin for 10 minutes at 12,000 g at 4°C. Transfer the supernatant to a fresh tube and re-spin (discard the pellet).
                    </LI><LI>
                        Transfer the upper 80% of the supernatant to a fresh tube and add to it 1/6 volume of 20% PEG/2.5 M NaCl. Allow the phage to precipitate at 4°C for at least 2 hours, preferably overnight.
                    </LI></OL><H5>DAY 2</H5><OL><LI>
                        Inoculate 10mL of LB+Tet medium with ER2738 for titration.
                    </LI><LI>
                        Spin the PEG precipitation at 12,000 g for 15 minutes at 4°C. Decant and discard the supernatant, re-spin the tube briefly, and remove residual supernatant with a pipette. The phage pellet should be a white finger print sized smear on the side of the tube.
                    </LI><LI>
                        Suspend the pellet in 1 ml of TBS. Transfer the suspension to a micro¬centrifuge tube and spin at maximum (14,000 rpm) for 5 minutes at 4°C to pellet residual cells.
                    </LI><LI>
                        Transfer the supernatant to a fresh microcentrifuge tube and reprecipitate by adding 1/6 volume of 20% PEG/2.5 M NaCl. Incubate on ice for 15–60 minutes. Microcentrifuge at 14,000 rpm for 10 minutes at 4°C, discard the supernatant, re-spin briefly, and remove residual supernatant with a micropipet.
                    </LI><LI>
                        Suspend the pellet in 200 μl of TBS. Microcentrifuge for 1 minute to pel¬let any remaining insoluble material. Transfer the supernatant to a fresh tube. This is the amplified eluate.
                    </LI><LI>
                        Titer the amplified eluate as described in the Phage Titering protocol. The eluate can be stored for up to 3 weeks at 4°C. For long-term storage, add an equal volume of sterile glycerol and store at –20° C.
                    </LI><LI>                       Incubate overnight at 4°C a new piece of fabric/thread in 1mL of blocking buffer for the 2nd round of panning.
                    </LI><LI>
                        Inoculate 10 ml of LB+Tet with ER2738 and incubate with shaking overnight at 37°C.
                    </LI></OL><H5>DAY 3</H5><OL><LI>
                        Count blue plaques from the titering plates in Step 6 of DAY 2 and determine the phage titer, which should be on the order of 1013–14 pfu/ml. Use this value to calculate an input volume corresponding to the input titer in Step 3 of DAY 1. If the phage titer of the amplified eluate is too low, succeeding rounds of panning can be carried out with as little as 109 pfu of input phage.
                    </LI><LI>
                        Carry out a second round of panning: repeat DAY 1 using the calculated amount of the first round amplified eluate as input phage, and raising the Tween concentration in the wash steps to 0.5% (v/v).
                    </LI></OL><H5>DAY 4</H5><OL><LI>
                        Repeat DAY 2 with the panning eluate from the 2nd round.
                    </LI><LI>
                        Incubate overnight at 4°C a new piece of fabric/thread with 1mL of blocking buffer for the 3rd round of panning.
                    </LI></OL><H5>DAY 5</H5><OL><LI>
                        Carry out a third round of panning: repeat DAY 1 (steps 1 to 9 only), using the second round amplified eluate at an input titer equivalent to what was used in the first round, again using 0.5% Tween in the wash steps. It is not necessary to amplify the third round eluate. <EM>
                            Plaques from the titering can be used for sequencing: time the procedure so that plates are incubated at 37°C for no longer than 18 hours, as deletions may occur if plates are grown longer. Stored at 4°C, plates should be used to pick plaques within 1–3 days of plating. The remaining eluate can be stored at 4°C for at least one week.
                        </EM></LI><LI>
                        For preparation of individual clones for sequencing or ELISA, set up a 10-ml overnight culture of ER2738 from a colony, not by diluting the titering culture. Proceed with plaque amplification. Do not amplify the third round eluate and carry out a fourth round of panning unless the characterization shows no clear consensus sequence or phage ELISA results are negative.
                    </LI></OL><H4>Plaque Amplification for ELISA or Sequencing</H4><P class="input">
                Selected phage clones can be identified by DNA sequencing, and target specificity can be confirmed by phage ELISA. In both cases it is necessary to amplify phage, either from individual plaques or from the eluted pool, to obtain sufficient quantities to work with.
            </P><OL><LI>
                    Dilute an overnight culture of ER2738 1:100 in LB. Dispense 1 ml of diluted culture into culture tubes, one for each clone to be characterized. 10–20 clones from the third round are usually sufficient to detect a consensus binding sequence.
                </LI><LI>
                    Use a sterile wooden stick or pipette tip to stab a blue plaque from a titering plate (important: plates should be less than 1–3 days old, stored at 4°C and have less than 100 plaques) and transfer to a tube containing the diluted culture. Pick well-separated plaques. This will ensure that each plaque contains a single DNA sequence.
                </LI><LI>
                    Incubate the tubes at 37°C with shaking for 4.5–5 hours (no longer).
                </LI><LI>
                    Transfer the cultures to microcentrifuge tubes, and microfuge at 14,000 rpm for 30 seconds (see next section to purify sequencing template). Transfer the supernatant to a fresh tube and re-spin. Using a pipette, transfer the upper 80% of the supernatant to a fresh tube. This is the amplified phage stock and can be stored at 4°C for several weeks with little loss of titer. For longterm storage (up to several years), dilute 1:1 with sterile glycerol and store at –20°C.
                </LI></OL><H4>Sequencing of phage DNA</H4>
    
                This extremely rapid procedure produces template of sufficient purity for manual or automated dideoxy sequencing, without the use of phenol or chromatography.
    
            <OL><LI>
                    Carry out the plaque amplification procedure described in the Phage Amplification protocol. After the first centrifugation in Step 4, transfer 500 µl of the phage-containing supernatant to a fresh microfuge tube.
                </LI><LI>
                    Add 200 µl of 20% PEG/2.5 M NaCl. Invert several times to mix, and let stand for 10–20 minutes at room temperature.
                </LI><LI>
                    Microfuge at 14,000 rpm for 10 minutes at 4°C and discard the supernatant. Phage pellet may not be visible.
                </LI><LI>
                    Re-spin briefly. Carefully pipet away and discard any remaining supernatant.
                </LI><LI>
                    Suspend the pellet thoroughly in 100 µl of Iodide Buffer by vigorously tapping the tube. Add 250 µl of ethanol and incubate 10–20 minutes at room temperature. Short incubation at room temperature will preferentially precipitate single-stranded phage DNA, leaving most phage protein in solution.
                </LI><LI>
                    Spin in a microfuge at 14,000 rpm for 10 minutes at 4°C, and discard the supernatant. Wash the pellet with 0.5 ml of 70% ethanol (stored at –20°C), re-spin, discard the supernatant, and briefly dry the pellet under vacuum.
                </LI><LI>
                    Suspend the pellet in 30 µl of TE buffer. The template can be suspended in H2O instead of TE if desired, but this is not recommended for long-term storage. In TE buffer, the phage DNA should be stable indefinitely at –20°C.
                </LI><LI>
                    Quantitate product by using a Nanodrop.
                </LI><LI>
                    Send to sequencing.
                </LI></OL><H5>Sequencing Guidelines</H5><OL><LI>
                        The sequence being read corresponds to the anticodon strand of the template. Write out the complementary strand and check against the top strand of the insert sequence shown in the figure below (the primers hybridize downstream of the insert). Check that the third nucleotide of each codon in the randomized region is G or T.
                    </LI><LI>
                        TAG stop codons are suppressed by glutamine in ER2738 (glnV). If the library was amplified in this strain or any glnV (also known as supE) strain, TAG should be considered a glutamine codon when translating.
                    </LI><LI>
                        Libraries often contain a small percentage (less than 1%) of clones containing multiple inserts of the randomized region. Preferential selection and amplification of these clones may occur when panning against targets whose ligand specificity spans a length greater than that specified by the insert. When interpreting sequence data, be sure the sequence outside the restriction sites used for inserting the randomized sequence (Acc65I and EagI) matches the sequence shown in Figure 5. Once the flanking regions of the library insert have been identified, the insert size is easily determined.
                    </LI></OL></DIV><H2 class="red">Phage ELISA:</H2><DIV><P><OL><LI>
                                Carry out the plaque amplification procedure described previously- After the first centrifugation, save the phage-containing supernatants at 4°C.
                            </LI><LI>
                                For each clone to be characterized,  dilute an overnight culture of ER2738 1:100 in 20 ml of LB.
                            </LI><LI>
                                Add 5 μl of phage stock from Step 1(or a single phage plaque) to a 20 ml culture for each clone and incubate with vigorous aeration for 4.5–5 hours at 37°C.
                            </LI><LI> 
                                Transfer the culture to a centrifuge tube and spin at 12,000 g for 10 minutes at 4°C. Transfer the supernatant to a fresh tube and re-spin (discard the pellet).
                            </LI><LI> 
                                Pipette the upper 80% of the supernatant to a fresh tube and add 1/6 volume of 20% PEG/2.5 M NaCl. Allow the phage to precipitate at 4°C for at least 2 hours or overnight.
                            </LI><LI>
                                Spin the PEG precipitation at 12,000 g for 15 minutes at 4°C. Decant and discard the supernatant, re-spin briefly, and remove residual supernatant with a pipette.
                            </LI><LI>
                                Suspend the pellet in 1 ml of TBS. Transfer the suspension to a microcentrifuge tube and spin at 14,000 rpm for 5 minutes at 4°C to pellet residual cells.
                            </LI><LI>
                                Transfer the supernatant to a fresh microcentrifuge tube and re-precipitate with 1/6 volume of 20% PEG/2.5 M NaCl. Incubate 15–60 minutes on ice. Microcentrifuge at 14,000 rpm for 10 minutes at 4°C. Discard the supernatant, re-spin briefly, and remove residual supernatant with a micropipet.
                            </LI><LI>
                                Suspend the pellet in 50 μl of TBS and Titer. The titer should be approximately 10^14 pfu/ml. The phage stock can be stored for several weeks at 4°C. 
                            </LI><LI>
                                Elisa was first performed in the 96 well plate with the target (fabric) developed by the Assay team. Fill each well completely with blocking buffer. Additionally, one row of uncoated wells per clone to be characterized should also be blocked to test for binding of each selected sequence to BSA-coated plastic. A second, fully uncoated microtiter plate should be blocked for use in serial dilutions of phage (step 13) before addition to the target-coated plate. Dilutions are done in a separate blocked plate to ensure that phage are not absorbed onto the target during the course of performing dilutions, which would result in a sudden “falling-off” of signal as the phage is diluted. Incubate the plates filled with blocking buffer for 1–2 hours at 4°C.
                            </LI><LI>
                                Shake out the blocking buffer and wash each plate 10 times with TBST, slapping the plate face-down onto a clean section of paper towel each time. The percentage of Tween should be the same as the concentration used in the panning wash steps.
                            </LI><LI>
                                In the separate blocked plate, carry out fourfold serial dilutions of the phage in 200 μl of TBS/Tween per well, starting with 10^12 virions in the first well of a row and ending with 2 x 10^5 virions in the 12th well.
                            </LI><LI>
                                Using a multichannel pipettor, transfer 100 μl from each row of diluted phage to a row of target-coated wells, and transfer 100 μl to a row without target. Incubate at room temperature for 1–2 hours with agitation.Wash plate 10 times with TBST as in Step 12.
                            </LI><LI>
                                Dilute HRP-conjugated anti-M13 monoclonal antibody (GE Healthcare. #27-9421-01) in blocking buffer to the final dilution recommended by the manufacturer. Add 200 μl of diluted conjugate to each well and incubate at room temperature for 1 hour with agitation.
                            </LI><LI>
                                Wash 10 times with TBST as in Step 12.
                            </LI><LI>
                                Prepare the HRP substrate solution as follows: a stock solution of ABTS can be prepared in advance by dissolving 22 mg of azino-bis(3-ethylben- zothiazole sulfonic acid) diammonium salt (ABTS) (Sigma, cat. # A-1888) in 100 ml of 50 mM sodium citrate, pH 4.0. Filter sterilize and store at 4°C. Immediately prior to the detection step, add 36 μl of 30% H2O2 to 21 ml of ABTS stock solution per plate to be analyzed.
                            </LI><LI>
                                Add 200 μl of substrate solution to each well, and incubate for 10–60 minutes at room temperature with gentle agitation.
                            </LI><LI>
                                Read the plates using a microplate reader set at 405–415 nm. For each phage concentration, compare the signals obtained with and without target protein.
                            </LI><LI>
                                The protocol was later performed in micro-centrifuge tubes to reduce non specific binding of peptides to the polystyrene. All the reagent volumes were the same. Washes were increased to 1ml * 5 times.
                            </LI></OL></P></DIV><H2 class="red"> GOLDEN GATE ASSEMBLY </H2><DIV><H4> GOLDEN GATE ASSEMBLY/CLONING PROTOCOL</H4>

For the Golden Gate assembly (GG) here we fused two protocols, using part of the one facilitated by NEB Golden Gate Assembly Mix, which contains BsaI and T4 DNA Ligase, and another which we had by courtesy of our advisor Nadine Bongaerts: <B>uL</B>
x 40 fmol of vector and insert
1ul Buffer 10x 
0.5ul Assembly mix
x dH2O (until 10ul)
-----------------------------<B>10 uL total</B><B>Settings</B>

1. 37 C - 5 min
2. 37 C - 2 min
3. 16 C - 5 min
4. 50X back to 2
5. 37 C - 5 min (final digest)
6. 50 C - 10 min
7. 80 C - 10 min
Once this is done the product of the reaction can be transformed avoiding to use electroporation, since it could be ions still in the mix

</DIV><H2 class="red"> Over-expression of bpul </H2><DIV><P>
In this protocol volumes and other quantities used are proportional. By applying simple mathematical operations of multiplication or division one can adjust the protocol based on his/her needs.

<OL><LI>
Grow minimum 25 mL of overnight culture of BL21(DH3)-bpul in liquid Luria-Bertani (LB) broth with chloramphenicol.
</LI><LI>
In the morning, prepare 1L of LB broth in 2L Erlenmeyer flask and add 20 mL of overnight BL21(DH3)-bpuI cell culture. One part of the cell culture is diluted in 50 parts of LB broth. Grow until optical density of 1-2 at 37 oC
</LI><LI>
Add 0,1 mM of IPTG (1 mL of 100 mM IPTG stock) for induction of T7-polymerase. Add 0,25 mM CuCl2 (250 uL of 1 M stock solution) for assuring fully copper loaded CotA enzymes.
</LI><LI>
Grow cells for 4 h at 25 oC with shaking.
</LI><LI>
Grow cells for 20 h at 25 oC without shaking.
</LI><LI>
Harvest the cells by centrifugation and wash them with PBS.
</LI><LI>
Store cells at -20 oC for improving the yeald after B-per protein extraction
</LI></OL>

References:
1. DOI: 10.1186/1472-6750-11-9
2. DOI: 10.1007/s00775-007-0312-0
</P></DIV><H2 class="red"> Protein (BG1, catA, POO2, tanLpI, xyleE) overexpression </H2><DIV><H3>POO2 - copper binding enzyme</H3><B>WARNING!</B> The enzyme seems to be insoluble and is forming inclusion bodies. Careful with B-PER extraction. Protocol for bpuI and POO2 seem to be very similar, so use one you are more comfortable with.
<OL><LI>
Recombinant E. coli BL21 (DE3) is grown at 37 ◦ C with shaking to mid-logarithmic phase in LB solution containing antibiotic and 20 umol/l Cu++ (Sullivan et al., 2004)
</LI><LI>
Induce by adding IPTG (500 umol/l) and grown for an additional 3 h. 
</LI><LI>
The cells were harvested by centrifugation at 4000 × g for 1 min and resuspended in PBS buffer.
</LI><B>Extracting inclusion bodies:</B><LI>
Lysozyme was added to 100 ␮g ml−1, and the suspen sion was incubated at room temperature for 15 min.
</LI>
All subsequent steps were carried out at 4 ◦ C or on ice:
<LI>
The inclusion bodies were separated from the soluble extracts by centrifuging at 10,000 ×g for 10 min. 
</LI><LI>
Soluble and insoluble fractions were analyzed by 12% SDS-PAGE (sodium dodecyl sulfate polyacrylamide gel electrophoresis).
</LI></OL>

References:

doi:10.1016/j.jbiotec.2009.10.008

<H3>tanLpI - Long Growth</H3><OL><LI>
Cells carrying the recombinant plasmid were grown at 37 °C in LB + antibiotic (100 μg/mL)
</LI><LI>
(I guess at OD 0.6 were) induced by adding 0.4 mM IPTG.
</LI><LI>
After induction, the cells were grown at 22 °C during 20 h and collected by centrifugation. 
</LI><LI>
Cells were resuspended in 20 mM Tris-HCl, pH 8.0, 100 mM NaCl.
</LI></OL>
References:

DOI:10.1021/jf901045s

<H3>BG1, catA, xylE - standard protocol</H3><OL><LI>
Pick colonies to inoculate small (5 ml) LB + antibiotic cultures and shake overnight at 37°C
</LI><LI>
In the morning, inoculate a new culture by diluting the overnight culture 1/100 to 1/1000 with LB + antibiotic
</LI><LI>
Grow shaking at 37ºC until an OD of 0.5–0.8 is reached, and then add IPTG to a concentration of 0.5 - 1 mM. Continue shaking at 37°C for 5 hr.
</LI><LI>
Centrifuge samples at 4000rpm for 10 min at 4°C, remove supernatant, they are ready to do the cell extract, with agents such as B-PER for example, or resuspend them in glycerol 30% in water to be stored at -80C.
</LI></OL>
References:

DOI 10.1007/s10725-014-9932-x

DOI 10.1007/s10930-015-9637-7

DOI 10.1002/bmb.20328

Clontech Laboratories, Inc. pET Express &amp; Purify Kits User Manual
</DIV><H2 class="red"> SDS-PAGE electrophoresis </H2><DIV><H4>Materials:</H4>
10% per-made BioRad mini-protean gel

2x Laemmili sample sample loading buffer + 1:40 beta-mercaptoethanol

10x Tris/Glycine/SDS buffer BIORAD
<H4>Sample preparation</H4><B>Cells</B><OL><LI>
Grown cells for 5h in 0.5 mM IPTG
</LI><LI>
Measure the OD600 of the cells, and harvest them by centrifugation
</LI><LI>
Resuspend the cells in the appropriate volume of 2x Laemmili sample buffer so that OD600 of the cells is 10
</LI><LI>
Cook the cells for 10 min at 95oC
</LI></OL><B>Cell extract</B><OL><LI>
The cell extract was mixed 1:1 with Laemmili sample buffer
</LI><LI>
Heating 10 min at 95 oC
</LI></OL><B>The electrophoresis is run under the following conditions:</B>
 - The current is set to 25 mA for 10 min

 - After 10 min the samples have reached the separating gel, so the current is increased to 50 mA and ran until the front line reached the bottom of the gel.
<H4>Staining:</H4><OL><LI>
Gels are washed 3x for 5 min in miliQ water to remove the SDS.
</LI><LI>
Staining is performed with BIO-RAD Comassie dye for 30 min with gentle shaking
</LI><LI>
De-stain for 1h
</LI></OL></DIV><H2 class="red"> Binding strength of the Fabric Binding Domains</H2><DIV><H4> Assaying the strength or the Fabric Binding Domains using the assay developed by our team </H4>

We assayed the strength or the Fabric Binding Domains using the assay developed by our team, carrying cotton, wool, silk and linen fabrics. <B>In our experiments</B> we measure the fluorescence of the GFP fused with the FBD, therefore we set the plate reader to excite the samples with a wavelength of 475nm and for the emission a wavelength of 515nm.
We also wash the sample with different washing solutions, these were:<OL><LI>Water</LI><LI>Phosphate-buffered saline (PBS) </LI><LI>Bovine serum albumin (BSA) 5% (w/v)</LI><LI>Ethanol 70% (v/v)</LI><LI>Catechol 0,03M</LI></OL>
The steps we followed are the next:
1. Produce the cell extracts of BL21(DE3) strains carrying GFP-FBD fusion proteins were performed and 1/10 dilutions were made.  
2. A volume of 200ul is incubated 24 hours in each well of the assay plate.
3. Measure the signal of the sample before removing the cell extract
4. Remove the cell extract and measure
5. First wash by adding 200ul of washing solution and removing it after 30 seconds
6. Measure again after the first wash
7. Repeat the step 5
8. Finally, measure again after the second wash
This protocol is based in the one carried out by Imperial College London iGEM team 2014, which project was called <I>Aqualose</I>. The protocol can be found in its web site and it is called “CBD binding strength assay, using fluorescence”. </DIV></DIV></DIV><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:igem2016parisbettencourt@gmail.com">igem2016parisbettencourt@gmail.com</A><CENTER><A href="https://2016.igem.org/">
    2016.igem.org
</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>