<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Notebook_Microbiology skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Notebook/Microbiology</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navmenubkg"><DIV style="margin-left:-15px" id="navmenubox"><UL id="nav"><LI><A style="margin-left:10px" href="https://2016.igem.org/Team:Paris_Bettencourt/Description">Project</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Description">Overview</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Assay">Assay</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Microbiology">Microbiology</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Enzyme">Enzyme</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Binding">Binding</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Project/Indigo">Indigo</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Model">Model</A></LI></UL><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/About_Perc">Practices</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/About_Perc">About Perc</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Integrated_Practices">Dry Cleaners</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Design">Design</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Human_Practices">Events</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Safety">Safety </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Safety">Lab Safety</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Product_Safety" style="margin-top:-10px; margin-bottom:30px;">ProductSafety</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Achievements">Achievements</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Interlab_Study">Interlab Study</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Medal_Requirements">MedalRequirements</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI style="margin-left:10px"><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook">Daily Note</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Protocols">Protocols</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Bibliography">Bibliography</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Bettencourt/Team">Team </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Team">Our Team </A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI><LI><A class="liste" href="https://2016.igem.org/Team:Paris_Bettencourt/Sponsors">Sponsors</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><DIV id="subheader"><TABLE id="ancre"><TBODY><TR><TD><A href="#ancre">Notebook</A></TD><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Protocols">Protocols</A></TD><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Bibliography">Bibliography</A></TD></TR></TBODY></TABLE></DIV><DIV id="subheader2"><TABLE><TBODY><TR><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Assay">Assay</A></TD><TD><A href="#ancre2">Microbiology</A></TD><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Enzyme">Enzyme search</A></TD><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Indigo">Mission Indigo</A></TD><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Binding">Binding Domains</A></TD><TD><A href="https://2016.igem.org/Team:Paris_Bettencourt/Notebook/Anthocyanin">Anthocyanin</A></TD></TR></TBODY></TABLE></DIV><DIV id="ancre2"><DIV class="input"><H2 class="red">Introduction</H2>
The microbiology team aims to identify wine stain degrading microorganisms.
			    
		        Basing this bioremediation-inspired approach on the two following articles:
				    <UL><LI> Bioremediation of phenol by alkaliphilic bacteria isolated from alkaline lake of Lonar, India  P.P. Kanekar, S.S. Sarnaik and A.S. Kelkar. <I>Journal of applied microbiology supplement</I> 1999. 
					    </LI><LI> Bacteria Subsisting on Antibiotics  Gautam Dantas,  Morten O. A. Sommer,  Rantimi D. Oluwasegun, George M. Church. <I>Science</I>2012
				        </LI></UL>
		        We hypothesize that these microorganisms could be preferably discovered in vineyard soil samples.
		        
		        Indeed, it is highly probable that a rich-anthocyanin environment such as a vineyard, would host microbes with the desired degradation skills. In addition, the chance to find a microbe able to digest efficiently a wine stain, with its proper anthocyanin composition, in terms of anthocyanin diversity, and abundance, is theoretically increased.
			    
			    To ensure the widest microorganisms diversity, we used soil samples from all around the world (Australia, Spain, Namibia, France, Croatia), particularly through collaboration with different iGEM teams.
			    This bacteria identification was based on two different approaches:
				    <UL><LI> We are creating a bacterial database. It implies culture of the bacteria on different selective and non-selective media, and characterization of the strains. This identification was managed through 16s rRNA PCR, known as the most common housekeeping genetic marker in bacteria. At the end, we would test the database directly on fabrics stained with wine, or only anthocyanin, looking for color degradation. We also plan to build a phylogenetic tree based on our database that would help us understanding how related are the species able to degrade anthocyanin.
                        </LI></UL>
                The main advantage here is to select potential useful bacteria from soil, isolate them, and screen all of them, on fabrics, thanks to a high throughput assay. Nevertheless, it implies a selective bias, as some interesting microorganisms may not grow in the media we chose, or cannot compete with other microorganisms present in the soil. 
                
                To tackle this problem, we also followed another approach: 
                <UL><LI> We screened our samples for a microorganism growth directly on anthocyanin-enriched media, and for anthocyanin degradation by absorbance measurement. The idea here is to screen for bacteria that could use anthocyanin as their sole carbon source, and thus degrade it, or that could simply metabolize it. After identification of potential interesting microorganisms, we would isolate them, characterize them, and then test them on stained fabrics. 
                        </LI></UL>
                Therefore, the main point of our team is to put in evidence already existing anthocyanin-degradation metabolism in nature, so that we could isolate the enzymes, and potentially optimize them thanks to the binding domain team results. 
        <H2 class="red">Week 27th June - 3rd July</H2>
                The first step is to define a systematic protocol for the selection of microorganisms from our samples.
                
                It includes :
                <OL><LI>Determine samples conditions of collection, storage, and treatment before microbes culture.
                        </LI><LI>Determine the selective and nonselective media we will use for culture and the number of dilutions per samples.
                        </LI><LI>After streak and isolation, determine an efficient way to characterize the strains.
                        </LI><LI>For the second approach, where we test directly an anthocyanin-degradation natural metabolism, define the controls.
                        </LI></OL>
                Therefore, during this week, we focused our work reading articles dealing with bioremediation and isolation of bacteria from soil samples.
                
                Nonetheless, we rapidly had to tackle the following issue: Such approaches implies protocols of microorganisms culture on anthocyanin enriched media or even as a single carbon source. Or, anthocyanin isolation and purification is hard to achieve, and consequently, this chemical compound is really expensive. We cannot afford to buy it in high quantity.
                
                Considering this problem, we decided to take a deeper look in the anthocyanin chemical structure, so that we could potentially find a cheaper substitute molecule for our assays.
                
                Anthocyanin is a phenolic compound which belongs to the Flavonoid family. Flavonoids have a basic structure of C6–C3–C6. Depending on their structures, flavonoids may be classified into about a dozen groups, such as chalcones, flavones, flavonols and anthocyanins.
                
                If we take a deeper look in the anthocyanin structure, we observe that it’s composed of :
                    <UL><LI> a chroman ring </LI><LI> an additional aromatic ring on C2 </LI></UL>
                There are 31 different monomeric anthocyanins known. They differ from number and position of the hydroxyl and/or methyl, ether groups.
                
                But, 90% of the naturally occurring anthocyanins are based on only six common anthocyanidins. among them :
                    <UL><LI> Cyanidin,the major form in nature 
                        </LI><LI> Malvidin, the most commonly found in wine 
                        </LI></UL>
                As we can see in this figure, they only differ by their cycle B groups, so the chroman ring remains unchanged. 
                
                Therefore, we believe that enzymes able of degrade a lot of different types of anthocyanins would preferably attack the chroman ring structure as it is well conserved among the family.
                <B>Thus, we thought that the ideal cheap substitute molecule should present a very similar basic structure.</B>
                After some researches, Quercetin, a flavonol molecule, appeared to be a good competitor.
                
                We can in fact see that the only difference remains in the presence of a carbonyl group in 4. Another advantage is that Quercetin is present in Wine, and is partly responsible for its color. Thus, even though the assays may find some enzymes able to degrade specifically quercetin instead of anthocyanin, we should have a decrease in the wine stain color intensity. In addition, because of the co-pigmentation chemical interaction between quercetin and anthocyanin, degrading quercetin could also have an impact on anthocyanin stability.
                <B>For these reasons, quercetin was chosen as our substitute molecules for running the microbiological assays.</B>
                However, in parallel, we decided to develop a protocol for purifying anthocyanin from grapes, so that we could also tests our microbes directly on anthocyanin. (See the Anthocyanin section for the evolution of the protocol).
                <H2 class="red">Week 11th - 17th July</H2>We are looking into the microbes that could be able to degrade quercetin.  Because we want a high diversified database, we asked iGEM team from all around the world to send us soil,grape and grape's leaves samples.
At the end of the competition, we got samples from Melbourne and Macquarie iGEM teams in Australia, from Valencia iGEM team, from Barcelona iGEM team from INSA-Lyon iGEM team, but also from Namibia, Croatia, Algeria, from Cochin and Clos Montmartre in Paris and from Grignon and Suresnes in France.
Also, we are looking into some human gut strains for their quercetin degradation ability. Indeed, wine is know for its cardiovascular protect effect. And we figured out that the compound enabling this effect is actually phenol compound.
On top of that, quercetin and anthocyanin seems to be the most powerful wine antioxydant, and mostly explain this cardiovascular protector effect .
The epithelium from the gut is not able to digest the flavonoid and to cleave it into phenol compound. That's why it is higly probable that the microbiome is responsible for its degradation.If we were able to isolate some bacteria from human feces on quercetin plate, we may have some very interesting results.
The problem is that we need an authorization to work with human sample. We sent a request for it but it may take a lot of time, with the hope of a quick answer.
Zhang Z, Peng X, Li S, Zhang N, wang Y, Wei H (2014) Isolation and Identification of quercetin Degrading Bacteria from Human Fecal Microbes. <I>PLoS</I> ONE 9(3): e90531. doi:10.1371/journal.pone.0090531
In the same way, It would also be very interesting to work with herbivorous animals, as they eat a lot of flavonoids, responsible for many plants color.So we can expect that some microbes isolated from these animals' gut could also degrade theses flavonoids into phenolic compounds. But once again we need some authorizations to work on animals samples. 
<H2 class="red">Week 17th - 25th July</H2>
                We discovered Church's article, describing the isolation of antibiotic degrading bacteria from soil samples. It appears that the goal was very similar to ours, the only difference remaining in the nature of the molecule.  As they wanted to isolate bacteria that can grow on antibiotics as a unique carbon source, their main concern was carbon contamination from the soil sample. 
                
                To avoid this contamination they inoculated the samples in a SCS (single carbon source) liquid medium. They let it grow seven days at 22°C then use the broth to inoculate a new SCS liquid medium.
This step is repeated two more times. Then the culture broth is plated on SCS plate and the degrading bacteria is isolated.  In these conditions the carbon from the soil is consumed during the first cultivation steps and the bacteria that cannot use antibiotics as a carbon source do not survive during the next cultivation step.  
                
                However, this protocol seems to have some issues.
                    <UL><LI>It take 21 days before plating to isolate the microorganisms, this is very long. 
                        </LI><LI>Microorganisms that can degrade quercitin without using it as a single carbon source won't be selected.
                        </LI></UL>
                As we found in several papers, the regular technique to isolate microorganisms from the soil is to dilute the sample on PBS and directly plate on agar. A dilution of 10^(-1) correspond to a dilution of 1g of soil in 9 mL of PBS. <H3>Filtration of the soil sample </H3>
                A member from the protein group told us that he used a different protocol to isolate a toluene degrading bacteria. In fact, yo remove the carbon contamination from the soil he filtered the sample with a 0.22µL filter. Then he cultivated the filter in a liquid medium before plating. We decided to test this protocol. 
                
We filtrate a solution of soil sample under vacuum.  Then we place the filter (with the microbes stuck in it without carbons sources) in an erlenmeyer with some M9 + Quercetin. And we waited for a few days. We had the same results as the protocole with soil sample diluted at 10^-3. And as you can see in the picture, the green color is disappearing and some fungi is growing in the media. <H3>Screening assay design </H3>
                In church lab's article, they cultivate the bacteria at 22°C and pH 5.5 . We found that pH 5.5 inhibit most bacterial growth so we decided to stay at a neutral pH. 
                
                We did not have an incubator that could be set at a temperature of 22°C, after a discussion with our advisors and a microbiologist from Cochin hospital it was decided that 30°C should be sufficient for our experiement. Some bacterial species would not survive at 30°C but a sufficient number would, and they may grow faster at 30°C than at 22°C. 
                We decided to chose 5 dilutions, from 10^(-2) to 10^(-6). Instead of only plating at the end of the 3 liquid cultivation, we decided to plate 100 μL of medium just after inoculation and at the end of each cultivation. The goal of this experiment is to know the dilution needed and to test the impact of the several liquid medium cultivation steps.
                We decided to chose 4 different medium (for broth and plate). 
                -M9 : a medium with salts but no carbon source, this is our negative control medium, if something grows on that medium, it means that there is a carbon contamination.
                -M9 glucose (M9 G) : M9 salts plus 1g/L glucose, this medium is our positive control, if nothing grows on it, it means that there were no bacteria in the medium.          -M9 quercetin (M9 Q): M9 salts plus 1g/L quercetin (similar condition than in Church's article), this is the medium where we want to isolate bacterias.
                -M9 glucose quercetin (M9 GQ) : this medium is necessary to test if quercetin is toxic for microorganisms. Indeed if we have less growth than in M9 glucose it could mean that quercetin is toxic. 
                As it is done in church's protocol the incubation time between each cultivation step is 7 days. <H2 class="red">Week 25th - 31th July</H2><H3>Quercetin medium preparation </H3>

                As soon as the quercetin was purchased from Sigma®, we decided to start the assay. However it was found that unlike anthocyanin, quercetin was mostly insoluble in water. This is a major issue for medium preparation as the organisms could not be able to degrade an insoluble molecule in a liquid broth.
But it as also a problem for quercitin quantification, as we thought at the beggining that we could measure the degradation of quercetin with the absorbance, which is not possible if quercitin isn't soluble.
                
                We tried to solubilize quercetin at different concentration in water and ethanol : 1g/L, 0.1g/L, 0.01g/L and 0.001g/L . We did not succeed in dissolving totally the quercetin, so an absorbance assay was impossible.  However, even at 1g/L the liquid was a little yellow, so quercetin was at least slightly soluble.  We thought that the little quantity of soluble quercetin could be enough for growth, as long as the medium is agitated and the quercetin constantly solubilized during the consumption. So we decided to start the assay anyway. Me managed to have relatively homogenous plate. <H3>Results after two days </H3><UL>
Growth results in plate:
 <LI>The control plates with M9 showed growth until 10^(-3), that means that if we want to avoid carbon contamination in plate, we need to dilute the samples at 10^(-3) at least</LI><LI> The plate with M9 G always showed growth. Even in the more diluted samples there was microorganisms.</LI><LI> The plates with M9 QG showed less growth than the plate with M9G and there was no growth after a dilution of 10^(-3), it could mean that quercetin is toxic for some organisms.</LI> The plates with M9 QG showed growth until a dilution of 10^(-2) because of the carbon contamination.</UL><B>This part of the experiment seemed compromised as there was no growth in the condition M9Q at a dilution were there was no carbon contamination (below 10^(-3))</B>

Growth results in tubes:
<UL><LI> M9 growth with a dilution of 10^-2, the carbon contamiation is avoided with a dilution of 10^-3 and below.</LI><LI> M9 G groth in every tube, there is always microorganisms in the samples. </LI><LI>M9 Q and M9 QG, it is difficult to say, the quercitin is precipitated and it seams that the experiment is a failure, that the quercetin cannot be used in liquide medium, maybe because the agitation is too low. </LI></UL><H2 class="red">Week 01th - 7th August</H2><H3>Results after a week in solid medium</H3>       
The results were surprising. In the plates with dilutions of 10^(-2) and 10^(-3), with or without glucose, we could see transparent circles. The quercetin seemed to have disappeared. In the middle of these circles, there were always some microorganisms that looked like filamentous fungi, with the hyphae in the middle and spores in the whole circle.
  
It is to be noted that it is written that the dilution is 10^(-1). This is an annotation mistake. At the beginning we did not counted the initial dilution of the soil sample in water. We have considered that our original sample was 1g of soil in 9mL of water.<H3>Results after a week in liquid medium</H3>
With dilution of 10(-2) and 10(-3), in quercetin glucose and in M9 quercetin, there was fungal growth and as the medium was less yellow, it seems that quercetin was partly degraded.
<CENTER>Tube with quercitin</CENTER><CENTER>Tube with fungal growth (dilution 10^(-2),medium M9Q)</CENTER>

As our laboratory didn't have at this moment a working experience with fungi, we decided to keep the plates at 4°C before working with them, hoping to find bacterias meanwhile. 
 	<H2 class="red">Week 8th-14th August</H2><H3> Filamentous fungi lab practice</H3>       
We looked for protocol to work safely with fungi. We found a quite complete fungi protocol book &quot;Laboratory protocol in fungal biology&quot; (M. Ayyachamy and al). There was a safety part. The more important was to prevent spore contamination, knowing that ethanol is not sufficient to kill fungal spores.  A way to disinfect the hood needed to be found. It was said in the book to disinfect with bleach between every experiment and , at the end of the day, to disinfect during 20 minutes with a strong disinfectant like those used in hospitals. 
We were able to obtain the disinfectant SURFA’SAFE preminim of the laboratory Anios from the microbiologist of Cochin.<BT>  It was decided with our advisor to disinfect with bleach between each work with fungi. 
Disinfect during 20 minutes with anios was performed each time a member had finished to use fungi.

<H3> Quercetin analysis</H3>       

We still tried to solubilize quercetin. In fact, quercitin was almost insoluble, even at a concentration of 1mg/L, the quercetin precipitated. We tried to heat it but after 1 hour, the quercetin was still not solubilized.  We thought that the solubility of the molecule was of 200g/L so we were very surprised. We looked again on Sigma Aldrich information and that was actually the solubility in DMSO, an organic solvent, which explain our negative results.

The insolubility of quercetin didn't seem to be a real problem for the microorganisms, both in liquid and solid medium. However, it was still a problem for the degradation  quantification.

Another problem was the separation between the cells and the quercetin. We tried to filter a sample of water with quercetin with a 0.22 um² filter. All the pigment stayed fixed to filter. The pores were much bigger than the quercetin. It seemed that the quercetin has a strong affinity to the membrane. As the quercetin is mostly insoluble in water, a centrifugation was a way to remove the cells and the pigment from the broth. All we needed, then, was a medium were the quercetin was soluble, preferably not an expensive and toxic organic solvent.

 	<H2 class="red">Week 15th-21th August</H2><H3> Fungal strains collection</H3>       
We had all we needed to work with filamentous fungi. We isolated the identified strains and inoculated them in Sabouraud Dextrose Agar, a common plate for filamentous fungi culture.  After 4 days the plates were covered of mycelium. Then, we inoculated them on M9Q agar Plate and in M9 plates.

   <H3>Test in liquid medium</H3>       
The isolated fungi were tested in 3 liquid medium : <OL><LI>M9</LI><LI>M9 + 1g/L quercetin </LI><LI>M9 + 1g/L quercetin + 1g/L glucose </LI></OL>
There was also a blank without fungi<H3>Quercitin analysis </H3>       
We wanted to see if the disappearance of the color in plates could be due to a pH modification. So we took 1g/L of quercetin and changed the pH with hydrochloric acid. There was no color change at a pH of 1.5. 
Then we added base and there was a change. Around a pH of 12 we could see a solubilization of quercetin. After a pH of 12.5, quercetin was completely solubilized. The color was orange instead of yellow. There was no color change between a pH of 12.5 and 13.5. <B>We had solubilized quercetin</B>. We managed to solubilize up to 50g/L of quercetin in alkaline water. 
We measured the absorbance spectrum and we could see that the maximum of absorbance was at 315nm. 
To found the link between quercetin concentration and absorbance, we did a calibration. 
The R² is good, that confirm that the calibration is working. We have a relation between absorbance and concentration : A = 56,767*C, that mean that C=1/(56,767)*A
Finally C = 0,176*A

 	<H2 class="red">Week 22th-28th August</H2><H3> Result of the previous assay  </H3>       
There was growth on M9 glucose and M9 quercetin glucose for plates with dilutions of 20^(-2) and 10^(-3). However, the variability was much less important. No quercetin degradation organisms were found. This might be due to the osmotic choc caused by the dilution of the samples in distilled water. We will remember not to repeat this mistake.

   <H3> Third assay </H3>
We did a third assay with a soil sample from Croatia. The conditions were the same than for the two first assays (with PBS and not distilled water !).

   <H3> Fungal isolation</H3> 
There was still growth on M9Q plates, we could still see the circles of quercetin degradation. It was confirmed that the strains have been isolated.

<H2 class="red">Week 29th August-4th September</H2><H3> Fourth assay </H3>
We chose to make a simple selection process:

For that, we put some soil sample at 10^-3 of dilution in a single carbon source liquid media. The only carbon source is Quercetin at 1g/L.  We waited for 4 days with the falcon tubes in the shaking incubator at 30°C. In an other hand, we made an other experiment with two carbon sources: Quercetin and Glucose both at concentration of 1g/L. The idea was to give a chance to some strains that are not maybe able to use Quercetin as a carbon source, but able to destroy this coumpound with some other pathways  By providing them glucose, we could maybe maintain them alive and in the meantime theses strains would be able to degrade Quercetin.
We also had a negative and a positive control.  The negative control was M9 Quercetin without any microbes. The positive control was Pseudomonas putida. We made also some conditions with M9 + Glucose with the positive control, the negative control, and the two different soil samples we assayed. This was done to have an idea of the consumption of glucose by the different strains, measuring the growth of these bacteria at OD 600 nm. We wanted to see if the degradation of Quercetin is enhanced or inhibited by the presence of glucose. 


Surprisingly, Glucose didn’t have the effect we expected. Pseudomonas was slower but almost unaffected in its degradation of Quercetin while, the microbes from the two soil sample were not able to degrade Quercetin! We thought that the diauxie phenomena described by Monod, could explain these results. 
It is important for us because it’s a way of understanding how the strains are using the Quercetin: is it used as a carbon source via the carbon cycle for growth of the strains or, is it used by another pathway that do not permit the growth of the microbe but is effective in the degradation of the Quercetin (for a toxic compound for instance)
The only solution we found to solubilize quercetin was putting it at pH 12, which would obviously have an impact on bacterial growth.
Thus, this prevent us of doing a kinetic experiment to see the evolution of the growth of a strain with quercetin and glucose at the same time by measuring OD600nm. Nevertheless, we tried to make an experiment to see if there was a glucose repression with Pseudomonas when we put at the same concentration (1g/L) Glucose and Lactose. 

As you can see on the graph M9+P+G+L (M9+Pseudomonas+Glucose+Lactose) there is no shift in the growth curve of Pseudomonas in presence of Glucose and Lactose. That means there is no Glucose repression. Further reading of paper confirmed us that there is no Glucose repression for Pseudomonas putida.

We tried to find a new method to measure the growth of strains in liquid media in the presence of Quercetin by measuring absorbance at 600nm. We tried to solubilize Quercetin in mineral oil and make an emulsion with a two phase solution: Quercetin in the oily phase, Strains in the water phase.  As you can see, when we begin to shake the emulsion, the separation in two phase is not perfect and Quercetin pass through the aqueous phase.

We made some plating at different time: time0, time2, time4. The idea was to explain the absence of bacteria in our experiments. Indeed, when we were making theses experiments, we always had at the end fungi at the end. But we should have also some bacteria because bibliography shows that some bacteria are able to degrade Quercetin. By plating at different time we hope to see the difference of diversity of microbes in the plate.  Unfortunately, we were not able to observe some bacteria on Quercetin agar plate. In fact, Quercetin is green and it’s hide the bacteria that are growing on the plate. During two days we didn’t see any growth of microbes and at day 3, fungi appears. We have to keep in mind also that fungi, thanks to micellium are able to search for nutrients in the plate while bacteria are not. There is a sort of competition between bacteria and fungi for the use of Quercetin.
Above: two fungi isolated on M9 Quercetin agar plate. We can easily see a clear halo that mean the Quercetin was removed from the plate by the fungi. 
<H3>Result of the third assey</H3><P class="input">
A fungi was isolated from the plate M9Q 10^(-3) dilution and another from M9Q glucose 10^(-2) plate. 
Strains labelling and conservation
The strains were given a database name and saved at -80°C in 30% glycerol
•	FS_F1 : from cochin M9 quercetin plate 10^(-1) dilution
•	FS_F2 : from cochin M9 quercetin plate 10^(-2) dilution
•	FS_F3 : from cochin M9 glucose quercetin plate 10^(-1) dilution
•	FS_F4 : from cochin M9 glucose quercetin plate 10^(-2) dilution
•	FS_F5 : from Croatia M9 glucose quercetin plate 10^(-2) dilution
•	FS_F6 : from cochin M9 tube plate 10^(-1) dilution
•	FS_F7 : from cochin M9 tube 10^(-2) dilution
•	FS_F8 : from cochin M9 tube 10^(-1) dilution
•	FS_F9 : from cochin M9 tube 10^(-2) dilution
•	FS_F10 : from Croatia M9 glucose quercetin plate 10^(-2) dillution</P><H2 class="red">Week 5th-11th September</H2><H3>Quercetin degradation assey </H3><H3> Separation in two project </H3>
	At this point we really started to work on the first approach, and we combined it to the second one.
The goal of the first approachis making a huge library of strains that were isolated from soil sample. 
	We would like, from a given soil sample, to isolate as much of bacterial diversity as we can. 
	Then we could test each strain on Quercetin liquid media and report Quercetin degradation. 
	Thanks to our collaboration with other iGEM team, we got samples from mant places around the world. 
	At the end we would have a library with some strains that are able to degrade Quercetin and some others that are not. 
	We could quantify the strength of the degradation and also make a link between the species that are able to degrade the Quercetin to make hypothesis about a common enzyme responsible for this degradation, through the creation of a phylogenetic tree.
	
	To achieve that, here are the step of the protocol:
	<OL><LI>We plate the soil samples at 10^-2 and 10-3 dilutions (we choose these dilutions thanks to previous experiments) on non-selective agar media (LB, TSA, M9Glucose, M9) but also on selective media (FTO for micrococcus, Agar from Mossel for Bacillus Cereus, M9 Quercetin).</LI><LI>We then incubate for one or two days at 30°C.</LI><LI>When we have some colonies, we streak some that seems differents on LB or TSA plate in order to easy their growth. Once the streak bacteria grew enough, we give them a name to store them in the library.</LI>
For instance, FS_M69 stand for Frank&amp;Stain_Microbesgroup n°=69
	For a given strain we have the date of the stock, the species once its sequenced, the original agar media from where it was taken, the origin of the strain etc…
	<LI>Then we make an overnight culture in Tryptic Soy Broth and systematically, we characterise our strain with a 16s rRNA PCR from colonies streaked on non-selective media plates.</LI><LI> When the overnight culture is over, we inoculate 100ul of it directly in a 5ml M9 1g/L Quercetin falcon tube in triplicate. </LI><LI> We also make two glycerol stocks: One at -80°C and the other at -20°C (to be reused more easily).</LI></OL>
	For the assay on quercitin liquid media, we are quantifying Quercetin degradation. 
	To see if there was a degradation, we need to use a negative and a positive control.
	The negative control is just M9 + Quercetin. The positive control is M9 + Quercetin + Pseudomonas Putida. 
	Indeed, some article show that Pseudomonas is a good candidate for Quercetin degradation. The iGEM Evry team gave us this strain to use it for our assay.
	We made an experiment to check how long does pseudomonas needs to degrade Quercetin.
	We found that 4 days are enough to see a significant decrease in Quercetin absorbance. But to give a chance to every strains we will assay, we decided to make a 6 days experiment.
	<H2 class="red">Week 12th-18th September</H2>
	We designed the experiment with a Quercetin quantification at Time 0 and at Time 6 to see the evolution of Quercetin concentration. 
	
	We used Pseudomonas Putida K2440 given from Evry as a positive control. 
	We made two quantifications:
<OL><LI> At T0 to make sure that each tube had the same amount of Quercetin.</LI><LI>And at T6 because it’s enough to see a significant decrease in Quercetin concentration if there is a strain able to use it as a carbon source.</LI></OL>
All culture were made in triplicate.
To present data, we made a ratio of the absorbance at T6 over the absorbance at T0 for every culture. Then we average the three values. And we are able to calculate the standard deviation.
The values in y axis are a percentage of remaining Quercetin. 
	
Here you can see the T6 sample. We took 100µl from the 5ml culture after vortexing it and we put it in a eppendorf tube. Then we add 900µl of NaOH 0.5mM solution at pH=12.  You can see the difference on each tube depending of the % of Quercetin remaining.
  As you can see, none of the strains are able to degrade Quercetin except our positive control.
The issue is that these strains were collected from M9 Quercetin plates, so at least we were expecting some Quercetin degradation.
We suggest that it was a problem with the dilution factor. Indeed 10^-3 is maybe not enough. The other problem is that it is quite hard to streak small and isolated colonies from these plates, because of the quercetin green color on plate. Thus, we are just able to see big colonies.
	The other important point is that some strains are not able to use Quercetin as a carbon source to grow. These strains won’t be selected with this technique using a single source carbon. But these strains maybe able to degrade Quercetin as a substrate for other metabolism. 
	
Here are the first 16S PCR we did for strains 52 to 104. We have to notice that 16S rRNA is present in every bacteria, which means that this PCR is really sensitive : if there is contamination of the strain, usually e.coli in a lab, it's 16s rRNA can be amplified instead of the desired strain's. Knowing that, we did numerous negative control to ensure that our PCR were pure, and that no contamination interfere with our sequencing procedure, following exactly the same PCR protocol but without adding bacteria to the mix.
<B>Thus, all the PCR were maid under the hood, which was anyway essential for our safety, as we were working with unknown bacteria isolated from soil, in order to identify them</B>
 All the PCR were made at the same time which explain why the negative control don't necessarily appear in the same gel of all their related PCR products. Of course, it is essential to check.
        
C stands for control : here, you can see that they are completely negative, which means no contamination should have interfered with our samples.
<H2 class="red">Week 19th-25th September</H2>
For the next assay, we chose to increase the amount of cells in the media.
	We inoculate 500 µl instead of 100 µl of overnight culture in 5 ml of M9 Quercetin liquid media.
	Also, concerning plating, we noticed that we most of the time had more or less 4 species of bacteria on non-selective agar. That is annoying because our aim is to have a maximum of bacterial diversity for the database. 
	Yellow plots are some contamination with some fungi in it. But we have at least 3 strains that seems to be very efficient in the degradation.
	We took these strains and we plate them from the liquid media to some agar plate and then we re sequenced them to make sure they are the same as those previously tested.
	
        So as shows this histogram, we have some good results with E.coli K12 (a strain known to degrade Quercetin but in a different pathway as Pseudomonas Putida. 
        The strains B1 and B2 are strains that were taken from Anthocyanin degradation assay.
        The strains number 75 seems to be very promising.
         
The inoculation was the same as last assay (500ul of overnight culture).  The important thing to notice is that the strains 96, 97, 98, 99, 102, 103, 104 were taken from M9 Quercetin Plate.  We were expecting a degradation...

One reason of this failure maybe that the overnight culture were not enough fresh. Indeed, we had some troubles to organize ourself to test so many strains.
Some overnight culture were 5 days old when we assayed them.
In fact, our 6 incubators are all busy because of the assay, so we have to wait until one is free to begin the assay. We will organize ourself better next time to avoid this kind of problem.

For this assay we had one contamination and also no degradation from the other strains. 
In this experiment we tested also the strains from F1 to F8 that were isolated on Quercetin plate. As you can see, only F4 give a good result with the liquid experiment.
Strains 118 to 136 come from non selective plate (LB agar, Tryptic Soy Agar, M9 agar)

For this experiment, we tested also 5 E. coli strains that were designed by the Protein group to express some enzyme responsible for Anthocyane degradation. We had no results with these strains but there is maybe a problem of enzyme secretion. We should restart the experiment with cell extract.<H2 class="red">Week 26th September-2th October</H2>
For this experiments, the strains come from Australia. We isolated them on non selective agar plate.
<H3>Fungal identification</H3>       
The standard sequence used to identify fungi is the ITS sequence. Amplify it we need to purify fungal DNA then do a PCR with ITS 1 and ITS 4 primer.
Fungal DNA purification 
We tried to purify fungal DNA, the experiment failed because at a moment open tubes fall and the fungal DNA was lost. 
<H2 class="red">Week 9th-16th October</H2>
We isolated a lot of Pseudomonas from strains 201 to 222 that's why we had that much degradation in Quercetin.
Finally with the 9 experiments we made to assay 187 strains, we were able to produce this histogramme. 
To be more clear and avoid confusion with the name of the strains we renamed all the strains we have tested on Quercetin degradation. We named them with a S.number if the strain was isolated on a Selective media (M9 + Quercitin agar). We named them NS.number is the strain was isolated on a Non Selective media. 

In this histogram, the strains are rated by chronological order. At the beginning of our experiments, we had only strains isolated on Quercetin that worked on Quercetin degradation. But then we tested more and more NS strains and we had more positiv results, especially in the end of our experiments when we tested on Australia sample and we had lot of Pseudomonas.
<H3>Fungal identification</H3>       

The fungal DNA was purified. We amplified the ITS region in PCR. The gel electrophoresis showed the sequence in all the strains except the FS_F7 and the negative control. 

The DNA concentration was quantified  : 

-negative control : 5 ng/µ
-FS_F1 : 40,4 ng/µL
-FS_F2 : 66,3 ng/µL
-FS_F3 : 62,0 ng/µL
-FS_F4 : 53,0 ng/µL
-FS_F5 : 35,2 ng/µL
-FS_F6 : 23,6 ng/µL
-FS_F8 : 30,0 ng/µL
-FS_F9 : 68,9 ng/µL
-FS_F10: 45,3 ng/µL
As all the concentration were between 20 and 80 ng/µL the DNA was sent to GATC for sequencing
<H2 class="red">Week 3d October-8th October</H2><H3>Fungal identification</H3>       

The fungi were identified : 
-FS_F1 : Aspergillus fumigatus
-FS_F2 : Aspergillus fumigatus
-FS_F3 : Aspergillus fumigatus
-FS_F4 : Fusarium solani
-FS_F5 : Aspergillus alliaceus
-FS_F6 : Aspergillus fumigatus
-FS_F7 : no identification
-FS_F8 : Aspergillus fumigatus
-FS_F9 : Aspergillus fumigatus
-FS_F10 : Aspergillus niger
So we had 4 different fungi, 6 of our fungi were A.fumigatus, it is to be noted that these 6 fungi all come from the same soil sample. 
<H3>Quercetin degradation assey</H3>

As the standard error bars in the first quercetin degradation assey were really bad we decided to test another time the quercetin degradation after 4 days. 
We did a first measure after 2 days  : 

We noted that A.niger was able to degrade quercetin very fast. 
 
We were able to see a difference in quercetin degradation by the different fungi. We were also able to see a difference between the fungi and P.putida. 
We wanted to have a picture of each of the fungi in plates so a little ball of mycelium was inoculated on quercetin plates for 3 days 

We can see the plate becoming transparent where the mycelium have grown.

<H2 class="red">Week 17th-19th October</H2>
We made some kinetic of the most promising strains to see the evolution of Quercetin degradation. All the culture were made in triplicate.
 As you can see, there is some variations in the absorbance that makes the curve non linear.
Here we chose not show the standard deviation to make the graph more clear.
 To make the graph easier to understand, we chosed to normalized every daily absorbance by the control: Abs(strains)Tx/AbsTx(control).  Thanks to that we get an easy graph to read.
On the Y axis, we can read the % of Quercetin remaining.
We sent to genome sequencing the strains FS_M75(Lysinibacillus), B2(Stenothrophomonas maltophilia), B3(Oerskovia Paurometabola), FS_M121(Microccocus Luteus). 
We would like to compare, between theses species, the enzyme commonly responsible for Quercetin degradation.fimage


To finish our project, we assayed two strains FS_M75 and Pseudomonas putida with real wine stain on cotton fabric. 

The first experiment was made by putting a circle of cotton into a beaker with wine and then drying it by waiting a few hours. We had some cells that were in a transparent media composed of M9. This M9 is composed by cells without LB (LB have been removed by centrifugation, cells were re-suspended in PBS).
And we have fabric in M9 without strains and the piece of cotton with the wine stain.
As you can see, the results are not evidential. It seems that our negative control wash the wine stain in an as efficient way as the two other strains!

We redo the experiment but we waited three days until the cotton was enough stained. Then we autoclaved the cotton to make sure there is no others microorganisms. We used the same experimental conditions as before.


We made also an experiment with the platform the assay group designed. 

We made three different dilution and we tested once again the ability of Pseudomonas and FS_M75 to degrade wine stain at 30°C.


We used Geneious to generate consensus sequences from each bacteria isolated and sequenced, through de novo alignement between the two chromatogram files (.ab1) provided vy GATC for each sequencing. 
Thus, we were able to increase the quality of our sequences, to make them more reliable. We also trimmed them when the quality was too low.
Because of bad quality, or errors in sequencing, some strains of the 187 database were deleted.
Which explain why only 173 bacteria were used to create the phylogenetic tree.
We ran the tree thanks to Geneious software with the following parameters:
<OL><LI> The Tree build model chosen was neighbor joining. </LI><LI> Jukes-Cantor was chosen as the genetic distance model </LI><LI> NR_102450 , a cyanobacteria, was chosen as the outgroup </LI><LI>We chose a cost matrix for alignment of 93% similarity (the highest proposed by Geneious)</LI></OL><B>Pay attention to put every sequences in the same direction (using the Reverse complementary option), as Geneious don't reverse them automatically !  If you forget to do that, tou will have actually 2 similar trees: on assembling the forward sequences, the other one with the reverse ones.</B>
Here is the first phylogenetic tree we were able to get. The bold strains with an * are the one that were considered as the most able to degrade quercetin, according to the previous assay. The taxonomy is summed up in the legend.<H3>A.niger kinteics</H3>       

A.niger degradation kinetics 
As A.niger was able to degrade all the quercetin in 48h we decided to measure precidly the degradation kinetics. We followed the quercetin degradation for 32h (almost all the quercetin was degraded

We could clearly see the exponential aspect of the curb. Almost all the quercetin degradation happened between 16h and 29h. We calculated the degreadation speed between these two moments :
V=(C(17)-C(29))/(29-17) = 0,8g/h
This value is important to study the impact of a gene on the A.niger’s ability to degrade quercetin. If we had more time we would have done this kind of kinetics on all the fungi to compare their ability to degrade quercetin. 
	</BT></DIV></DIV><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:igem2016parisbettencourt@gmail.com">igem2016parisbettencourt@gmail.com</A><CENTER><A href="https://2016.igem.org/">
    2016.igem.org
</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>