Team:Paris Bettencourt/Notebook/Binding
Project
Overview
Assay
Microbiology
Enzyme
Binding
Indigo
Model
Practices
About Perc
Dry Cleaners
Design
Events
Safety
Lab Safety
Product
Safety
Achievements
Results
Interlab Study
MedalRequirements
Parts
Notebook
Daily Note
Protocols
Bibliography
Team
Our Team
Attributions
Collaborations
Sponsors
@iGEM_Paris
Binding Domains Group Notebook - iGEM 2016 Paris Bettencourt
Notebook
Protocols
Bibliography
Assay
Microbiology
Enzyme search
Mission Indigo
Binding Domains
Anthocyanin
Introduction
The Binding Domain Group aims at finding novel peptides that can bind tightly and specifically to a particular fabric. We hope to improve the efficiency of enzymes tested by the Enzymes search group by localizing and stabilizing the enzymes at the sites of action (fabric) by fusing the peptide sequences we find to those enzymes. To achieve this goal we are using phage display technique.
Phage display describes a selection technique in which phages with genetic material encoding variants of peptide sequences express these peptides on the protein coat. And based on the binding affinity of these peptides to a given target molecule by an in vitro selection process called panning, selective phages can be separated and enriched. We adapted the NEB protocol to accommodate our target, so panning is carried out by incubating a library of phage-displayed peptides (2*10^11 phages) with our fabric of choice in a micro centrifuge tube, washing away the unbound phage, and eluting the specifically bound phage. The eluted phage are then amplified and taken through additional binding/amplification cycles to enrich the pool in favor of binding se¬quences. After 3–4 rounds of panning, individual clones will be characterized by DNA sequenc¬ing and later ELISA.
The peptide library we received from NEB has 10^9 distinct clones. The M13 phages are modified for a pentavalent display of peptides as N-terminal fusions to the minor coat protein pIII. This protein modulates phage infectivity by binding to the F-pilus of the recipient bacterial cell. The E. coli host strain used is ER2738 (F´ proA+B+ lacIq Δ(lacZ)M15 zzf::Tn10(TetR)/fhuA2 glnV Δ(lac-proAB) thi-1 Δ(hsdS-mcrB)5. [rk– mk– McrBC–]), which is a robust F+ strain with a rapid growth rate and is supE (GlnV) in order to suppress amber (UAG) stop codons within the library with glutamine.
In summary, the main focus of our group is to find novel peptide sequences that specifically bind to different fabrics. Later, collaborate with other groups to optimize the efficiency of the potential enzymes they find on wine stained fabrics using our peptide sequences.
Week 11th - 17th July
NEB Ph.D.™-7 Phage Display Peptide Library Kit solutions preparation
NEB protocol recommends to prepare a mixed solution of ITPG and X-gal. Because I already have IPTG stock, I prefer to do 2 separate solution for IPTG and X-gal (also avoids throwing both IPTG and X-gal if anything were to happen to the tubes).
IPTG
M.W.= 238.31g/mol
NEB protocol recommends to put 1.25g in 25mL DMF, which is [IPTG]=50mg/mL
[IPTG]mass=[IPTG]/MW=0.05/238.31=0.21mol/L=210mM.
They recommend diluting 1000-fold to prepare plates. So [IPTG]working_concentration=50µg/mL which is 210µM.
In order to simplify things, I'll use [IPTG]working_concentration=200µM.
IPTG stocks are at 1M and 0.1M (100mM).
If preparing 1M stock, add 2.383g of IPTG to 10mL of milli-Q water. For 0.1M, add 238.3mg of IPTG to 10mL of milli-Q water.
2µL IPTG / mL medium should be added if using the 0.1M solution (0.2µL / mL of using the 1M solution).
X-gal
M.W.= 408.63g/mol
NEB protocol recommends to put 1g in 25mL DMF, which is [X-gal]=40mg/mL.
Sigma-Aldrich recommends having stock solution at 20mg/mL.
I decided to do a stock at 20mg/mL.
2 µL X-gal / mL of medium should be added.
Tetracycline
NEB protocol recommends doing a 20mg/mL stock solution in 50% EtOH. Most of the protocol found online recommend 70% EtOH, so that's what I did (it probably does not matter much).
300mg were dissolved in 15mL 70% EtOH.
TBS
NEB protocol recommends 50mM Tris-HCl (pH7.5), 150mM NaCl.
Tris-HCl 1M, pH 7.5:
M.W. Tris-Base=121.14g/mol
12.11g of Tris-Base were dissolved in 60mL of H2O.
pH was adjusted to 7.5 using HCl 5M.
Volume was adjusted to 100mL.
25 mL of Tris-HCl 1M, pH 7.5 were diluted in 500mL H2O.
M.W. NaCl=58.44g/mol
mNaCl=(58.44/1000)*150=8.766g for 1L
So 8.766/2=4.383g were added to the solution of Tris-HCl 50mM.
pH was measured after completion of the solution and was around 7.7.
Blocking Buffer
M.W. NaHCO3=84.007g/mol
2.1g of NaHCO3 were put in 225mL osmosed H2O.
pH was adjusted to 8.6 with 5M HCl.
Volume was adjusted to 250mL.
1.25 BSA was added to final concentration 5mg/mL.
Solution was kept 6 days at 4°C.
On 21/07/16, 0.05g NaN3 (0.02%) were added to the solution to avoid growth of microorganisms.
Solution was filter sterilized and aliquoted by ~75mL.
PEG 20%/NaCl 2.M
Put 20g of polyethylene glycol–8000 in 100mL distilled H2O.
M.W. NaCl=58.44g/mol.
In 100mL PEG 20%, add 14.61g NaCl.
Autoclave, mix well to combine separated layers while still warm. Store at room temperature.
Top Agar
For 1L:
10g Bacto-Tryptone
5g yeast extract
5g NaCl
7g Bacto Agar (or electrophoresis grade agarose)
Autoclave.
Dispose in 50mL aliquots.
Week 18th - 24th July
M13KE Phage Titering
In order to train ourselves with the phage titration that we will need to perform when doing the phage display from the NEB kit, we decided to do a phage titration using M13KE. The following protocol is the one given by NEB with their M13KE phages.
The number of plaques will increase linearly with added phage only when the multiplicity of infection (MOI) is much less than 1 (i.e., cells are in considerable excess).
Note: We used MGZ1 F+ to perform this titration
Inoculate 10 ml of LB with both MGZ1 F+ from a plate and incubate with shak­ing 4–6 hrs (mid-log phase, OD600 ~ 0.5).
Media was inoculated at 8h56.
Time after inoculation
OD 600nm
3h14
0.0584
4h11
0.2717
4h28
0.4022
4h28
0.5644
While cells are growing, melt Top Agar in microwave and dispense 3 ml into sterile culture tubes, one per expected phage dilution. Maintain tubes at 45°C.
Pre-warm, for at least one hour, one LB/IPTG/Xgal plate per expected dilu­tion at 37°C until ready for use.
Prepare 10 to 1000-fold serial dilutions of phage in LB; 1 ml final volumes are convenient. Suggested dilution ranges: for amplified phage culture super­natants, 10^8 –10^11 ; for unamplified panning eluates, 10^1–10^4. Use aerosol-resistant pipette tips to prevent cross-contamination, and use a fresh pipette tip for each dilution.
As I expect to have 10^11 pfu/mL after amplification and that they recommend 10^8 to 10^11 dilutions from that, beginning from M13KE tube (10^13 pfu/mL), I used the following dilutions: 10^10, 10^11, 10^12 and 10^13 (shift of 2 order of magnitude).
When the culture in Step 1 reaches mid-log phase, dispense 200 μl into microfuge tubes, one for each phage dilution.
To carry out infection, add 10 μl of each phage dilution to each tube, vortex quickly, and incubate at room temperature for 1–5 minutes. Here cells were incubated with the phages 3 to 4 minutes.
Transfer the infected cells one infection at a time to culture tubes containing 45°C Top Agar. Vortex briefly and IMMEDIATELY pour culture onto a pre-warmed LB/IPTG/Xgal plate. Gently tilt and rotate plate to spread top agar evenly.
Allow the plates to cool for 5 minutes, invert, and incubate overnight at 37°C.
Count plaques on plates that have approximately 100 plaques. Multiply each number by the dilution factor for that plate to get phage titer in plaque form­ing units (pfu) per 10 μl.
Results:
Dilution
Number of plaques
10^10
25
10^11
16
10^12
26
10^13
16
M13KE phage amplification
This phage amplification protocol is the one given by NEB.
Inoculate a 20 ml culture in a 250 ml Erlenmyer flask with 200 µL overnight E. coli culture. Add 1 µL phage suspension. Shake flask at 37°C, 150 rpm for 4 -5 hrs.
Remove cells by centrifugation at 4500 g for 10 min. Transfer supernatant to a fresh tube. Repeat centrifugation.
Transfer top 16 ml of supernatant to a new tube and add 4 mL of 2.5 M NaCl/20 % PEG-8000 (w/v). Briefly mix. Precipitate phage for 1 hr or overnight at 4°C.
Pellet phage by centrifugation at 12000 g for 15 min. 55 min at 4000 rpm (Rotor radius: 195 mm, 4000 rpm -> ~3400 g, 12000/~3400=3.5, 3.5*15 min=52.5 min) used was done instead due to problems with centrifuge. Decant supernatant. Resuspend pellet in 1 mL TBS. Transfer to an eppendorf tube. Spin briefly to remove any cell debris.
Transfer supernatant to a fresh tube. Add 200 µL of 2.5 M NaCl/20% PEG-8000. Incubate on ice for 60 min. Spin 14000 rpm in a benchtop centrifuge for 10 min. Discard supernatant. Spin again briefly and remove remaining supernatant with pipette. Resuspend pellet in 200 µL TBS.
Week 25th - 31th July
NEB Phage Display on Cotton - DAY 1
Cotton used: Khadi (from Khadi and Co - Bess Nielsen, given by Teja) - Hand Woven - 100% cotton.
A square of 1cm x 1cm was used (total surface: 2 cm² counting both sides).
Inoculate 10 ml of LB+Tet medium with ER2738. This culture will be used for titering in Step 8 and can be used in ~4 hours. Incubate at 37°C with vigorous shaking. Incubate the titering culture until needed.
Incubate for at least 1 hour at 4°C a piece of fabric with blocking buffer.
Remove the piece of fabric from the blocking solution. Wash the fabric rapidly 10 times with TBST (TBS + 0.1% [v/v] Tween-20) by dipping the fabric in a beaker filled with TBST and going back and forth through the washing solution.
Dilute a 25-fold representation of the library (e.g., 2 x 10 11 phage for a library with 2 x 10 9 clones) with 250µl of TBST (therefore add 10µL of phage library). Pipette onto piece of fabric in Eppendorf tube and rock gently for 60 minutes at room temperature.
Rem: The square shape of the piece of fabric does not seem best suited for the incubaction with the phages  as the sides seem not to be immersed in the phage library solution. It is probably better to use a rectangle shape thereafter so that the piece of fabric fits nicely into the tube and is fully immersed.
Remove piece of fabric from the Eppendorf tube.
Wash fabric 20 times with TBST as in step 3.
Elute bound phage with 1 ml of an ap­propriate elution buffer for the interaction being studied. A general buffer for nonspecific disruption of binding interactions is 0.2 M Glycine-HCl (pH 2.2), 1 mg/ml BSA. Rock gently for 15 minutes. Discard the piece of fabric (save it in TBST). Neutralize supernatant with 150 μl of 1 M Tris-HCl, pH 9.1.
Inoculate 20 mL of LB medium in a 250-ml Erlenmeyer flask with ER2738. Incubate culture at 37°C with vigorous shaking. Carefully, monitor the 20-ml culture so that it does not grow beyond early-log phase (OD600 0.01–0.05), for use in Step 10.
Titer a small amount (~2 μl) of the eluate as described in General M13 titration protocol. Plaques from the first or second round eluate titering can be sequenced if desired.
Rem: As this eluate is not amplified, dilutions used for titration are as follow: 10, 100, 1000, 10000.
Amplify the rest of the eluate by adding the eluate to the 20-ml ER2738 culture from Step 1 (should be early-log at this point) and incubating with vigorous shaking for 4.5 hours at 37°C.
Transfer the culture to a centrifuge tube and spin for 10 minutes at 12,000 g at 4°C. Transfer the supernatant to a fresh tube and re-spin (discard the pellet).
Transfer the upper 80% (here 12.8mL) of the supernatant to a fresh tube and add to it 1/6 volume of 20% PEG/2.5 M NaCl (here ~2.1mL). Allow the phage to precipitate at 4°C for at least 2 hours, preferably overnight.
Time after inoculation - Titration culture
OD 600nm - Titration culture
Time after inoculation - Amplification culture
OD 600nm - 20mL Amplification culture
0
0
0
0
3h30
0.3811
0h40
0.008/0.006/0.009
3h40
0.5101
1h50
0.0116
2h25
0.0193
NEB Phage Display on Cotton - DAY 2
Results titration unamplified phage eluate 1st round:
Dilution
Number of plaques
10
>1000
100
230
1000
17
10000
3
Therefore, we have a phage concentration of 230x(10^2)x(10^2) = 2.3 x 10^6 pfu/mL.
Inoculate 10mL of LB+Tet medium with ER2738 for titration.
Spin the PEG precipitation at 12,000 g for 15 minutes at 4°C. Decant and discard the supernatant, re-spin the tube briefly, and remove residual supernatant with a pipette. The phage pellet should be a white finger print sized smear on the side of the tube.
Suspend the pellet in 1 ml of TBS. Transfer the suspension to a micro­centrifuge tube and spin at maximum (14,000 rpm) for 5 minutes at 4°C to pellet residual cells.
Transfer the supernatant to a fresh microcentrifuge tube and reprecipitate by adding 1/6 volume of 20% PEG/2.5 M NaCl (here 166µL). Incubate on ice for 15–60 minutes. Microcentrifuge at 14,000 rpm for 10 minutes at 4°C, discard the supernatant, re-spin briefly, and remove residual supernatant with a micropipet.
Suspend the pellet in 200 μl of TBS. Microcentrifuge for 1 minute to pel­let any remaining insoluble material. Transfer the supernatant to a fresh tube. This is the amplified eluate.
Titer the amplified eluate on LB/IPTG/Xgal plates following the general M13 titration protocol. The eluate can be stored for up to 3 weeks at 4°C. For long-term storage, add an equal volume of sterile glycerol and store at –20° C.
OD measurements:
Time after inoculation
OD 600nm
0
0
3h08
0.3167
3h21
0.4415
3h25
0.4955
Incubate overnight at 4°C a new piece of fabric (both a 1 cm x 1cm square and 0.5 cm x 2 cm were incubated) in 1 mL of Blocking Buffer for the 2 nd round of panning.
Week 01st - 07 August
NEB Phage Display on Cotton - DAY 3:
Count blue plaques from the titering plates in Step 17 and determine the phage titer, which should be on the order of 1013–14 pfu/ml. Use this value to calculate an input volume corresponding to the input titer in Step 4. If the phage titer of the amplified eluate is too low, succeeding rounds of panning can be carried out with as little as 109 pfu of input phage.
Results titration amplified phage eluate 1st round:
Dilution
Number of plaques
10^8
>1000
10^9
92
10^10
8
10^11
1
Therefore, we have a phage concentration of 92x(10^9)x(10^2) = 9.2 x 10^12 pfu/mL.
Inoculate 10 ml of LB+Tet medium with ER2738. This culture will be used for titering in Step 9 and can be used in ~4 hours. Incubate at 37°C with vigorous shaking. Incubate the titering culture until needed.
Remove the piece of fabric from the blocking solution. Wash the fabric rapidly 10 times with TBST (TBS + 0.5% [v/v] Tween-20) by dipping the fabric in a beaker filled with TBST and going back and forth through the washing solution.
Dilute a 11.5-fold representation of the library (e.g., 2 x 10 11 phage for a library with 2 x 10 9 clones) with 250µl of TBST (therefore add 10µL of phage library). Pipette onto piece of fabric in Eppendorf tube and rock gently for 60 minutes at room temperature.
Rem: The square shape of the piece of fabric does not seem best suited for the incubaction with the phages  as the sides seem not to be immersed in the phage library solution. It is probably better to use a rectangle shape thereafter so that the piece of fabric fits nicely into the tube and is fully immersed.
Remove piece of fabric from the Eppendorf tube.
Wash fabric 20 times with TBST as in step 3.
Elute bound phage with 1 ml of an ap­propriate elution buffer for the interaction being studied. A general buffer for nonspecific disruption of binding interactions is 0.2 M Glycine-HCl (pH 2.2), 1 mg/ml BSA. Rock gently for 15 minutes. Discard the piece of fabric (save it in TBST). Neutralize supernatant with 150 μl of 1 M Tris-HCl, pH 9.1.
Inoculate 20 mL of LB medium in a 250-ml Erlenmeyer flask with ER2738. Incubate culture at 37°C with vigorous shaking. Carefully, monitor the 20-ml culture so that it does not grow beyond early-log phase (OD600 0.01–0.05), for use in Step 10.
Titer a small amount (~2 μl) of the eluate on LB/IPTG/Xgal following general M13 titration protocol. Plaques from the first or second round eluate titering can be sequenced if desired.
OD measuerments:
Time after inoculation
OD 600nm
0
0
3h33
0.1130
4h03
0.2107
4h35
0.3251
4h50
0.5356
Amplify the rest of the eluate by adding the eluate to the 20-ml ER2738 culture from Step 1 (should be early-log at this point) and incubating with vigorous shaking for 4.5 hours at 37°C.
OD measuerments:
Time after inoculation
OD 600nm
0
0
1h01
0
1h33
0.006
2h06
0.0388
Transfer the culture to a centrifuge tube and spin for 10 minutes at 12,000 g at 4°C. Transfer the supernatant to a fresh tube and re-spin (discard the pellet).
Transfer the upper 80% (here 14.4mL) of the supernatant to a fresh tube and add to it 1/6 volume of 20% PEG/2.5 M NaCl (here 2.4mL). Allow the phage to precipitate at 4°C for at least 2 hours, preferably overnight.
NEB Phage Display on Cotton - DAY 4
Results titration unamplified phage eluate 2nd round:
Dilution
Number of plaques
10
>1000
100
788
1000
38
10000
10
Therefore, we have a phage concentration of 38x(10^3)x(10^2) = 3.8 x 10^6 pfu/mL
Inoculate 10mL of LB+Tet medium with ER2738 for titration.
Spin the PEG precipitation at 12,000 g for 15 minutes at 4°C. Decant and discard the supernatant, re-spin the tube briefly, and remove residual supernatant with a pipette. The phage pellet should be a white finger print sized smear on the side of the tube.
Suspend the pellet in 1 ml of TBS. Transfer the suspension to a micro­centrifuge tube and spin at maximum (14,000 rpm) for 5 minutes at 4°C to pellet residual cells.
Transfer the supernatant to a fresh microcentrifuge tube and reprecipitate by adding 1/6 volume of 20% PEG/2.5 M NaCl (here 166µL). Incubate on ice for 15–60 minutes. Microcentrifuge at 14,000 rpm for 10 minutes at 4°C, discard the supernatant, re-spin briefly, and remove residual supernatant with a micropipet.
Suspend the pellet in 200 μl of TBS. Microcentrifuge for 1 minute to pel­let any remaining insoluble material. Transfer the supernatant to a fresh tube. This is the amplified eluate.
Titer the amplified eluate on LB/IPTG/Xgal plates following the general M13 titration protocol. The eluate can be stored for up to 3 weeks at 4°C. For long-term storage, add an equal volume of sterile glycerol and store at –20° C.
OD measurements:
Time after inoculation
OD 600nm
0
0
3h38
1.032 => 10-fold dilution
4h18
0.2009
4h42
0.42
4h48
0.5094
NEB Phage Display on Cotton - DAY 5
Count blue plaques from the titering plates in Step 17 and determine the phage titer, which should be on the order of 1013–14 pfu/ml. Use this value to calculate an input volume corresponding to the input titer in Step 4. If the phage titer of the amplified eluate is too low, succeeding rounds of panning can be carried out with as little as 109 pfu of input phage.
Results titration amplified phage eluate 1st round:
Dilution
Number of plaques
10^8
>1000
10^9
277
10^10
40
10^11
4
Therefore, we have a phage concentration of (277x(10^9)x(10^2)+40x(10^10)x(10^2))/2 = 3.36 x 10^13 pfu/mL
Inoculate 10 ml of LB+Tet medium with ER2738. This culture will be used for titering in Step 8 and can be used in ~4 hours. Incubate at 37°C with vigorous shaking. Incubate the titering culture until needed.
Remove the piece of fabric from the blocking solution. Wash the fabric rapidly 10 times with TBST (TBS + 0.5% [v/v] Tween-20) by dipping the fabric in a beaker filled with TBST and going back and forth through the washing solution.
Dilute a 168-fold representation of the library with 250µl of TBST (therefore add 1.5µL of phage library to 248.5µL TBST). Pipette onto piece of fabric in Eppendorf tube and rock gently for 60 minutes at room temperature.
Remove piece of fabric from the Eppendorf tube.
Wash fabric 20 times with TBST as in step 3.
Elute bound phage with 1 ml of an ap­propriate elution buffer for the interaction being studied. A general buffer for nonspecific disruption of binding interactions is 0.2 M Glycine-HCl (pH 2.2), 1 mg/ml BSA. Rock gently for 15 minutes. Discard the piece of fabric (save it in TBST). Neutralize supernatant with 150 μl of 1 M Tris-HCl, pH 9.1.
Titer a small amount (~2 μl) of the eluate on LB/IPTG/Xgal following general M13 titration protocol. Plaques from the first or second round eluate titering can be sequenced if desired.
OD measuerments:
Time after inoculation
OD 600nm
0
0
3h16
0.2936
3h45
0.2936
4h02
0.4242
4h11
0.5237
It is not necessary to amplify the third round eluate. Plaques from this titering can be used for sequencing: time the procedure so that plates are incubated at 37°C for no longer than 18 hours, as deletions may occur if plates are grown longer. Stored at 4°C, plates should be used to pick plaques within 1–3 days of plating. The remaining eluate can be stored at 4°C for at least one week.
Plaque amplification Sequencing - Cotton
Selected phage clones can be identified by DNA sequencing, and target specificity can be confirmed by phage ELISA. In both cases it is necessary to amplify phage, either from individual plaques or from the eluted pool, to obtain sufficient quantities to work with.
Dilute an overnight culture of ER2738 1:100 in LB. Dispense 1 ml of diluted culture into culture tubes, one for each clone to be characterized. 10–20 clones from the third round are usually sufficient to detect a consensus binding sequence. 20 clones were selected.
Use a sterile wooden stick or pipette tip (tip used here) to stab a blue plaque from a titering plate (important: plates should be less than 1–3 days old, stored at 4°C and have less than 100 plaques) and transfer to a tube containing the diluted culture. Pick well-separated plaques. This will ensure that each plaque contains a single DNA sequence.
Incubate the tubes at 37°C with shaking for 4.5–5 hours (4h45').
Transfer the cultures to microcentrifuge tubes, and microfuge at 14,000 rpm for 30 seconds. Transfer the supernatant to a fresh tube and re-spin. Using a pipette, transfer the upper 80% of the supernatant to a fresh tube. This is the amplified phage stock and can be stored at 4°C for several weeks with little loss of titer. For long-term storage (up to several years), dilute 1:1 with sterile glycerol and store at –20°C.
Week 08th - 14th August
Purification of Phage DNA from Cotton Phage Display for sequencing
After the first centrifugation in Step 4 in the phage amplification protocol, transfer 500 μl of the phage-containing supernatant to a fresh microfuge tube.
Add 200 μl of 20% PEG/2.5 M NaCl. Invert several times to mix, and let stand for 10–20 minutes at room temperature.
Microfuge at 14,000 rpm for 10 minutes at 4°C and discard the supernatant. Phage pellet may not be visible.
Re-spin briefly. Carefully pipet away and discard any remaining supernatant.
Suspend the pellet thoroughly in 100 μl of Iodide Buffer by vigorously tap­ping the tube. Add 250 μl of ethanol and incubate 10–20 minutes at room temperature. Short incubation at room temperature will preferentially pre­cipitate single-stranded phage DNA, leaving most phage protein in solution.
Spin in a microfuge at 14,000 rpm for 10 minutes at 4°C, and discard the supernatant. Wash the pellet with 0.5 ml of 70% ethanol (stored at –20°C), re-spin, discard the supernatant, and dry pellet (here it was dried 1h at RT with ~15µL EtOH remaining after discarding supernatant).
Suspend the pellet in 30 μl of H2O. The template can be suspended in TE if desired, this is recommended for long-term storage. In TE buffer, the phage DNA should be stable indefinitely at –20°C.
Quantitate product by using a Nanodrop.
Note: Make sure to use single strand DNA measurement
Sample
Concentratio (ng/µL)
260/280 ratio
R3P1
52
ND
R3P2
57
ND
R3P3
14
ND
R3P4
68
2.46
R3P5
29
2.53
R3P6
61
2.00
R3P7
97
2.01
R3P8
61
1.97
R3P9
61
2.55
R3P10
41
2.16
R3P11
80
2.04
R3P12
64
2.04
R3P13
64
2.15
R3P14
66
2.02
R3P15
65
1.96
R3P16
44.5
2.05
R3P17
45
2.10
R3P18
50
2.00
R3P19
43
2.00
R3P20
53
1.90
Sequencing of phage DNA from NEB phage Display on Cotton
All samples were sent to sequencing by GATC Biotech.
Results are available below:
Sequences Phage R3P6 to R3P15
Sequences Phage R3P1 to R3P5 and R3P16 to R3P20
NEB Phage Display on Cotton - Finding a consensus sequence
Sequences were analysed using Geneious 9.1.5.
The random peptide library sequence was found between the linker (motif: 5'-ACC TCC ACC-3') and the end of the pIII leader sequence (motif: 5'-AGA GTG AGA-3'). We used only the -96 primer (see sequence below) to sequence as the -28 seemed too close to the library sequence for the sequencing to give a good result.
Some samples (R3P4, R3P13, R3P17) did not exhibit those motifs. Deletion?
One of the sequences was longer (22 nucleotides) and contained non-determined nucleotides (N).
Those four sequences were not used in the analysis.
The 21 nucleotides of the library of 16 remaining sequences were then translated.
Using the Multiple-Align function of Geneious and setting the threshold for the consensus sequence at 25% (amino acid present at least 25% of the time at a given position), we get the following consensus sequence:
NEB Phage Display on Polyester and Silk - DAY 1 (Part 1)
Silk used: Silk thread 1003 Au ver a Soie - Colour: Crème (Cream) - 100% Soie - Length: 50 cm - Weight: 16.5mg.
Polyester used: Polyester thread Mediac Ref: 960 - Colour: 400 - 100% Polyester - Length: 50cm - Weight: 16mg.
DAY 1 from Panning Procedure (see protocol page) was followed. Amplification was not started this day.
OD600 measurements of titration culture were as follow:
Time after inoculation
OD 600nm
3h45
0.4984 => dilution 200-fold
6h20
0.1111
6h52
0.3191
7h02
0.4210
7h07
0.4567
7h12
0.4885
10mL LB+Tet were inoculated with a single ER2738 colony at 37°C with shaking. This culture will be diluted 100-fold the next day to amplify the unamplified panning eluates.
NEB Phage Display on Polyester and Silk - DAY 1 (Part 2)
Results titration unamplified phage eluates 1st round:
Dilution
Number of plaques
Silk
Polyester
10
864
504
100
75
34
1000
16
3
10000
5
0
Therefore, the pfu of the unamplified phage eluates are:
Silk: 75x(10^2)x(10^2) = 7.5 x 10^5 pfu/mL
Polyester: 34x(10^2)x(10^2) = 3.4 x 10^5 pfu/mL
End of DAY 1 (steps 9-11) was performed.
Week 15th - 21st August
NEB Phage Display on Polyester and Silk - DAY 2
DAY 2 from Panning Procedure (see protocol page) was followed.
OD600 measurements of titration culture were as follow:
Time after inoculation
OD 600nm
2h51
0.3813
3h01
0.4980
NEB Phage Display on Polyester and Silk - DAY 3
Results titration amplified phage eluates 1st round:
Dilution
Number of plaques
Silk
Polyester
10^8
>1000
>1000
10^9
148
130
10^10
10
21
10^11
2
5
Therefore, the pfu of the amplified phage eluates are:
Silk: 148x(10^9)x(10^2) = 1.48 x 10^13 pfu/mL
Polyester: 130x(10^9)x(10^2) = 1.3 x 10^13 pfu/mL
DAY 3 from Panning Procedure (see protocol page) was followed.
Dilution of the amplified eluate for the second panning was done using the following dilution factors and volumes:
Silk: 74-fold dilution => ~3.38µL of amplified phage eluate added to ~246.5µL of TBST 0.5%
Polyester: 65-fold dilution => 3.85%L of amplified phage eluate added to ~246.5µL of TBST 0.5%
OD600 measurements of titration culture were as follow:
Time after inoculation
OD 600nm
2h38
0.3811
2h57
0.6282 => dilution 20-fold
3h35
0.0688
4h16
0.2626
4h34
0.4569
OD600nm of overnight culture was 4.833. This culture was therefore diluted 200-fold to use as amplification cultures.
NEB Phage Display on Polyester and Silk - DAY 4
Results titration unamplified phage eluates 2nd round:
Dilution
Number of plaques
Silk
Polyester
10
146
113
100
4
3
1000
2
1
10000
0
0
Therefore, the pfu of the unamplified phage eluates are:
Silk: 146x10x5x(10^2) = 7.3 x 10^5 pfu/mL
Polyester: 113x10x5x(10^2) = 5.65 x 10^5 pfu/mL
DAY 4 was performed.
OD600 measurements of titration culture were as follow:
Time after inoculation
OD 600nm
3h29
0.1474
3h53
0.3094
4h03
0.3862
4h09
0.4469
4h16
0.5200
NEB Phage Display on Wool and Linen - DAY 1
Wool used: Pure wool gauze twill fabric 100%wool 3.2 oz/linear yard- From Dharma trading.
Linen used: 100% bleached linen, 3.8 oz/linear yard- From Dharma trading.
A rectange of 0.5cm x 2cm was used (total surface: 2 cm² counting both sides).
Followed Day1 of the phage display protocol but some things to note -
During step7, I removed the fabric from the elution buffer and then I centrifuged the solution to remove the lint particles from the wool fabric and then neutralized the supernatant.
Note- Wool fabric used is loosely woven, so while washing it is coming apart, I did not have the same problem with linen.
Time after inoculation - Titration culture
OD 600nm
0
>0
2h 15
0.25
2h 35
0.43
2h 48
0.48
For amplification, 1:100 dilution of the overnight culture of ER2738 was used.
NEB Phage Display on Polyester and Silk - DAY 5
Results titration amplified phage eluates 2nd round:
Dilution
Number of plaques
Silk
Polyester
10^8
>1000
>1000
10^9
152
150
10^10
14
15
10^11
1
3
Therefore, the pfu of the amplified phage eluates are:
Silk: 152x(10^9)x(10^2) = 1.52 x 10^13 pfu/mL
Polyester: 150x(10^9)x(10^2) = 1.50 x 10^13 pfu/mL
DAY 5 from Panning Procedure (see protocol page) was followed.
Dilution of the amplified eluate for the second panning was done using the following dilution factors and volumes:
Silk: 76-fold dilution => ~3.3µL of amplified phage eluate added to ~246.7µL of TBST 0.5%
Polyester: 75-fold dilution => 3.35µL of amplified phage eluate added to ~246.6µL of TBST 0.5%
OD600 measurements of titration culture were as follow:
Time after inoculation
OD 600nm
3h46
0.3272
4h01
0.4319
4h05
0.4869
NEB Phage Display on Wool and Linen - DAY 2
Results of 1st round unamplified phage eluate titration:
Dilution
Number of plaques
Wool
Linen
10
>1000
>1000
100
87
357
1000
10
46
10000
0
1
Therefore, we have following phage concentration -
Wool  -  87x(10^2)x(10^2) = 8.7 x 10^5 pfu/mL.
Linen - 357x(10^2)x(10^2) = 3.57 x 10^6 pfu/mL.
Followed Day2 protocol exactly.
OD measurements:
Time after inoculation
OD 600nm
0
0
3h30
2.42
Diluted 1:20
4h00
0.443
4h07
0.5409
Plaque amplification for Sequencing - Polyester and Silk
Plaque amplification protocol was followed. 20 plaques for each fabric were amplified.
NEB Phage Display on Wool and Linen - DAY 3
Results of 1st round amplified phage eluate titration:
Dilution
Number of plaques
Wool
Linen
10^8
>1000
>1000
10^9
267
277
10^10
41
22
10^11
2
4
Therefore, we have following phage concentration -
Wool  - 267x(10^9)x(10^2) = 2.67 x 10^13 pfu/mL.
Linen - 277x(10^9)x(10^2) = 2.77 x 10^13 pfu/mL.
Round 2 panning:
To get a final phage number per reaction approximately 2*10^11, add 10µl of the amplified phage solution in              240µl TBST(0.5% Tween 20) to get a final volume of 250µl for both wool and linen
Followed Day3 protocol of Phage display protocol.
OD measurement of Culture for Titration
Time after inoculation
OD 600nm
0
0
2h20
0.303
2h32
0.521
Diluted 1:5
Because of Hood availability
3h32
0.363
3h45
0.4759
OD measurements of Culture for Amplification
Time after inoculation
OD 600nm
Wool
Linen
1h00
>0.0129
>0.009
1h25
0.0179
0.0139
NEB Phage Display on Wool and Linen - DAY 4
Results of 2nd round unamplified phage eluate titration:
Dilution
Number of plaques
Wool
Linen
10
48
>1000
100
4
150
1000
1
18
10000
0
1
Therefore, we have following phage concentration -
Wool  - 48x10x(10^2) = 4.8 x 10^4 pfu/mL.
Linen - 150x(10^2)x(10^2) = 1.5 x 10^6 pfu/mL.
Followed Day4 of the protocol exactly..
OD measurements:
Time after inoculation
OD 600nm
0
0
3h05
0.421
3h10
0.534
NEB Phage Display on Wool and Linen - DAY 5
Results of 2nd round amplified phage eluate titration:
Dilution
Number of plaques
Wool
Linen
10^8
>1000
>1000
10^9
394
269
10^10
50
35
10^11
4
3
Therefore, we have following phage concentration -
Wool  - 394x(10^9)x(10^2) = 3.94 x 10^13 pfu/mL.
Linen - 269x(10^9)x(10^2) = 2.69 x 10^13 pfu/mL.
Round 3 panning:
To get a final phage number per reaction approximately 2*10^11, add-
Wool  - 5.1µl of the amplified phage solution in 244.9µl TBST(0.5% Tween 20) to get a final volume of 250µl
Linen - 10µl of the amplified phage solution in 240µl TBST(0.5% Tween 20) to get a final volume of 250µl
Followed Day5 protocol of Phage display protocol.
OD measurements:
Time after inoculation
OD 600nm
0
0
3h00
0.364
3h15
0.522
NEB Phage Display on Wool and Linen - DAY 6
Results of 3rd round unamplified phage eluate titration:
Dilution
Number of plaques
Wool
Linen
10
499
>1000
100
38
547
1000
5
58
10000
0
6
Therefore, we have following phage concentration -
Wool  - 38x100x(10^2) = 3.8 x 10^5 pfu/mL.
Linen - 58x(10^3)x(10^2) = 5.8 x 10^6 pfu/mL.
Week 22nd - 28th August
Purification of Phage DNA from Phage Display on Polyester and Silk for sequencing
Purification of phage DNA was carried out following the Purification of phage DNA protocol.
Note: The 500µL of phages R3P15 and R3P16 were by mistake mixed together. This was thrown away and subsequent purification was carried out with the remaining 250µL of phage stock.
Quantification was done using a Nanodrop2000. Phage DNA concentrations and purities were as follow:
Polyester
Silk
Sample Number
Concentration (ng/µL)
260/280 ratio
260/230 ratio
Concentration (ng/µL)
260/280 ratio
260/230 ratio
R3P1
69
3.17
2.27
49
2.06
0.18
R3P2
50
2.77
0.69
66
2.95
2.25
R3P3
36.5
2.88
0.23
72
3.64
2.54
R3P4
47
2.28
0.21
79
2.34
1.24
R3P5
57
3.44
1.70
27
4.45
0.20
R3P6
84
3.66
3.91
65
4.04
2.39
R3P7
50
2.23
0.28
76
5.72
3.92
R3P8
59
2.06
0.20
25
3.02
0.10
R3P9
46
3.54
1.12
29
10.32
0.96
R3P10
54
2.54
0.60
30
9.51
0.98
R3P11
77
2.95
2.34
25
10.35
0.57
R3P12
74
3.07
2.29
56
3.68
1.81
R3P13
48
2.12
0.15
27
2.21
0.06
R3P14
29
3.09
0.14
38
9.76
1.55
R3P15
48.5
2.24
0.18
8
9.21
0.02
R3P16
64
2.13
0.45
44.5
33.87
2.14
R3P17
54.5
2.74
0.89
19.5
6.71
0.20
R3P18
50
2.19
0.21
27
6.04
0.52
R3P19
39
2.55
0.18
31.5
4.44
0.52
R3P20
59.5
3.53
1.78
42.5
3.74
1.16
Some of the samples (actually almost all of them) have weird 260/280 and 260/230 values. I do not know if this is going to be a problem for sequencing. To test, I will send only 5 samples (Samples R3P1 to R3P5 of the SILK experiment) to check that sequencing is OK.
Sequencing of phage DNA from NEB phage Display on Silk and Polyester
All samples were sent to sequencing by GATC Biotech.
Results are available below:
Sequences Phages NEB Silk
Sequences Phages NEB Polyester
Sequencing of Phage DNA from Phage Display on Wool and Linen
I proceeded with plaque isolation and Phage DNA isolation for sequencing. I picked 5 plaque each for R1 and R2 panning
unamplifies titration plates of wool and linen and 20 each for R3 panning unamplified titration plates of wool and linen.
Please refer following protocols from the protocol page:
Plaque Amplification for ELISA or Sequencing - for plaque isolation and amplification and
Rapid Purification of Sequencing Templates - for sample preparation for sequencing
DNA concentration and 260/280 ratio of purified templates for sequencing:
Linen_Round3 Panning
Concentration (ng/µL)
260/280 ratio
R3P1
78
1.80
R3P2
12
3.12
R3P3
15
2.03
R3P4
22
2.45
R3P5
34
1.98
R3P6
18
2.27
R3P7
14
3.78
R3P8
21
1.78
R3P9
19
2.08
R3P10
19
2.76
R3P11
15
2.53
R3P12
19
1.92
R3P13
24
3.10
R3P14
15
2.48
R3P15
30
1.65
R3P16
79
8.36
R3P17
31
1.93
R3P18
23
1.93
R3P19
24
2.20
R3P20
25
1.75
Linen_Round1
Concentration (ng/µL)
260/280 ratio
R1P1
35
2.73
R1P2
20
2.59
R1P3
28
2.40
R1P4
22
2.43
R1P5
26
2.51
Linen_Round2
Concentration (ng/µL)
260/280 ratio
R2P1
27
2.06
R2P2
56
2.04
R2P3
66
2.06
R2P4
32
2.47
R2P5
25.8
2.05
Wool_Round3 Panning
Concentration (ng/µL)
260/280 ratio
R3P1
94
1.88
R3P2
72
1.98
R3P3
68
1.94
R3P4
81
1.77
R3P5
82
1.85
R3P6
87
1.86
R3P7
80
1.85
R3P8
61
3.31
R3P9
73
1.85
R3P10
59
1.95
R3P11
68
1.85
R3P12
64
1.88
R3P13
57
1.93
R3P14
70
1.93
R3P15
69
1.87
R3P16
66
2.22
R3P17
63
1.88
R3P18
94
2.14
R3P19
73
2.03
R3P20
65
1.91
Wool_Round1
Concentration (ng/µL)
260/280 ratio
R1P1
16
2.79
R1P2
26
4.00
R1P3
24
2.21
R1P4
19
2.15
R1P5
34
2.28
Wool_Round2
Concentration (ng/µL)
260/280 ratio
R2P1
56
2.05
R2P2
45
2.00
R2P3
54
2.01
R2P4
52
2.08
R2P5
61
1.97
All Round 3 samples were sent to sequencing by GATC Biotech.
Results are available below:
NEB Phage Display on Wool and Linen - Finding a consensus sequence
Sequences were analysed using Geneious 9.0.2.
The random peptide library sequence was found between the linker (motif: 5'-ACC TCC ACC-3') and the end of the pIII leader sequence (motif: 5'-AGA GTG AGA-3'). We used only the -96 primer to sequence as the -28 seemed too close to the library sequence for the sequencing to give a good result.
Some samples (R3P5, R3P12, R3P17,R3P18 of wool and R3P3, R3P8,R3P11 of linen) did not exhibit those motifs. Deletion?
The 21 nucleotides of the library of remaining sequences were then translated.
Consensus sequence for Linen:
Using the Multiple-Align function of Geneious and setting the threshold for the consensus sequence at 25% (amino acid present at least 25% of the time at a given position), we get the following consensus sequence:
Consensus sequence for Wool:
At 25% threshold-
NEB Phage Display on Polyester and Silk - Finding a consensus sequence
Sequences were analysed using Geneious 9.1.5.
As for the sequences obtained with cotton, the random peptide library sequence was found between the linker (motif: 5'-ACC TCC ACC-3') and the end of the pIII leader sequence (motif: 5'-AGA GTG AGA-3'). We again used only the -96 primer (see sequencing guidelines scheme in protocol page) to sequence as the -28 seemed too close to the library sequence for the sequencing to give a good result.
Among the 20 plaques picked for polyester, one (R3P19) did not exhibit the randomized region (nor did the linker sequence and the end of the mature pIII protein coding sequence). Deletion?
This sequence was not used in the analysis.
Additionally, four of the sequences of phages selected on silk showed multiple randomized sequences:
R3P9: 2 randomized sequences in the same orientation.
R3P10: 3 randomized sequences with first and last one having the same orientation.
R3P11: 2 randomized sequences in opposite orientation.
R3P17: 2 randomized sequences in the same orientation.
Those sequences will be treated separately.
The 21 nucleotides of the library of the 19 remaining sequences for polyester and the 16 remaining sequences for silk were then translated.
Polyester:
Silk:
Using the Multiple-Align function of Geneious and setting the threshold for the consensus sequence at 0% (amino acid majoritarily present at a given position), we get the following consensus sequences:
Polyester:
Silk:
Week 29th August - 4th September
Spoke to one of our advisor - Ariel Lindner - He had some concerns about specificity and suggested few experiments.
This is what he suggested-
For panning to find wool binding peptides- incubate every other fabric before wool, so that it removes any sticky peptides from the pool.
He also suggested to do a panning with M13 phage without peptides to see the number of phage particles that bind.
For later experiments at quantifying the binding affinity of the peptides for the target we should use enzymes that are easy to quantify using calorimeter like HRP and the likes instead of GFP, like this we hope to avoid the problem of autofluorescence from the target.
I started amplification of the 3rd round panning eluate of wool and linen- It would be Day 6 I guess..
OD measurements-
Time after incubation
OD 600nm
Wool
Linen
0
0
0
1h
0.008
0.003
1h 45
0.0198
0.0186
Day 7 - Completed amplification and titred for both wool and linen.
Following the suggestion from Ariel-
Fourth round of panning - I started with Linen first.                                      Plaques counted from the amplified R3 panning eluate-
Dilution
Number of plaques
Wool
Linen
10^8
>1000
>1000
10^9
133
279
10^10
13
29
10^11
2
3
Wool--> 133*10^9*10^2 =1.33*10^13 pfu/ml
Linen-->279*10^9*10^2 =2.79*10^13 pfu/ml
I tested two diferent parameters today-
The wash time, as a reaction's rate mainly depends on its rate of dissociation.
Panning with only M13 phage without peptides.. to check if these phages bind non specifically to the fabric.
Calculation for Linen-
2.79 *10^13 phages in 1000μl-
To get final phage count as ~2*10^11--> 2*10^11/2.79*10^10=6.79μl.
I decided to reduce the size of the fabric used to provide some sense of competition for the target - I used 1cm long fibre of the fabric- accordingly, I reduced the volume to 100μl so that all the phages can come in contact with the target.
Also, to remove all the sticky peptide sequences, I incubated the phages first with 4 fabrics together- wool, silk, polyester, cotton.
Note: There were more than usual number(0-3) of white plaques along with the blue ones in the titration plates.. Contamination?? I will be autoclaving all my solution just in case.
The M13 phage concentration was 1.5*10^14 pfu/ml
to get final phage count as ~2*10^11--> 2*10^11/1.5*10^11=2μl
Fabric size-
Silk- 10cm long thread
Polyester- 10cm long thread
Cotton - 0.25 cm2 square
Wool- 0.25 cm2 square
Day 8-Followed the day 1 of Phage display protocol with slight changes-
Washed the fabric/threads in ethanol/autoclaved water.
Blocked for 1h at 4 deg.
Incubated all fabric except linen in the phage solution for 1hr at room temperature.
Removed the fabrics, centrifuged and collected the supernatant.
Incubated linen (1cm fibre) with the remaining non sticky phages for 1h at room temperature.
Elute, neutralize and titre- this time I titred only 10^1 ,10^2,10^3 for different condition and two different phage type.
OD measurement for the titration culture:
Time after incubation
OD 600nm
0
0
3h
0.298
3h 30
0.61=> Diluted 2:1
3h 35
0.492
Week 5th September - 11th September
Re-analyzing the results
During the week-end, I retook a look at the seqences and at the manual to make sure we did everything correctly so that I could upload everything on the wiki. That was very fortunate as I realized we were reading the wrong frame to translate our sequences (since all the codons are supposed to have their third nucleotide be G or T which was not the case). I therefore reversed complemented everything (which I should have done from the beginning as indicated in the manual) then translated in frame 1 (which is also frame -1 of the sequences we originally got from sequencing).
The "new" results (the real ones this time) are as follow:
Cotton:
Translation of the sequences:
Consensus sequence:
The geneious files containing the sequence of the library for all clones, the translation of all those sequences and the alignement can be found here:
Geneious files NEB PhD Cotton
Silk:
Translation of the sequences:
Consensus sequence:
Geneious files NEB PhD Silk
Polyester:
Translation of the sequences:
The geneious files containing the sequence of the library for all clones, the translation of all those sequences and the alignement can be found here:
Consensus sequence:
The geneious files containing the sequence of the library for all clones, the translation of all those sequences and the alignement can be found here:
Geneious files NEB PhD Polyester
Linen:
Translation of the sequences:
Consensus sequence:
Wool:
Translation of the sequences:
Consensus sequence:
From approximately 20 sequences for 5 different fabrics each we picked total of 9 peptide sequence to pass on to Enzyme search group to fuse it with enzymes they are working on and study their efficiency in binding fabrics.
The peptides are named FBD 1-9 for Fabric binding domain - 1 to 9.
Out of the 9 sequences 5 are specific, i.e, one sequence specific for each fabric. The sequences were not selected based on the consensus sequence but their repeatition in that particular fabric or some unique feature in the peptide. and the other 4 sequences are non specific, found in multiple fabrics.
FBD-1, 2, 3, 4 -  Are non specific, they are repeated in all 5 fabrics.
FBD5 - Specific to Wool, it was repeated 3 times out of 16 sequences.
FBD6 - Specific to cotton
FBD7 - Specific to Silk
FBD8 - Specific to Linen, the sequence was selected because it has multiple proline residues
FBD9 - Specific to polyester, it was selected because of its high number of positively charged amino acids
FBD10 - Picked from literature, Guo et.al, (2013) it will serve as a positive control - shown to bind to cellulose
FDB11 - Random amino acids put together(7aa) to be used as negative control.
Continuing from previous week-                         Plaques Counted:
Dilution
M13 Long wash
M13 Quick wash
PHD Long wash
PHD Quick wash
10
0
18
63
22
100
0
1
5
2
1000
0
0
1
0
With long wash the non specifically binding M13 phages are washed off.
I picked 10 plaques each from PHD long wash and quick wash. Amplified and purified the templates for sequencing.
DNA concentration and 260/280 ratio:
Linen_Counter selection_Quick wash
Concentration (ng/µL)
260/280 ratio
R4P1
59
1.89
R4P2
61
1.73
R4P3
28
1.74
R4P4
40
1.76
R4P5
52
1.77
R4P6
63
1.8
R4P7
68
1.74
R4P8
68
1.8
R4P9
75
1.83
R4P10
44
1.95
I sent the quick wash phage DNA for sequencing-
Out of 10 seqs, 1 contained non-determined nucleotides (N), that sequence was not used in the analysis. .
Translation of the sequences:
Week 12th September - 18th September
Workshop..
Week 19th September - 25th September
We realized we need to have atleast 50 sequences for each fabric in order to find a emerging similarity in the sequences..                                                  Started new culture of ER2738
Titred Cotton R3 and linen R4 long wash phage eluate.
I did 2 dilutions each  Cotton - 10^2 and 10^3 linen - 10^1 *2
Cotton titration plates looked good but linen plates had only 2 plaques each.
So I decided to do another paning from R3 amplified eluate of linen and wool. (Since I had already amplified wool earlier, and I need to pick plaques fron unamplified eluate)
I picked 30 plaques from cotton plates - followed the protocol for plaque amplification and rapid purification of sequencing templates
Cotton
Concentratio (ng/µL)
260/280 ratio
R3P21
46
1.98
R3P22
51
2.43
R3P23
72
2.03
R3P24
65
2.19
R3P25
69
2.15
R3P26
47
2.17
R3P27
41
2.84
R3P28
125
11.19
R3P29
55
2.25
R3P30
61
2.01
R3P31
59
2.05
R3P32
60
1.94
R3P33
38
1.93
R3P34
61
2.48
R3P35
49
2.07
R3P36
58
1.98
R3P37
65
1.93
R3P38
49
2.15
R3P39
77
1.97
R3P40
58
1.91
R3P41
58
2.25
R3P42
62
2.01
R3P43
71
1.97
R3P44
64
1.99
R3P45
62
1.95
R3P46
68
1.94
R3P47
66
1.96
R3P48
38
2.25
R3P49
35
2.14
Week 26th September - 02nd October
For ELISA I needed higher number of virions, so I amplified the selected peptide displaying phages
ELISA Protocol:
Carry out the plaque amplification procedure described previously- After the first centrifugation, save the phage-containing supernatants at 4°C.
For each clone to be characterized,  dilute an overnight culture of ER2738 1:100 in 20 ml of LB.
Add 5 μl of phage stock from Step 1(or a single phage plaque) to a 20 ml culture for each clone and incubate with vigorous aeration for 4.5–5 hours at 37°C.
Transfer the culture to a centrifuge tube and spin at 12,000 g for 10 minutes at 4°C. Transfer the supernatant to a fresh tube and re-spin (discard the pellet).
Pipette the upper 80% of the supernatant to a fresh tube and add 1/6 volume of 20% PEG/2.5 M NaCl. Allow the phage to precipitate at 4°C for at least 2 hours or overnight.
Spin the PEG precipitation at 12,000 g for 15 minutes at 4°C. Decant and discard the supernatant, re-spin briefly, and remove residual supernatant with a pipette.
Suspend the pellet in 1 ml of TBS. Transfer the suspension to a microcentrifuge tube and spin at 14,000 rpm for 5 minutes at 4°C to pellet residual cells.
Transfer the supernatant to a fresh microcentrifuge tube and re-precipitate with 1/6 volume of 20% PEG/2.5 M NaCl. Incubate 15–60 minutes on ice. Microcentrifuge at 14,000 rpm for 10 minutes at 4°C. Discard the supernatant, re-spin briefly, and remove residual supernatant with a micropipet.
Suspend the pellet in 50 μl of TBS and Titer. The titer should be approximately 10^14 pfu/ml. The phage stock can be stored for several weeks at 4°C. For long-term storage, add an equal volume of sterile glycerol and store at –20°C.
Dilutions
MPRLPPA
ADARYKS
SILPVTR
ASSHIHH
ADIRHIK
MRLSVPN
AWPYVTL
QFPPPPG
FRKKRKS
10^8
>1000
>1000
>1000
>1000
>1000
>1000
>1000
>1000
>1000
10^9
>1000
>1000
>1000
>1000
>1000
>1000
>1000
>1000
>1000
10^10
392
645
>1000
>1000
>1000
>1000
>1000
411
>1000
10^11
42
70
107
96
98
114
113
56
88
Fill each well completely with blocking buffer. Additionally, one row of uncoated wells per clone to be characterized should also be blocked to test for binding of each selected sequence to BSA-coated plastic (this test for background signal is extremely important, especially if panning was carried out on a polystyrene surface). A second, fully uncoated microtiter plate should be blocked for use in serial dilutions of phage (step 13) before addition to the target-coated plate. Dilutions are done in a separate blocked plate to ensure that phage are not absorbed onto the target during the course of performing dilutions, which would result in a sudden “falling-off” of signal as the phage is diluted. Incubate the plates filled with blocking buffer for 1–2 hours at 4°C.
Shake out the blocking buffer and wash each plate 6 times with TBST, slapping the plate face-down onto a clean section of paper towel each time. The percentage of Tween should be the same as the concentration used in the panning wash steps.
In the separate blocked plate, carry out fourfold serial dilutions of the phage in 200 μl of TBS/Tween per well, starting with 10^12 virions in the first well of a row and ending with 2 x 10^5 virions in the 12th well.
Using a multichannel pipettor, transfer 100 μl from each row of diluted phage to a row of target-coated wells, and transfer 100 μl to a row without target. Incubate at room temperature for 1–2 hours with agitation.Wash plate 6 times with TBST as in Step 12.
Dilute HRP-conjugated anti-M13 monoclonal antibody (GE Healthcare. #27-9421-01) in blocking buffer to the final dilution recommended by the manufacturer. Add 200 μl of diluted conjugate to each well and incubate at room temperature for 1 hour with agitation.
Wash 6 times with TBST as in Step 12.
Prepare the HRP substrate solution as follows: a stock solution of ABTS can be prepared in advance by dissolving 22 mg of azino-bis(3-ethylben- zothiazole sulfonic acid) diammonium salt (ABTS) (Sigma, cat. # A-1888) in 100 ml of 50 mM sodium citrate, pH 4.0. Filter sterilize and store at 4°C. Immediately prior to the detection step, add 36 μl of 30% H2O2 to 21 ml of ABTS stock solution per plate to be analyzed.
Add 200 μl of substrate solution to each well, and incubate for 10–60 minutes at room temperature with gentle agitation.
Read the plates using a microplate reader set at 405–415 nm. For each phage concentration, compare the signals obtained with and without target protein.
ELISA results:
The results from the Elisa did not give much information as the non specific signal was just as high as the signal from target. Washing more thoroughly might be a solution for this problem.
We wanted to perform ELISA in such a way as to obtain atleast semi quantitative data.
Since the difference in the signal is seen only in the higher dilution, I decided to use 10^12 virion per well for further experiment.
The experiment would involve washing the target with increasing volume starting from 1ml to 5ml and reading the OD.
Titred Wool, linen, Silk and polyester with only 2 dilution - 10^2 and 10^3 - for picking colonies for sequencing.
Week 10th October - 16th October
I performed another set of experiment with increasing volume starting from 1ml to 5ml and also, I used microcentrifuge tubes instead of the 96 well plate. This was done to reduce the background noise.
We are hoping to see a proportional decrease in the signal with increased washing and with this calculate the binding constant.                     The protocol is same as before but with increased wash volume..
Result
From the graph we see that there is no proportional decrease with more washes. But what we can say is that the peptides are bound with good affinty even though we cannot quantify it. The value with 1ml wash and 5 ml wash is similar and also, the background noise is very less in comparison almost 10 fold.
I will be performing same assay on other fabrics (silk, wol, linen, polyester)
I picked plaques, amplified, purified sequencing template from silk, linen, polyester and sent it for sequencing.
Next, I will  picked  one fabric and one FBD- FBD9 and increase the wash volume progressively from 1ml to 10ml over a period of 1hr and also, I changed the detergents used - 1% Tween 20, 1% Tritin X100, 1% SDS-in the washes.
Week 17th October - 23rd October
Centre for Research and Interdisciplinarity (CRI) Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor Paris Descartes University 24, rue du Faubourg Saint Jacques 75014 Paris, France
+33 1 44 41 25 22/25
igem2016parisbettencourt@gmail.com
2016.igem.org
