Team:Paris Bettencourt/Notebook/Enzyme
Project
Overview
Assay
Microbiology
Enzyme
Binding
Indigo
Model
Practices
About Perc
Dry Cleaners
Design
Events
Safety
Lab Safety
Product
Safety
Achievements
Results
Interlab Study
MedalRequirements
Parts
Notebook
Daily Note
Protocols
Bibliography
Team
Our Team
Attributions
Collaborations
Sponsors
@iGEM_Paris
Notebook
Protocols
Bibliography
Assay
Microbiology
Enzyme search
Mission Indigo
Binding Domains
Anthocyanin
Week 18th -24th July
This week we designed the strategy for cloning all the proteins that we want to test in the pCOLA vector, and at the same time make everything compatible with the phytobrick format.
The table of proteins that we are working on is the following:
BG1 - beta-glucosidase from Vinis vinifera
bpul - laccase from Bacillus pumilus
catA - catechol-1,2-dioxygenase from Acinetobacter pittii
Cellulose Binding Domain
POO2 - polyphenol oxidase from Camellia sinensis
tanLpI - tannin acyl hydrolase from Lactobacillus plantarum
xylE - catechol-2,3-dioxygenase from Pseudomonas putida
How to clone using the pDUET – Golden Gate adapted plasmids
The first thing that you have to do it to codon optimise your sequence for E. coli. For that we used the IDT’s Codon Optimisation Tool (https://eu.idtdna.com/CodonOpt).
After doing the codon optimisation we checked if our sequence had any recognition site for the BpiI, BsaI and BsmBI. Those enzymes are widely used in the Phytobricks and there can therefore be no recognition size outside of the purposely designed (https://2016.igem.org/Resources/Plant_Synthetic_Biology/PhytoBricks). Correct the recognition sites using a codon usage table.
Attach the following to your optimised sequence:
To the 5’- attach a BsaI recognition site that will allow you to fit the part in the Phytobrick. The sequence must be as follows, to be able to fit in the already defined iGEM parts:
In order to be able to get a singe primer for amplifying all the different CDS after synthesis, we need to add a tail before the BsaI site that will allow us to create a universal primer.
We call this structure with the NNNNN + the BsaI site 5’ the extremity A (this will make sense later on, we promise).
When adding the extremity A to the sequence take into account that the extremity ends with a start codon, therefore eliminate the one from your CDS to get it in frame.
To the 3’- attach the His-tag and a BsaI recognition site. This will allow us to purify the protein and also fit the part in the Phytobrick.
Once again, take into account the fact that the sequence must be as follows to fit in the Phytobrick. If not designed like this the BsaI recognition site will continue to exist in our sequence after cloning and that could be a problem.
Before the BsaI 3’ recognition site, add a His-tag. Do not forget removing the STOP codon from the CDS to allow for fusion with the His-tag. This two together will create what we call the extremity C (once again, will make sense!).
/>
The extremities A and C will permit us to clone the entire CDS + His-tag in our plasmid. Nonetheless, we might not want to clone the His-tag in the iGEM Phytobricks. In order to clone the CDS without the his-tag we will design some specific primers.
The common primers FW and RV will allow us to amplify all our CDS with the His-tag and directly clone them in our desired vector. They will also allow cloning the entire part in the Phytobricks.
The common primer FW and the CDS-specific RV will allow to clone only the CDS without the His-tag into the Phytobricks, and will also allow to attach a CBD to our proteins.
Example of primers (ignore the sequence of BsaI that has been added to the CDS sequence, it is only there because it is easier to design the primers like that in the software, they will not exist in the synthesised DNA, they will be present as TAILS. Take the STOP codons always into account!
Until here, a short resume of what has to have been done:
Codon optimise
Check for restriction enzymes and correct (BpiI, BsaI and BsmBI)
Attach the 5’ extremity (A) and delete ATG
Attach the 3’ extremity (C) and delete the STOP codon
Design primers that are specific
Next, we need to design our CDB and attach to it the BsaI sites to clone it in the vector, Phytobricks and attach it to our CDS.
To the 5’- attach a BsaI recognition site that will allow you to fit the part in the Phytobrick with our gene in frame. The sequence must be as follows, to be able to fit in the already defined iGEM parts.
To the 3’- attach the His-tag and a BsaI recognition site as before. This will allow us to purify the protein and also fit the part in the Phytobrick.
Ordering of primers for golden gate and Gibson assembly
We also ordered the following primers for carrying out our strategy:
Name Sequence Scale Purification
FNS_1 GTAGTAGCTGCTATATGGTCTCAA 24 Standard desalting
FNS_2 CGATGGTCTCAAAGCTCAGT 20 Standard desalting
FNS_4 CGATGGTCTCAGGCTGAGGCTGAAGCACGGCGA 33 Standard desalting
FNS_6 CGATGGTCTCAGGCTGAGGGAGAATTCAAGAAGTTCTTAAACCA 44 Standard desalting
FNS_8 CGATGGTCTCAGGCTGACTGAATAATATCCATCGGGCG 38 Standard desalting
FNS_10 CGATGGTCTCAGGCTGAAGAGTCAAATTCAATTTTTACACCACCG 45 Standard desalting
FNS_12 CGATGGTCTCAGGCTGACTGACACAGACCGTCAATCCA 38 Standard desalting
FNS_14 CGATGGTCTCAGGCTGAAGTCAAAACAGTCATAAAACGTTCATTC 45 Standard desalting
FNS_15 GCTGACGACCGAGTCTCCGCA 21 Standard desalting
FNS_16 TACCGAAGATAGCTCATGTTATATCCCGC 29 Standard desalting
FNS_17 TTGCTCAGCGGTGGCAGCAG 20 Standard desalting
FNS_18 ATCGTATTGTACACGGCCGCAT 22 Standard desalting
FNS_19 TCGGAATCGCAGACCGATACCAGGA 25 Standard desalting
FNS_20 ATTTATGCCTCTTCCGACCATCAAGC 26 Standard desalting
FNS_21 ATGTTCGTCAGGGGGGCG 18 Standard desalting
FNS_22 TTGGGGAACTGCTTAACCTGGTAACT 26 Standard desalting
FNS_23 GTGAAAAGAAAAACCACCCTGGCG 24 Standard desalting
FNS_24 GTAATTCAGCTCCGCCATCGCC 22 Standard desalting
FNS_25 GACGCGCCGAGACAGAACTT 20 Standard desalting
FNS_26 CATGTTAGTCATGCCCCGCG 20 Standard desalting
Week 25th -31th July
This week we started the kombucha growing for the Imperial Protocol reproduction. We are redoing the protocol from their 2014 team to test for the binding of the Cellulose Binding Domains that we are using.
We are carrying out a similar protocol to the one we can find at https://2014.igem.org/Team:Imperial/Protocols
Such experiment aims to measure the affinity of the CBD to the cellulose after several washes with different solutions. The affinity is measured by reading the fluorescence of the Green Fluorescent Protein, GFP, fused with the CBD. The difference between the fluorescence before and after the washes can give a reference of the affinity of the Cellulose Binding Domain under different conditions of media.
We are also testing more washing solutions, such toluene and grape extract in addition to those tested by the Imperial College iGEM team, since they are more related with our future experiments.
This experiment aims to improve the available data regarding the part BBa_K1321357, submitted by iGEM imperial College team in 2014, and also to generate data and compare affinity of the GFP-CBD and the future proteins that we will fuse with CBD in our project.
Kombucha growing protocol
We got some old Kombucha from Juan Manuel García Arcos (iGEM team from 2014 and manager of the Open Science School).
We grew the kombucha as it follows:
800mL of ddH2O heated up
Add 100g of brown sugar and let dissolve
Add 3 sachets of green tea and let them infuse until it cools down (under 30ºC!)
Add 100mL of cider vinegar
Add 200mL of starter culture (commercial kombucha in juice)
Add a big piece of kombucha culture
Grow at 30ºC without shacking!
Preparation of the crude lysate for phusion protein assays
The first thing was to grow a 20ml inocule of the strain carrying the GFP-CBD protein, in our case it was the strain FS_S2, which is transformed with the part BBa_K1321357, submitted by the Imperial College team in 2014. Chloramphenicol (Crm) was added, since the plasmid carrying this part has Cm resistance.
Following we used the 20ml of such preculture as inocule of 1L of media, and let them grow overnight.
The cells grown overnight were collected by centrifuging at 4000rpm during 20min. They were then resuspended in 10ml of PBS, and collected again to remove all the residual media.
The remaining cells were resuspended in B-PER Bacterial Protein Extraction Reagent, from Thermo Scientific. 4ml per gram of cells.
Incubated for 15mins and centrifuged 5minuetes at 15000rpm.
The supernatant was recovered and the GFP-CBD detected in it by looking at the fluorescence of the sample.
Then the cell lysates were stored at 4C until the day of the assay.
Week 1st-7th August
Preparation of the 96-well plates
The 96 wells plates where we will measure the fluorescence of the GFP-CBD have to treated with cellulose in order to retain the CBDs and carry out the experiment.
In order to do that we mixed 6.4g of pure cellulose (435236-250G, Sigma-Aldrich) in 40ml of distilled water, to fill two 96 wells plates (with black plates for fluorescence measurement). Since the cellulose is almost insoluble in water the mix has to be shaken all the time while 200ul of it are added to each well. Also, due to the high viscosity of the mix we used the pipette of 1000ul to get 200ul.
Once we had all the wells full, the plates must be dried in the incubator at 37C one over night.
200 ul of cell extract was added to each well of the 96 wells plate already treated with cellulose, also the same was done with another plate but with 200ul of cell extract 1/10 diluted.
Rows A-H, columns 1-6 --> the concentration of cell extract is the original
Rows A-H, columns 7-12 --> the concentration of cell extract is 1/10 respect the original
The plate was incubated 24hours at 4C before the assay.
We carried out the assay as follows:
Measuring absorbance with 475 excitation 510 emission, when the wells were full and when the liquid is removed in each case (just in the Imperial College iGEM team is described).
3 washes were done with the following solutions: water, PBS1x, ethanol 70%, BSA 5% w/w and grape extract to a concentration of 1mg/mL.
The positioning of each of the washes in the plate is as follows:
We got the following results for the experiment:
The following table shows the comparison between our preliminary data and the one that was supplied by the Imperial team in 2014. As we can see, the results are quite different. In the case of Water and PBS, they are completely opposite from what was observed by the Imperial team, and in the case of the other solutions, they are more similar. The date shown in the graph is for remaining fluorescence after washes (relative fluorescence).
Because the results are too different and also because we need to test other subtracts, we will repeat the experiment in the future.
Weeks 8-14th and 15-21th August
On August the 11th our gBlocks finally arrived, so we could really start our work on the enzyme studies.
The first step was to clone the vector in which we will clone all of our enzymes. The vector (pCOLA-FS) is based on the pCOLA plasmid, only it only contains one T7 promoter, no cloning sites incompatible with the PhytoBricks, and also contains the lac operon under the remaining T7 promoter.
The idea of cloning the lac operon is to simplify the screening for the rightly cloned plasmids.
Gibson Assembly of the pCOLA-FS plasmid
The pCOLA-FS plasmid was assembled by Gibson Assembly from the following fragments:
pCOLA-FS synthetic fragment This fragment is 1925bp long.
pCOLA PCR product, amplified from primers FNS_15 and FNS_16. This fragment is 2128bp long.
The final size of the plasmid, after the Gibson Assembly is 4002bp.
For the Gibson Assembly of the pCOLA-FS itself, we calculated a molar ratio of 1:1 BackBone:insert, due to the size similarity of the two fragments, which did not justify having a 1:2 ratio. We used 100ng of the PCR fragment and 93ng of the synthetic fragment, in a total of 15uL of Gibson Assembly Mix (using New England’s Biolabs’ 2X Hifi DNA Assembly MasterMix. We incubated the mix 1h at 50ºC (protocol says to incubate it for 15min, but in our experience it works better if you incubate it for longer).
2uL of the final Gibson Assembly mix was transformed through chemical transformation to E. coli DH5alfa and plated in the corresponding antibiotics.
Golden Gate of the various pCOLA-FS plasmids
After we obtained the pCOLA-FS plasmid and checked it by sequencing, we began the cloning of the different pCOLA-FS plasmids carrying the different enzymes for testing.
For doing that, the first step was getting the fragments to clone using BsaI cloning.
The PCR reaction was carried out using ThermoFischer Phusion polymerase, because of its proofreading activity, and the PCR was carried out in a volume of 50uL (10uL 5X HF Buffer, 1uL 10mM dNTPs, 2.5uL of each primer, 2uL of the template (for 20bg), 1,5uL DMSO, 0.5uL pol and 30uL water). The PCR cycle was as follows: 98ºC 30’ – (98ºC 10’’ – 60ºC 20’’ – 72ºC 1’30’’)x33 – 72ºC 10’
After purification we run a gel to confirm the amplification:
Wells: 1.CBD – 2.pbuI_Ph – 3. bpuI – 4.tanLpI_ph – 5.tanLpI – 6.BG1_Ph – 7.BG1 – 8.xylE_Ph – 9.xylE – 10.PPO2_Ph – 11.PPO2 – 12.catA_Ph – 13.catA
Weeks 22-28th and 29th August – 4th September
During these two weeks we performed the Golden Gate to get our several plasmids.
It took us longer than we had originally expected because we run into some problems regarding the construction of the plasmids.
Luckily, we were able to figure the mistakes out. There were no mistakes in our designing of the cloning (uffff, we are happy about that, it would have been a disaster). There were nonetheless mistakes in the way we performed the protocol for Golden Gate.
We used the NEB Golden Gate Mix throughout all the attempts.
We started first by amplifying the fragments for the Golden Gate (as said already in the past week). IDT says that PCR should not be performed on their GBlocks in the case of them being bigger than 1000pb, which in all of our cases they are. Nonetheless, we needed to perform the PCR on the fragments because we needed different tails added to them.
The first Golden Gate protocol that we did was the one that is proposed by NEB for Golden Gates with up to 5 fragments (which is our case).
Total volume of reaction is 20uL. We added 2uL of the NEB Golden Gate Buffer, 75ng of plasmid (in our case 1.39uL), we kept a 2:1 molar ratio for each of the inserts, added 1uL of the Golden Gate Assembly Mix and added water up to the final volume.
We performed the proposed protocol of 1h at 37ºC an 5 min at 55ºC, and transformed the results into E. coli DH5alfa.
The following day we had a high number of colonies in our plates, all of them white (correct colour for the blue/white screening that we are performing).
Nonetheless, we performed PCR with our checking primers (ch1 and ch2/ch8) and all our bands were smaller than they were supposed to be. They seemed to be the size of what would be in the plasmid if the lacZ fragment had been correctly excised, and something quite small (up to 300pb) had been inserted.
The PCR was with Dreamtaq polymerase and as follows:
Volume per reaction 10ul:
- Mastermix --> 5ul
- ch1 --> 0.5 ul
- ch8 --> 0.5 ul
- DNA* --> 4ul
* DNA is actually the selected colony diluted in 20ul of distilled water.
PCR's settings:
1st- 10min -- 98C --> Cell lysis
2nd- 3min -- 95C --> Activation
3th- 30sec -- 95C --> Denaturailzation
4th- 30sec -- 55C --> Hybridation
5th- 2min -- 72C --> Elongation
Repeat from 3th to 5th 25 times
7 minutes -- 72C --> Final elongation
After the PCR we run an electrophoresis with the product:
Excel sheet for computing Golden Gate reactions
The settings for the Golden Gate reaction were as follows:
1. 37 C - 5 min
2. 37 C - 2 min
3. 16 C - 5 min
4. 50X back to 2
5. 37 C - 5 min (final digest)
6. 50 C - 10 min
7. 80 C - 10 min
The Golden Gate product was transformed by heat shock into thermo competent E.coli DH5-alpha and they were growth in LB plates with Kanamycin (50ug/ml), X-gal (20ng/ml) and IPTG (0.1mM), and growth overnight at 37C.
We did so to do Blue/White screening.
(Following the protocols in the protocol section). We also transformed a negative control with the intact plasmid without being processed by the GG assembly.
Next day we observed white and blue colonies in the plates, and all blue in the negative control as is was expected. Therefore we selected those colonies of each clone which were white and we did colony PCR on them.
The PCR was performed with Dreamtaq polymerase and as follows:
Primers ch1 and ch8.
Volume per reaction 25ul:
- Mastermix --> 12.5ul
- Water --> 5ul
- ch1 --> 2.5 ul
- ch8 --> 2.5 ul
- DNA* --> 2.5ul
* DNA is actually the selected colony diluted in 20ul of distilled water.
PCR's settings:  1st- 10min -- 98C --> Cell lysis
2nd- 3min -- 95C --> Activation
3th- 30sec -- 95C --> Denaturailzation
4th- 30sec -- 55C --> Hybridation
5th- 2min -- 72C --> Elongation
Repeat from 3th to 5th 25 times
7 minutes -- 72C --> Final elongation
After the PCR we ran an electrophoresis with the product but the gel was empty, it might have been because the polymerase Dream taq we were using was spoilt since the control wasn't amplified, therefore we repeated the PCR with another aliquot of Dream taq polymerase under the same conditions of PCR and after we run a new electrophoresis.
The Control plasmids is a PCR where only clean plasmid (after miniprep) was added.
The Control colony corresponds with a common colony PCR with an strain which contains the pFNS0, our backbone plasmid.
Finally we had reason to be happy :D. Since we had some positives. In this case all the clones except BGI.
Therefore we could start doing the phusion proteins, since we had all the clones for the proteins with only the His-tag.
In parallel we run another PCR but changing a detail, we realized that if we did the cell lysis in the thermocycler, as it had been done until now (see above), the resolution of the electrophoresis was not so good, then we decided to repeat the electrophoresis but doing the lysis before in a thermoblock.
The protocol followed is the following:
1º 5min - 100C --> Cell lysis
2º 5sec - vortex --> Homogenization
3º 3min - centrifuge 6000g --> Pelleting the cell debris
4º collect SuperNatant
The PCR was with Dreamtaq polymerase and as follows:
Primers ch1 and ch2.
Volume per reactio 50ul: (50ul to ensure that we have DNA in the 2ul of DNA we use)
- Mastermix --> 25ul
- Water --> 19ul
- ch1 --> 2 ul
- ch2 --> 2 ul
- DNA* --> 2 ul
*It is the SN collected after the cell lysis.
PCR's settings:
1nd- 3min -- 95C --> Activation
2th- 30sec -- 95C --> Denaturailzation
3th- 30sec -- 55C --> Hybridation
4th- 2min -- 72C --> Elongation
Repeat from 2th to 5th 25 times
7 minutes -- 72C --> Final elongation
After this an electrophoresis (agarose 1%) was carried out:
As we can see this gel shows again positives , even one for BGI. We decided to do miniprep of the following clones:
1.3 and 1.4 (bpuI)
2.3 and 2.5 (catA)
3.3 and 3.4 (tanLpI)
4.4 and 4.5 (PPO)
5.3 (BGI)
The results of the miniprep of this clones are:
After the normal proteins were golden gated, we started the GG for the six fusion proteins. To do that we did PCR first.
This GG mix contains the gblocks amplified two weeks ago with the primers which allow to have cohesive end in 3' with the 5' of the CBD protein, once it has been digested with BsaI, and the CBD gblock we used is the one coming directly from the product IDT supplied us.
The protocol we followed was the same that worked before:
For a final volume of 10 uL
x 40 fmol of vector and insert
1ul Buffer 10x
0.5ul Assembly mix
x dH2O (until 10ul)
-----------------------------
10 uL total
Settings
1. 37 C - 5 min
2. 37 C - 2 min
3. 16 C - 5 min
4. 50X back to 2
5. 37 C - 5 min (final digest)
6. 50 C - 10 min
7. 80 C - 10 min
To compute the quantities to add of DNA we used the excel sheet shown before.
Week 5th-11th September
This week we continue the Golden Gate protocols for producing the fusion proteins. This time, since what we did before did not work, we began by amplifying the genes from the plasmids that were built last week directly. The primers are those that allow for complementary ends A-B to fuse directly the IDT proteins with the CBD, which carries the cohesive A-C ends.
For this we performed Phusion PCR because we needed high fidelity amplification.
The PC mix is as follows: 30uL of water, 10uL of buffer, 2uL of template (pFNS1, 3, 5, 9, and 11), 2.5uL of FNS01 primer and 2.5uL of one of the following primers (FNS4, 6, 8, 10, 12 and 14), 1uL of dNTPs, 1.5uL of DMSO and 0.5uL of the phusion polymerase.
Primer list
We then run a gel to see the results of the amplification. Looking at the following picture we realised that we made a mistake in our logic. All the bands we see correspond to the supercoiled plasmid, and none to the amplicons. This is due to the fact that our primers do not work, because FNS1 does not hybridise with the template, since after cutting the gBlocks for golden gating them into the plasmids the complementary sequence for this primer is partially lost.
Therefore, we suggested to change the strategy to our advisors; who agreed that it would be the best option. We decided to forget about the fusion proteins with CBD and amplify instead the genes with the best Fabric Binding Domains that had been found by the binding domains group.
First. we are going to fuse this fragments with the GFP protein to do binding assays and select the best FBD out of the existing ones. Then, we are going to fuse these FBD to our candidate enzymes to see if their activity changes. If the activity on liquid is not perturbed, we will test to see whether they optimise their activity by directing it to the fabrics.
We did so, we designed all the new primers and ordered also a new set of GFP proteins to do the testing of the FBD. The problem we faced was that all the GP proteins that we currently have in the lab had bsaI cutting sites, and therefore we needed to synthesise the gene from scratch to get rid of the;. Since we had 8kb left from the IDT free synthesising sponsor, we ordered 8 versions of GFP, each Codon Optimised for E. coli, each fused with one of the 8 different FBDs. We chose one FBS with affinity for just one f the fabrics, one with affinity for cellulose (positive control) and two with affinity for all of the fabrics.
The FBD was added to the proteins in the N-terminal, because the phage display group told us that the domains they chose were designed to be expressed as such. Therefore, we run into a new problem. Some of the FBD did begin with a met in the first position, and therefore we had no problems. But for those which first aminoacid was not methionine, there is now an extra aminoacid, a met to start the protein translation.  This way we do not need to amplify the sequences and can clone them directly once they arrive, into our backbone. After we get the stains, we will perform assays like the ones we performed for the imperial college improvement of the GFP-CBD biobricks.
Week 12th-18th September
This week we had to wait for the designed things to arrive, ad in the meantime, we did some electrocompetent BL21 and BL21(DE3) cells, using the normal protocol that we always use.
We also started working on a collaboration with Evry's team, in which we need to characterise one of their biobricks.
We began by amplifying their GFP, from the backbone they gave us. We decided to amplify with Dreamtaq because it is quite easy to understand if the amplification has gone wrong, given that it will not shine.
In order to build their plasmid we needed to perform old/school restriction-ligation protocols, and therefore we sorted out the common buffer for both enzymes that they told us we needed to cut with, we performed the restriction protocol at 37C for 1hour. We then added ATP to the mix and performed ligation at room temperature during the night.
Week 19th-25th September
We recieved all the GFP-FBD sequences and start the clonning by Golden Gate, the process is the same as for the before proteins we cloned, it is to say using common primers in 3' but specifics in 5' because the FBD different for each was there. Also in the design of primers we took in account the fact that the FBD is in 5' and there we need a codon start, since only 2 of the FBD started by a methionin codon. Therefore we added one extra aminoacids to these FBD which didn't start by Methionin's codon.
After do the cloning we transform the result of golden gate in DH5-alpha to amplify the plasmid and do Blue/White screening, in order to select positives (white colonies) and an additional PCR with the primers FNS17 and FNS18 which amplify the region of the plasmid flanking the inserted gene.
Here the photo of the colony PCR with the DH5-alpha cells, the negative control is the result of golden gate but with water instead of insert, therefore is the empty plasmid, with LacZ gene, from a blue colony.
We have them, since almost 1.5kb is the length we expected for the amplicons. YUJU!!
The next steps, are sequencing an ascertain that the plasmid has no mutations, and transform with this plasmid our strain BL21-DE3 to carry out the gene over expression. To do that we need previously to do a miniprep.
During this week we also design the primers to do the phusion proteins, XylE and CatE with the two different FBD, specifically: FBD1 and FBD10, We chose this ones because FBD is not specific and it could work a priori in any fabric and FBD10 becuase is a control which binds to cellulose.
This primers are quite long, more than 70 bp, since they include the hybridation region, the FBD (7aa  therefore 21bp) and the cutting region. It will be a challenge to do the PCR with such a long primers. The template will be the primers already constructed for the native proteins.
This week we also repeat the Imperial College experiment, exactly the same as the first time but adding one new washing solution, catechol 0.03M since the it is the substrates for the enzymes which we are going to test with the FBD.
We will improve thi Biobrick because is also important for us, since we will use it as control in our binding experiments.
Week 26th September to 2nd October
We ascertain that all the plasmid we send to sequencing were good, without mutations, except for one of the FBD7, luckily we isolate 2 plasmids of each type, therefore if one fails as in this case we could chose the other to follow the experiments.
Once we now that the plasmids are ok, we transform them in BL21-DE3 and carry out the ove expression.
But first we will carry out the assays with the native enzymes we slected as potential candidates to egrade anthocyanins.
To do this we cloned the plasmids into BL21-DE3 in 5ml of LB during an whole night, then in the morning we inoculate this 5ml in 500ml of fresh LB, and let grow them till OD=0.4-0.6, then induce the expression by adding IPTG 0.5mM final concentration. And wait 5 hours.
The cell extract was obtained by following the protocol of B-PER from Thermo Fisher, then we carry out the SDS-PAGE and enzymatic assays to see if there were protein and if they are active.
We used a kit from BioRad, all ready even the gels, which are 10% polyacrlamide.
As we can see we didn't achieve to express the protein product of BGI, maybe because is a eucariotic protein which could from inclusion boides, therefore we desist to express it anymore as well as for PPO2, for the same reason.
Once this is done, lets do the enzymatic assays.
First, for XylE, it was done in potassium phosphate 100mM, with different concentrations of catechol but the best was 0.03M. In 200ul of reaction mix was added 5um of cell extract.
For CatA gene product the reaction was carrried out in Sodium phosphate buffer 50mM, with the same concentration of catechol, 0.03M aswell. In 200ul of reaction mix was added 5um of cell extract.
Finally BpuI, carried out in citrate buffer 100mM, with 5ul of cell extract in 200ul of reaction mix with 0.4mMof substrate, ABTS.
Week 3rd-9th October
We recieved the long primers to do the phusion proteins, XylE and CatA with FBD1 and FBD10 each.
To do that we follow a standard protocol of Phusion taq PCR, but adding DMSO in the final mix, 1.5%, instead of the corresponding volume of water (0.75ul per reaction of 50ul). The template was the plasmids we already construct to express XylE and CatE. The amplicons, once purified from the PCR mix, were used to do Golden gate cloning to introduce the new fusion proteins in the backbones, the same backbounds as for the rest of genes. And the product of golden gate transformed into DH5-alpha also for blue/white screening, and colony PCR for the positives.
Once we know that the gene is there we followed by cloning them into our strain to do gene expression and express the proteins just as we did for the XylE, CatA and BpuI genes. And tested them together with the others in the same conditions to see if the activity was affected by fusing the FBD to them.
Week 10th-16th October
This week we return to the GFP-FBD, to test the binding strength in different fabrics and un different washing conditions. Teja cut prepare 3 plates for different fabrics to test all the binding domains. After express all the 10 fabric domains only 8 were properly expressed, we realized of this by doing an SDS-PAGE, but also because the results of the experiment show that some samples have the same fluorescence as the negative control, it is to say, no fluorescence. Our "official" negative control didn't work neither, it is to say, GFP wt. Therefore we used as negative control only cell extract of BL21-DE3 which don't express the GFP.
The experiments consisted in:
- 24h incubation with the cell extracts in the plates at 4C.
- measure fluorescence of the cell extract (excitation 475, emmision 515)
- remove cell extract and measure
- wash with the corresponding washing solution (water, PBS, BSA 5%, ethanol 70%, Catechol 0.03M)
- remove the washing solution
- measure
- wash again
- measure
Week 17th-23rd October
WIKI WEEK!
Centre for Research and Interdisciplinarity (CRI) Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor Paris Descartes University 24, rue du Faubourg Saint Jacques 75014 Paris, France
+33 1 44 41 25 22/25
igem2016parisbettencourt@gmail.com
2016.igem.org
