Team:SDU-Denmark/Tour54
Welcome
Interactive wiki-tour
Menu
You really should try our interactive wiki tour instead!
But if you are looking for a specific page - you are in the right place.
Introduction
The Team
Attributions
Gallery
Medal Requirements
Antibody Issue
Antibody Production
Animal Ethics
The Alternative
PAST
Two-Hybrid Screening
System Design
Business
Entrepreneurship
Process
Complete Journal
Collaborations
SOPs
Safety
Results
Characterization
Two-Hybrid System
Submitted Parts
Interlab Study
Practices
Survey
Education
Ethics
A Children's Book
What is Next?
Challenges and Limitations
Medical Aspects
Lab Work
The End
Introduction
The Team
Attributions
Gallery
Medal Requirements
Antibody issue
Antibody production
Animal Ethics
The Alternative
PAST
Two-Hybrid Screening
System Design
Business
Entrepreneurship
Process
Complete Journal
Collaborations
SOPs
Safety
Results
Characterization
Two-Hybrid System
Submitted Parts
Interlab Study
Practices
Survey
Education
Ethics
A Children’s Book
What is Next
Challenges and Limitations
Medical Aspects
Lab Work
The End
"In a dark place we find ourselves, and a little more knowledge lights our way." - George Lucas
Interlab study
Our team has been participating in the iGEM Interlab Study , to help further characterize three different promoters. This was done by using GFP-fluorescence as a way of measuring how strong each of the promoters are compared to each other.
Figure 1:The three constructs made for this experiment.
The three constructs were made through iGEM standard biobrick assembly, inserting gfp into pSB1C3 plasmids containing the different promoters. The gfp-gene were originally cut out of pSB1A2, before ligation with each of the three promotes. Transformation of the constructs was done into E. coli K12 Top10, because it has been optimized for transformation and high levels of plasmid synthetisation. Reference:One Shot® Top10 Chemically Competent E. coli. Catalog number: C4040-10. Visited: 17.09.15. [ThermoFisher] According to iGEM headquarter protocol, biological triplets and technical triplicates was made for each promoter. Each construct has been sequenced and verified with colony PCR. For the complete protocol click here .
Each construct were sequenced; two of the constructs were correctly assembled, the third had the correct promoter, but the remaining did not match gfp.
For the measurements conducted in this project we used a FACSAriaII. The used method of the FACS were flow cytometry which functions through singling out droplets containing one cell each. By exciting the proteins with light on the absorption frequency of the protein the cells will be fluorescent and emission intensity can be measured.
We used E. coli Top10 without inserted plasmids as our negative control and one of our own biobricks; pSB1C3-T18-LinkerGFP for the positive control. Top10 was used to calibrate the FACS and the arbitrary units (a.u.) was measured by gating. Arbitrary units are relative to the measurements, it describes how strong one reading is compared to a defined reference. Reference:Kate B. Bechtol, David K. Hanzel, and Bee Choo Liang, Amersham Biosciences, Inc. - Understanding Fluorescence. [GelifeSciences]
The three promoters are part of a promoter family originally from a combinatorial library made by John Anderson in 2006. WT J23119 was made from synthetic oligonucleotides using overlap extension, and is the strongest of all the promoters in the familyREFERENCE WORD. Reference:iGEM Parts Registry - BBa_J23119:Design. Visited: 17.09.15. [Parts] J23101 is a constitutive promoter mutated form the WT J23119, and contains one point mutation in position -33, subsstituting G with T. J23106 and J23117, like J23101, are derivatives of J23119, and are also constitutive promoters and modified in the same manner. J23106 have three mutations one in the -10 region and two in the -35 region, 8- and -9 plus -33 and -32, respectively. J23117 have two mutations in its -10 region, at position -8 and -12. All three promoters are 35 bp long. Reference:JCAnderson. iGEM Parts Registry - Visited: 17.09.15. [Parts]
Figure 2: The diagram shows the flourescence measured in a.u. by FACS for the 5 constructs. The P-value derived from the data is P<0.0001, marked by ****. J23106+I13504 has the highest flourescence measured, while Top10 has the lowest.
BBa_I13504 is a composite part composed of and RBS-site, a FACS optimized GFPmut3b and two standard biobrick terminators. The GFPmut3b has amino acid substitutions in position 65 and 72, a Glycine and Alanine respectively. Reference:Brendan P. Cormack, Raphael H. Valdiviaa, Stanley Falkow - FACS-optimized mutants of the green fluorescent protein (GFP). DOI:10.1016/0378-1119(95)00685-0. [ScienceDirect]
Top10 is the reference for the a.u. measurements compared to each of the other samples. Our expectations of the promoters are listed from weakest to strongest in relative expression: J23117 < J23106 < J23101, based on the original measurements done in 2006.
In figure 2 a graph comparing each construct’s a.u. to the positive control, T18-Linker-GFP is depicted. The scale is logarithmic due to the relatively large difference in the measured expression levels of the promoters.
Sequence data for the three constructs was compared to the sequence retrieved from parts registry. It was correct for J23106-I13504 and J23117-I13504 , but the data for J23101-I13504 did not show the gene encoding GFP, a BLAST search showed some alignment in the beginning of the sequence, to pFM46. Promoter J23106 showed the highest fluorescence of the constructs, with J23117 having a significantly (p < 0.0001) reduced fluorescence, at 40 times lower.
The J23106 construct showed a much higher fluorescence than the J23117 construct, with a mean value of 85932 a.u. compared to 814 a.u. see figure 2 for graph. The measurement on J23101 is inconclusive in this study, due to the wrongful sequence, possibly caused by contamination.
Next chapter
Next chapter
Dig deeper
End of tour
Want more? Go to iGEM.org!
If you want to learn more about Synthetic Biology you should visit iGEM's homepage, which contains loads on the subject. You can even visit other teams' wikis and take a look at what they accomplished. Or maybe join a team yourself?
