Team:Bio Without Borders/Results
Home
TEAM
Team Members
Collaborations
PROJECT
Description
Design
Notebook
InterLab
Model
Results
Demonstrate
Improve
Attributions
PARTS
Parts Overview
Basic Parts
Composite Parts
Part Collection
SAFETY
H. PRACTICES
Human Practices
Education & Engagement
AWARDS
Applied Design
Entrepreneurship
Hardware
Measurement
Model
Plant
Software
JUDGING FORM
Results
What we accomplished
We succeeded in cloning the two proteins that make up the test system: Factor C and our substrate (GFP-linker-CDB-CBD)
We submitted them to the iGEM registry in pSB1C3.
We attempted to express Factor C in Pichia pastoris, a yeast expression system.
We attempted to express the substrate in E. coli.
Cloning and expression of Factor C
Our first choice was whether to express the Factor C gene from the Atlantic horseshoe crab (Limulus polyphemus) or from the Japanese Horseshoe crab (Tachypleus tridentatus). The latter had been described more fully in the literature. But we wanted to use the one from our homeland. We codon-optimized it for our preferred expression system, Pichia pastoris, and sent it for synthesis. Unfortunately, after several failed attempts, IDT was unable to synthesize it. So we switched to the Japanese horseshoe crab and had it synthesized as G-blocks for later assembly, and also as a complete gene. Our attempts at G-block assembly failed, and we ended up using the full gene synthesized in the plasmid pIDT.
We designed the IDT plasmid to have Biobrick prefix and suffix at either end of the Factor C open reading frame (ORF). We used these sites to clone the ORF into pSB1C3.
We then turned our attention to Pichia. We obtained a vector from BioGrammatics (pJAG-1)that would put the Factor C ORF under the control of the AOX promoter that responds to methanol, which Pichia can use as a substrate. It also adds a signal to translocate the protein from the inside of the cell into the media. This would make harvesting of Factor C easier for us to test. We used a BioGrammatics electroporation protocol and cloning strategy. Seamless Cloning Description Pichia Transformation Protocol: Pichia Transformation Protocol
We got yeast colonies, and picked five for further study. We grew them in methanol as directed, but when we ran a protein gel we did not see any bands in the media. We hypothesize that the large size of the Factor C (greater than 100kd) may have made it difficult to translocate. Our next step is to lyse the cell pellets and look for Factor C in the lysate.
Cloning and expression of Substrate (GFP-linker-CBD-CBD)
We first decided to take two separate Biobrick parets, a GFP and a CBD, and design a linker with overlap to use in NEBuilder cloning. However, we had trouble retrieving some of the parts from the distribution, so we switched our strategy and found a part that was GFP-CBD-CBD (BBa_K12341348). There are 2 unique restriction sites BsrG1 and Kas1 in between the GFP and the first CBD, and we designed a linker that was 2 oligos that when annealed had sticky ends matching these restriction sites. The oligo had the amino acid sequence that Factor C cleaves when activated in its immediate substrate, Factor B. The cloning was successful and we submitted the part in pSB1C3. To identify clones that had the linker, we cut the plasmids with BsrG1 and Pst1, which should give a fragment of 1131bp for the original insert but with the linker should be about 20bp larger. This was a subtle difference so we ran both in the same lane and looked for double bands when the gel was run really far. You can see below we found two clones that showed this. In addition, we moved the part, which was an open reading frame, into pUC18 where it would be behind a promoter and expressed. Right before the wiki freeze we got a positive colony that clearly shows GFP being produced!
