Team:Bio Without Borders/LabBook
Home
TEAM
Team Members
Collaborations
PROJECT
Description
Design
Notebook
InterLab
Model
Results
Demonstrate
Improve
Attributions
PARTS
Parts Overview
Basic Parts
Composite Parts
Part Collection
SAFETY
H. PRACTICES
Human Practices
Education & Engagement
AWARDS
Applied Design
Entrepreneurship
Hardware
Measurement
Model
Plant
Software
JUDGING FORM
Lab Book
This page documents all of our lab notes, featuring various protocols, experiments, and details about our wet and dry lab work that was conducted for our project. We thought it was necessary to document our performance in the lab especially because it highlights the changes that we made to our experimental design.
Week 1 (June 4-8)
•Made competent cells and aliquoted them into 25 1.5 mL tubes (50 µl each)
Week 2 (June 11-15)
•Testing competent cells using iGem pSB1C3.
• Tested to see if they grew better on a freezer block versus an ice bath. They grow better in an ice bath.
• Ran colony PCR and plated bacteria on antimicrobial plate.
• cPCR, chloramphenicol, and culturing protocols
• Ran 1% agarose gel for cPCR
• Plasmid prep of pSB1C3
Week 3 (June 18-22)
•Made chloramphenicol LB plates
•Restriction enzyme digest of iGem plasmid backbone pSB1C3, pSB1A3, and pSB1K3 using EcoR1 and Pst1
•Ligated digested backbones with j04450 insert
•Grew transformed competent cells with pSB1A3 with j04450 insert on ampicillin plate
•Created a 50mg/mL kanamycin solution and then aliquoted it to about 80 1.5mL tubes.
•Inoculated LB with ampicillin colonies (with j04450 insert).
•Streaked ampicillin (1A3) colony onto new plates.
•Conducted cPCR for pSB1A3 colonies with j04450 insert.
•Filled out first draft of safety form
•Went to Brighton Beach to examine horseshoe crabs' habitat and found a part of the carcass
Week 4 (June 25-29)
•Conducted cPCR for pSB1K3 colonies with j04450 insert (the colonies took longer to turn red).
•Ran 1% agarose gel of pSB1A3 with the insert (colonies that were re-streaked).
•Ran 1% agarose gel of pSB1K3 with insert.
•Cleaned laboratory material.
•Set up pSB1C3 backbone amplification PCR using Q5.
•Ran gel of pSB1C3 with Q5 PCR to confirm size.
•Amplification of j04450 insert with Phusion PCR (because Q5 did not work). Confirmed size with gel electrophoresis.
•Plasmid prep of pSB1A3
•Digest of amplified j04450 insert (Phusion PCR).
•Amplified linearized backbone from pSB1C3 and pSB1A3 with Phusion PCR
•Ran a gel for pSB1A3 and pSB1C3 plasmid prep from day before to check band size
•Purified (accidentally-amplified with Phusion) j04450 insert, as well as the circular (made into linear) pSB1A3 and pSB1C3 backbones with PCR clean-up
Week 5 (July 2-6)
•Went to Jones Beach to examine horseshoe crabs' habitat and found a carcass
•Worked on GoFundMe campaign page
•Digested pSB1K3 backbone, ligated it with the J04450 insert (from purified J04450 insert from 6/27), and transformed cells for Kanamycin resistance & plated them
•Ran cPCR for transformed kanamycin resistant colonies
•Coded format for iGEM website
•Inoculated LB broth with transformed kanamycin resistant colonies
•Made iGEM Facebook page
Week 6 (July 7-13)
•Human practices: PCR and Pizza.
•Ran cPCR of pSB1K3 ith j04450 insert.
•Re-did cPCR of pSB1K3 and ran gel (failed again).
•Made chloramphenicol solution, as well as chloramphenicol plates.
•Transformed the cellulose binding domain (BBa_K1478001) from distribution plate into competent cells and plated them on chloramphenicol.
(We need to figure out which primers to so that we can observe a band from cPCR of pSB1K3).
• Inoculated LB broth with transformed cellulose binding domain containing colonies
• Made new VF2 and VR primers and performed cPCR with them for BBa_K1478001 and pSB1K3
• Ran gel for both pSB1K3 and BBa_K1478001 (failed again; a band seemed to form at ~700bp for K1478001), but is not the correct length of 309bp). Maybe we order new primers and change PCR master mix.
• Resuspended from distribution and transformed BBa_E1010(RFP,chloramphenicol backbone), BBa_K648013(GFP, chloramphenicol backbone) and BBa_E0040(GFP, ampicillin backbone) in competent cells.
• Plasmid prep of two pSB1K3 samples and two of cellulose binding domain BBa_K1478001 samples.
• Plated all transformation cells
Week 7 (July 16-20)
• BBa_E1010(RFP,chloramphenicol backbone) and BBa_K648013(GFP, chloramphenicol backbone) transformed plates did not work. BBa_E0040(GFP, ampicillin backbone) transformed plate had two colonies present.
• cPCR was conducted for two BBa_E0040 (GFP) transformed colonies.
•Digested E0040 (GFP) cPCR product (still contained pSB1A3 backbone).
• Ran gel of digested BBa_E0040 (a little shy of 720 bp). (7/17)
• Made plates of LB with chloramphenicol.
• Performed transformation with 1 µl of E1010(RFP) and K648013(GFP), on chloramphenicol. Result: There were no colonies on plates with E1010(RFP),and approximately 3 or 4 colonies grew on plate with K648013(GFP).
• Did cPCR of K648013 (GFP) and plasmid prep of GFP E0440.
• Ran digest of K648013 GFP (unecessary because we accidentally used VF2 and VR during cPCR which only amplified the insert, rather than the backbone, which makes the digest uncessary) and confirmed size with gel electrophoresis
• Did plasmid prep of GFP k648013.
• Performed transformation of k1478001 (CBD) and K648013 (GFP) and plated them. Also inoculated the left over k14 and k648013 with 200 µl of SOC and incubated.
Week 8 (July 23-27)
• Started our Interlab measurement study by performing a transformation of the measurement parts each into 8 tubes (one tube per part) 25 µl DH5-alpha ultra competent cells (New England Biolabs).
• Left the cells to rotate in a 37 C incubator for two hours before plating them on chloramphenicol plates to grow overnight
• Observed colonies in each of the plates and inoculated two colonies from each plate in 5mL of LB+Chloramphenicol and put them on the rotator to grow overnight.
• Made more chloramphenicol plates for the CFU E.coli measurement protocol.
• Remade our chloramphenicol plates for the CFU protocol with appropriate LB agar powder and performed the cell measurement protocol study with our cultures that were grown overnight on Wednesday.
• After troubleshooting, we attempted to perform the Calibration 1 protocol with the Fluostar Galaxy fluorometer, but ran into issues with reading plates and the output data.
• Because we were short on time, we were only able to perform the CFU protocol, and had to put our culture in the fridge overnight and save them for the cell measurement protocol (which requires a 6 hour incubation period) the next day.
• We performed the CFU serial dilution protocol using a spectrophotometer to calculate absorbance values. We redid our dilution the next day when our fluorometer was up and running.
• Called BMG Technologies to troubleshoot Fluostar issues, and were told that the Fluostar Galaxy has 1000x (micro) the scale required for iGEM (milli), so we converted all of our values by moving the decimal place over three digits and changed our data in Excel.
• Performed all three fluorescence & absorbance calibration measurements with clear plates as well as the cell measurement protocol.
• Performed calculations to discover how much microliters of each control and testing device we should place in 16 tubes with varying amounts of LB broth.
• Transferred 500 microliters from the 16 50 mL tubes to corresponding 1.5 mL eppendorf tubes.
• Left the eppendorf tubes in an ice bath to represent 0 hours and placed the 16 50 mL tubes in the incubator for 6 hours.
• Counted colonies, did appropriate CFU calculations and  filled in information on the iGEM measurement forms.
Week 9 (July 30-August 3)
• Took two new transformed colonies from plates we had prepared during our first go at the Interlab study and inoculated for growth overnight.
• During our first run at the Interlab study we were using clear well plates, which may have affected our fluorescence and absorbance values, so for our re-do we decided to order new black well with clear bottom plates. The plates hadn’t arrived yet so we didn’t proceed in the study further. We did prepare all of our tubes and reagents for the following day.
• Our new 96-well plates arrived and we performed the cell measurement study. We also re-performed our calibration studies even though they were not incorrect the first time because we needed to standardize our entire experiment with the new plates. During our first trial we ran into error because our positive control exhibited less fluorescence than our negative control after 6 hours in incubation. Due to time constraints from the previous experiment we also could not wait for the full 6 hour incubation, but the second time we were able to do so, likely giving us better results. We were happy to see that in our second trial the numbers made more sense and our positive was more fluorescent than the negative.
• We were also going to perform our CFU study again because we didn’t dilute our cultures to an absorbance of OD600 = 0.1 the first time we performed the study, however, our cultures at this time may have been too concentrated and we ran into issues with our fluorometer (FLUOstar Galaxy from BMG Technology). We contacted the company who manufactured our fluorometer and tried to do some troubleshooting, however, our fluorometer uses a beam of light that measures at one target point as opposed to measuring in a spiral throughout the well, which meant that if we had too many cells or clumping that our readings were not accurate. Although we tried to dilute our cells as much as possible, the fluorometer had limits as to how far diluted a culture could be before it was unable to take the measurement. In light of this, we decided to use a spectrophotometer.
• We used the Thermofisher Genesys Spectrophotometer to conduct our CFU measurement study. For our four control cultures, we initially created a 1:8 dilution, however, both of our positive controls were too large and so we had to perform various dilutions until we received an OD600 of around 0.1. We performed the serial dilutions and left our plates at 37 C incubation overnight.
• Counted CFUs, did calculations and submitted forms.
Week 10 (August 6-August 10)
• Used NEBuilder to attached psb1c3 backbone to G-block 1 and G-block 2 of limulus factor C (experimental group), for the control group we added the NEB master mix with the NEB control backbone. We transformed the experimental and control group in DH5Alpha and plated the transformations in a chloramphenicol plate for the experimental group and ampicillin plate for the control. We observed colonies in both plates.
• We picked 5 colonies from the plate (some colonies were pink and some were white) with the experimental group (psb1c3 with G-blocks) and inoculated it in 10 microliters of water to initiate colony PCR. After we made two tubes of each colony (1-5) and placed Q5 polymerase in one tube and Taq polymerase in another tube in order to compare the efficacy of each polymerase for colony PCR. After colony PCR we ran a 1% agarose gel of the PCR products to determine whether or not the G-blocks are attached to the psb1c3 backbone.
• Although the 5 colonies we picked from the transformation plate were white and not red (meaning that they did not contain the GFP insert), we confirmed on the gel that the size was too small to have contained the G-Block insert and that the backbone likely recircularized with nothing in it. There are 5 or 6 extra base pairs that may have interfered with the NEBuilder which were added in the event that we needed to digest the blocks, as the EcoRI site is on the end of the block and needed more base pairs. To confirm that we can make good usage of the NEBuilder, we have decided to switch our efforts to making the GFP-Cellulose Binding Domain fusion protein. We confirmed that we had the right part for the cellulose binding domain with sequencing, but our GFP sample was dried during delivery and therefore was never sequenced. We re-did PCR of our GFP sample to send out for sequencing the same day.
• Performed PCR of BBa_k648013 (GFP) and ran gel to confirm size to then be sent for sequencing.
• In an attempt to redo the NEBuilder reaction, we are first digesting the G-Blocks to cut the additional ends added using EcoRI and PstI.
• We transform DH5-alpha with psb1c3 with the G-blocks attached and then plated the transformations and placed them in the iincubator.
Week 11 (August 13-August 17)
• Amplified pSB1C3 backbone using Q5 polymerase and G1002 and G1003 primers.
• Performed Digest of G-Block 1 and G-Block 2.
• After closely examining the sequence of the pSB1C3 backbone, we realized that the backbone was not appropriate for NEBuilder because it did not contain the entire biobrick prefix and suffix.
• Attempted to use NEBuilder to attach G-Block 1 and G-Block 2 to pSB1C3 backbone, however, we ran the wrong block of the thermocycler so it didn't work.
• We made another attempt to use NEBuilder to attach the G-Blocks to pSB1C3, and then performed a transformation of the resulting NEB part.
• We observed the pSB1C3 plates with G-Block 1 and G-Block 2 and noticed that there were several red colonies.
• We performed a transformation for the pUCIDT plasmid with the Factor C gene, and then performed a digest
• We digested pSB1C3 backbone. We also did colony PCR for the plates with pSB1C3 and G-block 1 and G-block 2, and the plates with the pUCIDT plasmid.
Week 12 (August 20-August 24)
• Plasmid preps of the Cellulose Binding Domain part and the GFP orf part in pSB1C3.
• pUCIDT/Factor C digested with EcoR1-HF and Pst1. Digest mixed with pSB1C3 backbone and ligated, followed by transformation into Turbo E. coli
• Performed colony PCR on 6 colonies.
Lab closed August 25-September 8)
Week 13 (September 9 - September 15)
Week 14 (September 16 - September 22)
Week 15 (September 23 - September 29)
Week 16 (September 30 - October 6)
• 10/01/2018 Did a PCR to confirm clone factor C in Psb1c3 1 factor C on psb1c3. Cbd cellulose binding domain. GFP - B.M.
• 10/01/2018 Designed sequencing primers In Snapgene annotated stuff Copied factor C out in snapgene used primer3 to get sequencing primers. - B.M.
• 10/01/2018 There was something weird in our YPD stock so had to make more based off this protocol - B.M.
• 10/01/2018 Picked colonies and inoculated culture ypd media + antibiotic - B.M.
• 10/01/2018 Picked small and big colonies from Pichia+pJAG-F plates labeled them SF1, SF2, ... ,SF10 respectively based on if they where big or small. - B.M.
• 10/01/2018 Each culture was picked and added to a 2ml of YPD + 100x (75µl) G-418 antibiotic. - B.M.
• 10/01/2018 Let these colonies incubate overnight in a spin column. - B.M.
• 10/02/2018 Made 75% Glycerol - B.M.
• 10/02/2018 Found BMY (buffered media for yeast w/o carbon) then Made BMY media - B.M.
• 10/02/2018 Pelleted Pichia cells drained media then resuspended into BMY + Glycerol - B.M.
• 10/02/2018 Made LB + chloramphenicol media - B.M.
• 10/02/2018 Spent time researching the Pichia Pastoris expression vectors - B.M.
• 10/02/2018 Did DNA purification of factor C, CBD, and GFP - B.M.
• 10/02/2018 Transformed turbo+ k1321348 plate5well20a and plated on lb+chlor - B.M.
• 10/02/2018 Developed alternative strategy, to take CBD+gfp backbone from registry - B.M.
• 10/03/2018 Ran around the city trying to get MeOH - B.M.
• 10/03/2018 Pelleted Pichia cells drained media then resuspended into BMY + MeOH - B.M.
• 10/03/2018 Plates containing K1321348 in Turbo didn't form any colonies :( - B.M.
• 10/03/2018 Re-plated on other plates maybe there was a mix-up - B.M.
• 10/03/2018 1st attempt to transform this time 2017 version of GFP-CBD-CBD in Turbo - B.M.
• 10/04/2018 2nd attempt to transform 2017 version of GFP-CBD-CBD in top10 - B.M.
• 10/05/2018 3rd attempt to transform this time 2015 version of GFP-CBD-CBD in top10 - B.M.
• 10/06/2018 picked 5 colonies a,b,c,d,e and inoculated tubes see picture attached - B.M.
Week 17 (October 7 - October 13)
• 10/07/2018 Did mini-prep of GFP-CBD-CBD psb1c3 picked from colonies a,b,c,d,e - B.M.
• 10/07/2018 Also did digest of gfp-cbd-cbd plasmid - B.M.
• 10/08/2018 ran gel of digest see attached - B.M.
• 10-09-2018 double digested gfp-cbd-cbd bsrG1 & kas1 restriction sites - B.M.
• 10-09-2018 annealed piece of linker DNA with Bsrg1 and Kas1 overhangs and Arg Ser in the center - B.M.
• 10-09-2018 ligation of linker into plasmid w/ gfp-cbd-cbd ran 2 control's and 1 experiment - B.M.
• 10-09-2018 Plated gfp-linker-cbd-cbd 4 plates - 2 control plates, 1 gfp-linker-cbd-cbd in turbo E. coli, 1 gfp-linker-cbd-cbd in top10 E. coli - B.M.
• 10-10-2018 picked 6 colonies of gfp-linker-cbd-cbd - B.M.
• 10-10-2018 did plasmid-preps of 6 colonies - B.M.
• 10-10-2018 Prepared DNA for parts submission .  - B.M.
• 10-10-2018 Made parts page for k2860000 Limulus factor C BBa_K2860000 - B.M.
• 10-10-2018 Made parts page k2860003 gfp-linker-cbd-cbd BBa_K2860003 - B.M.
• 10-12-2018 did plasmid miniprep of k2860003 in psb1c3 - B.M.
• 10-12-2018 then digested k2860003 in Kas1 and Bsrg1 - B.M.
• 10-12-2018 used http://primer3.ut.ee to design primers for verification? - B.M.
• 10-12-2018 ran a gel w/ k2860003 in lane 1, k2860003 cut w/ Kas1 in lane 2, k2860003 cut w/ BsrG1 in lane 3. - B.M.
• 10/12/2018 changed Pichia media to BMY+MeOH - B.M.
• 10/13/2018 added MeOH to Pichia media - B.M.
• 10/13/2018 Used NEB Cloner to determine buffer 3.1 - B.M.
• 10/13/2018 Did a double digest of BsrGI,PstI - B.M.
• 10/13/2018 Did ran PCR of plasmid-preps from 10-10-2018 - B.M.
• 10/13/2018 did PCR clean ups of PCR product plasmid-preps from 10-10-2018 - B.M.
• 10/13/2018 Ran gel as verification of insert BBa_K2860003 Part-only sequence (1131 bp) - B.M.
Week 18 (October 14 - October 20)
• 10/14/2018 Gel had two bands where we expected in sample 5 and sample 6. This is promising since sample 6 is what we submitted. - B.M.
• 10/14/2018 spun down Pichia factor can split product into two sets of tubes put one in -20c and one in fridge. - B.M.
• 10/14/2018 added LB + chor to E coli w/ BBa_K2860003 - B.M.
• 10/15/2018 someone messed up and used up all the PCR product in sample 5 :( - B.M.
• 10/15/2018 digested 1/2 of sample 6 to cut gfp-linker-cbd-cbd w EcoR1-HF and PST1 - B.M.
• 10/15/2018 digested pUC18 cut w EcoR1-HF and PST1 - B.M.
• 10/15/2018 ligated gfp-linker-cbd-cbd and pUC18 - B.M.
• 10/15/2018 transformed ligation into E. coli Turbo. Plated on Amp plates. - B.M.
