Team:Bio Without Borders/Protocols
Home
TEAM
Team Members
Collaborations
PROJECT
Description
Design
Notebook
InterLab
Model
Results
Demonstrate
Improve
Attributions
PARTS
Parts Overview
Basic Parts
Composite Parts
Part Collection
SAFETY
H. PRACTICES
Human Practices
Education & Engagement
AWARDS
Applied Design
Entrepreneurship
Hardware
Measurement
Model
Plant
Software
JUDGING FORM
Creating competent Cells
1)Select one colony from a streaked plate of your desired strain (cell grown on regular LB plate?) Grow miniculture in 37C incubator with shaking.
2)Inoculate 100mL of broth with 1mL starter. Grow until absorbance is between 0.3-0.4.
3)Chill culture on ice for 30 minutes.
4)Transfer prechilled 50mL tubes. Pellet cells at 3000g for 15 minutes. Centrifuge for increments of 5 minutes if centrifuge is not refrigerated.
5)Discard supernatant. Resuspend in half total volume in sterile, ice cold 100mM MgCl2. Incubate 10 minutes on ice.
6)Pellet cold (how long?)
7)Discard supernatant. Resuspend in ¼ original volume in sterile, ice cold 100mM CaCl2. Incubate at least 20 minutes on ice.
8)Pellet cold.
9)Discard supernatant. Rsuspend in 2mL sterile, ice cold 85mM CaCl2+15% glycerol. Aliquot in single servings (prechill the microfuge tubes). Store in -80 C.
cPCR
1)Use Taq premade beads or liquid Taq.
2)In the event where we didn’t use Taq beads, we used 12.5 microliters of Taq polymerase
3)Add 2.5 µL of each primer (VF2 and VR)
4)Add 5 µL of template DNA
5)Add water to reach a final volume of 25uL.
The PCR protocol we used was:
Initial step, 98 degree C, 120seconds
Number of cycles: 35
1st step, 98 degree C, 30 Seconds
2nd step, 55 degree C, 30 Seconds
3rd step, 72 degree C, 120 seconds
final step,  72 degree C, 600 seconds
hold: 4 degree C
Digestion and Ligation
Digestion
Create master mix for plasmid backbone:
5 µL of 2.1 buffer
0.5 µL of EcoRI-HF
0.5 µL of PstI
0.5 µL of DpnI*
18 µL of dH20
*Dpn1 helps to digest methylated DNA from PCR reactions. Therefore, Dpn1 isn’t necessary if you are digesting PCR products. Use Dpn1 when digesting backbone.
Into PCR tubes, use:
4 µL of master mix
4 µL of product (25 ng/uL for total of 100 ng)
Digest at 37°C for 30 minutes, then heat kill 80°C for 20 min
Ligation
Into PCR tubes:
2 µL of digested product (25 ng)
Add equimolar amount of EcoRI-HF PstI digested fragment (less than 3 µl, about 2ul)
Add 1 µl T4 DNA ligase buffer
0.5 µl T4 DNA ligase
Water (to reach final volume) (final volume = 10, use 4.5ul)
Ligate at 16°C for 30 minutes, then heat kill 80°C for 20 min
Transform with 1-2 µl of ligation product
iGEM transformations
Materials
dH2O
Ice
50 microL competent cells (one tube for your part and another tube for your control)
DNA (plasmid backbone) for the control
SOC media
antibiotic agar plates
42 C water bath
Resuspension
Locate and mark your well on the distribution plate
Punch hole with pipette tip into foil
Resuspend dry DNA with 10 microL of dH2O
Inoculation and Transformation
Inoculate competent cells (MUST REMAIN ON ICE and be thawed) with 1 microL DNA (resuspended and positive control)
Leave on ice for 30 minutes
Put into 42 C water bath for 1 minute
Put back on ice for 5 minutes
Add 200 microL of SOC to each tube
Incubate for 2 hours on rotator at 37 C
Plating
Each tube needs two plates: one for 20 microL, and one for the remainder.
Incubate overnight.
Making 1% agarose
Weigh 0.5g of agarose mix in a weigh boat
Measure 50mL of TBE or TAE and place it in a microwavable flask
Pour agarose mix into the same microwavable flask
Microwave the solution until the agarose mix completely dissolves
Drop 5µL of ethidium bromide in the solution then pour the solution in the gel electrophoresis chamber. (make sure you stir it)
Add comb and wait for the gel to solidify and cool.
Once cool add buffer (TBE or TAE, be consistent) into chamber
Place 9 µL of each DNA in a new PCR tube with 1 µL loading dye
Place 10 µL of the DNA ladder into a well
Then place DNA into each well.
Wait a half an hour.
Phusion PCR
Phusion PCR:
In total, we will run a 50µL reaction
Take a PCR tube and add:
0.5 µL Phusion (its a polymerase, add last)
1 µL of template DNA
2.5 µL of forward primer
2.5 µL of reverse primer
1 µL of dNTPs
10 µL of HF buffer
32.5 µL of water
PCR Temperatures:
Initial step:   98°C for 30 sec
30 cycles:
Denaturation: 98°C for 10 s
Annealing: 69°C for 20s
Extending: 72C for 40s
Final: 72°C for 10 mins
Hold:  4°C
Centrifuge tube before putting in thermocycler.
Plasmid Prep
New England Biolab Protocol
Prepping plasmids:
Pellet 1mL bacterial culture (centrifuge for 30 seconds). Discard supernatant.
Resuspend pellet in 200 µL Plasmid Resuspension Buffer(B1). (Vortex or pipet to ensure cells are suspended.
Prep the next step before conducting this step!!!! Add 200 µL Plasmid Lysis Buffer (B2) gently invert tube 5-6 times.and incubate for 1 minute at room temp. DO NOT VORTEX!!!!!!
Add 400 µL of Plasmid Neutralization Buffer (B3), gently invert until neutralized and incubate at room temp for 2mins. DO NOT VORTEX!!!!!
Centrifuge lysate for 2-5 mins
Transfer supernatant to the spin column and centrifuge for 1 min.
Re-insert column and add 200 µL of plasmid wash buffer 1. Centrifuge for 1 min.
Add 400 µL of plasmid wash buffer 2 and centrifuge for 2mins.
Transfer column to a clean 1.5mL microcentrifuge tube. Discard the flow through, and do not allow the tip of the column to touch the flow through.
Add 50 µL of DNA Elution Buffer to the center of the matrix. Wait for 1 minute then spin for 1 minute to elute DNA.
PCR Cleanup
New England Biolabs Protocol
Protocol for PCR cleanup
Dilute sample with DNA clean up binding buffer. If the same is 50uL in total add 100uL of binding buffer.
Load sample onto column and close the cap. Spin for 1 minute then discard the flow through.
Re-insert column into a collection tube. Add 200uL of DNA wash buffer and spin for 1 minute.
Repeat step 3.
Transfer column to a clean 1.5mL microfuge tube.
Add 15uL of DNA elution buffer to the center of the matrix. Wait for 1 min and then spin for 1 min.
Q5 PCR
Protocol for Q5® High-Fidelity 2X Master Mix PCR
Component    25 µl   50 µl  Final Concentration
Q5 High-Fidelity 2X Master Mix 12.5 µl  25 µl 1X
10 µM Forward Primer  1.25 µl  2.5 µl 0.5 µM
10 µM Reverse Primer  1.25 µl  2.5 µl 0.5 µM
Template DNA variable  variable  less than 1,000 ng
Nuclease-Free Water to 25 µl to 50 µl
Initial denaturation: 98C for 30 secs
30 cycles: 98°C for 10 secs
71°C for 20 secs
72°C for 40 secs
Final extension: 72°C for 120 secs
Hold: 4-10C
Put 5uL of DNA
Transformation for Competent cells
Transformation Protocol:
Thaw competent cells on ice (usually takes 10-30 minutes).
Add 5 µL of DNA to competent cells and sit for 30 min.
Heat shock at 42°C for 30 seconds.
Add 200 µL of SOC growth media.
Rotate in incubator at 37 C for an hour.
Spread 200 µL of cells onto appropriate antibiotic plate.
Incubate overnight at 37°C.
Pichia Transformation
Pichia Transformation Protocol. Electroporation of BioGrammatics competent Pichia cells, 2017-06-01. bG-Customer Protocol – 0XX.
Expectations
Each “one-shot” tube of BioGrammatics electroporation-competent Pichia pastoris (E-comp) cells has 25 – 30 ul of frozen competent cells. Using the following protocol, electroporation with 2 ul of the 5 ng/ul control transformation control plasmid, pJAG-ARS, should generate ~1000 colonies. This is similar to number of transformants one can obtain with an expression vector linearized in the AOX1 promoter with PmeI to targeted integration into the Pichia genome at the AOX1 locus. Lower transformation frequencies are obtained targeting alternative loci, and with different forms of DNA.
E-comp cells can be stored at -80°C for up to 6 months without significant loss of competency.
Note, the following protocol uses cuvettes with a 1 mm gap width for electroporation; most published protocols for Pichia use 2 mm cuvettes. Furthermore, pre-programmed electroporation settings may be set for 2 mm cuvettes. Protocols for either type of cuvette are acceptable, however, the appropriate electroporation settings must match the cuvette gap width (1 mm or 2 mm). Higher voltages required for the 2 mm cuvettes will cause arching with the 1 mm cuvettes.
DNA
Expression vectors are most often electroporated into Pichia cells as linear DNA molecules. For example, the restriction enzyme Pme I recognizes the 5’-GTTTAAAC-3’ site in the middle of the AOX1 promoter; expression vectors linearized with Pme I preferentially integrate into the AOX1 promoter in the Pichia genome. DNA should be cleaned and concentrated after restriction enzyme digestion, prior to transformation. Re-suspension of the DNA in a solution with low conductivity, i.e. in water, or in low concentrations of Tris/EDTA, is best for electroporation.
�Electroporation Protocol.
1. Label and chill sterile 1 mm electroporation cuvettes in an ice/water slurry at least 5 minutes prior to electroporation. The cuvettes, and sample, should be as close to 0°C as possible at the time of electroporation. Note, the settings provided in this protocol are for 1 mm cuvettes - not 2 mm cuvettes.
2. Remove E-comp cells from -80°C freezer. Thaw and place on ice. Warming tubes by hand during transport from the freezer works well; rapid thawing may actually be slightly better than a slow thaw.
3. Add DNA to cells. Results with DNA volumes up to 5 ul per “one shot” tube (~30 ul) are similar; larger volumes may result in lower numbers of transformants. Most importantly, reduce the amount of salt/ions added to the E-comp cells to minimize conductivity of the sample during electroporation. Water or low concentrations of Tris are best.
Electroporation with 200 ng of a linear expression vector should generate 1000’s of transformants if the AOX1 locus is targeted, efficiencies at other loci vary. More DNA will result in more transformants, up to ~1ug of DNA/sample.
4. Gently mix the DNA and E-comp cells, then, transfer the entire sample to a sterile, ice-cold 1 mm cuvette. Make sure the sample is inserted between the metal plates.
5. Rapidly, place the cuvette between the electrodes in the “shock chamber” of your electroporation device, activate and discharge the device - all in ~5 seconds.
Note, warming of the cuvette/sample can significantly reduce the number of transformants. The time after the sample-cuvette is removed from the ice bath and placed in the Electroporator, until the current is discharged, is critical. This “Ice to Zap” time should be as short as possible.
Electroporation settings:
• BioRad (Gene Pluser Xcell®, MXcell®, II, and E. coli Pulser®): 10 uF capacitance, 600 ohms resistance, and 1150 volts.
• BTX machines (ECM 399 or 630) and the BioRad GenePuler I: 25 uF, 200 ohms and 1150 volts.
• Eppendorf (Eporator®, Multiporator®): 1150 volts (1200V if 1150 is not possible), capacitance and resistance are fixed.
�In general, the electroporation should be conducted at a voltage as close to 1150 volts as possible with a theoretical time constant of 5-6 milliseconds (25 uF and 200 ohms, or 10 uF and 600 ohms). Actual time constants of 4 – 6 milliseconds are best.
6. After the electroporation discharge (zapping the cells), add ~1 ml Pichia Electroporation Recovery Solution (PERS, a 1:1 mixture of YPD and 1 M Sorbitol) to the cuvette and mix it with the cells. Transfer the sample from the cuvette to a tube for incubation. The original E-comp cell tube works well. Inverting the cuvette to draw out the sample from between the electrodes with a pipet tip helps to extract more sample.
7. Incubate samples at at room temperature up to 30°C, shaking at ~100 rpm for ~3 hr. Shorter recovery times will yield fewer transformants; longer times, for example overnight, can result in cell division/“sister” clones.
8. Spread the cells onto the appropriate YPD-agar plates for selection, and incubate the plates at 30°C for 2 - 3 days. Multiple dilutions are recommended For example, plate 10 ul of the sample on one plate, and the 100 ul of the sample on another plate. For “hard to target” loci more of the transformation sample can be plated. Centrifugation at 8k rpm in a microfuge for 30 sec will pellet cells from PERS for plating. [Drug concentrations for selection: G418 at 800-1000 ug/ml, and Nourseothricin (Nat), or Zeocin, at 100 ug/ml].
9. Carefully pick single colonies from the original selection plates and transfer/patch them to a second selection plate to ensure that any “picked” non-transformed cells will not grow. Single colony isolate transformed cells is also recommended. Incubate at 30°C overnight. Subsequent testing should be performed with cells originating from the 2nd selective plate. Furthermore, no selection is required for stably transformed cells in subsequent testing.
10. Glycerol stocks of select clones should be made from cells off the 2nd selective plate. Add sterile glycerol to an overnight YPD culture, (30% v/v glycerol).
Quick Electroporation Protocol Outline. 1) Linearize plasmid DNA (clean and concentrate). 2) Chill cuvette (ice-water slurry). 3) Add DNA to “one shot” E-comp cells; transfer to 1 mm cuvette. 4) Electroporate sample with settings of 1150 V and 5-6 ms (10 uF and 600 Ohms, or, 25 uF and 200 Ohms). Actual time constants of 4-5 ms should be obtained. 5) Add 1 ml of Pichia Electroporation Recovery Solution (PERS) to the cuvette and transfer the sample to a tube for incubation at 30°C, 100 rpm for ~3 hr. 6) Spread samples on selective plates; incubate at 30° C for 2-3 days.
Create Chloramphenicol (plates and solution)
Chloramphenicol is 34 mg/mL of ethanol.
Make appropriate proportion solution in 15 mL tube.
Making chloramphenicol plates with LB -- volume of chloramphenicol solution added to LB broth should be the same volume except in microliters
Measure 250mL of DH2O and add 8.75g of LB powder
Sterilize either in autoclave or in microwave (microwave at low setting in intervals of 5 minutes until boiling).
Add 250 microliters of chloramphenicol into sterile LB.
Pour plates in hood.
Creating Competent Cells
Select one colony from a streaked plate of your desired strain (cell grown on regular LB plate?) Grow miniculture in 37C incubator with shaking.
Inoculate 100mL of broth with 1mL starter. Grow until absorbance is between 0.3-0.4.
Chill culture on ice for 30 minutes.
Transfer prechilled 50mL tubes. Pellet cells at 3000g for 15 minutes. Centrifuge for increments of 5 minutes if centrifuge is not refrigerated.
Discard supernatant. Resuspend in half total volume in sterile, ice cold 100mM MgCl2. Incubate 10 minutes on ice.
Pellet cold (how long?)
Discard supernatant. Resuspend in ¼ original volume in sterile, ice cold 100mM CaCl2. Incubate at least 20 minutes on ice.
Pellet cold.
Discard supernatant. Rsuspend in 2mL sterile, ice cold 85mM CaCl2+15% glycerol. Aliquot in single servings (prechill the microfuge tubes). Store in -80 C.
Digest G-Blocks
G-Block 1 Tube 1:
Add 9ul of G-block 1 to G-block tube 1.
Add 1ul of cutsmart
0.5ul of EcoR1
G-Block 2 Tube 2:
Add 9ul of G-block 2 to G-block tube 2
Add 1ul 2.1 Buffer
0.5 Pst1
Primer Stock
For VF2 and VR primers:
180 microliters of MGBW
31.4 nanomol (0.19 mg) VR
34.8 nano mol (0.21 mg) VF2
10 microliters VR and VF2 to make 5x primer stock
