Team:Warwick/Proof
iGEM Warwick 2016 - Human Practices
Home Team
Project
Overview Design Parts Model Software Progress/Proof of Concept
Labs
Lab Book Safety Interlab Protocols
Attributions
Collaborations
Human Practices
Summary Silver Gold Integrated Practices Education
Awards
Silver Gold
Progress/Proof of Concept
Creating Our System
Design of Plasmids
The first fundamental decision the team had to determine once selecting upon a project was the number of plasmids that must be constructed in order for the system to be operable. We had to be able to experimentally prove that the proteins were transacting, so had to take this requirement into consideration during the design phase – the sgRNA and dCas9 had to be located on different plasmids as they need to interact, whilst the fusion proteins must be located on a third plasmid as they must be able to interact with the sgRNA/dCas9 complex formed from the other two plasmids. This led to the decision to design three plasmids.
Another factor taken into consideration was the type of backbone that would be used for each plasmid. We identified the following aspects of synthetic biology that had to be acknowledged within the design:
- Each plasmid must have a different replication origin. If two had the same, competition between the plasmids would result in only one of the plasmids existing within a cell.
- Each plasmid must be resistant to a different antibiotic in order for individual plasmids to be selected for separately.
- The plasmid copy number must be suitable for it’s function.
Having reviewed these points, we selected pSB1C3, pSB3T5 and pSB4A5 as our backbones, with the plasmids having resistance to chloramphenicol, tetracycline and ampicillin respectively. It became apparent later in the project that there were possible issues with utilizing tetracycline resistance (see Lab Safety page for further detail) and the combination of pSB3T5 and pSB4A5, so we aim to alter our prototype model accordingly in the future. We’ve incorporated the sgRNA sequence into pSB1C3 because the high copy number nature of this backbone ensured high expression rates of the sgRNA, improving binding and therefore detection of our target substance. The fusion proteins have been designed as part of the pSB3T5 backbone, due to its relatively low copy number. This reduces the background expression of the reporter gene so that the change in expression (indicating a positive result) is more obvious. Finally, we used the pSB4A5 plasmid to carry the dCas9 and GFP parts. There is some conflict as to how this backbone will affect expression – according to previously published scientific literature, 4A5 is a low copy number backbone. However, when characterised with RFP and mini-prepped, it appeared to have high copy characteristics. Consequently, the dCas9/GFP plasmid could have a higher than desired background expression, but a greater increase in expression rate. On the other hand, if pSB4A5 behaves as a low copy number plasmid, the colour change would be less obvious and our detection system would be less effective. We had no issues when we utilized pSB4A5 for this purpose, with the apparent high copy number nature resulting in successful expression of the Cas9/GFP plasmid.
Experimental Proof of Successful Plasmid Construction
To confirm that our plasmids had the desired structure, the wet lab team performed a triple transformation of RFP in all three of them, plating them on the three corresponding antibiotics. Red colonies appeared on all plates, indicating successful existence of all plasmids together within the same cells.
Construction of Plasmids 1 and 2
We amplified target genes encoded on double stranded DNA ordered from, using pre-designed primers corresponding to the target gene. These primers were adapted to include Gibson or Golden Gate adaptors, so that an assembly reaction could be used to sub-clone the gene into a new backbone.
All plasmids carrying the inserted gene combinations were transformed into competent Top10 cells, except those with the sigma 54 fusion proteins. The Top10 cells used were appropriate for this task, as they had been made electro-competent and had a rapid growth rate.  Sigma 54 fusion protein plasmids were transformed into Rpoz- cells, which, unlike Top10 cells, don’t produce the sigma 54 protein naturally. Our competent cells had poor transformation efficiency (around 3 x 105), hence when we discovered some issues with construction of the T7 variants, we invested in cells that had a transformation efficiency of 1010.
Transformed cells were plated, and multiple colonies of each variation were inoculated with LB. The colonies selected were those that did not express the red fluorescent protein, as this indicated that the target gene had been sub-cloned successfully. After extracting the plasmids from the cells by mini-prep , a purified sample and the relevant primer were sent for sequencing. We also completed diagnostic digests to gain a greater insight into how effective the construction had been.
Currently, we have managed to fully construct all of the following variants of Plasmid 2:
- Sigma 54 with MS2
- Omega with MS2
- Sigma 54 with PP7
- Omega with PP7
- Sigma 54 with COM
- Omega with COM
We are still working to successfully synthesize the T7 polymerase variants.
Construction of Plasmid 3
Creation of Plasmid 3 was hindered by numerous unsuccessful PCR reactions due to the formation of primer dimers in solution. The sgRNA cassette was amplified from the sgRNA g Block once the antisense primer had been redesigned using …., and Gibson assembly was attempted with a PSB1C3 backbone. Unfortunately, re-circulisation prevented the desired insertion. To resolve this issue, we utilized the compatibility of the biobrick prefix and suffix and the new plasmid backbone with the Xba1 and Pst1 digestion enzyme cut sites. Potentially, this ligation reaction may be affected by insufficient base length at the 3’ end of the sgRNA Pst1 binding site, which would increase the probability of the plasmid re-circularising. We combatted this by performing a digestion reaction using shrimp alkaline phosphatase, removing phosphates from the 3’ and 5’ end of the plasmid backbone. This prevents re-ligation of the two ends, encouraging ligation of the insert instead. Unfortunately, even when using this chemical and commercial cells, there was no significant increase in ligation efficiency.
Proof of Concept
Testing the System – Positive and Negative Controls
Once we obtain the correct form of all three plasmids, we will introduce a PAM staggered promoter sequence into the 5’ end of the GFP gene encoded on Plasmid 1.  Following this, we combined all parts of the system by transforming all the plasmids into a single cell - for our trial, we only tested the PP7 omega plasmid out of the possible 9 combinations of fusion protein. To experimentally test functionality, we altered the sgRNA to incorporate all 20 possible variations of targeting sites, each with a fully formed operation PP7 handle.
We conducted an OD600 and fluorescence assay using 87 colonies taken from a library of 20 sgRNAs transformed into competent cells, carrying the PP7 omega plasmid, and the PAM staggered GFP dCas9 plasmid with a weakened omega promoter. As a negative control, we also transformed sgRNA not targeted towards the PAM staggered sequence. This assay was conducted in a Tecan plate reader, where the first plate was run for 6 hours and 23 minutes before being refreshed in a new 96 well plate. This was to ensure that the cells were measured during log phase. The following graphs illustrates the data collected from the second plate of this experiment.
As shown in Figure 1, the transformed cells had a growth curve following the expected trend. The distribution of OD600 data correlates well with the growth curve of the negative control, with the majority of colonies lying within the error bars of the control. This illustrates that the targeting sites have no negative impact on the ability of cells to replicate. Figure 2 shows that the library sgRNAs did not express GFP at a greater rate than the control samples. When combined with Figure 3, this indicates there is little significant difference in GFP expression, between cells with and without a targeting site. This implies that targeting sites have no effect on expression of GFP, resulting from either the PP7 fusion protein not binding to the sgRNA, or the sgRNA not binding to its target site. Alternatively, the omega protein may not be sufficiently upregulating expression of the gene. By swapping omega and PP7 for other proteins, or by extending the linkers between binding motifs on the sgRNA to reduce steric crowding between proteins, this issue could be resolved.
We are currently testing different sgRNA binding sites within the PAM staggered sequence to gauge whether there are binding sites that generate the greatest difference in expression in comparison to the negative control. As we are still in the experimentation phase, the data will unfortunately not be available prior to the wiki freeze, but we hope to present our findings at the Boston Jamboree. We are also still in the process of investigating the GFP expression of other variants of fusion protein plasmid , to determine whether other combinations show a more significant change in GFP expression, indicating that the system is fully operable. When a more appropriate target site/RBP handle combination is identified, it will be used in the future steps, detailed as follows.
Figure 1: Growth curve of triple transformed cells with sgRNAs from library
Figure 2: Fluorescence assay of GFP taken at an excitation wavelength of 465 nm, with emission at 530 nm
Figure 3: OD600/fluorescence as a measure of GFP fluorescence per cell, monitoring select colonies over a period of 4 hours and 27 minutes
Future Steps
Testing the System – Introducing the sRNA
In a digestion ligation reaction, we aim to subclone the sRNA (sensing RNA) gene, by using biobrick-compatible cut sites, into Plasmid 2. We selected Plasmid 2 (already encoding the fusion protein) as the host to illustrate that it is capable of transacting with the sgRNA plasmid.
Testing the System – In Vivo Formation of the RBP Handle
We plan to test three different variations of Plasmid 3 – 3 types of misfolded RBP handle in conjunction with the target site that resulted in the greatest GFP expression during the positive control investigation. To do this, we will transform all three versions of Plasmid 3 into Top10 cells. Having combined each of these with copies of Plasmids 1 and 2, we will monitor the rate at which the system expresses green fluorescence. Given that all the RBP handles are misfolded, GFP expression should be low (with exact values to be determined) as the binding site should not refold if the RNA being detected is not present.
Testing the System – In Vitro Formation of the RBP Handle
To complete an in vitro test, we will employ a version of Plasmid 2 that does not synthesize the sRNA, combined with each of the three versions of Plasmid 3 separately. Once this system has been freeze-dried onto a piece of filter paper, a concentrated solution of the RNA being sensed will be dropped onto the construct, and GFP expression will be measured. The sRNA (RNA being sensed) will be extracted from an external source, or ordered from a DNA synthesis company. Having gained a positive result during this experimentation period, we endeavor to accurately gauge the concentrations of sRNA required for complete saturation of GFP expression, and the minimum amount that still results in a significant indication. Determining these parameters will enable us to select the version of Plasmid 3 that causes the greater change in colour when sensing RNA is present.
Other Issues
During the course of the project, we encountered several issues that temporarily hindered the progression of the project. By discussing as a team and consulting our advisors, we managed to resolve most problems by introducing alternative viable solutions.
Poor colony PCR success was one of the main issues we faced. Initially, we used Thermo Fisher Taq Polymerase, but the conditions of this reaction resulted in multiple negative results, although diagnostic restriction digests showed positive results. We believe that this problem may be due to human error during the mixing of reagents, or damage resulting from repeated freezing and thawing. To rectify this, we used a 2X Taq Mastermix, reducing the influence of experimental error as fewer active reagents have to be combined.
Having mistakenly selected two restriction enzymes that had compatible overhangs, our progress when trying to insert parts into plasmids was limited. When first trying to introduce the GFP dCas9 (previously assembled in PSB1C3) into PSB4A5, we used the biobrick compatible enzymes Xba1 and Spe1. This digestion resulted in the majority of bacterial colonies containing re-circularised plasmids. Once we realised our error, we used EcoR1 instead of Spe1 as there is a greater difference in their overhangs, hence overlap of the two plasmids was far less common. The colonies synthesized from this second digestion ligation contained the PSB4A5 backbone carrying the GFP dCas9, compatible with the biobrick registry.
Sequencing failures were unfortunately a common occurrence during our project, causing many practical experiments to be repeated. Measures were taken at every possible stage of the construction to ensure the assemblies were as accurate as possible. However, when sent for sequencing, some of the parts had a large amount of variance from the original design. Potentially, these differences could have arisen during any of the synthesis stages – primer design, PCR, gel electrophoresis, Gibson/Goldengate assembly, digestion and ligation, transformation, plating or mini-prepping. These may have been human, systematic or random errors, such as contamination.
Even though we were able to control primer melting point and length using the G-block design tools available, we had to overcome further complications resulting from the uncommon formation of primer dimers. The nuPack tool enabled correction of this by identifying any pairs of primers capable of binding. We therefore altered the PCR conditions for any possibly binding primers – using a higher concentration of DMSO and a buffer rich in guanine and cytosine. This reduced the probability of primer dimers forming, as hydrogen bonding between base pairs is interrupted, enabling PCR to be successful. However, this environment may have resulted in higher than normal error rates, potentially contributing towards the sequencing issues we experienced.
Designing the sgRNA also poised a slight economic problem – for our system to be successful, we had to design and order 120 different combinations of sgRNA, as we required 20 different target sites and 6 different RNA binding protein sites. With each sgRNA being approximately 250 base pairs long, we would have to order 3000 specifically designed base pairs, as well as including spacers between each individual sgRNA in the G block, and primers to bind to each sgRNA. All these factors would have made the sgRNA order very expensive, so to avoid this, we amended our design procedure. Instead of ordering each particular sgRNA, we created a single sgRNA cassette that contained Golden Gate adapters for the insertion of target and binding sites. This meant that to successfully assemble the construct using the Golden Gate technique, we simply had to design and purchase a set of two primers corresponding to the target inserts, with each being around 25 nucleotides long. With only two primers then being required to amplify the sgRNA cassette, we dramatically reduced our costs and the time scale, successfully assembling the 120 different combinations of the sgRNA.
We also found an issue with the version of our sensor altered to detect heavy metals. Our detection system relies upon blocking the dCas9 binding site by introducing a stem and loop structure upstream of the dCas9 handle, altering the tertiary folding of the sgRNA. To ensure the sensor was selective, a metal-specific aptamer was incorporated into the dCas9 hindering loop. Upon complexation with the correspondent ion, the aptamer would undergo a conformational change resulting in the unblocking of the dCas9 binding site. By restricting ions capable of binding, only those corresponding to the particular aptamer included are capable of producing a positive result.
For the system to be successful, the aptamer is required to be linear in its free state and only folded in the presence of the complementary ion. To test the feasibility of the project, we modelled each aptamer in its free state. This led to the unfortunate discovery that the arsenic aptamer was heavily pre-folded and therefore unsuitable. On the other hand, both the mercury and lead aptamers were predicted as linear structures when in free solution when modelled using standard Watson-Crick base pairing calculations. To establish the fundamental binding of the metal ions to the aptamers, we analysed 3D crystal structures using Pymol software. This confirmed that the lead ion was bound by a guanine quadruplex within the aptamer. In contrast, ionic interactions between the mercury ion and each C4-carbonyl of two uracil bases are responsible for the binding of the mercury aptamer to form a linear complex. However, during our research, we found that the lead aptamer was pre-folded via interactions that had not been considered during our earlier modelling. This discovery led the team to initially believe that the lead aptamer would be unsuitable for the project. We overcame this obstacle by adding linkers to each end, causing the aptamer to unfold and become linear.
© Warwick iGem 2016.
Home Team Project Design Parts Safety Attributions Human Practices Awards
