<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Warwick_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Warwick/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Theory by TEMPLATED</TITLE></P><DIV class="inner"><NAV id="nav"><A href="https://2016.igem.org/Team:Warwick">Home</A><A href="https://2016.igem.org/Team:Warwick/Team">Team</A><DIV class="dropdown"><A href="https://2016.igem.org/Team:Warwick/Description">Project</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Warwick/Description">Description</A><A href="https://2016.igem.org/Team:Warwick/Design">Design</A><A href="https://2016.igem.org/Team:Warwick/Parts">Parts</A><A href="https://2016.igem.org/Team:Warwick/Model">Model</A><A href="https://2016.igem.org/Team:Warwick/Software">Software</A></DIV></DIV><DIV class="dropdown"><A href="https://2016.igem.org/Team:Warwick/LabBook">Labs</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Warwick/LabBook">Lab Book</A><A href="https://2016.igem.org/Team:Warwick/Safety">Safety</A><A href="https://2016.igem.org/Team:Warwick/Interlab">Interlab</A><A href="https://2016.igem.org/Team:Warwick/Protocols">Protocols</A></DIV></DIV><DIV class="dropdown"><A href="https://2016.igem.org/Team:Warwick/Attributions">Attributions</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Warwick/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><A href="https://2016.igem.org/Team:Warwick/Human_Practices">Human Practices</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Warwick/Human_Practices">Summary</A><A href="https://2016.igem.org/Team:Warwick/HP/Silver">Silver</A><A href="https://2016.igem.org/Team:Warwick/HP/Gold">Gold</A><A href="https://2016.igem.org/Team:Warwick/Integrated_Practices">Integrated Practices</A><A href="https://2016.igem.org/Team:Warwick/Education">Education</A></DIV></DIV></NAV></DIV><SECTION id="banner"><H1>Design</H1><P>CRISPR/Cas9</P></SECTION><DIV id="brainstorming"><SECTION id="three" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>What is CRISPR/Cas9</H2></HEADER><DIV class="flex flex-2"><ARTICLE><HEADER><H3>The Idea</H3></HEADER><P>CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) are prokaryotic DNA segments containing short repeated base sequences. The CRISPR-Cas9 system has two constituent parts: </P><UL><LI>The Cas9 enzyme This is the ‘active part’ of the system. It essentially acts as a pair of scissors that cut the double-stranded DNA at the target location, enabling editing at a specific point within the genome</LI><LI>The guide RNA (gRNA) The single stranded gRNA ensures that the Cas9 enzyme cuts the genome in the correct location. In nature, the gRNA is found as two single stranded RNA that bind have complimentary sites. It is a small (~ 20 base pairs) modifiable RNA sequence composed of two parts: target-specific CRISPR RNA (crRNA) and auxiliary trans-activating crRNA (tracrRNA).</LI></UL></ARTICLE></DIV></DIV></SECTION></DIV><DIV id="lepto"><SECTION id="four" class="wrapper style1 special"><DIV class="inner"><HEADER class="align-center"><H2>The CRISPR-Cas9 System: Step by Step DNA Cutting in Nature </H2></HEADER><DIV class="flex flex-2"><ARTICLE><HEADER><H3>Cas9 Binding</H3></HEADER><OL><LI>A bacteriophage infects a bacteria, inflicting damage that the cell survives. </LI><LI>The bacteria digests the bacteriophage DNA, producing a copy of a 20nt section of the bacteriophage genome and incorporating it into a CRIPSR array within the bacteria's genome as a protospacer.</LI><LI>A recurrence of infection from the same bacteriophage triggers a response from the bacteria.</LI><LI>The bacteria compares the DNA from the bacteriophage against the CRISPR array, expressing the CRISPR RNA (crRNA) that matches. </LI><LI>The crRNA binds with trans activating CRISPR RNA (tracrRNA), forming the full guide RNA (gRNA).</LI><LI>Cas9 enzyme binds to the Cas9 handle on the tracrRNA section of the gRNA.</LI><LI>The Cas9 enzyme compares the 20nt section of the crRNA against multiple places on the bacteriophage genome, binding when it finds a complimentary section next to a NGG-3' section called the protospacer adjacent motif (PAM).</LI><LI>The Cas9 enzymes expresses dual nuclease functionality, cutting both strands of the DNA it identifies.</LI><LI>The bacteriophage genome is cleaved by Cas9 preventing successful infection of the host cell.</LI></OL><P>This system can be modified by synthetic biologist to allow targeted DNA cutting. The 20nt crRNA section can be edited to allow the Cas9 enzyme to target almost any position on a genome, provided it contains a PAM. By combining the crRNA and tracrRNA into a single guide RNA (sgRNA), editing is simplified and further secondary structures can be added to the sgRNA further altering the function of the system. The Cas9 enzyme can also be edited to reduce its nuclease action so that it only cuts a single strand of the target DNA, or eliminate it entirely so that it acts more as a targetable DNA binding enzyme. Genome editing by CRISPR/Cas9 can range from single base changes, to insertion or deletion of entire genes.</P></ARTICLE><ARTICLE class="nopicture"><HEADER><H3>Genome Editing: A step by step CRISPR guide</H3></HEADER><OL><LI>Target DNA sequence is identified and a complementary strand of gRNA is designed.</LI><LI>Cas9 enzyme binds to the gRNA, targeting it towards a specific DNA sequence.</LI><LI>Cas9 binds to the target PAM site.</LI><LI>Cas9 enzyme cuts both strands of the double-stranded DNA.</LI><LI>When the cell detects the damage inflicted upon the DNA by the dCas9, it attempts to repair the two strands.</LI><LI>Genome mutations induced by flaws in cell DNA repair mechanisms results in a base changes due to non-homologous end joining.</LI><LI>Introducing sections of DNA with similar ends to both sides of the cut allows homology-directed repair to occur, allowing the insertion of the introduced DNA into the cut site.</LI></OL></ARTICLE></DIV></DIV></SECTION></DIV><DIV id="detection"><SECTION id="five" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>Advantages and Disadvantages of CRISPR Technologies in Gene Editing</H2></HEADER><DIV class="flex flex-2"><ARTICLE class="nopicture"><P>CRISPR technology is one of the most recently developed gene editing tools. Its predecessors include ZNFs (Zinc Finger Proteins) and TALENs (Transcription Activator-like Effector Nucleases). The advantages of CRISPR technology compared to these previous methods include:</P><UL><LI>Simplified design targeting – targeting does not rely on unpredictable DNA-protein interactions that are difficult to modify. As the desired binding site can be changed, sgRNAs can be designed easily and cheaply to allow complementary binding to the majority of sequences within the genome.</LI><LI>Simultaneous transcription – as the targeting of Cas9 is programmable, we can design for targeting of multiple sites within the same system.</LI></UL><P>However, as with many existing methods, there are some drawbacks to this form of gene editing. If the sgRNA is not fully complementary to the DNA sequence, it may be unable to bind, preventing the required transcription of the gene. There is also an issue with off-target binding, however as we are using the modified nuclease-deficient Cas9 the potential harm caused by off-target binding is greatly reduced. In systems where highly specific binding is an absolute necessity, as it currently exists, CRIPSR/Cas9 may have too frequent off-target binding for it to be reliable enough in these situations. However there are multiple developments in CRISPR technology to reduce the level of off-target binding: reducing the length of the crRNA can lower off-target binding by reducing overall binding efficiency; or using two single-strand cutting Cas9 targeting opposite strands at the same site to create the double strand cut so that two systems need to fail before an off-target cut occurs.</P></ARTICLE></DIV></DIV></SECTION></DIV><DIV id="nextstep"><SECTION id="three" class="wrapper style1 special"><DIV class="inner"><HEADER class="align-center"><H2>Use of CRISPR/Cas9 in Our System</H2></HEADER><DIV class="flex flex-2"><ARTICLE><P>We elected to use CRISPR/Cas9 as the DNA targeting system in our device over using more conventional DNA binding proteins due to the inherently modular nature of CRISPR. By creating an activation system that relies on CRISPR we create a system whereby modifying 20 nucleotides can target the activation system to completely different genes. Or by including two gRNAs in one device, activate multiple genes simultaneously without the need for introducing further large enzymes. Through this we future-proof our idea by allowing it to be used for a multitude of targets, allowing the other parts of our system to be interchangeable without needing to rework the entire system. Furthermore, as CRISPR relies on folding RNA, it opens up RNA detection as a possibility for methods of detecting stimuli. With the introduction of RNA aptamers that can potentially bind a multitude of compounds, it becomes possible to create a detection system for each of those compounds by only modifying the gRNA while maintaining the rest of the system. As the nature of our detection system requires activation and regulation of transcription, not strand cutting, we use a deactivated version of the Cas9 enzyme – dCas9. This prevents the cleavage of the double stranded DNA, but still allows binding to the specific target site. In the majority of previous research, the role of dCas9 as a repressor has been analysed. In our project, Spirosensr, we aim to investigate and utilise dCas9 as an activator. One of the key features of our detection system is it’s easily altered modular nature. CRISPR/Cas9 technology is well suited to our project, as the cutting and binding of DNA is easily controlled to change specificity.</P></ARTICLE></DIV></DIV></SECTION></DIV><DIV id="brainstorming"><SECTION id="three" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>Why CRISPR/Cas9</H2></HEADER><DIV class="flex flex-2"><ARTICLE class="nopicture"><P>The Warwick iGEM 2016 team dedicated a lot of time to researching novel technologies that were potential project focal points. We elected to use CRISPR technology as it possesses a myriad of potential uses, whilst being a very precise method of genetic manipulation. Given that this field of research is relatively new, we were excited by the concept of investigating potential novel applications of the CRISPR/Cas9 system.</P></ARTICLE></DIV></DIV></SECTION></DIV><DIV id="brainstorming"><SECTION id="three" class="wrapper style1 special"><DIV class="inner"><HEADER class="align-center"><H2>Selecting a Chassis Organism</H2></HEADER><DIV class="flex flex-2"><ARTICLE><P>We elected to use Escherichia coli as our chassis organism. Although the original paper that provided the inspiration for our project (Zalatan et al. 2015) used a CRISPR/Cas9 system as a method of gene activation in eukaryotes, a similar method for activation in prokaryotes did not exist. The idea of 
CRISPR/Cas9 activation in prokaryotes is feasible, however, the execution of the idea is more complex than in eukaryotes - a reversal of the usual roles.In eukaryotes, its a simple matter of fusing the RNA binding protein with a chromatin modifier, increasing the transcription and subsequent translation of the target genes. However, prokaryotes lack chromatin. Despite these potential complications, the otherwise simple nature of modifying prokaryotes make E. coli a suitable chassis.Escherichia coli was selected over chassis such as yeast, which the Zalatan paper used, due to it being a well documented and characterised chassis in synthetic biology, and it's ease of use for transformations as well as its low doubling time, allowing us to increase our throughput and efficiency. </P></ARTICLE></DIV></DIV></SECTION></DIV><DIV id="brainstorming"><SECTION id="three" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>Selecting An Activation Method/Activation of Genes Using CRISPR/Cas9</H2></HEADER><DIV class="flex flex-2"><ARTICLE class="nopicture"><P>Our system of gene activation consists of 3 parts that interact and guide activation to specific regions of the E. coli genome. The activation is done by effector proteins that are guided to the gene that reports the activation. In order to guide the effector protein it is fused to a RNA binding protein (RNABP) that recognises and binds a specific RNA motif. This RNA motif is present in our extended sgRNA molecule that interacts with the CRISPR/Cas9 system targeting upstream of the promoter of the gene to be activated. The sgRNA, and by extension the RNABP-effector fusion, is guided to the target gene by a nuclease-deficient Cas9 enzyme (dCas9). The reporter gene we are using is GFP, that when activated produces a green fluorescent protein that is measurable as a colour change in the cell.We are testing 3 different effector proteins, and 3 different RNA binding domains in order to maximise activation by comparing activity between 9 different effector-RNABP fusions.  </P></ARTICLE></DIV></DIV></SECTION></DIV><DIV id="brainstorming"><SECTION id="three" class="wrapper style1 special"><DIV class="inner"><HEADER class="align-center"><H2>Choosing Parts</H2></HEADER><DIV class="flex flex-2"><ARTICLE class="nopicture"><P>Our system requires the collaboration of many proteins that are not naturally found together. It was therefore important that these proteins were selected very carefully to give our project the greatest chance of success. In the design phase, we considered many candidate parts and combinations. Through a selection process dependent upon the fulfilment of target criteria, and the results of rigorous experimentation, we decided upon our final design. The criteria used to choose the proteins were their use in previous working systems, their tropism in nature, and the strength of the RNA binding.</P></ARTICLE></DIV><DIV class="flex flex-2"><ARTICLE class="nopicture"><H3>RNA Binding Proteins</H3><P>The RNA binding protein (RBP) recognises and binds to a specific RNA sequence. Our CRISPR system will incorporate these RNA sequences in the gRNA, enabling the RBP to specifically target the site where the dCas9 has bound. The RNA binding proteins that we ultimately selected for experimentation were taken from the Zalatan paper, where they were fused with chromatin modifying proteins and targeted towards the 3' end of the gRNA. </P></ARTICLE></DIV><DIV class="flex flex-2"><ARTICLE><H3>MS2 Phage Coat Protein</H3><P>The bacteriophage MS2 is a virus infecting E. coli and other members of the Enterobacteriacea family.  In nature, the coat protein binds a particular RNA stem loop structure in the shine-delgarno sequence of the phage replicase locus in the MS2 genome, preventing translation once the cell becomes saturated with coat protein. A modified version of this coat protein is commonly used for MS2 tagging, a method of RNA detection in living cells where the MS2 coat protein is fused with a Green Fluorescent Protein (GFP) allowing the visualisation of mRNA. The strongly binding nature of the MS2 to the RBS makes this protein suitable for inclusion in our system. http://www.sciencedirect.com/science/article/pii/S1044577397901207 </P></ARTICLE></DIV><DIV class="flex flex-2"><ARTICLE><H3>PP7 Phage Coat Protein</H3><P>The bacteriophage PP7 infects Pseudomonas bacteria and encodes for a coat protein in a similar manner to MS2. The recognition specificity of the PP7 coat protein differs only slightly from that of the MS2 RNA binding protein previously described, targeting a larger stem loop instead with different nucleotide compositions in the loop.  Both MS2 and PP7 coat proteins are excellent candidates for our project, due to their RNA binding being well characterised and having a strong binding affinity with its target sequence. They are also able to discriminately bind in favour of their target RNA, with a 1000-fold stronger binding to target RNA compared to non-target RNA. http://www.nature.com/nsmb/journal/v15/n1/full/nsmb1327.html</P></ARTICLE></DIV><DIV class="flex flex-2"><ARTICLE><H3>COM Protein</H3><P>Com protein originates in the Mu bacteriophage where it acts as a regulator of mom protein mRNA by binding 5' to the open reading frame, altering the RNA structure of the mRNA exposing the translation start site. The Com RNA binding protein itself is only 62 amino acids long and contains a zinc finger like structure allowing it to target specific RNA sequences. Its binding differs from that of PP7 and MS2 RNA binding proteins by the location of its binding in the RNA hairpin. Where PP7 and MS2 bind along unpaired nucleotides along the length and end of the hairpin, Com binds to the base of the hairpin. This difference provides some variety in the RNA binding proteins we are testing while maintaining specificity and binding strength. http://www.ncbi.nlm.nih.gov/pubmed/10665835 </P></ARTICLE></DIV></DIV></SECTION></DIV><DIV id="brainstorming"><SECTION id="three" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>Activator Proteins</H2></HEADER><DIV class="flex flex-2"><ARTICLE><H3>T7 Phage RNA Polymerase</H3><P>T7 RNA polymerase (T7 RNAP), found in T7 bacteriophage, catalyses the transcription of RNA from DNA. As well as extremely high specificity for its promoter sequence, T7 RNA polymerase can copy up to 15,000 nucleotides with a very low error rate. T7 RNA polymerase is orthogonal to native transcription and the T7 promoters are tightly off in the absence of the controller, ensuring that only the desired effector gene will be transcribed. On top of this, T7 RNAP pauses less frequently, and for a shorter time, during transcription than other comparable polymerase enzymes. These factors make T7 RNAP an excellent choice for an activator protein in our project.The groove on the enzyme surface provides the main binding site for DNA. The large size and shape specificity of this region generates a strong Van der Waals attraction between the enzyme and the DNA. An additional six hydrogen bonds between three enzymatic residues (one threonine and two arginine) and the unpaired bases in the uncoiled DNA molecule further increases binding strength. As well as these surface interactions, there is partial Pi-Pi stacking within the folds of the polymerase structure itself, specifically between multiple T7RNAP aromatic amino acids and the unpaired DNA bases. These factors contribute to the strong binding strength of T7RNAP to its consensus promoter, making it suitable for our needs.http://www.pnas.org/content/109/18/7085.fullhttp://mds.marshall.edu/cgi/viewcontent.cgi?article=1112</P></ARTICLE></DIV><DIV class="flex flex-2"><ARTICLE><H3>ω factor </H3><P>Omega protein is a subunit of the core RNA polymerase enzyme. It is the smallest subunit in the RNAP complex, where it stabilises the assembled enzyme. By bringing this enzyme closer to the transcription start site of the reporter gene, it will facilitate transcription of the gene by stabilising any RNAP complex that forms around it. By using a weakened promoter, background expression of the gene is reduced when there is no omega factor present. </P></ARTICLE></DIV><DIV class="flex flex-2"><ARTICLE><H3>σ-54 factor</H3><P>Sigma factors are proteins necessary for transcription initiation. Each sigma factor recognises and catalyses the transcription of its specific transcription start site. Sigma-54 in nature in encoded by the rpoN gene that is an important regulator of genes involved in nitrogen metabolism. It is found in some strains of E. coli, however not in the ones that we intend to use to test the system. By guiding the protein near its promoter site, we hope to increase the rate of transcription of the gene far over its background transcription rate.</P></ARTICLE></DIV></DIV></SECTION></DIV><SECTION id="three" class="wrapper style1 special"><DIV class="inner"><HEADER class="align-center"><H2>Designing the Fusion Protein</H2></HEADER><DIV class="flex flex-2"><ARTICLE><P>The fusion protein is designed to include the relevant domains from each of the two fused proteins and a glycine-serine linker providing flexibility between the domains so that the effector can act semi-independently of the RNA binding domain.</P></ARTICLE></DIV></DIV></SECTION><SECTION id="three" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>Selecting Promoter Sequences</H2></HEADER><DIV class="flex flex-2"><ARTICLE><P>In order to control the expression of our reporter, and minimise background activation, our promoters were designed specifically to reduce their efficiency under normal conditions. Furthermore, as we are testing 3 different effector proteins, they each require distinct promoters. We selected weakened sigma-54, omega, and T7 promoters among previously characterised promoters. As this type of activation has not been attempted before, there is little data on the ideal position upstream of the promoter in which to bind our dCas9-sgRNA-RNABP-effector complex. For this purpose we designed a sequence ahead of the promoter containing multiple PAM sites and PAM proximal regions that are individually targetable with little similarity between regions, reducing the risk of off-target binding within this area. We have dubbed this sequence the Multiple Aligned PAM Proximal Sequence (MAPPS). This allows us to test different binding sites upstream of the promoter in order to determine at what distance activation is at its highest.</P></ARTICLE></DIV></DIV></SECTION><SECTION id="three" class="wrapper style1 special"><DIV class="inner"><HEADER class="align-center"><H2>Selecting the Reporter Gene</H2></HEADER><DIV class="flex flex-2"><ARTICLE><P>The reporter gene is expressed when the system activates gene expression successfully. For this we needed to select a reporter whose expression is easily measured, is a fairly small gene reducing load on the cell, and does not harm the cell when expressed. For this we had two main candidates: Green Fluorescent Protein (GFP) and malachite green aptamer.GFP is a well characterised protein originally extracted from jellyfish that is commonly used in many different synthetic biology settings as a reporter. GFP is a fairly small protein, only 26.9 kDa in size, and from previous experiences throughout iGEM it does not harm the cell through expression.Malachite green aptamer is a small RNA strand that itself is not fluorescent, however in the presence of the organic compound malachite green it forms a complex fluorescing brightly. In order to measure the expression of the malachite green aptamer, malachite green compound must first be added. Malachite green compound itself fluoresces faintly, however in the presence of the aptamer fluorescence multiplies up to 2360-fold, comparable with the fluorescence of GFP. The aptamer is only 38 bp long, meaning it inflicts very little pressure on the cell minimising cell load, however the malachite green compound is known to be toxic to cells in sufficient concentration. In order to determine the optimum concentration of malachite green that causes clear fluorescence over background fluorescence while not damaging the cells, the aptamer was characterised by the team.In order to test our system in vivo, GFP was selected as the primary reporter gene, however, further down the line, when the device is expressed in an in vitro transcription/translation system mounted on filter paper it is worth revisiting the issue. GFP expression requires both transcription and translation delaying measurement while malachite green aptamer only requires transcription so that results are evident within minutes instead of hours. Furthermore in an in vitro system it is possible that malachite green toxicity becomes a non-issue.  </P></ARTICLE></DIV></DIV></SECTION><SECTION id="three" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>Designing the gRNA</H2></HEADER><DIV class="flex flex-2"><ARTICLE><P>As a fundamental part of our system, designing our gRNA within specifications is very important for ensuring the function of the device. For this we used Nupack to design and fold our gRNA within specification to prevent unnecessary modification to the basic gRNA system that we know works.</P></ARTICLE></DIV><DIV class="flex flex-2"><ARTICLE><H3>RNA Binding Protein Motifs</H3><P>In order to create our detection system it was necessary to add RNA binding protein motifs to the gRNA. However we encountered multiple design decisions while incorporating these motifs into the CRISPR/Cas9 system. The first choice was selecting where in the gRNA to add the motifs. In papers it has been shown that modifying the 5' end of the gRNA, past the 20nt crRNA region, does not negatively affect the binding or activity of Cas9. Furthermore it has been shown that adding nucleotide linkers between the Cas9 handle and the terminator loop of the tracrRNA similarly does not negatively impact Cas9 activity. We elected to place the motifs between the Cas9 handle and the terminator loop. This was based on the binding of Cas9 to its target site. When Cas9 binds, it binds with the 5' end of the gRNA upstream of the target. As our target is upstream of the promoter region of our reporter, by placing the RNA binding protein motif further towards the 3' end of the gRNA it is placed closer to the promoter region, potentially aiding the fusion protein in activating the expression of the gene.</P></ARTICLE></DIV><DIV class="flex flex-2"><ARTICLE><H3>Single guide or cr/tracr RNA </H3><P>The first design decision we made was whether to use the CRISPR system as it appears in nature - where the crRNA that selects the target for the Cas9 to bind to, and the tracrRNA, that allows the 
Cas9 to bind to the gRNA are two separate entities – or to fuse the crRNA and tracrRNA into a single guide RNA (sgRNA) that combines the function of both and is now the standard in CRISPR research. Considering the modifications it is necessary to make to the gRNA in order to combine it with RNA binding protein motifs it is worth considering both option. If we were to use a two-part system, the tracrRNA would the only part that needed to be modified with the extra stem and loop structure to incorporate the binding motif. The crRNA could be kept the same in the case of RNA detection, or modified separately to include a metal aptamer domain. This simplifies the system in a setting where multiple target sites are required to be activated by the same trigger, as multiple crRNAs could be encoded targeting different sites while a single tracrRNA that is sensitive to a single trigger is maintained. Or potentially multiple tracrRNA sensitive to different triggers could be expressed while only one crRNA is needed to activate a reporter. This would be inefficient in a system where only one trigger and response is necessary as sgRNAs have been shown to guide and activate Cas9 quicker in these cases. It is also a disadvantage that in this system triggers and responses cannot be tied together, as without further modification any tracrRNA would complex with any crRNA.  Using sgRNA each target site and Cas9 handle / RNA binding motif would need to be expressed as a single RNA increasing the workload of designers in cases where multiple sgRNA are expressed. In systems that have one trigger and one response this would be far more efficient however. Furthermore, in systems where individual responses are tied to the stimuli, using sgRNA would be far better, as cross activation of one targeted site across sgRNAs is extremely unlikely. For example, if in response one form of foreign RNA the system needed to activate expression of GFP, but in response to another RNA the system needed to activate expression of RFP, sgRNA are the preferred choice. In a system where in response to a single stimuli, expression of multiple genes at disparate loci is required it is worth considering using cr/tracr RNA. In our prototype version of our sensor we are only seeking to measure a single stimuli, presence of a specific foreign RNA or a specific metal ion. As a result we have elected to use sgRNA as the basis for our detection system.</P></ARTICLE></DIV></DIV></SECTION><SECTION id="three" class="wrapper style1 special"><DIV class="inner"><HEADER class="align-center"><H2>Terminator Loops</H2></HEADER><DIV class="flex flex-2"><ARTICLE><P>When designing our sgRNA we came across the problem of selecting terminator loops. In the original CRISPR/Cas9 system, the gRNA contains a S. pyrogenes terminator at the 3' end of the gRNA. While this has been shown to retain terminator function when placed in organisms other than S. pyrogenes, we elected to replace this terminator in our system with more conventional E. coli terminator in order to guarantee function. However when we modeled the folding of the terminator in Nupack we found that the removal of the S. pyrogenes terminator affected the folding of the Cas9 handle. The correct structure of the Cas9 handle returned upon returning the S. pyrogenes terminator.However, when placing the RNA binding protein motif between the Cas9 handle and the S. pyrogenes terminator, the structure of the handle was once again disrupted. This was eventually worked around by maintaining the  Cas9 handle-terminator structure while truncating four uracil from the S. pyrogenes terminator loop, preventing proper transcription termination. The RNA binding motif could then be appended to the 3' end of the gRNA without affecting the rest of the gRNA structure. The gRNA was capped at the 3' end with a E. coli terminator.</P></ARTICLE></DIV></DIV></SECTION><SECTION id="three" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>Linkers</H2></HEADER><DIV class="flex flex-2"><ARTICLE class="nopicture"><P>A nucleotide linker was added between the RNA binding motif and the Cas9 handle-terminator structure in order to introduce physical space between the binding sites of the dCas9 enzyme and the RNA binding protein. This was introduced to prevent interference with the binding of either protein by the presence of the other.</P></ARTICLE></DIV></DIV></SECTION><SECTION id="three" class="wrapper style1 special"><DIV class="inner"><HEADER class="align-center"><H2>Modelling Detection of sRNA</H2></HEADER><DIV class="flex flex-2"><ARTICLE><P>In order to make the system activate only upon the detection of specific foreign RNA (dubbed sRNA), the structure of either the Cas9 handle, crRNA region, or the RNA binding motif must be disrupted when no sRNA is present. Then in the presence of sRNA the structure must then reform.Based on our previous issues with maintaining the structure of the Cas9 handle, it as decided that the structure of the RNA binding motif would be disrupted. This was done by introducing a stretch of RNA between the first terminator loop and the binding motif that would bind to the nucleotides making up the motif disrupting the structure. Further nucleotides were introduced at either end of this disrupting sequence that were complimentary to the sRNA that was to be detected. When the sRNA is introduced to the system, these complimentary regions (dubbed sensing regions) would bind to the sRNA, preventing the disrupting region from binding to the motif, allowing the motif to reform its original structure. This was modelled on Nupack in order to determine the lengths of the disrupting region and the two sensing regions. The binding energy of the disrupting region must be greater than the binding energy ====of the motif, and the binding energy of both sensing regions must be greater than that of the disrupting region.</P></ARTICLE></DIV></DIV></SECTION><SECTION id="three" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>Activation via Aptamers</H2></HEADER><DIV class="flex flex-2"><ARTICLE class="nopicture"><P>The previous system works only for detecting sRNA. In order to detect metal ions, incorporating a metal ion aptamer that disrupts the structure of the sgRNA under normal circumstances and reforms the structure when a metal ion binds was necessary. However it was unfeasible to use any RNA folding prediction program to predict the folding of the metal aptamer when the metal has bound. From papers it was clear that the ordinarily linear aptamer scrunches up into a complex that was impossible to predict. From this it was decided that attempting to reform either the Cas9 handle or the RNA binding protein motif was unlikely. But, forming a structure with the aptamer and the crRNA region similarly prevents sgRNA activity and 3' additions to the sgRNA do not affect Cas9 binding. By adding the aptamer to the 3' end of the sgRNA it can bind to the crRNA preventing activity, and as it is only within the proximity of a single structure it is less likely to cause structural issues when ions bind. When an ion binds the aptamer fold in on itself, unbinding the crRNA.</P></ARTICLE></DIV></DIV></SECTION><SECTION id="three" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>Metal Aptamers</H2></HEADER><DIV class="flex flex-2"><ARTICLE><P>Initially, our group elected to construct detection systems for the free ion forms of mercury, arsenic and lead, due to the significant impact they have on human life and other ecological systems. Despite the lack of affordable sensors at present, we found that the aptamers for binding these elements were well documented and readily available, so we chose to incorporate these into our design. In order to produce a metal detecting biosensor, it is necessary to modify our existing sgRNA to make it compatible with metal aptamers. In a recent paper it was found [4] that appending nucleotides to the 5’ end of the sgRNA did not significantly affect the binding of Cas9. In our design, these additional nucleotides are complementary to the targeting region of the sgRNA, so forming an additional stem and loop that represses dCas9 activity. By incorporating known metal binding aptamers into the new sensing region formed by the extra nucleotides, the presence of specific metal ions results in a conformational change. This results in the de-repression of our system, allowing dCas9 to bind and the reporter genes to be activated. The left aptamer binds mercury via the complexation of the C4-carbonyl of two uracils and the divalent mercury(II) cation. When mercury is not present, the aptamer is a linear unfolded RNA strand. However, up to ten mercury ions can be bound by each aptamer, with the RNA folding into a symmetrical ‘mercury locked hairpin’[4]. At a pH greater than 7.5, hydroxide ions interfere with the likelihood of a metal ion binding with its aptamer. This is because a hydroxide complex forms, reducing the effective mercury concentration. A similar concept applies in a pH less than 7 – the nitrogen atoms within uracil become protonated, reducing its affinity with Hg2+. For this reason, our detector would be most sensitive within the pH range of 7  and 7.5. The aptamer on the right binds lead (II) ions through ionic interaction with eight guanines, to form a lead-guanine quadruplex. Like the mercury aptamer, the lead aptamer is naturally unfolded, but binds a single Pb2+ ion in a square prismatic structure using the guanine C6 carbonyls. Other metals, such as potassium and sodium, are also capable of triggering quadruplex formation. However, they have a much lower affinity and form less stable structures. Addition of cyanide, thiocyanate or 18-crown-6-ether significantly increase aptamer selectivity for lead.[3] [http://www.who.int/mediacentre/factsheets/fs379/en/ (World Health Organisation accessed on 30/07/2016) https://www.dropbox.com/home/iGEM%202016/Metal%20ap.%20project?preview=MERCURY_DATA_PAPER.pdf[4] L.Trasande, P. J. Landrigan, C. Schechter, Environ. Health Perspect., 2005, 113, 590 – 596</P></ARTICLE></DIV></DIV></SECTION><SECTION id="three" class="wrapper style1 special"><DIV class="inner"><HEADER class="align-center"><H2>Why Use A Paper Based Sensor</H2></HEADER><DIV class="flex flex-2"><ARTICLE class="nopicture"><P>When developing a frontline diagnosis tool, the timing of diagnosis is critical when considering options for treatment. Current Lyme disease, and other spirochete disease, testing techniques require samples to be sent to an analytical lab – delaying results, potentially leading to complications for the patient. Less economically developed places in the world do not have access to these specialist laboratories, preventing the poor who are more likely to come into contact with tick-borne diseases from being diagnosed and subsequently treated. Furthermore analytical tests are expensive to carry out, separating populations from treatment based on their wealth. To counteract this, we designed a portable and stable framework which allows the test to be conducted anywhere in the world, cheaply and quickly. This would enable the test to be more widely and immediately available to a greater demographic. Paper based sensing has been tested on diagnosis kits for Zika virus and Ebola, it is only natural that the technology becomes cheaper over time so that more and more diseases will be diagnosable by these tests. While currently freeze-drying an in vitro transcription/translation kits carries a considerable set up cost, the equipment required is already common in well-stocked labs. Cost is further reduced by the small size of the sensors, increasing the throughput of each cycle of freeze-drying reducing the cost per test. Being able to mass-produce diagnosis tools is far preferable to lengthy individual diagnoses that currently dominate.  The paper based sensor in our design consists of a freeze-dried  transcription/translation system that contains the genes encoding our device and the compounds that allow it to report activation, while being able to be stored en masse in dry areas at room temperature. This makes transport and storage of testing kits cheap and easy, allowing people in areas that may lack refrigeration to keep a stock of tests at all times. Samples can be collected and tested at the same time, so that transporting samples to labs is unnecessary and diagnosis is immediate. </P></ARTICLE></DIV></DIV></SECTION><FOOTER id="footer"><DIV class="inner"><DIV class="flex"><DIV class="copyright">© Warwick iGEM 2016.
                <P>We thankfully acknowledge generous funding support from our sponsors below.</P></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>