<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Warwick_Software skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Warwick/Software</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Warwick 2016 - Page</TITLE></P><DIV class="inner"><NAV id="nav"><A href="https://2016.igem.org/Team:Warwick">Home</A><A href="https://2016.igem.org/Team:Warwick/Team">Team</A><DIV class="dropdown"><A href="https://2016.igem.org/Team:Warwick/Description">Project</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Warwick/Description">Description</A><A href="https://2016.igem.org/Team:Warwick/Design">Design</A><A href="https://2016.igem.org/Team:Warwick/Parts">Parts</A><A href="https://2016.igem.org/Team:Warwick/Model">Model</A><A href="https://2016.igem.org/Team:Warwick/Software">Software</A></DIV></DIV><DIV class="dropdown"><A href="https://2016.igem.org/Team:Warwick/LabBook">Labs</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Warwick/LabBook">Lab Book</A><A href="https://2016.igem.org/Team:Warwick/Safety">Safety</A><A href="https://2016.igem.org/Team:Warwick/Interlab">Interlab</A><A href="https://2016.igem.org/Team:Warwick/Protocols">Protocols</A></DIV></DIV><DIV class="dropdown"><A href="https://2016.igem.org/Team:Warwick/Attributions">Attributions</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Warwick/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><A href="https://2016.igem.org/Team:Warwick/Human_Practices">Human Practices</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Warwick/Human_Practices">Summary</A><A href="https://2016.igem.org/Team:Warwick/HP/Silver">Silver</A><A href="https://2016.igem.org/Team:Warwick/HP/Gold">Gold</A><A href="https://2016.igem.org/Team:Warwick/Integrated_Practices">Integrated Practices</A><A href="https://2016.igem.org/Team:Warwick/Education">Education</A></DIV></DIV></NAV></DIV><SECTION id="banner"><H1>Software</H1><P>Tunable gRNA Designer</P></SECTION><SECTION id="three" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>Abstract</H2></HEADER><DIV class="flex flex-2"><ARTICLE><P>We have created a software tool that uses a DNA sequence input, and designs a list of gRNA sequences as output. These gRNAs will bind to their target DNA with a strength specified by the user, as a percentage of the maximum binding strength of the dCas9 protein to that DNA site. It also calculates the approximate structure of the gRNA and displays first the ones which disturb the Cas9 handle the least. To achieve these outcomes, it incorporates open source code from RNAfold [1] and code developed by Iman Farasat and Howard M. Salis [2], generously provided by them. The aim is to use this tool for de novo design of CRISPR-repressed (CRISPRi) and CRISPR activated (CRISPRa) promoters, as there is currently no method available for designing promoter sequences of pre-defined repression/ activation strengths.</P></ARTICLE><ARTICLE><DIV class="image fit"><P>Fig. 1: The fold change for the dCas9 [3] is plotted against its binding energy as calculated by the Cas9 Calculator.</P></DIV></ARTICLE></DIV></DIV></SECTION><SECTION id="three" class="wrapper style 1special"><DIV class="inner"><HEADER class="align-center"><H2>1. Calculation of the binding energy</H2></HEADER><DIV class="flex flex-2"><ARTICLE><P>The Cas9Calculator, written by Iman Farasat and Howard M. Salis and published in their latest paper [2], is used for all calculations determining the strength of gRNA:DNA binding. For its creation they analysed datasets from different experiments and took into account the effects of supercoiling on adjacent DNA sites, different PAM sites, gRNA:DNA site mismatches, Cas9 and gRNA expression levels, organisms and growth conditions. This allowed them to predict approximately the dynamics of dCas9-based binding and Cas9-based cleavage at all DNA sites.</P></ARTICLE><ARTICLE><DIV class="image fit"><P>Fig. 2: One over the repression activity of the dCas9 [4] is plotted against its binding energy as calculated by the Cas9 Calculator.</P></DIV></ARTICLE></DIV></DIV></SECTION><SECTION id="three" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>2. Relation between fold change and binding energy</H2></HEADER><DIV class="flex flex-2"><ARTICLE><P>
The fold change is the fold difference in gene expression between the case when the dCas9 is bound to the promoter and when it's not bound. Since binding of dCas9 in a CRISPRi system prevents transcription by blocking access to the promoter, the strength of binding of dCas9 determines the fold-repression.To estimate the relation between these, data from two papers is used. For a given DNA sequence one paper presents the fold change in binding of dCas9 for different number of mutations [3]. The second paper presents the repression activity of a single mutation at different locations[4].In both cases the RNA:DNA binding is calculated with the Cas9Calculator and then plotted on a logarithmic scale against the fold change (Fig. 1) or one over the repression activity, respectively (Fig. 2). With the exception of few points, correlations seems to be linear and hence the relation is fitted with the following formula:ùëÉ=ùëí^(‚àíùõΩ(ùëöùëéùë•ùê∫‚àíùê∫))Where maxG is the Gibson free energy when there are no mismatches and G is the energy for the software-designed sequence. P is the fold change between the two states. The coefficient %beta = 1.003074571 is the average of all individual coefficients satisfying the formula (with the exception of the few points noted above).</P></ARTICLE><ARTICLE><DIV class="image fit"><P>Fig. 3: (a) Starting interface for the web tool (b) Starting interface with the user‚Äôs input.</P></DIV></ARTICLE></DIV></DIV></SECTION><SECTION id="three" class="wrapper style1 special"><DIV class="inner"><HEADER class="align-center"><H2>3. The Algorithm</H2></HEADER><DIV class="flex flex-2"><ARTICLE><P>Once the user inputs a DNA sequence and the desired fold change, the software calculates the corresponding RNA+dCas9:DNA binding energy required to achieve that change. Then, starting from the perfectly matching gRNA, a random number of mutations is chosen and then for each mutation two further random numbers indicate its position and the replaced nucleotide (A/T/G/C). This process is repeated 10,000 times where the binding energy of each sequence is calculated using the Cas9calculator. At the end of this step, only sequences with energy within the allowed range are selected.Next, the dCas9 handle and terminator are added to the gRNA and RNAfold is used to calculate the structure of the final sequence. This is to determine whether the gRNA will disturb the structure of the Cas9 handle by hybridising with it. For higher binding strengths (closer to 100% of maximum), the output gRNA sequence mainly depends on the initial DNA input, the tool cannot enforce an unstructured DNA-binding region. Therefore, we selected the sequences with at least half of their bases unpaired and list them in ascending order of structure in the DNA-binding region, for the user to make their own choice based on other design criteria.</P></ARTICLE></DIV></DIV></SECTION><SECTION id="three" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>4. Next Steps</H2></HEADER><DIV class="flex flex-2"><ARTICLE><P>Our aim is to launch a web-based tool which will run the code described above in real time making it easier to use, and reaching a wider range of users. So far, a web interface has been created and the code has been successfully hosted on a webserver kindly provided by the Institute of Systems &amp; Synthetic Biology (iSSB), Genopole, France (See snapshots Fig. 3,4,5). The tool could be further improved by increasing its precision or decreasing the time it takes to return results. This could happen by fitting the data from section 2 with a different function or developing a more efficient algorithm for creating gRNA sequences. While the idea behind this tool is simple, it could still be very effective in tunable engineering of CRISPR/Cas9 systems.The tool is currently available at <A href="http://rikpu.lnx.warwick.ac.uk/">http://rikpu.lnx.warwick.ac.uk/</A> and <A href="http://warwickigem2016.issb.genopole.fr/">http://warwickigem2016.issb.genopole.fr</A> 
*We thank Warwick ITS and iSSB for hosting our web tool, and especially thank David Smyth (Warwick) and Joan H√©risson (iSSB) for all the help.
</P></ARTICLE><ARTICLE><DIV class="image fit"><P>Fig. 4: Confirmation page for submission of input</P><P>Fig. 5: Results page</P></DIV></ARTICLE></DIV></DIV></SECTION><SECTION id="three" class="wrapper rtyle1 special"><DIV class="inner"><HEADER class="align-center"><H2>References</H2></HEADER><DIV class="flex flex-2"><ARTICLE class="nopicture"><P>[1] Lorenz, Ronny, Stephan H Bernhart, Christian H√∂ner zu Siederdissen, Hakim Tafer, Christoph Flamm, Peter F Stadler, and Ivo L Hofacker. 2011. ‚ÄúViennaRNA Package 2.0.‚Äù Algorithms for Molecular Biology 6 (1): 26. doi:10.1186/1748-7188-6-26.[2] Farasat, Iman, and Howard M. Salis. 2016. ‚ÄúA Biophysical Model of CRISPR/Cas9 Activity for Rational Design of Genome Editing and Gene Regulation.‚Äù PLOS Computational Biology 12 (1): e1004724. doi:10.1371/journal.pcbi.1004724.[3] Bikard, David, Wenyan Jiang, Poulami Samai, Ann Hochschild, Feng Zhang, and Luciano A Marraffini. 2013. ‚ÄúProgrammable Repression and Activation of Bacterial Gene Expression Using an Engineered CRISPR-Cas System.‚Äù Nucleic Acids Research 41 (15): 7429‚Äì37. doi:10.1093/nar/gkt520.[4] Qi, Lei S, Matthew H Larson, Luke a Gilbert, Jennifer a Doudna, Jonathan S Weissman, Adam P Arkin, and Wendell a Lim. 2013. ‚ÄúRepurposing CRISPR as an RNA-Guided Platform for Sequence-Specific Control of Gene Expression.‚Äù Cell 152 (5). Elsevier: 1173‚Äì83. doi:10.1016/j.cell.2013.02.022.</P></ARTICLE></DIV></DIV></SECTION><FOOTER id="footer"><DIV class="inner"><DIV class="flex"><DIV class="copyright">¬© Warwick iGEM 2016.
                <P>We thankfully acknowledge generous funding support from our sponsors below.</P></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>