<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Warwick_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Warwick/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Theory by TEMPLATED</TITLE></P><DIV class="inner"><NAV id="nav"><A href="https://2016.igem.org/Team:Warwick">Home</A><A href="https://2016.igem.org/Team:Warwick/Team">Team</A><DIV class="dropdown"><A href="https://2016.igem.org/Team:Warwick/Description">Project</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Warwick/Description">Description</A><A href="https://2016.igem.org/Team:Warwick/Design">Design</A><A href="https://2016.igem.org/Team:Warwick/Parts">Parts</A><A href="https://2016.igem.org/Team:Warwick/Model">Model</A><A href="https://2016.igem.org/Team:Warwick/Software">Software</A></DIV></DIV><DIV class="dropdown"><A href="https://2016.igem.org/Team:Warwick/LabBook">Labs</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Warwick/LabBook">Lab Book</A><A href="https://2016.igem.org/Team:Warwick/Safety">Safety</A><A href="https://2016.igem.org/Team:Warwick/Interlab">Interlab</A><A href="https://2016.igem.org/Team:Warwick/Protocols">Protocols</A></DIV></DIV><DIV class="dropdown"><A href="https://2016.igem.org/Team:Warwick/Attributions">Attributions</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Warwick/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><A href="https://2016.igem.org/Team:Warwick/Human_Practices">Human Practices</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Warwick/Human_Practices">Summary</A><A href="https://2016.igem.org/Team:Warwick/HP/Silver">Silver</A><A href="https://2016.igem.org/Team:Warwick/HP/Gold">Gold</A><A href="https://2016.igem.org/Team:Warwick/Integrated_Practices">Integrated Practices</A><A href="https://2016.igem.org/Team:Warwick/Education">Education</A></DIV></DIV></NAV></DIV><SECTION id="banner"><H1>Our Project</H1><P>From Design to Product.</P></SECTION><SECTION id="one" class="wrapper"><DIV class="inner"><DIV class="flex flex-4"><ARTICLE><HEADER><H3>The Brainstorm</H3></HEADER><FOOTER><A href="#brainstorming" class="morebutton">More</A></FOOTER></ARTICLE><ARTICLE><HEADER><H3>Activation Using CRISPR/Cas9</H3></HEADER><FOOTER><A href="#lepto" class="morebutton">More</A></FOOTER></ARTICLE><ARTICLE><HEADER><H3>Detection Via RNA</H3></HEADER><FOOTER><A href="#detection" class="morebutton">More</A></FOOTER></ARTICLE><ARTICLE><HEADER><H3>The Next Step</H3></HEADER><FOOTER><A href="#nextstep" class="morebutton">More</A></FOOTER></ARTICLE></DIV></DIV></SECTION><DIV id="brainstorming"><SECTION id="three" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>Brainstorming</H2></HEADER><DIV class="flex flex-2"><ARTICLE class="nopicture"><HEADER><H3>The Idea</H3></HEADER><P>There is no global challenge worth greater investment than improving quality of life. During initial discussions, we identified a wide range of problems that we felt warranted further scientific development: increasing crop yields; sewage filtration; reclamation of rare earth metals. Upon evaluation, the team decided that synthesizing a modular biosensor detection kit would be most influential, due to the potential application to a wide range of diseases and environmental health issues. This affordable, accessible system could tackle multiple issues over a large demographic, making the venture truly worthwhile. Lyme disease, although not at the forefront of discussion, was the inceptive focus of the investigation, as current detection methods can only be applied after debilitating symptoms have manifested. However, upon reviewing data gathered from primary research and conferring with Lyme disease experts, we determined that diagnosis of another spirochetal disease would be more appropriate for our device. When surveying members of the public in Lyme disease hotspots, the results showed a distinct lack of knowledge of both symptoms and prevalence, but more importantly a distrust in self-testing. Furthermore, Dr Tim Brooks, Director of Public Health England Rare and Imported Pathogens Laboratory, identified a very significant issue with implementing our technology to diagnose Lyme disease directly from blood samples. Because of this, our previously secondary focus, leptospirosis, became the primary target for our diagnosis tool.   We are developing our device as a frontline diagnosis kit for leptospirosis infection in developing countries. The adaptable nature of the bio-detector could allow replacement of the expensive antibody-reliant detection systems currently used. We also aim to modify this technology such that it can be used to monitor toxic lead and mercury levels in water supplies.  Our detection system for infectious agents and environmental pollutants will be based on CRISPR/Cas9 technology, relying on conformational changes in an RNA-based sensor that will trigger transcriptional regulation by a dCas9 protein. We plan to produce a modular gene circuit that can detect the presence of either RNA from an infectious agent or metallic ion and output a fluorescent signal. The RNA sensor will be a modified sgRNA containing a motif binding either Borrelia/Leptospira RNA or lead/mercury aptamers. This binding will cause a conformational change such that the dCas9 enzyme may bind upstream of the transcriptional start site of a fluorescent reporter gene inducing transcription. Afterwards, the sensor will be freeze-dried onto a paper scaffold for ease of use.   We hope to create a paper based sensor that will be able to be used in a low-tech, out-of-lab, environment. We hope this will make it available to less economically developed countries as a frontline diagnostic tool and make a real impact in the fight against infectious diseases and pollution.</P></ARTICLE></DIV></DIV></SECTION></DIV><DIV id="lepto"><SECTION id="four" class="wrapper style1 special"><DIV class="inner"><HEADER class="align-center"><H2>Activation Using CRISPR/Cas9</H2></HEADER><DIV class="flex flex-2"><ARTICLE><HEADER><H3>dCas9 Binding</H3></HEADER><P>The single guide RNA (sgRNA) gene that forms part of plasmid 3 when expressed, forms a stand of RNA with a designed secondary structure. The dCas9 enzyme has a binding domain that targets the dCas9 handle on the sgRNA, resulting in a strong binding with high affinity. The dCas9 'reads' the 20nt targeting region at the 5' end of the sgRNA, guiding it to a location adjacent to a PAM that is complimentary to the targeting region.</P></ARTICLE><ARTICLE><HEADER><H3>RBP Effector Fusion</H3></HEADER><P>Our RNA binding protein (RBP) is fused to an effector protein that ordinarily upregulates the transcription of nearby genes with appropriate promotors. The effector domain is linked to the binding domain via a glycine-serine linker, a flexible length of amino acids that allows the effector to dangle freely when the RBP has bound to the appropriate stem loop structure on the sgRNA.</P></ARTICLE><ARTICLE><HEADER><H3>Targeting Activation</H3></HEADER><P>Once the fusion protein is expressed, it can bind to the complementary binding region on the sgRNA.  The CRISPR/Cas9 system will have guided the dCas9 to bind upstream of the transcription start site for the reporter gene, e.g. green fluorescent protein (GFP). The RBP-effector fusion recognises the stem loop structure present in the sgRNA and binds adjacent to the dCas9. This fusion protein acts as a RNA polymerase recruiter, as the fusion protein recruits RNA polymerase, it is positioned in an ideal location to initiate transcription. The reporter gene is expressed indicating gene activation.</P></ARTICLE><ARTICLE><HEADER><H3>Gene Expression</H3></HEADER><P>In this system, the sgRNA can be targeted to any site that contains a PAM sequence, and when placed upstream of a promoter, it will upregulate transcription of the gene downstream. </P></ARTICLE></DIV></DIV></SECTION></DIV><DIV id="detection"><SECTION id="five" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>Detection Via RNA</H2></HEADER><DIV class="flex flex-2"><ARTICLE><HEADER><H3>Detecting RNA</H3></HEADER><P>Through this system, it is possible with small changes to create a simple detection system that is capable of taking single inputs and activating multiple genes. By modifying the structure of the sgRNA so that the RBP handle is misfolded under normal conditions, the activation of the reporter gene becomes dependent on the refolding of this handle. The handle can be reformed upon the introduction of a specific single stranded RNA that binds, changing the conformation of the sgRNA. This &quot;sRNA&quot; can be tailored to almost any specification. By editing the RBP handle so that it reforms in the presence of sRNA from bacteria, it becomes possible to create a detection kit for bacterial infection. </P></ARTICLE><ARTICLE><HEADER><H3>Modular Sensor</H3></HEADER><P>For our current system, we are aiming to target Leptospirosis, aiding in the diagnosis of this disease. We are also looking into incorporating metal aptamers into the design such that our sensor can detect hazardous levels of heavy metals, creating a portable paper-based water monitor.</P></ARTICLE><ARTICLE><HEADER><H3>Metal Sensor</H3></HEADER><P>Incorporating a metal aptamer in the 5' end of the sgRNA in such a way that it prevents the binding of dCas9 to the sgRNA, allows this metal sensing to occur. When a metal ion binds to the aptamer, the aptamer changes conformation, exposing the previously bound RNA in the 20nt targeting site and dCas9 handle. This de-represses the dCas9 action, allowing the system to activate expression of the reporter gene.</P></ARTICLE></DIV></DIV></SECTION></DIV><DIV id="nextstep"><SECTION id="three" class="wrapper style1 special"><DIV class="inner"><HEADER class="align-center"><H2>The Next Step</H2></HEADER><DIV class="flex flex-2"><ARTICLE class="nopicture"><H3>The future of our biosensor.</H3><P>As our sensor is RNA based, any form of aptamer can be made compatible with the system. While the system is currently designed to detect mercury and lead, by modifying the aptamers it can become sensitive to any compound which an aptamer exists to bind to it. This modular system could potentially replace many other types of biosensor.  Furthermore, where the sensing regions in the current sgRNA design are sensitive to an RNA analogue, it is a proof of concept that the system activates in the presence of specific RNA. From this it is possible to design the biosensor to react to RNA from any bacteria or virus, creating a potentially universal infection diagnosis kit.  As it exists currently, our prototype is capable of detecting a single stimuli, be it foreign RNA or metal ion, and reacting by activating a single gene. By modifying our system it would be simple to have the detector react to multiple stimuli with individual responses for each. In this way it would be possible to have a single cell detect multiple stimuli, outputting different reporters for each stimuli, for example, a cell that fluoresces red in the presence of lead, and green in the presence of mercury. Or a diagnosis kit for multiple common infections.  This system could also selected to activate multiple genes in response to a single stimuli. By targeting and activating multiple genes simultaneously, complex cellular changes – such as cell fate –can be initiated by simple controllable stimuli. If this system were transposed to mammalian cells, multiple controllable gene expression could be the start of a renaissance in mammalian genetic engineering. The system is therefore useful not only as a modular biosensor, but also as a foundational advance in gene control. </P></ARTICLE></DIV></DIV></SECTION></DIV><SECTION id="three" class="wrapper special"><DIV class="inner"><HEADER class="align-center"><H2>Part Improvement</H2></HEADER><DIV class="flex flex-2"><ARTICLE class="nopicture"><P>Whilst planning our project, we decided that the use of three interacting plasmids would be the most optimal method of deploying our device. We decided to investigate the function of the bio brick standard plasmids further, in order to ascertain whether it was possible to clone three plasmids into the same competent cells. We found that only minimal information was available on the iGEM registry concerning this subject, as each plasmid had only been characterised individually, and few teams had ever attempted triple transformations as an integral part of their work. As a result, we decided to make our research findings available to the wider iGEM community, so that any teams in the future pursuing similar projects may have greater confidence in aspects of their designs.In order to test whether triple transformations using biobrick standard plasmids was possible, we  selected three biobrick plasmids differing significantly in copy number and antibiotic resistance. We elected to use pSB1C3, pSB3T5, and pSB4A5. In the registry, pSB1C3 is listed as having a high copy number of 200 copies, and pSB3T5 a medium copy number of around 20 copies. The last plasmid, pSB4A5, is listed as having a low copy number in the registry. However, from data gathered by the Warwick iGEM team in 2015, and further sequencing completed by the registry,  a mutation in the origin of replication was identified that caused the copy number to be significantly higher than the recorded value. pSB4A5 is a BioBrick standard vector with a pSC101 replication origin, whilst pSB3T5 has a p15A replication origin. As both of these origins rely on iterons as the negative copy number control, the competition between the plasmids may render them incompatible when in the same cell[1],  potentially limiting triple transformation efficiency.However, by successfully triple transforming the three plasmids, all containing the RFP gene J04450, we demonstrate that the plasmids can be used simultaneously in a single construct. To illustrate this we transferred the triple transformed cells onto plates containing chloramphenicol, tetracycline, and ampicillin, as shown in Figure 1 below.</P><P>Figure 1: RFP in pSB1C3, pSB3T5 and pSB4A5, having been transformed and platedFrom this plate, we inoculated a single colony into water and streaked this water onto one half of three different plates containing each of the antibiotics. On the other half of the plates, we streaked a colony of top10 cells that had been grown on a streptomycin plate. The results can be seen below. </P><P>Figure 2: Full colour image of the triple transformed and untransformed Top10 cells plated on three different antibiotics</P><P>Figure 3: A UV light image of the same plates, clearly showing the difference between the cultivated transformed and untransformed cells.References[1] http://openwetware.org/wiki/Escherichia_coli/Vectors, 19/10/16</P></ARTICLE></DIV></DIV></SECTION><FOOTER id="footer"><DIV class="inner"><DIV class="flex"><DIV class="copyright">© Warwick iGEM 2016.<P>We thankfully acknowledge generous funding support from our sponsors below.</P></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>