<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tianjin_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tianjin/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tianjin/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tianjin/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tianjin/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tianjin/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tianjin/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><A href="/wiki/index.php?title=Template:Https://2011.igem.org/Team:Peking_S/bannerhidden&amp;action=edit&amp;redlink=1" class="new" title="Template:Https://2011.igem.org/Team:Peking S/bannerhidden (page does not exist)">Template:Https://2011.igem.org/Team:Peking S/bannerhidden</A><A href="/wiki/index.php?title=Template:Https://2011.igem.org/Team:Peking_S/back2&amp;action=edit&amp;redlink=1" class="new" title="Template:Https://2011.igem.org/Team:Peking S/back2 (page does not exist)">Template:Https://2011.igem.org/Team:Peking S/back2</A><TITLE>Untitled</TITLE></P><DIV id="global"><DIV id="main"><DIV id="content"><P>Athough “Harry Potter and the Deathly Hallows” marks the termination of J. K. Rowling’s popular novels, we recreate a fantastic world where patronus are produced by Saccharomyces cerevisiae to fight against dementors inside the cell. Ethanol fermented by yeast from lignocellulosic materials can be an environmentally friendly fuel. However, rapid and efficient fermentation of lignocellulosic hydrolysates is limited because of inhibitors generated during pretreatment in addition to monomeric sugars. Inhibitors strongly affect the normal physiology of yeast as well as its ethanol productivity, just like the dementors taking away people’s hope and happiness. Nevertheless, we reconstruct TOR protein, a central component of major signaling transduction network controlling cell growth, to increase the tolerance of yeast. A new TOR after directional mutation will play the role of patronus to defend the influence of inhibitors, keep the overall signaling networks in good order, and finally provide a prosperous world for ethanol production.
</P><P>
This year we are aimed at increasing the tolerance of Saccharomyces cerevisiae to composite inhibitors in lignocellulosic hydrolysates, such as <B>furans, acetate and phenol (“FAP” for short)</B> formed during pretreatment and hydrolysis. Lignocellulosic materials such as wood provide abundant and renewable energy sources. Lignocellulosics contain sugars polymerised to cellulose and hemicelluloses which can be liberated by hydrolysing the material using <SPAN style="color:#7f7f00">industrial waste acid</SPAN>, and subsequently fermented to ethanol by microorganisms, such as yeast. Lignocellulose-derived ethanol can be used as an environmentally friendly liquid fuel. However, rapid and efficient fermentation of the hydrolysates is limited because a range of toxic compounds are generated during steam pretreatment and hydrolysis of lignocellulosics in addition to monomeric sugars.</P><P>
The inhibiting compounds are divided in three main groups based on origin: weak acids, furan derivatives, and phenolic compounds. Weak acids, especially acetate, which is widely known as a kind of food preservatives, could inhibit cell growth. The growth-inhibiting effect on microorganisms has been proposed to be due to the inflow of undissociated acid into the cytosol. Undissociated weak acids are liposoluble and can diffuse across the plasma membrane. In the cytosol, dissociation of the acid occurs due to the neutral intracellular pH, thus decreasing the cytosolic pH. With the obvious decrease of intracellular PH, acetate could cause severe amino acid starvation, repress the normal center carbon metabolism and thus damage the physiology of whole cell. Acetic acid has been shown to induce apoptosis in yeast, and TOR pathways (Tor1p) are involved in the signaling of acetic acid-induced apoptosis. Furan derivatives have been shown to reduce the specific growth rate, the cell-mass yield on ATP, the volumetric and specific ethanol productivities. They could give rise to production of oxidative stress, and consumption of enegy (as ATP) and reducing power (as NADPH, NADH). For example, HMF has been shown to cause accumulation of lipids and decrease the protein content in yeast cells. And furfural reduction to furfuryl alcohol by NADH dependent dehydrogenases has a higher priority than reduction of dihydroxyacetone phosphate to glycerol, and furfural causes inactivation of cell replication. Phenolic compounds partition into biological membranes may cause loss of integrity, thereby affecting their ability to serve as selective barriers and enzyme matrices. They have been suggested to exert a considerable inhibitory effect in the fermentation of lignocellulosic hydrolysates, the low molecular weight phenolic compounds being most toxic. However, the mechanism of the inhibiting effect has not been elucidated, largely due to a lack of accurate qualitative and quantitative analyses. What’s more, the three groups of inhibitors have been shown to interact synergistically or antagonistically when they coexist, making the inhibition mechanism more complicated.
With the treatment of composite inhibitors, the yeast will suffer from the raise in protein degradation, and ROS (Reactive Oxygen Species), unfolded protein response as well as excessive consumption of ATP, which may lead to the stasis of ethanol production and cell growth, and even autophagy.
</P><P>
Clarification of the inhibiting mechanism is indispensible for the manipulation and domestication of inhibitor-resistant yeast. However, traditional methods that only focus on the characterization of a few genes or proteins are too limited for elucidating the composite mechanism. Therefore, there is plainly a tendency that system biology, as well as synthetic biology in which varieties of “Omics” (proteomics, metabolomics, genomics, transcriptomics etc,.) is integrated, would become an advanced high-throughput technology to shed light on the key mechanism. We should regard the yeast cell as an entity and a system, and look into the overall comnination of the <SPAN style="color:#7f7f00">metabolite network</SPAN>, rather than several isolated genes and pathways. Following this discipline, we finally settled our target at TOR pathway.
</P><P>
The core of our project is the reconstruction of TOR (target of rapamycin) pathway, which is the core of yeast signaling transduction network. TOR protein is a central node of signaling network, and is involved in massive physiological processes, such as cell growth and nutrient uptake, etc. One of the most significant clues that relate TOR to the resistance of inhibitors is transcriptional profiling. Different “omics” analysis results revealed that expressions of TOR-relevant genes, RNAs and proteins differ greatly between tolerant strains to parental ones. The following heat map reveals that transcriptions of the Atg proteins in tolerant strains discriminate dramatically from parental ones in the presence of inhibitors.</P><P>
Atg proteins are the key players involved in autophagy. While TORC1 is a generally-known negative regulator of autophagy whose activity controls the phosphorylation status of Atg13. When TORC1 is active (inhibitors absent), Atg13 is hyperphosphorylated, whereas rapamycin addition induces a rapid dephosphorylation of Atg1. The latter apparently stimulates the affinity of Atg13 for Atg1 and promotes Atg1–Atg13 complex formation which is a requirement for autophagy. It is possible that PP2A (a downstream substrate of TORC1) is involved in TORC1-dependent regulation of Atg13 phosphorylation, since it was recently shown that autophagy is negatively regulated by the Tap42-PP2A pathway (a major way for TORC1 to regulate nutrients uptake and amino acids synthesis).
Another strong evidence which links TOR to the allergy to FAP is the comparison between growth curves of deletion strains and parental ones. In the following image, several deletion strains lacking the gene of some transcriptional factor, such as msn2/4, rtg1/2, gln3, gcn2, crf1 and so on.</P><P>
As shown above, the growth rate of original strain differs greatly from deletion strains, especially with the presence of FAP. However, some of the deletion effects are negative, and others are positive, indicating various physiological functions regulated by different transcriptional factors.
Seeing that TORC1-relevant regulators play crucial roles in inhibitors treatment, it should be useful to relieve or even switch the allergic symptoms of yeast through reconstructions of TOR pathway, especially modifications on the signaling transduction node “TORC1”, and regulations on corresponding downstream pathways.
</P><P>
With the summed up assumptions above, The core of our project is the directional modification of TOR protein(The target of rapamycin), a highly conserved Ser/Thr protein kinase which is the central component of a major signaling transduction network that controls cell growth in diverse eukaryotic organisms, ranging from yeast to human beings. Massive physiological processes, such as amino acid biosynthesis, transcription and translation machinery, carbohydrate metabolism, nucleotide biosynthesis, stress response, protein turnover and cell cycle are directly or indirectly linked with TOR pathway.</P><P>
In contrast to most eukaryotes, yeast contains two TOR homologues, Tor1 and Tor2. In Saccharomyces cerevisiae, TOR1 and TOR2 genes encode two large (*280 kDa) and homologous (67% identical) proteins that belong to a family of phosphatidylinositol kinase-related kinases (PIKKs). Despite their resemblance to lipid kinases, they are thought to function solely as Ser/Thr protein kinases. Based on the homologous proteins Tor1 and Tor2, two functionally and structurally distinct TOR multiprotein complexes exist: TOR complex 1 (TORC1) and TOR complex 2 (TORC2). Both Tor1 as well as Tor2 can be found in the multiprotein TORC1 together with Lst8, Kog1 and Tco89. A separate pool of Tor2 also associates with Lst8, Avo1, Avo2, Avo3, Bit61 and Bit2 to form TORC2. However, only TORC1 is specifically inhibited by rapamycin.</P><P>
The addition of rapamycin induces dramatic phenotypic changes such as cell cycle arrest and entry into G0, general downregulation of protein synthesis, accumulation of the reserve carbohydrate glycogen and the stress protectant trehalose, upregulation of stress response genes, autophagy and alterations in nitrogen and carbon metabolism, which became the research foundation of TOR pathway. It appears that TORC1 signalling controls the temporal aspects of cell growth in response to the quality of the available nitrogen and carbon sources. On the other hand, TORC2, which is insensitive to rapamycin and is less well characterized in comparison to TORC1, is thought to regulate the spatial aspects of growth, such as the control of actin polarization. Here, we will focus on TORC1, as only this complex modulates nutrient-induced signaling in response to starvations mainly of nitrogen sources and carbon sources triggered by FAP presence.</P><P>
The TOR protein consists of several distinct domains (illustrated in the image above). The N-terminal half of TOR comprises 20 tandemly HEAT repeats. HEAT repeats mediate protein-protein interactions and are required for localization of TOR to the plasma membrane. The HEAT repeats are accompanied by a so-called FAT domain that possibly serves as a scaffold or protein interaction domain. Following the FAT domain are the FKBP-rapamycin binding domain, the kinase catalytic domain, and a C-terminal FATC domain. Because TOR contains a number of domains that may mediate protein-protein interactions, TOR has been proposed to exist in a multiprotein complex and multi pathways. The most well-characterized TORC1 mediates the rapamycin-sensitive signaling branch that positively regulates anabolic processes such as translation and ribosome biogenesis and negatively regulates catabolic processes such as RNA degradation, autophagy, and other degradative pathways. 
The precise function of these TOR interacting proteins is not known yet. They might play a role in the binding of the TOR complexes to their substrates, be the receivers of upstream signals and/or determine the localization of the complexes. The TOR complexes are likely dimeric built on a TOR–TOR dimer. Only TORC1 can bind FKBP12-rapamycin while in TORC2, the Tor2 FKBP12-rapamycin binding domain is probably not exposed for binding, explaining why only TORC1 signaling is sensitive to rapamycin treatment. Interestingly, the constitution of TORC1 appears to be unaffected by rapamycin, implying that rapamycin does not inhibit TORC1 signaling by interfering with TORC1 stability. Both TOR complexes are essential for viability, since deletion of TOR2 (inactivation of TORC2) or deletion of both TOR1 and TOR2, or rapamycin treatment (inactivation of TORC1) are lethal to yeast. Deletion of TOR1 alone, however, is not lethal, indicating that Tor1 and Tor2 have a redundant role in TORC1 signaling. Several studies investigated the localization of TORC1 and TORC2. Various different localization patterns were observed, which is possibly a reflection of the fact that TOR signaling controls a multitude of processes. TORC1 is mainly found at the vacuolar membrane, which is intriguing knowing that the vacuole is a reservoir of nutrients and that TORC1 signaling is believed to be regulated by nutrients. According to a recent study, TORC1 is also targeted to the nucleus where it induces 35S rRNA synthesis under favorable growth conditions.
In our project, increase the resistance of Saccharomyces cerevisiae to FAP equals removing the allergy of TORC1 to intracellular FAP. That is to say, to ignore the inhibiting signals from the extracellular environment to maintain normal physiological activity and keep a pretty high growth rate and ethanol productivity so that therefore it won’t trigger severe reactions like autophagy and apoptosis. Since the target has been identified on Tor protein, we should elucidate the corresponding mechanism.
</P><P>
Former transcriptional profiling data showed that inhibitors, especially acetate, would cause a series of phenotypic and intracellular changes induced by TORC1, including the entry of some transcription factors to the nucleus, some cell cycle arrest and entry into G0, general downregulation of protein synthesis, accumulation of the reserve carbohydrate glycogen, upregulation of stress response genes, autophagy and alterations in nitrogen and carbon metabolism. All the phenomena are originally controlled by an TOR initiated phosphorylation signaling network as the following image shows.</P><P>
A major part of signaling transduction is regulated by the rapamycin-sensitive TORC1 complex either via the Tap42-Sit4/PPA2c or the recently identified Sch9 branches. In normal state (ample nutrients and no FAP), nutrients activate TORC1, resulting in the phosphorylation of Tap42-Sit4/PPA2 and Sch9, which would further transfer their phosphate groups to downstream transcription factors like Rtg1/3, Gln3, Maf1, Sfp1, Crf1, Msn2/4 and Rim15.</P><P>
The precise mechanism how TORC1 regulates its downstream effectors are often not well-understood. As will be discussed in the section below, several processes appear to be regulated via the PP2A and the PP2A-related protein phosphatases. These phosphatases consist of heteromeric protein complexes. The PP2A holoenzyme contains one of the two redundant catalytic subunits (PP2Ac), Pph21, Pph22, a scaffolding subunit, Tpd3, and one of the two regulatory subunits, Cdc55 or Rts1. The PP2A-related phosphatase is mainly found as a complex between the catalytic subunit, Sit4, and one of the four regulatory subunits, Sap4, Sap155, Sap185 and Sap190. TORC1 controls the activity of these phosphatases via Tap42. When Tap42 is phosphorylated by TORC1, it will compete for binding the catalytic subunits of the phosphatases leading to the exclusion of other subunits of the phosphatase holoenzymes. Thereby, TORC1 stimulates the formation of a Tap42-associated phosphatase complex that further includes either one of the regulatory proteins Rrd1 or Rrd2, both of which are known to confer phosphotyrosyl phosphatase activity to the catalytic phosphatase subunits in vitro. Tap42 as well as Rrd1 and Rrd2 may redirect the substrate specificity of the catalytic phosphatase subunits, and as such, it is not surprising that the proteins have been attributed both inhibitory as well as activating roles, dependent on the substrate being studied. In actively growing cells, the Tap42-associated phosphatase complexes reside mainly at membranes where they associate with TORC1. Rapamycin treatment or nutrient starvation abrogates the TORC1 association and releases the Tap42-associated phosphatase complex into the cytosol. Once cytoplasmic, this complex then slowly dissociates, presumably concomitant with the dephosphorylation of Tap42. Several studies revealed an important role for yet another player in TORC1-dependent regulation of PP2Ac and Sit4, i.e. Tip41. This protein was initially identified as an inhibitor that could specifically interact with dephosphorylated Tap42. However, more recent data suggest that both Tip41 and Tap42 cooperate in determining the substrate specificity of PP2Ac and Sit4, and that both proteins may fulfill essentially a similar function in TORC1 signaling.</P><P>
The complicated network has been characterized in the above image, which is also the foundation where we build our modeling section. FAP takes the place of Rapamycin (Rapamycin-Fpr1) to render the inhibiting effects. However, our model has been simplified to just several major regulatory proteins to avoid some immeasurable proteins interacting parameters and obscure mechanism.
Another downstream substrate of TORC1 is AGC kinase (homologous to protein kinases A, G, and C) Sch9. Six amino acids in the C terminus of Sch9 are directly phosphorylated by TORC1. Phosphorylation of these residues is lost upon rapamycin treatment as well as carbon or nitrogen starvation and transiently reduced following application of osmotic, oxidative, or thermal stress.
</P><P>
These observations support the notion that TORC1 activity is regulated by nutrient abundance and inhibited by noxious stress. TORC1-dependent phosphorylation is required for Sch9 activity, and replacement of residues phosphorylated by TORC1 with Asp/Glu (turn to continuous phosphorylation) renders Sch9 activity TORC1 independent. Sch9 is required for TORC1 to properly regulate ribosome biogenesis, translation initiation, and entry into G0 phase, but not expression of Gln3-dependent genes. Using TORC1-independent versions of Sch9, Scientists found that Sch9 is a major effector of TORC1 that appears to function similarly to the mTORC1 substrate S6K1. In Saccharomyces cerevisiae, Sch9 is directly phosphorylated by TORC1 at multiple C-terminal sites and by the yeast PDK1 orthologs in the activation loop. Both phosphorylation events are independently required for Sch9 activity. As Sch9 is contemporarily controlled by both TORC1 and Pkh1/2, and shares rather overlapping functions with cAMP-PKA pathway, this branch is not our emphasis presently. Nevertheless, for the central role Sch9 plays in nutrient-mediated signaling, this pathway deserves further interests and deeper exploration.
Briefly speaking, when TORC1 is active, it could keep the downstream transcription factors phosphorylated , thus they be excluded outside the nucleus and not accessible for the initiation of stress response genes, autophagy and several pathways that allow growth on poor nitrogen sources. However, when it comes to adverse environment with FAP or poor nutrients, transcription factors would be dephosphorylated because of the deactivation of TORC1 and enter the nucleus, causing the repression of protein synthesis and the stimulation of amino acid biosynthesis, stress response genes, autophagy and nitrogen discrimination pathway.
</P><P>
As TOR pathway is the core controller of intracellular phosphorylation, mutations or genetic engineering on TORC1, in other words TOR1 or TOR2, would be the key to increase the tolerance to FAP and solve our problem. However, even though TOR pathway is proved to be a central element of signaling transduction, research on TORC1 of yeast are not so clear and well-understood for us to directly learn from. Fortunately, noticeable work has been done on mTORC1 (mammalian TOR complex 1) that modulation in certain amino acid sites could improve the activity in response to adverse nutrient status and inhibitors. The high homogeneity between TOR in yeast and mammalian TOR (43% of TOR2 and 39% of TOR1) makes it possible to transplant the mutations of mTOR into the yeast. Mammalian target of rapamycin (mTOR) is a key regulator of eukaryotic cell growth. In particular, mTORC1, one of the two complexes that contain mTOR, is involved in the regulation of protein synthesis, proliferation, cell cycle and autophagy. Hyperactivation of the mTOR signaling pathway is observed in human cancer. A variety of approaches including deletion analysis, yeast genetic screens and mining of human cancer genome databases were taken that resulted in the identification of activating mutations of TOR. These studies suggest that the FAT, FRB and kinase domains are the three regions of TOR where activating mutations can be identified. </P><P>
Experiments also showed that TOR2-mTOR hybrid protein in which the carbon terminal of TOR2 was replaced with corresponding part of mTOR was stably expressed in yeast, localized to the vacuolar surface, and associated with a phosphatidylinositol-4 kinase activity. What’s more, TOR2-mTOR and TOR1-mTOR hybrid proteins mutated at the position corresponding to rapamycin-resistant TOR mutants (S2035I) conferred rapamycin resistance. These findings directly indicate that the kinase domain in TOR and mTOR have been functionally conserved from yeast to man. The comparison of yeast TOR and mTOR domains also shows high similarity in the carbon terminal where most activating sites concerning the resistance of inhibitors is located.</P><P>
The overlap in functions and activating domains render us to start with hyperactivating sites on mTOR appeared in Human cancer and drug resistance. In terms of several literatures, mutation sites can be sorted into three groups. Firstly, L1460P mutation in FAT domain and E2419K mutation in kinase domain would render mTOR independent from upstream signals and keep active while intracellular amino acids starvation; secondly, mutations on V2198A, L2216H and L2260P could obtain higher kinase activity by binding with Lst8; Thirdly, mutations on S2215Y and R2505P exists in cancer cells would keep the downstream substrates in continuous phosphorylation and maintain strong constitutive activity. But not all of these sites are suitable for yeast TOR despite their similar structures. In accordance to comparison of mutation sites between TOR2 (1st row), TOR1 (2nd row), mTOR (3rd row), only the following four sites share the same original amino acids.</P><P>
Besides, the hyperactive effect in cancer cells may bring about unexpected consequences in yeast in addition to FAP resistance. In the end, we picked out four point mutations of amino acids sites with sequence replacement and functions listed in the following form. As yeast TOR2 shares more similarity with mTOR and deeper involved in cell growth, we decided to use tor2 gene rather than tor1 as target of the whole project.</P><P>
With the confirmation of four mutation sites, a new question came out. How could we manipulate on such a large fragment of 7800bp?</P><P>
The large size of eukaryotes coding gene increase the difficulty of getting the original gene from yeast genome; condon optimization and chemical synthesis also seemed nearly impossible. While the design of mutation methods and primers became rather important and challenging, since we had to avoid massive common restriction sites to guarantee the insertion of tor gene into proper vectors. After rounds of discussions and brain-storming, two solutions are determined. We divided our experimenters into two groups and the two methods got started in parallel.<B>3.2.1  Circular PCR (emphasis)</B>
Our inspiration came from <B><I>Fast Mutagenesis System Kit</I></B> from <B>Transgen Biotech</B>. Different from traditional mutagenesis on linear fragment, this mutagenesis system is conducted in circular plasmid after our target gene has been inserted into the vector. Using high fidelity polymerase EasyPfu and FastPfu, we got the whole 7.8kb fragment of tor2 and inserted it into our modified plasmid backbone <B>pSB1A11</B> for mutation.
Its principle is quite simple but classical: Two primers anneal with template chains and synthesize mutagenesis chain with FastPfu Polymerase. The mutation base pair locate in the overlapping region of two primers. And then <B>DMT enzyme</B> will digest the unmutated template chain with methylation. The digested product will be transformed into <B>DMT Competent Cell</B>, with following steps of mutagenesis seletion, which is similar to traditional cloning and mutagenesis. Sequencing is necessary for final examine of the mutagenesis.</P><P>
As one of the mutation sites L-P locates in the middle 1/3 part of TOR2, and the other three (I-T V-A E-K) locate in the C-terminal 1/3 part, we started with introduction of single mutagenesis in one plasmid, and combine these mutagenesis respectively to get two, three and all sites mutation, meanwhile compare the resistance and activity brought in by our modification.<B>3.2.2  Overlap PCR (postponed)</B>
Overlap PCR is one of the most classical methods for ligation of fragment and site mutagenesis. Considering the size of tor2, we had to divide the whole fragment into three – 11tor, 22tor and 33tor. The part of 01tor contains the promoter and RBS region in addition to 11tor. Restriction sites are reserved in the overlapping region for later ligation and assembly of an integrated gene.</P><P>
Due to the different sequences, mutation strategy varied from site to site. For L-P and E-K, we designed primers with 40bp overlap including the mutation sites. But for I-T and E-K, we introduced restriction enzyme site and mutation site in the 5’ end of one primer and then connect this piece back to the whole gene. We have already got all single mutagenesis on each part so far. But the discriminating GC content strongly hindered effective primer design for PCR, and the time spent on connecting and assemble different parts is far out of our expectation. Presently we have postponed the overlapping way. While all BioBricks based on pEASY vector are reserved and we hope teams with developed overlapping technology could continue this unfinished work.
</P><P>
With the mutated gene fragment built in E. coli, the subsequent step is transformation into the yeast. However, pSB1A11 is derived from PUC19 that cannot be normally expressed in yeast. Thus we turned to pYD1 for help. In Tianjin University 2010, pYD1 is used for surface display. But this year we need the yeast elements instead.</P><P>
In the very beginning, we intended to use circular PCR to modify pYD1, keep pGAL1, PUC Origin, ampicillin resistance gene, CEN6/ARS4, TRP1 orf and terminator regions, and use specific primers to add the restriction sites for tor2 gene insertion. With time went by, we found that the circular PCR is not so accurate and stable when come across large fragment. Therefore we changed the strategy that only CEN4/ARS6, TRP1 orf and terminator regions in pYD1 are amplified and inserted into pSB1A11 by a single restriction sites. On the other hand, tor2 gene is already inserted before. The promoter and ampicillin resistance gene already exist in the vector, and consequently an E. coli vector is successfully transformed to a recombinant yeast vector named <B>pYD11</B>. What’s more, with CEN element, pYD11 could convey target gene into the yeast nucleus, functioning like another yeast chrosome. Our engineered tor2 gene was eventually electrotransformed into Saccharomyces cerevisiae BY4742 strain.
</P><P>With the planning mutagenesis, our experiment still needs to be continued. Now that we intend to release the allergic symptom and increase the tolerance of Saccharomyces cerevisiae to FAP, the module should work under a detecting and self-controlled mechanism. That is to say, trigger the Mut-tor2 when FAP has disrupted the normal intracellular physiology and phosphrylation, and terminate the process when environment is recovered. Our goal is to reach a steady state in a relatively short period that the yeast cell growth and ethanol productivity is not inhibited by FAP. In fact, we may not eliminate the inhibition thoroughly with the feedback system, but at least the allergy is released and tolerance is improved to a certain degree.</P>
To reach the goal, a detecting and delivering part is essential. We picked up four inducible promoters regulated by transcription factors, whose entry into nucleus are stimulated by FAP generating dephosphorylation. Insert these promoters into the vector before Mut-tor2 gene and reporter gene of EGFP. The production of Mut-TOR2 would form Mut-TORC1 resistant to FAP. The rising quantities of active mTORC1 could help recover the cell growth and intracellular phosphorylation, leading to more RTG1/3 being excluded out of the nucleus, and expression of Mut-TOR would decrease. Eventually the yeast cell will grow into a balance and EGFP strength have the following tendency.<P>
After the reconstruction of TOR pathway, data and results are necessary for testing the system. Two methods are applied here. One is qualitative: detecting the location transfer of transcription factors. The other is quantitative: measure the growth curve of recombinant strains and compare with parental ones.
</P><P>
With the inducible promoter and fussion protein of Rtg1/3, we could observe the fluorescence location and strength. Details will be shown in presentation and poster.</P><P>
Yeast growing rate, ethanol yield and productivity data will be shown in presentation and poster.
</P><P>
imited by the competition deadline, we could just concentrate on the reconstruction of TOR pathway. Yet we still have various project brunches to increase the tolerance of yeast to in other perspectives. For example, we can bring in detoxification pathways to speed up the rate of FAP degradation. Up-regulation of ADH6 would obviously remove furfurals in cytoplasm; exogenous genes such as laccase and phenol hydroxylase would transform phenols into less poisonous chmicals; getting through the pathway from acetate to ethanol will be helpful to release acetate accumulation; PH-sensitive promoter and modifications on acetate efflux pumps proteins will keep it from heavily inflowing. There are endless areas to explore only if you have passion, perseverance and patience. Actually, the parts and modules we built presently still needs improving, combinations and testing, to find out the best group.
Finally, all the modules will be integrated into a pYD11 (or other vectors having functions like Yeast Artificial Chromosome to express gene in nucleus). We hope to make it become the 17th chromosome of the yeast (normally 16). With this additional chromosome, the yeast will gain stable growth, high yield and productivity of ethanol and an extension in life span regardless of composite inhibitors in lignocellulosic hydrolysates. Furthermore, we also want to apply this integrated pYD11 with different functions and into others kinds of organism. That is to say, with this universal and standard module, we can put it into any organism needs to be remodeled, just like an USB, to be a universe vector.<B>Summary:</B>
Decades of years have passed since the concept of &quot;Synthetic Biology&quot; being introduced, it is now essential to view cells as true ‘programmable’ entities, and develop effective strategies for assembling devices and modules into intricate, customizable larger scale systems rather than just creation and perfection of genetic devices and small modules. Meanwhile, regulations on gene expression have evolved from transcriptional level, to translational level, and finally signaling transduction level. Based on the foundation of functionalized modules, we finally extend synthetic biology to industrialized application.
</P><P><SPAN style="color:#7f7f00"><B>[1]</B></SPAN>	I. Dilova, E. Easlon, S.-J. Lin. <B>Calorie restriction and the nutrient sensing signaling pathways.</B> Cell. Mol. Life Sci. 2007, <B>64:</B> 752 - 767.<SPAN style="color:#7f7f00"><B>[2]</B></SPAN>	H. Lempiäinen, A. Uotila, Jo. Urban <I>et al</I>. <B>Sfp1 interaction with TORC1 and Mrs6 reveals feedback regulation on TOR signaling.</B> Molecular Cell. 2009, <B>33:</B> 704 - 716.<SPAN style="color:#7f7f00"><B>[3]</B></SPAN>	Y.-X. Wang, N. L. Catlett, L. S. Weisman. <B>Vac8p, a vacuolar protein with armadillo repeats, functions in both vacuole inheritance and protein targeting from the cytoplasm to vacuole.</B> The Journal of Cell Biology. 1998, <B>140(5):</B> 1064 - 1074.<SPAN style="color:#7f7f00"><B>[4]</B></SPAN>	J. Urban,1 A. Soulard, A. Huber <I>et al</I>. <B>Sch9 is a major target of TORC1 in <I>Saccharomyces cerevisiae</I>.</B> Molecular Cell 2007, <B>26:</B> 663 - 674.<SPAN style="color:#7f7f00"><B>[5]</B></SPAN>	J. R. Rohde1, R. Bastidas, R. Puria <I>et al</I>. <B>Nutritional control via TOR signaling in <I>Saccharomyces cerevisiae</I>.</B> Current Opinion in Microbiology 2008, <B>11:</B> 153 - 160.<SPAN style="color:#7f7f00"><B>[6]</B></SPAN>	M. Wei, P. Fabrizio, F. Madia <I>et al</I>. <B>Tor1/Sch9-regulated carbon source substitution is as effective as calorie restriction in life span extension.</B> PLoS Genetics 2009, <B>5(5):</B> 1 - 15.<SPAN style="color:#7f7f00"><B>[7]</B></SPAN>	C. M. Alarcon, M. E. Cardenas, J. Heitman. <B>Mammalian RAFT1 kinase domain provides rapamycin-sensitive TOR function in yeast.</B> Genes Dev. 1996, <B>10:</B> 279 - 288.<SPAN style="color:#7f7f00"><B>[8]</B></SPAN>	I. Georis, J. J. Tate, A. Feller. <B>Intranuclear function for protein phosphatase 2A: Pph21 and Pph22 are required for rapamycin-induced GATA factor binding to the DAL5 promoter in Yeast.</B> Molecular and Cellular Biology, 2011, <B>31(1):</B> 92 - 104.<SPAN style="color:#7f7f00"><B>[9]</B></SPAN>	S. Wullschleger, R. Loewith, W. Oppliger <I>et al</I>. <B>Molecular organization of target of Rapamycin Complex 2.</B> The Journal of Biological Chemistry, 2005, <B>280(35):</B> 30697 - 30704.<SPAN style="color:#7f7f00"><B>[10]</B></SPAN>	E. Jacinto. <B>What Controls TOR?</B> Life, 2008, <B>60(8):</B> 483 - 496.<SPAN style="color:#7f7f00"><B>[11]</B></SPAN>	A. Adami, B. García-Álvarez, E. Arias-Palomo <I>et al</I>. <B>Structure of TOR and its complex with KOG1.</B> Molecular Cell 2007, <B>27:</B> 509 - 516.<SPAN style="color:#7f7f00"><B>[12]</B></SPAN>	L. Kuepfer, M. Peter, U. Sauer <I>et al</I>. <B>Ensemble modeling for analysis of cell signaling dynamics.</B> Nature Biotechnology 2007, <B>25(9):</B> 1001 - 1006.<SPAN style="color:#7f7f00"><B>[13]</B></SPAN>	B. Smets, R. Ghillebert, P. D. Snijder <I>et al</I>. <B>Life in the midst of scarcity: adaptations to nutrient availability in <I>Saccharomyces cerevisiae</I>.</B> Curr. Genet. 2010, <B>56:</B> 1 - 32.<SPAN style="color:#7f7f00"><B>[14]</B></SPAN>	M. A. Romanos, C. A. Scorer, J. J. Clare. <B>Foreign gene expression in Yeast: a review.</B> Yeast 1992, <B>8:</B> 423 - 488.<SPAN style="color:#7f7f00"><B>[15]</B></SPAN>	A. G. Hinnebusch, K. Natarajan. <B>Gcn4p, a master regulator of gene expression, is controlled at multiple levels by diverse signals of starvation and stress.</B> Eukaryotic Cell 2002, <B>1(1):</B> 22 - 32.<SPAN style="color:#7f7f00"><B>[16]</B></SPAN>	A. G. Hinnebusch. <B>Translational regulation of Yeast GCN4.</B> The Journal of Biological Chemistry 1997, <B>272(35):</B> 21661 - 21664.<SPAN style="color:#7f7f00"><B>[17]</B></SPAN>	B. Scherens, A. Feller, F. Vierendeels <I>et al</I>. <B>Identification of direct and indirect targets of the Gln3 and Gat1activators by transcriptional profiling in response to nitrogen availability in the short and long term.</B> FEMS Yeast Res 2006, <B>6:</B> 777 - 791.<SPAN style="color:#7f7f00"><B>[18]</B></SPAN>	J. L. Crespo, T. Powers, B. Fowler <I>et al</I>. <B>The TOR-controlled transcription activators GLN3, RTG1, and RTG3 are regulated in response to intracellular levels of glutamine.</B> PNAS 2002, <B>99(10):</B> 6784 - 6789.<SPAN style="color:#7f7f00"><B>[19]</B></SPAN>	S. M. Kingsman, D. Cousens, C. A. Stanway <I>et al</I>. <B>High-efficiency Yeast expression vectors based on the promoter of the phosphoglycerate kinase gene.</B> Methods in Enzymology 1990, <B>185(27):</B> 329.<SPAN style="color:#7f7f00"><B>[20]</B></SPAN>	M. Brunner, H. Bujard. <B>Promoter recognition and promoter strength in the Escherichia coli system.</B> The EMBO Journal 1987, <B>6(10):</B> 3139 - 3144.<SPAN style="color:#7f7f00"><B>[21]</B></SPAN>	T. Kodadek, D. Sikder, K. Nalley. <B>Keeping transcriptional activators under control.</B> Cell 2006, <B>127:</B> 261 - 264.<SPAN style="color:#7f7f00"><B>[22]</B></SPAN>	K. Natarajan, M. R. Meyer, B. M. Jackson <I>et al</I>. <B>Transcriptional profiling shows that Gcn4p is a master regulator of gene expression during amino acid starvation in Yeast.</B> Molecular and Cellular Biology 2001, <B>21(13):</B> 4347 - 4368.<SPAN style="color:#7f7f00"><B>[23]</B></SPAN>	A. G. Hinnebusch. <B>Translational regulation of GCN4 and the general amino acid control of Yeast.</B> Annual Review of Microbiology 2005, <B>59:</B> 407 - 450.<SPAN style="color:#7f7f00"><B>[24]</B></SPAN>	E. Nevoigt, J. Kohnke, C. R. Fischer <I>et al</I>. <B>Engineering of promoter replacement cassettes for fine-tuning of gene expression in <I>Saccharomyces cerevisiae</I>.</B> Applied and Encironmental Microbiology 2006, <B>72(8):</B> 5266 - 5273.<SPAN style="color:#7f7f00"><B>[25]</B></SPAN>	K.EJ Tyo, K. Kocharin, J. Nielsen. <B>Toward design-based engineering of industrial microbes.</B> Current Opinion in Microbiology 2010, <B>13:</B> 255–262.<SPAN style="color:#7f7f00"><B>[26]</B></SPAN>	Y. Ohne, T. Takahara, R. Hatakeyama <I>et al</I>. <B>Isolation of hyperactive mutants of mammalian target of rapamycin.</B> The Journal of Biological Chemistry 2008, <B>283(46):</B> 31861 - 31870.<SPAN style="color:#7f7f00"><B>[27]</B></SPAN>	J. Urano, T. Sato, T. Matsuo <I>et al</I>. <B>Point mutations in TOR confer Rheb-independent growth in fission yeast and nutrient-independent mammalian TOR signaling in mammalian cells.</B> PNAS 2007, <B>104(9):</B> 3514 - 3519.<SPAN style="color:#7f7f00"><B>[28]</B></SPAN>	T. W. Sturgill, M. N. Hall. <B>Activating mutations in TOR are in similar structures as oncogenic mutations in PI3KCα.</B> ACS Chemical Biology 2009, <B>4(12):</B> 999 - 1015.<SPAN style="color:#7f7f00"><B>[29]</B></SPAN>	M. Hardt, N. Chantaravisoot, F. Tamanoi. <B>Activating mutations of TOR (target of rapamycin).</B> Genes to Cells 2011, <B>16:</B> 141–151.<SPAN style="color:#7f7f00"><B>[30]</B></SPAN>	E. Palmqvist, B. Hahn-Hägerdal. <B>Fermentation of lignocellulosic hydrolysates. I: inhibition and detoxification.</B> Bioresource Technology 2000, <B>74:</B> 17 - 24.<SPAN style="color:#7f7f00"><B>[31]</B></SPAN>	E. Palmqvist, B. Hahn-Hägerdal. <B>Fermentation of lignocellulosic hydrolysates. II: inhibition and detoxification.</B> Bioresource Technology 2000, <B>74:</B> 25 - 33.<SPAN style="color:#7f7f00"><B>[32]</B></SPAN>	S. I. Mussatto, I. C. Roberto. <B>Alternatives for detoxification of diluted-acid lignocellulosic hydrolyzates for use in fermentative processes: a review.</B> Bioresource Technology 2004, <B>93:</B> 1 - 10.<SPAN style="color:#7f7f00"><B>[33]</B></SPAN>	Z. L. Liu. <B>Molecular mechanisms of yeast tolerance and in situ detoxification of lignocellulose hydrolysates.</B> Appl. Microbiol. Biotechnol. 2011, <B>90:</B> 809 - 825.<SPAN style="color:#7f7f00"><B>[34]</B></SPAN>	A. Breitkreutz, H.Choi, J. R. Sharom <I>et al</I>. <B>A global protein kinase and phosphatase interaction network in Yeast.</B> Science 2010, <B>328:</B> 1043 - 1046.<SPAN style="color:#7f7f00"><B>[35]</B></SPAN>	Z. D. Sharp. <B>Aging and TOR: interwoven in the fabric of life.</B> Cell. Mol. Life Sci. 2011, <B>68:</B> 587 - 597.<SPAN style="color:#7f7f00"><B>[36]</B></SPAN>	D. Carmona-Gutierrez, T. Eisenberg, S Büttner <I>et al</I>. <B>Apoptosis in yeast: triggers, pathways, subroutines.</B> Cell Death and Differentiation 2010, <B>17:</B> 763 - 773.<SPAN style="color:#7f7f00"><B>[37]</B></SPAN>	P. Fabrizio, V. D. Longo. <B>Chronological aging-induced apoptosis in yeast.</B> Biochem. Biophys. Acta. 2008, <B>1783(7):</B> 1280 - 1285.<SPAN style="color:#7f7f00"><B>[38]</B></SPAN>	D. S. Evansa, P. Kapahic, W. - C. Hsueha <I>et al</I>. <B>TOR signaling never gets old: aging, longevity and TORC1 activity.</B> Ageing Research Reviews 2011, <B>10:</B> 225 - 237.<SPAN style="color:#7f7f00"><B>[39]</B></SPAN>	B. Almeida, S. Ohlmeier, A. J. Almeida <I>et al</I>. <B>Yeast protein expression profile during acetic acid-induced apoptosis indicates causal involvement of the TOR pathway.</B> Proteomics 2009, <B>9:</B> 720 - 732.<SPAN style="color:#7f7f00"><B>[40]</B></SPAN>	C. J. Bashor, N. C. Helman, S. Yan <I>et al</I>. <B>Using engineered scaffold interactions to reshape MAP kinase pathway signaling dynamics.</B> Science 2008, <B>319:</B> 1539 - 1543.<SPAN style="color:#7f7f00"><B>[41]</B></SPAN>	E. Nevoigt. <B>Progress in metabolic engineering of <I>Saccharomyces cerevisiae</I>.</B> Microbiology and Molecular Biology Reviews 2008, <B>72(3):</B> 379 - 412.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Tianjin/Project">http://2011.igem.org/Team:Tianjin/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tianjin/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tianjin/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tianjin/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tianjin/Project&amp;oldid=234034" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>