<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="wrapper"><NAV class="navbar navbar-inverse navbar-fixed-top" id="sidebar-wrapper" role="navigation"><UL class="nav sidebar-nav"><LI class="sidebar-brand"><A href="#">
                       Project
                    </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Project"> Back</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Parts </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Improve">Improve</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Human Practice </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/HP/Gold_Integrated ">integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Collaborations">Collaboration</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/HP/Silver">Silver</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Team </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Team" target="_blank">Team</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Attributions">Attribution</A></LI></NAV></DIV><P>



/* OVERRIDE IGEM SETTINGS */
</P><DIV id="title"><H1>Demonstrate</H1></DIV><DIV id="normal"><SECTION class="nav"><DIV class="langtab"><DIV class="collapse-card"><DIV class="collapse-card__heading"><H3 class="collapse-card__title">
				Mating-type Switch and Mating Switcher 
			</H3></DIV><DIV class="collapse-card__body"><H4>Obtaining the chassis </H4><P>Aiming to achieve MTS for environmental use, it is essential to make sure that when the <I>MAT</I> locus has DSB (double strands break) cleaved by <I>HO</I>, our type-a (<I>MATa</I>) yeast can only become type-α (<I>MATα</I>). Therefore, we used a <I>Ura-tag</I> to replace the<I> HMRa</I> domain in <I>chromosome Ⅲ</I>. In this way the <I>HMRa</I> will no longer be the donor for the homologous recombination in the repairing process for MAT cleavage. After selection, by homologous recombination, we deleted the <I>Ura-tag</I> for further usage. We selected the target colonies (<B><I>SynⅩ-dUra</I></B>) via <I>5-FOA</I> plates. </P><DIV class="zxx_zoom_demo"><DIV class="small_pic_demo" style="float:left;"><P style="font-size:15px;text-align:center"> Fig. 1-1. The PCR strategy for testing whether we deleted the HMRa in <B><I>SynⅩ-dUra</I></B>. 
</P></DIV><DIV class="small_pic_demo" style="float:right;"><P style="font-size:15px;text-align:center">Fig. 1-2. As we can see in the gel photo above, the <B>UP</B> and <B>DOWN</B>  segments hasn’t been amplified in our <I><B>SynⅩ-dUra</B></I> comparing to the BY4741 as control, which indicated that the HMRa gene has been successfully eliminated. 
</P></DIV></DIV><DIV id="pic_ten" style="display:none;"><P style="font-size:15px;text-align:center"> Fig. 1-1. The PCR strategy for testing whether we deleted the HMRa in <B><I>SynⅩ-dUra</I></B>.      
</P></DIV><DIV id="pic_eleven" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 1-2. As we can see in the gel photo above, the <B>UP</B> and <B>DOWN</B>  segments hasn’t been amplified in our <I><B>SynⅩ-dUra</B></I> comparing to the BY4741 as control, which indicated that the HMRa gene has been successfully eliminated. 
</P></DIV><H4>	The result for constructing the <I>Gal</I> systems</H4><P>In this pathway, we chose <I>Gal1</I> as our inducible promoter for the expression of <I>HO</I> gene, <I>CYC1</I> as the terminator, and <I>PRS416</I>(with <I>Ura-tag</I>) as our vector. As for segments ligation, we designed the cutting sites for <I>Bsa1</I> enzyme in each part, hoping to achieve seamless ligation of these three parts.</P><P>We adopted the PCR method to amplify the 
<B><I>Gal1-part</I><A href="http://parts.igem.org/Part:BBa_K2407001"> (BBa_K2407001) </A></B>
 and 
 <B><I>CYC1-part</I><A href="http://parts.igem.org/Part:BBa_K2407107"> (BBa_K2407107) </A></B>
  from a <I>Gal1-Vika</I> plasmid we had used in our former lab work with specially designed primers for this procedure. After PCR, the <I>Gal1</I> has the cutting sites for <I>SalⅠ</I>and <I>BsaⅠ</I>on both ends, and <I>CYC1</I> has that for <I>BsaⅠ</I>and <I>BamhⅠ</I>on both ends. Meanwhile, the 
  <B><I>HO</I><A href="http://parts.igem.org/Part:BBa_K2407109"> (BBa_K2407109) </A></B>
   gene was obtained by gene synthesis, flanked by specific hangtags for <I>BsaⅠ</I>in order to be cohesive with <I>Gal1</I> (upstream) and <I>CYC1</I> (downstream). Thus, we have built our composite part (<I>GHC</I>).</P><P>After the ligation of<B><I>GHC</I><A href="http://parts.igem.org/Part:BBa_K2407100"> (BBa_K2407100) </A></B> and <I>PRS416</I> Plasmid (<B><I>GHC-416</I></B>), we transformed the E. coli for the augment of our new plasmid——<B><I>GHC-416</I></B>. We examined the transformation result by PCR method to amplify the <I>HO</I> gene in the E. coli which we randomly selected in the plate.   
</P><DIV class="zxx_zoom_demo_qqq" align="center"><DIV class="small_pic_demo_qqq" align="center"><P style="font-size:15px;text-align:center">Fig. 1-3. The results of PCR of #6, #7, #16, #20, #27, #36, #37, #55 colonies. <I>HO</I> gene (length of 1770bp). As we can see, <I>HO</I> gene in all 8 colonies has been amplified, which indicated that we succeeded in constructing the device for <I>HO</I> gene expression.</P></DIV></DIV><DIV id="pic_twentyone" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 1-3. The results of PCR of #6, #7, #16, #20, #27, #36, #37, #55 colonies. <I>HO</I> gene (length of 1770bp). As we can see, <I>HO</I> gene in all 8 colonies has been amplified, which indicated that we succeeded in constructing the device for <I>HO</I> gene expression.</P></DIV><H4>The result of mating type switching(<B>MTS</B>)</H4><P>We transformed the chassis yeasts for the new device——<B><I>GHC-416</I></B>, the new yeasts we selected in the <I>Sc-Ura</I> plate is named as <B><I>SynⅩ-dUra-416</I></B>.</P><P>In this section, we only got to test the <I>Gal</I> System due to time limit. And we figured that the result for <I>Gal</I> System is adequately enough to represent the feasibility of our designed strategy for <B>MTS</B>. 
</P><P>The whole test process can be divided into three steps.</P><H5> 1)	Step one</H5><P> Activate the <I>Gal1</I> promoter. After that, the expression of HO gene in the <B><I>SynⅩ-dUra-416</I></B> can be initiated.</P><H5>2)   Step two</H5><P> Cultivate two groups of yeasts together. (one is <B><I>SynⅩ-dUra-416</I></B>, the other is normal <I>BY4741 MATa</I>) If the <B>MTS</B> has been accomplished (<B><I>SynⅩ-dUra-416</I></B> can become <I>MATα</I>), the two groups of haploids can mate with each other and become diploids. </P><H5>3)	Step three</H5><P>Test the results of mating by PCR method. We designed the primers for both <I>MATa</I> and <I>MATα</I> loci. The amplification of both <I>MATa</I> locus and <I>MATα</I> locus indicates that the yeasts has turned into diploids, the <B>MTS</B> has been achieved in other words. </P><P>According to our design, after activating the <I>Gal1</I> promoter, the expression of <B><I>HO</I></B> gene in the <B><I>SynⅩ-dUra-416</I></B> can be initiated.</P><P>Then we cultivated two groups of yeasts together. (one is <B><I>SynⅩ-dUra-416</I></B>, the other is normal <I>BY4741 MATa</I>) If the <B>MTS</B> has been accomplished (<B><I>SynⅩ-dUra-416</I></B> can become <I>MATα</I>), the two groups of haploids can mate with each other and become diploids. </P><P>To test whether <B>MTS</B> has happened, we selected some colonies in the selective plates (<I>Sc-Ura</I> ) and adopted PCR method. With designed primers for both <I>MATa</I> locus and <I>MATα</I> locus, the amplification of both <I>MATa</I> locus and <I>MATα</I> locus can indicate that the yeasts has mated with each other, and turned into diploids, in other words, the <B>MTS</B> has been achieved.</P><DIV class="zxx_zoom_demo"><DIV class="small_pic_demo" style="float:left;"><P style="font-size:15px;text-align:center">Fig. 1-4. (a) showed the PCR results for MATa locus. The MATa gene was amplified in all colonies except the first 24 colonies.
</P></DIV><DIV class="small_pic_demo" style="float:right;"><P style="font-size:15px;text-align:center">Fig. 1-4. (b) showed the PCR results for MATα locus. The MATα gene was amplified in all 96 colonies.</P></DIV></DIV><DIV id="pic_twentytwo" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 1-4. (a) showed the PCR results for MATa locus. The MATa gene was amplified in all colonies except the first 24 colonies.
Cadmium concentration: 5mM for the first and the third experiments; 10 mM for the second experiment.
</P></DIV><DIV id="pic_twentythree" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 1-4. (b) showed the PCR results for MATα locus. The MATα gene was amplified in all 96 colonies.</P></DIV><H4>Discussion &amp; Expectation</H4><P>To our delight, we verified that our strategy, namely <I>Gal</I> System, is able to trigger the <I>Mating Type Switching</I> in yeasts when induced by galactose. Thereby, we succeeded in building a new kind of on-off device in our target yeast——<B><I> SynⅩ-dUra-416</I></B>, where the whole system is ready to perform as a mating switcher to turn on/off other functional genes for further usage in our project——<B><I>Romantic Switcher</I></B>.</P><DIV class="reference"><H4>Reference</H4><P>[1] Sara J. Hanson, and Kenneth H. Wolfe.   An Evolutionary Perspective on Yeast Mating-Type Switching.</P><P>[2] James E. Haber,   Mating-Type Genes and MAT Switching in Saccharomyces cerevisiae.</P></DIV><DIV class="collapse-card__close_handler mt1 align-right mouse-pointer">
				UP </DIV></DIV></DIV></DIV><DIV class="langtab"><DIV class="collapse-card"><DIV class="collapse-card__heading"><H3 class="collapse-card__title">
				 Characterization of Mating Switcher
			</H3></DIV><DIV class="collapse-card__body"><H4>OVERVIEW</H4><P>After realizing that we need more intuitive characterization of Mating Switcher, we thought of two kinds of gene expression products in different colors, <I>red fluorescent protein</I> and <I>β-carotene</I>. We carried out a reasonable experimental design, and decided to realize functional conversion from <I>yEmRFP</I> to <I>β-carotene</I> by Mating Switcher.</P><P>At first, we built expression vector with <I>TEF</I> promoter, which was a strong promoter in <I>Saccharomyces cerevisiae</I>. Although we obtained results as we expected, it is not so perfect that we decided to change a stronger promoter. Then, we constructed another expression vector with <I>TDH3</I> promoter. We redid the same qualitative and quantitative experiments to characterize our results.
</P><H4>CONSTRUCTION</H4><P>In the early stage of the project, we constructed two device parts with <I>TEF</I> promoter: BBa_K2407306, BBa_K2407307. At the end of our project, we also constructed one device part with <I>TDH3</I> promoter:BBa_K2407314. Among them, <I>yEmRFP</I> is modified from a mCherry mRFP to adapt to the transcription environment in yeast. We did overlap PCR to construct them together. After that, we sequenced these parts, and sequencing results showed that these construction were successful.</P><DIV class="zxx_zoom_demo" align="center"><DIV class="small_pic_demo" align="center"><P style="font-size:15px;text-align:center">Fig.2-1. Red fluorescent protein expression vector construction flow chart.</P></DIV></DIV><DIV id="pic_fortyone" style="display:none;"><P style="font-size:15px;text-align:center">Fig.2-1. Red fluorescent protein expression vector construction flow chart.</P></DIV><P>Then we first inserted BBa_K2407306 to the <I>SynⅤ</I> of <I>Saccharomyces cerevisiae</I> . Through the screening of <I>SC-Ura</I>  solid medium and PCR experiments, we obtained the required strains called <B><I>PVUVC</I></B>. Second, we integrated the second device part into this chromosome through homologous recombination, allowing the <I>yEmRFP</I> gene to replace the <I>Ura3</I> gene. The <I>5-FOA</I> solid medium and PCR experiments were used to screen correct colony <B><I>PVRVC</I></B>. The insertion of the last part referred to the previous method. This process is graphically displayed on the above figure.</P><P>To achieve mating, another mating type of wild haploid yeast <I>Saccharomyces cerevisiae BY4742</I> was used for modification. By digestion and ligation, we constructed vika gene on plasmid <I>pRS416</I> which contains a selective marker <I>Ura3</I>, and plasmid <I>pRS413</I> which contains a selective marker <I>His</I>. Then we introduced those two different plasmids into <I>BY4742</I> respectively.</P><H4>Results of Characterization of Mating Switcher</H4><H5>1) Proof of Existence of Device Parts</H5><P>We built three device parts in total. They were integrated into the chromosomes of <I>Saccharomyces cerevisiae</I> by transformation. We used colony PCR to prove the existence of these three parts in our strain. The result was showed as below.</P><P style="text-align:center;font-size:1em;">Fig.2-2. The results of PCR of <B><I>PVUVC</I></B>, <B><I>TVUVC</I></B>, <B><I>PVUVC</I></B> colonies. (length of 7607bp, 7865bp, 8131bp) As we can see, three parts and all fragments had been amplified, which indicated that we succeeded in constructing them.</P><P>The PCR’s results confirmed that the target genes were ligated into chromosome correctly.</P><H5>2) Verification of yEmRFP in the colonies</H5><P>The main characterization method of verification of <I>yEmRFP</I> in the <B><I>TVRVC</I></B> applied by us was observing the expression of <I>red fluorescent protein</I> under the fluorescence microscope. By this way, it will be much more intuitive so that we can directly get the results. We took pictures under different visions and the results are as follows.The experiments of <B><I>PVRVC</I></B> regulation system used this assay method as well.</P><P style="text-align:center;font-size:1em;">Fig.2-3. Microscope image of yeast
cultured with <I>SC-Leu</I> with <I>yEmRFP</I> gene transformed.</P><P style="text-align:center;font-size:1em;">Fig.2-3. Microscope image of yeast
cultured with <I>SC-Leu</I> with <I>yEmRFP</I> gene transformed.</P><P>From these images we can clearly see the expression of <I>yEmRFP</I>. These images undoubtedly verify the <I>yEmRFP</I> gene has been transformed succeessfully.</P><H5>3) Result of Mating</H5><P>After we got the strain that introduced the <I>yEmRFP</I> gene, we let it mate with another mating type haploid yeast, which had plasmid with <I>vika</I> gene. The result was showed as follows:</P><P style="font-size:15px;text-align:center;margin:0 6em">Fig.2-4. Three modified colonies and one resulting colony.
The upper left corner of the microorganism is synthetic <I>Saccharomyces cerevisiae</I>, we integrated modified fragment into its <I>synthetic chromosome V</I>. (<B><I>PVUVC</I></B>) The upper right corner is also synthetic <I>Saccharomyces cerevisiae</I>. (<B><I>PVRVC</I></B>) It is imported <I>red fluorescent protein</I> gene based on the upper left corner of the yeast. Both of them are single-celled organism called a. The lower right corner of the yeast is another mating type of haploid yeast called α. It has plasmid <I>pRS416</I> with <I>vika</I> gene. The yeast in the lower left corner is diploid <I>Saccharomyces cerevisiae</I>, which is obtained by mating the two yeasts on the right side of the figure.</P><DIV id="pic_fortythree" style="display:none;"><P style="font-size:15px;text-align:center">Fig.2-4. Three modified colonies and one resulting colony.
The upper left corner of the microorganism is synthetic <I>Saccharomyces cerevisiae</I>, we integrated modified fragment into its <I>synthetic chromosome V</I>. (<B><I>PVUVC</I></B>) The upper right corner is also synthetic <I>Saccharomyces cerevisiae</I>. (<B><I>PVRVC</I></B>) It is imported <I>red fluorescent protein</I> gene based on the upper left corner of the yeast. Both of them are single-celled organism called a. The lower right corner of the yeast is another mating type of haploid yeast called α. It has plasmid <I>pRS416</I> with <I>vika</I> gene. The yeast in the lower left corner is diploid <I>Saccharomyces cerevisiae</I>, which is obtained by mating the two yeasts on the right side of the figure.</P></DIV><P>The yellow colony in the figure was mating successfully. After the induction of <I>galactose</I>, <I>vika recombinase</I> was expressed, and <I>yEmRFP</I> gene and terminator was deleted so that <I>β-carotene</I> expresses. The color of colony was changed from white to yellow. In addition to it, we also tried other methods to turn on the switch.</P><DIV class="zxx_zoom_demo_angry" align="center"><DIV class="small_pic_demo_angry" align="center"><P style="font-size:15px;text-align:center">Fig.2-5. Yeast after mating cultivated on the Sc-His plate.There are 377 yellow colonies and 365 white colonies in the field of view.</P></DIV></DIV><DIV id="pic_fortyfour" style="display:none;">Fig.2-5. Yeast after mating cultivated on the Sc-His plate.
There are 377 yellow colonies and 365 white colonies in the field of view.</DIV><P>We used another α-type yeast named <I>BY4742</I>, which has a plasmid <I>pRS413</I> with selective marker <I>His</I>. It could express <I>vika recombinase</I> before mating. It mated with a-type <I>Saccharomyces cerevisiae</I><B><I>PVRVC</I></B>, and then yeast cultured on Sc-His plate. As can be seen from the figure above, the reorganization efficiency is high, which reaches 50.8 percent. This proves that our Mating switcher is fast and efficient.</P><DIV class="zxx_zoom_demo_angry" align="center"><DIV class="small_pic_demo_angry" align="center"><P style="font-size:15px;text-align:center">Fig.2-6. Yeast after induction cultivated on the Sc-Ura plate.
There are 325 yellow colonies and 31 white colonies in the field of view.</P></DIV></DIV><DIV id="pic_fortyfive" style="display:none;">Fig.2-6. Yeast after induction cultivated on the Sc-Ura plate.
There are 325 yellow colonies and 31 white colonies in the field of view.</DIV><P>Apart from mating, we also transformed plasmid <I>pRS416</I> with <I>vika</I> gene into the <B><I>PVRVC</I></B>. The efficiency is up to 91.3 percent in this figure.</P><P>Compare above two methods, we found that mating was not as efficient as the transformation of the plasmid. After analysis, we came to the conclusions as follows. For the mating method, <I>vika recombinase</I> has stopped expressing when <I>BY4742</I> mated with <I><B>PVRVC</B></I> in YPD medium. The previously expressed <I>vika recombinase</I> may be degraded during the growth. In contrast to this, with another method that the plasmid was transformed into <I><B>PVRVC</B></I> directly, <I>vika recombinase</I> was continuously expressing during growth. So the efficiency of the second method is higher than the first method.</P><DIV class="zxx_zoom_demo_qqqqq" align="center"><DIV class="small_pic_demo_qqqqq" align="center"><P style="font-size:15px;text-align:center">Fig.2-7. Four modified coloniesinserted with promotor-vox-RFP-terminators-vox-crt structure</P></DIV></DIV><DIV id="pic_fortysix" style="display:none;">Fig.2-7. Four modified coloniesinserted with promotor-vox-RFP-terminators-vox-crt structure</DIV><P>We also used other <I>Saccharomyces cerevisiae</I> with mating type of a to achieve mating switcher. After changing <I>TEF</I> promotor to <I>TDH3</I> promotor, we repeated the test according to the above two methods. The four strains are all haploid synthetic <I>Saccharomyces cerevisiae</I> with mating type of a named <I><B>TVRVC</B></I> NO.2 (upper left), NO.4 (upper right), NO.11 (lower left) and NO.19 (lower right) respectively. The color appeared to be white because <I>β-carotene</I> did not express.</P><DIV class="zxx_zoom_demo_qqqqq" align="center"><DIV class="small_pic_demo_qqqqq" align="center"><P style="font-size:15px;text-align:center">Fig.2-8. Successful mating colonies</P></DIV></DIV><DIV id="pic_fortyseven" style="display:none;">Fig.2-8. Successful mating colonies</DIV><P>These are parts of successful results of mating mentioned above.</P><P>To sum up, the mating switcher can be presented in kinds of yeast with different forms. This proved that our Mating switcher was fast, flexible and efficient.</P><P>Meantime, we cultured the transformed yeast in 5mL liquid <I>SC-Leu</I> at 30℃ and 220 rpm for 12 hours ( Take three samples at a time). We used one sample for centrifugation to precipitate the yeast and the remaining two remained unchanged. The difference was the fluorescence value we need, then we calculated the average value of them. The excitation wavelength was set at 540nm and the emission wavelength was set at 635nm. Hereafter, we measured the yeast concentration at OD<SUB>600</SUB>. At last, we divided the fluorescence value by OD<SUB>600</SUB> to normalize the value and the result data was as follows.
</P><DIV class="zxx_zoom_demo_qqqqq" align="center"><DIV class="small_pic_demo_qqqqq" align="center"><P style="font-size:15px;text-align:center">Fig.2-9. Normalized fluorescence value was calculated by dividing fluorescent value by cell concentration(OD<SUB>600</SUB>)</P></DIV></DIV><DIV id="pic_fiftythree" style="display:none;"><P style="font-size:15px;text-align:center">Fig.2-9
Normalized fluorescence value was calculated by dividing fluorescent value by cell concentration(OD<SUB>600</SUB>)</P></DIV><P>From the data we can found that the fluorescence intensity of the modified yeast was more than twice that of the wild type. Low red fluorescence was detected after yeast mating, which can be attributed to the influence of <I>β-carotene</I>.</P><H4>DISCUSSION &amp; FUTURE WORK</H4><P>Our mating switch plays an important role in many respects, such as including heavy metal treatment and cell signal switching. And we created a novel method to prove the effectiveness of the switch in an intuitive and effective way. The terminator of the first part (<B><I>PVUVC</I></B>) terminates the expression of the downstream gene, proving the validity of the switcher, and the second part (<B><I>PVRVC</I></B>) creates an evident method of color conversion to determine the state of the switcher.</P><P>Aiming to increase the <I>Vika-vox</I> system efficiency, we let <I>Vika</I> enzyme saturate expression, but the efficiency was still relatively low. We hypothesized that this phenomenon was caused by degradation of the <I>Vika</I> enzyme in the YPD culture medium. We’d better change the composition or proportion of YPD ingredients to find out the best culture conditions. We are looking forward to more research in this field so that we can make this system work better and even perfectly.</P><P>We use the <I>RFP</I> as the reporting protein. But there exists a drawback that it’s detected with an expensive device. A more intuitive reporting strategy need to be developed, maybe it can be seen by bare eyes like <I>E.coli</I> in the near future.</P><DIV class="reference"><H4>Reference</H4><P>[1]Altamura E, Borgatti M, Finotti A, Gasparello J, Gambari R, Spinelli M, et al. (2016) Chemical-Induced Read-Through at Premature Termination Codons Determined by a Rapid Dual-Fluorescence System Based on S. cerevisiae. PLoS ONE11(4)</P><P>[2]Leslie A. Mitchell, James Chuang, Neta Agmon, Chachrit Khunsriraksakul, Nick A. Phillips, Yizhi Cai, David M. Truong, Ashan Veerakumar, Yuxuan Wang, María Mayorga, Paul Blomquist, Praneeth Sadda, Joshua Trueheart, Jef D. Boeke; Versatile genetic assembly system (VEGAS) to assemble pathways for expression in <I>S. cerevisiae</I>,<I> Nucleic Acids Research</I>, Volume 43, Issue 13, 27 July 2015, Pages 6620–6630
</P></DIV><DIV class="collapse-card__close_handler mt1 align-right mouse-pointer">
				Show less </DIV></DIV></DIV></DIV><DIV class="langtab"><DIV class="collapse-card"><DIV class="collapse-card__heading"><H3 class="collapse-card__title">   
				 SCRaMbLE: Improve Resistance to Heavy Metal Ions 
			</H3></DIV><DIV class="collapse-card__body"><H4>OVERVIEW</H4><P>After doing relevant literature reading, we found that yeast’s tolerance level of ambient copper and cadmium ions has a threshold concentration, approximately 3mM and 0.5mM in SC culture media respectively.</P><P> In order to increase yeast strains’ inherent tolerance of copper or/and cadmium ions in their growing environment, we used this cutting-edge biological technology—SCRaMbLE, which stands for Synthetic Chromosome Rearrangement and Modification by <I>Loxpsym</I>-mediated Evolution, to obtain yeast strain with better tolerance to heavy metal ions<SUP>[1]</SUP>. </P><P>We constructed three yeast strains namely 079, 160, and 085. They all have a plasmid containing the CRE-EBD sequence and different nutritional labels<SUP>[2]</SUP>. 079 and 160 strains have a URA3 label, 085 strain has a HIS label. After proper induction and screening, we successfully obtained mutated 079, 085 and 160 strains that have a manifest growing advantage over control groups when cultured in SC solid media which contain 0.14 mM cadmium ions or 4.8 mM copper ions. We named those mutated strains with increased tolerance capacity of cadmium ions S1, S2, S3, and S4, and as for copper, S5, S6, S7, and S8.</P><P>In order to characterize their increased tolerance of copper or/and cadmium ions, we designed and conducted two different sets of experiments, in both visible and quantitative manner, to test their ability to cope with adverse environmental conditions.</P><H4>CONSTRUCTION</H4><P>This vector consists of three parts, an estrogen-inducible PCLB2 promoter <A href="http://parts.igem.org/Part:BBa_K2407002">(BBa_K2407002)</A>, the Cre-EBD sequence <A href="http://parts.igem.org/Part:BBa_K2407005">(BBa_K2407005)</A> and a CYC1 terminator<A href="http://parts.igem.org/Part:BBa_K2407003">(BBa_K2407003)</A>. We used overlap PCR to ligate these three parts and then the plasmids with URA3 and HIS nutritional label respectively through enzymatic digestion and ligation. Then this composite part <A href="http://parts.igem.org/Part:BBa_K2407011">(BBa_K2407011）</A>,was sequenced and proved to be accurate by using the promoter's forward primer and the terminator's reverse primer. The electrophoresis results below also showcased that the sequence length (about 2800bp) was correct.</P><DIV class="zxx_zoom_demo" align="center"><DIV class="small_pic_demo" align="center"><P style="font-size:15px;text-align:center">Fig.3-1 This is a simplified version of this vector expressing the Cre recombinase enzyme. Cre-EBD is the coding sequence of Cre recombinase; PCLB2 is a constitutive  promoter in yeast; CYC1 is a terminator.</P></DIV></DIV><H4>CHARACTERIZATION</H4><H5>Dilution Assay </H5><P>We conducted dilution assay on SC solid media containing 0.14 mM cadmium ions. Experimental groups are S1, S2, S3, and S4; control groups are <I>synX</I> (the yeast strain containing a synthetic chromosome X), <I>BY4741</I> (wild-type haploid yeast), and <I>BY4743</I> (wild-type diploid yeast). Results are shown in the picture below.</P><DIV class="zxx_zoom_long" align="center"><DIV class="small_pic_long" align="center"><P style="font-size:15px;text-align:center">Fig.3-2 Experimental groups: S1, S2, S3, and S4; control groups: <I>synX</I>, <I>BY474</I>, and <I>BY4743</I>. Yeast cells were picked up from single colonies on solid growth media and diluted by 10, 100, 1000, 10000, and 100000 times.</P></DIV></DIV><P>Apparently, the experimental groups have a survival advantage over control groups. In this picture, S1 is able to develop a large single colony even after it is diluted to 100000 times on SC solid media containing 0.14 mM cadmium ions; S3 and S4 are able to grow when diluted to 100000 times but the colonies are much smaller than that of S1. Although S2 is not as good as the other three, it still shows higher resistance to cadmium ions than the control groups do. Wild-type yeast strains <I>BY4741</I> and <I>BY4743</I> can barely grow on this growth media, while <I>synX</I> cannot grow, which means that <I>synX</I> is unable to survive such high concentration of cadmium ions. The results clearly demonstrate that these mutated yeast strains have an improved phenotype-increased resistance to cadmium ions.</P><P>Another assay was conducted on SC solid media containing 4.8 mM copper ions. Results are shown in the picture below. </P><DIV class="zxx_zoom_long" align="center"><DIV class="small_pic_long" align="center"><P style="font-size:15px;text-align:center">Fig.3-3 Experimental groups: S5, S6, S7, and S8; control groups: <I>synX</I>, <I>BY4741</I>, and <I>BY4743</I>. Yeast cells were picked up from single colonies on solid growth media and diluted by 10, 100, 1000, 10000, and 100000 times. </P></DIV></DIV><P>The experimental groups also have a survival advantage over control groups. From this picture, S5, S6, and S8 are able to develop a large single colony after diluted to 100000 times on SC solid media containing 4.8 mM copper ions. S7 is not as tough as the other three experimental groups, but it still shows higher resistance to copper ions compared with <I>BY4741</I> when diluted to 100 times. <I>BY4743</I> can hardly grow on this media, while <I>synX</I> cannot grow, which means that <I>synX</I> is unable to tolerate such high concentration of copper ions. The results clearly showcase that the mutated yeast strains have an improved phenotype-increased resistance to copper ions.</P><H5>Survival Rate Experiments </H5><P>This experiment aims to quantify mutated yeast strains’ ability to survive in copper or cadmium ions solution. Same amount of yeast cells are added to the copper or cadmium ions solution at the beginning; after that, a certain amount of this solution is taken out at regular intervals, namely 10min, 30min, 1h, and 2h, then diluted and plated on YPD solid media. After yeast colonies emerge from the growth media, the number of the colonies are counted and recorded to calculate the survival rate of this strain in this solution. </P><P>We choose the seemingly best strain, S1, as the experimental strain to test its ability to survive high concentration of cadmium compared with the control strain, <I>synX</I>. Results are shown in the pictures and tables below.</P><DIV class="zxx_zoom_joinpic" align="center"><DIV class="small_pic_joinpic" align="center"><P style="font-size:15px;text-align:center">Fig.3-4 The left: Colony numbers of <I>synX</I> and S1. Experiments were repeated for three times. Cadmium concentration: 5mM for the first and the third experiments; 10 mM for the second experiment.The right: Survival rates of <I>synX</I> and S1 in cadmium ion solution. In order to eliminate the error brought by the different amount of cells picked up from the media, the survival rate equals the value of the number of the colonies developed on growth media after immersed in cadmium solution for a certain amount of time divided by that of the colonies developed on growth media after immersed in cadmium solution for 10 minutes. Mean values are shown in the graph. The error line stands for the standard deviation.</P></DIV></DIV><DIV class="zxx_zoom_figthree" align="center"><DIV class="small_pic_figthree" align="center"><P style="font-size:15px;text-align:center">Fig.3-5 Picture of the third survival rate experiment results. Growth media: YPD. Experimental strain: S1; control strain: synX. Cells were plated onto the media after immersed in 5 mM cadmium solution for 10 minutes, 30minutes, 1 hour, and 2hours.</P></DIV></DIV><P>The Fig.3-4 and Fig.3-5 showcase that the survival rate of S1 is higher than that of <I>synX</I> after yeast cells are immersed in cadmium ions solutions of identical concentration for the same amount of time. We painstakingly counted and recorded the number of the colonies on each individual growth media. The quantitative results are that compared with the control strain, the experimental strain S1's ability to tolerate cadmium ions has increased by 23.8% (30 minutes), 231.9% (1 hour), and 192.4% (2 hours). The longer the time of immersion is, the more obvious the difference of survival rates is. The results are consistent with the dilution assay, which is that the mutated strain has a better resistance level of cadmium ions .</P><P>As for copper, the seemingly best strain, S5, is chosen as the experimental strain to test its ability to survive high concentration of copper ions compared with <I>synX</I>. Results are shown in the pictures and tables below.</P><DIV class="zxx_zoom_joinpic" align="center"><DIV class="small_pic_joinpic" align="center"><P style="font-size:15px;text-align:center">Fig.3-6 The left: Colony numbers of <I>synX</I> and S5. Due to the time limit, we conducted this  experiment for only one time. Yeast cells were immersed in 50mM copper solutions for 10 minutes, 30 minuets, 1 hour, and 2 hours.The right: Survival rates of <I>synX</I> and S5 in copper solution. In order to eliminate the error brought by the different amount of cells picked up from the media, the survival rate equals the value of the number of the colonies developed on growth media after immersed in copper solution for a certain amount of time divided by that of the colonies developed on growth media after immersed in copper solution for 10 minutes. Mean values are shown in the graph. The error line stands for the standard deviation.</P></DIV></DIV><DIV class="zxx_zoom_figseven" align="center"><DIV class="small_pic_figseven" align="center"><P style="font-size:15px;text-align:center">Fig.3-7 Picture of survival rate experiment results. Growth media: YPD. Experimental strain: S5; control strain: synX. Cells were plated onto the media after immersed in 50 mM copper solution for 10 minutes, 30minutes, 1 hour, and 2hours.</P></DIV></DIV><P>The Fig.3-6 and Fig.3-7 showcase that the survival rate of S5 is higher than that of <I>synX</I> after yeast cells are immersed in copper ions solutions of identical concentration for the same amount of time. The quantitative results are that compared with the control strain, the experimental strain S5's ability to tolerate copper ions has increased by 74% (1 hour), 72% (2 hours), and 698% (3 hours). It also can be extrapolated that the gap of survival rates between the mutated strain and <I>synX</I> strain will continue to widen as the immersion time increases. The results are consistent with the dilution assay too. </P><H4>EXPECTATIONS</H4><P>We are exhilarated to see that SCRaMbLE is really a feasible technology to enhance the yeast's ability to cope with adverse environmental conditions. Not just heavy metal ions, we are looking forward to seeing its future applications, be they, for example, alcohol tolerance or heat tolerance. </P><DIV class="reference"><H4>REFERENCE</H4><P>[1]Shen, Y., Stracquadanio, G., Wang, Y., Yang, K., Mitchell, L. A., &amp; Xue, Y., et al. (2016). Scramble generates designed combinatorial stochastic diversity in synthetic chromosomes. <I>Genome Research</I>, 26(1), 36.   
[2]Lindstrom, D. L., &amp; Gottschling, D. E. (2009). The mother enrichment program: a genetic system for facile replicative life span analysis in <I>Saccharomyces cerevisiae</I>.<I>Genetics</I>, 183(2):413.</P></DIV><DIV class="collapse-card__close_handler mt1 align-right mouse-pointer">
				Show less </DIV></DIV></DIV></DIV><DIV class="langtab"><DIV class="collapse-card"><DIV class="collapse-card__heading"><H3 class="collapse-card__title">
				 Biosensor for Cu Ion's Detection
			</H3></DIV><DIV class="collapse-card__body"><H4>OVERVIEW</H4><P>After communicating with professors, teachers, and factory superintendents, our HPers found that it was difficult to monitor the concentration of the copper ions in solution in real-time. Using a biosensor seems to be a good solution to this problem.

</P><P>This idea was inspired by the naturally-occurring metal-ion-induced promoters. Ligating this kind of promoters with a reporter gene such as RFP is a common idea to visibly monitor the concentration of metal ions. Take copper as an example: after browsing through parts, we found copper-ion-induced promoters in both <I>E.coli</I> and <I>S.cerevisiae</I>. Actually, the ability of <I>E.coli</I> and <I>S.cerevisiae</I> to tolerate copper ions differs from each other. <I>E.coli</I>'s maximum tolerance level to copper ions is 1 mM, which is much less than that of <I>S.cerevisiae</I>'s (over 15mM in YPD medium and 6.25 mM in SC medium). Considering the response range, the budding yeast is a much better host for copper detection. </P><P>We built a biosensor based on the <I>CUP1</I> promoter and <I>yEmRFP</I> to monitor the concentration of copper ions. The response range of this biosensor was characterized by a fluorescent spectrophotometer (Hitachi F-2700). To improve the sensitivity of the biosensor and enlarge the response intensity when it is induced, we used error-prone PCR to obtain lots of promoter mutants and then characterized them.
</P><H4>CONSTRUCTION</H4><P>The biosensor consists of two main parts. One is the Cu-induced promoter <I>CUP1p</I>, the other is <I>yEmRFP</I>, which is modified from a <I>mCherry mRFP</I> to adapt to the transcription environment in yeast. The promoter was synthesized without RFC sites (XbaI) and the RFP was amplified by PCR. We used overlap PCR to combine the two parts and added two restriction sites on the ends. By digestion and ligation, we constructed this biosensor on the plasmid pRS416 which contains a selective marker URA3. After that, we sequenced this part with M13F and M13R as primers. The sequencing result showed that this construction was successful, so we can take the next step – characterization.  
</P><DIV class="row"><DIV align="center"><DIV class="col-md-8"><DIV class="zxx_zoom_left" align="center"><DIV class="small_pic" align="center"><DIV align="center"><P style="font-size:15px;text-align:center;">Fig. 4-1. Sequencing result of <I>CUP1p</I> and <I>yEmRFP</I> (alignment support by SnapGene®)</P></DIV></DIV></DIV><DIV id="pic_one" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 4-1. Sequencing result of <I>CUP1p</I> and <I>yEmRFP</I> (alignment support by SnapGene®) </P></DIV></DIV></DIV></DIV><H4>CHARACTERIZATION</H4><P>Based on the <I>CUP1</I> promoter (<A href="http://parts.igem.org/Part:BBa_K2165004">BBa_K2165004</A>) provided by iGEM16_Washington, we constructed this biosensor. To characterize this biosensor, strains of <I>S. cerevisiae BY4742</I> containing the plasmid with an initial OD<SUB>600</SUB> of 0.1 were grown for 24 hours in SC-URA medium at 30 degrees Celsius, and then were induced with copper sulfate. Samples in different copper concentration were tested with the fluorescent spectrophotometer (Hitachi F-2700) after 1, 6, 12, and 24 hours. This protocol was based on the experience used by Waterloo and Washington iGEM teams and amended by our team.</P><DIV class="row"><DIV align="center"><DIV class="col-md-8"><DIV class="zxx_zoom_demo_qqqqqq" align="center"><DIV class="small_pic_demo_qqqqqq" align="center"><P style="font-size:15px;text-align:center">Fig. 4-2. The fluorescence intensity of CUP1p-yEmRFP biosensor with different Cu concentration induced</P></DIV></DIV><DIV id="pic_two" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 4-2. The fluorescence intensity of CUP1p-yEmRFP biosensor with different Cu concentration induced </P></DIV></DIV></DIV></DIV><P>Figure 4-2 showed the relationship between fluorescence intensity with induction time and Cu concentration. With 0.1 mM CuSO<SUB>4</SUB> induced, the fluorescence intensity is 2 times over a control with no induction at 1 hour. As time went on, the fluorescence intensity slightly reduced. Moreover, as the Cu concentration increased, the fluorescence intensity decreased, and when the concentration reached 1 mM, the intensity was close to the control group. This might be due to the higher copper ion concentration influences the transcription, expression and even growth of yeast.</P><P>This result will be useful for teams who will use the parts <A href="http://parts.igem.org/Part:BBa_K2165004">(BBa_K2165004)</A>&amp;<A href="http://parts.igem.org/Part:BBa_K2407012">(BBa_K2407012)</A> to build an effective Cu-induced biosensor in budding yeast. We noticed that this result is a little different with works done by Waterloo team. It may be due to the differences between Cu ion’s concentration and yeast species. However, we both verified the possibility of building a biosensor based on <I>CUP1</I> promoter in yeast.</P><P>This result was provided for modeling of this biosensor and try to find a proper function to accurately describe the response procedure. <A href="https://2017.igem.org/Team:Tianjin/Model"><I>Click here to see more information.</I></A></P><H4>PARTS IMPROVEMENT</H4><P>The Cu-induced promoter <I>CUP1</I> promoter is a previous BioBrick used by iGEM16_Washington, iGEM16_Waterloo, and other iGEM teams. However, the detailed characterization like what we did this year hasn't been shown on iGEM parts page. Moreover, this part hasn’t be improved by any means or in any way. Under this situation, we plan to work on this promoter to improve its sensitivity and response peak, reduce the leakage expression, and create new parts for future work.</P><H5>1)	Redesign of <I>CUP1</I> promoter</H5><P>First, based on the part <A href="http://parts.igem.org/Part:BBa_K2165004">(BBa_K2165004)</A> provided by iGEM16_Washington, we tried to ensure the core sequence for transcription. Researches about this promoter mainly published in the 1990s, and the mechanism of induction has been researched thoroughly. There exist 5 <I>ACE1</I> binding sites (UAS), 2 <I>TATA</I> boxes, and one initiation element in the promoter. The complex of <I>ACE1</I> and copper ions will bind the promoter, which causes the activation of <I>CUP1</I> promoter with <I>TATA</I> boxes’ help. <I>ACE1</I> complex’s binding directly increases the possibility for TBP (<I>TATA</I>-Box Binding Protein) to bind the promoter, which can enhance the expression.</P><DIV class="row"><DIV align="center"><DIV class="col-md-8"><DIV class="zxx_zoom_demo_qqqqqq" align="center"><DIV class="small_pic_demo_qqqqqq" align="center"><DIV align="center"><P style="font-size:15px;text-align:center">Fig. 4-3. structure of redesigned <I>CUP1</I> promoter used in our project, based on BBa_K2165004</P></DIV></DIV></DIV><DIV id="pic_three" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 4-3. structure of redesigned <I>CUP1</I> promoter used in our project, based on BBa_K2165004 </P></DIV></DIV></DIV></DIV><P>Based on this mechanism, we redesigned the part sequence provided by iGEM16_Washington. We deleted irrelevant based on the two ends of this promoter and retained the core sequence. In this way, this promoter played its key role with fewer bases. Strains of <I>S. cerevisiae BY4742</I> containing either BBa_K2165004-yEmRFP and BBa_K2407000-yEmRFP with an initial OD<SUB>600</SUB> of 0.1 were grown for 24 hours in SC-URA medium at 30 degrees Celsius, and then were induced with 0.1 mM Cu<SUP>2+</SUP>. Samples were tested with fluorescent spectrophotometer (Hitachi F-2700) after 1, 3, 6, 12, and 24 hours.</P><DIV class="row"><DIV align="center"><DIV class="col-md-8"><DIV class="zxx_zoom_demo_qqqqqq" align="center"><DIV class="small_pic_demo_qqqqqq" align="center"><DIV align="center"><P style="font-size:15px;text-align:center">Fig. 4-4. structure of redesigned CUP1 promoter used in our project, based on BBa_K2165004</P></DIV></DIV></DIV><DIV id="pic_eighty_one" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 4-4. structure of redesigned CUP1 promoter used in our project, based on BBa_K2165004 </P></DIV></DIV></DIV></DIV><P>Figure 4-4 shows the expression of <I>yEmRFP</I> with both the two promoters were very similar, so we can tell the deletion didn’t influence the core function of <I>CUP1</I> promoter. Based on the new part, we carried out a further improvement.</P><H5>2)	Error-Prone PCR</H5><P>In our experiment, we noticed that <I>CUP1</I> promoter still has a certain degree of leakage expression. To make a better biosensor, we planned to reduce the leakage expression and increase the sensitivity. To reach this goal, we took the fluorescence intensity at both inductions or not into evaluation indexes.</P><P>The technology of error-prone PCR was taken into our experiment. Although there are many methods to introduce genetic diversity into a parent sequence, error-prone PCR is the most common way of creating a combinatorial library based on a single sequence. By adding some heavy metal ions into the PCR buffer and preparing dNTPs with different composition, new mutants were introduced into the CUP1 promoter. The whole procedure is shown in the following figure.</P><DIV class="row"><DIV align="center"><DIV class="col-md-8"><DIV class="zxx_zoom_demo_qqqqqqq" align="center"><DIV class="small_pic_demo_qqqqqqq" align="center"><DIV align="center"><P style="font-size:15px;text-align:center">Fig. 4-5. The procedure for error-prone PCR experiment</P></DIV></DIV></DIV><DIV id="pic_four" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 4-5. The procedure for error-prone PCR experiment</P></DIV></DIV></DIV></DIV><P>The library of promoter mutants obtained from error-prone PCR were ligated into plasmid pRS416 with two restriction sites (BamHI and XbaI). After that, we enriched different plasmids from <I>E.coli</I> and established the plasmid library with 132 samples. Then, different plasmids were transferred into <I>S.cerevisiae BY4742</I> to test the fluorescence intensity under different conditions.</P><P>First, we tested and selected mutants with less leakage. Compared with control group, we test the fluorescence intensity with no induction and selected two mutants with lower fluorescence intensity. Actually, we test a lot of mutants, but most of them were not positive result. We picked EP-3, EP-5, EP-9, and EP-28 whose fluorescence intensity was less or close to the control group, and sequenced them. The sequencing results can be found in the parts' information: <A href="http://parts.igem.org/Part:BBa_K2407013">BBa_K2407013 (EP-3)</A>, <A href="http://parts.igem.org/Part:BBa_K2407014">BBa_K2407014 (EP-5)</A>, <A href="http://parts.igem.org/Part:BBa_K2407015">BBa_K2407015 (EP-9)</A>, <A href="http://parts.igem.org/Part:BBa_K2407016">BBa_K2407016 (EP-28)</A>.</P><DIV class="row"><DIV align="center"><DIV class="col-md-8"><DIV class="zxx_zoom_demo_qqqqqq" align="center"><DIV class="small_pic_demo_qqqqqq" align="center"><DIV align="center"><P style="font-size:15px;text-align:center">Fig. 4-6. The leakage expression of different promoters</P></DIV></DIV></DIV><DIV id="pic_five" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 4-6. The leakage expression of different promoters</P></DIV></DIV></DIV></DIV><P>Second, we worked on the sensitivity of the biosensor. Leakage expression was not the only thing needed to be solved, and we also needed to increase the response range when it was induced. A good biosensor needs less leakage and more sensitivity. </P><P>We tested the 4 selected biosensors and control group with 0, 10, 100, 500, and 1000 μM Cu<SUP>2+</SUP> induced for 20 min, and the result is shown below with logarithmic coordinates.</P><DIV class="row"><DIV align="center"><DIV class="col-md-8"><DIV class="zxx_zoom_demo_qqqqqq" align="center"><DIV class="small_pic_demo_qqqqqq" align="center"><DIV align="center"><P style="font-size:15px;text-align:center">Fig. 4-7. The leakage expression of different promoters</P></DIV></DIV></DIV><DIV id="pic_six" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 4-7. The leakage expression of different promoters</P></DIV></DIV></DIV></DIV><P>The figure shows the response ranges of biosensors with different promoters within 20 min. For most biosensors, the fluorescence intensity increases as copper ion’s concentration increases from 0 to 100 μM. However, when the concentration exceeds 100 μM, the responses of most biosensor become slow, and the fluorescence intensity decreases. A reasonable explanation is that high concentrations of copper can inhibit the biosensor's response within a short time. </P><P>Fortunately, we still found a biosensor who met the requirements of an excellent biosensor. EP-5 has a less leakage and a higher sensitivity. Its fluorescence intensity is lower than the control group by 17 units with no induction and is higher by 21 units with 100-μM-Cu induction. By aligning the sequence with the CUP1 promoter, we found altered bases mainly located at both sides of UASs and a deletion of one base even occurred between two UASs. We suspected that the change of sensitivity and leakage expression mainly due to the change of spatial distribution and the increase of A/T concentration, which both could influence the binding procedure of transcription factors.</P><DIV class="row"><DIV align="center"><DIV class="col-md-8"><DIV class="zxx_zoom_demo_qqqqqq" align="center"><DIV class="small_pic_demo_qqqqqq" align="center"><DIV align="center"><P style="font-size:15px;text-align:center">Fig. 4-8. The base changes in EP-5</P></DIV></DIV></DIV><DIV id="pic_seven" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 4-8. The base changes in EP-5</P></DIV></DIV></DIV></DIV><H4>DISCUSSION &amp; FUTURE WORK</H4><P>In our characterization of both primary and improved promoters, we found the effect of induction is not as obvious as expected (Previous iGEM team’s results). After reading some references, we found the activation process is related to the acetylation of H3 and H4 located at the  <I>CUP1</I> promoter, which showed nucleosome reposition and transcription factors binding might be the main reason for the activation. However, our biosensors were ligated on plasmid pRS416, which usually exists in the nucleus in a supercoiled state. There is only little possibility for a plasmid to binds to histones, so the transcription process shows less activation than that on a chromosome.</P><P>In the future, we plan to construct this biosensor on chromosomes to see whether the result will be more positive. Meanwhile, we will continue enlarging the response peak and range to improve this biosensor.</P><DIV class="reference"><H4>Reference</H4><P>[1]Badi, L., &amp; Barberis, A. (2002). The CUP1 upstream repeated element renders CUP1 promoter activation insensitive to mutations in the RNA polymerase II transcription complex. <I>Nucleic acids research</I>, 30(6), 1306-1315.</P><P>[2]Koller, A., Valesco, J., &amp; Subramani, S. (2000). The CUP1 promoter of Saccharomyces cerevisiae is inducible by copper in Pichia pastoris. <I>Yeast</I>, 16(7), 651-656.</P><P>[3]Labbé, S., &amp; Thiele, D. J. (1999). [8] Copper ion inducible and repressible promoter systems in yeast. <I>Methods in enzymology</I>, 306, 145-153.</P><P>[4]Leblanc, B. P., Benham, C. J., &amp; Clark, D. J. (2000). An initiation element in the yeast CUP1 promoter is recognized by RNA polymerase II in the absence of TATA box-binding protein if the DNA is negatively supercoiled. <I>Proceedings of the National Academy of Sciences</I>, 97(20), 10745-10750.</P><P>[5]Shen, C. H., Leblanc, B. P., Neal, C., Akhavan, R., &amp; Clark, D. J. (2002). Targeted histone acetylation at the yeast CUP1 promoter requires the transcriptional activator, the TATA boxes, and the putative histone acetylase encoded by SPT10. <I>Molecular and cellular biology</I>, 22(18), 6406-6416.</P></DIV><DIV class="collapse-card__close_handler mt1 align-right mouse-pointer">
				Show less </DIV></DIV></DIV></DIV><DIV class="langtab"><DIV class="collapse-card"><DIV class="collapse-card__heading"><H3 class="collapse-card__title">
				 Heavy Metals' Enrichment
			</H3></DIV><DIV class="collapse-card__body"><H4>overview</H4><P>Our HPers have conducteded research in Shanxi Province. They found factories directly choose to deal with only one main kind of the heavy metals, then abandon others. The convenient treatment obviously can't satisfy the environmental requirements. Thus, we demand a type of bacterium to handle two types of heavy metal ions together.</P><P>The genetic circuit based on the <I>Vika-Vox</I> system enables stepwise treatment, owing to a switch from the expression of <I>Cup1</I> (copper accumulation) to <I>LIMT</I> (cadmium accumulation). We grow our yeasts and measure the concentration of heavy metal ions in the supernatant at equal intervals to test the efficiency of our system. </P><H4>Construction</H4><P>The <I>TEF</I> promoter, the <I>Cup1</I> gene, and the <I>Ura3</I> terminator are ligated together, integrated into <I>vox-ura3-vox</I> system by homologous recombination. <I>5-FOA</I> plate helps us to screen the correct cell. Similarly, the <I>LIMT</I> gene and the <I>Ura3</I> nutritional label are integrated into the synthetic chromosome <I>V</I>, too.</P><P>PCR is used to check if we successfully completed the molecular biology construction.</P><DIV class="zxx_zoom_demo_qqqqqqq" align="center"><DIV class="small_pic_demo_qqqqqqq" align="center"><P style="font-size:15px;text-align:center">Fig.5-1 The results of PCR of our <I>S.C-Cu</I>. <I>LIMT</I> gene (length of 319bp) 、<I>Cup1</I>(length of 186bp) and complete sequence（length of 3114bp）have been amplified. which indicated that we succeeded in the construction of genetic circuit.</P></DIV></DIV><DIV id="pic_seventy-five" style="display:none;"><P style="font-size:15px;text-align:center">Fig.5-1 The results of PCR of our <I>S.C-Cu</I>. <I>LIMT</I> gene (length of 319bp) 、<I>Cup1</I>(length of 186bp) and complete sequence（length of 3114bp）have been amplified. which indicated that we succeeded in the construction of genetic circuit.</P></DIV><P>Fig.5-1 the results of PCR. We use <I>2k plus Ⅱ</I> as the marker. On four parallel lanes of the gel (number 1,2,3,4), run were four set of DNA molecules of known size (319bp for number 1, the <I>LIMT</I>; 186bp for number 2 and 3, the <I>Cup1</I>; 3114bp for number 4,the whole sequence contained <I>Cup1</I>). From the DNA band of number 1, we could analyze that <I>vika</I> has been expressed to delete the <I>Cup1</I> and its terminor, so we can get the <I>LIMT</I>. From the DNA band of number 2, 3 and 4, we could delightedly prove that the fragments (<I>TEF</I> promoter, <I>Cup1</I> and <I>ura3</I> terminator) have successfully transformed to synthetic chromosome <I>V</I>. </P><H4>Accumulation</H4><P>Firstly, <I>S.C-Cu</I>, <I>S8</I> (screened by SCRaMbLE) and <I>BY4741</I> are cultured in YPD liquid media for 24 hours. Then add the 430 mg/L copper ions solution. Cells are cultured for another 45 hours (30℃). Atomic absorption spectroscopy is used to measure the concentration of copper ions in the supernatant every 5 hours. We depict the adsorption curve of copper ions.</P><P>In the same way, we culture Cd Yeast, <I>S1</I> (screened by SCRaMbLE) and <I>BY4741</I> in YPD liquid medium for 24 hours and then add 16 mg/L cadmium ions solution to the media for another 40 hours (30℃). the concentration of cadmium ions in the supernatant is measured every 5 hours.
</P><P>In terms of the respective ability to adsorb copper and cadmium, we compare genetically-engineered yeast, SCRaMbLE yeast and original one.

</P><P>As is illustrated in Fig.5-1 and Fig.5-2, engineered yeast significantly absorbs more ion than the control group without any improvement. Furthermore, SCRaMbLE yeast also shows excellent adsorption capacity, comparable to genetically-engineered one. Fig.5-1 reveals the adsorption of copper ion, which relatively faster than cadmium, showed in Fig.5-2.</P><DIV class="zxx_zoom_demo_qqqqqqq"><DIV class="small_pic_demo_qqqqqqq" style="float:left;"><P style="font-size:15px;text-align:center">Fig.5-2 The variations of copper(II) consumption with time for <I>S.C-Cu</I>、<I>S8</I> and <I>BY4741</I> at 430 mg/L copper(II) concentrations.
</P></DIV><DIV class="small_pic_demo_qqqqqqq" style="float:right;"><P style="font-size:15px;text-align:center">Fig.5-3 The variations of cadmium(II) consumption with time for <I>S.C-Cd</I>、<I>S1</I> and <I>BY4741</I> at 16 mg/L cadmium(II) concentrations</P></DIV></DIV><DIV id="pic_Seventy-six" style="display:none;"><P style="font-size:15px;text-align:center">Fig.5-2 The variations of copper(II) consumption with time for <I>S.C-Cu</I>、<I>S8</I> and <I>BY4741</I> at 430 mg/L copper(II) concentrations.
</P></DIV><DIV id="pic_Seventy-seven" style="display:none;"><P style="font-size:15px;text-align:center">Fig.5-3 The variations of cadmium(II) consumption with time for <I>S.C-Cd</I>、<I>S1</I> and <I>BY4741</I> at 16 mg/L cadmium(II) concentrations </P></DIV><P>Afterwards, we check if the <I>vika</I> enzyme could work well. The Cu yeast with a plasmid expressing <I>vika</I> enzyme is grew in the medium with <I> raffinose</I>, then transferred to heavy metal solution.
 </P><DIV class="zxx_zoom_demo_qqqqqqq" align="center"><DIV class="small_pic_demo_qqqqqqq" align="center"><P style="font-size:15px;text-align:center">Fig.5-4 <I>S.C-Cu</I> is cultivated in medium with <I> raffinose </I> including 320mg/L copper ions and 6mg/L Cadmium.<I>galactose</I> is added at 12 hours to turn on &quot;switch&quot;.</P></DIV></DIV><DIV id="pic_eighty" style="display:none;"><P style="font-size:15px;text-align:center">Fig.5-4 <I>S.C-Cu</I> is cultivated in medium with <I> raffinose </I> including 320mg/L copper ions and 6mg/L Cadmium.<I>galactose</I> is added at 12 hours to turn on &quot;switch&quot;.</P></DIV><P>Fig.5-4 clearly shows the change of the concentration of heavy metal ions in the supernatant. Firstly, the Cu yeast works smoothly. The concentration of copper ions declines over time while that of cadmium ions barely changes. 12 hours later, we add <I>galactose</I> to the solution. Situation changes. <I>Galactose</I> induces the enzyme, changing Cu yeast to Cd yeast. It leads to faster adsorption of cadmium but slower for copper.</P><H4> DISCUSSION &amp; FUTURE WORK</H4><P>In the experiment, we find the <I>LIMT</I> doesn’t appear as ideal as we thought. In the future work, we plan to read involved references in order to complete protein modification by rational design.</P><P>Moreover, it’s a pity that we have no time to combine our genetic circuit and SCRaMbLE yeast. In subsequent experiments, it deserves a try to transform fragments into SCRaMbLE yeast, checking if it will double the absorption capacity or even better.</P><P>In the nutshell, we will be dedicated to improve the absorption efficiency and better our genetic circuit.</P><DIV class="reference"><H4>Reference</H4><P>[1]Wang J, Chen C. Biosorption of heavy metals by Saccharomyces cerevisiae: A review[J]. Biotechnology Advances, 2006, 24(5):427.</P><P>[2]C. Baumann, A. Beil, S. Jurt, M. Niederwanger, O. Palacios, M. Capdevila, S. Atrian, R. Dallinger, O. Zerbe, Angew. Chem. Int. Ed. 2017, 56, 4617.</P><P>[3]Dönmez G, Aksu Z. The effect of copper(II) ions on the growth and bioaccumulation properties of some yeasts[J]. Process Biochemistry, 1999, 35(1–2):135-142.</P></DIV><DIV class="collapse-card__close_handler mt1 align-right mouse-pointer">
				go bottom and find a surprise </DIV></DIV></DIV></DIV></SECTION></DIV><FOOTER class="site-footer" style="text-align:center"><DIV class="footer-content"><DIV><H2 class="footerwords"><A href="https://twitter.com/iGEMTianjin2017" target="_blank">@iGEMTianjin2017</A> |  Tianjin University, NO.135 Yaguan Road, Jinnan District, Tianjin City, China</H2></DIV></DIV><DIV class="footer-mobile"><H2><A href="https://twitter.com/iGEMTianjin2017" target="_blank">Twitter:@iGEMTianjin2017</A>
			Tianjin University, NO.135 Yaguan Road, Jinnan District, Tianjin City, China</H2></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>