Team:Tianjin/Contribution
Parts
Back
Project
Description
Design
Demonstrate
Model
Notebook
Safety
Achievements
Parts
Parts
Contribution
Improve
Human Practice
Public Engagement
integrated HP
Collaboration
Silver
Team
Team
Attribution
/* OVERRIDE IGEM SETTINGS */
Contribution
Overview
The improvement of characterization was focused on both previous and current parts. Previous parts BBa_K2165004 made by iGEM16_Washington and BBa_K319036 made by iGEM10_uOttawa were characterized carefully and served for biosensor and selective label in our project. Besides, we have also characterized some core parts we used for verification, and they are BBa_K2407303 , BBa_K2407112 , BBa_K2407000 , and other functionally improved parts.
The characterization results were shown
BBa_K2165004
Based on the CUP1 promoter (BBa_K2165004) provided by iGEM16_Washington, we constructed a biosensor with yEmRFP. To characterize this promoter, strains of S. cerevisiae BY4742 containing the plasmid with an initial OD600 of 0.1 were grown for 24 hours in SC-URA medium at 30 degrees Celsius, and then were induced with copper sulfate. Samples in different copper concentration were tested with fluorescent microplate reader after 1, 6, 12, and 24 hours. This protocol was based on the experience used by Waterloo and Washington iGEM teams and amended by our team.
Fig.1 The fluorescence intensity of CUP1p-yEmRFP biosensor with different Cu concentration induced
Figure 1 showed the relationship between fluorescence intensity with induction time and Cu2+ concentration. With 0.1 mM CuSO4 induced, the fluorescence intensity is 2 times over a control with no induction at 1 hour. As time went on, the fluorescence intensity slightly reduced. Moreover, as the Cu2+ concentration increased, the fluorescence intensity decreased, and when the concentration reached 1 mM, the intensity was close to the control group. This might be due to the higher copper ion concentration influences the transcription, expression and even growth of yeast.
BBa_K319036
Redesign of CUP1 promoterDescription
URA3, a gene on chromosome V in Saccharomvces cerevisiae, is widely used in researches concerning yeasts as a “marker gene” (systematic name YEL021W. URA3) and used as a label for chromosomes orplasmids. URA3 encodes Orotidine 5'-phosphate decarboxylase—an enzyme that catalyzes one reaction in the synthesis of pyrimidine ribonucleotides. We obtained the URA3 gene from the PRS416 plasmid，which worked as a vector for our functional genes.
Principle of operation
1) Pyrimidine biosynthetic pathway of S. cerevisiae
Fig.2 The pyrimidine metabolism pathway in yeasts
In Saccharomyces cerevisiae, the biosynthesis of pyrimidines involves the de novo synthesis of UMP from glutamine. Carbamoyl phosphate, derived from glutamine, undergoes a condensation reaction with aspartic acid, resulting in the formation of N-carbamoyl aspartic acid. Both the formation and subsequent condensation of carbamoyl phosphate are performed by Ura2p. The pyrimidine ring of N-carbamoyl aspartic acid is closed by the elimination of water to form dihydroorotic acid (DHO), which is subsequently oxidized to form orotic acid (OA), and a ribose-phosphate group is then added to form orotidine 5′-monophosphate (OMP). The formation of OMP is performed by two isoenzymes, Ura5p and Ura10p. OMP is then decarboxylated to yield UMP, which may subsequently be processed to form other pyrimidines. Regulation of this pathway occurs at several levels. First, UTP down-regulates the enzymatic activity of Ura2p and transcription of the URA2gene. Second, under conditions of pyrimidine starvation, transcription of the URA1, URA3, URA4, and URA10 genes (the URA genes) is increased some three- to eightfold. This increase in transcription is dependent on a transcriptional activator, Ppr1p.
2) Usage in yeast research
The URA3 gene in the yeasts used in lab has already been deleted. Hence the loss of ODCase activity leads to a lack of cell growth unless uracil or uridine is added to the media. The presence of the URA3 gene in yeast restores ODCase activity, facilitating growth on media not supplemented with uracil or uridine, thereby allowing selection for yeast carrying the gene. In contrast, if 5-FOA (5-Fluoroorotic acid) is added to the media, the active ODCase will convert 5-FOA into the toxic compound (a suicide inhibitor) 5-fluorouracil causing cell death, which allows for selection against yeast carrying the gene.
Since URA3 allows for both positive and negative selection, it has been developed as a genetic marker for DNA transformations and other genetic techniques in bacteria and many fungal species. It is one of the most important genetic markers in yeast genetic modification. While URA3 is a powerful selectable marker it has a high background. This background is because cells that pick up mutations in URA3 may also grow on 5-FOA. Colonies should be verified by a second assay such as PCR to confirm the desired strain has been created.
Characterization
Positive selection
For characterizing the positive selecting function of URA3 part, we designed the following experiment:
1. Incubate 2 test tubes of yeasts-BY4741, and numbered as #1, #2. (cultured in YPD, 30℃,6 months)
2. After incubation, the yeasts in #1 tube are transformed by PRS416 plasmid, which contains URA3 gene as report gene.
3. Two groups of yeasts are spread on two Sc-URA plates, and hatch in the 30℃ incubator for 48 hr.
4. Examine the growth situation of yeasts on both plates.
Fig.3 The plasmid profile for PRS416
Fig.4 The result of our experiment to characterize the URA3 gene for positive selection
Showed in the photograph above, yeasts in group 1#, with PRS416 plasmid, has proliferated on the Sc-URA plate. Meanwhile, the other group without URA3 gene are not able to grow on the Sc-URA plate. The result indicates the URA3 gene is necessary for our yeasts (BY4741) to grow and proliferate on media not supplemented with uracil, thereby allowing selection for yeast carrying the gene.
negative selection
In order to characterize the negative selecting function of URA3 gene, after the transformation of #1 yeasts, we also spread the yeasts of group 1 on a 5-FOA plate, and spread group 2 yeasts on another 5-FOA plate in the meantime as control.
Fig.5 The result of our experiment to characterize the URA3 gene for negative selection
As presented above, yeasts #1(with URA3 gene) haven’t amplified in the 5-FOA plate, while yeasts #2 can proliferate on it, which testified the negative selecting function of URA3 gene.
BBa_K2407303
Theory
Fig.6 The carotenogenic pathway
crtYB, crtI, and crtE are originate from X. dendrorhous. Just as shown above, the carotenogenic pathway in X. dendrorhous consists of GGPP synthase encoded by crtE, the bifunctional enzyme phytoene synthase and lycopene cyclase encoded by crtYB, and phytoene desaturase encoded by crtI. S. cerevisiae contains a GGPP synthase, encoded by BTS1, which is able to convert FPP into GGPP. HMG1 encodes HMG-CoA reductase, which is the main regulatory point in the ergosterol biosynthetic pathway in many organisms. IPP, isopentenyl diphosphate; DMAP, dimethylallyl diphosphate; GPP, geranyl diphosphate. (René Verwaal, et al, 2007)
Like X. dendrorhous, S. cerevisiae is able to produce FPP and converts it into GGPP, the basic building block of carotenoids. Conversion of FPP into GGPP is catalyzed by GGPP synthase encoded by BTS1 in S. cerevisiae. Therefore, overexpression of only crtYB and crtI from X. dendrorhous in S. cerevisiae should generally be sufficient to transform S. cerevisiae into a β-carotene-producing organism. Additional overexpression of crtE from X. dendrorhous or BTS1 from S. cerevisiae will increase GGPP levels and thereby enhance β-carotene production.
Verification
We succeeded to integrate this part into the chromosome of Saccharomyces cerevisiae by homologous recombination. With the expression of β-carotene, cell is orange. The result is showed below:
Fig.7 The left side of the figure is the original Synthetic Saccharomyces cerevisiae, whose color is white. The orange strain on the right is the yeast what we obtained
BBa_K2407112
Metallothioneins (MTs) form a class of cysteine-rich polypeptides that are capable of binding a large number of metal ions.
LlMT is a kind of metallothioneins from the snail Littorina littorea and is capable of binding 9 Zn2+ or 9 Cd2+ ions. we obtained it from gene synthesis company after codon optimization.
Characterization
As an essential part in our heavy metal project, LIMT gene is expressed in our yeast after switching. To characterize the effect of LIMT gene , strains of S. cerevisiae SynV containing this gene is accumulated in YPD Medium without cadmuim ions.Then,cadmuim ions is added into the culture medium so that the concentration of cadmuim ions in the environment is 8mg/L. Atomic absorption spectroscopy is used to measure the concentration of cadmuim ions in the supernatant at equal intervals. The results are plotted as the adsorption curve in the figure.
Fig.8 the concentration curve of cadmuim ions.
The figure clearly shows the change of the concentration of cadmuim ions in the supernatant. The concentration of cadmuim ions declines over time and S.C-Cd(genetically-engineered yeast) is more capable of binding cadmium ions than control group(BY4741).
BBa_K2407000
Design
This is a shortened CUP1 promoter, improved from BBa_K2165004 (
BBa_K2165004
). This promoter is designed to be deleted with a few bases on the two ends. We retained 5 ACE1/CUP1 binding sites, 2 TATA boxes, and one initiation element in the promoter. The complex of ACE1 and copper ions will bind the promoter, which causes the activation of CUP1 promoter with TATA boxes' help. In this way, this promoter played its key role with less bases.
Fig.9 Structure of redesigned CUP1 promoter used in our project, based on BBa_K2165004
Characterization
Strains of S. cerevisiae BY4742 containing either BBa_K2165004-yEmRFP and BBa_K2407000-yEmRFP with an initial OD600 of 0.1 were grown for 24 hours in SC-URA medium at 30 degrees Celsius, and then were induced with 0.1 mM Cu2+. Samples were tested with fluorescent microplate reader after 1, 3, 6, 12, and 24 hours.
Fig.9 The characterization of both BBa_K2165004 and BBa_K2407000
Figure 9 shows the expression of yEmRFP with both the two promoters were very similar, so we can tell the deletion didn’t influence the core function of CUP1 promoter. Based on the new part, we carried out a further improvement.
Further Improvement
The technology of Error-Prone PCR was taken into our experiment to reduce the leakage expression and increase the response rage. Visist
BBa_K2407013
,
BBa_K2407014
,
BBa_K2407015
, and
BBa_K2407016
.
@iGEMTianjin2017 |  Tianjin University, NO.135 Yaguan Road, Jinnan District, Tianjin City, China
Twitter:@iGEMTianjin2017
Tianjin University, NO.135 Yaguan Road, Jinnan District, Tianjin City, China
