<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Contribution skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Contribution</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="wrapper"><NAV class="navbar navbar-inverse navbar-fixed-top" id="sidebar-wrapper" role="navigation"><UL class="nav sidebar-nav"><LI class="sidebar-brand"><A href="#">
                       Parts
                    </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Parts/"> Back</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Parts </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Improve">Improve</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Human Practice </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/HP/Gold_Integrated ">integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Collaborations">Collaboration</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/HP/Silver">Silver</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Team </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Team" target="_blank">Team</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Attributions">Attribution</A></LI></NAV></DIV><P>

/* OVERRIDE IGEM SETTINGS */
</P><DIV id="title"><H1>Contribution</H1></DIV><DIV id="tongyong"><DIV class="demo-2"><DIV class="container"><SECTION class="main"><H2>Overview</H2><P>The improvement of characterization was focused on both previous and current parts. Previous parts <A href="http://parts.igem.org/Part:BBa_K2165004">BBa_K2165004</A> made by iGEM16_Washington and <A href="http://parts.igem.org/Part:BBa_K319036">BBa_K319036</A> made by iGEM10_uOttawa were characterized carefully and served for biosensor and selective label in our project. Besides, we have also characterized some core parts we used for verification, and they are <A href="http://parts.igem.org/Part:BBa_K2407303">BBa_K2407303</A>, <A href="http://parts.igem.org/Part:BBa_K2407112">BBa_K2407112</A>, <A href="http://parts.igem.org/Part:BBa_K2407000">BBa_K2407000</A>, and other functionally improved parts. </P><P>The characterization results were shown
</P><H2>BBa_K2165004</H2><P>Based on the <I>CUP1 promoter</I> (BBa_K2165004) provided by iGEM16_Washington, we constructed a biosensor with yEmRFP. To characterize this promoter, strains of <I>S. cerevisiae BY4742</I> containing the plasmid with an initial OD<SUB>600</SUB> of 0.1 were grown for 24 hours in SC-URA medium at 30 degrees Celsius, and then were induced with copper sulfate. Samples in different copper concentration were tested with fluorescent microplate reader after 1, 6, 12, and 24 hours. This protocol was based on the experience used by Waterloo and Washington iGEM teams and amended by our team.</P><DIV class="zxx_zoom_demo" align="center"><DIV class="small_pic_demo" align="center"><P style="font-size:15px;text-align:center">Fig.1 The fluorescence intensity of CUP1p-yEmRFP biosensor with different Cu concentration induced</P></DIV></DIV><P>Figure 1 showed the relationship between fluorescence intensity with induction time and Cu<SUP>2+</SUP> concentration. With 0.1 mM CuSO<SUB>4</SUB> induced, the fluorescence intensity is 2 times over a control with no induction at 1 hour. As time went on, the fluorescence intensity slightly reduced. Moreover, as the Cu<SUP>2+</SUP> concentration increased, the fluorescence intensity decreased, and when the concentration reached 1 mM, the intensity was close to the control group. This might be due to the higher copper ion concentration influences the transcription, expression and even growth of yeast.</P><H2>BBa_K319036</H2><H3>Redesign of CUP1 promoterDescription</H3><P><B><I>URA3</I></B>, a gene on <I>chromosome V</I> in <I>Saccharomvces cerevisiae</I>, is widely used in researches concerning yeasts as a <I>“marker gene”</I> (systematic name <B><I>YEL021W. URA3</I></B>) and used as a label for chromosomes orplasmids. <B><I>URA3</I></B> encodes <I>Orotidine 5'-phosphate decarboxylase</I>—an enzyme that catalyzes one reaction in the synthesis of <I>pyrimidine ribonucleotides</I>. We obtained the <B><I>URA3</I></B> gene from the <I>PRS416</I> plasmid，which worked as a vector for our functional genes.</P><H3>Principle of operation</H3><H4>1) Pyrimidine biosynthetic pathway of<I> S. cerevisiae</I></H4><DIV id="fudong"><DIV id="img1"><P align="center">Fig.2 The <I>pyrimidine</I> metabolism pathway in yeasts</P></DIV><P>In Saccharomyces cerevisiae, the biosynthesis of pyrimidines involves the de novo synthesis of <I>UMP</I> from <I>glutamine</I>. <I>Carbamoyl phosphate</I>, derived from <I>glutamine</I>, undergoes a condensation reaction with <I>aspartic acid</I>, resulting in the formation of <I>N-carbamoyl aspartic acid</I>. Both the formation and subsequent condensation of <I>carbamoyl phosphate</I> are performed by <I>Ura2p.</I> The <I>pyrimidine</I> ring of <I>N-carbamoyl aspartic acid</I> is closed by the elimination of water to form <I>dihydroorotic acid (DHO)</I>, which is subsequently <I>oxidized</I> to form <I>orotic acid (OA)</I>, and a <I>ribose-phosphate</I> group is then added to form orotidine <I>5′-monophosphate (OMP)</I>. The formation of <I>OMP</I> is performed by two <I>isoenzymes</I>, <I>Ura5p</I> and <I>Ura10p</I>. <I>OMP</I> is then <I>decarboxylated</I> to yield <I>UMP</I>, which may subsequently be processed to form other <I>pyrimidines</I>. Regulation of this pathway occurs at several levels. First, <I>UTP</I> down-regulates the enzymatic activity of <I>Ura2p</I> and transcription of the <I>URA2</I>gene. Second, under conditions of <I>pyrimidine</I> starvation, transcription of the <I>URA1, <B>URA3</B>, URA4</I>, and <I>URA10</I> genes (the <I>URA</I> genes) is increased some three- to eightfold. This increase in transcription is dependent on a transcriptional activator, <I>Ppr1p</I>.<A>2) Usage in yeast research</A>
     The <B><I>URA3</I></B> gene in the yeasts used in lab has already been deleted. Hence the loss of <I>ODCase</I> activity leads to a lack of cell growth unless <I>uracil</I> or <I>uridine</I> is added to the media. The presence of the <B><I>URA3</I></B> gene in yeast restores <I>ODCase</I> activity, facilitating growth on media not supplemented with <I>uracil</I> or <I>uridine</I>, thereby allowing selection for yeast carrying the gene. In contrast, if <I>5-FOA (5-Fluoroorotic acid)</I> is added to the media, the active <I>ODCase</I> will convert <I>5-FOA</I> into the toxic compound (a suicide inhibitor) <I>5-fluorouracil</I> causing cell death, which allows for selection against yeast carrying the gene. </P><P>Since <B><I>URA3</I></B> allows for both positive and negative selection, it has been developed as a genetic marker for DNA transformations and other genetic techniques in bacteria and many fungal species. It is one of the most important genetic markers in yeast genetic modification. While <B><I>URA3</I></B> is a powerful selectable marker it has a high background. This background is because cells that pick up mutations in <B><I>URA3</I></B> may also grow on <I>5-FOA</I>. Colonies should be verified by a second assay such as PCR to confirm the desired strain has been created. </P></DIV><H3>Characterization</H3><H4>Positive selection</H4><P>For characterizing the positive selecting function of <B><I>URA3</I></B> part, we designed the following experiment: </P><P>1. Incubate 2 test tubes of <I>yeasts-BY4741</I>, and numbered as #1, #2. (cultured in YPD, 30℃,6 months) </P><P>2. After incubation, the yeasts in #1 tube are transformed by <I>PRS416</I> plasmid, which contains <B><I>URA3</I></B> gene as report gene. </P><P>3. Two groups of yeasts are spread on two <I>Sc-URA</I> plates, and hatch in the 30℃ incubator for 48 hr. </P><P>4. Examine the growth situation of yeasts on both plates. </P><DIV class="zxx_zoom_demo" align="center"><DIV class="small_pic_demo" align="center"><P style="font-size:15px;text-align:center">Fig.3 The plasmid profile for <I>PRS416</I></P></DIV></DIV><DIV class="zxx_zoom_demo" align="center"><DIV class="small_pic_demo" align="center"><P style="font-size:15px;text-align:center">Fig.4 The result of our experiment to characterize the <B><I>URA3</I></B> gene for positive selection</P></DIV></DIV><P>Showed in the photograph above, yeasts in group 1#, with <I>PRS416</I> plasmid, has proliferated on the <I>Sc-URA</I> plate. Meanwhile, the other group without <B><I>URA3</I></B> gene are not able to grow on the <I>Sc-URA</I> plate. The result indicates the <B><I>URA3</I></B> gene is necessary for our yeasts (<I>BY4741</I>) to grow and proliferate on media not supplemented with uracil, thereby allowing selection for yeast carrying the gene. </P><H4>negative selection</H4><P>In order to characterize the negative selecting function of <B><I>URA3</I></B> gene, after the transformation of #1 yeasts, we also spread the yeasts of group 1 on a <I>5-FOA</I> plate, and spread group 2 yeasts on another <I>5-FOA</I> plate in the meantime as control.</P><DIV class="zxx_zoom_demo" align="center"><DIV class="small_pic_demo" align="center"><P style="font-size:15px;text-align:center">Fig.5 The result of our experiment to characterize the <B><I>URA3</I></B> gene for negative selection</P></DIV></DIV><P>As presented above, yeasts #1(with <B><I>URA3</I></B> gene) haven’t amplified in the <I>5-FOA</I> plate, while yeasts #2 can proliferate on it, which testified the negative selecting function of <B><I>URA3</I></B> gene.</P><H2>BBa_K2407303</H2><H3>Theory</H3><DIV class="zxx_zoom_demo" align="center"><DIV class="small_pic_demo" align="center"><P style="font-size:15px;text-align:center">Fig.6 The carotenogenic pathway</P></DIV></DIV><P>crtYB, crtI, and crtE are originate from <I>X. dendrorhous</I>. Just as shown above, the carotenogenic pathway in <I>X. dendrorhous</I> consists of GGPP synthase encoded by crtE, the bifunctional enzyme phytoene synthase and lycopene cyclase encoded by crtYB, and phytoene desaturase encoded by crtI. <I>S. cerevisiae</I> contains a GGPP synthase, encoded by BTS1, which is able to convert FPP into GGPP. HMG1 encodes HMG-CoA reductase, which is the main regulatory point in the ergosterol biosynthetic pathway in many organisms. IPP, isopentenyl diphosphate; DMAP, dimethylallyl diphosphate; GPP, geranyl diphosphate. (René Verwaal, et al, 2007)</P><P>Like <I>X. dendrorhous</I>, <I>S. cerevisiae</I> is able to produce FPP and converts it into GGPP, the basic building block of carotenoids. Conversion of FPP into GGPP is catalyzed by GGPP synthase encoded by BTS1 in S. cerevisiae. Therefore, overexpression of only crtYB and crtI from <I>X. dendrorhous</I> in <I>S. cerevisiae</I> should generally be sufficient to transform <I>S. cerevisiae</I> into a β-carotene-producing organism. Additional overexpression of crtE from <I>X. dendrorhous</I> or BTS1 from <I>S. cerevisiae</I> will increase GGPP levels and thereby enhance β-carotene production. </P><H3>Verification</H3><P>We succeeded to integrate this part into the chromosome of <I>Saccharomyces cerevisiae</I> by homologous recombination. With the expression of β-carotene, cell is orange. The result is showed below:</P><DIV class="zxx_zoom_demo" align="center"><DIV class="small_pic_demo" align="center"><P style="font-size:15px;text-align:center">Fig.7 The left side of the figure is the original Synthetic Saccharomyces cerevisiae, whose color is white. The orange strain on the right is the yeast what we obtained</P></DIV></DIV><H2>BBa_K2407112</H2><P>Metallothioneins (MTs) form a class of cysteine-rich polypeptides that are capable of binding a large number of metal ions.</P><P>LlMT is a kind of metallothioneins from the snail <I>Littorina littorea</I> and is capable of binding 9 Zn<SUP>2+</SUP> or 9 Cd<SUP>2+</SUP> ions. we obtained it from gene synthesis company after codon optimization. </P><H3>Characterization</H3><P>As an essential part in our heavy metal project, LIMT gene is expressed in our yeast after switching. To characterize the effect of LIMT gene , strains of S. cerevisiae SynV containing this gene is accumulated in YPD Medium without cadmuim ions.Then,cadmuim ions is added into the culture medium so that the concentration of cadmuim ions in the environment is 8mg/L. Atomic absorption spectroscopy is used to measure the concentration of cadmuim ions in the supernatant at equal intervals. The results are plotted as the adsorption curve in the figure. </P><DIV class="zxx_zoom_demo" align="center"><DIV class="small_pic_demo" align="center"><P style="font-size:15px;text-align:center">Fig.8 the concentration curve of cadmuim ions.</P></DIV></DIV><P>The figure clearly shows the change of the concentration of cadmuim ions in the supernatant. The concentration of cadmuim ions declines over time and <I>S.C-Cd</I>(genetically-engineered yeast) is more capable of binding cadmium ions than control group(<I>BY4741</I>). </P><H2>BBa_K2407000</H2><H3>Design</H3><P>This is a shortened <I>CUP1</I> promoter, improved from BBa_K2165004 (<PARTINFO>BBa_K2165004</PARTINFO>). This promoter is designed to be deleted with a few bases on the two ends. We retained 5 <I>ACE1/CUP1</I> binding sites, 2 TATA boxes, and one initiation element in the promoter. The complex of <I>ACE1</I> and copper ions will bind the promoter, which causes the activation of <I>CUP1</I> promoter with TATA boxes' help. In this way, this promoter played its key role with less bases.</P><DIV class="zxx_zoom_demo" align="center"><DIV class="small_pic_demo" align="center"><P style="font-size:15px;text-align:center">Fig.9 Structure of redesigned CUP1 promoter used in our project, based on BBa_K2165004</P></DIV></DIV><H3>Characterization</H3><P>Strains of <I>S. cerevisiae BY4742</I> containing either BBa_K2165004-yEmRFP and BBa_K2407000-yEmRFP with an initial OD<SUB>600</SUB> of 0.1 were grown for 24 hours in SC-URA medium at 30 degrees Celsius, and then were induced with 0.1 mM Cu<SUP>2+</SUP>. Samples were tested with fluorescent microplate reader after 1, 3, 6, 12, and 24 hours.</P><DIV class="zxx_zoom_demo" align="center"><DIV class="small_pic_demo" align="center"><P style="font-size:15px;text-align:center">Fig.9 The characterization of both BBa_K2165004 and BBa_K2407000</P></DIV></DIV><P>Figure 9 shows the expression of yEmRFP with both the two promoters were very similar, so we can tell the deletion didn’t influence the core function of CUP1 promoter. Based on the new part, we carried out a further improvement.</P><H3>Further Improvement</H3><P>The technology of Error-Prone PCR was taken into our experiment to reduce the leakage expression and increase the response rage. Visist <PARTINFO>BBa_K2407013</PARTINFO>, <PARTINFO>BBa_K2407014</PARTINFO>, <PARTINFO>BBa_K2407015</PARTINFO>, and <PARTINFO>BBa_K2407016</PARTINFO>.



                  



 









	</P></SECTION></DIV></DIV></DIV><FOOTER class="site-footer" style="text-align:center"><DIV class="footer-content"><DIV><H2 class="footerwords"><A href="https://twitter.com/iGEMTianjin2017" target="_blank">@iGEMTianjin2017</A> |  Tianjin University, NO.135 Yaguan Road, Jinnan District, Tianjin City, China</H2></DIV></DIV><DIV class="footer-mobile"><H2><A href="https://twitter.com/iGEMTianjin2017" target="_blank">Twitter:@iGEMTianjin2017</A>
			Tianjin University, NO.135 Yaguan Road, Jinnan District, Tianjin City, China</H2></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>