<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tianjin_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tianjin/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="wrapper"><NAV class="navbar navbar-inverse navbar-fixed-top" id="sidebar-wrapper" role="navigation"><UL class="nav sidebar-nav"><LI class="sidebar-brand"><A href="#">
                       Project
                    </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Project"> Back</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Parts </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Improve">Improve</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Human Practice </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/HP/Gold_Integrated ">integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Collaborations">Collaboration</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/HP/Silver">Silver</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Team </A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Team" target="_blank">Team</A></LI><LI><A href="https://2017.igem.org/Team:Tianjin/Attributions">Attribution</A></LI></NAV></DIV><P>




/* OVERRIDE IGEM SETTINGS */
</P><DIV id="title"><H1 align="center">Design</H1></DIV><DIV id="normal"><DIV id="p1"><H2>   Background</H2><P>Human existence on earth is almost impossible without the heavy metals. Even though important to mankind, exposure to them during production, usage and their uncontrolled discharge into the environment has caused lots of hazards to man, other organisms and the environment itself. Heavy metals can enter human tissues and organs via inhalation, diet, and manual handling. As the process of urbanization and industrialization goes deeper and deeper, heavy metal pollution, a noticeable threaten to almost all the creatures, has become an essential problem to solve.</P><P>According to our human practice, the situation of heavy metal pollution (copper and cadmium ions) is marked on a world map, and the severity of heavy metal pollution has been increasing all over this map. Places with serious pollution include middle Asia, eastern Asia, southern Europe, and Latin America. In addition, not only fresh water source, but also soil and crops are seriously contaminated by heavy metals. On average, during three out of ten suppers we have, we absorb excess heavy metals over the standard concentration.</P><P>Considering the rigorous situation we face, our team decided to design an advanced system for typical toxic heavy metal disposal based on Saccharomyces cerevisiae.</P></DIV><SECTION class="nav"><DIV class="langtab"><DIV class="collapse-card"><DIV class="collapse-card__heading"><H3 class="collapse-card__title">			 Mating-type Switch and Mating Switcher
			</H3></DIV><DIV class="collapse-card__body"><H4>OVERVIEW </H4><P> We found there might be revolutionary usage about the mating type switch (<B>MTS</B>) of yeasts in our heavy metal deposition system. Whereas, in laboratory, the species of budding yeast we usually use are <I>BY4741</I> and <I>BY4742</I>, whose <I>HO</I> gene are knocked out. Therefore, we intended to use two groups of <I>MATa</I> yeasts to realize the mating switcher by a specially designed system. </P><P>One of these groups was required to achieve <B>MTS</B>. We decided to achieve <B>MTS</B> by introducing the <I>HO</I> gene into this group of yeasts— Saccharomyces cerevisiae (<I>BY4741</I>, in our lab, whose chromosome Ⅹ has been switched by synthetic chromosome Ⅹ. And it has been renamed as <I>SynⅩ</I> similarly hereinafter). To make the <B>MTS</B> controllable, it was necessary for us to adopt inducible promoters to initiate the expression of <I>HO</I> gene or create a pathway functioning as single gene regulator. Eventually, we landed on the <I>Gal1</I> promoter first, for its convenience and efficiency. As we read in R. Scott McIsaac’s work, their rapid, tunable, single-gene specificity control system of single gene in yeasts has given us much impression. Therefore, we decided to use this system as one of our pathway designs for the expression of <I>HO</I> gene. </P><H4>GETTING THE CHASSIS </H4><P>Aiming to achieve <B>MTS</B> for environmental use, it is essential to make sure that when the<I>MATa</I> locus has DSB(double strands break) cleaved by <I>HO</I>, our type-a (<I>MATa</I>) yeast can only become type-α (<I>MATα</I>). Therefore, we used a Ura-tag to replace the HMR(a) domain in chromosome Ⅲ. In this way, the HMR will no longer be the donor for the homologous recombination in the repairing process for <I>MATa</I> cleavage. After selection, by homologous recombination, we deleted the Ura-tag for further usage. We selected the target colonies(<I>SynⅩdUra</I>)  via 5Foa plates.  </P><DIV class="zxx_zoom_demo_long" align="center"><DIV class="small_pic_demo_long" align="center"><P style="font-size:15px;text-align:center">Fig. 1-1. Getting the chassis.</P></DIV></DIV><DIV id="pic_one" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 1-1. Getting the chassis.</P></DIV><H4>CONSTRUCTION of SYSTEMS</H4><H5> 1) <I>Gal1</I> System</H5><P>In this pathway, we chose <I>Gal1</I> as our promoter for the expression of <I>HO</I> gene, <I>CYC1</I> as the terminator, and PRS416(with Ura-tag) as our vector. As for segments ligation, we design the cutting sites for the Bsa1 enzyme in each part, hoping to achieve seamless ligation of these three parts. </P><P>We adopted the PCR method to amplify the <I>Gal1</I>-part and <I>CYC1</I>-part from a <I>Gal1-Vika</I> plasmid we had used in our former lab work. We designed specific primers for this procedure. After PCR, the <I>Gal1</I> has the cutting sites for SalⅠand BsaⅠon both ends, and <I>CYC1</I> has that for BsaⅠand BamhⅠon both ends. Meanwhile, the <I>HO</I> gene was obtained by gene synthesis, flanked by specific hangtags for BsaⅠto be cohesive with <I>Gal1</I> upstream and <I>CYC1</I> downstream. Thus, we have built our composite part (<I>GHC</I>).</P><P>After the ligation, we transformed the E.coli for the augment of the PRS416 plasmid with <I>GHC</I>. (<I>GHC-416</I>) And eventually, we transformed our <I>SynⅩ-dUra</I> for the <I>GHC-416</I> to get our target yeasts——<I>SynⅩ-dUra-416</I>. </P><H5> 2) <I>Modified. Gal1</I> system </H5><P>In this pathway, we introduced one kind of artificial transcription factor (ATF)—<I>Z4EV</I> into the regulation of <I>HO</I> gene expression. With <I>Z4EV</I> working with our <I>Modified. Gal1</I> promoter, we hoped to reach the on off-target dynamic control of <I>HO</I> gene expression. 
Our design for getting the <I>Modified. Gal1-HO-CYC1</I> parts (<B>MGHC</B>) is quite the same as that for <I>GHC</I> mentioned above, only that we acquired our Modified. <I>Gal1</I> part from the gene synthesis. </P><DIV class="zxx_zoom_demo_long" align="center"><DIV class="small_pic_demo_long" align="center"><P style="font-size:15px;text-align:center">Fig. 1-2. <I>Modified. Gal1</I> system</P></DIV></DIV><DIV id="pic_two" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 1-2. <I>Modified. Gal1</I> system</P></DIV><P>As for the expression of <I>Z4EV</I> gene, we intended to induce it into the <I>SynX</I> chromosome in <I>SynⅩ-dUra</I> by homologous recombination. With overlap PCR strategy, we put a homologous domain <I>CanA</I> (originally from Can gene in <I>chromosome X</I>) in the upstream of the promoter of <I>Z4EV</I>. Thus, we got our <I>CanA-TEF-Z4EV</I> part. Then we planned to use Tdh2t as terminator attached with Leu-CanB (another part of Can gene). </P><P>Next, for double using the Leu-tag, we introduce <I>Vika/vox</I> system. We intended to attach the <I>Vika</I> operator (<I>Tdh3p-Vika-Tdh2t</I>, TVT) following the <I>Z4EV</I> gene. We had two groups of yeasts, as mentioned above, one of them were aimed to accomplish <B>MTS</B> and become <I>MATα</I>, the other with functional genes remained as <I>MATa</I>. According to our design, the former will express <I>Vika</I> gene and generate <I>Vika</I> recombinase, and the other contains functional genes whose expressions are controlled by vox-Terminator-vox structure. Thus, the function genes' expression will be initiated during the cell fusion in yeast mating process with Terminator being deleted.  </P><DIV class="zxx_zoom_mid" align="center"><DIV class="small_pic_mid" align="center"><P style="font-size:15px;text-align:center">Fig. 1-3. <I>Z4EV</I> and <I>Vika</I>. </P></DIV></DIV><DIV id="pic_hao" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 1-3. <I>Z4EV</I> and <I>Vika</I>.
</P></DIV><H4>TEST of <B>MTS</B></H4><P>In this section, we only got to test the <I>Gal1</I> System due to time limits. And we figured that the result for <I>Gal1</I> System is adequately enough to represent the feasibility of our designed strategy for <B>MTS</B>.</P><P>The whole test process can be divided into three steps. </P><H5>Step 1:</H5><P>Activate the <I>Gal1</I> promoter. After that, the expression of <I>HO</I> gene in the <I>SynⅩ-dUra-416</I> can be initiated.</P><H5>Step 2:</H5><P> Cultivate two groups of yeasts together. (one is <I>SynⅩ-dUra-416</I>, the other is normal <I>BY4741</I><I>MATa</I>) If the <B>MTS </B> has been accomplished (<I>SynⅩ-dUra-416</I> can become <I>MATα</I>), the two groups of haploids can mate with each other and become diploids. </P><H5>Step 3:</H5><P> Test the results of mating by PCR method. We designed the primers for both <I>MATa</I> locus and <I>MATα</I> locus. The amplification of both <I>MATa</I> locus and <I>MATα</I> locus indicates that the yeasts have turned into diploids, the <B>MTS</B> has been achieved in other words. </P><H4>BACKGROUND</H4><H5> Mating switch of yeasts</H5><P>Saccharomyces cerevisiae is a single-celled organism with three types, called a, α, and a/α. In Saccharomyces cerevisiae, three cell types differ from each other in their DNA content at the <I>MAT</I> locus which specifies the cell types. In nature, the two haploid cell types (a and α) of budding yeast are able to interconvert in a reversible manner by DNA-rearrangement with a DSB at the <I>MAT</I> locus, and this process is called mating-type switch.</P><P>The DSB at <I>MAT</I> locus is caused by <I>HO</I> endonuclease (a kind of site-specific endonuclease expressed by <I>HO</I> gene). DSBs in chromosomes can be repaired either by homologous recombination (HR) or by nonhomologous end-joining (NHEJ). In S.C haploids, the DSB caused by <I>HO</I> endonuclease mostly repaired by HR with <I>HML(α)</I> and <I>HMR(a)</I> as donors. If the donor is <I>HML(α)</I>, the mating-type will become α, and vice versa. In this way, a haploid budding yeast is able to achieve mating-type switch. </P><DIV class="zxx_zoom_demo" align="center"><DIV class="small_pic_demo" align="center"><P style="font-size:15px;text-align:center">Fig. 1-4. Mating switch of yeasts</P></DIV></DIV><DIV id="pic_four" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 1-4. Mating switch of yeasts</P></DIV><H5>Artificial Transcription Factors——<I>Z4EV</I></H5><P>Thanks to R. Scott McIsaac and Benjamin L. Oakes’s former work, we learned that <I>Z4EV</I> is a kind of fusion protein with three domains –a DNA binding domain (DBD), an estrogen receptor (ER) and a VP16 activation domain. In the absence of β-estradiol, the ER interacts with Hsp90 chaperone complex and keep the ATF out of the nucleus. This AFT will provide a strong transcriptional activator that is dependent on the presence of β-estradiol. By using a synthetic 4-time-repeated zinc-finger DBD array from the mouse TF Zif268, residual off-target effects have been totally avoided. </P><P><I>Z4EV</I> (the <I>Z4EV</I> gene has been introduced into the <I>SynX chromosome</I> of this group of haploids) to strictly control the expression of <I>HO</I> gene. Unlike common β-estradiol-induced or galactose-induced promoters, this modified promoter is designed to be activated only when it is specifically bound with the activated <I>Z4EV</I> factor. </P><DIV class="zxx_zoom_demo" align="center"><DIV class="small_pic_demo" align="center"><P style="font-size:15px;text-align:center">Fig. 1-5. Artificial Transcription Factors——<I>Z4EV</I></P></DIV></DIV><DIV id="pic_fours" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 1-5. Artificial Transcription Factors——<I>Z4EV</I></P></DIV><DIV class="collapse-card__close_handler mt1 align-right mouse-pointer">
				UP </DIV></DIV></DIV></DIV><DIV class="langtab"><DIV class="collapse-card"><DIV class="collapse-card__heading"><H3 class="collapse-card__title">
				Characterization of Mating Switcher (RFP to CRT)
			</H3></DIV><DIV class="collapse-card__body"><H4>OVERVIEW</H4><P><I> Vika-vox</I> system is used in our project in order to switch the expression from <I>RFP</I> to <I>β-carotene</I>, as a characterization of our Mating Switcher. In this way, we can easily visualize the function of our switcher through its color, as well as measure its efficiency and error rate.

</P><P><I>Vika-vox</I> system mainly consists of <I>vox</I> sites and reporting parts. At first the expression of <I>yEmRFP</I> will be activating and the expression of <I>β-carotene</I> will be inhibited so that we can detect red fluorescence when recombinase <I>Vika</I> doesn’t exist in <I>Saccharomyces cerevisiae</I>. After the expression of recombinase <I>Vika</I>, with the deletion of <I>yEmRFP</I> and terminators flanked by <I>vox</I> locus, <I>β-carotene</I> expresses and the strains take on orange. This is the whole characterization process of Mating Switcher.

</P><H4>THEORETICAL BACKGROUND</H4><H5>1) <I>Vika-vox</I> System</H5><P>Genome editing is emerging as a powerful technology platform which paved the way for exploring the nature of life comprehensively and systematically. Site-specific DNA recombinases have been tamed as a powerful tool in genome editing, such as <I>Vika/vox</I> and <I>Cre/loxp</I>. Site-specifc DNA recombinase <I>Vika</I>, originally identifed in a gram-negative bacterium Vibro coralliilyticus, could functionally and specifcally deleted genomic DNA fragment via recognizing specifc DNA site <I>vox</I> in yeast <I>Saccharomyces cerevisiae</I> and other spices, including mammal cell and bacteria. 

</P><P> Recently a milestone was reached with achieved total synthesis of functional <I>chromosome Ⅴ</I> DNA in yeast, <I>Saccharomyces Cerevisiae</I>. This chromosome owns lots of <I>loxPsym</I> sites. In spite of DNA sequence in recognizing site of <I>loxP</I> and <I>vox</I> shares a high similarity, it was demonstrated that activities of the two recombination systems were strictly independent to each other in yeast. Furthermore, the <I>Vika/vox</I> system functioned properly even in yeast cell carrying the synthesized chromosome on which lots of <I>loxPsym</I> sites are encoded.

</P><H5>2) <I>Carotenogenic</I> Pathway</H5><P>Carotenoids are a class of pigments of commercial interest that have important biological functions. Certain carotenoids can be synthesized by biotechnology, by either homologous or heterologous production. One example is that <I>β-carotene</I> can be overexpressed in <I>S. cerevisiae</I>, by introduced carotenogenic genes, <I>crtE</I>, <I>crtI</I>, and <I>crtYB</I>, from the carotenoid-producing yeast <I>Xanthophyllomyces dendrorhous</I>. Like <I>X. dendrorhous</I>, <I>S. cerevisiae</I> is able to produce <I>FPP</I> and converts it into <I>GGPP</I>, the basic building block of carotenoids. Conversion of <I>FPP</I> into <I>GGPP</I> is catalyzed by <I>GGPP</I> synthase encoded by <I>BTS1</I> in <I>S. cerevisiae</I>. Therefore, overexpression of only <I>crtYB</I> and <I>crtI</I> from <I>X. dendrorhous</I> in <I>S. cerevisiae</I> should generally be sufficient to transform <I>S. cerevisiae</I> into a β-carotene-producing organism. Additional overexpression of <I>crtE</I> from <I>X. dendrorhous</I> will increase <I>GGPP</I> levels and thereby enhance <I>β-carotene</I> production.

</P><DIV class="zxx_zoom_demo" align="center"><DIV class="small_pic_demo" align="center"><P style="font-size:15px;text-align:center">Fig.2-1. The carotenogenic pathway in <I>X. dendrorhous</I></P></DIV></DIV><DIV id="pic_fiftyfive" style="display:none;"><P style="font-size:15px;text-align:center">Fig.2-1. The carotenogenic pathway in <I> X. dendrorhous</I></P></DIV><H4>EXPERIMENT DESIGN</H4><H5>1) Construction of <I>vox-ura3-terminator-vox</I> Structure </H5><P> We will use <I>SynⅤ</I> of <I>Saccharomyces cerevisiae</I> to load our device, which is a haploid with mating type of a. First of all, we use PCR to amplify basic parts including <I>TEF</I> promoter, <I>ura3</I> gene, <I>ura3-terminator</I> and <I>β-carotene</I> gene. Among them, <I>ura3</I> gene and <I>ura3-terminator</I> are flanked by <I>vox</I> locus. Then we use overlap PCR to combine these parts together. The next step is transform this composite part into <I>Saccharomyces cerevisiae</I>. We screen for the correctly transformed cell by using the <I>Sc-Ura</I> plate. For the purpose of verifying desired strain <I><B>PVUVC</B></I>, we use colony PCR to amplify the <I>TEF promoter-vox-ura3</I> structure and <I>ura3 terminator-vox-β-carotene</I> structure. The length of the strip will be observed by agarose gel electrophoresis.

</P><DIV class="zxx_zoom_mid" align="center"><DIV class="small_pic_mid" align="center"><P style="font-size:15px;text-align:center">Fig.2-2. The obtention of <I>vox-ura3-terminator-vox</I> structure.</P></DIV></DIV><DIV id="pic_fiftyseven" style="display:none;"><P style="font-size:15px;text-align:center">Fig.2-2. The obtention of <I>vox-ura3-terminators-vox</I> structure.</P></DIV><H5>2) Construction of <I>vox-RFP-terminators-vox</I> Structure</H5><P> This structure has a great similarity to the <I>vox-ura3-terminator-vox</I> structure above. Therefore, it is easy to construct because we only need to change the <I>ura3</I> gene to the <I>yEmRFP</I> gene. We use PCR to amplify five basic prats including <I>TEF promoter</I>, <I>yEmRFP gene</I>, <I>Adh1 terminator</I>, <I>ura3-terminator</I> and <I>β-carotene</I> gene. Among them, <I>yEmRFP</I> gene and <I>ura3-terminator</I> are flanked by <I>vox</I> locus. Then we use overlap PCR to combine these parts together. After that we use the lithium acetate conversion method to transfer this composite part into <I><B>TVUVC</B></I>. We screen for the correctly transformed cell by the 5-FOA plate. This part will integrate into <I>SynⅤ</I> by homologous recombination, and we will get another desired strain called <I><B>TVRVC</B></I>.

</P><DIV class="zxx_zoom_mid" align="center"><DIV class="small_pic_mid" align="center"><P style="font-size:15px;text-align:center">Fig.2-3. The obtention of <I>vox-RFP-terminators-vox</I> structure.</P></DIV></DIV><DIV id="pic_fiftysix" style="display:none;"><P style="font-size:15px;text-align:center">Fig.2-3. The obtention of <I>vox-RFP-terminators-vox</I> structure.</P></DIV><H5>3) Verification of <I>yEmRFP</I> in the PVRVC</H5><P> The verification of <I>yEmRFP</I> is carried out by using colony PCR to amplify the homologous arm-TEF promoter-vox-RFP gene and terminator-vox-crtE gene, which determines the existence of <I>vox</I> sites and <I>yEmRFP</I> gene. Then we can detect the red fluorescence.

</P><H5>4) Method of Red Fluorescence Assay</H5><P> We use a variant of the mCherry red fluorescent protein (RFP). The variant sequence was codon-optimized for the expression in <I>Saccharomyces cerevisiae</I> as yeast-enhanced mRFP (<I>yEmRFP</I>) and can combine fluorescence and a purple visible phenotype. Unfortunately, the <I>RFP</I> can’t be directly observed by bare eyes, we decided to use the Fluorescence spectrophotometer(F-2500) and use OD<SUB>600</SUB> to determination cell concentration. Meanwhile, we will observe using Fluorescence microscopy for fluorescent proteins expression. The red color can be observed if <I>yEmRFP</I> is expressed.

</P><H5>5) Construction of <I>Vika</I> System</H5><P>We use a common expression vector plasmid, <I>pRS416</I>, to load <I>Vika</I> part. First of all, we use corresponding restriction endonuclease <I>Sal1</I> and <I>Not1</I> to cut plasmid <I>pRS416</I> and plasmid <I>pRS415-Vika</I>, a gift from Y.J lab, and then use <I>T4</I> DNA ligase to link them together, we can obtain the complete device we want. Finally, we transform this device into <I>BY4742</I> by the lithium acetate conversion method, and we screen for the correctly transformed cell by using the <I>Sc-Ura</I> plate. <I>BY4742</I> is a haploid with mating type of <I>α</I>.

</P><P> We also use another plasmid <I>pRS413</I> with previous method.

</P><H5>6) The Induction of Mating</H5><P> The <I>Saccharomyces cerevisiae</I> called <I><B>PVRVC</B></I> is haploid with mating type of <I>a</I>. At first, we cultivate <I>pRS416-Vika</I> in <I>Sc-Ura</I> medium without glucose for three hours. To induce the expression of <I>Vika</I>, they will be cultured to saturation in <I>Sc</I> medium with <I>raffinose</I> and <I>galactose</I> for twelve hours. After that, recombinase <I>Vika</I> are induced to express and we make <I>α-pRS416-Vika</I> cell and a-TVRVC cell mate in <I>YPD</I> medium for eight hours. Two types cells are fused and form diploid yeasts, in which recombinase <I>Vika</I> bind with <I>vox</I> locus, and then delete <I>yEmRFP</I> gene and <I>Adh1</I> terminator flanked by <I>vox</I> sites. After the Mating Switcher, <I>β-carotene</I> expresses and the color of cell will transform from white to orange. At last we smear yeast solution on <I>Sc-Ura-Leu</I> plate to screen for the correctly mating cell. We can judge the existence of recombinase <I>Vika</I> by the color of the colony, and obtain the efficiency of mating.

</P><H5>7) Culture and Expression Condition of <I>Saccharomyces cerevisiae</I> in this Experiment </H5><P>Traditional <I>YPD</I> culture medium (22g/L <I>glucose</I>, 20g/L <I>peptone</I>, 10g/L <I>yeast extracts</I>) is used by us. <I>Sc-Ura</I> solid culture medium (22g/L <I>glucose</I>, 6.7g/L <I>yeast nitrogen base</I>, 1.224g/L <I>nutrient deficiency mixture</I> without <I>Ura</I>, <I>His</I>, <I>Leu</I> and <I>Trp</I>, 20g/L <I>agar powder</I>, 5mg/L <I>Trp</I>, <I>His</I> and <I>Leu</I>) is used to screen for correctly transformed cell. <I>5-FOA</I> solid culture medium (22g/L <I>glucose</I>, 6.7g/L <I>yeast nitrogen base</I>, 1.224g/L <I>dropout</I>, 20g/L <I>agar powder</I>, 1ml/L <I>His</I>, <I>Trip</I>, <I>Leu</I> and 2.5ml/L <I>Ura</I>) is used to screen for correctly transformed cell. Sc medium with <I>raffinose</I> and <I>galactose</I> culture medium (20g/L <I>raffinose</I>, 6.7g/L <I>yeast nitrogen base</I>, 1.224g/L <I>nutrient deficiency mixture</I> without <I>Ura</I>, <I>His</I>, <I>Leu</I> and <I>Trp</I>, 20g/L <I>agar powder</I>, 10x <I>galactose</I>, 5mg/L <I>Trp</I>, <I>His</I> and <I>Leu</I>) is used to induce to express<I> vika</I> recombinase. <I>Sc-Ura-Leu</I> solid culture medium(22g/L <I>glucose</I>, 6.7g/L <I>yeast nitrogen base</I>, 1.224g/L <I>nutrient deficiency mixture</I> without <I>Ura</I>, <I>His</I>, <I>Leu</I> and <I>Trp</I>, 20g/L <I>agar powder</I>, 5mg/L <I>Trp</I> and <I>His</I>) is used to screen for the correctly mating cell. All the cells are cultured in 5mL medium at 30℃ with shaking speed of 220rpm.

</P><H4>EXPECTED RESULTS</H4><P>In our design, Mating Switcher is a means of gene regulation. We can transform from one functional system to another system through this switch conveniently. To show the function of Mating Switcher more intuitively, we construct this<I> RFP</I> system to be a characterization.

</P><P> First, we can successfully construct the corresponding expression vector.

</P><P> Second, we expect to observe the expression of <I>yEmRFP</I> before the switch is on. We expect red fluorescent protein to be observed under a fluorescence microscope, and it is desirable that the fluorescence spectrophotometer can measure a high and stable fluorescence value.

</P><P> Third, <I>Vika-vox</I> system can play a role on function conversion. It is means that β-carotene gene can express after mating. We will observe orange colonies on nutrient label medium.

</P><DIV class="reference"><H4>REFERENCE</H4><P>[1] Lin, Qiuhui et al. “Robust Orthogonal Recombination System for Versatile Genomic Elements Rearrangement in Yeast <I>Saccharomyces Cerevisiae</I>.” <I>Scientific Reports</I> 5 (2015).

</P><P>[2] Karimova, Madina et al. “Vika/vox, a Novel Efficient and Specific Cre/loxP-like Site-Specific Recombination System.” <I>Nucleic Acids Research</I> 41.2 (2013).

</P><P> [3]Verwaal, René et al. “High-Level Production of Beta-Carotene in Saccharomyces Cerevisiae by Successive Transformation with Carotenogenic Genes from X.anthophyllomyces Dendrorhous .” <I>Applied and Environmental Microbiology</I> 73.13 (2007): 4342–4350. PMC. Web. 27 Oct. 2017.

</P></DIV><DIV class="collapse-card__close_handler mt1 align-right mouse-pointer">
				Show less </DIV></DIV></DIV></DIV><DIV class="langtab"><DIV class="collapse-card"><DIV class="collapse-card__heading"><H3 class="collapse-card__title">
				 Resistance to Heavy Metals (SCRaMbLE)
			</H3></DIV><DIV class="collapse-card__body"><H4>Overview</H4><P>Synthetic chromosome rearrangement and modification by <I>loxP</I>-mediated evolution(SCRaMble) generates combinatorial geomic diversity through rearrangements at designed recombinase sites. We applied SCRaMble to Saccharomyces cerevisiae（<I>synX</I>）to attain strains with better tolerance to high concentration of cadmium ion and cupric ion solution and compared the growing condition with the original strains to demonstrate the validity of SCRaMble.</P><H4>Theoretical Background</H4><H5>Sc2.0 Project</H5><P>Synthetic Yeast Genome Project (Sc2.0) is the world’s first synthetic eukaryotic genome project that aims to create a novel, rationalized version of the genome of the yeast species Saccharomyces cerevisiae. On March 10th, 7 articles related to Sc 2.0 were published on Science. As a member of Sc 2.0, YJ lab has completed two synthetic yeast chromosomes, and two articles are published on Science discussing challenging but exciting task of building synthetic chromosomes V, X .</P><H5>Cre-<I>loxpsym</I> System</H5><P>As a part of the Sc2.0 Project , yeast chromosomes are targets, named <I>loxPsym</I> sites. <I>loxPsym</I> sites are substrates for Cre-EBD , which is an inducible form of the appropriate site-specific recombinase. Results of the recombination events are dependent on the directionality of the loxp sites. Because of the asymmetry of the loxp sites, when a pair of loxp sites is present in the same DNA strand and they are in opposite orientations, the Cre recombinase will catalyze the inversion of the gene in between the loxp sites; when the loxp sites are in same orientation, this gene will be deleted. Unlike the native directional <I>loxP</I> site, which permits a single orientation for recombination, the synthetic <I>loxPsym</I> site’s symmetry ensures that any pair of sites can recombine in either orientation. Then, controlled expression of Cre-EBD leads to deletions and inversions with chromosome segments flanked by <I>loxPsym</I> sites<SUP>[1]</SUP>.This characteristic allows more possibilities of recombination on yeast chromosome which lives up to our expectations. </P><DIV class="zxx_zoom_demo_long_q" align="center"><DIV class="small_pic_demo_long_q" align="center"><P style="font-size:15px;text-align:center">Fig. 3.1 The left : when the <I>loxp</I> sites are in same orientation, this gene will be deleted. The middle : when the <I>loxp</I> sites are in opposite orientations, this gene will be inversed. The right : when the <I>loxp</I> sites are on different chromosomes, the gene will be exchanged<SUP>[2]</SUP>.</P></DIV></DIV><DIV id="pic_ones" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 3.1 The left : when the <I>loxp</I> sites are in same orientation, this gene will be deleted. The middle : when the <I>loxp</I> sites are in opposite orientations, this gene will be inversed. The right : when the <I>loxp</I> sites are on different chromosomes, the gene will be exchanged.</P></DIV><H4>Experiment Design</H4><H5>Construction of Cre-<I>loxpsym</I> System </H5><P>First, We use two common expression vector plasmid, PRS416 and PRS413(with different nutrition label), in Saccharomyces cerevisiae to load our device, which consists of heterologous gene part (PCLB2 promoter, Cre-EBD gene, CYC1 terminator) . Second, we transform the pSCW11-CRE/EBD plasmid into <I>synX</I> strain , respectively and get three strains with Cre-EBD, 079 and 160 with ura tag ,085 with his tag.</P><DIV class="zxx_zoom_demo_long_q" align="center"><DIV class="small_pic_demo_long_q" align="center"><P style="font-size:15px;text-align:center">Fig. 3.2 Construction process</P></DIV></DIV><DIV id="pic_twos" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 3.2 Construction process</P></DIV><H5>Screening of Strains with High Tolerance</H5><P>Under the cell culture environment with traces of estradiol(1μL of 5mM estradiol / 5mL media), three  strains are incubated at 30℃for 6 hours. After dilute 1000 times and wash 2 times with water to remove estradiol and spot on on SC plates with gradient concentrations of copper ion and cadmium ion.</P><P>Incubate plates for 3 days at 30℃ and observe the growth of strains.</P><P>Aiming at screening strains with tolerance to high concentration of cadmium ion and cupric ion solution, we prepare SC culture medium with 3mM, 4mM, 4.5mM, 5mM, 5.5mM, 6mM, 6.5mM, 7mM copper ion and SC culture medium with 0.01mM, 0.05mM, 0.1mM, 0.15mM, 0.2mM cadmium ion. Through detecting the number, size of colony of different strains on SC plates with gradient concentrations. We further narrow the range of concentrations and screen strains with optimal characteristics.</P><DIV class="zxx_zoom_demo_long_q" align="center"><DIV class="small_pic_demo_long_q" align="center"><P style="font-size:15px;text-align:center">Fig. 3.3 Screening of Strains (Take 160 as an example)</P></DIV></DIV><DIV id="pic_threes" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 3.3 Screening of Strains (Take 160 as an example)</P></DIV><H5>Verification of Cre-EBD Effect (Dilution Assay and Measurement of Cell Survival Rate)</H5><P>To demonstrate the verification of Cre-EBD effect, we diluted optimal strains to 10<SUP>-1</SUP>、10<SUP>-2</SUP>、10<SUP>-3</SUP>、10<SUP>-4</SUP>、10<SUP>-5</SUP> and made dilution assay on SC culture medium with copper ion and cadmium ion. Obviously if optimal strains grow better than other blanks, the answer is YES.</P><DIV class="zxx_zoom_demo_long_q" align="center"><DIV class="small_pic_demo_long_q" align="center"><P style="font-size:15px;text-align:center">Fig. 3.4 Dilution assay</P></DIV></DIV><DIV id="pic_fours" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 3.4 Dilution assay</P></DIV><P>What's more, we measure the survival rate of optimal strains with original strains <I>synX</I>. After cultivating yeasts in YPD overnight,  we take 200μL culture medium into ultrahigh concentration copper ion and cadmium ion solution. Then, plate after 10min, 30min, 1h and 2h. Through counting the number of colonies, we can obtain and compare the curve describing the cell survival rate of optimal strain and <I>synX</I>.</P><DIV class="zxx_zoom_demo_long_q" align="center"><DIV class="small_pic_demo_long_q" align="center"><P style="font-size:15px;text-align:center">Fig. 3.5 Measurement of cell survival rate</P></DIV></DIV><DIV id="pic_fours" style="display:none;"><P style="font-size:15px;text-align:center">Fig. 3.5 Measurement of cell survival rate</P></DIV><H4>REFERENCE</H4><P>[1] Shen, Y., Stracquadanio, G., Wang, Y., Yang, K., Mitchell, L. A., &amp; Xue, Y., et al. (2016). Scramble generates designed combinatorial stochastic diversity in synthetic chromosomes. <I>Genome Research</I>, 26(1), 36.
</P><P>[2]García-Otín, A. L., &amp; Guillou, F. (2006). Mammalian genome targeting using site-specific recombinases. <I>Frontiers in Bioscience A Journal &amp; Virtual Library</I>, 11(1), 1108.


			</P><DIV class="collapse-card__close_handler mt1 align-right mouse-pointer">
				Show less </DIV></DIV></DIV></DIV><DIV class="langtab"><DIV class="collapse-card"><DIV class="collapse-card__heading"><H3 class="collapse-card__title">
				 Detection of Copper ions (Parts’ Improvement)
			</H3></DIV><DIV class="collapse-card__body"><H4>Overview</H4><P>This idea was inspired by the naturally-occurring metal-ion-induced promoters. Ligating this kind of promoters with a reporter gene such as RFP is a common idea to visibly monitor the concentration of metal ions.</P><P>In <I>Saccharomyces cerevisiae S288C</I>, there naturally exists a copper-induced promoter – <I>CUP1p</I>. The <I>CUP1</I> promoter is activated by <I>ACE1</I>, a transcription factor which binds to copper ions. It is previously available as a standalone part as <A href="http://parts.igem.org/Part:BBa_K945002">BBa_K945002</A>, produced by Tec-Monterrery’s 2012 iGEM team, and team iGEM16_Washington modified this part with illegal restriction sites removed to make this part (<A href="http://parts.igem.org/Part:BBa_K2165004">BBa_K2165004</A>) easier to control and operate.

Our goal is to more perfectly characterize this promoter and improve it to create some handy parts.

</P><H4>Improvement</H4><P>The characterization of this promoter wasn’t perfectly completed, so we first construct a biosensor based on CUP1 promoter (<A href="http://parts.igem.org/Part:BBa_K2165004">BBa_K2165004</A>) and yEmRFP (<A href="http://parts.igem.org/Part:BBa_K2407012">BBa_K2407012</A>) to improve the characterization and function of this BioBrick.</P><P>To improve this promoter, we first want to shorten this promoter with all core sequence retained. Compared with other promoters used in the prokaryote, this promoter is much longer, which is not beneficial for BioBricks to easily assembly. Under this condition, we want to shorten its sequence as much as possible without functional changes.</P><P>In fact, this promoter is leaky in the absence of inducer. To make it more sensitive and lower the threshold of expression, we chose to transform the promoter with error-prone PCR. After finishing the library, the sensitivity and response range of engineered promoters will be characterized by the fluorescence intensity.</P><P>After the detection, we will use Mating Switcher to rapidly open following gene’s expression. We can either overexpress <I>CUP1</I> to enrich copper in the yeast cell or display the Metallothionein on the surface of budding yeast.</P><DIV class="collapse-card__close_handler mt1 align-right mouse-pointer">
				Show less </DIV></DIV></DIV></DIV><DIV class="langtab"><DIV class="collapse-card"><DIV class="collapse-card__heading"><H3 class="collapse-card__title">
				 Separation of Different Ions (Copper and Cadmium ions) 
			</H3></DIV><DIV class="collapse-card__body"><H4>OVERVIEW </H4><P>The genetic circuit based on the <I>Vika-Vox</I> system in genetically-engineered yeast enables stepwise treatment of heavy metal ions by a switch from the expression of <I>Cup1</I> metallothionein (copper accumulation) to the expression of <I>LIMT</I> metallothionein (cadmium accumulation). We measure the concentration of heavy metal ions in the supernatant at equal intervals to test the efficiency of our system. </P><H4>BACKGROUND</H4><P><I>Cup1</I>, a kind of functional metallothionein, can combine with copper ions in yeast. 
</P><P>As a metallothionein in Littorina littorea, <I>LIMT</I> is capable of binding a large number of cadmium ions. </P><H4>EXPERIMENT DESIGN</H4><H5>1) construction of <I>S.C-Cu(Cd)</I></H5><P>The <I>TEF</I> promoter, the <I>Cup1</I> gene, and the <I>Ura3</I> terminator are ligated together by overlap PCR, and then the sequence is integrated into <I>vox-ura3-vox</I> system by yeast's homologous recombination. <I>5-FOA</I> plate helps us to screen the correct cell after transferring. Similarly, the <I>LIMT</I> gene and the <I>Ura3</I> nutritional label are integrated into the same chromosome. Screened by <I>SC-ura3</I>, we complete genetic circuit as figure 5-1,5-2.showing below.</P><DIV class="zxx_zoom_mid" align="center"><DIV class="small_pic_mid" align="center"><P style="font-size:15px;text-align:center">Fig.5-1 the construction of expressing copper metallothionein.</P></DIV></DIV><DIV id="pic_seventy-one" style="display:none;"><P style="font-size:15px;text-align:center">Fig.5-1 the construction of expressing copper metallothionein.</P></DIV><DIV class="zxx_zoom_mid" align="center"><DIV class="small_pic_mid" align="center"><P style="font-size:15px;text-align:center">Fig.5-2 the construction of expressing cadmium metallothionein.</P></DIV></DIV><DIV id="pic_seventy-two" style="display:none;"><P style="font-size:15px;text-align:center">Fig.5-2 the construction of expressing cadmium metallothionein.</P></DIV><H5>2) Switch </H5><P>A vector expressing the <I>Vika</I> recombinase is also transformed into the yeast strain. The expression of <I>Vika</I> enzymes can be induced by <I>galactose</I>. Then the <I>Cup1</I> gene along with its terminator will be deleted by <I>Vika</I> recombinase hence the initiation of the expression of <I>LIMT</I> metallothionein. <I>S.C-Cu</I> is turned to <I>S.C-Cd</I>.</P><H5>3) Copper Ions Accumulation</H5><P> After completing the construction of the genetic circuit, we start to test the efficiency of copper accumulation. Firstly, <I>S.C-Cu</I>, <I>S8</I> (screened by SCRaMbLE) and <I>BY4741</I> are all cultured in YPD liquid media for 24 hours for accumulation. after that the 430 mg/L copper ions solution is added to the media. The yeast cells are cultured in this solution for another 45 hours (30℃). Atomic absorption spectroscopy is used to measure the concentration of copper ions in the supernatant every 5 hours. According to the concentration change of copper ions at equal time intervals. We depict the adsorption curve of copper ions. </P><H5>4) Cadmium Ions Accumulation </H5><P>In order to reveal the yeast's influences on cadmium ions, we culture Cd Yeast, <I>S1</I> and <I>BY4741</I> in YPD liquid medium for 24 hours. After that the 16 mg/L cadmium ions solution is added to the media. The yeast cells are cultured in this solution for another 40 hours (30℃). The concentration of cadmium ions in the supernatant is measured every 5 hours.</P><H5>5) Separation of Copper and Cadmium Ions</H5><P>In the end, the accumulated <I>S.C-Cu</I> yeast strain is cultured in <I>Sc</I> medium with <I>raffinose</I> including copper ions and cadmium ions for 32 hours. <I>Galactose</I> is added after 12 hours of culture. Switch will be activated. The separation of Copper and Cadmium ions will be observed in heavy metal curve. </P><H4>REFERENCE</H4><P>[1]Wang J, Chen C. Biosorption of heavy metals by Saccharomyces cerevisiae: A review[J]. <I>Biotechnology Advances</I>, 2006, 24(5):427.</P><P>[2]C. Baumann, A. Beil, S. Jurt, M. Niederwanger, O. Palacios, M. Capdevila, S. Atrian, R. Dallinger, O. Zerbe, Angew. <I>Chem. Int. Ed</I>. 2017, 56, 4617.</P><P>[3]Dönmez G, Aksu Z. The effect of copper(II) ions on the growth and bioaccumulation properties of some yeasts[J]. <I>Process Biochemistry</I>, 1999, 35(1–2):135-142.</P><DIV class="collapse-card__close_handler mt1 align-right mouse-pointer">
				go bottom and find a surprise </DIV></DIV></DIV></DIV></SECTION></DIV><FOOTER class="site-footer" style="text-align:center"><DIV class="footer-content"><DIV><H2 class="footerwords"><A href="https://twitter.com/iGEMTianjin2017" target="_blank">@iGEMTianjin2017</A> |  Tianjin University, NO.135 Yaguan Road, Jinnan District, Tianjin City, China</H2></DIV></DIV><DIV class="footer-mobile"><H2><A href="https://twitter.com/iGEMTianjin2017" target="_blank">Twitter:@iGEMTianjin2017</A>
			Tianjin University, NO.135 Yaguan Road, Jinnan District, Tianjin City, China</H2></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>