"
Page
Discussion
View source
History
teams
Log in
 
Team:HKUST-Hong Kong/overview1a.html
From 2011.igem.org
1. ASM
1.2. Method of assembly
Using this mutualistic relation, the desired pDummy can be maintained once the host bacterium develops an addiction it, and pToolkit can be lost in bacteria propagation if the expression of G can be shut off manually. Eventually, the bacteria not obtain any new antibiotic resistance genes but keep pDummy.
Development of addiction – use of the lambda RED recombination system
To develop the addiction in the host bacterium to pDummy, an essential gene for survival is to be deleted from the bacteria genome, provided that the bacteria can survive on extra-genomic copies after the deletion.
The deletion here is mediated through the lambda RED recombination system.
The lambda RED recombination cassette is located on the pToolkit (and hence the name of the plasmid). Once the recombination is successful, it can be eliminated from the host bacterium together with the antibiotic resistance gene.
Therefore, once the co-transformation of pDummy and pToolkit is successful, linear dsDNAs having a reporter gene flanked by homologous sequences to the essential gene can be introduced into the bacteria. When the recombination is kicked started, the essential gene will be swapped out and the reporter gene will be incorporated into the bacteria genome.
Since the linear dsDNAs do not have origin of replications, they are not inherited in daughters unless they are swapped into the genomes. Thus, any observable signals from the reporter would allow identification of successful recombination. Identified colonies can then be further treated to induce loss of pToolkit, which afterwards would be the completed strain of EX.
Complementation between reporter genes – manifesting completion of EX engineering
To ensure that the final strain of EX has: 1. successfully had its essential gene deleted from genome, 2. maintained the pDummy, a complementation reporter system between the pDummy and swapped gene is preferred over a single reporter at the swapped site.
Different methods can achieve the above aim:
i. Alpha complementation can be used in E. Coli strains where the lacZ gene is completely removed. The larger fragment ω can be swapped for the essential gene while the smaller α fragment can stay on pDummy. In a X-gal rich medium, blue colonies suggest the desired engineered strains.
ii. Complementation between split fluorescent proteins (sFP). 2010 iGEM Slovenia team has demonstrated the principle that N-terminal and C-terminal fragments of sFPS are able to complement in vivo and two sets of sfFPS are able to undergo Förster resonance energy transfer (FRET). This idea is adopted but an alternative set of candidate, split superfolder GFPs (sfGFP), was developed.
Summary of construction flow:
1. Assembly pDummy and pToolkit
2. Co-transform both plasmid into E Coli and maintain stable strains
3. Introduce linear dsDNAs and induce recombination
4. Isolate recombinants
5. Induce loss of pToolkit
1.3. Component details
Temperature-sensitive origin of replication_oriR101 & repA101-ts (BBa_K524000) oriR101 & repA101-ts is a set of low copy origin of replication derived from the pSC101 origin of replication. The repA101-ts gene codes for a heat-labile protein that is required in trans for the initiation of replication at oriR101. In our construct, our characterization has shown that plasmids with this origin of replication can only be maintained below than 300C, and partial maintenance of plasmid was observed within temperature range from 290C to 330C. This part was cloned out from pKD46 plasmid (courtesy of The Coli Genetic Stock Center), and standardized by a nucleotide mutation.
Continue
Overview & Background
1 ASM
1.2. Method of assembly
1.3 Component details
Next on 1.3 Component details
Retrieved from " http://2011.igem.org/Team:HKUST-Hong_Kong/overview1a.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
