<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HKUST-Hong_Kong_future_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:HKUST-Hong_Kong/future.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HKUST-Hong_Kong/future.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HKUST-Hong_Kong/future.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HKUST-Hong_Kong/future.html&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HKUST-Hong_Kong/future.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HKUST-Hong Kong/future.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><H3><B>Future Plans</B></H3><FONT color="black"><H4 align="left">1. Constructing EX – the bacterial strain that allows selection without use of antibiotics</H4><P align="justify" style="margin: 20px 20px 20px 20px">


To study the population dynamics and behavior of a certain antibiotics sensitive strain of <I>E. coli</I> in a medium of antibiotic, our <I>E. Trojan</I> that is introduced into the culture medium must not process a wide spectrum of antibiotic resistance that impose a selective advantage. At the same time, <I>E. Trojan</I> needs to be transformed with the T4MO gene to carry out its job of signal disruption. 
Summarizing the above criteria, a solution where the bacteria can be transform with the gene of interest while remaining sensitive to antibiotics is needed. Therefore the requisite is to construct a new bacterial strain that can perform plasmid selection without the use of antibiotics, and contains as little antibiotics resistance gene as possible.
<A href="#top">[Top]</A></P><H4 align="left">2. How to select against EX without the vector plasmid? Our alternative selection method</H4><P align="justify" style="margin: 20px 20px 20px 20px">


Our EX will have one of its essential genes (genes that are required for viability) removed from its genome, and relocated onto an engineered plasmid pDummy. As illustrated, in order to survive, EX must rely on those extra-chromosomal copies of the essential gene; therefore, EX is addicted to pDummy. By having direct control over the replication of pDummy, we dictate the life and death of EX (and hence the name pDummy).


Here, we introduce a heat-sensitive origin of replication as the only origin of pDummy. When we intend to switch off the replication of pDummy, we can incubate EX at above 30°C. This origin would then cease to function, and pDummy cannot be maintained. Deprived of the essential gene and the corresponding vital product, EX cannot propagate, unless, it receives an alternative but heat insensitive analog of pDummy. 

This analog, named pCarrier, is the essentially our vector in cloning. Under an unfavorably high temperature, only those EX that are transformed with the insert-bearing pCarrier will be able to propagate and survive, while the others cannot undergo division and are virtually eliminated from the population. Eventually, the pDummy can be considered to be &quot;shuffled out&quot; by pCarrier. Our designed selection system, in short, bases itself on plasmid shuffling, and thus eliminates involvement of antibiotic resistance genes in any of the cloning steps.<A href="#top">[Top]</A></P><H4 align="left">3. Stepping in the heart of construction - methods of assembly</H4><P align="justify" style="margin: 20px 20px 20px 20px"><B>3.1 Construction and maintenance of an antibiotic-resistance-gene-free plasmid through antibiotic selection – the unavoidable evil two plasmid system</B>

Our ultimate goal is to construct our EX without conferring it any new antibiotic resistance. For this reason no resistance gene should be found in our dummy plasmid pDummy. 

Yet, such a plasmid would not be maintained by itself unless the host bacterium develops an addiction to it (i.e. losses the essential gene in its genome and depends on extra-chromosomal copies on pDummy), and inconveniently, the addiction can only be achieved after the introduction of the plasmid.

The solution is to develop a mutualistic relation between two plasmids and we planned to exploit positively regulated origin of replications. 

Well studied examples are those in pSC101 and R6K origins of replication, where the origins of replication (OR) appear together with a constitutive gene (G). Initiation of replication happens if and only if the trans element of the gene is provided.</P><P>
Let’s consider the following scenario: 
i.   G is placed on pDummy with no selection marker but with a normal replication origin
ii.  OR is the sole origin of replication of another plasmid (here we introduce a new plasmid pToolkit) with a selection marker
iii. pDummy and pToolkit are co-transformed to a bacterium which is under selection stress</P><P>
We would obtain three possible outcomes:<B>1. only pDummy is uptaken</B>
- since pDummy has no selection marker, the host bacteria die under selection pressure and cannot propagate<B>2. only pToolkit is uptaken</B>
- the host bacterium that uptakes pToolkit survives. Yet during propagation, pToolkit is not replicated because proteins of G are absent. Therefore daughter cells of the host bacterium will not receive copies of the pToolkit and die under selection pressure.<B>3. both pDummy and pToolkit are uptaken</B>
- in presence of pDummy, pToolkit is maintained and confers the host bacterium with stress resistance. Daughters that receive copies of both plasmids will survive and eventually develop into a colony.</P><P>
Using this mutualistic relation, the desired pDummy can be maintained once the host bacterium develops an addiction it, and pToolkit can be lost in bacteria propagation if the expression of G can be shut off manually. Eventually, the bacteria not obtain any new antibiotic resistance genes but keep pDummy.

</P><P align="justify" style="margin: 20px 20px 20px 20px"><B>3.2 Development of addiction – use of the lambda RED recombination system</B>

To develop the addiction in the host bacterium to pDummy, an essential gene for survival is to be deleted from the bacteria genome, provided that the bacteria can survive on extra-chromosomal copies after the deletion.

The deletion here is mediated through the lambda RED recombination system

[Nat is still writing...z.z]

The lambda RED recombination cassette is located on the pToolkit (and hence the name of the plasmid). Once the recombination is successful, it can be eliminated from the host bacterium together with the antibiotic resistance gene. 

Therefore, once the co-transformation of pDummy and pToolkit is successful, linear dsDNAs having a reporter gene flanked by homologous sequences to the essential gene can be introduced into the bacteria. 

When the recombination is kicked started, the essential gene will be swapped out and the reporter gene will be incorporated into the bacteria genome.

Since the linear dsDNAs do not have origin of replications, they are not inherited in daughters unless they are swapped into the genomes. Thus, any observable signals from the reporter would allow identification of successful recombination. Identified colonies can then be further treated to induce loss of pToolkit, which afterwards would be the completed strain of EX.<B>3.3 Complementation between reporter genes – manifesting completion of EX engineering</B>

To ensure that the final strain of EX has: 1. successfully had its essential gene deleted from genome, 2. maintained the pDummy, a complementation reporter system between the pDummy and swapped gene is preferred over a single reporter at the swapped site.

Different methods can achieve the above aim:
i. Alpha complementation can be used in <I>E. coli</I> strains where the lacZ gene is completely removed. The larger fragment ω can be swapped for the essential gene while the smaller α fragment can stay on pDummy. In a X-gal rich medium, blue colonies suggest the desired engineered strains.

ii. Complementation between split fluorescent proteins (sFP). 2010 iGEM Slovenia team has demonstrated the principle that N-terminal and C-terminal fragments of sFPS are able to complement in vivo and two sets of sfFPS are able to undergo Forster resonance energy transfer (FRET). This idea is adopted but an alternative set of candidate, split superfolder GFPs (sfGFP), was developed.</P><P align="justify" style="margin: 20px 20px 20px 20px"><B>3.4 Summary of construction flow:</B>
1. Assembly pDummy and pToolkit
2. Co-transform both plasmid into <I>E. coli</I> and maintain stable strains
3. Introduce linear dsDNAs and induce recombination
4. Isolate recombinants
5. Induce loss of pToolkit

<A href="#top">[Top]</A></P><H4 align="left">4. Details of the components – a closer look to the molecular basis of assembly</H4><P align="justify" style="margin: 20px 20px 20px 20px"><B>4.1 Temperature-sensitive origin of replication_oriR101 &amp; repA101-ts (BBa_K524000)</B>

oriR101 &amp; repA101-ts is a set of low copy origin of replication derived from the pSC101 origin of replication. The repA101-ts gene codes for a heat-labile protein that is required in trans for the initiation of replication at oriR101. In our construct, our characterization has shown that plasmids with this origin of replication can only be maintained below than 300°C, and partial maintenance of plasmid was observed within temperature range from 290°C to 330°C. This part was cloned out from pKD46 plasmid (courtesy of The Coli Genetic Stock Center), and standardized by a nucleotide mutation.</P><P align="justify" style="margin: 20px 20px 20px 20px"><B>4.2 split superfolder green fluroscent protein_split sfGFP
sfGFP1-10 (BBa_K524001) [Twins: BBa_K524006]
sfGFP11 (BBa_K524002) [Twins: BBa_K524007]</B>

The sfGFPs are mutated variants of GFPs that has improved folding kinetics and resistance to chemical denaturants. Split sfGFPs at amino acid residues 214 and 215 have been reported to undergo spontaneous complementation to give green fluorescence. The two split constructs were produced from an existing biobrick – pBAD driven sfGFP BBa_I746908. CDS of sfGFP amino acid residues 1-214 were copied out for sfGFP1-10 using PCR and stop codon was added to the end. The sfGFP11 was produced in a similar fashion, with a start codon added to the front of the CDS of amino acid residues 215 to 238.

</P><P align="justify" style="margin: 20px 20px 20px 20px"><B>4.3 Essential gene <I>nadE</I> (BBa_K524003)</B><I>nadE</I> is a vital gene in <I>E. coli</I>. It codes for NAD+ synthetase. In principle, removal of such gene from the genome would cause addiction of bacteria to a plasmid that has a copy of the gene. CyaR (a sRNA) regulates the expression of <I>nadE</I> post-transcriptionally. This feature is retained in our construct. Transcription of <I>nadE</I> operon requires the sigma-70 factor and is terminated by downstream extragenic sites. The <I>nadE</I> gene was cloned out from the genome of strain BL21(DE3), and was completed the <I>nadE</I> by having B0015 terminator assembled to its end.



</P><P align="justify" style="margin: 20px 20px 20px 20px"><B>4.4 Replication initiator pi protein encoded by pir gene (BBa_K524004) and ori-gamma from R6K plasmid</B> 

ori-gamma is one of three replication origins (the other two being alpha and beta) of the R6K origin. Initiation of replication at ori-gamma requires the pi protein in trans, which is encoded by the pir gene. Yet doubling the concentration of pi protein would effectively shut down the replication as well. Expression of pi protein is autogenously regulated. The pir construct was cloned out from the genome of strain BW25141 (courtesy of The Coli Genetic Stock Center) and standardized. The ori-gamma was adopted from the R6K origin of replication BBa_J61001.



</P><P align="justify" style="margin: 20px 20px 20px 20px"><B>4.5 iGEM 2010 Slovenia Split/FRET constructs</B> 

The split CFP and YFP from the biobricks of Slovenia team last year were used as alternative reporters. The idea is to put one of the terminal fragments of a split fluorescence protein into the pDummy, and swap out the essential <I>nadE</I> gene from the genome with the other terminal fragment. Driven by pLac R0010, both fragments should express simultaneously when induced by IPTG and fluorescence signal would be observed as an indicator of successful recombination. <A href="#top">[Top]</A></P></FONT><H2>Future Plans</H2><A href="#constructing">1. Constructing EX</A><A href="#method">2. How to Select? </A><A href="#assembly">3. Methods of Assembly </A><A href="#component">4. Component Details</A><P align="center"><B><FONT face="Verdana, Arial, Helvetica, sans-serif" size="2" color="#FFE1E1"><A href="https://2011.igem.org/Team:HKUST-Hong_Kong" target="_top"><FONT face="Verdana, Arial, Helvetica, sans-serif" size="4" color="#FFE1E1" font="" color="white"><SPAN style="font-weight:700">Home</SPAN></FONT></A></FONT></B></P><P align="center" valign="baseline"><B><FONT face="Verdana, Arial, Helvetica, sans-serif" size="3" color="green">

Our Project</FONT></B></P><P align="left"><FONT size="1" face="Verdana, Arial, Helvetica, sans-serif" color="green"><A href="overview.html" target="_top">Overview</A> | 
<A href="data.html" target="_top">Data Page</A><SPAN style="line-height:1; font-weight:600">Experiments and Results</SPAN><A href="asm.html" target="_top">Strain Construction</A> | 
<A href="mic.html" target="_top">Culture Tests</A> | 
<A href="modeling.html" target="_top">Modeling</A><SPAN style="line-height:1; font-weight:600">Miscellaneous</SPAN><A href="future.html" target="_top">Future Plans</A> | 
<A href="notebook.html" target="_top">Notebook</A></FONT></P><P align="left"><B><FONT face="Verdana, Arial, Helvetica, sans-serif" size="3" color="#FFF4D0"> 
           
iGEM Resources</FONT></B></P><P align="left"><FONT face="Verdana, Arial, Helvetica, sans-serif" size="1" color="#FFFFFF"><A href="acknowledgement.html" target="_top">Acknowledgements</A><SPAN style="line-height:0.7; font-weight:600">The Team</SPAN><A href="team.html" target="_top">iGEM Member List</A> | 
<A href="contribution.html" target="_top">Contributions</A><SPAN style="line-height:0.7; font-weight:600">Achievements</SPAN><A href="medal.html" target="_top">Medal Requirements</A> | 
<A href="biosafety.html" target="_top">BioSafety</A><SPAN style="line-height:0.7; font-weight:600">Biobricks</SPAN><A href="characterization.html" target="_top">Master List &amp; Characterization Data</A></FONT></P><P align="left" valign="baseline"><B><FONT face="Verdana, Arial, Helvetica, sans-serif" size="3" color="#FFE0E0">
Human Practice</FONT></B></P><P align="left"><FONT face="Verdana, Arial, Helvetica, sans-serif" size="1" color="#FFFFFF"><A href="workshop.html" target="_top">Workshop</A> |
<FONT face="Verdana, Arial, Helvetica, sans-serif" size="1" color="#FFFFFF"><A href="survey.html" target="_top">Survey</A></FONT></FONT></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:HKUST-Hong_Kong/future.html">http://2011.igem.org/Team:HKUST-Hong_Kong/future.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HKUST-Hong_Kong/future.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HKUST-Hong_Kong/future.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HKUST-Hong_Kong/future.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HKUST-Hong_Kong/future.html&amp;oldid=222209" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>