"
Page
Discussion
View source
History
teams
Log in
 
Team:HKUST-Hong Kong/asm.html
From 2011.igem.org
How to Select · Methods of Assembly · Details of Components · References
Strain Construction
1. How to select against E. CRAFT cells that fail to take up the vector plasmid - our alternative selection method
Our E. CRAFT (Escherichia coli Re-engineered for Antibiotics-Free Transformation) is designed to have one of its essential genes (genes that are required for viability) removed from its genome, and relocated into an engineered plasmid “Dummy”. This would result in E. CRAFT’s dependency on this extra- chromosomal copy of the essential gene for survival, and hence the addiction to the pDummy. By having direct control over the replication of pDummy, we dictate the life and death of E. CRAFT (and hence the name pDummy).
Here, we introduce a heat-sensitive origin of replication as the sole origin of pDummy. When we intend to switch off the pDummy’s replication, we can incubate E. CRAFT at a temperature above its optimum 30ᵒC . This origin would then cease to function, and pDummy cannot be maintained.[11] Deprived of the essential gene and its corresponding vital product, E. CRAFT will not be able to propagate unless it receives a heat insensitive analog of pDummy.
This analog plasmid, named “pCarrier”, is essentially our E. CRAFT- compatible vector in cloning. Under an unfavorably high incubation temperature, only E. CRAFT cells that are transformed with the insert-bearing pCarrier would be able to propagate and survive. The remaining E. CRAFT cells would not be able to undergo division and would eventually be eliminated from the population. In this sense, the pDummy can be considered to be "shuffled out" by pCarrier. Our designed selection system, in short, bases itself on plasmid shuffling, with no involvement of antibiotic resistance genes in any cloning step. [Top]
2. Stepping into the heart of construction - methods of assembly
2.1 Construction and maintenance of an antibiotic-resistance-gene-free plasmid through antibiotic selection – the unavoidable evil two plasmid system
Our ultimate goal is to construct the E. CRAFT without conferring any new antibiotic resistance on it. For this reason, no resistance gene should be found in our dummy plasmid: the pDummy.
Yet, ensuring the maintenance of such a plasmid in its host bacterium would be a challenge, unless the cell needs the plasmid for survival (essential- gene- loss induced addiction: loss of the essential gene in bacterial genome causes dependence on the extra-chromosomal copy in pDummy). Inconveniently, however, this addiction can only be achieved after the introduction of the plasmid.
The solution to this problem is to develop mutuality between pDummy and another plasmid by exploiting the nature of positively- regulated origins of replication. Well studied examples of such origins include those of pSC101 [2] and R6K plasmids [4, 5, 7, 8], where the origins of replication (OR) appear together with a constitutive gene (G). Initiation of replication happens if and only if the trans- element of the gene is provided.
Let’s consider the following scenario:
i. G is placed on the pDummy, which lacks a selection marker but is equipped with a normal replication origin
ii. OR is the sole origin of replication of another plasmid (here we introduce a new plasmid, pToolkit) with a selection marker
iii. pDummy and pToolkit are co-transformed to a bacterium which is under selection stress. [Top]
Three possible outcomes could be expected:
1. Only pDummy is uptaken
Since pDummy has no selection marker, the host bacterium would die under selection pressure and fail to propagate.
2. Only pToolkit is uptaken
The host bacterium that uptakes pToolkit survives. During its propagation, however, the pToolkit is not replicated because protein products of G are absent. Therefore, daughter cells of the host bacterium would not receive copies of the pToolkit and die under selection pressure.
3. Both pDummy and pToolkit are uptaken
In the presence of pDummy, pToolkit would be maintained and confer resistance to selection pressure on the host bacterium. Daughter cells that receive copies of both plasmids would survive and eventually form a colony.
Owing to this mutualistic relation, retention of the desired pDummy would be possible once the host bacterium develops an addiction it, while pToolkit can be lost in bacterial propagation if the expression of G can be shut off manually. Eventually, the bacteria would not obtain any new antibiotic resistance genes but keep pDummy.
2.2 Development of addiction – use of the λ RED recombination system [1]
Performed with a purpose to induce host bacterium’s addiction to the pDummy, removal of the essential gene nadE from the bacterial genomic DNA is mediated by the phage λ RED recombination system.
DNA Recombination is in fact inherent in E. coli, and this can be done via the RecBCD Recombination Pathway. Even so, it is not the most ideal method for engineering new strains of E. coli containing genomic modification, in part because this pathway requires a recognition site known as “χ” on the linear “to-be inserted” DNA sequence (called “Transformation DNA” henceforth). While this makes work for a molecular biologist more challenging, naturally, this is quite logical as the RecBCD pathway is meant to be a DNA Repair pathway when the E. coli’s genome is damaged (rather than a means to introduce new DNA material into the genome).
Even so, E. coli has its own natural “enemies” and one known enemy is the phage λ. As phage λ is a lysogenic phage (phage able to enter lysogenic phase), it has mechanisms to ensure that its DNA can be integrated into the host’s genome. One mechanism involves the proteins γ. Exonuclease (Exo) and β. Collectively, these three proteins are called “λ-RED”. Essentially, λ-RED hijacks the DNA repair machinery of the E. coli to promote site-specific homologous recombination. While the exact mechanism is still being contested and investigated, some insights have been gained:
γ will first bind to E. coli’s RecBCD, inhibiting RecB’s nuclease activity, thus shutting down this recombination pathway. Exo will then begin degrading the linear Transformation DNA (dsDNA) in the 5’→3’ direction, resulting in either 3’ ssDNA overhangs on both sides of the dsDNA or a sole ssDNA (proposed mechanistic differences). β will then bind to the 3’ ssDNA and encourage the ssDNA to anneal to the target dsDNA.
This in turn will activate E. coli’s alternative DNA repair system, RecFOR Recombination Pathway, whereby RecFOR will recruit RecA on to the ssDNA-dsDNA complex (at the homology sites). As the complementation of the linear Transformation and genomic DNA will result in a Holliday junction, RuvABC will be recruited to “resolve” this junction, cleaving away the target DNA on the bacterial genome and integrating the Transformation DNA into the genome. (It should be noted that RecA and RuvABC are also a shared downstream pathway for the RecBCD Recombination Pathway).
The λ RED recombination cassette is located in our third plasmid “Toolkit”. Upon successful co-transformation of pDummy and pToolkit, loss of genomic essential gene can be stimulated by introducing- into the bacterial cell- linear dsDNA molecules carrying a reporter gene flanked by sequences homologous to those of the essential gene. An expected outcome of this introduction is the swapping out of the nadE gene with the reporter gene.
Since the linear dsDNAs do not have origin of replications, they would not be inherited in daughters unless the swapping has taken place properly. Thus any observable signals from the reporter would allow identification of successful recombination. Once the recombination is completed, the toolkit plasmid and the cell’s antibiotic resistance gene can be eliminated from the host bacterium, giving us the completed strain of E. CRAFT.
2.3 Complementation between reporter genes – manifesting completion of E. CRAFT engineering
To ensure that the final strain of E. CRAFT has:
(1) successfully had its essential nadE gene deleted from the genome;
(2) maintained the pDummy,
a complementation reporter system between the pDummy and the swapped gene is preferred over a single reporter at the swap site.
Different methods can be employed to achieve this aim:
i. α complementation can be used in E. coli strains which have had their lacZ gene completely removed. The larger fragment ω can be swapped in place for the essential gene while the smaller α fragment can stay on pDummy. In an X-gal rich medium, blue colonies would indicate the desired engineered strains.
ii. Complementation between split fluorescent proteins (sFP). 2010 iGEM Slovenia team has demonstrated the principle that N-terminal and C-terminal fragments of sFPS are able to complement in vivo, and two sets of sfFPS are able to undergo Forster Resonance Energy Transfer (FRET). Using this system has been considered in this project, but an alternative reporter candidate, the split superfolder GFPs (sfGFP) [6, 12, 13] invented by other researchers, was tested instead.
2.4 Summary of construction flow:
1. Assembly pDummy and pToolkit
2. Co-transformation of both plasmids into E. coli and maintenance of stable strains
3. Introduction of linear ds DNAs and induction of recombination
4. Isolation of recombinants
5. Induction pToolkit loss [Top]
3. Details of the components – a closer look to the molecular basis of assembly
3.1 Temperature-sensitive origin of replication_oriR101 & repA101-ts (BBa_K524000)
oriR101 & repA101-ts is a set of low copy origins of replication derived from the pSC101 replication origin.[11] The repA101-ts gene encodes for a heat-labile protein that is required in trans for the initiation of replication at oriR101. Characterization of our construct has shown that plasmids with this origin of replication could only be fully maintained if incubated at lower than 30ᵒC, while partial maintenance of plasmid was observed at 33ᵒC. The origin ceases function at 37ᵒC.
This part was cloned out from pKD46 plasmid (courtesy of The Coli Genetic Stock Center), and standardized by nucleotide mutation.
3.2 split superfolder green fluroscent protein_split sfGFP
sfGFP1-10 (BBa_K524001) [Twins: BBa_K524006]
sfGFP11 (BBa_K524002) [Twins: BBa_K524007]
The sfGFPs are mutant variants of GFPs, with characteristics of improved folding kinetics and resistance to chemical denaturants. [10] Split sfGFPs have been reported to be able to undergo complementation at amino acid residues 214 and 215 to give green fluorescence. [6, 12, 13]
3.3 Essential gene nadE (BBa_K524003)
nadE, which encodes the enzyme NAD+ synthetase, is a vital gene in E. coli. [2] In principle, removal of such a gene from the bacterial genome would cause the cells to be addicted to a plasmid that has a copy of the gene. CyaR (a sRNA) regulates the expression of nadE post-transcriptionally, and this feature is retained in our construct. Transcription of nadE operon requires the sigma-70 initiation factor and is terminated by downstream extragenic sites.
The nadE gene was cloned out from the genome of E. coli strain BL21(DE3), and its construction was completed by ligating B0015 double terminator to its end.
3.4 π replication initiator protein encoded by pir gene (BBa_K524004) and γ-origin of replication (ori-γ) from R6K plasmid
Ori-γ is one of the three replication origins (the other two being α and β) of the R6K origin. Initiation of replication at ori-γ is regulated in trans by the π protein encoded by pir gene. [4, 5] While the presence of the appropriate amount of π protein is required for replication initiation, doubling the concentration of the same protein would effectively shut down the process. [8] Expression of π protein is autogenously regulated. [7]
In the fabrication of these two constructs, the pir gene was cloned out from the genome of BW25141 E. coli strain (courtesy of The Coli Genetic Stock Center) and standardized.
The ori-γ was adopted from the R6K origin of replication BBa_J61001.
3.5 iGEM 2010 Slovenia Split/FRET constructs
The split Cyan Fluorescent Protein (CFP) and Yellow Fluorescent Protein (YFP) biobricks constructed by the 2010 Slovenia Team are used as alternatives to the superfolder GFP reporter. The idea here is to put one of the terminal fragments of either CFP or YFP into the pDummy, and swap out the essential nadE gene from the bacterial genome with the other terminal fragment of the fluorescent protein. Driven by pLac R0010, both fragments should be expressed simultaneously when induced by IPTG. Fluorescent signal observed could be used as an indicator of successful recombination. [Top]
4. References
1. Datsenko KA, Wanner BL.(2000). One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products, Proc Natl Acad Sci U S A. 2000 Jun 6;97(12):6640-5.
2. D. Manen, L. Caro.(1991). The replication of plasmid pSC101, Mol Microbiol. 1991 Feb;5(2):233-7.
3. Ecocyc: Encyclopedia of Escherichia coli K-12 Genes and Metabolism
4. F Wu, I Goldberg, and M Filutowicz.(1992). Roles of a 106-bp origin enhancer and Escherichia coli DnaA protein in replication of plasmid R6K, Nucleic Acids Res. 1992 February 25; 20(4): 811–817.
5. F Wu, I Goldberg, and M Filutowicz.(1994). Binding of DnaA protein to a replication enhancer counteracts the inhibition of plasmid R6K γ origin replication mediated by elevated levels of R6K π protein, J Bacteriol. 1994 November; 176(22): 6795–6801.
6. Jun Zhou, Jian Lin, Cuihong Zhou, Xiaoyan Deng and Bin Xia.(2011). An improved bimolecular fluorescence complementation tool based on superfolder green fluorescent protein, Acta Biochim Biophys Sin 43 (3): 239-244.
7. M Filutowicz, G Davis, A Greener, and D R Helinski.(1985). Autorepressor properties of the π-initiation protein encoded by plasmid R6K, Nucleic Acids Res. 1985 January 11; 13(1): 103–114.
8. M Filutowicz, M J McEachern, and D R Helinski.(1986). Positive and negative roles of an initiator protein at an origin of replication, Proc Natl Acad Sci U S A. 1986 December; 83(24): 9645–9649.
9. Peubez I, Chaudet N, Mignon C, Hild G, Husson S, Courtois V, De Luca K, Speck D, Sodoyer R.(2010). Antibiotic-free selection in E. coli: new considerations for optimal design and improved production, Microb Cell Fact. 2010 Sep 7;9:65
10. Pédelacq JD, Cabantous S, Tran T, Terwilliger TC, Waldo GS.(2006). Engineering and characterization of a superfolder green fluorescent protein, Nat Biotechnol. 2006 Jan;24(1):79-88. Epub 2005 Dec 20.
11. Phillips GJ.(1995). New Cloning Vectors with Temperature-Sensitive Replication, Plasmid. 1999 Jan;41(1):78-81.
12. Stéphanie Cabantous, Thomas C Terwilliger & Geoffrey S Waldo.(2004). Protein tagging and detection with engineered self-assembling fragments of green fluorescent protein, Nature Biotechnology 23, 102 – 107
13. Stéphanie Cabantous & Geoffrey S Waldo.(2006). In vivo and in vitro protein solubility assays using split GFP, Nature Methods - 3, 845 – 854
[Top]
Home
Our Project
Overview | Data Page
Experiments and Results
Strain Construction | Culture Tests | Modeling
Miscellaneous
Notebook
iGEM Resources
Acknowledgements
The Team
iGEM Member List | Contributions
Achievements
Medal Requirements | BioSafety
BioBricks
Master List & Characterization Data
Human Practice
Workshop | Survey
Retrieved from " http://2011.igem.org/Team:HKUST-Hong_Kong/asm.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
