<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HKUST-Hong_Kong_characterization_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:HKUST-Hong_Kong/characterization.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HKUST-Hong_Kong/characterization.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HKUST-Hong_Kong/characterization.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HKUST-Hong_Kong/characterization.html&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HKUST-Hong_Kong/characterization.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HKUST-Hong Kong/characterization.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P align="left"><FONT face="Verdana, Arial, Helvetica, sans-serif" size="6" color="white">
BioBricks Master List and Characterization Data 
</FONT></P><TABLE style="border-collapse: collapse" width="963"><TBODY><TR bgcolor="white"><TD><BLOCKQUOTE>
1. <A href="http://partsregistry.org/Part:BBa_K524000"><FONT color="blue">BBa_K524000</FONT></A> –oriR101 &amp; repA101-ts: This heat sensitive origin of replication has been shown to be functional at 30°C. Partial loss of plasmid happens at 33°C, and complete loss of plasmid is observed at 37°C.

2. <A href="http://partsregistry.org/Part:BBa_K524001"><FONT color="blue">BBa_K524001</FONT></A> –pLac + RBS + sfGFP1-10: This contains coding DNA sequences for superfolder GFP amino residues 1-214 driven by RBS BBa_B0034 and pLac BBa_R0010.

3. <A href="http://partsregistry.org/Part:BBa_K524002"><FONT color="blue">BBa_K524002</FONT></A> –pLac + RBS + sfGFP11 + double terminator: This contains coding DNA sequences for superfolder GFP amino residues 215-238 driven by RBS BBa_B0034 and pLac BBa_R0010, with double terminator B0015 at the end.

4. <A href="http://partsregistry.org/Part:BBa_K524003"><FONT color="blue">BBa_K524003</FONT></A> – nadE gene + double terminator: an essential gene that encodes constitutively- expressed NAD+ synthetase.

5. <A href="http://partsregistry.org/Part:BBa_K524004"><FONT color="blue">BBa_K524004</FONT></A> –pir gene: the pir gene encodes the autogenously regulated pi protein.

6. <A href="http://partsregistry.org/Part:BBa_K524005"><FONT color="blue">BBa_K524005</FONT></A> – pToolkit: A plasmid that is only maintained in strains with pir+ genotype. The plasmid was modified from the plasmid pKD46, which has been shown to trigger DNA recombination and subsequent gene knockout under induction of arabinose.


7. <A href="http://partsregistry.org/Part:BBa_K524006"><FONT color="blue">BBa_K524006</FONT></A> –RBS + sfGFP1-10: This contains coding DNA sequences for superfolder GFP amino residues 1-214 driven by RBS BBa_B0034.

8. <A href="http://partsregistry.org/Part:BBa_K524007"><FONT color="blue">BBa_K524007</FONT></A> –RBS + sfGFP11 + double terminator: This contains coding DNA sequences for superfolder GFP amino residues 215-238 driven by RBS BBa_B0034, with double terminator B0015 at the end.


9. <A href="http://partsregistry.org/Part:BBa_K524100"><FONT color="blue">BBa_K524100</FONT></A> – bcr CDS + double terminator B0015: The gene product is responsible for multidrug efflux system that pumps out antibiotics. Effect of pump was documented by other researchers: it confers varying degrees of resistance to 5 different antibiotics.</BLOCKQUOTE><P><FONT size="3"><B><U>Characterization Data for BBa_K524000</U></B></FONT></P><P align="left"><STRONG>Heat Sensitive Origin of  Replication (oriR101 &amp; repA101-ts) </STRONG></P><P align="left"><U><STRONG>Abstract</STRONG></U></P><P>oriR101 &amp; repA101-ts constitute a low copy origin of replication derived from the pSC101 replication origin. Initiation of replication at oriR101 is regulated in <I>trans</I> by the heat-labile protein product of repA101-ts, which loses activity significantly at 37°C. Hence when bacteria harboring plasmids with this origin are incubated at 37°C, the activity of this origin would decrease. Temperatures higher than 42°C would cause the construct to cease its functions completely. (Phillips GJ, 1995).
  In the process of its fabrication, a point mutation was introduced (base substitute of C to G at nucleotide residue 1391) so as to make the construct compatible with Biobrick standard assembly requirements. Nevertheless, our characterization results do not seem to suggest great impacts to the construct's functions, causing only a slight increase in its thermosensitivity.
 </P><P align="left"><U><STRONG>Construction of This Part</STRONG></U></P><P>This construct was cloned out from the pKD46 plasmid harboured in the <I>E. coli</I> strain BW25113 (courtesy of The Coli Genetic Stock Center). 
A single nucleotide mutation, performed through overlapping PCR, was introduced to eliminate a SpeI cut site originally present inside the construct, specifically between the promoter and CDS of the ori101R &amp; repA101-ts. The mutation has been successfully confirmed by restriction digestion (Figure 1) and nucleotide sequencing.</P><P align="left"><STRONG>#Figure 1:  Digestion with SpeI for Confirmation of Successful Mutation</STRONG></P><P align="left"><U><STRONG>Characterization</STRONG></U></P><UL><LI><B>Construct Preparation for Heat Sensitivity  Test</B></LI></UL><P align="left">Characterization of the oriR101 &amp; repA101-ts was done with the insert ligated to RFP reporter and harbored in partregistry.org's standard assembly pSB1AK3 plasmid. The enzymes PstI and SmaI were then used to cut out the normal replication origin (pMB1) of pSB1AK3 and part of its KanR gene. After the blunt end treatment, the linearized plasmid was self-ligated and transformed into <EM>E. coli</EM> DH10b. Restriction digestion test, along with the presence of red fluorescence and absence of Kanamycin resistance, is used for plasmid verification. The final construct used for thermosensitivity test is referred to as “<I>ori-ts</I>” for convenience.</P><UL><LI><B>Characterization Methods</B></LI></UL><UL><UL><LI><B>Qualitative Test</B></LI></UL></UL><P align="left">  To test the thermosensitivity of the origin, we employed the plasmid-loss assay. When the cells were incubated in non-permissive temperatures (i.e. temperatures above 37°C), the plasmid containing the antibiotic-resistance gene will be lost and no growth of cells harboring <I>ori-ts</I> will be seen on the antibiotic-containing plates (here we used ampicillin with working concentrations of 150μg/ml); while cells that contain a plasmid with the corresponding antibiotic-resistance gene and a normal replication origin will survive and proliferate.

In our tests, serial dilution was performed on overnight cultures of <I>ori-ts</I>(+) cells. Dilutions were performed to give the following bacterial concentrations: 1 cell/7μl, 10 cells/7μl, 100 cells/7μl, and 1000 cells/7μl. 7μl of the abovementioned solutions were applied on LB and LB(Amp) plates. The plates were then incubated at 30°C, 33°C, 37°C, and 42°C. LB plates were prepared as a control to reflect the actual titration of <EM>E.coli</EM> in each dilution.</P><UL><UL><LI><B>Quantitative Test</B></LI></UL></UL><P align="left">To quantitatively determine the elimination rate of plasmids bearing our oriR101 &amp; repA101-ts, (approximated) equal numbers of <I>ori-ts</I>-harboring bacterial cells were spread on LB plates (labeled as LB_ori-ts) and LB (Amp) plates (labeled as Amp_ori-ts). Four replicas of this experimental setup were prepared for overnight incubations at four different temperatures: 30°C, 33°C, 37°C, and 42°C. Then, colonies formed on the plates after overnight incubation were quantified. For each set, the ratios of colonies on Amp_ori-ts plates vs. LB_ori-ts plates were calculated. The same protocol described above was applied to the same <I>E. coli</I> strain (DH10b) containing only the plasmids pKD46 and pBlueScriptKS+ to serve as controls.
  
  The total plasmid elimination rate with the thermosensitive replication origins (both the mutated version in pSB1AK3 and original version in pKD46) is computed by the following  formula:</P><UL><LI><B>Characterization Results</B></LI></UL><UL><UL><LI><B>Qualitative Test</B></LI></UL></UL><P align="left">Several sets of  serial dilution tests were done at 30°C, 33°C, 37°C and 42°C. The evidence and results are shown below.<STRONG>#Figure 2: Qualitative Description of the Temperature Sensitivity Test data.</STRONG></P><P align="left"><I>For each temperature point in the gradient, the most concentrated O/N culture was applied on the leftmost-side, and serially diluted by 10-fold towards the rightward direction.</I></P><P align="left">A robust growth of <I>ori-ts</I>(+) cells was observed when the cells were incubated at 30°C. This robustness (indicated by the density of bacterial colonies in each droplet, and associated with the proper functioning of the plasmid's heat sensitive origin) steadily declined as the incubation temperature was increased. Partial replication origin dysfunction was observed after 33°C incubation, and near-complete loss of function was observed at 37°C. Comparing this trend with the tested result of the original oriR101 &amp; repA101-ts in pKD46 (partial loss of function observed at 37°C, and near-complete loss of function at 42°C), the mutated one shows higher thermosensitivity.</P><UL><UL><LI><B>Quantitative test</B></LI></UL></UL><P align="left">The result of the quantitative thermosensitive test and trend of the % elimination of <I>ori-ts</I> is shown by the following graph:</P><P align="left"><STRONG>#Figure 3: Quantitative Analysis of the temperature sensitivity origin.</STRONG></P><P align="left"><U><STRONG>Conclusion</STRONG></U>
  This pKD46-derived oriR101 &amp; repA101-ts displays intensified thermosensitive properties, which might be due to the point mutation introduced during the fabrication of the construct. The optimal incubation temperature for  plasmid maintenance purposes is 30°C, whereas our recommended temperature for  plasmid loss induction is 37°C or above. </P><P><B><U>References</U></B></P><P>
Datsenko KA, Wanner BL.(2000).One-step inactivation of chromosomal genes in <I>Escherichia coli</I> K-12 using PCR products, Proc Natl Acad Sci U S A. 2000 Jun 6;97(12):6640-5.

D. Manen, L. Caro.(1991).The replication of plasmid pSC101, Mol Microbiol. 1991 Feb;5(2):233-7.

Phillips GJ.(1995). New Cloning Vectors with Temperature-Sensitive Replication, Plasmid. 1999 Jan;41(1):78-81.
</P></TD></TR></TBODY></TABLE><TABLE border="0" cellspacing="0" cellpadding="10"><TBODY><TR><TD width="100px" height="150px"><P align="center"><A href="https://2011.igem.org/Team:HKUST-Hong_Kong" target="_top"><B><FONT color="#FFE1E1" size="3">Home</FONT></B></A></P></TD><TD width="382px" bgcolor="#CCFF99" valign="baseline"><P align="center" valign="baseline"><B><FONT color="green">Our Project</FONT></B></P><P align="center" valign="baseline"><A href="https://2011.igem.org/Team:HKUST-Hong_Kong/overview.html" target="_top">Overview</A><FONT color="green"> | </FONT><A href="https://2011.igem.org/Team:HKUST-Hong_Kong/data.html" target="_top">Data Page</A></P><P align="center" valign="baseline"><B><FONT color="green">Experiments and Results</FONT></B></P><P align="center" valign="baseline"><A href="https://2011.igem.org/Team:HKUST-Hong_Kong/asm.html" target="_top">Strain Construction</A><FONT color="green"> | </FONT><A href="https://2011.igem.org/Team:HKUST-Hong_Kong/mic.html" target="_top">Culture Tests</A><FONT color="green"> | </FONT><A href="https://2011.igem.org/Team:HKUST-Hong_Kong/modeling.html" target="_top">Modeling</A></P><P align="center" valign="baseline"><B><FONT color="green">Miscellaneous</FONT></B></P><P align="center" valign="baseline"><A href="https://2011.igem.org/Team:HKUST-Hong_Kong/notebook.html" target="_top">Notebook</A></P></TD><TD width="302px" bgcolor="#D09C00" valign="baseline"><P align="center" valign="baseline"><B><FONT color="#FFF4D0">iGEM Resources</FONT></B></P><P align="center" valign="baseline"><A href="https://2011.igem.org/Team:HKUST-Hong_Kong/acknowledgement.html" target="_top">Acknowledgements</A></P><P align="center" valign="baseline"><B><FONT color="#FFF4D0">The Team</FONT></B></P><P align="center" valign="baseline"><A href="https://2011.igem.org/Team:HKUST-Hong_Kong/team.html" target="_top">iGEM Member List</A><FONT color="#FFF4D0"> | </FONT><A href="https://2011.igem.org/Team:HKUST-Hong_Kong/contribution.html" target="_top">Contributions</A></P><P align="center" valign="baseline"><B><FONT color="#FFF4D0">Achievements</FONT></B></P><P align="center" valign="baseline"><A href="https://2011.igem.org/Team:HKUST-Hong_Kong/medal.html" target="_top">Medal Requirements</A><FONT color="#FFF4D0"> | </FONT><A href="https://2011.igem.org/Team:HKUST-Hong_Kong/biosafety.html" target="_top">BioSafety</A></P><P align="center" valign="baseline"><B><FONT color="#FFF4D0">BioBricks</FONT></B></P><P align="center" valign="baseline"><A href="https://2011.igem.org/Team:HKUST-Hong_Kong/characterization.html" target="_top">Master List &amp; Characterization Data</A></P></TD><TD width="180px" bgcolor="#980000" valign="baseline"><P align="center" valign="baseline"><B><FONT color="#FFE0E0">Human Practice</FONT></B></P><P align="center" valign="baseline"><A href="https://2011.igem.org/Team:HKUST-Hong_Kong/workshop.html" target="_top">Workshop</A><FONT color="white"> | </FONT><A href="https://2011.igem.org/Team:HKUST-Hong_Kong/survey.html" target="_top">Survey</A></P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:HKUST-Hong_Kong/characterization.html">http://2011.igem.org/Team:HKUST-Hong_Kong/characterization.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HKUST-Hong_Kong/characterization.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HKUST-Hong_Kong/characterization.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HKUST-Hong_Kong/characterization.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HKUST-Hong_Kong/characterization.html&amp;oldid=253992" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>