Team:SYSU CHINA/Note
Project
Design
Result
Safety
Note
Human Practice
Team
Note
Method
Parts
Typical parts we constructed
Parts List
Recombinase library
Note
2015-04-16
2015-04-25
2015-05-01
2015-05-08
2015-05-12
2015-05-14
2015-05-26
2015-06-01
2015-06-07
2015-06-12
2015-06-18
2015-06-25
2015-07-01
2015-07-10
2015-07-15
2015-07-18
2015-07-23
2015-07-24
2015-07-29
2015-08-01
2015-08-04
2015-08-10
2015-08-17
2015-08-25
2015-08-28
2015-09-01
2015-09-06
2015-09-10
2015-09-12
2015-09-14
-culture the competence cell
-test the experiment kits
-Molecular cloning for Bba_J06602, BBa_B0012, Bba_K592024, Bba_I13453, Bba_B0010
-Tried to reverse pBAD(BBa_I13453)
-preparation of the target bio-brick
-Ligation: flpe-BFP, loxP-pBAD, B0010-loxP, eGFP-B0015, FRT-pBAD
-Preservation of the plasmid pAUR135 in E.coli; Transformation of the commercial plasmid pAUR135(from TAKARA, Item No.D3604); select monoclonal colony and extract plasmid(pAUR135) from the culture.
-Prepared Yeast Extract Peptone Dextrose(YPD) Medium
-Patch: strain: S.cerevisiae W303; medium: YPD agar
-Extract the genome of the budding yeast as the template used in the subsequence PCR, yeast stains was Saccharomyces cerevisiae laboratory strain W303 (haploid), and the extraction procedure is based on the fungal genome extraction kit(Omega Bio-Tec).
-Poor yield, partly because the yeast concentration in the suspension is low for genome extraction. The time for yeast incubation should be prolong for extraction next time.
-ligation: pBAD-eGFP- B0010;pBAD-EFP- B0010;PBAD-mCherry- B0010.
-induction and test the intensity of fluorescence
-HOWEVER our results went bad
-Extract genome DNA from W303 with yeast genome extraction kit(Omega Bio-Tec).
-Acquire homologous franking sequence for target integrating site with PCR procedure, using the extracted genome from W303 as a template.
-Ultraviolet exposure result of the identification gel shows nonspecific amplification. We plan to employ nested PCR.
-Preservation of W303 strain.
-Extract genome DNA from W303 with yeast genome extraction kit(Omega Bio-Tec).
-Employ nest PCR to acquire homologous sequence and M phase Promoter sequence using W303 genome.
-Ultraviolet exposure result of the identification gel shows no target band.
-Employ nest PCR and gradient anneal temperature from 50-60℃ to define the best anneal temperature.
-Anneal at 60℃ should be better.
-The ligation of pBAD-FR, pBAD-flpe is successful, we tried to insert Cre segment to pSB1A2
-Employ nest PCR and gradient anneal temperature from 56-66℃ to define the best anneal temperature.
-The most appropriate anneal temperature should be 65℃.
-Extract genome DNA from W303 with yeast genome extraction kit(Omega Bio-Tec).
-Ligation again: pBAD-mCherry-B0010, pBAD-BFP-B0010, pBAD-eGFP-B0015
-induction and test the intensity of fluorescence
-Again, we ligated pBAD with FRT
-We designed qPCR experiments to test whether our system work.
-We ordered our IDT gBlocks products.
-We tested whether our ligation products were corrected by sequence and enzyme digestion, and we found that BBa_B0010 in distribution plate was a fake one!
-Acquire M phase promoter sequence with GenStar 2x Taq PCR Star(Mix with Loading Dye) using W303 genome.
-Clean up the PCR product(M phase promoter nest sequence).
-Mutation of the plasmid pAUR135.( Muta-direct™ Kit from SBS Genetech Co.,Ltd. SDM-15).
-We found that some of the clones turned red after overnight grown in LB broth. All of these clones were transferred with plasmids contain parts BBa_B0010, indicating that these plasmids were polluted by BBa_J04450.
-Acquire M phase promoter (prefix and suffix added M phase promoter sequence) with PCR procedure, using nest PCR product as template, and gel extraction.
-Acquire homologous sequence (prefix and suffix added homologous sequence) with PCR procedure, using nest PCR product as template, and gel extraction.
-Add RBS and ssrA(strong) to mCherry by PCR and Extract PCR products by using Gel Extraction Kit.
-Double digestion of RBS-mc-ssrA(strong) PCR product and pSB1C3(r) with Spe I and EcoR I. Extract double digestion products by using Gel Extraction Kit.
-Ligation of RBS-mc-ssrA(strong).
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+cm.
-We conducted digestion identification and sequencing. No sequencing results were obtained, and the sizes of digestion products were incorrect!
-Sequencing result of pAUR135 mutant: pAUR135 mutant have  A1041C mutation in CDS of AurR gene, which result in the elimination of Pst I restriction site.
-Enzyme digestion with FD Pst I (Takara)
-Extract the product(pAUR135 backbone with pst I sticky end) of 5000+bp
-End-filling with S1 nuclease.
-Ligation with T4 DNA polymerase.
-Pick single colonies of RBS-mc-ssrA(strong), and streak them onto another plate to expand culture.
-Perform colony PCR of RBS-mc-ssrA(strong).
-Pick correct colonies of RBS-mc-ssrA(strong), and inoculate a culture of 4 ml LB medium containing chloromycetin.
-We noted that BBa_B0010(fake) can ligate with our parts, and thus any part ligated with it must be reconstructed!
-We began to reconstruct our parts, and some of our parts, like Cre and eGFP run out. We amplified BBa_B1006 and BBa_B0015 in DH5a.
-Transform 5ul T4 ligation product (Jul.15) into 50ul DH5α.
-Select monoclonal colony and extract plasmid (pAUR135 circular backbone without pst I).
-Checkout of the removal of Pst I enzyme site.
-Use Plasmid Miniprep Kit to isolate mc-ssrA(strong) from the cultures.
-We received our booked IDT gBlocks products arrived!
-We changed our circuits by replacing some devide parts with composite parts. we converted BFP to BBa_E0422(ECFP), converted eGFP to E0840 (GFP with terminator), such changes can accelerate our work
-Double enzyme digestion (HF，NEB) with EcoR I and Sac I.
-Gel extraction (Digestion product with EcoR I and Sac I sticky end) With Tiangen gel extraction kit.
-Anneal of MCS oligonucleotide.
-Ligation with double digested backbone and MCS anneal product.
-Transformation with ligation product.
-no colony found on the plate
-Double enzyme digestion did not work, as enzyme Sac I has very low(~20%) efficiency when the space between the two enzyme sites is very close(2bp in this experiment).
-Add RTS sequences (loxp) to the flanks of RBS-mC-ssrA(strong)-1 by PCR.
-Extract PCR products by using Gel Extraction Kit.
-Double digestion of loxp-mcss-1 and B0015 (B0015 digest with Xba I and EcoR I, Fragments digest with Spe I and EcoR I). Extract double digestion products by using Gel Extraction Kit.
-Ligation of loxp-mcss-T.
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+cm.
-Later sequence result shows the template we used, RBS-mC-ssrA(strong)-1, is not correct. So the clones made by this day are not correct too.
-We tested and confirmed that mCherry, E0422, E0840, Cre and B1006. They were all corrected.
-We ligated E0840 with loxP(reverse) and mCherry with B1006. But nearly no clone grown.
-We obtained some clones with ligation products of mCherry-B1006
-The first part, BBa_K1641213, was successfully constructed!
-Parts construction
-Adjust double enzyme digestion into two single digestions, first Sac I then EcoR I.
-MSC modification fails.
-Change MSC modification primer using Kpn I and EcoR I as sticky end.
-Double enzyme digestion with Kpn I and EcoR I and then ligate with MCS annealed product.
-Transformation.
-Extract plasmid: pAUR135-KE.
-Sequencing result shows incorrect ligation.
-Add RTS sequences (loxp) to the flanks of RBS-mC-ssrA(strong)-2 by PCR. Extract PCR products by using Gel Extraction Kit.
-Ligated E0840 with loxP and pBAd-flpe-ECFP. We tried several methods of ligation. Preliminary "2A" assembly mthod was developed. However, colony PCR showed high false positive rate of these clones.
-Double digestion of loxp-mcss and B0015. (B0015/B1006 digested with Xba I and EcoR I, Fragments digested with Spe I and EcoR I).Extract double digestion products by using Gel Extraction Kit.
-Ligation of loxp-mcss-T.
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+cm.
-Pick single colonies of loxp-mcss-T, and streak them onto another plate to expand culture.
-Perform colony PCR of loxp-mcss-T and record the correct colonies.
-Pick correct colonies of loxp-mcss-T, and inoculate a culture of 4 ml LB medium containing chloromycetin.
-Use Plasmid Miniprep Kit to isolate loxp-mcss-T from the cultures.
-We used PCR to reconstruct RBS-Cre-ssrA, and inserted this fragment into pSB backbone.
-All clones grown well! But somebody placed our enzymes out of refrigerator overnight. We went crazy! So our process was delayed.
-We placed the gBlocks products into pSB backbones. Most of them were successful. Also we gained some correct clones of loxP-GFP(BBa_K16416209).
-Ligations: Cre with mCherry and K1641206 with flpe-ECFP
-pAUR135-MCS modification
-Double enzyme digestion with Kpn I and EcoR I and then ligate with MCS annealed product.
-Transformation;Extract plasmid: pAUR135-MCS-KE
-Clone biobrick:KRE9UTR;GFP-PEST191;SV40NLS;Venus YFP
-Patch:bxb1gp35 RFC25optimized-puc19
-Transformation and Plasmid Preparation-MCS-KE
-Sequencing confirmed.
-Again, we tried to ligate the short intermediates with the long parts:pBAD(reverse) with FPs, etc.
-We successfully ligated pBAD(reverse) with flpe-ECFP.
-We used endonuclease and sequencing skills to test the correctness of ligation
-Clone cyclic promoters;PCR :YMR013C/Cln3p/Met16p from yeast genome add attB/P sites, prefix and suffix
-transformation and plasmid preparation:MCS-KE
-Assemble:Pgal+kozak-1C3;bxb1gp35+SV40NLS-1C3;;Pbad-RBS-1C3
-Double digestion, ligation and transformation
-Mutation:MCS-KE to RFC25 optimized
-Substrate digested via Fast Digest-Dpn I.
-product transformed in TOP10 50, Amp
-Double digestion of loxp-mc-1, loxp-mc-2, B0015 and CG, CGS, B1006 (B0015/B1006 digested with Xba I and EcoR I. Fragments digested with Spe I and EcoR I). Extract double digestion products by using Gel Extraction Kit.
-Ligation of loxp-mc-1-T, loxp-mc-2-T and CGT, CGST.
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+cm.
-Use Plasmid Miniprep Kit to isolate I0500 from the cultures.
-We ligated pBAD(forward) with other parts contains FPs
-We test whether pBAD and FPs can worked, but we met high false positive rates.
-We found that the ECFP sequence of our BBa_K1641209 sample was reverse! Oh my God!
-Clone biobrick:ECFP;EYFP;mRFP1
-PCR:reversed mRFP1;reversed KRE9UTR
-Plasmid preparation: pAUR135 RFC25 optimized
-Assemble:reversed KRE9UTR+reversed mRFP1;eCFP+KRE9UTR
-Double digestion, ligation and transformation
-Add sequence via PCR: prefix-RBS-eCFP-terminatior-frt;loxP-RBS-Cre-ssrA;prefix-RBS-GFP-Terminator-loxp;prefix-ECFP-frt-suffix;prefix-loxP-Cre-suffix;prefix-GFP-loxp-suffix
-Assemble:Met16p-eCFP;YMR013C-eCFP;Loxp-cre+gfp-loxp;frt-flpe-cfp-frt
-Transformation.
-Plasmid preparation.
-Double digestion of dre, vika, scre, vcre, pSB1C3(r) with Spe I and EcoR I. Extract double digestion products by using Gel Extraction Kit.
-Ligation of dre, vika, scre, vcre and pSB1C3(r).
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+cm.
-Pick single colonies of dre(1c3), vika(1c3), vcre(1c3), scre(1c3), and streak them onto another plate to expand culture.
-Perform colony PCR of dre(1c3), vika(1c3), vcre(1c3), scre(1c3) and record the correct colonies.
-Pick correct colonies of dre(1c3), vika(1c3), vcre(1c3), scre(1c3), and inoculate a culture of 3 ml LB medium containing chloromycetin
-Add RTS sequences(loxp) to the flanks of RBS-mC-ssrA(strong) by PCR. Electrophoresis result didn’t shows correct bands.
-Use Plasmid Miniprep Kit to isolate dre(1c3), vika(1c3), vcre(1c3), scre(1c3) from the cultures.
-We tried to amplify BBa_K1641203, BBa_K1641201, BBa_K1641209
-We ligated mCherry with BBa_K1641208.
-Assemble:reversed KRE9UTR+reversed mRFP1;eCFP+KRE9UTR
-Restriction analysis:loxp-cre-gfp-loxp
-Assemble:YFP-KRE9UTR;reversed KRE9UTR+reversed mRFP1
-Sequencing:loxp-cre-gfp-loxp
-Prepare RBS-Flpe-fc by PCR.
-Use Plasmid Miniprep Kit to isolate CG, CGS, B0015, B1006, I0500 from the cultures.
-Add RTS sequences(loxp) to the flanks of RBS-mC-ssrA(strong) by PCR. Electrophoresis result didn’t shows correct bands.
-Add RTS sequences(FRT, Rox, SloxpM1, Vloxp, vox) to the flanks of RBS-mC-ssrA(strong) by PCR. Electrophoresis result didn’t shows correct bands.
-We had many colony PCR experiments to find a correct clone, we did.
-We eventually had a real BBa_K1641209. Also we have correct BBa_1641203 and BBa_K1641201. So we can use these parts to construct the Circuit2, which was improved on the basis of Circuit1.
-clone and assemble:loxp-cre-gfp-loxp;frt-flpe-cfp-frt
-sequencing inconsistant
-Double digestion of PCG-1, PCGS-1, B1006. (B1006 digested with Xba I and EcoR I, fragment digested with Spe I and EcoR I). Extract double digestion products by using Gel Extraction Kit.
-Ligation of PCGT, PCGST.
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+cm.
-Perform 20ul system PCR to test if the primers of FRT, Rox, SLM, VL, vox work.
-Add RTS sequences to the flanks of FRT, Rox, SLM, VL, vox by PCR. Extract PCR products by using Gel Extraction Kit.
-Double digestion of FRT, Rox, SLM, VL, vox and pSB1C3(r) with EcoR I and Pst I. Extract double digestion products by using Gel Extraction Kit.
-Ligation of FRT, Rox, SLM, VL, vox.
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+cm.
-Use Plasmid Miniprep Kit to isolate PCGT-1, PCGST-1, pSB3k3 from the cultures.
-Double digestion of PCGT-1, PCGST-1 and pSB3k3. Extract double digestion products by using Gel Extraction Kit.
-Ligation of PCGT-1(3k3), PCGST-1(3k3).
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+kan.
-Perform colony PCR of FRT-mcsm, Rox-mcsm, VL-mcsm, vox-mcsm.
-Pick correct colonies of FRT-mcsm, Rox-mcsm, VL-mcsm, vox-mcsm, and inoculate a culture of 4 ml LB medium containing chloromycetin.
-Perform colony PCR of RBS-mc-ssrA(strong).
-Pick correct colonies of RBS-mc-ssrA(strong), and inoculate a culture of 4 ml LB medium containing chloromycetin.
-We successfully ligated pBAD(reverse) with flpe-ECFP and we also got correct BBa_K1641214.
-We tried to induce BBa_K1641214 in DH5a and Top10 strains with L-arabinose in LB broth, then test the intensity of fluorescence
-Use Plasmid Miniprep Kit to isolate FRT-mcsm-3, Rox-mcsm-3, SLM-mcsm-4, VL-mcsm-4, vox-mcsm-2,VGC-3, PCGT(3k3), PCGST(3k3), mc-ss-2 from the cultures.
-Double digestion of ScGS-1, VGS-3, VcGS-4, FGS-1, DGS-1 and I0500. Extract double digestion products by using Gel Extraction Kit.
-Ligation of PDGS, PFGS, PVcGS, PVGS, PScGS.
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+cm.
-Perform colony PCR of PDGS, PFGS, PScGS, PVcGS.
-Pick correct colonies of PDGS, PFGS, PScGS, PVcGS, and inoculate a culture of 4 ml LB medium containing chloromycetin.
-Double digestion of RT-mcsm-3, Rox-mcsm-3, SLM-mcsm-4, VL-mcsm-4, vox-mcsm-2 and B0015. (B0015 digested with Xba I and EcoR I, fragment digested with Spe I and EcoR I). Extract double digestion products by using Gel Extraction Kit.
-Ligation of FRT-mcsm-T, Rox-mcsm-T, SLM-mcsm, VL-mcsm-T.
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+cm.
-Use Plasmid Miniprep Kit to isolate PDGS-2, PFGS-5, PVcGS-1, PVGS-4, p23101, B0015, loxp-mcsm-Inv-rep from the cultures.
-Double digestion of PDGS-2, PFGS-5, PVcGS-1, PVGS-4 and B1006. (B1006 digested with Xba I and EcoR I, fragment digest with Spe I and EcoR I). Extract double digestion products by using Gel Extraction Kit.
-Ligation of PDGST, PFGST, PScGST, PVcGST.
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+cm.
-Transform PCGT with loxp-mcsm-Inv-rep, PCGST with loxp-mcsm-Inv-rep, plate the transformed cells on LB+kan+cm.
-Use Plasmid Miniprep Kit to isolate FRT-mcsm-T, Rox-mcsm-T, SLM-mcsm, VL-mcsm-T from the cultures.
-Extract double digestion products by using Gel Extraction Kit.
-Ligation of P-FRT-mcsm-T(FRT-reporter), P-Rox-mcsm-T(Rox-reporter), P-SLM-mcsm(SLM-reporter), P-VL-mcsm-T(VL-reporter).
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+cm.
-Use Plasmid Miniprep Kit to isolate PDGST-5, PFGST-1, PScGST-4, PVcGST-1, VGS-4, VGS-5 from the cultures.
-Double digestion of PDGST-5, PFGST-1, PScGST-4, PVcGST-1 and pSB3K3. Extract double digestion products by using Gel Extraction Kit.
-Ligation of PDGST-5(3k3), PFGST-1(3k3), PScGST-4(3k3), PVcGST-1(3k3).
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+kan.
-pcr (add sequence):prefix-RBS-flpe-FC, FC-Mcherry-T7-frt-suffix from biobrick
-overlap pcr:RBS-flpe-FC-Mcherry
-Pick single colonies of PDGST-5(3k3), PFGST-1(3k3), PScGST-4(3k3), PVcGST-1(3k3), and streak them onto another plate to expand culture.
-Perform colony PCR of PDGST-5(3k3), PFGST-1(3k3), PScGST-4(3k3), PVcGST-1(3k3). Colony No. 2, 3, 4, 6 of PDGST-5(3k3), No. 2, 3, 4, 5 of PFGST-1(3k3), No. 1, 2, 3, 4, 5, 6 of PScGST-4(3k3), No. 3, 4, 5, 6 of PVcGST-1(3k3) are correct.
-Pick correct colonies of PDGST-5(3k3), PFGST-1(3k3), PScGST-4(3k3), PVcGST-1(3k3), and inoculate a culture of 4 ml LB medium containing kanamycin.
-Use Plasmid Miniprep Kit to isolate PDGST-5-2, PFGST-1-2, PScGST-4-1, PVcGST-1-4 from the cultures.
-Double digestion of PDGST-5-2, PFGST-1-2, PScGST-4-1, PVcGST-1-4 and pSB1K3 with EcoR I and Pst I. Extract double digestion products by using Gel Extraction Kit.
-Ligation of PDGST-5-2(1k3), PFGST-1-2(1k3), PScGST-4-1(1k3), PVcGST-1-4(1k3).
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+kan.
-Transform PDGST-2(3k3) with Rox-repoter, PFGST-4(3k3) with FRT-reporter, PScGST-1(3k3) with SLM-repoter, PVcGST-2(3k3) with vox-repoter, plate the transformed cells on LB+kan+cm.
-We detected ECFP in M9 broth!
-BBa_K1641225 correctly constructed!
-pcr (add sequence):prefix-flpe-FC, FC-Mcherry-T7TE-frt-suffix from biobrick
-overlap pcr:flpe-FC-Mcherry;overlap pcr worked
-assemble:Pbad-frt-RBS-flpe-FC-Mcherry-T7TE-frt in pSB1A2
-Use Plasmid Miniprep Kit to isolate cgs, vgs, gp38 from the cultures.
-Pick single colonies of PDGST-4(1k3), PFGST-4(1k3), PScGST-4(1k3), PVcGST-1(1k3) , and streak them onto another plate to expand culture.
-Perform colony PCR of PDGST-4(1k3), PFGST-4(1k3), PScGST-4(1k3), PVcGST-1(1k3). Colony No. 1, 2, 3, 4, 5, 6 of PDGST-4(1k3), No. 2, 4, 6, 7 of PFGST-4(1k3), No. 1, 2, 3, 4, 5, 6 of PScGST-4(1k3), No. 1, 2, 3, 4, 5, 6 of PVcGST-1(1k3) are correct.
-Pick correct colonies of PDGST-4(1k3), PFGST-4(1k3), PScGST-4(1k3), PVcGST-1(1k3), and inoculate a culture of 4 ml LB medium containing kanamycin.
-Use Plasmid Miniprep Kit to isolate PDGST(1k3), PFGST(1k3), PScGST(1k3), PVcGST(1k3), Test-mc-T, Test-rep, p23101, I0500, B1006 from the cultures.
-Double digestion of FRT-mcsm-T, Rox-mcsm-T, SLM-mcsm-T, VL-mcsm-T (Spe I and Pst I) and p23101(Xba I and Pst I). Extract double digestion products by using Gel Extraction Kit.
-Ligation of p-FRT-mcsm-T, p-Rox-mcsm-T, p-SLM-mcsm-T and p-VL-mcsm-T.
-Transform the ligation reactions into standard bacterial cloning cells(DH5α), plate the transformed cells on LB+cm.
-We conducted fluorescence measurements to confirmed whether BBa_K1641225 can work. We still tried to construct a full plasmid in Circuit 2.
-pcr (add sequence):prefix-loxP-RBS-Cre-FC, FC-eGFP-T7TE-loxP-suffix from biobrick
-overlap pcr:loxP-RBS-Cre-FC-eGFP-T7TE-loxP;overlap pcr worked
-assemble:Pbad-frt-RBS-flpe-FC-Mcherry-T7TE-frt-loxP-RBS-Cre-FC-eGFP-T7TE-loxP in pSB1A2
+ -
Method
1. Molecular cloning
Preparation of E. coli competent cell
Wild type E. coli DH5α, BL21 (DE3), or Top10 is inoculate on a LB plate and incubated in 37℃ overnight. Single colony is inoculated into 3 mL LB broth for overnight growth. This culture is 1:100 inoculated into a second 50 mL LB and let growth for 2-5 h until OD600 reach around 0.5. The culture is then ice-bathed, centrifuged and resuspended by 50 mL 10% glycerol with 0.1 M CaCl2 for twice, and all the way 4℃. Finally, the bacterial cells are resuspended into 7 mL solution of 10% glycerol with 0.1 M CaCl2, transferred into each EP tube for 120 μL, and stored at -80℃.
PCR
We use 3 kinds of DNA polymerase to deal with different situation, that is, Takara PrimeSTAR max (Cat. R045A), Vazyme Phanta max (Cat. P515), or Genstar Starmix (Cat. A112). The protocol refers to the instruction of corresponding enzyme. The product is purified through electrophoresis in 1.0-2.0% agarose gel and OMEGA Gel Extract Kit (Cat. D2500-02).
Preparation of Plasmid DNA
At least 4 mL culture of E. coli DH5α is gathered after 16-18 hours of growth in LB with proper antibiotics. The plasmid is purified through Genstar StarPrep Plasmid Miniprep Kit StarPrep (Cat. D201-01) or OMEGA Plasmid Mini Kid I (Cat. D6943-02).
Digestion
20 μL (about 1.5 μg plasmid or 1 μg PCR purified product) substrates are added into totally 50 reaction solution, with at least 1 μL each restriction enzyme (NEB). After incubation for at 37℃ for least 1.5h, the product is purified through electrophoresis in 1.0-2.0% agarose gel and OMEGA Gel Extract Kit (Cat. D2500-02).
Gel Purification
We use HiPure Gel Pure DNA Micro Kit (Cat. D2110-01) offered by Magen to purify DNA. It is suitable for a variety of routine applications including sequencing and ligation.
Ligation
The ligation is directed by 3A assembly. Add 25ng Fast DNA-Backbone, appropriate DNA-fragment (The amount of substance of DNA-fragment shall be 3-10 times as much as DNA-Backbone), and 1μL T4 Buffer 10 X, 1μL T4 ligase into a PCR tube. Then add MINI-Q water to ensure that the volume of the mix is 10μL. Incubate the tube at 4℃ overnight.
DNA Clean-Up
We used Genstar StarPre PCR & DNA Fragment Purification Kit to purify PCR products, and digestion products in “2A” assembly method.
2. Transformation
Yeast
Grow yeast cells overnight in YPD
Harvest cells by centrifugation
Resuspend cell pellet in 5 mL TE + 0.1 M lithium acetate and centrifuge.
Resuspend cell pellet in 2 mL TE + 0.1 M lithium acetate. Incubate at 30℃ for 1 h and rapidly chill on ice
In a tube, add Plasmid DNA for transformation, 0.2 mL cells treated with LiOAc from above and 1 mL 40% PEG 4000, 1XTE pH 7.5, 0.1 M Lithium acetate.
Mix by vortexing and incubate at 30 degrees for 30 min.
Heat shock cells for 15 min. at 42 degrees.
Shortly centrifuge and remove supernatant
Resuspend cell pellet in 1 mL TE
Shortly centrifuge and remove supernatant
Resuspend cell pellet in TE by vortexing.
Plate to selective plates.
E.coli
Transformation must be conducted at once after ligation is over. 20 μL ligation product or 1-2 μL plasmid is added to about 120 μL E. coli competent cell, and incubated on ice for at least 20 min. Transfer the solution into 42℃ and keep for 80s, and return to ice and keep for 3 min. Afterward, add 700 μL LB into each tube and shake for 220 rpm at 37℃, for 30-60 min. Finally, centrifuge and remove 600 μL LB, resuspend the bacterial and spread them on LB plates with proper antibiotics.
3. Real-time invertase dynamics measurement
Strain preparation
Inoculate a strain into 3 mL LB with 2 proper antibiotics (for us, Km and Cm) and cultivate them overnight. This culture is 1:100 inoculated into a second 8 mL LB (in Φ18mm tube), and pre-shaking for 2 h 30 min.
Induction
For T7-lacO promoter in BL21 (DE3), add IPTG at final concentration of 125nM; for PBAD promoter in Top10, add arabinose at final concentration of 1mM. The measurement starts from this point.
Sampling & measurement
At each sampling time point, transfer 200 μL into a well of 96-well plate, 2 parallel tubes for each strain. When adding samples finished, immediately return the tube to the shaker. For each well we read OD 700, OD 600, eGFP fluorescence 480/515, and mCherry fluorescence 580/615 using a multi-function plate reader. Usually we pick sample and measure the data at 0 h, 1 h, 2 h, 3 h, 4.5 h, 6 h, 8 h, 10 h, 12 h, 15 h, 19 h, and 24 h for each strain.
Data process
The background value of LB is subtracted from each well, then average value of OD 700, OD 600 (usually useless for us), eGFP RFU, and mCherry RFU for each sample is calculated. Finally, we use RFU per OD 700 as data to describe the dynamics of invertase in certain number of bacteria. This data is further analyzed by our modeling work.
Measuring fluorescence with microplate reader
Colony PCR for positive check
Prepare Taq PCR solution without any template. Add 15 μL such solution to each PCR tube and pick a tiny spot of a colony and dip it into the PCR tube and then start PCR. For this method, the PCR pre-denature step (more than 94℃) should be more than 5 min.
4. Genomic DNA extraction from yeast (Omega bio-tek).
Grow yeast culture in YPD medium and harvest by centrifugation at 4000x g for 10 min at room temperature.
Discard medium and resuspend cells in 480 μL Buffer SE, 10 μL 2-mercaptoethanol and 20 μL lyticase solution. Incubate at 30℃ for at least 30 min.
Pellet spheroblast by centrifuging 5 min at 4000x g at room temperature.
Add 200 μL Buffer YL and 50 mg glass beads to the sample. Vortex at max speed for 3-5 min. Let it stand to allow the beads to settle. Transfer supernatant to a new 1.5 mL centrifuge tube.
Add 25 μL proteinase K solution and vortex to mix well. Incubate at 65℃ in a shaking water bath for 30 min.
Add 5 μL RNase A to the sample and invert tube several times to mix. Incubate at room temperature for 10 min.
Add 220 μL Buffer YDL and 220 μL absolute ethanol to the sample and mix thoroughly by vortexing at maxi speed for 20s.
Assemble a HiBind DNA Mini Column in a 2 mL collection, Transfer the entire sample from Step 8 into the column, including any precipitate that may have formed. Centrifuge at 10000x g for 1 min to bind DNA. Discard the collection tube and filtrate.
Place the column into a second 2 mL tube and wash by adding 500 μL buffer HB. Centrifuge at 10000x g for 1 min. Discard flow-through and reuse the collection tube.
Place the column into the same collection tube and wash by adding 700 μL DNA Wash Buffer diluted with ethanol. Centrifuge at 10000x g for 1 min. Discard flow-through and reuse the collection tube.
Wash the column with a second 700 μL DNA Wash Buffer and centrifuge as above.
Using the same 2 mL collection tube, centrifuge HiBind DNA Mini Column at maxi speed for 2 min to dry the column.
Place the column into nuclease-free 1.5 mL microfuge tube and add 50-100 μL of preheated Elution Buffer to HiBind DNA Mini Column matrix.
To elute DNA from the column, centrifuge at 10000x g for 1 min.
5. Site specific mutation
Program:
Add 1μL Dpn I and cultured in 37℃ for 1h.Transform 10 μL product into 50 μL Top10 (competent cell) and cultivate in 37℃ overnight. Select monoclonal colony for plasmid extraction.
6. End-filling with S1 nuclease
Prepare 10x S1 nuclease buffer:
300 mM Sodium acetate, pH 4.6
2800 mM NaCl
10 mM ZnSO4
Selective degradation of single-stranded DNA:
incubate at 23°C, 15min
Add EDTA
Ligation with T4 DNA polymerase:
Incubate at 22℃ for 1 h
Incubate at 65℃ for 10 min to inactivate the enzyme
Transformation and plasmid extraction.
7. MSC modification of a plasmid
Double enzyme digestion (HF，NEB)
Incubate at 37℃ for 2 h
Purify digestion product with Gel extraction kit (Tiangen)
Anneal of MCS oligonucleotide (MCS anneal product )
Ligation
Incubate at 22℃ for 1h
Transformation and plasmid extraction.
8. Integration of DNA fragment into yeast genome
Insert DNA fragment into pAUR135 vector
Transform yeast cells by acetate lithium method
Culture in YPD medium for more than 6h and spread onto YPD selective medium plate containing AbA
Select resistance marked yeast cells.
9. Strain storage
A strain of overnight culture can be stored by adding glycerol at a final concentration of 15-20% and keeping them at -80℃.
Parts
Typical parts we constructed
Typical parts in Real-time invertase dynamics testing system.
A. Parts of novel Cre-like invertase CDS. (e.g. Dre, BBa_K1641002 );
B. Parts of RTS corresponding to novel invertases (e.g. Rox, BBa_K1641010 );
C. Parts of pInv-gen, the genertase of invertases. (e.g. pInv-gen-DGS, BBa_K1641040 );
D. Parts of pInv-rep, the invertase activity reporter. (e.g. BBa_K1641029 ).
BBa_K1641900 : bxb1gp35 is a modified sequence derived from the eukaryote recombinase bxb1 which has no standard exonuclease sites
BBa_K1641999 : We transformed a commercial plasmid pAUR135 into a standard biobrick shuttle vector, which contains 4 standard exonuclease sites and can use for fragment integration into yeast chromosome.
Parts List
Part No.
Name
Cat.
Backbone
Description
BBa_K1641000
Vcre
Basic_Coding
pSB1C3
Recombinase Vcre
BBa_K1641001
Scre
Basic_Coding
pSB1C3
Recombinase Scre
BBa_K1641002
Dre
Basic_Coding
pSB1C3
Recombinase Dre
BBa_K1641003
Vika
Basic_Coding
pSB1C3
Recombinase Vika
BBa_K1641004
RBS-CGS
Basic_Translational Unit
pSB1C3
Fusion protein of Cre-EGFP-ssra
BBa_K1641005
RBS-FGS
Basic_Translational Unit
pSB1C3
Fusion protein of Flp-EGFP-ssra
BBa_K1641006
RBS-VcGS
Basic_Translational Unit
pSB1C3
Fusion protein of Vcre-EGFP-ssra
BBa_K1641007
RBS-ScGS
Basic_Translational Unit
pSB1C3
Fusion protein of Scre-EGFP-ssra
BBa_K1641008
RBS-DGS
Basic_Translational Unit
pSB1C3
Fusion protein of Dre-EGF-ssra
BBa_K1641009
RBS-VGS
Basic_Translational Unit
pSB1C3
Fusion protein of Vika-EGFP-ssra
BBa_K1641010
Rox
Basic_DNA
pSB1C3
Rox
BBa_K1641011
Vox
Basic_DNA
pSB1C3
Vox
BBa_K1641012
VLoxP
Basic_DNA
pSB1C3
VLoxP
BBa_K1641013
SLoxM1
Basic_DNA
pSB1C3
SLoxM1
BBa_K1641014
(rev)2lox-RBS-mcsm
Basic_Translational Unit
pSB1C3
Up-side-down mcherry-ssra with LoxP and RBS
BBa_K1641015
(rev)2FRT-RBS-mcsm
Basic_Translational Unit
pSB1C3
Up-side-down mcherry-ssra with FRT and RBS
BBa_K1641016
(rev)2Rox-RBS-mcsm
Basic_Translational Unit
pSB1C3
Up-side-down mcherry-ssra with Rox and RBS
BBa_K1641017
(rev)2Vox-RBS-mcsm
Basic_Translational Unit
pSB1C3
Up-side-down mcherry-ssra with Vox and RBS
BBa_K1641018
(rev)2SLM-RBS-mcsm
Basic_Translational Unit
pSB1C3
Up-side-down mcherry-ssra with SLoxM1 and RBS
BBa_K1641019
(rev)2VL-RBS-mcsm
Basic_Translational Unit
pSB1C3
Up-side-down mcherry-ssra with VloxP and RBS
BBa_K1641022
(rev)2lox-RBS-mc
Basic_Translational Unit
pSB1C3
Up-side-down mcherry with LoxP and RBS
BBa_K1641023
101Lox-M
Composite_Reporter
pSB1C3
pInv-rep of Cre-loxP with J23101
BBa_K1641024
100Lox-M
Composite_Reporter
pSB1C3
pInv-rep of Cre-loxP with J23100
BBa_K1641025
106Lox-M
Composite_Reporter
pSB1C3
pInv-rep of Cre-loxP with J23106
BBa_K1641026
110Lox-M
Composite_Reporter
pSB1C3
pInv-rep of Cre-loxP with J23110
BBa_K1641027
116Lox-M
Composite_Reporter
pSB1C3
pInv-rep of Cre-loxP with J23116
BBa_K1641028
101FRT-M
Composite_Reporter
pSB1C3
pInv-rep of Flp-FRT with J23101
BBa_K1641029
101Rox-M
Composite_Reporter
pSB1C3
pInv-rep of Dre-Rox with J23101
BBa_K1641030
101SLM-M
Composite_Reporter
pSB1C3
pInv-rep of Vika-Vox with J23101
BBa_K1641031
101Vox-M
Composite_Reporter
pSB1C3
pInv-rep of Scre-SloxM1 with J23101
BBa_K1641032
101Vlox-M
Composite_Reporter
pSB1C3
pInv-rep of Vcre-VloxP with J23101
BBa_K1641035
101Lox-O
Composite_Reporter
pSB1C3
pInv-rep of Cre-loxP with J23101 (no ssra)
BBa_K1641036
1C3-CGS
Composite_Generator
pSB1C3/pSB1K3
"pInv-Gen
BBa_K1641037
1C3-FGS
Composite_Generator
pSB1C3/pSB1K3
"pInv-Gen
BBa_K1641038
1C3-VcGS
Composite_Generator
pSB1C3/pSB1K3
"pInv-Gen
BBa_K1641039
1C3-ScGS
Composite_Generator
pSB1C3/pSB1K3
"pInv-Gen
BBa_K1641040
1C3-DGS
Composite_Generator
pSB1C3/pSB1K3
"pInv-Gen
BBa_K1641041
1C3-VGS
Composite_Generator
pSB1C3/pSB1K3
"pInv-Gen
BBa_K1641042
RBS-CG
Basic_Translational Unit
pSB1C3
Fusion protein of Cre-EGFP
BBa_K1641043
1C3-CG
Composite_Generator
pSB1C3/pSB1K3
"pInv-Gen
BBa_K1641200
pBAD(forward)-FRT(forward)-LoxP(reverse)
Composite
N/A
pBAD forward promoter+recombinase Flpe recognition site (forward)+recombinase Cre recognition site (reverse)
BBa_K1641201
pBAD(forward)-FRT(forward)-loxP(reverse)-pBAD(reverse)
basic_DNA
N/A
pBAD forward promoter+recombinase Flpe recognition site (forward)+recombinase Cre recognition site (reverse)+pBAD reverse promoter
BBa_K1641202
FRT(forward)-LoxP(reverse)-pBAD(reverse)
basic_DNA
N/A
recombinase Flpe recognition site (forward)+recombinase Cre recognition site (reverse)+pBAD reverse promoter
BBa_K1641203
pBAD(forward)-FRT(forward)-pBAD(reverse)
basic_DNA
N/A
pBAD forward promoter+recombinase Flpe recognition site (forward)+pBAD reverse promoter
BBa_K1641204
FRT(forward)-pBAD(reverse)
basic_DNA
N/A
recombinase Flpe recognition site (forward)+pBAD reverse promoter
BBa_K1641205
FRT(reverse)-pBAD(reverse)
basic_DNA
pSB1K3
recombinase Flpe recognition site (reverse)+pBAD reverse promoter
BBa_K1641206
FRT(reverse)-loxP(forward)-pBAD(reverse)
basic_DNA
pSB1C3
recombinase Flpe recognition site (reverse)+recombinase Cre recognition site (forward)+pBAD reverse promoter
BBa_K1641207
loxP(reverse)
basic_DNA
N/A
recombinase Cre recognition site (reverse)
BBa_K1641208
loxP(reverse)-BBa_E0840
Composite
pSB1C3
composite part: K1641208+E0840
BBa_K1641209
flpe-ECFP
Composite
pSB1C3
composite part: K313002+E0422
BBa_K1641210
mCherry-loxP(reverse)-GFP
Composite
N/A
composite part: J06602+K1641208+E0840
BBa_K1641211
RBS-flpe-ssrA(M0051)
basic_Coding
pSB1C3
composite part: B0034+K313002+M0051
BBa_K1641212
B0034-Cre-ssrA
basic_Coding
pSB1K3
composite part: B0034+cre+M0052
BBa_K1641213
RBS-mCherry-B1006
Composite
pSB1C3
mCherry with B0034 and B1006
BBa_K1641214
FRT(reverse)-pBAD(reverse)-Cre
Composite
pSB1K3
FRT(reverse) pBAD(reverse) Cre
BBa_K1641215
pBAD(forward)-FRT(forward)
Composite
pSB1A3
pBAD FRT
BBa_K1641216
pBAD(forward)-B0034
Composite
pSB1C3
pBAD B0034
BBa_K1641217
killerred-B0015
Composite
pSB1C3
killerred B0015
BBa_K1641218
FRT(reverse)
basic_DNA
N/A
FRT(reberse)
BBa_K1641219
Standard for qPCR
basic_DNA
N/A
qPCR standard
BBa_K1641220
pBAD(forward)-FRT(forward)-loxP(reverse)
basic_DNA
pSB1A3
pBAD FRT loxP(reverse)
BBa_K1641221
FRT(reverse)-loxP(forward)
basic_DNA
pSB1K3
pBAD forward promoter+recombinase Flpe recognition site (forward)
BBa_K1641222
flpe-ssrA
basic_Coding
pSB1C3
RBS-flpe-ssrA
BBa_K1641223
Cre-ssrA
basic_Coding
N/A
RBS-Cre-ssrA
BBa_K1641224
mCherry-ssrA
basic_Coding
N/A
RBS mCherry M0052
BBa_K1641225
Invertible element
Composite
pSB1K3
Invertible device in Circuit 2.
BBa_K1641900
bxb1gp35 RFC25 optimized
Basic_Coding
pSB1C3
"A ser reombinase bxb1gp35 optimized for RFC25
BBa_K1641999
pAUR135-RFC10 optimized
Backbone
--
Yeast-bacteria shuttle vector optimized for RFC 10
Recombinase library
Recombinase is the basic element of our design of Micro-timer. Each recombinase attacks its specific recombination target sites (RTSs) and cause the recombination between them. In our project, for the construction of counting unit with different timer span, we found a collection of recombinase and its RTSs (shown as followed). Find more choices of recombinase !
recombinase
RTS
Part.No
Name
Part.No
Name
BBa_K1641000
Vcre
BBa_K1641013
VLoxP
BBa_K1641001
Scre
BBa_K1641013
SLoxM1
BBa_K1641002
Dre
BBa_K1641010
Rox
BBa_K1641003
Vika
BBa_K1641011
Vox
BBa_K1641222
flpe-ssrA
BBa_K1641218
FRT(reverse)
BBa_J61020
FRT(forward)
BBa_K1641223
Cre-ssrA
BBa_K1641207
loxP(reverse)
BBa_K1641046
LoxP(forward)
recombinase
Educt Site
Part.No
Name
Name
Sequence
BBa_K1641900
bxb1gp35
attB(forward)
TCGGCCGGCTTGTCGACGACGGCGGTCTCCGTCGTCAGGATCATCCGGGC
attP(reverse)
GGGTTTGTACCGTACACCACTGAGACCGCGGTGGTTGACCAGACAAACCACGA
▲
Sponsor
Name: SYSU-China   School: Sun Yat-sen University
Address: No. 135, Xingang Xi Road, Guangzhou, 510275, P. R. China
Contact: nichy5@mail2.sysu.edu.cn
