<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SYSU_CHINA_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SYSU CHINA/Design</SPAN></H1><DIV id="bodyContent"><DIV class="sui-page-content" style="width: 100%;background-color:#fff;padding-bottom: 0;"><DIV class="main-wrapper"><UL class="sui-menu"><LI style="background-color: #D7E79F;"><A style="color: #fff;" href="/Team:SYSU_CHINA/Project">Project</A></LI><LI style="background-color: #FFE075;"><A style="color: #fff;" href="/Team:SYSU_CHINA/Design">Design</A></LI><LI style="background-color: #FFB436;"><A style="color: #fff;" href="/Team:SYSU_CHINA/Result">Result</A></LI><LI style="background-color: #EF9870;"><A style="color: #fff;" href="/Team:SYSU_CHINA/Safety">Safety</A></LI><LI style="background-color: #E76B56;"><A style="color: #fff;" href="/Team:SYSU_CHINA/Note">Note</A></LI><LI style="background-color: #B63232;"><A style="color: #fff;" href="/Team:SYSU_CHINA/Practices">Human Practice</A></LI><LI style="background-color: #363D68;"><A style="color: #fff;" href="/Team:SYSU_CHINA/Team">Team</A></LI></UL><DIV class="sui-content"><DIV class="sui-side-nav"><UL><LI><A href="#Matching-and-Testing">Matching and Testing</A></LI><LI><A href="#Prokaryotic-Timer">Prokaryotic Timer</A></LI><LI><A href="#Telomeric-Timer">Telomeric Timer</A></LI><LI><A href="#Eukaryotic-Timer">Eukaryotic Timer</A></LI><LI><A href="#Reference">Reference</A></LI></UL></DIV><DIV class="sui-article"><DIV id="Matching-and-Testing" class="scrollto"><H1>Matching and Testing</H1><H3>Introduction of purpose</H3><P>The basic idea of Micro-time system is to separate a long-period timing into small invertase device modules, and through appropriate combination of them, we can obtain a wide range of aimed time length for users to choose. However, for both E. coli and yeast, a successful timer must be based on precise definition and measurement of “time unit” – how long each invertase module exactly represents. Hence, the major consideration of our testing group is to measure the time unit for different invertase modules, and provide a systematic solution with optimized synthetic elements to gain a Micro-timer for any length of time (see Fig-T1).</P><P class="figure">Fig-T-1:The mission of testing and optimization group. We design different invertase modules, and fathom into their dynamics, providing valuable information to optimize our timing system.</P><H3>System construction</H3><P>The real-time invertase dynamics testing system contains two different plasmids in E. coli (see Fig-T2). The first one is an invertase generation vector, namely pInv-gen, that produces invertase-EGFP fusion protein through induction. The second one is called pInv-rep, a reporter vector that produce mcherry signal to indicate the inversion successfully happens. The invertase-EGFP on pInv-gen is controlled by an inducible promoter (T7-LacO promoter or Pbad). The target sequence (RTS) of corresponding invertase locates in the pInv-rep, surrounding a mcherry gene which is yet upside-down and transcribed by a constructive promoter (e.g. BBa_J23101). This mcherry-coding sequence can be inverted and restored to 5’ – 3’ direction at the existence of Cre-GFP, rendering red signal. Additionally, an ssra tag that intensifies the protein degradation may be fused to the C-terminus of invertase-EGFP and mcherry to be in tune with our final device that aims to clean up the redundant invertase not participating in a second round inversion.</P><P class="figure">
           Fig-T-2: The construction of our real-time invertase dynamics testing system. A bacteria containing two vectors, one expressing invertase when induced and another as target and reporter. 
         </P><P>Once if the inducer is added into the culture, the green fluorescence will increase at first due to the expression of invertase-EGFP. Then, the red fluorescence is generated because the Cre-EGFP restores the reversed mcherry sequence (see Fig-T-3). The length of interval between green and red indicates the corresponding single timing length of the invertase module. In our study, the variants to render different time length are invertase itself, promoter, and the degrading rate by ssra. Specifically, the activity level of invertase directly determines the time need to invert most of pInv-reps, and the promoter decides the rate of generation of invertase, which also contribute sigfificant to the speed of module. The ssra-tag, on the contrary, reduces the speed of inversion while effectively inhibiting the leakage expression when inducer is not in the culture.</P><P class="figure">
          Fig-T-3: A typical pattern of expression of both Cre-EGFP fusions and mcherry in reporter. When inducer is added into the culture, the green signal begin to accumulate, and when its product – restored mcherry CDS – is enough to reach the resolution of plate reader, red signal can be detected. K1, time of 1/2 max increasing rate of mcherry; K2, time of max increasing rate of mcherry; K3, beginning of plateau phase of red signal; K4, beginning of plateau phase of green signal.
        </P><H3>Achievement</H3><P>We uses this system to measure totally 21 pairs of different combination of pInv-gens and pInv-reps. There are 6 different invertases we have tested using the Real-time system. While Cre and Flp are most commonly used recombinases in Biobrick plates, we newly contributed 4 brand-new invertases: Dre, Vcre, Scre, and Vika, all of which are Cre-family recombinases with different and non-intervolving RTS. We successfully proved that all these invertase work pretty good in our system, which you can see in <A href="">RESULTS</A>. All of the data we gather are analyzed by modeling group to render its corresponding time length. This work could guide other groups for their final design.</P><P class="figure">Fig-T-4: All recombinases (invertases) used in this study. Each invertase has their own recognition sites and will not interfere with one another.</P></DIV><DIV id="Prokaryotic-Timer" class="scrollto"><H1>Prokaryotic Timer</H1><H3>Introduction</H3><P>  A report from Science, by which we were inspired, tries to explain that synthetic gene networks can be constructed to emulate a cellular counter that would enable complex synthetic programming and a variety of biotechnology applications.</P><P>One of the figures from this article, with introduction of Single Invertase Memory Module (SIMM), indicates how genes can work in a counting system by flipping of recombinases. Two recombinases in the circuit, Flpe and Cre, in conjunction with their specific targeting sites, FRT and loxP, accomplishes the whole flipping process in a plasmid they call DIC 3-Counter (Fig-P-1A, 1B) . We made a slight improvement on the circuit mentioned above and we call it circuit 1 (Fig-P-1C) , which we construct to verify its feasibility(An analyzing tool Snapgene was used to export plasmid figures).</P><P class="figure">Fig-P-1 Plasmid construction and circuit mechanism. (A) DIC 3-Counter constructed by Ari E. Friedland et. al. (B) Mechanism of SIMM. (C)Circuit 1 plasmid.</P><P> We noted that stage 1 might not be robust, for the residual flpe might make Stage 1 plasmids return to Stage 0. To solve this problem, we design that our micro-timer should be placed on a low-copy plasmid, with strong Promoter, RBS and degradation tags (Fig-P-2). Hence, recombinases and reporters can be strongly expressed and fast turned over, which makes our system more robust.</P><P class="figure">Fig-P-2 Three stages of Circuit 2 

A. Stage 0 In this stage, flpe and ECFP express, while no Cre, mCherry and GFP can express without a promoter. Only ECFP signal can be detected.

B. Stage 1 When the concentration of flpe reaches a threshold, sequence between two FRTs can be flipped. In this flipping, a pBAD change its position and initiates the expression of Cre and mCherry.  In this stage, we can find a decline of ECFP signal and an increase of mCherry signal.

C. Stage 2 Similar to Stage 1, Cre accumulates, reaches a threshold, and finally triggers a second flipping. This flipping reverses the sequence between two loxPs. After this flipping, GFP expresses robustly for the turn-off of flpe and Cre expression.
</P><P>  Circuit 2 continues transcripting and flipping in a circulation once induced. It happens theoretically because it may be bothered by objective resistance, but it provides us with a possibility to time gene reaction and control certain protein expression in a time scale.</P><P>Circuit 2 can transfer certain DNA sequences unit after unit. Imagine if there is a target gene between the first FRT and loxP in the initial phase, it would pass continuously unit after unit.</P><H3> Construction </H3><P>  For circuit 2, we added florescent protein ECFP (BBa_E0422) and mCherry (BBa_J06602) right in the downstream of the ssrA-tag of recombinase gene flpe and cre, respectively, for enhanced sensitivity and robust of the system. And we add a final GFP (BBa_E0840) as a reporter. pSB1C3 was used as vector for cloning and we tried to transfer the entire circuit to pSB3K3 in order to test its viability (Fig-P-3) .</P><P class="figure">Fig-P-3 Circuit 2 plasmid.</P><P> Worth of attention, we developed a method of &quot;2A&quot; assembly, based on 3A assembly, by using DNA clean-up kit to clear small fragments less than 75bp (principally fragments between EcoRI and XbaI, or SpeI and PstI) after digestion. As a result of facing with inaccessibility through 3A assembly during our experiment, we managed to join target gene segments directly to upstream or downstream of another plasmid, which should have been abandoned already in 3A assembly(Fig-P-4) . In this method, we could get more well-ligated products as highly purified linearized backbones were dispensable, although results of &quot;2A&quot; assembly relied on the quality of DNA clean-up kits.</P><P class="figure">Fig-P-4 Process of “2A” assembly.</P><H3>Testing</H3><P>Experimental proof must be accomplished after construction and circuits can be tested by the methods below.
</P><P>1.Measurement of the intensity of fluorescence of ECFP, mCherry and GFP. The result, from which we infer that flipping actually happens, exhibit an expression of each fluorescence with a rise and then a decrease in different time scale (Fig-P-5).</P><P>Analyse by Real-time Quantitative PCR (qPCR) (Fig-P-6A). With  primers designed, shown in the picture , we can get 3 different kinds of amplification curves, whose tendency presented may almost be the same as the fluorescence intensity.</P><P>Digestion (Fig-P-6B, 6C) . When sequences flip, certain restriction endonuclease cutting sites remain constant while what have, in fact, changed, are their locations. Gene length can be altered when sequences flip, which can be visualized in an electrophoretic way.</P><P class="figure">Fig-P-5 Simulation of the result we planned to test on the intensity of fluorescences.</P><P class="figure">Fig-P-6 Testing by qPCR and digestion. (A) Ideograph for theoretical model of qPCR testing. Parallel primers were designed at the initial phase and PCR product would produce after flipping. (B) Ideograph for theoretical model of digestion testing. (C) Simulated agarose gel for digestion in EcoRV of different stages of circuit 2.
</P></DIV><DIV id="Telomeric-Timer" class="scrollto"><H1>Telomeric Timer: Micro-timer 2.0</H1><P>Telomeric timer，also called Micro-timer 2.0, a telomere-like device, the recombinases are flanked with two identically oriented recombination target sites (RTSs), in which the recombination of two RTSs will lead to deletion of the intervening sequence. As was shown in Fig-Y-1, the termination was included in each deletion unit. In state 0 (before the cell division), none of the signal will be expressed. The integrated sequence remains intact.</P><P class="figure">Fig-Y-1:A three-step Micro-timer 2.0. In each deletion unit, a recombinase is fused with a reporter, followed by a transcriptional terminator. </P><P>At the first cell division (see Fig-Y-2), the cell cycle specific promoter initiates the expression of recombinase 1 fused with mCherry, which can induce the recombination between two FRT sites. Consequently, the specific sequence flanked by two FRT will be deleted, forming the following state 1.</P><P class="figure">Fig-Y-2: Intervening sequence between two equally oriented FRTs is removed by the recombination process. </P><P> In state 1 (see Fig-Y-3), the terminator 1, which can block the transcription of cre-EGFP RNA in the former state, has been deleted.</P><P class="figure">Fig-Y-3: The state before the second initiation of the promoter.</P><P> At the second cell division (see Fig-Y-4), the expression of recombinase 2 fused with EGFP will induce the recombination between two LoxP sequences, and the sequence flanked by two direct LoxP will be deleted.</P><P class="figure">Fig-Y-4: Intervening sequence between two equally oriented LoxPs is removed by the recombination process.</P><P> In state 2 (see Fig-Y-5), because the terminator 2, which can block the transcription of mCherry in the former state, has been deleted, the cell cycle specific promoter can trigger the expression of YFP at the third cell division. If we substitute the mCherry gene with cell toxin gene, the initiation of cell cycle specific promoter can induce cell death.</P><P class="figure">Fig-Y-5: The final state of the telomere-like device, in which the reporter or cell toxins can be expressed during the next initiation of the promoter.</P></DIV><DIV id="Eukaryotic-Timer" class="scrollto"><H1>Eukaryotic Timer: Micro-timer 3.0</H1><P> In the design of Micro-timer 3.0 (See Fig-Y-6), each Eukaryotic timer integrase motif (EIM) contains a reporter, a promoter flanked with two inversely oriented educt site attB and attP, and an Ser family integrase gene. The recombination between two educt sites will yield two product sites attL and attR because the sequence of recombinase binding elements of attB and attP was different. Without its excisionase, the integrase can’t catalyze the inversion between two product sites.
</P><P>In state 0, all the signals are expressed when promoters fire. And when we add inducer to express integrase1, the system starts. In state 1, integrase 1 is produced to attack target site 1, inverting the attB/P flanking promoter, forming the state 2. In state 2, signal 1 was silenced whereas integrase 2 can be expressed. And to state n, if we do not want it to work any longer, we can easily let it die. But we can add another inducer, producing excisionases to restore the orientation of target sites. The system can go back to state 0 and start over again after a short period.</P><P class="figure">Fig-Y-6: Construction of Micro-timer 3.0. The reporter gene is at the downstream of the transcription direction. Before the system starts, the initiating promoter would express reporter gene rather than integrase gene. A “set” order drives the expression of the first integrase to revert the first target site, and with every initiation of the promoter, each target site can be reverted sequentially, and a “reset” order can drive the system back to state 0.</P></DIV><DIV id="Reference" class="scrollto"><H1>Reference</H1><P>[1] Friedland A E, Lu T K, Wang X, et al. Synthetic gene networks that count[J]. science, 2009, 324(5931): 1199-1202.
[2] Buchholz F, Angrand P O, Stewart A F. Improved properties of FLP recombinase evolved by cycling mutagenesis[J]. Nature biotechnology, 1998, 16(7): 657-662.
[3] Xu Z, Thomas L, Davies B, et al. Accuracy and efficiency define Bxb1 integrase as the best of fifteen candidate serine recombinases for the integration of DNA into the human genome[J]. BMC biotechnology, 2013, 13(1): 87.
[4] Buchholz F, Ringrose L, Angrand P O, et al. Different thermostabilities of FLP and Cre recombinases: implications for applied site-specific recombination[J]. Nucleic acids research, 1996, 24(21): 4256-4262.
[5] Bai Flagfeldt D, Siewers V, Huang L, et al. Characterization of chromosomal integration sites for heterologous gene expression in Saccharomyces cerevisiae[J]. Yeast, 2009, 26(10): 545-551.
[6] Xu Q, Reed J C. Bax inhibitor-1, a mammalian apoptosis suppressor identified by functional screening in yeast[J]. Molecular cell, 1998, 1(3): 337-346.
[7] Lewis J A, Hatfull G F. Control of directionality in integrase-mediated recombination: examination of recombination directionality factors (RDFs) including Xis and Cox proteins[J]. Nucleic acids research, 2001, 29(11): 2205-2216.
[8] Anastassiadis K, Fu J, Patsch C, et al. Dre recombinase, like Cre, is a highly efficient site-specific recombinase in E. coli, mammalian cells and mice[J]. Disease models &amp; mechanisms, 2009, 2(9-10): 508-515.
[9] Suzuki E, Nakayama M. VCre/VloxP and SCre/SloxP: new site-specific recombination systems for genome engineering[J]. Nucleic acids research, 2011, 39(8): e49-e49.
[10] S, Bode J. Site-specific recombinases: from tag-and-target-to tag-and-exchange-based genomic modifications[J]. The FASEB Journal, 2011, 25(12): 4088-4107.
[11] Bonnet J, Subsoontorn P, Endy D. Rewritable digital data storage in live cells via engineered control of recombination directionality[J]. Proceedings of the National Academy of Sciences, 2012, 109(23): 8884-8889.
[12] O’Brien E L, Van Itallie E, Bennett M R. Modeling synthetic gene oscillators[J]. Mathematical biosciences, 2012, 236(1): 1-15.
[13] Karimova M, Abi-Ghanem J, Berger N, et al. Vika/vox, a novel efficient and specific Cre/loxP-like site-specific recombination system[J]. Nucleic acids research, 2013, 41(2): e37-e37.
</P></DIV></DIV><DIV class="to-top-btn"><BUTTON>▲</BUTTON></DIV></DIV></DIV></DIV><DIV style="background-color: #FFE075; color:#fff; height: 240px;padding-top:20px;text-align: center; padding-bottom: 30px;"><H5 style="text-align: center; margin-bottom: 30px;">Sponsor</H5></DIV><DIV style="background-color:#444; color: #fff; height: 70px;text-align: center;padding-top: 10px;">
Name: SYSU-China   School: Sun Yat-sen University
Address: No. 135, Xingang Xi Road, Guangzhou, 510275, P. R. China
Contact: nichy5@mail2.sysu.edu.cn
</DIV></DIV></DIV></DIV></BODY></HTML>