<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SYSU_CHINA_Result skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SYSU CHINA/Result</SPAN></H1><DIV id="bodyContent"><DIV class="sui-page-content" style="width: 100%;background-color:#fff;padding-bottom: 0;"><DIV class="main-wrapper"><UL class="sui-menu"><LI style="background-color: #D7E79F;"><A style="color: #fff;" href="/Team:SYSU_CHINA/Project">Project</A></LI><LI style="background-color: #FFE075;"><A style="color: #fff;" href="/Team:SYSU_CHINA/Design">Design</A></LI><LI style="background-color: #FFB436;"><A style="color: #fff;" href="/Team:SYSU_CHINA/Result">Result</A></LI><LI style="background-color: #EF9870;"><A style="color: #fff;" href="/Team:SYSU_CHINA/Safety">Safety</A></LI><LI style="background-color: #E76B56;"><A style="color: #fff;" href="/Team:SYSU_CHINA/Note">Note</A></LI><LI style="background-color: #B63232;"><A style="color: #fff;" href="/Team:SYSU_CHINA/Practices">Human Practice</A></LI><LI style="background-color: #363D68;"><A style="color: #fff;" href="/Team:SYSU_CHINA/Team">Team</A></LI></UL><DIV class="sui-content"><DIV class="sui-side-nav"><UL><LI><A href="#Matching-and-Testing">Matching and Testing</A></LI><LI><A href="#Prokaryotic-Timer">Prokaryotic Timer</A></LI><LI><A href="#Telomeric-Timer">Telomeric Timer</A></LI><LI><A href="#Eukaryotic-Timer">Eukaryote Timer</A></LI><LI><A href="#Modelling">Modelling</A></LI></UL></DIV><DIV class="sui-article"><DIV id="Matching-and-Testing" class="scrollto"><H1>Matching and Testing</H1><H3>The dynamics pattern of each pair of pInv-gen and pInv-rep</H3><P class="figure">Fig-T-5: The elements of our concern in invertase module. Through changing such components we wish to understand their property and illustrate an optimized combination of them.</P><P> We co-transform pairs of pInv-gen and pInv-rep into E. coli BL21(DE3) or Top10, and have measured till now 13 different combinations them (see Table-T-1). A detailed pattern of relationships between time and RFU/OD can be hence revealed. Most of our data indicate the RFU/OD – time graphs shares a similar pattern (See Fig-T-6). The signal of Cre fused with EGFP stably increase after induction, perhaps a linear relationship; the mcherry signal, however, resembles an S-type curve that show a tiny or no growth and suddenly erupt a period after induction. Later, the increasing rate damped and the curve moves to a plateau phase. But technically, each pattern is slightly different because the molecular element of in this pathway varies from each other. </P><P>There are 3 major variants in this study, invertase (itself), promoter, and the ssra-mediated degradation, and additionally the fusion site of EGFP onto invertase does significantly count (See Fig-T-5). Using a series of combination of these variants we can understand their effect on invertase module. </P><P class="figure">Table-T-1: A detailed list of information of each pInv-rep and pInv-gen in this study.</P><P class="figure">Fig-T-6: The measured dynamics pattern of all 13 combinations of pInv-gen and pInv-rep. For relationship of group name and there corresponding device, please check Table-T-1.</P><H3>The properties of elements in invertase module</H3><P>1.Promoter intensity is positively-correlated to expression.</P><P>It is easy to understand that the higher intensity of a promoter, the better it performs to initiate transcription. We tested 5 constructive promoters, that is, J23100, J23101, J23106, J23110, and J23116, on pInv-rep (see Table-T-1 and Fig-T-7). On the one hand, the plateau phase RFU for different promoter is positively-correlated with its promoter intensity; on the other hand, another important value, the length of time from adding IPTG to burst of expression rate, is negatively-correlated to promoter intensity. The later value is perhaps more meaningful, because it can be used as a standard to help us define “inversion time” of an invertase module.</P><P>Through this series of data we can understand that promoter element has a significant potential to control the inversion time. Specifically, a stronger promoter leads to a shorter timing length. This helps us to modify the module into intended timing length. </P><P class="figure"> Fig-T-7:  The comparison of 5 constructive promoters on pInv-rep. The curves represents mcherry signals by pInv-rep. All condition are controlled expect for promoter of pInv-rep.</P><P> 2.The N-term fusion of FP onto invertase deteriorate its activity but is more stable</P><P> All the invertase in this study is a fusion protein to EGFP. However, we must be cautious when using such material, in that a fusion (especially of an entire protein) might interfere its 3D-structure and folding due to steric hindrance.</P><P>Hence, we prepared EGFP fusions onto either N-term or C-term of Cre (namely EGFP-Cre or Cre-EGFP), with an 8 AA flexible chain as linker to provide larger space for proper folding. The dynamics of the two proteins show dramatically different pattern (see Fig-T-8). When EGFP is linked to the C-term of Cre, it can be expressed at a slow but stable rate. Cre-EGFP works pretty good since we obtained obvious burst and increasing of mcherry signal. However, if EGFP is fused onto Cre’s N-term, it renders dramatically high concentration of EGFP-Cre expression, but the invertase activity is almost lost. The phenomenon indicates a possibility that if the N-term of Cre is linked to EGFP, its 3D structure might be interfered. </P><P class="figure"> Fig-T-8: The comparison of effect of different fusion sites on Cre. The curves represents Cre-EGFP and mcherry signals. All conditions are controlled except for fusion sites. </P><P class="figure"> Fig-T-9: The 3D structure of tetramer of Cre, the active form of Cre. Image is obtained from RCSB Protein Data Bank ( Http://www.rcsb.org/pdb/explore/explore.do?structureId=3MGV). A, the Z-axis view of Cre tetramer, 4 colors indicates 4 monomers. B, the N-term of Cre, blue arrows marks the M28, the starting amino acid of simplified Cre of BBa_K1179058; linking to a large protein like EGFP at this point take risks to generate steric hindrance. C, the C-term of Cre. Yellow arrow marks the final amino acid; a fusion to this site can be much better to maintain the structure.  </P><P>We find some evidence to support this hypothesis. The activity of invertase requires the formation of tetramer of Cre in combination with target DNA (see Fig-T-9 A), forming the structure so-called Holiday Junction. At this stage, Both N and C terminus seems to be loose with relatively open space, which is a good structure compatible of fusion (see Fig-T-9 B, C). However, the Cre we used, BBa_K1179058, is a truncated type of Cre that only maintains perhaps necessary part of the protein, so actually the translation starts from the No.28 Met, as marked in Fig-T-8 B, which is at the middle of an alpha helix locating pretty close to other part of the protein. Hence, we believe this is one of the reason why N-term linking to EGFP deteriorate its activity. Yet although this protein shows less activity, the expression rate is far higher than Cre-EGFP, probably because such structure can be folded quicker. </P><P> Therefore, since other invertases belongs to Cre-family, we construct only invertase-EGFP fusion with no EGFP-invertase issues.</P><P>3. Ssra can significantly avoid leakage but slightly reduce the inversion efficiency</P><P> Ssra-tag (e.g. BBa_M0051), if linked to the C-term of a protein in E. coli, will lead this protein ClpX or ClpA protease, rendering effective degradation. In this study we wish to utilize this tool to clean up certain invertases when going to next round of timing, and reduce the leakage of expression when not induced. Hence, ssra-tag, theoretically, will decrease both protein expression and leakage, prolonging the time for total inversion of an invertase module. </P><P class="figure"> Fig-T-10: the effect of ssra-tag on invertase dynamics. A, a comparison of EFGP expression of all pInv-gen. Adding an ssra tag can always reduce net expression rate, comparing to corresponding group. B, a comparison between group “pET-CG &amp; 101Lox-M” and “pET-CGS &amp; 101Lox-M”. The relatively slow net expression rate of Cre-EGFP-ssra renders a reduced intensity of mcherry expression.  </P><P class="figure">Fig-T-11: Adding an ssra-tag can restore the activity of EGFP-Cre. Although strains with EGFP-Cre grows at a significantly slow speed and shows tremendous accumulation of EGFP-Cre while hardly express mcherry, adding an ssra tag dramatically restarts the inversion.  </P><P>This is confirmed by our experiment. When adding an ssra-tag onto either mcherry or Cre-EGFP, its expression can be repressed at a certain level, comparing to those without ssra (see Fig-T-10). However, we do not have to worry about the risk that ssra is so strong that no enough invertase can be generated: even if we uses ssra-tag to modify Cre-EGFP, it still performs pretty good (See Fig-T-10 B).</P><P>Additionally, ssra tag can even restore the enzymatic activity of EGFP-Cre. When EGFP-Cre-ssra is expressed, it successfully overturns the reporter, although at a lower speed than Cre-EGFP-ssra (see Fig-T-11).</P><P>We decide to use ssra on other invertases.
</P><H3> The comparison of efficiency of different invertases (In fact, a good lesson!)</H3><P>There are 5 other Tyr-family recombinases, which have invertase activity. 4 among them, Dre, Vcre, Scre, and Vika are Cre-like invertases. Our team this year synthesized these 4 Cre-like invertases which is not previously in the biobricks and wish to test their function and utilize them to construct more timer.</P><P> Yes, perhaps we can catch up with the DDL of sending parts of them and will show the result in our poster and presentation. But we should have been able to finish this job far earlier and should have already measured their real-time dynamics – if it is not due to a horrible oligo synthesis company (Guangzhou IGE Biotechnology Ltd.) that totally ruined our clones by providing impure primers (even so-called “PAGE purified”) with deleted bases and making tremendous numbers of frameshift CDS.</P><P> This is a good lesson; we strongly recommend all iGEMers be careful to choose primer producer.</P><H3> The strategy to define inversion time of each invertase module </H3><P> To understand the length of timing for each combination of pInv-gen and pInv-rep is our primary concern. Roughly, this time can be considered as the time interval between the initiation of burst of green and red signal. This definition through visual observation is pretty subjective and not yet precise. We hence wish to design a precise method to understand inversion time for each module. </P><P>Luckily, due to our modeling work aims to solve this problem, we can find a far better solution. In real-time invertase dynamics measurement system, we can describe this process totally through mathematical function by data fitting. One of the greatest outcome is that we can define “time interval” be quantificational method. We will use the first derivative of both green and red signals, and the timing length of the invertase module is defined as the length of time between the two points when growth rate of two signal reaches 1/2 maximum level. </P><P>To see how data is further processed and invertase module timing length is calculated, please turn to <A href="https://2015.igem.org/Team:SYSU_CHINA/Result#Modelling">MODELING</A> for detail. </P></DIV><DIV id="Prokaryotic-Timer" class="scrollto"><H1>Prokaryotic Timer</H1><H3>Construction</H3><P>Small fragments served as joints between recombinases and fluorescences, which were sent to IDT (Integrated DNA Technology, Inc.) for accurate synthesis. These fragments were ligated to linear pSB1C3 and pSB1K3 backbones for further experiments.</P><P>We ligated our circuits by what we developed as &quot;2A&quot; assembly and attempted to get different segments. These segments, in conjunction with small fragments, are shown as biobricks below (Fig-P-7).</P><P class="figure">Fig-P-7 Successfully constructed biobricks presented with black lines below the circuits.</P><P>All segments were well sequenced, among which we successfully ligated BBa_K1641225, a major part of circuit 2 which was able to reverse from stage 0 to stage 1 (Fig-P-8).</P><P class="figure">Fig-P-8 BBa_K1641225 presented in pSB1K3, stage 0 as the initial phase and stage 1 comes after flipping of flpe-ecfp.</P><H3> Flipping test</H3><P>Due to time limits, we simply tested fluorescence expression of mid-body BBa_K1641225.</P><P>We managed to get BBa_K1641225 (pSB1K3 as backbone) transformed into E. coli Top10 strain, exerting a complete fluorescence detection. Result showed ECFP (433/476) can be induced normally and to some extent, an increased expression of mCherry (580/610) after a 4 hours' inducement (Fig-P-9), indicating effective flipping of flpe-ecfp in circuit 2.</P><P class="figure">Fig-P-9 Levels of ECFP and mCherry expression tested in BBa_K1631225 by microplate reader</P><P> Levels of fluorescent expression was calculated by the formula below (Fig-P-10)</P><P class="figure">Fig-P-10 Formula calculating fluorescent levels. </P><P> However, a prospective apparent descend of eCFP failed to be found and the efficiency of flipping wasn't that much as expected, in comparison with what the report has exhibited, probably on account of the control of plasmid copies, the efficiency of Flpe recombinase as well as the way we operate an inducement.</P><H3>Notes</H3><P>Due to influence of LB background, we measured our fluorescent level with M9 broth, which is different from what is mentioned in our wiki’s “Note” section. </P><P>E. coli strain Top10 or DH5a was  inoculated in 5ml M9 broth for 24h, 37C 220rpm, with 0.1% suitable antibiotic. 1% tryptone was added for better growth of bacteria. Then the grown cultures were diluted 1:5 in 5 ml of fresh M9 broth with 1% tryptone and incubated in the same condition. 1% L- arabinose was added to induce expression while culture without induction was measured as control group. Fluorescence background of M9+1% tryptone was also measured. </P><P>200 μL culture of each tube were transferred to a clean sterilized 96 well plate per hour. Then this plate was detected by BioTek Synergy H1 microplate reader with the following program: Room temperature (about 27 to 29 ℃); Sampling time about 5 min; linear shaking for 10 seconds; filter was 600 nm; ECFP filters were 433 nm(ex)/476 nm(em); mCherry filters were 580 nm(ex)/610 nm(em); GFP filters were 485 nm(ex)/511 nm(em).</P></DIV><DIV id="Telomeric-Timer" class="scrollto"><H1>Telomeric Timer</H1><P>For Microtimer 2.0, we successfully constructed a 1 step telomere-like system based on loxP flanking Cre-eGFP-T7TE device(CG) and had its dynamics tested.</P><P class="figure">Fig-Y-7 Plasmid map of PBAD-CG-1A2. The vector is primarily optimized for multi-step system in which flpe is located on the primary site, which is the reason why there is a frt site prior to the loxP site.</P><P>CG primarily is designed to be located on the secondary site while frt flanking flpe-Mcherry-T7TE device(FM) on the primary site. Due to the poor activity of recombinase flpe at 37℃ , however, we have not harvested positive results from FM device and 2 step telomeric system.</P><P class="figure">Fig-Y-8 graph of CG’s dynamic testing. The plateau phase indicates that the loxp flanking Cre-eGFP-T7TE fragment has been eliminated and thus the expression was stopped.</P><P>We applied nonlinear fit on the data harvested from CG-1A2 system, the equation is presented below</P><P>(68.556/(0.8014/(1+((0.8014-0.2268)*exp(-0.6023*x)/0.2268))))＋2058+2767*x－232.9*x^2</P></DIV><DIV id="Eukaryotic-Timer" class="scrollto"><H1>Eukaryotic Timer</H1><P>We transformed a commercial plasmid pAUR135 into a standard biobrick vector (Fig-Y-9) through 3 main steps. First we modified a PstI site located in the resistance coding region by mutating the site. We then used PstI exonuclease and S1 nuclease to modify another two closely located PstI sites. Thirdly, we inserted a MCS (multiple cloning sequence) into the plasmid which supplies the 4 standard exonuclease sites to it. After the transformation, this plasmid can be used as the backbone in our 3.0 timer. </P><P class="figure">Fig-Y-9 Plasmid map of pAUR135-RFC10 optimized. 3 PstI exonuclease sites have been modified and an MCS has been inserted.</P><P>We also constructed the part of bxb1gp35-RFC25 (see Fig-Y-10), which is for the first step in the microtimer 3.0. This part was meant to work in eukaryotes but as the culturing environment for yeast is unsuitable in our lab, and the time for transformation of yeast is too long per round (at least 4 days), we did not transform it into yeast. And due to the same reasons, we decided not to test our system in yeast.  </P>
         
 <P class="figure">Fig-Y-10 plasmid map of bxb1gp35-RFC25 optimized. bxb1gp35 is a modified sequence derived from the eukaryote recombinase bxb1 which has no standard exonuclease sites.</P>
         
      </DIV><DIV id="Modelling" class="scrollto"><H1>Modelling</H1><P> First of all, we have two tables of one certain combination, including different kinds of plasmid with 3 parts: certain type of promoter, invertase with its recognition sites on the reporter, and, a ssrA tag with specific intensity, which will be calculated later. </P><P> From the first table, we can see the OD700 of the culture, representing the population quantity. In order to get the exact function of the quantity (N) with respect to time (t), we use Logistic equation (equ.1) to fit it by using cftool in Matlab.</P><P>From the second table in each group, in biological aspect, Cre-EGFP produced by pInv-gen after inducer (IPTG) is added to the culture, we show the RFU changing according to time and it reflects the quantity changes of the protein, which contain two separate parts, the leakage and the actual production (which form is a linear function).</P><P class="figure"> The equation above is the first column of table 1 and Gre0 is the first column of table 2. Now we can obtain the exact function G(t) of table 2 using the data and fitting tool of custom function in matlab, which represents the amount of enzyme with respect to time .</P><P>Let’s move forward to working out how the degradation tag contribute to the final result of expression. In our experiments, we use two different tags. The first one is the O &amp; M which is on the mCherry. So we have the scatter data (form 1) through the difference of row OCG &amp; MCG, OCGS &amp; MCGS, OGC &amp; MGC, OGCS&amp; MGCS in the third table.</P><P class="figure"> Fig-M-1</P><P> In the same way, we obtain the scatter data (form 2) of another tag, ssrA-tag. It shows how many reactant it degrades at the moment 0, 1, 2 and so on.</P><P class="figure"> Fig-M-1</P><P>So we can use Michaelis-Menten equation (3) to describe it.</P><P class="figure"> Fig-M-2</P><P>It is obvious that we can get the total degradation amount by integrating the velocity V0 with respect to time t, thus we have function as follows:</P><P>Use G0(t) to substitute the complex G(t) and solve the equation (9) to obtain the Sp(t).</P><P class="figure"> 
Equ(5) Pp(t) is the quantity of the promotor that have been inverted in the bacterial population per volume .Sp(t) is the quantity of the promotor to be inverted in the bacterial population per volume .(here S is known)

(6) we use the Michaelis-Menten equation to describe the enzymatic activity.

(7)the total reaction rate equals to the product of enzymatic activity  in the bacterial population per volume and the amount of enzyme with respect of time t, G(t).

(8) By integrating the rate Vi, we get the quantity of the product.

(9) derived from the equation set, we get the ODE model. 
</P><P>Use G0(t) to substitute the complex G(t) and solve the equation (9) to obtain the Sp(t).</P><P> As for the last expression (10),	we represent the intensity of every promotor. The additional correction (*) part is to translate the figure because of the pressure from the environment. When there are too much products in one certain environment, the population would need a while to react to the situation  of nutrition deficiency. So a time delay could be observed in the figure that shows the changing of the red light.</P><H3>Abbreviation of the terms used in model part</H3><H3>Modeling for Testing Group's Result</H3><P>To see what the data means in biology, please turn to <A href="https://2015.igem.org/Team:SYSU_CHINA/Result#Matching-and-Testing">Matching and Testing</A> for detail. </P><P>•The strategy to define inversion time of each invertase module
To understand the length of timing for each combination of pInv-gen and pInv-rep is our primary concern. Roughly, this time can be considered as the time interval between the initiation of burst of green and red signal. This definition through visual observation is pretty subjective and not yet precise. We hence wish to design a precise method to understand inversion time for each module.</P><P>Luckily, due to our modeling work aims to solve this problem, we can find a far better solution. In real-time invertase dynamics measurement system, we can describe this process totally through mathematical function by data fitting. One of the greatest outcome is that we can define “time interval” be quantificational method. We will use the first derivative of both green and red signals, and the timing length of the invertase module is defined as the length of time between the two points when growth rate of two signal reaches 1/2 maximum level.</P><P>Here is the model and fitting result of Green fluorescent protein's expression in Testing Group's system.</P><P> Formula: K/(1+((K-N0)*exp(-r*x)/N0)) &amp; expression (2)

1. pET-CG&amp;101Lox-O</P><P> od: N(t)=0.7253/(1+((0.7253-0.2106)*exp(-0.6703*t)/ 0.2106))

leakage=119.5/(0.7253/(1+((0.7253-0.2106)*exp(-0.6703*t)/ 0.2106)))</P><P> G(t)=414*t+199.6+(119.5/(0.7253/(1+((0.7253-0.2106)*exp(-0.6703*t)/ 0.2106))))

2. pET-CGS&amp;101Lox-O</P><P> od: N(t)=0.7487/(1+((0.7487-0.2241)*exp(-0.7123*t)/0.2241))
leakage=73/(0.7487/(1+((0.7487-0.2241)*exp(-0.7123*t)/0.2241)))</P><P> G(t)=145*t-406.8+(73/(0.7487/(1+((0.7487-0.2241)*exp(-0.7123*t)/0.2241))))
3. pET-GC&amp;101Lox-O</P><P> od: N(t)=78/(0.4105/(1+((0.4105-0.1078)*exp(-1.343*t)/ 0.1078)))
leakage=78/(0.4105/(1+((0.4105-0.1078)*exp(-1.343*t)/ 0.1078)))</P><P> G(t)=559.6*t+7631+(78/(0.4105/(1+((0.4105-0.1078)*exp(-1.343*t)/ 0.1078))))
4. pET-GCS&amp;101Lox-O</P>
od: N(t)=0.7437/(1+((0.7437-0.2354)*exp(-0.7228*t)/ 0.2354))
leakage=120.5/(0.7437/(1+((0.7437-0.2354)*exp(-0.7228*t)/ 0.2354)))


G(t)=170.1*t-18.15+(120.5/(0.7437/(1+((0.7437-0.2354)*exp(-0.7228*t)/ 0.2354))))
5. pET-CG&amp;101Lox-M<P> od: N(t)=0.7084/(1+((0.7084-0.1961)*exp(-0.6928*t)/ 0.1961))
leakage=147.5/(0.7084/(1+((0.7084-0.1961)*exp(-0.6928*t)/ 0.1961)))

</P><P> G(t)=196.5*t+386.5+(147.5/(0.7084/(1+((0.7084-0.1961)*exp(-0.6928*t)/ 0.1961))))
6. pET-CGS&amp;101Lox-M</P><P> od: N(t)=0.7066/(1+((0.7066-0.2564)*exp(-0.6644*t)/ 0.2564))
leakage=95.5/(0.7066/(1+((0.7066-0.2564)*exp(-0.6644*t)/ 0.2564)))</P><P> G(t)=118.6*t-287.4+(95.5/(0.7066/(1+((0.7066-0.2564)*exp(-0.6644*t)/ 0.2564))))
7. pET-GC&amp;101Lox-M</P><P> od: N(t)=0.3358/(1+((0.3358-0.05478)*exp(-1.842*t)/ 0.05478))
leakage=100.5/(0.3358/(1+((0.3358-0.05478)*exp(-1.842*t)/ 0.05478)))</P><P> G(t)=663.3*t+6989+(100.5/(0.3358/(1+((0.3358-0.05478)*exp(-1.842*t)/ 0.05478))))

8. pET-GCS&amp;101Lox-M</P><P> od: N(t)=0.7216/(1+((0.7216-0.2279)*exp(-0.6804*t)/ 0.2279))
leakage=107/(0.7216/(1+((0.7216-0.2279)*exp(-0.6804*t)/ 0.2279)))</P><P> G(t)=162.8*t+26.57+(107/(0.7216/(1+((0.7216-0.2279)*exp(-0.6804*t)/ 0.2279)))) </P></DIV></DIV><DIV class="to-top-btn"><BUTTON>▲</BUTTON></DIV></DIV></DIV></DIV><DIV style="background-color: #FFE075; color:#fff; height: 240px;padding-top:20px;text-align: center; padding-bottom: 30px;"><H5 style="text-align: center; margin-bottom: 30px;">Sponsor</H5></DIV><DIV style="background-color:#444; color: #fff; height: 70px;text-align: center;padding-top: 10px;">
Name: SYSU-China   School: Sun Yat-sen University
Address: No. 135, Xingang Xi Road, Guangzhou, 510275, P. R. China
Contact: nichy5@mail2.sysu.edu.cn
</DIV></DIV></DIV></DIV></BODY></HTML>