Team:US AFRL CarrollHS/Results
Team
Team Members Collaborations
Project
Description Design Experiments Notebook Interlab Model Results Demonstrate Improve Attributions
Parts
Parts Overview Composite Parts Part Collection
Safety(current)
Human Practices
Human Practices Education & Engagement
Awards
Applied Design Best New Composite Part Best Part Collection Education & Engagement Integrated Human Practices Model
Judging Form(current)
Detect and Deliver
The LabPats were able to complete their sense and respond module as planned. The goal of the project was to base a plasmid design very similar to last years. The students wanted a simple circuit design capable of delivering the microbe to the C4-HSL and biofilm. They first started with a constitutive promoter, PJ23117, to continually express RhlR. By continuing expressing RhlR, it would allow for the binding of C4-HSL whenever it became present. Afterwards, it would activate the PRhl promoter and start the transcription and translation of our CheZ gene, which would allow the flagellum of the microbe to navigate in a straight line path. When C4-HSL was no longer expressed, CheZ production would stop and the degradation tag would kick in, allowing the cell to start tumbling again. Hopefully through expressing CheZ and tumbling, a majority of the bacterium will be able to make it to the biofilm.
Figure 1. Genetic Circuit for the detect and deliver mechanism
With this goal in mind, the LabPats set to work with the “Detect” and “Deliver” mechanism. In their lab, Dr. Goodson had a similar construct readily available, however the gene expressed was GFP. In the first couple weeks, the LabPats were able to ligate Dr. Goodson’s construct within the iGEM backbone. This plasmid was sent off to IDT and sequenced, and the LabPats were ecstatic to see a perfectly expected sequence. They then utilized primers to insert the CheZ in place of the GFP. This proved to be quite a task as we experimented different methods to insert the CheZ within the plasmid correctly. After multiple failed attempts, the LabPats tried it over and over again, going as far as experimenting with a gradient PCR and ordering brand new primers. Eventually, after a lot of trials, the LabPats were able to send off the final construct to IDT for sequencing, achieving favorable results.
While in the design phase of constructing their plasmid, the students also completed motility tests with E. coli cells with the CheZ knocked out. E. coli by nature have produce CheZ, so by testing with a strain without CheZ (courtesy of Professor Sandy Parkinson), the LabPats were able to make a clear distinction that the E. coli without CheZ knocked out were non-motile. Thus, they could insert their finished plasmid within the non-motile E. coli and observe movement towards C4-HSL.
We also performed some experiments with GFP to ensure the detect mechanism worked correctly, and these results can be found on our Demonstrate page.
Destroy
Chitinase C-1 was successfully cloned with the ice nucleation protein into the iGEM backbone plasmid for part submission. Chitinase B4A was successfully cloned with the ice nucleation protein into the iGEM backbone plasmid for part submission.   The three enzymes in order to produce cinnamaldehyde, which was originally in three parts, was assembled into a DNA vector with two more ribosomal binding sites to improve efficiency, but was not able to be successfully cloned into the iGEM backbone plasmid.
We also performed a number of tests to determine the efficacy of chitinase and cinnamaldehyde against fungi and bacteria, and these results can be found on our Demonstrate page
Figure 2. Genetic Circuit for the chitinase mechanism
Contact us @iGEMLabPats or iGEM@carrollhs.org
