<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Edinburgh_UG_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Edinburgh UG/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><P><TITLE>Edinburgh iGEM 2018</TITLE></P><DIV class="collapse navbar-collapse" id="navbarNavDropdown"><UL class="navbar-nav"><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:Edinburgh_UG">Home <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Team
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Team">Team Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Collaborations">Collaborations</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Project
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Experiments">Experiments</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/InterLab">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Model">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Demonstrate">Demonstrate</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Improve">Improve</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Attributions">Attributions</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Parts
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Parts">Parts Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Basic_Part">Basic Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Composite_Part">Composite Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Part_Collection">Part Collection</A></DIV></LI><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:Edinburgh_UG/Safety">Safety <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:Edinburgh_UG/Human_Practices">Human Practices <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Awards
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Model">Model</A></DIV></LI><LI class="nav-item active"><A class="nav-link" href="https://igem.org/2018_Judging_Form?team=Edinburgh_UG">Judging Form <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:Edinburgh_UG/Medal_Criteria">Medal Criteria<SPAN class="sr-only">(current)</SPAN></A></LI></UL></DIV><HEADER class="masthead"><DIV class="intro-body"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H1 class="brand-heading" align="center">Design</H1><H1 class="brand-heading">Why Do We Need Maxicells?</H1></DIV></DIV></DIV></DIV></HEADER><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><P style="text-align:left">Bacteria and other unicellular prokaryotes may seem somewhat insignificant in the ecosystems we see around us. How can something so small have any influence on the 
             hugely complex multicellular life sharing the environment? We (think we) know all about the bacteria involved in the nitrogen cycle, carbon cycle - bacterial species that we know have a 
             direct affect on the anthropogenic sphere. But what about microbial species that are not directly involved in these processes, should we be concerned about the effects that human activity 
             have on them if it will have no effect on us? The answer is YES.</P><P style="text-align:left">Recent studies in metagenomics have revealed a complex network of dynamic interactions in microbial communities. Species that 
             were previously thought to be unimportant have been found to impact these processes by providing services to those species performing the key, central reactions. With this knowledge it is clear that any human activity, such as environmental release of reproductive micro-organisms that interfere with natural ecosystem dynamics, can have unknown, possibly long-lasting effects on the biogeochemical cycles which are necessary for life on Earth. It is therefore imperative that any Synthetic Biology venture intended for environmental release should not have any undesired effects on the local ecology.</P><P style="text-align:left">We considered the possible reasons for why environmental release is so dangerous, and decided that there were two main problems which must be overcome:</P><OL><LI style="text-align:left">The possibility that our released strain could proliferate uncontrollably</LI><LI style="text-align:left">The risk that genes from our strain could escape by horizontal gene transfer and provide a competitive advantage to native prokaryotes </LI></OL><P style="text-align:left">Both of these outcomes have to potential to alter the balance of ecosystem interaction networks, which could have a myriad of unforeseen consequences.</P><P style="text-align:left">In order to tackle the first problem, and to a large extent the second, we first looked at minicells: cells generated after abnormal septum formation during replication due to misplacement of the FtsZ ring - leading to the production of small cells that do not possess a chromosome.</P></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><P style="font-size:10px"> Reeve JN, Mendelson NH, Coyne SI, Hallock LL, Cole RM. Minicells of Bacillus subtilis [Internet]. Vol. 114, JOURNAL OF BACTERIOLOGY. 1973. Available from: http://jb.asm.org/</P></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><P style="text-align:left">As they have no chromosome, they are unable to reproduce and proliferate in the environment. Minicells have been shown to retain plasmids, and are therefore presented as a promising novel chassis for Synthetic Biology. It would be possible for minicells to perform any plasmid-encoded function you desire, and would solve the first problem for environmental release. However, minicells presented problems with purification due to their small, inconsistent size. After discussion with experts in the field of biotechnology to try and resolve these issues, it was suggested that we instead try to remove the chromosome by some other means. This suggestion led us to completely change the design of our project; to instead look at maxicells.</P><P style="text-align:left">Maxicells are bacterial cells which have lost their chromosome, and are therefore unable to reproduce and proliferate in the environment. These maxicells can be used for Synthetic Biology, as they retain any plasmids they possessed prior to chromosome destruction, which we have defined as the “instructor plasmids”. Maxicells would then be as functional and safe as minicells, without the costly, time-consuming purification steps.</P><P style="text-align:left">While maxicells are unable to reproduce and cannot release any chromosomal genes into the environment, it is nonetheless still possible that advantageous genes from the instructor plasmid could escape by horizontal gene transfer, the most notable of which being the antibiotic resistance marker used for plasmid selection. We have approached this problem in three ways, which we have termed our Triple Lock System:</P><OL><LI style="text-align:left">The colicin kill switch, which aims to fully degrade any plasmid DNA in the maxicell after a defined, programmable time-frame. </LI><LI style="text-align:left">Our semantic containment system, which ensures that any native prokaryotes will be unable to read and express the instructor plasmid DNA, as it will be written in a different language.</LI><LI style="text-align:left">The alternative selection marker, triclosan, which removes the need for an antibiotic resistance gene to be encoded on the instructor plasmid, thereby preventing the spread of antibiotic resistance.</LI></OL><P style="text-align:left">The design of our project then fell into three overarching sections, which will be detailed below:</P><UL><LI style="text-align:left">Evaluating and Optimising Maxicell Protocols</LI><LI style="text-align:left">Quantifying the Active Timeframe of Maxicells</LI><LI style="text-align:left">The Triple Lock System</LI></UL></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H1 class="brand-heading" align="left">Evaluating and Optimising Maxicell Protocols</H1><P style="text-align:left">We investigated 3 ways in which to produce maxicells in order to present the easiest, most efficient production method for current and future synthetic biologists wishing to use them as a chassis! These three methods were by: </P><UL><LI style="text-align:left"> Ultraviolet light exposure 
                    </LI><LI style="text-align:left"> Cleaving a palindromic hairpin loop on the chromosome 
                    </LI><LI style="text-align:left"> Cutting the chromosome with a homing endonuclease </LI></UL><P style="text-align:left"> We then attempted to modularise the UV production method by creating knockdowns for DNA repair genes using RNA interference. </P><H2 style="text-align:left">UV Production Method</H2><P style="text-align:left">This is the established method of maxicell production, which uses ultraviolet light to damage the DNA within the cell. The exposure to UV light causes pyrimidine dimers to form, which in regular cells would be restored  by nucleotide excision repair, however, the maxicell producing strain CSR603 is a knockout for <I>recA</I> - an enzyme involved in DNA repair by homologous recombination - and <I>uvrA</I> - an enzyme involved in nucleotide excision repair. As the cell has no method of repairing the UV-induced damage, the entire chromosome is degraded by exonucleases, leaving an achromosomal maxicell. </P><P style="text-align:left">We decided to evaluate the efficiency of maxicell production using two <I>Escherichia coli</I> strains: CSR603, the <I>uvrA<SUP>-</SUP></I><I>recA<SUP>-</SUP></I> strain; and DH5-alpha, a standard <I>recA<SUP>-</SUP></I> lab strain. DH5-alpha was used to test how effective standard, <I>recA<SUP>+</SUP></I> lab strains are as maxicell producers. After the effectiveness of maxicell production by these strains was investigated, we set out to further increase the modularity of this system by producing a MicC-RecA-UvrA construct to knockdown <I>recA</I> and <I>uvrA</I> using RNA interference. This would allow theoretically any <I>E. coli</I> strain of your choosing to be used as a maxicell producer, and potentially many other bacterial species.</P><H2 style="text-align:left">Palindromic Cleavage Method</H2><P style="text-align:left">The next maxicell production method we investigated was suggested to us by experts in Molecular and Cell Biology from the Leach Lab. This uses an <I>E. coli</I> strain developed in their lab, DL 3355, which contains a palindromic site on the chromosome that causes a hairpin loop to form. This hairpin loop may be cleaved by sbcCD exonuclease, causing a double strand break in the DNA. As the strain is <I>recA<SUP>-</SUP></I>, this break cannot be repaired, and the chromosome is degraded by exonucleases.</P><H2 style="text-align:left">Homing Endonuclease Method</H2><P style="text-align:left">The last method of maxicell production was also suggested by the Leach Lab, and uses an <I>E. coli</I> strain - DL2524 - that contains an 18 bp recognition site for the homing endonuclease I-SceI, and the <I>I-SceI</I> gene under the control of an arabinose-inducible promoter. When the cells are cultured in 0.5% arabinose, <I>I-SceI</I> is expressed, which then cuts the chromosomal DNA at the recognition site producing a double strand break that leads to complete degradation of the chromosome.</P><H1 class="brand-heading" align="left">Quantifying Maxicell Timeframe</H1><P style="text-align:left">After we established that we could reliably produce maxicells, we attempted to get a quantitative measurement of how long they could be metabolically active. This would have obvious applications in the field, as you would know the time-frame in which maxicells can be used for your desired process. We took a three-pronged approach to this quantification, beginning by looking at their ATP levels over time, the level of protein retention, and their ability to functionally express a gene at various times after their production.</P><H2 style="text-align:left">ATP Quantification</H2><P style="text-align:left">We added the enzymes D-Luciferin and Luciferase to the maxicell and cell samples in order to produce luminescence by hydrolysis of ATP by the enzymes. This luminescence can be directly correlated to the level of ATP in a cell. After calibration with known ATP levels, we used a luminometer to determine the levels of ATP in our maxicells over a 72 hour period at 4 °C, 21 °C and 37 °C. We picked these temperatures as they are often used in the lab and are therefore would be most commonly used in experiments. We also thought these temperatures related quite well to temperatures in the world. Average of 21 °C during the spring and summer in the majority of countries; 4 °C for the winter months and 37 °C for hotter climates around the world. We also thought if we can show the differences in ATP levels at different temperatures it will allow us to better visualise what climates around the world will be best suited for our chassis.</P><H2 style="text-align:left">Protein Retention</H2><P style="text-align:left">We next looked at the protein levels within maxicells at various time points and concentrations. We figured that the half-life of proteins would be much longer than in non-maxicells, as the half-life would no longer be intrinsically tied to the rate of division; in normal cells the level of protein halves at every cell-division. The protein levels within maxicells would therefore be solely dependent on protein export and degradation rate. These values were unknown in maxicells, as they have an altered metabolism, so we sought to determine the overall protein levels using SDS-PAGE and compare them at different time-points. As with ATP quantification, this was performed at 4 °C, 25 °C and 37 °C.</P><H2 style="text-align:left">Functional Gene Expression</H2><P style="text-align:left">The above characterisations give us some idea of how long maxicells can operate, but we considered that there may be another, unknown factor that affects their functionality. As a definitive test to determine their functional lifespan, we decided to insert a construct into our maxicell instructor plasmid to see how well it can express a gene after a given time. We settled on two constructs:</P><UL><LI style="text-align:left">ars-mCherry - the fluorescent protein mCherry under control of an arsenite inducible promoter</LI><LI style="text-align:left">araC-mCherry - mCherry under an arabinose promoter</LI><P style="text-align:left">The added benefit of using these parts was to test how effectively our maxicells can function as a biosensor; in these cases for arsenic and arabinose. This would be a characterisation of how well maxicells <I>actually</I> work in the field, for one of their potential applications. We planned to transform the maxicell producing <I>E. coli</I> strain CSR603 with these constructs and then induce mCherry expression with arsenic or arabinose after 18 hours, 19 hours and 24 hours. This would give us a better idea of how long they can survive and still have the potential for gene expression.</P></UL></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H1 class="brand-heading" align="left">The Triple Lock System</H1><H2 style="text-align:left">Colicin Kill Switch</H2><P style="text-align:left">As the first ‘lock’ in our triple lock system, the function of the colicin kill switch is to prevent the instructor plasmid from being released from the maxicell into the 
             environment in the first place. This 2-part system was inspired by  <A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/KILLswitch"> Darmstadt 2016 </A> iGEM team, but altered and optimised for use in maxicells. The kill switch works by the action of colicin E2, a nicking endonuclease which cuts single and double stranded DNA at random sites. The counterpart to E2 is Imm2; an immunity protein which binds to E2 and inhibits its endonuclease activity. Imm2 binds in a 1:1 stoichiometric ratio, so if the level of Imm2 in the cell should fall below the level of E2, then E2 is free to function. We hypothesised that by fine-tuning of the expression of these genes, we could cause the Imm2 concentration to drop below that of E2 at a defined time. This time point would be the point at which the instructor plasmid would start to be degraded. We therefore made some alteration to Darmstadt's design to make it applicable for the release of maxicells.</P><P style="text-align:left">The first alteration was to the Immunity protein construct (Imm2). We have inserted I-Sce1 sites flanking the <I>imm2</I> gene in the plasmid. This makes the system compatible with the                         
             ‘Homing Endonuclease’ (I-SceI) method of maxicell production. When <I>I-SceI</I> expression is induced to digest the chromosome, the <I>imm2</I> gene will also be cut out and therefore is no longer expressed. Imm2 
             protein levels will fall over time until Colicin-E2 protein is no longer bound by Imm2 and is free to digest the plasmid. </P><P style="text-align:left">Following environmental release, conditions for the maxicell will be uncontrolled and variable. Therefore, the longer the maxicell is in the environment, the higher the probability 
             of a chance maxicell lysis event, and release of the plasmid into the surroundings. The plasmid may then be picked up by other prokaryotes in the environment. It is important that the instructor plasmid is 
             degraded before this occurs. However, we do not want to inhibit maxicell function (whatever that may be) by destroying the instructor plasmid too early. Ideally, plasmid degradation should occur immediately 
             prior the to the end of the active metabolic lifetime of the maxicell. Timing of plasmid destruction by Colicin should be finely tuned meaning selection of promoters and ribosome binding sites for E2 and Imm2 
             expression is crucial in order for degradation to occur at the desired point in time. </P><P style="text-align:left">We have used rational design in order to select the promoters and ribosome binding sites necessary for triggering plasmid degradation at the point we estimate our maxicells will no longer be 
             metabolically active (see 'Quantifying Maxicell Timeframe' above). We have designed a model that predicts the degradation time point for every possible promoter-RBS combination for Colicin and Imm2 
             using the Anderson promoters and ribosome binding sites on the iGEM registry. From this model we selected promoter <A href="http://parts.igem.org/Part:BBa_J23105"> BBa_J23105 </A> for Imm2, and promoter <A href="http://parts.igem.org/Part:BBa_J23106"> BBa_J23106 </A> for Colicin E2. Both parts had RBS <A href="http://parts.igem.org/Part:BBa_J61107"> BBa_J61107 </A> and Terminator <A href="http://parts.igem.org/Part:BBa_B0010"> BBa_B0010 </A></P></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H2 style="text-align:left">Semantic Containment</H2><P style="text-align:left">The second lock in our triple lock system ensures that even if the first lock fails, an environmental cell would not be able to express and utilise a gene obtained from our 
             maxicells. The basis for this came from the <A href="https://2012.igem.org/Team:Paris_Bettencourt/Semantic_containment"> Paris-Bettencourt 2012 </A> project but we have taken it a step further. This involved rewriting the DNA of our instructor plasmid so that it cannot be read by a foreign organism. Essentially we're giving our plasmid a different language.</P><P style="text-align:left">The basis of semantic containment lies in the mutation of the <I>supD</I> gene - a tRNA amber suppressor - which causes amber STOP codons to be read as a serine codon instead. Our sequence may be re-coded to change serine codons to amber STOP codons, and as long as it has the <I>supD</I> mutation the genes will be translated correctly. However, if a foreign organism were to take up the DNA, it would read the amber codons as a STOP, producing a non-functional, truncated protein. This was achieved by Paris-Bettencourt using one and two serine codon replacements with amber STOP codons in the kanamycin resistance gene, which they designated P1003* and P1003**. This allows easy quantification of the effective expression of recoded genes by non-<I>supD</I> mutants simply by testing growth rates on kanamycin. Paris-Bettencourt's results showed that non-<I>supD</I> mutants with the recoded P1003* and P1003** genes showed limited growth on kanamycin, suggesting that the recoding was successful to a certain extent. We decided, however, that these results did not show a high enough level of security, and so we planned to go a step further.</P><P style="text-align:left">Before designing parts for this, we again approached the problem by utilising rational design. We modelled the probability of a translational read-through with increasing numbers of amber             stop codons present in a coding sequence. With 2 amber codons, 1 in every 10,000 translation attempts will be able to generate a read through and express the recoded gene. When aiming for complete safety and containment, this is not good enough… We therefore designed 2 new parts building on Paris-Bettencourt’s: P1003 5* <A href="http://parts.igem.org/Part:BBa_K2725012"> [BBa_K2725012] </A> and P1003 10* <A href="http://parts.igem.org/Part:BBa_K2725013"> [BBa_K2725013] </A>. These have 5 and 10 serine codons from the P1003 coding sequence substituted with amber stop codons. With 5 amber codons present in a coding sequence, probability of a read through and subsequent expression is reduced to 1 in every 10<SUP>10</SUP>  translation attempts, and with 10 amber codons this is reduced further to 1 in 10<SUP>18</SUP> translation attempts. For more information on our probability model, see our <A href="https://2018.igem.org/Team:Edinburgh_UG/Semantic_Containment_Modelling"> semantic containment modelling </A> page. </P><P style="text-align:left">Addition of more amber codons however, also presents a problem for expression of the recoded gene in our chassis. In order to solve this we used the same probability model in order 
             to maximise read through in our chassis. We took the supD + terminator sequence from Paris-Bettencourt's part, <A href="http://parts.igem.org/Part:BBa_K914000"> BBa_K914000 </A> and assembled this downstream of a higher strength constitutive Anderson promoter. 
             Higher levels of amber suppressor supD increases the likelihood of the tRNA interacting with an amber codon instead of Release Factor 1 that would release the nascent polypeptide from the ribosome and halt 
             translation. We designed 3 of these parts with varying strength anderson promoters in order to test for proof of concept:</P><UL><LI style="text-align:left">J23102-supD - <A href="http://parts.igem.org/Part:BBa_K2725014"> [BBa_K2725014] </A> (strength 0.82)</LI><LI style="text-align:left">J23103-supD - <A href="http://parts.igem.org/Part:BBa_K2725015"> [BBa_K2725015] </A> (strength 0.01)</LI><LI style="text-align:left">J23108-supD - <A href="http://parts.igem.org/Part:BBa_K2725016"> [BBa_K2725016] </A> (strength 0.51)</LI></UL></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H2 style="text-align:left">Alternative Selection</H2><P style="text-align:left">The third lock in our triple lock system ensures that even if an environmental cell takes up an instructor plasmid, and can read and express the genes present, that those genes 
             confer no competitive advantage to the environmental cell. This is done by replacing the antibiotic resistance gene in the standard iGEM backbone with a gene that gives resistance to the biocide, triclosan <SUP>[1,2]</SUP>. 
             Triclosan is a chemical that has previously been used in health and beauty products such as toothpaste, but today it is very rarely used. Critically, it is not used in medicine and healthcare, therefore if 
             a pathogen became triclosan resistant this would not interfere in our ability to treat such an infection. The triclosan resistance gene, FabV, offers no cross resistance to any commonly used 
             antibiotics. It is also cheaper than conventional antibiotics as lower concentrations need to be used for selection: &lt;1p per litre compared to 56p per litre for ampicillin (prices from Alfa Aesar).</P><P style="text-align:left">Triclosan is a biocide which acts upon FabI protein in <I>E. coli</I>, an enoyl ACP-reductase involved in fatty acid biosynthesis. We planned to investigate the effectiveness of resistance by two mechanisms. The first is by overexpression of <I>fabI</I>, which gives triclosan resistance as there is so much of the target that no matter how much FabI is broken down, there is still enough for the system to function. The second method of resistance is by heterologous expression of <I>fabV</I> – an enoyl ACP-reductase found in Vibrio species. Research has shown that FabV confers greater resistance than FabI, as it is refractory to inhibition by triclosan.</P></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H4 style="text-align:left">References:</H4><UL><LI style="text-align:left">Ali S, Chew Y, Omar T, Azman N. Use of FabV-Triclosan Plasmid Selection System for Efficient Expression and Production of Recombinant Proteins in Escherichia coli. PLOS ONE. 2015;10(12):e0144189.</LI><LI style="text-align:left">Ali S, Chew Y. FabV/Triclosan Is an Antibiotic-Free and Cost-Effective Selection System for Efficient Maintenance of High and Medium -Copy Number Plasmids in Escherichia coli. PLOS ONE. 2015;10(6):e0129547.</LI></UL></DIV></DIV></DIV></SECTION><SECTION id="contact" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H2>Contact EdiGEM18</H2><P>Feel free to leave us a comment on social media!</P><UL class="list-inline banner-social-buttons"><LI class="list-inline-item"><A href="https://twitter.com/EdiGEM2018" class="btn btn-default btn-lg"><SPAN class="network-name">Twitter</SPAN></A></LI><LI class="list-inline-item"><A href="https://www.instagram.com/igemedinburgh/" class="btn btn-default btn-lg"><SPAN class="network-name">Instagram</SPAN></A></LI><LI class="list-inline-item"><A href="https://www.facebook.com/EdiGEM2018/?ref=br_rs" class="btn btn-default btn-lg"><SPAN class="network-name">Facebook</SPAN></A></LI></UL></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>