<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Edinburgh_UG_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Edinburgh UG/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><P><TITLE>Edinburgh iGEM 2018</TITLE></P><DIV class="collapse navbar-collapse" id="navbarNavDropdown"><UL class="navbar-nav"><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:Edinburgh_UG">Home <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Team
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Team">Team Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Collaborations">Collaborations</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Project
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Experiments">Experiments</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/InterLab">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Model">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Demonstrate">Demonstrate</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Improve">Improve</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Attributions">Attributions</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Parts
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Parts">Parts Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Basic_Part">Basic Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Composite_Part">Composite Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Part_Collection">Part Collection</A></DIV></LI><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:Edinburgh_UG/Safety">Safety <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:Edinburgh_UG/Human_Practices">Human Practices <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Awards
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Model">Model</A></DIV></LI><LI class="nav-item active"><A class="nav-link" href="https://igem.org/2018_Judging_Form?team=Edinburgh_UG">Judging Form <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:Edinburgh_UG/Medal_Criteria">Medal Criteria<SPAN class="sr-only">(current)</SPAN></A></LI></UL></DIV><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H1 class="brand-heading" align="center">Results</H1><H2 style="text-align:left">Achievements</H2><UL><LI style="text-align:left">We have characterised various methods of maxicell production, to optimise their manufacture for synthetic biology</LI><LI style="text-align:left">The active time frame of maxicells was quantified, to determine their potential “lifespan”</LI><LI style="text-align:left">We have created parts which facilitate maxicell production in any E. coli strain of your choosing</LI><LI style="text-align:left">We have introduced a functional biosensor into the maxicell chassis, which has shown that maxicells have the potential for gene expression for at least 18 hours after their  
                   production</LI><LI style="text-align:left">Various methods of preventing horizontal gene transfer were explored, to increase the security of maxicells as a chassis designed for environmental release</LI><LI style="text-align:left">Triclosan resistance was introduced into a standard backbone as an alternative to antibiotic selection, which would prevent the unwanted release of antibiotic resistance genes</LI></UL></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H1 class="brand-heading" align="left"><FONT color="bronze">Evaluating and Optimising Maxicell Protocols</FONT></H1><P style="text-align:left">Three methods of maxicell production were evaluated to identify the most efficient process. Each method causes a double-strand break in the chromosome, leading to its degradation by native exonucleases (the rec system, in recA knockout cells). After maxicell induction, all of these methods involved treatment by cycloserine to kill the non-maxicells. Cycloserine is an antibiotic that targets cell wall synthesis, thereby killing any actively dividing cells. The level of maxicell production was determined by fluorescence microscopy of samples stained with DAPI; a fluorescent stain that binds to double stranded DNA.</P><H2 style="text-align:left">UV Production Method</H2><P style="text-align:left">We first started by looking at the <I>Escherichia coli</I> strains DH5-alpha (<I>recA<SUP>-</SUP></I>, <I>uvrA<SUP>+</SUP></I>) and CSR603 (<I>recA<SUP>-</SUP></I>, <I>uvrA<SUP>-</SUP></I>). It was found that when attempting maxicell formation in strains with functional uvrA, such as DH5-aplha, all stages post-UV exposure until the end of a 15-18 hour incubation must be carried out in complete darkness, to prevent photoreactivation by the UVR system. We also found that while CSR603 is a more efficient maxicell producer than DH5-alpha, it grows slower and the cells are significantly shorter in length and diameter than DH5-a. We surmised that this was because the cells are, in essence, sicker due to the mutations in DNA repair systems. When cultured in LB and exposed to 40 seconds of UV and then incubated for a further 18 hours, 70.0% CSR603 showed no DAPI fluorescence. When grown under the same conditions but exposed to no UV, 2.9% CSR603 showed no DAPI fluorescence.</P><P style="text-align:left">Next, we used the <I>E. coli</I> strain MC4100 to produce a <I>recA</I>, <I>uvrA</I> knockdown, as this strain was recommended in literature for maxicell production. Through utilisation of RNA interference by MicC sRNAs, we created a <I>recA</I>, <I>uvrA</I> knockdown construct. The objective was two-fold: to develop a method of maxicell production which could be theoretically applied to any strain of E. coli; and to create a healthier maxicell producing strain - after it was observed that CSR603 has impaired growth due to its sickness. Both of these objectives were successful. For the knockdown strain (MC4100), we found that the colony forming units per mL culture 18 hours post UV-exposure was vastly lower than that of MG1655 (fully functioning <I>recA</I> and <I>uvrA</I>), and only slightly higher than CSR603 (fully non-functioning <I>recA</I> and <I>uvrA</I>) (Figure 1.). Furthermore, MC4100 growth rate in LB media and M9 glucose media is vastly higher than that of CSR603. When cells are cultured in M9 glucose media, there is no significant difference in CFU/mL 18 hours post-UV exposure when cells are exposed to light or kept in complete darkness. When cells are cultured in LB media, CFU/mL is significantly higher if the post-UV culture is exposed to light (Figure 2.).</P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 1. Colony forming units/mL after 18 hours of UV treatment, when cells are exposed to light afterwards.</FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 2. Colony forming units/mL of MC4100 starin after 18 hours of UV treatment, when cells are exposed or not to light afterwards.</FIGCAPTION></FIGURE><P style="text-align:left">The MicC constructs we produced were as follows:</P><UL><LI style="text-align:left"><A href="http://parts.igem.org/Part:BBa_K2725006">BBa_K2725006</A>: micC construct for recA knock down</LI><LI style="text-align:left"><A href="http://parts.igem.org/Part:BBa_K2725007">BBa_K2725007</A>: micC construct for uvrA knock down</LI><LI style="text-align:left"><A href="http://parts.igem.org/Part:BBa_K2725008">BBa_K2725008</A>: micC construct for uvrA and recA knock down</LI><LI style="text-align:left"><A href="http://parts.igem.org/Part:BBa_K2725009">BBa_K2725009</A>: micC construct for the mCherry gene that produces increased mCherry fluorescence</LI></UL><H2 style="text-align:left">Palindromic Cleavage Method</H2><P style="text-align:left">Another method of maxicell induction that was investigated involved using a strain of <I>E. coli</I>, DL3355 ( lacIq lacZχ- ParaBAD-sbcDC lacZ::pal460 recA::Cm), which contains a palindromic site in the chromosome that forms a hairpin loop. This hairpin loop can be cut by an sbcCD exonuclease that was introduced to the chromosome under control of an arabinose-sensitive promoter. This causes a double strand break in the chromosome. leading to digestion by native exonucleases. However, this method was found to be largely unsuccessful: a spot test of <I>E. coli</I> DL3355 with the <I>sbcCD</I> exonuclease gene under an arabinose inducible promoter inserted into the chromosome showed that there was no decrease in colony forming units after <I>sbcCD</I> induction. It was surmised that this was due to a high rate of mutation in this strain, as the palindromic sequence is a burden on the cell there is an inherent rate of loss of the palindrome site of 1/10000 per genome per generation. The rate of strain reversion is therefore very high, so this method of maxicell production is inefficient and not recommended.</P><H2 style="text-align:left">Homing Endonuclease Method</H2><P style="text-align:left">The last method we looked at centres around <I>E. coli</I> DL2524 (MG1655 recA::CmR araB::PBAD-I-SceI lacZ::I-SceIcs  fnr-267), a strain that contains in its chromosome an 18 base pair recognition site for the homing endonuclease I-SceI, and the <I>I-SceI</I> gene under control of an arabinose-sensitive promoter. Maxicell production by this strain was initiated by culturing in arabinose, which activates <I>I-SceI</I> expression, leading to a double-strand break in the chromosome at the I-SceI recognition site. This method was the most successful at maxicell production: the lowest percentage of cells seen to be fluorescing with DAPI were in DL2524 arabinose induced samples (compared to all UV induced maximal methods and DL3355).</P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 3. Microscopy images of DL2524 cells incubated in glucose (top - no induction of homing endonuclease expression) and in arabinose (bottom - induction of homing endonuclease expression). </FIGCAPTION></FIGURE></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H1 class="brand-heading" align="left"><FONT color="green">Quantifying Maxicell Timeframe</FONT></H1><H2 style="text-align:left">ATP Quantification</H2><P style="text-align:left">All ATP assays was conducted in the DL2524 <I>E. coli</I> strain (Endonuclease Method of producing maxicells). This strain and method were used as it was the easiest way of forming Maxicells as it was controlled by an inducible promoter. The ATP levels in Maxicells were compared at 3 Different temperature (37 °C; 21 °C; 4 °C, Figure 4) as we wanted to test the effect of varying temperature on the functionality of our chassis.</P><H4 style="text-align:left">ATP Decline in Maxicells when Stored in the Incubator (37 °C):</H4><P style="text-align:left">Variation of ATP levels seem normal over 18 hours after induction between maxicells and non-maxicells (Fig 3a). As expected non-maxicells showed an increase of ATP after 18 hours as they used the nutrients in the medium to proliferate and produce more ATP. However, after 18 hours we start the decline in ATP in our maxicell strain. At 20 hours the concentration of ATP in maxicells is 0.54 mM; while the concentrations of ATP in non-maxicells is 0.62 mM. This divergence in ATP levels continues until ~26 hours in maxicells where it appears to have plateaued at ~0.48 mM while non-maxicells are at 0.61 mM and continue to increase over the rest of the time frame. It appears maxicells have ~24-hour time frame before the ATP inside the cell is depleted to baseline and the cell stops to function when stored at 37 °C. However, this still needs to be confirmed via protein synthesis assays. </P><H4 style="text-align:left">ATP Decline in Maxicells when Stored at Room Temperature (21 °C):</H4><P style="text-align:left">ATP concentration for non-maxicells show a similar trajectory to non-maxicells at 37 °C, however with a lower overall concentration (Fig 3b). This is because <I>E. coli</I> proliferate better at 37 °C compared to 21 °C, therefore the non-maxicells at 37 °C have a higher ATP concentration on average. We can also see that maxicells at 21 °C show a similar trajectory as maxicells at 37 °C. However, seem to plateau earlier than the maxicells at 37 °C. They plateau at ~16 hours to 0.51mM. This is unusual as our initial thought were the cells at the 37 °C will metabolise ATP faster as enzymes are at their optimal temperature, however, this data suggests that this may not be the case and that the main factor driving ATP concentration decline is natural degradation/hydrolysis of ATP over time. This could be confirmed as there may just be discrepancy recording or other errors.</P><H4 style="text-align:left">ATP Decline in Maxicells when Stored in the Fridge (4 °C):</H4><P style="text-align:left">Again, non-maxicells increase in ATP concentration albeit at a lower rate the non-maxicells at 21 °C or 37 °C (Fig 3c). The maxicells show plateauing at 24 hours to 0.49 mM of ATP. Again, very similar to the other two temperatures. However, this data supports the theory that the main factor driving ATP degradation in our maxicells is not an enzymatic activity but rather natural degradation/hydrolysis of ATP over time.</P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 4. Concentration of ATP (mM) over Time (hours) in DL2524 either induced - Maxicells (grown in Arabinose) OR not induced - Non-Maxicells (grown in glucose). a: shows Maxicell and Non-Maxicell ATP conc. when stored at 37 °C. b: shows Maxicell and Non-Maxicell ATP conc. when stored at 21 °C. c: shows Maxicell and Non-Maxicell ATP conc. when stored at 4 °C</FIGCAPTION></FIGURE><H4 style="text-align:left">Conclusions: </H4><P style="text-align:left">The general trend seems to follow a pattern between all the data sets and temperatures that is: Plateau of ATP concentration at ~24 hours to ~0.5mM. This was similar for all temperatures (37 °C; 21 °C and 4 °C) which suggests may suggest the main factor that causes ATP degradation in our maxicells is naturally degradation of the molecule; as if the main factor was metabolism we would expect to see a larger difference in the time it takes to plateau for each temperature. This is because enzymes within the cell are most optimal at 37 °C and least at 4 °C. However, they plateau at the same time. Therefore this raises the question of the cause of ATP decline over time.
It is also important to not that the natural concentration of ATP within <I>E. coli</I> is 1.54 ± 1.22 mM. Which our system does fall into as 1 hour after induction we see both maxicells and non-maxicells are ~0.62 mM of ATP, which fall within the normal range. Albeit the DL2524 seems to fall at the lower normal range that wild-type, however, this may be due to the fact our cells seem to be sicker than wild-type and therefore may not be able to produce ATP as efficiently as wild type cells. 
The potential active time frame of ~24 hours in the environment may not be limited to this maximum number as we know we can make our cells healthier by not using <I>recA</I> KO mutant and instead using <I>recA</I> knockdown, as seen in the MicC constructs. As the cell is healthier it will be able to proliferate quicker and able to store more ATP. It may also be possible to increase intracellular ATP before induction to boost potential active timeframe via other methods. These are all experiments future iGEM teams can conduct to further characterise our chassis and boost potential usage. 
</P><H4 style="text-align:left">ATP levels of Induced Maxicells when stored in the Freezer (-20°C) for 2 weeks:</H4><P style="text-align:left">To see if our maxicell could be frozen for transport we decided to test ATP concentrations of maxicells that were frozen and compared this to our other temperatures to see if they held a similar trajectory. The maxicells were frozen along with non-maxicells (see method here). They were kept in the freezer for two weeks before being thawed to simulate transport in real life. They were then assayed. Figure 5 shows that non-maxicell act as normal; as they gradually increase in ATP concentrations. 
However, our maxicells seem to decline in ATP concentration much quicker than our other temperatures. They seem to reach 0.46mM of ATP at ~16 hours after being thawed. This may be due to lysis of the cells during the freezing or thawing process which results in reduced ATP within the cell, or there may be some unknown factor for the rapid decline in ATP seen in the frozen maxicells.
Due to time restraint, we could not conduct more experiments, but we would have like to repeat this experiment at different time points to see the effect of storage on the cell, i.e. test at 2 days frozen; 1 week frozen; 1 month frozen etc. We would have also liked to try other methods of storage such as freeze-drying and the effect of this on ATP levels.
</P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 5. Conc. of ATP (mM) over Time (hours) in DL2524 either induced - maxicells (grown in arabinose then frozen) OR not induced - non-maxicells (grown in glucose then frozen). Shows maxicell and non-maxicell ATP concentration over time for maxicells that have been induced at then frozen.</FIGCAPTION></FIGURE><H2 style="text-align:left">Protein Retention</H2><P style="text-align:left">Next, we decided to investigate the level of protein degradation in our maxicells, to find out whether or not all the cellular machinery remains intact over long periods of time. To do so, we took samples of maxicells stored 4°C for 0 h and 24 h, and at 25°C &amp; 37°C for 0 h, 24 h and 48 h. We then ran these samples through an SDS-PAGE gel, the results of which can be seen in Figure 6. below. These results show that there is no major decrease in protein concentration within maxicells over a period of 48 h, not even at 37°C, suggesting that the level of protein degradation is minimal. Although we cannot determine absolute protein concentrations between the samples due to potential differences in sample preparation, we can see that they are not orders of magnitude different. It can, therefore, be inferred that protein degradation within maxicells is not a significant factor for their functionality over long periods of time.</P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 6. SDS Page of All Maxicell Proteins Over Time when Stored at Different Tempratures. (see <A href="https://2018.igem.org/Team:Edinburgh_UG/Experiments"> experiments page </A> for details)</FIGCAPTION></FIGURE><H2 style="text-align:left">Functional Gene Expression</H2><P style="text-align:left">Finally, CSR603 UV-induced maxicells were used to investigate the ability of maxicells to transcribe and translate a chosen gene, in this case mCherry under an inducible arabinose promoter (<A href="http://parts.igem.org/Part:BBa_K2725010">BBa_K2725010</A>). These maxicells were induced by arabinose to produce mCherry 18-25 hours after UV exposure. Fluorescence by mCherry was observed after arabinose treatment 24 hours after each sample was induced. The proportion of cells fluorescing that were induced 18 hours after UV exposure was 57.9%, which significantly decreases to 4.7% in samples induced 20 hours after UV exposure, and a maximum of 6% of cells fluorescing was seen in samples induced 25 hours post-UV exposure (figure 7.). In contrast, it was found that cells with intact chromosomes fluoresce only faintly after arabinose treatment, and mCherry is not visible to the naked eye in colonies cultured on LB+arabinose plates. It can be surmised, therefore, that maxicells can be induced to produce a gene product after their production, and can act as a biosensor: for arabinose in this case.</P><P style="text-align:left">Unfortunately, due to difficulties in transformation of CSR603 we were unable to transform our ars-mCherry construct (<A href="http://parts.igem.org/Part:BBa_K2725011">BBa_K2725011</A>) into this strain. This made it impossible to test the ability of maxicells to act as a biosensor for arsenic.</P></DIV></DIV></DIV></SECTION><CENTER><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 7. Graph of mCherry fluorescence 24 h after arabinose induction against &quot;age&quot; of maxicells (time in hours after maxicell production). </FIGCAPTION></FIGURE></CENTER><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H1 class="brand-heading" align="left"><FONT color="blue">The Triple Lock System</FONT></H1><H2 style="text-align:left">Colicin Kill Switch</H2><P style="text-align:left">The results for the modelling section of our colicin kill switch can be seen <A href="https://2018.igem.org/Team:Edinburgh_UG/Degradation_Switch">here</A>. We <A href="https://2018.igem.org/Team:Edinburgh_UG/Collaborations">collaborated</A><A> with Team Vilnius to develop a protocol for applying this model.</A></P><H2 style="text-align:left">Semantic Containment</H2><P style="text-align:left">Our aim was to recode the kanamycin resistance gene with differing numbers of serine codons replaced with amber codons (1, 2, 5 and 10). E. coli TOP 10 was transformed with each of these 4 plasmids and their growth measured on 8 concentrations of kanamycin - concentrations decreasing 2 fold from 400 ug/mL to 3.125 ug/mL. The outcome of this can be seen in Figure 8, which shows that replacing a single serine codon with an amber STOP codon does not fully inhibit kanamycin resistance in this strain. The fact that 1* showed growth on kanamycin is due to an inherent level of amber suppression. This phenomenon has been investigated by <A href="https://2018.igem.org/Team:Edinburgh_UG/Semantic_Containment_Modelling"><I>in silico</I> modelling</A> to determine the rates of read-through for the different numbers of mutations.</P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 8. Growth curves of Top10 <I>E.coli</I> transformed with P1003, P1003* (1*), P1003** (2*), P1003 5* (5*) and P1003 10* (10*) grown in the given concentration of kanamycin. These growth curves show the extent of kanamycin resistance conferred by each part. </FIGCAPTION></FIGURE><P style="text-align:left">After it was determined that sufficient kanamycin resistance could not be conferred by the 2*, 5* (<A href="http://parts.igem.org/Part:BBa_K2725012">BBa_K2725012</A>) and 10* (<A href="http://parts.igem.org/Part:BBa_K2725013">BBa_K2725013</A>) genes when <I>supD</I> is functional, we then sought to test for resistance when <I>supD</I> is mutated as an amber suppressor. We also tested the extent of kanamycin resistance when the amber suppressor <I>supD</I> was expressed under different strength Anderson promoters. The results of this can be seen in Figure 9 below. </P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 9. Growth curves of 8 Top10 <I>E.coli</I> double transformants in the given concentration of kanamycin . These growth curves show the extent of kanamycin resistance conferred by each part combination. </FIGCAPTION></FIGURE><P style="text-align:left"> It is clear that when the amber suppressor <I>supD</I> is expressed under medium strength promoter J23108, P1003 with 1, 2, and 5 amber codons can be expressed at a sufficient enough level that confers kanamycin resistance. This means the rate of serine integration at amber codons is much higher than the removal of the nascent polypeptide by Release Factor 1. However, there was no kanamycin resistance conferred by the P1003 10* gene. This means that the amber suppressor <I>supD</I> was not expressed at a high enough level to outcompete Release Factor 1 at 10 successive amber STOP codons. We attempted higher level expression of amber suppressor <I>supD</I> (<A href="http://parts.igem.org/Part:BBa_K2725014">BBa_K2725014</A>), however no double transformants with this part had any kanamycin resistance. We think this is due to the presence of amber STOP codons in the chromosome. If chromosomal genes that use amber STOP codons for termination of translation consistently have serine incorporated (and continued translation) instead, this would place a significant metabolic burden on the cell. This, therefore, generates significant selection pressure for a mutation that decreases the expression level of the amber suppressor <I>supD</I> or prevents it completely. Because the construct was under a constitutive promoter this mutation likely occurred before the cells were transferred to a kanamycin medium. </P><P style="text-align:left">The Anderson Promoter-SupD constructs are as follows:</P><UL><LI style="text-align:left"><A href="http://parts.igem.org/Part:BBa_K2725014">BBa_K2725014</A>: Serine amber suppressor tRNA gene under J23102 Anderson promoter</LI><LI style="text-align:left"><A href="http://parts.igem.org/Part:BBa_K2725015">BBa_K2725015</A>: Serine amber suppressor tRNA gene under J23103 Anderson promoter</LI><LI style="text-align:left"><A href="http://parts.igem.org/Part:BBa_K2725016">BBa_K2725016</A>: Serine amber suppressor tRNA gene under J23108 Anderson promoter</LI></UL><H2 style="text-align:left">Alternative Selection</H2><P style="text-align:left"><U>FabI</U></P><P style="text-align:left">The <I>fabI</I> gene was PCR amplified from DH5-α genomic DNA and the biobrick prefix and suffix was added. The PCR product was introduced into the biobrick site of pSB1C3. The FabI biobrick was then expressed under a high constitutive expression cassette (<A href="http://parts.igem.org/Part:BBa_K314100">BBa_K314100</A>) to produce <A href="http://parts.igem.org/Part:BBa_K2725001">BBa_K2725001</A> and growth was observed on triclosan at 1 µM (Figure 10.), while untransformed DH5-alpha showed no growth (Figure 11.).</P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 10. 1 µM triclosan agar plate, plated with <I>E. coli</I> DH5-alpha transformed with BBa_2725001</FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 11. 1 µM triclosan agar plate, plated with untransformed <I>E. coli</I> DH5-alpha</FIGCAPTION></FIGURE><P style="text-align:left"><U>FabV</U></P><P style="text-align:left">As a proof of concept for our final design, the <I>fabV</I> gene was introduced into pSB1C3, replacing chloramphenicol resistance by Gibson Assembly to produce a new plasmid backbone: <A href="http://parts.igem.org/Part:BBa_K2725005">pSB1Tcs1</A>. The <I>fabV</I> gene was therefore expressed under the chloramphenicol acetyltransferase promoter and ribosome binding site. This new plasmid conferred resistance to triclosan above 16 mM, showing similar growth profiles at all concentrations below 16 mM, while the growth of  untransformed DH5-α was inhibited at 0.25 µM and was severely inhibited above 4 µM (Figure 12.). It is therefore advised that 4-16 µM triclosan be used for selection in liquid culture. However, It was found that 1 µM triclosan is sufficient for selection on agar plates, with no colonies being observed when untransformed DH5-ɑ was plated at 1 µM (Figure 11.), as lower concentrations can be used for selection when plating.</P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 12. Growth curves of untransformed DH5-alpha at triclosan concentrations ranging from 0 to 64 µM (left) and DH5-alpha transformed with pSB1Tcs1 (BBa_K2725005) at concentrations of triclosan ranging from 0 to 16 mM.</FIGCAPTION></FIGURE><P style="text-align:left">The <I>fabV</I> gene was then PCR amplified from our pSB1Tr1 plasmid to add the Biobrick prefix and suffix. This <I>fabV</I> Biobrick (<A href="http://parts.igem.org/Part:BBa_K2725002">BBa_K2725002</A>) was inserted after a high constitutive expression cassette (<A href="http://parts.igem.org/Part:BBa_K314100">BBa_K314100</A>) to produce <A href="http://parts.igem.org/Part:BBa_K2725003">BBa_K2725003</A> and a low constitutive expression cassette (<A href="http://parts.igem.org/Part:BBa_K314101">BBa_K314101</A>) to produce <A href="http://parts.igem.org/Part:BBa_K2725004">BBa_K2725004</A>, both of which showed growth on triclosan at 1  µM (Figure 13.).</P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 13. 1 µM triclosan agar plates, plated with <I>E. coli</I> DH5-alpha transformed with BBa_2725003 (left) and BBa_K2725004 (right)</FIGCAPTION></FIGURE><P style="text-align:left"><U>What's Next?</U></P><UL><LI style="text-align:left">The triclosan resistant backbone should be recoded to replace all serine codons with amber stop codons, so that it is compatible with the semantic containment system</LI><LI style="text-align:left">Triclosan resistance is due to a higher than normal expression of Fab proteins, or more specifically, it is due to an increased number of triclosan binding sites. It may, therefore, be possible to decrease the metabolic burden on the cell or increase resistance by trimming the FabI or FabV protein down to just the domain which binds to triclosan. This would further optimise this system of triclosan resistance</LI></UL></DIV></DIV></DIV></SECTION><SECTION id="contact" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H2>Contact EdiGEM18</H2><P>Feel free to leave us a comment on social media!</P><UL class="list-inline banner-social-buttons"><LI class="list-inline-item"><A href="https://twitter.com/EdiGEM2018" class="btn btn-default btn-lg"><SPAN class="network-name">Twitter</SPAN></A></LI><LI class="list-inline-item"><A href="https://www.instagram.com/igemedinburgh/" class="btn btn-default btn-lg"><SPAN class="network-name">Instagram</SPAN></A></LI><LI class="list-inline-item"><A href="https://www.facebook.com/EdiGEM2018/?ref=br_rs" class="btn btn-default btn-lg"><SPAN class="network-name">Facebook</SPAN></A></LI></UL></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>