<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Edinburgh_UG_Modelling_Collaboration skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Edinburgh UG/Modelling Collaboration</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><P><TITLE>Edinburgh iGEM 2018</TITLE></P><DIV class="collapse navbar-collapse" id="navbarNavDropdown"><UL class="navbar-nav"><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:Edinburgh_UG">Home <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Team
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Team">Team Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Collaborations">Collaborations</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Project
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Experiments">Experiments</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/InterLab">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Model">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Demonstrate">Demonstrate</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Improve">Improve</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Attributions">Attributions</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Parts
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Parts">Parts Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Basic_Part">Basic Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Composite_Part">Composite Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Part_Collection">Part Collection</A></DIV></LI><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:Edinburgh_UG/Safety">Safety <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:Edinburgh_UG/Human_Practices">Human Practices <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Awards
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Edinburgh_UG/Model">Model</A></DIV></LI><LI class="nav-item active"><A class="nav-link" href="https://igem.org/2018_Judging_Form?team=Edinburgh_UG">Judging Form <SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:Edinburgh_UG/Medal_Criteria">Medal Criteria<SPAN class="sr-only">(current)</SPAN></A></LI></UL></DIV><HEADER class="masthead"><DIV class="intro-body"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H1 class="brand-heading" align="center">Modelling Collaboration</H1></DIV></DIV></DIV></DIV></HEADER><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H2 style="text-align:left">Introduction</H2><P style="text-align:left">Team Vilnius-Lithuania aims to use the PURE cell free system <SUP>[1]</SUP> in order to integrate proteins into the membrane of the liposome from the inside. The BamA complex is responsible for integrating these proteins; OmpA and lgA hence in order to ensure quick integration BamA needs to be consistently present at high levels throughout the expression of OmpA and lgA. This mechanistic model aims to examine the simultaneous expression of these proteins and compare the results obtained from different starting parameters, primarily different starting volumes of <I>bamA</I> RNA in order to quantify the effectiveness of an initial addition of RNA to ensure fast production of BamA.</P></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H2 style="text-align:left">Mass Action Equations</H2><P style="text-align:left">Mass Action Equations are commonly used to represent chemical reactions and provide a starting point for mechanistic modelling of a variety of phenomena. The laws of mass action state that the rate of any chemical reaction is proportional to the product of the masses of the reacting substances, with each mass raised to a power equal to the coefficient that occurs in the chemical equation <SUP>[2]</SUP>. The mass action equations in Figure 1 can be used to represent protein expression: </P><P style="text-align:left">Each of these equations is used in triplicate to represent expression of BamA, OmpA and lgA respectively and from these mass action equations a system of ordinary differential equations can be derived.</P></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H2 style="text-align:left">Ordinary Differential Equations</H2><P style="text-align:left">The model uses a simple set of ordinary differential equations (ODEs):</P><P style="text-align:left">In order to solve this system it is first necessary to derive values for all the parameters used:</P><P style="text-align:left">copies<SUB>BamA</SUB>, copies<SUB>OmpA</SUB>, copies<SUB>lgA</SUB> - Number of plasmid copies - In order to take into account the effect of different starting masses of DNA for BamA, OmpA and lgA it is neccesary to calculate the number of plasmids present from which each protein may be expressed. DNA added to the PURE system should be between 25 and 1000ng per reaction <SUP>[1]</SUP> and hence it was decided to screen over a number of values in this range. A single base pair has mass of 650 Daltons <SUP>[3]</SUP> hence it is possible to calculate mass in kDa of each plasmid with its particular insert when each plasmid length is known <SUP>[4,5]</SUP>:</P><UL><LI style="text-align:left">peT2Ab with BamA - 5089.5 kDa</LI><LI style="text-align:left">pRSETb with OmpA - 2550.95 kDa</LI><LI style="text-align:left">pRSETb with lgA - 2490.8 kDa</LI></UL><P style="text-align:left">The conversion 1 ng = 6.022*10<SUP>17</SUP> kDa allows the calculation of the number of plasmids present for a particular number of ng of DNA added (as seen below in Table 1): </P><P style="text-align:left"><B>Table 1. Number of Plasmid Copies</B></P><TABLE class="table"><THEAD><TR><TH>peT2AB with BamA</TH></TR></THEAD><TBODY><TR><TD>DNA added (ng)</TD><TD>Number of Copies</TD></TR><TR><TD>25</TD><TD>2.985*10<SUP>15</SUP></TD></TR><TR><TD>250</TD><TD>2.985*10<SUP>16</SUP></TD></TR><TR><TD>1000</TD><TD>1.183*10<SUP>17</SUP></TD></TR></TBODY></TABLE><TABLE class="table"><THEAD><TR><TH>pRSETb with OmpA</TH></TR></THEAD><TBODY><TR><TD>DNA added (ng)</TD><TD>Number of Copies</TD></TR><TR><TD>25</TD><TD>5.902*10<SUP>15</SUP></TD></TR><TR><TD>250</TD><TD>5.902*10<SUP>16</SUP></TD></TR><TR><TD>1000</TD><TD>2.361*10<SUP>17</SUP></TD></TR></TBODY></TABLE><TABLE class="table"><THEAD><TR><TH>pRSETb with lgA</TH></TR></THEAD><TBODY><TR><TD>DNA added (ng)</TD><TD>Number of Copies</TD></TR><TR><TD>25</TD><TD>6.044*10<SUP>15</SUP></TD></TR><TR><TD>250</TD><TD>6.044*10<SUP>16</SUP></TD></TR><TR><TD>1000</TD><TD>2.418*10<SUP>17</SUP></TD></TR></TBODY></TABLE><P style="text-align:left">Although it would have been desirable to screen over a greater number of values within the 25-1000ng range the resulting increase in size of the already large parameter space made this intractable.</P><P style="text-align:left">tr<SUB>BamA</SUB>, tr<SUB>OmpA</SUB>, tr<SUB>lgA</SUB> - Transcription Rate - Transcription is known to occur at approximately 60 nucleotides per second <SUP>[6]</SUP>. Length of each gene was found to be 2430 nucleotides for BamA <SUP>[7]</SUP>, 1038 nucleotides for OmpA <SUP>[8]</SUP> and 945 nucleotides for lgA <SUP>[9]</SUP>. Computing the transcription rate per minute for each of BamA, OmpA and lgA:</P><UL><LI style="text-align:left">tr<SUB>BamA</SUB> - (60/2430)∗60 = 1.48 mRNAs per minute</LI><LI style="text-align:left">tr<SUB>OmpA</SUB> - (60/1038)∗60 = 3.47 mRNAs per minute</LI><LI style="text-align:left">tr<SUB>lgA</SUB> - (60/945)∗60 = 3.81 mRNAs per minute</LI></UL><P style="text-align:left">deg<SUB>mRNA BamA</SUB>, deg<SUB>mRNA OmpA</SUB>, deg<SUB>mRNA lgA</SUB> - mRNA Degradation Rate - Average mRNA half-life is known to be approximately 5 minutes however to reflect variation in this number between different mRNAs, we screen here over half-lives of 1, 5, 10 and 15 minutes for each protein <SUP>[10]</SUP>. Degradation rate is calculable from half-life using the formula; deg = ln(2)/halflife <SUP>[11]</SUP>. Using this formula degradation rates per minute are calculated as:</P><UL><LI style="text-align:left">1 minute half-life - 0.69 mRNAs per minute</LI><LI style="text-align:left">5 minute half-life - 0.14 mRNAs per minute</LI><LI style="text-align:left">10 minute half-life - 0.07 mRNAs per minute</LI><LI style="text-align:left">15 minute half-life - 0.05 mRNAs per minute</LI></UL><P style="text-align:left"> trl<SUB>BamA</SUB>, trl<SUB>OmpA</SUB>, trl<SUB>lgA</SUB> - Translation Rate - Translation is known to occur at approximately 20 amino acids per second <SUP>[12]</SUP>. Length of each protein was found  for BamA <SUP>[7]</SUP>, OmpA <SUP>[8]</SUP> and lgA <SUP>[9]</SUP> by searching through protein databases. Here we show the calculation used to find the translation rate per minute for BamA, OmpA and lgA:</P><UL><LI style="text-align:left">trl<SUB>BamA</SUB> - (20/810)∗60 = 1.48 proteins per minute</LI><LI style="text-align:left">trl<SUB>OmpA</SUB> - (20/346)∗60 = 3.47 proteins per minute</LI><LI style="text-align:left">trl<SUB>lgA</SUB> - (20/315)∗60 = 3.81 proteins per minute</LI></UL><P style="text-align:left"> deg<SUB>BamA</SUB>, deg<SUB>OmpA</SUB>, deg<SUB>lgA</SUB> - Protein Degradation Rate - Protein half-life was determined using ProtParam Tool <SUP>[13]</SUP>. ProtParam uses the N-end rule <SUP>[14]</SUP> to determine protein half-life, the estimates given for each of BamA, OmpA and lgA are (&gt;)10hrs in E.coli and 30hrs in reticulocytes hence an average 20hr half-life. In order to reflect the inexact nature of these computationally derived half-lives, we screen over possible half-lives of 10, 20 and 30 hours for each of BamA, OmpA and lgA. Applying the same degradation rate formula as previously deg = ln(2)/halflife <SUP>[11]</SUP> this yields degredation rates per minute of: </P><UL><LI style="text-align:left">10 hour half-life - 1.16 * 10 <SUP>-3</SUP> proteins per minute</LI><LI style="text-align:left">20 hour half-life - 5.78 * 10 <SUP>-4</SUP> proteins per minute</LI><LI style="text-align:left">30 hour half-life - 3.85 * 10 <SUP>-4</SUP> proteins per minute</LI></UL></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H2 style="text-align:left">Starting Conditions</H2><P style="text-align:left">In order to examine the effects of higher initial mass of BamA RNA 6 different values were screened over. The guidance for the PURE system <SUP>[1]</SUP> suggests addition of RNA between 1 and 5 ug. Using the masses of sense and anti-sense strands of BamA RNA in kDA <SUP>[15]</SUP> (830.382 and 820.8 respectively) and conversion 1 ug = 6.022*10<SUP>20</SUP> kDa the number of RNA molecules added can be calculated using the formula (ug added *(6.022*10<SUP>20</SUP>)/((830.382 + 820.8)/2)))) as seen below in table 2:</P><P style="text-align:left"><B>Table 2. Initial RNA Levels</B></P><TABLE class="table"><THEAD><TR><TH>RNA added (ug)</TH><TH>RNA added (molecules)</TH></TR></THEAD><TBODY><TR><TD>0</TD><TD>0</TD></TR><TR><TD>1</TD><TD>7.29*10<SUP>17</SUP></TD></TR><TR><TD>2</TD><TD>1.46*10<SUP>18</SUP></TD></TR><TR><TD>3</TD><TD>2.19*10<SUP>18</SUP></TD></TR><TR><TD>4</TD><TD>2.92*10<SUP>18</SUP></TD></TR><TR><TD>5</TD><TD>3.65*10<SUP>18</SUP></TD></TR></TBODY></TABLE></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H2 style="text-align:left">Results</H2><P style="text-align:left">The primary aim of this model is to identify parameters leading to fast and consistently high levels of BamA under conditions of co-expression of BamA, OmpA and lgA. In order to identify those particular parameters leading to these conditions it is first prudent to examine some general trends. Figure 2 displays the minimum, maximum and average levels of mRNAs and protein for BamA, OmpA and lgA. The value at each time point was obtained from the range of values at the same time point across each run of the model. </P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 2. Average, minimum and maximum number of protein molecules after 2 hours</FIGCAPTION></FIGURE><P style="text-align:left"> Table 3 summarizes this data in the form of number of molecules of mRNA and protein for each average, minimum and maximum plot and each of BamA, OmpA and lgA after 2 hours. </P><P style="text-align:left"><B>Table 3. Protein Levels After 2 Hours</B></P><TABLE class="table"><THEAD><TR><TH>Protein</TH><TH>Number of Molecules</TH></TR></THEAD><THEAD><TR><TH>Average</TH></TR></THEAD><TBODY><TR><TD>BamA</TD><TD>2.8*10<SUP>20</SUP></TD></TR><TR><TD>OmpA</TD><TD>2.78*10<SUP>21</SUP></TD></TR><TR><TD>lgA</TD><TD>3.44*10<SUP>21</SUP></TD></TR></TBODY><THEAD><TR><TH>Minimum</TH></TR></THEAD><TBODY><TR><TD>BamA</TD><TD>1.95*10<SUP>18</SUP></TD></TR><TR><TD>OmpA</TD><TD>2.13*10<SUP>19</SUP></TD></TR><TR><TD>lgA</TD><TD>2.63*10<SUP>19</SUP></TD></TR></TBODY><THEAD><TR><TH>Maximum</TH></TR></THEAD><TBODY><TR><TD>BamA</TD><TD>1.28*10<SUP>21</SUP></TD></TR><TR><TD>OmpA</TD><TD>1.28*10<SUP>22</SUP></TD></TR><TR><TD>lgA</TD><TD>1.58*10<SUP>22</SUP></TD></TR></TBODY></TABLE><P style="text-align:left"> In the average mRNA graph in Figure 2 we can see <I>ompA</I> and <I>lgA</I> mRNA present in rapidly increasingly levels until an equilibrium is reached (around 1 hour), the increased rate and higher equilibrium value of lgA mRNA are due to its shorter gene length and hence faster mRNA synthesis than OmpA. Average <I>bamA</I> mRNA starts of at comparatively high levels due to the initial additions of <I>bamA</I> mRNA however the mass of DNA present in the average case is not sufficient to maintain these high mRNA levels and hence <I>bamA</I> mRNA drops to an equilibrium level around 25% of OmpA and lgA mRNA. </P><P style="text-align:left"> Minimum mRNA levels occur when no addition of initial mRNA is made as well as when screening over low DNA masses and fast mRNA degredation rates. In this area of the parameter space mRNA for all 3 proteins quickly reach low equilibrium levels characterized primarily by effect of gene length on speed of mRNA synthesis. </P><P style="text-align:left"> Maximum mRNA levels are reached with large additions of <I>bamA</I> mRNA and large masses of DNA. The resulting synthesis of <I>bamA</I> mRNA maintains a high equilibrium value throughout the entire model run. mRNA of OmpA and lgA in the maximal case quickly increases to a high equilibrium value after around 100 minutes.</P><P style="text-align:left"> By comparing the mRNA levels in average, minimum and maximum cases we can observe as might be expected that increasing mass of DNA means that equilibrium levels of mRNA take longer to reach and occur at a higher value.</P><P style="text-align:left"> In average, minimum and maximum cases the protein expression of OmpA, lgA and BamA follows the same trend which is unaffected by fluctuations in mRNA level. Each protein is expressed at a rate primarily proportional to its length and to a magnitude primarily dependent on the mass of available DNA.</P><P style="text-align:left"> Figure 3 displays the distribution of number of BamA molecules present at 20 minute intervals, appendix B contains analogous plots for OmpA and lgA.</P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 3. Distribution of BamA at 20 minute intervals</FIGCAPTION></FIGURE><P style="text-align:left"> At each 20 minute interval we can observe the distribution of BamA expression stretches to the right as more protein is synthesized. Within the initial 40 minutes the distribution is heavily grouped together as there has not been enough time for BamA to be present in high levels. Between 60 and 100 minutes the distribution spreads out and settles into distinct groups corresponding to particular parameter settings which increasingly move to the right as more protein is synthesized.</P><P style="text-align:left"> Figure 4 shows the distribution of number of <I>bamA</I> mRNA molecules present at 20 minute intervals, appendix B contains analogous plots for OmpA and lgA.</P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 4. Distribution of <I>bamA</I> mRNA at 20 minute intervals</FIGCAPTION></FIGURE><P style="text-align:left"> Distribution of <I>bamA</I> mRNA is extremely spread out initially during the first 40 minutes due to the variation in starting conditions between runs and rapid initial changes in mRNA level in cases of low initial mRNA and very high DNA masses or cases of high initial mRNA and very low DNA masses. By 120 minutes the distribution has settled in a number of distinct peaks corresponding to parameter sets leading to a particular <I>bamA</I> mRNA equilibrium value. </P><P style="text-align:left"> Figure 5 displays average levels of BamA expression per combination of starting conditions. </P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 5. Average BamA per combination of starting conditions</FIGCAPTION></FIGURE><P style="text-align:left"> By examining the model results broken down by starting conditions we can make recommendations about how best to proceed in the wet lab. From Figure 5 we can see by comparing BamA expression with 25 nanograms DNA and 0 micrograms RNA added compared to when RNA is added for the same mass of DNA there is a much increased rate of initial BamA synthesis as a result of mRNA addition. The increased rate of BamA synthesis due to mRNA addition is present in a lesser extent as higher initial masses of DNA are used. When using the maximum DNA mass of 1000 nanograms addition of mRNA in the quantities supported by the PURE system make little effect to protein expression.</P></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H2 style="text-align:left">Sensitivity Analysis</H2><P style="text-align:left"> Sensitivity Analysis is a vital part of the modelling work flow, by analyzing which parameters contribute most to the uncertainty of the models output we can recognize those parameters that are most important to fine tune from wet lab results.</P><P style="text-align:left"> Fourier Amplitude Sensitivity Testing (FAST) is a computationally efficient method to calculate variance based sensitivity indices used here via the SALib library <SUP>[16]</SUP>. Intuitively variance represents the spread of a set of numbers. FAST indices represent the proportion of the output variance of the model attributable to a particular variable and its interactions. Focusing on BamA expression as the protein of interest total order FAST sensitivity indices were calculated using the BamA protein level each 20 minutes as the model output. </P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 5. FAST Total order sensitivity indices</FIGCAPTION></FIGURE><P style="text-align:left">  Number of BamA plasmid copies contributes most to output variance over the whole time span due to having an extremely high value and having a major effect on where mRNA equilibrium occurs. <I>bamA</I> mRNA degredation rate is considerably faster than BamA degredation rate - with mRNA halflife of the order of minutes and protein halflife the order of hours - hence the greater FAST index. </P></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H2 style="text-align:left">Conclusion</H2><P style="text-align:left"> Whilst the addition of <I>bamA</I> mRNA does increase the rate of protein expression this effect is proportional to the mass of initial DNA. Initial mass of BamA DNA determines at how high an expression level the equilibrium for BamA expression is reached. In order for addition of mRNA to give the most benefits it should be used where the mass of expressed protein does not need to be particularly high but where the protein needs to quickly present. Due to the shorter size of proteins lgA and OmpA compared to BamA it is not necessary to have maximal expression of these proteins to attain them in an equivalent quantity to BamA. In cases where maximal expression level of BamA is desired (by using 1000 nanograms DNA) the addition of mRNA in the quantities supported by the PURE system produces little change to BamA expression and hence may not be justified given the extra wet lab work incurred.</P></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H2 style="text-align:left">References</H2><OL><LI style="text-align:left">PURE cell free system - https://international.neb.com/products/e6800-purexpress-invitro-protein-synthesis-kit</LI><LI style="text-align:left">A. B. Kudryavtsev, R. F. Jameson, and W. Linert.  The Law of Mass Action.  Springer Science &amp; Business Media, 2001.</LI><LI style="text-align:left">IDT Molecular Facts and Figures - http://sfvideo.blob.core.windows.net/sitefinity/docs/default-source/biotech-basics/molecular-facts-and-figures.pdf?sfvrsn=4563407_4</LI><LI style="text-align:left"> peT2AB - https://www.addgene.org/vector-database/2568/</LI><LI style="text-align:left"> pRSETb - https://www.addgene.org/vector-database/3991/</LI><LI style="text-align:left"> Transcription Rate - http://book.bionumbers.org/what-is-faster-transcription-or-translation/</LI><LI style="text-align:left">BamA - https://www.uniprot.org/uniprot/P0A940</LI><LI style="text-align:left"> OmpA - https://www.uniprot.org/uniprot/P0A910</LI><LI style="text-align:left"> lgA - https://www.uniprot.org/uniprot/A6Y9S5</LI><LI style="text-align:left"> mRNA half-life - http://book.bionumbers.org/how-fast-do-rnas-and-proteins-degrade/</LI><LI style="text-align:left"> Degradation Rate Equation - https://en.wikipedia.org/wiki/Exponential_decay</LI><LI style="text-align:left"> Translation Rate - http://book.bionumbers.org/what-is-faster-transcription-or-translation/</LI><LI style="text-align:left"> ProtParam - https://web.expasy.org/protparam/</LI><LI style="text-align:left"> N-end rule -  Bachmair A, Finley D, Varshavsky A (1986).  ”In vivo half-life of a protein is a function of itsamino-terminal residue”.  Science.</LI><LI style="text-align:left"> Mass <I>bamA</I> mRNA - https://www.aatbio.com/tools/calculate-RNA-molecular-weight-mw</LI><LI style="text-align:left"> SALib Sensitivity Analysis Library - https://salib.readthedocs.io/en/latest/</LI></OL></DIV></DIV></DIV></SECTION><SECTION id="about" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H2 style="text-align:left">Supplementary Information</H2><FIGURE class="figure"><FIGCAPTION class="figure-caption">Distribution of OmpA at 20 minute intervals</FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption">Distribution of OmpA mRNA at 20 minute intervals</FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption">Distribution of lgA at 20 minute intervals</FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption">Distribution of lgA mRNA at 20 minute intervals</FIGCAPTION></FIGURE></DIV></DIV></DIV></SECTION><SECTION id="contact" class="content-section text-center"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 mx-auto"><H2>Contact EdiGEM18</H2><P>Feel free to leave us a comment on social media!</P><UL class="list-inline banner-social-buttons"><LI class="list-inline-item"><A href="https://twitter.com/EdiGEM2018" class="btn btn-default btn-lg"><SPAN class="network-name">Twitter</SPAN></A></LI><LI class="list-inline-item"><A href="https://www.facebook.com/EdiGEM2018/?ref=br_rs" class="btn btn-default btn-lg"><SPAN class="network-name">Facebook</SPAN></A></LI></UL></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>