Team:Edinburgh UG/Degradation Switch
Edinburgh iGEM 2018
Home (current)
Team
Team Members Collaborations
Project
Description Design Experiments Notebook InterLab Model Results Demonstrate Improve Attributions
Parts
Parts Overview Basic Parts Composite Parts Part Collection
Safety (current)
Human Practices (current)
Awards
Model
Judging Form (current)
Medal Criteria(current)
DNA Degradation Switch
Introduction
We aim to provide a novel chassis with improved bio-safety capabilities. This will take the form of a Maxicell - achromosomal E. coli - which is incapable of reproducing or of performing horizontal gene transfer. In order to prevent horizontal gene transfer we intend to take a two pronged approach; Semantic Containment will mean that other organisms should not be able to 'read' the DNA of the Maxicell if they should take it up and a DNA degrading killswitch will break up the Maxicell plasmid(s) after a given time span.
DNA Degradation Switch Concept
The implementation of our DNA degradation system requires 2 constructs:
Colicin E2 plasmid (pCol) -  Colicin E2 is a nicking endonuclease which acts without specificity to cut both single and double stranded DNA. This plasmid will contain Constitutive Anderson Promoter, RBS, Colicin E2 coding sequence and T1 Terminator.
Immunity Plasmid - Imm2 is the immunity protein for Colicin E2. Imm2 binds to Colicin E2 with high affinity and when present in equimolar concentration prevents its DNase activity. This plasmid will contain Constitutive Anderson Promoter, RBS, Imm2 Coding Sequence and T1 Terminator. In addition a site for the homing endonuclease I-SceI will be present that when cut turns off imm2 expression.
Prior to Maxicell formation our E. coli will be transformed with pImm, this will cause a build up of Imm2 within the cell. At this phase of the model only equations for Imm2 synthesis are evaluated.
Once Imm2 concentration has reached equilibrium the E. coli can then be transformed with pCol. Colicin E2 will be expressed within the cell however the build-up of Imm2 and its continued expression ensure it remains inactive. At this phase in the model all equations are evaluated.
In order to break up the chromosome of our E. coli and form Maxicells we have obtained a strain from the Leach Lab containing a number of I-SceI sites within the chromosome. When I-SceI is introduced and Maxicells are formed pImm will also be cut 'turning off' expression of Imm2. At this phase in the model all equations are evaluated however rate of imm2 transcription is 0 and halflives for Imm2 and Colicin E2 are altered to reflect that replication is no longer taking place.
By modeling this simple killswitch we can determine the plausibility of the system as well as screen across different Anderson Promoters and RBS to find the killswitch activation time closest to the length of time for which Maxicells can continue expressing protein.
Mass Action Equations
Mass Action Equations are commonly used to represent chemical reactions and provide a starting point for mechanistic modelling of a variety of phenomena. The laws of mass action state that the rate of any chemical reaction is proportional to the product of the masses of the reacting substances, with each mass raised to a power equal to the coefficient that occurs in the chemical equation [2]. The mass action equations in Figure 1 can be used to represent protein expression when taking into account the effect of ribosome binding sites and promoters:
Ordinary Differential Equations
The model uses a simple set of ordinary differential equations derived from mass action equations:
pimm and pcol - Promoter Strength - The Anderson Promoter Collection [1] consists of 19 constitutive promoters that have been characterized by their relative strengths in E. coli.
trimm - Rate of Imm Transcription - Transcription proceeds at an average speed of 60 nucleotides per second [2]. Imm is 258 nucleotides in length [3] giving transcription rate of 60/258. In Maxicells imm is not transcribed and hence this rate is set to 0.
degmRNA - Rate of mRNA Degradation - Average mRNA halflife in E. coli is 5 minutes [4]. degmRNA = ln(2)/halflife [4].
trcol - Rate of Col Transcription - Transcription proceeds at an average speed of 60 nucleotides per second [2]. Col is 1743 nucleotides in length [5] giving transcription rate of 60/581.
rbsimm and rbscol - RBS Strength - The Community RBS Collection [6] consists of ribosome binding sites that have been characterized by their relative strengths in E. coli. We evaluated our model using RBS strengths 0.6, 0.3 and 0.07.
trlimm - Rate of Imm2 Translation - Translation proceeds at an average speed of 20 amino acids per second [2]. Imm2 is 83 amino acids in length [3] giving translation rate of 20/86.
degimm - Rate of Imm2 Degradation - in replicating E. coli effective halflife is the time taken for a generation approximately 20 minutes. In Maxicells halflife was determined using ProtParam Tool [7]. ProtParam uses the N-end rule to determine protein halflife, the estimates given for Imm2 are >10hrs in E. coli and 30hrs in reticulocytes hence an average 20hr halflife is used. degimm = ln(2)/halflife [4]
trlcol - Rate of Colicin E2 Translation - Translation proceeds at an average speed of 20 amino acids per second [2]. Colicin E2 is 581 amino acids in length [5] giving translation rate of 20/581.
degcol - Rate of Colicin E2 Degradation - in replicating E. coli effective halflife is the time taken for a generation approximately 20 minutes. In Maxicells halflife was determined using ProtParam Tool [7]. ProtParam uses the N-end rule to determine protein halflife, the estimates given for Colicin E2 are >10hrs in E. coli and 30hrs in reticulocytes hence an average 20hr halflife is used. degcol = ln(2)/halflife [4]
Results
Figure 2 demonstrates the results obtained from an initial non-screening run to investigate the plausibility of the killswitch design. Results indicate that free Colicin E2 will be present after 19.6 hours which is promising given the range of promoters and ribosome binding sites over which to screen. Quantities given of Imm2 mRNA, Colicin E2 mRNA, Imm2 and Colicin E2 are presented in relative units.
Figure 2 - Initial model results - free Colicin E2 is present after 19.6 hours with pimm ,pcol ,rbsimm and rbscol equal to 1.
The results from this initial run are characteristic of those obtained when screening over Anderson Promoters and Ribosome Binding Sites. With only pImm transformed (left of first red line) we observe Imm2 mRNA very rapidly reaching equilibrium and Imm2 also reaching equilibrium shortly after. pCol is transformed (between first and second red lines) and reaches a much lower equilibirum at a slower rate compared to Imm2 due its larger size and hence slower rates of transcription (of Colicin E2 mRNA) and translation. Maxicell formation occurs (at second red line) and pImm is as a result linearized, we observe an immediate spike in Imm2 production due to the now longer halflife of Imm2 when in non-dividing Maxicells however this quickly drops as no new Imm2 mRNA is produced. We can also observe considerably higher Colicin E2 expression once maxicells are formed which is again due the longer halflife of proteins when not being diluted due to cell division. Free Colicin E2 becomes present at 19.6 hours and hence DNA degradation occurs and maxicells are deactivated.
Figure 3 displays the distribution of different killswitch activation times obtained by screening across Anderson Promoters and Ribosome Binding Sites. We can observe that there is a wide variation in activation times between 1.8 minutes and 194 hours interestingly we can also observe that the distribution closely approximates a power law distribution with fast activation time (0-50 hours) being significatly more numerous than those with longer activation times (>50 hours).
Figure 3 - Distribution of Degradation Switch Activation Times
Figure 4 displays the protein levels over time for runs 20, 457 and 2585 which correspond to the slowest, fastest and average activation times of 194 hours, 0.03 hours (1.8 minutes) and 33.28 hours.
Figure 4 - Protein Levels for Average, Minimum and Maximum Activation Times
Figure 4 demonstrates our success in generating a wide variation in expression patterns by our screening approach. Table 1 gives the parameter values used in each of these model runs for reference:
Model Run
Parameter
Relative Strength
20
Imm2 Promoter Strength
1.0
Colicin E2 Promoter Strength
0.01
Imm2 Ribosome Binding Site Strength
0.6
Colicin E2 Ribosome Binding Site Strength
0.07
457
Imm2 Promoter Strength
0.01
Colicin E2 Promoter Strength
0.47
Imm2 Ribosome Binding Site Strength
0.6
Colicin E2 Ribosome Binding Site Strength
0.6
2585
Imm2 Promoter Strength
0.15
Colicin E2 Promoter Strength
0.58
Imm2 Ribosome Binding Site Strength
0.6
Colicin E2 Ribosome Binding Site Strength
0.07
Sensitivity Analysis
Sensitivity Analysis is a vital part of the modelling work flow, by analyzing which parameters contribute most to the uncertainty of the models output we can recognize those parameters that are most important to fine tune from wet lab results.
Table 2 below summarises the average change in Degradation Switch Activation Time per 1% change in parameter value.
Parameter
Average change to Degradation Switch activation time per 1% change in parameter value (hours)
Imm2 Promoter Strength
0.549
Colicin E2 Promoter Strength
0.928
Imm2 Ribosome Binding Site Strength
0.591
Colicin E2 Ribosome Binding Site Strength
0.659
Fourier Amplitude Sensitivity Testing (FAST) is a computationally efficient method to calculate variance based sensitivity indices used here via the SALib library [8]. Intuitively variance represents the spread of a set of numbers. FAST indices represent the proportion of the output variance of the model attributable to a particular variable and its interactions. Focusing on activation time as the primary result of interest total order FAST sensitivity indices were calculated and are displayed in Table 3.
Parameter
Total Order FAST index
Imm2 Promoter Strength
0.482
Colicin E2 Promoter Strength
0.505
Imm2 Ribosome Binding Site Strength
0.316
Colicin E2 Ribosome Binding Site Strength
0.514
The main take away from our sensitivity analysis is the importance of those parameters which affect Colicin E2 expression versus those acting on Imm2 expression, using both sensitivity analysis methods we find that Colicin E2 promoter and ribosome binding site strengths contribute more to both the activation time and output variance than do Imm2 promoter and ribosome binding site strengths. If we consider the action of the Degradation Switch within maxicells pImm has been linearized which means in the period leading up to Degradation switch activation only those parameters affecting Colicin E2 expression will have any impact hence their greater effect on the model output.
Conclusions
By utilising our DNA Degradation Switch Model we were able to verify that our initial design concept was plausible as well as identify exactly those promoters and ribosome binding sites which in combination lead to particular activation times directly informing our part design. Figure 5 shows the results for model run 684 with . Run 684 was selected to be used as the basis for our part design due to its 11.45 hour activation time which we deemed long enough when tested in the wet lab to validate the production of a significant delay before degradation switch activation and yet short enough not to make validation too laborious. To read more about the wet lab protocol we designed in collaboration with Team Vilnius-Lithuania for degradation switch validation see our collaboration page here .
Figure 5 - mRNA and Protein levels for 11.45 hour activation time Degradation Switch
Despite the success of our modelling approach there are some simplifications which were necessary to make due to lack of knowledge about parameter values invloved in a more complex solution. The main simplification which may contribute to inaccurate results was to not account for leakiness in the expression of I-SceI which may cause premature linearization of pImm and hence premature activation of degradation switch due to a lack of data on the leakiness of the arabinose promoter under which I-SceI is expressed. We felt secure that this simplification would not have a major impact on our predictions due to the observation that maxicell progenitor strain DL2524 does not spontaneously decay into maxicells without induction of I-SceI by addition of arabinose as would happen if this promoter was significantly leaky. Despite our inability to evaluate such a system we have designed a system of Ordinary Differential Equations to take leakiness into account which is presented here as our extended model.
Extended Model
Figure 6 - Extended DNA Degradation Switch Ordinary Differential Equations
Examining the extended model we can see the main changes from our simple degradation switch model are in the equations governing:
Rate of Change of Imm2 mRNA - We modulate the production term of Imm2 mRNA based on the percentage of pImm plasmids remaining unlinearized allowing us to keep track of the exact rate of change of Imm2 mRNA as plasmids are linearized.
Rate of Change of I-Sce1 mRNA - I-Sce1 is expressed under an arabinose promoter. The addition term in this equation represents the standard method to represent expression under an inducible promoter with trleak being the transcription rate due to leakiness (transcription rate when promoter is uninduced) and hill the hill coefficient representing the effect that the binding of a ligand encourages the binding of additional ligands.
Rate of Change of Arabinose - after initial addition of arabinose the arabinose will degrade over time resulting in lower and lower expression of I-Sce1
Interaction of I-Sce1 to linearize pImm (Equations 7-10) - Derived from the Michaelis Mentens Enzyme Kinetics Model [9] to represent the interaction of I-Sce1 with its substrate; the recognition site present in circularized pImm and the resulting production of product in the form of linearized pImm.
References
Anderson Promoter Collection - http://parts.igem.org/Promoters/Catalog/Anderson
Bionumbers - http://book.bionumbers.org/
Imm2 - https://www.uniprot.org/uniprot/P04482
Exponential Decay - https://en.wikipedia.org/wiki/Exponential\_decay
Colicin E2 -  https://www.uniprot.org/uniprot/P04419
RBS Collection - http://parts.igem.org/Ribosome_Binding_Sites/Prokaryotic/Constitutive/Community_Collection
ProtParam - https://web.expasy.org/protparam/
SALib Sensitivity Analysis - https://salib.readthedocs.io/en/latest/basics.html#what-is-sensitivity-analysis
Michaelis Mentens Enzyme Kinetics- https://www.ncbi.nlm.nih.gov/books/NBK22430/
Contact EdiGEM18
Feel free to leave us a comment on social media!
Twitter
Facebook
