"
Page
Discussion
View source
History
teams
Log in
 
Team:LCG-UNAM-Mexico:WTM
From 2009.igem.org
Home
Record
Project
Description>>
Design
Delivery
Defense
Parts
Resources
Safety
Results
Conclusions
Modelling
Wet Lab
Objectives
Team
Team
Acknowledgments
Collaborations
Sponsors
Photos!!!
Site map
WILD-TYPE MOLECULAR MODEL (WTM)
WTM Simbiology ® diagram
WTM is a stochastic molecular model of bacteriophage T7 life cycle , it was constructed to simulate wild-type development of this phage in E Coli.
Both models, WTM and KZM , were constructed to monitor the evolution and abundance of molecular species in the systems. For the purpose of this project we are particularly concerned in the abundance of phage T7 (burst size) at the end of the cycle (720 seconds) in each infection event.
Ensembles of runs of WTM and KZM will provide us with data to build Burst Size Distributions for each model.  BSDs are next utilized by the population models to recreate the impact of phage infections at the population level.
See also KZM , BSD , Population Model , Cellular Automata .
Contents
1 WILD-TYPE MOLECULAR MODEL (WTM)
2 Insertion and translocation of T7 DNA at different times
3 Transcription of different T7 DNA segments into polycistronic mRNAs
4 Degradation of phage mRNAs
5 Translation of phage mRNAs into proteins
6 T7 DNA replication
7 Procapsid Assembly
8 DNA packaging and final assembly
9 Simulation, performance and BSD construction
10 References
The following critical processes are accounted in this model:
Insertion and translocation of T7 DNA at different times
Entry of T7 DNA into the host cell occurs in several distinct stages.
Phage's DNA is arranged in three classes of genes depending on their positions, it is translocated into the cell between 6 to 10 minutes after attachment, so this order and timing drives the phage's development. This phenomenon of DNA translocation is modeled here taking into account reported insertion speeds F. Buchholtz et al. 1987[1].
Transcription of different T7 DNA segments into polycistronic mRNAs
It has been shown that T7 genes are expressed in overlapped polycistronic mRNAs.
Transcription of T7 polycistronic mRNAs occurs if and only if its coding DNA segment is available in the cell. Transcription is dependent of the set of genes inserted at a time.
We define a set of transcription rates for every polycistronic mRNA taking into account constant bacterial or T7 RNA polymerase elongation rates and the length of the polycistronic mRNA, these transcription rates will be our rate limiting steps at the transcriptional level.
Degradation of phage mRNAs
We assume the same degradation rate for all T7 polycistronic mRNAs. Until now impact of this phenomenon had not been studied. It has been found that phage messengers are stabler than Bacterial mRNAs .Yoshihiko Yamada et al 1975 [2].
Translation of phage mRNAs into proteins
In this model, translation is simulated assuming an environment of unlimited amino acids and ribosomes. We also assume that the rate at which ribosomes incorporate amino acids is constant over all T7 mRNA.
As it has been done for transcription we define a set of translation rates for every protein taking into account a constant ribosome elongation rate and the length of the protein, these translation rates will be our rate limiting steps at the translational level.
T7 DNA replication
DNA synthesis is simulated by taking elongation of T7 DNA polymerase as the rate-limiting step.
We assume an environment of limited free nucleotides so we can set a maximum number of T7 genomes produced in a single infection taking into account the size of the host genome (and this includes bacterial chromosomes, plasmids and other sources of free nucleotides).
Procapsid Assembly
This phenomenon is simulated in almost the same way as Drew Endy et al. 1996 [3] using mass action kinetics.
DNA packaging and final assembly
Both processes are modeled using mass action kinetics as well. This last step requires complete procapsids, T7 DNA, and enough of each structural protein to complete the phage. The simulation assumes that packaging of DNA into the procapsid is the rate-limiting step for T7 progeny formation.
Simulation, performance and BSD construction
The next video has been recorded from a single simulation of WTM, with this algorithm we can monitor the evolution and abundance of molecular species in time.
For the purpose of our project we're just concerned in the burst size an infected cell could produce in each infection process, as we said above BSDs for WTM and KZM can be assembled with many runs of each model so we can compare the performance of our synthetic kamikaze and the wild-type systems at molecular and population levels.
Generated BSDs are next used by Cellular Automaton to choose the burst size an infected cell (with and without our kamikaze system) is going to release into the medium.
BSDs are also use by the agent-based model, in both cases BZDs drives the fate of the population.
In this video there are two molecular species plotted, ABproteins or Major head proteins, this protein is the main component of T7 procapsids, it is evident the upper limit of molecules at which a procapsid assembly occurs. Production of this protein grows faster at late stages when polycistronic mRNAs responsible for them become more and more abundant in the cell. T7 plotted in red grows until it reach a burst size of about 250 phages at the end of the cycle/simulation (lysis time 720 seconds).
References
[1] F. Buchholtz and F.W. Schneider. 1987. Computer simulation of T3/T7 phage infection using lag times
[2] Yoshihiko Yamada, Patricia A. Whitaker and Dai Nakada. 1975. Chemical Stability of Bacteriophage T7 Early mRNA
[3] Drew Endy, Deyu Kong, John Yin. 1996. Intracellular Kinetics of a Growing Virus: A Genetically Structured Simulation for Bacteriophage T7
Retrieved from " http://2009.igem.org/Team:LCG-UNAM-Mexico:WTM "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
