"
Page
Discussion
View source
History
teams
Log in
 
Team:LCG-UNAM-Mexico:KZM
From 2009.igem.org
Home
Record
Project
Description>>
Design
Delivery
Defense
Parts
Resources
Safety
Results
Conclusions
Modelling
Wet Lab
Objectives
Team
Team
Acknowledgments
Collaborations
Sponsors
Photos!!!
Site map
KAMIKAZE MOLECULAR MODEL (KZM)
KZM Simbiology ® diagram
As WTM , KZM is a stochastic molecular model of bacteriophage T7 life cycle , it was constructed to simulate the interplay between the kamikaze system and the phage T7 infection.
Both models, WTM and KZM, were constructed to monitor the evolution and abundance of molecular species in the systems.
For the purpose of this project we are particularly concerned in the abundance of phage T7 (burst size) at the end of the cycle (720 seconds) in each infection event.
Ensembles of runs of WTM and KZM will provide us with data to build Burst Size Distributions for each model. A Kamikaze BSD is next utilized by the population models to recreate the impact of our synthetic circuit in the infection at population level.
See also WTM , BSD , Population Model , Cellular Automata .
Contents
1 KAMIKAZE MOLECULAR MODEL (KZM)
2 Insertion and translocation of T7 DNA at different times
3 Transcription of different T7 DNA segments into polycistronic mRNAs
4 Transcription of kamikaze system
5 Degradation of phage mRNAs
6 Translation of phage mRNAs into proteins
7 Translation of kamikaze mRNA
8 T7 DNA replication
9 Procapsid Assembly
10 DNA packaging and final assembly
11 Sabotage: Inactivation and decay of ribosomes
12 Simulation, performance and BSD construction
13 References
The following critical processes are accounted in this model:
Insertion and translocation of T7 DNA at different times
Entry of T7 DNA into the host cell occurs in several distinct stages.
Phage's DNA is arranged in three classes of genes depending on their positions, it is translocated into the cell between 6 to 10 minutes after attachment, so this order and timing drives the phage's development. This phenomenon of DNA translocation is modeled here taking into account reported insertion speeds by Drew Endy et al. 1996 [1] and F. Buchholtz et al. 1987 [2].
Transcription of different T7 DNA segments into polycistronic mRNAs
It has been shown that T7 genes are expressed in overlapped polycistronic mRNAs.
Transcription of T7 polycistronic mRNAs occurs if and only if its coding DNA segment is available in the cell. Transcription is dependent of the set of genes inserted at a time.
We define a set of transcription rates for every polycistronic mRNA taking into account constant bacterial or T7 RNA polymerase elongation rates and the length of the polycistronic mRNA, these transcription rates will be our rate limiting steps at the transcriptional level.
Transcription of kamikaze system
Expression of kamikaze system occurs when T7 RNA polymerase is present in the cell, this protein activates transcription through a T7 RNA pol promoter.
Rate limiting step for this reaction is the elongation rate of T7 RNA polymerase, a translation rate for this polycistronic mRNA was constructed taking into account this limiting rate and the length of the produced messenger.
Kamikaze polycistronic mRNA codes for GFP, rRNAse domain of colicin E3(ColE3) and luxI.
Degradation of phage mRNAs
We assume the same degradation rate for all T7 polycistronic mRNAs. Until now impact of this phenomenon had not been studied. It has been found by Yoshihiko Yamada et al. 1976 [3] that phage messengers are stabler than Bacterial mRNAs .
Translation of phage mRNAs into proteins
In this model, translation is simulated assuming an environment of unlimited amino acids
On  the other hand, Quantity of ribosomes changes over time depending on the toxic action of ribosome-inactivating rRNAse domain of colicin E3.
We also assume that the elongation rate at which ribosomes incorporate amino acids is constant over all T7 mRNAs.
We define a set of initial translation rates and they are coupled to the ribosome decay so they change over time as well. Kamikaze system affect T7 cycle by reducing the number of bacteriophage proteins and therefore lowering the number of assembled T7 particles at the end of phage life cycle.
Translation of kamikaze mRNA
Translation occurs as any other protein, its production decreases as a function the abundance of ribosomes and its translation rates.
T7 DNA replication
DNA synthesis is simulated by taking elongation of T7 DNA polymerase as the rate-limiting step.
We assume an environment of limited free nucleotides so we can set a maximum number of T7 genomes produced in a single infection taking into account the size of the host genome (and this includes bacterial chromosomes, plasmids and other sources of free nucleotides).
Procapsid Assembly
This phenomenon is simulated in almost the same way as Drew Endy et al. 1996[1] using mass action kinetics.
DNA packaging and final assembly
Both processes are modeled using mass action kinetics as well. This last step requires complete procapsids, T7 DNA, and enough of each structural protein to complete the phage. The simulation assumes that packaging of DNA into the procapsid is the rate-limiting step for T7 progeny formation.
Sabotage: Inactivation and decay of ribosomes
The main actor of this sabotage is rRNAse domain of colicin E3 (colE3), this toxin cleaves 16s rRNAs in active ribosomes rendering them useless to translate or inactive, it is expected a decay in the number of functional ribosomes and in the overall bacterial translation.
Simulation, performance and BSD construction
The next video has been recorded from a single simulation of KZM, molecular species monitoring in time is one of the fundamental objectives of this mode.
In order to accurately simulate the infection at population scale we study the amount of phages an infected bacteria will produce at the end of the lytic cycle of T7, burst size. Furthermore we construct burst size distributions of KZM (BSD) and WTM so we can compare the performance of our synthetic kamikaze system vs wild-type behavior.
The following video plots the abundance of ABproteins a main procapsid component of viral coat and assembled T7 phages, it is noteworthy to see the impact of the toxin in the amount of viral proteins and in the final number of assembled particles of T7, burst sizes are significantly lower than the wild-type burst sizes, for more details go to BSD .
Ribosome decay can be monitored in the next video. The action and abundance of the toxin bring about an overall translation decay in the cell, as a consequence of this viral protein production is reduced, this in turn provokes an unfavorable amount of phages at the end of the cycle. We also plot the amount of ribosome-inactivating colicinE3 in purple, GFP in green and LuxI (A quorum sensing protein) in red.
References
[1] Drew Endy, Deyu Kong, John Yin. 1996. Intracellular Kinetics of a Growing Virus: A Genetically Structured Simulation for Bacteriophage T7
[2] F. Buchholtz and F.W. Schneider. 1987. Computer simulation of T3/T7 phage infection using lag times
[3] Yoshihiko Yamada, Patricia A. Whitaker and Dai Nakada. 1975. Chemical Stability of Bacteriophage T7 Early mRNA
Retrieved from " http://2009.igem.org/Team:LCG-UNAM-Mexico:KZM "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
