<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_LCG-UNAM-Mexico_AbrahamJurnal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:LCG-UNAM-Mexico/AbrahamJurnal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:LCG-UNAM-Mexico/AbrahamJurnal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/AbrahamJurnal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/AbrahamJurnal&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:LCG-UNAM-Mexico/AbrahamJurnal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:LCG-UNAM-Mexico/AbrahamJurnal</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV class="MenuBar" id="LCG"><UL><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico" onmouseover="window.status= 'Click here to return Home'; return true;">Home
                </A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Record">Record
                </A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description">Project</A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description"><SPAN><SPAN>Description</SPAN>&gt;&gt;</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description"><SPAN><SPAN>Design</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description#a)_Delivery"><SPAN><SPAN>Delivery</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description#b)_Defense"><SPAN><SPAN>Defense</SPAN></SPAN></A></LI></UL><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Parts"><SPAN><SPAN>Parts</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Resources"><SPAN><SPAN>Resources</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Safety"><SPAN><SPAN>Safety</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Results"><SPAN><SPAN>Results</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Conclusions"><SPAN><SPAN>Conclusions</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Modelling">Modelling</A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Wet_Lab/Objectives">Wet Lab</A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Wet Lab/Objectives"><SPAN><SPAN>Objectives</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team#The_team"><SPAN><SPAN>Team</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team#Acknowledgments"><SPAN><SPAN>Acknowledgments</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team#Collaborations"><SPAN><SPAN>Collaborations</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team#Sponsors"><SPAN><SPAN>Sponsors</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team/Photos"><SPAN><SPAN>Photos!!!</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Site map">Site map</A></LI></DIV><H2><SPAN class="mw-headline" id="Abraham.27s_lab_journal"><B>Abraham's lab journal</B></SPAN></H2><P><B>June 26th</B></P><P>I started working in the new lab. 
</P><P><B>== Objectives: ==</B></P><P>	The final aim is to earn all the following devices and them all together into the plasmid we are suggesting, that will be extracted from P4
</P><P>I learned to obtain the plasmid sent by iGEM in the 2009 kit plates, and to transform it, the biobricks we are trying to obtain are:
</P><PRE>  BBa_R1062		Promoter activated by LuxR-HSL complex   BBa_I1352 		RFP constitutively expressed and repressed with tetracycline  BBa_J37033   		LuxR protein  BBa_C0261  		AHL making enzyme  BBa_B0015 		Double transcriptional terminator  BBa_P1003     	Kanamycine ressistance casette  BBa_J04450    	mRFP</PRE><P><B>June 27th-July 7th</B></P><P>Colony PCR from grown colonies and plasmid extraction from true possitives.
</P><P><B>June 8th</B></P><P>My birthday! enjoying it was all my work.
</P><P><B>July 10th</B></P><P>We noticed that we needed some other biobricks in order to test the function of our principal components, so we transformed the plasmids coming in the 2009 kit plates:</P><PRE> BBa_I715038 		ready to use T7 RNA polymerase inducible with IPTG
 BBa_K093005		RBS-RFP
 BBa_I13507 		RBS-RFP-Transcriptional terminator
</PRE><P>The T7 RNA polymerase is going to be used to test the functionality of the final device and of the partial constructions with this promoter.
</P><P>RBS-RFP is going to be used after the asRNA-double_transcriptional_terminator as a reporter gene. (Check asRNA design)
</P><P>RBS-RFP-T.terminator If is the case that we can not afford the asRNA after other constructions are finished, we will use this biobrick as a reporter, It would be with a promoter inducible with luxR-HSL dimer (BBa_R1062).
</P><P><B>July 14th</B></P><P>Plan to ligate double terminators (BBa_B0015) at the end of the HSL making enzyme (BBa_C0261) the kanamycine resistance cassette (BBa_P1003)
</P><P><B>July 17-26th </B></P><P>We MUST take vacation, the lab’s people will be on vacation and we’re not able to stay here without supervision of the advisor. We are supposed to read about general things involved in our project and also about applications in human’s health.
</P><P><B>Aug 6th</B></P><P>We recieved an e-mail from Mr. Gene, They say that E3 and E9 colicin domains have always been tricking to synthesize, The E3 toxin has already been amplified by PCR, but the E9 sequence has been dificult to obtain.
The GFP biobrick is cloned into any plasmid from Mr.Gene 
</P><P>It seems that they were not able to clone neither E3 nor E9 colicines, we are worried about the dificulties that we will have to clone them.
</P><P><B>Aug 7th</B></P><P>From the transformations we performed on Aug 5th we observed some colonies, we expected them to grow in just 24hrs, but it took almost 2 days. we decided to perform colony PCR from.
</P><PRE>     Ligations				Number of colonies	Lines
     BBa_R1062 + BBa_I13507 		5			1 - 5 big
     Cox--&gt;BBa_J04450*			2			6, 7 big 
     BBa_C0261  + BBa_B0015 		2			8, 9 big
     BBa_K116640 + BBa_K116640		8			11-18 big
     OGR +  BBa_B0015 			5			1-5 small
     PCR positive control		/			6 small
     PCR negative control		/  			7 small	
</PRE><UL><LI>The arrow means that we want to clone the Cox gene into the plasmid pSB1T3 containing BBa_J04450 as a marker.
</LI></UL><P>As seen in the 1%  Agarose gel, we got 4 true positive colonies in the OGR +  BBa_B0015 transformation, and 2 true positive for  BBa_J04450 + Cox.
</P><P><B>September 7th</B></P><P>We reasoned that the idea of asRNA.  If we design and prove that is functional we will repress the fage infection easier.
The idea is that infected bacteria will send HSL to alarm their neighbors about the infection, afterwards the bacteria close to this place will express the asRNA, so if lytic phase of phages is faster than the transcription machinery and the toxin’s action  time (This implies that the first infected bacteria will “lose” even when contains the construction), the bacterial population next to the infected will express an antisense that if the lytic phages infect one of the alerted bacteria, it will be already prepared with the antisense. 
</P><P><B>Sept 8th – 18th Design of the antisensense RNA</B></P><P>We looked for any target that was mentioned as essential in the literature. We decided to attack phages in their replication process, because of the targets we found, so we reasoned that if the phage tries to replicate it’s genome the asRNA is going to block the process reducing dramatically the burst size or if the efficiency is high, there wont be any new phage.
The target is NOT the RNA polymerase because we want it to express the kamikaze device.
</P><P>The design included literature reading of examples to block the translation of proteins in bacteria, and particularly to those in which the target sequences were phages’.  The 5’ of the transcript should be blocked. The replication is a good target, in some articles some of the components of the replication were targeted and the efficiency of plaquing diminished as well as the size of the plaques.
</P><P>We met Dr. José Luis Reyes Taboada who is expert in the field of RNAs. With his guidance we looked for structures in RNAFold with high Gibbs energy values (the less negative, the better), we also checked that the structures were not blocking the RBS.
</P><P>We planned to include a marker such as mRFP in the 5' side of the antisense, but we though that the translation of this protein would interfer with the function of the antisense, thus we will include this biobrick induced by the complex LuxR-HSL (as well as the mRFP), but in different cistrons.
</P><P>Why not to use the same target mRNA for both phages? Because the upstream sequences of the essential genes are not conserved more than 90%, and the efficiency of silencing would decrease dramatically for one if we take the antisense sequence of the other.
Why not use the two different sequences but for the same target in the two phages (for example, both DNA polymerases)? Because they are more similar than any other pair of sequences, so they would interfere with each other. The alternative strategy is atack the same process but with different components
</P><P>The final choices are the following sequences: 
</P><P>GGGTGGCCTTTATGATTATCATTTAGCACGAAACCAAAGGAGGGCATTATGCTCGTAAGTGACATTGAGG
t3 DNA polymerase
AAACGAAACCTAAAGGAGATTAACATTATGGCTAAGAAGATTTTCACCTCTGCGCTGGGTACCGCTGAACCTTACGCTTACAT
t7  gp2.5 ssDNA Binding Prot
</P><P>
Their reverse complement sequences are: 
</P><UL><LI>CCTCAATGTCACTTACGAGCATAATGCCCTCCTTTGGTTTCGTGCTAAATGATAATCATAAAGGCCACCC.
</LI></UL><P>Form two possible secondary structures of -10 and -11 kcal/mol.
</P><UL><LI>ATGTAAGCGTAAGGTTCAGCGGTACCCAGCGCAGAGGTGAAAATCTTCTTAGCCATAATGTTAATCTCCTTTAGGTTTCGTTT
</LI></UL><P>Three possible secondary structures between -13.3 and 14.3 kcal/mol
</P><P> 
Concatenated: 
CCTCAATGTCACTTACGAGCATAATGCCCTCCTTTGGTTTCGTGCTAAATGATAATCATAAAGGCCACCCATGTAAGCGTAAGGTTCAGCGGTACCCA
GCGCAGAGGTGAAAATCTTCTTAGCCATAATGTTAATCTCCTTTAGGTTTCGTTT
</P><P>With 4 possible folds between -29.3 and -30.4 kcal/mol
</P><P>In order to send the sequence to synthesis, we added the prefix, the suffix, after the concatenation of the asRNA with the double terminator (BBa_B0015), without any scar. 
</P><P>At the end we send to synthesis the following sequence.
</P><P>&gt;Prefix_PLux_AsRNA(t3DNApol)_asRNA(t7SSBProt)_DoubleTerminator_Suffix
GAATTCGCGGCCGCTTCTAGACCTGTAGGATCGTACAGGTTGACACAAGAAAATGGTTTGTTGATACTCGAATAAACCTCAATGTCACTTACGAGCATA
ATGCCCTCCTTTGGTTTCGTGCTAAATGATAATCATAAAGGCCACCCATGTAAGCGTAAGGTTCAGCGGTACCCAGCGCAGAGGTGAAAATCTTCTTAG
CCATAATGTTAATCTCCTTTAGGTTTCGTTTCCAGGCATCAAATAAAACGAAAGGCTCAGTCGAAAGACTGGGCCTTTCGTTTTATCTGTTGTTTGTCGGT
GAACGCTCTCTACTAGAGTCACACTGGCTCACCTTCGGGTGGGCCTTTCTGCGTTTATATACTAGTAGCGGCCGCTGCAG
</P><P><B>September 17th</B></P><P>Mr.Gene shipment arrives. In this point my task is going to be the work with the sequences that just have arrived.
</P><PRE><B>== New biobricks work ==</B></PRE><P>Goals
Final goals: 
</P><OL><LI>Ensamble and prove the functionality of the kamikaze device.
</LI><LI>Get the time in wich a colicin, prefferentially E3, kills E. coli c1-alpha.
</LI><LI>Clone the bioparts received from Gene Art into any iGEM plasmid vector. Send them to the Registry of standard biological parts.
</LI></OL><P>Partial goals:
</P><OL><LI>Get the time in wich a colicin, prefferentially E3, kills Escherichia coli c1-alpha, the election of this is due to a <A href="https://2009.igem.org/Team:LCG-UNAM-Mexico:BSD#BSD_using_the_Kamikaze_System" class="external text" rel="nofollow">modeling suggestion</A>.
</LI><LI>Clone both colicines into any iGEM vector.
</LI><LI>Clone both colicines at the suffix of the biobrick BBa_R0010 in order to get an IPTG inducible device.
</LI><LI>Measure the time of death after the induction with IPTG.
</LI></OL><LI>Clone the MultiPromoter_GFP with all the death device
</LI><LI>Insert the MP_GFP biobrick into any iGEM vector.
</LI><LI>Insert the 2 colicines after the MP_GFP.
</LI><LI>Join this product with all the other biobricks.
</LI><P><B>September 21th.</B></P><P>High quality PCR performed to the synthesized  biobricks. This bioparts just arived from GeneArt shipment
</P><P>MultiPromoter T3/T7_GFP (furthermore refered as MP_GFP)-- We recieved 5ng of plasmid containing this biobrick
E3 RNase domain (furthermore called E3. Later registered as BBa_K242000)-- We recieved just 0.1ng of PCR product of this biobrick
E9 DNase domain (furthermore called E9. Later registered as BBa_K242029)-- We recieved just 0.1ng of PCR product of this biobrick
(Check Aug 6th for details)
</P><P>PCR results were with the expected size for all three sequences.
</P><P>Line 1.- DNA ladder
Line 2.- E9 DNase domain
Line 3.- E3 RNase domain
Line 4.- MP_GFP (there's a band in the size of the plasmid length)
</P><P><B>Oct 1st-3rd</B></P><P>Digestion reactions of the parts comming from mr gene in order to clone them.
</P><P><B>Oct 4th-5th</B></P><P>Ligation and transformation to the previus digested DNA.
</P><P><B>Oct6th</B></P><P>PCR colony for those that are still white. We expected to find true transformants, but from 25 colonies, just one contains a PCR product of the expected size.
Almost all the few colonies with the pSB1T3 as a plasmid are red. In the case of those that are withe (the Kanamicine resistance conteiners) has just kanamicine ligated! we are surprised becouse we weren't expecting that the kanamicine resistance was able to clone without the colicin, just becouse of restriction enzymes combinations!
</P><P><B>Oct 7th</B></P><P>Plasmid extraction and digestion of the transformant colony whose PCR seemed to amplify a fragment of the expected size. 
Now we have a clone with the biobrick multipromoter_GFP inserted into the plasmid pSB1T3.
</P><P><B>Oct 8th</B></P><P>Preparation by dilution of Preffix and Suffix primers.
High quality PCRs to E3 and E9, just to earn more of these DNA fragments in order to continue the efforts to clone them
</P><P><B>October 9th</B></P><P>Cut and  dephosphatation of BBa_J04450 cut by EcoRI and PstI which expeled RFP from the plasmid. This new plasmid pSB4A5 is used due to the lack of transformants for the plasmid pSB4K5, both of them are low copy number plasmid. We reasoned that a high copy number plasmid may be the cause of our failure to clone E3 and E9 bacteriocines.</P><P><B>Oct 10th</B></P><P>Digestion with different enzyme combinations to different biobricks.
</P><UL><LI>E9 &amp; E3
</LI></UL><P>	EcoRI/PstI
	Xba/PstI
	EcoRI/Spe
</P><UL><LI>Kanamicine resistance casette BBa_J04450
</LI></UL><P>	Xba/PstI
</P><UL><LI>Double transcriptional terminator BBa_B0015
</LI></UL><P>	Eco/Spe
</P><P>All of these digestions were performed in order to clone E3 and E9 into pSB4A5. 
Double transcriptional terminator will be inserted just before the colicines, as a control to stop basal transcription.
</P><P>
We also started the registration of the biobricks we will provide to the Registry of Standard Biological Parts
</P><P><B>Oct 11th</B></P><P>I performed ligations according to the protocol of the following biobricks:
</P><PRE> E3 --&gt;BBa_J04450, plasmid pSB4A5
 E9 --&gt;BBa_J04450, plasmid pSB4A5
 E9 + BBa_P1003  --&gt;BBa_J04450, plasmid pSB4A5 
 E9 + BBa_P1003 --&gt;BBa_J04450, plasmid pSB4A5
 BBa_R0010 + E3 (In the suffix of BBa_R0010)
 BBa_R0010 + E9 (In the suffix of BBa_R0010)
</PRE><P>he cultures containing BBa_P1003 are plated into Km antibiotic, thus it is supposed  that if the ligation is good then we will se just true positives or new resistant colonies.
</P><P><B>Octuber 13th-14th</B></P><P>The petri boxes now contain some colonies
</P><PRE> E3 --&gt;BBa_J04450*   3 colonies
 E9 --&gt;BBa_J04450*   0 colonies
 E9 + BBa_P1003  --&gt;BBa_J04450* many little colonies
 E9 + BBa_P1003 --&gt;BBa_J04450* many little colonies
 BBa_R0010 + E3   2 colonies
 BBa_R0010 + E9   6 colonies
 Negative control  0 colonies
 Ligation without insert control  1 colony
</PRE><UL><LI>Low copy number plasmid pSB1A2
</LI></UL><P>I managed to do colony PCR and placed the boxes with BBa_P1003  into the incubator again, this is because we suspect that the toxines are being detrimental for E. coli's fitness and that is the reason to observe that their growth is so slow.
</P><P>Lane 1.- DNA ladder
Lane 2.- Colony PCR positve control.
Lanes 3-4.-  BBa_R0010 + E3
Lanes 5-6.-  E3 --&gt;BBa_J04450
Lane 7.- PCR Negative control
Lane 8.- PCR of E3 from Mr Gene shipment.
Lanes 9-15.- BBa_R0010 + E9
</P><P>We can see in lines 3, 4, 9-12 with a band arround 200 bp, these are from the BBa_R0010 promoter, so again, we just obtained clones with one of the two inserts even when the restriction enzymes used are supposed to avoid this result.
</P><P><B>Octuber 15th-16th</B></P><P>We placed the biggest colonies of the plates with Km into the antibiotic for wich they have resistance into the plasmid, Ampiciline. In parallel we performed colony PCR.
</P><P>The results were negatives :( and the cultures in Amp were succesfull. But the morphology of the colonies don't look like E. coli. 
At the same time we observe the original plates with some fungi, so we conclude that the little colonies were contaminated with some mushroom.
</P><P><B>Octuber 18th</B></P><P>Due to the failed efforts to clone the colicines, we will try again after the wiki fresh. Maybe with the topo cloning technique.
</P><P><B>September 19th-20th</B></P><P>Registry of the bioparts into the Registry of Standard Biological parts and in work in the wiki. :)
</P><P><B>References of the asRNA design</B></P><P>Antisense RNA directed against the major capsid protein of Lactococcus lactis subsp. cremoris bacteriophage confers partial resistance to the host 
Dae Kyun Chung, Sung Kyun Chung and Carl A. Batt
Applied Microbiology and Biotechnology, Springer Berlin / Heidelberg, 0175-7598 1432-0614
</P><P>Antisense mRNA-Mediated Bacteriophage Resistance in Lactococcus lactis subsp. lactis
Sung Guk Kim and Carl A. Batt
Appl Environ Microbiol. 1991 April; 57(4): 1109-1113
</P><P>Engineering of the mRNA-interfering complementary RNA immune system against viral infection.
A Hirashima, S Sawaki, Y Inokuchi, and M Inouye
Proc Natl Acad Sci U S A. 1986 October; 83(20): 7726–7730. 
</P><P>Artificial Immune System against Viral Infection Involving Antisense RNA Targeted to the 5'-Terminal Noncoding Region of Coliphage SP RNA
The Journal of Biochemistry, Volume 106, Number 1, Pp. 163-166
Akikazu Hiroshima, Saeko Sawaki, Takafumi Mizuno, Nicole Houba-Herin and Masayori Inouye
</P><P>An Explosive Antisense RNA Strategy for Inhibition of a Lactococcal 
Applied and Environmental Microbiology, January 2000, p. 310-319, Vol. 66, No. 1
0099-2240/0/$04.00+0
</P><P>Bacteriophage T7 gene 2.5 protein: An essential protein for
DNA replication
Proc. Natl. Acad. Sci. USA Vol. 90, pp. 10173-10177, November 1993 Biochemistry
Young Tae Kim and Charles C. Richardson
</P><P>
Acidic Carboxyl-terminal Domain of Gen2e. 5 Protein of
Bacteriophage T7 Is Essential for Protein-Protein Interactions*
Young Tae Kim4 and Charles C. Richardson5
Vol. 269, No. 7, Issue of February 18, pp. 5270-5278, 1994 Printed in U.S.A.
</P><P>Essential Residues in the C Terminus of the Bacteriophage T7 Gene 2.5 Single-stranded DNA-binding Protein*
Boriana Marintchev,. Samir M. Hamdan,. Seung-Joo Lee and. Charles C. Richardson3
28, 2006, doi: 10.1074/jbc
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:LCG-UNAM-Mexico/AbrahamJurnal">http://2009.igem.org/Team:LCG-UNAM-Mexico/AbrahamJurnal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:LCG-UNAM-Mexico/AbrahamJurnal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:LCG-UNAM-Mexico/AbrahamJurnal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/AbrahamJurnal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/AbrahamJurnal&amp;oldid=167567" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>