"
Page
Discussion
View source
History
teams
Log in
 
Team:LCG-UNAM-Mexico:Molecular model
From 2009.igem.org
Home
Record
Project
Description>>
Design
Delivery
Defense
Parts
Resources
Safety
Results
Conclusions
Modelling
Wet Lab
Objectives
Team
Team
Acknowledgments
Collaborations
Sponsors
Photos!!!
Site map
Stochastic molecular models of bacteriophage T7 life cycle
Contents
Overview
Molecular Models
Wild Type Model
Kamikaze Model
The Burst Size Distribution
Results and achievements
References
Overview
The growth of a virus in its host cell is a complex and highly orchestrated multimolecular process; this complex molecular phenomenon gains importance when you want to study the behavior of designed systems that impact intracellular scales at first stages.
Based on previous works from Drew Endy et al.1996 we simulate chemical-kinetic systems of the life cycle of phage T7 in E. coli using a stochastic framework, one of the systems corresponds to the Wild-Type Model ( WTM ) of the life cycle of phage T7, in order to simulate the performance of our designed kamikaze system on the life cycle of phage T7 we made a second model or the KamikaZe Model ( KZM ), KZM integrates contribution of total number of ribosomes to overall translation rates and attack of toxins over  bacterial translation machinery.
Data published over the last 40 years on various fields about phage T7 is integrated here. These stochastic models incorporate entry of T7 genome into the host, transcription and expression of T7 genes, translation from phage polycistronic mRNAs into proteins, degradation of mRNAs, replication of T7 DNA, assembly of T7 procapsids, packaging of T7 DNA, T7 final particle assembly and in the case of the KZM, contribution of ribosomes to global translation rates, inactivation of ribosomes and decay of the global translation rates.
With a fairly big number of runs of both models we can assemble probability distributions for each molecular species as a function of time. To integrate this information into a multi-scale model we are particularly interested in the burst size Distribution ( BSD ); burst size is the number of phages an infected cell produce.
Once we have the BSD we are ready for the Spatial Population Model and finally for the whole integrative multi-scale model .
Molecular Models
WTM Simbiology ® diagram
KZT Simbiology ® diagram
Click on the text bellow the image to see description
WTM
KZM
Results and achievements
We decide to model the performance of the kamikaze system using stochastic simulation algorithms . Dan Gillespie 1977 [2] first implemented stochastic algorithms and methods to simulate the time evolution for chemically reacting systems.  One of the most important advantages of the stochastic framework over the deterministic methods is that stochastic fluctuations of the reactions can play an important role in single cell/system behavior, which is hidden when you use deterministic procedures.  We assembled WTM and KZM based on existing deterministic chemical-kinetic models [referencia a Endy y a Lingchong You] and resolved them using stochastic solvers from Simbiolgy ®. Both models integrate various events and chemical flows that determine the development of life cycle of phage T7 without and with kamikaze system.
In WTM we simulate different phenomena of the phage's development, from entry of phage’s DNA to final particle assembly and lysis, the importance of this model is that we reproduce the wild type life cycle of phage T7 and with ensembles of runs we build wild type BSD which it's supported by existing experimentally measured T7 Burst Sizes [3],[4],[5],[6],[7].
On the other hand, KZM tested the performance of our kamikaze. Apart from all wild-type processes, KZM incorporates the contribution of the ribosomes to the translation rates; this has been done to simulate the ribosome decay by the ribosome-inactivating toxin from the kamikaze construction. New BSDs will result from many runs of KZM, which will be used to study the behavior of our construction at the population scale.
See also KZM , BSD , Population Model , Cellular Automata .
References
[1] Drew Endy, Deyu Kong, John Yin. 1996. Intracellular Kinetics of a Growing Virus: A Genetically Structured Simulation for Bacteriophage T7.
[2] Daniel Gillespie. 1977. Exact Stochastic Simulation of Coupled Chemical Reactions.
[3] Thomas D. Brock. 1990. Emergence of Bacterial Genetics.
[4] Delbrück. 1945. Burst Size Distribution in the Growth of Bacterial Viruses(Bacteriophages).
[5] Richard H. Heineman and James J. Bull. 2007. Testing Optimality with experimental evolution: Lysis Time in time.
[6] De Paepe. 2006. Viruses Life History: Towards a Mechanistic Basis of a Trade-Off between Survival and Reproduction among Phage.
[7] Paul D. Sadowski. 1973. Suppression of a Mutation in Gene 3 of Bacteriophage T7 (T7 Endonuclease) by Mutations in Phage and Host Polynucleotide Ligase.
Retrieved from " http://2009.igem.org/Team:LCG-UNAM-Mexico:Molecular_model "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
