<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_LCG-UNAM-Mexico_Description"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:LCG-UNAM-Mexico/Description">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:LCG-UNAM-Mexico/Description&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/Description&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/Description&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:LCG-UNAM-Mexico/Description" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:LCG-UNAM-Mexico/Description</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV class="MenuBar" id="LCG"><UL><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico" onmouseover="window.status= 'Click here to return Home'; return true;">Home
                </A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Record">Record
                </A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description_null">Project</A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description_null"><SPAN><SPAN>Description</SPAN>&gt;&gt;</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description_null"><SPAN><SPAN>Design</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description_null"><SPAN><SPAN>Delivery</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description_null"><SPAN><SPAN>Defense</SPAN></SPAN></A></LI></UL><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Parts"><SPAN><SPAN>Parts</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Resources"><SPAN><SPAN>Resources</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Results"><SPAN><SPAN>Results</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Conclusions"><SPAN><SPAN>Conclusions</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Modelling">Modelling</A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Wet_Lab/Objectives">Wet Lab</A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Wet Lab/Objectives"><SPAN><SPAN>Objectives</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Wet Lab/Experiments"><SPAN><SPAN>Experiments</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team#The_team"><SPAN><SPAN>Team</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team#Acknowledgments"><SPAN><SPAN>Acknowledgments</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team#Collaborations"><SPAN><SPAN>Collaborations</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team#Sponsors"><SPAN><SPAN>Sponsors</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Help">?</A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Help/Site map"><SPAN><SPAN>Site map</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Help/Glossary"><SPAN><SPAN>Glossary</SPAN></SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Description"><B>Description</B></SPAN></H1><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Description"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Description</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Relevance_of_the_project"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Relevance of the project</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Logic_design_of_the_project_subsystems"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Logic design of the project subsystems</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Project_subsystems"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project subsystems</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#a.29_Delivery"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">a) Delivery</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Main_objective"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Main objective</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Background"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-4 tocsection-8"><A href="#P2-P4_bacteriophages"><SPAN class="tocnumber">2.1.2.1</SPAN><SPAN class="toctext">P2-P4 bacteriophages</SPAN></A></LI><LI class="toclevel-4 tocsection-9"><A href="#P4_genome_structure"><SPAN class="tocnumber">2.1.2.2</SPAN><SPAN class="toctext">P4 genome structure</SPAN></A></LI><LI class="toclevel-4 tocsection-10"><A href="#P4_sid_mutation"><SPAN class="tocnumber">2.1.2.3</SPAN><SPAN class="toctext">P4 sid mutation</SPAN></A></LI><LI class="toclevel-4 tocsection-11"><A href="#Cos_sites"><SPAN class="tocnumber">2.1.2.4</SPAN><SPAN class="toctext">Cos sites</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#P4sid1_standardized_production"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">P4sid1 standardized production</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Delivery_system:_benefits_and_perspectives"><SPAN class="tocnumber">2.1.4</SPAN><SPAN class="toctext">Delivery system: benefits and perspectives</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#b.29_Defense"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">b) Defense</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Main_objective_2"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Main objective</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Phage_detection"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Phage detection</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Translation_sabotage"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">Translation sabotage</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Gossip_and_Paranoia"><SPAN class="tocnumber">2.2.4</SPAN><SPAN class="toctext">Gossip and Paranoia</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Model_Validation"><SPAN class="tocnumber">2.2.5</SPAN><SPAN class="toctext">Model Validation</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Defense_system:_benefits_and_perspectives"><SPAN class="tocnumber">2.2.6</SPAN><SPAN class="toctext">Defense system: benefits and perspectives</SPAN></A></LI><LI class="toclevel-4 tocsection-21"><A href="#1.29Lytic_phage-induced_responses"><SPAN class="tocnumber">2.2.6.1</SPAN><SPAN class="toctext">1)Lytic phage-induced responses</SPAN></A></LI><LI class="toclevel-4 tocsection-22"><A href="#2.29Internal_Negative_Autoregulation"><SPAN class="tocnumber">2.2.6.2</SPAN><SPAN class="toctext">2)Internal Negative Autoregulation</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#References"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><TABLE><TBODY><TR valign="top"><TD width="900px" style="padding: 0 20px 0 0;"><H2><SPAN class="mw-headline" id="Relevance_of_the_project"><B>Relevance of the project</B></SPAN></H2><P>Bacteria play a fundamental role in human life and they are still the preferred models to study the molecular dynamics of organisms. One example are probiotics because of their vital importance in industry and food manufacturing. Infection by phages represents a relevant and expensive problem in these areas. For this reason we decided to construct a system to contend with bacteriophage infection at a population level.
</P><P>Given this global protection vision emerges the idea of dividing our project in two subsystems: <I><B><A href="/Team:LCG-UNAM-Mexico/Description#a.29_Delivery" title="Team:LCG-UNAM-Mexico/Description"> delivery</A></B></I> and <I><B><A href="/Team:LCG-UNAM-Mexico/Description#b.29_Defense" title="Team:LCG-UNAM-Mexico/Description"> defense</A></B></I>. Their coupled expression leads to a cascade dependent on the presence of an infectious phage. This property gives an extra versatility to the project because the defense subsystem turns on faster enough to hold back the infection and then lasts enough to give &quot;immunity&quot; to the population. Therefore, it is feasible to achieve a faster and wider protection response by amplifying the infection signal delivered by the phage in order to increase the number of &quot;immune&quot; bacteria at every lytic cycle.
</P><H2><SPAN class="mw-headline" id="Logic_design_of_the_project_subsystems"><B>Logic design of the project subsystems</B></SPAN></H2><P>The first level of the expression cascade consists on the delivery (<I><B><A href="/Team:LCG-UNAM-Mexico/Description#a.29_Delivery" title="Team:LCG-UNAM-Mexico/Description"> Delivery susbsystem</A></B></I>) of the protection system that will be immediately activated when a phage is detected (<I><B><A href="/Team:LCG-UNAM-Mexico/Description#Phage_detection" title="Team:LCG-UNAM-Mexico/Description"> Defense system: Phage detection and sabotage</A></B></I>). At the same time the formation of infectious viral particles is hold back, a diffusing signal warns the neighboring cells of the presence of infection (<I><B><A href="/Team:LCG-UNAM-Mexico/Description#Gossip_and_Paranoia" title="Team:LCG-UNAM-Mexico/Description"> Defense system: Gossip</A></B></I>). The reaction of alarmed cells consists on turning on a device, which allows a delay in case the defense system is turned on(<I><B><A href="/Team:LCG-UNAM-Mexico/Description#Gossip_and_Paranoia" title="Team:LCG-UNAM-Mexico/Description"> Defense system: Paranoia</A></B></I>).
</P><H1><SPAN class="mw-headline" id="Project_subsystems"><B>Project subsystems</B></SPAN></H1><H2><SPAN class="mw-headline" id="a.29_Delivery"><B>a) Delivery</B></SPAN></H2><H3><SPAN class="mw-headline" id="Main_objective"><B>Main objective</B></SPAN></H3><P><B>The main goal of the delivery device is the construction of a new iGEM vector capable of being used in a system for transduction of biobricks and synthetic devices in several bacterial hosts.</B></P><P>We propose a complete, standardized and controllable system for production of phage vectors for delivery of over 25 kb of synthetic constructs to a wide range of bacterial hosts. The relationships between bacteria and phages is quite rich and dynamic, so hacking this system for our control will be an interesting challenge! 
</P><P>The system starts with a modified bacteriophage P4 genome. This viral vector will be modified to be compatible with iGEM standards for biobrick assembly. The second part of the system involves <I><B><A href="/Team:LCG-UNAM-Mexico/Description#P4sid1_standardized_production" title="Team:LCG-UNAM-Mexico/Description"> production</A></B></I> of the modified P4 phages under our control. 
</P><P>As can be imagined, this promises to be a powerful tool in Synthetic Biology with a great potential for expansions and applications. We like the analogy of this modified P4 as an <B>USB *** memory device</B>, where you can store information or entire programs and just &quot;plug&quot; them into another machine that will handle such information.
</P><H3><SPAN class="mw-headline" id="Background"><B>Background</B></SPAN></H3><H4><SPAN class="mw-headline" id="P2-P4_bacteriophages">P2-P4 bacteriophages </SPAN></H4><P>Bacteriophage P2 and P4 are double stranded DNA enterobacteria viruses. Phage P4 is a satellite phage because it is 
dependent on the machinery of  P2. <A href="/Team:LCG-UNAM-Mexico/Description#References" title="Team:LCG-UNAM-Mexico/Description">(1,2,3)</A> Sometimes, it is called a &quot;parasite phage&quot;, since it takes over the elements of P2 and leaves its &quot;host phage&quot; practically neutralized. There are several interesting features of the P4 genome, including transactivation zones (the genes that respond to the presence of P2 along with P4 and vice versa) that 
function in domination of the late P2 genes. Important elements of this kind are gene P4 delta and P2 ogr<A href="/Team:LCG-UNAM-Mexico/Description#References" title="Team:LCG-UNAM-Mexico/Description">(12,15,16)</A>, which work synergistically together in activating P2 genes. Given these interesting properties, P4 has been exhaustively studied.
</P><H4><SPAN class="mw-headline" id="P4_genome_structure">P4 genome structure</SPAN></H4><P>We can divide P4 genome into two main regions: the essential and non-essential region. The essential region contains 
operons intended for replication and hijacking of P2, and the non-essential region contains accessory genes for special situations as lambda infections, as well as the integrase and attachment site<A href="/Team:LCG-UNAM-Mexico/Description#References" title="Team:LCG-UNAM-Mexico/Description">(3)</A>. Removing the latter two would result in a permanent plasmid-state P4 with a unique multicopy replication system.
</P><H4><SPAN class="mw-headline" id="P4_sid_mutation">P4 sid mutation</SPAN></H4><P>As P4 thoroughly depends on P2 for capsid, tail and lysis functions, the difference in size between both genomes (+- 33kb for P2 whereas +-11kb for P4) came to attention. P4 protein sid is able to scaffold a smaller capsid with the  same structural proteins as P2. A sid mutant was found that made P4 pack its genome inside bigger-sized capsids, which can hold up to 1, 2 or 3 copies of its genome<A href="/Team:LCG-UNAM-Mexico/Description#References" title="Team:LCG-UNAM-Mexico/Description">(6)</A>. The extra genome copies could be “something else”; this means P4 can transport over 20 kbs of extra foreign DNA attached to its genome. 
</P><H4><SPAN class="mw-headline" id="Cos_sites">Cos sites</SPAN></H4><P>Another important point is that the signal for encapsidation is located in the “cos” sites. It means that you only need this region (about 20 pb, but you increase the efficency of transduction with a region of  more than 100 pb) to encapsidate a double DNA strand disregarding the sequence in between the cos sites.
</P><H3><SPAN class="mw-headline" id="P4sid1_standardized_production"><B>P4sid1 standardized production</B></SPAN></H3><P>We thought of a way <B>to overproduce our viral particles</B> without being forced to infect with P2 or getting P2 particles as a byproduct. The solution planned was to construct an E. coli strain containing all the useful genes for P4 in P2 (capsid, tail and lysis operons). In addition to these genes, the<B> helper cell</B> would also contain the main P2  transactivators (cox and ogr)<A href="/Team:LCG-UNAM-Mexico/Description#References" title="Team:LCG-UNAM-Mexico/Description">(12,15,16)</A> under a lac operator. This way, after we transform the helper cell with our desired P4  plasmid, <B>we would decide when to promote stock production by lysis of the helper bacteria</B> by adding IPTG. Then we have our biobrick assembled inside ready-to-use phages that can deliver their genome to wildtype bacteria.
</P><P>We have also biobricked the cos sites of P4. This biobrick should be cloned in any vector with your construction. If you transform the P4 producing strain with this vector and then infect with P4, you will have as a result some 
P4 phages and some of your vectors with the P4 cos sites inside a capsid. It means you can encapsidate up to 33 kb 
with this system. Until your production is pure, you can add a marker in the plasmid (e.g. an antibiotic resistance or color) so after infection you select the colonies with the desired plasmid.
</P><H3><SPAN class="mw-headline" id="Delivery_system:_benefits_and_perspectives">Delivery system: benefits and perspectives</SPAN></H3><P>One of the main motivations for the construction of this delivery system using P4 as the vector is to achieve insertion of devices  into cells by transduction as an alternative way from traditional transformation. <B>This extends the panorama of synthetic biology to the whole P4 host range, which involves especies of genera such as Rhizobium, Klebsiella, and Serratia besides Enterobacteria like E. coli</B>. The delivery of parts into wildtype bacteria could be a pool for innovative applications and properties, such as the following:
</P><P>1)<A href="/Team:LCG-UNAM-Mexico/Description#b.29_Defense" title="Team:LCG-UNAM-Mexico/Description"><B> A defense system against another phages for E. Coli delivered by P4 phage. </B></A></P><P>The idea we love is <B>hacking one system that is harmful</B> (if you are a bacterium) <B>and using it for your own protection</B> against similar systems.
</P><P>2) <B>Refined phage therapy.</B></P><P>In addition to bacterial protection, we propose the use of this system to protect humans. This could be done using phage therapy to transduce DNA into pathogenic bacteria. It would be an advantage in cases where extra control is needed, as in degrading toxins before killing the pathogen and so avoiding further immune response. Another benefit would be response specificity in <B>hacking pathogen-specific regulators</B> while <B>the system is bypassed in non-hazardous strains</B>.
</P><P>As a first step in this area, we have adapted the <A href="/Team:LCG-UNAM-Mexico/Description#Translation_sabotage" title="Team:LCG-UNAM-Mexico/Description"> kamikaze system</A> to detect pathogenicity instead of phages. The target pathogen is EHEC and EPEC (Enterohemorragic and Enteropathogenic Escherichia coli). P4 will introduce a specific binding site for a pathogenicity-specific regulator LER, which in turn will activate the <A href="/Team:LCG-UNAM-Mexico/Description#Translation_sabotage" title="Team:LCG-UNAM-Mexico/Description"> kamikaze system</A> at the moment LER activates pathogenic <A href="http://en.wikipedia.org/wiki/Locus_of_Enterocyte_Effacement" class="external text" rel="nofollow">effacement</A>.
</P><P>3) <B>Phage mediated training </B></P><P>Another usage could be to &quot;train&quot; the bacterial population by P4 infection so that it is sensitive to a future 
stimulus, like indirect activation of medicine producing devices inside bacteria through phage contact. For instance, imagine you load a device of interest in the P4 genome and you transduce native E. coli with it. Now this E. coli will turn the device on in response to a new stimulus, say a body substance like a hormone.
</P><H2><SPAN class="mw-headline" id="b.29_Defense"><B>b) Defense</B></SPAN></H2><H3><SPAN class="mw-headline" id="Main_objective_2"><B>Main objective</B></SPAN></H3><P><B>The main goal of the defense device is to significantly reduce the burst size in order to allow bacteria to survive a phage infection process.</B> To achieve this, we designed a <A href="/Team:LCG-UNAM-Mexico/Description#Translation_sabotage" title="Team:LCG-UNAM-Mexico/Description"> kamikaze system</A> that will prevent the spread of phage infections. When phages T7 and T3 infect protected E. coli, these will start producing toxins that deactivate ribosomes. The result: no translation machinery, no phages produced and heroic bacterial suicide. 
Initially, a viral infection is a process that takes place inside an individual but the real consequences of the infection become important at the population scale. In order to efficiently and accurately simulate the behaviour of the defence device we need to implement two different kinds of approaches: an individual-based simulation and a population simulation, and then integrate them in a Multi-Scale Model.</P><P>In order to simulate the spatial dynamics of the defence device we designed and implemented a <A href="/Team:LCG-UNAM-Mexico:CA" title="Team:LCG-UNAM-Mexico:CA">Celluar Automaton (CA).</A> Using the CA we can approach several problems at the same time:  E. Coli movement and duplication, AHL and phage diffusion and the infection process.</P><P>The detailed parts of the defense subsystem are the following:
</P><H3><SPAN class="mw-headline" id="Phage_detection"><B>Phage detection</B></SPAN></H3><P>When phage T7 or T3 transduce their DNA into the host cell, the phage's polymerase will be able to bind the multi-promoter of the system, which will activate two subsequent actions: production of toxins to inhibit further phage propagation, and a neighborhood alarm. The first thing translated is GFP.
</P><P>The part contents, in order of appearance, are as follows:
</P><H3><SPAN class="mw-headline" id="Translation_sabotage"><B>Translation sabotage</B></SPAN></H3><P>One of the elements transcribed by T7 RNA polymerase at early stages of T7 cycle in our transformed bacteria is the <A href="/Team:LCG-UNAM-Mexico/Description#Translation_process_sabotage" title="Team:LCG-UNAM-Mexico/Description"> kamikaze system</A> which consists of a polycistronic mRNA that codes, among other proteins, the rRNAse domain of colicin E3. This toxin cleaves 16s rRNAs in active ribosomes from E. Coli, which causes inactivation of the ribosome and a subsequent decay in the overall bacterial translation. This response of our system affect T7 cycle by reducing the number of bacteriophage proteins and then lowering the number of T7 phages at the end of the cycle.
</P><H3><SPAN class="mw-headline" id="Gossip_and_Paranoia"><B>Gossip and Paranoia</B></SPAN></H3><P>luxI is another product from the kamikaze system. Infected cells produce it in order to warn surrounding cells of phages' presence through AHL.
When a neighboring cell has been reached by AHL, it turns on an antisense RNA against a T7 messenger to interrupt its life cycle if it becomes infected. This delay in the life cycle of T7 gives more time to colicins to act upon the translation machinery reducing active ribosomes to zero before the assembly of any T7 particle.
</P><H3><SPAN class="mw-headline" id="Model_Validation">Model Validation</SPAN></H3><P>We expect the Burst-Size to be significantly reduced. An optimal result would be a Burst-Size of 0. Implementing a <A href="/Team:LCG-UNAM-Mexico:BSD" title="Team:LCG-UNAM-Mexico:BSD">sensitivity analysis</A> we found that the <A href="/Team:LCG-UNAM-Mexico:BSD" title="Team:LCG-UNAM-Mexico:BSD">burst size distribution</A> is dependent on the rate of ribosome inactivation by colicin E3. The <A href="/Team:LCG-UNAM-Mexico:BSD" title="Team:LCG-UNAM-Mexico:BSD">wild type Burst Size Distribution</A> has mean 176 and standard deviation 102. This results are consistent with <A href="/Team:LCG-UNAM-Mexico:BSD" title="Team:LCG-UNAM-Mexico:BSD">existing experimental data</A>, the reported values for the burst size present a wide variation.
The <A href="/Team:LCG-UNAM-Mexico:CA" title="Team:LCG-UNAM-Mexico:CA">Celluar Automaton</A> and the system of <A href="/Team:LCG-UNAM-Mexico:odes" title="Team:LCG-UNAM-Mexico:odes">Delay Differential Equations</A> generate growth curves that can be compared with those obtained experimentally.
Results generated by the <A href="/Team:LCG-UNAM-Mexico:CA" title="Team:LCG-UNAM-Mexico:CA"> Cellular Automaton</A> are in good agreement with those obtained experimentally.
Our <A href="/Team:LCG-UNAM-Mexico:Molecular_model" title="Team:LCG-UNAM-Mexico:Molecular model">Molecular Model</A> has proved to be a reliable tool for sampling molecular distributions in order to make sensitivity analysis and to assemble more complex models as we did with our Cellular Automaton.
See <A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Modelling" class="external text" rel="nofollow">Modelling Section</A> for detailed information.
</P><H3><SPAN class="mw-headline" id="Defense_system:_benefits_and_perspectives"><B>Defense system: benefits and perspectives</B></SPAN></H3><P>One of the most important mechanisms concerning the <A href="/Team:LCG-UNAM-Mexico/Description#Defense" title="Team:LCG-UNAM-Mexico/Description">defense subsystem</A> is the use of toxins as the main element in the disruption of phage’s assembly and scattering. Even though the contention of the infection implies that some bacteria will die, the use of a RNAse and a DNAse induces a delay of the phages production by beating host machinery. This in turn, avoids the possibility of the phage to gain resistance against toxins.
</P><H4><SPAN class="mw-headline" id="1.29Lytic_phage-induced_responses">1)Lytic phage-induced responses</SPAN></H4><P>One interesting internal property of this system that could be exploited for novel purposes is the use of an entire wild-type biological entity (phages) as the activator of an internal system. Phage infection neutralization makes the population survive to the initial signal, while phage-mediated signal triggering makes the stimulus arrive to non-infected cells. This makes phages act merely as &quot;external activators&quot; analogous to quorum sensing molecules.
</P><H4><SPAN class="mw-headline" id="2.29Internal_Negative_Autoregulation">2)Internal Negative Autoregulation</SPAN></H4><P>One of the ideal situations in synthetic biology is the &quot;friendly get-along&quot; of humans and biological machines for the benefit of the former, as projected with medicine production from genetic circuits. We propose that activation of such circuits could rely on the usage of bacteriophages and their population equilibrium with bacteria. While a bacterial population could tolerate a phage infection with our alarm system and hence initiate extra responses (like medicine production), an overdose of the initial phage activation signal, instead of killing individuals by icreasing the production of medicine, could cause the extintion of the biological machine inside the body. This would bring the individual to an initial pre-medication state.
</P><H2><SPAN class="mw-headline" id="References"><B>References</B></SPAN></H2><P>1.Propagation of satellite phage P4 as a plasmid. Goldstein R, Sedivy J, Ljungquist E. Proc Natl Acad Sci U S A. 1982 Jan;79(2):515-9
</P><P>2.Nonessential region of bacteriophage P4: DNA sequence, transcription, gene products, and functions. Ghisotti D, Finkel S, Halling C, Deh˜ G, Sironi G, Calendar R. J Virol. 1990 Jan;64(1):24-36.
</P><P>3.Mechanisms of Genome Propagation and Helper Exploitation by Satellite Phage P4. Lindqvist BH, Deh˜ G, Calendar R. Microbiol Rev. 1993 Sep;57(3):683-702.
</P><P>4.Phasmid P4: manipulation of plasmid copy number and induction from the integrated state. Lagos R, Goldstein R. J Bacteriol. 1984 Apr;158(1):208-15.
</P><P>5.Integration of satellite bacteriophage P4 in Escherichia coli. DNA sequences of the phage and host regions involved in site-specific recombination. Pierson LS 3rd, Kahn ML. J Mol Biol. 1987 Aug 5;196(3):487-96.
</P><P>6.Determination of capsid size by satellite bacteriophage P4.  Shore D, Deh˜ G, Tsipis J, Goldstein R. Proc Natl Acad Sci U S A. 1978 Jan;75(1):400-4.
</P><P>7.Recombinant P4 Bacteriophages Propagate as Viable Lytic Phages or as Autonomous Plasmids in Klebsiella pneumoniae. David W. Ow and Frederick M. Ausubel. Molec. gen. Genet. 180, 165 175 (1980)
</P><P>8.Engineered bacteriophage-defence systems in bioprocessing. Sturino JM, Klaenhammer TR. Nat Rev Microbiol. 2006 May;4(5):395-404.
</P><P>9.Interactions between a satellite bacteriophage and its helper. Barrett KJ, Marsh ML, Calendar R. J Mol Biol. 1976 Sep 25;106(3):683-707
</P><P>10.Engineering BioBrick vectors from BioBrick parts. Shetty RP, Endy D, Knight TF Jr. J Biol Eng. 2008 Apr 14;2:5.
</P><P>11.The Locus of Enterocyte Effacement (LEE)-Encoded Regulator Controls Expression of Both LEE- nd Non-LEE-Encoded Virulence Factors in Enteropathogenic and Enterohemorrhagic Escherichia coli. Eliott J. et al.(2000).  Infection and immnity, Nov. 2000, p 6115-6126
</P><P>12.Activation of prophage P4 by the P2 Cox protein and the sites of action of the Cox protein on the two phage genomes. PNAS. Vol 86:pp. 3973-3977 Shamol Saha et al.(1989)
</P><P>13.Eliott J. et al.(2000). The Locus of Enterocyte Effacement (LEE)-Encoded RegulatorControls Expression of Both LEE- and Non-LEE-Encoded
</P><P>14.Virulence Factors in Enteropathogenic and Enterohemorrhagic Escherichia coli. Infection and immnity, Nov. 2000, p 6115-6126
</P><P>15.Bacteriophage P2 ogr and P4 delta genes act independently and are essential for P4 multiplication. Journal of Bacteriology. Halling, C. (1990)  172(7):3549-3558
</P><P>16.Regulation of bacteriophage P2 late-gene expression: The ogr gene. Christie, G. (1986)  PNAS Vol.83 3238-3242
</P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:LCG-UNAM-Mexico/Description">http://2009.igem.org/Team:LCG-UNAM-Mexico/Description</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:LCG-UNAM-Mexico/Description" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:LCG-UNAM-Mexico/Description" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/Description&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico/Description&amp;oldid=168937" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>