"
Page
Discussion
View source
History
teams
Log in
 
Team:LCG-UNAM-Mexico:BSD
From 2009.igem.org
Home
Record
Project
Description>>
Design
Delivery
Defense
Parts
Resources
Safety
Results
Conclusions
Modelling
Wet Lab
Objectives
Team
Team
Acknowledgments
Collaborations
Sponsors
Photos!!!
Site map
The Burst-Size Distribution
Content
The Burst Size Distribution
Simulated BSD
BSD using the Kamikaze System
References
The Burst Size Distribution
In 1939 Ellis and Delbrück obtained values which indicated an exceedingly wide variation of the burst sizes for different bacteriophages. In 1945 Delbrück published the first Burst-Size distribution using an improved technique for bacteriophage T1.
The reported values for the burst size of T7 are in the range 100-300, this values are used in many population models for bacteriophage infection [][][].
With the Stochastic Molecular simulations of the intracellular dynamics  we can sample values of the Burst Size Distribution. Creating a Burst-Size distribution is one of the most important things of our work since it will be the link between the intracellular scale and the population scale simulations.  The BSD is by no means the only distributions generated by our intracellular simulations , distributions for each species in the model are generated indeed.
T7 Reported Burst Size
WT Measured T7 BS
Reference
266±16
Heineman 2007
130±50
Stanley 1989
260
De Paepe 2006
214
Sadowski 1973
300
Brock 1990}
Table1.Experimentally measured T7 Burst Size
Simulated BSD
Burst Size Distribution obtained from the simulation Results of the Molecular Simulations.
Without kamikaze system our simulated BSD has mean 176 and standard deviation 102. This distribution was created running 1000 simulations of the intracellular model.
We see that experimentally measured values fall within 1 standard deviation of the distribution's mean. The large variance seen by Delbrück and the dispersion in experimental values is congruent with our results (table 1).
It's important to remember that the observed dispersion in our simulations its due only to stochastic fluctuations in ocurrence of chemical reactions.
So using our model we can sample distributions for any of the biochemical species in the system and use those values to assemble more complex stochastic models as we did with the Cellular Automata .
Sensitivity Analysis for T7 mRNA half-life
Sensitivity analysis for the T7 mRNA half-life. For each value of the parameter we performed 1000 simulations of the molecular model and the mean was computed. Standard deviation bars are shown.  .
Previous intracellar simulations for bacteriophage devolpment didn't take into account the half-life of the phage mRNA.
To answer the question of whether this parameter affects the Burst Size or not, we implemented a sensitivity analysis. The average half-life reported for early T7 messengers is ~6.5 min. (Yamada, 1975). We performed 1000 simulations of the molecular model for a wide range of half-life values.
The image on the left shows the results of the simulations, we observe the way in which the Burst Size is affected by the mRNA half-life time.
We observe a wide standard deviation in the distributions (vertical bars), this is consistent with existing experimental data.
BSD using the Kamikaze System
Sensitivity Analysis for the Ribosome Inactivarion Rate. We used simulations that include the action of toxine E3. Burst Size of 0 was obtained for rate in the range (10e-1,10e-4).
Results by Yin,2002 suggested an efficient way to reduce phage cycle efficieny: "...phage growth was found to be most sensitive to the host translation machinery, specifically, the level and elongation rate of the ribosomes."
We added to our Wild type Molecular Model the action of colicin E3 (rRNAsa): cleavage of rRNA of 16s subunit of the ribosome. We couldn't find the rate of this reaction in the literature so we decided to perform a sensitivity analysis: change the reaction rate over a wide range of values and see wether the burst size is reduced significantly or not.
The image on the left shows the results of the sensitivity analysis: Burst Size was reduced to 0 when the ribosome inactivation rate takes values greater or equal than 10e-4. When the rate inactivation is 10e-5 the mean burst size is 5.8. Using the BSD distribution with mean=5.8 in the Cellular Automata simulations we observed that the bacteria population, after a brave struggle with phages, sadly dies. This result was expected since the latency period is smaller than the duplication time and each infected bacterio will produce an average of 6 phages!
Our system work as expected for burst size values less or equal to 1. Sensitivity analysis shows that our system works for a wide range of values for the ribosome inactivation rate but even a small burst size value like 6 will eventually kill the whole population.
References
Thomas D. Brock. Emergence of Bacterial Genetics, 1990.
Delbrück. Burst Size Distribution in the Growth of Bacterial Viruses(Bacteriophages). 1945
Richard H. Heineman and James J. Bull . Testing Optimality with experimental evolution: Lysis Time in time . 2007
De Paepe. Viruses Life History: Towards a Mechanistic Basis of a Trade-Off between Survival and Reproduction among Phage. 2006.
Paul D. Sadowski. Suppression of a Mutation in Gene 3  of Bacteriophage T7 (T7 Endonuclease) by Mutations in Phage and Host Polynucleotide Ligase. 1973
Yamada. Chemical Stability of Bacteriophage T7 Early mRNA. 1975
Retrieved from " http://2009.igem.org/Team:LCG-UNAM-Mexico:BSD "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
