"
Page
Discussion
View source
History
teams
Log in
 
Team:LCG-UNAM-Mexico:Journals:Nando's
From 2009.igem.org
Home
Record
Project
Description>>
Design
Delivery
Defense
Parts
Resources
Safety
Results
Conclusions
Modelling
Wet Lab
Objectives
Team
Team
Acknowledgments
Collaborations
Sponsors
Photos!!!
Site map
Nando's lab journal
My main goal is described in these points.
1) I'm in charge of the assembly of the P4 vector. 2) I need to obtain the essential region of P4 sid 1 by pcr 3) I need to check for the characterization of p4 as an iGEM plasmid. 4) I need to ligate the P4sid1 plasmid to the BiteBack system and transform an E.coli  C1-a strain with it. 5) then we can start checking for transduction of our system.
During the months of June and July, I was in charge of obtaining the phages to work with, T3, T7, P2 and P4. In order to accomplish this, I emailed the "Phage people". They are:
Dr. Richard Calendar .- Professor of Molecular Biology at UCB. He has gently handed phage p4 vir1, P2 vir1 and some strains to work with them: C-520, C-2423, C-1895 and C-331. Dr. Giuseppe (Joe) Bertani from CalTech. He kindly sent C1a , C-117, C1906. Dr. Mario Soberón and Sabino Pacheco from Instituto de Biotecnología UNAM (Institute for Biotechnology of the National Autonomous University of Mexico) have kindly given us a phage T7 lysate. Dr. Ian Molineux from the University of Texas gently sent us phage T3. P4 sid1.- obtained from ATCC
August 12, 09.
A very important piece of data is the phage titer you use, since you cannot know what properties the infection is developing if you don't know how many phages you threw into battle. So T3 and T7 are diluted in LB medium, which is a good buffer for them.
Another important point is the making of an agar or agarose mix called Top agarose. This Top agarose tops the normal agar plate (hence its name) with an agar in a lower concentration that promotes proper plaque expansion.
I started organizing the phage titration. You start from a phage lysate as a 1:1 sample, make a dilution of 1:9 of that one for a 1/10, and repeat for the new dilutions until you reach the desired dilution: from 10^-7 to 10^-13.  This way you can correlate the number of plaques formed with the dilution of  the stock. For example, if you only get one plaque in a 10^-10 dilution, you can state that your lysate has 10^10 plaque forming units (pfu) in the volume unit you started with.
Today,  overnight standing cultures of C1a , BL-21 were left to grow at 37 degrees in order to assay for the viability of T3 tomorrow. One of [[Team:LCG-UNAM-Mexico/Resources/Strains| C117 ]] was placed for storing the supernatant next day.
August 13, 09
We also need to verify that T3 and T7 infect C1-a without any problem. Since the most suspicious phage in terms of viability is T3 because of all the time it waited in the border, i added .15 ul of the overnight cultures to M9LB  for it to reach the log phase. when they reached log phase (3 hours), I added  50 ul of phage to each flask. Then I proceeded to place the cultures in Miguel's Incubators (and one in  someone else's incubator).
After some time, the cultures were crystal clear, what qualitatively demonstrated  the lysis of the culture by phage T3.
The [[Team:LCG-UNAM-Mexico/Resources/Strains| C117 ]] overnight culture was centrifuged at 13000 rpm to pellet the cells and store the supernatant.
Today, an overnight of C 2423 and C1a were put to grow.
Plan for tomorrow.- prepare the plaque assay for P4 sid1.
August 14, 2009
We need to verify the viability of P4 sid1, so many things have to be assessed.
As the legend tells, p4sid1 must:
a) not make plaques in C1-a or P4-containing bacterial lawns b)not make plaques qhere there's no bacteria b)not make plaques where there is no phage present c)make plaques in p2-containing bacterial lawns
When going to check the C1-a standing culture, it happens it didn't grow as an overnight culture is expected to, so all the experiments will be put off until later in the day. This means that I won't be able to see the plaques myself until monday since I'll party my birthday away in Mexico City.
August 17, 2009
Looking at the plaque assay results.- see August 14
1.NCLB- bacteria grow.- probably due to mistake in procedure. 2.NC C1a .- Negative control for C1a (contamination control).- grows well without plaques 3. C1a P4sid1 1.- some small plaques really far away from each other. probably because some P2  and coinfection in the ATCC lysate 4.- C1a P4sid1 2- some small plaques really far away from each other. probably because some P2  and coinfection in the ATCC lysate (a really curious effect) 5.- C2423 P4sid1 3- many well-defined plaques (internal control discards contamination) 6.- C2423 P4sid1 4- many well-defined plaques 7.- C2423 P4sid1 5- many well-defined plaques 8.- C2423 CN.- no plaques :D
Conclusion of the first part of the experiment.- P4 sid1 does work in the formation of plaques. P2 is actually produced in lysis, but in a really low number.
Today's plan.-  the next part  is to isolate colonies fro the non-plaque regions of 3 and 4 that could contain a plasmid-state or lysogenic P4.
August 18, 2009
Yesterday's experiments summary.-
The plates were arranged this way.
plate 1
colonies
1, 2, 3, 4, 5, 6.- different culture zones in the C1a plate which  are not plaques . theoretically, may be one of these colonies could have been infected and transduced by P4, so I must assay for the presence of plaques. However, it comes into mind how i will assess the presence ot P4 inside of these bacteria by only means of plaque assays, given that P2 is a complete phage and could perform lysis on its own. This would mean to make a comparison between lysis of P2 alone and along with P4, and it's probably non-trivial to distinguish P4 plaques from P2 plaques. given all this, We should investigate about a better way to test with an internal control. Of course, the biobrick with a color signal could be excelent as an indicator, but we cannot wait for that. x,y,z.- zones  of the bacterial plate generated that include plaques from P2 presence.
the presence of plaques reveals an interesting phenomenon; the P4 sid1 purchased stock has some P4 contamnation because it's a lysate. Dr. Bertani argues that it is of really low significance since  one P2 is produced by 100 P4.  The fact that you can see plaques involves that the P2 that come out are really efficient infectors of the neighbor cells, which happen to be in a P4-filled environment. If this werent the real way, the P2 phage wold be lost in the medium and, ocasionally, a cell would die, but that wouldn't necessarily produce a plaque in the agar.
E,F,G.- C2423 zones from August 14, 09 which were picked very carefully trying not to obtain any plaques. These work as pneative controls.
In other respects, we tried to infect  bacteria in  M9LB without calcium to see if that worked. The incubations were made with 100 ul of phage. Lysis didn't start, so we cannot combine the protocols for both  phages and calcium is really important for P4 adsorption. I need to find out how to carry out the protocol with the materials we have here.
August 19, 2009.
I tried generating lysis in liquid medium by addition of the P2 vir 1 lysate to a series of overnight cultures which were added 70 ul P2 and 40 ul CaCl2. the cultures were:
bacteria                                   expected No bacteria                                clear dish C1a being tested for P4 sid 1 x 2    plaques if infected with P4, no plaques if not [[Team:LCG-UNAM-Mexico/Resources/Strains| C117 ]]                                         small plaques C2423 large plaques C1906                                       no plaques
August 21, 2009.
We have officially confirmed yeast contamination  in the lab. Many petri plates have acquired a shiny white appearance. This is a big problem, since maybe many of the results expected before now are modified by the contamination. Undoubtedly, this will lag the development of the project. The next best option will be to produce streptomycin resistant strains of every strain present until this moment.  The only thing left to be done to leave this round of experiments, since the negative results from the last experiment may be due to the lack of virulence of phage P2, is to probe the absence of P4 through colony PCR product analysis. The PCR was performed.
August 24, 2009.
The PCR results almost prove the absence of P4 thanks to a lack of PCR  product amplification The only thing needed for a complete confirmation. Now, let's move on. We have asked Juan Pablo the technician to prepare streptomycin for growth of resistants. By the way, we asked him to prepare top agarose to go on with the phage plaque assays among other things, like M9LB medium and a 1M CaCl2 stock. Meanwhile, we left the bacteria growing in order to be ready for when the streptomycin will be used. The cultures left for tonight resistance round are as follows:
1.C1895 2 C2423 3. C331 4.C1906 5.C520 6.[[Team:LCG-UNAM-Mexico/Resources/Strains| C117 ]]a 7.[[Team:LCG-UNAM-Mexico/Resources/Strains| C117 ]]b 8. C1a
Almost at the end of the day, we noticed that Juan Pablo was not told by anyone to prepare the streptomycin solution, but fortunately, we found him and he gave us some sp 100 solution, assuring that it was streptomycin. So we proceeded to prepare plates for tomorrow's assay.
August 25, 2009.
Early in the morning, I centrifugated and resuspended cells in 100 ul to plate and scan for resistants. At night, some resistants finally appeared, but we have to further prove that they are what we want them to be. We will discuss this with Miguel.
August 26, 2009.
C 1895.- small colony spots... C2423 .- 2 or 3 colonies. let's check for morphology C331 nothing yet. let's wait a bit more. C1906.- really lots of colonies. maybe the antibiotic went wrong. C520.- too many colonies. check for resistance again. [[Team:LCG-UNAM-Mexico/Resources/Strains| C117b .- let's wait for them to grow a bit more. C1a .- nothing clear. [[Team:LCG-UNAM-Mexico/Resources/Strains| C117a .- practically nothing CN.- nothing
I planned to repeat the resistant-growth assay, but the cultures I left inan overday showed contamination in the negative control.
Late at night, I offered Paz my help in purifying PCR products. I commited a really stupid mistake when using the elution buffer before the wash buffer and treating the flowthrough as waste. I was completely ashamed of my error!!! Luckily, the PCR could be done again.
Laura had left some cultures she asked me to plate.
August 27, 2009
Nothing resistant has grown yet. I'll wait a bit more. Laura's cultures show a bit different behaviour from yesterday's. C520 returned to normal (that means, yesterday's was contaminated). none of the others has obtained resistance yet.
Miguel finally told us what tests we can make for assessing a real resistance acquisition:
1.- to draw a resistant colony with a copper stick in a petri dish with the wildtype bacteria at its side. 2. assess for correlation of cfu with serial dilutions. (to assess that bacteria don't become spontaneously resistant) 3. Do a contrast of the colony against another antibiotic. In this case, since we failed in believing Juan Pablo's lie (sp means spectinomycin, not streptomycin) we will use streptomycin as our test antibiotic.
If everything is fine, the next step is to make a glycerol stock of them. I'll find out how to do that.
Both streptomycin and spectinomycin are translation blockers as they attach to essential sites of the ribosome.
August 28, 09
Yeserday I plated the hypothetical P4-infected C1a for resistants. the results are the following:
Negative control.- not valid since has colonies and yeast. 1. colonies of distinct sizes. 2. the same as 1. 3. spotted lawn... 4.everything alright and small colonies. 5. different sized colonies.
All this, except box 4, state that contamination is inminent and it is not good to try a PCR with this.
As some of the strains have not generated resistants, yesterday Laura put overnight cultures for C1895, C2423 and C520. I have plated them.
August 31, 2009
I learned how to  plate for colonies on a single petri plate. The idea is to draw the colony, burn the copper stick, take a bit from the last edge touched by the copper stick before and draw another colony again in a new space. By repeating the process, you will start isolating clones in the same petri plate. The goal of this is to check individual colony morpholoogy if it is double or single; in the former case, contamination will be proved; in the latter one, we asses individual colony morphology.
September 1, 2009.- tracing yeast
The same-plate assay for resistants and wild type worked. The resistant colonies were the only ones to grow.
colony morphologies.-
C1906 sp+.- normal, somehow brown. doesn't become yellow even after a lot of time.
[[Team:LCG-UNAM-Mexico/Resources/Strains| C117 ]] a1 and a2.- it's becoming fatter and fatter each time and happens to overlap with another colony.
before resistance, C1906 was yellowish with a bright halo around the colony. internal appearance looks good.
the plate with C2423 didn't grow anything before and  now we have some orange round colonies in the agar. Undoubtedly, this means contamination.
This day I also ran a gel for the PCRs that Laura performed for cox, ogr and P4 from the resistant colonies. Surprisingly, nothing came out and the gel was possitively controlled by the ladder, so we cannot discard mistakes in the performance of the PCR.
While having my first steps in loading a gel electrophoresis, I noted some tips:
It may be helpful...
not to unload the second pipette stop, so you don't push air and your sample doesn't drift away. not to bury the pipette tip inside the gel!!! this is partially or completely prone to destruction of your attempt to load. not to wash the  loading plastic (array for samples) with water before use! If you really need to, use alcohol and  dry it to perfection.
I will try to check colony 6 for infection with p4 sid1. First, I  must prepare the cell extract.
September 2, 2009
Objective of yesterday's PCRs.- 1. verify the correct functioning of the primers 2. Presence of infection of P4 sid1 en strain 6 of phage infection.
All the lanes in the gel are dim and noisy except for the negative control. many doubts are left by now:
Are the primers right? Did the PCR ocurr correctly? no positive control present. Why does colony 2 have a brighter DNA shearing? hypotheses.-
The genomic DNA was sheared Primer annealing was non-specific the polymerase wasn't processive enough to finish
Plans for the rest of the week.-
obtain  sp resistant C1a grow log phase C1a and add P4sid1. then plate for individual colonies.
we have to manage to prove that our primers work.
The first attempt will be to extract genomic DNA from C1906, so I'll grow an overnight culture.
The scan for resistance continues.
We have to design a method for detection the differences between P4, P4 vir1 and P4 sid1 genomes. Let's look for restriction enzymes.
September 3, 2009
Continuing colony transfer and scavenge for resistants.
I will  put a PCR reaction today with the aim of amplifying the needed P4 3kb region of C1906 and thus verify the correct functionality of the primers. Elongation time is 3 min.
2 tubes for C1906 DNA C1a .- Negative control 1 No DNA Straight PCR NO TAQ! must wait for tomorrow to carry out the PCR.
September 4, 2009
Todays goals: Set yesterday's PCR Plate C1a (this is the most needed strain for infecting with P4)
Laura gave me the job of taking the photographs for her phage plates. unfortunately, I didn't receive full instructions about the management of the plates and didn't think that the plaques would extend much at room temperature. well, they did. She got a bit angry because we lost a pair of plates that were krystal clear.
The PCR didn't work. Possible reasons:
Lots of DNA chelated magnesium ions. No primer spots. What could be the reason for this? I need to run a gel to observe the quality of my template.
September 5, 2009
continuing scavenge for resistants.
I will grow an overnight of C1906 and C2423 to prove presence of P4 and P2.
if we can observe a template DNA,PCR will be performed.
September 6, 2009
Let's see if we've got template for kit-purified C1906 and the strains that were extracted DNA manually. They are.- ones are C1906 sp resistant colonies. twos  are C2423 sp resistant colonies.
11 12 13 14 15 21 22
At the end of the day, we can see that there isn't a successful genomic C2423 DNA extraction, but C1906 happens to show (dirty) DNA template, so I will run a gel for this
September 7 2009
I plan to run a 3% agarose gel for verification of the presence of primers. Nothing can be seen. not even the ladder. Possible causes.-  the agarose is so thick it doesn't allow the DNA to migrate. the primers drifted away from the gel. less probable. ¿Was the wash appropriate?
Next step is to repeat the PCR of C1906. Corrections will be.- dilute 1 part C1906 DNA in 10 water. I'll check the annealing temperature.
this time, were short of DNTPs and there is a stock that was made to test. Sadly, I was a victim of [[Team:LCG-UNAM-Mexico/Team/ Paz and he gave me useless DNTPs.
September 8, 2009
Plans for the day.- wait for DNTPs to perform PCR. Extract DNA from C1895 and C331 I'm planning the obtention of lysate from P4 sid1 this time.- Obtain an aliquot of P4sid 1 by diluting it in LB supplemented with calcium. Put an overnight culture of C1895 and add phage to it. follow growth of p4 protocol .
The PCR was interrupted by careless connection of the PCR cycler to the direct current. this was around cycle 24.
September 9, 09
The C 1895 used was a sp+ one which was really slow growing. Two lysis experiments were made.- one flask was added 80 ul diluted lysate from ATCC and will be called Stock (S) the  other one was carefully picked with 2 plaques from the C2423 lysis  produced in a plaque assay on August 19. Cynthia (nené) helped me understand how to use the spectrophotometer. I turned it on for my measures. I found the spec turned off twice while I was  taking my measures.
Critical observations.-
I'll use smaller flasks since the fernbach doesn't fit inside our incubator. The bacteria grew really slow. That means, about .01 A600 in 3 hours. My best bet will be to leave the culture overnight and see what happened in the morning. I consider the experiment failed ( check plaque assay on Sep. 19 ), so I'll have to try with a wt bacterium.
Today I also ran a pcr with various controls and primers.
only with int P4 primers.- CN
C1906 colony 2- expected: 3 kb band
C1906 colony 3-expected: 3 kb band
C1906 non-resistant, kit purified from stock-expected:3 kb band
--- P4 int and P2 ogr primers.-
C2423 colony 2-expected:300 bp band C2423 colony 3-expected:300 bp band C331 -expected: 3kb and 300 bp band --- P2 ogr primers only [[Team:LCG-UNAM-Mexico/Resources/Strains| C117 - expected:300 bp band C1906 non resistant-expected: nothing C2423 colony 2- expected-nothing
results.- a strange +- 6 b band appeared in C331 which twists my mind out. It is possible that I am retrieving  a product specific for the primer combination used, so the PCR isn't really valid. Hypotheses are that this newly combined pair of primers
September 10, 2009
I notice a strange phenomenon in yesterday's flasks: they have some things floating. I checked the absorbance of the culture and it was just opposite in quantity from last night's measures; in other words,  one was higher than the other and now roles have changed . I added EGTA and MgCl2. An hour later, I rechecked the absorbance and it had diminished by half in both cultures. just when lysis stopped, I added more MgCl2 to stabilize the phage and stored the hypothetical lysates in the fridge.
September 15, 2009
Today I'll repeat the P4 producng assay. Problems to solve:
Using our spectrophotometer Making space for the fernbach flask getting sterilized centrifuge tubes
Issue.- the incubator is at 250 rpm and I need it at 60- 80
Today I'll learn how to prepare LB medium and will prepare 1 liter.
September 16 09
P4 production assay (reloaded)
Note.- the incubator was modified in order that the fernbach flask fitted.
The incubated overnight (with 100 ul of phage concentrate) was added to the flask (400 ml supplemented LB). According to Dr. Calendar, lysis should start when  absorbance gets between .7 and 1.5 on growing bacteria.
I waited very long until the absorbance reached 1.17 and didn't observe any lysis.
What could be happening is encountered next.
check for problems with the rehydrated phage. control for culture aeration as a cause for the fail control for whole protocol with p4vir1gas exchange
Next Idea is to try again, this time adjusting the cotton lid to fit inside the neck of the tube, so there can be air exchange with the environment.
Plans for tomorrow.- Assay for presence of P$ in test growth. I will need more top agarose, so I'll prepare some tomorrow. What I'll do.- Plate C2423 on top agarose:
no phage 100 ul S supernatant 100 ul P supernatant 100 ul phage stock one incubated with a plaque.
State of the art of work on P4
Plan 1.- create a P4 sid1 infected strain Plan 2.- detect a P4 infected strain via PCR We didn't obtain any P4 PCR products from C1906 genome,  but we've got (not necessarily expected) products from the resistant reseeding. We hypothesize that there is no target DNA, so direct PCR from  a reproduced P4 stock is a good proposal. The primers don't seem to be wrongly constructed There have been some problems with generating the stock Once we have P4 stock ready: Digest with proteinase K Perform PCR for P4 parts: int and main (sid?) perform C1a p4 sid1 infection assay and verify with PCR. Tomorrow.- Infect [[Team:LCG-UNAM-Mexico/Resources/Strains| C2423 wt/sp+
September 18, 2009
I will attempt to produce plaques with P4 from different sources in the same host strain.- C2423 . S- stock lysate from september 9 involving slow-growing  c1895 mutant. P.- plaque lysate from september 9 wt- wildtype colony... +s.- plus ATCC stock +p.- plus a picked plaque This is to assess the best technique to produce P4 stock.
September 19, 2009
Fortunately, plaques were produced. herewith the results.-
s.- individual colonies are seen, probably phage resistants. there is simething in the culture that prevents bacteria to form a lawn. p.- looks exactly like cell control.- lysis from the picked plaque culture isn't observed. It could have been an aeration effect. wt+p.- like cell control (nothing) wt+s.- control for stock functioning.- ok: many small plaques
Conclusion.- It seems that the stock produced from the slow growing mutant is far better that the one produced from the sole stock. I will try producing the stock with the wild-type strain before keeping and characterizing this mutation.
My plan for today was stopped, since the culture for C1895 didn't grow at all.
possible explanations.- the bacterial colony picked was one that came from a glycerol stock. The reseed should have been picked. Another explanation could be an aeration effect. The tube wasn't covered with a cotton lid. Today's work was reduced to.- 1. find out how the centrifuge works. 2. find out the characteristics of the rotor used in order to concentrate the phage as the calendar protocol. 3. concentrate the phage in Falcon tubes with P4 buffer
about 2.- failed abruptly since no info on manuals could be found (there are no manuals!)and no internet info. Primo R is a discontinued subclass of centriguges. Apparently, this rotor has a max speed of 8500 rpm, but we don't know the rcf measure, which is a real problem. 3. the lysate was balanced (to perfection) and, after lots of efforts,  centrifuged at 8500 rpm (max rotor speed). there was a warning when  I tried to centrifugate at the indicated protocol speed.
After centrifugation, the tubes were drained in a paper towel and added .25 ml of P4 buffer as the protocol indicates. The pellets will be allowed to resuspend all night in the refrigerator, then centrifugated in eppendorfs to further remove cell debris.
September 20, 2009
Plaque assays for today.-
the concentrated phage were put to test, along with p4 vir1, everyone with 10 ul.
Although the overnight already had CaCl2, no more calcium was added. Next time, extra calcium will be added to help lysis.
september 21, 2009
The first part was completed succesfully, but 1 ml extra 1mM CaCl2 was added to the overnight culture. It is important to mention that the overnight culture was incubated with two plaques. More  supplemented LB medium was prepared for stock growth.
September 23, 2009
So, having already created stock of  P4, we have to manage how to extract the DNA from phage capsids. this is the next goal since we cannot amplify P$ regions from the lysogens we have. In turn we have to really control for the presence of P4 with the lack of amplification in order to discard the primers. Meanwhile, we have designed two more primers around the sid gene region.
the techniques involved use phenol chloroform extractions, formamide and isoamyl alcohol. It is said that phenol, the aqueous phase, is the one that stays with  the DNA and Chloroform, the organic phase, stays with the proteins, although this doesn't make complete sense, because the same mix is used to isolate phage particles.
September 23, 2009
Phage assay.-
Vir1 didn't present phage. ccells.- no plaques cN no cells. Phage concentrates.- S1.- the whole plate is a plaque with some resistants arising. S2 S3
These tubes can now be titered.
september 24, 2009
Phage DNA extraction.-
I have been reading protocols  and they vary dependind on many things.-
How debris-clean is the sample? Do you want to remove RNA? What substance will you use to separate capsid proteins from DNA?
After talk with Miguel , many things used int he protocols were clarified.-
glycogen.- used to condense DNA. formamide.- to further denature proteins so many substances are needed that maybe it's better to try other methods first (phenol chloroform is also toxic) and many things are lacking, like cetyl trimethyl ammonium bromide.
September 27, 2009
There are 2 things that are worth testing.  Dr. Calendar isolates Phage P4 DNA from a Diethyl amino ethyl cellulose (DEAE) column. The most alike element is  a Phage DNA extraction kit. While the protocol handed by Rosa lasts about 3 days, this kit promises DNA within 5 hours. The other one is a protocol that Dr. Kameyama from Cinvestav uses. Hopefully he will send it to us soon. We have to manage to get funding for the purchase of such kit by now.
September 28, 2009
We need to exchange at least one of the TOPO XL cloning kits we bought in advance. Dr. Pablo Vinuesa has to check whether he needs the TOPO cloning kit or not.
September 29, 2009
I have found another alternative: to apply the DNA extraction method from T7. it involves usage of ethanol , EDTA and heat.
I commited a small mistake in which i may have mistaken the correct phage-containing tube.
No pellet is seen at the bottom of the  tube after addition of ethanol and centrifugation. This is suspitious.
September 30, 2009
The DNA heat burst and ethanol precipitation failed. the suspects are the following:
No DNA was extracted from the capsids (The phage are still viable) The phages didn't burst.- we need a better denaturing agent, like urea . This could be due to extreme phage capsid stability. DNAse I somehow managed to destroy the DNA when it was just liberated from the capsid.
Tomorrow.- produce more P4 stock.
October 2 to 4, 2009
Several attempts to produce Phage P4 stock have been tried. Something strange is happening. lysis won't continue after it starts and I add EGTA and MgCl2. We suspect that P4 sid1 virulence is even more dependent on calcium in the medium, so for the next attempts, I won't add the chelator at lysis start.
october 5, 2009
The day started at almost 10:00, when an over(day) of c-1895 was put to grow to perform a lysis assay at night.
The plans for this day were.-
1.- centrifugate and test candidate lysates left on friday in order to prepare for the kit reactions, which need a starter 50 ml lysate. 2.- autoclave all the stuff needed and prepare supplemented P4 medium for the night assay. 3 prepare petri plates for testing the lysates.
I poured fractions of Friday's lysate flask 1 and 2 and named them as follows.-
oct 2 1.1
1.2        2.1        2.2
I also used two tubes from september 30's lysate which I wasn't able to centrifugate due to rotor space. the main idea of centrifugating is to clear out the cell debris from the lysate
and to further process the lysate until concentrating the phage or using it as the kit starting material. First I have to test for the presence of phage, so I will spend today  concentrating the phage completely and testing for  the infection of C2423 , the indicator strand, in plaque production before and after concentrating the phage.
I also started a protocol to produce more lysate, this time without adding the EGTA, so the calcium is not chelated too soon.  I verified lysis start in two flasks previously added bacteria incubated with phage about 2 hours after growth start. one of the flasks started lysing at A600 .66 and the other one at .4.
October 6, 2009
We had an iGEM meeting where we discussed about some aspects of the presentation, wiki, poster and other stuff. I checked the A600 of the cultures I left last night and it wass about 0.65 in both, so lysis stopped at some point and the bacteria continued growing.
I'm really excited because the lambda DNA purification kit just arrived around 4 pm! so I'll get my DNA today and start with PCRs tomorrow (or maybe today).
In general terms, the kit has a  standard protocol in manipulating phages. what changes is the anion exchange column. the protocol is a bit large, but time till getting purified DNA is a few hours : like half a day.
the steps are.- digest RNA and DNA, precipitate phage with PEG, lyse the phage and extract DNA, bind te DNA to a column, wash it and elute it.
From my perspective, carrying out the protocol was limiting in several respects. First, centrifugation on falcon tubes at 15000 g is not achievable on normal desktop centrifuge, so more power is needed. I need to ask Rosy how she does it. The other way around is to pour everything into eppendorf tubes and do it in a microcentrifuge, but that discards the 4 C temperature. as you use a narrow pipette tip, DNA may be shattered. Second, you need the tubes to contain exactly the same of everything in order to avoid balancing all the time (the other way is to have a balance...¬¬).
It seems that the best way to centrifuge the mixes is in a supercentrifuge and an ss-34 rotor.
October 7, 2009
Results from kit DNA isolation.- no DNA was seen in any of the fractions. There are many possible reasons, but the most likely ones is 1 .- the phage titer and 2.- the procedure bumps. everything will be clear in one more attempt and with a positive control.
Results of the day.- The application of the protocol improved dramatically. While yesterday I wasn't able to  centrifugate at 15000xg, thanks to otto geiger and his supercentrifuge, now I could do it. the outcome was a real pellet of potassium dodecyl sulfate and proteins , which could be poured into a fresh tube  with the loss of almost all the salt. Another thing I noticed last was that I could actually cold-centrifugate at >15000 x g with our own centrifuge and that RIGHT AFTER ELUTING,  I MUST CONTINUE  TO THE NEXT STEPS IN EPPENDORF TUBES  for the DNA not to irreversibly sediment at the bottom of the big centrifuge tube.
I took small aliquots of the steps after denaturing in order to check for the quality of the procedure. I expect a small, but at least visible yield of DNA. I can start working with that.
All the plaque assays  made in the morning had lots of phage, but maybe not as much as the Assays made on September 21 since the plaques were countable in most of them.
Gel electrophoresis.- The gel with the purification fractions is considered a complete success. Although the band seen is really thin and a bit difficult to see, the size is precisely the expected one. One thing to learn about the gel is that  a high voltage and more time are required. Next time, I will try running the gel for 1:30 hrs and at 80 volts.
Given that result, I was motivated enough to run a PCR for the essential and non-essential parts of P4. The essential part is a bit over 7 kb, and the non-essential part is around 3 kb. So the PCRs were performed this way: a reaction for each of the samples containing the template band, namely DNA1, which comes from centrifuge tube 1; DNA2, from centrifuge tube 2; Fraction 5 of the purification, the elution from the column, also had the band present.
one set of reactions attempts to amplifies P4 essential, and the other one, the non essential region. therefore, I prepared stocks for six reactions (3+3 with different primers each) and a negative control with the 7kb primers. I used lots of DNTPS so my reaction won't run out of them.
lanes 5 and 6 have a really dim band around 11 kb.
October 8, 2009
The Gel electrophoresis of last night's PCR has practically failed with both primer sets. Though we cannot discard the  fact that the polymerase itself didn't work due to lack of positive controls, lanes of the essential region for sample DNA 1 shows two blurred spots at the size of the template  at the expected size of amplification (7kb). It's not overkill to try again changing some parameters, like the polymerase or the annealing time.
For today, I will  make a restriction assay of my template in order to find out if it's the actual expected template. For this, I have decided to use the enzyme EcoRI, which has three restriction sites in the P4 chromosome. so the expected bands are as follows:
3 kb from site 1 to 2
Around 4 kb from site 2 to 3
around 7kb from 3 to 1
Partial restrictions.-
entire genome.- 11600 kb
1 to 3.- 3500 kb
2 to 1.- 7800 kb.- may be undistinguishable
3 to 2.- 10000 kb... may be difficult to see.
The restriction was completed to 75 ul with 15 ul of DNA when it was expected to be at 40. if it went wrong, I'll do it again.
The pcrs that were performed with RTth hot start were those of DNA1, DNA2 and fraction 5 (elution step) which happened to show the band of the genome.
if something goes wrong, we could blame the elution buffer's substances.
In the meantime, I'll try to obtain some concentrated DNA from the eluted fraction. It needs to be added 1 volume of isopropanol and centrfugated for 30 min at 15000 x g.
The result of concentrating the phage DNA from the resting elution was completely successful. The band can be seen completely at the corresponding size and the concentration is considerably high. Restrictions and PCRs will be made with this new stock. at 9 o'clock, the restriction time will have ended.
'October 10, 2009
Lane 8 shows the EcoRI restriction of the P4 genome.
There's something really strange going on in the restriction. Even though the DNA was considerably concentrated, after restriction with EcoRI nothing can be seen, not even the template DNA.
There's a possibility that the DNA, thanks to the reagents added in the restriction, runs away from the lane.
October 13, 2009
So yesterday I purified DNA again, this time enhancing the technique as mentioned when I started it.
The results of each DNA containing tube are here.-
At the left we see the steps pre-protein denaturing. between the ladders, we observe all the DNA contained in tubes from 1 to 5. tube 3 was spilled before centrifugation, so some DNA was lost. Tube five contains a real lot of DNA, so let's hope it helps see the digestion.
October 14, 2009
Digestions were planned once again for EcoRI alone. Two more were made ith NotI and boh NotI and Xba.
the expected bands for NotI alone  are 4.5 kb and 7 kb
The expected bands for NotI and XbaI are: +-2.3 kb, close to 5 kb, a 1.2 kb band and the NotI 4.5 kb band.
The first three lanes are what matters. First observation.- the DNA sample added to the restriction was of low concentration, what c 1.NotI bands observed.-
+- 11.5 kb.- entire genome; incomplete restriction. +- 7 kb +- 4500 kb
conclusion.- totally consistent
2.NotI, XbaI +-11.5 kb,.- incomplete restriction seemingly 2 bands below 4 5 kb and  over 4kb no clue about the 2.3 b, but there could be a band around 1000-1200 b
conclusion.- almost completely consistent. maybe the 2.3 kb band was somehow inefficiently cut.
3.EcoRI
This cut better than the others since there is less initial DNA and the DNA illumination  is a bit fuzzier.
What can be seen is a clear 8 kb band, a bit bright spot at 4.5 kb, and  a 3 kb band (almost unseen). So this is consistent (though not evidence) with the other restriction patterns. For now, let us assume that the P4 genome does behave like it should du to the quality of the bands, which don't let us discard  the alternative hypothesis.
Once with a good quantity and clues about P4 presence, there are less things to control for in the PCR.
October 15, 2009
Today I came really early to start the protocol for chemically competent C331 and C2423 cells. The idea is use the for transformation of the cox+ogr plasmid and activate with IPTG to look for phage production as plaques. These strains have to do with P4: one assesses its production and the other one has it lisogenic along with P2.
Other thing I did today is revisit the only aparent PCR product I obtained. It was from September 9. C331 was used for a PCR containing 2 sets of primers: P4 int and ogr. The bands expected were 300 bp and 3000 bp, but the band that appeared was of 700 bp and it's quite clear. We first had discarded this as something secondary, but it didn't contradict the possibility of having a good PCR with the correct primers used. What lacked in that experiment is more combinations of DNA and primers to consistently advance with the next part.
In response, I have planned and prepared another PCR reaction in which I prove all the primers against C331 and contrast with the results observed in the P4 sid1 genome.
Conclusion: don't fear repeating a suspicious experiment!!!!
October 16.- Finally the suspition is over. I ran the PCR  gel for testing P4 and P2 inside C331 . Here it is.-
Lanes. 1negative control (no DNA) 2 cox 3 ogr 4 C331 +int 5 ladder 6 C331 + P4 7 C331 INTp4 8 P4 DNA P4 9 P4 DNA 10. p4 DNA INT p4
So the mysterious band is still present... well. another experiment variation is to use different combinations of primers. I have set the PCR to come out tomorrow. let's see what happens.
October 17, 2009
I ran the gel testing different combinations of primers. there was no product either, so after all, we can conclude that the primers don't work. It has always been strange that both pairs of primers wouldn't work, but it seems to be the case this time. Final possibilities for lack of function.-
1. there is no phage DNA. The restriction assays don't show this, neither does the +- 11 kb band in the gel. 2. there are no sites for both primer sets.- valid  option 3. the genome is an isomorphism which doesn't allow amplification because of primer orientation.- at least some incomplete products should be seen.
October 19, 2009.
Yesterday I transformed C331 and C2423 cells with plasmid 18 + cox and ogr under the control of a lac operator. C2423 where the only resistants obtained, and negative controls are valid. So it's time to plan the experiment to validate the presence of cox and ogr inside the bacteria. It cannot be done from direct PCR, since C2423 already had both genes. It cannot be done only from plasmid extraction, unless the precise band is checked. the best method would be extracting plasmid and  performing PCR to it. At the same time, plating on IPTG plus controls will be done, eventhough it is not sure that it will work. The expected results are P2 plaque formation. Too sad that C331 didn't transform, because we cannot observe direct interaction with P4.  So growth experiment will be as follows.-
for a plasmid containing colony, there will be these plates.-
simple LB LB with IPTG simple LB added .1 M IPTG on half of the plate.
Retrieved from " http://2009.igem.org/Team:LCG-UNAM-Mexico:Journals:Nando%27s "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
