<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_LCG-UNAM-Mexico_WTM"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:LCG-UNAM-Mexico:WTM">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:LCG-UNAM-Mexico:WTM&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico:WTM&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico:WTM&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:LCG-UNAM-Mexico:WTM" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:LCG-UNAM-Mexico:WTM</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV class="MenuBar" id="LCG"><UL><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico" onmouseover="window.status= 'Click here to return Home'; return true;">Home
                </A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Record">Record
                </A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description">Project</A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description"><SPAN><SPAN>Description</SPAN>&gt;&gt;</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description"><SPAN><SPAN>Design</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description#a)_Delivery"><SPAN><SPAN>Delivery</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Description#b)_Defense"><SPAN><SPAN>Defense</SPAN></SPAN></A></LI></UL><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Parts"><SPAN><SPAN>Parts</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Resources"><SPAN><SPAN>Resources</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Safety"><SPAN><SPAN>Safety</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Results"><SPAN><SPAN>Results</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Conclusions"><SPAN><SPAN>Conclusions</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Modelling">Modelling</A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Wet_Lab/Objectives">Wet Lab</A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Wet Lab/Objectives"><SPAN><SPAN>Objectives</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team#The_team"><SPAN><SPAN>Team</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team#Acknowledgments"><SPAN><SPAN>Acknowledgments</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team#Collaborations"><SPAN><SPAN>Collaborations</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team#Sponsors"><SPAN><SPAN>Sponsors</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Team/Photos"><SPAN><SPAN>Photos!!!</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:LCG-UNAM-Mexico/Site map">Site map</A></LI></DIV><H2><SPAN class="mw-headline" id="WILD-TYPE_MOLECULAR_MODEL_.28WTM.29"><B>WILD-TYPE MOLECULAR MODEL (WTM)</B></SPAN></H2><TABLE><TBODY><TR valign="top"><TD width="900px" style="padding: 0 20px 0 0;"><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>WTM Simbiology ® diagram</B></DIV></DIV></DIV><P>WTM is a <A href="/Team:LCG-UNAM-Mexico:Molecular_model" title="Team:LCG-UNAM-Mexico:Molecular model">stochastic molecular model of bacteriophage T7 life cycle</A>, it was constructed to simulate wild-type development of this phage in E Coli.
Both models, WTM and <A href="/Team:LCG-UNAM-Mexico:KZM" title="Team:LCG-UNAM-Mexico:KZM">KZM</A>, were constructed to monitor the evolution and abundance of molecular species in the systems.
For the purpose of this project we are particularly concerned in the abundance of phage T7 (burst size) at the end of the cycle (720 seconds) in each infection event.
Ensembles of runs of WTM and KZM will provide us with data to build Burst Size Distributions for each model. 
BSDs are next utilized by the population models to recreate the impact of phage infections at the population level. 
<I>See also <A href="/Team:LCG-UNAM-Mexico:KZM" title="Team:LCG-UNAM-Mexico:KZM">KZM</A>, <A href="/Team:LCG-UNAM-Mexico:BSD" title="Team:LCG-UNAM-Mexico:BSD">BSD</A>, <A href="/Team:LCG-UNAM-Mexico/Modelling" title="Team:LCG-UNAM-Mexico/Modelling">Population Model</A>, <A href="/Team:LCG-UNAM-Mexico:CA" title="Team:LCG-UNAM-Mexico:CA">Cellular Automata</A>.</I></P></TD></TR><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#WILD-TYPE_MOLECULAR_MODEL_.28WTM.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">WILD-TYPE MOLECULAR MODEL (WTM)</SPAN></A></LI><LI class="toclevel-1"><A href="#Insertion_and_translocation_of_T7_DNA_at_different_times"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Insertion and translocation of T7 DNA at different times</SPAN></A></LI><LI class="toclevel-1"><A href="#Transcription_of_different_T7_DNA_segments_into_polycistronic_mRNAs"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Transcription of different T7 DNA segments into polycistronic mRNAs</SPAN></A></LI><LI class="toclevel-1"><A href="#Degradation_of_phage_mRNAs"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Degradation of phage mRNAs</SPAN></A></LI><LI class="toclevel-1"><A href="#Translation_of_phage_mRNAs_into_proteins"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Translation of phage mRNAs into proteins</SPAN></A></LI><LI class="toclevel-1"><A href="#T7_DNA_replication"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">T7 DNA replication</SPAN></A></LI><LI class="toclevel-1"><A href="#Procapsid_Assembly"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Procapsid Assembly</SPAN></A></LI><LI class="toclevel-1"><A href="#DNA_packaging_and_final_assembly"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">DNA packaging and final assembly</SPAN></A></LI><LI class="toclevel-1"><A href="#Simulation.2C_performance_and_BSD_construction"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Simulation, performance and BSD construction</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><P>The following critical processes are accounted in this model:
</P><UL><LI><H2><SPAN class="mw-headline" id="Insertion_and_translocation_of_T7_DNA_at_different_times">Insertion and translocation of T7 DNA at different times</SPAN></H2></LI></UL><P>Entry of T7 DNA into the host cell occurs in several distinct stages.
Phage's DNA is arranged in three classes of genes depending on their positions, it is translocated into the cell between 6 to 10 minutes after attachment, so this order and timing drives the phage's development. This phenomenon of DNA translocation is modeled here taking into account reported insertion speeds F. Buchholtz et al. 1987[1]. 
</P><UL><LI><H2><SPAN class="mw-headline" id="Transcription_of_different_T7_DNA_segments_into_polycistronic_mRNAs">Transcription of different T7 DNA segments into polycistronic mRNAs</SPAN></H2></LI></UL><P>It has been shown that T7 genes are expressed in overlapped polycistronic mRNAs.
Transcription of T7 polycistronic mRNAs occurs if and only if its coding DNA segment is available in the cell. Transcription is dependent of the set of genes inserted at a time. 
We define a set of transcription rates for every polycistronic mRNA taking into account constant bacterial or T7 RNA polymerase elongation rates and the length of the polycistronic mRNA, these transcription rates will be our rate limiting steps at the transcriptional level.
</P><UL><LI><H2><SPAN class="mw-headline" id="Degradation_of_phage_mRNAs">Degradation of phage mRNAs</SPAN></H2></LI></UL><P>We assume the same degradation rate for all T7 polycistronic mRNAs. Until now impact of this phenomenon had not been studied. It has been found that phage messengers are stabler than Bacterial mRNAs .Yoshihiko Yamada et al 1975 [2].  
</P><UL><LI><H2><SPAN class="mw-headline" id="Translation_of_phage_mRNAs_into_proteins">Translation of phage mRNAs into proteins</SPAN></H2></LI></UL><P>In this model, translation is simulated assuming an environment of unlimited amino acids and ribosomes. We also assume that the rate at which ribosomes incorporate amino acids is constant over all T7 mRNA.
 As it has been done for transcription we define a set of translation rates for every protein taking into account a constant ribosome elongation rate and the length of the protein, these translation rates will be our rate limiting steps at the translational level.
</P><UL><LI><H2><SPAN class="mw-headline" id="T7_DNA_replication">T7 DNA replication</SPAN></H2></LI></UL><P>DNA synthesis is simulated by taking elongation of T7 DNA polymerase as the rate-limiting step.
We assume an environment of limited free nucleotides so we can set a maximum number of T7 genomes produced in a single infection taking into account the size of the host genome (and this includes bacterial chromosomes, plasmids and other sources of free nucleotides). 
</P><UL><LI><H2><SPAN class="mw-headline" id="Procapsid_Assembly">Procapsid Assembly</SPAN></H2></LI></UL><P>This phenomenon is simulated in almost the same way as Drew Endy et al. 1996 [3] using mass action kinetics. 
</P><UL><LI><H2><SPAN class="mw-headline" id="DNA_packaging_and_final_assembly">DNA packaging and final assembly</SPAN></H2></LI></UL><P>Both processes are modeled using mass action kinetics as well. This last step requires complete procapsids, T7 DNA, and enough of each structural protein to complete the phage. The simulation assumes that packaging of DNA into the procapsid is the rate-limiting step for T7 progeny formation.
</P><UL><LI><H2><SPAN class="mw-headline" id="Simulation.2C_performance_and_BSD_construction">Simulation, performance and BSD construction</SPAN></H2></LI></UL><P>The next video has been recorded from a single simulation of WTM, with this algorithm we can monitor the evolution and abundance of molecular species in time. 
</P><P>For the purpose of our project we're just concerned in the burst size an infected cell could produce in each infection process, as we said above BSDs for WTM and KZM can be assembled with many runs of each model so we can compare the performance of our synthetic kamikaze and the wild-type systems at molecular and population levels.
Generated BSDs are next used by Cellular Automaton to choose the burst size an infected cell (with and without our kamikaze system) is going to release into the medium.
BSDs are also use by the agent-based model, in both cases BZDs drives the fate of the population. 
In this video there are two molecular species plotted, ABproteins or Major head proteins, this protein is the main component of T7 procapsids, it is evident the upper limit of molecules at which a procapsid assembly occurs. Production of this protein grows faster at late stages when polycistronic mRNAs responsible for them become more and more abundant in the cell. T7 plotted in red grows until it reach a burst size of about 250 phages at the end of the cycle/simulation (lysis time 720 seconds).
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>[1] F. Buchholtz and F.W. Schneider. 1987. <B>Computer simulation of T3/T7 phage infection using lag times</B>[2] Yoshihiko Yamada, Patricia A. Whitaker and Dai Nakada. 1975. <B>Chemical Stability of Bacteriophage T7 Early mRNA</B>[3] Drew Endy, Deyu Kong, John Yin. 1996. <B>Intracellular Kinetics of a Growing Virus: A Genetically Structured Simulation for Bacteriophage T7</B></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:LCG-UNAM-Mexico:WTM">http://2009.igem.org/Team:LCG-UNAM-Mexico:WTM</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:LCG-UNAM-Mexico:WTM" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:LCG-UNAM-Mexico:WTM" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico:WTM&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:LCG-UNAM-Mexico:WTM&amp;oldid=162911" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>