Team:NUDT CHINA/Design
MENU
HOME
TEAM
Team
Collaborations
PROJECT
Description
Design
Experiments
Notebook
InterLab
Contribution
Model
Results
Demonstrate
Improve
Attributions
PARTS
Parts
Basic Parts
Composite Parts
Part Collection
SAFETY
HUMAN PRACTICES
Silver HP
Integrated and Gold
Public Engagement
AWARDS
Applied Design
Entrepreneurship
Hardware
Measurement
Model
Plant
Software
JUDGING FORM
HOME
TEAM
Team
Collaborations
PROJECT
Description
Design
Experiment
Notebook
Results
Demonstrate
Improve
PARTS
Parts
Basic Parts
Composite Parts
Part Collection
Attributions
InterLab
Model
SAFETY
HP
Silver HP
Integrated & Gold
Public Engagement
Design
Our project attempts to demonstrate a novel design of miRNA inhibitor, named as miRNA Locker, which can be easily assembled using modularized DNA parts from a set of chemically synthetic oligo DNA library. Generally, the project design can be divided into two portions: miRNA Locker assembly and the construction of the oligo DNA library.
MiRNA Locker assembly
MiRNA Lockers are unsealed dumbbell shaped single-stranded DNAs (ssDNA) that is capable of binding specific miRNAs (Figure 1). The double-stranded region, named as the supporting module, serves as a “handle” to improve the stability of Locker. The loop regions, known as the functional modules, are designed to bind specific miRNAs through Watson-Crick base pairing. Intervals are inserted between modules for the convenience of the assembly procedure.
Figure 1. Structural diagram of miRNA locker
Though short miRNA Lockers can be directly synthesized, it is noticeable that the cost of direct synthesis raise rapidly as the length of the Locker increases. Hence, a low-cost and rapid assemble protocol is needed. Due to the difficulties in ligating ssDNAs, our strategy is to assemble double-strand DNA (dsDNA) containing the Locker strand and its complementary strand for the first step, then use techniques such as asymmetric PCR1 to generate ssDNAs.
Noticing that the Locker-containing dsDNA can be divided into several modularized DNA parts according to the structure of the Locker, we modified a previously reported oligo-linker mediated assembly (OLMA) method to assemble the Locker-containing dsDNA 2.
Figure 2. The construction and assembly of miRNA lockers
In our new method, named as Overlapped Oligo Assembly (OOA) method, modularized DNA parts is designed into the chemically synthesized DNA oligo, and overlapped 5’-overhangs are added for assembly. After that, dsDNAs can be obtained by annealing of two complementary single-strand oligonucleotides, then phosphorylated by T4 Polynucleotide Kinase to facilitate the following ligation reaction. T4 DNA ligase is use for their ligation onto pSB1C3 plasmid backbone. High-throughput one-step assemble of multiple short dsDNA nucleotides can therefore be achieved, and the correct assembling order is guaranteed by using identical overlap sequences among oligos, serving as zipcodes. The zipcodes are introduced as the intervals in a miRNA Locker.
The dsDNA assembles are capable of being sub-cloned into plasmid backbones for amplification and long-term storage. In view of that, a prefix and a suffix is added to the dsDNA oligos forming the 5’ and 3’ terminus of the assembly, ensuring that the plasmids we constructed are compatible with BioBrick RFC[10].
Finally, molecular cloning technique is used for the amplification and purification of the target gene, then asymmetric PCR is performed for the production of ssDNA from dsDNA, which generates ssDNA due to the unequal concentrations of primers used in the reaction1. Asymmetric PCR is the most cost effective method for ssDNA production, the PCR conditions for asymmetric PCR were exactly the same as those of symmetric PCR except for the ratios of primers used1. Asymmetric PCR products can then be harvested and purified using PAGE purification method.
The construction of the oligo DNA library
Another unique feature of our design is the construction of a set of chemically synthetic double-stranded DNA oligo library containing functional modules and supporting modules flanked by different combination of overhangs sequences.
A miRNA Locker with the functional of specific miRNAs inhibition can be easily assembled by choosing the related modularized DNA parts with proper overhangs.
However, it is worth noticing that the intervals inserted between modules have a great impact on the secondary structure of Locker, further affecting its effectiveness. Thus, a model is introduced to our project to select the optimal intervals for a specific miRNA Locker.
References
1 Citartan, M., Tang, T. H., Tan, S. C. & Hoe, C. H. Asymmetric PCR for good quality ssDNA generation towards DNA aptamer production. Songklanakarin Journal of Science & Technology 34, 125-131 (2012).
2 Zhang, S., Zhao, X., Tao, Y. & Lou, C. A novel approach for metabolic pathway optimization: Oligo-linker mediated assembly (OLMA) method. J Biol Eng 9, 23, doi:10.1186/s13036-015-0021-0 (2015).
