<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Saint_Rose_School_A_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Saint Rose School A/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><DIV class="navbar navbar-default navbar-fixed-top"><DIV class="container-fluid"><DIV class="collapse navbar-collapse navHeaderCollapse"><UL class="nav navbar-nav navbar-right"><LI style="font-family:helvetica, tahoma;"><A href="https://2016.igem.org/Team:Saint_Rose_School_A">Home</A></LI><LI style="font-family:helvetica, tahoma;"><A href="https://2016.igem.org/Team:Saint_Rose_School_A/Description">Project</A></LI><LI style="font-family:helvetica, tahoma;"><A href="https://2016.igem.org/Team:Saint_Rose_School_A/Notebook">Notebook</A></LI><LI style="font-family:helvetica, tahoma;"><A href="https://2016.igem.org/Team:Saint_Rose_School_A/Results">Results</A></LI><LI style="font-family:helvetica, tahoma;"><A href="https://2016.igem.org/Team:Saint_Rose_School_A/Safety">Safety</A></LI><LI style="font-family:helvetica, tahoma;"><A href="https://2016.igem.org/Team:Saint_Rose_School_A/Human_Practices">Human Practices</A></LI><LI style="font-family:helvetica, tahoma;"><A href="https://2016.igem.org/Team:Saint_Rose_School_A/Collaborations">Collaborations</A></LI><LI style="font-family:helvetica, tahoma;"><A href="https://2016.igem.org/Team:Saint_Rose_School_A/Attributions">Attributions</A></LI><LI style="font-family:helvetica, tahoma;"><A href="https://2016.igem.org/Team:Saint_Rose_School_A/Team">Team</A></LI></UL></DIV></DIV></DIV><H1 style="
    color: black;
    text-align: center;
    padding: 8%;
    font-size: 90px;
">Project</H1><H1 id="problem" style="
    color: black;
    text-align: center;
    padding: 2%;
    font-size: 50px;
">Problem</H1><DIV id="name"><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 20px;
    text-align: justify;
"> The Metropolitan Region of Chile is a basin enclosed by mountain ranges that do not allow a free circulation of the pollutant particles, causing an inestability of the pollution levels in different areas of the city.

During several months of 2015, the Metropolitan Region had high polution rates, those high rates in a long-term cause the inhabitants different health problems. These rates have mantained, despite the measures that have been taken to decrease this situation.
One of the main pollutants that exists in the Metropolitan Region is the carbon monoxide, an invisible and odourless gas. After our research, we became aware of how dangerous this gas is for the health. The damage that provokes to the body depends on the amount in which is present, if the levels are too high, it might cause death.

Carbon monoxide is 200 times more related to the hemoglobin than the oxygen, when the carbon monoxide gets together with the hemo group, it creates the carboxyhemoglobin, blocking the entry of oxygen to the blood, causing hypoxia and eventually, death. For these reasons, the efficient and precise detection is essential.

The presence of CO is related to industrial processes such as paper factories and steel foundries. It is also present in fires, machines and homes with heating systems. That is why many people are exposed everyday to this dangerous pollutant.
</P><H2 id="solution" style="
    color: black;
    text-align: center;
    padding: 2%;
    font-size: 50px;
"> Solution </H2><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 20px;
    text-align: justify;
"> In Chile, the Metropolitan Region, which is the most populated area in the country, is constantly exposed to very high pollution levels, with frequent environmental emergencies. Due to this situation, at first instance we wanted to detect the most harmful pollutants to our health, but after carefully analysing the issue we decided to focus on carbon monoxide specifically.
Through synthetic biology we were able to genetically modify living organisms that react to the presence of CO. This process is achieved by transferring DNA sequences of the Rhodospirillum rubrum protobacteria to an E. coli bacterium. We will implement these genes into the E.coli to use them as carbon monoxide detectors, since its organism is easier to work with because there is a better understanding of its genome. The bacteria is also going to contain chromoproteins in order to indicate the presence of CO.
The modified E. coli will be inside an accessory like a bracelet or a wristband, and it is going to react to the presence of CO by changing its colour to purple or pink depending on the quantity of the pollutant in the air. 
All of this will be made using the knowledge gained at the Academia de emprendimiento científico.    
Synthetic biology and the study of the genome of different organisms, allow nowadays access to scientific information stored in databases available to general public, which is necessary to do the application of sequences, genes, biological processes made by living organisms to replicate the metabolism of programmed bacteria to solve daily problems. After a bibliographic research we identified Rhodospirillum rubrum protobacteria, which is capable of living under either aerobic or anaerobic conditions. This protobacteria can oxidize carbon monoxide to seize the electrons released in this proceses to generade energy; to detect CO, it possesses a mechanism that works with the transcription factor CooA, which contains a heme group with iron that binds to the molecule.
 </P><H2 id="design" style="
    color: black;
    text-align: center;
    padding: 2%;
    font-size: 50px;
"> Design</H2><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 20px;
    text-align: justify;
"> After roughly explaining the genes we will use, it's time to explain our circuit step by step first of all.
Identifying the Rhodospirillum Rubrum protobacteria that can live in both aerobically and anaerobically conditions, it is a versatile organism that can obtain energy through various processes, especially photosynthesis. It can also use carbon monoxide as its sole power source, so it has a detection mechanism that works with a COOA transcription factor, which has a heme group that binds to the CO molecule as it has iron on its structure. In this transcription factor we have decided to implement a LacI inducible promoter through the IPTG molecule, because if it wasn’t there, the bacteria would be in constant operation until being disintegrated. Notably, LacI will not affect the detection of carbon monoxide by COOA, because they have no relationship to each other.
The COOA transcription factor activates the pCoof and / or pCooM promoters, also from the R.rubrum protobacteria, these promoters have the characteristic of being transcribed according to the toxicity of carbon monoxide in the atmosphere, completely depending of COOA. To verify this procedure, the circuit will have chromoproteins of purple and pink coloration
It is important to mention that we implement these genes in the E. coli bacteria, as we have knowledge of its genome and its ease of using.
As it can be represented in the following image: 

 </P><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 20px;
    text-align: justify;
"> However, after the chromoproteins were activated, we decided to implement &quot;quorum sensing&quot; tools,  which is a mechanism for regulating gene expression, which will be referred to two &quot;lux&quot; to generate a third promoter. Designating for pCooF Luxi promoter and pCooM LuxR. Once LuxR is expressed , it will join the AHL molecule that is synthesized by SAM through LuxI activity, provided that LuxI has been expressed.
After the AHL molecule has joined LuxR expression, it will allow the transcription of genes under the Plux promoter, generating the third level of detection of CO, this will have a green fluorescence protein.
In the end, the result will be three levels of detection, pCooF detecting low levels promoter, the promoter Plux having an intermediate level of detection, and to detect high levels we will have pCooM as a promoter.

 </P><H2 id="parts" style="
    color: black;
    text-align: center;
    padding: 2%;
    font-size: 50px;
margin-top: 100px;
"> Parts</H2><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 20px;
    text-align: justify;
"><STRONG> LacI:</STRONG>This part is an inverting regulator sensitive to LacI and CAP.
It contains two protein binding sites. The first binds the CAP protein, which is generally present in E.coli and is asocciated with cell health and availability of glucose. The second binds LacI protein.
-In the absence of LacI protein and CAP protein, this part promotes transcription.
-LacI can be inhibited by IPTG.
 </P><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 20px;
    text-align: justify;
"><STRONG>RBS:</STRONG> A Ribosome Binding Site or Ribosomal Binding Site is a sequence of nucleotides upstream of the start codon of an mRNA transcript that is responsible for the recruitment of a ribosome during the initiation of protein translation.
 </P><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 20px;
    text-align: justify;
"><STRONG>CooA:</STRONG> A CO-sensing transcription factor from Rhodospirillum rubrum, is a CO-binding heme protein
 </P><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 20px;
    text-align: justify;
"><STRONG>pCooF and pCooM:</STRONG> The two CooA-regulated R. rubrum promoters, contain 2-fold symmetric DNA sequences which are CooA-binding sites and are similar to the CRP consensus sequence. This similarity corralates with the tantamount helix-turn-helix motifs CooA and CRP. The CooA binding sites lie at the 243.5 and 238.5 positions relative to the transcription start sites in PcooF and PcooM.
 </P><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 20px;
    text-align: justify;
"><STRONG>Chromoprotein purple: </STRONG>This chromoprotein from the coral Acropora millepora, amilCP, naturally exhibits strong color when expressed. The protein has an absorbance maximum at 588 nm giving it a blue/purple color visible to the naked eye, thereby requiring no instruments to observe. The strong color is readily observed in both LB or agar culture, in less than 24 hours of incubation.

 </P><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 20px;
    text-align: justify;
"><STRONG>Chromoprotein pink:</STRONG>This chromoprotein “eforRed” naturally exhibits red/pink color when expressed. The color is slightly weaker than RFP. On agar plates and in liquid culture, the color is readily visible to naked eye in less than 24 hours of incubation.

 </P><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 20px;
    text-align: justify;
"><STRONG> LuxI:</STRONG>is a synthase that converts SAM into a small molecule called an acyl-homoserine lactone (AHL). It can diffuse across cell membranes and is stable in growth media at a range of pH. In the natural system, the pLuxR promoter controls transcription of the LuxI enzyme leading to a positive feedback loop that increases transcription from the right hand lux promoter. In addition controlling the transcription of luxI, the promoter also controls transcription of luciferase.

 </P><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 20px;
    text-align: justify;
"><STRONG>LuxR:</STRONG> is a constitutively expressed protein that can bind AHL. When bound to AHL it can stimulate transcription from the right hand lux promoter (pLuxR). 

 </P><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 20px;
    text-align: justify;
"><STRONG>SAM:</STRONG>benzoic acid/salicylic acid carboxyl methyltransferase I; converts salicylic acid to methyl sale<STRONG>AHL:</STRONG> [N-]acyl-homoserine lactones
These are small signalling molecules which are employed in &quot;quorum sensing&quot; systems. They are also known as autoinducers (AIs) and are present in many Gram-negative bacteria. 
<STRONG>Plux:</STRONG>The lux cassette of V. fischeri contains a left and a right promoter. The right promoter gives weak constitutive expression of downstream genes. This expression is up-regulated by the action of the LuxR activator protein complexed with the autoinducer, 3-oxo-hexanoyl-HSL. Two molecules of LuxR protein form a complex with two molecules of the signalling compound homoserine lactone (HSL). This complex binds to a palindromic site on the promoter, increasing the rate of transcription.<STRONG>Green fluorescent protein</STRONG> derived from jellyfish Aequeora victoria wild-type GFP


 </P><H1 id="method" style="
    color: black;
    text-align: center;
    padding: 2%;
    font-size: 50px;
">Method</H1><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 20px;
    text-align: justify;
"> We are developing our project using knowledge in molecular biology, genetic engineering and laboratory protocols.
We will describe the procedures followed by the team:
 </P><P style="color: black; margin: 100px 110px 100px 110px; font-size: 20px; text-align: justify;"><LI style="text-align: left; margin-left: 10%;list-style: none;"><STRONG>Method for culturing:</STRONG></LI><LI style="text-align: left; margin-left: 10%; "> 5 ml of LB in control tube falcon </LI><LI style="text-align: left; margin-left: 10%; "> 5 ml of LB falcon tube 5 ul control of antibiotic.</LI><LI style="text-align: left; margin-left: 10%;"> In a 100 ul sample falcon LuxR + 5 ml of LB + antibiotic tube 5 ul.</LI><LI style="text-align: left; margin-left: 10%;"> In a 100 ul sample falcon LuxI + 5 ml of LB + antibiotic tube 5 ul.</LI></P><P style="color: black; margin: 100px 110px 100px 110px; font-size: 20px; text-align: justify;"><LI style="text-align: left; margin-left: 10%;list-style: none;"><STRONG>Purified plasmid extraction procedure:</STRONG></LI><LI style="text-align: left; margin-left: 10%; ">Add 1000 ul sample in an Eppendorf tube</LI><LI style="text-align: left; margin-left: 10%; ">We remove the liquid to leave only the pellet (sample).</LI><LI style="text-align: left; margin-left: 10%; ">Add 250 ul of solution resuspension</LI><LI style="text-align: left; margin-left: 10%; ">Add 250 ul of lysis solution and flip the sample 4 times.</LI><LI style="text-align: left; margin-left: 10%; ">We add 10 ul of alkaline protease to flip 4 times</LI><LI style="text-align: left; margin-left: 10%; ">Incubate 5 minutes at room temperature.</LI><LI style="text-align: left; margin-left: 10%; ">Add 350 ul of neutralizing solution and flip 4 times.</LI><LI style="text-align: left; margin-left: 10%; ">We centrifuged at maximum speed for 10 minutes.</LI><LI style="text-align: left; margin-left: 10%; ">600 ul insert the sample into a column and this one in the collection tube.</LI><LI style="text-align: left; margin-left: 10%; ">We centrifuged at maximum speed for 1 minute.</LI><LI style="text-align: left; margin-left: 10%; ">We removed and reinserted lysate column in the collection tube</LI><LI style="text-align: left; margin-left: 10%; ">We add 750 ul of wash solution.</LI><LI style="text-align: left; margin-left: 10%; ">We centrifuged at maximum speed for 1 minute.</LI><LI style="text-align: left; margin-left: 10%; ">Discard the supernatant and reinserted in the column.</LI><LI style="text-align: left; margin-left: 10%; ">Add 250 ul of wash solution.</LI><LI style="text-align: left; margin-left: 10%; ">We centrifuged at maximum speed for 3 minutes.</LI><LI style="text-align: left; margin-left: 10%; ">Column transferred to a sterile tube.</LI><LI style="text-align: left; margin-left: 10%; ">Add 80 ul of nuclease-free water.</LI><LI style="text-align: left; margin-left: 10%; ">We centrifuged at maximum speed for 1 minute.</LI><LI style="text-align: left; margin-left: 10%; ">We store the sample at -20 ° C.</LI></P><P style="color: black; margin: 100px 110px 100px 110px; font-size: 20px; text-align: justify;"><LI style="text-align: left; margin-left: 10%;list-style: none;"><STRONG>Preparation competent calcium:</STRONG></LI><LI style="text-align: left; margin-left: 10%; ">Falcon tube with 5 ml of bacteria and. DHSα coli grown overnight. </LI><LI style="text-align: left; margin-left: 10%; ">Add 500 ul culture and bring eppendorf tube.</LI><LI style="text-align: left; margin-left: 10%; ">Centrifuge at maximum speed 5 minutes, discard the supernatant </LI><LI style="text-align: left; margin-left: 10%; ">Add 300 ul of cold CaCl, resuspend and centrifuge again at full speed for 5 minutes, discard the supernatant.</LI><LI style="text-align: left; margin-left: 10%; ">Add 50 ul of cold CaCl and proceed to transformation.</LI><LI style="text-align: left; margin-left: 10%;list-style: none; margin-top:10%;"><STRONG>LuxI transformation procedure, LuxR and Plux:</STRONG></LI><LI style="text-align: left; margin-left: 10%; ">Add 2 ul of DNA and mix with light taps between the pipes. </LI><LI style="text-align: left; margin-left: 10%; ">Let it stand for 20 to 30 minutes.</LI><LI style="text-align: left; margin-left: 10%; ">Give a thermal shock by immersing eppendorf tubes in a beaker with water at a temperature of 42 ° C for one minute.</LI><LI style="text-align: left; margin-left: 10%; ">Place on ice for 2 minutes.</LI><LI style="text-align: left; margin-left: 10%; ">Add 200 ul of LB medium and incubated at 37 ° C for 20 or 30 minutes.</LI><LI style="text-align: left; margin-left: 10%; ">Use 5ml LB plus 5 ul of antibiotic.</LI><LI style="text-align: left; margin-left: 10%; ">Incubate at 37 ° C overnight.</LI><LI style="text-align: left; margin-left: 10%; ">Check colony growth 12 hours of incubation.</LI><LI style="text-align: left; margin-left: 10%;list-style: none;margin-top:10%;"><STRONG>Procedure for cutting:</STRONG></LI><LI style="text-align: left; margin-left: 10%; ">10 Ul MQ</LI><LI style="text-align: left; margin-left: 10%; "> Ul sample -5 </LI><LI style="text-align: left; margin-left: 10%; "> 1 Ul BSA</LI><LI style="text-align: left; margin-left: 10%; ">2 Ul buffer</LI><LI style="text-align: left; margin-left: 10%; ">2 Ul of enzyme (1 and 1) -EcoR1, Pstl</LI><LI style="text-align: left; margin-left: 10%;list-style: none;margin-top:10%;"><STRONG>Procedure for cryogenics:</STRONG></LI><LI style="text-align: left; margin-left: 10%; ">200ul Glycerol.</LI><LI style="text-align: left; margin-left: 10%; ">800ul Bacteria.</LI></P><H1 style="
    color: black;
    text-align: center;
    padding: 2%;
    font-size: 50px;
">Bibliography</H1><P style="
    color: black;
    margin: 100px 110px 100px 110px;
    font-size: 18px;
    text-align: justify;
"> 1. Youn H, Kerby RL, Conrad M, Roberts GP. Functionally Critical Elements of CooA-Related CO Sensors. J Bacteriol. 2004;186(5):1320–9.


2. Shelver D, Kerby RL, He Y, Roberts GP. CooA, a CO-sensing transcription factor from Rhodospirillum rubrum, is a CO-binding heme protein. Biochemistry. 1997;94(October):11216–20.



3. He Y, Gaal T, Karls R, Donohue TJ, Gourse RL, Roberts GP. Transcription Activation by CooA , the CO-sensing Factor from Rhodospirillum rubrum. Biochemistry. 1999;274(16):10840–5.



4. Exposures E, Atrazine TO, The IN. 2 . Relevance To Public Health. Public Health. 1997;3(Iii):11–20.


5 Gera C, Srivastava S. Quorum-sensing: The phenomenon of microbial communication. Curr Sci. 2006;90(5):666–76.


 </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>