"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Bielefeld-Germany/Theory
From 2011.igem.org
Home
Team
Students
Supervisors
Photos
Bielefeld University
Acknowledgements
Contact
Project
Description
Achievements
Approach
Background
Bisphenol A
S-layer
NAD+ detection
Technology
DIY Nanobiotech Guide
Future Applications
Labjournal
Protocols
Overview
Molecular Genetics
Production
Analytics
Materials
Results
Data Page
Summary
Bisphenol A
S-layer
NAD+ detection
Since Regionals
Model
Human Practices
Overview
Summary
Cell-free Biosafety
Outreach
Open Lab Course
Science Cafe
High School
CeBiTec Symposium
Student Convention
Strategic Process
Beacon Box
Press
Safety
Partner
Contents
1 Bisphenol A
1.1 Bisphenol A and its effects on mammals
1.2 Bisphenol A degradation
1.3 Further applications of bisphenol A degrading BioBricks and enzymes
2 S-layer
2.1 Further applications of S-layer proteins
3 NAD+ detection
3.1 NAD+ in general
3.2 Biosynthesis of NAD+
3.3 Bacterial DNA ligases
3.4 Applications for molecular beacons
3.5 Further applications for the NAD+ bioassay
4 References
Bisphenol A
monomer, plastic, polycarbonate, epoxy resins, food containers, forbidden in EU and canada in baby flasks etc.
Bisphenol A and its effects on mammals
endocrine disruptor, estrogenic, teratogenic, infertility etc.
Bisphenol A degradation
There exist a lot of bacteria in nature that can degrade xenobiotic substances such as phenolic compounds or endocrine disruptors. In a lot of soil samples that were taken from contaminated soil to find organisms that do so, sphingomonads were extraordinarily often isolated ( Stolz, 2009 ). In 2005, Sasaki et al. isolated a soil bacterium from the Sphingomonas genus which is able to degrade the endocrine disruptor bisphenol A (BPA) with a unique rate and efficiency compared to other BPA degrading organisms. This strain, called Sphingomonas bisphenolicum AO1, is able to completely decompose 120 mg BPA L-1 in about 6 hours while other strains need days of cultivation (e.g. Sphingomonas strain BP-7 isolated by Sakai et al. (2007) ).
The full bisphenol A degradation pathway which is found in nature is shown in figure X (taken from KEGG database). BPA is metabolized to 4-Hydroxybenzaldehyde, 4'-Hydroxyacetophenone and 4-Hydroxybenzoate which can be used by some bacteria as carbon source. These and other metabolites of the BPA degradation pathway can be found in the supernatant of BPA containing cultivations of S. bisphenolicum AO1. They can grow on BPA as the only carbon source, too ( Sasaki et al., 2005a ).
Fig. X: blabla
Bisphenol A is mainly hydroxylated into the products 1,2-Bis(4-hydroxyphenyl)-2-propanol and 2,2-Bis(4-hydroxyphenyl)-1-propanol by some kind of oxidoreductase acting with NADH or NADPH. A total of three genes are responsible for this BPA hydroxylation: a cytochrome P450 (CYP, bisdB), a ferredoxin (Fd, bisdA) and a ferredoxin-NAD+ oxidoreductase (FNR) ( Sasaki et al., 2005b ). The three gene products act together to reduce BPA while oxidizing NADH + H+. The cytochrome P450 (BisdB) reduces the BPA and is oxidized during this reaction. BisdB in its oxidized status is reduced by the ferredoxin (BisdA) so it can reduce BPA again. The oxidized BisdA is reduced by a ferredoxin-NAD+ oxidoreductase consuming NADH + H+ so the BPA degradation can continue ( Sasaki et al., 2005b ). The bisdAB genes from S. bisphenolicum AO1 were isolated, transformed into and expressed in E. coli and enabled this bacterium to degrade BPA, too ( Sasaki et al., 2008 ). In addition, the BisdAB proteins from S. bisphenolicum AO1 were able to degrade BPA in a cell-free system (enzyme assay) in which spinach reductase ( EC 1.18.1.2 ) was added ( Sasaki et al., 2005b ).
In 2008, the iGEM team from the University of Alberta submitted the bisdAB genes from S. bisphenolicum AO1 to the registry of standard biological parts in the so called Freiburg BioBrick assembly standard (<partinfo>K123000</partinfo> and <partinfo>K123001</partinfo>). In order to degrade BPA in a cell free system, a FNR BioBrick is also needed. As demonstrated, the BisdA and BisdB work together in a cell-free system with spinach reductase and they work intracellular in E. coli, it can be assumed that it is possible to produce all components necessary to degrade BPA in a cell free system in E. coli. To achieve this a ferredoxin-NADP+ oxidoreductase BioBrick, isolated from E. coli, is provided by the Bielefeld-Germany 2011 iGEM team (<partinfo>K525499</partinfo>).
The whole electron transport chain between the three enzymes involved in BPA degradation and the BioBricks needed to enable this reaction in vitro are shown in the following figure (please have some patience, it's an animated .gif file):
File:Bielefeld-Germany2011-BPAdegrad2.gif
Fig. X: blabla
Further applications of bisphenol A degrading BioBricks and enzymes
S-layer
Further applications of S-layer proteins
NAD+ detection
NAD+ in general
Nicotinamide adenine dinucleotide, abbreviated NAD+, is an ubiquitous and indispensable cofactor for all living organisms ( Bi et al., 2011 ). The molecule has a wide range of physiological functions in cellular processes (fig. 2) and has a great impact on the cell integrity ( Grahnert et al., 2011 ). It is integrated into the energy metabolism and determines the redox status of the cell due to its purpose as a reduction equivalent. This is strongly linked to the control of signaling and transcriptional events. For instance, NAD+ has been recently described as a modulator of immune functions ( Grahnert et al., 2011 ) and a master regulator of transcription ( Ghosh et al., 2010 ). Several enzymes are tightly regulated by the cellular balance between the oxidized and reduced form of NAD+ and therefore regarded as a potential target for therapeutics ( Sauve, 2008 ). Indeed, NAD+ acts as a substrate for a wide range of proteins including NAD+-dependent protein deacetylases, poly(ADP-ribose) polymerases and several transcription factors ( Houtcooper et al., 2010 ). A disorder of the cellular NAD+ level has usually relevance in symptoms and diseases like stroke, cardiac ischemia, epilepsy, huntington disease, wallerian degeneration, cancer, type two diabetes mellitus (T2DM), neutrophil survival, longevity, obesity as well as cardiovascular and neurodegenerative disease ( Sauve, 2008 ; Houtcooper et al., 2010 ). The regulation of NAD+-dependent pathways may have a major contribution to oxidative metabolism and life span extension ( Houtcooper et al., 2010 ). The biosynthetic pathways for NAD+ are proposed as promising novel antibiotics targets against pathogens ( Bi et al., 2011 ).
Figure 2: NAD(P) physiological function (taken from Bi et al., 2011 ).
Biosynthesis of NAD+
NAD+ is embedded in the cellular metabolism and has a great impact on essential cellular processes. The biosynthesis in most bacteria and plants starts with the anaerobic conversion of the amino acid aspartate to quinoline. Alternatively, in yeast, humans and some bacterial microbes the quinoline derives from tryptophan in an aerobic manner (fig 3). This is a crucial step that leads under consumption of nicotinic acid to the nicotinate ring system via formation of nicotinic acid monoculeotide (NAMN), respectively ( Sauve, 2008 ). After conjugation of adenosine mononucleotide (AMP) the NAD synthetase forms nicotinamide adenine dinucleotide (NAD). Additionally, it has been recently reported that in humans exists an nicotinic acid-independent reaction catalysed by the enzyme nampt/PBEF, which uses directly nicotinamide as a substrate ( Rongvaux, 2002 ). Diet or pharmalogical agents can thereby serve as a source for nicotinamide or nicotinic acid ( Sauve, 2008 ). Fundamental processes as the cellular energy metabolism determine the balance between the oxidized (NAD+) and reduced (NADH) form of NAD.
Figure 3: Principle of de novo NAD+ biosynthesis of organisms that use tryptophan as a source. Bottom: paths of metabolism for human nicotinamide, nicotinic acid and nicotinamide riboside that lead to formation of NAD+ (taken from Sauve, 2008 ).
Bacterial DNA ligases
Bacterial DNA ligases are important enzymes for DNA repair and replication. The big difference between bacterial and eukaryotic DNA ligases is their specifity for cofactors. While eukaryotic DNA ligases use ATP as a source for building a phosphodiester bond in the DNA`s phosphate backbone the bacterial ones display a NAD+-dependent biochemical catalysis. These enzymes are totally essential for bacterial survival and are therefore in focus for designing highly specific antiinfectives ( Gajiwala & Pinko, 2004 ). All bacterial DNA ligases possess highly conserved functional domains. Generally, they can be devided into four functionally distinguishable domains, each with characteristic motifs:
Domain 1: Ia, Ib
Domain 2: Oligomer-binding (OB) beta-barrel
Domain 3: Zinc-finger, helix-hairpin-helix motif (HhH)
Domain 4: BRCT domain
All bacterial DNA ligases share three fundamental steps during reaction mechanism:
Adenylation of DNA ligase by transferring hydrolysed AMP from NAD+ to the active site lysine and release of NMN
Tranfer of the AMP from DNA ligase to the 5'-phosphate group of nicked DNA
Formation of a phosphodiester bond by attack of the nick 3'-OH on the 5'-phosphoanhydride linkage and release of free AMP
Figure 7: During the adenylation reaction cycle of NAD+-dependent DNA ligases the linker between the two subdomains Ia (gray) and Ib (yellow) undergoes conformational change to form open and closed state. A: Open state of Ia domain exposing the adenylation site to solvent. B: Closed state of Ia domain is achieved by NAD+ binding. C: AMP-bound form providing adenylated site exposed to solvent and an open form accessible to incoming DNA. D: Deadenylated (NMN-bound) form (taken from Gajiwala & Pinko, 2004 ).
Figure 6: Model of LigA from E. coli bound to DNA. B: The different colours indicate N-terminal domain Ia (blue), nucleotidyltransferase (NTase) domain (cyan), OB-fold domain (magenta), helix-hairpin-helix (HhH) domain (beige) and a zinc (Zn)-finger domain (green). Typically, there is a BRCT domain at the C-terminus (not shown). C: Aspartic acid, arginine and glycine residues in the NTase and HhH domains are responsible for intramolecular interaction promoting protein/DNA interaction within the DNA major groove opposite the adenylated nick, called AppDNA (taken from et al., 2007 ).
Applications for molecular beacons
Molecular beacons can be applied to a wide range of biological and diagnostical studies. Whenever there is a specific nucleic acid which is supposed to be detected they can be utilized. Molecular beacons are used for the detection and distinction of retroviral DNA from different types of HIV, for instance ( Marras et al., 2006 ). Even for indentification of proteins which interact with DNA or RNA they are useful. In sum, there are a lot of opportunities to apply molecular beacons ( Antony & Subramaniam, 2001 ):
Detection of PCR amplification products
Mutational analysis
Genotyping and allele discrimination
Clinical diagnosis
Oligonucleotide degradation studies
Analysis of DNA/RNA hybridization
Realtime visualization of hybridization in cells
Analysis of triplex DNA formation
Protein/DNA interaction
Enzymatic assays
Biosensors
Figure 4: Hybridization of the molecular beacon and a complementary target forms the molecular beacon`s open state which leads to an increase of the fluorescence intensity. The molecular beacon can be immmobilized using specific biomolecular interaction, e.g. streptavidin/biotin or covalent cross-linking (taken from Kim et al., 2008 ).
Figure 5: Single nucleotide polymorphism of nucleic acids, including different alleles as indicated, can be detected by differently designed and labeled molecular beacons (taken from Kim et al., 2008 ).
Figure 6: Realtime in vivo imaging of specific mRNAs utilizing two MBs that target adjacent positions on the mRNA allowing FRET to take place (taken from Kim et al., 2008 ).
Further applications for the NAD+ bioassay
The reviewed molecular beacon based NAD+ bioassay can be applied to biochemical and biomedical studies ( Tang et al., 2011 ). Accordingly, it can be utilized to detect NAD+/NADH-dependent enzymatic processes. The low limit of detection, its reliability and manageability provides a practical alternative to present colorimetric, fluorometric, chemiluminescent, electrochemical or mass spectrometric methods detecting NAD+ or NADH. In context of clinical applications and therapeutics the NAD+ bioassay can be useful to monitor cellular NAD+ levels during treatments of tissues or cell cultures for identification of drug targets, for instance. It may also be applied in the field of diagnostics. Finally, the principle of the proposed NAD+ bioassay can be used for cell-free optical biosensors by taking molecular beacons either immobilized on the sensor surface or being present in the reaction medium ( Tang et al., 2011 ).
References
Antony T, Subramaniam V (2001) Molecular Beacons: Nucleic Acid Hybridization and Emerging Applications, Journal of Biomolecular Structure & Dynamics 19(3):497-504 .
Bi J, Wang H, Xiel J (2011) Comparative genomics of NAD(P) biosynthesis and novel antibiotic drug targets, Journal of Cellular Physiology 226(2):331-340 .
Gajiwala KS, Pinko C (2004) Structural Rearrangement Accompanying NAD+ Synthesis within a Bacterial DNA Ligase Crystal, Structure 12(8):1449-1459 .
Ghosh S, George S, Roy U, Ramachandran D, Kolthur-Seetharam U (2010) NAD: A master regulator of transcription, Biochimica et Biophysica Acta 1799(10-12):681-693 .
Grahnert A, Grahnert A, Klein C, Schilling E, Wehrhahn J, Hauschildt S (2011) NAD+: A modulator of immune functions, Innate Immunity 17(2):212-233 .
Houtkooper RH, CantÃ³ C, Wanders RJ, Auwerx J (2010) The Secret Life of NAD+: An Old Metabolite Controlling New Metabolic Signaling Pathways, Endocrine Reviews 31(2):194-223 .
Kim Y, Sohn D, Tan W (2008) Molecular Beacons in Biomedical Detection and Clinical Diagnosis, International Journal of Clinical and Experimental Pathology 1:105-116 .
Marras SA, Tyagi S, Kramer FR (2006) Real-time assays with molecular beacons and other fluorescent nucleic acid hybridization probes, Clinica chimica acta 363(1-2):48-60 .
Nandakumar J, Nair PA, Shuman S (2007) Last Stop on the Road to Repair: Structure of E. coli DNA Ligase Bound to Nicked DNA-Adenylate, Molecular Cell 26(2):257-271 .
Rongvaux A, Shea RJ, Mulks MH, Gigot D, Urbain J, Leo O, Andris F (2002) Pre-B-cell colony-enhancing factor, whose expression is up-regulated in activated lymphocytes, is a nicotinamide phosphoribosyltransferase, a cytosolic enzyme involved in NAD biosynthesis, European Journal of Immunology 32(11):3225-3234 .
Sakai K, Yamanaka H, Moriyoshi K, Ohmoto T, Ohe T (2007) Biodegradation of Bisphenol A and Related Compounds by Sphingomonas sp. Strain BP-7 Isolated from Seawater, Bioscience, Biotechnology, and Biochemistry 71(1):51-57 .
Sauve AA (2008) NAD+ and Vitamin B3: From Metabolism to Therapies, The Journal of Pharmacology and Experimental Therapeutics 324(3):883-893 .
Sasaki M, Maki J, Oshiman K, Matsumura Y, Tsuchido T (2005a) Biodegradation of bisphenol A by cells and cell lysate from Sphingomonas sp. strain AO1, Biodegradation 16(5):449-459 .
Sasaki M, Akahira A, Oshiman K, Tsuchido T, Matsumura Y (2005b) Purification of Cytochrome P450 and Ferredoxin, Involved in Bisphenol A Degradation, from Sphingomonas sp. Strain AO1, Appl Environ Microbiol 71(12):8024-8030 .
Sasaki M, Tsuchido T, Matsumura Y (2008) Molecular cloning and characterization of cytochrome P450 and ferredoxin genes involved in bisphenol A degradation in Sphingomonas bisphenolicum strain AO1, J Appl Microbiol 105(4):1158-1169 .
Stolz A (2009) Molecular characteristics of xenobiotic-degrading sphingomonads, Appl Microbiol Biotechnol 81:793-811 .
Retrieved from " http://2011.igem.org/Team:Bielefeld-Germany/Theory "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
