<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Project_Background_S-Layer"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Project/Background/S-Layer">Page               </A></LI><LI><A href="/Talk:Team:Bielefeld-Germany/Project/Background/S-Layer">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Background/S-Layer&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Background/S-Layer&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Project/Background/S-Layer" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Project/Background/S-Layer</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu-igem"><UL id="menu" class="sf-menu"><LI><A style="border-left:0px;" href="https://2011.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Students">Students</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Advisors">Supervisors</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Photos">Photos</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Bielefeld-University‎‎">Bielefeld University</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></UL><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Description">Description</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Achievements">Achievements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Appoach">Approach</A></LI><LI><A href="#">Background</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/Technology">Technology</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer/Guide">DIY Nanobiotech Guide</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Future">Future Applications</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Labjournal">Labjournal</A></LI><LI><A href="#">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Genetics">Molecular Genetics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing">Production</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Analytics">Analytics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials">Materials</A></LI><LI><A href="#">Results</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Data_Page">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/SinceAmsterdam">Since Regionals</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Modell">Model</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Cell-free">Cell-free Biosafety</A></LI><LI><A href="#">Outreach</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_lab_course">Open Lab Course</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_science_cafe">Science Cafe</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/high-school">High School</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/cebitec-symposium">CeBiTec Symposium</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/student-convention">Student Convention</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/strategic-process">Strategic Process</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Black-box">Beacon Box</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Press">Press</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#S-layer"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">S-layer</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#S-layers_in_general"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">S-layers in general</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#The_S-layer_protein_PS2_of_Corynebacterium_glutamicum"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">The S-layer protein PS2 of Corynebacterium glutamicum</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#The_S-layer_protein_SgsE_of_Geobacillus_stearothermophilus_NRS_2004.2F3a"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">The S-layer protein SgsE of Geobacillus stearothermophilus NRS 2004/3a</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#The_S-layer_protein_SbpA_of_Lysinibacillus_sphaericus_CCM_2177"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">The S-layer protein SbpA of Lysinibacillus sphaericus CCM 2177</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Further_applications_of_S-layer_proteins"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Further applications of S-layer proteins</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#References"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="S-layer">S-layer</SPAN></H1><P>Molecular nanotechnology, especially nanobiotechnology, starts to use and modify functionalized surfaces. Particularly the immobilization of self-assembling biomolecules draws an increasing attention. The advantages of using immobilized enzymes in well-defined positions on nanostructured surfaces may even be greater. Whereas there exist various possibilities to immobilize enzymes like covalent binding, adsorption and physical adsorption, immobilization using S-layer enyzme fusion proteins provides and additional  advantage due to the defined position of the enzymes. Therefore functional groups of enzymes can be immobilized in directed orientation (compare figure on the right).
</P><P>Self-assembly is the organization of molecules into defined structures, lowering the free energy of the system. Interaction between the molecules is non-covalent (<I>e.g.</I> hydrophobic, van der Waals forces, molecular stacking) (<A href="http://onlinelibrary.wiley.com/doi/10.1002/smll.200700200/abstract" class="external text" rel="nofollow">Schäffer <I>et al.</I>, 2007</A>). 
Many biomolecules such as proteins, polysaccharides and lipids have the ability to self-assemble into different shapes (<I>e.g.</I> spherical, rod- or sheet-like shapes), allowing several specific functions as virus capsids, cytoskeleton components or extracellular surface layer protein.
The so-called paracrystalline cell surface-layers (S-layer) are built up from S-layer proteins and are one of the most common surface structures in bacteria and archaea. 
</P><P>They are regarded as the outmost cell envelope of prokaryotic organisms (<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6968.2006.00573.x/full" class="external text" rel="nofollow">Sleytr <I>et al.</I>, 2007</A>).
</P><H2><SPAN class="mw-headline" id="S-layers_in_general">S-layers in general</SPAN></H2><P>S-layer proteins fulfill various functions as molecular sieves, ion traps and protective coats (<A href="http://onlinelibrary.wiley.com/doi/10.1002/3527600035.bpol7011/abstract" class="external text" rel="nofollow">Sleytr <I>et al.</I>, 2005</A>). They build up periodic structures, posses pores of identical size and morphology and show equal physicochemical properties on each molecular unit. Isolated they have the ability to reassemble into two-dimensional crystals equal to the S-layer structures that are found on intact bacterial cells. S-layers have the ability to form self-assembly products in solution and to recrystallize into monomolecular layers on solid supports, at air-water interface and on lipid films. They can completely cover liposomes and nanocapsules as well as small beads. S-layers are mainly composed of a single (glyco-)protein species, assembled into layers that are completely covering the outer part of the bacterial cell. In organisms they may represent up to 20 % of the total protein content of a bacterial cell. Most S-layer proteins are weakly acidic (pI 4 - 6) and contain a high proportion of hydrophobic amino acids as well as few or no sulphur-containing amino acids. Their molecular mass varies between 40 – 200 kDa and is often strain-specific. The assembled S-layer lattices in bacteria are generally 5 – 20 nm thick and in archaea up to 70 nm thick. The protein subunits of S-layers are arranged in lattices with different symmetry: oblique (p1, p2), square (p4) or hexagonal (p3, p6) with a center-to-center spacing of the subunits of 3 – 35 nm (compare figure on the right). S-layers are highly porous with a porosity of 30 – 70 % (<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1742-4658.2006.05606.x/abstract" class="external text" rel="nofollow">Sleytr <I>et al.</I>, 2007</A>).
</P><P>Various S-layer proteins from archaea and eubacteria are glycosylated, with strain-specific modifications. S-layer proteins were the first prokaryotic proteins that were shown to exhibit this characteristic. Until now, glycosylation has been proven for several archaeal S-layer proteins. Among the bacterial species glycosylation was demonstrated only for S-layer proteins of Bacillaceae (<A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1997.d01-1868.x/abstract" class="external text" rel="nofollow">Chami <I>et al.</I>, 1997</A>, <A href="http://onlinelibrary.wiley.com/doi/10.1002/smll.200700200/abstract" class="external text" rel="nofollow">Schäffer <I>et al.</I>, 2007</A>).
</P><P>In gram-negative bacteria, S-layer proteins are the exclusive cell wall component. In gram-positive bacteria and archaea, S-layers assemble on the outmost part of a firm wall matrix, which is composed mostly of peptidoglycan and pseudomurein. In gram-negative bacteria, S-layers are linked to specific lipopolysaccharides (LPS) (<A href="http://onlinelibrary.wiley.com/doi/10.1002/3527600035.bpol7011/abstract" class="external text" rel="nofollow">Sleytr <I>et al.</I>, 2005</A>). For gram-positive bacteria, a cell-wall-targeting domain could be identified at the N-terminal end of many S-layer proteins. The domain facilitates binding to a specific secondary cell wall polymer (SCWP) by a lectin-type binding (<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1742-4658.2006.05606.x/abstract" class="external text" rel="nofollow">Sleytr <I>et al.</I>, 2007</A>). It was found that some S-layer proteins consist of two distinct domains with different functions. One domain is involved in the assembly with other S-layer protein monomers and the other domain mediates the interaction with the cell wall. Several SLH (S-layer homologous) domains have been identified at the amino-terminal region of different S-layer proteins and at the carboxy-terminal region of cell-associated exoproteins. The domain may be repeated within the sequence and is involved in anchoring the S-layer proteins to the cell surface (<A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1997.d01-1868.x/abstract" class="external text" rel="nofollow">Chami <I>et al.</I>, 1997</A>, <A href="http://www.sciencedirect.com/science/article/pii/S0966842X99015139" class="external text" rel="nofollow">Sleytr &amp; Beveridge, 1999</A>). In various S-layer proteins from <I>bacillacaea</I> the deletion of significant parts of the carboxy-terminus or amino-terminus did not affect self-assembly and the capability of the S-layer proteins to form lattices (<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1742-4658.2006.05606.x/abstract" class="external text" rel="nofollow">Sleytr <I>et al.</I>, 2007</A>).
</P><P>The supramolecular structure as well as the mechanism of binding the outermost cell wall vary between S-layers of different species, leading to the development of different isolation procedures. S-layers normally are attached to the cell wall through non-covalent binding, and can therefore be isolated and completely disintegrated in dissociating agents (<I>e.g.</I> lithium chloride), metal-chelating agents (<I>e.g.</I> ethylendiaminetetraacetic acid EDTA), chaotropic denaturants such as urea or guanidine hydrochloride and by raising or lowering pH. After removal of the disrupting agent the reassembly takes place. (<A href="http://onlinelibrary.wiley.com/doi/10.1002/3527600035.bpol7011/abstract" class="external text" rel="nofollow">Sleytr <I>et al.</I>, 2005</A>).
</P><H2><SPAN class="mw-headline" id="The_S-layer_protein_PS2_of_Corynebacterium_glutamicum">The S-layer protein PS2 of <I>Corynebacterium glutamicum</I></SPAN></H2><P>The S-layer of the gram-positive bacterium <I>Corynebacterium glutamicum</I> ATCC 14067 is formed by the PS2 protein. The protein is encoded by the gene <I>cspB</I>. The mature protein has a molecular mass of 52.5 kDa. It is devoid of any sulfur-containing amino acids, whereas its hydrophobic nature is due to a high content of hydrophobic amino acids. Although there exist a lot of different S-layer proteins, PS2 has no similarities to any other protein in the EMBL database. The S-layer of <I>C. glutamicum</I> is characterized by a hexagonal lattice symmetry. Attachment between S-layer and cell wall was found to be due to the hydrophobic carboxy-terminus of the PS2 protein. It was found that peptidoglycan is probably not involved in interaction between the PS2 S-layer and the cell because the interaction between PS2 and the cell is disrupted by adding detergents. Also the S-layer protein from <I>C. glutamicum</I> does not contain a SLH domain, which is characteristic for several S-layer proteins and other enzymes bound to the peptidoglycan. Besides some other S-layer proteins show a carboxy-terminal hydrophobic sequence of 20 – 24 amino acids. (<I>e.g. Halobacterium halobium, Haloferax volcanii, Rickettsia rickettsii</I>) (<A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1997.d01-1868.x/abstract" class="external text" rel="nofollow">Chami <I>et al.</I>, 1997</A>, <A href="http://www.sciencedirect.com/science/article/pii/S016816560400241X" class="external text" rel="nofollow">Hansmeier <I>et al.</I>, 2004</A>).
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">TEM Microscopy of PS2 with hexagonal lattice structure.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">AFM Microscopy of PS2 with hexagonal lattice structure.</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="The_S-layer_protein_SgsE_of_Geobacillus_stearothermophilus_NRS_2004.2F3a">The S-layer protein SgsE of <I>Geobacillus stearothermophilus</I> NRS 2004/3a</SPAN></H2><P>SgsE monomers are naturally assembled in a lattice with oblique symmetry (p2) exhibiting a well-defined periodicity and distances of 9.4 – 11.6 nm between the proteinaceous subunits. The S-layer protein SgsE of <I>Geobacillus stearothermophilus</I> NRS 2004/3a consists of 903 amino acids, including a 30 amino acid signal peptide (SLH-domain) at the amino-terminus. 
The carboxy-terminal part of SgsE is the larger part of the protein, encoding the self-assembly information. The protein is formed by the <I>sgsE</I> gene, has a calculated mass of 93.7 kDa and an isoelectric point of 6.1. When isolated SgsE maintains its ability to self-assemble. Depending on salt concentration, duration of dialysis to remove the detergent and its amino acid sequence it builds up five types of self-assembly products. These products are formed like flat sheets and cylinders (<A href="http://onlinelibrary.wiley.com/doi/10.1002/smll.200700200/abstract" class="external text" rel="nofollow">Schäffer <I>et al.</I>, 2007</A>).
</P><H2><SPAN class="mw-headline" id="The_S-layer_protein_SbpA_of_Lysinibacillus_sphaericus_CCM_2177">The S-layer protein SbpA of <I>Lysinibacillus sphaericus</I> CCM 2177</SPAN></H2><P>The S-layer protein SbpA of <I>Lysinibacillus sphaericus</I> CCM 2177 consists of 1,268 amino acids and is encoded by the <I>sbpA</I> gene. The lattice symmetry formed through self-assembly is a square structure (p4) with a center-to-center spacing of the morphological units of 13.1 nm. <A href="http://pubs.acs.org/doi/abs/10.1021/bc800445r" class="external text" rel="nofollow">Badelt-Lichtblau <I>et al.</I> (2009)</A> found that the self-assembly process is strongly dependent on presence of bivalent cations. Therefore the assembly is dirigible, because in absence of bivalent cations SbpA stays water-soluble. At the amino-terminal end the protein consists of three typical SLH (S-layer homologous) domains and an additional 58 amino acid-long SLH-like motif. Those domains recognize a distinct type of secondary cell wall polymers (SCWP), consisting of disaccharide units. The S-layer protein SbpA is extremely resistant to deletions changing its properties. The deletion of up to 237 carboxy-terminal amino acids does not affect the ability to self-assemble into square lattices. After deletion of 350 amino acids the lattice structure changes from square (p4) to oblique (p1) symmetry <A href="http://pubs.acs.org/doi/abs/10.1021/bc800445r" class="external text" rel="nofollow">(Badelt-Lichtblau <I>et al.</I>, 2009)</A>.
</P><H2><SPAN class="mw-headline" id="Further_applications_of_S-layer_proteins">Further applications of S-layer proteins</SPAN></H2><P>The ability of self-assembly and forming defined nanostructures enables the use of S-layer proteins as building blocks in a biomolecular construction kit for the generation of a universal nanobiotechnological matrix. By using the matrix as a nano pinboard for further components or just the nanostructure itself, a great variety of applications in modular nanobiotechnology, biomimetics, bioanalytics and medical diagnostic are imaginable <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6968.2006.00573.x/full" class="external text" rel="nofollow">(Sleytr <I>et al.</I>, 2007)</A>.
</P><P>A current topic in S-layer research is the design of more sensitive and improved optical and electrochemical sensors. One example is the luminescence lifetime based oxygen sensor, described by <A href="http://www.ncbi.nlm.nih.gov/pubmed/19765970" class="external text" rel="nofollow">Schleicher <I>et al.</I> (2009)</A>. Therefore metalloporphyrin dyes which change luminescence properties in presence and absence of oxygen were bound to S-layer proteins. These proteins were used to form monolayer coatings on optical fiber surfaces allowing evanescent field fluorescence excitation of the linked fluorophores. The key benefits of using S-layer proteins for this type of sensor are the imparted anti-fouling properties and the good biocompatibility. This is especially valuable for sensing applications in complex biological fluids or implantable sensors.
</P><P>Concentrating on the repetitive features of S-layer proteins to form a well defined nano lattice, the use as filtration units becomes reasonable. <A href="http://jb.asm.org/cgi/content/full/182/4/859" class="external text" rel="nofollow">Sára and Sleytr (2000)</A> described a so-called isoporous S-layer ultrafiltration membrane (SUM). This filtration system is based on a nylon microfiltration membrane as recrystallization surface for S-layer fragments of <I>Bacillus sphaericus</I> CCM 2120, to build up a constant lattice with a well-defined pore size of 12.5 nm. 
Another advantage of this membrane is the presence of functional groups in defined positions and orientations. These functional groups could be used for highly reproducible chemical modifications to optimize molecular sieving properties and non-specific adsorption (antifouling) characteristics.
</P><P>The self-assembly and immobilization properties of S-layer fusion proteins also open new innovative perspectives in clinical applications like immune therapy, blood purification and drug targeting. <A href="http://www.jimmunol.org/content/172/11/6642.full" class="external text" rel="nofollow">Bohle <I>et al.</I> (2004)</A> showed that a genetic fusion of an allergen to SbsC S-layer proteins from <I>Geobacillus stearothermophilus</I> combine reduced allergenicity with immunomodulatory capacity. These qualities make them to an ideal allergen carrier/adjuvants in specific immunotherapy, which is the only causative treatment for type I allergy. 
Fusion proteins of a synthetic analogue of the immunoglobulin G (IgG)-binding B-domain of protein A of <I>Staphylococcus aureus</I> and SbpA S-layer proteins from <I>Bacillus sphaericus</I> CCM 2177 are proposed as a microsphere-based extracorporeal blood detoxification system to remove IgG from human plasma from patients suffering from autoimmune disease. The use of microparticles to detoxify the plasma allows rapid removal of the pathogenic substances and leads to higher binding capacities than commercially available immunoadsorbents <A href="http://aem.asm.org/cgi/content/short/70/3/1514" class="external text" rel="nofollow">(Völlenkle<I>et al.</I>, 2003)</A>.
The natural/original ability of crystallization on lipid membranes/liposomes for heightening the mechanical and thermal stability of S-liposomes and the possibility for immobilizing biologically active molecules can be cross-linked and exploited as a matrix for the covalent attachment of functional molecules to built up a biocompatible drug targeting or gene therapy system.
</P><P>But the main advantage of S-layer proteins is their application as universal matrices for the immobilization. This versatility could be upgraded by fusing the S-layers with minimum-sized core-streptavidin to build up a universal affinity matrix for any kind of biotinylated molecule. This concept fits exactly in the primary aim of constructing a biomolecular construction kit. <A href="http://onlinelibrary.wiley.com/doi/10.1002/smll.200500147/abstract" class="external text" rel="nofollow">Huber <I>et al.</I> (2006)</A> used the streptavidin matrix for building up functional sensor surface by recrystallization of these heterotetramers on gold chips. This could be exploited for the development of cheaper DNA or protein chips.
</P><P>In conclusion, S-layer proteins are one of the most innovative and promising discoveries for combining biology with engineering in the new field of nanobiotechnology.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>Badelt-Lichtblau H, Kainz B, Völlenkle C, Egelseer EM, Sleytr UB, Pum D, Ilk N (2009) Genetic engineering of the S-layer protein SbpA of Lysinibacillus sphaericus CCM 2177 for the generation of functionalized nanoarrays, <A href="http://pubs.acs.org/doi/abs/10.1021/bc800445r" class="external text" rel="nofollow">Bioconjug Chem. 20(5):895-903</A>.
</P><P>Bohle B, Breitwieser A, Zwölfer B, Jahn-Schmid B, S´ara M, Sleytr UB &amp; Ebner C (2004) A novel approach to specific allergy treatment: the recombinant fusion protein of a bacterial cell surface (S-layer) protein and the major birch pollen allergen Bet v 1 (rSbsC-Bet v 1) combines reduced allergenicity with immunomodulating capacity, <A href="http://www.jimmunol.org/content/172/11/6642.full" class="external text" rel="nofollow">J Immunol 172: 6642–6648</A>.
</P><P>Chami M, Bayan N, Peyret JL, Gulik-Krzywicki T, Leblon G, Shechter E (1997) The S-layer protein of Corynebacterium glutamicum is anchored to the cell wall by its C-terminal hydrophobic domain, <A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1997.d01-1868.x/abstract" class="external text" rel="nofollow">Mol Microbiol. 23(3):483-92</A>.
</P><P>Hansmeier N, Bartels FW, Ros R, Anselmetti D, Tauch A, Pühler A, Kalinowski J (2004) Classification of hyper-variable Corynebacterium glutamicum surface-layer proteins by sequence analyses and atomic force microscopy, <A href="http://www.sciencedirect.com/science/article/pii/S016816560400241X" class="external text" rel="nofollow">J Biotechnol. 26;112(1-2):177-93</A>.
</P><P>Huber C, Liu J, Egelseer EM, Moll D, Knoll W, Sleytr UB &amp; S´ara M (2006b) Heterotetramers formed by an S-layer-streptavidin fusion protein and core-streptavidin as nanoarrayed template for biochip development, <A href="http://onlinelibrary.wiley.com/doi/10.1002/smll.200500147/abstract" class="external text" rel="nofollow">Small 2: 142–150</A>.
</P><P>Sára M and Sleytr U B (2000) S-layer proteins: A minireview, <A href="http://jb.asm.org/cgi/content/full/182/4/859" class="external text" rel="nofollow">Journal of bacteriology 182(4): 859–868</A>.
</P><P>Scheicher SR, Kainz B, Köstler S, Suppan M, Bizzarri A, Pum D, Sleytr UB, Ribitsch V (2009) Optical oxygen sensors based on Pt(II) porphyrin dye immobilized on S-layer protein matrices, <A href="http://www.ncbi.nlm.nih.gov/pubmed/19765970" class="external text" rel="nofollow">Biosensors and Bioelectronics 25: 797–802</A>.
</P><P>Schäffer C, Novotny R, Küpcü S, Zayni S, Scheberl A, Friedmann J, Sleytr UB, Messner P (2007) Novel biocatalysts based on S-layer self-assembly of Geobacillus stearothermophilus NRS 2004/3a: a nanobiotechnological approach, <A href="http://onlinelibrary.wiley.com/doi/10.1002/smll.200700200/abstract" class="external text" rel="nofollow">Small 3(9):1549-59</A>.
</P><P>Sleytr UB, Beveridge TJ (1999) Bacterial S-layers. <A href="http://www.sciencedirect.com/science/article/pii/S0966842X99015139" class="external text" rel="nofollow">Trends Microbiol. 7(6):253-60</A>.
</P><P>Sleytr UB, Sára M, Pum D, Schuster B, Messner P, Schäffer C (2005) Self-assembling protein systems: microbial S-layers, in: <A href="http://onlinelibrary.wiley.com/doi/10.1002/3527600035.bpol7011/abstract" class="external text" rel="nofollow">Steinbüchel A, Fahnestock SR (Eds.), Polyamides 34 and complex proteinaceous materials, Wiley-VCH, Weinheim, pp. 285-338</A>.
</P><P>Sleytr UB, Egelseer EM, Ilk N, Pum D, Schuster B (2007) S-Layers as a basic building block in a molecular construction kit, <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1742-4658.2006.05606.x/full" class="external text" rel="nofollow">FEBS J 274(2):323-34</A>.
</P><P>Sleytr UB, Huber C, Ilk N, Pum D, Schuster B, Egelseer EM (2007) S-layers as a tool kit for nanobiotechnological applications, <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6968.2006.00573.x/full" class="external text" rel="nofollow">FEMS Microbiol Lett. 267(2):131-44</A>.
</P><P>Völlenkle C, Weigert S, Ilk N et al. (2004) Construction of a functional S-layer fusion protein comprising an immunoglobulin G-binding domain for development of specific adsorbents for extracorporeal blood purification,<A href="http://aem.asm.org/cgi/content/short/70/3/1514" class="external text" rel="nofollow">Appl Environ Microbiol 70: 1514–1521</A>.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Bielefeld-Germany/Project/Background/S-Layer">http://2011.igem.org/Team:Bielefeld-Germany/Project/Background/S-Layer</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Project/Background/S-Layer" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Project/Background/S-Layer" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Background/S-Layer&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Background/S-Layer&amp;oldid=260063" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>