"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-Germany/Results/S-Layer/SbpA
From 2011.igem.org
Home
Team
Students
Supervisors
Photos
Bielefeld University
Acknowledgements
Contact
Project
Description
Achievements
Approach
Background
Bisphenol A
S-layer
NAD+ detection
Technology
DIY Nanobiotech Guide
Future Applications
Labjournal
Protocols
Overview
Molecular Genetics
Production
Analytics
Materials
Results
Data Page
Summary
Bisphenol A
S-layer
NAD+ detection
Since Regionals
Model
Human Practices
Overview
Summary
Cell-free Biosafety
Outreach
Open Lab Course
Science Cafe
High School
CeBiTec Symposium
Student Convention
Strategic Process
Beacon Box
Press
Safety
Partner
Contents
1 SbpA from Lysinibacillus sphaericus CCM 2177
1.1 Expression in E. coli
1.2 Purification of SbpA fusion protein
1.2.1 Final purification strategy
1.2.2 Alternative purification strategy
1.3 Immobilization behaviour
1.3.1 Optimal bead to protein ratio for immobilization
SbpA from Lysinibacillus sphaericus CCM 2177
Fig.1: Square structure of S-layer lattices.
The S-layer protein SbpA of Lysinibacillus sphaericus CCM 2177 consists of 1,268 amino acids and is encoded by the sbpA gene. The lattice symmetry formed through self-assembly is a square structure (p4) with a center-to-center spacing of the morphological units of 13.1 nm (see Fig. 1). Badelt-Lichtblau et al. (2009) found that the self-assembly process is strongly dependent on presence of bivalent cations. Therefore the assembly is dirigible, because in absence of bivalent cations SbpA stays water-soluble. At the amino-terminal end the protein consists of three typical SLH (S-layer homologous) domains and an additional 58 amino acid-long SLH-like motif. Those domains recognize a distinct type of secondary cell wall polymers (SCWP), consisting of disaccharide units. The S-layer protein SbpA is extremely resistant to deletions changing its properties. The deletion of up to 237 carboxy-terminal amino acids does not affect the ability to self-assemble into square lattices. After deletion of 350 amino acids the lattice structure changes from square (p4) to oblique (p1) symmetry (Badelt-Lichtblau et al., 2009) .
Expression in E. coli
The SbpA gene under the control of a T7 / lac promoter (<partinfo>K525403</partinfo>) was fused to mCitrine ( BBa_J18931 ) using Freiburg BioBrick assembly for characterization experiments.
The SbpA|mCitrine fusion protein was overexpressed in E. coli KRX after induction of T7 polymerase by supplementation of 0.1 % L-rhamnose and 1 mM IPTG using the autinduction protocol by Promega.
Fig.2: Growth curve of E. coli KRX expressing the fusion protein of SbpA and mCitrine with and without induction, cultivated at 37 °C in autoinduction medium with and without inductor, respectively. A curve depicting KRX wildtype is shown for comparison. After induction at approximately 4 h the OD600 of the induced K525405  drops slightly when compared to the uninduced culture. Both cultures grow significantly slower than KRX wildtype.
Fig.3: RFU to OD600 ratio of E. coli KRX expressing the fusion protein of SbpA and mCitrine with and without induction. A curve depicting KRX wildtype is shown for comparison. After induction at approximately 4 h the RFU to OD600 ratio starts to rise in the induced culture. Compared to the uninduced culture the ratio is roughly three times higher. The KRX wildtype shows no variation in the RFU to OD600 ratio.
Purification of SbpA fusion protein
Figure 4: Fluorescence of collected fractions during purification of <partinfo>K525405</partinfo> fusion protein. Abbreviations: pell.: pellet, s.n.: supernatant, ret.: retentate, perm.: permeate.
As seen in the analysis of the cultivations with expression of SbpA | mCitrine fusion proteins, these proteins form inclusion bodies in E. coli. Inclusion bodies have the advantage that they are relatively easy to clean-up and are resistant to proteases. So the first purification step is to solve and set-free the inclusion bodies. This step is followed by two filtrations (300 kDa UF and 100 kDa DF/UF) to further concentrate and purify the S-layer proteins. After the filtrations, the remaining protein solution is dialyzed against ddH2O for 18 h at 4 °C in the dark. The dialysis leads to a precipitation of the water-insoluble proteins. After centrifugation of the dialysate the water-soluble S-layer monomers remain in the supernatant and can be used for recrystallization experiments.
The fluorescence of some collected, important fractions of this purification strategy is shown in Figure 4.
A lot of protein is lost during the purification especially after centrifugation steps (compared to filtrations). The fluorescence in the urea containing fractions is lowered due to denaturation of the fluorescent protein. This purification strategy is very simple and can be carried out by nearly everyone in any lab being one first step to enable real do it yourself nanobiotechnology.
Final purification strategy
Scheme of purification strategy for SbpA (fusion) proteins:
First, SbpA is expressed in E. coli under the control of a T7 / lac promoter for separation of growth and production phase due to metabolic stress of the S-layer expression. Because the SbpA protein is forming inclusion bodies in E. coli, an inclusion body purification with urea follows the cell lysis. The S-layers are further concentrated and purified by two ultrafiltration / diafiltration steps (300 kDa and 100 kDa) and afterwards dialysed against water leading to the precipitation of water-insoluble proteins. The supernatant contains the monomeric SbpA solution.
Click for detailed information
Alternative purification strategy
Sometimes the inclusion body purification does not give very clean fractions. That's why we developed another purification strategy for <partinfo>K525405</partinfo> monomers after inclusion body purification. This strategy consists of a capture with an IEX chromatography followed by a purification with HIC chromatography. Because of issues with the PES membranes we used to rebuffer our samples (see below - PES binds most of the S-layers) we lost a lot of our proteins in this step between the two chromatographies. So there was not enough protein left to stain it with Coomassie in a SDS-PAGE and time ran out to perform a silver staining. The fluorescence measurements of the collected fractions of the chromatographies are shown in the figures below.
Fig. 5: Fluorescence in the fractions of an anion exchange chromatography (IEX) with monomeric SbpA | mCitrine solution. A 1 mL HighTrap DEAE sepharose column by GE Healthcare was used for purification. The binding buffer (buffer A) contained 25 mM sodium chloride and the elution buffer (buffer B) 1 M sodium chloride (both 25 mM acetate buffer, pH 6.0). Abbreviations: FT = flow through, W = wash, E = elution.
Fig. 6: Fluorescence in the fractions of a hydrophobic interaction chromatography (HIC) with monomeric SbpA | mCitrine solution eluted from an IEX (see above). A 1 mL HighTrap butyl sepharose column by GE Healthcare was used for purification. The binding buffer (buffer A) contained 1200 mM ammonium sulfate and the elution buffer (buffer B) no ammonium sulfate (both 50 mM ammonium phosphate buffer, pH 7.0). Abbreviations: FT = flow through, E = elution.
Immobilization behaviour
Figure 7: Measured fluorescence of collected fractions of immobilization of purified <partinfo>K525405</partinfo> on silica dioxide beads (n = 3, 100 mg mL-1 SiO2, time of recrystallization: 4 h).
After purification, solutions of monomeric SbpA S-layer proteins can be recrystallized and immobilized on silicon dioxide beads in recrystallization buffer (0.5 mM Tris-HCl, pH 9, 10 mM CaCl2). After the recrystallization procedure the beads are washed with and stored in ddH2O at 4 °C in the dark. The fluorescence of the collected fractions of  recrystallization experiments with <partinfo>K525405</partinfo> are shown in Fig. 7. 100 mg beads were coated with 100 µg of protein. The figure shows, that not all of the protein is immobilized on the beads (supernatant fraction) but the immobilization is pretty stable (very low fluorescence in the wash). After the immobilization, the beads show a high fluorescence indicating the binding of the SbpA | mCitrine fusion protein.
Optimal bead to protein ratio for immobilization
To determine the optimal ratio of silica beads to protein for immobilization, the degree of clearance ϕC in the supernatant is calculated and plotted against the concentration of silica beads used in the accordant immobilization experiment (compare Fig. 8):
(1)
The data was collected in three independent experiments. The fluorescence of the samples was measured in the supernatant of the immobilization experiment after centrifuging the silica beads. The fluorescence of the control was measured in a sample which was treated exactly like the others but no silica beads were added. 100 µg protein were used for one immobilization experiment. The data was fitted with a sigmoidal dose-response function of the form
(2)
with the Hill coefficient p, the bottom asymptote A1, the top asymptote A2 and the switch point log(x0) (R² = 0.997).
The fit indicates that a good silica concentration for 100 µg of protein is 200 - 250 mg mL-1. This set-up leads to saturated beads with low waste of protein. So a good protein / bead ratio to work with is 7 - 9 * 10-4.
Figure 8: Degree of clearance of the fluorescence in the supernatant plotted against the concentration of silicium dioxide beads used to immobilize <partinfo>K525405</partinfo> (n = 3). Data is fitted with dose-reponse function (R² = 0.997).
Retrieved from " http://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer/SbpA "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
