"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-Germany/Results/BPA
From 2011.igem.org
Home
Team
Students
Supervisors
Photos
Bielefeld University
Acknowledgements
Contact
Project
Description
Achievements
Approach
Background
Bisphenol A
S-layer
NAD+ detection
Technology
DIY Nanobiotech Guide
Future Applications
Labjournal
Protocols
Overview
Molecular Genetics
Production
Analytics
Materials
Results
Data Page
Summary
Bisphenol A
S-layer
NAD+ detection
Since Regionals
Model
Human Practices
Overview
Summary
Cell-free Biosafety
Outreach
Open Lab Course
Science Cafe
High School
CeBiTec Symposium
Student Convention
Strategic Process
Beacon Box
Press
Safety
Partner
Contents
1 Sequencing results
2 Bisphenol A degradation with E. coli
2.1 Specificity of bisphenol A degradation with E. coli
2.2 BPA degradation with constructs containing FNR
2.3 Comparison of all constructs used for BPA degradation
3 Modeling of intracellular bisphenol A degradation
3.1 Regular model
3.2 Model for the fusion protein between FNR, BisdA and BisdB
3.3 Comparison between modeled and measured data
4 Interpretation of the results
5 Summary of results
6 References
Sequencing results
The iGEM team from the University of Alberta sent in BioBricks for BPA degradation in 2008 (<partinfo>K123000</partinfo> and <partinfo>K123001</partinfo>). Sequencing of these BioBricks by the iGEM HQ and by us led to negative results so the sequences entered into the partsregistry are not correct although these sequences are the ones from Sphingomonas bisphenolicum AO1 (compare genbank entry ). In addition, there are "illegal" AgeI and NgoMIV restriction sites in the BioBrick which are used for Freiburg BioBrick assembly standard (RFC 25). After translating and comparing the original sequences from the partsregistry and the sequences from our sequencing results in silico, we saw that the amino acid sequences were identical. These BioBricks were probably synthesized in the Freiburg assembly standard 25 because they have the accordant restriction sites and they were codon optimized for Escherichia coli. But the original sequence from S. bisphenolicum AO1 was entered into the registry because amino acid sequence of the real sequence and the sequence that was entered are identical. The alignments are shown in Figures 1 - 4.
Figure 1: Alignment of the DNA sequence of <partinfo>K123000</partinfo> entered into the partsregistry (K123000 registry) with our sequencing results (K123000 real) for this BioBrick (made with Clonemanager).
Figure 2: Alignment of the aminoacid sequence (translated in silico) of <partinfo>K123000</partinfo> entered into the partsregistry (K123000 registry) with our sequencing results (K123000 real) for this BioBrick (made with Clonemanager).
Figure 3: Alignment of the DNA sequence of <partinfo>K123001</partinfo> entered into the partsregistry (K123001 registry) with our sequencing results (K123001 real) for this BioBrick (made with Clonemanager).
Figure 4: Alignment of the aminoacid sequence (translated in silico) of <partinfo>K123001</partinfo> entered into the partsregistry (K123001 registry) with our sequencing results (K123001 real) for this BioBrick (made with Clonemanager).
Bisphenol A degradation with E. coli
The bisphenol A degradation with the BioBricks <partinfo>K123000</partinfo> and <partinfo>K123001</partinfo> works in E. coli KRX in general. Because Sasaki et al. (2008) reported problems with protein folding in E. coli which seem to inhibit a complete BPA degradation, we did not cultivate at 37 °C and we did not use the strong T7 promoter as Sasaki et al. (2008) did for expressing these BioBricks. Instead, we cultivated at 30 °C and we used a medium strong constitutive promoter (<partinfo>J23110</partinfo>). Incidentally, 30 °C is announced as the cultivation temperature of S. bisphenolicum AO1. With the chosen promoter upstream of a polycistronic bisdAB gene, we were able to completely degrade 120 mg L-1 BPA in about 30 - 33 h. By fusing <partinfo>K123000</partinfo> and <partinfo>K123001</partinfo> (using Freiburg BioBrick assembly standard) we could improve the BPA degradation of E. coli even further so that 120 mg L-1 BPA can be degraded in 21 - 24 h. This data is shown in the following figure:
Figure 5: BPA degradation by E. coli KRX carrying genes for BisdA and BisdB (only bisdA (black), polycistronic bisdAB (red) and fusion protein between BisdA and BisdB (green)) behind the medium strong constitutive promoter <partinfo>J23110</partinfo> with RBS <partinfo>B0034</partinfo>. Cultivations were carried out at 30 °C in LB + Amp + BPA medium for 24 h and 36 h, respectively, with automatic sampling every three hours in 300 mL shaking flasks without baffles with silicon plugs. At least three biological replicates were analysed (three for bisdA alone, seven for bisdAB polycistronic and five for the fusion protein).
Additionally, we carried out these cultivations at different temperatures and BPA concentrations, but the chosen conditions (30 °C and 120 mg L-1 BPA) seemed to be the best. Higher BPA concentrations had an effect on the growth of E. coli and higher temperature led to a worse BPA degradation (probably due to misfolding of the enzymes). Lower temperature also led to less BPA degradation (probably due to slower growth, expression and reaction rate at lower temperatures). The data dealing with the effect of the temperature on the BPA degradation is shown in Figure 6.
Figure 6: BPA degradation by E. coli KRX carrying genes for BisdA and BisdB (polycistronic bisdAB (black) and fusion protein between BisdA and BisdB (striped)) behind the medium strong constitutive promoter <partinfo>J23110</partinfo> with RBS <partinfo>B0034</partinfo>. Cultivations were carried out at different temperatures in LB + Amp + BPA medium (starting concentration 120 mg L-1 BPA) for 24 h in 300 mL shaking flasks without baffles with silicon plugs. Samples were taken at the end of the cultivation. Three biological replicates were analysed (n = 3).
As shown by Sasaki et al. (2008) , BisdB expressed in E. coli leads to almost no BPA degradation. In our experiments, we could not detect the BPA degradation products 1,2-Bis(4-hydroxyphenyl)-2-propanol and 2,2-Bis(4-hydroxyphenyl)-1-propanol in cultivations with E. coli expressing <partinfo>K123000</partinfo> or <partinfo>K123001</partinfo> alone (neither via UV- nor MS-detection). The BPA degradation products 1,2-Bis(4-hydroxyphenyl)-2-propanol and 2,2-Bis(4-hydroxyphenyl)-1-propanol were identified via MS-MS (m/z: 243 / 225 / 211 / 135) and only occured in cultivations with E. coli expressing BisdA and BisdB simultaneously. Sasaki et al. (2005) reported the same MS-MS results for 1,2-Bis(4-hydroxyphenyl)-2-propanol and 2,2-Bis(4-hydroxyphenyl)-1-propanol when degrading BPA with S. bisphenolicum AO1 as we observed in our BPA degradation experiments.
We could also identify the BPA degradation products when working with E. coli TOP10 and MACH1 (data not shown). But because we aim to fuse BisdA and BisdB to S-layer proteins which are supposed to be expressed in E. coli KRX, we carried out the characterizations of <partinfo>K123000</partinfo> and <partinfo>K123001</partinfo> in this strain.
Specificity of bisphenol A degradation with E. coli
In order to access the specificity of the bisphenol A degradation by the bisdA | bisdB fusion protein we tested how well two similar bisphenols, bisphenol F (BPF) and bisphenol S (BPS), were utilized. The structure of those bisphenols differs only in the chemical group linking the two phenols from that of bisphenol A (see Figure 7).
Figure 7: Chemical structure of BPA, BPF and BPS showing the different chemical groups linking the two phenols.
BPF and BPS are used in a broad range of applications that involve the use of polycarbonates or epoxy resins and thus can often be found were BPA is also present. Accordingly, their presence is a potential disruptive factor that could lead to a false positive signal with our biosensor. This is especially true for BPS that in some cases is used as a substitute for BPA in baby bottles [1] . Studies concerning the environmental pollution with BPF ( Fromme et al. (2002) ) and the acute toxicity, mutagenicity and estrogenicity of BPF and BPS ( Chen et al. (2001) and Kitamura et al. (2005) ) indicate their potential harmfulness but further research is needed to fully assess their impact on human health.
E. coli KRX carrying genes for the bisdA | bisdB fusion protein behind the medium strong constitutive promoter <partinfo>J23110</partinfo> with RBS <partinfo>B0034</partinfo> was cultivated at 30 °C for 36 h with LB-Medium containing 120 mg L-1 BPA, BPF respectively BPS. The BPF and BPS concentration were determined with a HPLC using the same method as with BPA. Figure 8 shows the degradation of the respective bisphenol after 24 h of cultivation in percent.
Figure 8: Degradation of BPA, BPF and BPS after 24 h cultivation with E.coli KRX carrying genes for the bisdA | bisdB fusion protein behind the medium strong constitutive promoter <partinfo>J23110</partinfo> with RBS <partinfo>B0034</partinfo>. Cultivations were carried out at different temperatures in LB + Amp + bisphenol medium (starting concentration 120 mg L-1 BPA, BPF or BPS respectively) for 24 h in 300 mL shaking flasks without baffles with silicon plugs. Samples were taken every 3 hours. Two biological replicates were analyzed (n=2).  While BPA is fully degraded only a small fraction of BPF (~7%) and BPS (~3%) was degraded.
The results of the experiment indicate that the bisdA | bisdB fusion protein has a high specificity for the degradation of BPA. In addition it is possible that the decrease in BPF and BPS concentration is due to internalization of those bisphenols or a endogenous enzyme of E. coli KRX and that the bisdA | bisdB fusion protein was not responsible. It can be assumed that false positive signals because of BPF or BPS present in a sample are unlikely.
BPA degradation with constructs containing FNR
Constructs containing FNR, BisdA and BisdB polycistronic <partinfo>K525551</partinfo>, FNR and a fusion protein of BisdA and BisdB <partinfo>K525582</partinfo> and a fusion protein of FNR, BisdA and BisdB <partinfo>K525560</partinfo> were tested for their ability to degrade BPA. Cultivations were done using the same conditions as with previous constructs. The results are shown in Figure 9 below.
Figure 9: Degradation of BPA after 24 h cultivation with E.coli KRX carrying genes for FNR, BisdA and BisdB polycistronic <partinfo>K525551</partinfo>, FNR and a fusion protein of BisdA and BisdB <partinfo>K525582</partinfo> and a fusion protein of FNR, BisdA and BisdB <partinfo>K525560</partinfo> behind the medium strong constitutive promoter <partinfo>J23110</partinfo> with RBS <partinfo>B0034</partinfo>. Cultivations were carried out at 30 °C temperatures in LB + Amp + 120 mg L-1 BPA for 24 h in 300 mL shaking flasks without baffles with silicon plugs. Samples were taken every 3 hours. Two to four biological replicates were analyzed. All constructs were able to degrade BPA, with the constructs containing fusion proteins outperforming the construct where all proteins were expressed polycistronic.
The results indicate that the BPA degradation was improved using fusion proteins compared to the completely polycistronic construct. Also the fusion protein of all three enzymes (<partinfo>K525560</partinfo>) involved in the degradation of BPA worked as intended which is quite impressive.
Comparison of all constructs used for BPA degradation
Figure 10 gives an overview of the constructs employed for the degradation of BPA.
Figure 10: Schematic depiction of all constructs used for BPA degradation: BBa_K525511 , <partinfo>K525515</partinfo>, <partinfo>K525551</partinfo>, BBa_K525580 and <partinfo>K525560</partinfo>. Constructs are divided into those without FNR (left) and with FNR (right).
All constructs were cultivated under the same conditions as described under Cultivations . BPA concentrations were measured using HPLC . Figure 11 shows the percentage of BPA degraded in 24 h of cultivation and the specific BPA degradation rate calculated with the regular model and in the case of <partinfo>K525560</partinfo> with the model for the fusion protein between FNR, BisdA and BisdB .
Figure 11: Results of the BPA degradation experiments with BBa_K525511 , <partinfo>K525515</partinfo>, <partinfo>K525551</partinfo>, BBa_K525580 and <partinfo>K525560</partinfo> and the corresponding specific BPA degradation rates calculated with the model. The construct consisting only of BisdA shows no degradation activity as confirmed by LC-MS (the bar in the chart is within the margin of measurement error), which was expected since BisdA alone can't facilitate the degradation of BPA. In comparison the fusion protein of BisdA and BisdB degraded BPA considerably more efficent than the polycistronic construct, indicating that the fusing of the two proteins improved its activity. Regarding the constructs containing FNR the completely polycistronic construct lags distinctively behind the other two constructs. Again, the fusion of the BisdA and BisdB proteins improved the BPA degradation activity. Using the fusion protein between FNR, BisdA and BisdB we measured the highest maximal specific BPA degradation rate of all constructs. Allthough it has to be taken into account, that a different model had to be used for this particular construct.
We are especially impressed that the fusion protein of FNR, BisdA and BisdB (<partinfo>K525560</partinfo>) not only was capable to degrade BPA but also was the fastest construct we employed. This also contributes to the feasibility of our approach for a cell free BPA biosensor since in a cell free environment fusion proteins are beneficial when compared to their non-fused counterparts.
Modeling of intracellular bisphenol A degradation
Regular model
The modeling was done with the software Berkeley Madonna using the common fourth-order Runge-Kutta method to solve the equations. The model was fitted to the measured data shown above by the function "curve fit" in Berkeley Madonna to calculate the parameters, constants, etc..
To model the BPA degradation by E. coli carrying BioBricks for BPA degradation (<partinfo>K123000</partinfo> and <partinfo>K123001</partinfo>), first the cell growth has to be described to calculate a specific BPA degradation rate per cell. The observed growth of E. coli on (our) LB medium was diauxic with two different growth phases. Cell growth is a first-order reaction and is mathematically described as
(1)
with the specific growth rate µ and the cell count X. The specific growth rate is dependent on the concentration of the growth limiting substrate (e.g. glucose) and can be described as
(2)
with the substrate concentration S, the Monod constant KS and the maximal specific growth rate µmax ( Monod, 1949 ). Because LB medium is a complex medium we cannot measure the substrate concentration so we have to assume an imaginary substrate concentration. Due to the diauxic growth two different substrates with different Monod constants and consumption rates are necessary to model the cell growth. The amount of a substrate S can be modeled as follows
(3)
with the specific substrate consumption rate per cell qS. The whole model for the diauxic growth of E. coli on LB medium with two not measurable (imaginary) substrates looks like:
(4)
The specific BPA degradation rate per cell qD is modeled with an equation like eq. (3). In the beginning of the cultivations, when E. coli growths on the "good" imaginary substrate S1, no BPA degradation is observed. When this substrate is consumed, the BPA degradation starts. The model for this diauxic behavior is as follows:
(5)
Figures 9 - 12 show a comparison between modeled and measured data for cultivations with BPA degrading E. coli. In Table 1 the parameters for the model are given, obtained by curve fitting the model to the data.
Model for the fusion protein between FNR, BisdA and BisdB
The cultivations and BPA degradation of E. coli KRX carrying a fusion protein consisting of ferredoxin-NADP+ oxidoreductase, ferredoxinbisd and cytochrome P450bisd differ from the cultivations with the other BPA degrading BioBricks. Thus, the model for these cultivations has to be adjusted to this behaviour. First of all, no diauxic growth is observed so the growth can be modeled more easily like
(6)
The BPA degradation starts when the imaginary substrate is depleted, like observed in the other cultivations with BPA degrading BioBricks. But it seems that the BPA degradation is getting slower with longer cultivation time. So it is modeled with a Monod-like term in which the specific BPA degradation rate is dependent from the BPA concentration:
(7)
with the maximal specific BPA degradation rate qD,max and the constant KD.
Figures 13 shows a comparison between modeled and measured data for cultivations with BPA degrading fusion protein E. coli. In Tab. 1 the parameters for the model are given, obtained by curve fitting the model to the data.
Comparison between modeled and measured data
Figure 12: Comparison between modeled (lines) and measured (dots) data for cultivations of E. coli KRX carrying BPA degrading BioBrick <partinfo>K525512</partinfo>. The BioBrick <partinfo>K525512</partinfo> (polycistronic bisdAB genes behind medium strong promoter) was cultivated seven times in E. coli KRX in LB + Amp + BPA medium at 30 °C, using 300 mL shaking flasks without baffles with silicon plugs. The BPA concentration (closed dots) and the cell density (open dots) is plotted against the cultivation time.
Figure 13: Comparison between modeled (lines) and measured (dots) data for cultivations of E. coli KRX carrying BPA degrading BioBrick <partinfo>K525517</partinfo>. The BioBrick <partinfo>K525517</partinfo> (fusion protein between BisdA and BisdB behind medium strong promoter) was cultivated five times in E. coli KRX in LB + Amp + BPA medium at 30 °C, using 300 mL shaking flasks without baffles with silicon plugs. The BPA concentration (closed dots) and the cell density (open dots) is plotted against the cultivation time.
Figure 14: Comparison between modeled (lines) and measured (dots) data for cultivations of E. coli KRX carrying BPA degrading BioBrick <partinfo>K525552</partinfo>. The BioBrick <partinfo>K525552</partinfo> (polycistronic fnr : bisdA : bisdB genes behind medium strong promoter) was cultivated four times in E. coli KRX in LB + Amp + BPA medium at 30 °C, using 300 mL shaking flasks without baffles with silicon plugs. The BPA concentration (closed dots) and the cell density (open dots) is plotted against the cultivation time.
Figure 15: Comparison between modeled (lines) and measured (dots) data for cultivations of E. coli KRX carrying BPA degrading BioBrick <partinfo>K525582</partinfo>. The BioBrick <partinfo>K525582</partinfo> (polycistronic fnr : BisdAB (fusion protein) genes behind medium strong promoter) was cultivated four times in E. coli KRX in LB + Amp + BPA medium at 30 °C, using 300 mL shaking flasks without baffles with silicon plugs. The BPA concentration (closed dots) and the cell density (open dots) is plotted against the cultivation time.
Figure 16: Comparison between modeled (lines) and measured (dots) data for cultivations of E. coli KRX carrying BPA degrading BioBrick <partinfo>K525562</partinfo>. The BioBrick <partinfo>K525562</partinfo> (fusion protein between FNR, BisdA and BisdB behind medium strong promoter) was cultivated four times in E. coli KRX in LB + Amp + BPA medium at 30 °C, using 300 mL shaking flasks without baffles with silicon plugs. The BPA concentration (closed dots) and the cell density (open dots) is plotted against the cultivation time.
Tab. 1: Parameters of the model.
Parameter
<partinfo>K525512</partinfo>
<partinfo>K525517</partinfo>
<partinfo>K525552</partinfo>
<partinfo>K525562</partinfo>
<partinfo>K525582</partinfo>
X0
0.112 108 mL-1
0.138 108 mL-1
0.109 108 mL-1
0.115 108 mL-1
0.139 108 mL-1
µmax
1.253 h-1
1.357 h-1
0.963 h-1
1.730 h-1
0.858 h-1
KS,1
2.646 AU-1
1.92 AU-1
5.35 AU-1
13.87 AU-1
3.05 AU-1
KS,2
265.1 AU-1
103.1 AU-1
82.6 AU-1
-
32.5 AU-1
S1,0
1.688 AU
1.166 AU
4.679 AU
3.003 AU
2.838 AU
qS,1
0.478 AU 10-8 cell-1
0.319 AU 10-8 cell-1
0.883 AU 10-8 cell-1
0.240 AU 10-8 cell-1
0.544 AU 10-8 cell-1
S2,0
16.091 AU
6.574 AU
3.873 AU
-
2.402 AU
qS,2
0.295 AU 10-8 cell-1
0.191 AU 10-8 cell-1
0.082 AU 10-8 cell-1
-
0.056 AU 10-8 cell-1
BPA0
0.53 mM
0.53 mM
0.41 mM
0.45 mM
0.53 mM
qD
8.76 10-11 mM cell-1
1.29 10-10 mM cell-1
5.67 10-11 mM cell-1
-
1.13 10-10 mM cell-1
qD,max
-
-
-
1.32 10-10 mM cell-1
-
KD
-
-
-
0.121 mM-1
-
The specific BPA degradation rate per cell qD is about 50 % higher when using the fusion protein compared to the polycistronic bisdAB gene. On average, this results in a 9 hours faster, complete BPA degradation by E. coli carrying <partinfo>K525517</partinfo> compared to <partinfo>K525512</partinfo> as observed during our cultivations . The fusion protein between BisdA and BisdB improves the BPA degradation by E. coli. Introducing a polycistronic ferredoxin-NADP+ reductase gene into these systems does not lead to a higher specific BPA degradation rate. The rates are a bit lower, though. The effect that the BisdA | BisdB fusion protein is more efficient than BisdB when expressed polycistronically with BisdA can still be observed in this setup. The cultivations with the expression of the fusion protein FNR | BisdA | BisdB differ from the expression of the other BPA degrading BioBricks. The BPA degradation rate is concentration dependent which is typical for enzymatic reactions but was not observed in the other cultivations. In addition, the growth was faster and not diauxic. The maximal specific BPA degradation rate of this BioBrick is higher than the observed specific BPA degradation rates in the other cultivations. But due to the dependence of the BPA degradation on the BPA concentration, the BPA degradation with this BioBrick is not as efficient and not complete. Anyway, the fact that this BioBrick is working is impressive.
Interpretation of the results
Misfolding seems to be a problem when expressing BisdA and BisdB in E. coli. To reduce this, the cultivation conditions were improved for the BPA degradation with the polycistronic bisdAB gene in E. coli first, compared to the literature ( Sasaki et al., 2008 ). Problems with misfolding of BisdA and BisdB could be reduced by lowering the temperature as well as growth rate and by using a weaker promoter for expression.
When degrading BPA with E. coli using the BisdA | BisdB fusion protein, both domains (BisdA and BisdB) are active and correctly folded because otherwise there would be no BPA degradation measured and no BPA degradation products 1,2-Bis(4-hydroxyphenyl)-2-propanol and 2,2-Bis(4-hydroxyphenyl)-1-propanol would be detectable, which could definetely be identified via MS-MS. The about 50 % higher specific BPA degradation rate in E. coli expressing the BisdA | BisdB fusion protein could be explained either by improved folding properties of the fusion protein or by the closer distance of BisdA and BisdB in the fusion protein leading to a faster electron transfer and therefore a more efficient reaction. In this case, which has to be further analysed by cell-free enzyme assays, a natural cytochrome P450 class I electron transport system was converted into a more effective class V electron transport system, demonstrating the new possibilities of synthetic biology. In addition, this new class V system would work without alanine-rich linker, potentially changing the view on cytochrome P450 depending electron transport chains.
Summary of results
Tab. 2: Important parameters of <partinfo>K525512</partinfo>, <partinfo>K525517</partinfo>, <partinfo>K525552</partinfo>, <partinfo>K525562</partinfo> and <partinfo>K525582</partinfo>.
Experiment
Characteristic
Result <partinfo>K525517</partinfo>
Result <partinfo>K525512</partinfo>
Result <partinfo>K525552</partinfo>
Result <partinfo>K525562</partinfo>
Result <partinfo>K525582</partinfo>
Expression in E. coli
Compatibility
E. coli KRX, TOP10, MACH1, BL21(DE3)
Expression
Constitutive
Optimal temperature
30 °C
BPA working concentration
120 mg L-1 (0.53 mM)
Purification
Molecular weight
59.3 kDa
11.2 and 48.3 kDa
28.0, 11.2 and 48.3 kDa
87.1 kDa
28.0 and 59.3 kDa
Theoretical pI
4.99
4.31 and 5.27
6.17, 4.31 and 5.27
5.26
6.17 and 4.99
High absorbtion
450 nm (due to CYP)
Degradation of BPA
Completely degradation of 0.53 mM BPA
21 - 24 h
30 - 33 h
> 36 h
> 36 h
30 - 33 h
Specific BPA degradation rate
1.29 10-10 mM cell-1
8.76 10-11 mM cell-1
5.67 10-11 mM cell-1
1.32 10-10 mM cell-1 *
1.13 10-10 mM cell-1
* Different model: the maximal specific BPA degradation rate was calculated.
References
Monod J (1949) The growth of bacterial cultures, Annu Rev Microbiol 3:371-394 .
Sasaki M, Maki J, Oshiman K, Matsumura Y, Tsuchido T (2005) Biodegradation of bisphenol A by cells and cell lysate from Sphingomonas sp. strain AO1, Biodegradation 16(5):449-459 .
Sasaki M, Tsuchido T, Matsumura Y (2008) Molecular cloning and characterization of cytochrome P450 and ferredoxin genes involved in bisphenol A degradation in Sphingomonas bisphenolicum strain AO1, J Appl Microbiol 105(4):1158-1169 .
Retrieved from " http://2011.igem.org/Team:Bielefeld-Germany/Results/BPA "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
