<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Theory"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Theory">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Theory&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Theory&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Theory&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Theory" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Theory</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu-igem"><UL id="menu" class="sf-menu"><LI><A style="border-left:0px;" href="https://2011.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Students">Students</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Advisors">Supervisors</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Photos">Photos</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Bielefeld-University‎‎">Bielefeld University</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></UL><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Description">Description</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Achievements">Achievements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Appoach">Approach</A></LI><LI><A href="#">Background</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/Technology">Technology</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer/Guide">DIY Nanobiotech Guide</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Future">Future Applications</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Labjournal">Labjournal</A></LI><LI><A href="#">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Genetics">Molecular Genetics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing">Production</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Analytics">Analytics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials">Materials</A></LI><LI><A href="#">Results</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Data_Page">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/SinceAmsterdam">Since Regionals</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Modell">Model</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Cell-free">Cell-free Biosafety</A></LI><LI><A href="#">Outreach</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_lab_course">Open Lab Course</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_science_cafe">Science Cafe</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/high-school">High School</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/cebitec-symposium">CeBiTec Symposium</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/student-convention">Student Convention</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/strategic-process">Strategic Process</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Black-box">Beacon Box</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Press">Press</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Bisphenol_A"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Bisphenol A</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Bisphenol_A_and_its_effects_on_mammals"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Bisphenol A and its effects on mammals</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Bisphenol_A_degradation"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Bisphenol A degradation</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Further_applications_of_bisphenol_A_degrading_BioBricks_and_enzymes"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Further applications of bisphenol A degrading BioBricks and enzymes</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#S-layer"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">S-layer</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Further_applications_of_S-layer_proteins"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Further applications of S-layer proteins</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#NAD.2B_detection"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">NAD<SUP>+</SUP> detection</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#NAD.2B_in_general"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">NAD<SUP>+</SUP> in general</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Biosynthesis_of_NAD.2B"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Biosynthesis of NAD<SUP>+</SUP></SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Bacterial_DNA_ligases"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Bacterial DNA ligases</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Applications_for_molecular_beacons"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Applications for molecular beacons</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Further_applications_for_the_NAD.2B_bioassay"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Further applications for the NAD<SUP>+</SUP> bioassay</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Bisphenol_A">Bisphenol A</SPAN></H1><P>monomer, plastic, polycarbonate, epoxy resins, food containers, forbidden in EU and canada in baby flasks etc.
</P><H2><SPAN class="mw-headline" id="Bisphenol_A_and_its_effects_on_mammals">Bisphenol A and its effects on mammals</SPAN></H2><P>endocrine disruptor, estrogenic, teratogenic, infertility etc. 
</P><H2><SPAN class="mw-headline" id="Bisphenol_A_degradation">Bisphenol A degradation</SPAN></H2><P>There exist a lot of bacteria in nature that can degrade xenobiotic substances such as phenolic compounds or endocrine disruptors. In a lot of soil samples that were taken from contaminated soil to find organisms that do so, <I>sphingomonads</I> were extraordinarily often isolated (<A href="http://www.springerlink.com/content/t717l15u85507706/" class="external text" rel="nofollow">Stolz, 2009</A>). In 2005, <A href="http://www.springerlink.com/content/q7864l02734wg32m/" class="external text" rel="nofollow">Sasaki <I>et al.</I></A> isolated a soil bacterium from the <I>Sphingomonas</I> genus which is able to degrade the endocrine disruptor bisphenol A (BPA) with a unique rate and efficiency compared to other BPA degrading organisms. This strain, called <I>Sphingomonas bisphenolicum</I> AO1, is able to completely decompose 120 mg BPA L<SUP>-1</SUP> in about 6 hours while other strains need days of cultivation (e.g. <I>Sphingomonas</I> strain BP-7 isolated by <A href="http://www.jstage.jst.go.jp/article/bbb/71/1/71_51/_article" class="external text" rel="nofollow">Sakai <I>et al.</I> (2007)</A>). 
</P><P>The full bisphenol A degradation pathway which is found in nature is shown in figure X (taken from KEGG database). BPA is metabolized to 4-Hydroxybenzaldehyde, 4'-Hydroxyacetophenone and 4-Hydroxybenzoate which can be used by some bacteria as carbon source. These and other metabolites of the BPA degradation pathway can be found in the supernatant of BPA containing cultivations of <I>S. bisphenolicum</I> AO1. They can grow on BPA as the only carbon source, too (<A href="http://www.springerlink.com/content/q7864l02734wg32m/" class="external text" rel="nofollow">Sasaki <I>et al.</I>, 2005a</A>).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Fig. X: blabla</B></DIV></DIV></DIV></DIV><P>Bisphenol A is mainly hydroxylated into the products 1,2-Bis(4-hydroxyphenyl)-2-propanol and 2,2-Bis(4-hydroxyphenyl)-1-propanol by some kind of oxidoreductase acting with NADH or NADPH. A total of three genes are responsible for this BPA hydroxylation: a cytochrome P450 (CYP, <I>bisdB</I>), a ferredoxin (Fd, <I>bisdA</I>) and a ferredoxin-NAD<SUP>+</SUP> oxidoreductase (FNR) (<A href="http://aem.asm.org/cgi/content/abstract/71/12/8024" class="external text" rel="nofollow">Sasaki <I>et al.</I>, 2005b</A>). The three gene products act together to reduce BPA while oxidizing NADH + H<SUP>+</SUP>. The cytochrome P450 (BisdB) reduces the BPA and is oxidized during this reaction. BisdB in its oxidized status is reduced by the ferredoxin (BisdA) so it can reduce BPA again. The oxidized BisdA is reduced by a ferredoxin-NAD<SUP>+</SUP> oxidoreductase consuming NADH + H<SUP>+</SUP> so the BPA degradation can continue (<A href="http://aem.asm.org/cgi/content/abstract/71/12/8024" class="external text" rel="nofollow">Sasaki <I>et al.</I>, 2005b</A>). The <I>bisdAB</I> genes from <I>S. bisphenolicum</I> AO1 were isolated, transformed into and expressed in <I>E. coli</I> and enabled this bacterium to degrade BPA, too (<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.2008.03843.x/full" class="external text" rel="nofollow">Sasaki <I>et al.</I>, 2008</A>). In addition, the BisdAB proteins from <I>S. bisphenolicum</I> AO1 were able to degrade BPA in a cell-free system (enzyme assay) in which spinach reductase (<A href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC1/18/1/2.html" class="external text" rel="nofollow">EC 1.18.1.2</A>) was added (<A href="http://aem.asm.org/cgi/content/abstract/71/12/8024" class="external text" rel="nofollow">Sasaki <I>et al.</I>, 2005b</A>). 
</P><P>In 2008, the iGEM team from the <A href="https://2008.igem.org/Team:The_University_of_Alberta/Parts" class="external text" rel="nofollow">University of Alberta</A> submitted the <I>bisdAB</I> genes from <I>S. bisphenolicum</I> AO1 to the <A href="http://partsregistry.org/Main_Page" class="external text" rel="nofollow">registry of standard biological parts</A> in the so called <A href="http://partsregistry.org/Assembly_standard_25" class="external text" rel="nofollow">Freiburg BioBrick assembly standard</A> (&lt;partinfo&gt;K123000&lt;/partinfo&gt; and &lt;partinfo&gt;K123001&lt;/partinfo&gt;). In order to degrade BPA in a cell free system, a FNR BioBrick is also needed. As demonstrated, the BisdA and BisdB work together in a cell-free system with spinach reductase and they work intracellular in <I>E. coli</I>, it can be assumed that it is possible to produce all components necessary to degrade BPA in a cell free system in <I>E. coli</I>. To achieve this a ferredoxin-NADP<SUP>+</SUP> oxidoreductase BioBrick, isolated from <I>E. coli</I>, is provided by the Bielefeld-Germany 2011 iGEM team (&lt;partinfo&gt;K525499&lt;/partinfo&gt;). 
</P><P>The whole electron transport chain between the three enzymes involved in BPA degradation and the BioBricks needed to enable this reaction <I>in vitro</I> are shown in the following figure (please have some patience, it's an animated .gif file): 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><A href="/File:Bielefeld-Germany2011-BPAdegrad2.gif" title="File:Bielefeld-Germany2011-BPAdegrad2.gif">File:Bielefeld-Germany2011-BPAdegrad2.gif</A><DIV class="thumbcaption"><B>Fig. X: blabla</B></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Further_applications_of_bisphenol_A_degrading_BioBricks_and_enzymes">Further applications of bisphenol A degrading BioBricks and enzymes</SPAN></H2><H1><SPAN class="mw-headline" id="S-layer">S-layer</SPAN></H1><H2><SPAN class="mw-headline" id="Further_applications_of_S-layer_proteins">Further applications of S-layer proteins</SPAN></H2><H1><SPAN class="mw-headline" id="NAD.2B_detection">NAD<SUP>+</SUP> detection</SPAN></H1><H2><SPAN class="mw-headline" id="NAD.2B_in_general">NAD<SUP>+</SUP> in general</SPAN></H2><P>Nicotinamide adenine dinucleotide, abbreviated NAD<SUP>+</SUP>, is an ubiquitous and indispensable cofactor for all living organisms (<A href="http://onlinelibrary.wiley.com/doi/10.1002/jcp.22419/full" class="external text" rel="nofollow">Bi <I>et al.</I>, 2011</A>). The molecule has a wide range of physiological functions in cellular processes (fig. 2) and has a great impact on the cell integrity (<A href="http://ini.sagepub.com/content/17/2/212.abstract" class="external text" rel="nofollow">Grahnert <I>et al.</I>, 2011</A>). It is integrated into the energy metabolism and determines the redox status of the cell due to its purpose as a reduction equivalent. This is strongly linked to the control of signaling and transcriptional events. For instance, NAD<SUP>+</SUP> has been recently described as a modulator of immune functions (<A href="http://ini.sagepub.com/content/17/2/212.abstract" class="external text" rel="nofollow">Grahnert <I>et al.</I>, 2011</A>) and a master regulator of transcription (<A href="http://www.sciencedirect.com/science/article/pii/S1874939910001057" class="external text" rel="nofollow">Ghosh <I>et al.</I>, 2010</A>). Several enzymes are tightly regulated by the cellular balance between the oxidized and reduced form of NAD<SUP>+</SUP> and therefore regarded as a potential target for therapeutics (<A href="http://jpet.aspetjournals.org/content/324/3/883.abstract" class="external text" rel="nofollow">Sauve, 2008</A>). Indeed, NAD<SUP>+</SUP> acts as a substrate for a wide range of proteins including NAD<SUP>+</SUP>-dependent protein deacetylases, poly(ADP-ribose) polymerases and several transcription factors (<A href="http://edrv.endojournals.org/content/31/2/194.abstract" class="external text" rel="nofollow">Houtcooper <I>et al.</I>, 2010</A>). A disorder of the cellular NAD<SUP>+</SUP> level has usually relevance in symptoms and diseases like stroke, cardiac ischemia, epilepsy, huntington disease, wallerian degeneration, cancer, type two diabetes mellitus (T2DM), neutrophil survival, longevity, obesity as well as cardiovascular and neurodegenerative disease (<A href="http://jpet.aspetjournals.org/content/324/3/883.abstract" class="external text" rel="nofollow">Sauve, 2008</A>; <A href="http://edrv.endojournals.org/content/31/2/194.abstract" class="external text" rel="nofollow">Houtcooper <I>et al.</I>, 2010</A>). The regulation of NAD<SUP>+</SUP>-dependent pathways may have a major contribution to oxidative metabolism and life span extension (<A href="http://edrv.endojournals.org/content/31/2/194.abstract" class="external text" rel="nofollow">Houtcooper <I>et al.</I>, 2010</A>). The biosynthetic pathways for NAD<SUP>+</SUP> are proposed as promising novel antibiotics targets against pathogens (<A href="http://onlinelibrary.wiley.com/doi/10.1002/jcp.22419/full" class="external text" rel="nofollow">Bi <I>et al.</I>, 2011</A>).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Figure 2: NAD(P) physiological function (taken from <A href="http://onlinelibrary.wiley.com/doi/10.1002/jcp.22419/full" class="external text" rel="nofollow">Bi <I>et al.</I>, 2011</A>).</B></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Biosynthesis_of_NAD.2B">Biosynthesis of NAD<SUP>+</SUP></SPAN></H2><P>NAD<SUP>+</SUP> is embedded in the cellular metabolism and has a great impact on essential cellular processes. The biosynthesis in most bacteria and plants starts with the anaerobic conversion of the amino acid aspartate to quinoline. Alternatively, in yeast, humans and some bacterial microbes the quinoline derives from tryptophan in an aerobic manner (fig 3). This is a crucial step that leads under consumption of nicotinic acid to the nicotinate ring system via formation of nicotinic acid monoculeotide (NAMN), respectively (<A href="http://jpet.aspetjournals.org/content/324/3/883.abstract" class="external text" rel="nofollow">Sauve, 2008</A>). After conjugation of adenosine mononucleotide (AMP) the NAD synthetase forms nicotinamide adenine dinucleotide (NAD). Additionally, it has been recently reported that in humans exists an nicotinic acid-independent reaction catalysed by the enzyme nampt/PBEF, which uses directly nicotinamide as a substrate (<A href="http://www.ncbi.nlm.nih.gov/pubmed/12555668" class="external text" rel="nofollow">Rongvaux, 2002</A>). Diet or pharmalogical agents can thereby serve as a source for nicotinamide or nicotinic acid (<A href="http://jpet.aspetjournals.org/content/324/3/883.abstract" class="external text" rel="nofollow">Sauve, 2008</A>). Fundamental processes as the cellular energy metabolism determine the balance between the oxidized (NAD<SUP>+</SUP>) and reduced (NADH) form of NAD.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption"><B>Figure 3: Principle of de novo NAD<SUP>+</SUP> biosynthesis of organisms that use tryptophan as a source. Bottom: paths of metabolism for human nicotinamide, nicotinic acid and nicotinamide riboside that lead to formation of NAD<SUP>+</SUP> (taken from <A href="http://jpet.aspetjournals.org/content/324/3/883.abstract" class="external text" rel="nofollow">Sauve, 2008</A>).</B></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Bacterial_DNA_ligases">Bacterial DNA ligases</SPAN></H2><P>Bacterial DNA ligases are important enzymes for DNA repair and replication. The big difference between bacterial and eukaryotic DNA ligases is their specifity for cofactors. While eukaryotic DNA ligases use ATP as a source for building a phosphodiester bond in the DNA`s phosphate backbone the bacterial ones display a NAD<SUP>+</SUP>-dependent biochemical catalysis. These enzymes are totally essential for bacterial survival and are therefore in focus for designing highly specific antiinfectives (<A href="http://www.cell.com/structure/abstract/S0969-2126%2804%2900235-7" class="external text" rel="nofollow">Gajiwala &amp; Pinko, 2004</A>). All bacterial DNA ligases possess highly conserved functional domains. Generally, they can be devided into four functionally distinguishable domains, each with characteristic motifs: 
</P><UL><LI> Domain 1: Ia, Ib
</LI><LI> Domain 2: Oligomer-binding (OB) beta-barrel
</LI><LI> Domain 3: Zinc-finger, helix-hairpin-helix motif (HhH)
</LI><LI> Domain 4: BRCT domain
</LI></UL><P>All bacterial DNA ligases share three fundamental steps during reaction mechanism:
</P><UL><LI> Adenylation of DNA ligase by transferring hydrolysed AMP from NAD<SUP>+</SUP> to the active site lysine and release of NMN
</LI><LI> Tranfer of the AMP from DNA ligase to the 5'-phosphate group of nicked DNA
</LI><LI> Formation of a phosphodiester bond by attack of the nick 3'-OH on the 5'-phosphoanhydride linkage and release of free AMP
</LI></UL><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:567px;"><DIV class="thumbcaption"><B>Figure 7: During the adenylation reaction cycle of NAD<SUP>+</SUP>-dependent DNA ligases the linker between the two subdomains Ia (gray) and Ib (yellow) undergoes conformational change to form open and closed state. A: Open state of Ia domain exposing the adenylation site to solvent. B: Closed state of Ia domain is achieved by NAD<SUP>+</SUP> binding. C: AMP-bound form providing adenylated site exposed to solvent and an open form accessible to incoming DNA. D: Deadenylated (NMN-bound) form (taken from  <A href="http://www.cell.com/structure/abstract/S0969-2126%2804%2900235-7" class="external text" rel="nofollow">Gajiwala &amp; Pinko, 2004</A>).</B></DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:287px;"><DIV class="thumbcaption"><B>Figure 6: Model of LigA from <I>E. coli</I> bound to DNA. B: The different colours indicate N-terminal domain Ia (blue), nucleotidyltransferase (NTase) domain (cyan), OB-fold domain (magenta), helix-hairpin-helix (HhH) domain (beige) and a zinc (Zn)-finger domain (green). Typically, there is a BRCT domain at the C-terminus (not shown). C: Aspartic acid, arginine and glycine residues in the NTase and HhH domains are responsible for intramolecular interaction promoting protein/DNA interaction within the DNA major groove opposite the adenylated nick, called AppDNA (taken from <A href="http://www.cell.com/molecular-cell/abstract/S1097-2765%2807%2900144-XNandakumar" class="external text" rel="nofollow"><I>et al.</I>, 2007</A>).</B></DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Applications_for_molecular_beacons">Applications for molecular beacons</SPAN></H2><P>Molecular beacons can be applied to a wide range of biological and diagnostical studies. Whenever there is a specific nucleic acid which is supposed to be detected they can be utilized. Molecular beacons are used for the detection and distinction of retroviral DNA from different types of HIV, for instance (<A href="http://www.sciencedirect.com/science/article/pii/S0009898105004249" class="external text" rel="nofollow">Marras <I>et al.</I>, 2006</A>). Even for indentification of proteins which interact with DNA or RNA they are useful. In sum, there are a lot of opportunities to apply molecular beacons (<A href="http://www.jbsdonline.com/c3008/c4017/Molecular-Beacons-Nucleic-Acid-Hybridization-and-Emerging-Applications-p-497-504-p10542.html" class="external text" rel="nofollow">Antony &amp; Subramaniam, 2001</A>):
</P><UL><LI> Detection of PCR amplification products
</LI><LI> Mutational analysis
</LI><LI> Genotyping and allele discrimination
</LI><LI> Clinical diagnosis
</LI><LI> Oligonucleotide degradation studies
</LI><LI> Analysis of DNA/RNA hybridization
</LI><LI> Realtime visualization of hybridization in cells
</LI><LI> Analysis of triplex DNA formation
</LI><LI> Protein/DNA interaction
</LI><LI> Enzymatic assays
</LI><LI> Biosensors
</LI></UL><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 4: Hybridization of the molecular beacon and a complementary target forms the molecular beacon`s open state which leads to an increase of the fluorescence intensity. The molecular beacon can be immmobilized using specific biomolecular interaction, e.g. streptavidin/biotin or covalent cross-linking (taken from <A href="http://www.ijcep.com/708009A.html" class="external text" rel="nofollow">Kim <I>et al.</I>, 2008</A>).</B></DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:313px;"><DIV class="thumbcaption"><B>Figure 5: Single nucleotide polymorphism of nucleic acids, including different alleles as indicated, can be detected by differently designed and labeled molecular beacons (taken from <A href="http://www.ijcep.com/708009A.html" class="external text" rel="nofollow">Kim <I>et al.</I>, 2008</A>).</B></DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:277px;"><DIV class="thumbcaption"><B>Figure 6: Realtime <I>in vivo</I> imaging of specific mRNAs utilizing two MBs that target adjacent positions on the mRNA allowing FRET to take place (taken from <A href="http://www.ijcep.com/708009A.html" class="external text" rel="nofollow">Kim <I>et al.</I>, 2008</A>).</B></DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Further_applications_for_the_NAD.2B_bioassay">Further applications for the NAD<SUP>+</SUP> bioassay</SPAN></H2><P>The reviewed molecular beacon based <A href="http://pubs.acs.org/doi/abs/10.1021/ac102742k" class="external text" rel="nofollow">NAD<SUP>+</SUP> bioassay</A> can be applied to biochemical and biomedical studies (<A href="http://pubs.acs.org/doi/abs/10.1021/ac102742k" class="external text" rel="nofollow">Tang <I>et al.</I>, 2011</A>). Accordingly, it can be utilized to detect NAD<SUP>+</SUP>/NADH-dependent enzymatic processes. The low limit of detection, its reliability and manageability provides a practical alternative to present colorimetric, fluorometric, chemiluminescent, electrochemical or mass spectrometric methods detecting NAD<SUP>+</SUP> or NADH. In context of clinical applications and therapeutics the <A href="http://pubs.acs.org/doi/abs/10.1021/ac102742k" class="external text" rel="nofollow">NAD<SUP>+</SUP> bioassay</A> can be useful to monitor cellular NAD<SUP>+</SUP> levels during treatments of tissues or cell cultures for identification of drug targets, for instance. It may also be applied in the field of diagnostics. Finally, the principle of the proposed <A href="http://pubs.acs.org/doi/abs/10.1021/ac102742k" class="external text" rel="nofollow">NAD<SUP>+</SUP> bioassay</A> can be used for cell-free optical biosensors by taking molecular beacons either immobilized on the sensor surface or being present in the reaction medium (<A href="http://pubs.acs.org/doi/abs/10.1021/ac102742k" class="external text" rel="nofollow">Tang <I>et al.</I>, 2011</A>).
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>Antony T, Subramaniam V (2001) Molecular Beacons: Nucleic Acid Hybridization and Emerging Applications, <A href="http://www.jbsdonline.com/c3008/c4017/Molecular-Beacons-Nucleic-Acid-Hybridization-and-Emerging-Applications-p-497-504-p10542.html" class="external text" rel="nofollow"><I>Journal of Biomolecular Structure &amp; Dynamics</I> 19(3):497-504</A>.
</P><P>Bi J, Wang H, Xiel J (2011) Comparative genomics of NAD(P) biosynthesis and novel antibiotic drug targets, <A href="http://onlinelibrary.wiley.com/doi/10.1002/jcp.22419/full" class="external text" rel="nofollow"><I>Journal of Cellular Physiology</I> 226(2):331-340</A>.
</P><P>Gajiwala KS, Pinko C (2004) Structural Rearrangement Accompanying NAD<SUP>+</SUP> Synthesis within a Bacterial DNA Ligase Crystal, <A href="http://www.cell.com/structure/abstract/S0969-2126%2804%2900235-7" class="external text" rel="nofollow"><I>Structure</I> 12(8):1449-1459</A>.
</P><P>Ghosh S, George S, Roy U, Ramachandran D, Kolthur-Seetharam U (2010) NAD: A master regulator of transcription, <A href="http://www.sciencedirect.com/science/article/pii/S1874939910001057" class="external text" rel="nofollow"><I>Biochimica et Biophysica Acta</I> 1799(10-12):681-693</A>.
</P><P>Grahnert A, Grahnert A, Klein C, Schilling E, Wehrhahn J, Hauschildt S (2011) NAD<SUP>+</SUP>: A modulator of immune functions, <A href="http://ini.sagepub.com/content/17/2/212.abstract" class="external text" rel="nofollow"><I>Innate Immunity</I> 17(2):212-233</A>.
</P><P>Houtkooper RH, Cantó C, Wanders RJ, Auwerx J (2010) The Secret Life of NAD<SUP>+</SUP>: An Old Metabolite Controlling New Metabolic Signaling Pathways, <A href="http://edrv.endojournals.org/content/31/2/194.abstract" class="external text" rel="nofollow"><I>Endocrine Reviews</I> 31(2):194-223</A>.
</P><P>Kim Y, Sohn D, Tan W (2008) Molecular Beacons in Biomedical Detection and Clinical Diagnosis, <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2480550/?tool=pmcentrez&amp;rendertype=abstract" class="external text" rel="nofollow"><I>International Journal of Clinical and Experimental Pathology</I> 1:105-116</A>.
</P><P>Marras SA, Tyagi S, Kramer FR (2006) Real-time assays with molecular beacons and other fluorescent nucleic acid hybridization probes, <A href="http://www.sciencedirect.com/science/article/pii/S0009898105004249" class="external text" rel="nofollow"><I>Clinica chimica acta</I> 363(1-2):48-60</A>.
</P><P>Nandakumar J, Nair PA, Shuman S (2007) Last Stop on the Road to Repair: Structure of <I>E. coli</I> DNA Ligase Bound to Nicked DNA-Adenylate, <A href="http://www.cell.com/molecular-cell/abstract/S1097-2765%2807%2900144-X" class="external text" rel="nofollow"><I>Molecular Cell</I> 26(2):257-271</A>.
</P><P>Rongvaux A, Shea RJ, Mulks MH, Gigot D, Urbain J, Leo O, Andris F (2002) Pre-B-cell colony-enhancing factor, whose expression is up-regulated in activated lymphocytes, is a nicotinamide phosphoribosyltransferase, a cytosolic enzyme involved in NAD biosynthesis, <A href="http://www.ncbi.nlm.nih.gov/pubmed/12555668" class="external text" rel="nofollow"><I>European Journal of Immunology</I> 32(11):3225-3234</A>.
</P><P>Sakai K, Yamanaka H, Moriyoshi K, Ohmoto T, Ohe T (2007) Biodegradation of Bisphenol A and Related Compounds by <I>Sphingomonas sp.</I> Strain BP-7 Isolated from Seawater, <I>Bioscience, Biotechnology, and Biochemistry</I><A href="http://www.jstage.jst.go.jp/article/bbb/71/1/71_51/_article" class="external text" rel="nofollow">71(1):51-57</A>.
</P><P>Sauve AA (2008) NAD<SUP>+</SUP> and Vitamin B3: From Metabolism to Therapies, <A href="http://jpet.aspetjournals.org/content/324/3/883.abstract" class="external text" rel="nofollow"><I>The Journal of Pharmacology and Experimental Therapeutics</I> 324(3):883-893</A>. 
</P><P>Sasaki M, Maki J, Oshiman K, Matsumura Y, Tsuchido T (2005a) Biodegradation of bisphenol A by cells and cell lysate from <I>Sphingomonas</I> sp. strain AO1, <A href="http://www.springerlink.com/content/q7864l02734wg32m/" class="external text" rel="nofollow"><I>Biodegradation</I> 16(5):449-459</A>.
</P><P>Sasaki M, Akahira A, Oshiman K, Tsuchido T, Matsumura Y (2005b) Purification of Cytochrome P450 and Ferredoxin, Involved in Bisphenol A Degradation, from <I>Sphingomonas</I> sp. Strain AO1, <A href="http://aem.asm.org/cgi/content/abstract/71/12/8024" class="external text" rel="nofollow"><I>Appl Environ Microbiol</I> 71(12):8024-8030</A>.
</P><P>Sasaki M, Tsuchido T, Matsumura Y (2008) Molecular cloning and characterization of cytochrome P450 and ferredoxin genes involved in bisphenol A degradation in <I>Sphingomonas bisphenolicum</I> strain AO1, <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.2008.03843.x/full" class="external text" rel="nofollow"><I>J Appl Microbiol</I> 105(4):1158-1169</A>.
</P><P>Stolz A (2009) Molecular characteristics of xenobiotic-degrading sphingomonads,  <A href="http://www.springerlink.com/content/t717l15u85507706/" class="external text" rel="nofollow"><I>Appl Microbiol Biotechnol</I> 81:793-811</A>.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Bielefeld-Germany/Theory">http://2011.igem.org/Team:Bielefeld-Germany/Theory</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Theory" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Theory" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Theory&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Theory&amp;oldid=121404" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>