<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Project_Background_NAD"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Project/Background/NAD">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Project/Background/NAD&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Background/NAD&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Background/NAD&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Project/Background/NAD" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Project/Background/NAD</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu-igem"><UL id="menu" class="sf-menu"><LI><A style="border-left:0px;" href="https://2011.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Students">Students</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Advisors">Supervisors</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Photos">Photos</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Bielefeld-University‎‎">Bielefeld University</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></UL><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Description">Description</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Achievements">Achievements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Appoach">Approach</A></LI><LI><A href="#">Background</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/Technology">Technology</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer/Guide">DIY Nanobiotech Guide</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Future">Future Applications</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Labjournal">Labjournal</A></LI><LI><A href="#">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Genetics">Molecular Genetics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing">Production</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Analytics">Analytics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials">Materials</A></LI><LI><A href="#">Results</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Data_Page">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/SinceAmsterdam">Since Regionals</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Modell">Model</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Cell-free">Cell-free Biosafety</A></LI><LI><A href="#">Outreach</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_lab_course">Open Lab Course</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_science_cafe">Science Cafe</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/high-school">High School</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/cebitec-symposium">CeBiTec Symposium</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/student-convention">Student Convention</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/strategic-process">Strategic Process</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Black-box">Beacon Box</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Press">Press</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#NAD.2B_detection"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">NAD<SUP>+</SUP> detection</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#NAD.2B_in_general"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">NAD<SUP>+</SUP> in general</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Bacterial_DNA_ligases"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Bacterial DNA ligases</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Applications_of_molecular_beacons"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Applications of molecular beacons</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Further_applications_of_the_NAD.2B_bioassay"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Further applications of the NAD<SUP>+</SUP> bioassay</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#References"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="NAD.2B_detection">NAD<SUP>+</SUP> detection</SPAN></H1><H2><SPAN class="mw-headline" id="NAD.2B_in_general">NAD<SUP>+</SUP> in general</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 1: Reduced (right) and oxidized (left) form of NAD.</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 2: NAD as a key metabolite embedded in the cellular metabolism.</B></DIV></DIV></DIV><P>Nicotinamide adenine dinucleotide, abbreviated NAD<SUP>+</SUP>, is an ubiquitous and indispensable cofactor for all living organisms (<A href="http://onlinelibrary.wiley.com/doi/10.1002/jcp.22419/full" class="external text" rel="nofollow">Bi <I>et al.</I>, 2011</A>). The molecule has a wide range of physiological functions in cellular processes and a great impact on the cell integrity (<A href="http://ini.sagepub.com/content/17/2/212.abstract" class="external text" rel="nofollow">Grahnert <I>et al.</I>, 2011</A>). <A href="http://onlinelibrary.wiley.com/doi/10.1002/jcp.22419/full" class="external text" rel="nofollow">Bi <I>et al.</I> (2011)</A> announced the following physiological functions for NAD(P): cellular metabolism, energy metabolism, cofactor, DNA repair, redox reaction, precursor for vitamine B<SUB>12</SUB>, telomere maintenance, gene silencing, cell longevity, immune response, Ca<SUP>2+</SUP> signaling and regulation of incretion. Thus, NAD is integrated into the energy metabolism and determines the redox status of the cell due to its purpose as a reduction equivalent (see Figures 1 and 2). This is strongly linked to the control of signaling and transcriptional events. For instance, NAD<SUP>+</SUP> has been recently described as a modulator of immune functions (<A href="http://ini.sagepub.com/content/17/2/212.abstract" class="external text" rel="nofollow">Grahnert <I>et al.</I>, 2011</A>) and a master regulator of transcription (<A href="http://www.sciencedirect.com/science/article/pii/S1874939910001057" class="external text" rel="nofollow">Ghosh <I>et al.</I>, 2010</A>). Several enzymes are tightly regulated by the cellular balance between the oxidized and reduced form of NAD<SUP>+</SUP> and are therefore regarded as a potential target for therapeutics (<A href="http://jpet.aspetjournals.org/content/324/3/883.abstract" class="external text" rel="nofollow">Sauve, 2008</A>). Indeed, NAD<SUP>+</SUP> acts as a substrate for a wide range of proteins including NAD<SUP>+</SUP>-dependent protein deacetylases, poly(ADP-ribose) polymerases and several transcription factors (<A href="http://edrv.endojournals.org/content/31/2/194.abstract" class="external text" rel="nofollow">Houtcooper <I>et al.</I>, 2010</A>). An imbalance of the cellular NAD<SUP>+</SUP> level has usually relevance in symptoms and diseases like stroke, cardiac ischemia, epilepsy, huntington disease, wallerian degeneration, cancer, type two diabetes mellitus (T2DM), neutrophil survival, longevity, obesity as well as cardiovascular and neurodegenerative disease (<A href="http://jpet.aspetjournals.org/content/324/3/883.abstract" class="external text" rel="nofollow">Sauve, 2008</A>; <A href="http://edrv.endojournals.org/content/31/2/194.abstract" class="external text" rel="nofollow">Houtcooper <I>et al.</I>, 2010</A>). Additionally, the regulation of NAD<SUP>+</SUP>-dependent pathways may have an influence on oxidative metabolism and life span extension (<A href="http://edrv.endojournals.org/content/31/2/194.abstract" class="external text" rel="nofollow">Houtcooper <I>et al.</I>, 2010</A>). Because of the essentialities of DNA ligases for fundamental processes in bacterial cells, the biosynthetic pathways for NAD<SUP>+</SUP> are proposed as promising novel antibiotics targets against pathogens (<A href="http://onlinelibrary.wiley.com/doi/10.1002/jcp.22419/full" class="external text" rel="nofollow">Bi <I>et al.</I>, 2011</A>).
NAD<SUP>+</SUP> is embedded in the cellular metabolism and has a great impact on essential cellular processes.
The biosynthesis in most bacteria and plants starts with the anaerobic conversion of the amino acid aspartate to quinolinic acid (QA). As indicated in Figure 3, in yeast, humans and some bacterial microbes the quinolinic acid (QA) derives from tryptophan in an aerobic manner. This is a crucial step that leads under consumption of nicotinic acid (Na) to the nicotinate ring system via formation of nicotinic acid monoculeotide (NaMN), respectively (<A href="http://jpet.aspetjournals.org/content/324/3/883.abstract" class="external text" rel="nofollow">Sauve, 2008</A>). After conjugation of adenosine mononucleotide (AMP) the NAD synthetase forms nicotinamide adenine dinucleotide (NAD). Additionally, it has been reported recently that in humans exists an nicotinic acid-independent reaction catalysed by the enzyme nampt/PBEF, which uses nicotinamide (Nam) directly as a substrate (<A href="http://www.ncbi.nlm.nih.gov/pubmed/12555668" class="external text" rel="nofollow">Rongvaux, 2002</A>). Diet or pharmalogical agents can thereby serve as a source for nicotinamide or nicotinic acid (<A href="http://jpet.aspetjournals.org/content/324/3/883.abstract" class="external text" rel="nofollow">Sauve, 2008</A>). Fundamental processes as the cellular energy metabolism determine the balance between the oxidized (NAD<SUP>+</SUP>) and reduced (NADH) form of NAD (see Figures 1 and 2).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption"><B>Figure 3: NAD salvage: for synthesis and recycling of NAD vertebrates use tryptophan (Trp), nicotinic acid (Na), nicotinamide (Nam) and/or nicotinamide riboside (NR).</B></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Bacterial_DNA_ligases">Bacterial DNA ligases</SPAN></H2><P>Bacterial DNA ligases are important enzymes for DNA repair and replication. The big difference between bacterial and eukaryotic DNA ligases is their specifity for cofactors. While eukaryotic DNA ligases use ATP as a source for building a phosphodiester bond in the DNAs phosphate backbone, the bacterial ones display a NAD<SUP>+</SUP>-dependent biochemical catalysis. These enzymes are essential for bacterial survival and are therefore in focus for designing highly specific antiinfectives (<A href="http://www.cell.com/structure/abstract/S0969-2126%2804%2900235-7" class="external text" rel="nofollow">Gajiwala &amp; Pinko, 2004</A>). All bacterial DNA ligases possess highly conserved functional domains. Generally, they can be devided into four functionally distinguishable domains, each with characteristic motifs: 
</P><UL><LI> Domain 1: Ia, Ib (NTase)
</LI><LI> Domain 2: Oligomer-binding (OB) beta-barrel
</LI><LI> Domain 3: Zinc (Zn)-finger, helix-hairpin-helix motif (HhH)
</LI><LI> Domain 4: BRCT domain
</LI></UL><P>All bacterial DNA ligases share three fundamental steps during their reaction mechanism (<A href="http://www.cell.com/molecular-cell/abstract/S1097-2765%2807%2900144-X" class="external text" rel="nofollow">Nandakumar <I>et al.</I>, 2007</A>):
</P><UL><LI> Adenylation of DNA ligase by transferring hydrolysed AMP from NAD<SUP>+</SUP> to the active site lysine and release of NMN
</LI><LI> Tranfer of the AMP from DNA ligase to the 5'-phosphate group of nicked DNA
</LI><LI> Formation of a phosphodiester bond by attack of the nick 3'-OH on the 5'-phosphoanhydride linkage and release of free AMP
</LI></UL><H2><SPAN class="mw-headline" id="Applications_of_molecular_beacons">Applications of molecular beacons</SPAN></H2><P><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Description#NAD.2B_detection" class="external text" rel="nofollow">Molecular beacons</A> can be applied to a wide range of biological and diagnostical studies. Some examples for applying molecular beacons in different biological fields are described in Figures 4, 5 and 6. Whenever there is a specific nucleic acid which is supposed to be detected they can be utilized. Molecular beacons are used for the detection and distinction of retroviral DNA from different types of HIV, for instance (<A href="http://www.sciencedirect.com/science/article/pii/S0009898105004249" class="external text" rel="nofollow">Marras <I>et al.</I>, 2006</A>). Even for identification of proteins which interact with DNA or RNA they are useful. In sum, there are a lot of opportunities to apply molecular beacons (<A href="http://www.jbsdonline.com/c3008/c4017/Molecular-Beacons-Nucleic-Acid-Hybridization-and-Emerging-Applications-p-497-504-p10542.html" class="external text" rel="nofollow">Antony &amp; Subramaniam, 2001</A>):
</P><UL><LI> Detection of PCR amplification products
</LI><LI> Mutational analysis
</LI><LI> Genotyping and allele discrimination
</LI><LI> Clinical diagnosis
</LI><LI> Oligonucleotide degradation studies
</LI><LI> Analysis of DNA/RNA hybridization
</LI><LI> Realtime visualization of hybridization in cells
</LI><LI> Analysis of triplex DNA formation
</LI><LI> Protein/DNA interaction
</LI><LI> Enzymatic assays
</LI><LI> Biosensors
</LI></UL><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 4: Hybridization of the molecular beacon and a complementary target forms the molecular beacons open state which leads to an increase of the fluorescence intensity. The molecular beacon can be immmobilized using specific biomolecular interaction, e.g. streptavidin/biotin or covalent cross-linking (taken from <A href="http://www.ijcep.com/708009A.html" class="external text" rel="nofollow">Kim <I>et al.</I>, 2008</A>).</B></DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:313px;"><DIV class="thumbcaption"><B>Figure 5: Single nucleotide polymorphism of nucleic acids, including different alleles as indicated, can be detected by differently designed and labeled molecular beacons (taken from <A href="http://www.ijcep.com/708009A.html" class="external text" rel="nofollow">Kim <I>et al.</I>, 2008</A>).</B></DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:277px;"><DIV class="thumbcaption"><B>Figure 6: Realtime <I>in vivo</I> imaging of specific mRNAs utilizing two molecular beacons that target adjacent positions on the mRNA allowing FRET to take place (taken from <A href="http://www.ijcep.com/708009A.html" class="external text" rel="nofollow">Kim <I>et al.</I>, 2008</A>).</B></DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Further_applications_of_the_NAD.2B_bioassay">Further applications of the NAD<SUP>+</SUP> bioassay</SPAN></H2><P>The reviewed molecular beacon based <A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Description#NAD.2B_detection" class="external text" rel="nofollow">NAD<SUP>+</SUP> bioassay</A> can be applied to biochemical and biomedical studies (<A href="http://pubs.acs.org/doi/abs/10.1021/ac102742k" class="external text" rel="nofollow">Tang <I>et al.</I>, 2011</A>). Accordingly, it can be utilized to detect NAD<SUP>+</SUP>/NADH-dependent enzymatic processes. The low limit of detection, its reliability and manageability provides a practical alternative to present colorimetric, fluorometric, chemiluminescent, electrochemical or mass spectrometric methods detecting NAD<SUP>+</SUP> or NADH. In context of clinical applications and therapeutics the NAD<SUP>+</SUP> bioassay can be useful to monitor cellular NAD<SUP>+</SUP> levels during treatments of tissues or cell cultures for identification of drug targets, for instance. It may also be applied in the field of diagnostics. Finally, the principle of the proposed NAD<SUP>+</SUP> bioassay can be used for cell-free optical biosensors by taking molecular beacons either immobilized on the sensor surface or being present in the reaction medium.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>Antony T, Subramaniam V (2001) Molecular Beacons: Nucleic Acid Hybridization and Emerging Applications, <A href="http://www.jbsdonline.com/c3008/c4017/Molecular-Beacons-Nucleic-Acid-Hybridization-and-Emerging-Applications-p-497-504-p10542.html" class="external text" rel="nofollow"><I>Journal of Biomolecular Structure &amp; Dynamics</I> 19(3):497-504</A>.
</P><P>Bi J, Wang H, Xiel J (2011) Comparative genomics of NAD(P) biosynthesis and novel antibiotic drug targets, <A href="http://onlinelibrary.wiley.com/doi/10.1002/jcp.22419/full" class="external text" rel="nofollow"><I>Journal of Cellular Physiology</I> 226(2):331-340</A>.
</P><P>Gajiwala KS, Pinko C (2004) Structural Rearrangement Accompanying NAD<SUP>+</SUP> Synthesis within a Bacterial DNA Ligase Crystal, <A href="http://www.cell.com/structure/abstract/S0969-2126%2804%2900235-7" class="external text" rel="nofollow"><I>Structure</I> 12(8):1449-1459</A>.
</P><P>Ghosh S, George S, Roy U, Ramachandran D, Kolthur-Seetharam U (2010) NAD: A master regulator of transcription, <A href="http://www.sciencedirect.com/science/article/pii/S1874939910001057" class="external text" rel="nofollow"><I>Biochimica et Biophysica Acta</I> 1799(10-12):681-693</A>.
</P><P>Grahnert And, Grahnert Anj, Klein C, Schilling E, Wehrhahn J, Hauschildt S (2011) NAD<SUP>+</SUP>: A modulator of immune functions, <A href="http://ini.sagepub.com/content/17/2/212.abstract" class="external text" rel="nofollow"><I>Innate Immunity</I> 17(2):212-233</A>.
</P><P>Houtkooper RH, Cantó C, Wanders RJ, Auwerx J (2010) The Secret Life of NAD<SUP>+</SUP>: An Old Metabolite Controlling New Metabolic Signaling Pathways, <A href="http://edrv.endojournals.org/content/31/2/194.abstract" class="external text" rel="nofollow"><I>Endocrine Reviews</I> 31(2):194-223</A>.
</P><P>Kim Y, Sohn D, Tan W (2008) Molecular Beacons in Biomedical Detection and Clinical Diagnosis, <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2480550/?tool=pmcentrez&amp;rendertype=abstract" class="external text" rel="nofollow"><I>International Journal of Clinical and Experimental Pathology</I> 1:105-116</A>.
</P><P>Marras SA, Tyagi S, Kramer FR (2006) Real-time assays with molecular beacons and other fluorescent nucleic acid hybridization probes, <A href="http://www.sciencedirect.com/science/article/pii/S0009898105004249" class="external text" rel="nofollow"><I>Clinica chimica acta</I> 363(1-2):48-60</A>.
</P><P>Miesel L, Kravec C, Xin AT, McMonagle P, Ma S, Pichardo J, Feld B, Barrabee E, Palermo R (2007) High-throughput assay for the adenylation reaction of bacterial DNA ligase, <A href="http://medical-journals.healia.com/doc/17493575/A-high-throughput-assay-for-the-adenylation-reaction-of-bacterial-DNA-ligase" class="external text" rel="nofollow">Analytical Biochemistry 366:9-17</A>.
</P><P>Nandakumar J, Nair PA, Shuman S (2007) Last Stop on the Road to Repair: Structure of <I>E. coli</I> DNA Ligase Bound to Nicked DNA-Adenylate, <A href="http://www.cell.com/molecular-cell/abstract/S1097-2765%2807%2900144-X" class="external text" rel="nofollow"><I>Molecular Cell</I> 26(2):257-271</A>.
</P><P>Rongvaux A, Shea RJ, Mulks MH, Gigot D, Urbain J, Leo O, Andris F (2002) Pre-B-cell colony-enhancing factor, whose expression is up-regulated in activated lymphocytes, is a nicotinamide phosphoribosyltransferase, a cytosolic enzyme involved in NAD biosynthesis, <A href="http://www.ncbi.nlm.nih.gov/pubmed/12555668" class="external text" rel="nofollow"><I>European Journal of Immunology</I> 32(11):3225-3234</A>.
</P><P>Sauve AA (2008) NAD<SUP>+</SUP> and Vitamin B3: From Metabolism to Therapies, <A href="http://jpet.aspetjournals.org/content/324/3/883.abstract" class="external text" rel="nofollow"><I>The Journal of Pharmacology and Experimental Therapeutics</I> 324(3):883-893</A>.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Bielefeld-Germany/Project/Background/NAD">http://2011.igem.org/Team:Bielefeld-Germany/Project/Background/NAD</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Project/Background/NAD" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Project/Background/NAD" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Background/NAD&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Background/NAD&amp;oldid=256294" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>