<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Protocols_Genetics"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Protocols/Genetics">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Protocols/Genetics&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Protocols/Genetics&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Protocols/Genetics&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Protocols/Genetics" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Protocols/Genetics</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu-igem"><UL id="menu" class="sf-menu"><LI><A style="border-left:0px;" href="https://2011.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Students">Students</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Advisors">Supervisors</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Photos">Photos</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Bielefeld-University‎‎">Bielefeld University</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></UL><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Description">Description</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Achievements">Achievements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Appoach">Approach</A></LI><LI><A href="#">Background</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/Technology">Technology</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer/Guide">DIY Nanobiotech Guide</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Future">Future Applications</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Labjournal">Labjournal</A></LI><LI><A href="#">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Genetics">Molecular Genetics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing">Production</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Analytics">Analytics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials">Materials</A></LI><LI><A href="#">Results</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Data_Page">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/SinceAmsterdam">Since Regionals</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Modell">Model</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Cell-free">Cell-free Biosafety</A></LI><LI><A href="#">Outreach</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_lab_course">Open Lab Course</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_science_cafe">Science Cafe</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/high-school">High School</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/cebitec-symposium">CeBiTec Symposium</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/student-convention">Student Convention</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/strategic-process">Strategic Process</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Black-box">Beacon Box</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Press">Press</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></LI></DIV><P><B>Molecular Genetics:</B> This is a list of the protocols we used in our project to modify DNA and bacterias.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Isolation_of_complete_genomes"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Isolation of complete genomes</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Transformation_via_electroporation"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Transformation via electroporation</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Generating_electrocompetent_cells"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Generating electrocompetent cells</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Transformation_of_Single_Step_.28KRX.29_Competent_Cells_by_Promega"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Transformation of Single Step (KRX) Competent Cells by Promega</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Standard_BioBrick_Assembly"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Standard BioBrick Assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Suffix_Insertion"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Suffix Insertion</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Prefix_Insertion"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Prefix Insertion</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Variations"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Variations</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-9"><A href="#Standard_Freiburg_BioBrick_Assembly"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Standard Freiburg BioBrick Assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Suffix_Insertion_2"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Suffix Insertion</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Prefix_Insertion_2"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Prefix Insertion</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Variations_2"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Variations</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Standard_3A_assembly"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Standard 3A assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Digestion"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Ligation"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Freiburg_3A_assembly"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Freiburg 3A assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Digestion_2"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Ligation_2"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Gibson_assembly"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Gibson assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Preparation_of_DNA_molecules_for_in_vitro_recombination"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Preparation of DNA molecules for in vitro recombination</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#In_vitro_recombination"><SPAN class="tocnumber">9.2</SPAN><SPAN class="toctext">In vitro recombination</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Restriction_analysis"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Restriction analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#Colony_PCR"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Site_Directed_Mutagenesis"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Site Directed Mutagenesis</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Isolation_of_complete_genomes">Isolation of complete genomes</SPAN></H2><P>Using the <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> Fermentas GeneJET™ Plasmid Miniprep Kit</A></P><UL><LI> centrifuge 10 mL of over-night liquid culture
</LI><LI> resuspend pellet in 800 µL of resuspension solution
</LI><LI> ribolyse three times for 60 s with 6500 rpm (30 s break between every run, 400 mg beads)
</LI><LI> centrifuge 3 min at 10,000 rpm
</LI><LI> transfer 500 µL of supernatant into fresh, 2 mL eppi
</LI><LI> mix with 500 µL of lysis buffer, invert 4 - 6 times
</LI><LI> mix with 700 µL of neutralisation buffer, invert 4 - 6 times
</LI><LI> centrifuge 10 min at 12,000 rpm
</LI><LI> transfer 2 x 600 µL of supernatant into spin column (centrifuge down each time for 30 s at 12,000 rpm)
</LI><LI> wash two times with 750 µL of washing buffer
</LI><LI> dry column
</LI><LI> elute in 75 µL of elution buffer
</LI></UL><H2><SPAN class="mw-headline" id="Transformation_via_electroporation"> Transformation via electroporation </SPAN></H2><UL><LI> Thaw 50 µL competent <I>E.coli</I> cells on ice, dilute with icecold 50 µL glycerine (10 %) if necessary
</LI><LI> Add 0.5-5 µL plasmid to 50 µl electrocompetent cells
</LI><LI> Store cells on ice for 1 minute 
</LI><LI> Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ώ 
</LI><LI> Transfer transformation reaction to 450 µL SOC-Medium and shake 1 h at 37 °C 
</LI><LI> Centrifuge 2 min at 800 rpm and plate on selective LB-Medium
</LI></UL><H2><SPAN class="mw-headline" id="Generating_electrocompetent_cells">Generating electrocompetent cells</SPAN></H2><P>Materials:
</P><UL><LI>550 mL LB-Medium
</LI><LI>1 L cooled bidest. H<SUB>2</SUB>O
</LI><LI>150 mL cooled 10 % glycerine
</LI><LI>10 pre-cooled 50 mL Falcons
</LI></UL><P>Protocol:
</P><UL><LI>Inoculate 2x3 mL LB with bacterial stock; incubate over night at 37 °C and 200 rpm
</LI><LI>Inoculate 2x250 mL LB with the over night cultures in 1-litre-flask at 37 °C and 140 rpm
</LI><LI>Incubate until OD<SUB>600</SUB> 0,4-0,6
</LI><LI>Cool the culture 15-30 minutes on ice
</LI></UL><P>Important: keep your cells at 2-4 °C onwards from this step
</P><UL><LI>Divide the cultures into cooled 50 mL Falcons and centrifugate at 4000 rpm, 4 °C for 15 minutes, make sure to slowly accelerate and deccelerate 
</LI><LI>Discard supernatant
</LI><LI>Resuspend pellet in 5 mL cooled bidest H<SUB>2</SUB>O (and don't get frustrated while doing it, keep shaking gently)
</LI><LI>Pool two suspensions each, add bidest H<SUB>2</SUB>O up to 50 mL and centrifugate again (see centrifugation above)
</LI><LI>Discard supernatant
</LI><LI>Resuspend pellet in 5 mL cooled bidest H<SUB>2</SUB>O
</LI><LI>Add bidest H<SUB>2</SUB>O up to 50 mL and centrifugate again (see centrifugation above)
</LI><LI>Discard supernatant
</LI><LI>Resuspend pellet in 5 mL cooled 10 % glycerine
</LI><LI>Transfer suspensions in two 50 mL Falcons and centrifugate again (see centrifugation above)
</LI><LI>Discard supernatant
</LI><LI>Add volume of 10 % glycerine that is approximately equal to the volume of the pellet and resuspend
</LI><LI>Divide cells in 100 μL aliquots and freeze in liquid N<SUB>2</SUB> immediately
</LI><LI>Store at -80 °C
</LI></UL><H2><SPAN class="mw-headline" id="Transformation_of_Single_Step_.28KRX.29_Competent_Cells_by_Promega">Transformation of Single Step (KRX) Competent Cells by Promega</SPAN></H2><P>Using <A href="http://www.promega.com/~/media/Files/Resources/Protocols/Technical%20Bulletins/101/Single%20Step%20Competent%20Cells%20Protocol.ashx" class="external text" rel="nofollow">protocol <I>E. coli</I> KRX single step competent cells by Promega</A></P><UL><LI> Remove Single Step (KRX) Competent Cells from –70 °C, and place on ice for 5 minutes or until just thawed.
</LI><LI> Add 1–50 ng of DNA (in a volume not greater than 5 μL) to the Single Step (KRX) Competent Cells. Move the pipette tip through the cells while dispensing. Quickly flick the tube several times. Do not vortex!
</LI><LI> Immediately return the tubes to ice for 5–30 minutes
</LI><LI> Heat-shock cells for 15–20 seconds in a water bath at exactly 42 °C. Do not shake.
</LI><LI> Immediately place the tubes on ice for 2 minutes.
</LI><LI> Add 450 μL of room-temperature SOC-Medium to each transformation reaction, and incubate for 60 minutes at 37 °C with shaking (approximately 225 rpm). For best transformation efficiency, lay the tubes on their sides and tape them to the platform.
</LI><LI> For each transformation reaction, we recommend plating 100 μL of undiluted cells and 1:10 and 1:100 cell dilutions on antibiotic plates. Incubate the plates at 37 °C over night.
</LI></UL><H2><SPAN class="mw-headline" id="Standard_BioBrick_Assembly"> Standard BioBrick Assembly </SPAN></H2><P>modified from <A href="http://openwetware.org/wiki/Silver:_BB_Strategy" class="external text" rel="nofollow">Silver lab</A>:
</P><P>This assembly method can be used for BioBricks which are bigger than 150 bp. The BioBrick should be at least 500 bp bigger or smaller than the backbone. The BioBrick, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other used BioBrick, called vector. So you have to differentiate between a prefix and a suffix insertion.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Silver Suffix Insertion</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Silver Prefix Insertion</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Suffix_Insertion"> Suffix Insertion </SPAN></H3><UL><LI> Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x Tango buffer, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> XbaI</A>, 1 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> PstI</A>. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI><LI> Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> SpeI</A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> PstI</A>. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> SAP (shrimp alcaline phosphatase)</A> and 1.2 µL 10 x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> PCR clean-up kit</A>. 
</LI><LI> Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> T4-Ligase</A>. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
</LI></UL><H3><SPAN class="mw-headline" id="Prefix_Insertion"> Prefix Insertion </SPAN></H3><UL><LI> Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x BamHI buffer, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> EcoRI</A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> SpeI</A>. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI><LI> Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10 x Tango buffer, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> EcoRI</A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> XbaI</A>. Digest for 2h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> SAP (shrimp alcaline phosphatase)</A> and 1.2 µL 10 x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> PCR clean-up kit</A>. 
</LI><LI> Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> T4-Ligase</A>. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
</LI></UL><H3><SPAN class="mw-headline" id="Variations"> Variations </SPAN></H3><UL><LI> A digestion over night is possible. If you digest over night use only 0.1 µL restriction enzyme. 
</LI><LI> It is also possible to use PCR product as insert. Digest after PCR with corresponding restriction enzymes and clean up with <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> PCR clean-up kit</A>. This could lead to higher yields of insert DNA because a lot of DNA gets lost during the gel electrophoresis clean up. 
</LI><LI> Sometimes some BioBricks are hard to assemble. Then you have to clean up the vector by gel electrophoresis as well.
</LI></UL><H2><SPAN class="mw-headline" id="Standard_Freiburg_BioBrick_Assembly"> Standard Freiburg BioBrick Assembly </SPAN></H2><P>Modified from <A href="http://openwetware.org/wiki/Silver:_BB_Strategy" class="external text" rel="nofollow">Silver lab</A> and <A href="http://partsregistry.org/Assembly_standard_25" class="external text" rel="nofollow">Assembly standard 25</A>:
</P><P>This assembly method can be used for fusion protein assemblies with BioBricks which are bigger than 150 bp. The BioBrick should be at least 500 bp bigger or smaller than the backbone. The BioBrick, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other used BioBrick, which is called vector and needs to be available in the BioBrick <A href="http://partsregistry.org/Assembly_standard_25" class="external text" rel="nofollow">Assembly standard 25</A>. You have to differentiate between a prefix and a suffix insertion.
</P><H3><SPAN class="mw-headline" id="Suffix_Insertion_2"> Suffix Insertion </SPAN></H3><UL><LI> Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x NEB buffer 4 + 0.1 µL 100x BSA, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> NgoMIV (NEB)</A>, 1 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> PstI</A>. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI><LI> Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> AgeI</A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> PstI</A>. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> SAP (shrimp alcaline phosphatase)</A> and 1.2 µL 10x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> PCR clean-up kit</A>. 
</LI><LI> Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> T4-Ligase</A>. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
</LI></UL><H3><SPAN class="mw-headline" id="Prefix_Insertion_2"> Prefix Insertion </SPAN></H3><UL><LI> Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> EcoRI</A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> AgeI</A>. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI><LI> Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10 x NEB buffer 4, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> EcoRI</A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> NgoMIV (NEB)</A>. Digest for 2h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> SAP (shrimp alcaline phosphatase)</A> and 1.2 µL 10 x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> PCR clean-up kit</A>. 
</LI><LI> Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> T4-Ligase</A>. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
</LI></UL><H3><SPAN class="mw-headline" id="Variations_2"> Variations </SPAN></H3><UL><LI> A digestion over night is possible. If you digest over night use only 0.1 µL restriction enzyme. 
</LI><LI> It is also possible to use PCR product as insert. Digest after PCR with corresponding restriction enzymes and clean up with <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> PCR clean-up kit</A>. This could lead to higher yields of insert DNA because a lot of DNA gets lost during the gel electrophoresis clean up. 
</LI><LI> Sometimes some BioBricks are hard to assemble. Then you have to clean up the vector by gel electrophoresis as well.
</LI></UL><H2><SPAN class="mw-headline" id="Standard_3A_assembly">Standard 3A assembly</SPAN></H2><P>Modified from <A href="http://ginkgobioworks.com/support/BioBrick_Assembly_Manual.pdf" class="external text" rel="nofollow">BioBrick Assembly Manual by Ginkgo BioWorks</A></P><H3><SPAN class="mw-headline" id="Digestion">Digestion</SPAN></H3><UL><LI> Thaw DNA from upstream and downstream part and the destination plasmid on ice. 
</LI><LI>  Destination plasmid has to carry the <I>ccdB</I> gene &lt;partinfo&gt;P1010&lt;/partinfo&gt; as insert and has to have a different antibiotic resistance than the plasmids carrying the upstream and downstream parts
</LI><LI>  DNA has to be cleaned (by <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> MiniPrep or after a PCR</A>)
</LI></UL><LI> 500 ng DNA / digestion mix for upstream part, downstream part and destination plasmid (total volume of mix 10 µL, dilute with ddH<SUB>2</SUB>0 if necessary)
</LI><LI> Add 1 µL of 10x buffer and <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> restriction enzymes</A> as shown in the following table: 
</LI><CENTER><TABLE cellpadding="10" style="border-collapse: collapse; border-width: 1px; border-style: solid; border-color: #000"><TBODY><TR><TH style="border-style: solid; border-width: 1px"> Upstream part
</TH><TH style="border-style: solid; border-width: 1px"> Downstream part
</TH><TH style="border-style: solid; border-width: 1px"> Destination plasmid
</TH></TR><TR><TD style="border-style: solid; border-width: 1px"> enzyme 1
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL EcoRI
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL XbaI
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL EcoRI
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> enzyme 2
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL SpeI
</TD><TD style="border-style: solid; border-width: 1px"> 1 µL PstI
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL PstI
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> buffer
</TD><TD style="border-style: solid; border-width: 1px"> 4 + BSA
</TD><TD style="border-style: solid; border-width: 1px"> Tango
</TD><TD style="border-style: solid; border-width: 1px"> Orange
</TD></TR></TBODY></TABLE></CENTER><UL><LI> Incubation of the digestion mixes at 37 °C
</LI><LI> After 2 h: add 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> SAP (shrimp alcaline phosphatase)</A> and 1.15 µL 10x SAP buffer to destination plasmid mix, continue incubation at 37 °C
</LI><LI> After another hour: heat inactivation of all mixes for 20 min at 80 °C
</LI><LI> Continue with ligation or freeze the mixes
</LI></UL><H3><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H3><UL><LI> Ligation mix: 
</LI><LI> 2 µL ddH<SUB>2</SUB>O
</LI><LI> 5 µL of every digestion mix (so 15 µL in total)
</LI><LI> 2 µL T4-DNA-ligase buffer (thaw on ice!)
</LI><LI> 1 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> T4-DNA-ligase</A></LI></UL><LI> Incubate at least 20 min at room temperature, afterwards heat inactivation for 5 min at 70 °C (optional)
</LI><LI> Freeze ligation mix or continue with transformation (heatshock or electroporation)
</LI><H2><SPAN class="mw-headline" id="Freiburg_3A_assembly">Freiburg 3A assembly</SPAN></H2><P>Modified from <A href="http://ginkgobioworks.com/support/BioBrick_Assembly_Manual.pdf" class="external text" rel="nofollow">BioBrick Assembly Manual by Ginkgo BioWorks</A> and <A href="http://partsregistry.org/Assembly_standard_25" class="external text" rel="nofollow">Assembly standard 25</A></P><H3><SPAN class="mw-headline" id="Digestion_2">Digestion</SPAN></H3><UL><LI> Thaw DNA from upstream and downstream part (=N-terminal and C-terminal protein domain) and the destination plasmid on ice. 
</LI><LI>  Destination plasmid has to carry the <I>ccdB</I> gene &lt;partinfo&gt;P1010&lt;/partinfo&gt; as insert and has to have a different antibiotic resistance than the plasmids carrying the upstream and downstream parts
</LI><LI>  DNA has to be cleaned (by <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> MiniPrep or after a PCR</A>)
</LI></UL><LI> 500 ng DNA / digestion mix for upstream part, downstream part and destination plasmid (total volume of mix 10 µL, dilute with ddH<SUB>2</SUB>0 if necessary)
</LI><LI> Add 1 µL of 10x buffer and <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> restriction enzymes</A> as shown in the following table: 
</LI><CENTER><TABLE cellpadding="10" style="border-collapse: collapse; border-width: 1px; border-style: solid; border-color: #000"><TBODY><TR><TH style="border-style: solid; border-width: 1px"> Upstream part
</TH><TH style="border-style: solid; border-width: 1px"> Downstream part
</TH><TH style="border-style: solid; border-width: 1px"> Destination plasmid
</TH></TR><TR><TD style="border-style: solid; border-width: 1px"> enzyme 1
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL EcoRI
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL NgoMIV
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL EcoRI
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> enzyme 2
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL AgeI
</TD><TD style="border-style: solid; border-width: 1px"> 1 µL PstI
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL PstI
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> buffer
</TD><TD style="border-style: solid; border-width: 1px"> Orange
</TD><TD style="border-style: solid; border-width: 1px"> NEB buffer 4 + BSA
</TD><TD style="border-style: solid; border-width: 1px"> Orange
</TD></TR></TBODY></TABLE></CENTER><UL><LI> Incubation of the digestion mixes at 37 °C
</LI><LI> After 2 h: add 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> SAP (shrimp alcaline phosphatase)</A> and 1.15 µL 10x SAP buffer to destination plasmid mix, continue incubation at 37 °C
</LI><LI> After another hour: heat inactivation of all mixes for 20 min at 80 °C
</LI><LI> Continue with ligation or freeze the mixes
</LI></UL><H3><SPAN class="mw-headline" id="Ligation_2">Ligation</SPAN></H3><UL><LI> Ligation mix: 
</LI><LI> 2 µL ddH<SUB>2</SUB>O
</LI><LI> 5 µL of every digestion mix (so 15 µL in total)
</LI><LI> 2 µL T4-DNA-ligase buffer (thaw on ice!)
</LI><LI> 1 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> T4-DNA-ligase</A></LI></UL><UL><LI> Incubate at least 20 min at room temperature, afterwards heat inactivation for 5 min at 70 °C (optional)
</LI><LI> Freeze ligation mix or continue with transformation (heatshock or electroporation)
</LI></UL><H2><SPAN class="mw-headline" id="Gibson_assembly"> Gibson assembly </SPAN></H2><P>Modified from <A href="http://www.nature.com/nmeth/journal/v6/n5/full/nmeth.1318.html" class="external text" rel="nofollow">Gibson <I>et al.</I> (2009)</A></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Gibson Assembly</DIV></DIV></DIV><P>This assembly method is an isothermal, single-reaction method for assembling multiple overlapping DNA molecules. By coordinating the activity of a 5‘ exonuclease, a DNA polymerase and a DNA ligase two adjacent DNA fragments with complementary terminal sequence overlaps can be joined into a covalently sealed molecule, without the use of any restriction endonuclease.
</P><H3><SPAN class="mw-headline" id="Preparation_of_DNA_molecules_for_in_vitro_recombination">Preparation of DNA molecules for <I>in vitro</I> recombination</SPAN></H3><UL><LI> Generate the complementary sequence overlaps by PCR using the <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> Phusion DNA-polymerase</A>. If necessary add 5 M Betain in the reaction mix by reducing the amount of H<SUB>2</SUB>O to decrease the number of false PCR products.
</LI><LI> Identify the PCR products of interest by gel electrophoresis with known DNA standards.
</LI><LI> Extract the PCR products from the gel by cutting out the DNA fragments and clean them up by using a commercial <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" class="external text" rel="nofollow">clean up kit</A>.
</LI></UL><H3><SPAN class="mw-headline" id="In_vitro_recombination"><I>In vitro</I> recombination</SPAN></H3><UL><LI> assembly mixture 
</LI><LI> 320 µL <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials#5x_isothermal_reaction_buffer_for_Gibson_assembly" class="external text" rel="nofollow">5x isothermal reaction buffer</A></LI><LI> 0.64 µL of 10 U mL<SUP>-1</SUP><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" class="external text" rel="nofollow">T5 exonuclease</A> (for DNA molecules overlapping by greater than 150 bp add 3.2 µL of 10 U ml<SUP>–1</SUP><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" class="external text" rel="nofollow">T5 exonuclease</A>)
</LI><LI> 20 µL of 2 U mL<SUP>-1</SUP><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" class="external text" rel="nofollow">Phusion DNA polymerase</A></LI><LI> 160 µL of 40 U mL<SUP>-1</SUP><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" class="external text" rel="nofollow"><I>taq</I> DNA ligase</A></LI><LI> add ddH<SUB>2</SUB>O water up to a final volume of 1.2 mL
</LI><LI> aliquote 15 µL of the reagent-enzyme mix and store it at –20 ˚C
</LI></UL><LI> Thaw 15 µL assembly mixture aliquot and keep it on ice until use.
</LI><LI> Add 5 µL of the purified DNA molecules in equimolar amounts (between 10 and 100 ng of each DNA fragment).
</LI><LI> Incubate the resulting mixture at 50 ˚C for 15 to 60 min, with 60 min being optimal.
</LI><LI> Transformation (heatshock or <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Genetics#Transformation_via_electroporation" class="external text" rel="nofollow">electroporation</A>) without cleaning up the assembly product.
</LI><H2><SPAN class="mw-headline" id="Restriction_analysis"> Restriction analysis </SPAN></H2><UL><LI> Digest BioBrick of interest: about 400 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> NotI or PstI</A>. Digest for 2 h at 37 °C. NotI is used to determine the length of the BioBrick and the plasmid backbone, PstI ist used to determine the length of the BioBrick in the plasmid backbone. 
</LI><LI> Gel electrophoresis: add 2 µL loading buffer to every digestion mix, apply about 100 - 200 ng DNA / pocket in gel. Don't forget to apply the uncut BioBrick as well. A good agarose concentration for BioBricks between 0.2 and 3 kb is 1.5 %. The smaller your BioBrick of interest is the higher the agarose concentration should be and vice versa. The gel electrophoresis is made with <A href="/Team:Bielefeld-Germany/Protocols/Materials#TAE_buffer" title="Team:Bielefeld-Germany/Protocols/Materials"> TAE-buffer</A>. Be sure that you melt your agarose gel in the same buffer you use for the electrophoresis later.
</LI></UL><H2><SPAN class="mw-headline" id="Colony_PCR"> Colony PCR </SPAN></H2><UL><LI> Pick one colony with a sterile tip and elute it in 100 µL ddH<SUB>2</SUB>0 or medium
</LI><LI> Store the colony in 4 °C while colony PCR is running
</LI><LI> One reaction mix contains: 
</LI><LI> 10 µL 5x buffer
</LI><LI> 2 µL MgCl<SUB>2</SUB> (25 mM stock)
</LI><LI> 1 µL dNTPs
</LI><LI> 0.5 µL primer mix (prefix/suffix primers or sequencing primers)
</LI><LI> 35.25 µL ddH<SUB>2</SUB>O
</LI><LI> 0.25 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> GoTaq polymerase (Promega)</A></LI><LI> 1 µL template
</LI></UL><LI> PCR program: 
</LI><LI> Start: 3 min, 98 °C
</LI><LI> 30 cycles of: 
</LI><LI> 30 s, 98 °C
</LI><LI> 30 s, 55 °C
</LI><LI> 30 s / 1 kb template, 72 °C
</LI><LI> Finish: 5 min, 72 °C
</LI><UL><LI> Gel electrophoresis: check the fragment size
</LI><LI> Plate the correct colony
</LI></UL><H2><SPAN class="mw-headline" id="Site_Directed_Mutagenesis">Site Directed Mutagenesis</SPAN></H2><UL><LI>Design Primers using the <A href="http://www.genomics.agilent.com/CollectionSubpage.aspx?PageType=Tool&amp;SubPageType=ToolQCPD&amp;PageID=15" class="external text" rel="nofollow">QuickChange Primer Design tool</A></LI><LI>Run mutagenesis PCR
</LI><LI>One reaction mix contains:
</LI><LI>5 µL 10x buffer
</LI><LI>0.4 µL <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> Pfu polymerase</A></LI><LI>1 µL dNTPs
</LI><LI>39.6 µL H<SUB>2</SUB>O
</LI><LI>1 µL DMSO
</LI><LI>2 µL primer mix
</LI><LI>1 µL template
</LI></UL><LI>PCR program:
</LI><LI>Start: 2 min, 94 °C
</LI><LI>16 cycles of:
</LI><LI> 45 s, 95 °C
</LI><LI> 30 s, 55 °C
</LI><LI> 2 min/kb, 73 °C
</LI><LI>Finish: 5 min, 72 °C
</LI><LI>Control the amplified PCR-fragments by gel electrophoresis
</LI><LI>Digest the template plasmid by adding 1 µL of <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_enzymes" title="Team:Bielefeld-Germany/Protocols/Materials"> DpnI</A> and incubate for 2-3 h
</LI><LI>Purify the PCR product with a <A href="/Team:Bielefeld-Germany/Protocols/Materials#Used_Kits" title="Team:Bielefeld-Germany/Protocols/Materials"> PCR clean-up kit</A></LI><LI>Transform 3 µL of the purified PCR product into <A href="/Team:Bielefeld-Germany/Protocols/Genetics#Generating_electrocompetent_cells" title="Team:Bielefeld-Germany/Protocols/Genetics"> electrocompetent <I>E. coli</I> XL1 blue cells</A></LI><LI>Screen the transformants using restiction digest and sequencing
</LI><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Bielefeld-Germany/Protocols/Genetics">http://2011.igem.org/Team:Bielefeld-Germany/Protocols/Genetics</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Protocols/Genetics" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Protocols/Genetics" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Protocols/Genetics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Protocols/Genetics&amp;oldid=236981" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>