<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Results_S-Layer_CspB_CH"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Results/S-Layer/CspB_CH">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Results/S-Layer/CspB_CH&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/S-Layer/CspB_CH&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/S-Layer/CspB_CH&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Results/S-Layer/CspB_CH" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Results/S-Layer/CspB CH</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu-igem"><UL id="menu" class="sf-menu"><LI><A style="border-left:0px;" href="https://2011.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Students">Students</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Advisors">Supervisors</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Photos">Photos</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Bielefeld-University‎‎">Bielefeld University</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></UL><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Description">Description</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Achievements">Achievements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Appoach">Approach</A></LI><LI><A href="#">Background</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/Technology">Technology</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer/Guide">DIY Nanobiotech Guide</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Future">Future Applications</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Labjournal">Labjournal</A></LI><LI><A href="#">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Genetics">Molecular Genetics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing">Production</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Analytics">Analytics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials">Materials</A></LI><LI><A href="#">Results</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Data_Page">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/SinceAmsterdam">Since Regionals</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Modell">Model</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Cell-free">Cell-free Biosafety</A></LI><LI><A href="#">Outreach</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_lab_course">Open Lab Course</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_science_cafe">Science Cafe</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/high-school">High School</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/cebitec-symposium">CeBiTec Symposium</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/student-convention">Student Convention</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/strategic-process">Strategic Process</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Black-box">Beacon Box</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Press">Press</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#CspB_from_Corynebacterium_halotolerans"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">CspB from Corynebacterium halotolerans</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#CspB_without_TAT-sequence_and_lipid_anchor"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">CspB without TAT-sequence and lipid anchor</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Cultivation_and_protein_expression"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Cultivation and protein expression</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Identification_and_localization"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Identification and localization</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-5"><A href="#CspB_without_TAT-sequence_and_with_lipid_anchor"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">CspB without TAT-sequence and with lipid anchor</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Cultivation_and_protein_expression_2"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Cultivation and protein expression</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Identification_and_localization_2"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Identification and localization</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#CspB_with_TAT-sequence_and_without_lipid_anchor"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">CspB with TAT-sequence and without lipid anchor</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Cultivation_and_protein_expression_3"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Cultivation and protein expression</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Identification_and_localisation"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Identification and localisation</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Purification"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Purification</SPAN></A></LI><LI class="toclevel-4 tocsection-12"><A href="#Final_purification_strategy"><SPAN class="tocnumber">1.3.3.1</SPAN><SPAN class="toctext">Final purification strategy</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="CspB_from_Corynebacterium_halotolerans">CspB from <I>Corynebacterium halotolerans</I></SPAN></H1><H2><SPAN class="mw-headline" id="CspB_without_TAT-sequence_and_lipid_anchor">CspB without TAT-sequence and lipid anchor</SPAN></H2><H3><SPAN class="mw-headline" id="Cultivation_and_protein_expression">Cultivation and protein expression</SPAN></H3><P>For characterization, PS2 also named CspB <A href="http://partsregistry.org/Part:BBa_K525222" class="external text" rel="nofollow">(K525222)</A> was fused with a monomeric RFP <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">(BBa_E1010)</A> using <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Gibson_assembly" class="external text" rel="nofollow">Gibson assembly</A>.
</P><P>The mRFP|CspB fusion protein was overexpressed in <I>E. coli</I> KRX after induction of T7 polymerase by supplementation of 0.1 % L-rhamnose using the <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing#Expression_of_S-layer_genes_in_E._coli" class="external text" rel="nofollow">autoinduction protocol</A>.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 1: Growthcurve of <I>E. coli</I> KRX expressing the fusion protein of CspB and mRFP with and without induction, cultivated at 37 °C in autoinduction medium with, respectively, without inductor. A curve depicting KRX wildtype is shown for comparison. After induction at approximately 6 h the OD<SUB>600</SUB> of the induced K525222 visibly drops when compared to the uninduced culture. While the induced culture grow significantly slower than KRX wildtype the uninduced seems to be unaffected.</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 2: RFU to OD<SUB>600</SUB> ratio of <I>E. coli</I> KRX expressing the fusion protein of CspB and mRFP with and without induction. A curve depicting KRX wildtype is shown for comparison. After induction at approximately 6 h the RFU to OD<SUB>600</SUB> ratio starts to rise in the induced culture. Compared to the uninduced culture the ratio is roughly five times higher. Most likely due to basal transcription the RFU to OD<SUB>600</SUB> ratio of the uninduced culture starts to rise after 12 hours. The KRX wildtype shows no variation in the RFU to OD<SUB>600</SUB> ratio.</B></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Identification_and_localization">Identification and localization</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 3: Fluorescence progression of the mRFP <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">(BBa_E1010)</A>/CspB fusion protein initiating with the cultivation fractions up to the detergent fractions of the separate denaturations. Cultivations were carried out in autoinduction medium at 37 ˚C. The cells were mechanically disrupted and the resulting biomass was washed with ddH<SUB>2</SUB>O and resuspended in the respective detergent. The used detergent acronyms stand for: SDS = 10 % sodium dodecyl sulfate; UTU = 7 M urea and 3 M thiourea; U = 10 M urea; NLS = 10 % n-lauroyl sarcosine; 2 % CHAPS = 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate.</B></DIV></DIV></DIV><P>After a cultivation time of 18 h the mRFP|CspB fusion protein was localized in <I>E. coli</I> KRX. Therefore a part of the produced biomass was mechanically disrupted and the resulting lysate was washed with ddH<SUB>2</SUB>O. Then the lysate was treated with ionic, nonionic and zwitterionic detergents to release the mRFP|CspB out of the membranes, if it integrates. The periplasm was detached of a part of the biomass by using an osmotic shock.
The fluorescence in all cultivation fractions, the fluorescence in the lysis and wash fraction show that the fusion protein is water soluble and does not sediment during centrifugation. Together with the absence of fluorescence in the detergent fractions this verifies that the fusion protein is not integrated into the cell membrane (Fig. 3) and is not forming inclusion bodies.
In comparison with the mRFP fusion protein of &lt;partinfo&gt;K525224&lt;/partinfo&gt;, which has a TAT-sequence, a minor relative fluorescence in all cultivation and detergent fractions was detected (Fig. 3). Together with the decreasing RFU after 14 h of cultivation (Fig. 2) this result indicates the postive effect of the TAT-sequence on the protein stability. This could be due to a digestion of &lt;partinfo&gt;K525222&lt;/partinfo&gt; by proteases in the cytoplasm. 
</P><P>
MALDI-TOF analysis was used to identify the location of the fusion protein in different fractions. Fractions of medium supernatant after cultivation, periplasmatic isolation, cell lysis and following denaturation in 6 M urea were loaded onto a SDS-PAGE. After denaturation with 6 M urea the remaining pellet (after centrifuging 15,000 <I>g</I> for 30 min) was washed with 2 % (v/v) Triton X-100, 2 % SDS (w/v). This fraction was also loaded onto the SDS-PAGE and fragments of the gel were measured with MALDI-TOF.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 4: SDS-PAGE of CspB/mRFP <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">(BBa_E1010)</A> fusion protein. Lanes are fractions of medium (M), periplasmatic isolation (PP), cell lysis (L), denaturation (D) and wash of the remaining pellet with Triton X-100 (T). Used marker is PageRuler <SUP>TM</SUP> Prestained Protein Ladder SM0671 from <A href="http://www.fermentas.com/en/products/all/protein-electrophoresis/prestained-ladders/sm067-pageruler-prestained-protein" class="external text" rel="nofollow">Fermentas</A>. Marked regions were cut out and prepared for MALDI TOF analysis.</B></DIV></DIV></DIV></DIV><P>The following table shows the sequence coverage (in %) of our measurable gel samples with the amino acid sequence of fusion protein CspB/mRFP <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">(BBa_E1010)</A>.
</P><CENTER><TABLE style="border-collapse: separate; border-spacing: 0; border-width: 1px; border-style: solid; border-color: #000; padding: 0"><TBODY><TR><TH style="border-style: solid; border-width: 0 1px 1px 0"> number of gel sample
</TH><TH style="border-style: solid; border-width: 0 0 1px 0"> sequence coverage (%)
</TH></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 1
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 2
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 3
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 4
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 5
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 6
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 7
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 8
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 9
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 10
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 11
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 12
</TD><TD style="border-style: solid; border-width: 0"> 14.6
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 13
</TD><TD style="border-style: solid; border-width: 0"> 9.3
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 14
</TD><TD style="border-style: solid; border-width: 0"> 6.3
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 15
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 16
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 17
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 18
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 19
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 20
</TD><TD style="border-style: solid; border-width: 0"> 1.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 21
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 22
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 23
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 24
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR><TR><TD style="border-style: solid; border-width: 0 1px 0 0"> 25
</TD><TD style="border-style: solid; border-width: 0"> 0.0
</TD></TR></TBODY></TABLE></CENTER><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 5: MALDI-TOF measurement of CspB/mRFP <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">(BBa_E1010)</A> fusion protein. Samples are arranged after estimated molecular mass of the gel slice. Measurement was performed with a ultrafleXtreme<SUP>TM</SUP> by Bruker Daltonics using the software FlexAnalysis, Biotools and SequenceEditor.</B></DIV></DIV></DIV><P>Fig. 5 shows these data. The gel samples were arranged after estimated molecular mass cut out from the gel. As expected, no sequence coverage was found in the periplasmatic fraction, due to absent TAT-sequence located at the amino-terminus. Little sequence coverage was found in the fraction of supernatant of the media, indicating that the protein can not be transported to the periplasm and thus secretion into the medium does not take place. The denaturation fraction and the Triton X-100 fraction show no or very few sequence coverage, however the lysis fraction shows significant higher sequence coverage. Both results indicate that the fusion protein is solely present in the cytoplasm and thus only identified in the lysis fraction. Fig. 5 and Fig. 6 show that the protein can be found mainly in the lysis fraction, but in smaller amounts in the periplasmatic and the media fraction as well, which can be explained due to the absence of the lipid anchor. The anchor normally binds to the cell membrane, so no protein is found in other fractions than the lysis fraction.
</P><P>To obtain more specific informations about the location of the S-layer fusion protein, after comparison with same treated fraction of <I>E. coli</I> KRX all gel bands in a defined size area were cut out of the gel and analysed with MALDI-TOF. Results are shown in Fig. 6.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 6: MALDI-TOF measurement of CspB/mRFP <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">(BBa_E1010)</A> fusion protein in different fractions. Abbreviations are Ma: Marker (PageRuler <SUP>TM</SUP> Prestained Protein Ladder SM0671), M (medium), PP (periplasm), L (cell lysis with ribolyser), W (wash with ddH<SUB>2</SUB>O). In the left half of the gel fractions of <I>E. coli</I> KRX with induced production of fusion protein, the right half shows fractions of <I>E. coli</I> KRX without carrying the plasmid coding the fusion protein. Colours show the sequence coverage of the gel lane, cutted out of the gel. Colours mark the sequence coverage measured with MALDI-TOF. fractions M of &lt;partinfo&gt;K525222&lt;/partinfo&gt; and <I>E. coli</I> KRX were switched and are therefore special named.</B></DIV></DIV></DIV></DIV><P>MALDI-TOF measurement shows sequence coverage in the supernatant fraction of the cultivation, the periplasmatic fraction and the lysis fraction, indicating that the fusion protein of &lt;partinfo&gt;K525222&lt;/partinfo&gt; and <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">BBa_E1010</A> without TAT-sequence and lipid anchor is not only found in the cytoplasm. The periplasmatic isolation does destroy a few complete cells, which could lead to the sequence coverage found in the periplasmatic fraction. The gel lanes in the periplasmatic fraction are clearly less intense, the protein concentration is very low compared to the lysis fraction. Sequence coverage in the media fraction shows, that the protein is probably secreted into the media. Cell lysis could lead to this effect. The sequence coverage in a MALDI-TOF analysis does not automatically correlate with protein concentrations. Small amounts of proteins can give the same results when cutted out of a clean protein band of a single protein. When looking at the intensity of the analysed gel pieces it can be summarized, that there is some CspB found in the periplasm and the culture supernatant but most of the protein stays in the cytoplasm which was the expected result. No fluorescence and sequence coverage was found in the cell membrane fractions, giving another proof that the lipid anchor from <I>Corynebacterium</I> binds to the cell membrane of <I>E. coli</I>.
</P><H2><SPAN class="mw-headline" id="CspB_without_TAT-sequence_and_with_lipid_anchor">CspB without TAT-sequence and with lipid anchor</SPAN></H2><H3><SPAN class="mw-headline" id="Cultivation_and_protein_expression_2">Cultivation and protein expression</SPAN></H3><P>For characterization the modified CspB <A href="http://partsregistry.org/Part:BBa_K525223" class="external text" rel="nofollow">(K525223)</A> gen was fused with a monomeric RFP (<A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">BBa_E1010</A>) using <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Gibson_assembly" class="external text" rel="nofollow">Gibson assembly</A>.
</P><P>The fusion protein was overexpressed in <I>E. coli</I> KRX after induction of T7 polymerase by supplementation of 0,1 % L-rhamnose using the <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing#Expression_of_S-layer_genes_in_E._coli" class="external text" rel="nofollow">auinduction protocol</A>.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 7: Growthcurve of <I>E. coli</I> KRX expressing the fusion protein of CspB and mRFP with and without induction, cultivated at 37 °C in autoinduction medium with, respectively, without inductor. A curve depicting KRX wildtype is shown for comparison. After induction at approximately 6 h the OD<SUB>600</SUB> of the induced K525223 visibly drops when compared to the uninduced culture. While the induced culture grow significantly slower than KRX wildtype the uninduced seems to be unaffected.</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 8: RFU to OD<SUB>600</SUB> ratio of <I>E. coli</I> KRX expressing the fusion protein of CspB and mRFP with and without induction. A curve depicting KRX wildtype is shown for comparison. After induction at approximately 6 h the RFU to OD<SUB>600</SUB> ratio starts to rise in the induced culture. Compared to the uninduced culture the ratio is roughly five times higher. Most likely due to basal transcription the RFU to OD<SUB>600</SUB> ratio of the uninduced culture starts to rise after 12 hours. The KRX wildtype shows no variation in the RFU to OD<SUB>600</SUB> ratio.</B></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Identification_and_localization_2">Identification and localization</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 9: Fluorescence progression of the mRFP<A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">(BBa_E1010)</A>/CspB fusion protein initiating with the cultivation fractions up to the detergent fractions of the separate denaturations. Cultivations were carried out in autoinduction medium at 37 ˚C. The cells were mechanically disrupted and the resulting biomass was washed with ddH<SUB>2</SUB>O and resuspended in the respective detergent. The used detergent acronyms stand for: SDS = 10 % sodium dodecyl sulfate; UTU = 7 M urea and 3 M thiourea; U = 10 M urea; NLS = 10 % n-lauroyl sarcosine; 2 % CHAPS = 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate.</B></DIV></DIV></DIV><P>After a cultivation time of 18 h the mRFP|CspB fusion protein was localized in <I>E. coli</I> KRX. Therefore a part of the produced biomass was mechanically disrupted and the resulting lysate was washed with ddH<SUB>2</SUB>O. From the other part the periplasm was detached by using an osmotic shock. 
The S-layer fusion protein could not be found in the polyacrylamide gel after a SDS-PAGE of the lysate. This indicated that the fusion protein integrates into the cell membrane with its lipid anchor. For testing this assumption the washed lysate was treated with ionic, nonionic and zwitterionic detergents to release the mRFP|CspB out of the membranes.
The existance of fluorescence in two of the detergent fractions (10 % SDS and 10 % N-lauroyl sarcosine) and the low fluorescence in the wash fraction confirm the hypothesis of an insertion into the cell membrane (Fig. 9). An insertion of these S-layer proteins might stabilize the membrane structure and increase the stability of cells against mechanical and chemical treatment. A stabilization of <I>E. coli</I> expressing S-layer proteins was described by <A href="http://www.ncbi.nlm.nih.gov/pubmed/20829284" class="external text" rel="nofollow">Lederer <I>et al.</I>, (2010)</A>.
An other important fact is, that there is actually mRFP fluorescence measurable in such high concentrated detergent solutions. The S-layer seems to stabilize the biologically active conformation of mRFP.
In comparison with the mRFP fusion protein of <A href="http://partsregistry.org/Part:BBa_K525224" class="external text" rel="nofollow">K525224</A>, which has a TAT-sequence, a minor relative fluorescence in all cultivation and detergent fractions was detected (Fig. 9). Together with the decreasing RFU after 9 h of cultivation (Fig. 7/ Fig. 8) this results indicate a postive effect of the TAT-sequence on the protein stability. This could be due to a digestion of &lt;partinfo&gt;K525223&lt;/partinfo&gt; by proteases in the cytoplasm. 
</P><P>To obtain more specific information about the location of the S-layer fusion protein, after comparison with same treated fraction of <I>E. coli</I> KRX all gel bands in a defined size area were cut out of the gel and analysed with MALDI-TOF. Results are shown in Fig. 10. The fusion protein CspB/mRFP <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">(BBa_E1010)</A> features a lipid anchor at the carboxy-terminus, but no amino-terminal TAT-sequence. In accordance with other protein variants with and without this features, the protein should be located mainly in the cytoplasm as inclusion bodies or incooperated with its lipid anchor into the cell membrane. Thus, the fraction with 10 % (v/v) SDS as detergent to disintegrate the protein from the cell wall was measured with MALDI-TOF. Results are shown in Fig. 10. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 10: MALDI-TOF measurement of CspB/mRFP <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">(BBa_E1010)</A> fusion protein in different fractions. Abbreviations are Ma: Marker (PageRuler <SUP>TM</SUP> Prestained Protein Ladder SM0671), M (medium), PP (periplasm), L (cell lysis with ribolyser), W (wash with ddH<SUB>2</SUB>O). In the left half of the gel fractions of <I>E. coli</I> KRX with induced production of fusion protein, the right half shows fractions of <I>E. coli</I> KRX without carrying the plasmid coding the fusion protein. Colours show the sequence coverage of the gel lane, cutted out of the gel.</B></DIV></DIV></DIV></DIV><P>Fig. 10 shows, that the protein could be identified in all measured gel bands. The results indicate, that the protein is incorperated into the cell membrane. No fluoresence could be detected in the fractions using urea as detergent (see Fig. 9), thus the protein probably does not form inclusion bodies. The fact that there is a comparable low overall fluorescence in the cells is a strong indication that the protein is degraded by <I>E. coli</I> proteases. The fluorescence in the periplasm and the supernatant is probably due to cell lysis during the periplasm isolation and the cultivation because the measured fluorescence values are comparable low, too.
</P><H2><SPAN class="mw-headline" id="CspB_with_TAT-sequence_and_without_lipid_anchor">CspB with TAT-sequence and without lipid anchor</SPAN></H2><H3><SPAN class="mw-headline" id="Cultivation_and_protein_expression_3">Cultivation and protein expression</SPAN></H3><P>For characterization the modiefied CspB <A href="http://partsregistry.org/Part:BBa_K525224" class="external text" rel="nofollow">(K525224)</A> gene was fused with a monomeric RFP (<A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">BBa_E1010</A>) using <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Gibson_assembly" class="external text" rel="nofollow">Gibson assembly</A>.
</P><P>The fusion protein was overexpressed in <I>E. coli</I> KRX after induction of T7 polymerase by supplementation of 0,1 % L-rhamnose using the <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing#Expression_of_S-layer_genes_in_E._coli" class="external text" rel="nofollow">auinduction protocol</A>.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 11: Growthcurve of <I>E. coli</I> KRX expressing the fusion protein of CspB and mRFP with and without induction, cultivated at 37 °C in autoinduction medium with, respectively, without inductor. A curve depicting KRX wildtype is shown for comparison. After induction at approximately 6 h the OD<SUB>600</SUB> of the induced K525224 visibly drops when compared to the uninduced culture. While the induced culture grow significantly slower than KRX wildtype the uninduced seems to be unaffected.</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 12: RFU to OD<SUB>600</SUB> ratio of <I>E. coli</I> KRX expressing the fusion protein of CspB and mRFP with and without induction. A curve depicting KRX wildtype is shown for comparison. After induction at approximately 6 h the RFU to OD<SUB>600</SUB> ratio starts to rise in the induced culture. Compared to the uninduced culture the ratio is roughly seven times higher at its highest point but starts to drop during the cultivation due to degradation of the fusion protein. Most likely due to basal transcription the RFU to OD<SUB>600</SUB> ratio of the uninduced culture starts to rise after 12 hours. The KRX wildtype shows no variation in the RFU to OD<SUB>600</SUB> ratio.</B></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Identification_and_localisation">Identification and localisation</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 13: Fluorescence progression of the mRFP <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">(BBa_E1010)</A>/CspB fusion protein initiating with the cultivation fractions up to the detergent fractions of the seperate denaturations. Cultivations were carried out in autoinduction medium at 37 ˚C. The cells were mechanically disrupted and the resulting biomass was washed with ddH<SUB>2</SUB>O and resuspended in the respective detergent. The used detergent acronyms stand for: SDS = 10 % sodium dodecyl sulfate; UTU = 7 M urea and 3 M thiourea; U = 10 M urea; NLS = 10 % n-lauroyl sarcosine; 2 % CHAPS = 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate.</B></DIV></DIV></DIV><P>After a cultivation time of 18 h the mRFP|CspB fusion protein was localized in <I>E. coli</I> KRX. Therefore a part of the produced biomass was mechanically disrupted and the resulting lysate was washed with ddH<SUB>2</SUB>O. Then the lysate was treated with ionic, nonionic and zwitterionic detergents to release the mRFP|CspB out of the membranes, if it integrates. From the other part of the cells the periplasm was detached by using an osmotic shock. The existance of fluorescene in the periplasm fraction, showed in Fig. 13, indicates that <I>C. halotolerans</I> TAT-signal sequence is at least in part functional in <I>E. coli</I> KRX.
</P><P>Specific for &lt;partinfo&gt;K525224&lt;/partinfo&gt; fused with mRFP is the proportional to the mRFP fusion proteins of <A href="http://partsregistry.org/Part:BBa_K525222" class="external text" rel="nofollow">K525222</A> and <A href="http://partsregistry.org/Part:BBa_K525223" class="external text" rel="nofollow">K525223</A> high fluorescence in the culture supernatant. This indicates that the fusion protein is secreted into the periplasm via the TAT-pathway and partly released into the culture medium. Because there is no known release pathway for S-layer proteins in <I>E. coli</I> the periplasm might burst in consequence of the overexpression. 
The fluorescence in all cultivation fractions plus the fluorescence in the lysis und wash fraction shows that the fusion protein is water soluble and does not sediment during centrifugation.
The absence of fluorescence indicates that the expressed fusion protein does not form inclusion bodies during cultivation.
</P><P>To obtain specific information about the location of the S-layer fusion protein, after comparison with same treated fraction of <I>E. coli</I> KRX all gel bands in a defined size area were cut out of the gel and analysed with MALDI-TOF. Results are shown in Fig. 14.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 14: MALDI-TOF measurement of CspB/mRFP <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">(BBa_E1010)</A> fusion protein in different fractions. Abbreviations are Ma: Marker (PageRuler <SUP>TM</SUP> Prestained Protein Ladder SM0671), M (medium), PP (periplasm), L (cell lysis with ribolyser), W (wash with ddH<SUB>2</SUB>O). In the left half of the gel fractions of <I>E. coli</I> KRX with induced production of fusion protein, the right half shows fractions of <I>E. coli</I> KRX without carrying the plasmid coding the fusion protein. Colours show the sequence coverage of the gel lane, cutted out of the gel.</B></DIV></DIV></DIV></DIV><P>Sequence coverage could be identified in the supernatant of the media, in the periplasmatic fraction and in the cell lysis fraction, indicating that the TAT-sequence is working and that the protein is furthermore secreted into the media. This covers with the fluorescence measurements of the different fractions. A high fluorescence has been measured in the supernatant of the media, which is a remarkable characteristic because normally <I>E. coli</I> is only limited capable for secretion.
</P><P>The influence of other detergents to disintegrate the S-layer fusion protein was tested after disrupting the cells with a ribolyser. The cell pellet was incubated in 10 % (v/v) Sodium dodecyl sulfate (SDS), in 7 M urea and 3 M thiourea (UTU), in 10 M urea (U) in 10 % (v/v) n-lauroyl sarcosine (NLS) and in 2 % CHAPS (C). Samples of the incubations with these detergents were loaded onto a SDS-PAGE prior to measurement with MALDI-TOF (see Fig. 15).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 15: Influence of diffent detergents on the disintegration of the fusionprotein CspB/mRFP <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">(BBa_E1010)</A> from the cell membrane of <I>E. coli</I> KRX. Abbreviations are: SDS (Sodium dodecyl sulfate 10 % (w/v)), NLS ((10 % (w/v) m-lauroyl sarcosine), UTU (3 M thiourea, 7 M urea), U (10 M urea), CH (2% CHAPS 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate), Ma Marker (PageRuler <SUP>TM</SUP> Prestained Protein Ladder SM0671). On the left half of the gel fractions of the S-layer fusion protein are displayed, on the right half fractions of <I>E. coli</I> KRX are displayed.</B></DIV></DIV></DIV></DIV><P>Altough most of the protein should be transported to the periplasm or secreted proteins should be found in the detergent fractions. So the SDS fraction was measured with MALDI TOF, because SDS is able to solubilize nearly every protein. In the fractions of SDS sequence coverage over 20 % could be identified, indicating that a considerable amount of the fusion protein is located in the cytoplasm.
</P><H3><SPAN class="mw-headline" id="Purification">Purification</SPAN></H3><P>After the localisation of the S-layer protein in <I>E. coli</I>, different methods for purification were tested. The results of these methods are shown in Fig. 16. The CspB protein does not form inclusion bodies in <I>E. coli</I> and most of the protein is transported out of the cell into the periplasm and a lot of protein is even secreted into the medium (all fractions were concentrated by filtration and precipitation, respectively). The secretion into the culture medium is very interesting because the purification is much faster (no cell disruption necessary). 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Figure 16: Fluorescence of collected fractions of different methods to release and concentrate &lt;partinfo&gt;K525234&lt;/partinfo&gt; protein from a cultivation in <I>E. coli</I>. </B></DIV></DIV></DIV></DIV><P>The highest fluorescence could be obtained by a precipitation with ammonium sulfate of the culture supernatant followed by an ultrafiltration with a 300 kDa membrane and a diafiltration with a 50 kDa membrane. The diafiltration was against a binding buffer for an anion exchange chromatography (25 mM sodium acetate, 25 mM sodium chloride) with pH 6, due to the theoretical pI of &lt;partinfo&gt;k525234&lt;/partinfo&gt;. The fluorescence of the collected fractions of the following anion exchange chromatography are shown in Fig. 17. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Figure 17: Fluorescence of collected fractions of an anion exchange chromatography of &lt;partinfo&gt;K525234&lt;/partinfo&gt; after concentration from the culture supernatant. </B></DIV></DIV></DIV></DIV><P>The binding conditions are well chosen because nearly all of the protein binds to the column. The protein is eluted from the column with rising sodium chloride concentrations. The highest fluorescence is in the elution fraction with 400 mM sodium chloride.  600 mM sodium chloride elutes all of the S-layer fusion proteins.
</P><H4><SPAN class="mw-headline" id="Final_purification_strategy">Final purification strategy</SPAN></H4><P>Scheme of purification strategy for CspB (fusion) proteins without lipid anchor:
</P><P>First, CspB is expressed in <I>E. coli</I> under the control of a T7 promoter for separation of growth and production phase due to metabolic stress of the S-layer expression. Because the CspB protein with TAT-sequence and without lipid anchor is secreted to the culture medium in appreciable amounts by <I>E. coli</I>, an ammonium sulfate precipitation of the culture supernatant follows the cultivation. The S-layers are further concentrated and purified by two ultrafiltration / diafiltration steps (300 kDa and 100 kDa) with anion exchange chromatography binding buffer. The permeate of the last filtration is used for an anion exchange chromatography for capture and purification of the S-layer protein. 
</P><P><A href="/Team:Bielefeld-Germany/Protocols/Downstream-processing#Fusion_proteins_of_CspB_without_lipid_anchor_with_TAT-sequence" title="Team:Bielefeld-Germany/Protocols/Downstream-processing"> Click for detailed information</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer/CspB_CH">http://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer/CspB_CH</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Results/S-Layer/CspB_CH" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Results/S-Layer/CspB_CH" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/S-Layer/CspB_CH&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/S-Layer/CspB_CH&amp;oldid=261707" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>