<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Backup-Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Backup-Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Backup-Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Backup-Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Backup-Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Backup-Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Backup-Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu-igem"><UL id="menu" class="sf-menu"><LI><A style="border-left:0px;" href="https://2011.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Students">Students</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Advisors">Supervisors</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Photos">Photos</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Bielefeld-University‎‎">Bielefeld University</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></UL><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Description">Description</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Achievements">Achievements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Appoach">Approach</A></LI><LI><A href="#">Background</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/Technology">Technology</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer/Guide">DIY Nanobiotech Guide</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Future">Future Applications</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Labjournal">Labjournal</A></LI><LI><A href="#">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Genetics">Molecular Genetics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing">Production</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Analytics">Analytics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials">Materials</A></LI><LI><A href="#">Results</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Data_Page">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/SinceAmsterdam">Since Regionals</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Modell">Model</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Cell-free">Cell-free Biosafety</A></LI><LI><A href="#">Outreach</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_lab_course">Open Lab Course</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_science_cafe">Science Cafe</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/high-school">High School</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/cebitec-symposium">CeBiTec Symposium</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/student-convention">Student Convention</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/strategic-process">Strategic Process</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Black-box">Beacon Box</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Press">Press</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Transformation_via_electroporation"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Transformation via electroporation</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Transformation_of_Single_Step_.28KRX.29_Competent_Cells_by_Promega"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Transformation of Single Step (KRX) Competent Cells by Promega</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Standard_BioBrick_Assembly"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Standard BioBrick Assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Suffix_Insertion"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Suffix Insertion</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Prefix_Insertion"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Prefix Insertion</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Variations"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Variations</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#Standard_Freiburg_BioBrick_Assembly"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Standard Freiburg BioBrick Assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Suffix_Insertion_2"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Suffix Insertion</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Prefix_Insertion_2"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Prefix Insertion</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Variations_2"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Variations</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Standard_3A_assembly"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Standard 3A assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Digestion"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Ligation"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Freiburg_3A_assembly"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Freiburg 3A assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Digestion_2"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Ligation_2"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Gibson_assembly"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Gibson assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Preparation_of_DNA_molecules_for_in_vitro_recombination"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Preparation of DNA molecules for in vitro recombination</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#In_vitro_recombination"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">In vitro recombination</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Restriction_analysis"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Restriction analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#Colony_PCR"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Cultivation"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Cultivation</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Expression_of_different_S-layer_genes_in_E._coli"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Expression of different S-layer genes in E. coli</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Expression_of_bisphenol_A_degrading_BioBricks_in_E._coli"><SPAN class="tocnumber">10.2</SPAN><SPAN class="toctext">Expression of bisphenol A degrading BioBricks in E. coli</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Bioreactor_cultivations_with_E._coli_KRX"><SPAN class="tocnumber">10.3</SPAN><SPAN class="toctext">Bioreactor cultivations with E. coli KRX</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#Purification_methods"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Purification methods</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Release_of_periplasmic_protein_fraction_from_E._coli_by_cold_osmotic_shock"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Release of periplasmic protein fraction from E. coli by cold osmotic shock</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Inclusion_body_clean-up"><SPAN class="tocnumber">11.2</SPAN><SPAN class="toctext">Inclusion body clean-up</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Ultra-.2FDiafiltration"><SPAN class="tocnumber">11.3</SPAN><SPAN class="toctext">Ultra-/Diafiltration</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Ion_exchange_chromatography_.28IEX.29_for_S-layer_proteins_from_Corynebacterium"><SPAN class="tocnumber">11.4</SPAN><SPAN class="toctext">Ion exchange chromatography (IEX) for S-layer proteins from Corynebacterium</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Ni-NTA_spin_columns"><SPAN class="tocnumber">11.5</SPAN><SPAN class="toctext">Ni-NTA spin columns</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#His-tag_affinity_chromatography"><SPAN class="tocnumber">11.6</SPAN><SPAN class="toctext">His-tag affinity chromatography</SPAN></A></LI><LI class="toclevel-1 tocsection-33"><A href="#Measuring_of_mRFP"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Measuring of mRFP</SPAN></A></LI><LI class="toclevel-1 tocsection-34"><A href="#Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE)</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#Pouring_the_polyacrylamide_gel"><SPAN class="tocnumber">13.1</SPAN><SPAN class="toctext">Pouring the polyacrylamide gel</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#Preparing_the_sample"><SPAN class="tocnumber">13.2</SPAN><SPAN class="toctext">Preparing the sample</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#Running_the_gel"><SPAN class="tocnumber">13.3</SPAN><SPAN class="toctext">Running the gel</SPAN></A></LI><LI class="toclevel-1 tocsection-38"><A href="#Polyacrylamide_gel_staining"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Polyacrylamide gel staining</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#Colloidal_Coomassie_Blue_staining"><SPAN class="tocnumber">14.1</SPAN><SPAN class="toctext">Colloidal Coomassie Blue staining</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#Fairbanks_Coomassie_staining"><SPAN class="tocnumber">14.2</SPAN><SPAN class="toctext">Fairbanks Coomassie staining</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#Silver_staining"><SPAN class="tocnumber">14.3</SPAN><SPAN class="toctext">Silver staining</SPAN></A></LI><LI class="toclevel-1 tocsection-42"><A href="#Tryptic_digest_of_gel_lanes_for_analysis_with_MALDI-TOF"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">Tryptic digest of gel lanes for analysis with MALDI-TOF</SPAN></A></LI><LI class="toclevel-1 tocsection-43"><A href="#Preparation_and_Spotting_for_analysis_of_peptides_on_Bruker_AnchorChips"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Preparation and Spotting for analysis of peptides on Bruker AnchorChips</SPAN></A></LI><LI class="toclevel-1 tocsection-44"><A href="#Bisphenol_A_analysis"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">Bisphenol A analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-45"><A href="#Extraction_with_ethylacetate"><SPAN class="tocnumber">17.1</SPAN><SPAN class="toctext">Extraction with ethylacetate</SPAN></A></LI><LI class="toclevel-2 tocsection-46"><A href="#HPLC_method"><SPAN class="tocnumber">17.2</SPAN><SPAN class="toctext">HPLC method</SPAN></A></LI><LI class="toclevel-1 tocsection-47"><A href="#NAD.2B_detection"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">NAD<SUP>+</SUP> detection</SPAN></A></LI><LI class="toclevel-2 tocsection-48"><A href="#Characterisation_of_molecular_beacons"><SPAN class="tocnumber">18.1</SPAN><SPAN class="toctext">Characterisation of molecular beacons</SPAN></A></LI><LI class="toclevel-2 tocsection-49"><A href="#Purification_of_DNA_ligase"><SPAN class="tocnumber">18.2</SPAN><SPAN class="toctext">Purification of DNA ligase</SPAN></A></LI><LI class="toclevel-2 tocsection-50"><A href="#Deadenylation"><SPAN class="tocnumber">18.3</SPAN><SPAN class="toctext">Deadenylation</SPAN></A></LI><LI class="toclevel-2 tocsection-51"><A href="#NAD.2B_bioassay"><SPAN class="tocnumber">18.4</SPAN><SPAN class="toctext">NAD<SUP>+</SUP> bioassay</SPAN></A></LI><LI class="toclevel-1 tocsection-52"><A href="#Used_enzymes"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">Used enzymes</SPAN></A></LI><LI class="toclevel-1 tocsection-53"><A href="#Used_Kits"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">Used Kits</SPAN></A></LI><LI class="toclevel-1 tocsection-54"><A href="#Media.2C_buffer.2C_solutions_etc."><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">Media, buffer, solutions etc.</SPAN></A></LI><LI class="toclevel-2 tocsection-55"><A href="#Ampicillin_stock_solution"><SPAN class="tocnumber">21.1</SPAN><SPAN class="toctext">Ampicillin stock solution</SPAN></A></LI><LI class="toclevel-2 tocsection-56"><A href="#Chloramphenicol_stock_solution"><SPAN class="tocnumber">21.2</SPAN><SPAN class="toctext">Chloramphenicol stock solution</SPAN></A></LI><LI class="toclevel-2 tocsection-57"><A href="#TAE_buffer"><SPAN class="tocnumber">21.3</SPAN><SPAN class="toctext">TAE buffer</SPAN></A></LI><LI class="toclevel-2 tocsection-58"><A href="#DNA_loading_buffer"><SPAN class="tocnumber">21.4</SPAN><SPAN class="toctext">DNA loading buffer</SPAN></A></LI><LI class="toclevel-2 tocsection-59"><A href="#LB_medium"><SPAN class="tocnumber">21.5</SPAN><SPAN class="toctext">LB medium</SPAN></A></LI><LI class="toclevel-2 tocsection-60"><A href="#Autoinduction_medium_for_KRX"><SPAN class="tocnumber">21.6</SPAN><SPAN class="toctext">Autoinduction medium for KRX</SPAN></A></LI><LI class="toclevel-2 tocsection-61"><A href="#M9_medium"><SPAN class="tocnumber">21.7</SPAN><SPAN class="toctext">M9 medium</SPAN></A></LI><LI class="toclevel-2 tocsection-62"><A href="#HSG_medium"><SPAN class="tocnumber">21.8</SPAN><SPAN class="toctext">HSG medium</SPAN></A></LI><LI class="toclevel-2 tocsection-63"><A href="#5x_isothermal_reaction_buffer_for_Gibson_assembly"><SPAN class="tocnumber">21.9</SPAN><SPAN class="toctext">5x isothermal reaction buffer for Gibson assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-64"><A href="#Cold_osmotic_shock_buffers_for_the_release_of_periplasmic_protein_fraction"><SPAN class="tocnumber">21.10</SPAN><SPAN class="toctext">Cold osmotic shock buffers for the release of periplasmic protein fraction</SPAN></A></LI><LI class="toclevel-2 tocsection-65"><A href="#Denaturation_buffer_for_inclusion_bodies"><SPAN class="tocnumber">21.11</SPAN><SPAN class="toctext">Denaturation buffer for inclusion bodies</SPAN></A></LI><LI class="toclevel-2 tocsection-66"><A href="#Buffers_for_S-layer_IEX"><SPAN class="tocnumber">21.12</SPAN><SPAN class="toctext">Buffers for S-layer IEX</SPAN></A></LI><LI class="toclevel-2 tocsection-67"><A href="#SDS-PAGE_gel"><SPAN class="tocnumber">21.13</SPAN><SPAN class="toctext">SDS-PAGE gel</SPAN></A></LI><LI class="toclevel-2 tocsection-68"><A href="#SDS_running_buffer"><SPAN class="tocnumber">21.14</SPAN><SPAN class="toctext">SDS running buffer</SPAN></A></LI><LI class="toclevel-2 tocsection-69"><A href="#4x_Laemmli-buffer"><SPAN class="tocnumber">21.15</SPAN><SPAN class="toctext">4x Laemmli-buffer</SPAN></A></LI><LI class="toclevel-2 tocsection-70"><A href="#Colloidal_Coomassie_Brilliant_Blue_G-250_staining_solution"><SPAN class="tocnumber">21.16</SPAN><SPAN class="toctext">Colloidal Coomassie Brilliant Blue G-250 staining solution</SPAN></A></LI><LI class="toclevel-2 tocsection-71"><A href="#Fairbanks_Coomassie_staining_solutions"><SPAN class="tocnumber">21.17</SPAN><SPAN class="toctext">Fairbanks Coomassie staining solutions</SPAN></A></LI><LI class="toclevel-2 tocsection-72"><A href="#Silver_staining_solutions"><SPAN class="tocnumber">21.18</SPAN><SPAN class="toctext">Silver staining solutions</SPAN></A></LI><LI class="toclevel-2 tocsection-73"><A href="#Binding_buffer"><SPAN class="tocnumber">21.19</SPAN><SPAN class="toctext">Binding buffer</SPAN></A></LI><LI class="toclevel-2 tocsection-74"><A href="#NPI-10_.28lysis_buffer.29"><SPAN class="tocnumber">21.20</SPAN><SPAN class="toctext">NPI-10 (lysis buffer)</SPAN></A></LI><LI class="toclevel-2 tocsection-75"><A href="#NPI-20_.28wash_buffer.29"><SPAN class="tocnumber">21.21</SPAN><SPAN class="toctext">NPI-20 (wash buffer)</SPAN></A></LI><LI class="toclevel-2 tocsection-76"><A href="#NPI-500_.28elution_buffer.29"><SPAN class="tocnumber">21.22</SPAN><SPAN class="toctext">NPI-500 (elution buffer)</SPAN></A></LI><LI class="toclevel-2 tocsection-77"><A href="#Deadenylation_buffer"><SPAN class="tocnumber">21.23</SPAN><SPAN class="toctext">Deadenylation buffer</SPAN></A></LI><LI class="toclevel-2 tocsection-78"><A href="#DNA_ligase_buffer"><SPAN class="tocnumber">21.24</SPAN><SPAN class="toctext">DNA ligase buffer</SPAN></A></LI><LI class="toclevel-2 tocsection-79"><A href="#NAD.2B_bioassay_buffer"><SPAN class="tocnumber">21.25</SPAN><SPAN class="toctext">NAD<SUP>+</SUP> bioassay buffer</SPAN></A></LI><LI class="toclevel-2 tocsection-80"><A href="#HCCA_matrix_solution"><SPAN class="tocnumber">21.26</SPAN><SPAN class="toctext">HCCA matrix solution</SPAN></A></LI><LI class="toclevel-1 tocsection-81"><A href="#Used_chemicals"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">Used chemicals</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Transformation_via_electroporation"> Transformation via electroporation </SPAN></H2><UL><LI> Thaw 50 µL competent <I>E.coli</I> cells on ice, dilute with icecold 50 µL glycerol (10 %) if necessary
</LI><LI> Add 0.5-5 µL plasmid to 50 µl electrocompetent cells
</LI><LI> Store cells on ice for 1 minute 
</LI><LI> Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ώ 
</LI><LI> Transfer transformation reaction to 450 µL SOC-Medium and shake 1 h at 37 °C 
</LI><LI> Centrifuge 2 min at 800 rpm and plate on selective LB-Medium
</LI></UL><H2><SPAN class="mw-headline" id="Transformation_of_Single_Step_.28KRX.29_Competent_Cells_by_Promega">Transformation of Single Step (KRX) Competent Cells by Promega</SPAN></H2><P>using <A href="http://www.promega.com/~/media/Files/Resources/Protocols/Technical%20Bulletins/101/Single%20Step%20Competent%20Cells%20Protocol.ashx" class="external text" rel="nofollow">protocol for <I>E. coli</I> KRX single step competent cells by Promega</A></P><UL><LI> Remove Single Step (KRX) Competent Cells from –70 °C, and place on ice for 5 minutes or until just thawed.
</LI><LI> Add 1–50 ng of DNA (in a volume not greater than 5 μL) to the Single Step (KRX) Competent Cells. Move the pipette tip through the cells while dispensing. Quickly flick the tube several times. Do not vortex!
</LI><LI> Immediately return the tubes to ice for 5–30 minutes
</LI><LI> Heat-shock cells for 15–20 seconds in a water bath at exactly 42 °C. Do not shake.
</LI><LI> Immediately place the tubes on ice for 2 minutes.
</LI><LI> Add 450 μL of room-temperature SOC medium to each transformation reaction, and incubate for 60 minutes at 37 °C with shaking (approximately 225 rpm). For best transformation efficiency, lay the tubes on their sides and tape them to the platform.
</LI><LI> For each transformation reaction, we recommend plating 100 μL of undiluted cells and 1:10 and 1:100 cell dilutions on antibiotic plates. Incubate the plates at 37 °C overnight.
</LI></UL><H2><SPAN class="mw-headline" id="Standard_BioBrick_Assembly"> Standard BioBrick Assembly </SPAN></H2><P>modified from <A href="http://openwetware.org/wiki/Silver:_BB_Strategy" class="external text" rel="nofollow">Silver lab</A>:
</P><P>This assembly method can be used for BioBricks which are bigger than 150 bp. The BioBrick should be at least 500 bp bigger or smaller than the backbone. The BioBrick, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other used BioBrick, called vector. So you have to differentiate between a prefix and a suffix insertion.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Silver Suffix Insertion</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Silver Prefix Insertion</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Suffix_Insertion"> Suffix Insertion </SPAN></H3><UL><LI> Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x Tango buffer, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> XbaI</A>, 1 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> PstI</A>. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI></UL><UL><LI> Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> SpeI</A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> PstI</A>. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> SAP (shrimp alcaline phosphatase)</A> and 1.2 µL 10 x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a <A href="/Team:Bielefeld-Germany/Protocols#Used_kits" title="Team:Bielefeld-Germany/Protocols"> PCR clean-up kit</A>. 
</LI></UL><UL><LI> Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> T4-Ligase</A>. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
</LI></UL><H3><SPAN class="mw-headline" id="Prefix_Insertion"> Prefix Insertion </SPAN></H3><UL><LI> Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x BamHI buffer, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> EcoRI</A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> SpeI</A>. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI></UL><UL><LI> Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10 x Tango buffer, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> EcoRI</A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> XbaI</A>. Digest for 2h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> SAP (shrimp alcaline phosphatase)</A> and 1.2 µL 10 x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a <A href="/Team:Bielefeld-Germany/Protocols#Used_kits" title="Team:Bielefeld-Germany/Protocols"> PCR clean-up kit</A>. 
</LI></UL><UL><LI> Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> T4-Ligase</A>. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
</LI></UL><H3><SPAN class="mw-headline" id="Variations"> Variations </SPAN></H3><UL><LI> A digestion over night is possible. If you digest over night use only 0.1 µL restriction enzyme. 
</LI></UL><UL><LI> It is also possible to use PCR product as insert. Digest after PCR with corresponding restriction enzymes and clean up with PCR clean-up kit. This could lead to higher yields of insert DNA because a lot of DNA gets lost during the gel electrophoresis clean up. 
</LI></UL><UL><LI> Sometimes some BioBricks are hard to assemble. Then you have to clean up the vector by gel electrophoresis as well.
</LI></UL><H2><SPAN class="mw-headline" id="Standard_Freiburg_BioBrick_Assembly"> Standard Freiburg BioBrick Assembly </SPAN></H2><P>Modified from <A href="http://openwetware.org/wiki/Silver:_BB_Strategy" class="external text" rel="nofollow">Silver lab</A> and <A href="http://partsregistry.org/Assembly_standard_25" class="external text" rel="nofollow">Assembly standard 25</A>:
</P><P>This assembly method can be used for fusion protein assemblies with BioBricks which are bigger than 150 bp. The BioBrick should be at least 500 bp bigger or smaller than the backbone. The BioBrick, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other used BioBrick, which is called vector and needs to be available in the BioBrick <A href="http://partsregistry.org/Assembly_standard_25" class="external text" rel="nofollow">Assembly standard 25</A>. You have to differentiate between a prefix and a suffix insertion.
</P><H3><SPAN class="mw-headline" id="Suffix_Insertion_2"> Suffix Insertion </SPAN></H3><UL><LI> Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x NEB buffer 4 + 0.1 µL 100x BSA, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> NgoMIV (NEB)</A>, 1 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> PstI</A>. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI></UL><UL><LI> Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> AgeI</A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> PstI</A>. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> SAP (shrimp alcaline phosphatase)</A> and 1.2 µL 10x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a <A href="/Team:Bielefeld-Germany/Protocols#Used_kits" title="Team:Bielefeld-Germany/Protocols"> PCR clean-up kit</A>. 
</LI></UL><UL><LI> Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> T4-Ligase</A>. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
</LI></UL><H3><SPAN class="mw-headline" id="Prefix_Insertion_2"> Prefix Insertion </SPAN></H3><UL><LI> Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> EcoRI</A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> AgeI</A>. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI></UL><UL><LI> Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10 x NEB buffer 4, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> EcoRI</A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> NgoMIV (NEB)</A>. Digest for 2h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> SAP (shrimp alcaline phosphatase)</A> and 1.2 µL 10 x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a <A href="/Team:Bielefeld-Germany/Protocols#Used_kits" title="Team:Bielefeld-Germany/Protocols"> PCR clean-up kit</A>. 
</LI></UL><UL><LI> Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> T4-Ligase</A>. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
</LI></UL><H3><SPAN class="mw-headline" id="Variations_2"> Variations </SPAN></H3><UL><LI> A digestion over night is possible. If you digest over night use only 0.1 µL restriction enzyme. 
</LI></UL><UL><LI> It is also possible to use PCR product as insert. Digest after PCR with corresponding restriction enzymes and clean up with <A href="/Team:Bielefeld-Germany/Protocols#Used_kits" title="Team:Bielefeld-Germany/Protocols"> PCR clean-up kit</A>. This could lead to higher yields of insert DNA because a lot of DNA gets lost during the gel electrophoresis clean up. 
</LI></UL><UL><LI> Sometimes some BioBricks are hard to assemble. Then you have to clean up the vector by gel electrophoresis as well.
</LI></UL><H2><SPAN class="mw-headline" id="Standard_3A_assembly">Standard 3A assembly</SPAN></H2><P>Modified from <A href="http://ginkgobioworks.com/support/BioBrick_Assembly_Manual.pdf" class="external text" rel="nofollow">BioBrick Assembly Manual by Ginkgo BioWorks</A></P><H3><SPAN class="mw-headline" id="Digestion">Digestion</SPAN></H3><UL><LI> Thaw DNA from upstream and downstream part and the destination plasmid on ice. 
</LI><LI>  Destination plasmid has to carry the <I>ccdB</I> gene &lt;partinfo&gt;P1010&lt;/partinfo&gt; as insert and has to have a different antibiotic resistance than the plasmids carrying the upstream and downstream parts
</LI><LI>  DNA has to be cleaned (by <A href="/Team:Bielefeld-Germany/Protocols#Used_kits" title="Team:Bielefeld-Germany/Protocols"> MiniPrep or after a PCR</A>)
</LI></UL><UL><LI> 500 ng DNA / digestion mix for upstream part, downstream part and destination plasmid (total volume of mix 10 µL, dilute with ddH<SUB>2</SUB>0 if necessary)
</LI></UL><UL><LI> Add 1 µL of 10x buffer and <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> restriction enzymes</A> as shown in the following table: 
</LI></UL><CENTER><TABLE cellpadding="10" style="border-collapse: collapse; border-width: 1px; border-style: solid; border-color: #000"><TBODY><TR><TH style="border-style: solid; border-width: 1px"> Upstream part
</TH><TH style="border-style: solid; border-width: 1px"> Downstream part
</TH><TH style="border-style: solid; border-width: 1px"> Destination plasmid
</TH></TR><TR><TD style="border-style: solid; border-width: 1px"> enzyme 1
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL EcoRI
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL XbaI
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL EcoRI
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> enzyme 2
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL SpeI
</TD><TD style="border-style: solid; border-width: 1px"> 1 µL PstI
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL PstI
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> buffer
</TD><TD style="border-style: solid; border-width: 1px"> BamHI
</TD><TD style="border-style: solid; border-width: 1px"> Tango
</TD><TD style="border-style: solid; border-width: 1px"> Orange
</TD></TR></TBODY></TABLE></CENTER><UL><LI> Incubation of the digestion mixes at 37 °C
</LI></UL><UL><LI> After 2 h: add 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> SAP (shrimp alcaline phosphatase)</A> and 1.15 µL 10x SAP buffer to destination plasmid mix, continue incubation at 37 °C
</LI></UL><UL><LI> After another hour: heat inactivation of all mixes for 20 min at 80 °C
</LI></UL><UL><LI> Continue with ligation or freeze the mixes
</LI></UL><H3><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H3><UL><LI> Ligation mix: 
</LI><LI> 2 µL ddH<SUB>2</SUB>O
</LI><LI> 5 µL of every digestion mix (so 15 µL in total)
</LI><LI> 2 µL T4-DNA-ligase buffer (thaw on ice!)
</LI><LI> 1 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> T4-DNA-ligase</A></LI></UL><UL><LI> Incubate at least 20 min at room temperature, afterwards heat inactivation for 5 min at 70 °C (optional)
</LI></UL><UL><LI> Freeze ligation mix or continue with transformation (heatshock or electroporation)
</LI></UL><H2><SPAN class="mw-headline" id="Freiburg_3A_assembly">Freiburg 3A assembly</SPAN></H2><P>Modified from <A href="http://ginkgobioworks.com/support/BioBrick_Assembly_Manual.pdf" class="external text" rel="nofollow">BioBrick Assembly Manual by Ginkgo BioWorks</A> and <A href="http://partsregistry.org/Assembly_standard_25" class="external text" rel="nofollow">Assembly standard 25</A></P><H3><SPAN class="mw-headline" id="Digestion_2">Digestion</SPAN></H3><UL><LI> Thaw DNA from upstream and downstream part (=N-terminal and C-terminal protein domain) and the destination plasmid on ice. 
</LI><LI>  Destination plasmid has to carry the <I>ccdB</I> gene &lt;partinfo&gt;P1010&lt;/partinfo&gt; as insert and has to have a different antibiotic resistance than the plasmids carrying the upstream and downstream parts
</LI><LI>  DNA has to be cleaned (by <A href="/Team:Bielefeld-Germany/Protocols#Used_kits" title="Team:Bielefeld-Germany/Protocols"> MiniPrep or after a PCR</A>)
</LI></UL><UL><LI> 500 ng DNA / digestion mix for upstream part, downstream part and destination plasmid (total volume of mix 10 µL, dilute with ddH<SUB>2</SUB>0 if necessary)
</LI></UL><UL><LI> Add 1 µL of 10x buffer and <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> restriction enzymes</A> as shown in the following table: 
</LI></UL><CENTER><TABLE cellpadding="10" style="border-collapse: collapse; border-width: 1px; border-style: solid; border-color: #000"><TBODY><TR><TH style="border-style: solid; border-width: 1px"> Upstream part
</TH><TH style="border-style: solid; border-width: 1px"> Downstream part
</TH><TH style="border-style: solid; border-width: 1px"> Destination plasmid
</TH></TR><TR><TD style="border-style: solid; border-width: 1px"> enzyme 1
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL EcoRI
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL NgoMIV
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL EcoRI
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> enzyme 2
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL AgeI
</TD><TD style="border-style: solid; border-width: 1px"> 1 µL PstI
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL PstI
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> buffer
</TD><TD style="border-style: solid; border-width: 1px"> Orange
</TD><TD style="border-style: solid; border-width: 1px"> NEB buffer 4 + BSA
</TD><TD style="border-style: solid; border-width: 1px"> Orange
</TD></TR></TBODY></TABLE></CENTER><UL><LI> Incubation of the digestion mixes at 37 °C
</LI></UL><UL><LI> After 2 h: add 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> SAP (shrimp alcaline phosphatase)</A> and 1.15 µL 10x SAP buffer to destination plasmid mix, continue incubation at 37 °C
</LI></UL><UL><LI> After another hour: heat inactivation of all mixes for 20 min at 80 °C
</LI></UL><UL><LI> Continue with ligation or freeze the mixes
</LI></UL><H3><SPAN class="mw-headline" id="Ligation_2">Ligation</SPAN></H3><UL><LI> Ligation mix: 
</LI><LI> 2 µL ddH<SUB>2</SUB>O
</LI><LI> 5 µL of every digestion mix (so 15 µL in total)
</LI><LI> 2 µL T4-DNA-ligase buffer (thaw on ice!)
</LI><LI> 1 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> T4-DNA-ligase</A></LI></UL><UL><LI> Incubate at least 20 min at room temperature, afterwards heat inactivation for 5 min at 70 °C (optional)
</LI></UL><UL><LI> Freeze ligation mix or continue with transformation (heatshock or electroporation)
</LI></UL><H2><SPAN class="mw-headline" id="Gibson_assembly"> Gibson assembly </SPAN></H2><P>Modified from <A href="http://www.nature.com/nmeth/journal/v6/n5/full/nmeth.1318.html" class="external text" rel="nofollow">Gibson <I>et al.</I> (2009)</A></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Gibson Assembly</DIV></DIV></DIV><P>This assembly method is an isothermal, single-reaction method for assembling multiple overlapping DNA molecules. By coordinating the activity of a 5‘ exonuclease, a DNA polymerase and a DNA ligase two adjacent DNA fragments with complementary terminal sequence overlaps can be joined into a covalently sealed molecule, without the use of any restriction endonuclease.
</P><H3><SPAN class="mw-headline" id="Preparation_of_DNA_molecules_for_in_vitro_recombination">Preparation of DNA molecules for <I>in vitro</I> recombination</SPAN></H3><UL><LI> Generate the complementary sequence overlaps by PCR using the <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Used_enzymes" class="external text" rel="nofollow">Phusion DNA-polymerase</A>. If necessary add 5 M Betain in the reaction mix by reducing the amount of H<SUB>2</SUB>O to decrease the number of false PCR products.
</LI></UL><UL><LI> Identify the PCR products of interest by gel electrophoresis with known DNA standards.
</LI></UL><UL><LI> Extract the PCR products from the gel by cutting out the DNA fragments and clean them up by using a commercial <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Used_Kits" class="external text" rel="nofollow">clean up kit</A>.
</LI></UL><H3><SPAN class="mw-headline" id="In_vitro_recombination"><I>In vitro</I> recombination</SPAN></H3><UL><LI> assembly mixture (6 mL):
</LI><LI> 320 µL <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#5x_isothermal_reaction_buffer_for_Gibson_assembly" class="external text" rel="nofollow">5x isothermal reaction buffer</A></LI><LI> 0.64 µL of 10 U mL<SUP>-1</SUP><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Used_enzymes" class="external text" rel="nofollow">T5 exonuclease</A> (for DNA molecules overlapping by greater than 150 bp add 3.2 µL of 10 U ml<SUP>–1</SUP><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Used_enzymes" class="external text" rel="nofollow">T5 exonuclease</A>)
</LI><LI> 20 µL of 2 U mL<SUP>-1</SUP><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Used_enzymes" class="external text" rel="nofollow">Phusion DNA polymerase</A></LI><LI> 160 µL of 40 U mL<SUP>-1</SUP><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Used_enzymes" class="external text" rel="nofollow"><I>taq</I> DNA ligase</A></LI><LI> add ddH<SUB>2</SUB>O water up to a final volume of 1.2 mL
</LI><LI> aliquote 15 µL of the reagent-enzyme mix and store it at –20 ˚C
</LI></UL><UL><LI> Thaw 15 µL assembly mixture aliquot and keep it on ice until use.
</LI></UL><UL><LI> Add 5 µL of the purified DNA molecules in equimolar amounts (between 10 and 100 ng of each DNA fragment).
</LI></UL><UL><LI> Incubate the resulting mixture at 50 ˚C for 15 to 60 min, with 60 min being optimal.
</LI></UL><UL><LI> Transformation (heatshock or <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Transformation_via_electroporation" class="external text" rel="nofollow">electroporation</A>) without cleaning up the assembly product.
</LI></UL><H2><SPAN class="mw-headline" id="Restriction_analysis"> Restriction analysis </SPAN></H2><UL><LI> Digest BioBrick of interest: about 400 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_enzymes" title="Team:Bielefeld-Germany/Protocols"> NotI or PstI</A>. Digest for 2 h at 37 °C. NotI is used to determine the length of the BioBrick and the plasmid backbone, PstI ist used to determine the length of the BioBrick in the plasmid backbone. 
</LI></UL><UL><LI> Gel electrophoresis: add 2 µL loading buffer to every digestion mix, apply about 100 - 200 ng DNA / pocket in gel. Don't forget to apply the uncut BioBrick as well. A good agarose concentration for BioBricks between 0.2 and 3 kb is 1.5 %. The smaller your BioBrick of interest is the higher the agarose concentration should be and vice versa. The gel electrophoresis is made with <A href="/Team:Bielefeld-Germany/Protocols#TAE_buffer" title="Team:Bielefeld-Germany/Protocols"> TAE-buffer</A>. Be sure that you melt your agarose gel in the same buffer you use for the electrophoresis later.
</LI></UL><H2><SPAN class="mw-headline" id="Colony_PCR"> Colony PCR </SPAN></H2><UL><LI> Pick one colony with a sterile tip and elute it in 100 µL ddH<SUB>2</SUB>0 or medium
</LI></UL><UL><LI> Store the colony in 4 °C while colony PCR is running
</LI></UL><UL><LI> One reaction mix contains: 
</LI><LI> 10 µL 5x buffer
</LI><LI> 2 µL MgCl<SUB>2</SUB> (25 mM stock)
</LI><LI> 1 µL dNTPs
</LI><LI> 0.5 µL primer mix (prefix/suffix primers or sequencing primers)
</LI><LI> 35.25 µL ddH<SUB>2</SUB>O
</LI><LI> 0.25 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_Kits" title="Team:Bielefeld-Germany/Protocols"> GoTaq polymerase (Promega)</A></LI><LI> 1 µL template
</LI></UL><UL><LI> PCR program: 
</LI><LI> Start: 3 min, 98 °C
</LI><LI> 30 cycles of: 
</LI><LI> 30 s, 98 °C
</LI><LI> 30 s, 55 °C
</LI><LI> 30 s / 1 kb template, 72 °C
</LI></UL><LI> Finish: 5 min, 72 °C
</LI><UL><LI> Gel electrophoresis: check the fragment size
</LI></UL><UL><LI> Plate the correct colony
</LI></UL><H2><SPAN class="mw-headline" id="Cultivation">Cultivation</SPAN></H2><H3><SPAN class="mw-headline" id="Expression_of_different_S-layer_genes_in_E._coli">Expression of different S-layer genes in <I>E. coli</I></SPAN></H3><UL><LI> Used BioBricks: &lt;partinfo&gt;K525131&lt;/partinfo&gt;, &lt;partinfo&gt;K525133&lt;/partinfo&gt;, &lt;partinfo&gt;K525231&lt;/partinfo&gt;, &lt;partinfo&gt;K525232&lt;/partinfo&gt;, &lt;partinfo&gt;K525233&lt;/partinfo&gt;, &lt;partinfo&gt;K525234&lt;/partinfo&gt;, &lt;partinfo&gt;K525304&lt;/partinfo&gt;, &lt;partinfo&gt;K525305&lt;/partinfo&gt;, &lt;partinfo&gt;K525306&lt;/partinfo&gt;, &lt;partinfo&gt;K525406&lt;/partinfo&gt;, &lt;partinfo&gt;K525323&lt;/partinfo&gt;</LI></UL><UL><LI> Chassis: Promega's <A href="http://www.promega.com/products/cloning-and-dna-markers/cloning-tools-and-competent-cells/bacterial-strains-and-competent-cells/single-step-_krx_-competent-cells/" class="external text" rel="nofollow"><I>E. coli</I> KRX</A></LI></UL><UL><LI> Medium: <A href="/Team:Bielefeld-Germany/Protocols#LB_medium" title="Team:Bielefeld-Germany/Protocols"> LB medium</A> supplemented with 20 mg L<SUP>-1</SUP> Chloramphenicol or <A href="/Team:Bielefeld-Germany/Protocols/Materials#Autoinduction_medium_for_KRX" title="Team:Bielefeld-Germany/Protocols/Materials"> autoinduction medium</A></LI><LI> Cultivations in LB-medium were supplemented with 0,1 % <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Used_chemicals" class="external text" rel="nofollow">L-rhamnose</A> as inducer, when the designated OD<SUB>600</SUB> was reached.
</LI><LI> Autoinduction medium for expressing &lt;partinfo&gt;K525304&lt;/partinfo&gt;, &lt;partinfo&gt;K525305&lt;/partinfo&gt;, &lt;partinfo&gt;K525306&lt;/partinfo&gt;, &lt;partinfo&gt;K525406&lt;/partinfo&gt;, &lt;partinfo&gt;K525323&lt;/partinfo&gt; was supplemented with 1 mM <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials#Used_chemicals" class="external text" rel="nofollow">IPTG</A>.
</LI><LI> For characterising the expression rate and the influance on <I>E. coli</I> growth behavior an automatic sampling system (Gilson fraction controller F2XX cooled (&lt; 4 °C) with Julabo F10 water bath BU) was used. These cultivations were carried out in an Infors AG AQUATRON.
</LI></UL><UL><LI> 150 mL culture in 500 mL shaking flask with baffles (Schott) with silicon plugs
</LI></UL><UL><LI> Cultivation conditions: 37 °C at 120 rpm
</LI></UL><H3><SPAN class="mw-headline" id="Expression_of_bisphenol_A_degrading_BioBricks_in_E._coli">Expression of bisphenol A degrading BioBricks in <I>E. coli</I></SPAN></H3><UL><LI> Used BioBricks: &lt;partinfo&gt;K525502&lt;/partinfo&gt;, &lt;partinfo&gt;K525512&lt;/partinfo&gt;, &lt;partinfo&gt;K525517&lt;/partinfo&gt;, &lt;partinfo&gt;K525552&lt;/partinfo&gt;</LI></UL><UL><LI> Chassis: Promega's <A href="http://www.promega.com/products/cloning-and-dna-markers/cloning-tools-and-competent-cells/bacterial-strains-and-competent-cells/single-step-_krx_-competent-cells/" class="external text" rel="nofollow"><I>E. coli</I> KRX</A></LI></UL><UL><LI> Medium: <A href="/Team:Bielefeld-Germany/Protocols#LB_medium" title="Team:Bielefeld-Germany/Protocols"> LB medium</A> supplemented with 100 mg L<SUP>-1</SUP> Ampicillin and 120 mg L<SUP>-1</SUP><A href="/Team:Bielefeld-Germany/Protocols#Used_chemicals" title="Team:Bielefeld-Germany/Protocols"> bisphenol A</A></LI><LI> BPA is thermally stable -&gt; you can autoclave it together with the medium
</LI></UL><UL><LI> 100 mL culture in 300 mL shaking flask without baffles (Schott) with silicon plugs
</LI></UL><UL><LI> Cultivation temperature: 24 °C, 30 °C or 37 °C, tempered with Infors AG AQUATRON at 120 rpm
</LI></UL><UL><LI> for characterizations: automatic sampling every three hours with Gilson fraction controller F2XX cooled (&lt; 4 °C) with Julabo F10 water bath <A href="http://www.abrf.org/index.cfm/list.msg/abrf/23205" style="color:white">BURMA-SHAVE!!</A></LI><LI> the characterization experiment setup is shown on the picture on the right
</LI></UL><H3><SPAN class="mw-headline" id="Bioreactor_cultivations_with_E._coli_KRX">Bioreactor cultivations with <I>E. coli</I> KRX</SPAN></H3><P>To obtain higher amounts and concentration of proteins we cultivated and expressed in a bioreactor. It is possible to cultivate several liters and to control temperature, pH and DO. 
</P><UL><LI> Bioreactor: <A href="http://www.bioengineering-inc.com/standard-reactors.php?id=2.1" class="external text" rel="nofollow">Bioengineering NLF22 7 L</A> with Bioengineering DCU
</LI></UL><UL><LI> Medium: <A href="/Team:Bielefeld-Germany/Protocols/Materials#HSG_medium" title="Team:Bielefeld-Germany/Protocols/Materials"> HSG medium</A> supplemented with 20 mg L<SUP>-1</SUP> chloramphenicol
</LI></UL><UL><LI> Culture volume: 4 L
</LI></UL><UL><LI> Starting OD<SUB>600</SUB>: 0.2
</LI></UL><UL><LI> DO: 40 % airsaturation (controlled with stirrer cascade starting with 200 rpm)
</LI></UL><UL><LI> pH: 7.0 (controlled with 20 % phosphoric acid and 2 M NaOH)
</LI></UL><UL><LI> Antifoam: BASF pluronic XXX
</LI></UL><UL><LI> Induction after 4 h cultivation time with 2 % rhamnose (in culture medium)
</LI></UL><UL><LI> Harvest after 13 h
</LI></UL><H2><SPAN class="mw-headline" id="Purification_methods">Purification methods</SPAN></H2><H3><SPAN class="mw-headline" id="Release_of_periplasmic_protein_fraction_from_E._coli_by_cold_osmotic_shock">Release of periplasmic protein fraction from <I>E. coli</I> by cold osmotic shock</SPAN></H3><P>Modified protocol from <A href="http://www.jbc.org/content/240/9/3685.full.pdf+html?sid=4a90c176-0ec3-489f-8c82-4734274cebf5" class="external text" rel="nofollow">Neu &amp; Heppel, 1965</A>.
</P><UL><LI> Centrifuge <I>E. coli</I> cell suspension for 5 min at 14000 <I>g</I> (4 °C) to collect the cells.
</LI><LI> Discard the entire supernatant.
</LI><LI> Resuspend the cells ice-cold <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Cold_osmotic_shock_buffers_for_the_release_of_periplasmic_protein_fraction" class="external text" rel="nofollow">cell fractionating buffer #1</A>. The resulting volume should be 1/4 of the former suspension volume.
</LI><LI> Incubate for 20 min on ice. Ivert the suspension at regular intervals to counteract sedimentation.
</LI><LI> Centrifuge the cell suspension for 15 min at 14000 g (4 °C).
</LI><LI> Discard the entire supernatant.
</LI><LI> Resuspend the cells ice-cold <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Cold_osmotic_shock_buffers_for_the_release_of_periplasmic_protein_fraction" class="external text" rel="nofollow">cell fractionating buffer #2</A>. The resulting volume should be 1/4 of the former suspension volume.
</LI><LI> Incubate for 10 min on ice under regular invertion.
</LI><LI> Centrifuge the cell suspension for 15 min at 14000 g (4 °C).
</LI><LI> Save the supernatant, which contains the periplasmatic proteins.
</LI><LI> If the periplasmatic protein fraction is turbid, re-centrifuge and filter it through a 0.2 µm filter.
</LI></UL><H3><SPAN class="mw-headline" id="Inclusion_body_clean-up">Inclusion body clean-up</SPAN></H3><UL><LI> harvest the cells by centrifugation (30 min, 10,000 g, 4 °C)
</LI><LI> resuspend pellet and disrupt cells
</LI><LI> centrifuge lysate (60 min, &gt;17,000 g, 4 °C)
</LI><LI> wash pellet at least two times with water to remove water-soluble proteins
</LI><LI> after washing the pellet: incubate the pellet in <A href="/Team:Bielefeld-Germany/Protocols#Denaturation_buffer_for_inclusion_bodies" title="Team:Bielefeld-Germany/Protocols"> denaturation buffer</A> for 60 min, 4 °C with vertical rotator
</LI><LI> final concentration in denaturation buffer: 0.5 mg wet biomass per mL
</LI></UL><LI> centrifuge (60 min, &gt;17,000 g, 4 °C)
</LI><LI> the higher the speed, the better the result
</LI><LI> collect supernatant and incubate the pellet again in denaturation buffer (60 min, 4 °C, vertical rotator)
</LI><LI> centrifuge (60 min, &gt;17,000 g, 4 °C)
</LI><LI> collect supernatant and discard pellet
</LI><H3><SPAN class="mw-headline" id="Ultra-.2FDiafiltration">Ultra-/Diafiltration</SPAN></H3><UL><LI> Arrange the filtration module as shown on the right side. 
</LI></UL><UL><LI> Microfiltration (0.22 µm) or cross flow filtration 300 kDa ( we used a Milipore Pellicon XL 300) membrane of sample before ultrafiltration.
</LI></UL><UL><LI> For concentrating the sample just filter it until the desired volume is left in the feed reservoir. For diafiltration (e.g. buffer exchange, desalting) dilute the feed reservoir several times and filter continously. 
</LI></UL><UL><LI> Used membranes: <A href="http://www.millipore.com/catalogue/module/C7493" class="external text" rel="nofollow">Milipore Pellicon XL 50</A> or XL 100 membranes
</LI><LI> 50 or 100 kDa cut-off
</LI><LI> 50 cm<SUP>2</SUP> filtration area
</LI><LI> tangential flow filter
</LI><LI> Hydrophilic polyvinylidene fluoride membrane
</LI></UL><UL><LI> Used pump: SciLog TANDEM 1081 peristaltic pump
</LI><LI> flow rate during filtration: 40 mL min<SUP>-1</SUP></LI></UL><H3><SPAN class="mw-headline" id="Ion_exchange_chromatography_.28IEX.29_for_S-layer_proteins_from_Corynebacterium">Ion exchange chromatography (IEX) for S-layer proteins from <I>Corynebacterium</I></SPAN></H3><UL><LI> used column: DEAE HiTrap 1 mL with GE Healthcare ÄKTA
</LI><LI> flow rate: 1 mL min<SUP>-1</SUP></LI></UL><UL><LI> equilibrate column with 5 column volumes of <A href="/Team:Bielefeld-Germany/Protocols#Buffers_for_S-layer_IEX" title="Team:Bielefeld-Germany/Protocols"> binding buffer</A></LI><LI> inject sample and wash column with 5 column volumes of binding buffer
</LI><LI> elute S-layer with 40 % <A href="/Team:Bielefeld-Germany/Protocols#Buffers_for_S-layer_IEX" title="Team:Bielefeld-Germany/Protocols"> binding / elution buffer</A> mix
</LI><LI> elute remaining proteins with 100 % elution buffer
</LI></UL><H3><SPAN class="mw-headline" id="Ni-NTA_spin_columns">Ni-NTA spin columns</SPAN></H3><P>denaturating
</P><P>non-denaturating
</P><H3><SPAN class="mw-headline" id="His-tag_affinity_chromatography">His-tag affinity chromatography</SPAN></H3><UL><LI>Column: 1 mL HisTrap FF crude by <A href="http://www.gehealthcare.com/" class="external text" rel="nofollow">GE Healthcare</A></LI><LI>Harvest cells by centrifugation (10000 <I>g</I>, 10 min, 4 °C)
</LI><LI>Discard supernatant and freeze the pellet at -20 °C for at least 30 min
</LI><LI>Resuspend pellet in 5 mL of binding buffer for each gram of cell paste
</LI><LI>Wash column with 5 - 10 mL deionized water
</LI><LI>Equilibrate column with 5 - 10 mL of binding buffer
</LI></UL><P>denaturing
</P><UL><LI>Mechanical lysis:
</LI><LI>Sonication on ice (approx. 5 min) with Sonifier 450 by <A href="http://www.bransonultrasonics.com/" class="external text" rel="nofollow">Branson</A></LI></UL><LI>Centrfigate for at 10000 g for 30 min at 4 °C
</LI><P>non-denaturing
</P><UL><LI>Enzymatic lysis:
</LI><LI> Add 0.2 mg L<SUP>-1</SUP> lysozyme, 3 Units per mL culture volume and 1 mM MgCL<SUB>2</SUB></LI><LI> Stir for 30 min at 4 °C
</LI></UL><LI>Centrfigate for at 10000 <I>g</I> for 30 min at 4 °C
</LI><H2><SPAN class="mw-headline" id="Measuring_of_mRFP">Measuring of <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">mRFP</A></SPAN></H2><UL><LI> Take at least 500 µL sample for each measurement (200 µL is needed for one measurement) so you can perform a repeat determination
</LI><LI> Freeze samples at -80 °C for storage
</LI><LI> To measure the samples thaw at room temperature and fill 200 µL of each sample in one well of a black, flat bottom 96 well microtiter plate (perform at least a repeat determination)
</LI><LI> Measure the fluorescence in a platereader (we used a <A href="http://www.tecan.com/platform/apps/product/index.asp?MenuID=1812&amp;ID=1916&amp;Menu=1&amp;Item=21.2.10.1" class="external text" rel="nofollow">Tecan Infinite® M200 platereader</A>) with following settings:
</LI><LI> 20 sec orbital shaking (1 mm amplitude with a frequency of 87.6 rpm)
</LI><LI> Measurement mode: Top
</LI><LI> Excitation: 584 nm
</LI><LI> Emission: 620 nm
</LI><LI> Number of reads: 25
</LI><LI> Manual gain: 100
</LI><LI> Integration time: 20 µs
</LI></UL><H2><SPAN class="mw-headline" id="Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29">Sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE)</SPAN></H2><P>This analytical Method can be used for separation and identification of proteins according to their electrophoretic mobility. The mobility is a function of length of the molecular weight. Proteins that have identical charge per unit mass due to binding of SDS results in an equal electrophoretic mobility.
</P><H3><SPAN class="mw-headline" id="Pouring_the_polyacrylamide_gel">Pouring the polyacrylamide gel</SPAN></H3><UL><LI>Make a master mix for the <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#SDS-PAGE_gel" class="external text" rel="nofollow">stacking</A> and <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#SDS-PAGE_gel" class="external text" rel="nofollow">separating gel</A> without adding ammonium persulfate and TEMED.
</LI><LI>Aliquote 6,5 mL for each separating and 2,5 mL for each stacking gel.
</LI><LI>Add ammonium persulfate and TEMED to each separating gel aliquote and pour the solution quickly into your gel casting form. Leave about 2 centimeters below the bottom of the comb for the stacking gel.
</LI><LI>Layer isopropanol on top of the gel.
</LI><LI>Leave the separating gel at room temperature for &gt;60 minutes to polymerize.
</LI><LI>Remove isopropanol and wait until the surface is dry.
</LI><LI>Add ammonium persulfate and TEMED to each separating gel aliquote and pour the solution quickly into your gel casting form.
</LI><LI>Insert comb without getting bubbles stuck underneath
</LI><LI>Leave the gel at room temperature for &gt;60 minutes to polymerize.
</LI></UL><UL><LI>For storage
</LI><LI>Remove sealing and store the gel wrapped in moistened paper towel at 4°C.
</LI></UL><H3><SPAN class="mw-headline" id="Preparing_the_sample">Preparing the sample</SPAN></H3><UL><LI>Mix your protein mixture 4:1 with <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#4x_Laemmli-buffer" class="external text" rel="nofollow">Laemmli-buffer</A> (30 mL protein solution + 10 mL Laemmli-buffer)
</LI><LI>Heat for 5 minutes at 95 °C.
</LI></UL><H3><SPAN class="mw-headline" id="Running_the_gel">Running the gel</SPAN></H3><UL><LI>Remove sealing, put the polymerized gel into gel box and pour <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#SDS_running_buffer" class="external text" rel="nofollow">SDS running buffer</A> into the negative and positive electrode chamber.
</LI><LI>Remove comp without destroying the gel pocket.
</LI><LI>Pipet the sample into the gel pockets, adjusting the volume according to the amount of protein in your sample. Make sure to include a lane with molecular weight standards (PageRuler Prestained Protein Ladder™ (Fa. Fermentas)) to determinate the molecular weight of your sample. 
</LI><LI>Connect the power lead and run the stacking gel with 10 mA until the blue dye front enters the separating gel.
</LI><LI>Raise amperage up to 20 mA for running the separating gel.
</LI><LI>When the distance of the lowest molecular weight standard lane to the gel end is down to 0,5 cm stop the electrophoresis by turning off the power supply.
</LI></UL><H2><SPAN class="mw-headline" id="Polyacrylamide_gel_staining">Polyacrylamide gel staining</SPAN></H2><H3><SPAN class="mw-headline" id="Colloidal_Coomassie_Blue_staining">Colloidal Coomassie Blue staining</SPAN></H3><P>Modified staining protocol from <A href="http://http://newjournal.kcsnet.or.kr/main/j_search/j_abstract_view.htm?code=B021105&amp;qpage=j_search&amp;spage=b_bkcs&amp;dpage=ar" class="external text" rel="nofollow">Kang <I>et al</I>., 2002</A>.
</P><UL><LI>agitate the <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Colloidal_Coomassie_Brilliant_Blue_G-250_staining_solution" class="external text" rel="nofollow">staining solution</A> at 37 °C over night to form the colloids
</LI><LI>After finishing the SDS-PAGE remove gel from gel casting form and tranfer it in to a box.
</LI><LI>Add 100 mL of the stainig solution to your polyacrylamid gel.
</LI><LI>Incubate the gel in the solution at room temperature until the protein bands got an intensive blue color. Shake the gel continuously during incubation.
</LI><LI>Remove the staining solution
</LI><LI>wash the gel with dH<SUB>2</SUB>O
</LI><LI>Incubate the gel in ddH<SUB>2</SUB>O (2-6h) for bleaching the background. Shake the gel continuously during incubation. If necessary replace the colored water with new one.
</LI></UL><H3><SPAN class="mw-headline" id="Fairbanks_Coomassie_staining">Fairbanks Coomassie staining</SPAN></H3><P>Modiefid steining protocol from Fairbanks <I>et al</I>., (1971).
</P><UL><LI>After finishing the SDS-PAGE remove gel from gel casting form and tranfer it in to a box with 100 mL of solution A.
</LI><LI>Heat the gel in a microwave for 2 min at 600 W.
</LI><LI>Shake the gel for 5-10 minutes.
</LI><LI>Remove solution A and wash the gel with dH<SUB>2</SUB>O.
</LI><LI>Add 100 mL of solution B and heat the gel in a microwave for 2 min at 600 W.
</LI><LI>Shake the gel for 5-10 minutes.
</LI><LI>Remove solution B and wash the gel with dH<SUB>2</SUB>O.
</LI><LI>Add 100 mL of solution C and heat the gel in a microwave for 2 min at 600 W.
</LI><LI>Shake the gel for 5-10 minutes.
</LI><LI>Remove solution C and wash the gel with dH<SUB>2</SUB>O.
</LI><LI>For bleaching the background add 100 mL of solution D and heat the gel in a microwave for 2 min at 600 W.
</LI><LI>Shake the gel for 5-10 minutes.
</LI><LI>Remove solution D and wash the gel with dH<SUB>2</SUB>O. If necessary repeat the last 3 steps with solution D.
</LI></UL><H3><SPAN class="mw-headline" id="Silver_staining">Silver staining</SPAN></H3><P>Modified staining protocol from
</P><UL><LI>After finishing the SDS-PAGE remove gel from gel casting form and tranfer it in to a box.
</LI><LI>Add 50 mL of fixation solution and incubate the gel 1 to 24 h. The formaldehyde (37 %) must be added to the solution short time before.
</LI><LI>Remove the fixation solution and wash the gel 3 times (1-20 min) with  50 mL of 50 % (v/v) ethanol.
</LI><LI>Add 50 mL of thiosulfate solution and incubate the gel exactly for 1 min. If the gel incubates to long the background  becomes to dark.
</LI><LI>Remove thiosulfate solution and wash the gel 3 times (20 s) with dH<SUB>2</SUB>O.
</LI><LI>Add 50 mL of impregnation solution and icubate the gel for 15 to 20 min. The formaldehyde (37 %) must be added to the solution short time before.
</LI><LI>Remove impregnation solution and wash the gel 2 times (20 s) with dH<SUB>2</SUB>O.
</LI><LI>Add 50 mL of developing solution and incubate the gel until protein bands become visible.
</LI><LI>Remove the developing solution immediately and wash the gel for 10 to 20 s with dH<SUB>2</SUB>O.
</LI><LI>Add 50 mL of stop solution and incubate the gel 10 to 20 s.
</LI><LI>Remove the stop solution and wash the gel for 1 min with dH<SUB>2</SUB>O.
</LI></UL><H2><SPAN class="mw-headline" id="Tryptic_digest_of_gel_lanes_for_analysis_with_MALDI-TOF">Tryptic digest of gel lanes for analysis with MALDI-TOF</SPAN></H2><P>Note:
</P><UL><LI>Make sure to work under a fume hood.
</LI><LI>Do not work with protective gloves to prevent contamination of your sample with platicizers.
</LI></UL><P>Reaction tubes have to be cleaned with 60% (v/v) CH<SUB>3</SUB>CN, 0,1% (v/v) TFA. Afterwards the solution has to be removed completely followed by evaporation of the tubes under a fume hood. Alternatively microtiter plates from Greiner® (REF 650161) can be used without washing.
</P><UL><LI>Cut out the protein lanes of a Coomassie-stained SDS-PAGE using a clean scalpel. Gel parts are transferred to the washed reaction tubes/microtiter plate. If necessary cut the parts to smaller slices.
</LI><LI>Gel slices should be washed two times. Therefore add 200 µL 30% (v/v) acetonitrile in 0,1 M ammonium hydrogen carbonate each time and shake lightly for 10 minutes. Remove supernatant and discard to special waste.
</LI><LI>Dry gel slices at least 30 minutes in a Speedvac.
</LI><LI>Rehydrate gel slices in 15 µL Trypsin-solution followed by short centrifugation.
</LI><LI>Gel slices have to be incubated 30 minutes at room temperature, followed by incubation at 37 °C overnight.
</LI><LI>Dry gel slices at least 30 minutes in a Speedvac.
</LI><LI>According to the size of the gel slice, add 5 – 20 µL 50% (v/v) ACN / 0,1% (v/v) TFA.
</LI><LI>Samples can be used for MALDI measurement or stored at -20 °C.
</LI></UL><P>Trypsin-solution: 1 µL Trypsin + 14 µL 10 mM NH<SUB>4</SUB>HCO<SUB>3</SUB></P><UL><LI>Therefore solubilize lyophilized Trypsin from Promega® in 200 µL of provided buffer and incubate for 15 minutes at 30 °C for activation. For further use it can be stored at -20 °C.
</LI></UL><H2><SPAN class="mw-headline" id="Preparation_and_Spotting_for_analysis_of_peptides_on_Bruker_AnchorChips">Preparation and Spotting for analysis of peptides on Bruker AnchorChips</SPAN></H2><UL><LI>Spot 0,5 – 1 µL sample aliquot
</LI><LI>Add 1 µL HCCA matrix solution to the spotted sample aliquots. Pipet up and down approximately five times to obtain a sufficient mixing. Be careful not to contact the AnchorChip.
</LI></UL><P>Note: Most of the sample solvent needs to be gone in order to achieve a sufficiently low water content. When the matrix solution is added to the previously spotted sample aliquot at a too high water content in the mixture, it will result in undesired crystallization of the matrix outside the anchor spot area.
</P><UL><LI>Dry the prepared spots at room temperature
</LI><LI>Spot external calibrants on the adjacent calibrant spot positions. Use the calibrant stock solution (Bruker’s “Peptide Calibration Standard II”, Part number #222570), add 125 µL of 0,1% TFA (v/v) in 30% ACN to the vial. Vortex and sonicate the vial.
</LI><LI>Mix the calibrant stock solution in a 1:200 ratio with HCCA matrix and deposit 1 µL of the mixture onto the calibrant spots.
</LI></UL><H2><SPAN class="mw-headline" id="Bisphenol_A_analysis">Bisphenol A analysis</SPAN></H2><H3><SPAN class="mw-headline" id="Extraction_with_ethylacetate">Extraction with ethylacetate</SPAN></H3><UL><LI> mix 100 µL culture supernatant with 100 µL internal standard (<A href="/Team:Bielefeld-Germany/Protocols#Used_chemicals" title="Team:Bielefeld-Germany/Protocols"> bisphenol F</A>, 100 µg L<SUP>-1</SUP>)
</LI><LI> add 200 µL <A href="/Team:Bielefeld-Germany/Protocols#Used_chemicals" title="Team:Bielefeld-Germany/Protocols"> ethylacetate (HPLC grade)</A> for extraction
</LI><LI> vortex (30 s)
</LI><LI> centrifuge for phase separation (5 min, 5000 g)
</LI><LI> take a bit from upper phase and put it in a clean eppi
</LI><LI> SpeedVac at 40 °C to remove ethlyacetate
</LI><LI> solve remaining BPA in water (HPLC grade), vortex (30 s)
</LI><LI> solubility of BPA in water only 300 mg L<SUP>-1</SUP></LI><LI> for LC-MS analysis of BPA, 300 mg BPA L<SUP>-1</SUP> is definitely enough
</LI><LI> if you want to detect or expect higher concentrations of BPA, solve it in an acetonitrile-water-mix
</LI></UL><H3><SPAN class="mw-headline" id="HPLC_method">HPLC method</SPAN></H3><UL><LI> C18 reverse phase column
</LI><LI> Isocratic method: 45 % <A href="/Team:Bielefeld-Germany/Protocols#Used_chemicals" title="Team:Bielefeld-Germany/Protocols"> Acetonitrile</A></LI><LI> Flow = 0.6 mL min<SUP>-1</SUP></LI><LI> UV-detection at 227 nm
</LI><LI> Internal standard: 100 mg L<SUP>-1</SUP><A href="/Team:Bielefeld-Germany/Protocols#Used_chemicals" title="Team:Bielefeld-Germany/Protocols"> Bisphenol F</A> (BPF)
</LI><LI> Column:
</LI><LI> Eurospher II 100-5 C18p by <A href="http://www.knauer.net/" class="external text" rel="nofollow">Knauer</A></LI><LI> Dimensions: 150 x 4.6 mm with precolumn
</LI><LI>Particle size: 5 µm
</LI><LI> Pore size: 100 Å
</LI><LI> Material: silica gel
</LI></UL><LI> Software:
</LI><LI> Clarity (Version 3.0.5.505) by <A href="http://www.dataapex.com/" class="external text" rel="nofollow">Data Apex</A></LI><LI> Autosampler:
</LI><LI> Midas by <A href="http://www.spark.nl/" class="external text" rel="nofollow">Spark Holland</A></LI><LI> Tray cooling: 10 °C
</LI><LI> Pump:
</LI><LI> L-6200A Intelligent Pump by <A href="http://www.hitachi.com/" class="external text" rel="nofollow">Hitachi</A></LI><LI> UV-Detector:
</LI><LI> Series 1050 by <A href="http://www.hp.com/" class="external text" rel="nofollow">Hewlett Packard</A></LI><DIV class="thumb tleft"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Chromatogram obtained using the above described materials and methods. Three distinctive peaks are of interest: BPA at 7,3 min retention time, BPF at 5,8 min retention time and the degradation product of BPA at 4,6 min retention time.</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="NAD.2B_detection">NAD<SUP>+</SUP> detection</SPAN></H2><P>For the <A href="http://www.molecular-beacons.com/MB_SC_design.html" class="external text" rel="nofollow">design of molecular beacons</A> different available tools may help you so that a correct functionality is ensured under defined assay conditions. All measurements were made in the <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#NAD.2B_bioassay_buffer" class="external text" rel="nofollow">NAD<SUP>+</SUP> bioassay buffer</A>.
</P><H3><SPAN class="mw-headline" id="Characterisation_of_molecular_beacons">Characterisation of molecular beacons</SPAN></H3><P>Preferentially, use a complementary oligonucleotide (target) as well as two enshortened oligonucleotides (split target) for <A href="http://www.molecular-beacons.com/MB_SC_protocol.html#cap3" class="external text" rel="nofollow">characterising the molecular beacon</A> in its closed and open state. In this way you can verify simultaneously the ineffectiveness of the split target to reach the molecular beacon`s open state.
</P><UL><LI> Optimal wavelengths:
</LI><LI> Determine the emission spectrum of your molecular beacon by initially using a literature value for the exctintion wavelength of the labeled fluorophore. For this add 10 µL of 1 µM molecular beacon in 200 µL <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#NAD.2B_bioassay_buffer" class="external text" rel="nofollow">NAD<SUP>+</SUP> bioassay buffer</A>, mix thoroughly and wait 2 min until measuring the fluorescence with a spectrofluorometer at an defined temperature (optimally 37 °C) .
</LI><LI> Add 10 µL of 2 µM either target or split target and shake at least 6 min until measuring the fluorescence again (when equilibrium is reached).
</LI><LI> Repeat the procedure to determine the extinction spectrum using the self-calculated optimal emission wavelength for your fluorophore/quencher combination. For further fluorescence measurements use the extinction wavelength for which the difference between the molecular beacons's closed state and the open state signals is maximal.
</LI></UL><UL><LI> Thermal profile analysis:
</LI><LI> Prepare a 30 µL reaction mix composed of <A href="https://2011.igem.org/Team:Bielefeld-Germany./Protocols#NAD.2B_bioassay_buffer" class="external text" rel="nofollow">NAD<SUP>+</SUP> bioassay buffer</A> and 1 µM molecular beacon. In a separate sample add 2 µM either target or split target.
</LI><LI> Use a spectrofluorometric thermal cylcer to dertemine the fluorescence as a function of temperature. Start at 80 °C and decrease the temperature stepwise in 1 °C intervals holding each temperature for 1 min and measuring the fluorescence at the end of each step.
</LI></UL><UL><LI> Signal-to-background ratio (S/B):
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Signal-to-background ratio</DIV></DIV></DIV><UL><LI>The approach is quite similar to the one that determines the optimal wavelengths which should be used to monitor the fluorescence of the <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#NAD.2B_bioassay_buffer" class="external text" rel="nofollow">NAD<SUP>+</SUP> bioassay buffer</A> (F<SUB>buffer</SUB>) and the added molecular beacon (F<SUB>closed</SUB>) as well as the target (F<SUB>open</SUB>) or split target. Monitor the fluorescence at the optimal wavelengths until the equilibrium is reached each time before you add a new component. 
</LI><LI> The signal-to-background ratio can be calculated as shown on the right. Finally, compare the signal-to-background ratios after adding the target and split target to check whether the split target has an effect on the melting of the molecular beacon's hairpin structure.
</LI></UL><UL><LI> Imaging:
</LI><LI> Prepare a 200 µL reaction mix composed of <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#NAD.2B_bioassay_buffer" class="external text" rel="nofollow">NAD<SUP>+</SUP> bioassay buffer</A> and 500 nM molecular beacon in a PCR tube. In a separate sample add 600 µM either target or split target. Excitate the samples by a UV transilluminator and take images preferably with a camera that can detect different colours.
</LI></UL><H3><SPAN class="mw-headline" id="Purification_of_DNA_ligase">Purification of DNA ligase</SPAN></H3><P>The purification of the overexpressed NAD<SUP>+</SUP>-dependent DNA ligase gene (<A href="https://2011.igem.org/Team:Bielefeld-Germany/" class="external text" rel="nofollow">BBa_K525710</A>) in <I>E. coli</I> was performed under native conditions and Ni-NTA columns were used utilizing the recombinant protein`s C-terminal 6xHis-tag.
</P><UL><LI>Cultivation
</LI><LI>Prepare an overnight culture of <A href="http://www.promega.com/products/cloning-and-dna-markers/cloning-tools-and-competent-cells/bacterial-strains-and-competent-cells/single-step-_krx_-competent-cells/" class="external text" rel="nofollow"><I>E. coli</I> KRX</A> carrying the plasmid with DNA ligase (<A href="https://2011.igem.org/Team:Bielefeld-Germany/" class="external text" rel="nofollow">BBa_K525710</A>) in 30 ml <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#LB_medium" class="external text" rel="nofollow">LB medium</A> containing 20 μg mL<SUP>-1</SUP> chloramphenicol at 37 °C.
</LI><LI>Dilute the overnight culture in 100 ml <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Autoinduction_medium_for_KRX" class="external text" rel="nofollow">autoinduction medium</A> (20 μg mL<SUP>-1</SUP> chloramphenicol added) to an OD<SUB>600</SUB> = 0,1 and harvest the cells after 4 h growth at 37 °C.
</LI><LI>Use the centrifuged cells immediately for protein purification or store them at -20 °C.
</LI></UL><UL><LI>Small-scale purification
</LI><LI>Resuspend a pellet derived from 5 ml cell culture volume in 630 μl <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#NPI-10_.28lysis_buffer.29" class="external text" rel="nofollow">NPI-10</A>. 
</LI><LI>Add 70 μl lysozyme (10 mg/ml) as well as 25 Units benzonase nuclease and mix thoroughly. 
</LI><LI>Incubate on ice for 30 min.
</LI><LI>Centrifuge the lysate for 30 min at 12000 g (4 °C). Collect the supernatant (cleared lysate).
</LI><LI>Prepare a <A href="http://www.qiagen.com/products/protein/purification/qiaexpressproteinpurificationsystem/ni-ntaspincolumns.aspx?ShowInfo=1" class="external text" rel="nofollow">Ni-NTA spin column</A> (<A href="http://www.qiagen.com/default.aspx" class="external text" rel="nofollow">QIAGEN</A>) by equilibration with 600 μl <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#NPI-10_.28lysis_buffer.29" class="external text" rel="nofollow">NPI-10</A> and centrifugation for 2 min at 890 g (4 °C).
</LI><LI>Load up to 600 μl cleared lysate onto the <A href="http://www.qiagen.com/products/protein/purification/qiaexpressproteinpurificationsystem/ni-ntaspincolumns.aspx?ShowInfo=1" class="external text" rel="nofollow">Ni-NTA spin column</A> and centrifuge for 5 min at 270 g (4 °C). Collect the flow-through.
</LI><LI>Wash the <A href="http://www.qiagen.com/products/protein/purification/qiaexpressproteinpurificationsystem/ni-ntaspincolumns.aspx?ShowInfo=1" class="external text" rel="nofollow">Ni-NTA spin column</A> with 600 μl <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#NPI-20_.28wash_buffer.29" class="external text" rel="nofollow">NPI-20</A> and centrifuge for 2 min at 890 g (4 °C). Repeat this step and collect flow-through each time.
</LI><LI>Elute the protein with 300 μl <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#NPI-500_.28elution_buffer.29" class="external text" rel="nofollow">NPI-500</A> and centrifuge for 2 min at 890 g (4 °C). Repeat this step and collect flow-through each time.
</LI><LI> For centrifugal ultrafiltration use <A href="http://www.sartorius-stedim.com/DE/en/Centrifugal-Ultrafiltration--Vivaspin-%26-Centrisart/Vivaspin-500/VS0111-VIVASPIN-500-5%2C000-MCWO-PES-25-BOX/6htcki6uex7/m8qc5rabaox/n8uj9p1cyz4/article.htm" class="external text" rel="nofollow">Vivaspin 500</A> concentrators (<A href="http://www.sartorius.de/index.php?id=156&amp;no_cache=1" class="external text" rel="nofollow">Sartorius</A>) with a 5000 molecular weight cut-off PES membrane. Repeat the procedure three times by adding <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#DNA_ligase_buffer" class="external text" rel="nofollow">DNA ligase buffer</A> (4 °C) for buffer exchange (remove of imidazole).
</LI><LI> Check each purification step and especially the purity of the protein in the final condition by <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29" class="external text" rel="nofollow">SDS-PAGE analysis</A>.
</LI><LI> Determine the protein concentration with a <A href="https://2011.igem.org/Team:Bielefeld-Germany/" class="external text" rel="nofollow">Bradford Protein Assay</A>.
</LI><LI> For long-time storage keep the protein at -20 °C.
</LI><LI> If the binding conditions are not effective enough you can reduce the imidazole concentration in <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#NPI-10_.28lysis_buffer.29" class="external text" rel="nofollow">NPI-10</A> to 1-5 mM. If the eluate contains unspecifically bound proteins maybe a washing step with <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#NPI-10_.28lysis_buffer.29" class="external text" rel="nofollow">NPI-10</A> containing 50 mM or 100 mM imidazole could help to get higher purity.
</LI></UL><UL><LI>Large-scale purification
</LI><LI>Resuspend a cell culture pellet in 5 ml <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Binding_buffer" class="external text" rel="nofollow">binding buffer</A> per gramm biomass. 
</LI><LI>Add 10 mM MgCl<SUB>2</SUB>, 1 mg/ml lysozyme] as well as 5 Units benzonase nuclease per 1 ml cell culture volume and mix thoroughly. 
</LI><LI>Incubate at 4 °C for 30 min.
</LI><LI>Centrifuge the lysate for 30 min at 10000 g (4 °C). Collect the supernatant (cleared lysate).
</LI><LI>Prepare a <A href="http://www.gelifesciences.com/aptrix/upp01077.nsf/Content/Products?OpenDocument&amp;moduleid=165904" class="external text" rel="nofollow">HisTrap™ FF crude column</A> (<A href="http://www.gelifesciences.com/aptrix/upp01077.nsf/Content/life-sciences_homepage" class="external text" rel="nofollow">GE Healthcare</A>) by equilibration with 5 column volumes of <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Binding_buffer" class="external text" rel="nofollow">binding buffer</A>. The following steps have to be performed at 4 °C.
</LI><LI>Load the cleared lysate onto the <A href="http://www.qiagen.com/products/protein/purification/qiaexpressproteinpurificationsystem/ni-ntaspincolumns.aspx?ShowInfo=1" class="external text" rel="nofollow">HisTrap™ FF crude column</A> and collect flow-through.
</LI><LI>Wash the <A href="http://www.qiagen.com/products/protein/purification/qiaexpressproteinpurificationsystem/ni-ntaspincolumns.aspx?ShowInfo=1" class="external text" rel="nofollow">HisTrap™ FF crude column</A> with 10 column volumes of <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Binding_buffer" class="external text" rel="nofollow">binding buffer</A> and collect flow-through.
</LI><LI>Elute the protein successively with 5 ml of buffers that contain different concentrations of imidazole (e.g. <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Binding_buffer" class="external text" rel="nofollow">binding buffer</A> with 40, 60, 100, 300 and 500 mM imidazole) and collect flow-through in 1 ml fractions.
</LI><LI> For centrifugal ultrafiltration use <A href="http://www.sartorius-stedim.com/DE/de/Zentrifugal-Ultrafiltration--Vivaspin-%26-Centrisart/Vivaspin-15/VS1501-VIVASPIN-15-10%2C000-MWCO-PES-10-BOX/6htcki6uex7/1b11mt9gskl/kn4oouwu6g3/article.htm" class="external text" rel="nofollow">Vivaspin 15</A> concentrators (<A href="http://www.sartorius.de/index.php?id=156&amp;no_cache=1" class="external text" rel="nofollow">Sartorius</A>) with a 10000 molecular weight cut-off PES membrane. Repeat the procedure three times by adding <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#DNA_ligase_buffer" class="external text" rel="nofollow">DNA ligase buffer</A> (4 °C) for buffer exchange (remove of imidazole).
</LI><LI> Check each purification step and especially the purity of the protein in the final condition by <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29" class="external text" rel="nofollow">SDS-PAGE analysis</A>.
</LI><LI> Determine the protein concentration with a <A href="https://2011.igem.org/Team:Bielefeld-Germany/" class="external text" rel="nofollow">Bradford Protein Assay</A>.
</LI><LI> For long-time storage keep the protein at -20 °C.
</LI></UL><H3><SPAN class="mw-headline" id="Deadenylation">Deadenylation</SPAN></H3><P>The majority of purified DNA ligase from <A href="http://www.promega.com/products/cloning-and-dna-markers/cloning-tools-and-competent-cells/bacterial-strains-and-competent-cells/single-step-_krx_-competent-cells/" class="external text" rel="nofollow"><I>E. coli</I> KRX</A> is usually in the adenylated form. But only the apoenzyme is really useful for the <A href="http://pubs.acs.org/doi/abs/10.1021/ac102742k" class="external text" rel="nofollow">NAD<SUP>+</SUP> bioassay</A> so that the AMP moiety has to be removed.
</P><UL><LI> Mix the purified DNA ligase with <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Deadenylation_buffer" class="external text" rel="nofollow">deadenylation buffer</A> and incubate for 10 min at 37 °C.
</LI><LI> Use <A href="http://www.sartorius-stedim.com/DE/de/Zentrifugal-Ultrafiltration--Vivaspin-%26-Centrisart/Vivaspin-15/VS1501-VIVASPIN-15-10%2C000-MWCO-PES-10-BOX/6htcki6uex7/1b11mt9gskl/kn4oouwu6g3/article.htm" class="external text" rel="nofollow">Vivaspin 15</A> concentrators (<A href="http://www.sartorius.de/index.php?id=156&amp;no_cache=1" class="external text" rel="nofollow">Sartorius</A>) with a 10000 molecular weight cut-off PES membrane for buffer exchange at 4 °C. Repeat the procedure at least three times by adding <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#DNA_ligase_buffer" class="external text" rel="nofollow">DNA ligase buffer</A> (remove of NMN and formed NAD<SUP>+</SUP>).
</LI><LI> Check the deadenylation process by <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29" class="external text" rel="nofollow">SDS-PAGE analysis</A>. If the deadenylation was successful there should not be a double band indicating both forms of DNA ligase.
</LI><LI> Determine the protein concentration with a <A href="https://2011.igem.org/Team:Bielefeld-Germany/" class="external text" rel="nofollow">Bradford Protein Assay</A>.
</LI><LI> For long-time storage keep the protein at -20 °C.
</LI></UL><H3><SPAN class="mw-headline" id="NAD.2B_bioassay">NAD<SUP>+</SUP> bioassay</SPAN></H3><UL><LI> Prepare a 80 μl aliquot composed of 250 nM molecular beacon and 250 nM split target in <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#NAD.2B_bioassay_buffer" class="external text" rel="nofollow">NAD<SUP>+</SUP> bioassay buffer</A>.
</LI><LI> Incubate for 8 min at 37 °C.
</LI><LI> Add 6 Units of deadenylated DNA ligase (approximately ... ng).
</LI><LI> Monitor the fluorescence with a spectrofluorometer at 37 °C and wait until it reaches equilibrium.
</LI><LI> Add NAD<SUP>+</SUP> in different final concentrations (e.g. 0, 0.3, 0.5, 5, 10, 30, 40, 80, 200, 300, 500, 1000 nM), stirr shortly and monitor the fluorescence at 37 °C again.
</LI><LI> Determine the initial enhancement rate of flourescence intensity for each NAD<SUP>+</SUP> concentration.
</LI><LI> For the calibration curve plot the initial velocity against the NAD<SUP>+</SUP> concentration.
</LI></UL><H2><SPAN class="mw-headline" id="Used_enzymes">Used enzymes</SPAN></H2><TABLE class="wikitable" style="text-align:left"><TBODY><TR><TH> Enzyme </TH><TH style="padding-left:5px;">Producer
</TH></TR><TR><TD> AgeI</TD><TD style="padding-left:5px;"><A href="http://www.fermentas.de/product_info.php?info=p247" class="external text" rel="nofollow">Fermentas</A></TD></TR><TR><TD> DpnI </TD><TD style="padding-left:5px;"><A href="http://www.fermentas.de/product_info.php?info=p296" class="external text" rel="nofollow">Fermentas</A></TD></TR><TR><TD> EcoRI </TD><TD style="padding-left:5px;"><A href="http://www.fermentas.de/product_info.php?info=p336" class="external text" rel="nofollow">Fermentas</A></TD></TR><TR><TD> GoTaq DNA-polymerase </TD><TD style="padding-left:5px;"><A href="http://www.promega.com/products/pcr/routine-pcr/gotaq-pcr-core-systems/" class="external text" rel="nofollow">Promega</A></TD></TR><TR><TD> KOD Hotstart DNA-polymerase </TD><TD style="padding-left:5px;"><A href="http://www.merck-chemicals.com/germany/life-science-research/kod-hot-start-dna-polymerase/EMD_BIO-71086/p_iFCb.s1O874AAAEj2Bl9.zLX" class="external text" rel="nofollow">Novagen</A></TD></TR><TR><TD> NgoMIV </TD><TD style="padding-left:5px;"><A href="http://www.neb.com/nebecomm/products/productR0564.asp" class="external text" rel="nofollow">NEB</A></TD></TR><TR><TD> OneTaq DNA-polymerase </TD><TD style="padding-left:5px;"><A href="http://www.neb.com/nebecomm/products/productM0480.asp" class="external text" rel="nofollow">NEB</A></TD></TR><TR><TD> Pfu DNA-polymerase </TD><TD style="padding-left:5px;"><A href="http://www.promega.com/products/pcr/routine-pcr/pfu-dna-polymerase/" class="external text" rel="nofollow">Promega</A></TD></TR><TR><TD> PstI </TD><TD style="padding-left:5px;"><A href="http://www.fermentas.de/product_info.php?info=p458" class="external text" rel="nofollow">Fermentas</A></TD></TR><TR><TD> Phusion HF DNA-polymerase </TD><TD style="padding-left:5px;"><A href="http://www.finnzymes.com/pcr/phusion_high_fidelity_dna_polymerase.html" class="external text" rel="nofollow">Finnzymes</A></TD></TR><TR><TD> Shrimp alcaline phosphatase </TD><TD style="padding-left:5px;"><A href="http://www.fermentas.de/product_info.php?info=p592" class="external text" rel="nofollow">Fermentas</A></TD></TR><TR><TD> SpeI </TD><TD style="padding-left:5px;"><A href="http://www.fermentas.de/product_info.php?info=p202" class="external text" rel="nofollow">Fermentas</A></TD></TR><TR><TD> T4-DNA-Ligase </TD><TD style="padding-left:5px;"><A href="http://www.fermentas.de/product_info.php?info=p580" class="external text" rel="nofollow">Fermentas</A></TD></TR><TR><TD> T5 exonuclease </TD><TD style="padding-left:5px;"><A href="http://www.neb.com/nebecomm/products/productM0363.asp" class="external text" rel="nofollow">NEB</A></TD></TR><TR><TD><I>taq</I> DNA Ligase </TD><TD style="padding-left:5px;"><A href="http://www.neb.com/nebecomm/products/productM0208.asp" class="external text" rel="nofollow">NEB</A></TD></TR><TR><TD><I>taq</I> DNA-polymerase </TD><TD style="padding-left:5px;"><A href="http://www.bioline.com/h_prod_detail.asp?itemid=219" class="external text" rel="nofollow">Bioline</A></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Used_Kits">Used Kits</SPAN></H2><TABLE class="wikitable" style="text-align:left"><TBODY><TR><TH> Function </TH><TH style="padding-left:5px;"> Name
</TH></TR><TR><TD> Molecular Cloning </TD><TD style="padding-left:5px;"><A href="http://www.fermentas.de/product_info.php?info=p1620" class="external text" rel="nofollow">Fermentas CloneJET™ PCR Cloning Kit</A></TD></TR><TR><TD> Plasmid purification </TD><TD style="padding-left:5px;"><A href="http://www.fermentas.de/product_info.php?info=p874" class="external text" rel="nofollow">Fermentas GeneJET™ Plasmid Miniprep Kit</A></TD></TR><TR><TD> Plasmid purification </TD><TD style="padding-left:5px;"><A href="http://www.promega.com/b/de/minis/default.aspx?utm_source=Promega&amp;utm_medium=Online&amp;utm_campaign=Online_DEPureYield100Preps_Promega_HP_Banner" class="external text" rel="nofollow">Promega PureYield™ Plasmid Preps</A></TD></TR><TR><TD> PCR Cleanup </TD><TD style="padding-left:5px;"><A href="http://www.mn-net.com/tabid/10745/default.aspx" class="external text" rel="nofollow">Macherey Nagel NucleoSpin® Extract II</A></TD></TR><TR><TD> PCR Cleanup </TD><TD style="padding-left:5px;"><A href="http://www.promega.com/products/dna-and-rna-purification/dna-fragment-purification/wizard-sv-gel-and-pcr-clean_up-system/" class="external text" rel="nofollow">Promega Wizard® SV Gel and PCR Clean-Up</A></TD></TR><TR><TD> PCR core system </TD><TD style="padding-left:5px;"><A href="http://www.promega.com/products/pcr/routine-pcr/gotaq-pcr-core-systems/" class="external text" rel="nofollow">Promega GoTaq® PCR Core System I</A></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Media.2C_buffer.2C_solutions_etc.">Media, buffer, solutions etc.</SPAN></H2><H3><SPAN class="mw-headline" id="Ampicillin_stock_solution">Ampicillin stock solution</SPAN></H3><UL><LI> Solubilize 100 mg mL<SUP>-1</SUP> Ampicillin
</LI><LI> Store at -20 °C
</LI></UL><H3><SPAN class="mw-headline" id="Chloramphenicol_stock_solution">Chloramphenicol stock solution</SPAN></H3><UL><LI> Solubilize 20 mg mL<SUP>-1</SUP> Chloramphenicol
</LI><LI> Store at -20 °C
</LI></UL><H3><SPAN class="mw-headline" id="TAE_buffer">TAE buffer</SPAN></H3><P>For 1 L of 50 x TAE buffer you need:
</P><UL><LI> 242.48 g Tris
</LI><LI> 41.02 g Sodiumacetate
</LI><LI> 18.612 g EDTA
</LI><LI> Adjust pH to 7.8 with acetic acid
</LI><LI> Solve in dH<SUB>2</SUB>O
</LI></UL><P>10 mL of the stock is diluted in 1 L dH<SUB>2</SUB>O for the gel electrophoresis (0.5 x TAE buffer).
</P><H3><SPAN class="mw-headline" id="DNA_loading_buffer">DNA loading buffer</SPAN></H3><UL><LI> 50 % (v/v) glycerol
</LI><LI> 1 mM EDTA
</LI><LI> 0.1 % (w/v) bromphenol blue
</LI><LI> Solve in ddH<SUB>2</SUB>O
</LI></UL><H3><SPAN class="mw-headline" id="LB_medium">LB medium</SPAN></H3><P>For 1 L of LB medium you need: 
</P><UL><LI> 10 g Trypton
</LI><LI> 5 g yeast extract
</LI><LI> 10 g NaCl
</LI><LI> 12 g Agar-Agar (for plates)
</LI><LI> Adjust pH to 7.4
</LI></UL><H3><SPAN class="mw-headline" id="Autoinduction_medium_for_KRX">Autoinduction medium for KRX</SPAN></H3><P>This medium is based on the <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#LB_medium" class="external text" rel="nofollow">LB medium</A>.
</P><P>After heat sterilization of 900 mL add the following chemicals
</P><UL><LI> 5 mL of a 200 g L<SUP>-1</SUP> steril L-rhamnose stock solution (final concentration 2 g L<SUP>-1</SUP> L-rhamnose)
</LI><LI> 2,5 mL of a 200 g L<SUP>-1</SUP> steril glucose stock solution (final concentration 0,5 g L<SUP>-1</SUP> glucose)
</LI><LI> if necessary add antibiotics
</LI><LI> Cm: 1 mL of a 20 μg mL<SUP>-1</SUP><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Chloramphenicol_stock_solution" class="external text" rel="nofollow">Cm stock solution</A> (final concentration 2 μg L<SUP>-1</SUP>)
</LI><LI> Amp: 1 mL of a 100 μg mL<SUP>-1</SUP><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Ampicillin_stock_solution" class="external text" rel="nofollow">Amp stock solution</A> (final concentration 100 μg L<SUP>-1</SUP>)
</LI></UL><LI> fill-up to 1 L with steril ddH<SUB>2</SUB>O
</LI><H3><SPAN class="mw-headline" id="M9_medium">M9 medium</SPAN></H3><P>For 250 mL M9 medium you need 175 mL sterile water (for plates add 4 g Agar-Agar as well). Then add (in the following order:
</P><UL><LI> 250 µL 100 mM CaCl<SUB>2</SUB></LI><LI> 2.5 mL trace salts
</LI><LI> store this stock solution in the dark
</LI></UL><LI> 250 µL MgSO<SUB>4</SUB></LI><LI> 250 µL 50 mM FeCl<SUB>3</SUB> / 100 mM citrate (one solution, citrate is iron carrier)
</LI><LI> store this stock solution cold and in the dark
</LI><LI> carbon source stock solution (<I>e.g.</I> glucose)
</LI><LI> 50 mL 5x M9 salts stock solution
</LI><LI> 64 g L<SUP>-1</SUP> Na<SUB>2</SUB>HPO<SUB>4</SUB> * 7 H<SUB>2</SUB>O
</LI><LI> 15 g L<SUP>-1</SUP> KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI> 2.5 g L<SUP>-1</SUP>NaCl
</LI><LI> 5 g L<SUP>-1</SUP> NH<SUB>4</SUB>Cl
</LI><LI> antibiotic stock solution
</LI><LI> fill up to 250 mL with sterile water
</LI><H3><SPAN class="mw-headline" id="HSG_medium">HSG medium</SPAN></H3><UL><LI> 14.9 g L<SUP>-1</SUP> glycerine
</LI><LI> 13.5 g L<SUP>-1</SUP> soy peptone
</LI><LI> 7 g L<SUP>-1</SUP> yeast extract
</LI><LI> 2.5 g L<SUP>-1</SUP> NaCl
</LI><LI> 2.3 g L<SUP>-1</SUP> K<SUB>2</SUB>HPO<SUB>4</SUB></LI><LI> 1.5 g L<SUP>-1</SUP> KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI> 0.249 g L<SUP>-1</SUP> MgSO<SUB>4</SUB> * 7 H<SUB>2</SUB>O
</LI></UL><H3><SPAN class="mw-headline" id="5x_isothermal_reaction_buffer_for_Gibson_assembly">5x isothermal reaction buffer for Gibson assembly</SPAN></H3><P>storage -20˚C
</P><UL><LI>3 mL of 1 M Tris-HCl (pH 7.5)
</LI><LI>150 µL of 2 M MgCl2,
</LI><LI>60 µL of 100 mM dGTP
</LI><LI>60 µL of 100 mM dATP
</LI><LI>60 µL of 100 mM dTTP,
</LI><LI>60 µL of 100 mM dCTP
</LI><LI>300 µL of 1 M DTT
</LI><LI>1.5 g PEG-8000 and
</LI><LI>300 µL of 100 mM NAD
</LI></UL><H3><SPAN class="mw-headline" id="Cold_osmotic_shock_buffers_for_the_release_of_periplasmic_protein_fraction">Cold osmotic shock buffers for the release of periplasmic protein fraction</SPAN></H3><P>Cell fractionating buffer #1 (pH 8)
</P><UL><LI>0,2 M Tris
</LI><LI>200 g L <SUP>-1</SUP> sucrose
</LI><LI>0,1 M EDTA
</LI></UL><P>Cell fractionating buffer #2 (pH 8)
</P><UL><LI>0,01 M Tris
</LI><LI>0,005 M MgSO<SUB>4</SUB></LI></UL><H3><SPAN class="mw-headline" id="Denaturation_buffer_for_inclusion_bodies">Denaturation buffer for inclusion bodies</SPAN></H3><UL><LI> 6 M urea
</LI><LI> 50 mM Tris-HCl
</LI><LI> 10 mM MgCl<SUB>2</SUB></LI></UL><H3><SPAN class="mw-headline" id="Buffers_for_S-layer_IEX">Buffers for S-layer IEX</SPAN></H3><P>Binding Buffer: 
</P><UL><LI> 25 mM Sodiumacetate, pH 6
</LI><LI> 25 mM NaCl
</LI></UL><P>Elution Buffer:
</P><UL><LI> 25 mM Sodiumacetate, pH 6
</LI><LI> 1 M NaCl
</LI></UL><H3><SPAN class="mw-headline" id="SDS-PAGE_gel">SDS-PAGE gel</SPAN></H3><P>The following amouts are for one gel.
Stacking gel 5 %:
</P><UL><LI>775 μL H<SUB>2</SUB>O
</LI><LI>1.25 mL 0,25 M Tris (pH 6,8)
</LI><LI>425 μL Bis/Acrylamide (0,8 %, 30 %)
</LI><LI>50 μL 5 % SDS
</LI><LI>25 μL 10 % Ammonium persulfate
</LI><LI>3 μL TEMED
</LI></UL><P>
Separating gel 12 %:
</P><UL><LI>1.5 mL H<SUB>2</SUB>O
</LI><LI>2.8 mL 1 M Tris (pH 8,8)
</LI><LI>3.0 mL Bis/ Acrylamide (0,8%, 30%)
</LI><LI>150 μL 5% SDS
</LI><LI>37.5 μL 10% Ammonium persulfate
</LI><LI>5 μL TEMED
</LI></UL><H3><SPAN class="mw-headline" id="SDS_running_buffer">SDS running buffer</SPAN></H3><UL><LI>25 mM Tris [pH 8,3]
</LI><LI>192 mM Glycerol
</LI><LI>0,1 % SDS
</LI></UL><H3><SPAN class="mw-headline" id="4x_Laemmli-buffer">4x Laemmli-buffer</SPAN></H3><UL><LI>250 mM Tris-HCl
</LI><LI>40 % [v/v] Glycerol
</LI><LI>20 % [v/v] 2-Mercapthoethanol
</LI><LI>80 g L<SUP>-1</SUP> SDS
</LI><LI>0,04 g L<SUP>-1</SUP> BPB
</LI></UL><H3><SPAN class="mw-headline" id="Colloidal_Coomassie_Brilliant_Blue_G-250_staining_solution">Colloidal Coomassie Brilliant Blue G-250 staining solution</SPAN></H3><P>for 1 L staining solution
</P><UL><LI>dissolve 50 g L<SUP>-1</SUP> (NH4)<SUB>2</SUB>SO<SUB>4</SUB> in ddH<SUB>2</SUB>O
</LI><LI>add 10 % (v/v) ethanol
</LI><LI>dissolve 0,2 g L<SUP>-1</SUP> Coomassie Brilliant Blue G-250
</LI><LI>add 2 % (v/v) phosphoric acid
</LI><LI>fill up to 1 L with ddH<SUB>2</SUB>O
</LI></UL><H3><SPAN class="mw-headline" id="Fairbanks_Coomassie_staining_solutions"> Fairbanks Coomassie staining solutions</SPAN></H3><P>Solution A:
</P><UL><LI>25 % (v/v) ispropanol
</LI><LI>10 % (v/v) acetic acid
</LI><LI>0,5 g L<SUP>-1</SUP> Coomassie Brilliant Blue R-250
</LI></UL><P>Solution B:
</P><UL><LI>10 % (v/v) ispropanol
</LI><LI>10 % (v/v) acetic acid
</LI><LI>0,05 g L<SUP>-1</SUP> Coomassie Brilliant Blue R-250
</LI></UL><P>Solution C:
</P><UL><LI>10 % (v/v) acetic acid
</LI><LI>0,025 g L<SUP>-1</SUP> Coomassie Brilliant Blue R-250
</LI></UL><P>Solution D:
</P><UL><LI>10 % (v/v) acetic acid
</LI></UL><H3><SPAN class="mw-headline" id="Silver_staining_solutions">Silver staining solutions</SPAN></H3><P>Fixation solution:
</P><UL><LI>50 % (v/v) ethanol
</LI><LI>12 % (v/v) acetic acid
</LI><LI>1 mL L<SUP>-1</SUP> formaldehyde (37 %)
</LI></UL><P>Thiosulfate solution:
</P><UL><LI>0.1 g L<SUP>-1</SUP> Na<SUB>2</SUB>S<SUB>2</SUB>O<SUB>3</SUB></LI></UL><P>Impregnation solution:
</P><UL><LI>2 g L<SUP>-1</SUP> silver nitrate (AgNO<SUB>3</SUB>)
</LI><LI>0.75 mL L<SUP>-1</SUP> formaldehyde (37 %)
</LI></UL><P>Developing solution:
</P><UL><LI>120 g L<SUP>-1</SUP> sodium carbonate (Na<SUB>2</SUB>CO<SUB>3</SUB>)
</LI><LI>1 mL L<SUP>-1</SUP> formaldehyde (37 %)
</LI></UL><P>Stop solution:
</P><UL><LI>18.6 g L<SUP>-1</SUP> EDTA
</LI></UL><H3><SPAN class="mw-headline" id="Binding_buffer">Binding buffer</SPAN></H3><UL><LI> 20 mM Na<SUB>3</SUB>PO<SUB>4</SUB>, pH 7,4
</LI><LI> 500 mM NaCl
</LI><LI> 20 mM Imidazole
</LI></UL><H3><SPAN class="mw-headline" id="NPI-10_.28lysis_buffer.29">NPI-10 (lysis buffer)</SPAN></H3><UL><LI> 50 mM NaH<SUB>2</SUB>PO<SUB>4</SUB>, pH 8,0
</LI><LI> 300 mM NaCl
</LI><LI> 10 mM Imidazole
</LI><LI> 2 mM PMSF
</LI></UL><H3><SPAN class="mw-headline" id="NPI-20_.28wash_buffer.29">NPI-20 (wash buffer)</SPAN></H3><UL><LI> 50 mM NaH<SUB>2</SUB>PO<SUB>4</SUB>, pH 8,0
</LI><LI> 300 mM NaCl
</LI><LI> 20 mM Imidazole
</LI></UL><H3><SPAN class="mw-headline" id="NPI-500_.28elution_buffer.29">NPI-500 (elution buffer)</SPAN></H3><UL><LI> 50 mM NaH<SUB>2</SUB>PO<SUB>4</SUB>, pH 8,0
</LI><LI> 300 mM NaCl
</LI><LI> 500 mM Imidazole
</LI></UL><H3><SPAN class="mw-headline" id="Deadenylation_buffer">Deadenylation buffer</SPAN></H3><UL><LI> 20 mM Tris-HCl, pH 7,5
</LI><LI> 50 mM NaCl
</LI><LI> 4 mM MgCl<SUB>2</SUB></LI><LI> 1 mM EDTA
</LI><LI> 1 mM DTT
</LI><LI> 1 mM NMN
</LI></UL><H3><SPAN class="mw-headline" id="DNA_ligase_buffer">DNA ligase buffer</SPAN></H3><UL><LI> 20 mM Tris-HCl, pH 7,5
</LI><LI> 50 mM NaCl
</LI><LI> 20 % [v/v] Glycerol
</LI></UL><H3><SPAN class="mw-headline" id="NAD.2B_bioassay_buffer">NAD<SUP>+</SUP> bioassay buffer</SPAN></H3><UL><LI> 50 mM Tris-HCl, pH 8,0
</LI><LI> 10 mM MgCl<SUB>2</SUB></LI><LI> 2,5 mM CaCl<SUB>2</SUB></LI><LI> 5 mM DTT
</LI><LI> 0,05 % BSA
</LI></UL><H3><SPAN class="mw-headline" id="HCCA_matrix_solution">HCCA matrix solution</SPAN></H3><UL><LI>Solvent:
</LI></UL><UL><LI>90 % ACN
</LI></UL><UL><LI>10 % H<SUB>2</SUB>O
</LI></UL><UL><LI>0,1 % TFA
</LI></UL><UL><LI>1 mM NH<SUB>4</SUB>H<SUB>2</SUB>PO<SUB>4</SUB></LI></UL><UL><LI>HCCA concentration: 0,7 mg mL<SUP>-1</SUP> (α-Cyano-4-hydroxycinnamic acid) 
</LI></UL><H2><SPAN class="mw-headline" id="Used_chemicals">Used chemicals</SPAN></H2><TABLE class="wikitable" style="text-align:left"><TBODY><TR><TH> Chemical </TH><TH style="padding-left:5px;"> Producer </TH><TH style="padding-left:5px;"> Purity
</TH></TR><TR><TD> Acetonitrile </TD><TD style="padding-left:5px;"><A href="https://de.vwr.com/app/catalog/Product?article_number=20048.320" class="external text" rel="nofollow">VWR</A></TD><TD style="padding-left:5px;"> 99.9 %, HPLC Grade
</TD></TR><TR><TD> Bisphenol A </TD><TD style="padding-left:5px;"><A href="http://www.sigmaaldrich.com/catalog/Lookup.do?N5=All&amp;N3=mode+matchpartialmax&amp;N4=bisphenol+a&amp;D7=0&amp;D10=bisphenol+a&amp;N1=S_ID&amp;ST=RS&amp;N25=0&amp;F=PR" class="external text" rel="nofollow">Sigma</A></TD><TD style="padding-left:5px;"> 97 %
</TD></TR><TR><TD> Bisphenol F </TD><TD style="padding-left:5px;"><A href="http://www.alfa.com/de/GP100W.pgm?DSSTK=A11417&amp;rnd=097380668" class="external text" rel="nofollow">Alfa Aesar</A></TD><TD style="padding-left:5px;"> 98 %
</TD></TR><TR><TD> Ethylacetate </TD><TD style="padding-left:5px;"><A href="https://de.vwr.com/app/catalog/Product?article_number=23882.321" class="external text" rel="nofollow">VWR</A></TD><TD style="padding-left:5px;">&gt; 99.5 %, <I>p.a.</I></TD></TR><TR><TD> Isopropyl β-D-1-thiogalactopyranoside </TD><TD style="padding-left:5px;"><A href="http://www.carlroth.com/catalogue/catalogue.do;jsessionid=D5D58FA281A3E75342D9198F07546935?id=17413&amp;favOid=000000010000911300020023&amp;act=showBookmark&amp;lang=de-de&amp;market=DE" class="external text" rel="nofollow">Roth</A></TD><TD style="padding-left:5px;"> ≥ 99 %
</TD></TR><TR><TD> L-rhamnose </TD><TD style="padding-left:5px;"><A href="http://www.sigmaaldrich.com/catalog/ProductDetail.do?lang=de&amp;N4=83650|FLUKA&amp;N5=SEARCH_CONCAT_PNO|BRAND_KEY&amp;F=SPEC" class="external text" rel="nofollow">Fluka</A></TD><TD style="padding-left:5px;"> ≥ 99 %
</TD></TR><TR><TD> Chemical A </TD><TD style="padding-left:5px;"><A href="http://www.link.com" class="external text" rel="nofollow">Producer A</A></TD><TD style="padding-left:5px;"> XX.X %
</TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Bielefeld-Germany/Backup-Protocols">http://2011.igem.org/Team:Bielefeld-Germany/Backup-Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Backup-Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Backup-Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Backup-Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Backup-Protocols&amp;oldid=121146" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>