"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-Germany/Backup-Protocols
From 2011.igem.org
Home
Team
Students
Supervisors
Photos
Bielefeld University
Acknowledgements
Contact
Project
Description
Achievements
Approach
Background
Bisphenol A
S-layer
NAD+ detection
Technology
DIY Nanobiotech Guide
Future Applications
Labjournal
Protocols
Overview
Molecular Genetics
Production
Analytics
Materials
Results
Data Page
Summary
Bisphenol A
S-layer
NAD+ detection
Since Regionals
Model
Human Practices
Overview
Summary
Cell-free Biosafety
Outreach
Open Lab Course
Science Cafe
High School
CeBiTec Symposium
Student Convention
Strategic Process
Beacon Box
Press
Safety
Partner
Contents
1 Transformation via electroporation
2 Transformation of Single Step (KRX) Competent Cells by Promega
3 Standard BioBrick Assembly
3.1 Suffix Insertion
3.2 Prefix Insertion
3.3 Variations
4 Standard Freiburg BioBrick Assembly
4.1 Suffix Insertion
4.2 Prefix Insertion
4.3 Variations
5 Standard 3A assembly
5.1 Digestion
5.2 Ligation
6 Freiburg 3A assembly
6.1 Digestion
6.2 Ligation
7 Gibson assembly
7.1 Preparation of DNA molecules for in vitro recombination
7.2 In vitro recombination
8 Restriction analysis
9 Colony PCR
10 Cultivation
10.1 Expression of different S-layer genes in E. coli
10.2 Expression of bisphenol A degrading BioBricks in E. coli
10.3 Bioreactor cultivations with E. coli KRX
11 Purification methods
11.1 Release of periplasmic protein fraction from E. coli by cold osmotic shock
11.2 Inclusion body clean-up
11.3 Ultra-/Diafiltration
11.4 Ion exchange chromatography (IEX) for S-layer proteins from Corynebacterium
11.5 Ni-NTA spin columns
11.6 His-tag affinity chromatography
12 Measuring of mRFP
13 Sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE)
13.1 Pouring the polyacrylamide gel
13.2 Preparing the sample
13.3 Running the gel
14 Polyacrylamide gel staining
14.1 Colloidal Coomassie Blue staining
14.2 Fairbanks Coomassie staining
14.3 Silver staining
15 Tryptic digest of gel lanes for analysis with MALDI-TOF
16 Preparation and Spotting for analysis of peptides on Bruker AnchorChips
17 Bisphenol A analysis
17.1 Extraction with ethylacetate
17.2 HPLC method
18 NAD+ detection
18.1 Characterisation of molecular beacons
18.2 Purification of DNA ligase
18.3 Deadenylation
18.4 NAD+ bioassay
19 Used enzymes
20 Used Kits
21 Media, buffer, solutions etc.
21.1 Ampicillin stock solution
21.2 Chloramphenicol stock solution
21.3 TAE buffer
21.4 DNA loading buffer
21.5 LB medium
21.6 Autoinduction medium for KRX
21.7 M9 medium
21.8 HSG medium
21.9 5x isothermal reaction buffer for Gibson assembly
21.10 Cold osmotic shock buffers for the release of periplasmic protein fraction
21.11 Denaturation buffer for inclusion bodies
21.12 Buffers for S-layer IEX
21.13 SDS-PAGE gel
21.14 SDS running buffer
21.15 4x Laemmli-buffer
21.16 Colloidal Coomassie Brilliant Blue G-250 staining solution
21.17 Fairbanks Coomassie staining solutions
21.18 Silver staining solutions
21.19 Binding buffer
21.20 NPI-10 (lysis buffer)
21.21 NPI-20 (wash buffer)
21.22 NPI-500 (elution buffer)
21.23 Deadenylation buffer
21.24 DNA ligase buffer
21.25 NAD+ bioassay buffer
21.26 HCCA matrix solution
22 Used chemicals
Transformation via electroporation
Thaw 50 µL competent E.coli cells on ice, dilute with icecold 50 µL glycerol (10 %) if necessary
Add 0.5-5 µL plasmid to 50 µl electrocompetent cells
Store cells on ice for 1 minute
Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ώ
Transfer transformation reaction to 450 µL SOC-Medium and shake 1 h at 37 °C
Centrifuge 2 min at 800 rpm and plate on selective LB-Medium
Transformation of Single Step (KRX) Competent Cells by Promega
using protocol for E. coli KRX single step competent cells by Promega
Remove Single Step (KRX) Competent Cells from –70 °C, and place on ice for 5 minutes or until just thawed.
Add 1–50 ng of DNA (in a volume not greater than 5 μL) to the Single Step (KRX) Competent Cells. Move the pipette tip through the cells while dispensing. Quickly flick the tube several times. Do not vortex!
Immediately return the tubes to ice for 5–30 minutes
Heat-shock cells for 15–20 seconds in a water bath at exactly 42 °C. Do not shake.
Immediately place the tubes on ice for 2 minutes.
Add 450 μL of room-temperature SOC medium to each transformation reaction, and incubate for 60 minutes at 37 °C with shaking (approximately 225 rpm). For best transformation efficiency, lay the tubes on their sides and tape them to the platform.
For each transformation reaction, we recommend plating 100 μL of undiluted cells and 1:10 and 1:100 cell dilutions on antibiotic plates. Incubate the plates at 37 °C overnight.
Standard BioBrick Assembly
modified from Silver lab :
This assembly method can be used for BioBricks which are bigger than 150 bp. The BioBrick should be at least 500 bp bigger or smaller than the backbone. The BioBrick, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other used BioBrick, called vector. So you have to differentiate between a prefix and a suffix insertion.
Silver Suffix Insertion
Silver Prefix Insertion
Suffix Insertion
Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x Tango buffer, 0.5 µL XbaI , 1 µL PstI . Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert.
Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL SpeI , 0.5 µL PstI . Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL SAP (shrimp alcaline phosphatase) and 1.2 µL 10 x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a PCR clean-up kit .
Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL T4-Ligase . Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
Prefix Insertion
Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x BamHI buffer, 0.5 µL EcoRI , 0.5 µL SpeI . Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert.
Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10 x Tango buffer, 0.5 µL EcoRI , 0.5 µL XbaI . Digest for 2h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL SAP (shrimp alcaline phosphatase) and 1.2 µL 10 x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a PCR clean-up kit .
Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL T4-Ligase . Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
Variations
A digestion over night is possible. If you digest over night use only 0.1 µL restriction enzyme.
It is also possible to use PCR product as insert. Digest after PCR with corresponding restriction enzymes and clean up with PCR clean-up kit. This could lead to higher yields of insert DNA because a lot of DNA gets lost during the gel electrophoresis clean up.
Sometimes some BioBricks are hard to assemble. Then you have to clean up the vector by gel electrophoresis as well.
Standard Freiburg BioBrick Assembly
Modified from Silver lab and Assembly standard 25 :
This assembly method can be used for fusion protein assemblies with BioBricks which are bigger than 150 bp. The BioBrick should be at least 500 bp bigger or smaller than the backbone. The BioBrick, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other used BioBrick, which is called vector and needs to be available in the BioBrick Assembly standard 25 . You have to differentiate between a prefix and a suffix insertion.
Suffix Insertion
Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x NEB buffer 4 + 0.1 µL 100x BSA, 0.5 µL NgoMIV (NEB) , 1 µL PstI . Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert.
Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL AgeI , 0.5 µL PstI . Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL SAP (shrimp alcaline phosphatase) and 1.2 µL 10x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a PCR clean-up kit .
Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL T4-Ligase . Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
Prefix Insertion
Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL EcoRI , 0.5 µL AgeI . Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert.
Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10 x NEB buffer 4, 0.5 µL EcoRI , 0.5 µL NgoMIV (NEB) . Digest for 2h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL SAP (shrimp alcaline phosphatase) and 1.2 µL 10 x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a PCR clean-up kit .
Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL T4-Ligase . Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
Variations
A digestion over night is possible. If you digest over night use only 0.1 µL restriction enzyme.
It is also possible to use PCR product as insert. Digest after PCR with corresponding restriction enzymes and clean up with PCR clean-up kit . This could lead to higher yields of insert DNA because a lot of DNA gets lost during the gel electrophoresis clean up.
Sometimes some BioBricks are hard to assemble. Then you have to clean up the vector by gel electrophoresis as well.
Standard 3A assembly
Modified from BioBrick Assembly Manual by Ginkgo BioWorks
Digestion
Thaw DNA from upstream and downstream part and the destination plasmid on ice.
Destination plasmid has to carry the ccdB gene <partinfo>P1010</partinfo> as insert and has to have a different antibiotic resistance than the plasmids carrying the upstream and downstream parts
DNA has to be cleaned (by MiniPrep or after a PCR )
500 ng DNA / digestion mix for upstream part, downstream part and destination plasmid (total volume of mix 10 µL, dilute with ddH20 if necessary)
Add 1 µL of 10x buffer and restriction enzymes as shown in the following table:
Upstream part
Downstream part
Destination plasmid
enzyme 1
0.5 µL EcoRI
0.5 µL XbaI
0.5 µL EcoRI
enzyme 2
0.5 µL SpeI
1 µL PstI
0.5 µL PstI
buffer
BamHI
Tango
Orange
Incubation of the digestion mixes at 37 °C
After 2 h: add 0.5 µL SAP (shrimp alcaline phosphatase) and 1.15 µL 10x SAP buffer to destination plasmid mix, continue incubation at 37 °C
After another hour: heat inactivation of all mixes for 20 min at 80 °C
Continue with ligation or freeze the mixes
Ligation
Ligation mix:
2 µL ddH2O
5 µL of every digestion mix (so 15 µL in total)
2 µL T4-DNA-ligase buffer (thaw on ice!)
1 µL T4-DNA-ligase
Incubate at least 20 min at room temperature, afterwards heat inactivation for 5 min at 70 °C (optional)
Freeze ligation mix or continue with transformation (heatshock or electroporation)
Freiburg 3A assembly
Modified from BioBrick Assembly Manual by Ginkgo BioWorks and Assembly standard 25
Digestion
Thaw DNA from upstream and downstream part (=N-terminal and C-terminal protein domain) and the destination plasmid on ice.
Destination plasmid has to carry the ccdB gene <partinfo>P1010</partinfo> as insert and has to have a different antibiotic resistance than the plasmids carrying the upstream and downstream parts
DNA has to be cleaned (by MiniPrep or after a PCR )
500 ng DNA / digestion mix for upstream part, downstream part and destination plasmid (total volume of mix 10 µL, dilute with ddH20 if necessary)
Add 1 µL of 10x buffer and restriction enzymes as shown in the following table:
Upstream part
Downstream part
Destination plasmid
enzyme 1
0.5 µL EcoRI
0.5 µL NgoMIV
0.5 µL EcoRI
enzyme 2
0.5 µL AgeI
1 µL PstI
0.5 µL PstI
buffer
Orange
NEB buffer 4 + BSA
Orange
Incubation of the digestion mixes at 37 °C
After 2 h: add 0.5 µL SAP (shrimp alcaline phosphatase) and 1.15 µL 10x SAP buffer to destination plasmid mix, continue incubation at 37 °C
After another hour: heat inactivation of all mixes for 20 min at 80 °C
Continue with ligation or freeze the mixes
Ligation
Ligation mix:
2 µL ddH2O
5 µL of every digestion mix (so 15 µL in total)
2 µL T4-DNA-ligase buffer (thaw on ice!)
1 µL T4-DNA-ligase
Incubate at least 20 min at room temperature, afterwards heat inactivation for 5 min at 70 °C (optional)
Freeze ligation mix or continue with transformation (heatshock or electroporation)
Gibson assembly
Modified from Gibson et al. (2009)
Gibson Assembly
This assembly method is an isothermal, single-reaction method for assembling multiple overlapping DNA molecules. By coordinating the activity of a 5‘ exonuclease, a DNA polymerase and a DNA ligase two adjacent DNA fragments with complementary terminal sequence overlaps can be joined into a covalently sealed molecule, without the use of any restriction endonuclease.
Preparation of DNA molecules for in vitro recombination
Generate the complementary sequence overlaps by PCR using the Phusion DNA-polymerase . If necessary add 5 M Betain in the reaction mix by reducing the amount of H2O to decrease the number of false PCR products.
Identify the PCR products of interest by gel electrophoresis with known DNA standards.
Extract the PCR products from the gel by cutting out the DNA fragments and clean them up by using a commercial clean up kit .
In vitro recombination
assembly mixture (6 mL):
320 µL 5x isothermal reaction buffer
0.64 µL of 10 U mL-1 T5 exonuclease (for DNA molecules overlapping by greater than 150 bp add 3.2 µL of 10 U ml–1 T5 exonuclease )
20 µL of 2 U mL-1 Phusion DNA polymerase
160 µL of 40 U mL-1 taq DNA ligase
add ddH2O water up to a final volume of 1.2 mL
aliquote 15 µL of the reagent-enzyme mix and store it at –20 ˚C
Thaw 15 µL assembly mixture aliquot and keep it on ice until use.
Add 5 µL of the purified DNA molecules in equimolar amounts (between 10 and 100 ng of each DNA fragment).
Incubate the resulting mixture at 50 ˚C for 15 to 60 min, with 60 min being optimal.
Transformation (heatshock or electroporation ) without cleaning up the assembly product.
Restriction analysis
Digest BioBrick of interest: about 400 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL NotI or PstI . Digest for 2 h at 37 °C. NotI is used to determine the length of the BioBrick and the plasmid backbone, PstI ist used to determine the length of the BioBrick in the plasmid backbone.
Gel electrophoresis: add 2 µL loading buffer to every digestion mix, apply about 100 - 200 ng DNA / pocket in gel. Don't forget to apply the uncut BioBrick as well. A good agarose concentration for BioBricks between 0.2 and 3 kb is 1.5 %. The smaller your BioBrick of interest is the higher the agarose concentration should be and vice versa. The gel electrophoresis is made with TAE-buffer . Be sure that you melt your agarose gel in the same buffer you use for the electrophoresis later.
Colony PCR
Pick one colony with a sterile tip and elute it in 100 µL ddH20 or medium
Store the colony in 4 °C while colony PCR is running
One reaction mix contains:
10 µL 5x buffer
2 µL MgCl2 (25 mM stock)
1 µL dNTPs
0.5 µL primer mix (prefix/suffix primers or sequencing primers)
35.25 µL ddH2O
0.25 µL GoTaq polymerase (Promega)
1 µL template
PCR program:
Start: 3 min, 98 °C
30 cycles of:
30 s, 98 °C
30 s, 55 °C
30 s / 1 kb template, 72 °C
Finish: 5 min, 72 °C
Gel electrophoresis: check the fragment size
Plate the correct colony
Cultivation
Expression of different S-layer genes in E. coli
Used BioBricks: <partinfo>K525131</partinfo>, <partinfo>K525133</partinfo>, <partinfo>K525231</partinfo>, <partinfo>K525232</partinfo>, <partinfo>K525233</partinfo>, <partinfo>K525234</partinfo>, <partinfo>K525304</partinfo>, <partinfo>K525305</partinfo>, <partinfo>K525306</partinfo>, <partinfo>K525406</partinfo>, <partinfo>K525323</partinfo>
Chassis: Promega's E. coli KRX
Medium: LB medium supplemented with 20 mg L-1 Chloramphenicol or autoinduction medium
Cultivations in LB-medium were supplemented with 0,1 % L-rhamnose as inducer, when the designated OD600 was reached.
Autoinduction medium for expressing <partinfo>K525304</partinfo>, <partinfo>K525305</partinfo>, <partinfo>K525306</partinfo>, <partinfo>K525406</partinfo>, <partinfo>K525323</partinfo> was supplemented with 1 mM IPTG .
For characterising the expression rate and the influance on E. coli growth behavior an automatic sampling system (Gilson fraction controller F2XX cooled (< 4 °C) with Julabo F10 water bath BU) was used. These cultivations were carried out in an Infors AG AQUATRON.
150 mL culture in 500 mL shaking flask with baffles (Schott) with silicon plugs
Cultivation conditions: 37 °C at 120 rpm
Expression of bisphenol A degrading BioBricks in E. coli
Used BioBricks: <partinfo>K525502</partinfo>, <partinfo>K525512</partinfo>, <partinfo>K525517</partinfo>, <partinfo>K525552</partinfo>
Chassis: Promega's E. coli KRX
Medium: LB medium supplemented with 100 mg L-1 Ampicillin and 120 mg L-1 bisphenol A
BPA is thermally stable -> you can autoclave it together with the medium
100 mL culture in 300 mL shaking flask without baffles (Schott) with silicon plugs
Cultivation temperature: 24 °C, 30 °C or 37 °C, tempered with Infors AG AQUATRON at 120 rpm
for characterizations: automatic sampling every three hours with Gilson fraction controller F2XX cooled (< 4 °C) with Julabo F10 water bath BURMA-SHAVE!!
the characterization experiment setup is shown on the picture on the right
Bioreactor cultivations with E. coli KRX
To obtain higher amounts and concentration of proteins we cultivated and expressed in a bioreactor. It is possible to cultivate several liters and to control temperature, pH and DO.
Bioreactor: Bioengineering NLF22 7 L with Bioengineering DCU
Medium: HSG medium supplemented with 20 mg L-1 chloramphenicol
Culture volume: 4 L
Starting OD600: 0.2
DO: 40 % airsaturation (controlled with stirrer cascade starting with 200 rpm)
pH: 7.0 (controlled with 20 % phosphoric acid and 2 M NaOH)
Antifoam: BASF pluronic XXX
Induction after 4 h cultivation time with 2 % rhamnose (in culture medium)
Harvest after 13 h
Purification methods
Release of periplasmic protein fraction from E. coli by cold osmotic shock
Modified protocol from Neu & Heppel, 1965 .
Centrifuge E. coli cell suspension for 5 min at 14000 g (4 °C) to collect the cells.
Discard the entire supernatant.
Resuspend the cells ice-cold cell fractionating buffer #1 . The resulting volume should be 1/4 of the former suspension volume.
Incubate for 20 min on ice. Ivert the suspension at regular intervals to counteract sedimentation.
Centrifuge the cell suspension for 15 min at 14000 g (4 °C).
Discard the entire supernatant.
Resuspend the cells ice-cold cell fractionating buffer #2 . The resulting volume should be 1/4 of the former suspension volume.
Incubate for 10 min on ice under regular invertion.
Centrifuge the cell suspension for 15 min at 14000 g (4 °C).
Save the supernatant, which contains the periplasmatic proteins.
If the periplasmatic protein fraction is turbid, re-centrifuge and filter it through a 0.2 µm filter.
Inclusion body clean-up
harvest the cells by centrifugation (30 min, 10,000 g, 4 °C)
resuspend pellet and disrupt cells
centrifuge lysate (60 min, >17,000 g, 4 °C)
wash pellet at least two times with water to remove water-soluble proteins
after washing the pellet: incubate the pellet in denaturation buffer for 60 min, 4 °C with vertical rotator
final concentration in denaturation buffer: 0.5 mg wet biomass per mL
centrifuge (60 min, >17,000 g, 4 °C)
the higher the speed, the better the result
collect supernatant and incubate the pellet again in denaturation buffer (60 min, 4 °C, vertical rotator)
centrifuge (60 min, >17,000 g, 4 °C)
collect supernatant and discard pellet
Ultra-/Diafiltration
Arrange the filtration module as shown on the right side.
Microfiltration (0.22 µm) or cross flow filtration 300 kDa ( we used a Milipore Pellicon XL 300) membrane of sample before ultrafiltration.
For concentrating the sample just filter it until the desired volume is left in the feed reservoir. For diafiltration (e.g. buffer exchange, desalting) dilute the feed reservoir several times and filter continously.
Used membranes: Milipore Pellicon XL 50 or XL 100 membranes
50 or 100 kDa cut-off
50 cm2 filtration area
tangential flow filter
Hydrophilic polyvinylidene fluoride membrane
Used pump: SciLog TANDEM 1081 peristaltic pump
flow rate during filtration: 40 mL min-1
Ion exchange chromatography (IEX) for S-layer proteins from Corynebacterium
used column: DEAE HiTrap 1 mL with GE Healthcare ÄKTA
flow rate: 1 mL min-1
equilibrate column with 5 column volumes of binding buffer
inject sample and wash column with 5 column volumes of binding buffer
elute S-layer with 40 % binding / elution buffer mix
elute remaining proteins with 100 % elution buffer
Ni-NTA spin columns
denaturating
non-denaturating
His-tag affinity chromatography
Column: 1 mL HisTrap FF crude by GE Healthcare
Harvest cells by centrifugation (10000 g, 10 min, 4 °C)
Discard supernatant and freeze the pellet at -20 °C for at least 30 min
Resuspend pellet in 5 mL of binding buffer for each gram of cell paste
Wash column with 5 - 10 mL deionized water
Equilibrate column with 5 - 10 mL of binding buffer
denaturing
Mechanical lysis:
Sonication on ice (approx. 5 min) with Sonifier 450 by Branson
Centrfigate for at 10000 g for 30 min at 4 °C
non-denaturing
Enzymatic lysis:
Add 0.2 mg L-1 lysozyme, 3 Units per mL culture volume and 1 mM MgCL2
Stir for 30 min at 4 °C
Centrfigate for at 10000 g for 30 min at 4 °C
Measuring of mRFP
Take at least 500 µL sample for each measurement (200 µL is needed for one measurement) so you can perform a repeat determination
Freeze samples at -80 °C for storage
To measure the samples thaw at room temperature and fill 200 µL of each sample in one well of a black, flat bottom 96 well microtiter plate (perform at least a repeat determination)
Measure the fluorescence in a platereader (we used a Tecan Infinite® M200 platereader ) with following settings:
20 sec orbital shaking (1 mm amplitude with a frequency of 87.6 rpm)
Measurement mode: Top
Excitation: 584 nm
Emission: 620 nm
Number of reads: 25
Manual gain: 100
Integration time: 20 µs
Sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE)
This analytical Method can be used for separation and identification of proteins according to their electrophoretic mobility. The mobility is a function of length of the molecular weight. Proteins that have identical charge per unit mass due to binding of SDS results in an equal electrophoretic mobility.
Pouring the polyacrylamide gel
Make a master mix for the stacking and separating gel without adding ammonium persulfate and TEMED.
Aliquote 6,5 mL for each separating and 2,5 mL for each stacking gel.
Add ammonium persulfate and TEMED to each separating gel aliquote and pour the solution quickly into your gel casting form. Leave about 2 centimeters below the bottom of the comb for the stacking gel.
Layer isopropanol on top of the gel.
Leave the separating gel at room temperature for >60 minutes to polymerize.
Remove isopropanol and wait until the surface is dry.
Add ammonium persulfate and TEMED to each separating gel aliquote and pour the solution quickly into your gel casting form.
Insert comb without getting bubbles stuck underneath
Leave the gel at room temperature for >60 minutes to polymerize.
For storage
Remove sealing and store the gel wrapped in moistened paper towel at 4°C.
Preparing the sample
Mix your protein mixture 4:1 with Laemmli-buffer (30 mL protein solution + 10 mL Laemmli-buffer)
Heat for 5 minutes at 95 °C.
Running the gel
Remove sealing, put the polymerized gel into gel box and pour SDS running buffer into the negative and positive electrode chamber.
Remove comp without destroying the gel pocket.
Pipet the sample into the gel pockets, adjusting the volume according to the amount of protein in your sample. Make sure to include a lane with molecular weight standards (PageRuler Prestained Protein Ladder™ (Fa. Fermentas)) to determinate the molecular weight of your sample.
Connect the power lead and run the stacking gel with 10 mA until the blue dye front enters the separating gel.
Raise amperage up to 20 mA for running the separating gel.
When the distance of the lowest molecular weight standard lane to the gel end is down to 0,5 cm stop the electrophoresis by turning off the power supply.
Polyacrylamide gel staining
Colloidal Coomassie Blue staining
Modified staining protocol from Kang et al., 2002 .
agitate the staining solution at 37 °C over night to form the colloids
After finishing the SDS-PAGE remove gel from gel casting form and tranfer it in to a box.
Add 100 mL of the stainig solution to your polyacrylamid gel.
Incubate the gel in the solution at room temperature until the protein bands got an intensive blue color. Shake the gel continuously during incubation.
Remove the staining solution
wash the gel with dH2O
Incubate the gel in ddH2O (2-6h) for bleaching the background. Shake the gel continuously during incubation. If necessary replace the colored water with new one.
Fairbanks Coomassie staining
Modiefid steining protocol from Fairbanks et al., (1971).
After finishing the SDS-PAGE remove gel from gel casting form and tranfer it in to a box with 100 mL of solution A.
Heat the gel in a microwave for 2 min at 600 W.
Shake the gel for 5-10 minutes.
Remove solution A and wash the gel with dH2O.
Add 100 mL of solution B and heat the gel in a microwave for 2 min at 600 W.
Shake the gel for 5-10 minutes.
Remove solution B and wash the gel with dH2O.
Add 100 mL of solution C and heat the gel in a microwave for 2 min at 600 W.
Shake the gel for 5-10 minutes.
Remove solution C and wash the gel with dH2O.
For bleaching the background add 100 mL of solution D and heat the gel in a microwave for 2 min at 600 W.
Shake the gel for 5-10 minutes.
Remove solution D and wash the gel with dH2O. If necessary repeat the last 3 steps with solution D.
Silver staining
Modified staining protocol from
After finishing the SDS-PAGE remove gel from gel casting form and tranfer it in to a box.
Add 50 mL of fixation solution and incubate the gel 1 to 24 h. The formaldehyde (37 %) must be added to the solution short time before.
Remove the fixation solution and wash the gel 3 times (1-20 min) with  50 mL of 50 % (v/v) ethanol.
Add 50 mL of thiosulfate solution and incubate the gel exactly for 1 min. If the gel incubates to long the background  becomes to dark.
Remove thiosulfate solution and wash the gel 3 times (20 s) with dH2O.
Add 50 mL of impregnation solution and icubate the gel for 15 to 20 min. The formaldehyde (37 %) must be added to the solution short time before.
Remove impregnation solution and wash the gel 2 times (20 s) with dH2O.
Add 50 mL of developing solution and incubate the gel until protein bands become visible.
Remove the developing solution immediately and wash the gel for 10 to 20 s with dH2O.
Add 50 mL of stop solution and incubate the gel 10 to 20 s.
Remove the stop solution and wash the gel for 1 min with dH2O.
Tryptic digest of gel lanes for analysis with MALDI-TOF
Note:
Make sure to work under a fume hood.
Do not work with protective gloves to prevent contamination of your sample with platicizers.
Reaction tubes have to be cleaned with 60% (v/v) CH3CN, 0,1% (v/v) TFA. Afterwards the solution has to be removed completely followed by evaporation of the tubes under a fume hood. Alternatively microtiter plates from Greiner® (REF 650161) can be used without washing.
Cut out the protein lanes of a Coomassie-stained SDS-PAGE using a clean scalpel. Gel parts are transferred to the washed reaction tubes/microtiter plate. If necessary cut the parts to smaller slices.
Gel slices should be washed two times. Therefore add 200 µL 30% (v/v) acetonitrile in 0,1 M ammonium hydrogen carbonate each time and shake lightly for 10 minutes. Remove supernatant and discard to special waste.
Dry gel slices at least 30 minutes in a Speedvac.
Rehydrate gel slices in 15 µL Trypsin-solution followed by short centrifugation.
Gel slices have to be incubated 30 minutes at room temperature, followed by incubation at 37 °C overnight.
Dry gel slices at least 30 minutes in a Speedvac.
According to the size of the gel slice, add 5 – 20 µL 50% (v/v) ACN / 0,1% (v/v) TFA.
Samples can be used for MALDI measurement or stored at -20 °C.
Trypsin-solution: 1 µL Trypsin + 14 µL 10 mM NH4HCO3
Therefore solubilize lyophilized Trypsin from Promega® in 200 µL of provided buffer and incubate for 15 minutes at 30 °C for activation. For further use it can be stored at -20 °C.
Preparation and Spotting for analysis of peptides on Bruker AnchorChips
Spot 0,5 – 1 µL sample aliquot
Add 1 µL HCCA matrix solution to the spotted sample aliquots. Pipet up and down approximately five times to obtain a sufficient mixing. Be careful not to contact the AnchorChip.
Note: Most of the sample solvent needs to be gone in order to achieve a sufficiently low water content. When the matrix solution is added to the previously spotted sample aliquot at a too high water content in the mixture, it will result in undesired crystallization of the matrix outside the anchor spot area.
Dry the prepared spots at room temperature
Spot external calibrants on the adjacent calibrant spot positions. Use the calibrant stock solution (Bruker’s “Peptide Calibration Standard II”, Part number #222570), add 125 µL of 0,1% TFA (v/v) in 30% ACN to the vial. Vortex and sonicate the vial.
Mix the calibrant stock solution in a 1:200 ratio with HCCA matrix and deposit 1 µL of the mixture onto the calibrant spots.
Bisphenol A analysis
Extraction with ethylacetate
mix 100 µL culture supernatant with 100 µL internal standard ( bisphenol F , 100 µg L-1)
add 200 µL ethylacetate (HPLC grade) for extraction
vortex (30 s)
centrifuge for phase separation (5 min, 5000 g)
take a bit from upper phase and put it in a clean eppi
SpeedVac at 40 °C to remove ethlyacetate
solve remaining BPA in water (HPLC grade), vortex (30 s)
solubility of BPA in water only 300 mg L-1
for LC-MS analysis of BPA, 300 mg BPA L-1 is definitely enough
if you want to detect or expect higher concentrations of BPA, solve it in an acetonitrile-water-mix
HPLC method
C18 reverse phase column
Isocratic method: 45 % Acetonitrile
Flow = 0.6 mL min-1
UV-detection at 227 nm
Internal standard: 100 mg L-1 Bisphenol F (BPF)
Column:
Eurospher II 100-5 C18p by Knauer
Dimensions: 150 x 4.6 mm with precolumn
Particle size: 5 µm
Pore size: 100 Å
Material: silica gel
Software:
Clarity (Version 3.0.5.505) by Data Apex
Autosampler:
Midas by Spark Holland
Tray cooling: 10 °C
Pump:
L-6200A Intelligent Pump by Hitachi
UV-Detector:
Series 1050 by Hewlett Packard
Chromatogram obtained using the above described materials and methods. Three distinctive peaks are of interest: BPA at 7,3 min retention time, BPF at 5,8 min retention time and the degradation product of BPA at 4,6 min retention time.
NAD+ detection
For the design of molecular beacons different available tools may help you so that a correct functionality is ensured under defined assay conditions. All measurements were made in the NAD+ bioassay buffer .
Characterisation of molecular beacons
Preferentially, use a complementary oligonucleotide (target) as well as two enshortened oligonucleotides (split target) for characterising the molecular beacon in its closed and open state. In this way you can verify simultaneously the ineffectiveness of the split target to reach the molecular beacon`s open state.
Optimal wavelengths:
Determine the emission spectrum of your molecular beacon by initially using a literature value for the exctintion wavelength of the labeled fluorophore. For this add 10 µL of 1 µM molecular beacon in 200 µL NAD+ bioassay buffer , mix thoroughly and wait 2 min until measuring the fluorescence with a spectrofluorometer at an defined temperature (optimally 37 °C) .
Add 10 µL of 2 µM either target or split target and shake at least 6 min until measuring the fluorescence again (when equilibrium is reached).
Repeat the procedure to determine the extinction spectrum using the self-calculated optimal emission wavelength for your fluorophore/quencher combination. For further fluorescence measurements use the extinction wavelength for which the difference between the molecular beacons's closed state and the open state signals is maximal.
Thermal profile analysis:
Prepare a 30 µL reaction mix composed of NAD+ bioassay buffer and 1 µM molecular beacon. In a separate sample add 2 µM either target or split target.
Use a spectrofluorometric thermal cylcer to dertemine the fluorescence as a function of temperature. Start at 80 °C and decrease the temperature stepwise in 1 °C intervals holding each temperature for 1 min and measuring the fluorescence at the end of each step.
Signal-to-background ratio (S/B):
Signal-to-background ratio
The approach is quite similar to the one that determines the optimal wavelengths which should be used to monitor the fluorescence of the NAD+ bioassay buffer (Fbuffer) and the added molecular beacon (Fclosed) as well as the target (Fopen) or split target. Monitor the fluorescence at the optimal wavelengths until the equilibrium is reached each time before you add a new component.
The signal-to-background ratio can be calculated as shown on the right. Finally, compare the signal-to-background ratios after adding the target and split target to check whether the split target has an effect on the melting of the molecular beacon's hairpin structure.
Imaging:
Prepare a 200 µL reaction mix composed of NAD+ bioassay buffer and 500 nM molecular beacon in a PCR tube. In a separate sample add 600 µM either target or split target. Excitate the samples by a UV transilluminator and take images preferably with a camera that can detect different colours.
Purification of DNA ligase
The purification of the overexpressed NAD+-dependent DNA ligase gene ( BBa_K525710 ) in E. coli was performed under native conditions and Ni-NTA columns were used utilizing the recombinant protein`s C-terminal 6xHis-tag.
Cultivation
Prepare an overnight culture of E. coli KRX carrying the plasmid with DNA ligase ( BBa_K525710 ) in 30 ml LB medium containing 20 μg mL-1 chloramphenicol at 37 °C.
Dilute the overnight culture in 100 ml autoinduction medium (20 μg mL-1 chloramphenicol added) to an OD600 = 0,1 and harvest the cells after 4 h growth at 37 °C.
Use the centrifuged cells immediately for protein purification or store them at -20 °C.
Small-scale purification
Resuspend a pellet derived from 5 ml cell culture volume in 630 μl NPI-10 .
Add 70 μl lysozyme (10 mg/ml) as well as 25 Units benzonase nuclease and mix thoroughly.
Incubate on ice for 30 min.
Centrifuge the lysate for 30 min at 12000 g (4 °C). Collect the supernatant (cleared lysate).
Prepare a Ni-NTA spin column ( QIAGEN ) by equilibration with 600 μl NPI-10 and centrifugation for 2 min at 890 g (4 °C).
Load up to 600 μl cleared lysate onto the Ni-NTA spin column and centrifuge for 5 min at 270 g (4 °C). Collect the flow-through.
Wash the Ni-NTA spin column with 600 μl NPI-20 and centrifuge for 2 min at 890 g (4 °C). Repeat this step and collect flow-through each time.
Elute the protein with 300 μl NPI-500 and centrifuge for 2 min at 890 g (4 °C). Repeat this step and collect flow-through each time.
For centrifugal ultrafiltration use Vivaspin 500 concentrators ( Sartorius ) with a 5000 molecular weight cut-off PES membrane. Repeat the procedure three times by adding DNA ligase buffer (4 °C) for buffer exchange (remove of imidazole).
Check each purification step and especially the purity of the protein in the final condition by SDS-PAGE analysis .
Determine the protein concentration with a Bradford Protein Assay .
For long-time storage keep the protein at -20 °C.
If the binding conditions are not effective enough you can reduce the imidazole concentration in NPI-10 to 1-5 mM. If the eluate contains unspecifically bound proteins maybe a washing step with NPI-10 containing 50 mM or 100 mM imidazole could help to get higher purity.
Large-scale purification
Resuspend a cell culture pellet in 5 ml binding buffer per gramm biomass.
Add 10 mM MgCl2, 1 mg/ml lysozyme] as well as 5 Units benzonase nuclease per 1 ml cell culture volume and mix thoroughly.
Incubate at 4 °C for 30 min.
Centrifuge the lysate for 30 min at 10000 g (4 °C). Collect the supernatant (cleared lysate).
Prepare a HisTrap™ FF crude column ( GE Healthcare ) by equilibration with 5 column volumes of binding buffer . The following steps have to be performed at 4 °C.
Load the cleared lysate onto the HisTrap™ FF crude column and collect flow-through.
Wash the HisTrap™ FF crude column with 10 column volumes of binding buffer and collect flow-through.
Elute the protein successively with 5 ml of buffers that contain different concentrations of imidazole (e.g. binding buffer with 40, 60, 100, 300 and 500 mM imidazole) and collect flow-through in 1 ml fractions.
For centrifugal ultrafiltration use Vivaspin 15 concentrators ( Sartorius ) with a 10000 molecular weight cut-off PES membrane. Repeat the procedure three times by adding DNA ligase buffer (4 °C) for buffer exchange (remove of imidazole).
Check each purification step and especially the purity of the protein in the final condition by SDS-PAGE analysis .
Determine the protein concentration with a Bradford Protein Assay .
For long-time storage keep the protein at -20 °C.
Deadenylation
The majority of purified DNA ligase from E. coli KRX is usually in the adenylated form. But only the apoenzyme is really useful for the NAD+ bioassay so that the AMP moiety has to be removed.
Mix the purified DNA ligase with deadenylation buffer and incubate for 10 min at 37 °C.
Use Vivaspin 15 concentrators ( Sartorius ) with a 10000 molecular weight cut-off PES membrane for buffer exchange at 4 °C. Repeat the procedure at least three times by adding DNA ligase buffer (remove of NMN and formed NAD+).
Check the deadenylation process by SDS-PAGE analysis . If the deadenylation was successful there should not be a double band indicating both forms of DNA ligase.
Determine the protein concentration with a Bradford Protein Assay .
For long-time storage keep the protein at -20 °C.
NAD+ bioassay
Prepare a 80 μl aliquot composed of 250 nM molecular beacon and 250 nM split target in NAD+ bioassay buffer .
Incubate for 8 min at 37 °C.
Add 6 Units of deadenylated DNA ligase (approximately ... ng).
Monitor the fluorescence with a spectrofluorometer at 37 °C and wait until it reaches equilibrium.
Add NAD+ in different final concentrations (e.g. 0, 0.3, 0.5, 5, 10, 30, 40, 80, 200, 300, 500, 1000 nM), stirr shortly and monitor the fluorescence at 37 °C again.
Determine the initial enhancement rate of flourescence intensity for each NAD+ concentration.
For the calibration curve plot the initial velocity against the NAD+ concentration.
Used enzymes
Enzyme
Producer
AgeI
Fermentas
DpnI
Fermentas
EcoRI
Fermentas
GoTaq DNA-polymerase
Promega
KOD Hotstart DNA-polymerase
Novagen
NgoMIV
NEB
OneTaq DNA-polymerase
NEB
Pfu DNA-polymerase
Promega
PstI
Fermentas
Phusion HF DNA-polymerase
Finnzymes
Shrimp alcaline phosphatase
Fermentas
SpeI
Fermentas
T4-DNA-Ligase
Fermentas
T5 exonuclease
NEB
taq DNA Ligase
NEB
taq DNA-polymerase
Bioline
Used Kits
Function
Name
Molecular Cloning
Fermentas CloneJET™ PCR Cloning Kit
Plasmid purification
Fermentas GeneJET™ Plasmid Miniprep Kit
Plasmid purification
Promega PureYield™ Plasmid Preps
PCR Cleanup
Macherey Nagel NucleoSpin® Extract II
PCR Cleanup
Promega Wizard® SV Gel and PCR Clean-Up
PCR core system
Promega GoTaq® PCR Core System I
Media, buffer, solutions etc.
Ampicillin stock solution
Solubilize 100 mg mL-1 Ampicillin
Store at -20 °C
Chloramphenicol stock solution
Solubilize 20 mg mL-1 Chloramphenicol
Store at -20 °C
TAE buffer
For 1 L of 50 x TAE buffer you need:
242.48 g Tris
41.02 g Sodiumacetate
18.612 g EDTA
Adjust pH to 7.8 with acetic acid
Solve in dH2O
10 mL of the stock is diluted in 1 L dH2O for the gel electrophoresis (0.5 x TAE buffer).
DNA loading buffer
50 % (v/v) glycerol
1 mM EDTA
0.1 % (w/v) bromphenol blue
Solve in ddH2O
LB medium
For 1 L of LB medium you need:
10 g Trypton
5 g yeast extract
10 g NaCl
12 g Agar-Agar (for plates)
Adjust pH to 7.4
Autoinduction medium for KRX
This medium is based on the LB medium .
After heat sterilization of 900 mL add the following chemicals
5 mL of a 200 g L-1 steril L-rhamnose stock solution (final concentration 2 g L-1 L-rhamnose)
2,5 mL of a 200 g L-1 steril glucose stock solution (final concentration 0,5 g L-1 glucose)
if necessary add antibiotics
Cm: 1 mL of a 20 μg mL-1 Cm stock solution (final concentration 2 μg L-1)
Amp: 1 mL of a 100 μg mL-1 Amp stock solution (final concentration 100 μg L-1)
fill-up to 1 L with steril ddH2O
M9 medium
For 250 mL M9 medium you need 175 mL sterile water (for plates add 4 g Agar-Agar as well). Then add (in the following order:
250 µL 100 mM CaCl2
2.5 mL trace salts
store this stock solution in the dark
250 µL MgSO4
250 µL 50 mM FeCl3 / 100 mM citrate (one solution, citrate is iron carrier)
store this stock solution cold and in the dark
carbon source stock solution (e.g. glucose)
50 mL 5x M9 salts stock solution
64 g L-1 Na2HPO4 * 7 H2O
15 g L-1 KH2PO4
2.5 g L-1 NaCl
5 g L-1 NH4Cl
antibiotic stock solution
fill up to 250 mL with sterile water
HSG medium
14.9 g L-1 glycerine
13.5 g L-1 soy peptone
7 g L-1 yeast extract
2.5 g L-1 NaCl
2.3 g L-1 K2HPO4
1.5 g L-1 KH2PO4
0.249 g L-1 MgSO4 * 7 H2O
5x isothermal reaction buffer for Gibson assembly
storage -20˚C
3 mL of 1 M Tris-HCl (pH 7.5)
150 µL of 2 M MgCl2,
60 µL of 100 mM dGTP
60 µL of 100 mM dATP
60 µL of 100 mM dTTP,
60 µL of 100 mM dCTP
300 µL of 1 M DTT
1.5 g PEG-8000 and
300 µL of 100 mM NAD
Cold osmotic shock buffers for the release of periplasmic protein fraction
Cell fractionating buffer #1 (pH 8)
0,2 M Tris
200 g L -1 sucrose
0,1 M EDTA
Cell fractionating buffer #2 (pH 8)
0,01 M Tris
0,005 M MgSO4
Denaturation buffer for inclusion bodies
6 M urea
50 mM Tris-HCl
10 mM MgCl2
Buffers for S-layer IEX
Binding Buffer:
25 mM Sodiumacetate, pH 6
25 mM NaCl
Elution Buffer:
25 mM Sodiumacetate, pH 6
1 M NaCl
SDS-PAGE gel
The following amouts are for one gel. Stacking gel 5 %:
775 μL H2O
1.25 mL 0,25 M Tris (pH 6,8)
425 μL Bis/Acrylamide (0,8 %, 30 %)
50 μL 5 % SDS
25 μL 10 % Ammonium persulfate
3 μL TEMED
Separating gel 12 %:
1.5 mL H2O
2.8 mL 1 M Tris (pH 8,8)
3.0 mL Bis/ Acrylamide (0,8%, 30%)
150 μL 5% SDS
37.5 μL 10% Ammonium persulfate
5 μL TEMED
SDS running buffer
25 mM Tris [pH 8,3]
192 mM Glycerol
0,1 % SDS
4x Laemmli-buffer
250 mM Tris-HCl
40 % [v/v] Glycerol
20 % [v/v] 2-Mercapthoethanol
80 g L-1 SDS
0,04 g L-1 BPB
Colloidal Coomassie Brilliant Blue G-250 staining solution
for 1 L staining solution
dissolve 50 g L-1 (NH4)2SO4 in ddH2O
add 10 % (v/v) ethanol
dissolve 0,2 g L-1 Coomassie Brilliant Blue G-250
add 2 % (v/v) phosphoric acid
fill up to 1 L with ddH2O
Fairbanks Coomassie staining solutions
Solution A:
25 % (v/v) ispropanol
10 % (v/v) acetic acid
0,5 g L-1 Coomassie Brilliant Blue R-250
Solution B:
10 % (v/v) ispropanol
10 % (v/v) acetic acid
0,05 g L-1 Coomassie Brilliant Blue R-250
Solution C:
10 % (v/v) acetic acid
0,025 g L-1 Coomassie Brilliant Blue R-250
Solution D:
10 % (v/v) acetic acid
Silver staining solutions
Fixation solution:
50 % (v/v) ethanol
12 % (v/v) acetic acid
1 mL L-1 formaldehyde (37 %)
Thiosulfate solution:
0.1 g L-1 Na2S2O3
Impregnation solution:
2 g L-1 silver nitrate (AgNO3)
0.75 mL L-1 formaldehyde (37 %)
Developing solution:
120 g L-1 sodium carbonate (Na2CO3)
1 mL L-1 formaldehyde (37 %)
Stop solution:
18.6 g L-1 EDTA
Binding buffer
20 mM Na3PO4, pH 7,4
500 mM NaCl
20 mM Imidazole
NPI-10 (lysis buffer)
50 mM NaH2PO4, pH 8,0
300 mM NaCl
10 mM Imidazole
2 mM PMSF
NPI-20 (wash buffer)
50 mM NaH2PO4, pH 8,0
300 mM NaCl
20 mM Imidazole
NPI-500 (elution buffer)
50 mM NaH2PO4, pH 8,0
300 mM NaCl
500 mM Imidazole
Deadenylation buffer
20 mM Tris-HCl, pH 7,5
50 mM NaCl
4 mM MgCl2
1 mM EDTA
1 mM DTT
1 mM NMN
DNA ligase buffer
20 mM Tris-HCl, pH 7,5
50 mM NaCl
20 % [v/v] Glycerol
NAD+ bioassay buffer
50 mM Tris-HCl, pH 8,0
10 mM MgCl2
2,5 mM CaCl2
5 mM DTT
0,05 % BSA
HCCA matrix solution
Solvent:
90 % ACN
10 % H2O
0,1 % TFA
1 mM NH4H2PO4
HCCA concentration: 0,7 mg mL-1 (α-Cyano-4-hydroxycinnamic acid)
Used chemicals
Chemical
Producer
Purity
Acetonitrile
VWR
99.9 %, HPLC Grade
Bisphenol A
Sigma
97 %
Bisphenol F
Alfa Aesar
98 %
Ethylacetate
VWR
> 99.5 %, p.a.
Isopropyl β-D-1-thiogalactopyranoside
Roth
≥ 99 %
L-rhamnose
Fluka
≥ 99 %
Chemical A
Producer A
XX.X %
Retrieved from " http://2011.igem.org/Team:Bielefeld-Germany/Backup-Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
