<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu-igem"><UL id="menu" class="sf-menu"><LI><A style="border-left:0px;" href="https://2011.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Students">Students</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Advisors">Supervisors</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Photos">Photos</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Bielefeld-University‎‎">Bielefeld University</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></UL><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Description">Description</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Achievements">Achievements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Appoach">Approach</A></LI><LI><A href="#">Background</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/Technology">Technology</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer/Guide">DIY Nanobiotech Guide</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Future">Future Applications</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Labjournal">Labjournal</A></LI><LI><A href="#">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Genetics">Molecular Genetics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing">Production</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Analytics">Analytics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials">Materials</A></LI><LI><A href="#">Results</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Data_Page">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/SinceAmsterdam">Since Regionals</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Modell">Model</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Cell-free">Cell-free Biosafety</A></LI><LI><A href="#">Outreach</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_lab_course">Open Lab Course</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_science_cafe">Science Cafe</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/high-school">High School</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/cebitec-symposium">CeBiTec Symposium</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/student-convention">Student Convention</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/strategic-process">Strategic Process</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Black-box">Beacon Box</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Press">Press</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Bisphenol_A_degradation"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Bisphenol A degradation</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Sequencing_results"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Sequencing results</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Bisphenol_A_degradation_with_E._coli"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Bisphenol A degradation with E. coli</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Modelling_of_intracellular_bisphenol_A_degradation"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Modelling of intracellular bisphenol A degradation</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Interpretation_of_the_results"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Interpretation of the results</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Summary_of_results"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Summary of results</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#References"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Bisphenol_A_degradation">Bisphenol A degradation</SPAN></H1><H2><SPAN class="mw-headline" id="Sequencing_results">Sequencing results</SPAN></H2><P>The iGEM team from the University of Alberta sent in BioBricks for BPA degradation in 2008 (&lt;partinfo&gt;K123000&lt;/partinfo&gt; and &lt;partinfo&gt;K123001&lt;/partinfo&gt;). Sequencing of these BioBricks by the iGEM HQ and by us led to negative results so the sequences entered into the partsregistry are not correct although these sequences are the ones from <I>Sphingomonas bisphenolicum</I> AO1 (compare <A href="http://www.ncbi.nlm.nih.gov/nuccore/AB255167.1" class="external text" rel="nofollow">genbank entry</A>). In addition, there are &quot;illegal&quot; AgeI and NgoMIV restriction sites in the BioBrick which are used for Freiburg BioBrick assembly standard (<A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A>). After translating and comparing the original sequences from the partsregistry and the sequences from our sequencing results <I>in silico</I> we saw, that the amino acid sequences were identically. These BioBricks were probably synthesized in the Freiburg assembly standard 25 because they have the accordant restriction sites and they were codon optimized for <I>Escherichia coli</I> but the original sequence from <I>S. bisphenolicum</I> AO1 were entered into the registry because amino acid sequence of the real sequence and the sequence that was entered are identical. The alignments are shown in fig. 1 - 4. 
</P><CENTER><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>Fig. 1: Alignment of the DNA sequence of &lt;partinfo&gt;K123000&lt;/partinfo&gt; entered into the partsregistry (K123000 registry) with our sequencing results (K123000 real) for this BioBrick (made with Clonemanager).
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>Fig. 2: Alignment of the aminoacid sequence (translated <I>in silico</I>) of &lt;partinfo&gt;K123000&lt;/partinfo&gt; entered into the partsregistry (K123000 registry) with our sequencing results (K123000 real) for this BioBrick (made with Clonemanager).
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>Fig. 3: Alignment of the DNA sequence of &lt;partinfo&gt;K123001&lt;/partinfo&gt; entered into the partsregistry (K123001 registry) with our sequencing results (K123001 real) for this BioBrick (made with Clonemanager).
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>Fig. 4: Alignment of the aminoacid sequence (translated <I>in silico</I>) of &lt;partinfo&gt;K123001&lt;/partinfo&gt; entered into the partsregistry (K123001 registry) with our sequencing results (K123001 real) for this BioBrick (made with Clonemanager).
</P></DIV></DIV></TD></TR></TBODY></TABLE></CENTER><H2><SPAN class="mw-headline" id="Bisphenol_A_degradation_with_E._coli">Bisphenol A degradation with <I>E. coli</I></SPAN></H2><P>The bisphenol A degradation with the BioBricks &lt;partinfo&gt;K123000&lt;/partinfo&gt; and &lt;partinfo&gt;K123001&lt;/partinfo&gt; works in <I>E. coli</I> KRX in general. Because <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.2008.03843.x/full" class="external text" rel="nofollow">Sasaki <I>et al.</I> (2008)</A> reported problems with protein folding in <I>E. coli</I> which seem to avoid a complete BPA degradation, we did not cultivate at 37 °C and we did not use the strong T7 promoter as <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.2008.03843.x/full" class="external text" rel="nofollow">Sasaki <I>et al.</I> (2008)</A> did for expressing these BioBricks but we cultivated at 30 °C and we used a medium strong constitutive promoter (&lt;partinfo&gt;J23110&lt;/partinfo&gt;). 30 °C is in addition the cultivation temperature of <I>S. bisphenolicum</I> AO1. With this promoter upstream of a polycistronic <I>bisdAB</I> gene we were able to completely degrade 120 mg L<SUP>-1</SUP> BPA in about 30 - 33 h. By fusing &lt;partinfo&gt;K123000&lt;/partinfo&gt; and &lt;partinfo&gt;K123001&lt;/partinfo&gt; together (using Freiburg BioBrick assembly standard) we could <B>improve the BPA degradation</B> of <I>E. coli</I> even further, so 120 mg L<SUP>-1</SUP> BPA can be degraded in 21 - 24 h. This data is shown in the following figure:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption"><B>Figure 5: BPA degradation by <I>E. coli</I> KRX carrying genes for BisdA and BisdB (only <I>bisdA</I> (black), polycistronic <I>bisdAB</I> (red) and fusion protein between BisdA and BisdB (green)) behind the medium strong constitutive promoter &lt;partinfo&gt;J23110&lt;/partinfo&gt; with RBS &lt;partinfo&gt;B0034&lt;/partinfo&gt;. <A href="/Team:Bielefeld-Germany/Protocols/Downstream-processing#Expression_of_bisphenol_A_degrading_BioBricks_in_E._coli" title="Team:Bielefeld-Germany/Protocols/Downstream-processing"> Cultivations</A> were carried out at 30 °C in LB + Amp + BPA medium for 24 h and 36 h, respectively, with automatic sampling every three hours in 300 mL shaking flasks without baffles with silicon plugs. At least three biological replicates were analysed (three for <I>bisdA</I> alone, seven for <I>bisdAB</I> polycistronic and five for the fusion protein).</B></DIV></DIV></DIV></DIV><P>We also carried out these cultivations at different temperatures and BPA concentrations, but the chosen conditions (30 °C and 120 mg L<SUP>-1</SUP> BPA) seem to be the best. Higher BPA concentrations have an effect on the growth of <I>E. coli</I> and higher temperature leeds to a worse BPA degradation (probably due to misfolding of the enzymes). Lower temperature also leeds to less BPA degradation (probably due to slower growth, expression and reaction rate at lower temperatures). These data on the effect of the temperature on the BPA degradation is shown in fig. 6. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption"><B>Figure 6: BPA degradation by <I>E. coli</I> KRX carrying genes for BisdA and BisdB (polycistronic <I>bisdAB</I> (black) and fusion protein between BisdA and BisdB (striped)) behind the medium strong constitutive promoter &lt;partinfo&gt;J23110&lt;/partinfo&gt; with RBS &lt;partinfo&gt;B0034&lt;/partinfo&gt;. <A href="/Team:Bielefeld-Germany/Protocols/Downstream-processing#Expression_of_bisphenol_A_degrading_BioBricks_in_E._coli" title="Team:Bielefeld-Germany/Protocols/Downstream-processing"> Cultivations</A> were carried out at different temperatures in LB + Amp + BPA medium (starting concentration 120 mg L<SUP>-1</SUP> BPA)  for 24 h in 300 mL shaking flasks without baffles with silicon plugs. Samples were taken at the end of the cultivation. Three biological replicates were analysed. </B></DIV></DIV></DIV></DIV><P>As shown by <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.2008.03843.x/full" class="external text" rel="nofollow">Sasaki <I>et al.</I> (2008)</A>, BisdB expressed in <I>E. coli</I> leeds to hardly no BPA degradation. In our experiments we could not detect the BPA degradation products 1,2-Bis(4-hydroxyphenyl)-2-propanol and 2,2-Bis(4-hydroxyphenyl)-1-propanol in cultivations with <I>E. coli</I> expressing &lt;partinfo&gt;K123000&lt;/partinfo&gt; or &lt;partinfo&gt;K123001&lt;/partinfo&gt; alone (neither via UV- nor MS-detection). The BPA degradation products 1,2-Bis(4-hydroxyphenyl)-2-propanol and 2,2-Bis(4-hydroxyphenyl)-1-propanol were identified via MS-MS (<I>m/z</I>: 243 / 225 / 211 / 135) and only occured in cultivations with <I>E. coli</I> expressing BisdA and BisdB together. <A href="http://www.springerlink.com/content/q7864l02734wg32m/" class="external text" rel="nofollow">Sasaki <I>et al.</I> (2005)</A> reported the same MS-MS results for 1,2-Bis(4-hydroxyphenyl)-2-propanol and 2,2-Bis(4-hydroxyphenyl)-1-propanol when degrading BPA with <I>S. bisphenolicum</I> AO1 as we observed in our BPA degradation experiments.
</P><P>We could also identify the BPA degradation products when working with <I>E. coli</I> TOP10 and MACH1 (data not shown) but because we want to fuse BisdA and BisdB to S-layer proteins which we express in <I>E. coli</I> KRX the characterizations of &lt;partinfo&gt;K123000&lt;/partinfo&gt; and &lt;partinfo&gt;K123001&lt;/partinfo&gt; were carried out in this strain.
</P><H2><SPAN class="mw-headline" id="Modelling_of_intracellular_bisphenol_A_degradation">Modelling of intracellular bisphenol A degradation</SPAN></H2><P>The modelling was done with the software <A href="http://www.berkeleymadonna.com/" class="external text" rel="nofollow">Berkeley Madonna</A> using the <A href="http://en.wikipedia.org/wiki/Runge–Kutta_methods#Common_fourth-order_Runge.E2.80.93Kutta_method" class="external text" rel="nofollow">common fourth-order Runge-Kutta</A> method to solve the equations. The model was fitted to the measured data shown above by the function &quot;curve fit&quot; in Berkeley Madonna to calculate the parameters, constants <I>etc</I>.
</P><P>To model the BPA degradation by <I>E. coli</I> carrying BioBricks for BPA degradation (&lt;partinfo&gt;K123000&lt;/partinfo&gt; and &lt;partinfo&gt;K123001&lt;/partinfo&gt;) the cell growth has to be described first to calculate a specific BPA degradation rate per cell. The observed growth of <I>E. coli</I> on (our) LB medium was <A href="http://en.wikipedia.org/wiki/Diauxie" class="external text" rel="nofollow">diauxic</A> with two different growth phases. Cell growth is a <A href="http://en.wikipedia.org/wiki/First_order_kinetics#First-order_reactions" class="external text" rel="nofollow">first-order reaction</A> and is mathematically described as 
</P><DIV align="right">(1)</DIV><P>
with the <B>specific growth rate</B><B>µ</B> and the <B>cell count</B><B>X</B>. The specific growth rate is dependent on the concentration of the growth limiting substrate (e.g. glucose) and can be described as 
</P><DIV align="right">(2)</DIV><P>
with the <B>substrate concentration</B><B>S</B>, the <B>Monod constant</B><B>K<SUB>S</SUB></B> and the <B>maximal specific growth rate</B><B>µ<SUB>max</SUB></B> (<A href="http://www.annualreviews.org/doi/abs/10.1146/annurev.mi.03.100149.002103" class="external text" rel="nofollow">Monod, 1949</A>). Because LB medium is a complex medium we cannot measure the substrate concentration so we have to assume an imaginary substrate concentration. Due to the diauxic growth two different substrates with different Monod constants and consumption rates are necessary to model the cell growth. The amount of a substrate S can be modelled as follows
</P><DIV align="right">(3)</DIV><P>
with the <B>specific substrate consumption rate per cell</B><B>q<SUB>S</SUB></B>. The whole model for the diauxic growth of <I>E. coli</I> on LB medium with two not measurable (imaginary) substrates looks like: 
</P><DIV align="right">(4)</DIV><P>
The <B>specific BPA degradation rate per cell</B><B>q<SUB>D</SUB></B> is modelled with an equation like eq. (3). In the beginning of the cultivations, when <I>E. coli</I> growths on the &quot;good&quot; imaginary substrate S<SUB>1</SUB>, no BPA degradation is observed. When this substrate is consumed, the BPA degradation starts. The model for this diauxic behavior is as follows:
</P><DIV align="right">(5)</DIV><P>
Fig. 7 shows a comparison between modelled and measured data for cultivations with BPA degrading <I>E. coli</I>. In Tab. 1 the parameters for the model are given, obtained by curve fitting the model to the data. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption"><B>Fig. 7: Comparison between modelled (lines) and measured (dots) data for <A href="/Team:Bielefeld-Germany/Protocols/Downstream-processing#Expression_of_bisphenol_A_degrading_BioBricks_in_E._coli" title="Team:Bielefeld-Germany/Protocols/Downstream-processing"> cultivations</A> of <I>E. coli</I> KRX carrying BPA degrading BioBricks. The BioBricks &lt;partinfo&gt;K525512&lt;/partinfo&gt; (polycistronic <I>bisdAB</I> genes behind medium strong promoter, shown in black) and &lt;partinfo&gt;K525517&lt;/partinfo&gt; (fusion protein between BisdA and BisdB, expressed with medium strong promoter, shown in red) were cultivated at least five times in <I>E. coli</I> KRX in LB + Amp + BPA medium at 30 °C, using 300 mL shaking flasks without baffles with silicon plugs. The BPA concentration (closed dots) and the cell density (open dots) is plotted against the cultivation time. </B></DIV></DIV></DIV></DIV><CENTER><P><B>Tab. 1: Parameters of the model. </B></P><TABLE cellpadding="10" style="border-collapse: collapse; border-width: 1px; border-style: solid; border-color: #000"><TBODY><TR><TH style="border-style: solid; border-width: 1px"> Parameter
</TH><TH style="border-style: solid; border-width: 1px">&lt;partinfo&gt;K525512&lt;/partinfo&gt;</TH><TH style="border-style: solid; border-width: 1px">&lt;partinfo&gt;K525517&lt;/partinfo&gt;</TH></TR><TR><TH style="border-style: solid; border-width: 1px"> X<SUB>0</SUB></TH><TD style="border-style: solid; border-width: 1px">0.112 10<SUP>8</SUP> mL<SUP>-1</SUP></TD><TD style="border-style: solid; border-width: 1px">0.138 10<SUP>8</SUP> mL<SUP>-1</SUP></TD></TR><TR><TH style="border-style: solid; border-width: 1px"> µ<SUB>max</SUB></TH><TD style="border-style: solid; border-width: 1px">1.253 h<SUP>-1</SUP></TD><TD style="border-style: solid; border-width: 1px">1.357 h<SUP>-1</SUP></TD></TR><TR><TH style="border-style: solid; border-width: 1px"> K<SUB>S,1</SUB></TH><TD style="border-style: solid; border-width: 1px">2.646 AU<SUP>-1</SUP></TD><TD style="border-style: solid; border-width: 1px">1.92 AU<SUP>-1</SUP></TD></TR><TR><TH style="border-style: solid; border-width: 1px"> K<SUB>S,2</SUB></TH><TD style="border-style: solid; border-width: 1px">265.1 AU<SUP>-1</SUP></TD><TD style="border-style: solid; border-width: 1px">103.1 AU<SUP>-1</SUP></TD></TR><TR><TH style="border-style: solid; border-width: 1px"> S<SUB>1,0</SUB></TH><TD style="border-style: solid; border-width: 1px">1.688 AU
</TD><TD style="border-style: solid; border-width: 1px">1.166 AU
</TD></TR><TR><TH style="border-style: solid; border-width: 1px"> q<SUB>S,1</SUB></TH><TD style="border-style: solid; border-width: 1px">0.478 AU 10<SUP>-8</SUP> cell<SUP>-1</SUP></TD><TD style="border-style: solid; border-width: 1px">0.319 AU 10<SUP>-8</SUP> cell<SUP>-1</SUP></TD></TR><TR><TH style="border-style: solid; border-width: 1px"> S<SUB>2,0</SUB></TH><TD style="border-style: solid; border-width: 1px">16.091 AU
</TD><TD style="border-style: solid; border-width: 1px">6.574 AU
</TD></TR><TR><TH style="border-style: solid; border-width: 1px"> q<SUB>S,2</SUB></TH><TD style="border-style: solid; border-width: 1px">0.295 AU 10<SUP>-8</SUP> cell<SUP>-1</SUP></TD><TD style="border-style: solid; border-width: 1px">0.191 AU 10<SUP>-8</SUP> cell<SUP>-1</SUP></TD></TR><TR><TH style="border-style: solid; border-width: 1px"> BPA<SUB>0</SUB></TH><TD style="border-style: solid; border-width: 1px">0.53 mM
</TD><TD style="border-style: solid; border-width: 1px">0.53 mM
</TD></TR><TR><TH style="border-style: solid; border-width: 1px"> q<SUB>D</SUB></TH><TD style="border-style: solid; border-width: 1px">8.76 10<SUP>-11</SUP> mM cell<SUP>-1</SUP></TD><TD style="border-style: solid; border-width: 1px">1.29 10<SUP>-10</SUP> mM cell<SUP>-1</SUP></TD></TR></TBODY></TABLE></CENTER><P>The specific BPA degradation rate per cell q<SUB>D</SUB> is about 50 % higher when using the fusion protein compared to the polycistronic <I>bisdAB</I> gene. This results in an average 9 hours faster, complete BPA degradation by <I>E. coli</I> carrying &lt;partinfo&gt;K525517&lt;/partinfo&gt; compared to &lt;partinfo&gt;K525512&lt;/partinfo&gt; as observed during <A href="/Team:Bielefeld-Germany/Results#Bisphenol_A_degradation_with_E._coli" title="Team:Bielefeld-Germany/Results"> our cultivations</A>. The fusion protein between BisdA and BisdB improves the BPA degradation by <I>E. coli</I>.
</P><H2><SPAN class="mw-headline" id="Interpretation_of_the_results">Interpretation of the results</SPAN></H2><P>Misfolding seems to be a problem when expressing BisdA and BisdB in <I>E. coli</I>. To reduce this, the cultivation conditions were improved for the BPA degradation with the polycistronic <I>bisdAB</I> gene in <I>E. coli</I> first, compared to the literature (<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.2008.03843.x/full" class="external text" rel="nofollow">Sasaki <I>et al.</I>, 2008</A>). Problems with misfolding of BisdA and BisdB could be reduced by lowering the temperature and growth rate and by using a weaker promoter for expression. 
</P><P>When degrading BPA with <I>E. coli</I> using the BisdA | BisdB fusion protein, both domains (BisdA and BisdB) are active and correctly folded because otherwise there would be no BPA degradation measured and no BPA degradation products 1,2-Bis(4-hydroxyphenyl)-2-propanol and 2,2-Bis(4-hydroxyphenyl)-1-propanol could be detected, which could definetely be identified via MS-MS. The about 50 % higher specific BPA degradation rate in <I>E. coli</I> expressing BisdA | BisdB fusion protein could be explained either by improved folding properties of the fusion protein or by the closer distance of BisdA and BisdB in the fusion protein leeding to a faster electron transfer and therefore a more efficient reaction. In this case, which has to be further analysed by cell-free enzyme assays, a natural cytochrome P450 class I electron transport system was converted into a more effective class V electron transport system, demonstrating the new possibilities of synthetic biology. In addition, this new class V system would work without alanine-rich linker, potentially changing the view on cytochrome P450 depending electron transport chains. 
</P><H2><SPAN class="mw-headline" id="Summary_of_results">Summary of results</SPAN></H2><CENTER><P><B>Tab. 2: Important parameters of &lt;partinfo&gt;K525512&lt;/partinfo&gt; and &lt;partinfo&gt;K525517&lt;/partinfo&gt;.</B></P><TABLE cellpadding="10" style="border-collapse: collapse; border-width: 1px; border-style: solid; border-color: #000"><TBODY><TR><TH style="border-style: solid; border-width: 1px">Experiment
</TH><TH style="border-style: solid; border-width: 1px">Characteristic
</TH><TH style="border-style: solid; border-width: 1px">Result K525512
</TH><TH style="border-style: solid; border-width: 1px">Result K525517
</TH></TR><TR><TH rowspan="4" style="border-style: solid; border-width: 1px">Expression in <I>E. coli</I></TH><TD style="border-style: solid; border-width: 1px">Compatibility
</TD><TD colspan="2" style="border-style: solid; border-width: 1px"><I>E. coli</I> KRX, TOP10, MACH1, BL21(DE3)
</TD></TR><TR><TD style="border-style: solid; border-width: 1px">Expression
</TD><TD colspan="2" style="border-style: solid; border-width: 1px">Constitutive
</TD></TR><TR><TD style="border-style: solid; border-width: 1px">Optimal temperature
</TD><TD colspan="2" style="border-style: solid; border-width: 1px">30 °C
</TD></TR><TR><TD style="border-style: solid; border-width: 1px">BPA working concentration
</TD><TD colspan="2" style="border-style: solid; border-width: 1px">120 mg L<SUP>-1</SUP> (0.53 mM)
</TD></TR><TR><TH rowspan="3" style="border-style: solid; border-width: 1px">Purification
</TH><TD style="border-style: solid; border-width: 1px">Molecular weight
</TD><TD style="border-style: solid; border-width: 1px">59.3 kDa
</TD><TD style="border-style: solid; border-width: 1px">11.2 and 48.3 kDa
</TD></TR><TR><TD style="border-style: solid; border-width: 1px">Theoretical pI
</TD><TD style="border-style: solid; border-width: 1px">4.99
</TD><TD style="border-style: solid; border-width: 1px">4.31 and 5.27
</TD></TR><TR><TD style="border-style: solid; border-width: 1px">High absorbtion
</TD><TD colspan="2" style="border-style: solid; border-width: 1px">450 nm (due to CYP)
</TD></TR><TR><TH rowspan="2" style="border-style: solid; border-width: 1px">Degradation of BPA
</TH><TD style="border-style: solid; border-width: 1px">Completely degradation of 0.53 mM BPA
</TD><TD style="border-style: solid; border-width: 1px">21 - 24 h
</TD><TD style="border-style: solid; border-width: 1px">30 - 33 h
</TD></TR><TR><TD style="border-style: solid; border-width: 1px">Specific BPA degradation rate
</TD><TD style="border-style: solid; border-width: 1px">1.29 10<SUP>-10</SUP> mM cell<SUP>-1</SUP></TD><TD style="border-style: solid; border-width: 1px">8.76 10<SUP>-11</SUP> mM cell<SUP>-1</SUP></TD></TR></TBODY></TABLE></CENTER><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>Monod J (1949) The growth of bacterial cultures, <I>Annu Rev Microbiol</I><A href="http://www.annualreviews.org/doi/abs/10.1146/annurev.mi.03.100149.002103" class="external text" rel="nofollow">3:371-394</A>.
</P><P>Sasaki M, Maki J, Oshiman K, Matsumura Y, Tsuchido T (2005) Biodegradation of bisphenol A by cells and cell lysate from <I>Sphingomonas</I> sp. strain AO1, <A href="http://www.springerlink.com/content/q7864l02734wg32m/" class="external text" rel="nofollow"><I>Biodegradation</I> 16(5):449-459</A>.
</P><P>Sasaki M, Tsuchido T, Matsumura Y (2008) Molecular cloning and characterization of cytochrome P450 and ferredoxin genes involved in bisphenol A degradation in <I>Sphingomonas bisphenolicum</I> strain AO1, <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.2008.03843.x/full" class="external text" rel="nofollow"><I>J Appl Microbiol</I> 105(4):1158-1169</A>.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Bielefeld-Germany/Results">http://2011.igem.org/Team:Bielefeld-Germany/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results&amp;oldid=144671" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>