<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Labjournal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Labjournal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Labjournal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Labjournal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Labjournal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu-igem"><UL id="menu" class="sf-menu"><LI><A style="border-left:0px;" href="https://2011.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Students">Students</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Advisors">Supervisors</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Photos">Photos</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Bielefeld-University‎‎">Bielefeld University</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></UL><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Description">Description</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Achievements">Achievements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Appoach">Approach</A></LI><LI><A href="#">Background</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/Technology">Technology</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer/Guide">DIY Nanobiotech Guide</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Future">Future Applications</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Labjournal">Labjournal</A></LI><LI><A href="#">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Genetics">Molecular Genetics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing">Production</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Analytics">Analytics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials">Materials</A></LI><LI><A href="#">Results</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Data_Page">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/SinceAmsterdam">Since Regionals</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Modell">Model</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Cell-free">Cell-free Biosafety</A></LI><LI><A href="#">Outreach</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_lab_course">Open Lab Course</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_science_cafe">Science Cafe</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/high-school">High School</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/cebitec-symposium">CeBiTec Symposium</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/student-convention">Student Convention</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/strategic-process">Strategic Process</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Black-box">Beacon Box</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Press">Press</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></LI></DIV><P><B>Labjournal:</B> On this page we summarize the (successful) results and achievements of our teamwork. 
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_1:_2nd_-_8th_May"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 1: 2nd - 8th May</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Week_2:_9th_-_15th_May"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Week 2: 9th - 15th May</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Week_3:_16th_-_22nd_May"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Week 3: 16th - 22nd May</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Week_4:_23rd_-_29th_May"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Week 4: 23rd - 29th May</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Week_5:_30th_May_-_5th_June"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Week 5: 30th May - 5th June</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Week_6:_6th_-_12th_June"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Week 6: 6th - 12th June</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Week_7:_13th_-_19th_June"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Week 7: 13th - 19th June</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Week_8:_20th_-_26th_June"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Week 8: 20th - 26th June</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Week_9:_27th_June_-_3rd_July"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Week 9: 27th June - 3rd July</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Week_10:_4th_July_-_10th_July"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Week 10: 4th July - 10th July</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Week_11:_11th_July_-_17th_July"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Week 11: 11th July - 17th July</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Week_12:_18th_July_-_24th_July"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Week 12: 18th July - 24th July</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Week_13:_25th_July_-_31th_July"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Week 13: 25th July - 31th July</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Week_14:_1st_August_-_7th_August"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Week 14: 1st August - 7th August</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#Week_15:_8th_August_-_14th_August"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">Week 15: 8th August - 14th August</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Week_16:_15th_August_-_21st_August"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Week 16: 15th August - 21st August</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Week_17:_22nd_August_-_28th_August"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">Week 17: 22nd August - 28th August</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Week_18:_29th_August_-_4th_September"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">Week 18: 29th August - 4th September</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Week_19:_5th_September_-_11th_September"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">Week 19: 5th September - 11th September</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Week_20:_12th_September_-_18th_September"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">Week 20: 12th September - 18th September</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#Week_21:_19th_September_-_25th_September"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">Week 21: 19th September - 25th September</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Week_22:_26th_September_-_2nd_October"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">Week 22: 26th September - 2nd October</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#Week_23:_3rd_October_-_9th_October"><SPAN class="tocnumber">23</SPAN><SPAN class="toctext">Week 23: 3rd October - 9th October</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Week_24:_10th_October_-_16th_October"><SPAN class="tocnumber">24</SPAN><SPAN class="toctext">Week 24: 10th October - 16th October</SPAN></A></LI><LI class="toclevel-1 tocsection-25"><A href="#Week_25:_17th_October_-_23th_October"><SPAN class="tocnumber">25</SPAN><SPAN class="toctext">Week 25: 17th October - 23th October</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#Week_26:_24th_October_-_30th_October"><SPAN class="tocnumber">26</SPAN><SPAN class="toctext">Week 26: 24th October - 30th October</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Week_1:_2nd_-_8th_May">Week 1: 2nd - 8th May</SPAN></H2><P>Bisphenol A:
</P><UL><LI> cloning of &lt;partinfo&gt;K123000&lt;/partinfo&gt; and &lt;partinfo&gt;K123001&lt;/partinfo&gt; behind weak (&lt;partinfo&gt;J23103&lt;/partinfo&gt;) and medium strong (&lt;partinfo&gt;J23110&lt;/partinfo&gt;) constitutive promoter (each part and both parts polycistronic)
</LI><LI> cloning of fusion protein between &lt;partinfo&gt;K123000&lt;/partinfo&gt; and &lt;partinfo&gt;K123001&lt;/partinfo&gt;, also assembly behind weak and medium strong constitutive promoter
</LI><LI> testing and establishing of HPLC method for <A href="http://en.wikipedia.org/wiki/Bisphenol_A" class="external text" rel="nofollow">BPA</A> detection
</LI><LI> expression of the successfully assembled BPA degrading BioBricks in <I>E. coli</I><A href="http://openwetware.org/wiki/E._coli_genotypes#TOP10_.28Invitrogen.29" class="external text" rel="nofollow">TOP10</A></LI></UL><P>S-layer:
</P><UL><LI> successful PCR on the S-layer genes of <A href="http://expasy.org/sprot/hamap/CORGL.html" class="external text" rel="nofollow"><I>Corynebacterium glutamicum</I></A> and <I>Corynebacterium crenatum</I></LI><LI> successful cloning of the S-layer gene of <I>Brevibacterium flavum</I> without TAT-sequence
</LI></UL><P>Organizational:
</P><UL><LI> meeting with <A href="http://www.bielefeld-marketing.de/de/index.html" class="external text" rel="nofollow">Bielefeld Marketing</A> to discuss our participation at the science festival <A href="http://www.geniale-bielefeld.de/" class="external text" rel="nofollow">GENIALE</A> in Bielefeld
</LI></UL><H2><SPAN class="mw-headline" id="Week_2:_9th_-_15th_May">Week 2: 9th - 15th May</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 1: HPLC results of the first experiment on BPA degradation in <I>E. coli</I> TOP10. Cultivations were carried out in LB medium with 100 mg L<SUP>-1</SUP> BPA at 30 °C. Samples were taken at the beginning of the cultivation and after one day. The HPLC results are shown above (area of the BPA peak). BisdA + BisdB is the polycistronic gene and BisdABisdB is the fusion protein. </B></DIV></DIV></DIV><P>Bisphenol A: 
</P><UL><LI> assembly of &lt;partinfo&gt;K157011&lt;/partinfo&gt; behind existing BPA degrading parts (for purification and testing of &lt;partinfo&gt;K123000&lt;/partinfo&gt; and &lt;partinfo&gt;K123001&lt;/partinfo&gt; in a cell free system)
</LI><LI> HPLC results: Fusion protein between &lt;partinfo&gt;K123000&lt;/partinfo&gt; and &lt;partinfo&gt;K123001&lt;/partinfo&gt; can degrade BPA and seems to work better than the polycistronic version (compare Figure 1)
</LI></UL><P>S-layer: 
</P><UL><LI> expression of the S-layer gene without TAT-sequence of <I>C. glutamicum</I> in <I>E. coli</I><A href="http://www.promega.com/products/cloning-and-dna-markers/cloning-tools-and-competent-cells/bacterial-strains-and-competent-cells/single-step-_krx_-competent-cells/" class="external text" rel="nofollow">KRX</A> and <A href="http://openwetware.org/wiki/E._coli_genotypes#BL21.28DE3.29" class="external text" rel="nofollow">BL21-Gold(DE3)</A> after sequencing gave correct results
</LI><LI> successful PCR on the S-layer gene of <A href="http://ijs.sgmjournals.org/cgi/content/abstract/54/3/779" class="external text" rel="nofollow"><I>Corynebacterium halotolerans</I></A></LI></UL><P>Organizational:
</P><UL><LI> moving to our own room in the <A href="http://www.cebitec.uni-bielefeld.de/" class="external text" rel="nofollow">CeBiTec</A></LI></UL><H2><SPAN class="mw-headline" id="Week_3:_16th_-_22nd_May">Week 3: 16th - 22nd May</SPAN></H2><P>Bisphenol A: 
</P><UL><LI> successful PCR on the <A href="http://www.brenda-enzymes.org/php/result_flat.php4?ecno=1.18.1.2" class="external text" rel="nofollow">NADP<SUP>+</SUP>  oxidoreductase</A> gene from <I>E. coli</I> TOP10
</LI></UL><P>S-layer: 
</P><UL><LI> successful cloning of the complete S-layer gene <I>cspB</I> of <I>C. glutamicum</I>, <I>C. crenatum</I> and <I>C. halotolerans</I></LI><LI> successful cloning of the S-layer genes of <I>B. flavum</I> and <I>C. halotolerans</I> without TAT-sequence, without lipid anchor and without both (only self-assembly domain)
</LI></UL><H2><SPAN class="mw-headline" id="Week_4:_23rd_-_29th_May">Week 4: 23rd - 29th May</SPAN></H2><P>Bisphenol A: 
</P><UL><LI> establishing a new method for analysis of BPA concentrations (extraction + LC-ESI-QTOF-MS)
</LI></UL><P>Organizational:
</P><UL><LI> arrange a BBQ for our workgroup in the CeBiTec to get to know our co-workers
</LI><LI> substantiating our contribution to the GENIALE
</LI></UL><H2><SPAN class="mw-headline" id="Week_5:_30th_May_-_5th_June">Week 5: 30th May - 5th June</SPAN></H2><P>Bisphenol A: 
</P><UL><LI> beginning of first characterization experiments for BPA degrading BioBricks (&lt;partinfo&gt;K123000&lt;/partinfo&gt; and &lt;partinfo&gt;K123001&lt;/partinfo&gt;)
</LI></UL><P>Organizational: 
</P><UL><LI> meeting with Prof. <A href="http://de.wikipedia.org/wiki/Alfred_Pühler" class="external text" rel="nofollow">Alfred Pühler</A> to plan our contribution to the <A href="http://www.cebitec.uni-bielefeld.de/content/view/209/88/" class="external text" rel="nofollow">CeBiTec symposium</A> in July
</LI></UL><H2><SPAN class="mw-headline" id="Week_6:_6th_-_12th_June">Week 6: 6th - 12th June</SPAN></H2><P>Organizational: 
</P><UL><LI> presentation of the iGEM competition at the <A href="http://www.bio.nrw.de/studentconvention" class="external text" rel="nofollow">2nd BIO.NRW (PhD) Student Convention</A></LI></UL><H2><SPAN class="mw-headline" id="Week_7:_13th_-_19th_June">Week 7: 13th - 19th June</SPAN></H2><P>S-layer:
</P><UL><LI> successful fusion of modified <I>cspB</I> genes of <A href="http://hamap.expasy.org/proteomes/CORGL.html" class="external text" rel="nofollow"><I>C. glutamicum</I></A> and <A href="http://ijs.sgmjournals.org/content/54/3/779.abstract" class="external text" rel="nofollow"><I>C. halotolerans</I></A> with a monomeric RFP (<A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">BBa_E1010</A>) using <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Gibson_assembly" class="external text" rel="nofollow">Gibson assembly</A>.
</LI><LI><I>C.glutamicum</I>:
</LI><LI> K525131: fusion of <I>cspB</I> including  lipid anchor and TAT-sequence &lt;partinfo&gt;K525121&lt;/partinfo&gt; with <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">BBa_E1010</A></LI><LI> K525133: fusion of <I>cspB</I> including  lipid anchor without TAT-sequence &lt;partinfo&gt;K525123&lt;/partinfo&gt; with <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">BBa_E1010</A></LI></UL><LI><I>C. halotolerans</I>:
</LI><LI> K525233: fusion of <I>cspB</I> including  lipid anchor without TAT-sequence &lt;partinfo&gt;K525223&lt;/partinfo&gt; with <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">BBa_E1010</A></LI><LI> K525224: fusion of <I>cspB</I> including TAT-sequence without lipid anchor &lt;partinfo&gt;K525224&lt;/partinfo&gt; with <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">BBa_E1010</A></LI><LI> First expression of K525133 in <A href="http://www.promega.com/products/cloning-and-dna-markers/cloning-tools-and-competent-cells/bacterial-strains-and-competent-cells/single-step-_krx_-competent-cells/" class="external text" rel="nofollow"><I>E. coli</I> KRX</A> to test different induction time points und L-rhamnose concentrations. A higher inducer concentration results in a decreasing maximum and final optical density (OD<SUB>600</SUB>) but in a higher fluorescence level. The variations of the induction time point has no significant influence on growth and the progression of fluorescence. 
</LI><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 2: Growth curve of <A href="http://www.promega.com/products/cloning-and-dna-markers/cloning-tools-and-competent-cells/bacterial-strains-and-competent-cells/single-step-_krx_-competent-cells/" class="external text" rel="nofollow"><I>E. coli</I> KRX</A> expressing K525133 using different inducer (L-rhamnose) concentrations and varying the time point of induction. Cultivations were carried out in LB medium. Inducing time points and inducer concentration are named with following indices: </B>(E1)<B> induced early (cultivation time = 2,75 h) L-rhamnose end concentration = 0,1 %; </B>(E2)<B> induced early (cultivation time = 2,75 h) L-rhamnose end concentration = 0,2 %; </B>(L1)<B> induced late (cultivation time = 4,25 h) L-rhamnose end concentration = 0,1 %; </B>(L2)<B> induced late (cultivation time = 4,25 h) L-rhamnose end concentration = 0,2 %</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 3: Fluorescence development of  <A href="http://www.promega.com/products/cloning-and-dna-markers/cloning-tools-and-competent-cells/bacterial-strains-and-competent-cells/single-step-_krx_-competent-cells/" class="external text" rel="nofollow"><I>E. coli</I> KRX</A> expressing K525133 during  cultivation using different inducer (L-rhamnose) concentrations and varying the time point of induction. Cultivations were carried out in LB medium. Inducing time points and inducer concentration are named with following indices: </B>(E1)<B> induced early (cultivation time = 2,75 h) L-rhamnose end concentration = 0,1 %; </B>(E2)<B> induced early (cultivation time = 2,75 h) L-rhamnose end concentration = 0,2 %; </B>(L1)<B> induced late (cultivation time = 4,25 h) L-rhamnose end concentration = 0,1 %; </B>(L2)<B> induced late (cultivation time = 4,25 h) L-rhamnose end concentration = 0,2 %</B></DIV></DIV></DIV><P>Bisphenol A: 
</P><UL><LI> cloning of NADP oxidoreductase in <A href="http://www.bioinfo.pte.hu/f2/pict_f2/pJETmap.pdf" class="external text" rel="nofollow">pJET1.2</A> finally successful -&gt; waiting for sequencing results to remove illegal restriction sites
</LI></UL><H2><SPAN class="mw-headline" id="Week_8:_20th_-_26th_June">Week 8: 20th - 26th June</SPAN></H2><P>S-layer:
</P><UL><LI> successful cloning of K525232 using <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Gibson_assembly" class="external text" rel="nofollow">Gibson assembly</A>.
</LI><LI> K525232: fusion of modified cspB of <A href="http://ijs.sgmjournals.org/content/54/3/779.abstract" class="external text" rel="nofollow"><I>C. halotolerans</I></A> without  lipid anchor and TAT-sequence &lt;partinfo&gt;K525222&lt;/partinfo&gt; with <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">BBa_E1010</A>.
</LI></UL><P>Organizational: 
</P><UL><LI> finishing our first press release
</LI><LI> all devices (thermocycler etc.) and materials (competent cells, polymerase, kits) from our sponsors arrived
</LI></UL><H2><SPAN class="mw-headline" id="Week_9:_27th_June_-_3rd_July">Week 9: 27th June - 3rd July</SPAN></H2><P>EXAMS !
</P><H2><SPAN class="mw-headline" id="Week_10:_4th_July_-_10th_July">Week 10: 4th July - 10th July</SPAN></H2><P>Bisphenol A:
</P><UL><LI> experiments on the influence of temperature, promoter strength and the characteristics of the fusion protein &lt;partinfo&gt;K123000&lt;/partinfo&gt; || &lt;partinfo&gt;K123001&lt;/partinfo&gt; on BPA degradation
</LI></UL><P>S-layer:
</P><UL><LI> Expression of K525131 (with TAT-sequence and lipid anchor), K5252133 (with lipid anchor), K525232 (nothing); K525233 (with lipid anchor) and K525234 (with TAT-sequence) in <I>E. coli</I> KRX to test the functional efficiency of <I>Corynebacterium</I> TAT-sequence in <I>E.coli</I> and the effect of the autoinduction protocol. After one day of cultivation, an increased fluorescence in all periplasm fractions of <I>E. coli</I> expressing S-layer constructs with TAT-sequence could be recognized. This is an indication that the <I>Corynebacterium</I> TAT-signal sequence is functional in <I>E. coli</I>. Comparing the manual and the autoinduction protocol, no significant changes in fluorescence (after 14 h) are identifiable. 
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 4: Relative fluorescence of the S-layer/mRFP fusion proteins in the cell suspension less the distinct fluorescence of <I>E. coli</I> KRX. All cultivations were carried out in LB medium at 37 ˚C except K525131<SUB>Glc</SUB> which was in autoinduction medium. The expression of the S-layers in LB medium were induced adding 0,1 % L-rhamnose at OD<SUB>600</SUB> = 0,6.</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 5: Relative fluorescence of the S-layer/mRFP fusion proteins in the periplasm less the distinct fluorescence of <I>E. coli</I> KRX periplasm. All cultivations were carried out in LB medium at 37 ˚C except K525131<SUB>Glc</SUB> which was in autoinduction medium. The expression of the S-layers in LB medium were induced adding 0,1 % L-rhamnose at OD<SUB>600</SUB> = 0,6.</B></DIV></DIV></DIV><P>Organizational:
</P><UL><LI> presenting our posters from the teams of 2010 and 2011 at the congress <A href="http://www.biotechnologie2020plus.de/BIO2020/Navigation/DE/root,did=121262.html" class="external text" rel="nofollow">Biotechnologie2020+</A> in Berlin hosted by the &quot;Bundesministerium für Bildung und Forschung&quot; (Federal Ministry of Education and Research).
</LI></UL><H2><SPAN class="mw-headline" id="Week_11:_11th_July_-_17th_July">Week 11: 11th July - 17th July</SPAN></H2><P>Bisphenol A / S-layer:
</P><UL><LI> our BioBrick order (some fluorescent proteins and cleavage sites) from iGEM HQ arrived
</LI></UL><P>S-layer:
</P><UL><LI> our synthesized S-layers SgsE and SbpA finally arrived
</LI></UL><P>Organizational:
</P><UL><LI> presenting the iGEM competition and our team project at the secondary school <A href="http://www.rg-herford.de/" class="external text" rel="nofollow">Ravensberger Gynmasium</A> in Herford (<A href="http://www.flickr.com/photos/igem-bielefeld/sets/72157627214780020/" class="external text" rel="nofollow">Photos</A>)
</LI></UL><H2><SPAN class="mw-headline" id="Week_12:_18th_July_-_24th_July">Week 12: 18th July - 24th July</SPAN></H2><P>Organizational:
</P><UL><LI> presenting our projects from 2010 and 2011 at the <A href="http://www.cebitec.uni-bielefeld.de/content/view/209/88/" class="external text" rel="nofollow">CeBiTec Symposium 2011</A> in Bielefeld
</LI><LI> meeting with the iGEM teams from Delft/NL, Edinburgh/UK, Odense/DK. Freiburg/DE and Ljubljana/SL at the Symposium (<A href="http://www.flickr.com/photos/igem-bielefeld/sets/72157627300144576/" class="external text" rel="nofollow">Photos</A>) was fun!
</LI></UL><P>Bisphenol A / S-Layer:
</P><UL><LI> removing illegal restriction sites to get valid BioBricks
</LI></UL><P>NAD<SUP>+</SUP> detection:
</P><UL><LI> cloning the NAD<SUP>+</SUP>-dependent DNA ligase from <I>E. coli</I> into BioBrick backbones
</LI></UL><H2><SPAN class="mw-headline" id="Week_13:_25th_July_-_31th_July">Week 13: 25th July - 31th July</SPAN></H2><P>S-layer:
</P><UL><LI> fusing the synthesized S-Layers to a bunch of fluorescent proteins
</LI><LI> successful cloning of K525231 using <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#Gibson_assembly" class="external text" rel="nofollow">Gibson assembly</A>.
</LI><LI> K525231: fusion of modified <I>cspB</I> of <A href="http://ijs.sgmjournals.org/content/54/3/779.abstract" class="external text" rel="nofollow"><I>C. halotolerans</I></A> including lipid anchor and TAT-sequence &lt;partinfo&gt;K525221&lt;/partinfo&gt; with <A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow">BBa_E1010</A>.
</LI></UL><LI> testing the basal transcription of <I>E. coli</I> KRX by adding inducer (L-rhamnose) or not. The increasing RFU per OD<SUB>600</SUB> reveals the basal transcription of <I>E. coli</I> KRX. L-rhamnose in the medium leads to a ten times higher RFU per OD<SUB>600</SUB>.
</LI><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 6: Growth curves of <I>E. coli</I> KRX expressing fluorescence tagged S-layer constructs of <A href="http://expasy.org/sprot/hamap/CORGL.html" class="external text" rel="nofollow"><I>C. glutamicum</I></A>. Cultivations were carried out in autoinduction  medium at 37 ˚C with and without adding inducer (L-rhamnose). The induced cultivations are marked with index I and the uninduced are marked with index U.</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 7: Growth curves of <I>E. coli</I> KRX expressing fluorescence tagged S-layer constructs of <A href="http://ijs.sgmjournals.org/content/54/3/779.abstract" class="external text" rel="nofollow"><I>C. halotolerans</I></A>. Cultivations were carried out in autoinduction  medium at 37 ˚C with and without adding inducer (L-rhamnose). The induced cultivations are marked with index I and the uninduced are marked with index U.</B></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 8: Development of relative fluorescence per OD<SUB>600</SUB> during cultivation of <I>E. coli</I> KRX expressing fluorescence tagged S-layer constructs of <A href="http://expasy.org/sprot/hamap/CORGL.html" class="external text" rel="nofollow"><I>C.glutamicum</I></A> and <A href="http://ijs.sgmjournals.org/content/54/3/779.abstract" class="external text" rel="nofollow"><I>C. halotolerans</I></A>. Cultivations were carried out in autoinduction medium at 37 ˚C.</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 9: Development of relative fluorescence per OD<SUB>600</SUB> during uninduced cultivation of <I>E. coli</I> KRX containing fluorescence tagged S-layer gens constructs of <A href="http://expasy.org/sprot/hamap/CORGL.html" class="external text" rel="nofollow"><I>C.glutamicum</I></A> and <A href="http://ijs.sgmjournals.org/content/54/3/779.abstract" class="external text" rel="nofollow"><I>C. halotolerans</I></A>. Cultivations were carried out in autoinduction medium at 37 ˚C without adding inducer.</B></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 10: Development of relative fluorescence per OD<SUB>600</SUB> in periplasm of <I>E. coli</I> KRX expressing fluorescence tagged S-layer constructs of <A href="http://expasy.org/sprot/hamap/CORGL.html" class="external text" rel="nofollow"><I>C. glutamicum</I></A> and <A href="http://ijs.sgmjournals.org/content/54/3/779.abstract" class="external text" rel="nofollow"><I>C. halotolerans</I></A> during cultivation. Cultivations were carried out in autoinduction medium at 37 ˚C.</B></DIV></DIV></DIV></DIV><P>Bisphenol A:
</P><UL><LI> testing new BPA extraction protocols for LC-MS including an internal standard (bisphenol F)
</LI></UL><H2><SPAN class="mw-headline" id="Week_14:_1st_August_-_7th_August">Week 14: 1st August - 7th August</SPAN></H2><P>Bisphenol A: 
</P><UL><LI> BPA analysis with <A href="/Team:Bielefeld-Germany/Protocols#Extraction_with_ethylacetate" title="Team:Bielefeld-Germany/Protocols"> extraction</A> and LC-MS finally works and is very accurate
</LI><LI> Better results for BPA degradation in <I>E. coli</I> -&gt; our fusion protein (&lt;partinfo&gt;K123000&lt;/partinfo&gt; to &lt;partinfo&gt;K123001&lt;/partinfo&gt;) can completely degrade BPA
</LI><LI> Measuring characterization results for different BPA degrading BioBricks
</LI></UL><P>NAD<SUP>+</SUP> detection:
</P><UL><LI> Successful characterization of two differently labeled (6-FAM or TAMRA with Dabcyl) molecular beacons as a preparation for the <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols#NAD.2B_bioassay" class="external text" rel="nofollow">NAD<SUP>+</SUP> bioassay</A> including autonomously produced NAD<SUP>+</SUP>-dependent DNA ligase (&lt;partinfo&gt;K525710&lt;/partinfo&gt;) from <I>E. coli</I> (results shown below)
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 11: Emission spectra of 6-FAM labeled molecular beacon in its closed and open state (target added) at an extinction wavelength 495 nm (n=3).</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 12: Emission spectra of 6-FAM labeled molecular beacon in its closed state (split target added) at an extinction wavelength 495 nm (n=3).</B></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 13: Extinction spectra of 6-FAM labeled molecular beacon in its closed and open state (target added) at an emission wavelength 530 nm (n=3).</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 14: Extinction spectra of 6-FAM labeled molecular beacon in its closed state (split target added) at an emission wavelength 530 nm (n=3).</B></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 15: Emission spectra of TAMRA labeled molecular beacon in its closed or open state (target added) at an extinction wavelength 544 nm (n=3).</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 16: Emission spectra of TAMRA labeled molecular beacon in its closed state (split target added) at an extinction wavelength 544 nm (n=3).</B></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 17: Extinction spectra of TAMRA labeled molecular beacon in its closed or open state (target added) at an emission wavelength 590 nm (n=3).</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 18: Extinction spectra of TAMRA labeled molecular beacon in its closed state (split target added) at an emission wavelength 590 nm (n=3).</B></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 19: Signal-to-background ratio (S/B) determination of 6-FAM labeled molecular beacon in its closed or open state at an extinction wavelength 495 nm and emission wavelength 530 nm. Molecular beacons and the target were added one after another (see gaps) each after equilibrium was reached. Calculated S/B: 45.52 (n=3).</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 20: Signal-to-background ratio (S/B) determination of 6-FAM labeled molecular beacon in its closed state at an extinction wavelength 495 nm and emission wavelength 530 nm. Molecular beacons and the split target were added one after another (see gaps) each after equilibrium was reached. Calculated S/B: 3.36 (n=3).</B></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 21: Signal-to-background ratio (S/B) determination of TAMRA labeled molecular beacon in its closed or open state at an extinction wavelength 552 nm and emission wavelength 590 nm. Molecular beacons and the target were added one after another (see gaps) each after equilibrium was reached. Calculated S/B: 18.21 (n=3).</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 22: Signal-to-background ratio (S/B) determination of TAMRA labeled molecular beacon in its closed state at an extinction wavelength 552 nm and emission wavelength 590 nm. Molecular beacons and the split target were added one after another (see gaps) each after equilibrium was reached. Calculated S/B: 2.31 (n=3).</B></DIV></DIV></DIV><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 23: Thermal profile of 6-FAM labeled molecular beacon alone and with either target or split target added (n=5).</B></DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:277px;"><DIV class="thumbcaption"><B>Figure 24: Imaging 6-FAM labeled molecular beacon alone and with either target or split target added.</B></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Week_15:_8th_August_-_14th_August">Week 15: 8th August - 14th August</SPAN></H2><P>Bisphenol A: 
</P><UL><LI> BPA analysis with <A href="/Team:Bielefeld-Germany/Protocols/Analytics#Extraction_with_ethylacetate" title="Team:Bielefeld-Germany/Protocols/Analytics"> extraction</A> and HPLC with UV detector leads to very similar results as the analysis with LC-MS (except for low BPA concentrations -&gt; LOD / LOQ of LC-MS is lower than that of &quot;normal&quot; HPLC, compare Figure 25)
</LI><LI> measuring of more samples from cultivations with BPA degrading BioBricks for further characterization
</LI><LI> we discovered some interesting results in our MS data - soon more
</LI><LI> testing methods to purify his-tagged BisdA and BisdB for cell free BPA degradation and further characterization of these proteins
</LI><LI> testing the influence of BPA on the growth of <I>E. coli</I></LI><LI> developing a model for BPA degradation by <I>E. coli</I> (compare Figure 26)
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 25: HPLC results and optical density of experiments on BPA degradation of <I>E. coli</I> KRX. Cultivations were carried out in LB medium with ~ 100 mg L<SUP>-1</SUP> BPA at 30 °C. Samples were taken every three hours over one day. BPA concentration was measured by HPLC with either UV detector or ESI-qTOF-MS. 502: negative control (&lt;partinfo&gt;K123000&lt;/partinfo&gt;), 512: polycistronic &lt;partinfo&gt;K123001&lt;/partinfo&gt; and &lt;partinfo&gt;K123000&lt;/partinfo&gt;, 517: fusionprotein between &lt;partinfo&gt;K123001&lt;/partinfo&gt; and &lt;partinfo&gt;K123000&lt;/partinfo&gt;, every part behind medium strong constitutive promoter. </B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 26: Modelling of BPA degradation (filled squares) by and OD<SUB>600</SUB> (open squares) of <I>E. coli</I> KRX carrying genes for BisdA and BisdB (polycistronic <I>bisdAB</I> (black) and fusion protein between BisdA and BisdB (red)) behind the medium strong promoter &lt;partinfo&gt;J23110&lt;/partinfo&gt;. Cultivations were carried out at 30 °C in LB + Amp + BPA medium for 24 h with automatic sampling every three hours in 300 mL shaking flasks without baffles with silicon plugs. Three biological replicates were analysed. </B></DIV></DIV></DIV><P>
S-layer: 
</P><UL><LI> MALDI-TOF analysis of SDS-PAGEs to characterize the function of the TAT-sequence and the lipid anchor of PS2 (encoded by <I>cspB</I> gene) in <I>E. coli</I> KRX.
</LI></UL><H2><SPAN class="mw-headline" id="Week_16:_15th_August_-_21st_August">Week 16: 15th August - 21st August</SPAN></H2><P>S-layer: 
</P><UL><LI> finally found our S-layer proteins in <I>E. coli</I> by analyzing the range of sizes in the polyacrylamid with a MALDI-TOF method-&gt; now we can plan purification strategy
</LI><LI> fusion of FPs to <I>sgsE</I> successful
</LI></UL><P>Bisphenol A: 
</P><UL><LI> all metabolites of natural BPA degradation pathway found during degradation of BPA in <I>E. coli</I> by LC-MS -&gt; now MS/MS to check structure of these metabolites to be sure
</LI><LI> some &quot;metabolites&quot; could also be found due to fragmentation during ionization of the product of BPA degradation
</LI></UL><LI> testing whether <I>E. coli</I> can grow on BPA as the only carbon source (on M9 plates)
</LI><LI> seems like they can't
</LI><H2><SPAN class="mw-headline" id="Week_17:_22nd_August_-_28th_August">Week 17: 22nd August - 28th August</SPAN></H2><P>S-layer: 
</P><UL><LI> cultivation and test purification of <I>SgsE</I> fusion proteins (&lt;partinfo&gt;K525304&lt;/partinfo&gt;, &lt;partinfo&gt;K525305&lt;/partinfo&gt; and &lt;partinfo&gt;K525306&lt;/partinfo&gt;)
</LI><LI> SDS-PAGEs showed inclusion bodies formation in <I>E coli</I> KRX expressing <I>sgsE</I>.
</LI></UL><P>Bisphenol A: 
</P><UL><LI> purification of his-tagged BisdA and BisdB
</LI><LI> first characterization results entered into partsregistry
</LI><LI> cloning of fusion protein FNR:BisdA:BisdB successful
</LI></UL><H2><SPAN class="mw-headline" id="Week_18:_29th_August_-_4th_September">Week 18: 29th August - 4th September</SPAN></H2><P>S-layer:
</P><UL><LI> developing IEX clean-up for S-layers from <I>Corynebacterium halotolerans</I></LI><LI> Searching new purification methods for <I>SgsE</I> fusion proteins.
</LI></UL><P>Bisphenol A:
</P><UL><LI> successful cloning of polycistronic FNR + BisdA + BisdB
</LI></UL><P>NAD<SUP>+</SUP> detection:
</P><UL><LI>Successful overexpression of NAD<SUP>+</SUP>-dependent DNA ligase (&lt;partinfo&gt;K525710&lt;/partinfo&gt;) in <I>E. coli</I> KRX and purification with Ni-NTA columns utilizing the protein`s C-terminal His-tag. Trying out utility for the NAD<SUP>+</SUP> bioassay now.
</LI></UL><P>Organizational:
</P><UL><LI> participating at the science festival <A href="http://www.geniale-bielefeld.de/" class="external text" rel="nofollow">GENIALE</A> in Bielefeld with a <A href="http://www.geniale-bielefeld.de/programm/detail/date/456/show/Event/" class="external text" rel="nofollow">lab tour</A> and a <A href="http://www.geniale-bielefeld.de/programm/detail/date/295/show/Event/" class="external text" rel="nofollow">presentation with public discussion afterwards (science café)</A>. Our participation is also mentioned in the regional newspaper <A href="http://www.westfalen-blatt.de/nachricht/2011-08-30-wie-ein-hamster-im-laufrad/?cHash=b8421d55eb00ebf8532ebfa677a7e18f" class="external text" rel="nofollow">Westfalen Blatt</A> and in the internet on <A href="http://www.direkt-bielefeld.de/Wissenschaft-im-Doppelpack-108856.html" class="external text" rel="nofollow">Bielefeld Direkt</A>.
</LI></UL><H2><SPAN class="mw-headline" id="Week_19:_5th_September_-_11th_September">Week 19: 5th September - 11th September</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Fig. 27: Immobilization behaviour of fluorescent S-layer fusion protein SgsE :: mCitrine as a function of silica bead concentration. Data fitted with Michaelis-Menten function. </B></DIV></DIV></DIV><P>S-layer:
</P><UL><LI> coating silica beads with fluorescent S-layer fusion protein SgsE | mCitrine, remove these proteins again and find them in SDS-PAGE
</LI><LI> characterizing immobilization behaviour of S-layers on silica beads -&gt; seems to work (Figure 27)
</LI><LI> cultivate bigger amounts of and develop new purification strategy for S-layer fusion proteins
</LI><LI> testing purification of S-layer fusion proteins with his-tag
</LI><LI> cultivations of fusion protein between NADP<SUP>+</SUP>-ferredoxin oxidoreductase and SgsE / SbpA
</LI><LI> also SbpA | mCitrine fusion protein
</LI></UL><P>NAD<SUP>+</SUP> detection:
</P><UL><LI> Suitability of autonomously produced DNA ligase (&lt;partinfo&gt;K525710&lt;/partinfo&gt;) from <I>E. coli</I> for the NAD<SUP>+</SUP> bioassay could be demonstrated after treatment with NMN for deadenylation. Checking out the limit of detection right now.
</LI></UL><H2><SPAN class="mw-headline" id="Week_20:_12th_September_-_18th_September">Week 20: 12th September - 18th September</SPAN></H2><P>S-layer:
</P><UL><LI> testing a purification and coating protocol for S-layer fusion protein mCitrine|SbpA.
</LI><LI> characterizing immobilization behaviour of mCitrine|SbpA <A href="http://partsregistry.org/Part:BBa_K525405" class="external text" rel="nofollow">(K525405)</A> on silica beads
</LI><LI> optimizing <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing#Production_and_purification_strategies" class="external text" rel="nofollow">purification</A> and <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing#Recrystallization_of_S-layer_proteins" class="external text" rel="nofollow">coating protocol</A> for <I>SgsE</I> fusion proteins.
</LI><LI> last characterization cultivations of <A href="http://partsregistry.org/Part:BBa_K525304" class="external text" rel="nofollow">K525304</A>, <A href="http://partsregistry.org/Part:BBa_K525305" class="external text" rel="nofollow">K525305</A>, <A href="http://partsregistry.org/Part:BBa_K525306" class="external text" rel="nofollow">K525306</A>, <A href="http://partsregistry.org/Part:BBa_K525405" class="external text" rel="nofollow">K525405</A> and <A href="http://partsregistry.org/Part:BBa_K525406" class="external text" rel="nofollow">K525406</A>.
</LI></UL><P>Bisphenol A: 
</P><UL><LI> successful cloning of fusion protein FNR + BisdA | BisdB and polycistronic FNR + BisdA + BisdB behind constitutive promoter -&gt; characterizing these last constructs
</LI></UL><P>NAD<SUP>+</SUP> detection:
</P><UL><LI> NAD<SUP>+</SUP> assay works :)
</LI></UL><H2><SPAN class="mw-headline" id="Week_21:_19th_September_-_25th_September">Week 21: 19th September - 25th September</SPAN></H2><P>Organizational:
</P><UL><LI> lab work is finished, the last measurements and cloning experiments into pSB1C3 are done
</LI><LI> now it is time to fill the Wiki and Partsregistry with brilliant data until European Wikifreeze @ September 21st 11:59 pm EDT!
</LI></UL><H2><SPAN class="mw-headline" id="Week_22:_26th_September_-_2nd_October">Week 22: 26th September - 2nd October</SPAN></H2><P>Preparation of the presentation and the poster for the European Regional Jamboree.
</P><UL><LI> practicing the presentation with the Coryne and fermentation engineering working group as audience.
</LI></UL><P><B>We are qualified for the final in Boston!!! Now labwork can continue.</B></P><H2><SPAN class="mw-headline" id="Week_23:_3rd_October_-_9th_October">Week 23: 3rd October - 9th October</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 28: LigA shows high selectivity for NAD<SUP>+</SUP>.</B> The final concentration of all analytes was 100 nM. The responses were evaluated on the basis of the average fluorescence enhancement rate in a range of 200 s after addition of each analyte into the NAD<SUP>+</SUP> bioassay. The dotted line marks the threshold indicating the intensity of background signal. All data are normalized to the NAD<SUP>+</SUP> value.</DIV></DIV></DIV><P>S-layer:
</P><UL><LI>scale-up cultivation of <I>E. coli</I> expressing <A href="http://partsregistry.org/Part:BBa_K525305" class="external text" rel="nofollow">mCitrine|SgsE (K525305)</A> and <A href="http://partsregistry.org/Part:BBa_K525405" class="external text" rel="nofollow">mCitrine|SbpA (K525405)</A> fusion protein.
</LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing#Fusion_proteins_of_SgsE_and_SbpA" class="external text" rel="nofollow">inclusion body purification</A> and testing further purification strategies.
</LI><LI>assembly of <A href="http://partsregistry.org/Part:BBa_K525422" class="external text" rel="nofollow">K525422</A> containing mCitrine|SbpA fused with a His-tag (<A href="http://partsregistry.org/Part:BBa_K157011" class="external text" rel="nofollow">K157011</A>) for easer purification.
</LI><LI>assembly of <A href="http://partsregistry.org/Part:BBa_K525311" class="external text" rel="nofollow">K525311</A> consisting of the SbpA S-layer fused with firefly-luciferase (<A href="http://partsregistry.org/Part:BBa_K525999" class="external text" rel="nofollow">K525999</A>) for testing an enzyme reaction on immobilized S-layer proteins.
</LI><LI>ordering silicon wafer as a new surface for immobilization.
</LI><LI>successful cloning of fusion protein of K525303 and K525560 <A href="http://partsregistry.org/Part:BBa_K525001" class="external text" rel="nofollow">(K525001)</A>,<A href="http://partsregistry.org/Part:BBa_K525311" class="external text" rel="nofollow">(K525311)</A>, <A href="http://partsregistry.org/Part:BBa_K525006" class="external text" rel="nofollow">SbpA|LigA (K525006)</A>, <A href="http://partsregistry.org/Part:BBa_K525422" class="external text" rel="nofollow">(K525422)</A>,<A href="http://partsregistry.org/Part:BBa_K525322" class="external text" rel="nofollow">(K525322)</A></LI></UL><P>NAD<SUP>+</SUP> detection:
</P><UL><LI> testing selectivity of LigA (&lt;partinfo&gt;BBa_K525710&lt;/partinfo&gt;) for its substrate NAD<SUP>+</SUP> to verify further applications of the molecular beacon based NAD<SUP>+</SUP> bioassay dealing with NAD<SUP>+</SUP> detection in analyte mixtures such as cell lysates.
</LI></UL><H2><SPAN class="mw-headline" id="Week_24:_10th_October_-_16th_October">Week 24: 10th October - 16th October</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 29: Fluorescence enhancement rate after addition of LDH reaction mix with various pyruvate concentrations into a LigA-dependent NAD<SUP>+</SUP> bioassay.</B></DIV></DIV></DIV><P>S-layer:
</P><UL><LI>first expression of K525422 in <I>E. coli</I> KRX and His-tag affinity chromatography based purification using denaturing conditions.
</LI><LI>Test purifications were carried out using binding buffer with and without imidazole.
</LI></UL><LI>new inclusion body purification of K525305 and K525405 to get new biological material to test the further purification methods.
</LI><LI>establishment of ion exchange chromatography (IEX) and hydrophobic interaction chromatography (HIC) as futher purification steps for S-layer proteins from <I>Lysinibacillus sphaericus</I> and <I>Geobacillus stearothermophilus</I>.
</LI><LI>diethylaminoethyl cellulose (DEAE) as IEX and butyl sepharose™ 4 Fast Flow as HIC purification media.
</LI><LI>test and scale-up expression of <A href="http://partsregistry.org/Part:BBa_K525322" class="external text" rel="nofollow">mCitrine|SbpA|His-tag (K525322)</A> in <I>E. coli</I> KRX.
</LI><LI>after enzymatic cell lysis with lysozyme and inclusion body purification highest luminescence was measured in the lysis supernatant. This implicates that the fusion protein does not form inclusion bodies.
</LI><LI>successful cloning of fusion protein of CspB and luciferase with <A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Genetics" class="external text" rel="nofollow">Gibson assembly</A></LI><P>NAD<SUP>+</SUP> detection:
</P><UL><LI>the NAD<SUP>+</SUP> bioassay was successfully coupled to the NADH-dependent reaction of lactic acid dehydrogenase and could therefore be used to quantify pyruvate
</LI></UL><P>BPA-degradation:
</P><UL><LI>successful cloning of <A href="http://partsregistry.org/Part:BBa_K525562" class="external text" rel="nofollow">(K525562)</A> ( Fusion protein of <A href="http://partsregistry.org/Part:BBa_K525499" class="external text" rel="nofollow">(K525499)</A>, <A href="http://partsregistry.org/Part:BBa_K123000" class="external text" rel="nofollow">(K123000)</A>and <A href="http://partsregistry.org/Part:BBa_K123001" class="external text" rel="nofollow">(K5123001)</A> with middle strong promoter) and <A href="http://partsregistry.org/Part:BBa_K525534" class="external text" rel="nofollow">(K525534)</A></LI></UL><H2><SPAN class="mw-headline" id="Week_25:_17th_October_-_23th_October">Week 25: 17th October - 23th October</SPAN></H2><P>S-layer:
</P><UL><LI>new expression of &lt;partinfo&gt;K525311&lt;/partinfo&gt; to develop an individual purification strategy without inclusion body purification.
</LI><LI>testing a new purification strategy for the fusion protein of the SgsE and the firefly-luciferase based on the IEX and HIC
</LI><LI>first expression of <A href="http://partsregistry.org/Part:BBa_K525322" class="external text" rel="nofollow">mCitrine|SbpA|His-tag (K525322)</A> and successful His-tag affinity chromatography based purification using denaturing conditions.
</LI></UL><P>NAD<SUP>+</SUP> detection:
</P><UL><LI>testing LigA for molecular cloning
</LI></UL><H2><SPAN class="mw-headline" id="Week_26:_24th_October_-_30th_October">Week 26: 24th October - 30th October</SPAN></H2><P>S-layer:
</P><UL><LI>test expression of <A href="http://partsregistry.org/Part:BBa_K525534" class="external text" rel="nofollow">reductase|His-tag (K525534)</A> in <I>E. coli</I> KRX. and testing non denaturating His-tag affinity chromatography.
</LI><LI>recrystallization of <A href="http://partsregistry.org/Part:BBa_K525305" class="external text" rel="nofollow">mCitrine|SgsE (K525305)</A> and <A href="http://partsregistry.org/Part:BBa_K525405" class="external text" rel="nofollow">mCitrine|SbpA (K525405)</A> on silicon wafer using two different immobilization strategies.
</LI><LI>trying to get pictures of the S-layer nanostructures on coated silicon wafers using reflection electron microscope (REM) and atomic force microscopy (AFM)
</LI><LI>not enough time left to find a successful method
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Bielefeld-Germany/Labjournal">http://2011.igem.org/Team:Bielefeld-Germany/Labjournal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Labjournal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Labjournal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal&amp;oldid=257642" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>