<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu-igem"><UL id="menu" class="sf-menu"><LI><A style="border-left:0px;" href="https://2011.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Students">Students</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Advisors">Supervisors</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Photos">Photos</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Bielefeld-University‎‎">Bielefeld University</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></UL><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Description">Description</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Achievements">Achievements</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Appoach">Approach</A></LI><LI><A href="#">Background</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Project/Background/Technology">Technology</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer/Guide">DIY Nanobiotech Guide</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Future">Future Applications</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Labjournal">Labjournal</A></LI><LI><A href="#">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Genetics">Molecular Genetics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Downstream-processing">Production</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Analytics">Analytics</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Protocols/Materials">Materials</A></LI><LI><A href="#">Results</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Data_Page">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/BPA">Bisphenol A</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/S-Layer">S-layer</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/NAD">NAD<SUP>+</SUP> detection</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/SinceAmsterdam">Since Regionals</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Modell">Model</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Public_relations_Summary">Summary</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Cell-free">Cell-free Biosafety</A></LI><LI><A href="#">Outreach</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_lab_course">Open Lab Course</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Geniale_science_cafe">Science Cafe</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/high-school">High School</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/cebitec-symposium">CeBiTec Symposium</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/student-convention">Student Convention</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/strategic-process">Strategic Process</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Black-box">Beacon Box</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Human_Practices/Press">Press</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Bielefeld-Germany/Partner">Partner</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Project_description"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Project description</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#S-layer"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">S-layer</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Bisphenol_A_degradation"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Bisphenol A degradation</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#NAD.2B_detection"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">NAD<SUP>+</SUP> detection</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Project_description">Project description</SPAN></H2><P>The development of sensitive and selective <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Biosensor" target="_blank" title="A biosensor is an analytical device for the detection of an analyte that combines a biological component with a physicochemical detector component.">biosensors</A></SPAN> is an important topic in <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Synthetic_biology" target="_blank" title="Synthetic biology is the newest development in modern biology. It connects principles of engineering and informatics with genetic engineering, molecular biology and organic chemistry to build biological systems with novel functions. ">synthetic biology</A></SPAN>. Biosensors can be applied in a wide range - from the detection of environmental toxics up to clinical diagnostics. Because cells have to sense their surroundings, there are a lot of natural systems that are similar to a biosensor. Prejudicial cellular biosensors often show negative side effects that complicate any practical application. Common problems are the <SPAN id="igem-tooltip"><A href="http://europa.eu/legislation_summaries/agriculture/food/l28130_en.htm" target="_blank" title="In Europe, it is not allowed to release genetically modified organisms (GMOs) into the environment or place them on the market. You are only allowed to work with GMOs inside of laboratories that fit special requirements to avoid their release (EU directive 2001/18/EC and 90/220/EEC).">limited use</A></SPAN> outside a gene laboratory due to the use of genetically engineered cells, the low durability because of the usage of living cells and the appearance of undesired signals induced by endogenous metabolic pathways.
</P><P>
To solve these problems, the <A href="/Team:Bielefeld-Germany/Team" title="Team:Bielefeld-Germany/Team"> iGEM-Team Bielefeld 2011</A> aims at developing a cell-free <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Bisphenol_a" target="_blank" title="Bisphenol A (abbreviated BPA) is a potentially harmful substance used in the production of polycarbonates and epoxy resins. These plastics are used in many food containing bins so we are exposed to BPA every day. ">bisphenol A</A></SPAN> (BPA) biosensor based on a coupled enzyme reaction fused to S-layer proteins for everyday use. Bisphenol A is a supposedly harmful substance which is used in the production of polycarbonate. To detect BPA it is degraded by a <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Fusion_protein" target="_blank" title="Fusion proteins are proteins created through the joining of two or more genes which originally coded for separate proteins. Translation of this fusion gene results in a single polypeptide with functional properties derived from each of the original proteins. Recombinant fusion proteins are created artificially by recombinant DNA technology. ">fusion protein</A></SPAN> under formation of <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/NAD%2B" target="_blank" title="Nicotinamide adenine dinucleotide, abbreviated NAD&lt;sup&gt;+&lt;/sup&gt;, is a coenzyme found in all living cells. In metabolism, NAD&lt;sup&gt;+&lt;/sup&gt; is involved in redox reactions, carrying electrons from one reaction to another. The enzymes that synthesize and use NAD&lt;sup&gt;+&lt;/sup&gt; and NADH are important in both current pharmacology and the research into future treatments for disease.">NAD<SUP>+</SUP></A></SPAN> which is detected by an NAD<SUP>+</SUP> dependent enzymatic reaction with a <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Molecular_beacon" target="_blank" title="Molecular beacons are oligonucleotide hybridization probes that can report the presence of specific nucleic acids in homogenous solutions. The terms more often used is molecular beacon probes. Molecular beacons are hairpin shaped molecules with an internally quenched fluorophore whose fluorescence is restored when they bind to a target nucleic acid sequence.">molecular beacon</A></SPAN>. Both enzymes are fused to S-layer proteins which build up well-defined nanosurfaces and are attached to the surface of beads. By providing these nanobiotechnological building blocks the system is expandable to other applications.
</P><P>
An overview of our project is shown in the figure below. The background and the current status of each subproject is described <A href="/Team:Bielefeld-Germany/Project#S-layer" title="Team:Bielefeld-Germany/Project"> further below</A>. To have a quick insight of what is happening in our lab take a look at our <A href="/Team:Bielefeld-Germany/Labjournal" title="Team:Bielefeld-Germany/Labjournal"> Labjournal</A>.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:932px;"><DIV class="thumbcaption"><B>Overview of the projectidea of iGEM team Bielefeld 2011.</B></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="S-layer">S-layer</SPAN></H2><P>S-layers (crystalline bacterial surface layer) are <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Crystal_structure" target="_blank" title="The crystal structure of a material can be described in terms of its unit cell. The unit cell is a small box containing one or more atoms, a spatial arrangement of atoms. The unit cells stacked in three-dimensional space describe the bulk arrangement of atoms of the crystal. The unit cell is given by its lattice parameters which are the length of the cell edges and the angles between them.">crystal-like</A></SPAN> layers consisting of multiple protein <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Monomer" target="_blank" title="A monomer is an atom or a small molecule that may bind chemically to other monomers to form a polymer. An example is the monomer bisphenol A that forms the plastic polycarbonate, a polymer. The term monomeric protein may also be used to describe one of the proteins making up a multiprotein complex.">monomers</A></SPAN> and can be found in various (archae-)bacteria. They constitute the outermost part of the cell wall. Especially their ability for <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Self-assembly" target="_blank" title="Self-assembly is a term used to describe processes in which a disordered system of pre-existing components forms an organized structure or pattern as a consequence of specific, local interactions among the components themselves, without external direction.">self-assembly</A></SPAN> into distinct geometries is of scientific interest. At phase boundaries, in solutions and on a variety of surfaces they form different lattice structures. The geometry and arrangement is determined by the C-terminal self assembly-domain, which is specific for each S-layer protein. The most common lattice geometries are oblique, square and hexagonal. By modifying the characteristics of the S-layer through combination with functional groups and <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Protein_domain" target="_blank" title="A protein domain is a part of a protein sequence and structure that can evolve, function, and exist independently of the rest of the protein chain. Each domain forms a compact three-dimensional structure and often can be independently stable and folded. Many proteins in nature consist of several structural domains. ">protein domains</A></SPAN> as well as their defined position and orientation to each other (determined by the S-layer geometry) it is possible to realize various practical applications (<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6968.2006.00573.x/full" class="external text" rel="nofollow">Sleytr <I>et al.</I>, 2007</A>). The usability and industrial potential of such well-defined nano-lattice structures is far-reaching from ultrafiltration membranes to the development of immobilized biosensors.
</P><P>
Especially for the production of cell-free biosensors, functional fusion proteins are of great importance. <A href="http://pubs.acs.org/doi/abs/10.1021/bm901071b" class="external text" rel="nofollow">Kainz <I>et al.</I> (2010)</A> fused fluorescent proteins with an S-layer glycoprotein from <A href="http://en.wikipedia.org/wiki/Geobacillus_stearothermophilus" class="external text" rel="nofollow"><I>Geobacillus stearothermophilus</I></A>. They demonstrated that the properties of the fusion protein were similar to the native fluorescent protein. The intensity of the fluorescence, the lifetime and the adsorption spectra showed comparable behavior at different pH-values. Enzymes fused to immobilized S-layers showed a significantly longer durability (<A href="http://onlinelibrary.wiley.com/doi/10.1002/smll.200700200/pdf" class="external text" rel="nofollow">Schäffer <I>et al.</I>, 2007</A>).
</P><P>
The iGEM-Team Bielefeld aims at the assembly, production and immobilization of S-layer fusion proteins for the detection of BPA by a coupled enzymatic reaction. S-layers from four different organisms are employed. The provision of various S-layers with different geometries offers the possibility for the scientific community to create functional <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Nanobiotechnology#Nanobiotechnology" target="_blank" title="Bionanotechnology and nanobiotechnology are terms that refer to the intersection of nanotechnology and biology. Many new medical technologies involving nanoparticles as delivery systems or as sensors would be examples of nanobiotechnology since they involve using nanotechnology to advance the goals of biology. ">nanobiotechnological</A></SPAN> surfaces with simple and standardized methods, so to say <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/DIY_biology" target="_blank" title="Do it yourself biology (DIY biology, DIY bio) is a growing movement in which individuals, or sometimes small informal organizations, change the genetics of life forms, with small resources, and often little or no formal training, oversight by professionals, or regulation by governments. This may be done as a hobby, sometimes called bio-hacking, or for profit, to start a business. ">do it yourself nanobiotechnology</A></SPAN>. First, different fusion proteins with fluorescent proteins and a luciferase are created. The functionality and efficiency of the immobilization to various materials such as silicon dioxide or cellulose is then characterized by measuring the fluorescence and luminescence, respectively.
</P><H2><SPAN class="mw-headline" id="Bisphenol_A_degradation">Bisphenol A degradation</SPAN></H2><P>In 2005, <A href="http://www.springerlink.com/content/q7864l02734wg32m/" class="external text" rel="nofollow">Sasaki <I>et al.</I></A> isolated a soil bacterium from the <I>Sphingomonas</I> genus which is able to degrade the environmental toxin <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Bisphenol_a" target="_blank" title="Bisphenol A (abbreviated BPA) is a potentially harmful substance used in the production of polycarbonates and epoxy resins. These plastics are used in many food containing bins so we are exposed to BPA every day. ">bisphenol A</A></SPAN> (BPA) with a unique rate and efficiency compared to other BPA degrading organisms. This strain was called <I>Sphingomonas bisphenolicum</I> AO1 and is able to completely decompose 120 mg BPA L<SUP>-1</SUP> in about 6 hours. Three genes which are responsible for the first step of this effective BPA degradation by <I>S. bisphenolicum</I> AO1 were identified: a <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Cytochrome_P450" target="_blank" title="The cytochrome P450 superfamily (officially abbreviated as CYP) is a large and diverse group of enzymes. The function of most CYP enzymes is to catalyze the oxidation of organic substances. The substrates of CYP enzymes include metabolic intermediates such as lipids and steroidal hormones, as well as xenobiotic substances such as drugs and other toxic chemicals. ">cytochrome P450</A></SPAN> (<I>bisdB</I>), a <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Ferredoxin" target="_blank" title="Ferredoxins (often abbreviated Fd) are small proteins containing iron and sulfur atoms organized as iron-sulfur clusters. These biological capacitors can accept or discharge electrons, the effect being change in the oxidation states (+2 or +3) of the iron atoms. This way, ferredoxin acts as electron transfer agents in biological redox reactions. ">ferredoxin</A></SPAN> (<I>bisdA</I>) and a <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Ferredoxin—NADP(%2B)_reductase" target="_blank" title="Ferredoxin: NADP&lt;sup&gt;+&lt;/sup&gt; reductase (abbreviated FNR) is the last enzyme in the transfer of electrons during photosynthesis from photosystem I to NADPH. In nonphotosynthetic organisms, the FNR primarily works in reverse to provide reduced ferredoxin for various metabolic pathways. These pathways include nitrogen fixation, steroid metabolism, oxidative stress response, and iron–sulfur protein biogenesis. ">ferredoxin-NAD<SUP>+</SUP> oxidoreductase</A></SPAN> (<I>FNR</I>) (<A href="http://aem.asm.org/cgi/content/abstract/71/12/8024" class="external text" rel="nofollow">Sasaki <I>et al.</I>, 2005b</A>). The <I>bisdAB</I> genes from <I>S. bisphenolicum</I> AO1 were isolated, transformed into and expressed in <I>E. coli</I> and enabled this bacterium to degrade BPA, too (<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.2008.03843.x/full" class="external text" rel="nofollow">Sasaki <I>et al.</I>, 2008</A>). In addition, the BisdAB proteins from <I>S. bisphenolicum</I> AO1 were able to degrade BPA in a cell free system in which spinach reductase was added (<A href="http://aem.asm.org/cgi/content/abstract/71/12/8024" class="external text" rel="nofollow">Sasaki <I>et al.</I>, 2005b</A>). So we assume that the BisdAB proteins also work in a cell free system together with the ferredoxin-NAD(P)<SUP>+</SUP> oxidoreductase from <I>E. coli</I>. The suggested reaction mechanism of the first BPA degradation step is shown in the project overview image above or in the animated .gif in figure 1:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><A href="/File:Bielefeld-Germany2011-BPAdegrad2.gif" title="File:Bielefeld-Germany2011-BPAdegrad2.gif">File:Bielefeld-Germany2011-BPAdegrad2.gif</A><DIV class="thumbcaption"><B>Figure 1: Suggested reaction mechanism of bisphenol A degradation and BioBricks needed for this reaction <I>in vitro</I>. </B></DIV></DIV></DIV></DIV><P>In 2008, the iGEM team from the <A href="https://2008.igem.org/Team:The_University_of_Alberta/Parts" class="external text" rel="nofollow">University of Alberta</A> submitted the codon usage optimized <I>bisdAB</I> genes from <I>S. bisphenolicum</I> AO1 to the registry of standard biological parts in the so called <A href="http://partsregistry.org/Assembly_standard_25" class="external text" rel="nofollow">Freiburg BioBrick assembly standard</A> (&lt;partinfo&gt;K123000&lt;/partinfo&gt; and &lt;partinfo&gt;K123001&lt;/partinfo&gt;). Via this assembly standard it is very easy to build fusion proteins. We were able to fuse these already existing protein domains together with the NAD(P)<SUP>+</SUP> oxidoreductase gene from <I>E. coli</I> to the fusion protein FNR:Fd<SUB>bisd</SUB>:P450<SUB>bisd</SUB> which subsequently was fused to an S-layer gene. Moreover we could show that the constructed fusion protein FNR:Fd<SUB>bisd</SUB>:P450<SUB>bisd</SUB> is degrading BPA intracellular. We have already shown that the Fd<SUB>bisd</SUB>:P450<SUB>bisd</SUB> fusion protein is degrading BPA more effective in <I>E. coli</I> than the polycistronic <I>bisdAB</I> gene (<A href="/Team:Bielefeld-Germany/Labjournal#Week_15:_8th_august_-_14th_august" title="Team:Bielefeld-Germany/Labjournal">data here</A>).
</P><H2><SPAN class="mw-headline" id="NAD.2B_detection">NAD<SUP>+</SUP> detection</SPAN></H2><P>Our selected NAD<SUP>+</SUP> detection method displays a <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/Molecular_beacon" target="_blank" title="Molecular beacons are oligonucleotide hybridization probes that can report the presence of specific nucleic acids in homogenous solutions. The terms more often used is molecular beacon probes. Molecular beacons are hairpin shaped molecules with an internally quenched fluorophore whose fluorescence is restored when they bind to a target nucleic acid sequence.">molecular beacon</A></SPAN> based approach. These have been initially described in 1996 as nucleic acid probes that fluoresce upon hybridization (<A href="http://www.nature.com/nbt/journal/v14/n3/abs/nbt0396-303.html" class="external text" rel="nofollow">Tyagi <I>et al.</I>, 1996</A>). For this effect the ends of a single-stranded DNA molecule are labeled with a fluorophore as well as with an appropriate quencher. Both are in close proximity to each other due to a formed stem-loop, so that the detection of any fluorescence signal is prevented.
</P><P>
The molecular beacon’s closed state can be applied to a bioassay detecting <SPAN id="igem-tooltip"><A href="http://en.wikipedia.org/wiki/NAD%2B" target="_blank" title="Nicotinamide adenine dinucleotide, abbreviated NAD&lt;sup&gt;+&lt;/sup&gt;, is a coenzyme found in all living cells. In metabolism, NAD&lt;sup&gt;+&lt;/sup&gt; is involved in redox reactions, carrying electrons from one reaction to another. The enzymes that synthesize and use NAD&lt;sup&gt;+&lt;/sup&gt; and NADH are important in both current pharmacology and the research into future treatments for disease.">NAD<SUP>+</SUP></A></SPAN> even in very low concentrations (<A href="http://pubs.acs.org/doi/abs/10.1021/ac102742k" class="external text" rel="nofollow">Tang <I>et al.</I>, 2011</A>). Using two complementary targets hybridizing side-by-side with the hairpin enables NAD<SUP>+</SUP>-dependent DNA ligation by <I>E. coli</I> DNA ligase (figure 2). Only after closing the gap between both hybridized targets the stem melts and the secondary structure gets broken down to a linearized probe-target hybrid. The immediate consequence is a disruption of the close proximity of the fluorophore and the quencher, so that an excitation with light is converted into a visible fluorescence signal. Hence, NAD<SUP>+</SUP> concentration determines DNA ligase activity, which is responsible for the formation of the molecular beacon’s open state and therefore directly correlates with the emerging fluorescence signal. Additionally, the highly selective <A href="http://pubs.acs.org/doi/abs/10.1021/ac102742k" class="external text" rel="nofollow">NAD<SUP>+</SUP> bioassay</A> has a low limit of detection (0,3 nM) compared to other methods.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2: Principle of the NAD<SUP>+</SUP> bioassay using molecular beacons and NAD<SUP>+</SUP>-dependent DNA ligase from <I>E. coli</I> (taken from <A href="http://pubs.acs.org/doi/abs/10.1021/ac102742k" class="external text" rel="nofollow">Tang <I>et al.</I>, 2011</A>).</B></DIV></DIV></DIV></DIV><P>Because of the signal’s stability and the suitability for daily use the <A href="http://pubs.acs.org/doi/abs/10.1021/ac102742k" class="external text" rel="nofollow">NAD<SUP>+</SUP> bioassay</A> can be coupled to NADH-dependent BPA degradation in the context of biosensing. See the results of our NAD<SUP>+</SUP> degration <A href="https://2011.igem.org/Team:Bielefeld-Germany/Results/NAD" class="external text" rel="nofollow">here</A>.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>Kainz B, Steiner K, Möller M, Pum D, Schäffer C, Sleytr UB, Toca-Herrera JL (2010) Absorption, Steady-State Fluorescence, Fluorescence Lifetime, and 2D Self-Assembly Properties of Engineered Fluorescent S-Layer Fusion Proteins of <I>Geobacillus stearothermophilus</I> NRS 2004/3a, <A href="http://pubs.acs.org/doi/abs/10.1021/bm901071b" class="external text" rel="nofollow"><I>Biomacromolecules</I> 11(1):207-214</A>.
</P><P>Sasaki M, Maki J, Oshiman K, Matsumura Y, Tsuchido T (2005a) Biodegradation of bisphenol A by cells and cell lysate from <I>Sphingomonas</I> sp. strain AO1, <A href="http://www.springerlink.com/content/q7864l02734wg32m/" class="external text" rel="nofollow"><I>Biodegradation</I> 16(5):449-459</A>.
</P><P>Sasaki M, Akahira A, Oshiman K, Tsuchido T, Matsumura Y (2005b) Purification of Cytochrome P450 and Ferredoxin, Involved in Bisphenol A Degradation, from <I>Sphingomonas</I> sp. Strain AO1, <A href="http://aem.asm.org/cgi/content/abstract/71/12/8024" class="external text" rel="nofollow"><I>Appl Environ Microbiol</I> 71(12):8024-8030</A>.
</P><P>Sasaki M, Tsuchido T, Matsumura Y (2008) Molecular cloning and characterization of cytochrome P450 and ferredoxin genes involved in bisphenol A degradation in <I>Sphingomonas bisphenolicum</I> strain AO1, <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.2008.03843.x/full" class="external text" rel="nofollow"><I>J Appl Microbiol</I> 105(4):1158-1169</A>.
</P><P>Schäffer C, Novotny R, Küpcü R, Zayni S, Scheberl A, Friedmann J, Sleytr UB, Messner P (2007) Novel Biocatalysts Based on S-Layer Self-Assembly of <I>Geobacillus Stearothermophilus</I> NRS 2004/3a: A Nanobiotechnological Approach, <A href="http://onlinelibrary.wiley.com/doi/10.1002/smll.200700200/pdf" class="external text" rel="nofollow"><I>Small</I> 3(9):1549-1559</A>.
</P><P>Sleytr UB, Huber C, Ilk N, Pum D, Schuster B, Egelseer EM (2007) S-layers as a tool kit for nanobiotechnological applications, <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6968.2006.00573.x/full" class="external text" rel="nofollow"><I>FEMS Microbiol Lett</I> 267(2):131-144</A>.
</P><P>Tang Z, Liu P, Ma C, Yang X, Wang K, Tan W, Lv X (2011) Molecular Beacon Based Bioassay for Highly Sensitive and Selective
Detection of Nicotinamide Adenine Dinucleotide and the Activity of Alanine Aminotransferase, <A href="http://pubs.acs.org/doi/abs/10.1021/ac102742k" class="external text" rel="nofollow"><I>Anal Chem</I> 83(7):2505-2510</A>.
</P><P>Tyagi S, Kramer FR (1996) Molecular beacons: probes hat fluoresce upon hybridization, <A href="http://www.nature.com/nbt/journal/v14/n3/abs/nbt0396-303.html" class="external text" rel="nofollow"><I>Nature Biotechnology</I> 14:303-308</A>.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Bielefeld-Germany/Project">http://2011.igem.org/Team:Bielefeld-Germany/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project&amp;oldid=256359" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>