Team:UI-Indonesia/Description
<!DOCTYPE html>
Home - Hi,Vax! UI iGEM Team 2016
Home
Our Project
Part
Basic Part
Part
Modelling
Human Practice
Field Activities
Hand Book
Integrated Human Practice
Education and Public Engagement
Medal Criteria
Team
References
Notebook
Attributions
Photo Collection
Description
HIV is one of the biggest health issues the world has ever faced, and although antiretroviral therapy (ART) has made it possible to prevent mortality in HIV patients, it has several limitations, including the logistic and economic burden that arises from the need for lifelong treatment. To end HIV, we still need a cure. Today, the best hope for a cure is DNA vaccines, which are DNA molecules that harbour a gene encoding an antigen. When administered, cells will take up these molecules and produce the antigen. Dendritic cells will take this antigen to lymph nodes and present them to T cells, activating them and licensing them to hunt and kill infected cells. In theory, this would allow for the elimination of viral reservoir in the body. In reality, however, DNA vaccination for HIV hasn’t worked in humans.
Although the key component of a DNA vaccine is the antigen-encoding gene, the vaccine backbone, as the name implies, accounts for most of its mass and governs its activity. So, we wondered, can we design a better vaccine backbone? We have designed genetic elements that, when put together, will hopefully achieve improved transgene expression (promoter, intron, polyadenylation sequence and barrier element) and immune stimulation (immunostimulatory DNA motif and RNA gene as well as coding sequences for peptides that direct antigen processing) with respect to existing vaccine backbones. We’re having the parts synthesized and assembling them soon. We hope that this new vaccine backbone will give rise to new, safer and more potent DNA vaccines for HIV—maybe even a cure.
Figure 1. Hi,Vax Plasmid Map
The design of this plasmid was attributed to M. Farhan Maruli, one of our labworkers. This plasmid have advantages over the conventional DNA vaccination plasmid. Our plasmid would be able to overexpress the desired antigen, two same antigen under one promoter. The antigen would be expressed out from the cell and the other would be degraded and be transferred to the MHC class I protein. This way, host’s immune response would be triggered as efficient and effective as possible.  The parts we were going to send as a contribution to the iGEM’s HQ are human-improved version of enchanced Green Flourescent Protein (EGFP) [BBa_K2050420], Thosea asigna Peptide 2A Linker [BBa_K2050421], mouse H1 promoter [BBa_K2050422], tRNA(val) with its linker [BBa_K2050423] and adenoviral VA1 RNA gene [BBa_K2050424].
In the end, we were able to send four out of the five parts. The omitted part was BBa_K2050421. See our basic part’s page for further explanation.
We were planning to use Gibson’s assembly method to assemble our plasmid. We designed four gBlocks of 1800 to 1900 bp length each, with unique 80 overlapping basepair. Below was the overview of Gibson’s assembly method.
Figure 2. Gibson assembly method. Boxes of the same colour resemble overlapping sequences.
As if the grand scheme was vast and complex after the assemble of HiVax, we were not able to complete any of our plan because HiVax was not completely assembled.
We knew our assemble had failed because there was no colony on our plates. Throughout the work, we always use Eschericia coli strain TOP-10 and find no problem at transforming plasmid. The control worked and spawned many colonies on ampicillin plates. This way we also knew that our Gibson reagents still worked.
Figure 3. Unsuccesful Gibson assembly.
We knew our assemble had failed because there was no colony on our plates. Throughout the work, we always use Eschericia coli strain TOP-10 and find no problem at transforming plasmid. The control worked and spawned many colonies on ampicillin plates. This way we also knew that our Gibson reagents still worked.
After consulting to our PI and advisors, we found out that the overlapping regions are too long. We put the region with 80bp overlaps, while the recommendation was 30 to 40 bp overlaps. As with this trouble, we couldn’t continue to work on our real project until had assembled HiVax. To troubleshoot this, we order primers to shorten the overlapping sequence and continue the project afterward. The thing is this happened a month before send-by date deadline, and we ordered the primer a week later. Until this page was written, the primers hadn’t come. Because we knew we would not be able to succeed the original workplan, we improvised to test the EGFP on E. coli, that way, we can validate at least one of our part even though the parts were originally to be tested on Rhabdomyosarcoma and HeLa cells. We were not able to characterize the other parts because all those basic parts work as a system, and the only thing that can be characterize the easiest is EGFP [BBa_K2050420].
Figure 4. Eschericia coli containing MAX promoter coupled with Bba_K2050420 on UV light exposure
We choose BBa_J04450 as a part to be improved. We saw that iGEM’s registry has always been using E. coli 10b as the standard chassis, while TOP-10 is also popular among laboratories. Because this part is also the same part that is used to test the competency of cells; which also is included on every DNA distribution kit, it’s important to know how it behave under different chassis, in particular, E. coli strain TOP-10.
We found out that the line that states RFP could be seen after 16h of incubation doesn’t apply to TOP10 cells. 16 hours of incubation with TOP10 cells results in white colonies. Even the smallest concentration of RFP didn’t show any colony. Following wiki statement, any person would judge their cell was not competent enough or their transformation procedure had failed. When we tested to increase the incubation time, colonies turn wholefully red after incubation up to 24 hours. We were making competent cells with iGEM procedure, and incubation at 37 degree celcius.
© 2016 UI iGEM Team. All rights reserved.
