Team:Macquarie Australia/Collaboration
Project
Description
Design
Protocols
Results
Safety
Notebook
Wet lab
Dry lab
Parts
Summary
Composite parts
Validation
Interlab
Interlab Overview
Human Practices
Overview
Interviews
Public Engagement
Intergrated Human Practices
Customer discovery toolkit
Chlorofella chronicles
Team
Meet The Team
Attributions
Modelling
Our Model
Collaboration
Overview
Judging Form
Judging Form
Major Collaboration- wet lab with  Nanyang Technology University Singapore
Bidirectional Collaboration With NTU Singapore
For the third consequtive year, Macquarie University has collaborated with the NTU Singapore iGEM team . After a number of discussions, we were able to organise ways so that we could both assist each other.
Following our success last year in the purification and in vitro assays of their dCas9 constructs, they requested that we repeat said process with their new mutant variants this year.
To assist us, we required them to perform a quantitative PCR on two of our composite Chlorophyll biosynthesis parts. BBa_K2664007 which adds a tail to protochlorophyllide, turning it into chlorophyll a (Figure 4) and BBa_K2664006 which adds a magnesium ion to protoporphyrin IX (Figure 5). As we have changed the promoters from lac to trc on a number of parts, we wished to quantify protein expression after induction with IPTG in our parts.
What We Did For Them:
The in vitro assay that we performed was on four truncated dCas9 mutants: 3ple, 3ple 4QHP, Rec2_HNH, and Rec2_HNH 4QHP.  This project used the dCas9 enzyme to form a Ribonucleoprotein (RNP) complex with gRNA, and subsequently bind to the target sequence in our PCR product (similar to what we did for the last two years). The results from the electrophoretic mobility shift assay (EMSA) show that the dCas9 enzymes (3ple, 3ple 4QHP, Rec2_HNH, and Rec2_HNH 4QHP) no longer retain their nuclease activity (see Figures 2,3). This is evident from the gel shift present, the shift of the target sequence protein, indicating the mutant dCas9 has bound the target sequence of DNA from its slower migration through the gel (see Figure 2,3).
Protein Purification
The plasmids were transformed into BL21 (DE3) E. coli and expressed under auto-induction conditions. These cells were then harvested, lysed and purified using a Ni-NTA resin with standard His-tag purification protocols. Eluates were then run on an SDS-PAGE gel (Figure 1). The darkest bands observed above 100 kDa correspond to the expected dCas9 sizes which suggests the expression and purification was successful.
Figure 1. SDS-PAGE gel of the His-tagged dCas9 constructs. The darkest bands observed above 100 kDa are the expected sizes of the dCas9 proteins.  Lane 1 contains a 250kDa ladder. Lane 2 contains 3ple lysate. Lane 3 contains 3ple fraction 1. Lane 4 contains 3ple fraction 2. Lane 5 contains 3ple4QHP lysate. Lane 6 contains 3ple4QHP fraction 1. Lane 7 contains 3ple4QHP fraction 2. Lane 8 contains a 250kDa ladder. Lane 9 contains Rec2_HNH lysate. Lane 10 contains Rec2_HNH fraction 1. Lane 11 contains Rec2_HNH fraction 2. Lane 12 contains Rec2_HNH4qhp lysate. Lane 13 contains Rec2_HNH4QHP fraction 1.
In vitro CRISPR assay + EMSA
To test these dCas9 constructs we again included an electrophoretic migration shift assay as last year. As the dCas9 constructs were designed to not have nuclease activity, while retaining the ability to bind to the gRNA and target DNA, we set up variations of the in vitro CRISPR assay to track the progression of the target DNA, gRNA, cleavage products, and protein migration on a DNA retardation gel (6% acrylamide in TBE, ThermoFisher Scientific).
The four purified samples were subjected to three treatments (Figure 2, 3):
Pre-incubated: gRNA + enzymes (incubated at room temperature for 5 minutes in all samples)
Incubated: gRNA + enzyme + PCR product (further incubated for 1 hour at 37°C)
Deactivated: gRNA + enzyme + PCR product (further incubated for 1 hour at 37°C) and a deactivation step of 20 minutes at 80°C.
Three protein bands were observed in the pre-incubated samples (Lane 1 and 5 in Figures 2, 3). The dCas9 proteins were expected to be the highest bands, indicated as “Native protein”. Post incubation of the enzymes with the PCR product binding and gel shifting was observed, with little migration through the gel (Lane 2 and 5 in Figures 2, 3). After inactivation of the enzymes (Lanes 3 and 6 in Figures 2, 3) protein aggregation was observed at the top of the lanes. This indicates that the binding to the PCR was reversed.
Figure 2. Retardation gel (6%) electrophoresis of the 3ple dCas9 and its mutant variant 3ple4QHP. The gel was imaged using both Gel Red (shown in black) and Coomassie Blue (shown in green). These two images were then overlayed. Lane 1 contains 3ple only. Lane 2 contains 3ple RNP with the PCR product. Lane 3 contains 3ple RNP with the PCR product, incubated at 37 °C and deactivated at 80°C. Lane 4 contains 3ple4QHP only. Lane 5 contains 3ple4QHP RNP with the PCR product. Lane 6 contains 3ple4QHP RNP with the PCR product, incubated at 37 °C and deactivated at 80°C.
Figure 3. Retardation gel (6%) electrophoresis of the Rec2_HNH dCas9 and its mutant variant Rec2_HNH 4QHP. The gel was imaged using both Gel Red (shown in black) and Coomassie Blue (shown in green). These two images were then overlayed. Lane 1 contains Rec2_HNH only. Lane 2 contains Rec2_HNH RNP with the PCR product. Lane 3 contains Rec2_HNH RNP with the PCR product, incubated at 37 °C and deactivated at 80°C. Lane 4 contains Rec2_HNH 4QHP only. Lane 5 contains Rec2_HNH 4QHP RNP with the PCR product. Lane 6 contains Rec2_HNH 4QHP RNP with the PCR product, incubated at 37 °C and deactivated at 80°C.
What They Did For Us:
The two plasmids were first transformed into DH5α cells and grown overnight at room temperature in LB broth with chloramphenicol added. The two plasmids were treated under three conditions: one uninduced, one induced with 0.5 mM IPTG and one induced with 1 mM IPTG. The Protochlorophyllide to chlorophyll a operon was tested with four genes, all of which showed good expression rates (Figure 4.). The Mg Chelatase operon was tested for five genes, however the ChlH and ChlD genes showed lower than expected or no expression at all (Figure 5.). We are investigating as to why these two genes had lower expression once inducted.
Formula:
2^-(ΔΔCt) is used to calculate the fold change of gene expression after induction, in which
ΔCt(control)1 = Ct(uninduced operon genes) - Ct(Chloramphenicol gene)
ΔCt(test)2 = Ct(Induced operon genes) - Ct(Induced Chloramphenicol gene)
Finally, use ΔCt(test)2 - ΔCt(control)1 to get ΔΔCt
Figure 4. Showing the results of the qPCR of the Protochlorophyllide to chlorophyll a operon normalised against the Chloramphenicol backbone. Cultures were subjected to three treatments: without IPTG, 0.5 mM IPTG and 1 mM IPTG.
Figure 5. Showing the results of the qPCR of the Mg chelatase operon normalised against the Chloramphenicol backbone. Cultures were subjected to three treatments: without IPTG, 0.5 mM IPTG and 1 mM IPTG.
Other collaborations
The Macquarie University iGEM team skyped, emailed and responded to surveys from numerous teams around the world. The team also participated in a virtual conference and a symposium that allowed us to leave the university grounds and explain our project to the community. Though this communication will not be evaluated for any medal criteria, we feel that it is important to reference the teams that helped us along our path to Boston.
Human Practices survey
A survey created by the Macquarie University iGEM team was sent to other iGEM teams to provide us with an insight into the resources used for the human practices component of their project. As seen on the map above, teams from around the world completed our survey and using their feedback, our team was able to incorporate this data for our customer discovery toolkit. Furthermore, we responded to several other teams’ surveys, many of which completed our own survey, in an attempt to aid them with their own projects.
UNSW symposium
We would like to thank the UNSW iGEM team for organising an event that provided the Macquarie University iGEM team with the opportunity to present our project for the first time. Hearing from other teams and scientists that spoke on the panel gave us a better understanding of our project and how we could improve before we present it at the jamboree.
Virtual Conference
iGEM NYU Abu Dhabi ran a virtual conference which enabled 6 iGEM teams (iGEM HKUST, iGEM Rec Chennai, iGEM VIT, iGEM Saint Joseph, iGEM NYU Abu Dhabi and iGEM Macquarie) to virtually meet one another to discuss their projects. The event had an emphasis on questions and discussion of each project. The conference gave us a chance to learn from other teams and discuss any issues that were unclear to help each team throughout their journey.
Additional Collaborations
iGEM Macquarie was contacted by a couple of teams asking to send science related photos to unite the science community from different countries. Two of the main highlights were when  iGEM TecMonterrey GDL asked us to take a photo with a sign showing how far away we were from the Jamboree, showing that ‘science is everywhere’ and when iGEM Estonia created a huge mosaic of photos showing fashion in the lab.
#ScienceEverywhere collaboration with iGEM TecMonterrey GDL
Lab Safety Fashion collaboration with iGEM Estonia
