<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Macquarie_Australia_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Macquarie Australia/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin-top: 15px;"><LI class="dropdown"><A class="dropdown-toggle active" style="font-family: 'Raleway', sans-serif; font-size: 12;">Project</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Description">Description</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Design">Design</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Protocols">Protocols</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Results">Results</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle active" style="font-family: 'Raleway', sans-serif;font-size: 12;">Notebook</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Notebook">Wet lab</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Notebookdry">Dry lab</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Parts</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Parts">Summary</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/CompositeParts">Composite parts</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/PartsValidation">Validation</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Interlab</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/InterLab">Interlab Overview</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Human Practices</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Human_Practices">Overview</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Interviews">Interviews</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Public_Engagement">Public Engagement</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/IntergratedHumanPractices">Intergrated Human Practices</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Toolkit">Customer discovery toolkit</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Chlorofella">Chlorofella chronicles</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Team</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Team">Meet The Team</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Attributions">Attributions</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Modelling</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Model">Our Model</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Collaboration</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Collaborations">Overview</A></LI><LI class="dropdown"><A class="dropdown-toggle active" style="font-family: 'Raleway', sans-serif;font-size: 12;">Judging Form</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://igem.org/2018_Judging_Form?id=2664">Judging Form</A></LI></DIV></DIV><DIV style="width: 50%; margin-left: auto; margin-right: auto; display: block;"><H style="text-align: justify; font-size: 25px; font-weight: bold;">
                Wet lab notebook
            </H><BUTTON class="accordion">Week 1: July 2nd-July 8th</BUTTON><DIV class="panel"><UL class="normal"><LI>
                        Project planning - no wet lab work this week.
                    </LI></UL></DIV><BUTTON class="accordion">Week 2: July 9th-July 15th</BUTTON><DIV class="panel"><UL class="normal"><LI>
                        We went through lab safety induction to ensure safety procedures were followed at all times in the lab
                    </LI><LI>
                        We learnt how to perform digestion, ligation, transformation and how to make and run agarose gels
                    </LI><LI>
                        We used digestion to screen the Photosystem II (PSII) parts DCA, psbELJTB and psbMZtlwkOPQR. psbELJTB and psbMZtlwkOPQR showed successful digestion and correct sizes. DCA showed signs of contamination in all samples.
                    </LI><LI>
                        We decided to focus of the Chlorophyll plasmid rather than PS II. We proceeded with digestion and ligation of ChlD with ChlI2 as well as DRV1 with ChlG, both into Kanamycin backbones, following the 3A assembly protocol.
                    </LI><LI>
                        Renee, Areti and Karl also began the interlab study:
                    </LI></UL><UL class="indented"><LI>
                        Made LB media + cam plates
                    </LI><LI>
                        Transformed <I>Εscherichia coli</I> DH5α cells with the interlab testing devices.Transformation was performed twice, due to poor colony growth in the LB + cam media at the first round of transformations.
                    </LI></UL><P style="text-align: justify; font-size: 12px; font-family: 'Quicksand', sans-serif;"><B><I>Figure 1.</I></B> Agarose gel (1%) electrophoresis of single (E) and double (E+S or E + X) digests of Photosystem II composite parts (ELJTB and MZHWKOPQR). Both ELJTB samples showed correct banding in single and double digests for the 2.1kb part (same size as backbone). 1 samples for MZHWKOPQR showed correct digestion with the 3.2kb part
                </P></DIV><BUTTON class="accordion">Week 3: July 16th-July 22nd</BUTTON><DIV class="panel"><UL class="normal"><LI>
                        We transformed the DVR1-ChlG and ChlD-ChlI2 ligation samples into competent DH5α <I>E.coli</I> cells. DVR1-ChlG had successful transformants which were then incubated in LB broth + Kan. ChlD-ChlI2 had given an unsuccessful transformation, so the process was repeated and was successful the second time. The samples were then liquid cultured.
                    </LI><LI>
                        Interlab experiments were completed this week:
                    </LI></UL><UL class="indented"><LI>
                        Calibrations 1, 2 and 3 were completed early on in the week
                    </LI><LI>
                        Cell measurements (Abs600 and fluorescence) were taken as per the interlab protocol
                    </LI><LI>
                        Colony forming units per OD600 = 0.1 of the negative and positive control devices was also determined (in triplicates)
                    </LI></UL></DIV><BUTTON class="accordion">Week 4: July 23rd-July 29th</BUTTON><DIV class="panel"><UL class="normal"><LI>
                        Set up backbone insertions for trc-POR, trc-ChlI1 (Digestion, ligation and transformation
                    </LI><LI>
                        Interlab experiments:
                    </LI></UL><UL class="indented"><LI>
                        Colony forming units per OD600 = 0.1 of the negative and positive control devices was completed again.
                    </LI><LI> 
                        Data analysis and submission. 
                    </LI></UL></DIV><BUTTON class="accordion">Week 5: July 30th-August 5th</BUTTON><DIV class="panel"><UL class="normal"><LI>
                        The liquid cultures of DVR1-ChlG and ChlD-ChlI2 were miniprepped and digested (single, double digest) to screen for successful ligations by agarose gel electrophoresis (1% agarose).
                    </LI></UL><UL class="indented"><LI>
                        The gel showed unsuccessful ligations so we switched our ligation technique to standard assembly and backbone swapping of necessary parts
                    </LI></UL><UL class="normal"><LI>
                        We backbone-swapped DVR1 to Kan and were in the process of swapping ChlD, trc-POR and GUN4
                    </LI><LI>
                        Made up more plasmid stock of current parts by plasmid transformation, culturing and miniprep
                    </LI><LI>           Begun Standard assembly of DVR1 + ChlG
                    </LI></UL><UL class="indented"><LI>
                        Miniprepped all samples
                    </LI></UL><UL class="normal"><LI>
                        Begun induction and SDS-PAGE to compare lac-ChlH and trc-ChlH
                    </LI></UL><P style="text-align: justify; font-size: 12px; font-family: 'Quicksand', sans-serif;"><B><I>Figure 2.</I></B> Agarose gel (1%) electrophoresis of single (E) and double (EP) digests of lac-ChlM, lac-ChlG and lac-ChlI2. This gel was checking parts we had transformed to make up more stock of for this years project, all part sizes were correct.
                </P><P style="text-align: justify; font-size: 12px; font-family: 'Quicksand', sans-serif;"><B><I>Figure 3.</I></B> Agarose gel (1%) electrophoresis of single (E) and double (EP) digests of plasto, ChlD, ChlP (In both Cam and Kan backbones) and POR. This gel was checking parts we had transformed to make up more stock of for this years project, Correct part sizes were observed for each part.
                </P></DIV><BUTTON class="accordion">Week 6: August 6th-August 12th</BUTTON><DIV class="panel"><UL class="normal"><LI>
                        Ran GUN4, ChlD and trc-POR backbone swaps as well as DVR1-ChlG on a 1% agarose gel
              </LI><LI>
                        Sent off successful samples of each for sequencing
                    </LI><LI>
                        We inserted synthesised DNA biobricks into Amp and Cam backbones for trc-POR,YidC and trc-ChlI1, trc-ycf39-HliD
                    </LI><LI>
                        We then transformed, liquid-cultured, miniprepped and screened on a 1% agarose gel
                    </LI><LI>
                        We had successes for trc-ChlI1 in Amp, which we sent for sequencing confirmation
                    </LI><LI>
                        Proceeded with standard assembly for:
                    </LI></UL><UL class="indented"><LI>
                        CTH1-ycf54-ChlM-ter + trc-FNR-fdx or trc-FNR
                    </LI><LI>
                        trc-ChlH + GUN4
                    </LI><LI>
                        trc-POR + ChlP
                    </LI><LI>
                        ChlI2 + ChlD
                    </LI><LI>
                        We sent CTH1-ycf54-ChlM-ter-trc-FNR-fdx as well as CTH1-ycf54-ChlM-ter-trc-FNR for sequencing
                    </LI><LI>
                        Sequencing came back for trc-ChlH to show that the promoter swap was unsuccessful. A reattempt of promoter swap was then undertaken
                    </LI><LI>
                        Transformed trc-POR-ChlP and ChlI2-ChlD
                    </LI></UL><P style="text-align: justify; font-size: 12px; font-family: 'Quicksand', sans-serif;"><B><I>Figure 4.</I></B> Agarose gel (1%) electrophoresis of single (E) and double (EP) digests of trc-POR (Backbone insertion into Cam/Kan) and DVR1-ChlG ligation. Samples 2 &amp; 3 of trc-POR (Kan) and Sample 2 of trc-POR (Cam) showed correct band size. DVR1-ChlG showed correct band size for our very first successful standard assembly (Biobrick and plasmid backbone similar size forming a large single band in double digests).
                </P></DIV><BUTTON class="accordion">Week 7: August 13th-August 19th</BUTTON><DIV class="panel"><UL class="normal"><LI>Screened trc-ChlI1 and sent 1 successful plasmid away for sequencing</LI><LI>Performed SDS-PAGE comparing trc-FNR-fdx and lac-FNR-fdx</LI><LI>Liquid-cultured, Miniprepped and screened trc-ChlH, trc-POR-ChlP, trc-ycf39-HliD, trc-FNR and trc-FNR-fdx and ChlI2-ChlD </LI><UL class="indented"><LI>
                            sent ChlH, trc-POR, trc-ycf39-HliD, trc-FNR-fdx and ChlI2-ChlD plasmids for sequencing
                        </LI></UL><LI>Digestion/ligation for trc-POR+ChlP, ChlI1+ChlI2-ChlD, trc-ChlH-GUN4</LI><LI>Glycerol stock made for ChlH</LI></UL><P style="text-align: justify; font-size: 12px; font-family: 'Quicksand', sans-serif;"><B><I>Figure 5.</I></B> Agarose gel (1%) electrophoresis of single (E) and double (EP) digests of 
                    ChlI2-ChlD and trc-POR-ChlP standard assembly ligations. All samples of ChlI2-ChlD had successful band sizes. trc-POR-ChlP did not show correct banding.
                </P></DIV><BUTTON class="accordion">Week 8: August 20th-August 26th</BUTTON><DIV class="panel"><UL class="normal"><LI>Transformed, Liquid-cultured and miniprepped  trc-ChlH-GUN4, trc-ChlI1-ChlI2-ChlD and trc-POR-ChlP</LI><LI>Made glycerol stock for trc-POR and trc-ChlH</LI><LI>Digested and screened trc-ChlH-GUN4, trc-FNR and trc-FNR-fdx. Only trc-FNR-fdx and trc-ChlH-GUN4 had successes, which were sent off for sequencing</LI><LI>Induced trc-POR cells and ran the cell lysate on SDS-PAGE</LI></UL></DIV><BUTTON class="accordion">Week 9: August 27th-September 2nd</BUTTON><DIV class="panel"><UL class="normal"><LI>Digested and screened  trc-ChlH-GUN4, trc-ChlI1-ChlI2-ChlD and trc-POR-ChlP on a 1% agarose gel and sent successful samples for sequencing</LI><LI>Backbone-swapped lac-CTH1-ycf54-ChlM (Kan)
                    </LI><LI>Performed standard assembly on trc-POR-ChlP-DVR1-ChlG and sent successful ones for sequencing</LI><LI>Induced new trc-POR samples and ran cell lysate on an SDS-PAGE </LI><LI>Grew barley for protochlorophyllide extraction</LI><LI>Made up more plasmid stock of trc-POR</LI><LI>Made up glycerol stock of ChlI1</LI><LI>Redesigned and ordered sequencing primers</LI><LI>Tried to insert G-block for yidC and trc-FNR into ampicillin backbone, no successes from screened samples</LI><LI>Digestion/ Ligation of trc-ChlI1-ChlI2-ChlD + ChlH-GUN4. Backbone swap of trc-POR (switched to Cam to send to iGEM) and lac-CTH1-ycf54-ChlM + trc-POR</LI></UL><P style="text-align: justify; font-size: 12px; font-family: 'Quicksand', sans-serif;"><B><I>Figure 6.</I></B> Agarose gel (1%) electrophoresis of single (E) and double (EP) digests of trc-ChlH-GUN4, trc-POR-ChlP and trc-ChlI1-ChlI2-ChlD standar assembly ligations. trc-ChlH-GUN4 and trc-POR-ChlP showed successful ligations with correct band sizes.
                </P></DIV><BUTTON class="accordion">Week 10: September 3rd-September 9th</BUTTON><DIV class="panel"><UL class="normal"><LI>Received sequencing results for trc-POR-ChlP and sent off trc-POR-ChlP-DVR1-ChlG samples for sequencing </LI><LI>Digested, ligated, transformed and liquid-cultured lac-CTH1-ycf54-ChlM + trc-FNR-fdx</LI><LI>Transformed trc-ChlI1-ChlI2-ChlD + trc-ChlH-GUN4 (shortened to trc-IH),  Backbone-swapped trc-POR (to Cam to send to iGEM) and lac-CTH1-ycf54-ChlM + trc-POR</LI><UL class="indented"><LI>Liquid-cultured, miniprepped and performed single/double screening of digests on a 1% agarose gel</LI><LI>Sent 3 IH samples and 1 of both trc-POR (Cam) and lac-CTH1-ycf54-ChlM-trc-POR for sequencing</LI></UL><LI>Made induction plates and induced trc-ChlI1-ChlI2-ChlD-trc-ChlH-GUN4 (Mg-Chelatase) to see if we could get Mg-PPIX formation</LI><LI>Digested and ligated  trc-ChlI1-ChlI2-ChlD-ChlH-GUN4 + lac-CTH1-ycf54-ChlM-trc-POR (Streamlined operon)</LI><LI>Bradford assay of trc-POR samples to determine concentration of protein in the supernatant and pellet fractions of the cell lysate</LI><UL class="indented"><LI>POR is a membrane-associated protein, more (total) dissolved proteins in the supernatant</LI></UL><LI>Extract protochlorophyllide from etiolated seedlings of barley </LI><UL class="indented"><LI>C<SUB>18</SUB> Sep-Pak used for further purification</LI></UL></UL><P style="text-align: justify; font-size: 12px; font-family: 'Quicksand', sans-serif;"><B><I>Figure 7.</I></B> Agarose gel (1%) electrophoresis of single (E) and double (EP) digests of trc-POR-ChlP-DVR1-ChlG formed by standard assembly. Both sets of samples showed correct banding for the part and backbone.
                </P></DIV><BUTTON class="accordion">Week 11: September 10th-September 16th</BUTTON><DIV class="panel"><UL class="normal"><LI>Received sequencing results for trc-ChlI1-ChlI2-ChlD-ChlH-GUN4, discovered trc-ChlI1 had not ligated initially but all other genes were present and sequence was confirmed</LI><UL class="indented"><LI>Stopped streamline operon and digested and ligated trc-ChlI1 to ChlI2-ChlD-ChlH-GUN4</LI></UL><LI>Miniprepped 7 liquid cultures of lac-CTH1-ycf54-ChlM-trc-FNR-fdx (Kan)</LI><UL class="indented"><LI>Performed single/double digests then ran on 1.5% agarose gel</LI><LI>Sent the 1 successful plasmid off for sequencing</LI><LI>Liquid cultured 8 more colonies of lac-CTH1-ycf54-ChlM-trc-FNR-fdx (Kan) </LI></UL><LI>Performed a functional assay of trc-POR induced cells using protochlorophyllide as substrate</LI><UL class="indented"><LI>Results were inconclusive and redone by Professor Robert Willows</LI></UL></UL></DIV><BUTTON class="accordion">Week 12: September 17th-September 23rd</BUTTON><DIV class="panel"><UL class="normal"><LI>Received sequencing results for  lac-CTH1-ycf54-ChlM-trc-FNR-fdx (Kan) and found frameshift mutation</LI><UL class="indented"><LI>8 other liquid cultures were miniprepped and screened and 2 successful results were sent off for sequencing</LI></UL><LI>Digestion and ligation of lac-CTH1-ycf54-ChlM-trc-FNR-fdx + trc-POR-ChlP-DVR1-ChlG and backbone swap of trc-POR into Cam</LI><LI>First round of transformations were unsuccessful for lac-CTH1-ycf54-ChlM-trc-FNR-fdx-trc-POR-ChlP-DVR1-ChlG, trc-POR (Cam) and trc-ChlI1-ChlI2-ChlD-ChlH-GUN4; we deduced the issue was with the competent cells we used</LI><UL class="indented"><LI>Commercial competent cells were used to re-transform samples and succeeded</LI><LI>8 Colonies from each plate were liquid-cultured</LI></UL></UL></DIV><BUTTON class="accordion">Week 13: September 24th-September 30th</BUTTON><DIV class="panel"><UL class="normal"><LI>Miniprepped and screened through PCR and agarose gel: lac-CTH1-ycf54-ChlM-trc-FNR-fdx-trc-POR-ChlP-DVR1-ChlG, trc-POR (Cam) and trc-ChlI1-ChlI2-ChlD-ChlH-GUN4</LI><UL class="indented"><LI>Sent 2 samples of trc-ChlI1-ChlI2-ChlD-ChlH-GUN4 and 4 samples of lac-CTH1-ycf54-ChlM-trc-FNR-fdx-trc-POR-ChlP-DVR1-ChlG for sequencing</LI><LI>trc-POR backbone swap did not work so we redigested the original G-Block DNA for backbone insertion into Cam.</LI></UL><LI>Induced  trc-ChlI1-ChlI2-ChlD-ChlH-GUN4 (Mg-Chelatase) plasmids into competent cells and used Fluorescence testing to check for enzyme expression and function (formation of Mg-PPIX which is fluorescent).</LI><UL class="indented"><LI>Growth was too little to yield results at that moment.</LI></UL><LI>We are performing Cas9 purification as part of our collaboration with NTU so we prepared all necessary solutions for Cas9 purification in preparation for the arrival of their biobricks</LI></UL><P style="text-align: justify; font-size: 12px; font-family: 'Quicksand', sans-serif;"><B><I>Figure 8.</I></B> Agarose gel (1%) electrophoresis of single (E) and double (EP) digests of trc-ChlI1-ChlI2-ChlD-trc-ChlH-GUN4 and trc-POR Cam. trc-ChlI1-ChlI2-ChlD-trc-ChlH-GUN4 showed some successes that will be screened by PCR as the ligation once inserted a 1100bp which is difficult to see on a part this size. trc-POR showed incorrect banding in single digests showing multiple bands.
                </P><P style="text-align: justify; font-size: 12px; font-family: 'Quicksand', sans-serif;"><B><I>Figure 9.</I></B> Agarose gel (1%) electrophoresis of single (E) and double (EP) digests of lac-CTH1-ycf54-ChlM-trc-FNR-fdx-trc-POR-ChlP-DVR1-ChlG standard assembly ligation. The banding size shows correct insertion of trc-POR-ChlP-DVR1-ChlG into lac-CTH1-ycf54-ChlM-trc-FNR-fdx. This forms the second half of our Chlorophyll biosynthesis operon.
                </P></DIV><BUTTON class="accordion">Week 14: October 1st-October 7th</BUTTON><DIV class="panel"><UL class="normal"><LI>Performed the standard assembly of our last 2 parts together for our Chlorophyll plasmid:   trc-ChlI1-ChlI2-D-trc-ChlH-GUN4 +  lac-CTH1-ycf54-ChlM-trc-FNR-fdx-trc-POR-ChlP-DVR1-ChlG</LI><UL class="indented"><LI>Screened by colony PCR which was able to narrow down our samples, we miniprepped samples and performed single/double digests for screening</LI></UL><LI>Mg-PPIX induction testing showed high formation of Zn-PPIX which had a similar fluorescence to Mg-PPIX. This dampened out any meaningful results</LI><UL class="indented"><LI>We proceeded using M9 minimal salt plates for our induction experiments to reduce this problem</LI></UL><LI>Received constructs from NTU and transformed and liquid cultured them, then proceeded to begin with Cas 9 extraction</LI></UL><P style="text-align: justify; font-size: 12px; font-family: 'Quicksand', sans-serif;"><B><I>Figure 10.</I></B> Agarose gel (1%) electrophoresis of single (E) and double (EP) digests of trc-POR G-block insertion into Cam biobrick backbone for iGEM submission. Samples 4 and 7 showed correct banding of trc-POR biobrick and Cam backbone.
                </P></DIV><BUTTON class="accordion">Week 15: October 8th-October 12th</BUTTON><DIV class="panel"><UL class="normal"><LI>Received sequencing results back for 1st half and last half of complete Chlorophyll pathway plasmid.</LI><UL class="indented"><LI>Miniprepped sequence confirmed cultures and used PCR to test for presence of genes from each operon of the plasmid (ChlI1, GUN4, trc-FNR-fdx and trc-POR)</LI><LI>Set up induction plates of full chlorophyll plasmid samples</LI></UL><LI>Used SpeedVac to dry out samples in iGEM DNA sample wells for Biobrick sending to iGEM headquarters</LI><UL class="indented"><LI>Sent trc-ChlH, trc-FNR-fdx, trc-POR, trc-ChlI1-ChlI2-ChlD-trc-ChlH-GUN4, trc-POR-ChlP-DVR1-ChlG and the Complete Chlorophyll pathway biobrick:  trc-ChlI1-ChlI2-ChlD-trc-ChlH-GUN4-lac-CTH1-ycf54-ChlM-trc-FNR-fdx-trc-POR-ChlP-DVR1-ChlG</LI><LI>Set up show gel for small parts (trc-ChlH, trc-FNR-fdx and trc-POR)</LI></UL><LI>Transformed lac-CTH1-ycf54-ChlM (w/ and w/out trc-FNR-fdx) into trc-ChlI1-ChlI2-ChlD-trc-ChlH-GUN4 competent cells for functional assay</LI><LI>Set up trc-ChlI1-ChlI2-ChlD-trc-ChlH-GUN4 on M9 minimal salt plates with ALA and IPTG and used fluorescence assay to test for Mg-PPIX
                    </LI><UL class="indented"><LI>PPIX formation was visible but Mg-PPIX was not</LI></UL><LI>Extracted His-tagged proteins from NTU constructs and ran SDS-PAGE showing successful extraction</LI><UL class="indented"><LI>Used protein concentrators with a molecular weight cut-off (MWCO) of 100kDa to spin down and remove protein bands under 100 kDA (protein was above that)</LI><LI>Attempted to fuse Cas9 with guide RNA. Unfortunately commercial guide RNA was not functioning to bind to Cas9</LI><LI>Buffer exchange by adding fresh Cas9 working buffer and spinning it down (3 times)</LI></UL><LI>Performed functional assay of Mg-chelatase (trc-ChlI1-ChlI2-ChlD-trc-ChlH-GUN4)</LI><UL class="indented"><LI>Used large scale inductions; unfortunately cell number was too low for assay</LI></UL></UL></DIV><BUTTON class="accordion">Week 16: October 15th-October 17th</BUTTON><DIV class="panel"><UL class="normal"><LI>Completed show gel of composite parts</LI><LI>Performed fluorescence assay of our induced Mg-chelatase plasmid incubated with PPIX substrate</LI><UL class="indented"><LI>Used verified Mg-chelatase subunit proteins (ChlI1, ChlI2, ChlD, ChlH and GUN4) as positive control, showed Mg-PPIX formation</LI><LI>Each subunit was mixed with the induced samples to see if protein expression of any subunit had been a limiting factor in our construct, no individual subunit was shown be limiting factor (none of them showed Mg-PPIX production)</LI></UL><LI>Ran SDS-PAGE gel of our IH3 induced construct</LI><UL class="indented"><LI>ChlH and ChlI1 expression was visible</LI></UL></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>