<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Macquarie_Australia_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Macquarie Australia/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin-top: 15px;"><LI class="dropdown"><A class="dropdown-toggle active" style="font-family: 'Raleway', sans-serif; font-size: 12;">Project</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Description">Description</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Design">Design</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Protocols">Protocols</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Results">Results</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle active" style="font-family: 'Raleway', sans-serif;font-size: 12;">Notebook</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Notebook">Wet lab</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Notebookdry">Dry lab</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Parts</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Parts">Summary</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/CompositeParts">Composite parts</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/PartsValidation">Validation</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Interlab</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/InterLab">Interlab Overview</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Human Practices</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Human_Practices">Overview</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Interviews">Interviews</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Public_Engagement">Public Engagement</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/IntergratedHumanPractices">Intergrated Human Practices</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Toolkit">Customer discovery toolkit</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Chlorofella">Chlorofella chronicles</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Team</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Team">Meet The Team</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Attributions">Attributions</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Modelling</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Model">Our Model</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Collaboration</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Collaborations">Overview</A></LI><LI class="dropdown"><A class="dropdown-toggle active" style="font-family: 'Raleway', sans-serif;font-size: 12;">Judging Form</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://igem.org/2018_Judging_Form?id=2664">Judging Form</A></LI></DIV></DIV><DIV style="width: 50%; margin-left: auto; margin-right: auto; display: block;"><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
The ultimate goal of our project was to produce vesicles in <I>E. coli</I>. We set out to create a system that could enable simplified protein purification, as well as provide an isolated space for biochemistry within cells. During the first iteration of our design process, these concepts were our main focus. After reviewing the literature, we discovered that we could use the chlorophyll biosynthesis pathway to produce semi-crystalline aggregates of phospholipids and proteins called prolamellar bodies. When these aggregates are exposed to light, they spontaneously form vesicles. We saw this as a great opportunity to give chlorophyll biosynthesis an entirely new application within the biotechnology world. We envision that our vesicles could be used in research labs, as well as in industrial synthesis in a broad range of projects. Before starting work in the lab, we needed to thoroughly design and plan our project, and to do this we established several guiding engineering principles.


    </P></DIV><DIV style="width: 50%; margin-left: auto; margin-right: auto; display: block;"><H style="font-size: 20px; font-weight: bold;">Engineering Principles

    </H><P style="text-align: justify; font-size: 14px;"><UL class="normal"><LI>
We wanted our vesicles to be simple to use. Multiple plasmids, unusual or complicated growing conditions, or an unusual expression vector were all to be avoided.
</LI></UL><UL class="indented"><LI>
- To address this, we planned to assemble all of our genes onto a single plasmid with a single antibiotic resistance gene. This would enable any cell line to produce vesicles with a single transformation. Additionally, it would leave our cells useful for further transformations using other antibiotic resistance genes. Finally, we planned to optimise our project for use in <I>E. coli</I>, which is a ubiquitous cell with well established utility in industry and research, and an extensive catalogue of compatible synthetic biology parts available.
</LI></UL><UL class="normal"><LI>
We wanted our vesicles to be easy to isolate.
</LI></UL><UL class="indented"><LI>
- To accomplish this, we envisioned bulk purification of target proteins or small molecules. We could capture compounds within our vesicles and separate them using centrifugation, or antibody affinity chromatography utilising membrane bound proteins within the vesicle membranes. Centrifugation was particularly desirable, as this would minimise the need for time and labour intensive chromatography steps in purification protocols.
</LI></UL><UL class="normal"><LI>
We wanted our vesicles to be easy to detect. Simple detection of vesicles would make purification of target compounds more efficient.
</LI></UL><UL class="indented"><LI>
- In addressing this, we planned to use the unique spectral properties of chlorophyll to enable the simple and rapid spectroscopic identification of our vesicles.
</LI></UL><UL class="normal"><LI>
We wanted our vesicles to be useful and tunable as research tools and to comply with current synthetic biology standards.
</LI></UL><UL class="indented"><LI>
- This involved packaging our final assembly of genes into a familiar biobrick layout, as well as providing sequences and samples of each of the simple parts. This would enable future researchers to make adjustments as necessary.
</LI></UL><UL class="normal"><LI>
We had to be able to test for our vesicles, and make sure they were forming from the prolamellar bodies.
</LI></UL><UL class="indented"><LI>
- We planned to observe our vesicles microscopically, using the naturally embedded chlorophyll as a UV reactive stain.
</LI></UL></P></DIV><DIV style="width: 50%; margin-left: auto; margin-right: auto; display: block;"><H style="font-size: 20px; font-weight: bold;">Plasmid Design - Putting Parts Together

    </H><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
With engineering principles established, we were able to begin identifying the genes we would need to construct our plasmid. We chose to use genes from <I>Chlamydomonas reinhardtii</I>, as considerable research already existed on its chlorophyll biosynthesis pathway. Additionally, we were able to use some of the parts previously assembled by former Macquarie teams.

Our plasmid was divided into three functional portions. The first was a series of genes that formed the magnesium chelatase complex. These genes encode proteins that function to insert a magnesium ion into the porphyrin ring of protoporphyrin, a natural metabolite within <I>E. coli</I>. This would produce Mg-protophorphyrin IX. The genes required for this portion of the plasmid were:

<UL class="normal"><LI>
ChlH
</LI><LI>
GUN4
</LI><LI>
ChlI1
</LI><LI>
ChlID
</LI><LI>
ChlI2
</LI></UL></P><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
Next, we identified a series of genes that would be necessary to convert Mg-protophorphyrin IX into protochlorophyllide. This made up the second portion of our plasmid. These genes were:

<UL class="normal"><LI>
ChlM
</LI><LI>
CTH1
</LI><LI>
Ycf54
</LI></UL></P><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
Following this, we needed a series of genes which would enable the conversion of protochlorophyllide into chlorophyll. This was the third and final portion of our plasmid. These genes were:
<UL class="normal"><LI>
POR
</LI><LI>
ChIP
</LI><LI>
DVR1
</LI><LI>
ChIG
</LI></UL></P><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
In addition to chlorophyll biosynthesis genes, we planned to incorporate the ycf39 gene. This was based on literature reports which indicated this protein may be involved in the formation of prolamellar bodies (Rast, Heinz and Nickelsen, 2015). Similarly, FNR and FDX genes were included based upon reports that they may aid chlorophyll biosynthesis (Herbst et al. 2018).
</P></DIV><DIV style="width: 50%; margin-left: auto; margin-right: auto; display: block;"><H style="font-size: 20px; font-weight: bold;">Modelling our Design

    </H><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
Before starting work in the lab, we looked at the work of previous Macquarie teams and the plasmids they had partially constructed. We computationally modelled chlorophyll production and realised we could improve the efficiency of this process by increasing the expression of several genes. In particular, increasing transcription of the ChlI1, POR, fdx and FNR genes improved the efficiency of chlorophyl production. As such, we decided not to use many of the parts assembled by previous Macquarie teams. We created the second iteration of our design, in which a trc protomoter was placed before the aforementioned genes. Modelling this promoter swap based on gene expression data obtained by Tegel, Ottosson, and Hober (2011), we were confident our improved plasmid would be more efficient at producing chlorophyll.


    </P></DIV><DIV style="width: 50%; margin-left: auto; margin-right: auto; display: block;"><H style="font-size: 20px; font-weight: bold;">Finalising our Assembly Plan

    </H><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
After initial experiments, we found that standard assembly was significantly more efficient than 3A assembly. Based on these preliminary results, we planned out our plasmid design using standard assembly. This meant that we had to perform backbone swaps on some of the parts we were going to use, such as GUN4 <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1080003">(BBa_K1080003)</A>. Despite this additional step, we were confident using standard assembly would ultimately make the construction of our plasmid more efficient. This led to our final plasmid design.


    </P></DIV><DIV style="width: 50%; margin-left: auto; margin-right: auto; display: block;"><H style="font-size: 20px; font-weight: bold;">Testing our Design

    </H><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
With the plasmid designed, it was necessary to consider how we would test if each step worked. In order to confirm our project was successfully progressing, we needed to think of a way to verify that our assembly steps and transformations were working. To do this, we planned to employ both PCR and gene sequencing on plasmids extracted from transformed cell cultures. Where possible, we planned to use gene sequencing, however owing to the large size of some of our final parts (in excess of 18000 bp) it would have been necessary to rely on PCR. Additionally, we planned to perform a basic qualitative assessment of our digestion and ligation steps based on agarose gel electrophoresis. This technique would also be useful in assessing the identity of plasmids within cells.

Similarly, it would be necessary to confirm that our desired proteins were being expressed within our cells. We planned to do this using SDS-PAGE.

Additionally, due to the unique UV-Vis spectra of each of our porphyrin intermediates, we planned to use this technique extensively to confirm that each biosynthetic step was working.

Finally, we planned to use confocal microscopy to optically observe our vesicles. Interestingly, because of the natural fluorescence of the chlorophyll molecules bound to the vesicles, we would not need additional fluorescent stains to visualize the vesicles.


    </P></DIV><DIV style="width: 50%; margin-left: auto; margin-right: auto; display: block;"><H style="font-size: 20px; font-weight: bold;">References

    </H><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
Rast, A., Heinz, S. and Nickelsen, J., 2015. Biogenesis of thylakoid membranes. Biochimica et Biophysica Acta (BBA)-Bioenergetics, 1847(9), pp.821-830.

Herbst, J., Girke, A., Hajirezaei, M.R., Hanke, G. and Grimm, B., 2018. Potential roles of YCF 54 and ferredoxin‐NADPH reductase for magnesium protoporphyrin monomethylester cyclase. The Plant Journal, 94(3), pp.485-496.

Tegel, H., Ottosson, J. and Hober, S., 2011. Enhancing the protein production levels in Escherichia coli with a strong promoter. The FEBS journal, 278(5), pp.729-739.
    </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>