<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Macquarie_Australia_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Macquarie Australia/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin-top: 15px;"><LI class="dropdown"><A class="dropdown-toggle active" style="font-family: 'Raleway', sans-serif; font-size: 12;">Project</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Description">Description</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Design">Design</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Protocols">Protocols</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Results">Results</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle active" style="font-family: 'Raleway', sans-serif;font-size: 12;">Notebook</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Notebook">Wet lab</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Notebookdry">Dry lab</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Parts</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Parts">Summary</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/CompositeParts">Composite parts</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/PartsValidation">Validation</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Interlab</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/InterLab">Interlab Overview</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Human Practices</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Human_Practices">Overview</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Interviews">Interviews</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Public_Engagement">Public Engagement</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/IntergratedHumanPractices">Intergrated Human Practices</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Toolkit">Customer discovery toolkit</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Chlorofella">Chlorofella chronicles</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Team</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Team">Meet The Team</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Attributions">Attributions</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Modelling</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Model">Our Model</A></LI><LI class="dropdown"><A style="font-family: 'Raleway', sans-serif;font-size: 12;" class="dropdown-toggle active">Collaboration</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://2018.igem.org/Team:Macquarie_Australia/Collaborations">Overview</A></LI><LI class="dropdown"><A class="dropdown-toggle active" style="font-family: 'Raleway', sans-serif;font-size: 12;">Judging Form</A></LI><LI><A style="font-family: 'Raleway', sans-serif;font-size: 12;" href="https://igem.org/2018_Judging_Form?id=2664">Judging Form</A></LI></DIV></DIV><DIV style="width: 70%; margin-left: auto; margin-right: auto; display: block;"><DIV id="toc_container"><P class="toc_title">Contents</P><UL class="toc_list"><LI><A href="#Electrophoresis">1. Electrophoresis</A></LI><LI><A href="#Manual Trypsin Digest">2. Manual Trypsin Digest</A></LI><LI><A href="#Buffers and Reagents">3. Buffers and Reagents</A></LI><LI><A href="#Competent Cells">4. Competent Cells</A></LI><LI><A href="#Heat Shock Transformation">5. Heat Shock Transformation</A></LI><LI><A href="#Media and Plates">6. Media and Plates</A></LI><LI><A href="#Plasmid Prep">7. Plasmid Prep</A></LI><LI><A href="#Protein Expression">8. Protein expression</A></LI><LI><A href="#SDS-PAGE">9. SDS-PAGE</A></LI><LI><A href="#PCR">10. PCR</A></LI><LI><A href="#Standard Assembly">11. Standard assembly</A></LI><LI><A href="#Protochlorophyllide Extraction from Barley">12. Protochlorophyllide Extraction from Barley</A></LI></UL></DIV></DIV><DIV style="width: 50%; margin-left: auto; margin-right: auto; display: block;"><H2 id="Electrophoresis">Electrophoresis</H2><H3>Gel Preparation</H3><OL style="line-height:1.5;"><LI>Mix 1g of agarose powder with 100ml of 1x TAE buffer and heat for 1 minute or until all agarose is dissolved.</LI><LI>Wait until it has cooled (not set), and add 1ul of GelRed into the mixture.</LI><LI>Pour the solution into a cast with an appropriate comb.</LI><LI>Leave to set.</LI></OL><H3>Running the Gel</H3><OL><LI> Mix 1ul of 1kbp DNA ladder with 6ul of loading dye (bromophenol blue) and 4ul of 1 x TAE buffer (total 6ul) and load onto first well.</LI><LI> Mix 5ul of PCR products with 1ul of loading dye and load into wells.</LI><LI> Run gel at 90V products with 1ul of loading dye and load onto wells.</LI><LI> Run gel at 90V for 45 minutes approximately.</LI><LI> Photograph gels under UV light.</LI></OL><H2 id="Manual Trypsin Digest">Manual Trypsin Digest</H2><OL><LI> For Coomassie stained gels, de-stain gels by washing briefly with 200uL NH<SUB>4</SUB>HCO<SUB>3</SUB> (Solution A) to make sure gel pieces are at the correct pH. </LI><LI>Add 200uL 50% acetonitrile / 50% 100mM NH<SUB>4</SUB>HCO<SUB>3</SUB> (Solution B) into each well. Vortex to mix and incubate for 10 minutes. Remove liquid. </LI><LI>Repeat step 2. Gel pieces should be clear at this stage. If they are still blue, repeat as necessary until colour is gone. </LI><LI>Wash for 5 min with 50uL of 100% acetonitrile (Solution C) to dehydrate gel pieces. Vortex during incubation. </LI><LI>Remove acetonitrile, then let air-dry for 10 min. The gel pieces should be noticeably shrunken and probably white. </LI><LI>Reduction and Alkylation: Cover gel pieces with 50uL 10mM DTT in 50mM NH<SUB>4</SUB>HCO<SUB>3</SUB> (Solution D). Let proteins be reduced for 45-60 min in a 37<SUP>o</SUP>C incubator. </LI><LI>Remove DTT solution and add 50uL of 55mM iodoacetamide in 50mM NH<SUB>4</SUB>HCO<SUB>3</SUB> (Solution E). Incubate for 45 min in dark place at room temperature. </LI><LI>Remove iodoacetamide, discard. </LI><LI>Wash gel pieces with 200uL of NH<SUB>4</SUB>HCO<SUB>3</SUB> (Solution A) for 5 min with vortexing before adding 100uL of 100% acetonitrile (Solution C). </LI><LI>Remove liquid after 5 min, discard. </LI><LI>Wash gel pieces with 50uL 100mM NH<SUB>4</SUB>HCO<SUB>3</SUB> (Solution A) for 10 min, then twice with 200uL 50% acetonitrile / 50% 100mM NH<SUB>4</SUB>HCO<SUB>3</SUB> (Solution B) for 10min. </LI><LI>Dehydrate with 100 uL 100% acetonitrile (Solution C) for 5 min as above. </LI><LI>Remove remaining liquid and let the gel dry. </LI><LI>Trypsin Digestion: Prepare trypsin mix to final concentration of 13ng/uL in 50mM Ammonium bicarbonate. Add 30uL (or more if required) of mixed trypsin solution to cover the gel pieces. Allow 30 min for gel rehydration at 4<SUP>o</SUP>C (on ice). Digest overnight at 37<SUP>o</SUP>C. </LI></OL><H3>Peptide Extraction</H3><OL><LI>Transfer the digest solution supernatant (if any) into clean 0.65ml Eppendorf tubes.</LI><LI>To the gel pieces, add 50 uL of 50% acetonitrile / 2% formic acid, incubate 30 min. Spin, remove supernatant and combine with initial digest solution supernatant. Please note that total volume may vary depending on the gel sizes.</LI><LI>Vortex the extracted digests, speed vac to reduce volume to 10 uL. If the remaining volume is less than 10uL, use 2% formic acid to bring the volume up to 10 uL. </LI><LI>Spin at 14,000 rpm for 30 min to remove any micro particulates. </LI><LI>Transfer the supernatant to a fresh 0.65ml eppendorf tube for storage at 4<SUP>o</SUP>C fridge OR directly into PCR plate for Mass spec for analysis. </LI></OL><H3>Trypsin Preparation</H3><OL><LI>Trypsin is prepared by adding 200uL of Resuspension Buffer into one vial containing 20ug of Proteomics grade trypsin.</LI><LI>Add 1.2ml of 50mM NH<SUB>4</SUB>HCO<SUB>3</SUB> into each trypsin vial (final concentration of Trypsin is 16ng/uL). Mix thoroughly.</LI><LI>This will provide enough Trypsin for 46 digestions.</LI></OL><H2 id="Buffers and Reagents">Buffers and Reagents</H2><H3>50x TAE Buffer</H3><TABLE><TBODY><TR><TD>Tris Base (MW=121.1)</TD><TD>242.4g</TD></TR><TR><TD>EDTA Disodium Salt</TD><TD>18.61g</TD></TR><TR><TD>Glacial Acetic acid</TD><TD>57.1mL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Dissolved Tris base and EDTA in 700mL distilled water, then added glacial acetic acid. Additional water was added to bring final volume up to 1L. Diluted to 1x for use.</P><H3>50x M (100mL)</H3><TABLE><TBODY><TR><TD>Na<SUB>2</SUB>HPO<SUB>4</SUB></TD><TD>17.75g</TD></TR><TR><TD>KH<SUB>2</SUB>PO<SUB>4</SUB></TD><TD>17g</TD></TR><TR><TD>NH<SUB>4</SUB>Cl</TD><TD>13.4g</TD></TR><TR><TD>Na<SUB>2</SUB>SO<SUB>4</SUB></TD><TD>3.55g</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Made up to 100mL with distilled water. Autoclaved for 15 minutes to sterilise.</P><H3>ZY Media (400mL)</H3><TABLE><TBODY><TR><TD>Tryptone</TD><TD>4g</TD></TR><TR><TD>Yeast Extract</TD><TD>2g</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Made up to 400mL with distilled water. Autoclaved for 15 minutes to sterilise.</P><H3>50x 5052 (100mL)</H3><TABLE><TBODY><TR><TD>Glycerol</TD><TD>25g</TD></TR><TR><TD>Glucose</TD><TD>2.5g</TD></TR><TR><TD>Alpha Lactose</TD><TD>10g</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Dissolved glycerol in 40mL of distilled water. Dissolved glucose and alpha lactose in 40mL of distilled water. Combined two solutions, adjusted final volume to 100mL. Autoclaved for 15 minutes to sterilise.</P><H3>TB Buffer (500mL)</H3><TABLE><TBODY><TR><TD>KCI(MW=74.5513)</TD><TD>9.3g</TD></TR><TR><TD>CaCl<SUB>2</SUB>.2H<SUB>2</SUB>O (MW=147.0146)</TD><TD>1g</TD></TR><TR><TD>PIPES (MW=302.4)</TD><TD>1.5g</TD></TR><TR><TD>MnCl<SUB>2</SUB>.4H<SUB>2</SUB>O</TD><TD>5.44g</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Dissolved KCl, PIPES and CaCl<SUB>2</SUB> in 300mL of Milli-Q water. pH adjusted to 6.7 using KOH. Dissolved 5.44g of MnCl<SUB>2</SUB> in 100mL of Milli-Q water. Gradually added MnCl2 to the other solution. Final volume was adjusted to 500mL, filter sterilised with 0.22um filter, then stored at 4°C </P><H3>HEPES, pH 7.5 (200 mL)</H3><TABLE><TBODY><TR><TD>HEPES</TD><TD>47.66g</TD></TR><TR><TD>KOH</TD><TD>As required</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Dissolved Tris in 75% of total volume of Milli-Q water. Buffered pH to correct levels with HCl, then adjusted final volume to 200mL.</P><H3>Tris-HCl, pH 8.0 (200ml)</H3><TABLE><TBODY><TR><TD>Tris</TD><TD>24.228g</TD></TR><TR><TD>HCl</TD><TD>As required</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Dissolved Tris in 75% of total volume of Milli-Q water. Buffered pH to correct levels with HCl, then adjusted final volume to 200mL.</P><H3>1M Imidazole, pH 8.0 (400ml)</H3><TABLE><TBODY><TR><TD>Imidazole</TD><TD>27.232g</TD></TR><TR><TD>HCl</TD><TD>As required</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Dissolve imidazole in 75% of total volume of Milli-Q water. Buffered pH to correct levels with HCl, then adjusted final volume to 400mL. </P><H3>1M EDTA, pH 8.0 (200ml)</H3><TABLE><TBODY><TR><TD>EDTA</TD><TD>29.22g</TD></TR><TR><TD>NaOH</TD><TD>As required</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Dissolved EDTA in 75% of total volume of Milli-Q water while adding NaOH to assist. Buffered pH to correct levels with NaOH, then adjusted final volume to 200mL.</P><H3>Dithiotheritol 1M (5mL)</H3><TABLE><TBODY><TR><TD>DTT</TD><TD>0.7715g</TD></TR></TBODY></TABLE><P>Dissolved DTT in Milli-Q water.</P><H3>TE Buffer (100ml)</H3><TABLE><TBODY><TR><TD>TRIS (1M) pH 8.0</TD><TD>1mL</TD></TR><TR><TD>EDTA(0.5M) pH 8.0</TD><TD>0.2mL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Combined reagents, adjusted final volume to 100mL with Milli-Q water.</P><H3>Cas9 Buffer (100ml)</H3><TABLE><TBODY><TR><TD>HEPES(1M) pH 7.5</TD><TD>2mL</TD></TR><TR><TD>KCl(1M)</TD><TD>15mL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Combined reagents, adjusted final volume to 100mL with Milli-Q water. </P><H3>CaCl<SUB>2</SUB> 1M (100mL)</H3><TABLE><TBODY><TR><TD>CaCl<SUB>2</SUB></TD><TD>11.098g</TD></TR><TR><TD>KCl(1M)</TD><TD>15mL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Dissolved CaCl2 in 100mL of Milli-Q water. </P><H3>Ammonium Bicarbonate 100mM (400mL)</H3><TABLE><TBODY><TR><TD>Ammonium Bicarbonate</TD><TD>3.164g</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Dissolved ammonium bicarbonate in 400mL of Milli-Q water.</P><H3>Ammonium Bicarbonate 50mM (200mL)</H3><TABLE><TBODY><TR><TD>Ammonium Bicarbonate</TD><TD>0.791g</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Dissolved ammonium bicarbonate in 200mL of Milli-Q water.</P><H3>50% 100mM Ammonium Bicarbonate/50% acetonitrile (200ml)</H3><TABLE><TBODY><TR><TD>Ammonium bicarbonate 100mM</TD><TD>100mL</TD></TR><TR><TD>acetonitrile</TD><TD>100mL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Combined reagents in Schott bottle.</P><H3>Coomassie Destain (1L)</H3><TABLE><TBODY><TR><TD>Acetic Acid</TD><TD>100mL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Mixed with 900mL Milli-Q water</P><H3>Coomassie Fixing Solution (400mL)</H3><TABLE><TBODY><TR><TD>Acetic Acid</TD><TD>40mL</TD></TR><TR><TD>Ethanol</TD><TD>200mL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Combined reagents and adjusted final volume to 400mL with Milli-Q water.</P><H3>Iodoacetamide (DTT)</H3><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Add 15 mL of 100 mM ammonium bicarbonate to 153 mg of iodoacetamide to give a solution of 55 mM iodoacetamide - 15 mL of each is enough for up to 96 samples.</P><H3>Peptide Extraction Solution - 2% formic Acid/ 50% acetonitrile Solution</H3><P>Add 2000uL of formic acid and 50 mL of acetonitrile to 75 mL of ddH2O. 15 mL is enough for up to 96 samples.</P><H2 id="Competent Cells">Competent Cells</H2><OL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI> A sterilized plastic loop was used to gather 10-12 large (2-3mm in diameter) colonies from the plate. Inoculated to 150mL of SOB medium in a 1L flask and grown overnight at 18-22°C, 200-250 rpm.</LI><LI>A600 readings were between 0.2-0.8 when harvested. Cells were in mid log phase with A600 ~ 0.5.</LI><LI>The flask was placed on ice for 10 minutes after being removed from incubator.</LI><LI>Culture was transferred to a 50mL centrifuge tube and spun at 2,500 x g for 10 minutes at 4°C. </LI><LI>Supernatant was discarded and the tube was placed on ice.</LI><LI>Cells were resuspended in 1mL of ice-cold TB buffer, ensuring no clumps of cells remained and kept cold.</LI><LI>Cold TB-buffer was added to bring up volume up to 1/5th of the original culture volume (~30mL) and mixed gently and inverted 3 times.</LI><LI>Tube was incubated on ice for 10 minutes and centrifuged at 2,500 x g for 7 minutes at 4°C and the supernatant was discarded.</LI><LI>Cells were resuspended in ~1/20th of the original culture volume of ice cold TB-buffer.</LI><LI>930uL of cell suspension was added to pre-chilled 1.5mL Eppendorf tubes.</LI><LI>70uL of DMSO was added to the 930uL cell suspension and mixed gently while placed on ice.</LI><LI>100uL of the competent cell/DMSO mixture was added into fresh microcentrifuge tubes, labelled and snapped frozen with liquid nitrogen.</LI><LI>Steps 10-12 was repeated to the remaining for the rest of the suspension from step 9.</LI></FONT></OL><H2 id="Heat Shock Transformation">Heat Shock Transformation</H2><OL><FONT face="Corbel" font="" style="line-height:1.5" font="" size="3" color="#404040"><LI> Obtain competent cells from -80<SUP>o</SUP>C.</LI><LI>Defrost gently on ice. 100uL is sufficient for 2 transformations.</LI><LI>Add 1-10uL of plasmid DNA/ ligation mix to each tube. Incubate on ice for 30 min.</LI><LI>Heat shock in 42<SUP>o</SUP>C water bath for 30 seconds, then back on ice for 2 min.</LI><LI>Add 200uL of SOC media to each tube, and incubate in the 37<SUP>o</SUP>C shaker for 1h.</LI><LI>For each tube of cells, spread 50uL onto one LB plate with appropriate antibiotic, and 100uL onto a second plate, using aseptic technique. Place your plate upside-down in the 37<SUP>o</SUP>C incubator.</LI></FONT></OL><H2 id="Media and Plates">Media and Plates</H2><H3>LB Media (400mL)</H3><TABLE><TBODY><TR><TD>Tryptone</TD><TD>4g</TD></TR><TR><TD>Yeast Extract </TD><TD>2g</TD></TR><TR><TD>NaCl</TD><TD>4g</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Made up to 400mL with Milli-Q Water in a 500mL Schott bottle. Autoclaved for 15 minutes to sterilize.</P><H3>LB Agar (400mL)</H3><TABLE><TBODY><TR><TD>Tryptone</TD><TD>4g</TD></TR><TR><TD>Yeast Extract </TD><TD>2g</TD></TR><TR><TD>NaCl</TD><TD>4g</TD></TR><TR><TD>Bacto Agar</TD><TD>6g</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Made up to 400mL with Milli-Q water in a 500mL in a Schott bottle. Autoclaved for 15 minutes to sterilize. 400uL of antibiotic was added once the bottle was cool enough to handle. Plates were poured using antiseptic technique.</P><H3>SOB Medium (400mL)</H3><TABLE><TBODY><TR><TD>Tryptone</TD><TD>8g</TD></TR><TR><TD>Yeast Extract </TD><TD>2g</TD></TR><TR><TD>5M NaCl</TD><TD>0.8mL</TD></TR><TR><TD>1M KCl</TD><TD>1mL</TD></TR><TR><TD>1M MgCl<SUB>2</SUB></TD><TD>4mL</TD></TR><TR><TD>1M MgSO<SUB>4</SUB></TD><TD>4mL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Made up to 400mL with distilled water. Autoclaved for 15 minutes to sterilize.</P><H3>SOC Medium</H3><TABLE><TBODY><TR><TD>Tryptone</TD><TD>8g</TD></TR><TR><TD>Yeast Extract </TD><TD>2g</TD></TR><TR><TD>5M NaCl</TD><TD>0.8mL</TD></TR><TR><TD>1M KCl</TD><TD>1mL</TD></TR><TR><TD>1M MgCl<SUB>2</SUB></TD><TD>4mL</TD></TR><TR><TD>1M MgSO<SUB>4</SUB></TD><TD>4mL</TD></TR><TR><TD>2M D-Glucose</TD><TD>4mL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Made up to 400mL with Milli-Q water. Autoclaved for 15 minutes to sterilize</P><H3>1% Agarose Gel</H3><TABLE><TBODY><TR><TD>DNA Grade Agarose</TD><TD>1g</TD></TR><TR><TD>1x TAE Buffer</TD><TD>100mL</TD></TR><TR><TD>GelRed</TD><TD>4uL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Mixed components in 250mL flask and microwaved until completely dissolved. Cooled slightly before 4uL of GelRed added.</P><H3>M9 Glycerol Agar</H3><TABLE><TBODY><TR><TD>M9 Salts (5x) </TD><TD>100mL</TD></TR><TR><TD>1M MgSO<SUB>4</SUB></TD><TD>2mL</TD></TR><TR><TD>1M CaCl<SUB>2</SUB></TD><TD>50uL</TD></TR><TR><TD>20% Glycerol </TD><TD>10mL</TD></TR><TR><TD>Milli-Q Water</TD><TD>113mL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                The next reagents were added to the mixture via filter sterilization:</P><TABLE><TBODY><TR><TD>1% Thiamine HCl</TD><TD>15mL</TD></TR><TR><TD>Cas Amino Acids</TD><TD>=10mL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Plates were then prepared using aseptic technique.</P><H3>M9 Glucose Agar</H3><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Agar solution was prepared by autoclaving 7.5g of agar in 250mL of water.
                The following reagents were combined and autoclaved before being aseptically added to the hot agar:</P><TABLE><TBODY><TR><TD>M9 Salts (5x) </TD><TD>100mL</TD></TR><TR><TD>1M MgSO<SUB>4</SUB></TD><TD>2mL</TD></TR><TR><TD>1M CaCl<SUB>2</SUB></TD><TD>50uL</TD></TR><TR><TD>20% Glucose </TD><TD>10mL</TD></TR><TR><TD>Milli-Q Water</TD><TD>113mL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                The next reagents were added to the mixture via filter sterilization:</P><TABLE><TBODY><TR><TD>1% Thiamine HCl</TD><TD>15mL</TD></TR><TR><TD>Cas Amino Acids</TD><TD>10mL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                Plates were then prepared using aseptic technique.</P><H2 id="Plasmid Prep">Plasmid Prep</H2><OL><LI> Centrifuge at 13,200 rpm for 10 min of 1.8mL of overnight cultures in 2mL Eppendorfs. </LI><LI>Discard supernatant and add 1.9 mL of culture and centrifuge again at 13,200 rpm for 10 min. </LI><LI>Re-suspend pelleted cells in P1 Buffer (250uL). </LI><LI>Add 250uL of P2 buffer and invert 4-6 times (turns homogenous blue). </LI><LI>Add 350uL of N3 and mix by inverting (turns colourless). </LI><LI>Centrifuge at 10min 13,000 rpm to obtain a pellet. </LI><LI>Transfer supernatant in QIA Prep Spin Column by pipetting. </LI><LI>Centrifuge 30-60 sec - discard flow through. </LI><LI>Wash QIA Prep Spin Column with 0.5mL of PB. </LI><LI>Centrifuge for 30-60 seconds - discard flow through. </LI><LI>Wash spin column by adding 0.75 mL PE buffer. </LI><LI>Centrifuge for 30-60 seconds - discard flow through and centrifuge for a further 1 min. </LI><LI>Place QIA Prep Column in clean Eppendorf. </LI><LI>Elute DNA by adding 30uL of water, stand for 1 min, centrifuge for 1 min. </LI><LI>Plasmid concentration is measured using NanoDrop 2000 spectrophotometer. </LI></OL><H2 id="Protein Expression">Protein Expression</H2><H3>IPTG Induction</H3><OL><LI>Re-transform restriction-digest screened plasmids according to the heat shock transformation protocol. </LI><LI>Screen plates for transformant colonies and place in 5mL of LB broth with appropriate antibiotic concentration at 37<SUP>o</SUP>C with shaking until OD600¬ is 2.0 – 3.0. </LI><LI>Take whole volume and place in 45mL of LB broth in a sterile conical flask containing appropriate antibiotic concentration. Incubate at 37<SUP>o</SUP>C with shaking until OD600 is 0.5-0.7. </LI><LI>Transfer 1M IPTG in a 1:1000 dilution into flask. Incubate at 15<SUP>o</SUP>C with shaking overnight. </LI><LI>Cell pellets were collected through centrifugation at 12,000 rpm for 5 mins. </LI><LI>Cell pellets were resuspended in re-suspension buffer (Containing: 10% glycerol w/v, 50mM Tricene NaOH pH 8.0, 2mM MgCl2, 1mM DTT) and whole lysate was collected through a French Press for functional assays of operon protein products in lysate. </LI></OL><H3>Auto Induction</H3><OL><LI>Start with glycerol stock or freshly transformed plate.</LI><LI>Inoculate 2ml starter culture with a single colony from plate, grow at 37<SUP>o</SUP>C with vigorous shaking to OD600 ~ 1.</LI><LI>Transfer to a larger culture (50ml of ZYM-5052 medium in 250ml flask) and grow at room temperature (or 18<SUP>o</SUP> C) overnight ~ 200rpm. For small scale, same starter flask can be grown at room temperature for ~24h. Amount of protein expressed peaks at OD600 ~ 8-10.</LI><LI>Spin down 5000 x g for 20 min at 4<SUP>o</SUP>C, discard supernatant.</LI><LI>Resuspend pellet in 1/10th of the original volume.</LI><LI>To run on NuPage Bis-Tris gels, make sample mixture which contains: 20 uL resuspended lysate, 20 uL milli-Q water, 16uL NuPage LDS sample buffer, and 5uL 1M DTT.</LI><LI>Boil for 20 min, then spin down max speed for 20 min. Load 10-20uL of supernatant on gels.</LI><LI>Run gel at 150V for 1.5h, then fix with Coomassie Fixing solution for 10 min, stain with Coomassie Stain Solution for 5 min and destain overnight with Destain Solution.</LI></OL><H2 id="SDS-PAGE">SDS-PAGE</H2><OL><LI> Re-suspend pelleted bacterial cells in 200uL of Milli-Q-water</LI><LI>Transfer 50 uL of suspension into new Eppendorf tubes and combine with 50uL of 2xTruSep sample buffer.</LI><LI>Shear the cells using a Hamilton syringe.</LI><LI>Centrifuge the preparation for 3 minutes at 13,000 rpm.</LI><LI>Load 20 uL of the supernatant into gel.</LI><LI>Conduct electrophoresis at a constant voltage (200V) for 1 hour.</LI><LI>Coomassie Stain for ~30 minutes.</LI></OL><H2 id="PCR">PCR</H2><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                General mastermix recipe:</P><OL><UL>4 uL of 5x phusion buffer</UL><UL>0.4 uL dNTPs</UL><UL>0.6 uL of DMSO</UL><UL>0.2 uL of polymerase</UL><UL>11.8 uL of water</UL></OL><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                To this mixture, add:</P><OL><UL>1 uL of forward primer</UL><UL>1 uL of reverse primer</UL><UL>1 uL of template</UL><UL>Total volume = 20 uL</UL></OL><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                PCR settings:</P><UL><OL>A general program for PCR is as follows:</OL><OL>Initial denaturation at 98<SUP>o</SUP>C for 30 seconds</OL><P>Followed by 30 repeats of:</P><OL><UL>Denaturation – 98<SUP>o</SUP>C for 10 seconds</UL><UL>Annealing – 60<SUP>o</SUP>C for 10 seconds</UL><UL>Extension – 72<SUP>o</SUP>C for 2 minutes</UL><UL>Final extension – 72<SUP>o</SUP>C for 10 minutes</UL></OL></UL><H2 id="Standard Assembly">Standard Assembly</H2><H3>Digestion</H3><P>1. Digest Upstream Part with EcoRI-HF and SpeI.</P><TABLE><TBODY><TR><TD>Upstream Part Plasmid</TD><TD>400ng</TD></TR><TR><TD>EcoRI-HF </TD><TD>0.5uL</TD></TR><TR><TD>SpeI</TD><TD>0.5uL</TD></TR><TR><TD>10 x NEBuffer CutSmart</TD><TD>1uL</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>Make up to 10uL  </TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                2. Digest the Destination Plasmid that contains the Downstream Part. with EcoRI-HF and XbaI: The Destination Plasmid should also have a different antibiotic resistance marker from the plasmid containing the Upstream Part to avoid the need to purify the Upstream Part.</P><TABLE><TBODY><TR><TD>Downstream Part Plasmid</TD><TD>400ng</TD></TR><TR><TD>EcoRI-HF </TD><TD>0.5uL</TD></TR><TR><TD>XbaI</TD><TD>0.5uL</TD></TR><TR><TD>10 x NEBuffer CutSmart</TD><TD>1uL</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>Make up to 10uL  </TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                3. Incubate all 3 restriction digest reactions at 37°C for 80 min, then heat inactivate at 80°C for 20 minutes.</P><H3>Ligation</H3><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                4. Ligate the Upstream and Downstream Parts into the digested Destination Plasmid with a 4:1 insert to vector ratio. The volume of insert required are calculated using.</P><P>V = (((X/Y)*4)*A)/B  </P><TABLE><TBODY><TR><TD>V= Volume of Insert digest required in uL</TD><TD>X= length of Insert (upstream part) in bp</TD></TR><TR><TD>Y= length of Vector (downstream part +backbone) in bp</TD><TD>A= Amount of vector used in ng</TD></TR><TR><TD>B= Concentration of Insert Digest in ng/uL</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>Upstream Part Insert digestion</TD><TD>V = (((X/Y)*4)*A)/B</TD></TR><TR><TD>Downstream Part in Destination Plasmid Vector digestion</TD><TD>20 ng</TD></TR><TR><TD>10 x T4 DNA Ligase Buffer</TD><TD>1uL</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>0.5uL</TD></TR><TR><TD>10 x T4 DNA Ligase Buffer</TD><TD>0.5uL  </TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>Make up to 10uL</TD></TR></TBODY></TABLE><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                *It is important to note that the combined volume of both digests used in the ligation must not exceed 8uL and is preferably &lt;5uL.</P><P style="text-align: justify; font-size: 14px; font-family: 'Quicksand', sans-serif;">
                5. Incubate the ligation mix at 37°C for 90 min, then heat inactivate at 80°C for 20 min.</P><H2 id="Protochlorophyllide Extraction from Barley">Protochlorophyllide Extraction from Barley</H2><P>Etiolated barley was prepared by growing the seeds for 8-9 days after germination in the dark. </P><OL><LI> The stalks of the Yellow sprouts were collected and macerated with liquid nitrogen and soaked in ~600mL acetone for 10 minutes.</LI><LI>The sample was then filtered using filter paper to remove the plant solids and washed with acetone several times or until no more colored pigment is extracted, followed by a final wash with ~400mL  of diethyl ether. </LI><LI>The solution was then transferred to a separatory funnel and shaken vigorously.</LI><LI>The lower now colourless aqueous phase was drained and discarded whilst the organic phase was washed several times with roughly 30°C water to remove traces of acetone. 
                </LI><LI>Redistilled light petroleum (b.p.40-60°C) was then added to the ether followed by approx. 10% the volume of the organic layer of methanol/ 0.01 M-NH3 (4:1v/v). </LI><LI>The solution was left to evaporate until it separated into clearly defined phases. </LI><LI>The two phases were separated and the presence of pigments in only the bottom methanol phase was confirmed by fluorescence of solution under uv light. </LI><LI>The sample was then collected and stored in methanol at -20°C.</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>