<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TJU_China_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TJU China/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="row"><DIV class="navbar-header" style="
       margin-top: 4px;
"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="https://2017.igem.org/Team:TJU_China" style="padding: 0"><DIV class="text-center" style="float: right;padding-top: 7px;padding-bottom: 7px;">TJUCHINA</DIV></A></DIV><DIV class="col-sm-12 collapse navbar-collapse navbar-right navbar-ex1-collapse"><UL class="nav navbar-nav"><LI class="menuItem"><A href="https://2017.igem.org/Team:TJU_China">HOME</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                        PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Demonstrate">Demonstrate</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                        PARTS</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Improve">Improve</A></LI><LI class="menuItem"><A href="https://2017.igem.org/Team:TJU_China/Model">MODEL</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                        HUMAN PRACTICE</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/HP">HP</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/HP/Gold_Integrated">Integrated and Gold </A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Engagement">Engagement</A></LI><LI class="menuItem"><A href="https://2017.igem.org/Team:TJU_China/Safety">SAFETY</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                        TEAM</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:TJU_China/Attributions">Attributions</A></LI></DIV></DIV></DIV><DIV class="container"><DIV class="row text-center"><DIV class="col-md-12"><H2>Experiments</H2></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3 class="h3">Molecular Cloning</H3><H3>Take smURFP standardization for instance</H3><H4>Transfer </H4><DIV class="mycontent">
                1. Mix the smURFP Plasmid (pET 21-b, Amp+, 1μL 100ng/μL) with the E.coli DH5α competent cells in the EP tubes. Incubate them on ice for 20-30 minutes. * Plasmid pET21-b plasmid containing the Amp-resistant gene of smURFP was selected.
                2. Heat shock the cells by immersion in a pre-heated water bath at 42ºC for 90 seconds without shake. Transport the tubes in ice for 3-5 minutes.
                3. Add 550 μL of LB media to each transformation. Incubate the cells at 37ºC, 220RPM for 30 minutes while the tubes are shaking to express the Amp resistant gene.
                4. Amp-resistant plates were preheated in a 37℃ incubator.
                5. Put the tube centrifugal 2 min, 3000 RPM. Remove 500μl the supernatant.
                6. Plate 100 µl of the transformation onto the dishes with LB agar (AMP), and spread.
                7. Resuspend E. coli pellets. Incubate the plates at 37ºC for 12-25 hours, making sure the agar side of the plate is up.
            </DIV><H4>Pick Monoclonal colonies and transferred to ampicillin-resistant LB medium for subculture  </H4><DIV class="mycontent">In the clean bench, a single colony was picked up from the medium with a pipette tip, and introduced into a test tube containing 5 mL of LB broth and 5 μL Amp, covered with a stopper, shaken at 220 rpm at 37 ° C Bed culture 12 ~ 16h. </DIV><H4>Extract the amplified plasmid from the cultured Escherichia coli </H4><DIV class="mycontent">
               1. Add the E. coli solution into the EP tube
                2. Re-suspend pelleted bacterial cells in Buffer P1 BL (kept at 4 °C) and transfer to a micro-centrifuge tube.
                3. Add Buffer P2 and gently invert the tube 4–6 times to mix.
                4. Add 350μl Buffer N3 and invert the tube immediately and gently 4–6 times.
                5. Centrifuge for 10 min at 12,000 rpm in a micro-centrifuge.
                6. Regenerate column CP3 while centrifugation. Add 500μl Buffer BL. Centrifuge for 1 min at 12,000 rpm after static for 2min. discard the flow-through.
                7. Add supernatant from the EP tube to the column and put it into collection canals. Add 500μl Buffer BL and centrifuging for 1min at 12000rpm. Discard the flow-through. Spin for 60 seconds produces good results.
                8. Adding 600μl Buffer PW and centrifuging for 60s after static for 2min. Discard the flow-through.
                9. Repeat step 8.
                10. Centrifuging for 2min at 12000rpm to shake off the rest of the Buffer PW.
                11. Place the column in a new EP tube and the opening was allowed to stand for 5 minutes, so that the ethanol in the PW can be sufficiently volatilized.
                12. Add 50μL 75℃ of sterile distilled water at 75 ° C drop wise to the middle of the adsorbed film. Static for 2min.
                Discard the flow-through. Centrifuging for 2min at 12000rpm.
            </DIV><H4>Examine the concentration of the extracted plasmid </H4><DIV class="mycontent">Add 1 drop of sterile distilled water at the light hole of nano Drop and erase it after 90s for two times. Add 1 drop of the solution in EP tube at the light hole of nano Drop with pipette and erase it after measure.</DIV><H4>PCR</H4><DIV class="mycontent">We use overlap PCR to get all the fusion protein. To splice two DNA molecules, special primers are used at the ends that are to be joined. In first cycle we use a couple of primers of and the same to B to PCR. By this, we design a same length of base sequence. In second cycle, due to the homologous sequence, when adding the beginning primers and the final primers, A and B can be primers to each other. They extend to a combined fragment. </DIV><H4>Agarose Gel Electrophoresis(Plasmid &amp; PCR product) ： </H4><DIV class="mycontent">1. Prepare sufficient lx TAE to fill the electrophoresis tank and to cast the gel.
                2. Prepare a solution of agarose in electrophoresis buffer at a concentration of 1%: Add 0.9g powdered agarose to 90ml of TAE in an Erlenmeyer flask.
                3. Heat the slurry in a boiling-water bath or a microwave oven until the agarose dissolves.
                4. Use insulated gloves or tongs to transfer the flask/bottle into a water bath at 55°C. When the molten gel has cooled, add ethidium bromide to a final concentration of 0.5 μg/ml. Mix the gel solution thoroughly by gentle swirling.
                5. While the agarose solution is cooling, choose an appropriate comb for forming the sample slots in the gel. Position the comb 0.5-1.0 mm above the plate so that a complete well is formed when the agarose is added to the mold.
                6. Pour the warm agarose solution into the mold.
                7. Allow the gel to set completely (30-45 minutes at room temperature), then carefully remove the comb. Pour off the electrophoresis buffer and carefully remove the tape Mount the gel in the electrophoresis tank.
                8. Add just enough electrophoresis buffer to cover the gel to a depth of ~1 mm.
                9. Mix the samples of DNA with 10 μl green buffer
                10. Slowly load the sample mixture into the slots of the submerged gel using a disposable micropipette, an automatic micropipettor. Load size standards into slots on both the right and left sides of the gel.
                11. Close the lid of the gel tank and attach the electrical leads so that the DNA will migrate toward the positive anode (red lead). Apply a voltage of 1-5 V/cm (measured as the distance between the positive and negative electrodes). If the leads have been attached correctly, bubbles should be generated at the anode and cathode (due to electrolysis), and within a few minutes, the bromophenol blue should migrate from the wells into the body of the gel. Run the gel until the bromophenol blue and xylene cyanol FF have migrated an appropriate distance through the gel.
                12. When the DNA samples or dyes have migrated a sufficient distance through the gel, turn off the electric current and remove the leads and lid from the gel tank.
            </DIV><H4>Plastic recycling </H4><DIV class="mycontent">1. Column regeneration: add 500μl balance liquid to adsorption column CA2 12000 rpm centrifugal 1 minutes, pour out the filtrate and put it back to the collecting canals.
                2. Cut the DNA band from the gal, weighing in the clean centrifuge tube and weight.
                3. Add equal volume solution PN (such as 0.1 g gal, to join 100μl PN) in rubber block, then put in 50 ℃ water bath place, turning up and down continuously in order to ensure fully dissolved. Cooling to ambient temperature.
                4. Add the solution into an adsorption column CA2 under the ambient temperature for 2 minutes. 12000 rpm centrifugal 30-60 seconds, pour out the filtrate, placed at ambient temperature for a few minutes to ewnsure fully getting rid of the alcohol. Put the adsorption column CA2 into the collection canals.
                5. Drift lotion to column to join 600 μl PW, 12000 rpm centrifugal 1 minutes, pour out the filtrate.
                6. Repeat step 5.
                7. 12000 rpm centrifugal for 2 minutes, pour out the filtrate, placed at ambient temperature for a few minutes to ensure fully getting rid of the filtrate.
                8. Put CA2 in clean centrifuge tube, add ddH2O and wait for 10 minutes at ambient temperature, 12000 RPM, centrifugal 2 min. collect the DNA solution.
                Saved the DNA concentration at -20 degrees Celsius to prevent DNA cleavage.
            </DIV><H4>Digestion of PCR products</H4><DIV class="mycontent">Add 1μL DMT enzyme, PCR products, buffer and water to 20 µL, mix, 37 ℃ water bath heating 1h.</DIV><H4>Ligation</H4><DIV class="mycontent">Mix 1 µL T4 DNA Ligase, 2 µL ligase buffer, 50ng Vector Plasmid, Insert DNA (molar amount is 5 more than plasmid.) and sterile water (Make reaction up to 20 µL). Then incubate at25℃ for 30 - 60 min or at 16℃ for 60min.</DIV><H4>Electro-transformation (B. longum)</H4><DIV class="mycontent">1. Incubate bacteria anaerobically in MRSS broth (+0.5M sucrose) over night.
                2. Transfer culture to 50ml MRSS broth and incubated at 37°C until the culture reached OD600 of 0.5 to 0.7.
                3. The cells washed third times in ice-cold sucrose-citrate buffer (0.5 M sucrose, 1 mM ammonium citrate [pH 6.0]).
                4. Gather the cells in 200 μl of the same buffer.
                5. Electroporation was performed at 25 μF, 200 Ω, 12.5 kV (0.2mm) and 4ms for 100μl cells and 0.6ng plasmid.
                6. The cells were immediately diluted in 900 μl of MRS broth and incubated for 2.5 h-3h.
                7. Spread the cells onto Amp+ MRS agar plate.
                8. Plates were incubated for 2 to 3 days at 37°C under anaerobic conditions.
            </DIV><H3 class="h3">Structure Analysis of Protein</H3><H4>A large number of cultured protein expression bacteria </H4><DIV class="mycontent">1. Extract 5μL of the bacterial strain from the bacterial strain retained in the test expression, and add 5μL Amp to a test tube containing 5 mL of LB culture solution.
                2. Incubate the plates at 37 ° C overnight at 220 rpm in a shaker
                3. Pour the bacteria in the test tube into a 2L Erlenmeyer flask containing 1 L of LB medium, and add 500μL Amp.
                4. 37 ℃, 220rpm in the shaker culture 4 ~ 5h, on behalf of bacteria growth to logarithmic phase (OD value 0.6-0.8) 16 ℃, 220rpm temperature drop 1h (temperature dropped to 16 degrees), and then add 500μL IPTG induced E. coli (4) Producing the protein of interest.
                5. 16 ℃, 220rpm overnight culture 12h ~ 15h.
                6. Collection: The bacteria were centrifuged at 4000rpm for 16min at 4000rpm and the supernatant was discarded. The precipitated Escherichia coli was transferred to a 50mL centrifuge tube and stored at -30 ℃ (total of 6 vials)
            </DIV><H4>Purification of SmURFP </H4><DIV class="mycontent">1. Disrupt the bacteria by high-pressure sterilizers, and the broken bacteria were ultrasonographed for 15 minutes (4S ultrasound, 6S interval) to remove the nucleic acids carried by the protein of interest. Followed by equilibration in a high speed centrifuge tube, 18000 rpm for 20 minutes, separating the supernatant containing the protein and the precipitate containing the cellular debris.
                2. Nickel column：The supernatant was applied to the His-Accept nickel column. After washing unbound proteins with the lysis buffer (50mM Tris-HCl, pH 7.5, 300mM NaCl, 20mM imidazole), the bound proteins were eluted with elution buffer (50mM Tris-HCl, pH 7.5, 300mM NaCl, 250mM imidazole)
                3. A / B Formulation A: pH 5.0 50mM Sodium Acetate 50mM NaCl B: pH 5.0 50mM Sodium Acetate 1 M NaCl
                4. Cation exchange column: The sample eluted from the nickel column was added to a 10kd concentrating tube. The solution was concentrated at 3500 rpm to a solution of cation exchange column A, diluted ten times with A solution to 5mL. And then separated and purified by Hitrap SP HP 5mL column to obtain a peak shape. According to the shape of the peak were collected in the collection tube sample run glue.
                5. Molecular Sieve: Samples obtained from the S-column were changed (75% solution A + 15% solution B) and concentrated to 500μL. Using AKTA system, the superdex75 molecular sieves with appropriate separation range but higher resolution were used to purify the target protein. Therefore, the corresponding protein samples were collected. Concentrated and re-separated through molecular sieves, respectively. Respectively,
                6. Crystallization: Take the corresponding position of the sample, measured with NanoDrop its concentration. According to the residual amount of protein, we used the conditions to do local screening of samples.
            </DIV><H3 class="h3">Standard Curves of BV, smURFP and RFU</H3><DIV class="mycontent">1. Get the pure smURFP as shown above and dissolve in protein solution.
                2. Dissolve BV in DMSO for stock, keep in the dark place.
                3. Add different amount of protein solution and BV stock solution in black 96-well plate in the dark.
                4. Use protein solution for the rest volume (up to 50μl).
                5. Incubate for 30 min.
                6. Use micro plate reader for RFU.
                7. Use Origin for standard curves.
            </DIV><H3 class="h3">Protein Expression in Host Bacteria</H3><H3>Take E.coli BL21 with plasmid pET28b for instance</H3><H4>Western blot</H4><H4>【Materials】 </H4><H4 class="h4">⒈ Apparatus: </H4><DIV class="mycontent">Apparatus of SDS-PAGE, Electroblotting Apparatus, Power supply, PVDF membrane （Millipore Immobion-P #IPVH 000 10）, Whatman 3MM paper, Additional Tools: Forceps, sponge pad, scissor, gloves, small plastic or glass container, Shallow tray. </DIV><H4 class="h4">⒉ Reagents: </H4><DIV class="mycontent">⑴ 10x buffer (660mL): Add 484.8g Tris , 2307.2g Glycine in ddH2O to final volume 660mL
                ⑵ 4x transfer buffer (5280mL):660mL 10x buffer, 3.3mL 20%SDS, 4620mL ddH2O
                ⑶ 1x transfer buffer (1 L): Add 200 ml Methanol, 250 ml 4x transfer buffer in ddH2O to final volume 1L
                ⑷ PBS buffer: Add 8g NaCl (137mM), 0.2g KCl (2.7mM), 1.44g Na2HPO4 (10mM), 0.24g KH2PO4 (2mM) to 1L ddH2O and adjust pH to 7.4 with HCl.
                ⑸ PBST buffer: 1L TBS buffer add 0.5ml Tween 20 (0.05%).
                ⑹ First antibody
                ⑺ Second antibody
                ⑻ 5% Blocking buffer (10ml): Add 0.5g Non-fat milk powde in PBST buffer to final volume 10ml, keep at 4°C to prevent bacterial contamination.
                ⑼ Developing reagent: Volume ratio: A reagent:B reagent=1:1.
                ⑽ Staining buffer: Add 1g amido black 18B (0.1%), 250ml isopropanol (25%) and 100 ml acetic acid (10%) to distilled water with final volume 1L.
                ⑾ Destaining buffer: Add 350ml isopropanol (35%) and 2 ml acetic acid(2%) to distilled water with final volume 1L.
            </DIV><H4>【Procedure】  </H4><H4 class="h4">⒈ Separation of Protein </H4><DIV class="mycontent">Apparatus of SDS-PAGE, Electroblotting Apparatus, Power supply, PVDF membrane （Millipore Immobion-P #IPVH 000 10）, Whatman 3MM paper, Additional Tools: Forceps, sponge pad, scissor, gloves, small plastic or glass container, Shallow tray. </DIV><H4 class="h4">⒉ Electrotransfer </H4><DIV class="mycontent">⑴ Preparation of membrane
                Cut a piece of PVDF membrane (Millipore Immobion-P #IPVH 000 10) according to the size of gel. Incubate in methanol for about 1 min on a rocker at room temp. Remove methanol and equilibrate membrane in 1x transfer buffer until ready to use.
                ⑵ Arrange gel-membrane sandwich
                In a shallow tray, open the transfer cassette. Put a well-soaked sponge pad on the black piece of the transfer cassette and a wetted 3MM paper on the sponge pad. Place the gel on the paper and arrange well so that all air bubbles are removed. Lay the PVDF membrane on the top of gel and remove any air bubbles. Place a wetted sheet of 3MM paper over the PVDF membrane and remove the bubble. Covered with the second well-soaked pad. Close the sandwich with the white piece of the cassette. Mount the sandwich in the transfer tank; put the black sides near the black side of the device. Fill the buffer tank with the transfer buffer.

                ⑶ Electrotransfer:
                Attach the electrodes. Set the power supply to 100V (constant voltage) for 1h at 4° C.
            </DIV><H4 class="h4">⒊ Immunodetection </H4><DIV class="mycontent">⑴ Membrane staining
                Disconnect transfer apparatus, remove transfer cassette, and peel 3MM paper from membrane. Remove the membrane to a small container. Add 10 ml PBST buffer and wash for short time. Cut out one stripe with 5mm width and put in another clean container. Stain this stripe in staining buffer for 1 min. Destain for 30 min in destaining buffer to check whether protein has been transferred from gel to membrane or not.
                ⑵ Membrane blocking and washing
                For other part of membrane, pour off PBST buffer. Add 5% blocking buffer，rock gently for at least 1 h. Pour off 5% blocking buffer and rinse briefly with PBST buffer three times, 5 minutes for per time.
            ⑶ First antibody
                Pour off PBST buffer. Add first antibody at appropriate dilution in 5 ml 0.5% blocking buffer. Rock gently for at least 1 h; pour off first antibody solution from membrane and wash twice for 10 minutes with PBST buffer.
                ⑷ Second antibody
                Pour off PBST buffer. Add second antibody at appropriate dilution in 5 ml 0.5% blocking buffer. Rock gently for 30min, pour off second antibody solution from membrane and wash twice for 10 minutes with TTBS buffer.
                ⑸ Detection
                Pour off PBST buffer from membrane and add developing reagent.Image with an imaging equipment.
            </DIV><H4>Small plate reader for fluorescence</H4><DIV class="mycontent">The procedure is similar with Standard curve. Protein can be replaced by the engineered bacteria. </DIV><H4>Immunofluorescence detection </H4><DIV class="mycontent">1. Take 200uL induced culture medium at 4℃, 12000rpm, centrifugation 1min. Retain the bacteria.
                2. Resuspend the recombinant yeast cells in PBS buffer, centrifuged at 12000rpm for 1 min at 4℃, and the supernatant was discarded. Retain the bacteria.
                3. The cells were resuspended in 200μL of 3mg/mL BSA in PBS buffer for 1h at room temperature. The cells were mixed up and down at 4℃, 12000rpm and centrifuged for 1min. Retain the bacteria.
                4. The cells were resuspended in 200uL of PBS buffer containing 3 mg / mL BSA containing 1μL of primary antibody.
                Treat the cells overnight at 4℃, and the cells were allowed to mix up and down. The next day at 4 ℃, 12000rpm, centrifugation 1min. Retain the bacteria.
                5. Recombinant yeast cells were resuspended in PBS buffer, treated at room temperature for 10 min, centrifuged at 12000rpm for 1 min at 4 ℃. Retain the bacteria.
                6. Repeat step 5 twice.
                7. The cells were resuspended in 200uL of PBS buffer containing 1μL of secondary antibody and incubated for 1 h at room temperature. The cells were mixed up and down at 4℃ and 12000 rpm for 1 min. Retain the bacteria.
                8. The recombinant yeast cells were resuspended in PBS buffer for 5 min at room temperature. The cells were centrifuged at 12000rpm for 1 min at 4 ℃. The supernatant was discarded and retain the bacteria.
                9. Repeat step 8.
                10. Resuspended in 200uL of PBS buffer, each taking 10uL on clean static slide production.
            </DIV><H4>Fluorescent examination in vivo</H4><DIV class="mycontent">1. Incubate engineered bacteria overnight and induce them as shown above.
                2. Gather the cells and wish twice with PBS buffer. Resuspend them in 100 μL with 10^8 and 10^11 CFU, respectively.
                3. Get the experimental nude mice before the test.
                4. Use intraperitioneal injection to hocus the mice with 10% chloral hydrate.
                5. Use lavage needle to feed the mice with bacteria suspense.
                6. Keep photographing the mice in In-vivo Imaging System every 30 min for 6 hours.
            </DIV></DIV></DIV></DIV><FOOTER><DIV class="container"><DIV class="text-center">© iGEM 2017 TJU_China. All Rights Reserved. |<A href="mailto:tjusls2017igem@163.com">Contact us</A>| （Designed by ZhaoYinJie and PanShi）</DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>