<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU-Eindhoven_logbook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU-Eindhoven/logbook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>var logJSONObj = {
</P><PRE>   &quot;Monday June 11, 2018&quot;:{
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of methacrylated dextran (iG03).</P>&quot;
<PRE>     }
   },
   &quot;Wednesday June 13, 2018&quot;:{
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The reaction of Dex-MA was quenched and the dialysis was started.</P>&quot;
<PRE>     }
   },
   &quot;Thursday June 14, 2018&quot;:{
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Changing the water of the dialysis each 2/3 hours.</P>&quot;
<PRE>     }
   },
   &quot;Friday June 15, 2018&quot;:{
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Freezedrying of the Dex-MA started.</P>&quot;
<PRE>     }
   },
   &quot;Monday June 18, 2018&quot;:{
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>NMR performed to determine the Degree of Substitution.</P>&quot;
<PRE>     }
   },
   &quot;Tuesday June 26, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Making_LB_medium:"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Making LB medium:</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_Pre-Cultures_for_binding_experiments"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Making Pre-Cultures for binding experiments</SPAN></A></LI><LI class="toclevel-1"><A href="#IPTG_stock_preparation"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">IPTG stock preparation</SPAN></A></LI><LI class="toclevel-1"><A href="#Diluting_bacteria_for_induction"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Diluting bacteria for induction</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_pellets_of_the_cells"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Making pellets of the cells</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_page_gel_examination"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">SDS page gel examination</SPAN></A></LI><LI class="toclevel-1"><A href="#Nickel_column"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Nickel column</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_gel_electrophoresis"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">SDS gel electrophoresis</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_agar_plates"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Making agar plates</SPAN></A></LI><LI class="toclevel-1"><A href="#Preparing_cultures_for_FACS_experiments_tomorrow"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Preparing cultures for FACS experiments tomorrow</SPAN></A></LI><LI class="toclevel-1"><A href="#FACS_preparations"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">FACS preparations</SPAN></A></LI><LI class="toclevel-1"><A href="#Preparing_2YT_stock_solution"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Preparing 2YT stock solution</SPAN></A></LI><LI class="toclevel-1"><A href="#Started_cultures_from_frozen_stock"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Started cultures from frozen stock</SPAN></A></LI><LI class="toclevel-1"><A href="#Frozen_permanent_made_.28glycrol_stock.29"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Frozen permanent made (glycrol stock)</SPAN></A></LI><LI class="toclevel-1"><A href="#SACS_experiment"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">SACS experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#FACS_device"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">FACS device</SPAN></A></LI><LI class="toclevel-1"><A href="#Preparing_2YT_stock_solution_2"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">Preparing 2YT stock solution</SPAN></A></LI><LI class="toclevel-1"><A href="#Kanamycin_stock_solution_is_prepared"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">Kanamycin stock solution is prepared</SPAN></A></LI><LI class="toclevel-1"><A href="#LB_cultures_of_R18_and_R15_are_prepared_from_colony_plates"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">LB cultures of R18 and R15 are prepared from colony plates</SPAN></A></LI><LI class="toclevel-1"><A href="#Make_new_plates_of_R18_and_R15"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">Make new plates of R18 and R15</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_agar_plates_2"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">Making agar plates</SPAN></A></LI><LI class="toclevel-1"><A href="#Experimenting_with_dextran_hydrogel_and_bacteria"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">Experimenting with dextran hydrogel and bacteria</SPAN></A></LI><LI class="toclevel-1"><A href="#Experimenting_with_dextran_hydrogel_and_bacteria_2"><SPAN class="tocnumber">23</SPAN><SPAN class="toctext">Experimenting with dextran hydrogel and bacteria</SPAN></A></LI><LI class="toclevel-1"><A href="#Plating_the_gel_medium"><SPAN class="tocnumber">24</SPAN><SPAN class="toctext">Plating the gel medium</SPAN></A></LI><LI class="toclevel-1"><A href="#Analyzing_the_data_from_yesterday"><SPAN class="tocnumber">25</SPAN><SPAN class="toctext">Analyzing the data from yesterday</SPAN></A></LI><LI class="toclevel-1"><A href="#LB_cultures_of_R18_and_R15_.2B_WT_are_prepared_from_colony_plates"><SPAN class="tocnumber">26</SPAN><SPAN class="toctext">LB cultures of R18 and R15 + WT are prepared from colony plates</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR"><SPAN class="tocnumber">27</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Prepare_LB_broth_and_make_agar-LB_plates"><SPAN class="tocnumber">28</SPAN><SPAN class="toctext">Prepare LB broth and make agar-LB plates</SPAN></A></LI><LI class="toclevel-1"><A href="#Incubate_dextran_hydrogels_and_induce_cultures"><SPAN class="tocnumber">29</SPAN><SPAN class="toctext">Incubate dextran hydrogels and induce cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_analysis"><SPAN class="tocnumber">30</SPAN><SPAN class="toctext">PCR analysis</SPAN></A></LI><LI class="toclevel-1"><A href="#Experimenting_with_dextran_hydrogel_and_bacteria_3"><SPAN class="tocnumber">31</SPAN><SPAN class="toctext">Experimenting with dextran hydrogel and bacteria</SPAN></A></LI><LI class="toclevel-1"><A href="#New_culture_plates"><SPAN class="tocnumber">32</SPAN><SPAN class="toctext">New culture plates</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_of_CL3"><SPAN class="tocnumber">33</SPAN><SPAN class="toctext">PCR of CL3</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_purification"><SPAN class="tocnumber">34</SPAN><SPAN class="toctext">PCR purification</SPAN></A></LI><LI class="toclevel-1"><A href="#Experimenting_with_dextran_hydrogel_and_bacteria_4"><SPAN class="tocnumber">35</SPAN><SPAN class="toctext">Experimenting with dextran hydrogel and bacteria</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_of_CL3_2"><SPAN class="tocnumber">36</SPAN><SPAN class="toctext">PCR of CL3</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_of_vector_TetO_and_insert_HlyB.2FD"><SPAN class="tocnumber">37</SPAN><SPAN class="toctext">PCR of vector TetO and insert HlyB/D</SPAN></A></LI><LI class="toclevel-1"><A href="#LB_cultures_of_R18_and_R15_.2B_WT_were_prepared_from_colony_plates"><SPAN class="tocnumber">38</SPAN><SPAN class="toctext">LB cultures of R18 and R15 + WT were prepared from colony plates</SPAN></A></LI><LI class="toclevel-1"><A href="#SEM"><SPAN class="tocnumber">39</SPAN><SPAN class="toctext">SEM</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_agar_plates_3"><SPAN class="tocnumber">40</SPAN><SPAN class="toctext">Making agar plates</SPAN></A></LI><LI class="toclevel-1"><A href="#Repeat_of_PCR_of_HlyB.2FD"><SPAN class="tocnumber">41</SPAN><SPAN class="toctext">Repeat of PCR of HlyB/D</SPAN></A></LI><LI class="toclevel-1"><A href="#Culturing_of_pBAD_vector_in_Nova_Blue_cells"><SPAN class="tocnumber">42</SPAN><SPAN class="toctext">Culturing of pBAD vector in Nova Blue cells</SPAN></A></LI><LI class="toclevel-1"><A href="#Plasmid_purification_of_pBAD"><SPAN class="tocnumber">43</SPAN><SPAN class="toctext">Plasmid purification of pBAD</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_of_purified_DNA_from_pBAD"><SPAN class="tocnumber">44</SPAN><SPAN class="toctext">PCR of purified DNA from pBAD</SPAN></A></LI><LI class="toclevel-1"><A href="#Prepare_LB_broth_and_make_agar-LB_plates_2"><SPAN class="tocnumber">45</SPAN><SPAN class="toctext">Prepare LB broth and make agar-LB plates</SPAN></A></LI><LI class="toclevel-1"><A href="#LB_cultures_of_R18_and_R15_.2B_WT_were_prepared_from_colony_plates_2"><SPAN class="tocnumber">46</SPAN><SPAN class="toctext">LB cultures of R18 and R15 + WT were prepared from colony plates</SPAN></A></LI><LI class="toclevel-1"><A href="#SEM_2"><SPAN class="tocnumber">47</SPAN><SPAN class="toctext">SEM</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_analysis_of_pBAD"><SPAN class="tocnumber">48</SPAN><SPAN class="toctext">PCR analysis of pBAD</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_purification_2"><SPAN class="tocnumber">49</SPAN><SPAN class="toctext">PCR purification</SPAN></A></LI><LI class="toclevel-1"><A href="#DpnI_digestion.2C_Gibson_Assembly_and_Transformation"><SPAN class="tocnumber">50</SPAN><SPAN class="toctext">DpnI digestion, Gibson Assembly and Transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#Incubate_dextran_hydrogels_and_induce_cultures_2"><SPAN class="tocnumber">51</SPAN><SPAN class="toctext">Incubate dextran hydrogels and induce cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_picking_of_pBAD-Lysostaphin-HlyA-His."><SPAN class="tocnumber">52</SPAN><SPAN class="toctext">Colony picking of pBAD-Lysostaphin-HlyA-His.</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_PCR_of_pBAD-Lysostaphin-HlyA."><SPAN class="tocnumber">53</SPAN><SPAN class="toctext">Colony PCR of pBAD-Lysostaphin-HlyA.</SPAN></A></LI><LI class="toclevel-1"><A href="#Plating."><SPAN class="tocnumber">54</SPAN><SPAN class="toctext">Plating.</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_new_Tris_buffer"><SPAN class="tocnumber">55</SPAN><SPAN class="toctext">Making new Tris buffer</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_washing_experiment"><SPAN class="tocnumber">56</SPAN><SPAN class="toctext">Gel washing experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Analysis.2C_purification_and_storage_of_the_pBAD-Lysostaphin-HlyA-His_constructs."><SPAN class="tocnumber">57</SPAN><SPAN class="toctext">Analysis, purification and storage of the pBAD-Lysostaphin-HlyA-His constructs.</SPAN></A></LI><LI class="toclevel-1"><A href="#LB_cultures_of_R18_and_R15_.2B_WT_were_prepared_from_colony_plates_3"><SPAN class="tocnumber">58</SPAN><SPAN class="toctext">LB cultures of R18 and R15 + WT were prepared from colony plates</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_picking_and_PCR_of_pTetO_.2B_HlyB.2FD"><SPAN class="tocnumber">59</SPAN><SPAN class="toctext">Colony picking and PCR of pTetO + HlyB/D</SPAN></A></LI><LI class="toclevel-1"><A href="#Incubate_dextran_hydrogels_and_induce_cultures_3"><SPAN class="tocnumber">60</SPAN><SPAN class="toctext">Incubate dextran hydrogels and induce cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_analysis_and_colony_PCR"><SPAN class="tocnumber">61</SPAN><SPAN class="toctext">PCR analysis and colony PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_washing_experiment_2"><SPAN class="tocnumber">62</SPAN><SPAN class="toctext">Gel washing experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Kanamycin_stock_solution_is_prepared_2"><SPAN class="tocnumber">63</SPAN><SPAN class="toctext">Kanamycin stock solution is prepared</SPAN></A></LI><LI class="toclevel-1"><A href="#Analysis_of_colony_PCR"><SPAN class="tocnumber">64</SPAN><SPAN class="toctext">Analysis of colony PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Transformation_of_pBAD-Lysostaphin_into_BL21_.28DE3.29_cells"><SPAN class="tocnumber">65</SPAN><SPAN class="toctext">Transformation of pBAD-Lysostaphin into BL21 (DE3) cells</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_picking"><SPAN class="tocnumber">66</SPAN><SPAN class="toctext">Colony picking</SPAN></A></LI><LI class="toclevel-1"><A href="#Experimenting_with_dextran_hydrogel_and_bacteria_5"><SPAN class="tocnumber">67</SPAN><SPAN class="toctext">Experimenting with dextran hydrogel and bacteria</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_PCR_and_PCR_analysis."><SPAN class="tocnumber">68</SPAN><SPAN class="toctext">Colony PCR and PCR analysis.</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_PCR_and_analysis"><SPAN class="tocnumber">69</SPAN><SPAN class="toctext">Colony PCR and analysis</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_cultures_for_SEM_and_following_experiments"><SPAN class="tocnumber">70</SPAN><SPAN class="toctext">Making cultures for SEM and following experiments</SPAN></A></LI><LI class="toclevel-1"><A href="#Second_attempt_for_the_Gibson_Assembly_of_pTetO_.2B_HlyB.2FD"><SPAN class="tocnumber">71</SPAN><SPAN class="toctext">Second attempt for the Gibson Assembly of pTetO + HlyB/D</SPAN></A></LI><LI class="toclevel-1"><A href="#New_small_cultures"><SPAN class="tocnumber">72</SPAN><SPAN class="toctext">New small cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_agar_plates_4"><SPAN class="tocnumber">73</SPAN><SPAN class="toctext">Making agar plates</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_Dissolving_Experiment"><SPAN class="tocnumber">74</SPAN><SPAN class="toctext">Gel Dissolving Experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Culture_OD600_measurement_check"><SPAN class="tocnumber">75</SPAN><SPAN class="toctext">Culture OD600 measurement check</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_cultures.2C_PCR_and_analysis_of_new_pTetO_.2B_HlyB.2FD"><SPAN class="tocnumber">76</SPAN><SPAN class="toctext">Small cultures, PCR and analysis of new pTetO + HlyB/D</SPAN></A></LI><LI class="toclevel-1"><A href="#Lysostaphin_induction"><SPAN class="tocnumber">77</SPAN><SPAN class="toctext">Lysostaphin induction</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_Dissolving_Experiment_2"><SPAN class="tocnumber">78</SPAN><SPAN class="toctext">Gel Dissolving Experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Culture_OD600_measurement_check_2"><SPAN class="tocnumber">79</SPAN><SPAN class="toctext">Culture OD600 measurement check</SPAN></A></LI><LI class="toclevel-1"><A href="#Incubate_dextran_hydrogels_and_induce_cultures_4"><SPAN class="tocnumber">80</SPAN><SPAN class="toctext">Incubate dextran hydrogels and induce cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#Plasmid_purification"><SPAN class="tocnumber">81</SPAN><SPAN class="toctext">Plasmid purification</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_washing_experiment_3"><SPAN class="tocnumber">82</SPAN><SPAN class="toctext">Gel washing experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Checking_the_Gibson_Assembly_by_Digestion_with_Ndel"><SPAN class="tocnumber">83</SPAN><SPAN class="toctext">Checking the Gibson Assembly by Digestion with Ndel</SPAN></A></LI><LI class="toclevel-1"><A href="#Glycerol_stock"><SPAN class="tocnumber">84</SPAN><SPAN class="toctext">Glycerol stock</SPAN></A></LI><LI class="toclevel-1"><A href="#Double_transformation_of_pBAD-lyso_.2B_pSTV_HlyB.2FD"><SPAN class="tocnumber">85</SPAN><SPAN class="toctext">Double transformation of pBAD-lyso + pSTV HlyB/D</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_washing_experiment_4"><SPAN class="tocnumber">86</SPAN><SPAN class="toctext">Gel washing experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#New_culture_plates_2"><SPAN class="tocnumber">87</SPAN><SPAN class="toctext">New culture plates</SPAN></A></LI><LI class="toclevel-1"><A href="#Formaldehyde_solution"><SPAN class="tocnumber">88</SPAN><SPAN class="toctext">Formaldehyde solution</SPAN></A></LI><LI class="toclevel-1"><A href="#Experimenting_with_dextran_hydrogel_and_bacteria_6"><SPAN class="tocnumber">89</SPAN><SPAN class="toctext">Experimenting with dextran hydrogel and bacteria</SPAN></A></LI><LI class="toclevel-1"><A href="#Repeat_digestion_of_pTetO_with_Ndel_and_BamH1_and_PstlHf"><SPAN class="tocnumber">90</SPAN><SPAN class="toctext">Repeat digestion of pTetO with Ndel and BamH1 and PstlHf</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_cultures_for_for_experiments:"><SPAN class="tocnumber">91</SPAN><SPAN class="toctext">Making cultures for for experiments:</SPAN></A></LI><LI class="toclevel-1"><A href="#Chloramphenicol_stock_solution_is_prepared"><SPAN class="tocnumber">92</SPAN><SPAN class="toctext">Chloramphenicol stock solution is prepared</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_new_tris_buffers"><SPAN class="tocnumber">93</SPAN><SPAN class="toctext">Making new tris buffers</SPAN></A></LI><LI class="toclevel-1"><A href="#Incubate_dextran_hydrogels_and_induce_cultures_5"><SPAN class="tocnumber">94</SPAN><SPAN class="toctext">Incubate dextran hydrogels and induce cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#Double_transformation"><SPAN class="tocnumber">95</SPAN><SPAN class="toctext">Double transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#Bigger_culture_.28150_mL.29"><SPAN class="tocnumber">96</SPAN><SPAN class="toctext">Bigger culture (150 mL)</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_washing_experiment_5"><SPAN class="tocnumber">97</SPAN><SPAN class="toctext">Gel washing experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#SEM_experiment"><SPAN class="tocnumber">98</SPAN><SPAN class="toctext">SEM experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Midi_prep"><SPAN class="tocnumber">99</SPAN><SPAN class="toctext">Midi prep</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_cultures."><SPAN class="tocnumber">100</SPAN><SPAN class="toctext">Small cultures.</SPAN></A></LI><LI class="toclevel-1"><A href="#Digestion_of_lysostaphin_construct_and_T7_promoter"><SPAN class="tocnumber">101</SPAN><SPAN class="toctext">Digestion of lysostaphin construct and T7 promoter</SPAN></A></LI><LI class="toclevel-1"><A href="#Ligation_of_lysostaphin_construct_to_T7_promoter"><SPAN class="tocnumber">102</SPAN><SPAN class="toctext">Ligation of lysostaphin construct to T7 promoter</SPAN></A></LI><LI class="toclevel-1"><A href="#Transformation_of_T7-lysostaphin_into_NovaBlue"><SPAN class="tocnumber">103</SPAN><SPAN class="toctext">Transformation of T7-lysostaphin into NovaBlue</SPAN></A></LI><LI class="toclevel-1"><A href="#Protein_induction"><SPAN class="tocnumber">104</SPAN><SPAN class="toctext">Protein induction</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_picking_2"><SPAN class="tocnumber">105</SPAN><SPAN class="toctext">Colony picking</SPAN></A></LI><LI class="toclevel-1"><A href="#Plasmid_purification_.28midi_prep.29."><SPAN class="tocnumber">106</SPAN><SPAN class="toctext">Plasmid purification (midi prep).</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_PAGE"><SPAN class="tocnumber">107</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_PCR"><SPAN class="tocnumber">108</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_cultures_for_for_experiments"><SPAN class="tocnumber">109</SPAN><SPAN class="toctext">Making cultures for for experiments</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_cultures"><SPAN class="tocnumber">110</SPAN><SPAN class="toctext">Small cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#Make_fresh_cultures_starting_from_old_ones"><SPAN class="tocnumber">111</SPAN><SPAN class="toctext">Make fresh cultures starting from old ones</SPAN></A></LI><LI class="toclevel-1"><A href="#Culture_OD600_measurement_check_3"><SPAN class="tocnumber">112</SPAN><SPAN class="toctext">Culture OD600 measurement check</SPAN></A></LI><LI class="toclevel-1"><A href="#NaCl_stock_was_prepared"><SPAN class="tocnumber">113</SPAN><SPAN class="toctext">NaCl stock was prepared</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_Dissolving_Experiment_II"><SPAN class="tocnumber">114</SPAN><SPAN class="toctext">Gel Dissolving Experiment II</SPAN></A></LI><LI class="toclevel-1"><A href="#Double_transformation_2"><SPAN class="tocnumber">115</SPAN><SPAN class="toctext">Double transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_cultures_2"><SPAN class="tocnumber">116</SPAN><SPAN class="toctext">Small cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#Plasmid_purification_.28mini_prep.29."><SPAN class="tocnumber">117</SPAN><SPAN class="toctext">Plasmid purification (mini prep).</SPAN></A></LI><LI class="toclevel-1"><A href="#Digestion"><SPAN class="tocnumber">118</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-1"><A href="#Ligation_and_transformation"><SPAN class="tocnumber">119</SPAN><SPAN class="toctext">Ligation and transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#Protein_induction_2"><SPAN class="tocnumber">120</SPAN><SPAN class="toctext">Protein induction</SPAN></A></LI><LI class="toclevel-1"><A href="#Double_transformation_3"><SPAN class="tocnumber">121</SPAN><SPAN class="toctext">Double transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_PCR_2"><SPAN class="tocnumber">122</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Fresh_transformation_with_adhesion"><SPAN class="tocnumber">123</SPAN><SPAN class="toctext">Fresh transformation with adhesion</SPAN></A></LI><LI class="toclevel-1"><A href="#New_culture_plates_3"><SPAN class="tocnumber">124</SPAN><SPAN class="toctext">New culture plates</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_PAGE_2"><SPAN class="tocnumber">125</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-1"><A href="#Protein_induction_3"><SPAN class="tocnumber">126</SPAN><SPAN class="toctext">Protein induction</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_culture_with_direct_induction."><SPAN class="tocnumber">127</SPAN><SPAN class="toctext">Small culture with direct induction.</SPAN></A></LI><LI class="toclevel-1"><A href="#Digestion_2"><SPAN class="tocnumber">128</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-1"><A href="#Ligation_and_transformation_2"><SPAN class="tocnumber">129</SPAN><SPAN class="toctext">Ligation and transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_PCR_of_SBD_colonies"><SPAN class="tocnumber">130</SPAN><SPAN class="toctext">Colony PCR of SBD colonies</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_cultures_3"><SPAN class="tocnumber">131</SPAN><SPAN class="toctext">Small cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_PCR_3"><SPAN class="tocnumber">132</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_cultures_4"><SPAN class="tocnumber">133</SPAN><SPAN class="toctext">Small cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#Miniprepping"><SPAN class="tocnumber">134</SPAN><SPAN class="toctext">Miniprepping</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_cultures_for_for_experiments:_2"><SPAN class="tocnumber">135</SPAN><SPAN class="toctext">Making cultures for for experiments:</SPAN></A></LI><LI class="toclevel-1"><A href="#Protein_induction_4"><SPAN class="tocnumber">136</SPAN><SPAN class="toctext">Protein induction</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_PAGE_3"><SPAN class="tocnumber">137</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-1"><A href="#Prepare_samples_for_sequencing"><SPAN class="tocnumber">138</SPAN><SPAN class="toctext">Prepare samples for sequencing</SPAN></A></LI><LI class="toclevel-1"><A href="#Miniprepping_lysostaphin"><SPAN class="tocnumber">139</SPAN><SPAN class="toctext">Miniprepping lysostaphin</SPAN></A></LI><LI class="toclevel-1"><A href="#Glycerol_Stock_BLR"><SPAN class="tocnumber">140</SPAN><SPAN class="toctext">Glycerol Stock BLR</SPAN></A></LI><LI class="toclevel-1"><A href="#Incubate_dextran_hydrogels_and_induce_cultures_6"><SPAN class="tocnumber">141</SPAN><SPAN class="toctext">Incubate dextran hydrogels and induce cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_cultures_5"><SPAN class="tocnumber">142</SPAN><SPAN class="toctext">Small cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_PAGE_4"><SPAN class="tocnumber">143</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-1"><A href="#Transformation_lysostaphin_from_Stockholm_2016_into_BL21"><SPAN class="tocnumber">144</SPAN><SPAN class="toctext">Transformation lysostaphin from Stockholm 2016 into BL21</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_washing_experiment_6"><SPAN class="tocnumber">145</SPAN><SPAN class="toctext">Gel washing experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_PAGE_5"><SPAN class="tocnumber">146</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-1"><A href="#HlyA_.2B_HlyB.2FD_.28TetO.29_induction"><SPAN class="tocnumber">147</SPAN><SPAN class="toctext">HlyA + HlyB/D (TetO) induction</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_PCR_4"><SPAN class="tocnumber">148</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Miniprepping_2"><SPAN class="tocnumber">149</SPAN><SPAN class="toctext">Miniprepping</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_PAGE_6"><SPAN class="tocnumber">150</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_agar_plates_5"><SPAN class="tocnumber">151</SPAN><SPAN class="toctext">Making agar plates</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_cultures_for_for_experiments:_3"><SPAN class="tocnumber">152</SPAN><SPAN class="toctext">Making cultures for for experiments:</SPAN></A></LI><LI class="toclevel-1"><A href="#Replenishing_solutions"><SPAN class="tocnumber">153</SPAN><SPAN class="toctext">Replenishing solutions</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_cultures_6"><SPAN class="tocnumber">154</SPAN><SPAN class="toctext">Small cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#Digestion_3"><SPAN class="tocnumber">155</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-1"><A href="#Agarose_gel_purification"><SPAN class="tocnumber">156</SPAN><SPAN class="toctext">Agarose gel purification</SPAN></A></LI><LI class="toclevel-1"><A href="#Incubate_dextran_hydrogels_and_induce_cultures_7"><SPAN class="tocnumber">157</SPAN><SPAN class="toctext">Incubate dextran hydrogels and induce cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#Protein_induction_5"><SPAN class="tocnumber">158</SPAN><SPAN class="toctext">Protein induction</SPAN></A></LI><LI class="toclevel-1"><A href="#Ligation_and_transformation_3"><SPAN class="tocnumber">159</SPAN><SPAN class="toctext">Ligation and transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_washing_experiment_7"><SPAN class="tocnumber">160</SPAN><SPAN class="toctext">Gel washing experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_PAGE_7"><SPAN class="tocnumber">161</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-1"><A href="#Continue_incubation_of_plates"><SPAN class="tocnumber">162</SPAN><SPAN class="toctext">Continue incubation of plates</SPAN></A></LI><LI class="toclevel-1"><A href="#Experimenting_with_dextran_hydrogel_and_bacteria_7"><SPAN class="tocnumber">163</SPAN><SPAN class="toctext">Experimenting with dextran hydrogel and bacteria</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment"><SPAN class="tocnumber">164</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_PAGE.3CH.2F3.3EWe_loaded_the_cell_media_of_the_pET24a-HlyA_.2B.E2.80.AFpTetO-HlyB.2FD_again._There_were_4_bands_visible_that_showed_that_this_time_there_was_HlyA_secretion..3CH3.3EProtein_induction"><SPAN class="tocnumber">165</SPAN><SPAN class="toctext">SDS PAGE&lt;H/3&gt;We loaded the cell media of the pET24a-HlyA + pTetO-HlyB/D again. There were 4 bands visible that showed that this time there was HlyA secretion.&lt;H3&gt;Protein induction</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_2"><SPAN class="tocnumber">166</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_PAGE_8"><SPAN class="tocnumber">167</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_cultures_for_for_experiments:_4"><SPAN class="tocnumber">168</SPAN><SPAN class="toctext">Making cultures for for experiments:</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_cultures_7"><SPAN class="tocnumber">169</SPAN><SPAN class="toctext">Small cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_purification"><SPAN class="tocnumber">170</SPAN><SPAN class="toctext">Gel purification</SPAN></A></LI><LI class="toclevel-1"><A href="#Miniprepping_3"><SPAN class="tocnumber">171</SPAN><SPAN class="toctext">Miniprepping</SPAN></A></LI><LI class="toclevel-1"><A href="#Induce_cultures"><SPAN class="tocnumber">172</SPAN><SPAN class="toctext">Induce cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_3"><SPAN class="tocnumber">173</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Double_transformation_4"><SPAN class="tocnumber">174</SPAN><SPAN class="toctext">Double transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_culture_of_T7-LacO-Lysostaphin-HlyA_biobrick"><SPAN class="tocnumber">175</SPAN><SPAN class="toctext">Small culture of T7-LacO-Lysostaphin-HlyA biobrick</SPAN></A></LI><LI class="toclevel-1"><A href="#Protein_induction_6"><SPAN class="tocnumber">176</SPAN><SPAN class="toctext">Protein induction</SPAN></A></LI><LI class="toclevel-1"><A href="#Digestion_4"><SPAN class="tocnumber">177</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_4"><SPAN class="tocnumber">178</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Glycerol_stock_2"><SPAN class="tocnumber">179</SPAN><SPAN class="toctext">Glycerol stock</SPAN></A></LI><LI class="toclevel-1"><A href="#Protein_induction_7"><SPAN class="tocnumber">180</SPAN><SPAN class="toctext">Protein induction</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_culture"><SPAN class="tocnumber">181</SPAN><SPAN class="toctext">Small culture</SPAN></A></LI><LI class="toclevel-1"><A href="#New_double_transformation"><SPAN class="tocnumber">182</SPAN><SPAN class="toctext">New double transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_PAGE_9"><SPAN class="tocnumber">183</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_2"><SPAN class="tocnumber">184</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_purification_PET_vector"><SPAN class="tocnumber">185</SPAN><SPAN class="toctext">Gel purification PET vector</SPAN></A></LI><LI class="toclevel-1"><A href="#Digestion_5"><SPAN class="tocnumber">186</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-1"><A href="#Ligation_and_transformation_4"><SPAN class="tocnumber">187</SPAN><SPAN class="toctext">Ligation and transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_5"><SPAN class="tocnumber">188</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_6"><SPAN class="tocnumber">189</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Protein_induction_.281_L_culture.29"><SPAN class="tocnumber">190</SPAN><SPAN class="toctext">Protein induction (1 L culture)</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_PAGE_10"><SPAN class="tocnumber">191</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_PAGE_11"><SPAN class="tocnumber">192</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_analysis_2"><SPAN class="tocnumber">193</SPAN><SPAN class="toctext">PCR analysis</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_3"><SPAN class="tocnumber">194</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Sequencing"><SPAN class="tocnumber">195</SPAN><SPAN class="toctext">Sequencing</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_PCR_5"><SPAN class="tocnumber">196</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_7"><SPAN class="tocnumber">197</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#PAMM"><SPAN class="tocnumber">198</SPAN><SPAN class="toctext">PAMM</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_analysis_3"><SPAN class="tocnumber">199</SPAN><SPAN class="toctext">PCR analysis</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_PCR_6"><SPAN class="tocnumber">200</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Transformation_into_NovaBlue"><SPAN class="tocnumber">201</SPAN><SPAN class="toctext">Transformation into NovaBlue</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_cultures_.2825_mL.29"><SPAN class="tocnumber">202</SPAN><SPAN class="toctext">Small cultures (25 mL)</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_PCR_7"><SPAN class="tocnumber">203</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_8"><SPAN class="tocnumber">204</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Glycerol_stock_3"><SPAN class="tocnumber">205</SPAN><SPAN class="toctext">Glycerol stock</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_cultures_.285_mL.29"><SPAN class="tocnumber">206</SPAN><SPAN class="toctext">Small cultures (5 mL)</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_purification_3"><SPAN class="tocnumber">207</SPAN><SPAN class="toctext">PCR purification</SPAN></A></LI><LI class="toclevel-1"><A href="#Miniprepping_4"><SPAN class="tocnumber">208</SPAN><SPAN class="toctext">Miniprepping</SPAN></A></LI><LI class="toclevel-1"><A href="#Ligation_and_transformation_5"><SPAN class="tocnumber">209</SPAN><SPAN class="toctext">Ligation and transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_9"><SPAN class="tocnumber">210</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Protein_induction_8"><SPAN class="tocnumber">211</SPAN><SPAN class="toctext">Protein induction</SPAN></A></LI><LI class="toclevel-1"><A href="#DpnI_digestion"><SPAN class="tocnumber">212</SPAN><SPAN class="toctext">DpnI digestion</SPAN></A></LI><LI class="toclevel-1"><A href="#Gibson_Assembly"><SPAN class="tocnumber">213</SPAN><SPAN class="toctext">Gibson Assembly</SPAN></A></LI><LI class="toclevel-1"><A href="#Transformation_.28C2987_cells.29"><SPAN class="tocnumber">214</SPAN><SPAN class="toctext">Transformation (C2987 cells)</SPAN></A></LI><LI class="toclevel-1"><A href="#Preparing_samples_for_sequencing"><SPAN class="tocnumber">215</SPAN><SPAN class="toctext">Preparing samples for sequencing</SPAN></A></LI><LI class="toclevel-1"><A href="#Transformation"><SPAN class="tocnumber">216</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_10"><SPAN class="tocnumber">217</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS_PAGE_12"><SPAN class="tocnumber">218</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-1"><A href="#Up_concentrating_the_cell_media"><SPAN class="tocnumber">219</SPAN><SPAN class="toctext">Up concentrating the cell media</SPAN></A></LI><LI class="toclevel-1"><A href="#Transformation_.28C2987_cells.29_2"><SPAN class="tocnumber">220</SPAN><SPAN class="toctext">Transformation (C2987 cells)</SPAN></A></LI><LI class="toclevel-1"><A href="#Double_transformation_5"><SPAN class="tocnumber">221</SPAN><SPAN class="toctext">Double transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_cultures_for_for_experiments:_5"><SPAN class="tocnumber">222</SPAN><SPAN class="toctext">Making cultures for for experiments:</SPAN></A></LI><LI class="toclevel-1"><A href="#Fresh_transformation_with_adhesion_2"><SPAN class="tocnumber">223</SPAN><SPAN class="toctext">Fresh transformation with adhesion</SPAN></A></LI><LI class="toclevel-1"><A href="#Washed_the_cells"><SPAN class="tocnumber">224</SPAN><SPAN class="toctext">Washed the cells</SPAN></A></LI><LI class="toclevel-1"><A href="#PAMM_2"><SPAN class="tocnumber">225</SPAN><SPAN class="toctext">PAMM</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_cultures_for_for_experiments:_6"><SPAN class="tocnumber">226</SPAN><SPAN class="toctext">Making cultures for for experiments:</SPAN></A></LI><LI class="toclevel-1"><A href="#Fresh_transformation_with_adhesion_3"><SPAN class="tocnumber">227</SPAN><SPAN class="toctext">Fresh transformation with adhesion</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_4"><SPAN class="tocnumber">228</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Shipping"><SPAN class="tocnumber">229</SPAN><SPAN class="toctext">Shipping</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_cultures_for_for_experiments:_7"><SPAN class="tocnumber">230</SPAN><SPAN class="toctext">Making cultures for for experiments:</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_11"><SPAN class="tocnumber">231</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_12"><SPAN class="tocnumber">232</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_analysis_4"><SPAN class="tocnumber">233</SPAN><SPAN class="toctext">PCR analysis</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_purification_2"><SPAN class="tocnumber">234</SPAN><SPAN class="toctext">Gel purification</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_purification_4"><SPAN class="tocnumber">235</SPAN><SPAN class="toctext">PCR purification</SPAN></A></LI><LI class="toclevel-1"><A href="#DpnI_digestion_2"><SPAN class="tocnumber">236</SPAN><SPAN class="toctext">DpnI digestion</SPAN></A></LI><LI class="toclevel-1"><A href="#Gibson_assembly_2"><SPAN class="tocnumber">237</SPAN><SPAN class="toctext">Gibson assembly</SPAN></A></LI><LI class="toclevel-1"><A href="#Transformation_.28C2987_cells.29_3"><SPAN class="tocnumber">238</SPAN><SPAN class="toctext">Transformation (C2987 cells)</SPAN></A></LI><LI class="toclevel-1"><A href="#Shipping_2"><SPAN class="tocnumber">239</SPAN><SPAN class="toctext">Shipping</SPAN></A></LI><LI class="toclevel-1"><A href="#Overnight_induction_of_T7-lysostaphin-HlyA"><SPAN class="tocnumber">240</SPAN><SPAN class="toctext">Overnight induction of T7-lysostaphin-HlyA</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_13"><SPAN class="tocnumber">241</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_14"><SPAN class="tocnumber">242</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_5"><SPAN class="tocnumber">243</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#DpnI_digestion_3"><SPAN class="tocnumber">244</SPAN><SPAN class="toctext">DpnI digestion</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_purification_5"><SPAN class="tocnumber">245</SPAN><SPAN class="toctext">PCR purification</SPAN></A></LI><LI class="toclevel-1"><A href="#Gibson_assembly_3"><SPAN class="tocnumber">246</SPAN><SPAN class="toctext">Gibson assembly</SPAN></A></LI><LI class="toclevel-1"><A href="#Transformation_.28C2987.29"><SPAN class="tocnumber">247</SPAN><SPAN class="toctext">Transformation (C2987)</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_analysis_5"><SPAN class="tocnumber">248</SPAN><SPAN class="toctext">PCR analysis</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_PCR_8"><SPAN class="tocnumber">249</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Preparing_samples_for_sequencing_2"><SPAN class="tocnumber">250</SPAN><SPAN class="toctext">Preparing samples for sequencing</SPAN></A></LI><LI class="toclevel-1"><A href="#Ligation_and_transformation_T7-lysostaphin"><SPAN class="tocnumber">251</SPAN><SPAN class="toctext">Ligation and transformation T7-lysostaphin</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_15"><SPAN class="tocnumber">252</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Colony_PCR_.28biobrick.29"><SPAN class="tocnumber">253</SPAN><SPAN class="toctext">Colony PCR (biobrick)</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR_analysis_6"><SPAN class="tocnumber">254</SPAN><SPAN class="toctext">PCR analysis</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_cultures_8"><SPAN class="tocnumber">255</SPAN><SPAN class="toctext">Small cultures</SPAN></A></LI><LI class="toclevel-1"><A href="#Inducer_mixture"><SPAN class="tocnumber">256</SPAN><SPAN class="toctext">Inducer mixture</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_cavity_experiment_16"><SPAN class="tocnumber">257</SPAN><SPAN class="toctext">Gel cavity experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Washed_the_cells_2"><SPAN class="tocnumber">258</SPAN><SPAN class="toctext">Washed the cells</SPAN></A></LI><LI class="toclevel-1"><A href="#PAMM_3"><SPAN class="tocnumber">259</SPAN><SPAN class="toctext">PAMM</SPAN></A></LI><LI class="toclevel-1"><A href="#Small_cultures_.28double.29"><SPAN class="tocnumber">260</SPAN><SPAN class="toctext">Small cultures (double)</SPAN></A></LI><LI class="toclevel-1"><A href="#Washed_the_cells_3"><SPAN class="tocnumber">261</SPAN><SPAN class="toctext">Washed the cells</SPAN></A></LI><LI class="toclevel-1"><A href="#PAMM_4"><SPAN class="tocnumber">262</SPAN><SPAN class="toctext">PAMM</SPAN></A></LI></UL></DIV><H3><SPAN class="mw-headline" id="Making_LB_medium:">Making LB medium:</SPAN></H3><UL><LI>LB Broth powder dissolved in miliQ </LI></UL><H3><SPAN class="mw-headline" id="Making_Pre-Cultures_for_binding_experiments">Making Pre-Cultures for binding experiments</SPAN></H3><P><B>Sample 1 (Control Sample):  MP15RPS</B><UL><LI> Cell type BL21(DE3).</LI><LI> Plasmid type PET24a (His tag on C -terminus).</LI></UL></P><P><B> Sample 2 (test condition with GFP): GFP15rps </B><UL><LI> Same cell type and plasmid </LI><LI> Region V of WT protein replaced with E. Coli sequence </LI><LI> GFP (mRuby) areplacing WYT RI </LI></UL></P>&quot;
<PRE>     }
   },
   &quot;Wednesday June 27, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="IPTG_stock_preparation">IPTG stock preparation</SPAN></H3><P>Making a concentration of 100 mM.</P><P> 0.5mL should be later added to culture medium (normally 0.5mM - 1mM </P><H3><SPAN class="mw-headline" id="Diluting_bacteria_for_induction">Diluting bacteria for induction</SPAN></H3><P>Induced at OD600 values: GFPM15HA - 1.046 A;  MP15RPS - 1.023 A </P><P>Culture flasks left to incubate overnight (22°C, 150 RPM)</P>&quot;
<PRE>     }
   },
   &quot;Thursday June 28, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Making_pellets_of_the_cells">Making pellets of the cells</SPAN></H3><P>Cells were centrifuged and a pellet was created.</P><P> A lysis buffer was prepared.</P><P> Cells were suspended in lysis buffer.</P><P> Lysed buffer was centrifuged and the supernatant was transferred to different tubes.</P><P> Dextran beads were used to see if the loose adhesin in the buffer binds to dextran (using SDS page).</P>&quot;
<P>}
</P><PRE>   },
   &quot;Friday June 29, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="SDS_page_gel_examination">SDS page gel examination</SPAN></H3><P>A band is visible above 250 kDA (our adhesin) in the pellet.</P>&quot;
<PRE>     }
   },
   &quot;Monday July 2, 2018&quot;:{
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of a new batch of methacrylated dextran was started (iG04). The first dextran hydrogels were synthesized from the other Dex-MA batch (iG03).</P>&quot;
<PRE>     }
   },
   &quot;Tuesday July 3, 2018&quot;:{
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The polymerisation reaction was quenched and the gels were washed. Some gels were not quenched, so they can polymerize a little bit longer.</P>&quot;
<PRE>     }
   },
   &quot;Wednesday July 4, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Nickel_column">Nickel column</SPAN></H3><P>Nickel column performed with resin.</P><P>Seperated these samples:</P><UL><LI>GFP supernatant.</LI><LI>GFP pellet (bound to resin).</LI><LI>WT supernatant.</LI><LI>WT pellet (bound to resin).</LI></UL><H3><SPAN class="mw-headline" id="SDS_gel_electrophoresis">SDS gel electrophoresis</SPAN></H3><P>SDS gel electrophoresis performed on the samples above.</P><P>The purification by his-tag was not very effective. This may be due to protein binding domain binding to  other proteins.</P><P>The GFP proteins did not bind well to the Ni resin. This is likely due to degradation at the C-terminus and should not affect  binding to dextran.</P> &quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The reaction for the new batch of Dex-MA (iG04) was quenched and dialysis started.</P>&quot;
<PRE>     }
   },
   &quot;Thursday July 5, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The plan is to test two variants for dextran binding: 1) Normal modified  Adhesin and 2) Bacteria expressing the Adhesin without the binding domains (control). </P><H3><SPAN class="mw-headline" id="Making_agar_plates">Making agar plates</SPAN></H3><P>Made agar plates in the microwave using 25 g/L LB broth and 15 g/L bacto-agar.</P><H3><SPAN class="mw-headline" id="Preparing_cultures_for_FACS_experiments_tomorrow">Preparing cultures for FACS experiments tomorrow</SPAN></H3><UL><LI><B>mRII18HA</B> - mRuby Adhesin with 18 repeats of region II and no binding domain (negative control)</LI><LI><B>mR15HA</B> - mRuby Adhesin with 15 repeats of region II and a binding domain.</LI></UL>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The other gels from 18-07-02 were quenched and washed.</P>&quot;
<PRE>     }
   },
   &quot;Friday July 6, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="FACS_preparations">FACS preparations</SPAN></H3><P>Performed an OD measurement (for FACS we will need approximately 10^7 cells) and diluted the cells.</P><P>Prepared Tris buffer (10 mM tris, 150 mM NaCl and 2 mM CaCl).</P><P>Made a pellet from the cells and resuspended in Tris buffer. This was used for a measurement on the FACS device with dextran beads that have a fluorescent label. This was done for both <B>mR15HA</B> and <B>mRII18HA</B>.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Freezedring of iG04 started.</P>&gt;&quot;
<PRE>     }
   },
   &quot;Monday July 9, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Preparing_2YT_stock_solution">Preparing 2YT stock solution</SPAN></H3><P>2YT medium was prepared and sterilized to grow bacteria that have been frozen for a long time.</P><H3><SPAN class="mw-headline" id="Started_cultures_from_frozen_stock">Started cultures from frozen stock</SPAN></H3><UL><LI><B>mRII18HA</B> - mRuby Adhesin with 18 repeats of region II and no binding domain (negative control)</LI><LI><B>mR15HA</B> - mRuby Adhesin with 15 repeats of region II and a binding domain.</LI></UL>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>NMR performed to determine the Degree of Substitution of iG04.</P>&quot;
<PRE>     }
   },
   &quot;Tuesday July 10, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Cultures that have been grown have been induced are tested with fluorescent antibodies (his tag) to see if they can bind to Adhesin and therefore we know it is fully expressed and exported to the outside. </P><H3><SPAN class="mw-headline" id="Frozen_permanent_made_.28glycrol_stock.29">Frozen permanent made (glycrol stock)</SPAN></H3><UL><LI><B>mRII18HA</B> - mRuby Adhesin with 18 repeats of region II and no binding domain (negative control)</LI><LI><B>mR15HA</B> - mRuby Adhesin with 15 repeats of region II and a binding domain.</LI></UL><H3><SPAN class="mw-headline" id="SACS_experiment">SACS experiment</SPAN></H3><P>Cultures were induced and the ODs are measured over time in multiple conditions (20,0°C, 37,0°C). Samples of the cultures were taken at different timepoints. Pellets of the cultures were made and resuspended. Antibodies were added and the cells were measured using SACS.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of a new batch of methacrylated dextran was started (iG05).</P>&quot;
<PRE>     }
   },
   &quot;Wednesday July 11, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Same procedure was done as yesterday.</P>&quot;
<PRE>     }
   },
   &quot;Thursday July 12, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="FACS_device">FACS device</SPAN></H3><P>Today we discussed with our supervisor what we would do next week in his absence. We also got an introduction on how to work with the FACS device.</P><H3><SPAN class="mw-headline" id="Preparing_2YT_stock_solution_2">Preparing 2YT stock solution</SPAN></H3><P>2YT medium was prepared and sterilized to grow bacteria that have been frozen for a long time.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The reaction for the new batch of Dex-MA (iG05) was quenched and dialysis started.</P>&quot;
<PRE>     }
   },
   &quot;Friday July 13, 2018&quot;:{
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Freezedring of iG05 started.</P>&quot;
<PRE>     }
   },
   &quot;Monday July 16, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Kanamycin_stock_solution_is_prepared">Kanamycin stock solution is prepared</SPAN></H3><P>A kanamycin stock solution was prepared (50 mg/mL) in miliQ.</P><H3><SPAN class="mw-headline" id="LB_cultures_of_R18_and_R15_are_prepared_from_colony_plates">LB cultures of R18 and R15 are prepared from colony plates</SPAN></H3><UL><LI><B>mRII18HA</B> - mRuby Adhesin with 18 repeats of region II and no binding domain (negative control)</LI><LI><B>mR15HA</B> - mRuby Adhesin with 15 repeats of region II and a binding domain.</LI></UL><H3><SPAN class="mw-headline" id="Make_new_plates_of_R18_and_R15">Make new plates of R18 and R15</SPAN></H3><P>The permanently frozen bacteria were taken from the freezer, and used to make new plates of <B>R15</B> and <B>R18</B>.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>NMR performed to determine the Degree of Substitution of iG05. Synthesis of dextran hydrogels from iG04.</P>&quot;
<PRE>     }
   },
   &quot;Tuesday July 17, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Making_agar_plates_2">Making agar plates</SPAN></H3><P>Made agar plates in the microwave using 25 g/L LB broth and 15 g/L bacto-agar containing Kanamycin and Chloramphenicol.</P><H3><SPAN class="mw-headline" id="Experimenting_with_dextran_hydrogel_and_bacteria">Experimenting with dextran hydrogel and bacteria</SPAN></H3><P>Pieces of dextran hydrogel were added to <B>mR15HA</B> and <B>mRII18HA</B> cultures and shaken overnight. This was to seed the hydrogel with the respective bacteria.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The polymerisation reaction of 18-07-16 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   },
   &quot;Wednesday July 18, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Experimenting_with_dextran_hydrogel_and_bacteria_2">Experimenting with dextran hydrogel and bacteria</SPAN></H3><P>The gels that incubated overnight were transferred to new medium with roughly equal OD and then were induced for protein expression. After incubating again they were washed with water, pictured for red fluorescence and  then placed again in cell-free medium, adding EDTA to one <B>R15</B>  culture. EDTA should chelate the calcium and denature the Adhesin, causing the cells to detach from the gel. If Adhesin binding works as we hope it should make this gel less fluorescent than the <B>R15</B> without EDTA. Pictures of the red fluorescence were taken.</P><H3><SPAN class="mw-headline" id="Plating_the_gel_medium">Plating the gel medium</SPAN></H3><P>From each growth tube a 20μL was pipetted onto an agar plate.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of dextran hydrogels from iG05.</P>&quot;
<PRE>     }
   },
   &quot;Thursday July 19, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Analyzing_the_data_from_yesterday">Analyzing the data from yesterday</SPAN></H3><P>The agar-LB plates were inspected for colonies. The images from yesterday were analyzed.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of dextran hydrogels from iG05 in a 96-well plate. The polymerisation reaction of 18-07-18 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   },
   &quot;Friday July 20, 2018&quot;:{
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The polymerisation reaction of 18-07-19 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   },
   &quot;Monday July 23, 2018&quot;:{
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of dextran hydrogels from iG05 in a 96-well plate.</P>&quot;
<PRE>     }
   },
   &quot;Tuesday July 24, 2018&quot;:{
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of dextran hydrogels from iG05 in a 96-well plate. The polymerisation reaction of 18-07-23 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   },
   &quot;Wednesday July 25, 2018&quot;:{
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of dextran hydrogels from iG05 in a 96-well plate. The polymerisation reaction of 18-07-24 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   },
   &quot;Thursday July 26, 2018&quot;:{
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The polymerisation reaction of 18-07-25 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   },
   &quot;Monday July 30, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="LB_cultures_of_R18_and_R15_.2B_WT_are_prepared_from_colony_plates">LB cultures of R18 and R15 + WT are prepared from colony plates</SPAN></H3><UL><LI><B>mRII18HA</B> - mRuby Adhesin with 18 repeats of region II and no binding domain (negative control)</LI><LI><B>mR15HA</B> - mRuby Adhesin with 15 repeats of region II and a binding domain.</LI><LI><B>WT15HA</B> – Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain.</LI></UL>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="PCR">PCR</SPAN></H3><P>A PCR was performed to synthesize the correct pieces of DNA to perform a Gibson Assembly on.</P><P>The following constructs were chosen:</P><P><B>Lysostaphin-HlyA-His (CL2)</B>. This construct represents natural lysostaphin with our new secretion system. </P><UL><LI>1500 bp</LI><LI>Annealing temp. = 51.7 °C</LI></UL><P><B>Sec-Delay-Lysostaphin-His (CL3)</B>. This represents natural lysostaphin and its natural secretion system.</P><UL><LI>1266 bp</LI><LI>Annealing temp. = 50.3 °C</LI></UL><P><B>HlyA-His (CL4)</B>. This represents only the secretion system.</P><UL><LI>657 bp</LI><LI>Annealing temp. = 50.1 °C</LI></UL>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of dextran hydrogels from iG05 in a 96-well plate.</P>&quot;
<PRE>     }
   },
   &quot;Tuesday July 31, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Prepare_LB_broth_and_make_agar-LB_plates">Prepare LB broth and make agar-LB plates</SPAN></H3><P>For LB broth (25 g/L LB broth) and LB-agar (25 g/L LB broth and 15 g/L bacto-agar)</P><H3><SPAN class="mw-headline" id="Incubate_dextran_hydrogels_and_induce_cultures">Incubate dextran hydrogels and induce cultures</SPAN></H3><P>Each of the 3 cultures (<B>mRII18HA, mR15HA,  WT15HA</B>) were induced  and two intact dextran hydrogels were added.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="PCR_analysis">PCR analysis</SPAN></H3><P>To analyze whether or not the PCR had succeeded in amplifying our constructs of DNA, the samples were run on an agarose gel. After analyzing the samples using an UV reader, it could be concluded that CL2 and CL4 had a clear and sharp band at the correct length and were thus successfully amplified. CL3 however, was vague and had no clear band at the desired length and thus not amplified correctly. Another PCR of CL3 will be performed tomorrow.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The polymerisation reaction of 18-07-30 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   },
   &quot;Wednesday August 1, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Experimenting_with_dextran_hydrogel_and_bacteria_3">Experimenting with dextran hydrogel and bacteria</SPAN></H3><P>Dextran hydrogels were transferred to culture tubes and shaken to wash out loosely bound bacteria. Then transferred again to new medium, crushed and plated (see iGEM Eindhoven protocols). This was done for three different conditions: <B>mRII18HA, mR15HA and WT15HA</B>.&lt;p&gt;</P><H3><SPAN class="mw-headline" id="New_culture_plates">New culture plates</SPAN></H3>&lt;p&gt;Culture plates from frozen stock mRII18HA, mR15HA and WT15HA were made.&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="PCR_of_CL3">PCR of CL3</SPAN></H3><P>Since CL3 (Sec-Delay-Lysostaphin-His) was not correctly amplified with PCR the day before, it was tried again with stricter conditions and a different temperature gradient. When it was analyzed on an UV reader after running it on an agarose gel we concluded that there was no improvement and it was still not correctly amplified. A third try of PCR will be performed tomorrow.</P><H3><SPAN class="mw-headline" id="PCR_purification">PCR purification</SPAN></H3><P>The CL2 Lysostaphin-HlyA-His and CL4 HlyA-His constructs have been purified and measured on a Nanodrop to give the following concentrations: <UL><LI><B>Lysostaphin-HlyA-His (CL2) </B> = 26.7 ng/µL</LI></UL><UL><LI><B>HlyA-His (CL4) </B> = 30.0 ng/µL</LI></UL></P>&quot;
<P>	  }
</P><PRE>   },
   &quot;Thursday August 2, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Experimenting_with_dextran_hydrogel_and_bacteria_4">Experimenting with dextran hydrogel and bacteria</SPAN></H3><P>The plates from yesterday were analysed and a cell count was done.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="PCR_of_CL3_2">PCR of CL3</SPAN></H3><P>We tried to to a PCR of CL3 for the third time with an even greater temperature gradient. After analysis we found out that there was still no improvement. We had decided to not continue with it, since we would be using the HlyA secretion system anyway instead of the natural secretion system.</P>&quot;
<P>}
</P><PRE>   },
   &quot;Friday August 3, 2018&quot;:{
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="PCR_of_vector_TetO_and_insert_HlyB.2FD">PCR of vector TetO and insert HlyB/D</SPAN></H3><P>A PCR was performed on the vector TetO and insert HlyB/D (for the secretion system). The following constructs were amplified:</P><P><B>Vector TetO</B></P><UL><LI>2598 bp</LI><LI>Annealing temp. = 50.7°C</LI></UL><P><B>Insert HlyB/D</B></P><UL><LI>3579 bp</LI><LI>Annealing temp. = 50.0°C</LI></UL><P>Analysis of the PCR products showed that the vector TetO had one clear band at the correct height, but the insert HlyB/D showed only a band around the size of the amplification primers</P>&quot;
<P>}
</P><PRE>   },
   &quot;Monday August 6, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="LB_cultures_of_R18_and_R15_.2B_WT_were_prepared_from_colony_plates">LB cultures of R18 and R15 + WT were prepared from colony plates</SPAN></H3><UL><LI><B>mRII18HA</B> - mRuby Adhesin with 18 repeats of region II and no binding domain (negative control)</LI><LI><B>mR15HA</B> - mRuby Adhesin with 15 repeats of region II and a binding domain.</LI><LI><B>WT15HA</B> – Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain.</LI></UL><H3><SPAN class="mw-headline" id="SEM">SEM</SPAN></H3><P>Small culture with dextran hydrogel inside was left induced overnight to be used for SEM tomorrow</P><H3><SPAN class="mw-headline" id="Making_agar_plates_3">Making agar plates</SPAN></H3><P>Made agar plates in the microwave using 25 g/L LB broth and 15 g/L bacto-agar containing Kanamycin and Chloramphenicol.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Repeat_of_PCR_of_HlyB.2FD">Repeat of PCR of HlyB/D</SPAN></H3><P>HlyB/D would be run twice, once with the dilution from Friday, August 3rd with 0.95 ng/µL and once undiluted with 95.2 ng/µL. After analyzing it was seen that there were two clear bands at the correct height, and thus the amplification of HlyB/D had succeeded this time.</P><H3><SPAN class="mw-headline" id="Culturing_of_pBAD_vector_in_Nova_Blue_cells">Culturing of pBAD vector in Nova Blue cells</SPAN></H3><P>The pBAD vector in NB cells was cultured in 8 mL LB and 8 µL Ampicillin. It was cultured overnight in the incubator at 37°C, 250 rpm.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of a new batch of methacrylated dextran was started (iG06).</P>&quot;
<PRE>     }
   },
   &quot;Tuesday August 7, 2018&quot;:{
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Plasmid_purification_of_pBAD">Plasmid purification of pBAD</SPAN></H3><P>Plasmid purification was performed for 2x 1.5 mL cell culture. DNA concentration was measured with the NanoDrop.<UL><LI>1. pBAD = 35 ng/µL&lt;/il&gt;&lt;il&gt;2. pBAD = 41 ng/µL&lt;/il&gt;</LI></UL></P><H3><SPAN class="mw-headline" id="PCR_of_purified_DNA_from_pBAD">PCR of purified DNA from pBAD</SPAN></H3>&lt;p&gt;A PCR was performed on the DNA obtained by the purification performed this morning on the overnight cultures. A construct pBAD of 3957 bp and an annealing temperature of 54.9°C was used. PCR was run overnight.&quot;
<PRE>     }
   },
   &quot;Wednesday August 8, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Prepare_LB_broth_and_make_agar-LB_plates_2">Prepare LB broth and make agar-LB plates</SPAN></H3><P>For LB broth (25 g/L LB broth) and LB-agar (25 g/L LB broth and 15 g/L bacto-agar).</P><H3><SPAN class="mw-headline" id="LB_cultures_of_R18_and_R15_.2B_WT_were_prepared_from_colony_plates_2">LB cultures of R18 and R15 + WT were prepared from colony plates</SPAN></H3><P>Previous cultures didn’t grow well, therefore the following bacteria were cultured again:</P><UL><LI><B>mRII18HA</B> - mRuby Adhesin with 18 repeats of region II and no binding domain (negative control)</LI><LI><B>mR15HA</B> - mRuby Adhesin with 15 repeats of region II and a binding domain.</LI><LI><B>WT15HA</B> – Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain.</LI></UL><H3><SPAN class="mw-headline" id="SEM_2">SEM</SPAN></H3><P>Pictures of our hydrogel containing bacteria were made using SEM.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="PCR_analysis_of_pBAD">PCR analysis of pBAD</SPAN></H3><P>After analyzing with an UV reader, a clear band could be seen at the correct height.</P><H3><SPAN class="mw-headline" id="PCR_purification_2">PCR purification</SPAN></H3><P>A PCR purification was then performed on the three constructs and measured with Nanodrop:</P><UL><LI>pTetO = 32 ng/µL</LI><LI>HlyB/D = 45.9 ng/µL (2 PCR products added together)</LI><LI>pBAD = 37 ng/µL</LI></UL><H3><SPAN class="mw-headline" id="DpnI_digestion.2C_Gibson_Assembly_and_Transformation">DpnI digestion, Gibson Assembly and Transformation</SPAN></H3><P>A digestion was performed on the pTetO and pBAD constructs. This was followed by a Gibson Assembly with the pBAD + Lysostaphin-HlyA-His and pTetO + HlyB/D constructs. These constructs were then transformed into C2987 cells. After an overnight incubation it was seen that pBAD + lysostaphin-HlyA-His cells showed some colonies, whilst the pTetO+HlyB/D had no colonies.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The reaction for the new batch of Dex-MA (iG06) was quenched and dialysis started.</P>&quot;
<PRE>     }
   },
   &quot;Thursday August 9, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Incubate_dextran_hydrogels_and_induce_cultures_2">Incubate dextran hydrogels and induce cultures</SPAN></H3><P>Each of the 3 cultures (<B>mRII18HA, mR15HA,  WT15HA</B>) were induced  and two intact dextran hydrogels were added.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Colony_picking_of_pBAD-Lysostaphin-HlyA-His.">Colony picking of pBAD-Lysostaphin-HlyA-His.</SPAN></H3><P>After colony picking, we made 6 small cultures of the pBAD-Lyso-HlyA-His cells that were transformed the day before.</P><H3><SPAN class="mw-headline" id="Colony_PCR_of_pBAD-Lysostaphin-HlyA.">Colony PCR of pBAD-Lysostaphin-HlyA.</SPAN></H3><P>A Colony PCR was performed on the pBAD-Lysostaphin-HlyA cells to see if the lysostaphin gene has been produced. 2 out of the 6 cultures had the correct gene after analyzing results overnight.</P><H3><SPAN class="mw-headline" id="Plating.">Plating.</SPAN></H3><P>PTetO + HlyB/D was plated again to what went wrong yesterday. This time there were a lot of colonies.</P>&quot;
<PRE>     },
   },
   &quot;Friday August 10, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Making_new_Tris_buffer">Making new Tris buffer</SPAN></H3><P>Prepared Tris buffer (10 mM tris, 150 mM NaCl and 2 mM CaCl).</P><H3><SPAN class="mw-headline" id="Gel_washing_experiment">Gel washing experiment</SPAN></H3><P>Performed the Gel washing experiment (see iGEM Eindhoven protocols), this time with different washing and different conditions. For example EDTA was added to see if this makes a difference.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Analysis.2C_purification_and_storage_of_the_pBAD-Lysostaphin-HlyA-His_constructs.">Analysis, purification and storage of the pBAD-Lysostaphin-HlyA-His constructs.</SPAN></H3><P>The PCR products of the 6 pBAD-Lysostaphin-HlyA-His small cultures were analyzed with an agarose gel and UV reader. Colony 3 and 4 contained the correct insert. After that, a 50% glycerol stock was made for colony 3 and 4 and then it was purified with the following results:</P><UL><LI>pBAD-Lysostaphin-HlyA-His 3 = 54.8 ng/µL </LI><LI>pBAD-Lysostaphin-HlyA-His 4 = 61.8 ng</LI></UL>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Freezedring of iG06 started.</P>&quot;
<PRE>     }
   },
   &quot;Monday August 13, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="LB_cultures_of_R18_and_R15_.2B_WT_were_prepared_from_colony_plates_3">LB cultures of R18 and R15 + WT were prepared from colony plates</SPAN></H3><UL><LI><B>mRII18HA</B> - mRuby Adhesin with 18 repeats of region II and no binding domain (negative control)</LI><LI><B>mR15HA</B> - mRuby Adhesin with 15 repeats of region II and a binding domain.</LI><LI><B>WT15HA</B> – Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain.</LI><LI><B>WT15HA NO SECRETION SYSTEM</B> - Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain, however this bacteria doesn’t have the plasmid to secrete the adhesin. </LI></UL>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Colony_picking_and_PCR_of_pTetO_.2B_HlyB.2FD">Colony picking and PCR of pTetO + HlyB/D</SPAN></H3><P>6 colonies were picked from the plated pTetO + HlyB/D of yesterday to create 6 small cultures. A colony PCR was then performed on these small cultures to check for the presence of HlyB/D.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>NMR performed to determine the Degree of Substitution of iG06. Synthesis of dextran hydrogels from iG06 in a 96-well plate.&lt;/p&quot;
      }
    },
    &quot;Tuesday August 14, 2018&quot;:{
      &quot;Adhesin&quot;:{
        &quot;html&quot;:&quot;</P><H3><SPAN class="mw-headline" id="Incubate_dextran_hydrogels_and_induce_cultures_3">Incubate dextran hydrogels and induce cultures</SPAN></H3>&lt;p&gt;Each of the 4 cultures (<B>mRII18HA, mR15HA,  WT15HA and WT15HA NO SECRETION SYSTEM</B>) were induced  and two intact dextran hydrogels were added.&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="PCR_analysis_and_colony_PCR">PCR analysis and colony PCR</SPAN></H3><P>The products from the PCR of the yesterday were analyzed with an agarose gel and UV reader, but only bands at the heights of the primers was visible. Therefore, a new colony PCR was performed on the same 6 cultures.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The polymerisation reaction of 18-08-13 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   },
   &quot;Wednesday August 15, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_washing_experiment_2">Gel washing experiment</SPAN></H3><P>Performed the Gel washing experiment (see iGEM Eindhoven protocols), with 4 different bacteria present in hydrogels and 5 conditions in total.</P><H3><SPAN class="mw-headline" id="Kanamycin_stock_solution_is_prepared_2">Kanamycin stock solution is prepared</SPAN></H3><P>A kanamycin stock solution was prepared (50 mg/mL) in miliQ.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Analysis_of_colony_PCR">Analysis of colony PCR</SPAN></H3><P>Colony PCR was analyzed with agarose gel and UV reader, and it did not succeed. There were no clear bands, only some vague bands around the heights of the primers.</P><H3><SPAN class="mw-headline" id="Transformation_of_pBAD-Lysostaphin_into_BL21_.28DE3.29_cells">Transformation of pBAD-Lysostaphin into BL21 (DE3) cells</SPAN></H3><P>Colony 3 and 4 of the purification on Friday, the 10th of August were used for the transformation into BL21 (DE3) cells. </P><H3><SPAN class="mw-headline" id="Colony_picking">Colony picking</SPAN></H3><P>New colonies were picked from the pTetO-HlyB/D plates to check for the presence of HlyB/D. 10 small cultures were made.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of dextran hydrogels from iG06 in a 96-well plate.</P>&quot;
<PRE>     }
   },
   &quot;Thursday August 16, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Experimenting_with_dextran_hydrogel_and_bacteria_5">Experimenting with dextran hydrogel and bacteria</SPAN></H3><P>The plates from yesterday were analysed and a cell count was done.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Colony_PCR_and_PCR_analysis.">Colony PCR and PCR analysis.</SPAN></H3><P>A colony PCR was performed on the 10 small cultures created yesterday with pTetO-HlyB/D. After that, the colony PCR of yesterday was analyzed. Once again, there were no results as there were only bands arounds the primers.</P> &quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The polymerisation reaction of 18-08-15 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   },
   &quot;Friday August 17, 2018&quot;:{
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Colony_PCR_and_analysis">Colony PCR and analysis</SPAN></H3><P>A new colony PCR was performed, but this time with a control sample that had the construct HlyB/D and the 10 new small cultures. It failed again, as there were no bands from the small cultures (only at the heights of the primers). However, the control samples gave a clear band at the right height. </P>&quot;
<PRE>     }
   },
   &quot;Monday August 20, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Making_cultures_for_SEM_and_following_experiments">Making cultures for SEM and following experiments</SPAN></H3><UL><LI><B>WT15HA</B> – Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain.</LI><LI><B>WT15HA NO SECRETION SYSTEM</B> - Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain, however this bacteria doesn’t have the plasmid to secrete the adhesin. </LI></UL>. <P>A part of these cultures were induced and a hydrogel was added.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Second_attempt_for_the_Gibson_Assembly_of_pTetO_.2B_HlyB.2FD">Second attempt for the Gibson Assembly of pTetO + HlyB/D</SPAN></H3><P>Both the insert (pTetO) and the vector (HlyB/D) were digested and then transformed into C2987 cells. This resulted in a lot of colonies. </P><H3><SPAN class="mw-headline" id="New_small_cultures">New small cultures</SPAN></H3><P>New small cultures were made from the colonies pBAD-Lysostaphin-HlyA-His 3 and 4.</P>&quot;
<PRE>     }
   },
   &quot;Tuesday August 21, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Making_agar_plates_4">Making agar plates</SPAN></H3><P>Made agar plates in the microwave using 25 g/L LB broth and 15 g/L bacto-agar containing Kanamycin and Chloramphenicol.</P><H3><SPAN class="mw-headline" id="Gel_Dissolving_Experiment">Gel Dissolving Experiment</SPAN></H3><P>Tried to dissolve our dextran hydrogel using trueGel3D Enzymatic Cell Recovery Solution from Sigma-Aldrich.</P><H3><SPAN class="mw-headline" id="Culture_OD600_measurement_check">Culture OD600 measurement check</SPAN></H3><P>The OD600 values of the cultures set yesterday were to low, much lower than the 1.4A we were aiming for. Bacteria seem to grow slowly. This may be due to high antibiotics concentration, or due to culture plates being too old. On advice new cultures were taken from two colonies:</P><UL><LI><B>WT15HA</B> – Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain.</LI><LI><B>WT15HA NO SECRETION SYSTEM</B> - Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain, however this bacteria doesn’t have the plasmid to secrete the adhesin. </LI></UL>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Small_cultures.2C_PCR_and_analysis_of_new_pTetO_.2B_HlyB.2FD">Small cultures, PCR and analysis of new pTetO + HlyB/D</SPAN></H3><P>New small cultures were made of pTetO+HlyB/D which were incubated for 5 hours in the morning, and then a colony PCR was run with a control construct that has the HlyB/D. It turned out that colony 3, 4 and 5 had a band at the right height but were quite thin and samples 4 and 5 had an extra band. </P><H3><SPAN class="mw-headline" id="Lysostaphin_induction">Lysostaphin induction</SPAN></H3><P> An induction of lysostaphin was done to see if it gets produced. This was done on the transformed cells from Wednesday the 15th of August. Induction was done with arabinose. It was analyzed with an SDS-PAGE in which we could see a clear protein production pattern over time, but it seemed as if lysostaphin-HlyA was cleaved over time since the bands at 50kDa (combined weight) dissappeared and bands at 25 kDa (separate weights) occurred. </P>&quot;
<PRE>     },
   },
   &quot;Wednesday August 22, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_Dissolving_Experiment_2">Gel Dissolving Experiment</SPAN></H3><P>Tried to dissolve our dextran hydrogel using trueGel3D Enzymatic Cell Recovery Solution from Sigma-Aldrich using different concentrations than yesterday.</P><H3><SPAN class="mw-headline" id="Culture_OD600_measurement_check_2">Culture OD600 measurement check</SPAN></H3><P>Cultures that were set yesterday were still growing very slowly.</P><H3><SPAN class="mw-headline" id="Incubate_dextran_hydrogels_and_induce_cultures_4">Incubate dextran hydrogels and induce cultures</SPAN></H3><P>Each of the cultures (WT15HA and WT15HA NO SECRETION SYSTEM) were induced and 3 intact dextran hydrogels were added to each condition.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Plasmid_purification">Plasmid purification</SPAN></H3><P>The plasmid purification was done with the mini prep kit. It was done on the 3 colonies that seemed to express HlyB/D based on colony PCR from August 21. The following concentrations were obtained with Nanodrop. </P><UL><LI>pTetO-HlyB/D 3 = 24 ng/µL</LI><LI>pTetO-HlyB/D 4 = 20.5 ng/µL</LI><LI>PTetO-HlyB/D 5 = 33.5 ng/µL</LI></UL>&quot;
<PRE>     }
   },
   &quot;Thursday August 23, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_washing_experiment_3">Gel washing experiment</SPAN></H3><P>Performed the Gel washing experiment (see iGEM Eindhoven protocols), with 2 different bacteria present in hydrogels and 3 conditions in total.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Checking_the_Gibson_Assembly_by_Digestion_with_Ndel">Checking the Gibson Assembly by Digestion with Ndel</SPAN></H3><P>This was performed on the 3 samples from yesterday. It was unclear whether or not the Gibson Assembly had worked, since the digestion time was too short and the concentrations of the samples was too low. Only sample 5 showed two light bands around the expected sizes. </P><H3><SPAN class="mw-headline" id="Glycerol_stock">Glycerol stock</SPAN></H3><P>50% glycerol stocks were made from the three samples to be stored in the -80°C freezer.</P><H3><SPAN class="mw-headline" id="Double_transformation_of_pBAD-lyso_.2B_pSTV_HlyB.2FD">Double transformation of pBAD-lyso + pSTV HlyB/D</SPAN></H3><UL><LI>pBAD-Lysostaphin and pSTV-HlyB/D were double transformed into BL21 (DE3) cells. There were colonies and it had thus succeeded. </LI></UL>&quot;
<PRE>     }
   },
   &quot;Friday August 24, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_washing_experiment_4">Gel washing experiment</SPAN></H3><P>The wash buffers that were left from the gel washing experiment were also plated (no time yesterday).</P><H3><SPAN class="mw-headline" id="New_culture_plates_2">New culture plates</SPAN></H3><P>Culture plates from frozen stock <B>WT15HA</B> and <B>WT15HA NO SECRETION SYSTEM</B> were made.</P><H3><SPAN class="mw-headline" id="Formaldehyde_solution">Formaldehyde solution</SPAN></H3><P>A new 4% formaldehyde solution was made for fixing bacteria in hydrogels for SEM.</P><H3><SPAN class="mw-headline" id="Experimenting_with_dextran_hydrogel_and_bacteria_6">Experimenting with dextran hydrogel and bacteria</SPAN></H3><P>The plates from yesterday were analysed and a cell count was done.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Repeat_digestion_of_pTetO_with_Ndel_and_BamH1_and_PstlHf">Repeat digestion of pTetO with Ndel and BamH1 and PstlHf</SPAN></H3><P>When the samples were run over an agarose gel, it turned out that both the pBAD-Lysostaphin-HlyA-His and pTetO-HlyB/D showed the correct bands at the correct height.</P>&quot;
<PRE>     }
   },
   &quot;Monday August 27, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Making_cultures_for_for_experiments:">Making cultures for for experiments:</SPAN></H3><UL><LI><B>WT15HA</B> – Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain.</LI><LI><B>WT15HA NO SECRETION SYSTEM</B> - Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain, however this bacteria doesn’t have the plasmid to secrete the adhesin. </LI></UL><H3><SPAN class="mw-headline" id="Chloramphenicol_stock_solution_is_prepared">Chloramphenicol stock solution is prepared</SPAN></H3><P>A chloramphenicol stock solution was prepared (30 mg/mL) in ethanol</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Only new autoclaved LB was made this day.</P> &quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>New molds with cavity have been designed.</P>&quot;
<PRE>     }
   },
   &quot;Tuesday August 28, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Making_new_tris_buffers">Making new tris buffers</SPAN></H3><UL><LI>10 mM tris pH 9, 150 mM NaCl, 2 mM CaCl</LI><LI>10 mM tris pH 9, 150 mM NaCl, 0.5 mM EDTA</LI></UL><H3><SPAN class="mw-headline" id="Incubate_dextran_hydrogels_and_induce_cultures_5">Incubate dextran hydrogels and induce cultures</SPAN></H3><P>Each of the cultures (<B>WT15HA and WT15HA NO SECRETION SYSTEM</B>) were induced and 5 intact dextran hydrogels were added to each condition.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Double_transformation">Double transformation</SPAN></H3><P>A new double transformation of pBAD-Lysostaphin-HlyA-His and pTetO-HlyB/D was performed into BL21 (DE3) cells. This resulted in 50-100 colonies. </P><H3><SPAN class="mw-headline" id="Bigger_culture_.28150_mL.29">Bigger culture (150 mL)</SPAN></H3><P>Big cultures were made from the pBAD-Lysostaphin-HlyA-His (C2987 cells) and pTetO-HlyB/D (C2987 cells).</P>&quot;
<PRE>     }
   },
   &quot;Wednesday August 29, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_washing_experiment_5">Gel washing experiment</SPAN></H3><P>Performed the Gel washing experiment (see iGEM Eindhoven protocols), with 2 different bacteria present in hydrogels and 3 conditions in total.</P><H3><SPAN class="mw-headline" id="SEM_experiment">SEM experiment</SPAN></H3><P>Also prepared dextran hydrogels in bacterial medium (<B>WT15HA and WT15HA NO SECRETION SYSTEM</B>). Afterwards formaldehyde solution was added to fixate the dextran hydrogels.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Midi_prep">Midi prep</SPAN></H3><P>A midi prep was performed on pBAD-Lysostaphin-HlyA-His in C2987 cells and pTetO-HlyB/D in C2987 cells. The concentrations measured with the Nanodrop were too low so the midi prep failed.</P><H3><SPAN class="mw-headline" id="Small_cultures.">Small cultures.</SPAN></H3><P>Several small cultures were made for protein induction and mini prep that will be performed later this week. </P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Digestion_of_lysostaphin_construct_and_T7_promoter">Digestion of lysostaphin construct and T7 promoter</SPAN></H3><P>Performed digestion of lysostaphin (from Stockholm 2016) and T7 promoter as described in iGEM Eindhoven protocols.</P><H3><SPAN class="mw-headline" id="Ligation_of_lysostaphin_construct_to_T7_promoter">Ligation of lysostaphin construct to T7 promoter</SPAN></H3><P>Performed ligation of lysostaphin to T7 promoter as described in iGEM Eindhoven protocols. </P><H3><SPAN class="mw-headline" id="Transformation_of_T7-lysostaphin_into_NovaBlue">Transformation of T7-lysostaphin into NovaBlue</SPAN></H3><P>The T7-lysostaphin construct is ligated into pSB1C3, followed by transformation into NovaBlue competent cells. The procedure for ligation and transformation is described in iGEM Eindhoven protocols. The transformed bacteria were plated on chloramphenicol plates and left in the incubator at 37 °C for overnight.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>We received the new PMMA molds for the experiments with the gels with a cavity.</P>&quot;
<PRE>     }
   },
   &quot;Thursday August 30, 2018&quot;:{
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Protein_induction">Protein induction</SPAN></H3><P>Protein induction was performed to test the production and secretion of lysostaphin, analyzed with SDS-PAGE. Protein induction was performed on:</P><UL><LI>pBAD-Lysostaphin-HlyA-His 3 (BL21(DE3)), induced by arabinose.</LI><LI>pBAD-Lysostaphin-HlyA-His + pTeto-HlyB/D (BL21(DE3)), induced by arabinose and ATC. </LI><LI>pBAD-Lysostaphin-HlyA-His +pSTV- HlyB/D (BL21(DE3)), induced by arabinose and IPTG.</LI></UL><P>Will be analyzed on Friday</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Colony_picking_2">Colony picking</SPAN></H3><P>Colony picking of bacteria transformed with T7-lysostaphin is performed. The picked colonies are grown in LB medium and incubated overnight at 300 rpm and 37 °C.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>We received a new PDMS mold for the gels with a cavity. Synthesis of the first gels with a cavity was started.</P>&quot;
<PRE>     }
   },
   &quot;Friday August 31, 2018&quot;:{
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Plasmid_purification_.28midi_prep.29.">Plasmid purification (midi prep).</SPAN></H3><P>A new plasmid purification was done, but this time with mini prep. It was performed on C2987 pBAD-Lysostaphin-HlyA-His and pTetO-HlyB/D. This resulted in the following concentrations:</P><UL><LI>C2987 pBAD-Lysostaphin-HlyA-His 1 = 70 ng/µL</LI><LI>C2987 pBAD-Lysostaphin-HlyA-His 2 = 70 ng/µL</LI><LI>C2987 pTetO-HlyB/D 1 = 22 ng / µL</LI><LI>C2987 pTetO-HlyB/D 2 = 18.5 ng / µL</LI></UL><H3><SPAN class="mw-headline" id="SDS_PAGE">SDS PAGE</SPAN></H3><P>The SDS PAGE of Thursday was analyzed. There were no clear bands and thus no proof that lysostaphin had been expressed or secreted.</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Colony_PCR">Colony PCR</SPAN></H3><P>Colony PCR of the bacteria transformed with lysostaphin-T7 in pSB1C3 was performed as described in iGEM Eindhoven protocols. </P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The polymerisation reaction of the gels with a cavity was quenched and the gels were washed. The PDMS mold gave the best gels, so we continued working with this mold instead of with the PMMA mold.</P>&quot;
<PRE>     }
   },
   &quot;Monday September 3, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Making_cultures_for_for_experiments">Making cultures for for experiments</SPAN></H3><UL><LI><B>WT15HA</B> – Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain.</LI><LI><B>WT15HA NO SECRETION SYSTEM</B> - Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain, however this bacteria doesn’t have the plasmid to secrete the adhesin. </LI></UL>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Small_cultures">Small cultures</SPAN></H3><P>Small cultures were made from glycerol stocks from the -80 °C freezer of the pBAD-Lysostaphin-HlyA-His (C2987) and pTetO-HlyB/D (C2987) for DNA purification. Also, primer dilutions were made for sequencing. </P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Make_fresh_cultures_starting_from_old_ones">Make fresh cultures starting from old ones</SPAN></H3><P>Fresh LB medium is inoculated with the culture of transformed cells from August 30.</P> &quot;
<PRE>     }
   },
   &quot;Tuesday September 4, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Culture_OD600_measurement_check_3">Culture OD600 measurement check</SPAN></H3><P>Cultures didn’t grow enough to work with. Probably the plates we set cultures from were damaged, therefore we decided to start everything from scratch. We discarded the cultures.</P><H3><SPAN class="mw-headline" id="NaCl_stock_was_prepared">NaCl stock was prepared</SPAN></H3><P>A 5M stock of NaCl in miliQ was prepared.</P><H3><SPAN class="mw-headline" id="Gel_Dissolving_Experiment_II">Gel Dissolving Experiment II</SPAN></H3><P>LB Medium was made with pH 7.4 made by adding kOH and HCl to LB and a dextran hydrogel was added. Different concentrations of trueGel3D Enzymatic Cell Recovery Solution were used to dissolve the hydrogel.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Double_transformation_2">Double transformation</SPAN></H3><P>We wanted to make the following constructs:</P><UL><LI>pET28a-HlyA + pSTV-HlyB/D --&gt; had no colonies (most likely due to using the diluted pSTV samples, instead of the undiluted. This meant that there was not enough DNA for a transformation.</LI><LI>pET28a-HlyA + pTetO-HlyB/D --&gt; had colonies</LI></UL><H3><SPAN class="mw-headline" id="Small_cultures_2">Small cultures</SPAN></H3><P>Small cultures were made of the following constructs to be used for protein induction.</P><UL><LI>pBAD-Lysostaphin-HlyA-His + pTeto-HlyB/D (BL21(DE3))</LI><LI>pBAD-Lysostaphin-HlyA-His + HlyB/D (BL21(DE3)).</LI></UL><H3><SPAN class="mw-headline" id="Plasmid_purification_.28mini_prep.29.">Plasmid purification (mini prep).</SPAN></H3><P>This was done to prepare the samples to be sequenced. First, the DNA had to be purified. </P><P>After mini prepping, the following concentrations were obtained via Nanodrop:</P><UL><LI>pBAD-Lysostaphin-HlyA-His 3 = 125 ng/μL</LI><LI> pTeto-HlyB/D 5 = 54 ng/μL. </LI></UL><P>These samples were then send to be sequenced. </P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Digestion">Digestion</SPAN></H3><P>The sugar binding domain (BBa_K2812000) and linearized pSB1C3 are digested. </P><H3><SPAN class="mw-headline" id="Ligation_and_transformation">Ligation and transformation</SPAN></H3><P>The sugar binding domain is ligated into pSB1C3. The resulting vector is transformed into competent NovaBlue cells. The transformed cells were plated on agar LB plates with chloramphenicol and were incubated overnight at 37 °C.</P>&quot;
<PRE>     }
   },
   &quot;Wednesday September 5, 2018&quot;:{
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Protein_induction_2">Protein induction</SPAN></H3><P>Protein induction was performed again to check for lysostaphin expression. The following constructs were used for this:</P><UL><LI>pBAD-Lysostaphin-HlyA-His + pTetO-HlyB/D (BL21(DE3)), induced by arabinose and ATC.</LI><LI>pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D (BL21(DE3)), induced by arabinose and IPTG.</LI></UL><P>Will be analyzed on Thursday.</P><H3><SPAN class="mw-headline" id="Double_transformation_3">Double transformation</SPAN></H3><P>Since on Tuesday, a diluted pSTV sample was used which resulted in a failed double transformation, it was performed again with the undiluted pSTV. This resulted in colonies. </P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Colony_PCR_2">Colony PCR</SPAN></H3><P>Colony PCR is performed with colonies of the bacteria transformed with the sugar binding domain. The resulting amplified DNA is analyzed with an agarose gel.</P> &quot;
<PRE>     }
   },
   &quot;Thursday September 6, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Fresh_transformation_with_adhesion">Fresh transformation with adhesion</SPAN></H3><P>A fresh transformation was done using the following plasmids:</P><UL><LI>R1R15HA - SBD adhesin with Kanamycin resistance. </LI><LI>PSTV HlyB/D - Secretion system type I with Chloramphenicol resistance.</LI><LI>This was done following the iGEM Eindhoven transformation protocol. This was done twice in E. Coli BLR Strain. Once using only the R1R15HA plasmid (No secretion system) and once using both plasmids.</LI></UL><H3><SPAN class="mw-headline" id="New_culture_plates_3">New culture plates</SPAN></H3><P>Culture plates from frozen stock <B>WT15HA</B> and <B>WT15HA(No secretion system)</B> were made.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="SDS_PAGE_2">SDS PAGE</SPAN></H3><P>No clear bands that showed Lysostaphin expression in the cell.&lt;/p</P><H3><SPAN class="mw-headline" id="Protein_induction_3">Protein induction</SPAN></H3>&lt;p&gt;This time, protein induction was done to check Lysostaphin expression <B>and</B> secretion.<P>Constructs:</P><UL><LI>pBAD-Lysostaphin-HlyA-His + pTetO-HlyB/D</LI><LI>pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D.</LI></UL><P>We used different concentrations of IPTG and 200 μM arabinose for induction. The induction started at an OD around 0.7. The induction was done at 30 °C at 180 rpm. The samples were taken at t0 and overnight. </P><H3><SPAN class="mw-headline" id="Small_culture_with_direct_induction.">Small culture with direct induction.</SPAN></H3><P>This was done to only induce HlyA, no lysostaphin should be present.</P><P>Constructs used:</P><UL><LI>pET28a-HlyA + pSTV-HlyB/D, induced by IPTG.</LI><LI>pET28a-HlyA + pTetO-HlyB/D, induced by IPTG and ATC.</LI></UL>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Digestion_2">Digestion</SPAN></H3><P>Linearized pSB1C3 is digested. The digested plasmid is purified with agarose gel purification as described in iGEM Eindhoven protocols. </P><H3><SPAN class="mw-headline" id="Ligation_and_transformation_2">Ligation and transformation</SPAN></H3><P>The sugar binding domain and lysostaphin are ligated into pSB1C3. The resulting plasmids are transformed into competent XLI cells. The transformed cells are spreaded on plates with chloramphenicol and incubated overnight at 37 °C.</P>&quot;
<PRE>     }
   },
   &quot;Friday September 7, 2018&quot;:{
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Colony_PCR_of_SBD_colonies">Colony PCR of SBD colonies</SPAN></H3><P>Colony PCR of the colonies transformed with sugar binding domain is performed. The resulting DNA is analyzed with agarose gel analysis. </P><H3><SPAN class="mw-headline" id="Small_cultures_3">Small cultures</SPAN></H3><P>Small cultures are made from colonies of with successfully transformed cells with sugar binding domain construct.</P>&quot;
<PRE>     }
   },
   &quot;Monday September 10, 2018&quot;:{
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Colony_PCR_3">Colony PCR</SPAN></H3><P>A colony PCR is performed with bacteria transformed with the lysostaphin construct. The resulting DNA is analyzed with an agarose gel.</P><H3><SPAN class="mw-headline" id="Small_cultures_4">Small cultures</SPAN></H3><P>Small cultures are made of the sucesfully transformed cells with lysostaphin construct.</P><H3><SPAN class="mw-headline" id="Miniprepping">Miniprepping</SPAN></H3><P>The cultures of bacteria with lysostaphin are used for miniprepping. The DNA concentration in the miniprepped samples is measured with the Nanodrop technique. </P> &quot;
<PRE>     }
   },
   &quot;Tuesday September 11, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Making_cultures_for_for_experiments:_2">Making cultures for for experiments:</SPAN></H3><P>This time from the freshly transformed bacteria.</P><UL><LI><B>WT15HA</B> – Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain.</LI><LI><B>WT15HA NO SECRETION SYSTEM</B> - Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain, however this bacteria doesn’t have the plasmid to secrete the adhesin. </LI></UL>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Protein_induction_4">Protein induction</SPAN></H3><P>Only induction of Lysostaphin (no secretion) based on the small cultures from yesterday. The construct used is pBAD-Lysostaphin-HlyA-His. It was induced at several concentrations of arabinose. These ranged from 0.2% - 0.000002%. Samples were taken before induction, after 4h and overnight. </P><H3><SPAN class="mw-headline" id="SDS_PAGE_3">SDS PAGE</SPAN></H3><P>Analyzing the SDS PAGE from Monday. There was only secretion by the pSTV-HlyB/D cells. </P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Prepare_samples_for_sequencing">Prepare samples for sequencing</SPAN></H3><P>Samples for sequencing were prepared of the miniprepped vector with sugar binding domain. </P><H3><SPAN class="mw-headline" id="Miniprepping_lysostaphin">Miniprepping lysostaphin</SPAN></H3><P>Cultures of bacteria with lysostaphin were miniprepped. The concentration of the resulting samples is measured with the nanodrop technique. </P>&quot;
<PRE>     }
   },
   &quot;Wednesday September 12, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Glycerol_Stock_BLR">Glycerol Stock BLR</SPAN></H3><P>Glycerol Stock was prepared for both <B>WT15HA</B> and <B>WT15HA (No secretion system)</B>. </P><H3><SPAN class="mw-headline" id="Incubate_dextran_hydrogels_and_induce_cultures_6">Incubate dextran hydrogels and induce cultures</SPAN></H3><P>Each of the cultures <B>(WT15HA and WT15HA NO SECRETION SYSTEM)</B> were induced and 5 intact dextran hydrogels were added to each condition. In this case we used Erlenmeyer’s for the hydrogel seeding.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Small_cultures_5">Small cultures</SPAN></H3><P>Small cultures were made for induction from the following constructs:</P><UL><LI>pET24a-HlyA + pTetO-HlyB/D</LI><LI>pBAD-Lysostaphin-HlyA-His</LI></UL><H3><SPAN class="mw-headline" id="SDS_PAGE_4">SDS PAGE</SPAN></H3><P>We did an SDS again to check for lysostaphin expression at different inducer concentrations. The construct used was pBAD-Lysostaphin-HlyA-His. There was no clear expression, and the samples were a bit overloaded. </P><P>Loaded 5 μL of the pellet samples. Still no clear overexpression of Lysostaphin, also the samples were a bit overloaded.</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Transformation_lysostaphin_from_Stockholm_2016_into_BL21">Transformation lysostaphin from Stockholm 2016 into BL21</SPAN></H3><P>The lysostaphin from Stockholm 2016 and are own designed lysostaphin construct are transformed into competent BL21 cells. The transformed cells are plated and incubated overnight at 37 °C. </P> &quot;
<PRE>     }
   },
   &quot;Thursday September 13, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_washing_experiment_6">Gel washing experiment</SPAN></H3><P>The dextran hydrogels seem to have degraded overnight. Therefore we did not have any hydrogels to continue the experiments with. This was probably due to the vigorous shaking within the Erlenmeyer’s at 250 RPM.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="SDS_PAGE_5">SDS PAGE</SPAN></H3><P>Loaded the same samples as yesterday only this time 2 μL for the pellets instead of 5. </P><P>Still no clear expression was visible. But there were darker bands visible in the higher concentrations of inducers.</P><H3><SPAN class="mw-headline" id="HlyA_.2B_HlyB.2FD_.28TetO.29_induction">HlyA + HlyB/D (TetO) induction</SPAN></H3><P>These experiments were performed to find the optimal ATC inducer concentration. The concentrations ATC ranged from 0.0001 µM - 10 µM. Induction failed however, since calcium was forgotten. Construct: pET24a-HlyA + pTetO-HlyB/D</P> &quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Colony_PCR_4">Colony PCR</SPAN></H3><P>Colony PCR is performed of the BL21 cells transformed with lysostaphin. The resulting DNA is analyzed with an agarose gel. </P><H3><SPAN class="mw-headline" id="Miniprepping_2">Miniprepping</SPAN></H3><P>Cultures with lysostaphin from Stockholm 2016 are miniprepped. The resulting DNA concentration is determined with Nanodrop. </P> &quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of a new cubic gel with a cavity was started.</P>&quot;
<PRE>     }
   },
   &quot;Friday September 14, 2018&quot;:{
     &quot;cloning&quot;:{
</PRE>      &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="SDS_PAGE_6">SDS PAGE</SPAN></H3><P>The goal was to test the expresion and secretion of HlyA with two promotors (pSTV and pTetO). The following constructs were used for this. pET28a-HlyA + pSTV-HlyB/D and pET28a-HlyA + pTetO-HlyB/D. There was clear expression but no clear secretion. This is likely due to calcium not being added.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The polymerisation reaction of the cavity gel of 18-09-13 was quenched and the gel washed.</P>&quot;
<PRE>     }
   },
   &quot;Monday September 17, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Making_agar_plates_5">Making agar plates</SPAN></H3><P>Made agar plates in the microwave using 25 g/L LB broth and 15 g/L bacto-agar.</P><H3><SPAN class="mw-headline" id="Making_cultures_for_for_experiments:_3">Making cultures for for experiments:</SPAN></H3><UL><LI><B>WT15HA</B> – Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain.</LI><LI><B>WT15HA NO SECRETION SYSTEM</B> - Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain, however this bacteria doesn’t have the plasmid to secrete the adhesin. </LI><LI><B>mRII18HA</B> - mRuby Adhesin with 18 repeats of region II and no binding domain (negative control).</LI></UL>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Replenishing_solutions">Replenishing solutions</SPAN></H3><P>Made 200 mL of Sep 2x YT solution for a better and richer medium used for expression. Also, a 1M arabinose stock solution was made.</P><H3><SPAN class="mw-headline" id="Small_cultures_6">Small cultures</SPAN></H3><P>Small cultures were prepared for induction. The following constructs were used: pET24a-HlyA + pTetO-HlyB/D and pBAD-Lysostaphin-HlyA-His </P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Digestion_3">Digestion</SPAN></H3><P>Linearized pSB1C3, pBAD-lysostaphin and pBAD-pyocin are digested. </P><H3><SPAN class="mw-headline" id="Agarose_gel_purification">Agarose gel purification</SPAN></H3><P>The pSB1C3 is purified with agarose gel purification.</P>&quot;
<PRE>     }
   },  &quot;Tuesday September 18, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Incubate_dextran_hydrogels_and_induce_cultures_7">Incubate dextran hydrogels and induce cultures</SPAN></H3><P>Each of the 3 cultures <B>(mR15HA,  WT15HA, WT15HA (no secretion system))</B> were induced  and four intact dextran hydrogels were added.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Protein_induction_5">Protein induction</SPAN></H3><P>An induction was started to check for lysostaphin expression and HlyA secretion via the following constructs: pET24a-HlyA + pTetO-HlyB/D (OD of 0.5) and pBAD-Lysostaphin-HlyA-His 3 (OD of 0.8). The following inducer concentrations were used:</P><UL><LI>arabinose (0.1 mM; 1 mM; 10 mM).</LI><LI>IPTG (0.2 mM).</LI><LI>Calcium (5 mM).</LI><LI>ATC (0.001 μM; 0.01 μM;  0.1 μM; 1 μM; 2 μM; 10 μM).</LI></UL>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Ligation_and_transformation_3">Ligation and transformation</SPAN></H3><P>The pBAD-lysostaphin and pBAD-pyocin are ligated into the pSB1C3 plasmid. Then, competent NovaBlue cells are transformed with these plasmids. The transformed cells are plated on plates with chloramphenicol and incubated overnight at 37 °C. </P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>New PDMS molds were ordered to upskill the cavity gel synthesis.</P>&quot;
<PRE>     }
   },
   &quot;Wednesday September 19, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_washing_experiment_7">Gel washing experiment</SPAN></H3><P>Performed the Gel washing experiment (see iGEM Eindhoven protocols), with 3 different bacteria present in hydrogels and 3 conditions in total.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="SDS_PAGE_7">SDS PAGE</SPAN></H3><P>Small band visible that showed HlyA secretion, but still no clear overexpression of Lysostaphin.</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Continue_incubation_of_plates">Continue incubation of plates</SPAN></H3><P>The plates with lysostaphin and pyocin bacteria are incubated one more day because in order to allow the colonies to grow larger. </P>&quot;
<PRE>     }
   },
   &quot;Thursday September 20, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Experimenting_with_dextran_hydrogel_and_bacteria_7">Experimenting with dextran hydrogel and bacteria</SPAN></H3><P>The plates from yesterday were analysed and a cell count was done.</P><H3><SPAN class="mw-headline" id="Gel_cavity_experiment">Gel cavity experiment</SPAN></H3><P>Performed a variant of the gel cavity experiment as described by iGEM Eindhoven protocols. This time, tris buffer (10 mM tris pH 9, 150 mM NaCl, 5 mM CaCl) was used and no IPTG was added in the medium. Also the gels were not soaked in LB or any other nutrient buffer.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="SDS_PAGE.3CH.2F3.3EWe_loaded_the_cell_media_of_the_pET24a-HlyA_.2B.E2.80.AFpTetO-HlyB.2FD_again._There_were_4_bands_visible_that_showed_that_this_time_there_was_HlyA_secretion..3CH3.3EProtein_induction">SDS PAGE&lt;H/3&gt;<P>We loaded the cell media of the pET24a-HlyA + pTetO-HlyB/D again. There were 4 bands visible that showed that this time there was HlyA secretion.</P>&lt;H3&gt;Protein induction</SPAN></H3><P>The goal was to get lysostaphin expression with a higher concentration of arabinose and to additionaly take samples after 3 hours. Construct used: pBAD-Lysostaphin-HlyA-His (OD of 0.8).Induced at several arabinose concentrations (1, 5, 10, 20, 100 mM). This induction was performed overnight.</P>&quot;
<PRE>     }
   },
   &quot;Friday September 21, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_cavity_experiment_2">Gel cavity experiment</SPAN></H3><P>Took a sample of the experiment that was set up yesterday (see iGEM Eindhoven protocols). </P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="SDS_PAGE_8">SDS PAGE</SPAN></H3><P>Analyzing SDS results of pBAD-Lysostaphin-HlyA-His from Thursday. The 3 highest concentrations of arabinose showed a darker band around 50 kD, which shows overexpression op Lysostaphin-HlyA-His.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The new PDMS (4x) molds were ready and we could pick them up.</P>&quot;
<PRE>     }
   },
   &quot;Monday September 24, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Making_cultures_for_for_experiments:_4">Making cultures for for experiments:</SPAN></H3><UL><LI><B>WT15HA</B> – Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain.</LI><LI><B>WT15HA NO SECRETION SYSTEM</B> - Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain, however this bacteria doesn’t have the plasmid to secrete the adhesin. </LI><LI><B>mRII18HA</B> - mRuby Adhesin with 18 repeats of region II and no binding domain (negative control).</LI></UL>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Small_cultures_7">Small cultures</SPAN></H3><P>Small cultures were prepared from BL21 (DE3) cells containing pBAD-Lysostaphin-HlyA-His + pTetO-HlyB/D or pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D plasmids.</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_purification">Gel purification</SPAN></H3><P>T7-Lysostaphin is purified on an agarose gel. </P><H3><SPAN class="mw-headline" id="Miniprepping_3">Miniprepping</SPAN></H3><P>Miniprep the PET vector. Measure the DNA concentration with nanodrop. </P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of 5 new cubic gels with a cavity was started.</P>&quot;
<PRE>     }
   },
   &quot;Tuesday September 25, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Induce_cultures">Induce cultures</SPAN></H3><P>Each of the 3 cultures <B>(mR15HA,  WT15HA, WT15HA (no secretion system))</B> were induced.</P><H3><SPAN class="mw-headline" id="Gel_cavity_experiment_3">Gel cavity experiment</SPAN></H3><P>Performed a variant of the gel cavity experiment as described by iGEM Eindhoven protocols. This time, miliQ salt buffer (150 mM NaCl, 5 mM CaCl) was used and no IPTG was added in the medium. Also the gels were not soaked in LB or any other nutrient buffer. Older cultures of <B>mR15HA</B> and <B>WT15HA</B> were used.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Double_transformation_4">Double transformation</SPAN></H3><P>Two new cultures of double transformed cells for new lysostaphin secretion have been made. The constructs used for transformation in BL21 (DE3) cells are pBAD-Lysostaphin-HlyA-His + pTetO-HlyB/D or pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D.</P><P>Overnight incubating showed no colonies, so the double transformation had failed. Will do again with fresh plates tomorrow.</P><H3><SPAN class="mw-headline" id="Small_culture_of_T7-LacO-Lysostaphin-HlyA_biobrick">Small culture of T7-LacO-Lysostaphin-HlyA biobrick</SPAN></H3><P>A 25 mL small culture of C2987 cells containing T7-LacO-Lysostaphin-HlyA plasmid was made in 5 separate culture tubes.</P><H3><SPAN class="mw-headline" id="Protein_induction_6">Protein induction</SPAN></H3><P>A protein induction was performed to check for secretion of lysostaphin. Performed with BL21 (DE3) cells containing the pBAD-Lysostaphin-HlyA-His + pTetO-HlyB/D (OD= 0.57), or pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D (OD= 0.77) plasmids. Induced with aTc (1 µM), arabinose (20 mM) and calcium (5 mM). Took samples before induction, after 3h and overnight.</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Digestion_4">Digestion</SPAN></H3><P>The PET vector and T7-lysostaphin are digested with NdeI and SacI. </P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The polymerisation reaction of the cavity gels of 18-09-24 was quenched and the gel washed.</P>&gt;&quot;
<PRE>     }
   },
   &quot;Wednesday September 26, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_cavity_experiment_4">Gel cavity experiment</SPAN></H3><P>Took a sample of the experiment that was set up yesterday (see iGEM Eindhoven protocols).</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Glycerol_stock_2">Glycerol stock</SPAN></H3><P>Made a 50% glycerol stock of C2987 cells containing the T7-LacO-Lysostaphin-HlyA plasmid for the -80°C freezer.</P><H3><SPAN class="mw-headline" id="Protein_induction_7">Protein induction</SPAN></H3><P>A protein induction of C2987 cells containing T7-LacO-Lysostaphin-HlyA plasmid was planned, but the OD increased too fast to start with the induction. It was therefore postponed until tomorrow.</P><H3><SPAN class="mw-headline" id="Small_culture">Small culture</SPAN></H3><P>Because of the failed induction, a new small culture of C2987 cells containing T7-LacO-Lysostaphin-HlyA plasmid(2x 25 mL) was started.</P><H3><SPAN class="mw-headline" id="New_double_transformation">New double transformation</SPAN></H3><P>A new double transformation was performed, see Day 42 – Monday September 25. Same construct, but on fresh plates. This time there were colonies.</P><H3><SPAN class="mw-headline" id="SDS_PAGE_9">SDS PAGE</SPAN></H3><P>The SDS of the double transformed cells from the small culture on Monday September 24 with the pellet and medium of the pBAD-Lysostaphin-HlyA-His + pTetO-HlyB/D and pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D constructs. There were some bands in the cell media of pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D.</P><H3><SPAN class="mw-headline" id="PCR_2">PCR</SPAN></H3><P>The goal was to get a vector + insert with the correct overlap for Gibson Assembly. This was done with pBAD (4691 bp, Tm = 60.1 °C) and Pyocin 1 and 2 (1494 bp, 50.2°C). As it was performed overnight, it will be analyzed tomorrow.</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_purification_PET_vector">Gel purification PET vector</SPAN></H3><P>The PET vector is purified with agarose gel purification. </P><H3><SPAN class="mw-headline" id="Digestion_5">Digestion</SPAN></H3><P>The pSB1C3 and T7-lysostaphin are digested with EcoRI and PstI. </P><H3><SPAN class="mw-headline" id="Ligation_and_transformation_4">Ligation and transformation</SPAN></H3><P>Digested T7-lysostaphin is ligated into PET and pSB1C3 vectors. These plasmids are used for transformation of competent NovaBlue cells. The cells are plated and incubated overnight at 37 °C. </P>&quot;
<PRE>     }
   },
   &quot;Thursday September 27, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_cavity_experiment_5">Gel cavity experiment</SPAN></H3><P>Took a sample of the experiment that was set up two days ago (see iGEM Eindhoven protocols).</P><H3><SPAN class="mw-headline" id="Gel_cavity_experiment_6">Gel cavity experiment</SPAN></H3><P>Performed a variant of the gel cavity experiment as described by iGEM Eindhoven protocols. This time, miliQ salt buffer (150 mM NaCl, 5 mM CaCl) was used and no IPTG was added in the medium. Also the gels were not soaked in LB or any other nutrient buffer. New cultures of <B>mR15HA</B> and <B>WT15HA</B> were used.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Protein_induction_.281_L_culture.29">Protein induction (1 L culture)</SPAN></H3><P>Protein induction was performed on BL21 (DE3) T7-LacO-Lysostaphin-HlyA (OD=0.5), induced with 0.5 mM IPTG, to determine lysostaphin secretion. After 3h we did the French press to get the cell lysate. We stored the lysate under 3 different conditions, in the fridge, in 20% glycerol in the freezer and fast freezing in nitrogen in the -80 °C freezer. Also, the pellet of the lysate was stored in the -80 °C freezer.</P><H3><SPAN class="mw-headline" id="SDS_PAGE_10">SDS PAGE</SPAN></H3><P>Of the cell growth media of BL21 (DE3) pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D with different concentrations. But still no clear secretion band of Lysostaphin was visible.</P><H3><SPAN class="mw-headline" id="SDS_PAGE_11">SDS PAGE</SPAN></H3><P>Of the cell lysate supernatant and cell lysate pellet of BL21 (DE3) T7-LacO-Lysostaphin-HlyA. There are bands that show Lysostaphin expression around 25 and 50 kDa.</P><H3><SPAN class="mw-headline" id="PCR_analysis_2">PCR analysis</SPAN></H3><P>PCR analysis was performed on the PCR run yesterday with pBAD, Pyocin 1 and Pyocin 2. Ran an agarose gel and analyzed the results with the UV reader. The pyocins both show a band at right height. pBAD did not show any band.</P><H3><SPAN class="mw-headline" id="PCR_3">PCR</SPAN></H3><P>Since the PCR analysis showed non-convincing results, it was performed again with pBAD (4691 bp, Tm = 60.1 °C) and Pyocin 1 and 2 (1494 bp, 50.2°C)</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Sequencing">Sequencing</SPAN></H3><P>Samples of Pyocin and lysostaphin from Stockholm 2016 are prepared for sequencing. </P><H3><SPAN class="mw-headline" id="Colony_PCR_5">Colony PCR</SPAN></H3><P>Colony PCR is performed of the colonies from September. The resulting DNA is analyzed with an agarose gel. </P> &quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of 5 new cubic gels with a cavity was started.</P>&quot;
<PRE>     }
   },
   &quot;Friday September 28, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_cavity_experiment_7">Gel cavity experiment</SPAN></H3><P>Used the first gel cavity experiment that was set up a week ago and performed the Performed the Gel washing (see iGEM Eindhoven protocols) to see if there were still bacteria within the gel.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="PAMM">PAMM</SPAN></H3><P>We could test our cell lysate from BL21 (DE3) T7-LacO-Lysostaphin-HlyA on Staphylococcus aureus. There was a clear dead zone, but we forgot to also apply a negative control.</P><H3><SPAN class="mw-headline" id="PCR_analysis_3">PCR analysis</SPAN></H3><P>PCR analysis was performed on the PCR run yesterday with pBAD, Pyocin 1 and Pyocin 2. Ran an agarose gel and analyzed the results with the UV reader. This time all constructs showed a band at the right size.</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Colony_PCR_6">Colony PCR</SPAN></H3><P>Colony PCR is performed of the colonies from September 26 with PET-T7-lysostaphin. The resulting DNA is analyzed with an agarose gel. </P><H3><SPAN class="mw-headline" id="Transformation_into_NovaBlue">Transformation into NovaBlue</SPAN></H3><P>The transformation of September 26 was repeated. </P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The polymerisation reaction of the cavity gels of 18-09-27 was quenched and the gel washed.</P>&quot;
<PRE>     }
   },
   &quot;Monday October 1, 2018&quot;:{
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Small_cultures_.2825_mL.29">Small cultures (25 mL)</SPAN></H3><P>Small cultures were made from BL21 (DE3) cells containing one of  the following plasmids:</P><UL><LI>pBAD-Lysostaphin-HlyA-His + pTetO-HlyB/D</LI><LI>pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D</LI><LI>pET24a-HlyA + pSTV-HlyB/D</LI><LI>T7-LacO-Lysostaphin-HlyA</LI></UL>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Colony_PCR_7">Colony PCR</SPAN></H3><P>Colony PCR of T7-lysostaphin in the pET 24a vector is performed. </P> &quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of new cube dextran hydrogels with a cavity and dextran hydrogel patches.</P>&quot;
<PRE>     }
   },
   &quot;Tuesday October 2, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_cavity_experiment_8">Gel cavity experiment</SPAN></H3><P>Performed a variant of the gel cavity experiment as described by iGEM Eindhoven protocols. This time, LB medium containing IPTG and CaCl was used as buffer. Old cultures of <B>mR15HA</B> and <B>WT15HA</B> were used.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Glycerol_stock_3">Glycerol stock</SPAN></H3><P>A 50% glycerol stock was made from BL21 (DE3) cells with the pET24a-HlyA + pSTV-HlyB/D plasmid.</P><H3><SPAN class="mw-headline" id="Small_cultures_.285_mL.29">Small cultures (5 mL)</SPAN></H3><P>Small cultures were made from BL21 (DE3) cells containing one of the following plasmids:</P><UL><LI>pBAD-Lysostaphin-HlyA-His + pTetO-HlyB/D</LI><LI>pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D</LI><LI>pET24a-HlyA + pSTV-HlyB/D</LI><LI>T7-LacO-Lysostaphin-HlyA</LI></UL><H3><SPAN class="mw-headline" id="PCR_purification_3">PCR purification</SPAN></H3><P>A PCR purification was performed on the PCR products of 27 September containing pBAD and pyocin.</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Miniprepping_4">Miniprepping</SPAN></H3><P>The pET and T7-lysostaphin in pET vector are miniprepped. Subsequently the pET vector is digested with NdeI and SacI. </P><H3><SPAN class="mw-headline" id="Ligation_and_transformation_5">Ligation and transformation</SPAN></H3><P>T7-lysostaphin in the pET 24a vector is transformed into competent NovaBlue cells. The T7-lysostaphin in the pSB1C3 vector is transformed into competent BL21 cells. </P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of new cube dextran hydrogels with a cavity. The polymerisation reaction of 18-10-01 was quenched and the gels were washed.</P>&gt;&quot;
<PRE>     }
   },
   &quot;Wednesday October 3, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_cavity_experiment_9">Gel cavity experiment</SPAN></H3><P>Took a sample of the experiment that was set up yesterday (see iGEM Eindhoven protocols). Performed OD measurement to see if the bacterial content in the medium changed.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Protein_induction_8">Protein induction</SPAN></H3><P>A protein induction was performed to check for the expression and secretion of lysostaphin. This was done with BL21 (DE3) cells containing the plasmids pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D, and pET24a-HlyA + pSTV-HlyB/D. Induction was done with 20 mM Arabinose, 0.2 mM IPTG and 5 mM of calcium (added after 3h of induction). After 4h of induction we washed the cells and did the French press. We took also samples of the cell media.</P><H3><SPAN class="mw-headline" id="DpnI_digestion">DpnI digestion</SPAN></H3><P>A DpnI digestion was performed on the pBAD and Pyocin 1 products from the PCR run on Friday.</P><H3><SPAN class="mw-headline" id="Gibson_Assembly">Gibson Assembly</SPAN></H3><P>A Gibson Assembly was done on the pBAD and Pyocin 1 products that have just been digested.</P><H3><SPAN class="mw-headline" id="Transformation_.28C2987_cells.29">Transformation (C2987 cells)</SPAN></H3><P>After Gibson Assembly, the pBAD + Pyocin 1 was transformed into C2987 cells which yielded no colonies after overnight incubation.</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Preparing_samples_for_sequencing">Preparing samples for sequencing</SPAN></H3><P>Samples of T7-lysostaphin and Pyocin are prepared for sequencing by BaseClear. </P><H3><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H3><P>The T7-lysostaphin is transformed into Novablue cells and BL21 DE3 cells. The transformed cells are plated on plates with chloramphenicol and incubated overnight at 37 °C. </P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of new cube dextran hydrogels with a cavity. The polymerisation reaction of 18-10-02 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   },
   &quot;Thursday October 4, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_cavity_experiment_10">Gel cavity experiment</SPAN></H3><P>Took a sample of the experiment that was set up two days ago (see iGEM Eindhoven protocols). Performed OD measurement to see if the bacterial content in the medium changed.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="SDS_PAGE_12">SDS PAGE</SPAN></H3><P>An SDS PAGE was performed on the BL21 (DE3) cells containing pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D, and pET24a-HlyA + pSTV-HlyB/D plasmids. There was no clear secretion since the calcium was added too late.</P><H3><SPAN class="mw-headline" id="Up_concentrating_the_cell_media">Up concentrating the cell media</SPAN></H3><P>Upconcentration was performed on BL21 (DE3) cells containing the pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D, and pET24a-HlyA + pSTV-HlyB/D plasmids. We added 2 times 15 mL of media to a 10x column and washed it twice with wash buffer (consisting of 15 mM Tris, 100 mM NaCl and 0.5 mM CaCl2)</P><H3><SPAN class="mw-headline" id="Transformation_.28C2987_cells.29_2">Transformation (C2987 cells)</SPAN></H3><P>The pBAD + pyocin 1 were transformed into C2987 cells, however this yielded no colonies due to the wrong antibiotics being used on the plates.</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Double_transformation_5">Double transformation</SPAN></H3><P>Both lysostaphin-HlyA and HlyB/D in the pET vector are transformed into competent BLR cells. The cells are plated and incubated overnight at 37 °C. </P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of new cube dextran hydrogels with a cavity. The polymerisation reaction of 18-10-03 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   },
   &quot;Friday October 5, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Making_cultures_for_for_experiments:_5">Making cultures for for experiments:</SPAN></H3><UL><LI><B>WT15HA</B> – Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain.</LI><LI><B>WT15HA NO SECRETION SYSTEM</B> - Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain, however this bacteria doesn’t have the plasmid to secrete the adhesin. </LI><LI><B>mRII18HA</B> - mRuby Adhesin with 18 repeats of region II and no binding domain (negative control).</LI></UL><H3><SPAN class="mw-headline" id="Fresh_transformation_with_adhesion_2">Fresh transformation with adhesion</SPAN></H3><P>A fresh transformation was done using the following plasmids:</P><UL><LI>R1R15HA - SBD adhesin with Kanamycin resistance.</LI><LI>PSTV HlyB/D - Secretion system type I with Chloramphenicol resistance</LI></UL><P>This was done following the iGEM Eindhoven transformation protocol. The double transformation was done once in E. Coli BLR Strain. However the double transformation failed.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Washed_the_cells">Washed the cells</SPAN></H3><P>To make sure no antibiotics are left on the cells, we washed them twice with LB.</P><H3><SPAN class="mw-headline" id="PAMM_2">PAMM</SPAN></H3><P>C2987 cells containing the pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D, and pET24a-HlyA + pSTV-HlyB/D plasmids were used to test the working of Lysostaphin. We took samples of the lysate, cell culture, cell media to see if the Lysostaphin is secreted and functioning. We took the construct producing and secreting HlyA as a negative control.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of new cube dextran hydrogels with a cavity. The polymerisation reaction of 18-10-04 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   },
   &quot;Monday October 8, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Making_cultures_for_for_experiments:_6">Making cultures for for experiments:</SPAN></H3><UL><LI><B>WT15HA</B> – Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain.</LI><LI><B>WT15HA NO SECRETION SYSTEM</B> - Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain, however this bacteria doesn’t have the plasmid to secrete the adhesin. </LI><LI><B>mRII18HA</B> - mRuby Adhesin with 18 repeats of region II and no binding domain (negative control).</LI></UL><H3><SPAN class="mw-headline" id="Fresh_transformation_with_adhesion_3">Fresh transformation with adhesion</SPAN></H3><P>A fresh transformation was done using the following plasmids:</P><UL><LI>R1R15HA - SBD adhesin with Kanamycin resistance.</LI><LI>PSTV HlyB/D - Secretion system type I with Chloramphenicol resistance</LI></UL><P>This was done following the iGEM Eindhoven transformation protocol. The double transformation was done once in E. Coli BLR Strain. However the double transformation failed.</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="PCR_4">PCR</SPAN></H3><P>A PCR was performed on the constructs pBAD-Lysostaphin-HlyA-His (3356 bp, 56.8 °C) and pSTV-HlyB/D (6516 bp, 59.9 °C). Our goal is to get both plasmids in one plasmid.</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Shipping">Shipping</SPAN></H3><P>Miniprepped samples of the constructs to be shipped are added to a 96-wells plate. The plate is dried overnight in a fumehood. </P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of new cube dextran hydrogels with a cavity. The polymerisation reaction of 18-10-05 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   },
   &quot;Tuesday October 9, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Making_cultures_for_for_experiments:_7">Making cultures for for experiments:</SPAN></H3><UL><LI>&lt;b?HlyA +BD&lt;/b&gt;Type 1 secretion system only. Only control so far with no adhesin at all.</LI><LI>Freshly transformed <B>WT15HA</B> – Wild type N-terminus (anchor) with 15 repeats of region II and a binding domain.</LI>&lt;/ul&gt;<H3><SPAN class="mw-headline" id="Gel_cavity_experiment_11">Gel cavity experiment</SPAN></H3><P>Performed the gel cavity experiment as described by iGEM Eindhoven protocols. This time, LB medium containing IPTG and CaCl was used as buffer. This experiment was done twice. Once for cultures of <B>induced WT15HA</B> and <B>uninduced WT15HA</B> were used. For the second experiment cultures of <B>induced WT15HA</B> and induced <B>HlyA +BD</B> were used.</P><H3><SPAN class="mw-headline" id="Gel_cavity_experiment_12">Gel cavity experiment</SPAN></H3><P>Took a sample of the experiment that was set today after a few hours (see iGEM Eindhoven protocols).</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="PCR_analysis_4">PCR analysis</SPAN></H3><P>A PCR analysis was performed on the constructs pBAD-Lysostaphin-HlyA-His, and pSTV-HlyB/D. We ran an agarose gel and analyzed the results with the UV reader. The pBAD had one clear band at the right seize, but pSTV-HlyB/D contained multiple bands.</P><H3><SPAN class="mw-headline" id="Gel_purification_2">Gel purification</SPAN></H3><P>A gel purification was performed on the construct pSTV-HlyB/D which yielded a concentration of 7 ng/uL.</P><H3><SPAN class="mw-headline" id="PCR_purification_4">PCR purification</SPAN></H3><P>A PCR purification was performed on the construct pBAD-Lysostaphin-HlyA-His which yielded a concentartion of 41.3 ng/uL.</P><H3><SPAN class="mw-headline" id="DpnI_digestion_2">DpnI digestion</SPAN></H3><P>The pBAD-Lysostaphin-HlyA-His construct was DpnI digested.</P><H3><SPAN class="mw-headline" id="Gibson_assembly_2">Gibson assembly</SPAN></H3><P>A Gibson Assembly was performed on the constructs pBAD-Lysostaphin-HlyA-His and pSTV-HlyB/D.</P><H3><SPAN class="mw-headline" id="Transformation_.28C2987_cells.29_3">Transformation (C2987 cells)</SPAN></H3><P>After the Gibson Assembly, the pSTV-HlyB/D and pBAD-Lysostaphin-HlyA-His were transformed in C2987 cells which yielded no colonies.</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Shipping_2">Shipping</SPAN></H3><P>The constructs are shipped in the 96-wells plate to the iGEM headquarters. </P><H3><SPAN class="mw-headline" id="Overnight_induction_of_T7-lysostaphin-HlyA">Overnight induction of T7-lysostaphin-HlyA</SPAN></H3><P>Cells with T7-lysostaphin-HlyA are induced with IPTG. </P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of new cube dextran hydrogels with a cavity. The polymerisation reaction of 18-10-08 was quenched and the gels were washed. additionally, new PDMS molds were ordered to further opskill the cavity gel synthesis.</P>&quot;
<PRE>     }
   },
   &quot;Wednesday October 10, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_cavity_experiment_13">Gel cavity experiment</SPAN></H3><P>Performed the gel cavity experiment as described by iGEM Eindhoven protocols. This time, LB medium containing IPTG and CaCl was used as buffer. This experiment was done twice. Cultures of <B>induced WT15HA</B> and induced <B>HlyA +BD</B> were used. This experiment was done in triplo.</P><H3><SPAN class="mw-headline" id="Gel_cavity_experiment_14">Gel cavity experiment</SPAN></H3><P>Took a sample of the experiment that was set up yesterday (see iGEM Eindhoven protocols). </P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="PCR_5">PCR</SPAN></H3><P>A PCR was performed on the pSTV-HlyB/D (6516 bp, 59.9 and 62 °C) construct.</P><H3><SPAN class="mw-headline" id="DpnI_digestion_3">DpnI digestion</SPAN></H3><P>A DpnI digestion was performed on the pSTV-HlyB/D 1 and pSTV-HlyB/D 2 constructs.</P><H3><SPAN class="mw-headline" id="PCR_purification_5">PCR purification</SPAN></H3><P>The constructs pSTV-HlyB/D 1  and pSTV-HlyB/D 2 were PCR purified which yielded 136 ng/µL  and 110 ng/µL respectively.</P><H3><SPAN class="mw-headline" id="Gibson_assembly_3">Gibson assembly</SPAN></H3><P>A Gibson Assembly was performed on the pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D 1, and pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D 2 constructs.</P><H3><SPAN class="mw-headline" id="Transformation_.28C2987.29">Transformation (C2987)</SPAN></H3><P>A transformation into C2987 cells was done for the pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D 1 (gave colonies), or pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D 2 (no colonies) constructs. </P><H3><SPAN class="mw-headline" id="PCR_analysis_5">PCR analysis</SPAN></H3><P>A PCR analysis was performed on the constructs pSTV-HlyB/D 1, and pSTV-HlyB/D 2. We ran an agarose gel and analyzed the results with the UV reader. Both contained a lot of different bands, however the band at the right size was very bright.</P>&quot;
<PRE>     },
     &quot;biobrick&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Colony_PCR_8">Colony PCR</SPAN></H3><P>Colonies with the sugar binding domain, lysostaphin-HlyA, pyocin-HlyA and pBAD-pyosin-HlyA in pSB1C3 are analyzed with colony PCR. The amplified DNA is analyzed on an agarose gel. </P><H3><SPAN class="mw-headline" id="Preparing_samples_for_sequencing_2">Preparing samples for sequencing</SPAN></H3><P>Samples for sequencing are prepared with pSB1C3 plasmids with sugar binding domain, lysostaphin-HlyA, pyocin-HlyA and pBAD-pyosin-HlyA. </P><H3><SPAN class="mw-headline" id="Ligation_and_transformation_T7-lysostaphin">Ligation and transformation T7-lysostaphin</SPAN></H3><P>The T7-lysostaphin in pTET 24a is transformed into BLR cells. The transformed cells are plated and incubated overnight at 37 °C. </P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of new cube dextran hydrogels with a cavity. The polymerisation reaction of 18-10-09 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   },
   &quot;Thursday October 11, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_cavity_experiment_15">Gel cavity experiment</SPAN></H3><P>Took samples of the experiments that were set up 2 days ago and yesterday (see iGEM Eindhoven protocols).</P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Colony_PCR_.28biobrick.29">Colony PCR (biobrick)</SPAN></H3><P>A colony PCR from 10 colonies was performed on C2987 cells containing pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D 1 plasmid.</P><H3><SPAN class="mw-headline" id="PCR_analysis_6">PCR analysis</SPAN></H3><P>A PCR analysis was performed on the pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D 1 construct. However, no bands were visible.</P><H3><SPAN class="mw-headline" id="Small_cultures_8">Small cultures</SPAN></H3><P>Small cultures were made of pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D and pET24a-HlyA + pSTV-HlyB/D.</P><H3><SPAN class="mw-headline" id="Inducer_mixture">Inducer mixture</SPAN></H3><P>An inducer mixture was prepared containing 1 mL IPTG (100 mM), 5 mL arabinose (2 M) and 1.25 mL calcium (2 M).</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Received 8 new PDMS molds for upscaling the synthesis of the cube dextran hydrogels with a cavity.</P>&quot;
<PRE>     }
   },
   &quot;Friday October 12, 2018&quot;:{
     &quot;Adhesin&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Gel_cavity_experiment_16">Gel cavity experiment</SPAN></H3><P>Took samples of the experiments that were set up a few days ago (see iGEM Eindhoven protocols). </P>&quot;
<PRE>     },
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Washed_the_cells_2">Washed the cells</SPAN></H3><P>To make sure no antibiotics are on the cells we wash them twice with LB.</P><H3><SPAN class="mw-headline" id="PAMM_3">PAMM</SPAN></H3><P>This time we wanted to test the diffusion of Lysostapin in our hydrogel. We used the gels with the cavity and loaded them with 7 μL of cells. Those cells did not contain the sugar binding protein. We saw that there was a dead zone around the gel but also some cells leaked out.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>Synthesis of new cube-shaped dextran hydrogels with a cavity.</P>&quot;
<PRE>     }
   },
   &quot;Sunday October 14, 2018&quot;:{
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Small_cultures_.28double.29">Small cultures (double)</SPAN></H3><P>Small cultures were made in duplicate from BL21 (DE3) cells containing pBAD-Lysostaphin-HlyA-His + pSTV-HlyB/D, pET24a-HlyA + pSTV-HlyB/D, or pET24a-Lysostaphin-HlyA + pSTV-HlyB/D plasmids.</P>&quot;
<PRE>     }
   },
   &quot;Monday October 15, 2018&quot;:{
     &quot;cloning&quot;:{
</PRE>
        &quot;html&quot;:&quot;<H3><SPAN class="mw-headline" id="Washed_the_cells_3">Washed the cells</SPAN></H3><P>To make sure no antibiotics are on the cells we wash them twice with LB.</P><H3><SPAN class="mw-headline" id="PAMM_4">PAMM</SPAN></H3><P>This time we wanted to test the diffusion of Lysostapin in our hydrogel and tested the secretion working of  pET24a-Lysostaphin-HlyA + pSTV-HlyB/D. We used the gels with the cavity and loaded them with 7 μL of cells. Those cells did not contain the sugar binding protein.</P>&quot;
<PRE>     },
     &quot;organiclab&quot;:{
</PRE>
        &quot;html&quot;:&quot;<P>The polymerisation reaction of 18-10-12 was quenched and the gels were washed.</P>&quot;
<PRE>     }
   }
 }
</PRE></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>