<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU-Eindhoven_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU-Eindhoven/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Medal_Requirements">Medal Requirements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Lab</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/InterLab">InterLab</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Results</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Results">Results Overview</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Lab/Hydrogel">Hydrogel</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Lab/Adhesin">Adhesin</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Lab/Bacteriocin">Bacteriocin</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Composite_Part">Composite Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Edu_n_Public_Engagement">Education &amp; Public Engagement</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Sponsors">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven/Attributions">Attributions</A></LI></DIV></DIV><DIV class="jumbotron neat model-heading"><DIV class="container"><H1 style="color:#FF8054">Modelling</H1><DIV class="hp-tabs" id="hpTop"><DIV class="btn-group" role="group" aria-label="hp-tabs"><BUTTON type="button" class="int-btn activeBtn btn btn-default" onclick="changeHPTabs(this)" data-target="#overview-container">Modelling Overview</BUTTON><BUTTON type="button" class="eth-btn btn btn-default" onclick="changeHPTabs(this)" data-target="#MI-container">Diffusion in Porous Media</BUTTON><BUTTON type="button" class="saf-btn btn btn-default" onclick="changeHPTabs(this)" data-target="#MII-container">Production of lysostaphin</BUTTON><BUTTON type="button" class="sta-btn btn btn-default" onclick="changeHPTabs(this)" data-target="#MIII-container">Patch Particle Transport</BUTTON></DIV></DIV></DIV></DIV><DIV class="container-fluid main-wrapper"><DIV id="overview-container" class="container hptabcontent integrated-content activeTab round white"><DIV class="row docs"><DIV class="col-xs-12 docs"><CENTER><H1>Overview</H1></CENTER><P>We established three models, one for modelling diffusion of lysostaphin, one for the bacterial production of lysostaphin per cell and the last one for a full model on our hydrogel patch, integrating the results of the two other models. The first model was supported by two diffusion experiments to determine diffusivity of lysostaphin in water and also diffusivity of lysostaphin in dextran hydrogel. For the second model, results of eradication of <EM>S. aureus</EM> were used to calculate the total production of lysostaphin during that experiment, and along with the cell count, the lysostaphin production per cell. In the end, the third model was modelled in COMSOL Multiphysics in 3D to simulate a layered structure consisting of a nutrient layer, cell layer and hydrogel layer. The diffusivity values through the gel of model 1 and the lysostaphin production per cell of model 2 were implemented to make statements about how the gel should be composed or any future insights on the future development of the gel.</P><P>After thinking about a fitting shape for our hydrogel, model results were used to adapt the hydrogel thickness to suit our wound healing application. We modelled which patch thickness would be feasible to create a patch that is as thin as possible, but also allows enough lysostaphin to reach the skin to eradicate <EM>S. aureus</EM> biofilm using experimental input as lysostaphin production rate and hydrogel parameters describing the microstructure. Our model showed  that a patch with a thickness of the order of magnitude of approximately a millimeter allows a high enough concentration of lysostaphin to eradicate <EM>S. aureus</EM>. Subsequently, we adapted our hydrogel patch design to synthesize a hydrogel patch with a thickness of 0.75 mm. We could show that this hydrogel could be easily created and yielded flexible, stable hydrogels. Furthermore, viewing the future of our hydrogel patch, we think a hydrogel, lysostaphin producing layer, with a removable nutrient layer would be convenient to nourish the cells. The removable layer makes it easily replacable, decreasing the amount of times the total gel has to be removed which is usually painful for the patient. </P></DIV></DIV></DIV><DIV id="MI-container" class="container hptabcontent ethics-content round white"><DIV class="row"><DIV id="introDiv" class="col-xs-12 introduction"><H1 style="text-align: center;">Diffusion in Porous media</H1><P><CENTER><I>What is the effect of a hydrogel on the diffusion of particles in a medium?</I></CENTER></P><H2>Summary</H2><P>The processes of diffusion in a porous media can be related to properties of the porous media. There relations were modelled to allow us to find what the effect was of changing properties of the hydrogel on the effective diffusion of lysostaphin within our gel. By measuring the effective diffusion of the enhanced green fluorescent protein (eGFP, 26.9 KDa) within our Dextran hydrogel using an experiment, we gained insights in the transport behaviour of lysostaphin (27 KDa). We estimated the tortuosity of our hydrogel to be between 1 and √1.5, and could conclude that timescales for the transport processes in our patch in the range of 24 hours.</P><P>Future research in the design of the patch could lead to a preference for another effective diffusion constant in the hydrogel. Using both the model and the experiment we have the tools to estimate what the properties of the hydrogel should be and to validate this effective diffusion constant is indeed what we would have expected.</P></DIV></DIV><DIV class="row"><DIV class="col-xs-12 col-sm-12 col-md-3 col-lg-2 col-xl-2 toc-column"><DIV class="toc-wrapper"><LI><UL id="tableOfContent"><P>The best that most of us can hope to achieve in physics is simply to misunderstand at a deeper level.</P></UL><A class="backToTop" href="#hpTop">▲</A></LI></DIV></DIV><DIV class="col-xs-12 col-sm-12 col-md-9 col-lg-10 col-xl-8 docs"><SECTION><H2>Introduction</H2><P>The effective diffusivity (D<SUB>e</SUB>) of particles being transported through a porous structure is a certain liquid is smaller than particles being transported in the same liquid but without porous structure (D<SUB>0</SUB>). The ratio D<SUB>e</SUB> / D<SUB>0</SUB> ,often referred to as the diffusibility, can be very different from one porous medium to the other. Therefore it makes sense to relate the diffusibility to variables that characterize the porous medium in multiples ways. A simple way to model this is by relating D<SUB>e</SUB> / D<SUB>0</SUB> to the porosity (ɛ<SUB>t</SUB>), constricitivy (δ) and tortuosity (τ) following equation 1 as is described in (1).</P><P>The porosity of a porous material is defined as the volume fraction of the vacancies and the total volume, V<SUB>v</SUB> / V<SUB>T</SUB>. The tortuosity is a parameter of a porous medium that relates to the shortest paths or main paths where diffusion takes place. The constrictivity relates to the change in cross-section of a segment over its length. So whereas the constrictivity can be determined locally, the tortuosity is a parameter that describes a region.</P><P>Knowledge about the behaviour of the diffusibility regarding the three parameters described above gives us insight in how a desired effective diffusion constant De can be obtained for our hydrogel. This is relevant because the effective diffusion constant De describes the main transport process in the hydrogel patch that will be used to transport lysostaphin to the skin in order to fight <I>Staphylococcus aureus</I>.</P><P>The volume fraction of the hydrogel is found to be 0.9 (2). For a volume fraction above 0.4 it can be assumed that the constricitivy equals 1 (1). Since we will not be investigating hydrogels with low volume fractions we can take the constrictivity out of the formula resulting in equation 2. Since we know the diffusion of eGFP in free medium (3) to be 0.87 e-10 m<SUP>2</SUP>/s we can calculate the tortuosity once we have experimentally obtained a value for the effective diffusivity.</P></SECTION><SECTION><H2>Experiment</H2><P>To determine the hydrogel properties an experiment was conducted to measure the effective diffusion of eGFP in the hydrogel with water as medium. Using eGFP (26.9 KDa) is relevant as its weight is similar to that of lysostaphin (27 KDa).</P><P>A cylindrical piece of hydrogel was submerged in eGFP for a day so it would contain eGFP. This piece was then aligned at the circular cross-sections with another piece of hydrogel. Over time eGFP diffuses into the piece of hydrogel where no eGFP is present. The speed of this process depends on the effective diffusion constant. Therefore pictures of the gel were made using a fluorescence camera at different timesteps, after excitation of the GFP at 480 nm UV light exposing for 1 second. These pictures where then analysed digitally. Longitudinal plots were obtained for each timestep, by averaging the intensity perpendicular to the longitudinal direction. The process of the experiment and the way of measuring the images are illustrated in figure 1.</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;"><B>Figure 1:</B> A Illustration of the experiment to determine the diffusion, showing the initial situation, the process of diffusion resulting in changing concentrations of eGFP overtime and the way the measurements were processed.</P></DIV></DIV></DIV></SECTION><SECTION><H2>Results</H2><P>Results of the sample were obtained at 6 timepoints and can be seen in figure 2. As can be seen, not all pictures where taken with the same settings of the fluorescence microscope. This was because other people had to use the device as well, in-between measurements settings might have been adjusted. Therefore the measurements needed to be normalized in order to be able to compare them.</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;"><B>Figure 2:</B> 6 Fluorescence photos of the Dextran-based hydrogel containing GFP that diffuses overtime.</P></DIV></DIV></DIV><P>Using FIJI software (previously known as ImageJ) a longitudinal intensity plot of the hydrogel was made. This data was later processed in MATLAB to ensure the samples to have the same offset and also an equal scaling. The result of these steps can be seen in figure 3. As can be seen, the samples have the same offset outside the hydrogel (0-0.1 mm). Also the plateaus of all samples, around 1mm were set to have the same intensity, because it was expected that this would not change throughout the experiment.</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;"><B>Figure 3:</B> Averaged longitudinal sections of the fluorescence intensity of the experiment for different timesteps.</P></DIV></DIV></DIV><P>In figure 3 it can be seen that the measurement combined with the method of rescaling is reasonably accurate, since the measurement at 3 hours and 50 minutes is fairly equal to the measurement after 4 hours. While performing the experiment, it became clear that after more than 20 hours the gel suffered from dehydration, hence they should not be included in the determination of the effective diffusion constant. Also were the results not entirely what was expected, because the gel that was seeded did not illuminate homogeneously. It was therefore hard to process the data in order to obtain a diffusion constant.</P><P>Nevertheless this was tried for the samples taken after 0, 3:50, 4 and 20 hours, by looking at the behaviour of the concentration indicated with red lines at figure 4. Behaviour of the concentration along this line overtime was investigated using COMSOL Multiphysics, by comparing simulations with the results displayed in figure 4. From comparison with these simulations it became clear that the tortuosity of the hydrogel is between 1 and √1.5.</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;"><B>Figure 4:</B> Most relevant results of the diffusion experiment. Red lines indicate the parameterisation that is used to compare the data to COMSOL Multiphysics simulations.</P></DIV></DIV></DIV></SECTION><SECTION><H2>Discussion</H2><P>There seem to be a lot of things that can be approved in this experiment. Figure 3 and 4 clearly show an irregular concentration of eGFP being present in the hydrogel part that has been seeded. It was seeded by submerge it in a eGFP solution overnight. The irregularity can likely be resolved by submerging the hydrogel block longer in eGFP so in can be guaranteed that the centre of the hydrogel block has the same concentration eGFP as the outer regions. Another way to improve the experiment is by having a clearer cut. The surfaces of the hydrogel parts with whom there were aligned were quite rough and did not have contact along the entire surface. It is likely that this has influenced the results. The final way to improve the experiment is by making sure the hydrogel does not dry out overtime. This would extend the measuring time past the current limit of 20 hours. A way to do this could be by protecting the gel by encapsulating it better during storage and exposing it to the environment for a shorter amount of time. Also a more humid environment is expected to help. Adding new fluid to the gel will not be a good idea, since this will convection into the hydrogel.</P><P>From the combination of experiments and modelling we could estimate that the tortuosity of our hydrogel should lie somewhere between 1 and √1.5. In order to safely conclude on an effective diffusion constant of eGFP in our Dextran hydrogel the experiment still needs to be improved upon.</P></SECTION><SECTION><H2>Conclusion</H2><P>By using eGFP, which has a similar weight to that of lysostaphin gained insight in transport of lysostaphin. We saw that diffusion times of lysostaphin within our patch will lie in the range of 24 hours. It can be concluded that the experiment that was done is a promising way to measure the effective diffusion constant of a particle in a hydrogel. In order to improve the accuracy of the experiment one should let the source part of the hydrogel incubate longer than one day with eGFP. Also cleaner cuts in the gel should be made. </P></SECTION><SECTION><H2>References</H2><OL class="reference"><LI id="sbdref1">Brakel JVAN, Heertjes PM. ANALYSIS OF DIFFUSION IN MACROPOROUS MEDIA IN TERMS OF A POROSITY , A TORTUOSITY AND A CONSTRICTIVITY FACTOR. Int J Heat Mass Transf [Internet]. 1974;17:1093–103. Available from: https://doi.org/10.1163/156856206778530722</LI><LI id="sbdref2">Plieva F, Oknianska A, Degerman E, Galaev IY, Mattiasson B. Novel supermacroporous dextran gels. J Biomater Sci Polym Ed. 2006;17(10):1075–92.</LI><LI id="sbdref3">Dayel MJ, Hom EFY, Verkman AS. Diffusion of Green Fluorescent Protein in the Aqueous-Phase Lumen of Endoplasmic Reticulum. Biophys J [Internet]. 1999;76(5):2843–51. Available from: http://dx.doi.org/10.1016/S0006-3495(99)77438-2</LI></OL></SECTION></DIV></DIV></DIV><DIV id="MII-container" class="container hptabcontent safe-content round white"><DIV class="row"><DIV id="introDiv" class="col-xs-12 introduction"><H1 style="text-align: center;">Production of lysostaphin</H1><P><CENTER><I>How much lysostaphin is produced by a single cell?</I></CENTER></P><H2>Summary</H2><P>A model based on Fick’s law for diffusion and the mass balance equation was proposed to determine production rates of lysostaphin produced by <I>Escherichia coli</I>, based on experimental data. This experimental data consisted of modified <I>E. coli</I> cultures that were applied to agar plates containing <I>Staphylococcus aureus</I>. The modified <I>E. coli</I> produced and secreted lysostaphin overtime, which eradicated the <I>S. aureus gradually</I>, creating a band around the <I>E. coli</I> colony. The radius of those bands were measured and used as model input determining the boundary conditions of differential equations used. Integration of the concentration profile determined the total amount of produced lysostaphin. By including a cell count, we were able to determine the lysostaphin production rate for an individual cell. This was however expected to be inaccurate, due to the inability of tracking the number of cells overtime. Nevertheless a reasonable production rate was found and the total production of lysostaphin for a single colony could be determined.</P></DIV></DIV><DIV class="row"><DIV class="col-xs-12 col-sm-12 col-md-3 col-lg-2 col-xl-2 toc-column"><DIV class="toc-wrapper"><LI><UL id="tableOfContent"><P>The best that most of us can hope to achieve in physics is simply to misunderstand at a deeper level.</P></UL><A class="backToTop" href="#hpTop">▲</A></LI></DIV></DIV><DIV class="col-xs-12 col-sm-12 col-md-9 col-lg-10 col-xl-8 docs"><SECTION><H2>Introduction</H2><P><I>Escherichia coli</I> will be used as factory to produce lysostaphin. The amount of lysostaphin a single cell produces is relevant, because there is a certain concentration needed in order to eradicate <I>Staphylococcus aureus</I>. This concentration is referred to as the minimum biofilm eradication concentration or MBEC in short (1). It is therefore important that the concentration within the patch exceeds this concentration there where the patch touches the skin.</P><P>A model was designed that estimates the production of lysostaphin for a single <I>E. coli</I> bacteria from experimental data. This model will uses basic physics in order to do an estimate of the total amount of lysostaphin that was produced by the culture. This can be done by combining Fick’s law for diffusion with the mass balance equation and assuming a steady state situation, where there is as much production of lysostaphin as is diffusing out of the system. Such a steady state situation can be observed by a stationary profile of the concentration. This profile will be evaluated in cylindrical coordinates and integrated to obtain a total amount of lysostaphin.</P><P>By estimating the number of cells that were present during the experiment we can also get an estimate for production rate of lysostaphin for an individual <I>E. coli bacteria</I>. In later research we can use these estimates to adjust the design of our hydrogel patch, on order to make sure that enough lysostaphin is present at the wound surface, while ensuring that as few cells are present in the cell as possible. This is of importance since a larger amount of cells will likely increase the risk of bacterial escape.</P></SECTION><SECTION><H2>Experiment</H2><P>Initially an experiment was conducted where an amount of medium containing <I>E. coli</I> cells was added to an agar plate containing <I>S. aureus</I>. Over time the E. coli produced lysostaphin, which eradicates nearby <I>S. aureus</I>. In figure 1 it can be seen that this forms a circle around the <I>E. coli culture</I>. By modelling this region one could obtain the rate of lysostaphin production for this <I>E. coli culture</I>. If the amounts of cells in this culture is known, also the production rate for a single cell could be determined. From the experiments shown in figure 2 parameters were obtained. Those are listed in table 1.</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;">Figure 1: An illustration showing an agar plate seeded with S. aureus in yellow. The inner circle represents the E. coli colony that was seeded and the orange ring the lysostaphin that was excreted by the E. coli. Overtime the band of lysostaphin will grow to a maximum size.</P></DIV></DIV></DIV><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;">Figure 2: Drops of modified E. coli that produce  and excrete lysostaphin, on a agar plate containing S. aureus. The bands around indicate that S. aureus is dissolved. At the inner band the concentration is set to be equal to MBEC. At the outer band the concentration is set at zero.</P></DIV></DIV></DIV><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><A class="figureModal"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;"><B>Table 1:</B> a listing of all the parameters used in this model. Cell stain diameter A, MBEC diameter B, Diameter where lysostaphin is present C, the pipetted volume of cell culture medium V<SUB>p</SUB>, The OD 600 value of the cell culture medium OD, the total number of E. coli cells N<SUB>c</SUB>, the source of lysostaphin concentration S<SUB>c</SUB>.</P></DIV></A><TABLE style="width:80%" class="figure"><TBODY><TR><TH>Colony</TH><TH>A [mm]</TH><TH>B [mm]</TH><TH>R [mm]</TH><TH>V<SUB>P</SUB> [μl]</TH><TH>OD [3E-8 ml<SUP>-1</SUP>]</TH></TR><TR><TD>1</TD><TD>5.7</TD><TD>7.1</TD><TD>8.6</TD><TD>5</TD><TD>1.22</TD></TR><TR><TD>2</TD><TD>5.6</TD><TD>7.0</TD><TD>8.9</TD><TD>5</TD><TD>1.22</TD></TR><TR><TD>3</TD><TD>4.8</TD><TD>6.4</TD><TD>7.8</TD><TD>3</TD><TD>1.22</TD></TR><TR><TD>4</TD><TD>3.8</TD><TD>5.2</TD><TD>6.9</TD><TD>3</TD><TD>1.22</TD></TR></TBODY></TABLE></DIV></DIV></SECTION><SECTION><H2>Model</H2><P>First we assume that the region that is affected by the lysostaphin is perfectly round, hence we can use cylindrical coordinates. The concentration of lysostaphin is assumed to be characterized by only diffusion and Fick’s law (eq. 1) can be applied.</P><P>By assuming  a steady state situation we can use the equation for mass balance (eq. 2) where we take the source term S(r) to be dependent on the radius, we can solve for this source term ones we have enough boundary conditions available. The source term will be modelled as a step function being equal to S<SUB>c</SUB> for radius r &lt; a, and 0 for a &lt; r &lt; R (eq. 3).
            </P><P>By substituting Ficks law into Mass balance equation we can solve the differential equation for both source terms separately. For r &lt; a this is done in equations 4-7, where C<SUB>a</SUB> is a constant that still needs to be determined. The equation is rewritten in such a way that C<SUB>a</SUB> will correspond to the concentration at r = a.</P><P>For the region a &lt; r &lt; R the approach is similar. We obtain equation 7, where we still need to determine integration constant C<SUB>2</SUB>.</P><P>In order to determine both C<SUB>a</SUB> and C<SUB>2</SUB> we make use of the fact that the concentration is continuous in r = a and also its derivative is continuous is r = a. This is done in equation 8-12.</P><P>Since we know the concentration at b, namely the MBEC concentration c<SUB>MBEC</SUB> we can plug this in in order to obtain a relation that describes the concentration of lysostaphin throughout the entire radius (eq. 13).</P><P>A direct relation for the source of lysostaphin S<SUB>c</SUB> can be found, as shown in equation 14. From this relation we can find a value for the production of lysostaphin in mg/m<SUP>3</SUP>s. This is not yet the production rate of one single E. coli bacteria. For this we will also be needing the number of cells that were seeded in the experiment.</P><P>To determine the number of <I>E. coli</I> bacteria present on the plate, we first need to assume that the <I>E. coli</I> do not proliferate during the experiment, and that the optical density that was measured before applying the cell medium is a proper indication of the amount of cells that are present. Since this might not be the case, we will keep this in mind in later steps of determining the production rate of lysostaphin. For the number of cells at the beginning of the experiment (N<SUB>c_init</SUB>) we can write down equation 15, where the optical density (OD) is multiplied by 3.7E8 (2) in order to the amount of cells in mg<SUP>-3</SUP>. The applied cell medium (VP) is used in µl. Because the <I>E. coli</I> will also divide during the plating we will assume that the number of <I>E. coli</I> cells (N<SUB>e</SUB>) will rise to a saturated state at an OD 600 of 8, relates to 0.75E10 cells/ml (3) and that this amount is representable during the entire experiment.</P><P>Now there are two ways to determine the production rate of lysostaphin for a single cell. The first one is to use equation 14 and 15 resulting in equation 16, which look rather straightforward, but probably result in low numbers than expected. This is logical because of the low diffusion rate and the fact that we assumed to concentration shape to be constant (steady state situation). In practise both of these assumptions are likely to be wrong due to two reasons. Firstly, after applying a drop of cell medium it is likely that some convection took place and had spread more lysostaphin than you would expect from only diffusion processes. The second one is that this experiment was done overnight (16 hours). For a diffusion constant of 0.78E-10 m<SUP>2</SUP>/s (4) and a spatial scale of  centimetres this is too little time for a steady state to occur.</P><P>A better approach would be to assume that the shape of the concentration is more or less correct and then integrate the concentration over the entire surface and multiply it with the thickness of the layer. This would give an idea what the entire colony of <I>E. coli</I> had produced in a 10 hour period (P<SUB>T</SUB>), which could in turn be divided by the time and the amount of cells to find a production rate of lysostaphin per second, per hour. This was done in equation 17-24.</P><P><CENTER>Integration by parts gives us:</CENTER></P></SECTION><SECTION><H2>Results</H2><P>The parameters from the experiment were fed into equation 18. Also was MATLAB used to visualize the behaviour of the concentration along the radius, for all four samples. This is visualized in figure x. It can be seen how for r &lt; a the concentration follows a r<SUP>2</SUP> behaviour, while for a &lt; r &lt; R the concentration follows an ln(1/r) profile.</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;">Figure 3: The concentration of all four samples, based on the parameters listed in table 1.</P></DIV></DIV></DIV><P>For all the samples the number of cells, total lysostaphin production and lysostaphin production per single cell were also obtained and can be found in table 3.</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><A class="figureModal"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;"><B>Table 2:</B> Results obtained from the model.</P></DIV></A><TABLE style="width:100%" class="figure"><TBODY><TR><TH>Sample</TH><TH>N [-]</TH><TH>Total production [mg]</TH><TH>Productionrate per bacteria P<SUB>C</SUB> [mg/s]</TH></TR><TR><TD>1</TD><TD>4.51E7</TD><TD>3.25E-3</TD><TD>2.41E-15</TD></TR><TR><TD>2</TD><TD>4.51E7</TD><TD>2.79E-3</TD><TD>2.07E-15</TD></TR><TR><TD>3</TD><TD>2.71E7</TD><TD>2.10E-3</TD><TD>2.59E-15</TD></TR><TR><TD>4</TD><TD>2.71E7</TD><TD>1.73E-3</TD><TD>2.141E-15</TD></TR></TBODY></TABLE></DIV></DIV><P>The mean production rate of lysostaphin for a single <I>E. coli</I> in our experiment was found to be 2.30E-15 mg cell<SUP>-1</SUP> s<SUP>-1</SUP>. Determination though the first method, where steady state and purely diffusion was required would estimate the production rate to be 1.76E-15 mg cell<SUP>-1</SUP> s<SUP>-1</SUP>, which is slightly lower that the first measurement.</P></SECTION><SECTION><H2>Discussion</H2><P>Determination of the number of cells for a given colony has been proven to be difficult. Measuring the optical density of a suspension might give a good indication for the initial cell concentration, but after applying the <I>E. coli</I> on the agar LB plate the <I>E. coli</I> continues to divide. This can be observed on the agar plate over time with the naked eye. Besides cell division a part of the explanation might be the growing of the individual cells over time (5), hence the cell counting by tracking the expanse of the culture is more difficult.</P><P>Even though the determination of the number of cells was inaccurate we can still get an idea of what the production rate of lystostaphin for a single cell could be. Results of method 1, were steady state was assumed, in the end did not differ much from the results obtained by integration.</P><P>The reaction rates per cell from this experiment are expected to give a fairly good insight in what the thickness of the hydrogel will need to be, therefore the production rate will be implemented in the finite element model that describes transport of lysostaphin to the skin. </P><P>Finally, the total amount of lysostaphin that was produced by the colony during the entire experiment, does not depend on the cell count itself and is therefore considered to be more accurate. To really prove the accuracy of the setup and model a similar approach should be investigated for different sizes, incubation times. Also comparison with different approaches to determine this total production of lysostaphin could give insight in the accuracy of this approach that was used.</P></SECTION><SECTION><H2>Conclusion</H2><P>Due the poor accuracy of tracking the numbers of cells during the experiment no accurate production rate could be obtained. This result is nevertheless a reasonable assumption of the real production rate, that in reality also varies over time a per cell and could be used for further exploration. The total amount of produced lysostaphin by the <I>E. coli</I> colony is expected to be more accurate since the calculation does not depend on determination of the amount of cells.</P></SECTION><SECTION><H2>References</H2><OL class="reference"><LI id="sbdref1">Jones RN, Farrell DJ, Helio SS. Lysostaphin and clarithromycin: a promising combination of the eradication of Staphylococcus aureus biofilms. Int J Antimicrob Agents. 2011;37:585–7.</LI><LI id="sbdref2">Eppendorf. OD600 Measurements Using Different Photometers. 2015;(28).</LI><LI id="sbdref3">Sezonov G, Joseleau-Petit D, D’Ari R. Escherichia coli physiology in Luria-Bertani Broth. J Bacteriol. 2007;189(23):8746–9. </LI><LI id="sbdref4">Dayel MJ, Hom EFY, Verkman AS. Diffusion of Green Fluorescent Protein in the Aqueous-Phase Lumen of Endoplasmic Reticulum. Biophys J [Internet]. 1999;76(5):2843–51. Available from: http://dx.doi.org/10.1016/S0006-3495(99)77438-2</LI><LI id="sbdref5">Martijn Wehrens, Dmitry Ershov, Rutger Rozendaal, Noreen Walker, Daniel Schultz, Roy Kishony, Petra Anne Levin, Sander Tans. Size Laws and Division Ring Dynamics in Filamentous Escherichia coli cells. Curr Biol [Internet]. 2018;28(6):972–9. Available from: https://www.sciencedirect.com/science/article/pii/S096098221830160X?via%3Dihub</LI></OL></SECTION></DIV></DIV></DIV><DIV id="MIII-container" class="container hptabcontent stakeholder-content round white"><DIV class="row"><DIV id="introDiv" class="col-xs-12 introduction"><H1 style="text-align: center;">Patch particle transport</H1><P><CENTER><I>With what density should the hydrogel be seeded with <I>E. coli</I> bacteria?</I></CENTER></P><P><CENTER><I>What is the optimal composition of the patch to maximize lysostaphin delivery and minimize nutrient delivery to  the exudate?</I></CENTER></P><H2>Summary</H2><P>For a hydrogel patch with a fitting shape to cover a wound on the arm, COMSOL was used to simulate the transport of lysostaphin and nutrients in this patch. Parameters, variables and boundary conditions were set according to literature, previous models and experiments on diffusion and lysostaphin production. It was found that a patch with a thickness of around a millimeter lysostaphin definetely reached the skin but in concentrations too low to eradicate <I> S. aureus</I> on the wound. This might be caused by a boundary condition we set on the bloodstream, as we set concentration to zero because blood would flow away from the wound with the lysostaphin. Later we realised this is probably not realistic since proteins diffuse very poorly through the line of cells forming the bloodvessel</P><P>With nutrients having higher diffusivity constants than lysostaphin, nutrients feeding our infected wound seemed a problem at first glance. However, we eventually concluded that the amount of nutrients in exudate neglects the nutrients flowing from applied nutrients to the infected wound, not causing problems. </P></DIV></DIV><DIV class="row"><DIV class="col-xs-12 col-sm-12 col-md-3 col-lg-2 col-xl-2 toc-column"><DIV class="toc-wrapper"><LI><UL id="tableOfContent"><P>The best that most of us can hope to achieve in physics is simply to misunderstand at a deeper level.</P></UL><A class="backToTop" href="#hpTop">▲</A></LI></DIV></DIV><DIV class="col-xs-12 col-sm-12 col-md-9 col-lg-10 col-xl-8 docs"><SECTION><H2>Introduction</H2><P>This simulation was designed to investigate the amount of lysostaphin and nutrients present at the line where the hydrogel ends and wound begins for different designs of patches. In order to effectively eradicate the <EM>Staphylococcus aureus</EM> culture that is present in the wound, it is important that the amount of lysostaphin exceeds a certain concentration at the wound. Furthermore, nutrients should not diffuse to the wound since not only our engineered <EM> E. coli</EM> likes nutrients, but also harmful bacteria in the wound will thrive well on them.</P><P>For convenience’s sake, we decided it to be the easiest if we feed the <EM>E. coli</EM> by adding a nutrient rich layer on top of the hydrogel patch. This way the nutrient rich layer can be easily replaced once depleted. Because replacement requires the help of a specialist like a nurse, we aim to maximize the time a patch can stay on the skin without anyone touching it. We will try to find an optimal composition of the hydrogel patch, so no nutrients diffuse entirely through the hydrogel to reach the malignant bacteria. However, this should not inhibit the delivery of lysostaphin to the infected wound.</P></SECTION><SECTION><H2>Model</H2><P>COMSOL Multiphysics 5.3a was used to simulate the transport of lysostaphin and nutrient particles within a porous medium. COMSOL uses the finite element method (FEM) to calculate the value of variables within an element for a given timestep. COMSOL allows the user to easily select physical processes, such as diffusion. Also, one can simply construct a grid, set boundary conditions and select simulation times. Its simplicity made the choice for us to use this program very easy.</P><P>The physical properties included in the simulation are diffusion following Ficks law (equation 1), and an equation describing the effective diffusivity of particles within a porous medium (equation 2). In this equation the diffusivity (D<SUB>e</SUB>/D<SUB>o</SUB>) scales with the inverse of the porosity, while we used a quadratic inverse relation with the porosity in the first model, as was argued to be correct in (1). In order to avoid confusion between the tortuosity used in COMSOL and the tortuosity determined in the first model, the tortuosity in COMSOL will be referred to as τ<SUB>c</SUB> where equation 3 holds.</P><P>In the first model, where we experimented with the diffusivity of GFP, we got a tortuosity (τ) value between 1 and √1.5. For this model we will use the average of these values, which leads to τ = √1.237 and τ<SUB>c</SUB> = 1.237. </P><H3>Patch composition</H3><P>First the shape of the hydrogel was chosen to be a square 30x30 mm block. We imagine it being used for an infected wound on the arm. Its total thickness was chosen to have a value of 4 mm, including the nutrient rich layer. Our decision fell on this thickness, because you do not want a very thick layer of hydrogel on your skin as a patient. That would be inconvenient and more prone to being damaged by shear stresses when wearing clothing for instance. We compose the patch of three different layers as can be seen in figure 1. The top layer is the one containing nutrients that nourishes the cells and allows growth of the cells and lysostaphin production by the cells. However, the most important layer, namely the layer that houses the lysostaphin secreting <EM>E. coli</EM> cells, can be found on a buffer layer. A buffer layer of hydrogel sits between the cell layer and the wound. This buffer layer should be small enough to ensure a sufficient amount of lysostaphin flowing from <EM>E. coli</EM> cells towards the wound, but the buffer layer should be large enough, along with the gel where the cells reside, to ensure that as little nutrients as possible end up in the wound by diffusion.</P><P>In physics, displacement of molecules can be influenced by diffusivity as said above, but flux can also have influence on this displacement. Through movement of the patch or forces on the gel, while it is on the skin of a patient, flux can be generated. However, these external influences are small and unpredictable. For this reason they are difficult to put in a model. So for simplicity we assume an ideal situation where there are not any external forces or movements that cause flux through the gel.</P><H3>Skin</H3><P>Since we want to simulate our patch on top of the skin, we also had to simulate a piece of skin in COMSOL. However, we assumed that the wound is not just a superficial wound, but an open wound because superficial wounds are generally able to heal by themselves and are not easily infected. So we assumed that the wound was open and covered with exudate. Moreover, as we are working with an open wound, the blood vessels are extremely close to the surface if not just separated by a single line of cells from the exudate. Because of the low amount of cells, we assumed the permeability constant of the skin to be negligible. In conclusion, the diffusivity through both exudate and blood were assumed to be equal to water due to the high water contents (approximately 92% for blood plasma (2)) and our target, <EM>S. aureus</EM>, thrives in exudate with high water content. (3) Furthermore, for the reason that blood flows to the wound and away from the wound we set a boundary where the blood is. This boundary works as a sink in the sense that nutrient and lysostaphin concentration are zero here, because there is flux away from the wound additional to diffusion. </P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;">Figure 1: Structure of the 3D model in COMSOL Multiphysics. The hydrogel parts are shown in grey, from top to bottom: nutrient layer, cell layer, buffer layer. The layers fulfilling the goals of exudate and blood are the top and bottom blue layers respectively.</P></DIV></DIV></DIV><H3>Cell layer</H3><P>As can be seen in the second model on lysostaphin production, we assumed a completely saturated hydrogel for cells to get a lysostaphin production rate per cell. For this model we do not assume growth of cells, actually, we assume that the cells’ growth and death rate are in equilibrium, establishing a steady state where there is a constant number of cells. Since we obtained a lysostaphin production rate per cell and nutrient uptake rate, along with diffusion constants, we can vary the amount of cells in the gel and the thickness of the gel to get results on possibly fitting values. </P><P>As can be seen further in this model, due to the fact that we vary the thickness of the buffer layer and not the cell layer, we argue that for convenience in handling a minimum thickness of 1 mm of the patch is needed (excluding the nutrient layer). This means that the minimum thickness is equal to the thickness of the cell layer, because the buffer layer’s thickness can be zero. Since using 1 mL as the cell layer’s volume eases calculations a lot, we fixed the thickness of the cell layer to 1.111 mm causing the cell layer to have an exact volume of 1 mL with both other dimensions being 30 mm.</P><H3>Nutrient layer</H3><P>The parameters in Table 1 will be used in the COMSOL Multiphysics model. We will be using the concentration of nutrients of standard LB Broth (Miller) which we also use in our laboratory. This nutrient-rich layer has to be able to be easily replaced by a new nutrient layer. Just like we argued for the cell layer in the patch, handling of this layer should be made easy by its thickness to enable replacement. Therefore we chose to use a fixed thickness of 1 mm of this layer.</P><H3>Parameters</H3><P>Using literature, calculations and results from previous models and experiments, we came up with parameters and variable ranges for our simulation in COMSOL, which can be found in table 1 and 2 respectively. </P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;">Table 1: Parameters used for the 3D model in COMSOL Multiphysics.</P></DIV></DIV></DIV></SECTION><SECTION><H2>Calculation 1</H2><P>To decide what nutrient source we would use, we decided to look at commonly used nutrient sources for the bacteria we use in our hydrogel: <EM>E. coli</EM>. For general laboratorial growth of <EM>E. coli</EM> often Lysogeny broth (LB) medium is used, so we decided to take this nutrient rich medium as a first source to feed our bacteria.</P><P>The LB we use in the lab of our university is called ‘LB Broth (Miller)’ from Sigma-Aldrich. As stated on their website, this type of LB consists of 10 g/L tryptone, 10 g/L NaCl and 5 g/L yeast extract. (8) We neglected the mass of NaCl, since NaCl is not that important to the total uptake of nutrients that is used to produce lysostaphin. Tryptone is a mixture composed of amino acids (AAs) and longer peptides. (9) Yeast extract is not much different since it consists of various growth factors, sources of sulfur and vitamin B complex, but mainly peptides and AAs since yeast extract is mainly produced by digestion of proteins by proteases. (10) Because both tryptone and yeast extract are largely amino acids and peptides we treated them as if they were only AAs and peptides. We then assumed that the mass ratio single AAs versus oligopeptides was 50/50. The average molecular weight for a single amino acid was calculated by averaging molecular weights of the twenty most common AAs in nature as shown in table 2. Values for molecular weight of amino acids were obtained through the database of the National Centre of Biotechnology Information (NCBI) called PubChem Compound.</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;">Table 2: Amino acids with their respective molecular weights in gram per mol.</P></DIV></DIV></DIV><P>For the oligopeptides we determined the average oligopeptide MW by multiplying the average oligopeptide length of 13 (2 to 25 AAs) by the calculated average MW of a single amino acid, resulting in MW<SUB>oligo</SUB>= 1779.72 g/mol. (11,12)  Since we have 10 g/L tryptone and 5 g/L yeast extract, this results in a 15 g/L mixture of AAs and oligopeptides. By assuming a 50/50 mass ratio of AAs and oligopeptides, so 7.5 grams of both, we could conclude the following concentrations with the average MW values: 
            c<SUB>AAs</SUB> = 0.05478 mol/L, c<SUB>oligo</SUB> = 0.004214 mol/L. </P><P>We then took the average of these concentrations to estimate the concentration nutrients in the LB we used and thus in the nutrient rich layer we put on top of our patch: c<SUB>nut</SUB> = 0.029499 mol/L</P></SECTION><SECTION><H2>Calculation 2</H2><P>After calculating the concentration of nutrients in LB Broth (Miller) we of course want to put this into the model, however, we will need a diffusivity value for that. We came across an article by <EM>Polson, A.</EM> where he experimentally determined the diffusivity of several amino acids in water. (13) We will not elaborate on these values, but we took the averages for AAs that <EM>Polson, A.</EM> determined and took the average of these values to get an averaged diffusivity for AAs: D<SUB>0,AAs</SUB> = 7E-10 m<SUP>2</SUP>/s </P>.
            <P>Since we said that the mass ratio within the nutrients is 50% amino acids and 50% oligopeptides (see Calculation 1), we also have to determine a separate diffusivity value for oligopeptides. Since we calculated a plausible molecular weight for oligopeptides earlier (in Calculation 1), and with the knowledge that MW scales approximately with the diffusivity, we assumed that a molecule with similar MW as MW<SUB>oligo </SUB>had a similar diffusivity (7). We found a diffusivity value for the neuropeptide Cortistatin of rats with MW<SUB>corstatin </SUB>= 1721.0 g/mol. So with MW<SUB>oligo </SUB>= 1779.72 g/mol, we determined the following: D<SUB>0,oligo </SUB>≈ D<SUB>0,corstatin </SUB>= 3.70E-10 m<SUP>2</SUP>/s (14) </P>.

            <P> Just like the concentration of nutrients, we want the diffusivity of the nutrients to have one value and not be split up. So again we took the average, although we discussed the fact this would favour small values, but we came across a website of the FCSXpert Team (Fluorescence Correlation Spectroscopy) of Sensor Technologies LLC. They stated that, whatever average calculation you used, it is not a bad way to calculate it as long as it is noted how it is calculated. (15) So for our calculation we took D<SUB>0,AAs </SUB>and D0,oligo, summed them and divided them by 2 to get: D<SUB>0,nut </SUB>= 5.35E-10 m<SUP>2</SUP>/s. </P><P>For the diffusivity of NaCl through water we found the value of D<SUB>0,NaCl </SUB>=  1.607 * 10<SUP>-9</SUP> m<SUP>2</SUP>/s(16). However, since NaCl is a common salt present in all kinds of solutions, we did not think that taking the average would actually realistically influence the total diffusion constant, because the medium in the gel itself will already contain salts like NaCl. Moreover, as said before, it does not contribute to the production of lysostaphin.
            Going back to the nutrient diffusivity, thus without taking NaCl into account, we do not want the diffusivity through water (D<SUB>0</SUB>) but through the hydrogel, the effective diffusivity D<SUB>e</SUB>. We use equation 2 to calculate D<SUB>e </SUB>from D<SUB>0,nut</SUB>, ε<SUB>t </SUB>and τ<SUB>c</SUB>. For the nutrients we then get:
            D<SUB>e,nut </SUB>= 3.98E-10 m<SUP>2</SUP>/s. </P></SECTION><SECTION><H2>Calculation 3</H2><P>To determine the uptake of <EM>E. coli</EM> cells of nutrients, we first assumed that the <EM>E. coli</EM> bacteria in our gel would always use the maximum amount of nutrients available, whether it is used for growth, protein production, division or other processes. <EM>Selvarasu et al.</EM> conducted experiments to determine the uptake of amino acids by <EM>E. coli</EM> that are programmed to maximize growth rate during the growth phase using linear programming (LP)(17). They showed AA uptake during three growth phases in which the uptake was maximized in the first growth phase. So to match our assumption we chose to use their data acquired during the first growth phase in column three, Table III of their article, the biosynthetic demand per amino acid in mmol/gDCW/h (DCW is short for Dry Cell Weight).</P><P>We divided the biosynthetic demand per AA by 3600 to get the demand per AA in mg/gDCW/s. Next, we divided it by 3600 to get the values in mg/gDCW/s, since COMSOL Multiphysics commonly works with seconds instead of hours. To get the average demand of per AA we summed all the synthetic demand values per AA and divided them by the amount of amino acids registered (twenty). After that, to get from gDCW to cells divided the synthetic demand by the average dry cell weight per cell of 280 fg (18). This gets us the average AA demand per cell in mmol/cell/s. We multiply this by 1000 to get the value in mol/cell/s, but to get the total amount of AAs needed per cell we still have to multiply the current value by 20, since our current value is for a single average AA, giving:</P><P>Uptake of nutrients per cell, U<SUB>nut,cell </SUB>=4.22E-19 mol/cell/s</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;">Table 3: Variables used to obtain various combinations of models to review the contribution of patch composition and seeding density to the total hydrogel patch.</P></DIV></DIV></DIV><P>To get results out of the model we take the measure that the MBEC for lysostaphin should be  reached at the latest moment at 12 hours after the bacteria are induced to produce lysostaphin.</P><P>We will determine the moment of reaching MBEC at the skin for lysostaphin for several thicknesses of hydrogels with varying seeding densities. Three thicknesses of hydrogel will be modelled, starting with the hydrogel with a nutrient layer of 1 mm, cell layer of 1,111 mm and a buffer layer of 1.8889 mm for a total of 4 mm hydrogel. Secondly, a hydrogel with nutrient layer of 1 mm, cell layer of 1.111 mm and buffer layer of 0.9445 (half of the original) for a total thickness of 3.056 mm. Lastly, the buffer layer will be completely removed. Only the cell layer and nutrient layer will be left for a thickness of 2.111 mm.</P><P>Moreover, for the three mentioned variants of hydrogel four seeding densities will be tested in steps of x10. The start will be at the amount of cells in OD = 0.01 which is 3 million per millilitre and we will end at 3 * 10<SUP>10 </SUP>cells per millilitre for which we definitely know we reach MBEC. The experiments used in model 2 show that with 7.5 * 10<SUP>9 </SUP>cells enough lysostaphin is produced to eradicate the biofilm of <EM>S. aureus</EM>. For the reason we wanted to scale up consistently, we chose to end at 3 * 10<SUP>10 </SUP>to make sure we establish MBEC: 2.963 * 10<SUP>-6 </SUP> mol/L = 2.963 * 10<SUP>-3 </SUP> mol m<SUP>-3</SUP>.</P></SECTION><SECTION><H2>Results</H2><P>Starting for the hydrogel with a 1 mm nutrient layer, a 1.111 mm cell layer and 1.889 mm of buffer layer with a cell count of 3 million. The model shows a high concentration of nutrients at t = 0h in the nutrient layer just like expected. (figure 2)</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;">Figure 2: Nutrient layer of 1 mm, cell layer of 1.111 mm and buffer layer of 1.889 mm. t=0h. Nutrient concentration is shown. Axis shown in mol/m<SUB>-3</SUB>.</P></DIV></DIV></DIV><P>Over time we see diffusion of the nutrients through the gel, but not only diffusion, the uptake of nutrients by cells is also taken into account. However, after twelve hours we still see a large concentration of nutrients of around 5 * 10<SUP>-4</SUP> mol/m<SUP>-3</SUP> or 5 *10<SUP>-7</SUP> mol/L all throughout the gel as can be seen in figure 3.</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;">Figure 3: Nutrient layer of 1 mm, cell layer of 1.111 mm and buffer layer of 1.889 mm. t=12h. Nutrient concentration is shown. Axis shown in mol/m<SUB>-3</SUB>.</P></DIV></DIV></DIV><P>For the lysostaphin production, at t = 0h in figure 4 an extremely small concentration of lysostaphin can be seen in the cell layer.</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;">Figure 4: Nutrient layer of 1 mm, cell layer of 1.111 mm and buffer layer of 1.889 mm. t=0h. Lysostaphin concentration is shown. Axis shown in mol/m<SUB>-3</SUB>.</P></DIV></DIV></DIV><P>At t = 12h, as can be seen in figure 5, a low concentration of lysostaphin has been established.</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;">Figure 5: Nutrient layer of 1 mm, cell layer of 1.111 mm and buffer layer of 1.889 mm. t=12h. Lysostaphin concentration is shown. Axis shown in mol/m<SUB>-3</SUB>.</P></DIV></DIV></DIV><P>Since we want to see if the MBEC for S. aureus is reached, we determine the concentration of lysostaphin along the x,y-plane between gel and exudate, the exudate-plane. We also determine the concentration of nutrients on this plane to see what amounts of nutrients are flowing through the gel into the exudate with malignant bacteria.  Figure 6 shows the mentioned plane with an approximate average nutrient concentration of 7 * 10<SUP>-5</SUP> mol/m<SUP>3</SUP> across the plane. Figure 7 depicts the approximate average lysostaphin concentration across the same plane, showing a concentration of 10-15 * 10<SUP>-13</SUP> mol/m<SUP>-3</SUP>.</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;">Figure 6: Nutrient layer of 1 mm, cell layer of 1.111 mm and buffer layer of 1.889 mm. t=12h. Nutrient concentration is shown. Axis shown in mol/m<SUB>-3</SUB>.</P></DIV></DIV></DIV><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;">Figure 7: Nutrient layer of 1 mm, cell layer of 1.111 mm and buffer layer of 1.889 mm. t=12h. Lysostaphin concentration is shown. Axis shown in mol/m<SUB>-3</SUB>.</P></DIV></DIV></DIV><P>Now that we have shown what results we obtain and how we obtain them with COMSOL, we will not show all the pictures for the other hydrogel composition and seeding density combination, however, we put the obtained values in Table 4.</P><DIV class="figureWrapper figureMiddle"><DIV class="figureFrame"><DIV class="figureCaption" style="text-align: center;"><P style="text-align: center;">Table 4: Results</P></DIV></DIV></DIV></SECTION><SECTION><H2>Discussion</H2><P>As we established the 3D in COMSOL, the complete hydrogel patch looked promising since there are nice gradients that can be seen for diffusion of particles. We kept encountering these gradients, but the surfaces we wanted to use to determine concentrations have varying values all over as can been seen in figure 6 and 7. The axes showing the values of the gradient always went from the maximum to zero, making the gradients' values not very accurate. We kept trying to make educated guesses for averages of the gradients, but sometimes the axis gave values making it almost impossible to get decent results. However, the trend of increasing lysostaphin concentration stays the same over increasing amounts of cells.</P><P>This also accounts for the concentration of nutrients. When the amount of cells increases, the concentration of nutrients reaching the exudate of the wound decreases. However, it is worth noting that reading errors are much more influential for nutrient concentrations because of the small differences for varying amounts of cells. In fact, we later realized that this parameter is not worth investigating. The concentration of nutrients reaching the exudate (and in that sense the <I>S. aureus</I> through the gel is not relevant, because exudate is already incredibly soaked with nutrients. This is why deeper wounds get infected more quickly than more superficial wounds in the first place. In wounds where blood vessels are close to the surface, a lot of small molecules are pushed out of the bloodstream producing exudate. Blood is full of nutrients and this way the exudate is as well.</P><P>The most important feature why we established this model, enough lysostaphin expression within a hydrogel seeded with bacteria, does not come forward very well unfortunately. The concentrations of lysostaphin that are shown on the selected surface where these concentrations matters most, at the border of gel and exudate, are strikingly low. This does not meet our expectations at all. With the lysostaphin production per cell found with model 2  and the moderately high diffusion constant we calculated, we expected to reach MBEC at some point with some amount of cells. Since the experiment shown in the model for Production of Lysostaphin clearly showed cells are capable of reaching MBEC after some time. We think this result in the model is caused by our blood boundary condition. Because we put the boundary condition that in the blood the lysostaphin concentration should go to zero, this 'sucks' the lysostaphin from the exudate and the gel. In hindsight it is not logical to put a boundary condition of zero there. Proteins, even smaller ones, go through the thin wall of cells of the blood vessels very poorly. So even though the blood flows away from a wound, carrying away all that diffuses to it, lysostaphin should not have been included here. This would greatly increase the model's lysostaphin concentration on the border of gel and exudate, agreeing with the results we got from the experiments: MBEC can be reached for <I>S. aureus</I> by lysostaphin that is produced by <I>E. coli</I>.</P><P>For future modelling there should be done more experiments to increase the accuracy of certain constants used in this model. Other models we established were also partly used in this model, but their individual inaccuracy was already discussed separately. Improving the first model will already improve the accuracy of the properties of the gel and this way indirectly diffusion constants on which this model relies.</P></SECTION><SECTION><H2>Conclusion</H2><P>Considering the inaccuracies in model parameters due to literature errors, human errors and small errors in formulas, we cannot conclude anything quantitatively on the ideal dimensions of the patch from this model. However, since trends in the obtained data are discernible, we still think we can carefully draw some minor conclusions.</P><P>The lysostaphin showing up in the exudate had relatively low rates, not reaching the MBEC we calculated. Even at cell densities of 3 * 10<SUP>16</SUP> m<SUP>-3</SUP> (which is very high) not enough lysostaphin reaches the exudate while this amount of cells is definitely able to produce enough. We do see, however, that diffusion of nutrients and thus nourishing the cells is not a problem in the patch.</P></SECTION><SECTION><H2>Future insight</H2><P>We think, concluding from our model that with a different boundary condition lysostaphin concentrations will show higher values as if the cells were actually producing some lysostaphin. We think, using this new boundary condition that concentrations of lysostaphin in the exudate will be reaching MBEC, just like we found in experiments. Moreover, we think it is not necessary to have a thick hydrogel but rather a thin one. This will lead to more convenience and it will need less cells producing less lysostaphin in the patch because the layer to diffuse through is thinner.</P><P>Lastly, since nutrients will no longer be a problem, even though they easily diffuse through the gel. Taking into account the concentrations of nutrients in exudate and infected, open wounds that can already be treated and healed, we no longer see feeding nutrients to the wound as an obstacle. For designing a patch, it does not need a minimum thickness anymore to keep nutrients away from the infected wound. Only the fact that MBEC should be reached and the ease to handle should be taken into account. In conclusion, after establishing a new model continuing from this one, we should be able to determine accurate dimensions of our patch, bringing effectiveness and ease to handle together. We can already say that the optimal thickness of the patch will be as thin as possible, optimizing lysostaphin concentrations but not thinner half a millimeter, to optimize handling by nurses and doctors. Also, the stability and durability of the gel decrreases when the patch becomes thinner, this should also be taken into account. However, this should be researched by trial and error since such stability is extremely difficult to model.</P></SECTION><SECTION><H2>References</H2><OL class="reference"><LI id="sbdref1">Jones RN, Farrell DJ, Helio SS. Lysostaphin and clarithromycin: a promising combination of the eradication of Staphylococcus aureus biofilms. Int J Antimicrob Agents. 2011;37:585–7.</LI><LI id="sbdref2">Beilin LJ, Knight GJ, Anderson J. The Sodium , Potassium, and Water Contents of Red Blood Cells of Healthy Human Adults. 1966;45(11):1817–25.</LI><LI id="sbdref3">Cutting KF. Wound exudate: composition and functions. Br J Community Nurs. 2003;(February):3–9. </LI><LI id="sbdref4">Plieva F, Oknianska A, Degerman E, Galaev IY, Mattiasson B. Journal of Biomaterials Science , Novel supermacroporous dextran gels. 2012;(November):37–41.</LI><LI id="sbdref5">Brakel JVAN, Heertjes PM. ANALYSIS OF DIFFUSION IN MACROPOROUS MEDIA IN TERMS OF A POROSITY , A TORTUOSITY AND A CONSTRICTIVITY FACTOR. Int J Heat Mass Transf [Internet]. 1974;17:1093–103. Available from: https://doi.org/10.1163/156856206778530722</LI><LI id="sbdref6">Dayel MJ, Hom EFY, Verkman AS. Diffusion of Green Fluorescent Protein in the Aqueous-Phase Lumen of Endoplasmic Reticulum. Biophys J [Internet]. 1999;76(5):2843–51. Available from: http://dx.doi.org/10.1016/S0006-3495(99)77438-2</LI><LI id="sbdref7">Valencia DP, González FJ. Understanding the linear correlation between diffusion coefficient and molecular weight. A model to estimate diffusion coefficients in acetonitrile solutions. Electrochem commun [Internet]. 2011;13:129–32. Available from: http://dx.doi.org/10.1016/j.elecom.2010.11.032</LI><LI id="sbdref8">Sigma-Aldrich. LB Broth (Miller) L3522 [Internet]. [cited 2018 Oct 16]. Available from: https://www.sigmaaldrich.com/catalog/product/sigma/l3522?lang=en&amp;region=NL</LI><LI id="sbdref9">GRiSP Research Solutions. Tryptone: Product Information – Culture Media and Components. 2016;1:1–2. Available from: http://www.grisp.pt/docs/gcm23-tryptone.pdf</LI><LI id="sbdref10">Sigma-Aldrich. Yeast Extracts - Base Ingredients for Media [Internet]. [cited 2018 Oct 16]. Available from: https://www.sigmaaldrich.com/analytical-chromatography/microbiology/microbiology-products.html?TablePage=8657623</LI><LI id="sbdref11">John Wiley &amp; Sons. Polymer synthesis: an Industrial Perspective. In: Schlüter AD, Hawker CJ, Sakamoto J, editors. Synthesis of Polymers: New Structures and Methods. WILEY-VCH Verlag GmbH &amp; Co. KGaA; 2012. p. 215.</LI><LI id="sbdref12">Brakel JVAN, Heertjes PM. ANALYSIS OF DIFFUSION IN MACROPOROUS MEDIA IN TERMS OF A POROSITY , A TORTUOSITY AND A CONSTRICTIVITY FACTOR. Int J Heat Mass Transf [Internet]. 1974;17:1093–103. Available from: https://doi.org/10.1163/156856206778530722</LI><LI id="sbdref13">Polson A. CCXXXIII. ON THE DIFFUSION CONSTANTS OF THE AMINO-ACIDS. Biochem J. 1937;31(10):1903–27. </LI><LI id="sbdref14">Hosoya O, Chono S, Saso Y, Juni K, Morimoto K, Seki T. Determination of diffusion coefficients of peptides and prediction of permeability through a porous membrane. J Pharm Pharmacol [Internet]. 2004;56:1501–7. Available from: http://doi.wiley.com/10.1211/0022357044878</LI><LI id="sbdref15">FCSXpert Team. Averaging Correlation Times vs. Averaging Diffusion Coefficients [Internet]. [cited 2018 Oct 16]. Available from: http://www.fcsxpert.com/classroom/analysis/note_averaging.html</LI><LI id="sbdref16">UK E. Diffusion Coefficient of 2m NACL in Water. [Internet]. 2013. Available from: https://www.ukessays.com/essays/biology/measurement-of-liquid-diffusion-coefficient-biology-essay.php?vref=1</LI><LI id="sbdref17">Selvarasu S, Ow DSW, Lee SY, Lee MM, Oh SKW, Karimi IA, et al. Characterizing escherichia coli DH5α growth and metabolism in a complex medium using genome-scale flux analysis. Biotechnol Bioeng. 2008;102(3):923–34. </LI><LI id="sbdref18">Neidhardt FC. Escherichia coli and Salmonella. 1st ed. Curtiss III R, Ingraham JL, Lin ECC, Brooks Low K, Magasanik B, Reznikoff WS, et al., editors. Washington: ASM Press; 1996. 14 p. </LI><LI id="sbdref19">Eppendorf. OD600 Measurements Using Different Photometers. 2015;(28). </LI></OL></SECTION></DIV></DIV></DIV><DIV id="wideImageModal" class="figureModalWindow"><SPAN class="closeFigure">×</SPAN><DIV class="modalCaption">Lorem Ipsum</DIV></DIV><DIV id="imageModal" class="figureModalWindow"><SPAN class="closeFigure">×</SPAN><DIV class="modalCaption">Lorem Ipsum</DIV></DIV></DIV><DIV class="container footer" id="footer-container"><DIV class="neat row contactus"><DIV class="col-xs-12 col-lg-4" style="background-color: #FF8054"><A href="mailto:igem@tue.nl"><SPAN> igem@tue.nl</SPAN></A><A href="https://www.facebook.com/iGEMTUEindhoven"><SPAN> iGEMTUEindhoven</SPAN></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>