<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Software_Linker_Software"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Software/Linker_Software">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Software/Linker_Software&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Software/Linker_Software&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Software/Linker_Software&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Software/Linker_Software" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Software/Linker Software</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container" style="padding:0;background-color:white;"><HEADER id="navbar" role="banner" class="navbar navbar-static-top navbar-white"><DIV class="container" style=" background-color:black;"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse collapse" style="margin-bottom:10px;"><NAV role="navigation"><UL class="menu nav navbar-nav navbar-right"><LI class="dropdown"><A href="/Team:Heidelberg/Team" class="dropdown-toggle">Team</A></LI><LI><A href="/Team:Heidelberg/Team/Members">Members</A></LI><LI><A href="/Team:Heidelberg/Team/Collaborations">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="/Team:Heidelberg/Project" class="dropdown-toggle">Project</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Background">Background</A></LI><LI><A href="/Team:Heidelberg/Project/Toolbox">Toolbox</A></LI><LI><A href="/Team:Heidelberg/Toolbox/Circularization">Toolbox: Circularization</A></LI><LI><A href="/Team:Heidelberg/Project/PCR_2.0">PCR 2.0</A></LI><LI><A href="/Team:Heidelberg/Project/Xylanase">Xylanase</A></LI><LI><A href="/Team:Heidelberg/Project/Linker_Screening">Linker Screening</A></LI><LI><A href="/Team:Heidelberg/Project/Reconstitution">Fluorescence Assembly</A></LI><LI><A href="/Team:Heidelberg/Project/LOV">Light-Induction</A></LI><LI><A href="/Team:Heidelberg/Project#Achievements">Achievements</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Parts" class="dropdown-toggle">Parts</A></LI><LI><A href="/Team:Heidelberg/Parts/RFC">RFC Heidelberg 2014</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Favorite Parts">Favorite Parts</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Sample Data Page">Sample Data Page</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#allParts">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Backbones">Backbones</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Intein Library">Intein Library</A></LI><LI><A href="/Team:Heidelberg/Parts/Part_Improvement">Part Improvement</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Software" class="dropdown-toggle">Software</A></LI><LI><A href="/Team:Heidelberg/Software/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Software/Linker_Software">Linker Software</A></LI><LI><A href="/Team:Heidelberg/Software/MidnightDoc">MidnightDoc</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Modeling" class="dropdown-toggle">Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Linker_Modeling">Linker Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Enzyme_Modeling">Enzyme Kinetics Modeling</A></LI><LI><A href="/Team:Heidelberg/Toolbox_Guide" class="dropdown-toggle">Toolbox Guide</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Human_Practice" class="dropdown-toggle">Human Practice</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Ethics">Religion &amp; Synthetic Biology</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Education">Education</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Public_Relations">Public Relations</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Experts">Experts</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Notebook" class="dropdown-toggle">Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="/Team:Heidelberg/AwesomeSheet">Database for Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook/Materials">Materials</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Notebook/Safety">Safety</A></LI></NAV></DIV><DIV class="col-lg-12"><DIV class="title-wrapper"><SPAN class="title">CRAUT</SPAN></DIV></DIV></DIV></HEADER><DIV class="container main"><DIV class="row" style="position:relative;"><DIV class="col-lg-offset-1 col-lg-11 subtitle"><H2><P>– Circularization with Rods and Angles of Unlinked Termini</P></H2></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><H2 class="titles"><A href="#Background" class="label label-default">Background</A><A href="#Results" class="label label-default">Results</A></H2></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="well well-lg abstract"><P><B>ABSTRACT</B></P>
As already <A href="https://2014.igem.org/Team:Heidelberg/Toolbox/Circularization" class="external text" rel="nofollow">introduced</A>, artificially circularized proteins may gain some heat stability by restraining the C- and N-terminus from moving around  freely. This  circularization may be trivial when the protein termini are very close to each other, which seems to be reasonably common <A href="#References">[1]</A>. However, if the ends are too far from each other, a long linker is needed to  connect them. This linker should not change the natural conformation of the protein and should constrain the relative position of the ends to restrict the degrees of freedom and thus to stabilize the structure even when heated up. On top, these linkers should not affect any of the protein functions. Consequently it is important to prevent linkers from passing through the active site or from covering binding domains to other molecules for example. Therefore one  needs to be able to define the shape of possible linkers. This section describes the software we developed to design such linkers. We would like to stress that this work has been made possible thanks to the feedback between computer modeling and experimental work: We could first design linkers in silico, test them experimentally and use the results to further calibrate the software. To our knowledge, this is the first time that such an approach is used to customly design rigid linkers with angles to connect protein extremities.

		</DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#General_procedure"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">General procedure</SPAN></A></LI><LI class="toclevel-1"><A href="#Background"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2"><A href="#PDB_analysis"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">PDB analysis</SPAN></A></LI><LI class="toclevel-2"><A href="#Generation_of_geometric_paths"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Generation of geometric paths</SPAN></A></LI><LI class="toclevel-3"><A href="#Step_1"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Step 1</SPAN></A></LI><LI class="toclevel-3"><A href="#Step_2"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Step 2</SPAN></A></LI><LI class="toclevel-3"><A href="#Step_3"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">Step 3</SPAN></A></LI></UL><LI class="toclevel-2"><A href="#Sorting_out_of_paths"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Sorting out of paths</SPAN></A></LI><LI class="toclevel-2"><A href="#Shifting_paths_to_the_patterns"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Shifting paths to the patterns</SPAN></A></LI><LI class="toclevel-2"><A href="#Weighting_of_paths"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Weighting of paths</SPAN></A></LI><LI class="toclevel-3"><A href="#Calibrating_the_weighting_function"><SPAN class="tocnumber">2.5.1</SPAN><SPAN class="toctext">Calibrating the weighting function</SPAN></A></LI><LI class="toclevel-2"><A href="#Translating_paths_to_sequence"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Translating paths to sequence</SPAN></A></LI><LI class="toclevel-2"><A href="#Clustering_of_paths"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Clustering of paths</SPAN></A></LI><LI class="toclevel-1"><A href="#Results"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2"><A href="#DNMT1"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">DNMT1</SPAN></A></LI><LI class="toclevel-2"><A href="#Feedback_from_wet_lab"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Feedback from wet lab</SPAN></A></LI><LI class="toclevel-1"><A href="#Discussion"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="General_procedure">General procedure</SPAN></H1><P>In short, the software can provide a weighted list of linkers to circularize any protein of interest with a known structure. Those linkers are made of rigid alpha helices segments connected with defined angles. Contrary to flexible linkers, those rigid linkers were expected to constrain the protein extremities and to confer better heat stability. Such an idea was already developed <A href="#References">[2]</A> but only with alpha helices defining simple rods, and without any possibility to introduce angles. To generate those linkers, we first defined the geometrical paths, with segments and angles, that they should follow. The geometrical paths that are biologically feasible are afterwards translated into amino acid sequences. Both the compatibility of paths with possible structures and the translation were made possible thanks to our <A href="https://2014.igem.org/Team:Heidelberg/Modeling/Linker_Modeling" class="external text" rel="nofollow">modeling approaches</A>. The first approach consisted in performing a statistical analysis of more than 17000 known non-homologous structures containing alpha helices connected with angles. For the second approach, we modeled the conformation of linkers circularizing proteins of known structure and analyzed them for certain properties. This second approach was run for a large number of proteins thanks to our distributing computing system <A href="https://2014.igem.org/Team:Heidelberg/Software/igemathome" class="external text" rel="nofollow">igemathome</A>. The software provides different possible linkers with weights that provide the ranking of the linkers depending on their capacity to maintain protein activity at higher temperatures. They were generated thanks to an extensive <A href="https://2014.igem.org/Team:Heidelberg/Project/Linker_Screening" class="external text" rel="nofollow">linker screening</A> on the target protein lambda-lysozyme, using the first modeling approach.
The documentation of our CRAUT software can be found <A href="https://2014.igem.org/Team:Heidelberg/Software/Linker_Software/Documentation" class="external text" rel="nofollow">here</A>.
</P><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 0)</SPAN><P>A representation of the general concept of CRAUT. At first the user provides it with a protein structure, by providing it with a PDB file. Then he can add relevant data, like binding sites, that he has found in databases. Then he chooses which parts of the protein the software should circularize and which parts of the protein should be ignored. After the calculations have finished, the user gets a sequence of the best linker, with which he could circularize the target protein.</P></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Background">Background</SPAN></H1><P>Classically,  protein  linkers were designed in three different manners. The  easiest way is to define the length that a linker should  cover and then simply  use a flexible glycine-serine peptide with the  right amount of amino acids  to match this length. Glycine is used for flexibility, as it has no sidechain and does not produce any steric  hindrance, while serine is used for solubility, as it has a small polar side chain. This solubility is important, as the linkers  should not pass through the hydrophobic core of the protein, but should be dissolved in the surrounding medium. These flexible linkers were normally used for circularization but also for connecting different proteins, when the  main goal is that the different parts are connected, but not how they are connected, or when the flexiblity of the linker was required for specific applications. 
</P><P>A second strategy consists in using rigid helical linkers to keep  proteins or protein domains at a certain distance from each other. This  is especially important for signalling proteins and fluorescent proteins. One major property of alpha helices is that they always fold in a defined way with well defined angles and lengths. There are also many different helical patterns that differ in stability and solubility. Although they have been used to design cirularizing linkers <A href="#References">[2]</A>. One big disadvantage of this strategy is that one can only build straight linkers with helices. So in the context of circularization, if an artificial line that would connect protein extremities is crossing the protein, this strategy is not an option.
</P><P>The third option, which served as a base to develop our approach and which came from discussions with the group of Rebecca Wade in Heidelberg, Germany, consists in designing customly tailored linkers for each specific application. These linkers can be obtained from protein structure prediction. At first one needs to define the path that the linker should take to connect the protein ends. Afterwards, one designs a possible linker sequence that  might fit well. Next  one makes a structure prediction of the linker  attached to the proteins to validate the prediction. Several different  linkers, with slight  changes, can be compared. This is repeated several  times until the  linker effectively follows the expected path. This method requires a strong knowledge on  protein folding and protein structure prediction and is computationaly intensive. On the other hand, the  benefit can be important as the interaction of the linker with the protein surface can be taken into account and as one can accurately define the path taken by the linker to the resolution of protein structure.
</P><P>We  have set up a completely new strategy to design rigid linkers. As further detailed in the <A href="https://2014.igem.org/Team:Heidelberg/Modeling/Linker_Modeling" class="external text" rel="nofollow">modeling</A> part, it is possible to define the shape of a linker, by combining rigid alpha helical rods with well-defined angle patterns. Therefore, by defining, in a geometrical way, the possible paths of the  circularizing linkers for a given protein, we can then propose potential linkers. This definition of the geometrical path can be very difficult, especially for large proteins with complex shapes. Moreover, this definition is further constrained by the fact that linkers must avoid hiding active sites of the protein of interest. Finally the paths have rotational degrees of freedom at the extremities of the protein, and depending on their orientation, they may or may not match the geometry of the protein. The tool we present here covers the two steps: defining geometrical paths with some weights and translate them into feasible linkers, also with weights. This tool is universal as it has the capacity to design circularizing linkers for any protein with a known structure. Moreover it is modular as, thanks to our <A href="https://2014.igem.org/Team:Heidelberg/Modeling/Linker_Modeling" class="external text" rel="nofollow">modeling approach</A>, we have designed linkers as exchangeable blocks of rods of different lengths and of angle patterns. The following sections detail the different steps followed by our software to design proper linkers.
</P><H2><SPAN class="mw-headline" id="PDB_analysis">PDB analysis</SPAN></H2><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 1) Accessible ends</SPAN><P>When checking whether the ends of the protein are covered, at first for all the directions it is checked, whether some of the protein points are in the way. This was done for discrete angles incremented at 5°</P></DIV></DIV></DIV><P>At first, the PDB file containing the structure of the target protein is parsed and the coordinates of the atoms are stored, in the metric unit. After this, some initial tests are made with the protein structure. First, we checked whether the C- and N-termini lie on the surface of the protein and are accessible to the solvent, which is crucial for circularization. We defined a line originating from an extremity of the protein with the two angles of the spherical coordinates around the z-axis. From that, we could determine the accessible angles by rejecting all the lines that are too close to the protein. As the future linker will be made of alpha helices and will therefore have a radius of 5 Å, we used this length as the minimal allowed distance.
Those allowed angles are stored for the coming linker generation.
</P><H2><SPAN class="mw-headline" id="Generation_of_geometric_paths">Generation of geometric paths</SPAN></H2><P>As our strategy consists in building linkers with helical rods and connecting angles, a path is completely defined by the coordinates of the angle points. Advancing one step from an existing point is always done by adding a displacement vector on this point. This vector is defined by the two spherical angles, chosen here in a discrete manner with an increment of 5 degrees, and by a length, also chosen in a discrete manner. This discrete length was used in two different contexts: it may correspond to the length of an alpha helix or to the length of the flexible part that appears at the extremity of the protein.  The coordinates that are reached thanks to this vector defines the new coordinates of an angle point, no matter if the vector corresponds to an alpha helix or to a flexible part. 
</P><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 2) Linker going around torus</SPAN><P>The worst shape we could think of for circularization was a torus without hole with ends in the middle. Even this shape could be circularized with our linkers.</P></DIV></DIV></DIV><P>As we screen for all possible angles in a discrete manner, those angle points coordinates are regularly distributed on a sphere. As further detailed in the next sections, those spheres are defined from both ends, either once or in several steps. Then the software checks for possible straight connections of given lengths for each pair of angle points originated from both extremities. 
</P><DIV class="pull-right col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 3) The generation of the paths</SPAN><P>Easy representation of the process of points generation. At first all the points from the start are generated, left. Then the points from the end are generated. Then all possible connections between the points are checked for their validity. This is done for every point from the beginning.</P></DIV></DIV></DIV><P>The linkers are built in a modular way, with blocks of well-defined size. From the <A href="https://2014.igem.org/Team:Heidelberg/Modeling/Linker_Modeling" class="external text" rel="nofollow">modeling</A> of potential linkers, we could derive 8 different alpha helical rods, all with different lengths. On top, the length of the two segments inside an angle block was always 8Å, so exchanging angle blocks do not affect the length of the linker. This means that the distance between the angle points is well defined, an essential aspect of our strategy of linker design.
The software proceeds in three steps. First, it checks for the possibility of direct single alpha helix linker. for this, it applies the procedure just mentioned with spheres of radius that reasonably corresponds to the length of the short parts at the extremity of the protein. Second, it tests if a linker containing two alpha helices connected with a right angle allows the circularization. Finally it searches the possible linkers with three angle points. The next parts will explain those three steps in detail.
This method has been chosen, because it could be implemented easily and efficiently in our program. However, this strategy generated paths that crossed the protein. Therefore we put big efforts in the sorting out of the paths.
</P><H3><SPAN class="mw-headline" id="Step_1">Step 1</SPAN></H3><P>As a simple rigid linker with no angle would be easier to design and likely more thermostable than the ones containing angles, the software first checks if this simple solution is possible.
</P><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 4) Step 1</SPAN><P>Only one single alpha helix connects the flexible ends of the protein.</P></DIV></DIV></DIV>

For this, we took into account the fact that proteins have some flexible amino acids at their extremities. This  flexible part may come  from the protein itself, but also from the 2 glycines that are included at the N-terminal part and from the extein at the C-terminal part. Those two latter parts comes from our linkers. Those parts have no preferential angles and offers a large amount of possibilities to insert fitting linkers. But this flexibility is also a drawback as we have to include this large amount of possible angles and length to our path search.
In this first step, the software explicitely takes these flexible parts into account to check for the possibility of straight linkers. As the angles and the length of the flexible parts are variable, the software position their extremity on a sphere centered on the last fixed position of the structure as explained above. The radius of this sphere is incremented in a discrete manner, in 4 steps, from 5.25 Å to the maximum length of the flexible part.
Then all possible straight segments between the points and the lastpoints are tested. If they are closer than 5 Å to the protein, of if they cross it, then they are rejected. If they are kept, then the software checks whether the length of the segments is compatible with the feasible alpha helices in terms of length: if the length of a given segment equal one of the 8 alpha helix lengths plus or minus 0.75 Å, then the path is eventually saved.
<H3><SPAN class="mw-headline" id="Step_2">Step 2</SPAN></H3><P>The next possibilty to design more complex rigid linkers while still taking flexible ends into account with a reasonable calculation time was to reduce amount of possible angles. As we originally thought that 90° angle would be practically feasible, the software was designed to generate  linkers with flexible ends and one 90° angle. 
</P><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 5) Step 2</SPAN><P>The linker forms an angle of 90°.</P></DIV></DIV></DIV>

This choice was notably made because of the simplicity to calculate lengths of right triangle edges.  We already saw in Step 1 that the length of an edge can only take 8 different values. As the linkers have to start from the extremities of the protein, and as we impose a right angle, the number of possible paths is therefore low, making them easy to compute. Practically, the extremities of the proteins are positioned in a flexible way as in Step 1. From each of the positions allowed by this flexibility, the software searches for all the allowed right triangles. This was mainly done, as the degrees of freedom needed to be restricted to keep calculations feasible.
<H3><SPAN class="mw-headline" id="Step_3">Step 3</SPAN></H3><P>Finally the software also provides the possibility to find paths with up to 4 edges, meaning 4 alpha helices and 3 angles. Thanks to the modularity of the possible linkers, such paths can offer the possibility to circularize theoretically any kind of protein, see figure 2).
</P><P>To keep the calculation feasible in a reasonable time, we design the searching strategy so that the flexible part at the extremity are oriented in the same direction as the consecutive alpha helix. This is obviously restricting the search but as these orientations are allowed for the flexible part, this approach remains fully correct. 
First, potential ending points of the first alpha helical rod are calculated from the N-terminal point of the protein. The orientation is chosen in a discrete manner, with an incrementation of 5 degrees for the two angles of the spherical coordinates. The distance from the origin corresponds to the 8 possible lengths allowed by the alpha helices, as already seen in Step 2, plus a length of 0, which mimics a linker with 3 instead of 4 edges. The exact same procedure is repeated to define all the potential ending points of the second alpha helical rod starting from all the possible ending points of the first alpha helical rod. Thanks to the possibility of a length of 0 for the first and the second rods, the software also calculate paths with 2 edges. Then, the same is done only once from the C-terminal point of the protein, defining 1 edge. The final step consists in checking if the points originating from the N- and C-terminal points can be linked by an potential alpha helix, i.e. if they are separated by the appropriate distance. If any of the potential alpha helix length lies within the distance between two points plus or minus 0.75 Å, then the path is eventually saved. In the same way, if two points are directly closer than 0.75 Å, then the path is also saved.
</P><H2><SPAN class="mw-headline" id="Sorting_out_of_paths">Sorting out of paths</SPAN></H2><P>The previous part described the generation of paths that can connect the two extremeties of the protein irrespective of the position of these paths relative to the protein. While this allows a fast computing of the geometrical paths, this also implies that the paths that are not practically feasible need to be sorted out. This is the most time consuming part of the computing as about 1 billion paths are generated. Three criteria are considered for the sorting. The first one is the feasibility of the linker: can the software find angle patterns that correspond to the one defined by the geometrical path? This question was part of the motivation for a large modeling effort (link) to determine the possible angles between consecutive angles. This was achieved by analyzing the <A href="https://2014.igem.org/Team:Heidelberg/Modeling/Linker_Modeling" class="external text" rel="nofollow">distribution of angles</A> between alpha helices found in the ArchDB database. As nearly any angle could be found between 20 and 170 degrees, only few paths were actually rejected at that step. The next criteria was the position of the angle point: if they appear inside the protein, then the path is rejected. Finally, the software checks if any of the atoms of the protein is less than 5Å away from any of the alpha helices, then the path is also rejected.
</P><H2><SPAN class="mw-headline" id="Shifting_paths_to_the_patterns">Shifting paths to the patterns</SPAN></H2><P>The strategy described in step 3 gives a certain freedom for the rod that connect the last two angle points that were generated from the N- and C-terminal points. As this freedom is actually not permitted by the alpha helix and the angle pattern, but is permitted by the flexible part for example at the C-terminal end, the software slighty refine the path by rotating the segment that originates from the C-terminal point.
</P><H2><SPAN class="mw-headline" id="Weighting_of_paths">Weighting of paths</SPAN></H2><P>Before translating the paths into sequences and thus into linkers that can be expressed, each path needs to be evaluated for its potential capacity to enhance heat stability. For this we have identified different contributions that should be combined into one value that defines how good the linker may be and that consequently defines its ranking among all possible linkers. The smaller this value, the more we expect that the linker will enhance thermostability. An important step for all these contributions is the normalization, as  explained in the next paragraphs.
The first contribution we considered was the linker length. We assumed that a short linker is better to constrain the protein extremities, and that a long helix might give more flexibility. Because we wanted this value to be independent of the size of the protein, the length of the linker is normalized to the distance between the two termini.
The second contribution relates to the angles used in the linker. We learned from the <A href="https://2014.igem.org/Team:Heidelberg/Modeling/Linker_Modeling" class="external text" rel="nofollow">modeling</A> that angles formed by a certain angle pattern follow a certain distribution. First, we assumed that the narrower the distribution, the more likely the alpha helices would actually produce this angle. Second, the angles found by the software should be as close as possible to those well-defined angles. In this case, the weight value from this contribution should be low.
Then the distance of the linker to the protein is taken into account. Because the linkers should not disturb the protein in its normal environment, linkers that pass close to the protein surface are considered better linkers. The distance was defined as the minimal distance between the linker and all the atoms of the protein. As already mentioned for the sorting of the paths, a linker cannot come closer than 5 Å and this distance was used for normalization of calculated distances.
After this, the places a linker should avoid are calculated. Each protein can interact with other molecules on some oarts of its surface. The user can specify where and how big those parts are. If a linker passes in front a potential molecule binding domain, the value of the corresponding path goes to infinity, so that the linker is discarded. Conversely the farther a linker is from a potential ligand binding domain, the smaller its weighting value. The user can also specify the importance of certain regions. In the end the total weighting is normalized to the amount of binding domains.
</P><H3><SPAN class="mw-headline" id="Calibrating_the_weighting_function">Calibrating the weighting function</SPAN></H3><P>Every contribution has its own distribution. You can see an example in figure   
</P><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 6) Distribution of length contribution of lysozyme</SPAN><P>Each weight has it's own distribution. As an example here the distribution of the length weighting of lambda lysozyme is shown. One can clearly see the gaps due to the discrete lengthes of the building blocks.</P></DIV></DIV></DIV>

, but all of them have different shapes. The aim is to find the paths that globally minimize all of these distributions. Therefore, for simplicity,in the weighting function the four mentioned contributions were combined in a linear manner:
\[ W(p) = \alpha  L(p) + \beta A(p) + \gamma D(p) + u(p) \]
where W is the final weighting, p the path, L the length contribution, A the angle contribution, D the distance contribution and u the contribution from the forbidden regions. $\alpha, \beta, \gamma, \delta$ are the weighting constants that needed to be found. The normalization performed for each of the contribution were made so that each of them is dimensionless and that all have reasonably similar values.
The weighting constants were obtained from the <A href="https://2014.igem.org/Team:Heidelberg/Project/Linker_Screening" class="external text" rel="nofollow">linker screening</A> performed with lysozyme and the <A href="https://2014.igem.org/Team:Heidelberg/Modeling/Enzyme_Modeling" class="external text" rel="nofollow">modeling of the enzyme activity</A>. Their calculation is presented in the results below.
<H2><SPAN class="mw-headline" id="Translating_paths_to_sequence">Translating paths to sequence</SPAN></H2><P>As already mentioned before the software is provided with two databases, one for the possible angle patterns and one for the helix patterns. The choice of the patterns was inspired by known crystal structures extracted from databases and described in different papers.
A huge in silico screening for refining the preferences of the patterns was then set up using the <A href="https://2014.igem.org/Team:Heidelberg/Software/igemathome" class="external text" rel="nofollow">distribution calculation</A> system. For the complete description of search for suitable patterns, one can read the <A href="https://2014.igem.org/Team:Heidelberg/Modeling/Linker_Modeling" class="external text" rel="nofollow">modeling</A> page. 
All the possible paths are now split up at the angles and compared with the possible patterns in the databases. The most suitable patterns are identified and added together to build the paths sequence. It is important to notice that this is only possible because of the modularity of our linker patterns used as building blocks: each block, being an alpha helix or an angle pattern, is not affected by the other. Thus for each possible path, one sequence is produced.
</P><H2><SPAN class="mw-headline" id="Clustering_of_paths">Clustering of paths</SPAN></H2><P>Many different paths are represented by the same sequence <A href="#.23.23Figure_that_shows.2C_different_paths_have_same_properties.2C_already_before_in_the_text_.23.23.23">###Figure that shows, different paths have same properties, already before in the text ###</A> and we therefore clustered such paths. The weigths for those clustered paths were then calculated by averaging the weights of the different paths that compose a cluster.
</P><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><H2><SPAN class="mw-headline" id="DNMT1">DNMT1</SPAN></H2><P>A  major motivation of our effort to design rigid linkers with angles was  the <A href="https://2014.igem.org/Team:Heidelberg/Project/PCR_2.0" class="external text" rel="nofollow">circularization of the DNA methyltranferase Dnmt1</A>. The  truncation form used in our project is composed of 900 amino acids and  the N- and C-terminal extremities are well separated. To circularize it,  two linkers were designed: a flexible one made of glycine and serine, and a rigid one designed by the software. The rigif linkers for DNMT1  were obtained from an early state of the software.  At that time the  calculation took 11 days on a laptop computer with  intel i5 processor  and 8GB of RAM, which shows the importance of a  distributed computing  system for large proteins. But from that state on, the software has  still improved a lot, resulting in reduced calculation time to about 1 day for DNMT1.
</P><H2><SPAN class="mw-headline" id="Feedback_from_wet_lab">Feedback from wet lab</SPAN></H2><P>The   results from the software for lysozyme were tested as described in the <A href="https://2014.igem.org/Team:Heidelberg/Project/Linker_Screening" class="external text" rel="nofollow">linker-screening</A> part and evaluated as described in the <A href="https://2014.igem.org/Team:Heidelberg/Modeling/Enzyme_Modeling" class="external text" rel="nofollow">enzyme modeling</A> part. We have performed a large linker screening on 10 different lysozymes with different linkers. As the purpose of the lysozyme screen was the calibration of the   software,those linkers (Table 1) were designed according to the four contributions previously mentioned. One of them was the shortest possible, one had the best possible angle, and so on.
</P><TABLE class="table table-hover" style="text-align: center;"><CAPTION><B>Table 1</B>: Linker and their amino acid sequence. Green: attachment sequences to prevent the flexible regions from being perturbed; Blue: angle; Purple: extein.
</CAPTION><TBODY><TR><TH> Linker
</TH><TH> Amino acid sequence
</TH><TH> activity
</TH><TH> length- contribution
</TH><TH> angle- contribution
</TH><TH> binding site contribution
</TH><TH> distance from surface
</TH><TH> weightingvalue after calibration
</TH></TR><TR><TD><B>Very good linkers</B></TD></TR><TR><TD> sgt2
</TD><TD><SPAN style="color:#3ADF00;">GG</SPAN>AEAAAK<SPAN style="color:#00BFFF;">AAAHPEA</SPAN>AEAAAK<SPAN style="color:#A901DB;">RGTCWE</SPAN></TD><TD> 0.7477
</TD><TD> 1.9205
</TD><TD> 6.7789
</TD><TD> 0.002259
</TD><TD> 10.525
</TD><TD> 114912
</TD></TR><TR><TD> rigid
</TD><TD><SPAN style="color:#3ADF00;">GG</SPAN>AEAAAKEAAAKAA<SPAN style="color:#3ADF00;">P</SPAN><SPAN style="color:#A901DB;">RGKCWE</SPAN></TD><TD> 0.9447
</TD></TR><TR><TD><B>Average linkers</B></TD></TR><TR><TD> may1
</TD><TD><SPAN style="color:#3ADF00;">GG</SPAN>AEAAAKEAAAKA<SPAN style="color:#00BFFF;">AAAHPEA</SPAN>AEAAAK EAAAKA<SPAN style="color:#00BFFF;">KTA</SPAN>AEAAAKEAAAKA<SPAN style="color:#A901DB;">RGTCWE</SPAN></TD><TD> 0.7489
</TD><TD> 6.2225
</TD><TD> 13.19
</TD><TD> 0.00384
</TD><TD> 1095.2
</TD><TD> 196414
</TD></TR><TR><TD> ord1
</TD><TD><SPAN style="color:#3ADF00;">GG</SPAN>AEAAAKEAAAK<SPAN style="color:#00BFFF;">ATGDLA</SPAN>AEAAAKAA<SPAN style="color:#A901DB;">RGTCWE</SPAN></TD><TD> 0.956
</TD><TD> 4.936
</TD><TD> 4.639
</TD><TD> 0.00055708
</TD><TD> 220.8
</TD><TD> 27985
</TD></TR><TR><TD> ord3
</TD><TD><SPAN style="color:#3ADF00;">GG</SPAN>AEAAAKEAAAK<SPAN style="color:#00BFFF;">ASLPAA</SPAN>AEAAAKEAAAK<SPAN style="color:#A901DB;">RGTCWE</SPAN></TD><TD> 1.390
</TD><TD> 4.949
</TD><TD> 7.116
</TD><TD> 0.000545
</TD><TD> 261.2
</TD><TD> 28557
</TD></TR><TR><TD><B>Short linkers</B></TD></TR><TR><TD> sho1
</TD><TD><SPAN style="color:#3ADF00;">GG</SPAN><SPAN style="color:#A901DB;">RGTCWE</SPAN></TD><TD> 0.7087
</TD></TR><TR><TD> sho2
</TD><TD><SPAN style="color:#3ADF00;">GG</SPAN>AEAAAK<SPAN style="color:#A901DB;">RGTCWE</SPAN></TD><TD> 0.5743
</TD></TR><TR><TD> flexible linker
</TD><TD> GGSGGGSGRGKCWE
</TD><TD> 0.6851
</TD></TR><TR><TD> linear lysozyme
</TD><TD> no linker
</TD><TD> 0.7039
</TD></TR></TBODY></TABLE><P>In figures 7 and 8 one can compare the modeled structure of the linker to the predicted path of the software. The lysozyme is oriented nearly in the same directions.
</P><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Fig 7) Circular lambdalysozyme structure</SPAN><P>A linker calculated by the software was modeled using modeller.</P></DIV></DIV></DIV><DIV class="pull-left col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Fig 8) Path predicted by software</SPAN><P>A path the software predicted in step 1. The points resemble the turning points, the green cross shows the size of an alpha helix.</P></DIV></DIV></DIV><P>In  the end we obtained a ranking of the in vitro tested linkers from the  <A href="https://2014.igem.org/Team:Heidelberg/Project/Linker_Screening" class="external text" rel="nofollow">linker-screening</A> and chose the parameters $\alpha, \beta, \gamma,  \delta$ of the weighting function so that the ranking from the  software represented the ranking from the assays. These values were at first fitted, so that the ranking predicted by the software resembles. But this could not work out perfectly because for example may1 linker was worse in every contribution than sgt2 but was tested better. Therefore the different parameters were adjusted afterwards by hand. The final values,  $\alpha = 1.85 * 10 ^{-6}, \beta = 0.57 , \gamma = 50.8 * 10^6$, set up a function that could reproduce the ranking oberved in the wetlab experiments.
</P><H1><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H1><P>The software described here allowed us to design rigid linkers with well-defined angles. This represents a major advance compared to previous approaches like <A href="#References">[2]</A> as these linkers can circularize any protein of known structure with any complex geometry.
The feedback between the modeling and the experiment work on lysozyme activity was a crutial step in the development of the software. It allowed the testing of our approach and the calibration of the contribution of different features of the linkers to heat stability. This calibration was performed on one enzyme, and can improve in the future with the testing of more enzymes. This will also be refined thanks to a complete modeling and analysis of protein structures with linkers.
Further on we could refine our assumptions on the different contributions of the weighing function. At first we assumed, that length would dominate, but the data suggests, that the contribution from omitting the substrate would be most important.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>[1] Thornton, J.M. &amp; Sibanda, B.L. Amino and carboxy-terminal regions in globular proteins. Journal of molecular biology 167, 443-460 (1983).
</P><P>[2] Wang, C.K.L., Kaas, Q., Chiche, L. &amp; Craik, D.J. CyBase: A database  of cyclic protein sequences and structures, with applications in protein  discovery and engineering. Nucleic Acids Research 36, (2008).
</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Heidelberg/Software/Linker_Software">http://2014.igem.org/Team:Heidelberg/Software/Linker_Software</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Software/Linker_Software" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Software/Linker_Software" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Software/Linker_Software&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Software/Linker_Software&amp;oldid=394519" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>