<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Toolbox_Circularization_Constructs"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Toolbox/Circularization_Constructs">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Toolbox/Circularization_Constructs&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Toolbox/Circularization_Constructs&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Toolbox/Circularization_Constructs&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Toolbox/Circularization_Constructs" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Toolbox/Circularization Constructs</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container" style="padding:0;background-color:white;"><HEADER id="navbar" role="banner" class="navbar navbar-static-top navbar-white"><DIV class="container" style="background-image:url(/wiki/images/a/aa/Header_circularization.jpg);background-position: bottom right; background-repeat:no-repeat; background-size: 100% auto; background-color:black;"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse collapse" style="margin-bottom:10px;"><NAV role="navigation"><UL class="menu nav navbar-nav navbar-right"><LI class="dropdown"><A href="/Team:Heidelberg/Team" class="dropdown-toggle">Team</A></LI><LI><A href="/Team:Heidelberg/Team/Members">Members</A></LI><LI><A href="/Team:Heidelberg/Team/Collaborations">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="/Team:Heidelberg/Project" class="dropdown-toggle">Project</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Background">Background</A></LI><LI><A href="/Team:Heidelberg/Project/Toolbox">Toolbox</A></LI><LI><A href="/Team:Heidelberg/Toolbox/Circularization">Toolbox: Circularization</A></LI><LI><A href="/Team:Heidelberg/Project/PCR_2.0">PCR 2.0</A></LI><LI><A href="/Team:Heidelberg/Project/Xylanase">Xylanase</A></LI><LI><A href="/Team:Heidelberg/Project/Linker_Screening">Linker Screening</A></LI><LI><A href="/Team:Heidelberg/Project/Reconstitution">Fluorescence Assembly</A></LI><LI><A href="/Team:Heidelberg/Project/LOV">Light-Induction</A></LI><LI><A href="/Team:Heidelberg/Project#Achievements">Achievements</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Parts" class="dropdown-toggle">Parts</A></LI><LI><A href="/Team:Heidelberg/Parts/RFC">RFC Heidelberg 2014</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Favorite Parts">Favorite Parts</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Sample Data Page">Sample Data Page</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#allParts">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Backbones">Backbones</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Intein Library">Intein Library</A></LI><LI><A href="/Team:Heidelberg/Parts/Part_Improvement">Part Improvement</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Software" class="dropdown-toggle">Software</A></LI><LI><A href="/Team:Heidelberg/Software/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Software/Linker_Software">Linker Software</A></LI><LI><A href="/Team:Heidelberg/Software/MidnightDoc">MidnightDoc</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Modeling" class="dropdown-toggle">Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Linker_Modeling">Linker Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Enzyme_Modeling">Enzyme Kinetics Modeling</A></LI><LI><A href="/Team:Heidelberg/Toolbox_Guide" class="dropdown-toggle">Toolbox Guide</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Human_Practice" class="dropdown-toggle">Human Practice</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Ethics">Religion &amp; Synthetic Biology</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Education">Education</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Public_Relations">Public Relations</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Experts">Experts</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Notebook" class="dropdown-toggle">Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="/Team:Heidelberg/AwesomeSheet">Database for Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook/Materials">Materials</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Notebook/Safety">Safety</A></LI></NAV></DIV><DIV class="col-lg-12"><DIV class="title-wrapper"><SPAN class="title">CIRCULARIZATION</SPAN></DIV></DIV></DIV></HEADER><DIV class="container main"><DIV class="row" style="position:relative;"><DIV class="col-lg-offset-1 col-lg-11 subtitle"><H2><P>– CONSTRUCTS</P></H2></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="well well-lg abstract"><P><B>ABSTRACT</B></P><P>...</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1"><A href="#NpuDnaE_intein_RFC_.5Bi.5D_circularization_constructs"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">NpuDnaE intein RFC [i] circularization constructs</SPAN></A></LI><LI class="toclevel-2"><A href="#Design"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-2"><A href="#Usage"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Usage</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Sortase_A_circularization_constructs"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Sortase A circularization constructs</SPAN></A></LI><LI class="toclevel-2"><A href="#Design_2"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-2"><A href="#Usage_2"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Usage</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H3><P>The most promising approaches to <A href="/Team:Heidelberg/Toolbox/Circularization" title="Team:Heidelberg/Toolbox/Circularization"> circularize</A> proteins are protein trans-splicing using <A href="/Team:Heidelberg/Project/Background" title="Team:Heidelberg/Project/Background"> split inteins</A><A href="#References">[1]</A> and Sortase A-catalyzed cyclization <A href="#References">[2]</A>. Both methods require the addition of specific proteins domains or peptides to the protein to be circularized. Consequently, on DNA level, creating circular proteins is equivalent to creating fusion proteins. However, existing protein fusion standards like <A href="http://parts.igem.org/Help:Standards/Assembly/RFC23" class="external text" rel="nofollow">RFC[23</A>] cause scars. Those scars on protein level may affect protein function and further complicate 3D-structure modeling. 
Therefore, we decided to create the new RFC[i] that allows scarless cloning of inteins. Our intein circularization constructs apply to this standard, while our sortase constructs are closely related and can be used similarly. Detailed instructions on how to use our constructs are provided in our <A href="/Team:Heidelberg/Toolbox_Guide" title="Team:Heidelberg/Toolbox Guide"> Toolbox Guide</A>. 
</P><H3><SPAN class="mw-headline" id="NpuDnaE_intein_RFC_.5Bi.5D_circularization_constructs">NpuDnaE intein RFC [i] circularization constructs</SPAN></H3><DIV class="pull-right col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 1) BBa_K1362000</SPAN><P>NpuDnaE intein RFC [i] circularization construct</P></DIV></DIV></DIV><DIV class="pull-right col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 2) BBa_K1362001</SPAN><P>NpuDnaE intein RFC[i] circularization construct (with FLAG and Smt3)</P></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Design">Design</SPAN></H4><P>Between the coding sequences of the <I>Npu</I> DnaE C-intein and the N-intein we placed <A href="http://parts.igem.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A>, an mRFP selection marker flanked by BsaI sites that can be replaced by the protein to be circularized. A strong RBS <A href="http://parts.igem.org/Part:BBa_K1362090" class="external text" rel="nofollow">(BBa_K1362090)</A> was added. A version containing Smt3 was created for use with proteins that are difficult to express. 
<A href="http://parts.igem.org/Part:BBa_K1362000" class="external text" rel="nofollow">BBa_K1362000</A> was assembled by [ref to mat/met CPEC] from PCR products of [ref to pSBX1K3], <A href="http://parts.igem.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A>, pVS07 and pVS41 <A href="#References">[3]</A>. <A href="http://parts.igem.org/Part:BBa_K1362001" class="external text" rel="nofollow">BBa_K1362001</A> was assembled by CPEC from PCR products of <A href="http://parts.igem.org/Part:BBa_K1362000" class="external text" rel="nofollow">BBa_K1362000</A> and pRSFDuet-1-mDNMT1(731-1602) <A href="#References">[4]</A>. 
</P><H4><SPAN class="mw-headline" id="Usage">Usage</SPAN></H4><DIV class="pull-right col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 3) Primer design for inserts to be used with RFC[i] split intein circularization constructs.</SPAN><P>The linker is optional. A strategy to clone long linkers is provided in the Toolbox guide.</P></DIV></DIV></DIV><P>Exteins, RFC [i] standard overhangs and BsaI sites have to be added to the coding sequence of the protein to be circularized without start- and stop codons by PCR. By Golden Gate assembly, the mRFP selection marker has to be replaced with the protein insert. After addition of an inducible promotor the circular protein is ready to be expressed. For detailed step-by-step instructions please use our <A href="/Team:Heidelberg/Toolbox_Guide" title="Team:Heidelberg/Toolbox Guide">Toolbox Guide</A>. 
</P><P>Upon expression of the fusion protein, the split intein domains reassemble to the active <A href="/Team:Heidelberg/Project/Background" title="Team:Heidelberg/Project/Background">intein</A> and thus ligate the termini of the protein to be circularized in trans-splicing reaction. 
</P><P>These constructs were successfully used to circularize <A href="/Team:Heidelberg/Project/Linker_Screening" title="Team:Heidelberg/Project/Linker Screening">lambda lysozyme</A> and <A href="/Team:Heidelberg/Project/Xylanase" title="Team:Heidelberg/Project/Xylanase">xylanase</A> and probably <A href="/Team:Heidelberg/Project/PCR_2.0" title="Team:Heidelberg/Project/PCR 2.0">DNMT1</A>. 
</P><H3><SPAN class="mw-headline" id="Sortase_A_circularization_constructs">Sortase A circularization constructs</SPAN></H3><DIV class="pull-right col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 4) BBa_K1362002</SPAN><P>Sortase A circularization construct (with His6)</P></DIV></DIV></DIV><DIV class="pull-right col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 5) BBa_K1362003</SPAN><P>Sortase A circularization construct (with Smt3 and His6)</P></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Design_2">Design</SPAN></H4><P>The mRFP selection marker <A href="http://parts.igem.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A>, which can be removed by restriction with BsaI, is flanked by TEV protease cleavage site (left) and a sorting signal (right) with a His6 tag. By Golden gate assembly, the selection marker can be replaced by the protein to be circularized. A version containing Smt3 and a FLAG tagwas created for use with proteins that are difficult to express. 
The constructs <A href="http://parts.igem.org/Part:BBa_K1362202" class="external text" rel="nofollow">BBa_K1362202</A> and <A href="http://parts.igem.org/Part:BBa_K1362003" class="external text" rel="nofollow">BBa_K1362003</A> were assembled in a row of CPEC and biobrick cloning steps from the following sources <A href="http://parts.igem.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A>, <A href="http://parts.igem.org/Part:BBa_K1362093" class="external text" rel="nofollow">BBa_K1362093</A>, <A href="http://parts.igem.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A>, pDONR and pRSFDuet-1-mDNMT1(731-1602) <A href="#References">[4]</A>. Please use our <A href="/Team:Heidelberg/Notebook" title="Team:Heidelberg/Notebook">notebook</A> for detailed information. 
</P><H4><SPAN class="mw-headline" id="Usage_2">Usage</SPAN></H4><DIV class="pull-right col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 6) Primer design for inserts to be used with sortase circularization constructs.</SPAN><P>The linker is optional. A strategy to clone long linkers is provided in the Toolbox guide.</P></DIV></DIV></DIV><P>BsaI sites and overhangs as in figure 6 have to be added to the coding sequence of the protein to be circularized without start- and stop codons by PCR. By Golden Gate assembly, the mRFP selection marker has to be replaced with the protein insert. After addition of an inducible promotor protein is ready to be expressed. The protein has to be purified and treated with TEV protease  and sortase A. The sortase A catalyzes the transpeptidation reaction that leads to backbone circularization. For detailed step-by-step instructions please use our <A href="/Team:Heidelberg/Toolbox_Guide" title="Team:Heidelberg/Toolbox Guide">Toolbox Guide</A>. 
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P>[1]   Iwai, H., Lingel, a &amp; Pluckthun, a. Cyclic green fluorescent protein produced in vivo using an artificially split PI-PfuI intein from Pyrococcus furiosus. J. Biol. Chem. 276, 16548–54 (2001).
</P><P>[2]   Antos, J. M. et al. A straight path to circular proteins. J. Biol. Chem. 284, 16028–36 (2009).
</P><P>[3]   Zettler, J., Schütz, V. &amp; Mootz, H. D. The naturally split Npu DnaE intein exhibits an extraordinarily high rate in the protein trans-splicing reaction. FEBS Lett. 583, 909–14 (2009).
</P><P>[4]   Song, J., Rechkoblit, O., Bestor, T. H. &amp; Patel, D. J. Structure of DNMT1-DNA complex reveals a role for autoinhibition in maintenance DNA methylation. Science 331, 1036–40 (2011).
</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Heidelberg/Toolbox/Circularization_Constructs">http://2014.igem.org/Team:Heidelberg/Toolbox/Circularization_Constructs</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Toolbox/Circularization_Constructs" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Toolbox/Circularization_Constructs" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Toolbox/Circularization_Constructs&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Toolbox/Circularization_Constructs&amp;oldid=249810" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>