<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Toolbox_Guide_Circularization"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Toolbox_Guide/Circularization">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Toolbox_Guide/Circularization&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Toolbox_Guide/Circularization&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Toolbox_Guide/Circularization&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Toolbox_Guide/Circularization" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Toolbox Guide/Circularization</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="myContainer" class="container"><HEADER id="navbar" role="banner" class="navbar navbar-static-top navbar-white"><DIV class="container" style=" "><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse collapse" style="margin-bottom:10px;"><NAV role="navigation"><UL class="menu nav navbar-nav navbar-right"><LI class="dropdown"><A href="/Team:Heidelberg/Team" class="dropdown-toggle">Team</A></LI><LI><A href="/Team:Heidelberg/Team/Members">Members</A></LI><LI><A href="/Team:Heidelberg/Team/Collaborations">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="/Team:Heidelberg/Project" class="dropdown-toggle">Project</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Background">Background</A></LI><LI><A href="/Team:Heidelberg/Project/Toolbox">Toolbox</A></LI><LI><A href="/Team:Heidelberg/Toolbox/Circularization">Toolbox: Circularization</A></LI><LI><A href="/Team:Heidelberg/Project/PCR_2.0">PCR 2.0</A></LI><LI><A href="/Team:Heidelberg/Project/Xylanase">Xylanase</A></LI><LI><A href="/Team:Heidelberg/Project/Linker_Screening">Linker Screening</A></LI><LI><A href="/Team:Heidelberg/Project/Reconstitution">Fluorescence Assembly</A></LI><LI><A href="/Team:Heidelberg/Project/LOV">Light-Induction</A></LI><LI><A href="/Team:Heidelberg/Project#Achievements">Achievements</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Parts" class="dropdown-toggle">Parts</A></LI><LI><A href="/Team:Heidelberg/Parts/RFC">RFC Heidelberg 2014</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Favorite Parts">Favorite Parts</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Sample Data Page">Sample Data Page</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#allParts">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Backbones">Backbones</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Intein Library">Intein Library</A></LI><LI><A href="/Team:Heidelberg/Parts/Part_Improvement">Part Improvement</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Software" class="dropdown-toggle">Software</A></LI><LI><A href="/Team:Heidelberg/Software/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Software/Linker_Software">Linker Software</A></LI><LI><A href="/Team:Heidelberg/Software/MidnightDoc">MidnightDoc</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Modeling" class="dropdown-toggle">Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Linker_Modeling">Linker Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Enzyme_Modeling">Enzyme Kinetics Modeling</A></LI><LI><A href="/Team:Heidelberg/Toolbox_Guide" class="dropdown-toggle">Toolbox Guide</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Human_Practice" class="dropdown-toggle">Human Practice</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Ethics">Religion &amp; Synthetic Biology</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Education">Education</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Public_Relations">Public Relations</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Experts">Experts</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Notebook" class="dropdown-toggle">Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="/Team:Heidelberg/AwesomeSheet">Database for Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook/Materials">Materials</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Notebook/Safety">Safety</A></LI></NAV></DIV></DIV></HEADER><DIV class="container main" style="color: white;"><DIV class="row"><DIV class="col-lg-8"><DIV class="row"><DIV class="col-lg-12"><SPAN class="middle">use the <SPAN class="red-text">intein</SPAN></SPAN></DIV></DIV><DIV class="row"><DIV class="col-lg-12" style="position: relative; top: -20px;"><SPAN class="large">TOOLBOX </SPAN><SPAN class="middle">to</SPAN></DIV></DIV></DIV><DIV class="col-lg-4 toolbox-icons-bar"><DIV class="col-lg-6 col-md-6 col-sm-6 col-xs-12"><A href="https://2014.igem.org/Team:Heidelberg/Toolbox/Circularization">CIRCULARIZE</A></DIV></DIV></DIV><DIV class="col-lg-12" style="color:white;"><H3>Please go to <A href="http://www.rcsb.org/">www.rcsb.org</A> and get a pdb file of your protein. If the 3D structure of your protein is known, we will be able to provide you an appropriate linker.</H3><H4>If the 3D structure is unknown, we will help you to find a set of potentially suitable linkers. </H4><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
					I’ve found one.
				</LABEL></DIV><DIV class="radio"><LABEL>
					The 3D structure of my protein is unknown
				</LABEL></DIV></DIV></DIV><DIV data-bind="fadeVisible: data.q9A" style="display:none;"><H4>Additionally, check the DNA sequence of your proteins for EcoRI, XbaI, SpeI, PstI and BsaI recognition sites. If there are E/X/S/P sites, you might have problems to change your backbone or add a promotor. If there is a BsaI recognition site, the cloning will be more difficult.</H4><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
						There is no BsaI site.
					</LABEL></DIV><DIV class="radio"><LABEL>
						There is a BsaI site.
					</LABEL></DIV></DIV></DIV></DIV><DIV data-bind="fadeVisible: data.q0A"><H3>Do you want to use split inteins or sortase to circularize your protein?</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
						NpuDnaE intein RFC  circularization construct
					</LABEL><UL><LI>Successfully used in our project</LI><LI>High efficiency</LI><LI>In vivo circularization</LI></UL></DIV><DIV class="radio"><LABEL>
						Sortase A circularization construct
					</LABEL><UL><LI>In vitro more efficient</LI><LI>Reversible</LI><LI>Not successfully tested by us</LI></UL></DIV></DIV></DIV></DIV><DIV data-bind="fadeVisible: data.q1A"><H3>Can your protein be easily expressed in <I>E. coli</I>?</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
						Yes, no problem.
					</LABEL></DIV><DIV class="radio"><LABEL>
						No, I will have to add <A href="http://parts.igem.org/Part:BBa_K1362429">Smt3</A>.
					</LABEL></DIV></DIV></DIV></DIV><DIV data-bind="fadeVisible: data.q2A() &amp;&amp; !data.useSortase()"><H3>Which exteins do you want to use? They will remain as scars in your circular protein.</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>/
								()
							</LABEL></DIV></DIV></DIV></DIV><DIV data-bind="fadeVisible: data.q3A() || (data.q2A() &amp;&amp; data.useSortase())"><H3>If you want to save time, check manually whether the ends are close together (approx. Å or closer). For example, you can use the <A href="http://spdbv.vital-it.ch/">Swiss-PdbViewer</A> or <A href="http://www.pymol.org/">PyMOL</A>.</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
						They are close together.
					</LABEL></DIV><DIV class="radio"><LABEL>
						I want/have to use .
					</LABEL></DIV></DIV></DIV></DIV><DIV data-bind="fadeVisible: data.q4A() &amp;&amp; !data.endsAreClose()">
	Please use  to generate a linker for your circular protein. [LINK] 

	This step might take up to 11 days.

	NILS – hier könnte dein instruction-file-ersatz stehen 


	NILS – hier auch


	NILS – hier immernoch


	NILS – hier ebnfalls und auch gerne noch umfangreicher


	Please save this link:<CODE id="comebackurl" style="display:inline-block; width:80%; overflow-y:hidden; vertical-align: middle;">https://2014.igem.org/#</CODE><DIV style="display:inline-block; width: 20%; color:#de4230; vertical-align: middle;" id="copy-to-clip">  Copy to clipboard</DIV>


	In order to generate your linker,  needs to know the scar amino acid sequence that is caused by circularization. In your case, it is .
	</DIV><DIV data-bind="fadeVisible: data.comingBack"><H3>Hello again. What is the result of ?</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
						I do not need a linker.
					</LABEL></DIV><DIV class="radio"><LABEL>
						The linker + scars are &lt; 15 amino acids in length.
					</LABEL></DIV><DIV class="radio"><LABEL>
						The linker + scars are betweeen 16 and 30 amino acids in length.
					</LABEL></DIV><DIV class="radio"><LABEL>
						The linker + scars are &gt; 30 amino acids in length.
					</LABEL></DIV></DIV></DIV></DIV><DIV data-bind="fadeVisible: data.q5A"><H3>Have you decided to use one linker or try different linkers?</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
						I trust . One should be enough.
					</LABEL></DIV><DIV class="radio"><LABEL>
						I want to test several linkers. 
					</LABEL></DIV></DIV></DIV></DIV><DIV data-bind="fadeVisible: !data.gotProteinStructure() &amp;&amp; (data.q3A() || data.q2A() &amp;&amp; data.useSortase())"><H3>How large is your protein?</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
						Less than 50 amino acids
					</LABEL></DIV><DIV class="radio"><LABEL>
						50-150 amino acids
					</LABEL></DIV><DIV class="radio"><LABEL>
						151-300 amino acids
					</LABEL></DIV><DIV class="radio"><LABEL>
						301-500 amino acids
					</LABEL></DIV><DIV class="radio"><LABEL>
						501-700 amino acids 
					</LABEL></DIV><DIV class="radio"><LABEL>
						More than 700 amino acids 
					</LABEL></DIV></DIV></DIV></DIV></DIV><DIV class="col-lg-12"><DIV data-bind="fadeVisible: (data.q4A() &amp;&amp; data.endsAreClose()) || (data.q5A() &amp;&amp; data.linkerLength() == 0) || data.q6A() || data.q7A()"><H3>Protocol:</H3><P>
				Get BBa_K136200<SPAN data-bind="if: !data.useSortase()&amp;&amp;!data.needsSmt3()">0</SPAN><SPAN data-bind="if: !data.useSortase()&amp;&amp;data.needsSmt3()">1</SPAN><SPAN data-bind="if: data.useSortase()&amp;&amp;!data.needsSmt3()">2</SPAN><SPAN data-bind="if: data.useSortase()&amp;&amp;data.needsSmt3()">3</SPAN><SPAN data-bind="if: !data.useSortase()">NpuDnaE intein RFC </SPAN><SPAN data-bind="if: data.useSortase()">Sortase A</SPAN> circularization construct <SPAN data-bind="if: !data.useSortase()&amp;&amp;data.needsSmt3()">(with FLAG and Smt3)</SPAN><SPAN data-bind="if: data.useSortase()&amp;&amp;!data.needsSmt3()">(with His6)</SPAN><SPAN data-bind="if: data.useSortase()&amp;&amp;data.needsSmt3()">(with Smt3 and His6)</SPAN> from the registry.
			</P><P>Get the DNA of the protein you want to circularize.</P><P data-bind="if: !data.gotProteinStructure()">We recommend you to try circularization without a linker and with flexible GS-linkers of different lenghts up to <SPAN data-bind="if: data.proteinSize() == 0">8</SPAN><SPAN data-bind="if: data.proteinSize() == 1">10</SPAN><SPAN data-bind="if: data.proteinSize() == 2">15</SPAN><SPAN data-bind="if: data.proteinSize() == 3">20</SPAN><SPAN data-bind="if: data.proteinSize() == 4">25</SPAN><SPAN data-bind="if: data.proteinSize() == 5">30</SPAN> amino acids (including exteins).</P> 
			Backtranslate the amino acid sequence of your linker<SPAN data-bind="if: data.testSeveral() || !data.gotProteinStructure()">s</SPAN> to a nucleic acid sequence. Be aware of:
			<UL><LI>Balanced GC-content</LI><LI>Restriction sites (especially PstI)</LI><LI>Codon usage</LI><LI>Avoid self annealing</LI></UL><P>Design and order primers for your <SPAN data-bind="if: data.linkerLength() == 2 || data.linkerLength() == 3">protein</SPAN> insert. </P><OL><LI><P>In 5'-3' direction, the forward primer should have the following sequence:</P><P>CT GGTCTCA <SPAN data-bind="if: !data.useSortase()">CAAC</SPAN><SPAN data-bind="if: data.useSortase()">GGGT</SPAN><SPAN data-bind="if: data.exteins().N == 'RGK' || data.exteins().N == 'RGT'">TGCTGGGAA</SPAN><SPAN data-bind="if: data.exteins().N == 'AEY'">TGCTTCAAC</SPAN><SPAN data-bind="if: data.exteins().N == 'XXX'">TGC</SPAN> + binding part
				(=additional random nucleotides, BsaI site, <SPAN data-bind="if: !data.useSortase()">RFC  E</SPAN><SPAN data-bind="if: data.useSortase()">overhang</SPAN><SPAN data-bind="if: data.exteins().N == 'RGK' || data.exteins().N == 'RGT'">, CWE</SPAN><SPAN data-bind="if: data.exteins().N == 'AEY'">, CFN</SPAN><SPAN data-bind="if: data.exteins().N == 'XXX'">, Cys</SPAN>)
				</P></LI><LI><P>In 5'-3' direction, the reverse primer should have the following sequence:</P><P><SPAN data-bind="if: data.linkerLength() == 2 || data.linkerLength() == 3 || data.proteinSize() == 3 || data.proteinSize() == 4 || data.proteinSize() == 5">CT GGTCTCT + binding part 
				(=additional random nucleotides, BsaI site)</SPAN><SPAN data-bind="if: !(data.linkerLength() == 2 || data.linkerLength() == 3 || data.proteinSize() == 3 || data.proteinSize() == 4 || data.proteinSize() == 5)">CT GGTCTCT <SPAN data-bind="if: !data.useSortase()">AGCA</SPAN><SPAN data-bind="if: data.useSortase()">GAAG</SPAN><SPAN data-bind="if: data.exteins().N == 'RGK'">TTTACCACG</SPAN><SPAN data-bind="if: data.exteins().N == 'RGT'"> GGTACCACG</SPAN><SPAN data-bind="if: data.exteins().N == 'AEY'">ATATTCCGC</SPAN><SPAN data-bind="if: data.linkerLength() == 1 || data.proteinSize() == 0 || data.proteinSize() == 1 || data.proteinSize() == 2">+ linker sequence (reverse complement)</SPAN> + binding part 
				(=additional random nucleotides, BsaI site, <SPAN data-bind="if: !data.useSortase()">RFC  B</SPAN><SPAN data-bind="if: data.useSortase()">overhang</SPAN><SPAN data-bind="if: data.exteins().N == 'RGK'">, RGK</SPAN><SPAN data-bind="if: data.exteins().N == 'RGT'">, RGT</SPAN><SPAN data-bind="if: data.exteins().N == 'AEY'">, AEY</SPAN><SPAN data-bind="if: data.exteins().N == 'RGK' || data.exteins().N == 'RGT' || data.exteins().N == 'AEY'"> reverse complement</SPAN>)</SPAN></P></LI><LI data-bind="style: { display: ((data.linkerLength() == 1 &amp;&amp; data.testSeveral()) || data.proteinSize() == 0 || data.proteinSize() == 1 || data.proteinSize() == 2) ? 'inline' : 'none'}"><P>For any additional linker you want to test, you just need one additional reverse primer with the corresponding reverse complement linker DNA sequence.</P></LI></OL><P>The Tm values of the binding parts should be between 60 °C and 70 °C and as similar as possible.</P><SPAN data-bind="if: data.linkerLength() == 2 || data.linkerLength() == 3 || data.proteinSize() == 3 || data.proteinSize() == 4 || data.proteinSize() == 5">
				Design and order oligos for your linker insert<SPAN data-bind="if: data.testSeveral() || !data.gotProteinStructure()">s</SPAN>. 
				<OL><LI><P>
							In 5'-3' direction, the forward <SPAN data-bind="if: !data.linkerLength() == 3">oligo</SPAN><SPAN data-bind="if: data.linkerLength() == 3">strand</SPAN> should have the following sequence:
						</P><P>
							Last 4 nucleotides of your protein + linker sequence + <SPAN data-bind="if: data.exteins().N == 'RGK'">CGTGGTAAA (=RGK)</SPAN><SPAN data-bind="if: data.exteins().N == 'RGT'">CGTGGTACC (=RGT)</SPAN><SPAN data-bind="if: data.exteins().N == 'AEY'">GCGGAATAT (=AEY)</SPAN></P></LI><LI><P>
							In 5'-3' direction, the reverse <SPAN data-bind="if: !data.linkerLength() == 3">oligo</SPAN><SPAN data-bind="if: data.linkerLength() == 3">strand</SPAN> should have the following sequence:</P><P><SPAN data-bind="if: !data.useSortase()">AGCA</SPAN><SPAN data-bind="if: data.useSortase()">GAAG</SPAN><SPAN data-bind="if: data.exteins().N == 'RGK'">TTTACCACG</SPAN><SPAN data-bind="if: data.exteins().N == 'RGT'">GGTACCACG</SPAN><SPAN data-bind="if: data.exteins().N == 'AEY'">ATATTCCGC</SPAN> + linker sequence (reverse complement)
							(=<SPAN data-bind="if: !data.useSortase()">RFC  B</SPAN><SPAN data-bind="if: data.useSortase()">overhang</SPAN><SPAN data-bind="if: data.exteins().N == 'RGK'">, RGK)</SPAN><SPAN data-bind="if: data.exteins().N == 'RGT'">, RGT</SPAN><SPAN data-bind="if: data.exteins().N == 'AEY'">, AEY</SPAN><SPAN data-bind="if: data.exteins().N == 'RGK' || data.exteins().N == 'RGT' || data.exteins().N == 'AEY'">reverse complement</SPAN>)
							
						</P></LI></OL></SPAN><SPAN data-bind="if: data.linkerLength() == 3">Probably these oligos would be longer than 90 nucleotides. In this case we recommend you to split the insert into several smaller inserts and order them as pairs of oligos. Use overhangs of 4-6 nucleotides and be careful that every overhang remains unique. 
				The largest linker we have successfully cloned this way was composed of 65 amino acids. ]
				<SPAN data-bind="if: data.testSeveral() || !data.gotProteinStructure()">For any additional linker you want to test, you just need another linker insert with the corresponding linker sequence.</SPAN></SPAN><P data-bind="if: data.linkerLength() == 2 || data.linkerLength() == 3 || data.proteinSize() == 3 || data.proteinSize() == 4 || data.proteinSize() == 5">Anneal your linker insert<SPAN data-bind="if: data.testSeveral() || !data.gotProteinStructure()">s</SPAN> as described here -LINK ZU MAT/MET ANNEALING-. </P><P>Use a Touchdown-Twostep PCR -LINK ZU MAT/MET TOUCHDOWNTWOSTEPPCR- to create your <SPAN data-bind="if: data.linkerLength() == 2 || data.linkerLength() == 3 || data.proteinSize() == 3 || data.proteinSize() == 4 || data.proteinSize() == 5">protein</SPAN> insert and purify it.</P>
			Use Golden Gate assembly –LINK ZU MAT/MET GOLDEN GATE ASSEMBLY (CYCLING) to replace the mRFP selection marker in the circularization construct with <SPAN data-bind="if: !(data.linkerLength() == 2 || data.linkerLength() == 3 || data.proteinSize() == 3 || data.proteinSize() == 4 || data.proteinSize() == 5)">your insert.</SPAN><SPAN data-bind="if: data.linkerLength() == 2 || data.linkerLength() == 3 || data.proteinSize() == 3 || data.proteinSize() == 4 || data.proteinSize() == 5">both your protein insert and your linker insert</SPAN><SPAN data-bind="if: data.testSeveral() || !data.gotProteinStructure()">s</SPAN>. 
			<P data-bind="if: data.linkerLength() == 3">Because your annealed oligos are not 5'-phosphorylated, you have to add T4-polynucleotide kinase to the Golden Gate reaction.</P><P data-bind="if: data.hasBsaI()">Since there is a BsaI site in your protein, you have to religate – LINK ZU MAT/MET RELIGATION after the Golden Gate reaction.</P><P>
			Add an inducible promotor by standard biobrick cloning or use one of our pSBX vectors –LINK ZU PSBX VECTOR-TEXT. 
			</P><P>Express your circular protein. If you are using our pSBX vectors, you can follow this protocol –LINK ZU PROTOKOLL EXPRESSION</P><SPAN data-bind="if: data.useSortase()">SILVAN SCHREIBT HIER NOCH WAS ZU SORTASE</SPAN></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Heidelberg/Toolbox_Guide/Circularization">http://2014.igem.org/Team:Heidelberg/Toolbox_Guide/Circularization</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Toolbox_Guide/Circularization" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Toolbox_Guide/Circularization" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Toolbox_Guide/Circularization&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Toolbox_Guide/Circularization&amp;oldid=384260" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>