<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Toolbox_Guide_Tagging"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Toolbox_Guide/Tagging">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Toolbox_Guide/Tagging&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Toolbox_Guide/Tagging&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Toolbox_Guide/Tagging&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Toolbox_Guide/Tagging" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Toolbox Guide/Tagging</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="myContainer" class="container"><HEADER id="navbar" role="banner" class="navbar navbar-static-top navbar-white"><DIV class="container" style=" "><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse collapse" style="margin-bottom:10px;"><NAV role="navigation"><UL class="menu nav navbar-nav navbar-right"><LI class="dropdown"><A href="/Team:Heidelberg/Team" class="dropdown-toggle">Team</A></LI><LI><A href="/Team:Heidelberg/Team/Members">Members</A></LI><LI><A href="/Team:Heidelberg/Team/Collaborations">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="/Team:Heidelberg/Project" class="dropdown-toggle">Project</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Background">Background</A></LI><LI><A href="/Team:Heidelberg/Project/Toolbox">Toolbox</A></LI><LI><A href="/Team:Heidelberg/Toolbox/Circularization">Toolbox: Circularization</A></LI><LI><A href="/Team:Heidelberg/Project/PCR_2.0">PCR 2.0</A></LI><LI><A href="/Team:Heidelberg/Project/Xylanase">Xylanase</A></LI><LI><A href="/Team:Heidelberg/Project/Linker_Screening">Linker Screening</A></LI><LI><A href="/Team:Heidelberg/Project/Reconstitution">Fluorescence Assembly</A></LI><LI><A href="/Team:Heidelberg/Project/LOV">Light-Induction</A></LI><LI><A href="/Team:Heidelberg/Project#Achievements">Achievements</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Parts" class="dropdown-toggle">Parts</A></LI><LI><A href="/Team:Heidelberg/Parts/RFC">RFC Heidelberg 2014</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Favorite Parts">Favorite Parts</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Sample Data Page">Sample Data Page</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#allParts">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Backbones">Backbones</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Intein Library">Intein Library</A></LI><LI><A href="/Team:Heidelberg/Parts/Part_Improvement">Part Improvement</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Software" class="dropdown-toggle">Software</A></LI><LI><A href="/Team:Heidelberg/Software/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Software/Linker_Software">Linker Software</A></LI><LI><A href="/Team:Heidelberg/Software/MidnightDoc">MidnightDoc</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Modeling" class="dropdown-toggle">Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Linker_Modeling">Linker Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Enzyme_Modeling">Enzyme Kinetics Modeling</A></LI><LI><A href="/Team:Heidelberg/Toolbox_Guide" class="dropdown-toggle">Toolbox Guide</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Human_Practice" class="dropdown-toggle">Human Practice</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Ethics">Religion &amp; Synthetic Biology</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Education">Education</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Public_Relations">Public Relations</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Experts">Experts</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Notebook" class="dropdown-toggle">Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="/Team:Heidelberg/AwesomeSheet">Database for Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook/Materials">Materials</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Notebook/Safety">Safety</A></LI></NAV></DIV></DIV></HEADER><DIV class="container main" style="color: white;"><DIV class="row"><DIV class="col-lg-12"><SPAN class="middle">use the <SPAN class="red-text">intein</SPAN></SPAN></DIV></DIV><DIV class="row"><DIV class="col-lg-12" style="position: relative; top: -20px;"><SPAN class="large">TOOLBOX </SPAN><SPAN class="middle">to</SPAN></DIV></DIV><DIV class="row toolbox-icons-bar"><DIV class="col-lg-2 col-lg-offset-1 col-md-offset-1 col-md-2 col-sm-4 col-xs-6"><A href="https://2014.igem.org/Team:Heidelberg/Toolbox/Circularization">CIRCULARIZE</A></DIV><DIV class="col-lg-2 col-md-2 col-sm-4 col-xs-6"><A href="https://2014.igem.org/Team:Heidelberg/Toolbox/Oligomerization">OLIGOMERIZE</A></DIV><DIV class="col-lg-2 col-md-2 col-sm-4 col-xs-6"><A href="https://2014.igem.org/Team:Heidelberg/Toolbox/Fusion">FUSE &amp; TAG</A></DIV><DIV class="col-lg-2 col-md-2 col-sm-4 col-xs-6"><A href="https://2014.igem.org/Team:Heidelberg/Toolbox/Purification">PURIFY</A></DIV><DIV class="col-lg-2 col-md-2 col-sm-4 col-xs-6"><A href="https://2014.igem.org/Team:Heidelberg/Toolbox/OnOff">ACTIVATE &amp;DEACTIVATE</A></DIV></DIV><DIV class="col-lg-12" style="color:white;"><H3>First of all, check the DNA sequence of your proteins for EcoRI, XbaI, SpeI, PstI and BsaI sites. If there are E/X/S/P sites, you might have problems to change your backbone or add a promotor. If there is a BsaI site, the cloning will be more difficult.</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
				There is no BsaI site.
			</LABEL></DIV><DIV class="radio"><LABEL>
				There is a BsaI site.
			</LABEL></DIV></DIV></DIV><DIV data-bind="if: data.q0A"><H3>Choose the type of tag you want to add or remove.</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
					iGEMHD Tags (collection of usefull tags based on Biobricks)
				</LABEL></DIV><DIV class="radio"><LABEL>
					Custom tag
				</LABEL></DIV></DIV></DIV></DIV><DIV data-bind="if: data.q1A"><H3>Please select the organism in that the tag should be used.</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL><I>Escherichia coli</I></LABEL></DIV><DIV class="radio"><LABEL>
					eukaryotic organism
				</LABEL></DIV></DIV></DIV></DIV><DIV data-bind="if: data.q2A()"><H3>Which tag would you like to use?</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL> ()
					</LABEL></DIV></DIV></DIV></DIV><DIV data-bind="if: (data.q3A() &amp;&amp; data.isiGEMHD()) || (data.q1A()  &amp;&amp; !data.isiGEMHD())"><H3>Do you want to add or remove the tag?</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
					Add tag to protein
				</LABEL></DIV><DIV class="radio"><LABEL>
					Remove tag via intein protease
				</LABEL></DIV></DIV></DIV></DIV><DIV data-bind="if: data.q4A() &amp;&amp; !data.isiGEMHD()"><H3>At which position do you want to <SPAN data-bind="if: data.addTag">add</SPAN><SPAN data-bind="if: !data.addTag()">remove</SPAN> the tag?</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
					N-Terminus
				</LABEL></DIV><DIV class="radio"><LABEL>
					C-Terminus
				</LABEL></DIV></DIV></DIV></DIV><DIV data-bind="if: (data.isiGEMHD() &amp;&amp; data.q4A() &amp;&amp; data.selectedTag().askFirstStop) || (data.q5A() &amp;&amp; !data.isiGEMHD())"><H3>Does your protein of interest contain methionin as a first amino acid?</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
					Yes
				</LABEL></DIV><DIV class="radio"><LABEL>
					No
				</LABEL></DIV></DIV></DIV><H3>Does your protein of interest contain a stop codon?</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
					Yes
				</LABEL></DIV><DIV class="radio"><LABEL>
					No
				</LABEL></DIV></DIV></DIV></DIV><DIV data-bind="if: (!data.isiGEMHD() &amp;&amp; data.q7A()) || (data.isiGEMHD() &amp;&amp; data.q7A() &amp;&amp; data.selectedTag().askFirstStop) || (data.isiGEMHD() &amp;&amp; !data.selectedTag().askFirstStop &amp;&amp; data.q4A())"><H3>How do you want to induce the trans-splicing reaction?</H3><DIV class="panel panel-default"><DIV class="panel-body"><DIV class="radio"><LABEL>
					regulation of gene expression
				</LABEL></DIV><DIV class="radio"><LABEL>
					light induction
				</LABEL></DIV></DIV></DIV></DIV><DIV data-bind="if: data.q8A"><H1>Protocol</H1><UL><LI>Get BBa_K1362105 (RBS + NpuDnaE C-intein RFC [] assembly construct) from the registry of standard biological parts.</LI><LI>Get BBa_K1362104 (RBS + NpuDnaE N-intein RFC [] assembly construct) from the registry of standard biological parts.</LI><LI>Get <SPAN data-bind="if: !data.isiGEMHD()">your tag</SPAN> from the registry of standard biological parts.</LI><LI>Get the DNA of your protein of interest.</LI><LI><SPAN data-bind="if: data.isiGEMHD() &amp;&amp; data.selectedTag().tagname == 'MinD MTS'">
		Use Golden Gate Assembly to replace the mRFP selection marker in BBa_K1362105 (RBS   + NpuDnaE C-intein RFC [] assembly construct) with  the C-terminal tag.
	</SPAN><SPAN data-bind="if: data.isiGEMHD() &amp;&amp;  data.selectedTag().tagname != 'MinD MTS'">
		Use Golden Gate Assembly to replace the mRFP selection marker in BBa_K1362104 (RBS + NpuDnaE N-intein RFC [] assembly construct) with  the N-terminal tag.
	</SPAN></LI><LI><P>Design and order primers for your tag:</P><P>In 5'-3 direction, the forward primer should have the following sequence:</P><P><SPAN data-bind="if: data.terminus() == 'C' &amp;&amp; data.addTag()">NNNNNNGGTCTCCCAACTGCTGGGAA + binding part</SPAN><SPAN data-bind="if: data.terminus() == 'N' &amp;&amp; data.addTag() &amp;&amp; !data.methionin()">NNNNNNGGTCTCAGATG + binding part</SPAN><SPAN data-bind="if: data.terminus() == 'N' &amp;&amp; data.addTag() &amp;&amp; data.methionin()">NNNNNNGGTCTCAGATG + binding part</SPAN></P><P>In 5'-3' direction, the reverse primer should have the following sequence:</P><P><SPAN data-bind="if: data.terminus() == 'C' &amp;&amp; data.addTag()">NNNNNNGGTCTCTATTA + binding part</SPAN><SPAN data-bind="if: data.terminus() == 'N' &amp;&amp; data.addTag()">NNNNNNGGTCTCGAGCACTTGCCCCT + binding part</SPAN><SPAN data-bind="if: data.terminus() == 'N' &amp;&amp; data.addTag() &amp;&amp; data.stopCodon()">Be careful not to include stop codons of your tag into the binding part.</SPAN></P></LI><LI><P>Use PCR to create your insert and purify it.</P></LI><LI><P>Use Golden Gate Assembly to replace the mRFP selection marker in 
		<SPAN data-bind="if: data.terminus() == 'C' &amp;&amp; data.addTag()">BBa_K1362105 (RBS + SspDnaE C-intein RFC [] assembly construct) with your C-terminal tag</SPAN><SPAN data-bind="if: data.terminus() == 'N' &amp;&amp; data.addTag()">BBa_K1362104 (RBS + SspDnaE N-intein RFC [] assembly construct) with your N-terminal tag.</SPAN></P></LI><LI><P>Backtranslate the amino acid sequence of your protein of interest into a nucleotide sequence.</P></LI><LI><P>Design and order primers:</P><P>In 5'-3' direction, the forward primer should have the following sequence:</P><SPAN data-bind="if: data.addTag() &amp;&amp; ((data.isiGEMHD() &amp;&amp; data.selectedTag().tagname == 'MinD MTS') || (!data.isiGEMHD() &amp;&amp; data.terminus() == 'C'))"><P><SPAN data-bind="if: !data.methionin()">NNNNNNGGTCTCAGATG + binding part</SPAN><SPAN data-bind="if: data.methionin()">NNNNNNGGTCTCAG + binding part</SPAN></P><P>In 5'-3' direction, the reverse primer should have the following sequence:</P><P>NNNNNNGGTCTCGAGCACTTGCCCCT + binding part</P><SPAN data-bind="if: data.stopCodon">
				Be careful not to include stop codons of your protein of interest into the binding part.
			</SPAN></SPAN><SPAN data-bind="if: (data.isiGEMHD() &amp;&amp; data.selectedTag().tagname != 'MinD MTS') || (!data.isiGEMHD() &amp;&amp; data.terminus() == 'N')"><P>NNNNNNGGTCTCCCAACTGCTGGGAA + binding part</P><P>In 5'-3' direction, the reverse primer should have the following sequence:</P><P>NNNNNNGGTCTCTATTA + binding part</P></SPAN></LI><LI><P>Use PCR to create your insert and purify it.</P></LI><LI><P>Use Golden Gate assembly to  to exchange the mRFP selection marker in 
			<SPAN data-bind="if: data.addTag() &amp;&amp; ((data.isiGEMHD() &amp;&amp; data.selectedTag().tagname == 'MinD MTS') || (!data.isiGEMHD() &amp;&amp; data.terminus() == 'C'))">
				BBa_K1362104 ( SspDnaE N-intein RFC [] assembly construct)  with your insert.</SPAN><SPAN data-bind="if: (data.isiGEMHD() &amp;&amp; data.selectedTag().tagname != 'MinD MTS') || (!data.isiGEMHD() &amp;&amp; data.terminus() == 'N')">
				BBa_K1362105 ( SspDnaE C-intein RFC [] assembly construct)  with your insert.
			</SPAN></P><P>Since there is a BsaI site in your protein, you have to religate after the Golden Gate reaction.</P></LI><LI><SPAN data-bind="if: !data.lightInduced()"><P>Clone both assembly constructs into compatible expression vectors.</P><P>You can use [verweise zu pSBXen].</P></SPAN><SPAN data-bind="if: data.lightInduced()"><P>Please use: 
			<A href="#">LOV</A> to induce splicing
			</P></SPAN></LI></UL></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Heidelberg/Toolbox_Guide/Tagging">http://2014.igem.org/Team:Heidelberg/Toolbox_Guide/Tagging</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Toolbox_Guide/Tagging" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Toolbox_Guide/Tagging" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Toolbox_Guide/Tagging&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Toolbox_Guide/Tagging&amp;oldid=290372" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>