"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/pages/Circularization Constructs
From 2014.igem.org
Contents
1 Introduction
2 NpuDnaE intein RFC [i] circularization constructs
2.1 Design
2.2 Usage
3 Sortase A circularization constructs
3.1 Design
3.2 Usage
4 References
Introduction
The most promising approaches to circularize proteins are protein trans-splicing using split inteins [1] and Sortase A-catalyzed cyclization [2] . Both methods require the addition of specific proteins domains or peptides to the protein to be circularized. Consequently, on DNA level, creating circular proteins is equivalent to creating fusion proteins. However, existing protein fusion standards like RFC[23 ] cause scars. Those scars on protein level may affect protein function and further complicate 3D-structure modeling.  Therefore, we decided to create the new RFC[i] that allows scarless cloning of inteins. Our intein circularization constructs apply to this standard, while our sortase constructs are closely related and can be used similarly. Detailed instructions on how to use our constructs are provided in our Toolbox Guide .
NpuDnaE intein RFC [i] circularization constructs
Figure 1) BBa_K1362000
NpuDnaE intein RFC [i] circularization construct
Figure 2) BBa_K1362001
NpuDnaE intein RFC[i] circularization construct (with FLAG and Smt3)
Design
Between the coding sequences of the Npu DnaE C-intein and the N-intein we placed BBa_J04450 , an mRFP selection marker flanked by BsaI sites that can be replaced by the protein to be circularized. A strong RBS (BBa_K1362090) was added. A version containing Smt3 was created for use with proteins that are difficult to express. BBa_K1362000 was assembled by [ref to mat/met CPEC] from PCR products of [ref to pSBX1K3], BBa_J04450 , pVS07 and pVS41 [3] . BBa_K1362001 was assembled by CPEC from PCR products of BBa_K1362000 and pRSFDuet-1-mDNMT1(731-1602) [4] .
Usage
Figure 3) Primer design for inserts to be used with RFC[i] split intein circularization constructs.
The linker is optional. A strategy to clone long linkers is provided in the Toolbox guide.
Exteins, RFC [i] standard overhangs and BsaI sites have to be added to the coding sequence of the protein to be circularized without start- and stop codons by PCR. By Golden Gate assembly, the mRFP selection marker has to be replaced with the protein insert. After addition of an inducible promotor the circular protein is ready to be expressed. For detailed step-by-step instructions please use our Toolbox Guide .
Upon expression of the fusion protein, the split intein domains reassemble to the active intein and thus ligate the termini of the protein to be circularized in trans-splicing reaction.
These constructs were successfully used to circularize lambda lysozyme and xylanase and probably DNMT1 .
Sortase A circularization constructs
Figure 4) BBa_K1362002
Sortase A circularization construct (with His6)
Figure 5) BBa_K1362003
Sortase A circularization construct (with Smt3 and His6)
Design
The mRFP selection marker BBa_J04450 , which can be removed by restriction with BsaI, is flanked by TEV protease cleavage site (left) and a sorting signal (right) with a His6 tag. By Golden gate assembly, the selection marker can be replaced by the protein to be circularized. A version containing Smt3 and a FLAG tagwas created for use with proteins that are difficult to express.  The constructs BBa_K1362202 and BBa_K1362003 were assembled in a row of CPEC and biobrick cloning steps from the following sources pSB1A3 , BBa_K1362093 , BBa_J04450 , pDONR and pRSFDuet-1-mDNMT1(731-1602) [4] . Please use our notebook for detailed information.
Usage
Figure 6) Primer design for inserts to be used with sortase circularization constructs.
The linker is optional. A strategy to clone long linkers is provided in the Toolbox guide.
BsaI sites and overhangs as in figure 6 have to be added to the coding sequence of the protein to be circularized without start- and stop codons by PCR. By Golden Gate assembly, the mRFP selection marker has to be replaced with the protein insert. After addition of an inducible promotor protein is ready to be expressed. The protein has to be purified and treated with TEV protease  and sortase A. The sortase A catalyzes the transpeptidation reaction that leads to backbone circularization. For detailed step-by-step instructions please use our Toolbox Guide .
References
[1]   Iwai, H., Lingel, a & Pluckthun, a. Cyclic green fluorescent protein produced in vivo using an artificially split PI-PfuI intein from Pyrococcus furiosus. J. Biol. Chem. 276, 16548–54 (2001).
[2]   Antos, J. M. et al. A straight path to circular proteins. J. Biol. Chem. 284, 16028–36 (2009).
[3]   Zettler, J., Schütz, V. & Mootz, H. D. The naturally split Npu DnaE intein exhibits an extraordinarily high rate in the protein trans-splicing reaction. FEBS Lett. 583, 909–14 (2009).
[4]   Song, J., Rechkoblit, O., Bestor, T. H. & Patel, D. J. Structure of DNMT1-DNA complex reveals a role for autoinhibition in maintenance DNA methylation. Science 331, 1036–40 (2011).
Retrieved from " http://2014.igem.org/Team:Heidelberg/pages/Circularization_Constructs "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
