<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Software_igemathome"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Software/igemathome">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Software/igemathome&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Software/igemathome&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Software/igemathome&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Software/igemathome" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Software/igemathome</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container" style="padding:0;background-color:white;"><HEADER id="navbar" role="banner" class="navbar navbar-static-top navbar-white"><DIV class="container" style=" background-color:black;"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse collapse" style="margin-bottom:10px;"><NAV role="navigation"><UL class="menu nav navbar-nav navbar-right"><LI class="dropdown"><A href="/Team:Heidelberg/Team" class="dropdown-toggle">Team</A></LI><LI><A href="/Team:Heidelberg/Team/Members">Members</A></LI><LI><A href="/Team:Heidelberg/Team/Collaborations">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="/Team:Heidelberg/Project" class="dropdown-toggle">Project</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Background">Background</A></LI><LI><A href="/Team:Heidelberg/Project/Toolbox">Toolbox</A></LI><LI><A href="/Team:Heidelberg/Toolbox/Circularization">Toolbox: Circularization</A></LI><LI><A href="/Team:Heidelberg/Project/PCR_2.0">PCR 2.0</A></LI><LI><A href="/Team:Heidelberg/Project/Xylanase">Xylanase</A></LI><LI><A href="/Team:Heidelberg/Project/Linker_Screening">Linker Screening</A></LI><LI><A href="/Team:Heidelberg/Project/Reconstitution">Fluorescence Assembly</A></LI><LI><A href="/Team:Heidelberg/Project/LOV">Light-Induction</A></LI><LI><A href="/Team:Heidelberg/Project#Achievements">Achievements</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Parts" class="dropdown-toggle">Parts</A></LI><LI><A href="/Team:Heidelberg/Parts/RFC">RFC Heidelberg 2014</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Favorite Parts">Favorite Parts</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Sample Data Page">Sample Data Page</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#allParts">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Backbones">Backbones</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Intein Library">Intein Library</A></LI><LI><A href="/Team:Heidelberg/Parts/Part_Improvement">Part Improvement</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Software" class="dropdown-toggle">Software</A></LI><LI><A href="/Team:Heidelberg/Software/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Software/Linker_Software">Linker Software</A></LI><LI><A href="/Team:Heidelberg/Software/MidnightDoc">MidnightDoc</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Modeling" class="dropdown-toggle">Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Linker_Modeling">Linker Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Enzyme_Modeling">Enzyme Kinetics Modeling</A></LI><LI><A href="/Team:Heidelberg/Toolbox_Guide" class="dropdown-toggle">Toolbox Guide</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Human_Practice" class="dropdown-toggle">Human Practice</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Ethics">Religion &amp; Synthetic Biology</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Education">Education</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Public_Relations">Public Relations</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Experts">Experts</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Notebook" class="dropdown-toggle">Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="/Team:Heidelberg/AwesomeSheet">Database for Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook/Materials">Materials</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Notebook/Safety">Safety</A></LI></NAV></DIV><DIV class="col-lg-12"><DIV class="title-wrapper"><SPAN class="title">iGEM@home</SPAN></DIV></DIV></DIV></HEADER><DIV class="container main"><DIV class="row" style="position:relative;"><DIV class="col-lg-offset-1 col-lg-11 subtitle"><H2><P>– iGEM´s software for distributed computing</P></H2></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><H2 class="titles"><A href="#Introduction" class="label label-default">Introduction</A><A href="#Project_Components" class="label label-default">Project Components</A><A href="#Conclusions" class="label label-default">Conclusions</A><A href="#References" class="label label-default">References</A></H2></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="well well-lg abstract"><P><B>ABSTRACT</B></P>
iGEM@home is a platform build upon the Berkeley Open Infrastructure for Network Computing (BOINC). It is supposed to deal with the common problem of insufficient computing resources for iGEM teams. Volunteers download client software that runs computations when their computer is idle. We used the platform to distribute different parts of our Linker Generation program for protein circularization. The most important contribution of our software is the extension of the BOINC framework to support Java and Python applications and the exploration of using distributed computing in the iGEM community.
<P><I>If you want to learn more about the communication platform iGEM@home, please visit the corresponding <A href="https://2014.igem.org/Team:Heidelberg/Human_Practice/igemathome" class="external text" rel="nofollow">site in the Human practice section</A>.</I></P></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2"><A href="#Overview"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Project_Components"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project Components</SPAN></A></LI><LI class="toclevel-2"><A href="#Communication_related_side:"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Communication related side:</SPAN></A></LI><LI class="toclevel-2"><A href="#Scientific_calculations:"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Scientific calculations:</SPAN></A></LI><LI class="toclevel-2"><A href="#Implementation_Details_of_the_scientific_software_component:"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Implementation Details of the scientific software component:</SPAN></A></LI><LI class="toclevel-3"><A href="#Python_Applications:"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">Python Applications:</SPAN></A></LI><LI class="toclevel-3"><A href="#Linker_Generator:"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">Linker Generator:</SPAN></A></LI><LI class="toclevel-3"><A href="#Modeller:"><SPAN class="tocnumber">2.3.3</SPAN><SPAN class="toctext">Modeller:</SPAN></A></LI><LI class="toclevel-3"><A href="#Linker_Evaluator:"><SPAN class="tocnumber">2.3.4</SPAN><SPAN class="toctext">Linker Evaluator:</SPAN></A></LI><LI class="toclevel-1"><A href="#Conclusions"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Conclusions</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H1><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">BOINC logo</SPAN></DIV></DIV></DIV>

Back in 1999 the Search for Extra-Terrestrial Intelligence (SETI) project suffered from insufficient computing resources, while trying to analyze extraterrestrial data from radio telescopes for the purpose of registering potential alien activity. To circumvent the problem of building a server cluster to analyze all the collected data, they developed a software called <A href="http://setiathome.ssl.berkeley.edu/" class="external text" rel="nofollow">SETI@home</A>. It is based on the idea to exploit the unused computing power of thousands of home PCs for scientific research. The SETI team developed a client that enlists the computer of volunteers on a central server which then distributes computation jobs to them. After the successful completion of the calculation the result is send back to the scientists.
<H3><SPAN class="mw-headline" id="Overview">Overview</SPAN></H3><P>We, the iGEM Team Heidelberg, were confronted with a similar problem. Working on protein modifications, we wanted to know what effect our modifications could have on the protein by simulating them, before stepping to expensive and time intensive wet lab screenings. Yet iGEM Teams often lack the ability to do resource intensive computations, because they have no access to server clusters. We therefore decided to start iGEM@home, as a universal platform for iGEM-Teams to distribute computing jobs to users all over the world.
</P><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Overview about iGEM@home</SPAN></DIV></DIV></DIV><P>Right now we are using the platform to <A href="/Team:Heidelberg/Modeling/Linker_Modeling" title="Team:Heidelberg/Modeling/Linker Modeling">calculate linkers</A> to enable the simple and efficient circularization of proteins. Further information on the type of computing tasks we are distributing may be found on the Modelling page. In future we hope to see the project grow and prosper when new iGEM teams take over and run their own computations on iGEM@home.
</P><H1><SPAN class="mw-headline" id="Project_Components">Project Components</SPAN></H1><P>The project is separated into two main components:
</P><OL><LI><B>Communication related side</B>: The software for the user management, communication between server and client and distribution of jobs
</LI><LI><B>Scientific calculations</B>: The computing intensive software that is distributed to the volunteers, which run the software and send back the results after completion
</LI></OL><P>Each of these components again is separated into different subcategories.
</P><H3><SPAN class="mw-headline" id="Communication_related_side:">Communication related side:</SPAN></H3><P>For the server architecture and the communication between server and client software, we relied on the Berkley Open Infrastructure for Network Computing (<A href="http://boinc.berkeley.edu/trac/wiki/BoincIntro" class="external text" rel="nofollow">BOINC</A>). It was created during the development of SETI@home, and has since then been developed further. It includes:
</P><OL><LI>User management system
</LI><LI>Job management system
</LI><LI>Database
</LI><LI>Backend Interface and forum software
</LI></OL><P>These components were adapted to our needs, for example we added scripts for the easy deployment of our jobs. In addition we implemented a workflow for distributing jobs at multiple levels of the linker generation, which includes waiting for successful results and joining them for the next step.
</P><H3><SPAN class="mw-headline" id="Scientific_calculations:">Scientific calculations:</SPAN></H3><DIV class="panel panel-default"><DIV class="panel-body&quot;" style="background-color:lightgray; padding:15px">This paragraph focuses on the implementation details of the distributed software. If you want to know more about the theory behind the modelling please visit our <A href="/Team:Heidelberg/Modeling/Linker_Modeling">Modelling page</A>.
</DIV></DIV>

We are distributing three types of software to our user base, which reflect the different steps of our modelling. Our software starts with a protein (a protein data bank file (pdb)) that one wants to circularize. The pdb is fed into the first and most important part of our software the Linker Generator. 
<OL><LI>The Linker Generator generates a list of all possible linkers that are able to connect the N- and C-terminus of the protein. It sorts out Linkers that cross the protein and then ranks them based on some features of the Linker. In the end it returns a list of the 300 best Linkers for further analysis.
</LI><LI>The server receives the list of possible Linkers and redistributes their pdb’s to the client. Now the second part of the software (Modeller) analyzes the super-secondary structure of the protein and updates the location of the amino acids of the Linker to the place that seems most natural. The pdb with the updated structure is returned to the server.
</LI><LI>The server collects all results and redistributes them to the Linker Evaluator.  It analyzes the list of all proteins, calculates an alignment score based on the FatCat Algorithm with the natural protein structure. Furthermore to evaluate the assumptions of the Linker Generator it calculates the length of the linker parts and the angles in between and writes them to a text file that allows the dynamic enhancement of the linker prediction.
</LI></OL><P>The result is a simple file that contains all important information on the process of the Linker Generation and can easily be evaluated on a single laptop.
</P><H3><SPAN class="mw-headline" id="Implementation_Details_of_the_scientific_software_component:">Implementation Details of the scientific software component:</SPAN></H3><H4><SPAN class="mw-headline" id="Python_Applications:">Python Applications:</SPAN></H4><P>As multiple components of our workflow are programmed in python, we needed to find a way to make these python applications portable and executable on computers without python installed, so they can be distributed via the BOINC system. Although python is widely installed on Linux distributions, it is underrepresented on Microsoft Windows operating systems. For the propose of packaging the python script with the required components, we implemented an extraction and version management system in the loader application, which extracts a delivered zip file into a separate folder for each application version and removes old versions if they are no longer needed. Additionally it implements methods for accessing the <A href="http://boinc.berkeley.edu/trac/wiki/BasicApi" class="external text" rel="nofollow">BOINC-API</A> via a python accessible “loader” module and file signing checking for input files to avoid licensing issues when using software for academic use only.
</P><H4><SPAN class="mw-headline" id="Linker_Generator:">Linker Generator:</SPAN></H4><P>The linker generator is written in Python. Being one of the most spread programming languages in the scientific community, Python allows to access many modules for the implementation of complex data analysis required for scientific applications. The linker generator relies heavily on the <A href="http://www.numpy.org/" class="external text" rel="nofollow">NumPy</A> module of the Python <A href="http://www.scipy.org/" class="external text" rel="nofollow">SciPy</A> package, which extends the functionality of python to cope with large multi-dimensional arrays and matrices at high computational speed. This is implemented by using compiled C-code and processor specific optimizations for time critical operations.  For distribution of the linker generator, we needed to use an alternative approach compared to the distribution of the Modeller software, as there seemed to be incompatibilities between NumPy and the software used for bundling, Nuitka . For further details on the implementation of the Linker Generator or on the principles used for bundling python applications please visit the <A href="/wiki/index.php?title=/Team:Heidelberg/Software/Linker-Generator&amp;action=edit&amp;redlink=1" class="new" title="/Team:Heidelberg/Software/Linker-Generator (page does not exist)">Linker-Generator</A> or <A href="/wiki/index.php?title=/Team:Heidelberg/Software/igemathome/Implementation&amp;action=edit&amp;redlink=1" class="new" title="/Team:Heidelberg/Software/igemathome/Implementation (page does not exist)">Bundeling Implementation</A> pages.
</P><H4><SPAN class="mw-headline" id="Modeller:">Modeller:</SPAN></H4><P>After talking to multiples experts in the field protein folding analysis, we decided to rely on the state of the art software widely used in this field of research: [<A href="https://salilab.org/modeller/|Sali" class="external text" rel="nofollow">Labs Modeller</A>] allows modeling of super secondary structures via homology analysis and discrete energy minimization. Further information on Modeller itself may be found on the <A href="/wiki/index.php?title=/Team:Heidelberg/Modeling/Linker&amp;action=edit&amp;redlink=1" class="new" title="/Team:Heidelberg/Modeling/Linker (page does not exist)"> Linker Modeling page</A>. To enable the distribution of Modeller we used a software called [<A href="http://nuitka.net/|Nuitka" class="external autonumber" rel="nofollow">[1]</A>], which defines itself as a python compiler. This means, that the python code is translated into C++-Code which results in a 2-3 fold acceleration of execution speed. Nuitka also allows embedding the complete python runtime into a single executable, thus making the application executable without a python runtime installed. Yet it was required to rewrite parts of the Nuitka software, so the process works flawlessly with the python loader. For further details please visit the <A href="/wiki/index.php?title=/Team:Heidelberg/Solftware/igemathome/implementation&amp;action=edit&amp;redlink=1" class="new" title="/Team:Heidelberg/Solftware/igemathome/implementation (page does not exist)"> implementation page</A>.
</P><H4><SPAN class="mw-headline" id="Linker_Evaluator:">Linker Evaluator:</SPAN></H4><P>The Linker Evaluator is written in pure Java and usesthe bioinformatics framework BioJava. This is in a bold contrast to the support of the BOINC framework for only the languages C and C++. One of the biggest advantages of Java is that a program can be written and debugged on a single platform and then be run on all the others, which is especially important if one targets a versatile community as with BOINC. To be able to use the Java with BOINC we decided to port the JRE to BOINC. Another approach that was previously discussed in the BOINC community  favored the use of Java that is already installed on the computer of the user, but as the number preinstalled JRE’s is declining in the last year we decide to make our program independent of this trend and deploy it as  a standalone program. 
</P><DIV class="panel panel-default"><DIV class="panel-heading">Java</DIV><DIV class="panel-body&quot;" style="background-color:lightgray; padding:15px">Java is a programming language created by Sun Corporation in 1995. Due to its rich content of libraries, simple memory management and platform independency it quickly became one of the most important and popular (TIOBE and RedMonk Index) programming languages in the world. Most programming languages including the famous C and C++ are platform dependent. That means they can only be executed on the platform on which they were compiled. In contrast Java offers with the Java Virtual Machine (JVM) a general platform in which the programs are executed and allows the developer to “write once and run anywhere”.
To achieve this independence the Java Runtime Environment (JRE), that includes the JVM, is written for each platform. So the basic idea is that each user has its own JRE installed and that any Java application can be run on it. But in the last years Java has been affected by some security issues and today many people stopped installing it. That’s why the Java community developed a tool called javafxpackager that bundles a java application directly with its own runtime environment and thereby removes the need for the user to install the JRE.

</DIV></DIV><P>We therefore utilized the tool javafxpackager that helps bundeling a JRE with a finished java program (JAR file). To incorporate it into the BOINC infrastructure we had to rewrite the Loader program, that is written in C and starts the JVM and executes the actual program.
</P><P>To reduce the size of the bundle we removed some packages from the JRE, for example all programs that are responsible for displaying graphics (AWT and Swing) and some more files as it was suggested in the JRE Readme (<A href="http://www.oracle.com/us/technologies/java/jre-7-readme-430162.html" class="external free" rel="nofollow">http://www.oracle.com/us/technologies/java/jre-7-readme-430162.html</A>).
</P><P>The BOINC architecture brings a bunch of methods that are necessary for running applications. Some of these are essential like starting the BOINC process that communicates with the BOINC manager or resolving a file name to the encoded representation that BOINC is internally using. To give users access to these method that were implemented in the C and C++ we had to write a BOINC API Wrapper. It uses the Java Native Interface that gives Java code the ability to call native C and C++ methods. Our BOINCWrapperAPI Java class is distributed with the JRE to give application developers the most natural access to its method and it implements access to the most important BOINC methods. In addition it is easy to extend due to its modularity.
</P><H1><SPAN class="mw-headline" id="Conclusions">Conclusions</SPAN></H1><P>With all this components in place it is very easy to develop applications for BOINC. One can use the popular scripting language Python together with one of the most used science frameworks: SciPy. In addition one can use Java a simple, though powerful fully featured object oriented language and basically all of its libraries to develop applications that can be scaled up from the single development machine to the distribution to thousands of home computers.
</P><P>With our effort to expand the list of supported languages by the BOINC system with more high level languages, we significantly reduced the implementation effort and required expertise for future BOINC projects and especially for future iGEM teams that want to use iGEM@home and benefit from the available computing power.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>[1] <A href="http://freegamedev.net/wiki/Portable_binaries#System_libraries_that_cannot_be_bundled" class="external free" rel="nofollow">http://freegamedev.net/wiki/Portable_binaries#System_libraries_that_cannot_be_bundled</A></P><P>[2] <A href="http://tldp.org/HOWTO/Program-Library-HOWTO/shared-libraries.html" class="external free" rel="nofollow">http://tldp.org/HOWTO/Program-Library-HOWTO/shared-libraries.html</A></P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Heidelberg/Software/igemathome">http://2014.igem.org/Team:Heidelberg/Software/igemathome</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Software/igemathome" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Software/igemathome" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Software/igemathome&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Software/igemathome&amp;oldid=390451" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>