<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_pages_PCR_2_0"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/pages/PCR_2.0">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/pages/PCR_2.0&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/pages/PCR_2.0&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/pages/PCR_2.0&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/pages/PCR_2.0" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/pages/PCR 2.0</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="panel panel-default"><DIV class="panel-body" style="background-color:lightgray; padding:15px"><DIV class="row"><DIV class="col-md-6"><P style="font-size:25px; font-weight:bold">Abstract</P><P>DNA methylation is the most abundant DNA modification and essential for embryonic development, gene regulation and genomic stability. Although several methods for the detection of methylation patterns exist, there is no easy way to amplify methylated DNA for <I>in vitro</I>  or <I>in vivo</I> studies.</P><P>To empower epigenetic research, we envisioned a PCR2.0 which maintains DNA methylation patterns during amplification. The central element of this PCR2.0 is a heat-resistant DNA methyltransferase - DNMT1 (731-1602) - which we created by circularization using our intein toolbox and the CRAUT linker software.</P><P>So far, our DNMT1 represents the largest circularized protein which highlights the usefulness of our intein toolbox in combination with the CRAUT linker software. Increased heat resistance of our circular DNMT1 which was observed in initial assays smoothens the path for the establishment of a PCR2.0 and illustrates the suitability of intein-mediated circularization for the advancement of heat resistant proteins.</P><P style="font-size:25px; font-weight:bold">Highlights</P><UL><LI>circularization seems to increase heat stability of mDNMT1 (731-1602): successful first step towards PCR2.0.</LI><LI>mDNMT1 is the largest circularized protein (105kDa) so far.</LI><LI>efficient expression and purification of active linear and circular mDNMT1 (731-1602) in E.coli.</LI><LI>successful application of CRAUT linker design software and intein toolbox for circularization.</LI></UL></DIV></DIV></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Motivation"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Motivation</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Epigenetics_-_there_is_more_than_just_A.2C_T.2C_C_and_G.E2.80.A6"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Epigenetics - there is more than just A, T, C and G…</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#A_closer_look_at_DNA_Methyltransferases"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">A closer look at DNA Methyltransferases</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Circularization_-_The_missing_link"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Circularization - The missing link</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Experimental_procedures"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Experimental procedures</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Constructs"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Expression_and_purification"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Expression and purification</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Methylation_activity_assay"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Methylation activity assay</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Results"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Optimizing_expression_and_purification_of_linear_mDNMT1_.28731-602.29"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Optimizing expression and purification of linear mDNMT1 (731-602)</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Activity_and_specificity_of_linear_mDNMT1_.28731-1602.29"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Activity and specificity of linear mDNMT1 (731-1602)</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Circular_mDNMT1_.28731-1602.29_-_The_pivotal_element_of_PCR2.0"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Circular mDNMT1 (731-1602) - The pivotal element of PCR2.0</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Expression_and_purification_of_mDNMT1_.28731-1602.29_for_circularization"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Expression and purification of mDNMT1 (731-1602) for circularization</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Thermal_stability_of_circular_mDNMT1s.28731-1602.29"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Thermal stability of circular mDNMT1s(731-1602)</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Outlook"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Outlook</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Discussion"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Achievements_and_impact"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Achievements and impact</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Considerations_for_further_PCR2.0_development"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Considerations for further PCR2.0 development</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H1><H2><SPAN class="mw-headline" id="Motivation">Motivation</SPAN></H2><P>The invention of the polymerase chain reaction in 1983 by Kary Mullis revolutionized our world by enabling amplification of DNA in an exponential manner. Further improvements including the use of thermo-stable DNA polymerases made the method even more efficient, allowing its widespread use in nearly every field of modern diagnostics and research. However, a major part of information is lost when using conventional PCR as DNA modifications are not copied. The most abundant modification is DNA methylation, which is a prominent regulator of gene expression in all kingdoms of life. Methylation and other modifications that influence the DNA function without changing its actual sequence are studied in the fast-growing field of epigenetics. Even though the detection and mapping of methylation patterns have become feasible in a high-throughput manner by using bisulfite sequencing and array techniques <A href="#References">[1]</A>, further functional analysis is still hindered by the small amount of primary material. Up to now, there is no method available to amplify methylated DNA without knowledge of methylation patterns and expensive de-novo synthesis.
</P><P>To empower epigenetics, we propose the <B>“PCR 2.0”</B> as an easy and efficient way to amplify DNA templates in an exponential manner while maintaining their specific methylation pattern. The pivotal element of this PCR reaction is a heat-stable DNA methyltransferase (DNMT). Although comparable enzymes exist in thermophile organisms <A href="#References">[2]</A>, until now no suitable protein has been found or synthesized that withstands the harsh conditions of a PCR.
</P><P>To achieve this goal of a PCR2.0 that can be used to amplify DNA including its intrinsic methylation pattern, our main aim was the generation of a heat-stable Dnmt1 that can be produced in a larger scale and shows efficient and specific methylation of DNA. It was reported that the insertion of additional intramolecular bonds can increase the stability of proteins <A href="#References">[3]</A><A href="#References">[4]</A>, and that specifically joining the C- and N-terminus improves the thermostability of smaller peptides <A href="#References">[5]</A>. This is why our approach to generate a heat-stable Dnmt1 applies:
</P><UL><LI> protein circularization using our intein toolbox
</LI><LI> creating a potent <A href="https://2014.igem.org/Team:Heidelberg/Software/Linker_Software" class="external text" rel="nofollow">Software</A> to design efficient linker peptides connecting C- and N-terminus
</LI></UL><P>To enable the PCR2.0 we started the circularization of the largest protein so far…
</P><H2><SPAN class="mw-headline" id="Epigenetics_-_there_is_more_than_just_A.2C_T.2C_C_and_G.E2.80.A6">Epigenetics - there is more than just A, T, C and G…</SPAN></H2><P>In mammals and other vertebrates DNA methylation occurs at the C5 position of cytosine nucleotides that are followed by guanines (CpG). It is a prominent key regulator of transcription, embryonic development, X chromosome inactivation and many other cellular functions <A href="#References">[6]</A><A href="#References">[7]</A>. The human genome contains about 28 million CpGs of which 60-80% are methylated <A href="#References">[28]</A>. Due to its great prevalence and the heredity of this modification, the 5-methylcytosine (5mC) is also known as the “fifth base” of eukaryotic genomes <A href="#References">[9]</A>. DNA methylation is preferentially occurring at intergenic regions and repetitive sequences, where it is known to silence gene expression <A href="#References">[10]</A><A href="#References">[11]</A><A href="#References">[12]</A>. Inappropriate cytosine methylation and silencing has been reported to play a crucial role in the development of numerous diseases, including cancer, imprinting diseases and repeat-instability based diseases such Huntington's disease <A href="#References">[13]</A>.
</P><H2><SPAN class="mw-headline" id="A_closer_look_at_DNA_Methyltransferases">A closer look at DNA Methyltransferases</SPAN></H2><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 1) Principle of mDNMT1-mediated maintenance methylation</SPAN><P>The enzyme DNMT1 is of particular interest for methylation maintenance PCR, as it allows exact duplication of the original methylation pattern to every new DNA copy.</P></DIV></DIV></DIV><P>
The family of enzymes called DNA methyltransferases (DNMTs) is responsible for the establishment and maintenance of cell-type specific DNA methylation patterns. Whereas the two enzymes DNMT3a and DNMT3b contribute to de novo methylation of DNA during development, DNMT1 is preserving existing methylation patterns throughout cell divisions. To do so, DNMT1 exploits the principle of semi-conservative replication, using the parental strand as a template to create an exact copy of methylation patterns on the daughter strand (Figure 1). After DNA synthesis, the DNA consists of a methylated and an unmethylated strand. DNMT1 recognizes these so called hemi-methylated CpG sites, and  transfers a methyl group from the methyl donor S-Adenosyl-methionine (SAM) to the cytosine belonging to the newly synthesized strand <A href="#References">[15]</A><A href="#References">[19]</A>.
</P><DIV class="pull-left col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 2) Crystal structure of full-length and truncated (731-1602) mDNMT1</SPAN><P>Structural overview of full length mDNMT1 and truncated mDNMT1 (731–1602) (A) Color-coded schematic overview of domain structure and numbering of mDNMT1 sequence. (B) Ribbon presentation of full-length DNMT1 with the CXXC, BAH1, BAH2,and methyltransferase domain colored in light blue, yellow, orange and red, respectively. (C) Crystal structure of truncated DNMT1, missing the CXXC domain and schowing more adjacent protein termini that the full-length version.</P></DIV></DIV></DIV><P>The complex structure of DNMT1 and several truncated versions have been recently solved by X-ray crystallography <A href="#References">[19]</A>. For our experiments we used the smallest truncated version of DNMT1 - mDNMT1 (731-1602) - that has been reported to efficiently mediate methylation maintenance. This truncated version is derived from <I>Mus musculus</I> and comprises the amino acids 731-1602 of the full length protein <A href="#References">[15]</A><A href="#References">[19]</A> (kindly provided by Dr. Bashtrykov after approval from Prof. Patel). Murine DNMT1 (731-1602) contains a C-terminal catalytic methyltransferase domain as well as the bromo-adjacent homology (BAH) domains, which prevent the binding of unmethylated DNA to the catalytic core. The truncated enzyme is missing the CXXC domain, which has a high affinity to hemi-methylated CpG dinucleotides, but was shown to be dispensable for protein function <A href="#References">[16]</A>. In contrast to the full-length DNMT1, mDNMT1 (731-1602) has the great advantage of being efficiently expressed in E.coli. Another benefit of the truncated DNMT1 is that the N- and C- terminus are closer together than in the full-length DNMT1 (Figure 2). Thus circularization might cause less deformation and show lower impact on the overall activity, which makes this truncated mDNMT1 an ideal candidate for our purposes.
</P><H2><SPAN class="mw-headline" id="Circularization_-_The_missing_link">Circularization - The missing link</SPAN></H2><P>Heat stabilization of mDNMT1 as a pivotal element of our methylation-maintaining PCR2.0 was approached by circularization of the protein. Until now, only smaller proteins have been stabilized using this method. Therefore, to our knowledge, mDNMT1 – even in its shortest truncated version of 871 amino acids and a molecular weight of approximately 100kDa – is the largest protein that has ever been tried to circularize. According to the crystal structure, the distance between the termini of the truncated DNMT1 is 48 Å. We suspected that circularization of the protein by direct fusion of both termini might cause deformation of the protein structure. Therefore we used our own <A href="https://2014.igem.org/Team:Heidelberg/Software/Linker_Software" class="external text" rel="nofollow">CRAUT Linker Software</A> to create a linking amino acid sequence, adapted to the structure of DNMT1. The software is able to design protein linkers with the required length to bridge the gap between the protein termini while bypassing the catalytic core of the enzyme.
</P><P>Since we could not estimate the impact of the linker introduction, we focused on two different kinds of linkers: a so called rigid linker that has been calculated and optimized during the establishment of our linker software and a flexible peptide connection consisting mostly of glycine and serine. To perform circularization we have used the split NpuDnaE Intein, since it is used as the <I>golden standard</I> for protein splicing and has been efficiently tested by us through circularization of GFP, Lysozyme and Xylanase. Moreover, we tried to implement protein circularization by using sortase A, which is recognizing and cleaving a carboxyterminal sorting signal followed by a transpeptidation reaction that can be exploited for protein circularization. This method has been reported to be very efficient <A href="#References">[4]</A> and was therefore included in our study as possible candidate for large scale productions of circular proteins.
</P><H1><SPAN class="mw-headline" id="Experimental_procedures">Experimental procedures</SPAN></H1><H2><SPAN class="mw-headline" id="Constructs">Constructs</SPAN></H2><P>We designed constructs in order to characterize the efficiency of sortase A as well as intein mediated circularization of DNMT1. Both approaches are based on the comparison of the circularized protein with a corresponding linear counterpart at different temperatures and incubation times.
</P><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Construct for circularization of mDNMT1 using inteins.</SPAN></DIV></DIV></DIV>

For circularization of DNMT1 with inteins we fused the obtained truncated version of DNMT1 (731-1602) <A href="#References">[15]</A> with an appropriate linker and split NpuDnaE domains at either site of the construct. For efficient purification of the protein, all constructs contained a hexa-histidine tag. Moreover, we have cloned the ubiquitin-like Smt3 protein from <I> Saccharomyces cerevisiae </I> in front of the DNMT1-intein complex. This attachment is used to increase the yield when expressing large mammalian proteins like DNMT1 in <I>E.coli</I><A href="#References">[17]</A>. The constructs have been designed in a way that Smt3 is no longer included in the final circular protein, and therefore cannot interfere with its function. 
All cloning steps were carried out using our new RFC[i] standard procedure. 
<P>
The mDNMT1 (731-1602) constructs that have been designed to allow sortase A mediated cirularization are flanked by the sortase A recognition sequence LPETGG on the C–terminus as well as N-terminal glycines that that can be exposed through previous TEV treatment. Since the transpeptidation via sortase requires an additional <I> in vitro </I> reaction, subsequent purification is necessary. To facilitate this step and to enrich the circular product in the sample, a hexa-histidine tag is present in the initial protein that is lost during circularization. Hence, an additional affinity chromatography via His trap can be used to separate the histidine tag containing sortase and not circularized educt from the circular flow-through. 
All cloning steps were carried out using our sortase standard. 
</P><P>The following linkers that had been optimized along with our <A href="https://2014.igem.org/Team:Heidelberg/Software/Linker_Software" class="external text" rel="nofollow">CRAUT Linker Software</A>:
</P><TABLE class="table table-hover"><TBODY><TR><TH> Definition</TH><TH> Sequence
</TH></TR><TR><TD>Rigid linker</TD><TD>    GGAEAAAKEAAAKVNLTAAEAAAKEAAAKEAAAKEAAAKEAAAKAVNLTAAEAAAKAHHHHHHSGRGT
</TD></TR><TR><TD>Flexible linker</TD><TD>     CWEGGGSGGGSGGGSGGGSGGGSGGGSGGGSGGGSGGGSGGGSHHHHHHSGRGT
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Expression_and_purification"> Expression and purification</SPAN></H2><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 3) Purification of recombinant mDNMT1</SPAN><P>The purification of recombinantly expressed mDNMT1 is a challenging task including large scale induction, efficient cell lysis, reduction of sample volume via ultracentrifugation, immobilized Ni ion affinity chromatography, dialysis and concentration through centrifugation.</P></DIV></DIV></DIV><P>In comparison to the full length murine DNMT1, our modified mDNMT1 (731-1602) can be expressed in <I>E.coli (Rosetta DE3)</I> and does not need the establishment of Baculovirus based system and the use of insect cell cultures. First experiments were conducted to determine the optimal conditions for protein induction and expression. We tested several concentrations of IPTG as well as different temperatures to maximize the yield of mDNMT1. Still, even though expression of the constructs can be simplified by using <I>E.coli</I>, purification of the recombinant protein remains a challenging task requiring multiple different steps that had to be adapted and established from published protocols <A href="#References">[19]</A>. 
</P><P>In a first step, we optimized the process of cell lysis, to be able to extract a maximum amount of active protein from our cultures. Therefore we compared the two commonly used methods sonication and disruption via French press.  Subsequently, the sample was cleared from cell debris by ultracentrifugation. The protein of interest containing a hexa-histidine tag was further enriched via immobilized metal ion affinity chromatography (IMAC) using a His trap and eluted with imidazol. In a next step, low molecular weight solutes such as salts and imidazol that would interfere with protein function were removed from the sample by dialysis. Finally, the purified protein was concentrated by size-exclusion centrifugation using filters with an appropriate mass cutoff.
</P><H2><SPAN class="mw-headline" id="Methylation_activity_assay">Methylation activity assay</SPAN></H2><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 4) Sequence of 40-mer DNA template for methylation activity assay</SPAN><P>Sequence of the 40-mer DNA template. CpG dinucleotides are shaded in gray, Sau3AI and HpaII recognition sites are depicted in red. The first CpG dinucleotide is methylated on the reverse strand.</P></DIV></DIV></DIV><P>To measure the efficiency and specificity of mDNMT1 maintenance methylation activity, we used an assay that is based on the two methylation-sensitive restriction enzymes Sau3AI and HpaII. It relies on the inhibition of restriction enzymes when methyl groups are attached to the cleavage site. We used a 40-mer DNA template with one hemi-methylated and one unmethylated CpG site which are located within the Sau3AI and HpaII restriction site, respectively (adapted from Bashtrykov et colleagues <A href="#References">[16]</A>.
</P><DIV class="pull-left col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 4) Principle of the mDNMT1 methylation activity assay</SPAN><P>Principle of mDNMT1 methylation assay using a 40-mer substrate with Sau3AI and HpaII cleavage sites that are blocked upon further CpG methylation. The lack of methylation activity leads to fragmentation of the DNA template with either of the two restriction enzymes. Whereas specific methylation of hemimethylated CpGs hampers only cleavage by Sau3AI, <I>de novo</I> methylation activity of mDNMT1 can be detected when HpaII cleavage is blocked as well.</P></DIV></DIV></DIV><P>The restriction enzyme Sau3AI is capable of cleaving the template despite of the hemimethylated state of the first CpG nucleotide. Similarly, HpaII can attack its unmethylated native site, leading to fragmentation of the template as long as no further methylation occurs. Impairment of Sau3AI cleavage indicates maintenance methylation activity, whereas a decrease in HpaII activity would result from <I>de novo</I> methylation. Therefore the assay is not only detecting activity of the enzyme but also includes its specificity for hemimethylated DNA over unmethylated DNA.
</P><P>To quantify maintenance and de novo methylation activity, the 40-mer fragments remaining after DNMT1 incubation and restriction digest were separated by gel electrophoresis. The intensities of the detected bands corresponding to different fragments were measured using ImageJ Gel after background substraction. 
</P><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><H2><SPAN class="mw-headline" id="Optimizing_expression_and_purification_of_linear_mDNMT1_.28731-602.29">Optimizing expression and purification of linear mDNMT1 (731-602)</SPAN></H2><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 5) Optimisation of induction</SPAN><P>As soon as transformed Rosetta (DE3) cultures reached an OD600 of approximately 0.6, induction of linear mDNMT1 (731-1602) was perfomed over night at either 15°C or 25°C, using different concentrations of IPTG. Higest expression of DNMT1 was observed at 15°C using 0.7mM IPTG.</P></DIV></DIV></DIV><P>Before starting circularization of DNMT1, we first tested if we can actually express and purify active linear DNMT1. Since DNMT1 expression in E.coli normally results in low yields (&lt;1mg/liter of culture) <A href="#References">[15]</A>, we started with an optimization of the expression conditions. 
</P><P>In general, low temperatures and low IPTG concentrations are known to prevent the occurrence of insoluble inclusion bodies under conditions of overexpression. Thus, we tested different IPTG concentrations and temperatures during induction in order to maximize yields of active mDNMT1 (731-1602). We observed an optimal induction of our mDNMT1 construct after incubation at 15 °C using 0.7 mM IPTG (Figure 5).
</P><P>After successful expression, the next critical step for recombinant protein purification is protein extraction. The lysis method has to be efficient but still gentle, especially when dealing with sensitive and low expressed proteins such as DNMT1. 
In order to optimize protein extraction we therefore compared cell lysis by sonification with disruption via French press (Figure 6A and 6B). Due to the low overall yield, mDNMT1 (721-1602) does not appear as a prominent band on the Coomassie-stained gel as normally expected for overexpressed recombinant proteins (Figure 6) . Since the efficiency of DNMT1 extraction could not be estimated from the Coomassie-stained gel, a western blot was performed. French press efficiently extracts soluble mDNMT1 (731-1602) whereas sonification leads to mDNMT1 accumulation in the pellet after ultracentrifugation (see Figure 6B, SN and pellet for sonification ). Indeed, the disruption of E.coli using French press has been reported to preserve the biological activity of susceptible proteins more efficiently than sonication <A href="#References">[18]</A>. Therefore we used the French press for further extraction of linear as well as circular mDNMT1 (731-1602).
</P><P>In the western blot, several other peptides with lower molecular weight were detected. These peptides possibly result from incomplete translation or degradation processes, since the construct used for initial mDNMT1 (731-1602) expression and purification (Figure 6) comprised an N-terminal hexa-histidine tag. To minimize the amount of incomplete protein products that would be enriched in the following purification, we optimized the DNMT1 construct by shifting the His-tag to the C-terminus. 
</P><P>Further purification of the protein using a His-trap resulted in successful enrichment of mDNMT1f(731-1602) (Figure 6C). Corresponding eluates were pooled and further concentrated for use in functional assays. 
</P><DIV class="pull-left col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 6A) Comparison of DNMT1 extraction from bacteria by French press and sonification</SPAN><P>mDNMT1 (731-1602) expression in Rosetta (DE3) was performed over night at 15°C. Subsequently, the bacteria were lysed by sonification or French press and ultracentrifuged. After each step, samples were collected for analysis by SDS-PAGE. The resulting protein-membrane pellets and lysate supernatants were analyzed by SDS-PAGE. (Untransformed negative control (UnTra), intervals of cell disruption (1, 2),  supernatant (SN)).</P></DIV></DIV></DIV><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 6B) Lysis via French press efficiently etracts mDNMT1 (731-1602)</SPAN><P>Western Blot of lysates was performed with an anti-hexa-histidine antibody for specific detection of mDNMT1. Full length mDNMT1 (731-1602) is indicated with an arrow.</P></DIV></DIV></DIV><DIV class="pull-left col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 6C) Purification of linear mDNMT1 (731-1602)</SPAN><P>Coomassie stained SDS-PAGE showing efficient purification of linear mDNMT1 (731-1602) via His-column. Arrow indicates peptides corresponding to the molecular weight of mDNMT1 (731-1602).</P></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Activity_and_specificity_of_linear_mDNMT1_.28731-1602.29">Activity and specificity of linear mDNMT1 (731-1602)</SPAN></H2><P>After successful expression and purification of linear DNMT1, we characterized the proteins activity and specificity for hemi-methylated DNA by performing the methylation activity assay described above. To test the maintenance methylation activity of our linear DNMT1, we performed the methylation assay with different incubation times of DNMT1 (731-1602). We observe an increasing amount of completely methylated template over time (Figure 7), represented by increasing amounts of template that are unsusceptible towards SauAI cleavage. 
</P><P>The truncated DNMT1 (731-1602) showed maximal methylation of the template after approximately 30min. Therefore, our data agrees with efficiencies that have been reported in literature for <I>wild type</I> and truncated versions of mDNMT1 <A href="#References">[15]</A><A href="#References">[16]</A> .Overall, the reaction kinetics of the linear protein seem to be promising for later use in a <B>&quot;PCR2.0&quot;</B>, assuming that circularization of the protein will not ablate its function. 
</P><P>In oder to be used in the targeted <B>&quot;PCR2.0&quot;</B> mDNMT1 needs to be highly specific for hemi-methylated sites and should not display <I>de novo</I> methylation activity. Therefore, we included HpaII digestion in our assay to determine <I>de novo</I> methylation of CpG sites. The first results prove that the purified mDNMT1 (731-1602) does not exhibit <I>de novo</I>  methylation activity (Figure 8). Even for increasing incubation times, HpaII is not repressed which indicates absence of <I>de novo</I> methylation. 
</P><P>However, the analysis of the gel pictures is difficult since restriction efficiency of HpaII was not 100% as it can be seen from the hemimethylated control. Therefore, uncut template is possibly not resulting from unspecific methylation activity of the enzyme. Further tests, especially with the circularized protein will have to be performed to confirm these initial data.
</P><DIV class="pull-left col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 7A) Maintenance Methylation activity of linear Dnmt1 (731-1602) over time</SPAN><P>Purified linear mDNMT1 (731-1602) was incubated with 40mer DNA template (see Figure4) for indicated times at 37°C. Subsequently the samples were digested with Sau3AI. Fragments were separated by PAGE and stained with ethidium bromide. (Homo me= homomethylated substrate, control for enzyme inhibition through methylation, Hemi me=hemimethylated substrate positive control for digest).</P></DIV></DIV></DIV><DIV class="pull-left col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 7B) Quantification of maintenance methylation activity of linear Dnmt1 (731-1602) over time</SPAN><P>Amount of methylation was calculated by normalizing the intensity of the upper band to the overall intensity of both bands. Graph shows technical duplicate of the assay. All measurements were performed with ImageJ.</P></DIV></DIV></DIV><DIV class="pull-left col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 8A) <I>De novo</I> methylation activity of mDNMT1</SPAN><P>Purified linear mDNMT1 (731-1602) was incubated with 40mer DNA template (see Figure4) for indicated times at 37°C. Subsequently the samples were digested with HpaII. Fragments were separated using gel-electrophoresis and visualized with ethidiumbromid staining. (Homo me= homomethylated substrate,control for enzyme inhibition through methylation, Hemi me=hemimethylated substrate positive control for digest).</P></DIV></DIV></DIV><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 8B) Quantification of linear mDNMT1 <I>de novo</I> methylation activity</SPAN><P>Amount of methylation was calculated by normalizing the intensity of the upper band to the overall intensity of both bands. All measurements were performed with ImageJ.</P></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Circular_mDNMT1_.28731-1602.29_-_The_pivotal_element_of_PCR2.0">Circular mDNMT1 (731-1602) - The pivotal element of PCR2.0</SPAN></H1><H2><SPAN class="mw-headline" id="Expression_and_purification_of_mDNMT1_.28731-1602.29_for_circularization">Expression and purification of mDNMT1 (731-1602) for circularization</SPAN></H2><P>After we succeeded in purifying active and specific linear mDNMT1, we went on with cloning, expression and purification of the circular DNMTs. 
</P><P>Expression and purification of mDNMT1 (731-1602) for the intein as well as the sortase approach were conducted as they had been established for the linear DNMT1. Elutions from the His-Trap showed an enrichment and concentration of peptides that correlate with the expected molecular weight of the truncated mDNMT1 (731-1602) (Figure 9A and 9B). Nevertheless, purification with a His-Trap alone does not seem to exclude a variety of impurities that either result from degraded mDNMT1 that still contains a His-Tag or unspecific binding to the affinity column. Advanced protocols as they were used by Song and collegues <A href="#References">[19]</A> for analysis of the proteins crystal structure are therefore including several more steps. Nevertheless, additional steps of purification increase the risk of reducing the overall protein yield and activity, necessitating greater amounts of starting material. Since the active protein should already be included and efficiently enriched in our sample despite of present impurities, we continued with the functional analysis of the sample.
</P><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 9A) Efficiency of mDNMT1 (731-1602) purification for the intein approach</SPAN><P>Induction of Rosetta (DE3) and purification were performed as described above. After each step, samples were collected for analysis by Coomassie-stained SDS-PAGE. Samples are numbered according to their chronology of collection. Arrow indicates mDNMT1 (731-1602).</P></DIV></DIV></DIV><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 9B) Sortase approach - Efficiency of Purification</SPAN><P>Induction of Rosetta (DE3) and purification were performed as described above. After each step, samples were collected for analysis by Coomassie stained SDS-PAGE. Samples are numbered according to their chronology of collection. Arrow indicates mDNMT1 (731-1602).</P></DIV></DIV></DIV><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 10) Expression and Purification of intein and sortase constructs with different linkers</SPAN><P>Expression and Purification of different constructs in Rosetta (DE3) was performed as described above. Unpurified as well as purified samples were collected for analysis by Coomassie-stained SDS-PAGE.(Ctrl indicates the untransformed control. D= no linker, R= Rigid linker, F=Flexible linker, L= linear DNMT1. Arrow indicates peptides with molecular weight corresponding to mDNMT1 (731-1602). Asterix indicates possibly spliced Intein constructs).</P></DIV></DIV></DIV><P>Our approach of heat-stabilization includes the testing of different linkers that had been calculated by our <A href="https://2014.igem.org/Team:Heidelberg/Software/Linker_Software" class="external text" rel="nofollow">CRAUT Linker Software</A>. We tested a flexible linker, consisting mostly of glycines and serines and a rigid linker, which was introduced to maximize stabilization of the protein through cicularization. Purification of the mDNMT1 (731-1602) variants with different linkers for circularization was successful (Figure 10), but the yields were lower compared to different batches of linear mDNMT1 (731-1602) that had been produced in a similar way. 
</P><P>Interestingly, a shift between mDNMT1 (731-1602) expressed from the intein construct compared to the sortase or linear versions was detected. Since protein circularization via inteins is an autocatalytic process that takes place inside the bacteria, this shift could possibly result from successful splicing activity of the inteins. The resulting product lacks Smt3 and is therefore approximately 30kDa smaller than its precursor (Figure 10, indicated by an asterix). Since intein splicing likely occurred, flexible- and rigid-linked-DNMT1 could be circularized.  
</P><P>To prove circularization by inteins, we cooperated with the core facility &quot;Protein Analysis&quot; of the German Cancer Research Center (DKFZ). In the facility, circularized and linear DNMT1 were extracted from a provided gel, digested with trypsin and analyzed by ESI mass spectrometry. Unfortunately, we do not have the final results yet but are confident to be able to present them at the jamboree.
</P><P>In contrast to the intein DNMT1 construct, mDNMT1 (731-1602) with sortase recognition sequences needs to be further processed before a circular protein is obtained. Unfortunately, TEV cleavage of the purified protein, which is necessary for subsequent circularization with sortase was not very efficient. Therefore sortase treatment did not yield in a detectable amount of processed mDNMT1 (731-1602). Therefore, we fully concentrated on the promising intein approach for further mDNMT1 (731-1602) circularization.
</P><H2><SPAN class="mw-headline" id="Thermal_stability_of_circular_mDNMT1s.28731-1602.29">Thermal stability of circular mDNMT1s(731-1602)</SPAN></H2><P>In our final assay we compared the effect of a PCR-relevant heat shock on the activity of linear and circular DNMT1s. Thereby, we wanted to examine the ability of circular DNMT1 to withstand higher temperatures and the feasibility of a PCR 2.0 using circularized DNMT1. Linear DNMT1 (L) as well as rigid linker (RC) and flexible linker-containing (FC) DNMT1s were exposed for 5s to temperatures between 55°C and 72°C that would also be reached in a PCR and subsequently analyzed for methylation activity. 
</P><P>In the gel picture (Figure 11A and B) a darker upper band in the FC sample compared to the linear DNMT1 sample indicates higher methylation activity  of circular DNMT1 with flexible Linker at all temperatures Quantification of the band intensities using Image J shows that  activity of circular DNMT1 with the flexible linker (black) is exceeding the linear and circular with rigid linker at least by a factor of 2. Surprisingly, the methylation activity of circular DNMT1 with a flexible linker seems to increase with higher temperatures.
</P><P>For significant evaluation, more data has to be collected. When applying Michael Menten kinetics, the methylation reaction for the controls already approaches saturation. Therefore, the normalization  to the original activity might be misleading. For further improvement of the assay, less volume of DNMT1 has to be taken for the enzyme reaction.
</P><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 11A) Higher maintenance methylation of circular mDNMT1 (731-1602) with flexible linker after heat-shock</SPAN><P>Circular DNMT1 with rigid linker (RC), circular DNMT1 with flexible linker (FC) and linear DNMT1 were heat shocked for 5 sec at 55°C and 60°C.</P></DIV></DIV></DIV><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 11B) Higher maintenance methylation activity of ciruclar mDNMT1 with flexible linker after heat-shock at PCR-relevant temperatures.</SPAN><P>Circular DNMT1 with rigid linker (RC), circular DNMT1 with flexible linker (FC) and linear DNMT1 were heat shocked for 5 sec at 65°C and 72°C.Methylation after one hour was measured. The dark upper band for FC indicates higher methylation activity.</P></DIV></DIV></DIV><DIV><DIV class="pull-right col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 11C) Amount of methylation activity kept after heat shock</SPAN><P>Circular DNMT1 with rigid linker (RC) in blue, circular DNMT1 with flexible linker (FC) in red and linear DNMT1 (L) in blue over a temperature range from 55°C to 72°C.</P></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Outlook">Outlook</SPAN></H2><P>Our next steps until the jamboree will be to prove efficient linkage of N- and C- terminus for our circularized mDNMT1 (731-1602) via mass spectrometry. Further methylation activity assays with circularized protein are planned to validate significance of the obtained data. Moreover, we are looking forward to use our heat-stable mDNMT1 (731-1602) in the first <B>”PCR2.0”</B>.
</P><H1><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H1><H2><SPAN class="mw-headline" id="Achievements_and_impact"> Achievements and impact</SPAN></H2><P>Taken together, we could show that intein-mediated circularization of mDNMT1 (731-1602) conceivably increased heat stability of the enzyme without impeding specific methylation activity. We proved that circularization with inteins is an easy method to increase heat stability of complex and sensitive proteins like mDNMT1 (731-1602). In comparison, other methods used for protein stabilization such as introduction of disulfide bonds or hydrophobic amino acids, require sophisticated structure analysis or non-physiological reaction conditions <A href="#References">[20]</A><A href="#References">[21]</A>. In fact, mDNMT1 (731-1602) is the largest protein that has ever been circularized so far and even more might follow.
</P><P>Our <A href="https://2014.igem.org/Team:Heidelberg/Software/Linker_Software" class="external text" rel="nofollow">CRAUT Linker Software</A> was essential to achieve circularization of mDNMT1 (731-1602) while preserving its natural function. In first experiments, the two designed linker compositions resulted in divergent heat stabilities of the protein. Our software successfully narrowed down the infinite amount of possibilities to circularize mDNMT1 (731-1602). Further experimental data of enzyme-specific linker variants will help to further improve the software predictions. Circularized mDNMT1 (731-1602) represents a valuable tool for the realization of a methylation maintaining <B>”PCR2.0”</B>. This foundational advance aims to enable broad functional analysis of DNA methylation patterns without prior knowledge on methylation frequency or dependency on other methods. For the detection of DNA methylation using bisulfite sequencing, handling of limited amounts of primary material still represents a major challenge: 
</P><TABLE class="table table-hover"><TBODY><TR><TD><I>“Future developments will undoubtedly allow information to be recovered from most genomic CpGs, the key being the ability to amplify DNA before bisulfite conversion. The capacity to capture the DNA methylome from individual cells will be critical for a full understandingof early embryonic development, cancer progression and generation of induced pluripotent stem cells.”</I> - Smallwood et al., <I>Nat. Methods (20014)</I><A href="#References">[29]</A></TD></TR></TBODY></TABLE><P>Approaches including single molecule sequencing have been reported, but have not been established in an affordable and efficient manner yet <A href="#References">[27]</A>. Furthermore, these methods are limited to the detection of methylation patterns and do not allow functional analysis without complete analysis and cost-intensive synthesis. Therefore, our <B>”PCR2.0”</B> aims to revolutionize the field of epigenetics by linking complex methylation patterns to their actual function. <I>In vitro</I> as well as <I>in vivo</I> experiments, requiring large amounts of methylated DNA will be feasible by using our <B>”PCR2.0”</B>. Those include for example the detection of DNA methylation binding proteins using Chromatin Immunoprecipitation or Electrophoretic Mobility Shift Assays. Further functional studies could investigate cellular regulation upon transfection with methylated template. 
</P><P>A very interesting application for the pharmaceutical use of our <B>&quot;PCR2.0&quot;</B> could be the large scale production of methylated gene therapy vectors. Since methylation of gene transfer vectors has been reported to extend transgene product expression by lowering cellular immunogenicity, the efficiency gene therapies might be improved with our method.<A href="#References">[22]</A></P><H2><SPAN class="mw-headline" id="Considerations_for_further_PCR2.0_development">Considerations for further PCR2.0 development</SPAN></H2><P>Since significance of the obtained methylation activity data still has to be proven in biological replicates, we could not yet precisely quantify the effect of DNMT1 stabilization by circularization. Therefore further steps in addition to the circularization of mDNMT1 (731-1602) could be necessary to obtain the required heat stability and specificity for our methylation maintaining <B>&quot;PCR 2.0&quot;</B>. Those for example include further protein stabilization by enhancing the hydrophobicity of the protein core or by introducing direct links to limit possible conformations <A href="#References">[23]</A>.
</P><P>Nevertheless, it is unlikely that mDNMT1 (731-1602) can be stabilized in a way to bear temperatures of 95°C, which are essential for conventional PCR. Therefore, another upcoming topic is the method of low temperature PCRs that exploit the function of helicases to segregate DNA strands. Even though these PCRs can be performed at lower temperatures than conventional PCRs, none of them has reached widespread establishment yet. Main reasons are that isothermal PCRs require either too long incubation times per cycle <A href="#References">[24]</A> or still exceed physiological temperatures of most enzymes, introducing new candidates for circularization... <A href="#References">[25]</A></P><P>Despite enduring activity of mDNMT1 (731-1602) at high temperatures, the heat-stability of its cofactors also plays a crucial role for establishing a <B>”PCR 2.0”</B>. Here, a still challenging factor is methyl-donor S-adenosyl methionine (SAM) that exhibits only low stability at high temperatures <A href="#References">[30]</A>. Nevertheless, addition of the quite cheap methyl-donor after each cycle of replication could circumvent this problem until a more stable variant of the cofactor is available.
</P><P>If there are occurring problems with the specificity of the enzyme at higher temperatures which have not been assessable from our data, the <B>“PCR 2.0”</B> could be complemented with enzymes such as UHRF1, that have been reported to increase the specificity of mDNMT1 (731-1602) <A href="#References">[31]</A>. Wild type DNMT1 is known to be associated to the replication complex for direction of its activity. Design of an appropriate Polymerase-DNMT-fusion protein could imitate this connection and therefore further reduce stochastic methylation events  <A href="#References">[26]</A>.
</P><P>Overall, the iGEM Team Heidelberg has achieved important milestones in establishing a methylation maintaining <B>&quot;PCR 2.0&quot;</B> by expressing and purifying the largest circular protein ever. Despite of its use in a <B>&quot;PCR 2.0&quot;</B>, the stabilized mDNMT1 (731-1602) can also be used in other methylation assays.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>[1] Meissner, A. et al. Genome-scale DNA methylation maps of pluripotent and differentiated cells. Nature 454, 766–70 (2008).
</P><P>[2] Watanabe, M., Yuzawa, H., Handa, N., &amp; Kobayashi, I.. Hyperthermophilic DNA methyltransferase M.PabI from the archaeon Pyrococcus abyssi. Applied and Environmental Microbiology, 72(8), 5367–75 (2006).
</P><P>[3] Vieille, C. &amp; Zeikus, G. Hyperthermophilic enzymes: sources, uses, and molecular mechanisms for thermostability. Microbiol. Mol. Biol. Rev. 65, 1–43 (2001).
</P><P>[4] Antos, J. M., Popp, M. W.-L., Ernst, R., Chew, G.-L., Spooner, E., &amp; Ploegh, H. L.. A straight path to circular proteins. The Journal of Biological Chemistry, 284(23), 16028–36 (2009).
</P><P>[5] Tam, J. P. &amp; Wong, C. T. T. Chemical synthesis of circular proteins. J. Biol. Chem. 287, 27020–5 (2012).
</P><P>[6] Robertson, K. D. DNA methylation and human disease. Nat. Rev. Genet. 6, 597–610 (2005).
</P><P>[7] Jaenisch, R. &amp; Bird, A. Epigenetic regulation of gene expression: how the genome integrates intrinsic and environmental signals. Nat. Genet. 33 Suppl, 245–54 (2003).
</P><P>[8] Ehrlich, M.. Amount and distribution of 5-methycytosine in human DNA from different types of tissues or cells. Nucleic Acids Res. 10: 2709–2721 (1982).
</P><P>[9] Lister, R. &amp; Ecker, J. Finding the fifth base: genome-wide sequencing of cytosine methylation. Genome Res. 959–966 (2009).
</P><P>[10] Bird, A.. DNA methylation patterns and epigenetic memory. Genes &amp; Development 16, 6-21 (2002).
</P><P>[11] Jaenisch, R., and Bird, A.. Epigenetic regulation of gene expression: how the genome integrates intrinsic and environmental signals. Nat Genet 33, 245-254 (2003).
</P><P>[12] Weichenhan, D., and Plass, C.. The evolving epigenome. Human Molecular Genetics 22, R1 (2013).
</P><P>[13] Robertson, K. D. DNA methylation and human disease. Nat. Rev. Genet. 6, 597–610 (2005).
</P><P>[14] Jurkowska, R. Z., Jurkowski, T. P. &amp; Jeltsch, A. Structure and function of mammalian DNA methyltransferases. Chembiochem 12, 206–22 (2011).
</P><P>[15] Song, J., Teplova, M., Ishibe-Murakami, S. &amp; Patel, D. J. Structure-based mechanistic insights into DNMT1-mediated maintenance DNA methylation. Science 335, 709–12 (2012).
</P><P>[16] Bashtrykov, P. et al. Specificity of Dnmt1 for methylation of hemimethylated CpG sites resides in its catalytic domain. Chem. Biol. 19, 572–8 (2012).
</P><P>[17] Lee, C.-D., Sun, H.-C., Hu, S.-M., Chiu, C.-F., Homhuan, A., Liang, S.-M., Wang, T.-F. An improved SUMO fusion protein system for effective production of native proteins. Protein Science : A Publication of the Protein Society, 17(7), 1241–8 (2008).
</P><P>[18] Benov, L. &amp; Al-ibraheem, J. Disrupting Escherichia coli : A Comparison of Methods. 35, 428–431 (2002).
</P><P>[19] Song, J., Rechkoblit, O., Bestor, T. H. &amp; Patel, D. J. Structure of DNMT1-DNA complex reveals a role for autoinhibition in maintenance DNA methylation. Science 331, 1036–40 (2011).
</P><P>[20] Betz, S. F. Disulfide bonds and the stability of globular proteins. 1551–1558 (1993).
</P><P>[21] Bulaj, G. Formation of disulfide bonds in proteins and peptides. Biotechnol. Adv. 23, 87–92 (2005).
</P><P>[22] Reyes-Sandoval, a &amp; Ertl, H. C. J. CpG methylation of a plasmid vector results in extended transgene product expression by circumventing induction of immune responses. Mol. Ther. 9, 249–61 (2004).
</P><P>[23] Vieille, C. &amp; Zeikus, G. Hyperthermophilic enzymes: sources, uses, and molecular mechanisms for thermostability. Microbiol. Mol. Biol. Rev. 65, 1–43 (2001).
</P><P>[24] Vincent, M., Xu, Y. &amp; Kong, H. Helicase-dependent isothermal DNA amplification. EMBO Rep. 5, 795–800 (2004).
</P><P>[25] Xu, G. et al. Cross priming amplification: mechanism and optimization for isothermal DNA amplification. Sci. Rep. 2, 246 (2012).
</P><P>[26] Vilkaitis, G., Suetake, I., Klimasauskas, S. &amp; Tajima, S. Processive methylation of hemimethylated CpG sites by mouse Dnmt1 DNA methyltransferase. J. Biol. Chem. 280, 64–72 (2005).
</P><P>[27] Krueger, F., Kreck, B., Franke, A. &amp; Andrews, S. R. DNA methylome analysis using short bisulfite sequencing data. Nat. Methods 9, 145–51 (2012).
</P><P>[28] Smith, Z. D. &amp; Meissner, A. DNA methylation: roles in mammalian development. Nat. Rev. Genet. 14, 204–20 (2013).
</P><P>[29] Smallwood, S. a et al. Single-cell genome-wide bisulfite sequencing for assessing epigenetic heterogeneity. Nat. Methods 11, 6–11 (2014).
</P><P>[30] Matos, J. &amp; Wong, C. S-adenosylmethionine: Stability and stabilization. Bioorg. Chem. 80, 71–80 (1987). 
</P><P>[31] Bashtrykov, P., Jankevicius, G., Jurkowska, R. Z., Ragozin, S. &amp; Jeltsch, A. The UHRF1 protein stimulates the activity and specificity of the maintenance DNA methyltransferase DNMT1 by an allosteric mechanism. J. Biol. Chem. 289, 4106–15 (2014).
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Heidelberg/pages/PCR_2.0">http://2014.igem.org/Team:Heidelberg/pages/PCR_2.0</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/pages/PCR_2.0" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/pages/PCR_2.0" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/pages/PCR_2.0&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/pages/PCR_2.0&amp;oldid=398911" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>