<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="myContainer" class="container-fluid"><HEADER id="navbar" role="banner" class="navbar navbar-static-top navbar-white"><DIV class="container" style=" "><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse collapse" style="margin-bottom:10px;"><NAV role="navigation"><UL class="menu nav navbar-nav navbar-right"><LI class="dropdown"><A href="/Team:Heidelberg/Team" class="dropdown-toggle">Team</A></LI><LI><A href="/Team:Heidelberg/Team/Members">Members</A></LI><LI><A href="/Team:Heidelberg/Team/Collaborations">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="/Team:Heidelberg/Project" class="dropdown-toggle">Project</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Background">Background</A></LI><LI><A href="/Team:Heidelberg/Project/Toolbox">Toolbox</A></LI><LI><A href="/Team:Heidelberg/Toolbox/Circularization">Toolbox: Circularization</A></LI><LI><A href="/Team:Heidelberg/Project/PCR_2.0">PCR 2.0</A></LI><LI><A href="/Team:Heidelberg/Project/Xylanase">Xylanase</A></LI><LI><A href="/Team:Heidelberg/Project/Linker_Screening">Linker Screening</A></LI><LI><A href="/Team:Heidelberg/Project/Reconstitution">Fluorescence Assembly</A></LI><LI><A href="/Team:Heidelberg/Project/LOV">Light-Induction</A></LI><LI><A href="/Team:Heidelberg/Project#Achievements">Achievements</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Parts" class="dropdown-toggle">Parts</A></LI><LI><A href="/Team:Heidelberg/Parts/RFC">RFC Heidelberg 2014</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Favorite Parts">Favorite Parts</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Sample Data Page">Sample Data Page</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#allParts">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Backbones">Backbones</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Intein Library">Intein Library</A></LI><LI><A href="/Team:Heidelberg/Parts/Part_Improvement">Part Improvement</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Software" class="dropdown-toggle">Software</A></LI><LI><A href="/Team:Heidelberg/Software/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Software/Linker_Software">Linker Software</A></LI><LI><A href="/Team:Heidelberg/Software/MidnightDoc">MidnightDoc</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Modeling" class="dropdown-toggle">Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Linker_Modeling">Linker Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Enzyme_Modeling">Enzyme Kinetics Modeling</A></LI><LI><A href="/Team:Heidelberg/Toolbox_Guide" class="dropdown-toggle">Toolbox Guide</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Human_Practice" class="dropdown-toggle">Human Practice</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Ethics">Religion &amp; Synthetic Biology</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Education">Education</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Public_Relations">Public Relations</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Experts">Experts</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Notebook" class="dropdown-toggle">Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="/Team:Heidelberg/AwesomeSheet">Database for Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook/Materials">Materials</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Notebook/Safety">Safety</A></LI></NAV></DIV><DIV class="col-lg-12"><DIV class="title-wrapper"><SPAN class="title"><SPAN style="font-size: 35px;">Welcome to the</SPAN> <SPAN style="color:#F3CA11;">MidnightDoc</SPAN></SPAN></DIV></DIV></DIV></HEADER><DIV class="container-fluid main" style="color: white;"><DIV class="row" style="position:relative;"><DIV class="col-lg-offset-1 col-lg-11 subtitle"><H2><P>– the different way of documentation</P></H2></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="panel panel-default"><DIV class="panel-body" style="background-color:lightgray; padding:15px"><DIV class="row"><DIV class="col-md-12" style="color:black;"><H4>Welcome to our graphical lab notebook representation.</H4>
This view was generated from <A href="https://2014.igem.org/wiki/index.php?title=Team:Heidelberg/Software/MidnightDoc">MidnightDoc</A>, our new wetlab documentation system. Each circle below represents one experiment, for example a restriction digest or a PCR reaction. The grey lines show the connections between the experiments and let you retrace our steps. Scroll with your mousewheel or click on the weekly descriptions on the right to jump to the corresponding entries, hover over the circles to see what's in there, and click on them to get a detailed description.
</DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><H2 class="titles"><SPAN class="label label-default active" id="general">General</SPAN><SPAN class="label label-default" id="dnmt1">DNMT I</SPAN><SPAN class="label label-default" id="lov">LOV</SPAN><SPAN class="label label-default" id="linker-screening">Linker-Screening</SPAN><SPAN class="label label-default" id="toolbox">Toolbox</SPAN><SPAN class="label label-default" id="xylanase">Xylanase</SPAN></H2></DIV></DIV><DIV class="row" style="height:90vh;"><DIV id="outerWrapper" class="col-lg-3" style="height:100%;overflow-y:scroll; padding-left:0; padding-right:0;"><DIV id="innerWrapper" style="width:100%;"><DIV class="weekentry general" data-timestamp="1402268400"><DIV class="weekbox"><P>Providing basic tools and supplies for working in the wet lab:</P><P>Making competent cells, transformation and plasmid purification of some parts shipped with the iGEM distribution. We also designed our first circularisation constructs and ordered primer. Furthermore intein constructs we planned to used were requested from different groups or ordered from addgene.</P></DIV></DIV><DIV class="weekentry general" data-timestamp="1407106800"><DIV class="weekbox"><P>In order to see that inteins splice in our lab too, we coexpressed Ssp and Npu Dna E C and N intein constructs and analysed transsplicing using SDS-PAGE and Western Blotting with Penta His Antibodies.</P></DIV></DIV><DIV class="weekentry general" data-timestamp="1411340400"><DIV class="weekbox"><P>Parts Submission</P><P>Almost all parts we created are in our pSBX expression vectors. Consequently, they all have to be cloned into pSB1C3 in order to be sent in. We began getting an overview over our parts and deciding which should be sent in. </P><P>We also prepared a lot of pSB1C3 cut with E and P.</P></DIV></DIV><DIV class="weekentry general" data-timestamp="1411945200"><DIV class="weekbox"><P>Parts Submission:</P><P>A first load of parts was cloned into pSBX1C3. We only used colony PCR to test whether the cloning was successful, since the parts itself had already been sequenced. </P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1402268400"><DIV class="weekbox"><P>The AsLOV2 domain was chosen as the most promising candidate for photo-activating the splicing reaction of inteins. We met with Dominik Niopek and developed a plan for photocaging inteins in the C-terminal J-alpha helix of the LOV domain.</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1402873200"><DIV class="weekbox"><P>Caging the C-intein within the J-alpha helix requires very small amino-acid sequences. We discovered artificially split inteins (S11) with extremely small C-inteins (6aa), namely Ssp DnaX, Ssp GyrB and Ter DnaE3 (Xiang-Qin Liu et al. April 2013) during literature research.</P><P>The inteins were back-translated from AA-sequences, codon-optimised for E.Coli K12.</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1403478000"><DIV class="weekbox"><P>A phototoxicity test for E. Coli was planned. Synthetic split inteins were ordered from life-technologies. The outlines for a split fluorescent protein assay was made and constructs were designed for fusing any two split proteins together -&gt; Toolbox Assembly Construct</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1404082800"><DIV class="weekbox"><P>To determine if illumination had any toxic effect on our E. coli strains of choice we conducted a phototoxicity assay. Cultures were induced for different amounts of time and their ability to grow was deduced from the number of colony forming units (CFUs). The test showed no effect on bacterial growth.</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1404687600"><DIV class="weekbox"><P>Induction test: HEK 293 cells expressing an mCherry-AsLOV2-NLS construct were illuminated in our chamber and relocalisation was observed under the flourescent microscope to test if photoinduction works using our 470nm LEDs. The images show clear induction-dependent relocalisation of the cytoplasmic LOV-bound mCherry to the nucleus.</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1405292400"><DIV class="weekbox"><P>Primers for the intein assembly constructs were designed and ordered this week.</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1405897200"><DIV class="weekbox"><P>Positions for the caging were determined. It seemed clear that a broader screening is necessary, as there is no completely reliable way of predicting the behaviour of the changed J-alpha. Primers arrived and the assembly construct cloning started.</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1406502000"><DIV class="weekbox"><P>Toolbox Assembly construct cloning made some problems, CPEC doesn’t seem to work as planned. “Backup” primers were designed and ordered to first assemble the inserts via PCA and then add X, S and P restriction sites to gain the constructs with standard cloning.</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1407106800"><DIV class="weekbox"><P>We selected different reporter proteins from the registry to run simple screening assays for inteins. Therefore we chose to split GFP, mRFP, mCherry, Renilla Luciferase and Firefly Luciferase at diverse positions.</P><P>GFP at positions 64/65 and 157/158, mRFP at positions 154/155, and 168/169, mCherry at position 168/169, RenLuc at position 229/230, FireLuc at position 437/438</P><P>Upon splicing reaction of the inteins the protein halves should reassemble again and restore their functionality. The reporters were bisected using a PCR reaction which simultaneously inserted overhangs including BsaI sides that are compatible with our fusion standard.</P><P>The backup strategy worked and Assembly Constructs were finished.</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1407711600"><DIV class="weekbox"><P>We developed a fast track cloning strategy to achieve a working split FP assay system required for screening the LOV variants earlier and more certainly. Therefore we designed and ordered new CPEC primers.</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1408316400"><DIV class="weekbox"><P>Primers arrived. We conducted PCRs and CPEC-assemblies for the fast track split GFP constructs including positive and negative control constructs.</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1408921200"><DIV class="weekbox"><P>N-terminal sfGFP half  (split 1-64) was cloned in front of the NpuDnaE-N split intein and the C-terminal sfGFP half (split 65-225) after the NpuDnaE-C intein. Also a non-splicing control was obtained by inserting a C-&gt;G mutation in NpuDnaE-N and a SN-&gt; AG mutation in NpuDnaE-C, which prevents the inteins from splicing.</P><P>The C-terminal version was cloned behind the N-terminal construct using normal biobrick cloning (insertion in S and P restriction sides). The constructs were transformed in BL21 to for expression. The cultures (N-half, N-half with mutation, C-half, C-half with mutation, coexpressing construct) were lysed and prepared for a readout in the plate reader to measure the reconstitution of fluorescence.</P><P>The split sfGFP assay was run on the plate-reader. Results look good. </P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1409526000"><DIV class="weekbox"><P>The sfGFP experiment was repeated on the FACS and with the right positive control This includes a T to C mutation on position 65 which will T65C mutation will be introduced by the splicing reaction of the inteins.</P><P>Since very small inteins are required for caging in the alpha helix of the LOV domain, we cloned SspDnaX, GyrB and SspDnaE3 in our standard 2-Gate construct of the Toolbox using CPEC.</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1410130800"><DIV class="weekbox"><P>Insertion of His in 2Gate -&gt; Results in 1Gate constructs</P><P>Insertion of all split FP variants in 1Gate Npu</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1410735600"><DIV class="weekbox"><P>Preliminary assay to screen for the best split fluorescent protein.2x mRFP, mCherry and GFP were tested and the decision was made for split mRFP (168/169).Design of fast track LOV constructs including mRFP (168/169) version and sfGFP(64/65) followed.</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1411340400"><DIV class="weekbox"><P>The Primers for LOV constructs finally arrived and cloning started. Parts were PCR amplified and assembled via CPEC into pSBX. The constructs were then transformed into TOP10, prepped and prepared for sequencing.</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1411945200"><DIV class="weekbox"><P>SDS-PAGE and western blot were run with the samples from sfGFP time series. Results didn’t look too good, we repeated the western blot and ran a gel with coomassie stain. The western blot showed a problem with the expression of the N-construct.</P><P>CPECs for LOV constructs were sequenced, unfortunately only few worked. New clones were picked, additionally CPEC was repeated. Parts were then cloned on one backbone for coexpression. We transformed them into BL21(DE3) and made preparation for the first assay. </P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1412550000"><DIV class="weekbox"><P>The first DnaX assay was run. Additionally we light-induced all LOV-construct we already had finished cloning. The first assay was a complete mess: time-points were messed up and the FACS settings were wrong. The few reliable data we got did not show any splicing activity of the DnaX-S11 intein.</P><P>We repeated the assay with longer expression (10h + 20h) at room temperature to ensure the inteins had enough time fully splice.</P></DIV></DIV><DIV class="weekentry lov" data-timestamp="1413154800"><DIV class="weekbox"><P>Western blot from the last assay was evaluated. The final LOV assay was run.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1402268400"><DIV class="weekbox"><P>First literature research revealed Dnmt1 as interesting candidate for heat stabilisation through circularization. Correspondence with Prof. Dr. Rippe,  Prof. Dr. Plass, Prof. Dr. Lyko (University of Heidelberg) and Prof. Dr. Jeltsch (University Stuttgart) for detailed information.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1402873200"><DIV class="weekbox"><P>Full length protein expression only seems to be possible in insect cells using Baculovirus system Dr. Weichenhan (DKFZ). Further literature research and correspondence with Dr. Bashtrykov revealed Dnmt1 (731-1602) as shortest active and specific version that can be expressed in E.coli.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1403478000"><DIV class="weekbox"><P>Dnmt1 (731-1602) construct (Song et al., Science 2010) provision by Dr. Bashtrykov upon approval from Prof. Dr. Song. Planning of protein purification with His column and further assays including the proof of cirularisation and activity/specificity</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1404082800"><DIV class="weekbox"><P>A pRSFDuet-1-derived plasmid encoding murine DNMT1(731-1602) (Song et al., Science 2010) under the control of a T7 promoter was obtained and sequenced. Protein purification was prepared with AG Prof. Dr. Russel (Bioquant). E. coli Rosetta(DE3) were transformed for expression, and induction with IPTG was performed in a 3 l scale.</P><P>The constructs for circularization of the truncated DNMT1 were planned using the obtained and verified sequences.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1404687600"><DIV class="weekbox"><P>The cells expressing DNMT1 were harvested and lysed by sonication or using a french press.</P><P>Primers for the sortase cloning standard were designed and ordered.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1405292400"><DIV class="weekbox"><P>The efficiencies of DNMT1 expression and the different lysis methods were analyzed by SDS-PAGE and western blotting. Cloning of circularisation standard construct with sortases proceeded.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1405897200"><DIV class="weekbox"><P>Two further lots (1.5 l culture-scale) of mDNMT1(731-1602) were prepared to rule out that the low expression level of the first try was caused by human error. Substrates for the activity assays were designed and ordered.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1406502000"><DIV class="weekbox"><P>Native mDNMT1(731-1602) was purified from the first two rounds of expression using Ni-NTA affinity chromatography. SDS-PAGE analysis of the eluted fractions showed successful expression of Dnmt1(731-1602) at low but detectable levels.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1407106800"><DIV class="weekbox"><P>The purification of the native mDNMT1(731-1602) was finished. To verify expression, an analytic gel was made from samples of the purified lots. Nanodrop measure and band portion on the gel was used to estimate concentration of purified linear DNMT1. Material for methylation activity assay was prepared: annealing of DNA substrate, reaction buffer, TAE buffer for polyacrylamid gel.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1407711600"><DIV class="weekbox"><P>This week was used to proof feasibility of the methylation assay for DNMT1. We establish the conditions for restriction digest detection via polyacrylamid gel. Starting with pretty bad results of blurry gels, we changed  from TAE to TBE gels and determined the right volume of restriction enzymes and reaction times for complete cleavage of the DNA substrates.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1408316400"><DIV class="weekbox"><P>After setting up the right conditions for the DNA-Page, we tried to run the DNA methylated by Dnmt1. Unfortunately we had some complications with the gel polymerisation and salt concentration. We improved the conditions.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1408921200"><DIV class="weekbox"><P>We continued to test activity of Dnmt1. After changing the reaction buffer which before consists out of HEPES as buffer substrate to Tris and adapting the pH to 7.0 we finally managed to get nicely and sharp looking gel pictures.</P><P>We also received our first precursor linkers for Dnmt1 out of the Modelling software and ordered them as Oligos. Using the RFC standard we constructed our standard construct for circularisation with inteins and Sortase A. The constructs contain a mRFP or ccdB selection marker as placeholder for different linkers.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1409526000"><DIV class="weekbox"><P>We tested our linear Dnmt protein at 37°C and 65°C and over a wide timespan. We could clearly see an increase in methylation over time. Although we are using a truncated version of Dnmt1 our protein is strongly active and quite specific as it almost no de novo methylation could be detected. The starting construct for circularisation was transformed in our pSBX1K3 Expression vector.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1410130800"><DIV class="weekbox"><P>Further cloning of circularisation standard for Dnmt1(731-1602) using golden gate cloning after our RFC standard method was conducted.</P><P>Dnmt1 was exposed to a temperature gradient between 37°C to 57°C and behaviour was characterised. Unfortunately this method failed, as DNMT1 did not show any methylation even at 37°C.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1410735600"><DIV class="weekbox"><P>Essays with new substances was run to ensure methylation was still possible. The linker oligos arrived and were cloned into the circularisation construct.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1411340400"><DIV class="weekbox"><P>We expressed of 4 circular DNMT1 constructs in Rosetta (DE3). Constructs were: DNMT1 without linker and with rigid linker for sortase circularisation and DNMT1 with flexible linker and with rigid linker for circularisation with inteins.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1411945200"><DIV class="weekbox"><P>The four previously expressed proteins were lysed using french press and purified using a His trap column. Analytic gels were run to ensure purification of DNMT1. In parallel, sortase reaction was tried using GFP, but showed not to be working. Subsequently we decided to drop sortase due to little time left. </P><P>Samples were prepared for mass spectrometric analysis of circularisation..First activity assays were run, showing activity of all modified proteins.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1412550000"><DIV class="weekbox"><P>Behaviour of circular DNMT1 compared to linear DNMT1 after a 5 sec heat shock at PCR conditions was analysed. </P><P>In another experiment, we attempted to prove circularisation with thrombin cleavage, which did not work to not pure enough sample solution.</P></DIV></DIV><DIV class="weekentry dnmt1" data-timestamp="1413154800"><DIV class="weekbox"><P>Additional activity assays to support the data were run.</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1403478000"><DIV class="weekbox"><P>First PCRs: With ordered primers to extract inteins Ssp DnaE and Npu DnaE CDS for plasmids that we got from Addgene.</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1404082800"><DIV class="weekbox"><P>In this week we did our first CPECs to build our pre-circularization standard with a ccdb insert as protein of interest. We build constructs with and without a His-tag. Constructs with a His-tag we could use for Western Blots.</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1404687600"><DIV class="weekbox"><P>Toolbox:</P><P>Assembly Construct: Planning for a universal two-part assembly construct. The construct should be available for all inteins and enable the user to scarlessly clone any protein inside using Golden Gate Cloning.</P><P>Circularization-Test:</P><P>By cutting it with BsaI and a GFP CDS amplified by PCR with restriction sites with EcoRI and SpeI we used our first Npu Circularisation construct to yield a first GFP circularisation construct.</P><P>Expression vectors:</P><P>As we planned to generate a whole bunch of genes whose function need to be tested after an induced expression, we decided to construct expression backbones with a strong inducible promoter in front of the the Biobrick prefix to get rid of one extra cloning step for all constructs. We chose a well established LacI repressible T7 promoter.</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1405292400"><DIV class="weekbox"><P>Toolbox:</P><P>Assembly Construct: A standard was designed for universal restriction overhangs that lie inside the N- and C-terminal splicing sites of most inteins (N: Cystein, C: Asparagin). This standard will allow to clone a protein once PCR-amplified together with any of our inteins. Should also be adopted for the Circularisation Construct.</P><P>Expression vektors: Oligos containing the lacI repressible T7 promoter were cloned into the EcoRI site of the BioBrick prefix of all pSB1X3 and pSB4X5</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1405897200"><DIV class="weekbox"><P>Construct was cloned into an expression vector (pSBX1C3) with NpuDnaE-GFP-H6 and Ssp-GFP-H6 (ID 44,45).</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1406502000"><DIV class="weekbox"><P>Literature research on different applications of split inteins. Collection of ideas what can be included in a toolbox.</P><P>Circularization-Test:</P><P>First Inductions for Western Blots next week. Incubating at 37°C for 4h, induction with IPTG.</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1407106800"><DIV class="weekbox"><P>A concept for the toolbox has been set including all major possibilities split inteins provide: circularization, fusion (including translocation by tagging, posttranslational modifications, purification, oligomerization), and the intein protease. </P><P>For test on circularisation: </P><P>Really bad blot, you could not see anything in matters of circularisation. Lane of the GFP-control (pSB1C3-BBa_I746907) is overloaded. In the next blot I have to use different volumes of the different lanes.</P><P>Expression Vektors:</P><P>pSBX Expression vectors were cloned and tested qualitatively on expression.</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1407711600"><DIV class="weekbox"><P>Development of a cloning strategy for the intein protease construct based on the split intein SspDnaB. Planning of a cloning strategy for a test construct to show intein protease activity. It incorporates GFP fused to a membrane targeting signal, which is cleaved off by the intein protease. Primer order for these two constructs.</P><P>Circularization-Test:</P><P>Inducing several samples and driving first Western Blots. Samples on the Western Blots include two different inteins, Npu DnaE and Ssp DnaE, as well as two different backbones. Problems with Western Blots, so I tried different protocols and methods. Last Western blot was quite better but I mentioned that one of my constructs must have a mistake in the sequence. Checking the sequence showed that my foreboding was right. So I decided to clone this construct again. Furthermore I decided to continue only with one of two used backbones.</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1408316400"><DIV class="weekbox"><P>Cloning of the intein protease was begun. Cloning of the NpuDnaE assembly construct was begun.</P><P>Circularization:</P><P>In this week I cloned the second construct called pSB1AK8-BBa_I712074-2ZSspH6:GFP. At the end of the week I induced with IPTG this constructs after growing to an OD of 0.8 for 4h at 37°C.</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1408921200"><DIV class="weekbox"><P>Cloning of the test construct to show intein protease activity was begun. Cloning of the NpuDnaE assembly construct was successful (proven by sequencing)</P><P>Expression Vektors:</P><P>Quantitative expression test with FACS scan. + Sequencing</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1409526000"><DIV class="weekbox"><P>Cloning of the intein protease construct was successful (proven by sequencing). Cloning of the test construct for the intein protease was not successful.</P><P>Development of a cloning strategy for the translocation tags and the SsrA degradation tag. The strategy was to clone these tags into pSB1C3 carrier plasmids containing BsaI sites to enable users of the registry of standard biological parts to easily clone these tags into intein assembly constructs by Golden Gate assembly.</P><P>Expression vektors:</P><P>Expression vectors were tested with facs after induction of 2h, 12h, 24h. Furthermore they were given to Aachen to be tested with their own designed fluorescence measurement device.</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1410130800"><DIV class="weekbox"><P>Primer order for the tags and a chitin binding domain (for the purufication tool)</P><P>Cloning of the tags was begun by attaching BsaI overhangs by PCR. Cloning of the carrier constructs was begun.</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1410735600"><DIV class="weekbox"><P>The PCR products of the tags (containing BsaI restriction sites) were cloned into the carrier constructs by Golden Gate assemby. Positive sequencing results.</P><P>Cloning of SspDnaB 2Gate assembly constructs was begun (SSpDnaB N, SspDnaB C, SspDnaB  N*, SspDnaB C*)</P><P>OmpA was cloned into a carrier construct as the last remaining tag. </P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1411340400"><DIV class="weekbox"><P>OmpA cloning was successful.</P><P>Cloning of the SspDnaB assembly construct failed.</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1411945200"><DIV class="weekbox"><P>Cloning of the SspDnaB assembly consructs failed again. The mistake was the usage of a wrong template for the CPECs.</P><P>A new attempt was started.</P></DIV></DIV><DIV class="weekentry toolbox" data-timestamp="1412550000"><DIV class="weekbox"><P>The rest of the parts was successfully cloned into pSB1C3. Additionally, mRFP selection maker inserts were cloned into our pSBX vectors in order to fulfill the submission criteria.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1402268400"><DIV class="weekbox"><P>Software for calculating the best path for semi-rigid linkers between the beginning and the end was started. At first a parser for PDB files was written in python. A calibration of lengths was made based on the C-alpha distances in the protein. The test whether the ends could be directly linked was inserted and it was decided how the general setup would be. The linkers will be stored in arrays of the points where the angles lie. A maximum linker would have 3 additional angles.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1402873200"><DIV class="weekbox"><P>The function that sorts linkers out, if the angles don't fit was introduced. All the paths were generated, first tests about the runtimes were performed and first sorting steps were made. The decision for the brute-forcing method was made.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1403478000"><DIV class="weekbox"><P>The function that sorts the linkers out if they pass through the protein was included.</P><P>First complete run of the software was made. The protein can now be shown correctly and all sorting steps run in principle.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1404082800"><DIV class="weekbox"><P>The function that sorts the linkers out if they pass through the protein was included.</P><P>First complete run of the software was made. The protein can now be shown correctly and all sorting steps run in principle.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1404687600"><DIV class="weekbox"><P>The last sort-out step of the last linkers (second to thirdpoints) was made to be running. This was difficult as it is just a huge amount of data. It needed to be stored on the harddisk using h5py module in Python. Also the weighting of the distance from the protein was introduced.</P><P>Also the variables were now sliced in shapes so that the calculations would fit into the RAM.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1405292400"><DIV class="weekbox"><P>In this week we mainly searched for a screening protein. In the end of the week we decided to take GFP and lambda-phage lysozyme as targets. So there was lots of literature-work done concerning lysozyme. It was tried to get substrates for the screenings. We came up with the plan to use the light-cycler for screening of GFP.</P><P>Softwarewise the recognition of PDBs was refined. Now the user can choose many more different things concerning the PDB, for example if he wants to ignore whole subunits or just single residues. Also it was allowed to have missing parts in the PDB and flexible ends at the linkers.</P><P>We also met Prof. Wade who gave us valuable council on structure predicting software and emphasized that the linker shouldn't become too rigid. She also came up with the idea of helix-turn-helix motifs.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1405897200"><DIV class="weekbox"><P>Wet lab:</P><P>In order to screen linkers generated by our software, we need an easy way to circularize several proteins with different linkers. The circularization construct that is already beeing used for GFP causes some scars due to cloning. Since it would be hard for the software to precisely deal with scars, we had to invent a new circularization construct. Split NpuDnaE seems to work fine, so we chose to use only this intein. The main difference compared to our old circularization construct is that the BsaI overhangs are now in the inteins.</P><P>Software:</P><P>Setting up a chart for the coverage of each project.We discussed the modularity of the linkers, first ideas for the library were exchanged, we decided not to submit a library for all possible linkers to the iGEM community, because it wouldn't help the other teams and would make lots of work for us. But we wanted to submit a smaller library, that can be used in 80% of the cases. A refined linker for DNMT1 was given to the DNMT1 group.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1406502000"><DIV class="weekbox"><P>Wet lab:</P><P>Our plan was to assemble our new circularization construct, 2Zv2Npu in one step by CPEC cloning. We planned to use three inserts, NpuC, mrfp selection marker and NpuN, which will be assembled into one of our pSBX vectors. Most of the necessary primers were already available from the assembly constucts. Only two new primers had to be designed and ordered. Additionally, we also created primers for a CPEC to get linear lambda-lysozyme and GFP into and expression vector as biobricks.</P><P>Software:</P><P>In this week some bugs in the function that sorts points out, if the linkers pass through the protein were eliminated. The weighting of different angles was also included in the weighting function.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1407106800"><DIV class="weekbox"><P>Wet lab:</P><P>This week's aim was to get our circularization construct (2Zv2Npu) and linear lambda-lysozyme (LambdaLys(lin)) in an expression vector via CPEC. Inserts and backbones with overhangs were created via PCR. Since the assembly via CPEC failed, we repeated those PCRs and conducted a lot of CPECs again. The CPECs continued failing. An attempt to use PCA to assemble the 3 inserts of 2Zv2Npu followed by CPEC to clone the PCA product into the backbone failed. Using a new protocol and a different backbone, the CPEC of pSBX-LambdaLys(lin) finally worked. We designed new primers for a PCA that could be used to clone the PCA product into the backbone via biobrick-cloning.</P><P>Software:</P><P>This week was mainly literature work and discussion about the screening for the calibration of the software. We set up the plan to divide the screen ing into two parts: The in silico screening for the different linkerpatterns done on the iGEM@home system and the in vivo screening for the calibration of the weighting function. In this week we also elaborated the screening set up and clarified how large our library would need to be. There were also papers read on the supersecondary structure, that provide us the building blocks for our linkers. The value of the different patterns was evaluated and first thoughts on the set up of the library were made. Software-wise, a short-track for small proteins was implemented and the flexible ends were really taken into account.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1407711600"><DIV class="weekbox"><P>Wet lab:</P><P>As last week, our aim was still to get pSBX-2Zv2Npu. In addition, we wanted to express LambdaLys(lin). For some mysterious reason, we ordered primers for inserts that can be cloned into our pSBX-2Zv2Npu (LambdyLys(rigid), LambdaLys(H6) = Lambda-Lysozyme with different Linkers for circularization). pSBX1K3-LambdaLys(lin) was successfully (!) cloned into BL21(DE3) and a 100ml culture was induced with IPTG. A CPEC of the PCA-product and pSBX1K3 with overhangs finally seemed to work, but sequencing results were not yet available.Since pSBX1K3 contains a BsmBI site, we cloned the potential 2Zv2Npu-biobrick into pSBX4C5. An attempt to get LambdyLys(rigid) and LambdaLys(H6) into pSBX1K3-2Zv2Npu via GoldenGate cloning failed.</P><P>Software:</P><P>In this week we have coded the translation from the software calculated paths to amino acids sequences. It was some literature work to be done for the right angle-patterns. On the other hand the set up of the screening library was refined. We decided not to screen a whole library of 60 linkers but we should construct it and then just take some few representative linkers out. For the screening-proteins GFP was discarded as the ends are too close together. Softwarewise also the clustering of linkers with the same structure was finished and the scoring on the clusters was implemented. A preliminary angle-library was also included in the software. First good linkers were obtained from the software and pictures of them were made. Another important step was the modelling of ligands, that should be omitted with sizes.  The software ran for the first time from beginning to the end and provided good sequences to us. The search for a target protein for screening began again, leading to the idea to use hemoglobin.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1408316400"><DIV class="weekbox"><P>Wet lab:</P><P>The lysozyme assay was planned andwas performed twice and first data was obtained. There was also writtena script that analyzes the data from the platereader automatically andproduces plots. This week we wanted to create the plasmids to circularize lambda lysozyme with some test linkers. BsaI sites, overhangs and the linker were added to the lambda lysozyme gene via primer overhangs. Goldengate cloning worked fine, but sequencing showed that there were mismatches in the primer region. Other clones might be sequenced as well, but there is not much hope. That is why we have put more efforts in designing our final inserts that do not rely on large overhangs.</P><P>Software:</P><P>A script for the archdb library for linkerturns was written, but the results have to get interpretated.</P>
			The lysozyme assay was planned andwas performed twice and first data was obtained. There was also writtena script that analyzes the data from the platereader automatically andproduces plots. 
		</DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1408921200"><DIV class="weekbox"><P>Wet lab:</P><P>Our aim was to create a plasmid that circularizes lambda-lysozyme with any linker that can easily be added. Therefore the mrfp selection marker in the circularization standard plasmid had to be exchanged with both a lambda-lys insert and a new selection marker, ccdb. First attempts using Golden Gate cloning + religation failed. However, using new competent cells both the Golden Gate attempt and the alternative, a 3-way ligation, worked well. About the attempts to insert lambda-lys + some test linkers at once: “Other clones might be sequenced as well, but there is not much hope.” (see last week) - Other clones were indeed successfully sequenced.</P><P>Software:</P><P>The coordination with the iGEM@home project was planned.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1409526000"><DIV class="weekbox"><P>Software:</P><P>In this week we achieved the setting up of the various softwares (Modeller, Linkerconstruction) for the BOINC system. We got an overview over the function of modeller and have seen, that it is not relying on loopdatabases for loopmodelling, so it can be used for our purpose. </P><P>Wet lab:</P><P>The sequencing results for the circularization construct with both the lambda lysozyme and ccdb insert were ok for both pSBX4C5 and 1K3. Moreover, we designed oligos to clone linear lambda lysozyme with his6 and exteins by Golden Gate assembly. This was needed to prove circularization of lambda lysozyme with his6 linker on a western blot. </P><P>The lysozyme assays were further evaluated and we have seen more strange results. We decided to have a closer look on the different errors occurring in the process of the assay and refined it again, to see the variability in wells, that should contain the same material. Furthermore we decided that a new substrate for lysozyme would be needed and we found a promising approach, that we want to test next week. We also decided to try some enzyme-kinetics modelling on the lysozyme data, that hopefully would make the analysis of the data more easy.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1410130800"><DIV class="weekbox"><P>Wet lab:</P><P>Linear lambda lysozyme with his6 and exteins was successfully created by Golden gate cloning. A new attempt to get circular lamba lysozyme with a flexible linker failed, this time by cloning the linker insert into the circularization construct with lambda lysozyme and ccdb.</P><P>In this week we performed many assays with the new circularized lysozymes to see a shift in temperature behaviour for the different kind of lysozyme. The results weren't that clear, so we tested many different temperatures. We also saw, that activity goes down for circular lysozymes at 75°C, whereas activity of linear lysozyme there has a maximum.</P><P>Software:</P><P>A version of the linkersoftware for distribution was started, butthere were many problems, as it is quite memory consuming. Therefore this took us lots of work to rewrite the software, so that it takes less RAM of the clients.Also the workflow of the in silico screening needed to be implemented, where the linker team was needed.Another big effort has been made in the analysis of the archdb database, where a new sort of plots was made. Now analysis can be much easier and the preliminary linker patterns were fixed. A problem was that the dnmt1 and the lysozyme group waited heavily for good linkers.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1410735600"><DIV class="weekbox"><P>In this week the analysis of the lysozyme assays was refined. Real error bars were included and also fitting of Michaelis-Menten kinetics was included. But the fitting of Michaelis-Menten kinetics isn't as reliable as we would need it to be for an automatic analysis.</P><P>Furthermore a substrate concentrations against OD curve was measured, so that the substrate concentration can be calculated from OD curves. In this week we also set up the list of linkers to be tested in the lysozyme assay. Also for DNMT1 the software gave out completely software-predicted linkers.</P><P>Linkersoftwarewise many bugfixes were made and it was tested to run it on our distributed computing system.The Golden Gate assembly to get circular lambda lysozyme with a flexible linker was successfully repeated. However, the plasmid could not be cloned into Bl21 (DE3), our expression strain. We could never really figure out the reason, but something must be wrong with pSBX4C5.</P><P>The amino acid sequences of 20 new linkers generated by the linker software were backtranslated, split into shorter oligos and ordered.</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1411340400"><DIV class="weekbox"><P>In this week we mainly focused on how to analyse the data from the lysozyme assays, different modelling approaches were taken and lots of refinements to the existing assays were discussed. We now want to try whether lysozyme is able to recover within a night and want to test, whether it would help if we add protease inhibitors to the protein mix. We started to set up an enzyme kinetics system to have a better modelling of the data achieved in multiple experiments.</P><P>Softwarewise we have made several bugfixes and worked a lot on running-stability of the software, so now we will try to distribute it, when the servers run again.</P><P>Some of the linkers that arrived earlier than the rest were cloned into pSBX4C5-2Zv2Npu:Lalys+ccdb by Golden Gate assembly. Again, the cloning and sequencing was successful for most of the plasmids, but they could not be transformed into Bl21 (DE3). </P><P>Everything was repeated using pSBX1K3-... instead. We also conducted the Golden gate assemblies for the remaining 15 linkers. We changed our Golden gate protocol to 50 cycles as recommended by the team Freiburg 2012</P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1411945200"><DIV class="weekbox"><P>The sequencing was successful for the linkers may1, sgt2, sho1, sho2 and flex. bad1 and sgt1 had mutations. Transformation into Bl21 (DE3) was successful. </P><P>The sequencing results for the other 15 linkers were disastrous. Only 2 of 15 linkers, ord1 and ord3 were ok. In most of the cases, completely different overhangs had ligated. Because of this bad ratio, we decided not to sequence further clones. We also decided not to retry the Golden Gate assembly, because their would not be enough time to screen all linkers anyway. </P><P>Since we did not have any bad linker yet, we decided to use standard biobrick cloning to switch the backbone of bad1 from pSBX4C5, which caused expression problems, to pSBX1K3. </P></DIV></DIV><DIV class="weekentry linker-screening" data-timestamp="1412550000"><DIV class="weekbox"><P>Cloning for bad1 was finished</P></DIV></DIV><DIV class="weekentry xylanase" data-timestamp="1407106800"><DIV class="weekbox"><P>Starting a new project for circularization of a protein. The requirements were:</P><UL><LI>Easy to circularize</LI><LI>Cool and useful application</LI><LI>Good chances for increased heat stability</LI></UL></DIV></DIV><DIV class="weekentry xylanase" data-timestamp="1407711600"><DIV class="weekbox"><P>The aim of this project is to circulate Xylanase to increase its stability against heat, pH shifts and exopeptidases. Xylanase was amplified from the genome of Bacillus subtilis 168 with primers that fit our circularization standard. These primers include BsaI recognition sides which enable golden gate cloning.</P><P>For different constructs were amplified:</P><OL><LI>Xylanase with a His tag that fits in the circularization standard.
				</LI><LI>Xylanase for our standard without His tag.</LI><LI>Xylanase that fits in our standard, but contains mutated C-extein residues. Cysteine of the C- extein which is essential for the intein splicing reaction is mutated to Glycine. This construct serves as negative control</LI><LI>Xylanase that fits in the biobrick standard</LI></OL><P>The constructs were cloned in the circularization standard, which itself is not yet proved for functionality.Sequencing is still requested.
			</P></DIV></DIV><DIV class="weekentry xylanase" data-timestamp="1408316400"><DIV class="weekbox"><P>Our aim was to insert the xylanase gene into the circularization standard plasmid. Since the xylanase gene contains a BsaI site, this was (is) not a trivial task. It was hard to find colonies that were not red (a mrfp selection marker was replaced by the insert). We really hope for positive sequencing results.</P><P>A first attempt to get linear xylanase by simple biobrick cloning failed.</P></DIV></DIV><DIV class="weekentry xylanase" data-timestamp="1408921200"><DIV class="weekbox"><P>The sequencing results for circular xylanase with his6 tag and the version with a mutated c-extein were positive. The Golden Gate assembly of circular xylanase without his6 tag had to be repeated. This time it was succesful. </P></DIV></DIV><DIV class="weekentry xylanase" data-timestamp="1409526000"><DIV class="weekbox"><P>Primers were ordered for linear Xylanase controls with his6 tag (functional control) and his6 tag + exteins (control for gel).</P></DIV></DIV><DIV class="weekentry xylanase" data-timestamp="1410130800"><DIV class="weekbox"><P>A new attempt to get linear xylanase in pSXB1K3 was successful</P></DIV></DIV><DIV class="weekentry xylanase" data-timestamp="1410735600"><DIV class="weekbox"><P>The Golden Gate assemblies for linear Xylanase controls with his6 tag (functional control) and his6 tag + exteins (control for gel) were successful.</P></DIV></DIV><DIV class="weekentry xylanase" data-timestamp="1411340400"><DIV class="weekbox"><P>As the constructs are all finished by now, the bacteria were induced and the different Xylanase were expressed. A first Coomassi Gel was run to check for a shift of the Xylanase band due to the faster traveling of the circular proteins. This was succesful for the constructs without His-Tag, but there was some problem with the sample of the circular xylanase with His-Tag.</P></DIV></DIV><DIV class="weekentry xylanase" data-timestamp="1411945200"><DIV class="weekbox"><P>The first heat shock assays were prepared. Coomassie Gels were run to ensure that the same amounts enzyme. The best estimation for the ratio between linear and circular concentration is 1:8. As a next step the Xylanase activity assay was tested. To ensure optimal measurements the gain of the plate reader, in which the assay was conducted, had to calibrated.</P></DIV></DIV><DIV class="weekentry xylanase" data-timestamp="1412550000"><DIV class="weekbox"><P>The heat shock assays to find out how our modifications influenced the enzyme were conducted. First the assays with the purified linear and circular Xylanases with His-Tag were conducted. But because of problems in equalizing the protein activity and amounts at 37 °C, later the week the assays were repeated with the non-purified linear and circular Xylanases.</P></DIV></DIV></DIV></DIV><DIV class="col-lg-1" style="height:100%; position:relative;"><A href="#" class="month active" style="top:5%;" data-timestamp="1402268400"><SPAN class="month-text">june </SPAN></A><A href="#" class="month" style="top:25%;" data-timestamp="1404169200"><SPAN class="month-text">july </SPAN></A><A href="#" class="month" style="top:45%;" data-timestamp="1406847600"><SPAN class="month-text">august </SPAN></A><A href="#" class="month" style="top:65%;" data-timestamp="1409526000"><SPAN class="month-text">september </SPAN></A><A href="#" class="month" style="top:85%;" data-timestamp="1412118000"><SPAN class="month-text">october </SPAN></A></DIV></DIV></DIV></DIV><DIV id="viewExperimentModal" class="modal fade"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal"><SPAN aria-hidden="true">×</SPAN><SPAN class="sr-only">Close</SPAN></BUTTON></DIV><DIV class="modal-body" data-bind="ifnot: loading"><P data-bind="if: experiment().performer">Performed by </P><TABLE id="experimentsTable" class="hierarchical"><TBODY><TR><TH class="section" colspan="2"><H3>Parameters</H3></TH></TR><TR><TH class="ul"><DIV class="li">:</DIV></TH></TR><TR><TH class="section" colspan="2"><H3>Steps</H3></TH></TR><TR><TH><H3>Finished on</H3></TH></TR><TR><TH><H3>Notes</H3></TH></TR><TR><TH class="section" colspan="2"><H3>Products</H3></TH></TR><TR><TD data-bind="with: $root.experiment().products[md5(name)]"><H4>
									()
								</H4></TD></TR></TBODY></TABLE></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Heidelberg/Notebook">http://2014.igem.org/Team:Heidelberg/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook&amp;oldid=400710" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>