<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_pages_Circularization"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/pages/Circularization">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/pages/Circularization&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/pages/Circularization&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/pages/Circularization&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/pages/Circularization" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/pages/Circularization</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Circular_Proteins_in_Nature"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Circular Proteins in Nature</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Synthetically_Circularized_Proteins"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Synthetically Circularized Proteins</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Our_Circularization_Project"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Our Circularization Project</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#The_Impact_of_Circularization_on_Thermostability"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">The Impact of Circularization on Thermostability</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Why_introducing_circularization_to_iGEM.3F"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Why introducing circularization to iGEM?</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H3><P>Enzymes represent a major tool for many branches of the chemical industry, including food, brewing, paper, detergent or biofuel. Millions of years of evolution have allowed these proteins to performed extremely specific chemical modifications that are not only essential for living organisms but can also be of great benefit to produce useful molecules for our life, efficiently and at low cost. A major limitation of the use of enzyme for industrial application and in general out of their natural environment is their stability. They can be destroyed by other enzymes and they can unfold and take non-functional conformation when exposed to non-physiological temperature and pH. Such limitations has motivated research in species that can grow at extreme temperatures <A href="#References">[1]</A>. Another major area of chemical research is the design of strategies to stabilize enzymes, and more generally proteins and peptides <A href="#References">[2][3]</A>. Protein circularization, meaning ligation of the N- and C-terminal ends of a protein, represents a promising way to achieve this stabilization. While conserving the functionality of their linear counterpart, circular proteins can be superior in terms of thermostability <A href="#References">[4][5][6]</A>, resistance against chemical denaturation <A href="#References">[7]</A> and protection from exopeptidases <A href="#References">[5][7]</A>. Moreover, a circular backbone can improve <I>in vivo</I> stability of therapeutical proteins and peptides <A href="#References">[8]</A>. All these remarkable properties motivated us to develop new tools to circularize any protein of interest. 
Our <A href="/Team:Heidelberg/Toolbox_Guide" title="Team:Heidelberg/Toolbox Guide"> Toolbox Guide</A> provides a step-by-step strategy to clone a circularization linker and express it in <I>E. coli</I>. Moreover, in case of complex structures where the protein extremities are far from each other, we have developed the software tool <A href="/Team:Heidelberg/Software/Linker_Software" title="Team:Heidelberg/Software/Linker Software"> CRAUT</A> that will design the appropriate rigid linkers.
</P><H3><SPAN class="mw-headline" id="Circular_Proteins_in_Nature">Circular Proteins in Nature</SPAN></H3><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 1) 3D Structure and sequence of the cyclotide katala B1 <A href="https://2014.igem.org/Team:Heidelberg/Toolbox/Circularization#References">[15]</A></SPAN><P>The terminal amino acids G and N are linked, resulting in a circular backbone. Six cysteine residues form a knot that greatly enhances stability. (source)</P></DIV></DIV></DIV>

Due to their extraordinary features, circular proteins have emerged independently in different organisms. Small circular peptides from microorganisms that are produced by non-ribosomal peptide synthesis are well-known and used in medicine. For example, cyclosporine, a small circular peptide from fungi, is able to suppress the human immune system and thus important for transplant surgery <A href="#References">[9]</A>. 
<P>A different type of circular peptides, the plant cyclotides, were discovered more recently <A href="#References">[10]</A>. Those circular peptides are true proteins, since their linear precursors are produced by ribosomes <A href="#References">[9]</A>. The peptide ligase catalyzing the circularization of cyclotides, butelase 1, was discovered a few months ago <A href="#References">[11]</A>. 
Cyclotides are rather small (&lt;79 aa) and most often them are additionally stabilized by disulfide bonds <A href="#References">[9]</A>. 
The first cyclotide to be discovered, katala B1, has been used by African women to accelerate childbirth and was obtained by boiling the dry plant leaves, demonstrating its considerable thermostability <A href="#References">[10]</A>. The unique cyclotide framework (Fig 1) is used in the development of drugs against HIV and cancer <A href="#References">[12]</A>. 
</P><P>Even in primates circular proteins have been discovered. Rhesus θ-defensin-1, found in rhesus macaque leukocytes, is a highly potent antibiotic peptide <A href="#References">[13]</A>. In the human genome there is a similar gene that is disrupted by a stop codon. The peptide called retrocyclin was synthesized chemically and it was shown to protect human lymphocytes from HIV-1 infection <I>in vitro</I><A href="#References">[14]</A>. 
</P><P>Importantly, naturally occurring circularization of more complex and larger proteins have not yet been described. Considering the benefit that one can expect from such a modification, we anticipated that developing tools to circularize proteins would be of great help for the scientific and industrial communities.
</P><H3><SPAN class="mw-headline" id="Synthetically_Circularized_Proteins">Synthetically Circularized Proteins</SPAN></H3><DIV class="pull-right col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 2) Approaches to circularize proteins</SPAN><P>Sortase circularization (left) requires an N-termin G and the sorting signal LPXTG. In a transpeptidation reaction, the sortase reversibly circularizes the backbone. Split intein circularization (left) requires the fusion of split inteins to the termini of the protein of interest. They assemble, cleave out themselves and join the exteins. This process is called protein trans-splicing.</P></DIV></DIV></DIV><P>
The properties that enzymes may acquire after circularization have motivated many studies that designed strategies to artificially achieve it <A href="#References">[16][17]</A>. These strategies followed two different directions: the use of chemistry and the use of biological tools. Here, we focused on the latter ones, and especially two of them: inteins and sortase. The best-researched method to circularize large proteins is based on artificially split <A href="/Team:Heidelberg/Project/Background" title="Team:Heidelberg/Project/Background">inteins</A>. Inteins are integrated as extraneous polypeptide sequences into proteins of interest. They do not contribute to the original protein function but perform an autocatalytic splicing reaction after protein translation. By Analogy to intron splicing on RNA level, this post translational modification was named intein splicing. The protein segments were called intein for internal protein sequence, and extein for external protein sequence, with upstream and downstram exteins termed N-exteins and C-exteins, respectively. Inteins excise themselves out of the host protein while reconnecting the remaining N and C exteins, via a new peptide bond. Interestingly, the host  protein maintains its normal structure and function after splicing. <A href="#References">[18]</A>. 
Some inteins like <I>Npu</I> DnaE  are naturally split in two fragments, which can reassemble to to the active intein and thus ligate the exteins <A href="#References">[19]</A>. This process is called <A href="/Team:Heidelberg/Project/Background#A_detailed_description_of_trans-splicing_reaction" title="Team:Heidelberg/Project/Background">protein trans-splicing</A>. If corresponding split inteins are added to both termini of a protein as illustrated in figure 2), the trans-splicing reaction results in a circular backbone. 
Several proteins were circularized using split inteins as a proof of principle, such as dihydrofolate reductase <A href="#References">[4]</A> and GFP <A href="#References">[7]</A>. In these cases, The proteins were rather small and their termini were in close proximity. Both circular dihydrofolate reductase and GFP showed enhanced resistance against proteases and denaturation. The largest published circularized protein is maltose-binding protein (MBP) (395 aa) from E. coli <A href="#References">[20]</A>. However, stability was not tested in this case. 
</P><P>Another interesting approach to circularize proteins is based on sortases <A href="#References">[21]</A>. Naturally, sortases catalyze the linkage of surface proteins to the peptides in peptidoglycan precursors. The sortase cleaves proteins after a specific sorting signal and forms an acyl enzyme intermediate, which is susceptible to a nucleophilic attack by amino groups, resulting in a transpeptidation <A href="#References">[22]</A>. The attacking nucleophile is not necessarily a pentaglycine of the peptidoglycan, and can also be any N-terminal glycine. As shown in figure 2), sortases can be used to circularize the backbone proteins containing a both sorting signal and a glycine at the N-terminus <A href="#References">[21]</A>. As a proof of principle, a GFP variant with an N-terminal polyglycine and LPETG-His6 at the C-terminus was circularized <I>in vitro</I> using sortase A. In this case, faster recovery of fluorescence after denaturation was observed in comparison with a linear control <A href="#References">[21]</A>.
</P><H3><SPAN class="mw-headline" id="Our_Circularization_Project">Our Circularization Project</SPAN></H3><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 3) Circularization using split inteins</SPAN><P>In a single step the protein of interest is cloned into the <A href="http://parts.igem.org/Part:BBa_K1362000">split intein circularization construct</A> (1.). The resulting plasmid (2.) can be used to express a fusion protein (3.). In an autocatalytic <I>in vivo</I> reaction, the circular protein (4.) is formed.</P></DIV></DIV></DIV>

Our interest for protein circularization originated from our ambitious idea of creating a thermostable DNA methyltransferase that could be included in a polymerase chain reaction (PCR) to transfer the methylation state of the original DNA to the product. Knowing the importance of DNA methylation in gene expression regulation, such a PCR assay would be a revolution for epigenetics. In practice, we aimed at circularizing DNA (cytosine-5)-methyltransferase 1 <A href="/Team:Heidelberg/Project/PCR_2.0" title="Team:Heidelberg/Project/PCR 2.0"> (DNMT1)</A>. If DNMT1 and the cofactor S-adenosyl methionine (SAM) could simply be added to a conventional PCR reaction, DNA could be amplified without losing methylation. Consequently, we planned to circularize DNMT1 in order to improve its thermostability as much as possible. 
<P>We decided to standardize the cloning process that is necessary to express circular proteins. Several <A href="/Team:Heidelberg/Toolbox/Circularization_Constructs" title="Team:Heidelberg/Toolbox/Circularization Constructs"> circularization constructs</A> based on the split intein <I>Npu</I> DnaE and Sortase A were designed. By Golden Gate cloning an mRFP selection marker can easily be exchanged with the protein to be circularized. 
Using our <A href="http://parts.igem.org/Part:BBa_K1362000" class="external text" rel="nofollow">split intein circularization construct</A>, a fusion protein of the C-intein, the protein to be circularized and the N-intein is created. Upon expression, the C- and the N-intein assemble and catalyze their own excision and the formation of a peptide bond circularizing the protein. 
Our Sortase A based construct adds TEV and Sortase A recognition sites to the protein to be circularized. The expressed protein has to be purified and treated with TEV protease and Sortase A to be circularized in vitro. We used GFP to test both split intein and sortase circularization. 
</P><DIV class="pull-left col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 4) Circular and linear lambda lysozyme</SPAN><P>The 3D structure of linear lambda lysozyme (purple) is compared with a model predicted by modeller (ref?) of circular lambda lysozyme (yellow) with a rigid linker (green).</P></DIV></DIV></DIV>

However, even the most truncated yet functional form of DNMT1 (872 aa) is much larger than GFP or MBP and its termini are far away from each other. Therefore, it is necessary to put a large linker between the termini. A long flexible linker would probably not be able to significantly enhance thermostability, since the termini would not be locked in one position. 
So we came up with the idea to design <A href="/Team:Heidelberg/Modeling/Linker_Modeling" title="Team:Heidelberg/Modeling/Linker Modeling"> rigid linkers with angles</A> and developed a  <A href="/Team:Heidelberg/Software/Linker_Software" title="Team:Heidelberg/Software/Linker Software"> software</A> to generate linkers for any protein. We circularized <A href="/Team:Heidelberg/Project/Linker_Screening" title="Team:Heidelberg/Project/Linker Screening"> lambda lysozyme</A> in order to test and calibrate our linker model. Circular lambda lysozymes with 9 different linkers were screened for thermostability and we saw a nice trend that confirms the hypothesis that rigid linkers are a way good to circularize a protein. As a proof of principle for an industrial application we also circularized xylanase, an enzyme that is used by the paper industry. Visit our <A href="https://2014.igem.org/Team:Heidelberg/Project/achievements" class="external text" rel="nofollow">achievements page</A> to see what we have reached!
<H3><SPAN class="mw-headline" id="The_Impact_of_Circularization_on_Thermostability">The Impact of Circularization on Thermostability</SPAN></H3><P>Backbone circularization of proteins increases the thermostability of a protein in a similar way to the introduction of disulfide bridges. Either way, a covalent linkage is introduced that decreases the entropy of the unfolded state of the protein <A href="#References">[1]</A>. This principle has already been shown in 1956 in the context of fibrous proteins with covalent linkages <A href="#References">[6]</A>. A short explanation of &quot;the entropy of the unfolded state is decreased&quot;: an unfolded linear polypeptide can move around, since each angle between the peptide bonds can adopt any possible state. If the polypeptide is circular, the state of those angles is much more restricted. Consequently, an unfolded state is less likely to occur, resulting in a higher thermostability. 
</P><P>Another factor that might improve thermostability is the fixation of the termini. They are usually flexible and thus a possible starting point of the denaturation process <A href="#References">[1]</A>. Especially if the termini are close together or a <A href="/Team:Heidelberg/Modeling/Linker_Modeling" title="Team:Heidelberg/Modeling/Linker Modeling"> rigid linker</A> is used to join them, this should cause an additional increase of thermostability.
</P><H3><SPAN class="mw-headline" id="Why_introducing_circularization_to_iGEM.3F">Why introducing circularization to iGEM?</SPAN></H3><P>Many iGEM projects deal with industrially or medically oriented applications, which would greatly benefit from having “super” proteins that can handle better the conditions under which they are requested to work. Therefore we thought that circularization was a perfect new tool for iGEM, being all the while very innovative and original in concept. Because we were fascinated by the power of inteins beyond circularization, we decided to expand our project and to create a <A href="/Team:Heidelberg/Project/Toolbox" title="Team:Heidelberg/Project/Toolbox">toolbox</A> that iGEM teams can use in the future to control their proteins of interest in vivo via post-translational modifications.  These other applications of our toolbox – independent of circularization – are more relevant to cell biology projects, which are also often found in iGEM.
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P>[1]	Vieille, C. &amp; Zeikus, G. J. Hyperthermophilic enzymes: sources, uses, and molecular mechanisms for thermostability. Microbiol. Mol. Biol. Rev. 65, 1–43 (2001).
</P><P>[2]     Gentilucci, L., De Marco, R., and Cerisoli, L., Chemical modifications designed to improve peptide stability: incorporation of non-natural amino acids, pseudo-peptide bonds, and cyclization. Curr Pharm Des 16 (28), 3185 (2010)
</P><P>[3]     Singh, R. K., Tiwari, M. K., Singh, R., and Lee, J. K., From protein engineering to immobilization: promising strategies for the upgrade of industrial enzymes. Int J Mol Sci 14 (1), 1232 (2013)
</P><P>[4]	Scott, C. P., Abel-Santos, E., Wall, M., Wahnon, D. C. &amp; Benkovic, S. J. Production of cyclic peptides and proteins in vivo. Proc. Natl. Acad. Sci. 96, 13638–13643 (1999).
</P><P>[5]	Iwai, H. &amp; Plückthun, A. Circular beta-lactamase: stability enhancement by cyclizing the backbone. FEBS Lett. 459, 166–72 (1999).
</P><P>[6]	Flory, J. &amp; Yol, S. Theory of Elastic Mechanisms in Fibrous Proteins. 715, 5222–5235 (1956)
</P><P>[7]	Iwai, H., Lingel, a &amp; Pluckthun, a. Cyclic green fluorescent protein produced in vivo using an artificially split PI-PfuI intein from Pyrococcus furiosus. J. Biol. Chem. 276, 16548–54 (2001).
</P><P>[8]	Trabi, M. &amp; Craik, D. J. Circular proteins--no end in sight. Trends Biochem. Sci. 27, 132–8 (2002).
</P><P>[9]	Craik, D. J. Chemistry. Seamless proteins tie up their loose ends. Science 311, 1563–4 (2006).
</P><P>[10]	A, P. K. B. et al. Elucidation of the Primary and Three-Dimensional Structure of the Uterotonic. 4147–4158 (1995).
</P><P>[11]     Nguyen, G. K. T. et al. Butelase 1 is an Asx-specific ligase enabling peptide macrocyclization and synthesis. Nat. Chem. Biol. 10, 732–8 (2014).
</P><P>[12]     Henriques, S. T. &amp; Craik, D. J. Cyclotides as templates in drug design. Drug Discov. Today 15, 57–64 (2010).
</P><P>[13]	Tang, Y. A Cyclic Antimicrobial Peptide Produced in Primate Leukocytes by the Ligation of Two Truncated -Defensins. Science (80-. ). 286, 498–502 (1999).
</P><P>[14]     Cole, A. M. et al. Retrocyclin: a primate peptide that protects cells from infection by T- and M-tropic strains of HIV-1. Proc. Natl. Acad. Sci. U. S. A. 99, 1813–8 (2002).
</P><P>[15]     KatalaB1. Cyclotide structure. (2007). at &lt;<A href="http://en.wikipedia.org/wiki/Cyclotide#mediaviewer/File:Cyclotide_structure.jpg" class="external free" rel="nofollow">http://en.wikipedia.org/wiki/Cyclotide#mediaviewer/File:Cyclotide_structure.jpg</A>&gt;</P><P>[16]    Aboye, T. L. and Camarero, J. A., Biological synthesis of circular polypeptides. J Biol Chem 287 (32), 27026 (2012).
</P><P>[17]    Sancheti, H. and Camarero, J. A., &quot;Splicing up&quot; drug discovery. Cell-based expression and screening of genetically-encoded libraries of backbone-cyclized polypeptides. Adv Drug Deliv Rev 61 (11), 908 (2009).
</P><P>[18]     Perler, F. B. et al. Protein splicing elements: inteins and exteins — a definition of terms and recommended nomenclature. Nucleic Acids Res. 22, 1125–1127 (1994).
</P><P>[19]     Zettler, J., Schütz, V. &amp; Mootz, H. D. The naturally split Npu DnaE intein exhibits an extraordinarily high rate in the protein trans-splicing reaction. FEBS Lett. 583, 909–14 (2009).
</P><P>[20]	Evans, T. C. Protein trans-Splicing and Cyclization by a Naturally Split Intein from the dnaE Gene of Synechocystis Species PCC6803. J. Biol. Chem. 275, 9091–9094 (2000).
</P><P>[21]	Antos, J. M. et al. A straight path to circular proteins. J. Biol. Chem. 284, 16028–36 (2009).
</P><P>[22]     Marraffini, L. A., Dedent, A. C., Marraffini, L. A., Dedent, A. C. &amp; Schneewind, O. Sortases and the Art of Anchoring Proteins to the Envelopes of Gram-Positive Bacteria Sortases and the Art of Anchoring Proteins to the Envelopes of Gram-Positive Bacteria. 70, (2006).
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Heidelberg/pages/Circularization">http://2014.igem.org/Team:Heidelberg/pages/Circularization</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/pages/Circularization" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/pages/Circularization" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/pages/Circularization&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/pages/Circularization&amp;oldid=361733" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>