<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_Xylanase"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Project/Xylanase">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project/Xylanase&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Xylanase&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Xylanase&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project/Xylanase" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project/Xylanase</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container" style="padding:0;background-color:white;"><HEADER id="navbar" role="banner" class="navbar navbar-static-top navbar-white"><DIV class="container" style=" background-color:#000000;"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse collapse" style="margin-bottom:10px;"><NAV role="navigation"><UL class="menu nav navbar-nav navbar-right"><LI class="dropdown"><A href="/Team:Heidelberg/Team" class="dropdown-toggle">Team</A></LI><LI><A href="/Team:Heidelberg/Team/Members">Members</A></LI><LI><A href="/Team:Heidelberg/Team/Collaborations">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="/Team:Heidelberg/Project" class="dropdown-toggle">Project</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Background">Background</A></LI><LI><A href="/Team:Heidelberg/Project/Toolbox">Toolbox</A></LI><LI><A href="/Team:Heidelberg/Toolbox/Circularization">Toolbox: Circularization</A></LI><LI><A href="/Team:Heidelberg/Project/PCR_2.0">PCR 2.0</A></LI><LI><A href="/Team:Heidelberg/Project/Xylanase">Xylanase</A></LI><LI><A href="/Team:Heidelberg/Project/Linker_Screening">Linker Screening</A></LI><LI><A href="/Team:Heidelberg/Project/Reconstitution">Fluorescence Assembly</A></LI><LI><A href="/Team:Heidelberg/Project/LOV">Light-Induction</A></LI><LI><A href="/Team:Heidelberg/Project#Achievements">Achievements</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Parts" class="dropdown-toggle">Parts</A></LI><LI><A href="/Team:Heidelberg/Parts/RFC">RFC Heidelberg 2014</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Favorite Parts">Favorite Parts</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Sample Data Page">Sample Data Page</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#allParts">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Backbones">Backbones</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Intein Library">Intein Library</A></LI><LI><A href="/Team:Heidelberg/Parts/Part_Improvement">Part Improvement</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Software" class="dropdown-toggle">Software</A></LI><LI><A href="/Team:Heidelberg/Software/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Software/Linker_Software">Linker Software</A></LI><LI><A href="/Team:Heidelberg/Software/MidnightDoc">MidnightDoc</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Modeling" class="dropdown-toggle">Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Linker_Modeling">Linker Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Enzyme_Modeling">Enzyme Kinetics Modeling</A></LI><LI><A href="/Team:Heidelberg/Toolbox_Guide" class="dropdown-toggle">Toolbox Guide</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Human_Practice" class="dropdown-toggle">Human Practice</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Ethics">Religion &amp; Synthetic Biology</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Education">Education</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Public_Relations">Public Relations</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Experts">Experts</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Notebook" class="dropdown-toggle">Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="/Team:Heidelberg/AwesomeSheet">Database for Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook/Materials">Materials</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Notebook/Safety">Safety</A></LI></NAV></DIV><DIV class="col-lg-12"><DIV class="title-wrapper"><SPAN class="title">XYLANASE</SPAN></DIV></DIV></DIV></HEADER><DIV class="container main"><DIV class="row" style="position:relative;"><DIV class="col-lg-offset-1 col-lg-11 subtitle"><H2><P>– Applying the toolbox</P></H2></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><H2 class="titles"><A href="#Motivation" class="label label-default">Motivation</A><A href="#Experimental_Procedures" class="label label-default">Experimental Procedures</A><A href="#Results" class="label label-default">Results</A></H2></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="well well-lg abstract"><P><B>ABSTRACT</B></P><P>Enzymes used for industrial applications often have to resist high temperatures. To increase heat stability of commercially relevant enzymes, our intein toolbox could be routinely applied to test the effect of circularization. As a proof of principle, we used the intein toolbox to circularize and probably increase heat stability of an important industrial enzyme – Xylanase from <I>B. subtilis</I>. We managed to express linear and circular xylanase but our first assays could not be used to quantify temperature-dependent changes in activity. Overall we could show that the intein toolbox can be used to efficiently circularize industrially-relevant enzymes.</P><P><B>Highlight:</B></P><P>Successful application of the intein toolbox for circularization of xylanase </P></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Motivation"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Motivation</SPAN></A></LI><LI class="toclevel-1"><A href="#Experimental_Procedures"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Experimental Procedures</SPAN></A></LI><LI class="toclevel-2"><A href="#Design_and_cloning_of_constructs"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Design and cloning of constructs</SPAN></A></LI><LI class="toclevel-2"><A href="#Xylanase_Assay"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Xylanase Assay</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Results"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2"><A href="#1.29_Expression_of_linear_and_circular_Xylanase"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">1) Expression of linear and circular Xylanase</SPAN></A></LI><LI class="toclevel-2"><A href="#2.29_Activity_of_linear_and_circular_xylanases_after_heat_shock"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">2) Activity of linear and circular xylanases after heat shock</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Motivation">Motivation</SPAN></H1><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Fig. 1 3D Structure of Xylanase</SPAN><P>Image of the pdb 2DCY file.The ends were highlighted and the distance has been labeled (4.6 Å)</P></DIV></DIV></DIV><P>Industrial enzymes are a fast growing global market which was estimated to reach $6 billion in 2016 (<A href="#References">1,2</A>). An important player in this market is xylanase, an enzyme used in various industrial fields such as textile, paper and energy as well as baking and fruit juice industries. Since many applications of xylanase require high temperatures, engineering of heat-stable xylanases is an active research topic (reviewed in <A href="#References">3</A>). Common approaches to this problem are to screen for xylanase-producing microorganisms (e.g. <A href="#References">4</A>) or to introduce disulfide bonds in order to generate more heat-stable xylanases (<A href="#References">5</A>).
</P><P>Considering that circularization of smaller peptides enhanced their thermostability (<A href="#References">6</A>), we wanted to apply our intein toolbox for circularization of xylanase as an alternative approach to increase its heat stability.
</P><P>From the multitude of described xylanases, we chose the xylanase of Bacillus subtilis because:
</P><P>-It is a promising enzyme (<A href="#References">7</A>) to be used in pulp bleaching
</P><P>-The N- and C-terminus of this xylanase are in close  proximity (4.6 Å , see Figure 1) which facilitates circularization.
</P><H1><SPAN class="mw-headline" id="Experimental_Procedures">Experimental Procedures</SPAN></H1><H2><SPAN class="mw-headline" id="Design_and_cloning_of_constructs">Design and cloning of constructs</SPAN></H2><P>To generate a circular, heat-stable xylanase we chose to follow two strategies and circularize not only the native form but also a xylanase including a histidine tag for purification.   For circularization we used the split NpuDnaE intein, since it is used as the <A href="/Team:Heidelberg/Toolbox/Circularization_Constructs" title="Team:Heidelberg/Toolbox/Circularization Constructs">golden standard for protein splicing</A>.  All cloning steps were carried out using our new RFC[i] standard procedure.
An overview of all constructs including controls for circularization is shown in the following table:
</P><TABLE class="table table-hover"><TBODY><TR><TD><B>Construct Names</B></TD><TD><B>Motivation</B></TD></TR><TR><TD>Linear
</TD><TD>native xylanase
</TD></TR><TR><TD>Linear +His
</TD><TD>control for “Circular with His6-Tag” construct
</TD></TR><TR><TD>Linear+His+ Ext
</TD><TD>control with similar amino acid composition as the circular His6-tag xylanase  (“Circular with His-tag”)
</TD></TR><TR><TD>Circular
</TD><TD>native xylanase with split NpuDnaE split intein
</TD></TR><TR><TD>Circular + His
</TD><TD>native xylanase with split NpuDnaE split intein and His6-tag
</TD></TR><TR><TD>Circular +His+ destr.Ext.
</TD><TD>control with mutation in C-Ext. (Cys-&gt; Gly)
</TD></TR></TBODY></TABLE><P>For the construct to express linear xylanase <A href="http://parts.igem.org/Part:BBa_K1362020" class="external text" rel="nofollow">(BBa_K1362020)</A> the coding sequence of xylanase without start codon and export signal was amplified from <I>B. subtilis</I>  by PCR. Overhangs containing an
EcoRI site, the RBS <A href="http://parts.igem.org/Part:BBa_B0034" class="external text" rel="nofollow">B0034</A>, a start codon (forward) and a SpeI site and a PstI site (reverse) were added by primer overhangs. The PCR product was cut with X and P and ligated into the expression vector <A href="http://parts.igem.org/Part:BBa_K1362093" class="external text" rel="nofollow">pSBX1K30</A>. In order to create the constructs to express linear xylanase with His6-Tag <A href="http://parts.igem.org/Part:BBa_K1362023" class="external text" rel="nofollow">BBa_K1362023</A> and linear xylanase with His6-tag and exteins, primers to linearize <A href="http://parts.igem.org/Part:BBa_K1362020" class="external text" rel="nofollow">BBa_K1362020</A> between the xylanase coding sequence and the stop codon and add BsaI sites at the ends were designed. Inserts coding for the sequence GSHHHHHHSG (His6-Tag) respectively HHHHHHRGKCWE (His6-tag + exteins) with overhangs corresponding to the overhangs caused by BsaI restriction of the vector PCR product were designed and ordered as DNA-oligomers. The inserts were annealed and inserted into the purified PCR product of <A href="http://parts.igem.org/Part:BBa_K1362020" class="external text" rel="nofollow">BBa_K1362020</A> by Golden Gate cloning.
</P><P>The constructs to express circular xylanase <A href="http://parts.igem.org/Part:BBa_K1362022" class="external text" rel="nofollow">(BBa_K1362022)</A>, circular xylanase with His6-tag <A href="http://parts.igem.org/Part:BBa_K1362023" class="external text" rel="nofollow">(BBa_K1362023)</A> and non-circular xylanase with His6-tag (with a Cys-&gt;Gly mutation in the C-extein) are based on the <A href="http://parts.igem.org/Part:BBa_K1362000">NpuDnaE intein RFC[i] circularization construct</A>. For <A href="http://parts.igem.org/Part:BBa_K1362022" class="external text" rel="nofollow">BBa_K1362022</A>, the coding sequence of xylanase without start codon, export signal and stop codon was amplified from <I>B. subtilis</I>  by PCR. Overhangs containing RFC[i] E standard overhang, BsaI sites, the coding sequences of the exteins CWE (forward) and RGK, the RFC[i] B standard overhang, BsaI (reverse) were added by primer overhangs. The PCR product was purified and it was inserted into the <A href="http://parts.igem.org/Part:BBa_K1362000">NpuDnaE intein RFC[i]</A> circularization construct] by Golden Gate assembly. <A href="http://parts.igem.org/Part:BBa_K1362023" class="external text" rel="nofollow">BBa_K1362023</A> and the construct to express non-circular xylanase with His6-tag (with a Cys-&gt;Gly mutation in the C-extein) were cloned in the same way, except for the additional His6-tags in the primer overhangs.
</P><H2><SPAN class="mw-headline" id="Xylanase_Assay">Xylanase Assay</SPAN></H2><P>To test if circularization increases heat stability of <I>B. subtilis</I> xylanase we determined xylanase activity after 30min incubation at different temperatures using the EnzChek® Ultra Xylanase Assay Kit. This kit includes a fluorescently labelled hemicellulose substrate. Xylanase-mediated hydrolysis of the substrate results in the unquenching of the attached fluorescent dyes and thus increased fluorescence. Fluorescence intensity over time was measured using the Tecan Infinite M200 reader and xylanase from Thermomyces lanuginosus was used as an initial positive control. Data were analyzed and visualized using a python script similar to the one for the lysozyme assay.
</P><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><H3><SPAN class="mw-headline" id="1.29_Expression_of_linear_and_circular_Xylanase">1) Expression of linear and circular Xylanase</SPAN></H3><P>To generate a circular, heat-stable xylanase we chose to follow two strategies and circularize not only the native form but also a His6-tagged xylanase.  His-tagged xylanase can be easily purified and thus, xylanase activity can be determined more reliably and without interfering molecules that are otherwise present in the lysate. On the other hand, the His-tag might influence thermostability and xylanase activity. Therefore we also included the unmodified xylanase and measured activity directly in bacterial lysate similar to the lysozyme assay.
</P><P>For circularization we used the split NpuDnaE intein and performed cloning using our new RFC[i] standard procedure.
</P><DIV class="pull-left col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Fig. 2 Coomassie gel of circular and linear xylanase constructs</SPAN><P>BL21 cultures were induced with IPTG, harvested, lysed and analyzed by SDS-PAGE. Compared to the negative control (BL21), prominent bands at 15-20kDa indicate that all constructs except the circular His-tagged xylanase (Circ.His) are efficiently expressed. Circularization is specific and not achieved with destroyed exteins (no band of circular xylanase at ~18kDa for Circ.destr.Ext+His) (Lin.=linear, Circ.=circular, His.=Histidine tag, Ext.=extein, dest.=destroyed, pur=purified with His-trap column).</P></DIV></DIV></DIV><P>Successful cloning and expression of the different linear and circular xylanase variants was verified by a Coomassie stained SDS polyacrylamide gel (see Figure 2). The prominent bands at ~20kDa indicate that linear xylanase with and without His-Tag (Lin., Lin.His., Figure 2) could be efficiently expressed in E.coli BL21. From the circular xylanase constructs the circular variant (Circ.) was produced in satisfying amounts whereas the His-tagged variant was not efficiently expressed (Circ, Circ.His, Figure 2) Nevertheless, the His-tagged circular xylanase could be enriched by purification using a His-Trap column (Circ. (p)+His, Figure 2). 
</P><P>Interestingly, the circular xylanases (Circ, Circ.(p).His) appear to have a lower molecular weight (at ~18kDa) on the gel than their linear counterparts (at ~20kDa) although they include six amino acids of exteins not present in the linear forms. This observation can be explained by the different migration properties of linear and circular molecules during gel electrophoresis. Similar to circular plasmids which migrate faster than their respective linearized versions in agarose gels, circularized proteins such as GFP move faster than their linear counterparts (<A href="#References">8</A>). Since the circularization control construct with the destroyed extein (Circ., destr. extein)+His) does not show a band at the same height as the circular xylanases, circularization seems to be specific. Taken together, the circularization control and migration behavior of our circular constructs strongly indicate that we succeeded in circularizing xylanase. 
</P><P>To proof circularization, we cooperated with Dr. Martina Schnölzer and Dr. Uwe Warnken from the mass spectrometry facility of the German Cancer Research center. In the facility, circularized and linear xylanase were extracted from the gel, digested with chymotrypsin and analyzed by ESI mass spectrometry. Unfortunately, we do not have the final results yet but are confident to present them at the jamboree.
</P><H3><SPAN class="mw-headline" id="2.29_Activity_of_linear_and_circular_xylanases_after_heat_shock">2) Activity of linear and circular xylanases after heat shock</SPAN></H3><P>After the successful expression of linear and circular xylanase we analyzed how circularization influences resistance towards heat. We measured the activity of purified linear and circular xylanase after 30min heat exposure.
</P><P>To be able to compare the activity of linear and circular forms, the enzyme amounts in the different samples were estimated by absorption measurement at 280 nm and verified by  SDS PAGE (see Figure 3). By determining intensities of the Xylanase bands with ImageJ we estimated a ratio of 1:8 of linear and circular form, respectively.
</P><DIV class="pull-left col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Fig. 3 Coomassie -stained gel to determine ratio of purified linear to circular xylanase</SPAN><P>Different amounts of linear and circular Xylanase with His-Tag were added assuming a ratio specified on top relative to the volume for 4 µg of the linear Xylanase.</P></DIV></DIV></DIV><P>Our initial results show an inconsistent behavior of purified linear and circular xylanase activity after heat shock at different temperatures. Since we did not observe similar activity at 37°C for circular and linear xylanase, the activity after heat shock cannot be compared. To exclude potential effects of the His-tag, we repeated the activity assay with the unpurified, native linear and circular xylanase. Both, unpurified linear and circular xylanase lose completely their activity over 50 °C which has been previously reported (<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC239434/" class="external autonumber" rel="nofollow">[1]</A>). Similar to the His-tag containing xylanases, temperature-dependent activity of the unpurified linear and circular xylanase cannot be compared due to different activity at 37°C.
</P><DIV class="row thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Fig. 4 Activity of purified linear and circular xylanase after incubation at different temperatures</SPAN><P>Linear and circular xylanase were incubated in assay buffer for 30min at the indicated temperatures and added to the hemicellulose substrate. Fluorescence intensities (relative fluorescence units) were measured over 45min</P></DIV></DIV><DIV class="row thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Fig. 5 Activity of unpurified linear and circular xylanase after incubation at different temperatures</SPAN><P>Linear and circular xylanase were incubated in assay buffer for 30min at the indicated temperatures and added to the hemicellulose substrate. Fluorescence intensities (relative fluorescence units) were measured over 160min</P></DIV></DIV><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><OL><LI> Sharma, M. and A. Kumar, Xylanases : An Overview. 2013. 3: p. 1-28.
</LI><LI><A href="http://www.bccresearch.com/market-research/biotechnology/enzymes-industrial-applications-markets-bio030g.html" class="external free" rel="nofollow">http://www.bccresearch.com/market-research/biotechnology/enzymes-industrial-applications-markets-bio030g.html</A> (2012) bcc Research
</LI><LI> Beg, Q. K., Kapoor, M., Mahajan, L. &amp; Hoondal, G. S. Microbial xylanases and their industrial applications: a review. Appl. Microbiol. Biotechnol. 56, 326–338 (2001).
</LI><LI> Guo, G. et al. Purification and characterization of a xylanase from Bacillus subtilis isolated from the degumming line. J. Basic Microbiol. 52, 419–28 (2012).
</LI><LI> Wakarchuk, W. W. et al. Thermostabilization of the Bacillus circulans xylanase by the introduction of disulfide bonds. Protein Eng. 7, 1379–86 (1994).
</LI><LI> Tam, J. P. &amp; Wong, C. T. T. Chemical synthesis of circular proteins. J. Biol. Chem. 287, 27020–5 (2012).
</LI><LI> Bajpai, P. Application of enzymes in the pulp and paper industry. Biotechnol. Prog. 15, 147–57 (1999).
</LI><LI> Iwai, H., Lingel, a &amp; Pluckthun, a. Cyclic green fluorescent protein produced in vivo using an artificially split PI-PfuI intein from Pyrococcus furiosus. J. Biol. Chem. 276, 16548–54 (2001).
</LI></OL></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Heidelberg/Project/Xylanase">http://2014.igem.org/Team:Heidelberg/Project/Xylanase</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project/Xylanase" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project/Xylanase" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Xylanase&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Xylanase&amp;oldid=383494" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>