<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_Background"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Project/Background">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project/Background&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Background&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Background&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project/Background" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project/Background</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container" style="padding:0;background-color:white;"><HEADER id="navbar" role="banner" class="navbar navbar-static-top navbar-white"><DIV class="container" style=" background-color:black;"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse collapse" style="margin-bottom:10px;"><NAV role="navigation"><UL class="menu nav navbar-nav navbar-right"><LI class="dropdown"><A href="/Team:Heidelberg/Team" class="dropdown-toggle">Team</A></LI><LI><A href="/Team:Heidelberg/Team/Members">Members</A></LI><LI><A href="/Team:Heidelberg/Team/Collaborations">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="/Team:Heidelberg/Project" class="dropdown-toggle">Project</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Background">Background</A></LI><LI><A href="/Team:Heidelberg/Project/Toolbox">Toolbox</A></LI><LI><A href="/Team:Heidelberg/Toolbox/Circularization">Toolbox: Circularization</A></LI><LI><A href="/Team:Heidelberg/Project/PCR_2.0">PCR 2.0</A></LI><LI><A href="/Team:Heidelberg/Project/Xylanase">Xylanase</A></LI><LI><A href="/Team:Heidelberg/Project/Linker_Screening">Linker Screening</A></LI><LI><A href="/Team:Heidelberg/Project/Reconstitution">Fluorescence Assembly</A></LI><LI><A href="/Team:Heidelberg/Project/LOV">Light-Induction</A></LI><LI><A href="/Team:Heidelberg/Project#Achievements">Achievements</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Parts" class="dropdown-toggle">Parts</A></LI><LI><A href="/Team:Heidelberg/Parts/RFC">RFC Heidelberg 2014</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Favorite Parts">Favorite Parts</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Sample Data Page">Sample Data Page</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#allParts">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Backbones">Backbones</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Intein Library">Intein Library</A></LI><LI><A href="/Team:Heidelberg/Parts/Part_Improvement">Part Improvement</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Software" class="dropdown-toggle">Software</A></LI><LI><A href="/Team:Heidelberg/Software/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Software/Linker_Software">Linker Software</A></LI><LI><A href="/Team:Heidelberg/Software/MidnightDoc">MidnightDoc</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Modeling" class="dropdown-toggle">Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Linker_Modeling">Linker Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Enzyme_Modeling">Enzyme Kinetics Modeling</A></LI><LI><A href="/Team:Heidelberg/Toolbox_Guide" class="dropdown-toggle">Toolbox Guide</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Human_Practice" class="dropdown-toggle">Human Practice</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Ethics">Religion &amp; Synthetic Biology</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Education">Education</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Public_Relations">Public Relations</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Experts">Experts</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Notebook" class="dropdown-toggle">Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="/Team:Heidelberg/AwesomeSheet">Database for Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook/Materials">Materials</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Notebook/Safety">Safety</A></LI></NAV></DIV><DIV class="col-lg-12"><DIV class="title-wrapper"><SPAN class="title">BACKGROUND</SPAN></DIV></DIV></DIV></HEADER><DIV class="container main"><DIV class="row" style="position:relative;"><DIV class="col-lg-offset-1 col-lg-11 subtitle"><H2><P>– Explore the world of inteins</P></H2></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><P>
In the variety of proteins that exist, inteins are clearly among the most mysterious.
They behave in such a peculiarly way that scientists are still puzzle about their initial role in the host organisms and are even more astonished when discovering their endless seeming abilities.
</P><P>Inteins are integrated as extraneous polypeptide sequences into ordinary proteins. They do not contribute to the original protein function but perform an autocatalytic splicing reaction after protein translation. Analog to intron splicing on RNA level, this posttranslational modification was named protein splicing. Consequentially the protein segments were called inteins, derived from “internal protein sequence”, and the flanking protein chains exteins, “external protein sequences”.  Inteins excise themselves out of the host protein while reconnecting the remaining N and C exteins via a new peptide bond. Despite this immense invasion the original protein regains its normal structure and function after splicing <A href="#References">[1]</A>. 
</P><DIV><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 1) Trans-splicing mechanism reaction by split inteins.</SPAN><P>The N-Intein is fused to the C’ terminal end of the N-extein. Complementary, C-Intein is located at the N’ end of the C-extein. After assembly of the two intein fragments, a splicing reaction takes place, where the intein removes itself from the precursor protein and simultaneously ligates the exteins via a peptide bond.</P></DIV></DIV></DIV></DIV><DIV class="pull-left col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Trans-splicing mechanism reaction by split inteins.</SPAN><P>Animation of the split intein splicing reaction</P></DIV></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#History"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">History</SPAN></A></LI><LI class="toclevel-1"><A href="#Evolution:_Inteins_as_parasitic_genes"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Evolution: Inteins as parasitic genes</SPAN></A></LI><LI class="toclevel-1"><A href="#Structure_and_Classification"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Structure and Classification</SPAN></A></LI><LI class="toclevel-1"><A href="#A_detailed_description_of_the_trans-splicing_reaction"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">A detailed description of the trans-splicing reaction</SPAN></A></LI><LI class="toclevel-1"><A href="#Use_of_inteins_in_molecular_biology_and_biotechnology"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Use of inteins in molecular biology and biotechnology</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="History">History</SPAN></H2><P>Inteins were discovered in 1990 when dissimilarities between the mature protein sequence of the yeast vacuolar ATPase (Vma1) and its corresponding mRNA were investigated. Surprisingly the mature protein had a lower molecular weight than expected from the encoding sequence, indicating the loss of one part of the protein after translation. 
Indeed, a region of 454 amino acids was found to be translated and subsequently removed from the Vma1 protein <A href="#References">[2]</A><A href="#References">[3]</A>. Since then, over 600 different inteins have been reported in all three domains of life as well as in viruses <A href="#References">[4]</A>. 
</P><P>Dependent on the organism they belong to, the intein’s name consists out of the genus and species abbreviation followed by the host gene <A href="#References">[5]</A>. For example, the &quot;golden standard&quot; split-intein, NpuDnaE has derived from the DNA polymerase III (DnaE) in <I>Nostoc punctiforme</I> PCC73102 (<I>Npu</I>).
</P><H2><SPAN class="mw-headline" id="Evolution:_Inteins_as_parasitic_genes">Evolution: Inteins as parasitic genes</SPAN></H2><P>Despite many years of research the initial role of inteins in their host organism remains still unclear.
Many natural inteins contain a homing endonuclease (HEN) domain, an enzyme that cleaves DNA and introduces new sequences at homing sites via homologous recombination or reverse transcription. This characteristic led to the perception of the intein as selfish DNA sequence - a gene generating copies of itself without an advantage for the host organism <A href="#References">[6]</A>. This allows horizontal gene transfer of inteins. <A href="#References">[7]</A>. Data showing that 27% of the intein's host proteins are related to DNA metabolism and involved in DNA replication or repair is further supporting the &quot;selfish gene&quot; hypothesis. This ensures expression of the inteins and the HEN domain during DNA replication, where they can take advantage of the DNA replication and repair system for introducing changes in the DNA <A href="#References">[8]</A>.
</P><P>However recent work on conditional protein splicing has shown sensitivity of some inteins to redox state, temperature and small molecules (reviewed in <A href="#References">[9]</A>, <A href="#References">[10]</A>). 
This might be a sign for inteins having a role in post-translational protein regulation upon environmental signals. Inserted close to the catalytic core of an protein the intein deactivates the protein function until the splicing reaction has taken place. Evolutionary said, selfish inteins might have adapted due to positive selection pressure to provide a beneficial mechanism for the host cell, thereby becoming selfless <A href="#References">[8]</A>.
</P><H2><SPAN class="mw-headline" id="Structure_and_Classification">Structure and Classification</SPAN></H2><P>The structure of inteins contains several conserved motifs. The splicing domains are located at the N and C terminal ends. Before splicing takes place the intein rearranges itself from its initial linear structure into to a horseshoe like structure where the termini are brought in close proximity making up the catalytic core <A href="#References">[14]</A>. 
Inteins are divided into three groups: bifunctional (large) inteins, mini inteins and split inteins.
</P><P>Large inteins carry both a splicing domain and an endonuclease (HEN) domain whereas mini inteins lack the HEN domain <A href="#References">[11]</A>. 
The most promising inteins for biotechnology are split inteins, which are basically mini-inteins, just divided in two fragments and expressed separately connected different proteins. After translation, they assemble with high affinity to become catalytic active and perform a splicing reaction. Split enzymes occure naturally <A href="#References">[12]</A> but can also be engineered artificially <A href="#References">[13]</A>. 
</P><P>In our project we focused on split inteins, as they present a powerful tool to insert posttranslational modifications, offering a plethora of applications in biotechnology. We have characterized the most promising inteins in our <A href="/Team:Heidelberg/Parts#Intein_Library" title="Team:Heidelberg/Parts">Intein Library</A>.
</P><H2><SPAN class="mw-headline" id="A_detailed_description_of_the_trans-splicing_reaction">A detailed description of the trans-splicing reaction</SPAN></H2><P>In contrast to the mini and large inteins that mediate cis-splicing reactions, split inteins are responsible for trans-splicing and fusion of protein parts. The trans-splicing reaction can be divided into the following steps:
</P><OL><LI> N-Intein and C-Intein first assemble together to form a dimer like structure with a newly formed catalytic core next to the exteins.
</LI><LI> The tertiary structure of the intein, once correctly formed, facilitates an <I>N→O/S</I> acyl rearrangement at its N-terminal serine or cysteine residue. The result is an ester or thioester bond, respectively between the side-chain and the peptide backbone of the N-extein.
</LI><LI> The two exteins are then linked by trans(thio)esterification involving the N-terminal serine or cysteine residue of the C-extein. The C-terminus of the N-extein is now covalently bound to the N-terminal side-chain of the C-extein, while its backbone still retains its normal peptide bond to the intein.
</LI><LI> The C-terminal asparagine of the intein undergoes self-cyclisation to form a succinimidyl moiety. The peptide bond between the intein and the extein is thereby broken, resolving the branched intermediate.
</LI><LI> In a final reaction, an <I>O/S→N</I> acyl shift results in the two exteins now being linked by an amide bond, indistinguishable from a ribosome-assembled fusion protein.
</LI></OL><P>(as reviewed in <A href="#References">[15]</A>) 
</P><DIV class="pull-right col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 2) Trans-splicing mechanism reaction by split inteins.</SPAN></DIV></DIV></DIV><P>For intein efficiency not only the structure of the intein domain itself is important but also the nature of the flanking extein residues, as they are heavily involved in the splicing reaction. It has been shown, that the first residue has to be a nucleophilic amino acid, preferable Cysteine, Serine or Threonine. Effectiveness of the intein can be mold by changing the these amino acids <A href="#References">[17]</A>.
</P><H2><SPAN class="mw-headline" id="Use_of_inteins_in_molecular_biology_and_biotechnology">Use of inteins in molecular biology and biotechnology</SPAN></H2><P>Due to their distinct characteristics intein are a powerful tool for molecular biology and biotechnology. Performing an autocatalytic reaction, the inteins are neither dependent on their host protein nor on any other additional substrate. This makes it them broadly applicable for in vivo and in vitro applications. <A href="#References">[16]</A>.
</P><P>Inteins have been used for protein purification, achieving a higher yield of protein due to more specific targeting. The production of semi-synthetic proteins and the attachment of synthetic groups is of huge interest for recombinant proteins. <A href="#References">[16]</A>. Split Intein-mediated Circular Ligation Of Peptides a ProteinS (SICLOPPS) is a method to produce circular peptides of eight amino acids length, which could be potent therapeutic drugs <A href="#References">[18]</A>. Inteins, fused to split fluorescent proteins as reporter and to the protein of investigation have been used to detect protein-protein interactions in vivo (reviewed in <A href="#References">[10]</A>).
Nevertheless most of this applications are performed in vitro and do not exploit the full potential of inteins as regulatory element for post-translational modification. The iGEM Team Heidelberg employs this excellent mechanism of protein splicing to specifically change whole amino-acid sequences and thereby regulate proteins via (dis-) assembly, protein cleavage or circularization of enzymes in vivo. All this and much more was embedded into our versatile intein toolbox. With this universal toolkit we provide a foundational advance for protein control - introducing the full potential of post-translational modification and therefore a new dimension of genetic engineering to Synthetic Biology. There is much more to <A href="/Team:Heidelberg/Project" title="Team:Heidelberg/Project">explore</A>!
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>[1] Perler FB, Davis EO, Dean GE, Gimble FS, Jack WE, et al. (1994) Protein splicing elements: inteins and exteins–a definition of terms and recommended nomenclature. Nucleic Acids Res 22: 1125–1127
</P><P>[2] Kane P.M., et al. Protein Splicing Converts the Yeast TEPI Gene Product to the 69-kD Subunit of the Vacuolar. 13253, (1990)
</P><P>[3] Hirata, R. et al. Molecular structure of a gene, VMA1, encoding the catalytic subunit of H(+)-translocating adenosine triphosphatase from vacuolar membranes of Saccharomyces cerevisiae. J. Biol. Chem. 265 , 6726–6733 (1990)
</P><P>[4] Perler, F. B. InBase : the Intein Database. 30, 383–384 (2002)
</P><P>[5] Perler, F. B. et al. Protein splicing elements : inteins and exteins — a definition of terms and recommended nomenclature. 22, 1125–1127 (1994)
</P><P>[6] Barzel, A., Naor, A., Privman, E., Kupiec, M. &amp; Gophna, U. Homing endonucleases residing within inteins: evolutionary puzzles awaiting genetic solutions. Biochem. Soc. Trans. 39, 169–73 (2011)
</P><P>[7] Pietrokovski, S. Intein spread and extinction in evolution. Trends Genet. 17, 465-472 (2001)
</P><P>[8] Novikova, O., Topilina, N. &amp; Belfort, M. Enigmatic distribution, evolution and function of inteins. J. Biol. Chem. (2014)
</P><P>[9] Shah, N. H., and Muir, T. W. Inteins: nature's gift to protein chemists. Chem. Sci. 5, 446- 461 (2014) 
</P><P>[10] Topilina, N. I. &amp; Mills, K. V. Recent advances in in vivo applications of intein-mediated protein splicing. Mob. DNA 5, 5 (2014)
</P><P>[11] Eryilmaz, E., Shah, N., Muir, T. &amp; Cowburn, D. Structural and Dynamical Features of Inteins and Implications on Protein Splicing. J. Biol. Chem. (2014)
</P><P>[12] Carvajal-Vallejos, P., Pallissé, R., Mootz, H. D. &amp; Schmidt, S. R. Unprecedented rates and efficiencies revealed for new natural split inteins from metagenomic sources. J. Biol. Chem. 287, 28686–96 (2012) 
</P><P>[13] Lin, Y. et al. Protein trans-splicing of multiple atypical split inteins engineered from natural inteins. PLoS One 8, e59516 (2013). 
</P><P>[14] Eryilmaz, E., Shah, N., Muir, T. &amp; Cowburn, D. Structural and Dynamical Features of Inteins and Implications on Protein Splicing. J. Biol. Chem. (2014)
</P><P>[15] Mills, K. V, Johnson, M. a &amp; Perler, F. B. Protein Splicing: How Inteins Escape from Precursor Proteins. J. Biol. Chem. (2014)
</P><P>[16] Mootz, H. D. Split inteins as versatile tools for protein semisynthesis. Chembiochem 10, 2579–89 (2009).
</P><P>[17] Amitai, G., Callahan, B. P., Stanger, M. J., Belfort, G. &amp; Belfort, M. Modulation of intein activity by its neighboring extein substrates. Proc. Natl. Acad. Sci. U. S. A. 106, 11005–10 (2009).
</P><P>[18] Tavassoli, A. &amp; Benkovic, S. J. Split-intein mediated circular ligation used in the synthesis of cyclic peptide libraries in E. coli. Nat. Protoc. 2, 1126–1133 (2007).
</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Heidelberg/Project/Background">http://2014.igem.org/Team:Heidelberg/Project/Background</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project/Background" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project/Background" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Background&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Background&amp;oldid=361641" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>