"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/testing/Circularization
From 2014.igem.org
Toggle navigation
Team
Members
Collaborations
Attributions
Sponsoring
Project
Overview
Background
Toolbox
Toolbox: Circularization
PCR 2.0
Xylanase
Linker Screening
Fluorescence Assembly
Light-Induction
Achievements
Parts
RFC Heidelberg 2014
Favorite Parts
Sample Data Page
Biobricks
Backbones
Intein Library
Part Improvement
Software
iGEM@home
Linker Software
MidnightDoc
Modeling
Linker Modeling
Enzyme Kinetics Modeling
Toolbox Guide
Human Practice
iGEM@home
Religion & Synthetic Biology
Education
Public Relations
Experts
Notebook
Notebook
Database for Notebook
Materials
Methods
Safety
CIRCULARIZATION
– This is a test subtitle
{{{titles}}}
ABSTRACT
Promoters are the key regulators of gene expression. Possessing promoters which are active under a desired condition, at a desired strength and in a specified tissue is of great value for Plant Biotechnology, Gene Therapy and fundamental research in Bioscience. Therefore, it has become a desire to synthetically construct promoters responsive to a variety of pathways. We explore two ways to the synthesis of promoters: On one hand, we have developed a bioinformatical model and database (HEARTBEAT) describing the structure of promoters responsive to user-defined inputs. On the other hand, we have developed a biochemical method for the synthesis of randomized promoter libraries. Using this method, we have created a library of constitutive promoters of varying strength. Also, we have created libraries of promoters putatively responsive to a variety of pathways. We have screened these libraries for functional, pathway responsive promoters and present a detailed characterization of a NF-κB responsive promoter of our making. We finally discuss ways to combine randomized biochemical synthesis and bioinformatical modeling to propose a method towards the generation of promoters of complex regulation (i.e. by multiple pathways).
Contents
1 Introduction
2 Results
2.1 RA-PCR, a method for the generation of randomized promoter libraries
2.2 Generation of a library of constitutive promoters
2.3 References
Introduction
http://2014.igem.org/wiki/images/c/c9/Heidelberg_Anna.jpg http://2014.igem.org/wiki/images/c/c9/Heidelberg_Anna.jpg
<ref name="Bezeichner">Blablablub</ref> Controlling gene expression in a condition and tissue specific manner is a desirable tool for a wide variety of applications ranging from fundamental bioscience research to gene therapy. Having this idea in mind, it is inevitable to consider the starting point of gene expression: the promoter. Most efforts of obtaining any promoter focus on cloning them from nature. In eukaryotes, this approach leads to several problems. The one major issue is the complexity of eukaryotic systems resulting in cross-talk between pathways. In particular, promoters tend to respond to several transcription factors and are thus activated by more than one pathway. That way, one cannot determine which signal originally induced the promoter. Alongside this specificity, induction under a desired combination of conditions and at defined strengths would be valuable implementations.
Therefore, efforts have emerged to synthetically construct promoters. Two concepts of synthetic promoters in mammalian cells co-exist independently from each other. One is the concept of "genetic switches" (see [2] for a recent review) - promoters which can specifically be induced by a stimulus mammalian cells and are usually insensitive to, e.g. tetracycline [3] . Much fewer efforts have been put into developing promoters sensitive to endogenous signals (referred to as "synthetic promoters" in the rest of this article). Such promoters are of very high value for a broad variety of applications. Three examples should demonstrate this: First, in virotherapy for cancer and other diseases, it has become a desire to express toxic genes only in affected cells (reviewed in [4] ). For example, breast cancer cells are characterized by high levels of estrogen receptor. Constructing a promoter which is active only at high estrogen receptor levels (preferable only in cells which are irradiated, as ER can be very active in other tissues of the female reproductive tract [17] ) might therefore help developing novel breast cancer therapies. Second, biologists studying pathway interactions are in need for transcriptional assays, that being promoters which are specifically activated by a single transcription factor. Third, the concept can be transferred to plants, where synthetic promoters can be very valuable, as plant biotechnology is always in need for novel tissue- or development-specific promoters.
Figure 2) Ein anderer Titel
Ganz viel Beschreibung, ganz viel Beschreibung, ganz viel Beschreibung, ganz viel Beschreibung... etc
Three approaches exist to construct synthetic promoters responsive to endogenous factors. First, the structure of promoters is modeled by generating large data sets describing the relative spacing and coincidence of transcription factors (reviewed in [5] ). To our knowledge, such predictions have not been tested in vivo. Second, promoters are generated by randomly or repeatedly cloning response elements upstream of a core promoter. In our understanding, repeated cloning of response elements works well [6] and is frequently applied, but no suggestions exist on how to apply this strategy to the generation of more complexly regulated promoters. The random creation of promoters works well to generate constitutive promoters [7] and was even applied to broadly identify activating elements [8] , but no promoters of specific regulation have been described for this approach. A third approach is the randomization of spacer elements between transcription factor binding sites, which is applied to generate libraries of promoters of varying strength [9] , [10] .
In order to be able to design synthetic promoters, an understanding of natural promoters is required. Mammalian promoters can be subdivided into several "domains". The core promoter is the binding site of the basal transcription machinery, i.e. RNA polymerase and associated factors. Core promoters differ in composition, but are more or less similar for most genes (reviewed in [11] ). The main regulatory domain is the proximal promoter, which is where regulatory elements bind. It can be very large (4kb), meaning that some transcription factors regulate transcription despite being very far away from the RNA polymerase. This is mainly possible because of the three-dimensional structure the DNA adopts. In addition to this, there are even more distal elements that are referred to as "enhancers" and "silencers". A further challenge is that some transcription factors are not able to initiate transcription on their own, but rather they require other transcription factors for their activity [5] .
[TOP]
Results
RA-PCR, a method for the generation of randomized promoter libraries
File:HD09 rapcr.jpg
Figure 1: The method of RA-PCR. The combination of oligos added to the PCR reaction determines the specificity and strengths of the synthesized promoters. If knowing the response elements for the transcription factors of a pathway, any promoter can be assembled by this method.
We have developed a standard method (termed "Random Assembly PCR / RA-PCR") for the construction of randomized promoter libraries. We modified Assembly PCR [12] to create promoters with randomized spacing and frequency of Transcription Factor Binding Sites. It relies on using different oligos containing a transcription factor binding site (or random DNA) plus two annealing sequences (see Fig. 1 for a comprehensive explanation of the method).  We use two sets of oligos, one for the top strand, one for the bottom strand. The oligos for each strand have the same annealing sequences (which are complementary to the annealing sequences of the other strand). If these oligos are pooled, they will randomly anneal to each other, thus generating randomized repeats of the transcription factor binding sites of interest at varying spacing. In order to be able to clone the construct, we also add two stop oligos (termed stop 5' and top 3') which contain only one annealing sequence, plus a cutsite (SpeI 5', HindIII 3'). Double-stranded DNA is created by running a seven-cycle PCR, and amplified by a  25-cycle PCR. Then, the resulting (proximal) promoter is cloned 5' of a core promoter (we used the core promoter of JeT [10] ) by inserting it into pSMB_MEASURE ( Part:BBa_K203100 ), the promoter measurement plasmid we developed (from there, it can be excised like any standard biological part in a submission plasmid). Thus, a mixture of different promoters in the same plasmid backbone is generated. These can then be transformed into bacteria. Each colony represents a single putative promoter, which can be transfected into mammalian cell. 24 hours after transfection cells are exposed to the conditions of interest and control conditions, serving as a reference. Promoters active under the desired conditions, but not under control conditions, are selected for further characterization.
Please see a detailed protocol for RA-PCR below .
[TOP]
Generation of a library of constitutive promoters
Most efforts of obtaining such promoters focus on cloning them from nature. This approach is, in eukaryotes, flawed for three reasons: First, promoters in eukaryotic cells are very complexly regulated by a wide variety of transcription factors, and thus, pathways [1] . One is the concept of "genetic switches" (see [2] for a recent review)
References
[1] Alberts, Bruce. "3 Blabla." Molecular Biology of the Cell. 4th ed. New York: Garland Science, 2002. Print.
[2] Fussenegger, M. & Weber, W. Engineering of Synthetic Mammalian Gene Networks. Chemistry and Biology 16: 287-297 (2009).
Retrieved from " http://2014.igem.org/Team:Heidelberg/testing/Circularization "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
