<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_pages_Linker_Screening"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/pages/Linker_Screening">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/pages/Linker_Screening&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/pages/Linker_Screening&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/pages/Linker_Screening&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/pages/Linker_Screening" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/pages/Linker Screening</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P>
We developed a modeling approach to design linkers that can circularize proteins and that  may confer some rigidity in particular after heat shock. The general concept of the approach can be found in the <A href="https://2014.igem.org/Team:Heidelberg/Software" class="external text" rel="nofollow">Software</A>, and the detailed theoretical development can be found in the software as well as in the <A href="https://2014.igem.org/Team:Heidelberg/Modeling" class="external text" rel="nofollow">Modeling</A> sections. 
The experimental work presented in this section has two main goals. The first one was testing the validity of the idea: Can rigid linkers with angles indeed provide heat stability to a protein, and can it do this better than flexible linkers? And the second one was to measure this stabilization depending on the properties of the linkers to calibrate the <A href="https://2014.igem.org/Team:Heidelberg/Software/Linker_Software" class="external text" rel="nofollow">Linker Software</A>. As already seen in the software part, these properties are the total length of the linker, the angles between consecutive alpha helices, the regions that the linker passes and the distance from the proteins surface. As  we had no a priori knowledge on the contribution of those different properties for the heat stability of protein, we performed an extensive linker screening on the lambda phage lysozyme. The tested linkers were predicted to belong to different groups by the software: Very good, good, average, bad and too short. By measuring their activity after heatshock, we could transform this qualitative classification into a quantitative one.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Main_results"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Main results</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Experimental_procedure"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Experimental procedure</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Cloning_of_the_expression_constructs"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Cloning of the expression constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Expression_and_detection_of_circular_lysozyme_variants"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Expression and detection of circular lysozyme variants</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Development_of_assay_to_measure_lysozyme_activity"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Development of assay to measure lysozyme activity</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#Materials_and_Methods"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Results"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Discussion"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#The_long_way_to_setup_the_assay"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">The long way to setup the assay</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#14th_of_August"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">14th of August</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#17th_of_August"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">17th of August</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#19th_of_August"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">19th of August</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#20th_of_August"><SPAN class="tocnumber">7.4</SPAN><SPAN class="toctext">20th of August</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#21st_of_August"><SPAN class="tocnumber">7.5</SPAN><SPAN class="toctext">21st of August</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#22nd_of_August"><SPAN class="tocnumber">7.6</SPAN><SPAN class="toctext">22nd of August</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#26th_of_August"><SPAN class="tocnumber">7.7</SPAN><SPAN class="toctext">26th of August</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#30th_of_August"><SPAN class="tocnumber">7.8</SPAN><SPAN class="toctext">30th of August</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#5th_of_September"><SPAN class="tocnumber">7.9</SPAN><SPAN class="toctext">5th of September</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#9th_of_September"><SPAN class="tocnumber">7.10</SPAN><SPAN class="toctext">9th of September</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#10th_of_September_.281.29"><SPAN class="tocnumber">7.11</SPAN><SPAN class="toctext">10th of September (1)</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#10th_of_September_.282.29"><SPAN class="tocnumber">7.12</SPAN><SPAN class="toctext">10th of September (2)</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#11th_of_September_.281.29"><SPAN class="tocnumber">7.13</SPAN><SPAN class="toctext">11th of September (1)</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#11th_of_September_.282.29"><SPAN class="tocnumber">7.14</SPAN><SPAN class="toctext">11th of September (2)</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#12th_of_September"><SPAN class="tocnumber">7.15</SPAN><SPAN class="toctext">12th of September</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#14th_of_September"><SPAN class="tocnumber">7.16</SPAN><SPAN class="toctext">14th of September</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#15th_of_September"><SPAN class="tocnumber">7.17</SPAN><SPAN class="toctext">15th of September</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#19th_of_September"><SPAN class="tocnumber">7.18</SPAN><SPAN class="toctext">19th of September</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#28th_of_September"><SPAN class="tocnumber">7.19</SPAN><SPAN class="toctext">28th of September</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#29th_of_September"><SPAN class="tocnumber">7.20</SPAN><SPAN class="toctext">29th of September</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#30th_of_September"><SPAN class="tocnumber">7.21</SPAN><SPAN class="toctext">30th of September</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#1st_of_October"><SPAN class="tocnumber">7.22</SPAN><SPAN class="toctext">1st of October</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#4th_of_October"><SPAN class="tocnumber">7.23</SPAN><SPAN class="toctext">4th of October</SPAN></A></LI><LI class="toclevel-1 tocsection-34"><A href="#References"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H1><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">figure 1) Lambda Lysozyme</SPAN><P>Secondary structure of lambda lysozyme. The distance between the two ends is 27.3 Ångström.</P></DIV></DIV></DIV>

Designing linkers for circularization is a non trivial task: Circularization is  a narrow path between gaining heat-stability and loosing function due to deformation. 
The choice of the protein to perform the linker screen was constrained by different requirements. First, it needed to be easily and fast expressed in <I>E.coli</I>. We needed to be able to measure its functionality without purification and in an easy, fast, cheap and reliable way. It needed to be known for having a certain stability at high temperature with, at the same time, a loss of functionality so that the heat stabilization could be tested. Finally, the major constrain for the choice was the structure of the protein.
<P>First, we needed a crystal structure of the complete protein at high resolution. Second, the ends should be separated by a distance of 15 to 30 Ångström, so that the rigid linkers containing alpha helices would be relevant. On the other hand, the linker should not pass over the active site of the protein. Furthermore it should be easy to obtain. 
</P><P>Lysozymes are well characterized enzymes that are able to digest the peptidoglycans that form the bacterial wall. This process is performed by many different species for different applications, including antibacterial defence by plants and animals <A href="#References">[1]</A> or bacterial penetration by viruses <A href="#References">[2]</A>. On top, lysozyme is applied in different fields of biotechnology and medicine. It is notably one of the most  important proteins in food preservation and is produced in the 100  tons scale a year. 
We anticipated that the lysozyme of the bacteriophage lambda could reasonably fulfill the requirements for our linker screen. 
</P><P>Its crystal structure is known <A href="#References">[2]</A> and the ends are 27.3 Angströms apart (figure.1). It is easy to obtain: one can clone it from the bacteriophage lambda genome, and express it in <I>E.coli</I>. The fact that the overexpression of the enzyme eventually lyse the cells was not considered as a problem as long as enough enzyme would  be produced. Finally substrates to measure the enzymatic activity are commercially available.
</P><H1><SPAN class="mw-headline" id="Main_results">Main results</SPAN></H1><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 5) Lysozyme assay</SPAN><P>Lysozyme assay with linker sho2 with a heat shock span between 45.8°C and 56.5°C. You can see very clearly a difference between the temperatures.</P></DIV></DIV></DIV>
 
We could clone and express the lysozyme of bacteriophage lambda and its fusion to different linkers (table 1: Linker and their amino acid sequence) chosen to calibrate our linker software. We used lyophilized <I>Micrococcus lysodeikticus</I> as substrate to measure lysozyme activity and we established a heat shock assay to measure the heat-stability of the enzyme. We observed that while lysozyme shows a clear activity at 37°C, this activity is gradually lost when the enzyme is pretreated at higher temperatures up to 57°C. Thanks to our measurement of the kinetics of substrate processing and to our model on enzymatic activity with product inhibition (see our model in section <A href="https://2014.igem.org/Team:Heidelberg/Modeling/Linker_Modeling" class="external text" rel="nofollow">Linker-Modeling</A>), we could extract the activity of the enzyme after different heat shock temperatures and with different linkers. This quantitative information was then used to calculate the weighting constants of the different linker features. 
The following figure (figure 5)  shows the substrate processing measured from the decrease of optical density of the <I>M. lysodeikticus</I> for the different temperatures of heat shock and the different linkers.
<TABLE class="table table-hover"><CAPTION><B>table 1</B>: Linker and their amino acid sequence. Green: attachment sequences to prevent the flexible regions from being perturbed; Blue: angle; Purple: extein.
</CAPTION><TBODY><TR><TH> Predicted linker quality
</TH><TH> Amino acid sequence
</TH></TR><TR><TD><B>Very good linkers</B></TD></TR><TR><TD> sgt2
</TD><TD><SPAN style="color:#3ADF00;">GG</SPAN>AEAAAK<SPAN style="color:#00BFFF;">AAAHPEA</SPAN>AEAAAK<SPAN style="color:#A901DB;">RGTCWE</SPAN></TD></TR><TR><TD> rigid
</TD><TD><SPAN style="color:#3ADF00;">GG</SPAN>AEAAAKEAAAKAA<SPAN style="color:#3ADF00;">P</SPAN><SPAN style="color:#A901DB;">RGKCWE</SPAN></TD></TR><TR><TD><B>Average linkers</B></TD></TR><TR><TD> may1
</TD><TD><SPAN style="color:#3ADF00;">GG</SPAN>AEAAAKEAAAKA<SPAN style="color:#00BFFF;">AAAHPEA</SPAN>AEAAAKEAAAKA<SPAN style="color:#00BFFF;">KTA</SPAN>AEAAAKEAAAKA<SPAN style="color:#A901DB;">RGTCWE</SPAN></TD></TR><TR><TD> ord1
</TD><TD><SPAN style="color:#3ADF00;">GG</SPAN>AEAAAKEAAAK<SPAN style="color:#00BFFF;">ATGDLA</SPAN>AEAAAKAA<SPAN style="color:#A901DB;">RGTCWE</SPAN></TD></TR><TR><TD> ord3
</TD><TD><SPAN style="color:#3ADF00;">GG</SPAN>AEAAAKEAAAK<SPAN style="color:#00BFFF;">ASLPAA</SPAN>AEAAAKEAAAK<SPAN style="color:#A901DB;">RGTCWE</SPAN></TD></TR><TR><TD><B>Short linkers</B></TD></TR><TR><TD> sho1
</TD><TD><SPAN style="color:#3ADF00;">GG</SPAN><SPAN style="color:#A901DB;">RGTCWE</SPAN></TD></TR><TR><TD> sho2
</TD><TD><SPAN style="color:#3ADF00;">GG</SPAN>AEAAAK<SPAN style="color:#A901DB;">RGTCWE</SPAN></TD></TR><TR><TD><B>Flexible linker</B></TD></TR><TR><TD> flex
</TD><TD> GGSGGGSG<SPAN style="color:#A901DB;">RGKCWE</SPAN></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Experimental_procedure">Experimental procedure</SPAN></H1><H2><SPAN class="mw-headline" id="Cloning_of_the_expression_constructs">Cloning of the expression constructs</SPAN></H2><P>The construct to express linear lambda lysozyme <A href="http://parts.igem.org/Part:BBa_K1362011" class="external text" rel="nofollow">BBa_K1362011</A> was assembled by <A href="https://2014.igem.org/Team:Heidelberg/Notebook/Methods#CPEC" class="external text" rel="nofollow">CPEC</A> from PCR products of <A href="http://parts.igem.org/Part:BBa_K1362093" class="external text" rel="nofollow">pSBX1K30</A> and phage lambda genome.  The RBS <A href="http://parts.igem.org/Part:BBa_K1362090" class="external text" rel="nofollow">BBa_K1362090</A> was added by primer overhangs.
In order to create the construct to express linear lambda lysozyme with His6-tag and exteins, primers to linearize <A href="http://parts.igem.org/Part:BBa_K1362011" class="external text" rel="nofollow">BBa_K1362011</A> between the lambda lysozyme coding sequence and stop codon and add BsaI sites at the ends were designed. An insert coding for the sequence GSHHHHHHSGRGKCWE (His6-tag + exteins) with overhangs corresponding to the overhangs caused by BsaI restriction of the vector PCR product was designed and ordered as DNA-oligos. The insert was annealed and inserted into the purified PCR product of <A href="http://parts.igem.org/Part:BBa_K1362011" class="external text" rel="nofollow">BBa_K1362011</A> by <A href="https://2014.igem.org/Team:Heidelberg/Notebook/Methods#Golden_Gate_Assembly" class="external text" rel="nofollow">Golden Gate assembly</A>.
</P><P>The constructs to express circular lambda lysozyme with a rigid linker <A href="http://parts.igem.org/Part:BBa_K1362013" class="external text" rel="nofollow">(BBa_K1362013)</A>, a His6 linker <A href="http://parts.igem.org/Part:BBa_K1362012" class="external text" rel="nofollow">(BBa_K1362012)</A> and all other linkers in shown in table 1 are based on the <A href="http://parts.igem.org/Part:BBa_K1362000" class="external text" rel="nofollow">NpuDnaE intein RFC(105) circularization construct</A>. These cloning steps were carried out according to our new RFC[105].
</P><P>For <A href="http://parts.igem.org/Part:BBa_K1362013" class="external text" rel="nofollow">BBa_K1362013</A> and <A href="http://parts.igem.org/Part:BBa_K1362012" class="external text" rel="nofollow">BBa_K1362012</A>, the coding sequence of lambda lysozyme without start and stop codon was aquired from genomic lambda DNA by PCR. Overhangs containing a BsaI site, the RFC[105] E standard overhang, the coding sequences of the extein CWE (forward) and the corresponding linker sequence, RGK, the RFC[105] B standard overhang, a BsaI site (reverse) were added by primers. The PCR products were purified and inserted each into the <A href="http://parts.igem.org/Part:BBa_K1362000" class="external text" rel="nofollow">NpuDnaE intein RFC(105) circularization construct</A> by Golden Gate assembly.
</P><P>For the constructs with the linkers sgt2, may1, ord1, ord3, sho1, sho2 and flex, a cloning intermediate containing both lambda lysozyme and the selection marker ccdB instead of the mRFP selection marker was created by Golden Gate assembly of <A href="http://parts.igem.org/Part:BBa_K1362000" class="external text" rel="nofollow">NpuDnaE intein RFC(i) circularization construct</A> and two inserts with subsequent <A href="https://2014.igem.org/Team:Heidelberg/Notebook/Methods#Religation_after_Golden_gate_Assembly" class="external text" rel="nofollow">religation</A>. The first insert was created by a PCR of the coding sequence of lambda lysozyme without start and stop codon aquired from genomic lambda DNA. Overhangs containing a BsaI site, the RFC[105] E standard overhang, the coding sequences of the extein CWE (forward) and a BsaI site (reverse) were added by primers. The ccdB insert was created by PCR of the ccdB selection marker from pDONR. Overhangs containing a BsaI site, the last 4 nucleotides of lambda lysozyme (forward) and the RFC[105] B standard overhang and a BsaI site (reverse) were added by primers.
Inserts coding for the linker sequences with overhangs corresponding to the overhangs caused by BsaI restriction of the cloning intermediate were ordered as DNA-oligos. The linker inserts were annealed and inserted to the cloning intermediate by Golden gate assembly. 
</P><H2><SPAN class="mw-headline" id="Expression_and_detection_of_circular_lysozyme_variants">Expression and detection of circular lysozyme variants</SPAN></H2><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 5) Measurement of substrate degradation after recovery</SPAN><P>All samples are active and not degraded.</P></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 4) Measurement of substrate degradation before recovery</SPAN><P>There is a difference between the activity of the lysozyme samples.</P></DIV></DIV></DIV>

With a Coomassie Gel and a Western Blot we tested the expressed construct on circularization. To do a Western Blot we built a circular form of lambda lysozyme with a His-tag as linker structure and a linear control including a His-tag, too. Circular proteins run faster in a gel than proteins of the same length because of their structure <A href="#References">[3]</A>. So you can see a shift on the gel between the linear and the circular form of a protein (figure 2). But what you could see on the Western Blot is that the product of the splicing reaction is not entirely circular. So in samples of circular lambda lysozyme there is still linear lysozyme.
<P>Because of the high expression level, tested with a Coomassie Gel (figure 3), we did not purify the lysozyme constructs and used non-purified protein mix. On the gel in figure 3 one can see that there is a difference between the expression level of the different constructs. But all constructs were expressed enough to measure their activity. 
To make sure our lysozyme samples were not degraded by proteases we measures samples two times, direct and after 20 hours at 37°C. But the activity persisted on the same level (figure 4 and 5).
</P><P>The linker screening was performed with ten different linkers. We wanted to show that different linkers have different effects on the thermostability and function of a protein, as claimed by our <A href="https://2014.igem.org/Team:Heidelberg/Modeling/Linker_Modeling" class="external text" rel="nofollow">Linker-Modeling</A>. The linker software calculated linkers classified according to fitness as great, bad, short and proper linker. The length of a linker had a significant effect on folding structure of proteins.
</P><DIV><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 3) Coomassie Gel of the linker constructs</SPAN><P>The expression levels of the linker constructs are different. In all lanes you can see intein fragments as result of the trans-splicing reaction.</P></DIV></DIV></DIV><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 2) Western Blot to prove circularization</SPAN><P>To test if lambda lysozyme is circular after the splicing reaction of the intein Npu Dna E we did a Western Blot with Penta·His Antibody. There is a clear shift between linear and circular samples.</P></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Development_of_assay_to_measure_lysozyme_activity">Development of assay to measure lysozyme activity</SPAN></H2><P>To test and quantify the differences in thermostability of the linear and several circular forms with different linkers of lambda lysozyme we needed an assay that is simple and reproducible enough. To have a comparison and a great check on functionality we tested two different methods to measure lysozyme activity and established the range of temperature to test the heat stability of the enzyme.
Lysozyme of the bacteriophage lambda like the other lysozymes cleaves glycosidic bonds between 1,4-beta-linkages between N-acetylmuramic acid (NAM) and N-acetyl-D-glucosamine residues (NAG) in peptidoglycan and between N-acetyl-D-glucosamine residues in. As a really small protein built of only 158 amino acids its mechanism is different from other lysozymes: the hydroxyl function OH on the C6 of the muramic acid is the nucleophile on cleavage of the muropeptide instead of the more common water molecule. So the protein is a transglycosidase, not a hydrolase <A href="#Reference">[4]</A>. 
The scientific background of the first assay was the capacity of lambda lysozyme to degrade the cell wall of <I>Micrococcus lysodeikticus</I>, a gram-positive bacterium often used as a lysozyme substrate  <A href="#Reference">[5]</A>. The rate of lysis of <I>Micrococcus lysodeikticus</I> was measured by a change of OD of the substrate solution at 600 nm. Using 96-well plates we have the possibility to test on several constructs within one assay.
</P><P>The basis of the second assay was the labeling of peptidoglycan with fluorescein isothiocyanate (FITC), an amine-reactive derivative of fluorescein dye. When the FITC-labeled substrate was subjected to the enzyme digestion, an increase of fluorescence intensity and a decrease of fluorescence polarization value should be observable. The labeling of peptidoglycane was done by ourself <A href="#Reference">[6]</A>.
</P><P>Performing both assays we realized in an early state that the first assay is less error-prone than the fluorescence intensity measuring assay. Both assays were measured in the plate reader Tecan Infinite F200 PRO. Even after several washing steps we could not exactly identify how effective the labeling was. Furthermore we had problems with the adjustment of the wavelength and the gain.
The assay measuring the degradation of the substrate had disadvantages, too. Because of the substrate <I>M. lysodeicticus</I> there were unknown values we could not predict. 
</P><P>The degradation of the substrate and associated with it the decrease of the OD was measured every 2 minutes for two or three hours to get all kinetic information and recognize a potential refolding.
At the beginning of the assay setup we measured the behavior of lambda lysozyme after a heat shock at temperatures up to 70°C until 90°C. Because of special characteristics of lambda lysozyme such as non-enzymatic behaviour around 70°C <A href="#Reference">[7]</A> we did not enhance the measurement due to higher temperatures.
For the linker screening the samples were heated for 1 minute between 37°C and 57°C. 
</P><P>Focus of the assay is to test the heated protein on stability not on functionality. If the lysozyme still degrades the substrate and retains function it shows that the protein is still folded correctly. We measured the activity and associated the heat-stability of lambda lysosyme after a heat shock to see behavior of refolding. So the first minutes after heating are very important to see potential refolding. So the speed the assay were performed is critical, too.
</P><H1><SPAN class="mw-headline" id="Materials_and_Methods">Materials and Methods</SPAN></H1><DIV><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 6) Composition of the final assay.</SPAN></DIV></DIV></DIV>

Bacteria were transformed with lambda lysozyme constructs, grown to an OD of 0.6 and induced with 1 mM IPTG. After 4 hours of expression, the samples were diluted to the same OD of 2.0 and centrifuged down at 2,850 rpm. The pellet was resuspended in 10 mM potassium phosphate buffer with a pH of 6.24 and sonicated for two minutes on ice. After centrifugation a second time at 2,850 rpm, the supernatant containing the lambda lysozyme was kept.
For substrate preparation, the lyophilized cells of <I>M.lysodeikticus</I> (Sigma Aldrich), were resuspended in ultrapure water. The supernantant with the protein mix was transfered and aliquoted for biological replicates and to prepare dilution series. 
Subsequently the samples were transfered into the thermocycler to do a one minute heat-shock in a temperature span between 45 and 55°C. 
After mixing enzyme and substrate the OD was measured every two minutes in a plate reader over 100 minutes at 37°C (figure 6). 
The expression of the lysozyme samples for the assay using FITC-labeled peptidoglycan was carried out in the same as we did it in the first assay. We resuspended the pellet in PBS. The protocol of  labeling of peptidoglycan with fluorescein isothiocyanate (FITC) you can find in the Materials and Methods. 
</DIV><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><P>We have established a protocol with which one is able to make a quantitative analysis on the effect of different linkers heat stability of lysozyme. This method can not only be performed in large scale, but is also resistant to various influences, like different expression levels of the constructs. With this assay in short time sufficient data of high quality can be produced, so that an extensive enzyme modeling approach can be made for their evaluation. 
</P><P>For the evaluation the data of the degradation curves from the assays were analyzed for the change in activity after different heatshocks. This was done applying different models to the data and analyzing it by use of Profile Likelihood Estimation, like we described in our <A href="/Team:Heidelberg/Modeling/Enzyme_Modeling" title="Team:Heidelberg/Modeling/Enzyme Modeling"> enzyme modeling</A> part. To each degradation curve by itself we did not apply a complex model, but the advantage lies in modeling the data in total. The complexity lies in the combination of the different model parameters. The models were always fitted to the data of multiple experiments at once, leading to an increased reliability than fitting single data.
</P><P>We could show, that the data was well described by applying product inhibited Michaelis Menten kinetics. Further on we found out, that the model's quality did not increase significantly by letting the enzyme kinetics ($K_M, K_I$) parameters change over temperature. This was done by making identifiability analysis of model parameters. We showed, that when not restraining $K_M$ and $K_I$ for different temperatures none of the important parameters was identifiable anymore. For better readability we introduced parameters representing the loss in activity after heat shock. These were called activities at a temperature and were built as the ratio between $V^T_{Max}$ and $V^{37°C}_{Max}$. These parameters were the most interesting ones for us. When each lysozyme was modeled with constant $K_M$ and $K_I$ for different temperatures nearly the activities were identifiable within 95% confidence level. For some very good datasets even the kinetic parameters $K_M$ and $K_I$ could be identified on a lower confidence level. Therefore we concluded that the change in these parameters for the different temperatures was minor and could be neglected in the further evaluation.
</P><P>We showed that the different linker constructs show different changes in activity after heat shock, figure 7. 
</P><DIV class="pull-right col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 7, Activity after heatshock</SPAN><P>Ratio of activity after a heatshock of a certain temperature. The single points come from different technical and biological replicates. For clarification a simple spline was fitted to the data. The fits produced 95% confidence intervals, that would exceed the scale. But as on can see that mostly the different replicates don't scatter too much, these errors are not too relevant. A clear difference can be seen for ord1- and ord3-linker in comparison to the flexible linker and the linear lysozyme</P></DIV></DIV></DIV><P>For further clarification we have calculated the average activity between 45 and 50°C, because there the biggest change in activity could be observed for the different constructs. Furthermore the values of the activities vary a lot, what can especially be seen at circsho1 linker. This averaging was made for clarification and for obtaining a final ranking of the linkers to feed back to the linker software (see table 2)
</P><TABLE class="table table-hover"><CAPTION><B>table 2</B>: Ranking of different linkers on enhancement of thermostability and their classification
</CAPTION><TBODY><TR><TH> Linker
</TH><TH> classification
</TH><TH> average activity from 45°C to 50°C
</TH></TR><TR><TD> ord3
</TD><TD> Average
</TD><TD> 1.390
</TD></TR><TR><TD> ord1
</TD><TD> Average
</TD><TD> 0.956
</TD></TR><TR><TD> rigid
</TD><TD> Very good
</TD><TD> 0.945
</TD></TR><TR><TD> may1
</TD><TD> Average
</TD><TD> 0.749
</TD></TR><TR><TD> sgt2
</TD><TD> Very good
</TD><TD> 0.748
</TD></TR><TR><TD> sho1
</TD><TD> bad
</TD><TD> 0.709
</TD></TR><TR><TD> linlys
</TD><TD> linear
</TD><TD> 0.704
</TD></TR><TR><TD> flex
</TD><TD> Flexible
</TD><TD> 0.685
</TD></TR><TR><TD> sho2
</TD><TD> bad
</TD><TD> 0.574
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H1><P>Our Linker screening was the first time that rigid linkers with angles were tested on their effect on thermostability. Further on we could extract the relevant data to calibrate <A href="/Team:Heidelberg/Software/Linker_Software" title="Team:Heidelberg/Software/Linker Software"> CRAUT</A> software with the results. Still important enzymatic parameters could not be identified reliably. Due to the lack of time the amount of replicates was restricted. Mainly the time between adding the enzyme to the substrate and the first measurement influences the quality of the data. Further refinement there could lead to better identifiability of the parameters. The errors on the parameters obtained from the fitting would need to be reduced. On the other hand one is tightly limited in the initial substrate concentration in a range until 0.7 mg/ml, which makes it difficult to identify all the parameters. Further on the linkers could be tested with different proteins to make predictions more independent of the protein.
</P><H1><SPAN class="mw-headline" id="The_long_way_to_setup_the_assay">The long way to setup the assay</SPAN></H1><P>Enzyme kinetics and protein interactions are very complex fields of modeling and biochemistry. For the present assay, there were many different dimensions that has to be tested and optimized: concentration of enzymes, of substrates, temperature for the enzyme-substrate reaction, heat shock temperature and length, as well as length of recovery after heat shock. So we had to test many factors, solve many problems and find new ways to establish our linker screening to a standardized and ideal method for testing the effect of different linkers on proteins. The following part details the establishment of our assay.
The main issues were finding the correct temperatures, the correct lengthes of heatshock and the right concentrations.
</P><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">22th of August</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">21th of August</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">20th of August</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">19th of August</SPAN></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="14th_of_August">14th of August</SPAN></H4><P>In the first assay we used HEWL (ordered from Sigma Aldrich) and <I>E.coli</I> lysate without lysozyme to see how lysozyme behaves in general and how the assay works in total. Furthermore we wanted to see a difference between HEWL containing samples and <I>E.coli</I> lysate. All samples were heated from 70°C to 90°C in the cycler.
The results of this first test were curious. It seems that even without lysozyme and without <I>E.coli</I> lysate, the substrate is degraded.
</P><H4><SPAN class="mw-headline" id="17th_of_August">17th of August</SPAN></H4><P>Repeating the first assay we could improve the results and saw first temperature dependence. This assay did not give great results, too. But we figured out that the concentration of HEWL was too high, which might explain the behavior of the samples, because at the higher temperatures we observed big coagulation. Furthermore we mentioned that we are too slow preparing and performing the assay, as most of the substrate is already degraded, when we had the first measurement. The most information about the enzyme kinetics can only be seen in the degradation of the <I>Micrococcus lysodeicticus</I> in the very first minutes. 
</P><H4><SPAN class="mw-headline" id="19th_of_August">19th of August</SPAN></H4><P>After using HEWL in the last assays we now tested a linear lambda lysozyme construct we had cloned and expressed the weeks before. The samples were heated at a different temperature span between 50°C and 70°C. This was the first time we really could see different curve progressions after a heat-shock at different temperatures. We changed our method, so that we were faster in the platereader after adding the lysozyme. The sobering message: The activity grows with higher temperature.
</P><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">26th of August</SPAN></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="20th_of_August">20th of August</SPAN></H4><P>To explain the behavior of the lysozyme around 70°C we saw in the last assay we measured the degradation of substrate after a heat-shock in a temperature span between 70°C and 95°C. It seems that lysozyme has his highest activity around 70°C. Later we detected that lysozymes has a non-enzymatic activity <A href="#References">[7]</A>, which explains the increase of activity around 70°C. Controls containing no lysozyme did not show any activity. All graphs showed &quot;shoulders&quot; and not a clean exponentiell curve. Troubleshooting possibilities like salt, measuring temperature had to be checked before performing the next assay.
</P><H4><SPAN class="mw-headline" id="21st_of_August">21st of August</SPAN></H4><P>To find out the best concentration of the lysate we measured a dilution series. High concentrated samples were coagulating, protein lysate less diluted did not showe this effect. It was the first time we saw a significant activity of lysozyme at 37°C. The 1 to 10 dilution is seems to be the appropriate dilution, because activity was high enough.
</P><H4><SPAN class="mw-headline" id="22nd_of_August">22nd of August</SPAN></H4><P>Testing lower temperatures we decided to do a heat shock at temperatures between 30°C and 50°C. The activity of lambda lysozyme decreases until 50°C, the maximum activity samples showed that were heated at lower temperatures. 
</P><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">30th of August</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">30th of August</SPAN></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="26th_of_August">26th of August</SPAN></H4><P>Next to enzyme dilutions we tested substrate dilutions to. All constructs were kept on 37°C, even during the measurement the plate reader had this temperatur. We found out the perfect concentration for the substrate, next assays were done with 0.5 % (w/v). With lower concentrations the errors were large. 
</P><H4><SPAN class="mw-headline" id="30th_of_August">30th of August</SPAN></H4><P>Next to the speed the accuracy of pipetting is significant, too. In this assay we payed heed to exact pipetting. Furthermore we measured many replicates on one well to check on correctness of pipetting. The differences between the replicates were low.
</P><H4><SPAN class="mw-headline" id="5th_of_September">5th of September</SPAN></H4><P>The cloning of the circular form of lambda lysozyme was finished, so we tested this construct the first time to see, if the circular form of lysozyme is still active. Because it was the first assay with circular constructs, the rigid-linker construct and the His-linker construct, we did not heated them. All samples stayed on 37°C.
</P><DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">5th of September</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">9th of September</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">10th of September (1)</SPAN></DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="9th_of_September">9th of September</SPAN></H4><P>Aim of this assay was to see differences in the temperature stability of the different lysozyme constructs, linear and circular, after a heat shock of half of an hour between 35°C and 55°C. The circular lysozyme constructs are clearly working and the probes with the rigid linker work better than the construct including a His-tag. Before declaring a difference between circular and linear constructs we needed more results for great statistics.
</P><H4><SPAN class="mw-headline" id="10th_of_September_.281.29">10th of September (1)</SPAN></H4><P>Next to the first assay we tested a second assay based on an increase of fluorescence intensity. We performed the assay with many different dilutions of substrate and of lysozymes, testing the fluorescein labeled peptidoglycane.To proof the labeling we added the supernatants of the washing steps to see whether they are still fluorescing or not. The fluorescence absolutely doesn't behave as we expected. We don't see any enlargement of the fluorescence. The different curves have huge variations between them.
</P><DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">10th of September (2)</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">10th of September (2)</SPAN></DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="10th_of_September_.282.29">10th of September (2)</SPAN></H4><P>To have a higher statistical coverage we reproduced the last assays. The temperature span of the heat shock was 37°C until 54°C. From this assay on, we could test and evaluate different linkertypes properly, as we have introduced a fitting method, showing the activity of the lysozymes as the exponent of the exponential decay. As we always had different expression levels, we introduced a normalization of the activity to the activity at 37°C.
There seems to exist some offset activity, the lysozymes always keep.
In the plots above you can see, how important a normalization is, as the activity plots sometimes are too crowded. But on the other hand, these plots can be misleading, as they overestimate, when there is low activity already.
</P><H4><SPAN class="mw-headline" id="11th_of_September_.281.29">11th of September (1)</SPAN></H4><P>To have a comparison between biological replicates we repeated the last assay  under the same conditions. The results are similar, our assay indeed is reproducable with different biological samples.
</P><DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">12th of September</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">11th of September</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">11th of September</SPAN></DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="11th_of_September_.282.29">11th of September (2)</SPAN></H4><P>To find a temperature span where we can finally screen our linkers we did the last assay at higher temperatures up to 70°C. 
</P><H4><SPAN class="mw-headline" id="12th_of_September">12th of September</SPAN></H4><P>In this assay we increased the temperature level of the heat shock, again. Aim of the last and following assays is to have data for the complete span up to 95°C. One can see, that while linear lysozyme has an activity maximum at 75°C circular lysozyme has not.
</P><H4><SPAN class="mw-headline" id="14th_of_September">14th of September</SPAN></H4><P>The samples were heat shocked for half of an hour at 78°C up to 95°C. 
At least at a heat shock of 95°C all the samples loose activity, but somehow are always clearly distinguishable from the controls.
</P><H4><SPAN class="mw-headline" id="15th_of_September">15th of September</SPAN></H4><P>In the last assays we recognized that the temperature span between 45°C and 55°C is interesting one for us related to a difference between the linear lysozyme and the lysozyme with the rigid linker in loosing the conformation from 37°C. So we repeated the assay in this span. There can be seen huge differences in the temperature behaviour of the different lysozymes after normalization. But it is not clear, whether they come from differences in expression level, or not.
</P><DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">15th of September</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">15th of September</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">14th of September</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">14th of September</SPAN></DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="19th_of_September">19th of September</SPAN></H4><P>On this assay OD calibration was made. Also we tried to have same &quot;standard&quot; activity at 37°C for all constructs, so that normalization isn't that grave anymore. The OD can be calculated to <I>M. lysodeicticus</I> concentration in the linear range until dilution of substrate of 0.65 mg/ml with 1.160 ml/mg + 0.13. So we can extend our range to 0.66 mg/ml, so that we have more time where we can measure. 
</P><H4><SPAN class="mw-headline" id="28th_of_September">28th of September</SPAN></H4><P>To test the recovery of the lysozyme constructs after heat-shock we did two assays with the same heated samples. One part we let recover for 20h at 37°C. The temperature span is from 40.3°C-54 °C with a reference temperature at 37°C. The heat shock was 30 minutes. The substrate concentration was after calculation of the last assay 0.66 mg/ml. We had two aims: Is there protease activity and is there some effect of heatshock.
</P><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">29th of September</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">28th of September</SPAN></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="29th_of_September">29th of September</SPAN></H4><P>This assay was made after a recovery time of the lysozyme samples of 20h. The samples recovered at 37°C over night. It was measured at 37°C in the platereader as well. This is the continuation of the test of 29th of september. The rest of the samples that could recover over night was put on a plate. We could not see any recovery from the heatshock and protease activity. The activity levels for both lysozyme constructs were nearly the same before and after the recovery time.
</P><H4><SPAN class="mw-headline" id="30th_of_September">30th of September</SPAN></H4><DIV><P>We want to find the appropriate length of heat shock for our assay to see the effect of different heatshock lengths and figure out whether there is a difference between the different lysozyme constructs. The samples were heated for 1 and for 7 minutes. For such short heatshocks in the temperatures where normally the activity is lost, most of the activitiy for all of the constructs remained still. For the ridid linker activity even increased and had a maximum activity at 46 degrees. Unfortunately the errors are enormous as we only had duplicates on the wellplate. The 1 minute heatsock seems the range where we could observe differences between the different linkers.
</P><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">1st of October</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">4th of October</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">4th of October</SPAN></DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="1st_of_October">1st of October</SPAN></H4><P>The samples were heated for 14 and for 21 minutes. With this longer heatshocks we observe the same behaviour as with the 30 minutes, so no clear tendencies can be seen. So we decided to perform all following assays of the linker screening with a heat shock time of 1 minute. The last preperations to set up the final assay were finished.
</P><H4><SPAN class="mw-headline" id="4th_of_October">4th of October</SPAN></H4><P>The final protocol you can find in in Materials and Methods or in parts of the text above!
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>[1] Callewaert, L. &amp; Michiels, C. W. Lysozymes in the animal kingdom. J. Biosci. 35, 127–160 (2010).
</P><P>[2] Evrard, C., Fastrez, J. &amp; Declercq, J. P. Crystal structure of the lysozyme from bacteriophage lambda and its relationship with V and C-type lysozymes. J. Mol. Biol. 276, 151–64 (1998).
</P><P>[3] Iwai, H., Lingel, a &amp; Pluckthun, a. Cyclic green fluorescent protein produced in vivo using an artificially split PI-PfuI intein from Pyrococcus furiosus. J. Biol. Chem. 276, 16548–54 (2001).
</P><P>[4] Callewaert, L. et al. Purification of Ivy, a lysozyme inhibitor from Escherichia coli, and characterisation of its specificity for various lysozymes. Enzyme Microb. Technol. 37, 205–211 (2005).
</P><P>[5] May, R. Turbidimetric Determination of Lysozyme with Micrococcus Cells : Reexamination of Reaction Conditions. 85, 77–85 (1983).
</P><P>[6] Maeda, H. A New Lysozyme Assay Based on Fluorescence Polarization or Fluorescence Intensity Utilizing a Fluorescent Peptidoglycan Substrate 1. 1191, 1185–1191 (1980).
</P><P>[7] Du, K., Porsch, P., Mahn, A., Brinkmann, O. &amp; Gie, W. The non-enzymatic microbicidal activity of lysozymes. 449, 93–100 (1999).
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Heidelberg/pages/Linker_Screening">http://2014.igem.org/Team:Heidelberg/pages/Linker_Screening</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/pages/Linker_Screening" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/pages/Linker_Screening" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/pages/Linker_Screening&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/pages/Linker_Screening&amp;oldid=403743" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>