<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Modeling_Linker_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Modeling/Linker_Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Modeling/Linker_Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Modeling/Linker_Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Modeling/Linker_Modeling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Modeling/Linker_Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Modeling/Linker Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container" style="padding:0;background-color:white;"><HEADER id="navbar" role="banner" class="navbar navbar-static-top navbar-white"><DIV class="container" style=" background-color:black;"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse collapse" style="margin-bottom:10px;"><NAV role="navigation"><UL class="menu nav navbar-nav navbar-right"><LI class="dropdown"><A href="/Team:Heidelberg/Team" class="dropdown-toggle">Team</A></LI><LI><A href="/Team:Heidelberg/Team/Members">Members</A></LI><LI><A href="/Team:Heidelberg/Team/Collaborations">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="/Team:Heidelberg/Project" class="dropdown-toggle">Project</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Background">Background</A></LI><LI><A href="/Team:Heidelberg/Project/Toolbox">Toolbox</A></LI><LI><A href="/Team:Heidelberg/Toolbox/Circularization">Toolbox: Circularization</A></LI><LI><A href="/Team:Heidelberg/Project/PCR_2.0">PCR 2.0</A></LI><LI><A href="/Team:Heidelberg/Project/Xylanase">Xylanase</A></LI><LI><A href="/Team:Heidelberg/Project/Linker_Screening">Linker Screening</A></LI><LI><A href="/Team:Heidelberg/Project/Reconstitution">Fluorescence Assembly</A></LI><LI><A href="/Team:Heidelberg/Project/LOV">Light-Induction</A></LI><LI><A href="/Team:Heidelberg/Project#Achievements">Achievements</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Parts" class="dropdown-toggle">Parts</A></LI><LI><A href="/Team:Heidelberg/Parts/RFC">RFC Heidelberg 2014</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Favorite Parts">Favorite Parts</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Sample Data Page">Sample Data Page</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#allParts">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Backbones">Backbones</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Intein Library">Intein Library</A></LI><LI><A href="/Team:Heidelberg/Parts/Part_Improvement">Part Improvement</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Software" class="dropdown-toggle">Software</A></LI><LI><A href="/Team:Heidelberg/Software/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Software/Linker_Software">Linker Software</A></LI><LI><A href="/Team:Heidelberg/Software/MidnightDoc">MidnightDoc</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Modeling" class="dropdown-toggle">Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Linker_Modeling">Linker Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Enzyme_Modeling">Enzyme Kinetics Modeling</A></LI><LI><A href="/Team:Heidelberg/Toolbox_Guide" class="dropdown-toggle">Toolbox Guide</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Human_Practice" class="dropdown-toggle">Human Practice</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Ethics">Religion &amp; Synthetic Biology</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Education">Education</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Public_Relations">Public Relations</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Experts">Experts</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Notebook" class="dropdown-toggle">Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="/Team:Heidelberg/AwesomeSheet">Database for Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook/Materials">Materials</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Notebook/Safety">Safety</A></LI></NAV></DIV><DIV class="col-lg-12"><DIV class="title-wrapper"><SPAN class="title">LINKER MODELING</SPAN></DIV></DIV></DIV></HEADER><DIV class="container main"><DIV class="row" style="position:relative;"><DIV class="col-lg-offset-1 col-lg-11 subtitle"><H2><P>– Building blocks for customized linkers</P></H2></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><H2 class="titles"><A href="#Introduction" class="label label-default">Introduction</A></H2></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><DIV class="well well-lg abstract"><P><B>ABSTRACT</B></P>
Artificially circularized proteins can acquire remarkable properties reviewed in our introduction on <A href="https://2014.igem.org/Team:Heidelberg/Toolbox/Circularization" class="external text" rel="nofollow">circularization</A>. These properties include heat stability thanks to the constrain of the relative positioning of the C- and the N-termini <A href="#References"> [1]</A>. circularization requires a linker that provide this constrain without changing the natural conformation of the protein. Designing such a linker is particularly non-trivial if the protein extremities are far from each other. Peptidic linkers are generally used in protein modification, not only for connecting protein subdomains but also for ligating their extremities to circularize them. Traditionally,  flexible linkers like Glycin-Serine peptides are used for this purpose <A href="#References"> [2]</A>. However, to keep domains of chimeric proteins in a certain distance, rigid peptides built of helical patterns are also often applied <A href="#References"> [3]</A><A href="#References"> [4]</A>. Here we show a novel approach to build customized rigid linkers that follow a desired shape. This is achieved by connecting peptides forming rigid helices with amino acids that produce a certain angle between them. We herein describe the building blocks with which one can build customized linkers. The building blocks we designed fall into two categories, the alpha helix blocks forming rigid rods, and the angle patterns. The latter were obtained from statistical analysis of supersecondary structures from structural databases, normally used for protein structure prediction.  The potential linkers were tested in a large screening in silico, in order to refine their properties. Additionally they were tested <I>in vitro</I> for <A href="https://2014.igem.org/Team:Heidelberg/Project/Linker_Screening" class="external text" rel="nofollow">circularizing lysozyme</A> from bacteriophage lambda as a model enzyme. This experimental work was also used to measure and calibrate the contribution of different linker features to heat stabilization. The outputs of our modeling approach and of the experimental work were combined in a <A href="https://2014.igem.org/Team:Heidelberg/Software/Linker_Software" class="external text" rel="nofollow">software</A> that designs circularization linkers for any protein with a known structure. Not only the modularity but also the reliability of our linkers are huge advantages compared to normally used linkers made of alanine or of a mix of glycine and serine. This part explains the statistical analysis performed to identify the building blocks and how we refined them by structure modeling. Finally, we show how this approach was used to design linkers used in our experiments.

		</DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Background"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2"><A href="#Supersecondary_structure"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Supersecondary structure</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Linker_building_block_design"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Linker building block design</SPAN></A></LI><LI class="toclevel-2"><A href="#Helix_patterns"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Helix patterns</SPAN></A></LI><LI class="toclevel-2"><A href="#Angle_patterns"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Angle patterns</SPAN></A></LI><LI class="toclevel-2"><A href="#Sequences_to_connect_the_alpha_helix_to_the_protein_extremity"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Sequences to connect the alpha helix to the protein extremity</SPAN></A></LI><LI class="toclevel-2"><A href="#Conclusion"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-1"><A href="#In_silico_refinement"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">In silico refinement</SPAN></A></LI><LI class="toclevel-1"><A href="#Conclusion_2"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Background">Background</SPAN></H1><P>Primary, secondary, tertiary and quaternary structures are the main levels of protein structure characterization. Primary structure designates the amino acid sequence, while the secondary structure describes the arrangement of consecutive amino acids  through their two dihedral angles $\phi$ and $\psi$. The Ramachandran plot, which represents the amino acid position in the space of those two angles, shows two particular arrangement commonly found in proteins: alpha helices and beta sheets. The next level of protein organization is the tertiary structure, which describes how the protein is organized in the three spatial dimensions, whereas the quaternary structure describes how different subunits of proteins cluster.
Finally, closely related to these standard structures, the supersecondary structure describes how secondary structure elements are connected to each other. While these connections look undefined at first sight, further analysis revealed that this wide variety of supersecondary structure motifs can be clustered to certain patterns <A href="#References">[5]</A>.
</P><H2><SPAN class="mw-headline" id="Supersecondary_structure">Supersecondary structure</SPAN></H2><P>When the properties of supersecondary structures were first described, only very few patterns were identified, mainly due to the lack of highly resolved protein structures. At that time, the structures were mainly classified by the Ramachandran plot regions where the amino acids could be found <A href="#References">[6]</A>. With growing amount of known crystal structures, the analysis of supersecondary structure improved and lead to databases with about 150 000 classified loop structures and elaborate clustering <A href="#References">[7]</A>. Nowadays supersecondary structures are defined as the structures built when two secondary structure elements are combined by a small peptide that is not clustered into one of the secondary structures. These loop peptides range from 1 to 9 amino acids.
Our aim was to build reliable stable linkers out of alpha helices connected by supersecondary structure motifs that produce certain angles. To achieve that, we searched for the most reliable alpha helix patterns that would form rigid rods and angle patterns covering the whole range of angles from 0 to 180 degrees.
</P><H1><SPAN class="mw-headline" id="Linker_building_block_design">Linker building block design</SPAN></H1><H3><SPAN class="mw-headline" id="Helix_patterns">Helix patterns</SPAN></H3><P>Various different patterns have been used to build helical linkers to connect protein ends <A href="/wiki/index.php?title=References&amp;action=edit&amp;redlink=1" class="new" title="References (page does not exist)"> [8]</A>. Moreover, in known protein structures, linkers between subdomains can be identified and their properties have already been analyzed <A href="#References"> [9]</A>. Two main criteria were used to build the alpha helix patterns: they should robustly for alpha helices, and they should be soluble in aqueous solution. Therefore, we could not just use linkers built of Alanine. So we decided to add some charged aminoacids to the pattern, and to position them physically close to each other so that they could stabilize themselves by Coulomb interaction. These amino acids needed to be separated by 3 amino acids as a helical turn takes about 3.6 aminoacids. The pattern we chose as most suitable for our purpose was also described to be one of the most stable <A href="#References"> [3]</A>.
8 alpha helix building blocks were eventually chosen: AEAAAK, AEAAAKA, AEAAAKAA, AEAAAKEAAAK, AEAAAKEAAAKA, AEAAAKEAAAKEAAAKA, AEAAAKEAAAKEAAAKEAAAKA, AEAAAKEAAAKEAAAKEAAAKEAAAKA, with a respective estimated length of 9, 10.5, 12, 16.5, 18, 25.5 and 33Å.
</P><H3><SPAN class="mw-headline" id="Angle_patterns">Angle patterns</SPAN></H3><P>The angle patterns for our model were obtained from the ArchDB database <A href="#References"> [5]</A>, which classifies loops from known proteins structures. About 17 000 non-homologous proteins from PDB database were analyzed and from them, 150 000 loop structures, i.e. regions connecting two secondary structure elements, were identified. The classification took into account not only the length of the loop, its conformation, meaning φ and ψ backbone dihedral angles of the residues in the loop, but also the distance between the attachments of the loop to the surrounding secondary structures. Furthermore the secondary structures surrounding the loop and the geometry defined by the super-secondary structure motifs can be found in the database.
</P><P>To extract from ArchDB the relevant supersecondary structure motifs for our linker design, the complete database was downloaded and helix-loop-helix motifs were extracted using a <A href="https://github.com/igemsoftware/Heidelberg_2014" class="external text" rel="nofollow">self-written script</A> in Python programming language. From them we only took into account loops composed of 1 and 2 amino acids, because the longer the loops, the less frequent and therefore the less reliable they are, and the further the ends are from each other.
The interesting information for us was (1) the angle produced between the vectors defining the surrounding alpha helices, (2) the distance between the ends of the loop, and (3) the type of amino acids surrounding the loop. Furthermore we analyzed the statistical significance of the conformation. For each amino acid combination in the loop region, the angle distribution between the embracing alpha helices, the loop length distribution and a 2D heatmap of the surrounding amino acids were plotted automatically (Figures 1-4).
</P><P>These distributions were then visually analyzed to identify loops of interest for linker design. We focused on loops that showed a narrow angle distribution and that appeared frequently in the database and we extended the distribution analysis to the surrounding amino acids. For example, when we have identified that the amino acid T in a turn produces an interesting distribution (Figure 1), we would elongate it with the amino acid K in front (Figure 2), and with the amino acid A at the end (Figure 3), and with both (Figure 4) and investigate how the distributions behave. By restraining the possibilites, the occurrences go down tremendously, but the properties become more interesting.
</P><DIV class="pull-{{{align}}} col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><P>Figure 1) A loop composed of the amino acid T produces an interesting well-defined angle distribution (left panel). The middle panel represents the distribution of lengths found with this loop. The right panel represent the distribution of amino acids found before and after the amino acid T in all the corresponding loops. One can see that the amino acid K is the most frequent before and A the most frequent after.</P></DIV></DIV></DIV><DIV class="pull-{{{align}}} col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><P>Figure 2) These distributions are subsets of the ones of Figure 1, with only the loops composed of the amino acid T preceded by the amino acid K. This constrain considerably narrows down the angle and length distributions.</P></DIV></DIV></DIV><DIV class="pull-{{{align}}} col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><P>Figure 3) Same procedure as in Figure 2. with the loops composed of the amino acid T followed by the amino acid A. This also narrows down the angle and length distributions compared to Figure 1., although not as much as in Figure 2.</P></DIV></DIV></DIV><DIV class="pull-{{{align}}} col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><P>Figure 4) The distributions correspond to loops composed of the amino acids KTA and therefore to subsets of all the preceeding ones. One can notice that the angle distribution becomes very narrow, but also that the frequency is reduced compared to the preceeding ones.</P></DIV></DIV></DIV><P>This step had two main goals: narrowing down the angle distribution and finding loops with no preferences for the amino acid surrounding them. This last point was important for the modularity of our approach: the angle blocks should not be affected by the surrounding alpha helices. Using this approach, 10 different angle motifs (table 1) could be identified producing different angles. Importantly, all these motifs were chosen so that the length of the two segments starting from the turning point was the same for all of them (put figure turning point). It should also be noticed that 3 of the patterns, KTA, LVA and AAIAP, produce the same angles. There were nevertheless all kept for the <I>in silico</I> refinement described below. But only one was used for the <A href="https://2014.igem.org/Team:Heidelberg/Software/Linker_Software" class="external text" rel="nofollow">CRAUT software</A>.
</P><TABLE class="table table-hover" style="text-align: center;"><CAPTION><B>Table 1</B>: The span of parameters.
</CAPTION><TBODY><TR><TH colspan="10">Angle Patterns
</TH></TR><TR><TD>Pattern
</TD><TD> NVL
</TD><TD> KTA
</TD><TD> LVA
</TD><TD>   AAIAP
</TD><TD>  AADGTL
</TD><TD> VNLTA
</TD><TD> AAAHPEA
</TD><TD> ASLPAA
</TD><TD> ATGDLA
</TD></TR><TR><TD>Mean
</TD><TD>  29.7
</TD><TD>  38.7
</TD><TD>   35
</TD><TD>   36.5
</TD><TD> 60
</TD><TD>  74.5
</TD><TD>   117
</TD><TD>  140
</TD><TD>   160
</TD></TR><TR><TD> Variation
</TD><TD> 8.5
</TD><TD> 30
</TD><TD>    29
</TD><TD>      27
</TD><TD>   12
</TD><TD>   27
</TD><TD>   12
</TD><TD>  15
</TD><TD>     5
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Sequences_to_connect_the_alpha_helix_to_the_protein_extremity">Sequences to connect the alpha helix to the protein extremity</SPAN></H3><P>Helical patterns often affect the folding of the attached sequences. To prevent them from affected the structure of the protein of interest, we analyzed the effects of various aminoacids in silico using online tools like <A href="http://bioserv.rpbs.univ-paris-diderot.fr/services/PEP-FOLD/" class="external text" rel="nofollow">pep-fold</A> on helix formation. We identified glycines and prolines as reliable amino acids to interrupt helix formation. We then decided to use glycine pairs to connect the protein of interest to the linker, because they give more flexibility to the initial orientation of the initial helix.
This only concerns the protein end that was connected to the linker through the coding sequence. The other end is ligated through exteins or sortase scar, both treated as unstructured flexible regions.
</P><H3><SPAN class="mw-headline" id="Conclusion"><I>Conclusion</I></SPAN></H3><P>The last three parts show how we could design alpha helix and angle pattern blocks and connect them to each other and to the protein of interest. They provide the material that allowed us to transform a linker defined as a geometrical path into a real amino acid sequence in our <A href="https://2014.igem.org/Team:Heidelberg/Software/Linker_Software" class="external text" rel="nofollow">software</A>.
</P><P>Thanks to this we could design linkers to circularize <A href="https://2014.igem.org/Team:Heidelberg/Project/PCR_2.0" class="external text" rel="nofollow">DNMT1</A> and <A href="https://2014.igem.org/Team:Heidelberg/Project/Linker_Screening" class="external text" rel="nofollow">lysozyme</A>. Additionally it was shown for lysozyme, that a customly tailored linker enhances heat-stability compared to a badly designed linker and even to the flexible linker.
</P><H1><SPAN class="mw-headline" id="In_silico_refinement"><I>In silico</I> refinement</SPAN></H1><P>As some of the interesting patterns could not be found often enough to be statistically significant, we decided to make a further refinement <I>in silico</I> by modeling the structure of proteins with circularizing linkers. To perform this for realistic situations, we selected, from the RCSB database, structures of non-homologous target proteins with extremities that are separated enough to require a linker for circularization. First, the <A href="https://2014.igem.org/Team:Heidelberg/Software/Linker_Software" class="external text" rel="nofollow">CRAUT software</A> generates possible fitting linkers for various proteins. From these possible linkers, the 100 shortest were taken. Among the 3 possible angle patterns that generate the same angle, the software provides only one. But the linker refinement developed here used the three of them for comparison. We assume that the linkers connect the ends of the protein without setting tension on the protein, so that the protein can fold in its natural way.
</P><P>After this the circularized proteins with the specific linkers are modelled using a software called Modeller <A href="#References">[10]</A>. This software is widely used for comparative structure prediction. It is well established in the scientific community and should be most suitable for prediction of loop regions attached to existing structures <A href="#References">[11]</A>. It is freely available for academical usage from the  <A href="http://salilab.org/modeller/" class="external text" rel="nofollow">salilab</A> webpages. The program is able to predict the 3d structure of a given sequence based on an alignment with a given structure. In our script Modeller needed to be provided with a sequence with the linker attached and with the PDB file of the protein of interest. This latter file is the only structural information used here. Modeller is recommended when at least 30% sequence similarity exists between the provided structure and the one that the user wish to model. Here, only the linker is different, so this similarity if around 90% in our case. At first Modeller makes an alignment between the provided structure and the sequence of our linked protein, identifying the regions that cannot be found in the structure. Based on that, Modeller generates 4 initial models. One of the strengths of the software is its capacity to further refine only certain parts of the protein. Thus we let Modeller refine the loops, which are defined as any part of the protein that could not be found in the structure file. The <I>ab initio</I> modelling of our linkers is made by minimizing energy functions with different methods like conjugate gradients and molecular dynamics. Eventually, each modeled structure is provided with energy values, thanks to which different models of the same structure can be compared. From Modeller we received about 8 different models and choose the one with the best energy scores to further proceed. For these refinement steps, one can choose different levels of optimization. We always decided for accuracy instead of velocity of the program. The result of a prediction for lysozyme of bacteriophage lambda can be seen in figure 5.
</P><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 5) Circular lysozyme</SPAN><P>The structure of circular lysozyme (yellow) was predicted and aligned to the linear structure (purple). The linker calculated by the software was marked in green.</P></DIV></DIV></DIV><P>Modeller was run by distributing calculation via the <A href="https://2014.igem.org/Team:Heidelberg/Software/igemathome" class="external text" rel="nofollow">iGEM@home system</A> system. The modelling of one linker took about 10 hours of calculation time on average, a value that is highly depending on the size of the protein. Then the best model is evaluated  by another self-written program to analyze the behaviour of the linker patterns in their natural surroundings.
Finally, all the models for the different structures and the different linkers are analyzed for their properties like the length of the helical patterns, the shape of the attachment structures of the linker and the angles produced by the angle patterns <A href="/wiki/index.php?title=Figure_helix_winkel_messung.png&amp;action=edit&amp;redlink=1" class="new" title="Figure helix winkel messung.png (page does not exist)"> Figure helix_winkel_messung.png</A>. First the modeled structure and the natural structure are fitted, to see how big the differences between those are. If the protein has been disturbed too much, the model is discarded. We control that the alpha helix patterns really generate the expected pattern by measuring the distance between the first and the last amino acids of each of them and checking if it is compatible with an alpha helix. These first and last amino acids of each rods define a vector that is then used to calculate the angles between consecutive helical patterns. The frequency of all the determined lengths and angles is then further analyzed, using a similar strategy to the ArchDB analysis presented above.
</P><P>The whole process for the verification of the different linker patterns was set up on the <A href="https://2014.igem.org/Team:Heidelberg/Software/igemathome" class="external text" rel="nofollow">distributed computing system</A>. Due to lack of time, only few results could be analyzed, resulting in distributions for the different helices, see for example figures 6 and 7.
This lead to a refinement of the length of the 8 different alpha helix blocks presented above to 8.7, 10, 10.8, 15.6, 16.8, 24.8, 32.3 Å.
</P><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Fig. 7. Length distribution of AEAAAKEAAAK</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Fig. 6. Length distribution of AEAAAKA</SPAN></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Conclusion_2">Conclusion</SPAN></H1><P>The patterns that we identified by analyzing structure databases provide an easy and fast tool to build customly shaped peptides. The main achievement is the identification of the angle patterns. These are designed as building blocks for enhanced applicability. The shapes were identified from a database of non-homologous proteins and the patterns were refined until the distribution of surrounding amino acids looked randomly distributed. Thus we can exclude  that the angle distributions we observed is not due to the surrounding sequences, but to the identified patterns. We have not observed any evidence for certain helix patterns being preferred in the database.
From figures 6 and 7 we have learned that the lengths we have assumed for the helices needed to be refined. For example we had assumed the AEAAAKA motif to span a distance of 10.5 Å but have observed it to be only 10 Å long. Our <A href="https://2014.igem.org/Team:Heidelberg/Software/Linker_Software" class="external text" rel="nofollow">CRAUT software</A> was accordingly corrected.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>[1]  Vieille, C. &amp; Zeikus, G.J. Hyperthermophilic enzymes: sources, uses, and molecular mechanisms for thermostability. Microbiology and molecular biology reviews : MMBR 65, 1-43 (2001).
</P><P>[2] Yu, Y. &amp; Lutz, S. Circular permutation: a different way to engineer enzyme structure and function. Trends Biotechnol. 29, 18-25 (2011).
</P><P>[3] Arai, R., Ueda, H., Kitayama, a, Kamiya, N. &amp; Nagamune, T. Design of the linkers which effectively separate domains of a bifunctional fusion protein. Protein Eng. 14, 529-532 (2001).
</P><P>[4] Wang,  C.K.L., Kaas, Q., Chiche, L. &amp; Craik, D.J. CyBase: A database  of  cyclic protein sequences and structures, with applications in protein   discovery and engineering. Nucleic Acids Research 36, (2008).
</P><P>[5] Efimov, a V. Standard structures in proteins. Prog. Biophys. Mol. Biol. 60, 201-2–39 (1993).
</P><P>[6] Donate, L. E., Rufino, S. D., Canard, L.   H. &amp; Blundell, T. L. Conformational analysis and clustering of   short and medium size loops connecting regular secondary structures: a   database for modeling and prediction. Protein Sci. 5, 2600-26–16   (1996).
</P><P>[7] Bonet, J. et al. ArchDB 2014: structural classification of loops in proteins. Nucleic Acids Res. 42, D315-D31–9 (2014).
</P><P>[8] George, R. a &amp; Heringa, J. An analysis of protein domain linkers: their classification and role in protein folding. Protein Eng. 15, 871-879 (2002).
</P><P>[9] Chen, X., Zaro, J. L. &amp; Shen, W.-C. Fusion protein linkers: property, design and functionality. Adv. Drug Deliv. Rev. 65, 1357-1369 (2013).
</P><P>[10] Fiser,  a et al. Modeling of loops in protein structures. Protein science : a publication of the Protein Society 9, 1753-73 (2000).
</P><P>[11] Fiser, A. &amp; Sali, A. ModLoop: Automated modeling of loops in protein structures. Bioinformatics 19, 2500-2501 (2003).
</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Heidelberg/Modeling/Linker_Modeling">http://2014.igem.org/Team:Heidelberg/Modeling/Linker_Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Modeling/Linker_Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Modeling/Linker_Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Modeling/Linker_Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Modeling/Linker_Modeling&amp;oldid=348335" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>