"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/pages/Notebook/Methods
From 2014.igem.org
(Redirected from Team:Heidelberg/pages/MidnightDoc/Methods )
Find here a collection of all the methods we used for cloning our constructs and performing our assays. For the individual experiments visit our Notebook, the MidnightDoc , for the Materials we used, visit Materials .
Contents
1 Make a liquid culture without any plasmid
2 Make a liquid culture carrying three plasmids
3 Streak bacteria without any plasmid onto an agar plate
4 Dilute non-plasmid dsDNA
5 PCA
6 DNA purification (Qiagen PCR Purification Kit)
7 Make a liquid culture carrying a plasmid
8 Dilute plasmid DNA
9 Golden Gate Assembly
10 Religation after Golden gate Assembly
11 Make competent cells already carrying a plasmid
12 Make a glycerol stock of bacteria carrying a plasmid
13 Restriction digest (NEB)
14 Make competent cells not yet carrying any plasmid
15 Pick a colony carrying three plasmids from an agar plate
16 Make a liquid culture carrying two plasmids
17 Gel Electrophoresis
18 PCR from Bacteria (colony PCR or genomic amplification)
19 Miniprep
20 Heat shock plasmid transformation (1 -> 2 plasmid)
21 Plasmid purification (Qiagen PCR Purification Kit)
22 Anneal complementary oligonucleotides
23 Phusion Flash PCR
24 Recovery of plasmid DNA from filter paper
25 Streak bacteria carrying three plasmid onto an agar plate
26 Gel extraction of a plasmid
27 Dissolve plasmid DNA from the registry distribution
28 Gel extraction
29 Heat shock plasmid transformation (2 -> 3 plasmids)
30 Streak bacteria carrying two plasmids onto an agar plate
31 Streak bacteria carrying a plasmid onto an agar plate
32 Heat shock plasmid transformation (0 -> 1 plasmid)
33 Pick a colony carrying a plasmid from an agar plate
34 Make a glycerol stock of bacteria carrying two plasmid
35 Make Medium
36 Plasmid ligation with T4 DNA Ligase (NEB)
37 Gather bacteria carrying a plasmid from an agar plate
38 Heat shock plasmid transformation (0 -> 2 plasmids)
39 Heat shock plasmid transformation (1 -> 3 plasmids)
40 Make a glycerol stock of bacteria without any plasmid
41 Gather bacteria carrying two plasmids from an agar plate
42 Expression of Dnmt1 (linear)
43 Make competent cells already carrying two plasmid
44 Pick a colony without any plasmid from an agar plate
45 Pick a colony carrying two plasmids from an agar plate
46 Make a glycerol stock of bacteria carrying three plasmids
47 CPEC
48 Touchdown-Twostep PCR (Phusion Flash)
49 Lysozyme Assay
Make a liquid culture without any plasmid
Inoculate medium
Grow at the preferred temperature of the organism.
Make a liquid culture carrying three plasmids
Inoculate medium
Grow at the preferred temperature of the organism.
Streak bacteria without any plasmid onto an agar plate
Streak the bacteria onto the agar plate
Dilute non-plasmid dsDNA
Dilute the DNA.
PCA
For a 20 µl reaction, mix equimolar amounts of the DNA fragments (10 ng to 15 ng in total) and add water to a volume of 10 µl.
Add 10 µl 2X Phusion Flash Master Mix.
Place the reaction in a thermocycler and run the following program: Initial denaturation at 98 °C for 10 s; 10 cycles of 1 s denaturation at 98 °C, 5 s annealing at 55 °C, extension at 72 °C for 15 to 30 s/kb of the product; final extension at 72 °C for 5 min.
Add 5 µl of the reaction to 25 µl Phusion Flash Master mix. Add primers to a final concentration of 0.5 µM and water to a final volume of 50 µl.
Place the reaction in a thermocycler and run the following program: Initial denaturation at 98 °C for 10 s; 20 to 30 cycles of 1 s denaturation at 98 °C, 5 s annealing, extension at 72 °C for 15 to 30 s/kb of the product; final extension at 72 °C for 5 min.
DNA purification (Qiagen PCR Purification Kit)
Follow the Qiagen protocol.
Measure the concentration with a NanoDrop.
Make a liquid culture carrying a plasmid
Inoculate medium
Grow at the preferred temperature of the organism.
Dilute plasmid DNA
Dilute the plasmid DNA.
Golden Gate Assembly
Mix 150 ng of the backbone and equimolar amounts of the insert(s) in water in a PCR tube for a total volume of 15 µl.
Add 1.5 µl of 10X T4 Ligase Buffer and, when using BsaI or another restriction enzyme that requires it, BSA (Bovine Serum Albumin) at a final concentration of 1X.
Add 1 µl of each restriction enzyme and 1 µl of T4 DNA Ligase (400,000 cohesive end ligation units/ml).
Optional: Add 1 µl of T4 Polynucleotide Kinase, if several inserts without 5’-phosphorylation (like annealed oligos) are used.
Place the reaction in a thermocycler and run the following program: 25 cycles of 4 min ligation at 16 °C and 3 min restriction at 37°C, followed by 5 min at 50 °C for final restriction and 5 min at 80 °C for heat inactivation.
Religation after Golden gate Assembly
Important: This will have a drastic effect on the efficiency, since the original backbone will be able to reassemble. If you don’t use a selection marker, a crazy colony PCR screening will be necessary to find the desired plasmid.
Add 12.5 µl of water, 1.5 µl of T4 Ligase Buffer and 1 µl of T4 DNA Ligase to your Golden Gate reaction.
Incubate for 20 min at 16 °C, then heat inactivate for 10 min at 65 °C.
Make competent cells already carrying a plasmid
Do some stuff
Make a glycerol stock of bacteria carrying a plasmid
Mix an aliquot of a LB culture with the same volume of storage buffer
Freeze at -80 °C
Restriction digest (NEB)
Mix purified DNA with water
Add buffer
Add restriction enzyme(s)
Incubate
Optional heat inactivation
Make competent cells not yet carrying any plasmid
Do some stuff
Pick a colony carrying three plasmids from an agar plate
Pick up a sterile tip with a pipette. Dip it into a single colony so it attaches to the tip without applying suction.
Make a liquid culture carrying two plasmids
Inoculate medium
Grow at the preferred temperature of the organism.
Gel Electrophoresis
Measure out 7.5 g of agarose.
Pour agarose powder into microwavable flask along with 500mL of 1xTAE.
Microwave for 1-3min (until the agarose is completely dissolved and there is a nice rolling boil).
Let agarose solution cool down for 5min.
Add 10 µl of 10 mg/l ethidium bromide (EtBr) stock solution (final concentration 0.2 mg/l). EtBr binds to the DNA and allows you to visualize the DNA under ultraviolet (UV) light.
Pour the agarose into a gel tray with the well comb in place.
Place newly poured gel at 4°C for 10-15 minutes OR let sit at room temperature for 20-30 minutes, until it has completely solidified.
Add loading buffer to each of your digest samples.
Once solidified, place the agarose gel into the gel box (electrophoresis unit).
Fill gel box with 1xTAE until the gel is covered.
Carefully load your samples into the additional wells of the gel (recommended: 15 µl).
Run the gel at 80-150 V until the dye line is approximately 75-80 % of the way down the gel.
Turn OFF power, disconnect the electrodes from the power source, and then carefully remove the gel from the gel box.
Using any device that has UV light, visualize your DNA fragments.
PCR from Bacteria (colony PCR or genomic amplification)
Suspend the picked colony in 10 µl to 20 µl of sterile H2O (or use inoculated medium).
Mix 12.5 µl OneTaq 2X Master Mix (NEB) with 1 µl of the bacterial suspension.
Add primers to a final concentration of 0.5 µM.
Add H2O to a final volume of 25 µl.
Run your thermocycler.
Miniprep
Follow the Qiagen protocol
Measure the concentration on a NanoDrop
Heat shock plasmid transformation (1 -> 2 plasmid)
Set a heat block to 42 °C.
Thaw an aliquot chemically competent bacteria on ice.
Add plasmid DNA to the aliquot and mix gently by flicking the tube.
Incubate the bacteria on ice for 10 min to 30 min.
Heat shock the bacteria for exactly 30 s (20 s for E. coli BL21(DE3)).
Incubate the bacteria on ice for approx. 5 min.
Add 1 ml of SOC or LB medium and recover the plasmids for 1 h at 37 °C while shaking (optional if the new plasmid only adds an Ampicillin resistance).
Optionally spin down the recovered suspension and resuspend it in a minimal volume of medium.
Plate the suspension on agar supplemented with the appropriate antibiotics (fill in the appropriate protocol here!).
Plasmid purification (Qiagen PCR Purification Kit)
Follow the Qiagen protocol.
Measure the concentration with a NanoDrop.
Anneal complementary oligonucleotides
Mix equal amounts of the oligonucleotides in an appropriate buffer.
Heat the mixture to 95 °C, then slowly cool it down to room temperature over approx. 45 min (by switching of the heat block or using a ramp-cool method in a thermocycler).
Phusion Flash PCR
Add H2O to a final volume of 50 µl.
Add 25 µl 2X Phusion Flash Master Mix.
Add primers to a final concentration of 0.5 µM.
Add 2.5 pg to 25 ng template DNA.
Run your thermocycler.
Recovery of plasmid DNA from filter paper
Use clean gloves and cut the marked circle area that contains the dried plasmid DNA.
Using clean forceps, insert the filter paper into a 1.5 ml micro centrifuge tube. Add 50 µl of pure water (or until the paper is completely soaked), vortex briefly and incubate at room temperature for 5 minutes. Vortex again and centrifuge the tube for a few seconds.
Remove about 5 µl of supernatant for use in transfecting E. coli by electroporation or chemical means. Please do not try to use the DNA directly for any application other than to transform bacteria and prepare a plasmid stock.
Store the remainder of the filter paper/water mix at -20 or -80 °C as a permanent archive in case that your plasmid stock ever gets lost or if something turns out to be wrong with it.
Streak bacteria carrying three plasmid onto an agar plate
Streak the bacteria onto the agar plate or use sterile glass beads.
Gel extraction of a plasmid
Using a clean scalpel, excise the desired band(s) from the agarose gel.
Follow the Qiagen protocol.
Mesaure the concentration on a NanoDrop.
Dissolve plasmid DNA from the registry distribution
Add 10 µl H2O to the well of the registry distribution if it was not previously used.
Use 1-2 µl for transformation.
Gel extraction
Using a clean scalpel, excise the desired band(s) from the agarose gel.
Follow the Qiagen protocol.
Mesaure the concentration on a NanoDrop.
Heat shock plasmid transformation (2 -> 3 plasmids)
Set a heat block to 42 °C.
Thaw an aliquot chemically competent bacteria on ice.
Add plasmid DNA to the aliquot and mix gently by flicking the tube.
Incubate the bacteria on ice for 10 min to 30 min.
Heat shock the bacteria for exactly 30 s (20 s for E. coli BL21(DE3)).
Incubate the bacteria on ice for approx. 5 min.
Add 1 ml of SOC or LB medium and recover the plasmids for 1 h at 37 °C while shaking (optional if the new plasmid only adds an Ampicillin resistance).
Optionally spin down the recovered suspension and resuspend it in a minimal volume of medium.
Plate the suspension on agar supplemented with the appropriate antibiotics (fill in the appropriate protocol here!).
Streak bacteria carrying two plasmids onto an agar plate
Streak the bacteria onto the agar plate or use sterile glass beads.
Streak bacteria carrying a plasmid onto an agar plate
Streak the bacteria onto the agar plate or use sterile glass beads.
Heat shock plasmid transformation (0 -> 1 plasmid)
Set a heat block to 42 °C.
Thaw an aliquot chemically competent bacteria on ice.
Add plasmid DNA to the aliquot and mix gently by flicking the tube.
Incubate the bacteria on ice for 10 min to 30 min.
Heat shock the bacteria for exactly 30 s (20 s for E. coli BL21(DE3)).
Incubate the bacteria on ice for approx. 5 min.
Add 1 ml of SOC or LB medium and recover the plasmids for 1 h at 37 °C while shaking (optional if the new plasmid only adds an Ampicillin resistance).
Optionally spin down the recovered suspension and resuspend it in a minimal volume of medium.
Plate the suspension on agar supplemented with the appropriate antibiotics (fill in the appropriate protocol here!).
Pick a colony carrying a plasmid from an agar plate
Pick up a sterile tip with a pipette. Dip it into a single colony so it attaches to the tip without applying suction.
Resuspend the colony in sterile medium or water.
Make a glycerol stock of bacteria carrying two plasmid
Mix an aliquot of a LB culture with the same volume of storage buffer.
Freeze at -80 °C.
Make Medium
Put appropriate amount of Mediums powder, put it into correct bottle, label it correctly and put in the correct box for autoclavation
Plasmid ligation with T4 DNA Ligase (NEB)
Mix water and T4 DNA Ligase Buffer (NEB).
Add DNA.
Incubate at room temperature.
Gather bacteria carrying a plasmid from an agar plate
Using a sterile item, gather some bacteria from the plate.
Heat shock plasmid transformation (0 -> 2 plasmids)
Set a heat block to 42 °C.
Thaw an aliquot chemically competent bacteria on ice.
Add plasmid DNA to the aliquot and mix gently by flicking the tube.
Incubate the bacteria on ice for 10 min to 30 min.
Heat shock the bacteria for exactly 30 s (20 s for E. coli BL21(DE3)).
Incubate the bacteria on ice for approx. 5 min.
Add 1 ml of SOC or LB medium and recover the plasmids for 1 h at 37 °C while shaking.
Optionally spin down the recovered suspension and resuspend it in a minimal volume of medium.
Plate the suspension on agar supplemented with the appropriate antibiotics (fill in the appropriate protocol here!).
Heat shock plasmid transformation (1 -> 3 plasmids)
Set a heat block to 42 °C.
Thaw an aliquot chemically competent bacteria on ice.
Add plasmid DNA to the aliquot and mix gently by flicking the tube.
Incubate the bacteria on ice for 10 min to 30 min.
Heat shock the bacteria for exactly 30 s (20 s for E. coli BL21(DE3)).
Incubate the bacteria on ice for approx. 5 min.
Add 1 ml of SOC or LB medium and recover the plasmids for 1 h at 37 °C while shaking.
Optionally spin down the recovered suspension and resuspend it in a minimal volume of medium.
Plate the suspension on agar supplemented with the appropriate antibiotics (fill in the appropriate protocol here!).
Make a glycerol stock of bacteria without any plasmid
Mix an aliquot of a LB culture with the same volume of storage buffer.
Freeze at -80 °C.
Gather bacteria carrying two plasmids from an agar plate
Using a sterile item, gather some bacteria from the plate.
Expression of Dnmt1 (linear)
Transform the plasmid harboring DNMT1 (Kanamycin resistant) into Rosetta DE3 competent cells (Chloramphenicol resistant) ( negative control untransformed)
Inoculate a few colonies to 40 ml LB+Kan+Cm medium to grow the cells overnight at 37 °C (Prepare 4l LB+Kan+Cm medium to autoclave)
Dispense 10 ml of the growth culture into each 1 l flask and grow the cells at 37 °C. When the cell OD reaches ~0.6, lower the temperature to 15-20 °C
30 min later, add IPTG (0.4 mM final concentration) and ZnCl2 (0.1 mM final concentration) into the media and continue to grow the cells overnight.
Spin down the cells at 4000 rpm for 20 min (4 °C, 5th or 6th floor, fill 500 ml buckets for Ja10 rotor to 375 ml), and resuspend the cell paste in HisTrap loading buffer, with approximately 20 ml buffer for each liter of cell paste. (Take sample S1, accordingly C1 from untransformed control)
SONICATION of the cells (Lohmann group, cold room 4th floor) while keeping them on ice, disrupt the cells using sonication. To avoid the protein degradation and aggregation, add PMSF and DTT to 1 μM and 0.5 mM respectively.
Place the sonication tip in the sample, close to the bottom, but not touching the tube. Select the right program: Microtip: yes; Amplitude 60; Working time 2 min; On 0.02; Off 0.05 After 5 minutes, check if the sample did not heat up and repeat the program one more time. Clean the tip with some ethanol after use.
After the cell is lysed, spin down the cell lysate using high speed or ultracentrifuge (17000 rpm x 45 min, Ti70 rotor, thick wall tubes [355631], filled to 15 ml)
Collect the supernatant. If any aggregates or particles are visible, filter the supernatant with a 0.2 µm filter using a syringe. (Take sample from supernatant S2 and pellet S3, accordingly C2 and C3 from untransformed control)
Load the supernatant to pre-equilibrated HisTrap column, wash the column using 200 ml of chilled His trap loading buffer, and elute the fusion protein using elution buffer.
Collect the eluent and dialyze the protein sample against dialysis buffer overnight at 4C. > Optional: adding protease inhibitor cocktail tablets will help DNMT1 from proteolytic degradation.
Take out sample from dialysis tube, spin down the sample and remove any precipitate
Load the supernatant to the Heparin column, then apply the salt gradient with Buffer A and B. The target protein is expected to elute at 0.4-0.5 M NaCl.
Collect the fractions and run the SDS Page to examine the protein (~100 kD). Pool together the protein fraction with significant purity
Concentrate the fractions to about 1 ml using concentration spin columns (25MWCO)
The protein is finally purified through a 16/60 Superdex 200 size exclusion column using Gel filtration buffer
After the gel filtration, the protein will be examined by SDS page. The fractions with >90% purity will be pooled
Concentration of sample and storage at -80C freezer.
Make competent cells already carrying two plasmid
Do some stuff
Pick a colony without any plasmid from an agar plate
Pick a single colony from the agar plate using a sterile pipette tip.
Resuspend the colony in appropriate medium to make a culture or in sterile water.
Pick a colony carrying two plasmids from an agar plate
Pick up a sterile tip with a pipette. Dip it into a single colony so it attaches to the tip without applying suction.
Resuspend the colony in sterile medium or water.
Make a glycerol stock of bacteria carrying three plasmids
Mix an aliquot of a LB culture with the same volume of storage buffer.
Freeze at -80 °C.
CPEC
Design of the insert and backbone primers
Assemble your desired construct in silico by pasting the fragments. Feel free to add tags and other small stuff between them.
Define a “homology region” with a Tm of 60 °C - 70 °C (~15 - 35 bp) at each fragment border. The Tms of all homology regions should be very similar. It is your choice whether the regions are right next to the border in one fragment, include parts of both fragments, include inserted sequences. These regions should be checked carefully for annoying secondary structures.
You need a forward and a reverse primer for any fragment. Any primer consists of a part that binds to the template DNA and an overlap. The binding part should have a G or C at the 3’ end and a Tm between 60 °C and 72 °C. The binding part Tms of one primer pair should be similar. In addition to the binding part, any primer must have an overlap part. Just extend the primers until the chosen homology region is fully covered. Check the complete primers for secondary structures. If there are really bad ones, you should change the homology region…
PCRs of the inserts
Please follow the Touchdown-Twostep PCR protocol , purify the products and measure their concentration.
The CPEC
Mix your fragments. At least 100 ng vector are required, we recommend to use around 500 ng/µl. Add equimolar amounts of your insert(s). A vector: insert ratio of 1:3 works as well.
Add 10 µl 2X Phusion Flash Master Mix.
Add H2O to a final volume of 20 µl.
Run your thermocycler:
Step
Temperature
Time
Initial denaturation
98 °C
10 s
1 insert: 10 cycles, >1 insert: 20-30 cycles
- Denaturation
98 °C
1 s
- Annealing
Slow ramping from 70°C to 55°C (-0.1 °C/ s)
~3 min
- Elongation
72 °C
15 s / kb (desired plasmid)
End elongation
72 °C
120 s
Touchdown-Twostep PCR (Phusion Flash)
Add H2O to a final volume of 50 µl.
Add 25 µl 2X Phusion Flash Master Mix.
Add primers to a final concentration of 0.5 µM.
Add 2.5 pg to 25 ng template DNA.
Run your thermocycler:
Step
Temperature
Time
Initial denaturation
98 °C
10 s
Touchdown (~12 cycles)
- Denaturation
98 °C
1 s
- Annealing
Primer Tm + 4 °C, -0.5 °C each cycle
5 s
- Elongation
72 °C
15 s / kb
Twostep (~24 cycles)
- Denaturation
98 °C
1 s
- Elongation
72 °C
5 s + 15 s /kb
End elongation
72 °C
120 s
Lysozyme Assay
Bacteria were transformed with lambda lysozyme constructs, grown to an OD of 0.6 and induced with 1 mM IPTG.
After 4 hours of expression, the samples were diluted to the same OD of 2.0 and centrifuged down at 2,850 rpm. The pellet was resuspended in 10 mM potassium phosphate buffer with a pH of 6.24 and sonicated for two minutes on ice. After centrifugation a second time at 2,850 rpm, the supernatant containing the lambda lysozyme was kept.
For substrate preparation, the lyophilized cells of M.lysodeikticus (Sigma Aldrich), were resuspended in ultrapure water. The supernantant with the protein mix was transfered and aliquoted for biological replicates and to prepare dilution series.
Subsequently the samples were transfered into the thermocycler to do a one minute heat-shock in a temperature span between 45 and 55°C. After mixing enzyme and substrate the OD was measured every two minutes in a plate reader over 100 minutes at 37°C (figure 4). === Flourescence labeled peptidoglycane assay The expression of the lysozyme samples for the assay using FITC-labeled peptidoglycan was carried out in the same way as we did it in the first assay. We resuspended the pellet in PBS. The protocol of labeling of peptidoglycan with fluorescein isothiocyanate (FITC) you can find in this methods section as well.
Retrieved from " http://2014.igem.org/Team:Heidelberg/pages/Notebook/Methods "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
