<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_Toolbox"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Project/Toolbox">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project/Toolbox&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Toolbox&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Toolbox&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project/Toolbox" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project/Toolbox</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="container" style="padding:0;background-color:white;"><HEADER id="navbar" role="banner" class="navbar navbar-static-top navbar-white"><DIV class="container" style=" background-color:black;"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse collapse" style="margin-bottom:10px;"><NAV role="navigation"><UL class="menu nav navbar-nav navbar-right"><LI class="dropdown"><A href="/Team:Heidelberg/Team" class="dropdown-toggle">Team</A></LI><LI><A href="/Team:Heidelberg/Team/Members">Members</A></LI><LI><A href="/Team:Heidelberg/Team/Collaborations">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsoring">Sponsoring</A></LI></UL><LI class="dropdown"><A href="/Team:Heidelberg/Project" class="dropdown-toggle">Project</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Background">Background</A></LI><LI><A href="/Team:Heidelberg/Project/Toolbox">Toolbox</A></LI><LI><A href="/Team:Heidelberg/Toolbox/Circularization">Toolbox: Circularization</A></LI><LI><A href="/Team:Heidelberg/Project/PCR_2.0">PCR 2.0</A></LI><LI><A href="/Team:Heidelberg/Project/Xylanase">Xylanase</A></LI><LI><A href="/Team:Heidelberg/Project/Linker_Screening">Linker Screening</A></LI><LI><A href="/Team:Heidelberg/Project/Reconstitution">Fluorescence Assembly</A></LI><LI><A href="/Team:Heidelberg/Project/LOV">Light-Induction</A></LI><LI><A href="/Team:Heidelberg/Project#Achievements">Achievements</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Parts" class="dropdown-toggle">Parts</A></LI><LI><A href="/Team:Heidelberg/Parts/RFC">RFC Heidelberg 2014</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Favorite Parts">Favorite Parts</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Sample Data Page">Sample Data Page</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#allParts">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Backbones">Backbones</A></LI><LI><A href="https://2014.igem.org/Team:Heidelberg/Parts#Intein Library">Intein Library</A></LI><LI><A href="/Team:Heidelberg/Parts/Part_Improvement">Part Improvement</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Software" class="dropdown-toggle">Software</A></LI><LI><A href="/Team:Heidelberg/Software/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Software/Linker_Software">Linker Software</A></LI><LI><A href="/Team:Heidelberg/Software/MidnightDoc">MidnightDoc</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Modeling" class="dropdown-toggle">Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Linker_Modeling">Linker Modeling</A></LI><LI><A href="/Team:Heidelberg/Modeling/Enzyme_Modeling">Enzyme Kinetics Modeling</A></LI><LI><A href="/Team:Heidelberg/Toolbox_Guide" class="dropdown-toggle">Toolbox Guide</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Human_Practice" class="dropdown-toggle">Human Practice</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/igemathome">iGEM@home</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Ethics">Religion &amp; Synthetic Biology</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Education">Education</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Public_Relations">Public Relations</A></LI><LI><A href="/Team:Heidelberg/Human_Practice/Experts">Experts</A></LI><LI class="dropdown"><A href="/Team:Heidelberg/Notebook" class="dropdown-toggle">Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook">Notebook</A></LI><LI><A href="/Team:Heidelberg/AwesomeSheet">Database for Notebook</A></LI><LI><A href="/Team:Heidelberg/Notebook/Materials">Materials</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Notebook/Safety">Safety</A></LI></NAV></DIV><DIV class="col-lg-12"><DIV class="title-wrapper"><SPAN class="title">TOOLBOX</SPAN></DIV></DIV></DIV></HEADER><DIV class="container main"><DIV class="row" style="position:relative;"><DIV class="col-lg-offset-1 col-lg-11 subtitle"><H2><P>– giving YOU the full power of inteins</P></H2></DIV></DIV><DIV class="row"><DIV class="col-lg-12"><P>A primary mission of the iGEM team Heidelberg 2014 is to provide a new toolbox based on appropriate standards that introduces the highly functional <A href="https://2014.igem.org/Team:Heidelberg/Project/Background" class="external text" rel="nofollow">INTEINS</A> to the parts registry and thereby paving the way to a new level of posttranslational modification possibilities for the iGEM community. Inteins are self-excising peptide segments that mediate the reaction of protein trans-splicing. They don't require any additional energy sources to perform the splicing reaction. Another valuable feature of inteins is their specific recognition, also in the presence of other proteins, which makes them functional in vitro as well as in vivo. On this page we describe experimental approaches to provide and evaluate five related but still quite different applications using inteins and their splicing capabilities. Thereby we present scientific prove of the diversity and efficiency of an intein toolbox that can be used for a wide range of posttranslational modifications of proteins. Explore the new possibilities, intein usage provides! On this page we describe the scientific work in form of modeling and experiments that lead to the establishment of the first intein toolbox in the history of iGEM! 
</P><P><I>Jump right into splicing and use the <A href="https://2014.igem.org/Team:Heidelberg/Toolbox_Guide" class="external text" rel="nofollow">toolbox guide</A> to design your intein-equipped proteins of choice!</I></P><DIV class="row toolbox-icons"><DIV class="col-lg-2 col-lg-offset-1 col-md-offset-1 col-md-2 col-sm-4 col-xs-6"><A href="https://2014.igem.org/Team:Heidelberg/Toolbox/Circularization">CIRCULARIZATION</A></DIV><DIV class="col-lg-2 col-md-2 col-sm-4 col-xs-6"><A href="https://2014.igem.org/Team:Heidelberg/Project/Toolbox#Oligomerization">OLIGOMERIZATION</A></DIV><DIV class="col-lg-2 col-md-2 col-sm-4 col-xs-6"><A href="https://2014.igem.org/Team:Heidelberg/Project/Toolbox#Fusion_and_Tagging">FUSING &amp; TAGGING</A></DIV><DIV class="col-lg-2 col-md-2 col-sm-4 col-xs-6"><A href="https://2014.igem.org/Team:Heidelberg/Project/Toolbox#Purification">PURIFICATION</A></DIV><DIV class="col-lg-2 col-md-2 col-sm-4 col-xs-6"><A href="https://2014.igem.org/Team:Heidelberg/Project/Toolbox#On_Off">ON OFF</A></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Circularization"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Circularization</SPAN></A></LI><LI class="toclevel-2"><A href="#Circularization_Constructs"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Circularization Constructs</SPAN></A></LI><LI class="toclevel-2"><A href="#Split_Intein_Circularization"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Split Intein Circularization</SPAN></A></LI><LI class="toclevel-2"><A href="#Sortase_Circularization"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Sortase Circularization</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Oligomerization"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Oligomerization</SPAN></A></LI><LI class="toclevel-2"><A href="#Standardization_of_oligomerization"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Standardization of oligomerization</SPAN></A></LI><LI class="toclevel-2"><A href="#The_mechanism"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">The mechanism</SPAN></A></LI><LI class="toclevel-1"><A href="#Fusion_and_Tagging"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Fusion and Tagging</SPAN></A></LI><LI class="toclevel-2"><A href="#Introduction"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2"><A href="#Standard_Construction"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Standard Construction</SPAN></A></LI><LI class="toclevel-2"><A href="#Posttranslational_Modifications"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Posttranslational Modifications</SPAN></A></LI><LI class="toclevel-3"><A href="#Introduction_2"><SPAN class="tocnumber">3.3.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-3"><A href="#Standard_construction_2"><SPAN class="tocnumber">3.3.2</SPAN><SPAN class="toctext">Standard construction</SPAN></A></LI><LI class="toclevel-2"><A href="#Translocation"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Translocation</SPAN></A></LI><LI class="toclevel-3"><A href="#Introduction_3"><SPAN class="tocnumber">3.4.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-3"><A href="#Standard_construction_3"><SPAN class="tocnumber">3.4.2</SPAN><SPAN class="toctext">Standard construction</SPAN></A></LI><LI class="toclevel-3"><A href="#Results"><SPAN class="tocnumber">3.4.3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1"><A href="#Purification"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Purification</SPAN></A></LI><LI class="toclevel-1"><A href="#On_Off"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">On Off</SPAN></A></LI><LI class="toclevel-2"><A href="#On"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">On</SPAN></A></LI><LI class="toclevel-2"><A href="#Off"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Off</SPAN></A></LI><LI class="toclevel-3"><A href="#Intein_protease_standard_construction"><SPAN class="tocnumber">5.2.1</SPAN><SPAN class="toctext">Intein protease standard construction</SPAN></A></LI><LI class="toclevel-3"><A href="#Results_2"><SPAN class="tocnumber">5.2.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3"><A href="#SsrA_degradation_tag_standard_construction"><SPAN class="tocnumber">5.2.3</SPAN><SPAN class="toctext">SsrA degradation tag standard construction</SPAN></A></LI><LI class="toclevel-3"><A href="#Results_3"><SPAN class="tocnumber">5.2.4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Circularization">Circularization</SPAN></H1><P>Enzymes represent a major tool for many branches of the chemical industry, including food, brewing, paper, detergent or biofuel. Millions of years of evolution have allowed these proteins to performed extremely specific chemical modifications that are not only essential for living organisms but can also be of great benefit to produce useful molecules for our life, efficiently and at low cost. A major limitation of the use of enzyme for industrial application and in general out of their natural environment is their stability. They can be destroyed by other enzymes and they can unfold and take non-functional conformation when exposed to non-physiological temperature and pH. Such limitations has motivated research in species that can grow at extreme temperatures . Another major area of chemical research is the design of strategies to stabilize enzymes, and more generally proteins and peptides. Protein circularization, meaning ligation of the N- and C-terminal ends of a protein, represents a promising way to achieve this stabilization. While conserving the functionality of their linear counterpart, circular proteins can be superior in terms of thermostability, resistance against chemical denaturation and protection from exopeptidases . Moreover, a circular backbone can improve <I>in vivo</I> stability of therapeutical proteins and peptides. All these remarkable properties motivated us to develop new tools to circularize any protein of interest. 
Our <A href="/Team:Heidelberg/Toolbox_Guide" title="Team:Heidelberg/Toolbox Guide"> Toolbox Guide</A> provides a step-by-step strategy to clone a circularization linker and express it in <I>E. coli</I>. Moreover, in case of complex structures where the protein extremities are far from each other, we have developed the software tool <A href="/Team:Heidelberg/Software/Linker_Software" title="Team:Heidelberg/Software/Linker Software"> CRAUT</A> that will design the appropriate rigid linkers.
</P><P><I>Please find more information about the circularization of proteins and the theory that laid the foundation for the circularization kit of the toolbox on the <A href="https://2014.igem.org/Team:Heidelberg/Toolbox/Circularization" class="external text" rel="nofollow">Circularization pages</A></I>. 
</P><H2><SPAN class="mw-headline" id="Circularization_Constructs">Circularization Constructs</SPAN></H2><P>The most promising approaches to <A href="/Team:Heidelberg/Toolbox/Circularization" title="Team:Heidelberg/Toolbox/Circularization"> circularize</A> proteins are protein trans-splicing using <A href="/Team:Heidelberg/Project/Background" title="Team:Heidelberg/Project/Background"> split inteins</A> and Sortase A-catalyzed cyclization. Both methods require the addition of specific proteins domains or peptides to the protein to be circularized. Consequently, on DNA level, creating circular proteins is equivalent to creating fusion proteins. However, existing protein fusion standards like <A href="http://parts.igem.org/Help:Standards/Assembly/RFC23" class="external text" rel="nofollow">RFC[23</A>] cause scars. Those scars on protein level may affect protein function and further complicate 3D-structure modeling. 
Therefore, we decided to create the new RFC[i] that allows scarless cloning of inteins. Our intein circularization constructs apply to this standard, while our sortase constructs are closely related and can be used similarly. Detailed instructions on how to use our constructs are provided in our <A href="/Team:Heidelberg/Toolbox_Guide" title="Team:Heidelberg/Toolbox Guide"> Toolbox Guide</A>. 
</P><H2><SPAN class="mw-headline" id="Split_Intein_Circularization">Split Intein Circularization</SPAN></H2><DIV class="pull-right col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 1) <A href="http://parts.igem.org/Part:BBa_K1362000"> BBa_K1362000</A></SPAN><P>NpuDnaE intein RFC [i] circularization construct</P></DIV></DIV></DIV><P>Between the coding sequences of the <I>Npu</I> DnaE C-intein and the N-intein we placed <A href="http://parts.igem.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A>, an mRFP selection marker flanked by BsaI sites that can be replaced by the protein to be circularized. 
</P><P>Exteins, RFC [i] standard overhangs and BsaI sites have to be added to the coding sequence of the protein to be circularized without start- and stop codons by PCR. By Golden Gate assembly, the mRFP selection marker has to be replaced with the protein insert. After addition of an inducible promotor the circular protein is ready to be expressed. For detailed step-by-step instructions please use our <A href="/Team:Heidelberg/Toolbox_Guide" title="Team:Heidelberg/Toolbox Guide">Toolbox Guide</A>. 
</P><P>Upon expression of the fusion protein, the split intein domains reassemble to the active <A href="/Team:Heidelberg/Project/Background" title="Team:Heidelberg/Project/Background">intein</A> and thus ligate the termini of the protein to be circularized in trans-splicing reaction. 
</P><P>These constructs were successfully used to circularize <A href="/Team:Heidelberg/Project/Linker_Screening" title="Team:Heidelberg/Project/Linker Screening">lambda lysozyme</A> and <A href="/Team:Heidelberg/Project/Xylanase" title="Team:Heidelberg/Project/Xylanase">xylanase</A> and probably <A href="/Team:Heidelberg/Project/PCR_2.0" title="Team:Heidelberg/Project/PCR 2.0">DNMT1</A>.
</P><H2><SPAN class="mw-headline" id="Sortase_Circularization">Sortase Circularization</SPAN></H2><DIV class="pull-right col-lg-12 col-md-12 col-sm-12 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Figure 2) <A href="http://parts.igem.org/Part:BBa_K1362002"> BBa_K1362002</A></SPAN><P>Sortase A circularization construct (with His6)</P></DIV></DIV></DIV><P>An mRFP selection marker <A href="http://parts.igem.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A>, which can be removed by restriction with BsaI, is flanked by TEV protease cleavage site (left) and a sorting signal (right) with a His6 tag. 
</P><P>BsaI sites and overhangs corresponding to the sortase A circularization construct have to be added to the coding sequence of the protein to be circularized without start- and stop codons by PCR. By Golden Gate assembly, the mRFP selection marker has to be replaced with the protein insert. After addition of an inducible promotor protein is ready to be expressed. The protein has to be purified and treated with TEV protease  and sortase A. The sortase A catalyzes the transpeptidation reaction that leads to backbone circularization. 
</P><P><I>Please find more information about our the toolkit for cirularization on the <A href="https://2014.igem.org/Team:Heidelberg/Toolbox/Circularization_Constructs" class="external text" rel="nofollow">Circularization Construct pages</A></I>. 
</P><H1><SPAN class="mw-headline" id="Oligomerization">Oligomerization</SPAN></H1><P>Split inteins constitute a useful tool to produce huge polymers in vivo: Hauptmann et al. managed to fabricate synthetic spider silk with microfiber structure. The results using an easy-to-handle split intein system were stunning: The polymers had a molecular weight of 250 kDa and more <A href="#References">[7]</A>. Further application of oligomerization by inteins includes the posttranslational complexation of multi-domain proteins, after their domains have been expressed individually. This approach is very valuable considering great difficulities of expressing large eukaryotic proteins in <I>E.coli</I>.
</P><H4><SPAN class="mw-headline" id="Standardization_of_oligomerization"> Standardization of oligomerization </SPAN></H4><P>The valuable properties of spider silk, for example its exceptional strength and elasticity, result from numerous repeats of certain protein motifs. Convenitonal methods to multimerize these motifs bear a lot of difficulties: Often genetic and mRNA instability constitute a barrier for the production of multimers as fusion proteins <A href="#References">[1]</A>. Posttranslational assembly through split inteins is therefore the solution to overcome these problems. The successfull polimerization of spider silk potein motifs demonstrates the potential of split inteins to be a useful tool for the production of new biomaterials by performing oligomerization reactions with split inteins. The iGEM team Heidelberg standardized (lik to toolbox guide) the oligomerization procedure with split inteins to allow easy handling with different proteins.
</P><P>The use of non-orthogonal split inteins can further be exploited to direct the oligomerisation of several protein domains at once.
</P><H4><SPAN class="mw-headline" id="The_mechanism"> The mechanism </SPAN></H4><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Oligomerization with inteins.</SPAN></DIV></DIV></DIV>

Oligomerization reactions require the same constructs as the ones used for protein circularization.
<P>Circularization is achieved by bringing the N and C terminus of a protein very close together, so both intein parts can asseble, cut out off the protein and thereby circularize it. In contrast, oligomerization occurs when both termini of a protein cannot reach each other and the intein parts of two neighbouring proteins assemble. 
</P><H1><SPAN class="mw-headline" id="Fusion_and_Tagging">Fusion and Tagging</SPAN></H1><H4><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H4><P>Post-translational modifications are present in nature in great numbers.Synthetic Biology, however, has not yet made use of the innumerable possibilities nature has developed. With our collection of intein assembly constructs we expand the arsenal of synthetic biology by enabling unlimited changes of a protein's amino acid sequence even after translation. 
</P><H4><SPAN class="mw-headline" id="Standard_Construction">Standard Construction</SPAN></H4><P>To be able <B>to fuse</B> any two halves of a protein together can have many different uses. We therefore saw the need for a standardised construct, the intein assembly part. This BioBrick part allows the user to clone two DNA-sequences coding for two parts of a peptide into a plasmid prepared with selection markers and standardised overhangs. Those parts were all send in with additional hexahistidine-tags to enable quick analysis on a western blot, however there are highly customisable parts available as well.Visit our <A href="https://2014.igem.org/Team:Heidelberg/Parts#allParts" class="external text" rel="nofollow">parts page</A> to get an overview of our assembly constructs.
In an extensive assay we proved the principle behind split protein assembly by showing that GFP can be artificially split into two halves and thereafter be reassembled so the fluorescence is restored. Visit the <A href="https://2014.igem.org/Team:Heidelberg/Project/Reconstitution" class="external text" rel="nofollow">split Fluorescent Protein</A>.
</P><DIV class="pull-left col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">N-intein fusion construct.</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">C-intein fusion construct.</SPAN></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Posttranslational_Modifications"> Posttranslational Modifications </SPAN></H3><H4><SPAN class="mw-headline" id="Introduction_2">Introduction</SPAN></H4><P>Posttranslational modifications are highly prevalent in nature: Almost every protein in a cell is modified after having been translated, adding numerous varieties of the protein to the mere protein backbone.
Synthetic Biology, however, can expand the possibilities offered by nature and introduce synthetic posttranslational modifications or attachments, such as biophysical probes. As the two parts of a split intein assemble in a highly specific manner, the modifications are introduced controllable a certain locus.
There are different publications on intein-based introduction of posttranslational modifications, including phosphorylation, lipidation, glycosylation, acetylation and ubiquitination <A href="#References">[4]</A>.
Phosphorylation, for example has been applied with tyrosine kinase C-terminal Src kinase (Csk) in order to be able to study the structure and function of this specifically modified protein <A href="#References">[5]</A>.
</P><H4><SPAN class="mw-headline" id="Standard_construction_2">Standard construction</SPAN></H4><P>Applying the intein assembly constructs the iGEM team Heidelberg provides a tool for all kinds of natural as well as synthetic posttranslational modifications by chemoselective addition of a peptide to a recombinant protein.
The principle is based on intein-mediated protein fusion using the <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1362141" class="external text" rel="nofollow">SspDnaB</A> split intein for N-terminal modifications and SspDnaX-S11 for C-terminal ones. The split SspDnaB intein has a very short N-terminal part, consisting of only 11 amino acids intein and 5 amino acids extein sequence and a much longer C-terminal part. By contrast, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1362110" class="external text" rel="nofollow">SspDnaX-S11</A> has a C-terminal part consisting of only 6 amino acids intein and 3 amino acids extein sequence, but a much longer N-terminal part.
The short part including the desired modification is easy to obtain by chemical synthesis. This  offers the possibility to introduce this modification at a specific locus.
</P><P><I>In general, split inteins are powerful tools to easily introduce all kinds of posttranslational modifications in a highly chemoselective manner.</I>
Use our <A href="https://2014.igem.org/Team:Heidelberg/Toolbox_Guide" class="external text" rel="nofollow">toolbox guide</A> to attach posttranslational modifications and attachments to your protein of interest!
</P><H3><SPAN class="mw-headline" id="Translocation">Translocation</SPAN></H3><H4><SPAN class="mw-headline" id="Introduction_3">Introduction</SPAN></H4><P>Adding and removal of translocation tags is one application example for split intein-mediated fusion of two protein domains. Translocation tags offer the possibility to transfer proteins to a certain locus inside the cell by attaching a short tag sequence to the terminus of one's protein of interest. Expressed fused to the protein, which is the conventional way, such a tag is neither removable nor attachable to a protein at a certain time point. Usage of split inteins for tagging offers new dimensions of mobility and control: Tags can be attached (using fusion constructs) or removed (using the intein protease) posttranslationally at a specific time point.
</P><H4><SPAN class="mw-headline" id="Standard_construction_3">Standard construction</SPAN></H4><P>Regulated targeting of proteins to certain loci inside the cell can be achieved using the iGEM Heidelberg 2014 translocation tags for either E. coli or eucaryotic expression systems. Many of them are based on parts created by previous iGEM teams, but with split intein usage now can be applied in much diverser ways.
We provide following tags:
</P><TABLE class="table table-hover"><CAPTION><B>Our toolbox provides the following tags:</B></CAPTION><TBODY><TR><TD> MinD membrane targeting sequence: <A href="http://parts.igem.org/Part:BBa_K1362055" class="external text" rel="nofollow">BBa_K1362055</A></TD><TD> → membrane localization (E. coli)
</TD></TR><TR><TD> OmpA:  <A href="http://parts.igem.org/Part:BBa_K1362060" class="external text" rel="nofollow">BBa_K1362060</A></TD><TD> → surface display (E. coli)
</TD></TR><TR><TD> pelB leader sequence: <A href="http://parts.igem.org/Part:BBa_K1362058" class="external text" rel="nofollow">BBa_K1362058</A></TD><TD> → periplasmic localization (E. coli)
</TD></TR><TR><TD> myristooilation signal sequence: <A href="http://parts.igem.org/Part:BBa_K1362056" class="external text" rel="nofollow">BBa_K1362056</A></TD><TD> → membrane localization (eucaryotic)
</TD></TR><TR><TD> PKI-NES: <A href="http://parts.igem.org/Part:BBa_K1362059" class="external text" rel="nofollow">BBa_K1362059</A></TD><TD> → cytoplasmic localization (eucaryotic)
</TD></TR><TR><TD> NLS: <A href="http://parts.igem.org/Part:BBa_K1362057" class="external text" rel="nofollow">BBa_K1362057</A></TD><TD> → nuclear localization (eucaryotic)
</TD></TR></TBODY></TABLE><P><I>Use our <A href="https://2014.igem.org/Team:Heidelberg/Toolbox_Guide" class="external text" rel="nofollow">toolbox guide</A> to post-translationally assemble proteins, attach tags or change the location of your protein!</I></P><P>All translocation tags except from the OmpA surface protein tag were ordered as DNA oligos containing the tag sequence, extein sequence and BsaI oberhangs. OmpA surface protein was obtained from the Spring Distribution 2012 and also PCR amplified with overhangs. By Golden Gate Assembly the tags were cloned into the carrier constructs BBa_K1362052 (for C-terminal tags) and BBa_K1362053 (for N-terminal tags). These carrier constructs, pSB1C3 backbones with two BsaI sites as an insert, were used to send the tags to the registry of standard biological parts. The user can afterwards easily use Golden Gate Assembly to clone the tags from the carrier constructs into intein assembly constructs.
</P><H4><SPAN class="mw-headline" id="Results">Results</SPAN></H4><P>The parts were sequenced, the results were positive. You can obtian them here:
</P><P><A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/8/8d/Heidelberg_orig_MTS.png" target="_blank">MinD MTS:</A>BBa_K1362055 
</P><P><A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/4/48/Heidelberg_orig_pelB.png" target="_blank">pelB leader sequence:</A>  BBa_K1362058
</P><P><A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/d/de/Heidelberg_orig_myristoilation.png" target="_blank">myristoilation signal sequence:</A>  BBa_K1362056
</P><P><A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/5/57/Heidelberg_orig_PKI_NES.png" target="_blank">PKI-NES:</A>  BBa_K1362059
</P><P><A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/5/5c/Heidelberg_orig_NLS.png" target="_blank">NLS:</A>  BBa_K1362057
</P><P>OmpA (<A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/d/d7/Heidelberg_orig_ompa-1.png" target="_blank">part1</A>, <A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/0/02/Heidelberg_orig_ompa1.png" target="_blank">part2</A> , <A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/1/1b/Heidelberg_orig_ompa2.png" target="_blank">part3</A> , <A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/b/bd/Heidelberg_orig_ompa3.png" target="_blank">part4</A> ): BBa_K1362060

    </P><H1><SPAN class="mw-headline" id="Purification">Purification</SPAN></H1><P>Therapeutic proteins have changed modern medicine. For instance, the monoclonal antibody Trastuzumab, is used to treat breast cancers that are positive for the epidermal growth factor HER2/neu (Ref. Clinical outcome in women with HER2-positive de novo or recurring stage IV breast cancer receiving trastuzumab-based therapy). 
For application in medicine proteins have to be of very high purity. However, purification at industrial scale remains time consuming and cost intensive. The purification procedure includes numerous steps: First of which is the binding of the protein to an affinity matrix, followed by washing and finally release of the protein from the affinity column. The affinity tag, used to achieve selective binding of the intended protein to the matrix remains bound to the protein of interest and has to be selectively removed  by protease cleavage. This often leads to undesirable secondary cleavage within the sequence of the protein. Proteases are the most expensive component of a industrial scale purification process. Eventually, the affinity tag has to be separated from the protein causing further effort and costs.
The split-intein based system is manifold easier: All these steps can be combined into one by using Ssp DnaB mini-intein, an artificial split intein. It performs the trans-splicing reaction only under certain pH and temperature conditions. A mutation of the SspDnaB N-intein at Cys1 to Ala prevents cleavage at the N-terminus, cleavage at the C-terminus still occurs. The N-terminal intein part, attached to a chitin binding domain can be used as an affinity ligand, while the C-terminal part, fused to a protein of interest, can be used as an affinity tag binding highly specific to the ligand part. After loading the protein of interest on a chitin column, the trans-splicing reaction can be induced by adjusting temperature and pH: the both split inteins fuse together, disrupting the fusion to the protein of interest in one single step <A href="#Reference">[6]</A>. 
Thereby, split-inteins simplify conventional protein purification in many aspects: The system is cheaper, faster and more efficient than other purification procedures (Ref.). In fact, only a single chromatographic step is necessary for both, binding of the protein fused to an affinity tag to the affinity column and tag removal. Also, there is no need for expensive proteases. 
</P><P>We standardized this split-intein purification system, so it is easy applicable to all kinds of proteins. Visit our <A href="https://2014.igem.org/Team:Heidelberg/Toolbox_Guide" class="external text" rel="nofollow">toolbox guide</A> to purify your protein!
</P><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Purification with inteins.</SPAN></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="On_Off">On Off</SPAN></H1><H2><SPAN class="mw-headline" id="On">On</SPAN></H2><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">NpuDnaE N-intein assembly construct</SPAN><P>Illustration of the part BBa_K1362100, the N-construct.</P></DIV></DIV></DIV><DIV class="pull-right col-lg-3 col-md-4 col-sm-6 col-xs-8"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">NpuDnaE C-intein assembly construct</SPAN><P>Illustration of the part BBa_K1362101, the C-construct.</P></DIV></DIV></DIV>

In many systems biology projects, it is desirable to have an external trigger to turn a protein’s function ON and OFF in cells at a specific time point. The most prominent approach to achieve the ON-OFF switch is the regulation via gene expression. However this method is unfavorable in terms of temporal and spatial resolution. Using our intein toolbox it could become possible to activate a specific protein in a shorter period of time in single cell resolution. Imaging the most prominent system currently used in molecular biology- the CRISPR/Cas technology temporal and spatial activated within a single cell. This could be achieved using our assembly construct with two insertion sides to fuse proteins or protein domains combined with our light induction approach. Cas9 could be split in two parts resulting in an inactive state of each half. Upon splicing reaction the protein can be reassembled and its functionality restored. Unfortunately, we ran out of time to apply our assembly construct for the experiments with Cas9. However using our toolbox this can easily be achieved within few days. As proof of principle for the fusion and activation of proteins, we showed the reconstitution of fluorescence of <A href="https://2014.igem.org/Team:Heidelberg/Project/Reconstitution" class="external text" rel="nofollow">superfold GFP</A> that was previously split and connected to split NpuDnaE inteins. 
<P>Our assembly construct with two insertion sides provides an easy to handle approach to clone in front and behind of each split intein. Initially N- and C-terminal parts are separated on two plasmids and each intein provides the two insertion sides. The iGEM team Heidelberg 2014 created both constructs: <A href="http://parts.igem.org/Part:BBa_K1362100" class="external text" rel="nofollow">BBa_K1362100</A>, the N-construct and <A href="http://parts.igem.org/Part:BBa_K1362101" class="external text" rel="nofollow">BBa_K1362101</A>, the C-construct. As place holder for the insertion sides mRFP selection markers were included that are easily exchangeable via Golden Gate cloning using the restriction enzyme BsaI. The user only needs to make sure to clone the protein halves, which should reassemble, into the correct insertion sides; namely the N-terminal protein half into the N-terminal “gate” and the C-terminal protein into the C-terminal “gate”. 
</P><P>Confused? All your questions will be answered once looking into our <A href="https://2014.igem.org/Team:Heidelberg/Parts/RFC" class="external text" rel="nofollow">RFC</A>!
</P><P>In this manner the separated plasmids containing either N- or C-terminal splicing construct can be coexpressed by transforming both plasmid into one cell. Alternatively they can be assembled onto one plasmid by standard Biobrick cloning. 
</P><P>We generated the assembly construct by joining and amplification of the three inserts – first insertion side, Npu DnaE intein and second insertion side- via polymerase cycling assembly (PCA) cloning method. PCA is a modification of CPEC cloning, resulting only in the amplification of the assembled insert fragment. Using CPEC this fragment was incorporated in our expression vector. 
The sequencing of our assembly construct with two insertion sides is available.
</P><H2><SPAN class="mw-headline" id="Off">Off</SPAN></H2><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">The intein protease construct.</SPAN></DIV></DIV></DIV><DIV class="pull-right col-lg-6 col-md-6 col-sm-8 col-xs-12"><DIV class="thumbnail"><DIV class="caption"><SPAN style="font-weight:bold;">Principle of the intein protease.</SPAN></DIV></DIV></DIV><P>We provide two different tools for deactivation of proteins. One of them is the C-terminal degradation tag SsrA, the other one the intein protease.The SsrA tag directs the tagged protein to the ClpXP protease, causing degradation of this protein <A href="#References">[10]</A>. 
The intein protease is a singular tool for site-specific in vivo protein cleavage. Under in vitro conditions, protease usage is relatively easy compared to in vivo conditions. The inside of a cell constitutes a very complex environment, so that conventional proteases would simply cleave not only the desired site of the protein of interest, but also other proteins inside this cell, causing huge interference with pathways crucial for cell functions. So far, only the tobacco etch virus (TEV) protease has been used with success for in-vivo cleavage without severe decline of cell viability.<A href="#References">[12]</A>
Volkmann et al. first showed in vitro functionality of the intein protease <A href="#References">[11]</A>. Later it was shown,  that the intein proteease is also applicable in vivo in E. coli and yeast cells without impairment of cell functions.<A href="#References">[12]</A></P><P>The intein protease consists of the 144-aa SspDnaB S1 C-intein (with an Asn-to-Ala mutation at the end of the C-intein to prevent protein splicing) followed by the extein sequence. This C-terminal part of the split intein recognizes its counterpart, the N-terminal split intein inserted into the sequence of the proteine to be cleaved, and cleaces there between extein and intein sequence. 
</P><H4><SPAN class="mw-headline" id="Intein_protease_standard_construction">Intein protease standard construction</SPAN></H4><P>SspDnaB C intein was PCR amplified from a plasmid (pCL20) received from Prof. Dr. Henning Mootz. The PCR primers contained overhangs to get the following constuct:
</P><P>XbaI site – T7RBS - His6 – SspDnaB C intein (N → A) – SspDnaB C extein – SpeI site – PstI site
</P><P>
This PCR product and pSB1C3 were digested with XbaI and PstI. The digested PCR product was ligated into pSB1C3. The resulting plasmid is BBa_K1362050 (Intein protease – T7RBS-His-SspDnaBC ).
</P><P>The next step was to add a promotor. Therefore, BBa_K1362050 was digested with EcoRI and XbaI. BBa_K808000, an Ara promotor, was digested with SpeI and PstI. These two parts were ligated, resulting in BBa_K1362051 (Intein protease with arabinose inducible regulatory promoter/ repressor araC-Pbad-T7RBS-His-SspDnaBC).
</P><H4><SPAN class="mw-headline" id="Results_2">Results</SPAN></H4><P>Both intein protease constructs BBa_K1362050 and BBa_K1362051 were sequenced. We obtained positive results. Here you can see the sequencing results of BBa_K1362051, the longer of the constructs (and including BBa_K1362050),
</P><P>
BBa_K1362051 
</P><P><A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/8/8c/Heidelberg_orig_ARA1.png" target="_blank">part1:</A><A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/2/23/Heidelberg_orig_ARA2.png" target="_blank">part2:</A><A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/b/be/Heidelberg_orig_ARA3.png" target="_blank">part3:</A><A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/9/9e/Heidelberg_orig_ARA4.png" target="_blank">part4:</A><A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/b/b6/Heidelberg_orig_ARA5.png" target="_blank">part5:</A><A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/7/72/Heidelberg_orig_ARA6.png" target="_blank">part6:</A><A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/2/25/Heidelberg_orig_ARA7.png" target="_blank">part7:</A><A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/9/9d/Heidelberg_orig_ARA8.png" target="_blank">part8:</A></P><H4><SPAN class="mw-headline" id="SsrA_degradation_tag_standard_construction">SsrA degradation tag standard construction</SPAN></H4><P>The SsrA degradation tag was ordered as DNA oligos containing the tag sequence, extein sequence and BsaI overhangs. By Golden Gate Assembly the tag was cloned into the carrier construct BBa_K1362052 (for C-terminal tags). This carrier construct consists of a pSB1C3 backbone with two BsaI sites as an insert. It was used to send the tag to the registry of standard biological parts. The user can afterwards easily use Golden Gate Assembly to clone the tag from the carrier constructs into intein assembly constructs.
</P><H4><SPAN class="mw-headline" id="Results_3">Results</SPAN></H4><P>The part was sequenced, the results were positive. You can see the sequencing results here:
</P><P><A class="img-enlarge" href="https://static.igem.org/mediawiki/2014/3/3c/Heidelberg_orig_Ssra.png" target="_blank">SsrA:</A> BBa_K1362054
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>[1] McGinness, KE et al.: Engineering controllable protein degradation. Molecular Cell
22, 701–707, June 9, 2006. DOI 10.1016/j.molcel.2006.04.027
</P><P>[2] Volkmann, G. et al: Controllable protein cleavages through intein fragment complementation. Protein Sci, 18 (2009), pp. 2393–2402. DOI: 10.1002/pro.249
</P><P>[3] Volkmann, Gerrit et al.: Site-specific protein cleavage in vivo by an intein-derived protease.     
FEBS Letters 586 (2012) 79–84. doi:10.1016/j.febslet.2011.11.028.
</P><P>[4] Vila-Perello, Miquel et al.: Biological Applications of Protein Splicing. Cell 143. October 15,   
2010. DOI 10.1016/j.cell.2010.09.031.
</P><P>[5] Muir, Tom W. et al.: Expressed protein ligation: A general method for protein engineering. Proc.   
Natl. Acad. Sci. USA 95 (1998). 
</P><P>[6] Lu, Wei et al.: Split intein facilitated tag affinity purification for recombinant proteins with controllable tag removal by inducible auto-cleavage. J. Chromatogr. A 1218 (2011) 
</P><P>[7] Hauptmann, V. et al.: Native-sized spider silk proteins synthesized in planta via intein-based multimerization. Transgenic Res (2013) 22:369–377. DOI 10.1007/s11248-012-9655-6.
</P><P>[8/D] Vila-Perello, Miquel et al.: Biological Applications of Protein Splicing. Cell 143. October 15,  
2010. DOI 10.1016/j.cell.2010.09.031.
</P><P>[9/E] Muir, Tom W. et al.: Expressed protein ligation: A general method for protein engineering. Proc. Natl. Acad. Sci. USA 95 (1998).
</P><P>[10] McGinness, KE et al.: Engineering controllable protein degradation. Molecular Cell
22, 701–707, June 9, 2006. DOI 10.1016/j.molcel.2006.04.027
</P><P>[11] Volkmann, G. et al: Controllable protein cleavages through intein fragment complementation. Protein Sci, 18 (2009), pp. 2393–2402. DOI: 10.1002/pro.249
</P><P>[12] Volkmann, Gerrit et al.: Site-specific protein cleavage in vivo by an intein-derived protease.     
FEBS Letters 586 (2012) 79–84. doi:10.1016/j.febslet.2011.11.028.
</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Heidelberg/Project/Toolbox">http://2014.igem.org/Team:Heidelberg/Project/Toolbox</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project/Toolbox" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project/Toolbox" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Toolbox&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Toolbox&amp;oldid=343556" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>