<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_pages_Parts"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/pages/Parts">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/pages/Parts&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/pages/Parts&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/pages/Parts&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/pages/Parts" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/pages/Parts</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1 id="Favorite Parts">Favorite Parts.</H1><P>The iGEM Team Heidelberg 2014 had built a new biological system for the iGEM community integrating split-inteins. 
	Intein splicing is a natural process that excises one part of a protein and leaves the remaining parts irreversibly attached. This great function allows you to modify your protein in numerous ways.</P><P>Creating a toolbox including all great functions and possibilities of inteins, we need a new standard for the scientific world of iGEM. This standard, the RFC of the iGEM Team Heidelberg 2014, allows us to easily and modulary work with split inteins.</P><P>Our favorite Parts represent the basic constructs of our toolbox – the Assembly and the Circularization construct, which are both tested in many methods and applications. </P><P>In the following we present you 
		<A href="http://parts.igem.org/Part:BBa_K1362000">BBa_K1362000</A>, the construct for circularization, 
		<A href="http://parts.igem.org/Part:BBa_K1362100">BBa_K1362100</A> and 
		<A href="http://parts.igem.org/BBa_K1362101">BBa_K1362101</A>, the N- and the C-construct for assembly. Take a look and visit the Partsregistry to read the associated documentation.</P><H3> Circularization Construct. BBa_K1362000 </H3><DIV class="row"><DIV class="col-md-4 col-sm-12 col-xs-12"><H4>BBa_K1362000</H4>
				This part is an element of the Team Heidelberg 2014 intein toolbox and can be used as described in the new RFC[i]. The purpose of this tool is to provide an easy way to circularize any linear protein. While conserving the functionality of their linear counterpart, circular proteins can be superior in terms of thermostability, resistance against chemical denaturation and protection from exopeptidases. Moreover, a circular backbone can improve in vivo stability of therapeutical proteins and peptides.
Using this part, Xylanase, lambda lysozyme and DNA methyltransferase (DNMT1) were successfully circularized.



			</DIV></DIV><H3 id="Assembly">  Assembly Constructs. BBa_K1362100 and BBa_K1362101 </H3><DIV class="row"><DIV class="col-md-4 col-sm-12 col-xs-12"><H4>BBa_K1362100</H4><P>This intein assembly construct is part of our strategy for cloning with split inteins. Inteins are naturally occuring peptide sequences that splice out of a precursor protein and attach the remaining ends together to form a new protein. When splitting those intein sequence into an N-terminal and a C-terminal split intein one is left with a powerful tool to post-translationally modify whole proteins on the amino-acid sequence level. This construct was designed to express any protein of interest fused to the Nostoc punctiforme DnaE N-terminal split intein. </P></DIV></DIV><DIV class="row"><DIV class="col-md-4 col-sm-12 col-xs-12"><H4>BBa_K1362101</H4>
				BBa_K1362101 is the corresponding C-terminal construct to BBa_K1362100. Upon coexpression or mixture of the N- and C-constructs protein splicing takes place and the N- and C-terminal proteins of interest are irreversibly assembled via a newly formed peptide bond.<P>
				This mechanism can be applied for a variety of different uses such as the activation of a protein through reconstitution of individually expressed split halves. See our <A href="https://2014.igem.org/Team:Heidelberg/Project/Reconstitution">split sfGFP experiment</A> set a foundation that you guys can build on!</P></DIV></DIV><H1 id="Sample Data Page">Sample Data Page for our favorite Parts.</H1><H3> Circularization Construct. BBa_K1362000 </H3><DIV class="row"><DIV class="col-md-5 col-sm-12 col-xs-12"><DIV class="well well-sm">
					This part represents an easy way to circularize any protein. In a single step you can clone your protein in the split intein circularization construct. Exteins, RFC [i] standard overhangs and BsaI sites have to be added to the coding sequence of the protein to be circularized without start- and stop codons by PCR. By Golden Gate assembly, the mRFP selection marker has to be replaced with the protein insert.
					If the distance of the ends of your protein of interest aren't close enough to connect them you will need a linker. <A href="http://parts.igem.org/Part:BBa_K1362000">BBa_K1362000</A>, the split intein circularization construct, includes a strong T7 RBS (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1362090">BBa_K1362090</A>), we sent to the parts registry as well, and the split intein Npu DnaE. The T7 RBS derived from the T7 phage gene 10a (major capsid protein). </DIV><DIV class="well well-sm">
						The resulting plasmid can be used to express the protein of interest with the obligatory linker and the N- and C-intein.
					</DIV><DIV class="well well-sm">
						In an autocatalytic in vivo reaction, the circular protein is formed. To read more about the trans-splicing reaction visit our <A href="https://2014.igem.org/Team:Heidelberg/Project/Background">Intein Background</A> page. If corresponding split inteins are added to both termini of a protein, the trans-splicing reaction results in a circular backbone.    
					</DIV><DIV class="well well-sm">
						Circular proteins offers many advantages. While conserving the functionality of their linear counterpart, circular proteins can be superior in terms of thermostability, resistance against chemical denaturation and protection from exopeptidases. Moreover, a circular backbone can improve in vivo stability of therapeutical proteins and peptides.
					</DIV></DIV></DIV><H3> Assembly Construct. BBa_K1362100 and BBa_K1362101 </H3><DIV class="row"><DIV class="col-md-5 col-sm-12 col-xs-12"><DIV class="well well-sm">
						These parts represent an easy way to build fusion constructs of intein parts with any protein or peptide of interest (POI). In a single step you can fuse your protein to a split intein part. Desired extein residues, RFC [i] standard overhangs and BsaI sites generating these overhangs have to be added to the coding sequence of the POI by PCR. Facilitating the highly efficient GoldenGate assembly reaction, the mRFP selection marker can be replaced with the POI insert.</DIV><DIV class="well well-sm">
							The resulting parts can be concatenated using standard BioBrick cloning into an expression backbone to coexpress the POIs fused to a N- and a C-intein.
						</DIV><DIV class="well well-sm">
							In an auto catalytic in vivo reaction, the inteins will cleave themselves out and ligate the exteins - here your POIs - together. For example we used this mechanism to <A href="https://2014.igem.org/Team:Heidelberg/Project/Reconstitution">reconstitute the fluorescence of split sfGFP</A> . To read more about the trans-splicing reaction visit our <A href="https://2014.igem.org/Team:Heidelberg/Project/Background">Intein Background</A> page. The whole process becomes significant more interesting with <A href="https://2014.igem.org/Team:Heidelberg/Project/LOV">conditional protein trans splicing</A>.   
						</DIV><DIV class="well well-sm">
							If performing experiments with split inteins make sure you always have a non-splicing negative control. Check out our non splicing assembly constructs (with part names ending with 2 or 3) for example: <A href="http://parts.igem.org/Part:BBa_K1362102">BBa_K1362102</A> and <A href="http://parts.igem.org/Part:BBa_K1362103">BBa_K1362103</A></DIV></DIV></DIV><H1 id="Intein Library">Intein Library.</H1>
				
				Inteins are the basic unity of our toolbox. They are integrated as extraneous polypeptide sequences into habitual proteins and do not follow the original protein function. Inteins perform an autocatalytic splicing reaction, where they excite themselves out of the host protein while reconnecting the remaining chains on both end, so called N and C exteins, via a new peptide bond. Read more about it in our <A href="https://2014.igem.org/Team:Heidelberg/Project/Background|" class="external text" rel="nofollow">project background</A>!
<P>				To characterize the different types and groups of split-inteins and inteins we collect many details about them to develop a intein library. It gives you a great and clear overview about the most important facts.
</P><TABLE class="table table-hover"><TBODY><TR><TH>Split intein
</TH><TH>Special features
</TH><TH>Size N [aa]
</TH><TH>Size C [aa]
</TH><TH>Reaction properties
</TH><TH>Origin
</TH><TH>References
</TH></TR><TR><TD> Npu DnaE</TD><TD>fast; robust at high temperature range and high-yielding trans-splicing activity, well characterised requirements</TD><TD>102</TD><TD>36</TD><TD>t1/2 = 63s , 37°C , k=~1x10^-2 (s^-1); activity range 6 to 37°C</TD><TD>S1 natural split intein, Nostoc punctiforme</TD><TD><A href="#References">[1]</A><A href="#References">[2]</A></TD></TR><TR><TD> Ssp DnaX</TD><TD>cross-reactivity with other N-inteins, transsplicing in vivo and in vitro, high yields</TD><TD>127</TD><TD>6</TD><TD>k=~1.7x10^-4(s^-1); efficiency 96%</TD><TD>engineered from Synechocystis species</TD><TD><A href="#References">[3]</A><A href="#References">[4]</A></TD></TR><TR><TD> Ssp GyrB</TD><TD> very short Nint  facilitates trans-splicing of synthetic peptides</TD><TD>6</TD><TD>150</TD><TD>k=~1x10^-4(s^-1), efficiency 40-80%</TD><TD>S11 split intein enginered from Synechocystis species, strain PCC6803</TD><TD><A href="#References">[4]</A><A href="#References">[5]</A></TD></TR><TR><TD> Ter DnaE3</TD><TD>trans-splicing activity with high yields</TD><TD>102</TD><TD>36</TD><TD>k=~2x10^-4(s^-1), efficiency 87%</TD><TD>natural split intein, Trichodesmium erythraeum</TD><TD><A href="#References">[4]</A><A href="#References">[6]</A></TD></TR><TR><TD> Ssp DnaB</TD><TD>relatively fast</TD><TD>t1/2=12min,  25°C, k=~1x10^-3(s^-1)</TD><TD>engineered from Synechocystis species, strain PCC6803</TD><TD><A href="#References">[2]</A></TD></TR><TR><TD> Gp41-1</TD><TD>fastes known reaction </TD><TD>88</TD><TD>38</TD><TD>t1/2=20-30s, 37°C, k=~1.8x10^-1 (s^-1); activity range 0 to 60°C</TD><TD>natural split intein, Cyanophage</TD><TD><A href="#References">[7]</A><A href="#References">[8]</A></TD></TR></TBODY></TABLE><H3>References</H3><P>[1] Iwai, H., Züger, S., Jin, J. &amp; Tam, P.-H. Highly efficient protein trans-splicing by a naturally split DnaE intein from Nostoc punctiforme. FEBS Lett. 580, 1853–8 (2006).</P><P>[2] Zettler, J., Schütz, V. &amp; Mootz, H. D. The naturally split Npu DnaE intein exhibits an extraordinarily high rate in the protein trans-splicing reaction. FEBS Lett. 583, 909–14 (2009).</P><P>[3] Song, H., Meng, Q. &amp; Liu, X.-Q. Protein trans-splicing of an atypical split intein showing structural flexibility and cross-reactivity. PLoS One 7, e45355 (2012).</P><P>[4] Lin, Y. et al. Protein trans-splicing of multiple atypical split inteins engineered from natural inteins. PLoS One 8, e59516 (2013).</P><P>[5] Appleby, J. H., Zhou, K., Volkmann, G. &amp; Liu, X.-Q. Novel Split Intein for trans-Splicing Synthetic Peptide onto C Terminus of Protein. J. Biol. Chem. 284, 6194–6199 (2009).</P><P>[6] Liu, X.-Q. &amp; Yang, J. Split dnaE genes encoding multiple novel inteins in Trichodesmium erythraeum. J. Biol. Chem. 278, 26315–8 (2003).</P><P>[7] Carvajal-Vallejos, P., Pallissé, R., Mootz, H. D. &amp; Schmidt, S. R. Unprecedented rates and efficiencies revealed for new natural split inteins from metagenomic sources. J. Biol. Chem. 287, 28686–96 (2012).</P><P>[8] Dassa, B., London, N., Stoddard, B. L., Schueler-Furman, O. &amp; Pietrokovski, S. Fractured genes: a novel genomic arrangement involving new split inteins and a new homing endonuclease family. Nucleic Acids Res. 37, 2560–73 (2009).</P><H1 id="Backbones">Our Backbones.</H1><P>Standard BioBrick cloning is a universal way of putting two BioBrick parts together to build a new BioBrick part. Despite several alternative cloning methods allow the assembly of multiple parts at one its simplicity and the broad availability of compatible parts keep it the 'de facto' standard of the iGEM-community.</P><P>Using standard BioBrick cloning, the generation of translationally active parts requires often more than one round of cloning. The ability to easily test the functionality of a protein before cloning them into complicated circuits has the potential to prevent many unsuccessful experiments of iGEM teams and may improve the characterization of the parts in the parts registry. However the extra amount of work required to clone such an additional construct may inhibit this behavior. We therefore improved the standard plasmids pSB1X3 and pSB4X5 by inserting a lacI repressible T7 promoter directly upstream to the BioBrick prefix of those plasmids. This promoter is completely inactive in 'E. coli' strains lacking a T7 RNA polymerase such as TOP10 or DH10beta bute inducible in strains carrying the T7 RNA polymerase under a lacI repressible promoter such as DE3 strains. This enables the use of the same backbone for cloning and over expression. Using 3A assembly a translational active part can be cloned from an RBS and a coding part in one step while maintaining the full flexibility of standard BioBrick assembly. These new RFC 10 conform backbones eliminate one cloning step needed for the expression and thus the characterization of a newly BioBricked protein. Version number 30 was claimed for the high copy variants and version number 50 for the low copy variants.</P><P>High copy BioBrick expression backbone:</P><UL><LI><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1362091">pSB1A30</A>(Part:BBa_K1362091): High copy BioBrick cloning/expression backbone carrying Amp resistance</LI><LI><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1362092">pSB1C30</A>(Part:BBa_K1362092): High copy BioBrick cloning/expression backbone carrying Cm resistance</LI><LI><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1362093">pSB1K30</A>(Part:BBa_K1362093): High copy BioBrick cloning/expression backbone carrying Kan resistance</LI><LI><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1362094">pSB1T30</A>(Part:BBa_K1362094): High copy BioBrick cloning/expression backbone carrying Tet resistance</LI></UL><P>Low copy BioBrick expression backbone:</P><UL><LI><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1362095">pSB4A50</A>(Part:BBa_K1362095): High copy BioBrick cloning/expression backbone carrying Amp resistance</LI><LI><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1362096">pSB4C50</A>(Part:BBa_K1362096): High copy BioBrick cloning/expression backbone carrying Cm resistance</LI><LI><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1362097">pSB4K50</A>(Part:BBa_K1362097): High copy BioBrick cloning/expression backbone carrying Kan resistance</LI></UL><P>Because of the great experience we had using our expression vectors, we sent them to the iGEM team Aachen and Tuebingen. <A href="https://2014.igem.org/Team:Heidelberg/Team/Collaborations">We helped them</A> solving their problems with the expression of their products.</P><H1 id="allParts"><SPAN style="font-size:170%;">List of Parts</SPAN></H1><DIV class="col-xs-12"><DIV id="partsTable">Loading...</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Heidelberg/pages/Parts">http://2014.igem.org/Team:Heidelberg/pages/Parts</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/pages/Parts" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/pages/Parts" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/pages/Parts&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/pages/Parts&amp;oldid=406478" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>