<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UFMG_Brazil_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UFMG Brazil/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="row"><DIV class="col-md-2"><P class="text">Project</P><P class="sub-text hide" data-key="overview" style="position:relative;top:23px">Overview</P><P class="sub-text hide" data-key="problemAndSolution" style="top:25px;font-size: 15px;line-height:110%;">Problem andSolution</P><P class="sub-text hide" data-key="chassis" style="top:30px;font-size: 15px;line-height:110%;">Chassis</P><P class="sub-text hide" data-key="devicesAndKillSwitch" style="top:37px;font-size: 15px;line-height:110%;">Devices andkill switch</P></DIV><DIV class="col-md-2"><P class="text">Lab Work</P><P class="sub-text hide" data-key="safety" style="position:relative;top:20px">Safety</P><P class="sub-text hide" data-key="notebook" style="top:28px;">Notebook</P><P class="sub-text hide" data-key="protocols" style="top:38px;">Protocols</P><P class="sub-text hide" data-key="results" style="top: 45px;">Results</P></DIV><DIV class="col-md-2"><P class="text" data-key="modeling">Modeling</P></DIV><DIV class="col-md-2"><P class="text" data-key="practices">Practices</P><P class="sub-text hide" style="position:relative;top:22px" data-key="practicesOverview">Overview</P><P class="sub-text hide" style="position:relative;top:27px;font-size: 14px;line-height:110%;" data-key="integratedHumanPractices">Integrated HumanPractices</P><P class="sub-text hide" style="top:23px;font-size: 15px;line-height:110%;" data-key="publicEngagement">PublicEngagement</P></DIV><DIV class="col-md-2"><P class="text" data-key="synenergene">Synenergene</P><P class="sub-text hide" style="position:relative;top:22px" data-key="synenergeneOverview">Overview</P><P class="sub-text hide" style="top:25px;font-size: 15px;line-height:110%;" data-key="applicationScenarios">ApplicationScenarios</P><P class="sub-text hide" style="position:relative;top:27px" data-key="technoMoral">Techno-moral</P></DIV><DIV class="col-md-2"><P class="text">Team</P><P class="sub-text hide" style="position:relative;top:22px" data-key="ourTeam">Our Team</P><P class="sub-text hide" style="top:30px;" data-key="attributions">Attributions</P><P class="sub-text hide" style="top:40px;" data-key="collaborations">Collaborations</P><P class="sub-text hide" style="top: 46px;" data-key="sponsors">Sponsors</P></DIV></DIV><DIV class="row"><DIV class="col-md-11 content"><H3>Results</H3><P> DNA coding sequence of Mouse IFN-β linked to signal peptide that allow protein secretion and latency-associated peptide (LAP) that possibilite IFN-β activation only in inflammatory site was designed with optimized codons for expression in <I>Leishmania</I> host (Figure 1).</P><H5><B>Figure 1</B>: LAP-IFN-β synthetic gene synthetized. Highlighted in green is signal peptide for secretion and highlighted in purple is LAP sequence.</H5><P> The final sequence was synthesized by IDT (California, USA) in pUC_IDT standard vector with resistance to ampicillin. <I>E. coli</I> strain DH10b/TOP10 was transformed with the plasmid and resistant colonies were observed after incubation by 16 hours at 37° C (Figure 2).</P><H5><B>Figure 2</B>: Plate of <I>E. coli</I> colonies transformed with pUC_IDT vector containing recombinant IFN-β. Red arrows label positive colonies.</H5><P>Miniprep to recover the plasmids were carried out and the quality of extraction was confirmed by 1.2% agarose gel stained with SYBR Safe DNA Gel Stain from Invitrogen (Waltham, USA) (Figure 3).</P><H5><B>Figure 3</B>: Electrophoresis in 1.2% agarose gel of plasmid pUC_IDT containing recombinant IFN-β. Labels of lanes: 1) Molecular weight markers; 2) Negative control; 3 and 4) Plasmids recovered by miniprep protocol.</H5><P>Plasmids containing IFN-β were used as DNA template in a PCR with specific primers pair to the synthetic gene (Figure 4). The forward primer had restriction site of enzyme <I>Xba</I>I in the 5’ region and the reverse oligonucleotide sequence had site of <I>Bam</I>HI in the 5’ region.</P><H5><B>Figure 4</B>: Electrophoresis in 1.2% agarose gel of IFN-β a PCR product containing restriction sites to <I>Xba</I>I and <I>Bam</I>HI enzymes. The molecular weight of correct PCR product should be 1461 base pairs. Labels of lanes: 1) Molecular weight markers; 2, 3 and 4) Positive PCR for three transformed <I>E. coli</I> colonies; 5) Negative control.</H5><P>The PCR product was joined with pGEM-T Easy vector from Promega (Fitchburg, USA). <I>E. coli</I> DH10b/TOP10 were transformed with the ligated product (Figure 5) or pLpNEO vector specific to <I>Leishmania</I> expression. After resistant bacteria selection and cultivation in liquid medium, miniprep protocol was performed to recovered the plasmids (Figures 6 and 7).</P><H5><B>Figure 5</B>: Plate of <I>E. coli</I> colonies transformed with pGEM-T Easy vector containing recombinant IFN-β. Red arrows label positive colonies. </H5><H5><B>Figure 6</B>: Electrophoresis in 1.2% agarose gel of plasmid pGEM-T Easy containing synthetic IFN-β. Labels of lanes: 1) Molecular weight markers; 2, 3, 4 and 5) Plasmids recovered by miniprep protocol.</H5><H5><B>Figure 7</B>: Electrophoresis in 1.2% agarose gel of pLpNEO vector. Labels of lanes: 1) Molecular weight markers; 2 and 3) Plasmids recovered by miniprep protocol.</H5><P>Our experiments could not be finished. We had a major setback because of the Brazilian import policy, and the biobricks only arrived in July. We also spent a lot of timw standardizing the protocols. Moreover, the limited amount of DNA made it to hard to transform the biobricks into the bacteria (since we had to repeat it several times).</P><P>However, despite the technical limitations, we have plans to return the experiments and complete the project after the competition.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>