<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_uOttawa_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:uOttawa/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:uOttawa/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:uOttawa/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:uOttawa/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:uOttawa/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:uOttawa/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P><TITLE>banner_with_menu.gif</TITLE></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Project_Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Project Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#New_selection_markers"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">New selection markers</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Results"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Ade4_targeting_vector"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Ade4 targeting vector</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Results_2"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Proximal_.E2.80.93_Distal_promoter_distinction"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Proximal – Distal promoter distinction</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Characterization_of_the_ADH1_promoter"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Characterization of the ADH1 promoter</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Results_3"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Power_of_yeast_mating"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Power of yeast mating</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Possible_network_construction_with_our_parts_.28Including_toggle_switches.29"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Possible network construction with our parts (Including toggle switches)</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#References"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Project_Overview">Project Overview</SPAN></H2><P>Yeast (<I>Saccharomyces cerevisiae</I>) is one of the earliest organisms used in biotechnology, even if we did not know that we were doing science at the time. Humans have been experimenting with yeast ever since our ancestors brewed their first beer. However, there is more to yeast than just making beer and bread; it also happens to be one of the simplest eukaryotic organisms. Due to its homology with mammalian organisms and its ease of genetic manipulation, yeast has become a model organism for the study of systems biology. Many key genes in mammalian organisms were first identified in yeast. For this reason, we believe it is important to develop tools that improve our ability to use yeast as a standard BioBrick organism. 
</P><P>In summary we have BioBricked two yeast selection markers and have shown that they work, constructed a novel plasmid which greatly increases the efficiency of genomic integration into yeast, BioBricked several new yeast promoters and characterized one, developed a novel strategy for dissecting promoters. Please, browse through the following sections in order to learn more about each aspect of our project. We hope that these contributions will encourage more iGem teams to use yeast as part of their projects!
</P><H2><SPAN class="mw-headline" id="New_selection_markers">New selection markers</SPAN></H2><P>There are generally two types of selection markers used in yeast, auxotropic markers and selection markers that confer drug resistance. Standard yeast strains have deletions and mutations in genes in a pathway that generates an essential metabolite (such as an amino acid). These yeast strains must be grown on media supplemented with that particular metabolite. When integrating constructs into yeast, a functional copy of that gene can be brought over. Only yeast cells that contain that functional copy will be able to grow on media lacking that specific metabolite. From our experience we have learned that there are two major drawbacks with such selection. Firstly, this type of selection is not very strong, small background colonies that do not contain the integrated construct often appear on plates. Secondly, since the selection marker shares substantial homology with the mutated / partially deleted ORF in the wild type strain, it is possible that the selection marker can undergo homologous recombination with the wild type ORF and this results in a yeast colony that can survive on the restricted media but does not contain the construct of interest. Drug selection markers are much stronger than auxotrophic markers in yeast and for this reason they have less background on plates. We BioBricked natMX6 and kanMX, which are completely heterologous and do not share any homology with the yeast genome. There is less chance that they will recombinate into the incorrect locus. They provide resistance to the drugs Nourseothricin sulfate and G418 respectively.
</P><H3><SPAN class="mw-headline" id="Results">Results</SPAN></H3><P><B>Figure 1:</B>YPH500 cells transformed with our ADE4 targeting vector containing the natMX6 resistance marker. The plate contains YPD media with 100 ug/mL Nourseothricin. White colonies indicate a successful transformation into the ADE4 locus, all colonies are white.
</P><P><B>Figure 2:</B>Wild type YPH500 cells plated on YPD media with 100 ug/mL Nourseothricin, no growth is observed
</P><P><B>Figure 3:</B>YPH500 cells transformed with our ADE4 targeting vector containing the kanMX resistance marker. The plate contains YPD media with 0.2 mg/mL G418. White colonies indicate a successful transformation into the ADE4 locus. Transformation efficiency is very high, single colonies are hard to discern, cells seem to be growing as a lawn. Areas of high cell density appear red presumably due to cells growing on top of each other.  Areas of lower cell density are white. 
</P><P><B>Figure 4:</B>Wild type YPH500 cells plated on YPD media with 0.2 mg/mL , no growth is observed
</P><H2><SPAN class="mw-headline" id="Ade4_targeting_vector">Ade4 targeting vector</SPAN></H2><P>When implementing finely balanced synthetic networks in vivo it is very important to tightly control the copy number of network components (Guido, 2006). In yeast the easiest way to control the copy number is to keep it at 1 by integrating constructs into the yeast genome. Constructs can be integrated into the yeast genome by homologous recombination, which is a natural yeast mechanism used to repair damaged DNA (Hua, 1997). The length of the homology between the insert and the genome increases the specificity of targeting and cloning efficiency (Gray, 2001). Although there are several yeast integrating vectors that already exist (Sikorski, 1989), they have their own drawbacks. Namely, they do not comply by the BioBrick standards, they have short homologies with the yeast genome, they require special yeast deletion strains in order to function, and most do not provide colorimetric validation of successful integration. We therefore constructed a new plasmid (BBa_K319043) that is able to integrate into the ADE4 locus, knocking out the ADE4 ORF with high specificity. The ADE4 locus which is flanked by ATM1 and DYN3 was chosen as the site for integration because knocking out the ADE4 gene in an ADE2 mutant strain (such as YPH500) restores the white colony phenotype by undercutting the adenine biosynthesis pathway (Ugolini, 1996).
</P><P><B>Figure 5:</B> Adenine biosynthesis pathway retreived from <A href="http://www.phys.ksu.edu/gene/GENEFAQ.html" class="external free" rel="nofollow">http://www.phys.ksu.edu/gene/GENEFAQ.html</A>. AIR gets converted into a red pigment. Mutations in ADE1 and ADE2 genes lead to the accumulation of the red colour in yeast. Mutations in upstream genes in the pathway do not allow for the generation of AIR, and therefore can restore the white phenotype in ADE1 and ADE2 mutants.
</P><P>Therefore a white colony in this background indicates successful integration at the correct locus and eliminates the need for time consuming and costly PCR validation. BBa_K319043 is a BioBrick backbone that conforms to Assembly standard 23 (Phillips, 2006). The plasmid contains 200 bp homologous to the ATM1 terminator followed by the BioBrick cloning sites, followed by 200 bp homologous to the ADE4 terminator. Two version of the plasmid contain KanMX and NatMx expression cassettes (pADE4TAK and pADE4TAN respectively) between the ATM1 terminator and the BioBrick cut sites.
</P><H3><SPAN class="mw-headline" id="Results_2">Results</SPAN></H3><P>Figure 1 shows a transformation into YPH500 of the ADE4 targeting vector with a natMX6 resistance cassette. 100% of the colonies are white, this means that in combination with the herterologous marker natMX6, the ADE4 targeting vector has a specificity of 100%. For comparison the following image shows a typical yeast transformation performed in our lab where a construct of interest in targeted to the ADE2 locus with Ura3 selection using 40 base pairs of homology added as an PCR primer overhang. Since the ADE2 locus is being targeted, red colonies indicate a successful transformation with targeting to the correct locus, and white colonies indicate targeting to the incorrect locus. Targeting to a specific locus in yeast is important because each locus has its own suceptipility to chomosomal positioning effects. If two constructs are to be compared, then this comparison must be performed in the same locus and the experimenter must ensure that the correct locus is targeted. 21 colonies are red and the other 64 colonies are white, this represents a 25% specificity. If loci that do not allow for colour selection are used, the experimenter must pick several colonies and screen them by PCR if only using 40 base pair homologies. This can be a tedious and costly process in high throughput applications such as library construction or screening and demonstrates why the ADE4 targeting vectors are useful.
</P><P><B>Figure 6:</B> A transformation of a construct of interest with URA3 selection into the ADE2 locus of BY4742 using 40 base pair overhangs. There are 21 red colonies and 64 white colonies.
</P><H2><SPAN class="mw-headline" id="Proximal_.E2.80.93_Distal_promoter_distinction">Proximal – Distal promoter distinction</SPAN></H2><P>Ellis et al. developed a library of 40 variants of the inducible Gal1 promoter that contain operator sites for the binding of the TetR and the LacI repressors. These promoter libraries were generously donated to us. Eukaryotic promoters can be split into two halves, the proximal component and the distal component. The distal component is thought to be responsible for binding upstream activators. The proximal component is responsible for binding to the TATA binding protein and is directly involved in the recruitment of transcriptional machinery. We therefore hypothesized that it would be possible to shuffle around distal and proximal promoter components from various promoters. We proceeded to BioBrick the Act1 distal promoter and combine it with the proximal elements from the Ellis libraries. We were not able to finish this aspect of the project, however we have made all the BioBricks available to the registry. If this type of promoter shuffling in Eukaryotes is possible, then it can have a significant impact on eukaryotic synthetic biology. There are many interesting wild type promoters in nature that respond to a particular signal and these promoters are often used in biosensors. A problem with many of these interesting promoters is that although they respond correctly to a signal, they may not produced the desired output. By adding the correct proximal promoter, one should be able to tune the desired output rationally.
</P><H2><SPAN class="mw-headline" id="Characterization_of_the_ADH1_promoter">Characterization of the ADH1 promoter</SPAN></H2><P>Although there are other versions of the ADH1 promoter already available in the registry, they are longer than needed for optimal expression from the promoter. Furthermore, since these promoters are longer, they are inhibited by ethanol which is used as a source of energy during the ethanol phase of the yeast life cycle. This information has been published by Ruohonen et al. We therefore decided to contribute an improved version of the ADH1 promoter in yeast. The length that was chosen for this ADH1 promoter is more manageable for the construction of larger networks, and the promoter is not inhibited by ethanol. We characterized the expression of the ADH1 promoter by cloning it in front of yEGFP with a ADH1 terminator. This construct was cloned into a pRS403 vector and integrated into the YPH500 genome. The resulting strain was innoculated in 3 mL of YPD + 2% Glucose + 2% Adenine overnight. The overnight culture was reinoculated into 2 mL of the same media at an OD of 0.02. After three hours the strain was analyzed on a Cyan ADP flow cytometer, 488 laser was used for exitation and emission was captured with a standard FITCH filterset. We observed roughly a 100 fold increase in GFP expression over wild type YPH500.
</P><H3><SPAN class="mw-headline" id="Results_3">Results</SPAN></H3><P><B>Figure 7</B>: Fluorescence of wild type YPH500
</P><P><B>Figure 8</B>: Fluorescence of ADH1 promoter driving yEGFP expression in a YPH500 background.
</P><H2><SPAN class="mw-headline" id="Power_of_yeast_mating">Power of yeast mating</SPAN></H2><P>Yeast cells can live as hapoid organisms and as diploid organisms, the following schematic describes how two haploid yeast cells of different mating types that each contain part of a synthetic network can be combined to form a diploid yeast cell containing the full network.
</P><P><B>Figure: 9</B> Schematic describing the yeast mating process
</P><P>The two versions of the previously mentioned ADE4 targeting vector containing Nat and Kan selection are compatible with a yeast mating approach for combinatorial synthetic network construction. Half of a network can be cloned into the nat plasmid, and other half into the kan plasmid. The nat plasmid can be transformed into YPH499 and the kan plasmid can be transformed into YPH500, for example. If there are several different versions of the network in the nat plasmid (for instance each containing a certain promoter of different strength), it is much easier to mate them together than to clone them individually.
</P><H2><SPAN class="mw-headline" id="Possible_network_construction_with_our_parts_.28Including_toggle_switches.29">Possible network construction with our parts (Including toggle switches)</SPAN></H2><P>One of the goals of the 2010 iGEM team was to create a library of toggle switches. Although we did not manage to create this library we have contributed many of the BioBricks necessary for its construction. The rational behind choosing this project was not to generate the library, but to generate a toolkit of useful yeast components and use the library as a proof of principle. We have generated several useful BioBricks, and we hope they they will be used by the iGEM community! 
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI>Guido, N. et al. A bottom-up approach to gene regulation. Nature 439, 856-860, doi:10.1038/nature04473 (2006).
</LI><LI>Hua, S.-B., Qiu, M., Chan, E., Zhu, L. &amp; Luo, Y. Minimum length of sequence homology required for in vivo cloning by homologous recombination in yeast. Plasmid 38, 91-96 (1997).
</LI><LI>Gray, M. &amp; Honigberg, S. M. Effect of chromosomal locus, GC content and length of homology on PCR-mediated targeted gene replacement in Saccharomyces. Nucleic Acids Research 29, 5156-5162 (2001).
</LI><LI>Sikorski, R. S. &amp; Hieter, P. A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in Saccharomyces cerevisiae. Genetics 122, 19-27 (1989).
</LI><LI>Ugolini, S. &amp; Bruschi, C. V. The red/white colony color assay in the yeast Saccharomyces cerevisiae: Epistatic growth advantage of white ade8-18, ade2 cells over red ade2 cells. Curr Genet 30, 485-492 (1996).
</LI><LI>Phillips, I. &amp; Silver, P. A new biobrick assembly strategy designed for facile protein engineering. dspace.mit.edu (2006).
</LI><LI>Ellis et al. Diversity-based, model-guided construction of synthetic gene networks with predicted functions. Nat Biotechnol (2009) vol. 27 (5) pp. 465-71
</LI><LI>Ruohonen et al. Modifications to the ADH1 promoter of Saccharomyces cerevisiae for efficient production of heterologous proteins. Journal of Biotechnology (1995) vol. 39 (3) pp. 193-203
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:uOttawa/Project">http://2010.igem.org/Team:uOttawa/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:uOttawa/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:uOttawa/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:uOttawa/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:uOttawa/Project&amp;oldid=207550" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>