<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Project_Gene_Modelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Project_Gene_Modelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Project_Gene_Modelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Project_Gene_Modelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Project_Gene_Modelling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Project_Gene_Modelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Project Gene Modelling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">Project AuxIn<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London">Home</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Tour">At a Glance</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project/Background">The Problem</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Overview">M1: Phyto-Route</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Overview">M2: Auxin Xpress</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Overview">M3: Gene Guard</A></LI></UL><LI><A class="sf-with-ul" href="#">Achievements<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Achievements">Main Results</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Judging">Judging Criteria</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Europe_Jamboree">Europe Jamboree</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Overview">Informing Design</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human/Outreach">Outreach</A></LI><LI><A class="sf-with-ul" href="#">Extras<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Extras/Collaboration">Collaboration</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Protocols_General">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Software">Software</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Notebook/week1">Diary</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Brainstorming_Health">Brainstorming</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Team">Team</A></LI></DIV><H1>Module 3: Gene Guard</H1><P><B>Containment is a serious issue concerning the release of genetically modified organisms (GMOs) into the environment. To prevent horizontal gene transfer of the genes we are expressing in our chassis, we have developed a system based on the genes encoding holin, anti-holin and endolysin. We are engineering anti-holin into the genome of our chassis, where it acts as an anti-toxin, and holin and endolysin on plasmid DNA. In the event of horizontal gene transfer with a soil bacterium, holin and endolysin will be transferred without anti-holin, rendering the recipient cell non-viable and effectively containing the Auxin Xpress and Phyto-Route genes in our chassis.</B></P><DIV class="underlinemenu"><UL><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Specifications">Specifications</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Design">Design</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Modelling">Modelling</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Assembly">Assembly</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Testing">Testing &amp; Results</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Future">Future Work</A></LI></UL></DIV><H1>Modelling</H1><P style="font-size:1.3em;"><A href="#" onclick="ddaccordion.collapseall('technology'); return false">Collapse all</A>  | <A href="#" onclick="ddaccordion.expandall('technology'); return false">Expand all</A></P><DIV class="technology">1. Introduction</DIV><DIV class="thelanguage"><P>The Gene Guard device involves three proteins:</P><P>1.	<B>Holin</B> is a protein that forms pore complexes in the inner membrane of bacteria and these pores allow endolysin to move into the periplasmic space. </P><P>2.	<B>Endolysin</B> is an enzyme that can break down the cell wall and induce cell lysis. </P><P>3.	<B>Anti-holin</B> binds to holin and prevents it from forming pores. </P><P>Our system will have the holin and endolysin genes together with the Phyto-Route and the Auxin Xpress genes on the plasmid, and the anti-holin gene on the genome under the control of a strong promoter. The production of anti-holin will prevent cell lysis of the genetically modified (GM) bacteria by deactivating holin. If the plasmid is transferred to a different bacterium without anti-holin on its genome (<I>e.g.</I> any wild type soil bacteria), the expression of holin and endolysin will induce cell lysis. This will prevent the spread of the AuxIn plasmid, containing the Phyto-Route and Auxin Xpress genes, to naturally occurring soil bacteria.</P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 0); return false">Collapse</A></P></DIV><DIV class="technology">2. Objectives</DIV><DIV class="thelanguage"><P>    The key to the success of this device is the correct ratio of holin and anti-holin production such that the anti-holin can deactivate all the holin produced in our modified bacteria, and the holin expression level in wild type bacteria is high enough to induce cell lysis. As gene expression is controlled by promoter strength and ribosome binding site (RBS) strength, modelling of the Gene Guard is important to help us choose a promoter and RBS of appropriate strength. </P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 1); return false">Collapse</A></P></DIV><DIV class="technology">3. Description</DIV><DIV class="thelanguage"><P>   The modelling of the Gene Guard consists of two parts, one is the holin production in wild type bacteria and the other is the deactivation of holin by anti-holin in our modified bacteria. Since all the genes in our system are constitutively expressed, we only consider the steady state. </P><H4>Holin production in wild type soil bacteria</H4><P>This part of the modelling is focused on deriving the relationships between promoter strength and RBS strength of holin on a single plasmid. In order to model this, the following assumptions were made: </P><P>1)	Gene Guard is contained on a pSB1C3 plasmid which contains pUC19-derived pMB1 replication origin (~100-300 molecules per cell)<SUP>[5]</SUP>. The number of plasmids in bacteria ranges from 100 to 300 per cell. To make sure the promoter and RBS are strong enough for the holin/endolysin to be effective, we underestimated the number of plasmids in soil bacteria to be 50, although they will always have at least 100. Therefore our system should work whenever the number of plasmids in a soil bacterium is greater than 50.  </P><P>2)	The literature indicates that cell lysis occurs at a holin concentration equal to 1000 molecules/cell<SUP>[1],[5],[6]</SUP>, therefore we used this as the threshold concentration for cell lysis in our system. </P><P>3)      Since the construct used to characterise the Pveg promoter last year contains a mutation, we assume that the<B><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K515010">Pveg2</A></B>  promoter we used for anti-holin has same the strength as Pveg. 
</P><P>Based on these assumptions, the kinetics of cell lysis in wild type bacteria was modelled using ordinary differential equations (ODEs) for the holin mRNA and holin protein (Equation 1). At steady state (<I>i.e.</I> d[mRNA<SUB>holin</SUB>] /dt = 0 and d[holin]/dt = 0), rearranging Equation 1 with [holin] = 1000, we obtained an expression for P<SUB>holin</SUB> K<SUB>holin</SUB> as shown in Equation 2.</P><H4>Holin inhibition in our modified bacteria</H4><P>    In our bacteria the strength of the promoter and RBS controlling expression of anti-holin on the genome should be higher than that of holin on a single plasmid, such that all the holin produced by 300 plasmid copies (the maximum copy number in a single bacterium) can be inhibited. In our system, anti-holin binds to holin to form a dimer, which is defined as the deactivated holin. The mechanism in our modified bacteria is described by Equation 3. We defined the ratio of promoter and RBS strength for anti-holin and holin respectively as &quot;m&quot;, as seen in Equation 4.</P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 2); return false">Collapse</A></P></DIV><DIV class="technology">4. Results and discussion</DIV><DIV class="thelanguage"><P>     By varying the ratio m in Equation 4, we found that the holin concentration in GM bacteria decreased to zero at ratio m ≥ 300 (see Figure 1). Then, we tested m = 400 with the anti-holin promoter (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K515104">J23100</A>) we have with strength 13.1 pg RNA/min/μg substrate DNA, the result is displayed in Figure 2.</P><DIV class="imgbox" style="width:820px;margin:0 auto;"><P><I> Figure 1: The holin concentration in GM bacteria vs. ratio m (m = (P<SUB>anti-holin</SUB>K<SUB>anti-holin</SUB>)/(P<SUB>holin</SUB>K<SUB>holin</SUB>)). This graph shows that the intracellular concentration of holin remains at zero if m &gt; 300. (Modelling by Imperial College London iGEM team 2011).</I></P></DIV><DIV class="imgbox" style="width:820px;margin:0 auto;"><P><I> Figure 2: The evolution of different protein species vs. time at m = 400. All the holin proteins in the GM cell are inhibited as the holin concentration remains at 0 all the time, while the holin concentration in wild type bacteria reaches 1000 per cell after 5000 seconds.(Modelling by Imperial College London iGEM team 2011).</I></P></DIV><P>In summary, the modelling of gene guard informed us that the promoter strength and RBS strength for holin and anti-holin can be arbitrarily chosen as long as their ratio (<I>i.e.</I> P<SUB>anti-holin</SUB>K<SUB>anti-holin</SUB>)/(P<SUB>holin</SUB>K<SUB>holin</SUB>) &gt;300), to make it certain, the ratio value 400 was selected for our project.
</P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 3); return false">Collapse</A></P></DIV><DIV class="technology">5. Parameters</DIV><DIV class="thelanguage"><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 4); return false">Collapse</A></P></DIV><DIV class="technology">6. Matlab code</DIV><DIV class="thelanguage"><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 5); return false">Collapse</A></P></DIV><DIV class="technology">7. References</DIV><DIV class="thelanguage"><P>   [1] Shetty Rp, Endy D, Knight TF Jr (2008) Engineering Biobrick vectors from biobrick parts. <I>J Biol Eng.</I>Apr 14;2:5. doi: 10.1186/1754-1611-2-5</P><P>   [2]  Savva GG, Dewey JS, Deaton J, White RL, Struck DK, Holzenburg A and Young R (2008) The holin of bacteriophage lambda forms rings with large diameter. <I>Molecular Microbiology</I><B>69(4):</B> 784–793.</P><P>   [3] K. Nath, A.L. Koch (1971) Protein degradation in E. coli. The journal of biological chemistry vol. 246, No. 22, Issue of November 25, pp 6956-6967,1971</P><P>   [4] Gründling et al. (2001) Holins kill without warning. <I>PNAS</I><B>98(16):</B> 9348-9352</P><P>   [5] Young, Ry, Ing-Nang Wang and William D. Roof. “Phages will out: strategies of host cell lysis.” Trends in Microbiology 2000; 8(3):120-8.</P><P>   [6] Christos G. Savva, Jill S. Dewey, John Deaton, Rebecca L. White, Douglas K. Struck, Andreas Holzenburg and Ry Young. The holin of bacteriophage lambda forms rings with large diameter. Molecular Microbiology 69(4), 784–793. 2008.</P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 6); return false">Collapse</A></P></DIV><H2><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Design" style="text-decoration:none;color:#728F1D;float:left;">
M3: Design
</A><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Assembly" style="text-decoration:none;color:#728F1D;float:right;">
M3: Assembly
</A></H2><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Imperial_College_London/Project_Gene_Modelling">http://2011.igem.org/Team:Imperial_College_London/Project_Gene_Modelling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Project_Gene_Modelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Project_Gene_Modelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Project_Gene_Modelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Project_Gene_Modelling&amp;oldid=237899" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>