"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College London/test7
From 2011.igem.org
Project AuxIn »
Home
At a Glance
The Problem
M1: Phyto-Route
M2: Auxin Xpress
M3: Gene Guard
Achievements »
Main Results
Data
Judging Criteria
Europe Jamboree
Human Practice »
Informing Design
Safety
Outreach
Extras »
Collaboration
Protocols
Software
Diary
Brainstorming
Team
Module 3: Gene Guard
Containment is a serious issue concerning the release of genetically modified organisms (GMOs) into the environment. To prevent horizontal gene transfer of the genes we are expressing in our chassis, we have developed a system based on the genes encoding holin, anti-holin and endolysin. We are engineering anti-holin into the genome of our chassis, where it acts as an anti-toxin, and holin and endolysin on plasmid DNA. In the event of horizontal gene transfer with a soil bacterium, holin and endolysin will be transferred without anti-holin, rendering the recipient cell non-viable and effectively containing the Auxin Xpress and Phyto-Route genes in our chassis.
Overview
Specifications
Design
Modelling
Assembly
Testing & Results
Future Work
Testing
Horizontal gene transfer is currently an increasingly pressing topic in the field of Synthetic Biology. It is the main limiting factor which is halting the release of GMO's to the environment as while we know what our genetic construct will do in our own chassis we are unsure what it will do in another organism. In order to prevent this issue of genetic containment we have designed a novel toxin/anti-toxin system, the Gene Guard.
In this module we wanted to first test the necessity of this system by performing experiments to see how long E. coli would retain the plasmid in soil as well as under optimal conditions. Then we tested whether the Gene Guard was assembled correctly at each stage. Finally, we wanted to perform an experiment where we transform control cells and our Gene Guard cells with the holin-endolysin plasmid. Is Gene Guard necessary? Moreover, does it work? Stay tuned to find out.
Collapse all | Expand all
1. E.coli survivability in soil and plasmid retainment
1.1 Survivability in soil
Figure 1: Colonies recovered from filter discs and grown on LB plates containing selective antibiotics imaged using a LAS-3000 gel imager. a) Sample taken from non-sterilised soil b) Sample taken from sterilised soil  (Data by Imperial College London iGEM team 2011).
We set up a soil experiment to test how long our E. coli chassis can retain its plasmid in soil. We initially transformed chemically competent E. coliDH5α cells with superfolder GFP. These cells were inoculated on small filter discs (about 0.5 cm diameter), which were placed in autoclaved and non-autoclaved soil. We periodically grew up cultures from these filter discs over the course of seven weeks.
After seven weeks, we were able to recover fluorescent bacteria from sterilised and non-autoclaved soil (Figure 1).
Figure 2: Gel digests of bacteria displaying colony morphology typical of E. coli recovered from non-sterilised and sterilised soil. These bacteria exhibited colony morphologies typical of E. coli. (Data by Imperial College iGEM team 2011).
As is visible from these plates, fluorescence was present in bacteria recovered from both sterile and non-sterile soil. A control plate grown from a filter disc inoculated in non-sterilised soil without fluorescent bacteria showed that there was no contamination with other fluorescent lab bacteria. In order to investigate whether the fluorescence observed was due to the presence of the original sfGFP construct and whether the E. coli-like colonies from the non-sterile sample had retained a plasmid, we extracted plasmid DNA using a miniprep kit and did a digest with EcoRI and PstI and with EcoRI on its own to check for presence of the original insert and size of the unfolded vector, respectively (Figure 2).
The insert is very clearly visible at just below 2 kb. This confirms the presence of superfolder GFP in both cultures. Sequencing of the GFP insert revealed that no mutations had taken place in the superfolder GFP gene contained in the bacteria inoculated in non-sterile and sterile soil.This result was obtained in three separate replicates.
In addition, small colonies appeared on the non-sterile plate that had very different colony morphology. We grew this colony up in LB medium containing selective antibiotic and subsequently performed a separate miniprep. No DNA was yielded in this miniprep. It is therefore likely that the plasmid was not transferred to these bacteria but that they either possess natural antibiotic resistance or were able to survive on plates that whose antibiotics had already been depleted by the presence of resistant engineered bacteria.
All three samples, the sample from the sterile plate, and the two from the non-sterile plate showing E. coli-like and non-E. coli-like morphologies will be used for 16S ribosomal RNA sequencing (using commonly used primers [1]) to determine the bacterial species.
This result is extremely important as it shows that plasmids can be retained in E. coli for a very long period of time even in the presence of competition when inoculated in soil. This gives us an indication of the life-span our chassis would have in the soil in its implementation stage. In addition, long retainment of the plasmid means that the chance for horizontal gene transfer increases, rendering this result very important for the Gene Guard module.
1.2 Plasmid retainment
After observing survaviblity and growth of E. coli in soil, we assumed that the plasmid with sfGFP ( BBa_K515105 ) has been retained in the cells for 7 weeks either due to static state or very slow growth of cells. Therefore we wanted to test the retainment of plasmid within our chassis in optimal growth conditions without the presence of antibiotics. For this observation we required cells, which would be continuously dividing. As experimental set up we have grown cells in LB for a number of days with supplying fresh LB each day to a number of bacteria, which would then each day go through the growth phases of bacterial maturation. Also each day a small proportion of the culture was plated on a petri dish containing antibiotic and grown overnight to observe the extent of plasmid loss.
Figure 2: E. coli cells grown in LB without antibiotic and subsequently plated on the selective plate have been imaged using Fujifilm LAS 3000 Imager. The samples have been plated after a) 1 day, b) 14 days. In sample after 1 day growth single colonies can not be observed due to high cell density which was plated. In sample after 14 days of growth, decrease in the bacterial density and decrease in fluorescence can be seen. The bacteria were imaged using GFP excitation and emission wavelengths and the picture was kept in the original greyscale format.(Data by Imperial College iGEM team 2011).
We could observe that even after fourteen days of continuous growth in optimal conditions without the presence of antibiotics, our chassis still managed to express sfGFP and therefore contained the plasmid. However considerable loss of fluorescence can be seen due to progressive loss of plasmid. Also decrease in number of colonies can be observed. The lower number of colonies is due to the bacteria losing the plasmid. The loss of plasmid can be attributed to the replication rate of the bacteria being faster than the replication rate of the plasmid.
Collapse
2. Anti-holin expression
We have completed stage 1 of the assembly of the Gene Guard. A protein gel showed a clear band of the appropriate size right at the bottom of the gel when compared to a control cell. Therefore, we have sequence verified and shown that the BBa_K515104 expresses a protein of the appropriate size (Figure 3).
Figure 3: Protein gel showing an over-expression band (higher in intensity) of a small protein. Lane 1 and 2 are control and lane 3 and 4 are the anti-holin expressing cells.(Data by Imperial College London iGEM team 2011).
Collapse
3. Gene integration of the anti-holin
New since regional jamboree
Once we knew that the anti-holin construct was alright and seemed to be producing a small protein we had to excise the insert and ligate it into the CRIM plasmid. This was complicated as the high copy number plasmid was placing a heavy metabolic burden on the cells making them grow incredibly slowly (something that is not desirable for when you have to be working for a deadline). Not only that, the cells seemed to be losing the plasmid after four days on a plate in a cold room and some of the colonies managed to alter the plasmid in a way that made it lose the sfGFP and allowed the cells to grow quickly and out-compete any cells with the correct plasmid. However, after a lot of trial and error we managed to isolate eight plasmids that were possible candidates for the plasmid we wanted to integrate.
Figure 4: Gel of colonies suspected of containing anti-holin in the genome. The amplified bands of all the colonies correspond to around 300 bp which is approximately the size of the anti-holin gene. The lane furthest to the right is a control in which the anti-holin gene is contained in a pSB1C3 plasmid.(Data by Imperial College London iGEM team 2011).
In order to test these eight colonies we first did an EcoRI and PstI digest and then looked at the size of the insert compared to a control which only contained sfGFP in the CRIM. According to our results, three of the colonies contained an insert that was larger than that of the control. However, the gel red we used to stain the gel seems to do something funny with the buffer we were using to digest making all the bands run higher than they should be running. Therefore, to conclude that the plasmid contained the inserts we wanted we looked for a restriction site that is unique on the anti-holin and the CRIM. We found that the anti-holin construct contains a ClaI site that is also present on the CRIM vector. We performed this digest in another buffer and obtained the expected bands for the three colonies confirming that they contain the anti-holin on the CRIM.
Then we performed the genome integration step which involves the transformation of a cell line containing the helper plasmid[2]. In order to test whether the colonies had integrated the CRIM in the correct location we ordered the primers that were used in the original CRIM paper and performed a colony PCR on all the colonies. A few of the colonies had integrated the CRIM twice which could be clearly seen under a blue box. However, the rest of the colonies had a single integration event which could also be seen by their phenotype under blue light.
Finally, we had to prove that the anti-holin gene was now present within the colonies. In order to achieve this we performed numerous PCR's with all kinds of buffers and polymerases until we finally got one to work. Using Taq polymerase and Barnes' buffer we managed to amplify a band of the appropriate size using the same primers that we used to initially amplify the anti-holin from its origin.
Collapse
4. Cell survival of holin-endolysin construct
Collapse
5. References
[1] Tanner M et al. (2000) Molecular phylogenetic evidence for noninvasive zoonotic transmission of Staphylococcus intermedius from a canine pet to a human. Journal of Clinical Microbiology 38(4): 1628-1631.
[2] Haldimann and Wanner, 2001. Conditional-Replication, Integration, Excision, and Retrieval Plasmid-Host Systems for Gene Structure-Function Studies of Bacteria. Journal of Bacteriology 183(21) p. 6384-6393.
Collapse
M3: Assembly M3: Future Work
Retrieved from " http://2011.igem.org/Team:Imperial_College_London/test7 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
