<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Achievements"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Achievements">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Achievements&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Achievements&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Achievements&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Achievements" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Achievements</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">Project AuxIn<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London">Home</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Tour">At a Glance</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project/Background">The Problem</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Overview">M1: Phyto-Route</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Overview">M2: Auxin Xpress</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Overview">M3: Gene Guard</A></LI></UL><LI><A class="sf-with-ul" href="#">Achievements<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Achievements">Main Results</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Judging">Judging Criteria</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Europe_Jamboree">Europe Jamboree</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Overview">Informing Design</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human/Outreach">Outreach</A></LI><LI><A class="sf-with-ul" href="#">Extras<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Extras/Collaboration">Collaboration</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Protocols_General">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Software">Software</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Notebook/week1">Diary</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Brainstorming_Health">Brainstorming</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Team">Team</A></LI></DIV><H1>Main Results</H1><H1>Phyto-Route</H1><H2>1. Uptake of bacteria into plant roots</H2><P>We observed uptake of GFP-expressing bacteria into plant roots (Figure 1). The fact that (at least some) plants can actively break up bacteria and use them for nutrients was only reported last year<SUP>[1]</SUP>. This finding is of extreme importance to our project as the exposure of roots to indole 3-acetic acid will vary significantly if the uptake of the compound takes place inside the roots themselves rather than from the outside. This also presents a novel platform of indirectly engineering plants as delivery of a whole range of compounds will be possible using this method.</P><DIV class="imgbox" style="width:470px;margin:0 auto;"><P><I>Figure 1: </I>Escherichia coli<I> cells expressing superfolder GFP (sfGFP) can be seen inside an </I>Arabidopsis thaliana<I> root using confocal microscopy after overnight incubation of the plants with bacteria. Roots were washed in PBS prior to imaging to avoid &quot;false positives&quot; of bacteria adhering to the outside of the root. (Data and imaging by Imperial College London iGEM team 2011).</I></P></DIV><P>To ensure that the bacteria we imaged were inside the roots, we took stack images that were later converted to three-dimensional views of the roots containing bacteria (Video 1).</P><DIV class="vidbox" style="width:600px;margin:0 auto;"><P><I>Video 1. Confocal stack imaging showing sfGFP-expressing bacteria inside plant roots showing blue autofluorescence (data and imaging by Imperial College London iGEM team 2011).</I></P></DIV><H2>2. Chemotaxis of our chassis expressing PA2652 construct towards malate</H2><DIV class="imgbox" style="width:920px;"><P><I>Figure 2: Dependence of bacterial chemotaxis to varied malate concentrations. Cells containing PA2652 (<A href="http://partsregistry.org/Part:BBa_K515102">BBa_K515102</A>) have shown increased number of cells in capillaries with increasing malate concentration, with a peak at 1 mM. The number of cells drops sharply after 1 mM due to saturation. Negative control were cells without construct. The cell count for negative control in each of the capillaries with increasing attractant concentration has not increased. (Data collected by Imperial iGEM 2011).</I></P></DIV><P>In the Phyto-Route module of our system we have expressed the malate-reponsive chemoreceptor PA2652 in our chassis. The result of quantitative assay demonstrates chemotaxis of <I>E. coli</I> expressing PA2652 towards L (-) malic acid. The response towards different concentrations of malate has been quantified and the strongest response can be seen at 1 mM. These results show functionality of the Phyto-Route construct that enables movement of our bacteria towards plant root exudate malate. Once the bacteria are attracted towards roots, then they can be naturally taken up into the plant's roots.</P><DIV class="newouterbox"><H4 class="newtext">NEW SINCE EUROPE JAMBOREE</H4><DIV class="newinnerbox"><H2>3. Modelling capillary assay </H2><P>For chemotaxis wet lab experiments, a user interface was designed to test the feasibility of experimental protocols. Initially, an agar plug assay experiment was designed in which one would place a certain number of bacteria 6 cm away from a chemoattractant source. The modelling informed us that the time it would take for the malate diffuse the bacteria would divide too many times. This issue could not be solved by placing the bacteria closer to the source because it would make it indistinguishable from random walk.</P><DIV class="vidbox" width="780px;" style="margin:0 auto;"><P><I>Video 2. Simulation of capillary assay. This model predicts the population dynamics of the bacteria within the sample. The user interface allows the wetlab user to run the simulations with customized parameter.</I></P></DIV><P>This new model integrates our experimental results from our capillary assay, refining the initial population dynamics model used previously for the agar plug assays which were based on general chemoreceptor paramaters. This refinement in turn is further informing the design of our assay which will result in more accurate data, thereby creating a refinement loop between modelling and the wet lab. </P></DIV></DIV><H1> Auxin Xpress </H1><H2>1. Expression of indole 3-acetic acid (IAA) by our modified bacteria</H2><H3>Salkowski Assay</H3><P>Through this assay we have determined that <I>E. coli</I> transformed with the Auxin Xpress construct produce about 50 µM IAA. Salkowski is a colourimetric assay that detects IAA and produces an increasingly vibrant pink colour with higher concentrations of IAA in the sample. </P><TABLE><TBODY><TR><TD><DIV class="imgbox" style="width:520px;"><P><I>Figure 3: Results from trial 1 of Salkowski assay with cell filtrate of <I>E. coli DH5α</I> transformed with the Auxin Xpress construct. Samples were filtered through a 0.2 µm pore filter. (Data by Imperial College London iGEM team 2011).</I></P></DIV></TD><TD><DIV class="imgbox" style="width:270px;"><P><I>Figure 4: Visual results correlating with OD measurements. The eppendorf on the right contains </I>E. coli<I> DH5α with the Auxin Xpress construct and the eppendorf on the left contains control </I>E. coli DH5α<I> cells. (Data by Imperial College London iGEM team 2011).</I></P></DIV></TD></TR></TBODY></TABLE><DIV class="newouterbox"><H4 class="newtext">NEW SINCE EUROPE JAMBOREE</H4><DIV class="newinnerbox"><H3> LCMS</H3><P>We used liquid chromatography mass spectrometry to confirm that IAA was being produced by <I>E. coli</I> transformed with the Auxin Xpress construct. </P><DIV class="imgbox" style="width:840px;margin:0 auto;"><P><I>Figure 5: LCMS showing IAA peak in extract from <I>E. coli</I> engineered with the Auxin Xpress construct.</I></P></DIV></DIV></DIV><H3>Plant response to IAA</H3><P>We exposed <I>Arabidopsis</I> seedlings to our IAA-producing bacteria to observe their effect on the roots. For this, we used a reporter line that expresses YFP in response to indole 3-acetic acid. As controls, we used a culture of seedlings that was not incubated with any bacteria and a control that was indicated with <I>E. coli</I> not expressing IAA. Fluorescence was brightest in the roots incubated with IAA-producing bacteria (Video 3).</P><DIV class="vidbox" style="width:600px;margin:0 auto;"><P><I>Video 3. Stack images taken by confocal microscopy converted into three-dimensional videos of </I>Arabidopsis thaliana DR5:3VENUS<I>roots (data and imaging by Imperial College London iGEM team 2011).</I></P></DIV><P>In order to quantify the expression of fluorescence in the plants exposed to different treatments we processed the images in ImageJ and analysed fluorescence intensity in individual particles. The relative fluorescence intensity of roots exposed to our IAA-secreting bacteria is 1.93x that of roots exposed to no bacteria and 2.9x that of roots exposed to non-IAA-producing bacteria. It therefore appears that bacteria normally suppress production of IAA in root tips. However, the bacteria we have engineered to express IAA produce enough of the compound to not only overcome this limitation but to also increase expression of the reporter gene. </P><DIV class="newouterbox"><H4 class="newtext">NEW SINCE EUROPE JAMBOREE</H4><DIV class="newinnerbox"><H2>2. Tracking of bacterial metabolic activity using Dendra2</H2><P>Dendra2 is a photoconvertable protein<SUP>[2]</SUP> that normally emits green fluorescence but can be switched to emit red fluorescence with single photon activation. This conversion from green to red is irreversible. Dendra2 is therefore very useful for observing the metabolism of cells: if all Dendra2 protein in a cell is photoconverted to emit red fluorescence and overtime begins to re-emit green fluorescence, the cell must be metabolically active. </P><P>Dendra2 is now availalbe as a BioBrick. </P><P>We used Dendra2 to assess whether the bacteria taken up by <I>Arabidopsis</I> roots remain metabolically active over time. The roots infected with Dendra2 expressing <I>E. coli</I> were imaged and photoconverted under a confocal microscope, left in agarose over night and re-imaged after 24 hours. The bacteria were shown to re-emit green fluorescence inside the root after overnight incubation, proving that they are metabolically active inside the roots.</P><P><B>This result is significant in that we have proven that <I>E. coli</I> remain metabolically active inside the roots for up to 5 days which would allow IAA production by our engineered bacteria once inside the root, thereby enhancing root growth.</B></P><DIV class="imgbox" style="width:600px;margin:0 auto"><P><I>Video 4. Photoconversion of Dendra2 in </I>Arabidopsis thaliana<I> roots. After 24 hours, the bacteria were re-imaged at the same settings. Expression of green Dendra2 had increased, showing for the first time ever that the bacteria are metabolically active inside the roots (Imaging by Mark Scott for Imperial College iGEM 2011).</I></P></DIV></DIV></DIV><DIV class="newouterbox"><H4 class="newtext">NEW SINCE EUROPE JAMBOREE</H4><DIV class="newinnerbox"><H2>3. 3D simulation of the <I>Arabidopsis</I> root system</H2><P>We integrated the 3D simulation of generalised root system with the data fitting experiment results from wetlab to produce a MATLAB toolbox, with IAA concentration as the input and the 3D root system as the output. We defined a valid IAA concentration range from 10<SUP>-3</SUP> mol/L (1 mM) to 10<SUP>-14</SUP> mol/L (0.01 pM).
 
</P><DIV class="vidbox" style="width:800px;margin-left:33px;"><P><I>Video 5. This video shows how the MATLAB toolbox give the prediction of the root system after 25 days with a certain input IAA concentration.(Please select the 720HD version when you play the video to see the details clearly. Video and interface made by Imperial College London iGEM team 2011.)</I></P></DIV><P>We designed a experiment to support the data fitting and built the MATLAB program based on the principle of the Lindenmayer system. For more details, please go to our Auxin Xpress modelling page <A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Modelling">here</A>.

</P></DIV></DIV><DIV class="newouterbox"><H4 class="newtext">NEW SINCE EUROPE JAMBOREE</H4><DIV class="newinnerbox"><H2>4. IAA improving soil stability </H2><P>We carried out a soil erosion experiment using <I>Arabidopsis</I> watered with different concentrations of IAA over 6 weeks. The results from this experiment show that less soil was washed away with water when the plants were supplemented with the optimal IAA concentration. These results correlate with <I>Arabidopsis</I> experiments carried out in liquid media showing the same bell shaped curve for longest average root length with optimal IAA concentration. At IAA concentrations that are too low, root growth is not enhanced, and at concentrations too high root growth is hindered, therefore at the optimal IAA concentration maximal root growth and therefore soil stability is achieved. </P><P><B>These results are significant in that they demonstrate that the right IAA concentrations improve <I>Arabidopsis</I> root growth and prevent soil erosion, therefore providing potential application for our project in preventing desertification.</B></P><DIV class="imgbox" style="width:720px;margin:0 auto;"><P><I>Figure 6: Mass of Soil eroded in different IAA concentration, control is soil supplemented with IAA but without <I>Arabidopsis</I>. The small box is a zoomed in image of the same graph without the control and 0M IAA applied and shows visually the bell shape that correlates to previous data on root length with varying IAA concentration. (Data by Imperial College London iGEM team 2011).</I></P></DIV></DIV></DIV><H1>Gene Guard</H1><H2>1. <I>E. coli</I> survivability in soil</H2><P>To assess how well our chassis <I>Escherichia coli</I> is able to survive in soil and for how long a plasmid can be retained, we set up an experiment that consisted of inoculating <I>E. coli</I> into sterile and non-sterile soil. After six weeks of incubation, samples grown up from sterile cultures still expressed fluorescence. Cultures exhibiting <I>E. coli</I>-like morphologies that were grown from non-sterile plates had lost fluorescence (Figure 7).
</P><DIV class="imgbox" style="width:520px;margin:0 auto;"><P><I>Figure 7: Colonies recovered from filter discs and grown on LB plates containing selective antibiotics imaged using a LAS-3000 gel imager. a) Sample taken from non-sterilised soil b) Sample taken from sterilised soil  (Data by Imperial College London iGEM team 2011).</I></P></DIV><P>A restriction digest revealed that cultures from sterile and non-sterile plates that display <I>E. coli</I>-like morphologies have both retained the plasmid. Sequencing of the insert revealed a frameshift mutation in the coding region of the sfGFP gene in the colonies from non-sterile soil. Both samples will be sent off for 16S rRNA sequencing to identify the bacterial species. However, it seems very likely that our GM <I>E. coli</I> have been able to survive in soil and retain their plasmid for six weeks despite competition and selective pressure against the plasmid.</P><H2>References:</H2><P>[1] Paungfoo-Lonhienne, C. et al. (2010) Turning the table: plants consume microbes as a source of nutrients. PLoS One, <B>5(7)</B> e11915.
</P><P>[2] Gurskaya N et al. (2006) Engineering of a monomeric green-to-red photoactivatable fluorescent protein induced by blue light. Nature Biotechnology <B>24:</B> 461-465.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Imperial_College_London/Achievements">http://2011.igem.org/Team:Imperial_College_London/Achievements</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Achievements" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Achievements" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Achievements&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Achievements&amp;oldid=262526" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>