<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Project_Auxin_Modelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Project_Auxin_Modelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Project_Auxin_Modelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Project_Auxin_Modelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Project_Auxin_Modelling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Project_Auxin_Modelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Project Auxin Modelling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">Project AuxIn<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London">Home</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Tour">At a Glance</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project/Background">The Problem</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Overview">M1: Phyto-Route</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Overview">M2: Auxin Xpress</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Overview">M3: Gene Guard</A></LI></UL><LI><A class="sf-with-ul" href="#">Achievements<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Achievements">Main Results</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Judging">Judging Criteria</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Europe_Jamboree">Europe Jamboree</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Overview">Informing Design</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human/Outreach">Outreach</A></LI><LI><A class="sf-with-ul" href="#">Extras<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Extras/Collaboration">Collaboration</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Protocols_General">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Software">Software</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Notebook/week1">Diary</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Brainstorming_Health">Brainstorming</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Team">Team</A></LI></DIV><H1>Module 2: Auxin Xpress</H1><P><B>Auxin, or Indole 3-acetic acid (IAA), is a plant growth hormone which is produced by several soil bacteria. We have taken the genes encoding the IAA-producing pathway from <I>Pseudomonas savastanoi</I> and expressed them in <I>Escherichia coli</I>. Following chemotaxis towards the roots and uptake by the <A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Overview">Phyto Route module</A>, IAA expression will promote root growth with the aim of <A href="https://2011.igem.org/Team:Imperial_College_London/Project/Background">improving soil stability</A>.</B></P><DIV class="underlinemenu"><UL><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Specifications">Specifications</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Design">Design</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Modelling">Modelling</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Assembly">Assembly</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Testing">Testing &amp; Results</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Future">Future Work</A></LI></UL></DIV><H1>Modelling</H1><P style="font-size:1.3em;"><A href="#" onclick="ddaccordion.collapseall('technology'); return false">Collapse all</A>  | <A href="#" onclick="ddaccordion.expandall('technology'); return false">Expand all</A></P><DIV class="technology">1. Introduction</DIV><DIV class="thelanguage"><P>   Adequate production of the plant hormone auxin or indole-3-acetic acid (IAA) from genetically modified (GM) <I>Escherichia coli</I> could efficiently promote plant root growth. However, IAA is toxic to plants if its concentration is too high. Therefore it is important to be able to predict the IAA expression level for a given promoter, and then adjust the promoter strength to ensure the IAA produced by our <I>E.coli</I> will optimally increase root growth.</P><P>IAA increases root growth in terms of root length and number of branches. In order to study how different concentrations of IAA affect root growth patterns, modelling tools were combined with wet lab results to predict and visualise <I>Arabidopsis</I> root growth.</P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 0); return false">Collapse</A></P></DIV><DIV class="technology">2. Modelling of IAA synthesis</DIV><DIV class="thelanguage"><H3>2.1 Objective</H3><P> Determine the IAA expression level of a single <I>E. coli</I> cell with IAA promoter strength 4.536 RNA/min/μg substrate DNA, and then predict the number of bacteria to be placed in the seed coat to induce optimal root growth. </P><H3>2.2 Description</H3><P>    The genetically engineered IAA pathway involves two genes, IaaM and IaaH, both of which are constitutively expressed. The IaaM gene encodes tryptophan-2-monooxygenase (T-2-monase) that catalyses the conversion of tryptophan (Trp) to indole-3-acetamide (IAM), which is then hydrolysed to release indole-3-acetic acid (IAA) by the hydrolase iaaH<SUP>[3]</SUP>. At the same time, the synthesised IAM and IAA will competitively inhibit the enzyme activity of tryptophan-2-monooxygenase, thereby inducing a negative feedback loop on the expression of IAA. The enzymatic reactions involved in the pathway are illustrated in Figure 1. </P><DIV class="imgbox" style="width:720px;margin:0 auto;"><P><I>Figure 1: IAA synthesis pathway. (Diagram by Imperial College London iGEM team).</I></P></DIV><P>    In addition, based on research carried out by mathematical biology research group from the <A href="http://mathbio.colorado.edu/mediawiki/index.php/MBW:Modelling_the_Tryptophan_Operon" target="_blank">University of Colorado</A><SUP>[4]</SUP>, tryptophan is also negatively controlled inside bacteria. Therefore the tryptophan synthesis pathway should be integrated into the above model. Furthermore, in order to reduce the numbers of parameters, as most of the parameters in <B>Figure 1</B> are not available to us, the IAA pathway is then simplified into two Michaelis Menten equations, which are combined with the tryptophan pathway and constitutive gene expression for T-2-monase and iaaH. The whole tryptophan IAA pathway model is described in Equation 1<SUP>[5]</SUP>, and the parameters are defined in the parameters section below. 
</P><P>    In this model, we made the following assumptions:
</P><P>    (1) We neglect the short time delay due to synthesis of Trp-T-2-monase (substrate-enzyme (ES) complex), IAM-iaaH (ES complex), IAM-T-2-monase (inhibitor-enzyme (EI) complex) and IAA- T-2-monase (inhibitor-enzyme (EI) complex) and assume that these species reach their equilibrium almost instantaneously.
</P><P>    (2) The degradation rate of IAA in the dark is extremely low compared to bacterial growth rate. Therefore we used bacterial growth rate as the IAA degradation rate in this model.
</P><P>    (3) During the modelling of the IAA pathway, we found that the rate-determining species for IAA synthesis is IAM, not the enzyme iaaH, since the production of IAM is inhibited by itself and IAA.  
</P><H3>2.3 Results and discussion</H3><P><B>Fig.2</B> below represents the output of our model. 
</P><P>1. How the concentration of each protein species varies with time. 
</P><P>2. The simulation of the enzymatic reaction for each of the species, with initial concentrations of O<SUB>F</SUB> = 1.54×10<SUP>-4</SUP> µM, M<SUB>F</SUB> = 3.78×10<SUP>-4</SUP> µM , E = 0.378 µM, Trp = 4.1 µM and all others = 0 <SUP>[4]</SUP>. 
</P><P>Also, <B>Fig.2</B> shows that the IAA expression level is 72.35 µM, which means that each bacterium produces 7.24×10<SUP>-14</SUP> µmole per bacteria at steady state with bacterial volume equals to 10<SUP>-15</SUP> dm<SUP>3</SUP>. From wet <A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Testing">lab experiments</A> we know that the optimal concentration of IAA to promote root growth is 0.1 nM, and the volume of an <I>Arabidopsis</I> seed coat approximately equals 3.6×10<SUP>-9</SUP> m<SUP>3</SUP><SUP>[7]</SUP>. Therefore the number of bacteria required to be present in seed coat to maximally increase root growth of roots is 4.97 x 10<SUP>6</SUP>. </P><P>The details of the calculation are listed below:
</P><P>1. Number of moles produced by a single bacterium is 72.35 µm * 10<SUP>-18</SUP> m<SUP>3</SUP> (7.235 * 10<SUP>-17</SUP>µmol).
</P><P>2. The number of moles of IAA need to be in one seed coat is 3.6 * 10<SUP>-9</SUP> m<SUP>3</SUP> * 0.1 nmol.
</P><P>3. The number of bacteria need to be place in one seed coat is (3.6 * 10<SUP>-10</SUP> µmol) / (7.235 * 10<SUP>-17</SUP>) = 4.97 * 10<SUP>6</SUP>.
</P><DIV class="imgbox" style="width:820px;margin:0 auto;"><P><I>Figure 2 (a): The evolution of IAM vs. time. Figure 2 (b): The evolution of IAA vs. time. </I></P></DIV><P>In conclusion, we obtained the concentration of IAA produced by the IAA DNA construct (72.25 μM). From this value, we calculated the number of bacteria that would need to enter the root under ideal growth conditions and ignoring death and division of bacteria, which was found to be 4.97x10<SUP>6</SUP>  bacteria for <I>Arabidopsis</I>. This value varies due to the variation in seed size of different of plants. Since bacterial cells can be lost from the plant and not all the bacteria from the seed coat will enter the plant, the number of bacteria required in the seed coat will be higher than this. Furthermore, the modelling of root growth in the next part helped us visualise the root growth. </P><P style="text-align:right;font-size:1.3em"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 1); return false"><B>Collapse</B></A></P></DIV><DIV class="technology">3. IAA uptake and root growth</DIV><DIV class="thelanguage"><H3>3.1 Objective</H3><P> 1. Create a graphic program to demonstrate the growing process of the <I>Arabidopsis</I> root system, based on the principles of Lindenmayer system and plant physiology.</P><P>2. Use Matlab data fitting tools to develop relations between IAA concentration and growth rate and number of branches. </P><H3>3.2 Description</H3><P> We used computational tools to visulise the root growth phenomena (primary rootlength, branching, root density, etc) in different enviromental conditions, and specifically considered root order and root length. Root order describes the branching “generation” of a root system; a root without branching is defined as a zero-order root. Root growth depends on environmental factors, such as gravitation and soil heterogeneities.</P><P><B>3.2.1 Tropisms</B></P><P>    A root system starts with a single root tip of a zero-order root. Then the root grows away from the plant stem in a conical way<SUP>[10]</SUP>.
</P><P>    Root growth depends on environmental factors, such as gravitation and soil heterogeneities. Therefore, two more variables are defined to describe the plant adaptation<SUP>[10]</SUP>:
</P><P><B>σ:-</B></P><P>    - how strong the roots direction changes per 1 cm growth
</P><P>    - a larger value indicates a more deflected root and a more twisted root system
</P><P><B>N:-</B></P><P>    - the number of trials for the roots to find the optimal angles α and β for the rotation 
</P><P>    of the downward movement
</P><P>    - N can be any real number. If N = 1.5, if means that N can be either 1 or 2.</P><DIV class="imgbox" style="width:820px;margin:0 auto;"><P><I>Figure 3: Differences between root systems with different values of N and σ. (Modelling based on Daniel Leitner et al from BOKU, and codes were modified by Imperial College London iGEM team 2011).</I></P></DIV><P><B>3.2.2 Lindenmayer system and root growth modelling</B></P><P>    An L-system is a parallel rewriting system, namely a variant of a formal grammar, most famously used to model the growth processes of plant development, but also able to model the morphology of a variety of organisms, due to the two main properties: recursiveness and self-similarity<SUP>[11]</SUP>. Plant models and natural-looking organic forms are easy to define, as by increasing the recursion level the form slowly 'grows' and becomes more complex.
</P><P>    Using L-systems for generating graphical images requires that the symbols in the model refer to elements of a drawing on the computer screen. It interprets each constant in an L-system model as a turtle command.
</P><P><B>3.2.3 Root growth</B></P><P>    The modelling of IAA uptake can give predictive answers to questions such as these:
</P><P><I>&quot;What is the primary root growth rate?&quot;</I></P><P><I>&quot;What does the root system look like after a certain period of time?&quot;</I></P><P><I>&quot;How does <I>Arabidopsis</I> respond to different IAA concentrations?&quot;</I></P><P>   We modified a MATLAB program developed by Daniel Leitner's research group<SUP>[6]</SUP> to demonstrate the 3D root system based on the principles of Lindenmayer system (turtle commands) and the root growth modelling toolbox developed by Daniel Leitner et al from BOKU (Universität für Bodenkultur Wien, University of Natural Resources and Life Sciences, Vienna)<SUP>[12]</SUP>.
</P><P><B>3.2.4 Data fitting</B></P><P>    We took growth rate parameters from the literature and analysed the raw data from our wet lab to give more accurate and suitable parameters for our project. <I>Arabidopsis</I> plants were planted and root length and number of branches recorded every two days from day 0 to day 9. Root length, daily root growth rate and number of branches were plotted against time and IAA concentration.
</P><H3>3.3 Results and discussion</H3><P><B>3.3.1 Visualisation of a root system</B></P><P>    The values from the literature give the relationship between external IAA concentration and elongation of the roots as 5*10<SUP>-5</SUP> mol/L → 200 µm elongation in 30 min. The modelling parameter of growth speed is therefore 9.6*10<SUP>-3</SUP> m/day<SUP>[9]</SUP>.
</P><P>    We observed real root growth patterns and modified our simulation to give a more reliable and accurate prediction of root growth. <I>Arabidopsis</I> has a primary root with zeroth order and it is thicker than the branches. <I>Arabidopsis</I> normally grows to the depth of 20-30 cm inside the soil and branches only once. The 3D picture shown below predicts the root growth with different elongation rate (with IAA = 0.96 cm/day; without IAA = 0.46 cm/day<SUP>[10]</SUP>). They can be compared with the photo of a real root system.
</P><DIV class="imgbox" style="width:820px;margin:0 auto;"><P><I>Figure 4: 3D visalisation of the arabidopsis root system. The graph Figure 4(a)(b) shows the demonstration of the </I>Arabidopsis<I> root system in different IAA concentration conditions with a growth time of 20 days as our simulation results. Figure 4(c)(d) are the actual photos of real </I>Arabidopsis<I> plants taken from the literature and our wet lab. (Modelling based on Daniel Leitner et al from BOKU, and codes were modified by Imperial College London iGEM team 2011).</I></P></DIV><DIV class="imgbox" style="width:850px;margin:0 auto;"><P><I>Figure 5: The graphic analysis of the 3D root system. By taking cross and longitudinal sections of the root system at different positions, Figure 5(a) is the root length distribution (by fraction) against the root depth. Figure 5(c) is the surface area distribution map. This graph will give guidance for placing GM </I>E. coli<I> inside the soil. </I>E. coli<I> should be placed at the depth with the maximal root surface area (6 cm) inside the soil for the maximal uptake efficiency. (Modelling based on Daniel Leitner et al from BOKU, and codes were modified by Imperial College London iGEM team 2011).</I></P></DIV><DIV class="newouterbox"><H4 class="newtext">NEW SINCE EUROPE JAMBOREE</H4><DIV class="newinnerbox"><P><B>3.3.2 Data fitting</B></P><P>  We designed an experiment to support the data fitting and tested it before the European Regional Jamboree. The results are integrated into Section 2. The effect of IAA on <I>Arabidopsis</I> roots. Click <A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Testing">here</A> to see the details of the first data-fitting experiment.

</P><P>  After the European Regional Jamboree, we improved the protocol to include the following considerations and improvements:-
</P><P>  a) The degradation of IAA:
</P><P>     The wet lab results shows that the IAA concentration in phytogel media is negligible after 9 days, the degradation of IAA is an important factor when root growth is modelled. The IAA concentration can be calculated everyday using the equation: <B>remaining IAA (%) = 95.75 - 2.9x + 0.05x<SUP>2</SUP></B>, where x is the number of days after the auxin has been added.<SUP>[14]</SUP></P><P>  b) The number of samples:
</P><P>     We prepared 20 replica for each auxin concentration to reduce the error.
</P><P>  c) The measurements technique:
</P><P>     Since the uptake rate and the IAA degradation rate changes rapidly, we measured <I>Arabidopsis</I> roots in phytogel everyday instead of once every three days. Therefore we have correlated the daily growth rate with IAA concentration. 
</P><DIV class="imgbox" style="width:800px;margin:0 auto;"><P><I>Figure 6: Daily growth rate of</I> Arabidopsis <I>against the logarithms of the additional IAA concentration. The relation can be described by the Gaussian equation.</I></P></DIV><P>Then, the mathematic model we got from the data fitting experiments was integrated with our 3D root simulation system. This produced a MATLAB toolbox with IAA concentration as the input and the 3D root model as the output. We defined a valid IAA concentration range from 10<SUP>-3</SUP> mol/L (1 mM) to 10<SUP>-14</SUP> mol/L (0.01 pM) for the simulation based on the following reasons:
</P><P>   a) When the IAA concentration is higher than 10<SUP>-3</SUP> mol/L (1 mM), the <I>Arabidopsis</I> will be killed. 
</P><P>b) When the IAA concentration is lower than 10<SUP>-14</SUP> mol/L (0.01 pM), the <I>Arabidopsis</I> will not be affected since the IAA produced by <I>Arabidopsis</I> itself is much more. Also, it is reasonable to define a limitation of IAA dilution to keep the percentage error low.
</P><P>c) The Gaussian equation can only describe the relation between the growth rate and the IAA concentration within this range accurately.
</P><P>Our wetlab results shows two different optimal IAA concentrations for the <I>Arabidopsis</I> root growth, 0.1nM in liquid media and 1pM in phytogel. The data fitting experiment is done inside phytogel.
</P><DIV class="imgbox" style="width:800px;margin:0 auto;"><P><I>Figure 7: The results of the simulation with different input.</I></P></DIV><P>The video below shows how the MATLAB toolbox predicting the root system after 25 days given a certain IAA concentration.The results simulated are supported by our wetlab experiments.(Please select the 720HD version when you play the video to see the details clearly.)

</P></DIV></DIV><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 2); return false">Collapse</A></P></DIV><DIV class="technology">4. Parameters</DIV><DIV class="thelanguage"><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K515010">Link to the Registry Page for details of Pveg2 promoter</A></P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 3); return false">Collapse</A></P></DIV><DIV class="technology">5. Matlab code</DIV><DIV class="thelanguage"><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 4); return false">Collapse</A></P></DIV><DIV class="technology">6. References</DIV><DIV class="thelanguage"><P>   [1] Hutcheson SW, Kosuge T (1985) Regulation of 3-indole acetic acid production in <I>Pseudomonas syringae pv savastanoi</I>: Purification and properties of tryptophan 2-monooxygenase. <I>The Journal of Biological Chemistry</I><B>260(10):</B> 6281-6287.
</P><P>   [2] Saepen S, Vanderleyden J, Remans R (2007) Indole-3-acetic acid in microbial and microorganism-plant signaling. <I>FEMS Microbiology Reviews</I><B>31(4)</B> 425-448.
</P><P>   [3] Cheng Y, Dai C, Zhao Y (2006), Auxin biosynthesis by the YUCCA flavin monooxygenases controls the formation of floral organs and vascular tissues in <I>Arabidopsis</I>. <I>Genes &amp; Dev</I> 1790-199. Doi: 10.1101/gad.1415106
</P><P>   [4] http://mathbio.colorado.edu/mediawiki/index.php/MBW:Modelling_the_Tryptophan_Operon
</P><P>   [5] Santillan M, Mackey MC (2001) Dynamic regulation of the tryptophan operon: a modeling study and comparison with experimental data. <I>PNAS</I><B>98(4)</B> 1364-1369.
</P><P>   [6] http://www.ccdb.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
</P><P>   [7] http://www.seedgenes.org/Tutorial.html
</P><P>   [8] Leitner D, Klepsch S, Bodner G, Schnepf A (2009) A dynamic root system growth model based on L-Systems: tropisms and coupling to nutrient uptake from soil. <I>Plant and Soil</I><B>332(1-2):</B> 177-192.
</P><P>   [9] Rayle DL, Evans ML, Hertel L (1970) Action of auxin on cell elongation. <I>Proc Nat Acd Sci USA</I><B>65(1):</B> 184-191.
</P><P>   [10] Beemster GTS, Baskin TI (1998) Analysis of cell division and elongation underlying the developmental acceleration of root growth in <I>Arabidopsis thaliana</I>. <I>Plant Physiology</I><B>116(4):</B> 1515-1526.
</P><P>   [11] Hu H, Cao X, Lin B (2003) Three dimensional Lindenmayer system. <I>The Journal of Engineering Graphics</I>.
</P><P>   [12] Leitner D, Schnepf A, Klepsch S, Roose T (2009) Comparison of nutrient uptake between 3-dimensional simulation and an averaged root system model. <I>Plant Biosystems</I><B>144(2):</B> 443-447.
</P><P>   [13] Brenda: The Comprehensive Enzyme Information System (http://www.brenda-enzymes.org/).
</P><P>   [14] James D, Stephen K,  Robert M (1986) The Effect of Salt Concentration on Auxin Stability in Culture
Media. <I>Plant Physiology</I><B>81:</B> 934-936.
</P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 5); return false">Collapse</A></P></DIV><H2><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Design" style="text-decoration:none;color:#728F1D;float:left;">
M2: Design
</A><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Assembly" style="text-decoration:none;color:#728F1D;float:right;">
M2: Assembly
</A></H2><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Modelling">http://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Modelling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Project_Auxin_Modelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Project_Auxin_Modelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Project_Auxin_Modelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Project_Auxin_Modelling&amp;oldid=262019" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>