"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College London/Project Gene Assembly
From 2011.igem.org
Project AuxIn »
Home
At a Glance
The Problem
M1: Phyto-Route
M2: Auxin Xpress
M3: Gene Guard
Achievements »
Main Results
Data
Judging Criteria
Europe Jamboree
Human Practice »
Informing Design
Safety
Outreach
Extras »
Collaboration
Protocols
Software
Diary
Brainstorming
Team
Module 3: Gene Guard
Containment is a serious issue concerning the release of genetically modified organisms (GMOs) into the environment. To prevent horizontal gene transfer of the genes we are expressing in our chassis, we have developed a system based on the genes encoding holin, anti-holin and endolysin. We are engineering anti-holin into the genome of our chassis, where it acts as an anti-toxin, and holin and endolysin on plasmid DNA. In the event of horizontal gene transfer with a soil bacterium, holin and endolysin will be transferred without anti-holin, rendering the recipient cell non-viable and effectively containing the Auxin Xpress and Phyto-Route genes in our chassis.
Overview
Specifications
Design
Modelling
Assembly
Testing & Results
Future Work
Assembly
The assembly of this module is the most challenging of all the modules. Not only are we starting it the latest, but we will be using standard BioBrick assembly because the genes we are using are already in the registry. The assembly of this module will go through three stages which must be completed sequentially. First we need to create the anti-holin construct with the insulator and strong RBS. Then we must insert the anti-holin sequence into the CRIM plasmid and integrate this into the genome. Once the anti-holin is in the genome we will build the complete holin-endolysin construct that will only be maintained within our modified strain of E. coli.
Figure 1: Diagram of the proposed assembly of the Gene Guard. (Picture by Imperial College iGEM team 2011).
Figure 2: This is an agarose gel showing a EcoRI + SpeI digest of one of the colonies on the anti-holin In-Fusion plate that showed promising results in the colony PCR's. Lane 1 is the ladder, lane 2 is colony 2 and Lane 3 is the ladder. (Data by Imperial College iGEM team 2011).
Stage 1: Anti-holin construct assembly
The first step of assembly will require us to place the anti-holin from the BBa_K112808 BioBrick under the J23100 promoter in BBa_K398500 . In order to perform this step we will be using PCR to add our new RBS and insulator sequence into the construct. The primers were designed to have 15 bp overhangs for In-Fusion. The PCR step was incredibly challenging due to the long non-homologous regions within the primers. This cost us almost two weeks. However, after a lot of hardship and numerous In-Fusion and CPEC attempts we managed to assemble this construct.
Stage 2: Integration of anti-holin into Escherichia coli genome
The next stage of the assembly is to integrate the insert from our anti-holin construct into the genome of E. coli. In order to do this we will ligate the insert into the CRIM plasmid. Once inserted we must transform the plasmid into pir+ cells in order to replicate the plasmid. Once we have a good yield of the plasmid we will co-transform the CRIM plasmid into cells containing the accessory plasmid and we will select for the colonies that contain the CRIM plasmid in the genome.
During this process we discovered that the CRIM plasmid SpeI site is non-functional due to the non-RFC10 insertion of a terminator. This was also discovered within our sfGFP construct which we had to repair via PCR, phosphorylation and ligation of the plasmid. This has altered our plans to run two insertions in parallel (one in the suffix and one in the prefix) to see if there would be any difference in anti-holin expression levels.
Stage 3: Holin-endolysin construct assembly
In order to assemble this construct we inserted the J23103 promoter itno BBa_K093005 . Then we PCR'd out the holin and endolysin genes from BBa_K112808 and added an XbaI site and suffix into the primers. The plan was to then ligate the J23103-RFP (BBa_K093005) construct into a pSB1C3 plasmid and then ligate the holin-endolysin PCR product after it. We have also recently discovered that the original BioBrick has a "barcode" that contains an RBS. This will be problematic in our assembly and will have to be inverse PCR'd.
Figure 3: Anti-holin construct (BBa_K515004) integrated into the genome. (Diagram by Imperial College London iGEM team 2011).
Figure 4: Holin/endolysin construct (BBa_K515106) (Diagram by Imperial College London iGEM team 2011).
M3: Modelling M3: Testing & Results
Retrieved from " http://2011.igem.org/Team:Imperial_College_London/Project_Gene_Assembly "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
