<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Project_Chemotaxis_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Project/Chemotaxis/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Project/Chemotaxis/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Project/Chemotaxis/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Project/Chemotaxis/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Project/Chemotaxis/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Project/Chemotaxis/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">Project AuxIn<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London">Home</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Tour">At a Glance</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project/Background">The Problem</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Overview">M1: Phyto-Route</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Overview">M2: Auxin Xpress</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Overview">M3: Gene Guard</A></LI></UL><LI><A class="sf-with-ul" href="#">Achievements<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Achievements">Main Results</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Judging">Judging Criteria</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Europe_Jamboree">Europe Jamboree</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Overview">Informing Design</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human/Outreach">Outreach</A></LI><LI><A class="sf-with-ul" href="#">Extras<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Extras/Collaboration">Collaboration</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Protocols_General">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Software">Software</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Notebook/week1">Diary</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Brainstorming_Health">Brainstorming</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Team">Team</A></LI></DIV><H1>Chemotaxis Results</H1><H1>Modelling</H1><H2>Introduction</H2><P>E.coli is a motile strain of bacteria, which is to say it can swim. It is able to do so by rotating its flagellum, which is a rotating tentacle like structure on the outside of cell.
   Chemotaxis is the movement up concentration gradient of chemoattractants (i.e. malate in our project) and away from poisons. E.coli is too small to detect any concentration
   gradient between the two ends of itself, and so they must randomly head in any direction and then compare the new chemoattractant concentration at new point to the previous
   3-4s point. Its motion is described by ‘runs’ and ‘tumbles’, runs refer to a smooth, straight line movement for a number of seconds, while tumble referring to reorientation of 
   bacteria [1]. Chemoattractant increases transiently raise the probability of ‘tumble’ (or bias), and then a sensory adaptation process returns the bias to baseline, enabling the cell
   to detect and respond to further concentration changes.  The response to a small step change in chemoattractant concentration in a spatially uniform environment increase the  
   response time occurs over a 2- to 4- s time span [2]. Saturating changes in chemoattractant can increase the response time to several minutes. </P><P>Many bacterial chemoreceptors belong to a family of transmemberane methyl-accepting chemotaxis proteins (MCPs) [3]. Each chemoreceptors on the bacterium has a periplasmic binding domain and a cytoplasmic signaling domain that communicates with the flagellar motors via a phosphorelay sequence involving the CheA, CheY, and CheZ proteins. Modeling of this chemotaxis pathway in single cell is very important, as it will help us to determine, with certain numbers of chemoreceptors, the threshold of chemoattractant concentration where the bacterium is able to detect and the saturation level of chemoattractant where the all the receptors on the bacterium are occupies.  A it is believed that the auxin should be kept at a very close region around the root (0.25 cm [4]), therefore it is very important to obtain the number of chemoreceptors needed on individual bacterium that enables the it to stay close to the plant root/seed from modeling. </P><P>In addition, modeling of chemotaxis of bacteria population is also valuable for us to capture the overview of movement of bacteria around the plant root; therefore it can potentially inform our project about how and where we can place our bacteria.  Under experiment condition, the chemoattractant diffuses all the time from the source. However, in real soil, the root produces malate all the time, therefore we assume that the distribution of chemoattractant outside the root is steady and time-independent. Hence, the modeling of bacteria population chemotaxis will be built with different patterns of chemoattractant distributions.</P><H2> Chemotaxis Pathway </H2><P>The chemotaxis pathway in E.Coli is demonstrated in Figure 1. MCPs form stable ternary complexes with the CheA and CheW proteins to generate signals that control the direction of rotation of the flagellar motors [5].  The signaling currency is in the form of phosphoryl groups (p), made available to the CheY and CheB effector proteins through autophosphorylation of CheA[1].CheY-p initiates flagellar responses by interacting with the motor to enance the probability of ‘run’ [1]. CheB-p is part of a sensory adaptation circuit that terminates motor responses [1]. MCP complexes have two alternative CheA autokinase activity; When the receptor is not occupied by chemoattractant, the receptor stimulates CheA activity [1]. The overall flux of phosphoryl groups to inhibited and stimulated states. Changes in attractant concentration shift this distribution, triggering a flagellar response [1]. The ensuing changes in CheB phosphorylation state alter its methylesterase activity, producing a net change in MCP methylation state that cancels the stimulus signal [1]. Therefore, studying of methylation level, phosphorylation level of CheB and CheY are important to understand chemotaxis of single cell. </P><P><B>Figure 1[1]: Chemotaxis signaling conponents  and oathways for E.Coli.</B></P><P>The model based on Spiro et al. (1997) [1] was used to identify candidates of the chemotaxis receptor pathway. The methylation level of receptors, phosphorylation level of CheY and CheB were studied from Spiro’s model(Figure 2-Figue3). With receptor concentration per cell equals to 8x10<SUP>-6</SUP>mole/L, the lower threshold concentration of chemoattractant that the bacterium start to detect is 10<SUP>-8</SUP>mole/L. The saturation level is 10<SUP>-6</SUP>mole/L  in which concentration or higher the bacteria’s movements to chemoattractant are less efficient. In addition, the relation between chemoreceptor concentration and the lower threshold and staturation level were also studied (Figure 5). </P><P>The quantity that links the CheY-p concentration with the type of motion (run vs. tumble) is called bias. It is defined as the fraction of time spent on the directed movement with respect to the total movement time. The relative concentration  of CheYp is converted into motor bias using a Hill function (Euqation 1)[5]. A graph describes bias against CheY-p concentration was shown in Figure 5.</P><P><B>Figure 2: [Phosphorylated CheY]/ [CheY]</B></P><P><B>Figure 3: [Phosphorylated CheY]/ [CheY]</B></P><P><B>Figure 4: [Phosphorylated CheB]/ [CheB]</B></P><P><B>Figure 6: The dependency of Bias on the concentration of CheY-p</B></P><H2>Simulation of chemotaxis of bacteria population</H2><P>The part of modeling focused on creating the movement model of bacteria population for chemotaxis. In order to accurately built this model, the following assumptions are made based on literature: </P><P>1)	During the directed movement phase, the mean speed of an E. coli equals 24.1 μm/s, varying speed between 17.3 μm/s  and 30.9 μm/s [7]. Whereas during the tumbling phase, the speed is significantly smaller and can be neglected. </P><P>2)	E.Coli usually take previous second as their basis on deciding whether the concentration has increased or not. Therefore, in our model the bacteria will be able to compare the concentration of chemattractant at t second and t-1 second. </P><P>3)	In our model, we ignored that E.Coli do not travel in straight line during run, but take curved paths due to unequal firing of flagella. </P><P>4)	Our model did not consider the size changing and dividing of bacteria. And the tendency of bacteria congregate into small area due to qurum sensing is also ignored. </P><H3>Model Designing</H3><P>In chemotaxis, receptors sensing an increase in the concentration of chemoattractant send a signal that suppresses tumbling, and, simultaneously, the receptor becomes more highly methylated. Conversely, a decrease in the chemoattractant concentration increases the tumble frequency and causes receptor demethylation. The tumbling frequency is approximately 1 Hertz, and decreased to almost zero as he bacteria move up a chemtoatic gradient [5]. </P><P>In the model, the bacteria should be able compare the chemoattractant concentration at current point to the concentration at previous second. If the concentration decreases (i.e. C_t1-C_t2  ≤0), the bacteria will tumble with frequency 1 Hertz. If the concentration increases (C_t1-C_t2  &gt;0), the tumble frequency decreases, and hence the probability of tumbling decreases. From equation 10 in ref [6], we known that even if C_t1-C_t2  &gt;0, the probability of tumbling could decreases to 39%. Therefore, we can conclude the above description into the following statement [8]: </P><H3>Chemotaxis of bacteria population under laboratory conditions</H3><P>Under laboratory condition, the chemoattractant diffuses from the source, hence the distribution pattern of chemoattratctant changes with time. In this case, error function (Equation 2) was used to describe the non-steady chemoattractant distribution. The simulation of chemotaxis of 100 bacteria placed 6cm away from the 5mM malate is shown in the movie below. </P><H3>Chemotaxis of bacteria population in Soil</H3><P>Malate is used as the chemoattractant in our project, the malate is constantly secreted in the root tip, and the concentration is 0.3mM[9]. In this case, the malate source is always replenished due to continuous secretion from the root tip, the distribution pattern can be considered as steady (i.e. independent of time), and steady state Keler-Segel model was used to demonstrate this distribution (Equation 3 and Equation 5). The distribution was displayed in Figure 7.  And Figure 8 shows the position of lower threshold where the bacteria start to response to malate and the saturation level where the chemoreceptors start to loss efficiency.  Finally, the animation of bacterial chemotaxis in steady chemoattractant distribution is demonstrated in video below. </P><P><B>Figure 7: Malate distribution (1D)</B></P><P><B>Figure 8: Malate distribution. Red: malate concentration = 10<SUP>-8</SUP>mol/L,Blue: malate concentration = 10<SUP>-6</SUP>mol/L</B></P><H4>Reference</H4><P>[1] Peter A. Spiro, John S. Parkinson, Hands G. Othmer. ‘A model of exciatation and adaptation in bacterial chemotaxis’. Proc. Natl. Acd. Sci. USA, Vol. 94, pp. 7263-7268, July 1997. Biochemistry</P><P>[2] Blocks S. M., Segall J. E. and Berg H.C. (1982) Cell 31, 215-226.</P><P>[3] Stock J. B. and Surette M. G. (1996) ‘Escherichia coli and salmonella: Cellular and molecular biology’. Am. Soc. Microbiol., Washington, DC). </P><P>[4] Andrea Schnepf. ‘3D simulation of nutrient uptake’ </P><P>[5] M D Levin, C J Morton-Firth, W N Abouhamad, R B Bourret, and D Bray, ‘Origins of individual swimming behavior in bacteria.’</P><P>[6] Vladimirov N, Lovdok L, Lebiedz D, Sourjik V (2008) ‘Dependence of Bacterial Chemotaxis on Gradient Shape and Adaptation Rate’ PloS Comput Biol 4(12): e1000242. Doi:10.1371/journal.pcb1.1000242. </P><P>[7] Zenwen Liu and K. Papadopoulos. ‘Unidirectional Motility of Escherichia coli’. 
APPLIED AND ENVIRONMENTAL MICROBIOLOGY, Oct. 1995, p. 3567–3572 Vol. 61, No. 100099-2240/95/$04.0010 Copyright q 1995, American Society for Microbiology</P><P>[8] https://2009.igem.org/Team:Aberdeen_Scotland/chemotaxis</P><P>[9] Enrico Martinoia and Doris  Rentsch. ‘Malate Compartmentation-Responses to a Complex Metabolism’ Annual Review of Plant Physiology and Plant Molecular Biology Vol. 45: 447-467 (Volume publication date June 1994) DOI: 10.1146/annurev.pp.45.060194.002311</P><P>[10] C.J. Brokaw. ‘Chemotaxis of bracken spermatozoids: Implications of electrochemical orientation’. </P><P>[11] D.L.Jones, A.M. Prabowo, L.V.Kochian, ‘Kinetics of malate transport and decomposition in acid soils and isolated bacterial populations the effect of microorganisms on root exudation of malate under Al stress.’ Plant and Soil 182:239-247, 1996. 
</P><H2>Malate concentration distribution</H2><P>For our project, malate is the chemoattractant that results in the movement of E.coli. In this section, we will first model the concentration distribution of the chemoattractant, malate in the soil. Then, we will model the bacteria concentration pattern as a result of this distribution of malate. Finally, we will infer some useful information by analysing the results of the modelling.</P><P>We will model the concentration distributions of malate and bacteria using the Keller-Segel model which is governed by the two equations shown below. Solving the equations will give the concentration distributions of the malate and the bacteria respectively

</P><P> ---------------------------------------------------------------------------------------------(1)</P><P> -------------------------------------------------------(2)</P><P>s = concentration of chemoattractant
D = diffusion coefficient of chemoattractant
f = degradation of chemoattractant
b = number concentration of bacteria
µ = bacterial diffusion coefficient (how fast bacteria spread)
χ = chemotactic coefficient (how sensitive bacteria are)
g = bacterial cell growth
h = bacterial cell death
</P><P>The values of the above parameters for E. coli are shown in the following table. These values will be used for the modelling. 

</P><P>Parameter description</P><P>Notation</P><P>Value</P><P>Initial bacterial concentration</P><P>b0</P><P>10^8 cells/ml</P><P>Initial attractant concentration</P><P>s0</P><P>0.1 mM or 0.1 mol/m3</P><P>Bacterial diffusion coefficient</P><P>µ</P><P>1.5*10-5 cm2/s</P><P>Bacterial chemotactic coefficient</P><P>χ</P><P>1.5-75*10-5 cm2/s</P><P>Attractant diffusion coefficient</P><P>D</P><P>10-5 cm2/s</P><P>Reference: Overview  of Mathematical Approaches Used to Model Bacterial Chemotaxis II: Bacterial  Populations
  The assumptions that we have made are as follow:</P><UL><LI>The entire root system is assumed to take the  shape of a long cylinder. Hence, a cylindrical coordinate system will be used.</LI><LI>The system is axisymmetric and there is no  variation along the vertical length of the root. Hence,  </LI><LI>The system has reached steady state and is  time-independent. Hence,  </LI><LI>Degradation of chemoattractant is first order  and is described by f = ks where k is the degradation rate of the  chemoattractant.</LI><LI>Bacterial cell growth and death are neglected.  Hence, g(b,s) = h(b,s) = 0</LI></UL><P> Applying the assumptions above, equation (1) becomes</P><P> Rearranging, -------------------------------------------------(3)
  Equation (3) is in the form of the modified Bessel equation.  Hence, the solution of equation (3) is given by,
  Where K0 is the modified Bessel function.
  Since it is unrealistic for the concentration to increase to  infinity, A=0. And applying the boundary condition, the solution becomes,  -----------------------------------------------------(4)</P><P>The modeling result show that the  steady-state pattern of malate distribution. The concentration of malate is a  variable against the distance</P><H1>Assembly</H1><P>The receptor genes were synthesised in two fragments. In order to assemble the construct, we
-use CPEC to combine the two fragments</P><P>-Cells have been transformed with backbone plasmid pSB1C3 carrying biobrick BBa_K398500, with constitutive promoter J23100, protocol for transformation can be found here.</P><P>-transformation of PA2652 fragments into E coli competent cells. Fragment numbers 22 &amp; 23.
-miniprep of the 22 &amp; 23 cells to obtain DNA.
-gibson assembly of 22&amp;23 fragments.
-Due to this we have transformed 5a strain with a high copy plasmid containing ampicillin and kanamycin resistance (AK3 backbone)and sfGFP. These cells have been numbered 17.</P><H2>26th of August</H2><P> Colony PCR results of CPEC assembled PA2652 construct look promising! Will know for sure when sequencing results arrive next week </P><P>Gel 12: Colony PCR of 19 colonies picked from cells transformed with CPEC assembled PA2652 construct, about half have the correct size insert, these will be inoculated and miniprepped. Gel 13: Two more colony PCRs which were unsuccesfull, followed by five colony PCRs from negative control colonies (assembly of backbone vector without insert)showing backbone vector (6a) in one. The next well is a positive control colony PCR with plasmid 6a. The final two wells are analytical PCRs of the CPEC assembly and negative control with sequencing primers showing the correct size band for the assembled insert.</P><H1>Testing</H1><P>Testing for chemotaxis can be split into qualitative and quantitative assays. Qualitative assays involve putting engineered <I>E. coli</I> and an attractant onto semi-solid agar plates and observe the movement of the microbes. If they can be observed to move towards the attractant source, they are likely to be attracted to the ligand. In quantitative assays, capillaries are filled with different concentrations of the attractant malate. Positive controls are provided by filling identical capillaries with different concentrations of serine, which <I>E. coli</I> naturally move towards. Negative controls are provided by filling capillaries with media that does not contain a source of attractant. The amount of bacteria that swim into each capillary is evaluated by FACS.</P><P> For simplicity, we will be working with Arabidopsis to observe the uptake of bacteria into plant roots.
Arabidopsis thaliana is a common plant model organism. It belongs to the mustard family and fulfils many important requirements for model organisms. As such, its genome has been almost completely sequenced and replicates quickly, producing a large number of seeds. It is easily transformed and many different mutant strains have been constructed to study different aspects (National Institute of Health, no date). While Arabidopsis may not represent plant populations naturally occurring in arid areas threatened by desertification, it is a handy model organism we will be using to study the effect of auxin on roots, observe chemotaxis towards them and look at uptake of bacteria into the roots.</P><P>We will be using Arabidopsis to look at the uptake of our engineered bacteria into the plants. For this, we will be using wild type Arabidopsis and E. coli that constitutively express green fluorescent protein. The natural fluorescence produced by plant roots and green fluorescence produced by the bacteria can be used to image the uptake of bacteria using confocal microscopy.</P><H2>1. Testing for chemotaxis towards malate</H2><P>Experiments involving chemotaxis can be split to two categories, qualitative and quantitative. In the qualitative experiments, we are able to show that bacteria, which we study do or do not move towards a source, however it does not inform us at all about the cell count.</P><H2>5th of August - Agar plug in assay</H2><P>The simplest method for studying chemotaxis is to use agar plug in assay, which is a simple experimental method to show bacterial movement towards a localised chemical source.</P><P>Experiment is performed as follows, bacteria are added into the plate on one side, and the attractant to the other. Result is then the shape of the formed colony which grows into distorted elipse shape within the agar if the attractant is present, however colony shape remains circular if no attractant was added.</P><P> In our experiment we have used <I>5-α Escherichia coli</I> cells. Cells used as positive control were transformed with pSB1C3 plasmid backbone, which conferred chloramphenicol resistance. These cells have not been modified in any other way and were used to show <I>5-α E. coli</I> are capable of chemotaxis, using serine as attractant, which acts as a ligand for Tsr, an endogenous chemoreceptor. <I>5-α Escherichia coli</I> competent cells were also transformed with pRK415 a working plasmid with mcpS gene and tetracycline resistance. These cells express mcpS gene from <I>P. putida KT2440</I>, which is chemoreceptor with malate as its ligand, and therefore malate was used as attractant for these cells during agar plug in assay. As a negative control, no attractant was added into the plate.</P><P> Amount of attractant added contributes greatly towards the result, as too little attractant added will not stimulate bacteria to move towards source, however too much attractant will result in the saturation of the medium and the bacteria will not chemotax towards source. Values (concentration of attractant, distance between colony and attractant at start etc.), necessary to perform this experiment have been worked out using modelling. </P>
     These are the values of attractant concentration we have used, when 5µl sample was added into the plate:<TABLE class="MsoTableGrid" style="border-collapse:collapse;border:none;mso-border-alt:solid windowtext .5pt;
 mso-yfti-tbllook:1184;mso-padding-alt:0cm 5.4pt 0cm 5.4pt" border="1" cellpadding="0" cellspacing="0"><TBODY><TR style="mso-yfti-irow:0;mso-yfti-firstrow:yes"><TD style="width:74.35pt;border:solid windowtext 1.0pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="99"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal" align="center"><SPAN style="mso-ansi-language:SK"> </SPAN></P></TD><TD style="width:74.45pt;border:solid windowtext 1.0pt;
  border-left:none;mso-border-left-alt:solid windowtext .5pt;mso-border-alt:
  solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="99"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal" align="center"><SPAN style="mso-ansi-language:SK">1</SPAN></P></TD><TD style="width:83.25pt;border:solid windowtext 1.0pt;
  border-left:none;mso-border-left-alt:solid windowtext .5pt;mso-border-alt:
  solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="111"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal" align="center"><SPAN style="mso-ansi-language:SK">2</SPAN></P></TD><TD style="width:83.25pt;border:solid windowtext 1.0pt;
  border-left:none;mso-border-left-alt:solid windowtext .5pt;mso-border-alt:
  solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="111"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal" align="center"><SPAN style="mso-ansi-language:SK">3</SPAN></P></TD><TD style="width:74.55pt;border:solid windowtext 1.0pt;
  border-left:none;mso-border-left-alt:solid windowtext .5pt;mso-border-alt:
  solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="99"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal" align="center"><SPAN style="mso-ansi-language:SK">4</SPAN></P></TD><TD style="width:74.55pt;border:solid windowtext 1.0pt;
  border-left:none;mso-border-left-alt:solid windowtext .5pt;mso-border-alt:
  solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="99"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal" align="center"><SPAN style="mso-ansi-language:SK">5</SPAN></P></TD></TR><TR style="mso-yfti-irow:1"><TD style="width:74.35pt;border:solid windowtext 1.0pt;
  border-top:none;mso-border-top-alt:solid windowtext .5pt;mso-border-alt:solid windowtext .5pt;
  padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="99"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="mso-ansi-language:SK">Concentration</SPAN></P></TD><TD style="width:74.45pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="99"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="mso-ansi-language:SK">1.884x10<SUP>-3</SUP> mol/L</SPAN></P></TD><TD style="width:83.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="111"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="mso-ansi-language:SK">1.884x10<SUP>-2</SUP> mol/L</SPAN></P></TD><TD style="width:83.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="111"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="mso-ansi-language:SK">1.884x10<SUP>-1</SUP> mol/L</SPAN></P></TD><TD style="width:74.55pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="99"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="mso-ansi-language:SK">0.5 </SPAN></P><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="mso-ansi-language:SK">mol/L</SPAN></P></TD><TD style="width:74.55pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="99"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="mso-ansi-language:SK">0.75</SPAN></P><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="mso-ansi-language:SK">mol/L</SPAN></P></TD></TR><TR style="mso-yfti-irow:2;mso-yfti-lastrow:yes"><TD style="width:74.35pt;border:solid windowtext 1.0pt;
  border-top:none;mso-border-top-alt:solid windowtext .5pt;mso-border-alt:solid windowtext .5pt;
  padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="99"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="mso-ansi-language:SK">Amount</SPAN></P></TD><TD style="width:74.45pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="99"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="mso-ansi-language:SK">9.42x10<SUP>-9</SUP> mol</SPAN></P></TD><TD style="width:83.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="111"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="mso-ansi-language:SK">9.42x10<SUP>-8</SUP> mol</SPAN></P></TD><TD style="width:83.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="111"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="mso-ansi-language:SK">9.42x10<SUP>-7</SUP> mol</SPAN></P></TD><TD style="width:74.55pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="99"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="mso-ansi-language:SK">2.5x10<SUP>-7</SUP> mol</SPAN></P></TD><TD style="width:74.55pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-left-alt:solid windowtext .5pt;
  mso-border-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="99"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="mso-ansi-language:SK">3.75x10<SUP>-6</SUP> mol</SPAN></P></TD></TR></TBODY></TABLE><P>Note: Concentration values for 4 &amp; 5 were not obtained from models. They were overestimations to show possible saturation of the medium with attractant. </P><P><B>Results</B>:</P><P><I>Figure 1: Negative control pictured for the attraction E. coli cells, with added attractant malate. Rising consentrations of malate were tested, 0 M, 10<SUP>-5</SUP> M, 10 <SUP>-4</SUP> M, 10 <SUP>-3</SUP> M, 10 <SUP>-2</SUP> M, 10 <SUP>-1</SUP> M. Circular colonies are observed with rising concentrations.</I></P><P><I>Figure 2: Negative control pictured for the attraction E. coli cells, with added attractant malate. Rising consentrations of malate in milimolar range were tested, 0 mM, 5 mM, 10 mM, 15 mM, 20 mM, 25 mM. Circular colonies are observed with rising concentrations.</I></P><P><I>Figure 3: Positive control pictured for the attraction E. coli cells, with added attractant serine, which is compatible with native E. coli chemoreceptors. Rising consentrations of serine were tested, 0 M, 10<SUP>-5</SUP> M, 10 <SUP>-4</SUP> M, 10 <SUP>-3</SUP> M, 10 <SUP>-2</SUP> M, 10 <SUP>-1</SUP> M. Circular colonies are observed with 0 M &amp; 10<SUP>-5</SUP> M concentrations, result for 10 <SUP>-4</SUP> M has been rendered void due to mishandling with semi - solid agar. Possible elipse shape of the 10 <SUP>-3</SUP> M sample can be observed. Directed colony shape towards the attractant source is observed at 10 <SUP>-2</SUP> M, and circular colony shape is observed in the 10 <SUP>-1</SUP> M sample.</I></P><P><I>Figure 4: Positive control pictured for the attraction E. coli cells, with added attractant serine, which is compatible with native E. coli chemoreceptors. Rising consentrations of serine in milimolar range were tested, 0 mM, 5 mM, 10 mM, 15 mM, 20 mM, 25 mM. Circular colony can be observed at 0 mM concentration as control, at all other milimolar concentrations directed colony shape towards attractant source can be observed.</I></P><P><I>Figure 5: Modified 5α E. coli cells with mcpS on pRK415 plasmid pictured with added attractant malate. Rising consentrations of malate were tested, 0 M, 10<SUP>-5</SUP> M, 10 <SUP>-4</SUP> M, 10 <SUP>-3</SUP> M, 10 <SUP>-2</SUP> M, 10 <SUP>-1</SUP> M. Circular colonies are observed with rising concentrations.</I></P><P><I>Figure 6: Modified 5α E. coli cells with mcpS on pRK415 plasmid pictured with added attractant malate. Rising consentrations of malate in milimolar range were tested, 0 mM, 5 mM, 10 mM, 15 mM, 20 mM, 25 mM. Circular colonies are observed with rising concentrations, with possible directed colony shape at 15mM and 20mM samples.</I></P><P><I>Figure 7: Modified 5α E. coli cells with PA2652 receptor on pSB1C3 plasmid pictured with added attractant malate. Rising consentrations of malate were tested, 0 M, 10<SUP>-5</SUP> M, 10 <SUP>-4</SUP> M, 10 <SUP>-3</SUP> M, 10 <SUP>-2</SUP> M, 10 <SUP>-1</SUP> M. Circular colony can be observed for control, elipse shape of colonies can be observed at 10<SUP>-5</SUP>&amp; 10 <SUP>-3</SUP>M concentrations. Elipse shaped colony can also be observed at 10 <SUP>-4</SUP> M however a large spread of bacteria can also be observed to the right side of the plate. Bacteria exposed to 10 <SUP>-2</SUP> M, 10 <SUP>-1</SUP> M concentrations, appear as circular colonies, however the colony spread is smaller it is possible due to saturation by malate at high concentrations.</I></P><P><I>Figure 8:Modified 5α E. coli cells with PA2652 receptor on pSB1C3 plasmid pictured with added attractant malate. Rising concentrations of malate in milimolar range were tested, 0 mM, 5 mM, 10 mM, 15 mM, 20 mM, 25 mM. Circular colonies can be observed for 0 mM, 10 mM, 15 mM, 25 mM concentrations. Colonies exposed to 5 mM and 20 mM concentration of attractant were rendered void due to mishandling with semi - solid agar.</I></P><H2>18th of August</H2><B>Capillary assay</B><P> quantitative, bacterial movement into a syringe/capillary based on attraction, number of bacteria measured in a capillary after 30min using FACS machine. motility medium &amp; high bacterial OD used..</P><H2>2. Tests for uptake of bacteria into roots</H2><H2>Wednesday, 3 August 2011</H2><P>One important part of our project is uptake of our bacteria into plant roots. The observation that this occurs (albeit under controlled lab settings) is new and was only published last year. We attempted to replicate these findings.</P><P>In preparation, we met with Dr Martin Spitaler who advised us on how to prepare samples for the confocal microscopy. Confocal microscopy is much more precise than conventional light field microscopy as it eliminates background light by focusing the laser through a pinhole (Mark Scott, oral communication). The confocal microscopy will focus on imaging GFP expressing bacteria inside Arabidopsis roots to show that uptake of the bacteria takes place. </P><P>Staining of wt roots with DiD, a lipophilic dye that stains the plant membranes and does not interfere with the absorption or emission spectra of GFP and Dendra, was unsuccessful. However, natural fluorescence was measured in a root in a spectrum that does not interfere with measuring GFP. We should therefore not need to dye the roots before imaging.</P><P><I>Stack of wt Arabidopsis root. The root can be imaged at around 488nm. Imaging carried out by Dr Martin Spitaler.</I></P><P>We may also try to stain the roots with propidium iodide, which is also a strong indicator of cell wall break down. </P><H2>Thursday, 4 August 2011</H2><P>We prepared the GFP-expressing bacteria for plant infection. They were spun down and media was exchanged prior to incubation at 37°C to reach exponential phase. Bacteria were then spun down and resuspended in wash buffer (5mM MES) to reach OD 30. 8ml, 4ml and 2ml were added to separate flasks, containing 100ml of half-MS media each. 4ml and 6ml of wash buffer were added to the flasks containing 4ml and 2ml bacteria, respectively. 8ml of wash buffer was added to the negative control. Ten Arabidopsis seedlings were distributed into each of the flasks. Incubation was carried out for 15 hours prior to imaging.</P><H2>Friday, 5 August 2011</H2><P>Prior to imaging, roots were washed in PBS to wash off bacteria and facilitate imaging. We imaged the plants incubated with 8ml of bacteria and were able to find bacteria inside one of the roots. A 3D picture was taken of uninfected roots and roots containing bacteria by taking a Z stackk image using confocal microscopy. </P><P><I>This video shows a zoom from the top to the bottom of the root. It was put together from successive images taken at different depth levels. The GFP-expressing bacteria are clearly visible within the root.</I></P><P>We also imaged roots that did not contain bacteria via a similar Z-stack scan. This image is shown below.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Imperial_College_London/Project/Chemotaxis/Results">http://2011.igem.org/Team:Imperial_College_London/Project/Chemotaxis/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Project/Chemotaxis/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Project/Chemotaxis/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Project/Chemotaxis/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Project/Chemotaxis/Results&amp;oldid=106927" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>