"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College London/Project/Auxin/Assembly
From 2011.igem.org
Project AuxIn »
Home
At a Glance
The Problem
M1: Phyto-Route
M2: Auxin Xpress
M3: Gene Guard
Achievements »
Main Results
Data
Judging Criteria
Europe Jamboree
Human Practice »
Informing Design
Safety
Outreach
Extras »
Collaboration
Protocols
Software
Diary
Brainstorming
Team
Module 2: Auxin Xpress
Auxin, or Indole 3-acetic acid (IAA), is a plant growth hormone which is produced by several soil bacteria. We have taken the genes encoding the IAA-producing pathway from Pseudomonas savastanoi and expressed them in Escherichia coli. Following chemotaxis towards the roots and uptake by the Phyto Route module , IAA expression will promote root growth with the aim of improving soil stability .
Overview
Specifications
Design
Modelling
Assembly
Testing & Results
Future Work
Assembly
We wish to build a single expression plasmid that can express IaaH and IaaM. While this task can be summarised in one sentence its execution is not as short. The first problem lies in the size of these two enzymes which both exceed 1kbp making their synthesis a problem. We therefore created a new standard for biobrick assembly to tackle this issue. We broke up these large sequences into four fragments that were ordered at the end of week 3. In preparation for the arrival of these fragments (circa 8-10 days) we started to transform our cells with the pVEg+pSB1C3 backbone constructs in order to make enough genetic material for a gibson assembly reaction. This chapter will describe our struggles and successes throughout this grueling and yet rewarding process.
29th of July
Our first transformations were a success! We have managed to transform our competent cell colonies with pSB1C3 containing BBa_K398500 and J23100 promoter to produce cell line 6. We also transformed the cells with part BBa_K316001 to produce cell line 7 and part BBa_K316005 to produce cell line 8. With these cell lines we will be able to make more copies of each part in preparation for the arrival of our synthesized sequences. We are under a tight schedule so efficiency is key.
Also, the cell line with the superfolded GFP integrated in its genome has been plated successfully. We have confirmed that these are the correct cell lines by looking at them under UV light. Their green glow was brighter than we expected.
However, cell line 1 did not grow in the liquid broth media at the same concentration of kanamycin. We will create an assay to ascertain the optimum kanamycin concentration. This is both a bizarre and unexpected result that must be rectified.
30th of July
The previous day, we had obtained the mcpS gene in a pRK415 plasmid from Spain. We then performed a transformation on the competent 5α cell line and obtained the results on this day. Out of the four plates one of them contained transformed cells. We named these cells 10.
1st of August
A month has already passed since we started our project. The experiments that were conducted on this day were a mini-prep on samples 6,7,8 and 10 that were inoculated into an LB broth culture the night before. We used a mini-prep kit to obtain the plasmid DNA that we wanted from the transformed cells. Then, once we obtained the DNA, we started a restriction digest using PstI and EcoRI for 1.5 hours to confirm that the samples from the mini-prep contained the DNA that we wanted (the pSB1C3 backbone with the appropriate promoters).
Sadly, once we tried to visualize the results on a gel, the results were less than satisfactory. Mini-prep 6a and 7b seems to have failed and the rest of the bands do not make much sense. The experiment had to be repeated.
2nd of August
Today we attempted the restriction digest again and the results verified that samples 6,7 and 8 were actually pSB1C3 constructs with the required components. Therefore, the mini-prep experiment had worked and did not need to be repeated. It is a great feeling when everything comes together.
Gel 1&2. Restriction digest of three backbone vectors with EcoRI(E) and PstI(P) to confirm backbone length. Gel1: Lane1-1 kb DNA ladder Lane 2-6a cut with E;Lane 3-6a cut with P; Lane 4-6a cut with E+P;Lane 6-6b cut with E; Lane 7- 6b cut with P; Lane 8- 6b cut with E+P; Lane 11- 7a cut with E; Lane 12- 7a cut with P; Lane 13- 7a cut with E+P; Lane 15- 7b cut with E; Lane 16- 7b cut with P; Lane 17- 7b cut with E+P. Gel 2: Lane 1 - 1kb ladder; lane 2 - 8a cut with E; lane 3- 8a cut with P; lane 4- 8a cut with E+P; lane 6- 8b cut with E; lane 7- 8b cut with P; lane 8 - 8b cut with E+P.
3rd of August
Today we attempted to PCR samples 6, 7 and 8. The PCR did not end up working so well so. The annealing temperature must have been too low because we obtained bands that we did not want. Also, there must have been too little Sybr safe in the gel because the bands that were there were not bright. The PCR will be repeated once again and we will use a temperature gradient for (hopefully) better results.
4th of August
Today we redid the PCR of samples 6, 7, and 8 but with a temperature gradient to improve primer annealing.... and it was a success! So tomorrow we can run the rest of the Dpn1 digested DNA on a gel and gel purify it, then the vectors are ready to be used for DNA assembly once our genes arrive!
Gels 3&4: Temperature gradient PCR of desired backbone with promoter and terminator out of plasmids. Gel 3:Lane 1- 1kb DNA ladder; lanes 2 to 7 - PCR of vector 6 from 57.1°C to 62.6°C; lanes 9 to 15 - PCR of vector 7 from 57.1°C to 62.6°C. Gel 4: lanes 2 to 7 - PCR of vector 8 from 57.1°C to 62.6°C
The backbone sequences have also returned. All of the samples are in order except a one base pair mutation in sample 8 within the pVEg promoter. For now, we are going to amplify it anyways in the hope that the one base pair mutation is just an error that occured during sequencing. Either way, sample 8 is a back-up of sample 7 so there should be no problems either way.
We also started experimenting with the Salkowski reagent. In particular, we tested the S2/1 method and found that the ideal wavelength for measuring the IAA concentration with our apparatus is at 554nm. This was done by scanning between 500nm and 600nm to obtain the individual absorption spectra of each sample. The wavelength at which the absorbance peaked was chosen. This experiment gave us a great standard curve from which we can roughly estimate the amount of IAA in a solution between 2 and 200 μg/ml.
8th of August
Today we ran a gel of the PCRd backbone DNA extracted from the previous gel to make sure that the DNA was pure. The gel results were succesful. We also transformed cells with the pure DNA to check that the Dpn1 digestion worked properly.
Gel 5: Gel extracted backbone vector DNA run on a gel to confirm purity. Lane 1- 1 kb DNA ladder; Lane 2- vector 6a; Lane 3- vector 6b; Lane 4- vector 7a; Lane 5- vector 7b; Lane 6- vector 8a; Lane 7- vector 8b.
9th of August
Transformations of auxin fragment 1 (20) and auxin fragment 4 (24) were successful. We obtained plenty of colonies to choose from on both the ampicillin and kanamycin plates. Also, the DpnI digest transformations created bacteria that had no resistance to chloramphenicol.
However, sample 8 has to be repeated because in the "rest" plates there were 1 or 2 colonies on each.
10th of August
Today we attempted to perform a midi-prep on the DNA fragments that had arrived from Germany. The experiment failed and we were not able to obtain a decent yield of DNA. Oh well, got to try again.
12th of August
After the failed results and the rather lethargic week we attempted to get our minds back to gear. The visit to Syngenta the day before was a moment of respite that allowed us to perform two mini-preps that worked. However, there was an issue when we digested our plasmid (pCR2.1) with MlyI. The genes had been placed in a plasmid that contains multiple MlyI sites which would make the gel extraction more difficult. We used Serialcloner to make a virtual cut and then used the predicted image to guide us. In the end we obtained a band for 20b and 24a in the right location (or so we hope!).
Gel 6: gel of MlyI restriction digest of the synthesised genes. Lane 1-Marker; Lane 2-20a; Lane 3-20b; Lane 4-22a; Lane 5-22b; Lane 6-23a; Lane 7-23b; Lane 8-24a.
15th of August
We gel extracted several gene fragemtns that were transformed yesterday.
16th of August
The transformations we did the previous day were a success and the last two remaining fragments were also mini-prepped. We will attempt the Gibson assembly. If all goes well, we'll have E. coli excreting auxin by Friday!
19th of August
Gel 7: Gel extracted DNA fragments for assembly of auxin expressing plasmid. Lane 1&2 - Auxin fragment 1; Lane 3 - Auxin fragment 4; Lane 4&5 - Auxin fragment 3; Lane 6&7 - Auxin fragment 2; Lane 8&9 - pVEG backbone vector
22nd of August
Today we obtained some disappointing results. It seems like the vector is just religating during the Gibson reaction. Maybe the sequences of the two ends of the vectors are too homologous for Gibson to work. Either way, we will be attempting CPEC today and hopefully we will obtain some bands that we can purify and transform bacteria with.
23rd of August
First try of CPEC assembly assembly of the auxin construct seems to have been successful by looking at the gel electrophoresis results of colony PCR and CPEC product PCR with sequencing primers. We expect sequencing results to arrive tomorrow for an accurate assessment, however a preliminary test of colony supernatant with the salkowski reagent showed promosing results with one colony which turned bright red, indicating the presence of auxin.
Gel 8: The first two lanes show that CPEC assembly of four auxin fragments at ~1kb each in a backbone of about 2kb. Lane one contains the assembled construct at ~6 kb and lane 2 contains the negative control assembly of backbone vector with no insert at ~2kb. The following two lanes show the analytical PCR of the CPEC assembled product with standard biobrick primers to PCR our the assembled auxin fragments. The first well shows the auxin assembly at ~4kb and the second (negative control) shows no PCR product because no insert is present. Gel 9: Colony PCR with standard biobrick primers of CPEC assembled auxin fragments showing the desired assembly size of about 4 kb. Gel 10: Colony PCR of negative control colonies (backbone vector 8 only and no insert) and positive control colony PCR of the same vector 8 but the entire plasmid. This result shows that the DpnI digest of PCRd backbone vector 8 was not completely efficient as some complete plasmid remains, but this residual amount did not hinder assembly.
25th of August
A restriction digest of the mini-prepped auxin assembled constructs with EcoRI and PstI show clearly the assembled auxin insert drop out of the backbone vector.
Gel 11: Restriction digest of auxin construct with EcoRI, PstI and both for three miniprepped colonies grown from transformed E. coli.
Retrieved from " http://2011.igem.org/Team:Imperial_College_London/Project/Auxin/Assembly "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
