<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Brainstorming_Health"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Brainstorming_Health">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Brainstorming_Health&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Brainstorming_Health&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Brainstorming_Health&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Brainstorming_Health" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Brainstorming Health</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">Project AuxIn<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London">Home</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Tour">At a Glance</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project/Background">The Problem</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Overview">M1: Phyto-Route</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Overview">M2: Auxin Xpress</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Overview">M3: Gene Guard</A></LI></UL><LI><A class="sf-with-ul" href="#">Achievements<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Achievements">Main Results</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Judging">Judging Criteria</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Europe_Jamboree">Europe Jamboree</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Overview">Informing Design</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human/Outreach">Outreach</A></LI><LI><A class="sf-with-ul" href="#">Extras<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Extras/Collaboration">Collaboration</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Protocols_General">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Software">Software</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Notebook/week1">Diary</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Brainstorming_Health">Brainstorming</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Team">Team</A></LI></DIV><H1>Brainstorming</H1><P><B>This page contains a summary of the ideas we developed throughout our brainstorming sessions at the beginning of the project. These ideas can be classified into 4 main categories as shown below. Click on the tabs to find out more about the ideas in each category.</B></P><DIV class="underlinemenu"><UL><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Brainstorming_Health">Health &amp; Medicine</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Brainstorming_Energy">Energy &amp; Environment</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Brainstorming_Bioremediation">Bioremediation</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Brainstorming_Others">Others</A></LI></UL></DIV><H1>Ideas</H1><P><A href="#Sunscreen">Bacterial sunscreen</A><A href="#Prodigiosin">Prodigiosin pigment</A><A href="#Venom">Anti-venom</A><A href="#Parkinsons">Alkaloid isorhy against Parkinson's</A><A href="#Vampiric">Vampiric bacteria</A><A href="#Dengue">Transportable dengue mosquitoes “simulated nepenthes”</A><A href="#Sensor">Bile acid sensor</A><A href="#Bandage">Inflammation-detecting bandage</A></P><H1><A name="Sunscreen" style="color:#225323;">Bacterial sunscreen</A></H1><P>- Evidence suggests that several components found in most sunscreens are harmful to us and can be carcinogenic, also most sunscreens only protect against UV B (315-280 nm) and not UV A </P><P>- These sunscreens use metal oxides (Zinc oxide) to absorb UV radiation, but the effects of absorbing these metals into your skin are not fully understood and are thought to lead to production of reactive oxygen species and could lead to melanomas rather than preventing them. </P><P>- Scytonemin is a pigment found in cyanobacteria which protects them from UV radiation, absorbing 325-425 nm. Its synthesis requires three enzymes, SycA-C Mycosporine-like amino acids (MAAs) are produced by organisms adapted to environments with high levels of sunlight (eg. cyanobacteria and algae), protecting them from UV radiation. There are 20 types and they also serve as anti-oxidants by stabilising free radicals (anti-ageing?). In a bioinformatics study the genes YP_324358 (predicted DHQ synthase) and YP_324357 (O-methyltransferase) were identified in <I>A. variabilis</I> PCC 7937 cyanobacteria. (http://www.sciencedirect.com/science/article/pii/S0888754309002353) MAAs have already been recognised to have sunscreen potential and are found in some anti-wrinkle creams</P><P>- Gene cluster encoding 4 enzymes required for all MAA synthesis. Expressed this gene cluster from a cyanobacterium into <I>E. coli</I> and got pigment production. (Text by Nikki)</P><P><A href="#Top">Back to top</A></P><H1><A name="Prodigiosin" style="color:#225323;">Prodigiosin pigment</A></H1><P>- UV protecting properties</P><P>- Synthesis controlled by quorum sensing - absorbs in UV range from 240 to 400 nm as well as in the visible spectrum from 400 to 600 nm - also antibiotic and anti-cancer (induces NAG-1 pro-apoptotic gene in human breast cancer cells) - Prodigiosin biosynthesis gene cluster (pig cluster) contains ~15 ORFs in Serratia strains (Harris et al., 2004) - PigS and PigP regulate prodigiosin biosynthesis in <I>Serratia</I> (Gristwood et al., 2011) - <I>Streptomyces griseoviridis</I> 2464-S5 produces prodigiosin R1, gene cluster of 24 open reading frames, including 21 genes (rphD-rphZ) homologous to prodigiosin biosynthesis genes in the red cluster in <I>Streptomyces coelicolor</I> A3(2). The expression of rphN in <I>S. coelicolor</I> lacking redN restored the production of prodigiosin (Kawasaki et al., 2009) - Could also be used as a dye in clothing. pigment production from microorganisms by large-scale fermentation would be environmentally friendly and sustainable. Could make clothes with inherent UV protection, but they would all be red.... (Text by Nikki)</P><P>Sci-Fi prodigiosin ideas (developed by Rebekka, Nick and CJ from the RCA): 
</P><P>- Use prodigiosin (red pigment) as the new &quot;colour of health&quot; (know something is sterile rather than assume it is) </P><P>- Possible future uses: in decontamination/ as a &quot;panic room&quot;/ sterile hospice -possible future products: hand gel, clothing (e.g. protective suits in bioreactor plants), decontamination paint (in hospitals etc) 
</P><P>- Actual uses: anti-cancer (could be in a red drip, red pill?), anti-malarial (drug verification because pigment colour is hard to fake) Eventually, this idea was scratched because optimising the production pathway does not contain enough synthetic biology. In addition, the compound is immuno-suppressive (http://pubs.rsc.org/en/Content/ArticleHtml/2008/CC/b719353j) and would therefore be disadvantageous to normal people. Many of the envisaged applications would only work with less problematic analogues of prodigiosin. 
</P><P><A href="#Top">Back to top</A></P><H1><A name="Venom" style="color:#225323;">Anti-venom</A></H1><P>- Traditional way of anti-venom production:</P><P>1. injecting venom or detoxified venom into a horse (tiny amount, multiple times)
2. After the antigen growing period, contract the horse blood plasma
3. Use stomach digestive enzyme to breakdown the anti-venom protein into smaller globin
4. Use (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB> to salt out the globin (purification) [1]</P><P>- Oxides like K<SUB>2</SUB>MnO<SUB>4</SUB> can neutralise venom by denaturing the polypeptide chains - complex ligase like AuCl2 can denature venom by binding with them, when preventing venom entering the tissues [2] -immune system has a stronger response to venom [3]. When the mast cells are stimulated, they release histamine. Histamine can subside the venom - histamine producing bacteria: found in tuna; 18 types of bacteria such as Clostridium perfringens and other anaerobic bacteria</P><P>- To conclude: 1. decide the working mechanism (toxicology) of a specific type of venom: does it attack neurons? brain cells? cardiovascular system? respiratory system? 2. does it react with histamine? 3. bacterial production of histamine</P><P>REFERENCES: 
</P><P>[1]http://en.wikipedia.org/wiki/Antivenom 
</P><P>[2] http://life.91sqs.com/html/zazhi/yixueyushehui/2011/0113/1453.html 
</P><P>[3] “ Development of antibody against Naja naja atra venom using phage display and single-chain Fv antibody technology ” Master Graduation Paper NUK http://ethesys.nuk.edu.tw/ETD-db/ETD-search-c/view_etd?URN=etd-0825110-170246 (Text by Nina)</P><P>We discussed the anti-venom idea with Koby (one of the RCA students). He helped us expand the idea to maybe provide protection against viruses as well. We even developed the idea to use a seasonal hand wash containing the purified antibodies from the season's viruses in order to create a world where a handshake would be more than just a form of greeting but also a way to pass on immunity.</P><P>RCA sci-fi story ideas: Nuclear winter leaving people without immune systems. Use external immune system to save everybody?</P><P>Synthetic life can only use certain amino acids, firewall? Biocomputers where man and machine are converged closer together. Floppy disk baby? Mock news articles (Times, science article, tabloid), comic strips (simplified version of our project), documentary type video?, Terrorist attacks (Text by Chris)</P><P>Antibody research: 
</P><P>- Single-domain antibodies such as the nurse shark derived IgNAR and the camelid derived VHH have been used for many purposes and have recently started to gain popularity among the scientific community. 
</P><P>-Contain CD3 loop that gives these Ig's an advantage when looking for cryptic viral epitopes. However, contain ten epitope copies and might still infect. Solved by increasing their mass. 
</P><P>-Both of these Ig's are stable enough to be administered orally. - Orally administered transformed lactobacilli were used to administer anti-TNF Ig. 
</P><P>-Small dimensions of VHHs allow it to be easily tagged. -Lactobacilli have been engineered that produces VHH's at a rate fast enough to prevent infection by p2 bacteriophage. Possibly use lactobacillus for screening and E. coli for secretion? -VHH's and IgNAR's have been effective in detecting poliovirus and inhibiting its replication in vitro, as well as preventing the assembly and secretion of hepatitis B. Possible to use VHH's as intrabodies vs. HIV-1? -N-glycosylation increases stability. -Studies demonstrated that pre-immune libraries can be used for rapid generation of Ig's against a large number of harmful antigens. Troublesome low sensitivity overcome by using phage-displayed instead of purified antibodies.[1]</P><P>References: [1]Ario de Marco, “Biotechnological applications of recombinant single-domain antibody fragments,” Microbial Cell Factories 10, no. 1 (2011): 44. (Text by Chris)</P><P>To generate cell surface anchored antibody fragments: VHH sequence is fused to the anchor sequence from proteinase P of <I>L. casei</I>.</P><P>To generate the secreted VHH, stop codon after E-tag. (Text by Chris)</P><P>Sorting cells - A method is needed to sort cells by their ability to bind the venom proteins - One such method would be to have the cells secrete their anti-venom proteins and then flood the cells with venom. The best cells would survive and the weak ones would be killed. However, this may cause the production of proteins that bind the venom components that are less dangerous such as phospholipases and oxidases. It would be more important for the anti-venom to inhibit the neuro-muscular disruptive proteins</P><P>- One effective method would be to use Fluorescence Activated Cell Sorting, which is able to sort cells by their fluorescence. See: http://www.bio.davidson.edu/courses/genomics/method/FACS.html - This would require a system that expresses a fluorescent reporter in response to the binding of venom proteins to the cell-surface proteins, but I cannot be too specific until I know how the anti-venom proteins are going to be expressed - Yet another method would cause the death of any cells that do not bind to the venom with high enough affinity - it would be similar to the methods employed in T-cell selection in eukaryotic immune systems, but would be more tricky as it has to occur in a bacterium - The ideal then, would be to have all the cells that have little or no binding affinity killed, and then those that do bind express GFP so that the fluorescence can be quantified and the binding can be rated - Again, I'd need to understand the anti-venom proteins before I could suggest any particular systems - Perhaps a second species of bacteria could be used to express the venom proteins on their surface, and come into contact with the anti-venom producing cells, causing contact-dependant stimulation, so that those that interact by the venom-antivenom complexes will be stimulated to divide whilst the remaining cells will potentially be killed See: http://www.ncbi.nlm.nih.gov/pubmed/21085179 (Text by Frank)</P><P>In vivo mutagenesis - we need a method for random mutagenesis of peptides to create high affinity binding proteins to the multiple components of venom - PCR based mutagenesis is namely used for site directed mutagenesis and has several biases that make it not ideal for random mutation - in vivo homologous recombination inherent to yeast can be exploited for protein mutagenesis (Pirakiticulr et al., 2010) MAGE (multiplex automated genome engineering) See: http://www.nature.com/nature/journal/v460/n7257/fig_tab/nature08187_F4.html - a method for large scale evolution of cells - has been used to optimize the 1-deoxy-d-xylulose-5-phosphate (DXP) biosynthesis pathway in E.coli for isoprenoid lycopene overexpression with a pool of synthetic DNA to modify 24 genes in the pathway creating &gt; 4.3 billion combinatorial genomic variants per day (Wang et al., 2009 - can do insertions, deletions and mismatches See: http://www.nature.com/nature/journal/v460/n7257/fig_tab/nature08187_F1.html (Text by Nikki)</P><P>However, all currently known methods of in vivo mutagenesis are not able to mutate a specific gene without requiring a transformation step. We therefore propose the following mutagenesis mechanism:</P><P>We would therefore use a novel mechanism based on an rt reaction (Text by Rebekka)</P><P>Type of venom Venom refers to varieties of toxins produced by certain types of animals. One of the most common venoms are produced by snakes where 15% of 3,000 species of snakes are found poisonous. Snake venom consists of proteins, enzymes, substances with a cytotoxins, neurotoxins and coagulants</P><P>Most snake envenomings and fatalities are found in South Asia, South East Asia and sub-Saharan Africa with the high fatality rate of 125,000 deaths per annual. Among these India is reported the most caused by big 4 including Russell's viper, Indian cobra, saw-scaled viper, and the common krait. Indian cobra is found the most famous and make the highest fatality rate (43%) in India and Southeast asia.</P><P>Indian cobra venom contains a potent post-synaptic neurotoxin which acts on the synaptic gaps of the nerves, thereby paralyzing muscles, and in severe bites leading to acute respiratory failure or cardiac arrest. The components of venom include lysis enzymes such as hyaluronidase which increase the spread of the venom. Its toxicity is found one of the highest based on LD50 value in mice. Symptoms of cobra envenomation can begin from 15 minutes to two hours after the bite, and can be fatal in less than an hour.</P><P>Despite the advance in emergency therapy, anti-venom is often only the effective treatment. In treatment anti-venom is injected into patient intravenously which could neutralize the toxin. Collecting of anti-venom is done by milking the venom and injected into the cattle. The subject will undergo immune response where its antibody produced can be collected. This common method of obtaining anti-venom can tackle many toxins in the venom but however is considered unproductive since only small amount of anti-venom is produced from the animal blood which is due to complicated serum purification process from the animal's serum. Waiting of animal recovery from venom also make its production quite slow and in several cases the animals die after injection.</P><P>Therefore bacteria should be introduced as a substituted platform utilizing synthetic biology to deal with the problems mentioned above. Apart from existing characterised anti-venom, other anti-venoms could be more easily discovered using the various mutagenesis of variable region of antivenom antibody. To develop this platform, well known venom could be produced to test this platform. Indian cobra snake is therefore chosen as the first target due to its generality, high toxicity and it is also one of the highest profile anti-venom discovered which could potentially save many victims from this fatal snakebite.</P><P>The way to engineer the bacteria is to mutagenise the shark antibody gene to allow different variable regions of the antibody to be expressed on the surface of the bacteria in different libraries. The venom is screened onto each plate of different bacteria libraries. The venom will bind to the right antibody and trigger the signal cascade which results in the expression of the fluorescence proteins which can be detected by FACs. machine. The bacteria that produces the antibody for the venom will be subjected to DNA sequencing which could be the platform for producing bacteria expressing antibody specific to the venom in the future. (Text by Ming)</P><P>So the method for the anti-venom generation begins as follows: 1: Take the venom proteins and bind biotin to either the C or N terminus so that it is able to be fixed to a large ferro-magnetic beads that are coated in streptavidin. Streptavidin has an extremely high affinity for biotin and the beads can be picked up with a neodynium magnet - so the bacteria with the 'antibody' on their surface that is able to bind to the venom proteins will be attached to the bead. 2: The bacteria will have a plasmid that encodes a gene for a single-chain variable fragment, a fusion protein of the variable regions of the heavy and light chains of an immunoglobulin. This will be displayed upon the surface of the bacterium and will be on a plasmid that has a very mutagenic effect as described by Rebekka.</P><P><A href="#Top">Back to top</A></P><H1><A name="Parkinsons" style="color:#225323;">Alkaloid isorhy against Parkinson's</A></H1><P>- It is rumoured to be a potential treatment for Parkinson's and it would make a good project if this were to be produced by bacteria</P><P>- On further research it turns out that the evidence for this drug as a treatment is weak and there is no information available about the gene or genes that encode it, so the idea was dropped. (Text by Frank)</P><P><A href="#Top">Back to top</A></P><H1><A name="Vampiric" style="color:#225323;">Vampiric bacteria</A></H1><P>-The aspect of a Vampiric bacteria that is designed to get rid of blood clots produced by trauma induced clotting or during complex medical procedures is intriguing.</P><P>-Expression of Hirudin is possible in systems such as <I>E. coli</I>. In 2007 Berkeley produced a chassis for a <I>E. coli</I> that could be introduced into the blood stream after inactivation. -However, it is difficult to have the non-viable cell lysis occur in the correct location and therefore an anticoagulant could just as well be injected into the patient. -For this to work, we would require an expression system that is able to express Hirudin (produced usually by leech salivary glands and has been successfully expressed in <I>E. coli</I> [1]), express anti-angiotensin (it is possible to express Fab fragments in <I>E. coli</I> [2]) and targeting the fibrin (can be done by expressing Tissue plasminogen activator). -The idea would be to have the chassis recognize a blood clot or an area of damage and prevent clotting and/or clear clots. A method for having the system recognize when to secrete hirudin would be by having the bacteria sense trauma related chemokines and have the chassis secrete the protein only when it senses above a certain threshold of these chemokines or we could try to express protease-activated receptors (GPCR) that are cleaved by activated thrombin (the target of hirudin). Direct application would only benefit over the use of leeches in that the chassis is more aseptic then a leech bite. The biggest issue remains the fact that for this to work we would have to inject the patient with living <I>E. coli</I> that can evade the human immune system.</P><P>A new method of boosting biosynthesis has been obtained through the use of RNA scaffolds:<A href="http://www.sciencemag.org/content/early/2011/06/22/science.1206938" rel="nofollow">http://www.sciencemag.org/content/early/2011/06/22/science.1206938</A></P><P>Reference: [1]Shuhua Tan et al., “Efficient expression and secretion of recombinant hirudin III in E. coli using the L-asparaginase II signal sequence,” Protein Expression and Purification 25, no. 3 (August 2002): 430-436. [2]Saad A Masri et al., “Cloning and expression in E. coli of a functional Fab fragment obtained from single human lymphocyte against anthrax toxin,” Molecular Immunology 44, no. 8 (March 2007): 2101-2106. [3]Ji Qiu, James R. Swartz, and George Georgiou, “Expression of Active Human Tissue-Type Plasminogen Activator in Escherichia coli,” Applied and Environmental Microbiology 64, no. 12 (December 1998): 4891-4896. (Text by Chris)</P><P><A href="#Top">Back to top</A></P><H1><A name="Dengue" style="color:#225323;">Transportable dengue mosquitoes “simulated nepenthes”</A></H1><P>Engineering bacteria which could be put into the water container in order to attract, trap and kill the dengue mosquitoes and therefore decoy the mosquito away from biting you.</P><P>Overview - Dengue virus (RNA) : fever, haemorrhage, shock, organ disfunction - Problems : no vaccine, difficult to engineer vaccine, acute but at the same time chronic - Vectors : Aedes aegypti &gt;&gt; target!!! - Why? : Lay egg at night (specific time), fragile, is the only 1 type of vector</P><P>Engineering bacteria Module 1 : Allurement - Odour &gt; lactic a, octenol, NH3 &gt; LDHB, LOX, AOS, GDH <A href="http://en.wikipedia.org/wiki/Aedes_aegypti" rel="nofollow">http://en.wikipedia.org/wiki/Aedes_aegypti</A>,<A href="http://chemse.oxfordjournals.org/content/30/2/145.full" rel="nofollow">http://chemse.oxfordjournals.org/content/30/2/145.full</A> - Light &gt; orange fluprescent protein (600 nm)<A href="http://journals.fcla.edu/flaent/article/view/75460" rel="nofollow">http://journals.fcla.edu/flaent/article/view/75460</A> Module 2 : Entrapment - Surfactant &gt; Flavobacterium &gt; GLD,<A href="https://colloque.inra.fr/flavobacterium/content/download/.../28Hunnicut.pdf" rel="nofollow">https://colloque.inra.fr/flavobacterium/content/download/.../28Hunnicut.pdf</A> Module 3 : Torment - Egg and Larva &gt; Bacillus thuringiensis &gt; Cry, Cyt, Chi - Adult &gt; Geranyl acetone &gt; Populus trichocarpa &gt; POPTRDRAFT_596199, <A href="http://www.ncbi.nlm.nih.gov/pubmed/20127888" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/20127888</A> (Text by Ming)</P><P><A href="#Top">Back to top</A></P><H1><A name="Sensor" style="color:#225323;">Bile acid sensor</A></H1><P>- The idea came from Si’s final year project.</P><P>- A biosensor can be produced for daily home use to detect the bile acid concentration level in blood to prevent a series of liver diseases, especially the Intrahepatic cholestasis of pregnancy (ICP) 
</P><P>- The basic concept behind this sensor: bile acid - enzyme binding with the acid molecules – promoter – triggering the FXR gene – production of GFP - GFP is normally used as an indicator of the bio-sensor. The main problem is that the fluorescent intensity is very hard to quantified for a home-user - being inspired by the cosmetic skin colour sample card, we can make a sample card of fluorescence to give the rough concentration level of bile acid - a threshold value is required to tell the patient when their blood bile acid level is dangerous and may need a medical treatment - therefore, we will modify the linear relationship between bile acid concentration and GFP intensity level into a Hill system using Hill equation to find the threshold value - also, we may use colour indicator instead of GFP (light indicator).</P><P>Background: 
</P><P>- Bile acids are 24-carbon steroids found in bile, which are subject to enterohepatic circulation - synthesized in liver and stored in gallbladder, helping in digestion and absorption of dietary fat and liposoluble vitamins - bile acids are highly toxic. Therefore their concentration must be tightly regulated - the level of bile acids is controlled through a negative feedback system mediated by a nuclear bile acid receptor FXR. FXR is highly expressed in liver, intestine and kidney cells. It responds to bile acids and has been shown to repress CYP7A1, a key gene associated with bile acid synthesis. - Defects in bile acid homeostasis due to functional variations of FXR result in cholestatic conditions such as Intrahepatic cholestasis of pregnancy (ICP). ICP is a pregnancy-specific liver disorder characterized by pruritus (intense itch) an abnormal liver function - The dysfunction of maternal liver could induce stress on the fetal liver as the fetus relies on maternal liver to remove bile. It has been shown that ICP pregnancies are more likely to suffer from meconium staining of the amniotic fluid (MSAF), cardiotocography (GCT) abnormalities and respiration distress syndrome (RDS). - The risk of complications such preterm labor, prenatal death and stillbirth are directly linked to severity of ICP - The morbidity of ICP is geographically and racially dependent. [1]</P><P>Country Morbidity(%) Sweden 4.2 Finland 1.0 Poland 1.5 Jugoslavia 1.1 Spain 1.6 UK 1.0 China 4.4 Aymara 13.8 Araucanian 27.6 Chile 15.6 Caucasian species 9.8</P><P>Mechanism: FXR based activation: - use eukaryotic transcription factors in bacterial gene expression - suggested use of FXR (known structure and DNA sequence) with constitutive promoter expression - a receptor protein of a number of bile acids, which would bind to ligand-bile acid - can act as a repressor or an activator, we would use it as an activator, binding to a promoter region B4-BARE (2.4 kb) taken from a gene UGT2B4, (known primers for promoter region) - specified exact FXR binding site, therefore possibility of integrating FXR binding site into another regulatory region - upon binding to bile acid it would activate output gene - In the experiment using FXR as transcription factor in human cells 30mM DCA was used, however presumably lower concentration should be sufficient for triggering of FXR.</P><P>Output: 
</P><P>- GFP: fluorescence = light indicator = hard to quantify 
&lt;&gt;- ligaments: colour = clear indicator = influenced by the red colour of blood cells = cell free mechanism for hospital use = accurate measurement = filter/ membrane mechanism for home use 
</P><P>- protein expressed on surface of bacteria causing aggregation (either of bacteria or of bacteria to a substance in blood)</P><P>Modelling: 
</P><P>- Binding of activator to GFP gene is a positive cooperative reaction. 
</P><P>- Once activator molecule is bound to the enzyme, its affinity for other activator molecules increases. 
</P><P>- Hence, Hill equation can be used as a model. Hill function for transcriptional activation: k1: Maximal transcription rate Km: Activation coefficient n: Hill coefficient A: [activator]</P><P>- This equation gives the % bound by activator as a function of activator concentration. - after Hill equation modification, the system behaves like a switch - In addition, we also need to model the diffusion of blood and GFP in the bacteria culture</P><P>Cell–free mechanism: 

</P><P>- For hospital use, the blood must be processed through a cell-free system to give an accurate test result, as well as the bacteria to reduce the risk if they leak from the container ( after the cell-free process, the bacteria are not able to reproduce) - separate certain organelles from whole cells for further analysis of specific parts of cells - in the process, a tissue sample is first homogenised to break the cell membranes and mix up the cell contents - homogenization is intensive blending of mutually related substances or groups of mutually related substances to form a constant of different insoluble phases - homogenate is then subjected to repeated centrifugations - each time removing the pellet and increasing the centrifugal force - Separation is based on size and density, with larger and denser particles pelleting at lower centrifugal forces. In the separating order in actual application: Whole cells and nuclei; Mitochondria, lysosomes and peroxisomes; Microsomes (vesicles of disrupted endoplasmic reticulum); ribosomes and cytosol.</P><P>Filter/semi-permeable membrane for the blood cells: RBC: 7-8.5 μm neutrophil: 10-12 μm eosinophil: 10-15 μm basophil: 10-12 μm monocyte: 14-20 μm lymphocyte: small: 6-8 μm, medium 9-12 μm, large: 13-20 μm taurocholic acid molecule roughly 0.4 nm - therefore, if the hole diameter of the filter is set to be at about 2-5 nm, all the blood cells can be filtered</P><P>Human practices (with help of LSE BIOS): A. why biosensors ? - possible to construct - has a potential market</P><P>- B. human practices 1. bio-safety - cell-free system to stop the bacteria from reproducing if there is a leakage of the sensor device - (unpredictable) mutation must be carefully prevented during the engineering part 2. bio-security - the access to DNA sequences and other genetic information must be controlled - “garage biologist” - the restriction to synthetic biology knowledge is not the way to prevent bio-terrorism, the key thing is the professional and correct guidance 3. IP (intellectual property issue) and patent 4. ethical and philosophy 5. global fairness - bio-sensing = faster disease detection - employment problem? discrimination? - classify the diseases into different levels of risks - social coordinate organization to optimize the occupation and personnel resources (Text by Si, Nina, Nick and Yuanwei)</P><P><A href="#Top">Back to top</A></P><H1><A name="Bandage" style="color:#225323;">Inflammation-detecting bandage</A></H1><P>- Inflammation is caused by the immune response to pathogens.</P><P>- It an be acute to chronic.</P><P>- Acute inflammation: 1. increased movement of plasma and leukocytes (granulocytes) from blood to injured tissues 2. causative agent = pathogens and injured tissues 3. cells involved: neutrophils, mononuclear cells (monocytes and macrophagens) 4. primary mediator = vasoactive amine and eicosanoids - to carry out the detection mechanism, we will set a target chemical to detect - at this stage, interferon gamma is taken into our consideration: Interferon-gamma (IFN-γ) is a dimerised soluble cytokine that is the only member of the type II class of interferon. This interferon was originally called macrophage-activating factor, a term now used to describe a larger family of proteins to which IFN-γ belongs. In humans, the IFN-γ protein is encoded by the IFNG gene.[2] - the goal of this project is to find a gene coding the protein which and react with interferon gamma and give an indication - a further step may be taken as a “damaged tissue cleaner”, which means that the bacteria can not only detect but also remove the inflammation tissue</P><P>[1] <A href="http://baike.baidu.com/view/676699.htm" rel="nofollow">http://baike.baidu.com/view/676699.htm</A></P><P>[2] <A href="http://en.wikipedia.org/wiki/Interferon-gamma" rel="nofollow">http://en.wikipedia.org/wiki/Interferon-gamma</A> (Text by Si, Nina, Nick and Yuanwei)</P><P><A href="#Top">Back to top</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Imperial_College_London/Brainstorming_Health">http://2011.igem.org/Team:Imperial_College_London/Brainstorming_Health</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Brainstorming_Health" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Brainstorming_Health" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Brainstorming_Health&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Brainstorming_Health&amp;oldid=154144" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>