"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College London/Project/Auxin/Overview
From 2011.igem.org
Project AuxIn »
Home
At a Glance
The Problem
M1: Phyto-Route
M2: Auxin Xpress
M3: Gene Guard
Achievements »
Main Results
Data
Judging Criteria
Europe Jamboree
Human Practice »
Informing Design
Safety
Outreach
Extras »
Collaboration
Protocols
Software
Diary
Brainstorming
Team
Module 2: Auxin Xpress
Auxin, or Indole 3-acetic acid (IAA), is a plant growth hormone which is produced by several soil bacteria. We have taken the genes encoding the IAA-producing pathway from Pseudomonas savastanoi and expressed them in Escherichia coli. Following chemotaxis towards the roots and uptake by the Phyto Route module , IAA expression will promote root growth with the aim of improving soil stability .
Overview
Specifications
Design
Modelling
Assembly
Testing & Results
Future Work
Overview
Auxin is a well known plant hormone that is responsible for plant growth in response to biotic and abiotic stresses. Usually, synthetic auxins like α-Naphthaleneacetic acid (αNAA) and 2,4-Dichlorophenoxyacetic acid (2,4-D) are used as herbicides. They have been been used effectively for the past fifty years due to their high effectiveness and cheap cost. This high efficiency stems from the interactions that these auxin analogues have with the TIR1 protein. When the analogues bind to the TIR1 protein promoting the formation of the Aux/IAA–SCFTIR1. This leads to the ubiquitylation of of Aux/IAA which usually acts as an inhibitor when bound to ARF transcription factors. Many of the plant's genes are under the control of the ARF transcription factor meaning that any alteration can have a large impact on the plant's morphology. When altered by persistent compounds that cannot be degraded easily (such as synthetic auxins), the plant suffers as it is not able to keep up with the high metabolic demand correlated with over-expression of genes.
Therefore, we believe that natural auxin will not function as a herbicide because it is a labile chemically that can be easily metabolized by the plants. However, there is still a danger that too much natural auxin can also be detrimental to the plants. Therefore, for this module we must consider this fact during our design process. The bacteria must be able to produce a certain amount of auxin and must not go above a certain threshold. This level of control makes it a perfect project for synthetic biology which is looking to design controllable components in natural systems.
Our specifications will therefore be influenced by our modelling as well as our experiments on Arabidopsis thaliana. These experiments include an auxin concentration assay where we will be looking at the effect of auxin on the root length as well as the total dry mass of the plant in phytogel and soil. We will also be looking through past literature in order to determine what concentrations of auxin are known to promote root growth. However, seeing as how we only have ten weeks for the project and these sorts of studies take a long time, we must create a construct that will be able to produce auxin and will be as modular as possible to allow for easy tinkering in later stages. We have decided to use the IAM pathway from Pseudomonas savastanoi since it has been shown to be expressed in Escherichia coli and is only composed of two enzymes.
For the first design of our Auxin Xpress module we decided to make it as modular as possible. We have introduced an insulator sequence which will allow us to test out different promoters in the future until we find a combination of components that will fit our specifications. We also decided to use a variant of the Pveg promoter to contribute further to our modularity as well as codon optimizing our sequences for both Bacillus subtilis and Escherichia coli. This flexibility allowed us to choose our chassis by calmly looking at the pros and cons of both bacteria. We came to the conclusion that E. coli would be a better chassis due to its ease of production as well as its inability to form spores which would make the issue of containment a nightmare (seeing as how we would be working with loose soil onto which the spores could attach themselves to and be blown far distances from the application zone).
Our modelling has looked into the rate constants of the two enzymes in our system (IaaM and IaaH) in order to calculate how much of the enzymes we would need to produce to create the desired output. However, in order to more accurately determine whether the equations give us a correct output we must first look at how much auxin our first version of this module will produce. The desired output has been modelled by using the data from the auxin effect experiments on A. thaliana through a process known as data fitting. Therefore, our modelling will allow us to calculate how many cells we would require on each seed or what promoter and RBS might be more suitable if a second version of our construct were to be built.
The first version of our module was built through the use of our new standard. Our new standard consists of ordering synthetically made DNA fragments with 50bp overlaps that are flanked by MlyI restriction sites. This allows for the easy blunt excision of the fragments from their expression vectors. Once the fragments have been excised one can put them together through the use of Gibson (making it a PCR-less Gibson) or CPEC. For this construct we used CPEC.
Finally, once the module was assembled we decided to use a simple colourimetric assay with Salkowski reagent. Salkowski reagent is commonly used in order to determine a rough estimate for the amount of Auxin present within a cell culture filtrate. If auxin is present, the reagent turns pinkish-red. For a more accurate analysis on the amount of auxin we are producing we are going to use HPLC with a reverse phase C18 column. Once these two assays are completed we fed the data back through the modelling and determined that
Retrieved from " http://2011.igem.org/Team:Imperial_College_London/Project/Auxin/Overview "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
