"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College London/test6
From 2011.igem.org
Engineering Cycle
Project AuxIn »
Home
At a Glance
The Problem
M1: Phyto-Route
M2: Auxin Xpress
M3: Gene Guard
Achievements »
Main Results
Data
Judging Criteria
Europe Jamboree
Human Practice »
Informing Design
Safety
Outreach
Extras »
Collaboration
Protocols
New Standards
Software
Diary
Brainstorming
Team
Main Results
1. Expression of indole 3-acetic acid (IAA) by our modified bacteria
Salkowski assay
We grew up cultures of our IAA-producing E. coli as well as a control culture overnight and performed a colourimetric assay on the cell filtrate in order to determine if there is a difference in the level of IAA. Our results showed that our Auxin Xpress cells produced more IAA than the control (see Figure 1 and 2).
Figure 1: Results from the Salkowski assay with cell filtrate of IAA-producing E. coli DH5α.(Data by Imperial College London iGEM team 2011).
Figure 2: Visual results correlating with OD measurements. The eppendorf on the right contains IAA producing E. coli DH5α and the eppendorf on the left contains control E. coli DH5α. (Data by Imperial College London iGEM team 2011).
Plant response
We exposed Arabidopsis seedlings to our IAA-producing bacteria to observe their effect on the roots. For this, we used a reporter line that expresses YFP in response to indole 3-acetic acid. As controls, we used a culture of seedlings that was not incubated with any bacteria and a control that was indicated with E. coli not expressing IAA. Fluorescence was brightest in the roots incubated with IAA-producing bacteria (Video 1).
Video 1. Stack images taken by confocal microscopy converted into three-dimensional videos of Arabidopsis thaliana DR5:3VENUSroots (data and imaging by Imperial College London iGEM team 2011).
In order to quantify the expression of fluorescence in the plants exposed to different treatments we processed the images in ImageJ and analysed fluorescence intensity in individual particles. The relative fluorescence intensity of roots exposed to our IAA-secreting bacteria is 1.93x that of roots exposed to no bacteria and 2.9x that of roots exposed to non-IAA-producing bacteria. It therefore appears that bacteria normally suppress production of IAA in root tips. However, the bacteria we have engineered to express IAA produce enough of the compound to not only overcome this limitation but to also increase expression of the reporter gene.
2. Uptake of bacteria into plant roots
We observed uptake of GFP-expressing bacteria into plant roots (Figure 3). The fact that (at least some) plants can actively break up bacteria and use them for nutrients was only reported last year[1]. This finding is of extreme importance to our project as the exposure of roots to indole 3-acetic acid will vary significantly if the uptake of the compound takes place inside the roots themselves rather than from the outside. This also presents a novel platform of indirectly engineering plants as delivery of a whole range of compounds will be possible using this method.
Figure 3: Escherichia coli cells expressing superfolder GFP (sfGFP) can be seen inside an Arabidopsis thaliana root using confocal microscopy after overnight incubation of the plants with bacteria. Roots were washed in PBS prior to imaging to avoid "false positives" of bacteria adhering to the outside of the root. (Data and imaging by Imperial College London iGEM team 2011).
To ensure that the bacteria we imaged were inside the roots, we took stack images that were later converted to three-dimensional views of the roots containing bacteria (Video 2).
Video 2. Confocal stack imaging showing sfGFP-expressing bacteria inside plant roots showing blue autofluorescence (data and imaging by Imperial College London iGEM team 2011).
3. Chemotaxis of our chassis expressing PA2652 construct towards malate
Figure 4: Dependence of bacterial chemotaxis to varied malate concetrations. Cells containing PA2652 ( BBa_K515102 ) have shown inreased number of cells in capillaries with inreasing malate concentration, with a peak at 1 mM. The number of cells drops sharply after 1 mM due to saturation. Negative control were cells without contruct. The cell count for negative control in each of the capillaries with increasing attractant concentration has not increased. Data collected by Imperial iGEM 2011.
In the Phyto-Route module of our system we have expressed the malate-reponsive chemoreceptor PA2652 in our chassis. The result of quantitative assay demonstrates chemotaxis of E. coli expressing PA2652 towards L (-) malic acid. The response towards different concentrations of malate has been quantified and the strongest response can be seen at 1 mM. These results show functionality of the Phyto-Route construct that enables movement of our bacteria towards plant root exudate malate. Once the bacteria are attracted towards roots, then they can be naturally taken up into the plant's roots.
4. E. coli survivability in soil
To assess how well our chassis Escherichia coli is able to survive in soil and for how long a plasmid can be retained, we set up an experiment that consisted of inoculating E. coli into sterile and non-sterile soil. After six weeks of incubation, samples grown up from sterile cultures still expressed fluorescence. Cultures exhibiting E. coli-like morphologies that were grown from non-sterile plates had lost fluorescence (Figure 5).
Figure 5: Colonies recovered from filter discs and grown on LB plates containing selective antibiotics imaged using a LAS-3000 gel imager. a) Sample taken from non-sterilised soil b) Sample taken from sterilised soil  (Data by Imperial College London iGEM team 2011).
A restriction digest revealed that cultures from sterile and non-sterile plates that display E. coli-like morphologies have both retained the plasmid. Sequencing of the insert revealed a frameshift mutation in the coding region of the sfGFP gene in the colonies from non-sterile soil. Both samples will be sent off for 16S rRNA sequencing to identify the bacterial species. However, it seems very likely that our GM E. coli have been able to survive in soil and retain their plasmid for six weeks despite competition and selective pressure against the plasmid.
5. Tracking of bacterial metabolic activity using Dendra2
Dendra2 is a photoconvertable protein[2] that normally exhibits green fluorescence but can be switched, using single photon activation, to fluoresce red. This conversion cannot be reversed. Dendra2 is therefore very useful for observing the metabolism of cells: if all protein in a cell is converted to red fluorescence and it subsequently expresses green fluorescence, the cell has to be metabolically active. As we wanted to know if our cells are metabolically active once they have been taken up into root cells, we performed another plant uptake experiment.
We have made a novel fluorescent protein, Dendra2 available as a BioBrick. Dendra2 normally fluoresces green but can be irreversibly converted to red fluorescence. This is very useful for tracking the viability of cells in different circumstances (for instance, after they have been taken up into root cells). We did a plant uptake experiment with Dendra2-expressing E. coli and converted the protein using a 405 nm laser on a confocal microscope (Figure 6 and Video 3).
Figure 6: Dendra2 photoconversion in bacteria taken up inside plant roots. The graph on the top displays averaged fluorescence over the entire photoconverted area and the amount of brightfield light (the background light used to see the outline of the roots) recorded is therefore very high. The graph on the bottom displays emission at green and red fluorescence over the same time span. (Data by Imperial College iGEM team 2011).
Video 3. This video shows the photoconversion of Dendra2 within E. coli cells that have been taken up into the plant roots as a time-lapse of pictures taken after each round of bleaching at 405 nm. The targeted area of cells being photoconverted corresponds to the top graph in Figure 5. There is a single bacterium visible on the right that was not targeted for photoconversion and serves as a control (data and imaging by Imperial College iGEM team 2011).
References:
[1] Paungfoo-Lonhienne, C. et al. (2010) Turning the table: plants consume microbes as a source of nutrients. PLoS One, 5(7), e11915.
[2] Gurskaya N et al. (2006) Engineering of a monomeric green-to-red photoactivatable fluorescent protein induced by blue light. Nature Biotechnology 24: 461-465.
Retrieved from " http://2011.igem.org/Team:Imperial_College_London/test6 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
