"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College London/Project Chemotaxis Assembly
From 2011.igem.org
Project AuxIn »
Home
At a Glance
The Problem
M1: Phyto-Route
M2: Auxin Xpress
M3: Gene Guard
Achievements »
Main Results
Data
Judging Criteria
Europe Jamboree
Human Practice »
Informing Design
Safety
Outreach
Extras »
Collaboration
Protocols
Software
Diary
Brainstorming
Team
Module 1: Phyto-Route
Chemotaxis is the movement of bacteria based on attraction or repulsion of chemicals. Roots secrete a variety of compounds that E. coli are not attracted to naturally. Accordingly, we engineered a chemoreceptor into our chassis that can sense malate, a common root exudate, so that it can swim towards the root. Additionally, E. coli are actively taken up by plant roots, which will allow targeted IAA delivery into roots by our system.
Overview
Specifications
Design
Modelling
Assembly
Testing & Results
Future Work
Assembly
Figure 1: Assembly strategy for our Phyto-Route construct. (Figure made by Imperial College London iGEM team 2001).
We ordered the PA2652 coding sequence in two fragments to minimise cost and time (Figure 1). We did not want to use PCR to amplify the fragments in order to avoid introducing mutations into our final construct. Accordingly, we engineered blunt end cut sites on either side of our synthesized sequences with the MlyI restriction enzyme. MlyI (a type II restriction enzyme) cuts bluntly at a distance of 5 bp from the recognition site. This property permitted us to excise only the coding sequence of each fragment. Each digested fragment was then gel extracted in preparation for assembly.
The pSB1C3 vector was simultaneously inverse PCR'd to amplify the backbone vector with the required overlaps.
We planned to combine the two fragments into the pSB1C3 vector by Gibson assembly. Unfortunately, our attempts failed. We postulate that this was due to homology on the backbone vector, causing it to re-anneal.
We reverted to CPEC to assemble the construct, a method that requires more extensive use of PCR than Gibson. This said, by introducing MlyI restrictin sites, we were able to halve the number of PCR steps required, thereby reducing the risk of potential mutation.
CPEC (Circular Polymerase Extension Cloning) is a primer-independent PCR assembly technique which relies on overlaping sequences between each part to be assembled. With a denaturing step, the double stranded DNA is melted, allowing compatible single stranded ends of each part to joined. For this reason it is essential that the parts are designed with homologous ends (the fragments we used were designed with 60 bp overlaps). The annealed overlapping ends then serve as primers for polymerase extension to join the parts into a seamless construct.
DNA was mini-prepped from colonies and sequenced by Eurofins. The sequences came back positive so we could move on and start characterizing the PA2652 construct.
The second sequencing fragment of our codon optimised mcpS receptor has still not arrived and could therefore not be assembled. However Juan L. Ramos of Consejo Superior de Investigaciones Científicas in Spain was able to provide us with a non-codon optimised mcpS gene with unknown promoter and ribosome binding site in the pRK415 vector. We have performed chemotaxis assays on this construct as well.
Figure 2: The assembled construct was digested with EcoRI and SpeI which cut at the backbone prefix and suffix respectively to determine whether the insert size correlates to the expected size of the assembled fragments. Lanes 1 and 4 contain a 1 kb+ DNA ladder as reference. Lane 2 shows the construct cut with EcoRI and lane 3 with SpeI. The band of the insert is about 2 kb and correlates with the expected size of PA2652. (Data by Imperial College London iGEM team 2001).
Figure 3: Schematic of the final construct. Click on the parts to be directed to the registry. (Figure made by Imperial College London iGEM team 2001).
M1: Modelling M1: Testing & Results
Retrieved from " http://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Assembly "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
