<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Project_Gene_Modelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Project/Gene/Modelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Project/Gene/Modelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Project/Gene/Modelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Project/Gene/Modelling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Project/Gene/Modelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Project/Gene/Modelling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">Project AuxIn<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London">Home</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Tour">At a Glance</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project/Background">The Problem</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Overview">M1: Phyto-Route</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Overview">M2: Auxin Xpress</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Overview">M3: Gene Guard</A></LI></UL><LI><A class="sf-with-ul" href="#">Achievements<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Achievements">Main Results</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Judging">Judging Criteria</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Europe_Jamboree">Europe Jamboree</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Overview">Informing Design</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human/Outreach">Outreach</A></LI><LI><A class="sf-with-ul" href="#">Extras<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Extras/Collaboration">Collaboration</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Protocols_General">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Software">Software</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Notebook/week1">Diary</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Brainstorming_Health">Brainstorming</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Team">Team</A></LI></DIV><H1>Module 3: Gene Guard</H1><P><B>Containment is a serious issue concerning the release of genetically modified organisms (GMOs) into the environment. To prevent horizontal gene transfer of the genes we are expressing in our chassis, we have developed a system based on the genes encoding holin, anti-holin and endolysin. We are engineering anti-holin into the genome of our chassis, where it acts as an anti-toxin, and holin and endolysin on plasmid DNA. In the event of horizontal gene transfer with a soil bacterium, holin and endolysin will be transferred without anti-holin, rendering the recipient cell non-viable and effectively containing the Auxin Xpress and Phyto-Route genes in our chassis.</B></P><DIV class="underlinemenu"><UL><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Specifications">Specifications</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Design">Design</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Modelling">Modelling</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Assembly">Assembly</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Testing">Testing &amp; Results</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Future">Future Work</A></LI></UL></DIV><H1>Modelling</H1><H2>I. INTRODUCTION</H2><P>The gene guard device involves three proteins:</P><P>1)	Holin is a protein forms pore complexes in the inner-membrane and this pores allow endolysin to move into periplasmic space. </P><P>2)	Endolysin is an enzyme that could break down the cell wall and causes cell lysis. </P><P>3)	Anti-holin binds to holin and prevent from forming pores. </P><P>Our system will have holin and endolysin genes together with axuin genes and chemoreceptor genes on the plasmids, and anti-holin genes on the genome under control of strong promoter. The production of anti-holin will prevent cell lysis of the genetic modified bacteria by deactivating holin. If the plasmid get horizontal transferred to a different cell without anti-holin on its genome(e.g. any wild type soil bacteria), the expression of holin and endolysin will induce cell lysis and hence prevent it from keeping the plasmid containing auxin gene.
</P><H2>II. OBJECTIVES</H2><P>    The key to the success of this device is the right ratio of holin and anti-holin production such that the anti-holin can deactivate all the holin produced in our genetic modified bacteria, and the holin expression level in wild type bacteria is high enough to induce cell lysis. As the gene expression is controlled by both promoter strength and ribosome binding site (RBS) strength, therefore modeling of gene guard is important for helping us to choose appropriate promoters and ribosome binding site (RBS) to inhibit and optimize cell behavior in our genetic modified cell and soil bacteria respectively. </P><H2>III. DESCRIPTION</H2><P>   The modeling of gene guard consists two parts, one is the holin production in wild type bacteria and the other part is the deactivation of holin by anti-holin in genetic modified bacteria. Since all the genes in our system are constitutively expressed, thereby we only consider steady state. </P><H3>1. Holin production in wild type soil bacteria</H3><P>This part of modeling focused on deriving the relationships between promoter strength and RBS strength of holin on single plasmid. In order to model this, the following assumptions were made: </P><P>1)	After plasmid transfer and duplication, there will be 50 copies of plasmids in one bacterium. </P><P>2)	The literature indicates that cell lysis happens at holin concentration equals to 1000/cell[1]. </P><P>Based on above assumptions, the kinetics of cell lysis in wild type bacteria was modeled using ordinary differential equations (ODEs) for the holin mRNA and holin (Equation 1).  </P><H3>2. Holin inhibition in Genetic Modified bacteria</H3><P>    In genetic modified bacteria the promoter strength and RBS strength of anti-holin on the genome should be stronger than promoter strength and RBS strength of holin on single plasmid such that all the holin produced by 300 copies of plasmids should be inhibited, where 300 is the maximum number of plasmids can be in single bacterium. In this system, the anti-holin can bind to holin to form dimer, which is defined as the deactivated holin. The mechanism in the genetic modified bacteria can be described in following ODEs in Equation 3.</P><H2>IV. RESULTS</H2><H3>1. Holin production in wild type soil bacteria</H3><P>At steady state (i.e. d[ ] /dt = 0), rearrange Equation 1 with [Holin] = 1000, we obtained an expression for P_(holin) K_(holin) as shown in Equation 2: </P><H3>2. Holin inhibition in Genetic Modified bacteria</H3><P>    Again at steady state with d[]/dt = 0, we obtained an equation of holin concentration (Equation 4).  By varying the ratio m in equation 4, we obtained that the holin concentration in genetic modified bacteria decreased to zero at ratio m ≥ 400 (see Figure 1). Then, we tested m = 400 with the anti-holin promoter we have with strength 13.1pg RNA/min/μg substrate DNA using ODEs shown in Equation 1 and 3, the result (Figure 2) shown that all the holin in the cell are inhibited as the holing concentration remains at 0 all the time, while the holin concentration in wild type bacteria reaches 1000 per cell after 5000 seconds. 

</P><P><B>Figure 1: The holin concentration against the ratio m</B></P><P><B>Figure 2: Evolution of anti-holin and hoin in genetic modified and wild type bacteria agiainst time(s)</B></P><H2>V. MATLAB CODE</H2><H2>VI. REFERENCE</H2><P>   [1]  Christos G. Savva, Jill S. Dewey, John Deaton, Rebecca L. White, Douglas K. Struck, Andreas Holzenburg and Ry Young. The holin of bacteriophage lambda forms rings with large diameter. Molecular Microbiology 69(4), 784–793. 2008.

</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Imperial_College_London/Project/Gene/Modelling">http://2011.igem.org/Team:Imperial_College_London/Project/Gene/Modelling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Project/Gene/Modelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Project/Gene/Modelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Project/Gene/Modelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Project/Gene/Modelling&amp;oldid=119873" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>