"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College London/Project/Auxin/Results
From 2011.igem.org
Project AuxIn »
Home
At a Glance
The Problem
M1: Phyto-Route
M2: Auxin Xpress
M3: Gene Guard
Achievements »
Main Results
Data
Judging Criteria
Europe Jamboree
Human Practice »
Informing Design
Safety
Outreach
Extras »
Collaboration
Protocols
Software
Diary
Brainstorming
Team
Auxin Results
Chapter 1: Assembly of genetic constructs
We wish to build a single expression plasmid that can express IaaH and IaaM. While this task can be summarised in one sentence its execution is not as short. The first problem lies in the size of these two enzymes which both exceed 1kbp making their synthesis a problem. We therefore created a new standard for biobrick assembly to tackle this issue. We broke up these large sequences into four fragments that were ordered at the end of week 3. In preparation for the arrival of these fragments (circa 8-10 days) we started to transform our cells with the pVEg+pSB1C3 backbone constructs in order to make enough genetic material for a gibson assembly reaction. This chapter will describe our struggles and successes throughout this grueling and yet rewarding process.
29th of July
Our first transformations were a success! We have managed to transform our competent cell colonies with pSB1C3 containing BBa_K398500 and J23100 promoter to produce cell line 6. We also transformed the cells with part BBa_K316001 to produce cell line 7 and part BBa_K316005 to produce cell line 8. With these cell lines we will be able to make more copies of each part in preparation for the arrival of our synthesized sequences. We are under a tight schedule so efficiency is key.
Also, the cell line with the superfolded GFP integrated in its genome has been plated successfully. We have confirmed that these are the correct cell lines by looking at them under UV light. Their green glow was brighter than we expected.
However, cell line 1 did not grow in the liquid broth media at the same concentration of kanamycin. We will create an assay to ascertain the optimum kanamycin concentration. This is both a bizarre and unexpected result that must be rectified.
30th of July
The previous day, we had obtained the mcpS gene in a pRK415 plasmid from Spain. We then performed a transformation on the competent 5α cell line and obtained the results on this day. Out of the four plates one of them contained transformed cells. We named these cells 10.
1st of August
A month has already passed since we started our project. The experiments that were conducted on this day were a mini-prep on samples 6,7,8 and 10 that were inoculated into an LB broth culture the night before. We used a mini-prep kit to obtain the plasmid DNA that we wanted from the transformed cells. Then, once we obtained the DNA, we started a restriction digest using PstI and EcoRI for 1.5 hours to confirm that the samples from the mini-prep contained the DNA that we wanted (the pSB1C3 backbone with the appropriate promoters).
Sadly, once we tried to visualize the results on a gel, the results were less than satisfactory. Mini-prep 6a and 7b seems to have failed and the rest of the bands do not make much sense. The experiment had to be repeated.
2nd of August
Today we attempted the restriction digest again and the results verified that samples 6,7 and 8 were actually pSB1C3 constructs with the required components. Therefore, the mini-prep experiment had worked and did not need to be repeated. It is a great feeling when everything comes together.
Gel 1&2. Restriction digest of three backbone vectors with EcoRI(E) and PstI(P) to confirm backbone length. Gel1: Lane1-1 kb DNA ladder Lane 2-6a cut with E;Lane 3-6a cut with P; Lane 4-6a cut with E+P;Lane 6-6b cut with E; Lane 7- 6b cut with P; Lane 8- 6b cut with E+P; Lane 11- 7a cut with E; Lane 12- 7a cut with P; Lane 13- 7a cut with E+P; Lane 15- 7b cut with E; Lane 16- 7b cut with P; Lane 17- 7b cut with E+P. Gel 2: Lane 1 - 1kb ladder; lane 2 - 8a cut with E; lane 3- 8a cut with P; lane 4- 8a cut with E+P; lane 6- 8b cut with E; lane 7- 8b cut with P; lane 8 - 8b cut with E+P.
3rd of August
Today we attempted to PCR samples 6, 7 and 8. The PCR did not end up working so well so. The annealing temperature must have been too low because we obtained bands that we did not want. Also, there must have been too little Sybr safe in the gel because the bands that were there were not bright. The PCR will be repeated once again and we will use a temperature gradient for (hopefully) better results.
4th of August
Today we redid the PCR of samples 6, 7, and 8 but with a temperature gradient to improve primer annealing.... and it was a success! So tomorrow we can run the rest of the Dpn1 digested DNA on a gel and gel purify it, then the vectors are ready to be used for DNA assembly once our genes arrive!
Gels 3&4: Temperature gradient PCR of desired backbone with promoter and terminator out of plasmids. Gel 3:Lane 1- 1kb DNA ladder; lanes 2 to 7 - PCR of vector 6 from 57.1°C to 62.6°C; lanes 9 to 15 - PCR of vector 7 from 57.1°C to 62.6°C. Gel 4: lanes 2 to 7 - PCR of vector 8 from 57.1°C to 62.6°C
The backbone sequences have also returned. All of the samples are in order except a one base pair mutation in sample 8 within the pVEg promoter. For now, we are going to amplify it anyways in the hope that the one base pair mutation is just an error that occured during sequencing. Either way, sample 8 is a back-up of sample 7 so there should be no problems either way.
We also started experimenting with the Salkowski reagent. In particular, we tested the S2/1 method and found that the ideal wavelength for measuring the IAA concentration with our apparatus is at 554nm. This was done by scanning between 500nm and 600nm to obtain the individual absorption spectra of each sample. The wavelength at which the absorbance peaked was chosen. This experiment gave us a great standard curve from which we can roughly estimate the amount of IAA in a solution between 2 and 200 μg/ml.
8th of August
Today we ran a gel of the PCRd backbone DNA extracted from the previous gel to make sure that the DNA was pure. The gel results were succesful. We also transformed cells with the pure DNA to check that the Dpn1 digestion worked properly.
Gel 5: Gel extracted backbone vector DNA run on a gel to confirm purity. Lane 1- 1 kb DNA ladder; Lane 2- vector 6a; Lane 3- vector 6b; Lane 4- vector 7a; Lane 5- vector 7b; Lane 6- vector 8a; Lane 7- vector 8b.
9th of August
Transformations of auxin fragment 1 (20) and auxin fragment 4 (24) were successful. We obtained plenty of colonies to choose from on both the ampicillin and kanamycin plates. Also, the DpnI digest transformations created bacteria that had no resistance to chloramphenicol.
However, sample 8 has to be repeated because in the "rest" plates there were 1 or 2 colonies on each.
10th of August
Today we attempted to perform a midi-prep on the DNA fragments that had arrived from Germany. The experiment failed and we were not able to obtain a decent yield of DNA. Oh well, got to try again.
12th of August
After the failed results and the rather lethargic week we attempted to get our minds back to gear. The visit to Syngenta the day before was a moment of respite that allowed us to perform two mini-preps that worked. However, there was an issue when we digested our plasmid (pCR2.1) with MlyI. The genes had been placed in a plasmid that contains multiple MlyI sites which would make the gel extraction more difficult. We used Serialcloner to make a virtual cut and then used the predicted image to guide us. In the end we obtained a band for 20b and 24a in the right location (or so we hope!).
Gel 6: gel of MlyI restriction digest of the synthesised genes. Lane 1-Marker; Lane 2-20a; Lane 3-20b; Lane 4-22a; Lane 5-22b; Lane 6-23a; Lane 7-23b; Lane 8-24a.
15th of August
We gel extracted several gene fragemtns that were transformed yesterday.
16th of August
The transformations we did the previous day were a success and the last two remaining fragments were also mini-prepped. We will attempt the Gibson assembly. If all goes well, we'll have E. coli excreting auxin by Friday!
19th of August
Gel 7: Gel extracted DNA fragments for assembly of auxin expressing plasmid. Lane 1&2 - Auxin fragment 1; Lane 3 - Auxin fragment 4; Lane 4&5 - Auxin fragment 3; Lane 6&7 - Auxin fragment 2; Lane 8&9 - pVEG backbone vector
22nd of August
Today we obtained some disappointing results. It seems like the vector is just religating during the Gibson reaction. Maybe the sequences of the two ends of the vectors are too homologous for Gibson to work. Either way, we will be attempting CPEC today and hopefully we will obtain some bands that we can purify and transform bacteria with.
23rd of August
First try of CPEC assembly assembly of the auxin construct seems to have been successful by looking at the gel electrophoresis results of colony PCR and CPEC product PCR with sequencing primers. We expect sequencing results to arrive tomorrow for an accurate assessment, however a preliminary test of colony supernatant with the salkowski reagent showed promosing results with one colony which turned bright red, indicating the presence of auxin.
Gel 8: The first two lanes show that CPEC assembly of four auxin fragments at ~1kb each in a backbone of about 2kb. Lane one contains the assembled construct at ~6 kb and lane 2 contains the negative control assembly of backbone vector with no insert at ~2kb. The following two lanes show the analytical PCR of the CPEC assembled product with standard biobrick primers to PCR our the assembled auxin fragments. The first well shows the auxin assembly at ~4kb and the second (negative control) shows no PCR product because no insert is present. Gel 9: Colony PCR with standard biobrick primers of CPEC assembled auxin fragments showing the desired assembly size of about 4 kb. Gel 10: Colony PCR of negative control colonies (backbone vector 8 only and no insert) and positive control colony PCR of the same vector 8 but the entire plasmid. This result shows that the DpnI digest of PCRd backbone vector 8 was not completely efficient as some complete plasmid remains, but this residual amount did not hinder assembly.
25th of August
A restriction digest of the mini-prepped auxin assembled constructs with EcoRI and PstI show clearly the assembled auxin insert drop out of the backbone vector.
Gel 11: Restriction digest of auxin construct with EcoRI, PstI and both for three miniprepped colonies grown from transformed E. coli.
Chapter 2: Auxin assays
How much auxin will we be producing? More importantly, has the module actually worked?! This chapter will look into the methods that we have decided to use in order to measure the amount of auxin in a solution. We have decided to use qualitative methods such as the Salkowski reagent (changes colour which is always good) as well as quantitative methods such as HPLC and GC-MS for more accurate results. Cross our fingers and hope this works!
5th of August
After successfully testing the S2/1 method we also attempted the PC method which is both more exact and more specific for IAA. However, it only works at a lower range of concentrations of IAA and will therefore be useless if our bacteria end up excreting more than 20 μg/ml of IAA into the solution. Either way, we now have two standard curves which can be used to measure the amount of IAA in a solution. We are ready for the synthetic genes to arrive!
8th of August
Today we performed the first PC Salkowski assay successfully. We even took a picture of the gradient:
This was one of the 3 repeats that gave us the following standard curve:
26th of August
Today we have determined that the Salkowski reagent that we are currently using is not specific enough to detect indole-3 acetic acid. Instead, it is reacting with random indoles found within the supernatant and the cell samples. After working the entire day in order to obtain a conclusive answer on whether the cells are producing auxin or not has led to a frustrating result where the current reagent's only use is that of a very elaborate cell density measurement system, ergo useless. We will attempt HPLC next week as well as do some further reading on what we might have done wrong.
10th of September
The graph below showed the auxin expression level in both E.coli transformed with auxin construct and the control E.coli. We are going to repeart this expriment with tryptophan supplment as the auxin pathway heavily denpends on tryptophan supply.
Chapter 3: effect of auxin on plants
We are not only interested in constructing the auxin-producing pathway in our bacteria but we also want to investigate what effect the auxin has on plants to verify our assumptions about indole 3-acetic acid's effects. This will help us with the human practices aspect of our project and it will also provide a good assay for the functionality of auxin-secreting bacteria.
Root growth at different auxin concentrations. Tuesday, 9 August 2011
To look at the effect of auxin on plants, we supplied differing indole 3-acetic acid concentrations to Arabidopsis seedlings in liquid culture.
Si modelled the concentration of auxin secreted by our bacteria to be 10mM. However, according to  Joseph et al (1995), increasing exogenous IAA concentrations from 0 - 0.1 nM increases root growth from 0 - 20%. From 0.1 nM to 10microM, the root length and decreases sequentially, while fibrosity increases. The plant dies at concentrations over 10 microM. However ___ state that the optimal auxin concentration lies in between 0.5 microM - 20 microM. Accordingly, we used concentrations starting from 10mM to 0.1nM to test the effect of different auxin concentrations on the length of the roots and their branching. We made the auxin concentrations by serial dilution and added 10ml of concentrated auxin solution to 100ml of half-MS media each. Twenty-five seeds were added to each flask. The seeds were incubated at 23°C and wrapped in aluminium foil to allow the plants to germinate in the dark. They will be allowed to grow in the light in 3 days' time. This follows a protocol described by King et al. (1995).  The results are stated in Friday 19th of August and Monday 22th of August
Root growth at different positions where Auxin is applied Wednesday, 10 August 2011
Looking at the effect of auxin on the roots also involves observing its effect on phytogels. On these gels, individual seedlings grow horizontally into a gel containing plant nutrients. These gels enable us to supply the plant with auxin at set distances from the seedling itself. Observing these effects is especially in case our bacteria do not get taken up into roots in nature, which has yet to be investigated. We are using DR:3VENUS seeds. These germinate into plants whose roots respond to auxin uptake by expressing YFP. We hope to be able to get an estimate of the effect of auxin by comparing intensities of fluorescence across plants supplied with different concentrations of auxin.
Si corrected the estimated auxin secretion of our bacteria to 0.0001 to 0.01 mM. Accordingly, we set up the auxin concentration gradient experiment with 0.0001, 0.001 and 0.01 mM of IAA. We injected a small volume of IAA dissolved in 70% ethanol at set distances from seeds, which were subsequently put onto the gel. Five replicates were set up for each concentration. On each plate, the seed seeds are sown at distances of 2,4,6,8, and 10 cm from the point where IAA is applied. Roots grow perpendicular to the line on which IAA and seeds are applied. The plates are kept in low light for 3 days in order to prevent photooxidation of auxin and also in 4 C to simulate the winter hibernation. The plates are later put in light for another 6 days to see root growth.
The split root experiment (Friday 12 and Monday 15 August 2011)
To visually compare the difference of the root growth between the applying and non applying of IAA in the same plant, we set up a split root experiment (which had been recommended to us by Dr Alex Milcu). This experiment extends beyond the scope of normal controls as the same plant is subjected to two different treatments. We supplied the following concentrations of IAA to one half of the roots: 0 microM (control), 0.1 microM, 1 microM and 10 microM, while the other half is grown in phytogel containing no IAA. 3 replicates were set up for each concentration. 7-day old seedling of A thaliana DR:3VENUS were used for this set up. This strain responds to auxin by expressing YFP. The plates are sealed and kept in the incubating room for 2 weeks in order to observe the length of the grown root.
Friday, 19 August 2011
We imaged the plants that had been incubated with differing concentrations of IAA using confocal microscopy. Plants incubated with 0.1mM of IAA showed strongly enhanced lateral root growth but also stunted growth.
A Z stack through an Arabidopsis root tip incubated with 0.1mM indole 2-acetic acid.
This did not occur at lower concentrations. However, fluorescence was still clearly visible:
Root of A. thaliana seedling incubated with 1uM indole 3-acetic acid.
At even lower concentrations, fluorescence was much weaker:
Incubated at 0.01nM.
Effect of auxin concentrations results Monday, 22 August 2011
The results shows that from low (0.1nM) to high concentrations (0.1mM)the root legth and the number of the leaves decrease seqeuntially and the plants all die at 10mM. This is an important information for the modelling team since auxin only operates well at liquid concentrations less than 0.1nM. To find the optimal concentrations of auxin for maximal root growth and splitting. We extend the contrations from 0.1 nM lower down from 10 pM, 1 pM, 0.1 pM and 0.01 pM. Hopefully we could produce the bell shape curve graph for both the root's length and its fibrousity
Effect of auxin concentrations in phytogel Tuesday, 23 August 2011
In order to modelling the effect of auxin concentrations on Arabidopsis and build the macros on lateral root initiation and elongation, root length should be measured day by day. This could not be done in liquid media since taken the plant out of liquid media makes the plant prone to fungal contamination. Also the resources we have, e.g. number of shakers, amounts of Venus Arabidopsis seeds are not enough to build up repeats to measure root length everyday. Therefore we decide to grow Arabidopsis in phytogel which is less reliable than the liquid media
Soil erosion experiment Thursday, 25 August 2011
In order to see which root architecture, the length and the dispersion, is the best to hold up the soil to prevent soil erosion and retain the moisture inside the soil. We then vary 8 concentrations from 0.01 pM, 0.1 pM, 1 pM, 10 pM, 0.1 nM, 10nM, 0.001mM, and 0.1mM as the same as the effect of different auxin concentrations experiment. These different auxin concentrations allow the roots to show different architectures. The experiments are planned up as shown in the diagrams below.
- The material for making the slope is the 20 cm x 10 cm pots placing on top of the 30 degree slope.
- The soil chosen is called M2 which composes of organic compost without any gravel to eliminate the error from different mixtures which contributes to different soil contents. The soil immitates normal
- The pressure of water is kept constant by the adaptable shower head which immitates the average speed of rainfall (9.8 m/s)
The experiment is set up where 18 (3 columns at 10 cm side x 6 rows on 20 cm side) arabidopsis are seeded on the pots which represents each concentration. At each concentration, the experiment is repeated 3 times. These altogether with 2 controls where no auxin is watered in the first one and no Arabidopsis is seeded in the second one, result in 30 pots in total. Estimately 12.5 kg of soil (1/4 of 50kg M2) is used for the experiment. The soil is pretreated with 2 litre of 1mM fungicide solution and left dryout for 1 day. It is then seperated to approximately 400g on each pot and is watered each with 25ml of water to moist up the soil conditions for Arabidopsis seeding. A small hole is digged onto each seeding place to fix the position of the seeds. Arabidopsis is seeded with the pattern already described. For the first 3 days, the seeds are watered everyday at 4 pm to promote the growth and the watering is decreased to once in a 2 days afterwards. Each pot is watered with 25 ml of auxin concentrations mentioned above.
Separate modeling page
Retrieved from " http://2011.igem.org/Team:Imperial_College_London/Project/Auxin/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
