<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Extras_Brainstorming_Energy_test"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Extras/Brainstorming/Energy_test">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Extras/Brainstorming/Energy_test&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Extras/Brainstorming/Energy_test&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Extras/Brainstorming/Energy_test&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Extras/Brainstorming/Energy_test" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Extras/Brainstorming/Energy test</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">Project AuxIn<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London">Home</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Tour">At a Glance</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project/Background">The Problem</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Overview">M1: Phyto-Route</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Overview">M2: Auxin Xpress</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Overview">M3: Gene Guard</A></LI></UL><LI><A class="sf-with-ul" href="#">Achievements<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Achievements">Main Results</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Judging">Judging Criteria</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Europe_Jamboree">Europe Jamboree</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Overview">Informing Design</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human/Outreach">Outreach</A></LI><LI><A class="sf-with-ul" href="#">Extras<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Extras/Collaboration">Collaboration</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Protocols_General">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Software">Software</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Notebook/week1">Diary</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Brainstorming_Health">Brainstorming</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Team">Team</A></LI></DIV><H1>Ideas</H1><P><A href="#Soil">Soil-Solidifying Bacteria</A><A href="#Rain">Making Rain with Bacterial Spores</A><A href="#Isoprene">Greenhouse Gases (Isoprene)</A><A href="#Green">Filling in the Green Gap</A><A href="#Auxin">Auxin-Secreting Bacteria</A><A href="#C4">Turning C3 Plants into C4 Plants</A><A href="#Nematodes">Bacteria Targeting Parasitic Nematodes</A></P><H1><A name="Soil">Soil-Solidifying Bacteria</A></H1><P>Sporosarcina pasteurii or Bacillus pasteurii from older taxonomies is a bacteria with the ability to solidify sand given a calcium and an organic nitrogen source through the process of biological cementation. This will be a good recycle of land waste, urea waste and a food waste.</P><P>However solidification requires a high pH and produce toxic ammonium waste. Even though ammonia increase the pH this should be control using synthetic biology to model the right amount. ammonium can be subjected to other products which we are still searching for. Another application might be using ammonium produce to tighten the dye we made using the pigment which might allow the full house to be made easily from the brick. (Text by Ming)</P><P><A href="#Top">Back to top</A></P><H1><A name="Rain">Making Rain with Bacterial Spores</A></H1><P>The surface of spores can form crystal structures that attract water vapour from the air and form rain drops. We wanted to use this priniciple to have a useful output from the spores when they reach the ground - which could be sand or soil.</P><P>There are several genes localized that contribute to the spore outer layer which could potentially be randomly mutated to optimize the geometric shape of the spore to promote water droplet formation.</P><P>One of the drawbacks to using spores is that we need germination to occur for gene expression and there is an obvious risk of releasing GMOs into nature. One option could be to engineer in a death response to have limited gene expression of mucin for example which could aid in water retention, or auxin to promote root growth deep into soil.</P><P><A href="#Top">Back to top</A></P><H1><A name="Isoprene">Greenhouse Gases (Isoprene)</A></H1><P>1.Water vapor</P><P>2. Carbon dioxide - done to death 3. Methane 4. Ozone</P><P>CFC's, VOC's and Nitrous oxide. CFC's are banned. However, chlorine radicals still a problem. isoprene is produced by plants to prevent oxidative stress and heat shock. VOC. Byproduct of the thermal cracking of naphtha or oil. Used to produce natural rubber... Produced from the precursor DMAPP DMAPP is made by isopentenyl pyrophosphate isomerase from IPP. IPP is also one of the precursors of of lycopene. You need 3 IPP, 1 DMAPP and 1 GGPP (Geranylgeranyl pyrophosphate) to produce Phytoene. Insert lycopene biosynthesis pathway and inhibit isoprene synthase? Probably too complicated.</P><P>1. Johan E. T. van Hylckama Vlieg et al., “Characterization of the Gene Cluster Involved in Isoprene Metabolism in Rhodococcus sp. Strain AD45,” Journal of Bacteriology 182, no. 7 (April 2000): 1956-1963.</P><P>Rhodococcus sp. strain AD45 Isoprene is supposed to recycle OH. NOx causes oxidant build-up. Mechanism for NOx removal? NO sources: Internal combustion engine. Cannot target. NO2 sources: Internal combustion engine, thermal power station, pulp mill. None of which could be targeted. Pulp mill industry is one of the largest producers of water pollution in the world. An alternative to paper could solve a lot of problems. Still unfeasible. Containment is an issue.</P><P><A href="#Top">Back to top</A></P><H1><A name="Green">Filling in the Green Gap</A></H1><P>-plants absorb light wavelength from 400-500nm and then 600-700nm however between 500 - 600nm, plants do not have any light accepting compounds in that light range</P><P>-there are two possible ways of solving the problem: 1.introduce lightharvesting pigments from red algae and couple them to plant photosystems 2.use any fluorophore which accepts light in the green gap and emits the light in wavelength higher than 600nm, localise it on the membrane on chloroplast and therefore transfer wavelength between 500-600nm to a wavelength higher than 600nm so plant phytochromes can accept it -it appears that red algae combine the two methods in phycobilisomes, where they use pigments which emit wavelengths a bit higher than the emmited wavelength until chlorophyll is capable of accepting light -possbility of introducing phycobilisomes or only specific phycobiliproteins, which accept (500-600nm) and re-emit it in 600-700nm -problem: does not seem to be a single protein for accepting and re-emitting of the corresponding wavelengths -problem: do not know the number of phycobiliproteins neccessary to significantly contribute towards excitation of plant phytochromes -problem: plant might be losing energy by synthesising extra pigment proteins (if it needs a lot of them) even though these proteins would contribute to produce more energy in the plant</P><P><A href="#Top">Back to top</A></P><H1><A name="Auxin">Auxin-Secreting Bacteria</A></H1><P>The project intends to enhance the ability of plants to grow roots towards specific locations. For example, this could be used to allow plants to quickly locate the nutrients they need with minimal energy expenditure. Another example is to increase the speed at which roots grow, in order to hold down soil that is in danger of being eroded.</P><P>This would be done using a two bacteria system, so that one strain of bacteria would associate closely with the roots of a specific plant and another strain could locate the target and secrete a long-distance signalling molecule that would cause the root-associated bacteria to secrete auxin, causing the roots to grow. To model this, a simple system will be used, using Azotobacter paspali, which associates specifically with the roots of paspalum grass. The bacteria would be modified to secrete auxin in a dose-dependent fashion in response to a signal. The exploratory strain of bacteria would be engineered to seek out specific conditions and to release a signalling molecule. The signalling molecule could be glutamate, calcium, possibly a homoserine lactone in a vesicle, Paris 2009. In this paper, <A href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0011915" rel="nofollow">http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0011915</A> , it was shown that tomatoes and arabidopsis plants are able to take up microbes to degrade them. The bacteria were alive for around 10 days before they were all completely gone. It might be possible to have these bacteria secrete nitrates and auxin once inside the cell, or over-express it so that it is released when the bacteria are lysed by the plant. The bacteria could potentially be engineered to thrive inside the plant roots. For this, it is important to note that it is the plants that break down their own cell walls and actively take up the bacteria. Could also use yeast.</P><P><A href="#Top">Back to top</A></P><H1><A name="C4">Turning C3 Plants into C4 Plants</A></H1><P>problem: How to make C3 plant operating in sunny and arid areas or how to reduce photorespiration</P><P>solution: Create a bacteria which penetrates plant cells, creates high concentration of HCO3− and packages it into vesicles, inactive Carbonic anhydrase is added to the vesicles, releases vesicles with chloroplast localisation signal, releases vesicles into the chloroplast, upon fusion CA is activated and changes HCO3− into carbon dioxide, which is then highly concentrated in a chloroplast and reduces rate of O2 binding to the Rubisco simply by increasing concentration of CO2. Chassis: E.coli or Sinorhizobium meliloti</P><P>Bacterial infection: Nod factors</P><P>Bacteria of Rhizobium spp. are capable of infecting a plant and forcing it to develop an extra organ - Nodule, where these bacteria then intracellularly (a bit like organelles) reside. They do this to develop a mutualistic relationship with plant. We could use this mechanism of infection and acceptance by using the entire &quot;Nod box&quot; a cluster of genes involved in signalling to the plant to allow entry through the specially deformed root (induced by the Nod factors) or by crack entry. Each of the two mechanisms involves plant release of the flavonoids in the first place to trigger the Nod factors in the first place.</P><P>problems: -A lot of plants do not have Nod factor receptors, as wild type Rhizobium infects only legumes, so we would have been restricted to legumes as well. Also there is specificity among different Nod factors and their receptors on the plants meaning that not every Nod box containing bacteria could infect every plant. -In theory inserting a whole &quot;Nod box&quot; of genes into E. coli should enable E.coli to function in relation to the plant much in the same way as Rhizobium does, however we can not be sure of that, even though there is evidence that some genes in Rhizobium (NodD) have orthologues in E. coli (glmS). -Plant accepts Rhizobium as a symbiont and expects to get something from it, therefore if we were to use Rhizobium as chassis we could leave the initial nitrogenase function intact, however there might be a problem using E. coli as it would not be capable of fixing nitrogen the plant might not accept its infection thread. -Rhizobium forces plant to form nodule on the root, however ideally we would want to set up infection into the leaves. Maybe possibility to send vesicles through the xylem to the leaves, however vesicles would face problem of crossing plant cell wall.</P><P>Accumulation of HCO3− and packaging into the vesicles: CaA and carboxysome A lot of cyanobacteria / algae, use specialised carboxysomes to accumulate HCO3− through a number of HCO3− transporters and carbon dioxide converting enzyme Carbonic anhydrase which performs interconversion of CO2 and HCO3−. Different genes in C. reinhardtii (cupA, cupB) act as transporters of CO2 and automatically convert it to HCO3−. There is a number of other transporters utilised by cyanobacteria, but these just transport HCO3− and do not convert it to CO2, and therefore are not useful to us. Then a number of genes involved in carboxysome production would have to be included in the chassis as well. Also normal carboxysome in a cyanobacterium contains a number of other protein products to convert CO2, however these are not necessary as carbon fixation would be performed by the plant itself. Finally a CaA - carbonic anhydrase converting HCO3− to CO2 would be included, also Cso3 a Carbonic anhydrase embedded in the carboxysome membrane would be present.</P><P>However it needs to be inactive within the carboxysome/vesicle and active only upon entry into chloroplast. Therefore possible fusion protein with 3 domains could be created containing CA on the inner end, then transmembrane subunit and a transit/fusion peptide targeting it to the chloroplast. Upon fusion into chloroplast the fusion protein would be cleaved and CA would become active.</P><P>problems: -Creation of carboxysome ( a whole &quot;organelle&quot;) within a chassis not previously having any. -Creating vesicles out of carboxysome, which would not release any of its content out into bacterial cytoplasm (whole compartmentalisation would not work) -This also raises a question of what concentration of HCO3− can be transported within one vesicle, if the concentration is too low it will not function.</P><P>Transport of vesicles from bacteroid into the chloroplast: OMV Could be largely based on the OMV-outer membrane transport, which has been worked out by igem team paris 2009. However a number of outer-transit/fusion peptides would have to be different to ensure targeting towards chloroplast and succesful fusion into the chloroplast. problems: -Usual transit peptide used for fusion protein targeting from cytoplasm into chloroplast (5kDa Rubisco subunit) might not work in targeting of the wholve vesicle into the chloroplast. -Previous igem team have developed OMV to transport proteins from cytoplasm to another bacteria. In this situation however we would use OMV to transport concentrated solution from carboxysome - &quot;organelle&quot;, therefore the OMV itself might not work on our setup. Ideal solution: Engineer carboxysome with Carbonic anhydrase within plants (possibly within chloroplast) and use it to generate high CO2 concentration. References: Moroney, J.V. &amp; Somanchi, A., (1999). How Do Algae Concentrate CO2 to Increase the Efficiency of Photosynthetic Carbon Fixation? Plant Physiology, 119 (1), 9 -16. Goodsell a S. Dutta, “Carbonic Anhydrase”, RCSB Protein Data Bank (january, 2004), <A href="http://www.pdb.org/pdb/101/motm.do?momID=49" rel="nofollow">http://www.pdb.org/pdb/101/motm.do?momID=49</A>. Nod factor interaction picture taken from: <A href="http://www.glycoforum.gr.jp/science/word/saccharide/SA-A02E.html" rel="nofollow">http://www.glycoforum.gr.jp/science/word/saccharide/SA-A02E.html</A> (Text by Nick)</P><P><A href="#Top">Back to top</A></P><H1><A name="Nematodes">Bacteria Targeting Parasitic Nematodes</A></H1><P>Plant parasitic nematodes Plant parasitic nematodes (PPNs) cause billions of dollars in crop damage annually and the only relatively effective solution is chemical which cause environmental damage.</P><P>Biological solutions: Nematode populations have been declined by biological microbes- mainly by two parasitic fungi, Nematophthora gynophila and Verticillium chlamydosporium, which attack the developing female on the root surface. 95 to 97 % of the females and eggs are destroyed (Kerry, Crump and Mullen, 1982). Thus the natural control of cereal-cyst nematode in a range of soils is predictable and effective, but slow acting. See:<A href="http://www.fao.org/docrep/V9978E/v9978e0b.htm" rel="nofollow">http://www.fao.org/docrep/V9978E/v9978e0b.htm</A> for a list of potential biological agents and their shortcomings</P><P>The two most problematic species, the root-knot and cyst nematodes, infect roots of plants and feed off of their tissue. Nematodes are known to find food by chemotaxis and one type of bacteria, Bacillus nematocida, attracts nematodes by secreting volatile organic compounds and is then ingested by the worms. When it adapts to the intestine, it secretes proteases to kill the nematode.</P><P>This principle could be applied to an engineered bacteria that secretes a compound to attract PPNs (it would have to attract only PPNs, because there are a lot of good nematodes in soil) and once ingested kill the nematodes by RNAi knockout of the parasitic genes or genes involved in reproduction. A problem with this is that PPNs eat plants, not bacteria.</P><P>Newly hatched larvae have to migrate through the soil efficiently to locate plants because they have limited nutrient resources. IAA (the main type of auxin) and other indole compound gradients have been shown to attract PPNs, allowing plant infection. As long as the RNAi target is specific to PPNs, attraction of the PPNs does not have to be as specific.</P><P>Successful RNAi PPN targets in GM crops are a gene encoding a splicing factor and a gene encoding an integrase to target the root knot nematode M incognita. Most of the genes expressed in the esophageal gland cells (generate molecules secreted into the plant through the stylet) of PPNs encode proteins secreted into the host root (include cell wall modifying enzymes, regulators of host cell cycle &amp; metabolism, suppressors of host defence, and mimics of plant molecules)</P><P>Problem: Can we get relatively specific chemo-attraction of PPNs? How do we introduce RNAi into the nematodes? How do we contain the GM bacteria in a field and will PPNs be able to migrate effectively to these contained areas?</P><P>Lymphatic filariasis</P><P>Second leading cause of longterm disability worldwide (as a result of lymphedema, elephantiasis, hydrocele and periodic fevers), is caused by mosquito-transmitted filarial worms, including Wuchereria bancrofti and Brugia malayi, which colonize the lymphatic system. Drug treatments are fairly effective at killing larval stages but don’t provide much benefit to infected hosts</P><P>Research is being done to determine important genes in the nematodes via RNAi.</P><P>How could we target this nematode to prevent mosquito infection and subsequent human infection?</P><P><A href="#Top">Back to top</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Imperial_College_London/Extras/Brainstorming/Energy_test">http://2011.igem.org/Team:Imperial_College_London/Extras/Brainstorming/Energy_test</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Extras/Brainstorming/Energy_test" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Extras/Brainstorming/Energy_test" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Extras/Brainstorming/Energy_test&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Extras/Brainstorming/Energy_test&amp;oldid=68759" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>