<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Project_Chemotaxis_Modelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Project_Chemotaxis_Modelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Project_Chemotaxis_Modelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Project_Chemotaxis_Modelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Project_Chemotaxis_Modelling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Project_Chemotaxis_Modelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Project Chemotaxis Modelling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">Project AuxIn<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London">Home</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Tour">At a Glance</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project/Background">The Problem</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Overview">M1: Phyto-Route</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Overview">M2: Auxin Xpress</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Overview">M3: Gene Guard</A></LI></UL><LI><A class="sf-with-ul" href="#">Achievements<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Achievements">Main Results</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Judging">Judging Criteria</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Europe_Jamboree">Europe Jamboree</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Overview">Informing Design</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human/Outreach">Outreach</A></LI><LI><A class="sf-with-ul" href="#">Extras<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Extras/Collaboration">Collaboration</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Protocols_General">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Software">Software</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Notebook/week1">Diary</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Brainstorming_Health">Brainstorming</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Team">Team</A></LI></DIV><H1>Module 1: Phyto-Route</H1><P><B>Chemotaxis is the movement of bacteria based on attraction or repulsion of chemicals. Roots secrete a variety of compounds that <I>E. coli</I> are not attracted to naturally. Accordingly, we engineered a chemoreceptor into our chassis that can sense malate, a common root exudate, so that it can swim towards the root. Additionally, <I>E. coli</I> are actively taken up by plant roots, which will allow targeted <A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Overview">IAA delivery</A> into roots by our system.</B></P><DIV class="underlinemenu"><UL><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Specifications">Specifications</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Design">Design</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Modelling">Modelling</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Assembly">Assembly</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Testing">Testing &amp; Results</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Future">Future Work</A></LI></UL></DIV><H1>Modelling</H1><P style="font-size:1.3em;"><A href="#" onclick="ddaccordion.collapseall('technology'); return false">Collapse all</A>  | <A href="#" onclick="ddaccordion.expandall('technology'); return false">Expand all</A></P><DIV class="technology">1. Introduction</DIV><DIV class="thelanguage"><P>Chemotaxis is the movement of bacteria up a concentration gradient of chemoattractants (<I>e.g.</I> malate in our project) and away from repellents (<I>e.g.</I> poisons). <I>E. coli</I> is too small to detect any concentration gradient between its two ends. These bacteria resample their surroundings every 3-4 seconds, causing the cells to either tumble or to run<SUP>[1]</SUP>. Chemoattractants transiently increase the probability of ‘tumbling’ (or bias). This is followed by a sensory adaptation process that returns the bias to baseline, enabling the cell to detect and respond to further concentration changes. A small step change in chemoattractant concentration in a spatially uniform environment increases the response time from 1 second to 2-4 seconds<SUP>[2]</SUP>. Changes to the saturating level of chemoattractant can increase the response time to several minutes. </P><P>Each chemoreceptor on the bacterium has a periplasmic binding domain and a cytoplasmic signalling domain that communicate with the flagellar motors via a phospho-relay sequence involving the CheA, CheY, and CheZ proteins. The results of modelling the chemotaxis pathway will determine the threshold chemoattractant concentration for bacterial detection and the level at which the bacterial chemoattractant detection becomes saturated, decreasing the efficiency of the bacterial response to the chemoattractant.</P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 0); return false">Collapse</A></P></DIV><DIV class="technology">2. Modelling of the chemotaxis pathway</DIV><DIV class="thelanguage"><H3>2.1 Objective</H3><P>     Use MATLAB to model the chemotaxis pathway and thereby determine the detection threshold and saturation level of chemoattractant for bacteria with 8 μM chemoreceptor. As it is believed that the auxin should be placed close to the seed (&lt; 0.25 cm<SUP>[4]</SUP>) for optimal growth. It is therefore crucial to determine whether our bacteria will be able to stay close to the seed.</P><H3>2.2 Description</H3><DIV class="imgbox" style="width:420px;float:right;"><P><I><B>Figure 1</B><SUP>[1]</SUP><B>:</B> Chemotaxis signaling components and pathways for </I>E. coli<I>.</I></P></DIV><P>The chemotaxis pathway in <I>E. coli</I> is demonstrated in <B>Fig.1</B>. Chemoreceptors form stable ternary complexes with the CheA and CheW proteins to generate signals that control the direction of rotation of the flagellar motors<SUP>[5]</SUP>. The signalling group's currency is in the form of phosphoryl (p), which is made available to the CheY and CheB effector proteins through autophosphorylation of CheA<SUP>[1]</SUP>. CheY<SUB>p</SUB> initiates flagellar responses by interacting with the motor to enhance the probability of ‘run’<SUP>[1]</SUP>. CheB<SUB>p</SUB> is part of a sensory adaptation circuit that terminates motor responses<SUP>[1]</SUP>. Therefore, studying the methylation and phosphorylation levels of CheB and CheY is important to understanding chemotaxis of a single cell. The model based on Spiro et al. (1997)<SUP>[1]</SUP> was used to study how the concentrations of proteins in the chemotaxis pathway change over time. 
</P><P>In addition, the quantity that links the CheY<SUB>p</SUB> concentration with the type of motion (run vs. tumble) is called bias. It is defined as the fraction of time spent on the run with respect to the total movement time. The relative concentration of CheY<SUB>p</SUB> is converted into motor bias using a Hill function (<B>Equation 1</B>). </P><H3>2.3 Results and discussion</H3><P>Based on the Spiro model<SUP>[1]</SUP>, the methylation levels of receptors and phosphorylation levels of CheY and CheB were studied <B>(Fig.2)</B>.</P><P>Also, we studied the dependence of the run/tumble decision making, which is based on the relative receptor occupancy. It means that the probability of any particular chemoreceptor being occupied depends only on the external concetration of the chemoattractant, but not the total number of the chemoreceptors. Even if the threshold and saturation level of chemoreception does not depend on the chemoattractant concentration <B>(Fig.3)</B>.  A different number of receptors would certainly allow for finer-grain adaptation, therefore the strongest promoter is chosen for the best sensitivity.</P><DIV class="imgbox" style="width:820px;margin:0 auto;"><P><I><B>Figure 2(a):</B> The phosphorylation level of CheY. <B>Figure 2(b):</B> Phosphorylation of CheB. <B>Figure 2(c):</B> Methylation level of Chemoreceptor. <B>Figure 2(d):</B> The probability of of bacteria in the running state at different levels of CheY<SUB>p</SUB> Figure 2 (a)(b)(c) show that the threshold detection concentration is 10<SUP>-8</SUP> M, and the saturation concentration is 10<SUP>-5</SUP> M. (modelling based on spiro et al <SUP>[1]</SUP>, codes were modified by Imperial College iGEM team 2011).</I></P></DIV><DIV class="imgbox" style="width:820px;margin:0 auto;"><P><I><B>Figure 3:</B> The threshold and saturation level of chemoreception detection vs. the chemoattractant concentration. This graph shows clearly that the threshold and saturation level of chemoreception does not depend on the chemoattractant concentration. </I></P></DIV><P>The results from our modelling of the chemotaxis pathway show that the threshold chemoattractant concentration for bacterial detection is 1x10<SUP>-8</SUP> M and the level where bacterial chemoattractant detection becomes saturated is 1x10<SUP>-5</SUP> M, causing the bacterial response to the chemoattractant to becomes less efficient. And the strongest promoter is chosen for the highest sensitivity. In addition, the modelling of bacterial population chemotaxis in the next segment will further inform us about the whether our bacteria will be able to move close to the root and at which actual distance the bacteria should be placed. </P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 1); return false">Collapse</A></P></DIV><DIV class="technology">3. Simulation of chemotaxis of a bacterial population</DIV><DIV class="thelanguage"><H3>3.1 Objective</H3><P>   Modelling the chemotaxis of bacterial population dynamics under experimental and natural conditions. The model of laboratory conditions will aid the wet lab team in designing their experiments. The model of real soil conditions will further inform our project about how and where we can place our bacteria. </P><P>    Under experimental conditions, the chemoattractant diffuses constantly from the source. However, in real soil, the root produces malate all the time. Therefore, we assume that the distribution of chemoattractant outside the seed is steady and time-independent. Hence, the modelling of bacterial population chemotaxis will be built with different patterns of chemoattractant distribution.</P><H3>3.2 Description</H3><P>This part of the modelling focused on creating a movement model of a bacterial population for chemotaxis. In chemotaxis, chemoreceptors sense an increase in the concentration of a chemoattractant and then send a signal that suppresses tumbling. Simultaneously, the receptor becomes increasingly methylated. Conversely, a decrease in the chemoattractant concentration increases the tumbling frequency and causes receptor demethylation. The tumbling frequency is approximately 1 Hz when flagellar movement is unbiased, and decreases to almost zero as the bacteria move up a chemotatic gradient<SUP>[5]</SUP>. In order to accurately built this model, the following assumptions were made based on literature: </P><P>1)	 During the directed movement phase, the mean velocity at which <I>E. coli</I> moves is 24.1 μm/s<SUP>[7]</SUP>. Whereas during the tumbling phase, the speed is insignificantly slower and can be neglected. </P><P>2)	<I>E. coli</I> usually takes the previous second as their basis on deciding whether the concentration has increased or not. Therefore, in our model the bacteria will be able to compare the concentration of chemoattractant at t seconds to that at t-1 seconds. </P><P>3)	In our model, we ignored that <I>E. coli</I> do not travel in a straight line during a run, but take curved paths due to unequal firing of flagella. </P><P>4)	Our model did not consider the growth of bacteria or the tendency of bacteria to congregate into a small area due to quorum sensing. </P><P>    In the model, the bacteria should be able to compare the chemoattractant concentration at the current time point to the concentration at the previous second. If the concentration decreases (<I>i.e.</I> C<SUB>t1</SUB>-C<SUB>t2</SUB>  ≤0), the bacteria will tumble with a frequency of 1 Hz. If the concentration increases (C<SUB>t1</SUB>-C<SUB>t2</SUB>&gt;0), the tumbling frequency decreases, hence the probability of tumbling decreases. From equation 10 in reference [6], we know that when C<SUB>t1</SUB>-C<SUB>t2</SUB>  &gt;0, the probability of tumbling decreases as an exponential function of chemostatic constant, bacterial velocity , concentration difference between adjacent time points and angle between the two time points. Therefore, we can condense the above description into the following statement<SUP>[8]</SUP>: </P><H3>3.3 Results and discussion</H3><H4>3.3.1 Baterial chemotaxis in lab</H4><P>Under laboratory conditions, the chemoattractant continuously diffuses from the source, hence the distribution pattern of chemoattractant changes with time. In this case, the error function (<B>Equation 2</B>) was used to describe the non-steady state chemoattractant distribution.</P><P>For chemotaxis wet lab experiments, a user interface was designed to test the feasibility of experimental protocols. Initially, an experiment was designed as placing certain number of bacteria 6 cm away from a chemoattractant source with different concentrations. A simulation of chemotaxis of 100 bacteria placed 6 cm away from a 5 mM malate source is shown in the movie below. And this MATLAB program has a graphic user interface (GUI)(<B>Fig.4</B>), which can be used by the wet lab students to simply type in parameters and generate static graphs or animations. </P><DIV class="imgbox" style="width:440px;float:left;"><P><I><B>Figure 4:</B> Matlab graphic user interface for our chemotaxis model. (Made by Imperial College iGEM team 2011).</I></P></DIV><P>The video above shows that the chemoattractant takes 14000 seconds to diffuse to the bacteria colony. During this time bacteria will replicate everywhere on the plate. Thus from modelling, we can deduce that these experimental results may not be able to clearly demonstrate chemotaxis (a result that was seen in our wet lab). Therefore, a new experimental set-up was designed as the chemoattractant is kept in the capillary tube and placed into well with bacterial suspension; attractant diffuses into the well setting up the concentration gradient. Bacteria swim up the concentration gradient into the capillary, after given time, capillary is removed and number of bacteria in the capillary is counted. The whole experimental design is shown demonstrated in <B>Fig.5</B>. This new model integrates our experimental results, which  has refined it. This refinement in turn is further informing the design of our assay which will result in more accurate data, thereby creating a refinement loop between modelling and wet lab of the Phyto-Route module. </P><DIV class="imgbox" style="width:520px;margin:0 auto;"><P><I><B>Figure 5:</B> Demonstration of experimental setup of the capillary assay. (Diagram by Imperial College London iGEM team 2011.) </I></P></DIV><P>The chemoattractant distribution profiles are different in capillary and well. Therefore two chemoattractant distribution profiles were derived, for details of derivation please download <A href="https://static.igem.org/mediawiki/2011/5/53/Capillary-Problem.pdf" target="_blank">here</A>. The distribution equation in capillary is shown in <B>Equation 3</B> and <B>Equation 4</B> shows the chemoattractant distribution in the well. </P><P>Again, a user interface was designed that enables wet lab students to vary the input parameters (<B>Fig. 6</B>).  The video below shows a simulation with 1000 bacteria chemotaxing towards a capillary with a diameter of 1mm filled with 1mM of chemoattractant. And <B>Fig. 7</B> shows the number of bacteria in the capillary at each time point. This model demonstrated that the maximum number of bacteria accumulated in the capillary is at 3600 seconds. Therefore our modelling suggests that the wet lab student should perform this experiment for 60mins and then collect the data. </P><TABLE><TBODY><TR><TD><DIV class="imgbox" style="width:420px;float:none;"><P><I><B>Figure 6:</B> Matlab graphic user interface for our chemotaxis capillary assay. (Made by Imperial College iGEM team 2011).</I></P></DIV></TD><TD><DIV class="imgbox" style="width:420px;float:none;"><P><I><B>Figure 7:</B> The number of bacteria in capillary vs. time (Made by Imperial College iGEM team 2011).</I></P></DIV></TD></TR></TBODY></TABLE><DIV class="newouterbox"><H4 class="newtext">NEW SINCE EUROPE JAMBOREE</H4><DIV class="newinnerbox"><DIV class="vidbox" width="800px" style="margin:0 auto;"><P><I>Video 2. Simulation of capillary assay. This model predicts the population dynamics of the bacteria within the sample. The user interface allows the wetlab user to run the simulations with customized parameter.</I></P></DIV></DIV></DIV><H4>3.3.2 Bacterial chemotaxis in soil</H4><P>In real soil, malate is used as the chemoattractant. Malate is constantly secreted from the root tip at a concentration of 3 mM<SUP>[9]</SUP>. In this case, the malate source is always replenished due to continuous secretion from the root and the distribution pattern can be considered as steady (i.e. independent of time). The steady-state Keler-Segel model was used to demonstrate this distribution (<B>Equation 5</B>).</P><P> The distribution of malate in real soil is displayed in <B>Fig.5</B>. The paths of bacterial chemotaxis when placing bacteria at different positions in steady malate distribution is demonstrated in <B>Fig.5</B>.  </P><DIV class="imgbox" style="width:820px;margin:0 auto;"><P><I><B>Figure 8(a):</B> Distribution of malate with distance (1D).Figure 8(b): Distribution of malate with radius (2D).<B>Figure 8(b)</B>shows the position of the threshold detection concentration (1e-8 M at radius = 0.028 m) and the saturation concentration (1e-5 M at radius = 0.012).</I></P></DIV><DIV class="imgbox" style="width:820px;margin:0 auto;"><P><I><B>Figure 9: Chemotaxis with bacteria placed at different starting positions.</B><B>Blue</B>: 2×10<SUP>5</SUP> seconds chemotaxis starts at radius = 0.015 m (in between 0.012 m and 0.028 m), <B>green</B>: 2×10<SUP>5</SUP> seconds  chemotaxis starts at radius = 0.008 m, which is less than 0.012. This shows that the chemotactic response of the bacteria is inefficient when they are placed between 0.0028 m and 0.0012 m from the root tip. The green line shows that the bacteria can be maintained close to the seed when it is placed at a distance of less than 0.012 m away from the root tip. Therefore, the model suggests that the bacteria should be placed at a distance of less than 0.012 m in our project implementation.</I></P></DIV><P> In conclusion, the threshold chemoattractant concentration for bacterial detection is 1x10<SUP>-8</SUP> M and the level where the bacterial chemoattractant detection becomes saturated is 1x10<SUP>-5</SUP> M. And we also demonstrated that the detection threshold and saturation level are independent of number of chemoattractant receptors, however, in order to enable maximum sensitivity, the strongest promoter should chosen for Phyto-route module . Furthermore, with malate concentration equal to 3 mM, the threshold detection concentration is at a distance of 0.028 m from the root, and the saturation concentration is reached at 0.012 m (1.2 cm) away from the root. From modelling the chemotaxis of the bacterial population, we observed that the chemotaxis to the root is slow if the bacteria are initially placed in between 0.012 m and 0.028 m, while, when placing the bacteria within 0.012 m from the root, the bacteria are likely to stabilise and remain close to the root. The distance for stable bacterial localisation (1.2 cm) is greater than 0.25 cm (the distance where nutrients can be effectively taken up by the root), therefore it is suggested that for our <A href="https://2011.igem.org/Team:Imperial_College_London/Human_Implementation"><B>seed coat</B></A> implementation, the bacteria should be placed at or closer than 0.25 cm away from the root. </P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 2); return false">Collapse</A></P></DIV><DIV class="technology">4. Parameters</DIV><DIV class="thelanguage"><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 3); return false">Collapse</A></P></DIV><DIV class="technology">5. Matlab code</DIV><DIV class="thelanguage"><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 4); return false">Collapse</A></P></DIV><DIV class="technology">6. References</DIV><DIV class="thelanguage"><P>[1] Spiro PA, Parkinson JS and Othmer HG (1997) ‘A model of exciatation and adaptation in bacterial chemotaxis’. <I>Proc Nat. Acd Sci USA</I><B>94:</B> 7263-7268.</P><P>[2] Blocks SM, Segall JE and Berg HC (1982) <I>Cell</I><B>31:</B> 215-226.</P><P>[3] Stock JB and Surette MG (1996) ‘Escherichia coli and salmonella: cellular and molecular biology’. <I>Am Soc Microbiol</I>, Washington, DC. </P><P>[4] Schnepf A ‘3D simulation of nutrient uptake’ </P><P>[5] Levin MD, Morton-Firth CJ, Abouhamad WN, Bourret RB, and Bray D, ‘Origins of individual swimming behavior in bacteria.’</P><P>[6] Vladimirov N, Lovdok L, Lebiedz D, Sourjik V (2008) ‘Dependence of bacterial chemotaxis on gradient shape and adaptation rate’ <I>PloS Comput Biol</I><B>4(12):</B> e1000242. Doi:10.1371/journal.pcb1.1000242. </P><P>[7] Liu Z and Papadopoulos K (1995) ‘Unidirectional motility of <I>Escherichia coli</I>’. <I>App Env Microbiol</I><B>61:</B> 3567–3572, 100099-2240/95/$04.0010</P><P>[8] https://2009.igem.org/Team:Aberdeen_Scotland/chemotaxis</P><P>[9] Martinoia E and Rentsch D (1994) ‘Malate compartmentation-responses to a complex metabolism’ <I>Annual Review of Plant Physiology and Plant Molecular Biology</I><B>45:</B> 447-467, DOI: 10.1146/annurev.pp.45.060194.002311</P><P>[10] Brokaw CJ ‘Chemotaxis of bracken spermatozoids: Implications of electrochemical orientation’. </P><P>[11] Jones DL, Prabowo AM, Kochian LV (1996) ‘Kinetics of malate transport and decomposition in acid soils and isolated bacterial populations the effect of microorganisms on root exudation of malate under Al stress.’ <I>Plant and Soil</I><B>182:</B> 239-247
</P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 5); return false">Collapse</A></P></DIV><H2><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Design" style="text-decoration:none;color:#728F1D;float:left;">
M1: Design
</A><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Assembly" style="text-decoration:none;color:#728F1D;float:right;">
M1: Assembly
</A></H2><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Modelling">http://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Modelling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Project_Chemotaxis_Modelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Project_Chemotaxis_Modelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Project_Chemotaxis_Modelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Project_Chemotaxis_Modelling&amp;oldid=262469" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>