"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College London/Project/Chemotaxis/Assembly
From 2011.igem.org
Project AuxIn »
Home
At a Glance
The Problem
M1: Phyto-Route
M2: Auxin Xpress
M3: Gene Guard
Achievements »
Main Results
Data
Judging Criteria
Europe Jamboree
Human Practice »
Informing Design
Safety
Outreach
Extras »
Collaboration
Protocols
Software
Diary
Brainstorming
Team
Module 1: Phyto-Route
Chemotaxis is the movement of bacteria based on attraction or repulsion of chemicals. Roots secrete a variety of compounds that E. coli are not attracted to naturally. Accordingly, we engineered a chemoreceptor into our chassis that can sense malate, a common root exudate, so that it can swim towards the root. Additionally, E. coli are actively taken up by plant roots, which will allow targeted IAA delivery into roots by our system.
Overview
Specifications
Design
Modelling
Assembly
Testing & Results
Future Work
Assembly
We have chosen to test two chemoreceptors. McpS from P. putida and PA2652 from P. aeruginosa. In order to test the receptor´s ability to rewire chemotaxis towards malate, we had to assemble the construct first. We have codon optimised the sequences for E. coli and B. subtilis. Then we have ordered the sequences from Eurofins. Both mcpS and PA2652 are long constructs (1923bp & 1689bp respectively, just for the coding sequence), therefore we have decided to get it synthesised in two fragments. The assembly into the vector pSB1C3 would follow one of the multifragment assembly strategies. One modification was made to the existing assembly strategies such as CPEC or Gibson, which have removed the first PCR step from the set of assembly steps that are required to follow. This has decreased the probability of the synthesised fragments mutating due to extensive use of PCR during assembly. The modification itself is based on the addition of MlyI restriction sites at the 5´ & 3´ end of the ordered fragments. Upon digestion with MlyI, which acts as a blunt end endonuclease that cuts five bp away from its recognition site, the synthesised fragment of interest can be gel extracted without having to utilise PCR for fragment isolation from the vector it has been delivered in.
In parallel to having codon optimised genes synthesised we have asked Dr Juan Luis Ramos, who has been working with mcpS gene and was at the discovery of its properties as chemoreceptor towards malate, to provide us with this gene. He has kindly agreed to help us and we have obtained from him a non codon optimised mcpS gene with unknown promoter and RBS in a vector pRK415 with tetracycline resistance.
Due to short timeframe we were operating on, Eurofins did not manage to synthesise mcpS fragments in time, and therefore we were not able to assemble these. However PA2652 gene fragments have been delivered on time and they have been assembled into pSB1C3 vector using CPEC assembly.
Assembly of PA2652 using CPEC
Colony PCR results of CPEC assembled PA2652 construct look promising! Will know for sure when sequencing results arrive next week
Gel 12: Colony PCR of 19 colonies picked from cells transformed with CPEC assembled PA2652 construct, about half have the correct size insert, these will be inoculated and miniprepped. Gel 13: Two more colony PCRs which were unsuccesfull, followed by five colony PCRs from negative control colonies (assembly of backbone vector without insert)showing backbone vector (6a) in one. The next well is a positive control colony PCR with plasmid 6a. The final two wells are analytical PCRs of the CPEC assembly and negative control with sequencing primers showing the correct size band for the assembled insert.
Retrieved from " http://2011.igem.org/Team:Imperial_College_London/Project/Chemotaxis/Assembly "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
