"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College London/Project Auxin Assembly
From 2011.igem.org
Project AuxIn »
Home
At a Glance
The Problem
M1: Phyto-Route
M2: Auxin Xpress
M3: Gene Guard
Achievements »
Main Results
Data
Judging Criteria
Europe Jamboree
Human Practice »
Informing Design
Safety
Outreach
Extras »
Collaboration
Protocols
Software
Diary
Brainstorming
Team
Module 2: Auxin Xpress
Auxin, or Indole 3-acetic acid (IAA), is a plant growth hormone which is produced by several soil bacteria. We have taken the genes encoding the IAA-producing pathway from Pseudomonas savastanoi and expressed them in Escherichia coli. Following chemotaxis towards the roots and uptake by the Phyto Route module , IAA expression will promote root growth with the aim of improving soil stability .
Overview
Specifications
Design
Modelling
Assembly
Testing & Results
Future Work
Assembly
Figure. 1: Assembly strategy for our Auxin Xpress construct. (Figure by Imperial College London iGEM team 2011).
We ordered both the IaaM and IaaH coding sequences as two fragments to minimise cost and time (fragments 1, 2, 3, and 4). We did not want to use PCR to amplify the fragments in order to avoid introducing mutations into our final construct, and so we engineered blunt end cut sites on either side of our synthesized sequences with the MlyI restriction enzyme. Mly1 (type II restriction enzyme) cuts bluntly, 5 bp away from the recognition site. This property allowed us to cut out only the coding sequence of each fragment. Each digested fragment was then gel extracted to prepare for assembly.
The pSB1C3 vector was simultaneously inverse PCR'd to amplify the backbone vector with the required overlaps.
We planned to combine the four fragments into the pSB1C3 vector by Gibson assembly, unfortunately our attempts failed, we postulate that this was due to homology on the backbone vector, causing it to re-anneal.
We reverted to CPEC to assemble the construct, a method that requires more extensive use of PCR than Gibson. This said, by introducing MlyI restriction sites, we were able to halve the number of PCR steps required, thereby reducing the potential mutation rate.
CPEC (Circular Polymerase Extension Cloning) is a primer-independent PCR assembly technique which relies on overlapping sequences between each part to be assembled. With a denaturing step, the double stranded DNA is melted, allowing compatible single stranded ends of each part to join. For this reason it is essential that the parts are designed with homologous ends (the fragments we used were designed with 50 bp overlaps). The annealed overlapping ends then serve as primers for polymerase extension to join the parts into a seamless construct.
CPEC was successful on the first try. DNA from E. coli DH5α colonies transformed with the assembled construct was miniprepped and sent to Eurofins for sequencing. The sequences were verified, so we proceeded to characterising the IAA producing construct.
Figure. 2: The assembled construct was restriction digested with EcoRI and PstI which cut at the backbone prefix and suffix respectively, allowing us to determine if the insert is the right size. Lane 1 contains a 1 kb+ ladder as reference. Lane 2 shows digestion with EcoRI, lane 3 with PstI and lane 4 with both. The resulting band sizes are approximately 4 kb which correlates with the size of the four assembled fragments. (Data by Imperial College London iGEM team 2011).
Figure. 3: A schematic diagram of the final Auxin Xpress construct. Click on each part to be directed to the relevant page of the Parts Registry. (Diagram by Imperial College London iGEM team 2011).
M2: Modelling M2: Testing & Results
Retrieved from " http://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Assembly "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
