"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College London/Project/Switch/Results
From 2011.igem.org
Project AuxIn »
Home
At a Glance
The Problem
M1: Phyto-Route
M2: Auxin Xpress
M3: Gene Guard
Achievements »
Main Results
Data
Judging Criteria
Europe Jamboree
Human Practice »
Informing Design
Safety
Outreach
Extras »
Collaboration
Protocols
Software
Diary
Brainstorming
Team
Chapter 1: Assembly strategy
The assembly of this module shall be the most challenging out of all of them. Not only are we starting it the latest, but we will be using parts from the registry to assemble it. The first step of assembly will require us to place the anti-Holin from the BBa_K112808 biobrick under the J23100 promoter in BBa_K398500. In order to perform this step we will be using a PCR which will contain non-homologous regions. These non-homologous sequences will contain the insulator, RBS (ITR obtained from modelling) and 15bp overhangs that will allow us to assemble the PCR products of both the biobricks through the use of In-Fusion. The PCR step will be incredibly challenging. Once the parts are correctly inserted into the pSB1C3 vector we will be able to extract it and use biobrick assembly to insert it into the Crim plasmid. Once in the Crim plasmid, the gene must be integrated into the genome. Once this step is completed we can proceed to the transformation of these cells (any attempts at transformation before we have these cells will just result in cell lysis).
We will also require the use of the J23103 promoter (the RPU which we have obtained from modelling)which can be found in a BBa_J61002 vector. We also have ordered an oligo of the promoter to run in parallel. Once this has been inserted into a pSB1C3 plasmids, we can extract the Holin and Endolysin genes from the BBa_K112808 biobrick using primers that will contain a SpeI or a PstI site for biobrick assembly. Once the J23103 is assembled with the Endolysin and Holin we can transform the E. coli that contain the anti-Holin gene in the genome.
Time is running out. Will this module be completed? We bloody hope it will.
3rd of August
Today we attempted to transform 5α cells with the BBa_K112808 kill switch cassette. These cells will be incredibly important for later steps in the assembly process.
4th of August
Today we performed a successful mini-prep on the previously transformed cells. This DNA is now ready for subsequent assembly.
8th of August
Today we attempted a transformation of cells with the BBa_K093005 biobrick. We will be using the RFP in this plasmid in order to make sure that the final constructs contain both the integrated Crim plasmid (contains GFP) and the pSB1C3 with the Endolysin (will contain the RFP).
9th of August
Today we performed a successful mini-prep on the previously transformed cells. This DNA is now ready for subsequent assembly. However, in order to proceed we will need to know the expression ratio between the genome promoter and the plasmid promoter. We have to make sure that the amount of anti-Holin is only slightly higher than the level of Holin as to not exhaust the cells too much. This is pretty difficult considering that one of the genes will be in a high copy plasmid whereas the other will be in the genome.
16th of August
Today we transformed the cells with the BBa_J61002 vector containg the J23103 promoter that will be needed for the plasmid.
18th of August
Today we attempted to run the digested gels. Unfortunately, the gel that was used had lost its Sybr safe somehow overnight meaning that no bands could be extracted. However, we were told that we have access to an oligo of the J23113 promoter which has a similar strength to the J23103 promoter. We shall attempt to use this promoter as well before the ordered oligo of J23103 arrives.
19th of August
Today we prepared the vector for the promoter insertion by digesting it with EcoRI and XbaI and then gel purifying the sample. We should be able to ligate the J23113 promoter into BBa_K093005 on Monday (provided that the gel extraction worked...)
22nd of August
The primers that we need for the assembly has arrived today. We prepared a PCR using the "short" primers. These allowed us to make the templates that we need for the "long" primers (contain large non-homologous regions that will allow us to use In-Fusion later on). We then PCR'd the Anti-holin with the template. Hopefully the PCR worked and we will be able to do a cheeky MlyI digest tomorrow.
23rd of August
The gel we ran today on the Anti-holin PCR product has given us an excellent yield of DNA which we will be able to use for the In-fusion attempt.
From left to right: Ladder from Baldwin, Anti-holin PCR, Anti-holin PCR, Ladder from Invitrogen.
We also performed the short and then the long PCR on the pSB1C# vector containing the J23100 promoter.
25th of August
We have been having problems obtaining a decent yield for the long J23100 PCR. We attempted to run another PCR again overnight in order to recitfy this issue. Today we also determined that the CRIM plasmid is missing either an XbaI site or a SpeI site. Depending on which one it is missing we will have to integrate the Anti-holin upstream or downstream of the uGFP within the plasmid.
From left to right: EcoRI+PstI; XbaI SpeI.
26th of August
We finally managed to obtain a yield of DNA larger than 10ng/μL with which we can attempt the In-Fusion reaction as well as the CPEC reaction. However, the yield might still be too low meaning that we will have to repeat the PCR once again using a different number of cycles and temperatures. We also ran another digest to determine which recognition site is missing on the CRIM plasmid. From the gel we can assume that the missing site is SpeI.
From left to right: GFP digested with EcoRI+PstI; XbaI+SpeI; Ladder; CRIM digested with XbaI; SpeI; XbaI+SpeI
1st of September
After two unsuccessful In-Fusion attempts we decided to go back to take a look at our primers and found that there was an error in the vector primers. We decided to rectify this by performing a PCR with a short and long primer. This was successful and it gave us a good yield of DNA to work with for the In-Fusion reaction.
5th of September
Today we ran a gel of a CPEC and In-Fusion reaction in order to try to get the Anti-Holin into the pSB1C3 plasmid. The gel showed no results, we will have to continue trouble-shooting and repeating these reactions.
6th of September
Temperature gradient PCR of superfolded GFP to get back Spe site. Reactions 1 and 4 worked and will be Dpn1 digested and PCR purified and then the concentration will be measured.
7th of September
Today it seems like the In-Fusion finally worked. There were no colonies on the control plate and there were at least fifty colonies on the In-Fusion plate. In order to prepare for the Anti-Holin in the genome we have started the PCR on the endolysin and holin genes from the kill switch cassette.
From left to right: Ladder; CRIM digest with E+X; Holin+Endolysin PCR product.
8th of September
Today we performed the PCR on the template of the Holin and endolysin we made the other day. This allowed us to add and XbaI site as well as the suffix to the template without much issue. We also digested the CRIM plasmid with EcoRI and XbaI in preparation for the insertion of the Anti-Holin into the genome.
From left to right: Lanes 1-5 are Holin+Endolysin PCR products; Lane 6 is the CRIM plasmid digested with EcoRI and XbaI.
9th of September
Today we performed a colony PCR on the Anti-Holin In-Fusion plate. Most of the results gave us no interest. However, since there was no cells on the control plate we believe it is due to the cells not bursting properly in the PCR reaction. On the other hand, colony 9 gave us a nice band at around the right size.
Top gel: Ladder; 14, 15, 16, 17, 18, 19, 20, Control, Ladder
Bottom gel: Ladder, 1,2,3,4,5,6,7,8,9,10,11,12,13,Ladder
10th of September
We have performed the colony PCR once again. This time colony two gave us a nice clear band of the appropriate size. Colonies two and nine have been chosen for further testing.
Modelling of Gene Guard
Introduction
The gene guard device involves three proteins:
1) Holin is a protein forms pore complexes in the inner-membrane and this pores allow endolysin to move into periplasmic space.
2) Endolysin is an enzyme that could break down the cell wall and causes cell lysis.
3) Anti-holin binds to holin and prevent from forming pores.
Our system will have holin and endolysin genes together with axuin genes and chemoreceptor genes on the plasmids, and anti-holin genes on the genome under control of strong promoter. The production of anti-holin will prevent cell lysis of the genetic modified bacteria by deactivating holin. If the plasmid get horizontal transferred to a different cell without anti-holin on its genome(e.g. any wild type soil bacteria), the expression of holin and endolysin will induce cell lysis and hence prevent it from keeping the plasmid containing auxin gene.
The key to the success of this device is the right ratio of holin and anti-holin production such that the anti-holin can deactivate all the holin produced in our genetic modified bacteria, and the holin expression level in wild type bacteria is high enough to induce cell lysis. As the gene expression is controlled by both promoter strength and ribosome binding site (RBS) strength, therefore modeling of gene guard is important for helping us to choose appropriate promoters and ribosome binding site (RBS) to inhibit and optimize cell behavior in our genetic modified cell and soil bacteria respectively.
Modelling Design
The modeling of gene guard consists two parts, one is the holin production in wild type bacteria and the other part is the deactivation of holin by anti-holin in genetic modified bacteria. Since all the genes in our system are constitutively expressed, thereby we only consider steady state.
Holin production in wild type soil bacteria
This part of modeling focused on deriving the relationships between promoter strength and RBS strength of holin on single plasmid. In order to model this, the following assumptions were made:
1) After plasmid transfer and duplication, there will be 50 copies of plasmids in one bacterium.
2) The literature indicates that cell lysis happens at holin concentration equals to 1000/cell[1].
Based on above assumptions, the kinetics of cell lysis in wild type bacteria was modeled using ordinary differential equations (ODEs) for the holin mRNA and holin (Equation 1).
At steady state (i.e. d[ ] /dt = 0), rearrange Equation 1 with [Holin] = 1000, we obtained an expression for P_(holin) K_(holin) as shown in Equation 2:
Holin inhibition in Genetic Modified bacteria:
In genetic modified bacteria the promoter strength and RBS strength of anti-holin on the genome should be stronger than promoter strength and RBS strength of holin on single plasmid such that all the holin produced by 300 copies of plasmids should be inhibited, where 300 is the maximum number of plasmids can be in single bacterium. In this system, the anti-holin can bind to holin to form dimer, which is defined as the deactivated holin. The mechanism in the genetic modified bacteria can be described in following ODEs in Equation 3.
Again at steady state with d[]/dt = 0, we obtained an equation of holin concentration (Equation 4).  By varying the ratio m in equation 4, we obtained that the holin concentration in genetic modified bacteria decreased to zero at ratio m ≥ 400 (see Figure 1). Then, we tested m = 400 with the anti-holin promoter we have with strength 13.1pg RNA/min/μg substrate DNA using ODEs shown in Equation 1 and 3, the result (Figure 2) shown that all the holin in the cell are inhibited as the holing concentration remains at 0 all the time, while the holin concentration in wild type bacteria reaches 1000 per cell after 5000 seconds.
Figure 1: The holin concentration against the ratio m
Figure 2: Evolution of anti-holin and hoin in genetic modified and wild type bacteria agiainst time(s)
Reference
[1]  Christos G. Savva, Jill S. Dewey, John Deaton, Rebecca L. White, Douglas K. Struck, Andreas Holzenburg and Ry Young. The holin of bacteriophage lambda forms rings with large diameter. Molecular Microbiology 69(4), 784–793. 2008.
Retrieved from " http://2011.igem.org/Team:Imperial_College_London/Project/Switch/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
