<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Human_Containment"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Human_Containment">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Human_Containment&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Human_Containment&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Human_Containment&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Human_Containment" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Human Containment</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">Project AuxIn<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London">Home</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Tour">At a Glance</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project/Background">The Problem</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Overview">M1: Phyto-Route</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Overview">M2: Auxin Xpress</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Overview">M3: Gene Guard</A></LI></UL><LI><A class="sf-with-ul" href="#">Achievements<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Achievements">Main Results</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Judging">Judging Criteria</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Europe_Jamboree">Europe Jamboree</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Overview">Informing Design</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human/Outreach">Outreach</A></LI><LI><A class="sf-with-ul" href="#">Extras<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Extras/Collaboration">Collaboration</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Protocols_General">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Software">Software</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Notebook/week1">Diary</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Brainstorming_Health">Brainstorming</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Team">Team</A></LI></DIV><H1>Informing Design</H1><P><B>We consulted numerous experts in various fields to ensure that the design of the AuxIn system respects all relevant social, ethical and legal issues.  One module of our system, <A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Overview"><B>Gene Guard</B></A>, is a direct result of brainstorming around the issues involved in the release of genetically modified organisms (GMOs). Although we have only reached the proof of concept stage, we have put a lot of thought into how AuxIn may be implemented as a product and the legal issues that would be involved.</B></P><DIV class="underlinemenu"><UL><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Panels">Discussion Panels</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Containment">GM Release</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Legal">Legal Issues</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Ecology">Ecology</A></LI><LI><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Implementation">Implementation</A></LI></UL></DIV><H1>GM release</H1><P>There are many different biological aspects that need to be taken into account when deliberating release of genetically modified organisms into the environment. A big issue associated with this is containment of genetic information and preventing its spread to other organisms. To prevent spread of our auxin-producing plasmid in the environment, we have devised <A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Testing"><B>Gene Guard</B></A>, a containment device that will actively prevent horizontal gene transfer. In addition, we have chosen <I>Escherichia coli</I> as our chassis to prevent spread to adjacent ecosystems because <I>E. coli</I> (unlike <I>Bacillus subtilis</I>) cannot form spores.</P><H2>Gene Guard design</H2><H3>1. Goal</H3><P>Bacteria frequently pass on genetic information via horizontal gene transfer. This can happen in a variety of ways, including conjugation, transformation and transduction<SUP>[1]</SUP>. As a result, a significant portion of bacterial genomes (12.8% of the <I>Escherichia coli</I> genome) consists of foreign DNA<SUP>[2]</SUP>. If the DNA we engineered into our bacteria were to be transferred to plant pathogenic bacteria, it may give them a competitive advantage when combined with their own virulence factors. This could negatively affect plants and the ecosystem. Therefore, we want to prevent spreading of the genetic information we have engineered into our chassis. </P><H3>2. Action</H3><P>Early on in the project, we recognised that containment of genetic information is an issue that has to be taken very seriously. We discussed many different ways in which we could ensure that the DNA we engineer into our bacteria does not spread to other microorganisms or adjacent ecosystems. We consulted our advisors in the synthetic biology centre at Imperial who informed us about already existing &quot;kill switches&quot;. Most of these are based on killing bacteria in response to a specific stimulus. However, many of the compounds that cause activation of these switches could not be applied to soil without side effects. In addition, we could never ensure that all bacteria in the soil would be eradicated through application of these compounds. Switches that kill bacteria after a certain number of replication cycles are also not necessarily applicable as the lysed bacteria leave behind genetic information that can be taken up by other soil microorganisms. </P><H3>3. Result</H3><P> We took a novel approach and, instead of designing a kill switch, we designed a containment device that is able to contain genetic information in our bacteria at the same time as preventing horizontal gene transfer to other microorganisms. <A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Testing"><B>Gene Guard</B></A> is based on a toxin/anti-toxin system taken from the lysis cassette made by the Berkeley 2008 team containing, holin, endolysin and anti-holin.


</P><H2>Chassis choice</H2><DIV class="imgbox" style="width:520px;float:right"><P><I>Figure 1. Our reasons for choosing </I>E. coli<I> as our chassis (graphic by Imperial College London iGEM team 2011).</I></P></DIV><H3>1. Goal</H3><P>In chassis choice, we had to consider several aspects. We wanted to choose a chassis that we would be able to transport to arid areas, preferably already enveloped inside a solid seed coat. In addition, we want the bacteria to be able to persist in the soil long enough to carry out their function. On the other hand, we also wanted to prevent spread of the bacteria into far-away ecosystems where they are more likely to have a detrimental effect on the ecological balance. </P><H3>2. Action</H3><P>We consulted two ecologists who are experts in above/below ground interactions and soil microbial ecology. They both advised us that while it may be more obvious to use naturally occurring soil bacteria such as <I>Bacillus subtilis</I>, <I>Escherichia coli</I> is less likely to survive in soil and may ensure better containment. <B>Dr Alexandru Milcu</B> pointed out that this is especially important considering that very high auxin secretion may skew plant populations. While this is not an issue in areas where the ecosystem is already badly affected, spread to other ecosystems, especially via spores, is a big issue. <B>Dr Robert Griffiths</B> also advised us that while engineering naturally occurring soil bacteria might lead to better persistence and cause our project to be more efficient, containment would be more easily achieved by using bacteria that do not normally occur in soil such as <I>E. coli</I> as they are more likely to be outcompeted.</P><P>These arguments caused us to pin-point our chassis choice on <I>B. subtilis</I>, a natural spore-forming bacterium that naturally occurs in soil and <I>E. coli</I>. We initially codon-optimised our genes for both of these species. At the first human practices panel, we thoroughly discussed the advantages and disadvantages associated with both chassis choices <B>(Figure 1)</B>.</P><H3>3. Result</H3><P>Containment and possible contamination of other areas is a very big human practices issue. With <I>B. subtilis</I> as our chassis we would never be able to ensure complete containment. On the other hand, enveloping <I>E. coli</I> in a seed coat is mostly  a mechanical issue that we should be able to overcome. We therefore chose to use <I>E coli</I> as the chassis for AuxIn.</P><H2>Survivability of our chassis</H2><H3>1. Goal</H3><P>Although <I>E. coli</I> have been shown to be able to survive in soil for significant amounts of time (more than 130 days in the case of the pathogenic strain O175, as shown by Maule, 2000<SUP>[3]</SUP>), the ability to survive even in manure sludge varies greatly between different strains<SUP>[4]</SUP>. We wanted to ensure that our chassis is able to survive in the soil for long enough to complete its purpose. At the same time, we wanted to test how long the bacteria would be able to retain their plasmid in an environment where it would be unfavourable to the survival of the bacteria and if we could observe any horizontal gene transfer.
</P><H3>2. Action</H3><P>To investigate the survival of lab strains of <I>E. coli</I> in soil, we set up a separate experiment. This experiment was started by our work experience student Kiran and carried on by us after he left. </P><DIV class="imgbox" style="width:340px;float:right"><P><I>Figure 2. Our work experience student Kiran and James discussing the survivability experiment set up (picture by Imperial College London iGEM team 2011).</I></P></DIV><P>We inoculated small discs of filter paper with GFP-expressing <I>E. coli</I> and put these into sterile and non-sterile soil. The fluorescent protein is expressed on a plasmid that also gives the bacteria antibiotic resistance against ampicillin and kanamycin. In the absence of antibiotic and presence of competition (through other soil bacteria, fungi and protozoans) we would expect the bacteria to mutate the plasmid as it presents an added metabolic burden. The bacteria were incubated for over six weeks to check for plasmid retainment. Each wee, cultures were grown up in medium containing both ampicillin and kanamycin to check for bacteria resistant to these two antibiotics. The grown-up cultures were plated out and checked for fluorescence, which would only be present in bacteria containing the plasmid. To ensure that we were not growing up antibiotic-resistant soil bacteria, we grew up cultures from non-inoculated soil as negative controls.</P><H3>3. Result</H3><P> After seven weeks, the bacteria could be recovered from sterile and non-sterile media and were shown to express GFP. Plasmid DNA was extracted from cultures showing <I>E. coli</I>-like appearance by miniprep. This DNA was digested with EcoRI and PstI to check for presence of the sfGFP insert. Bacteria from sterile and non-sterile soil contained the unmutated GFP insert, as confirmed by sequencing. In addition, there were bacteria showing different colony morphologies present on the plate inoculated from non-sterile soil. However, these are likely to be present due to the depletion of antibiotics in the plate by the resistant bacteria. A miniprep confirmed that these bacteria do not contain a plasmid. 
(For more extensive results on this experiment, please see our <A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Testing"><B>Gene Guard</B></A> page.)</P><H2>References:</H2><P>[1] Ochman H et. al. (2000) Lateral gene transfer and the nature of bacterial innovation. <I>Nature</I><B>405:</B> 299-305.
[2] Thomas C and Nielsen K (2005) Mechanisms of, and barriers to, horizontal gene transfer between bacteria. <I>Nature Reviews Microbiology</I><B>3:</B> 711-721.
[3] Maule A (2000) Survival of verocytotoxigenic <I>E. coli</I> O175 in soil, water and on surfaces. <I>Symp Ser Soc Appl Microbiol </I><B>29:</B> 71-78.
[4] Topp E et al. (2006) Strain-dependent variability in growth and survival of <I>Escherichia coli</I> in agricultural soil. <I>FEMS Microbiology Ecology</I><B>44:</B> 303-308.</P><H2><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Panels" style="text-decoration:none;color:#728F1D;float:left;">
Discussion Panels
</A><A href="https://2011.igem.org/Team:Imperial_College_London/Human_Legal" style="text-decoration:none;color:#728F1D;float:right;">
Legal Issues
</A></H2><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Imperial_College_London/Human_Containment">http://2011.igem.org/Team:Imperial_College_London/Human_Containment</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Human_Containment" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Human_Containment" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Human_Containment&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Human_Containment&amp;oldid=259548" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>