<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Macquarie_Australia_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Macquarie_Australia/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Macquarie_Australia/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Macquarie_Australia/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Macquarie_Australia/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Macquarie_Australia/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Macquarie Australia/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">Project<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Project">Project</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Project/background">Background</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/parts">Parts Submitted</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/medals">Achievements</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Project#flow">Project Workflow</A></LI></UL><LI><A class="sf-with-ul" href="#">Meet MQ<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Team">The Team</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Team/advisors">Our Advisors</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Team/instructors">Our Instructors</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Team/contributors">Other Contributors</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Macquarie_Australia">Official Profile</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Sponsors">Sponsors</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/HumanPracticeOverview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Ethics">Ethics</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Education">Education</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Survey">Survey</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Openday">Open Day</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/collaborations">Our Collaboration Research</A></LI><LI><A class="sf-with-ul" href="#">Labwork<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Results">Results and Characterisation</A></LI><LI><A class="sf-with-ul" href="#">Links<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia">MQ Wiki Home</A></LI><LI><A href="https://2012.igem.org/Main_Page">iGEM Home</A></LI><LI><A href="http://www.mq.edu.au/">Our University</A></LI><LI><A href="http://www.cbms.mq.edu.au/">Our Department</A></LI></DIV><H1><CENTER>Results and Characterisation</CENTER></H1><P><CENTER>To quickly skip to the section that you wish to read, click on the links below.</CENTER></P><CENTER><TABLE><TBODY><TR><TH colspan="2"><CENTER><FONT color="#c85a17" size="5"><B>Results</B></FONT></CENTER></TH></TR><TR><TD><A href="#1"><H3>Heme Oxygenase</H3></A></TD><TD><A href="#3"><H3>Bacteriophytochromes</H3></A></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH colspan="2"><CENTER><FONT color="#c85a17" size="5">Sequencing Results</FONT></CENTER></TH></TR><TR><TD><A href="#5"><H3>Heme Oxygenase</H3></A></TD><TD><A href="#6"><H3>Bacteriophytochromes</H3></A></TD></TR></TBODY></TABLE><TABLE><CENTER><TBODY><TR><TH colspan="4"><FONT color="#c85a17" size="5">Characterisation</FONT></TH></TR><TR><TD><A href="#2"><H3>Heme Oxygenase</H3></A></TD><TD><A href="#4"><H3>Bacteriophytochromes</H3></A></TD><TD><A href="#7"><H3>Building The Switch</H3></A></TD><TD><A href="#switch"><H3>Reversing The Switch</H3></A></TD></TR></TBODY></CENTER></TABLE></CENTER><CENTER><A name="1"><H3>Heme Oxygenase Results</H3></A></CENTER><P>We produced a Heme Oxygenase BioBrick that was codon optimized for <I>E. coli</I>. The Gibson assembly of the T7 promoter containing Heme Oxygenase was successful. The transformation was successful with numerous colonies grown using Chloramphenicol as the selecting agent. Six colonies were selected and then they were sequenced before digestion with EcoR1 and Spe1. The sequencing showed that all of the colonies contained the plasmid with a Heme oxygenase identical to the original protein sequence. The gel containing the digested Heme Oxygenase bearing plasmid can be seen in Figure 1.</P><CENTER><TABLE><TBODY><TR><TD><CENTER>Figure 1: The restriction digest showing the linearised plasmid backbone (Black Box)
 and the heme oxygenese gene (Green Box). We used a 1kb ladder 
(in increasing Kbp: 0.25, 0.5, 0.75, 1, 1.5, 2, 2.5, 3, 4, 5, 6, 8, 10).</CENTER></TD></TR></TBODY></TABLE></CENTER><P>The lower band is the T7 Heme Oxygenase BioBrick (771 bp) which was expected. Similarly, the plasmid backbone was visible at the expected length (2070 bp). The similarity in intensity of the bands also demonstrates that the assembly and the digest were successful. The sequencing results for the six plasmids above can be seen below.</P><CENTER><A name="5"><H3>Heme Oxygenase Sequencing Results</H3></A></CENTER><P>The Gibson Assembly performed was transformed into 6 different cultures of competent cells. Three of theses were selected for sequencing using both forward and reverse primers. They were sequenced using the forward and reverse primers for the BioBricks. We performed Blastx and Blastn pipelines (available <A href="http://blast.ncbi.nlm.nih.gov/">here</A>) to determine if there was a significant change in the protein sequence and to determine the identity of the plasmid. The sequences below are named to show: that it is heme oxygenase (1C), which transformation (4,5,6), and whether it is the forward or reverse primer (F or R).</P><CENTER><TABLE border="3" cellpadding="4" cellspacing="0"><TBODY><TR><TD>Sample</TD><TD>Identity</TD><TD>E-value</TD><TD>MaxID</TD></TR><TR><TD>1C-6F</TD><TD>Heme Oxygenase (Synechocystic sp. PCC603)</TD><TD>7e-176</TD><TD>99%</TD></TR><TR><TD>1C-6R</TD><TD>Heme Oxygenase (Synechocystic sp. PCC603)</TD><TD>1e-171</TD><TD>99%</TD></TR><TR><TD>1C-4F</TD><TD>Heme Oxygenase (Synechocystic sp. PCC603)</TD><TD>4e-176</TD><TD>99%</TD></TR><TR><TD>1C-4R</TD><TD>Heme Oxygenase (Synechocystic sp. PCC603)</TD><TD>6e-172</TD><TD>99%</TD></TR><TR><TD>1C-5F</TD><TD>Heme Oxygenase (Synechocystic sp. PCC603)</TD><TD>6e-172</TD><TD>99%</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR><TD><UL><LI>We successfully assembled the codon optimised Heme Oxygenase.</LI><LI>The probability that it is not Heme Oxygenase is negligibly small.</LI><LI>The MaxID score is not 100% due to sequence misreads producing small gaps in the sequence.</LI><LI>The E-value signifies the possibility of matches based purely on chance. A score of 
0 identifies no background noise and is expected to be an error-free match.</LI></UL></TD></TR></TBODY></TABLE></CENTER><P>The source of our gene was identified in the sequencing result which showed that the sequencing was accurate. We then compared to the original gBlock sequence and determined that the sequencing was accurate and confirmed the identity of the plasmid. The Blastn pipeline indicated that there was no significant change from the theoretical sequence. With this data we would assume that the protein would be functional and performed assays to determine if this was the case.</P><CENTER><A name="2"><H3>Characterisation of Heme Oxygenase</H3></A></CENTER><P>The T7 regulated Heme Oxygenase produced was characterised to determine its functionality. BL21 <I>E. coli</I> was transformed with the plasmid, selected for using chloramphenicol, and a culture was inoculated. The culture was then induced with ALA (d-aminolevulinic acid) for the heme pathway and IPTG to promote protein production. They were incubated overnight and the cells were spun down. We observed a functional Heme Oxygenase with the cells appeared a vibrant green after induction by ALA and IPTG. We observed this as well in our assembled switch. The image below demonstrates the green pigment produced by the cells compared to uninduced Heme Oxygenase and the bacteriophytochrome.</P><P>The cells were lysed and the protein extract run on an SDS page gel. The gel showed the protein bands near the expected size (27 kDa). This, along with our pigmentation, showed that the BioBrick was producing the functional protein.
</P><CENTER><A name="3"><H3>Bacteriophytochromes Results</H3></A></CENTER><P>Like the Heme Oxygenase, the bacteriophytochromes from <I>Deinococcus radiodurans</I> and <I>Agrobacterium tumefaciens</I> were codon optimised for use in <I>E. Coli</I>. We did not use genomic DNA or previous Macquarie University team DNA at any stage, everything was performed with synthetic DNA we designed. The identity of the plasmid was determined by sequencing and by digestion. The bacteriophytochromes were characterised by demonstrating their ability to bind biliverdin and by their characteristic absorption profile.</P><CENTER><TABLE><TBODY><TR><TD>The restriction enzymes used in this digestion were,</TD></TR><TR><TD><UL><LI>X= Xba1      P= Pst</LI><LI>E= EcoR1     S=Spe1</LI></UL></TD></TR><TR><TD>We used a 1kb ladder (in increasing Kbp: 0.25, 0.5, 0.75, 1, 1.5, 2, 2.5, 3, 4, 5, 6, 8, 10).</TD></TR></TBODY></TABLE></CENTER><P>The digest above shows only shows one band for the different digests performed on the Agro T7 plasmid. The length of the vector and the bacteriophytochrome are similar and make it difficult to resolve the two bands. Only the digested plasmid can be seen which indicates that the bacteriophytochrome component was assembled correctly. Based on the length in base pairs of the fragments (2.25 kbp) we determined that assembly was successful. The first Agro with no T7 digest (asterixed band) shows incomplete digestion of a BioBrick plasmid. This enacted as a control and indicated the size of the plasmid.</P><CENTER><A name="6"><H3>Bacteriophytochrome Sequencing</H3></A></CENTER><P>Plasmids extracted from the different transformations were sequenced. </P><CENTER><TABLE border="3" cellpadding="4" cellspacing="0"><TBODY><TR><TD>Sequence</TD><TD>Expected Identity</TD><TD>Identity</TD><TD>E value</TD><TD>Max ID</TD></TR><TR><TD>F3CE1</TD><TD>Deinococcus radiodurans Bacteriophytochrome</TD><TD>Chain A, Crystal Structure Of A Monomeric Infrared Fluorescent Deinococcus Radiodurans Bacteriophytochrome Chromophore Binding Domain</TD><TD>3.00E-160</TD><TD>99%</TD></TR><TR><TD>F3CE2</TD><TD>Deinococcus radiodurans Bacteriophytochrome</TD><TD>Chain A, Crystal Structure Of A Monomeric Infrared Fluorescent Deinococcus Radiodurans Bacteriophytochrome Chromophore Binding Domain</TD><TD>2.00E-161</TD><TD>99%</TD></TR><TR><TD>F3CE3</TD><TD>Deinococcus radiodurans Bacteriophytochrome</TD><TD>Chain A, Crystal Structure Of A Monomeric Infrared Fluorescent Deinococcus Radiodurans Bacteriophytochrome Chromophore Binding Domain</TD><TD>8.00E-163</TD><TD>99%</TD></TR><TR><TD>R4KE2</TD><TD>Agrobacterium tumefaciens Bacteriophytochrome</TD><TD>bacteriophytochrome protein [Agrobacterium tumefaciens str. C58]</TD><TD>0</TD><TD>99%</TD></TR><TR><TD>R5CE1</TD><TD>Agrobacterium tumefaciens Bacteriophytochrome</TD><TD>bacteriophytochrome protein [Agrobacterium tumefaciens str. C58]</TD><TD>0</TD><TD>99%</TD></TR><TR><TD>R5CE3</TD><TD>Agrobacterium tumefaciens Bacteriophytochrome</TD><TD>bacteriophytochrome protein [Agrobacterium tumefaciens str. C58]</TD><TD>0</TD><TD>99%</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR><TD><UL><LI>We successfully produced the bacteriophytochromes with protein sequences matching the theoretical sequence.</LI><LI>The difference in identity is due to the codon optimisation performed. It was also affected by short sequence reads.</LI><LI>The 0 E value shows that we have produced the appropriate bacteriophytochrome.</LI></UL></TD></TR></TBODY></TABLE></CENTER><P>The The Blastx pipeline showed that there was an identical match to the orginal source. This shows that the Gibson Assembly reactions were successful and we expect the protein to be functional. Blastn were run to determine the deviance from the theoretical sequence. The Blastn searches produced indicated that the sequences were nearly identical. The changed bases were examined on the sequencing output and determined to be possible misreads.</P><CENTER><A name="4"><H3>Bacteriophytochrome Characterisation</H3></A></CENTER><P>We demonstrated that the bacteriophytochrome were functional by displaying their ability to bind biliverdin as well as by their presence in an SDS PAGE gel stained with Coomassie Blue. The gel for the bacteriophytochromes can be seen below,</P><P>The gel shows the expected bands for the bacteriophytochrome (Lanes 2, 5, 6, 7, 8) of approximately 80 kDA.We demonstrated that the bacteriophytochrome was functional and able to bind biliverdin in results below.</P><CENTER><A name="7"><H3>The Switch</H3></A></CENTER><P>Two of the BioBricks produced (Heme Oxygenase and the Deinococcus Bacteriophytochrome) were ligated together to produce the light switch. We demonstrated that the switch was produced by inspecting a gel following ligation and then digestion. The gel can be seen below.</P><P>Gel 1: We have run against a Heme Oxygenase standard (Lane 1, 770 bp). The gel contains digested fragments from our composite BioBrick (Heme Oxygenase and Deinococcus). The upper band (Black Box) is the Heme Oxygenase with the bacteriophytochrome and the bottom band (blue) is the plasmid backbone.</P><P>This provided the evidence that the product had been successfully ligated.</P><A name="gel"><H2>The Gels</H2></A><P>An SDS page gel was run of the ligation products to observe if the heme oxygenase was able to produce biliverdin and then to determine if it was binding with the bacteriophytochrome. The biliverdin binds to a specific site in the bacteriophytochrome. As biliverdin is fluorescent this coupling can be observed by irradiation with infrared (IR) light. </P>
The SDS PAGE gels for the Switch Constructs confirmed that the genes were compatible and that we successfully produced a T7 controlled operon-like system. The annotated gel can be seen below,<CENTER><P>The gel contains the following constructs:</P></CENTER><CENTER><TABLE><TBODY><TR><TD><UL><LI>The Protein Molecular Weight Marker</LI><LI>1C3C, the heme oxygenase and Deinococcus bacteriophytochrome ligation products (Lanes 2, 5, 6).</LI><LI>1C, the heme oxygenase (Lanes 3, 4)</LI><LI>4KE, Agro bacteriophytochrome (Lanes 7, 8).</LI></UL></TD></TR></TBODY></TABLE></CENTER><P>We were able to show that the bacteriophytochrome was produced in the construct. Bands were observed at the same molecular weight as our bacteriophytochrome controls (4KE1 and 4KE3). We were able to visualise useing Coomassie Blue. Heme oxygenase was also produced in a low yield. We then determined that successfully the switch with heme oxygenase providing biliverdin for the bacteriophytochrome. This was observed under IR light to take advantage of biliverdin's fluorescence.</P><P>The image contains four different SDS PAGE gels. The 2 on the left have been visualised using infrared light and the two on the right are the same gel stained with Coomassie Blue. This shows that many bands visible after Coomassie Blue staining are still not visible in the presence of IR light. For the band to be visualised a fluorophore is required. The top two gels act as a control as they contain no bacteriophytochrome, these are the T7 regulated Heme Oxygenase BioBricks we produced. This demonstrates that there are no native proteins, in a high concentration, that will cause false positives when the bacteriophytochrome is introduced.</P><P>The bottom two gels contain the Heme Oxygenase-Deinococcus bacteriophytochrome construct (1C3C), Heme Oxygenase (1C), and Agrobacterium bacteriophytochrome (4KE). The boxed area has been used to show that on select bands in the Coomassie stained gel are fluorescent in the infrared image. The blue circled area indicated that the bacteriophytochrome is not fluorescent in its native state. The bands that are active in the Infrared gel are the composite part we produced. It is our functional switch. The infrared image provides the information that:</P><BLOCKQUOTE><UL><LI>Heme Oxygenase produces the fluorescent molecule Biliverdin.</LI><LI>In our switch, Biliverdin binds specifically to the bacteriophytochrome in <I>E. coli</I>.</LI><LI>The bacteriophytochrome is the only molecule that can be visualised in the infrared spectrum. It requires biliverdin to be present in the cell</LI></UL></BLOCKQUOTE><P>The infrared gel shows that the switch is capable of self assembly without an external source of biliverdin. This is seen in the infrared gel as only the Switch's bacteriophytochromes can be visualised.</P><P>The gel contains:  4K1 (Agrobacterium T7) no biliverdin (BV), 4K1-25 uM BV, 4K1-50 uM BV, 4K3-no biliverdin, 4K3-25 uM BV, 4K3-50 uM BV, 1C3C-3 (Heme Oxygenase + Deino) no BV, 1C3C-3 25 uM BV, 1C3C-3 50 uM BV, 1C3C-6 no BV, 1C3C-7 no BV.</P><P>The gel shows that with an increasing gradient of biliverdin supplied to the cell there is no change in the intensity of the band in the infrared spectrum. Therefore we determined that our heme oxygenase part was able to supply the cell with enough biliverdin to saturate the bacteirophytochrome. </P><P>The spectrum of the cleared cell lysate of the Heme Oxygenase-Deinococcus Bacteriophytochrome expression shows that we have successfully assembled the light switch and in the Far-Red form. The peaks at 700 and 751 nm correlate with the literature values for the Far Red form Deinococcus bacteriophytochrome (Wagner et al. THE JOURNAL OF BIOLOGICAL CHEMISTRY VOL. 283, NO. 18, pp. 12212–12226, May 2, 2008). We have placed underneath the spectrum the shape of the different peaks. We had no access to a light source to switch it back to the red form (760 nm) and so could not convert it back to the Red form.</P><CENTER><B><A name="switch"><H1>Reversing the Switch</H1></A></B></CENTER><H3>Following the regional Jamboree we were able to show that the light switch was functional and was able to rapidly and reversible convert between the red and far red form.</H3><P>A dual beam UV-Vis spectrophotometer was used to provide a constant reference to an uninduced cell line's lysate, which experienced the same irradiation. The Agrobacterium bacteriophytochrome (4K) was used due to greater yield of protein. The induced cell lysate with no biliverdin. After this spectrum was obtained, the induced cell line with no ALA added was used as a reference. The spectrum for the bacteriophytochrome was collected in its native state (no treatment) and showed that it was assembled in the red form (yellow trace). Upon irradiation with 680 nm red light, it was able to be converted to the far red form (blue trace). This was seen in the spectrum with an increase in absorbance at 750 nm and decrease at 650 nm. We were able to get the switch to convert back from the far-red form to the red form by irradiation with 680 nm light (blue trace) for one minute. This data can be seen in the spectra below. The lysates were stored in the dark at -20°C allowing for the bacteriophytochrome to relax back to the red form. After 3 days the spectra were collected and shown to be reproducible.</P><CENTER>The spectra show that the switch could successfully convert between the two spectral forms</CENTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Macquarie_Australia/Results">http://2012.igem.org/Team:Macquarie_Australia/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Macquarie_Australia/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Macquarie_Australia/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Macquarie_Australia/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Macquarie_Australia/Results&amp;oldid=282957" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>