"
Page
Discussion
View source
History
teams
Log in
 
Team:Macquarie Australia/Protocols/Designing Gibson Assembly Fragments
From 2012.igem.org
Project »
Project
Background
Parts Submitted
Achievements
Project Workflow
Meet MQ »
The Team
Our Advisors
Our Instructors
Other Contributors
Official Profile
Sponsors
Human Practice »
Overview
Ethics
Education
Survey
Safety
Open Day
Our Collaboration Research
Labwork »
Notebook
Protocols
Results and Characterisation
Links »
MQ Wiki Home
iGEM Home
Our University
Our Department
Contents
1Haemoxygenase
2Deinococcus radiodurans Bacteriophytochrome
3Agrobacterium tumefaciens Bacteriophytochrome
4BioBrick assembly using Gibson method: Fragment design
Haemoxygenase
Original Gene Sequence: http://www.ncbi.nlm.nih.gov/nuccore/BA000022
Team mates performing codon optimisation
Deinococcus radiodurans Bacteriophytochrome
Original Gene Sequence: http://www.ncbi.nlm.nih.gov/nuccore/15807672/?from=53908&to=56175&strand=true&report=genbank
Agrobacterium tumefaciens Bacteriophytochrome
Original Gene Sequence: http://www.ncbi.nlm.nih.gov/nuccore/159184118/?from=1954292&to=1956502&strand=true&report=genbank
BioBrick assembly using Gibson method: Fragment design
Codon optimisation:
The Amino acid sequences for Haemoxygenase (synechocystis) and Bacteriophytochrome (Deinococcus radiodurans and Agrobacterium tumefaciens) were initially run through the Reverse Translate program ( http://www.bioinformatics.org/sms2/rev_trans.html ) for codon optimisation in E. coli BL21 .
Retrieved from " http://2012.igem.org/Team:Macquarie_Australia/Protocols/Designing_Gibson_Assembly_Fragments "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
