<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Macquarie_Australia_Notebook2"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Macquarie_Australia/Notebook2">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Macquarie_Australia/Notebook2&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Macquarie_Australia/Notebook2&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Macquarie_Australia/Notebook2&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Macquarie_Australia/Notebook2" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Macquarie Australia/Notebook2</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">Project<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Project">Project</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Project/background">Background</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/parts">Parts Submitted</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/medals">Achievements</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Project#flow">Project Workflow</A></LI></UL><LI><A class="sf-with-ul" href="#">Meet MQ<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Team">The Team</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Team/advisors">Our Advisors</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Team/instructors">Our Instructors</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Team/contributors">Other Contributors</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Macquarie_Australia">Official Profile</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Sponsors">Sponsors</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/HumanPracticeOverview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Ethics">Ethics</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Education">Education</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Survey">Survey</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Openday">Open Day</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/collaborations">Our Collaboration Research</A></LI><LI><A class="sf-with-ul" href="#">Labwork<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia/Results">Results and Characterisation</A></LI><LI><A class="sf-with-ul" href="#">Links<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Macquarie_Australia">MQ Wiki Home</A></LI><LI><A href="https://2012.igem.org/Main_Page">iGEM Home</A></LI><LI><A href="http://www.mq.edu.au/">Our University</A></LI><LI><A href="http://www.cbms.mq.edu.au/">Our Department</A></LI></DIV><H1>Notebook</H1><H2>Week 1- Tuesday July 31st</H2><P> With the break between semesters over, the Macquarie iGEM returned to classes. For us this was the first day of iGEM 2012. We had our first meeting and discussed our project. Dr. Louise Brown and Associate Professor Rob Willows were introduced the team and we began to determine who would take on certain roles within the team. </P><P> We eagerly began our project, deciding to use the novel approach of Gibson Assembly to produce our optimised genes. We had decided to develop a gene-switch controlled by light. To do this a couple of genes would be required: 
<OL><LI>A bacteriophytochrome</LI><LI>Heme oxygenase</LI></OL></P><H3><A href="https://2012.igem.org/Team:Macquarie/Protocols/Designing_Gibson_Assembly_Fragments">Gibson Assembly</A></H3><P>
The bacteriophytochromes from <I>Deinococcus radiodurans</I> and <I>Agrobacterium tumefaciens</I> were chosen. Over the next week Matt Stclair started to develop the Gibson Blocks (gBlocks) by:
<OL><LI>Acquiring the DNA sequence</LI><LI>Translating into the protein sequence </LI><LI>Using the DNA sequence, optimise codon usage for <I>E. coli</I></LI><LI>Translating into protein sequence and ensuring no changes in amino acids relative to the original sequence </LI></OL></P><H2>Week 2- Tuesday 7th August</H2><CENTER><H3><A href="https://2012.igem.org/Team:Macquarie/Protocols/Making_LB_agar_plates">Media prep</A></H3></CENTER><P>Our lab work began today with the preparation of liquid media, plates and buffers. The protocols we followed are located <A href="https://2012.igem.org/Team:Macquarie/Protocols/Making_LB_agar_plates">here</A>.</P><P>We prepared the following:
</P><BLOCKQUOTE><UL><LI>Liquid LB Media
</LI><LI>SOC Solution
</LI><LI>SOB Solution
</LI><LI>LB Agar Plates
</LI><LI>Ampicillin LB Agar Plates: 31 plates
</LI><LI>Chloramphenicol LB Agar Plates: 33 Plates
</LI><LI>Kanamyacin LB Agar Plates: 32 Plates
</LI><LI>TB buffer.
</LI><LI>TAE buffer.
</LI><LI>EDTA buffer.</LI></UL></BLOCKQUOTE><H3><CENTER>gBlock Development</CENTER></H3><P>While the lab group produced the necessary materials, another team began developing the gBlocks. The gBlock team consisted of Ellaina, Matt Smith, Matt Stclair, Harry, Gazelle, Silas, Kim, and Miguel. Matt Stclair spearheaded the effort and proofread all the sequences we had developed to ensure that we had not changed the protein sequence or introduced restriction sites.</P><P>To develop our gBlocks we needed to construct sections of DNA that could be used as overlaps. This ment that we the end of one gBlock was identical to the start of the next. This would allow for them to assemble in the correct order. This took some careful planning as we needed to ensure that they all had a similar <B>melting point greater than 48°C</B>.</P><H3><CENTER>Wiki editing and Human Practice</CENTER></H3><P>Ryan volunteered to be the wiki chief with Erin helping out throughout the project. Ellaina ensured the key safety questions were answered, and relayed these to the rest of the team. Rob and Sarah took control of the seeking finances to get the team to Hong Kong.</P><P>In a Macquarie iGEM first, the team took a big interest in the idea of human outreach and we had a brainstorming session. Elle proposed that we visit a high school while Ellaina suggested that the University's open day would be a great opportunity to communicate with the wider community. Ellaina took control of developing our high school visit and Elle took control of Open Day. </P><H2>Week 3- Tuesday 14th August</H2><CENTER><H3>gBlock Development Continues</H3></CENTER><P>Over the week we finalised our gBlocks and sent them to IDT DNA for synthesis. The bacteriophytochrome genes were split into 5 gBlocks of 500 bp each (the current maximum size). The Heme Oxygenase gene was split into 2 gBlocks due to it being significantly smaller. The Agro and Heme Oxygenase genes were able to accomodate the T7 promoter (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I719005">Registry Part BBa_I719005</A>) and so we produced 3 gBlocks for the heme oxygenase and a total of six for the bacteriophytochrome. The extra gBlock is the result of possessing or not possessing the T7 promoter. The Deinococcus bacteriophytochrome was too large and so the T7 promoter could not be used. Our GC content was high for some of the gBlocks in particular the Deinococcus. However- we could not find any information from IDT that this was a problem. <B><U>Update:</U></B> This was a problem and it appears that our gBlocks were a case study in lowering GC content to allow for synthesis.  In anticipation of our gBlocks arriving next week, competent cells were prepared. The protocol followed can be seen <A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/Making_competent_Cells">here</A>.</P><H3><CENTER><A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/Outreach_Planning">Human Practice Preparation</A></CENTER></H3><P>Human Practice planning went into overdrive this week. To engage the wider community, fluorescent and bioluminescent parts from the registry were selected and transformed in <I>E. coli</I>. Red, Green, Orange, and Cyan Florescent protein as well as Luciferase were all located and transformed in the competent cells available. More details can be seen <A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/Outreach_Planning">here</A>. </P><H2>Week 3- Wednesday 15th August</H2><H3><CENTER>Open Day Preparation</CENTER></H3><P>Our open day preparation did not go as planned. Most of our transformations were unsuccessful, we believe this was because SOC was added after the heat shock. As disappointing as this was, there was one little piece of good news.</P><P>A single colony of Top 10 <I>E. coli</I> containing GFP insert was identified on a plate of LB agar media.
LB agar and nutrient agar plates were struck using this colony. The colony was also used to inoculate 5mL of nutrient broth. Plates and stock were then incubated at 37°C overnight. Both of the streaked plates and the broth showed significant growth of <I>E. coli</I> colonies which could fluoresce when exposed to UV light.</P><H3><CENTER>Finances</CENTER></H3><P>We got our first news on the finances front. Unfortunately, Qantas declined to sponsor the effort of getting our team to Hong Kong. This was soon forgotten when the Macquarie University School of Medicine, Agilent and the Defence Science Institute (thanks Yagiz) all agreed to sponsor our team. Hong Kong here we come.</P><H3><CENTER>gBlock Development 3: Revenge of the GC rich regions</CENTER></H3><P>Design of the gBlocks continued. The fragments were designed with melting temperatures above 48°C. However, IDT had significant problems synthesising them due to GC rich regions. Unfortunately for the MQ iGEM team, IDT is based in the USA and our lab sessions do not coincide with business hours. The gBlock saga continued for a couple of weeks with new GC rich regions that affected synthesis identified with each modification made.</P><H2>Week 4- Tuesday 21st August</H2><H3><CENTER><A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/Outreach_Planning_Part_Two">Open Day planning</A></CENTER></H3><P>We continued open day planning by transforming parts from the registry (<A href="http://partsregistry.org/Part:BBa_I13521">Part BBa_I13521</A>, <A href="http://partsregistry.org/Part:BBa_I13522">Part BBa_I3522</A>, <A href="http://partsregistry.org/Part:BBa_I13600">Part BBa_I3600</A>, <A href="http://partsregistry.org/Part:BBa_I712052">Part BBa_I712052</A>, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K274210">Part BBa_K274210</A>). These were transformed in BL21 <I>E. coli</I>.</P><H3><CENTER>gBlock Development 4: A New Hope</CENTER></H3>.
<P>The team received the best news of the iGEM experience so far. Our gBlocks finally had no GC rich regions that inhibited synthesis. With the gBlocks arriving in the next week, we prepared for the assembly and took the opportunity to spend more time planning our two aspects of human practice.</P><H3><CENTER><A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/CompetentCellTesting">Checking Competency</A></CENTER></H3><P>The <A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/CompetentCellTesting">efficiency of the competent cells</A> produced was determined by transforming with the fluorescent constructs being used for the open day activities.
</P><H2>Week 5- Tuesday 28th August</H2><H3><CENTER><A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/OutreachandSchoolPlanning">Outreach Planning</A></CENTER></H3><P>The two outreach teams prepared the presentations for next week. The school outreach team would be travelling up to Green Point Christian College on Tuesday next week to talk to their Year 12 HSC Biology class about ethics in synthetic biology.</P><P>The open day team developed more powerpoint presentations and posters for the activities planned. As part of this planning we drew pictures of different Australian icons with Green Fluorescent and Red Fluorescent proteins. The protocol we followed can be seen <A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/Labwork">here</A>.</P><H2>Week 5- Friday 31st August</H2><CENTER><H3><A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/ArrivalofGBlocks">Arrival of gBlocks: Full Steam Ahead</A></H3></CENTER><P>Our gBlocks finally arrived, after a long period of time getting the GC content down. Unfortunately for us, they arrived late on Friday afternoon, so the Gibson Assembly would need to wait to till the following Tuesday. Problems encountered from August 11th - August 26th included high GC content in the sequences to be synthesised, in particular, the Deinococcus fragments. These sequences had to be codon optimized and accepted by IDT before synthesis began. </P><H2>Week 6- Tuesday 4th September</H2><CENTER><H3><A href="https://2012.igem.org/Team:Macquarie_Australia/Education">School Visit for Human Outreach</A></H3></CENTER><P>The human outreach school team consisting of Ellaina, Ryan, Daniel and Matt Smith all visited Green Point Christian College to talk about the ethics and concerns surrounding synthetic biology. We received wonderful feedback and had a great discussion of what we were doing in our project. The students also had a few ideas for synthetic constructs that could be used in future iGEM teams.</P><P>The students completed surveys so we could gauge the change in their perspective of synthetic biology before and after our seminar. The results of our surveys can be seen here <A href="https://2012.igem.org/Team:Macquarie_Australia/Survey">here</A>.</P><P>Our original intention was to take up a culture of GFP expressing <I>E. coli</I>. Upon further investigation, we determined that this would be a breach of our agreement with the Australian government for working with Transgenic organisms. This turned out to be an excellent avenue for us to discuss legal issues surrounding the field.</P><CENTER><H3><A href="https://2012.igem.org/Team:Macquarie_Australia/Education">Open Day Planning Team</A></H3></CENTER><P>The activities for open day were finalised and all of our prosters were prepared. We decided to try to communicate to a younger audience by using a Lego theme. All of our posters can be seen by clicking on the heading above</P><CENTER><H3><A href="https://2012.igem.org/Team:Macquarie_Australia/Education">Gibson Assembly Begins</A></H3></CENTER><P>In anticipation of the Gibson assembly we prepared more <A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/Making_competent_Cells">competent cells</A>. Harry, Matt St Clair, Silas and Kim performed our first Gibson Assembly reaction for each gene we prepared with different linearised vector (Parts PSB-1C3, PSB-1K3, PSB-1A3 from the registry). The synthesised genes were transformed in the competent cells prepared. For a more detailed analysis of the 
</P><H2>Week 6- Wednesday 5th September</H2><P> Matt looked at the transformed cells we cultured the previous day (04/09/12). Two small colonies of the transformed Deinococcus bacteriophytochome without the T7 promoter were grown on kanamycin. The positive control was successful, so we could infer that the reactions were proceeding as expected. The positive control was also cultured on XGal and incubated overnight.</P><P>The Deinococcus fragment was cultured in 2 tubes of LB broth (2 mL) containing kanamycin. The transformations performed on the previous day were done again with a slightly different method. We choose to redo the transformations as the efficiency was relatively low. The transformations were re-innoculated onto fresh plates and incubated over night.</P><H2>Week 6- Thursday 6th September</H2><P>We continued on from the Gibson assembly and transformations we performed. Daniel did a plasmid prep using a QIAprep Spin Miniprep Kit by Qiagen, for two colonies from the Deinococcus bacteriophytochrome. The plasmid preparation protocol can be seen <A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/PlasmidPrep">here</A>.</P><H2>Week 6- Friday 7th September</H2><H3><CENTER>Plasmid Preparation and Nanodrop Results</CENTER></H3><P>Plasmid Preparation was continued from Thursday, by Daniel and Harry, using the QIAprep Spin Miniprep Kit by Qiagen. A total of 7 colony cultures were used as the remaining did not grow. Successful cultures included 6 colonies from 1C and 1 colony from 3K. The DNA concentration was then determined for each sample by NanoDrop, with EB buffer from the Miniprep Kit used as a blank. The concentration of the plasmid extracted was determined using a NanoDrop and the results can be seen <A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/NanoDropResults">here</A>.</P><H2>Week 7- Tuesday 11th September</H2><H3><CENTER>Sequencing Results</CENTER></H3><P>The team received excellent news from the sequences performed on Friday. We had produced our first gene that is suitable for a biobrick! We had successfully produced the T7 promoter containing Heme Oxygenase. We then took this plasmid and inserted into  BL21  <I>E. coli</I> to overproduce the protein and allow us to characterise the gene.</P><H3><CENTER>Gibson Assembly Round 2</CENTER></H3><P>Unfortunately none of the bacteriophytochrome Gibson reactions were successful, so these were repeated by Ryan, Kim, Silas and Matt. Unlike the first assembly, for the second round of Gibson assembly we decided to assmble into 3 different vectors for each reaction. That is, each bacteriophytochrome genes were assembled into one of three different vectors, each containing antibiotic resistance for kanamycin, chloramphenicol and ampicillin. In total 9 Gibson reactions were performed which are summarised below and details of this procedure can be seen <A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/GibsonAssembly2">here</A></P><CENTER><TABLE border="3" cellpadding="4" cellspacing="0"><TBODY><TR><TD>Gene</TD><TD colspan="3"><CENTER>Antibiotic Resistance (Registry Part)</CENTER></TD></TR><TR><TD><I>Deinococcus</I> bacteriophytochrome</TD><TD>Ampicillin (PSB-1A3)</TD><TD>Kanamycin (PSB-1K3)</TD><TD>Chloramphenicol (PSB-1C3)</TD></TR><TR><TD>
T7+<I>Agrobacterium</I> bacteriophytochrome</TD><TD>Ampicillin (PSB-1A3)</TD><TD>Kanamycin (PSB-1K3)</TD><TD>Chloramphenicol (PSB-1C3)</TD></TR><TR><TD><I>Agrobacterium</I> bacteriophytochrome</TD><TD>Ampicillin (PSB-1A3)</TD><TD>Kanamycin (PSB-1K3)</TD><TD>Chloramphenicol (PSB-1C3)</TD></TR></TBODY></TABLE><H3><CENTER>Gibson Assembly Transformations</CENTER></H3></CENTER><P>The previous Gibson reaction used heat transformations, we wanted to determine whether electroporation yielded greater efficiency. Therefore we performed transformations using both of these techniques to determine which is the better method.</P><CENTER><H3>Media Preparation</H3></CENTER><P>Using the standard procedures, more kanamycin and chloramphenicol plates were prepared.</P><H2>Week 7- Wednesday 12th September</H2><P>The transformations from the previous day were examined and colonies were counted. Growth was slow so they were left till the afternoon to be counted. The counts can be seen <A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/Construct_olonies_1209">here</A>. It was determined that electroporation was a significantly more efficient transformation procedure than heat shock. This can be seen easily in the graph below,</P><CENTER><P>Blue bars signify electroporation counts and red signify heat shock counts.</P><H2>Week 7- Thursday 13th September</H2><CENTER><H3>Plasmid Prep and Sequencing</H3></CENTER><P>Ryan inoculated cultures of the transformations in the morning so Andrew and Daniel could complete the plasmid prep in the afternoon. The concentrations of the plasmids obtained can be seen here. All of the plasmids were sent for sequencing to determine if the assembly had proceeded as expected.</P><H2>Week 8- Monday 17th September</H2><CENTER><H3>Restriction Digests</H3></CENTER><P>With our midsemester break beginning and no other classes we could commit more time to the project. With the deadline fast approaching and the sequencing data returning tomorrow, we took the opportunity to digest all of our plasmids and prepare them for ligations. We were able to characterise the biobricks we had produced by inspecting the digestion pattern on the gel. The digest we performed are summarised below,</P><CENTER><TABLE border="3" cellpadding="4" cellspacing="0"><TBODY><TR><TH>Biobrick Part</TH><TH>Restriction Enzymes Used</TH></TR><TR><TD>T7 promoter + Heme Oxygenase</TD><TD>EcoR1 and Spe1</TD></TR><TR><TD>Heme Oxygenase (No T7)</TD><TD>Xba1 and Spe1</TD></TR><TR><TD>Deinococcus Bacteriophytochrome</TD><TD>Xba1 and Pst1</TD></TR><TR><TD>T7 promoter + Agrobacterium</TD><TD>EcoR1 and Spe1</TD></TR><TR><TD>Agrobacterium</TD><TD>Xba1 and Pst1</TD></TR></TBODY></TABLE></CENTER><CENTER><H3>Characterisation</H3></CENTER><P>The Biobricks we had produced were characterised by running on a gel following the restriction digest. 

For the above gel image: 
</P><BLOCKQUOTE><OL><LI>There are relatively equal intensities of the two bands present. At approximately 750 bp is the heme oxygenase insert and at 2000 bp is the vector backbone. </LI><LI>In these lanes, there was less material loaded and thus the two bands present are poorly resolved. The Deinococcus phytochrome insert and the backbone are approximately the same size (2000 bp) as expected. The digestion may have been incomplete and thus there is a presence of circular and supercoiled plasmid, phytochrome insert and cut backbone. </LI>
For the above gel image:
<BLOCKQUOTE><OL><LI>No digest occurred and does not have the expected bands</LI><LI>Another incomplete digestion occurred and the enzymes used were the same as in the Deinococcus digestion with X + P and again, may not be reliable. </LI><LI>The lanes with expected bands are shown in the 1st and 3rd lanes of this Agro T7 treatment. </LI><LI>Linearized backbones </LI></OL></BLOCKQUOTE><CENTER><H3>Heme Oxygenase Functionality</H3></CENTER><P>The heme oxygenase BioBrick was characterised by incubating in the presence of ALA and IPTG. Old colonies of transformed BL21 were used. As our BioBrick was under control of the T7 promoter, the enzyme would need to be induced using IPTG. The heme pathway was also stimulated using d-aminolevulinic acid (ALA). We incubated overnight with a 1 mM final concentrations of ALA and IPTG.</P><H2>Week 8- Tuesday 18th September</H2><CENTER><H2>Sequencing Results</H2></CENTER><P>The sequencing results finally returned for our second round of Gibson Assembly (the bacteriophytochromes). The first look is very promising, Our Blast searches suggested that there were no significant mutations with the protein sequences identical. With this news we began to assemble the switch.</P><CENTER><H2>Assembling the switch</H2></CENTER><P>Using the digests we performed yesterday, we ligated our Heme Oxygenase and Bacteriophytochrome BioBricks and those from vectors not applicable to being a BioBrick. Ellaina performed the ligation using a standard procedure which is listed <A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/ligations">here</A>. The following ligations were performed,</P><CENTER><TABLE border="3" cellpadding="4" cellspacing="0"><TBODY><TR><TH>Reaction</TH><TH>Upstream Part</TH><TH>Downstream Part</TH><TH>Vector</TH></TR><TR><TD>1</TD><TD>Heme Oxygenase + T7 (1C)</TD><TD>Deino Bacteriophytochrome (3C)</TD><TD>Kanamycin (PSB-1K3)</TD></TR><TR><TD>2</TD><TD>Heme Oxygenase + T7 (1C)</TD><TD>Agro Bacteriophytochrome (5C)</TD><TD>Kanamycin (PSB-1K3)</TD></TR><TR><TD>3</TD><TD>Agro + T7 (4K)</TD><TD>Heme Oxygenase (2K)</TD><TD>Chloramphenicol (PSB-1C3)</TD></TR></TBODY></TABLE></CENTER><H2>Week 8- Wednesday 19th September</H2><P>Plates from the transformations of Top 10 cells with the ligation products were inspected. No colonies from the 2K-4K ligations grew. 9 colonies from the 1C-3C and single colony from 1C-5C colonies had grown. These colonies were used to inoculate into LB Kanamycin and left overnight for growth.</P><P>Rob and Matt spun down the BL21 colonies that had been transformed with plasmids containing the T7-HO genes (1C) and had been growing overnight after inducing with IPTG in the presence of ALA. 2 of the cultures were not induced but still had ALA added. These cultures had varying levels of green pigment visible in the pellet after being spun down. One culture was subjected to an additional 10ul of 100mM ALA and left to grow in the refrigerator. This culture will be left to grow for a further 48-72 hours to assess if there is any increase in pigmentation. Pellets from the remaining treatments are in the -18 freezer and will be used to run SDS-PAGE.</P><P>The liquid cultures used for the earlier minipreps were used to streak agar plates and these are now in the incubator. The cultures were then transferred into glycerol stocks and frozen.</P><H2>Week 8- Thursday 20th September</H2><CENTER><H3>Ligation product plasmid prep</H3></CENTER><P>Andrew and Gazelle performed the plasmid prep on our ligation products. The concentration of the ligation was determined using a nanodrop. The results can be seen below,</P><CENTER><TABLE border="1" cellspacing="0" cellpadding="0" valign="center"><TBODY><TR><TD width="200" height="20">Sample ID</TD><TD width="150">DNA conc (ng/uL)</TD><TD width="55">260/280</TD><TD width="55">260/230</TD></TR><TR><TD height="20">1 A (1C-3C) LB-KAN LOW</TD><TD align="right">92.7</TD><TD align="right">1.87</TD><TD align="right">2.1</TD></TR><TR><TD height="20">2 A (1C-3C) LB-KAN HIGH</TD><TD align="right">50.2</TD><TD align="right">1.9</TD><TD align="right">1.97</TD></TR><TR><TD height="20">3 A (1C-3C) LB-KAN HIGH</TD><TD align="right">35.1</TD><TD align="right">1.92</TD><TD align="right">1.77</TD></TR><TR><TD height="20">4 A (1C-3C) LB-KAN HIGH</TD><TD align="right">57</TD><TD align="right">1.9</TD><TD align="right">1.98</TD></TR><TR><TD height="20">5 A (1C-3C) LB-KAN LOW</TD><TD align="right">12.9</TD><TD align="right">1.99</TD><TD align="right">1.07</TD></TR><TR><TD height="20">6 A (1C-3C) LB-KAN HIGH</TD><TD align="right">62.3</TD><TD align="right">1.86</TD><TD align="right">1.76</TD></TR><TR valign="middle"><TD height="20">7 A (1C-3C) LB-KAN HIGH</TD><TD align="right">32.2</TD><TD align="right">1.85</TD><TD align="right">1.7</TD></TR><TR valign="middle"><TD height="20">8 B (1C-5C) LB-KAN</TD><TD align="right">6.1</TD><TD align="right">2.08</TD><TD align="right">1.11</TD></TR><TR valign="middle"><TD height="20">9 A (1C-3C) LB-KAN HIGH</TD><TD align="right">48.9</TD><TD align="right">1.88</TD><TD align="right">1.86</TD></TR><TR valign="center"><TD height="20">10 A (1C-3C) LB-KAN HIGH</TD><TD align="right">41.9</TD><TD align="right">1.85</TD><TD align="right">1.54</TD></TR></TBODY></TABLE></CENTER><P>BL21 <I>E. coli</I> was transformed with the plasmid. The plasmid was also digested with EcoR1 and Spe1.</P><H2>Week 8- Friday 21st September</H2><H3><CENTER>Gel Runs</CENTER></H3><P>The BioBricks produced and our ligation product were digested and a gel was run. The gels can be seen below with annotation.</P><P>Gel 1: We have run against a Heme Oxygenase standard (Lane 1). The gel contains digested fragments from our composite BioBrick (Heme Oxygenase and Deinococcus). The upper band (Black Box) is the Heme Oxygenase with the bacteriophytochrome and the bottom band (blue) is the plasmid backbone.</P><P>Gel 2: From left to right, this gel contains digests of: Mw ladder, T7-Heme Oxygenase (1C), Agro Bacteriophytochrome(5C), T7 Agrobacteriophytochrome (4C), Heme Oxygenase (2C), Kanamycin vector, chloramphenicol vector, 1C+5C ligation product in kanamycin vector, 4K+2K in chloramphenicol.</P><P>The gels provided substantial evidence that the ligation was successful and that we had produced the composite product.</P><CENTER><H3>Transformations</H3></CENTER><P>Using the <A href="https://2012.igem.org/Team:Macquarie/Protocols/TransformationProtocol">standard trasformation protocol</A> Matt and Ryan transformed BL21 to express T7 Heme Oxygenase, T7 Agro Bacteriophytochrome and the T7 Heme Oxygenase-Deinococcus ligation product. These were cultured onto Kanamycin selective plates and left to incubate at room temperature over the weekend. They would be used to test the function of the two components.</P><H2>Week 9- Monday 24th September</H2><H3><CENTER>Heme Oxygenase Characterisation continues</CENTER></H3><P> Today Rob, Harry and Matt performed an SDS-PAGE gel (protocol found <A href="https://2012.igem.org/Team:Macquarie_Australia/Protocols/SDSPAGE">here</A>) using the 1C (Heme Oxygenase) transformant cells. Samples were either uninduced or induced with 1 mM IPTG and 1 mM ALA. ALA was used to stimulate the heme pathway and results in the accumulation of heme. Samples were as follows:</P><CENTER><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="100" height="20">Sample ID</TD><TD width="400">Treatment</TD></TR><TR><TD height="20">2A</TD><TD>Induced over 4 hours with ALA</TD></TR><TR><TD height="20">2B</TD><TD>Induced with ALA overnight at room temperature then allowed to grow for a further 48 hours at 4°C </TD></TR><TR><TD height="20">3A</TD><TD>Non-induced over 4 hours</TD></TR><TR><TD height="20">3B</TD><TD>Non-induced overnight</TD></TR><TR><TD height="20">4A</TD><TD>Induced over 4 hours with ALA</TD></TR><TR><TD height="20">4B</TD><TD>Induced overnight with ALA at room temperature. An extra 20µL ALA was added and allowed to grow for a further 48 hours at 4°C. </TD></TR><TR><TD height="20">5A</TD><TD>Induced over 4 hours with ALA. </TD></TR><TR><TD height="20">5B</TD><TD>Induced overnight with ALA. </TD></TR></TBODY></TABLE></CENTER><P>The gel was stained and will be imaged tomorrow. </P><CENTER><H3>Making Glycerol Stocks</H3><P>Our BioBrick's (Heme Oxygenase, Agro, and Deino Bacteriophytochrome) plasmid were transformed in a TOP10 cell line. We then made glycerol stocks so the cells could be kept for a significantly longer time.</P><CENTER><H3>Developing the Switch</H3><P>The ligation of our Heme Oxygenase and Deino parts was transformed in BL21 on Friday. We continued working with this today. We inoculated cultures (20 mL and 2 mL) containing antibiotics (20 µL and 2 µL) for each our successful ligation products. The cultures were allowed to grow to an OD<SUB>600</SUB> of 0.5. They were then covered in aluminium foil and induced with ALA (0.7 mM) and IPTG (1 mM). The cultures needed to be grown in the dark due to photosensitisation caused by the production of porphyrins. 
</P><H2>Week 9- Tuesday 25th September</H2><P>Today was a very big day for the team with lots of lab work being done as the wiki freeze approached. Silas and Ryan performed a transformation of the heme oxygenase into BL21 in the event that the characterisation being performed was unsuccessful.</P><H3><CENTER>Heme Oxygenase Characterisation</CENTER></H3></CENTER><P>The cultures grown up over night were removed from the shaker. The aluminium foil was removed and we immediately noticed the cultures inoculated with our switch were significantly darker, they were slightly green. We spun the cells down by centrifugation and observed the cells to be an intense green. This provided the first piece of evidence that our Heme Oxygenase BioBrick was functional.</P><H3><CENTER>Bacteriophytochrome Characterisation</CENTER></H3></CENTER><P>Ridding on the excellent news that our oxygenase was functional we lysed the cells and ran an SDS PAGE gel to determine if the bacteriophytochrome was being expressed in the case of the BioBrick and if it was coupling with biliverdin for our composite.</P><P>The gel contains (left to right): Agro + T7 (4K) Sample 3, 4K Sample 1, Heme Oxygenase T7 (1C) + Deinococcus (3C) (1C+3C) sample 7, 1C+3C Sample 6, 1C Sample 6, 1C sample 5, 1C3C Sample 3, Marker.</P><P>This gel contains (left to right): 1C3C Sample 3 Induced, 1C3C Sample 5 Induced, 4K1 Induced, 4K3 Induced, 1C3C Sample 3 Uninduced, 1C3C Sample 5 Uninduced, 1C3C Sample 7, 4K1 Uninduced, 4K3 Uninduced, Marker.</P><H3><CENTER>The Switch</CENTER></H3><P>The cultures of the ligation product we induced with ALA and IPTG were spun down and observed to be green indicating that Heme Oxygenase was functional. The gel above contains the protein bands for these products. To determine if biliverdin had coupled to the bacteriophytochrome we observed the gel under infrared (IR) light.</P><P>The SDS PAGE Gel under IR light indicates the successful coupling of biliverdin with bacteriophytochromes, for a more indepth analysis of the gel see <A href="https://2012.igem.org/Team:Macquarie_Australia/Results#gel">here</A>. The analysis in the results section is much more indepth.</P><H2>Week 9- Wednesday 26th September</H2><P>Today, Rob and Harry used the French press to successfully lyse the BL21 cells that had previously been transformed. These were then spun down and the supernatant was analysed using the Nanodrop. The 1C-3C constructs displayed absorbance peaks at 400nm and 700nm, indicating the coupling of the bacteriophytochrome with the chromophore. The absorbance peak at 700nm was very wide, indicating that a percentage of the bacteriophytochrome construct had been photoconverted to the far-red conformation. The 4K constructs that did not contain biliverdin only gave the expected absorbance peak for protein solution at 280nm.</P><P> Overall, our results were promising in demonstrating the absorbance characteristics of our assembled switch, however it was agreed that a more comprehensive analysis was appropriate. As a result, Rob and Harry proceeded to incubate the 1C3C and 4KE samples with varying concentrations of BV for spectral analysis the following day using a plate-reader. Complete details of the analysis are provided here.</P><P>Rob and Harry continued with the spectral analysis of the BL21 cell samples yielded the spectra shown <A href="https://2012.igem.org/Team:Macquarie_Australia/Results#7">here</A>.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Macquarie_Australia/Notebook2">http://2012.igem.org/Team:Macquarie_Australia/Notebook2</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Macquarie_Australia/Notebook2" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Macquarie_Australia/Notebook2" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Macquarie_Australia/Notebook2&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Macquarie_Australia/Notebook2&amp;oldid=236732" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></OL></BLOCKQUOTE></CENTER></DIV></DIV></DIV></BODY></HTML>