"
Page
Discussion
View source
History
teams
Log in
 
Team:Yeshiva NYC/Project
From 2009.igem.org
You can write a background of your team here.  Give us a background of your team, the members, etc.  Or tell us more about something of your choosing.
Tell us more about your project.  Give us background.  Use this is the abstract of your project.  Be descriptive but concise (1-2 paragraphs)
Team Example 2
Home
The Team
The Project
Parts Submitted to the Registry
Modeling
Notebook
(Or you can choose different headings.  But you must have a team page, a project page, and a notebook page.)
Contents
1 Overall project
2 Spatially encoding temporal information: using diffusional escape of periplasmic reporter proteins as a clock.
3 Project Details
4 Results
Overall project
Spatially encoding temporal information: using diffusional escape of periplasmic reporter proteins as a clock.
Project Details
To achieve our final goal, we first had to make cells whose periplasmic membrane would allow our reporter proteins through to the periplasm. Rafi needed to increase the permeability of the periplasm.  Previous studies have shown that mutation of the Braun Lipoportein, expressed by the  lpp gene, can increase the permeability of the cell.  The approach taken was one in which the lpp gene was deleted using Red/ET recombination.
Next, the reporter molecules needed to be made, and Faige grabbed the reins for this stage. To make the proteins, she started by extracting the genomic DNA from XL1-Blue cells for use in most of the later PCR reactions and transforming cccdB vectors into survival cells. She grew 3 different types of ccdB, all identical except for resistance. One was kanamycin, one was chloramphenicol, and one was ampicillin resistant. After transforming the vectors into the cells, growing colonies from them, making overnights of the colonies, and miniprepping the overnights, she was ready to move on to adding parts. She found leader sequences that were likely to carry her proteins to the membrane and made them into simple Biobricks, ligated to ccdB Cam vectors. She also found a promoter, T7, and a ribosomal binding site (RBS) (Biobrick B0030) and ligated those two into ccdB Cam. This DNA, known as “F DNA”, would be ready for leader sequences to be attached to it when ready. She had chosen eight leader sequences to transform into Biobricks, but only six of them worked (mostly with some tweaking of annealing temperatures), so she decided to go ahead with those six and leave behind the other two. After ligating each leader sequence with ccdB vectors, she ligated those Biobricks with her F DNA to make a combination of promoter, RBS, and leader sequence that was ready to have a protein attached to the back. Here we ran into some trouble; we had chosen an RFP mutant (Biobrick E1010) and a cytoslac protein to attach to the leader sequences, which sounded simple enough but turned out to be harder than it seemed. We ordered primers for the RFP and ran PCR on them, but when we ran the PCR product in a gel, the result showed that the piece was 900 bp, while the Registry of Standard Parts claimed that it was only ~700 bp. Many of the above steps had taken extra tweaking to complete and more time than expected for trouble-shooting, so at this point we were running out of time and decided to go ahead with the RFP despite the strange gel result. Faige ligated the RFP to each leader sequence + promoter + RBS combination and hoped it would work. The ligation tested positive with colonies on the plates, so after miniprepping the DNA, she went ahead and transformed the DNA into BL21 DE3 cells to test for expression (we usually use XL1-Blue for transformations). She streaked the cells onto plates coated with IPTG and also put some IPTG onto plates that already had colonies of BL21 DE3 cells growing, but the colonies that grew were not red, as they should have been with RFP expression. She also tried a liquid culture of the cells in 2XYT media (we usually use LB media) and IPTG, but the cells that grew were not red. In the meantime, she was also attempting to attach the cytoslac protein to the leader sequences and it was working so far, but then she realized that her final result would be Kan resistant, as are Rafi’s weakened cells that her results would need to be transformed into. We select our positive ligation products by transforming into destination plasmids or cells with a resistance different from that of the inserts, so this was a problem. The cytoslac DNA’s resistance would have to be removed in order for the system to work. To do this, Faige digested cytoslac DNA in preparation for a 3-way ligation and then ran the digest on a gel, extracting the insert containing cytoslac and leaving behind the plasmid containing the resistance. She then ligated this extracted DNA to a leader sequence into non-Kan resistant ccdB. At the same time, she continued her ligation with the cytoslac that had the wrong DNA, so that she could later cut out the resistance, in case the first method of changing resistance didn’t work. These colonies didn’t grow well, so we stopped this method and continued with the gel-extraction instead, which did seem to be working well.
The purpose of Julie’s part of the experiment was to create a cost efficient, simple method for analyzing diffusion in one dimension. It was her job to come up with the plan of how to graph diffusion as a plot of intensity versus position from just a picture of a drop of a substance on an agar plate. To go about doing this she searched online as to what program to use that would solve this problem. She found that by taking a picture with a UV camera, the gel 100, with molecular imaging software and then exporting the document as a JPEG she could then use imageJ to take care of all the numbers. As she thought everything was taken care of there was a stumbling block- the numbers she got from imageJ were hundreds of pages long with intensities separated by rows of one pixel each. She then, with her professor (Dr. Mike here is a shot out), figured out that if we cropped the picture using Microsoft photo editor before exporting it into imageJ then we could narrow down the data to contain 10 pixels long and however wide the diffusion pattern contained. After cropping the picture, she exported the picture into imageJ and saved as a text image. The text image document was then opened up in Microsoft word and the best row of intensities was selected and copied and pasted into excel into columns with the help of the transpose icon in paste special. The data was then plotted with the x, y scatter plot chart. The graphs were upside down so she then flipped them, found the baseline, and corrected the data by subtracting intensity by the baseline to get the raw data for that set. Once the methodology of the process of deriving the data was achieved the next line of business for her was to decide what timeframe to monitor the diffusion. The timeframe decided would be 2, 4, 6, and 24 hours. But what would be her test subject. Her professor decided that dyes that would not easily precipitate would be the best to try out first. So she placed 6 dyes including: Bromothymol blue, Bromocresol purple, Janus Green, Methyl Green, Methyl Red, and Sudan Black. After 24hours the only ones that diffused the furthest were then further tested, which included: Methyl Green, Methyl Red, and Bromocresol purple. The only dye that showed up under the UV camera was methyl red and was therefore the only choice in our testing. The next element of the analysis that needed to be worked on was the agar plates on which she was going to test diffusion of methyl red. For one because we were making the plates with a certain resistance in the hope that we could test it with the biobricks in the knockout that was being made by Faige and Rafi, the LB medium had to be at a cool enough temperature to touch to add the antiobiotic and therebye she did not have enough time to pour the plates before they solidified. The second problem was she had to make the plates thin enough so that one dimensional diffusion could be tested. And as always, the third problem was that of reproducibility. The first step taken to solve this problem was to glue two razor blades together separated by 3-4 millimeters with melted plastic as a backbone. The problem with this was the razor blades when glued together would not cut the entire way through. With this idea thrown out, the idea of a mold was suggested made of 2 plastic pieces super glued to each other separated by 1-2 mm. The idea was if we pour the plates while they are hot in this mold then we could get a lot of strips and when the time would come to place antibiotic we would just place a stream of water whatever antibiotic resistance was needed. The baselines, Gaussians, and the diffusion coefficients were calculated and the results were that the Gaussians with the mold came out more Gaussian than those that were cut up with the two razor blades. She then tried the exact experiment of the dyes with cytochrome C, a protein from bovine heart. In order to dissolve the solid sample we had to get the concentration to equal that of the dyes, because we knew that that concentration worked. To do that we looked up the extinction coefficient for methyl red multiplied by the concentration and by the volume added has to equal the same product for cyctochrome C.
Results
Retrieved from " http://2009.igem.org/Team:Yeshiva_NYC/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
