<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Stockholm_23_August_2010"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Stockholm/23_August_2010">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Stockholm/23_August_2010&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Stockholm/23_August_2010&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Stockholm/23_August_2010&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Stockholm/23_August_2010" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Stockholm/23 August 2010</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV class="navbar"><CENTER><TABLE><TBODY><TR><TD><A href="/Team:Stockholm" title="Team:Stockholm">Home</A></TD><TD><A href="/Team:Stockholm/Project_Idea/Introduction" title="Team:Stockholm/Project Idea/Introduction">Project Idea</A></TD><TD><A href="/Team:Stockholm/Lab_work/Notebook" title="Team:Stockholm/Lab work/Notebook">Lab Work</A></TD><TD><A href="/Team:Stockholm/Results/BioBricks" title="Team:Stockholm/Results/BioBricks">Results</A></TD><TD><A href="/Team:Stockholm/Modelling/Introduction" title="Team:Stockholm/Modelling/Introduction">Modelling</A></TD><TD><A href="/Team:Stockholm/Team/Members" title="Team:Stockholm/Team/Members">Team</A></TD><TD>      Follow us on  and </TD><TD>       <A href="https://2010.igem.org/">2010.igem.org </A></TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="indented"><DIV class="navbar"><TABLE><TBODY><TR><TD>↳  <A href="/Team:Stockholm/Lab_work/Notebook" title="Team:Stockholm/Lab work/Notebook">Notebook</A></TD><TD><A href="/Team:Stockholm/Lab_work/Safety" title="Team:Stockholm/Lab work/Safety">Safety</A></TD><TD><A href="/Team:Stockholm/Lab_work/Protocols" title="Team:Stockholm/Lab work/Protocols">Protocols</A></TD></TR></TBODY></TABLE></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Nina"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Nina</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Sequencing_of_fusion_protein"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Sequencing of fusion protein</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Ligation_of_protein_A_in_peX"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Ligation of protein A in peX</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#PCR_of_ligation_product"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">PCR of ligation product</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Agarose_gel_on_protein_A"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Agarose gel on protein A</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Andreas"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Andreas</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Assembly_of_SOD.2FyCCS.E2.8B.85His_constructs_into_pSB1K3"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Assembly of SOD/yCCS⋅His constructs into pSB1K3</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Plasmid_prep"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Plasmid prep</SPAN></A></LI><LI class="toclevel-4 tocsection-9"><A href="#Sequencing"><SPAN class="tocnumber">2.1.1.1</SPAN><SPAN class="toctext">Sequencing</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Enzyme_inactivation"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Enzyme inactivation</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Transfer_of_RFP_coding_device_to_pEX"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Transfer of RFP coding device to pEX</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Colony_restreaks"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">Colony restreaks</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Gel_verification_of_Dig_pEX_X.2BP_and_Dig_pC.RFP_X.2BP"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">Gel verification of Dig pEX X+P and Dig pC.RFP X+P</SPAN></A></LI><LI class="toclevel-4 tocsection-14"><A href="#Results"><SPAN class="tocnumber">2.3.2.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Digestion_of_new_pEX_vector"><SPAN class="tocnumber">2.3.3</SPAN><SPAN class="toctext">Digestion of new pEX vector</SPAN></A></LI><LI class="toclevel-4 tocsection-16"><A href="#Gel_verification"><SPAN class="tocnumber">2.3.3.1</SPAN><SPAN class="toctext">Gel verification</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Ligation"><SPAN class="tocnumber">2.3.4</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Transformation"><SPAN class="tocnumber">2.3.5</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#ON_cultures"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">ON cultures</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Assembly_of_His.E2.8B.85SOD.2FyCCS_constructs"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Assembly of His⋅SOD/yCCS constructs</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Ligation_2"><SPAN class="tocnumber">2.5.1</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Transformation_2"><SPAN class="tocnumber">2.5.2</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#Johan"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Johan</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Nina">Nina</SPAN></H2><H3><SPAN class="mw-headline" id="Sequencing_of_fusion_protein">Sequencing of fusion protein</SPAN></H3><P>I prepared one tube of the fusion protein made up by protein A and IgG protease:
</P><P>15 ul plasmid from a mini-prep and 1.5 ul (10uM) primer VF2 of the bank vectors (C) verification primers.
</P><UL><LI>ASB0045 B00
</LI></UL><H3><SPAN class="mw-headline" id="Ligation_of_protein_A_in_peX">Ligation of protein A in peX</SPAN></H3><P>I performed a ligation of protein A with the peX vector in order to induce the protein with IPTG o be able to see that the protein can become overexpressed in BL21 cells.  
</P><P>Ligation:
</P><P>I wanted 25 ng vector
</P><P>90 ng/ul / X = 25 ng/ul
</P><P>X = 3 which means a 1:3 dilution. 1 ul vector sample and 2 ul dH2O. This results in 30 ng vector which is OK. 
</P><P>I wanted ~4 times more gene than vector. Usually it is enough with 3 times more gene than vector but since this gene is pretty small (180 bp) I prefer to have a 4X more gene. 
</P><P>4 * 30 = 120 
</P><P>120 ng/ul / 30ng/ul = 4 times more gene than vector. 
</P><UL><LI>Vector peX: 1 ul
</LI><LI>Gene protein A: 5 ul
</LI><LI>Quick ligase: 1 ul
</LI><LI>2X quick ligase buffer: 7 ul
</LI></UL><P>Tot Volume: 14 ul
</P><P>I incubated in RT for 15 min.  
</P><H3><SPAN class="mw-headline" id="PCR_of_ligation_product">PCR of ligation product</SPAN></H3><P>I wanted to check if I had a correct ligation before I transformed it into BL21 cells. Therefore I performed a PCR on the ligation product with peX verification primers. 
</P><P>PCR mix:
</P><UL><LI>MgCl 1 ul
</LI><LI>Buffer 5X 10 ul
</LI><LI>dNTP 1 ul
</LI><LI>primer peX forward 3 ul
</LI><LI>primer peX reverse 3 ul
</LI><LI>polymerase PjuX7 1 ul
</LI><LI>H2O 30 ul
</LI><LI>Template 1 ul
</LI></UL><P>PCR prgm:
</P><H3><SPAN class="mw-headline" id="Agarose_gel_on_protein_A">Agarose gel on protein A</SPAN></H3><P>I ran an agarose gel 1 % 100 V on the PCR product of protein A in the peX vector in order to verify that the ligation has occured properly.
</P><P>Ladder: FastRuler™ Low Range DNA Ladder, ready-to-use, 50-1500 bp Fermentas
</P><P>The gel results show that I have had a good ligation since I got a band just below 400 on the ladder which I expected. The protein A gene is 180 nt and the verification primers add 200 nt which give a 380 nt band. This allows for a transformation tomorrow of the ligation product into BL21 cells. 
</P><H2><SPAN class="mw-headline" id="Andreas">Andreas</SPAN></H2><H3><SPAN class="mw-headline" id="Assembly_of_SOD.2FyCCS.E2.8B.85His_constructs_into_pSB1K3">Assembly of SOD/yCCS⋅His constructs into pSB1K3</SPAN></H3><P><I>Continued from 21/8</I></P><H4><SPAN class="mw-headline" id="Plasmid_prep">Plasmid prep</SPAN></H4><TABLE border="1" cellpadding="1" cellspacing="0" align="right"><CAPTION align="bottom">†Samples concentrated in vacuum evaporator
</CAPTION><TBODY><TR><TH colspan="5">DNA concentrations
</TH></TR><TR><TD width="130"> </TD><TD colspan="2" align="center">Before sample conc.
</TD><TD colspan="2" align="center">After sample conc.
</TD></TR><TR><TH>Sample
</TH><TH width="90">Conc. [ng/μl]
</TH><TH width="70">A<SUB>260</SUB>/A<SUB>280</SUB></TH><TH width="90">Conc. [ng/μl]
</TH><TH width="70">A<SUB>260</SUB>/A<SUB>280</SUB></TH></TR><TR><TD>pSB1K3.SOD⋅His 1†</TD><TD align="center">61.50
</TD><TD align="center">1.86
</TD><TD align="center">100.9
</TD><TD align="center">1.80
</TD></TR><TR><TD>pSB1K3.SOD⋅His 2†</TD><TD align="center">76.45
</TD><TD align="center">1.78
</TD><TD align="center">126.1
</TD><TD align="center">1.83
</TD></TR><TR><TD>pSB1K3.yCCS⋅His 2
</TD><TD align="center">171.5
</TD><TD align="center">1.88
</TD><TD align="center">–</TD><TD align="center">–</TD></TR><TR><TD>pSB1K3.yCSS⋅His 3†</TD><TD align="center">84.15
</TD><TD align="center">1.88
</TD><TD align="center">133.6
</TD><TD align="center">1.84
</TD></TR></TBODY></TABLE><P><I>From 21/8 cultures</I></P><UL><LI>E.Z.N.A Plasmid Mini kit I
</LI><LI>50 μl elution volume
</LI></UL><H5><SPAN class="mw-headline" id="Sequencing">Sequencing</SPAN></H5><P>Samples prepared for sequencing:
</P><UL><LI>15 μl DNA (&gt;100 ng/μl)
</LI><LI>1.5 μl 10 μM pSB-VR primer
</LI></UL><TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TH>Sample
</TH><TH>Label
</TH><TH>Seq #
</TH></TR><TR><TD width="130">pSB1K3.SOD⋅His 1
</TD><TD width="50">pK-SH1
</TD><TD width="45" align="center">938
</TD></TR><TR><TD>pSB1K3.SOD⋅His 2
</TD><TD>pK-SH2
</TD><TD align="center">939
</TD></TR><TR><TD>pSB1K3.yCCS⋅His 2
</TD><TD>pK-yH2
</TD><TD align="center">940
</TD></TR><TR><TD>pSB1K3.yCCS⋅His 3
</TD><TD>pK-yH3
</TD><TD align="center">941
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Enzyme_inactivation">Enzyme inactivation</SPAN></H3><P>Inactivated restriction enzymes in digestion samples from 19/8 and 20/8 in 80 °C, 10 min.
</P><UL><LI>Dig pEX X+P
</LI><LI>Dig pC.RFP X+P
</LI><LI>Dig His E+A
</LI><LI>Dig m-yCCS N+P
</LI><LI>Dig m-SOD N+P
</LI></UL><H3><SPAN class="mw-headline" id="Transfer_of_RFP_coding_device_to_pEX">Transfer of RFP coding device to pEX</SPAN></H3><H4><SPAN class="mw-headline" id="Colony_restreaks">Colony restreaks</SPAN></H4><P><I>Results from 21/8</I></P><P>All clones grew well on the Amp plate, but not on Km, indicating RFP has indeed been transfered from pSB1K3 to a target AmpR vector, and that pSB1K3 does not express AmpR.
</P><P><I>I later realized that the transfer of RFP was not from pSB1K3, but from pSB1C3, making this restreak pointless.</I></P><H4><SPAN class="mw-headline" id="Gel_verification_of_Dig_pEX_X.2BP_and_Dig_pC.RFP_X.2BP">Gel verification of Dig pEX X+P and Dig pC.RFP X+P</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Gel verification of digested pEX and pSB1C3.RFP samples from 19/8.</B>Loading vol.: 4 μl λ; 4 μl sample.λ = O'GeneRuler 1 kb DNA ladder.</DIV></DIV></DIV><P>Ran a gel of the digestion samples from 19/8 to verify the sizes of vectors and inserts.
</P><P>1 % agarose, 100 V, 1 h 20 min
</P><P><B>Expected bands</B></P><UL><LI>Dig pEX X+P
</LI><LI>4453 bp (vector)
</LI><LI>1237 bp (insert)
</LI></UL><LI>Dig pC.RFP X+P
</LI><LI>2054 bp (pSB1C3 vector)
</LI><LI>1095 bp (RFP insert)
</LI><H5><SPAN class="mw-headline" id="Results">Results</SPAN></H5><P>The bands for &quot;pC.RFP X+P&quot; correspond well to what was expected. However, none of the &quot;pEX X+P&quot; bands do. It might be possible that this is actually a pMA vector, carrying a ≈800 bp insert (unclear what). This would explain why ligations and transformations from 19/8 resulted in red colonies, but colony PCR amplification with pEX primers doesn't result in any bands.
</P><H4><SPAN class="mw-headline" id="Digestion_of_new_pEX_vector">Digestion of new pEX vector</SPAN></H4><P>Used pEX vector prepared and verified 21/8 for new pEX digestion with XbaI and PstI.
</P><TABLE border="1" cellpadding="1" cellspacing="0"><CAPTION align="bottom">[pEX] = 55.52 ng/μl
</CAPTION><TBODY><TR><TH colspan="3">Digestion mix
</TH></TR><TR><TD width="110"><B>10X FD buffer</B></TD><TD width="30" align="center">3 μl
</TD><TD width="110" rowspan="6">[DNA] = 33.3 ng/μlInc.: 37 °C, 30 min
</TD></TR><TR><TD><B>dH<SUB>2</SUB>O</B></TD><TD align="center">7 μl
</TD></TR><TR><TD><B>1 μg DNA (pEX)</B></TD><TD align="center">18 μl
</TD></TR><TR><TD><B>FD XbaI</B></TD><TD align="center">1 μl
</TD></TR><TR><TD><B>FD PstI</B></TD><TD align="center">1 μl
</TD></TR><TR><TD> </TD><TD align="center"><B>30 μl</B></TD></TR></TBODY></TABLE>
Enzyme inactivation (<UL>after</UL> ligation): 80 °C, 5 min
<H5><SPAN class="mw-headline" id="Gel_verification">Gel verification</SPAN></H5><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Gel verification of new pEX vector digestion.</B>Loading vol.: 4 μl λ; 4 μl sample.λ = O'GeneRuler 1 kb DNA ladder.</DIV></DIV></DIV><P>1 % agarose, 100 V, 35 min
</P><P><B>Results</B>
pEX confirmed - bands correspond well.
</P><H4><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H4><TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TH colspan="4">Ligation mix
</TH></TR><TR><TD><B>100 ng vector (pEX)</B></TD><TD align="center">3 μl
</TD><TD rowspan="2">1/3ratio
</TD><TD rowspan="6">[Dig pEX X+P] = 33.3 ng/μl (23/8)[Dig pC.RFP X+P]=66.6 ng/μl (19/8)Inc.: 22 °C, 10 min
</TD></TR><TR><TD><B>165 ng insert (RFP)</B></TD><TD align="center">2.5 μl
</TD></TR><TR><TD><B>5X Rapid Ligation buf.</B></TD><TD align="center" colspan="2">4 μl
</TD></TR><TR><TD><B>dH<SUB>2</SUB>O</B></TD><TD align="center" colspan="2">9.5 μl
</TD></TR><TR><TD><B>T4 DNA Ligase</B></TD><TD align="center" colspan="2">1 μl
</TD></TR><TR><TD> </TD><TD align="center" colspan="2"><B>20 μl</B></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H4><P>Procedures based on quick transformation protocol:
</P><UL><LI>1.5 μl ligation mix. 15 min incubation on ice.
</LI><LI>30 s heat-shock in 42 °C
</LI></UL><P>Plating onto 100 Amp LB agar. 37 °C ON.
</P><H3><SPAN class="mw-headline" id="ON_cultures">ON cultures</SPAN></H3><P>Set ON cultures for preparation of glycerol stocks:
</P><UL><LI>3 ml LB + Km 50:
</LI><LI>pSB1K3.SOD⋅His 1
</LI><LI>pSB1K3.SOD⋅His 2
</LI><LI>pSB1K3.yCCS⋅His 2
</LI><LI>pSB1K3.yCCS⋅His 3
</LI></UL><LI>3 ml LB + Amp 100:
</LI><LI>pMA.His (Picked 19/8)
</LI><LI>pEX (Picked 19/8)
</LI><P>Inc. 30 °C, ON.
</P><H3><SPAN class="mw-headline" id="Assembly_of_His.E2.8B.85SOD.2FyCCS_constructs">Assembly of His⋅SOD/yCCS constructs</SPAN></H3><H4><SPAN class="mw-headline" id="Ligation_2">Ligation</SPAN></H4><P>Used digestion samples from 20/8 for ligations. Ligation protocol identical to that from 20/8.
</P><H4><SPAN class="mw-headline" id="Transformation_2">Transformation</SPAN></H4><P>Standard transformation protocol.
</P><UL><LI>1 μl ligation mix
</LI><LI>Plating on Km 50.
</LI></UL><H2><SPAN class="mw-headline" id="Johan">Johan</SPAN></H2><UL><LI> PCR of bFGF from original vector
</LI></UL><P>1 µl Pfu polymerase
</P><P>5 µl 10x Pfu buffer
</P><P>36 µl H2O
</P><P>1 µl DNA
</P><P>1 µl dNTP
</P><P>3 µl bFGF for primer
</P><P>3 µl bFGF rev primer
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Stockholm/23_August_2010">http://2010.igem.org/Team:Stockholm/23_August_2010</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Stockholm/23_August_2010" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Stockholm/23_August_2010" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Stockholm/23_August_2010&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Stockholm/23_August_2010&amp;oldid=204625" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>