<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Stockholm_9_August_2010"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Stockholm/9_August_2010">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Stockholm/9_August_2010&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Stockholm/9_August_2010&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Stockholm/9_August_2010&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Stockholm/9_August_2010" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Stockholm/9 August 2010</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV class="navbar"><CENTER><TABLE><TBODY><TR><TD><A href="/Team:Stockholm" title="Team:Stockholm">Home</A></TD><TD><A href="/Team:Stockholm/Project_Idea/Introduction" title="Team:Stockholm/Project Idea/Introduction">Project Idea</A></TD><TD><A href="/Team:Stockholm/Lab_work/Notebook" title="Team:Stockholm/Lab work/Notebook">Lab Work</A></TD><TD><A href="/Team:Stockholm/Results/BioBricks" title="Team:Stockholm/Results/BioBricks">Results</A></TD><TD><A href="/Team:Stockholm/Modelling/Introduction" title="Team:Stockholm/Modelling/Introduction">Modelling</A></TD><TD><A href="/Team:Stockholm/Team/Members" title="Team:Stockholm/Team/Members">Team</A></TD><TD>      Follow us on  and </TD><TD>       <A href="https://2010.igem.org/">2010.igem.org </A></TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="indented"><DIV class="navbar"><TABLE><TBODY><TR><TD>↳  <A href="/Team:Stockholm/Lab_work/Notebook" title="Team:Stockholm/Lab work/Notebook">Notebook</A></TD><TD><A href="/Team:Stockholm/Lab_work/Safety" title="Team:Stockholm/Lab work/Safety">Safety</A></TD><TD><A href="/Team:Stockholm/Lab_work/Protocols" title="Team:Stockholm/Lab work/Protocols">Protocols</A></TD></TR></TBODY></TABLE></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Hassan"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Hassan</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Folate"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Folate</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Possible_Genes"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Possible Genes</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Nina"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Nina</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Overnight_culture_of_Tyrosinase"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Overnight culture of Tyrosinase</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Digestion_of_protein_A_.28ZZ_domain.29_and_bank_vector_C"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Digestion of protein A (ZZ domain) and bank vector C</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Gel_clean_up_of_digest_samples"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Gel clean up of digest samples</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Second_digestion_of_protein_A_.28ZZ_domain.29_and_bank_vector_C"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Second digestion of protein A (ZZ domain) and bank vector C</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Mimmi"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Mimmi</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#MITF"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">MITF</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#-_Gel"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">- Gel</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#MITF_.2B_yCCS_and_RFP"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">MITF + yCCS and RFP</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Verification_PCR"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">Verification PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Andreas"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Andreas</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Vector_preparation_for_CPP_cloning"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Vector preparation for CPP cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Digestion"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Gel_verification"><SPAN class="tocnumber">4.1.2</SPAN><SPAN class="toctext">Gel verification</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Gel_extraction"><SPAN class="tocnumber">4.1.3</SPAN><SPAN class="toctext">Gel extraction</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Amp_plate_preparation"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Amp plate preparation</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Cloning_of_IgG_protease"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Cloning of IgG protease</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#yCCS_clone_analysis"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">yCCS clone analysis</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Gel_verification_2"><SPAN class="tocnumber">4.4.1</SPAN><SPAN class="toctext">Gel verification</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Hassan"> Hassan </SPAN></H2><H3><SPAN class="mw-headline" id="Folate"> Folate </SPAN></H3><P>below links and articles are meant to be study when I'm finished with interaction map and prove of concept!
</P><UL><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/12761857" class="external autonumber" rel="nofollow">[1]</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/17307853,18403248,17698004,17220358,16359333,10201405,15489424,2254281,15865426,17923103,11041843,2185835,11371182,9287033,1958664,8208138,8019142,8408018,8634297,1098865?report=docsum" class="external autonumber" rel="nofollow">[2]</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/15611104" class="external autonumber" rel="nofollow">[3]</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/17698004" class="external autonumber" rel="nofollow">[4]</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/8208138" class="external autonumber" rel="nofollow">[5]</A></LI><LI><A href="http://www.genome.jp/kegg-bin/show_pathway?org_name=eco&amp;mapno=00790&amp;mapscale=&amp;show_description=hide" class="external autonumber" rel="nofollow">[6]</A></LI></UL><H3><SPAN class="mw-headline" id="Possible_Genes"> Possible Genes </SPAN></H3><UL><LI> decrease in CAT (Catalase) : <A href="http://www.ncbi.nlm.nih.gov/pubmed/10537016" class="external autonumber" rel="nofollow">[7]</A><A href="http://onlinelibrary.wiley.com/doi/10.1034/j.1600-0749.2003.00087.x/abstract" class="external autonumber" rel="nofollow">[8]</A></LI><LI> increase in SOD (SOD3) : <A href="http://www.ncbi.nlm.nih.gov/pubmed/12692376" class="external autonumber" rel="nofollow">[9]</A></LI><LI> possible genes involved in Melanocytes adhesion: &quot;An imbalance in cell adhesion molecules was also observed in vitiligo melanocytes. Differentially downregulated genes in vitiligo melanocytes included integrin alpha 7, integrin beta-like 1, integrin beta 7 and catenin alpha-like 1 (cadherin-associated protein). The adhesion between the melanocytes and the basal membrane is mediated by integrins and the contacts between melanocytes and keratinocytes are mediated by cadherins (Hara et al., 1994; Tang et al., 1994; Zambruno et al., 1993). The importance of regulated expression levels of cadherins have also been shown during migration of melanoblasts (Nishimura et al., 1999). It is difficult to predict if this specific downregulation of adhesion molecules impairs the attachment of the melanocytes in the basal layer of the epidermis; however, detachment and transepidermal loss of melanocytes have previously been reported in vitiligo vulgaris (Gauthier et al., 2003).&quot;<A href="http://www.ncbi.nlm.nih.gov/pubmed/18426409" class="external autonumber" rel="nofollow">[10]</A>  May also look at <A href="http://www.ncbi.nlm.nih.gov/pubmed/17850512" class="external autonumber" rel="nofollow">[11]</A> if more interested.
</LI></UL><P>
And those from <A href="https://2010.igem.org/Team:Stockholm/22_July_2010" class="external text" rel="nofollow">July 22</A> with data from <A href="https://2010.igem.org/Team:Stockholm/Planning" class="external text" rel="nofollow">planning</A> will bring enough explanation for interaction map.
</P><P>I think by end of this week a wall of text will appear in modelling page!
</P><H2><SPAN class="mw-headline" id="Nina">Nina</SPAN></H2><H3><SPAN class="mw-headline" id="Overnight_culture_of_Tyrosinase">Overnight culture of Tyrosinase</SPAN></H3><P>I inoculated eight falcon tubes containing 12 ml LB and 24 ul amphicillin (50mg/ml) with the tyrsoinase site directed mutagenesis colonies number 1, 2, 3, 4, 5, 6, 7 &amp; 8. These were incubated overnight in 37 °C with shake and will be miniprepped tomorrow. 
</P><H3><SPAN class="mw-headline" id="Digestion_of_protein_A_.28ZZ_domain.29_and_bank_vector_C">Digestion of protein A (ZZ domain) and bank vector C</SPAN></H3><P>I cut a previously PCR product of protein A ZZ domain. The first digestion was carried out with the restriction enzyme NgoMIV. However since this enzyme is from New England BioLabs and has another buffer composition compared to the second digestion with the restriction enzyme AgeI from Fermentas (fast digest) I had to run a gel clean up before following up with the second digest after the first digest. 
</P><P>I also cut IgG protease in the bank vector C since I wanted to replace the insert of IgG protease in bank vector C with an insert of protein A ZZ domain. 
</P><P>First digest:
</P><UL><LI>DNA vector with IgG protease 20 ul
</LI><LI>Restriction enzyme  NgoMIV 1 ul
</LI><LI>Buffer #4 3 ul
</LI><LI>H2O 6 ul
</LI></UL><UL><LI>DNA PCR product of protein A ZZ domain 10 ul
</LI><LI>Restriction enzyme  NgoMIV 1 ul
</LI><LI>Buffer #4 3 ul
</LI><LI>H2O 6 ul
</LI></UL><P>I incubated the vector in 3 hours and the PCR product in 1 hour, both in 37 °C.
</P><P>After 3 hours of incubation of the vector I added 1.5 ul CIAP to this sample in order to dephosphorylate the vector to prevent the vector to couple back into a circle. The sample was yet again incubated in 37 °C for 30 min. 
</P><H3><SPAN class="mw-headline" id="Gel_clean_up_of_digest_samples">Gel clean up of digest samples</SPAN></H3><P>I ran the digested samples PCR product protein A ZZ domain  and IgG protease in bank vector C in an 1 % agarose gel 80 V. 
</P><P>Ladder: GeneRuler™ DNA Ladder Mix, ready-to-use, 100-10,000 bp Fermentas 
</P><P>Arrangement on gels: 
</P><P>I cut out the red marked bands with a scalpel and carried out a gel clean up according to the method decribed in protocols. 
</P><P>Gel clean up:
</P><UL><LI>Protein A ZZ domain: 300 mg * 3 = 900 ul
</LI><LI>IgG protease: 400 mg * 3 = 1200 ul
</LI></UL><P>In step 3 I added 10 ul  of QIAEXII to the protein A ZZ domain sample and 30 ul QIAEXII to the IgG protease sample (340 ng/ul * 20 ul = 6.8 ug). 
</P><P>In step 9 I eluted with 35 ul H2O and incubated the samples in RT for 5 min. 
</P><P>Spectophotometer:
</P><H3><SPAN class="mw-headline" id="Second_digestion_of_protein_A_.28ZZ_domain.29_and_bank_vector_C">Second digestion of protein A (ZZ domain) and bank vector C</SPAN></H3><P>After the gel clean up I digested the samples with the restriction enzyme (fast digest) AgeI. 
</P><P>Second digest:
</P><UL><LI>DNA vector with IgG protease 2 ul
</LI><LI>Restriction enzyme AgeI 1 ul
</LI><LI>Buffer fast digest 10X 2 ul
</LI><LI>H2O 15 ul 
</LI></UL><UL><LI>DNA PCR product of protein A ZZ domain 10 ul
</LI><LI>Restriction enzyme AgeI 1 ul
</LI><LI>Buffer fast digest 10X 2 ul
</LI><LI>H2O 17 ul 
</LI></UL><P>I incubated the vector in 5 minutes and the PCR product in 30 minutes, both in 37 °C. Then the samples had their AgeI heat inactivated in 80 °C for 5 min.
</P><H2><SPAN class="mw-headline" id="Mimmi"> Mimmi </SPAN></H2><H3><SPAN class="mw-headline" id="MITF"> MITF </SPAN></H3><H4><SPAN class="mw-headline" id="-_Gel"> - Gel </SPAN></H4><UL><LI>No products!
</LI></UL><DL><DD>--&gt; Trying more cycles of low annealing temperature
</DD></DL><H3><SPAN class="mw-headline" id="MITF_.2B_yCCS_and_RFP"> MITF + yCCS and RFP </SPAN></H3><H4><SPAN class="mw-headline" id="Verification_PCR"> Verification PCR </SPAN></H4><TABLE><TBODY><TR><TH> Mix
</TH><TD> (µl)
</TD><TD> X5
</TD><TD> X3
</TD><TH colspan="2"><B>yCCS/RFP</B></TH><TH colspan="2"><B>MITF</B></TH></TR><TR><TD> sH<SUB>2</SUB>O
</TD><TD> 22.5
</TD><TD> 112.5
</TD><TD> 67.5
</TD><TD colspan="2"> pSB1_VF2
</TD><TD colspan="2"> MITF_F
</TD></TR><TR><TD> F primer
</TD><TD> 1
</TD><TD> 5
</TD><TD> 3
</TD><TD colspan="2"> pSB1_VR
</TD><TD colspan="2"> MITF_R
</TD></TR><TR><TD> R primer
</TD><TD> 1
</TD><TD> 5
</TD><TD> 3
</TD><TD colspan="2"> 970/925bp
</TD><TD colspan="2"> ~1300bp
</TD></TR><TR><TD> DNA
</TD><TD> 0.5
</TD><TD> 4X0.5
</TD><TD> 2X0.5
</TD></TR><TR><TD align="right"> tot
</TD><TD> 25
</TD><TH colspan="2"> conditions
</TH><TH colspan="2"> conditions
</TH></TR><TR><TH> time
</TH><TH>°C
</TH><TH> time
</TH><TH>°C
</TH></TR><TR><TD> 2m
</TD><TD> 94
</TD><TD> 2m
</TD><TD> 94
</TD></TR><TR><TD> 30s
</TD><TD> 94
</TD><TD> )
</TD><TD> 30s
</TD><TD> 94
</TD><TD> )
</TD></TR><TR><TD> 30s
</TD><TD> 60
</TD><TD>&gt; 30 cycles
</TD><TD> 30s
</TD><TD> 60
</TD><TD>&gt; 30 cycles
</TD></TR><TR><TD> 2m
</TD><TD> 72
</TD><TD> )
</TD><TD> 2m40s
</TD><TD> 72
</TD><TD> )
</TD></TR><TR><TD> 10m
</TD><TD> 72
</TD><TD> 10m
</TD><TD> 72
</TD></TR><TR><TD> oo
</TD><TD> 10
</TD><TD> oo
</TD><TD> 10
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Andreas"> Andreas </SPAN></H2><H3><SPAN class="mw-headline" id="Vector_preparation_for_CPP_cloning"> Vector preparation for CPP cloning </SPAN></H3><P>pSB1C3 vectors need to be prepared for cloning of our CPPs.
</P><UL><LI>Digested with NgoMIV (N) &amp; SpeI (S) (for C-CPPs)
</LI><LI>Digested with XbaI (X) &amp; AgeI (A) (for N-CPPs)
</LI></UL><H4><SPAN class="mw-headline" id="Digestion">Digestion</SPAN></H4><P>Vector: pSB1C3.BBa_J18930 A (133.5 ng/μl)
</P><TABLE border="1" cellpadding="2" cellspacing="0"><TBODY><TR><TH>[μl]
</TH><TH>N+S
</TH><TH>X+A
</TH></TR><TR><TD>10X FD buffer
</TD><TD>3
</TD><TD>3
</TD></TR><TR><TD>Vector DNA
</TD><TD>25
</TD><TD>25
</TD></TR><TR><TD>dH<SUB>2</SUB>O
</TD><TD>0
</TD><TD>0
</TD></TR><TR><TD>FD XbaI
</TD><TD> --
</TD><TD>0.5
</TD></TR><TR><TD>FD AgeI
</TD><TD> --
</TD><TD>0.5
</TD></TR><TR><TD>FD SpeI
</TD><TD>1†</TD><TD> --
</TD></TR><TR><TD>NgoMIV (NEB)
</TD><TD>1
</TD><TD> --
</TD></TR><TR><TD> </TD><TD>30
</TD><TD>30
</TD></TR></TBODY></TABLE><P>†<I>Added after 1:30 of incubation with NgoMIV, since the latter is not a FD enzyme.</I></P><H4><SPAN class="mw-headline" id="Gel_verification">Gel verification</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Verification of pSB1C3 vector digestion.<B>Settings:</B> 1 % agarose, 110 V, 25 min.<B>Loading:</B> 1 kb λ, N+S, X+A. 3 μl λ, 4 μl sample.</DIV></DIV></DIV><P>1 % agarose, 110 V, 25 min
</P><P><B>Expected bands:</B> 2070 bp (pSB1C3), 714 bp (BBa_J18930)
</P><P><B>Results</B>
Good digestion, no uncut plasmid. Bands with correct sizes.
</P><H4><SPAN class="mw-headline" id="Gel_extraction">Gel extraction</SPAN></H4><P>1 % agarose, 70 V, 1 h
</P><TABLE border="1" cellpadding="2" cellspacing="0" align="right"><TBODY><TR><TH colspan="3">DNA concentrations
</TH></TR><TR><TD><B>Sample</B></TD><TD><B>Conc. [ng/μl]</B></TD><TD><B> A<SUB>260</SUB>/A<SUB>280</SUB></B></TD></TR><TR><TD>pSB1C3 X+A 1
</TD><TD>13.72
</TD><TD>2.04
</TD></TR><TR><TD>pSB1C3 X+A 2
</TD><TD>6.397
</TD><TD>2.31
</TD></TR><TR><TD>pSB1C3 N+S 1
</TD><TD>12.22
</TD><TD>2.09
</TD></TR><TR><TD>pSB1C3 N+S 2
</TD><TD>5.807
</TD><TD>2.46
</TD></TR></TBODY></TABLE><P>Loaded 30 μl of each sample (<B>X+A</B> and <B>N+S</B>). After electrophoresis, 260 mg samples containing digested vectors were excised. DNA purified using the E.Z.N.A. gel extraction kit, following the procedures provided. Each sample eluted twice, as follows:
</P><UL><LI><B>pSB1C3 X+A 1:</B> 40 μl
</LI><LI><B>pSB1C3 X+A 2:</B> 30 μl
</LI><LI><B>pSB1C3 N+S 1:</B> 40 μl
</LI><LI><B>pSB1C3 N+S 2:</B> 30 μl
</LI></UL><H3><SPAN class="mw-headline" id="Amp_plate_preparation">Amp plate preparation</SPAN></H3><P>Prepared 20X Amp 100.
</P><H3><SPAN class="mw-headline" id="Cloning_of_IgG_protease">Cloning of IgG protease</SPAN></H3><P>Re-transformed Top10 with the pSB1C3.IgGp ligation mixture from 3/8 (standard transformation protocol). Plated on Cm 25, incubation ON in 37 °C.
</P><H3><SPAN class="mw-headline" id="yCCS_clone_analysis">yCCS clone analysis</SPAN></H3><P>Ran a gel for Mimmi, where she/we investigated my yCCS clones from 15/7, prior to mutagenesis, comparing them to the RFP (BBa_J18932) gene.
</P><H4><SPAN class="mw-headline" id="Gel_verification_2">Gel verification</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Verification of yCCS clones by digestion. 1 % agarose, 100 V, 30 min.<B>Loading:</B> 1 kb λ, -, -, yCCS A, d.yCCS A, yCCS B, d.yCCS, RFP A, d.RFP A, RFP B, d.RFP B &quot;d.&quot; denotes digested sample.</DIV></DIV></DIV><P>1 % agarose, 100 V, 30 min.
</P><P><B>Expected bands:</B></P><UL><LI>yCCS (undig): 1076 bp
</LI><LI>yCCS (dig): 123 bp, 257 bp, 704 bp
</LI><LI>RFP (undig): 1036 bp
</LI><LI>RFP (dig): 1076 bp
</LI></UL><P><B>Results</B>
yCCS B was identified as RFP, which explains the failed yCCS sequencing results. However, yCCS A seems correct, so this clone/sample will be chosen for new site-directed mutagenesis (EcoRI site removal).
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Stockholm/9_August_2010">http://2010.igem.org/Team:Stockholm/9_August_2010</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Stockholm/9_August_2010" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Stockholm/9_August_2010" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Stockholm/9_August_2010&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Stockholm/9_August_2010&amp;oldid=157633" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>