<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Stockholm_Results_Assemblies"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Stockholm/Results/Assemblies">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Stockholm/Results/Assemblies&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Stockholm/Results/Assemblies&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Stockholm/Results/Assemblies&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Stockholm/Results/Assemblies" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Stockholm/Results/Assemblies</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV class="navbar"><CENTER><TABLE><TBODY><TR><TD><A href="/Team:Stockholm" title="Team:Stockholm">Home</A></TD><TD><A href="/Team:Stockholm/Project_Idea/Introduction" title="Team:Stockholm/Project Idea/Introduction">Project Idea</A></TD><TD><A href="/Team:Stockholm/Lab_work/Notebook" title="Team:Stockholm/Lab work/Notebook">Lab Work</A></TD><TD><A href="/Team:Stockholm/Results/BioBricks" title="Team:Stockholm/Results/BioBricks">Results</A></TD><TD><A href="/Team:Stockholm/Modelling/Introduction" title="Team:Stockholm/Modelling/Introduction">Modelling</A></TD><TD><A href="/Team:Stockholm/Team/Members" title="Team:Stockholm/Team/Members">Team</A></TD><TD>      Follow us on  and </TD><TD>       <A href="https://2010.igem.org/">2010.igem.org </A></TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="indented"><DIV class="navbar"><TABLE><TBODY><TR><TD>↳  <A href="/Team:Stockholm/Results/BioBricks" title="Team:Stockholm/Results/BioBricks">BioBricks</A></TD><TD><STRONG class="selflink">Assemblies</STRONG></TD><TD><A href="/Team:Stockholm/Medals_criteria" title="Team:Stockholm/Medals criteria">Medals criteria</A></TD></TR></TBODY></TABLE></DIV></DIV><TABLE width="800px"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Assemblies"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Assemblies</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#IdeS.C2.B7PtA-Z_fusion_protein"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">IdeS·PtA-Z fusion protein</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#CPP_fusions"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">CPP fusions</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Construction_of_a_sod1-yccs_operon"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Construction of a sod1-yccs operon</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Assemblies">Assemblies</SPAN></H2><H3><SPAN class="mw-headline" id="IdeS.C2.B7PtA-Z_fusion_protein">IdeS·PtA-Z fusion protein</SPAN></H3><P>One of our approaches for protecting affected Vitiligo skin from autoantibodies was to locally digest IgG immunoglobulins using the IgG degrading bacterial protease IdeS. In an attempt to increase the efficiency of IdeS in the skin we suggested that IdeS be fused to the Z-domain of Protein A, which possesses the ability of binding to the Fc (constant) region of immunoglobulins. By combining the features of these two proteins, we could possibly create a protein chimera which binds with a high(er) affinity to IgG, thus increasing the protease's access to antibodies, resulting in a higher antibody degradation than native IdeS.
</P><P>The PtA-Z·IdeS fusion protein was constructed as shown in the figure below:</P><H3><SPAN class="mw-headline" id="CPP_fusions">CPP fusions</SPAN></H3><P>As described in our project introduction, one of our aims was to target proteins to keratinocytes and melanocytes in human skin using cell-penetrating peptides. As we didn't know which of (or if) our CPPs would be most efficient in protein delivery, we set out to test all three in parallel.
</P>
At least two of our CPPs (LMWP and Tp10) have not previously been tested using recombinant technique, i.e. in protein fusions. Although these have been shown to successfully deliver protein cargo into cells and/or skin, these proteins have been chemically conjugated to the CPPs, <DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Cloning strategy for proteins fused to CPPs</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Cloning strategy for proteins fused to N-part CPPs</B></DIV></DIV></DIV>usually to amino acid side chains. We were therefore unsure as to whether membrane and skin penetration would be most effective when fusing the CPPs to our proteins' N- or C-termini; we also discussed that this may possibly vary for different proteins, as different cargo properties may interact and possibly disturb our CPPs. In relation to this, we also discussed the probability of the CPPs disturbing the enzymatic activity of our fused cargo proteins, and whether this risk would differ depending on to which side of the protein (N- or C-terminus) the CPP is fused.
<P>As time is very limited in the iGEM competition, and as we would not be able to test fusion to one side at the time, we decided to fuse the CPPs to both sides of our proteins, and draw our own conclusions by investigating both CPP and protein activity.
</P><P>Another factor we took into consideration was the additional amino acids added to parts adapted to the [<A href="http://partsregistry.org/Assembly_standard_25" class="external text" rel="nofollow">Freiburg assembly standard</A>]. Due to the small size of our CPPs (11-21 a.a.) we thought that even a very small variation to the amino acid sequence could affect their function. For our N-terminal protein fusions we therefore designed N-part versions of our CPPs, which prevents the addition of the two extra amino acids in the CPP N-terminal (Ala &amp; Gly) by removing the standard-specific NgoMIV restriction site.
</P><P>Finally, we also attached a 6x His tag to each one of our protein fusions to enable antibody detection and protein purification.
</P><P>The figures to the right illustrate the cloning strategy for our CPP·protein fusions, and insertion into the IPTG-inducible protein expression pEX (<A href="http://partsregistry.org/Part:BBa_K243033" class="external text" rel="nofollow">BBa_K243033</A>). Tables below show an overview of our many CPP·protein fusion combinations.
</P><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TH align="center" colspan="3">C-terminal CPP fusions
</TH></TR><TR><TH align="center">Purification tag
</TH><TH align="center">Protein
</TH><TH align="center">CPP
</TH></TR></TBODY></TABLE><TABLE border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TH align="center" colspan="3">N-terminal CPP fusions
</TH></TR><TR><TH align="center">N-part CPP
</TH><TH align="center">Protein
</TH><TH align="center">Purification tag
</TH></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Construction_of_a_sod1-yccs_operon">Construction of a <I>sod1</I>-<I>yccs</I> operon</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Cloning strategy for <I>sod-yccs</I> operon for co-expression in pEX.</B></DIV></DIV></DIV><P>For SOD1 to be fully active it needs to be co-expressed with its helper copper chaperone, which provides SOD1 with necessary copper ions prior to folding. We are using a yeast copper chaperone (yCCS) for this.
</P><P>Since we were (for several reasons) restricted to the pEX vector for protein expression, we wanted construct a <I>sod1</I>-<I>yccs</I> operon allowing us to co-express the two genes from the same P<SUB>tac</SUB> promoter, which comes pre-installed in pEX. Previous studies (e.g. by <A href="http://www.ncbi.nlm.nih.gov/pubmed/15358352" class="external text" rel="nofollow">Ahl <I>et al.</I> (2004)</A>) show that high copper content SOD1 is obtained with the protein is expressed in levels equal to yCCS. As we wanted to copy this behavior, transcription from the same promoter is an advantage. We then set out to look for a Shine-Dalgarno (SD) sequence with the same strength as the one found in pEX. Since no experimental comparison data with the pEX SD was  available we chose the most similar with respect to (1) the ribosome-binding nucleotides and (2) distance to the start codon; thus we selected <A href="http://partsregistry.org/Part:BBa_B0034" class="external text" rel="nofollow">BBa_B0034</A>.
</P><P>The operon was then assembled into pEX as illustrated. The same principle was used for constructing operons expressing CPP-SOD1 fusions.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Stockholm/Results/Assemblies">http://2010.igem.org/Team:Stockholm/Results/Assemblies</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Stockholm/Results/Assemblies" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Stockholm/Results/Assemblies" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Stockholm/Results/Assemblies&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Stockholm/Results/Assemblies&amp;oldid=190766" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>