<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Stockholm_Lab_work_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Stockholm/Lab_work/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Stockholm/Lab_work/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Stockholm/Lab_work/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Stockholm/Lab_work/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Stockholm/Lab_work/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Stockholm/Lab work/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV class="navbar"><CENTER><TABLE><TBODY><TR><TD><A href="/Team:Stockholm" title="Team:Stockholm">Home</A></TD><TD><A href="/Team:Stockholm/Project_Idea/Introduction" title="Team:Stockholm/Project Idea/Introduction">Project Idea</A></TD><TD><A href="/Team:Stockholm/Lab_work/Notebook" title="Team:Stockholm/Lab work/Notebook">Lab Work</A></TD><TD><A href="/Team:Stockholm/Results/BioBricks" title="Team:Stockholm/Results/BioBricks">Results</A></TD><TD><A href="/Team:Stockholm/Modelling/Introduction" title="Team:Stockholm/Modelling/Introduction">Modelling</A></TD><TD><A href="/Team:Stockholm/Team/Members" title="Team:Stockholm/Team/Members">Team</A></TD><TD>      Follow us on  and </TD><TD>       <A href="https://2010.igem.org/">2010.igem.org </A></TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="indented"><DIV class="navbar"><TABLE><TBODY><TR><TD>↳  <A href="/Team:Stockholm/Lab_work/Notebook" title="Team:Stockholm/Lab work/Notebook">Notebook</A></TD><TD><A href="/Team:Stockholm/Lab_work/Safety" title="Team:Stockholm/Lab work/Safety">Safety</A></TD><TD><STRONG class="selflink">Protocols</STRONG></TD></TR></TBODY></TABLE></DIV></DIV><DIV align="justify"><TABLE><TBODY><TR><TD width="10px"> </TD><TD width="590px"><H2><SPAN class="mw-headline" id="Protocols">Protocols</SPAN></H2><H3><SPAN class="mw-headline" id="Competent_cells_.28Nina_.26_Johan.29">Competent cells (Nina &amp; Johan)</SPAN></H3><P><I>From Morten Nørholm at the Department of Biochemistry &amp; Biophysics Stockholm University</I></P><OL><LI> Add 5 ml LB in two 50 ml falcon tubes. Add the top of a tip bacteria into the two 5 ml LB. Grow ON in shake incubator 37 degree C. 
</LI><LI> Subculture each 5 ml of starterculture into two 400 ml pre-warmed LB. Grow at 37 degree C until OD reaches 0.6. 
</LI><LI> Put cells on ice for 20 min. 
</LI><LI> Harvest cells at 4000 rpm for 20 min, 4 degree C. 
</LI><LI> Discard supernatant if it looks clear (or spinn longer if it is cloudy).
</LI><LI> Resuspend pellet carefully in 500 50 mM CaCl2 for a 1000 ml cell culture (1/2 the orifinal volume). 
</LI><LI> Put cells on ice 20 min.
</LI><LI> Repeat step 5.
</LI><LI> Resuspend cells in 16 ml CaCl2 + 15% Glycerol for a 800 ml starter culture (1:50 volume)
</LI><LI> Put metal blocks in -80 degree C.
</LI><LI> Snapfreeze 100 mikroL aliquots in ice-cold Epps (in pre-chilled blocks). Store in -80 degree C freezer.
</LI></OL><H3><SPAN class="mw-headline" id="Competent_cells_.28Andreas_.26_Mimmi.29">Competent cells (Andreas &amp; Mimmi)</SPAN></H3><P><I>Based and modified from the <A href="http://openwetware.org/wiki/TOP10_chemically_competent_cells" class="external text" rel="nofollow">Top10 protocol by the Knight lab</A></I></P><H5><SPAN class="mw-headline" id="Materials">Materials</SPAN></H5><BLOCKQUOTE><B>CCMB80 buffer</B><UL><LI>10 mM KOAc pH 7.0
</LI><LI>80 mM CaCl<SUB>2</SUB>.2H<SUB>2</SUB>O
</LI><LI>20 mM MnCl<SUB>2</SUB></LI><LI>10 mM MgCl<SUB>2</SUB>.6H<SUB>2</SUB>O
</LI><LI>10 % glycerol</LI></UL><H5><SPAN class="mw-headline" id="Procedures">Procedures</SPAN></H5><OL><LI> Set an ON starter culture by inoculating 5 ml LB and incubating ON in 37 °C with 250 rpm rotary shaking.
</LI><LI> Inoculate 250 ml new LB with 2 ml from the ON culture and grow in 30 °C, 250 rpm until an OD<SUB>600</SUB> of ≈0.3.
</LI><LI> Use a large, 1 l E-flask.
</LI><LI> Spin down cells at 3000 x <I>g</I> for 10 min in 4 °C.
</LI><LI> Remove supernatant and resuspend cells in 80 ml ice-cold <B>CCMB80 buffer</B>. Keep cells on ice for 20 min.
</LI><LI> Spin down cells again with same settings. Resuspend in 10 ml new CCMB80 buffer and keep on ice for 20 min.
</LI><LI> Aliquot 100 μl samples of competent cells into 1.5 ml vials and store in -80°C, or transform immediately.
</LI></OL><H3><SPAN class="mw-headline" id="Transformation_.28Nina_.26_Johan.29">Transformation (Nina &amp; Johan)</SPAN></H3><P><I>From NEB 5-alpha competent E.coli (High Efficiency) NEW ENGLAND BioLabs</I></P><OL><LI> Thaw a tube of NEB 5-alpha competent E.coli cells on ice for 10 min.
</LI><LI> Add 1-5 μl containing 1 pg-100 ng of plasmid DNA to the cell mixture. Carefully flick the tube 4-5 times to mix the cells and DNA. 
</LI><LI> Place the mixture on ice for 30 min.
</LI><LI> Heat shock at 42°C for 30 sec.
</LI><LI> Place on ice for 5 min.
</LI><LI> Pipette 950 μl of room temp. SOC into the mixture.
</LI><LI> Place at 37°C for 30-60 min, 250 rpm. 
</LI><LI> Warm selection plates to 37°C.
</LI><LI> Perform 10-fold serial dilutions in SOC (if necessary)
</LI><LI> Spread 50-100 μl of each sample onto a selection plate and incubate overnight at 37°C.
</LI></OL><H3><SPAN class="mw-headline" id="Transformation_.28Andreas_.26_Mimmi.29">Transformation (Andreas &amp; Mimmi)</SPAN></H3><OL><LI> Add 1 μl plasmid to 100 μl thawed, competent cells of choice. Hold cells on ice for 30 min.
</LI><LI> Heat-shock cells for 55 sec in 42 °C. Return to ice.
</LI><LI> Add 900 μl LB medium and grow cells in 37°C, 250 rpm for 1 hour.
</LI><LI> This allows cells to start expressing antibiotic resistance gene(s).
</LI><LI> Spin down cells at full speed (≈13.000 x <I>g</I>) for 15 sec.
</LI><LI> Remove 900 μl from the supernatant and gently resuspend pellet in remaining 100 μl.
</LI><LI> Plate 100 μl cells onto an LB agar plate with appropriate antibiotic(s).
</LI><LI> Incubate in 37 °C ON.
</LI></OL><H3><SPAN class="mw-headline" id="Quick_transformation_.28Andreas_.26_Mimmi.29">Quick transformation (Andreas &amp; Mimmi)</SPAN></H3><OL><LI> Add 1-3 μl plasmid to 100 μl thawed, competent Top10 cells. Hold cells on ice for 5 min.
</LI><LI> Heat-shock cells for 30 sec in 42 °C. Return to ice.
</LI><LI> Plate cells onto a pre-heated (37 °C) LB agar plate with appropriate antibiotic(s).
</LI><LI> Incubate in 37 °C ON.
</LI></OL><H3><SPAN class="mw-headline" id="Colony_PCR_verification_.28Andreas_.26_Mimmi.29">Colony PCR verification (Andreas &amp; Mimmi)</SPAN></H3><OL><LI> Pick four colonies and resuspend each in 10 μl LB.
</LI><LI> Let incubate in RT while preparing PCR tubes.
</LI><LI> Prepare each illustra PuReTaq Ready-To-Go PCR (GE Healthcare) tube as follows:
</LI><LI> 1.0 μl 10 μM forward primer
</LI><LI> 1.0 μl 10 μM reverse primer
</LI><LI> 22.5 μl dH<SUB>2</SUB>O
</LI><LI> 0.5 μl cell suspension (template DNA) to a <B>final volume of 25 μl</B>. Vortex to mix.
</LI><LI> Run PCR amplification.
</LI><LI> Denaturation: 95 °C - 10 min
</LI><LI> 30 cycles
</LI><OL><LI> Denaturation: 95 °C - 30 s
</LI><LI> Annealing: 55 °C - 30 s
</LI><LI> Elongation: 72 °C - <I>Calculate from expected sequence length, <B>≈1 min/kb</B></I></LI></OL><UL><LI> Elongation: 72 °C - 10 min
</LI></UL><LI> Analyze PCR products by agarose gel electrophoresis.
</LI></OL><H3><SPAN class="mw-headline" id="Site-directed_mutagenesis">Site-directed mutagenesis</SPAN></H3><P><I>Based on the QuikChange® Site-Directed Mutagenesis Kit</I></P><UL><LI> Design two complimentary oligonucleotides containing the desired mutation with <A href="https://www.genomics.agilent.com/CollectionSubpage.aspx?PageType=Tool&amp;SubPageType=ToolQCPD&amp;PageID=15" class="external free" rel="nofollow">https://www.genomics.agilent.com/CollectionSubpage.aspx?PageType=Tool&amp;SubPageType=ToolQCPD&amp;PageID=15</A></LI></UL><OL><LI> Prepare PCR (amounts for 100 µl reaction)
</LI><LI> 10 ng dsDNA template
</LI><LI> 125 ng primer #1
</LI><LI> 125 ng primer #2
</LI><LI> 84 µl ddH2O
</LI><LI> 2 µl 10 mM dNTPs
</LI><LI> 10 µl 10x Pfu reaction buffer
</LI><LI> 2 µl Pfu Turbo DNA polymerase 
</LI><LI> Run PCR
</LI><LI> 95 °C - 30 sec
</LI><LI> 22 cycles of
</LI><LI> 95 °C - 30 sec
</LI><LI> 55 °C - 1 min
</LI><LI> 68 °C - 1 min/kb plasmid
</LI></OL><LI> 4 °C - ∞
</LI><LI> Make sure the reaction is ≤ 37 °C before proceeding 
</LI><LI> Add 2 µl (for 100 µl reaction) of Dpn I restriction enzyme and mix thoroughly 
</LI><LI> Spin down for 1 min in microcentrifuge 
</LI><LI> Immediately incubate at 37 °C for 1 hour or more (more gives less background)
</LI><H3><SPAN class="mw-headline" id="SDS-PAGE_mixtures_.28Nina_.26_Johan.29">SDS-PAGE mixtures (Nina &amp;Johan)</SPAN></H3><P><I>From Robert Daniels at the Department of Biochemistry &amp; Biophysics Stockholm University</I></P><H3><SPAN class="mw-headline" id="Mini_prep_.28Nina_.26_Johan.29">Mini prep (Nina &amp; Johan)</SPAN></H3><P><I>Based on the QIAprep Spin Miniprep Kit Using a Microcentrifuge</I></P><P>1. Centrifuge sample in 4 °C, 10 min, 4000 rpm and resuspend pelleted bacterial cells with Buffer P1 (250 ul/5ml bacterial sample) and transfer to a microcentrifuge tube.
</P><P>2. Add 250 ul buffer P2, invert the tubes 4-6 times.
</P><P>3. Add 350 ul buffer N3, invert the tubes 4-6 times with powerful strokes.
</P><P>4. Centrifuge 10 min, 13000 rpm in a table-top microcentrifuge.
</P><P>5. Transfer the supernatant to the QIAprep spin column.
</P><P>6. Centrifuge for 1 min, 13000 rpm. Discard the flow-through. 
</P><P>7. Add 0.5 ml buffer PB and centrifuge for 1 min, 13000 rpm. Discard the flow-through. 
</P><P>8. Add 0.75 ml buffer PE and centrifuge for 1 min, 13000 rpm. 
</P><P>9. Discard the flow-through and centrifuge again the same way. 
</P><P>10. Place the column in a clean 1.5 ml microcentrifuge tube. Add 35 ul water to the center of the QIAprep spin column, let stand for 1 min, and centrifuge for 1 min, 13000 rpm.
</P><H3><SPAN class="mw-headline" id="Agarose_gel_clean_up_.28Nina_.26_Johan.29">Agarose gel clean up (Nina &amp; Johan)</SPAN></H3><P><I>Based on the QIAprep II Handbook</I></P><H3><SPAN class="mw-headline" id="IgG_protease_activity_assay">IgG protease activity assay</SPAN></H3><P>Based on an experiment by <A href="http://www.ncbi.nlm.nih.gov/pubmed/20441890" class="external text" rel="nofollow">Abuknesha, <I>et al</I> (2010)</A>.
</P><H4><SPAN class="mw-headline" id="Materials_2">Materials</SPAN></H4><UL><LI>10 ml culture tubes
</LI><LI>200 ml E-flasks
</LI><LI>10 ml Falcon tubes
</LI><LI>IPTG
</LI><LI>Protease dilution buffer
</LI><LI>Eppendorf tubes
</LI><LI><A href="http://en.wikipedia.org/wiki/McFarland_standards" class="external text" rel="nofollow">0.5 McFarland standard</A></LI><LI>0.05 ml 1 % BaCl<SUB>2</SUB></LI><LI>9.95 ml 1 %/0.18 M H<SUB>2</SUB>SO<SUB>4</SUB></LI></UL><LI><A href="http://openwetware.org/wiki/PBS" class="external text" rel="nofollow">Phosphate buffered saline (PBS), pH 7.4</A></LI><LI>Mouse IgG-Agarose (1 mg IgG/ml) (Sigma-Aldrich)
</LI><LI>Anti-mouse IgG peroxidase conjugate (Sigma Aldrich)
</LI><LI>SureBlue™ TMB Microwell Peroxidase Substrate
</LI><H4><SPAN class="mw-headline" id="Procedures_2">Procedures</SPAN></H4><H5><SPAN class="mw-headline" id="Day_0">Day 0</SPAN></H5><OL><LI>Set ON cultures in 5 ml LB + 100 μg/ml Amp, 37 °C, 225 rpm
</LI><LI>BL21 IgGp
</LI><LI>BL21 SOD (or other negative control)
</LI></OL><H5><SPAN class="mw-headline" id="Day_1">Day 1</SPAN></H5><OL><LI>Inoculate 200 μl of each ON culture (x2 for IgGp) into 20 ml fresh LB with 100 μg/ml Amp. Grow at 37 °C, 225 rpm until an OD<SUB>600</SUB> of ≈0.5.
</LI><LI>Induce protein expression by adding IPTG to a final concentration of 0.3 mM (e.g. 60 μl 0.1 M IPTG) to one of each culture type. Leave the second IgGp culture uninduced (negative control). Continue incubation for two hours.
</LI><LI>Transfer cultures to 10 ml Falcon tubes and spin down cells at 4,400 x g for 10 min.
</LI><LI>Resuspend enough pellet in 5 ml sterile PBS to yield a cell density of 1.5 x 10<SUP>8</SUP> CFU ml<SUP>-1</SUP> according to the McFarland standard 0.5. Make a 1000x dilution of each culture with PBS to 1.5 x 10<SUP>5</SUP>.
</LI><LI>Sonicate cells and spin down cell debris at 4,400 x g for 10 min. Recover and save supernatants for protease assay.
</LI><LI>Uninduced IgGp
</LI><LI>1.5 x 108 CFU ml-1
</LI><LI>1.5 x 105 CFU ml-1
</LI><LI>Induced IgGp
</LI><LI>1.5 x 108 CFU ml-1
</LI><LI>1.5 x 105 CFU ml-1
</LI><LI>Induced SOD
</LI><LI>1.5 x 108 CFU ml-1
</LI><LI>1.5 x 105 CFU ml-1
</LI><LI>Prepare IgG peroxidase dilutions by diluting the stock solution in PBS to 1:500 and 1:1000.
</LI><LI>Apply 30 μl IgG-Agarose suspension to 14 Eppendorf tubes, two for each protein extract dilution and two negative controls.
</LI><LI>Bind secondary antibodies by adding 100 μl anti-mouse IgG peroxidase, each dilution (1:500, 1:1000) to 7 tubes. Allow antibody binding by incubating at room temperature with tilt mixing for 30 min.
</LI><LI>Spin down IgG-Agarose at 13,000 x g for 3 min and wash 3 times with 200 μl PBS. Spin down again and remove supernatant.
</LI><LI>Add 100 μl of the protease extracts to the corresponding tubes (12 tubes). Add 100 μl PBS to the remaining two negative control tubes. Incubate at 37 °C ON (≈16 h) with tilt mixing.
</LI></OL><H5><SPAN class="mw-headline" id="Day_2">Day 2</SPAN></H5><OL><LI>Spin down the IgG-Agarose at 13,000 x g, 3 min. Transfer 80 μl of each supernatant (no pellet!) to new Eppendorf tubes. 
</LI><LI>Add 100 μl Sure Blue™ peroxidase substrate solution. Incubate in room temperature and tap gently until a blue color develops (≈30 min).
</LI><LI>Stop reaction by adding 100 μl 1 M HCl.
</LI><LI>Read color intensity/absorbance at 450 nm.
</LI></OL></BLOCKQUOTE></TD></TR></TBODY></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Stockholm/Lab_work/Protocols">http://2010.igem.org/Team:Stockholm/Lab_work/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Stockholm/Lab_work/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Stockholm/Lab_work/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Stockholm/Lab_work/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Stockholm/Lab_work/Protocols&amp;oldid=164973" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>