<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Stockholm_1_July_2010"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Stockholm/1_July_2010">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Stockholm/1_July_2010&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Stockholm/1_July_2010&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Stockholm/1_July_2010&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Stockholm/1_July_2010" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Stockholm/1 July 2010</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV class="navbar"><CENTER><TABLE><TBODY><TR><TD><A href="/Team:Stockholm" title="Team:Stockholm">Home</A></TD><TD><A href="/Team:Stockholm/Project_Idea/Introduction" title="Team:Stockholm/Project Idea/Introduction">Project Idea</A></TD><TD><A href="/Team:Stockholm/Lab_work/Notebook" title="Team:Stockholm/Lab work/Notebook">Lab Work</A></TD><TD><A href="/Team:Stockholm/Results/BioBricks" title="Team:Stockholm/Results/BioBricks">Results</A></TD><TD><A href="/Team:Stockholm/Modelling/Introduction" title="Team:Stockholm/Modelling/Introduction">Modelling</A></TD><TD><A href="/Team:Stockholm/Team/Members" title="Team:Stockholm/Team/Members">Team</A></TD><TD>      Follow us on  and </TD><TD>       <A href="https://2010.igem.org/">2010.igem.org </A></TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="indented"><DIV class="navbar"><TABLE><TBODY><TR><TD>↳  <A href="/Team:Stockholm/Lab_work/Notebook" title="Team:Stockholm/Lab work/Notebook">Notebook</A></TD><TD><A href="/Team:Stockholm/Lab_work/Safety" title="Team:Stockholm/Lab work/Safety">Safety</A></TD><TD><A href="/Team:Stockholm/Lab_work/Protocols" title="Team:Stockholm/Lab work/Protocols">Protocols</A></TD></TR></TBODY></TABLE></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Andreas"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Andreas</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Preparation_of_competent_BL21.28DE3.29_cells"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Preparation of competent BL21(DE3) cells</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Preparation_of_LB_agar_plates_with_antibiotics"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Preparation of LB agar plates with antibiotics</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Transformations"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Transformations</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Isolation_of_requested_parts"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Isolation of requested parts</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Maya"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Maya</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Hassan"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Hassan</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#What_I_did_for_today:"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">What I did for today:</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#TODO_for_tomorrow:"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">TODO for tomorrow:</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Johan"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Johan</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Digestion"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Andreas"> Andreas </SPAN></H1><H3><SPAN class="mw-headline" id="Preparation_of_competent_BL21.28DE3.29_cells"> Preparation of competent BL21(DE3) cells </SPAN></H3><P><I>Continued from 30/6</I></P><P>Competent cells were prepared according to <A href="https://2010.igem.org/Team:Stockholm/Protocols#Competent_cells_.28Andreas_.26_Mimmi.29" class="external text" rel="nofollow">protocol</A>.
</P><DL><DT>Modifications to protocol</DT><DD><UL><LI><I>Growth temperature changed to 37°C</I></LI><LI><I>Final OD(600) set to 0.57</I></LI></UL></DD></DL><H3><SPAN class="mw-headline" id="Preparation_of_LB_agar_plates_with_antibiotics"> Preparation of LB agar plates with antibiotics </SPAN></H3><UL><LI> 50 ug/ml Km (20 plates)
</LI><LI> 50 ug/ml Amp &amp; 25 ug/ml Cm (10 plates)
</LI><LI> 50 ug/ml Amp &amp; 25 ug/ml Km (10 plates)
</LI></UL><P><I>Plates were tested by plating with competent, non-transformed Top10 cells</I></P><H3><SPAN class="mw-headline" id="Transformations"> Transformations </SPAN></H3><P>Several transformations were performed. Top10 cells were transformed with pSB1x3 BioBrick vectors and grown on LB agar plates with appropriate antibiotic(s). BL21(DE3) cells were transformed with pEX.SOD and pEX.yCCS as well as with the pMA vector to test cell competence, and grown on Amp plates: 
</P><P><B> Top10 </B></P><UL><LI>pSB1A3.BBa_J04450
</LI><LI>pSB1C3.BBa_J04450
</LI><LI>pSB1K3.BBa_J04450
</LI><LI>pSB1AC3.BBa_J04450
</LI><LI>pSB1AK3.BBa_J04450
</LI></UL><P><I>The <B><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A></B> insert is an RFP-coding device, expressing RFP when transformed into cells and thereby resulting in red colonies</I>.
</P><P><B>BL21(DE3)</B></P><UL><LI>pEX.SOD
</LI><LI>pEX.yCCS
</LI><LI>pMA.BBa_K157011
</LI></UL><H3><SPAN class="mw-headline" id="Isolation_of_requested_parts"> Isolation of requested parts </SPAN></H3><P>Our three requested parts BBa_J18930 (mCerulean; CFP), BBa_J18931 (mCitrine; YFP) and BBa_J18932 (mCherry; RFP) arrived as agar stabs, carried in DH5alpha cells on pMA vectors. These were plated onto Amp plates according to <A href="http://partsregistry.org/Help:Requesting_Parts_2009#Using_agar_stabs" class="external text" rel="nofollow">Registry instructions</A>.
</P><H1><SPAN class="mw-headline" id="Maya">Maya</SPAN></H1><P>1 Microarrays - reliability
- How reliable is the protein regulation profile in the article &quot;Transcriptional profiling of melanocytes...&quot;
</P><P>2 How should one priorate different types of protein-protein interactions?
Focus on experimental (not predictive ) data and focus first of all at physical PPI second at co expression and co localisation. Skip the rest.
</P><P>To improve PPI reliability
</P><UL><LI> Find a good coverage from many primary databases. -How many and with?
</LI><LI> What experiments were used? If the same result is dereved from many different experiments then the PPI is more reliable.
</LI><LI> Look at 3D structures...?
</LI></UL><P>Compare with canonical pathways to get a better insight in the functional roles of the proteins in the network.
</P><H1><SPAN class="mw-headline" id="Hassan">Hassan</SPAN></H1><H2><SPAN class="mw-headline" id="What_I_did_for_today:"> What I did for today: </SPAN></H2><OL><LI><B>Prieto C, Risueño A, Fontanillo C, De Las Rivas J (2008) Human Gene Coexpression Landscape: Confident Network Derived from Tissue Transcriptomic Profiles. doi:10.1371/journal.pone.0003911</B></LI></OL><DL><DD><UL><LI><I>&quot;Despite the described interest, coexpression studies done at global “omic” scale are not focused in many cases on human samples [5], and, when they correspond to human, very often they include heterogeneous datasets, mixing “normal” samples with “disease altered” samples from patients suffering from some kind of pathological state. This is the case, for example, in several human gene expression large studies [2], [6]. The inclusion of many disease datasets (mainly from cancer) in such meta-analyses may introduce strong bias and produce a lot of biological noise in the results. In fact, it is well known that cancer cells have altered genomes. Therefore, these kind of studies cannot be used to clarify how a normal-healthy human cellular system works, and they cannot be used to draw a reliable map of the human gene coexpression landscape.&quot;</I></LI></UL></DD></DL><DL><DD><UL><LI><I>&quot;it has been shown that the inclusion of at least one non-parametric step based on ranks in the analyses of microarray data offers statistically more robust and more accurate estimation of expression values and expression correlations&quot;</I></LI></UL></DD></DL><DL><DD><UL><LI><I>&quot;It is difficult to achieve a proper gene coexpression study due to several obstacles that have to be taken in consideration: (i) the technical noise present in the microarrays at genomic scale (ii) the small number of samples used to define each gene expression signature (specially in comparison to the large number of genes); (iii) the strong heterogeneity of the data sets frequently studied, that include in many cases samples from pathological or altered states, which are not adequate samples to find “normal” gene expression behavior.&quot;</I> (continue from refs. here....)
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="TODO_for_tomorrow:"> TODO for tomorrow: </SPAN></H2><UL><LI>continue from article above.
</LI><LI>study reliability on microarray
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/12538238" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/12538238</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/17646307" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/17646307</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/19212713" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/19212713</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/17553856" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/17553856</A></LI><LI><A href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0003911" class="external free" rel="nofollow">http://www.plosone.org/article/info:doi/10.1371/journal.pone.0003911</A></LI><LI><A href="http://www.ploscompbiol.org/article/info:doi/10.1371/journal.pcbi.1000807?utm_source=feedburner&amp;utm_medium=feed&amp;utm_campaign=Feed:+ploscompbiol/NewArticles+(PLoS+Computational+Biology:+New+Articles)" class="external free" rel="nofollow">http://www.ploscompbiol.org/article/info:doi/10.1371/journal.pcbi.1000807?utm_source=feedburner&amp;utm_medium=feed&amp;utm_campaign=Feed:+ploscompbiol/NewArticles+(PLoS+Computational+Biology:+New+Articles)</A></LI></UL><H1><SPAN class="mw-headline" id="Johan">Johan</SPAN></H1><H2><SPAN class="mw-headline" id="Digestion">Digestion</SPAN></H2><P>pEX, pMA(130 ng/µl) and linker(in pMA vector)
</P><P>(Everything in µl)
</P><UL><LI>DNA vector: 
</LI></UL><P>pEX - 500 
</P><P>pMA - 20
</P><P>linker - 120			
</P><UL><LI>Enzyme (NgoMIV &amp; AgeI):
</LI></UL><P>pEX - 25+25
pMA - 1+1
linker - 6+6
</P><UL><LI> Buffer (Fermentas #1):
</LI></UL><P>pEX - 75
pMA - 3
linker - 18
</P><UL><LI> H2O:
</LI></UL><P>pEX: 125
pMA - 5
linker - 30
</P><UL><LI>3h digestion at 37 °C.
</LI></UL><UL><LI>CIAP alkaline phosphatase:
</LI></UL><P>pEX - 45
pMA - 1,5
linker - 10
</P><UL><LI> Incubated 60 min 37 °C
</LI></UL><UL><LI> Gel
</LI></UL><P>3 gels, each 8 lanes.
</P><P>1/10 loading dye (30+3, 170+17, 750+75), 45 µl loaded into gel of pEX. 20 µl of pMA and linker loaded as that gel had less agarose.
</P><P>All pEX could not be loaded, stored for tomorrow in freezer (with loading dye)
</P><P>All that loaded was cutted and stored in freezer for tomorrow.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Stockholm/1_July_2010">http://2010.igem.org/Team:Stockholm/1_July_2010</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Stockholm/1_July_2010" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Stockholm/1_July_2010" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Stockholm/1_July_2010&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Stockholm/1_July_2010&amp;oldid=191820" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>