"
Page
Discussion
View source
History
teams
Log in
 
Team:Stockholm/1 July 2010
From 2010.igem.org
Home
Project Idea
Lab Work
Results
Modelling
Team
      Follow us on
and
        2010.igem.org
↳   Notebook
Safety
Protocols
Contents
1 Andreas
1.1 Preparation of competent BL21(DE3) cells
1.2 Preparation of LB agar plates with antibiotics
1.3 Transformations
1.4 Isolation of requested parts
2 Maya
3 Hassan
3.1 What I did for today:
3.2 TODO for tomorrow:
4 Johan
4.1 Digestion
Andreas
Preparation of competent BL21(DE3) cells
Continued from 30/6
Competent cells were prepared according to protocol .
Modifications to protocol
Growth temperature changed to 37°C
Final OD(600) set to 0.57
Preparation of LB agar plates with antibiotics
50 ug/ml Km (20 plates)
50 ug/ml Amp & 25 ug/ml Cm (10 plates)
50 ug/ml Amp & 25 ug/ml Km (10 plates)
Plates were tested by plating with competent, non-transformed Top10 cells
Transformations
Several transformations were performed. Top10 cells were transformed with pSB1x3 BioBrick vectors and grown on LB agar plates with appropriate antibiotic(s). BL21(DE3) cells were transformed with pEX.SOD and pEX.yCCS as well as with the pMA vector to test cell competence, and grown on Amp plates:
Top10
pSB1A3.BBa_J04450
pSB1C3.BBa_J04450
pSB1K3.BBa_J04450
pSB1AC3.BBa_J04450
pSB1AK3.BBa_J04450
The BBa_J04450 insert is an RFP-coding device, expressing RFP when transformed into cells and thereby resulting in red colonies.
BL21(DE3)
pEX.SOD
pEX.yCCS
pMA.BBa_K157011
Isolation of requested parts
Our three requested parts BBa_J18930 (mCerulean; CFP), BBa_J18931 (mCitrine; YFP) and BBa_J18932 (mCherry; RFP) arrived as agar stabs, carried in DH5alpha cells on pMA vectors. These were plated onto Amp plates according to Registry instructions .
Maya
1 Microarrays - reliability - How reliable is the protein regulation profile in the article "Transcriptional profiling of melanocytes..."
2 How should one priorate different types of protein-protein interactions? Focus on experimental (not predictive ) data and focus first of all at physical PPI second at co expression and co localisation. Skip the rest.
To improve PPI reliability
Find a good coverage from many primary databases. -How many and with?
What experiments were used? If the same result is dereved from many different experiments then the PPI is more reliable.
Look at 3D structures...?
Compare with canonical pathways to get a better insight in the functional roles of the proteins in the network.
Hassan
What I did for today:
Prieto C, Risueño A, Fontanillo C, De Las Rivas J (2008) Human Gene Coexpression Landscape: Confident Network Derived from Tissue Transcriptomic Profiles. doi:10.1371/journal.pone.0003911
"Despite the described interest, coexpression studies done at global “omic” scale are not focused in many cases on human samples [5], and, when they correspond to human, very often they include heterogeneous datasets, mixing “normal” samples with “disease altered” samples from patients suffering from some kind of pathological state. This is the case, for example, in several human gene expression large studies [2], [6]. The inclusion of many disease datasets (mainly from cancer) in such meta-analyses may introduce strong bias and produce a lot of biological noise in the results. In fact, it is well known that cancer cells have altered genomes. Therefore, these kind of studies cannot be used to clarify how a normal-healthy human cellular system works, and they cannot be used to draw a reliable map of the human gene coexpression landscape."
"it has been shown that the inclusion of at least one non-parametric step based on ranks in the analyses of microarray data offers statistically more robust and more accurate estimation of expression values and expression correlations"
"It is difficult to achieve a proper gene coexpression study due to several obstacles that have to be taken in consideration: (i) the technical noise present in the microarrays at genomic scale (ii) the small number of samples used to define each gene expression signature (specially in comparison to the large number of genes); (iii) the strong heterogeneity of the data sets frequently studied, that include in many cases samples from pathological or altered states, which are not adequate samples to find “normal” gene expression behavior." (continue from refs. here....)
TODO for tomorrow:
continue from article above.
study reliability on microarray
http://www.ncbi.nlm.nih.gov/pubmed/12538238
http://www.ncbi.nlm.nih.gov/pubmed/17646307
http://www.ncbi.nlm.nih.gov/pubmed/19212713
http://www.ncbi.nlm.nih.gov/pubmed/17553856
http://www.plosone.org/article/info:doi/10.1371/journal.pone.0003911
http://www.ploscompbiol.org/article/info:doi/10.1371/journal.pcbi.1000807?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed:+ploscompbiol/NewArticles+(PLoS+Computational+Biology:+New+Articles)
Johan
Digestion
pEX, pMA(130 ng/µl) and linker(in pMA vector)
(Everything in µl)
DNA vector:
pEX - 500
pMA - 20
linker - 120
Enzyme (NgoMIV & AgeI):
pEX - 25+25 pMA - 1+1 linker - 6+6
Buffer (Fermentas #1):
pEX - 75 pMA - 3 linker - 18
H2O:
pEX: 125 pMA - 5 linker - 30
3h digestion at 37 °C.
CIAP alkaline phosphatase:
pEX - 45 pMA - 1,5 linker - 10
Incubated 60 min 37 °C
Gel
3 gels, each 8 lanes.
1/10 loading dye (30+3, 170+17, 750+75), 45 µl loaded into gel of pEX. 20 µl of pMA and linker loaded as that gel had less agarose.
All pEX could not be loaded, stored for tomorrow in freezer (with loading dye)
All that loaded was cutted and stored in freezer for tomorrow.
Retrieved from " http://2010.igem.org/Team:Stockholm/1_July_2010 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
