"
Page
Discussion
View source
History
teams
Log in
 
Team:Stockholm/24 August 2010
From 2010.igem.org
Home
Project Idea
Lab Work
Results
Modelling
Team
      Follow us on
and
        2010.igem.org
↳   Notebook
Safety
Protocols
Contents
1 Nina
1.1 Tranformation of protein A
1.2 Sequencing of tyrosinase
1.3 PCR on N_CPP cluster
2 Andreas
2.1 Glycerol stocks
2.2 Assembly of SOD/yCCS⋅His into pSB1K3
2.2.1 Transformation results
2.2.2 Colony PCR
2.2.3 Gel verification
2.2.3.1 Results
2.2.4 ON cultures
2.3 MITF BioBrick construction
2.3.1 New MITF primers
2.3.2 PCR amplification
2.3.3 Gel verification
2.3.3.1 Results
2.4 Transfer of MITF to pSB1C3
2.4.1 Digestion
2.4.2 Ligation
2.4.3 Quick transformation
2.5 Tranfer of RFP to pEX
2.5.1 Transformation results
2.5.2 Ligation
2.5.3 Quick transformation
3 Johan
Nina
Tranformation of protein A
I transformed 100 ul BL21 cells with both 1 and 3 ul of protein A inserted into the peX vector. This transformation is carried out in order to perform an IPTG induction on protein A in BL21 e.coli cells.
The transformation procedure is described in protocols. However, in step 1 I thawed the cells in 15 min instead of 10 min. In step 2 I added 1 ul of DNA sample to 100 ul BL21 cells and 3 ul of DNA sample to an additional 100 ul BL21 cells.
Sequencing of tyrosinase
Since the last sequencing of the two tyrosinase samples did not turn out well, I send two new tyrosinase samples for sequencing. This time I mixed the sample with primers complementary to the bank vector iGEM send the gene in. I therefore mixed with the primer VR. I choose VR instead of VF2 because I wanted the primer to bind closer to the site where I performed a site directed mutagenesis.
I prepared two tubes of tyrosinase:
15 ul plasmid from a mini-prep and 1.5 ul (10uM) primer VR of the vectors verification primers.
Colony #4: ASB0045 B01
Colony #6: ASB0045 B02
PCR on N_CPP cluster
We obtained our N_CPP in a lysophilized form in an eppendorf tube, to which I added 24 ul of dH2O, vortexed and spann down by centrifuging ~10 seconds.
I prepared a PCR mix with the N_CPP cluster as the DNA template. This PCR product will in following days become digested and ligated into both the pEX and shipping vector.
PCR mix:
Buffer Pfu buffer + MgSO4 10X 4.33 ul
dNTP 10 uM 1 ul
primer VF2 10 uM 3 ul
primer pgex 10 uM 3 ul
polymerase Pfu 1 ul
DNA template 1 ul
H2O 30 ul
I did a 1:5 dilution of primer pgex from 50 uM to 10 uM. 1 ul 50 uM primer and 4 ul dH2O.
PCR prgm:
Andreas
Glycerol stocks
From 23/8 ON cultures
1600 μl 100 % glycerol + 400 μl cell culture.
pEX vector 24/8
pSB1K3.SOD⋅His 1
pSB1K3.SOD⋅His 2
New cultures of the pSB1K3.yCCS⋅His will be set, as the current ones did not grow.
Assembly of SOD/yCCS⋅His into pSB1K3
Transformation results
From 23/8 transformations
pSB1K3.His⋅SOD: Good colony yield
pSB1K3.His⋅yCCS: Good colony yield
Colony PCR
Picked four colonies from each plate for colony PCR, as follows:
pSB1K3.His⋅SOD: HS1, HS2, HS3, HS4
pSB1K3.His⋅yCCS: Hy1, Hy2, Hy3, Hy4
Positive control: PC (pSB1K3.RFP plasmid)
Procedures according to colony PCR protocol.
Elongation time: 1:40
Gel verification
Gel verification of pSB1K3.His⋅SOD and pSB1K3.His⋅yCCS clones.
Loading volumes: 4 μl λ; 4 μl sample.
λ = O'GeneRuler 1 kb DNA ladder.
1 % agarose, 100 V, 1 h
Expected bands
pSB1K3.His⋅SOD: 815 bp
pSB1K3.His⋅yCCS: 1100 bp
Positive control: 1385 bp
Results
His⋅SOD: Well corresponding bands for samples HS1, HS3 and HS4, with HS1 possibly being slightly larger than the other two. This could also be an artifact from the gel.
His⋅yCCS: Well corresponding bands for samples Hy1 and Hy2.
ON cultures
Clones HS1, HS3, Hy1 and Hy2 were selected for verification by sequencing. ON cultures were set for plasmid and glycerol stock prep:
Plasmid prep
5 ml LB + 50 Km
37 °C, 250 rpm
Glycerol stocks
3 ml LB + 50 Km.
30 °C
MITF BioBrick construction
New MITF primers
MITF primers
>pRc/CMV_VF (24 bp) AATACGACTCACTATAGGGAGACC >pRc/CMV_VR (20 bp) CGTTACTAGTGGATCCGAGC >MITF_F_18aug (32 bp) CTGGAAATGCTAGAATATAATCACTATCAGGT >MITF_R_18aug (20 bp) ACAAGTGTGCTCCGTCTCTT >MITF_FB-F_18aug (64 bp) GAAGAATTCGCGGCCGCTTCTAGATGGCCGGC CTGGAAATGCTAGAATATAATCACTATCAGGT >MITF_FB-R_18aug (53 bp) GAACTGCAGCGGCCGCTACTAGTATTAACCGG TACAAGTGTGCTCCGTCTCTT
New primers for MITF and pRc/CMV vector arrived.
pRc/CMV verification primers
pRc/CMV_VF
pRc/CMV_VR
MITF amplification primers
MITF_F_18aug
MITF_R_18aug
MITF amplification primers with 5' prefix/suffix primer extensions
MITF_FB-F_18aug
MITF_FB-R_18aug
PCR amplification
Ran PCR reactions using the three primer pairs and pRc/CMV.MITF plasmid DNA:
MITF VF (pRC/CMV verification primers)
MITF FR (MITF amplification primers)
MITF FB (MITF BioBrick primers)
PCR tubes (25 μl total volume)
illustra Ready-to-Go PCR tubes
Forward primer: 1 μl
Reverse primer: 1 μl
pRc/CMV MITF plasmid DNA (95 ng/μl): 0.5 μl
dH2O: 22.5 μl
pRC/CMV verification
PCR program
MITF FB PCR program
1)
95 °C - 10:00
2)
95 °C - 00:30
x5
3)
55 °C - 00:30
4)
72 °C - 01:40
5)
95 °C - 00:30
x25
6)
72 °C - 00:30
7)
72 °C - 01:40
8)
72 °C - 10:00
MITF VF: Same as colony PCR settings above .
MITF VR: Same as colony PCR settings above .
MITF FB: See table to the right.
Gel verification
Gel verification of PCR amplified MITF with three different primer pairs
Loading volume: 4 μl λ; 4 μl sample
λ = O'GeneRuler 1 kb DNA ladder.
1 % agarose, 100 V, 45 min
Expected bands
MITF VF: ?
MITF FR: 1254 bp
MITF FB: 1313 bp
Results
Bands for both MITF FR and MITF FB correspond to expected fragment sizes, indicating successful MITF amplification! No band for MITF VF, but if the MITF FB band is correct, the pRc/CMV verification primers are not needed.
Transfer of MITF to pSB1C3
Digestion
Digested pSB1C3 vector (w/ RFP insert) as well as amplified MITF DNA with EcoRI and PstI.
[pSB1C3.RFP] = 201 ng/μl
Digestion tubes
[μl]
pSB1C3
MITF
10X FD buffer
3
3
DNA
10 (2 μg)
10
dH2O
15
10
FD EcoRI
1
1
FD PstI
1
1
Incubation: 37 °C, 10 min
Enzyme inactivation: 80 °C, 5 min
Ligation
Ligation tube
Vector DNA (pSB1C3)
1.5 μl
PCR DNA (MITF)
10 μl
dH2O
3.5 μl
5X Rapid Ligation buf.
4 μl
T4 DNA Ligase
1 μl
 
20 μl
Incubation: 22 °C, 10 min
Quick transformation
Procedures according to protocol
1.5 μl ligation mix
Cm 25 LB agar plates
Tranfer of RFP to pEX
Continued from 23/8
Transformation results
Very few and small red colonies, difficult to pick. Decided to make a new ligation with a higher insert:vector ratio.
Ligation
Ligation tube
Vector DNA (pEX, 33.3 ng/μl)
1 μl
Insert DNA (RFP, 66.6 ng/μl)
5 μl
dH2O
9 μl
5X Rapid Ligation buf.
4 μl
T4 DNA Ligase
1 μl
 
20 μl
Digested pEX and pSB1C3.RFP samples from 23/8 and 19/8, respectively.
Incubation: 22 °C, 10 min
Quick transformation
Procedures according to protocol
1.5 μl ligation mix. 20 min incubation on ice.
Amp 100 LB agar plates
Johan
Cut bFGF from original vector
10 µl DNA
2 µl 10x buffer
1 µl XbaI
1 µl AgeI
16 µl H2O
5 min 37 °C
45 min gel 100V 1% of bFGF
Gel cleanup of bFGF
Cut C-vector (that has igg)
14 µl H2O
2 µl 10x buffer
2 µl DNA
1 µ XbaI
1 µl AgeI
5 min 37 °C
Retrieved from " http://2010.igem.org/Team:Stockholm/24_August_2010 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
