Team:uOttawa/Description
Project
Parts
Modelling
Policy & Practice
Collaborations
Team
Attributions
Nav
Introduction
Logic Gate
Results
Yeast Biobrick Device
References
Introduction
Construction of synthetic guide RNA based gene regulatory network
The engineering of genetic networks often relies on transcriptional regulatory modules composed of a promoter and the transcription factor proteins that modulate the activity of that promoter. In this approach, the properties of a network are determined by the way modules are “wired” together, and one can enable new functionalities by changing the wiring of the network without introducing new parts.
Figure 1 An example of a gene network based on traditional transcription factors. Figure from genOway.
A key feature of many transcription factor proteins is that they bind DNA sequence elements with high specificity. This has some obvious advantages but also means that the regulation is “hard-wired”. With conventional transcriptional regulatory modules, it is not possible to redirect a transcription factor protein to a different promoter without changing the DNA sequence of that promoter.
The principle of regulated recruitment offers an interesting alternative to the conventional use of transcription factor proteins in genetic network engineering. In this principle, the transcription factor proteins don’t bind to DNA directly. Instead, they are recruited to a promoter by a second factor, an adaptor, which is capable of recognizing certain DNA sequences within the promoter. This makes possible to redirect a transcription factor protein to a different promoter by without changing the DNA sequence of that promoter.
Our project will explore if regulated recruitment using synthetic guide RNA (sgRNA) is a viable alternative to conventional transcriptional regulatory modules in genetic network engineering. These regulatory RNAs can modulate gene transcription by facilitating the recruitment of certain engineered transcription factors, and it should in principle be possible to create gene regulatory networks in which sgRNAs are used to regulate the expression of other sgRNAs molecules and the expression conventional transcription factor proteins. While sgRNAs networks have been built in the past, the ones we propose to create include feedback loops that enable more complex functionalities. To our knowledge, sgRNA feedback control has not been studied much in the past.
Figure 2 Schematic of sgRNA dcas9 based gene regulation. Figure from Igoshin et al.
Logic Gate
To explore the potential of sgRNAs in genetic network engineering, we will create and implement an sgRNA-based logic gate, the NOR gate. Logic gates are basic representations of digital circuits in terms of input and output flow under binary conditions. The behavior of a 2-input NOR gate (Figure 1) can be demonstrated with a truth table (Table 2). The output Z is true only when both input A and B are not true, which could be represented by the boolean expression:  (Z=(Not A or B)).
The suitability of sgRNAs in genetic network engineering depends critically on the availability of suitable promoters. The expression of sgRNAs is usually driven by so-called Pol III promoters, which function in ways that are very different from the Pol II promoters used to control the expression of protein-encoding genes. We simply don’t know if Pol III promoters will allow an sgDNA-based Genetic Toggle Switch to work. From mathematical modelling, we know that the two promoters in the network must have low basal activity or “leakiness”, high maximal expression rate, a switch-like response changing inputs, and low gene expression noise. Fortunately, a study published in May 2017 by Gander et al., suggests that it is possible to express functional sgRNA from PolII promoters in yeast and that sgRNAs and be used to control the expression of other sgRNA.
Figure 3 Design of dcas9 sgrna based logic gate.
Inspired by Gander’s study, “Digital logic circuits in yeast with CRISPR-dCas9 NOR gates”, we have decided to build a NOR gate whose output is controlled via the inducible production of gRNA-complexed dCas9. First, we will create four different strains of yeast, Saccharomyces cerevisiae, with each strain representing a stationary state in the truth Table 1. For instance, one strain contains a constitutive GFP reporter. This strain represents the first state in Table, in which neither gRNA A nor gRNA B are present, allowing for the production of GFP, which leads to an output of 1. Two strains contain only one type of gRNA along with dCas9. Since gRNA is complementary to a region in the promoter of GFP, it will complex with dCas9 to inhibit the production of GFP via steric hindrance, leading to an output of 0. The fourth strain contains both gRNAs as well as dCas9, with an expected output of 0. Finally, we will create another strain containing a NOR gate whose different output states can be modified. This can be achieved by controlling the production of both gRNAs using different inducible promoters. NOR gate parts of this project, notably dCas9, the GFP reporter, and the different gRNAs, have been adapted from the aforementioned study.
Results
We tested the potential of the sgrna dcas9 system as an alternative to traditional transcription factors to introduce gene regulation and repression. Our system consisted of a constitutive promoter for yeGFP, a dcas9-mxi1 plasmid, and two sgrnas (sgrna1 and sgrna2) transformed into a suitable yeast chassis. The sgrnas are designed to hybridize with specific regions of complementarity on the yeGFP promoter. All elements were successfully incorporated to complete the system.
Figure 4 Results measured by flow cytometry indicating expression of GFP for w303 yeast containing the dcas9 sgrna1 system shown in red. Compared to the negative control of the w303 yeast expressing baseline levels of yeGFP with dcas9 in absence of sgrna1.
The results shown in figure 4 compare the relative GFP expression in w303 S. Cerevisiae between a complete sgRNA dcas9 system including the sgrna1 (red) versus the negative control without sgRNA1 (green). The results show a decreased  population count of GFP colonies as measured by flow cytometry, indicating a successful repression by the sgRNA dcas9 system.
As a new team with minimal training and time constraints, there are further questions and data collection that we would have liked to explore and incorporate in our project. However, based on the results collected, we were able to demonstrate that sgRNA dcas9 system exhibit the potential as an alternative to traditional transcription factors in gene regulatory networks. In future works, we would like to explore the possibility of applying this mechanism to a toggle switch or increasingly complex gene networks.
Yeast Biobrick Device
Part of our project focused on improving on parts to support yeast genetic engineering. The main emphasis of iGEM is to create parts that can be expressed using E. coli as a chassis. This is reflected in the choice of pSB1C3 as the standard backbone, which must be used for all submitted parts. The disadvantage is that the backbone can only be used in E. coli.
The absence of available yeast backbone from the Registry and the relatively few parts available in the Distribution kit is a significant problem that we seek to address by improving on existing parts and by creating new ones. We will specifically focus on creating parts enabling the replication and propagation of genetic material in yeast.
To enable the selection and the propagation of foreign genes in yeast, it is necessary to create plasmids that are compatible with the yeast DNA replication machinery and capable of expressing a selectable marker. Replication and propagation can also be enabled by sequences that allows linearized DNA to be integrated at targeted sites in the yeast genome. Plasmids that allow integration into the genome are called targeting vectors. Several are used frequently in the yeast community, but none are in the standard BioBrick format.
Part of our iGEM project will be to create a series of devices that can convert any RFC 10 compatible E. coli plasmid into a vehicle for introducing and replicating foreign DNA in yeast S cerevisiae.
We will create the following basic parts in the standard BioBrick shipping vector:
- CEN/ARS centromere (completed and submitted)
- HIS3 selection cassette (completed and submitted)
- LEU2 selection cassette (completed and submitted)
- URA3 selection cassette (Pending)
- KanMX selection cassette (Pending)
- NatMX selection cassette (Pending)
- ADE2 genomic targeting cassette (Pending)
- ADE4 genomic targeting cassette (Pending)
- GAL4 genomic targeting cassette (Pending)
We also sought to address this issue by improving the characterization of available parts already existing in the registry. The shuttle vector, BBa_K530034 addresses the issue by enabling the replication and propagation of genetic material in both e coli and yeast.
The existence of this part presented a theoretical solution to bridging the gap between e coli biobricks and the limited compatibility in yeast. However, there is a lack of information regarding its practical usage and application. Therefore, iGEM uOttawa 2017 aimed to improve the characterization of Leu2 CEN/ARS yeast shuttle vector by examining its functionality and applicability to both e coli and yeast systems as a part of our project.
A central part of our project, dcas9 Mxi1, was examined in both e coli and yeast. Our plasmid incorporated the use of Leu2 CEN/ARS yeast shuttle vector that also contained an ampicillin selection (see in plasmid design).
We have proven that this yeast shuttle vector is compatible with ecoli strain DH5alpha as well as yeast strain W303 as shown by the growth on the ampicillin selection plate (ecoli right) and leucine selection plate (yeast left).
In conclusion, our improved informational characterization provides practical support for the use of yeast shuttle vectors as a solution to the long existing compatibility issue.
References
“Research Projects.” Oleg Igoshin Research Group, igoshin.rice.edu/research.html.
Gander, Miles W., et al. “Digital logic circuits in yeast with CRISPR-dCas9 NOR gates.” Nature Communications, vol. 8, 2017, p. 15459., doi:10.1038/ncomms15459.
Radzisheuskaya, Aliaksandra, et al. “Optimizing SgRNA Position Markedly Improves the Efficiency of CRISPR/dCas9-Mediated Transcriptional Repression.” Nucleic Acids Research, vol. 44, no. 18, 2016, doi:10.1093/nar/gkw583.
“Systems Biology and Complex Regulatory Networks.” Yeast, vol. 22, no. S1, 2005, doi:10.1002/yea.1272.
