<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Notebook_standardisation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Notebook/standardisation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Notebook/standardisation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Notebook/standardisation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Notebook/standardisation&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Notebook/standardisation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Notebook/standardisation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3></DIV><DIV id="menu"><UL id="navbar"><LI style="padding-left:15px"><A href="https://2013.igem.org/Team:Freiburg">Home</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Highlights">Highlights</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Introduction </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/induction">Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></LI></UL><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna">Targeting</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/method">uniBAss</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation">Standardization</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/protocols">Material and Methods</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/BioBricks">Our BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/favorite_parts">Favorite BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/improvement">Improvement of a Registry part</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/sharing">Sharing of our BioBricks </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/experts">Expert opinions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety/safety">Safety &amp; Security</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/politics">Politics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/society">Society</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/education">Education</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/art">Art</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/Facebook">Facebook / Media</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/News">News</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team2">Members</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Exolab">Exolab-Events</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Statistics">Statistics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/gallery">Photo Gallery</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Attributions">Our Partners</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/Coop">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/attributions"> Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/thanks">Thanks to</A></LI></DIV><DIV id="sidebar"><DIV id="sidebar_content"><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna"> Targeting </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector"> Effectors </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling"> Modeling </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/method"> uniBAss </A></P><P class="first_order"><A class="active" href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation"> Standardization </A></P><P class="third_order"><A href="#june"> June </A></P><P class="third_order"><A href="#july"> July </A></P><P class="third_order"><A href="#august"> August </A></P><P class="third_order"><A href="#september"> September </A></P><P class="first_order"><A id="link" href="https://2013.igem.org/Team:Freiburg/protocols"> Material and Methods </A></P></DIV></DIV><DIV id="main_contant_note"><DIV id="h1">
Assembly of Biobricks Notebook</DIV><H2>Theoretical cloning</H2>
Cas9 primers were design to exchange several bases - all as silent mutations – in order to remove all restriction recognition sides of the iGEM enzymes EcoI, NotI, XbaI, NgoMIV, SpeI, PstI, AgeI. Then the standardized Cas9 was brought into pSB1C3 (RFC25) and was used for fusion with with other parts.


<H2>Practical work</H2><DIV id="june"><P id="h2">
June
</P></DIV><DIV id="tag"><H3>17.06.2013</H3><P>PCRs for Gibson assembly 1:</P><P>PCR1: oIG22+oIG28 PCR2: oIG29+oIG26 PCR3: oIG27+oIG24 PCR4: oIG25+oIG23/</P><P>
Template: pX334

Annealing temperature dependent of fragment size:
<UL><LI>PCR1: 1130 bp </LI><LI>PCR2: 1369 bp </LI><LI>PCR3: 314 bp </LI><LI>PCR4: 1409 bp
</LI></UL></P><H4>PCR mixture</H4><TABLE><TBODY><TR><TH>per tube (50 µl)</TH></TR><TR><TD>10 µl </TD><TD> Q5 reaction buffer </TD></TR><TR><TD>10 µl </TD><TD> GC enhancer </TD></TR><TR><TD>4 µl</TD><TD> dNTPs </TD></TR><TR><TD>0.5 µl </TD><TD> Q5 Polymerase </TD></TR><TR><TD>22.5 µl</TD><TD>add with water up to 48 µl</TD></TR></TBODY></TABLE>

per reaction add 1 µl fw primer, rev primer, template each. 
<H4>PCR program</H4><TABLE><TBODY><TR><TH>Step</TH><TH>temperature</TH><TH> time</TH></TR><TR><TD><B>(1)</B></TD><TD>98°C </TD><TD> 30 s </TD></TR><TR><TD><B>(2)</B></TD><TD>98°C </TD><TD> 10 s </TD></TR><TR><TD>63°C </TD><TD> 20 s </TD></TR><TR><TD>72°C </TD><TD> 20-25 s/ kb </TD></TR><TR><TD><B>(3)</B></TD><TD>72°C </TD><TD>2 min </TD></TR><TR><TD><B>(4)</B></TD><TD>4°C  </TD><TD>hold </TD></TR></TBODY></TABLE><P>repeat step 2 25 cycles</P><H4> Digest of pSB1C3 from BBa_K323005 (RFC10) with AgeI und NgoMIV (buffer 4, 2 h, 37 °C )</H4><H3>06.06.2013</H3><H4>Gel run of PCR 1-4 and digest pSB1C3</H4>
30 min at 120 V, afterwards 1 h at 170 V, poststaining for 45 min (15 µl GelRed auf 50 ml TAE) 
<DIV id="floathleft"><P>PCR bands in wished size were cutted and frozen over night.
Digest was not functional (only one band), new pSB1C3 is searched with RFC25 (–&gt; Team Freiburg 2010 plasmid number 51).</P></DIV><H3>07.06.2013</H3><H4>Repeating of PCR (for Gibson I)</H4><H4>Gel run of PCR</H4>
1 % agarose, 120 V, 60 min; Poststaining
<DIV id="floathleft"><P> cutting bands out</P></DIV><H4> Gelextractionof PCRs from 6.6.+7.6. </H4>
Weigh gel sclice,  put liquidated duplicates onto one column, implementation via standard protocol.
<UL>
Concentrations:
<LI>PCR1: 165,5 ng/µl </LI><LI>PCR2: 146,5 ng/µl</LI><LI>PCR3: 56,3 ng/µl</LI><LI>PCR4: 160,5 ng/µl</LI></UL><H3> 10.06.2013 </H3><H4> Digest of pSB1C3 with new insert </H4>
No. 51 (BBa_K404152 pSB1C3_VP1up), insert size 411 bp, concentration 83 ng/µl; see 07.06.13,implementation see 06.06.
<H4> Gelextraction of digested BBa_K404152 pSB1C3_VP1up </H4>
83,5 ng/µl
<H4> Gibson I</H4>
cutted pSB1C3 + PCR1-4 analog to standard protocol
<H4>Retransformation of BBa_K404152 pSB1C3_VP1u and transformation of Gisbon from 10.06. </H4>
Use 4 µl Gibson mix for transformation and for retransformation 2 µl of 1:6 diluted BBa_K404152 pSB1C3_VP1up (implementation analog to standard protocol), plate on chloramphenicol dishes
<H3> 11.06.2013 </H3>
No colonies on Gibson plates, repeat of Gibson analog to 10.6., 
colonies of retrafo were minipreped after standard protocol
<H3> 12.06.2013 </H3>
Colonies of repeated Gibson were picked and plated out.
<H3> 13.06.2013 </H3><UL>
Minipreparation of plated clones from Gibson I of 11.06. 
<LI> clone 1: 137 ng/µl</LI><LI> clone 2: 158 ng/µl</LI><LI> clone 3: 112 ng/µl</LI><LI> clone 4: 104 ng/µl</LI><LI> clone 5: 141 ng/µl</LI><LI> clone 6: 38 ng/µl</LI><LI> clone 7: 62 ng/µl</LI></UL><H4> Test digest of miniprep from 13.06. </H4>
Digest with NgoMIV, 10 µl total volume, clones 1-5 2 µl each, clones 6+7 3 µleach, wished bands: 2500 bp, 2200 bp, 1400 bp, 80 bp
 <H4> Gel (1%) of test digest from 13.06. </H4>
no Poststaining (1 µl GelRed added into fluid agarose gel), 180 V, 1 h

Digest did not work, only one band is visible--&gt; religated bb? (AgeI and NgoMIV are compatible), assumption: Gibson did not work because of  spontaneous religation of digested backbone–&gt; no Gibson reaction possible
<H4> New digest of Ba_K404152 pSB1C3_VP1up with phosphatase treatment </H4>
3 µg vector digested, after 105 minutes: addition of antarctic phosphatase reaction buffer (1-fold, conzentration is 10-fold)+ 1 µl Phosphatase +
 water up tp 30 µl 
<UL><LI>15 min at 37 °C</LI><LI>5 min at 60 °C</LI></UL><H4> Test digest II of clne 7 of Gibson I (second trial) from 13.06. </H4>
digest with XbaI und AgeI in buffer 4, 1 h at 37 °C, 
wished sizes: 4119 bp (Cas9), 2076 bp (backbone), 287 bp 
<H4>Gel (1%) of digest from 13.06. andof test digest II from 13.06.</H4>
From left to right: Marker diluted 3 µl, Marker undiluted  3 µl, test digest, digest, Marker diluted 1 µl, Marker undiluted 1 µl;

test digest was not successful (only one band), bigger band of digest was cutted out and was being frozen
 <H3> 14.06.2013 </H3><H4> Gel extraction of digested and phosphatase treated BBa_K404152 pSB1C3_VP1u from 13.06. </H4>
analogous to 09.06., 11 ng/µl
<H4> Repeat of Gibson I with new digested and phosphatase treated BBa_K404152 pSB1C3_VP1u from 13.06. </H4>
analogous to 11.06., afterwards 4 µl Gibson Mix were transformated

        <H3> 15.06.2013 </H3>
        Clones were picked from Gibson plates from 14.06.and plated on Chloramphenicol plates
        <H3> 16.06.2013 </H3>
        Minipreparations of picked colonies from 15.06.
<H3> 17.08.2013 </H3><H4> Test digest of minipreps from 16.06.</H4>
Digest with XbaI und AgeI in Cut smart buffer, 1,5 h at 37°C

Sequencing of clone 1.1 and 1.3 with oIG0002 (forward Primer of Cas9 till base 1085)
  <H3> 17.06.2013 </H3><H4> Testdigest of Miniprep from 16.6. </H4>
Digested with XbaI and AgeI in Cut smart buffer, 1.5 h at 37°C

Sequencing of clone 1.1 and 1.3 with oIG0002 (forward Primer for Cas9 after 1085 bases)
 <H3> 24.06.2013 </H3><H4>Digestion of pSB1C3_VP1_up (51)</H4>
Digestion was performed with AgeI-HF and NgoMIV in buffer 4 (1,5h 37°C) followed by CIP treatment (as on the 13th of June).

0,7% agarose gel (normal marker(1µl), digest, empty, pSB1C3 control(uncut), ready to use gene ruler marker (1µl)
<P> Lower half of the band at 2000bp was cut out and purified with Roche PCR purification kit: 10ng/µl</P><DIV id="july"><P id="h2">
July
</P></DIV><H3> 02.07.2013 </H3><H4>Gibson with digested pSB1C3 from 24.06. </H4>
    Concentration of all PCRs and dilutions measured again (see below), volume for Gibson reaction calculated with excel sheet in dropbox. 
    DNA directly added to Gibson Master Mix, 1h at 50°C, 4µl in trafo with competent cells (21.6.), inoculated in 500µl LB, plated on chloramphenicol plates. Negative control without PCR4.
    <H4>Retrafo of pSB1C3_VP1up (51)</H4><H3> 03.07.2013 </H3>
    Some colonies on Gibson plate, none on negative control. Six colonies inoculated for miniprepping.
Retrafo of pSB1C3_VP1up (51): three colonies for minis inoculated.
<H4>Mini preps of pSB1C3_VP1up (51)</H4><H4>ini preps and test digest of Gibson from 2.7</H4>
Minis in freezer box (Standardisierung=St1-6).
 Test digest with AgeI-HF and XbaI in Cut-Smart buffer.
Expected fragments: 4119bp (Cas9), 2076bp (pSB1C3)
 <H3>19.07.2013 </H3><H4>PCR of pSB1C3 as well as all single fragments 1-7 for Cas9 standardization</H4>
PCR was made analog to 5.6.
For pSB1C3 60 °C were used as annealing temperature and pIG0037 and pIG0038 as primers. Two probes have been made: template amount of 10 pg and 1 ng (I/II).
<UL>
For Cas9 fragments:
<LI> PCR1: 839 bp, Primer oIG0022, oIG0036 </LI><LI>PCR2: 330 bp, Primer oIG0035, oIG0028  </LI><LI> PCR3: 1121 bp, Primer oIG0029, oIG0034 </LI><LI>  PCR4: 268 bp, Primer oIG0033,oIG0026</LI><LI> PCR5: 314 bp, Primer oIG0027, oIG0024 </LI><LI> PCR6: 945 bp, Primer oIG0025, oIG0032 </LI><LI>  PCR7: 484 bp, Primer oIG0031, oIG0023</LI></UL>

For pSB1C3: 2086 bp.
PCR was frozen afterwards (o.n.)
 <H3> 20.07.2013 </H3><UL>
 Loading scheme for 1 % Agarose gels:
<LI> 
Gel 1 (big): M I I II II 3 </LI><LI>Gel 2 (big): M 1 2 4 5 </LI><LI> Gel 3 (small): M 6 7 </LI></UL>
Gelpicture could not be saved due to error of the machine. Size was not as excpected unless from amplification of the backbone pSB1C3. Gelextraction followed. Afterwards a test gel was made with 2 µl of the extracted band. Size was as excpected.
<H4>Fusion PCR of the PCR products 1-7 </H4><UL><LI> Probe 1: PCR 1-7</LI><LI>Probe 2: PCR 1-3 </LI><LI> Probe 3: PCR 4-7</LI></UL><B>PCR I:</B> Annealing temperature was performed after the lowest overlapping temperature of the primer (1+3: 50°C, 2: 54°C)
Elongation time was performed after the longest fragment size of the several PCR fragments to be fused (1+2: 28 sec, 3: 24 sec; run all with 28 sec).
<B>PCR II:</B> primer added in each of the probes (oIG0022+oIG0023 for 1, oIG0022+oIG0034 for 2, oIG0033+oIG0023 for 3)
Elongation time was performed after the product size (1: 1 min 44 sec , 2: 57 sec, 3: 50 sec, 2+3 have been run on 57 sec)
<UL>
Estimated product size:
<LI>4162 bp for 1</LI><LI>2290 bp for 2 </LI><LI>2011 bp for 3</LI></UL><H4>Repetition of PCR of amplification of pSB1C3</H4>
1 ng has been used as template volume, template was pSB1C3 (RFC10–&gt; Probe I) and pSB1C3 (RFC25–&gt; Probe *).
PCRs were frozen o.n.
 <H3> 21.07.2013 </H3><H4> Agarose Gel from PCR of 20th of July </H4>
0.7 % Agarose Gel has been run
<H4> Gelextraction from PCRs of 20th of July </H4><UL>
Concentrations were measured with Nanodrop: 
<LI>Fusion 1: 26 ng /µl</LI><LI>Fusion 2: 40 ng /µl </LI><LI>Fusion 3: 62 ng /µl</LI><LI>PCR*: 8 ng /µl.</LI></UL><H3> 22.07.2013 </H3><H4> Fusion PCR of Fusion-PCR 2+3 of the 20th of July</H4><P> PCR 1: Elongationtime 1 min 2 sec, 1 µl of each PCR-Product, Annealing temperature: 56 °C. </P><P> PCR 2: Elongationtime 1 min 44 sec, Annealing temperature: 63 °C, primer added: oIG0022+oIG0023.</P><H4> FPCR of amplification PCR of the 20th of July (of pSB1C3_RFC 25 backbone)</H4>
2 µl template, 52 sec elongation, 60 °C annealing, primer oIG0037+38
 <H4> Gibson assembly of PCR products from the 20th of July (of pSB1C3_RFC 25 backbone)</H4><UL><LI>Gibson 1 (no. 1): Fusion PCR product 1 (from PCR 1-7 from 19.7.)+ PCR of pSB1C3_RFC25 bb (*)</LI><LI>Gibson 2 (no. 2/3: Fusion PCR Product 2 (from PCR 1-3 from 19.07.)+Fusion PCR Product 3 (from PCR 3-7 from 19.07.)+ PCR of pSB1C3_RFC25 bb (*)</LI><LI>Gibson 3 (no. -): Fusion PCR Product 3 (from PCR 3-7 from 19.07.)+ PCR of pSB1C3_RFC25 bb (*)</LI></UL><UL>
calculated amounts of DNA mastermix (with formula of Hanna):
<LI>Gibson 1: 1 µl fusion 1-7 (no 1), 3.1 µl bb, 0.9 water</LI><LI>Gibson 2: 0.6 µl fusion 1-3 (no 2), 0.4 µl fusion 3-7 (no 2), 3.1 µl bb, 0.9 water</LI><LI>Gibson 3: 0.4 µl fusion 3-7 (no 2), 3.1 µl bb, 1.5 water</LI></UL><H4> Trafo of Gibson assembly from the 22th of July </H4>
put TOP10 on Chloramphenicol o.n.

 <H4> Gelrun of the PCRs from the 22th of July </H4>
0.7 % Agarose gel, loading scheme: M, *(3 µl),*(all), *(all), *(3 µl), PCR 1(3 µl), PCR 1(all), empty, Fusion 2/3 (3 µl), Fusion 2/3 (all)
 <P>Expected size: * (2086 bp), PCR1 (4162 bp), Fusion2/3 (4162 bp)</P>
fragments were cutted out and frozen o.n. 
  <H3> 23.07.2013 </H3><H4> PCR over Fusion PCR product 1-7 was performed</H4>
        primer oIG0022+23, elongation time 1 min 44, product was frozen o.n.
          <H3> 24.07.2013 </H3><H4> Gelrun of PCR from 23.7. with 0.7 % Agarose and 120 V</H4>        
From left two right: Marker 2log from NEB, 5 µl PCRI, 56 µl PCRI,56 µl PCRII, 5 µl PCRI, 1 µl.
<P>Ciped pSB1C3_RFC25 bands were cutted out and gelextracted over one column (17 ng/µl, 30 µl total volume.</P><H4>Digest of Fusion PCR1-7 and pSB1C3_Vp1Up_RFC25 with EcoRIHF and SpeIHF</H4>
For both 40 µl as total volume:
<P> pSB1C3: 23 µl DNA [2,5 µg, conc.:111 ng/µl], Eco and Spe 1 µl each , buffer cut smart 4 µl, up to 40 µl with water (11 ml)</P><P> Fusion 1-7 [digest all of the gelectracted product]: 25 µl [425 ng, 17 ng/µl], Eco and Spe 0.5 µl each, , buffer cut smart 4 µl, 10 µl water up to 40 µl</P>
2 h, 37 °C 
<H4>PCR of Fusion PCR1-7 and PCR over non-nickase Cas9 (plasmid 2004)</H4>
PCR over non-nickase PCR with oIG27+24, elongation time 8 sec [plasmid 1 µl of 1:10 dilution, because of high concentration of plasmid–&gt; midiprep)
<P>PCR of Fusion PCR1-7 see 23.07. </P><H4>Gelrun of digest from 24.07.</H4>
excpected size: 2051 bp for pSB1C3 [insert left: 454 bp], 4144 bp for Fusion 1-7

We run gel very long for cutting out digested backbone using lower band for later ligation.

Because we cutted out Fusion 1-7 digested without being able to diverge the cutted and uncutted PCR product, 1:1 ratio is used for ligation

<H4>Gelextraction of the two digested fragments from 24.07.(pSB1C3_Vp1up and Fusion1-7)
pSB1C3_Vp1up: 28 ng/µl
Fusion1-7: 7 ng/µl

</H4><H4>Ligation of the two digested fragments from 24.07.(pSB1C3_Vp1up and Fusion1-7)</H4>
We have been used Cas9 as bb and pSB1C3 as insert with an 1:1 ratio.
<P>20 µl total volume, 15 min, 22 °C, 1 µl T4 ligase, 2 µl T4 ligase buffer, 30ng from Cas9 [4 µl], 15 ng of pSB1C3_RFC25 [0.5 µl]</P><H4>Transformation of ligation from 24.07.(pSB1C3_Vp1up and Fusion 1-7)</H4>
2.5 µl of ligation result to 25 µl TOP10 

<H4>Gelrun of PCR from 24.07. </H4>
excpected size: 4162 bp for PCR1-7, 314 bp for PCR over non-nickase Cas9 (plasmid 2004)

PCR over fusion PCR1-7 did not work, is repeated 
<P>right picture: marker log2, PCR over p2004 with oIG0027+24, fragments were cut out and gelextracted (32,1 ng/µl) </P><H3>  25.07.2013 </H3><H4>Result of Ligation from 23.07 </H4>
One clone was visible. Clone was picked and given into 3 ml LB over night and icubated with 37° C.

 <H4>PCR of Fusion PCR 1-7 on 23.07.13</H4>
Elongation time 1 min 44 (Rest like above)
<H4>Fusion PCR 1-7 with PCR Fragment 5 without Nickase (PCR Produkt from 23.07.13</H4>
Fusion PCR approach like above, except for amount of PCR Products: 3 µl of PCR Product 4 and 1 µl of the Rest. PCR Product 4 is now empty. Repeat PCR 4
<DIV id="floathleft"><H3>26.07</H3><H4>Result of Sequencing of Fusion PCR 1-7</H4>
Results were positive, Primers were able to bind at expected basepairs of Cas9. → Fusion PCR works!
<H4>Minipreps of Transformation from 24.07</H4>
Concentration: 76 ng/µl
<H4>Test digest of Transformation from 24.07</H4>
Digested with XbaI and AgeI HF
<H3>29.07</H3><H4> Digest of pSB1C3 RFC 10 and Fusion PCR Product 1-7 * (Double Mutant, without Nickase)</H4>
Fusion PCR was digested with EcoRI and SpeI; pSB1C3 was cutted with XbaI and AgeI HF (with phosphatase treatment)
<H4>Ligation of Fusion PCR Product for Cas9 without Nickase and pSB1c3 RFC10</H4>
BB to Insert ratio was set 1:1, ligation was incubated 15 minutes at room temperature
<H3>30.07</H3>
Two colonies were visible. They were plated out for minipreps.

<H3>31.07</H3><UL>
Minipreps
<LI>Colony 1: 344,9 ng/µl</LI><LI>Colony 2: 254,0 ng/µl</LI></UL><H4> Test digest of Transformation from 30.07</H4>
cutted with XbaI and AgI HF
<DIV id="august"><P id="h2">
August
</P></DIV><H3> 01.08</H3><H4>Sequencing result</H4>
Sequencing result of Cas9 in pSB1C3 looks good. The nickase mutation is in inserted. Cas9 is now in the shipping backbone. More primers were send to GATC in order to sequence the whole Cas9 (oIG0001, oIG0002, oIG0003, oIG0004, oIG0005) (See next day).

<H3> 02.08</H3><H4>Sequencing results</H4>
Cas9 is completely mutated. All ten iGEM restrictions sites were mutated (Primers oIG0001, oIG0002, oIG0003, oIG0004, oIG0005).


        <H3> 16.08.2013 </H3><H4> Transformation of CMV-Promotor(BBa_K747096) and AAV2-NLS (BBa_K404153) </H4>
        CMV-Promotor plasmid was extracted from the distribution-kit of 2013 (plate 1, 21P). E. coli were plated out on Chloramphenicol containing plates.
        <H4> Oligo annealing of oIG0047 and oIG0048 (HA-tag) </H4><TABLE class="tabelle"><TBODY><TR><TD> Oligo 1 (100 µM) </TD><TD> 50 µl </TD></TR><TR><TD> Oligo 1 (100 µM) </TD><TD> 50 µl </TD></TR><TR><TD> water </TD><TD> 150 µl </TD></TR></TBODY></TABLE> 

Incubate at 98 C for 4 minutes and cool down in turned off heating block for 3 hours.
        <H4> PCR of BGH terminator </H4><P>BGH was amplified from pGR2 with the primers oIG0049 + oIG0050.</P>
        Annealing temperature was 62 °C, elongation time was set to 7 seconds (wished product size: 242 bp). 
        <H4> Digest of pIG001 with NgoMIV and SpeI</H4><P> 2.5 µg of plasmid were digested with NgoMIV and SpeI HF in cut smart buffer for 2 hours at 37 °C.</P><P> Expected band size 2100 bp (bb) and 4121 (Cas9 Insert) </P><H4> Gelrun of digest and PCR from 16.08.</H4>
        PCR and digest were run on a 1% agarose gel and disered band sizes were cutted out.
        <H4> Gelrextraction from digest and PCR bands from 16.08.</H4><UL> 
        Concentrations:
        <LI> BGH: 47 ng/µl </LI><LI> pIG001 cutted with NgoMIV and SpeI: 18 ng/µl</LI></UL><H4> Ligation of pIG001 cutted with NgoMIV and SpeI and HA-oligo</H4><P>HA-oligo was diluted 1:100 and concentration measured with nanodrop (7 ng/ µl).</P><P>50 ng of bb was used (2.8 µl of cutted pIG001)and an 3 times  excess of Insert (17.3 ng, 2.5 µl of HA-oligo)</P><P>[50ng/bp vector*3=ng insert/bp insert].</P><P>Ligation was incubated 40 minutes at 22 °C and afterwards transformed. </P><H4> Gibson assembly of pIG001 cutted with NgoMIV and SpeI and BGH-Terminator</H4><P>1:4 ratio of bb:insert was used. </P><P>For DNA mix 1.4 µl of cutted pIG001, 0.25 µl of BGH terminator and 3.35 µl water were used. </P><P>Gibson mastermix was added to the DNA mix, incubated 1 hour at 50 °C and afterwards transformated.  </P><H4> Transformation of ligation and Gibson from 16.08.</H4>
        Top10 cells were plated out in chloramphenicol containing plates.
        
        <H4>Start standardisation of VP16 and KRAB and SV40 Promoter</H4>
PCR was performed at 60°C annealing temp: 
<UL><LI>oIG0042/0041 –&gt; KRAB ( 445bp)</LI><LI>oIG0040/0039 –&gt; VP16 (451bp)</LI><LI>oIG0043/0044 –&gt; pSV40 (411bp)</LI></UL>
Results:

PCR products were band isolated and a gel extraction was performed.
 <H4>Gibson of VP16 KRAB and SV40 into pSB1C3</H4>
Gibson was performed using the following approach:
<H3> 17.08.2013 </H3><H4> Picking and plating of colonies from the transformation of 16.08.</H4></DIV><H4> Continue standardisation of VP16 and KRAB and pSV40</H4><P> Colonie PCRs were performed using 60°C annealing temp and oIG6017 and oIG6018 after standard protocoll. </P><H4> Results: </H4>
The positive colonies (upper row) were picked for mini prep and one of each was send for sequencing.
<H3> 18.08.2013 </H3><H4> Miniprep of plated colonies from the 17.08.
        yield was between 30 and 50 ng/µl
        </H4><H3> 19.08.2013 </H3><H4> Test digest of the miniprep from 18.08.</H4><UL>
        from each plasmid 250 ng had been used; 1 h, 37 °C
<LI>HA-tag (4 minipreps): cut with AgeI (for gel NgoMIV and SpeI cut pIG0001 was used as control) </LI><LI>CMV (580 bp): cut with XbaRI and SpeI HF </LI><LI>BGH (4 minipreps, 222 bp): cut with AgeI HF and Eco HF </LI><LI>NLS(21 bp): only sent to sequencing </LI></UL><H4> Gel run of test digest from 19.08.</H4>
From left to right: Marker 2 log, HA (4x), HA Kontrolle, CMV, BGH (4x), Marker 2 log 
BGH terminator seemd not to have worked. HA, NLS and CMV were sent to sequencing. As sequencing primer pIG6017 has been used (fv primer binding around 100 bp before the insert).  

<H3>20.08.13</H3><H4>Sequencing results</H4>
All three parts are fully sequenced without mistakes.

  <H4>Digest of the parts NLS, HA-Tag, CMV- &amp; SV40-promoter and Cas9</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> 1 µg </TD></TR><TR><TD> restriction enzyme </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 3.5 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 35 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 2 h.
</DIV><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> plasmid </TH><TH align="right"> restriction enzymes </TH></TR><TR><TD> pIG001 (Cas9) </TD><TD align="right"> NgoMIV &amp; PstI HF</TD></TR><TR><TD> NLS </TD><TD align="right"> AgeI HF &amp; PstI HF</TD></TR><TR><TD> HA </TD><TD align="right"> EcoRI HF &amp; XbaI</TD></TR><TR><TD> CMV </TD><TD align="right"> EcoRI HF &amp; SpeI HF</TD></TR><TR><TD> SV40 </TD><TD align="right"> EcoRI HF &amp; SpeI HF</TD></TR></TBODY></TABLE></DIV><H4>Gel run</H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (1 kb Roth); SV40; HA; Cas; NLS; CMV </TD></TR></TBODY></TABLE><P>Expected bands: ~ 4kb for Cas9; ~ 2kb for NLS and HA (incl. Bb); 580 bp for CMV; 340 bp for SV40.</P></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (1 kb Roth); SV40; HA; Cas; NLS; CMV </TD></TR></TBODY></TABLE></DIV><H4>Gel extraction</H4><P>DNA were purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Changes to the protocol: 
<UL><LI>incubation at 56 °C for 10 min for gel dissolving</LI><LI>elution with dH<SUB>2</SUB>O (incubation at 50 °C for 4 min before centrifugation)</LI></UL></P><P>Yield: 5 ng/µl for CMV and SV40; ~ 12 ng/µl for Cas9 and HA; 61 ng/µl for NLS</P><H4>Ligation</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> NLS (with Bb) </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> Cas9 </TD><TD align="right"> 14.9 µl </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE></DIV><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> HA (with Bb) </TD><TD align="right"> 3 µl </TD></TR><TR><TD> CMV / SV40 </TD><TD align="right"> 6 / 3.5 µl </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Incubation at 22 °C for 30 min.
</DIV><H4>Transformation was made from the ligation of 20.08. and retrafo of standardized HA, NLS, CMV </H4><H4>Colony PCR of the BGH Gibson plate from 16.08</H4>
Colony PCR was made analog to standard protocol. As primer pIG6017 and pIG6018 are used. 12 clonies had been screened. Excpected size of product: 550 bp

Loading scheme: M,colony 1-12, positive control (standardized NLS with 349 bp product size)
Clone 6 has been plated out over night.
<H3>21.08.13</H3><H4> Minipreps from the plated BGH/retrafos from 20.08.
yield around 65 ng/µl; BGH Trafo was send to sequencing (with pIG6017)
</H4><H4>Colony PCR of clones of the ligation from 20.08.</H4>
12 colonies were tested for CMV-HA, SV40-HA, NLS-Cas9 with oIG6017+18.
<P>Colony PCR of CMV-HA (excpected size: 950 bp):</P><DIV id="floathleft">
loading scheme: M, 1-12, positive control (standardized NLS); Clone 4 was picked and plated out.
</DIV><P>Colony PCR of NLS-Cas9 (excpected size: 4.5 kbp):</P><DIV id="floathleft">
loading scheme: M, 1-12, M; Clone 3 was picked and plated out.
</DIV><P>Colony PCR of SV40-HA (excpected size: 700 bp):</P><DIV id="floathleft">
loading scheme: M, 1-12, M; Clone 11 was picked and plated out.
</DIV><H4> Oligo annealing of linker fv and linker rev (7 AS, oIG0051+52)</H4><TABLE class="tabelle"><TBODY><TR><TD> Oligo 1 (100 µM) </TD><TD> 50 µl </TD></TR><TR><TD> Oligo 1 (100 µM) </TD><TD> 50 µl </TD></TR><TR><TD> water </TD><TD> 150 µl </TD></TR></TBODY></TABLE> 
Incubate at 98 C for 4 minutes and cool down in turned off heating block o.n.

<H3>22.08.13</H3><H4>Sequencing results of BGH_pSB1C3 are completely correct. BGH can be used for further cloning.</H4><H4>Miniprep of pCMV-HA and pNLS-Cas9 from plated colonies from 21.08.</H4><H4>Digest of miniprep from 22.08. and BGH_pSB1C3</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> 2 µg </TD></TR><TR><TD> restriction enzyme </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 3.5 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 30 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 2 h.
</DIV><UL><LI>pNLS-Cas9: cut with NgoMIV + PstI </LI><LI>pCMV-HA: AgeI + PstI</LI><LI>pSV40-HA: AgeI + PstI</LI><LI>BGH_pSB1C3: NgoMIV + SpeI </LI><LI>pNLS already cut with AgeI and SpeI </LI></UL><H4>Gel run</H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (1 kb Roth); CMV-HA; BGH; NLS-Cas9; SV40-HA. </TD></TR></TBODY></TABLE><P>All bands were at the expected size; CMV-HA (2.5 kb), BGH (220 bp), NLS-Cas9 (4 kb), SV40-HA (2.5 kb) were cut out.</P></DIV><H4> Oligo annealing of linker fv and linker rev from Max (3 AS)(oIG0045+46)</H4><TABLE class="tabelle"><TBODY><TR><TD> Oligo 1 (100 µM) </TD><TD> 50 µl </TD></TR><TR><TD> Oligo 1 (100 µM) </TD><TD> 50 µl </TD></TR><TR><TD> water </TD><TD> 150 µl </TD></TR></TBODY></TABLE> 
Incubate at 98 C for 4 minutes and cool down in turned off heating block for 3 hours.

<H4> Ligation </H4><DIV>1. of annealed linker oligos (7AS and 3 AS linker [(oIG0051+52)+(oIG0045+46)] and pSB1C3 cut with NgoMIV and SpeI
<P>Linker-oligo was diluted 1:100 and concentration measured with nanodrop (X ng/ µl).</P><P>30 ng of bb was used (1.7µl of cutted pIG001)and an 3 times  excess of Insert (1.2 ng of annealed Linker-oligo)</P><P>[50ng/bp vector*3=ng insert/bp insert].</P><P>Ligation was incubated 30 minutes at 22 °C and afterwards transformed. </P></DIV><DIV>2. of NLS and BGH
   <P>30 ng Bb + 3 times  excess of BGH</P></DIV><DIV>3. of CMV-HA / SV40-HA and NLS-Cas9
   <P>15 ng Bb (CMV-HA / SV40-HA) + 2 times  excess of NLS-Cas9</P></DIV><H4>Transformation</H4><H3>23.08.13</H3><H4>Colony PCR</H4>
10 colonies were tested for Linker 3 aa, Linker 7 aa, NLS-BGH, SV40-... and CMV-... with oIG6017+18.
 
  <H4>Gel run</H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Upper row: Marker (2 log NEB); 8x Linker 3 aa; 5x NLS-BGH. Lower row: Marker (2 log NEB); 5x Linker 7 aa; 5x NLS-BGH, control (pNLS).</TD></TR></TBODY></TABLE><P>The bands of Linker 3 aa no. 1 &amp; 5 and Linker 7 aa no. 3 were at the right size. </P></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Upper row: Marker (1 kb Roth); 9x CMV-... Lower row: Marker (1 kb Roth); 9x SV40-...</TD></TR></TBODY></TABLE><P>The bands of CMV no. 5 &amp; 9 and SV40 no. 1, 5, 6, 8 &amp; 9 were at the right size.</P></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (1 kb Roth); CMV-...; SV40-... </TD></TR></TBODY></TABLE><P>The band of SV40 was at the right size.</P></DIV><H3>24.08.13</H3><H4>Miniprep of the positive clones</H4><P>DNA was purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Yield: ~ 90 ng/µl for the linkers; 170 ng/µl for CMV-...</P><H4>Sequencing of the linkers</H4><P>Linker 3 aa no. 1 and Linker 7 aa no. 3 were send in for sequencing (primer: oIG6018)</P><H4>Repeat of ligation of NLS-BGH</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> NLS (with Bb) </TD><TD align="right"> 0.5 µl (30 ng)</TD></TR><TR><TD> BGH </TD><TD align="right"> 2 µl </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE></DIV><H4>Trafo</H4><P>Ligation, HA, NLS and SV40 were transformed into <I>E. coli</I>. SV40-NLS-Cas9 no. 5 was spread on a new plate.</P><H3>25.08.13</H3><H4>Miniprep</H4><P>DNA was purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Yield: ~ 70 ng/µl</P><H4>Digest of the parts HA and NLS-Cas9</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> 1 µg </TD></TR><TR><TD> restriction enzyme </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 3 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 30 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 1:45 h.
</DIV><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> plasmid </TH><TH align="right"> restriction enzymes </TH></TR><TR><TD> HA </TD><TD align="right"> AgeI &amp; PstI HF</TD></TR><TR><TD> NLS-Cas9 </TD><TD align="right"> NgoMIV HF &amp; PstI HF</TD></TR></TBODY></TABLE></DIV><H4>Gel run</H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (1 kb Roth); HA; - ;NLS-Cas9. </TD></TR></TBODY></TABLE><P>Bands should be at the right size (HA: ~ 2 kb; NLS-Cas9 ~ 4 kb), so they were cut out.</P></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (1 kb Roth); HA; - ;NLS-Cas9. </TD></TR></TBODY></TABLE></DIV><H4>Gel extraction</H4><P>DNA were purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Changes to the protocol: 
<UL><LI>incubation at 56 °C for 10 min for gel dissolving</LI><LI>elution with dH<SUB>2</SUB>O (incubation at 50 °C for 4 min before centrifugation)</LI></UL></P><P>Yield: ~ 20 ng/µl</P><H4>Ligation</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> HA (with Bb) </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NLS-Cas9 </TD><TD align="right"> 14.9 µl </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE></DIV><H4>Trafo</H4><DIV class="floatleft"><OL><LI>4 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 300 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with chloramphenicol</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H3>26.08.13</H3><H4>Colony PCR</H4>
10 colonies were tested for HA-NLS-Cas9 and 16 for NLS-BGH with oIG6017+18.
<DIV class="floatleft">
Mix:
<TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> Colony </TD><TD align="right"> pipet tip </TD></TR><TR><TD> Primers (10 µM) </TD><TD align="right"> each 1 µl</TD></TR><TR><TD> Taq polymerase </TD><TD align="right"> 0.125 µl </TD></TR><TR><TD> Taq buffer </TD><TD align="right"> 2.5 µl </TD></TR><TR><TD> dNTPs (2.5 mM) </TD><TD align="right"> 2.5 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 25 µl </TD></TR></TBODY></TABLE></DIV><DIV class="floatright">
Program:
<TABLE class="tabelle"><TBODY><TR><TH> temperature </TH><TH align="right"> time </TH></TR><TR><TD> 95 °C </TD><TD align="right"> 10 min </TD></TR><TR><TD> 95 °C </TD><TD align="right"> 30 s </TD></TR><TR><TD> 60 °C </TD><TD align="right"> 40 s </TD></TR><TR><TD> 68 °C </TD><TD align="right"> 1 min/kb </TD></TR><TR><TD> 68 °C </TD><TD align="right"> 5 min </TD></TR></TBODY></TABLE>
Step 2 - 4 were repeated for 25 times.
</DIV><H4>Gel run</H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Upper row: Marker (1 kb Roth); 10x HA-NLS-Cas9; 3x NLS-BGH. Lower row: Marker (2 log NEB); 13x NLS-BGH.</TD></TR></TBODY></TABLE><P>All bands of HA-NLS-Cas9 were at the right size. The assembly of NLS-BGH was not correct planned, so the result of the colony PCR is not of interest.</P><P>HA-NLS-Cas9 no. 2 and 6 were spread on a new plate.</P></DIV><H3>27.08.13</H3><H4>Miniprep of HA-NLS-Cas9</H4><P>DNA was purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Yield: ~ 150 ng/µl</P><H4>Digest of Fusion parts and Linker</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> 1 - 2 µg </TD></TR><TR><TD> restriction enzyme </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right">2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 1:50 h.
</DIV><UL><LI>CMV-HA-NLS-Cas9: cut with EcoRI-HF and AgeI </LI><LI> CMV-HA-NLS-Cas: cut with PstI and SpeI-HF</LI><LI> HA-NLS-Cas9: cut with EcoRI-HF and AgeI-HF</LI><LI>BGH: cut with Pst and XbaI</LI><LI> Linker 7 aa: cut with EcoRI-HF and NgoMIV </LI><LI> Linker 3 aa: cut with EcoRI-HF and NgoMIV </LI><LI>pIG0001: cut with EcoRI-HF and SpeI-HF to isolate the pSB1C3 backbone </LI></UL><H4>Gel run</H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (1 kb Roth); linker 3 aa; CMV-... (cut with Age&amp;Eco); linker 7 aa; CMV-... (cut with Pst&amp;Spe); HA-NLS-Cas9; BGH; Marker (2 log NEB) </TD></TR></TBODY></TABLE></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (1 kb Roth); pSB1C3; CMV-... (cut with Pst&amp;Spe) </TD></TR></TBODY></TABLE><P>All bands were at the right size, so they were cut out: </P></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (1 kb Roth); linker 3 aa; CMV-... (cut with Age&amp;Eco); linker 7 aa; CMV-... (cut with Pst&amp;Spe); HA-NLS-Cas9; BGH; Marker (2 log NEB) </TD></TR></TBODY></TABLE></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (1 kb Roth); pSB1C3; CMV-... (cut with Pst&amp;Spe) </TD></TR></TBODY></TABLE></DIV><H4>Gel extraction</H4><P>DNA were purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Changes to the protocol: 
<UL><LI>incubation at 56 °C for 10 min for gel dissolving</LI><LI>elution with dH<SUB>2</SUB>O (incubation at 50 °C for 4 min before centrifugation)</LI></UL></P><P>Yield: ~ 30 - 40 ng/µl. </P><H4>Ligation</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> Linker 3 aa / Linker 7 aa (with Bb) </TD><TD align="right"> 1 µl (30 ng) </TD></TR><TR><TD> CMV-HA-NLS-Cas9 / HA-NLS-Cas9 </TD><TD align="right"> 5 µl </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE></DIV><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> CMV-HA-NLS-Cas9 (with Bb) </TD><TD align="right"> 1 µl (30 ng) </TD></TR><TR><TD> BGH </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE></DIV><H4>Trafo</H4><DIV class="floatleft"><OL><LI>4 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 300 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with chloramphenicol</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H3>28.08.13</H3><H4>Colony PCR of ligation from 27.08.</H4>
8 colonies were tested for CMV-HA-NLS-Cas9-L3/L7, HA-NLS-Cas9-L3, 2 colonies were tested for CMV-HA-NLS-Cas9-BGH,HA-NLS-Cas9-L7 each with oIG6017+18. Elongation time was set 5 min 18 sec.
<P>From left to right: upper row Marker; 1-8 CMV+L3; 1-5 Cas+L7; lower row Marker; 6-8 Cas+L7; 1-8 Cas+L3</P><P>From left to right:Marker; 1-4 CMV+BGH; 2. CMV+L7; empty; 1. CMV+L7</P><H3>29.08.13</H3><H4>Digest of Fusion parts and effector domains</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> 1 - 2 µg </TD></TR><TR><TD> restriction enzyme </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 3 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 30 µl </TD></TR></TBODY></TABLE>
Incubation at 37 °C for 2 h.
</DIV><UL><LI>CMV-HA-NLS-Cas9-Linker3: cut with PstI and AgeI </LI><LI> HA-NLS-Cas9-Linker3: cut with PstI and AgeI </LI><LI> VP16, KRAB, G9a: cut with PstI and NgoMIV</LI></UL><H4>Gel run</H4>
picture

<H4>Gel extraction</H4><P>DNA were purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Changes to the protocol: 
<UL><LI>incubation at 56 °C for 10 min for gel dissolving</LI><LI>elution with dH<SUB>2</SUB>O (incubation at 50 °C for 4 min before centrifugation)</LI></UL></P><P>Yield: ~ xx ng/µl. </P><H4>Ligation</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> VP16 / KRAB / G9a  </TD><TD align="right"> 0,7 µl / 0,5 µl / 1,75 µl </TD></TR><TR><TD> CMV-HA-NLS-Cas9-Linker3 / HA-NLS-Cas9-Linker3 (with Bb) </TD><TD align="right"> 30 ng </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE></DIV>
30 min, RT

  <H4>Trafo</H4><DIV class="floatleft"><OL><LI>4 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 300 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with chloramphenicol</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H3>30.08.13</H3>
Picking and plating of 4 colonies of each ligation on chloramphenicol plates for miniprep.

<H3>31.08.13</H3><H4>Miniprep of Ligations from 29.8.13</H4><P>DNA was purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Yield: ~ 200 ng/µl</P><H4>  Testdigest: </H4><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD>  200 ng </TD><TD> DNA </TD></TR><TR><TD> 1 </TD><TD> Cut smart buffer </TD></TR><TR><TD> 0.5 </TD><TD> Xba </TD></TR><TR><TD> 0.5 </TD><TD> SpeI </TD></TR><TR><TD> Add to 10 µl </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Temp.: 37°C </LI><LI> Incubation time: 1h </LI></UL></DIV><H4> Gel run </H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (1 kb Roth); 2x CMV-...-VP16; 2x CMV-...-KRAB; 2x CMV-...-G9a; control (CMV-...) </TD></TR></TBODY></TABLE></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (1 kb Roth); 2x HA-NLS-Cas-L3-VP16; 2x HA-NLS-Cas-L3-KRAB; 2x HA-NLS-Cas-L3-G9a; control (HA-NLS-Cas-L3) </TD></TR></TBODY></TABLE><P>The bands of G9a are at the right size, the bands of VP16 and KRAB are assumed to be right.</P></DIV><DIV id="september"><P id="h2">
September
</P></DIV><H3>01.09.13</H3><H4>Digest of the parts (CMV-)HA-NLS-Cas9-L3-Effector and NLS</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> 1 µg </TD></TR><TR><TD> restriction enzyme </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 3 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 30 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 2 h.
</DIV><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> plasmid </TH><TH align="right"> restriction enzymes </TH></TR><TR><TD> (CMV-)HA-NLS-Cas9-L3-Effector </TD><TD align="right"> AgeI &amp; EcoRI</TD></TR><TR><TD> NLS </TD><TD align="right"> NgoMIV &amp; EcoRI </TD></TR></TBODY></TABLE></DIV><H4>Ligation</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> NLS (with Bb)  </TD><TD align="right"> 30 ng </TD></TR><TR><TD> CMV-HA-NLS-Cas9-L3-Effector / HA-NLS-Cas9-L3-Effector </TD><TD align="right"> ?? </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
30 min, RT
</DIV><H4>Trafo</H4><DIV class="floatleft"><OL><LI>4 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 300 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with chloramphenicol</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H3>02.09.13</H3><H4>Colony PCR</H4>
7 colonies of each plate were tested with oIG6017+18.
<DIV class="floatleft">
Mix:
<TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> Colony </TD><TD align="right"> pipet tip </TD></TR><TR><TD> Primers (10 µM) </TD><TD align="right"> each 1 µl</TD></TR><TR><TD> Taq polymerase </TD><TD align="right"> 0.125 µl </TD></TR><TR><TD> Taq buffer </TD><TD align="right"> 2.5 µl </TD></TR><TR><TD> dNTPs (2.5 mM) </TD><TD align="right"> 2.5 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 25 µl </TD></TR></TBODY></TABLE></DIV><DIV class="floatright">
Program:
<TABLE class="tabelle"><TBODY><TR><TH> temperature </TH><TH align="right"> time </TH></TR><TR><TD> 95 °C </TD><TD align="right"> 10 min </TD></TR><TR><TD> 95 °C </TD><TD align="right"> 30 s </TD></TR><TR><TD> 60 °C </TD><TD align="right"> 40 s </TD></TR><TR><TD> 68 °C </TD><TD align="right"> 1:15 min </TD></TR><TR><TD> 68 °C </TD><TD align="right"> 5 min </TD></TR></TBODY></TABLE>
Step 2 - 4 were repeated for 25 times.
</DIV><H4>Gel run</H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Upper row: Marker (2 lob NEB); 7x HA-NLS-Cas9-L3-VP16-NLS; 6x HA-NLS-Cas9-L3-KRAB-NLS. Lower row: Marker (2 lob NEB); 1x HA-NLS-Cas9-L3-KRAB-NLS; 7x HA-NLS-Cas9-L3-G9a-NLS; 5x CMV-HA-NLS-Cas9-L3-VP16-NLS. </TD></TR></TBODY></TABLE></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Upper row: Marker (2 lob NEB); 2x CMV-HA-NLS-Cas9-L3-VP16-NLS; 7x CMV-HA-NLS-Cas9-L3-KRAB-NLS. Lower row: Marker (2 lob NEB); 7x CMV-HA-NLS-Cas9-L3-G9a-NLS. </TD></TR></TBODY></TABLE><P>The bands of Cas-VP16 no. 1 &amp; 5, Cas-G9a, CMV-VP16 no. 1 - 5, CMV-KRAB no. 4 &amp; 6 and CMV-G9a seem to be at the right size.</P></DIV><H4>Digest of pSB1C3</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> BGH in pSB1C3 RFC 25 (~ 150 ng/µl)</TD><TD align="right"> 14,2 µl </TD></TR><TR><TD> EcoRI </TD><TD align="right"> 1 µl </TD></TR><TR><TD> SpeI </TD><TD align="right"> 1 µl </TD></TR><TR><TD> NEB CutSmart buffer </TD><TD align="right"> 5 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 50 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 2 h.
</DIV><H3>03.09.13</H3><H4>Miniprep of Ligations from 1.9.13</H4><P>DNA was purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Yield: ~ 200 ng/µl</P><H4>Digest of the parts (CMV-)HA-NLS-Cas9-L3-Effector-NLS and BGH</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> 1 µg </TD></TR><TR><TD> restriction enzyme </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 3 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 30 µl </TD></TR></TBODY></TABLE>
Incubation at 37 °C for 2 h.
</DIV><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> plasmid </TH><TH align="right"> restriction enzymes </TH></TR><TR><TD> (CMV-)HA-NLS-Cas9-L3-Effector-NLS </TD><TD align="right"> Spe &amp; Pst</TD></TR><TR><TD> BGH </TD><TD align="right"> Xba &amp; Pst </TD></TR></TBODY></TABLE></DIV><H4>Gel run</H4>
picture

<H4>Gel extraction</H4><P>DNA were purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Changes to the protocol: 
<UL><LI>incubation at 56 °C for 10 min for gel dissolving</LI><LI>elution with dH<SUB>2</SUB>O (incubation at 50 °C for 4 min before centrifugation)</LI></UL></P><P>Yield: ~ 2-70 ng/µl. </P><H4>Ligation</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> BGH  </TD><TD align="right"> 1,4 µl / 1,5 µl / 1,4 µl /1,3 µl/1,3 µl </TD></TR><TR><TD> CMV-HA-NLS-Cas9-Linker3-VP16-NLS / CMV-HA-NLS-Cas9-Linker3-KRAB-NLS / CMV-HA-NLS-Cas9-Linker3-G9a-NLS / HA-NLS-Cas9-Linker3-VP16-NLS / HA-NLS-Cas9-Linker3-G9a-NLS(with Bb) </TD><TD align="right"> 30 ng </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE></DIV>
60 min, RT

<H4> PCR of BGH (RFC10) and pSV40 (RFC10)</H4><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 200-300 ng </TD><TD> Template </TD></TR><TR><TD> 1 </TD><TD> Primer1(pSV40: oIG0059; BGH: oIG0057) </TD></TR><TR><TD> 1 </TD><TD> Primer2 (pSV40: oIG0060; BGH: oIG0058) </TD></TR><TR><TD> 4 </TD><TD> dNTPs </TD></TR><TR><TD> 1 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="$floatright"><UL><LI> Annealing: 60°C </LI><LI> Extension: 13 s</LI></UL></DIV><H4> Gibson </H4>
Gibson-Assembly was performed of BGH with pSB1C3 (RFC10), UVR8 with pSB1C3 (RFC25), PIF with pSB1C3 (RFC25), PhyB with pSB1C3 (RFC25) according to protocol.

<H4>Repeat of PCR amplification of SV40</H4>
PCR was made from pIG3002 and the primers oIG0060 and oIG0061. Annealing temperature was set to 67 °C and elongation time 11 seconds.
<H4>Gelrun and Gelextraction of SV40 PCR</H4>
yield: around 50 ng/µl
Gelextraction was used for Gibson assembly.
Eco and Spe were used for linearizing pSB1C3.
<H4>Transformation of SV40-RFC25 Gibson assembly </H4><H3>04.09.13</H3><H4>Colony PCRs of the transformation from the 03.09.13</H4><UL>
Following PCRs were set
calculated amounts of DNA mastermix (with formula of Hanna):
<LI>BGH-RFC 10: 8 colonies</LI><LI>Cas-G9a-BGH: 8 colonies (2.1-2.8)(excpected size: 1500 bp) </LI><LI>Cas-VP16-BGH: 8 colonies (3.1-3.8)(excpected size: 1000 bp)</LI><LI>CMV-KRAB-BGH: 8 colonies (4.1-4.8)(excpected size: 1000bp) </LI><LI>CMV-VP16-BGH: 1 colony (5.1)</LI><LI>CMV-G9a-BGH: 2 colonies (6.1+6.2)(excpected size: 1500 bp)</LI><LI>SV40 RFC10: 5 colonies (7.1-7.5) (excpected size: 656 bp)</LI><LI>Cas9-KRAB: 8 colonies (8.1+8.8) (excpected size: 750 bp)</LI><LI>Cas9-KRAB-NLS: 7 colonies (9.1+9.7)(excpected size: 780 bp) </LI></UL><P> Two different mastermixes have been made: oIG6018+6017 were used for SV40 and BGH; oIG6018+0007 were used for all others.</P><P> Elongation time was set to 1 minute and 30 seconds for Cas-G9a-BGH, Cas-VP16-BGH, CMV-KRAB-BGH, CMV-VP16-BGH, CMV-G9a-BGH and 47 seconds for all others.</P> 
As controls for Cas-G9a-BGH and CMV-KRAB-BGH, Cas-G9a (excpected size: 780 bp) and CMV-KRAB (excpected size: 750) were used.

<H4>Gelrun of Colony PCRs from the 04.09.13</H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD>Gel 1: upper lane: M, 1.1-1.8, 7.1-7.5; lower lane: M, 8.1-8.8; 9.1-9.5 </TD></TR></TBODY></TABLE></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD>Gel 2: upper lane: M 2.1-2.6; 6.1-6.2; negative control G9a; lower lane: M; 3.1-3.7; 5.1; negative control of VP16 </TD></TR></TBODY></TABLE></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> Gel 3: upper lane: M, 14.1-4.8, M; lower lane: 2.7-2.8; 3.8; 9.6; 9.7, M
 </TD></TR></TBODY></TABLE></DIV>

Following clones have been plated out: 1.1, 2.2, 3.3, 4.1, 5.1, 6.2, 7.3, 9.7 

<H3>05.09.13</H3><H4>Miniprep</H4><P>DNA was purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><H3>06.09.13</H3><H4>Digest</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> 2 µg </TD></TR><TR><TD> restriction enzyme </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 3 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 30 µl </TD></TR></TBODY></TABLE>
Incubation at 37 °C for 2 h.
</DIV><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> plasmid </TH><TH align="right"> restriction enzymes </TH></TR><TR><TD> NLS </TD><TD align="right"> Spe &amp; Pst </TD></TR><TR><TD> BGH </TD><TD align="right"> Xba &amp; Pst </TD></TR><TR><TD> BGH </TD><TD align="right"> Xba &amp; Eco </TD></TR><TR><TD> SV40 </TD><TD align="right"> Spe &amp; Eco </TD></TR><TR><TD> HA-NLS-Cas9-L3 </TD><TD align="right"> Xba &amp; Eco </TD></TR><TR><TD> HA-NLS-Cas9-L3-VP16-NLS-BGH </TD><TD align="right"> Xba &amp; Eco </TD></TR><TR><TD> HA-NLS-Cas9-L3-KRAB-NLS </TD><TD align="right"> Xba &amp; Eco </TD></TR><TR><TD> HA-NLS-Cas9-L3-G9a-NLS-BGH </TD><TD align="right"> Xba &amp; Eco </TD></TR><TR><TD> COP1 </TD><TD align="right"> Xba &amp; Eco </TD></TR><TR><TD> L3 </TD><TD align="right"> Age &amp; Pst </TD></TR><TR><TD> HA-NLS-Cas9 </TD><TD align="right"> Xba &amp; Spe </TD></TR><TR><TD> HA-NLS-Cas9 </TD><TD align="right"> Pst &amp; Spe </TD></TR></TBODY></TABLE></DIV><H4>Gel run</H4><P>All bands were at the right size, so they were cut out and purified.</P><H4>Ligation</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> Backbone </TD><TD align="right"> 30 ng </TD></TR><TR><TD> Insert </TD><TD align="right"> 3 fold molar amount </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Ligations:
<UL><LI>NLS + BGH</LI><LI>HA-NLS-Cas9-L3-VP16-NLS-BGH + CMV</LI><LI>HA-NLS-Cas9-L3-KRAB-NLS + CMV</LI><LI>HA-NLS-Cas9-L3-G9a-NLS-BGH + CMV</LI><LI>HA-NLS-Cas9-L3 + CMV</LI><LI>HA-NLS-Cas9-L3-VP16-NLS-BGH + SV40</LI><LI>HA-NLS-Cas9-L3-KRAB-NLS + SV40</LI><LI>HA-NLS-Cas9-L3-G9a-NLS-BGH + SV40</LI><LI>HA-NLS-Cas9-L3 + SV40</LI><LI>CMV + COP1</LI><LI>L3 + VP16</LI><LI>L3 + BGH</LI><LI>L3 + G9a</LI><LI>HA-NLS-Cas9 + BGH</LI><LI>CMV + HA-NLS-Cas9 + BGH</LI></UL>
Incubation at RT for 30 min.
</DIV><H4>Trafo</H4><DIV class="floatleft"><OL><LI>4 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 300 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with ampicilline</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H3>07.09.13</H3><H4>Colony PCRs of the transformation of 06.09.13</H4><UL><LI>NLS + BGH with oIG6017 &amp; 18</LI><LI>HA-NLS-Cas9-L3-VP16-NLS-BGH + CMV with oIG6017 &amp; oIG0009</LI><LI>HA-NLS-Cas9-L3-KRAB-NLS + SV40 with oIG6017 &amp; oIG0009</LI><LI>CMV + COP1 with oIG6017 &amp; 18</LI><LI>L3 + VP16 with oIG6017 &amp; 18</LI><LI>L3 + KRAB with oIG6017 &amp; 18</LI><LI>L3 + G9a with oIG6017 &amp; 18</LI><LI>HA-NLS-Cas9 + BGH with oIG6018 &amp; oIG0007</LI><LI>CMV + HA-NLS-Cas9 + BGH with oIG6017 &amp; oIG0009</LI></UL><P>On all other plates there were no colonies.</P><H4>Gel run</H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Upper row: Marker (2 log NEB); 11x CMV + HA-NLS-Cas9 + BGH; HA-NLS-Cas9-L3-KRAB-NLS + SV40; HA-NLS-Cas9-L3-VP16-NLS-BGH + CMV. Lower row: Marker (2 log NEB); 10x CMV + HA-NLS-Cas9 + BGH; 3x HA-NLS-Cas9-L3-VP16-NLS-BGH + CMV.</TD></TR></TBODY></TABLE><P>cPCR did not work</P></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Upper row: Marker (2 log NEB); 6x L3 + G9a; 3x CMV + COP13x HA-NLS-Cas9 + BGH. Lower row: Marker (2 log NEB); 3x CMV + COP1; 6x L3 + VP16.</TD></TR></TBODY></TABLE><P>cPCR did not work</P></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Upper row: Marker (2 log NEB); 6x L3 + KRAB; 3x HA-NLS-Cas9 + BGH. Lower row: Marker (2 log NEB); 3x HA-NLS-Cas9 + BGH; 6x NLS + BGH.</TD></TR></TBODY></TABLE><P>The bands of L3-KRAB no. 1, HA-NLS-Cas9-BGH no. 4 - 6 and NLS-BGH no. 3 were at the right size.</P><P>L3-KRAB no. 1, HA-NLS-Cas9-BGH no. 4 and NLS-BGH no. 3 and additionally 3 clones of L3-VP16 &amp; L3-G9a as well as 2 clones of CMV-COP1 &amp; CMV-HA-NLS-Cas9-L3-VP16-NLS-BGH were plated for minipreps.</P></DIV><H4>Repeat of trafo</H4><P>The ligations that did not yield colonies were transformed again.</P><H3> 08.09.2013 </H3><H4>Miniprep</H4><P>DNA was purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><H4>Digest</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> 1 µg </TD></TR><TR><TD> restriction enzyme </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 3 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 30 µl </TD></TR></TBODY></TABLE>
Incubation at 37 °C for 2.5 h.
</DIV><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> plasmid </TH><TH align="right"> restriction enzymes </TH></TR><TR><TD> L3-VP16/L3-G9a/L3-KRAB </TD><TD align="right"> Age &amp; Eco </TD></TR><TR><TD> NLS-BGH </TD><TD align="right"> Ngo &amp; Eco </TD></TR><TR><TD> HA-NLS-Cas9-BGH </TD><TD align="right"> Xba &amp; Eco </TD></TR><TR><TD> CMV </TD><TD align="right"> Spe &amp; Eco </TD></TR></TBODY></TABLE></DIV><H4>Test digest</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> 250 ng </TD></TR><TR><TD> restriction enzyme </TD><TD align="right"> each 0.5 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 1 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 10 µl </TD></TR></TBODY></TABLE>
Incubation at 37 °C for 2.5 h.
</DIV><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> plasmid </TH><TH align="right"> restriction enzymes </TH></TR><TR><TD> CMV-COP1 </TD><TD align="right"> Not </TD></TR><TR><TD> HA-NLS-Cas9-BGH </TD><TD align="right"> Not &amp; Age </TD></TR><TR><TD> CMV-HA-NLS-Cas9-L3-VP16-NLS-BGH </TD><TD align="right"> Not &amp; Ngo </TD></TR></TBODY></TABLE></DIV><H4>Gel run</H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Upper row: 3x L3-VP16; 3x L3-G9a; L3-KRAB. Lower row: NLS-BGH; HA-NLS-Cas9-BGH; CMV; 2x CMV-COP1; HA-NLS-Cas9-BGH (test); 2x CMV-HA-NLS-Cas9-L3-VP16-NLS-BGH. </TD></TR></TBODY></TABLE><P>The bands of the digests of L3-VP16 no. 3, L3-G9a no. 2, L3-KRAB, NLS-BGH, HA-NLS-Cas9-BGH, CMV and CMV-COP no. 2 were at the right size.</P></DIV><H4>Ligation</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> Backbone </TD><TD align="right"> 30 ng </TD></TR><TR><TD> Insert </TD><TD align="right"> 3 fold molar amount </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Ligations:
<UL><LI>NLS + BGH</LI><LI>HA-NLS-Cas9-L3-KRAB-NLS + CMV</LI><LI>HA-NLS-Cas9-L3-G9a-NLS-BGH + CMV</LI><LI>HA-NLS-Cas9-L3-VP16-NLS-BGH + SV40</LI><LI>HA-NLS-Cas9-L3-KRAB-NLS + SV40</LI><LI>HA-NLS-Cas9-L3-G9a-NLS-BGH + SV40</LI><LI>HA-NLS-Cas9-L3 + SV40</LI><LI>L3-VP16 + NLS-BGH</LI><LI>L3-G9a + NLS-BGH</LI><LI>L3-KRAB + NLS-BGH</LI></UL>
Incubation at RT for 45 min.
</DIV><H4>Trafo</H4><DIV class="floatleft"><OL><LI>4 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 300 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with ampicilline</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H4> Colony PCR for  </H4><P> Clones were obtained. Colony PCR was performed to screen for potential positive clones. 22 clones were picked for CMV-HA-NLS-Cas9-bGH, 6clones for SV40-HA-NLS-Cas9-L3-VP16-NLS-bGH and 6 clones for CMV-HA-NLS-cas9-L3, transferred to Master Mix, transferred to Master Plate. 
				</P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 2.5 </TD><TD>standard Taq buffer </TD></TR><TR><TD> 1 </TD><TD> oIG6017 </TD></TR><TR><TD> 1 </TD><TD> oIG6018</TD></TR><TR><TD> 2.5 </TD><TD> dNTPs </TD></TR><TR><TD> 0.125 </TD><TD> Taq Polymerase </TD></TR><TR><TD> Add to 25 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Annealing: 60°C </LI><LI> Elongation: 80 sec</LI><LI> 25 cycles </LI></UL></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> cPCR: Expected band size is roughly 1kb. </TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> cPCR: Expected band size is roughly 1kb. </TD></TR></TBODY></TABLE></DIV><DIV id="tag"><P> Positive clones were obtained and streaked on plates for further analysis. The positive ones of CMV-HA-NLS-CAS9-bGH are looking suspicious, but it is the best guess. The other positive ones look quite fine. <B> loading scheme </B>: Gel1: <I>M-1-2-3-4-5-6-7-8-9 10-11-12-13-14-15-16-17-18-M gel2:M-19-20-21-22</I>-1-2-3-4-5M-6-<B>1-2-3-4-5-6</B><I>CMV-HA-NLS-CAS9-bGH</I> CMV-HA-NLS-Cas9-L3 <B> SV40-HA-NLS-CAS9-L3-VP16-NLS-bGH </B></P></DIV><H3> 09.09.2013 </H3><H4>Digest of RNA plasmid, CMV (RFC10), SV40 (RFC10), HA-NLS-Cas9-KRAB-NLS, HA-NLS-Cas9-VP16-NLS-BGH </H4><DIV id="floathleft"><TABLE class="tabelle"><TBODY><TR><TH> Plasmid	</TH><TH> Enzym </TH><TH> Enzym </TH><TH> buffer</TH><TH> water</TH><TH> total</TH></TR><TR><TD> CMV </TD><TD>28,57143 </TD><TD>1µl SpeI</TD><TD>1µl PstI </TD><TD> 3,5</TD><TD>0,93 </TD><TD>35 µl </TD></TR><TR><TD> SV40 </TD><TD>21,7 </TD><TD> 1µl SpeI</TD><TD>1µl PstI </TD><TD> 3,5</TD><TD>2,7 </TD><TD>30 µl </TD></TR><TR><TD> KRAB </TD><TD>9 </TD><TD> 1µl SpeI</TD><TD>1µl PstI </TD><TD> 3,5</TD><TD>5,5 </TD><TD>20 µl </TD></TR><TR><TD> VP16</TD><TD>14,3 </TD><TD> 1µl SpeI</TD><TD>1µl PstI </TD><TD> 3,5</TD><TD>0,2 </TD><TD>20 µl </TD></TR><TR><TD>RNA-Plasmid</TD><TD>16,1 </TD><TD> 1µl SpeI</TD><TD>1µl PstI </TD><TD> 3,5</TD><TD>3,4 </TD><TD>25 µl </TD></TR></TBODY></TABLE></DIV><H4>Oligo annealing</H4><P>The crRNA sequences for targetting pKM006 were annealed:</P><UL><LI>pIG2016 &amp; 17 for SEAP nt 650</LI><LI>pIG2018 &amp; 19 for SEAP nt 750</LI><LI>pIG2020 &amp; 21 for SEAP nt 850</LI><LI>pIG2022 &amp; 23 for SEAP nt 950</LI><LI>pIG2024 &amp; 25 for SEAP CMVmin1</LI></UL><P>25 µl of each oligo + 75 µl water were heated at 95 °C for 4 min and let cool down for 3 hours.</P><H4>Gelrun of Digest from 09.09.</H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Upper row: Marker (1kb Roth); CMV (2640 bp); SV40 (2404 bp); HA-NLS-Cas9-L3-VP16-NLS-BGH (4868 bp); HA-NLS-Cas9-L3-KRAB (4603 bp);  Lower row: Marker (1kb Roth); RNA Plasmid.</TD></TR></TBODY></TABLE><P>Bands were cutted out</P></DIV><H4>Ligation of extracted fragments of 09.09.</H4>
  
  Annealed oligos were ligated into the cut RNA-Plasmid (10 fold molar amount of oligos); KRAB construct and VP16 construct were ligated into each promotor (SV40 and CMV) (3 fold molar amount of inserts).
  
  <H4>Trafo of the ligation from 09.09. </H4><H4>Digest of Cas-VP16-BGH, Cas-KRAB-NLS, Cas-G9a-BGH and HA-NLS-Cas9-L3 </H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> 2 µg </TD></TR><TR><TD> restriction enzyme /Xba, Eco) </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 3 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 30 µl </TD></TR></TBODY></TABLE>
Incubation at 37 °C for 2 h.
</DIV><H4>Gel run</H4>

Ligations:
<UL><LI>HA-NLS-Cas9-L3 + SV40</LI><LI>Cas-VP16-BGH + CMV</LI><LI>Cas-KRAB-NLS + CMV</LI><LI>Cas-G9a-BGH + CMV</LI><LI>Cas-G9a-BGH + SV40</LI><LI>Cas-KRAB-NLS + SV40</LI></UL>
Incubation at RT for 1 h.
</DIV><H4>Trafo</H4><DIV class="floatleft"><OL><LI>4 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 300 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with ampicilline</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H3>10.09.13</H3><H4>Colony PCR (Michi)</H4><P>4 clones of each RNA plasmid ligation were screened with oIG6017 &amp; 18 and 8 clones of SV40-Cas-KRAB-NLS and SV40-Cas-VP16-BGH were screened with oIG6017 &amp; oIG009.</P><H4>Gel run (Michi)</H4><P>The colony PCR with oIG6017 &amp; oIG009 did not yield any bands. The bands with oIG6017 &amp; 18, that were visible, were all the same size (above the expected size).</P><H4>Miniprep of Cas9-L3-VP16-NLS-BGH, Cas9-L3-KRAB-NLS, Cas9-L3-G9a-NLS-BGH and HA-NLS-Cas9-L3</H4><P>DNA was purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><H4>Digest</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> 1 µg </TD></TR><TR><TD> restriction enzyme (Xba, Eco)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 3 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 30 µl </TD></TR></TBODY></TABLE>
Incubation at 37 °C for 2.5 h.
</DIV><H3>11.09.13</H3><H4>Miniprep (Michi)</H4><P>3 colonies of SV40-Cas-KRAB-NLS &amp; SV40-Cas-VP16-BGH as well as 1 colony of each RNA-Plasmid were prepped.</P><H3>12.09.13</H3><H4>Test digest (Michi)</H4><P>SV40-Cas-KRAB-NLS, SV40-Cas-VP16-BGH, CMV-Cas-KRAB-NLS, CMV-Cas-G9a-BGH &amp; SV40-Cas-G9a-BGH were digested to check if they contain a promoter.</P><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> 250 ng </TD></TR><TR><TD> Ngo &amp; Not </TD><TD align="right"> each 0,5 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 1 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 10 µl </TD></TR></TBODY></TABLE>
Incubation at 37 °C for 2.5 h.
</DIV><H4>Digest (Michi)</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> SV40-Cas-KRAB-NLS </TD><TD align="right"> 2 µg </TD></TR><TR><TD> Spe &amp; Pst </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Incubation at 37 °C for 2.5 h.
</DIV><H4>Gel run of digests (Michi)</H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Upper row: Marker (Roth 1 kb); 3x SV40-Cas-VP16-BGH; 3x SV40-Cas-KRAB-NLS; 2x SV40-Cas-G9a-BGH; CMV-Cas-G9a-BGH. Lower row: Marker (Roth 1 kb); CMV-Cas-G9a-BGH; 2x CMV-Cas-KRAB-NLS; 2x SV40-Cas-KRAB-NLS. </TD></TR></TBODY></TABLE><P>Only SV40-Cas-G9a-BGH showed the right banding pattern.</P></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (Roth 1 kb); 3x SV40-Cas-KRAB-NLS; CMV-Cas-VP16-BGH (test digest).</TD></TR></TBODY></TABLE><P>No bands were at the expected sizes.</P></DIV><H4>Colony PCR (Nadine)</H4><H4>Gel run of colony PCR (Nadine)</H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (NEB 2 log); ???????</TD></TR></TBODY></TABLE><P>????????</P></DIV><H4>Gel run of colony PCR (Nadine)</H4><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (NEB 2 log); ???????</TD></TR></TBODY></TABLE><P>????????</P></DIV><H4> Minipreps of L3-VP16-NLS-BGH, L3-KRAB-NLS-BGH, L3-G9a-NLS-BGH </H4><H4>Digest (Alina)</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> Plasmid </TD><TD align="right"> 2 µg </TD></TR><TR><TD> enzyme </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Incubation at 37 °C for 2 h.
</DIV>   
L3-VP16-NLS-BGH: Xba, Pst
L3-KRAB-NLS-BGH: Xba, Pst
L3-G9a-NLS-BGH: Xba, Pst
CMV-COP1: Spe, Pst
Phy-B: Xba, Pst
UVR8:Xba, Pst
PIF: Xba, Pst
CMV-HA-NLS-Cas9-L3: Spe, Pst
CMV: Spe, Pst

  <H4>Gel run of digests (Alina)</H4><H4>Ligation (Alina)</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> Backbone </TD><TD align="right"> 30 ng </TD></TR><TR><TD> Insert </TD><TD align="right"> 3 fold molar amount </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Ligations:
<UL><LI>CMV-COP1 + L3-VP16-NLS-BGH</LI><LI>CMV-COP1 + L3-KRAB-NLS-BGH</LI><LI>CMV-COP1 + L3-G9a-NLS-BGH</LI><LI>CMV-HA-NLS-Cas9-L3 + UVR8</LI><LI>CMV-HA-NLS-Cas9-L3 + PIF</LI></UL>
Incubation at RT for 1 h.
</DIV><H4>Trafo (Alina)</H4><DIV class="floatleft"><OL><LI>4 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 300 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with ampicilline</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H3>14.09.13</H3><H4>Digest (Michi)</H4><P>VP16, KRAB, G9a, PIF (B) and UVR8 (B) were digested with Eco &amp; Age; Linker 3 and NLS-BGH with Eco &amp; Ngo.</P><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> 2 µg </TD></TR><TR><TD> restriction enzymes </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Incubation at 37 °C for 2 h.
</DIV><H3>15.09.13</H3><H4>Colony PCR (Michi)</H4><P>VP16-NLS-BGH, KRAB-NLS-BGH, G9a-NLS-BGH, CMV-COP1-L3, CMV-PhyB, UVR8-NLS-BGH &amp; PIF-NLS-BGH were amplified with oIG6017 &amp; 18; CMV-HA-NLS-Cas-L3-UVR8, CMV-HA-NLS-Cas-L3-PIF &amp; CMV-HA-NLS-Cas-L3-NLS-BGH were amplified with oIG0007 &amp; oIG6018.</P><H4>Gel run (Michi)</H4><P>Only PIF-NLS-BGH showed bands. They were at the expected size (~ 900 bp).</P><H3>17.09.13</H3><H4>Digest (Alina)</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> CMV-Phy B </TD><TD align="right"> 2 µg </TD></TR><TR><TD> enzyme (Eco RI, Age I, Eco RV) </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 3 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 30 µl </TD></TR></TBODY></TABLE>
Incubation at 37 °C for 2 h.
</DIV><H4>Gel run of digests (Alina)</H4>
CMV-Phy B (ca. 2,5 kb) was cutted out.
  
 <H4>Ligation (Alina)</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> Linker 3 (Backbone) </TD><TD align="right"> 30 ng </TD></TR><TR><TD> CMV-Phy B (Insert) </TD><TD align="right"> 3 fold molar amount </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>

Incubation at RT for 1 h.
</DIV><H4>Trafo (Alina)</H4><DIV class="floatleft"><OL><LI>4 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 300 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with ampicilline</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H3>18.09.13</H3><H4>Colony PCR (Alina)</H4><P>CMV-Phy B-L3 was amplified with oIG6017 &amp; 18.</P><H4>Gel run (Alina)</H4><P>The bands were at the expected size (~ 2600 bp).</P><H3>19.09.13</H3><H4>Digest (Alina)</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> CMV-Phy B-L3 </TD><TD align="right"> 2 µg </TD></TR><TR><TD> enzyme (Age I, Pst I) </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Incubation at 37 °C for 2 h.
</DIV><H4>Gel run of digests (Alina)</H4>
CMV-Phy B-L3 in pSB1C3 was cutted out. <H4>Ligation (Alina)</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> CMV-Phy B-Linker 3 (Backbone) </TD><TD align="right"> 30 ng </TD></TR><TR><TD> VP16 / KRAB / G9a (Insert) </TD><TD align="right"> 3 fold molar amount </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>

Incubation at RT for 1 h.
</DIV><H4>Trafo (Alina)</H4><DIV class="floatleft"><OL><LI>4 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 300 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with ampicilline</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H3>20.09.13</H3><H4>Colony PCR (Alina)</H4><P>CMV-Phy B-L3-VP16,CMV-Phy B-L3-KRAB, CMV-Phy B-L3-G9a were amplified with oIG6017 &amp; 18.</P><H4>Gel run (Alina)</H4><P>The bands were at the expected size.</P><H4>Digest (Alina)</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> Plasmid </TD><TD align="right"> 2 µg </TD></TR><TR><TD> enzyme </TD><TD align="right"> each 1 µl </TD></TR><TR><TD> NEB buffer CutSmart </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Incubation at 37 °C for 2 h.
</DIV>   
NLS-BGH: Ngo, Pst
CMV-Phy B-L3-VP16: Age, Pst
CMV-Phy B-L3-VP16: Age, Pst
CMV-Phy B-L3-VP16: Age, Pst


  <H4>Gel run of digests (Alina)</H4><H4>Ligation (Alina)</H4><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> Backbone </TD><TD align="right"> 30 ng </TD></TR><TR><TD> Insert </TD><TD align="right"> 3 fold molar amount </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Ligations:
<UL><LI>CMV-Phy B-L3-VP16 + NLS-BGH</LI><LI>CMV-Phy B-L3-KRAB + NLS-BGH</LI><LI>CMV-Phy B-L3-G9a + NLS-BGH</LI></UL>
Incubation at RT for 1 h.
</DIV><H4>Trafo (Alina)</H4><DIV class="floatleft"><OL><LI>4 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 300 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with ampicilline</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H3>23.09.13</H3><H4>Colony PCR (Alina)</H4><P>CMV-Phy B-L3-VP16-NLS-BGH,CMV-Phy B-L3-KRAB-NLS-BGH, CMV-Phy B-L3-G9a-NLS-BGH were amplified with oIG6017 &amp; 18.</P><H4>Gel run (Alina)</H4><P>The bands were at the expected size.</P><H3>24.09.13</H3>

CMV-Phy B-L3-VP16-NLS-BGH, CMV-Phy B-L3-KRAB-NLS-BGH and CMV-Phy B-L3-G9a-NLS-BGH were sent for sequencing.

</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Freiburg/Notebook/standardisation">http://2013.igem.org/Team:Freiburg/Notebook/standardisation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Notebook/standardisation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Notebook/standardisation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Notebook/standardisation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Notebook/standardisation&amp;oldid=343418" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>