"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg/Project/method
From 2013.igem.org
Home
Highlights
Project
Abstract & Introduction
Targeting
Effectors
Effector Control
Modeling
Truncation
uniBAss
uniBOX
Manual
Application
Notebook
Targeting
Effectors
Effector Control
Modeling
uniBAss
Standardization
Material and Methods
BioBricks
Our BioBricks
Favorite BioBricks
Improvement of a Registry part
Sharing of our BioBricks
Safety
Human Practice
Expert opinions
Ethics
Safety & Security
Politics
Society
Education
Art
Facebook / Media
Team
News
Members
Exolab-Events
Statistics
Photo Gallery
Our Partners
Collaborations
Attributions
Thanks to
Abstract & Intro
Targeting
Effectors
Effector Control
Modeling
Truncation
uniBAss
Method
Development
Application & Outlook
uniBOX
Manual
Application
uniBAss - uniCAS Binding Assay
uniBAss - The innovative & easy screen for binding affinity
We developed a novel ELISA-based method to assess the binding efficiency of our proteins: the uniCAS Binding Assay  uniBAss. It is a powerful tool for the biochemical characterization of the binding capacity of our dCas9 fusion proteins and our truncated dCas9 versions with high throughput capabilities.
uniBAss ELISA
How uniBAss works
1. First a 96-well ELISA plate is coated with streptavidin. Then biotinylated oligonucleotides containing the DNA sequence complementary to the crRNA incorporated into dCas9 are applied, the strong affinity between biotin and streptavidin leads to wells coated with oligonucleotides.
2. 42h after transfection with the dCas9 fusion constructs and its desired target site on the RNAimer plasmid HEK-293T cells are taken up in dilution buffer and lysed by sonifying. Then the cell lysate containing dCas9/crRNA is ready to be applied to the wells.
3. The dCas9 proteins can now bind via their incorporated crRNAs to the complementary oligonucleotides on the bottom of the well.
4. After a washing step the first antibody is added. This mouse anti-HA antibody binds to the HA-tag at the N-terminus of dCas9.
5. After a washing step the secondary antibody can be added. The secondary anti-mouse-antibody is coupled to a horseradish peroxidase (HRP) that catalyzes the oxidation of ABTS by hydrogen peroxide resulting in a change of color.
6. The absorption at 405 nm is then measured and represents the amount of dCas9 bound in each well. To substract out differences in dCas9 expression levels the uniBAss data can be normalized to western blot data to reflect binding properties of dCas9 to its target site.
Principle of the uniBAss ELISA method.
For stepwise description see How uniBAss works above. For a detailed information on the development of uniBAss click here .
Results
Via the uniCAS Binding Assay we are able to examine the DNA binding capacity of different dCas9 fusion proteins . Thus uniBAss can be used to quantify the amount of HA-tagged DNA binding dCas9 and therefore helps us to characterize our constructs. Not only the dCas9-constructs can be investigated by the uniBAss but also every other DNA binding protein.
Figure 1: Binding affinity of dCas9 fusion constructs after normalizing the dCas9 amount with the help of a western blot
HEK-293T cells were either transfected in 6-well plates with dCas9 encoding an EMX1 crRNA on the same plasmid or co-transfected with dCas9 fusion constructs and the EMX1 RNAimer plasmid. Lysates were split and analyzed on uniBAss and western blot 42 h after transfection. The uniBAss ABTS readout seen above is normalized to dCas9 expression levels that were western blot quantified using ImageJ. Depicted values therefore represent comparable binding capacities of our dCas9 effector fusion proteins.
Mammalian HEK-293T cells were transfected with dCas9, dCas9-VP16, dCas9-KRAB, dCas9-G9a and the RNAimer plasmid targeting EMX1. 42h post transfection the standard uniBAss protocol was performed.To assess the accuracy of uniBAss we performed technical duplicates. For every construct the ABTS readout (background level substracted) was normalized to western blot dCas9 expression data obtained from the same cell lysates. Through this normalization we could compensate differences in protein expression between the constructs and thereby compare the actual binding properties. Figure 1 shows the normalized uniBAss data for dCas9, dCas9-VP16, dCas9-KRAB and dCas9-G9a + RNAimer(EMX1) when targeted to biotinylated EMX1 oligos. Transfection controls without dCas9 or the RNAimer plasmid did not show detectable signals in our binding assay.
Development of uniBAss
uniBAss Introduction
The need to characterize the properties of CRISPR/Cas is motivated by several means. Different fusion constructs have been built based on the CRISPR/Cas system, but it remained unclear if the binding characteristics of dCas9 fusion proteins differ from the native Cas9.
Simulating complex gene circuits in-silico generates new needs for straight forward characterization of the binding affinity followed by quantification. By this mean information can be gathered about a variety of promoters for different dCas9 fusion constructs. Different assays have been developed to characterize protein-DNA interplay to analyze their respective binding behavior [1] . Examples of these protein characterization assays are the Chromatin Immunoprecipitation (ChIP) and the DNA Electrophoretic Mobility Shift Assay (EMSA). Both systems have individual strength as well as limitations. ChIP can be used to quantify a sample when coupled with qPCR analysis. However ChIP is limited by the absence of high throughput possibilities [2] . EMSA is able to detect low abundance DNA binding proteins from lysate with a high sensitivity but it is difficult to quantify a sample [3] . However, those assays do not address the need to analyze the binding of a protein to its respective DNA sequence and thus perform a quantification with high capabilities. Starting from these needs a new assay for the biochemical characterization of dCas9 had to be established with the requirements of binding capacity test and quantification with high throughput capabilities. Therefore we developed our Enzyme-linked Immunosorbent Assay (ELISA) uniBAss. The assay is based on a streptavidin-biotin interaction [4] with the 5’ biotin tag of the oligonucleotide [5] . This was used to immobilize the target DNA.
Optimization
To obtain robust conditions for uniBAss, several experiments were performed. Different salt concentrations as well as different oligonucleotide lengths were examined to optimize this assay. Hereafter different dCas9 fusion proteins were tested regarding their DNA binding capacity. In addition Cas9 versions with one nickase domain and without a nickase domain were compared to investigate the influence of the nicking enzyme towards the binding behavior.  Subsequently to the optimization process the resulting buffer condition 10 mM Tris, 1 % BSA, 90 mM NaCl, 5 mM MgCl2 containing EDTA free Protease inhibitor was used to test different dCas9 fusion constructs.
To assess the influence of different salt compositions in the dilution buffer HEK-293T cells were transfected with a nickase Cas9 version containing a crRNA locus for EMX1. 24h after transfection the cells were taken up in standard dilution buffer b0 (10 mM Tris, 1 % BSA, 10 mM MgCl2, 10 mM NaCl) and sonified. After that lysate dilutions with different MgCl2 and NaCl concentrations were prepared in 96-well plates and analyzed via ELISA. Figure 2 and 3 show that the binding behaviour of the dCas9 / crRNA EMX1 strongly depends on the salt concentrations. For the detailed uniBAss protocol click here .
Figure 2: Binding restraint of the nickase Cas9/EMX1 crRNA complex for varying MgCl2 conditions
HEK-293T cells transfected with pIG9000 (nickase Cas9 plasmid containing an EMX1 target site) were lysed in dilution buffer (24 h post transfection) and subsequently analyzed for binding affinity to EMX1 oligos under varying buffer conditions. Here the MgCl2 concentration was varied and plotted against the ABTS readout of uniBAss. Error bars represent three technical replicates, negative control: no oligo coated.
Figure 3: Binding restraint of the nickase Cas9/EMX1 crRNA complex for varying NaCl conditions
HEK-293T cells transfected with pIG9000 (nickase Cas9 plasmid containing an EMX1 target site) were lysed in dilution buffer (24 h post transfection) and subsequently analyzed for binding affinity to EMX1 oligos under varying buffer conditions. Here the NaCl concentration was varied. Error bars represent three technical replicates, negative control: no oligo coated.
The experiment (figure 3) showed that the nickase Cas9 with EMX1 crRNA sequence acts differently for different buffer conditions. Therefore an optimization process was performed focusing on two main points. Firstly, different salt concentrations illustrate one parameter for optimization. Secondly, the length of the target oligonucleotides coated onto the ELISA plate was taken into account for characterization.
Different spacer length flanking the site targeted by Cas9 were designed to investigate differences in binding affinity of the Cas9/crRNA complex to its target (see figure 4). Sequences of all oligos can be found in our plasmids and oligo list . The sequences flanking the human EMX1 locus [6] were used to vary the lengths of the oligonucleotides.
Figure 4: Design of the different oligos
BTN = biotinylation on 5'end of one of the annealed oligonucleotides. Different spacer length flanking the EMX1 site targeted by Cas9 were designed to analyze differences in binding affinity of the Cas9/crRNA complex dependent on length of the presented double-stranded DNA oligonucleotide.
Figure 5: DNA binding capacity tested with different oligo-lengths
Annealed EMX1 oligonucleotides of varying lengths were coated on ELISA plate, transfection and uniBAss were performed as described here . A more robust binding of the nickase Cas9/EMX1 crRNA complex was achieved with longer oligonucleotide versions. Error bars represent three technical replicates, negative control: no oligo coated.
Using different oligonucleotide lengths to determine the best size of the coated oligonucleotides revealed that the presence of a longer sequence at the 5’ end with about 11 nucleotides consisting of the crRNA locus is essential to get the nickase Cas9 to bind. The presence of a second long sequence at the 3’ end of the crRNA however is not essential for simple binding assays but enables a more robust binding environment. Biotinylated oligonucleotides for prospective assays were designed analogue to the longest oligonucleotides used in the attempt to offer a point of interaction for fusion proteins.
The second parameter focused on optimizing the buffer conditions. The used buffer is composed of 10 mM Tris, 1 % BSA and varying concentrations of MgCl2 and NaCl. The salt concentrations were varied against each other generating 16 different buffers.
Factorial design
The second part of the optimization was done using a factorial design. This approach was used to test whether the MgCl2 concentration is linearly linked to the ABTS conversion.  Therefore different concentrations of MgCl2 – 30 mM, 20 mM, 10 mM and 5 mM – and of NaCl – 150 mM, 120 mM, 90 mM and 60 mM, were varied against each other. Indicating that for high MgCl2 concentrations the binding of dCas9 is close to zero (Figure 6). Whereas with descending MgCl2 concentrations more dCas9 is able to bind to the oligonucleotides, indicating an increase in an exponential matter. This is true for all the tested NaCl concentrations however it is to mention that 150 mM NaCl has the lowest bound nickase Cas9.
Figure 6: Factorial design experiment to test the behaviour of the nickase Cas9 towards different salt environments
After 24 h transfected HEK-293T cells (pIG9000) were taken up in dilution buffer b0 and the salt concentrations were adjusted individually resulting in the conditions (10 mM Tris, 1 % BSA, NaCl – 150 mM, 120 mM, 90 mM and 60 mM, MgCl2 – 30 mM, 20 mM, 10 mM and 5 mM respectively). The binding behaviour of the Cas9/crRNA towards EMX1 display similar behaviour for descending MgCl2 concentrations. This is true for all four NaCl concentrations however 150 mM NaCl has the smallest yield. Error bars represent three technical replicates, negative control: no oligo coated.
Figure 6 shows that the binding behaviour of the nickase Cas9/crRNA towards EMX1 display similar behaviour for descending MgCl2. This is true for all four NaCl concentrations however 150 mM NaCl has the smallest yield.
Use of factorial design
To test if the experimentally obtained curves can be fitted to an exponential increase within decreasing MgCl2 concentration a factorial design approach was performed (see table 1). To check if the conditions either behave in a linear or a non-linear way the experimental b0 values were compared to the calculated b0 values resulting from a central point. The results of this analysis are shown in Table 1. It was oberserved that the amount of bound dCas9 increases with a reduction of the MgCl2 concentration.
Table 1: Factorial design approach to test the behaviour of the Cas9/crRNA towards different salt environments
The results are sorted, from the highest difference (top) to the lowest difference (bottom). The lower the difference the more linear is the area surrounding the central point.
The plotting of the data in two dimensions indicated that there is an exponential increase of the ABTS conversion with descending MgCl2 concentrations. To verify this, a factorial design with different MgCl2 concentrations was performed within a range of 30 mM and 5 mM MgCl2. After having the data illustrated in three dimensions (figure 7), it occurred that the decreasing amounts of NaCl also contribute to an increasing yield.
Figure 7: Data from factorial design expriment plotted in a three dimensional matrix
To set the points into a common perspective the data from Figure 8 were plotted using MATLAB R2013a.
Subsequently to the optimization process the resulting buffer condition 10 mM Tris, 1 % BSA, 90 mM NaCl, 5 mM MgCl2 containing EDTA free Protease inhibitor was used to test different dCas9 fusion constructs.
Test of a fusion-construct
After having optimized the buffer conditions for the nickase Cas9 our dCas9–fusion proteins were studied aiming on two points. First to gather information about the target site for the CRISPR/Cas system in a straight forward manner and secondly to study which of the high yield buffer conditions displays the most robust readout when used with various dCas9–fusion proteins. The dCas9-VP16 fusion construct for gene activation was tested on different target on the human VEGF (Vascular endothelial growth factor) gene.
Figure 8: Three different VEGF biotinylated oligos were used as targets for dCas9-VP16 constructs containing the respective crRNA
The three different biotinylated and annealed oligonucleotides oIG9012/oIG9013, oIG9041/oIG9042 and oIG9015/oIG9016 (10 pmol/well) were coated by incubation at RT for 1 h. After 24 h transfected HEK-293T cells (dCas9-VP16 with VEGF locus 1, 2 or 3) were resuspended in dilution buffer b0 and subsequently the salt concentration was adjusted. The binding affinity of the dCas9–VP16/crRNA complex was analyzed during an ELISA. For different crRNAs the binding affinities seems to differ. This effect could however also be due to differential dCas9-VP16 from plasmids containing different VEGF crRNA loci. The best binding condition was obtained with 5 mM MgCl2 and 90 mM NaCl. Error bars represent three technical replicates, negative control: no oligo coated.
Figure 8 shows the dCas9-VP16 fusion proteins containing different crRNA targeting VEGF loci. This experiment aimed to examine whether there are different affinities between the different crRNAs. It can be seen, that the VEGF loci 1 and 3 behave similar, whereas the locus 2 reveals a higher ABTS readout in uniBAss. This indicates a higher binding affinity of the dCas9-VP16 construct to the VEGF locus 2 oligo but could also result from differential dCas9-VP16 expression levels from plasmids containing different VEGF crRNA loci. To compensate discrepancies in expression level a quantitative western blot could be performed.
Quantification of dCas9
To be able to quantify the amount of dCas9 binding to the oligos in the uniBAss ELISA a purified HA-tagged protein (FM) was used. As the ng/ml concentration of this protein is known, a dilution row was performed to obtain a standard curve and equation to convert the ABTS readout in ng/ml of bound HA-tagged protein. Figure 9 depicts the ng/ml of Cas9 or the different dCas9 constructs calculated via the formula (see figure 9) of the standard dilution curve. Additionally to dCas9-VP16 dCas9-KRAB was also tested.
Figure 9: Quantification of different Cas9 and dCas9 constructs
HEK-293T cells were transfected with the following constructs: nickase Cas9 with EMX1, dCas9 with EMX1, dCas9-KRAB with EMX1, dCas9-VP16 with VEGF2 and analyzed on uniBAss (standard protocol). Error bars represent three technical replicates, (-)cont no oligo coated, off target non EMX1 related oligo coated. ABTS readout was converted into ng/ml Cas9 by using the formula of an HA-tagged FM protein standard curve .
The uniBAss ELISA performed for figure 9 indicated that the fusion constructs are 3-fold less expressed than the constructs with the Cas9 or dCas9 only. The comparison between a dCas9 completely lacking the nickase activity and a Cas9 having one active nickase shows no significant difference. However the yield of the dCas9 construct without any nicking activity is slightly higher.
Application and Outlook
Find your perfect crRNA
As a part of our toolkit uniBAss is the perfect method to test different crRNA loci. Our experiments showed that not all crRNA sequences result in a similar efficiency of activation or repression of the desired locus. Here uniBAss can be used to test the binding affinity of the dCas9-fusion construct to a large amount of possible crRNAs for a desired gene locus at one. Hence uniBAss offers the possibility for a high throughput analysis of suitable crRNAs to use the unicast toolkit for every gene of personal desire.
High throughput on a microfluidic chip
To expand the high throughput feature of uniBAss from a 96-well plate to much more and faster analysis of crRNAs and samples a microfluidic chip could be used. By using an approach similar to the micro ELISA described by Ohashi T et al. [7] for Cas9-crRNA-DNA interaction it would be possible minimize the time for the analysis (20 times faster!) as well as the sample values (10 times less!). Therefore further work will optimize the uniBAss on a micro-fluidic chip device to save time and money for future iGEM teams when they are assessing the perfect crRNA target on the gene they want to regulate.
Test any DNA binding protein
On top, uniBAss can not only be used to characterize and analyze the DNA binding protein of our toolkit but also by all iGEM teams to assess the DNA binding capacity of their own DNA binding protein. uniBAss is the universal applicable method to test a huge variety of DNA binding proteins, such as TALEN and Zinc-finger. This helps all iGEM teams testing the functionality and DNA binding affinity of their DNA binding protein.
To put it in a nutshell
Team Freiburg 2013 developed and optimized uniBAss as a universally applicable tool for all iGEM teams not only to test Cas9-fusion constructs of our toolkit and assess the perfect crRNA for their gene locus of desire, but also to examine the DNA binding affinity of any DNA binding protein they wish in a fast, easy and robust assay.
References
(1) Lutolf, M.P., Hubbell, J. (2005). Synthetic biomaterials as instructive extracellular microenvironments for morphogenesis in tissue engineering. Nature biotechnology 23, 47–55.
(2) Gibson, D.G., et al. (2010). Creation of a bacterial cell controlled by a chemically synthesized genome. Science 329, 52–56.
(3) Hellman, L.M., Fried, M.G. (2007). Electrophoretic mobility shift assay (EMSA) for detecting protein-nucleic acid interactions. Nature protocols 2, 1849–61.
(4) Hyre, D.E., et al. (2006). Cooperative hydrogen bond interactions in the streptavidin – biotin system. 459–467.
(5) Weber, C.C., et al. (2005). Broad-spectrum protein biosensors for class-specific detection of antibiotics. Biotechnology and bioengineering 89, 9–17
(6) Zhang, Y., et al. (2013). Processing-Independent CRISPR RNAs Limit Natural Transformation in Neisseria meningitidis. Molecular Cell 50, 488–503.
(7) Ohashi, T., et al. (2009). A micro-ELISA system for the rapid and sensitive measurement of total and specific immunoglobulin E and clinical application to allergy diagnosis. Lab Chip 9, 991-995
Retrieved from " http://2013.igem.org/Team:Freiburg/Project/method "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
