"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg/Notebook/lab light
From 2013.igem.org
Home
Highlights
Project
Abstract & Introduction
Targeting
Effectors
Effector Control
Modeling
Truncation
uniBAss
uniBOX
Manual
Application
Notebook
Targeting
Effectors
Effector Control
Modeling
uniBAss
Standardization
Material and Methods
BioBricks
Our BioBricks
Favorite BioBricks
Improvement of a Registry part
Sharing of our BioBricks
Safety
Human Practice
Expert opinions
Ethics
Safety & Security
Politics
Society
Education
Art
Facebook / Media
Team
News
Members
Exolab-Events
Statistics
Photo Gallery
Our Partners
Collaborations
Attributions
Thanks to
Targeting
Effector
Effector Control
Light
July
August
September
October
Hormone
Modeling
uniBAss
Standardization
Material and Methods
Effector Control - Light
July
29.07.2013
Plasmids that we need as templates were transformed.
30.07.2013
Primers
Primers arrived today. Primers were diluted 1:10.
Minipreps
Transformed bacteria from yesterday with our template plasmids were minipreped.
pKM018: 83 ng/µl
pKM220: 216 ng/µl
pKM115: 177 ng/µl
pKM006: 118 ng/µl
pSAM200: 213 ng/µl
pSW42: 134 ng/µl
pSW45: 243 ng/µl
pIG2004: 212 ng/µl
T405: 213 ng/µl
Oligo Annealing of tetO crRNA
Oligos tetO13 1. CC and tetO13 2. CC were annealed
Therefore 20 µM diluted Primes were heated on 98° C for 4 minutes and then cooled down.
PCR list, program and composition
aIG4102: CMV Promotor from template T405 - Primers oIG4100F+R - 589 bp
aIG4103: Cas9 from template pIG2004 - Primers oIG4101F+R - 4170 bp
aIG4104: PIF and backbone from template pKM022- Primers oIG4102F+R - 3287 bp
aIG4105: KRAB from template pIG2004 - Primers oIG4103F+R - 390 bp
aIG4106: PhyB and backbone from template pKM018 - Primers oIG4104F+R - 4922 bp
aIG4107: UVR8 and backbone from template pKM168 - Primers oIG4105F+R - 3921 bp
aIG4108: COP1 and backbone frome template pKM115 - Primers oIG4106F+R - 3972 bp
aIG4109: CRY2 from template pSW42 - Primers oIG4107F+R - 1591 bp
aIG4110: Backbone from template pKM018 - Primers oIG4108F+R - 2924 bp
aIG4111: Backbone with VP16 from template pKM018 - Primers oIG4109F+R - 3282 bp
aIG4112: CIB from template pSW45 - Primers oIG4110F+R - 593 bp
aIG4113: Backbone from template pKM018 - Primers oIG4111F+R - 2881 bp
aIG4114: CIB from template pSW45 - Primers !! oIG4110F+oIG4112R - 638 bp
µl
Substance
10
Q5-HF Reaction Buffer
200 ng
Template
1
Primer forward
1
Primer reverse
4
dNTPs
1
DMSO
0,5
Q5-HF Polymerase
Add to 50
H2O
Temperature [° C]
Time
98
5 Min
98
30s
60 (or optimized)
30
72
30-40/kb
72
10 Min
4
hold
18 cycles
PCR results
aIG4102: 60° C - WORKED - GelExtraction: 87,9ng/ul
aIG4103: 60° C - 1 of 4 WORKED, Annealing Temperature lower? - GelExtraction: 50,5ng/ul Try 56°!
aIG4104: 60° C - DIDN'T WORK!!! Try with 58° - Was done, has to be tested on gel.
aIG4105: 60° C - WORKED - GelExtraction: 70,1ng/ul I and 57,7ng/ul II
aIG4106: 60° C - Was done, has to be tested on gel.
aIG4107: 60° C - ONLY SLIGHT BAND. TRY AGAIN!
aIG4108: 60° C - WRONG AMPLIFICATED BAND. Try again with different temperature!
aIG4109: 60° C - WORKED! - GelExtraction: 176,4ng/ul
aIG4110: 60° C - Was done, has to be tested on gel.
aIG4111: 60° C - WORKED! - GelExtraction: 84,4ng/ul
aIG4112: 60° C - Wasn't done yet.
aIG4113: 60° C - Was done, has to be tested on gel.
aIG4114: 60° C - Wan't done yet.
2-log Marker
aIG4102 (first 4) worked, 589 bp
aIG4105 (rest) worked, 390 bp
Roth Marker
aIG4103 only one PCR worked, 4170 bp
Roth Marker
aIG4104 didn't work
aIG4111 worked, 3282 bp
2-log Marker
aIG4109 (first two bands) worked, 1591 bp
aIG4107 (next two bands) didn't work, should have been 3921 bp
aIG4108 (one band) didnt work. Wrong band was amplified, should have been 3972 bp - another band of aIG4108 can be tested on a gel, didn't have space on this one.
31.07.13
PCR results
aIG4103: 54°-64° C - Everything WORKED!
aIG4104: 58° C - Gave three bands, upper band is supposed to be right. Was confirmed on a second gel with charging the gel only with a small amount of DNA.
aIG4106: 60° C - Didn't work. Try temperature gradient.
aIG4107: 54°-64° C - Everything Worked! But not very strong bands.
aIG4108: 54°-64° C - Didn't Work. Try seperate PCRs.
aIG4110: 60° C - Didn't work. Tried temperature gradient 54°-59° C - Worked with 55° and better with every additional ° C.
aIG4112: 60° C - Worked!
aIG4113: 60° C - Worked!
aIG4114: 60° C - Worked!
2-log marker
UPPER SIDE
aIG4103 (2 bands) - worked!
aIG4107 (2 bands) - worked!
aIG4106 (4 bands) - didn't work with 60° C, bands are too low!
LOWER SIDE
aIG4108 temperature gradient 54°-64° C - didn't work!
2-log marker
aIG4103 temperature gradient 54°-64° C - everything worked!
2-log marker
aIG4105 (2bands) - worked!
aIG4104 (2 bands) - upper band is considered to be the right band! - Worked!
aIG4110 - didn't work with 60° C.
2-log marker
aIG4107 temperature gradient 54°-64° C - worked, but only slight bands!
2-log marker
aIG4108 (2bands) - didn't work with 60° C!
aIG4113 (2 bands) - Worked!
aIG4110 - didn't work with 60° C.
2-log marker
aIG4110 (1 band) last column of temperature gradient (59° C) - worked!
aIG4112 (2 bands) - Worked!
aIG4114 - worked!.
2-log marker
aIG4110 temperature gradient (54°-59° C) - worked from 55° C upwards!
August
01.08.13
PCR results
aIG4104: Was gelexed and size was confirmed on a gel with a smaller amount of DNA. - Worked!
aIG4106: Temperature gradient 54° - 59° C and excess of reverse primer oIG4104R - Worked!
aIG4108: Seperate PCRs were tried out. Therefore the PCR was performed with template and at a time only one primer. After these PCRs (1x with forward Primer (58° C) and 1x with reverse Primer (62° C)) both columns were mixed and another PCR was performed. - Worked, but two bands appeared, upper band is considered to be the right fragment.
2-log marker
aIG4106 temperature gradient (54°-59° C) - worked!
2-log marker
aIG4108 seperate PCRs - worked, but other amplified products can be seen, too!
Gibsons
Fragments are mixed together to a total volume of 5µl and added to 15µl Gibson Master- Mix.
pIG4101: aIG4104- 0,83µl; aIG4103- 3,96µl; H2O- 0,21µl
pIG4103: aiG4107- 2,12µl; aiG4103- 2,88µl
pIG4105: aIG4110- 1,11µl; aIG4103- 3,51µl
pIG4106: aIG4111- 0,5µl;  aIG4112- 0,3µl, H2O- 4,2ul
pIG4107: aIG4113- 0,4µl;  aIG4105- 0,35µl; aIG4114- 0,35µl; H2O- 3,9µl
Gibson Mixes were transformed.
Ligation of pIG4100
Digest for pIG4100
µl
type
10
DNA (pKM006 or oIG4102) ~ 1000 ng
4
Cut Smart Buffer
1
EcoRI HF
1
Nhe1 HF
Add to 40µl
H2O
Temp.: 37°C
Incubation time: 1,5h
2-log marker
Digested aIG4102 with 601 bp
Digested pKM006 with 6393 bp and 126 bp.
Both were gelexed: aIG4102 19 ng/µl and pKM006 40 ng/µl
Ligation and Transformation of approach pIG4100
µl
type
1
T4 Ligase
2
T4 Ligase buffer
3 fold molar amount of insert
2
Backbone pKM006
1,3
Insert aIG4102
13,7
H2O
Temp.: RT
Incubation time: 15 minutes
After the ligation, 2,5 µl if this approach were transformed into E.coli and scratched out.
02.08.13
Gibson & Ligation results
Colonies could be seen on every plate except for one of the two ligation plates. Clones were picked and scratched out -> 51 Minis
Gibson
pIG4102: aIG4105- 0,4µl; aIG4106- 2,00µl; H2O- 2,6 µl
pIG4104: aiG4108- 3,2µl; aiG4105- 0,5µl; H2O- 1,3µl
Gibson Mixes were transformed.
03.08.13
Gibson results
Colonies could be seen on one of two plates, each. Colonies were scratched out for mini preps. Therefore we had clones for every construct.
Minipreps
51 Minis - concentrations between 50 and 200 ng/µl.
04.08.13
Minipreps
15 Minis.
05.08.13
Test digest
Today we made a test digest session of our 66 minis with following enzymes.
Plasmid
Enzymes
Size
pIG4100
EcoRI-HF & EcoRV-HF
1571 bp (Insert CMV 600bp) & 5417 bp
pIG4101
EcoRV-HF & NotI-HF
1952 bp (half the Cas9) & 5456 bp
pIG4102
EcoRI-HF & NotI-HF
1966 bp (with PhyB) & 3274 bp
pIG4103
BamHI-HF & XhoI
5337 bp (Cas9, UVR8) & 2682 bp
pIG4104
NotI-HF & BamHI-HF
1451 bp (with COP1) & 2839 bp
pIG4105
EcoRI-HF & EcoRV-HF
3040 bp (Half the Cas9 and half CRY2) & 5529 bp
pIG4106
NotI-HF & XbaI
970 bp (CIB, VP16) & 2833 bp
pIG4107
PstI-HF & NotI-HF
976 bp (CIB, KRAB) & 2821 bp
µl
type
~ 100-200 ng
DNA
1
Cut Smart buffer
0,5
Enzyme 1
0,5
Enzyme 2
Add to 10µl
H2O
Temp.: 37°C
Incubation time: 2h
2-log marker
Upper bands: pIG4100, expected sizes: Plasmid 3 and 5 were send to GATC
Bottom bands: pIG4101, expected sizes: Plasmid 4 and 7 were send to GATC
2-log marker
pIG4102, expected sizes, Plasmid 1 and 9 were send to GATC.
2-log marker
Upper bands left side: pIG4104, expected sizes: Plasmid 1 and 4 were send to GATC
Upper bands right side: pIG4105, expected sizes: Plasmid 1 and 6 were send to GATC
2-log marker
Upper bands: pIG4106,
not expected sizes: Plasmid 3 was send to GATC
Bottom bands: pIG4107, expected sizes: Plasmid 1 and 4 were send to GATC
For the sequencing we used GATC Primers: CMV-F for pIG4100 and EBV-RP (binds in pSV40) for all other plasmids.
Insertion of crRNA into RNA Plasmid
First RNA Plasmid backbone was digested with BbsI for 3 hours. Enzyme was taken out of - 80° C.
Then it was put on a gel and gelexed. Concentration: 50,5ng/ug. Backbone from Hormone- Group: 8,7ng/ul
Ligation: Digested backbone from Hormone- Induction and self- digested backbone was used. Molar ratio of                        backbone : insert - 1 : 3.
Mix: Backbone; Insert, 1ul T4- ligase, 2ul T4-ligase-buffer, H2O up to a total volume of 20ul
Incubated for 15min at RT
Transformed and scratched out.
06.08
Sequencing results
Sequencing results were
positive for pIG4101, pIG4102, pIG4103, pIG4105, pIG4106 and pIG4107 (sequenced with EBV-RP -> SV40 Terminator). pIG4100 didn't have the CMV promotor as an insert (-> new ligation) and pIG4104 didn't contain KRAB, although KRAB worked with pIG4102 (new Primers?)
Re-Trafos and inoculation for midi-preps
The positive Plasmids and already existing plasmids pKM018 and pKM115 were retransformed and put into 150 ml LB with 150 µl Ampicillin and incubated at 37° C over night.
New Primers for pIG4104
New Primers were designed for pIG4104.
Ligation of pIG4100
Ligation of pIG4100 was done again like on 31.07.13, only the ratio of backbone and insert was changed (3 µl backbone and 1,5 µl Insert).
2-log marker
Left band: Backbone, Right band: Inser
Result of Ligation of RNA Plasmid
There were clones on every plate. They were scratched out for minipreps.
New sequencing
Plasmids were send to GATC with different primers. Cas9 was sequenced completely in pIG4101.7. Other pIG4104 were screened for insertion of KRAB.
7.8.2013
Midipreps of pKM017; pIG4101.7 (?), pIG4102.1, pIG4103, pKM115, pIG4105.5, pIG4106.3, pIG4107.4
All midipreps were successfull
Concentrations:
pKM018: 474 ng/ul
pIG4101: 485 ng/ul
pIG4102: 528 ng /ul
pIG4103: 521 ng/ul
pKM115: 517 ng/ul
pIG4105.1: 492 ng /ul
pIG4106: 567 ng/ul
pIG4107: 476 ng /ul
Minipreps of RNA Plasmid pIG4201 and pIG4202
Minis were send to GATC. 3 of ligation with digested Backbone from Michi and tetO1 and 3 with tetO2. And 3 of ligation with newly digested Backbone from Gabi and tetO1 and 3 with tetO2.
Result of Ligation of pIG4100
There were clones on every plate. They were scratched out for minis.
8.8.2013
Sequencing of RNA Plasmid
1 of the 6 pIG4201 and 4 of the 6 pIG4202 were positiv with inserted crRNA.
Sequencing Order
Sequencing of pIG4107_2 and pIG4107_3 with Primer EBV-RP
Minis of pIG4100
12 Minis were prepped.
Re-Trafo for inoculation for Midi Preps
The following plasmids were retransformed: pIG1000, pIG4201, pIG4202
9.8.2013
Test digest of pIG4100
10.8.2013
Seeding of CHO cells
CHO- cells were seeded: 70.000 cells per well on 24 well- plates in HTS-medium
11.8.2013
Transfection of CHO cells
Protocol for 24- well plates:
1. Mix 40 µl Opti-MEM + 1.5 µl PEI-solution in a 1.5 ml Eppi
2.Prepare in another Eppi 0.5 µg of the DNA of interest
3.Add DNA to former Eppi, vortex thoroughly and incubate 15 min at RT
4.Resuspend solution gently and spread them drop-wise to the cells in the dish
5.Swing the dish in axis-form
6.Change of media 5h post transfection
12.8.2013
PCB- treatment of CHO-cells and illumination
Cells are treated with HTS-medium containing 15uM of phycocyanobilin
illumination started at 16.15. with 20uE light- quantity
plate 1 with activating red light (660nm)
plate 2 with inctivating far-red light (740nm), control
construction of pIG4104
pKM115 was digested with BamHI and BssHII for 4h. KRAB was amplified from pIG2004 with oIG4103FII and oIG4103RII
Subsequently the fragments were assembled via Gibson-cloning and transformed into E.coli
12.8.2013
Gibson results
Cell-colonies are picked and plated out anew
A mistake in the cloning strategy was discovered, new primers are designed
13.8.2013
Gibson results
CHO- cells are illuminated
14.8.2013
Seeding of CHO-cells
CHO- cells were seeded: 70.000 cells per well on 24 well- plates in HTS-medium
SEAP-measurements of illuminated cells
The results are all negative except for the positive control (-> see transfection Nr.6 of 11.8.2013)
The interaction with the cri-RNA may be responsible. The experiment is repeated with new target- sites
15.8.2013
Transfection of CHO-cells for red light and UV tests
The wiki- transfection protocoll is used:
1. Mix 40 µl Opti-MEM + 1.5 µl PEI-solution in a 1.5 ml Eppi
2.Prepare in another Eppi 0.5 µg of the DNA of interest
3.Add DNA to former Eppi, vortex thoroughly and incubate 15 min at RT
4.Resuspend solution gently and spread them drop-wise to the cells in the dish
5.Swing the dish in axis-form
6.Change of media 5h post transfection
The following pattern is transfected in 24- well plates:
16.8.2013
UV-b and red-/ far red light illumination of transfected cells from 15.8.2013
Cells for red- light illumination are illuminated with 20uE; 660nm or 720nm for 24h. Illumination is started at 14.20
Cells for UV-b light illumination are illiminated with 5uE; 311nm for 24h
17.8.2013
SEAP-measurements taken from illuminated- and untreated cells.
the following protocoll was obeyed for the whole process
200 µl of cell supernatant were transfered form the 24-well plate into one 96-well plate (round bottom)
Incubation for 30 min at 65 °C.
Centrifugation for 1 min at 1250 g.
100 µl of SEAP buffer were filled in the wells of a 96 well plate (flat bottom).
Addition of 80 µl supernatant from the round bottom plate to each well with SEAP buffer.
Addition of 20 µl pNPP (substrate).
96 plate was immediately placed in the blade reader.
Spectroscopic measurement every minute for 120 times (2h); wavelenght 405 nm.
numbers 1-6 are first six rows of the UV-b light approach -> look at transfections from 15.8.2013. numbers 7-12 are rows 1-6 of the red-light transfections from 15.8.2013
the system has to be improved, the controls showed the desired effect, but very weak. In the following the expression of Cas9- constructs has to be checked and improved. The tetO cri-RNA showed an insufficient binding pattern. New target sites for the plasmid have to be construct
20.8.2013
Construction of new cri-RNAs
6 new cri-RNAs are designed and ordered at sigma-aldrich. One new cri-RNA is provided by Max Stamnitz. All new cri-RNAs target pKM006!
Cloning of pIG4100; pIG4104; pIG4106; pIG4107
Primers for new cloning- strategies have arrived.
pIG4100: pKM006 is digested and the CMV-promotor from pAcGFP1 is amplified via PCR.
pIG4104: pKM115 is digested with BamHI and BssHII. KRAB is amplified via PCR from pIG2001
pIG4106: pKM018 is digested with EcoRI and NotI. CIB is amplified from pSW45
pIG4107: pIG4102 is digested with EcoRI and NotI. CIB is amplified from pSW45
21.8.2013
Cloning of pIG4100; pIG4104; pIG4106; pIG4107
Products from digestions and PCRs are purified via gel run in 1% agarose and gelexed The digest of pKM115 for pIG4104 has not worked and is repeated executing the same protocol.
Parts for pIG4100, pIG4106, pIG4107 are Gibson cloned and subsequently transformated into competent "TOP10" E.coli- cells
22.8.2013
Cloning of pIG4100; pIG4104; pIG4106; pIG4107
Clones of Gibsons are picked and plated out. Later to be picked and test- digested. For each plasmid 2 samples were sent to GATC for sequencing.
Gibson- cloning of pIG4104 is repeated with new digest and PCR-fragments. The constructs are subsequently transformated into competent "TOP10" E.coli- cells
Transfection control: Is Cas9-VP16 expresse? How Strong? Is it toxic?
HEK, HeLa and CHO- cells are seeded in 6-well plates
23.8.2013
Cloning of pIG4104
Colonies are picked and plated out for mini and test- digest on 24.- or 25.8.13
Transfection control: Is Cas9-VP16 expresse? How Strong? Is it toxic?
different DNA- concentrations are used for transfection in different cell lines. pIG1000 containing Cas9-VP16 is used. As a already tested sample, pIG8004 containing Cas9-G9a is used as positive- control. The following DNA- concentrations are transfected: 2ug; 3ug; 4ug; 6ug. One sample is tested with and addition of 0,5% DMSO into the medium attempting to boost the expression
The following pattern is transfected
24.8.2013
Oligo annealing
Targets T2, T3, T4, T5, T6, T7 and Max's random target were annealed.
Ligation of new RNA plasmid
Digested pIG3010 (50,1 ng/µl) and the annealed oliges were ligated with 3 molar amount of Insert.
25.8.2013
Result of ligation
Colonies were picked for mini preps.
26.8.2013
Minipreps of targets
Minipreps were sent for sequencing
27.8.2013
Result of sequencing
T2, T5 and T6 were positive.
Seeding of cells for UV light activation experiment
29.8.2013
Transfection of CHOs for UV light activation
30.8.2013
Light start
UV light was started at 15.30 o clock (two plexi glass panes)
31.8.2013
SEAP assay
SEAP assay of UV light activation was performed.
September
03.9.2013
Oligo annealing
5µl Oligo forward 100 µM to get 20 µM concentration
5 µl Oligo reverse 100 µM to get 20 µM concentration
10 µl NEB buffer 2
30 µl ddH2O
Mix was heated to 98° C, Heatblock was turned off and Oligos were cooled down
Digest of pIG3010
10 µl pIG3010
1 µl BbsI from -80° C
5 µl NEB buffer 2.1
up to 50 µl ddH2O
Digestion was put on a gel and purified (10 ng/µl).
Ligation of Target 3,4 and VEGF and transformation
Digested pIG3010 (10 ng/µl) and the annealed oliges were ligated with 3 molar amount of Insert.
04.9.2013
Result of Ligation
Colonies were picked
Gibson of pIG4104
Wrong primers were used for the last pIG4104 plasmid (was seen with the last sequencing). Therefore we made a new Gibson approach.
05.9.2013
Minipreps of T3, T4 and VEGF (plates) and pIG4104 (fluid culture)
Minipreps were send for sequencing (Primers for RNA Plasmid (in pIG3010) BGH-reverse and EBV-RP for pIG4104.)
06.9.2013
Sequencing results
pIG4104-3 was positive, T3, T4 and VEGF RNA plasmids were negative.
New Ligation of T4 target into standardized RNA plasmid
Procedure was done by standard protocol.
Seeding of CHO cells
4 24-well plates were ceeded with 65000 CHO cells per well.
07.9.2013
Status of CHO cells
CHO cells were all contaminated with bacteria again. Media was plated in a 10 cm dish. Result: No contamination here.
seeding of HEK cells
HEK cells were seeded for UV & BL activation and repression experiments at around 19/19.30. 4 24-well plates and 65000 cells per well.
08.09.2013
Transfection of HEKs for UV & BL activation and repression
HEK cells were transfected (standard protocol) at 19.30. Media was changed after 5 hours.
T4 RNA plasmid
6 colonies were scratched out.
09.09.2013
Minis of T4
3 of 6 plasmids were sent for sequencing (1-3).
Light start
Blue light was started at 21.30 with 450 nm and unsure intesity (4 colour box, 655 intensity   number). Dark control is in a light box without light on.
UV light was started at 20.30. Dark control is in the incubator (no UV light in the lab).
10.09.2013
Sequencing results
Sequencing results were positive for Bla Target 4. Bla Targets 2 and 3 were cloned by Philipp who   got positive results, too. Plasmids were retransformed and Midis were inoculated.
Seeding of HEK cells.
4 24 well plates were seeded with 65.000 HEK cells per well.
Light stop
Lightning of UV and BL were stopped after about 24 h and supernatants were taken and frozen.
11.09.2013
Midis
Midis of Bla target 2 (pIG4), Bla target 3 (pIG4, Bla target 4 (pIG4), EMX1 (pIG4 and VEGF4   (pIG4) were prepped.
SEAP of UV and BL activation and repression
Figure 1: SEAP measurement for blue light activation.
Figure 2: SEAP measurement for UV light activation.
Activation was okay, repression was in overflow.
Transfection for Activation-VP16 and Repression-KRAB target test row
In order to test which target we should use, we made test rows with different targets and   different target combinations. Medium was changed after 5 h.
Figure 3: Transfection plan: Target test row Activation 1
Figure 4: Transfection plan: Target test row Activation 1
Figure 5: Transfection plan: Target test row repression 1.
Figure 6: Transfection plan: Target test row repression 2.
12.09.2013
SEAP of Activation VP16 test row
Figure 7: SEAP measurement for target test row for Cas9-VP16.
Seeding of HEK cells for light activation experiments
13.09.2013
SEAP of Repression KRAB test row and UV and BL repression
Samples had to be diluted 1:10 because the constitutive CMV promoter is very strong. SEAP levels   were still in overflow.
Transfection for UV/BL/RL activation
Figure 8: Transfection plan for blue light activation. plan
Figure 9: Transfection plan for red light activation.
Figure 10: Transfection plan for UVB light activation.
14.09.2013
Light start for UV/BL/RL
15.09.2013
Light stop for UV and BL
SEAP of UV and BL activation
Figure 11: SEAP measurement for blue light activation.
Figure 12: SEAP measurement for UV light activation.
16.09.2013
Light stop for RL
SEAP for RL
Figure 13: SEAP measurement for red light activation.
18.09.2013
New charge of medium for CHOs
Medium was complemented with FCS, Pen Strep and Gentamicin.
20.09.13
New CHO-K1 & NIH cells
We received CHO-K1 cells from a freshly thawed cell batch. Thank you Shima from AG Ulbrich! These   cells were cultivated in DMEM.
We received NIH cells.
24.09.13
Seeding of CHO-K1 (from Toolbox) cells
6 24 well plates were seeded with about 65.000 cells per well.
25.09.13
Transfection for red/blue/UV light induction in CHO-K1 (from Toolbox)
CHO-K1 cells were transfected for red, blue and UV light induced gene activation. Cells that were   transfected for UV light were also transfected with Renilla. This was done for normalization, because UV light can kill the   cells.
Figure 14: Transfection plan for blue light activation.
Figure 15: Transfection plan for red light activation.
Figure 16: Transfection plan for UV light activation.
HTS medium was changed after 5 hours.
Seeding of NIH cells
2 24 well plates were seeded with NIH cells (65.000 cells per well).
26.09.13
Transfection for UV light induction in NIH cells
Transfection plan was the same as for CHOs on 25.09.
Light start for red/blue/UV light in CHO-K1 cells (from Toolbox)
1 plate was placed in UV light (311 nm). 1 plate was placed in a red light box (660) after PCB   treatment. 1 plate was placed in a far-red light box (740 nm) after PCB treatment. One plate was placed in a blue light box   (460 nm).
27.09.13
Light start for UV light in NIH cells
1 plate of transfected NIH cells was placed in UV light.
Light stop for UV (CHO-K1 from toolbox) & blue light (CHO-K1) cells
200 µl of supernatant of each well were removed and heated. After this the 96 well plate was   frozen (-20° C). SEAP measurement was done on the next day.
28.09.13
SEAP measurement of UV (CHO-K1 from toolbox) & blue light (CHO-K1) cells
Figure 17: SEAP measurement for blue light induced activation (CHO-K1)
Figure 18: SEAP measurement for UVB light induced activation (CHO-K1).
Light stop for red light (CHO-K1 from toolbox) and UV light (NIH)
200 µl of supernatant of each well were removed and heated. SEAP measurement was done according   to standard protocol.
Figure 19: SEAP measurement of red light induced activation (CHO-K1)
Figure 20: SEAP measurement for UVB light induced activation (NIH).
29.09.13
Seeding of CHO-K1 from AG Ulbrich and NIH cells
6 24 well plates with CHO-K1 were seeded and 6 24 well plates with NIH cells were seeded.
30.09.13
Transfection of CHO-K1 & NIH cells - red, blue and UV light
Alltogether 12 24 well plates were transfected.
Transfection plans
Figure 21: Transfection plan: Blue light activation.
Figure 22: Transfection plan: Red light activation.
Figure 23: Transfection plan: UVB light activation.
transfection calculation
Figure 24: Transfection calculation for blue light activation.
Figure 25: Transsfection calculation 2 for blue light activation.
Figure 26: Transfection calculation 1 for red light activation.
Figure 27: Transfection calculation 2 for red light activation.
Figure 28: Transfection calculation 1 for UVB light activation.
Figure 29: Transfection calculation 2 for UVB light activation
October
01.10.13
Light start
Both transfected cell lines were illuminated: 1 CHO plate & 1 NIH plate with red light 660 nm, 1   CHO plate & 1 NIH plate with blue 465 nm and 1 CHO plate & 1 NIH plate with UV light 311 nm. For every cell line and light   system there was a dark control plate (red light's dark control is far red light of 740 nm because the system is reversible)
02.10.13
Light stop
Light was stoped for UV and blue light illumination (after 24 hours) (CHO and NIH cells). 200 µl   supernatant were removed from each well for SEAP measurements. SEAP measurement was done according to standard protocol.
Figure 30: SEAP measurement for blue light activation in CHO-K1 cells.
HIER DANN ALLES ANDERE
Figure 31: SEAP measurement for blue light activation in NIH cells.
Figure 32: SEAP measurement for UVB light activation in CHO-K1 cells.
Figure 33: SEAP measurement for UVB light activation in NIH cells.
03.10.13
Light stop
Light was stoped for red light illumantion. 200 µl supernatant were removed from each well for   SEAP measurements. SEAP measurement was done according to standard protocol.
Figure 34: SEAP measurement for red light activation in NIH cells.
Figure 35: SEAP measurement for red light activation in CHO-K1 cells.
Retrieved from " http://2013.igem.org/Team:Freiburg/Notebook/lab_light "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
