<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Notebook_lab_activation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Notebook/lab_activation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Notebook/lab_activation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_activation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_activation&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Notebook/lab_activation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Notebook/lab activation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3></DIV><DIV id="menu"><UL id="navbar"><LI style="padding-left:15px"><A href="https://2013.igem.org/Team:Freiburg">Home</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Highlights">Highlights</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Introduction </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/induction">Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></LI></UL><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna">Targeting</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/method">uniBAss</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation">Standardization</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/protocols">Material and Methods</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/BioBricks">Our BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/favorite_parts">Favorite BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/improvement">Improvement of a Registry part</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/sharing">Sharing of our BioBricks </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/experts">Expert opinions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety/safety">Safety &amp; Security</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/politics">Politics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/society">Society</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/education">Education</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/art">Art</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/Facebook">Facebook / Media</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/News">News</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team2">Members</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Exolab">Exolab-Events</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Statistics">Statistics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/gallery">Photo Gallery</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Attributions">Our Partners</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/Coop">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/attributions"> Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/thanks">Thanks to</A></LI></DIV><DIV id="sidebar"><DIV id="sidebar_content"><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna"> Targeting </A></P><P class="first_order"><A class="active" href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector"> Effector </A></P><P class="second_order_note"><A class="active" href="https://2013.igem.org/Team:Freiburg/Notebook/lab_activation"> Activation </A></P><P class="third_order"><A href="#april"> April </A></P><P class="third_order"><A href="#may"> May </A></P><P class="third_order"><A href="#june"> June </A></P><P class="third_order"><A href="#july"> July </A></P><P class="third_order"><A href="#august"> August </A></P><P class="third_order"><A href="#september"> September </A></P><P class="second_order_note"><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_epigenetics"> Epigenetics </A></P><P class="second_order_note"><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_repression"> Repression </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling"> Modeling </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/method"> uniBAss </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation"> Standardization </A></P><P class="first_order"><A id="link" href="https://2013.igem.org/Team:Freiburg/protocols"> Material and Methods </A></P></DIV></DIV><DIV id="main_contant_note"><P id="h1">
Effector - Activation
</P><DIV id="tag"><H2> Experiments </H2><LI> E001: Making pKM602 (crRNAtargetsite(EMXI)-PCMVmin-SEAP-pA) </LI><LI> E002: Making pKM600 (PCAG-Cas9-VP16-DR_DR-pA) </LI><LI> E003: Making pKM603 (PCAG-Cas9-VP16-crRNA_EMXI-pA) </LI><LI> E004: Making pKM604 (PCAG-Cas9-VP16-crRNA_VEGF_VZ-8-pA) </LI><LI> E005: Making pKM605 (PCAG-Cas9-VP16-crRNA_VEGF_VZ-573-pA) </LI><LI> E006: Making pKM606 (PCAG-Cas9-VP16-crRNA_VEGF_VZ+434-pA) </LI><LI> E007: Making pKM607 (PCAG-Cas9-VP16-crRNA_VEGF_VZ-475-pA) </LI><LI> E008: Making pKM608 (crRNAtargetsite(VEGF_VZ-8)-PCMVmin-SEAP-pA) </LI><LI> E009: Making pKM609 (crRNAtargetsite(VEGF_VZ-573)-PCMVmin-SEAP-pA) </LI><LI> E010: Making pKM610 (crRNAtargetsite(VEGF_VZ+434)-PCMVmin-SEAP-pA) </LI><LI> E011: Making pKM611 (crRNAtargetsite(VEGF_VZ-475)-PCMVmin-SEAP-pA) </LI></DIV><DIV id="april"><P id="h2">
April
</P></DIV><DIV id="tag"><H2> 17.04.13 </H2><H3> E001: Oligo Annealing </H3><P> Oligos oKM509/510 were annealed. </P></DIV><DIV id="tag"><H2> 18.04.13 </H2><H3> E001: Digest </H3><P> Digest of pKM006 with AatII + NheI-HF. </P><P> Loading sheme: Marker (didn't work) - Digest of pKM006 - Failed PCR product - Failed PCR product. </P><P>Agarose gel</P><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 1: Gel pic 1 (Digest pKM006)</B></TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 19.04.13 </H2><H3> E001: Ligation </H3><P> Ligation with digested pKM006 and oligos oKM509/510. </P><H3> E001: Traffo with Ligation attempt </H3><P> Transformation with Top10 E.coli cells for getting pKM602. </P></DIV><DIV id="tag"><H2> 20.04.13 </H2><H3> E001-E011 Oligo Annealing </H3><LI> oKM507/508 </LI><LI> oKM509/510 </LI><LI> oKM511/512 </LI><LI> oKM513/514 </LI><LI> oKM515/516 </LI><LI> oKM517/518 </LI><LI> oKM519/520 </LI><LI> oKM521/522 </LI><LI> oKM523/524 </LI><LI> oKM525/526</LI></DIV><DIV id="tag"><H2> 20.04.13 </H2> E001: MiniPrep of Traffo(pKM602)<H3> E001: Test digest pKM602 with XmnI, EcoRI</H3><H3> E001: Agarose gel</H3><P> Checking test digest of pKM602 </P><P>Agarose gel</P><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 1: Gel pic 2 (Test-Digest pKM602)</B></TD></TR></TBODY></TABLE></DIV><P> Sending first attempt for sequencing --&gt; correct </P></DIV><DIV id="tag"><H2> 30.04.13 </H2> E008-E011: Digest<P> Digest of pKM006 with AatII and NheI-HF. </P><H3>  E008-E011: Agarose gel</H3><P>Agarose gel</P><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 1: Gel pic 3 (Digest pKM006)</B></TD></TR></TBODY></TABLE></DIV><H3> E008-E011: Ligation</H3><P> Ligation of pKM006 with Oligos: </P><LI> E008: oKM519/520 </LI><LI> E009: oKM521/522 </LI><LI> E010: oKM523/524 </LI><LI> E011: oKM525/526 </LI></DIV><DIV id="may"><P id="h2">
May
</P></DIV><DIV id="tag"><H2> 01.05.13 </H2> E008-E011: Traffo <P> Transformation with Top10 E.coli cells with ligation attempts. </P></DIV><DIV id="tag"><H2> 03.05.13 </H2> E008-E011: Miniprep pKM608, pKM609, pKM610, pKM611<H3> E008-E011: Test digest of pKM608-pKM611</H3><P> Test Digest with pKM608-pKM611 with XmnI, EcoRI. </P><H3>  E008-E011: Agarose gel</H3><P>Loading sheme: Marker - 1-6 (pKM608) - 1-6 (pKM609) - 1-6 (pKM610) - 1-6 (pKM611) </P><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 1: Gel pic 4 (TestDigest pKM608-pKM611)</B></TD></TR></TBODY></TABLE></DIV><P> Sending for Sequencing:</P><LI> pKM608_4 </LI><LI> pKM609_1 </LI><LI> pKM610_1 </LI><LI> pKM611_2 </LI><P> All attempts were correct! </P></DIV><DIV id="tag"><H2> 29.05.13 </H2> E008-E011: Retrafo of pKM608, pKM609, pKM610, pKM611<H3> E008-E011: MidiPrep of pKM608, pKM609, pKM610, pKM611</H3><P> 150 ml LB media + 200 Âµl amp. per attempt.</P><LI> pKM608_4: 655 ng/Âµl </LI><LI> pKM609_1: 490 ng/Âµl </LI><LI> pKM610_1: 1420 ng/Âµl </LI><LI> pKM611_2: 1670 ng/Âµl </LI></DIV><DIV id="june"><P id="h2">
June
</P></DIV><DIV id="tag"><H2> 04.06.13 </H2> E002: Digest <P> Digest of pX334a with NotI-HF and SacI </P></DIV><DIV id="tag"><H2> 05.06.13 </H2> E002: Agarose gel<P>Loading sheme: Marker - Digest pX334a - Marker </P><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 1: Gel pic 5 (Digest pX554a)</B></TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 06.06.13 </H2> New primers arrived (oKMxx1, oKMxx2, oKMxx3) <H3> E002: PCR 1 </H3><P> F2-3 (Cas9 D10A;H840) with oKMxx1/oKMxx2 </P><P> Size: 1947 bp </P><H3> E002: PCR 2 </H3><P> pKM018 with oKMxx3/oKM505 for getting VP16 </P><P> Size: 712 bp </P><H3> E002: Agarose gel</H3><P>Loading sheme: Marker - PCR1 (Cas9 D10A;H840) 1-4 - PCR2 (VP16) 1-4 - Marker </P><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 1: Gel pic 6 (PCR 1__PCR 2)</B></TD></TR></TBODY></TABLE></DIV></DIV><H3> E002: Digest </H3><P> Digest of PCR1 product (Cas9) with SacI and BamHI: Size: 1868 bp </P><P> Digest of PCR2 product (VP16) with BamHI and NotI: Size: 674 bp </P><DIV id="tag"><H2> 07.06.13 </H2> E002: Agarose gel <P>Loading sheme: Marker - Digest Cas9 - Digest VP16 - Marker </P><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 1: Gel pic 7 (Digest of PCR1 (Cas9) and digest of PCR2 (VP16)</B></TD></TR></TBODY></TABLE></DIV><H3> E002: Ligation</H3><P> Ligation with Cas9 (correct overhangs), VP16 (correct overhangs) and digested pX334a (SacI, NotI) </P><H3> E002: Traffo</H3><P> Transformation with Ligation attempt. </P></DIV><DIV id="tag"><H2> 09.06.13 </H2> E002: MiniPrep of pKM600 <P> pKM600_1 - pKM600_12 </P><H3> E002: Testdigest of pKM600 </H3><P> Test digest of pKM600_1 - pKM600_12 with SacI-HF </P><H3> E002: Agarose gel </H3><P>Loading sheme: Marker - Test digest pKM600_1 - pKM600_12 </P><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 1: Gel pic 8 Test digest of pKM600_1 - pKM600_12 with SacI-HF</B></TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 10.06.13 </H2> E003 - E007: Digest <P> Digest of pKM600_1 and pKM600_2 with BbsI </P><H3> E002: Ligation</H3><P> Ligation with digested pKM600_1: </P><LI> and oKM507/oKM508: pKM603   </LI><LI> and oKM519/520: pKM604    </LI><LI> and oKM521/oKM522: pKM605    </LI><LI> and oKM523/524: pKM606    </LI><LI> and oKM525/526: pKM607    </LI><H3> E003 - E007: Trafo </H3><P> Transformation of pKM603, pKM604, pKM605, pKM606, pKM607 </P><P> Sending for sequencing: pKM600_1 and pKM600_2 </P></DIV><DIV id="tag"><H2> 12.06.13 </H2><H3> E003 - E007: Miniprep</H3><P> MiniPrep of pKM603, pKM604, pKM605, pKM606, pKM607 </P><P> Sending for sequencing: </P><LI> pKM603_1 and pKM603_2 </LI><LI> pKM604_1 and pKM604_2 </LI><LI> pKM605_1 and pKM605_2 </LI><LI> pKM606_1 and pKM606_2 </LI><LI> pKM607_1 and pKM607_2 </LI><P> pKM603_1, pKM604_1, pKM605_1, pKM606_3,pKM607_2 were correct! </P></DIV><DIV id="tag"><H2> 18.06.13 </H2> Seeding HEK293T cells [24 well plate]</DIV><DIV id="tag"><H2> 19.06.13 </H2> Transfection <P> HEK293T cells were co-transfected with the following plasmid combination </P><P> 75000 cells/well </P><H3> Transfection sheme </H3><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 1: Transfection sheme with HEK293T cells</B></TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 20.06.13 </H2> SEAP Assay <P>SEAP is a secreted form of embryonic alkaline phosphatase and thus can be
easily detected in a sample of culture medium without destroying the cells and
time-consuming sample preparation. SEAP catalyzes the hydrolysis of
pNitrophenyl phosphate producing a yellow end product that can be read
spectrophotometrically at 405 nm. </P></DIV><DIV id="tag"><H2> 25.06.13 </H2> SEAP Assay 2 <P> Same conditions as SEAP Assay on 20.06.13 </P></DIV><DIV id="tag"><H2> 27.06.13 </H2> SEAP Assay 3 <P> Same conditions as SEAP Assay on 20.06.13 </P></DIV><DIV id="tag"><H2> 27.06.13 </H2> SEAP Assay 3 <P> Same conditions as SEAP Assay on 20.06.13 </P><H3> Results of SEAP Assays </H3><H3> Results SEAP Assay </H3><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 1: SEAP result with EMXI crRNA</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 1: SEAP result with four different VEGFA crRNAs</B></TD></TR></TBODY></TABLE></DIV></DIV><DIV id="september"><P id="h2">
September
</P></DIV><H2>09. September</H2><H3> Retrafo of Plasmids for the Midiprep</H3><UL>
Following plasmids are needed for activation and repression repeats:

<LI>pRSet</LI><LI>pKM600</LI><LI>pKM604</LI><LI>pKM605</LI><LI>pKM606</LI><LI>pKM607</LI><LI>pKM603</LI><LI>pIG2017</LI><LI>pIG2013</LI><LI>pSAM200</LI></UL>

Retrafo was inoculated into Ampicillin medium (concentration of 1:1000 from Ampicillin and medium). 

<H2>10. September</H2><H3>Midiprep</H3><P>Plasmids were prepped with the Midiprep kit of Promega according to the manufacturer's protocol.</P><H3>Seeding of cells</H3><P>4 24-well plates were seeded with 65,000 cells per well</P><H2>11. September</H2><H3>Transfection</H3><OL><LI>40 Âµl Opti-MEM + 2.25 Âµl PEI-solution were mixed in a 1.5 ml Eppi.</LI><LI>0.75 Âµg of the DNA of interest were prepaired in another Eppi .</LI><LI>Addition of the DNA to former Eppi, vortexing for 10 s and incubation for 15 min at RT</LI><LI>Solution was spread drop-wise to the cells in the dish</LI></OL><H3>Medium change</H3>
Medium was changed after 5 h.

<H2>13. September</H2><H3>Preparation for analyses</H3><UL><LI>Supernatant was collected for SEAP measurement.</LI><LI>Cells were frozen at - 80 Â°C for later on Western blot and luminescence measurement.</LI></UL><H2>14. September</H2><H3>SEAP measurement</H3><UL><LI>Collected supernatant and fresh cell culture medium was heated at 65 Â°C for 30 min, then centrifuged at 1250 g for 1 min.</LI><LI>Supernatant was diluted in cell culture medium 1:4 (25 Âµl supernatant + 75 Âµl medium).</LI><LI>80 Âµl of diluted supernatant was given to 100 Âµl of SEAP buffer.</LI><LI>After addition of 20 Âµl pNPP (substrate of SEAP) the absorbance of the mix was measured every minute for 2 h at 405 nm.
</LI></UL><H2>16. September</H2><H3>Cell lysis</H3><UL><LI>250 Âµl of lysis buffer (containing Tris-HCl, EGTA, MgSO<SUB>4</SUB>, DTT and protease inhibitor) were applied to the thawn cells of each well.</LI><LI>Incubation on ice for 10 min.</LI><LI>Centrifugation at 15,000 g for 4 min.</LI></UL><H3>Renilla luminescence measurement</H3><UL><LI>80 Âµl of supernatant were pipetted in a white 96 well plate.</LI><LI>Measurement of luminescence (every 2 min for 30 min) immediately after addition of 20 Âµl Renilla substrate in PBS.</LI></UL><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>SEAP activity normalized to Renilla expression</B>
				The value of TetR-VP16 is 64,6 which is too high to be shown.
			</TD></TR></TBODY></TABLE>
The different SEAP plasmids showed very differing background levels of SEAP expression. When Cas9-VP16 without a crRNA was cotransfected there was a reduction of SEAP expression in two cases, whereas in all other samples the SEAP level stayed unaltered. An activation in SEAP expression in comparison to the cells that were not transfected with Cas9-VP16 was only detectable for the EMX1 target sequence (5 fold).
SEAP activation with TetR-VP16 was much stronger, most probably because there are 13 binding sites for TetR-VP16, but only one for Cas9-VP16. 
</DIV><H2>17. September</H2><H3>Protein precipitation</H3><P>As the cell lysis needed to be done with a high amount of lysis buffer, proteins are too much diluted for western blotting. Thus, they were precipitated:</P><UL><LI>The remaining lysates of the triplikates (~ 80 Âµl) were put together in a new epi.</LI><LI>Addition of TCA to a final concentration of 10 % and 1 Âµg BSA.</LI><LI>Incubation for 30 min on ice.</LI><LI>Centrifugation for 30 min at full speed and 4 Â°C.</LI><LI>Removal of supernatant and addition of 500 Âµl acetone.</LI><LI>Incubation on ice over night.</LI><LI>Centrifugation for 15 min at full speed and 4 Â°C.</LI><LI>Removal of acetone and nair drying for 20 min.</LI><LI>Addition of 20 Âµl 2x SDS-loading dye and 0.5 Âµl Tris-HCl (pH = 8.8).</LI><LI>Heating at 95 Â°C for 5 min.</LI></UL><H2>18. September</H2><H3>SDS-gel run</H3><P>SDS-gel was loaded with 18 Âµl of each sample. A voltage of 80 V was applied till the loading dye bands reached the border between collection and separation gel, then the voltage was increased to 120 V.</P><H3>Western Blotting</H3><UL><LI>Activation of PVDF-membrane for 10 min in methanol.</LI><LI>Short incubation of Whatman papers and the PVDF-membrane in transfer buffer.</LI><LI>Loading of western blot apperature: (-) - Whatman paper - SDS-gel - PVDF-membrane - Whatman paper - (+).</LI><LI>200 mA were applied for 1.5 h.</LI></UL><H3>Antibody treatment I</H3><UL><LI>Incubation of PVDF membrane in blocking buffer (4 % milkpowder in TBS-T) for 1 h.</LI><LI>Incubation with anti HA antibody (in 2 % milkpowder in TBS-T) over night at 4 Â°C.</LI><LI>3 x washing with TBS-T for 5 min, each.</LI><LI>Incubation with anti mouse antibody for 1 h.</LI><LI>4 x washing with TBS-T for 10 min, each.</LI><LI>Measurement of luminescence after addition of ECL I + ECL II solution.</LI><LI>Air drying of the membrane for further antibody treatments.</LI></UL><H2>19. September</H2><H3>Seeding of cells</H3><P>3 24-well plates were seeded with 65,000 cells per well.</P><H3>Antibody treatment II</H3><UL><LI>Reactivation in methanol.</LI><LI>1 x washing with TBS-T for 5 min.</LI><LI>Incubation of PVDF membrane in blocking buffer (4 % milkpowder in TBS-T) for 30 min.</LI><LI>Incubation with anti beta-actin antibody (in 2 % milkpowder in TBS-T) over night at 4 Â°C.</LI><LI>3 x washing with TBS-T for 5 min, each.</LI><LI>Incubation with anti mouse antibody for 1 h.</LI><LI>4 x washing with TBS-T for 10 min, each.</LI><LI>Measurement of luminescence after addition of ECL I + ECL II solution (500 Âµl each).</LI></UL><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Western blot </B>
In every well that was transfected with Cas9-VP16 there was more or less the same expression of this fusion protein.				
			</TD></TR></TBODY></TABLE></DIV><H2>20. September</H2><H3>Transfection</H3><OL><LI>40 Âµl Opti-MEM + 2.25 Âµl PEI-solution were mixed in a 1.5 ml Eppi.</LI><LI>0.75 Âµg of the DNA of interest were prepaired in another Eppi .</LI><LI>Addition of the DNA to former Eppi, vortexing for 10 s and incubation for 15 min at RT</LI><LI>Solution was spread drop-wise to the cells in the dish</LI></OL><P>Ratio of DNA amount (mass): 
reporter plasmid (SEAP) : effector plasmid (Cas9) : RNA plasmid
1 : 4 : 4 </P><H3>Medium change</H3>
Medium was changed after 4.5 h.  

<H2>22. September</H2><H3>Medium removal</H3><UL><LI>Supernatant was collected for SEAP measurement.</LI><LI>Cells were frozen at - 80 Â°C for later on Western blotting.</LI></UL><H3>Preparation for SEAP measurement</H3><P>Collected supernatant and fresh cell culture medium was heated at 65 Â°C for 30 min, then centrifuged at 1250 g for 1 min.</P><H2>23. September</H2><H3>SEAP measurement</H3><UL><LI>Supernatant of wells containing CMV:SEAP was diluted in cell culture medium 1:10 (10 Âµl supernatant + 90 Âµl medium).</LI><LI>80 Âµl of (diluted) supernatant was given to 100 Âµl of SEAP buffer.</LI><LI>After addition of 20 Âµl pNPP (substrate of SEAP) the absorbance of the mix was measured every minute for 2 h at 405 nm.
</LI></UL><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>SEAP activity [U/L]</B>
				pKM600: Cas9-VP16 (not in iGEM standard); PhyB: negative control; all other Cas9 constucts are in iGEM standard (with the indicated promoter); bars represent three biological copies when there is an error bar (mean with standard deviation) or one when there is not.
			</TD></TR></TBODY></TABLE>
In comparison to the negative control the non standardized constructs repress the SEAP expression only slightly, whereas there is a strong repression by the standardized constructs, when driven by a CMV promoter. The repression of Cas9-KRAB seems to be a little stronger than the effect of Cas9 alone. But just like in the last experiment, the repression of Cas9-KRAB with a wrong crRNA is as strong as the repression with the right one.
In comparison to the negative control there was a reduction in SEAP expression when Cas-VP16 was cotransfected.
</DIV><H3>Seeding of cells</H3><P>4 24-well plates were seeded with 65,000 cells per well</P><H2>24. September</H2><H3>Repeat of seeding of cells</H3><P>4 24-well plates were seeded with 65,000 cells per well again, as the cell density of the first seeding was too low.</P><H2>25. September</H2><H3>Transfection</H3><OL><LI>40 Âµl Opti-MEM + 0.75 Âµg of the DNA of interest were mixed in a 1.5 ml Eppi.</LI><LI>Addition of 2.25 Âµl PEI-solution, vortexing for 10 s and incubation for at least 15 min at RT</LI><LI>Solution was spread drop-wise to the cells in the dish</LI></OL><P>Ratio of DNA amount (mass): 
reporter plasmid (SEAP) : effector plasmid (Cas9) : RNA plasmid
1 : 4 : 4 
Additionally a plasmid coding for Renilla luciferase was cotransfected (5 % of DNA amount)</P><H2>26. September</H2><H3>Medium change</H3>
Medium was changed after 8 h.

  <H3>Cell lysis of transfection of 20. September</H3><UL><LI>80 Âµl of RIPA lysis buffer were applied to the thawn cells of each well.</LI><LI>Incubation on ice for 10 min.</LI><LI>Sonification 3 x 30 s at maximum.</LI><LI>Centrifugation at 10,000 g for 10 min.</LI><LI>40 Âµl of supernatant were mixed with 10 Âµl 5 x SDS sample buffer.</LI><LI>Heating for 5 min at 95 Â°C.</LI></UL><H3>SDS-gel run</H3><P>SDS-gel was loaded with 40 Âµl of each sample. A voltage of 80 V was applied till the loading dye bands reached the border between collection and separation gel, then the voltage was increased to 120 V.</P><H3>Western Blotting</H3><UL><LI>Activation of PVDF-membrane for 10 min in methanol.</LI><LI>Short incubation of Whatman papers and the PVDF-membrane in transfer buffer.</LI><LI>Loading of western blot apperature: (-) - Whatman paper - SDS-gel - PVDF-membrane - Whatman paper - (+).</LI><LI>200 mA were applied for 1.5 h.</LI></UL><H3>Antibody treatment I</H3><UL><LI>Incubation of PVDF membrane in blocking buffer (4 % milkpowder in TBS-T) for 1 h.</LI><LI>Incubation with anti HA antibody (in 2 % milkpowder in TBS-T) over night at 4 Â°C.</LI><LI>3 x washing with TBS-T for 5 min, each.</LI><LI>Incubation with anti mouse antibody for 1 h.</LI><LI>4 x washing with TBS-T for 10 min, each.</LI><LI>Measurement of luminescence after addition of ECL I + ECL II solution.</LI><LI>Air drying of the membrane for further antibody treatments.</LI></UL><DIV><TABLE class="imgtxt" width="600px"><TBODY><TR><TD><B>Western blot of the SEAP assay from 23.09. </B>
				dCas-VP16 was expressed with each tested promoter, but strongest with CAG.
			</TD></TR></TBODY></TABLE></DIV><H2>27.09.13</H2><H3>Medium removal</H3><UL><LI>Supernatant was collected for SEAP measurement 48 h after transfection.</LI><LI>Cells were frozen at - 80 Â°C for later on Western blotting.</LI></UL><H2>28.09.13</H2><H3>SEAP measurement</H3><UL><LI>Collected supernatant and fresh cell culture medium was heated at 65 Â°C for 30 min, then centrifuged at 1250 g for 1 min.</LI><LI>Supernatant of wells containing CMV:SEAP was diluted in cell culture medium 1:15 (6.7 Âµl supernatant + 93.3 Âµl medium).</LI><LI>80 Âµl of (diluted) supernatant was given to 100 Âµl of SEAP buffer.</LI><LI>After addition of 20 Âµl pNPP (substrate of SEAP) the absorbance of the mix was measured every minute for 2 h at 405 nm.
</LI></UL><H3>Cell lysis for lyciferase measurement and western blot</H3><UL><LI>250 Âµl of lysis buffer (containing Tris-HCl, EGTA, MgSO<SUB>4</SUB>, DTT and protease inhibitor) were applied to the thawn cells of each well.</LI><LI>Incubation on ice for at least 10 min.</LI><LI>Centrifugation at 2,200 g for 30 min.</LI></UL><H3>Luciferase luminescence measurement</H3><UL><LI>80 Âµl of supernatant were pipetted in a white 96 well plate.</LI><LI>Measurement of luminescence (every 2 min for 15 min) immediately after addition of 20 Âµl Renilla substrate in PBS.</LI></UL><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>SEAP-Assay: Results </B>
All different tested loci could activate the SEAP-gene expression. Cas-VP16 under the control of the CMV promotor as well as the SV40 promotor worked fine. Using more than one target locus simultaniously increases gene activation.  				
			</TD></TR></TBODY></TABLE></DIV><H2>29.09.13</H2><H3>Protein precipitation</H3><P>As the cell lysis needed to be done with a high amount of lysis buffer, proteins are too much diluted for western blotting. Thus, they were precipitated:</P><UL><LI>340 Âµl of acetone was added to 85 Âµl of the remaining lysates of one well per triplicate.</LI><LI>Incubation at - 20 Â°C for 1:15 h.</LI><LI>Centrifugation for 5 min at 10,000 g and 4 Â°C.</LI><LI>Removal of acetone and air drying for 5 min.</LI><LI>Addition of 25 Âµl 2x SDS-loading dye.</LI><LI>Heating at 95 Â°C for 5 min.</LI></UL><H3>SDS-gel run</H3><P>SDS-gel was loaded with 20 Âµl of each sample. A voltage of 80 V was applied untill the loading dye bands reached the border between collection and separation gel, then the voltage was increased to 120 V.</P><H3>Western Blotting</H3><UL><LI>Activation of PVDF-membrane for 10 min in methanol.</LI><LI>Short incubation of Whatman papers and the PVDF-membrane in transfer buffer.</LI><LI>Loading of western blot apperature: (-) - Whatman paper - SDS-gel - PVDF-membrane - Whatman paper - (+).</LI><LI>200 mA were applied for 1.5 h.</LI></UL><H3>Antibody treatment I</H3><UL><LI>Incubation of PVDF membrane in blocking buffer (4 % milkpowder in TBS-T) for 1 h.</LI><LI>Incubation with anti HA antibody (in 2 % milkpowder in TBS-T) over night at 4 Â°C.</LI><LI>3 x washing with TBS-T for 5 min, each.</LI><LI>Incubation with anti mouse antibody for 1 h.</LI><LI>4 x washing with TBS-T for 10 min, each.</LI><LI>Measurement of luminescence after addition of ECL I + ECL II solution.</LI><LI>Air drying of the membrane for further antibody treatments.</LI></UL><DIV><TABLE class="imgtxt" width="400px"><TBODY><TR><TD><B>Western blot of the SEAP assay from 28.09. </B>
Western blotting did not work well. Maybe lysis due to relative long incubation time on Renilla lysis buffer destroyed cells to much beforehand. So maybe only degradation products are visible.				
			</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Freiburg/Notebook/lab_activation">http://2013.igem.org/Team:Freiburg/Notebook/lab_activation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Notebook/lab_activation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Notebook/lab_activation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_activation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_activation&amp;oldid=343441" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>