"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Freiburg/Project/crrna
From 2013.igem.org
Home
Highlights
Project
Abstract & Introduction
Targeting
Effectors
Effector Control
Modeling
Truncation
uniBAss
uniBOX
Manual
Application
Notebook
Targeting
Effectors
Effector Control
Modeling
uniBAss
Standardization
Material and Methods
BioBricks
Our BioBricks
Favorite BioBricks
Improvement of a Registry part
Sharing of our BioBricks
Safety
Human Practice
Expert opinions
Ethics
Safety & Security
Politics
Society
Education
Art
Facebook / Media
Team
News
Members
Exolab-Events
Statistics
Photo Gallery
Our Partners
Collaborations
Attributions
Thanks to
CrispR
Abstract & Intro
Targeting
RNAimer
Multiple targeting
crRNA design tool
Effectors
Effector Control
Modeling
Truncation
uniBAss
uniBOX
Manual
Application
Targeting
In our uniCAS Toolkit we engineered the CRISPR/Cas9 system for future applications to regulate gene expression. The key components for the binding of the catalytically inactive Cas9 (dCas9) to its targets are two small, non-coding RNAs: the CRISPR-RNA (crRNA) and the tracrRNA. These RNAs that guide our dCas9 to specific DNA sequences have to be cotransfected with the dCas9-plasmids. Therefore we designed an RNA plasmid, termed RNAimer , which contains all required RNAs for efficiently guiding the dCas9 protein to the required DNA - even if multiple DNA sites should be targeted.
As an essential part of our toolkit is the binding of our protein to DNA, we evaluated this in detail: Various DNA target sequences were compared to evaluate the best guiding crRNAs. Evaluated parameters were varying loci and GC-content. Based on these results, we tested if dCas9 is able to bind to multiple targets .
In order to simplify the search for potential target sequences, we programmed the crRNA design tool and provide it to the iGEM community. As the binding of the RNA-guided Cas9 requires a protospacer adjacent motif (PAM), the tool determines all possible regions of the desired DNA sequence that is pasted into the tool. Eventually, you obtain the position of your target sites and the sequences of the oligos that have to be inserted into the RNAimer - ready to order!
RNAimer - targeting dCas9 to its destination
Introduction
As dCas9 requires two small, non-coding RNAs to mediate binding to the DNA, we designed an RNA plasmid containing the tracrRNA and a site where the crRNA can be introduced. This is achieved by digestion using BbsI and annealed oligos corresponding to respective crRNA sequence. Two (or more) of these RNA plasmids (with different crRNAs) can be fused using the iGEM BioBrick cloning strategy thereby allowing for a RNAimer plasmid carrying multiple crRNAs.
Figure 1: RNAimer (BBa_K1150034)
Our RNA plasmid, termed the RNAimer, contains the tracrRNA and a site where the desired crRNA can be inserted. Transcription of both RNAs are driven by the H1 and the U6 promoter RNA polymerase III promoters, respectively. Assembly of multiple crRNAs can be easily done according to iGEM standard assembly.
As it is important that the RNAs are not being marked for protein expression and nuclear export, the RNA polymerase III is required for transcription. RNA polymerase III mainly synthesizes small non-coding RNAs (e.g. tRNAs or rRNAs) whereas the commonly used polymerase II is responsible for transcription of mainly mRNAs. [1, 2] . We chose the human U6- and H1-promoter to drive transcription of the RNAs as they are exclusively recognized by   polymerase III [2] .  In order to reduce the amount of plasmids for transfection when intending to target several genes or target sites at once, we wanted to join the required   crRNAs on one RNA plasmid. This was easily manageable by using the iGEM standard assembly: Oligos are inserted separately into the RNAimer plasmid (BBa_K1150034) by   digesting using BbsI and ligation. Afterwards the inserts located within the respective RNAimer plasmids can be combined by using the restriction enzymes of the prefix and suffix of the iGEM standard (Figure 6).
Figure 6: Assembly and function of the RNAimer
For each desired crRNA, one RNAimer plasmid (containing the crRNA after ligation) has to be digested with BbsI. Afterwards the annealed crRNA oligos will be ligated. The different RNA inserts can be assembled using the idempotent iGEM cloning strategy. Three different crRNAs (red, yellow, blue) will be transcribed.
Results
To test this strategy functionally we performed SEAP (Secreted Embryonic Alkaline Phosphatase) assays with dCas9-VP16 . HEK-293T cells were transfected with dCas9-VP16 and different RNAimer plasmids containing different crRNAs. We either co-transfected two crRNAs or a single plasmid carrying both crRNAs. This resulted in the same activation of SEAP expression (Figure 7).
Figure 7: SEAP activation with dCas9-VP16 and RNAimer.
HEK-293T cells were seeded at 65,000 cells in a 24-well format. After 24 hours cells were transfected following the standard procedure. 48 h post-transfection, supernatant was harvested and SEAP activity was evaluated following the standard protocol. Activation by dCas9-VP16 was performed using two targets at once. In the case of T3+4 the loci were on the same plasmid, whereas in the case of T3&T4 two separate plasmids were co-transfected. Both combinations display a strong activation of the CMVmin promoter, driving the SEAP reporter expression. The activation of the two conditions is comparable. This is a hint, that there is no decrease in efficiency when combining several loci on one plasmid.
Discussion
We have shown that our RNA plasmid "RNAimer" works as expected. The combination of two targets does not seem to decrease the efficiency of   targeting as the activation is comparable when transfecting two separate plasmids or combining both targets in one plasmid. With our toolkit it is possible to   induce gene expression by co-transfecting two plasmids (plasmid coding for the dCas9-effector fusion protein and the RNAimer plasmid carrying the crRNA and tracrRNA).
For future application researchers will be able to alter gene expression by transfecting only the plasmid containing the crRNA. This could be possible when the dCas9 will be   stably integrated into the genome of cells or model organisms, as recently done by Gilbert et al. [4] .
References
(1) Dieci G.,et al. (2007). The expanding RNA polymerase III transcriptome. Trends Genet. 23, 614-622.
(2) Myslinski E., et al. (2001). An unusually compact external promoter for RNA polymerase III transcription of the human H1RNA gene. Nucleic Acids Res. 29, 2502-2509.
(3) Cheng AW., et al. (2013). Multiplexed activation of endogenous genes by CRISPR-on, an RNA-guided transcriptional activator system. Cell Res.
(4) Gilbert LA.,et al. (2013). CRISPR-Mediated Modular RNA-Guided Regulation of Transcription in Eukaryotes. Cell 154, 442-451.
Multiple Targeting
Introduction
One of the greatest advantages of the CRISPR/Cas9 system compared to other transcription activators (e.g. Zinc fingers (ZFNs) or transcription-activator like effectors (TALEs)) is that only one protein is   required for targeting several DNA loci: For a new target just another crRNA has to be expressed. So a RNA plasmid was   designed containing the tracrRNA, where the crRNA can be easily introduced.
With this RNA plasmid and another plasmid containing the dCas9-effector fusion gene it is possible to target several DNA sites at once by co-transfecting two plasmids. This allows the simultaneous regulation of different genes or a stricter control of one gene by targeting several loci of this gene the same time.
First of all, we had to evaluate different DNA sequences for targeting, since the results of our first experiments (see labjournal for details) showed that various crRNAs resulted in different SEAP levels. Thus, we cloned five different target sites 26 bp upstream of the CMVmin promoter of a SEAP reporter plasmid.   Additionally we designed crRNAs in various distances upstream of this promoter (Tab. 1). These target sites were evaluated by activating SEAP expression   with dCas9-VP16.
Table 1: Different target sites
The VEGF and EMX1 target sites were cloned into the SEAP reporter plasmid. T2-T4 are original sequences found on this plasmid. All target loci are upstream of   the CMVmin promoter. VEGF and EMX1 are parts of the sequence upstream of the human VEGF or EMX1 gene, respectively.
Name
Distance to promoter
Sequence
GC content [%]
VEGF VZ-573
-26 bp
GTGTGCAGACGGCAGTCACTAGGGGGCGCT
70
VEGF VZ-475
-26 bp
GTGAGTGTGTGCGTGTGGGGTTGAGGGCGT
63
VEGF VZ-8
-26 bp
TTAAAAGTCGGCTGGTAGCGGGGAGGATCG
57
VEGF VZ+434
-26 bp
GACCTGCTTTTGGGGGTGACCGCCGGAGCG
70
EMX1
-26 bp
GGAAGGGCCTGAGTCCGAGCAGAAGAAGAA
57
T2
-148 bp
AAGCATTTATCAGGGTTATTGTCTCATGAG
37
T3
-222 bp
AATGCCGCAAAAAAGGGAATAAGGGCGACA
47
T4
-443 bp
GACCGAGTTGCTCTTGCCCGGCGTCAATAC
60
Results
Evaluation of different target sequences
At first we tested different target sequences at the same distance from the promoter. To do so we had to insert the target sequences into the SEAP reporter plasmid in front of the CMVmin.
HEK-293T cells were transfected with one of these SEAP plasmids and a plasmid containing dCas9-VP16, the tracrRNA and the appropriate crRNA. The   results show different activation efficiencies (Figure 8) in which no activation (VEGF VZ -573 and +434) and up to a 5 fold increase of SEAP activation could be observed. With the   tested target sequences that have a GC content of 70 % no activation was observed whereas induction of SEAP expression with the target   sequences with a GC content of about 60 % (compare Tab. 1) could be shown.
Figure 8: Targets with different sequences
The grey bars represent the SEAP activity of HEK-293T cells transfected with dCas9-VP16 but no crRNA (normalized to value 1). The blue bars show fold induction of SEAP activity by tansfecting dCas9-VP16 with crRNAs in comparison to the appropriate control without crRNA. An activation is visible for nearly all targets, but the fold-induction is differing strongly. All values are means of three biological replicates. Error bars represent standard deviation. For the experiment HEK-293T cells were seeded at 65,000 cells in a 24-well format. After 24 h cells were transfected following standard procedure. After 48 h of expression supernatant was harvested and SEAP activity was evaluated following the standard protocol.
Next, we evaluated different distances upstream of the promoter. Thus, we targeted four different loci on the reporter plasmid (EMX1, T2, T3 & T4; for sequence see Tab. 1).
HEK-293T cells were co-transfected with a CMVmin:SEAP plasmid that contains EMX1, CMV:dCas9-VP16 and the RNA plasmid, respectively. The results show an significant increase in   SEAP expression when EMX1, which is the closest target site to the promoter, is targeted (Figure 2). Activation decreases, the larger the distance   between promoter and target sites becomes (compare Tab. 1).
Figure 9: Targets with different distances to the promoter
Activation of CMVmin:SEAP via dCas9-VP16 with different distances to the promoter in HEK-293T cells. The grey bar represents the SEAP level without crRNA, the blue bars represent the activation using different crRNAs in increasing distance to the promoter (EMX1 the closest, T4 with the greatest distance). A clear decrease in activation is observed with increasing distance from promoter. All epxeriments were performed in biological triplicates. Error bars represent standard deviation. HEK-293T cells were seeded at 65,000 cells in 24-well format and 24 hours later transfected following the standard procedure. After 48 h of expression supernatant was harvested and SEAP activity was evaluated following the standard protocol.
Stricter gene regulation by targeting different loci simultaneously
In order to improve the efficiency of our gene regulation toolkit, we tried to target several loci upstream of the promoter of the reporter gene at once.   Thus, we designed crRNAs that are complementary to sequences on the SEAP reporter plasmid with different distances to the promoter ( Tab. 1 ).
HEK-293T cells were transfected with the CMVmin:SEAP reporter plasmid, dCas9-VP16 (iGEM standard), one or two RNA plasmids and a plasmid coding for Renilla   luciferase for an internal standard (to eliminate variabilities of different cell numbers or expression levels). The total amount of RNA plasmids is kept constant, so an increase of SEAP expression due to more available crRNA can be excluded.  By combining the targets EMX1 and T2 a higher SEAP activity than the sum of the single targets can be observed (Figure 13).
Figure 13: Effects of different target numbers
Activation of CMVmin:SEAP was evaluated by the usage of single targets and combined targets. Every target itself displays an activation of the minimal promoter. By combining two targets (EMX1 & T2), an even stronger activation is visible, that exceeds the additive effect of single plasmids.
HEK-293T cells were seeded at 65,000 cells in 24-well format. After 24 hours cells were transfected following standard procedure. After 48 h of expression supernatant was harvested and SEAP activity was evaluated following the standard protocol.
Discussion
From the few target sites we evaluated we could draw the conclusion that a target should be chosen like this:
The distance to the promoter should be kept at minimum.
Target sites with a GC content around 60 % should be preferred.
As we tested our target sequences with dCas9-VP16, stricly speaking, we can only make a statement about the best target sites for gene activation.
Moreover there may be other parameters that influence the effects of dCas9-VP16 on SEAP (e.g. the secondary structure of the crRNA). But for the   distance to the promoter Mali et al. [1] came to the same conclusion: Only the target site that had the shortest distance to the   promoter showed a high increase of activation.
By targeting different loci upstream of a promoter simultaneously the efficiency of transcription activation can be enhanced. While Cheng et al. [2] yielded an activation up to 8-fold by using at least three different targets in comparison to the highest   activation of a single target, we were able to achieve a three fold increase by using only two different targets.
References
(1) Mali P.,et al. (2013). Cas9 transcriptional activators for target specificity screening and paired nickases for cooperative genome  engineering. Nat Biotechnol.
(2) Cheng AW., et al. (2013). Multiplexed activation of endogenous genes by CRISPR-on, an RNA-guided transcriptional activator  system. Cell Res.
crRNA design tool
Enter Target Sequence
Choose Program
Which strand should be targeted?
non coding strand
coding strand
Submit
This tool helps you to design a crRNA-insert for dCas9 RNA plasmid: "uniCAS RNAimer" ( BBa_K1150034 ).
Using this tool you do not have to do this on your own. Just insert the desired target sequence and you get all different oligo possibilities and their positions. The oligos contain overhangs which fit to this plasmid's BbsI-overhangs and are ready to use.
The two different target possibilities are the coding and non-coding strand, depending on the desired target sequence.
a) For repression of gene transcription by targeting the coding strand the oligos must be designed as follows:
Search at your desired target sequence for a CCN (reverse complement of the PAM sequence) at the coding strand.
Extract the following (3') 30 nucleotides.
Extract the reverse complement.
Add AAAC at the 5' end and GT at the 3' end. This will be your fist oligo.
Take the sequence from step 2 and add TAAAAC at the 5' end. This will be your second oligo.
b) For repression of gene transcription by targeting the non-coding strand the oligos must be designed as follows:
Search at your desired target sequence for a NGG (the PAM sequence) at the coding strand.
Extract the 30 nucleotides before (5') the PAM sequence.
Extract the reverse complement.
Add TAAAAC at the 5' end. This will be your second oligo.
Take the sequence from step 2 and add AAAC at the 5' end and GT at the 3' end. This will be your first oligo.
Advice: To prevent BioBrick cloning from failing, avoid illegal iGEM restriction sites within the crRNAs.
(the oligos are designed analog to: Cong, L., et al. (2013). Multiplex Genome Engineering using CRISPR/Cas Systems. Science.)
Technical details about the crRNA design tool
Source Code
jQuery(document).ready( function() {  document.getElementById('button1').onclick = function() {    var TestVar = document.getElementById('form1').Sequenz.value;   var length = TestVar.length;     TestVar = TestVar.trim();     TestVar = TestVar.toUpperCase();   if(length & lt ; 32){alert("too short");return;}   var nonNuc = TestVar.search(/[^T|^A|^G|^C]/);   if (nonNuc != -1){alert("There are non nucleotide signs in your sequence.                  Please check your sequence!");return;                 }    for (Count = 0; Count & lt ; 2; Count++) {        if (document.getElementById('form1').rad[Count].checked){ break; }      }    if (Count==0) {       var pattern = /.GG/g;      var m;      var oligo4 = [];      var index1 = [];      var k = 0;     while (m = pattern.exec(TestVar)){       oligo4.push(m);       index1.push(m.index);       }            var oligo1 = [];      var index = [];            var i = 0;      for (var k = 0; k& lt ;index1.length; k++){       if (index1[k] & gt;  31){        oligo1[i] = TestVar.substring(index1[k]-30 ,index1[k]);        index[i] = index1[k]-30;        i++;        }       }          if (oligo1 == null ) {       alert("There are no matches found!");return;       }           for ( var k = 0; k& lt ;oligo1.length; k++){       oligo1[k] = oligo1[k].substring(0,30);       }          var oligo2 = oligo1.slice();      var index = oligo1.slice();             for( var k=0; k& lt ;oligo1.length; k++ ) {       index[k] = TestVar.indexOf(oligo1[k]);       }          for( var k=0; k& lt ;oligo2.length; k++ ) {       oligo2[k] = oligo2[k].split("").reverse().join("");         oligo2[k]=oligo2[k].replace(/A/gi,"X");         oligo2[k]=oligo2[k].replace(/T/gi,"A");         oligo2[k]=oligo2[k].replace(/X/gi,"T");         oligo2[k]=oligo2[k].replace(/G/gi,"Y");         oligo2[k]=oligo2[k].replace(/C/gi,"G");         oligo2[k]=oligo2[k].replace(/Y/gi,"C");         oligo2[k]="TAAAAC" + oligo2[k] ;       }           for( var k=0; k& lt ;oligo1.length; k++ ) {       oligo1[k]="AAAC"+ oligo1[k] +"GT";       }        var j = 0;      for( var k=0; k& lt ;oligo2.length; k++ ) {       TestVar = TestVar.substring(0,index[k]+30+j) +                                 TestVar.substring(index[k]+30+j,index[k]+33+j).fontcolor("red") + TestVar.substring(index[k]+33+j,TestVar.length);       j = j+25;       }    } else {        var pattern = /CC./g;      var m;      var oligo4 = [];      var index1 = [];      var k = 0;           while (m = pattern.exec(TestVar)){       oligo4.push(m);       index1.push(m.index);       }     var oligo1 = [];      var index = [];             var i = 0;      for (var k = 0; k& lt ;index1.length; k++){       if (index1[k] & lt ; TestVar.length-33){          oligo1[i] = TestVar.substring(index1[k]+3 ,index1[k]+33);        index[i] = index1[k];        i++;        }       }       var oligo2 = oligo1.slice();          for( var k=0; k& lt ;oligo2.length; k++ ) {           oligo2[k]="TAAAAC" + oligo2[k] ;       }           for( var k=0; k& lt ;oligo1.length; k++ ) {       oligo1[k] = oligo1[k].split("").reverse().join("");       oligo1[k]=oligo1[k].replace(/A/gi,"X");         oligo1[k]=oligo1[k].replace(/T/gi,"A");         oligo1[k]=oligo1[k].replace(/X/gi,"T");         oligo1[k]=oligo1[k].replace(/G/gi,"Y");         oligo1[k]=oligo1[k].replace(/C/gi,"G");         oligo1[k]=oligo1[k].replace(/Y/gi,"C");       oligo1[k]="AAAC"+ oligo1[k] +"GT";       }            var j = 0;      for( var k=0; k& lt ;oligo2.length; k++ ) {       TestVar = TestVar.substring(0,index[k]+j) +                                   TestVar.substring(index[k]+j,index[k]+3+j).                                 fontcolor("red") + TestVar.substring                                (index[k]+3+j,TestVar.length);       j = j+25;       }       }    var oligo2rev = oligo1.slice();   for( var k=0; k& lt ;oligo1.length; k++ ) {      oligo2rev[k] = oligo2[k].split("").reverse().join("");    }    TestVar = TestVar.replace(/& lt ;/g,"\n & lt ;");        var w = window.open("blank.html", "blank", "toolbar=yes, location=yes,                         directories=no, status=no, menubar=yes, scrollbars=yes, resizable=no,                          copyhistory=yes, width=1000, height=700");   w.document.writeln("& lt;h2 & gt; The inserted sequence is: & lt ;/h2 & gt; ");   w.document.writeln(TestVar);   w.document.writeln("& lt ;h1 & gt; possible crRNAs are: & lt ;/h1 & gt; & lt ;div & gt; ");   w.document.writeln("& lt ;table & gt; ");   w.document.writeln("& lt ;tr & gt; & lt ;td & gt; ");    for (i=0; i& lt ;oligo2.length; i++ ) {            w.document.writeln("& lt ;h3 & gt;  Oligo number: " +                          (parseInt(i, 10) + parseInt(1, 10)) + " was found at position " +                          (parseInt(index[i], 10) + parseInt(1, 10))  + "& lt ;/h3 & gt; ");        w.document.writeln("& lt ;p & gt; Oligo1: " + oligo1[i] +                          "& lt ;/p & gt; & lt ;p & gt; Oligo2: " + oligo2[i] + "& lt;                          /p & gt; ");          w.document.writeln("& lt ;pre & gt; Oligo1: " + oligo1[i] +                          "& lt ;br & gt;                                      ||||||||||||||||||||||||||||||||& lt ;br & gt; Oligo2:     " +                          oligo2rev[i] + "& lt ;/pre & gt; ");           }    w.document.writeln("& lt ;/td & gt; ");     w.document.writeln("& lt ;td & gt; ");     w.document.writeln("& lt ;/td & gt; & lt ;/tr & gt; & lt ;/table & gt; ");     w.document.writeln("& lt ;/div & gt; ");      w.document.close();      }  });
Retrieved from " http://2013.igem.org/Team:Freiburg/Project/crrna "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
