"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg/Project/1
From 2013.igem.org
Home
Highlights
Project
Abstract & Introduction
Targeting
Effectors
Effector Control
Modeling
Truncation
uniBAss
uniBOX
Manual
Application
Notebook
Targeting
Effectors
Effector Control
Modeling
uniBAss
Standardization
Material and Methods
BioBricks
Our BioBricks
Favorite BioBricks
Improvement of a Registry part
Sharing of our BioBricks
Safety
Human Practice
Expert opinions
Ethics
Safety & Security
Politics
Society
Education
Art
Facebook / Media
Team
News
Members
Exolab-Events
Statistics
Photo Gallery
Our Partners
Collaborations
Attributions
Thanks to
Abstract & Intro
Abstract
Introduction
CRISPR/Cas
Outlook
Targeting
Effectors
Effector Control
Modeling
Truncation
uniBAss
uniBOX
Manual
Application
Introduction
Abstract
Our team developed a universal toolkit, termed uniCAS, that enables customizable gene regulation in mammalian cells. Therefore, we engineered the recently understood and highly promising CRISPR/Cas9 system. The regulation is based on the RNA-guided Cas9 protein, which allows specific targeting of DNA sequences. Our toolkit comprises not only a standardized Cas9 protein, but also different effector domains for efficient gene activation or repression. We further engineered a modular RNA plasmid for easy implementation of RNA guide sequences. As an additional feature, we established an innovative screening method for assessing the functionality of our uniCAS fusion proteins. Single genes and even whole genetic networks can be modified using our uniCAS toolkit. We think that our toolbox of standardized parts of the CRISPR/Cas9 system offers broad application in research fields such as tissue engineering, stem cell reprogramming and fundamental research.
Click here for a multilingual project description
Introduction
Introduction to our project
The enormous amount of gene regulation, signal transduction and metabolic pathways gives us a slight idea about what complexity of life really means. This astonishing complexity is one of the most crucial issues to understand, if we want to understand life itself.
Many approaches have been investigated and given us insights into these biological pathways, resulting in more and more elaborate tools. Solutions for the treatment of systemic diseases like cancer or the fine-tuning of cellular processes remain challenging. There is a need for a powerful tool, which enables powerful regulation of whole genetic networks and yet is simple to use. In the last years there have been several approaches which were able to alter gene expression, but they have the flaw that they are only targeting one gene at a time and do not allow flexible modification of entire genomes.
In the last 8 months, our team has been planning, developing and building up a universal toolkit, which enables the regulation of complex networks. A toolkit that allows the simple and flexible regulation of multiple genes by only one protein.
Figure 1: Our uniCAS toolkit
The uniCAS toolkit provides 3 different effectors , 2 methods & 1 effector control ! By using our toolkit it is possible to efficiently activate or repress gene expression. We also provide devices for controlling our effectors by light. For multiple targeting different DNA sites can be addressed simultaneously by using our RNAimer plasmid.
Try working with uniCAS and receive custom-tailored instructions for your desired gene regulation experiment by using our Manual Tool .
The great potential harbored by the CRISPR/Cas9 system provided a great basis for us to modify it in a way that allows multiple, sequence specific gene regulation in mammalian cells.
The CRISPR/Cas9 system in short:
This CRISPR/Cas9 system is based on the interaction of three acting components: tracrRNA, crRNA and Cas9 protein. The crRNA detects complementary invasive DNA, tracrRNA enables the interaction of crRNA and Cas9. The RNA-Cas9 complex binds respective DNA and Cas9 cleaves the DNA sequcence specifically.
We used a human codon optimized Cas9 , mutated the remaining nickase function and standardized the 160 kDa protein. The result: The DNA binding protein dCas9 which can be guided specifically to any DNA target site due to complexation with crRNA (CRISPR RNA) and tracrRNA (transactivating crRNA). By fusing different effector domains to dCas9 we built fusion proteins for gene regulation in mammalian cells.
Figure 2: Engineering of the CRISPR/Cas9 system
The nuclease function of Cas9 has been mutated. The dCas9 is still able to bind the guiding RNAs and therewith can be directed towards every requested sequence. By fusing effectors to the dCas9, gene regulation of specitic loci can be engineered.
For gene regulation we used the trans-activator VP16 , the repressor-domain KRAB and the histone methyltransferase G9a . These effectors were able to efficiently change the expression rate of a reporter protein SEAP (secreted alcaline phosphatase) and the endogenous signal protein VEGF (vascular endothelial growth factor).
In order to allow for control of gene regulation in a time- and space-dependent manner, we designed various control elements. We engineered and tested light induction systems and also tried to activate the expression upon chemical stimuli .
As dCas9 is a relatively large protein consisting of 1367 amino acids with a molecular weight of 160 kDa, it influences expression rates and toxicity. We designed several truncated versions of dCas9 with the aim to minimize its size and increase the expression rate while retaining the RNA/DNA binding ability.
Last but not least we established a novel ELISA-based method to assess the binding efficiency of the dCas9 proteins: The uniCAS Binding Assay “ uniBAss ”. uniBAss was shown to be an innovative tool for the characterization of the binding capacity of our dCas9 fusion proteins and the truncated dCas9 versions. The method offers the possibility for high throughput implementation and technological expansion to DNA-binding proteins other than dCas9 such as TAL effectors (TALEs) and zinc fingers (ZFNs).
Background: The CRISPR/Cas9 system
Hidden as an uncharacterized E. coli locus for more than 15 years [1] , Barrangou et al. identified the CRISPR (Clusterd Regularly Interspaced Short Palindromic Repeats) array as a previously unknown adaptive prokaryotic immune system [2] . Almost half of all prokaryotes make use of this defense mechanism against unselective uptake through natural transformation, phage DNA transduction or horizontal gene transfer by conjugation. Invasive DNA or even RNA can be specifically recognized and efficiently cleaved [3] . This unique feature results from the interaction of non-coding RNAs and CRISPR associated (Cas) proteins. [4] , [5] . From a wide range of known CRISPR subtypes we used CRISPR type II b of S. pyogenes.
The recognition and degradation of invasive DNA by CRISPR/Cas type II occurs in three steps (see Fig. 2):
Acquisition: Invasive DNA is recognized via a protospacer adjacent motif (PAM) – the sequence NGG. A short sequence downstream of the PAM sequence is then integrated into the host CRISPR array and is termed spacer. Spacer sequences transcribe for CRISPR RNAs(crRNAs) which help to cleave sequence-specific invasive DNA. These sequences are located between short palindromic repeats, which are neccessary for the functionality of the crRNAs.
Expression/Transcription: The Cas9 endonuclease is expressed. CRISPR array is then transcribed and processed  by RNAse III into crRNAs. These contain the complementary spacer sequence and the direct repeat sequence. The crRNA guides the Cas9 protein specifically to invasive DNA sequences. Furthermore trans-activating crRNAs (tracrRNA) are transcribed and bind to the direct repeat part of the crRNA [6] . The tracrRNA is necessary for the formation of a Cas9-RNA complex.
Interference: Repeatedly invading DNA, which has been integrated into the CRISPR locus, is detected by the RNA-protein complex and cleaved by Cas9.
Figure 3: The CRISPR/Cas9 system
First step is the aquisition of invasive DNA sequences into the CRISPR array. Next step is the transcription of crRNA and tracrRNA. Together they form an RNA hybrid that interacts with Cas9 and enables the protein to bind and cleave invasive DNA in the interference step.
Outlook
Our toolkit opens up the possibility to deal with a wide range of yet unadressed scientific questions in the near future, including the field of systems biology and complex metabolic engineering approaches. Other advantages cover monetary and logistic aspects, as the only component for modification lies indeed within the crRNA itself. In turn, this can be ordered as two corresponding forward and reverse primers - see Team Freiburg's easy crRNA Design Tool . Financial, logistic and human ressources stay at a minimum using the uniCas toolkit for gene regulation. Accordingly, CRISPR/Cas9 systems have already been established for many model organisms, including Saccharomyces cerevisiae, Caenorhabditis elegans, Arabidopsis thaliana, Drosophila melanogaster, Danio rerio and Mus musculus.
The system could possibly be used for tissue engineering and stem cell reprogramming. Simultaneous regulation of the expression of different pathway enzymes may enhance industrial purposes - e.g. for the production of certain chemical compounds such as amino acids, biofuels or therapeutic drugs in eukaryotic cells. Therefore, stably transfected cells would be of interest.
Moreover, not only metabolic pathways could be targeted. Developmental Biology studies the embryonic patterning and organ formation. By transient dCas9-G9a interaction with different stem cell factor expressions, possibilities for better understanding and modeling of organism development arise. Induced pluripotent stem cells, for example, are yet made responsible for causing mutagenesis or immunogenicity [9] . Regenerative medicine still deals with basic understanding of reprogramming most cell types and may therefore gain benefits from efficient simultaneously acting dCas9-regulators.
Complex networks, as for instance can be found in cancer cell differentiation, migration and metabolism may be targeted. Frequently, transcriptional master regulators of house-keeping functions and signaling pathways, such as p53, show down- or upregulation in certain tumors [10] . From a speculative point of view, one could therefore suggest to perform sophisticated dCas9-fusion protein high-throughput transcriptome analysis of cancer-related promoters. Targeting should then first be obtained in cancer cell lines with a multiple crRNA plasmid library. Consequently, promising combinations of crRNAs might then be investigated more precisely and maybe, on a long-term perspective, lead to future therapeutic relevance.
References
(1) Ishino, Y., et al. (1987). Nucleotide Sequence of the iap Gene in Escherichia coli. Journal of Bacteriology 169, 5429-5433.
(2) Barrangou, R., et al. (2007). CRISPR provides acquired resistance against viruses in prokaryotes. Science 315, 1709-1712.
(3) Marraffini, L., and Sontheimer, E. (2008). CRISPR interference limits horizontal gene transfer in staphylococci by targeting DNA. Science 322, 1843-1845.
(4) Jansen, R., et al. (2002). Identification of genes that are associated with DNA repeats in prokaryotes. Molecular Microbiology 43, 1565-1575.
(5) Makarova, K., et al. (2011). Evolution and classification of the CRISPR-Cas systems. Nat Rev Microbiol 9, 467-477.
(6) Jinek, M., et al. (2012). A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity. Science 337, 816-821.
(7) Cong, L., et al. (2013). Multiplex genome engineering using CRISPR/Cas systems. Science 339, 819-823.
(8) Mali, P., et al. (2013). RNA-guided human genome engineering via Cas9. Science 339, 823-826.
(9) Yamanaka, S., et al. (2012). Induced Pluripotent Stem Cells: Past, Present and Future. Cell Stem Cell 10, 678-684.
(10) Kaneshiro, K., et al. (2007). An integrated map of p53-binding sites and histone modification in the human ENCONDE regions. Genomics, 177-188.
Retrieved from " http://2013.igem.org/Team:Freiburg/Project/1 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
