<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Notebook_lab_light"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Notebook/lab_light">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Notebook/lab_light&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_light&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_light&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Notebook/lab_light" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Notebook/lab light</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3></DIV><DIV id="menu"><UL id="navbar"><LI style="padding-left:15px"><A href="https://2013.igem.org/Team:Freiburg">Home</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Highlights">Highlights</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Introduction </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/induction">Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></LI></UL><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna">Targeting</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/method">uniBAss</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation">Standardization</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/protocols">Material and Methods</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/BioBricks">Our BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/favorite_parts">Favorite BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/improvement">Improvement of a Registry part</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/sharing">Sharing of our BioBricks </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/experts">Expert opinions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety/safety">Safety &amp; Security</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/politics">Politics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/society">Society</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/education">Education</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/art">Art</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/Facebook">Facebook / Media</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/News">News</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team2">Members</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Exolab">Exolab-Events</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Statistics">Statistics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/gallery">Photo Gallery</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Attributions">Our Partners</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/Coop">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/attributions"> Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/thanks">Thanks to</A></LI></DIV><DIV id="sidebar"><DIV id="sidebar_content"><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna"> Targeting </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector"> Effector </A></P><P class="first_order"><A class="active" href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control </A></P><P class="second_order_note"><A class="active" id="link" href="https://2013.igem.org/Team:Freiburg/Notebook/lab_light"> Light </A></P><P class="third_order"><A href="#july"> July </A></P><P class="third_order"><A href="#august"> August </A></P><P class="third_order"><A href="#september"> September </A></P><P class="third_order"><A href="#oktober"> October </A></P><P class="second_order_note"><A id="link" href="https://2013.igem.org/Team:Freiburg/Notebook/lab_hormon"> Hormone </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling"> Modeling </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/method"> uniBAss </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation"> Standardization </A></P><P class="first_order"><A id="link" href="https://2013.igem.org/Team:Freiburg/protocols"> Material and Methods </A></P></DIV></DIV><DIV id="main_contant_note"><DIV id="h1">
Effector Control - Light
</DIV><DIV id="july"><P id="h2">
July
</P></DIV><H2>29.07.2013</H2><P> Plasmids that we need as templates were transformed. </P><H2>30.07.2013</H2><H3>Primers</H3><P>Primers arrived today. Primers were diluted 1:10.</P><H3>Minipreps</H3><P>Transformed bacteria from yesterday with our template plasmids were minipreped.</P><UL><LI> pKM018: 83 ng/µl</LI><LI> pKM220: 216 ng/µl</LI><LI> pKM115: 177 ng/µl</LI><LI> pKM006: 118 ng/µl</LI><LI> pSAM200: 213 ng/µl</LI><LI> pSW42: 134 ng/µl</LI><LI> pSW45: 243 ng/µl</LI><LI>pIG2004: 212 ng/µl</LI><LI>T405: 213 ng/µl</LI></UL><H3> Oligo Annealing of tetO crRNA </H3><UL><LI> Oligos tetO13 1. CC and tetO13 2. CC were annealed
				</LI><LI> Therefore 20 µM diluted Primes were heated on 98° C for 4 minutes and then cooled down.
			</LI></UL><H3> PCR list, program and composition</H3><UL><LI> aIG4102: CMV Promotor from template T405 - Primers oIG4100F+R - 589 bp </LI><LI> aIG4103: Cas9 from template pIG2004 - Primers oIG4101F+R - 4170 bp </LI><LI> aIG4104: PIF and backbone from template pKM022- Primers oIG4102F+R - 3287 bp</LI><LI> aIG4105: KRAB from template pIG2004 - Primers oIG4103F+R - 390 bp </LI><LI> aIG4106: PhyB and backbone from template pKM018 - Primers oIG4104F+R - 4922 bp </LI><LI> aIG4107: UVR8 and backbone from template pKM168 - Primers oIG4105F+R - 3921 bp </LI><LI> aIG4108: COP1 and backbone frome template pKM115 - Primers oIG4106F+R - 3972 bp </LI><LI> aIG4109: CRY2 from template pSW42 - Primers oIG4107F+R - 1591 bp </LI><LI> aIG4110: Backbone from template pKM018 - Primers oIG4108F+R - 2924 bp </LI><LI> aIG4111: Backbone with VP16 from template pKM018 - Primers oIG4109F+R - 3282 bp </LI><LI> aIG4112: CIB from template pSW45 - Primers oIG4110F+R - 593 bp </LI><LI> aIG4113: Backbone from template pKM018 - Primers oIG4111F+R - 2881 bp </LI><LI> aIG4114: CIB from template pSW45 - Primers !! oIG4110F+oIG4112R - 638 bp </LI></UL><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH>µl </TH><TH> Substance </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 200 ng </TD><TD> Template </TD></TR><TR><TD> 1 </TD><TD> Primer forward </TD></TR><TR><TD> 1 </TD><TD> Primer reverse </TD></TR><TR><TD> 4 </TD><TD> dNTPs </TD></TR><TR><TD> 1 </TD><TD> DMSO </TD></TR><TR><TD> 0,5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H2O </TD></TR></TBODY></TABLE></DIV><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH> Temperature [° C] </TH><TH> Time </TH></TR><TR><TD> 98 </TD><TD> 5 Min </TD></TR><TR><TD> 98 </TD><TD> 30s </TD></TR><TR><TD> 60 (or optimized) </TD><TD> 30 </TD></TR><TR><TD> 72 </TD><TD> 30-40/kb </TD></TR><TR><TD> 72 </TD><TD> 10 Min </TD></TR><TR><TD> 4 </TD><TD> hold </TD></TR></TBODY></TABLE></DIV><DIV id="${18:floatright}"><UL><LI> 18 cycles </LI></UL></DIV><H3> PCR results </H3><UL><LI> aIG4102: 60° C - WORKED - GelExtraction: 87,9ng/ul </LI><LI> aIG4103: 60° C - 1 of 4 WORKED, Annealing Temperature lower? - GelExtraction: 50,5ng/ul <B>Try 56°!</B></LI><LI> aIG4104: 60° C - DIDN'T WORK!!! Try with 58° - <B>Was done, has to be tested on gel.</B></LI><LI> aIG4105: 60° C - WORKED - GelExtraction: 70,1ng/ul I and 57,7ng/ul II </LI><LI> aIG4106: 60° C - <B>Was done, has to be tested on gel.</B></LI><LI> aIG4107: 60° C - ONLY SLIGHT BAND. TRY AGAIN! </LI><LI> aIG4108: 60° C - WRONG AMPLIFICATED BAND. <B>Try again with different temperature! </B></LI><LI> aIG4109: 60° C - WORKED! - GelExtraction: 176,4ng/ul </LI><LI> aIG4110: 60° C - <B>Was done, has to be tested on gel.</B></LI><LI> aIG4111: 60° C - WORKED! - GelExtraction: 84,4ng/ul </LI><LI> aIG4112: 60° C - <B>Wasn't done yet.</B></LI><LI> aIG4113: 60° C - <B>Was done, has to be tested on gel.</B></LI><LI> aIG4114: 60° C - <B>Wan't done yet.</B></LI></UL><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log Marker</P><P>aIG4102 (first 4) <B>worked</B>, 589 bp</P><P>aIG4105 (rest) worked, 390 bp</P></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>Roth Marker</P><P><B>aIG4103</B><B>only one PCR worked</B>, 4170 bp</P></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>Roth Marker</P><P><B>aIG4104</B> didn't work</P><P><B>aIG4111 worked</B>, 3282 bp</P></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log Marker</P><P><B>aIG4109</B> (first two bands) <B>worked</B>, 1591 bp</P><P><B>aIG4107</B> (next two bands) didn't work, should have been 3921 bp</P><P><B>aIG4108</B> (one band) didnt work. Wrong band was amplified, should have been 3972 bp - another band of aIG4108 can be tested on a gel, didn't have space on this one.</P></TD></TR></TBODY></TABLE></DIV><H2>31.07.13</H2><H3>PCR results</H3><UL><LI> aIG4103: 54°-64° C - Everything WORKED! </LI><LI> aIG4104: 58° C - Gave three bands, upper band is supposed to be right. Was confirmed on a second gel with charging the gel only with a small amount of DNA.</LI><LI> aIG4106: 60° C - Didn't work. <B> Try temperature gradient. </B></LI><LI> aIG4107: 54°-64° C - Everything Worked! But not very strong bands. </LI><LI> aIG4108: 54°-64° C - Didn't Work. <B> Try seperate PCRs. </B></LI><LI> aIG4110: 60° C - Didn't work. Tried temperature gradient 54°-59° C - Worked with 55° and better with every additional ° C.</LI><LI> aIG4112: 60° C - Worked!</LI><LI> aIG4113: 60° C - Worked!</LI><LI> aIG4114: 60° C - Worked!</LI></UL><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P> UPPER SIDE </P><P>aIG4103 (2 bands) - worked!</P><P> aIG4107 (2 bands) - worked! </P><P> aIG4106 (4 bands) - didn't work with 60° C, bands are too low! </P><P> LOWER SIDE </P> aIG4108 temperature gradient 54°-64° C - didn't work!</TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P> aIG4103 temperature gradient 54°-64° C - everything worked! </P></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P> aIG4105 (2bands) - worked! </P><P> aIG4104 (2 bands) - upper band is considered to be the right band! - Worked! </P><P> aIG4110 - didn't work with 60° C.</P></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P>aIG4107 temperature gradient 54°-64° C - worked, but only slight bands! </P></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P> aIG4108 (2bands) - didn't work with 60° C! </P><P> aIG4113 (2 bands) - Worked! </P><P> aIG4110 - didn't work with 60° C.</P></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P> aIG4110 (1 band) last column of temperature gradient (59° C) - worked! </P><P> aIG4112 (2 bands) - Worked! </P><P> aIG4114 - worked!.</P></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P> aIG4110 temperature gradient (54°-59° C) - worked from 55° C upwards! </P></TD></TR></TBODY></TABLE></DIV><DIV id="august"><P id="h2">
August
</P></DIV><H2>01.08.13</H2><H3>PCR results</H3><UL><LI> aIG4104: Was gelexed and size was confirmed on a gel with a smaller amount of DNA. - Worked!
				</LI><LI> aIG4106: Temperature gradient 54° - 59° C and excess of reverse primer oIG4104R - Worked! </LI><LI> aIG4108: Seperate PCRs were tried out. Therefore the PCR was performed with template and at a time only one primer. After these PCRs (1x with forward Primer (58° C) and 1x with reverse Primer (62° C)) both columns were mixed and another PCR was performed. - Worked, but two bands appeared, upper band is considered to be the right fragment. </LI></UL><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P> aIG4106 temperature gradient (54°-59° C) - worked! </P></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P> aIG4108 seperate PCRs - worked, but other amplified products can be seen, too! </P></TD></TR></TBODY></TABLE></DIV><H3>Gibsons</H3><P>   Fragments are mixed together to a total volume of 5µl and added to 15µl Gibson Master- Mix.</P><P> pIG4101: aIG4104- 0,83µl; aIG4103- 3,96µl; H2O- 0,21µl </P><P> pIG4103: aiG4107- 2,12µl; aiG4103- 2,88µl </P><P> pIG4105: aIG4110- 1,11µl; aIG4103- 3,51µl </P><P> pIG4106: aIG4111- 0,5µl;  aIG4112- 0,3µl, H2O- 4,2ul </P><P> pIG4107: aIG4113- 0,4µl;  aIG4105- 0,35µl; aIG4114- 0,35µl; H2O- 3,9µl </P><P> Gibson Mixes were transformed. </P><H3>Ligation of pIG4100</H3><H4>Digest for pIG4100</H4><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> DNA (pKM006 or oIG4102) ~ 1000 ng </TD></TR><TR><TD> 4 </TD><TD> Cut Smart Buffer </TD></TR><TR><TD> 1 </TD><TD> EcoRI HF </TD></TR><TR><TD> 1 </TD><TD> Nhe1 HF </TD></TR><TR><TD> Add to 40µl </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Temp.: 37°C </LI><LI> Incubation time: 1,5h </LI></UL></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P> Digested aIG4102 with 601 bp </P><P> Digested pKM006 with 6393 bp and 126 bp. </P><P> Both were gelexed: aIG4102 19 ng/µl and pKM006 40 ng/µl</P></TD></TR></TBODY></TABLE></DIV><H4>Ligation and Transformation of approach pIG4100</H4><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 1 </TD><TD> T4 Ligase </TD></TR><TR><TD> 2 </TD><TD> T4 Ligase buffer </TD></TR><TR><TD> 3 fold molar amount of insert </TD></TR><TR><TD> 2 </TD><TD> Backbone pKM006 </TD></TR><TR><TD> 1,3 </TD><TD> Insert aIG4102 </TD></TR><TR><TD> 13,7 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Temp.: RT </LI><LI> Incubation time: 15 minutes </LI></UL></DIV><P> After the ligation, 2,5 µl if this approach were transformed into E.coli and scratched out.</P><H2>02.08.13</H2><H3>Gibson &amp; Ligation results</H3><P> Colonies could be seen on every plate except for one of the two ligation plates. Clones were picked and scratched out -&gt; 51 Minis </P><H3> Gibson </H3><P> pIG4102: aIG4105- 0,4µl; aIG4106- 2,00µl; H2O- 2,6 µl </P><P> pIG4104: aiG4108- 3,2µl; aiG4105- 0,5µl; H2O- 1,3µl </P><P> Gibson Mixes were transformed. </P><H2>03.08.13</H2><H3> Gibson results </H3><P> Colonies could be seen on one of two plates, each. Colonies were scratched out for mini preps. Therefore we had clones for every construct. </P><H3> Minipreps </H3><P> 51 Minis - concentrations between 50 and 200 ng/µl. </P><H2>04.08.13</H2><H3> Minipreps </H3><P> 15 Minis. </P><H2>05.08.13</H2><H3> Test digest </H3><P> Today we made a test digest session of our 66 minis with following enzymes. </P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> Plasmid </TH><TH> Enzymes </TH><TH> Size </TH></TR><TR><TD> pIG4100 </TD><TD> EcoRI-HF &amp; EcoRV-HF</TD><TD><B>1571 bp</B> (Insert CMV 600bp) &amp;<B>5417 bp</B></TD></TR><TR><TD> pIG4101 </TD><TD> EcoRV-HF &amp; NotI-HF</TD><TD><B>1952 bp</B> (half the Cas9) &amp;<B>5456 bp</B></TD></TR><TR><TD> pIG4102 </TD><TD> EcoRI-HF &amp; NotI-HF </TD><TD><B>1966 bp</B> (with PhyB) &amp;<B>3274 bp</B></TD></TR><TR><TD> pIG4103 </TD><TD> BamHI-HF &amp; XhoI </TD><TD><B>5337 bp</B> (Cas9, UVR8) &amp;<B>2682 bp</B></TD></TR><TR><TD> pIG4104 </TD><TD> NotI-HF &amp; BamHI-HF </TD><TD><B>1451 bp</B> (with COP1) &amp;<B>2839 bp</B></TD></TR><TR><TD> pIG4105 </TD><TD> EcoRI-HF &amp; EcoRV-HF </TD><TD><B>3040 bp</B> (Half the Cas9 and half CRY2) &amp;<B>5529 bp</B></TD></TR><TR><TD> pIG4106 </TD><TD> NotI-HF &amp; XbaI </TD><TD><B>970 bp</B> (CIB, VP16) &amp;<B>2833 bp</B></TD></TR><TR><TD> pIG4107 </TD><TD> PstI-HF &amp; NotI-HF </TD><TD><B>976 bp</B> (CIB, KRAB) &amp;<B>2821 bp</B></TD></TR></TBODY></TABLE></DIV><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> ~ 100-200 ng </TD><TD> DNA </TD></TR><TR><TD> 1 </TD><TD> Cut Smart buffer </TD></TR><TR><TD> 0,5 </TD><TD> Enzyme 1 </TD></TR><TR><TD> 0,5 </TD><TD> Enzyme 2 </TD></TR><TR><TD> Add to 10µl </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Temp.: 37°C </LI><LI> Incubation time: 2h </LI></UL></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P> Upper bands: pIG4100, expected sizes: Plasmid 3 and 5 were send to GATC</P><P> Bottom bands: pIG4101, expected sizes: Plasmid 4 and 7 were send to GATC </P></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P> pIG4102, expected sizes, Plasmid 1 and 9 were send to GATC.</P></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P> Upper bands left side: pIG4104, expected sizes: Plasmid 1 and 4 were send to GATC</P><P> Upper bands right side: pIG4105, expected sizes: Plasmid 1 and 6 were send to GATC </P></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P> Upper bands: pIG4106, not expected sizes: Plasmid 3 was send to GATC</P><P> Bottom bands: pIG4107, expected sizes: Plasmid 1 and 4 were send to GATC </P></TD></TR></TBODY></TABLE></DIV><P> For the sequencing we used GATC Primers: CMV-F for pIG4100 and EBV-RP (binds in pSV40) for all other plasmids. </P><H3> Insertion of crRNA into RNA Plasmid </H3><OL><LI> First RNA Plasmid backbone was digested with BbsI for 3 hours. Enzyme was taken out of - 80° C.</LI><LI> Then it was put on a gel and gelexed. Concentration: 50,5ng/ug. Backbone from Hormone- Group: 8,7ng/ul
			</LI><LI> Ligation: Digested backbone from Hormone- Induction and self- digested backbone was used. Molar ratio of                        backbone : insert - 1 : 3. </LI><LI> Mix: Backbone; Insert, 1ul T4- ligase, 2ul T4-ligase-buffer, H2O up to a total volume of 20ul </LI><LI> Incubated for 15min at RT </LI><LI> Transformed and scratched out.
		</LI></OL><H2>06.08</H2><H3> Sequencing results</H3><P> Sequencing results were positive for pIG4101, pIG4102, pIG4103, pIG4105, pIG4106 and pIG4107 (sequenced with EBV-RP -&gt; SV40 Terminator). pIG4100 didn't have the CMV promotor as an insert (-&gt; new ligation) and pIG4104 didn't contain KRAB, although KRAB worked with pIG4102 (new Primers?)</P><H3> Re-Trafos and inoculation for midi-preps</H3><P> The positive Plasmids and already existing plasmids pKM018 and pKM115 were retransformed and put into 150 ml LB with 150 µl Ampicillin and incubated at 37° C over night. </P><H3> New Primers for pIG4104 </H3><P> New Primers were designed for pIG4104. </P><H3> Ligation of pIG4100 </H3><P> Ligation of pIG4100 was done again like on 31.07.13, only the ratio of backbone and insert was changed (3 µl backbone and 1,5 µl Insert). </P><DIV><TABLE class="gelpic"><TBODY><TR><TD><P>2-log marker</P><P> Left band: Backbone, Right band: Inser</P></TD></TR></TBODY></TABLE></DIV><H3> Result of Ligation of RNA Plasmid </H3><P> There were clones on every plate. They were scratched out for minipreps.</P><H3> New sequencing </H3><P> Plasmids were send to GATC with different primers. Cas9 was sequenced completely in pIG4101.7. Other pIG4104 were screened for insertion of KRAB.</P><H2> 7.8.2013 </H2><H3> Midipreps of pKM017; pIG4101.7 (?), pIG4102.1, pIG4103, pKM115, pIG4105.5, pIG4106.3, pIG4107.4 </H3><P> All midipreps were successfull </P><P> Concentrations: </P><P> pKM018: 474 ng/ul </P><P> pIG4101: 485 ng/ul </P><P> pIG4102: 528 ng /ul </P><P> pIG4103: 521 ng/ul </P><P> pKM115: 517 ng/ul </P><P> pIG4105.1: 492 ng /ul </P><P> pIG4106: 567 ng/ul </P><P> pIG4107: 476 ng /ul </P><H3> Minipreps of RNA Plasmid pIG4201 and pIG4202</H3><P> Minis were send to GATC. 3 of ligation with digested Backbone from Michi and tetO1 and 3 with tetO2. And 3 of ligation with newly digested Backbone from Gabi and tetO1 and 3 with tetO2.</P><H3> Result of Ligation of pIG4100</H3><P> There were clones on every plate. They were scratched out for minis.</P><H2> 8.8.2013 </H2><H3> Sequencing of RNA Plasmid </H3><P> 1 of the 6 pIG4201 and 4 of the 6 pIG4202 were positiv with inserted crRNA.</P><H3> Sequencing Order </H3><P> Sequencing of pIG4107_2 and pIG4107_3 with Primer EBV-RP </P><H3> Minis of pIG4100</H3><P> 12 Minis were prepped. </P><H3> Re-Trafo for inoculation for Midi Preps</H3><P> The following plasmids were retransformed: pIG1000, pIG4201, pIG4202 </P><H2> 9.8.2013 </H2><H3> Test digest of pIG4100 </H3><H2> 10.8.2013 </H2><H3> Seeding of CHO cells </H3><P> CHO- cells were seeded: 70.000 cells per well on 24 well- plates in HTS-medium </P><H2> 11.8.2013 </H2><H3> Transfection of CHO cells </H3><H4>Protocol for 24- well plates: </H4><P> 1. Mix 40 µl Opti-MEM + 1.5 µl PEI-solution in a 1.5 ml Eppi </P><P> 2.Prepare in another Eppi 0.5 µg of the DNA of interest  </P><P> 3.Add DNA to former Eppi, vortex thoroughly and incubate 15 min at RT  </P><P> 4.Resuspend solution gently and spread them drop-wise to the cells in the dish  </P><P> 5.Swing the dish in axis-form  </P><P> 6.Change of media 5h post transfection  </P><H2> 12.8.2013 </H2><H3> PCB- treatment of CHO-cells and illumination </H3><P> Cells are treated with HTS-medium containing 15uM of phycocyanobilin </P><P> illumination started at 16.15. with 20uE light- quantity  </P><P> plate 1 with activating red light (660nm)  </P><P> plate 2 with inctivating far-red light (740nm), control  </P><H3> construction of pIG4104 </H3><P> pKM115 was digested with BamHI and BssHII for 4h. KRAB was amplified from pIG2004 with oIG4103FII and oIG4103RII  </P><P> Subsequently the fragments were assembled via Gibson-cloning and transformed into E.coli </P><H2> 12.8.2013 </H2><H3> Gibson results </H3><P> Cell-colonies are picked and plated out anew </P><P> A mistake in the cloning strategy was discovered, new primers are designed </P><H2> 13.8.2013 </H2><H3> Gibson results </H3><P> CHO- cells are illuminated </P><H2> 14.8.2013 </H2><H3> Seeding of CHO-cells </H3><P> CHO- cells were seeded: 70.000 cells per well on 24 well- plates in HTS-medium </P><H3> SEAP-measurements of illuminated cells </H3><P> The results are all negative except for the positive control (-&gt; see transfection Nr.6 of 11.8.2013) </P><P> The interaction with the cri-RNA may be responsible. The experiment is repeated with new target- sites  </P><H2> 15.8.2013 </H2><H3> Transfection of CHO-cells for red light and UV tests </H3><P> The wiki- transfection protocoll is used:  </P><P> 1. Mix 40 µl Opti-MEM + 1.5 µl PEI-solution in a 1.5 ml Eppi </P><P> 2.Prepare in another Eppi 0.5 µg of the DNA of interest  </P><P> 3.Add DNA to former Eppi, vortex thoroughly and incubate 15 min at RT  </P><P> 4.Resuspend solution gently and spread them drop-wise to the cells in the dish  </P><P> 5.Swing the dish in axis-form  </P><P> 6.Change of media 5h post transfection  </P><P> The following pattern is transfected in 24- well plates: </P><H2> 16.8.2013 </H2><H3> UV-b and red-/ far red light illumination of transfected cells from 15.8.2013 </H3><P> Cells for red- light illumination are illuminated with 20uE; 660nm or 720nm for 24h. Illumination is started at 14.20  </P><P> Cells for UV-b light illumination are illiminated with 5uE; 311nm for 24h  </P><H2> 17.8.2013 </H2><P> SEAP-measurements taken from illuminated- and untreated cells. </P><P> the following protocoll was obeyed for the whole process </P><LI>200 µl of cell supernatant were transfered form the 24-well plate into one 96-well plate (round bottom)</LI><LI>Incubation for 30 min at 65 °C.</LI><LI>Centrifugation for 1 min at 1250 g.</LI><LI>100 µl of SEAP buffer were filled in the wells of a 96 well plate (flat bottom).</LI><LI>Addition of 80 µl supernatant from the round bottom plate to each well with SEAP buffer.</LI><LI>Addition of 20 µl pNPP (substrate).</LI><LI>96 plate was immediately placed in the blade reader.</LI><P>Spectroscopic measurement every minute for 120 times (2h); wavelenght 405 nm.</P><P> numbers 1-6 are first six rows of the UV-b light approach -&gt; look at transfections from 15.8.2013. numbers 7-12 are rows 1-6 of the red-light transfections from 15.8.2013 </P><P> the system has to be improved, the controls showed the desired effect, but very weak. In the following the expression of Cas9- constructs has to be checked and improved. The tetO cri-RNA showed an insufficient binding pattern. New target sites for the plasmid have to be construct 
                    
               
               </P><H2> 20.8.2013 </H2><H3> Construction of new cri-RNAs </H3><P> 6 new cri-RNAs are designed and ordered at sigma-aldrich. One new cri-RNA is provided by Max Stamnitz. All new cri-RNAs target pKM006! </P><H3> Cloning of pIG4100; pIG4104; pIG4106; pIG4107 </H3><P> Primers for new cloning- strategies have arrived. </P><LI> pIG4100: pKM006 is digested and the CMV-promotor from pAcGFP1 is amplified via PCR. </LI><LI> pIG4104: pKM115 is digested with BamHI and BssHII. KRAB is amplified via PCR from pIG2001 </LI><LI> pIG4106: pKM018 is digested with EcoRI and NotI. CIB is amplified from pSW45 </LI><LI> pIG4107: pIG4102 is digested with EcoRI and NotI. CIB is amplified from pSW45 </LI><H2> 21.8.2013 </H2><H3> Cloning of pIG4100; pIG4104; pIG4106; pIG4107 </H3> Products from digestions and PCRs are purified via gel run in 1% agarose and gelexed  The digest of pKM115 for pIG4104 has not worked and is repeated executing the same protocol. <P> Parts for pIG4100, pIG4106, pIG4107 are Gibson cloned and subsequently transformated into competent &quot;TOP10&quot; E.coli- cells </P><H2> 22.8.2013 </H2><H3> Cloning of pIG4100; pIG4104; pIG4106; pIG4107 </H3> Clones of Gibsons are picked and plated out. Later to be picked and test- digested. For each plasmid 2 samples were sent to GATC for sequencing. <P> Gibson- cloning of pIG4104 is repeated with new digest and PCR-fragments. The constructs are subsequently transformated into competent &quot;TOP10&quot; E.coli- cells </P><H3> Transfection control: Is Cas9-VP16 expresse? How Strong? Is it toxic? </H3><P> HEK, HeLa and CHO- cells are seeded in 6-well plates </P><H2> 23.8.2013 </H2><H3> Cloning of pIG4104 </H3> Colonies are picked and plated out for mini and test- digest on 24.- or 25.8.13 <H3> Transfection control: Is Cas9-VP16 expresse? How Strong? Is it toxic? </H3><P> different DNA- concentrations are used for transfection in different cell lines. pIG1000 containing Cas9-VP16 is used. As a already tested sample, pIG8004 containing Cas9-G9a is used as positive- control. The following DNA- concentrations are transfected: 2ug; 3ug; 4ug; 6ug. One sample is tested with and addition of 0,5% DMSO into the medium attempting to boost the expression </P><P> The following pattern is transfected </P><H2> 24.8.2013 </H2><H3> Oligo annealing </H3><P> Targets T2, T3, T4, T5, T6, T7 and Max's random target were annealed.</P><H3> Ligation of new RNA plasmid </H3><P> Digested pIG3010 (50,1 ng/µl) and the annealed oliges were ligated with 3 molar amount of Insert. </P><H2> 25.8.2013 </H2><H3> Result of ligation </H3><P> Colonies were picked for mini preps.</P><H2> 26.8.2013 </H2><H3> Minipreps of targets </H3><P> Minipreps were sent for sequencing </P><H2> 27.8.2013 </H2><H3> Result of sequencing </H3><P> T2, T5 and T6 were positive. </P><H3> Seeding of cells for UV light activation experiment </H3><H2> 29.8.2013 </H2><H3> Transfection of CHOs for UV light activation</H3><H2> 30.8.2013 </H2><H3> Light start </H3><P> UV light was started at 15.30 o clock (two plexi glass panes) </P><H2> 31.8.2013 </H2><H3> SEAP assay </H3><P> SEAP assay of UV light activation was performed. </P><DIV id="september"><P id="h2">
September
</P></DIV><H2> 03.9.2013 </H2><H3> Oligo annealing </H3><UL><LI> 5µl Oligo forward 100 µM to get 20 µM concentration</LI><LI> 5 µl Oligo reverse 100 µM to get 20 µM concentration</LI><LI> 10 µl NEB buffer 2 </LI><LI> 30 µl ddH2O </LI></UL><P> Mix was heated to 98° C, Heatblock was turned off and Oligos were cooled down </P><H3> Digest of pIG3010 </H3><UL><LI> 10 µl pIG3010 </LI><LI> 1 µl BbsI from -80° C</LI><LI> 5 µl NEB buffer 2.1 </LI><LI> up to 50 µl ddH2O </LI></UL><P> Digestion was put on a gel and purified (10 ng/µl). </P><H3> Ligation of Target 3,4 and VEGF and transformation</H3><P> Digested pIG3010 (10 ng/µl) and the annealed oliges were ligated with 3 molar amount of Insert. </P><H2> 04.9.2013 </H2><H3> Result of Ligation </H3><P> Colonies were picked </P><H3> Gibson of pIG4104 </H3><P> Wrong primers were used for the last pIG4104 plasmid (was seen with the last sequencing). Therefore we made a new Gibson approach. </P><H2> 05.9.2013 </H2><H3> Minipreps of T3, T4 and VEGF (plates) and pIG4104 (fluid culture)</H3><P> Minipreps were send for sequencing (Primers for RNA Plasmid (in pIG3010) BGH-reverse and EBV-RP for pIG4104.) </P><H2> 06.9.2013 </H2><H3> Sequencing results </H3><P> pIG4104-3 was positive, T3, T4 and VEGF RNA plasmids were negative. </P><H3> New Ligation of T4 target into standardized RNA plasmid</H3><P> Procedure was done by standard protocol.</P><H3> Seeding of CHO cells</H3><P> 4 24-well plates were ceeded with 65000 CHO cells per well.</P><H2> 07.9.2013 </H2><H3> Status of CHO cells </H3><P> CHO cells were all contaminated with bacteria again. Media was plated in a 10 cm dish. Result: No contamination here.</P><H3> seeding of HEK cells</H3><P> HEK cells were seeded for UV &amp; BL activation and repression experiments at around 19/19.30. 4 24-well plates and 65000 cells per well.</P><H2> 08.09.2013 </H2><H3> Transfection of HEKs for UV &amp; BL activation and repression</H3><P> HEK cells were transfected (standard protocol) at 19.30. Media was changed after 5 hours.</P><H3> T4 RNA plasmid</H3><P> 6 colonies were scratched out. </P><H2> 09.09.2013 </H2><H3> Minis of T4 </H3><P> 3 of 6 plasmids were sent for sequencing (1-3).
			</P><H3> Light start </H3><P> Blue light was started at 21.30 with 450 nm and unsure intesity (4 colour box, 655 intensity 

number). Dark control is in a light box without light on.</P><P> UV light was started at 20.30. Dark control is in the incubator (no UV light in the lab).</P><H2>10.09.2013</H2><H3>Sequencing results</H3><P> Sequencing results were positive for Bla Target 4. Bla Targets 2 and 3 were cloned by Philipp who 

got positive results, too. Plasmids were retransformed and Midis were inoculated.</P><H3>Seeding of HEK cells.</H3><P> 4 24 well plates were seeded with 65.000 HEK cells per well.</P><H3> Light stop </H3><H3><P> Lightning of UV and BL were stopped after about 24 h and supernatants were taken and frozen.</P></H3><H2>11.09.2013</H2><H3> Midis </H3><P> Midis of Bla target 2 (pIG4), Bla target 3 (pIG4, Bla target 4 (pIG4), EMX1 (pIG4 and VEGF4 

(pIG4) were prepped.</P><H3> SEAP of UV and BL activation and repression</H3><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 1: SEAP measurement for blue light activation. </B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 2: SEAP measurement for UV light activation.</B></TD></TR></TBODY></TABLE></DIV><P> Activation was okay, repression was in overflow.</P><H3> Transfection for Activation-VP16 and Repression-KRAB target test row</H3><P> In order to test which target we should use, we made test rows with different targets and 

different target combinations. Medium was changed after 5 h.</P><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 3: Transfection plan: Target test row Activation 1</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 4: Transfection plan: Target test row Activation 1 </B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 5: Transfection plan: Target test row repression 1. </B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 6: Transfection plan: Target test row repression 2. </B></TD></TR></TBODY></TABLE></DIV><H2>12.09.2013</H2><H3> SEAP of Activation VP16 test row</H3><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 7: SEAP measurement for target test row for Cas9-VP16.</B></TD></TR></TBODY></TABLE></DIV><H3> Seeding of HEK cells for light activation experiments</H3><H2>13.09.2013</H2><H3> SEAP of Repression KRAB test row and UV and BL repression </H3><P> Samples had to be diluted 1:10 because the constitutive CMV promoter is very strong. SEAP levels 

were still in overflow.</P><H3> Transfection for UV/BL/RL activation</H3><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 8: Transfection plan for blue light activation. plan</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 9: Transfection plan for red light activation.</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 10: Transfection plan for UVB light activation.</B></TD></TR></TBODY></TABLE></DIV><H2>14.09.2013</H2><H3> Light start for UV/BL/RL</H3><H2>15.09.2013</H2><H3> Light stop for UV and BL</H3><H3> SEAP of UV and BL activation</H3><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 11: SEAP measurement for blue light activation.</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 12: SEAP measurement for UV light activation.</B></TD></TR></TBODY></TABLE></DIV><H2>16.09.2013</H2><H3> Light stop for RL</H3><H3> SEAP for RL</H3><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 13: SEAP measurement for red light activation.
</B></TD></TR></TBODY></TABLE></DIV><H2>18.09.2013</H2><H3> New charge of medium for CHOs</H3><P> Medium was complemented with FCS, Pen Strep and Gentamicin.</P><H2>20.09.13</H2><H3>New CHO-K1 &amp; NIH cells</H3><P> We received CHO-K1 cells from a freshly thawed cell batch. Thank you Shima from AG Ulbrich! These 

cells were cultivated in DMEM.</P><P> We received NIH cells.</P><H2>24.09.13</H2><H3>Seeding of CHO-K1 (from Toolbox) cells</H3><P> 6 24 well plates were seeded with about 65.000 cells per well.</P><H2>25.09.13</H2><H3>Transfection for red/blue/UV light induction in CHO-K1 (from Toolbox)</H3><P> CHO-K1 cells were transfected for red, blue and UV light induced gene activation. Cells that were 

transfected for UV light were also transfected with Renilla. This was done for normalization, because UV light can kill the 

cells.</P><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 14: Transfection plan for blue light activation.</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 15: Transfection plan for red light activation.</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 16: Transfection plan for UV light activation.</B></TD></TR></TBODY></TABLE></DIV><P> HTS medium was changed after 5 hours.</P><H3>Seeding of NIH cells</H3><P> 2 24 well plates were seeded with NIH cells (65.000 cells per well).</P><H2>26.09.13</H2><H3>Transfection for UV light induction in NIH cells</H3><P> Transfection plan was the same as for CHOs on 25.09.</P><H3>Light start for red/blue/UV light in CHO-K1 cells (from Toolbox)</H3><P> 1 plate was placed in UV light (311 nm). 1 plate was placed in a red light box (660) after PCB 

treatment. 1 plate was placed in a far-red light box (740 nm) after PCB treatment. One plate was placed in a blue light box 

(460 nm).</P><H2>27.09.13</H2><H3> Light start for UV light in NIH cells </H3><P> 1 plate of transfected NIH cells was placed in UV light.</P><H3> Light stop for UV (CHO-K1 from toolbox) &amp; blue light (CHO-K1) cells </H3><P> 200 µl of supernatant of each well were removed and heated. After this the 96 well plate was 

frozen (-20° C). SEAP measurement was done on the next day.</P><H2>28.09.13</H2><H3> SEAP measurement of UV (CHO-K1 from toolbox) &amp; blue light (CHO-K1) cells</H3><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 17: SEAP measurement for blue light induced activation (CHO-K1)</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 18: SEAP measurement for UVB light induced activation (CHO-K1).</B></TD></TR></TBODY></TABLE></DIV><H3> Light stop for red light (CHO-K1 from toolbox) and UV light (NIH)</H3><P> 200 µl of supernatant of each well were removed and heated. SEAP measurement was done according 

to standard protocol.</P><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 19: SEAP measurement of red light induced activation (CHO-K1)</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 20: SEAP measurement for UVB light induced activation (NIH).</B></TD></TR></TBODY></TABLE></DIV><H2>29.09.13</H2><H3> Seeding of CHO-K1 from AG Ulbrich and NIH cells</H3><P> 6 24 well plates with CHO-K1 were seeded and 6 24 well plates with NIH cells were seeded.</P><H2>30.09.13</H2><H3> Transfection of CHO-K1 &amp; NIH cells - red, blue and UV light</H3><P> Alltogether 12 24 well plates were transfected.</P><H4> Transfection plans </H4><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 21: Transfection plan: Blue light activation.</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 22: Transfection plan: Red light activation.</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 23: Transfection plan: UVB light activation. </B></TD></TR></TBODY></TABLE></DIV><H4> transfection calculation </H4><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 24: Transfection calculation for blue light activation.</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 25: Transsfection calculation 2 for blue light activation. </B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 26: Transfection calculation 1 for red light activation.</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 27: Transfection calculation 2 for red light activation. </B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 28: Transfection calculation 1 for UVB light activation. </B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 29: Transfection calculation 2 for UVB light activation </B></TD></TR></TBODY></TABLE></DIV><DIV id="oktober"><P id="h2">
October
</P></DIV><H2>01.10.13</H2><H3> Light start </H3><P> Both transfected cell lines were illuminated: 1 CHO plate &amp; 1 NIH plate with red light 660 nm, 1 

CHO plate &amp; 1 NIH plate with blue 465 nm and 1 CHO plate &amp; 1 NIH plate with UV light 311 nm. For every cell line and light 

system there was a dark control plate (red light's dark control is far red light of 740 nm because the system is reversible) 

</P><H2>02.10.13</H2><H3> Light stop </H3><P> Light was stoped for UV and blue light illumination (after 24 hours) (CHO and NIH cells). 200 µl 

supernatant were removed from each well for SEAP measurements. SEAP measurement was done according to standard protocol.
			</P><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 30: SEAP measurement for blue light activation in CHO-K1 cells.</B>
							HIER DANN ALLES ANDERE
						</TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 31: SEAP measurement for blue light activation in NIH cells.</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 32: SEAP measurement for UVB light activation in CHO-K1 cells.</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 33: SEAP measurement for UVB light activation in NIH cells.</B></TD></TR></TBODY></TABLE></DIV><H2>03.10.13</H2><H3> Light stop </H3><P> Light was stoped for red light illumantion. 200 µl supernatant were removed from each well for 

SEAP measurements. SEAP measurement was done according to standard protocol.
			</P><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 34: SEAP measurement for red light activation in NIH cells.</B></TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 35: SEAP measurement for red light activation in CHO-K1 cells.</B></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Freiburg/Notebook/lab_light">http://2013.igem.org/Team:Freiburg/Notebook/lab_light</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Notebook/lab_light" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Notebook/lab_light" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_light&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_light&amp;oldid=343457" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>