"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg/parts/favorite parts
From 2013.igem.org
Home
Highlights
Project
Abstract & Introduction
Targeting
Effectors
Effector Control
Modeling
Truncation
uniBAss
uniBOX
Manual
Application
Notebook
Targeting
Effectors
Effector Control
Modeling
uniBAss
Standardization
Material and Methods
BioBricks
Our BioBricks
Favorite BioBricks
Improvement of a Registry part
Sharing of our BioBricks
Safety
Human Practice
Expert opinions
Ethics
Safety & Security
Politics
Society
Education
Art
Facebook / Media
Team
News
Members
Exolab-Events
Statistics
Photo Gallery
Our Partners
Collaborations
Attributions
Thanks to
Our Favorite BioBricks
No. 1: BBa_K1150020 - uniCAS Activator
Figure 1: Construct design of dCas9-VP16 ( BBa_K1150020 ).
dCas9 was linked via a 3 amino acid linker to the 5’ end of the sequence coding for the transactivation domain VP16. To ensure nuclear localization of the construct, a nuclear localization signal (NLS) was fused to both ends of dCas9-VP16. For detection of expression the fusion protein was tagged with a HA-epitope coding sequence. Its expression was set under control of the CMV promoter and BGH terminator.
Our uniCAS Activator is a fusion protein consisting of dCas9 and Virus Protein 16 (VP16) for sequence-specific transactivation of a desired target locus. VP16 is a transcription factor encoded by the UL48 gene of Herpes simplex virus-1 (HSV-1). With the help of the transactivation domain of VP16, transcription factors are recruited and the pre-initiation complex can be built. We cloned this transactivation domain behind dCas9 in order to activate any gene of interest by placing this construct upstream of a promotor region. By co-transfecting our RNA plasmid [ uniCAS RNAimer ( BBa_K1150034 )] which includes the tracrRNA and the separately integrated, desired crRNA, the dCas9-VP16 specifically binds to the targeted DNA sequence.
For the exact constellation of the associated device take a look at Figure 1.
Figure 2: Results of the SEAP activation with dCas9-VP16 under control of the CMV promoter using different crRNAs.
To quantify the activation properties of dCas9-VP16 the amount of SEAP expression was measured and divided by the expression level of the luciferase Renilla (internal standard). Each sample was measured in biological triplicates.  Bright green columns reflect the negative controls while dark green ones reflect the different samples. The fold induction above each column is related to the basic SEAP expression level of the pRSet (junk DNA) control. T3+4 crRNAs are transcribed from one RNAimer plasmid ( BBa_K1150034 ) while T3 & T4 crRNAs are transcribed from two independent RNA plasmids.
dCas9-VP16 was able to induce gene expression of our tested reporter gene SEAP up to a 10-fold higher amount using a single target locus [EMX1]. Transfecting more target loci simultaneously, the expression increased up to 29-fold [EMX1 and T2]. For these two loci this effect was more than additive - so activation was higher than the addition of the single locus results (activation was two times higher than the additive effect); see Figure 2.
...for more information about activation click here .
No. 2: BBa_K1150024 - uniCAS Histone Modifier
Figure 3: CMV:dCas9-G9a ( BBa_K1150024 )
dCas9 was fused via a 3 amino acid linker to G9a. The resulting fusion protein is flanked by NLS sequences and tagged by an HA-epitope. The CMV promoter and BGH terminator were chosen to control gene expression.
This device combines the dCas9 protein with the SET-domain of the murine histone methyltransferase G9a. dCas9 enables not only sequence specific, but also multiple targeting of any desired DNA sequence. Hence, coupling of dCas9 to the effector G9a allows specific methylation of histone 3 at lysin 9 (Figure 3).
Such methylations are a hallmark of gene repression. One interesting fact about histone modification is the capability to spread the activity state over the surrounding chromatin via reader proteins. So the information of e.g. "repressed state" can, once specifically introduced, be propagated over a whole locus.
We included the G9a histone methyltransferase to the uniCAS toolkit for specific histone methylation. With the device BBa_K1150024 , consisting of dCas9 and G9a, we were able to decrease the endogenous VEGF expression in HEK-293T cells about 50%, depending on the locus targeted.
Chromatin remodelling, resulting in repression of endogenous genes, is possible by fusing the histone methyltransferase G9a to dCas9 (Figure 4).
dCas9-G9a is our most efficient repressive device !
...for more information about epigenetic repression click here .
Figure 4: Endogenous, stable repression by dCas9-G9a
HEK-293T cells have been transfected with the BBa_K1150024 device and the RNAimer plasmid containing the different crRNA target sites for the endogenous VEGF locus. 12 hours after transfection the medium was changed and 24 hours after medium change we harvested the supernatant and performed VEGF measurments by ELISA. As a control that the repressive effect of our proteins is not based on the sterical block of the transcription by the protein, we tested against the catalytic inactive dCas9. So every detectable effect is due to the G9a targeted to this locus.(n=3, p<0.05 is marked by asterisks)
No. 3: BBa_K1150034 - uniCAS RNAimer
Figure 5: RNAimer ( BBa_K1150034 )
As we are engineering the CRISPR/Cas9 system we needed two small, non-coding RNAs that mediate the interaction between our proteins and the desired DNA loci. Our RNA plasmid contains the tracrRNA and a site where the desired crRNA can be inserted. Both RNAs are driven by different human RNA polymerase III promoters, H1 and U6. The combination of multiple crRNAs can be easily done by digestion with enzymes of prefix and suffix and ligation of these parts and the RNAimer backbone according to iGEM standard assembly.
As dCas9 requires special RNAs for binding to the DNA, we designed a RNA plasmid containing the structure giving tracrRNA and the DNA binding crRNA. The crRNA is responsible for sequence specific DNA-binding of the entire RNA-protein complex.
The crRNA can be easily introduced to the plasmid by digesting the backbone with BbsI and ligation of a double stranded oligo. Two of these RNA plasmids (with different crRNAs) can be fused using the iGEM BioBrick system. This way it is possible to get two or more crRNAs on one plasmid (Figure 5).
We were able to show that a RNAimer plasmid ( BBa_K1150034 ) containing two crRNAs is as effective as co-transfecting two crRNAs on single plasmids (Figure 6). Furthermore the RNAimer enables multiple targeting, which means to use several crRNAs for one locus. Even for multiple targeting different RNAimers can be combined using the iGEM BioBrick system. And as the results show, multiple targeting is more effective concerning the influence the effector has on gene expression (Figure 7).
Figure 6: RNAimer in comparison to two RNA plasmids
HEK-293T cells were transfected with CMV:dCas9-VP16 ( BBa_K1150020 ) and either two RNA plasmids (right; T3 & T4) or one RNAimer plasmid (left; T3+4), containing the same crRNAs (T3, T4). The supernant was taken and SEAP activity measured according to standard protocol. The bars represent the mean of biological triplicates with standard deviation.
Figure 7: Effect of multiple targeting using the RNAimer plasmid
Mammalian HEK-293T cells were transfected with CMV:dCas9-VP16 and different crRNA targets for the same locus. After 48h the supprnant was harvested for SEAP measurement according to standard protocol. The bars represent the mean of biological triplicates with standard deviation. The last bar shows the activation of the SEAP gene with two different crRNAs target the CMV:dCas9-VP16 ( BBa_K1150020 ) construct to the same locus.
Retrieved from " http://2013.igem.org/Team:Freiburg/parts/favorite_parts "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
