<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Project_induction"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Project/induction">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Project/induction&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Project/induction&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Project/induction&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Project/induction" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Project/induction</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3></DIV><DIV id="menu"><UL id="navbar"><LI style="padding-left:15px"><A href="https://2013.igem.org/Team:Freiburg">Home</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Highlights">Highlights</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Introduction </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/induction">Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></LI></UL><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna">Targeting</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/method">uniBAss</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation">Standardization</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/protocols">Material and Methods</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/BioBricks">Our BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/favorite_parts">Favorite BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/improvement">Improvement of a Registry part</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/sharing">Sharing of our BioBricks </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/experts">Expert opinions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety/safety">Safety &amp; Security</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/politics">Politics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/society">Society</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/education">Education</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/art">Art</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/Facebook">Facebook / Media</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/News">News</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team2">Members</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Exolab">Exolab-Events</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Statistics">Statistics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/gallery">Photo Gallery</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Attributions">Our Partners</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/Coop">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/attributions"> Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/thanks">Thanks to</A></LI></DIV><DIV id="main"><DIV id="sidebar"><DIV id="sidebar_content"><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Intro </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/effector"> Effectors </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/induction" class="active"> Effector Control </A></P><P class="second_order"><A href="#light">Light </A></P><P class="second_order"><A href="#hormon">Hormone </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></P></DIV></DIV><DIV id="main_contant"><P id="h1">
Effector Control
</P><P>We engineered the CRISPR/Cas9 system that is relying on a protein-RNA-DNA interaction to generate a DNA binding protein. For this we used a catalytically inactive Cas9, the <B>dCas9</B>. </P><P> After generating a series of effectors using the CRISPR/Cas9 system, we aimed to control our system via <B>external stimulation</B>. We developed two approaches for induction, either on <B><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/induction#hormon">hormone stimulus</A></B> or on <B><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/induction#light">light stimulus</A></B>.
</P><P> The CRISPR/Cas9 system requires two small, non-coding RNAs for the binding to DNA. To test this system we performed functional tests, using the <B><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#rnaimer">RNAimer</A></B>. This plasmid codes for the RNAs that are necessary for the binding ability of dCas9.
</P><P> In those tests with this systems, SEAP (Secreted Embryonic Alkaline Phosphatase) assays were performed. This is a secreted enzyme that can easily be measured.</P><TABLE id="headline_pic"><TBODY><TR><TD style="width:1000px;"><P id="h2">
Light
</P></TD></TR></TBODY></TABLE><P id="h3">
Introduction
</P><P> Up to the present day, various networks have been developed to control cellular behavior. Small molecules such as antibiotics are often used as an inducer <SPAN id="refer"><A href="#(1.1)"> [1]</A></SPAN>, <SPAN id="refer"><A href="#(1.2)"> [2]</A></SPAN>.
However these systems often lack reversibility, exact spatial control and show toxic and pleiotropic side effects <SPAN id="refer"><A href="#(1.3)"> [3]</A></SPAN>, <SPAN id="refer"><A href="#(1.4)"> [4]</A></SPAN>. These problems can for instance be solved using optogenetic tools to control gene expression <SPAN id="refer"><A href="#(1.5)"> [5]</A></SPAN>, <SPAN id="refer"><A href="#(1.6)"> [6]</A></SPAN> or other desired interactions <SPAN id="refer"><A href="#(1.7)"> [7]</A></SPAN> e.g. activation of protein splicing <SPAN id="refer"><A href="#(1.8)"> [8]</A></SPAN>. The precise control of gene expression is of high value for basic research or further applications such as gene therapy. </P><P> The field of optogenetics made great progress within the past few years. Between the discovery of new mechanisms like the PhyB-PIF interaction <SPAN id="refer"><A href="#(1.9)"> [9]</A></SPAN> and the practical use in synthetic biology <SPAN id="refer"><A href="#(1.10)"> [10]</A></SPAN> only few years have passed. The recently emerged CRISPR/Cas system which is established in mammalian cells allowed first applications in 2011 <SPAN id="refer"><A href="#(1.11)"> [11]</A></SPAN>. The aim of this study is to connect both fields to form a new system for the use in mammalian cells. The combination of dCas9 with optogenetic tools has potential use for multiple applications enabling accurate genome editing. </P><P> Biotic and abiotic influences have major impact on evolution. All living beings are able to sense the state of their surroundings to a certain extent. Sensing light is crucial for most organisms.  For plants, light serves not only as orientation, but mainly as an energy source. 
Plants have developed mechanisms to perceive the intensity, periodicity and wavelength of light. The perception of light is directly linked to the metabolism. A unique set of photoreceptors allows them to sense exact quantities and qualities of light <SPAN id="refer"><A href="#(1.12)"> [12]</A></SPAN>.  </P><P> Our universal toolkit will allow you to use nearly the full spectrum of light as induction signal. All proteins are derived from <I>Arabidopsis thaliana</I>. Phytochrome B is responsible for red-light induction. Cryptochrome 2 controls blue-light induction and the UV-light receptor UVR8 is responsible for induction of UV-B light.  </P><DIV id="redlight"><P id="h3">
Effector Control by red-light stimulus
</P></DIV> One of the best described phytochromes is Phytochrome B (PhyB). In <I>Arabidopsis thaliana</I>, PhyB predominantly controls seedling establishment. PhyB is a protein with a molecular mass of 125 kDa which is predominantly located in the cytoplasm. Illumination with red light (660 nm wavelength) leads to binding of Phytochrome Interaction Factor 6 (PIF6). The interaction can be surpressed by illumination with far-red light (740 nm wavelength). <P> A framework using the unique properties of PhyB and PIF6 was designed to enable the recruitment of different effector proteins to any desired gene locus. The system was designed to work as described: Fusion proteins linking dCas9 to PIF6 were designed, as well as fusion proteins linking the interaction partner of PIF6 âPhyBâ to several effector molecules such as VP16 or KRAB. PhyB is a protein with a molecular mass of 125 kDa that is predominantly located in the cytoplasm. Its structure can be roughly divided in two parts: The N-terminal part, regulatory and photosensory, and the C-terminal part, simply regulatory. The N-terminal part is covalently bound to phytochromobilin. Phytochromobilin is a linear tetrapyrrol that undergoes conformational changes upon radiation of either red- or far red-light when bound to the phytochrome moiety. This conformational change is responsible for the photosensory properties of PhyB <SPAN id="refer"><A href="#(1.13)"> [13]</A></SPAN>, <SPAN id="refer"><A href="#(1.14)"> [14]</A></SPAN>.

</P><DIV><CENTER><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 1: Plasmid-cards of red-light effector-control plasmids </B>
             For gene-activation via red-light stimulus, we constructed a fusion of dCas9 linked to PIF6 for exact gene targeting <A id="link" href="http://parts.igem.org/Part:BBa_K1150025"> (BBa_K1150025)</A>. To enable gene activation, we linked PhyB to a VP16-activation domain <A id="link" href="http://parts.igem.org/Part:BBa_K1150026"> (BBa_K1150026)</A>. Both constructs contain a Nuclear Localization Sequence, necessary for usage in mammalian cells. Additionally the expression of dCas9-PIF6 can be quantified using the attached HA-tag.      
            </TD></TR></TBODY></TABLE></CENTER></DIV><P> The dCas9 protein has built a complex with crRNA and tracrRNA, additionaly it is covalently linked to PIF6. After expression, it will bind to its target locus. A red-light stimulus is recruiting the VP16 protein linked to PhyB and thereby initiating expression of a target gene. Red light does not carry any pleiotropic or toxic side effects. Red light additionally has no effects on other light-absorbing particles, that may be disrupted by illumination. The dCas9-protein is able to target even multiple gene-loci with only one transfection. Targeting one locus with multiple crRNAs may enhance the desired effect; either inhibition or induction of target genes. </P><DIV><CENTER><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 2: Mechanism of red-light induction </B>
The fusion protein consisting of dCas9 and PIF6 binds to its target locus. Illumination with red-light leads to the recruiting of the activating fusion-protein PhyB-VP16.
</TD></TR></TBODY></TABLE></CENTER></DIV><P id="h4"> Experiments </P><P> The experiments for the induction with red light were executed according to the following pattern:	
Day 0: Seeding of cells. 65,000 cells were seeded into a 24-well plate. 	
Day 1: Transfection of cells. The transfection was executed using PEI at a PEI/DNA-ratio of 3:1. Medium was changed 5 h post transfection.   	
Day 2: 1 h prior to illumination with red light, 660 nm, or far-red light, 740 nm, (20 ÂµE for both), the medium was renewed with fresh media containing 15 ÂµM phycocyanobilin  	
Day 3: Illumination (48 h total).  	
Day 4: The desired assay (SEAP-measurement) can be performed. </P><P id="h4"> Results </P><CENTER><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD colspan="2"><B>Figure 3: SEAP levels for HEK293T-cells.</B>
The HEK293T-cells were treated as described in the &quot;Experiments&quot; section above. The bars on the left side show SEAP levels of the negative control that was transfected without the targeting plasmid <A id="link" href="http://parts.igem.org/Part:BBa_K1150034"> (BBa_K1150034, RNAimer)</A>. The bars on the right side show SEAP levels of constructs transfected with target plasmid. Red bars show cell-samples illuminated with activating red-light, white bars show samples illuminated with inactivating far-red light. Error bars show standard deviation of biological triplicates. An increasing SEAP-level after illumination can be observed if target plasmid is present.
</TD></TR></TBODY></TABLE></DIV></CENTER>

HEK293T-cells were transfected with dCas9-PIF6, PhyB-VP16, the RNAimer plasmid with one target site (target 1) for the SEAP-reporter plasmid pKM611 and the reporter plasmid <SPAN id="refer"><A href="#(Supplements)"> pKM611</A></SPAN>. The cells were illuminated for 48 h with red light (660 nm) or inactivating far-red (740 nm) light. The data show absolute SEAP activity measured from 3 biological replicates. The &quot;no target&quot; control was transfected with an empty vector instead of the RNAimer target plasmid. The results show a 1.3-fold induction after illumination. 
The negative controls indicate that the used reporter plasmid containing the CMVmin promoter and the SEAP gene shows a high leaky SEAP expression. Due to this high background level, the induced activation of the light-system is only slightly observable. 
In order to improve the induction further experiments need to be performed. For example the usage of different cell-lines, other reporter systems could reduce the background level. More target sites and stronger activation domains (VP64, p65) could increase the inducible activation.

<P id="h3">
Effector control by UVB-light stimulus
</P><P>Another light system that we engineered is based on the induction by ultraviolet B light (UVB) with a wavelength of 311 nm. The main components are the UV resistance locus 8 (UVR8) and the WD40 domain of COP1. As COP1 stands for E3-ubiquitin ligase Constitutively Photomorphogenic 1 <SPAN id="refer"><A href="#(1.15)"> [15]</A></SPAN>, <SPAN id="refer"><A href="#(1.16)"> [16]</A><SPAN id="refer">. Both proteins are derived from <I>A. thaliana</I>.
UVR8 is a UVB light-responsive receptor that stays in its closed configuration without light stimulus. When exposed to UVB light the configuration changes to its open state and recruits COP1 <SPAN id="refer"><A href="#(1.6)"> [6]</A></SPAN>.</SPAN></SPAN></P><P>By combining our dCas9 protein and effector domains with these proteins we engineered a UVB light inducible gene regulation system.</P><DIV><CENTER><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 4: Plasmid-cards of UV-b light effector-control plasmids </B>
For gene-activation via UV-b light stimulus, we constructed a fusion of dCas9 linked to UVR8 for exact gene targeting <A id="link" href="http://parts.igem.org/Part:BBa_K1150029"> (BBa_K1150029)</A>. To enable gene activation, we linked COP1 to a VP16-activation domain <A id="link" href="http://parts.igem.org/Part:BBa_K1150030"> (BBa_K1150030)</A>. Both constructs contain a Nuclear Localization Sequence (NLS), necessary for usage in mammalian cells. Additionally the expression of dCas9-UVR8 can be quantified using the attached HA-tag.
</TD></TR></TBODY></TABLE></CENTER></DIV><P>The core domain of UVR8 was fused to our dCas9 protein. The other transcription factor consists of COP1 fused to the transactivation domain VP16 or repressor domain KRAB. When crRNA and tracrRNA bind to dCas9, the protein is able to bind complementary DNA. If the system is exposed to UV-B light (311 nm) the UVR8 receptor changes its conformation and recruits COP1. If dCas9  was fused to a transactivation domain like VP16 gene expression starts. Subsequently the gene of interest is activated.</P><DIV><CENTER><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 5: Mechanism of UV-B light induction </B>
The fusion protein consisting of dCas9 and UVR8 binds to its target locus. Illumination with UV-B light leads to recruitment of the activating fusion-protein COP1-VP16.
</TD></TR></TBODY></TABLE></CENTER></DIV><P id="h4"> Experiments </P><P> The experiments for the induction with UV-B light were executed according to the following pattern:	
Day 0: Seeding of cells: 65,000 cells were seeded into a 24-well plate. 	
Day 1: Transfection of cells: The transfection was executed using PEI at a PEI/DNA-ratio of 3:1. Medium was changed 5 h post transfection.   	
Day 2: Illumination with UV-B light: 5 ÂµE for 24 h  		
Day 3: The desired assay can be performed. 
(<A id="link" href="https://2013.igem.org/Team:Freiburg/protocols">Standard protocols</A>)
</P><P id="h4"> Results </P><CENTER><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD colspan="2"><B>Figure 6: SEAP levels for HEK-293T-cells.</B>
The HEK-293T-cells were treated as described in the &quot;Experiments&quot; section above. The bars on the left side show SEAP levels of the negative control that was transfected without the targeting plasmid <A id="link" href="http://parts.igem.org/Part:BBa_K1150034"> (BBa_K1150034, RNAimer)</A>. The bars on the right side show SEAP levels of constructs transfected with target plasmid containing 3 different target sites. Violet bars show cell-samples illuminated with activating UV-B light, white bars show samples that were kept in darkness. Error bars show standard deviation of biological triplicates. An increasing SEAP-level after illumination can be observed, if target plasmid is present. 
	</TD></TR></TBODY></TABLE></DIV></CENTER>

To test the UV light system HEK-293T cells were transfected with dCas9-UVR8, COP1-VP16, the RNAimer plasmid with three (target 1-3) target sites in front of the CMVmin promoter of the SEAP-reporter plasmid pKM611 and the reporter plasmid <SPAN id="refer"><A href="#(Supplements)"> pKM611</A></SPAN>. The cells were illuminated for 24 h with UV-B light (311 nm). The control was kept in darkness. The data show absolute SEAP activity taken from 3 biological replicates. The negative control was transfected with an empty vector instead of the RNAimer target plasmid. 
The results show an almost 2-fold induction after illumination. 
The negative controls without dCas9 target sites indicate that the used reporter plasmid has a highly leaking SEAP expression in HEK-293T cells. Due to this high background level, the induced activation by the light-system is only slightly observable. 
In order to improve the induction further experiments need to be performed, e.g. usage of different cell-lines, other reporter systems, more target sites and different effector proteins (VP64, p65). Other cell lines may also tolerate longer illumination times.

<P id="h3">
Effector control by blue-light stimulus
</P><DIV><P> The third light system we engineered is the blue light mediated induction. The system we used is based on dimerization of cryptochrome 2 (CRY2) and the cryptochrome-interacting basic-helix-loop-helix 1 (CIB1), both located in the nucleus.  
Cryptochrome 2 binds flavin and pterin on its conserved N-terminal photolyase homologe region. This interaction leads to the blue-light responsiveness of CRY2. So the blue light system requires no exogenous chromophore <SPAN id="refer"><A href="#(1.17)"> [17]</A></SPAN>, <SPAN id="refer"><A href="#(1.18)"> [18]</A></SPAN>. </P>
 Illumination with blue light (465nm) leads to interaction of CRY2 and CIB1.
These two components originate from <I>Arabidopsis thaliana</I>, and can be expressed in mammalian cells.
<P>
For our system we fused CRY2 to dCas9 and CIB1 to different effector domains. By fusing VP16 and KRAB to CIB1 we build a blue light responsive tool, enabeling activation or possibly repression of virtually any gene via blue light stimulus. </P><DIV><CENTER><TABLE class="imgtxt" width="600px"><TBODY><TR><TD><B>Figure 7: Plasmid cards of blue-light effector-control plasmids </B>
For gene-activation via blue-light stimulus, we constructed a fusion of dCas9 linked to CRY2 for exact gene targeting. To enable gene activation, we linked CIB1 to a VP16-activation domain. Both constructs contain a Nuclear Localization Sequence, necessary for usage in mammalian cells. Additionally the expression of dCas9-CRY2 can be quantified using the attached HA-tag.
</TD></TR></TBODY></TABLE></CENTER></DIV><P> When crRNA and tracrRNA bind to dCas9, the protein is able to bind complementary DNA. If the system is exposed to  blue light (465 nm) the Cry2 receptor recruits CIB with its fused transactivation domain VP16 which starts gene expression. Therefore the gene of interest is activated.</P><P id="h4"> Experiments </P><P>
Day 0: Seeding cells: 65,000 NIH/3T3 cells were seeded into 24-well plates. 
Day 1: The transfection was executed using PEI at a PEI/DNA-ratio of 3:1. Medium was changed 5 h post transfection. 
Day 2: Media change before light treatment, illumination with blue light (465 nm): 5 ÂµE for 24 h  
Day 3: The desired assay can be performed. </P><DIV><CENTER><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 8: Mechanism of blue-light induction </B>
The fusion protein consisting of dCas9 and CRY2 binds to its target locus. Illumination with blue light leads to recruition of the activating fusion-protein CIB1-VP16.
</TD></TR></TBODY></TABLE></CENTER></DIV><P id="h4"> Results </P><CENTER><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD colspan="2"><B>Figure 9: SEAP levels for NIH/3T3-cells.</B>
The NIH/3T3-cells were treated as described in the &quot;Experiments&quot; section above. The bars on the left side show SEAP levels of the negative control that was transfected without the targeting plasmid containing 4 different target sites <A id="link" href="http://parts.igem.org/Part:BBa_K1150034"> (BBa_K1150034, RNAimer)</A>. The bars on the right side show SEAP levels of constructs transfected with target plasmid. Blue bars show cell-samples illuminated with activating blue-light, white bars show samples that were kept in darkness. Error bars show standard deviation of biological triplicates. An increasing SEAP-level can be observed after illumination, if target plasmid is present. </TD></TR></TBODY></TABLE></DIV></CENTER>

To test the blue-light system NIH/3T3 cells were transfected with dCas9-CRY2, CIB1-VP16 and the RNAimer plasmid containing the four target sites (targets 1-4) binding in front of the CMVmin promoter of the SEAP-reporter plasmid pKM611 and the reporter plasmid <SPAN id="refer"><A href="#(Supplements)"> pKM611</A></SPAN>. The cells were illuminated for 24 h with blue-light (450 nm). The control was kept in darkness. The data show absolute SEAP activity taken from 3 biological replicates. The negative control was transfected with an empty vector instead of the RNAimer target plasmid. 
The results show an almost 5-fold induction after illumination. 
In general we could only observe lower absolute SEAP levels, but nevertheless a clear induction. This might be due to the lower transfection efficiency of NIH/3T3 cells and overall lower expression levels 
In return the background SEAP level in this cell line was quite low in contrast to the HEK-293T cells. Likewise to the other two systems, further optimization needs to be done. More target site or stronger effectors could improve this system.

<P id="h4">References</P><SMALL><DIV id="(1.1)"> (1) Weber, W., <I>et al.</I> (2002): Macrolide-based transgene control in mammalian cells and mice. Nature Biotechnology.<DIV id="(1.2)"> (2) Jakobus, K., <I>et al.</I> (2012): Synthetic mammalian gene networks as a blueprint for the design of interactive biohybrid materials. Chemical Society Reviews </DIV><DIV id="(1.3)"> (3) Yang, G., <I>et al.</I> (2013): Toxicity and adverse effects of Tamoxifen and other anti-estrogen drugs. Pharmacology &amp; Therapeutics.</DIV><DIV id="(1.4)"> (4) Ortiz, O., <I>et al.</I> (2013): Reversible and tissue-specific activation of MAP kinase signaling by tamoxifen in braf V637 ER T2 mice. Genesis. </DIV><DIV id="(1.5)"> (5) MÃ¼ller, K., <I>et al.</I> (2013a): A red/far-red light-responsive bi-stable toggle switch to control gene expression in mammalian cells. Nucleid Acids Research </DIV><DIV id="(1.6)"> (6) MÃ¼ller, K., <I>et al.</I> (2013b): Multi-chromatic control of mammalian gene expression and signaling. Nucleid Acids Research. </DIV><DIV id="(1.7)"> (7) Levskaya, A., <I>et al.</I> (2009): Spatiotemporal control of cell signalling using a light-switchable protein interaction. Nature. </DIV><DIV id="(1.8)"> (8) Tyszkiewicz, A.B., Muir, T.W. (2008): Activation of protein splicing with light in yeast. Nature Methods.</DIV><DIV id="(1.9)"> (9) Ni, M., <I>et al.</I> (1998): PIF3, a Phytochrome-Interacting Factor Necessary for Normal Photoinduced Signal Transduction, Is a Novel Basic Helix-Loop-Helix Protein. Cell. </DIV><DIV id="(1.10)"> (10) Shimizu-Sato, S., <I>et al.</I> (2002): A light-switchable gene promoter system. Nature Biotech. </DIV><DIV id="(1.11)"> (11) Jinek, M., <I>et al.</I> (2012): A Programmable Dual-RNA-Guided DNA Endonuclease in Adaptive Bacterial Immunity. Science.</DIV><DIV id="(1.12)"> (12) Lariguet, P. &amp; Dunand, C. (2005): Plant Photoreceptors: Phylogenetic Overview. Journal of Molecular Evolution.</DIV><DIV id="(1.13)"> (13) Essen, L.-O., <I>et al.</I> (2008): The structure of a complete phytochrome sensory module in the Pr ground state. PNAS. </DIV><DIV id="(1.14)"> (14) Quail, P.H. (2002): Phytochrome photosensory signalling networks. Nature Reviews Molecular Cell.</DIV><DIV id="(1.15)"> (15) Rizzini, L., <I>et al.</I> (2011): Perception of UV-B by the Arabidopsis UVR8 protein. Science 332. </DIV><DIV id="(1.16)"> (16) Favory, J.-J., <I>et al.</I> (2009): Interaction of COP1 and UVR8 regulates UV-B-induced photomorphogenesis and stress acclimation in Arabidopsis. The EMBO Journal 28, <I>591-600</I>. </DIV><DIV id="(1.17)"> (17) Kennedy, M., <I>et al.</I> (2010): Rapid blue-lightâmediated induction of protein interactions in living cells. Nature America.</DIV><DIV id="(1.18)"> (18) Bugaj, L., <I>et al.</I> (2012): Optogenetic protein clustering and signaling activation in mammalian cells. Nature America.</DIV><DIV id="(Supplements)"> Supplements <DIV id="(1.15)"><CENTER><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>Figure 10: Position of the target loci on the SEAP plasmid pKM611. </B>
The tested target loci are located in front of the SEAP promoter. Several distances and combinations of the related crRNAs were tested.
			</TD></TR></TBODY></TABLE></DIV></CENTER><DIV id="hormon"><P id="h2">
Hormone
</P></DIV><P id="h3">
Introduction
</P><P> The aim of this project was to control dCas9 (catalytically inactive CRISPR associated protein 9) activity by an external 4-OHT (4-hydroxytamoxifen) stimulus: Only when 4-OHT is applied, dCas9 should enter the nucleus. Thus, for a first try the Cas9 nickase was fused to the 4-OHT inducible receptor domain ERT2 (mutated hormone binding domain of the human estrogenic receptor). To facilitate the detection of the fusion protein an HA-tag and the fluorescent protein mCherry were added. </P><P> The native estrogenic receptor (ER) acts as an enhancer: Upon hormone binding this protein stimulates the transcription of specific genes. It mainly consists of a DNA and a hormone binding domain (HBD) <SPAN id="refer"><A href="#(1)"> [1]</A></SPAN>. 
Steroid receptors such as ER require chaperones (mostly Hsp90) to be functional: They open the hydrophobic hormone binding cleft and carry them after binding of the appropriate ligand with the help of immunophilins along the cytosceleton to the nucleus <SPAN id="refer"><A href="#(2)"> [2]</A></SPAN>. For the glucocorticoid receptor (GR) this interaction is crucial for nuclear import <SPAN id="refer"><A href="#(3)"> [3]</A></SPAN>. Even within the nucleus most steroid receptors are in complex with Hsp90 <SPAN id="refer"><A href="#(4)"> [4]</A></SPAN>. Without a ligand steroid receptors move continuously into and out of the nucleus. When the adequate hormone is bound GR localization becomes nuclear within a half life of about 4.5 min <SPAN id="refer"><A href="#(2)"> [2]</A></SPAN>. 
Tamoxifen is a competitive inhibitor of ER. For that reason it is widely used for treatment of breast cancer <SPAN id="refer"><A href="#(5)"> [5]</A></SPAN>. In the body of higher vertebrates tamoxifen becomes metabolized to 4-OHT <SPAN id="refer"><A href="#(6)"> [6]</A></SPAN>. This form is more potent in inhibiting the ER which leads to a stronger repression of estrogen-dependent cell growth <SPAN id="refer"><A href="#(7)"> [7] </A></SPAN>.
In 1993 Danielian et al. <SPAN id="refer"><A href="#(8)"> [8]</A></SPAN> mutated the murine ER, so that it is no more sensitive to estrogen, but becomes active after binding of tamoxifen. Littlewood et al. <SPAN id="refer"><A href="#(9)"> [9]</A></SPAN> took the HBD of this mutated ER and fused it to the c-Myc protein. Thus they were able to control Myc-induced proliferation and apoptosis. Later on the DNA recombinase Cre was fused with a more tamoxifen sensitive human HBD mutant named ERT2 <SPAN id="refer"><A href="#(10)"> [10]</A></SPAN>. Concerning the regulation 4-OHT rather than tamoxifen is preferred since it yields a higher effect with lower concentrations <SPAN id="refer"><A href="#(11)"> [11]</A></SPAN>. 
Besides the inhibition of the ER, tamoxifen can decrease the cellular ATP level via uncoupling of the mitochondrial membrane potential as well as the adenine nucleotide transferase concentration within the mitochondrial membrane. However the derivate 4-OHT is much less harmful <SPAN id="refer"><A href="#(7)">[7]</A></SPAN>. 

</P><P> According to Leone et al. <SPAN id="refer"><A href="#(12)"> [12]</A></SPAN> the mechanism of regulation of the ERT2-Cas9 fusion protein should be the following: Without 4-OHT the 

protein remains in the cytoplasm, due to the shielding of the NLS by Hsp90s. When 4-OHT is applied to the cell it diffuses through the cell membrane and 

binds to ERT2 (Figure 1 (1.) ). Thus the heat shock proteins should separate from ERT2, which allows the fusion protein to enter the nucleus, where 

Cas9 (when in complex with crRNA and tracrRNA) can target - and regulate - the desired genes (Figure 1 (2.) ). </P><CENTER><DIV><TABLE class="imgtxt" width="600px"><TBODY><TR><TD><B>Figure 1: Proposed mechanism</B>
Without (hydroxy)tamoxifen in the cell the fusion protein (Cas9, NLS &amp; ERT2) is in complex with Hsp90 chaperones, which should cover the NLS. After tamoxifen binding (1) to the HBD, the chaperones separate and the fusion protein will be imported into the nucleus (2).
			</TD></TR></TBODY></TABLE></DIV></CENTER><P id="h3">
Assembled plasmids
</P><P> Due to unknown sterical side effects, 5 constructs with different NLS and ERT2 positions were assembled (see also Figure. 2). As for first experiments only the subcellular localization should be investigated the Cas9 nickase was used: 
First of all an NLS and an ERT2 were fused to both ends of the fusion protein, as a Cas9-GFP fusion protein with an NLS on both sides was shown by Cong et al. <SPAN id="refer"><A href="#(13)"> [13] </A></SPAN> to be localized exclusively in the nucleus within HEK-293T cells. The Hsp90s which were acquired by ERT2 during the non induced state were thought to cover these NLSs and therefore inhibit the nuclear import of the fusion protein. (pIG3005) 
Secondly ERT2 was only fused to the N- or the C-terminal NLSs, since Cong et al. <SPAN id="refer"><A href="#(13)"> [13] </A></SPAN> showed a difference in nuclear import efficiency due to the position of the NLS. According to these studies the C-terminal NLS should have a higher impact on nuclear localization, but fusing ERT2 N-terminal leads to 

a tighter hormone mediated control <SPAN id="refer"><A href="#(14)"> [14]</A></SPAN>. (pIG3006: ERT2 only C-terminal &amp; pIG3007: ERT2 only N-terminal) 
Thirdly constructs only contain one NLS (with ERT2). This should reduce the amount of not induced nuclear import, while the size of the protein is lower 

than pIG3005. (pIG3008: NLS and ERT2 only C-terminal &amp; pIG3009: NLS and ERT2 only N-terminal) 
At last a control was assembled for every NLS variation: Cas9-mCherry without a NLS (pIG3001), NLSs fused to both ends (pIG3002) and one NLS fused to the C- (pIG3003) and to the N-terminus (pIG3004). </P><P> All planned plasmids could successfully be cloned through Gibson assembly. </P><CENTER><DIV><TABLE class="imgtxt" width="600px"><TBODY><TR><TD><B>Figure 2: Assembled fusion proteins</B>
All fusion proteins used for the hormone induction are pictured with all their essential parts.
pIG3001 - 4: controls; pIG3005 - 9: test constructs; HA: HA-tag; ERT2: hormone receptor domain; NLS: nuclear localization sequence.
			</TD></TR></TBODY></TABLE></DIV></CENTER><P id="h3">
Results
</P><P id="h4">Influence of the NLS on nuclear import</P><P> Fluorescent pictures taken with the confocal microscope clearly show that pIG3001 is mostly localized in the cytoplasm, whereas with pIG3002 - 4 transfected cells display a much stronger red fluorescence in the nucleus. It seems that this nuclear localization is even stronger for pIG3003 &amp; 4 (Fig. 3).
</P><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>Figure 3: Fluorescence pictures of HeLa cells expressing the control constructs</B>
Pictures were taken with a confocal fluorescence microscope. 
mCherry: fluorophor (part of the Cas9-ERT2 fusion protein); DRAQ5: DNA staining; pIG3001: Cas9-ERT2 fusion protein without a NLS; pIG3002: Cas9-ERT2 fusion protein with a NLS on both ends; pIG3003: Cas9-ERT2 fusion protein with a C-terminal NLS; pIG3004: Cas9-ERT2 fusion protein with a N-terminal NLS.
			</TD></TR></TBODY></TABLE></DIV>
This is confirmed by quantitative analysis of the mean fluorescence intensity: Figure 5 shows a slight increase in the ratio of nuclear to cytoplasmic 

fluorescence from pIG3002 to 3 &amp; 4, but within the error bars (standard deviation). The value of pIG3003 &amp; 4 is more or less the same. The intensity ratio 

of pIG3002 is 14 fold, the intensity ratio of pIG3003 &amp; 4 17 fold higher than the one of pIG3001. <P id="h4">Effects of 4-OHT induction on nuclear import</P><P> The red fluorescence of the Cas9-ERT2 fusion proteins is mostly localized in the nucleus both with 4-OHT and EtOH (negative control, as 4-OHT is 

dissolved in EtOH) treatment, a difference is not detectable as can be seen for pIG3007 &amp; 9 in Figure 4. </P><CENTER><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>Figure 4: Fluorescence pictures of HeLa cells expressing pIG3006 &amp; 8 (ERT2 at the  C-terminus)</B>
Pictures taken with a confocal fluorescence microscope were chosen as an example for the look of the cell expression of the Cas9-ERT2 fusion proteins.
mCherry: fluorophor (part of the Cas9-ERT2 fusion protein); DRAQ5: DNA staining; pIG3006: Cas9-ERT2 fusion protein with a NLS on both ends and an ERT2 at the C-terminus; pIG3008: Cas9-ERT2 fusion protein with a C-terminal NLS and ERT2.
The very bright, round red shapes are mostly dead cells.
			</TD></TR></TBODY></TABLE></DIV></CENTER><P> Quantitative analysis reveals a slight increase of the ratio of the mean nuclear red fluorescence intensity to the cytoplasmic upon 4-OHT treatment for 

pIG3005 (1.5-fold), pIG3007 (2-fold) and pIG3009 (1.3-fold), whereas this ratio drops upon 4-OHT treatment for pIG3006 (2-fold) and pIG3008 (1.6-fold), however still within the standard deviation. All values range between the positive and the negative controls (Fig. 5). </P><CENTER><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>Figure 5: Ratio of the mean intensity of nuclear and cytoplasmic fluorescence</B>
For each bar the mean intensities of nuclear and cytoplasmic red fluorescence of at least 7 cells on one picture taken by the confocal microscope were determined using ImageJ. This was done by selecting a region in the nucleus and in the cytoplasm of each cell. The figure shows the ration of nuclear to cytoplasmic fluorescence (background subtracted) with the standard deviation.
The pink bars represent the controls (no treatment): pIG3001 = negative control (no NLS); pIG3002 - 4 = positive controls (one or two NLSs, respectively). The purple bars represent the test constructs with either 0.1 % EtOH or 1 ÂµM 4-OHT treatment over night.
			</TD></TR></TBODY></TABLE></DIV></CENTER><P id="h4">Effects of 4-OHT on Cas9 functionality </P><P> As another aproach to regulate Cas9 activity, HEK-293T cells were transfected with a constitutive active SEAP plasmid, an RNA plasmid containing the tracrRNA and a crRNA, that is complementary to a sequence at the beginning of the SEAP coding sequence, and the Cas9-ERT2 fusion constructs, which contain two NLS. We hoped that the Hsp90s binding to ERT2 when there is no 4-OHT present would sterically hinder Cas9 from binding to the target DNA. 
Figure 6 only shows very little differences in SEAP expression between hormone and ethanol treatment (in both directions) but 1.5- to 3-fold repression when Cas9-ERT2 is transfected with a crRNA. </P><CENTER><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>Figure 6: Repression of SEAP expression</B>
The SEAP activities of technical triplicates were measured after the addition of a certain amount of pNPP (substrate of SEAP). The bars represent the mean with standard deviation.
pink: controls without Cas9-ERT fusion proteins; purple: tests with Cas9-ERT fusion proteins; 1/5: cotransfection of an RNA plasmid containing a crRNA targeting the SEAP coding region at the first 1/5; 1/10: cotransfection of an RNA plasmid containing a crRNA targeting the SEAP coding region at the first 1/10; -/-: no RNA plasmid transfected; 4-OHT: treatment with 4-OHT; EtOH: treatment with EtOH.
			</TD></TR></TBODY></TABLE></DIV></CENTER><P id="h4"> Toxicity of 4-OHT </P><P> In order to test if 4-OHT has some lethal side effects on HeLa cells, non transfected cells were treated with different 4-OHT concentrations (10 ÂµM, 1 ÂµM 

and 0.1 ÂµM), EtOH (1 % and 0.1 %) or none of them in two independent experiments for 18 or 24 h respectively. 
In the first experiment cells were observed by microscopy: All cells looked healthy, only the cells treated with 10 ÂµM 4-OHT had a quite compact shape, whereas all other had rather stretched shapes. 
With the second experiment the number of dead cells in the supernatant were determined. The well with untreated cells contained the lowest number of dead cells, the well with cells treated with 0.1 % EtOH the highest. Treatment with 4-OHT caused aslightly lower number of dead cells (Fig. 7). </P><CENTER><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 7: Number of dead cells in supernatant of HeLa cell culture after different treatments</B>
Cells were treated with 4-OHT or EtOH as shown in the figure for 18 h. Cells within the removed supernatant and 0.5 ml PBS of a washing step were counted and dead cells were calculated.
1 ÂµM = 1 ÂµM 4-OHT; 0.1 ÂµM = 0.1 ÂµM 4-OHT.
			</TD></TR></TBODY></TABLE></DIV></CENTER><P id="h3">
Discussion
</P><P id="h4">Influence of an NLS on nuclear import</P><P> The lack of an NLS leads to an almost exclusively cytoplasmic localization of the Cas9-mCherry fusion protein. The presence of an NLS increases the 

nuclear import about 14- to 17-fold. Surprisingly one NLS seems to show a greater effect than two of them, but this difference is not significant due to the 

overlapping standard deviations. The pictures may be misleading to this conclusion because of a different cell shape between pIG3002 and pIG3003/4 

which is caused by a different cell density. 
The observations that one NLS is at least as strong as two and that the N-terminal NLS is as strong as the C-terminal one do not agree with the results of 

Cong et al. <SPAN id="refer"><A href="#(13)"> [13]</A></SPAN>. They showed that two NLSs lead to the strongest nuclear import, whereas a C-terminal NLS has a lower effect and a N-terminal NLS 

causes almost no nuclear import. This divergence to our findings may be due to the fact that Cong et al. used HEK cells (instead of HeLa cells) and a Cas9-GFP 

fusion protein for their experiments. As GFP is derived from the fluorophor of <I>Aequorea victoria</I><SPAN id="refer"><A href="#(15)"> [15] </A></SPAN> and mCherry is a monomeric 

derivative of the fluorophor of <I>Discosoma sp.</I><SPAN id="refer"><A href="#(16)"> [16]</A></SPAN>, it should have a different tertiary structure, which may cause another 

exposure of at least the C-terminal NLS. </P><P id="h4">Effects of 4-OHT induction</P><P>None of the 5 test constructs showed a big difference in subcellular localization after treatment with 4-OHT or EtOH; all values range within the 

standard deviations (Figure 5). Thus no fusion protein could be termed hormone inducible. However due to the high standard deviations, the low amount of cells in the pictures and subjectively chosen mean fluorescence regions within the cells, the results can not be regarded as trustworthy. This could be an explanation for a stronger nuclear localization with EtOH treatment. 
The pictures of pIG3006 show a tendency of an increased nuclear localization after a 4-OHT treatment (Figure 5), whereas the quantitative analysis demonstrate the 

opposite. The problem may be caused by a wide variety of cells with different fusion protein distributions in case of the EtOH treatment. This fact is indicated by the high standard deviation. </P><P id="h4">Effects of 4-OHT on Cas9 functionality </P><P> Moreover the functionality of Cas9 could not be induced by hormones: The observed repression between Cas9 with crRNA and without crRNA  is independent of hormone addition. With and without 4-OHT there only is a very slight difference; in most samples the repression is even stronger with ethanol addition than with tamoxifen (Figure 7).

</P><P id="h4"> Toxicity of 4-OHT </P><P> 4-OHT in concentrations below 1 ÂµM does not seem to be toxic to HeLa cells. If it can be assumed that most dead cells will detach from the well, then 

the toxicity assay we performed shows that the number of dead cells does not increase with 4-OHT treatment in comparison to EtOH. It is more likely that 

the solvent EtOH has a negative effect on HeLa cells. To be sure about that, the assay would have to be repeated. 
The change in shape cells shown after treatment with 10 ÂµM 4-OHT may also occur because of a high cell density. </P><P id="h4">
References
</P><SMALL><DIV id="(1)">(1) Kumar V., <I>et al.</I> (1987). Functional domains of the human estrogen receptor. Cell. 51(6), <I>941-51</I>. </DIV><DIV id="(2)">(2) Pratt, W.B. &amp; Toft, D.O. (2003). Regulation of Signaling Protein Function and Trafficking by the hsp90/hsp70-Based Chaperone Machinery. Experimental Biology and Medicine 228, <I>111-33</I>. </DIV><DIV id="(3)">(3) Czar M.J., <I>et al.</I> (1997). Geldanamycin, a heat shock protein 90-binding benzoquinone ansamycin, inhibits steroid-dependent translocation of the glucocorticoid receptor from the cytoplasm to the nucleus. Biochemistry 36(25), <I>7776-85</I>. </DIV><DIV id="(4)">(4) Smith D.F. &amp; Toft D.O. (2008). Minireview: the intersection of steroid receptors with molecular chaperones: observations and questions. Mol Endocrinol. 22(10), <I>2229-40</I>. </DIV><DIV id="(5)">(5) Furr, B.J. &amp; Jordan, V.C. (1984). The pharmacology and clinical uses of tamoxifen. Pharmacol Ther. 25(2), <I>127-205</I>. </DIV><DIV id="(6)">(6) Borgna J.L. &amp; Rochefort H. (1981). Hydroxylated metabolites of tamoxifen are formed in vivo and bound to estrogen receptor in target tissues. J Biol Chem. 256(2), <I>859-68</I>. </DIV><DIV id="(7)">(7) Cardoso, C.M.P., <I>et al.</I> (2003). Comparison of the changes in adenine nucleotides of rat liver mitochondria induced by tamoxifen and 4-hydroxytamoxifen. Toxicology in Vitro 17, <I>663â70</I>. </DIV><DIV id="(8)">(8) Danielian, P.S., <I>et al.</I> (1993). Identification of Residues in the Estrogen Receptor That Confer Differential Sensitivity to Estrogen and Hydroxytamoxifen. Molecular Endocrinology 7 (2), <I>232-40</I>. </DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Freiburg/Project/induction">http://2013.igem.org/Team:Freiburg/Project/induction</A>&quot;</DIV></SMALL></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Project/induction" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Project/induction" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Project/induction&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Project/induction&amp;oldid=361423" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></SMALL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>