"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg/Notebook/lab repression
From 2013.igem.org
Home
Highlights
Project
Abstract & Introduction
Targeting
Effectors
Effector Control
Modeling
Truncation
uniBAss
uniBOX
Manual
Application
Notebook
Targeting
Effectors
Effector Control
Modeling
uniBAss
Standardization
Material and Methods
BioBricks
Our BioBricks
Favorite BioBricks
Improvement of a Registry part
Sharing of our BioBricks
Safety
Human Practice
Expert opinions
Ethics
Safety & Security
Politics
Society
Education
Art
Facebook / Media
Team
News
Members
Exolab-Events
Statistics
Photo Gallery
Our Partners
Collaborations
Attributions
Thanks to
Targeting
Effector
Activation
Epigenetics
Repression
April
May
June
July
September
Effector Control
Modeling
uniBAss
Standardization
Material and Methods
Effector - Repression - KRAB
April
21. April
Minipreps of pKM006, pKM102 & pSAM200
name
ng/µl
pSAM200
320.1
pKM006
113.3
pKM102
180.0
May
02. May
PCR of pX334 fragments
PCR 1
µl
type
10
Q5-HF Reaction Buffer
1
pX334
1
oIG2000
1
oIG2029
2.5
dNTPs
1.5
DMSO
0.5
Q5-HF Polymerase
Add to 50
H2O
Annealing: 60°C
Extension: 115 sec.
Cycles: 18
PCR 2
µl
type
10
Q5-HF Reaction Buffer
1
pX334
1
oIG2028
1
oIG2001
2.5
dNTPs
1.5
DMSO
0.5
Q5-HF Polymerase
Add to 50
H2O
Annealing: 60°C
Extension: 115 sec.
Cycles: 18
PCR 3
µl
type
10
Q5-HF Reaction Buffer
1
pX334
1
oIG2002
1
oIG2003
2.5
dNTPs
1.5
DMSO
0.5
Q5-HF Polymerase
Add to 50
H2O
Annealing: 60°C
Extension: 115 sec.
Cycles: 18
PCR 4
µl
type
10
Q5-HF Reaction Buffer
1
pX334
1
oIG2006
1
oIG2007
2.5
dNTPs
1.5
DMSO
0.5
Q5-HF Polymerase
Add to 50
H2O
Annealing: 60°C
Extension: 115 sec.
Cycles: 18
PCR 5
µl
type
10
Q5-HF Reaction Buffer
1
pX334
1
oIG2008
1
oIG2009
2.5
dNTPs
1.5
DMSO
0.5
Q5-HF Polymerase
Add to 50
H2O
Annealing: 60°C
Extension: 115 sec.
Cycles: 18
All samples were stored at -20°C
02. May
Agarose Gelelectrophoreses of PCR fragments No. 1-5
f.l.t.r.: marker, PCR 1 (1909 bp), PCR 2 (2610 bp), PCR 3 (1645 bp), PCR 4 (2272 bp), PCR 5 (1878 bp)
Gel Extraction of PCRs 1-6
name
ng/µl
Fragment No. 1
40.7
Fragment No. 2
53.6
Fragment No. 3
52.6
Fragment No. 4
41.6
Fragment No. 5
40.4
03. May
Fusion PCR of PCR fragments 2-3 and 5-1
Assembly fragment 2-3 (PCR 1)
µl
type
10
Q5-HF Reaction Buffer
1
Fragment No. 2
1
Fragment No. 3
4
dNTPs
1.5
DMSO
0.5
Q5-HF Polymerase
Add to 50
H2O
Annealing: 71°C
Extension: 105 sec.
Cycles: 5
Assembly fragment 2-3 (PCR 2)
Addition of 1 µl of oIG2028 and oIG2003 to PCR 1
Annealing: 60 °C
Extension: 2 min 40 sec
Cycles: 16
Assembly fragment 5-1 (PCR 1)
µl
type
10
Q5-HF Reaction Buffer
1
Fragment No. 5
1
Fragment No. 1
4
dNTPs
1.5
DMSO
0.5
Q5-HF Polymerase
Add to 50
H2O
Annealing: 71°C
Extension: 105 sec.
Cycles: 5
Assembly fragment 5-1 (PCR 2)
Addition of 1 µl of oIG2008 and oIG2029 to PCR 1
Annealing: 60 °C
Extension: 2 min 40 sec
Cycles: 16
Gel Extraction of Fusion fragments 2-3 and 5-1
name
ng/µl
Fragment No. 2-3 (a)
76.6
Fragment No. 2-3 (b)
57.5
Fragment No. 2-3 (c)
106.8
Fragment No. 5-1
23.4
17. May
PCR of the KRAB fragment with a N-terminal linker (GSAGSAG) overhang
PCR 1
µl
type
10
Q5-HF Reaction Buffer
1
pKM102
1
oIG2004
1
oIG2005
2.5
dNTPs
1.5
DMSO
0.5
Q5-HF Polymerase
Add to 50
H2O
Annealing: 60°C
Extension: 30 sec.
Cycles: 18
For higher Gel Extraction yields: 4x amount
Agarose Gelelectrophoreses of the KRAB fragment
f.l.t.r.: marker, KRAB at 490 bp (4x)
Gel Extraction of the KRAB fragment
name
ng/µl
KRAB (a)
76.0
KRAB (b)
64.7
Two bands were respectively pooled on one extraction column, in order to yield higher amount of DNA.
20. May
Gibson Assembly of pIG2004 (Cas9-KRAB)
Preparation of 5µl DNA mix, containing all four fragments
Melting a 15µl master-mix on ice, until it is ready to use
Addition of 5µl of DNA mix to the master mix
Immediately put the mix on 50°C and incubate for 1h
3 min on RT
3 min on ice
Transformation of 4 µl to TOP10 chemically competent E-coli cells
Preparation scheme for Gibson Assembly fragment mix
21. May
Evaluation of Gibson Assembly
Colony growth
Colony PCR will be performed with primers within KRAB and Cas9 fragment
Colony PCR Mastermix for Gibson Clones
µl
type
34.8
H2O
13.0
Taq Buffer
13.0
dNTPs
1.3
oIG2002
1.3
oIG2005
1.6
Taq Polymerase
5.0
H2O for each reaction, containing picked colony
Annealing: 60°C
Extension: 68°C
Extension: 2 min 10 sec.
Cycles: 30
Results of Colony PCR: Clones 5, 6 and 7 were striked out.
22. May
Miniprep of suspected pIG2004 Clones 5, 6 and 7
name
ng/µl
pIG2004, No. 5
207.9
pIG2004, No. 6
180.0
pIG2004, No. 7
198.1
Clones were sent to sequencing at GATC
23. May
Evaluation of Gibson Cloning Sequencing Results
oIG0007 sequencing reveals that the KRAB fragment was succesfully fused to Cas9 in each case. Further sequecing will be done to test for correct N-terminal assembly and H840A mutation.
PCR of pX334 fragment 6
PCR 1
µl
type
10
Q5-HF Reaction Buffer
1
pX334
1
oIG2002
1
oIG2007
2.5
dNTPs
1.5
DMSO
0.5
Q5-HF Polymerase
Add to 50
H2O
Annealing: 60°C
Extension: 2 min 40 sec.
Cycles: 18
Agarose Gelelectrophoreses of PCR fragments No. 6
f.l.t.r.: marker, PCR 6 (3908 bp), empty, PCR 6 (2x)
Gel Extraction of PCR 6
name
ng/µl
Fragment No. 6
43.6
Gibson Assembly of pIG2005 (dCas9)
Preparation of 5µl DNA mix, containing all four fragments
Melting a 15µl master-mix on ice, until it is ready to use
Addition of 5µl of DNA mix to the master mix
Immediately put the mix on 50°C and incubate for 1h
3 min on RT
3 min on ice
Transformation of 4 µl to TOP10 chemically competent E-coli cells
Preparation scheme for Gibson Assembly fragment mix
24. May
Digest of pIG2004 (No. 5, 6 and 7) with BbsI
µl
type
Volume
DNA
16
pIG2004
10
NEB-Buffer 2.1
2
BbsI
72
H2O
Temp.: 37°C
Incubation time: 2h
Agarose Gelelectrophoreses of pIG2004 Digests
f.l.t.r.: marker, pX334, empty, digested pIG2004 5 (2x), 6 (2x) and 7 (2x). Upper bands of clone no. 5 and 6 digests were cut out for Gel Extraction.
Gel Extraction of pIG2004 digest
name
ng/µl
pIG2004 (5)
38.0
pIG2004 (6)
17.0
Evaluation of Gibson Assembly of 23. May
Colony growth
Five clones were striked out
27. May
Evaluation of pIG2004 sequencing results (No. 5 and 6)
H840A mutation was inserted in both cases, leading to catalytic inactivation of Cas9
a 300 bp gap was inserted within the CAG promoter, probably due to error-prone PCR amplification
Very GC-rich part on pIG2004 template (part of the CAG promoter lacks nearly 300 bp after having been amplified via PCR and subsequently Gibson-assembled.
June
04. June
Minipreps of pIG2005 clones striked out on 23. May
name
ng/µl
pIG2005 - No. 1
77.1
pIG2005 - No. 2
68.4
pIG2005 - No. 3
59.6
pIG2005 - No. 4
43.7
pIG2005 - No. 5
61.2
07. June
Fixing of the 300 bp CAG promoter gap for Cas9-KRAB
Restriction digest of pIG2004 and pIG2005 - 1
µl
type
8
pIG2004 - No. 5 or pIG2005 - No. 1
8
NEB-Buffer 4
2
AgeI-HF
2
NotI-HF
2
SacII
2
BSA
Add to 80µl
H2O
Restriction digest of pX334 - 2
µl
type
8
pX334 DNA
8
NEB-Buffer 4
2
KpnI-HF
2
NotI-HF
2
SacI-HF
2
BSA
Add to 80µl
H2O
Restriction digest of pX334 - 3
µl
type
8
pX334 DNA
8
NEB-Buffer 4
2
KpnI-HF
2
AgeI-HF
2
BSA
Add to 80µl
H2O
Temp.: 37°C
Incubation time: 1h
Expected fragment lengths: 1 - 4500 bp (Cas9-KRAB), 2 - 4900 bp (Backbone), 3 - 890 bp (CAG promoter)
Top left. Upper band: Cas9-KRAB fragment, Top right. Upper band: dCas9 fragment. Bottom left. Upper band: pX334 Backbone fragment.
Top right. Lowest band (blurred): CAG promoter fragment.
Gel Extraction of digest fragments
name
ng/µl
Cas9-KRAB
26.7
dCas9
5.0
pX334 Backbone
14.4
CAG promoter
5.0
Ligation and transformation of intact pIG2004 and pIG2005
Ligation of pIG2005
µl
type
11.2
CAG fragment
4.4
dCas9 fragment
1.4
pX334 backbone
2
T4 ligase buffer
1
T4 ligase
Ligation of pIG2004
µl
type
0.8
Cas9-KRAB fragment
14.8
CAG fragment
1.4
pX334 backbone
2
T4 ligase buffer
1
T4 ligase
08. June
Evaluation of Transformations of pIG2005 and pIG2004
pIG2004 (30 µl strikeout): 24 colonies
pIG2005 (30 µl strikeout): 15 colonies
12 colonies of both approaches were striked out for further sequencing
09. June
Minipreps of pIG2004 and pIG2005
name
ng/µl
pIG2004 - No. 1
198.4
pIG2004 - No. 2
203.6
pIG2004 - No. 3
202.0
pIG2004 - No. 4
187.3
pIG2004 - No. 5
195.3
pIG2004 - No. 6
211.2
pIG2004 - No. 7
187.4
pIG2004 - No. 8
201.1
pIG2004 - No. 9
201.7
pIG2004 - No. 10
206.3
pIG2004 - No. 11
212.3
pIG2004 - No. 12
174.6
pIG2005 - No. 1
202.0
pIG2005 - No. 2
195.3
pIG2005 - No. 3
189.9
pIG2005 - No. 4
221.4
pIG2005 - No. 5
205.4
pIG2005 - No. 6
194.4
pIG2005 - No. 7
206.7
pIG2005 - No. 8
206.3
pIG2005 - No. 9
220.4
pIG2005 - No. 10
192.9
pIG2005 - No. 11
188.8
pIG2005 - No. 12
192.1
Colonies No. 2 and 3 of pIG2004 were sent to sequencing, intending to validate (i) KRAB-insertion (ii) H840A conversion and (iii) an intact CAG promoter
Colonies No. 1 and 4 of pIG2005 were sent to sequencing, intending to validate (i) H840A conversion and (ii) an intact CAG promoter
11. June
Evaluation of Sequencing results of pIG2004 and pIG2005
pIG2004 clone No. 2 misses the KRAB fragment, but contains the CAG promoter
pIG2004 clone No. 3 contains KRAB fragment, H840A and the CAG promoter - and will therefore be used for further studies
pIG2005 clones No. 1 and 4 contain both H840 mutation and the CAG promoter
Restriction digest of pIG2004 and pIG2005 with BbsI
µl
type
6
DNA
4
NEB-Buffer 2.1
2
BbsI
Add to 40µl
H2O
F.l.t.r.: marker, pIG2004 undigested, pIG2004 digested, pIG2005 undigested, pIG2005 digested (2x)
Gel Extraction of digested vectors pIG2004 and pIG2005
name
ng/µl
pIG2004
16.1
pIG2005
18.8
12. June
Oligo annealing of target crRNAs
µl
type
3
Oligo No. 1
3
Oligo No. 2
69
NEB-Buffer 2
Samples were heated up to 95 degrees for 5 minutes. Gradient cool down to room temperature for 2 hours.
Pair 1: oIG2010/oIG2011 - EMX1
Pair 2: oIG2022/oIG2023 - Target 200 bp upstream of SEAP transcription initiation on pKM006
Pair 3: oIG6008/oIG6009 - CMV
13-20. June
Ligation and transformation of crRNAs with Cas9-KRAB and dCas9
Ligation of pIG2013
µl
type
15
Annealed EMX1
3
oIG2004
2
T4 ligase buffer
1
T4 ligase
Ligation of pIG2017
µl
type
15
Annealed EMX1
3
oIG2005
2
T4 ligase buffer
1
T4 ligase
Ligation of pIG2019
µl
type
15
Annealed SEAP target
3
oIG2004
2
T4 ligase buffer
1
T4 ligase
Ligation of pIG2020
µl
type
15
Annealed SEAP target
3
oIG2005
2
T4 ligase buffer
1
T4 ligase
Ligation of pIG2011
µl
type
15
Annealed CMV target
3
oIG2004
2
T4 ligase buffer
1
T4 ligase
Ligation of pIG2015
µl
type
15
Annealed CMV target
3
oIG2005
2
T4 ligase buffer
1
T4 ligase
Evaluation of transformation results
Except for pIG2020, colonies were grown in each case
6 colonies of pIG2011, pIG2013, pIG2015, pIG2017 and pIG2019 were respectively striked out
All attempts had to be repeated at least twice, as BbsI digestion often lead to unspecific digestion of direct-repeats next to crRNA insertion spacer.
18-25. June
MIDI-preps of pIG2004, pIG2005, pIG2011, pIG2013, pIG2015, pIG2017, pIG2019 and also pIG9000
name
ng/µl
pIG2004 (Cas9-KRAB)
400
pIG2005 (dCas9)
203
pIG2011 (Cas9-KRAB vs. CMV)
610
pIG2013 (Cas9-KRAB vs. EMX1)
609
pIG2015 (dCas9 vs. CMV)
460
pIG2017 (dCas9 vs. EMX1)
520
pIG2017 (dCas9 vs. EMX1)
520
pIG2019 (Cas9-KRAB vs. SEAP)
430
pIG9000 (Cas9 vs. EMX1)
460
MIDIs had to be repeated three times
During the first attempt, an unkown white precipitate was obtained - which could not be separated from the desired DNA.
Within the second attempt, DNA was not adequately treated during isopropanol treatment (has to be shaken).
25.-27. June
Test of Cas9-KRAB on Western Blots
Cas9-KRAB and dCas9, both targetting the CMV promoter were respectively co-transfected to HEK 293 cells with another plasmid, encoding CMV driven CNK.
Both attempts were performed repeatedly, and cell lyses were than to be performed 12, 18, 36 and 42 hours post-transfection.
Transfections were performed with PEI
Lyses were performed with modified RIPA buffer
Biological double attempts were respectively done
Transfection schemes for one 6-well plate
Cas9-KRAB vs. CMV of CNK (2x)
name
µg
CNK
6
pIG2011
12
dCas9 vs. CMV of CNK (2x)
name
µg
CNK
6
pIG2015
12
CNK negative control (2x)
name
µg
CNK
6
Cas9-KRAB and dCas9 expression controls on one well
name
µg
pIG2004
1
pIG2005
1
Results
???
???
We expected a stronger decrease in CNK levels, when combining it with Cas9-KRAB than with dCas9
Cas9-KRAB was able to highly efficiently repress CMV driven CNK expression
Cas9-KRAB was expressed.
29. June
Test of Cas9-KRAB vs. CMV-driven GFP
Transfection of pIG6000 (CMV::GFP) in combination with either pIG2011 (Cas9-KRAB + crRNA agaisnt CMV) or Mock DNA
Cells were seeded previously seeded on 6-well plates
Incubation for about 36 hours
dCas9 vs. CMV of GFP
name
µg
CMV-GFP
1
pIG2011 (Cas9-KRAB)
2
PEI
9
Transfection scheme
July
01. July
Fluorescence Microscopy Evaluation
Fluorescence Images
ImageJ: Cells treated with GFP and with pRSET (Mock-DNA)
Cell
Size
Intensity
1
208
2920.9
2
146
3945.0
3
238
3064.8
4
166
3699.3
5
238
2071.3
6
192
2219.1
7
172
3819.9
8
228
3235.1
9
247
3510.0
10
146
3213.6
11
228
1977.4
12
224
3436.5
13
224
3436.5
14
136
1872.9
15
140
1611.4
ImageJ: Cells treated with GFP and Cas9-KRAB-DNA
Cell
Size
Intensity
1
268
2407.6
2
176
3329.2
3
217
1336.1
4
166
1138.5
5
102
2844.8
6
217
936.6
7
148
1236.4
8
214
1032.9
9
177
1566.6
10
96
1461.9
11
228
1977.4
12
938
3371.7
13
102
992.9
14
210
982.1
15
112
1016.6
Mean of GFP fluorescence intensity of cells treated with Cas9-KRAB is 1409.4 (SD: 880.6), cells co-transfected with Mock-DNA revealed 2718 (SD: 786.6)
Results should be of low validity, but are potentially indicative towards a repressive effect of Cas9-KRAB
02. July
MIDI-prep of pIG2005, pIG2019 and pRSET (Mock-DNA)
Failed in each case, for unknown reasons (white salt fall-out after elution from column). Attempt will be repeated on the next day.
03. July
MIDI-prep of pIG2005, pIG2019 and pRSET (Mock-DNA)
name
ng/µl
pIG2005 (dCas9)
470
pIG2019 (Cas9-KRAB vs. SEAP target)
430
pRSET (Mock DNA)
790
04. July
Transfection of pIG2019 in combination with pKM006 (SEAP plasmid) and pSAM200 (SV40::tetR-VP16)
Biological tetraplicates (each attempt was repeated independently, for four different wells of one 6-well plate
Negative control: pKM006 (500 ng) + Mock-DNA (2500 ng)
Positive control: pKM006 (500 ng) + pSAM200 (500 ng) + Mock-DNA (2000 ng)
Cas9-KRAB test: pKM006 (500 ng) + pSAM200 (500 ng) + pIG2019 (2000 ng)
07. July
SEAP measurement
300µl of medium supernatants were removed from each well
Samples were heated to 65°C for 30 minutes
Centrifugation at 600 rpm (table top) for 1 minute
80µl of each supernatant was transfered to one well of a 96-well plate, technical triplicates for each sample were pipetted
100µl 2x SEAP Buffer was added to each well
20µl pNPP substrate was added to each well
Air bubbles were quickly removed
Measurement was taken with a plate reader, for two consecutive hours with an interval lenght of one minute. Detection wavelength was 405 nm
SEAP Results
September
01. September
Test of Cas9-KRAB vs. CMV-driven GFP, Repetition of the Experiment from 29. June on 24 well plates
dCas9 vs. CMV of GFP
name
µg
CMV-GFP
0,04
pIG2011 (Cas9-KRAB)
0,46
PEI
1,5
dCas9 vs. CMV of GFP
name
µg
CMV-GFP
0,04
pRSET (Mock-DNA)
0,46
PEI
1,5
03. September
Fluorescence Microscopy of KRAB-mediated GFP-repression
Results of GFP repression. Fluorescence images were taken with a light exposure time of 250 ms (80% saturation). Experiment shall be repeated with off-target controls and parallel Flow Cytometry analysis.
09. September
Retrafo of Plasmids for the Midiprep
Following plasmids are needed for activation and repression repeats:
pRSet
pKM600
pKM604
pKM605
pKM606
pKM607
pKM603
pIG2017
pIG2013
pSAM200
Retrafo was inoculated into Ampicillin medium (concentration of 1:1000 from Ampicillin and medium).
10. September
Midiprep
Plasmids were prepped with the Midiprep kit of Promega according to the manufacturer's protocol.
Seeding of cells
4 24-well plates were seeded with 65,000 cells per well
11. September
Transfection
40 µl Opti-MEM + 2.25 µl PEI-solution were mixed in a 1.5 ml Eppi.
0.75 µg of the DNA of interest were prepaired in another Eppi .
Addition of the DNA to former Eppi, vortexing for 10 s and incubation for 15 min at RT
Solution was spread drop-wise to the cells in the dish
Medium change
Medium was changed after 5 h.
13. September
Preparation for analyses
Supernatant was collected for SEAP measurement.
Cells were frozen at - 80 °C for later on Western blot and luminescence measurement.
14. September
SEAP measurement
Collected supernatant and fresh cell culture medium was heated at 65 °C for 30 min, then centrifuged at 1250 g for 1 min.
Supernatant was diluted in cell culture medium 1:4 (25 µl supernatant + 75 µl medium).
80 µl of diluted supernatant was given to 100 µl of SEAP buffer.
After addition of 20 µl pNPP (substrate of SEAP) the absorbance of the mix was measured every minute for 2 h at 405 nm.
16. September
Cell lysis
250 µl of lysis buffer (containing Tris-HCl, EGTA, MgSO4, DTT and protease inhibitor) were applied to the thawn cells of each well.
Incubation on ice for 10 min.
Centrifugation at 15,000 g for 4 min.
Renilla luminescence measurement
80 µl of supernatant were pipetted in a white 96 well plate.
Measurement of luminescence (every 2 min for 30 min) immediately after addition of 20 µl Renilla substrate in PBS.
SEAP activity normalized to Renilla expression
Blue bars: The SEAP expression ranged between the expression of the CMV minimal promoter and the activation with TetR-VP16. Surprisingly the repression of Cas9-KRAB without a appropriate crRNA seemed to be stronger than the repression of Cas9-KRAB with this crRNA.
Red bars: When targeting the same locus, Cas9 leaded to less SEAP expression in combination with Cas9-VP16 than Cas9-KRAB in the same combination.
17. September
Protein precipitation
As the cell lysis needed to be done with a high amount of lysis buffer, proteins are too much diluted for western blotting. Thus, they were precipitated:
The remaining lysates of the triplikates (~ 80 µl) were put together in a new epi.
Addition of TCA to a final concentration of 10 % and 1 µg BSA.
Incubation for 30 min on ice.
Centrifugation for 30 min at full speed and 4 °C.
Removal of supernatant and addition of 500 µl acetone.
Incubation on ice over night.
Centrifugation for 15 min at full speed and 4 °C.
Removal of acetone and nair drying for 20 min.
Addition of 20 µl 2x SDS-loading dye and 0.5 µl Tris-HCl (pH = 8.8).
Heating at 95 °C for 5 min.
18. September
SDS-gel run
SDS-gel was loaded with 18 µl of each sample. A voltage of 80 V was applied till the loading dye bands reached the border between collection and separation gel, then the voltage was increased to 120 V.
Western Blotting
Activation of PVDF-membrane for 10 min in methanol.
Short incubation of Whatman papers and the PVDF-membrane in transfer buffer.
Loading of western blot apperature: (-) - Whatman paper - SDS-gel - PVDF-membrane - Whatman paper - (+).
200 mA were applied for 1.5 h.
Antibody treatment I
Incubation of PVDF membrane in blocking buffer (4 % milkpowder in TBS-T) for 1 h.
Incubation with anti HA antibody (in 2 % milkpowder in TBS-T) over night at 4 °C.
3 x washing with TBS-T for 5 min, each.
Incubation with anti mouse antibody for 1 h.
4 x washing with TBS-T for 10 min, each.
Measurement of luminescence after addition of ECL I + ECL II solution.
Air drying of the membrane for further antibody treatments.
19. September
Seeding of cells
3 24-well plates were seeded with 65,000 cells per well.
Antibody treatment II
Reactivation in methanol.
1 x washing with TBS-T for 5 min.
Incubation of PVDF membrane in blocking buffer (4 % milkpowder in TBS-T) for 30 min.
Incubation with anti beta-actin antibody (in 2 % milkpowder in TBS-T) over night at 4 °C.
3 x washing with TBS-T for 5 min, each.
Incubation with anti mouse antibody for 1 h.
4 x washing with TBS-T for 10 min, each.
Measurement of luminescence after addition of ECL I + ECL II solution (500 µl each).
Western blot
In all tested wells there was an expression of Cas9, but a shift between Cas9 and Cas9-VP16 or Cas9-KRAB and Cas9-VP16 is not detectable.
20. September
Transfection
40 µl Opti-MEM + 2.25 µl PEI-solution were mixed in a 1.5 ml Eppi.
0.75 µg of the DNA of interest were prepaired in another Eppi .
Addition of the DNA to former Eppi, vortexing for 10 s and incubation for 15 min at RT
Solution was spread drop-wise to the cells in the dish
Ratio of DNA amount (mass):
reporter plasmid (SEAP) : effector plasmid (Cas9) : RNA plasmid
1 : 4 : 4
Medium change
Medium was changed after 4.5 h.
22. September
Medium removal
Supernatant was collected for SEAP measurement.
Cells were frozen at - 80 °C for later on Western blotting.
Preparation for SEAP measurement
Collected supernatant and fresh cell culture medium was heated at 65 °C for 30 min, then centrifuged at 1250 g for 1 min.
23. September
SEAP measurement
Supernatant of wells containing CMV:SEAP was diluted in cell culture medium 1:10 (10 µl supernatant + 90 µl medium).
80 µl of (diluted) supernatant was given to 100 µl of SEAP buffer.
After addition of 20 µl pNPP (substrate of SEAP) the absorbance of the mix was measured every minute for 2 h at 405 nm.
SEAP activity [U/L]
pIG2004: Cas9-KRAB (not in iGEM standard); pIG2005: Cas9 (not in iGEM standard); PhyB: negative control; all other Cas9 constucts are in iGEM standard (with the indicated promoter)
In comparison to the negative control the non standardized constructs repressed the SEAP expression only slightly, whereas there was a strong repression by the standardized constructs, when driven by a CMV promoter. The repression of Cas9-KRAB seemed to be a little stronger than the effect of Cas9 alone. But just like in the last experiment, the repression of Cas9-KRAB with a wrong crRNA was as strong as the repression with the right one.
Seeding of cells
4 24-well plates were seeded with 65,000 cells per well
24. September
Repeat of seeding of cells
4 24-well plates were seeded with 65,000 cells per well again, as the cell density of the first seeding was too low.
25. September
Transfection
40 µl Opti-MEM + 0.75 µg of the DNA of interest were mixed in a 1.5 ml Eppi.
Addition of 2.25 µl PEI-solution, vortexing for 10 s and incubation for at least 15 min at RT
Solution was spread drop-wise to the cells in the dish
Ratio of DNA amount (mass):
reporter plasmid (SEAP) : effector plasmid (Cas9) : RNA plasmid
1 : 4 : 4
Additionally a plasmid coding for Renilla luciferase was cotransfected (5 % of DNA amount)
26. September
Medium change
Medium was changed after 8 h.
Cell lysis of transfection of 20. September
80 µl of RIPA lysis buffer were applied to the thawn cells of each well.
Incubation on ice for 10 min.
Sonification 3 x 30 s at maximum.
Centrifugation at 10,000 g for 10 min.
40 µl of supernatant were mixed with 10 µl 5 x SDS sample buffer.
Heating for 5 min at 95 °C.
SDS-gel run
SDS-gel was loaded with 40 µl of each sample. A voltage of 80 V was applied till the loading dye bands reached the border between collection and separation gel, then the voltage was increased to 120 V.
Western Blotting
Activation of PVDF-membrane for 10 min in methanol.
Short incubation of Whatman papers and the PVDF-membrane in transfer buffer.
Loading of western blot apperature: (-) - Whatman paper - SDS-gel - PVDF-membrane - Whatman paper - (+).
200 mA were applied for 1.5 h.
Antibody treatment I
Incubation of PVDF membrane in blocking buffer (4 % milkpowder in TBS-T) for 1 h.
Incubation with anti HA antibody (in 2 % milkpowder in TBS-T) over night at 4 °C.
3 x washing with TBS-T for 5 min, each.
Incubation with anti mouse antibody for 1 h.
4 x washing with TBS-T for 10 min, each.
Measurement of luminescence after addition of ECL I + ECL II solution.
Air drying of the membrane for further antibody treatments.
Western blot of the SEAP assay from 23.09.
CAG:dCas was strongly expressed, whereas CMV:dCas and dCas-KRAB were not visible on the western blot. Nevertheless a effect has been detected with the SEAP assay, so there should be an expression, but maybe too low for detection.
27.09.13
Medium removal
Supernatant was collected for SEAP measurement 48 h after transfection.
Cells were frozen at - 80 °C for later on Western blotting.
28.09.13
SEAP measurement
Collected supernatant and fresh cell culture medium was heated at 65 °C for 30 min, then centrifuged at 1250 g for 1 min.
Supernatant of wells containing CMV:SEAP was diluted in cell culture medium 1:15 (6.7 µl supernatant + 93.3 µl medium).
80 µl of (diluted) supernatant was given to 100 µl of SEAP buffer.
After addition of 20 µl pNPP (substrate of SEAP) the absorbance of the mix was measured every minute for 2 h at 405 nm.
Cell lysis for lyciferase measurement and western blot
250 µl of lysis buffer (containing Tris-HCl, EGTA, MgSO4, DTT and protease inhibitor) were applied to the thawn cells of each well.
Incubation on ice for at least 10 min.
Centrifugation at 2,200 g for 30 min.
Luciferase luminescence measurement
80 µl of supernatant were pipetted in a white 96 well plate.
Measurement of luminescence (every 2 min for 15 min) immediately after addition of 20 µl Renilla substrate in PBS.
SEAP-Assay: Results
All different tested loci showed no clear result. dCas9-KRAB under the control of the CMV promotor showed no significant repression in comparison to the off-target and CRISPRi control. dCas9-KRAB under the control of the SV40 promotor showed no repression at all. Maybe targeting a gene coded on a plasmid is a problem for dCas9-KRAB to repress.
29.09.13
Protein precipitation
As the cell lysis needed to be done with a high amount of lysis buffer, proteins are too much diluted for western blotting. Thus, they were precipitated:
340 µl of acetone was added to 85 µl of the remaining lysates of one well per triplicate.
Incubation at - 20 °C for 1:15 h.
Centrifugation for 5 min at 10,000 g and 4 °C.
Removal of acetone and air drying for 5 min.
Addition of 25 µl 2x SDS-loading dye.
Heating at 95 °C for 5 min.
SDS-gel run
SDS-gel was loaded with 20 µl of each sample. A voltage of 80 V was applied untill the loading dye bands reached the border between collection and separation gel, then the voltage was increased to 120 V.
Western Blotting
Activation of PVDF-membrane for 10 min in methanol.
Short incubation of Whatman papers and the PVDF-membrane in transfer buffer.
Loading of western blot apperature: (-) - Whatman paper - SDS-gel - PVDF-membrane - Whatman paper - (+).
200 mA were applied for 1.5 h.
Antibody treatment I
Incubation of PVDF membrane in blocking buffer (4 % milkpowder in TBS-T) for 1 h.
Incubation with anti HA antibody (in 2 % milkpowder in TBS-T) over night at 4 °C.
3 x washing with TBS-T for 5 min, each.
Incubation with anti mouse antibody for 1 h.
4 x washing with TBS-T for 10 min, each.
Measurement of luminescence after addition of ECL I + ECL II solution.
Air drying of the membrane for further antibody treatments.
Western blot of the SEAP assay from 28.09.
Western blotting did not work well. Maybe lysis due to relative long incubation time on Renilla lysis buffer destroyed cells to much beforehand. So maybe only degradation products are visible.
Retrieved from " http://2013.igem.org/Team:Freiburg/Notebook/lab_repression "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
