<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Project_effector"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Project/effector">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Project/effector&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Project/effector&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Project/effector&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Project/effector" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Project/effector</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3></DIV><DIV id="menu"><UL id="navbar"><LI style="padding-left:15px"><A href="https://2013.igem.org/Team:Freiburg">Home</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Highlights">Highlights</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Introduction </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/induction">Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></LI></UL><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna">Targeting</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/method">uniBAss</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation">Standardization</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/protocols">Material and Methods</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/BioBricks">Our BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/favorite_parts">Favorite BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/improvement">Improvement of a Registry part</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/sharing">Sharing of our BioBricks </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/experts">Expert opinions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety/safety">Safety &amp; Security</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/politics">Politics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/society">Society</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/education">Education</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/art">Art</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/Facebook">Facebook / Media</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/News">News</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team2">Members</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Exolab">Exolab-Events</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Statistics">Statistics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/gallery">Photo Gallery</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Attributions">Our Partners</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/Coop">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/attributions"> Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/thanks">Thanks to</A></LI></DIV><DIV id="main"><DIV id="sidebar"><DIV id="sidebar_content"><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Intro </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/effector" class="active"> Effectors </A></P><P class="second_order"><A href="#activation">Activation </A></P><P class="second_order"><A href="#epigenetics">Epigenetics </A></P><P class="second_order"><A href="#repression">Repression </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/induction"> Effector Control </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></P></DIV></DIV><DIV id="main_contant"><P id="h1">
Effectors
</P><P> We engineered the <B><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/1#background">CRISPR/Cas9</A></B> system that is relying on a protein-RNA-DNA interaction to generate a DNA binding protein. For this we used a catalytically inactive Cas9, termed dCas9.
	As we wanted to manipulate gene expression with our system we fused different effector domains to the dCas9. By doing so we created proteins that are able to <B><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/effector#activation">activate</A></B>, <B><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/effector#repression">repress</A></B> and alter the <B><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/effector#epigenetics">epigenetic</A></B> state of genes.
	</P><P>
	A main feature of our uniCAS system is the so-called <B><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#rnaimer">RNAimer</A></B>, a plasmid coding for the required RNAs. By using several RNAs mutliple targeting is an immense advantage in comparison to other DNA-binding proteins. 
To highlight the specificity of our system we conducted the experiments with an off-target control. Here, the fusion protein is led to a locus that is not related to our test locus. </P><P>For repression experiments we performed experiments that alter the endogenous VEGF-A expression. For activation we tested the expression of a secreted reporter protein, the secreted embryonic alkalyine phosphatase, short SEAP. </P><TABLE id="headline_pic"><TBODY><TR><TD style="width:1000px;"><P id="h2">
Activation
</P></TD></TR></TBODY></TABLE><P id="h3">
Introduction
</P><P>
The aim of this subproject was to engineer a novel transcriptional activation device, based on an RNA guided synthetic dCas9-VP16 fusion protein. To do so, we used a mutated, catalytically inactive form of Cas9 <A id="link" href="http://parts.igem.org/Part:BBa_K1150000">(dCas9)</A>. Impaired in its cleavage activity, dCas9 was fused to the 5' end of the trans-activation domain of <A id="link" href="http://parts.igem.org/Part:BBa_K1150001">VP16</A>. To ensure nuclear localization of the whole construct, a <A id="link" href="http://parts.igem.org/Part:BBa_K1150010">nuclear localization signal (NLS)</A> was fused to the 5' and 3' termini of dCas9-VP16. A <A id="link" href="http://parts.igem.org/Part:BBa_K1150016">HA-epitope coding sequence</A> was prefixed for efficient protein detection detections on western blots. Expression was put under control of a <A id="link" href="http://parts.igem.org/Part:BBa_K1150011">SV40</A> or <A id="link" href="http://parts.igem.org/Part:BBa_K747096">CMV</A> promoter, and also a <A id="link" href="http://parts.igem.org/Part:BBa_K1150012">BGH terminator</A> was suffixed. Figure 1 illustrates the design described above.
</P><CENTER><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 1: Design of the dCas9-VP16 fusion constuct.</B>
dCas9 was fused to VP16 via a 7 amino acid linker. The resulting fusion protein was flanked by NLS sequences and tagged by a HA epitope. The SV40 or CMV promoter and the BGH terminator were chosen to control gene expression. BBa_K1150019 is put under the control of the SV40 promoter, BBa_K1150020 is under the control of the CMV promoter. 
			</TD></TR></TBODY></TABLE></DIV></CENTER><P>
The Virus Protein 16 (VP16) is a transcription factor of Herpes simplex virus-1. Through complex formation with cellular host factors, VP16 can bind to a common regulatory element in the upstream promoter region of adjacent genes <SPAN id="refer"><A href="#(1,1)">[1]</A></SPAN>. Through the trans-activating activity of VP16, expression rates of these genes can be enhanced. The protein consists of 490 amino acids - with two important domains: first, a core domain in its central region, which is necessary for the indirect DNA binding. Furthermore, a carboxy-terminal trans-activation domain <SPAN id="refer"><A href="#(1,2)">[2]</A></SPAN><SPAN id="refer"><A href="#(1,3)">[3]</A></SPAN>. The latter part of VP16 can be fused to DNA-binding domains of other proteins - in order to efficiently upregulate expression of desired target genes <SPAN id="refer"><A href="#(1,4)">[4]</A></SPAN>.
</P><P id="h4"> 
Mechanism
</P><P>
By co-transfecting our <A id="link" href="http://parts.igem.org/Part:BBa_K1150034">RNA plasmid</A> which codes for the desired crRNA, dCas9-VP16 specifically binds to the targeted DNA sequence. With the help of the trans-activation domain of VP16, transcription factors are recruited and the pre-initiation complex can be built. By targeting this construct upstream of a promoter region any gene of interest can be activated. 
</P><CENTER><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 2: Principle of trans-activation of mammalian gene expression by the fusion protein dCas9-VP16</B>
				The non-cleaving dCas9 fused to the VP16 domain serves as a crRNA-guided DNA-binding protein and trans-activates gene expression. Due to its helicase function, dCas9 unwinds the DNA-strand and the crRNA can bind sequence specifically.
			</TD></TR></TBODY></TABLE></DIV></CENTER><P id="h4"> 
Experimental setup
</P><P>
For testing this device we used HEK-293T cells, which were seeded at a densitiy of 65,000 cells/well in 24-well plates. After 24 hours RNA plasmids targeted against the indicated loci and the referring CMVmin:SEAP reporter plasmids were co-transfected to this device. The reporter plasmids contain the secreted alkaline phosphatase (SEAP) gene under the control of a CMV minimal promoter. 48 hours post transfection the activity of SEAP in the cell culture medium was measured. Additionally the luciferase Renilla was measured as an internal standard to eliminate variabilities of protein expression levels caused by the expression of the transfected plasmids. The dCas9-VP16 expression was assessed by Western blot analysis of cell lysates. Different crRNAs were tested and compared for their activation properties of the referring reporter plasmid.<B>Our controls were the following:</B>
pRSet is a vector containing junk DNA and is transfected in the same DNA amount as the samples. As our negative control it reflects the basic SEAP level production in the absence of any activator protein. 
dCas9-VP16 driven by a SV40 or CMV promoter transfected without any crRNA is the off-target control. As dCas9-VP16 should only bind to the target locus in combination with the related crRNA no increase in the SEAP production should be visible in comparison to the pRSet control. As no such effect was observed we conclude that off-target effects are no major problem.
</P><P id="h3"> 
Results	
</P><P>
With the dCas9-VP16 fusion protein different target loci have been tested by the usage of a SEAP reporter plasmid with a minimal CMV promoter (pKM602). The target sites can be determined by directing the crRNA against the desired sequence of interest. Four Target sites with different distances to the promoter were tested (Figure 3 and Table 1).
We used two different promoters to drive the dCas9-VP16 expression: The SV40 promoter for a moderate level of dCas9-VP16 (BBa_K1150019) and the CMV promoter for a high level of our effector fusion protein (BBa_K1150020).
</P><CENTER><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>Figure 3: Position of the target loci on the SEAP reporter plasmid pKM602. </B>
The tested target loci are located in front of the SEAP promoter. Several distances and combinations of the related crRNAs were tested.
			</TD></TR></TBODY></TABLE></DIV></CENTER><DIV><TABLE><TBODY><TR><TD colspan="4"><B>Table 1: Overview over the tested crRNAs that guide our construct to different sites on the SEAP plasmid.</B>
 RNA plasmids containing the crRNAs EMX1, T2, T3 and T4 were transfected in combination with the activator dCas9-VP16. These RNAs target different loci on the SEAP plasmid. The crRNA sequences, their promoter distance and their binding site on the SEAP plasmid are displayed.
</TD></TR><TR><TH>	Name	</TH><TH>		Distance to promoter		</TH><TH>	Sequence	</TH><TH>		GC content [%]		</TH></TR><TR><TD>	EMX1	</TD><TD><CENTER>	-26 bp	</CENTER></TD><TD>	GGAAGGGCCTGAGTCCGAGCAGAAGAAGAA	</TD><TD><CENTER>	57	</CENTER></TD></TR><TR><TD>	T2	</TD><TD><CENTER>	-148 bp	</CENTER></TD><TD>	AAGCATTTATCAGGGTTATTGTCTCATGAG	</TD><TD><CENTER>	37	</CENTER></TD></TR><TR><TD>	T3	</TD><TD><CENTER>	-222 bp	</CENTER></TD><TD>	AATGCCGCAAAAAAGGGAATAAGGGCGACA	</TD><TD><CENTER>	47	</CENTER></TD></TR><TR><TD>	T4	</TD><TD><CENTER>	-443 bp	</CENTER></TD><TD>	GACCGAGTTGCTCTTGCCCGGCGTCAATAC	</TD><TD><CENTER>	60	</CENTER></TD></TR></TBODY></TABLE></DIV><P>
By targeting EMX1 and T2 with dCas9-VP16 driven by a SV40 promoter we gained 3 to 10 fold induction of SEAP expression (Figure 4). When we used these targets simultaneously a 17 fold activation could be achieved.
</P><CENTER><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 4: Results of the SEAP activation with dCas9-VP16 under control of the SV40 promoter using different crRNAs.</B>
HEK-293T cells have been transfected with different crRNA plasmids and dCas9 plasmids. The first row below the plot shows the different targets, the second the transfected plasmid. pRSet (junk DNA) and the SV40:dCas9-VP16 construct without crRNA remain as negative controls. To quantify the activation properties of dCas9-VP16 the activity of SEAP in the supernatant was measured and divided through the expression level of the luciferase Renilla, that served as an internal standard. Each sample was measured in biological triplicates. The bright green columns display the negative controls while the dark green bars reflect the different samples. The fold induction above each column is related to the basic SEAP expression level of the pRSet (junk DNA) control. 
			</TD></TR></TBODY></TABLE></DIV></CENTER><P>
For the dCas9-VP16 construct under control of the CMV promoter the crRNA single targets EMX1, T2, T3 and T4 has been tested. Additionally the combination of two targets (T3 &amp; T4 and EMX1 &amp; T2; Figure 5) was evaluated. All targets led to an activation of the SEAP expression in comparison to the off-target control. Compared to the samples without dCas9-VP16 only T4 showed no activation. The EMX1 crRNA seems to be the most efficient locus for single target activation. We achieved an 11 fold induction. T2 (BBa_K1150035) and EMX1 (BBa_K1150040) proved themselves to be activation sites.
By simultaneously using the EMX1 and T2 loci the highest SEAP induction could be achieved (29 fold induction; Figure 5).
</P><CENTER><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>Figure 5: Results of the SEAP activation with dCas9-VP16 under control of the CMV promoter using different crRNAs.</B>
HEK-293T cells have been transfected with different crRNA plasmids and dCas9 plasmids. The first row below the plot shows the different targets, the second row displays the transfected effector plasmid. pRSet (junk DNA), the CMV:dCas9 and the CMV:dCas9-VP16 construct without crRNA remain as negative controls. To quantify the activation properties of dCas9-VP16 the amount of SEAP activity in the supernatant was measured and divided by the expression level of the luciferase Renilla (internal standard). Each sample was measured in biological triplicates. Error bars display standard deviation. Bright green columns reflect the negative controls while dark green ones reflect the different samples. The fold induction above each column is related to the basic SEAP expression level of the pRSet (junk DNA) control. T3+4 crRNAs are transcribed from one RNA plasmid while T3 &amp; T4 crRNAs are transcribed from two independent RNA plasmids. 
			</TD></TR></TBODY></TABLE></DIV></CENTER><P id="h4">Activation of a fluorescent reporter </P><P> To demonstrate the broad functionality of activation via dCas9-VP16 we desired to activate a fluorescent mCherry reporter that is under the control of a CMV<SUB>min</SUB> promoter. This promoter is inactive as long as it is not activated externally. To do so we adressed dCas9-VP16 to different positions in front of the promoter in similar distances as for the previous SEAP activation. This should lead to an increase in fluorescence which can then to be measured via epifluorescent microscopy.</P><CENTER><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>Figure 6: Position of the target loci on the mCherry reporter plasmid pKM078. </B>
The tested target loci are located in front of the CMV<SUB>min</SUB> promoter. Several distances and combinations of the related crRNAs were tested.
			</TD></TR></TBODY></TABLE></DIV></CENTER><P> HEK-293T cells were transfected with the mCherry reporter as well as with dCas9-VP16 and the corresponding RNAimer plasmids coding for crRNAs adressing different target sites in front of the minimal promoter. Fluorescence was analyzed 48 hours post transfection using an AxioVert microscope.

</P><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>Figure 7: Results of fluorecent analysis of mCherry activation via dCas9-VP16. </B>
HEK-293T cells were transfected with a minimal pCMV:mCherry (pKM078) and dCas9-VP16 and corresponding RNAimer plasmids coding for different targets in front of the promoter. The first column shows the brightfield picutres, the second column the mCherry signal in red false-color. The third column shows the merge with mCherry in false-color red.
The first line displays the weak background of mCherry expression without activation. Nearly no fluorescence can be detected. When cotransfecting dCas9-VP16 and an RNA plasmid coding for targets 1 and 2, a mild increase in fluorescence can be observed (2nd row). The 3rd row displays strong fluorescence. Here two additional targets were adressed by contransfecting respective RNA coding plasmids. Adressing these 4 targets simultaneously leads to a even stronger increase in mCherry signal, what is in line with the previous SEAP results. When transfecting no RNA-coding plasmid at all (4th row) there is no difference in mCherry signal in contrast to the 1st row.  Exposure time = 1000ms. In every case the same amount of mCherry was transfected and was filled up to the needed amount with an empty vetor prSET. Scale bar = 200 um.
			</TD></TR></TBODY></TABLE></DIV><P>Fluorescence analysis confirmed our previous findings in SEAP activation. (Fig. 7) When transfecting dCas9-VP16 and RNAimer plasmids that adress positions in front of the promoter we see an increase in reporter expression. There is no observable increase in mCherry expression when no RNAimer plasmids are cotransfected. By adressing multiple targets simultaneously in front of the promoter we achieve a stricter activation of the mCherry.</P><P id="h3"> Discussion </P>

We were able to efficiently activate the expression of a reporter protein from a transfected plasmid (up to 29 fold). Different levels of activation were achieved by targeting different loci upstream of the promoter of the reporter protein. The best results were gained by targeting two loci simultaneously. 
Very recently a 25 fold activation of transient GFP expression by dCas9-VP64 was shown by Gilbert et al., however in comparison to cells that were not transfected with a GFP plasmid <SPAN id="refer"><A href="#(1,5)">[5]</A></SPAN>. So we were able to yield a higher effect of SEAP activation with dCas9-VP16.
With different promoters for dCas-VP16 the relations between the target sites were similar, but the activation of the SV40 driven dCas9-VP16 was slightly weaker. So SV40:dCas9-VP16 can be used to activate genes more moderately than with CMV:dCas9-VP16.<A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#targets">Click here</A> for detailed information about the target sites.



<P id="h4"> References </P><SMALL><DIV id="(1,1)">(1) Weir, J. (2001). Regulation of herpes simplex virus gene expression. Gene 271, <I>117-130</I>. </DIV><DIV id="(1,2)">(2) Triezenberg, S. <I>et al.</I> (1988). Functional dissection of VP16, the trans-activator of herpes simplex virus early gene expression. Genes Dev 2, <I>718-729</I>. </DIV><DIV id="(1,3)">(3) Greaves, R. and OâHare, P. (1989). Separation of requirements for protein-DNA complex assembly from those for functional activity in the herpes simplex virus regulatory protein Vmw65. J Virol 63, <I>1641-1650</I>. </DIV><DIV id="(1,4)">(4) Hirai, H., <I>et al.</I> (2010). Structure and functions of powerful trans-activators: VP16, MyoD and FoxA. Int. J. Dev. Biol. 54, <I>1589-1596</I>. </DIV><DIV id="(1,5)">(5) Gilbert, LA., <I>et al.</I> (2013). CRISPR-Mediated Modular RNA-Guided Regulation of Transcription in Eukaryotes. Cell. 154, <I>442-51</I>. </DIV></SMALL><TABLE id="headline_pic"><TBODY><TR><TD style="width:1000px;"><P id="h2">
Epigenetics
</P></TD></TR></TBODY></TABLE><P id="h3"> Histone modification by dCas9-G9a </P><P id="h4"> Introduction </P><P> For organisms it is crucial to have a tight control over their transcriptional machinery. As every cell has basically the same genetic information, different tissues have to be formed by alter the expression of this information. One of the most prominent mechanisms that give rise to this differential expression of genes are epigenetic modifications.
Epigenetics are, by definition, inheritable changes in gene expression, that are not accompanied by changes in the nucleotide sequence <SPAN id="refer"><A href="#(7)">[7]</A></SPAN>.
</P><P> There are several types of epigenetic modifications that may have severe impact on gene expression <SPAN id="refer"><A href="#(8)">[8]</A></SPAN>. Basically there are two main types of modifications. The first type are the chemical modification of cytosine residues of nucleotides, better known as DNA methylation. At so-called CpG islands, that can be found clustered in front of promoters the nucleotides can be altered by methylation. This methylation is a hallmark of repressive transcriptional states. 
But not only the DNA may be altered, but also the protein-nucleotide complex, called the chromatin, that forms the highly variable system that has massive impact on the differential regulation of expression. The probably most prominent epigenetic modifications are histone modifications. 
</P><P> Histones are proteins that work as multi-histone complexes and form a backbone for the DNA to be wound around. The termini of these nucleosomes are protruding from the histone core complex and can be targets for a lot of different modifications, which influence the chromatin state. One very prominent modification is methylation at lysine 9 of histone 3 (H3K9me) which renders before-open chromatin inactive. Such methylations are a hallmark of gene repression. 
One interesting fact about histone modification is the capability to spread the activity state over the surrounding chromatin via reader proteins. So the information of e.g. &quot;repressed state&quot; can, once specifically introduced, be propagated over a whole locus. 
</P><CENTER><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 6: CMV:dCas9-G9a </B>
dCas9 was fused via a 3 amino acid linker to G9a. The resulting fusion protein was flanked by NLS sequences and tagged by a HA epitope. The CMV promoter and BGH terminator were chosen to control gene expression. BBa_K1150023 is set under the control of the SV40 promoter, BBa_K1150024 is under the control of the CMV promoter. 
</TD></TR></TBODY></TABLE></DIV></CENTER><P> We extpand the uniCAS system by a histone methylase. By introducing specific histone modifications at several loci we should be able to regulate several genes at once. For this we fused a methyltransferase that is known to specifically perform certain histone tail methylations to the non-DNA-cleaving version of Cas9 (dCas9). </P><P>
For our device we used a part of the murine <I>EHMT2</I> gene, the G9a. It is described in literature that, when targeted to an open locus via zinc finger proteins, G9a is able to repress expression of this locus<SPAN id="refer"><A href="#(9)">[9]</A></SPAN>. </P><P>
We assayed the function as an epigenetic repressor. As a test subject we chose the VEGF locus, as it is
<OL><LI> well characterized </LI><LI> easy to measure by ELISA methods </LI><LI> VEGF is known to be involved in tumorgenesis and therefore an interesting target for future medical approaches </LI><LI> HEK293T cells have an open VEGF locus, so we do not have to artificially open the locus, before testing the system </LI></OL></P><CENTER><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 7: Principle of dCas9-G9a DNA interaction</B>
				The non-cleaving dCas9 fused to the histone methyltransferase G9a serves as a crRNA-guided DNA-binding protein and represses gene expression by histone methylation. Due to its helicase function, dCas9 unwinds the DNA-strand and the crRNA bind the DNA so that G9a can effect sequence specifically the DNA methylation state.
			</TD></TR></TBODY></TABLE></DIV></CENTER><P id="h4">Results </P><P> HEK293T cells were seeded into 24-well plates and transfected with our constructs, that were targeted to open regions <SPAN id="refer"><A href="#(10)"> [10] </A></SPAN> of the VEGF locus. </P><CENTER><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 8: Target sites on the endogenous VEGF locus. </B>
As the VEGF locus offers several accessible regions we chose four loci around the promoter. Arrows mark the regions. Numbers indicate position of the PAM sequence relative to the starting point of transcription.
</TD></TR></TBODY></TABLE></DIV></CENTER>
By specific histone modification through G9a we should see repression of VEGF secretion into the medium. 12 hours after transfection the medium was changed and 24 hours after medium change we harvested the supernatant and performed VEGF measurments by <B>ELISA</B>. Additionally we co-transfected a constitutive reporter (SEAP) as an internal standard to reduce mistake of non-transfected cells and cell number. A major problem of working with endogenous loci is the background of non-transfected cells which will display VEGF secretion, even though we have strong repression in other cells. <P> 
		As a control that the repressive effect of our proteins is not based on the sterical block of the transcription, we tested against the catalytic inactive dCas9. So every detectable effect is due to the G9a targeted to this locus<SPAN id="refer"><A href="#(11)">[11]</A></SPAN>.
		As we can see in the graph, we achieved a strong reduction in VEGF expression, especially when targeting the -8 and -573 loci. This is in line with literature results and plausible, when having in mind, that the promoter region is targeted and dense chromatin in promoters leads to repression.
	 </P><CENTER><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>Figure 9: Endogenous, stable repression by dCas9-G9a </B>
HEK-293T cells have been trancfected with the dCas9-G9a device and the RNAimer plasmid containing the different crRNA target sites for the endogenous VEGF locus. 12 hours after transfection the medium was changed and 24 hours after medium change we harvested the supernatant and performed VEGF measurments by ELISA. As a control that the repressive effect of our proteins is not based on the sterical block of the transcription, we tested against the catalytic inactive dCas9. So every detectable effect is due to the G9a targeted to this locus.(n=3, p&lt;0.05 is marked by asterisks) 
</TD></TR></TBODY></TABLE></DIV></CENTER><P id="h4"> Discussion </P><P> Our results obviously show a strong repression of VEGF expression around 50%, depending on the locus targeted. We can conclude from our results and literature data that we were able to specifically methylate histones and change the transcriptional state of the VEGF locus. Off-target controls do not display differences in VEGF expression, which indicates the specificity of our protein. 
As we always show the difference to the dCas9 protein without effector. This leads to the point that we only observe the catalytic activity of G9a.
This results in a valuable repressor tool that is able to specifically change histone methylation patterns and can change transcriptional states. It leads to many possible applications such as cancer research, fundamental epigenetic science or even tissue engineering. </P><SMALL><P id="h4"> References </P><DIV id="(7)">(7) Wolffe, A., <I>et al.</I> (1999). Epigenetics: Regulation Through Repression. Science 286. </DIV><DIV id="(8)">(8) Jones P. &amp; Baylin S. (2002). The Fundamental Role of Epigenetic Events in Cancer. Nature Reviews Genetics 3, <I>415-428</I>. </DIV><DIV id="(9)">(9) Snowden, A., <I>et al.</I> (2002). Gene-Specific Targeting of H3K9 Methylation Is Sufficient for Initiating Repression In Vivo. Curr Biol 12, <I>2159-2166</I>. </DIV><DIV id="(10)">(10) Liu, PQ., <I>et al.</I> (2000). Regulation of an Endogenous Locus Using a Panel of Designed Zinc Finger Proteins Targeted to Accessible Chromatin Regions.  Journ Biol Chem, 276, <I>11323-11334</I>. </DIV><DIV id="(11)">(11) Lee, D., <I>et al.</I> (2006). Histone 3 Lysine 9 Methyltransferase G9a Is a Transcriptional Coactivator for Nuclear Receptors. Journ Biol Chem 281, <I>8476-8485</I>. </DIV></SMALL><TABLE id="headline_pic"><TBODY><TR><TD style="width:1000px;"><P id="h2">
Repression
</P></TD></TR></TBODY></TABLE><P id="h3">
Introduction 
</P><DIV align="justify"><P>
KrÃ¼ppel-associated Box (KRAB) repressor domains are highly conserved polypeptide motifs and were first functionally characterized in 1991<SPAN id="refer"><A href="#(12)"> [12]</A></SPAN>. Their occurence in about one third of all human zinc finger transcription factors suggests key regulatory features in higher eukaryotic transcriptomics <SPAN id="refer"><A href="#(13)"> [13]</A></SPAN>. In terms of tetrapod evolution, the role of their great abundance has been extensively discussed <SPAN id="refer"><A href="#(14)"> [14]</A></SPAN>. Even though KRAB minimal domains are usually no longer than ~ 50-75 amino acids, their mechanism of function remains complex. Common biochemical models suggest a key role in epigenetic silencing, by recruiting a scaffold of diverse proteins to the zinc fingers' binding site - amongst others histone deacetylases and histone methyltransferases <SPAN id="refer"><A href="#(15)"> [15]</A></SPAN>. To date, KRAB domains were attached to several DNA binding proteins such as lacR and tetR, thereby silencing gene expression downstream of designed reporter targets.
</P><CENTER><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 10: Principle of transcriptional repression by a dCas9-KRAB fusion</B>
				The non-cleaving dCas9 was fused to repression domain KRAB to allow crRNA-guided DNA-binding. Due to its helicase function dCas9 unwinds the DNA-strand, the crRNA can bind the DNA and KRAB sequence specifically downregulates gene expression.
			</TD></TR></TBODY></TABLE></DIV></CENTER><P>
In this work, KRAB was fused to enzymatically inoperable dCas9. Thus, a transcriptional repressor with the flexibility to target almost any DNA sequence of interest was yielded. Transient SEAP expression could thus be reduced by almost 60%. In a second attempt, CMV-driven expression of the signaling scaffold protein CNK1 was targeted over 36 h - being partially knocked down to background amounts <SPAN id="refer"><A href="#(16)"> [16]</A></SPAN>. Furthermore, GFP reporter expression was shown to be drastically reduced by dCas9-KRAB in Fluorescence Microscopy imaging. Endogenous levels of VEGF-A, a key factor in hypoxic tumor angiogenesis <SPAN id="refer"><A href="#(17)"> [17]</A></SPAN>, were also successfully reduced and quantified through an Enzyme-Linked Immunosorbent Assay (ELISA). 
</P></DIV><CENTER><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>Figure 11: Design of the dCas9-KRAB fusion construct</B>
Through a seven amino acid linker, dCas9 was fused to KRAB. The resulting fusion protein was flanked by two NLS sequences. An HA-tag was also prefixed for Western blot detections. To drive gene expression, <A id="link" href="http://parts.igem.org/Part:BBa_K1150021">SV40</A> or <A id="link" href="http://parts.igem.org/Part:BBa_K1150022">CMV</A> promoters and the BGH terminator were chosen in our standardized constructs.
			</TD></TR></TBODY></TABLE></DIV></CENTER><P id="h3">
Results 
</P><P>
Repressional effects through our engineered dCas9-KRAB fusion protein were primarily evaluated by harnessing transient reporter assays. 
First, we used <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Assays">secreted embryonic alkaline phosphatase (SEAP)</A>, an enzyme which is thermostable and secreted into the medium of transfected mammalian cells. By photometrically measuring conversion rates of an exogenously added substrate (pNPP) at 405 nm, we were able to precisely determine the amounts of SEAP production. In our attempt, a tetO13-CMVmin-SEAP reporter plasmid was constitutively trans-activated by co-transfection of a second plasmid, coding for SV40-driven tetR-VP16. Thereby, CMVmin was efficiently upregulated and a high rate of SEAP production could be yielded. Thirdly, by co-transfecting dCas9-KRAB in combination with a crRNA that encodes a locus of 26 basepairs upstream of the CMVmin promoter, SEAP expression decreased by about 60 %. </P><CENTER><DIV><TABLE class="imgtxt" width="450px"><TBODY><TR><TD><B>Figure 12: dCas9-KRAB repression effects on SEAP production.</B>

Left: Positive controls consisted of co-transfections of SEAP vector with non-coding pRSET plasmid in HEK-293T cells. Right: dCas9-KRAB was crRNA-directed towards the tetR-VP16-<I>trans</I>activated SEAP gene. Similar DNA amounts as compared to pRSET were co-transfected with SEAP encoding plasmid. (<I>n=4</I>, error bars represent SD)
			</TD></TR></TBODY></TABLE></DIV></CENTER><P>A strongfold decreasement of trans-activated SEAP production was obtained when being co-transfected with dCas9-KRAB - as indicated by Figure 12. As a second proof-of-principle experiment, fluorescence microscopy was chosen. Therefore, we transfected HEK-293T cells with a <A id="link" href="http://parts.igem.org/Part:BBa_K1150033">self-built fluorescence-reporter plasmid</A>, which contained a CMV promoter and thus constitutively drove high expressions of GFP.</P><CENTER><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>Figure 13: dCas9-KRAB repression effects on GFP fluorescence.</B>
Upper images: As a negative control, non-coding pRSET was co-transfected with our GFP reporter in HEK-293T cells. Lower images: instead of pRSET (junk DNA), similar amounts of dCas9-KRAB were co-transfected with GFP reporter plasmid. Fluorescence images were taken with a light exposure time of 250 ms and 80% saturation.
			</TD></TR></TBODY></TABLE></DIV></CENTER><P>As shown in Figure 13, co-transfection of dCas9-KRAB - in combination with a crRNA corresponding to 30 basepairs of the CMV promoter - led to powerful disruption of fluorescence activity. Negative controls did not obtain any repressional effects. Next, in order to counteract gene expression of a signaling pathway component, ectopically expressed Connector Enhancer of KSR 1 (CNK1) was utilized for detection. CNK1 acts as a scaffold protein for a vast variety of signaling processes in cells, including those involved in proliferation and apoptosis <SPAN id="refer"><A href="#(16)"> [16]</A></SPAN>.</P><CENTER><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 14: dCas9-KRAB repression effects on <I>CNK1</I> protein expression.</B>
CNK1 was co-expressed in HEK-293T cells with either dCas9-KRAB or dCas9 alone. Cells were lysed at time points indicated and analysed by western blotting with anti-HA antibody to detect HA-CNK1 and HA-dCas9-KRAB expression. Endogenuous Akt was shown as loading control.
			</TD></TR></TBODY></TABLE></DIV></CENTER><P>In a time-series experiment, repressive effects were observed on Western Blots illustrated in Figure 14. Therefore, CNK1 was co-transfected with similar quantities of either dCas9 or dCas9-KRAB, providing an insight into the functional effects of the KRAB domain. It could be shown that co-expression of dCas9-KRAB, in comparison with dCas9 alone, dramatically decreases CNK1 levels - at least on basis of a transient attempt. Thus, potential effects of acting KRAB have risen in significance. Furthermore, as a key actor in signal transduction was transcriptionally repressed, these results represent a first step towards prospective impacts in medical research.</P><P> Next, investigation of further effects was to be proven on endogenous stages in human cells, as well. Similar to our experiments with epigenetically acting dCas9-G9a, we decided to also direct dCas9-KRAB towards the promoter region of the Vascular Endothelial Growth Factor A (VEGF-A) on Chromosome no 6. VEGF protein amounts could be detected through a specifically optimized sandwich version of an Enzyme-linked Immunosorbent Assay (ELISA). Normalization was performed by non-interacting SEAP. To assess dCas9-KRAB action, we co-transfected the fusion-protein plasmid with either crRNAs corresponding to the VEGF-A locus or to an EMX1 off-target on Chromosome no 2. The EMX1 protein is a prominent member of brain development stem cell regulators. On account of this, no effects on VEGF-A expression were expected for our off-target control of dCas9-KRAB actions.</P><CENTER><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 15: dCas9-KRAB regulation affects endogenous <I>VEGF</I> production.</B> dCas9-KRAB was crRNA-directed towards two targets of VEGF-A and EMX1 as an off-target control in HEK-293T cells. 12 hours after transfection, the medium was exchanged and 24 hours after medium change, we harvested supernatants and performed VEGF measurments by an ELISA. Additionally, we co-transfected a constitutive reporter (SEAP) as an internal standard to reduce mistake of non-transfected cells and cell number (<I>n=3</I>, error bars represent SD). 
			</TD></TR></TBODY></TABLE></DIV></CENTER><P>Regarding Figure 15, dCas9-KRAB has shown a clear ability to downregulate VEGF levels in a significant manner. By contrast with the applied off-target control, protein amounts detected were almost halved - both on sites upstream and downstream of the corresponding promoter. VEGF is a key factor in angiogenesis and has thus an important impact on tumor growth under hypoxic conditions. Therefore, an efficient repression of VEGF-A may lead to the delivery of insights into both fundamental spheres of complex endogenous regulation and cancer therapy. Future attempts of dCas9-KRAB directed repression may include different and more sophisticated endogenous targets - namely those involved in intricate cancer development.</P><P id="h3">
Expression problems
</P><P>However, standardized versions of both SV40:dCas9-KRAB and CMV:dCas9-KRAB have yet failed to yield successful results. Hitherto, protein expression of these constructs could not be truly shown on Western-Blots. Effects on transient secreted embryonic alkaline phosphatase reporters, similar to the first experiment, were shown nevertheless, while undesired off-target effects have also emerged.</P><CENTER><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 16: dCas9-KRAB off-target effects.</B>
Left: Positive controls consisted of co-transfections of SEAP vector with non-coding pRSET plasmid in HEK-293T cells. Middle: Standardized dCas9-KRAB was crRNA-directed towards the tetR-VP16-<I>trans</I>activated SEAP gene. Right: Standardized dCas9-KRAB without any crRNA providing plasmid was co-transfected with SEAP vector, thereby revealing unwanted amounts of repression activity. (<I>n=4</I>, error bars represent SD)
			</TD></TR></TBODY></TABLE></DIV></CENTER><P>Apparently, Figure 16 indicates an unfortunate off-target effect. Even though 30-nucleotide crRNAs against the SEAP promoter, repression effects were shown to yield a certain similarity for non-targeted SEAP reporter, too. To date, we interpret these unforeseen data as follows. Transfection of SV40:dCas9-KRAB and CMV:dCas9-KRAB may be crucially inhibited for several reasons. Firstly, one could suggest lower expression rates for both CMV and SV40 promoters which were used for our standardized constructs. This, in turn, does not fit with our findings that dCas9-VP16 and dCas9-G9a, our other transcriptional effectors, have been repeatedly well expressed and did not show any off-target effects. A more sophisticated hypothesis emerges from structural properties of the fusion protein. Both the enormous size of the transcript (4581 Bp), as well as a dozen structural coding deviances between standardized and non-standardized dCas9-KRAB, are very likely to having affected mRNA stability in a crucial manner. Expression values have been likewisely and repeatedly low for non-standardized constructs - Western Blot data not shown - but could be defected with certainty. Furthermore, following our assumptions, the associated fusion protein (1527 Aa) may have become impaired by unexpected post-translational modifications, especially referring to ubiquitylation and subsequent proteasomal degradation.  </P><P id="h3">
Outlook 
</P><P>Summarizing the results shown above, we have gained strong evidence of dCas9-KRAB effectivity - while specificity is still to be optimized and displayed on transient approaches in standardized vectors. Since we consider it as a duty to point out the lately observed and unwanted effects, we are keen to find out about the exact reasons. A very recent article of Gilbert et al. has already proven high functionality of this device - although strong repressive effects required stable lentiviral integration of dCas9-KRAB and HEK-cell pre-selection via Flow Cytometry had to be performed <SPAN id="refer"><A href="#(1,5)">[5]</A></SPAN>. Not least, our results of strongfold downregulation of endogenous VEGF-A promoter activity, equippes us with great enthusiasm. Especially for future medical advances and fundamental research in the field of multiplex gene regulation, we prospect that dCas9-KRAB will one day be utilized for more sophisticated transcriptional networks. Systemic diseases, stem cell differentiation, metabolic engineering: 21st century biology is about to take the next step forward towards complexity. Therefore, more sophisticated tools for diverse gene regulation will be required. dCas9-KRAB has a large potential in becoming an elegant synthetic biology device - so letâs share it with the iGEM community! </P><P id="h4"> References </P><DIV id="(12)">(12) Rosati, M. <I>et al.</I> (1991). Members of the zinc finger protein gene family sharing a conserved N-terminal module. Nucleic acids research 19, <I>5661-5667</I>. </DIV><DIV id="(13)">(13) Witzgall, R. <I>et al.</I> (1994). The KrÃ¼ppel-associated box-A domain of zinc finger proteins mediates transcriptional repression. PNAS 91, <I>4514-4518</I>. </DIV><DIV id="(14)">(14) Birtle, Z. and Ponting, C. (2006). Meisetz and the birth of the KRAB motif. Bioinformatics 22, <I>2841-2845</I>. </DIV><DIV id="(15)">(15) Urrutia, R. (2003). KRAB-containing zinc-finger repressor proteins. Genome Biology 4, <I>4:231</I>. </DIV><DIV id="(16)">(16) Fritz, R. and Radziwill, G. (2011). CNK1 and other scaffolds for Akt/FoxO signaling. Biochimica et biophysica acta 1813, <I>1971-1977</I>. </DIV><DIV id="(17)">(17) BaÅan, B. and SÅotwiÅski, R. (2008). VEGF and tumor angiogenesis. Centr Eur J Immunol 33, <I>232-236</I>. </DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Freiburg/Project/effector">http://2013.igem.org/Team:Freiburg/Project/effector</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Project/effector" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Project/effector" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Project/effector&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Project/effector&amp;oldid=361388" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>