<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Notebook_lab_repression"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Notebook/lab_repression">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Notebook/lab_repression&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_repression&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_repression&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Notebook/lab_repression" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Notebook/lab repression</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3></DIV><DIV id="menu"><UL id="navbar"><LI style="padding-left:15px"><A href="https://2013.igem.org/Team:Freiburg">Home</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Highlights">Highlights</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Introduction </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/induction">Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></LI></UL><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna">Targeting</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/method">uniBAss</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation">Standardization</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/protocols">Material and Methods</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/BioBricks">Our BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/favorite_parts">Favorite BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/improvement">Improvement of a Registry part</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/sharing">Sharing of our BioBricks </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/experts">Expert opinions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety/safety">Safety &amp; Security</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/politics">Politics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/society">Society</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/education">Education</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/art">Art</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/Facebook">Facebook / Media</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/News">News</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team2">Members</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Exolab">Exolab-Events</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Statistics">Statistics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/gallery">Photo Gallery</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Attributions">Our Partners</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/Coop">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/attributions"> Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/thanks">Thanks to</A></LI></DIV><DIV id="sidebar"><DIV id="sidebar_content"><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna"> Targeting </A></P><P class="first_order"><A class="active" href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector"> Effector </A></P><P class="second_order_note"><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_activation"> Activation </A></P><P class="second_order_note"><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_epigenetics"> Epigenetics </A></P><P class="second_order_note"><A class="active" href="https://2013.igem.org/Team:Freiburg/Notebook/lab_repression"> Repression </A></P><P class="third_order"><A href="#april"> April </A></P><P class="third_order"><A href="#may"> May </A></P><P class="third_order"><A href="#june"> June </A></P><P class="third_order"><A href="#july"> July </A></P><P class="third_order"><A href="#september"> September </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling"> Modeling </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/method"> uniBAss </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation"> Standardization </A></P><P class="first_order"><A id="link" href="https://2013.igem.org/Team:Freiburg/protocols"> Material and Methods </A></P></DIV></DIV><DIV id="main_contant_note"><DIV id="h1">
Effector - Repression - KRAB
</DIV><DIV id="april"><P id="h2">
April
</P></DIV><DIV id="tag"><H2> 21. April </H2><H3> Minipreps of pKM006, pKM102 &amp; pSAM200 </H3><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> pSAM200 </TD><TD> 320.1 </TD></TR><TR><TD> pKM006 </TD><TD> 113.3 </TD></TR><TR><TD> pKM102 </TD><TD> 180.0 </TD></TR></TBODY></TABLE></DIV><DIV id="may"><P id="h2">
May
</P></DIV><DIV id="tag"><H2> 02. May </H2><H3> PCR of pX334 fragments </H3><H4> PCR 1 <DIV><TABLE><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> pX334 </TD></TR><TR><TD> 1 </TD><TD> oIG2000 </TD></TR><TR><TD> 1 </TD><TD> oIG2029 </TD></TR><TR><TD> 2.5 </TD><TD> dNTPs </TD></TR><TR><TD> 1.5 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="$floatright"><UL><LI> Annealing: 60°C </LI><LI> Extension: 115 sec. </LI><LI> Cycles: 18 </LI></UL></DIV></H4><H4> PCR 2 <DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> pX334 </TD></TR><TR><TD> 1 </TD><TD> oIG2028 </TD></TR><TR><TD> 1 </TD><TD> oIG2001 </TD></TR><TR><TD> 2.5 </TD><TD> dNTPs </TD></TR><TR><TD> 1.5 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="$floatright"><UL><LI> Annealing: 60°C </LI><LI> Extension: 115 sec. </LI><LI> Cycles: 18 </LI></UL></DIV></H4><H4> PCR 3 <DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> pX334 </TD></TR><TR><TD> 1 </TD><TD> oIG2002 </TD></TR><TR><TD> 1 </TD><TD> oIG2003 </TD></TR><TR><TD> 2.5 </TD><TD> dNTPs </TD></TR><TR><TD> 1.5 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="$floatright"><UL><LI> Annealing: 60°C </LI><LI> Extension: 115 sec. </LI><LI> Cycles: 18 </LI></UL></DIV></H4><H4> PCR 4 <DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> pX334 </TD></TR><TR><TD> 1 </TD><TD> oIG2006 </TD></TR><TR><TD> 1 </TD><TD> oIG2007 </TD></TR><TR><TD> 2.5 </TD><TD> dNTPs </TD></TR><TR><TD> 1.5 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="$floatright"><UL><LI> Annealing: 60°C </LI><LI> Extension: 115 sec. </LI><LI> Cycles: 18 </LI></UL></DIV></H4><H4> PCR 5 <DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> pX334 </TD></TR><TR><TD> 1 </TD><TD> oIG2008 </TD></TR><TR><TD> 1 </TD><TD> oIG2009 </TD></TR><TR><TD> 2.5 </TD><TD> dNTPs </TD></TR><TR><TD> 1.5 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="$floatright"><UL><LI> Annealing: 60°C </LI><LI> Extension: 115 sec. </LI><LI> Cycles: 18 </LI></UL></DIV><P> All samples were stored at -20°C </P></H4><H2> 02. May </H2><H3> Agarose Gelelectrophoreses of PCR fragments No. 1-5 </H3><DIV><TABLE class="gelpic"><TBODY><TR><TD> f.l.t.r.: marker, PCR 1 (1909 bp), PCR 2 (2610 bp), PCR 3 (1645 bp), PCR 4 (2272 bp), PCR 5 (1878 bp) </TD></TR></TBODY></TABLE></DIV><H3> Gel Extraction of PCRs 1-6 </H3><TABLE class="tabelle"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> Fragment No. 1 </TD><TD> 40.7 </TD></TR><TR><TD> Fragment No. 2 </TD><TD> 53.6 </TD></TR><TR><TD> Fragment No. 3 </TD><TD> 52.6 </TD></TR><TR><TD> Fragment No. 4 </TD><TD> 41.6 </TD></TR><TR><TD> Fragment No. 5 </TD><TD> 40.4 </TD></TR></TBODY></TABLE><DIV id="tag"><H2> 03. May </H2><H3> Fusion PCR of PCR fragments 2-3 and 5-1  </H3><H4> Assembly fragment 2-3 (PCR 1) </H4><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> Fragment No. 2 </TD></TR><TR><TD> 1 </TD><TD> Fragment No. 3 </TD></TR><TR><TD> 4 </TD><TD> dNTPs </TD></TR><TR><TD> 1.5 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI>  Annealing: 71°C </LI><LI>  Extension: 105 sec. </LI><LI> Cycles: 5
		</LI></UL></DIV><H4> Assembly fragment 2-3 (PCR 2) </H4><DIV><UL><LI> Addition of 1 µl of oIG2028 and oIG2003 to PCR 1 </LI><LI> Annealing: 60 °C </LI><LI> Extension: 2 min 40 sec </LI><LI> Cycles: 16
	</LI></UL></DIV><H4> Assembly fragment 5-1 (PCR 1) </H4><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> Fragment No. 5 </TD></TR><TR><TD> 1 </TD><TD> Fragment No. 1 </TD></TR><TR><TD> 4 </TD><TD> dNTPs </TD></TR><TR><TD> 1.5 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI>  Annealing: 71°C </LI><LI>  Extension: 105 sec. </LI><LI> Cycles: 5
		</LI></UL></DIV><H4> Assembly fragment 5-1 (PCR 2) </H4><DIV><UL><LI> Addition of 1 µl of oIG2008 and oIG2029 to PCR 1 </LI><LI> Annealing: 60 °C </LI><LI> Extension: 2 min 40 sec </LI><LI> Cycles: 16
	</LI></UL><H3> Gel Extraction of Fusion fragments 2-3 and 5-1 </H3><TABLE class="tabelle"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> Fragment No. 2-3 (a) </TD><TD> 76.6 </TD></TR><TR><TD> Fragment No. 2-3 (b) </TD><TD> 57.5 </TD></TR><TR><TD> Fragment No. 2-3 (c) </TD><TD> 106.8 </TD></TR><TR><TD> Fragment No. 5-1 </TD><TD> 23.4 </TD></TR></TBODY></TABLE></DIV><H2> 17. May </H2><H3> PCR of the KRAB fragment with a N-terminal linker (GSAGSAG) overhang </H3><H4> PCR 1 <DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> pKM102 </TD></TR><TR><TD> 1 </TD><TD> oIG2004 </TD></TR><TR><TD> 1 </TD><TD> oIG2005 </TD></TR><TR><TD> 2.5 </TD><TD> dNTPs </TD></TR><TR><TD> 1.5 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="$floatright"><UL><LI> Annealing: 60°C </LI><LI> Extension: 30 sec. </LI><LI> Cycles: 18 </LI><LI> For higher Gel Extraction yields: 4x amount </LI></UL></DIV></H4><H3> Agarose Gelelectrophoreses of the KRAB fragment </H3><DIV><TABLE class="gelpic"><TBODY><TR><TD> f.l.t.r.: marker, KRAB at 490 bp (4x) </TD></TR></TBODY></TABLE></DIV><H3> Gel Extraction of the KRAB fragment </H3><TABLE class="tabelle"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> KRAB (a) </TD><TD> 76.0 </TD></TR><TR><TD> KRAB (b) </TD><TD> 64.7 </TD></TR></TBODY></TABLE></DIV><P> Two bands were respectively pooled on one extraction column, in order to yield higher amount of DNA. </P><H2> 20. May </H2><H3> Gibson Assembly of pIG2004 (Cas9-KRAB) </H3><DIV id="$floatright"><UL><LI> Preparation of 5µl DNA mix, containing all four fragments </LI><LI> Melting a 15µl master-mix on ice, until it is ready to use </LI><LI> Addition of 5µl of DNA mix to the master mix </LI><LI> Immediately put the mix on 50°C and incubate for 1h </LI><LI> 3 min on RT </LI><LI> 3 min on ice </LI><LI> Transformation of 4 µl to TOP10 chemically competent E-coli cells </LI></UL></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> Preparation scheme for Gibson Assembly fragment mix </TD></TR></TBODY></TABLE></DIV><H2> 21. May </H2><H3> Evaluation of Gibson Assembly </H3><DIV id="$floatright"><UL><LI> Colony growth </LI><LI> Colony PCR will be performed with primers within KRAB and Cas9 fragment </LI></UL></DIV><H4> Colony PCR Mastermix for Gibson Clones <DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 34.8 </TD><TD> H<SUB>2</SUB>O </TD></TR><TR><TD> 13.0 </TD><TD> Taq Buffer </TD></TR><TR><TD> 13.0 </TD><TD> dNTPs </TD></TR><TR><TD> 1.3 </TD><TD> oIG2002 </TD></TR><TR><TD> 1.3 </TD><TD> oIG2005 </TD></TR><TR><TD> 1.6 </TD><TD> Taq Polymerase </TD></TR><TR><TD> 5.0</TD><TD> H<SUB>2</SUB>O for each reaction, containing picked colony </TD></TR></TBODY></TABLE></DIV><DIV id="$floatright"><UL><LI> Annealing: 60°C </LI><LI> Extension: 68°C </LI><LI> Extension: 2 min 10 sec. </LI><LI> Cycles: 30 </LI></UL></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> Results of Colony PCR: Clones 5, 6 and 7 were striked out. </TD></TR></TBODY></TABLE></DIV></H4><H2> 22. May </H2><H3> Miniprep of suspected pIG2004 Clones 5, 6 and 7 </H3><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> pIG2004, No. 5 </TD><TD> 207.9 </TD></TR><TR><TD> pIG2004, No. 6 </TD><TD> 180.0 </TD></TR><TR><TD> pIG2004, No. 7 </TD><TD> 198.1 </TD></TR></TBODY></TABLE><P> Clones were sent to sequencing at GATC </P></DIV><H2> 23. May </H2><H3> Evaluation of Gibson Cloning Sequencing Results </H3><DIV><TABLE class="gelpic"><TBODY><TR><TD> oIG0007 sequencing reveals that the KRAB fragment was succesfully fused to Cas9 in each case. Further sequecing will be done to test for correct N-terminal assembly and H840A mutation. </TD></TR></TBODY></TABLE></DIV><H3> PCR of pX334 fragment 6 </H3><H4> PCR 1 <DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> pX334 </TD></TR><TR><TD> 1 </TD><TD> oIG2002 </TD></TR><TR><TD> 1 </TD><TD> oIG2007 </TD></TR><TR><TD> 2.5 </TD><TD> dNTPs </TD></TR><TR><TD> 1.5 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="$floatright"><UL><LI> Annealing: 60°C </LI><LI> Extension: 2 min 40 sec. </LI><LI> Cycles: 18 </LI></UL></DIV></H4><H3> Agarose Gelelectrophoreses of PCR fragments No. 6 </H3><DIV><TABLE class="gelpic"><TBODY><TR><TD> f.l.t.r.: marker, PCR 6 (3908 bp), empty, PCR 6 (2x) </TD></TR></TBODY></TABLE></DIV><H3> Gel Extraction of PCR 6 </H3><TABLE class="tabelle"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> Fragment No. 6 </TD><TD> 43.6 </TD></TR></TBODY></TABLE><H3> Gibson Assembly of pIG2005 (dCas9) </H3><DIV id="$floatright"><UL><LI> Preparation of 5µl DNA mix, containing all four fragments </LI><LI> Melting a 15µl master-mix on ice, until it is ready to use </LI><LI> Addition of 5µl of DNA mix to the master mix </LI><LI> Immediately put the mix on 50°C and incubate for 1h </LI><LI> 3 min on RT </LI><LI> 3 min on ice </LI><LI> Transformation of 4 µl to TOP10 chemically competent E-coli cells </LI></UL></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> Preparation scheme for Gibson Assembly fragment mix </TD></TR></TBODY></TABLE></DIV><H2> 24. May </H2><H3> Digest of pIG2004 (No. 5, 6 and 7) with BbsI </H3><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> Volume </TD><TD> DNA </TD></TR><TR><TD> 16 </TD><TD> pIG2004 </TD></TR><TR><TD> 10 </TD><TD> NEB-Buffer 2.1 </TD></TR><TR><TD> 2 </TD><TD> BbsI </TD></TR><TR><TD> 72 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Temp.: 37°C </LI><LI> Incubation time: 2h </LI></UL></DIV><H3> Agarose Gelelectrophoreses of pIG2004 Digests </H3><DIV><TABLE class="gelpic"><TBODY><TR><TD> f.l.t.r.: marker, pX334, empty, digested pIG2004 5 (2x), 6 (2x) and 7 (2x). Upper bands of clone no. 5 and 6 digests were cut out for Gel Extraction. </TD></TR></TBODY></TABLE></DIV><H3> Gel Extraction of pIG2004 digest </H3><TABLE class="tabelle"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> pIG2004 (5) </TD><TD> 38.0 </TD></TR><TR><TD> pIG2004 (6) </TD><TD> 17.0 </TD></TR></TBODY></TABLE><H3> Evaluation of Gibson Assembly of 23. May </H3><DIV id="$floatright"><UL><LI> Colony growth </LI><LI> Five clones were striked out </LI></UL></DIV><H2> 27. May </H2><H3> Evaluation of pIG2004 sequencing results (No. 5 and 6) </H3><UL><LI> H840A mutation was inserted in both cases, leading to catalytic inactivation of Cas9 </LI><LI> a 300 bp gap was inserted within the CAG promoter, probably due to error-prone PCR amplification </LI></UL><DIV><TABLE class="gelpic"><TBODY><TR><TD> Very GC-rich part on pIG2004 template (part of the CAG promoter lacks nearly 300 bp after having been amplified via PCR and subsequently Gibson-assembled.  </TD></TR></TBODY></TABLE></DIV><DIV id="june"><P id="h2">
June
</P></DIV><H2> 04. June </H2><H3> Minipreps of pIG2005 clones striked out on 23. May </H3><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> pIG2005 - No. 1 </TD><TD> 77.1 </TD></TR><TR><TD> pIG2005 - No. 2 </TD><TD> 68.4 </TD></TR><TR><TD> pIG2005 - No. 3 </TD><TD> 59.6 </TD></TR><TR><TD> pIG2005 - No. 4 </TD><TD> 43.7 </TD></TR><TR><TD> pIG2005 - No. 5 </TD><TD> 61.2 </TD></TR></TBODY></TABLE></DIV><H2> 07. June </H2><H3> Fixing of the 300 bp CAG promoter gap for Cas9-KRAB </H3><H4> Restriction digest of pIG2004 and pIG2005 - 1 <DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 8 </TD><TD> pIG2004 - No. 5 or pIG2005 - No. 1 </TD></TR><TR><TD> 8 </TD><TD> NEB-Buffer 4 </TD></TR><TR><TD> 2 </TD><TD> AgeI-HF </TD></TR><TR><TD> 2 </TD><TD> NotI-HF </TD></TR><TR><TD> 2 </TD><TD> SacII </TD></TR><TR><TD> 2 </TD><TD> BSA </TD></TR><TR><TD> Add to 80µl </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV></H4><H4> Restriction digest of pX334 - 2 <DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 8 </TD><TD> pX334 DNA </TD></TR><TR><TD> 8 </TD><TD> NEB-Buffer 4 </TD></TR><TR><TD> 2 </TD><TD> KpnI-HF </TD></TR><TR><TD> 2 </TD><TD> NotI-HF </TD></TR><TR><TD> 2 </TD><TD> SacI-HF </TD></TR><TR><TD> 2 </TD><TD> BSA </TD></TR><TR><TD> Add to 80µl </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV></H4><H4> Restriction digest of pX334 - 3 <DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 8 </TD><TD> pX334 DNA </TD></TR><TR><TD> 8 </TD><TD> NEB-Buffer 4 </TD></TR><TR><TD> 2 </TD><TD> KpnI-HF </TD></TR><TR><TD> 2 </TD><TD> AgeI-HF </TD></TR><TR><TD> 2 </TD><TD> BSA </TD></TR><TR><TD> Add to 80µl </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Temp.: 37°C </LI><LI> Incubation time: 1h </LI><LI> Expected fragment lengths: 1 - 4500 bp (Cas9-KRAB), 2 - 4900 bp (Backbone), 3 - 890 bp (CAG promoter) </LI></UL></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> Top left. Upper band: Cas9-KRAB fragment, Top right. Upper band: dCas9 fragment. Bottom left. Upper band: pX334 Backbone fragment. </TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> Top right. Lowest band (blurred): CAG promoter fragment. </TD></TR></TBODY></TABLE></DIV></H4><H3> Gel Extraction of digest fragments </H3><TABLE class="tabelle"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> Cas9-KRAB </TD><TD> 26.7 </TD></TR><TR><TD> dCas9 </TD><TD> 5.0 </TD></TR><TR><TD> pX334 Backbone </TD><TD> 14.4 </TD></TR><TR><TD> CAG promoter </TD><TD> 5.0 </TD></TR></TBODY></TABLE><H2> Ligation and transformation of intact pIG2004 and pIG2005 </H2><H3> Ligation of pIG2005 </H3><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 11.2 </TD><TD> CAG fragment </TD></TR><TR><TD> 4.4 </TD><TD> dCas9 fragment </TD></TR><TR><TD> 1.4 </TD><TD> pX334 backbone </TD></TR><TR><TD> 2 </TD><TD> T4 ligase buffer </TD></TR><TR><TD> 1 </TD><TD> T4 ligase </TD></TR></TBODY></TABLE></DIV><H3> Ligation of pIG2004 </H3><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 0.8 </TD><TD> Cas9-KRAB fragment </TD></TR><TR><TD> 14.8 </TD><TD> CAG fragment </TD></TR><TR><TD> 1.4 </TD><TD> pX334 backbone </TD></TR><TR><TD> 2 </TD><TD> T4 ligase buffer </TD></TR><TR><TD> 1 </TD><TD> T4 ligase </TD></TR></TBODY></TABLE></DIV><H2> 08. June </H2><H3> Evaluation of Transformations of pIG2005 and pIG2004 </H3><UL><LI> pIG2004 (30 µl strikeout): 24 colonies </LI><LI> pIG2005 (30 µl strikeout): 15 colonies </LI><LI> 12 colonies of both approaches were striked out for further sequencing </LI></UL><H2> 09. June </H2><H3> Minipreps of pIG2004 and pIG2005 </H3><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> pIG2004 - No. 1 </TD><TD> 198.4 </TD></TR><TR><TD> pIG2004 - No. 2 </TD><TD> 203.6 </TD></TR><TR><TD> pIG2004 - No. 3 </TD><TD> 202.0 </TD></TR><TR><TD> pIG2004 - No. 4 </TD><TD> 187.3 </TD></TR><TR><TD> pIG2004 - No. 5 </TD><TD> 195.3 </TD></TR><TR><TD> pIG2004 - No. 6 </TD><TD> 211.2 </TD></TR><TR><TD> pIG2004 - No. 7 </TD><TD> 187.4 </TD></TR><TR><TD> pIG2004 - No. 8 </TD><TD> 201.1 </TD></TR><TR><TD> pIG2004 - No. 9 </TD><TD> 201.7 </TD></TR><TR><TD> pIG2004 - No. 10 </TD><TD> 206.3 </TD></TR><TR><TD> pIG2004 - No. 11 </TD><TD> 212.3 </TD></TR><TR><TD> pIG2004 - No. 12 </TD><TD> 174.6 </TD></TR><TR><TD> pIG2005 - No. 1 </TD><TD> 202.0 </TD></TR><TR><TD> pIG2005 - No. 2 </TD><TD> 195.3 </TD></TR><TR><TD> pIG2005 - No. 3 </TD><TD> 189.9 </TD></TR><TR><TD> pIG2005 - No. 4 </TD><TD> 221.4 </TD></TR><TR><TD> pIG2005 - No. 5 </TD><TD> 205.4 </TD></TR><TR><TD> pIG2005 - No. 6 </TD><TD> 194.4 </TD></TR><TR><TD> pIG2005 - No. 7 </TD><TD> 206.7 </TD></TR><TR><TD> pIG2005 - No. 8 </TD><TD> 206.3 </TD></TR><TR><TD> pIG2005 - No. 9 </TD><TD> 220.4 </TD></TR><TR><TD> pIG2005 - No. 10 </TD><TD> 192.9 </TD></TR><TR><TD> pIG2005 - No. 11 </TD><TD> 188.8 </TD></TR><TR><TD> pIG2005 - No. 12 </TD><TD> 192.1 </TD></TR></TBODY></TABLE></DIV><UL><LI> Colonies No. 2 and 3 of pIG2004 were sent to sequencing, intending to validate (i) KRAB-insertion (ii) H840A conversion and (iii) an intact CAG promoter </LI><LI> Colonies No. 1 and 4 of pIG2005 were sent to sequencing, intending to validate (i) H840A conversion and (ii) an intact CAG promoter </LI></UL><H2> 11. June </H2><H3> Evaluation of Sequencing results of pIG2004 and pIG2005 </H3><UL><LI> pIG2004 clone No. 2 misses the KRAB fragment, but contains the CAG promoter </LI><LI> pIG2004 clone No. 3 contains KRAB fragment, H840A and the CAG promoter - and will therefore be used for further studies </LI><LI> pIG2005 clones No. 1 and 4 contain both H840 mutation and the CAG promoter </LI></UL><H4> Restriction digest of pIG2004 and pIG2005 with BbsI <DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 6 </TD><TD> DNA </TD></TR><TR><TD> 4 </TD><TD> NEB-Buffer 2.1 </TD></TR><TR><TD> 2 </TD><TD> BbsI </TD></TR><TR><TD> Add to 40µl </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> F.l.t.r.: marker, pIG2004 undigested, pIG2004 digested, pIG2005 undigested, pIG2005 digested (2x) </TD></TR></TBODY></TABLE></DIV></H4><H3> Gel Extraction of digested vectors pIG2004 and pIG2005 </H3><TABLE class="tabelle"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> pIG2004 </TD><TD> 16.1 </TD></TR><TR><TD> pIG2005 </TD><TD> 18.8 </TD></TR></TBODY></TABLE><H2> 12. June </H2><H3> Oligo annealing of target crRNAs </H3><TABLE><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 3 </TD><TD> Oligo No. 1 </TD></TR><TR><TD> 3 </TD><TD> Oligo No. 2 </TD></TR><TR><TD> 69 </TD><TD> NEB-Buffer 2 </TD></TR></TBODY></TABLE><UL><LI> Samples were heated up to 95 degrees for 5 minutes. Gradient cool down to room temperature for 2 hours. </LI><LI> Pair 1: oIG2010/oIG2011 - EMX1 </LI><LI> Pair 2: oIG2022/oIG2023 - Target 200 bp upstream of SEAP transcription initiation on pKM006 </LI><LI> Pair 3: oIG6008/oIG6009 - CMV </LI></UL><H2> 13-20. June </H2><H3> Ligation and transformation of crRNAs with Cas9-KRAB and dCas9 </H3><H4> Ligation of pIG2013 </H4><TABLE><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 15 </TD><TD> Annealed EMX1 </TD></TR><TR><TD> 3 </TD><TD> oIG2004 </TD></TR><TR><TD> 2 </TD><TD> T4 ligase buffer </TD></TR><TR><TD> 1 </TD><TD> T4 ligase </TD></TR></TBODY></TABLE><H4> Ligation of pIG2017 </H4><TABLE><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 15 </TD><TD> Annealed EMX1 </TD></TR><TR><TD> 3 </TD><TD> oIG2005 </TD></TR><TR><TD> 2 </TD><TD> T4 ligase buffer </TD></TR><TR><TD> 1 </TD><TD> T4 ligase </TD></TR></TBODY></TABLE><H4> Ligation of pIG2019 </H4><TABLE><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 15 </TD><TD> Annealed SEAP target </TD></TR><TR><TD> 3 </TD><TD> oIG2004 </TD></TR><TR><TD> 2 </TD><TD> T4 ligase buffer </TD></TR><TR><TD> 1 </TD><TD> T4 ligase </TD></TR></TBODY></TABLE><H4> Ligation of pIG2020 </H4><TABLE><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 15 </TD><TD> Annealed SEAP target </TD></TR><TR><TD> 3 </TD><TD> oIG2005 </TD></TR><TR><TD> 2 </TD><TD> T4 ligase buffer </TD></TR><TR><TD> 1 </TD><TD> T4 ligase </TD></TR></TBODY></TABLE><H4> Ligation of pIG2011 </H4><TABLE><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 15 </TD><TD> Annealed CMV target </TD></TR><TR><TD> 3 </TD><TD> oIG2004 </TD></TR><TR><TD> 2 </TD><TD> T4 ligase buffer </TD></TR><TR><TD> 1 </TD><TD> T4 ligase </TD></TR></TBODY></TABLE><H4> Ligation of pIG2015 </H4><TABLE><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 15 </TD><TD> Annealed CMV target </TD></TR><TR><TD> 3 </TD><TD> oIG2005 </TD></TR><TR><TD> 2 </TD><TD> T4 ligase buffer </TD></TR><TR><TD> 1 </TD><TD> T4 ligase </TD></TR></TBODY></TABLE><H3> Evaluation of transformation results </H3><UL><LI> Except for pIG2020, colonies were grown in each case </LI><LI> 6 colonies of pIG2011, pIG2013, pIG2015, pIG2017 and pIG2019 were respectively striked out </LI><LI> All attempts had to be repeated at least twice, as BbsI digestion often lead to unspecific digestion of direct-repeats next to crRNA insertion spacer. </LI></UL><H2> 18-25. June </H2><H3> MIDI-preps of pIG2004, pIG2005, pIG2011, pIG2013, pIG2015, pIG2017, pIG2019 and also pIG9000 </H3><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> pIG2004 (Cas9-KRAB) </TD><TD> 400 </TD></TR><TR><TD> pIG2005 (dCas9)</TD><TD> 203 </TD></TR><TR><TD> pIG2011 (Cas9-KRAB vs. CMV) </TD><TD> 610 </TD></TR><TR><TD> pIG2013 (Cas9-KRAB vs. EMX1) </TD><TD> 609 </TD></TR><TR><TD> pIG2015 (dCas9 vs. CMV) </TD><TD> 460 </TD></TR><TR><TD> pIG2017 (dCas9 vs. EMX1) </TD><TD> 520 </TD></TR><TR><TD> pIG2017 (dCas9 vs. EMX1) </TD><TD> 520 </TD></TR><TR><TD> pIG2019 (Cas9-KRAB vs. SEAP) </TD><TD> 430 </TD></TR><TR><TD> pIG9000 (Cas9 vs. EMX1) </TD><TD> 460 </TD></TR></TBODY></TABLE><UL><LI> MIDIs had to be repeated three times </LI><LI> During the first attempt, an unkown white precipitate was obtained - which could not be separated from the desired DNA. </LI><LI> Within the second attempt, DNA was not adequately treated during isopropanol treatment (has to be shaken). </LI></UL></DIV><H2> 25.-27. June </H2><H3> Test of Cas9-KRAB on Western Blots </H3><UL><LI> Cas9-KRAB and dCas9, both targetting the CMV promoter were respectively co-transfected to HEK 293 cells with another plasmid, encoding CMV driven CNK. </LI><LI> Both attempts were performed repeatedly, and cell lyses were than to be performed 12, 18, 36 and 42 hours post-transfection. </LI><LI> Transfections were performed with PEI </LI><LI> Lyses were performed with modified RIPA buffer </LI><LI> Biological double attempts were respectively done </LI></UL><H3> Transfection schemes for one 6-well plate </H3><H4> Cas9-KRAB vs. CMV of CNK (2x) </H4><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH> name </TH><TH>µg </TH></TR><TR><TD> CNK </TD><TD> 6 </TD></TR><TR><TD> pIG2011 </TD><TD> 12 </TD></TR></TBODY></TABLE></DIV><H4> dCas9 vs. CMV of CNK (2x) </H4><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH> name </TH><TH>µg </TH></TR><TR><TD> CNK </TD><TD> 6 </TD></TR><TR><TD> pIG2015 </TD><TD> 12 </TD></TR></TBODY></TABLE></DIV><H4> CNK negative control (2x) </H4><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH> name </TH><TH>µg </TH></TR><TR><TD> CNK </TD><TD> 6 </TD></TR></TBODY></TABLE></DIV><H4> Cas9-KRAB and dCas9 expression controls on one well </H4><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH> name </TH><TH>µg </TH></TR><TR><TD> pIG2004 </TD><TD> 1 </TD></TR><TR><TD> pIG2005 </TD><TD> 1 </TD></TR></TBODY></TABLE></DIV><H3> Results </H3><DIV><TABLE class="gelpic"><TBODY><TR><TD> ??? </TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> ??? </TD></TR></TBODY></TABLE></DIV><UL><LI> We expected a stronger decrease in CNK levels, when combining it with Cas9-KRAB than with dCas9 </LI><LI> Cas9-KRAB was able to highly efficiently repress CMV driven CNK expression </LI><LI> Cas9-KRAB was expressed. </LI></UL><H2> 29. June </H2><H3> Test of Cas9-KRAB vs. CMV-driven GFP </H3><UL><LI> Transfection of pIG6000 (CMV::GFP) in combination with either pIG2011 (Cas9-KRAB + crRNA agaisnt CMV) or Mock DNA </LI><LI> Cells were seeded previously seeded on 6-well plates </LI><LI> Incubation for about 36 hours </LI></UL><H4> dCas9 vs. CMV of GFP </H4><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH> name </TH><TH>µg </TH></TR><TR><TD> CMV-GFP </TD><TD> 1 </TD></TR><TR><TD> pIG2011 (Cas9-KRAB) </TD><TD> 2 </TD></TR><TR><TD> PEI </TD><TD> 9 </TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> Transfection scheme </TD></TR></TBODY></TABLE></DIV><DIV id="july"><P id="h2">
July</P></DIV><H2> 01. July </H2><H3> Fluorescence Microscopy Evaluation </H3><DIV><TABLE class="gelpic"><TBODY><TR><TD> Fluorescence Images </TD></TR></TBODY></TABLE></DIV><DIV id="${1:floatleft}"><H4> ImageJ: Cells treated with GFP and with pRSET (Mock-DNA) </H4><TABLE class="${2:tabelle}"><TBODY><TR><TH> Cell </TH><TH> Size </TH><TH> Intensity </TH></TR><TR><TD> 1 </TD><TD> 208 </TD><TD> 2920.9 </TD></TR><TR><TD> 2 </TD><TD> 146 </TD><TD> 3945.0 </TD></TR><TR><TD> 3 </TD><TD> 238 </TD><TD> 3064.8 </TD></TR><TR><TD> 4 </TD><TD> 166 </TD><TD> 3699.3 </TD></TR><TR><TD> 5 </TD><TD> 238 </TD><TD> 2071.3 </TD></TR><TR><TD> 6 </TD><TD> 192 </TD><TD> 2219.1 </TD></TR><TR><TD> 7 </TD><TD> 172 </TD><TD> 3819.9 </TD></TR><TR><TD> 8 </TD><TD> 228 </TD><TD> 3235.1 </TD></TR><TR><TD> 9 </TD><TD> 247 </TD><TD> 3510.0 </TD></TR><TR><TD> 10 </TD><TD> 146 </TD><TD> 3213.6 </TD></TR><TR><TD> 11 </TD><TD> 228 </TD><TD> 1977.4 </TD></TR><TR><TD> 12 </TD><TD> 224 </TD><TD> 3436.5 </TD></TR><TR><TD> 13 </TD><TD> 224 </TD><TD> 3436.5 </TD></TR><TR><TD> 14 </TD><TD> 136 </TD><TD> 1872.9 </TD></TR><TR><TD> 15 </TD><TD> 140 </TD><TD> 1611.4 </TD></TR></TBODY></TABLE></DIV><DIV id="${1:floatright}"><H4> ImageJ: Cells treated with GFP and Cas9-KRAB-DNA </H4><TABLE class="${2:tabelle}"><TBODY><TR><TH> Cell </TH><TH> Size </TH><TH> Intensity </TH></TR><TR><TD> 1 </TD><TD> 268 </TD><TD> 2407.6 </TD></TR><TR><TD> 2 </TD><TD> 176 </TD><TD> 3329.2 </TD></TR><TR><TD> 3 </TD><TD> 217 </TD><TD> 1336.1 </TD></TR><TR><TD> 4 </TD><TD> 166 </TD><TD> 1138.5 </TD></TR><TR><TD> 5 </TD><TD> 102 </TD><TD> 2844.8 </TD></TR><TR><TD> 6 </TD><TD> 217 </TD><TD> 936.6 </TD></TR><TR><TD> 7 </TD><TD> 148 </TD><TD> 1236.4 </TD></TR><TR><TD> 8 </TD><TD> 214 </TD><TD> 1032.9 </TD></TR><TR><TD> 9 </TD><TD> 177 </TD><TD> 1566.6 </TD></TR><TR><TD> 10 </TD><TD> 96 </TD><TD> 1461.9 </TD></TR><TR><TD> 11 </TD><TD> 228 </TD><TD> 1977.4 </TD></TR><TR><TD> 12 </TD><TD> 938 </TD><TD> 3371.7 </TD></TR><TR><TD> 13 </TD><TD> 102 </TD><TD> 992.9 </TD></TR><TR><TD> 14 </TD><TD> 210 </TD><TD> 982.1 </TD></TR><TR><TD> 15 </TD><TD> 112 </TD><TD> 1016.6 </TD></TR></TBODY></TABLE></DIV><UL><LI> Mean of GFP fluorescence intensity of cells treated with Cas9-KRAB is 1409.4 (SD: 880.6), cells co-transfected with Mock-DNA revealed 2718 (SD: 786.6) </LI><LI> Results should be of low validity, but are potentially indicative towards a repressive effect of Cas9-KRAB </LI></UL><H2> 02. July </H2><H3> MIDI-prep of pIG2005, pIG2019 and pRSET (Mock-DNA) </H3><P> Failed in each case, for unknown reasons (white salt fall-out after elution from column). Attempt will be repeated on the next day.</P><H2> 03. July </H2><H3> MIDI-prep of pIG2005, pIG2019 and pRSET (Mock-DNA) </H3><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> pIG2005 (dCas9) </TD><TD> 470 </TD></TR><TR><TD> pIG2019 (Cas9-KRAB vs. SEAP target)</TD><TD> 430 </TD></TR><TR><TD> pRSET (Mock DNA) </TD><TD> 790 </TD></TR></TBODY></TABLE></DIV><H2> 04. July </H2><H3> Transfection of pIG2019 in combination with pKM006 (SEAP plasmid) and pSAM200 (SV40::tetR-VP16) </H3><UL><LI> Biological tetraplicates (each attempt was repeated independently, for four different wells of one 6-well plate </LI><LI> Negative control: pKM006 (500 ng) + Mock-DNA (2500 ng) </LI><LI> Positive control: pKM006 (500 ng) + pSAM200 (500 ng) + Mock-DNA (2000 ng) </LI><LI> Cas9-KRAB test: pKM006 (500 ng) + pSAM200 (500 ng) + pIG2019 (2000 ng) </LI></UL><H2> 07. July </H2><H3> SEAP measurement </H3><UL><LI> 300µl of medium supernatants were removed from each well </LI><LI> Samples were heated to 65°C for 30 minutes </LI><LI> Centrifugation at 600 rpm (table top) for 1 minute </LI><LI> 80µl of each supernatant was transfered to one well of a 96-well plate, technical triplicates for each sample were pipetted </LI><LI> 100µl 2x SEAP Buffer was added to each well </LI><LI> 20µl pNPP substrate was added to each well </LI><LI> Air bubbles were quickly removed </LI><LI> Measurement was taken with a plate reader, for two consecutive hours with an interval lenght of one minute. Detection wavelength was 405 nm </LI></UL><DIV><TABLE class="gelpic"><TBODY><TR><TD> SEAP Results </TD></TR></TBODY></TABLE></DIV><DIV id="september"><P id="h2">
September
</P></DIV><H2> 01. September </H2><H3> Test of Cas9-KRAB vs. CMV-driven GFP, Repetition of the Experiment from 29. June on 24 well plates</H3><H4> dCas9 vs. CMV of GFP </H4><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH> name </TH><TH>µg </TH></TR><TR><TD> CMV-GFP </TD><TD> 0,04 </TD></TR><TR><TD> pIG2011 (Cas9-KRAB) </TD><TD> 0,46 </TD></TR><TR><TD> PEI </TD><TD> 1,5 </TD></TR></TBODY></TABLE></DIV><H4> dCas9 vs. CMV of GFP </H4><DIV id="${1:floatleft}"><TABLE class="${2:tabelle}"><TBODY><TR><TH> name </TH><TH>µg </TH></TR><TR><TD> CMV-GFP </TD><TD> 0,04 </TD></TR><TR><TD> pRSET (Mock-DNA) </TD><TD> 0,46 </TD></TR><TR><TD> PEI </TD><TD> 1,5 </TD></TR></TBODY></TABLE></DIV><H2> 03. September </H2><H3> Fluorescence Microscopy of KRAB-mediated GFP-repression </H3><DIV><TABLE class="gelpic"><TBODY><TR><TD> Results of GFP repression. Fluorescence images were taken with a light exposure time of 250 ms (80% saturation). Experiment shall be repeated with off-target controls and parallel Flow Cytometry analysis. </TD></TR></TBODY></TABLE></DIV><H2>09. September</H2><H3> Retrafo of Plasmids for the Midiprep</H3><UL>
Following plasmids are needed for activation and repression repeats:

<LI>pRSet</LI><LI>pKM600</LI><LI>pKM604</LI><LI>pKM605</LI><LI>pKM606</LI><LI>pKM607</LI><LI>pKM603</LI><LI>pIG2017</LI><LI>pIG2013</LI><LI>pSAM200</LI></UL>

Retrafo was inoculated into Ampicillin medium (concentration of 1:1000 from Ampicillin and medium). 

<H2>10. September</H2><H3>Midiprep</H3><P>Plasmids were prepped with the Midiprep kit of Promega according to the manufacturer's protocol.</P><H3>Seeding of cells</H3><P>4 24-well plates were seeded with 65,000 cells per well</P><H2>11. September</H2><H3>Transfection</H3><OL><LI>40 µl Opti-MEM + 2.25 µl PEI-solution were mixed in a 1.5 ml Eppi.</LI><LI>0.75 µg of the DNA of interest were prepaired in another Eppi .</LI><LI>Addition of the DNA to former Eppi, vortexing for 10 s and incubation for 15 min at RT</LI><LI>Solution was spread drop-wise to the cells in the dish</LI></OL><H3>Medium change</H3>
Medium was changed after 5 h.

<H2>13. September</H2><H3>Preparation for analyses</H3><UL><LI>Supernatant was collected for SEAP measurement.</LI><LI>Cells were frozen at - 80 °C for later on Western blot and luminescence measurement.</LI></UL><H2>14. September</H2><H3>SEAP measurement</H3><UL><LI>Collected supernatant and fresh cell culture medium was heated at 65 °C for 30 min, then centrifuged at 1250 g for 1 min.</LI><LI>Supernatant was diluted in cell culture medium 1:4 (25 µl supernatant + 75 µl medium).</LI><LI>80 µl of diluted supernatant was given to 100 µl of SEAP buffer.</LI><LI>After addition of 20 µl pNPP (substrate of SEAP) the absorbance of the mix was measured every minute for 2 h at 405 nm.
</LI></UL><H2>16. September</H2><H3>Cell lysis</H3><UL><LI>250 µl of lysis buffer (containing Tris-HCl, EGTA, MgSO<SUB>4</SUB>, DTT and protease inhibitor) were applied to the thawn cells of each well.</LI><LI>Incubation on ice for 10 min.</LI><LI>Centrifugation at 15,000 g for 4 min.</LI></UL><H3>Renilla luminescence measurement</H3><UL><LI>80 µl of supernatant were pipetted in a white 96 well plate.</LI><LI>Measurement of luminescence (every 2 min for 30 min) immediately after addition of 20 µl Renilla substrate in PBS.</LI></UL><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>SEAP activity normalized to Renilla expression</B></TD></TR></TBODY></TABLE>
Blue bars: The SEAP expression ranged between the expression of the CMV minimal promoter and the activation with TetR-VP16. Surprisingly the repression of Cas9-KRAB without a appropriate crRNA seemed to be stronger than the repression of Cas9-KRAB with this crRNA.
Red bars: When targeting the same locus, Cas9 leaded to less SEAP expression in combination with Cas9-VP16 than Cas9-KRAB in the same combination.
</DIV><H2>17. September</H2><H3>Protein precipitation</H3><P>As the cell lysis needed to be done with a high amount of lysis buffer, proteins are too much diluted for western blotting. Thus, they were precipitated:</P><UL><LI>The remaining lysates of the triplikates (~ 80 µl) were put together in a new epi.</LI><LI>Addition of TCA to a final concentration of 10 % and 1 µg BSA.</LI><LI>Incubation for 30 min on ice.</LI><LI>Centrifugation for 30 min at full speed and 4 °C.</LI><LI>Removal of supernatant and addition of 500 µl acetone.</LI><LI>Incubation on ice over night.</LI><LI>Centrifugation for 15 min at full speed and 4 °C.</LI><LI>Removal of acetone and nair drying for 20 min.</LI><LI>Addition of 20 µl 2x SDS-loading dye and 0.5 µl Tris-HCl (pH = 8.8).</LI><LI>Heating at 95 °C for 5 min.</LI></UL><H2>18. September</H2><H3>SDS-gel run</H3><P>SDS-gel was loaded with 18 µl of each sample. A voltage of 80 V was applied till the loading dye bands reached the border between collection and separation gel, then the voltage was increased to 120 V.</P><H3>Western Blotting</H3><UL><LI>Activation of PVDF-membrane for 10 min in methanol.</LI><LI>Short incubation of Whatman papers and the PVDF-membrane in transfer buffer.</LI><LI>Loading of western blot apperature: (-) - Whatman paper - SDS-gel - PVDF-membrane - Whatman paper - (+).</LI><LI>200 mA were applied for 1.5 h.</LI></UL><H3>Antibody treatment I</H3><UL><LI>Incubation of PVDF membrane in blocking buffer (4 % milkpowder in TBS-T) for 1 h.</LI><LI>Incubation with anti HA antibody (in 2 % milkpowder in TBS-T) over night at 4 °C.</LI><LI>3 x washing with TBS-T for 5 min, each.</LI><LI>Incubation with anti mouse antibody for 1 h.</LI><LI>4 x washing with TBS-T for 10 min, each.</LI><LI>Measurement of luminescence after addition of ECL I + ECL II solution.</LI><LI>Air drying of the membrane for further antibody treatments.</LI></UL><H2>19. September</H2><H3>Seeding of cells</H3><P>3 24-well plates were seeded with 65,000 cells per well.</P><H3>Antibody treatment II</H3><UL><LI>Reactivation in methanol.</LI><LI>1 x washing with TBS-T for 5 min.</LI><LI>Incubation of PVDF membrane in blocking buffer (4 % milkpowder in TBS-T) for 30 min.</LI><LI>Incubation with anti beta-actin antibody (in 2 % milkpowder in TBS-T) over night at 4 °C.</LI><LI>3 x washing with TBS-T for 5 min, each.</LI><LI>Incubation with anti mouse antibody for 1 h.</LI><LI>4 x washing with TBS-T for 10 min, each.</LI><LI>Measurement of luminescence after addition of ECL I + ECL II solution (500 µl each).</LI></UL><DIV><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Western blot </B></TD></TR></TBODY></TABLE>
In all tested wells there was an expression of Cas9, but a shift between Cas9 and Cas9-VP16 or Cas9-KRAB and Cas9-VP16 is not detectable.
</DIV><H2>20. September</H2><H3>Transfection</H3><OL><LI>40 µl Opti-MEM + 2.25 µl PEI-solution were mixed in a 1.5 ml Eppi.</LI><LI>0.75 µg of the DNA of interest were prepaired in another Eppi .</LI><LI>Addition of the DNA to former Eppi, vortexing for 10 s and incubation for 15 min at RT</LI><LI>Solution was spread drop-wise to the cells in the dish</LI></OL><P>Ratio of DNA amount (mass): 
reporter plasmid (SEAP) : effector plasmid (Cas9) : RNA plasmid
1 : 4 : 4 </P><H3>Medium change</H3>
Medium was changed after 4.5 h.  

<H2>22. September</H2><H3>Medium removal</H3><UL><LI>Supernatant was collected for SEAP measurement.</LI><LI>Cells were frozen at - 80 °C for later on Western blotting.</LI></UL><H3>Preparation for SEAP measurement</H3><P>Collected supernatant and fresh cell culture medium was heated at 65 °C for 30 min, then centrifuged at 1250 g for 1 min.</P><H2>23. September</H2><H3>SEAP measurement</H3><UL><LI>Supernatant of wells containing CMV:SEAP was diluted in cell culture medium 1:10 (10 µl supernatant + 90 µl medium).</LI><LI>80 µl of (diluted) supernatant was given to 100 µl of SEAP buffer.</LI><LI>After addition of 20 µl pNPP (substrate of SEAP) the absorbance of the mix was measured every minute for 2 h at 405 nm.
</LI></UL><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>SEAP activity [U/L]</B>
				pIG2004: Cas9-KRAB (not in iGEM standard); pIG2005: Cas9 (not in iGEM standard); PhyB: negative control; all other Cas9 constucts are in iGEM standard (with the indicated promoter)
			</TD></TR></TBODY></TABLE>
In comparison to the negative control the non standardized constructs repressed the SEAP expression only slightly, whereas there was a strong repression by the standardized constructs, when driven by a CMV promoter. The repression of Cas9-KRAB seemed to be a little stronger than the effect of Cas9 alone. But just like in the last experiment, the repression of Cas9-KRAB with a wrong crRNA was as strong as the repression with the right one.
</DIV><H3>Seeding of cells</H3><P>4 24-well plates were seeded with 65,000 cells per well</P><H2>24. September</H2><H3>Repeat of seeding of cells</H3><P>4 24-well plates were seeded with 65,000 cells per well again, as the cell density of the first seeding was too low.</P><H2>25. September</H2><H3>Transfection</H3><OL><LI>40 µl Opti-MEM + 0.75 µg of the DNA of interest were mixed in a 1.5 ml Eppi.</LI><LI>Addition of 2.25 µl PEI-solution, vortexing for 10 s and incubation for at least 15 min at RT</LI><LI>Solution was spread drop-wise to the cells in the dish</LI></OL><P>Ratio of DNA amount (mass): 
reporter plasmid (SEAP) : effector plasmid (Cas9) : RNA plasmid
1 : 4 : 4 
Additionally a plasmid coding for Renilla luciferase was cotransfected (5 % of DNA amount)</P><H2>26. September</H2><H3>Medium change</H3>
Medium was changed after 8 h.

  <H3>Cell lysis of transfection of 20. September</H3><UL><LI>80 µl of RIPA lysis buffer were applied to the thawn cells of each well.</LI><LI>Incubation on ice for 10 min.</LI><LI>Sonification 3 x 30 s at maximum.</LI><LI>Centrifugation at 10,000 g for 10 min.</LI><LI>40 µl of supernatant were mixed with 10 µl 5 x SDS sample buffer.</LI><LI>Heating for 5 min at 95 °C.</LI></UL><H3>SDS-gel run</H3><P>SDS-gel was loaded with 40 µl of each sample. A voltage of 80 V was applied till the loading dye bands reached the border between collection and separation gel, then the voltage was increased to 120 V.</P><H3>Western Blotting</H3><UL><LI>Activation of PVDF-membrane for 10 min in methanol.</LI><LI>Short incubation of Whatman papers and the PVDF-membrane in transfer buffer.</LI><LI>Loading of western blot apperature: (-) - Whatman paper - SDS-gel - PVDF-membrane - Whatman paper - (+).</LI><LI>200 mA were applied for 1.5 h.</LI></UL><H3>Antibody treatment I</H3><UL><LI>Incubation of PVDF membrane in blocking buffer (4 % milkpowder in TBS-T) for 1 h.</LI><LI>Incubation with anti HA antibody (in 2 % milkpowder in TBS-T) over night at 4 °C.</LI><LI>3 x washing with TBS-T for 5 min, each.</LI><LI>Incubation with anti mouse antibody for 1 h.</LI><LI>4 x washing with TBS-T for 10 min, each.</LI><LI>Measurement of luminescence after addition of ECL I + ECL II solution.</LI><LI>Air drying of the membrane for further antibody treatments.</LI></UL><DIV><TABLE class="imgtxt" width="600px"><TBODY><TR><TD><B>Western blot of the SEAP assay from 23.09. </B>
				CAG:dCas was strongly expressed, whereas CMV:dCas and dCas-KRAB were not visible on the western blot. Nevertheless a effect has been detected with the SEAP assay, so there should be an expression, but maybe too low for detection.
			</TD></TR></TBODY></TABLE></DIV><H2>27.09.13</H2><H3>Medium removal</H3><UL><LI>Supernatant was collected for SEAP measurement 48 h after transfection.</LI><LI>Cells were frozen at - 80 °C for later on Western blotting.</LI></UL><H2>28.09.13</H2><H3>SEAP measurement</H3><UL><LI>Collected supernatant and fresh cell culture medium was heated at 65 °C for 30 min, then centrifuged at 1250 g for 1 min.</LI><LI>Supernatant of wells containing CMV:SEAP was diluted in cell culture medium 1:15 (6.7 µl supernatant + 93.3 µl medium).</LI><LI>80 µl of (diluted) supernatant was given to 100 µl of SEAP buffer.</LI><LI>After addition of 20 µl pNPP (substrate of SEAP) the absorbance of the mix was measured every minute for 2 h at 405 nm.
</LI></UL><H3>Cell lysis for lyciferase measurement and western blot</H3><UL><LI>250 µl of lysis buffer (containing Tris-HCl, EGTA, MgSO<SUB>4</SUB>, DTT and protease inhibitor) were applied to the thawn cells of each well.</LI><LI>Incubation on ice for at least 10 min.</LI><LI>Centrifugation at 2,200 g for 30 min.</LI></UL><H3>Luciferase luminescence measurement</H3><UL><LI>80 µl of supernatant were pipetted in a white 96 well plate.</LI><LI>Measurement of luminescence (every 2 min for 15 min) immediately after addition of 20 µl Renilla substrate in PBS.</LI></UL><DIV><TABLE class="imgtxt" width="700px"><TBODY><TR><TD><B>SEAP-Assay: Results </B>
All different tested loci showed no clear result. dCas9-KRAB under the control of the CMV promotor showed no significant repression in comparison to the off-target and CRISPRi control. dCas9-KRAB under the control of the SV40 promotor showed no repression at all. Maybe targeting a gene coded on a plasmid is a problem for dCas9-KRAB to repress.
			</TD></TR></TBODY></TABLE></DIV><H2>29.09.13</H2><H3>Protein precipitation</H3><P>As the cell lysis needed to be done with a high amount of lysis buffer, proteins are too much diluted for western blotting. Thus, they were precipitated:</P><UL><LI>340 µl of acetone was added to 85 µl of the remaining lysates of one well per triplicate.</LI><LI>Incubation at - 20 °C for 1:15 h.</LI><LI>Centrifugation for 5 min at 10,000 g and 4 °C.</LI><LI>Removal of acetone and air drying for 5 min.</LI><LI>Addition of 25 µl 2x SDS-loading dye.</LI><LI>Heating at 95 °C for 5 min.</LI></UL><H3>SDS-gel run</H3><P>SDS-gel was loaded with 20 µl of each sample. A voltage of 80 V was applied untill the loading dye bands reached the border between collection and separation gel, then the voltage was increased to 120 V.</P><H3>Western Blotting</H3><UL><LI>Activation of PVDF-membrane for 10 min in methanol.</LI><LI>Short incubation of Whatman papers and the PVDF-membrane in transfer buffer.</LI><LI>Loading of western blot apperature: (-) - Whatman paper - SDS-gel - PVDF-membrane - Whatman paper - (+).</LI><LI>200 mA were applied for 1.5 h.</LI></UL><H3>Antibody treatment I</H3><UL><LI>Incubation of PVDF membrane in blocking buffer (4 % milkpowder in TBS-T) for 1 h.</LI><LI>Incubation with anti HA antibody (in 2 % milkpowder in TBS-T) over night at 4 °C.</LI><LI>3 x washing with TBS-T for 5 min, each.</LI><LI>Incubation with anti mouse antibody for 1 h.</LI><LI>4 x washing with TBS-T for 10 min, each.</LI><LI>Measurement of luminescence after addition of ECL I + ECL II solution.</LI><LI>Air drying of the membrane for further antibody treatments.</LI></UL><DIV><TABLE class="imgtxt" width="400px"><TBODY><TR><TD><B>Western blot of the SEAP assay from 28.09. </B>
Western blotting did not work well. Maybe lysis due to relative long incubation time on Renilla lysis buffer destroyed cells to much beforehand. So maybe only degradation products are visible.				
			</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Freiburg/Notebook/lab_repression">http://2013.igem.org/Team:Freiburg/Notebook/lab_repression</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Notebook/lab_repression" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Notebook/lab_repression" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_repression&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_repression&amp;oldid=343452" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>