<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Notebook_lab_epigenetics"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Notebook/lab_epigenetics">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Notebook/lab_epigenetics&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_epigenetics&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_epigenetics&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Notebook/lab_epigenetics" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Notebook/lab epigenetics</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3></DIV><DIV id="menu"><UL id="navbar"><LI style="padding-left:15px"><A href="https://2013.igem.org/Team:Freiburg">Home</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Highlights">Highlights</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Introduction </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/induction">Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></LI></UL><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna">Targeting</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/method">uniBAss</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation">Standardization</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/protocols">Material and Methods</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/BioBricks">Our BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/favorite_parts">Favorite BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/improvement">Improvement of a Registry part</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/sharing">Sharing of our BioBricks </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/experts">Expert opinions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety/safety">Safety &amp; Security</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/politics">Politics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/society">Society</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/education">Education</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/art">Art</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/Facebook">Facebook / Media</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/News">News</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team2">Members</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Exolab">Exolab-Events</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Statistics">Statistics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/gallery">Photo Gallery</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Attributions">Our Partners</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/Coop">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/attributions"> Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/thanks">Thanks to</A></LI></DIV><DIV id="sidebar"><DIV id="sidebar_content"><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna"> Targeting </A></P><P class="first_order"><A class="active" href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector"> Effector </A></P><P class="second_order_note"><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_activation"> Activation </A></P><P class="second_order_note"><A class="active" href="https://2013.igem.org/Team:Freiburg/Notebook/lab_epigenetics"> Epigenetics</A></P><P class="third_order"><A href="#june"> June </A></P><P class="third_order"><A href="#july"> July </A></P><P class="third_order"><A href="#august"> August </A></P><P class="third_order"><A href="#september"> September </A></P><P class="second_order_note"><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_repression"> Repression </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling"> Modeling </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/method"> uniBAss </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation"> Standardization </A></P><P class="first_order"><A id="link" href="https://2013.igem.org/Team:Freiburg/protocols"> Material and Methods </A></P></DIV></DIV><DIV id="main_contant_note"><DIV id="h1">
Effector - Repression - G9a
</DIV><DIV id="june"><P id="h2">
June
</P></DIV><DIV id="tag"><H2> 05.06.13 </H2><H3> G9a <I>mus musculus</I> DNA arrived from Stuttgart </H3><P> Lyophilized DNA on Whatman paper was incubated with 50µl sterile H<SUB>2</SUB>O (10min at RT) and eluted from the paper by centrifugation: 51.7ng/µl (NanoDrop).  20µl were sent for sequencing.
	    </P><H3> Transformation of G9a </H3><P> 4µl of G9a-DNA were transformed into 25µl Top10 competent <I>E. coli</I>. 250µl LB medium was added and cells were grown on 33°C for 1h30min before plating on an ampicillin agar plate.   
	    </P></DIV><DIV id="tag"><H2> 06.-07.06.13 </H2><H3> Clone selection </H3><P> Because no single clones could be picked, a dilution plating of the transformation was performed and two single clones were selected the next day and streaked out for mini preps. 
	    </P></DIV><DIV id="tag"><H2> 08.06.13 </H2><H3> Mini preps of G9a clones </H3><P> Clones were prepped with the Roche High Pure Plasmid Isolation Kit. Both clones were sent for sequencing with oIG8007 and oIG8008.
	    </P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> Clone </TH><TD> 1 </TD><TD> 2 </TD></TR><TR><TH> concentration in ng/µl </TH><TD> 61 </TD><TD> 76 </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 09.06.13 </H2><H3> Sequencing results </H3><P> Both clones contain G9a from <I>mus musculus</I>. 
	New primers to amplify G9a(mm) were designed and ordered. 
	    </P><P> insert primer list here? </P></DIV><DIV id="tag"><H2> 14.06.13 </H2><P> New primers arrived (oIG8009-oIG8012) 
	    </P><H3> PCR to amplify G9a-SET domaine </H3><P> aIG8000 (linker-G9a-NLS): oIG8009 (fw) and oIG8010 (rev) with G9a-Mini prep 1 (8.6.) as template. 
	  aIG8003 (NLS-G9a-linker): oIG8011 (fw) and oIG8012 (rev). 
	    </P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> Template </TD></TR><TR><TD> 1 </TD><TD> Primer1 </TD></TR><TR><TD> 1 </TD><TD> Primer2 </TD></TR><TR><TD> 4 </TD><TD> dNTPs </TD></TR><TR><TD> 1 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Annealing: 60°C </LI><LI> Elongation: 30 sec</LI><LI> 24 cycles </LI></UL></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> PCR aIG8000 and aIG8003 </TD></TR></TBODY></TABLE></DIV><P> Bands run too low, can be due to gelred. Check size after gelextraction by loading less DNA.
	    </P><H3> Gel extraction of PCR products aIG8000 and aIG8003d </H3><TABLE class="tabelle"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> aIG8000 </TD><TD> 115.6 </TD></TR><TR><TD> aIG8003 </TD><TD> 105.4 </TD></TR></TBODY></TABLE><H3> PCR to amplify Cas9 (on pX334a) </H3><P> We need the mutated non-cleaving dCas9. Therefore we use pIG2004 from Max as template to produce aIG8001 (oIG2000/oIG8002) and aIG8002 (oIG2006/oIG2009).
	  5µl of Q5 buffer were used instead of 10µl. Elongation: 4min20sec.
	    </P><DIV><TABLE class="gelpic"><TBODY><TR><TD> PCR aIG8001, Gelex aIG8003(size correct), marker, aIG8002 </TD></TR></TBODY></TABLE></DIV><H3> Gel extraction of PCR products aIG8001 and aIG8002 </H3><TABLE class="tabelle"><TBODY><TR><TH> name </TH><TH> ng/µl </TH></TR><TR><TD> aIG8001 </TD><TD> 19.2 </TD></TR><TR><TD> aIG8002 </TD><TD> 11.5 </TD></TR></TBODY></TABLE></DIV><DIV id="tag"><H2> 15.06.13 </H2><H3> PCR for aIG 8004, aIG8008, aIG 8007, aIG 8010 and aIG 8011 </H3><UL><LI> aIG8004: primers: oIG2008 and 8005; template: pIG2004 </LI><LI> aIG8007: primers: oIG8009 and 8014; template: G9a(mm) </LI><LI> aIG8008: primers: oIG8013 and 8010; template: G9a(mm) </LI><LI> aIG8010: primers: oIG8014 and 8011; template: G9a(mm) </LI><LI> aIG8011: primers: oIG8013 and 8012; template: G9a(mm) </LI></UL><P> PCR reaction was performed as described below </P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> Template </TD></TR><TR><TD> 1 </TD><TD> Primer1 </TD></TR><TR><TD> 1 </TD><TD> Primer2 </TD></TR><TR><TD> 4 </TD><TD> dNTPs </TD></TR><TR><TD> 1 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Annealing: 60°C </LI><LI> Elongation: 120 sec</LI><LI> 24 cycles </LI></UL></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> PCR aIG8001, Gelex aIG8003(size correct), marker, aIG8002 </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 19.06.13 </H2><H3> Gibson Cloning for pIG8002 and pIG8004 </H3><P> Even though aIG8001 did not yield good results by gelEx, we performed Gibson cloning, as PCR was not repeatable. Gibson mix was prepared as described below. </P><UL><LI> pIG8002 </LI><UL><LI> aIG8006: 0.35µl</LI><LI> aIG8001: 1.85µl</LI><LI> aIG8002: 2.80µl</LI></UL><LI> pIG8004</LI><UL><LI>aIG8001: 1.75µl</LI><LI>aIG8002: 2.64µl</LI><LI>aIG8009: 0.617µl</LI></UL></UL><DIV><P> DNA mix was added to the aliquoted Gibson reaction mix and was incubated for 60 minutes at 50°C, 3 minutes on ice and 3 minutes on RT. 5µl of the mixture was transformed into competent E.coli and plated on Amp(+) plates. 
		    </P></DIV></DIV><DIV id="tag"><H2> 20.06.13 </H2><P> Clones were obtained for pIG8002, but not for pIG8004. clones from pIG8002 were streaked out on plates for miniprep. 
		</P></DIV><DIV id="tag"><H2> 21.06.13 </H2><H3> test digest of pIG8002 </H3><P> Clones were prepped using the Roche kit and eluted in 50µl ddH<SUB>2</SUB>O. Concentrations were roughly 230ng/µl. A test digest was performed using EcoRV and NotI. If the correct plasmid is obtained, three bands will be visible (at 5850bp, 3400bp and 1700bp).
		</P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> Substance </TH></TR><TR><TD> 1 </TD><TD> Plasmid </TD></TR><TR><TD> 1 </TD><TD> buffer 4</TD></TR><TR><TD> 0.5 </TD><TD> EcoRV-HF </TD></TR><TR><TD> 0.5 </TD><TD>  NotI-HF </TD></TR><TR><TD> 7 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Incubation: 1.5 h at 37°C</LI></UL></DIV><H3> Results of test digest </H3><DIV><TABLE class="image"><TBODY><TR><TD> 15 out of 17 colonies were positive. Clone 3 and 5 were sent for sequencing with oIG0009, oIG8007 and oIG8008. </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 22.06.13 </H2><H3> Gibson for pIG8004 </H3><H3> Sequencing results of pIG8002 </H3><P> Sequencing confirmed clone 5 as pIG8002. The catalytically active dCAS9:G9a-SD is complete. Clone 3 showed a deletion in between promoter and ATG. We do not expect this mutation to influence the protein, but, nevertheless, clone 5 is completely correct. 
		</P><H3> Repetition of Gibson for pIG8004 (dCAS9:dG9a) </H3><P>As the first Gibson did not yield any clones we repeated it. DNA mix was prepared as described below. </P><UL><LI> aIG8001: 1,75ul </LI><LI> aIG8002: 2,64ul </LI><LI> aIG8009: 0,617ul </LI></UL><P> DNA mix was added to the aliquoted Gibson reaction mix and was incubated for 60 minutes at 50°C, 3 minutes on ice and 3 minutes on RT. 5µl of the mixture was transformed into competent <I>E. coli</I> and plated on Amp(+) plates. </P></DIV><DIV id="tag"><H2> 23.06.13 </H2><P> 11 Clones were obtained and picked. They were streaked out on plates for further analysis. </P></DIV><DIV id="tag"><H2> 24.06.13 </H2><H3> Test digest of pIG8004 </H3><P> Clones were prepped using the Roche kit. Test digest was performed using EcoRV and NotI. See table below for composition of test digest. 
		</P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> Substance </TH></TR><TR><TD> 1 </TD><TD> Plasmid </TD></TR><TR><TD> 1 </TD><TD> buffer 4</TD></TR><TR><TD> 0.5 </TD><TD> EcoRV-HF </TD></TR><TR><TD> 0.5 </TD><TD>  NotI-HF </TD></TR><TR><TD> 7 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Incubation: 1.5h at 37°C</LI></UL></DIV><H3> Results of digest </H3><DIV><TABLE class="image"><TBODY><TR><TD> 11 out of 12 clones were positive, as all 3 bands were visible as expected. Numer 3 and 5 were sent for sequencing with oIG0009, oIG8007 and oIG8008. </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="july"><P id="h2">
July
</P></DIV><DIV id="tag"><H2> 02.07.13 </H2><H3> Sequencing results of pIG8004 and starting of target site ligation </H3><P> pIG8004 was confirmed as positive. The next step is to ligate target sites designed by Manuel into both constructs (from here on called VEGF1-4, respectively pIG8002_1-4 and pIG8004_1-4), for functional testing of our construct. To do so, the construct will be opened with BbsI and afterwards the annealed oligos will be ligated into the construct with T4 ligase. Digest was performed as follows.</P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> Substance </TH></TR><TR><TD> 5 </TD><TD> Plasmid </TD></TR><TR><TD> 5 </TD><TD> buffer 2.1</TD></TR><TR><TD> 2 </TD><TD> BbsI </TD></TR><TR><TD> 39 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Incubation: 2h at 37°C </LI></UL></DIV><H3> Results of digest </H3><DIV><TABLE class="image"><TBODY><TR><TD> loading scheme: marker - pIG8002(digested) - pIG8002(control) - pIG8004(digested) - pIG8004(control). No big shift is visible when loading only undigested control.</TD></TR></TBODY></TABLE></DIV><DIV><P> digested bands were cut and extracted using the Qiagen gelEx kit. Yields were roughly 20ng/ul. The next day they will be ligated with the oligos. </P></DIV></DIV><DIV id="tag"><H2> 03.07.13 </H2><P> Ligation of the annealed oligos into pIG8002 and pIG8004. 50ng of opened vector is ligated with 150-200ng of insert. A strong excess of small oligo is suspected to work more efficiently than less. 
		</P><UL><LI> pIG8002+oKM511/512:pIG8002_1 </LI><LI> pIG8002+oKM513/514:pIG8002_2 </LI><LI> pIG8002+oKM515/516:pIG8002_3 </LI><LI> pIG8002+oKM517/518:pIG8002_4 </LI><LI> pIG8002+oKM511/512:pIG8004_1 </LI><LI> pIG8002+oKM513/514:pIG8004_2 </LI><LI> pIG8002+oKM515/516:pIG8004_3 </LI><LI> pIG8002+oKM517/518:pIG8004_4 </LI></UL><P> ligation was performed as follows: </P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> Substance </TH></TR><TR><TD> 2.5 </TD><TD> vector </TD></TR><TR><TD> 2 </TD><TD> insert </TD></TR><TR><TD> 1 </TD><TD> T4 buffer</TD></TR><TR><TD> 0.5 </TD><TD> T4 Ligase </TD></TR><TR><TD> 4 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Incubation: 1h at 22° C</LI></UL></DIV><P> 5µl of each transformation assay was transformed into competent <I>E. coli</I> (batch 8.5.), following the standard protocol. </P></DIV><DIV id="tag"><H2> 04.07.13 </H2><H3> Test digest of ligated crRNAs </H3><P> Clones were obtained for all constructs. Minipreps were performed and yielded roughly 230ng/µl. Test digest was performed with BbsI and XhoI as described several times above. Results are inconclusive, so pIG8002_1 and pIG8004_1 were sent for sequencing with oIG0015. </P><DIV><TABLE class="image"><TBODY><TR><TD> Test digest - no difference is visible</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 05.07.13 </H2><P> pIG8002_1 was sequenced as correct, pIG8004_1 was negative, BbsI seemed to have cut too much of the plasmid, one Direct Repeat is missing. Strange. rest was sent for sequencing with oIG0015. 
		</P></DIV><DIV id=""><H2> 09.07.13 </H2><P> Sequencing revealed positive clones for pIG8002_1 and 3, as well as for pIG8004_4. Negative colonies were discarded. More preps were done to screen for further clones.  
		</P><UL><LI> 8002_2.2 </LI><LI> 8002_4.2 </LI><LI> 8004_1.2 </LI><LI> 8004_2.2 </LI><LI> 8004_3.2 </LI></UL></DIV><DIV id="tag"><H2> 10.07.13 </H2><P> 8004_3.2 is positive, rest negative. For the missing constructs one of each clone was sent. 
		</P></DIV><DIV id="tag"><H2> 11.07.13 </H2><P> positive clones were obtained for 8002_2.3, 8002_4.3, and 8004_1.3. New clones were sent for sequencing. 
		</P></DIV><DIV id="tag"><H2> 12.07.13 </H2><P> positive clone for 8004_2.4 was found. All constructs are done. Midi preps were inoculated in 150ml LB with 150µl Amp. As well as control plasmids without target sites. Flasks were directely inoculated by giving a little bit of colony with a sterile tip into the flask. 
		</P></DIV><DIV id="tag"><H2> 15.07.13 </H2><P> Midi preps were performed using the JetStar Midi Kit. Pellet was resolved in 100µl of ddH<SUB>2</SUB>O. All preps yielded acceptable results (500-1900 ng/µl).  HEK293T cells were seeded into 24 well plates. Our construct should be able to efficiently repress VEGF expression by methylating H3K9. As HEK293 cells have an open locus we should see a reduction in VEGF levels. 65,000 cells were seeded per well.
			 Each construct will be transfected in biological triplicates, let grow for 48 hours and the supernatant harvested. ELISA measurments will be performed using the peprotech VEGF ELISA Development kit. </P></DIV><DIV id="tag"><H2> 16.07.13 </H2><P> Transfection of the cells was performed as described in the table below. PIF-GFP was transfected as a transfection control to have a fast impression of transfection efficiency. Next transfection should be calculated for 2-3 wells more to have a buffer for pipetting mistakes. Recalibrate pipette?
		</P><DIV><TABLE class="image"><TBODY><TR><TD> Transfection scheme 16.07.</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 17.07.13 </H2><P> Transfection seems to have worked, as we see green fluorescence in the nuclei of the cells.  VEGF ELISA kit arrived. Antibodies were aliquoted following the instructions and stored at -20°C. Buffers were prepared as follows:
		</P><UL><LI> PBS: dilute 100ml 10xPBS in 900ml ddH<SUB>2</SUB>O. </LI><LI> PBS-T: 1xPBS+0,05% Tween-20 </LI><LI> Blockbuffer: 31ml PBS+ 0,31g BSA </LI><LI> Diluent: 45ml PBS-T+0,045g BSA </LI></UL><P> Every buffer containing BSA is stored at 4°C. ELISA plate was coated with capture antibody as described in the instructions.
		</P></DIV><DIV id="tag"><H2> 18.07.13 </H2><H3> ELISA detection of VEGF content of supernatant </H3><P> All steps were performed following the protocol of Peprotech, except the following steps: Blocking time was elongated to 2 hours, and washing was performed 3 times with 300µl wash buffer (PBS-T). Samples were used diluted and 1:2 diluted for ELISA detection. The loading scheme of the ELISA plate is shown below.
		</P><DIV><TABLE class="image"><TBODY><TR><TD> ELISA and plate scheme from 16.07.</TD></TR></TBODY></TABLE></DIV><H3> results of ELISA </H3><DIV><TABLE class="image"><TBODY><TR><TD> Results of the first VEGF ELISA from 16.07., error bars represent standard deviation.</TD></TR></TBODY></TABLE></DIV><P> Direct comparison of 8002 and 8004 constructs show less VEGF in 8002 (active G9a) as expected. Controls show very low amounts as well. This is rather strange, but gives a hint that our primary idea is correct. Several possible problems and solutions wre identified. 
		</P><UL><LI> VEGF is secreted when cells are exposed to oxygen stress. The distribution of O2 could be varying inside the plate, messing with our results. The use of filters instead of lids could be a solution. </LI><LI> The use of an internal standard could be a possible solution for reducing cell toxicity of G9a and gives us as well a hint on cell number/transfection efficiency. A SV40:SEAP plasmid will serve as this. </LI><LI> Triplicates should be mandatory, if possible </LI></UL></DIV><DIV id="tag"><H2> 22.07.13 </H2><P> HEK293T cells were seeded at 70,000 cells/well in 24-well plates. </P></DIV><DIV id="tag"><H2> 23.07.13 </H2><P> Transfection was performed as described in scheme and protocol from 16.07.. Each well was co-transfected with 0.0375µg of SV40:SEAP (commercial pK2 SEAP control plasmid by CloneTech (R)) that will serve as internal standard. The cells were quite dense, so we will have to evaluate transfection efficiency carefully. </P><DIV><TABLE class="image"><TBODY><TR><TD> Transfection scheme for repeated experiment from 16.07. 5% SV-40 will serve as internal standard.</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 24.07.13 </H2><P> Cells were too dense, transfection efficiency was low =&gt; Cells were discarded. Seeding of cells will be repeated. </P></DIV><DIV id="tag"><H2> 27.07.13 </H2><P> HEKs were seeded at 65,000 cells per well in 24-well plates. We were advised by Bea to assure good distribution of cells. </P></DIV><DIV id="tag"><H2> 28.07.13 </H2><P> Cells show strange behaviour: They form a clump in the middle of the well, whereas the outer space is too lose. This will reduce the transfection efficiency strongly in the middle of the well, making our experiment quite difficult. Cells were discarded. again, cells were seeded with the help of Adrian. </P></DIV><DIV id="tag"><H2> 29.07.13 </H2><P> Same procedure as everyday. We suspect the cells to be misbehaving. We acquired new cells from AG Warscheidt for testing other cells. A contamination could explain the problems in growth.  for testing we transfected the old cells anyway to check efficiency. 
		</P></DIV><DIV id="tag"><P> New cells were seeded at 65.000 cells per well into 24 well plates. (HEK293 cells). Cells were resuspended and singled with great(!) caution. We checked the transfection efficiency of the old cells which was below 10%. The cells were discarded. 
	    </P></DIV><DIV id="august"><P id="h2">
August
</P></DIV><DIV id="tag"><H2> 01.08.13 </H2><P> Cells are not dense enough for transfection (~30%), so transfection was postponed to the next day. 
		</P></DIV><DIV id="tag"><H2> 02.08.13 </H2><P> Cells were transfected following the standard protocol as described in the following tables at 12.a.m.. 
		</P><DIV><TABLE class="image"><TBODY><TR><TD> Loading scheme of the plates used in the second ELISA experiment.</TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="image"><TBODY><TR><TD> Transfection scheme for ELISA experiment</TD></TR></TBODY></TABLE></DIV><P> The plates were covered with O<SUB>2</SUB> diffusible filters to prevent oxygen stress to occur and messing up our data. 
		</P></DIV><DIV id="tag"><H2> 03.08.13 </H2><P> Medium of transfected cells was changed at 10 a.m., to repress the native VEGF level to a minimum. Transfection efficiency was around 50% when assaying the PIF-GFP construct. We are confident, that this will suffice for our experiment. 
		</P></DIV><DIV id="tag"><H2> 04.08.13 </H2><P> Supernatant of cells was harvested (400µl) for further analysis. 200µl are used for the VEGF ELISA, whereas the other 200µl were frozen to perform a SEAP measurement the next days as internal standard. ELISA was performed as described above.  Additionally cell lysis was performed using modified RIPA buffer for expression control of our construct by Western blot. Lysates were cooked for 5 minutes in sample buffer and frozen for later blotting. 
		</P><P> VEGF locus 2 and 3 were displaying strong (!) repression this time (see graph) in comaprison to the control bar. But controls are varying strongly in VEGF expression. Even more, locus 1 behanves really strange. Maybe the cells have died in this wells? Western Blot will show the problems. 
		</P><DIV><TABLE class="image"><TBODY><TR><TD> Results of the second ELISA experiment performed. VEGF is measured using ELISA. Controls are varying strongly. Maybe normalization to SEAP will solve the problem. </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 05.08.13 </H2><P> SEAP measurement was performed as described in the standard protocol. All VEGF levels were divided by the SEAP level to get a normalization to an internal standard. Error was propagated using Gauß's mistake propagation.  by looking at the data now it is obvious that sample 4_1 can be discarded for further analysis. But nevertheless the rest of the samples is looking great. Even the controls are within range of one standard deviation. Strangely PIF:GFP displayed a very strong SEAP expression, lowering the bar in the normalized plot. But loci VEGF2 and VEGF3 display really strong effects. 
		</P><DIV><TABLE class="image"><TBODY><TR><TD> Results of the second ELISA experiment performed. VEGF is normalized to SEAP. Error bars represent standard deviation. </TD></TR></TBODY></TABLE></DIV><P> For further expression analysis we were using a Western blot approach. To do so, 2 8% Acrylamide gels were pourn with 50% Sucrose added. They were stored at 4°C and will be ran tomorrow. 
	    </P></DIV><DIV id="tag"><H2> 06.08.13 </H2><P> Protein samples were run on an 8% gel at 130 Volt. Afterwards a semidry blot was performed at 70mA to transfer protein onto an VDF-membrane. Primary antibody (anti-HA and anti-ß-act) was diluted 1:2500 in 2% milkpowder. Blocking was performed using 4% milk in PBS-T for 1 hour. Primary incubation took place overnight at 4°C on a vertical shaker.
		</P></DIV><DIV id="tag"><H2> 07.08.13 </H2><P> Membrane was washed 3x for 5 minutes with 1x PBS-T. Secondary antibody (anti-mouse) was diluted 1:5000 and applied to the membrane for 2 hrs at RT. An additional washing step was performed 3 times for 5 minutes in 1x PBS-T. </P><H3> Detection of Western Signal&lt;</H3><P> Membranes were treated with 600µl of ECLI and II each. Signal was detected using the LS imageQuant. Results are displayed in the following figure. 
		</P><DIV><TABLE class="image"><TBODY><TR><TD> Western blot with the samples of second ELISA experiment. Evident expression of Cas9-G9a is visible at all lanes (even for 4_1, but very weak). Nevertheless, loading controls are varying strongly, indicating badly performed cell lysis. </TD></TR></TBODY></TABLE></DIV><P> We see expression of dCAS9:G9a in each well, except 4_1, which supports our hypothesis, that here a problem with the cells occurred. The ß-actin levels are varying strongly what leads to the impression, that cell lysis was not performed perfectly . Nevertheless, we could detect our protein in each well were we saw effect. Today is a good day! 
		</P></DIV><DIV id="tag"><H2> 08.08.13 </H2><P> Further planning after discussion: A kinetic should be done, but should be planned carefully. </P></DIV><DIV id="tag"><H2> 12.08.13 </H2><P> Plans for Kinetics:</P><UL><LI> Kinetics will be performed in two ways: Either measuring production of VEGF in 24hours over 5 days, and VEGF levels over 5 days, without medium change. </LI><LI> To reduce variability in between plates we will transfect in 10cm dishes, and spread afterwards in 24 well plates. </LI><LI> 6 24-well plates will be used </LI><LI> Filters and SEAP standard will be used </LI></UL></DIV><DIV id="tag"><H2> 14.08.13 </H2><P> All constructs were transfected into HEK293 cells in 10cm-dishes. For DNA mix, see the following table. Every construct was treated the same. </P><DIV><TABLE class="image"><TBODY><TR><TD> Western blot with the samples of second ELISA experiment. Evident expression of Cas9-G9a is visible at all lanes (even for 4_1, but very weak). Nevertheless, loading controls are varying strongly, indicating badly performed cell lysis. </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 15.08.13 </H2><P> To detach cells from the plates, medium was removed. Afterwards they were washed in 5ml PBS.  PBS was removed and another 10ml of PBS was added. Cells detached using a cell scraper centrifuged and resuspended in 4ml DMEM. 75% of a plate was diluted in 9.5ml medium. 500µl was put to each well of a 24-well plate. Plates were seeded according to following scheme. 
		</P><DIV><TABLE class="image"><TBODY><TR><TD> Cell plate scheme and experimental setup. </TD></TR></TBODY></TABLE></DIV><DIV id="tag"><H2> 16-21.08.13 </H2><P> Everyday supernatant was harvested and lysates were taken using modified RIPA buffer. They were frozen and evaluated on the 22.08. </P></DIV><DIV id="tag"><H2> 16.08.13. </H2><H3> Standardisation of G9a in RFC 25 <P> Standardisation of G9a will be performed using a Gibson approach. 2 cutting sites have to be mutated in the protein sequence. Primers were designed that contain the mutation in the overhangs. With those primers PCR will be performed and Gibson will ligate the whole thing in the pSB1C3 backbone with RFC25 prefix and suffix.  
			All oligos arrived for the standardisation. PCR was performed on pIG8002 with: 
		</P><UL><LI> oIG8015 + 8017 =&gt; aIG8100 </LI><LI> oIG8018 + 8019 =&gt; aIG8101 </LI><LI> oIG8020 + 8016 =&gt; aIG8102 </LI></UL><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> Template </TD></TR><TR><TD> 1 </TD><TD> Primer1 </TD></TR><TR><TD> 1 </TD><TD> Primer2 </TD></TR><TR><TD> 4 </TD><TD> dNTPs </TD></TR><TR><TD> 1 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Annealing: 60°C </LI><LI> Elongation: 30 sec</LI><LI> 24 cycles </LI></UL></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> all PCRs yielded desired products.  loading scheme: aIG8100 - marker - aIG8101 - marker - aIG8102 </TD></TR></TBODY></TABLE></DIV><P> All bands were extracted using the Roche kit. Gibson was performed using the three fragments and the opened pSB1C3 backbone. for DNA mix see the following table. Reaction was performed as described in the standard protocol. 
	    </P><P> 5µl of the reaction mix were transformed into competent <I>E. coli</I> following the standard protocol. Bacteria was spread on Cap containing LB plates. 
	    </P></H3><H2> 16.08.13 </H2><H3> Red light inducible PhyB-G9A and Cas9-PIF system </H3><H4> PCR amplification of the backbone containing PhyB for Gibson </H4><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> pKM018 (30ng) </TD></TR><TR><TD> 1 </TD><TD> oIG8021 </TD></TR><TR><TD> 1 </TD><TD> oIG8022 </TD></TR><TR><TD> 2.5 </TD><TD> dNTPs </TD></TR><TR><TD> 1.5 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI>  Annealing: 60°C </LI><LI>  Extension: 2min 30sec </LI></UL></DIV><H4> PCR amplification of G9A with specific overhangs to the PhyB-bb </H4><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> pIG8002 </TD></TR><TR><TD> 1 </TD><TD> oIG8023 </TD></TR><TR><TD> 1 </TD><TD> oIG8024 </TD></TR><TR><TD> 2.5 </TD><TD> dNTPs </TD></TR><TR><TD> 1.5 </TD><TD> DMSO </TD></TR><TR><TD> 0.5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI>  Annealing: 60°C </LI><LI>  Extension: 40sec </LI></UL></DIV><H5> Results </H5><DIV><TABLE class="gelpic"><TBODY><TR><TD> Left side: 2log laddder; On the right side: G9A; everything else are different samples </TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> Left: 2log ladder, right: PhyB bb </TD></TR></TBODY></TABLE></DIV><P> All bands had the expected size (gelred standard) and were band isolated and purificated </P></DIV><DIV id="tag"><H2> 19.08.13 </H2><H3> Red light inducible PhyB-G9A and Cas9-PIF system </H3><P> A Gibson was performed using the PCR products of PhyB-bb and G9A in a ratio of 1/8 (58.39ng PhyB; 0.9 43.25ng G9A). The control plate showed nearly as many colonies and a test digest using Hind3 and EcoRI only revealed one positive colony. The sequencing showed a frameshift therefore a Gibson has to be repeated by using another ratio (maybe 1/4). </P></DIV><DIV id="tag"><H2> 22.08.13 </H2><H3> Red light inducible PhyB-G9A and Cas9-PIF system </H3><P> Another gibson approach of G9A PhyB-bb using a 1/4 ratio </P><P> Gibson approach: </P><H4> Test digest of some Gibson colonies using Sal1 and EcoR1 </H4><DIV><TABLE class="gelpic"><TBODY><TR><TD> Left: 2 log ladder; then tested Gibson colonies from 1-10. Colony 5-10 seem to be defenitly possitive. </TD></TR></TBODY></TABLE></DIV><P> Colony 5 was send for sequencing </P><H3> Ligation of two VEGF crRNAs into pIG3010 </H3><P> For light experiments a plasmid containing the tracrRNA and crRNA is needed. </P><H3> Ligation </H3><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl: </TH><TH> Substance: </TH></TR><TR><TD> 1 </TD><TD> pIG3010 </TD></TR><TR><TD> 5 </TD><TD> crRNA2/3 (2/3 refere to the different VEGF target sides) </TD></TR><TR><TD> 2 </TD><TD>  10X T4-Ligase Buffer </TD></TR><TR><TD> 1 </TD><TD> T4-Ligase </TD></TR><TR><TD> 11 </TD><TD> H<SUB>2</SUB>O </TD></TR><TR><TD> 20 </TD><TD> Total </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> 0.5h at RT </LI></UL></DIV></DIV><DIV id="tag"><H2> 27.08.13 </H2><H3> Red light inducible PhyB-G9A and Cas9-PIF system </H3><H4> Sequencing results of pIG8007 (PhyB-G9A) - the sixed colony from test digest was send for sequencing </H4><P> A frameshift is inside the sequencing. Colonies 9 and 10 were send for sequencing and prepared for midi prep 
		</P><H3> crRNA-tracr Plasmid with two VEGF target sides </H3><P> Colonies of crRNA ligation were minipreped and 2 colonies of each locus (2.1 2.2 3.1 3.2) were send for sequencing and emediatly prepared for minipreps.
		</P><H3> First cellculture experiment </H3><P> Two 24-well plates were prepared with HEK293T cells (65,000 cells/well). </P></DIV><DIV id="tag"><H2> 17.08.13 </H2><P> Colonies were obtained for the standardized G9a-SD. To screen for positive ones, colony-PCR was performed as described in the standard protocol. The following PCR mix was used. 
		</P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 2.5 </TD><TD> taq buffer</TD></TR><TR><TD> 1 </TD><TD> oIG6017 </TD></TR><TR><TD> 1 </TD><TD> oIG6018</TD></TR><TR><TD> 2.5 </TD><TD> dNTPs </TD></TR><TR><TD> 0.125 </TD><TD> Taq-polymerase</TD></TR><TR><TD> Add to 25 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Annealing: 60°C </LI><LI> Elongation: 45 sec</LI><LI> 24 cycles </LI></UL></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> One lane shows the desired band at roughly 1kb. Upper row, left lane.</TD></TR></TBODY></TABLE></DIV><P> Positive colony was streaked out for miniprep. </P></DIV><DIV id="tag"><H2> 18.08.13 </H2><P> Colony was miniprepped and sent for sequencing with oIG6018. 
		</P></DIV><DIV id="tag"><H2> 19.08.13 </H2><P> Sequencing confirmed the clone. <B> G9a is standardized! </B></P></DIV><DIV id="tag"><H2> 22-23.08.13 </H2><P> All the samples from the kinetic were evaluated following the usual protocols. VEGF content was measured using ELISA methods and the SEAP was assayed by measuring the activity following the standard protocol. The next day, the data will be evaluated.
		</P></DIV><DIV id="tag"><H2> 24.08.13 </H2><P> Data was evaluated. We do not see repressive effects in any of the kinetics. The experiment failed miserably (see figure below for details). Possible explanations are: 
		</P><UL><LI>VEGF is a stress response gene, that could have been activated by transfering the cells. </LI><LI>There is no effect of G9a - all effects are simply measuring mistakes </LI><LI>We screwed up transfection </LI></UL><P> For the sake of time, the experiment will  not be repeated, but the original experiment will be repeated to prove to reproducibility of our data. 
		</P><H4> Results of kinetic </H4><DIV><TABLE class="image"><TBODY><TR><TD> Results between the kinetic were the medium was not changed. We do not see any differences in between the constructs. </TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="image"><TBODY><TR><TD> Results of the kinetic where the medium was changed every 24 hours. Again, we do not see any differences in between the constructs. possible explanations above. </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="september"><P id="h2">
September
</P></DIV><DIV id="tag"><H2> 06.09.13 </H2><P> As several other experimentators were not able to reproduce our results, we decided to do it again, as described earlier in this labbook. </P><P> Cells were transfected following the standard protocol as described in the following tables at 12.a.m.. </P><P> The plates were covered with O<SUB>2</SUB> diffusible filters to prevent oxygen stress to occur and messing up our data. </P></DIV><DIV id="tag"><H2> 07.09.13 </H2><P> Medium was changed at 8.a.m. to lower the native VEGF level after transfection. Supernatant will be harvested 24 hours later. </P></DIV><DIV id="tag"><H2> 08.09.13 </H2><P> Supernatant was harvested, ELISA was performed and SEAP was assayed as described above. We can draw the following conclusions from it (see figure below).</P><UL><LI> the results are reproducible, even though not to the extend as last time </LI><LI> the endogenous VEGF contents are strongly varying - Even though of the usage of an internal standard </LI><LI> We should stop doing experiments, until the devices are finally built. </LI><LI> the number of passages may influence the chromatin structure of the VEGF locus - maybe use fresh cells next time </LI></UL><DIV><TABLE class="image"><TBODY><TR><TD> VEGF analysis to confirm our results - again, the first locus (-8) does not show any effect. But the other effects could at least be confirmed, concerning their tendencies. </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 24.09.13 </H2><P> Everything is stadardized. cells were seeded in 3 24-well plates according to following scheme. They were seeded at 70,000 cells/well. Fresh cells were acquired to ensure chromatin stability. 
		</P></DIV><DIV id="tag"><H2> 25.09.13 </H2><P> Cells were seeded too lose, maybe a problem with the cell counter. New cells were seeded usign half a 100% plate for 3 24-well plates. Also new midis were inoculated for VEGF target 2,3 and 1. 
        </P></DIV><DIV id="tag"><H2> 26.09.13 </H2><P> Cells reached optimal confluence. Transfection as performed following the standard protocol and the following schemes.</P><DIV><TABLE class="image"><TBODY><TR><TD>Transfesction scheme for final VEGF ELISA. </TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="image"><TBODY><TR><TD>Transfection calculations (1) for final VEGF ELISA. </TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="image"><TBODY><TR><TD>Transfection calculations (2) for final VEGF ELISA. </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2>27.09.13</H2><P> Transfection was assayed using the GFP construct, that was transfected. We received signal in nearly 90% of the cells. We deceided to advance with the experiment. Medium was changed at approx. 11am, to reduce native VEGF levels.</P></DIV><DIV id="tag"><H2> 28.09.13 </H2><P> Supernatant was harvested and ELISA detection was performed following the standard protocol provided by Peprotech. The loading scheme of the 96 well plate is done as follows.Evalutaion will be performed after SEAP assay was performed. </P><DIV><TABLE class="image"><TBODY><TR><TD>Loading scheme for ELISA plate. </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2>29.09.13</H2><P> SEAP assay was performed, diluted 1:10. Standard protocol was followed. SEAP levels seem to be quite small. ELISA and SEAP data was evaluated and the ratio of VEGF/SEAP was calculated. (see following graph)One can see, that we have the repression of the VEGF level, even more than the CRISPRi. This is interesting and important, as the Cas9 protein is higher expressed than the cas9:G9a fusion. These are very nice results, especially, that the off-target controls do not show any differences. Error bars are standard deviation of VEGF and SEAP propagated via Gauß. </P><DIV><TABLE class="image"><TBODY><TR><TD>VEGF/SEAP ration, reflecting VEGF levels. </TD></TR></TBODY></TABLE></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Freiburg/Notebook/lab_epigenetics">http://2013.igem.org/Team:Freiburg/Notebook/lab_epigenetics</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Notebook/lab_epigenetics" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Notebook/lab_epigenetics" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_epigenetics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_epigenetics&amp;oldid=343446" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>