<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Project_1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Project/1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Project/1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Project/1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Project/1&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Project/1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Project/1</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3></DIV><DIV id="menu"><UL id="navbar"><LI style="padding-left:15px"><A href="https://2013.igem.org/Team:Freiburg">Home</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Highlights">Highlights</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Introduction </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/induction">Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></LI></UL><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna">Targeting</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/method">uniBAss</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation">Standardization</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/protocols">Material and Methods</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/BioBricks">Our BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/favorite_parts">Favorite BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/improvement">Improvement of a Registry part</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/sharing">Sharing of our BioBricks </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/experts">Expert opinions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety/safety">Safety &amp; Security</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/politics">Politics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/society">Society</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/education">Education</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/art">Art</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/Facebook">Facebook / Media</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/News">News</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team2">Members</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Exolab">Exolab-Events</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Statistics">Statistics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/gallery">Photo Gallery</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Attributions">Our Partners</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/Coop">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/attributions"> Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/thanks">Thanks to</A></LI></DIV><DIV id="main"><DIV id="sidebar"><DIV id="sidebar_content"><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/1" class="active"> Abstract &amp; Intro </A></P><P class="second_order"><A href="#abstract"> Abstract </A></P><P class="second_order"><A href="#introduction"> Introduction </A></P><P class="second_order"><A href="#background"> CRISPR/Cas</A></P><P class="second_order"><A href="#outlook"> Outlook</A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/effector"> Effectors </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/induction"> Effector Control </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></P></DIV></DIV><DIV id="main_contant"><P id="h1">
Introduction
</P><DIV id="abstract"><P id="h2">
Abstract
</P></DIV><P> 
Our team developed a universal toolkit, termed uniCAS, that enables customizable gene regulation in mammalian cells. Therefore, we engineered the recently understood and highly promising CRISPR/Cas9 system. The regulation is based on the RNA-guided Cas9 protein, which allows specific targeting of DNA sequences. Our toolkit comprises not only a standardized Cas9 protein, but also different effector domains for efficient gene activation or repression. We further engineered a modular RNA plasmid for easy implementation of RNA guide sequences. As an additional feature, we established an innovative screening method for assessing the functionality of our uniCAS fusion proteins. Single genes and even whole genetic networks can be modified using our uniCAS toolkit. We think that our toolbox of standardized parts of the CRISPR/Cas9 system offers broad application in research fields such as tissue engineering, stem cell reprogramming and fundamental research.
</P><CENTER><A id="link" href="https://2013.igem.org/Team:Freiburg/description"> Click here for a multilingual project description </A></CENTER><DIV id="introduction"><P id="h2">
Introduction
</P></DIV><DIV id="project"><P id="h3">
Introduction to our project
</P><P>The enormous amount of gene regulation, signal transduction and metabolic pathways gives us a slight idea about what complexity of life really means. This astonishing complexity is one of the most crucial issues to understand, if we want to understand life itself.</P><P>Many approaches have been investigated and given us insights into these biological pathways, resulting in more and more elaborate tools. Solutions for the treatment of systemic diseases like cancer or the fine-tuning of cellular processes remain challenging. There is a need for a powerful tool, which enables powerful regulation of whole genetic networks and yet is simple to use. In the last years there have been several approaches which were able to alter gene expression, but they have the flaw that they are only targeting one gene at a time and do not allow flexible modification of entire genomes.</P><P>In the last 8 months, our team has been planning, developing and building up a universal toolkit, which enables the regulation of complex networks. A toolkit that allows the simple and flexible regulation of multiple genes by only one protein.</P><DIV style="margin-left:155px"><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 1: Our uniCAS toolkit </B>
The uniCAS toolkit provides 3 different <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/effector">effectors</A>, 2 methods &amp; 1 <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/induction">effector control</A>! By using our toolkit it is possible to efficiently activate or repress gene expression. We also provide devices for controlling our effectors by light. For multiple targeting different DNA sites can be addressed simultaneously by using our <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#rnaimer">RNAimer</A> plasmid.
Try working with uniCAS and receive custom-tailored <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/toolkit">instructions</A> for your desired gene regulation experiment by using our <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/toolkit">Manual Tool</A>.
</TD></TR></TBODY></TABLE></DIV><P>The great potential harbored by the <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/1#background"> CRISPR/Cas9 </A> system provided a great basis for us to modify it in a way that allows multiple, sequence specific gene regulation in mammalian cells.</P><P style=" border:thin solid  white; margin:15px; padding:15px;"><B>The CRISPR/Cas9 system in short</B>:
This CRISPR/Cas9 system is based on the interaction of three acting components: tracrRNA, crRNA and Cas9 protein.  The crRNA detects complementary invasive DNA, tracrRNA enables the interaction of crRNA and Cas9. The RNA-Cas9 complex binds respective DNA and Cas9 cleaves the DNA sequcence specifically.</P><P>We used a <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/attributions#Zhang">human codon optimized Cas9</A>, mutated the remaining nickase function and standardized the 160 kDa protein. The result: The DNA binding protein dCas9 which can be guided specifically to any DNA target site due to complexation with crRNA (CRISPR RNA) and tracrRNA (transactivating crRNA). By fusing different effector domains to dCas9 we built fusion proteins for gene regulation in mammalian cells.</P><DIV style="margin-left:160px"><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 2: Engineering of the CRISPR/Cas9 system </B>
The nuclease function of Cas9 has been mutated. The dCas9 is still able to bind the guiding RNAs and therewith can be directed towards every requested sequence. By fusing effectors to the dCas9, gene regulation of specitic loci can be engineered.
</TD></TR></TBODY></TABLE></DIV>
For gene regulation we used the trans-activator <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/effector#activation">VP16</A>, the repressor-domain <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/effector#repression">KRAB</A> and the histone methyltransferase <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/effector#epigenetics">G9a</A>. These effectors were able to efficiently change the expression rate of a reporter protein SEAP (secreted alcaline phosphatase) and the endogenous signal protein VEGF (vascular endothelial growth factor).<P>In order to allow for control of gene regulation in a time- and space-dependent manner, we designed various control elements. We engineered and tested <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/induction#light">light induction</A> systems and also tried to activate the expression upon <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/induction#hormon">chemical stimuli</A>.</P><P>As dCas9 is a relatively large protein consisting of 1367 amino acids with a molecular weight of 160 kDa, it influences expression rates and toxicity. We designed several <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/method#truncation">truncated versions</A> of dCas9 with the aim to minimize its size and increase the expression rate while retaining the RNA/DNA binding ability.</P><P>Last but not least we established a novel ELISA-based method to assess the binding efficiency of the dCas9 proteins: The uniCAS Binding Assay â<A id="link" href="https://2013.igem.org/Team:Freiburg/Project/method#elisa">uniBAss</A>â. uniBAss was shown to be an innovative tool for the characterization of the binding capacity of our dCas9 fusion proteins and the truncated dCas9 versions. The method offers the possibility for high throughput implementation and technological expansion to DNA-binding proteins other than dCas9 such as TAL effectors (TALEs) and zinc fingers (ZFNs). </P></DIV><DIV id="background"><P id="h3">
Background: The CRISPR/Cas9 system
</P></DIV><P>Hidden as an uncharacterized <I>E. coli</I> locus for more than 15 years <SPAN id="refer"><A href="#(1)"> [1]</A></SPAN>, Barrangou et al. identified the CRISPR (<B>C</B>lusterd <B>R</B>egularly <B>I</B>nterspaced <B>S</B>hort <B>P</B>alindromic <B>R</B>epeats) array as a previously unknown adaptive prokaryotic immune system <SPAN id="refer"><A href="#(2)"> [2]</A></SPAN>. Almost half of all prokaryotes make use of this defense mechanism against unselective uptake through natural transformation, phage DNA transduction or horizontal gene transfer by conjugation. Invasive DNA or even RNA  can be specifically recognized and efficiently cleaved <SPAN id="refer"><A href="#(3)"> [3]</A></SPAN>. This unique feature results from the interaction of non-coding RNAs and CRISPR associated (Cas) proteins. <SPAN id="refer"><A href="#(4)">[4]</A></SPAN>, <SPAN id="refer"><A href="#(5)">[5]</A></SPAN>. From a wide range of known CRISPR subtypes we used CRISPR type II b of <I>S. pyogenes</I>.  
</P><P>The recognition and degradation of invasive DNA by CRISPR/Cas type II occurs in three steps (see Fig. 2):
	<OL><LI><B>Acquisition</B>: Invasive DNA is recognized via a protospacer adjacent motif (PAM) â the sequence NGG. A short sequence downstream of the PAM sequence is then integrated into the host CRISPR array and is termed spacer. Spacer sequences transcribe for CRISPR RNAs(crRNAs) which help to cleave sequence-specific invasive DNA. These sequences are located between short palindromic repeats, which are neccessary for the functionality of the crRNAs.</LI><LI><B>Expression/Transcription</B>: The Cas9 endonuclease is expressed. CRISPR array is then transcribed and processed  by RNAse III into crRNAs. These contain the complementary spacer sequence and the direct repeat sequence. The crRNA guides the Cas9 protein specifically to invasive DNA sequences. Furthermore trans-activating crRNAs (tracrRNA) are transcribed and bind to the direct repeat part of the crRNA <SPAN id="refer"><A href="#(6)"> [6]</A></SPAN>. The tracrRNA is necessary for the formation of a Cas9-RNA complex. </LI><LI><B>Interference</B>: Repeatedly invading DNA, which has been integrated into the CRISPR locus, is detected by the RNA-protein complex and cleaved by Cas9.</LI></OL></P><DIV style="margin-left:160px"><TABLE class="imgtxt" width="500px"><TBODY><TR><TD><B>Figure 3: The CRISPR/Cas9 system</B>
First step is the aquisition of invasive DNA sequences into the CRISPR array. Next step is the transcription of crRNA and tracrRNA. Together they form an RNA hybrid that interacts with Cas9 and enables the protein to bind and cleave invasive DNA in the interference step.  
</TD></TR></TBODY></TABLE></DIV><DIV id="outlook"><P id="h2">
Outlook
</P></DIV><P> Our toolkit opens up the possibility to deal with a wide range of yet unadressed scientific questions in the near future, including the field of systems biology and complex metabolic engineering approaches. Other advantages cover monetary and logistic aspects, as the only component for modification lies indeed within the crRNA itself. In turn, this can be ordered as two corresponding forward and reverse primers - see Team Freiburg's easy <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#design_tool">crRNA Design Tool</A>. Financial, logistic and human ressources stay at a minimum using the uniCas toolkit for gene regulation. Accordingly, CRISPR/Cas9 systems have already been established for many model organisms, including <I>Saccharomyces cerevisiae</I>, <I>Caenorhabditis elegans</I>, <I>Arabidopsis thaliana</I>, <I>Drosophila melanogaster</I>, <I>Danio rerio</I> and <I>Mus musculus</I>. 
</P><P> The system could possibly be used for tissue engineering and stem cell reprogramming. Simultaneous regulation of the expression of different pathway enzymes may enhance industrial purposes - e.g. for the production of certain chemical compounds such as amino acids, biofuels or therapeutic drugs in eukaryotic cells. Therefore, stably transfected cells would be of interest.</P><P> Moreover, not only metabolic pathways could be targeted. Developmental Biology studies the embryonic patterning and organ formation. By transient dCas9-G9a interaction with different stem cell factor expressions, possibilities for better understanding and modeling of organism development arise. Induced pluripotent stem cells, for example, are yet made responsible for causing mutagenesis or immunogenicity <SPAN id="refer"><A href="#(9)"> [9]</A></SPAN>. Regenerative medicine still deals with basic understanding of reprogramming most cell types and may therefore gain benefits from efficient simultaneously acting dCas9-regulators.</P><P> Complex networks, as for instance can be found in cancer cell differentiation, migration and metabolism may be targeted. Frequently, transcriptional master regulators of house-keeping functions and signaling pathways, such as p53, show down- or upregulation in certain tumors <SPAN id="refer"><A href="#(10)">[10]</A></SPAN>. From a speculative point of view, one could therefore suggest to perform sophisticated dCas9-fusion protein high-throughput transcriptome analysis of cancer-related promoters. Targeting should then first be obtained in cancer cell lines with a multiple crRNA plasmid library. Consequently, promising combinations of crRNAs might then be investigated more precisely and maybe, on a long-term perspective, lead to future therapeutic relevance.</P><P id="h4"> References </P><SMALL><DIV id="(1)">(1) Ishino, Y., <I>et al.</I> (1987). Nucleotide Sequence of the iap Gene in Escherichia coli. Journal of Bacteriology 169, <I>5429-5433</I>. </DIV><DIV id="(2)">(2) Barrangou, R., <I>et al.</I> (2007). CRISPR provides acquired resistance against viruses in prokaryotes. Science 315, <I>1709-1712</I>. </DIV><DIV id="(3)">(3) Marraffini, L., and Sontheimer, E. (2008). CRISPR interference limits horizontal gene transfer in staphylococci by targeting DNA. Science 322, <I>1843-1845</I>.</DIV><DIV id="(4)">(4) Jansen, R., <I>et al.</I> (2002). Identification of genes that are associated with DNA repeats in prokaryotes. Molecular Microbiology 43, <I>1565-1575</I>.</DIV><DIV id="(5)">(5) Makarova, K., <I>et al.</I> (2011). Evolution and classification of the CRISPR-Cas systems. Nat Rev Microbiol 9, <I>467-477</I>.</DIV><DIV id="(6)">(6) Jinek, M., <I>et al.</I> (2012). A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity. Science 337, <I>816-821</I>.</DIV><DIV id="(7)">(7) Cong, L., <I>et al.</I> (2013). Multiplex genome engineering using CRISPR/Cas systems. Science 339, <I>819-823</I>.</DIV><DIV id="(8)">(8) Mali, P., <I>et al.</I> (2013). RNA-guided human genome engineering via Cas9. Science 339, <I>823-826</I>.</DIV><DIV id="(9)">(9) Yamanaka, S., <I>et al.</I>  (2012). Induced Pluripotent Stem Cells: Past, Present and Future. Cell Stem Cell 10,
<I>678-684</I>.</DIV><DIV id="(10)">(10) Kaneshiro, K., <I>et al.</I> (2007). An integrated map of p53-binding sites and histone modification in the human ENCONDE regions. Genomics,
<I>177-188</I>.</DIV></SMALL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Freiburg/Project/1">http://2013.igem.org/Team:Freiburg/Project/1</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Project/1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Project/1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Project/1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Project/1&amp;oldid=360394" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>