<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Notebook_lab_hormon"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Notebook/lab_hormon">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Notebook/lab_hormon&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_hormon&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_hormon&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Notebook/lab_hormon" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Notebook/lab hormon</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3></DIV><DIV id="menu"><UL id="navbar"><LI style="padding-left:15px"><A href="https://2013.igem.org/Team:Freiburg">Home</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Highlights">Highlights</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Introduction </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/induction">Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></LI></UL><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna">Targeting</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/method">uniBAss</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation">Standardization</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/protocols">Material and Methods</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/BioBricks">Our BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/favorite_parts">Favorite BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/improvement">Improvement of a Registry part</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/sharing">Sharing of our BioBricks </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/experts">Expert opinions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety/safety">Safety &amp; Security</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/politics">Politics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/society">Society</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/education">Education</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/art">Art</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/Facebook">Facebook / Media</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/News">News</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team2">Members</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Exolab">Exolab-Events</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Statistics">Statistics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/gallery">Photo Gallery</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Attributions">Our Partners</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/Coop">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/attributions"> Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/thanks">Thanks to</A></LI></DIV><DIV id="sidebar"><DIV id="sidebar_content"><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna"> Targeting </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector"> Effector </A></P><P class="first_order"><A class="active" href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control </A></P><P class="second_order_note"><A id="link" href="https://2013.igem.org/Team:Freiburg/Notebook/lab_light"> Light </A></P><P class="second_order_note"><A class="active" id="link" href="https://2013.igem.org/Team:Freiburg/Notebook/lab_hormon"> Hormone </A></P><P class="third_order"><A href="#april"> April </A></P><P class="third_order"><A href="#may"> May </A></P><P class="third_order"><A href="#june"> June </A></P><P class="third_order"><A href="#july"> July </A></P><P class="third_order"><A href="#august"> August </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling"> Modeling </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/method"> uniBAss </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation"> Standardization </A></P><P class="first_order"><A id="link" href="https://2013.igem.org/Team:Freiburg/protocols"> Material and Methods </A></P></DIV></DIV><DIV id="main_contant_note"><DIV id="h1">
Effector Control - Hormone
</DIV><H2>Planned constructs</H2><H2>Practical work: Localization</H2><DIV id="april"><P id="h2">
April
</P></DIV><H2>17.04.2013</H2><H3>Trafo of pMZ333</H3>
pMZ333 containing mCherry and the backbone (Bb)

<H2>18.04.2013</H2><H3>Picking of clones</H3>
3 colonies of pMItom (containing ERT2) and 1 of pMZ333.

<H2>19.04.2013</H2><H3>Minipreps of pMZ333 and pMItom </H3>
Yield: 100 - 300 ng/µl

<H2>23.04.2013</H2><H3>Digestion of pMZ333</H3>
Bb was cut out of pMZ333 by XbaI and EcoRI.
  <H3>Gel run</H3>
0,7 % agarose, marker: <I>GeneRuler 1 kb</I><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker - Test digestion of pMItom - xx - Digestion of pMZ333 (3x) </TD></TR></TBODY></TABLE>
The bands of the test digestion were as exspected.
pMZ333 was cut in two fragments, where the shorter one is the Bb (lower band).
</DIV><H3>Gel extraction</H3>
Bb was purified.
Yield: 20 - 30 ng/µl

<H2>25.04.2013</H2><H3>PCR of aIG3001 and aIG3006-8</H3><H3>Gel run</H3>
0,9 % agarose, marker: <I>GeneRuler 1 kb</I><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker - xx - aIG3001 - aIG3006 - aIG3007 - aIG3008 </TD></TR></TBODY></TABLE><P>All PCR products (the bright bands) had the exspected length.</P></DIV><H3>Gel extraction</H3>
PCR products were purified.
Yield: 30 - 90 ng/µl

<DIV id="may"><P id="h2">
May
</P></DIV><H2>03.05.2013</H2><H3>PCR of aIG3002-5</H3><DIV class="floatleft">
Mix:
<TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> Template </TD><TD align="right"> 1 µl</TD></TR><TR><TD> Primers (10 µM) </TD><TD align="right"> each 2.5 µl</TD></TR><TR><TD> Q5-HF MasterMix (NEB) 2x </TD><TD align="right"> 25 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 50 µl </TD></TR></TBODY></TABLE></DIV><H3>Gel run</H3>
0,9 % agarose, marker: <I>GeneRuler 1 kb</I><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: aIG3002 - aIG3004 - aIG3003 - aIG3005 - xx - Marker </TD></TR></TBODY></TABLE><P>All PCR products (the bright bands) had the exspected length.</P></DIV><H3>Gel extraction</H3>
PCR products were purified with the following changes to the standard protocol:
<UL><LI>After the first incubation, the samples were vortexed followed by another incubation for 2 min at 50°C.</LI><LI>Between the centrifugations after PE buffer incubation the epis were turned around for 180°.</LI><LI>After 13 min incubation with water, samples were incubated for 4 min at 55°C.</LI></UL>
Yield: 100 - 140 ng/µl

<H2>06.05.2013</H2><H3>Gibson assembly</H3>
Pipetting scheme:
<DIV id="floathleft"><P>to each symple 1 µl water was added</P></DIV><P>For pIG3005 and pIG3007 I used only the half amount of DNA as otherwise the total volume would have been higher than 5 µl.</P>
Gibson assembly was performed by the following protocol:
<OL><LI>5 µl of DNA mixes were added to 15 µl of Gibson master mix, each.</LI><LI>Incubation for 1 h at 50°C</LI><LI>Incubation for 3 min at RT</LI><LI>Incubation for 10 min on ice</LI></OL><H3>Trafo</H3><OL><LI>4 µl of each assemblies were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 13 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 500 µl LB medium</LI><LI>incubation for 15  min at 37 °C (shaking)</LI><LI>20 µl, 200 µl and 300 µl (concentrated) were distributed on agar plates with ampicilline</LI><LI>incubation over night at 37 °C</LI></OL><H2>07.05.2013</H2><H3>Trafo repeat</H3>
As there were very few colonies only on the 300 µl plates which were surrounded by &quot;satellite colonies&quot;, transformation was repeated by using the same protocol exept of:
<UL><LI>The first incubation at 37°C was 1:15 h.</LI><LI>Only 300 µl (not concentrated) were distributed.</LI><LI>Over night incubation lasted only 16 h.</LI></UL><H2>08.05.2013</H2><H3>Picking of clones</H3>
Colonies from plates pIG3001-8 (exept of pIG3006, because there were no colonies) of trafo repeat were spread on new agar plates.
Colonies of the first gibson trafo hopefully containing pIG3003,5,6 were spread on new plates to get single colonies.

<H2>09.05.2013</H2><H3>Picking of clones</H3>
Single colonies of the first gibson trafo were spread on new agar plates.
  <H3>Minipreps of the second gibson trafo</H3>
Yield: 40 - 350 µg/ml
  <H3>Test digest of gibson constructs</H3><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> approx. 1 µg </TD></TR><TR><TD> NcoI HF</TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NEB buffer 4 </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 2.5 h.
</DIV><H2>10.05.2013</H2><H3>Gel run of test digest</H3>
1 % agarose, marker: <I>GeneRuler 1 kb</I><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: <I>pIG300..,Colony:</I>
1,1 - 1,2 - 1,3 - 1,4 - 2,1 - 2,2 - Marker - 2,3 - 3,1 - 4,1 - 4,2 - 4,3 - 4,4 </TD></TR></TBODY></TABLE></DIV><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: <I>pIG300..,Colony:</I>
xx - 5,1 - 7,1 - 7,2 - 7,3 - 7,4 - Marker - 8,1 - 8,2 - 8,3 - 8,4 </TD></TR></TBODY></TABLE></DIV><P>The banding pattern of pIG3001 (all colonies); 2,1; 2,2; 4,1; 4,2; 4,4 and 8 (all colonies) has been proven to be the expected one by sequencing the linkers. Thus, the marker run has to be wrong.</P><H3>Minipreps of the first gibson trafo</H3>
Changes:
<UL><LI>Between the second washing step (centrifugation for 30 s) and the drying step (centrifugation for 1 min) the collection tube was changed.</LI><LI>The incubation with water took only 3 min.</LI></UL>
Yield: 40 - 180 ng/µl
  <H3>Test digest of gibson constructs</H3><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> plasmid </TD><TD align="right"> approx. 1 µg </TD></TR><TR><TD> NcoI HF</TD><TD align="right"> 0.3 µl </TD></TR><TR><TD> NEB buffer 4 </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 1:45 h.
</DIV><H3>Gel run of second test digest</H3>
1 % agarose, marker: <I>GeneRuler 1 kb</I><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: <I>pIG300..,Colony:</I> xx - 3,1 - 3,2 - 5,1 - Marker - 5,2 - 6,1 - 6,2 - xx - xx </TD></TR></TBODY></TABLE><P>The banding pattern of pIG3003 has been proven to be the expected one by sequencing the linkers. Like above the marker ran wrong.</P></DIV><H2>14.05.2013</H2><H3>Fusion PCR of aIG3009 &amp; 10</H3><P>aIG3005 &amp; 6 were fused to aIG3009, aIG3007 &amp; 8 to aIG3010 in order to reduce the fragments for Gibson assembly.</P><DIV class="floatleft">
Mix:
<TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> Templates </TD><TD align="right"> each 1 µl</TD></TR><TR><TD> Q5 polymerase </TD><TD align="right"> 0.3 µl </TD></TR><TR><TD> Q5 buffer (5x) </TD><TD align="right"> 10 µl </TD></TR><TR><TD> dNTPs (2.5 mM) </TD><TD align="right"> 4 µl </TD></TR><TR><TD> DMSO </TD><TD align="right"> 1 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 45 µl </TD></TR></TBODY></TABLE></DIV><DIV class="floatleft">
Program A:
<TABLE class="tabelle"><TBODY><TR><TH> temperature </TH><TH align="right"> time </TH></TR><TR><TD> 98 °C </TD><TD align="right"> 5 min </TD></TR><TR><TD> 98 °C </TD><TD align="right"> 30 s </TD></TR><TR><TD> ** </TD><TD align="right"> 30 s </TD></TR><TR><TD> 72 °C </TD><TD align="right"> * </TD></TR></TBODY></TABLE>
Step 2 - 4 were repeated for 5 times.
*  time calculation: size template [kb] x 30 s
** 72 °C - 0,5 per cycle for aIG3009; 69 °C - 0,5 per cycle for aIG3010
</DIV><DIV class="floatright">
Program B (after addition of 2,5 µl Primers, each):
<TABLE class="tabelle"><TBODY><TR><TH> temperature </TH><TH align="right"> time </TH></TR><TR><TD> 98 °C </TD><TD align="right"> 5 min </TD></TR><TR><TD> 98 °C </TD><TD align="right"> 30 s </TD></TR><TR><TD> 63 °C </TD><TD align="right"> 30 s </TD></TR><TR><TD> 72 °C </TD><TD align="right"> * </TD></TR><TR><TD> 72 °C </TD><TD align="right"> 10 min </TD></TR></TBODY></TABLE>
Step 2 - 4 were repeated for 5 times.
*  time calculation: size template [kb] x 30 s
</DIV><H3>Gel run of fusion PCR</H3>
Fragment´s lengths were as expected, though the band of aIG3009 was very slight.

<H2>15.05.2013</H2><H3>Gel extraction of fusion PCR</H3>
PCR products were purified as described on 03.05.2013, but DNA solution was loaded on the column again.
Yield: 6 / 49 ng/µl

<H2>16.05.2013</H2><H3>PCR of aIG3006 (repeat)</H3>
This PCR was repeated as the concentration yielded from the first attempt is too low for Gibson assembly.

<H2>21.05.2013</H2><H3>Repeat of fusion PCR of aIG3009</H3><P>Fusion PCR was repeated, as the concentration of the first attempt is too low for Gibson assembly.</P><H3>Gel run of PCR of aIG3006 and fusion PCR of aIG3009</H3><P>Gel did not show the desired bands.</P><H3>Gibson assembly 2</H3>
		Pipetting scheme:
			<DIV id="floathleft"><P>For pIG3005 and pIG3007 only the half amount of each PCR product was used as the concentration of aIG3006 was very 

low.</P><P>The volumes were calculated that 5 µl contain the required DNA amount.</P></DIV>
		Gibson assembly was performed by the following protocol:
			<OL><LI>5 µl of DNA mixes were added to 15 µl of Gibson master mix, each.</LI><LI>Incubation for 1 h at 50°C</LI><LI>Incubation for 3 min at RT</LI><LI>Incubation for 10 min on ice</LI></OL><H3>Trafo</H3><OL><LI>4 µl of each assemblies were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 13 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 500 µl LB medium</LI><LI>incubation for 15  min at 37 °C (shaking)</LI><LI>20 µl, 200 µl and 300 µl (concentrated) were distributed on agar plates with ampicilline</LI><LI>incubation over night at 37 °C</LI></OL><H2>22.05.2013</H2><H3>Midiprep of pIG3003, 4 &amp; 8</H3><P> As the sequencing results were positive, plasmids were amplified and midiprepped according to the manual of <I>Jetstar Plasmid 

Purification MIDI Kit</I></P><P>Yield: ~ 1 µg/µl for pIG3004 &amp; 8; no DNA was isolated for pIG3003.</P><H2>23.05.2013</H2><H3>Miniprep of Gibson assembly 2</H3><P>DNA was purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Yield: 30 - 90 ng/µl</P><H3>Test digest of Gibson assembly 2</H3><P>Mix 1 was used for DNA concentrations beneath 60 ng/µl, Mix 2 for concentrations above 60 ng/µl.</P><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> mix 1</TH><TH align="right"> mix 2</TH></TR><TR><TD> plasmid </TD><TD align="right"> 17.5 µl </TD><TD align="right"> 10 µl </TD></TR><TR><TD> NcoI HF</TD><TD align="right"> 0.2 µl </TD><TD align="right"> 0.2  µl</TD></TR><TR><TD> NEB buffer 4 </TD><TD align="right"> 2 µl </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 1.5 h.
</DIV><H2>24.05.2013</H2><H3>Gel run</H3><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (GeneRuler 1 kb); pIG3001 (4x); pIG3006 (4x); pIG3007 (4x). </TD></TR></TBODY></TABLE><P>The bands of pIG3001 &amp; 6 could be at the right size, so they were send in for sequencing.</P></DIV><H2>27.05.2013</H2><H3>Amplification of aIG3006 &amp; 9 by PCR</H3><DIV class="floatleft">
Mix:
<TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> aIG3006 / 9</TD><TD align="right"> 5 µl</TD></TR><TR><TD> Primers (10 µM) </TD><TD align="right"> each 1 µl</TD></TR><TR><TD> Q5 DNA polymerase </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> Q5 buffer </TD><TD align="right"> 10 µl </TD></TR><TR><TD> DMSO </TD><TD align="right"> 1 µl </TD></TR><TR><TD> dNTPs (2.5 mM) </TD><TD align="right"> 4 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 50 µl </TD></TR></TBODY></TABLE></DIV><DIV class="floatright">
Program:
<TABLE class="tabelle"><TBODY><TR><TH> temperature </TH><TH align="right"> time </TH></TR><TR><TD> 98 °C </TD><TD align="right"> 5 min </TD></TR><TR><TD> 98 °C </TD><TD align="right"> 30 s </TD></TR><TR><TD> 63 °C </TD><TD align="right"> 30 s </TD></TR><TR><TD> 72 °C </TD><TD align="right"> 160 s </TD></TR><TR><TD> 72 °C </TD><TD align="right"> 10 min </TD></TR></TBODY></TABLE>
Step 2 - 4 were repeated for 18 times.
</DIV><P>Yield after Gel extraction with <I>High Pure Plasmid Isolation Kit</I> of Roche: 40 ng/µl for aIG3006 and 82 ng/µl for aIG3009.</P><H2>28.05.2013</H2><H3>Gibson assembly 3</H3>
		Pipetting scheme:
			<DIV id="floathleft"><P>For pIG3005 and pIG3007 only the half amount of each PCR product was used as the concentration of aIG3006 was very 

low.</P><P>5 µl of each DNA mix was added to 15 µl Gibson master mix.</P></DIV>
		Gibson assembly was performed by the following protocol:
			<OL><LI>5 µl of DNA mixes were added to 15 µl of Gibson master mix, each.</LI><LI>Incubation for 1 h at 50°C</LI><LI>Incubation for 3 min at RT</LI><LI>Incubation for 10 min on ice</LI></OL><H3>Trafo</H3><OL><LI>4 µl of each assemblies were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 13 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 500 µl LB medium</LI><LI>incubation for 15  min at 37 °C (shaking)</LI><LI>20 µl, 200 µl and 300 µl (concentrated) were distributed on agar plates with ampicilline</LI><LI>incubation over night at 37 °C</LI></OL><H3>Transfection of pIG3004 and pIG3008</H3><UL><LI>0.75 ng of DNA per well were filled up to 50 µl with OptiMEM</LI><LI>2.5 µl of PEI per well were filled up to 50 µl with OptiMEM</LI></UL><P>These two solutions were mixed by immediately 2x vortexing for ~ 5 s. After 20 min incubation 100 µl were added dropwise to 

each well.</P><P>Medium change was performed after 2 h.</P><H2>28.05.2013</H2><H3>Cell fixation</H3><P>Cells on each well were washed with 500 µl PBS and fixed with 200 µl PFA (4 %) solution by incubating for 10 min on ice and at 

least 		20 min at RT.
		Cells attached on cover slides were stained with DAPI and DRAQ5 (incubation in DAPI or DRAQ5 solution consisting of 1 µl DAPI or 		

DRAQ5 in 5 ml dH2O for 5 s, respectively) and rinsed in dH2O. Cover slides were dried and mounted on a drop of Mowiol solution 		(with 

DABCO) with cells down. After a drying step at 37 °C for 15 min samples were sealed with nail polish and stored at 4 °C.</P><H3>Flourescence microscopy</H3><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> pIG3004 </TD></TR></TBODY></TABLE></DIV><DIV id="floathright"><TABLE class="gelpic"><TBODY><TR><TD> pIG3008  </TD></TR></TBODY></TABLE></DIV><P>These pictures show that pIG3004 is clearly localized in the nucleus whereas pIG3008 is localized in the cytoplasm. Nevertheless more cells have to be analyzed.</P><H2>30.05.2013</H2><H3>Midiprep of pIG3001, 3 &amp; 6</H3><P> As the sequencing results were positive, plasmids were amplified and midiprepped according to the manual of <I>Jetstar Plasmid Purification MIDI 

Kit</I></P><P>Yield: 1.4 µg/µl for pIG3006; 470 ng/µl for pIG3001; 71 ng/µl for pIG3003.</P><H3>Test digest of Gibson assembly 3</H3><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volumes</TH></TR><TR><TD> plasmid (~ 100 ng/µl) </TD><TD align="right"> 5 µl </TD></TR><TR><TD> NcoI HF</TD><TD align="right"> 0.2 µl </TD></TR><TR><TD> NEB buffer 4 </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 2.5 h.
</DIV><H2>31.05.2013</H2><H3>Gel run</H3><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (GeneRuler 1 kb); pIG3002 (3x); pIG3005<SUB>2</SUB> (3x); pIG3007 (3x). </TD></TR></TBODY></TABLE><P>The bands of pIG3002, 5 &amp; 7-a were at the right size, so they were send in for sequencing.</P></DIV><DIV id="june"><P id="h2">
June
</P></DIV><H2>04.06.2013</H2><H3>Transfection of pIG3001, 3, 4, 6 &amp; 8</H3><P>HeLa cells seeded onto cover slides in a 24 well plate were transfected by the following protocol:</P><UL><LI>0.75 ng of DNA per well were filled up to 50 µl with OptiMEM</LI><LI>2.5 µl of PEI per well were filled up to 50 µl with OptiMEM</LI></UL><P>These two solutions were mixed by immediately 2x vortexing for ~ 5 s. After 20 min incubation 100 µl were added dropwise to 

each well.</P>
Transfection scheme:
			<H2>05.06.2013</H2><H3>Second Miniprep of Gibson assembly 3</H3><P>As the sequencing of the only positive colony of pIG3007 pointed out a mutation, more colonies were prepped using <I>High Pure Plasmid Isolation 

Kit</I> of Roche.</P><P>Yield: ~ 60 ng/µl</P><H3>Second test digest of Gibson assembly 3</H3><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volumes</TH></TR><TR><TD> plasmid (~ 60 ng/µl) </TD><TD align="right"> 10 µl </TD></TR><TR><TD> NcoI HF</TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NEB buffer 4 </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 2.5 h.
</DIV><H3>Gel run</H3><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (GeneRuler 1 kb); pIG3007 (3x). </TD></TR></TBODY></TABLE><P>The bands were at the right size, so pIG3007 was send in for sequencing.</P></DIV><H2>06.06.2013</H2><H3>4-OHT treatment</H3><P>4-OHT was diluted in DMEM complete to a final concentration of 0.1, 1 and 10 µM; EtOH was diluted in DMEM complete to a final concentration of 0.1 

and 1 % (which is the same amount as in the medium with 4-OHT, as 4-OHT is dissolved in EtOH).</P><P>The old medium was removed from the cells and the new mixed media were added according to the transfection scheme.</P><H3>Cell fixation</H3><P>After 0.5, 1 and 3 h (according to the transfection scheme) cells on each well were washed with 500 µl PBS and fixed with 200 µl PFA (4 %) solution by 

incubating for 10 min on ice and at least 20 min at RT.
Cells attached on cover slides were stained with DAPI and DRAQ5 (incubation in DAPI or DRAQ5 solution consisting of 1 µl DAPI or 	DRAQ5 in 5 ml dH2O 

for 5 s, respectively) and rinsed in dH2O. Cover slides were dried and mounted on a drop of Mowiol solution (with DABCO) with cells down. After a drying 

step at 37 °C for 15 min samples were sealed with nail polish and stored at 4 °C.</P><H3>Midiprep of pIG3002, 3, 5 &amp; 7</H3><P> As the sequencing results were positive, plasmids were amplified and midiprepped according to the manual of <I>Jetstar Plasmid Purification MIDI 

Kit</I></P><P>Yield: 1.7 µg/µl for pIG3005; 670 ng/µl for pIG3007; 540 ng/µl for pIG3002; 335 ng/µl for pIG3003.</P><H2>07.06.2013</H2><H3>Microscopy</H3><UL><LI>All samples showed red flourescence (except of pIG3006 &amp; 8 treated with 10 µM 4-OHT, as there were almost no cells).</LI><LI>Treatment with 0.1 µM 4-OHT did not cause a difference to treatment with EtOH.</LI><LI>Treatment with 1 µM 4-OHT seemed to increase nuclear import for pIG3006 &amp; 8.</LI><LI>pIG3006 is always localized in the nucleus.</LI><LI>By cell shape concentrations of 4-OHT lower than 10 µM does not seem to harm HeLa cells.</LI></UL><H2>10.06.2013</H2><H3>Midiprep of pIG3009</H3><P> As experimental results indicate, that the 5' NLS has a maybe stronger influence on nuclear import than the 3' NLS, the wrong assembled plasmid with 

the insert <B>HA-ERT2-NLS-Cas9-mCherry (:= pIG3009)</B> was amplified and midiprepped according to the manual of <I>Jetstar Plasmid Purification MIDI 

Kit</I></P><P>Yield: 300 ng/µl.</P><H2>11.06.2013</H2><H3>Transfection of all constructs</H3><P>HeLa cells seeded onto cover slides in a 24 well plate were transfected by the following protocol:</P><UL><LI>0.75 ng of DNA per well were filled up to 50 µl with OptiMEM</LI><LI>2.5 µl of PEI per well were filled up to 50 µl with OptiMEM</LI></UL><P>These two solutions were mixed by immediately 2x vortexing for ~ 5 s. After 20 min incubation 100 µl were added dropwise to 

each well.</P>
Transfection scheme:
			<H2>12.06.2013</H2><H3>4-OHT treatment</H3><P>4-OHT was diluted in DMEM complete to a final concentration of 0.1 and 1 µM; EtOH was diluted in DMEM complete to a final concentration of 0.1 % 

(which is the same amount as in the medium with 4-OHT, as 4-OHT is dissolved in EtOH).</P><P>31 h after transfection the old medium was removed from the cells and the new mixed media were added according to the transfection scheme.</P><H3>Cell fixation</H3><P>After 3 and 16 h (according to the transfection scheme) cells on each well were washed with 500 µl PBS and fixed with 200 µl PFA (4 %) solution by 

incubating for 10 min on ice and at least 20 min at RT.
Cells attached on cover slides were stained with DAPI and DRAQ5 (incubation in DAPI or DRAQ5 solution consisting of 1 µl DAPI or 	DRAQ5 in 5 ml dH2O 

for 5 s, respectively) and rinsed in dH2O. Cover slides were dried and mounted on a drop of Mowiol solution (with DABCO) with cells down. After a drying 

step at 37 °C for 15 min samples were sealed with nail polish and stored at 4 °C.</P><H2>13.06.2013</H2><H3>Toxity assay for 4-OHT</H3><P>Untransfected HeLa cells were treated for one day with 4-OHT, EtOH (different concentrations, each) or no addition. Supernatant was removed and a 

washing step with 0.5 ml PBS was performed. The cells in the supernatant and PBS were counted with CASY counter. The number of dead cells was 

calculated by multiplying the number of living cell with (100 - percentage of living cells) / 100.</P><H3>Confocal fluorescence microscopy</H3><P>Pictures were taken of all different constructs by a confocal fluorescence microscope. The ration of the mean nuclear to cytoplasmic red flourescence 

was determined by ImageJ.</P><P>No significant difference in subcellular localisation upon 4-OHT stimulus was detectable this way.</P><H2>19.06.2013</H2><H3>Second transfection of all constructs</H3><P>HeLa cells seeded onto cover slides in a 24 well plate were transfected by the following protocol:</P><UL><LI>0.75 ng of DNA per well were filled up to 50 µl with OptiMEM</LI><LI>2.5 µl of PEI per well were filled up to 50 µl with OptiMEM</LI></UL><P>These two solutions were mixed by immediately 2x vortexing for ~ 5 s. After 20 min incubation 100 µl were added dropwise to 

each well.</P>
Transfection scheme:
			<H2>20.06.2013</H2><H3>4-OHT treatment</H3><P>4-OHT was diluted in DMEM complete to a final concentration of 1 and 5 µM; EtOH was diluted in DMEM complete to a final concentration of 0.1 % 

(which is the same amount as in the medium with 4-OHT, as 4-OHT is dissolved in EtOH).</P><P>31 h after transfection the old medium was removed from the cells and the new mixed media were added according to the transfection scheme.</P><H2>21.06.2013</H2><H3>Cell fixation</H3><P>After an over night incubation cells on each well were washed with 500 µl PBS and fixed with 200 µl PFA (4 %) solution by incubating for 10 min on ice 

and at least 20 min at RT.
Cells attached on cover slides were stained with DAPI and DRAQ5 (incubation in DAPI or DRAQ5 solution consisting of 1 µl DAPI or 	DRAQ5 in 5 ml dH2O 

for 5 s, respectively) and rinsed in dH2O. Cover slides were dried and mounted on a drop of Mowiol solution (with DABCO) with cells down. After a drying 

step at 37 °C for 15 min samples were sealed with nail polish and stored at 4 °C.</P><H3>Fluorescence microscopy</H3><P>No big difference between 4-OHT and EtOH treatment was detectable.</P><H2>25.06.2013</H2><H3>Transfection for western blotting</H3><P>HEK cells seeded in three 6 well plates were transfected with pIG3001, 2, 5, 7 &amp; 9 by the following protocol:</P><UL><LI>3.75 ng of DNA per well were filled up to 250 µl with OptiMEM</LI><LI>12.5 µl of PEI per well were filled up to 250 µl with OptiMEM</LI></UL><P>These two solutions were mixed by immediately 2x vortexing for ~ 5 s. After 20 min incubation 500 µl were added dropwise to 

each well.</P><H2>26.06.2013</H2><H3>4-OHT treatment</H3><P>4-OHT was diluted in DMEM complete to a final concentration of 1 µM; EtOH was diluted in DMEM complete to a final concentration of 0.1 % (which is 

the same amount as in the medium with 4-OHT, as 4-OHT is dissolved in EtOH).</P><P>30 h after transfection the old medium was removed from the cells and the new mixed media were added to the wells, both 4-OHT and EtOH for each 

construct.</P><H2>27.06.2013</H2><H3>Cell fractionation</H3><P>Cells were washed twice with 1 ml PBS, then incubated for 30 min on ice with 0.5 ml hypotonic lysis buffer (20 mM HEPES pH=7.4; 5 mM MgCl2; 1 mM 

EDTA; 1 Protease inhibitor cocktail tablet / 50 ml buffer). After mechanical detachment lysates were centrifuged for 5 min at 10,000 g. Supernatant and 

pellet were separated.</P><H3>Sample preparation for SDS-PAGE</H3><P>200 µl of supernatants were mixed with 50 μl of 5x SDS loading buffer (10 % SDS; 0.3125 M Tris-HCl pH=6.8; 50 % glycerol, 12.5 % 2-Mercaptoethanol; 

0.025 % Bromphenol blue) and heated for 5 min at 95 °C. The pellets resuspended in 15 μl 5x SDS loading buffer and heated for 5 min at 95 °C were diluted 

with 120 μl hypotonic lysis buffer and another 30 μl SDS loading buffer (5x) and sonified for 10 min at maximum.</P><H3>SDS-PAGE cast</H3><P>At first the separation gel was cast and covered with 200 μl isopropanol. After polymerisation the collection gel was cast on top.</P><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> separation gel (8 %) </TH><TH><CENTER> ingredient </CENTER></TH><TH align="right"> collection gel </TH></TR><TR><TD> 19.2 ml </TD><TD><CENTER> dH<SUB>2</SUB>O </CENTER></TD><TD align="right"> 7 ml </TD></TR><TR><TD> 10.4 ml </TD><TD><CENTER> 30 % acryl-bisacryl mix </CENTER></TD><TD align="right"> 2 ml </TD></TR><TR><TD> 10.4 ml </TD><TD><CENTER> 1.5 M Tris with 0.4 % SDS, pH=8.8 </CENTER></TD></TR><TR><TD><CENTER> 0.5 M Tris with 0.4 % SDS, pH=6.8 </CENTER></TD><TD align="right"> 3 ml </TD></TR><TR><TD> 400 µl </TD><TD><CENTER> APS (10%) </CENTER></TD><TD align="right"> 120 µl </TD></TR><TR><TD> 40 µl </TD><TD><CENTER> TEMED </CENTER></TD><TD align="right"> 12 µl </TD></TR><TR><TD> 40 ml </TD><TD><CENTER> sum </CENTER></TD><TD align="right"> 12 ml </TD></TR></TBODY></TABLE>
150 V were supplied for about 1.5 h.
</DIV><H3>Western blotting</H3><P>A PVDF-membrane was activated in methanol and washed in transfer buffer (192 mM glycin; 25 mM Tris; 10 % methanol). 3 sheets of whatman paper 

were incubated in transfer buffer, too. The blotting apparatus was loaded in the following order from bottom up: 2 sheets of whatman paper - PVDF-

membrane - gel - 1 sheet of whatman paper. An amperage of 0.35 A per membrane was supplied for 1.5 h.</P><H2>27.06.2013</H2><H3>Antibody treatment</H3><P>Membranes were incubated for 1 h in blocking buffer (PBS with 3 % milk powder), then for 2.5 h with anti HA mouse (1:1000 in blocking buffer). After 3 

times washing for 5 min with PBS-T (PBS with 0.05 % Tween 20) membranes were incubated for 1 h with anti mouse HRP (1:5000 in blocking buffer) and 

washed again 3 times for 5 min with PBS-T.
Imaging was performed with automatically determined exposure time after addition of ECL I (250 mM luminol; 90 mM coumaric acid; 1 M Tris pH=8.5) + ECL 

II (30 % hydrogen peroxide; 1 M Tris pH=8.5) solution (500 μl of each).</P><DIV id="floathleft"><P>Pictures of western blot of nuclear (A) and cytoplasmic (B) cell fraction.</P><P>Arrows indicate most probably the fusion proteins; other bands are unspecific AB-bindings or (N-terminal) fusion protein fragments.
Only the controls are detectable and act as expected.</P></DIV><B>These data suggest that altering the subcellular localization of a Cas9 fusion protein upon a 4-OHT stimulus is not possible. Because of this it ought to be tested, if the functionality of Cas9 can be controlled by hormone induction. Thus, in a new experiment a SEAP plasmid should be targeted.</B><H2>RNA Plasmid for SEAP Assay</H2>
pX334a digested with PstI and religated afterwards (&quot;pIG3010&quot;) was used as initial plasmid for the introduction of crRNAs (oligos designt for targeting two different SEAP loci).


<DIV id="july"><P id="h2">
July
</P></DIV><H2>14.07.2013</H2><H3>Digest of pX334a</H3><P>Cas9 was cut out of pX334a to get a plasmid containing only the tracrRNA and the crRNA.</P><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>ingredient</TH><TH align="right">sample 1</TH><TH align="right">sample 2</TH></TR><TR><TD>pX334a</TD><TD align="right">0.9 µg</TD><TD align="right">1.8 µg</TD></TR><TR><TD>PstI HF</TD><TD align="right">1 µl</TD><TD align="right">1 µl</TD></TR><TR><TD>NEB buffer 4</TD><TD align="right">5 µl</TD><TD align="right">5 µl</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD align="right">up to 50 µl</TD><TD align="right">up to 50 µl</TD></TR></TBODY></TABLE>
Incabation at 37 °C for 2 h.
</DIV><H3>Gel run</H3><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (2µl); digest 1 (55 µl); digest 1 (4 µl); digest 2 (55 µl); digest 2 (4 µl); Marker (0.5 µl) </TD></TR></TBODY></TABLE><P>Only the upper band of digest 1 was cut out, as the digest of sample 2 was not completely.</P></DIV><H2>15.07.2013</H2><H3>Gel extraction</H3><P>DNA was purified using <I>High Pure PCR Product Purification Kit</I> of Roche.</P><P>Changes to the protocol: 
<UL><LI>incubation at 56 °C for 10 min for gel dissolving</LI><LI>elution with dH<SUB>2</SUB>O (incubation at 50 °C for 4 min before centrifugation)</LI></UL></P><P>Yield: 30 ng/µl</P><H3>Religation of pX334a backbone</H3><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>ingredient</TH><TH align="right">amount</TH></TR><TR><TD>Backbone of pX334a</TD><TD align="right">150 ng</TD></TR><TR><TD>T4-Ligase</TD><TD align="right">1 µl</TD></TR><TR><TD>T4-Ligase buffer</TD><TD align="right">2 µl</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD align="right">up to 20 µl</TD></TR></TBODY></TABLE>
Incubation at 22 °C for 1 h.
</DIV><H3>Transformation</H3><DIV class="floatleft"><OL><LI>5 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 500 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 5, 50 and 300 µl on LB plates with ampicilline</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H2>16.07.2013</H2><H3>Picking of colonies</H3><P>4 colonies were spread on a 1/4 plate.</P><H2>17.07.2013</H2><H3>Miniprep</H3><P>DNA was purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Yield: ~ 75 ng/µl</P><H3>Oligo annealing</H3><P>Ordered oligos (single stranded) were annealed to create the insert for the crRNA locus:</P><P> oIG3013 &amp; oIG3014 for Targetsequence I   oIG3015 &amp; oIG3016 for Targetsequence II </P><P> Mix: </P><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> Oligo 1 (100 µM) </TD><TD align="right"> 50 µl </TD></TR><TR><TD> Oligo 2 (100 µM) </TD><TD align="right"> 50 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> 150 µl </TD></TR></TBODY></TABLE><P> Incubation at 98 °C for 4 min, then let cool down for 3 h (in turned off heating block) </P><H2>18.07.2013</H2><H3>Digestion of pIG3010 with BbsI</H3><P>RNA plasmid was opened in order to insert the crRNA coding sequences.</P><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> pIG3010 (75 ng/µl) </TD><TD align="right"> 10 µl </TD></TR><TR><TD> Bbs I </TD><TD align="right"> 1 µl </TD></TR><TR><TD> NEB buffer 2.1 </TD><TD align="right"> 5 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 50 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 2 h.
</DIV><H3>Gel run</H3><H3>Gelextraction</H3><P>DNA was purified using <I>High Pure PCR Product Purification Kit</I> of Roche.</P><P>Changes to the protocol: 
<UL><LI>incubation at 56 °C for 10 min for gel dissolving</LI><LI>elution with dH<SUB>2</SUB>O (incubation at 50 °C for 4 min before centrifugation)</LI></UL></P><P>Yield: 40 ng/µl</P><H3>Insertion of annealed Oligos</H3><P>Digested pIG3010 was ligated with oligo I or II in a 1:3 or 1:6 ratio.</P><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> opened pIG3010 </TD><TD align="right"> 30 ng </TD></TR><TR><TD> Insert (20 µM) </TD><TD align="right"> 1 ng </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>
Incubation at 22 °C for 1 h.
</DIV><H3>Transformation</H3><DIV class="floatleft"><OL><LI>5 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 500 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with ampicilline</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H2>19.07.2013</H2><H3>Picking of colonies</H3><P>2 colonies of each plate were spread on a 1/4 plate.</P><H2>20.07.2013</H2><H3>Miniprep</H3><P>DNA was purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Only 1 colony of each ligation approach was prepped.</P><P>Yield: ~ 150 ng/µl</P><H3>Test digest</H3><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> pIG3011 (~ 150 ng/µl) </TD><TD align="right"> 2 µl </TD></TR><TR><TD> PstI HF </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NdeI </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NEB buffer 4 </TD><TD align="right"> 1 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 10 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 1:10 h.
</DIV><DIV class="center"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> pIG3012 (~ 150 ng/µl) </TD><TD align="right"> 2 µl </TD></TR><TR><TD> EcoRI HF </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> PflMI </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> BSA </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NEB buffer 2 </TD><TD align="right"> 1 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 10 µl </TD></TR></TBODY></TABLE></DIV><H3>Gel run</H3><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (1 µl); pIG3011 (1:3 ligation); pIG3011 (1:6 ligation); pIG3012 (1:3 ligation); pIG3012 (1:6 ligation)</TD></TR></TBODY></TABLE><P>Expected bands with insert: pIG3011 [0.2 &amp; 5.4 kb], pIG3012 [0.55, 0.7 &amp; 4.3 kb]</P><P>Expected bands without insert: pIG3011 [0.17 &amp; 5.4 kb], pIG3012 [0.7 &amp; 4.8 kb]</P><P>In pIG3012 there most probably no insert, for pIG3011 it is unknown.</P></DIV><H2>20.07.2013</H2><H3>Colony PCR</H3><P>Fwd. primer: fwd. oligo of insert (oIG3013 for pIG3011 &amp; oIG3015 for pIG3012); rev. primer: Gibson primer of Cas9 (oIG3004).</P><P>Control: Fwd. primer: Cas9 sequencing primer (oIG0015); rev. primer: Gibson primer of Cas9 (oIG3004).</P><DIV class="floatleft">
Mix:
<TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> pIG3011/pIG3012 </TD><TD align="right"> pick of colony OR 0.5 µl</TD></TR><TR><TD> Primers (10 µM) </TD><TD align="right"> each 1 µl</TD></TR><TR><TD> Taq polymerase </TD><TD align="right"> 0.33 µl </TD></TR><TR><TD> Taq standard buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dNTPs (2.5 mM) </TD><TD align="right"> 3 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE></DIV><DIV class="floatright">
Program:
<TABLE class="tabelle"><TBODY><TR><TH> temperature </TH><TH align="right"> time </TH></TR><TR><TD> 95 °C </TD><TD align="right"> 5 min </TD></TR><TR><TD> 95 °C </TD><TD align="right"> 30 s </TD></TR><TR><TD> 63 °C </TD><TD align="right"> 30 s </TD></TR><TR><TD> 68 °C </TD><TD align="right"> 20 s </TD></TR><TR><TD> 68 °C </TD><TD align="right"> 10 min </TD></TR></TBODY></TABLE>
Step 2 - 4 were repeated for 18 times.
</DIV><H3>Gel run</H3><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (2-log; 1 µl); 3x pIG3011 (1:3 ligation); 3x pIG3011 (1:6 ligation); 3x pIG3012 (1:3 ligation); 3x pIG3012 (1:6 ligation); control </TD></TR></TBODY></TABLE><P>Expected band: ~ 600 bp</P><P>Colony PCR did not work, because even the control does not show a band at the expected size.</P></DIV><H2> 22.07.2013 </H2><H3>Repeat of Insertion of annealed Oligos</H3><P>Ligation of pIG3010 with SEAP-Oligos I and II.</P><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> pIG3010 </TD><TD align="right"> 1,25 µl </TD></TR><TR><TD> Seap I/II </TD><TD align="right"> 2,5 µl </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>

Incubation at 22 °C for 1 h.

</DIV><DIV class="floatleft"><OL><LI>4 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 40 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 300 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with ampicilline</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H3>Miniprep</H3><P>DNA of 4 more colonies (2x pIG3011 + 2x pIG3012) were purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Yield: ~ 300 ng/µl</P><H3>Test digest with new colonies</H3><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> pIG3011 (~ 300 ng/µl) </TD><TD align="right"> 1 µl </TD></TR><TR><TD> PstI HF </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NdeI </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NEB buffer 4 </TD><TD align="right"> 1 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 10 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 2 h (all digests).
</DIV><DIV class="center"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> pIG3012 (~ 300 ng/µl) </TD><TD align="right"> 1 µl </TD></TR><TR><TD> EcoRI HF </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> PflMI </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> BSA </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NEB buffer 2 </TD><TD align="right"> 1 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 10 µl </TD></TR></TBODY></TABLE></DIV><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> pIG3010 (75 ng/µl) </TD><TD align="right"> 4 µl </TD></TR><TR><TD> PstI HF </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NdeI </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NEB buffer 4 </TD><TD align="right"> 1 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 10 µl </TD></TR></TBODY></TABLE>
This digest was performed to have a comparison for digest of pIG3011.
</DIV><H3>Gel run</H3><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right: Marker (1 µl); pIG3011 (1:3 ligation); pIG3010; pIG3011 (1:6 ligation); pIG3012 (1:3 ligation); pIG3012 (1:6 ligation)</TD></TR></TBODY></TABLE><P>Expected bands with insert: pIG3011 [0.2 &amp; 5.4 kb], pIG3012 [0.55, 0.7 &amp; 4.3 kb]</P><P>Expected bands without insert: pIG3011 [0.17 &amp; 5.4 kb], pIG3012 [0.7 &amp; 4.8 kb]</P><P>In pIG3012 there is most probably no insert, for pIG3011, too, because there is no difference to pIG3010 (plasmid without insert -&gt; band at about 0.2 kb should be 10 bp shorter).</P></DIV><H3>Sequencing</H3><P>pIG3011-65 (1:6 ligation, colony 5) and pIG3012-61 (1:6 ligation, colony 1) were sequenced with oIG0015.</P><P>Both sequencings indicate that the sequenced plasmids are pIG3010. Thus, the BbsI digest did not work.</P><H2>23.07.2013</H2><H3>Repeat of opening pIG3010</H3><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> digest </TH><TH align="right"> control of BsbI functionality </TH></TR><TR><TD> pIG3010 (75 ng/µl) </TD><TD align="right"> 10 µl </TD><TD align="right"> - </TD></TR><TR><TD> pIG3005 (1.7 µg/µl) </TD><TD align="right"> - </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> BbsI </TD><TD align="right"> 1 µl </TD><TD align="right"> 1 µ l</TD></TR><TR><TD> NEB buffer 2.1 </TD><TD align="right"> 5 µl </TD><TD align="right"> 5 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 50 µl </TD><TD align="right"> up to 50 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 1.5 h.
</DIV><H3>Gel run</H3><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right:  Marker (1 µl); digested pIG3010 (40 µl); digested pIG3010 (4 µl); undigested pIG3010 (4 µl); pIG3005 (4 µl) </TD></TR></TBODY></TABLE><P> Expected bands: pIG3010 [5.6 kb], pIG3005 [3.5 &amp; 6 kb].</P><P> The bands of pIG3005 are at the expected size -&gt; BbsI did work. pIG3010 showed a band at 5.6 kb, which was cut out. The band below is the undigested pIG3010. </P></DIV><H3>Gel extraction</H3><P>DNA were purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Changes to the protocol: 
<UL><LI>incubation at 56 °C for 10 min for gel dissolving</LI><LI>elution with dH<SUB>2</SUB>O (incubation at 50 °C for 4 min before centrifugation)</LI></UL></P><P>Yield: 120 ng/µl</P><H2> 24.07.2013 </H2><H3>Repeat of insertion of annealed Oligos</H3><P>Ligation of digested pIG3010 with SEAP-Oligos I and II.</P><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> pIG3010 </TD><TD align="right"> 0,4 µl </TD></TR><TR><TD> Seap I/II </TD><TD align="right"> 2,5 µl </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>

Incubation at 22 °C for 1 h.

</DIV><DIV class="floatleft"><OL><LI>4 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 40 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 300 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with ampicilline</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H2> 25.07.2013 </H2><H3> Coloy PCR on pIG3010 with target sites SEAP </H3><P>Colonies were picked and transferred into the following master mix. Afterwards they were streaked out onto plates, containing Amp as selective.</P><P>oIG3013I and 3015II were used as forward primers, oIG3004 as reverse primer.</P><DIV class="floatleft">
Mix:
<TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> Template </TD><TD align="right"> 1 colonie </TD></TR><TR><TD> Primers (10 µM) </TD><TD align="right"> each 0.4 µl</TD></TR><TR><TD> polymerase </TD><TD align="right"> 0.3 µl </TD></TR><TR><TD> buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dNTPs (2.5 mM) </TD><TD align="right"> 1.6 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE></DIV><DIV class="floatright">
Program:
<TABLE class="tabelle"><TBODY><TR><TH> temperature </TH><TH align="right"> time </TH></TR><TR><TD> 95 °C </TD><TD align="right"> 5 min </TD></TR><TR><TD> 95 °C </TD><TD align="right"> 20 s </TD></TR><TR><TD> 52 °C for control, 62 °C for samples </TD><TD align="right"> 30 s </TD></TR><TR><TD> 68 °C </TD><TD align="right"> 30 s </TD></TR><TR><TD> 68 °C </TD><TD align="right"> 5 min </TD></TR></TBODY></TABLE>
Step 2 - 4 were repeated for 25 times.
</DIV><DIV class="floatleft"><P>No specific amplification was observed. Oligos do not seem to be suitable as primers for cPCR.</P></DIV><H3>Miniprep</H3><P>DNA of 6 colonies (3x pIG3011 + 3x pIG3012) were purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>Yield: ~ 30 ng/µl</P><H2>26.07.2013</H2><H3>Test digest</H3><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> pIG3011 (~ 30 ng/µl) </TD><TD align="right"> 8 µl </TD></TR><TR><TD> PstI HF </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NdeI </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NEB buffer 4 </TD><TD align="right"> 1 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 3.5 h (all digests).
</DIV><DIV class="center"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> pIG3012 (~ 30 ng/µl) </TD><TD align="right"> 7.5 µl </TD></TR><TR><TD> EcoRI HF </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> PflMI </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> BSA </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NEB buffer 2 </TD><TD align="right"> 1 µl </TD></TR></TBODY></TABLE></DIV><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> pIG3010 (75 ng/µl) </TD><TD align="right"> 4 µl </TD></TR><TR><TD> PstI HF </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NdeI </TD><TD align="right"> 0.5 µl </TD></TR><TR><TD> NEB buffer 4 </TD><TD align="right"> 1 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 10 µl </TD></TR></TBODY></TABLE>
This digest was performed to have a comparison for digest of pIG3011.
</DIV><H3>Gel run</H3><DIV id="floathleft"><TABLE class="gelpic"><TBODY><TR><TD> From left to right:  Marker (1 µl); pIG3011_1-3;  digested pIG3010; pIG3012 ; marker 1 ul </TD></TR></TBODY></TABLE><P>Expected bands with insert: pIG3011 [0.2 &amp; 5.4 kb], pIG3012 [0.55, 0.7 &amp; 4.3 kb]</P><P>Expected bands without insert: pIG3011 [0.17 &amp; 5.4 kb], pIG3012 [0.7 &amp; 4.8 kb]</P><P>pIG3011_1 and pIG3012_2 were already sequenced and contain no insert. pIG3011_2 &amp; 3 do not really differ from pIG3011_1, pIG3012_2 show 4 bands, 3 of them at the expeced size.</P></DIV><H3>Sequencing</H3><P>pIG3011_3 and pIG3012_2 were sequenced with oIG0015. Unfortunately they turned out to be undigested pIG3010.</P><H3>2nd repeat of opening pIG3010</H3><DIV><P>
As test digestions and sequencing of the days before suggests that BbsI did not digest properly a new batch of pIG3010 was digested using BbsI and incubated overnight to avoid closed plasmid.
Nevertheless, sample pIG3011_3 and pIG3012_2 were sent for sequencing.
</P></DIV><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> pIG3010 </TD><TD align="right"> 0,7 µg; 8 µl </TD></TR><TR><TD> BbsI </TD><TD align="right"> 2 µl </TD></TR><TR><TD> NEB buffer 2.1 </TD><TD align="right"> 5 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 50 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C ovenight.
</DIV><H3>Trafo of digested pIG3010</H3><P>To test if there is still undigested plasmid in the sample of digested pIG3010, <I>E. coli</I> were transformed with 40 ng of this sample and - as a control - with 40 ng of undigested pIG3010.</P><DIV class="floatleft"><OL><LI>40 ng of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 45 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 500 µl LB medium</LI><LI>incubation for 15 min at 37 °C (shaking)</LI><LI>distribution of 200 µl on LB plates with ampicilline</LI><LI>incubation over night at 37 °C </LI></OL><P>The plate with the digested pIG3010 contained half as much as the plate with the undigested pIG3010. Thus, the BbsI digest did not work properly. However it has to be considered that the bacteria are able to repare double strand breaks via non homologous end joining.</P></DIV><H3>Digest of pIG3010 with BbsI (Alina and Philipp)</H3><P> As the digest of pIG3010 seems to have still a lot of undigested backbone, digestion was repeated.</P><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> volume </TH></TR><TR><TD> pIG3010 </TD><TD align="right"> 1 µg </TD></TR><TR><TD> BbSI </TD><TD align="right"> 1 µl from stock (-80°C) </TD></TR><TR><TD> NEB buffer 2.1 </TD><TD align="right"> 5 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 50 µl </TD></TR></TBODY></TABLE>
Incabation at 37 °C for 3 h. 
</DIV><H2> 29.07.2013 </H2><H3>Repeat of insertion of annealed Oligos</H3><P>Ligation of digested pIG3010 with SEAP-Oligos I and II.</P><DIV class="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> ingredient </TH><TH align="right"> amount </TH></TR><TR><TD> pIG3010 </TD><TD align="right"> 30 ng </TD></TR><TR><TD> Seap I/II </TD><TD align="right"> 1 ng </TD></TR><TR><TD> T4-Ligase </TD><TD align="right"> 1 µl </TD></TR><TR><TD> T4-Ligase buffer </TD><TD align="right"> 2 µl </TD></TR><TR><TD> dH<SUB>2</SUB>O </TD><TD align="right"> up to 20 µl </TD></TR></TBODY></TABLE>

Incubation at 22 °C for 1 h.

</DIV><DIV class="floatleft"><OL><LI>4 µl of plasmid were added to 25 µl of chemically competent <I>E. coli</I> cells</LI><LI>incubation for 10 min on ice</LI><LI>heatshock (42 °C for 40 s)</LI><LI>incubation for 2 min on ice</LI><LI>addition of 300 µl LB medium</LI><LI>incubation for 1 h at 37 °C (shaking)</LI><LI>distribution of 300 µl on LB plates with ampicilline</LI><LI>incubation over night at 37 °C </LI></OL></DIV><H2>31.07.2013</H2><H3>Miniprep</H3><P>DNA was purified using <I>High Pure Plasmid Isolation Kit</I> of Roche.</P><P>8 colonies of each ligation approach was prepped.</P><P>Yield: ~ 250 ng/µl</P><P> randomized samples were sent for sequencing. </P><H2>31.07.2013</H2><H3> Sequencing results </H3><P> pIG3011_4 and pIG3012_7 proved themselves to be correct. 100ml LB containing Amp were inoculated with these strains, as well as the CMV:SEAP construct.</P><DIV id="august"><P id="h2">
August
</P></DIV><H2>02.08.2013</H2><H3>Midiprep of pIG3011 and pIG3012</H3><P>DNA was purified using <I>Jetstar Plasmid Purification midi kit</I>.</P><P>Yield: ~  2,5µg/µl</P><H2> 04.08.2013 (Alina &amp; Philipp) </H2><H3> Seeding of cells for functioal test of ERT as catalytic inhibitor of CAS9.</H3><P> As the plasmids are ready,  HEK293T cells were be seeded in 24 wells with 65.000 cells each well. This will allow a functional test of possible inhibtory effects of Cas9:ERT. A quantitative SEAP essay should give answer to the question wether we can induce CRISPRi via Tamoxifen.Seeding was performed according to standard protocol. </P><H2>06.08.2013</H2><H3> Transfection </H3><DIV class="floatleft">
Protocol:
<OL><LI>40 µl Opti-MEM + 1.5 µl PEI-solution were mixed in a 1.5 ml Eppi.</LI><LI>0.5 µg of the DNA of interest were prepaired in another Eppi .</LI><LI>Addition of the DNA to former Eppi, vortexing for 15 s and incubation for 15 min at RT</LI><LI>Solution was spread them drop-wise to the cells in the dish</LI>
Transfection scheme:
<H2>07.08.2013</H2><H3>Medium change</H3>
12 h after transfection DMEM complete was removed and replaced by 0.5 ml per well DMEM complete containing 1 µM 4-OHT or 0.1 % EtOH, respectively, according to the transfection scheme.

<H2>08.08.2013</H2><H3>Medium change</H3>
24 h later medium was changed again according to the transfection scheme, this time with a washing step (0.5 ml DPBS per well).

<H2>09.08.2013</H2><H3>SEAP measurement</H3><P>The following was done 22 h after the last medium change (58 h after transfection) with technical triplicates.</P><UL><LI>400 µl of cell supernatant were collected in Epis.</LI><LI>Incubation for 30 min at 65 °C.</LI><LI>Centrifugation for 1 min at 1250 g.</LI><LI>100 µl of SEAP buffer were filled in the wells of a 96 well plate.</LI><LI>Addition of 80 µl supernatant to each well.</LI><LI>Addition of 20 µl pNPP (substrate).</LI><LI>96 plate was immediately placed in the blade reader.</LI></UL><P>Spectroscopic measurement every minute for 150 times; wavelenght 405 nm.</P><P>Measurement was aborted, as the SEAP concentration was too high.</P><H2>10.08.13</H2><H3>Repeat of SEAP measurement</H3><P>Cell supernatant was diluted 1:10 with DMEM that was heated for 30 min at 65 °C and measurement was repeated.</P><DIV id="floathleft"><P>The results indicate that also the functionality of the Cas9-ERT2 fusion protein is not controlable with hormone induction, as the SEAP expression is more or less the same for wells with and without 4-OHT.  </P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Freiburg/Notebook/lab_hormon">http://2013.igem.org/Team:Freiburg/Notebook/lab_hormon</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Notebook/lab_hormon" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Notebook/lab_hormon" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_hormon&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Notebook/lab_hormon&amp;oldid=343460" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></OL></DIV></DIV></DIV></DIV></BODY></HTML>