"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg/Notebook/lab hormon
From 2013.igem.org
Home
Highlights
Project
Abstract & Introduction
Targeting
Effectors
Effector Control
Modeling
Truncation
uniBAss
uniBOX
Manual
Application
Notebook
Targeting
Effectors
Effector Control
Modeling
uniBAss
Standardization
Material and Methods
BioBricks
Our BioBricks
Favorite BioBricks
Improvement of a Registry part
Sharing of our BioBricks
Safety
Human Practice
Expert opinions
Ethics
Safety & Security
Politics
Society
Education
Art
Facebook / Media
Team
News
Members
Exolab-Events
Statistics
Photo Gallery
Our Partners
Collaborations
Attributions
Thanks to
Targeting
Effector
Effector Control
Light
Hormone
April
May
June
July
August
Modeling
uniBAss
Standardization
Material and Methods
Effector Control - Hormone
Planned constructs
Practical work: Localization
April
17.04.2013
Trafo of pMZ333
pMZ333 containing mCherry and the backbone (Bb)
18.04.2013
Picking of clones
3 colonies of pMItom (containing ERT2) and 1 of pMZ333.
19.04.2013
Minipreps of pMZ333 and pMItom
Yield: 100 - 300 ng/µl
23.04.2013
Digestion of pMZ333
Bb was cut out of pMZ333 by XbaI and EcoRI.
Gel run
0,7 % agarose, marker: GeneRuler 1 kb
From left to right: Marker - Test digestion of pMItom - xx - Digestion of pMZ333 (3x)
The bands of the test digestion were as exspected.
pMZ333 was cut in two fragments, where the shorter one is the Bb (lower band).
Gel extraction
Bb was purified.
Yield: 20 - 30 ng/µl
25.04.2013
PCR of aIG3001 and aIG3006-8
Gel run
0,9 % agarose, marker: GeneRuler 1 kb
From left to right: Marker - xx - aIG3001 - aIG3006 - aIG3007 - aIG3008
All PCR products (the bright bands) had the exspected length.
Gel extraction
PCR products were purified.
Yield: 30 - 90 ng/µl
May
03.05.2013
PCR of aIG3002-5
Mix:
ingredient
volume
Template
1 µl
Primers (10 µM)
each 2.5 µl
Q5-HF MasterMix (NEB) 2x
25 µl
dH2O
up to 50 µl
Gel run
0,9 % agarose, marker: GeneRuler 1 kb
From left to right: aIG3002 - aIG3004 - aIG3003 - aIG3005 - xx - Marker
All PCR products (the bright bands) had the exspected length.
Gel extraction
PCR products were purified with the following changes to the standard protocol:
After the first incubation, the samples were vortexed followed by another incubation for 2 min at 50°C.
Between the centrifugations after PE buffer incubation the epis were turned around for 180°.
After 13 min incubation with water, samples were incubated for 4 min at 55°C.
Yield: 100 - 140 ng/µl
06.05.2013
Gibson assembly
Pipetting scheme:
to each symple 1 µl water was added
For pIG3005 and pIG3007 I used only the half amount of DNA as otherwise the total volume would have been higher than 5 µl.
Gibson assembly was performed by the following protocol:
5 µl of DNA mixes were added to 15 µl of Gibson master mix, each.
Incubation for 1 h at 50°C
Incubation for 3 min at RT
Incubation for 10 min on ice
Trafo
4 µl of each assemblies were added to 25 µl of chemically competent E. coli cells
incubation for 13 min on ice
heatshock (42 °C for 45 s)
incubation for 2 min on ice
addition of 500 µl LB medium
incubation for 15  min at 37 °C (shaking)
20 µl, 200 µl and 300 µl (concentrated) were distributed on agar plates with ampicilline
incubation over night at 37 °C
07.05.2013
Trafo repeat
As there were very few colonies only on the 300 µl plates which were surrounded by "satellite colonies", transformation was repeated by using the same protocol exept of:
The first incubation at 37°C was 1:15 h.
Only 300 µl (not concentrated) were distributed.
Over night incubation lasted only 16 h.
08.05.2013
Picking of clones
Colonies from plates pIG3001-8 (exept of pIG3006, because there were no colonies) of trafo repeat were spread on new agar plates.
Colonies of the first gibson trafo hopefully containing pIG3003,5,6 were spread on new plates to get single colonies.
09.05.2013
Picking of clones
Single colonies of the first gibson trafo were spread on new agar plates.
Minipreps of the second gibson trafo
Yield: 40 - 350 µg/ml
Test digest of gibson constructs
ingredient
volume
plasmid
approx. 1 µg
NcoI HF
0.5 µl
NEB buffer 4
2 µl
dH2O
up to 20 µl
Incabation at 37 °C for 2.5 h.
10.05.2013
Gel run of test digest
1 % agarose, marker: GeneRuler 1 kb
From left to right: pIG300..,Colony:
1,1 - 1,2 - 1,3 - 1,4 - 2,1 - 2,2 - Marker - 2,3 - 3,1 - 4,1 - 4,2 - 4,3 - 4,4
From left to right: pIG300..,Colony:
xx - 5,1 - 7,1 - 7,2 - 7,3 - 7,4 - Marker - 8,1 - 8,2 - 8,3 - 8,4
The banding pattern of pIG3001 (all colonies); 2,1; 2,2; 4,1; 4,2; 4,4 and 8 (all colonies) has been proven to be the expected one by sequencing the linkers. Thus, the marker run has to be wrong.
Minipreps of the first gibson trafo
Changes:
Between the second washing step (centrifugation for 30 s) and the drying step (centrifugation for 1 min) the collection tube was changed.
The incubation with water took only 3 min.
Yield: 40 - 180 ng/µl
Test digest of gibson constructs
ingredient
volume
plasmid
approx. 1 µg
NcoI HF
0.3 µl
NEB buffer 4
2 µl
dH2O
up to 20 µl
Incabation at 37 °C for 1:45 h.
Gel run of second test digest
1 % agarose, marker: GeneRuler 1 kb
From left to right: pIG300..,Colony:
xx - 3,1 - 3,2 - 5,1 - Marker - 5,2 - 6,1 - 6,2 - xx - xx
The banding pattern of pIG3003 has been proven to be the expected one by sequencing the linkers. Like above the marker ran wrong.
14.05.2013
Fusion PCR of aIG3009 & 10
aIG3005 & 6 were fused to aIG3009, aIG3007 & 8 to aIG3010 in order to reduce the fragments for Gibson assembly.
Mix:
ingredient
volume
Templates
each 1 µl
Q5 polymerase
0.3 µl
Q5 buffer (5x)
10 µl
dNTPs (2.5 mM)
4 µl
DMSO
1 µl
dH2O
up to 45 µl
Program A:
temperature
time
98 °C
5 min
98 °C
30 s
**
30 s
72 °C
*
Step 2 - 4 were repeated for 5 times.
*  time calculation: size template [kb] x 30 s
** 72 °C - 0,5 per cycle for aIG3009; 69 °C - 0,5 per cycle for aIG3010
Program B (after addition of 2,5 µl Primers, each):
temperature
time
98 °C
5 min
98 °C
30 s
63 °C
30 s
72 °C
*
72 °C
10 min
Step 2 - 4 were repeated for 5 times.
*  time calculation: size template [kb] x 30 s
Gel run of fusion PCR
Fragment´s lengths were as expected, though the band of aIG3009 was very slight.
15.05.2013
Gel extraction of fusion PCR
PCR products were purified as described on 03.05.2013, but DNA solution was loaded on the column again.
Yield: 6 / 49 ng/µl
16.05.2013
PCR of aIG3006 (repeat)
This PCR was repeated as the concentration yielded from the first attempt is too low for Gibson assembly.
21.05.2013
Repeat of fusion PCR of aIG3009
Fusion PCR was repeated, as the concentration of the first attempt is too low for Gibson assembly.
Gel run of PCR of aIG3006 and fusion PCR of aIG3009
Gel did not show the desired bands.
Gibson assembly 2
Pipetting scheme:
For pIG3005 and pIG3007 only the half amount of each PCR product was used as the concentration of aIG3006 was very   low.
The volumes were calculated that 5 µl contain the required DNA amount.
Gibson assembly was performed by the following protocol:
5 µl of DNA mixes were added to 15 µl of Gibson master mix, each.
Incubation for 1 h at 50°C
Incubation for 3 min at RT
Incubation for 10 min on ice
Trafo
4 µl of each assemblies were added to 25 µl of chemically competent E. coli cells
incubation for 13 min on ice
heatshock (42 °C for 45 s)
incubation for 2 min on ice
addition of 500 µl LB medium
incubation for 15  min at 37 °C (shaking)
20 µl, 200 µl and 300 µl (concentrated) were distributed on agar plates with ampicilline
incubation over night at 37 °C
22.05.2013
Midiprep of pIG3003, 4 & 8
As the sequencing results were positive, plasmids were amplified and midiprepped according to the manual of Jetstar Plasmid   Purification MIDI Kit
Yield: ~ 1 µg/µl for pIG3004 & 8; no DNA was isolated for pIG3003.
23.05.2013
Miniprep of Gibson assembly 2
DNA was purified using High Pure Plasmid Isolation Kit of Roche.
Yield: 30 - 90 ng/µl
Test digest of Gibson assembly 2
Mix 1 was used for DNA concentrations beneath 60 ng/µl, Mix 2 for concentrations above 60 ng/µl.
ingredient
mix 1
mix 2
plasmid
17.5 µl
10 µl
NcoI HF
0.2 µl
0.2  µl
NEB buffer 4
2 µl
2 µl
dH2O
up to 20 µl
up to 20 µl
Incabation at 37 °C for 1.5 h.
24.05.2013
Gel run
From left to right: Marker (GeneRuler 1 kb); pIG3001 (4x); pIG3006 (4x); pIG3007 (4x).
The bands of pIG3001 & 6 could be at the right size, so they were send in for sequencing.
27.05.2013
Amplification of aIG3006 & 9 by PCR
Mix:
ingredient
volume
aIG3006 / 9
5 µl
Primers (10 µM)
each 1 µl
Q5 DNA polymerase
0.5 µl
Q5 buffer
10 µl
DMSO
1 µl
dNTPs (2.5 mM)
4 µl
dH2O
up to 50 µl
Program:
temperature
time
98 °C
5 min
98 °C
30 s
63 °C
30 s
72 °C
160 s
72 °C
10 min
Step 2 - 4 were repeated for 18 times.
Yield after Gel extraction with High Pure Plasmid Isolation Kit of Roche: 40 ng/µl for aIG3006 and 82 ng/µl for aIG3009.
28.05.2013
Gibson assembly 3
Pipetting scheme:
For pIG3005 and pIG3007 only the half amount of each PCR product was used as the concentration of aIG3006 was very   low.
5 µl of each DNA mix was added to 15 µl Gibson master mix.
Gibson assembly was performed by the following protocol:
5 µl of DNA mixes were added to 15 µl of Gibson master mix, each.
Incubation for 1 h at 50°C
Incubation for 3 min at RT
Incubation for 10 min on ice
Trafo
4 µl of each assemblies were added to 25 µl of chemically competent E. coli cells
incubation for 13 min on ice
heatshock (42 °C for 45 s)
incubation for 2 min on ice
addition of 500 µl LB medium
incubation for 15  min at 37 °C (shaking)
20 µl, 200 µl and 300 µl (concentrated) were distributed on agar plates with ampicilline
incubation over night at 37 °C
Transfection of pIG3004 and pIG3008
0.75 ng of DNA per well were filled up to 50 µl with OptiMEM
2.5 µl of PEI per well were filled up to 50 µl with OptiMEM
These two solutions were mixed by immediately 2x vortexing for ~ 5 s. After 20 min incubation 100 µl were added dropwise to   each well.
Medium change was performed after 2 h.
28.05.2013
Cell fixation
Cells on each well were washed with 500 µl PBS and fixed with 200 µl PFA (4 %) solution by incubating for 10 min on ice and at   least   20 min at RT.
Cells attached on cover slides were stained with DAPI and DRAQ5 (incubation in DAPI or DRAQ5 solution consisting of 1 µl DAPI or     DRAQ5 in 5 ml dH2O for 5 s, respectively) and rinsed in dH2O. Cover slides were dried and mounted on a drop of Mowiol solution   (with   DABCO) with cells down. After a drying step at 37 °C for 15 min samples were sealed with nail polish and stored at 4 °C.
Flourescence microscopy
pIG3004
pIG3008
These pictures show that pIG3004 is clearly localized in the nucleus whereas pIG3008 is localized in the cytoplasm. Nevertheless more cells have to be analyzed.
30.05.2013
Midiprep of pIG3001, 3 & 6
As the sequencing results were positive, plasmids were amplified and midiprepped according to the manual of Jetstar Plasmid Purification MIDI   Kit
Yield: 1.4 µg/µl for pIG3006; 470 ng/µl for pIG3001; 71 ng/µl for pIG3003.
Test digest of Gibson assembly 3
ingredient
volumes
plasmid (~ 100 ng/µl)
5 µl
NcoI HF
0.2 µl
NEB buffer 4
2 µl
dH2O
up to 20 µl
Incabation at 37 °C for 2.5 h.
31.05.2013
Gel run
From left to right: Marker (GeneRuler 1 kb); pIG3002 (3x); pIG30052 (3x); pIG3007 (3x).
The bands of pIG3002, 5 & 7-a were at the right size, so they were send in for sequencing.
June
04.06.2013
Transfection of pIG3001, 3, 4, 6 & 8
HeLa cells seeded onto cover slides in a 24 well plate were transfected by the following protocol:
0.75 ng of DNA per well were filled up to 50 µl with OptiMEM
2.5 µl of PEI per well were filled up to 50 µl with OptiMEM
These two solutions were mixed by immediately 2x vortexing for ~ 5 s. After 20 min incubation 100 µl were added dropwise to   each well.
Transfection scheme:
05.06.2013
Second Miniprep of Gibson assembly 3
As the sequencing of the only positive colony of pIG3007 pointed out a mutation, more colonies were prepped using High Pure Plasmid Isolation   Kit of Roche.
Yield: ~ 60 ng/µl
Second test digest of Gibson assembly 3
ingredient
volumes
plasmid (~ 60 ng/µl)
10 µl
NcoI HF
0.5 µl
NEB buffer 4
2 µl
dH2O
up to 20 µl
Incabation at 37 °C for 2.5 h.
Gel run
From left to right: Marker (GeneRuler 1 kb); pIG3007 (3x).
The bands were at the right size, so pIG3007 was send in for sequencing.
06.06.2013
4-OHT treatment
4-OHT was diluted in DMEM complete to a final concentration of 0.1, 1 and 10 µM; EtOH was diluted in DMEM complete to a final concentration of 0.1   and 1 % (which is the same amount as in the medium with 4-OHT, as 4-OHT is dissolved in EtOH).
The old medium was removed from the cells and the new mixed media were added according to the transfection scheme.
Cell fixation
After 0.5, 1 and 3 h (according to the transfection scheme) cells on each well were washed with 500 µl PBS and fixed with 200 µl PFA (4 %) solution by   incubating for 10 min on ice and at least 20 min at RT.
Cells attached on cover slides were stained with DAPI and DRAQ5 (incubation in DAPI or DRAQ5 solution consisting of 1 µl DAPI or  DRAQ5 in 5 ml dH2O   for 5 s, respectively) and rinsed in dH2O. Cover slides were dried and mounted on a drop of Mowiol solution (with DABCO) with cells down. After a drying   step at 37 °C for 15 min samples were sealed with nail polish and stored at 4 °C.
Midiprep of pIG3002, 3, 5 & 7
As the sequencing results were positive, plasmids were amplified and midiprepped according to the manual of Jetstar Plasmid Purification MIDI   Kit
Yield: 1.7 µg/µl for pIG3005; 670 ng/µl for pIG3007; 540 ng/µl for pIG3002; 335 ng/µl for pIG3003.
07.06.2013
Microscopy
All samples showed red flourescence (except of pIG3006 & 8 treated with 10 µM 4-OHT, as there were almost no cells).
Treatment with 0.1 µM 4-OHT did not cause a difference to treatment with EtOH.
Treatment with 1 µM 4-OHT seemed to increase nuclear import for pIG3006 & 8.
pIG3006 is always localized in the nucleus.
By cell shape concentrations of 4-OHT lower than 10 µM does not seem to harm HeLa cells.
10.06.2013
Midiprep of pIG3009
As experimental results indicate, that the 5' NLS has a maybe stronger influence on nuclear import than the 3' NLS, the wrong assembled plasmid with   the insert HA-ERT2-NLS-Cas9-mCherry (:= pIG3009) was amplified and midiprepped according to the manual of Jetstar Plasmid Purification MIDI   Kit
Yield: 300 ng/µl.
11.06.2013
Transfection of all constructs
HeLa cells seeded onto cover slides in a 24 well plate were transfected by the following protocol:
0.75 ng of DNA per well were filled up to 50 µl with OptiMEM
2.5 µl of PEI per well were filled up to 50 µl with OptiMEM
These two solutions were mixed by immediately 2x vortexing for ~ 5 s. After 20 min incubation 100 µl were added dropwise to   each well.
Transfection scheme:
12.06.2013
4-OHT treatment
4-OHT was diluted in DMEM complete to a final concentration of 0.1 and 1 µM; EtOH was diluted in DMEM complete to a final concentration of 0.1 %   (which is the same amount as in the medium with 4-OHT, as 4-OHT is dissolved in EtOH).
31 h after transfection the old medium was removed from the cells and the new mixed media were added according to the transfection scheme.
Cell fixation
After 3 and 16 h (according to the transfection scheme) cells on each well were washed with 500 µl PBS and fixed with 200 µl PFA (4 %) solution by   incubating for 10 min on ice and at least 20 min at RT.
Cells attached on cover slides were stained with DAPI and DRAQ5 (incubation in DAPI or DRAQ5 solution consisting of 1 µl DAPI or  DRAQ5 in 5 ml dH2O   for 5 s, respectively) and rinsed in dH2O. Cover slides were dried and mounted on a drop of Mowiol solution (with DABCO) with cells down. After a drying   step at 37 °C for 15 min samples were sealed with nail polish and stored at 4 °C.
13.06.2013
Toxity assay for 4-OHT
Untransfected HeLa cells were treated for one day with 4-OHT, EtOH (different concentrations, each) or no addition. Supernatant was removed and a   washing step with 0.5 ml PBS was performed. The cells in the supernatant and PBS were counted with CASY counter. The number of dead cells was   calculated by multiplying the number of living cell with (100 - percentage of living cells) / 100.
Confocal fluorescence microscopy
Pictures were taken of all different constructs by a confocal fluorescence microscope. The ration of the mean nuclear to cytoplasmic red flourescence   was determined by ImageJ.
No significant difference in subcellular localisation upon 4-OHT stimulus was detectable this way.
19.06.2013
Second transfection of all constructs
HeLa cells seeded onto cover slides in a 24 well plate were transfected by the following protocol:
0.75 ng of DNA per well were filled up to 50 µl with OptiMEM
2.5 µl of PEI per well were filled up to 50 µl with OptiMEM
These two solutions were mixed by immediately 2x vortexing for ~ 5 s. After 20 min incubation 100 µl were added dropwise to   each well.
Transfection scheme:
20.06.2013
4-OHT treatment
4-OHT was diluted in DMEM complete to a final concentration of 1 and 5 µM; EtOH was diluted in DMEM complete to a final concentration of 0.1 %   (which is the same amount as in the medium with 4-OHT, as 4-OHT is dissolved in EtOH).
31 h after transfection the old medium was removed from the cells and the new mixed media were added according to the transfection scheme.
21.06.2013
Cell fixation
After an over night incubation cells on each well were washed with 500 µl PBS and fixed with 200 µl PFA (4 %) solution by incubating for 10 min on ice   and at least 20 min at RT.
Cells attached on cover slides were stained with DAPI and DRAQ5 (incubation in DAPI or DRAQ5 solution consisting of 1 µl DAPI or  DRAQ5 in 5 ml dH2O   for 5 s, respectively) and rinsed in dH2O. Cover slides were dried and mounted on a drop of Mowiol solution (with DABCO) with cells down. After a drying   step at 37 °C for 15 min samples were sealed with nail polish and stored at 4 °C.
Fluorescence microscopy
No big difference between 4-OHT and EtOH treatment was detectable.
25.06.2013
Transfection for western blotting
HEK cells seeded in three 6 well plates were transfected with pIG3001, 2, 5, 7 & 9 by the following protocol:
3.75 ng of DNA per well were filled up to 250 µl with OptiMEM
12.5 µl of PEI per well were filled up to 250 µl with OptiMEM
These two solutions were mixed by immediately 2x vortexing for ~ 5 s. After 20 min incubation 500 µl were added dropwise to   each well.
26.06.2013
4-OHT treatment
4-OHT was diluted in DMEM complete to a final concentration of 1 µM; EtOH was diluted in DMEM complete to a final concentration of 0.1 % (which is   the same amount as in the medium with 4-OHT, as 4-OHT is dissolved in EtOH).
30 h after transfection the old medium was removed from the cells and the new mixed media were added to the wells, both 4-OHT and EtOH for each   construct.
27.06.2013
Cell fractionation
Cells were washed twice with 1 ml PBS, then incubated for 30 min on ice with 0.5 ml hypotonic lysis buffer (20 mM HEPES pH=7.4; 5 mM MgCl2; 1 mM   EDTA; 1 Protease inhibitor cocktail tablet / 50 ml buffer). After mechanical detachment lysates were centrifuged for 5 min at 10,000 g. Supernatant and   pellet were separated.
Sample preparation for SDS-PAGE
200 µl of supernatants were mixed with 50 μl of 5x SDS loading buffer (10 % SDS; 0.3125 M Tris-HCl pH=6.8; 50 % glycerol, 12.5 % 2-Mercaptoethanol;   0.025 % Bromphenol blue) and heated for 5 min at 95 °C. The pellets resuspended in 15 μl 5x SDS loading buffer and heated for 5 min at 95 °C were diluted   with 120 μl hypotonic lysis buffer and another 30 μl SDS loading buffer (5x) and sonified for 10 min at maximum.
SDS-PAGE cast
At first the separation gel was cast and covered with 200 μl isopropanol. After polymerisation the collection gel was cast on top.
separation gel (8 %)
ingredient
collection gel
19.2 ml
dH2O
7 ml
10.4 ml
30 % acryl-bisacryl mix
2 ml
10.4 ml
1.5 M Tris with 0.4 % SDS, pH=8.8
0.5 M Tris with 0.4 % SDS, pH=6.8
3 ml
400 µl
APS (10%)
120 µl
40 µl
TEMED
12 µl
40 ml
sum
12 ml
150 V were supplied for about 1.5 h.
Western blotting
A PVDF-membrane was activated in methanol and washed in transfer buffer (192 mM glycin; 25 mM Tris; 10 % methanol). 3 sheets of whatman paper   were incubated in transfer buffer, too. The blotting apparatus was loaded in the following order from bottom up: 2 sheets of whatman paper - PVDF-  membrane - gel - 1 sheet of whatman paper. An amperage of 0.35 A per membrane was supplied for 1.5 h.
27.06.2013
Antibody treatment
Membranes were incubated for 1 h in blocking buffer (PBS with 3 % milk powder), then for 2.5 h with anti HA mouse (1:1000 in blocking buffer). After 3   times washing for 5 min with PBS-T (PBS with 0.05 % Tween 20) membranes were incubated for 1 h with anti mouse HRP (1:5000 in blocking buffer) and   washed again 3 times for 5 min with PBS-T.
Imaging was performed with automatically determined exposure time after addition of ECL I (250 mM luminol; 90 mM coumaric acid; 1 M Tris pH=8.5) + ECL   II (30 % hydrogen peroxide; 1 M Tris pH=8.5) solution (500 μl of each).
Pictures of western blot of nuclear (A) and cytoplasmic (B) cell fraction.
Arrows indicate most probably the fusion proteins; other bands are unspecific AB-bindings or (N-terminal) fusion protein fragments.
Only the controls are detectable and act as expected.
These data suggest that altering the subcellular localization of a Cas9 fusion protein upon a 4-OHT stimulus is not possible. Because of this it ought to be tested, if the functionality of Cas9 can be controlled by hormone induction. Thus, in a new experiment a SEAP plasmid should be targeted.
RNA Plasmid for SEAP Assay
pX334a digested with PstI and religated afterwards ("pIG3010") was used as initial plasmid for the introduction of crRNAs (oligos designt for targeting two different SEAP loci).
July
14.07.2013
Digest of pX334a
Cas9 was cut out of pX334a to get a plasmid containing only the tracrRNA and the crRNA.
ingredient
sample 1
sample 2
pX334a
0.9 µg
1.8 µg
PstI HF
1 µl
1 µl
NEB buffer 4
5 µl
5 µl
dH2O
up to 50 µl
up to 50 µl
Incabation at 37 °C for 2 h.
Gel run
From left to right: Marker (2µl); digest 1 (55 µl); digest 1 (4 µl); digest 2 (55 µl); digest 2 (4 µl); Marker (0.5 µl)
Only the upper band of digest 1 was cut out, as the digest of sample 2 was not completely.
15.07.2013
Gel extraction
DNA was purified using High Pure PCR Product Purification Kit of Roche.
Changes to the protocol:
incubation at 56 °C for 10 min for gel dissolving
elution with dH2O (incubation at 50 °C for 4 min before centrifugation)
Yield: 30 ng/µl
Religation of pX334a backbone
ingredient
amount
Backbone of pX334a
150 ng
T4-Ligase
1 µl
T4-Ligase buffer
2 µl
dH2O
up to 20 µl
Incubation at 22 °C for 1 h.
Transformation
5 µl of plasmid were added to 25 µl of chemically competent E. coli cells
incubation for 10 min on ice
heatshock (42 °C for 45 s)
incubation for 2 min on ice
addition of 500 µl LB medium
incubation for 1 h at 37 °C (shaking)
distribution of 5, 50 and 300 µl on LB plates with ampicilline
incubation over night at 37 °C
16.07.2013
Picking of colonies
4 colonies were spread on a 1/4 plate.
17.07.2013
Miniprep
DNA was purified using High Pure Plasmid Isolation Kit of Roche.
Yield: ~ 75 ng/µl
Oligo annealing
Ordered oligos (single stranded) were annealed to create the insert for the crRNA locus:
oIG3013 & oIG3014 for Targetsequence I   oIG3015 & oIG3016 for Targetsequence II
Mix:
ingredient
volume
Oligo 1 (100 µM)
50 µl
Oligo 2 (100 µM)
50 µl
dH2O
150 µl
Incubation at 98 °C for 4 min, then let cool down for 3 h (in turned off heating block)
18.07.2013
Digestion of pIG3010 with BbsI
RNA plasmid was opened in order to insert the crRNA coding sequences.
ingredient
volume
pIG3010 (75 ng/µl)
10 µl
Bbs I
1 µl
NEB buffer 2.1
5 µl
dH2O
up to 50 µl
Incabation at 37 °C for 2 h.
Gel run
Gelextraction
DNA was purified using High Pure PCR Product Purification Kit of Roche.
Changes to the protocol:
incubation at 56 °C for 10 min for gel dissolving
elution with dH2O (incubation at 50 °C for 4 min before centrifugation)
Yield: 40 ng/µl
Insertion of annealed Oligos
Digested pIG3010 was ligated with oligo I or II in a 1:3 or 1:6 ratio.
ingredient
amount
opened pIG3010
30 ng
Insert (20 µM)
1 ng
T4-Ligase
1 µl
T4-Ligase buffer
2 µl
dH2O
up to 20 µl
Incubation at 22 °C for 1 h.
Transformation
5 µl of plasmid were added to 25 µl of chemically competent E. coli cells
incubation for 10 min on ice
heatshock (42 °C for 45 s)
incubation for 2 min on ice
addition of 500 µl LB medium
incubation for 1 h at 37 °C (shaking)
distribution of 300 µl on LB plates with ampicilline
incubation over night at 37 °C
19.07.2013
Picking of colonies
2 colonies of each plate were spread on a 1/4 plate.
20.07.2013
Miniprep
DNA was purified using High Pure Plasmid Isolation Kit of Roche.
Only 1 colony of each ligation approach was prepped.
Yield: ~ 150 ng/µl
Test digest
ingredient
volume
pIG3011 (~ 150 ng/µl)
2 µl
PstI HF
0.5 µl
NdeI
0.5 µl
NEB buffer 4
1 µl
dH2O
up to 10 µl
Incabation at 37 °C for 1:10 h.
ingredient
volume
pIG3012 (~ 150 ng/µl)
2 µl
EcoRI HF
0.5 µl
PflMI
0.5 µl
BSA
0.5 µl
NEB buffer 2
1 µl
dH2O
up to 10 µl
Gel run
From left to right: Marker (1 µl); pIG3011 (1:3 ligation); pIG3011 (1:6 ligation); pIG3012 (1:3 ligation); pIG3012 (1:6 ligation)
Expected bands with insert: pIG3011 [0.2 & 5.4 kb], pIG3012 [0.55, 0.7 & 4.3 kb]
Expected bands without insert: pIG3011 [0.17 & 5.4 kb], pIG3012 [0.7 & 4.8 kb]
In pIG3012 there most probably no insert, for pIG3011 it is unknown.
20.07.2013
Colony PCR
Fwd. primer: fwd. oligo of insert (oIG3013 for pIG3011 & oIG3015 for pIG3012); rev. primer: Gibson primer of Cas9 (oIG3004).
Control: Fwd. primer: Cas9 sequencing primer (oIG0015); rev. primer: Gibson primer of Cas9 (oIG3004).
Mix:
ingredient
volume
pIG3011/pIG3012
pick of colony OR 0.5 µl
Primers (10 µM)
each 1 µl
Taq polymerase
0.33 µl
Taq standard buffer
2 µl
dNTPs (2.5 mM)
3 µl
dH2O
up to 20 µl
Program:
temperature
time
95 °C
5 min
95 °C
30 s
63 °C
30 s
68 °C
20 s
68 °C
10 min
Step 2 - 4 were repeated for 18 times.
Gel run
From left to right: Marker (2-log; 1 µl); 3x pIG3011 (1:3 ligation); 3x pIG3011 (1:6 ligation); 3x pIG3012 (1:3 ligation); 3x pIG3012 (1:6 ligation); control
Expected band: ~ 600 bp
Colony PCR did not work, because even the control does not show a band at the expected size.
22.07.2013
Repeat of Insertion of annealed Oligos
Ligation of pIG3010 with SEAP-Oligos I and II.
ingredient
amount
pIG3010
1,25 µl
Seap I/II
2,5 µl
T4-Ligase
1 µl
T4-Ligase buffer
2 µl
dH2O
up to 20 µl
Incubation at 22 °C for 1 h.
4 µl of plasmid were added to 25 µl of chemically competent E. coli cells
incubation for 10 min on ice
heatshock (42 °C for 40 s)
incubation for 2 min on ice
addition of 300 µl LB medium
incubation for 1 h at 37 °C (shaking)
distribution of 300 µl on LB plates with ampicilline
incubation over night at 37 °C
Miniprep
DNA of 4 more colonies (2x pIG3011 + 2x pIG3012) were purified using High Pure Plasmid Isolation Kit of Roche.
Yield: ~ 300 ng/µl
Test digest with new colonies
ingredient
volume
pIG3011 (~ 300 ng/µl)
1 µl
PstI HF
0.5 µl
NdeI
0.5 µl
NEB buffer 4
1 µl
dH2O
up to 10 µl
Incabation at 37 °C for 2 h (all digests).
ingredient
volume
pIG3012 (~ 300 ng/µl)
1 µl
EcoRI HF
0.5 µl
PflMI
0.5 µl
BSA
0.5 µl
NEB buffer 2
1 µl
dH2O
up to 10 µl
ingredient
volume
pIG3010 (75 ng/µl)
4 µl
PstI HF
0.5 µl
NdeI
0.5 µl
NEB buffer 4
1 µl
dH2O
up to 10 µl
This digest was performed to have a comparison for digest of pIG3011.
Gel run
From left to right: Marker (1 µl); pIG3011 (1:3 ligation); pIG3010; pIG3011 (1:6 ligation); pIG3012 (1:3 ligation); pIG3012 (1:6 ligation)
Expected bands with insert: pIG3011 [0.2 & 5.4 kb], pIG3012 [0.55, 0.7 & 4.3 kb]
Expected bands without insert: pIG3011 [0.17 & 5.4 kb], pIG3012 [0.7 & 4.8 kb]
In pIG3012 there is most probably no insert, for pIG3011, too, because there is no difference to pIG3010 (plasmid without insert -> band at about 0.2 kb should be 10 bp shorter).
Sequencing
pIG3011-65 (1:6 ligation, colony 5) and pIG3012-61 (1:6 ligation, colony 1) were sequenced with oIG0015.
Both sequencings indicate that the sequenced plasmids are pIG3010. Thus, the BbsI digest did not work.
23.07.2013
Repeat of opening pIG3010
ingredient
digest
control of BsbI functionality
pIG3010 (75 ng/µl)
10 µl
-
pIG3005 (1.7 µg/µl)
-
0.5 µl
BbsI
1 µl
1 µ l
NEB buffer 2.1
5 µl
5 µl
dH2O
up to 50 µl
up to 50 µl
Incabation at 37 °C for 1.5 h.
Gel run
From left to right:  Marker (1 µl); digested pIG3010 (40 µl); digested pIG3010 (4 µl); undigested pIG3010 (4 µl); pIG3005 (4 µl)
Expected bands: pIG3010 [5.6 kb], pIG3005 [3.5 & 6 kb].
The bands of pIG3005 are at the expected size -> BbsI did work. pIG3010 showed a band at 5.6 kb, which was cut out. The band below is the undigested pIG3010.
Gel extraction
DNA were purified using High Pure Plasmid Isolation Kit of Roche.
Changes to the protocol:
incubation at 56 °C for 10 min for gel dissolving
elution with dH2O (incubation at 50 °C for 4 min before centrifugation)
Yield: 120 ng/µl
24.07.2013
Repeat of insertion of annealed Oligos
Ligation of digested pIG3010 with SEAP-Oligos I and II.
ingredient
amount
pIG3010
0,4 µl
Seap I/II
2,5 µl
T4-Ligase
1 µl
T4-Ligase buffer
2 µl
dH2O
up to 20 µl
Incubation at 22 °C for 1 h.
4 µl of plasmid were added to 25 µl of chemically competent E. coli cells
incubation for 10 min on ice
heatshock (42 °C for 40 s)
incubation for 2 min on ice
addition of 300 µl LB medium
incubation for 1 h at 37 °C (shaking)
distribution of 300 µl on LB plates with ampicilline
incubation over night at 37 °C
25.07.2013
Coloy PCR on pIG3010 with target sites SEAP
Colonies were picked and transferred into the following master mix. Afterwards they were streaked out onto plates, containing Amp as selective.
oIG3013I and 3015II were used as forward primers, oIG3004 as reverse primer.
Mix:
ingredient
volume
Template
1 colonie
Primers (10 µM)
each 0.4 µl
polymerase
0.3 µl
buffer
2 µl
dNTPs (2.5 mM)
1.6 µl
dH2O
up to 20 µl
Program:
temperature
time
95 °C
5 min
95 °C
20 s
52 °C for control, 62 °C for samples
30 s
68 °C
30 s
68 °C
5 min
Step 2 - 4 were repeated for 25 times.
No specific amplification was observed. Oligos do not seem to be suitable as primers for cPCR.
Miniprep
DNA of 6 colonies (3x pIG3011 + 3x pIG3012) were purified using High Pure Plasmid Isolation Kit of Roche.
Yield: ~ 30 ng/µl
26.07.2013
Test digest
ingredient
volume
pIG3011 (~ 30 ng/µl)
8 µl
PstI HF
0.5 µl
NdeI
0.5 µl
NEB buffer 4
1 µl
Incabation at 37 °C for 3.5 h (all digests).
ingredient
volume
pIG3012 (~ 30 ng/µl)
7.5 µl
EcoRI HF
0.5 µl
PflMI
0.5 µl
BSA
0.5 µl
NEB buffer 2
1 µl
ingredient
volume
pIG3010 (75 ng/µl)
4 µl
PstI HF
0.5 µl
NdeI
0.5 µl
NEB buffer 4
1 µl
dH2O
up to 10 µl
This digest was performed to have a comparison for digest of pIG3011.
Gel run
From left to right:  Marker (1 µl); pIG3011_1-3;  digested pIG3010; pIG3012 ; marker 1 ul
Expected bands with insert: pIG3011 [0.2 & 5.4 kb], pIG3012 [0.55, 0.7 & 4.3 kb]
Expected bands without insert: pIG3011 [0.17 & 5.4 kb], pIG3012 [0.7 & 4.8 kb]
pIG3011_1 and pIG3012_2 were already sequenced and contain no insert. pIG3011_2 & 3 do not really differ from pIG3011_1, pIG3012_2 show 4 bands, 3 of them at the expeced size.
Sequencing
pIG3011_3 and pIG3012_2 were sequenced with oIG0015. Unfortunately they turned out to be undigested pIG3010.
2nd repeat of opening pIG3010
As test digestions and sequencing of the days before suggests that BbsI did not digest properly a new batch of pIG3010 was digested using BbsI and incubated overnight to avoid closed plasmid.
Nevertheless, sample pIG3011_3 and pIG3012_2 were sent for sequencing.
ingredient
volume
pIG3010
0,7 µg; 8 µl
BbsI
2 µl
NEB buffer 2.1
5 µl
dH2O
up to 50 µl
Incabation at 37 °C ovenight.
Trafo of digested pIG3010
To test if there is still undigested plasmid in the sample of digested pIG3010, E. coli were transformed with 40 ng of this sample and - as a control - with 40 ng of undigested pIG3010.
40 ng of plasmid were added to 25 µl of chemically competent E. coli cells
incubation for 10 min on ice
heatshock (42 °C for 45 s)
incubation for 2 min on ice
addition of 500 µl LB medium
incubation for 15 min at 37 °C (shaking)
distribution of 200 µl on LB plates with ampicilline
incubation over night at 37 °C
The plate with the digested pIG3010 contained half as much as the plate with the undigested pIG3010. Thus, the BbsI digest did not work properly. However it has to be considered that the bacteria are able to repare double strand breaks via non homologous end joining.
Digest of pIG3010 with BbsI (Alina and Philipp)
As the digest of pIG3010 seems to have still a lot of undigested backbone, digestion was repeated.
ingredient
volume
pIG3010
1 µg
BbSI
1 µl from stock (-80°C)
NEB buffer 2.1
5 µl
dH2O
up to 50 µl
Incabation at 37 °C for 3 h.
29.07.2013
Repeat of insertion of annealed Oligos
Ligation of digested pIG3010 with SEAP-Oligos I and II.
ingredient
amount
pIG3010
30 ng
Seap I/II
1 ng
T4-Ligase
1 µl
T4-Ligase buffer
2 µl
dH2O
up to 20 µl
Incubation at 22 °C for 1 h.
4 µl of plasmid were added to 25 µl of chemically competent E. coli cells
incubation for 10 min on ice
heatshock (42 °C for 40 s)
incubation for 2 min on ice
addition of 300 µl LB medium
incubation for 1 h at 37 °C (shaking)
distribution of 300 µl on LB plates with ampicilline
incubation over night at 37 °C
31.07.2013
Miniprep
DNA was purified using High Pure Plasmid Isolation Kit of Roche.
8 colonies of each ligation approach was prepped.
Yield: ~ 250 ng/µl
randomized samples were sent for sequencing.
31.07.2013
Sequencing results
pIG3011_4 and pIG3012_7 proved themselves to be correct. 100ml LB containing Amp were inoculated with these strains, as well as the CMV:SEAP construct.
August
02.08.2013
Midiprep of pIG3011 and pIG3012
DNA was purified using Jetstar Plasmid Purification midi kit.
Yield: ~  2,5µg/µl
04.08.2013 (Alina & Philipp)
Seeding of cells for functioal test of ERT as catalytic inhibitor of CAS9.
As the plasmids are ready,  HEK293T cells were be seeded in 24 wells with 65.000 cells each well. This will allow a functional test of possible inhibtory effects of Cas9:ERT. A quantitative SEAP essay should give answer to the question wether we can induce CRISPRi via Tamoxifen.Seeding was performed according to standard protocol.
06.08.2013
Transfection
Protocol:
40 µl Opti-MEM + 1.5 µl PEI-solution were mixed in a 1.5 ml Eppi.
0.5 µg of the DNA of interest were prepaired in another Eppi .
Addition of the DNA to former Eppi, vortexing for 15 s and incubation for 15 min at RT
Solution was spread them drop-wise to the cells in the dish
Transfection scheme:
07.08.2013
Medium change
12 h after transfection DMEM complete was removed and replaced by 0.5 ml per well DMEM complete containing 1 µM 4-OHT or 0.1 % EtOH, respectively, according to the transfection scheme.
08.08.2013
Medium change
24 h later medium was changed again according to the transfection scheme, this time with a washing step (0.5 ml DPBS per well).
09.08.2013
SEAP measurement
The following was done 22 h after the last medium change (58 h after transfection) with technical triplicates.
400 µl of cell supernatant were collected in Epis.
Incubation for 30 min at 65 °C.
Centrifugation for 1 min at 1250 g.
100 µl of SEAP buffer were filled in the wells of a 96 well plate.
Addition of 80 µl supernatant to each well.
Addition of 20 µl pNPP (substrate).
96 plate was immediately placed in the blade reader.
Spectroscopic measurement every minute for 150 times; wavelenght 405 nm.
Measurement was aborted, as the SEAP concentration was too high.
10.08.13
Repeat of SEAP measurement
Cell supernatant was diluted 1:10 with DMEM that was heated for 30 min at 65 °C and measurement was repeated.
The results indicate that also the functionality of the Cas9-ERT2 fusion protein is not controlable with hormone induction, as the SEAP expression is more or less the same for wells with and without 4-OHT.
Retrieved from " http://2013.igem.org/Team:Freiburg/Notebook/lab_hormon "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
