<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Notebook_method"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Notebook/method">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Notebook/method&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Notebook/method&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Notebook/method&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Notebook/method" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Notebook/method</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3></DIV><DIV id="menu"><UL id="navbar"><LI style="padding-left:15px"><A href="https://2013.igem.org/Team:Freiburg">Home</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Highlights">Highlights</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Introduction </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/induction">Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></LI></UL><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna">Targeting</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/method">uniBAss</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation">Standardization</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/protocols">Material and Methods</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/BioBricks">Our BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/favorite_parts">Favorite BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/improvement">Improvement of a Registry part</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/sharing">Sharing of our BioBricks </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/experts">Expert opinions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety/safety">Safety &amp; Security</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/politics">Politics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/society">Society</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/education">Education</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/art">Art</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/Facebook">Facebook / Media</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/News">News</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team2">Members</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Exolab">Exolab-Events</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Statistics">Statistics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/gallery">Photo Gallery</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Attributions">Our Partners</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/Coop">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/attributions"> Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/thanks">Thanks to</A></LI></DIV><DIV id="sidebar"><DIV id="sidebar_content"><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna"> Targeting </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector"> Effectors </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling"> Modeling </A></P><P class="first_order"><A class="active" href="https://2013.igem.org/Team:Freiburg/Notebook/method"> uniBAss </A></P><P class="third_order"><A href="#may"> May </A></P><P class="third_order"><A href="#june"> June </A></P><P class="third_order"><A href="#july"> July </A></P><P class="third_order"><A href="#august"> August </A></P><P class="third_order"><A href="#september"> September </A></P><P class="third_order"><A href="#october"> October </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation"> Standardization </A></P><P class="first_order"><A id="link" href="https://2013.igem.org/Team:Freiburg/protocols"> Material and Methods </A></P></DIV></DIV><DIV id="main_contant_note"><DIV id="h1">
uniBAss - uniCAS Binding Assay Notebooks
</DIV><DIV id="may"><P id="h2">
May
</P></DIV><DIV id="tag"><H2> 15.05.13 </H2><H3>  Digest with BbsI </H3><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 8 </TD><TD> pX334a (243ng/µl)</TD></TR><TR><TD> 2µl </TD><TD> NEB-Buffer 2.1 </TD></TR><TR><TD> 1µl </TD><TD> BbsI </TD></TR><TR><TD> Add to 50µl </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><P> pX334a was digested for 2h at 37°C using BbsI for subsequent target insertion.
  </P><H3> Gel extraction of the digested pX334a </H3><P> Bands were cut out and extracted with Roche high pure PCR product purification kit. Eluation in 20µl H<SUB>2</SUB>O, before centrifugation: incubated 10min at room temp. Eluted liquid put on column again, incubated for 10 min and centrifuged.
concentrations: less than 16.5ng/µl. 
  </P><H3>  Ligation of digested pX334a with annealed oIG9007-oIG9008  (EMX1 locus) </H3><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 2.5 </TD><TD> pX334a (16.5 ng/µl)</TD></TR><TR><TD> 1µl </TD><TD> T4 ligase buffer </TD></TR><TR><TD> 0.5µl </TD><TD> T4 ligase </TD></TR></TBODY></TABLE></DIV><P> Five different ligation mixes (dilution row of insert, 1x, 10x 100x ,500x, 1000x diluted insert) + the negative control with water instead of oligo were incubated for 1h at 22°C. This resulted in pIG9000.
  </P><H3>  Transformation of Top10 <I>E. coli</I> with the pX334a containing the EMX1 locus (pIG9000) </H3><P> The 6 different samples were transformed following the protocol and were thus given on an agar plate with Ampicilline and incubated over night. </P></DIV><DIV id="tag"><H2> 16.05.13 </H2><H3> Picking colonies and streakout </H3><P> The negative control showed five colonies whereas the plate with 1x duluted insert had 12 colonies, 10x had 17 colonies, 100x had 17 colonies, 500 had 10 colonies, 1000x hat 18 colonies. From each of the two plates with the most colonies were 8 colonies picked. 16ten streakouts were performed on AMP agar plates and incubated over night at 37°C.  </P></DIV><DIV id="tag"><H2> 17.05.13 </H2><H3> Miniprep and testdigest </H3><P> The miniprep was performed as stated in the protocol. Afterwards a testdigest was performed. </P><H3>  Testdigest of 16. possible pIG9000 colonies, mastermix contained </H3><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> total 40µl </TD><TD> Sample</TD></TR><TR><TD> 4µl </TD><TD> BbsI </TD></TR><TR><TD> 4µl </TD><TD> NcoI </TD></TR><TR><TD> 20µl </TD><TD> NEB 2.1 </TD></TR><TR><TD> 132µl </TD><TD> H2O </TD></TR></TBODY></TABLE></DIV><P> After 2h at 37°C the samples were applied on an agarose gel (1% Agar, 0.5 TAE to 50 ml) to visualize band 3 ul gel red was used. The gel displayed that some colonies showed no additional band and therefore are likely to have the insert included. Those positive minipreps were send to sequencing at GATC.
</P><P> The sequencing results validated that the insert was successfully cloned into pX334a and therefore pIG9000 was ready for the testing on the ELISA. 
</P></DIV><DIV id="tag"><H2> 18.05.13 </H2><H3> Oligoannealing and transformation of <I>E. coli</I> with pIG9000 with the aim the perform a midi prep </H3><P> The oligoannealing with the biotinylated oligonucleotides oIG9000, oIG9001 and oIG9002 was performed at 95°C for 5 minutes, afterwards the heating block was turned off. The transformation was performed following the protocol. The transformed cells were given into 150ml LB-media containing AMP and were incubated overnight at 37°C.</P></DIV><DIV id="tag"><H2> 19.05.13 </H2><H3> Performing a midiprep </H3><P> A midiprep of the <I>E. coli</I> containing pIG9000 was performed following the protocol. Result: 2487ng/µl in a volume of 200µl. </P></DIV><DIV id="tag"><H2> 20.05.13 </H2><H3> Seeding cells for transfection </H3><P> HEK-293T cells were seeded at a density of 250,000 cells/ml. The conservation culture contained 60,000 cells/ml. </P><P> Human embryonic kidney cells (HEK-293T) were cultivated in Dulbecco's modified Eagle's medium (DMEM), supplemented with 10 % Fetal Calf Serum (FCS) and 1 % penicillin / streptomycin solution (DMEM complete) and incubated in a moisturized atmosphere containing 5 % CO2. Cells were detached with 2.4 ml Trypsin-EDTA solution, transferred into 7 ml fresh medium and centrifuged for 3 min at 900 x g. The supernatant was removed and the pelleted cells were resuspended in 5 - 7 ml medium. Cell numbers were determined by transferring 100µl cell solution to 10 ml of Casyton Buffer and counted in an Innovatis Casy Cell counter (Casy-Technology, Reutlingen, Germany, model TT). For maintaining cultures, HEK-293T cells were cultivated in petridishes with a starting concentration of 0.6 or 1.2*106 cells in 10 ml medium and incubated as described before.</P></DIV><DIV id="tag"><H2> 21.05.13 </H2><H3> Contamination &amp; seeding cells </H3><P> Contamination occured therefore the cells had to be seeded again following the protocol.</P></DIV><DIV id="tag"><H2> 23.05.13 </H2><H3> PEI Transfection and preparation of uniBAss experiment </H3><P> Seeded cells were transfected with pIG9000 following the protocol. 
Cells were transfected using the PEI transfection method with the following concentrations: Typically for a 200µl transfection mix, 1.5µg of DNA and 5µl PEI (Polysciences, Inc., PA, cat. no. 23966) were filled up with OptiMEM (Invitrogen GmbH, Darmstadt, Germany cat. no. 11058-021) to 100µl. Afterwards, the 100µl PEI solution was added to the 100µl DNA solution, vortexed and incubated at RT for 15 to 30 min. Afterwards 200µl of the PEI / DNA transfection mix was added drop-wise to each 1 ml culture medium containing cells in well plates and incubated at 37°C, 5 % CO2. After 5 h, the medium was exchanged , Dulbecco's modified Eagle's medium (DMEM), supplemented with 10 % Fetal Calf Serum (FCS) and 1 % penicillin / streptomycin solution (DMEM complete) </P><P> The ELISA plates (Corning, Inc., cat. no. 3590, NY, New York) were coated with 100µl streptavidin solution (20µl / 10 ml in dH<SUB>2</SUB>0 to obtain a final concentration of 4µg/ml) and incubated overnight at 4°C. 
</P></DIV><DIV id="tag"><DIV id="firstuniBAss"><H2> 24.05.13 </H2><H3> uniBAss experiment to observe the influence of different Ion concentrations </H3></DIV><P> The streptavidin coated plates were washed 3 x with TBST (TBS, 2 ml / l Tween20) and 300µl blocking buffer (1 x TBST, 1 % BSA) was added for 1 h. After 3 x washing with TBST, the biotinylated oligonucleotides (10pmol/well) were applied and incubated at RT for 1 h. Afterwards the plates were washed 3 x with TBST. 
To obtain the cell lysate with the Cas9 protein, the transfected HEK-293T cells (250,000 cells / ml) were resuspended in 250µl dilution buffer (10 mM Tris, 1 % BSA, 10 mM MgCl2, 10 mM NaCl) containing EDTA free Protease inhibitor per 125,000 cells / ml and sonified for 10 min. To remove the cell fractions the lysate was centrifuged for 5 min at 500xg.
</P><P> To adjust the buffer condition a dilution row with cell lysate comprising the Cas9 were performed in round 96-well plates. The dilution buffers used were b1 (10 mM Tris, 1 % BSA, 10 mM MgCl2, 250 mM NaCl) and b2 (10 mM Tris, 1 % BSA, 100 mM MgCl2, 10 mM NaCl) respectively. 
This resulted in the following conditions.
</P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> MgCl2 [mM] </TH><TH> NaCl [mM]</TH></TR><TR><TD> 10 </TD><TD> 250 </TD></TR><TR><TD> 10 </TD><TD> 125 </TD></TR><TR><TD> 10 </TD><TD> 62.5 </TD></TR><TR><TD> 10</TD><TD> 31.25 </TD></TR><TR><TD> 10 </TD><TD> 15.62 </TD></TR><TR><TD> 10 </TD><TD> 7.81 </TD></TR><TR><TD> 10 </TD><TD> 3.9 </TD></TR><TR><TD> 10 </TD><TD> 1.95 </TD></TR><TR><TD> 10 </TD><TD> 0.97 </TD></TR><TR><TD> 10 </TD><TD> 0.48 </TD></TR></TBODY></TABLE></DIV><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> NaCl [mM] </TH><TH> MgCl2 [mM]</TH></TR><TR><TD> 10 </TD><TD> 100</TD></TR><TR><TD> 10 </TD><TD> 50 </TD></TR><TR><TD> 10 </TD><TD> 25 </TD></TR><TR><TD> 10</TD><TD> 12.5 </TD></TR><TR><TD> 10 </TD><TD> 6.25 </TD></TR><TR><TD> 10 </TD><TD> 3.12 </TD></TR><TR><TD> 10 </TD><TD> 1.56 </TD></TR><TR><TD> 10 </TD><TD> 0.89 </TD></TR><TR><TD> 10 </TD><TD> 0.39 </TD></TR><TR><TD> 10 </TD><TD> 0.19 </TD></TR></TBODY></TABLE></DIV><P> Afterwards 100µl of the diluted cell lysate was transferred to the ELISA plates. After 1 h incubation at RT the plates were washed 3 x with TBST and 100µl anti-HA antibody solution (1000 x diluted in blocking buffer) was applied to each wells and incubated for 1 h. After 3 x washing with TBST, 100µl anti-mouse HRP antibody (4000 x diluted in blocking buffer,) were added. After 1 h incubation, plates were washed 3 x with TBST, probed with 100µl ELISA ABTS substrate and the absorbance was measured at 405 nm.
For the positive controls, wells were coated with 100µl FM protein (1.0 µg / ml) and directly with the sonificated cell lysate in the dilution buffer.
The negative controls performed were composed of, no Antibody, no oligonucleotide at all, not the matching oligonucleotide sequence.  
</P><P> in the following this protocol was always used only variations are mentioned. </P><P> Result: Figure 1 </P><DIV><TABLE class="imgtxt"><TBODY><TR><TD> Binding restrain of the Cas9 MX1 crRNA complex for bisected buffer </TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt"><TBODY><TR><TD> Binding restrain of the Cas9 MX1 crRNA complex for bisected buffer  </TD></TR></TBODY></TABLE></DIV><P> To check if the Cas9 protein was in the cell lysate a western blot was performed </P><P>For specific detection of proteins, semi-dry Western Blots were performed. The polyvinylidene fluoride membrane (PVDF) (0.45 µm, Millipore Corporation, Bedford, MA, cat. no. T831.1) was first activated in methanol then washed with dH<SUB>2</SUB>O and subsequently equilibrated in the transfer buffer. For Semi-dry transfer of proteins, a sandwich of Whatman paper / gel / membrane / Whatman paper (wetted with the transfer buffer) is placed directly between cathode and anode in a blotting chamber (Peqlab, Biotechnologie GmbH, Erlangen, Germany cat. no. 52-2020). The blot was performed at 450 mA for 1 h. To prevent unspecific binding of antibodies, the membrane was incubated in block
ing solution (2 % milk in PBST) at room temperature for 1 h at shaking conditions. Afterwards, the membrane was incubated in PBST 1 % milk supplemented with the primary antibody at room temperature for 1 h. Followed by washing the membrane 3 x with PBST for 15 min, the membrane was incubated with PBST 1 % milk containing a horse radish peroxidase (HRP) conjugated secondary antibody at room temperature for 1 h. After 3 x washing with PBST for 10 - 15 min, the blot was incubated with 1 ml Western Blotting Detection Solution and detection of the chemiluminescence, emanating from the HRP-mediated conversion of the substrate was performed in an image reader .
</P></DIV><DIV id="june"><P id="h2">
June
</P></DIV><DIV id="tag"><H2> 05.06.13 </H2><H3> Seeding cells for PEI transfection </H3><P> The seeding of the cell was done as mentioned above. 
</P></DIV><DIV id="tag"><H2> 06.06.13 </H2><H3> PEI transfection and coating of ELISA plate </H3><P> The seeding of the cell was done as mentioned above </P></DIV><DIV id="tag"><H2> 07.06.13 </H2><H3> Testing the best oligonucleotide length for Cas9/crRNA binding </H3><P> The three different biotinylated and annealed oligonucleotides oIG9000 / oIG9001, oIG9002 / oIG9003 and oIG9004 / oIG9005 (10 pmol / well) were coated by incubation at RT for 1 h. The transfected HEK-293T cells (250,000 cells / ml) were harvested by sonification for 10 minutes in the dilution buffer (10 mM Tris, 1 % BSA, 125 mM NaCl, 10 mM MgCl2) and (10 mM Tris, 1 % BSA, 50 mM MgCl2, 10 mM NaCl) containing one tablet of the EDTA free Protease inhibitor cocktail per 50 ml buffer. The Buffer dilution comprising the Cas9 were performed in round 96-well plates by a 1:1.
</P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> NaCl [mM] </TH><TH> MgCl2 [mM]</TH></TR><TR><TD> 125 </TD><TD> 50 </TD></TR><TR><TD> 125 </TD><TD> 25 </TD></TR><TR><TD> 125 </TD><TD> 12.5 </TD></TR><TR><TD> 125</TD><TD> 6.25 </TD></TR><TR><TD> 125 </TD><TD> 10 </TD></TR><TR><TD> 10 </TD><TD> 50 </TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt"><TBODY><TR><TD><B>The results show that a total length of 40 nucleotides is not enough for the Cas9 / RNA to bind. 50 nucleotides are sufficient for binding however 60 nucleotides show the most robust binding behaviour.</B></TD></TR></TBODY></TABLE></DIV><DIV id="tag"><H2> 12.06.13 </H2><H3> Seeding cells for PEI transfection </H3><P> The seeding of the cell was done as mentioned above. </P></DIV><DIV id="tag"><H2> 13.06.13 </H2><H3> PEI transfection and coating of ELISA plate </H3><P> The PEI transfection and coating of the ELISA plate was done as mentioned above. </P></DIV><DIV id="tag"><H2> 14.06.13 </H2><H3> Optimizing the buffer condition </H3><P> Different concentrations of MgCl2 – 30 mM, 20 mM, 10 mM and 5 mM – and of NaCl – 150 mM, 120 mM, 90 mM and 60 mM, were varied against each other. 
</P><P> 
The ELISA was performed as explained above. 
Dilution buffers containing (10 mM Tris, 1 % BSA, NaCl – 150 mM, 120 mM, 90 mM and 60 mM, MgCl2 – 30 mM, 20 mM, 10 mM and 5 mM respectivly) containing one tablet EDTA free Protease inhibitor cocktail per 50 ml were used to uptake the transfected HEK–293T cells and subsequently perform cell lysis.
</P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> NaCl [mM] </TH><TH> MgCl2 [mM]</TH></TR><TR><TD> 150 </TD><TD> 30</TD></TR><TR><TD> 150 </TD><TD> 20</TD></TR><TR><TD> 150 </TD><TD> 10</TD></TR><TR><TD> 150 </TD><TD> 5</TD></TR></TBODY></TABLE></DIV><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> NaCl [mM] </TH><TH> MgCl2 [mM]</TH></TR><TR><TD> 120 </TD><TD> 30</TD></TR><TR><TD> 120 </TD><TD> 20</TD></TR><TR><TD> 120 </TD><TD> 10</TD></TR><TR><TD> 120 </TD><TD> 5</TD></TR></TBODY></TABLE></DIV><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> NaCl [mM] </TH><TH> MgCl2 [mM]</TH></TR><TR><TD> 90 </TD><TD> 30</TD></TR><TR><TD> 90 </TD><TD> 20</TD></TR><TR><TD> 90 </TD><TD> 10</TD></TR><TR><TD> 90 </TD><TD> 5</TD></TR></TBODY></TABLE></DIV><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> NaCl [mM] </TH><TH> MgCl2 [mM]</TH></TR><TR><TD> 60 </TD><TD> 30</TD></TR><TR><TD> 60 </TD><TD> 20</TD></TR><TR><TD> 60 </TD><TD> 10</TD></TR><TR><TD> 60 </TD><TD> 5</TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt"><TBODY><TR><TD><B>The figure shows that the binding behaviour of the Cas9 / crRNA towards EMX1 display similar behaviour for descending MgCl2. This is true for all four NaCl concentrations however 150 mM NaCl has the smallest yield.</B></TD></TR></TBODY></TABLE></DIV><DIV id="tag"><H2> 17.06.13 </H2><H3> Seeding cells for PEI transfection </H3><P> The seeding of the cell was done as mentioned above. </P></DIV><DIV id="tag"><H2> 18.06.13 </H2><H3> PEI transfection and coating of ELISA plate </H3><P> The PEI transfection was performed with the corresponding plasmids and coating of the ELISA plate was done as mentioned above. </P></DIV><DIV id="tag"><H2> 19.06.13 </H2><H3> Testing of dCas9 – VP16 fusion proteins in different buffer conditions </H3><P> The ELISA was performed as mentioned above. The transfected HEK–293T cells were harvested using the dilution buffers with varying NaCl and varying MgCl2 contrations (10 mM Tris, 1 % BSA, 150 mM, 120 mM, 90 mM or 60 mM NaCl and 20 mM, 10 mM or 5 mM MgCl2) containing EDTA free Protease inhibitor.
</P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> NaCl [mM] </TH><TH> MgCl2 [mM]</TH></TR><TR><TD> 150 </TD><TD> 5 </TD></TR><TR><TD> 120 </TD><TD> 20</TD></TR><TR><TD> 120 </TD><TD> 10</TD></TR><TR><TD> 120 </TD><TD> 5</TD></TR><TR><TD> 90 </TD><TD> 5</TD></TR><TR><TD> 60 </TD><TD> 5</TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt"><TBODY><TR><TD> Three different VEGF crRNA targets of Cas9 - VP16. The binding affinity of the Cas9 – crRNA complex was analysed during an ELISA. The figure shows that for different crRNAs the binding affinities differs. The best binding condition was obtained with 5 mM MgCl2 and 90 mM NaCl. The error bars represent the standard deviation of 3 replicas.</TD></TR></TBODY></TABLE></DIV><DIV id="tag"><H2> 23.06.13 </H2><H3> Seeding cells for PEI transfection </H3><P> The seeding of the cell was done as mentioned above. </P></DIV><DIV id="tag"><H2> 24.06.13 </H2><H3> PEI transfection and coating of ELISA plate </H3><P> The PEI transfection was performed with the corresponding plasmids and coating of the ELISA plate was done as mentioned above. </P></DIV><DIV id="tag"><H2> 25.06.13 </H2><H3> Testing of different dCas9 fusion proteins in the optimized buffer condition.  </H3><P> The tested constructs can be divided into two groups. The Cas9 fusion constructs Cas9 - VP16, Cas9 - KRAB and the Cas9 with one active nickase domain and with two nickase domains. The composition of this experiment aimed to examine whether all constructs are expressed in the same pace and display the same binding behavior.
</P><P> The ELISA was performed as explained. The ELISA indicated that the fusion constructs are 3 - fold less expressed than the constructs with the Cas9 only. The comparison between a Cas9 completely lacking the nickase activity and a Cas9 having one active nickase shows no significant difference. However the yield of the Cas9 construct without any nicking activity is slightly higher.
A Bradford assay was used to adjust the FM protein concentration for the standard. The FM protein was coated in a 1 : 1 dilution row for 1 h at RT in duplicates onto the ELISA plate. The transfected HEK–293T cells were harvested using the optimized dilution buffer (10 mM Tris, 1 % BSA, 90 mM NaCl, 5 mM MgCl2) containing EDTA free Protease inhibitor. 
</P><DIV><TABLE class="imgtxt"><TBODY><TR><TD> The four different biotinylated and annealed oligonucleotides (oIG9004 / oIG9005, oIG9012 / oIG9013, oIG9041 / oIG9042 and oIG9015 / oIG9016 (10 pmol / well) were coated by incubation at RT for 1 h. After 24 h transfected HEK-293T cells were resuspended in the optimized buffer condition and the binding affinity of the different Cas9 / crRNA construct was analysed during an ELISA. A quantification of the different Cas9 constructs was performed by using a standard dilution row with FM protein. a) Concentration of the undiluted Cas9 constructs. The figure shows the different quantities of different Cas9 constructs as well as the disparity of undiluted sample compared to diluted samples. Error bars represent the standard deviation of 3 replicas. </TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt"><TBODY><TR><TD> The four different biotinylated and annealed oligonucleotides (oIG9004 / oIG9005, oIG9012 / oIG9013, oIG9041 / oIG9042 and oIG9015 / oIG9016 (10 pmol / well) were coated by incubation at RT for 1 h. After 24 h transfected HEK-293T cells were resuspended in the optimized buffer condition and the binding affinity of the different Cas9 / crRNA construct was analysed during an ELISA. A quantification of the different Cas9 constructs was performed by using a standard dilution row with FM protein. Concentration of the 1:1 diluted Cas9 constructs. The figure shows the different quantities of different Cas9 constructs as well as the disparity of undiluted sample compared to diluted samples. Error bars represent the standard deviation of 3 replicas. </TD></TR></TBODY></TABLE></DIV><DIV id="standardcurve"><P id="h4">
Quantification of Cas9 via HA-tagged FM protein
</P></DIV><P>
to quantify the amount of Cas9 with uniBAss HA-tagged FM protein with known protein concentration. like this  we were abele to performe a dilution row to get a standard to convert the ABTS readout in ng protein bound in the ELISA. 
</P><DIV><TABLE class="imgtxt"><TBODY><TR><TD> graph showing the standard curve for quantification of HA-tagged protein in the uniBAss ELISA</TD></TR></TBODY></TABLE></DIV><P>
To convert the ABTS readout of uniBAss with our Cas9 and dCas9 the formula y = 0,0793x + 0,0255 was used. Like this we succeded in receiving absolut ng of dCas9 with our ELISA. 
</P><DIV><TABLE class="imgtxt"><TBODY><TR><TD> Quantification of different Cas9 and dCas9 constructs</TD></TR></TBODY></TABLE></DIV><DIV id="july"><P id="h2">
July
</P></DIV><DIV id="tag"><H2> 23.07.13 </H2><H3> PCR Cas9 </H3><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> Q5-HF Reaction Buffer </TD></TR><TR><TD> 1 </TD><TD> Template: pIG2004 </TD></TR><TR><TD> 1 </TD><TD> oIG9100 </TD></TR><TR><TD>1</TD><TD> oIG9101 </TD></TR><TR><TD> 4 </TD><TD> dNTPs </TD></TR><TR><TD> 1 </TD><TD> DMSO </TD></TR><TR><TD> 0,5 </TD><TD> Q5-HF Polymerase </TD></TR><TR><TD> Add to 50 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI>  Annealing: 60°C </LI><LI>  Extension: 2min20sec </LI><LI> 18 cycles </LI></UL></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> Roth 1kb ladder - Cas9-PCR a - Cas9-PCR b - 2log ladder NEB 
         There is a second (unspecific?) band in b.</TD></TR></TBODY></TABLE></DIV><H3> Gel extraction of PCR Cas9 </H3><P>Bands a and b were cut out and extracted separately with Roche high pure PCR product purification kit. Eluation in 20 µl H<SUB>2</SUB>O, before centrifugation: incubated 10min at room temp. Eluted liquid put on column again, incubated for 10 min and centrifuged.
concentrations: less than 5ng, no digest performed. -&gt; repeat PCR tomorrow. 
  </P></DIV><DIV id="tag"><H2> 24.07.13 </H2><H3> PCR Cas9 </H3><P> PCR was repeated with same program but 25 cycles instead of 18. This time no unspecific bands were visible (picture could not be saved). Both bands were cut out and extracted over same column with Roche Kit. 
        Concentration: 65 ng/µl</P><H3>  Digest with SacII and KpnI-HF </H3><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 10 </TD><TD> pIG6000 (170ng/µl=1.7µg)</TD></TR><TR><TD> 2µl </TD><TD> NEB-Buffer Cut-Smart </TD></TR><TR><TD> 1µl </TD><TD> SacII </TD></TR><TR><TD> 1µl </TD><TD> KpnI-HF </TD></TR><TR><TD> Add to 20 µl </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 25 </TD><TD> Cas9-PCR (65 ng/µl=1,625µg)</TD></TR><TR><TD> 5µl </TD><TD> NEB-Buffer Cut-Smart </TD></TR><TR><TD> 1µl </TD><TD> SacII </TD></TR><TR><TD> 1µl </TD><TD> KpnI-HF </TD></TR><TR><TD> Add to 50 µl </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatleft"><UL><LI> Temp.: 37°C </LI><LI> Incubation time: 1.5h </LI></UL></DIV><P> 0.7% agarose gel, upper bands cut out, digest worked. </P><DIV><TABLE class="gelpic"><TBODY><TR><TD> Roth ladder - Cas9-KRAB (4826bp) - pIG6000 (upper band 2721bp, insert 1064bp) </TD></TR></TBODY></TABLE></DIV><H3>  Gel extraction of digested fragments </H3><P> Cas9: 16.2ng/µl 
dIG6000: 17.5ng/µl 
frozen at -20°C
  </P></DIV><DIV id="tag"><H2> 25.07.13 </H2><H3> Ligation of digested Cas9-PCR product and dIG6000 </H3><H4> Calculation: </H4><DIV id="floatleft"> Cas9-PCR = backbone (4826bp) dIG6000 = insert (2721bp) </DIV><DIV id="floatright"><P> (30ng backbone/bp backbone) x bp insert x 3 = ng insert 
      30ng Cas9-PCR/4826 x 2721 x 3 = 50.7ng dIG6000</P></DIV><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> amount </TH><TH> substance </TH></TR><TR><TD> 30ng = 1.85µl </TD><TD> Cas9-PCR (16.2ng/µl)</TD></TR><TR><TD> 50.7ng = 2.89µl </TD><TD> dIG6000 (17.5ng/µl) </TD></TR><TR><TD> 2µl </TD><TD> T4 ligase buffer (Fermentas) </TD></TR><TR><TD> 1µl </TD><TD> T4 Ligase (Fermentas) </TD></TR><TR><TD> Add to 10 µl </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><P> 30min at 22°C </P><H3> Transformation </H3><P> 2.5µl of ligation used. &gt;1h on 37°C shaker. 
plate 50µl, centrifuge &amp; discard supernatant, plate rest.

2 a.m.: colonies picked for mini prepping.</P></DIV><DIV id="tag"><H2> 26.07.13 </H2><H3> Miniprep of colonies 1-4 </H3><LI> with Roche kit. </LI><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH> Mini </TH><TD> 1 </TD><TD> 2 </TD><TD> 3 </TD><TD> 4 </TD></TR><TR><TH> concentration in ng/µl </TH><TD> 77.7 </TD><TD> 13.9 </TD><TD> 65 </TD><TD> 45 </TD></TR></TBODY></TABLE></DIV><H3>  Testdigest of Minis with SacII and KpnI </H3><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> about 200 ng </TD><TD> DNA </TD></TR><TR><TD> 1 </TD><TD> NEB-Buffer Cut Smart </TD></TR><TR><TD> 0.25 </TD><TD> SacII (undiluted) </TD></TR><TR><TD> 0.5 </TD><TD> KpnI-HF </TD></TR><TR><TD> Add to 10 µl </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Temp.: 37°C </LI><LI> Incubation time: 1-2h </LI><LI> expected fragments: </LI><OL><LI> 4826bp for Cas9-KRAB-bgh </LI><LI> 2722bp for pSB1C3-CMV (dIG6000) </LI></OL></UL></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> Roth marker, mini1, 2, 3, 4, pIG6000 
 Cas9-KRAB (4826bp) pIG6000 (upper band 2721bp, insert 1064bp) 
in mini2 slight Cas9 band is visible (marked by star). </TD></TR></TBODY></TABLE></DIV><P><B> Concentration too low to send for sequencing, try PCR with truncation primers instead. If Cas9 is there, PCR should work.</B></P></DIV><DIV id="tag"><H2> 27.07.13 </H2><H3> Test-PCR on Mini2 for Truncation 2 </H3><P> fw: oIG9032 rev: oIG9033 
     fragment size: 7250bp </P><DIV id="floatleft"><UL><LI>  Annealing: 63°C </LI><LI>  Extension: 4min </LI><LI> 25 cycles </LI></UL></DIV><H3> Mini preps of ligation colonies 5-7 and 2&amp;4 again to yield higher concentrations </H3><DIV><TABLE class="tabelle"><TBODY><TR><TH> Mini </TH><TD> 2 </TD><TD> 4 </TD><TD> 5 </TD><TD> 6 </TD><TD> 7 </TD></TR><TR><TH> concentration in ng/µl </TH><TD> 193.1 </TD><TD> 181.9 </TD><TD> 230.8 </TD><TD> 171.8 </TD><TD> 215.8 </TD></TR></TBODY></TABLE></DIV><H3> Test digest of Minis with SacII and KpnI </H3><P> For protocol see 26.7.13. 1µl of each Mini prep was used. </P><H3> 0.7% Agarose gel of PCR and Test Digest </H3><DIV><TABLE class="gelpic"><TBODY><TR><TD> up: Roth 1kb ladder - Test PCR1 - Test PCR2
down: Roth 1kb ladder - Mini2 - 4 - 5 - 6 - 7  
4, 5 and 7 seem to be positive! </TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> Roth 1kb ladder - Mini2 - 4 - 5 - 6 - 7  
Mini 5 is not completely digested, probably because of high concentration (230ng/µl!) </TD></TR></TBODY></TABLE></DIV><P> Minis 5 and 7 contain Cas9-KRAB and can be used as truncation PCR templates. </P></DIV><DIV id="tag"><H2> 29.07.13 </H2><H3> Truncation PCRs </H3><P> annealing: 64°C 
elongation: Truncation 1&amp;2 4min, Truncation 3,4 &amp; 5 3:30min </P><DIV><TABLE class="gelpic"><TBODY><TR><TD> Roth 1kb ladder - Truncation1 - 2 - 3 - 4 - 5 </TD></TR></TBODY></TABLE></DIV><H3> Gibson of truncated Cas9 plasmids </H3><H3> Trafo of Gibson </H3></DIV><DIV id="tag"><H2> 30.07.13 </H2><P> Truncation 3 and 4 yielded 10-15 colonies each. No colonies on other plates. 8 			colonies from each plate were picked and streaked out for mini preps. 
	</P><H3> Truncation PCR 1, 2 and 5 repeated as on 29.7. </H3><DIV><TABLE class="gelpic"><TBODY><TR><TD> Roth 1kb ladder - Truncation PCR 1 - 1 - 2 - 2 </TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> Roth 1kb ladder - Truncation PCR 5 - 5 - BbsI digested pIG3010 </TD></TR></TBODY></TABLE></DIV><H3> BbsI digest of RNA plasmid (from Hormon group) </H3><DIV><TABLE class="gelpic"><TBODY><TR><TD> Roth 1kb ladder - Truncation PCR 5 - 5 - BbsI digested pIG3010 </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 31.07.13 </H2><H3> Mini preps and test digest of truncation 3 and 4 </H3><P> Minis: &gt;200ng/µl, 1µl was digested with EcoRI and SpeI (both HF) for 2h at 37°C. </P><DIV><TABLE class="gelpic"><TBODY><TR><TD> Roth 1kb ladder - MiniT3 1-2-3-4-5-6-7-8- -pIG9100 
Roth 1kb ladder - MiniT4 1-2-3-4-5-6-7-8- - gelex PCR1-2-5- Roth ladder </TD></TR></TBODY></TABLE></DIV><H3> Gel Ex of truncation PCR 1, 2 and 5 </H3><P> 9-15ng/µl, test size on gel. </P><DIV><TABLE class="gelpic"><TBODY><TR><TD> gelex PCR1-2-5- Roth ladder </TD></TR></TBODY></TABLE></DIV><H3> Gibson of truncation 1, 2 and 5 </H3><P> 5µl of gel extracted PCR product were added to aliquoted Gibson master mix, 			1h 50°C. 300µl were plated on chloramphenicol plates.
  </P><H3> Testdigest </H3><P> as the sequenzing of T3 showed unclear results we did minipreps of 4 more colonies from the gibson plate and digested them with EcoRI and SpeI to yield another positive testdigest that could be sequenzed.
  </P><H3> RNA-Plasmid </H3><H4> Gel ex of BbsI digested plasmid </H4><H4> Oligo annealing of EGFP oligos </H4><H4> Ligation with EmxI and EGFP crRNA target oligos </H4></DIV><DIV id="august"><P id="h2">
August
</P></DIV><DIV id="tag"><H2> 06.08.13 </H2><H3> Seeding cells for transfection </H3><P> 120,000cells/ml were seeded in six well plates (3ml per well)
	</P><H3> Sequencing of RNA Plasmids </H3><P> Both EGFP and EMX1 were inserted correctly (enter clone number)
	</P></DIV><DIV id="tag"><H2> 07.08.13 </H2><H3> Inoculate Midis for T1, 2, 5, pIG9100(Cas9), pIG3010(EMX1, EGFP) </H3><P> 100ml LB + 100µl Chloramphenicol. For T2 four additional minis were inoculated for midi and sent for overnight sequencing.
	</P><H3> Truncation 4 </H3><P> Due to 200 missing base pairs more colonies could be screen from T4 plate: 9 clones streaked out for mini prepping (edit 8.8 plates put in 4°C bacterial fridge, prep when suitable).
	</P></DIV><DIV id="tag"><H2> 08.08.13 </H2><H3> Midipreps of plasmids </H3><P> All plasmids were prepped because sequencing was delayed. Concentrations: ? 
	</P><H3> Transfection </H3><P> Medium change was performed 7 hours before transfection. 
	</P><P> 3µg DNA were transfected per well (six well plate). Of truncation 2 all midis were transfected because sequencing results were still missing (pIG9102-3, -1, -5, -7 and -8 transfected). Transfection scheme?
	</P><H3> ELISA </H3><P> 96 well plate (flat, high-binding) was coated with streptavidin (100µl per well, 4µg/ml working solution), sealed and incubated over night at 4°C.
	</P><P> 10xTBS was prepared (50mM Tris, 125mM NaCl, pH 7.5) 
	</P></DIV><DIV id="tag"><H2> 09.08.13 </H2><H3> Cell lysis and ELISA (Fenja and Philipp)</H3><P> did not work; repeat everything.
	</P></DIV><DIV id="tag"><H2> 12.08.13 </H2><H3> Cell Seeding </H3><P> aprox. 65,000 cells/ml seeded in 6 well plates. 
  </P><H3> Sequencing Results for Truncation 2 </H3><P> T2.3 is positive
  </P></DIV><DIV id="tag"><H2> 13.08.13 </H2><H3> Transfection </H3><P> Natalie transfected  the truncations (two 6-wells)
  </P><H3> uniBAss II </H3><P> Coating plate with streptavidin
  </P></DIV><DIV id="tag"><H2> 14.08.13 </H2><H3> Cell lysis </H3><P> With dilution buffer (500µl per 6-well) and sonification (10min)
  </P><H3> uniBAss II </H3><P>  Triplicates of the probes were run on the plate but the ELISA did not work. It´s hard to say why we could not detect bound dCas9. Secondary antibody broken? Try again..
  </P></DIV><DIV id="tag"><H2> 15.08.13 </H2><H3> Cell Seeding </H3><P> approx. 65,000 cells/ml seeded in 6 well plates. 
  </P></DIV><DIV id="tag"><H2> 16.08.13 </H2><H3> Transfection </H3><P> PIF-GFP served as a transfection control.
  </P></DIV><DIV id="tag"><H2> 17.08.13 </H2><H3> uniBAss III </H3><P> No dCas9 in pSB1C3 detectable. Test if dCas is expressed: Western Blot with remaining cell lysate against HA-tag.
  </P><DIV><TABLE class="image"><TBODY><TR><TD> Results of uniBAss ELISA experiment</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 18.08.13 </H2><H3> Cell lysis and Blot</H3><P> Done at AG Warscheid. anti-HA antibody overnight.
  </P></DIV><DIV id="tag"><H2> 19.08.13 </H2><H3> Detection of blot </H3><P> Anti-mouse antibody (xh), washing steps, detection with ECL. illumination for 1000sec, only unspecific bands. Repeat western Blot: transfection, lysis, SDS PAGE, blot.
  </P><H3> Cell seeding and transfection </H3><P> HEK-293H from Adrian were seeded and transfected 5h afterwards.
  </P></DIV><DIV id="tag"><H2> 20.08.13 </H2><P> Transfection did not work, the cells died because they did not tolerate the transfection only a few hours after seeding. We therefore seeded new cells.
  </P></DIV><DIV id="tag"><H2> 21.08.13 </H2><H3> Transfection </H3><P> Transfection was performed by Lisa and we let cells grow for 48h! Medium change after 5h. 
  </P></DIV><DIV id="tag"><H2> 22.08.13 </H2><H3> SDS gels </H3><P> For the westernblot tomorrow SDS gels were poured with the invitrogen system. 
  </P></DIV><DIV id="tag"><H2> 23.08.13 </H2><H3> Cell Lysis </H3><P> Cell lysis was performed with modified RIPA buffer (200µl per 6-well), incubated on ice 10min, scraped off with pipet tip, in eppi, on ice for 5-10min, centrifuge 5min, 10000xg.  boil 150µl lysate with 50µl 4x sample buffer (5min, 95°C).
  </P><H3> SDS PAGE </H3><P> The probes were loaded onto the gel.
  </P><H3> Western Blot </H3><P> Semidry western blot was performed and after blocking anti-HA mouse 1:2000 (AG Weber) in TBS with BSA was applied over night.  
  </P></DIV><DIV id="tag"><H2> 24.08.13 </H2><H3> Detection </H3><P> Secondary antibody (1.5µl anti-mouse in 25ml TBS), wash for 2h.. 
  </P><P> Still no dCas9 in pSB1C3! </P></DIV><DIV id="tag"><H2> 29.08.13 </H2><H3> Truncation PCRs with standardized Cas9 (CMV-HA-NLS-Cas9-bGH) </H3><P> All PCRs were done as on 29.07.13 with 4min elongation time and 64 °C annealing temperature. All PCRs worked. 
  </P><DIV><TABLE class="gelpic"><TBODY><TR><TD> Truncation PCRs T1, T2, T3, T4, T5 </TD></TR></TBODY></TABLE></DIV><H3> Gel extraction of PCR products </H3><P> with Roche kit, eluted in 25?µl H<SUB>2</SUB>O. Concentrations between 10-40ng/µl.
  </P><H3> Gibson and transformation of standardized truncations</H3><P> 5 µl of PCR product (gelex) were mixed with Gibson mastermix, 1h 50°C, 3min RT, 3min on ice, trafo with 5 µl. Everything plated.
  </P></DIV><DIV id="tag"><H2> 30.08.13 </H2><H3> Picking colonies and minipreps </H3><P> colonies for all truncations were picked and inoculated for mini-prepping in liquid cultures. Mini preps were not successful because cultures were too thin. more colonies were streaked out on agar plates to grow over night. 
  </P></DIV><DIV id="tag"><H2> 31.08.13 </H2><H3> Mini Preps and test digest of standardized truncations </H3><P> four colonies of each T1, T2, T3, T4, T5 were preped with Roche kit and eluted in 50 µl H<SUB>2</SUB>O. 
  </P><P> Mini preps were test digested with XbaI and PstI-HF in Cut-Smart buffer (10 µl volume, 2h, 37 °C) and separated on a 1% agarose gel.
  </P><DIV><TABLE class="gelpic"><TBODY><TR><TD> 31.08.13 test digest. Log2 marker, T1-1,-2,-3,-4, T2-1, etc. </TD></TR></TBODY></TABLE></DIV><H3> Seeding Cells </H3><P> 2/3 of one 10cm plate were seeded into two 6well plates.
  </P><H3> Midi preps of standarized truncations </H3><P> Midi preps of pIG9200-9205 (chloramphenicol) + pIGCas-mCherry (ampicillin) were inoculated in 100 ml LB+100 µl antibiotic (Mini T1-1, T2-1, T3-1, T4-3, T5-1).
  </P></DIV><DIV id="september"><P id="h2">
September
</P></DIV><DIV id="tag"><H2> 01.09.13 </H2><H3> Midi preps of standardized truncations </H3><P> Plasmids were midi prepped with Promega vacuum &quot;Sau&quot;, 6ml buffer amounts used, columns were not dried from ethanol before eluting. Some columns were clogged and had to be scraped free with steril pipet tips. Plasmids were eluted in 300 µl nuclease free H<SUB>2</SUB>O. Where possible Midis were diluted to 400ng/µl.
  </P><DIV><TABLE class="tabelle"><TBODY><TR><TH> Plasmid </TH><TD> pIG9200 </TD><TD> pIG9201 </TD><TD> pIG9202 </TD><TD> pIG9203 </TD><TD> pIG9204 </TD><TD> pIG9205 </TD><TD> pIGCas9-mCherry </TD></TR><TR><TH> concentration in ng/µl </TH><TD> 400 </TD><TD> 400 </TD><TD> 400 </TD><TD> 400 </TD><TD> 400 </TD><TD> 240 </TD><TD> 400 </TD></TR></TBODY></TABLE></DIV><H3> Transfection of standardized truncations </H3><P> All midi-prepped plasmids were transfected (6well protocol, 3µg total DNA) with pIG9000, PIF-GFP and pIG2004 as controls. This transfection will be used for western blotting to see if the standardized Cas9 is expressed in pSB1C3. 
  </P><DIV><TABLE class="image"><TBODY><TR><TD> Transfection scheme for uniBAss ELISA experiment</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 02.09.13 </H2><H3> Sequencing </H3><P> Midi preps of standarized truncations 1-5 were sent for sequencing with CMV forward primer @ GATC or oIG6018.
  </P><H3> Cells seeded for ELISA </H3><P> 65,000 cells/well were seeded in 6-well plates as transfection will only be on wednesday (04.09.).
  </P></DIV><DIV id="tag"><H2> 03.09.13 </H2><H3> Sequencing results and cell harvesting </H3><P> All sequencing results were useless (sequence length 0nt). Therefore no western blot was done today but cell pellets were collected and frozen. wells transfected with fluorescent proteins were photographed (link pics..). For harvesting cells were washed with cold PBS and then rinsed from well bottom with 500 µl cold PBS and collected in 1.5ml Eppis. Samples were centrifuged 5min at full speed (21000xg), supernatant was decanted and cell pellets were frozen at -20°C.
  </P><P> pIG9200 and mini T1-1 (pIG9201) were sent for sequencing again with CMV-F and BGH-R (both @ GATC).
  </P></DIV><DIV id="tag"><H2> 04.09.13 </H2><H3> Western blot </H3><P> still no sequencing results. pellets blotted anyway. sample buffer, sonifyer, SDS PAGE, semidry blot (not enough transfer buffer?? gels dried out), block in TBST+milk 1h30, anti-HA-mouse 1:2500 in TBST+milk over night. 
  </P><P> GATC is repeating seuquencing of pIG9200 and pIG9201 with oIG6017.
  </P><P> retrafo and new minis (in 1.5ml eppis) of T1-5? Natalie.
  </P><H3> Transfection of pIG9200 truncations with EMX1-RNA Plasmid </H3><DIV><TABLE class="image"><TBODY><TR><TD> Transfection scheme for uniBAss ELISA experiment</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 05-07.09-13 </H2><H3> Sequencing results </H3><P> sequencing did not work because no CMV promoter in construct! finally sequenced with oIG6017: result: SV40(!)-NLS-Cas-BGH, no CMV promoter and no HA-tag! Therefore all our truncation constructs are useless. We have to wait for the standardized CMV-HA-NLS-Cas9-BGH construct. 
  </P></DIV><DIV id="tag"><H2> 09.09.13 </H2><H3> Cloning of CMV-HA-NLS-Cas9-BGH </H3><P> We received HA-NLS-Cas9-BGH from standardization group and have to ligate it with the CMV promoter.
  </P><H4> Mini prepping and test digest </H4><P> some colonies were prepped and test-digested with NgoMIV or NotI.(Natalie) 
  Mini1 (407ng/µl) was used for digest.
  </P><H4> Digestion with EcoRI/XbaI and EcoRI/SpeI </H4><P> HA-NLS-Cas9-BGH is cut open with EcoRI-HF and XbaI, the CMV promoter (pIG0019, 5ng/µl) is cut out with EcoRI-HF and SpeI-HF to then be ligated into the opened Cas9-backbone. 
  </P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 1-2µg (4µl Cas9, 20µl CMV) </TD><TD> DNA </TD></TR><TR><TD> 5 </TD><TD> Cut-Smart NEB buffer </TD></TR><TR><TD> 1 </TD><TD> EcoRI-HF </TD></TR><TR><TD> 1 </TD><TD> XbaI / SpeI-HF </TD></TR><TR><TD> Add to 50µl </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Temp.: 37°C </LI><LI> Incubation time: 2h </LI></UL></DIV><DIV><TABLE class="gelpic"><TBODY><TR><TD> Roth 1kb marker - HA-NLS-Cas9-BGH - CMV 
		    Cas9 backbone has correct size (6466bp), CMV insert (603bp) maybe lies in bromphenol blue band? Cut out and gelex anyway. </TD></TR></TBODY></TABLE></DIV><H4> Gelex </H4><P> both bands were gelexed with Roche Kit and eluted in 20µl H<SUB>2</SUB>O.
  </P><UL><LI> HA-NLS-Cas9-BGH = 38,9ng/µl </LI><LI> CMV = 5ng/µl </LI></UL><H4> Ligation </H4><P> 50ng of Cas9 backbone were ligated with 15ng of CMV insert. Two ligations were set up, with our CMV-digest (N+L) and one of the standardization group (St.)
  </P><DIV id="floatleft"><TABLE class="tabelle"><TBODY><TR><TH>µl </TH><TH> type </TH></TR><TR><TD> 1.5 </TD><TD> Cas9 backbone </TD></TR><TR><TD> 3 </TD><TD> CMV insert (N+L or St.) </TD></TR><TR><TD> 2 </TD><TD> T4 buffer </TD></TR><TR><TD> 1 </TD><TD> T4 Ligase (Fermentas?) </TD></TR><TR><TD> 12.5 </TD><TD> H<SUB>2</SUB>O </TD></TR></TBODY></TABLE></DIV><DIV id="floatright"><UL><LI> Room temperature </LI><LI> Incubation time: 30min </LI></UL></DIV><H4> Trafo </H4><P> 4µl were used for trafo of each ligation
  </P></DIV><DIV id="tag"><H2> 10.09.13 </H2><H3> Minis </H3><P> four colonies of each ligation were picked and inoculated in 1.5ml Eppis with 1ml LB+ 1µl Chloramphenicol at 37°C, 700rpm. Eppis were opened frequently to ensure O<SUB>2</SUB> supply.
  </P></DIV><DIV id="tag"><H2> 11/12.09.13 </H2><H3> Sequencing results </H3><P> no CMV promoter...
  </P></DIV><DIV id="tag"><H2> 12-18.09.13 </H2><H3> New cloning strategy with new CMV promoter </H3><P> Ligations done again to produce SV40-HA-NLS-dCas9-NLS-BGH (pIG0063) and CMV-HA-NLS-dCas9-NLS-BGH (pIG0062). Both used as templates for truncation PCRs. Only CMV constructs shall be tested at first, as we expect higher expression (and thus more easily detectable binding levels in uniBAss) here. 
  </P></DIV><DIV id="tag"><H2> 19-20.09.13 </H2><H3> Sequencing of truncation minis </H3><P> Minis for each truncation with CMV promoter were sent for sequencing, had to be repeated for T1,2,4 and 5 due to frameshifts/extra primer insertions etc.. Wait for results on Monday, but inoculate midis anyway on Sunday. 
  </P></DIV><DIV id="tag"><H2> 22.09.13 </H2><H3> Seeding cells </H3><P> Four 6-well plates were seeded with 130,000cells/ml=260,000cells/well.
  </P><H3> Midis inoculated </H3><P> Midis of T1-T5 were inoculated (except for T3, already done by Michi and Nadine). Gabi :)
  </P></DIV><DIV id="tag"><H2> 23.09.13 </H2><H3> Midis and transfection of CMV-T1-5 </H3><P> Finally sequencing results were correct (T4 was repeated:check tomorrow), inoculated midis were prepped with Promega kit (double buffer amounts) and eluted in 600µl nuclease free H<SUB>2</SUB>O.  
  </P><P> Transfection was performed as on plan in 6-well plates. Additionally all standardized constructs (CMV and SV40) and their non-standardized pX334a-versions were transfected to be compared on uniBAss. The Assay will be performed 48h after transfection. change of media required? check in the morning! 
  </P><DIV id="transfection1234"><TABLE class="image"><TBODY><TR><TD> Transfection scheme for uniBAss ELISA experiment</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="tag"><H2> 24.09.13 </H2><P> No medium change of the cells transfected the day before was performed. SDS-Gel for the westernblot which will be performed tomorrow poured. The ELISA plate for uniBAss is coated with streptadivin. 
	</P></DIV><DIV id="tag"><H2> 25.09.13 </H2><H3> uniBAss ELISA </H3><P> The HEK-293T cells were lysed about 42h after transfection. The uniBAss ELISA was performed according to the protocol. In parallel a westernblot was performed with 2/5 of the lysate of each well to ensure dCas9 expression.
  
  </P><H2> 25.09.13 </H2><H3> Westernblot of uniBAss probes</H3><P> The westernblot of yesterday was analyzed. Due to problems with resolving the protein pellet we had suboptimal results in the blot.</P><P>
 Therefore we want to repeat the hole experiment. As two days before we transfected HEK-293T cells in 6-wells: <A id="link" href="#transfection1234">transfection scheme</A>. This time we made biological duplicates for the standardised constructs. (together six 6-wells)  </P></DIV><DIV id="tag"><H2> 26.09.13 </H2><P> Medium of the six 6 wells was exchanged 12h after transfection. 	  
	</P></DIV><DIV id="tag"><H2> 27.09.13 </H2><H3> Westernblot of 2. uniBAss probes</H3><P> As an expression control we wanted to perform the westernbot of the transfected cells this time before we to the uniBAss ELISA. So we collected the cells in 500µl diluent buffer and took 170µl for western blotting. The 170µl cell suspension was centrifuged for 5min at 10,000xg, the supernatant decanted and the pellet resuspended in 100µl SDS-sample buffer. Then the suspension was sonified for a total of 35min. The probes were cooked for 15min and SDS-page was performed, followed by a semidry westernblot. After blocking, the primary antibody (mouse-anti-HA) was applied over night. </P></DIV><DIV id="tag"><H2> 28.09.13 </H2><P> Development of the westernblot. After washing and applying the secondary antibody we developed the membranes. </P><P> We could detect most of our constructs and had very nice bands. So we want to perform the uniBAss with all our probes tomorrow, therefore we coated an ELISA plate with streptavidin. 
</P></DIV><DIV id="tag"><H2> 29.09.13 </H2><H3> uniBAss ELISA V</H3><P> To perform the uniBAss we had to unfreeze our probes from the -80°C freezer. Then we sonified them and proceeded with our usual uniBAss-protocol.  Except for the constructs containing a CAG promoter all constructs are standardized and in pSB1C3. CMV-GFP serves as a negative control as it contains no dCas9 (and thus no HA-tag), CMV-dCas9 neg contains no RNAimer plasmid and also should not be able to bind the EMX1 oligo. As a control for specificity of the binding we coated one well for each construct without any oligo and an additional well with a VEGF oligo that is not complementary to the EMX1 target site on the RNAimer that was co-transfected with all dCas9-constructs.
</P><P>We received lower ABTS readouts for our dCas9 fusion constructs than for the dCas9 alone. But is due to significant weaker expression of the fusion-constructs. Therefore we normalized the values of the ABTS readout in the ELISA to the dCas9 amount with the help of the westernblot of the respective contructs. We substracted the no oligo values and then set the dCas9 alone to 100% expression.For the dCas9-VP16 construct we received a value of 25,6% expression after analysis with imageJ. For dCas9-KRAB 5,4% and for dCas9-G9a 53,7%.  Errorbars show the standard deviation of two technical replicates. 
</P><DIV><TABLE width="800px"><TBODY><TR><TD> uniBAss ELISA results 29.09.13. </TD></TR></TBODY></TABLE></DIV><P> Again only the non-standardized constructs show an ABTS signal, meaning that no dCas9 binding is detected in our standardized pSB1C3 constructs. This is probably due to the low expression of CMV-dCas9 and SV40-dCas9 we observed in the western blot which might not be sufficient to be detected with uniBAss. For the other oligo control we see strange off target effects that are probably due to a technical mistake. Maybe our BSA batch is not working properly as we get too high values for unspecific binding. The results show non the less that there is no binding detected for negative controls lacking dCas9 or the RNAimer plasmid.
</P><P> After evaluating the results of the assay we desided to repeat the experiment one more time. Next time we will try to use more cell lysate to get higher amounts of dCas9 protein so that effects for the dCas9 in pSB1C3 can be seen in uniBAss. Therefore four 6-well plates HEK-293T cells are seeded which will be transfected tomorrow. We will try the following changes in our uniBAss protocol: </P><OL><LI> Transfection in duplicates to gain more volume of lysate </LI><LI> take up cells in 100-200µl dilution buffer per 6-well to obtain higher dCas9 concentration in lysates </LI><LI> sonify cells for 10min </LI><LI> centrifuge lysate before applying it to the 96-well ELISA plate to get rid of cell debris </LI><LI> increase oligo concentration when coating the wells </LI><LI> try a different off-target oligo from AG Weber </LI></OL></DIV><DIV id="tag"><H2> 30.09.13 </H2><H3> Transfection</H3><P> This time we transfected only the standardized constructs but in biological duplicate to have enough cell lysate for uniBAss. Ilona did the transfection of the four 6-well plates.
</P></DIV><DIV id="october"><P id="h2">
October
</P></DIV><DIV id="tag"><H2> 01.10.13 </H2><P> New 10xTBS was made and the uniBAss ELISA plate was coated with streptavidin.
</P></DIV><DIV id="tag"><H2> 02.10.13 </H2><H3> last but not least uniBAss</H3><P> Today we performed the last trial to get results with uniBAss with the standardized constructs before the Jamboree in Lyon.
</P><P> The ELISA plate well were blocked and coated with oligos (EMX1, other oligo from AG Weber, no oligo). Then wells were blocked again with blocking buffer containing new BSA (Sigma). Cells were scraped off in 160µl dilution buffer (also with new BSA) per 6-well, sonified for 10min at 4°C and centrifuged 5min at 500xg. 100µl of lysate was applied to each well and incubated for more than 1h. Afterwards the standard protocol was followed. 
</P><P> All efforts taken to upconcentrate the amount of dCas9 in pSB1C3 were not enough to see an effect in uniBAss. Due to a lack of time we will not repeat the uniBAss again. 
</P></DIV><H2>06.10.13</H2><H3>Seeding of cells for functional test of all truncations</H3><P>2 24-well plates were seeded with 65,000 cells per well.</P><H2>07.10.13</H2><H3>Transfection</H3>
One plate was transfected with a reporter plasmid containing SEAP under the control of a CMVmin promoter, differnently truncated dCas9-VP16 constructs 

and two RNA plasmids (activation via VP16); the other with a reporter plasmid containing SEAP under the control of a SV40 promoter, differnently 

truncated dCas9 constructs and two RNA plasmids (repression via CRISPRi).
<OL><LI>40 µl Opti-MEM + 0.75 µg of the DNA of interest were mixed in a 1.5 ml Eppi.</LI><LI>Addition of 2.25 µl PEI-solution, vortexing for 10 s and incubation for at least 15 min at RT.</LI><LI>Solution was spread drop-wise to the cells in the dish</LI></OL><P>Ratio of DNA amount (mass): 
reporter plasmid (SEAP) : effector plasmid (Cas9) : RNA plasmid
1 : 4 : 4 
Additionally a plasmid coding for Renilla luciferase was cotransfected (5 % of DNA amount)</P><H2>08.10.13</H2><H3>Medium change</H3>
Medium was changed after 12 h.

<H2>09.10.13</H2><H3>Medium removal</H3><UL><LI>Supernatant was collected for SEAP measurement 36 h after transfection for repression and 46 h after transfection for activation.</LI><LI>Cells of repression were frozen at - 80 °C for later on Renilla measurement.</LI></UL><H3>SEAP measurement of repression</H3><UL><LI>Collected supernatant and fresh cell culture medium was heated at 65 °C for 30 min, then centrifuged at 1250 g for 1 min.</LI><LI>Supernatant was diluted in cell culture medium 1:5 and 1:10.</LI><LI>80 µl of (diluted) supernatant was applied to 100 µl of SEAP buffer.</LI><LI>After addition of 20 µl pNPP (substrate of SEAP) the absorbance of the mix was measured every minute for 2 h at 405 nm.
</LI></UL><H3>SEAP measurement of activation</H3><UL><LI>Collected supernatant and fresh cell culture medium was heated at 65 °C for 30 min.</LI><LI>80 µl of supernatant was applied to 100 µl of SEAP buffer.</LI><LI>After addition of 20 µl pNPP (substrate of SEAP) the absorbance of the mix was measured every minute for 2 h at 405 nm.
</LI></UL><H3>Cell lysis for lyciferase measurement</H3><UL><LI>250 µl of lysis buffer (containing Tris-HCl, EGTA, MgSO<SUB>4</SUB>, DTT and protease inhibitor) were applied to the (thawn) cells of each well.</LI><LI>Incubation on ice for at least 10 min.</LI><LI>Centrifugation at 2,200 g for 15 min.</LI></UL><H3>Luciferase luminescence measurement</H3><UL><LI>80 µl of supernatant were pipetted in a white 96 well plate.</LI><LI>Measurement of luminescence (every 2 min for 15 min) immediately after addition of 20 µl Renilla substrate in PBS.</LI></UL><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>Results of activation via VP16</B>
Truncation clearly activated SEAP expression, but afterwards sequencing revealed that the truncation did not work. So T4 is actually the same as the 

positive control.				
			</TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="imgtxt" width="800px"><TBODY><TR><TD><B>Results of repression via CRISPRi</B>
After normalization to Renilla expression the positive and the negative controls had almost the same values. So there can not be drawn a validated 

conclusion from this experiment. Nevertheless it is conspicuous that the SEAP level of truncation 4 is clearly below the ones of 2, 3 and 5.			

	
			</TD></TR></TBODY></TABLE></DIV><H2>10.10.13</H2><P>Digestion of uniCAS Truncation 4 with EcoRI and AgeI. Simultaneous digestion of G9a, Krab, KrabKrab, VP16, VP32, VP64 in RFC25 standard with NgoMIV and SpeI. Additionally, digestion of the BgH terminator in RFC10 with XbaI and EcoRI. Subsequent ligation of three fragments each. uniCas Truncation 4 and BgH terminator fragments were the same for all plasmids, whereby only the effector domains were exchanged among the constructs to-be cloned.</P><H2>14.10.13</H2><P> Striking out of four colonies per attempt: uniCAS T4-VP16, uniCAS T4-VP32, uniCAS T4-VP64, uniCAS T4-Krab, uniCAS T4-KrabKrab and uniCAS T4-G9a. </P><H2>15.10.13</H2><P> Miniprep of clones. Testdigest with NotI. All constructs have shown positive digest results, except for uniCAS T4-VP32.</P><DIV><TABLE class="gelpic"><TBODY><TR><TD> Roth 1kb marker, T4 (neg. control), T4-VP16, T4-VP64, T4-VP64, T4-Krab, T4-KrabKrab, T4-G9a, Roth 1kb marker 
		    After NotI digestion, a ~ 600 bp band, indicating the CMV promoter became recognizable for all obtained vectors. Moreover, according to the size of the upper band, it was also obvious to detect size differences among constructs such as VP16 and VP64. </TD></TR></TBODY></TABLE></DIV><H2>16.10.13</H2><H3>Seeding of cells for functional test of truncation 4</H3><P>2 24-well plates were seeded with 65,000 cells per well.</P><H2>17.10.13</H2><H3>Transfection</H3>
One plate was transfected with a reporter plasmid containing SEAP under the control of a CMVmin promoter, truncated dCas9-VP16 / VP64 constructs as 

well as two RNA plasmids (activation); the other with a reporter plasmid containing SEAP under the control of a CMV promoter, truncated dCas9-KRAB2 / 

G9a constructs and a RNA plasmid containing two crRNAs (repression).
<OL><LI>40 µl Opti-MEM + 0.75 µg of the DNA of interest were mixed in a 1.5 ml Eppi.</LI><LI>Addition of 2.25 µl PEI-solution, vortexing for 10 s and incubation for at least 15 min at RT.</LI><LI>Solution was spread drop-wise to the cells in the dish</LI></OL><P>Ratio of DNA amount (mass): 
reporter plasmid (SEAP) : effector plasmid (Cas9) : RNA plasmid
1 : 4 : 4 
Additionally a plasmid coding for Renilla luciferase was cotransfected (5 % of DNA amount)</P><H2>18.10.13</H2><H3>Medium change</H3>
Medium was changed after 14 h.

<H2>19.10.13</H2><H3>Medium removal</H3><UL><LI>Supernatant was collected for SEAP measurement 36 h after transfection for repression and 46 h after transfection for activation.</LI><LI>Cells of repression were frozen at - 80 °C for later on Renilla measurement.</LI></UL><H2>20.10.13</H2><H3>SEAP Assay</H3><DIV><TABLE class="image"><TBODY><TR><TD> Results of SEAP activation via uniCAS truncation No. 4 with VP16 and VP64 effectors. An activation by the truncated constructs could not be observed - as obvious by comparing the positive control (left) with any of the vectors.</TD></TR></TBODY></TABLE></DIV><DIV><TABLE class="image"><TBODY><TR><TD> Results of SEAP repression via uniCAS truncation No. 4 with G9a and Krab effectors. Strongfold downregulation could be observed in most cases. However, as off-target repression only occurs, it can be assumed that the repressive effect does not emerge from the domains or locus-specific binding.</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Freiburg/Notebook/method">http://2013.igem.org/Team:Freiburg/Notebook/method</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Notebook/method" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Notebook/method" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Notebook/method&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Notebook/method&amp;oldid=350768" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>