"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg/Project/truncation
From 2013.igem.org
Home
Highlights
Project
Abstract & Introduction
Targeting
Effectors
Effector Control
Modeling
Truncation
uniBAss
uniBOX
Manual
Application
Notebook
Targeting
Effectors
Effector Control
Modeling
uniBAss
Standardization
Material and Methods
BioBricks
Our BioBricks
Favorite BioBricks
Improvement of a Registry part
Sharing of our BioBricks
Safety
Human Practice
Expert opinions
Ethics
Safety & Security
Politics
Society
Education
Art
Facebook / Media
Team
News
Members
Exolab-Events
Statistics
Photo Gallery
Our Partners
Collaborations
Attributions
Thanks to
Abstract & Intro
Targeting
Effectors
Effector Control
Modeling
Truncation
uniBAss
uniBOX
Manual
Application
Truncation of the dCas9 protein
Reason for truncating dCas9
With a size of 160 kDa, the dCas9 protein of the CRISPR/Cas system  is the largest molecular tool among Zinc Fingers and Transcription Activator–Like Effectors. Difficulties that occurred during the light induced translocation of a dCas9 fusion construct in the nucleus indicated, that the size of dCas9 might be a bottleneck for efficient genome engineering - hence, a truncation strategy was developed.
Truncation of dCas9
How to truncate dCas9
To reduce the size of dCas9 we ran a PCR over the backbone with two primers binding in dCas9, one with an overlap for Gibson Assembly. After the PCR one-fragment gibson cloning was performed. We tried five different strategies to reduce the size of the dCas9 protein. In the first attempt we deleted 365 bp near the N-terminus of the protein. In a second try we erased 306 bp near the C-terminal part of dCas9. Here we assume the reverse transcription domain of the protein, which will probably be responsible for DNA binding. Therefore this truncation is thought to verify this assumption and can serve as a negative control. For truncation 3, 4, and 5 we just deleted the beginning, the middle and the end of dCas9 more drastically by throwing out about 1000 bp.
Figure 1: Design of the truncated dCas9 versions.
dCas9 is flanked by NLS sequences and tagged by a HA epitope. The CMV promoter and the BGH terminator were chosen to control gene expression. The red parts are missing in the truncated sequences. You can find truncation 1 in the registry under BBa_K1150047 , truncation 2 under BBa_K1150048 , truncation 3 BBa_K1150049 , truncation 4 under BBa_K1150050 and truncation 5 under BBa_K1150051 .
Results
The PCR over the pSB1C3 backbone of the dCas9 plasmid worked for all primer pairs. The different lengths of the PCR products, due to differently truncated dCas9, are clearly visible. After Gibson assembly with these PCR products the test digest of the plasmids also showed the expected length. We cut with NotI so that the insert of the plasmid is cut out. Only for truncation 4 we received a shorter fragment than expected.  To ensure the expression of the shortened dCas9 version we transfected the midi-preps of them into HEK-293T cells and performed a western blot analysis of the cell lysates. As it can be seen in figure 2 we could prove the expression of the standardized truncations. As expected truncations 1 and 2 are about 100 amino acids shorter resulting in an approximately 11 kDa shift in the western blot compared to the full-lengths dCas9.  Truncation 3 and 5 run at around 120 kDa. Here about 300bp are missing, resulting in a size reduction of about 33 kDa. Only truncation 4 is shorter than expected. It has about 80 kDa and the sequencing of the midi-prep of T4 revealed a nearly 2000 bp deletion in the dCas9.
Figure 2: Western blot with anti HA-antibody to show the expression of the different dcas9 truncations.
HEK-293T cells were transfected in 6-well plates with 5 different truncated CMV:dCas9 constructs in pSB1C3 and the EMX1 RNAimer plasmid in 6-well plates. 42 hours post transfection cells were taken up in 500 µl optimized dilution buffer. 170 µl were centrifuged, pelleted cells were resuspended in SDS sample buffer and used for semi-dry western blotting . All truncations are expressed (upper bands in T1-5) and could be detected via HA. Only truncation 4 is smaller than expected.
After the proof of expression we tested the binding capacity of the truncated dCas9 versions to DNA with the uniBAss ELISA ( see Results of uniBAss ). By now we could not show binding of the untruncated dCas9 in pSB1C3 indepent of CMV or SV40 promoter. This might be due to a very weak expression of dCas9 in pSB1C3 which results in amounts of dCas9 which are not sufficient for uniBAss detection levels. Because we cannot prove the binding of the full-length dCas9 in the iGEM backbone, we are hence not able to make a statement about the binding capacity of the truncated versions. To adress this problem we tested the DNA binding capacity of the truncated dCas9 versions in a CRISPRi approach
CRISPRi
CRISPR interference is the process that the catalytically dead Cas9 targeted to a gene can repress its transcription [1] . We used this effect to test if our truncated dCas9 versions are still able to bind to DNA and therefore can cause the CRISPRi effect which we measured by repression of our reporter gene SEAP. To get reliable data we co-transfected the luciferase Renilla as an internal standard. This enables the normalization of the SEAP values to the cell number. Figure 3 shows the result of the CRISPRi repression experiment. The full-length dCas9 served as a positive control and showed repression of the SEAP levels. But also the negative controls, one without crRNA and one without dCas9 showed smaller SEAP levels than truncation 2,3 and 5. As the positive and negative controls in this experiment show similar values not much conclusion could be drawn out of it. Nevertheless truncation 4 seems to be a very promising candidate for further investigation as it leads to SEAP levels as low as the positive control. Before the Renilla normalization this effect was even more prominent and therefore all following experiments to validate the DNA binding capacity of a truncated dCas9 version concentrate on truncation 4, BioBrick BBa_K1150050 , our uniCas.
Figure 3:Results of repression via CRISPRi
dCas9 constructs were transfected into HEK-293T cells with a crRNA plasmid targeting the SEAP gene of a constitutively active SEAP reporter plasmid which was co-transfected. After normalization to Renilla expression the positive (dcas9 with crRNA)   and the negative controls (no crRNA or no dCas9) had almost the same values. So there can not be drawn a validated   conclusion from this experiment. Nevertheless it is conspicuous that the SEAP level of truncation 4 is clearly below the ones of 2, 3 and 5.
Repression via KRAB or G9a
To further investigate the functionality of our truncated dCas9 we cloned our repressive domains KRAB and G9a three prime of the most promising candidate (truncation 4). We directed these constructs to the promoter region of a constitutively active SEAP-reporter gene and analyzed the decrease of the SEAP expression after normalizing with the internal standard by the luciferase renilla. Figure 4 shows the results of this experiment.
Figure 4:Results of repression via KRAB and G9a
dCas9 fusion constructs were transfected into HEK-293T cells with and without crRNA plasmid targeting the constitutive promoter of the also co-transfected SEAP reporter plasmid. The full-length dCas9-KRAB (positive control) shows a repression in SEAP levels when the corresponding crRNA plasmid is cotransfected and without crRNA the repressive effect is not visible. For the truncated dCas9-KRAB and dCas9-G9a repression is achieved regardless of whether a crRNA plasmid was co-transfected or not. pRSET transfected cells show the normal SEAP level in HEK-293T cells.
Full-length dCas9-KRAB was used as a positive control which shows reduction in SEAP levels when targeted to the SEAP promoter. Without co-transfected crRNA (= off-target control) no significant decrease in SEAP expression could be detected.   But when truncation 4-KRAB or truncation 4-G9a are transfected a severe repression of the SEAP levels qas visible no matter if crRNA was co-transfected or not, which makes the data of this experiment inconclusive. Either the truncated dCas9 has a very high off-target effect, or it doesn´t bind to the DNA at all and the repression in the SEAP levels are caused by an elusive mechanism.  Due to this inconclusive experiment we also tried to activate the SEAP gene with our truncation 4.
Activation via VP16
To be able to activate genes with our truncated dCas9 we cloned  the sequence coding for VP16 three prime of it. To be able to achieve even higher activation rates we also cloned VP64 three prime of truncation 4. As it can clearly be seen in Figure 5 only the full-length dCas9-VP16 positive control showed an increase of the SEAP level when targeted to SEAP gene's minimal promoter.
Figure 5:Results of activation via VP16
dCas9 fusion constructs were transfected into HEK-293T cells with and without crRNA plasmid targeting the  minimal promoter of the also co-transfected SEAP reporter plasmid. The full-length dCas9-VP16 (positive control) shows an increase in SEAP level when the corresponding crRNA plamid is cotransfected and without crRNA activation can be observed. For the truncated dCas9-VP16 and dCas9-VP64 no activation of the reporter gene was achieved. pRSET transfected cells show the minimal leakyness of the SEAP reporter in HEK-293T cells.
The truncated dCas-VP16 as well as the truncated dCas9-VP64 were not able to activate the expression of SEAP. This means that our size-reduced dCas9 version is probably not able to bind to DNA any more.
Discussion
Team Freiburg 2013 wanted to create its own DNA binding protein by decreasing the size of the dCas9 protein. We succeeded in truncating dCas9 in five different ways, reducing the size of the 4101 base pairs large gene from about 200 base pairs to 2000 base pairs. The truncation was performed with the BioBrick BBa_K1150000 where no effector domain is attached to the dCas9. By western-blotting we confirmed the size reduction of the five truncations and also verified their enhanced expression.
To test whether they can still bind DNA and are functional we first used a CRISPRi approach, because for this no effector is needed as the intended effect of repression is due to binding of the dCas9 in the gene of interest. This experiment indicated that truncation 4 might show the CRISPRi effect, so we decided to concentrate on this truncated version, BioBrick BBa_K1150050 , where about 2000 bp in the middle of dCas9 are missing. To confirm it´s DNA binding capacity and hence it´s functionality we cloned effectors to it to be able to activate and repress gene activity. When using KRAB and G9a as repressive domains the results of the experiment to repress the SEAP reporter gene activity were inconclusive. We achieved repression no matter whether we co-transfected the crRNA targeting the SEAP promoter or not. Furthermore we saw CRISPRi effect in this experiment although we did not target the SEAP gene directly but only it's promoter. Therefore this experiment did not help us to assess the functionality of truncation 4.
But when we fused VP16 or VP64 as activation domains to truncation 4 trying to activate the SEAP reporter gene expression we obtained clear results. This experiment indicates that our truncation 4 is not able to activate the SEAP gene and we conclude that probably truncation 4 is not able to bind to DNA properly anymore. As we gained contradictory results in different experiments the terminal prove of functionality of our truncation could not be generated. Further work will have figure out the DNA binding capacity of size-reduced dCas9 versions.
References
(1) Qi, L., et al. (2013). Repurposing CRISPR as an RNA-Guided Platform for Sequence-Specific Control of Gene Expression. Cell 1173–1183
Retrieved from " http://2013.igem.org/Team:Freiburg/Project/truncation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
