<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3></DIV><DIV id="menu"><UL id="navbar"><LI style="padding-left:15px"><A href="https://2013.igem.org/Team:Freiburg">Home</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Highlights">Highlights</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Introduction </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/induction">Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></LI></UL><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna">Targeting</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/method">uniBAss</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation">Standardization</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/protocols">Material and Methods</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/BioBricks">Our BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/favorite_parts">Favorite BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/improvement">Improvement of a Registry part</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/sharing">Sharing of our BioBricks </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/experts">Expert opinions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety/safety">Safety &amp; Security</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/politics">Politics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/society">Society</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/education">Education</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/art">Art</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/Facebook">Facebook / Media</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/News">News</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team2">Members</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Exolab">Exolab-Events</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Statistics">Statistics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/gallery">Photo Gallery</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Attributions">Our Partners</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/Coop">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/attributions"> Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/thanks">Thanks to</A></LI></DIV><DIV id="main"><DIV id="sidebar"><DIV id="sidebar_content"><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna"> Targeting </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector"> Effectors </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling"> Modeling </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/method"> uniBAss </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation"> Standardization </A></P><P class="first_order"><A class="active" href="https://2013.igem.org/Team:Freiburg/protocols"> Material and Methods </A></P><P class="second_order"><A href="#Molecular Cloning"> Molecular Cloning </A></P><P class="second_order"><A href="#Cell Culture"> Cell Culture </A></P><P class="second_order"><A href="#Assays"> Assays </A></P><P class="second_order"><A href="#plasmids"> Oligos and Plasmids </A></P></DIV></DIV><DIV id="main_contant"><DIV id="Protocols"><P id="h1">Standard Protocols
</P></DIV><DIV id="Molecular Cloning"><P id="h2">Molecular Cloning
</P></DIV><DIV id="PCR"><P id="h3"> Polymerase Chain Reaction (PCR)</P><P>
In order to amplify different DNA-templates, preferably from plasmids, different PCR approaches were used - with the following component amounts for 50 µl of a total volume. 31.5 µl of water, 10 µl of 5x Q5 Reaction Buffer, 4 µl of 2.5 mM dNTP solution, 1 µl of DNA template (200 ng), 1 µl of 10 µM Forward and Reverse Primer, 1 µl of DMSO and 0.5 µl of Q5 High-Fidelity DNA Polymerase. Apart from Touch-down variants and annealing temperature gradient analyses, thermocycler programs consisted of the guideline annotated below. </P></DIV><DIV id="Assembly PCR"><P id="h3"> Assembly PCR</P><P>
in case of Gibson Assemblies subsequent to fragment amplification, a reduction of fragment quantities to less than five was soon considered appropriate for more efficient Cloning. Assembly PCRs were established for halving the amounts, by using a consecutive double PCR strategy. Expect for Forward and Reverse Primers, all components of a Standard PCR were joined as mentioned above. A first PCR was done with an annealing temperature derived from the sequences of overlapping template regions. Elongation temperature was estimated by considering the fragment length of the larger DNA template (1 kBP per 30 seconds). Five cycles were performed. 2.5 µl of 10 µM Forward and Reverse Primers, both binding to the new 5‘ of both strands were subsequently added to the first PCR mix. A second PCR was performed, with an elongation temperature calculated on the basis of the assembled fragment‘s size. </P></DIV><DIV id="Preparative Digest"><P id="h3"> Preparative Enzymatic Digest</P><P>
Buffers for two or more combinatoral enzymes were selected by NEB Double Digest Finder. In order to gain high fragment concentrations in subsequent Gel Extractions, a total volume of 50 µl was mostly chosen. Typical Preparative Digests constisted of 2-3 µg vector DNA, 5 µl 10x NEB Buffer, 0.5 µl of 100x BSA, 1 µl per enzyme and were filled up to 50µl with water. Digests were commonly performed at 37 °C for two hours. </P></DIV><DIV id="Oligo Annealing"><P id="h3"> Oligo Annealing</P><P>
5 µl of 100 µM both Forward and Reverse primer solutions were diluted in 80 µl of water. 10 µl of NEB Buffer No. 2 were added. Samples were heated up to 95 °C for two minutes, whereupon the heating block was switched off. After a 2-hour step of gradual cooling down to RT, the annealed oligos were stored at -20 °C. </P></DIV><DIV id="Ligation"><P id="h3"> Ligation</P><P>
In order to avoid low ligation efficiencies or undesired byproduct vectors, fragment molarities and lengths were considererd as parameters for calculational schemes. Smaller fragments were mostly added excessively within between 4-fold and 8-fold molar amounts, compared to larger backbone fragments. Volume constraint often arose low Gel Extraction yields of particular fragments, so that total DNA quantities were fixed to values at around 25 ng. 5‘ dephosphorylation of single fragments was scarcely performed for reactions with high proportions of backbone religation, using Antarctic Phosphatase. Typical ligation approaches contained 2 µl 10x T4 Buffer, 1-7 µl of each fragment, 1 µl of T4 Ligase and a water fill-up to 20 µl. A 30-minute incubation step was performed at RT, preceding subsequent transformation. </P></DIV><DIV id="Gibson Assembly"><P id="h3"> Gibson Assembly</P><P>
Fragments with around 40 bp overlaps were obtained through PCRs with previously designed primer overhangs. According to rigidly calculated molar amounts, fragments were joined and filled up to 5 µl with of water. These 5 µl were added to 15 µl of ice-chilled Gibson-Mastermix, containing strictly defined amounts 5x ISO Buffer, T5 Exonuclease, Taq Ligase and Phusion Polymerase (Gibson et al., 2009). Samples were subsequently heated to 50 °C for one hour. Thereafter, transformation of 5 µl mix to a 25 µl aliquot of chemically compentent <I>E. coli</I> cells was performed.</P></DIV><DIV id="Transformation"><P id="h3"> Transformation</P><P>
3-4 µl of assembled plasmids, derived from either Gibson Assemblies or classical cloning steps, were added to a 25 µl aliquot of chemically competent Top10 <I>E. coli</I> cells on ice. After an incubation period of 30 minutes, a 42 °C heat shock was performed for exactly 45 seconds. Subsequently, a second incubational step on ice was conducted for 2 minutes. 500 µl of previously autoclaved LB medium were added to each aliquot of transformation mix. Samples were incubated at 37 °C and 300 rpm for one hour. Transformations were finalized with plating of 500 µl of transformed <I>E. coli</I> solution on an Ampicillin or Chloramphenicol containing agar plate. Subsequently, plates were stored at 37 °C over night.</P></DIV><DIV id="Colony PCR"><P id="h3"> Colony PCR</P><P> In order to screen for positive bacterial clones after transformation experiments, Colony PCRs with available primers were done as follows. One single plate colony was picked and dissolved in 17.8 µl of water in a PCR tube, subsequently 2.5 µl of 10x Standard Taq  Reaction Buffer, 1 µl of 10 µM Forward and Reverse Primer, 1 µl 2.5 mM dNTP solution and 0.125 µl Taq Polymerase were added from a previously prepared Mastermix. </P></DIV><DIV id="Plasmid Isolation"><P id="h3"> Plasmid Isolation</P><P>
To purify plasmid DNA from agar plates, either Roti-Prep Plasmid Mini Kit from Carl Roth or High Pure Plasmid Isolation Kit from Roche were used for Minipreps. For high yields of supercoiled, endotoxin-less DNA, Genomed‘s Jetstar Plasmid Purification MIDI Kit or Genopure plasmid MIDI Kit from Roche were used with 150 mL of transformed E. coli o/n culture. No deviances from the manufacturers‘ guidelines were incorporated by performing the isolation steps. </P></DIV><DIV id="Precipitation"><P id="h3"> Isopropanol Precipitation</P><P>
For an increased concentration of DNA, precipitations with Isopropanol were performed at cooled temperatures. Therefore, 1/10 of volume of 5M NaCl solution was added to the respective tube, followed by an additional 1 volume of chilled Isopropanol. Tubes were stored at -20°C for 30 minutes. A subsequent 30-minute centrifugation step at 4°C was then performed. Next, the remanent pellet was washed with 70% ethanol and centrifuged for another 15 minutes. Ethanol was finally carefully removed and the DNA resuspended in an adequate volume of deionized water. </P></DIV><DIV id="Analytical Digest"><P id="h3"> Analytical Enzymatic Digest</P><P>
In comparison with Preparative Digests, analytical controls were reduced to a significantly lower volume. Usual Master Mix aliquots contained 6 µl of water, 1 µl 10x  NEB Buffer, 0.5 µl of each enzyme, and 2 µl of test DNA (~300 ng) which were thereupon added to a PCR tube. Digests were commonly performed at 37 °C for two hours. </P></DIV><DIV id="Gel Electrophoresis"><P id="h3"> Agarose Gel Electrophoresis</P><P>
Towards PCR and enzymatic digest analyses, agarose gels were prepaired with 0,5x TAE and Agarose concentrations between 0.7 % and 0.9 % (w/v). At 65 °C, 1 µl of 10,000x GelRed was added to the Gel before a one hours polymerisation step. As a marker, 1 µl of GeneRuler 1 kb DNA ladder was loaded. Gel runs were commonly performed with voltage ranges between 80 V and 130 V for 45 minutes.</P></DIV><DIV id="Gel Extraction"><P id="h3"> Gel Extraction</P><P>
When extracting PCR bands or digested fragments from an agarose gel, the QIAquick Gel extraction Kit from Qiagen was used. Steps were performed in accord with provided user‘s manuals, including three deviances: centrifugation steps were always performed with or with less than 17,900 g, columns were rotated by 180° after washing and last, DNA-binding columns were heated to 50 °C for two minutes before the final elution step. </P></DIV><DIV id="Cell Culture"><P id="h2">Cell Culture</P></DIV><DIV id="h3">Growing Conditions </DIV><P> Cells were cultured in DMEM complemented with 10% FCS, glutamin (2 mM), 100 units/ml Penicillin and 100 ug/ml Streptomycin. Cells were grown in humid conditions at 37°C and 5% CO<SUB>2</SUB>.</P><DIV id="Cell Splitting"><P id="h3"> Cell Splitting</P><P>
DMEM, DPBS and Trypsin solutions were prewarmed to 37 °C. Old medium was aspirated and cells were washed with 5 ml of DPBS, which was subsequently aspirated as well. After an addition of 1 ml Trypsin, an incubation step at 37 °C was performed - until all cells showed signs of detachment. 5 ml of fresh DMEM were then added to recollect trypsinated cells from the dish. Cells were transferred to a Falcon tube and centrifuged at 900 g for 2 minutes. The DMEM-Trypsin supernatant was removed and the pellet finally resuspended in 6 ml DMEM. Seeding into a new 10 cm, 6-well or 24-well dish was performed and cells stored at 37 °C. Confluency was usually checked the day after.
</P></DIV><DIV id="Cell Seeding"><P id="h3"> Cell Seeding</P><P>
For a 24-well plate, 0.5 ml cell suspension was commonly utilized per well. Accordingly, 2 ml were added to 6-wells and 10 ml per 10 cm petridish. The amount of cells depended on the respective experiment, but usually ranged from 65,000 in 24 wells to 300,000 in 6-wells and 1,800,000 per 10 cm dishes.
</P></DIV><DIV id="PEI Transfection"><P id="h3"> PEI Transfection</P><P><B>10 cm dish</B>
24 µl of PEI-solution were mixed with ~ 600 µl of Opti-MEM. 8 µg of the DNA of interest was added to the solution, then the tube was immediately vortex thoroughly and had to incubate at room temperature for exactly 15 min. Finally, the solution was gently resuspended and drop-wisely spread to the cells in the dish. This was in turn swung in an 8-form.<B>6-well plates</B>
7.5 µl of PEI-solution, 200 µl of Opti-MEM and 3 µg of interest DNA were used.<B>24-well plates</B>
1.5 µl of PEI-solution, 50 µl of Opti-MEM and 0.5 µg of interest DNA were used.</P></DIV><DIV id="Cell Lysis"><P id="h3"> Cell Lysis with RIPA Buffer</P><P>
Cells of 24-well plates were washed with 500 µl of cooled DPBS. Then, cells were lysed in 100 µl of chilled RIPA Buffer, followed by a 10-minute incubation on ice. Next, cells were dissolved with a cell scrapper or inverted pipet tip and the developing lysate transferred to a 1.5 ml tube. Samples were vortexed thoroughly, preceding a second 10-minute incubation on ice. A 5 minute centrifugation step was done at 10,000 g. 60 µl of the supernatant were mixed with 20 µl of 4x Laemmli protein dye. Samples were heated to 95 °C for 5 minutes and afterwards either frozen at -20 °C or directly used for subsequent Western Blotting.
</P></DIV><P id="h3"> PCB preparation</P><P>
PCB preparation
25mg purified PCB was obtained from LivChem. The PCB- powder was diluted in 1,3ml DMSO and aliquoted into Eppendorf- tubes. The absorbance- spectrum at 300 to 750nm, as well as A680 was determined using the GeneQuant 1300- spectrometer. The concentrations were calculated according to Lambert- Beer´s Law (E= ε*c*d;ε680nm= 37900 M^(-1) cm^(-1)). 
The absorbance spectrum of PCB was measured aiming to proof the purity of the purchased PCB.
PCB was diluted in methanol/HCl (95:5) and subsequently the absorption was measured within the range of 450 to 800nm wavelength using the infinite M200 PRO- plate- reader (TECAN, Switzerland).


</P><P id="h3"> Red-Light experiments</P><P>
24 h post transfection, the medium was exchanged thereby adding 15µM phycocyanobilin (PCB). The cells were incubated for 1h in the CO2- incubator to allow the formation of holo-Phytochrome B. Subsequently the cells were illuminated under controlled conditions using light sources emitting distinct amounts of light: 20µE for 48h, 660nm or 740nm wavelength. The control-plate was kept in darkness.

</P><P id="h3"> UV-B Light experiments</P><P>
The cells were illuminated under controlled conditions using light sources emitting distinct amounts of light: 5µE for 24h, 311nm wavelength. The control-plate was kept in darkness.

</P><DIV id="Assays"><P id="h2">Assays
</P></DIV><DIV id="uniBAss"><P id="h3"> uniBAss</P><P>ELISA plates (Corning, Inc., cat. no. 3590, NY, New York) were coated with 100 µl streptavidin solution 
(20 µl/10 ml in dH20 to obtain a final concentration of 4 µg/ml) and incubated overnight at 4°C. 
Afterwards, plates were washed 3x with TBST (1x TBS, 0,02% Tween20) and 300 µl blocking buffer (1x 
TBST,  1 %  BSA)  were  added  for  1 h.  After  3x  washing  with  TBST,  the  biotinylated  oligonucleotides (10pmol/well) were applied and incubated at RT for 1 h. Afterwards the plates were washed 3x with 
TBST. To obtain the cell lysate with the Cas9 protein, the transfected HEK 293T cells (250.000 cells/ml) 
were  resuspended  in  250  µl  dilution  buffer  (10 mM Tris,  1 %  BSA,  10 mM  MgCl2,  10 mM  NaCl) 
containing EDTA free Protease inhibitor per 125.000 cells/ml and sonified for 10 min. To remove the 
cell fractions the lysate was centrifuged for 5 min at 500g.
Afterwards 80 µl or 100 µl of the diluted cell lysate were transferred to the ELISA plates. After 1 h 
incubation at RT the plates were washed 3x with TBST and 100 µl anti-HA antibody solution (1000x 
diluted in blocking buffer) was applied to each well and incubated for 1 h. After 3x washing with 
TBST,  100 µl  anti-mouse  HRP  antibody  (4000x  diluted  in  blocking  buffer)  were  added.  After  1 h 
incubation,  plates  were  washed  3x  with  TBST,  probed  with  100 µl  ELISA  ABTS  substrate  and  the 
absorbance was measured at 405 nm. 
For the positive controls, wells were coated with 100 µl FM protein (1.0 µg/ml) and directly with the 
sonificated cell lysate in the dilution buffer. 
</P></DIV><DIV id="Western Blot"><P id="h3"> SDS-PAGE &amp; Western Blot</P><P>
For efficient detection of proteins in our studies, especially dCas9 and it‘s targets, cells were transfected and after 48 h cell lysis followed. 

SDS-Gels (10 % SDS) were performed in Invitrogen-cassettes or Biorad-systems. Therefore, resolving- and stacking-gels were poured as listed in the according table.
The bags were loaded with 20 µl sample or 10 µl marker (cell lysates were boiled at 95 °C in 1x Loading Buffer for 5 minutes before). Runs were performed at 80 volt until the samples had reached the resoltution gel. Then, the voltage was increased to 120 volt. Gels run until the blue frontline had passed the gel completely.
Afterwards, blotting in a wet tank or semi-dry blotting followed. therefore PVDF-membranes were activated for 5 minutes in methanole. Then, they were passed to transfer-buffer just as the Whatman-Paper.  
wet-blotting:
The blotting sandwiches were prepared as follows, started from the bottom: 
<LI> one fiber pad
</LI><LI> 2 pieces of whatman-paper
</LI><LI> SDS-gel
</LI><LI> PVDF-membrane
</LI><LI> one piece of whatman-paper
</LI><LI> one fiber pad


Then, the sandwich was inserted into the wet tank. Blotting was performed for 1.5 h at 4 °C. For one membrane 200 mA was set (two membranes: 400 mA). 
semi-dry-blotting:
For semi-dry blotting the apparatuses of AG Weber were used.
</LI><LI> one piece of whatman-paper
</LI><LI> PVDF-membrane
</LI><LI> SDS-gel
</LI><LI> one piece of whatman-paper
 
Blotting was performed for 1.5 h. For one membrane 350 mA was set (two membranes: 700 mA). 
After blotting, the membrane was blocked in 10 ml of 4 % milkpouder-TBST solution for 1 h. The solution was aspirated before primary antibody in 2 % milkpouder-TBST solution was added (10 ml) and incubated over night on a shaker at 4 °C. In general primary antibodies were: anti-HA (dilution: 1:2500), anti B-Actin (dilution: 1:2500), anti-GAPDH (dilution: 1:5000) 
The next day, three washing steps with TBST and dH20 followed, each 5 minutes (primary antibody solution was stored and used 4 times). Secondary antibody solution (also in 2 % milkpouder-TBST solution) was added for 1 h at RT and decanted afterwards. In general secondary antibodies were: anti-mouse-HRP (1:5000) or anti-rabbit-HRP (1:5000) 
Another 3 washing steps with TBST followed. Proteins were detected by respectively adding 0.5 mL ECL solution 1 and 2 (Promega) to the membrane.


<TABLE id="tabelle"><TABLE border="3" frame="box"><TBODY><TR><TH>	Resolution Gel (1x)	</TH><TH>	Stacking Gel (1x)	</TH></TR><TR><TD>	1.34 ml H20	</TD><TD>	1.9 ml H2O	</TD></TR><TR><TD>	1.2 ml 50% Succrose	</TD></TR><TR><TD>	2.82 ml Tris/HCl pH 8,8	</TD><TD>	3.2 ml Tris/HCl pH 6,8	</TD></TR><TR><TD>	1.89ml 38% Acrylamid	</TD><TD>	0.33 ml 38% Acrylamid	</TD></TR><TR><TD>	75µl 10% SDS	</TD><TD>	31.5 µl 10% SDS	</TD></TR><TR><TD>	2µl TEMED	</TD><TD>	1.7 µl TEMED	</TD></TR><TR><TD>	94µl APS	</TD><TD>	83 µl APS	</TD></TR></TBODY></TABLE><TABLE id="tabelle"><TABLE border="3" frame="box"><TBODY><TR><TH>	Running Buffer (250 mM; 1,92 M Glycin; 1% SDS)	</TH></TR><TR><TD>	30 g Tris	</TD></TR><TR><TD>	144 g Glycin	</TD></TR><TR><TD>	100 ml 10% SDS	</TD></TR><TR><TD>	fill up to 1 L with dH20	</TD></TR></TBODY></TABLE><TABLE id="tabelle"><TABLE border="3" frame="box"><TBODY><TR><TH>	Transferpuffer (48 mM Tris; 39 mM Glycin; 20% Methanol; pH 9,2; 0,1% SDS)	</TH></TR><TR><TD>	5,82 g Tris	</TD></TR><TR><TD>	2, 93 g Glycin 	</TD></TR><TR><TD>	200 ml MeOH	</TD></TR><TR><TD>	10 ml 10% SDS 	</TD></TR><TR><TD>	fill up to 1 L with dH20	</TD></TR></TBODY></TABLE><TABLE id="tabelle"><TABLE border="3" frame="box"><TBODY><TR><TH>	TBST	</TH></TR><TR><TD>	100 ml 10x TBS	</TD></TR><TR><TD>	900 ml Milipore H2O	</TD></TR><TR><TD>	 2 ml Tween-20 (1:4 solution)	</TD></TR></TBODY></TABLE></TABLE></TABLE></TABLE></TABLE></LI></P></DIV><DIV id="Microscopy"><P id="h3"> Microscopy</P><P><B>Seeding</B> 
For microscopy experiments, 50,000 cells were seeded in 0.5 ml DMEM on sterile cover slips in 24-well plates.<B>Fixation</B>
Cells were treated with 200 μl of 4 % PFA solution and stored at 4°C for 30 minutes. In order to avoid any bleaching, cells were covered with aluminum foil.<B>DAPI-staining and mounting</B>
DAPI was used for visualizing the nucleus. Therefore, cells were dipped into a DAPI solution (1:10,000) for 5 seconds - right before mounting on microscope slides. Next, the cells were washed for 5 seconds in distilled water, had to be dabbed of with a tissue and were then placed upside down on a drop (8.5 μl) of mowiol. The cover slips stayed for an overnight drying at room temperature. </P></DIV><DIV id="SEAP Assay"><P id="h3"> SEAP Assay</P><P>
200 µl of cell supernatant was transferred from a 24-well plate into one 96-well plate (round bottom), followed by a heat incubation at 65°C for 30 min. Centrifugation was done for 1 min at 1250xg. 100 µl of 2x SEAP buffer (20 mM homoarginine, 1 mM MgCl<SUB>2</SUB>, 21 %(v/v) diethanolamine, pH 9.8) were filled in the wells of a 96 well plate (flat bottom). Addition of 80 µl supernatant from the round bottom plate to each well with SEAP buffer. Addition of 20 µl pNPP (120 mM para-nitrophenyl phosphate in H<SUB>2</SUB>O) and bubbles carefully removed. Immediately after this, the 96-well plate was placed into a plate reader. Spectroscopic measurement was taken every minute for 120 times (2h) at a wavelength of 405 nm. 
</P></DIV><DIV id="VEGF ELISA"><P id="h3"> VEGF ELISA </P><P>
The VEGF ELISA was performed with the VEGF ELISA Development Kit #900-K10 (PeproTech) according to the manufacturer's protocol. 96-well ELISA plates were used (Corning #3590 or Greiner #). HEK293T cells were seeded in 24-well plates, transfected with desired constructs and then cultured for 36h sealed by an oxygen diffusible filter. 12h post transfection the medium was exchanged to set VEGF levels to zero. 24h after medium change the supernatant was collected and directly used for ELISA or frozen at -20°C until further use. To normalize VEGF results a constitutive SEAP reporter was always co-transfected so that a SEAP assay could be performed in parallel. ABTS solution was prepared from ABTS aliquot by mixing 11ml Citrate buffer pH4.0 with 200µl ABTS and 20µl H<SUB>2</SUB>O<SUB>2</SUB>. ABTS solution was applied to wells and color development was measured immediately afterwards with the infinite M200 PRO plate reader at 405nm for 35min in 2.5min intervals. To calculate VEGF amounts values of time points lying in the linear standard range were used.
</P></DIV><DIV id="Materials"><P id="h2">
 Materials
</P><P id="h3"> Kits </P><TABLE id="tabelle"><TABLE border="3" frame="box"><TBODY><TR><TH>	Name 	</TH><TH>	Manufacturer	</TH></TR><TR><TD>	Roti-Prep Plasmid Mini 	</TD><TD>	Carl Roth, Karlsruhe, Germany	</TD></TR><TR><TD>	Jetstar Plasmid Purification MIDI Kit	</TD><TD>	Gemomed, Löhne, Germany	</TD></TR><TR><TD>	QIAquick Gel extraction kit 	</TD><TD>	QIAGEN, Hilden, Germany	</TD></TR></TBODY></TABLE><P id="h3"> Enzymes </P><TABLE id="tabelle"><TABLE border="3" frame="box"><TBODY><TR><TH>	Name 	</TH><TH>	Manufacturer	</TH></TR><TR><TD>	Q5 DNA polymerase 	</TD><TD>	New England Biolabs (NEB), Ipswich, USA	</TD></TR><TR><TD>	Restriction enzymes 	</TD><TD>	New England Biolabs (NEB), Ipswich, USA	</TD></TR><TR><TD>	T5 Exonuclease 	</TD><TD>	New England Biolabs (NEB), Ipswich, USA	</TD></TR><TR><TD>	Taq ligase	</TD><TD>	New England Biolabs (NEB), Ipswich, USA	</TD></TR><TR><TD>	Trypsin 	</TD><TD>	Life technologies, Carlsbad, USA	</TD></TR></TBODY></TABLE><P id="h3"> Buffers </P><TABLE id="tabelle"><TABLE border="3" frame="box"><TBODY><TR><TH>	Name 	</TH><TH>	Manufacturer	</TH></TR><TR><TD>	Restriction enzyme buffers 	</TD><TD>	New England Biolabs (NEB), Ipswich, USA	</TD></TR><TR><TD>	TAE buffer 	</TD><TD>	BIOSS	</TD></TR><TR><TD>	Q5 buffer 	</TD><TD>	New England Biolabs (NEB), Ipswich, USA	</TD></TR></TBODY></TABLE><P id="h3"> Chemicals </P><TABLE id="tabelle"><TABLE border="3" frame="box"><TBODY><TR><TH>	Name 	</TH><TH>	Manufacturer	</TH></TR><TR><TD>	Agarose 	</TD><TD>	Carl Roth, Karlsruhe, Germany	</TD></TR><TR><TD>	BSA	</TD><TD>	New England Biolabs (NEB), Ipswich, USA	</TD></TR><TR><TD>	DABCO 	</TD><TD>	Carl Roth, Karlsruhe, Germany	</TD></TR><TR><TD>	DAPI	</TD><TD>	Carl Roth, Karlsruhe, Germany	</TD></TR><TR><TD>	DMSO 	</TD><TD>	New England Biolabs (NEB), Ipswich, USA	</TD></TR><TR><TD>	dNTPs 	</TD><TD>	New England Biolabs (NEB), Ipswich, USA	</TD></TR><TR><TD>	DRAQ5 	</TD><TD>	Biostatus, Shepshed, UK 	</TD></TR><TR><TD>	DTT	</TD><TD>	Carl Roth, Karlsruhe, Germany	</TD></TR><TR><TD>	GelRed 	</TD><TD>	Biotium, Hayward, US	</TD></TR><TR><TD>	GeneRuler 1 kb 	</TD><TD>	Thermo Scientific, Waltham, USA	</TD></TR><TR><TD>	L-homoarginine 	</TD><TD>	Alfa Aesar, UK</TD></TR><TR><TD>	MgCl2	</TD><TD>	Carl Roth, Karlsruhe, Germany	</TD></TR><TR><TD>	Mowiol	</TD><TD>	Carl Roth, Karlsruhe, Germany	</TD></TR><TR><TD>	NAD	</TD><TD>	New England Biolabs (NEB), Ipswich, USA	</TD></TR><TR><TD>	PEG-8000	</TD><TD>	Carl Roth, Karlsruhe, Germany	</TD></TR><TR><TD>	PEI	</TD><TD>	Polyscience, Warrington, USA	</TD></TR><TR><TD>	PFA (4 %) solution 	</TD><TD>	Morphisto, Frankfurt/Main, Germany </TD></TR><TR><TD>	Tris 	</TD><TD>	Carl Roth, Karlsruhe, Germany	</TD></TR></TBODY></TABLE><P id="h3"> Nutritions </P><TABLE id="tabelle"><TABLE border="3" frame="box"><TBODY><TR><TH>	Name 	</TH><TH>	Manufacturer	</TH></TR><TR><TD>	DMEM 	</TD><TD>	Life technologies, Carlsbad, USA	</TD></TR><TR><TD>	LB medium	</TD><TD>	BIOSS, Uni Freiburg	</TD></TR><TR><TD>	Opti-MEM® 	</TD><TD>	Life technologies, Carlsbad, USA	</TD></TR><TR><TD>	HTS	</TD><TD>	Cell Culture Technologies LLC	</TD></TR></TBODY></TABLE><P id="h3"> Instruments</P><TABLE id="tabelle"><TABLE border="3" frame="box"><TBODY><TR><TH>	Name/Instrument 	</TH><TH>	Manufacturer	</TH></TR><TR><TD>	cell counter (CASY Innovatis)	</TD><TD>	Roche, Rotkreuz, Switzerland	</TD></TR><TR><TD>	CO2 incubator	(Heracell 240)	</TD><TD>	Thermo scientific, Waltham, USA	</TD></TR><TR><TD>	confocal microscope	(C2) 	</TD><TD>	Nikon, Düsseldorf, Germany	</TD></TR><TR><TD>	gel chambers	(PerfectBlue Gelsystem)	</TD><TD>	Peqlab, Erlangen, Germany	</TD></TR><TR><TD>	gel documentation	(E-BOX VX2) 	</TD><TD>	Vilber, Eberhardzell, Germany	</TD></TR><TR><TD>	heating block	(Thriller) 	</TD><TD>	Peqlab, Erlangen, Germany	</TD></TR><TR><TD>	incubator	(INE 300) 	</TD><TD>	Memmert, Schwabach, Germany	</TD></TR><TR><TD>	microscope	(EVOS fl)	</TD><TD>	Advanced Microscopy Group (AMG), 
		Mill Creek, USA	</TD></TR><TR><TD>	nano drop	(NanoDrop 2000c) 	</TD><TD>	Thermo scientific, Waltham, USA	</TD></TR><TR><TD>	power supply	(peqPOWER 250V) 	</TD><TD>	Peqlab, Erlangen, Germany	</TD></TR><TR><TD>	steril bench	(Safe 2020)	</TD><TD>	Thermo scientific, Waltham, USA	</TD></TR><TR><TD>	thermocycler I	(GeneAmp PCR System 9700)	</TD><TD>	Applied Biosystems, Foster City, USA	</TD></TR><TR><TD>	thermocycler II	(Mastercycler epgradient S)	</TD><TD>	Eppendorf, Hamburg, Germany	</TD></TR><TR><TD>	plate reader	(infinite M200 PRO)	</TD><TD>	TECAN, Männedorf, Switzerland	</TD></TR></TBODY></TABLE><DIV id="plasmids"><P id="h2">
Oligos and Plasmids
</P></DIV><P>
Klick <A id="link" href="https://2013.igem.org/Team:Freiburg/plasmids"> HERE </A> to see the List of all used Oligos and Plasmids used in this project.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Freiburg/protocols">http://2013.igem.org/Team:Freiburg/protocols</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/protocols&amp;oldid=343420" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></TABLE></TABLE></TABLE></TABLE></TABLE></TABLE></DIV></DIV></DIV></DIV></DIV></BODY></HTML>