<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Project_toolkit"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Project/toolkit">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Project/toolkit&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Project/toolkit&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Project/toolkit&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Project/toolkit" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Project/toolkit</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3></DIV><DIV id="menu"><UL id="navbar"><LI style="padding-left:15px"><A href="https://2013.igem.org/Team:Freiburg">Home</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Highlights">Highlights</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Introduction </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/induction">Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit"> Manual </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></LI></UL><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/crrna">Targeting</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/lab_effector">Effectors</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/induction"> Effector Control</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/method">uniBAss</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Notebook/standardisation">Standardization</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/protocols">Material and Methods</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/BioBricks">Our BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/favorite_parts">Favorite BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/improvement">Improvement of a Registry part</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/parts/sharing">Sharing of our BioBricks </A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/experts">Expert opinions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/ethics">Ethics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Safety/safety">Safety &amp; Security</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/politics">Politics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/society">Society</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/education">Education</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/art">Art</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/HumanPractice/Facebook">Facebook / Media</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/News">News</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Team2">Members</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Exolab">Exolab-Events</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Statistics">Statistics</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/gallery">Photo Gallery</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Attributions">Our Partners</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/Coop">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/attributions"> Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Freiburg/Project/thanks">Thanks to</A></LI></DIV><DIV id="main"><DIV id="sidebar"><DIV id="sidebar_content"><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/1"> Abstract &amp; Intro </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/crrna"> Targeting </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/effector"> Effectors </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/induction"> Effector Control </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/modeling"> Modeling </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/truncation"> Truncation </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/method"> uniBAss </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/unibox"> uniBOX </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/toolkit" class="active"> Manual </A></P><P class="first_order"><A href="https://2013.igem.org/Team:Freiburg/Project/application"> Application </A></P></DIV></DIV><DIV id="main_contant"><DIV id="preample"><DIV id="headline">
The uniCAS toolkit - Customize your experiments!</DIV>

You want to have a maximum of activation or repression of your genes by a minimal effort? Then you 

have to use the uniCAS toolkit provided by the iGEM team Freiburg 2013. All you have to do is:
<UL><LI> Click yourself through the routine below </LI><LI> Order the appropriate plasmids and oligos </LI><LI> Conduct a minimal of cloning </LI><LI> Start your personalized experiment </LI></UL>

By the end of the routine you will get a personal manual. All you need to do is using the uniCAS toolkit and follow the instructions. Best of all: The uniCAS toolkit is all open source and in iGEM standard!
</DIV><DIV id="toolkit" name="toBottom()"><DIV id="first_checkboxes"><LABEL>Activation </LABEL><LABEL>Repression </LABEL></DIV><DIV id="answer_first_checkboxes"><DIV id="check-1" class="toHide" style="display:none"><P> Choose an effector to activate your genes. Click <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/effector#activation"> here</A> to see the functional tests of the different activation effectors. </P><LABEL>VP16 (recommended) </LABEL></DIV><DIV id="check-2" class="toHide" style="display:none"><P> Effectively repress your genes using KRAB or G9a as functional effector.  Click <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/effector#repression"> here</A> to see the functional tests of the different activation effectors. </P><DIV><LABEL>KRAB </LABEL><LABEL> G9a </LABEL></DIV></DIV></DIV><DIV id="answer_second_checkboxes"><DIV id="check-3" class="toHide2" style="display:none"><P> Effectively activate your genes using VP16 as functional effector. </P><LABEL> No Induction </LABEL><LABEL> Red light induction </LABEL><LABEL> UV light induction </LABEL></DIV><DIV id="check-4" class="toHide2" style="display:none"><P> Effectively repress your genes using KRAB as functional effector. </P><LABEL> No Induction </LABEL><LABEL> Red light induction </LABEL><LABEL> UV light induction </LABEL></DIV><DIV id="check-5" class="toHide2" style="display:none"><P> Effectively repress your genes using <B>KRAB</B> as functional effector. </P><LABEL> No Induction </LABEL><LABEL> Red light induction </LABEL><LABEL> UV light induction </LABEL></DIV><DIV id="check-6" class="toHide2" style="display:none"><P> Effectively repress your genes using <B> G9a </B> as functional effector. </P><LABEL> No Induction </LABEL><LABEL> Red light induction </LABEL><LABEL> UV light induction </LABEL></DIV></DIV><DIV id="answer_third_checkboxes"><DIV id="check-7" class="toHide3" style="display:none"><DIV id="final_checkbox_background"><DIV id="final_checkbox_content"><P id="h1">
			Non inducible uniCAS Activator (dCas9-VP16 device)
			</P><P>
			You have chosen to activate a gene using a non inducible dCas9-VP16 device. Therefore you have to order the following plasmids from the <A id="link" href="http://parts.igem.org/Main_Page"> iGEM parts registry</A>. After receiving our plasmids, you will have to clone your target sequence into our crRNA plasmid (protocol see below).
			</P><TABLE><TBODY><TR><TH> No. </TH><TH> Biobrick </TH><TH> Device </TH><TH> Order </TH></TR><TR><TD> 1 </TD><TD> BBa_K1150017 </TD><TD> CMV:HA-NLS-dCas9-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150017"> order </A></TD></TR><TR><TD> 2 </TD><TD> BBa_K1150020 </TD><TD> CMV:HA-NLS-dCas9-L3-VP16-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150020"> order </A></TD></TR><TR><TD> 3 </TD><TD> BBa_K1150034 </TD><TD> RNAimer - plasmid (crRNA - plasmid) </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150034"> order </A></TD></TR></TBODY></TABLE><P>
			Note: 
			All plasmids are optimized for expression in mammalian systems. The devices are also available containing a SV40 instead of a CMV promotor (have a look at our <A id="link" href="https://2013.igem.org/Team:Freiburg/parts"> parts</A> side). This system was tested mainly in CHO-K1 and HEK-293T. 
			</P><P id="h3">
			Design of the crRNA plasmid:
			</P><P> 
			Use our crRNA design tool to design the crRNAs that are needed to target your gene of interest.
			</P><DIV align="center"><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#design_tool"><FONT size="5"><U>crRNA design tool</U></FONT></A></DIV><P>It will generate possible target sites and the appropriate oligos. Order the oligos by the company of your choice. We recommend to test several different loci to target your gene of interest because the efficiency of different crRNA-loci can differ.
			</P><OL><LI><B>Oligo annealing:</B> Anneal forward and reverse oligos to get the desired crRNA. Therefore mix 10 µl of 100 µM forward Oligo, 10 µl of 100 µM reverse Oligo and 80 µl of ddH<SUB>2</SUB>O. Heat the solution to 95° C for 5 minutes. Then turn off the heat block and let the solution cool down.</LI><LI><B>Digest plasmid BBa_K1150034 with Bbs1:</B> The restriction enzyme Bbs1 should always be stored at -80° C. Mix about 500 ng of BBa_K1150034 with 1 µl of Bbs1, appropriate amount of buffer and fill up to 50 µl with ddH<SUB>2</SUB>O. Digest for exactly 3 hours at 37° C. Load the digest on a gel and cut out the DNA band (2900 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate crRNAs (step 1) into Bbs1 cut backbone:</B> The insert (crRNAs) should be ligated into the backbone in 3 molar insert excess. Therefore use this formular: Required Volume of Insert = 3 x Volume(Backbone) x length(Insert) x concentration (Backbone) / [ length(Backbone)  x concentration(Insert) ]. Use about 50 ng of backbone. The length of insert is always 30 basepairs. The length of the backbone is 2900 basepairs. You have to mix the appropriate amount of Backbone and the appropriate amount of Insert with 1 µl of T4 ligase and 2 µl of 10xT4 ligase buffer. Then fill up to 20 µl with ddH<SUB>2</SUB>O. This mix should be incubated for 30 minutes at room temperature.</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI></OL><P> 
			Now that you have created the desired crRNA plasmids it is possible to use them indiviually or fuse different crRNA loci together into one crRNA plasmid (recommended).
			</P><P id="h3">
			Design of a multiple target crRNA plasmid:
			</P><P>
			It is shown that multiple targeting of one gene of interest <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#multiple_targeting">increases the efficiency of regulation</A>. If you want to fuse different crRNA loci together into one plasmid use the following protocol:
			</P><OL><LI><B>First of all digest crRNA plasmid</B> with PstI and SpeI in order to linearize it. Both enzymes cut in the suffix. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, it is possible to assemble multiple crRNA sequences into one plasmid. Therefore mix about 500 ng Backbone with 1 µl enzyme 1 and 1 µl enzyme 2, add an appropriate amount of compatible buffer and fill up to 30-50 µl. Incubate mix at 37° C for 2 hours. </LI><LI><B>Secondly digest crRNA plasmid</B> with PstI and XbaI. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, this is your insert (procedure see above).</LI><LI>  Load digests on a gel and cut out the DNA bands (Backbone 2900 bp, Insert 870 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate</B> the insert in 3 molar excess into the backbone (formular see paragraph above).</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 forward sequencing primer (sequence: GAGTGCCACCTGACGTCTAAGAAAC) and pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI><LI> These steps can be repeated various times. Using this method, you can engineer a plasmid with several crRNA targets!</LI></OL><P id="h3">
			Experimental design (recommendation for mammalian cell culture):
			</P><OL><LI><B>Transfect</B> BBa_K1150020 with all desired crRNA plasmids (seperate crRNA plasmid and/or multiple crRNAs plasmid) </LI><LI><B>Non-effector control:</B> Transfect the appropriate crRNA - plasmid together with BBa_K1150017 that has no effector.</LI><LI><B>Off target control:</B> Transfect BBa_K1150020 without any crRNA plasmid.</LI></OL></DIV></DIV></DIV><DIV id="check-8" class="toHide3" style="display:none"><DIV id="final_checkbox_background"><DIV id="final_checkbox_content"><P id="h1">
			Red light inducible uniCAS Activator (dCas9-PIF6 &amp; PhyB-VP16 devices)
			</P><P>
			You have chosen to activate a gene using a red light inducible dCas9-PIF6 &amp; PhyB-VP16 device. Therefore you have to order the following plasmids from the <A id="link" href="http://parts.igem.org/Main_Page"> iGEM parts registry</A>. After receiving our plasmids, you will have to clone your target sequence into our crRNA plasmid (protocol see below).
			</P><TABLE><TBODY><TR><TH> No. </TH><TH> Biobrick </TH><TH> Device </TH><TH> Order </TH></TR><TR><TD> 1 </TD><TD> BBa_K1150025 </TD><TD> CMV:HA-NLS-dCas9-L3-PIF6-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150025"> order </A></TD></TR><TR><TD> 2 </TD><TD> BBa_K1150026 </TD><TD> CMV:NLS-PhyB-L3-VP16-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150026"> order </A></TD></TR><TR><TD> 3 </TD><TD> BBa_K1150020 </TD><TD> CMV:HA-NLS-dCas9-L3-VP16-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150020"> order </A></TD></TR><TR><TD> 4 </TD><TD> BBa_K1150034 </TD><TD> RNAimer - plasmid (crRNA - plasmid)</TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150034"> order </A></TD></TR></TBODY></TABLE><P>
			Note: 
			All plasmids are optimized for expression in mammalian systems. The devices are also available containing a SV40 instead of a CMV promotor (have a look at our <A id="link" href="https://2013.igem.org/Team:Freiburg/parts"> parts</A> side). This system was tested mainly in CHO-K1 and HEK-293T cells. 
			</P><P id="h3">
			Design of the crRNA plasmid:
			</P><P> 
			Use our crRNA design tool to design the crRNAs that are needed to target your gene of interest.
			</P><DIV align="center"><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#design_tool"><FONT size="5"><U>crRNA design tool</U></FONT></A></DIV><P>It will generate possible target sites and the appropriate oligos. Order the oligos by the company of your choice. We recommend to test several different loci to target your gene of interest because the efficiency of different crRNA-loci can differ.
			</P><OL><LI><B>Oligo annealing:</B> Anneal forward and reverse oligos to get the desired crRNA. Therefore mix 10 µl of 100 µM forward Oligo, 10 µl of 100 µM reverse Oligo and 80 µl of ddH2O. Heat the solution to 95° C for 5 minutes. Then turn off the heat block and let the solution cool down.</LI><LI><B>Digest plasmid BBa_K1150034 with Bbs1:</B> The restriction enzyme Bbs1 should always be stored at -80° C. Mix about 500 ng of BBa_K1150034 with 1 µl of Bbs1, appropriate amount of buffer and fill up to 50 µl with ddH2O. Digest for exactly 3 hours at 37° C. Load the digest on a gel and cut out the DNA band (2900 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate crRNAs (step 1) into Bbs1 cut backbone:</B> The insert (crRNAs) should be ligated into the backbone in 3 molar insert excess. Therefore use this formular: Required Volume of Insert = 3 x Volume(Backbone) x length(Insert) x concentration (Backbone) / [ length(Backbone)  x concentration(Insert) ]. Use about 50 ng of backbone. The length of insert is always 30 basepairs. The length of the backbone is 2900 basepairs. You have to mix the appropriate amount of Backbone and the appropriate amount of Insert with 1 µl of T4 ligase and 2 µl of 10xT4 ligase buffer. Then fill up to 20 µl with ddH2O. This mix should be incubated for 30 minutes at room temperature.</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI></OL><P> 
			Now that you have created the desired crRNA plasmids it is possible to use them indiviually or fuse different crRNA loci together into one crRNA plasmid (recommended).
			</P><P id="h3">
			Design of a multiple target crRNA plasmid:
			</P><P>
			It is shown that multiple targeting of one gene of interest <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#multiple_targeting">increases the efficiency of regulation</A>. If you want to fuse different crRNA loci together into one plasmid use the following protocol:
			</P><OL><LI><B>First of all digest crRNA plasmid</B> with PstI and SpeI in order to linearize it. Both enzymes cut in the suffix. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, it is possible to assemble multiple crRNA sequences into one plasmid. Therefore mix about 500 ng Backbone with 1 µl enzyme 1 and 1 µl enzyme 2, add an appropriate amount of compatible buffer and fill up to 30-50 µl. Incubate mix at 37° C for 2 hours. </LI><LI><B>Secondly digest crRNA plasmid</B> with PstI and XbaI. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, this is your insert (procedure see above).</LI><LI>  Load digests on a gel and cut out the DNA bands (Backbone 2900 bp, Insert 870 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate</B> the insert in 3 molar excess into the backbone (formular see paragraph above).</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 forward sequencing primer (sequence: GAGTGCCACCTGACGTCTAAGAAAC) and pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI><LI> These steps can be repeated various times. Using this method, you can engineer a plasmid with several crRNA targets!</LI></OL><P id="h3">
			Experimental design (recommendation for mammalian cell culture):
			</P><P>
			For red light experiments transfect the following plasmid combinations on two different well plates. One plate will be illuminated with 660 nm for 48 hours. This is the red light induced plate. The other plate will be illuminated with 740 nm. This is the no-induction control. If not working in plant systems add 15µM of PCB prior to illumination and incubate for at least 1h. See our <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/induction#light">light induction project page</A>.
			</P><OL><LI><B>Transfect</B> BBa_K1150025, BBa_K1150026 with all desired crRNA plasmids (seperate crRNA plasmid and/or multiple crRNAs plasmid) </LI><LI><B>Off target control:</B> Transfect BBa_K1150025, BBa_K1150026 without any crRNA plasmid.</LI><LI><B>Target efficiency control:</B> Transfect BBa_K1150020 with the same crRNA plasmids as in the first transfection.</LI></OL></DIV></DIV></DIV><DIV id="check-9" class="toHide3" style="display:none">
bliii
</DIV><DIV id="check-10" class="toHide3" style="display:none"><DIV id="final_checkbox_background"><DIV id="final_checkbox_content"><P id="h1">
			UVB light inducible uniCAS Activator (dCas9-UVR8 &amp; COP1-VP16 devices)
			</P><P>
			You have chosen to activate a gene using a UVB light inducible dCas9-UVR8 &amp; COP1-VP16 device. Therefore you have to order the following plasmids from the <A id="link" href="http://parts.igem.org/Main_Page"> iGEM parts registry</A>. After receiving our plasmids, you will have to clone your target sequence into our crRNA plasmid (protocol see below).
			</P><TABLE><TBODY><TR><TH> No. </TH><TH> Biobrick </TH><TH> Device </TH><TH> Order </TH></TR><TR><TD> 1 </TD><TD> BBa_K1150029 </TD><TD> CMV:HA-NLS-dCas9-L3-UVR8-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150029"> order </A></TD></TR><TR><TD> 2 </TD><TD> BBa_K1150030 </TD><TD> CMV:NLS-COP1-L3-VP16-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150030"> order </A></TD></TR><TR><TD> 3 </TD><TD> BBa_K1150020 </TD><TD> CMV:HA-NLS-dCas9-L3-VP16-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150020"> order </A></TD></TR><TR><TD> 4 </TD><TD> BBa_K1150034 </TD><TD> RNAimer - plasmid (crRNA - plasmid)</TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150034"> order </A></TD></TR></TBODY></TABLE><P>
			Note: 
			All plasmids are optimized for expression in mammalian systems. The devices are also available containing a SV40 instead of a CMV promotor (have a look at our <A id="link" href="https://2013.igem.org/Team:Freiburg/parts"> parts</A> side). This system was tested mainly in CHO-K1 and HEK-293T cells. 
			</P><P id="h3">
			Design of the crRNA plasmid:
			</P><P> 
			Use our crRNA design tool to design the crRNAs that are needed to target your gene of interest.
			</P><DIV align="center"><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#design_tool"><FONT size="5"><U>crRNA design tool</U></FONT></A></DIV><P>It will generate possible target sites and the appropriate oligos. Order the oligos by the company of your choice. We recommend to test several different loci to target your gene of interest because the efficiency of different crRNA-loci can differ.
			</P><OL><LI><B>Oligo annealing:</B> Anneal forward and reverse oligos to get the desired crRNA. Therefore mix 10 µl of 100 µM forward Oligo, 10 µl of 100 µM reverse Oligo and 80 µl of ddH2O. Heat the solution to 95° C for 5 minutes. Then turn off the heat block and let the solution cool down.</LI><LI><B>Digest plasmid BBa_K1150034 with Bbs1:</B> The restriction enzyme Bbs1 should always be stored at -80° C. Mix about 500 ng of BBa_K1150034 with 1 µl of Bbs1, appropriate amount of buffer and fill up to 50 µl with ddH2O. Digest for exactly 3 hours at 37° C. Load the digest on a gel and cut out the DNA band (2900 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate crRNAs (step 1) into Bbs1 cut backbone:</B> The insert (crRNAs) should be ligated into the backbone in 3 molar insert excess. Therefore use this formular: Required Volume of Insert = 3 x Volume(Backbone) x length(Insert) x concentration (Backbone) / [ length(Backbone)  x concentration(Insert) ]. Use about 50 ng of backbone. The length of insert is always 30 basepairs. The length of the backbone is 2900 basepairs. You have to mix the appropriate amount of Backbone and the appropriate amount of Insert with 1 µl of T4 ligase and 2 µl of 10xT4 ligase buffer. Then fill up to 20 µl with ddH2O. This mix should be incubated for 30 minutes at room temperature.</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI></OL><P> 
			Now that you have created the desired crRNA plasmids it is possible to use them indiviually or fuse different crRNA loci together into one crRNA plasmid (recommended).
			</P><P id="h3">
			Design of a multiple target crRNA plasmid:
			</P><P>
			It is shown that multiple targeting of one gene of interest <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#multiple_targeting">increases the efficiency of regulation</A>. If you want to fuse different crRNA loci together into one plasmid use the following protocol:
			</P><OL><LI><B>First of all digest crRNA plasmid</B> with PstI and SpeI in order to linearize it. Both enzymes cut in the suffix. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, it is possible to assemble multiple crRNA sequences into one plasmid. Therefore mix about 500 ng Backbone with 1 µl enzyme 1 and 1 µl enzyme 2, add an appropriate amount of compatible buffer and fill up to 30-50 µl. Incubate mix at 37° C for 2 hours. </LI><LI><B>Secondly digest crRNA plasmid</B> with PstI and XbaI. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, this is your insert (procedure see above).</LI><LI>  Load digests on a gel and cut out the DNA bands (Backbone 2900 bp, Insert 870 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate</B> the insert in 3 molar excess into the backbone (formular see paragraph above).</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 forward sequencing primer (sequence: GAGTGCCACCTGACGTCTAAGAAAC) and pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI><LI> These steps can be repeated various times. Using this method, you can engineer a plasmid with several crRNA targets!</LI></OL><P id="h3">
			Experimental design (recommendation for mammalian cell culture):
			</P><P>
			For UVB light experiments transfect the following plasmid combinations on two different well plates. One plate will be illuminated with 311 nm at 5 µE for 24 hours. This is the UVB light induced plate. The other plate will be wrapped in aluminum foil. This is the no-induction control. See our <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/induction#light">light induction project page</A>.
			</P><OL><LI><B>Transfect</B> BBa_K1150029, BBa_K1150030 with all desired crRNA plasmids (seperate crRNA plasmid and/or multiple crRNAs plasmid) </LI><LI><B>Off target control:</B> Transfect BBa_K1150029, BBa_K1150030 without any crRNA plasmid.</LI><LI><B>Target efficiency control:</B> Transfect BBa_K1150020 with the same crRNA plasmids as in the first transfection.</LI></OL></DIV></DIV></DIV><DIV id="check-11" class="toHide3" style="display:none">
blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla
</DIV><DIV id="check-12" class="toHide3" style="display:none">
blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla
</DIV><DIV id="check-13" class="toHide3" style="display:none">
blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla
</DIV><DIV id="check-14" class="toHide3" style="display:none">
blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla
</DIV><DIV id="check-15" class="toHide3" style="display:none"><DIV id="final_checkbox_background"><DIV id="final_checkbox_content"><P id="h1">
			Non inducible uniCAS Repressor (dCas9-KRAB device)
			</P><P>
			You have chosen to repress a gene using a non inducible dCas9-KRAB device. Therefore you have to order the following plasmids from the <A id="link" href="http://parts.igem.org/Main_Page"> iGEM parts registry</A>. After receiving our plasmids, you will have to clone your target sequence into our crRNA plasmid (protocol see below).
			</P><TABLE><TBODY><TR><TH> No. </TH><TH> Biobrick </TH><TH> Device </TH><TH> Order </TH></TR><TR><TD> 1 </TD><TD> BBa_K1150017 </TD><TD> CMV:HA-NLS-dCas9-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150017"> order </A></TD></TR><TR><TD> 2 </TD><TD> BBa_K1150022 </TD><TD> CMV:HA-NLS-dCas9-L3-KRAB-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150022"> order </A></TD></TR><TR><TD> 3 </TD><TD> BBa_K1150034 </TD><TD> RNAimer - plasmid (crRNA - plasmid)</TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150034"> order </A></TD></TR></TBODY></TABLE><P>
			Note: 
			All plasmids are optimized for expression in mammalian systems. The devices are also available containing a SV40 instead of a CMV promotor (have a look at our <A id="link" href="https://2013.igem.org/Team:Freiburg/parts"> parts</A> side). This system was tested mainly in CHO-K1 and HEK-293T cells. 
			</P><P id="h3">
			Design of the crRNA plasmid:
			</P><P> 
			Use our crRNA design tool to design the crRNAs that are needed to target your gene of interest.
			</P><DIV align="center"><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#design_tool"><FONT size="5"><U>crRNA design tool</U></FONT></A></DIV><P>It will generate possible target sites and the appropriate oligos. Order the oligos by the company of your choice. We recommend to test several different loci to target your gene of interest because the efficiency of different crRNA-loci can differ.
			</P><OL><LI><B>Oligo annealing:</B> Anneal forward and reverse oligos to get the desired crRNA. Therefore mix 10 µl of 100 µM forward Oligo, 10 µl of 100 µM reverse Oligo and 80 µl of ddH2O. Heat the solution to 95° C for 5 minutes. Then turn off the heat block and let the solution cool down.</LI><LI><B>Digest plasmid BBa_K1150034 with Bbs1:</B> The restriction enzyme Bbs1 should always be stored at -80° C. Mix about 500 ng of BBa_K1150034 with 1 µl of Bbs1, appropriate amount of buffer and fill up to 50 µl with ddH2O. Digest for exactly 3 hours at 37° C. Load the digest on a gel and cut out the DNA band (2900 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate crRNAs (step 1) into Bbs1 cut backbone:</B> The insert (crRNAs) should be ligated into the backbone in 3 molar insert excess. Therefore use this formular: Required Volume of Insert = 3 x Volume(Backbone) x length(Insert) x concentration (Backbone) / [ length(Backbone)  x concentration(Insert) ]. Use about 50 ng of backbone. The length of insert is always 30 basepairs. The length of the backbone is 2900 basepairs. You have to mix the appropriate amount of Backbone and the appropriate amount of Insert with 1 µl of T4 ligase and 2 µl of 10xT4 ligase buffer. Then fill up to 20 µl with ddH2O. This mix should be incubated for 30 minutes at room temperature.</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI></OL><P> 
			Now that you have created the desired crRNA plasmids it is possible to use them indiviually or fuse different crRNA loci together into one crRNA plasmid (recommended).
			</P><P id="h3">
			Design of a multiple target crRNA plasmid:
			</P><P>
			It is shown that multiple targeting of one gene of interest <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#multiple_targeting">increases the efficiency of regulation</A>. If you want to fuse different crRNA loci together into one plasmid use the following protocol:
			</P><OL><LI><B>First of all digest crRNA plasmid</B> with PstI and SpeI in order to linearize it. Both enzymes cut in the suffix. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, it is possible to assemble multiple crRNA sequences into one plasmid. Therefore mix about 500 ng Backbone with 1 µl enzyme 1 and 1 µl enzyme 2, add an appropriate amount of compatible buffer and fill up to 30-50 µl. Incubate mix at 37° C for 2 hours. </LI><LI><B>Secondly digest crRNA plasmid</B> with PstI and XbaI. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, this is your insert (procedure see above).</LI><LI> Load digests on a gel and cut out the DNA bands (Backbone 2900 bp, Insert 870 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate</B> the insert in 3 molar excess into the backbone (formular see paragraph above).</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 forward sequencing primer (sequence: GAGTGCCACCTGACGTCTAAGAAAC) and pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI><LI> These steps can be repeated various times. Using this method, you can engineer a plasmid with several crRNA targets!</LI></OL><P id="h3">
			Experimental design (recommendation for mammalian cell culture):
			</P><OL><LI><B>Transfect</B> BBa_K1150022 (4 fold excess) with all desired crRNA plasmids (seperate crRNA plasmid and/or multiple crRNAs plasmid) </LI><LI><B>Non-effector control:</B> Transfect the appropriate crRNA - plasmid together with BBa_K1150017 that has no effector.</LI><LI><B>Off target control:</B> Transfect BBa_K1150022 without any crRNA plasmid.</LI></OL></DIV></DIV></DIV><DIV id="check-16" class="toHide3" style="display:none"><DIV id="final_checkbox_background"><DIV id="final_checkbox_content"><P id="h1">
			Red light inducible uniCAS Repressor (dCas9-PIF6 &amp; PhyB-KRAB devices)
			</P><P>
			You have chosen to repress a gene using a non inducible dCas9-KRAB device. Therefore you have to order the 

following plasmids from the <A id="link" href="http://parts.igem.org/Main_Page"> iGEM parts registry</A>. After receiving our 

plasmids, you will have to clone your target sequence into our crRNA plasmid (protocol see below).
			</P><TABLE><TBODY><TR><TH> No. </TH><TH> Biobrick </TH><TH> Device </TH><TH> Order </TH></TR><TR><TD> 1 </TD><TD> BBa_K1150025 </TD><TD> CMV:HA-NLS-dCas9-L3-PIF6-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150025"> 

order </A></TD></TR><TR><TD> 2 </TD><TD> BBa_K1150027 </TD><TD> CMV:NLS-PhyB-L3-KRAB-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150027"> 

order </A></TD></TR><TR><TD> 3 </TD><TD> BBa_K1150022 </TD><TD> CMV:HA-NLS-dCas9-L3-KRAB-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150022"> 

order </A></TD></TR><TR><TD> 4 </TD><TD> BBa_K1150034 </TD><TD> RNAimer - plasmid (crRNA - plasmid)</TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150034"> 

order </A></TD></TR></TBODY></TABLE><P>
			Note: 
			All plasmids are optimized for expression in mammalian systems. The devices are also available containing 

a SV40 instead of a CMV promotor (have a look at our <A id="link" href="https://2013.igem.org/Team:Freiburg/parts"> parts</A> 

side). This system was tested mainly in CHO-K1 and HEK-293T cells. 
			</P><P id="h3">
			Design of the crRNA plasmid:
			</P><P> 
			Use our crRNA design tool to design the crRNAs that are needed to target your gene of interest.
			</P><DIV align="center"><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#design_tool"><FONT size="5"><U>crRNA design tool</U></FONT></A></DIV><P>It will generate possible target sites and the appropriate oligos. Order the oligos by the company of 

your choice. We recommend to test several different loci to target your gene of interest because the efficiency of different 

crRNA-loci can differ.
			</P><OL><LI><B>Oligo annealing:</B> Anneal forward and reverse oligos to get the desired crRNA. Therefore 

mix 10 µl of 100 µM forward Oligo, 10 µl of 100 µM reverse Oligo and 80 µl of ddH2O. Heat the solution to 95° C for 5 minutes. 

Then turn off the heat block and let the solution cool down.</LI><LI><B>Digest plasmid BBa_K1150034 with Bbs1:</B> The restriction enzyme Bbs1 should always be 

stored at -80° C. Mix about 500 ng of BBa_K1150034 with 1 µl of Bbs1, appropriate amount of buffer and fill up to 50 µl with 

ddH2O. Digest for exactly 3 hours at 37° C. Load the digest on a gel and cut out the DNA band (2900 bp). Purify the gel slice and use 

DNA for the next step.</LI><LI><B>Ligate crRNAs (step 1) into Bbs1 cut backbone:</B> The insert (crRNAs) should be ligated 

into the backbone in 3 molar insert excess. Therefore use this formular: Required Volume of Insert = 3 x Volume(Backbone) x 

length(Insert) x concentration (Backbone) / [ length(Backbone)  x concentration(Insert) ]. Use about 50 ng of backbone. The length 

of insert is always 30 basepairs. The length of the backbone is 2900 basepairs. You have to mix the appropriate amount of 

Backbone and the appropriate amount of Insert with 1 µl of T4 ligase and 2 µl of 10xT4 ligase buffer. Then fill up to 20 µl with 

ddH2O. This mix should be incubated for 30 minutes at room temperature.</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the 

plasmids and sequence it with pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI></OL><P> 
			Now that you have created the desired crRNA plasmids it is possible to use them indiviually or fuse 

different crRNA loci together into one crRNA plasmid (recommended).
			</P><P id="h3">
			Design of a multiple target crRNA plasmid:
			</P><P>
			It is shown that multiple targeting of one gene of interest <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#multiple_targeting">increases the efficiency of regulation</A>. If you 

want to fuse different crRNA loci together into one plasmid use the following protocol:
			</P><OL><LI><B>First of all digest crRNA plasmid</B> with PstI and SpeI in order to linearize it. Both enzymes 

cut in the suffix. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, it 

is possible to assemble multiple crRNA sequences into one plasmid. Therefore mix about 500 ng Backbone with 1 µl enzyme 1 and 1 µl 

enzyme 2, add an appropriate amount of compatible buffer and fill up to 30-50 µl. Incubate mix at 37° C for 2 hours. 

</LI><LI><B>Secondly digest crRNA plasmid</B> with PstI and XbaI. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, this is your insert (procedure see 

above).</LI><LI>  Load digests on a gel and cut out the DNA bands (Backbone 2900 bp, Insert 870 bp). Purify 

the gel slice and use DNA for the next step.</LI><LI><B>Ligate</B> the insert in 3 molar excess into the backbone (formular see paragraph 

above).</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the 

plasmids and sequence it with pSB1C3 forward sequencing primer (sequence: GAGTGCCACCTGACGTCTAAGAAAC) and pSB1C3 reverse 

sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI><LI> These steps can be repeated various times. Using this method, you can engineer a plasmid 

with several crRNA targets!</LI></OL><P id="h3">
			Experimental design (recommendation for mammalian cell culture):
			</P><P>
			For red light experiments transfect the following plasmid combinations on two different well plates. One 

plate will be illuminated with 660 nm for 48 hours. This is the red light induced plate. The other plate will be illuminated with 

740 nm. This is the no-induction control. If not working in plant systems add 15µM of PCB prior to illumination and incubate for at least 1h. See our <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/induction#light">light induction project page</A>.
			</P><OL><LI><B>Transfect</B> BBa_K1150025, BBa_K1150027 (4 fold excess) with all desired crRNA plasmids 

(seperate crRNA plasmid and/or multiple crRNAs plasmid) </LI><LI><B>Off target control:</B> Transfect BBa_K1150025, BBa_K1150026 (4 fold excess) without any 

crRNA plasmid.</LI><LI><B>Target efficiency control:</B> Transfect BBa_K1150022 (4 fold excess) with the same crRNA 

Plasmids as in the first transfection.</LI></OL></DIV></DIV></DIV><DIV id="check-17" class="toHide3" style="display:none">
blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla
</DIV><DIV id="check-18" class="toHide3" style="display:none"><DIV id="final_checkbox_background"><DIV id="final_checkbox_content"><P id="h1">
			UVB light inducible uniCAS Repressor (dCas9-UVR8 &amp; COP1-KRAB devices)
			</P><P>
			You have chosen to repress a gene using a UVB light inducible dCas9-UVR8 &amp; COP1-KRAB device. Therefore you have to order the following plasmids from the <A id="link" href="http://parts.igem.org/Main_Page"> iGEM parts registry</A>. After receiving our plasmids, you will have to clone your target sequence into our crRNA plasmid (protocol see below).
			</P><TABLE><TBODY><TR><TH> No. </TH><TH> Biobrick </TH><TH> Device </TH><TH> Order </TH></TR><TR><TD> 1 </TD><TD> BBa_K1150029 </TD><TD> CMV:HA-NLS-dCas9-L3-UVR8-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150029"> order </A></TD></TR><TR><TD> 2 </TD><TD> BBa_K1150031 </TD><TD> CMV:NLS-COP1-L3-KRAB-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150031"> order </A></TD></TR><TR><TD> 3 </TD><TD> BBa_K1150022 </TD><TD> CMV:HA-NLS-dCas9-L3-KRAB-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150022"> order </A></TD></TR><TR><TD> 4 </TD><TD> BBa_K1150034 </TD><TD> RNAimer - plasmid (crRNA - plasmid)</TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150034"> order </A></TD></TR></TBODY></TABLE><P>
			Note: 
			All plasmids are optimized for expression in mammalian systems. The devices are also available containing an SV40 instead of an CMV promotor (have a look at our <A id="link" href="https://2013.igem.org/Team:Freiburg/parts"> parts</A> side). This system was tested mainly in CHO-K1 and HEK-293T cells. 
			</P><P id="h3">
			Design of the crRNA plasmid:
			</P><P> 
			Use our crRNA design tool to design the crRNAs that are needed to target your gene of interest.
			</P><DIV align="center"><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#design_tool"><FONT size="5"><U>crRNA design tool</U></FONT></A></DIV><P>It will generate possible target sites and the appropriate oligos. Order the oligos by the company of your choice. We recommend to test several different loci to target your gene of interest because the efficiency of different crRNA-loci can differ.
			</P><OL><LI><B>Oligo annealing:</B> Anneal forward and reverse oligos to get the desired crRNA. Therefore mix 10 µl of 100 µM forward Oligo, 10 µl of 100 µM reverse Oligo and 80 µl of ddH2O. Heat the solution to 95° C for 5 minutes. Then turn off the heat block and let the solution cool down.</LI><LI><B>Digest plasmid BBa_K1150034 with Bbs1:</B> The restriction enzyme Bbs1 should always be stored at -80° C. Mix about 500 ng of BBa_K1150034 with 1 µl of Bbs1, appropriate amount of buffer and fill up to 50 µl with ddH2O. Digest for exactly 3 hours at 37° C. Load the digest on a gel and cut out the DNA band (2900 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate crRNAs (step 1) into Bbs1 cut backbone:</B> The insert (crRNAs) should be ligated into the backbone in 3 molar insert excess. Therefore use this formular: Required Volume of Insert = 3 x Volume(Backbone) x length(Insert) x concentration (Backbone) / [ length(Backbone)  x concentration(Insert) ]. Use about 50 ng of backbone. The length of insert is always 30 basepairs. The length of the backbone is 2900 basepairs. You have to mix the appropriate amount of Backbone and the appropriate amount of Insert with 1 µl of T4 ligase and 2 µl of 10xT4 ligase buffer. Then fill up to 20 µl with ddH2O. This mix should be incubated for 30 minutes at room temperature.</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI></OL><P> 
			Now that you have created the desired crRNA plasmids it is possible to use them indiviually or fuse different crRNA loci together into one crRNA plasmid (recommended).
			</P><P id="h3">
			Design of a multiple target crRNA plasmid:
			</P><P>
			It is shown that multiple targeting of one gene of interest <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#multiple_targeting">increases the efficiency of regulation</A>. If you want to fuse different crRNA loci together into one plasmid use the following protocol:
			</P><OL><LI><B>First of all digest crRNA plasmid</B> with PstI and SpeI in order to linearize it. Both enzymes cut in the suffix. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, it is possible to assemble multiple crRNA sequences into one plasmid. Therefore mix about 500 ng Backbone with 1 µl enzyme 1 and 1 µl enzyme 2, add an appropriate amount of compatible buffer and fill up to 30-50 µl. Incubate mix at 37° C for 2 hours. </LI><LI><B>Secondly digest crRNA plasmid</B> with PstI and XbaI. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, this is your insert (procedure see above).</LI><LI>  Load digests on a gel and cut out the DNA bands (Backbone 2900 bp, Insert 870 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate</B> the insert in 3 molar excess into the backbone (formular see paragraph above).</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 forward sequencing primer (sequence: GAGTGCCACCTGACGTCTAAGAAAC) and pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI><LI> These steps can be repeated various times. Using this method, you can engineer a plasmid with several crRNA targets!</LI></OL><P id="h3">
			Experimental design (recommendation for mammalian cell culture):
			</P><P>
			For UVB light experiments transfect the following plasmid combinations on two different well plates. One plate will be illuminated with 311 nm at 5 µE for 24 hours. This is the UVB light induced plate. The other plate will be wrapped in aluminum foil. This is the no-induction control. See our <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/induction#light">light induction project page</A>.
			</P><OL><LI><B>Transfect</B> BBa_K1150029, BBa_K1150031 (4 fold excess) with all desired crRNA plasmids (seperate crRNA plasmid and/or multiple crRNAs plasmid) </LI><LI><B>Off target control:</B> Transfect BBa_K1150029, BBa_K1150031 (4 fold excess)without any crRNA plasmid.</LI><LI><B>Target efficiency control:</B> Transfect BBa_K1150022 (4 fold excess) with the same crRNA plasmids as in the first transfection.</LI></OL></DIV></DIV></DIV><DIV id="check-19" class="toHide3" style="display:none"><DIV id="final_checkbox_background"><DIV id="final_checkbox_content"><P id="h1">
			Non inducible uniCAS Histone modifier for Repression (dCas9-G9a device)
			</P><P>
			You have chosen to repress a gene using a non inducible dCas9-G9a device. Therefore you have to order the following plasmids from the <A id="link" href="http://parts.igem.org/Main_Page"> iGEM parts registry</A>. After receiving our plasmids, you will have to clone your target sequence into our crRNA plasmid (protocol see below).
			</P><TABLE><TBODY><TR><TH> No. </TH><TH> Biobrick </TH><TH> Device </TH><TH> Order </TH></TR><TR><TD> 1 </TD><TD> BBa_K1150017 </TD><TD> CMV:HA-NLS-dCas9-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150017"> order </A></TD></TR><TR><TD> 2 </TD><TD> BBa_K1150024 </TD><TD> CMV:HA-NLS-dCas9-L3-G9a-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150024"> order </A></TD></TR><TR><TD> 3 </TD><TD> BBa_K1150034 </TD><TD> RNAimer - plasmid (crRNA - plasmid)</TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150034"> order </A></TD></TR></TBODY></TABLE><P>
			Note: 
			All plasmids are optimized for expression in mammalian systems. The devices are also available containing an SV40 instead of an CMV promotor (have a look at our <A id="link" href="https://2013.igem.org/Team:Freiburg/parts"> parts</A> side). This system was tested mainly in CHO-K1 and HEK-293T cells. 
			</P><P id="h3">
			Design of the crRNA plasmid:
			</P><P> 
			Use our crRNA design tool to design the crRNAs that are needed to target your gene of interest.
			</P><DIV align="center"><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#design_tool"><FONT size="5"><U>crRNA design tool</U></FONT></A></DIV><P>It will generate possible target sites and the appropriate oligos. Order the oligos by the company of your choice. We recommend to test several different loci to target your gene of interest because the efficiency of different crRNA-loci can differ.
			</P><OL><LI><B>Oligo annealing:</B> Anneal forward and reverse oligos to get the desired crRNA. Therefore mix 10 µl of 100 µM forward Oligo, 10 µl of 100 µM reverse Oligo and 80 µl of ddH2O. Heat the solution to 95° C for 5 minutes. Then turn off the heat block and let the solution cool down.</LI><LI><B>Digest plasmid BBa_K1150034 with Bbs1:</B> The restriction enzyme Bbs1 should always be stored at -80° C. Mix about 500 ng of BBa_K1150034 with 1 µl of Bbs1, appropriate amount of buffer and fill up to 50 µl with ddH2O. Digest for exactly 3 hours at 37° C. Load the digest on a gel and cut out the DNA band (2900 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate crRNAs (step 1) into Bbs1 cut backbone:</B> The insert (crRNAs) should be ligated into the backbone in 3 molar insert excess. Therefore use this formular: Required Volume of Insert = 3 x Volume(Backbone) x length(Insert) x concentration (Backbone) / [ length(Backbone)  x concentration(Insert) ]. Use about 50 ng of backbone. The length of insert is always 30 basepairs. The length of the backbone is 2900 basepairs. You have to mix the appropriate amount of Backbone and the appropriate amount of Insert with 1 µl of T4 ligase and 2 µl of 10xT4 ligase buffer. Then fill up to 20 µl with ddH2O. This mix should be incubated for 30 minutes at room temperature.</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI></OL><P> 
			Now that you have created the desired crRNA plasmids it is possible to use them indiviually or fuse different crRNA loci together into one crRNA plasmid (recommended).
			</P><P id="h3">
			Design of a multiple target crRNA plasmid:
			</P><P>
			It is shown that multiple targeting of one gene of interest <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#multiple_targeting">increases the efficiency of regulation</A>. If you want to fuse different crRNA loci together into one plasmid use the following protocol:
			</P><OL><LI><B>First of all digest crRNA plasmid</B> with PstI and SpeI in order to linearize it. Both enzymes cut in the suffix. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, it is possible to assemble multiple crRNA sequences into one plasmid. Therefore mix about 500 ng Backbone with 1 µl enzyme 1 and 1 µl enzyme 2, add an appropriate amount of compatible buffer and fill up to 30-50 µl. Incubate mix at 37° C for 2 hours. </LI><LI><B>Secondly digest crRNA plasmid</B> with PstI and XbaI. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, this is your insert (procedure see above).</LI><LI>  Load digests on a gel and cut out the DNA bands (Backbone 2900 bp, Insert 870 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate</B> the insert in 3 molar excess into the backbone (formular see paragraph above).</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 forward sequencing primer (sequence: GAGTGCCACCTGACGTCTAAGAAAC) and pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI><LI> These steps can be repeated various times. Using this method, you can engineer a plasmid with several crRNA targets!</LI></OL><P id="h3">
			Experimental design (recommendation for mammalian cell culture):
			</P><OL><LI><B>Transfect</B> BBa_K1150024 with all desired crRNA plasmids (seperate crRNA plasmid and/or multiple crRNAs plasmid) </LI><LI><B>Non-effector control:</B> Transfect the appropriate crRNA - plasmid together with BBa_K1150017 that has no effector.</LI><LI><B>Off target control:</B> Transfect BBa_K1150024 without any crRNA plasmid.</LI></OL></DIV></DIV></DIV><DIV id="check-20" class="toHide3" style="display:none"><DIV id="final_checkbox_background"><DIV id="final_checkbox_content"><P id="h1">
			Red light inducible uniCAS Histone modifier for Repression (dCas9-PIF6 &amp; PhyB-G9a devices)
			</P><P>
			You have chosen to repress a gene using a non inducible dCas9-KRAB device. Therefore you have to order the following plasmids from the <A id="link" href="http://parts.igem.org/Main_Page"> iGEM parts registry</A>. After receiving our plasmids, you will have to clone your target sequence into our crRNA plasmid (protocol see below).
			</P><TABLE><TBODY><TR><TH> No. </TH><TH> Biobrick </TH><TH> Device </TH><TH> Order </TH></TR><TR><TD> 1 </TD><TD> BBa_K1150025 </TD><TD> CMV:HA-NLS-dCas9-L3-PIF6-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150025"> order </A></TD></TR><TR><TD> 2 </TD><TD> BBa_K1150028 </TD><TD> CMV:NLS-PhyB-L3-G9a-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150028"> order </A></TD></TR><TR><TD> 3 </TD><TD> BBa_K1150024 </TD><TD> CMV:HA-NLS-dCas9-L3-G9a-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150024"> order </A></TD></TR><TR><TD> 4 </TD><TD> BBa_K1150034 </TD><TD> RNAimer - plasmid (crRNA - plasmid)</TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150034"> order </A></TD></TR></TBODY></TABLE><P>
			Note: 
			All plasmids are optimized for expression in mammalian systems. The devices are also available containing an SV40 instead of an CMV promotor (have a look at our <A id="link" href="https://2013.igem.org/Team:Freiburg/parts"> parts</A> side). This system was tested mainly in CHO-K1 and HEK-293T cells. 
			</P><P id="h3">
			Design of the crRNA plasmid:
			</P><P> 
			Use our crRNA design tool to design the crRNAs that are needed to target your gene of interest.
			</P><DIV align="center"><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#design_tool"><FONT size="5"><U>crRNA design tool</U></FONT></A></DIV><P>It will generate possible target sites and the appropriate oligos. Order the oligos by the company of your choice. We recommend to test several different loci to target your gene of interest because the efficiency of different crRNA-loci can differ.
			</P><OL><LI><B>Oligo annealing:</B> Anneal forward and reverse oligos to get the desired crRNA. Therefore mix 10 µl of 100 µM forward Oligo, 10 µl of 100 µM reverse Oligo and 80 µl of ddH2O. Heat the solution to 95° C for 5 minutes. Then turn off the heat block and let the solution cool down.</LI><LI><B>Digest plasmid BBa_K1150034 with Bbs1:</B> The restriction enzyme Bbs1 should always be stored at -80° C. Mix about 500 ng of BBa_K1150034 with 1 µl of Bbs1, appropriate amount of buffer and fill up to 50 µl with ddH2O. Digest for exactly 3 hours at 37° C. Load the digest on a gel and cut out the DNA band (2900 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate crRNAs (step 1) into Bbs1 cut backbone:</B> The insert (crRNAs) should be ligated into the backbone in 3 molar insert excess. Therefore use this formular: Required Volume of Insert = 3 x Volume(Backbone) x length(Insert) x concentration (Backbone) / [ length(Backbone)  x concentration(Insert) ]. Use about 50 ng of backbone. The length of insert is always 30 basepairs. The length of the backbone is 2900 basepairs. You have to mix the appropriate amount of Backbone and the appropriate amount of Insert with 1 µl of T4 ligase and 2 µl of 10xT4 ligase buffer. Then fill up to 20 µl with ddH2O. This mix should be incubated for 30 minutes at room temperature.</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI></OL><P> 
			Now that you have created the desired crRNA plasmids it is possible to use them indiviually or fuse different crRNA loci together into one crRNA plasmid (recommended).
			</P><P id="h3">
			Design of a multiple target crRNA plasmid:
			</P><P>
			It is shown that multiple targeting of one gene of interest <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#multiple_targeting">increases the efficiency of regulation</A>. If you want to fuse different crRNA loci together into one plasmid use the following protocol:
			</P><OL><LI><B>First of all digest crRNA plasmid</B> with PstI and SpeI in order to linearize it. Both enzymes cut in the suffix. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, it is possible to assemble multiple crRNA sequences into one plasmid. Therefore mix about 500 ng Backbone with 1 µl enzyme 1 and 1 µl enzyme 2, add an appropriate amount of compatible buffer and fill up to 30-50 µl. Incubate mix at 37° C for 2 hours. </LI><LI><B>Secondly digest crRNA plasmid</B> with PstI and XbaI. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, this is your insert (procedure see above).</LI><LI>  Load digests on a gel and cut out the DNA bands out (Backbone 2900 bp, Insert 870 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate</B> the insert in 3 molar excess into the backbone (formular see paragraph above).</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 forward sequencing primer (sequence: GAGTGCCACCTGACGTCTAAGAAAC) and pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI><LI> These steps can be repeated various times. Using this method, you can engineer a plasmid with several crRNA targets!</LI></OL><P id="h3">
			Experimental design (recommendation for mammalian cell culture):
			</P><P>
			For red light experiments transfect the following plasmid combinations on two different well plates. One plate will be illuminated with 660 nm for 48 hours. This is the red light induced plate. The other plate will be illuminated with 740 nm. This is the no-induction control. If not working in plant systems add 15µM of PCB prior to illumination and incubate for at least 1h. See our <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/induction#light">light induction project page</A>.
			</P><OL><LI><B>Transfect</B> BBa_K1150025, BBa_K1150028 with all desired crRNA plasmids (seperate crRNA plasmid and/or multiple crRNAs plasmid) </LI><LI><B>Off target control:</B> Transfect BBa_K1150025, BBa_K1150028 without any crRNA plasmid.</LI><LI><B>Target efficiency control:</B> Transfect BBa_K1150024 with the same crRNA plasmids as in the first transfection.</LI></OL></DIV></DIV></DIV><DIV id="check-21" class="toHide3" style="display:none">
blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla blabla
</DIV><DIV id="check-22" class="toHide3" style="display:none"><DIV id="final_checkbox_background"><DIV id="final_checkbox_content"><P id="h1">
			UVB light inducible uniCAS Histone modifier for Repression (dCas9-UVR8 &amp; COP1-G9a devices)
			</P><P>
			You have chosen to repress a gene using a UVB light inducible dCas9-UVR8 &amp; COP1-G9a device. Therefore you have to order the following plasmids from the <A id="link" href="http://parts.igem.org/Main_Page"> iGEM parts registry </A>. After receiving our plasmids, you will have to clone your target sequence into our crRNA plasmid (protocol see below).
			</P><TABLE><TBODY><TR><TH> No. </TH><TH> Biobrick </TH><TH> Device </TH><TH> Order </TH></TR><TR><TD> 1 </TD><TD> BBa_K1150029 </TD><TD> CMV:HA-NLS-dCas9-L3-UVR8-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150029"> order </A></TD></TR><TR><TD> 3 </TD><TD> BBa_K1150022 </TD><TD> CMV:HA-NLS-dCas9-L3-G9a-NLS:BGH </TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150022"> order </A></TD></TR><TR><TD> 4 </TD><TD> BBa_K1150034 </TD><TD> RNAimer - plasmid (crRNA - plasmid)</TD><TD><A id="link" href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K1150034"> order </A></TD></TR></TBODY></TABLE><P>
			Note: 
			All plasmids are optimized for expression in mammalian systems. The devices are also available containing an SV40 instead of an CMV promotor (have a look at our <A id="link" href="https://2013.igem.org/Team:Freiburg/parts"> parts</A> side). This system was tested mainly in CHO-K1 and HEK-293T cells. 
			</P><P id="h3">
			Design of the crRNA plasmid:
			</P><P> 
			Use our crRNA design tool to design the crRNAs that are needed to target your gene of interest.
			</P><DIV align="center"><A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#design_tool"><FONT size="5"><U>crRNA design tool</U></FONT></A></DIV><P>It will generate possible target sites and the appropriate oligos. Order the oligos by the company of your choice. We recommend to test several different loci to target your gene of interest because the efficiency of different crRNA-loci can differ.
			</P><OL><LI><B>Oligo annealing:</B> Anneal forward and reverse oligos to get the desired crRNA. Therefore mix 10 µl of 100 µM forward Oligo, 10 µl of 100 µM reverse Oligo and 80 µl of ddH2O. Heat the solution to 95° C for 5 minutes. Then turn off the heat block and let the solution cool down.</LI><LI><B>Digest plasmid BBa_K1150034 with Bbs1:</B> The restriction enzyme Bbs1 should always be stored at -80° C. Mix about 500 ng of BBa_K1150034 with 1 µl of Bbs1, appropriate amount of buffer and fill up to 50 µl with ddH2O. Digest for exactly 3 hours at 37° C. Load the digest on a gel and cut out the DNA band (2900 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate crRNAs (step 1) into Bbs1 cut backbone:</B> The insert (crRNAs) should be ligated into the backbone in 3 molar insert excess. Therefore use this formular: Required Volume of Insert = 3 x Volume(Backbone) x length(Insert) x concentration (Backbone) / [ length(Backbone)  x concentration(Insert) ]. Use about 50 ng of backbone. The length of insert is always 30 basepairs. The length of the backbone is 2900 basepairs. You have to mix the appropriate amount of Backbone and the appropriate amount of Insert with 1 µl of T4 ligase and 2 µl of 10xT4 ligase buffer. Then fill up to 20 µl with ddH2O. This mix should be incubated for 30 minutes at room temperature.</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI></OL><P> 
			Now that you have created the desired crRNA plasmids it is possible to use them indiviually or fuse different crRNA loci together into one crRNA plasmid (recommended).
			</P><P id="h3">
			Design of a multiple target crRNA plasmid:
			</P><P>
			It is shown that multiple targeting of one gene of interest <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/crrna#multiple_targeting">increases the efficiency of regulation</A>. If you want to fuse different crRNA loci together into one plasmid use the following protocol:
			</P><OL><LI><B>First of all digest crRNA plasmid</B> with PstI and SpeI in order to linearize it. Both enzymes cut in the suffix. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, it is possible to assemble multiple crRNA sequences into one plasmid. Therefore mix about 500 ng Backbone with 1 µl enzyme 1 and 1 µl enzyme 2, add an appropriate amount of compatible buffer and fill up to 30-50 µl. Incubate mix at 37° C for 2 hours. </LI><LI><B>Secondly digest crRNA plasmid</B> with PstI and XbaI. Using the <A id="link" href="http://parts.igem.org/Help:Assembly/3A_Assembly">BioBrick Assembly method</A>, this is your insert (procedure see above).</LI><LI>  Load digests on a gel and cut out the DNA bands (Backbone 2900 bp, Insert 870 bp). Purify the gel slice and use DNA for the next step.</LI><LI><B>Ligate</B> the insert in 3 molar excess into the backbone (formular see paragraph above).</LI><LI><B>Transform</B> 3-5 µl of the mix following <A id="link" href="https://2013.igem.org/Team:Freiburg/protocols#Transformation">standard protocol</A>. Pick clones, miniprep the plasmids and sequence it with pSB1C3 forward sequencing primer (sequence: GAGTGCCACCTGACGTCTAAGAAAC) and pSB1C3 reverse sequencing primer (sequence: CGCCTTTGAGTGAGCTGATACCGC).</LI><LI> These steps can be repeated various times. Using this method, you can engineer a plasmid with several crRNA targets!</LI></OL><P id="h3">
			Experimental design (recommendation for mammalian cell culture):
			</P><P>
			For UVB light experiments transfect the following plasmid combinations on two different well plates. One plate will be illuminated with 311 nm at 5 µE for 24 hours. This is the UVB light induced plate. The other plate will be wrapped in aluminum foil. This is the no-induction control. See our <A id="link" href="https://2013.igem.org/Team:Freiburg/Project/induction#light">light induction project page</A>.
			</P><OL><LI><B>Transfect</B> BBa_K1150029, BBa_K1150032 with all desired crRNA plasmids (seperate crRNA plasmid and/or multiple crRNAs plasmid) </LI><LI><B>Off target control:</B> Transfect BBa_K1150029, BBa_K1150032 (4 fold excess)without any crRNA plasmid.</LI><LI><B>Target efficiency control:</B> Transfect BBa_K1150024 with the same crRNA plasmids as in the first transfection.</LI></OL></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Freiburg/Project/toolkit">http://2013.igem.org/Team:Freiburg/Project/toolkit</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Project/toolkit" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Project/toolkit" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Project/toolkit&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Project/toolkit&amp;oldid=356360" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>