<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Transmembrane_Proteins skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Transmembrane Proteins</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV class="row top-menu"><DIV class="col-md-8"><UL class=" text-left wtf-menu "><LI class="parent main-item"><A href="#">Project  </A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Overview">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Transmembrane_Proteins">Membrane Photoreceptor</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Proteins_Caging">Protein Caging</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Logic_Circuit">Logic Circuit</A></LI><LI class="parent"><A href="https://2017.igem.org/Team:Paris_Bettencourt/Biomaterials">Biomaterials</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Biomaterials#header2">PHA</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Biomaterials#header3">Calcium Carbonate</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Biomaterials#header4">Polysilicate</A></LI></UL><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/RNA">RNA</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Hardware_setup">Gel Optics</A></LI><LI class="parent main-item"><A href="https://2017.igem.org/Team:Paris_Bettencourt/Model">Modeling  </A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Model#firstmodel">Gel Optics</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Model#secondmodel">RNA</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Model#thirdmodel">Logic Circuit</A></LI><LI class="parent main-item"><A href="#">Achievements  </A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Medal_Criteria">Medal Criteria</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Contribution">Contributions</A></LI><LI class="parent main-item"><A href="#">Human Practices  </A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/HP/Gold_Integrated">Gold &amp; Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Safety">Safety</A></LI><LI class="parent main-item"><A href="https://2017.igem.org/Team:Paris_Bettencourt/Lab_Notebook">Lab Book</A></LI><LI class="parent main-item"><A href="#">Parts  </A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Improve">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Basic_Part">Basics Parts</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Part_Collection">Part Collection</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI class="parent main-item"><A href="#">Team  </A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Sponsors">Sponsors</A></LI><DIV id="logolist2"><DIV class="sidelogos top5"><DIV id="twitapp"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV></NAV><DIV id="header1" class="header">MEMBRANE PHOTORECEPTOR</DIV><DIV class="textbody"><H1>Introduction</H1><DIV class="text2"><DIV class="text2left">Light is one of the crucial determinants of the environment and as such, many naturally occurring organisms, including microorganisms , have evolved mechanisms to respond to light. This is done through photoreceptor proteins embedded in the cell membrane, that either change conformity or energy state when exposed to specific wavelengths to then induce a signalling cascade within the cell. </DIV><DIV class="text2right"> With the advent of genetic engineering, more specifically synthetic biology, scientists have been able to harness the power of these membrane photosensors and create powerful optogenetic tools that are used in a range of applications. Additionally, protein engineering have increased the sensitivity and portability of these enzymes, resulting in accurate modular systems.</DIV></DIV><H1>State of the art and inspiration</H1><DIV class="text2"><DIV class="text2left">These optogenetic tools have become vital in Synthetic biology, and have been used to construct genetic circuits, used for both fundamental and applicative research(Toettcher, 2011). Most recently, these proteins have been used to create logical circuits, allowing for control of expression of cells. </DIV><DIV class="text2right">The Chris Voigt Lab published a paper (Fernandez-Rodriguez et al., 2017) where they incorporate photoreceptor proteins into a circuit which when exposed to different light wavelengths produce specific coloured pigments in response to one corresponding light. Their system was used to express, red (gusA),  blue (lacZ) and green (bFMO) , proteins in response to red blue and green light respectively. (figure 1) </DIV></DIV><DIV class="text1"><SPAN><B>Figure 1</B>Three light systems leading to red, green and blue pigments allowing for images to be produced.</SPAN></DIV><H1> Aims </H1><DIV class="text6">In our design, we aim to activate the production of a biomaterial through an AND gate. For our design we used the multi-enzyme pathway formation of PHB. 
				<UL><LI>Create and AND gate using two different light wavelengths</LI><LI>See production of a biomaterial when activated at the specific location where two lights are shone. </LI><LI>Create a modular desing that can be used with different biomaterials.</LI></UL></DIV><H1>Design </H1><DIV class="text6"> We used two wavelengths of light to activate gene expression: red and blue. The photosensors we used are previously characterised parts from the iGEM registry.  </DIV><DIV class="text2"><DIV class="text2left"><H6>SENSING RED LIGHT</H6> 
The Cph8 photoreceptor system was used as a red light sensing module. This system contains the Cph8 fusion protein created with the photoreceptor Cph1, originating from <I> Cyanobacteria</I>, and the EnvZ histidine kinase native in <I>E.coli</I>(Levaskaya et al., 2005). The light sensing unit is connected to transcription via the EnvZ-OmpR signalling pathway, with phosphorylated OmpR acting as a transcription factor.  <B>When exposed to Red light, the protein is inactivated and the signal cascade is repressed.</B></DIV><DIV class="text2right"><H6>SENSING BLUE LIGHT </H6>
The YF1/FixJ photo-sensing system was used as a blue light sensing module. This system contains a fusion protein consisting of the YF1 protein is a fusion protein consisting of the LOV blue light sensor domain of <I>Bacillus subtilis</I>  and the heme-binding PAS sensor domain of FixL from<I> Bradyrhizobium japonicum</I>(Möglich, A et al, 2009) .  The light sensing domain is connected to transcription via the FixL/FixJ signalling pathway, with phosphorylated FixJ acting as a transcription factor.<B>When exposed to blue light, the protein is inactivated and the signal cascade is repressed.</B></DIV></DIV><DIV class="text6"> These two light sensors were used to activate two repressors: phIF by blue light and CI by the red light that then act on components that complete the T7 RNAP core, which in turn act to allow production of the genes of interest. 4 plasmids were used to express the system: <TABLE><TBODY><TR><TH>Role in circuit</TH><TH>Components</TH><TH>Backbone (resistance, ori)</TH></TR><TR><TD>Plasmid One</TD><TD> Photoreceptors: YF1/FixL and Cph8</TD><TD>YF1/FixL and Cph8</TD><TD>CamR, colE1</TD></TR><TR><TD>Plasmid Two</TD><TD> Repressor activated by YF1/FixL: phIF</TD><TD>phIF repressorin front of FixJ promotor.</TD><TD>SpecR, p15A</TD></TR><TR><TD>Plasmid Three</TD><TD>Repressor activated by Cph8: CI</TD><TD>CI repressor in front of OmpR promotor.photobilins: ho1 and pcyA</TD><TD>TrimR, incW</TD></TR><TR><TD>Plasmid Four</TD><TD>Genes of interest (Biomaterials),</TD><TD>PhaA and PhaB under K1F T7 RNAP promoterm and PhaC under T3 Promoter</TD><TD>AmpR, pSC101</TD></TR></TBODY></TABLE></DIV><DIV class="text2"><DIV class="text2left">

Our genes of interest were the enzymes responsible or production of PHB. Production of PHB depends a multi enzyme pathway;  phaA, phaB and phaC. All three enzymes are necessary for production of the biomaterial PH3B. The wild-type gene, originating in <I>Ralstonia eutropha</I>,  is under the control of a single promoter, which when activated, ensure production of all three enzymes and thus the biomaterial (Fig 2. A). The production and the gene have been throughly characterised in previous research. </DIV><DIV class="text2right">
By splitting up activation of the different enzymes using two promoters (Fig 2. B) in our pathway induced by light, we show that it is possible to create an AND gate. Not only does this have many applications, such as in our bio-printer but also allows a method to study the individual enzymes within a pathway. </DIV></DIV><DIV class="text6"><SPAN><B>Figure 2</B> A. Biobrick of wildtype PHA production gene, with enzymes phaA, phaB and phaC under one promoter. B. Biobrick of our composite part with phaA and phaB under T3 promoter control and phaC under KIF control</SPAN></DIV><H1> Results </H1><DIV class="text2"><DIV class="text2left"><H4>Testing with Fluorescent Reporters</H4>
We first tested the functionality by replicating Voigt's papers work. In this test, the circuit expresses reporter genes: BFP for blue light activation and RFP for red light activation.  The cells were co-transformed with 4 plasmids, one with the light sensors, two with the repressors and T7 RNAP resource allocator and a final one with the reporter genes under T7 promoters. The transformed cells were exposed to light, using our DIY Arduino controlled LED light device for eight hours at log phase. These cells were consequently analysed using Flow Cytometry, allowing single cell data to be obtained. 

 
Although we observed activation of the specific fluorescence corresponding to the light input, the difference was not very large and no significance was found in statistical analysis. We also found lack of specificity in response to different light signals, as heightened RFP expression was observed under blue light and heightened BFP expression was observed under red light (Figure 3) </DIV><DIV class="text2right"><SPAN><B>Figure 3</B>  A. RFP levels detected by Flow cytometry in cells transformed with the 4 plasmids exposed to red light, blue light and no lights.  Controls are cells without any plasmids transformed exposed to red light, blue light and no lights. B. BFP levels detected by Flow cytometry in cells transformed with the 4 plasmids exposed to red light, blue light and no lights. Controls are cells without any plasmids transformed exposed to red light, blue light and no lights </SPAN></DIV></DIV><DIV class="text2"><DIV class="text2left"><TSPAN><B>Figure 4</B>  A. Cells exposed to both red and blue light B. Cells exposed to one light (red) C. Cells exposed to no lights </TSPAN></DIV><DIV class="text2right"><H4> Testing with PHA </H4> 
Once we tested the logic gate circuitry, we transformed our composite PHB control plasmid with the light sensing and circuitry plasmids. The cells were plated on Nile Red plates, a fluorescent dye for PHB, and exposed to light for 8 hours. These plates were then imaged and contrasted (Fig 4). 
Although large and software detectable amounts of fluorescence were seen in plates exposed to both lights, there was high background expression. Even when grown under one light or no lights, fluorescence was observed at a single cell level. An analysis of the fluorescence showed no significant difference in the fluorescence.  </DIV></DIV><H1> Perspectives and Design Considerations </H1><DIV class="text3"> One of the main problems we encountered with out design was the high metabolic cost to the cell. The photoreceptors are expressed constitutively and also require additional genes for production of photobilins for  their production. This resulted in a large decrease of the growth rate when co-transformed (Fig 5.) We found a significant decrease in the growth rate when 3 plasmids were transformed and when 4 plasmids were transformed. (Figure 5)  </DIV><DIV class="text3"><TSPAN><B> Figure 5</B>A. Growth curves of strains transformed with two plasmids,
 three plasmids and four plasmids and growth of cell strain with no plasmids transformed. B.Maximal Growth rate of the 4 strains. A significant difference was found in growth rate between 3 plasmids and 3 plasmids (p=0.0006) as well as between 2 plasmids and 3 plasmids (p=0.0033)</TSPAN></DIV><DIV class="text2"><DIV class="text2left"> Moreover, the system is technically difficult to implement, requiring multiple different antibiotic resistances, and multiple transformations, as transformation with four plasmids has very low success. </DIV><DIV class="text2right"> Finally, the system that we designed has low modularity as the AND gate can be executed with a multi-enzyme pathway. A single enzyme pathway, such as the one we used in the production of Calcium Carbonate, could not be implemented in our circuit. It is for this reason that we also propose another optogenetic tool - <B><HREF>Protein Photocaging. </HREF></B></DIV></DIV><H1>References</H1><LI>
Toettcher, J.E., Voigt, C.A., Weiner, O.D. and Lim, W.A., 2011. The promise of optogenetics in cell biology: interrogating molecular circuits in space and time. Nature methods, 8(1), pp.35-38. 
       </LI><LI>
Fernandez-Rodriguez, J., Moser, F., Song, M. and Voigt, C.A., 2017. Engineering RGB color vision into Escherichia coli. Nature Chemical Biology.
        </LI><LI>
Levskaya, A., Chevalier, A.A., Tabor, J.J., Simpson, Z.B., Lavery, L.A., Levy, M., Davidson, E.A., Scouras, A., Ellington, A.D., Marcotte, E.M. and Voigt, C.A., 2005. Synthetic biology: engineering Escherichia coli to see light. Nature, 438(7067), pp.441-442. 
         </LI><LI>
Möglich, A., Ayers, R.A. and Moffat, K., 2009. Design and signaling mechanism of light-regulated histidine kinases. Journal of molecular biology, 385(5), pp.1433-1444.</LI><LI>
Toettcher, J.E., Voigt, C.A., Weiner, O.D. and Lim, W.A., 2011. The promise of optogenetics in cell biology: interrogating molecular circuits in space and time. Nature methods, 8(1), pp.35-38.
         </LI></DIV><DIV id="footerdiv"><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France<A href="mailto:bettencourt.igem2017@gmail.com">bettencourt.igem2017@gmail.com</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>