<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV class="row top-menu"><DIV class="col-md-8"><UL class=" text-left wtf-menu "><LI class="parent main-item"><A href="#">Project  </A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Overview">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Transmembrane_Proteins">Membrane Photoreceptor</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Proteins_Caging">Protein Caging</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Logic_Circuit">Logic Circuit</A></LI><LI class="parent"><A href="https://2017.igem.org/Team:Paris_Bettencourt/Biomaterials">Biomaterials</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Biomaterials#header2">PHA</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Biomaterials#header3">Calcium Carbonate</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Biomaterials#header4">Polysilicate</A></LI></UL><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/RNA">RNA</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Hardware_setup">Gel Optics</A></LI><LI class="parent main-item"><A href="https://2017.igem.org/Team:Paris_Bettencourt/Model">Modeling  </A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Model#firstmodel">Gel Optics</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Model#secondmodel">RNA</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Model#thirdmodel">Logic Circuit</A></LI><LI class="parent main-item"><A href="#">Achievements  </A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Medal_Criteria">Medal Criteria</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Contribution">Contributions</A></LI><LI class="parent main-item"><A href="#">Human Practices  </A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/HP/Gold_Integrated">Gold &amp; Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Safety">Safety</A></LI><LI class="parent main-item"><A href="https://2017.igem.org/Team:Paris_Bettencourt/Lab_Notebook">Lab Book</A></LI><LI class="parent main-item"><A href="#">Parts  </A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Improve">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Basic_Part">Basics Parts</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Part_Collection">Part Collection</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI class="parent main-item"><A href="#">Team  </A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Paris_Bettencourt/Sponsors">Sponsors</A></LI><DIV id="logolist2"><DIV class="sidelogos top5"><DIV id="twitapp"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV></NAV><DIV id="header1" class="header">MODELLING</DIV><DIV class="textbody"><DIV class="line"><A href="#firstmodel" class="buttons">Gel optics modelling</A><A href="#secondmodel" class="buttons">RNA organelle modelling</A><A href="#thirdmodel" class="buttons">Logic circuit modelling</A></DIV></DIV><DIV class="textbody"><DIV class="text2"><H1 id="firstmodel"> Gel Optics Modelling </H1><DIV class="text2left">
			
Characterizing the light signal inside the gel medium in which our bacteria grow is necessary to predict the performance of our optic control system. 
We developed a strategy to measure the light intensity landscape created by a laser going through a gel (See <A href="https://2017.igem.org/Team:Paris_Bettencourt/Hardware_setup#Fitting_Interpretation"><U>Gel Optics page </U></A>) . We described this intensity landscape by fitting its absorption and scattering. Since the scattering observed was low, we tested whether we could predict the intensity landscape in a given gel (0.5% Gelrite with LB media) based on a simple cuvette absorbance reading. </DIV><DIV class="text2right">
The absorbance readings were used to find [ε*C], the parameter which governs the logarithmic decrease of the laser (see <A href="https://2017.igem.org/Team:Paris_Bettencourt/Hardware_setup#Equation_Table"><U>Table 2</U></A> on Gel Optics page). 
We assumed the light scattering in 0.5% Gelrite to be the same as in 1% Gelrite. This is an oversimplification but predicting the scattering of a gel based on its concentration would make it necessary to acquire light diffusion data for a range of gel concentrations which was outside the scope of our model development. 
We observed that our predicted light profiles were close to the fit of the intensity data recorded. This suggest that the intensity profile of a laser in the gel can be characterize from a single absorbance reading.
</DIV></DIV><SPAN>Comparison Between The Fitted Data Profiles (Left 4 Graphs) And The Predicted Profiles (4 Right Gaphs) for 0.5% Gelrite</SPAN><DIV class="text2"><DIV class="text2left"><H3> Laser Beam Shape Modelling</H3>
To predict and visualize the volume of gel in which the immobilized bacteria would be activated, we modeled the 3D shapes of the laser in the different gels. We assumed that bacteria would be sharply activated in an ON/OFF manner when light reached an intensity threshold equal to 80% the intensity used in the optogenetic work of Fernandez-Rodriguez et al. This allowed us to predict the volume activated by our two intersecting laser at any point within the three gel characterized. 
At 6cm depth, the activating volume was of the order of a few mm<SUP>2</SUP> for the 1% gels. This is very promising as it suggests that a sub-mm<SUP>2</SUP> resolution could be obtained with lasers having a smaller beam diameter in less concentrated gels.  
</DIV><DIV class="text3"><DIV class="text3left"><DIV class="container"><VIDEO loop="" id="video1" width="160%" autoplay=""><SPAN> Modeled Laser Shapes in 1% Alginate </SPAN></VIDEO></DIV></DIV><DIV class="text3center"><DIV class="container"><VIDEO loop="" id="glass" class="videocenter" width="170%" autoplay=""><SPAN> Modeled Laser Shapes in 1% Gelrite </SPAN></VIDEO></DIV></DIV><DIV class="text3right"><DIV class="container"><VIDEO loop="" id="glass" class="videocenter" width="170%" autoplay=""><SPAN> Modeled Laser Shapes in 1% Agar </SPAN></VIDEO></DIV></DIV></DIV></DIV></DIV><DIV class="textbody"><H1 id="secondmodel">RNA </H1><H4> Introduction </H4><DIV class="text2"><DIV class="text2left"> Creating a microenvironment in a chassis is a groundbreaking way to allow the implementation of foreign introduced pathway to another species. Previous studies have observed that liquid phase separation occurs in mammalian cells, under the production of mutated RNA sequences with tri-nucleotide repeats - such as CAG. These RNA strands agglomerate to form a membrane-less organelle.
            </DIV><DIV class="text2right"> Therefore Medusa aimed at reproducing this phenomenon in a chosen chassis cell by designing synthetic RNA sequences made up of CAG repeats to different lengths. In first modeling, we characterised the organelle formation and kinetics, which in turn influenced our experimental design. In the second part, we did mathematical analysis based on reaction-diffusion system and revealed by axiomatic reduction that the membrane-less organelle, as we expect, could abolish the unwanted reactions and enhance the desired reactions, under the parameter constraints of real biological conditions.
             </DIV></DIV><H4> Liquid phase Separation - Flory Huggins lattice modeling</H4><DIV class="text1"> We wanted to first characterise the RNA organelles under different repeat lengths to ensure liquid phase separation. This was done through applying the Flory Huggins solution theory, which is a mathematical model of the thermodynamics involved in polymer-solvent solutions, in our case- water and RNA (Brangwynne et al. 2015).  

Where n1, and n2 define the concentration and Φ1 and Φ2 define the volume fraction of the water and the RNA ‘polymer’ respectively. The parameter χ12 is the interaction parameter defined as such:    

Where z is the lattice space interaction, k is the Boltzmann constant and T is the temperature. The interaction parameters are defined as: w12 solvent-polymer, w11 solvent-solvent, w22 polymer polymer.  

The interaction parameter in the Flory-Huggins Solution specifies whether phase separation occurs in the mixture.  

Therefore we tested this with various different length of RNA repeats, whose interaction parameters calculated through structural modelling of the RNA folding and cofolding using NuPack and cross referenced with SimRNA data (which was tested with the help of the IONIS iGEM team.)

<TSPAN><B>Figure 1</B> The free energies of the structures of co-folded RNA strands were computed, for strands ranging between 5 to 70 repeats. Each structure contains a cooled of 1 strand to 10 strands. </TSPAN></DIV><DIV class="text2"><DIV class="text2left"> This structural modelling demonstrated that the longer the repeat, the stronger the monomer- monomer interaction, which would be favourable for RNa organelle formation. 

The χ12 was calculated for each of the lengths, between 5 repeats and 70, to better understand the properties of the RNA strands. If the χ12 is positive, it means that the RNA polymers, when polymerised, are able to show liquid phase separation (Brangwynne et al. 2015) and are favourable to form membrane-less organelles like structures. We found that all lengths of RNA have positive, meaning that all lengths would be favourable to show liquid phase separation, if stable enough to form agglomerates.  </DIV><DIV class="text2right"><TABLE><TBODY><TR><TH>Strand</TH><TH>χ12</TH></TR><TR><TD>5xCAG</TD><TD>1.25E+21</TD></TR><TR><TD>15xCAG</TD><TD>9.06E+20</TD></TR><TR><TD>20xCAG</TD><TD>8.43E+20</TD></TR><TR><TD>24xCAG</TD><TD>7.55E+20</TD></TR><TR><TD>40xCAG</TD><TD>7.76E+20</TD></TR><TR><TD>45xCAG</TD><TD>7.7E+20</TD></TR><TR><TD>50xCAG</TD><TD>7.62E+20</TD></TR><TR><TD>70xCAG</TD><TD>9.68E+20</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="text2"><DIV class="text2left"><SPAN>Free energy of mixing of each of the different lengths using the χ12 values obtained above.   </SPAN></DIV> Calculating the Mean free energy of mixing for the Flory Huggins model shows that the higher the number of repeats, the more stable the structure is. Under 30 repeats, we found that they were not stable enough to form aggregates, which corresponds with previous experimental data. </DIV><U><A href="https://static.igem.org/mediawiki/2017/3/3d/PB_Florryhuggins.pdf"> See more in pdf</A></U><H4>Kinetic modelling</H4><DIV class="text2"><DIV class="text2left"> To test the kinetics of the RNA binding into organelles, we used rule based modelling rather than ODE modelling due to the combinatorial nature of RNA binding. Rule based models depend on single agents , representing a biological entity, which bind to one another according to user-defined rules. In our model, each repeat was modelled as one agent, with binding sites to connect multiple agents into strand of length n. Furthermore, there are three binding sites: C, A and G, that define RNA-RNA interactions (Fig. 1 )</DIV><DIV class="text2right"><TSPAN> One agent represented by the a blue circle, and binding sites represented by red circles. Interactions are seen between the different binding sites.</TSPAN></DIV></DIV><DIV class="text6">We show through this modelling that the RNA strands quickly bind, forming agglomerations <TSPAN>Figure 2. The percentage of bound c or g sites over time </TSPAN></DIV><H4> RNA organelle preference and selectivity on specific chemical reactions over non-specific ones - Analytical solution on reaction-diffusion systems </H4><DIV class="text2"><DIV class="text2left"> Once we proved that the RNA repeats could assemble together and form to an synthetic &quot;organelle&quot; rapidly, we wants to know if the RNA organelle could fulfill our need - reduce the underground reactions and improve the efficiency of desired reactions. Given the facts that in bacteria, all the components and reactants come from the cytoplasm,
 not synthesized inside the organelle itself, the situation turns to be very complicated. Besides, how the natural organelles works in this situation was still unclear --  which may be very important for us to understand the initial fitness emerged in organelles and to better design and engineer synthetic organelles. </DIV><DIV class="text2right">  In this study, we compare the composition reaction hosted by the organelles or the cytoplasm (considered as a homogeneous mixture). Using chemical reaction steady state analysis, we generated an analytical solution of the complex quadratic dynamic system for the cytoplasm behavior according to the law of mass action. For the reaction model when the organelle exists in the cell, we hybridized the model we used for the cytoplasm and a specific reaction-diffusion model to describe the specific behaviors on the surface and inside of the organelle.</DIV></DIV><DIV class="text1"><TSPAN>Figure RD 1. Schematic diagram of the reaction process when the organelle is present in the cell: in the homogeneous cytoplasm, two chemical agents react to each other (red and blue), and the organelle (grey ball) specifically binds to A (the red agents); as a result, B (the blue agents) need to diffuse to get into organelle and to keep the reaction going. This process, to be shown in our analytical solution, is very sensitive to the reaction rate constant &quot;k&quot; in the A and B composition reaction. </TSPAN></DIV><DIV class="text1"> In our analysis, we found that the RNA organelles prefer to enhance the reactions with high reaction rate constant (constant k) -- double the production at most, but make the whole-cell production rate significantly when the reaction rate constant is low<TSPAN>Figure RD 2. An example of whole-cell production rate difference between the situation with or without the organelle. When the reaction is non-specific and has a lower constant, the homogeneous mixture will have significantly high production, and vice versa. Therefore,
 the organelle prefers to host the &quot;specific&quot; and &quot;fast&quot; reactions. </TSPAN> 

This indicates that the organelles may serve as compartments that increases the specific experiments and remove the non-specific experiments. Guided by this results, we managed to design a testing system using proper split GFP version. To better understand the parameter sensitivity of the system, we scanned the system performance with different synthesis rate, degradation rate and reaction rate constant combinations.<TSPAN>Figure RD 3. The parameter sensitivity analysis on the production rate with or without the organelles. A and B, left: whole-cell production rate in cytoplasm, with different combination of synthesis (or degradation) and reaction rate constant; right: the situation with the presence of the organelle. The maximum of cytoplasm is only a half as the cell with an organelle. C and D, production rate difference under different parameter combinations. </TSPAN>

We found that when the synthesis rate is low, the advantages of organelle is getting smaller and disappears at the end -- also observed in our wet-lab experiments (Figure RD 3A, C) (<A href="https://2017.igem.org/Team:Paris_Bettencourt/RNA">see here for the experimental results </A>). A few specific parameter constraints are needed to make the advantages of organelle occur, which are usually true in the real biological conditions -- high affinity of organelles to their targets, big enough organelles and low diffusion rate. Specifically, we predicted that larger the cells are, more likely they will benefit from the organelles, even without counting into the cost to pay for the organelle. Also, interestingly, lower the degradation and dilution rate is, it seems that the cell could benefit more from the organelle. Together, these may indicate the reasons that why bacteria lack big independent organelles. For the detailed mathematical description and proofs, welcome to see more in our article attached, <A href="https://static.igem.org/mediawiki/2017/9/9e/PB_RNA_Modelling_Analytical_Proof.pdf"> Mathematical basis of organelle's preference and selectivity on chemical reaction</A>.
</DIV><H1 id="thirdmodel">Logic circuit modeling</H1><DIV class="text1">Recent work on transcription elements showed that assembling insulated synthetic operator upstream and downstream of a insulated T7 promoter core allowed for a more diverse control of gene expression and a more specific response time (Zong et al., 2017).  More importantly, the expression of a gene regulated by such repressible promoters can be well-described by a simple equation:</DIV><DIV><DIV class="text2"><DIV class="text2left">α, β , η<SUB>A</SUB>, K<SUB>A</SUB> are respectively the maximal and basal promoter activity, the Hill coefficient and the dissociation constant of the transcriptional activator-promoter core pair. η<SUB>R</SUB> and Κ<SUB>R</SUB> represent the Hill coefficient and dissociation constant of the binding of a repressor to its cognate operator. δ<SUB>R</SUB> represents the relaxation time, the expected time in which an operator is not bound to any repressor. </DIV><DIV class="text2right">Making the assumption that the elements are insulated, we can easily combine them to create not single but dually repressible promoters, and predict their performance by generalising equation 1. In Equation (2), the fact that  more than one repressor type binding to the promoter was taken into account and changes to relaxation time and the number of total microstates in the equilibrium were made accordingly.</DIV></DIV><DIV class="text2"><DIV class="text2left"><SPAN><B>Figure 1</B>: Structure of the insulated promoters used in the study by Zang <I>et al.</I><B>A</B> - Structure of promoters with a single insulated operator <B>B</B> - Structure of promoters with two insulated operators</SPAN></DIV><DIV class="text2right">Using the experimental data acquired from single repressible promoters, we were able to simulate the behavior of corresponding dually repressible promoters. The experimental data was obtained by testing the impact of different configurations of operators (Figure 1) on gene expression. Results from the data obtained in configuration A and B were combined  <I>in silico</I> to create a large number of promoters. Indeed, we worked with 11 different repressors, leading to a total of 110 promoters. The results obtained for the entirety of this library are available in pdf <A href="https://static.igem.org/mediawiki/2017/c/c7/Promoters_PDF_PB.pdf">here</A> and <A href="https://static.igem.org/mediawiki/2017/f/f6/Second_type_of_promoter.pdf">here</A>. We will focus for now on three repressors with interesting behaviors: </DIV></DIV><DIV class="text1"><SPAN><B>Figure 2</B>: Modeling results for an <I>in silico</I> dually repressible promoter.</SPAN></DIV><DIV class="text1">These three  candidates were chosen for further testing depending on parameter values: the dissociation constant of repressor-operator binding had to be reasonably low and the Hill coefficient η needed to allow for a cooperative behavior. They are also easily available on the iGEM registry and are very commonly used in research. Each promoter was designed to contain two out of the three chosen repressors - TetR, P22c2, HKcI. For each couple of repressors, four different arrangements of the operators were characterized experimentally under different input combinations. More information on the experimental part of this project can be found <A href="https://2017.igem.org/Team:Paris_Bettencourt/Logic_Circuit">here</A>.</DIV><H2>References</H2>
Brangwynne, C.P., Tompa, P. and Pappu, R.V., 2015. Polymer physics of intracellular phase transitions. Nature Physics, 11(11), pp.899-904.
Zong, Y., Zhang, H., Lyu, C., Ji, X., Hou, J., Guo, X., Ouyang, Q. and Lou, C. (2017). Insulated transcriptional elements enable precise design of genetic circuits. Nature Communications, 8(1).


</DIV></DIV><DIV id="footerdiv"><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France<A href="mailto:bettencourt.igem2017@gmail.com">bettencourt.igem2017@gmail.com</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>