Team:Brasil-USP/Notebook/Safety
Toggle navigation
Brasil-USP
Home
Project
Description
Brainstorming
Design
Reactors
Results
Modeling
Lcp Bioinformatics
Molecular Docking
Promoters and Exportation
Rubber degradation
Judging Criteria
Parts
Parts
Team Parts
Assembly Maps
Notebook
Protocols
Notebook
Safety
Policy&Practices
Policy & Practices
Human Practices
ANIP
CBL
São Carlos 8
RECUR
Women in Science
Patent
USP Innovation Agency
Events
Meetup (Jamboré)
Events
Team
Official Profile
Like our Facebook page
About our team
Collaborations
Attributions
Sponsors
Interlab
Entrepreneurship
Safety
Notebook
Table of contents
Safety Training
Risks of the Project
Precautions taken to avoid unexpected release of the GMO
If the organism were released
Institutional Biosafety Committee
Safety Forms
References
    Brasil-USP team is very concerned about the project’s biosafety, and considered it not only while working in the wet lab but also while designing our biological system. Below you can find more information about how our team handled this subject.
Safety Training
    All of the students involved in our project have received safety instructions for working at the laboratory. Graduated students have attended a course offered by São Carlos Institute of Physics ( IFSC/USP ) about basic safety aspects, and then instructed undergraduate students about the precautions to be taken. The course included topics such as: the use of appropriated clothing and personal protective equipment; correct use of equipments such as autoclave, laminar hood flow, electrophoresis chamber and centrifuge; handling safety aspects and disposal of biological and chemical agents; specific danger of the main reagents used at the lab; and precautions about bioaerosol. All of the knowledge obtained in the course was applied on the project developed for iGEM.
Risks of The Project
Parts
    The goal of our iGEM project 2015 is to produce roxA (Rubber oxygenase) and Lcp (Latex clearing protein) in E.coli under laboratory or industrial conditions. Both of these proteins are not entirely elucidated, but it is known that they are responsible to degrade poly(isoprene) into smaller molecules. The main molecule produced in this process is ODTD (12-oxo-4,8-dimethyltrideca-4,8-diene-1-al), however there are no reports about the threat that this compound offers to the environment and to humans. Both of proteins used only degradates the polymer of the rubber, therefore, they don’t offer any harm to humans once we do not have this polymer constituting cell process. Our circuit uses inducible promoters (BBa_K91403; BBa_R0080; BBa_R0010), so if the trigger reactant isn’t present our genic circuit is off, not producing the ODTD molecule, which might be toxic to the environment.
Chassis
    According to the DSMZ database, Acidithiobacillus ferroxidans and Acidithiobacillus thiooxidans are from the risk group 1, and does not offer any disease on healthy adult humans. Additionally, the E. coli BL21, contains several mutations silenced, and does not compete in the environment. There have been no reports showing serious harm in humans. Therefore, this strain is classificated in Risk group 1, and it meets the requirements of the US NIH. For the Interlab study, we used E. Coli DH5-α. Both of them are also from risk group 1, presenting no serious threats.
Precautions taken to avoid unexpected release of the GMO
Bioreactors
    While scaling up our project, bioreactors will be used for rubber devulcanization and degradation of natural rubber. These acts as a physical containment for the developed bacteria and, therefore, minimize the risks of releasing it unexpectedly to the environment.Also, as these processes are being idealized to be performed in factories, only few people of the staff would have contact with the developed microorganism; and these would be specifically trained to minimize contamination - granting an even lower chance of the microorganism being released unexpectedly. Unfortunately, physical containment alone isn’t enough to ensure that the GMO’s won’t be released accidentally. Physical containment is normally enough to ensure that GMO’s won’t be released accidentally, however it’s not 100% efficient. Therefore, other precautions were also taken, as described below.
Kill Switch
    A kill switch mechanism designed to be activated when the bacteria leaves the bioreactor was included in our DNA circuit. For this purpose, our team has worked with a simplified Hok-Sok mechanism - a highly studied kill switch mechanism that is effective on most gram-negative bacteria, and even in some gram-positive microorganisms.  This is how it works: the bacterial culture medium will contain rhamnose, activating the left part of our designed DNA circuit (Figure 1). This will lead to the production of tetR, which inhibits the formation of hokD. If the bacteria is released from the bioreactor, it will be in an environment without rhamnose, and won't produce tetR - leading to the production of HokD, a protein that causes cell membrane depolarization, killing the microorganism. The mechanism is expected to kill the cells almost immediately, with only a fraction of 1.4 x 10-4 the cells surviving after 90 minutes4.Therefore, the system is efficient for containing the microorganism spreading and its modified DNA transmission.
Figure 1. DNA circuit proposed by our team.
    Although the simplified Hok-Sok kill switch mechanism proposed by our team allows, with low energy cost, the efficient and rapid death of the cells after leaving the bioreactor, it doesn’t grant that cells inside the bioreactor which didn’t inherit the developed DNA circuit die. This may be a problem to the process efficiency, as cells that aren’t developing the desired processes will be competing for nutrients with the ones that contain the DNA circuit. For the proposed project, these contaminations will be controlled using antibiotics. As the bacteria desired to survive will contain a plasmid with antibiotic resistance, it won’t die - and all of the other bacteria will.
    As the continuum use of antibiotics can lead to resistant bacteria and has been avoided by scientific community5, our team proposes a circuit that would be able to control these problems. It will not be constructed due to time issues, but it is idealized here (Figure 2).
Figure 2. DNA circuit proposed by our team, with optimization.
The circuits includes sok, a RNA antitoxin, under the rhamnose promoter; tetR is removed from the circuit. Therefore, it will be produced constantly while the cell is inside the bioreactor.
The hok system promoter is changed to a constitutive one - so it will also be produced constantly. In order for it not to be produced in higher concentrations than necessary, which can lead to the risk of it not being completely inhibited by sok, we have inserted only one hok sequence in this design.
While the cells contain the DNA circuit and are inside the bioreactor, they will produce both the toxin and antitoxin, and will live. If they leave the bioreactor, sok won’t be produced, and the cell will die due to hok presence.
For cells that didn’t inherited the developed DNA circuit: these will contain Hok in their plasma, but not sok - this is due to the difference in their degradation rates; while sok has a half life of 30 seconds, the half life of hok is much higher: of 30 minutes. Therefore, these cells would die, avoiding the decrease in process efficiency. The mok gene helps in the hok translation.
Disposal of Biological Material at the Laboratory
    In order to minimize the release of the developed bacteria to the environment, we were very careful in the disposal of biological material at the laboratory. All liquid cell cultures were autoclaved and, after reaching room temperature, disposed on the sink. Used agar plates were autoclaved inside a bag and disposed on common waste. Pipette tips were incinerated before being disposed. There was also the possibility of using a powder disinfectant called Virkon. Usually, this compound is mixed with the contaminated material for 30 minutes. After that, the materials were discarded on common waste.
If the organism were released
Risks to the population
    We are using a strain of E. coli that it's not supposed to survive greatly in the environment. If the kill switch fails, the bacteria will only survive in extreme conditions, which was a low probability to occur. The bacteria used do not offer any harm to humans as stated before. The main molecule produced in this process is the 12-oxo-4,8-dimethyltrideca-4,8-diene-1-al (ODTD).  There are no reports about the threat that this compound offers to the environment and to humans.  Additionally, both of proteins used only degradates the polymer of the rubber, therefore, they don’t offer any harm to humans once we do not have this polymer constituting cell process.
Risks to the environment
    As we use inducible promoters for producing the rubber degrading enzymes, if rhamnose isn’t present on the environment, our genic circuit is off - presenting no threats. Besides E. coli being present in several nature systems, such as water, soil, sand, sediments, and algae, we didn’t find any evidence that free rhamnose is present in considerable concentrations on these environments. Therefore, the developed bacteria wouldn’t be producing neither Lcp nor roxA, presenting no threats to usable rubber or to the environment. Furthermore, Lcp and roxA enzymes are naturally produced by strains of Streptomyces and Xantomonas, respectively. Therefore, no relevant risks to the environment were detected. On our first project design, however, we still depend on antibiotic resistance for maintenance of the desired plasmid. Although this part is not present on our main circuit, it is on the plasmid where we assembled our developed DNA design. This could be an issue as, by horizontal gene transfer, other - possibly dangerous - bacteria could get resistant to antibiotics. In order to eliminate this issue, our team proposed inserting our circuit and the antibiotic resistance into the genome of the developed bacteria - avoiding horizontal gene transfer and the creation of potentially dangerous organisms.
Risks to security through malicious misuse by individuals, groups, or countries
     We did not identify any major risks in our project for malicious misuse. None of the used DNA parts in the main circuit has any direct relation with virulence and the organism presents low pathogenicity, besides being a commonly found bacteria. Besides the plasmid having an antibiotic resistance related part- on our first project design -, it is a known and common sequence, which can be easily obtained by DNA synthesis or acquiring the plasmid commercially, for example. Therefore, no major risks were identified on this subject.
Institutional Biosafety Committee
    At IFSC we have an Institutional Biosafety Committee named CIBio IFSC . All the procedures applied in our team’s project have been discussed with the members and no safety concerns were raised.
iGEM Safety Forms
About Our Lab
About Our Project
Final Safety Form
Safety Spreadsheet
References
https://www.dsmz.de/catalogues/details/culture/DSM-14882.html
http://www.ouhsc.edu/ibc/ibcnih.asp
https://2014.igem.org/Team:TU_Darmstadt
Conditional suicide system for the containment of bacteria and plasmids. S. Molin, P. Klemm, L. K. Poulsen, H. Biehl, K. Gerdes, P. Andersson. Nature Biotechnoogy, Vol 5, 1987
Competitive inhibition of natural antisense Sok-RNA interactions activates Hok-mediated cell killing in Escherichia coli. O. R. Faridani, A. Nikravesh, D. P. Pandey, L. Good, K. Gerdes
Building-in biosafety for synthetic biology. O. Wright, G. B. Stan, T. Ellis
http://www.ncbi.nlm.nih.gov/gene/948616
http://www.who.int/mediacentre/factsheets/fs194/en/
Escherichia coli in the Environment: Implications for Water Quality and Human Health. S. Ishii, M. J. Sadowsky
© 2015 Brasil-USP Team · iGEM · IFSC · USP
Back to top
