<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Brasil-USP_Project_Brainstorming skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Brasil-USP/Project/Brainstorming</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar-wrapper"><DIV class="container"><NAV class="navbar navbar-inverse navbar-static-top"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="https://2015.igem.org/Team:Brasil-USP"><SPAN class="teamname">Brasil-USP</SPAN></A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:Brasil-USP">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="/Team:Brasil-USP/Project/Overview">Description</A></LI><LI><A href="/Team:Brasil-USP/Project/Brainstorming">Brainstorming</A></LI><LI><A href="/Team:Brasil-USP/Project/Design">Design</A></LI><LI><A href="/Team:Brasil-USP/Project/Reactor">Reactors</A></LI><LI role="dropdown-header" class="dropdown-header">Results</LI><LI><A href="/Team:Brasil-USP/Modeling">Modeling</A></LI><LI><A href="/Team:Brasil-USP/LcpBioinformatics">Lcp Bioinformatics</A></LI><LI><A href="/Team:Brasil-USP/Project/Molecular_Binding">Molecular Docking</A></LI><LI><A href="/Team:Brasil-USP/Project/Results">Promoters and Exportation</A></LI><LI><A href="/Team:Brasil-USP/Project/RubberDegradation">Rubber degradation</A></LI><LI><A href="/Team:Brasil-USP/Judging">Judging Criteria</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="/Team:Brasil-USP/Parts">Parts</A></LI><LI><A href="/Team:Brasil-USP/Parts/Teamparts">Team Parts</A></LI><LI><A href="/Team:Brasil-USP/Project/AssemblyMaps">Assembly Maps</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="/Team:Brasil-USP/Notebook/protocols">Protocols</A></LI><LI><A href="/Team:Brasil-USP/Notebook/Calendar">Notebook</A></LI><LI><A href="/Team:Brasil-USP/Notebook/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Policy<SPAN style="color:#FF0;">&amp;</SPAN>Practices </A></LI><LI><A href="/Team:Brasil-USP/Practices">Policy &amp; Practices </A></LI><LI role="dropdown-header" class="dropdown-header">Human Practices</LI><LI><A href="/Team:Brasil-USP/Practices/ANIP">ANIP</A></LI><LI><A href="/Team:Brasil-USP/Practices/CBL">CBL</A></LI><LI><A href="/Team:Brasil-USP/Practices/SaoCarlos8">São Carlos 8</A></LI><LI><A href="/Team:Brasil-USP/Practices/RECUR">RECUR</A></LI><LI><A href="/Team:Brasil-USP/Practices/WomanInScience">Women in Science</A></LI><LI><A href="/Team:Brasil-USP/Practices/Patent">Patent</A></LI><LI><A href="/Team:Brasil-USP/Practices/USPInovacao">USP Innovation Agency</A></LI><LI role="dropdown-header" class="dropdown-header">Events</LI><LI><A href="/Team:Brasil-USP/PolicyPractices/Jambore">Meetup (Jamboré)</A></LI><LI><A href="/Team:Brasil-USP/Practices/Events">Events</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://igem.org/Team.cgi?id=1819">Official Profile</A></LI><LI><A href="http://www.facebook.com/brasilusp" target="_blank">Like our Facebook page</A></LI><LI><A href="/Team:Brasil-USP/Team/Members">About our team</A></LI><LI><A href="/Team:Brasil-USP/Collaborations">Collaborations</A></LI><LI><A href="/Team:Brasil-USP/Attributions">Attributions</A></LI><LI><A href="/Team:Brasil-USP/Sponsors">Sponsors</A></LI><LI><A href="/Team:Brasil-USP/interlabstudy">Interlab</A></LI><LI><A href="https://2015.igem.org/Team:Brasil-USP/Entrepreneurship">Entrepreneurship</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="jumbotron PageTitle"><DIV class="container"><DIV class="col-md-7"><H1 class="PageTitle">Brainstorming</H1><H2 class="PageTitle">Project</H2></DIV></DIV></DIV><DIV class="wikicontent"><P class="lead">    At the very beginning, we have divided the Brasil-USP team in four groups. Each group should suggest a project, research about its impact and viability, as well as how it would work using synthetic biology. After 3 meetings and lots of discussions, the chosen project was “Degradation of Natural Rubber”, mainly due to its environmental appeal and innovative features.    From scientific reviews, we learned that two different enzymes were found to degrade natural rubber: RoxA (Rubber oxygenase A) and Lcp (Latex clearing protein) <SUP>1,2,3,4</SUP>. They are naturally expressed by specific strains of <I>Xanthomonas sp.</I> and <I>Streptomyces sp.</I>, respectively, and act through distinct mechanisms to degrade the rubber polymer. RoxA mechanism consists of exo-cleavage degradation of the rubber polymer, poli(cis-1,4-isoprene), producing 12-oxo-4,8-dimethyltrideca-4,8-diene-1-al (ODTD). Lcp works in a similar way, but through endo-cleavage degradation and its final products are oligo-isoprenoides with aldehyde and ketone groups at the end, including ODTD.<SUP>5</SUP>    Our main project idea was to speed up the degradation process of rubber, since it may take from 500 to 1000 years for it to naturally decompose. Then, we thought: why not genetically engineer a microorganism to produce both enzymes, RoxA and Lcp, simultaneously?! We also planned to scale up the process in bioreactors.</P><H2>Chimeric Protein</H2><P class="lead">    One of our first ideas was to build a chimeric protein of RoxA and Lcp. This chimera could be really efficient on cleaving latex polymer by both degradation types - endo and exo cleavages - and it would be a major innovation in our project. But, after talking to one of the best protein structure researchers in our institute, Prof. Dr. Richard C. Garratt, we ruled out this possibility. Richard told us that RoxA structure was really complex and we could not exclude any of its parts to build the chimera. Additionally, Professor Richard noted RoxA similarity to peroxidases excepted by an extra structure that could be the reason why RoxA degrades rubber. This similarity has been already reported by Seidel <I>et al</I>.<SUP>19</SUP>. Then, considering the high molecular weight of both proteins, especially RoxA, and the inviability of reaching a more fundamental structure of RoxA, we concluded that unfortunately RoxA-Lcp chimeric protein would not be viable.
</P><H2>Microorganism choice</H2><P class="lead">    For the execution of the project, three organisms were considered: <B><I>Xanthomonas sp.</I> 35Y</B>, <B><I>Pichia pastoris</I></B>, and <B><I>Escherichia coli </I></B>. These have been chosen based on easy handling, data from literature correlating such organisms with the synthesis of the proteins of interest, as well as their availability.    Because <B><I>Xanthomonas sp. 35Y</I></B> naturally synthesizes RoxA and produces Lcp without great difficulties <SUP>5</SUP> , this strain was the main candidate to be used in our project. Furthermore, this bacteria is a simple organism and has a mechanism to export proteins to the extracellular space, what is a necessary characteristic for this project. Nevertheless, this specific strain of <I>Xanthomonas</I> is <B>not easy to be found</B>. We tried to contact researchers who had published scientific articles using this organism but the collaboration with them was not possible because of patent issues.    The yeast <B><I>Pichia pastoris</I></B> is generally used for heterologous expression and it owns a well characterized system for protein secretion <SUP>6</SUP>. Due to these reasons, this organism was considered for our project. However, because of the <B>lack of information in the iGEM’s registry</B> about this organism and some discussion with our advisor about the <B>difficulty to work with it</B>, we have preferred to discard this option.     <B><I>Escherichia coli</I></B> is the <B>most widely studied prokaryotic model organism</B>. This organism has great easiness of handling, availability and a variety of defined protocols for homologous recombination. 
Despite this, literature mentions the attempts of expressing the RoxA protein in <I>E. coli</I> stating that it has led to inclusion bodies formation.<SUP>7</SUP> This effect is probably a consequence of the limited secretion capacity of <I>E. coli</I> transport machinery. When this capacity is overwhelmed, the excess of expressed recombinant protein is likely to accumulate in inclusion bodies.<SUP>8</SUP> Other possible reason for this result is due to the fact that, in <I>Xanthomonas sp. 35Y </I>, RoxA is naturally exported to the extracellular space passing through the periplasmic space where disulfide bonds are formed which gives the protein a appropriate folding. A possibility is that, in the study that tried to express RoxA in <I>E. coli</I>, the signal peptide may not have been recognized by the bacteria or may even not be present. So, since the protein was not sent to the periplasmic space, the disulfide bonds were not formed properly, leading to incorrect protein folding. Disulfide bonds are relatively rare in intracellular proteins because the cytosol is a strongly reducing environment.<SUP>9</SUP>    By using a software for signal peptide prediction (predTAT), we have found the RoxA protein is presumably translocated across the cytoplasmic membrane by the Sec translocon, while Lcp is translocated across the cytoplasmic membrane by the twin-arginine translocation pathway (TAT), what is well corroborated by the literature <SUP>10</SUP>. Besides, heterologous expression in <I>E. coli</I> of the entire Lcp sequence of <I>Streptomyces sp. K30</I>, including the TAT signal peptide, was successful.<SUP>10</SUP> Thus, to enhance the secretion of both RoxA and Lcp in <I>E. coli</I>, avoiding the production of inclusion bodies, we decided to introduce the twin-arginine translocation (TAT) signal in our genetic circuit - to secret Lcp to the extracellular space - and to fusion RoxA with OmpA (Outer membrane protein A) - leading to the transport of RoxA to the bacterium external surface.<SUP>11</SUP>    Ultimately, because of the <B>availability, project adaptability and facilities of using <I>E. coli</I></B>, we chose this microorganism to host our genetic circuit.  </P><H2> Kill Switch System </H2><P class="lead">    In order not to spread our genetically modified bacteria to the environment, we projected a kill switch system - which is activated when the bacteria leaves the designated bioreactors. We found that there is a family of lethal genes in gram-negative bacteria <SUP>12</SUP>. One of these is the hok gene, which encodes a toxin responsible for causing the cell death by depolarization of the cell membrane <SUP>13</SUP>. In this way, our kill switch system is composed by a homologous of hok gene (hokD) and is controlled by a tetracycline-repressible promoter.     Among the factors that limit the effectiveness of suicide systems, mutations that make the cell insensitive to the killer protein and the respective selection of these cells to propagate are the major. The surviving phenotype may result primarily from mutation in the actual lethal gene, in the promoter, or yet in other genes that inactivate their functions <SUP>14</SUP>. Thus, we decided to duplicate the hokD part in our genetic circuit to improve the efficiency of our kill switch system and consequently decrease the survival chance of the organism outside the bioreactor.</P><H2>Devulcanization </H2><P class="lead">    Considering the fact that tires are made of vulcanized rubber to ensure durability and increase elasticity and strength, we included a complimentary pretreatment using a bacterium that naturally devulcanizes rubber. The protein capable of catalyzing cleavage of sulfur bonds is tetrathionate hydrolase (tetH), which functions in low pH and is expressed by <I>Acidithiobacillus ferrooxidans</I><SUP>15</SUP>. This process will be held in a separated bioreactor.</P><H2>Final Product Transformation </H2><P class="lead">    Moreover, we aimed not only to speed up the rubber degradation process, but also to transform the main product of degradation - the ODTD molecule, a triisoprene unit - into a product with high economic value. Some options came on, such as: 
</P><OL><LI>Polymerizing the ODTD molecule to re-obtain polyisoprene;</LI><LI>Creating a new polymer;</LI><LI>Transforming the ODTD molecule in a hydrocarbon chain with fuel properties.</LI></OL><P class="lead">    The first option, polymerizing the ODTD molecule to re-obtain polyisoprene, would be great, since it could diminish the expenditure of non-renewable carbon sources and also the emission of gases that cause the greenhouse effect.    The second option, creating a new polymer, would be more questionable: how would this new theoretical polymer be degraded?; would it have some economic value?     Finally, the third option, <B>creating a fuel</B>, seemed really attractive. We would then eliminate the burning step, <B>lowering the emission of sulfur</B>, and generate a high-energy molecule directly from a renewable source (namely, natural rubber) resulting in a more sustainable end for this material. Consequently, besides granting the final product considerable <B>economic interest</B>, it would also decrease the final pollutant emission.     After chemically analyzing all the three options, with the aid of organic chemistry professors, our theoretical studies indicated that transforming ODTD into a fuel would be the more practicable option.     Since the ODTD molecule has a ketone and an aldehyde groups in the C2 and C13 positions, as shown on the Figure 1, in order to transform this molecule in a hydrocarbon fuel, two different steps should be performed (Figure 2): the reduction of the aldehyde and ketone groups into alcohol groups; and then the dehydration of the alcohol groups.   </P><CENTER>Figure 1. ODTD 2D structure</CENTER><CENTER>Figure 2. Reduction and dehydration of ODTD molecule to obtain a hydrocarbon fuel</CENTER><P class="lead">    To achieve the reduction of the ketone and aldehyde groups to alcohol, a strong reducing agent, as sodium borohydride (NaBH<SUB>4</SUB>), could be used. The second step, the dehydration of alcohol groups, could be reached by introducing a strong acid catalyst, as concentrated sulphuric acid (H<SUB>2</SUB>SO<SUB>4</SUB>). However, these options do not seem to be industrially practicable, since sodium borohydride (NaBH<SUB>4</SUB>) is an expensive reagent. Considering that, it is interesting to have another option on mind.    Catalytic Hydrogenation has wide industrial application since the reactions are highly selective, the catalyst can often be recovered and recycled, and it generally is an economically viable process.<SUP>16 </SUP> 
For each application, a deeper study should be performed to determine the best catalyst and solvent, the specific conditions of pressure, temperature, concentration and time of the reaction.
From similar reactions and molecules<SUP>17,18</SUP>, we concluded that good initial conditions to the <B>catalytic hydrogenation</B> experiments with ODTD would be: </P><UL><LI>Catalyst: Pd\C;</LI><LI>Hydrogen Donor: H<SUB>2</SUB>, limonene, cyclohexane</LI><LI>Temperature: 0 °C; </LI><LI>Pressure: 2 atm to 3 atm </LI><LI>Solvent: ethanol, methanol.</LI></UL><H1>References</H1><FONT size="-1">
1 Jenrossek, D.; Reinhard, S. “Sequence analysis of a gene product synthesized by Xanthomonas sp. during growth on natural rubber latex”. FEMS Microbiology Letters, v. 224, p. 61– 5, 2003.
 
2 Rose, K; Tenberge, K. B.; Steinbuchel, A. “Identification and characterization of genes from Streptomyces sp. strain K30 responsible for clear zone formation on natural rubber latex and poly(cis-1,4-isoprene) rubber degradation”. Biomacromolecules, v. 6, p. 180-8, 2005.
 
3 Shah, Aamer Ali et al. “Biodegradation of natural and synthetic rubbers: a review”. International Biodeterioration &amp; Biodegradation 83 145e157, 2013.
 
4 Yikmis, Meral; Steinbüchel, Alexander. “Historical and Recent Achievements in the Field of Microbial Degradation of Natural and Synthetic Rubber”. Applied and environmental Microbiology, v. 78, number.13, 2012.

5 Birke, Jakob; Jendrossek, Dieter. “Rubber Oxygenase and Latex Clearing Protein Cleave Rubber to Different Products and Use Different Cleavage Mechanisms”. Applied and environmental Microbiology, v. 15, ed.16, 2014.

6 AHMAD, Mudassar et al. “Protein expression in Pichia pastoris: recent achievements and perspectives for heterologous protein production”. Applied microbiology and biotechnology, v. 98, n. 12, p. 5301-5317, 2014.

7 Schmitt, G.; Seiffert, G.; Kroneck, M.H.; Braaz, R.; Jendrossek, D. “Spectroscopic properties of rubber oxygenase RoxA from Xanthomonas sp., a new type of diheme dioxygenase”. Journal of Applied Microbiology, v. 156, p. 2537-48, 2010.
8 Mergulhão, F.J.M.; Summers, D.K.; Monteiro, G. A.; “Recombinant protein secretion in Escherichia coli”; Biotechnology Advances 23 177–202, 2005.
9 Narayan, Mahesh; Welker, Ervin; Wedemeyer, William J.; Scheraga, Harold A.; “Oxidative Folding of Proteins”; Acc. Chem. Res. 33, 805-812, 2000.
10 Yikmis, M; Arenskötter M, Rose K, Lange N, Wernsmann H, Wiefel L, Steinbüchel A. “Secretion and transcriptional regulation of the latex-clearing protein, Lcp, by the rubber-degrading bacterium Streptomyces sp. strain K30”; Appl Environ Microbiol. 74(17):5373-82., September 2008.
11 “OmpA outer membrane protein A fused to linker; displays proteins on cell surface”, available on: http://parts.igem.org/Part:BBa_K103006

12 Poulsen, L. K.; Larsen, N. M.; Molin, S.; Andersson, P; “A family of genes encoding a cell-killing function may be conserved in all Gram-negative bacteria”; Molecular Microbiology, 3(11), 1463-1472, 1989.

13 Molin, S.; Klemm, P.; Poulsen, L. K; Biehl, H.; Gerdes, K.;   Andersson, P; Andersson, P; “Conditional Suicide System for Containment of Bacteria and Plasmids”; Nature Biotechnology, v.5, December 1987. 

14 Marques, Daniel et al; “Plasmídeos suicidas e Contenção de Organismos Transgênico”; Procitropicos Informa, March 2015.

15 Kanao T.; Kamimura K.;, Sugio  T;  “Identification of a gene encoding a tetrathionate hydrolase in Acidithiobacillus ferrooxidans”; J Biotechnology, 132(1):16-22, 2007 Oct 21.

16 Nerozzi, Fabrizio; “Heterogeneous Catalytic Hydrogenation”; Platinum Metals Rev., 56, (4), 236–241, 2012.

17 Holleben, Maria Luiza; Silva, Silvana; Mauler, Raquel; “Hydrogenation of styrene-butadiene rubber by hydrogen transfer from limonene”; Polymer Bulletin 33, 203-208, 1994.

18 Alotaibi, Mshari A.; Kozhevnikova, Elena F.; Kozhevnikov, Ivan V.; “Efficient hydrodeoxygenation of biomass-derived ketones over bifunctional Pt-polyoxometalate catalyst”; Chem. Commun., 48, 7194-7196, 2012.

19 Seidel J, Schmitt G, Hoffmann M, Jendrossek D, Einsle O. &quot;Structure of the processive rubber oxygenase RoxA from Xanthomonas sp.&quot; Proceedings of the National Academy of Sciences of the United States of America. 2013;110(34):13833-13838. doi:10.1073/pnas.1305560110.
</FONT></DIV><DIV class="container"><DIV id="footer"><P>© 2015 Brasil-USP Team
	·<A href="https://2015.igem.org/">iGEM</A>·<A href="http://www.ifsc.usp.br/english/">IFSC</A>·<A href="http://www.usp.br">USP</A></P></DIV></DIV><A class="back-to-top" href="#top">Back to top</A></DIV></DIV></DIV></DIV></BODY></HTML>