Team:Brasil-USP/Notebook/protocols
Toggle navigation
Brasil-USP
Home
Project
Description
Brainstorming
Design
Reactors
Results
Modeling
Lcp Bioinformatics
Molecular Docking
Promoters and Exportation
Rubber degradation
Judging Criteria
Parts
Parts
Team Parts
Assembly Maps
Notebook
Protocols
Notebook
Safety
Policy&Practices
Policy & Practices
Human Practices
ANIP
CBL
São Carlos 8
RECUR
Women in Science
Patent
USP Innovation Agency
Events
Meetup (Jamboré)
Events
Team
Official Profile
Like our Facebook page
About our team
Collaborations
Attributions
Sponsors
Interlab
Entrepreneurship
Protocols
Notebook
Table of contents
Circuit Assembly Protocols
Calcium chloride transformation with heat shock in Escherichia coli DH5α
Plasmid extraction
Digestion of plasmidial DNA
Ligation reaction (Cohesive ends)
Agarose Gel Electrophoresis
Directed mutagenesis PCR for restriction site elimination using a plasmid template
PCR amplification (applied to lcp)
PCR amplification for difficult amplicons (applied to roxA)
Gibson assembly
Characterization Protocols
Pre-inoculum
Plate Reader
Flow Cytometer
Western Blotting
Calcium chloride transformation  with heat shock in Escherichia coli DH5α
Materials
Sterile LB agar plate supplemented with the appropriate antibiotic (ampicillin 100 μg ml-1 or chloramphenicol 34 μg ml-1 - SIGMA-ALDRICH®);
Sterile liquid LB media (SIGMA-ALDRICH®);
Competent DH5α cells (Novagen) prepared through heat shock with calcium chloride;
Plasmidial DNA.
Methodology
Put the 0.5 mL microtube containing 50 μL competent cells aliquot on ice;
Add 20-50 ng of plasmidial DNA or 10  μL of ligation reaction to the competent cells. Mix by pipetting carefully;
Place the tube into a 42°C water bath for 2 min;
Return the tube to the ice for 5 min;
Add 200 μL of liquid LB;
Incubate at 37°C, 250 rpm for 45 min;
Plate the liquid LB containing the bacterial suspension on a LB agar plate with the appropriate antibiotic;
Incubate overnight (14-16h) at 37°C.
Plasmid extraction
PureLink® Quick Plasmid Miniprep Kit-Life Technologies
Methodology
Cell Growth
After isolating a single colony from a LB agar plate, grow it in 6 mL of liquid LB within the appropriate antibiotic. Incubate overnight (14-16h) at 37°C in a shaking incubator.
Resuspension
Pellet the overnight culture in a 2 mL microtube and discard the supernatant. Repeat this step until the total liquid culture is finished. Resuspend the cell pellet in 240 μL of resuspension buffer by vortexing.
Lysis
Add 250 μL of the Lysis buffer. Mix by inversion 4-8 times and incubate at 37°C for 3-5 minutes. Do not exceed this period.
Neutralization
Add 350 μL of the neutralization solution. Mix by inversion 4-8 times and incubate at 37°C for 3-5 minutes. Do not exceed this time. Centrifuge at 16000 g for 10 minutes.
Washing
Transfer the supernatant to a new 1.5 mL microtube with the resin. Be careful not to transfer the white pellet.  Add 650 μL of Wash buffer. Centrifuge at 16000 g for 1 minute. Discard the supernatant. Centrifuge again for 2-4 min to remove ethanol remains.
Elution of plasmidial DNA
Put the resin in a new 1.5 mL microtube. Add 50 μL of nuclease free water at 65°C. Centrifuge at 16000 g for 3 minutes and discard the resin. Store DNA at -20°C.
Digestion of plasmidial DNA
Materials
Plasmidial DNA;
Restriction Enzyme 1: EcoRI or XbaI (FastDigest Thermo Scientific);
Restriction Enzyme 2: SpeI or PstI (FastDigest Thermo Scientific);
FastDigest Buffer (Thermo Scientific);
Nuclease free water.
Methodology
On ice, prepare the following mixture in a microtube:
- 500 -1000 ng of plasmidial DNA
- 1 μL of Restriction Enzyme 1
- 1 μL of Restriction Enzyme 2 (if necessary)
- 2 μL of 10x FastDigest Buffer
- Nuclease free water to complete 20 μL
Spin the mixture.
Incubate at 37°C for at least 3 hours.
Perform agarose gel electrophoresis to confirm the results
Ligation reaction (Cohesive ends)
Reference: https://www.neb.com/protocols/1/01/01/dna-ligation-with-t4-dna-ligase-m0202
Materials
Vector DNA digested;
Insert DNA digested;
10X T4 DNA Ligase Buffer* (Thermo Scientific);
T4 DNA Ligase (Thermo Scientific);
Nuclease free water.
Methodology
Set up the following reaction in a microcentrifuge tube on ice:
* The T4 DNA Ligase Buffer should be thawed and resuspended at room temperature.
q.s. = quantum sufficit
**
\[ ng_{insert} =\left (\frac{kb_{insert}}{kb_{vector}}  \right )\left (ng_{vector}  \right )\left (ratio  \right )\]
ratio was considered to be 3, but it can vary according to the vector
For cohesive ends, incubate at 22°C for 3 hours + 16°C for 9 hours;
Heat shock in E. coli DH5α with 10 μl of the reaction.
Agarose Gel Electrophoresis
Materials
1X TAE Buffer;
Electrophoresis apparatus (cell, gasket, power supply, gel caster and comb; BIO-RAD - http://www.bio-rad.com/cmc_upload/Literature/38717/M1704400B.PDF);
Gel analysis and documentation equipement (Gel DocTM EZ System, BIO-RAD);
UV light box
DNA ladder  (Invitrogen or Thermo Scientific);
10X Loading Buffer (Invitrogen);
Ethidium bromide (Promega);
x% (mass/volume) agarose gel (the concentration varies with the size of  the DNA sample; 1.2% is recommended to short DNA fragments - smaller than 200bp - and 0.8% is recommended to high length of DNA)
Methodology
Agarose gel: Mixture 1X TAE with agarose (1.2 or 0.8 grams for each 100ml of buffer depending of  x% agarose) and melting the mixture. Add ethidium bromide and after, transfer the melted gel into a gasket in a gel caster support with an appropriate comb);
Prepare samples by diluting in loading buffer to approximately 1X or higher;
Load the DNA ladder into the first well of the gel and the samples into the additional wells;
Transfer gel to a cell and apply DNA ladder and samples;
Run the gel for about 40 minutes at 100 volts;
Do analysis (in Gel Doc equipment) or cut the gel (UV light box).
Directed mutagenesis PCR for restriction site elimination using a plasmid template
Materials
1 μl of 10 ng/μl DNA template;
5 μl of each primer forward and reverse (diluted to 20μM) previously designed and purchased;
1 μl of dNTP mixture 10mM;
10 μl Phusion HF  5X buffer (NEB)  with MgCl2;
1 μl High Fidelity enzyme (2.5U μl-1, NEB);
Sterile deionized water to 50 μl.
Methodology
In a thermal cycler (BIO-RAD) set the following steps:
First (1X): 95°C for 3 min;
Second (18X): 95°C for 30s; 60°C for 30s (primers Tm); 72°C for 5 min (15-30s per kb - pUC9::roxA : 4462 bp)
Third (1X): 72°C for 15min;
Hold in 4°C.
Run a gel electrophoresis to analysis (10 μl) and purification (all remainder reaction);
Prepare a DNA digestion with only DpnI enzyme (Thermo Scientific);
Heat shock in E. coli DH5α with 10 μl of the digest reaction;
Do minipreps with some colonies;
Confirm the mutation with a digest reaction with two enzymes, one vector site containing and with the desired mutation site. Confirm with a gel electrophoresis.
PCR amplification (applied to lcp)
Materials
1μl DNA template at 10 ng μl-1;
5μl of each primers forward and reverse (diluted to 20μM) previously designed and purchased;
1μl of dNTP mixture 10mM;
5μl High Fidelity 10X buffer (Thermo scientific) with MgCl2;
2.5μl BSA protein (Promega);
0.5μl High fidelity enzyme (2.5 U μl-1, Thermo Scientific);
Sterile deionized water quantum sufficit for 50 μl.
Methodology
In a thermal cycler (BIO-RAD) set the following steps :
First (1X): 95°C for 3 min;
Second (30X): 95°C for 30s; 57°C for 30s; 72°C for 2 min (1-2 min per kb - lcp : 1128 bp);
Third (1X): 72°C for 10min;
Hold in 4°C.
Run a gel electrophoresis to analysis (3 μl) and purification (all remainder reaction).
PCR amplification for difficult amplicons (applied to roxA)
Materials
1μl DNA template at about 300 ng μl-1;
1.25 μl of each primer forward and reverse (diluted to 20μM) previously designed and purchased;
25 μl Q5 High-Fidelity 2X Master Mix (NEB);
Sterile deionized water quantum sufficit for 50 μl.
Methodology
In a thermal cycler (BIO-RAD) set the following steps :
First (1X): 98°C for 30s;
Second (30X): 95°C for 10 s; 63°C for 30s (primers Tm calculated by NEB TM calculator - http://tmcalculator.neb.com/#!/); 72°C for 1min (20-30s per kb);
Third (1X): 72°C for 2min;
Hold in 4°C.
Run a gel electrophoresis to analysis (3 μl) and purification (all remainder reaction)
Gibson assembly
For complete protocol : https://www.neb.com/products/e2621-nebuilder-hifi-dna-assembly-master-mix
Methodology
Reaction
\[ ng_{insert} =\left (\frac{kb_{insert}}{kb_{vector}}  \right )\left (ng_{vector}  \right )\left (ratio  \right )\] Ratio for inserts smaller than 200bp (1:5); for inserts bigger than 200bp use 1:5. Recommended 0.03 - 0.2 pmol; pmol \[ gBlock =\left (\frac{weight}{bp x 650 Daltons}  \right )\left (1000  \right )\] use weight in ng
Incubate at 50°C for 15 minutes;
Heat shock in E. coli DH5α with 10 μl of the reaction.
Pre-inoculum
Methodology
E. coli cells transformed were plated on LB agar (Sigma) plates supplemented with antibiotic:
chloramphenicol (34 ug/ul) or ampicillin (100ug/ul)
Grown for 18-20 hours at 37°C.
Tip 1 colony from the LB agar plates into independent sterile tubes with 5 ml of LB media (1:4) containing antibiotic
Repeat the previous step twice for 2 different colonies to obtain biological replicate.
Grown overnight (14 - 16 hours)  at 37°C, 80 - 300 rpm.
Plate Reader
Optical Density
Using a clear, flat- or round-bottomed, 96-well plate we measured optical density  at 600 nm (OD600).
Methodology
In each well add LB and pre-inoculum
usually in a proportion 1:10 we obtained OD600 ~ 0.1 with the previous pre-inoculum conditions
Measure OD600
If necessary, adjust LB:pre-inoculum proportion to the proper OD600.
Constitutive promoter: measure OD600
Inducible promoter: add inducer and measure OD600 in time (8-12 hours)
Fluorescence
Using a black, flat-bottomed, 96-well plate we measured fluorescence adjusting excitation and emission wavelength according to the fluorescent molecule.
Methodology
In each well add LB and pre-inoculum
usually in a proportion 1:10 we obtained OD600 ~ 0.1 with the previous pre-inoculum conditions
Measure OD600
If necessary, adjust LB:pre-inoculum proportion to the proper OD600.
Constitutive promoter: measure OD600
Inducible promoter: add inducer and measure OD600 in time (8-12 hours)
Induction response must be observed in time due to cellular growth and consequent gene expression. Induction during the mid-log phase of growth might maximize protein expression
Flow Cytometer
Methodology
Diluted cultured bacteria to approximately 8 x 106 bacteria mL-1 or OD600 of ~ 0.01 (cell concentration must be in the range of 1x106 to 2 x 107 bacteria mL-1 according to the equipment manufacturer’s recommendation).
Wash twice with phosphate buffer saline (PBS) at centrifugation 5000 g for 5 minutes.
Remove the supernatant
Resuspended the pellet in sheath fluid, same volume as used to obtain OD600 of 0.01.
Constitutive promoter: measure fluorescence using the appropriate channel
Inducible promoter: we previously induced the cultured bacteria and used non-induced cultures as control.
Western Blotting
SDS-PAGE running buffer
25mM Tris
190 mM glycine
0.1% SDS
Check the pH and adjust to pH 8.3
Transfer buffer
25mM Tris
190mM glycine
20% methanol
Check the pH and adjust to pH 8.3
Ponceau S staining buffer
20mM Tris pH 7.5
150mM NaCl
Tris-buffered saline Tween 20 (TBST) buffer
20mM Tris pH 7.5
150mM NaCl
0.1% Tween 20
Blocking buffer
5% milk in TBST
Separate the protein sample using electrophoresis gel. Apply the samples with a multicolor broad range marker to ensure the transference.
Place the gel in 1x transfer for 15 minutos.
Assemble the transfer sandwich as shown in the image below.
Place the cassette in a transfer tank and place a ice block in the tank.
Transfer for 1 hour, 100V, 350mA.
Stain the blot in ponceau staining buffer to check the transfer quality.
Rinse of the ponceau with 3 washes with 1x TBST.
Block with 5% milk in TBST for 1 hour at room temperature.
Rinse the blot 3 to 5 times with 1x TBST.
Incubate with the primary antibody solution (anti His-tag) for 3 hours at room temperature.
Rinse the blot 3 to 5 times with 1x TBST.
Incubate with the HRP-conjugated secondary antibody for 1 hour at room temperature.
Rinse the blot 3 to 5 times with 1x TBST.
Treat the membrane with luminol solution (6ml luminol, 20μl peroxidase).
Capture the luminescent signals with a CCD camera-based imager.
Use an image analysis to read the band intensity of the target proteins.
© 2015 Brasil-USP Team · iGEM · IFSC · USP
Back to top
