<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Brasil-USP_Project_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Brasil-USP/Project/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar-wrapper"><DIV class="container"><NAV class="navbar navbar-inverse navbar-static-top"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="https://2015.igem.org/Team:Brasil-USP"><SPAN class="teamname">Brasil-USP</SPAN></A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:Brasil-USP">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="/Team:Brasil-USP/Project/Overview">Description</A></LI><LI><A href="/Team:Brasil-USP/Project/Brainstorming">Brainstorming</A></LI><LI><A href="/Team:Brasil-USP/Project/Design">Design</A></LI><LI><A href="/Team:Brasil-USP/Project/Reactor">Reactors</A></LI><LI role="dropdown-header" class="dropdown-header">Results</LI><LI><A href="/Team:Brasil-USP/Modeling">Modeling</A></LI><LI><A href="/Team:Brasil-USP/LcpBioinformatics">Lcp Bioinformatics</A></LI><LI><A href="/Team:Brasil-USP/Project/Molecular_Binding">Molecular Docking</A></LI><LI><A href="/Team:Brasil-USP/Project/Results">Promoters and Exportation</A></LI><LI><A href="/Team:Brasil-USP/Project/RubberDegradation">Rubber degradation</A></LI><LI><A href="/Team:Brasil-USP/Judging">Judging Criteria</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="/Team:Brasil-USP/Parts">Parts</A></LI><LI><A href="/Team:Brasil-USP/Parts/Teamparts">Team Parts</A></LI><LI><A href="/Team:Brasil-USP/Project/AssemblyMaps">Assembly Maps</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="/Team:Brasil-USP/Notebook/protocols">Protocols</A></LI><LI><A href="/Team:Brasil-USP/Notebook/Calendar">Notebook</A></LI><LI><A href="/Team:Brasil-USP/Notebook/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Policy<SPAN style="color:#FF0;">&amp;</SPAN>Practices </A></LI><LI><A href="/Team:Brasil-USP/Practices">Policy &amp; Practices </A></LI><LI role="dropdown-header" class="dropdown-header">Human Practices</LI><LI><A href="/Team:Brasil-USP/Practices/ANIP">ANIP</A></LI><LI><A href="/Team:Brasil-USP/Practices/CBL">CBL</A></LI><LI><A href="/Team:Brasil-USP/Practices/SaoCarlos8">São Carlos 8</A></LI><LI><A href="/Team:Brasil-USP/Practices/RECUR">RECUR</A></LI><LI><A href="/Team:Brasil-USP/Practices/WomanInScience">Women in Science</A></LI><LI><A href="/Team:Brasil-USP/Practices/Patent">Patent</A></LI><LI><A href="/Team:Brasil-USP/Practices/USPInovacao">USP Innovation Agency</A></LI><LI role="dropdown-header" class="dropdown-header">Events</LI><LI><A href="/Team:Brasil-USP/PolicyPractices/Jambore">Meetup (Jamboré)</A></LI><LI><A href="/Team:Brasil-USP/Practices/Events">Events</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://igem.org/Team.cgi?id=1819">Official Profile</A></LI><LI><A href="http://www.facebook.com/brasilusp" target="_blank">Like our Facebook page</A></LI><LI><A href="/Team:Brasil-USP/Team/Members">About our team</A></LI><LI><A href="/Team:Brasil-USP/Collaborations">Collaborations</A></LI><LI><A href="/Team:Brasil-USP/Attributions">Attributions</A></LI><LI><A href="/Team:Brasil-USP/Sponsors">Sponsors</A></LI><LI><A href="/Team:Brasil-USP/interlabstudy">Interlab</A></LI><LI><A href="https://2015.igem.org/Team:Brasil-USP/Entrepreneurship">Entrepreneurship</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="jumbotron PageTitle"><DIV class="container"><DIV class="col-md-7"><H1 class="PageTitle">Promoter and Exportantion tests</H1><H2 class="PageTitle">Results</H2></DIV></DIV></DIV><DIV class="container jamboreh"><DIV id="ToC"><H3>Table of contents</H3><UL><LI><A href="#promotertest">Promoter Test</A></LI><UL class="sub"><LI><A href="#dh5alphabad">DH5α</A></LI><LI><A href="#BL21de3supercool">BL21 (DE3)</A></LI><LI><A href="#discussionpromotertest">Discussion</A></LI></UL><LI><A href="#exportationtest">Exportation</A></LI><UL class="sub"><LI><A href="#tatsignal">TAT signal</A></LI><LI><A href="#ompA">OmpA</A></LI><LI><A href="#discussionexportation">Discussion</A></LI></UL></UL></DIV><P class="lead">    Natural and synthetic rubber degradation has been reported for some microorganisms such as <I>Streptomyces sp.</I> strain K30, <I>Gordonia polyisoprenivorans</I>, <I>Nocardia sp.</I> strain 835A and <I>Xanthomonas sp.</I> strain 35Y.<SUP>1</SUP> However, most of these organisms show a slow growth when using rubber as a sole carbon source.<SUP>2-4</SUP>   To improve microbial rubber degradation efficiency, our project aims to create a gene circuit which will, among other things, be responsible for coding and expressing two enzymes: RoxA (Rubber oxygenase A) and Lcp (Latex clearing protein). These enzymes are fundamental for rubber degradation and have to be secreted or exposed in the exterior of the cell to be in direct contact with their substrate, the poly(cis-1,4 isoprene) - the main component of rubber. The proposed circuit provides the secretion of the proteins by the fusion with a specific signal sequence, the Twin-arginine translocation (TAT) sequence, or its binding to the bacterial outer membrane by the fusion with a specific protein, OmpA fused with a linker (<A href="http://parts.igem.org/Part:BBa_K1489002">BBa_K1489002</A>). 
</P><DIV class="wikicontent"><H1>Promoter Test</H1><P>    The Gram negative bacterium <I>Escherichia coli</I> is the most used chassis in Synthetic Biology. The bacterial system is often chosen due to its low cost, easy maintenance, high productivity and low time consumption. Moreover, there are several large scale system established for <I>E. coli</I> use in industry and, since our project aim for industrial applications, we have chosen the E. coli system allied with the previously named secretion routes that might help increase enzymes' stability. However, the choice of the promoter system is not as simple as the chassis'. That being said, we designed circuits to test the viability of three different inducible promoters: Plac, Para and Prha (figure 1). </P><DIV class="fig" style="width: 1000px;"><P><CENTER><FONT size="3">Figure 1 - Promoter test circuits (more information about the design of the circuits <A href="https://2015.igem.org/Team:Brasil-USP/Project/Design">here</A>). </FONT></CENTER></P></DIV><H2>DH5α</H2><P>    We first started with DH5α. For protocols, we refer to our <A href="https://2015.igem.org/Team:Brasil-USP/Notebook/protocols">Protocols section</A>. We evaluated the fluorescence per cell by measuring Optical Density (OD) of our colonies and their fluorescences. We show in figures 2 and 3 our results.</P><DIV class="fig" style="width: 600px;"><P><CENTER><FONT size="3">Figure 2 - Optical Density (OD) measurements over time, showing that all colonies were in their log phase.</FONT></CENTER></P></DIV><DIV class="fig" style="width: 750px;"><P><FONT size="3"><CENTER>Figure 3 - Fluorescence per cell for each promoter, normalized by the fluorescence with no induction. In the x-axis, the higher the concentration label, the higher the concentration, with zero means no induction at all. Only Rhamnose showed some actual induction. All errors may be considered about 0.05 a.u.</CENTER></FONT></P></DIV><P>    Except for Rhamose, fluorescence was very weak and basically at the level of our negative control (E. Coli without plasmid), confirmed by a Mann-Whitney U test. Rhamnose showed some level of activity which would be considered weak, compared to, for instance, our <A href="/Team:Brasil-USP/interlabstudy">Interlab Results</A>. Finally, no considerable changes were detected in the RFP measures. Since there results are not good indicatives, we performed this experiment three different times (to reduce possible experimental artifacts with assembling or other steps in our protocol). Yet we have always observed the same behavior consistently.</P><H2>BL21 (DE3)</H2><P>    We performed the same tests in BL21 strain. For this test, Arabinose was not used as we couldn't confirm its transformation in our gels. For protocols, we refer to our <A href="https://2015.igem.org/Team:Brasil-USP/Notebook/protocols">Protocols section</A>. Again, our colonies were all in log phase (omitted) when we measured their fluorescence over time. Our results are in figure 4 and 5.</P><DIV class="fig" style="width: 750px;"><P><CENTER><FONT size="3">Figure 4 - Fluorescence per cell for pLac, normalized by the fluorescence with no induction. We see that with IPTG, GFP activity incresases and RFP activity decreases, as it should happen.</FONT></CENTER></P></DIV><DIV class="fig" style="width: 750px;"><P><CENTER><FON size="3">Figure 5 - Fluorescence per cell for each promoter, normalized by the fluorescence with no induction. Not only the induced activity is lower than the activity without induction - for the first 3 concentrations of Rhamnose -, but also the RFP is also growing.</FON></CENTER></P><P><A href="http://parts.igem.org/Part:BBa_K1819008">BBa_K1819008</A> was submitted to the Registry of Standard Biological Parts as a Brasil-USP team new part.
</P></DIV><H2>Discussion</H2><P>    We successfully expressed and induced promoters pRha in DH5α strain and pLac in BL21 strain. Mann-Whitney U tests confirmed induction, i.e., induced fluorescence per cell was (statistically) significantly higher than our negative controls. Yet, the other promoters weren't successfully expressed or induced, , this fact can be explained by a mutation in the sequence, since we weren’t able to perform the sequencing of our circuits. Another possibility for our negative results is the wrong ligations between the parts, that again we just confirmed by agarose gel instead of sequencing. Since we had no issues at all with any of our constitutive promoters (see for instance our <A href="/Team:Brasil-USP/interlabstudy">Interlab Study</A>, <A href="/Team:Brasil-USP/Parts/KI">Pveg characterization in E. Coli</A>, <A href="/Team:Brasil-USP/Parts/Teamparts">Linker-GFP part</A>), our primary concern is that our protocols should be modified and optimized for induced promoters.</P><H1>Exportation Test</H1><P>    TAs mentioned earlier, our enzymes have to be secreted so they can be in contact with the substrate, moreover it’s been seen that the heterologous expression of RoxA wasn’t successful due to inclusion body formation. The exportation of RoxA ensures its passage through the bacterial periplasm which increases the chances of a correct folding of the protein. Therefor we planned circuits with two exportation systems, the Twin-arginine translocation (TAT) sequence and the fusion of the enzymes with the bacterial outer membrane, OmpA, to check which of these exportation systems were better for each of the enzymes. The designed circuits are shown in the figure below (Figure 6). </P><DIV class="fig" style="width: 800px;"><P><CENTER><FONT size="3">Figure 6 - Exportation test circuits (more information about the design of the circuits <A href="https://2015.igem.org/Team:Brasil-USP/Project/Design">here</A>). </FONT></CENTER></P></DIV><H2>TAT signal</H2><P>    The Twin-arginine translocation (TAT) system is classified as part of the type II bacterial secretion mechanism, which is composed of two steps. The first one aims to get the protein through the bacterial cytoplasmic membrane and, for that, it can use three different pathways (the SecB-dependent pathway, the signal recognition particle (SRP), and the twin-arginine translocation (TAT)). The second step is the protein translocation through the outer membrane, involving specific protein machinery known as the secreton.<SUP>5</SUP>    The TAT pathway, unlike the SecB or the SRP pathways, is capable of transporting folded proteins across the inner membrane independently of ATP using the transmembrane PMF (proton-motive force).<SUP>5</SUP> This translocation system is composed mainly of five proteins (TatA, TatB, TatC, TatD and TatE) but their specific function have not been firmly established yet. However, it is known that TatA is the most expressed gene of this pathway and it has 60% of homology with TatE, which can partially substitute TatA; TaTB can form a complex with TatC to prevent its degradation; there is evidence that TatC is the signal peptide binding protein and, finally, TatD produces no effect on protein translocation but has DNase activity.<SUP>5</SUP> The secretion is composed by 12-16 different proteins, but none of these proteins have their specific roles elucidated.<SUP>5</SUP></P><H2>OmpA</H2><P>    The ompA mechanism is inserted in the type I secretion system  which, in Gram negative bacteria, promotes the secretion of various proteins of different sizes and functions in one single step, without the necessity of a periplasmic intermediate. The secretion is driven by three proteins of the cell envelope<SUP>6</SUP>. The first one is located in the outer membrane, and is called OMP (Outer Membrane Protein); the other two are cytoplasmic membrane proteins, called ABC (ATP-Binding Cassette) and MFP (Membrane Fusion Protein). The protein being transported has a specific C-terminal signal sequence that usually has few to 50 glycine rich-repeats, that might be necessary for the protein function and, therefore, is not cleaved during the secretion6. This signal sequence recognizes the ABC protein and triggers the assembly of the whole complex for the translocation process. 
        </P><H2>Discussion</H2><P>    We intended to evaluate the exportations systems through GFP fluorescence, since this phenomenon depends on the correct folding of the protein, which in this case is linked with the folding of Lcp and RoxA. Unfortunately we weren’t able to perform these tests, first due to our problems with promoter test and second because we are still in the circuit construction phase.
        </P><H1>References</H1><P>
        1.  Rose, K. &amp; Steinbuchel, A. Biodegradation of natural rubber and related compounds: Recent insights into a hardly understood catabolic capability of microorganisms. Appl. Environ. Microbiol. 71, 2803-2812, doi:10.1128/aem.71.6.2803-2812.2005 (2005).
2.  Rose, K., Tenberge, K. B. &amp; Steinbuchel, A. Identification and characterization of genes from Streptomyces sp strain K30 responsible for clear zone formation on natural rubber latex and poly(cis-1,4-isoprene) rubber degradation. Biomacromolecules 6, 180-188, doi:10.1021/bm0496110 (2005).
3.  Tsuchii, A. &amp; Takeda, K. Rubber-degrading enzyme from a bacterial culture. Appl. Environ. Microbiol. 56, 269-274 (1990).
4.  Bode, H. B., Kerkhoff, K. &amp; Jendrossek, D. Bacterial degradation of natural and synthetic rubber. Biomacromolecules 2, 295-303, doi:10.1021/bm005638h (2001).
5.  Mergulhão, F. J. M.; Summers, D. K.; Monteiro, G. A. Recombinant protein secretion in Escherichia coli. Biotechnology Advances. 23, 177–202 (2005).
6.  Delepelaire, P. Type I secretion in g
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>