<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Brasil-USP_interlabstudy skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Brasil-USP/interlabstudy</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar-wrapper"><DIV class="container"><NAV class="navbar navbar-inverse navbar-static-top"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="https://2015.igem.org/Team:Brasil-USP"><SPAN class="teamname">Brasil-USP</SPAN></A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:Brasil-USP">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="/Team:Brasil-USP/Project/Overview">Description</A></LI><LI><A href="/Team:Brasil-USP/Project/Brainstorming">Brainstorming</A></LI><LI><A href="/Team:Brasil-USP/Project/Design">Design</A></LI><LI><A href="/Team:Brasil-USP/Project/Reactor">Reactors</A></LI><LI role="dropdown-header" class="dropdown-header">Results</LI><LI><A href="/Team:Brasil-USP/Modeling">Modeling</A></LI><LI><A href="/Team:Brasil-USP/LcpBioinformatics">Lcp Bioinformatics</A></LI><LI><A href="/Team:Brasil-USP/Project/Molecular_Binding">Molecular Docking</A></LI><LI><A href="/Team:Brasil-USP/Project/Results">Promoters and Exportation</A></LI><LI><A href="/Team:Brasil-USP/Project/RubberDegradation">Rubber degradation</A></LI><LI><A href="/Team:Brasil-USP/Judging">Judging Criteria</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="/Team:Brasil-USP/Parts">Parts</A></LI><LI><A href="/Team:Brasil-USP/Parts/Teamparts">Team Parts</A></LI><LI><A href="/Team:Brasil-USP/Project/AssemblyMaps">Assembly Maps</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="/Team:Brasil-USP/Notebook/protocols">Protocols</A></LI><LI><A href="/Team:Brasil-USP/Notebook/Calendar">Notebook</A></LI><LI><A href="/Team:Brasil-USP/Notebook/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Policy<SPAN style="color:#FF0;">&amp;</SPAN>Practices </A></LI><LI><A href="/Team:Brasil-USP/Practices">Policy &amp; Practices </A></LI><LI role="dropdown-header" class="dropdown-header">Human Practices</LI><LI><A href="/Team:Brasil-USP/Practices/ANIP">ANIP</A></LI><LI><A href="/Team:Brasil-USP/Practices/CBL">CBL</A></LI><LI><A href="/Team:Brasil-USP/Practices/SaoCarlos8">São Carlos 8</A></LI><LI><A href="/Team:Brasil-USP/Practices/RECUR">RECUR</A></LI><LI><A href="/Team:Brasil-USP/Practices/WomanInScience">Women in Science</A></LI><LI><A href="/Team:Brasil-USP/Practices/Patent">Patent</A></LI><LI><A href="/Team:Brasil-USP/Practices/USPInovacao">USP Innovation Agency</A></LI><LI role="dropdown-header" class="dropdown-header">Events</LI><LI><A href="/Team:Brasil-USP/PolicyPractices/Jambore">Meetup (Jamboré)</A></LI><LI><A href="/Team:Brasil-USP/Practices/Events">Events</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://igem.org/Team.cgi?id=1819">Official Profile</A></LI><LI><A href="http://www.facebook.com/brasilusp" target="_blank">Like our Facebook page</A></LI><LI><A href="/Team:Brasil-USP/Team/Members">About our team</A></LI><LI><A href="/Team:Brasil-USP/Collaborations">Collaborations</A></LI><LI><A href="/Team:Brasil-USP/Attributions">Attributions</A></LI><LI><A href="/Team:Brasil-USP/Sponsors">Sponsors</A></LI><LI><A href="/Team:Brasil-USP/interlabstudy">Interlab</A></LI><LI><A href="https://2015.igem.org/Team:Brasil-USP/Entrepreneurship">Entrepreneurship</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="jumbotron PageTitle"><DIV class="container"><DIV class="col-md-7"><H1 class="PageTitle">Interlab Study</H1></DIV></DIV></DIV><DIV class="container jamboreh"><H1 style="border-bottom: 0px; margin: 0px 0px 25px 0px;">2nd International InterLab Measurement Study in synthetic biology</H1><DIV id="ToC"><H3>Table of contents</H3><UL><LI><A href="#Introduction">Introduction &amp; Motivation</A></LI><LI><A href="#Results">Results</A></LI><UL class="sub"><LI><A href="#resfluor">Fluorescence</A></LI><LI><A href="#rescytometry">Cytometry</A></LI><LI><A href="#resRPU">Rel. Promoter Units</A></LI></UL><LI><A href="#Discussions">Discussions &amp; References</A></LI><LI><A href="#Protocols">Protocols</A></LI><UL class="sub"><LI><A href="#passemble">Assemble protocol</A></LI><LI><A href="#psample">Sample preparation</A></LI><LI><A href="#pdataacq">Data acquisition</A></LI></UL></UL></DIV><P class="lead">In order to characterize standard biological parts, fluorescence-based assays have been widely used with a variety of fluorophores such as fluorescent proteins (FPs). Green fluorescent protein (GFP) is frequently applied as reporter gene to confirm gene expression and analyse its regulation. The Second International InterLab Measurement Study in synthetic biology proposes to detect three promoters activity by measuring and comparing the levels of GFP expression of three devices expressing GFP and controlled by one of the three promoters (see Figure 1). The three promoters -  BBa_J23101, BBa_J23106, BBa_J23117 - are from the Anderson library (http://parts.igem.org/Promoters/Catalog/Anderson), a constitutive promoter family with different strengths.1
      </P><DIV class="fig" style="width: 500px;"><P>Figure 1: Interlab devices 1, 2 and 3, in order from top to down.</P></DIV></DIV><DIV class="wikicontent"><H1>Introduction and Motivation</H1><P>
          In the eleventh iGEM edition happens the second Interlab Study. This study is based on the characterization of standard biological parts and, as standard parts, it is fundamental to observe reproducibility and repeatability on their behaviour. For instance, even well characterized promoters in a given strain of E. Coli may behave reasonably different in another strain. Acknowledging this challenge, the Interlab Studies is a way to gather experiments from all around the world that only have one thing in common: the biological parts to be studied. Each research team has its own strains, plasmids and protocols. With this, we can construct a rich knowledge base of standard biological parts, together with several study cases of different protocols and other details. The value this have to the whole community of Synthetic Biology is beyond doubt.</P><P>We used a plate reader and a flow cytometer to measure fluorescence with biological and technical replicates, fulfilling InterLab study requirements and the extra credit.  We evaluated the promoter strength of all devices using Relative Promoter Units [2] using DH5α E. Coli with backbone plasmid pSB1C3. Results show interesting differences: Device 2 (J23106) shows half the strength it would be expected in the original library.
        </P><H1>Results</H1><P>
          In the following subsections, we present our fluorescence measurements over time, a cytometry study showing a more microscopy view of how our cells behaved and our calculations of promoter strength using Relative Promoter Units (RPU). Notice that all of our data is publicly available here, and all software developed to calculate the results presented below are available here.
        </P><H2>Sample definitions</H2><P>We have three different samples from which we have used biological and technical triplicates. In the following we define each sample and the names we will use as reference throughout our text.</P><UL><LI><B>Negative control</B> (-control1) - cells with no plasmid, refering then to autofluorescence from DH5α E. coli cells. It was be used as baseline for our evaluations.</LI><LI><B>Negative control (with plasmid)</B> (-control2) - as a second negative control, DH5α E. coli cells containing a chloramphenicol resistant plasmid (pSB1C3) were used and no significant difference in the cells autofluorescence was detected when compared with wild DH5α E. coli cells</LI><LI><B>Positive control</B> (+control) - used the part <A href="http://parts.igem.org/Part:BBa_I20270" target="_blank">I20270</A> in the pSB1C3 backbone (chloramphenicol resistant) was also measured.</LI><LI><B>Device 1</B> (Dev1) - <A href="http://parts.igem.org/Part:BBa_J23101" target="_blank">J23101</A> promoter with <A href="http://parts.igem.org/Part:BBa_I13504" target="_blank">I13504</A>.</LI><LI><B>Device 2</B> (Dev2) - <A href="http://parts.igem.org/Part:BBa_J23106" target="_blank">J23106</A> promoter with <A href="http://parts.igem.org/Part:BBa_I13504" target="_blank">I13504</A>.</LI><LI><B>Device 3</B> (Dev3) - <A href="http://parts.igem.org/Part:BBa_J23117" target="_blank">J23117</A> promoter with <A href="http://parts.igem.org/Part:BBa_I13504" target="_blank">I13504</A>.</LI></UL><H2>Fluorescence results - plate reader</H2><P>
          As described above, we have recorded fluorescence and optical density (OD) over time of a 96-well plate organized as mentioned before. We show below fluorescence measurements after 11h of experiments, when all colonies presented an OD of about 0.5 u.a., except of course LB and LB+Cam which do not have any bacteria. Notice that we have for three biological replicates of three technical replicates, as suggested for extra credit. If you are interested in our complete data, we have made it public here. 
        </P><DIV class="fig" style="width: 600px;"><P>Table 1 (click for larger view): Fluorescence readings after 11h of experiment, not normalized by OD yet (see data below).</P></DIV><P>
          LB and LB+Cam are our baseline controls: they set the zero-level for the fluorescence. Although their levels are very close indeed, a Mann-Whitney U-statistic test showed that the antibiotic Cam is inducing a fluorescence that should not be disregarded (p-value &lt; 0.5%). Thus, we will normalize all of our measures with respect to LB+Cam activity.
        </P><P>
          To better visualize this data, we show in the figure below The activity of each device averaged for each colony (considering the standard deviation). This fluorescence measure is normalized by LB+Cam. Notice how Device 3 fluorescence is close 1.0, i.e., not much is induced more than our zero-level activity. This is somehow expected, as this promoter is indeed very weak (see Relative Promoter Units below).
        </P><DIV class="fig" style="width: 600px;"><P>Figure 2 - Fluorescence per cell for each of the three colonies and for each device. While across colonies there is some difference, within the same colony the standard deviation is rather small. For all of our cases, Device 1 is the stronger promoter, followed by Device 2. Device 3 was always the weaker promoter.</P></DIV><P>
          The activity difference is definitely greater than the standard deviation of our measurements, guaranteeing our the hierarchy Device 1 &gt; Device 2 &gt; Device 3. We can see the time evolution of the fluorescence relative to the baseline control for each device.
        </P><DIV class="fig" style="width: 700px;"><P>Figure 3 - Time evolution of the relative fluorescence per cell, normalized by the LB+Cam activity. All colonies are in their log-phase, suitable for evaluating their promoter strengths (see below). Notice that for all times, Device 1 is stronger than Device 2, while Device 3 is by far the weaker.</P></DIV><P>
          The figure above clearly shows that Device 1 induced significantly more activity. Still, Device 2 seems to perform better than Device 3. However, to quantitatively ensure that these activity differences are statistically relevant, we have performed a Mann-Whitney U-statistic test (p-value under 0.01%). Therefore, Device 1 is definitely stronger than Device 2, which in turn is also stronger than Device 3.
        </P><H2>Cytometry</H2><P>
          Here, we performed flow cytometry with biological triplicate. Because populations are well distinguished, quadrant gating method can be used to discern GFP expressing cells from those not expressing the GFP selection marker. FL1-H vs. FL2-H plots were divided in four quadrants - Q4 (lower left), Q3 (lower right), Q2 (upper left)  and Q1 (upper right) - defined by analyzing negative control with no plasmid and fitting all events in the LL quadrant. Therefore, as FL1 detector (530/30 filter) collects green emission we obtained fluorescence intensity histograms for each sample, FL1-H vs. Counts. 
        </P><H3>Negative control (no plasmid)</H3><P>
         Autofluorescence from DH5α E. coli cells and histogram markers FL1-H-  and FL1-H+  were set, therefore, any event in the Q3 quadrant represents GFP fluorescence. 
       </P><DIV class="fig" style="width: 700px;"><P>Figure 4 - <I>E. coli</I> DH5α cells. The cells were excited at 488 nm and detected using FL1-H (FITC channel). At the left, dot plot obtained by flow cytometry correspondent to autofluorescence whereby the lower left quadrant was defined to contain all cells as control. Histogram on the right shows autofluorescence levels. Both dot plot and histogram were used to set baseline and threshold.</P></DIV><H3>Negative control (with plasmid)</H3><P>
Autofluorescence with chloramphenicol resistant plasmid. 
</P><DIV class="fig" style="width: 700px;"><P>Figure 5 - <I>E. coli</I> DH5α cells with chloramphenicol resistant plasmid pSB1C3 without insert DNA. The cells were excited at 488 nm and detected using FL1-H (FITC channel). At the left, dot plots obtained by flow cytometry correspondent to cells fluorescence and, at the right, histograms analysis of fluorescence levels. </P></DIV><H3>Positive control</H3><DIV class="fig" style="width: 700px;"><P>Figure 6 - <I>E. coli</I> cells GFP expression controlled by BBa_J23151 promoter. GFP was excited at 488 nm and detected using FL1-H (FITC channel). At the left dot plots obtained by flow cytometry correspondent to BBa_I20270 and histograms on the right show analysis of GFP expression in biological triplicate</P></DIV><H3>Device 1</H3><DIV class="fig" style="width: 700px;"><P>Figure 7 - <I>E. coli</I> cells GFP expression controlled by BBa_J23101 promoter. GFP was excited at 488 nm and detected using FL1-H (FITC channel). At the left dot plots obtained by flow cytometry correspondent to Device 1 (J23101 + I13504 ) and histograms on the right show analysis of GFP expression in biological triplicate </P></DIV><H3>Device 2</H3><DIV class="fig" style="width: 700px;"><P>Figure 8 - <I>E. coli</I> cells GFP expression controlled by BBa_J23101 promoter. GFP was excited at 488 nm and detected using FL1-H (FITC channel). At the left dot plots obtained by flow cytometry correspondent to Device 2 (J23106 + I13504 ) and histograms on the right show analysis of GFP expression in biological triplicate  </P></DIV><H3>Device 3</H3><DIV class="fig" style="width: 600px;"><P>Figure 9 - <I>E. coli</I> cells GFP expression controlled by BBa_J23101 promoter. GFP was excited at 488 nm and detected using FL1-H (FITC channel). At the left dot plots obtained by flow cytometry correspondent to Device 3 (J23117 + I13504 ) and histograms on the right show analysis of GFP expression in biological triplicate </P></DIV><H3>Comparison of expression rates of J23101, J23106 and J23117</H3><P>
         
    Using flow cytometry we it is suggested that promoter strength decreases like J23101 &gt; J23106 &gt; J23117 (Device 1 &gt; Device 2 &gt; Device 3).

      </P><DIV class="fig" style="width: 700px;"><P>Figure 10 -  Comparison of GFP expression controlled by BBa_J23101, BBa_J23106 and BBa_J23117 promoters. GFP was excited at 488 nm and detected using FL1-H (FITC channel).</P></DIV><H2>Relative Promoter Units</H2><P>
         According to the Anderson Promoter Collection [1], Device 1 should be stronger among the tested promoters in this experiment. Moreover, we can numerically measure a relative promoter strength and compare with the values available in the Anderson Collection. To measure then, we have to evaluate the following equation while our colonies are in the log-phase of their growth [2]:
      \[ RPU_{\phi} = \frac{ \left\langle \frac{dF_{\phi}}{dt}\frac{1}{OD_{\phi}} \right\rangle }{ \left\langle \frac{dF_{J101}}{dt}\frac{1}{OD_{J101}} \right\rangle } \]

Notice that this measure is evaluated with respect to Device 1, thus we will be able to measure the promoter strength of devices 2 (J23106) and 3 (J23117). In compliance with the Anderson Collection, we set to 0.7 the promoter strength of the device 1 (J23101). All our colonies were in the log-phase (colocar figura!). See below our results, the expected values and the relative error.</P><P>We have also measured the promoter strengths of all controls to compare with the devices being tested, as shown below. We have normalized everything to Relative Promoter Units (RPU) using Device 1 as reference. As expected [2], different strains (and even chassis and plasmids) present different promoter strengths. Possibly, different protocols may also result in under- or over-expressing genes. Originally, most of experiments for Anderson's library were performed with W3110 strain (plasmid pSB3K3) and it was shown with DH5α strain stronger or weaker results may be observed depending on the backbone plasmid alone. According to our results (DH5α strain, plasmid pSB1C3), J23117 is so weak that its promoter strength is comparable to our negative controls, and J23106 was considerably weaker than what was indicated in W3110. Using values from [1], if J23101 was normalized to 1.0 RPU, then J23106 would be 0.67 RPU, almost twice stronger than what we have observed in our experiments.</P><DIV class="fig" style="width: 700px;"><P>Figure 11 - Promoter strengths measured in Relative Promoter Units (RPU) for all controls and devices. J23101's (Device 1) strength was used as reference, set to 1.0 RPU. As expected, our negative controls (-control 1 and 2 in the figure) have present almost no strength at all. Devices 2 and 3 show strengths pretty close to what were expected from publicly RPU measurements from Anderson Collection. Finally, our positive control (+Control in the figure) has intermediary strength, which is reasonable since it is a mutation from another promter from Anderson Collection which should be stronger than Device 3, but weaker than Device 2.</P></DIV><P>The positive control shows similar activity as the device 2. The positive control <A href="http://parts.igem.org/Part:BBa_I20270" target="_blank">I20270</A> uses the <A href="http://parts.igem.org/Part:BBa_J23115" target="_blank">J23151</A> promoter, a mutant from one of the promoters in the Anderson Collection, namely, <A href="http://parts.igem.org/Part:BBa_J23114" target="_blank">J23114</A>. It would be natural to expect, then, to behave similarly, although nothing is really certain. Using the Anderson Collection catalogue available on Registry [1], the relative strength of J23114 part is 0.10, which is stronger than Device 3, but weaker than Device 2. We show in the above figure that this is also the case for J23151.
 </P><H2>Measuring Promoter Strength using a Camera + Gimp</H2><P>All these results were obtained using high-end equipaments capable of precise and well calibrated measurements. Conversely, we wanted to make some measurements ourselves. We then wondered: what if we have only a comercial camera and no money to buy expensive software? Here is what we have done:</P><P>1- Took pictures with smooth and homogeneous luminosity, using a <A href="
https://static.igem.org/mediawiki/2015/thumb/9/9e/Team-Brasil-USP-camera-eos600.png/1200px-Team-Brasil-USP-camera-eos600.png.jpeg" target="_blank">Canon EOS 600D</A>. White backgrounds may help, but they do not seem to be necessary. Here is an example of such picture.</P><DIV class="fig" style="width: 700px;"><P>Figure 12 - Picture of one of our 96-wells plate. Devices 1, 2 and 3 are in wells A09 through B05.</P></DIV><P>2- Then we used <A href="http://www.gimp.org/" target="_blank">GIMP</A>, a free and open software considered by many a professional software for image processing, to show us how the distribution of scales of green. See below how it is done.</P><DIV class="fig" style="width: 700px;"><P>Figure 12 - Analyzing the green distribution with GIMP. To do so, first select (for selection tool, press r) the area you want to analyze. Then, open Window / Dockable Dialogues / Histogram. Finally, on the top corner of GIMP window, select the green channel.</P></DIV><P>3- Using the histogram dialogue, you have access the average values and standard deviations. All you have to do is compare.</P><P>By eye, it is almost impossible to distinguish among the different samples. Conversely, using the histogram dialogue you will be able to have a finer measure. For instance, our Device 1 showed an average of \(160.2\), while the negative control was \(152.2\). Device 3 was \(152.6\), i.e., really close to our negative control (compare with Figure 3).Although it would be better to use the formula shown above, it is also a good definition of promoter strength the simple ratio of &quot;fluorescences&quot; - in our case, the average level of green. See in the table below our results for each of the devices:</P><DIV class="fig" style="width: 700px;"><P>Figure 13 - Results for promoter strength using a commercial camera and a free software (GIMP). Column &quot;Average&quot; shows the promoter strength (ratio of levels of green between a device and Device 1).</P></DIV><P>The method is not perfect, as you can see we have got the positive control twice as strong as we got with our equipament. However, we could see a clear distinction between devices 1, 2 and 3. Also notice that device 3 is really close to the negative control, as we observed in our experiments.</P><P>We are currently developing a 3d model of a support to hold a camera still while pictures are taken. This way, we could evaluate how the levels of green evolve through time. We believe a few adjustments are necessary to really make this method a competitive, low-resolution option for quick &amp; dirty experiments. With this 3d model, everyone in the world will be able to simply print this model (nowadays, ABS is pretty cheap!) and be ready to make their own experiments.</P><H1>Discussions and References</H1><P>  We presented the promoter strengths of all devices in DH5α E. Coli  with backbone plasmid pSB1C3. Using the plate reader we measured fluorescence over time (12 hours) at room temperature. Our findings were consistent with the reported Anderson promoters strength library. J23101 exhibited the stronger promoter activity followed by J23106 and J23117 was the weakest promoter and confirmed by fluorescence reading on a plate reader and a flow cytometer. Our J23106 showed half the strength it was observed in the original Anderson's library, which is probably due to the strain and plasmid we have used.
        </P><P>
          We finished showing a way to compare promoter strengths, with lower accuracy, using a commercial camera and a free software. The results agree pretty well, except for our positive control. We believe that with few adjustments, it might be possible to create a simple and cheap equipament that anyone would be able to mount and use.
        </P><P>
[1] Anderson, J. C. &amp; Team, B. i. Anderson Promoters Colletion, <A href="http://parts.igem.org/Promoters/Catalog/Anderson" target="_blank">http://parts.igem.org/Promoters/Catalog/Anderson</A> (2006).
</P><P>
[2] Kelly et al. <A href="http://www.jbioleng.org/content/pdf/1754-1611-3-4.pdf" target="_blank">Measuring the activity of BioBri ck promoters using an in vivo reference standard</A>, Journal of Biological Engineering, 3:4 2003.
</P><H1>Protocols</H1><H2>Assemble protocol</H2><P>
          All parts to assemble the devices were on pSB1C3 vector and were resuspended from iGEM kit 2015 in 10 μL of autoclaved Milli-Q water. 2.5 μL of these resuspended DNA were transformed in  Escherichia coli DH5α cells and miniprepped to analyse bacterial clones. BBa_K823005, BBa_K823008 and BBa_K823013 (see table below) were digested with SpeI and PstI restriction enzymes (SP digestion), maintaining the promoters linked to pSB1C3 vector and then digestion products could be used in the ligation reactions (see protocols) with BBa_I13504, which was digested with XbaI and PstI (XP digestion). Three independent ligations were performed to obtain the required devices - K823005 + I13504 (Device 1), K823008 + I13504 (Device 2) and K823013 + I13504 (Device 3) - and final clones were confirmed with a digestion reaction with NotI enzyme (prefix and suffix). For restriction analysis, we selected two colonies of each device and separated population of  DNA by agarose gel electrophoresis. 
        </P><DIV class="fig" style="width: 500px;"><P>Figure 2 (click for larger view): Restriction analysis of final clones using Gene Ruler 1Kb DNA Ladder (Thermo Scientific). Because contamination could come only from non-cleaved-plasmids, a restriction gel analysis was enough to confirm the correct size of each device. A 35 bp insert band (approximate promoters length) would be seen in a negative result and also, it would not be possible to recover a insert part from a negative result in a ligation procedure (note I13504 approximate length is 875 bp); MM - molecular marker; </P></DIV><H2>Sample preparation protocols</H2><P>
          E. coli DH5α cells transformed with plasmids containing device 1, device 2 and device 3 were plated on LB agar (SIGMA) plates supplemented with chloramphenicol and grown for 18-20 hours at 37°C. Into independent sterile tubes with 5 ml of LB media (1:4) containing 34 μg mL<SUP>-1</SUP> of chloramphenicol, three colonies of each device and control were tip from the LB agar plates and grown at 37 °C, 80 rpm overnight ( for 14-16 hours).

All tests were performed in E. coli DH5α cells in  high-copy plasmid pSB1C3.
        </P><H3>Plate reader</H3><P>
          Samples were prepared in a black, flat-bottomed, 96-well Costar plate by adding a volume of 200 µL per well of each sample (180 µL of LB media and 20 µL of pre-inoculum). We also measured optical density using a clear, round-bottomed, 96-well Greiner plate.

        </P><H3>Flow cytometer</H3><P>
          Cultured bacteria were diluted to approximately 8 x 10<SUP>6</SUP> bacteria mL<SUP>-1</SUP> or OD<SUB>600</SUB> of ~ 0.01 (cell concentration must be in the range of 1x10<SUP>6</SUP> to 2 x 10<SUP>7</SUP> bacteria mL<SUP>-1</SUP> according to the equipment manufacturer’s recommendation). At this concentration, we guarantee enough cells in the samples  and avoid clusters formation. To reduce background fluorescence originated from components of the media, samples were washed twice with phosphate buffer saline (PBS) at centrifugation 5000 g for 5 minutes. The supernatant was removed and the pellet resuspended in 1 ml sheath fluid, same volume as used to obtain OD<SUB>600</SUB> of 0.01.
        </P><H2>Data acquisition protocol</H2><H3>Plate reader</H3><P>
          Data acquisition was performed in a plate reader model SpectraMax® M3 (Molecular Devices), SoftMax Pro software. To prevent cross-talk and light scattering, samples were prepared in a black plate and top read was set to acquire fluorescence measurements. 
        </P><H3>Flow cytometer</H3><P>
        Flow cytometry measurements were acquired in a BD FACSCaliburTM Flow Cytometer (Becton, Dickinson and Company, BD Biosciences, San Jose, CA, USA)  equipped with BD FACSComp™ and BD CellQuest™ Pro software. First the equipment was calibrated to ensure optimal performance due to inherent instrument variability. As model calibrators we used BD Calibrite 3 Beads (BD Calibrite™) by running FACSComp program. 
       </P><DIV class="fig" style="width: 600px;"><P>Table 2 - 3-Color Lyse/Wash FACSComp Report, Software FACSComp 6.0 </P></DIV><DIV class="fig" style="width: 600px;"><P>Table 3 - Flow cytometer calibration</P></DIV><P>

For each sample, ten thousand events were acquired and fluorescence was measured in FL1 and FL2 channels. Voltage and amplifier gain were adjusted in the Detector/Amps window according to negative control (no plasmid) (Figure negative control no plasmid) to ensure all the events are in the plot FSC vs. SSC and the same parameters were used to read GFP expressing cells and other controls. FL1 vs. FL2 plot was divided in 4 quadrants, and the lower left quadrant (LL) contained all events from the negative controls. The lower right quadrant (LR) should show up  the GFP expressing cells and histogram FL1 vs. counts, the fluorescence levels. Although, the histogram do not represents the origin of fluorescence and populations, then FL1 vs. FL2 plot can show the origin of the fluorescence. 

   </P></DIV><DIV class="container"><DIV id="footer"><P>© 2015 Brasil-USP Team
	·<A href="https://2015.igem.org/">iGEM</A>·<A href="http://www.ifsc.usp.br/english/">IFSC</A>·<A href="http://www.usp.br">USP</A></P></DIV></DIV><A class="back-to-top" href="#top">Back to top</A></DIV></DIV></DIV></DIV></BODY></HTML>