<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Brasil-USP_Modeling_GeneExpression skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Brasil-USP/Modeling/GeneExpression</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar-wrapper"><DIV class="container"><NAV class="navbar navbar-inverse navbar-static-top"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="https://2015.igem.org/Team:Brasil-USP"><SPAN class="teamname">Brasil-USP</SPAN></A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:Brasil-USP">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="/Team:Brasil-USP/Project/Overview">Description</A></LI><LI><A href="/Team:Brasil-USP/Project/Brainstorming">Brainstorming</A></LI><LI><A href="/Team:Brasil-USP/Project/Design">Design</A></LI><LI><A href="/Team:Brasil-USP/Project/Reactor">Reactors</A></LI><LI role="dropdown-header" class="dropdown-header">Results</LI><LI><A href="/Team:Brasil-USP/Modeling">Modeling</A></LI><LI><A href="/Team:Brasil-USP/LcpBioinformatics">Lcp Bioinformatics</A></LI><LI><A href="/Team:Brasil-USP/Project/Molecular_Binding">Molecular Docking</A></LI><LI><A href="/Team:Brasil-USP/Project/Results">Promoters and Exportation</A></LI><LI><A href="/Team:Brasil-USP/Project/RubberDegradation">Rubber degradation</A></LI><LI><A href="/Team:Brasil-USP/Judging">Judging Criteria</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="/Team:Brasil-USP/Parts">Parts</A></LI><LI><A href="/Team:Brasil-USP/Parts/Teamparts">Team Parts</A></LI><LI><A href="/Team:Brasil-USP/Project/AssemblyMaps">Assembly Maps</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="/Team:Brasil-USP/Notebook/protocols">Protocols</A></LI><LI><A href="/Team:Brasil-USP/Notebook/Calendar">Notebook</A></LI><LI><A href="/Team:Brasil-USP/Notebook/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Policy<SPAN style="color:#FF0;">&amp;</SPAN>Practices </A></LI><LI><A href="/Team:Brasil-USP/Practices">Policy &amp; Practices </A></LI><LI role="dropdown-header" class="dropdown-header">Human Practices</LI><LI><A href="/Team:Brasil-USP/Practices/ANIP">ANIP</A></LI><LI><A href="/Team:Brasil-USP/Practices/CBL">CBL</A></LI><LI><A href="/Team:Brasil-USP/Practices/SaoCarlos8">São Carlos 8</A></LI><LI><A href="/Team:Brasil-USP/Practices/RECUR">RECUR</A></LI><LI><A href="/Team:Brasil-USP/Practices/WomanInScience">Women in Science</A></LI><LI><A href="/Team:Brasil-USP/Practices/Patent">Patent</A></LI><LI><A href="/Team:Brasil-USP/Practices/USPInovacao">USP Innovation Agency</A></LI><LI role="dropdown-header" class="dropdown-header">Events</LI><LI><A href="/Team:Brasil-USP/PolicyPractices/Jambore">Meetup (Jamboré)</A></LI><LI><A href="/Team:Brasil-USP/Practices/Events">Events</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://igem.org/Team.cgi?id=1819">Official Profile</A></LI><LI><A href="http://www.facebook.com/brasilusp" target="_blank">Like our Facebook page</A></LI><LI><A href="/Team:Brasil-USP/Team/Members">About our team</A></LI><LI><A href="/Team:Brasil-USP/Collaborations">Collaborations</A></LI><LI><A href="/Team:Brasil-USP/Attributions">Attributions</A></LI><LI><A href="/Team:Brasil-USP/Sponsors">Sponsors</A></LI><LI><A href="/Team:Brasil-USP/interlabstudy">Interlab</A></LI><LI><A href="https://2015.igem.org/Team:Brasil-USP/Entrepreneurship">Entrepreneurship</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="jumbotron PageTitle"><DIV class="container"><DIV class="col-md-7"><H1 class="PageTitle">Gene Expression</H1><H2 class="PageTitle">Modeling results</H2></DIV></DIV></DIV><DIV class="container jamboreh"><DIV id="ToC"><H3>Table of contents</H3><UL><LI><A href="#objective">Main objective</A></LI><LI><A href="#deterministic">Deterministic modeling</A></LI><LI><A href="#enzymeproduction">RoxA and Lcp Production</A></LI><LI><A href="#hokD">Kill Switch: hokD</A></LI><LI><A href="#fitting">Fitting the parameters</A></LI><LI><A href="#fluctuations">Fluctuation analysis</A></LI></UL></DIV><P class="lead">    Here we show our models to predict gene expression from our main circuits using roxA and lcp. We also show a detailed study of the kill switch mechanism we have used.
      </P><P class="lead">    The main process we want to model in this section is the expression and translation of the genes involved in our circuits. Qualitatively, this whole process is triggered by the attachment of an enzyme that synthesizes RNA, known as RNA polymerase (RNAP), to the DNA near a gene. Promoters contain specific DNA sequences such as response elements that provide a secure initial binding site for RNA polymerase and for proteins called transcription factors that recruit RNAP. Then, mRNA associated with that gene is generated which can then be translated into a protein by ribosomes. This second process (translation) is triggered when a ribosome binds at a special region, called Ribosomal Binding Site (RBS). At the end of the day, a protein is produced.
      </P><P class="lead">    At this point, the protein might be in its function conformal structure or not. We will stop at this point for now and model this whole process from rather coarser level of details: we will simulate populations of mRNAs and proteins and predict their concentration over time. As a sneaky-peek  of what is to come, this is the basic prediction of our model:
      </P><P>    Notice that when Rhamnose concentration (black line) drops, TetR also drops and HokD starts to be produced, killing the cell.
      </P></DIV><DIV class="wikicontent"><H1>Main objective</H1><P class="lead">    Before we start dwelling with our mathematical modeling, let's state what are our main objectives and goals with this model. We want to 
        </P><OL><LI>construct a model easily adjustable to a wide variety of cell strains, chassis and different protocols.</LI><LI>predict accurately protein/enzyme concentration after long periods (stationary states).</LI><LI>take into account the inducer concentration to evaluate enzyme production.</LI></OL><P style="margin-top: 25px;">    These are the main points we will keep in mind while developing our models. In especial, our objective 1 is the most fundamental of all: as this solution to rubber degradation sill still be scaled up to an industrial level, and considering that both chassis and strain will highly be modified in such process, fitting the exact parameters from our experiments should not be the number one priority. Conversely, we want a highly scalable model that can be easily adjusted according to easy experiments that can be performed once these parameters are chosen.</P><H1>Deterministic modeling</H1><P class="lead">    As mentioned above, we want to model the gene expression. Fundamentally, RNA polymerase (RNAP) binds to a transcription factor and triggers the transcription of a gene into mRNA; subsequently, ribosomes starts the translation of that mRNA into a protein/enzyme. The schematics below summarizes it.
        </P><P class="lead">    Each process is based on the combination of two elements: first, a RNAP and the gene; then, ribosome and mRNA. The simplest way to model such combinations and their products is to use a Dynamical Systems approach <SUP>1,2</SUP>, similar to techniques employed in Population Dynamics <SUP>3</SUP>. For a introductory text on this subject, click here.
      </P><H1>RoxA and Lcp Production</H1><P class="lead">    In particular, we assume an inducible promoter such as <A href="http://parts.igem.org/Part:BBa_R0010" tagret="_blank">pLac (BBa_R0010)</A> or <A href="http://parts.igem.org/Part:BBa_K914003" target="_blank">pRha (BBa_K914003)</A>. Let \(P_{p}(t)\) be the concentration of promoter inducer over time, which will control the level of expression of our circuit. The concentration of RoxA and Lcp proteins are \(P_{r}(t)\) and \(P_{\ell}(t)\), respectively. Finally, TetR concentration is denoted by \(P_{t}(t)\), which blocks the Kill Switch mechanism, hindering cell growth by the production of HokD with concentration \(P_{h}(t)\).
        </P><P class="lead">    Let's start defining the differential equation for the production of RoxA and Lcp. Their concentration depend only on the concentration of promoter inducer and their own degradation rate. Let's define \(\delta_r\) and \(\delta_{\ell}\) as the degradation rate of RoxA and Lcp respectively. Additionally, to set the promotion levels, we define the \(\beta\) parameter. Then, the differential equations become:
        </P>
        
        \[ \frac{dP_{r}(t)}{dt} = - \delta_r P_{r}(t) + \beta \frac{P^n_{p}(t)}{K^n + P^n_{p}(t)} \]
        \[ \frac{dP_{\ell}(t)}{dt} = - \delta_{\ell} P_{\ell}(t) + \beta \frac{P^n_{p}(t)}{K^n + P^n_{p}(t)} \]
        
        <P class="lead">    Notice that we have modeled the promoter activity as a Hill function. Jointly, tetR will be also expressed and its differential equation can be written as follows:
        </P>
        
        \[ \frac{dP_{t}(t)}{dt} = - \delta_{t} P_{t}(t) + \beta \frac{P^n_{p}(t)}{K^n + P^n_{p}(t)} \]
        
        <P>To adjust the parameters, see <A href="#fitting">below</A>. For instance, below we show simulations where \(P_r(t)\) is evaluated with several different values of \(\delta_r\). They essentially change the half-life of the protein.</P><H1>Kill Switch: hokD</H1><P class="lead">    The Kill Switch mechanism induces a self-destruction mechanism, based on a toxic enzyme, HokD. It is regulated by <A href="http://parts.igem.org/Part:BBa_R0040" tagret="_blank">pTet (BBa_R0040)</A>. This assures cells won't leaeve. expression will then depend on \(P_t (t)\),
        </P>
        
        \[ \frac{dP_{h}(t)}{dt} = - \delta_{h} P_{h}(t) + \beta_0 + \beta_t \frac{1}{\frac{P^n_{t}(t)}{K^n} + 1} , \]
        
        <P class="lead">    where \(\beta_0\) is the leakeage term <SUP>2</SUP> and \(\beta_t\) defines the level of promotion of the tetR promoter. It is interesting to note that hok-RNA has a pretty long half-life time (about 30min <SUP>4</SUP>) and it is believed that hokD-RNA may present similar properties. Thus, \(\delta_{h}^{-1}\) should be of order of several minutes. 
        </P><P>    The number of cells can also be estimated, and easily adjusted to experiments (see next section). In general, cells will tend to have a mostly steady growth rate. Particularly, this will in fact depend on RoxA and Lcp concentrations, since enzymes impair cellular growth <SUP>5</SUP>. When HokD is expressed, then two trends start to compete: HokD triggers a self-destruction mechanism as new cells appear. A very simple model can be proposed to describe this time evolution:</P>
        \[ \frac{dN(t)}{dt} = \eta \left( P_{r}(t), P_{\ell} (t) \right) - \delta_{k} P_{h}(t) , \]
      <P>    These to trends are given by the rates \( \eta \left( P_{r}(t), P_{\ell} (t) \right) \)  and \( \delta_{k} P_{h}(t) \). Below, we show some simulations demonstrating the usual behavior behind this model.</P><H1>Fitting the parameters</H1><P>    We have shown all graphs with arbitrary units following topic number 1 of our main objectives. In this section, we quickly describe experiments to fit our parameters. Regarding their units, to fix all units you will only need to choose your concentration unit and a time scale. Often, you will not have concentrations, but fluorescence instead. If normalized by optical density, they can be used interchangeably. For instance, if your data is measured in hours and your concentrations are, say, PFU (Particular Fluorescence Units, i.e., your own units of fluoresnce), then units of \( beta \) will be \( \mbox{PFU} h^{-1} \). Conversely, if you have concentrations, say, in \(n\mbox{g} \mu \ell^{-1}\), then units of \( beta \) will be \( n\mbox{g} \, \mu \ell^{-1} h^{-1} \).</P><P>    To measure parameters \(\beta\) and \(\beta_t\) part <A href="http://parts.igem.org/Part:BBa_K1819000" target="_blank">K1819000</A> can be easily used (<A href="https://2015.igem.org/Team:Brasil-USP/Parts/Teamparts#partlinkergfp">submitted this year by our team</A>) to link with RoxA, Lcp and TetR. Note that <A href="https://2015.igem.org/Team:Brasil-USP/Parts/Teamparts">we have also submitted roxA and lcp genes in the BioBrick format</A>, which should make this experiment easier. By measuring then the evolution of fluorescence per cell, \(\beta\) and (\beta_t\) can be easily fitted using a variety of methods such as minimizing quadratic error.</P><P>    Degradation parameters \(\delta_x\) (with \(x=r, \ell, t, h\)) can be fitted using the same methodology, but measuring the half-time of the fluorescence decay. This should give \(2 / \mbox{log}_2(\delta_x)\).</P><P>    In particular, to evaluate \( \delta_k \) a very simple experiment can be performed: using different concentrations of tetR, optical density (OD) can be measured over time. The lower the concentration of TetR proteins, the steeper the decay in OD. Measuring the half-life of this decay, we get \(2 / \mbox{log}_2(\delta_k)\). Unfortunately, due to time constraints our experiments were not conclusive.</P><H1>Fluctuation analysis</H1><P class="lead">    Deterministic models within a context of population dynamics (i.e., populations of chemicals evolving and interacting in time) are well suited for very large concentrations, where fluctuations are indeed suppressed and a clear average effect tends to dominate. If that is not the case, fluctuations are rather important and fundamental to describe the phenomenon. To describe some level of randomness in the system (such as the possibility of RNAp not binding to a promoter) we use <I><A href="https://en.wikipedia.org/wiki/Stochastic_process" target="_blank">Stochastic models</A></I>. In general, chemical reactions are somewhat at the edge of both cases: fluctuations are expected and may give rise to interesting effects (such as stochastic resonance), but averages is a very good first step to describe our quantities of interest. However, there is a price to using Stochastic Processes for modeling: they are considerably more expensive and complex than their deterministic counterparts. Thus, beyond doubt, it would be essential for us to have some kind of control or estimation on fluctuations over the deterministic approach presented above. To achieve this, we will next introduce a classical technique in Stochastic Processes and show some analytical and numerical procedures to embed and investigate fluctuations in our modeling.</P><P class="lead">    Interestingly enough, there is a way to closely related Stochastic and Deterministic models: it is not very hard to construct a Stochastic Process whose average dynamics matches quite well with a given Deterministic model. To better understand this bridge, suppose you have a stochastic process describing the evolution of a variable \(X(t)\). You want the average evolution of \(X(t)\) to match the dynamics a given Deterministic model. You can calculate the \(n\)-th order momentum \(\langle X^n \rangle (t) = E\big[X(t)^n\big]\), where \(E[\cdot]\) denotes a probability average (or expectation), as function of time. For instance, supposing you have a master equation (as we will below), then you can manipulate it into a differential equation for \(\langle X \rangle (t)\) (i.e., average of \(X\)) that depends on \(\langle X^2 \rangle (t)\). If you write another differential equation, from you master equation, for \(\langle X^2 \rangle (t)\) it will depend on \(\langle X^3 \rangle (t)\). Guess what? The equations will never close, always depending on higher and higher order momenta. This is a problem, since you would have an infinite number of equations to solve!!</P><P class="lead">    The simples possible way to solve this issue is as follows. Thus, as a first approximation to the equation for \(\langle X \rangle (t)\) you can neglect \(\langle X^2 \rangle\) term as simply \(\langle X \rangle^2\), having then a equation that depends only on the first order momentum \(\langle X \rangle\). As a second approximation, you can go up to the equation for \(\langle X^2 \rangle (t)\) and regard \(\langle X^3 \rangle (t)\) as \(\langle X \rangle^3 (t)\). Then you will end up with two equations that depend only on \(\langle X \rangle (t)\) and \(\langle X^2 \rangle\). Techniques based on this principle are called <B>momentum-closure techniques</B>.</P><H1>References</H1><FONT size="4"><P>
1. An Introduction to Systems Biology, by Uri Alon. Chapman and Hall/CRC.

2. Control Theory and Systems Biology, by Pablo Iglesias &amp; Brian Ingalls. MIT Press. 

3. Mathematical Biology, by James D. Murray. Springer.

4. Omid R. Faridani et al. Competitive inhibition of natural antisense Sok-RNA interactions activates Hok-mediated cell killing in Escherichia coli. Nucleic Acids Research, 2006, 34:20, p. 5922

5. Birke, J.; Rother, W.; Schmitt, G.; Jendrossek, D.Functional identification of rubber oxygenase (RoxA) in soil and marine myxobacteria. Appl Environ Microbiol, 2013, v.79(20), p. 6391-9
</P></FONT></DIV><DIV class="container"><DIV id="footer"><P>© 2015 Brasil-USP Team
	·<A href="https://2015.igem.org/">iGEM</A>·<A href="http://www.ifsc.usp.br/english/">IFSC</A>·<A href="http://www.usp.br">USP</A></P></DIV></DIV><A class="back-to-top" href="#top">Back to top</A></DIV></DIV></DIV></DIV></BODY></HTML>