Team:Brasil-USP/Project/Molecular Binding
Toggle navigation
Brasil-USP
Home
Project
Description
Brainstorming
Design
Reactors
Results
Modeling
Lcp Bioinformatics
Molecular Docking
Promoters and Exportation
Rubber degradation
Judging Criteria
Parts
Parts
Team Parts
Assembly Maps
Notebook
Protocols
Notebook
Safety
Policy&Practices
Policy & Practices
Human Practices
ANIP
CBL
São Carlos 8
RECUR
Women in Science
Patent
USP Innovation Agency
Events
Meetup (Jamboré)
Events
Team
Official Profile
Like our Facebook page
About our team
Collaborations
Attributions
Sponsors
Interlab
Entrepreneurship
Molecular Docking
Project
Isoprene and RoxA
    Computer simulation and analysis are good scientific tools for understanding enzymatic processes. Molecular docking is one of this tools which predicts the structure of protein-ligand complexes.1,2 Since RoxA has just one solved structure in Protein Data Bank that does not have any ligand bounded (PDB: 4B2N), we thought about doing a molecular docking of polyisoprene in RoxA structure. The first step was to analyze RoxA structure in search for its substrate (polyisoprene) binding site. Since, this structure was obtained by RoxA crystals without substrate, its binding site was really tight which suggests that protein must undergo conformational changes to fit polyisoprene. Then we did some manual conformational modifications on residues of RoxA ligand biding site to allow molecular docking of polyisoprene. We based this conformational changes on the comparison of binding sites of RoxA and CcpA (PDB: 3HQ6) which has a same signature motif and share a common ancestry to RoxA.3 We did an alignment of corresponding heme group of each protein using a tool named LigAlign4 installed in PyMOL. The alignment result is shows in Figure 1.
Figure 1. shows the difference between both protein structures, but no significant space available can be seen in CcpA ligand biding site.
    After this, we used another PyMOL tool, named Sculpt, to modify the conformation of RoxA biding site residues. The advantage of this tool is that it does not generate conformational absurd and adapts structures around the manual modifications (Figure 2).
Figure 2. Use of Sculpt PyMOL tool in the modification of residues.
    We preferred to modify residue with bulky side chain to get more space in binding site. We can see the difference between protein before and after modification in Figure 3.
Figure 3. Binding site residues conformational changes; in blue, structure before modifications; in green, structure after modifications.
    Then we could compare the binding site surface of each structures (before and after).
Figure 4. Binding sites surfaces; left, before conformational changes; right, after conformational changes.
    Although the space in binding site has been increased, it was not possible to dock any of the polyisoprene molecules (different number of isoprene units). We tried the molecular docking in two different programs. One of the program is a free download molecular docking software, named AutoDock, and the other software we used was Surflex of Sybyl. Both program did not give any score functions.5
References
1.   Brooijmans, N.;Kuntz, I. D. Molecular Recognition and Docking Algorithms. Annual Reviews  4139 El Camino Way, P.O. Box 10139, Palo Alto, CA 94303-0139, USA,  2003.
2.   Lengauer, T.; Rarey, M. .Computational methods for biomolecular docking. Current Opinion in Structural Biology. v.6 (3), p. 402–406, 1996.
3. Seidel, J.; Schmitt, G.; Hoffmann, M.; Jendrossek, D.; Einsle, O. .Structure of the processive rubber oxygenase RoxA from Xanthomonas sp. Proc Natl Acad Sci U S A. v.110, p.13833-8, 2013.
4.  http://compbio.cs.toronto.edu/ligalign/index.html
5.   http://autodock.scripps.edu/
© 2015 Brasil-USP Team · iGEM · IFSC · USP
Back to top
