Team:Brasil-USP/LcpBioinformatics
Toggle navigation
Brasil-USP
Home
Project
Description
Brainstorming
Design
Reactors
Results
Modeling
Lcp Bioinformatics
Molecular Docking
Promoters and Exportation
Rubber degradation
Judging Criteria
Parts
Parts
Team Parts
Assembly Maps
Notebook
Protocols
Notebook
Safety
Policy&Practices
Policy & Practices
Human Practices
ANIP
CBL
São Carlos 8
RECUR
Women in Science
Patent
USP Innovation Agency
Events
Meetup (Jamboré)
Events
Team
Official Profile
Like our Facebook page
About our team
Collaborations
Attributions
Sponsors
Interlab
Entrepreneurship
Lcp Bioinformatics
I-TASSER prediction method
   Due to lack of information on Lcp (latex clearing protein) properties and structure, we decided to applied some bioinformatic tools to obtain more of these information. Unlike RoxA (Rubber oxygenase A), Lcp crystallographic structure has not been solved yet (RoxA PDB: 4B2N). It has been indeed shown that Lcp is able to link itself to a prosthetic heme group in a non-covalent way (b-type), unlike what happens in RoxA (c-type)1.
   Using prediction methods, such as Iterative Threading ASSEmbly Refinement (I-TASSER)2, we attempt to predict Lcp structure based on its amino acid sequence but results did not show good structure models using I-TASSER (Figure 1). Although, it is possible to observe that secondary structure prediction reveals that Lcp could consist of many loops and unstructured regions that are also observed in RoxA (32% helical and 7% beta sheet)3, which might suggest some similarity between these two proteins. For RoxA, a complex structure composed by three large loops forming a tower (each loop formed by helices and non-structured regions) was proposed to determine the exo-cleavage rubber polymer (Figure 3).3 Such difficulty predicting a model might be due to Lcp high complexity, distinctly from RoxA which performs exo-cleavage while Lcp performs endo-cleavage. Notice that I-TASSER was not able to predict even an heme ligand.
Figure 1 - The best model predicted by I-TASSER. It is composed of helices present in sequence and approximately in the same plan, not looking like a center that can receive a heme group. The C-score and TM scores are under acceptable values (-3.22 and 0.35±0.12 respectively).
Figure 2 - RoxA structure is a complex globular protein that contains two heme groups in its core. The catalysis happens in group heme 13. The heme groups are surrounded by alpha helices as it is characteristic of this kind of protein. The oxygen responsible for attacking the polyisoprene chain is shown in red, the helices are shown in cyan and in purple, is the beta sheet.
Figure 3 - The RoxA structure simplifying helices and loops. In this figure, loop 1 is highlighted in dark blue, loop 2 in green and loop 3 in yellow.
HMM model
   To obtain a hidden Markov model (HMM) model for Lcp, we used the Jackhammer tool in HMMER package on-line server.4 In this analysis we used the lcp sequence without TAT signal as a start sequence. Eight iterations in the program were sufficient for the model to converge. The HMM model was designed using Skylign online server (http://skylign.org/) 5. Three main regions in the HMM model with highly conserved amino acid were recognized (Figure 4). The second one has a highly conserved histidine amino acid, that suggests an oxygen-amino acid coordination, known to heme groups.
(a)
(b)
Figure 4 - Figure 4a shows the final HMM analysis wherein three regions were recognized exhibiting significant conservation. Note histidine (H) is necessary to coordinate the oxygen atom in the heme group, allowing catalysis to procede. Figure 4b shows the alignment used in the HMM model revealing that Lcp has that same histidine.6
References
1. Birke, Jakob, Wolf Röther, and Dieter Jendrossek. "Latex Clearing Protein (Lcp) of Streptomyces sp. Strain K30 Is a b-Type Cytochrome and Differs from Rubber Oxygenase A (RoxA) in Its Biophysical Properties." Applied and environmental microbiology 81.11 (2015): 3793-3799.
2. Yang, Jianyi, Renxiang Yan, Ambrish Roy, Dong Xu, Jonathan Poisson, and Yang Zhang. "The I-TASSER Suite: protein structure and function prediction."Nature methods 12, no. 1 (2015): 7-8.
3.Seidel, Julian, Georg Schmitt, Maren Hoffmann, Dieter Jendrossek, and Oliver Einsle. "Structure of the processive rubber oxygenase RoxA from Xanthomonas sp." Proceedings of the National Academy of Sciences 110, no. 34 (2013): 13833-13838.
4. Finn, Robert D., Jody Clements, and Sean R. Eddy. "HMMER web server: interactive sequence similarity searching." Nucleic acids research (2011): gkr367.
5. Wheeler, Travis J., Jody Clements, and Robert D. Finn. "Skylign: a tool for creating informative, interactive logos representing sequence alignments and profile hidden Markov models." BMC bioinformatics 15, no. 1 (2014): 7.
6. Larkin, Mark A., Gordon Blackshields, N. P. Brown, R. Chenna, Paul A. McGettigan, Hamish McWilliam, Franck Valentin et al. "Clustal W and Clustal X version 2.0." Bioinformatics 23, no. 21 (2007): 2947-2948.
© 2015 Brasil-USP Team · iGEM · IFSC · USP
Back to top
