Team:Brasil-USP/Project/Design
Toggle navigation
Brasil-USP
Home
Project
Description
Brainstorming
Design
Reactors
Results
Modeling
Lcp Bioinformatics
Molecular Docking
Promoters and Exportation
Rubber degradation
Judging Criteria
Parts
Parts
Team Parts
Assembly Maps
Notebook
Protocols
Notebook
Safety
Policy&Practices
Policy & Practices
Human Practices
ANIP
CBL
São Carlos 8
RECUR
Women in Science
Patent
USP Innovation Agency
Events
Meetup (Jamboré)
Events
Team
Official Profile
Like our Facebook page
About our team
Collaborations
Attributions
Sponsors
Interlab
Entrepreneurship
Design
Project
Main assembly design
    As we said before, the attempts to express RoxA in E. coli were frustrated due to the formation of inclusion bodies. Knowing that heterologous expression in E. coli of the entire Lcp sequence of Streptomyces sp. K30, including the TAT signal peptide, was successful 1, we decided to use the twin-arginine translocation (TAT) signal as a way to export Lcp and RoxA, increasing the chances that RoxA would acquire a functional folding. Another possibility explored in our project was the fusion of RoxA and Lcp to OmpA (Outer membrane protein A), leading to the transport of RoxA to the bacterium external surface. 2 Following we show all the possibilities for a final assembly.
    To perform the fusion of these proteins, we had to add a SacI site between the protein sequence and the suffix (for signal TAT) and prefix (for RoxA and Lcp). We chose this site because it already existed in BBa_K103006 (OmpA). Although our priority was the first assembly once Lcp is already translocated across the cytoplasmic membrane by the twin-arginine translocation pathway (TAT) in its organism of origin, we planned other assemblies to test which of the exportation systems would be best for Lcp and RoxA individually.
    Another crucial part of our main circuit was the kill-switch, ensuring that our E. coli wouldn’t survive outside of the bioreactor. As a kill-switch we chose a homologous of hok gene (hokD) 3. To control the expression of this gene we used a tetracycline-repressed promoter, as shown below.
    Tetracycline would be produced at the same time as Lcp and RoxA, inhibiting the expression of hokD. As soon as the bacteria stopped receiving rhamnose the production of tetR would end allowing the expression of hokD and cell death.
Exportation test
    As mentioned before, we planned a test to check which of the exportation systems was better for each of the enzymes. An easy way to see the expression of folded Lcp and RoxA is to fusion a Green Fluorescent Protein (GFP). Again, because of the mixed site formed by the ligation between XbaI and SpeI, which translates as a stop codon, we chose to add a NdeI site between the prefix and the sequence in the case of GFP and between the suffix and the sequence for Lcp and RoxA. The assemblies are shown below.
Promoter test
    To check the viability of our kill switch we planned two tests, using different promoters, all induced by addiction of a sugar (arabinose, rhamnose and IPTG) in the culture medium.
    The first test monitors the production of two fluorescent proteins, GFP and RFP (Red Fluorescent Protein). The idea is that in the presence of the inductor (arabinose, rhamnose or IPTG) the bacteria produces GFP and tetracycline, which will inhibits the production of RFP and in the absence of the inductor the production of GFP and tetracycline would cease and the production of RFP would get higher. The assemblies are shown below.
    The second test follows the same principle but instead of two fluorescent proteins we used GFP only. In this case, GFP would be produced if the E. coli grew in a medium lacking the inductor. The assemblies are shown below.
Parts design
Latex Clearing Protein (Lcp) and Rubber Oxygenase A (RoxA)
    We received Lcp and RoxA sequences optimized for E. coli cloned in pUC9 from Prof. Dr. Dieter Jendrossek and the first thing we did was check their biobrick compatibility searching for EcoRI, PstI, SpeI and XbaI sites. Lcp didn’t possess any of these sites but we found an EcoRI site between bases 207 and 211 of RoxA sequence. To solve this problem we planned a set of primers changing a thymine for a cytosine, keeping the asparagine codon, to perform site directed mutagenesis.
    To amplify these sequences and clone in pSB1C3 we design the primers in a way that they would contain the biobrick prefix and suffix as well as NdeI and SacI sites for protein fusion, as mentioned previously.
The primers for Lcp are the following.
    The primers for RoxA are the following.
    The meaning of the colors is in the table below.
Other parts
    Some of the parts we needed for the main assembly and exportation test were synthesized as gBlocks® Gene Fragments by Integrated DNA Technologies (IDT). One of them was the signal TAT, in addition to its sequence this part had the prefix and suffix used in biobricks  as well as a SacI site. The sequence is shown below.
SignalTAT
    Another part we decided to synthesize was the GFP used in the exportation tests. In this case we had to change a thymine for a cytosine in the blue area from 282 to 284 bases to eliminate an undesirable NdeI site. Another modification was a linker (represented in grey in the sequence below) add so the GFP wouldn’t interfere with Lcp and RoxA folding.
References
1. Yikmis, M; Arenskötter M, Rose K, Lange N, Wernsmann H, Wiefel L, Steinbüchel A. “Secretion and transcriptional regulation of the latex-clearing protein, Lcp, by the rubber-degrading bacterium Streptomyces sp. strain K30”; Appl Environ Microbiol. 74(17):5373-82., September 2008.
2. “OmpA outer membrane protein A fused to linker; displays proteins on cell surface”, available on: http://parts.igem.org/Part:BBa_K103006
3. Poulsen, L. K.; Larsen, N. M.; Molin, S.; Andersson, P; “A family of genes encoding a cell-killing function may be conserved in all Gram-negative bacteria”; Molecular Microbiology, 3(11), 1463-1472, 1989.
© 2015 Brasil-USP Team · iGEM · IFSC · USP
Back to top
