"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/Project
From 2012.igem.org
Home
Homepage
News
Team
Members
Focus Groups
Seminars
Work Impressions
Project
Our Project
Results
Discussion/Outlook
Lab Work Flow
Notebook
Overview
Materials
Methods
Computational Data
Bioinformatical Tools
iGEM
What is iGEM?
Parts Submitted
Attributions
Safety
>
Göttingen
City
University of Göttingen
Max-Planck-Institute
S1-Demo Lab
Human Practice
Our Human Practice Projects
Public and Media
Survey
Synthetic Biology Day
Panel Discussion
Flash Coli
Sponsoring
Our Sponsors
Interested in...?
Deutsch
 /  English 
Contents
1Our Project
2Chemotaxis
2.1Sensing and the mechanism of chemotaxis
2.2Tar chemoreceptor of E. coli
2.3Sensory molecules
3Poster
Our Project
Our project was born from the idea to create a real champion: the fastest E. coli in the world. As funny as this may sound first, soon we were at the development of an ambitious plan to create our "Homing Coli" and apply its speed for selective purposes. The ultimate goal was a fast swimming E. coli strain which would be able to recognize specific molecules on a mutagenized receptor and head towards gradients of these substances on swimming agar plates. If this approach worked, it might be put to use for the recognition of various molecules such as pollutants, toxins or even cancer cell markers. As our planning moved on, we soon created three different focus groups which should work in parallel on the biggest and most crucial components of our project.
The first group focuses on the creation of effective swimming motility assays. All kinds of different media and swimming agar plates were to be tested, because fast E. coli can only show their potential under the right conditions. Furthermore, an efficient selection system should be created in order to separate the fast E. coli from the slower ones and to test potential attractants for our swimmers.
Creation of a fast strain represents the main task for the second group. The main question here is: which genes have the potential to make our E. coli faster and how do they need to be regulated to achieve this? Naturally, genes that code for parts of the bacterial motor, the flagellum, were selected for testing as well as FlhDC, a master regulator for motility and chemotaxis. The output is then measured as motility on the first group's swimming plates.
The last group focuses on the directed mutagenesis of the aspartate receptor Tar. Thereby, a library of numerous different and new Tar receptors can be created. Some of these might exhibit the ability to recognize a specific substance of interest. E. coli strains possessing such mutated receptors can then be screened for homing ability towards a selection of chemical compounds.
These three groups would focus mostly on their separate projects during the early phases of lab-work and also plan their schedules independently to minimize frictional losses. But as time progresses and the first results are obtained the work of our focus groups overlaps more and more in order to achieve our ultimate goal: the creation of Homing Coli.
Chemotaxis
Sensing and the mechanism of chemotaxis
Chemotaxis is a phenomenon whereby cells or organisms direct their orientation or movement in relation to a gradient of chemical agents (Fig 1). These chemical agents are known as chemoattractants and chemorepellants, which are inorganic or organic substances like amino acids and sugars. They are able to activate chemotaxis in motile cells. This chemotaxis behavior is triggered by binding of chemoattractants or chemorepellants to chemotaxis receptors such as the target of our iGEM project, the aspartate receptor Tar.
Figure 1: Chemotaxis of E. coli. (a) When no attractant is present E. coli switches from direct swimming to tumbling randomly. (b) In the presence of an attractant E. coli moves through the gradient in the direction of the attractant. (Attractant gradient is shown in green.)
Chemotaxis is based on high-order intracellular signaling structures. Clustered receptors in the cell wall of bacteria sense signals and mediate downstream signaling in the cell via associated proteins in a highly cooperative manner [2]. These high-order intracellular signaling structures are also known as two-component systems.
Figure 2: Schematic structure of a two-component system. A histidine kinase (HK) serves as sensing structure for attractants or repellents and mediates downstream signaling to autokinase (red). The response regulator (RR) consists of a receiver (purple) and an output module (green) which if activated induces gene expression [2].
A two-component system consists of a sensory histidine kinase and a phosphorylable response regulator [2] (Fig 1). Transfer of the phosphate group from a histidine residue of the kinase domain to an aspartate residue of the response regulator activates the output domain. This normally results in activation of gene expression.
Beside the aspect that the sensing in E. coli is coupled to flagella-based motion, the two-component system is more complex. There are five histidine-kinase-associated chemotaxis receptors of E. coli known. The receptors are typically arranged as a trimeric application of dimeric receptor subunits (trimers of dimers) that are spanning through the membrane. The receptors are methyl-accepting chemotaxis proteins (MCPs) that are directly associated with CheA, a histidine autokinase and CheW, an adaptor protein that couples CheA to the receptor protein.
There are two conformational states of receptor kinases possible: the kinase-on and kinase-off state [3]. In kinase-off state the counter-clockwise (CCW) rotation is active, which leads to forward swimming. In the kinase-on state CheA autophosphorylation is activated due to repellent binding whereas in the kinase-off state autophosphorylation is inactive due to attractant binding (Fig 3).
In the case of kinase-on state, the autophosphorylated CheA transfers a phosphate group to one of the two response regulators, CheY and CheB.CheY is responsible for motor control by binding to the flagellar rotary motor. This results in clockwise (CW) rotation, which is visible as random directional movement. CheZ, a phosphatase, dephosphorylates CheY to keep random movement low (Fig 3).
The methylesterase CheB and methyltransferase CheR are counterplayers in sensory adaptation. Here, the MCPs play a crucial role. Both MCP sites have glutamines in their structure. These are functional mimics of methyl glutamates. In the case of CheB is bound to a phosphate group from CheA, it mediates deamidation of glutamines to methyl-accepting glutamates. Therefore the receptor is in the off-state with a high attractant affinity and it is likely to be methylated but not demethylated [3]. Because the kinetics of methylation and demethylation are relatively slow, adaptation can take tens to hundreds of seconds [2].
All in all, E. coli switches from tumbling to swimming when it is surrounded by a gradient of attractants. Increased attractant stimulation results in both, terminating tumbling and activation of swimming towards the attractants [2].
Figure 3: Molecular mechanism of tumbling and swimming. Activated CheA transfers a phosphate group to CheY thus activating clockwise (CW) rotation which leads E. coli tumble. CheZ dephosphorylates CheY to activate counter-clockwise (CCW) flagella rotation that results in swimming.
Figure 4: Structure of E. coli chemoreceptor Tar. Left: Ribbon diagram and chematic show of the 3D structure of Tar [3]. Right: Detail view of the 3D structure ligand binding domain of Tar (PDB file: 1WAT).
Tar chemoreceptor of E. coli
The aspartate receptor Tar (taxis to aspartate and repellents) is one member of five classical methyl-accepting chemotaxis proteins in E. coli (Aer, Tar, Tsr, Trg and Tap) that mediate chemotactic response. The whole chemoreceptor is build of three parts: a transmembrane sensing domain, a signal conversion domain and a kinase control domain (Fig 4). The transmembrane sensing domain of Tar is a four helix bundle where one bundle consists of two antiparallel helices [3].
Tar is able to sense aspartate in a high sensitive manner and a lower sensitivity for glutamate and other compounds is known [3]. The ligand binding site involves some aminoacid residues of four helices. Binding of the ligand causes a conformational change. The signal is then transmitted across the membrane through the signal conversion domain to the kinase control domain (Koshland et al., 2001) which leads to flagellar motion.
Sensory molecules
Sensory molecules are organic or inorganic agents that can be divided into two groups: chemoattractants and chemorepellents. Chemoattractants are molecules like aminoacids, organic or inorganic acids, small peptides or chemokines. They induce the active motion of the bacteria towards the highest concentration of the attractant (Fig 5). Chemorepellents have a danger signaling function. If bacteria recognize repellents, they swim away from the source of repellents (Fig 5).
Sensory molecules can be recognized by various receptors. E. coli has five of these receptors: Aer for sensing oxygen, Tar for sensing aspartate and repellents, Tsr for sensing serine and repellents, Trg for sensing ribose and galactose and Tap for sensing dipeptides. Receptors are able to mediate taxis to other sensory molecules as well but with lower affinity. Therefore, we try to find new recpetors by mutagenesis of the sensory molecule binding site of Tar.
Figure 5: Reaction of E. coli to chemoattractants and chemorepellents.E. coli swims towards the highest concentration of the chemoattractant or away from the highest concentration of the chemorepellent.
Sources:
[1] Madigan M. T., Martinko J. M., Stahl D. A., Clark D. P. (2012). Brock Microbiology. Vol. 13. Pearson, San Francisco, 78 – 80
[2] Sourjik V., Armitage J. P. (2010). Spatial organization in bacterial chemotaxis. EMBO J. 29:16, 2724 - 2733
[3] Hazelbauer G. L., Falke J. J., Parkinson J. S. (2008). Bacterial chemoreceptors: high-performance signaling in networked arrays. Trends Biochem Sci. 33:1, 9 - 19
Poster
↑ Back to top!
Retrieved from " http://2012.igem.org/Team:Goettingen/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
