<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Goettingen_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Goettingen/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Goettingen/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Goettingen/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Goettingen/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TABLE cellspacing="0" cellpadding="0"><TBODY><TR><TD colspan="2"><DIV class="MenuBar" id="navi"><UL><LI><A href="https://2012.igem.org/Team:Goettingen" style="color: white;">Home</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen"><SPAN><SPAN>Homepage</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/News"><SPAN><SPAN>News</SPAN></SPAN></A></LI></UL><LI><A href="https://2012.igem.org/Team:Goettingen/Team" style="color: white;">Team</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Team"><SPAN><SPAN>Members</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Focus_Groups"><SPAN><SPAN>Focus Groups</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Seminars"><SPAN><SPAN>Seminars</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Work_Impressions"><SPAN><SPAN>Work Impressions</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project" style="color: white;">Project</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project"><SPAN><SPAN>Our Project</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Notebook/Results"><SPAN><SPAN>Results</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Notebook/Discussion"><SPAN><SPAN>Discussion/Outlook</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Lab_Work_Flow_eng"><SPAN><SPAN>Lab Work Flow</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Notebook" style="color: white;">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Notebook"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Materials"><SPAN><SPAN>Materials</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Methods"><SPAN><SPAN>Methods</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Computational_Data"><SPAN><SPAN>Computational Data</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Bioinformatical_Tool"><SPAN><SPAN>Bioinformatical Tools</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM" style="color: white;">iGEM</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM"><SPAN><SPAN>What is iGEM?</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM/Parts"><SPAN><SPAN>Parts Submitted</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM/Attributions"><SPAN><SPAN>Attributions</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM/Saftey"><SPAN><SPAN>Safety</SPAN></SPAN></A></LI>&gt;
                                <LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen" style="color: white;">Göttingen</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen"><SPAN><SPAN>City</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen/University"><SPAN><SPAN>University of Göttingen</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen/MPI"><SPAN><SPAN>Max-Planck-Institute</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen/S1-Demo_Lab"><SPAN><SPAN>S1-Demo Lab</SPAN></SPAN></A></LI><LI style="width: 160px"><A href="https://2012.igem.org/Team:Goettingen/Human_Practice" style="color: white;">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice"><SPAN><SPAN>Our Human Practice Projects</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Public_and_Media"><SPAN><SPAN>Public and Media</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Survey"><SPAN><SPAN>Survey</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Synthetic_Biology_Day"><SPAN><SPAN>Synthetic Biology Day</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Panel_Discussion"><SPAN><SPAN>Panel Discussion</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Flash_coli"><SPAN><SPAN>Flash Coli</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Sponsoring" style="color: white;">Sponsoring</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Sponsoring#Sponsors"><SPAN><SPAN>Our Sponsors</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Sponsoring#Interested_in_Sponsoring.3F"><SPAN><SPAN>Interested in...?</SPAN></SPAN></A></LI></DIV></TD></TR><TR><TD width="865px" style="padding-top: 10px; font-size:13px; font-family:Verdana" align="right"><A href="https://2012.igem.org/Team:Goettingen/Project_deu">Deutsch </A> / <A href="https://2012.igem.org/Team:Goettingen/Project">English </A></TD></TR></TBODY></TABLE></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Our_Project"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Our Project</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Chemotaxis"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Chemotaxis</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Sensing_and_the_mechanism_of_chemotaxis"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Sensing and the mechanism of chemotaxis</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Tar_chemoreceptor_of_E._coli"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Tar chemoreceptor of E. coli</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Sensory_molecules"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Sensory molecules</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Poster"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Poster</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Our_Project"> Our Project </SPAN></H2><P>Our project was born from the idea to create a real champion: the fastest <I>E. coli</I> in the world. As funny as this may sound first,
soon we were at the development of an ambitious plan to create our &quot;Homing Coli&quot; and apply its speed for selective purposes.
The ultimate goal was a fast swimming <I>E. coli</I> strain which would be able to recognize specific molecules on a mutagenized
receptor and head towards gradients of these substances on swimming agar plates. If this approach worked, it might be put to use
for the recognition of various molecules such as pollutants, toxins or even cancer cell markers. As our planning moved on, we soon
created three different focus groups which should work in parallel on the biggest and most crucial components of our project. 
The first group focuses on the creation of effective swimming motility assays. All kinds of different media and swimming agar
plates were to be tested, because fast <I>E. coli</I> can only show their potential under the right conditions. Furthermore, an
efficient selection system should be created in order to separate the fast <I>E. coli</I> from the slower ones and to test potential
attractants for our swimmers.
Creation of a fast strain represents the main task for the second group. The main question here is: which genes have the potential
to make our <I>E. coli</I> faster and how do they need to be regulated to achieve this? Naturally, genes that code for parts of
the bacterial motor, the flagellum, were selected for testing as well as FlhDC, a master regulator for motility and chemotaxis.
The output is then measured as motility on the first group's swimming plates. 
The last group focuses on the directed mutagenesis of the aspartate receptor Tar. Thereby, a library of numerous different and new Tar
receptors can be created. Some of these might exhibit the ability to recognize a specific substance of interest. <I>E. coli</I>
strains possessing such mutated receptors can then be screened for homing ability towards a selection of chemical compounds. 
These three groups would focus mostly on their separate projects during the early phases of lab-work and also plan their schedules
independently to minimize frictional losses. But as time progresses and the first results are obtained the work of our focus groups
overlaps more and more in order to achieve our ultimate goal: the creation of Homing Coli.</P><H2><SPAN class="mw-headline" id="Chemotaxis"> Chemotaxis </SPAN></H2><H3><SPAN class="mw-headline" id="Sensing_and_the_mechanism_of_chemotaxis">Sensing and the mechanism of chemotaxis</SPAN></H3><P>Chemotaxis is a phenomenon whereby cells or organisms direct their orientation or movement in relation to a gradient of
chemical agents (Fig 1). These chemical agents are known as chemoattractants and chemorepellants, which are inorganic or
organic substances like amino acids and sugars. They are able to activate chemotaxis in motile cells. This chemotaxis behavior is
triggered by binding of chemoattractants or chemorepellants to chemotaxis receptors such as the target of our iGEM project, the aspartate receptor Tar.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:698px;"><DIV class="thumbcaption"><B>Figure 1: Chemotaxis of <I>E. coli</I>.</B> (a) When no attractant is present <I>E. coli</I> switches from direct swimming to tumbling randomly. (b) In the presence of an attractant <I>E. coli</I> moves through the gradient in the direction of the attractant. (Attractant gradient is shown in green.)</DIV></DIV></DIV></DIV><P>Chemotaxis is based on high-order intracellular signaling structures.
Clustered receptors in the cell wall of bacteria sense signals and mediate downstream signaling in the cell via associated
proteins in a highly cooperative manner [2]. These high-order intracellular signaling structures are also known as two-component systems.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Figure 2: Schematic structure of a two-component system.</B> A histidine kinase (HK) serves as sensing structure for attractants or repellents and mediates downstream signaling to autokinase (red). The response regulator (RR) consists of a receiver (purple) and an output module (green) which if activated induces gene expression [2].</DIV></DIV></DIV><P>A two-component system consists of a sensory histidine kinase and a phosphorylable
response regulator [2] (Fig 1). Transfer of the phosphate group from a histidine residue of the kinase domain to an aspartate
residue of the response regulator activates the output domain. This normally results in activation of gene expression. 
Beside the aspect that the sensing in <I>E. coli</I> is coupled to flagella-based motion, the two-component system is more complex.
There are five histidine-kinase-associated chemotaxis receptors of <I>E. coli</I> known. The receptors are typically arranged as
a trimeric application of dimeric receptor subunits (trimers of dimers) that are spanning through the membrane.
The receptors are methyl-accepting chemotaxis proteins (MCPs) that are directly associated with CheA, a histidine autokinase
and CheW, an adaptor protein that couples CheA to the receptor protein. 
There are two conformational states of receptor kinases possible: the kinase-on and kinase-off state [3]. In kinase-off state the
counter-clockwise (CCW) rotation is active, which leads to forward swimming. In the kinase-on state CheA autophosphorylation is
activated due to repellent binding whereas in the kinase-off state autophosphorylation is inactive due to attractant binding (Fig 3). 
In the case of kinase-on state, the autophosphorylated CheA transfers a phosphate group to one of the two response regulators,
CheY and CheB.CheY is responsible for motor control by binding to the flagellar rotary motor. This results in clockwise (CW) rotation,
which is visible as random directional movement. CheZ, a phosphatase, dephosphorylates CheY to keep random movement low (Fig 3). 
The methylesterase CheB and methyltransferase CheR are counterplayers in sensory adaptation. Here, the MCPs play a crucial role.
Both MCP sites have glutamines in their structure. These are functional mimics of methyl glutamates. In the case of CheB
is bound to a phosphate group from CheA, it mediates deamidation of glutamines to methyl-accepting glutamates. Therefore the
receptor is in the off-state with a high attractant affinity and it is likely to be methylated but not demethylated [3]. Because
the kinetics of methylation and demethylation are relatively slow, adaptation can take tens to hundreds of seconds [2].
All in all, <I>E. coli</I> switches from tumbling to swimming when it is surrounded by a gradient of attractants. Increased
attractant stimulation results in both, terminating tumbling and activation of swimming towards the attractants [2].</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 3: Molecular mechanism of tumbling and swimming.</B> Activated CheA transfers a phosphate group to CheY thus activating clockwise (CW) rotation which leads <I>E. coli</I> tumble. CheZ dephosphorylates CheY to activate counter-clockwise (CCW) flagella rotation that results in swimming.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 4: Structure of <I>E. coli</I> chemoreceptor Tar.</B> Left: Ribbon diagram and chematic show of the 3D structure of Tar [3]. Right: Detail view of the 3D structure ligand binding domain of Tar (PDB file: 1WAT).</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Tar_chemoreceptor_of_E._coli">Tar chemoreceptor of <I>E. coli</I></SPAN></H3><P>The aspartate receptor Tar (taxis to aspartate and repellents) is one member of five classical methyl-accepting chemotaxis proteins
in <I>E. coli</I> (Aer, Tar, Tsr, Trg and Tap) that mediate chemotactic response. The whole chemoreceptor is build of three parts:
a transmembrane sensing domain, a signal conversion domain and a kinase control domain (Fig 4). The transmembrane sensing domain of
Tar is a four helix bundle where one bundle consists of two antiparallel helices [3]. 
Tar is able to sense aspartate in a high sensitive manner and a lower sensitivity for glutamate and other compounds is
known [3]. The ligand binding site involves some aminoacid residues of four helices. Binding of the ligand causes a
conformational change. The signal is then transmitted across the membrane through the signal conversion domain to the
kinase control domain (Koshland <I>et al.</I>, 2001) which leads to flagellar motion.</P><H3><SPAN class="mw-headline" id="Sensory_molecules"> Sensory molecules </SPAN></H3><P>Sensory molecules are organic or inorganic agents that can be divided into two groups: chemoattractants and chemorepellents.
Chemoattractants are molecules like aminoacids, organic or inorganic acids, small peptides or chemokines. They induce the active
motion of the bacteria towards the highest concentration of the attractant (Fig 5).
Chemorepellents have a danger signaling function. If bacteria recognize repellents, they swim away from the source of repellents (Fig 5). 
Sensory molecules can be recognized by various receptors. <I>E. coli</I> has five of these receptors: Aer for sensing
oxygen, Tar for sensing aspartate and repellents, Tsr for sensing serine and repellents, Trg for sensing ribose and galactose
and Tap for sensing dipeptides. Receptors are able to mediate taxis to other sensory molecules as well but with lower affinity.
Therefore, we try to find new recpetors by mutagenesis of the sensory molecule binding site of Tar.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 5: Reaction of <I>E. coli</I> to chemoattractants and chemorepellents.</B><I>E. coli</I> swims towards the highest concentration of the chemoattractant or away from the highest concentration of the chemorepellent.</DIV></DIV></DIV></DIV><P>Sources:
[1] Madigan M. T., Martinko J. M., Stahl D. A., Clark D. P. (2012). Brock Microbiology. Vol. 13. Pearson, San Francisco, 78 – 80
[2] Sourjik V., Armitage J. P. (2010). Spatial organization in bacterial chemotaxis. EMBO J. 29:16, 2724 - 2733
[3] Hazelbauer G. L., Falke J. J., Parkinson J. S. (2008). Bacterial chemoreceptors: high-performance signaling in networked arrays. Trends Biochem Sci. 33:1, 9 - 19</P><H2><SPAN class="mw-headline" id="Poster"> Poster </SPAN></H2><P><TABLE width="900px" cellspacing="0" cellpadding="0"><TBODY><TR><TD><A href="#top">↑ Back to top!</A></TD></TR></TBODY></TABLE></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Goettingen/Project">http://2012.igem.org/Team:Goettingen/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Goettingen/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Goettingen/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Goettingen/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Goettingen/Project&amp;oldid=135344" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>