"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/Notebook/Discussion
From 2012.igem.org
Home
Homepage
News
Team
Members
Focus Groups
Seminars
Work Impressions
Project
Our Project
Results
Discussion/Outlook
Lab Work Flow
Notebook
Overview
Materials
Methods
Computational Data
Bioinformatical Tools
iGEM
What is iGEM?
Parts Submitted
Attributions
Safety
>
Göttingen
City
University of Göttingen
Max-Planck-Institute
S1-Demo Lab
Human Practice
Our Human Practice Projects
Public and Media
Survey
Synthetic Biology Day
Panel Discussion
Flash Coli
Sponsoring
Our Sponsors
Interested in...?
Deutsch
 /  English 
Discussion
E. coli, the pet of many molecular biologists, suffers great underestimation. Although, it is capable of much more than just cloning procedures and protein expression, many utilize it like an old nag. In our project, we allowed it to follow its most natural instinct: swimming towards its favorite honeypot. By establishing a mutant library via saturation mutagenesis, we trained E. coli to become a “tracking dog” for uncommon substances. Furthermore, we “played fetch” all day long, which eventually enabled us to establish a completely new selection assay. Last but not least, speed improvement was the third backbone-piece of our work to make E. coli a highly evolved tracing machine.
The identification of two novel Tar receptor classes that were each associated with two or three novel chemoattractants, respectively, led us to the assumption that our saturation mutagenesis and selection assay worked as expected. This is supported by the results of the negative control with water as chemoattractant in our selection assays. Unfortunately, we were not able to identify highly specific receptors that are associated to a single substance. Nevertheless, this might be due to the small coverage of about 2x105 out of 3.2x106 numerically possible mutants. Furthermore, a structural similarity of the different chemoattractants, or the presence of a universal binding pocket of the receptor might be the reason for an unspecific recognition. Our finding of two very similar amino acid sequences within one class backs up this speculation (see Fig. 18 B – Results section).
For a fully functional chemoattractant tracking system, the receptor itself is just one brick of the puzzle. The promoter constructs for the related gene were also aim of improvement. To identify the perfect promoter for the Tar membrane protein, a quantitative real-time PCR was performed. Quantitative real-time PCR measures the amount of expressed mRNA while relative fluorescence measurements quantify on protein level. In perspective of stability and half-life periods of mRNA and proteins or due to protein modification, it is comprehensive to obtain varying data sets and expression rates. Another problem that occurred during our quantitative real-time measurements was the deviation in some of the biological replicates. This problem was also observed in another iGEM group’s experiments (Kelly et al., 2009). They mentioned variations throughout various experimental conditions in the absolute activity of the BioBrick promoters. To reduce variation in promoter activity, they measured the activity of promoters relative to BBa_J23101. Furthermore, the iGEM team of Groningen, which participated in 2009, also measured the relative fluorescence of TG1 strain with the promoters J23100, J23109, and J23106 via Relative Promoter Units (RPUs). Their values indicated the comparable tendency to our documented values (see their wiki ).
As our project proceeded, we wondered about the small phenotypic effects and low protein expression levels, even with the strongest constitutive promoters in charge. Finally, sequencing approaches revealed that a ribosomal binding site (RBS) is missing in our constructs. Including this in our analysis, the low impact of even the strongest promoter was plausible. However, we did see effects, which led us to the assumption that the existence of a RBS is no essential prerequisite for basal protein expression. A paper by Brock et al. from 2008 supports this outcome as well. Furthermore, we even found a correlation between the amount of mRNA in the cell and production of the encoded protein. Taking into consideration that the physiologically necessary number of receptor copies ranges in the small magnitude of three to four figures the findings mentioned above become more plausible.
Besides the recognition of chemoattractants, a directed movement towards these is an equally important quality of E. coli. Our attempts to improve its motility were largely successful. Overexpression of the genes fliC and yhjH led to an enhanced swimming ability. However, it was hard to control all parameters of the swimming agar assay, which might be caused by its special requirements. Therefore, it was difficult to compare and associate the produced results.
Outlook
To sum up our results and findings, one can say that we achieved many of the aspired aims that were formulated during the foundation phase of our project. We were able to establish a new selection assay and elucidated two adjusting screws for E. coli’s motility. Last but not least we identified two new receptors from our Tar library and their chemical counterparts triggering chemotaxis.
Nevertheless, there is room left for improvements: We would like to achieve a higher diversity with our library and many of our designed BioBricks should work better with a suitable ribosomal binding site. The selection system itself could be further improved and even better results could be achieved using fast and well-swimming bacteria. One important issue of Synthetic Biology are “ready for the market” products. If we start to think ahead, now that we are at the end of this year’s competition, many possible applications are imaginable. Exemplarily, we would want to apply our approach to the search for cancer or tumor cells and the tracking of specific mixtures of attractants. A paper from the Arkin group at UC-Berkley already describes various modified bacteria targeting tumor cells more often than healthy cells under certain conditions.
Some of our initial questions could be answered during our experiments, but many additional ones were raised. For one, would it be possible to use probiotic bacteria instead of E. coli, to prevent pathogenicity-related risks? Could this be the basis for an advanced and directed treatment of chronic inflammatory bowel disease? Furthermore, other fields of application are possible. One could, for instance, visualize intestinal sources of disease by fluorescence markers or other detecting methods. Finally, as we speak about Synthetic Biology, the expansion of the genetic code might open even more and completely new possibilities of improving what we have accomplished so far.
↑ Back to top!
Retrieved from " http://2012.igem.org/Team:Goettingen/Notebook/Discussion "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
