"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/week10-3
From 2012.igem.org
Home
Homepage
News
Team
Members
Focus Groups
Seminars
Work Impressions
Project
Our Project
Results
Discussion/Outlook
Lab Work Flow
Notebook
Overview
Materials
Methods
Computational Data
Bioinformatical Tools
iGEM
What is iGEM?
Parts Submitted
Attributions
Safety
>
Göttingen
City
University of Göttingen
Max-Planck-Institute
S1-Demo Lab
Human Practice
Our Human Practice Projects
Public and Media
Survey
Synthetic Biology Day
Panel Discussion
Flash Coli
Sponsoring
Our Sponsors
Interested in...?
Deutsch
 /  English 
#3 Chemoreceptor Library - 10th Week
Back to overview
V07_02
V07_02_1 SDS-PAGE of promoter constructs
Experiment:
An SDS-PAGE was performed to investigate and characterize the promoter strentgh on the protein level. We chose promoters J23105 (18M) and J23106 (18O) and compared it to the overall protein expression of E. coli DH10B and E. coliΔtar.
Observations & Results:
The results were surprising. We could not observe any overexpression of proteins for both E. coli strains transformed with the BioBrick plasmids.
V07_02_2 Startpoint of Saturated mutagenesis experiment!
Experiment:
We aim to apply directed evolution to reprogram the bacterial chemoreceptor TAR in order to enable the perception of novel substances. A saturated mutagenesis PCR was performed to mutate five individual amino acid residues in the ligand binding site of TAR. The primers that are utilized are the following:
TARR6973Fw:
gcgcaGGAAAGGTCTCACTGAGTnnNTCAGCGGTAnnNATGATGATGGATTCCTCCAATCAACAAAG
TARR6973Rv:
gcgcaGAGTAGGTCTCATCAGGTTAATGCGCGTTTGCAG
TARY149F150T154Fw:
gcgcaGGAAAGGTCTCAGGAGCTnnNnnNGCTCAGCCAnnNCAGGGAATGCAAAATGCAATGGGCGAAG
TARY149F150T154Rv:
gcgcaGAGTAGGTCTCACTCCAGTATTGCCATAATCTAGGTAATC
We firstly did a test PCR with both primer pairs to verify the mutagenesis PCR protocol and conditions. As a template we use the vector pSB1C3_TAR_QC_18C isolated from E. coli DH10B since this promoter is the strongest among the promoter constructs we use. After the amplification we have multiple linear vectors that contain every possible mutation.
Observations & Results:
According to the agarose gel the PCR worked well. A band of the expected size of 3769 bp was observed for both reactions.
V07_03
1st round of mutagenesis PCR (mutation of aa residues 149, 150 and 159)
Experiment:
The saturated mutagenesis PCR was set up in a 50 µL batch according to the established protocol V07_02). The following experiments are performed as "attempt a" 1st and 2nd round. Beginning V08_20 "attempt b" was carried out to increase mutant diversity!
Observations & Results:
The corresponding gel showed bands of the expected size.
V07_04
V07_04_1 1st round: PCR clean-up
Experiment:
The clean-up was performed using peqGOLD Cycle-Pure Kit (PeqLab) following the user manual. We modified the protocol by using columns from the peqGOLD Plasmid Miniprep Kit I (PeqLab) because these can bind a higher amount of DNA (30 µg/column). Thus, we lose fewer DNA material in the cleaning steps. 50 µL of elution buffer was used.
Observations & Results:
The corresponding gel showed a band of the expected size.
V07_04_2 1st round: DpnI/BsaI digestion
Experiment:
The digestion was performed with a total volume of 50 µL according to protocol .
V07_04_3 1st round: Digestion clean-up
Experiment:
The clean-up was performed using peqGOLD Cycle-Pure Kit (PeqLab) modified by using columns from the peqGOLD Plasmid Miniprep Kit I (PeqLab) with an EB volume of 50 µL.
Observations & Results:
The corresponding gel showed a band of the expected size.
V07_04_4 1st round: Ligation
Experiment:
The ligation was set up in a 100 µL batch according to protocol . Incubation over night at 16 °C.
V07_05
V07_05_1 1st round: Ethanol precipitation of mutated plasmids
Experiment:
An ethanol precipitation was performed.
for 2000 µL ligation
- 3 M Na-Acetate pH 5.2, volume 1/10
- 5 mL EtOH 100 % (final volume 70 %)
- incubate at room temperature (RT) for 5 min
- centrifuge at 13000 rpm for 15 min at RT
- discard supernatant
- resuspend pellet in 500 µL 70 % EtOH
- centrifuge at 13000 rpm for 15 min at RT
- discard supernatant
- dry pellet for 5 min at RT
- resuspend pellet in 10 µL sterile ddH2O
Protocol is adjusted for different ligation volumes.
Observations & Results:
Unfortunately we used the wrong amount of ethanol, so that the subsequent transformation did not work successfully!
V07_05_2 1st round: Transforamtion of electrocompetent cells wir the mutant plasmid mixture
Experiment:
Electrocompetent cells were prepared according to protocol . We transformed 200 µL of cells with 10 µL of DNA. A detailed protocol for the electroporation can be found here . The transformed cells were transferred to 100 mL liquid LB medium + CM and incubated over night at 37 °C, 180 rpm. We prepared three dilution plates LB+CM (104, 105, 106) for counting colonies to verify whether the desired library diversity is reached. Plates were incubated over night at 37 °C aswell.
Observations & Results:
Continuous bug see V07_05_1!
V07_06
V07_06_1 1st round: Analysis of transformation V07_05
Experiment:
Plates and liquid culture were checked for successful transformation.
Observations & Results:
No bacterial growth on neither plates nor in liquid culture. Continuous bug see V07_05_1!
V07_06_2 1st round: Restart of saturated mutagenesis PCR
Experiment:
The PCR V07_03 was set up again + 1 backup. The clean-up was performed with the peqGOLD Cycle-Pure Kit (PeqLab). Elution in 50 µL EB.
Observations & Results:
The corresponding gel showed bands of the expected size.
Back to overview
↑ Back to top!
Retrieved from " http://2012.igem.org/Team:Goettingen/week10-3 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
