<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Goettingen_Project_Bioinformatical_Tool"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Goettingen/Project/Bioinformatical_Tool">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Goettingen/Project/Bioinformatical_Tool&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/Project/Bioinformatical_Tool&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/Project/Bioinformatical_Tool&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Goettingen/Project/Bioinformatical_Tool" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Goettingen/Project/Bioinformatical Tool</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TABLE cellspacing="0" cellpadding="0"><TBODY><TR><TD colspan="2"><DIV class="MenuBar" id="navi"><UL><LI><A href="https://2012.igem.org/Team:Goettingen" style="color: white;">Home</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen"><SPAN><SPAN>Homepage</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/News"><SPAN><SPAN>News</SPAN></SPAN></A></LI></UL><LI><A href="https://2012.igem.org/Team:Goettingen/Team" style="color: white;">Team</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Team"><SPAN><SPAN>Members</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Focus_Groups"><SPAN><SPAN>Focus Groups</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Seminars"><SPAN><SPAN>Seminars</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Work_Impressions"><SPAN><SPAN>Work Impressions</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project" style="color: white;">Project</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project"><SPAN><SPAN>Our Project</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Notebook/Results"><SPAN><SPAN>Results</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Notebook/Discussion"><SPAN><SPAN>Discussion/Outlook</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Lab_Work_Flow_eng"><SPAN><SPAN>Lab Work Flow</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Notebook" style="color: white;">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Notebook"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Materials"><SPAN><SPAN>Materials</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Methods"><SPAN><SPAN>Methods</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Computational_Data"><SPAN><SPAN>Computational Data</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Bioinformatical_Tool"><SPAN><SPAN>Bioinformatical Tools</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM" style="color: white;">iGEM</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM"><SPAN><SPAN>What is iGEM?</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM/Parts"><SPAN><SPAN>Parts Submitted</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM/Attributions"><SPAN><SPAN>Attributions</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM/Saftey"><SPAN><SPAN>Safety</SPAN></SPAN></A></LI>&gt;
                                <LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen" style="color: white;">Göttingen</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen"><SPAN><SPAN>City</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen/University"><SPAN><SPAN>University of Göttingen</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen/MPI"><SPAN><SPAN>Max-Planck-Institute</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen/S1-Demo_Lab"><SPAN><SPAN>S1-Demo Lab</SPAN></SPAN></A></LI><LI style="width: 160px"><A href="https://2012.igem.org/Team:Goettingen/Human_Practice" style="color: white;">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice"><SPAN><SPAN>Our Human Practice Projects</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Public_and_Media"><SPAN><SPAN>Public and Media</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Survey"><SPAN><SPAN>Survey</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Synthetic_Biology_Day"><SPAN><SPAN>Synthetic Biology Day</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Panel_Discussion"><SPAN><SPAN>Panel Discussion</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Flash_coli"><SPAN><SPAN>Flash Coli</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Sponsoring" style="color: white;">Sponsoring</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Sponsoring#Sponsors"><SPAN><SPAN>Our Sponsors</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Sponsoring#Interested_in_Sponsoring.3F"><SPAN><SPAN>Interested in...?</SPAN></SPAN></A></LI></DIV></TD></TR><TR><TD width="865px" style="padding-top: 10px; font-size:13px; font-family:Verdana" align="right"><A href="https://2012.igem.org/Team:Goettingen/Project/Bioinformatical Tool">Deutsch </A> / <A href="https://2012.igem.org/Team:Goettingen/Project/Bioinformatical Tool">English </A></TD></TR></TBODY></TABLE></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Link_List_of_Bioinformatical_Tools"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Link List of Bioinformatical Tools</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#A_plasmid_Editor_ApE"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">A plasmid Editor ApE</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Primer_Design_with_ApE"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Primer Design with ApE</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Primer_Binding_with_ApE"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Primer Binding with ApE</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Find_restriction_sites_and_fragment_lenghts_with_ApE"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Find restriction sites and fragment lenghts with ApE</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Blasting_Sequences"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Blasting Sequences</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Finding_the_ORF"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Finding the ORF</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#General_Remarks"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">General Remarks</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-9"><A href="#Blasting_of_Sequences"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Blasting of Sequences</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Tm_calculator"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">T<SUB>m</SUB> calculator</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Double_Digest_Finder"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Double Digest Finder</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Enzyme_Finder"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Enzyme Finder</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#NEBcutter_V2.0"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">NEBcutter V2.0</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#ORF_Finder"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">ORF Finder</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#RE_Base"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">RE Base</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#WebLogo"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">WebLogo</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Link_List_of_Bioinformatical_Tools"> Link List of Bioinformatical Tools </SPAN></H2><P>Here, an useful link list of computational and bioinformatical tools is provided. All of these programs are regularly used by the students to do the cloning process.
</P><UL><LI><A href="http://biologylabs.utah.edu/jorgensen/wayned/ape/" class="external text" rel="nofollow">A plasmid Editor ApE</A> - Use of ApE 
</LI><LI><A href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" class="external text" rel="nofollow">Blasting of Sequences</A> - BLAST: Basic Local Alignment Search Tool of NCBI 
</LI><LI><A href="http://www.iit-biotech.de/iit-cgi/oligo-tm.pl" class="external text" rel="nofollow">T<SUB>m</SUB> calculator</A> - Calculator for Primer Melting Temperature 
</LI><LI><A href="http://www.fermentas.com/en/tools/doubledigest/" class="external text" rel="nofollow">Double Digest Finder</A> - Double Digest Finder of Thermo Scientific 
</LI><LI><A href="http://www.neb.com/nebecomm/EnzymeFinderSearchbySequence.asp" class="external text" rel="nofollow">Enzyme Finder</A> - Enzyme Finder of NEB 
</LI><LI><A href="http://tools.neb.com/NEBcutter2/" class="external text" rel="nofollow">NEBcutter V2.0</A> - NEBcutter V2.0 
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/gorf/gorf.html" class="external text" rel="nofollow">ORF Finder</A> - ORF Finder of NCBI 
</LI><LI><A href="http://rebase.neb.com/rebase/rebase.html" class="external text" rel="nofollow">RE Base</A> - The Restriction Enzyme Database RE Base of NEB 
</LI><LI><A href="http://weblogo.berkeley.edu/logo.cgi" class="external text" rel="nofollow">WebLogo</A> - WebLogo  
</LI></UL><H2><SPAN class="mw-headline" id="A_plasmid_Editor_ApE"> A plasmid Editor ApE </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:Goett_Ape_BG.gif" title="File:Goett Ape BG.gif">File:Goett Ape BG.gif</A><DIV class="thumbcaption">The Ape Logo</DIV></DIV></DIV> The plasmid Editor ApE by M. Wayne Davis is a free ware program conceived for both Windows (XP, Vista and 7) and Mac (OS X v10.5 and above). It can be downloaded <A href="http://biologylabs.utah.edu/jorgensen/wayned/ape/" class="external text" rel="nofollow">here</A>. The program offers lots of applications required for cloning processes, e.g. construction of plasmid maps, primer design, sequence alignments, management of sequences, ORF finder, Tm calculator, translation of nucleotide sequences and a lot more. ApE is compatible for the handling of common sequencing formatted .seq and .ab1 files. For further information visit the ApE homepage. Moreover, the hoster´s of the program take care about an <A href="http://ape-a-plasmid-editor.wikispaces.com/" class="external text" rel="nofollow">ApE Wiki</A> where you can find help and advice if questions in the use of ApE pop up.
<H3><SPAN class="mw-headline" id="Primer_Design_with_ApE"> Primer Design with ApE </SPAN></H3><P>Primers depend mainly on the chosen criteria. Yet, the specifity and the tendency to form hair-pins drastically reduces straight forward PCR amplification of genes.
ApE also offers a primer design feature. 
</P><OL><LI> Highlight the sequence to which the primer should bind. Recognize that given standard minimum and maximum length is 20 bp and 25 bp, respectively. Nevertheless, these options can be altered. 
</LI><LI> Select &quot;Tools -&gt; Find Primers&quot;. A new window Find primers will emerge offering lots of different manipulatable attributes. 
</LI><LI> Click &quot;OK&quot; to see the possible primers. 
</LI></OL><H3><SPAN class="mw-headline" id="Primer_Binding_with_ApE"> Primer Binding with ApE </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Fig. 1: ApE window for binding of primers</B></DIV></DIV></DIV><P>In this example, ApE is used to find the binding site on Lambda DNA for primer with a specific sequence.  
</P><OL><LI> Open ApE and paste the desired sequence, for which the primer is designated, in the sequence field (or open according .ape file, if made before. 
</LI><LI> Go to Edit&gt;Find (or use Ctrl+F). 
</LI><LI> Paste the primer sequence and check &quot;also find rev-com of string&quot; (the primer may be identical to (a part of) the opposite DNA strand.) and click the &quot;Find Next&quot; button, as shown in Fig. 1.  
</LI><LI> The in blue highlighted sequence appears to be the desired primer sequence annealing to the template sequence. Check also if primer is indeed the reverse complement. Below the icon bar, one can find information about the length of the primer 30 nucleotides and the binding site. 
</LI></OL><P>For an illustrative description of each steps, please follow <A href="http://www.bioinformatics.nl/molbi/SCLResources/Bioinformatics.htm#Finding_primer_binding_sites_using_ApE" class="external text" rel="nofollow">this link</A></P><H3><SPAN class="mw-headline" id="Find_restriction_sites_and_fragment_lenghts_with_ApE"> Find restriction sites and fragment lenghts with ApE </SPAN></H3><OL><LI> Start ApE and open desired sequence. 
</LI><LI> Go to the Enzyme Selector (or use Ctrl+E) and click on the wanted restriction enzymes; the amount of restriction sites will be indicated in brackets.
</LI><LI> One can now continue with different options:
</LI></OL><UL><LI> searching different restriction enzymes with particular properties applying the various buttons in the &quot;Select panel&quot;,
</LI><LI> or highlighting the restiction site within the given sequence by pressing &quot;Highlight&quot; button, 
</LI><LI> or simulating a restriction <I>in silico</I> by klicking on the &quot;Digest&quot; button.
</LI></UL><P>A new window will appear showing the digestion results. Additional information about the different bands can be received by hovering the mouse arrow over the bands, map or text. A convenient way to see the respective sequence behind a digested fragment is by simply klicking on the fragment within the digest window.
For a description with the use of pictures for each step, please follow <A href="http://www.bioinformatics.nl/molbi/SCLResources/ApE_and_lambda.htm" class="external text" rel="nofollow">this link</A>.
</P><H3><SPAN class="mw-headline" id="Blasting_Sequences"> Blasting Sequences </SPAN></H3><P>For blasting two Sequences:
</P><OL><LI> Open two sequences on ApE. 
</LI><LI> Select &quot;Tools -&gt; Align Two Sequences&quot;.
</LI></OL><P>For blasting multiple Sequences:
</P><OL><LI> Open multiple sequences on ApE. 
</LI><LI> Select &quot;Tools -&gt; Align Sequences&quot;. 
</LI></OL><P>It is important to note that the opening chronology of the ApE files will matter in the order of the alignment sequences. Those will show up in according sequence from top to bottom.
Mismatches will be highlighted in red color, whereas matches will use the respective nucleotide linked with a dash. By double-clicking on any base pair within the sequence alignment, the sequence corresponding to this location will appear in the sequence ApE window.
</P><H3><SPAN class="mw-headline" id="Finding_the_ORF"> Finding the ORF </SPAN></H3><OL><LI> Under &quot;ORFs -&gt; Find Next (or Ctrl+&gt;) / Find Previous (or Ctrl+&lt;)&quot; open reading frames, i. e sequences beginning with a start codon ATG and end with one of the stop codons, become visible.  
</LI><LI> These ORFs can be translated using &quot;ORFs -&gt; Selection Translate&quot; for direct amino acid sequence in the current displayed selection or by creating a new window &quot;ORFs (or Ctrl+T)-&gt; Translate&quot; listing the aa sequence and enabling to link the amino acids with the nucleotides, respectively. The translated sequence can be chosen in 1- or 3-letter code. 
</LI></OL><H3><SPAN class="mw-headline" id="General_Remarks"> General Remarks </SPAN></H3><UL><LI> While copying features from one file to another, it is obligatory to have the file containing those features still open.   
</LI><LI> Addition of extra enzymes is possible only via the &quot;Enzyme Editor (or Ctrl+E) -&gt; Enzymes -&gt; New Enzyme&quot;. To save these changes in the program folders ApE, use &quot;Enzyme Editor (or Ctrl+E) -&gt; File -&gt; Save Current Enzymes as Default&quot;. For permanent saving exceeding new installing of the program, copy your Default file within the ApE program pathway. After installing paste this file in the corresponding folder. 
</LI><LI> Features in the DNA sequences can be labeled, selecting &quot;Features (or Ctrl+.) -&gt; New Feature&quot; allowing to mark the standard BioBrick restriction sites obvious or e.g. antibiotic resistances. These annotations are also saved. Moreover, it has a preexisting Feature Library which can be used for easily see important regions. 
</LI></UL><H2><SPAN class="mw-headline" id="Blasting_of_Sequences"> Blasting of Sequences </SPAN></H2><P>This <A href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" class="external text" rel="nofollow">link</A> allows for alignment of two or even multiple given sequences.
To become more informed how to do the sequence alignment with BLAST follow <A href="http://www.bioinformatics.nl/molbi/SCLResources/Bioinformatics.htm#Alignment_of_two_given_sequences_" class="external text" rel="nofollow">here</A>.
</P><H2><SPAN class="mw-headline" id="Tm_calculator"> T<SUB>m</SUB> calculator </SPAN></H2><P>This <A href="http://www.iit-biotech.de/iit-cgi/oligo-tm.pl" class="external text" rel="nofollow">link</A> provides an online calculation form for the melting temperature T<SUB>m</SUB> of
PCR primers referring to the so-called Nearest Neighbour method. 
</P><H2><SPAN class="mw-headline" id="Double_Digest_Finder"> Double Digest Finder </SPAN></H2><P>This <A href="http://www.fermentas.com/en/tools/doubledigest/" class="external text" rel="nofollow">link</A> helps to find the appropriate reaction conditions while cleaving a DNA
substrate with two restriction enzymes simultaneously as double digestion is a common timesaving procedure. The Thermo Scientific Double Digest
Finder tool is applied by selecting the two restriction enzymes and submitting the query. It gives the reaction conditions amenable to any two
Thermo Scientific restriction enzymes. 
</P><H2><SPAN class="mw-headline" id="Enzyme_Finder"> Enzyme Finder </SPAN></H2><P>This <A href="http://www.neb.com/nebecomm/EnzymeFinderSearchbySequence.asp" class="external text" rel="nofollow">link</A> leads to a tool which allows for selection of restriction
enzymes by name, sequence, overhang, or type. It should be noted that the single letter code nomenclature should be entered for restriction sites.
The search results appear in a list having enzymes supplied by NEB at the top with corresponding displayed links.
</P><H2><SPAN class="mw-headline" id="NEBcutter_V2.0"> NEBcutter V2.0 </SPAN></H2><P>This <A href="http://tools.neb.com/NEBcutter2/" class="external text" rel="nofollow">link</A> will lead one to the NEBcutter: a program to cleave DNA with restriction enzymes.
By submitting a maximum size of the input file of 1 MByte or a maximum sequence length of 300 kb, a linearized sequence restriction map will
be the result containing the marked restriction enzyme sites. The map also indicates the fashion of the sticky or blunted end cutters.
Moreover, additional effects like methylation of the DNA sequence are given, too. 
</P><H2><SPAN class="mw-headline" id="ORF_Finder"> ORF Finder </SPAN></H2><P>&quot;The ORF Finder (Open Reading Frame Finder) is a graphical analysis tool which finds all open reading frames of a selectable minimum size
in a user's sequence or in a sequence already in the database. This tool identifies all open reading frames using the standard or alternative
genetic codes. The deduced amino acid sequence can be saved in various formats and searched against the sequence database using the WWW BLAST server.
The ORF Finder should be helpful in preparing complete and accurate sequence submissions. It is also packaged with the Sequin sequence submission software&quot;
(Quote from: <A href="http://www.ncbi.nlm.nih.gov/project/gorf/" class="external text" rel="nofollow">http://www.ncbi.nlm.nih.gov/project/gorf/; 06/29/2012</A>).
</P><P>This <A href="http://www.ncbi.nlm.nih.gov/gorf/gorf.html" class="external text" rel="nofollow">link</A> aids to find all open reading frames (ORFs) in a given sequence.
It is an ideal alternative to the ApE ORF finding feature if an ApE sequence file has not been constructed yet or if ApE is not installed at the
used computer. 
</P><H2><SPAN class="mw-headline" id="RE_Base"> RE Base </SPAN></H2><P>This <A href="http://rebase.neb.com/rebase/rebase.html" class="external text" rel="nofollow">link</A> encompasses comprehensive information about restriction enzymes and is the
official RE Database by NEB. It provides for instance information of suppliers,recognition sequence, isoschizomers and the restriction enzyme type.
In regard of a single enzyme also the organism and organism type with respective growth temperature behind the abbreviation is listed. 
</P><P>Choose the search enzyme names or the recognition sequence search catagory, and type a keyword (e.g. ecor or gaattc) to quickly find info about
specific enzymes and click REbase list to find more specialised information. In this example, BamHI was searched within the RE Base:
</P><UL><LI> Acronym: BamH 
</LI><LI> Prototype: <I>BamH</I>I 
</LI><LI> Org #: 208
</LI><LI> Organism: Bacillus amyloliquefaciens H
</LI><LI> Organism type: bacteria 
</LI><LI> Organism source: ATCC 49763 (ATCC LINK)
</LI><LI> Growth Temperature: 37°C
</LI><LI> Experimental Evidence: biochemistry
</LI><LI> Exhibits star activity 
</LI><LI> Enzyme gene cloned.
</LI><LI> Enzyme gene sequenced. 
</LI><LI> Crystal data present.
</LI><LI> Kinetics data present. 
</LI><LI> Molecular Weight: 24569   
</LI></UL><P>Have also a look at the <A href="http://www.neb.com/nebecomm/EnzymeFinder.asp" class="external text" rel="nofollow">Enzyme Finder</A> as alternative source of information.
</P><H2><SPAN class="mw-headline" id="WebLogo"> WebLogo </SPAN></H2><P>This <A href="http://weblogo.berkeley.edu/logo.cgi" class="external text" rel="nofollow">link</A> encompasses a web based application. It will help you to generate sequence logos easily. To create your own sequence logos it is important to enter the sequence data only in the allowed file type:
</P><UL><LI> FASTA format
</LI><LI> CLUSTALW format
</LI><LI> Flat format.
</LI></UL><P><TABLE width="900px" cellspacing="0" cellpadding="0"><TBODY><TR><TD><A href="#top">↑ Back to top!</A></TD></TR></TBODY></TABLE></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Goettingen/Project/Bioinformatical_Tool">http://2012.igem.org/Team:Goettingen/Project/Bioinformatical_Tool</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Goettingen/Project/Bioinformatical_Tool" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Goettingen/Project/Bioinformatical_Tool" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Goettingen/Project/Bioinformatical_Tool&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Goettingen/Project/Bioinformatical_Tool&amp;oldid=178767" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>