"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/Project/Methods
From 2012.igem.org
Home
Homepage
News
Team
Members
Focus Groups
Seminars
Work Impressions
Project
Our Project
Results
Discussion/Outlook
Lab Work Flow
Notebook
Overview
Materials
Methods
Computational Data
Bioinformatical Tools
iGEM
What is iGEM?
Parts Submitted
Attributions
Safety
>
Göttingen
City
University of Göttingen
Max-Planck-Institute
S1-Demo Lab
Human Practice
Our Human Practice Projects
Public and Media
Survey
Synthetic Biology Day
Panel Discussion
Flash Coli
Sponsoring
Our Sponsors
Interested in...?
Deutsch
 /  English 
Contents
1Agarose Gel Electrophoresis
1.1Pouring the Gel
1.2Loading the Gel
1.3Running the Gel
1.4Staining the Gel
2Cloning Protocols
2.1Cloning Protocols: Restriction (double digest)
2.2Cloning Protocols: Ligation
2.3Cloning Protocols: Chemical Transformation
3Competent Cells
3.1Chemical Competent Cells
3.2Electrocompetent Cells
4Transmission Electron Microscopy, negative staining with 2 % PTA
4.1Strains
4.2Media and buffers
4.3Execution
5Library Selection
5.1Attractants
5.2Media
5.3Execution
5.3.1First round of selection
5.3.2Second round of selection
5.3.3Third round of selection
5.3.4Plating of the selected clones
5.3.5Minipreparation and sequencing of plasmid DNA
5.3.6Retransformation of the plasmid DNA
5.3.7Determination of the swimming behaviour of the freshly transformed BL21 cells
6Overlap PCR for removal of restriction sites
7Separation Assay
7.1Strains
7.2Media
7.3Execution
7.3.1Applying the mixed culture to the 0.3 % tryptone swimming agar plates
7.3.2Separation of the different strains
8Sequencing
9Standard PCR
9.1-> 50 μl / reaction:
9.2Program:
10Swimming/Chemotaxis Assay
10.1Strains
10.2Media
10.3Execution
11QuikChange Protocol
11.1-> 20 μL / reaction:
11.2Program: -> 1 min 96 °C
11.3-> 5 min 72 °C -> Store at 4-8 °C
12Quantitative Real-Time PCR
12.1Experimental design
12.2Primer for qrtPCR:
12.320 μl / reaction:
12.4Program:
12.5Analysis via the 2–ΔΔCT (Livak) Method
13SDS-PAGE
Agarose Gel Electrophoresis
For the analysis of PCR-amplified products, agarose gel electrophoresis is the method of choice. This method takes advantage of the separation of DNA in dependance of the charge-mass ratio. The separation is based on the electric attraction of the negative charged DNA which is guided towards the positive charged anode upon application of a current. The PCR samples are run on agarose gels with different percentages according to the product sizes: small products run faster than bigger products. Later on, these fragments within the gel are made visible by examination under the UV light to ensure the correct DNA fragment length synthesized in the PCR reaction. Prior to UV analysis, a staining method of the DNA, here using ethidium bromide (EtBr), is obligatory. EtBr is an intercalating agent which embeddes itself within the DNA helix. Thus, the absorption spectrum is biased so that it is suitable for DNA detection. The determination of separated molecule sizes is done accodrding to a common DNA size standard.
Pouring the Gel
1 % agarose gels are standard to separate DNA. The percentages and thus the degree of polymerisation of the gel influences the degree of separation, i.e. PCR products of similar size can be distinguished by applying a percentage lower than 1 %. To pour a 1 % agarose gel weight 1 g of Ultra Pure Agarose for every 100 ml of 1x TAE buffer. For preparation of 1x TAE buffer fill 10 mL of 50x TAE buffer stock ad 500 mL ddH2O. Ensure that the 1 % agarose in 1x TAE buffer is boiled throughly and dissolved completely without streaks. Incomplete boiling will bias the separation results. The liquid is poured in a gel tray with the wanted comb which are assembled in a holder. After solification the gel is placed in a gel chamber and is fully covered with 1x TAE buffer post removal of the comb.
Loading the Gel
Prior to loading, the DNA samples are homogenized with 6x Loading Dye (LD) resulting in a 1x final concentration of the LD (4 μL 6x LD ad 20 μL PCR reaction). Be aware of the fact, that PCR samples should not be mixed with LD if further experiments with the samples are needed to be done. If this is the case, mix the PCR samples with LD on parafilm. Meanwhile, the PCR samples are kept on ice. The 1 kb DNA ladder is stored in the 4 °C fridge and is suitable for larger PCR products ranging from 250 bp to 10000 bp (GeneRuler 1 kb DNA Ladder (Fermentas)). Load the gel with x μL of LD-mixed samples next to the first well with 5 μL marker. The amount loaded for PCR samples depends on the concentration and the size of the products. Note that PCR tubes should be closed to prevent drying up exposed to air.
Running the Gel
When all samples have been loaded, connect the power supply. Ensure that the plus pol is at the opposite site of the loaded wells. Run big gels applying 100 V for a more accurate separation for about 1 h. The time may be adjusted according to the loading front.
Staining the Gel
To make the separated DNA bands visible under the UV light, the samples were stained in a EtBr bath for about 10 min. Then, the gel was rinsed in water for approximately 5 min. The EtBr stained gel was exposed to UV light under a gel documentation station, a gel photo was taken and saved in the iGEM2012 folder. The file should be labelled with the date, group number and short description of what was analyzed. A printed version should be pasted into the group notebook.
Cloning Protocols
Cloning Protocols: Restriction (double digest)
choose the correct restriction enzymes depending on your backbone and insert (e.g. backbone: SpeI and PstI; insert: XbaI and PstI)
calculate the appropriate amount of enzyme which is necessary for a 5 fold over-digestion. Therefore fill in all parameters of the following equation:
Keep in mind that you have to calculate the enzyme amounts separately for backbone AND insert (if the differ in size). The result of th the equation must be divided by 10 because most enzymes have a concentration of 10 U / µL. Finally, you have to multiply the value by factor 5 for 5 fold over-digestion in order to guarantee a complete digestion of the parts.
choose the buffer by using the following tool: Fermantas double digest
prepare reaction mix: 1 µg template, buffer, enzyme 1 and enzyme 2 (dependent on the concentration it is often necessary to add purified water)
Incubation for 1.5 h at 37 °C
Cloning Protocols: Ligation
Combination of 50 ng of vector with a 2-fold molar excess of insert.
Addition of an appropriate volume of T4 Buffer.
Addition of 0.5-1 μL T4 ligase.
Adjustment of volume to 20 μL with ddH2O
Incubation over night at 16 °C
Deactivation of ligase for 10 min at 65 °C
Cloning Protocols: Chemical Transformation
Thawing of competent cells on ice
Addition of 1 μL DNA
Incubation on ice for 20 min
Heat shock for 1 min at 42 °C
Incubation on ice for 5 min
Addition of 270 μL LB
Incubation in thermoblock for 45 min (incubation time dependent on used vector!) at 37 °C and 300 rpm
Smearing of cells on plates and incubation at 37 °C (centrifuge it, discard supernatant, resuspend cells in rest-medium, 20 μL on plate)
Competent Cells
Chemical Competent Cells
Preparation of CaCl2 buffer for competent cells! Before you start make sure that the CaCl2 buffer is ice-cold when needed and the centrifuge is cooled to 4 °C.
Inoculate LB-liquid medium with 1 ml of overnight culture.
Let cells grow to OD600 0.4-0.6.
Transfer them to 50 mL Falcons and centrifuge for 5 min at 1500 g and 4 °C.
Resuspend pellet in 5-10 mL CaCl2 buffer; here 7 mL (prechilled).
Centrifuge 5 min 1500 g, 4 °C
Resuspend pellet in 5-10 mL CaCl2 buffer; here 7 mL (prechilled).
Incubate cells 30-60 min on ice; here 30 min; Eppis prechilled in 5-10 mL on dry ice (or in liquid nitrogen)
Storage at -80 °C
Electrocompetent Cells
Cells grow to OD600 0.5
2x washing with 10 mL % 15 Glycerol
Pellets are resuspend in backwash, pooled to about 1mL
centrifuge at 4800 rpm at 4°C for 5 min
remove uper clear part
electroporation (100µL per cuvette)
Transmission Electron Microscopy, negative staining with 2 % PTA
The TEM and the sample preparation was conducted by Dr. Michael Hoppert.
Strains
MG1655
BL21 J61002_rfp
BL21 pSB1C3_QC_fliC_18C
BL21 pSB1C3_QC_flhDC_18C
Media and buffers
LB-broth
LB-agar
2 % PTA (phosphotungstic acid)
Execution
Inoculate the strains of interest in 5 mL L-broth with necessary antibiotics respectively
Incubate them at 37 °C with 100 rpm (to minimize the chance of the flagella breaking of) over night
Inoculate fresh 5 mL LB with necessary antibiotics with the overnight culture
For electron microscopy of flagella it is necessary to reach a high cell density without reaching a growth state where the flagella are discarded
The first time electron microscopy was conducted the new cultures were inoculated with 200 and 500 µL of the overnight culture, but for the cell density was not high enough the one ml of the cultures was spun down
One possibility to reach a high OD is to pour LB-agar (with antibiotics) in the glass test tubes and cover it with LB-broth (with antibiotics) and inoculate it, nutrients can freely diffuse through the agar as well as the broth but the cells are constricted to the liquid phase and thus a higher density can be reached
The second time electron microscopy was conducted the method described above was applied
Let grow until a suitable OD600 is reached and check the motility of the cells with the light microscope
Prepare the samples
Place a drop of the culture, a drop of ddH2O and a drop of 2 % PTA on a slice of parafilm
Pick a TEM grid (the microscopic slide in TEM) with forceps and place it on the drop of the culture for a suitable amout of time
Pick the grid up and let the cell suspension dry for a suitable amount of time
Place the grid on the 2 % PTA for a suitable amount of time
Let the grid dry for a suitable amount of time
Wash the grid through placing it on the ddH2O drop for a suitable amount of time and let it dry afterwards
Place the prepared grid in the electron microscope
Library Selection
The library containing vectors with the mutagenized tar-gene was transformed into the E. coli strain BL21. In order to determine certain receptor derivates that enables chemotaxis to a certain molecule a "Library Selection" protocol was determined.  For hints and advice in general for swimming assays view the method "swimming/chemotaxis assay".
Attractants
Coffein: stock solution x mM
2-Ethyl-1-Hexanol
Geraniol
Sodium-Cyclamat
D-aspartic acid
L-aspartic acid 4-benzylester
Vanillin
Media
LB-broth
0.3 % tryptone-swimming agar(1 % tryptone, 0.5 % NaCl, 0.3 % agar)
LB agar
Execution
First round of selection
Thaw one 1 ml cryostock of the library in BL21 and pour into a 200 mL flask filled with LB-media with chloramphenicol
Inoculate the BL21 strain with the parent plasmid in 5 mL LB broth with chloramphenicol
Grow the cultures over night at 37 °C with approx. 180 rpm
Fill 7 + 1 control (whatmanpaper with H2O) 12 cm petridishes with 0.3 % tryptone-swimming agar with chloramphenicol
Apply 100 µL of the attractant to a steril 2x2cm whatmanpaper respectivly and position it in the center of a petridish
Spin down at least 1.5 mL of the culture containing the library and at least 1.5 mL of the culture containing the BL21 strain with the parent plasmid with 1.5 X g for 10 minutes
Discard the supernatant and resuspend the pellet in the remaining medium
Drop 3 times 5 µL of the library and once 5 µl of the reference strain (BL21 with the parent plasmid) respectively, on each plate
Let the drops dry for at least 20 minutes until inverting the plates and placing in the incubator at 33 °C over night
Second round of selection
Determine the drop with the fastes and most directed swimming behaviour on each plate
In order to select the fastest cells the cells containing agar is cut out:
Cut the yellow eppendorf tips of to the first mark (approx 1 cm)
The first cut out is shortly befor the swimming front --> I
The second cut out is on the swimming front --> II
The third cut out is shortly behind the swimming front --> III
Place each cut out either with or without the tip in at least 0.5 ml LB media in an test tube or an E-cup
Incubate the cultures for at least 1 h at 37 °C with approx. 180 rpm
Meanwhile fill 7 + 1 control (whatmanpaper with H2O) 12 cm petridishes with 0.3 % tryptone-swimming agar with chloramphenicol
Apply 100 µL of the attractant to a steril 2x2 cm whatmanpaper respectivly and position it in the center of a petridish
Transfer the culturesinto an E-cup and spin them down with 1.5 X g for 10 minutes
Discard the supernatant and the resuspend the pellet in the remaining medium
Drop 5 µl of the 3 different library cut outs and 5 µl of the reference strain (BL21 with the parent plasmid) respectively on each plate
Let the drops dry for at least 20 minutes before inverting the plates and placing them in the incubator at 33 °C over night
Third round of selection
See second round of selection
Plating of the selected clones
The plates of the third round of selection are treated as described before, but the cultures are not spun down
100 µl of a 10-2 dilution is plated on LB-plates containing chloramphenicol respectively
Incubate the plates in an incubator over night at 33 °C
Minipreparation and sequencing of plasmid DNA
A suitable amount of clones are selected from each plate and used to inoculate 5 mL LB media with chloramphenicol respectively
Incubate the cultures over night at 37 °C with approx. 180 rpm
Isolate the plasmid DNA according to the instructions of the the peqlab kit
Sequence the plasmid DNAas described
Retransformation of the plasmid DNA
In order to determine wether the observed chemotaxis is dependent on the cells themselfes or on the inserted vector the isolated plasmid DNA is transformed into fresh BL21 cells according to the described protocol
Determination of the swimming behaviour of the freshly transformed BL21 cells
Colonies of the freshly transformed BL21 cells (Retrafo) as well as of the selected BL21 clones (Trafo) are used to inoculate 5 mL LB-media with chloramphenicol, respectively and grown over night at 37 °C with approx. 180 rpm
Pour 7 x 2 x 3 0.3 % tryptone-swimming agar plates
Each attractant has 2 additional controls: one time the whatmanpaper is soaked with H2Odest. and the other time with aspartate
The whole approach is conducted for the "Trafos" as well as for the "Retrafos"
Treat and drop the cultures as described
Let the drops dry for at least 20 minutes before inverting the plates and placing them in the incubator at 33 °C over night
Overlap PCR for removal of restriction sites
The DH10B fliC gene contained three PstI sites and one SpeI site that had to be removed. In order to achieve this in a relatively short time we applied overlap PCR. If you are not familiar with this method you can find a detailed description here .
Separation Assay
One of our goals was to be able to separate two strains that swim with a different speed from each other when they are in a mixed culture. Only when they are in a mixed culture they are incubated at the exactly same conditions and thus the true effect of an attractant can be determined.  In this case we utilized different resistance markers that were inserted into two different strains. This version of the separation assay is the final version determined through many tests with the strains Δtar+pSB1C3 (ampicillin resistance, rfp) and Δtar+pSB1C3_tar_QC_18C (chloramphenicol resistance). Specifications for this strain combination are marked in navy, variations in purple. For hints and advice in general for swimming assays view the method "swimming/chemotaxis assay".
Strains
Two strains containing plasmids with different resistance markers for example the biobrick vectors pSB1C3 (chloramphenicol resistance) and J61002 (ampicillin resistance) with different inserts.
Media
LB broth
LB agar with two different resistance markers respectively
0.3 % tryptone swimming agar (1 % tryptone, 0.5 % NaCl, 0.3 % agar)
Execution
Applying the mixed culture to the 0.3 % tryptone swimming agar plates
Inoculate the two strains of interest in 5 mL LB broth with the neccessary antibiotic respectively
Incubate cultures over night at 37 °C with approximately 180 rpm
Fill a suitable number of 12 cm petridishes with 0.3 % tryptone-swimming agar without antibiotic and let the agar solidify
Alternatively M9 agar can be used
Apply 100 µL of the to be tested attractant to a steril 2x2 cm whatmanpaper respectivly and position it in the center of the petridish
When the strains Δtar+pSB1C3 and Δtar+pSB1C3_tar_QC_18C were tested aspartate was used as a attractant
Measure the OD600 of the over night cultures
calculate the neccessary amout you have to take from each culture to gain a cell ratio of 1:1
Mix the cultures
Spin down the mixed culture and 1 ml of the not mixed ultures repectively with 1.5 X g for 10 minutes
No change in the results were observed when this step was not conducted
Remove the supernatant completly and add 100 µL of fresh LB broth
Discard the supernatant and resuspend the pellet in the remaining medium, this can only be applied, when the cultures are dropped on the plated immediatly
Drop two times 5 µL of the mixed culture and once 5 µL of each of the not mixed strains (references)
Let the drops dry for at least 20 minutes until inverting the plates and placing them in the incubator at 33 °C over night
Separation of the different strains
Determine the drop of the mixed culture with the fastes and most directed swimming behaviour on each plate
In order to determine the faster strain and to separate them the agar is cut out at three different positions:
Cut the yellow eppendorf tips of to the first mark (approx 1 cm)
The first cut out is at the swimming front --> I
The second cut out is between the swimming front and the center od the original drop -->II
The third cut out is in the center of the original drop --> III
Place each cut out either with or without the tip in at least 0.5 mL LB media in an test tube or an E-cup
Incubate the cultures for at least 1 h at 37 °C with approx. 180 rpm
Meanwhile prepare the same amout of LB agar plates (9 cm) with the different selection makers
When the strains Δtar+pSB1C3 and Δtar+pSB1C3_tar_QC_18C were tested LB agar plates containing either chloramphenicol or ampicillin were poured
Prepare a dilution series from 10-1 to 10-5 of each of the three cultures
Plate out 100 µL of the dilutions 10-3, 10-4 and 10-5 on LB agar plates with the two different selection markers, respectively
Alternatively: Plate out the 10-2 and the 10-4 dilution. Dependent on the incubation time a bacterial lawn will be observed on the 10-2 plates
Incubate the plates in an incubator over night at 33 °C
Count out the colonies
Sequencing
Sequencing is used to identify the nucleotide arrangement of a given DNA or RNA template. For example we used sequencing to check if our new designed plasmid-constructs include the correct insert. But before we could sequence our samples or better before we could give our samples to the sequence laboratory, we had to prepare the samples. First of all, the plasmids have to be isolated from cells. Therefore we used the PeqGOLD MiniPrep Kit I from Peqlab. After isolation of the plasmids the concentration has to be determined via Nanodrop. For sequencing we needed a concentration of 250-300 ng in a maximal volume of 4 µl purified water. Finally, 1 µL of the sequence primer (concentration of 5 pmol) was added. In most cases we used the standard Biobrick primer VF2 and VR.
Standard PCR
The polymerase chain reaction (PCR) is a method for in vitro-amplification of DNA sequences. For the amplification of a DNA fragment the heat resistent enzyme DNA polymerase is responsible. There are several types of DNA polymerases purchaseable, e.g. some of which are very fast or are not error-prone due to proof-reading activity. In order to choose the appropriate DNA polymerase, this link might be of interest: http://barricklab.org/twiki/bin/view/Lab/ProtocolsTaq; 06/30/2012.
To allow binding of the DNA polymerase primer are required. Thus, only the flanking sites for the sequence of interest needs to be known to synthesize specific primer. Each primer has a specific annealing temperature according to it GC-content. Primers used here were ordered by Sigma®.
A single PCR cycle encompasses three steps: denaturation, annealing and elongation. In the first step the DNA double strands are separated at about 95 °C. Next hybridization of the primers at their annealing temperature happens and finally the DNA seqthesis takes place at the optimal working temperature of the chosen DNA polymerase. This amplification cycle is normally conducted between 25-35 cycles depending on the amount of PCR product one wants to synthesize. In one PCR reaction following components are mixed: DNA template, forward and reverse primer, DNA polymerase with appropriate buffer and deoxynucleotides. Note that an increasing cycle number is prone to incorporate errors in the amplified DNA fragment. This is due to the fact that the dNTPs will be depleted and the saturation phase is reached.
-> 50 μl / reaction:
5 μL 10x Buffer
5 μL dNTPs (0.2 mM each)
2.5 μL Primer (0.5 µM) (2x forward and reverse)
0.5 μL Template (ca. 20-100 ng)
0.5 μL Pfu-Turbo (1.25 U)
ad 34 μL ddH2O
Program:
1 min 95 °C (initial denaturation)
30 s 95 °C
30 s 58 °C
2 min/kb 72 °C
10 min/kb 72 °C (final elongation)
30 Cycles
Swimming/Chemotaxis Assay
Another one of our aims was to invent a protocol whith enables us to observe and quantify swimming and chemotaxis of different E. coli strains. Here we use different agar combination dependent on the desired effect and soaked 2x2 cm Whatmanpapers as attractants for chemotaxis.
Strains
In general any strain can be used but some like for examle BL21 are more suitablte for chemotaxis assays, for they exibit the ability to swim.
Media
LB broth
0.3 % tryptone swimming agar
alternatively: M9 swimming agar
this agar is more suitable to observe chemotaxis for the nutrient content is lower
keep in mind that some E. coli strains are not able to grow on M9 media without additional supplements e.g. the strain DH10B requires the aminoacid leucin
Execution
Inoculate the strains of interest in 5 mL LB broth with the neccessary antibiotic respectively
Incubate cultures over night at 37 °C with approximately 180 rpm
Fill a suitable number of 12 cm petridishes with 0.3 % tryptone-swimming agar or M9 agar with antibiotics or additional supplements when necessary
in some cases the swimming ability could be enhanced through the addition L-methionine to the M9 agar
pour the plates fresh before use but let them dry with a slightly open lid under the hood until the condensation water is evaporated to reduce the risk of smearing of the drops
Apply 100 µL of the test-attractant to a steril 2x2 cm whatmanpaper and position it in the center of the petridish
Spin down 1 mL of the cultures with 1.5 X g for 10 minutes
No change in the results were observed when this step was not conducted, but for the cultures have a higher viscosity the risk of spillage and smearing is lower
Remove the supernatant completly and add the amount of fresh LB broth neccessary for the number of desired drops
Alternatively: Discard the supernatant and resuspend the pellet in the remaining medium
Drop 5 µL of the cultures on the petridish, at maximum 4 drops per dish
The best results were observed when the drop was placed in a distance of 2-2.5 cm from the whatmanpaper
Let the drops dry for at least 20 minutes until carefully inverting the plates and placing them in the incubator at 33 °C over night
QuikChange Protocol
To remove disturbing restriction sites within the gene for the successful usage of BioBrick system, the QuikChange reaction is used.
-> 20 μL / reaction:
2 μL 10x Buffer
0.4 μL dNTPs (10 mM each)
0.8 μL Primer (10 µM) (2x or as premix)
0.4 μL Template (ca. 20-100 ng)
0.4 μL Pfu-Turbo
ad 20 μL ddH2O
Program: -> 1 min 96 °C
20 s 96 °C
20 s 58 °C
1.5 min/kb 72 °C
15 Cycles
-> 5 min 72 °C -> Store at 4-8 °C
After PCR add 1 μL DpnI directly into PCR tube. Incubate reaction 1-2 h at 37 °C. Transform 5 µL into 50 µL competent cells.
Quantitative Real-Time PCR
Quantitative real-time PCR is a powerful tool for quantitation of nucleic acid. The quantitative rtPCR was performed by using SYBR® green 1 (5’ Prime). All real-time PCR systems detect a fluorescent dye and correlate this fluorescence signal to the amount of PCR product in the reaction. There are several methods that quantify the mRNA amount via fluorescence including the method SYBR® green 1 that is used in our experiments. SYBR® green 1 is a nonsequence-specific DNA binding dye that intercalates into dsDNA (it does not bind to single stranded DNA). It emits a strong fluorescent signal upon binding to dsDNA.
Experimental design
5 mL of LB-media with 50 µg/mL chloramphenicol were inoculated with E. coli strain BL21DE3 haboring the vector pSB1C3 with following combinations of promoters and Tar receptor constructs:
Constitutive promoter constructs expressed in E. coli:
Tar receptor under the control of constitutive promoter J23100
Tar receptor under the control of constitutive promoter J23104
Tar receptor under the control of constitutive promoter J23105
Tar receptor under the control of constitutive promoter J23106
Tar receptor under the control of constitutive promoter J23109
Tar receptor under the control of constitutive promoter J23112
Tar receptor under the control of constitutive promoter J23113
Tar receptor under the control of constitutive promoter J23114
(For more details click Parts Submitted )
Cultures were incubated overnight at 37 °C, shaking.
For quantitative real-time PCR, RNA was isolated from E. coli BL21DE3 by using the RNA isolation Kit “Nucleus Spin II (Machery and Nagel)”.  Cultures are inoculated over night at 37 °C to 109 cells.  The isolated RNA is then converted to cDNA using the “Reverse Transcription Kit from Qiagen”
Each primer (forward or reverse) concentration in the mixture was 5 pmol/μl
The housekeeping gene rrsD (ribosomal RNA, 16S) was used as reference
The experiment was set up by following the PCR reaction mix illustrated in the table
All experimental set ups were prepared in triplets for calculating the average
In addition: to avoid contaminations, a non-template control for our PCR assays was always carried with the samples.
The quantitative real-time PCR was performed by using ”LightCycler Software Version 4.05 (Roche)”
Primer for qrtPCR:
Primers for amplification of reference cDNA rrsD
rrsD_rev_transcript_fw:
cgtcagctcgtgttgtgaaatg
rrsD_rev_transcript_rev:
cgtgtgtagccctggtcgtaag
Primers for amplification of promoter constructs cDNAs: Primer Set 1
query_L1_fw:
tgacgtcaacctgggattta
query_R1_rev:
ggaggaatccatcatcatcc
20 μl / reaction:
9 µL 2.5x real master mix/ 20x SYBR
2 µL sense primer (5 pmol)
2 µL antisense prime (5 pmol)
1 µL cDNA (diluted 1:10 in H2O)
6 µL water
Program:
3 min 95 °C (initial denaturation)
30 s 95 °C (denaturation)
30 s 61 °C (annealing for our experiment)
2 min/kb 72 °C (elongation)
42 Cycles
Analysis via the 2–ΔΔCT (Livak) Method
First, we normalized the CT of the target gene to that of the reference (ref) gene, for both the test sample and the calibrator sample (calibrator sample was the weakest expressed promotor construct in our case):
ΔCT(test) = CT(target, test) – CT(ref, test)
ΔCT(calibrator) = CT(target, calibrator) – CT(ref, calibrator)
Second, we normalized the ΔCT of the test sample to the ΔCT of the calibrator:
ΔΔCT = ΔCT(test) – ΔCT(calibrator)
Finally, we calculated the expression ratio:
2–ΔΔCT = Normalized expression ratio
SDS-PAGE
Discontinuous SDS-PAGE (sodiumdodecylsulfate-polyacrylamide-gelelectrophoreses) serves to separate proteins in an electric field corresponding to their molecular weight. Two compounds of Laemmli-Buffer β-Mercaptoehanol and SDS are necessary for the separation: β-Mercaptoethanol is a reducing agent of disulfide bridges of proteins. SDS, a strong anionic detergent, binds to hydrophobic amino acid residues, which lead the proteins become negatively charged relative to their mass (~1.4 g SDS/g protein in a 1% SDS-solution). SDS also denaturizes proteins. The SDS-compensated proteins move from a cathode to an anode through a porous polyacrylamide gel. Depended on the size of the porous, which is due to the concentration of polyacrylamide, the proteins are separated.  Here a 5% stacking- and a 12% separation-gel was prepared for separating proteins in a range of 30 – 200 kDa. Protein samples were mixed 1:1 (v/v) with 2x Leammli-Buffer (20 mM Tris/HCl pH 6.5, 4% SDS, 10% β-Mercaptoehanol, 40% Glycerol, 0.002% Brompheolblue) and loaded on the gel. As size standard an unstained protein molecular weight marker (#26610, Thermo Scientific) was used. The Gel was developed at 35 mA for approximately one hour in 1x SDS-running buffer (25 mM Tris (10x), 192 mM Glycerin, 0.1% SDS) and afterwards stained with a 1% Coomassie Brilliant Blue solution (0.002% Coomassie Brilliant BlueR-250, 10% Ethanol, 5% Acetic acid) that fixes the proteins.
↑ Back to top!
Retrieved from " http://2012.igem.org/Team:Goettingen/Project/Methods "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
