<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Goettingen_Notebook_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Goettingen/Notebook/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Goettingen/Notebook/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/Notebook/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/Notebook/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Goettingen/Notebook/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Goettingen/Notebook/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:Goettingen/Notebook/Summary&amp;redirect=no" title="Team:Goettingen/Notebook/Summary">Team:Goettingen/Notebook/Summary</A>)</DIV><P><TABLE cellspacing="0" cellpadding="0"><TBODY><TR><TD colspan="2"><DIV class="MenuBar" id="navi"><UL><LI><A href="https://2012.igem.org/Team:Goettingen" style="color: white;">Home</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen"><SPAN><SPAN>Homepage</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/News"><SPAN><SPAN>News</SPAN></SPAN></A></LI></UL><LI><A href="https://2012.igem.org/Team:Goettingen/Team" style="color: white;">Team</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Team"><SPAN><SPAN>Members</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Focus_Groups"><SPAN><SPAN>Focus Groups</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Seminars"><SPAN><SPAN>Seminars</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Work_Impressions"><SPAN><SPAN>Work Impressions</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project" style="color: white;">Project</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project"><SPAN><SPAN>Our Project</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Notebook/Results"><SPAN><SPAN>Results</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Notebook/Discussion"><SPAN><SPAN>Discussion/Outlook</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Lab_Work_Flow_eng"><SPAN><SPAN>Lab Work Flow</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Notebook" style="color: white;">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Notebook"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Materials"><SPAN><SPAN>Materials</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Methods"><SPAN><SPAN>Methods</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Computational_Data"><SPAN><SPAN>Computational Data</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Project/Bioinformatical_Tool"><SPAN><SPAN>Bioinformatical Tools</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM" style="color: white;">iGEM</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM"><SPAN><SPAN>What is iGEM?</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM/Parts"><SPAN><SPAN>Parts Submitted</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM/Attributions"><SPAN><SPAN>Attributions</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/iGEM/Saftey"><SPAN><SPAN>Safety</SPAN></SPAN></A></LI>&gt;
                                <LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen" style="color: white;">Göttingen</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen"><SPAN><SPAN>City</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen/University"><SPAN><SPAN>University of Göttingen</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen/MPI"><SPAN><SPAN>Max-Planck-Institute</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Goettingen/S1-Demo_Lab"><SPAN><SPAN>S1-Demo Lab</SPAN></SPAN></A></LI><LI style="width: 160px"><A href="https://2012.igem.org/Team:Goettingen/Human_Practice" style="color: white;">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice"><SPAN><SPAN>Our Human Practice Projects</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Public_and_Media"><SPAN><SPAN>Public and Media</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Survey"><SPAN><SPAN>Survey</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Synthetic_Biology_Day"><SPAN><SPAN>Synthetic Biology Day</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Panel_Discussion"><SPAN><SPAN>Panel Discussion</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Human_Practice/Flash_coli"><SPAN><SPAN>Flash Coli</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Sponsoring" style="color: white;">Sponsoring</A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Sponsoring#Sponsors"><SPAN><SPAN>Our Sponsors</SPAN></SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Goettingen/Sponsoring#Interested_in_Sponsoring.3F"><SPAN><SPAN>Interested in...?</SPAN></SPAN></A></LI></DIV></TD></TR><TR><TD width="865px" style="padding-top: 10px; font-size:13px; font-family:Verdana" align="right"><A href="https://2012.igem.org/Team:Goettingen/Notebook/Results">Deutsch </A> / <A href="https://2012.igem.org/Team:Goettingen/Notebook/Results">English </A></TD></TR></TBODY></TABLE></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Homing_coli:_Engineering_E._coli_to_become_tracking_dogs"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Homing coli: Engineering E. coli to become tracking dogs</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#.231_-_Selection_.2F_Swimming"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">#1 - Selection / Swimming</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Fine_tuning_of_the_experimental_setup"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Fine tuning of the experimental setup</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Library_selection_assay"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Library selection assay</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Separation_Assay_and_RFP-interference"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Separation Assay and RFP-interference</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#References"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#.232_-_Speed_Improvement"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">#2 - Speed Improvement</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#motA_and_motB"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">motA and motB</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#yhjH"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">yhjH</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#fliC"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">fliC</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#flhDC"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">flhDC</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Electron_microscopy"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Electron microscopy</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#References_2"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#.233_-_Chemoreceptor_Library"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">#3 - Chemoreceptor Library</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Prologue"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Prologue</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Chapter_I:_Preparing_Tar_for_the_Library"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Chapter I: Preparing Tar for the Library</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Chapter_II:_Designing_the_Library"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Chapter II: Designing the Library</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Chapter_III:_Reading_the_Library"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Chapter III: Reading the Library</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Epilogue"><SPAN class="tocnumber">4.5</SPAN><SPAN class="toctext">Epilogue</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#References_3"><SPAN class="tocnumber">4.6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Homing_coli:_Engineering_E._coli_to_become_tracking_dogs"> Homing coli: Engineering <I>E. coli</I> to become tracking dogs </SPAN></H2><P>The model organism <I>Escherichia coli</I> is naturally capable of sensing substances in its environment and consequently moves directionally
towards these, a phenomenon known as chemotaxis. Here, we apply directed evolution to chemoreceptors by targeting five amino acid
residues in the ligand binding site to enable <I>E. coli</I> to perceive novel substances. In order to investigate mobility and directed
movement towards a substance, an effective mobility selection method using special &quot;swimming plates&quot; is designed. Additionally, we
attempt to improve <I>E. coli</I>'s swimming velocity by creating new parts derived from its own motility apparatus. Based on our selection
system, we identify variants of chemoreceptors with new binding specificities in the mutant library. By these means, we aim to train
the bacterium to detect new molecules such as tumor cell markers. Once having established <I>E. coli</I> as our &quot;tracking dog&quot;, the possible
applications in medicine but also to environmental issues are virtually countless.
</P><H2><SPAN class="mw-headline" id=".231_-_Selection_.2F_Swimming"> #1 - Selection / Swimming </SPAN></H2><H3><SPAN class="mw-headline" id="Fine_tuning_of_the_experimental_setup"> Fine tuning of the experimental setup </SPAN></H3><P>During the course of the experimental work it has been the task of group one to design and test the parameters for the optimal swimming and chemotaxis assays. Eventually, it was the goal to be able to select for fast <I>E.coli</I> cells that show a chemotactic behavior. Firstly, swimming is only possible in media with a low concentration of agar. After testing agar concentrations in a range between 0.2% - 1%, it turned out that 0.3% is the best concentration for fast <I>E. coli</I> cells. Lower concentrations were not appropriate due to a lack of solidity of the swimming-agar. Secondly, different nutrient compositions were tested. On the one hand, it was necessary to identify a convenient nutrient composition to trigger bacterial swimming. On the other hand, the nutrients in the media should enable <I>E. coli</I> cells to follow a chemoattractant. 
</P><TABLE align="right"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Fig. 1:</B><B>Chemotaxis and swimming of different <I>E. coli</I>-strains on tryptone- and M9 agar.</B> Chemotaxis strain MG1655 (left), laboratory strain BL21 (top), laboratory strain DH10B (right) and laboratory strain XL1 blue (bottom) were tested for swimming and chemotaxis properties on two different media. <B>(A)</B> MG1655 and BL21 both showed extensive swimming on 0.3% tryptone swimming agar after incubation at 33°C for one day. In contrast, the laboratory strains DH10B and XL1 blue did not swim at all.  Swimming towards the attractant L-aspartate in the central Whatman paper was not observed. <B>(B)</B> On 0.3% M9 minimal agar the <I>E. coli</I> strains MG1655 and BL21 displayed swimming, but had to be incubated at 33°C for 2 days. Strikingly, chemotaxis was clearly observed for both strains, MG1655 and BL21, towards the attractant L-aspartate.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Here, we focused on two different media: Tryptone-swimming agar for the induction of <I>E. coli</I> swimming after one day of incubation and M9-minimal agar for the observation of chemotaxis (Burkart et al. 1998; Farrel et al. 2005).  M9-minimal agar did not support intense swimming. Instead, swimming was initially observed after 2 days. In order to allow chemotaxis on M9-minimal agar, the addition of L-methionine was required, as described by Adler, 1972. In comparison to the tryptone-swimming agar, chemotaxis experiments were much more reasonable with the defined M9-minimal agar leading to a high degree of reproduction. Thirdly, different stocksolutions of attractants in the Whatman paper were tested, including L-aspartate (10 mM), a mix of amino acids (0,2% w/v) and tryptone (0,5% w/v). These components were used to trigger chemotaxis of the bacteria. Here, we used L-aspartate in most experiments when we worked with the aspartate receptor Tar of <I>E. coli</I>. We adjusted the volume of all attractants to 100 µl, which was the maximal amount that could be soaked into the Whatman paper (2 cm X 2 cm in size). Furthermore, it proved beneficial to pipette the 100 µl attractant solution on the Whatman paper before the paper was placed onto the center of the semisolid agar. Moreover, the incubation temperature turned out to be a crucial parameter for swimming assays. It is known that long-term bacterial growth at 37°C leads to inhibition of flagella growth. Therefore, incubation of swimming assay Petri dishes was conducted at 33°C.
The fine-tuning of the experimental setup was closely related to the choice of our <I>E. coli</I> strains. Since we worked a lot with the tar-knock-out mutant (Baba et al. 2006) that was created in the background of the laboratory strain DH10B, we tested all our media and parameters with DH10B in the first place. But, soon we realized that laboratory strains were not developed to be the best swimmers and we figured that other strains might have better swimming characteristics. Therefore, the chemotaxis strains MG1655 and RP437 joined our collection. In addition, the E. coli laboratory strain BL21 showed better swimming properties than the laboratory strains DH10B or XL1 blue (Fig. 1). 
</P><H3><SPAN class="mw-headline" id="Library_selection_assay"> Library selection assay </SPAN></H3><P>The main aim of our project was to “engineer <I>E. coli</I> to act as tracking dogs”. Seven substances were chosen as new attractants for our “tracking <I>E. colis</I>”: some were taken because of structural similarity to L-aspartate, the target of the tar receptor, others because of medical relevance and a third group was made up by just subjectively interesting chemicals. The first substance was D-aspartate, which is the enantiomer of L-aspartate. L-aspartate-4-benzyl-ester was chosen because of its structural similarity to L-aspartate. Another substance was 2-Ethyl-1-hexanol, which can be detected in the breath of lung cancer patients (Filipiak <I>et al.</I> 2009). The well known stimulant caffeine, the flavoring substance vanillin and  geraniol, the smell of roses, were tested additionally. Finally, sodium cyclamate was chosen, an artificially produced sweetener.
The mutational library of the tar receptor, our receptor of choice, was produced by focus group #3 . Here, focus group 1 designed a selection system for the “tracking <I>E. colis</I>” (Fig. 2A). 
In our system, the BL21 cells containing the mutagenesis library of the Tar receptor were dropped on 0.3 % tryptone agar plates with the attractant soaked into Whatman paper placed in the middle. The clones that moved the fastest towards the attractant and the reference strain were selected in three successive rounds and the plasmid DNA of single clones was sequenced (see results of group 3).  
</P><TABLE align="left"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Fig. 2A: </B><B>Schematic illustration of the library selection assay of the tar mutagenesis library. </B>1. One 1 mL cryostock of the library was thawed and used to inoculate 50 mL LB-broth with chloramphenicol, the culture was incubated over night at 37 °C and approximately 180 rpm. 2. In order to reach a high cell density in a small volume1 mL of the culture was spun down for 10 min at 1.5k X g. The supernatant was discarded and the pellet was resuspended in the amount necessary to place 24 x 5 µl drops LB broth, which is approximately 200 µl. Alternatively, the supernatant was discarded by inverting the E-cup and the pellet was resuspended in the remaining liquid. 3. 7 x 12 cm plus one control Petri dish were filled with 0.3% tryptone swimming agar, a 2 x 2 cm Whatman paper was soaked with an attractant and placed in the center of the agar plate. 5 µl of the resuspended pellet were dropped on the agar three times, the strain containing the parent plasmid was dropped once and marked as “Ref.”. 4. The Petri dishes were incubated over night at 33 °C. 5. The colony with the fastest swimming behavior was identified according to the size of the halo. 6. The yellow Eppendorf pipet tip was cut at its first mark with sterilized scissors. 7. Agar pieces at the marked sites were cut out using a shortened tip, respectively. 8. The pieces of agar were placed in a glass test tube or an e-cup with or without the tip. The cultures were incubated for at least one hour at 37 °C with 180 rpm. 9. The cultures were spun down for 10 min at 1.5k X g. 10. 8 x 12 cm plus one control Petri dish were filled with 0.3% tryptone swimming agar, a 2 x 2 cm Whatman paper was soaked with an attractant and placed in the center of the agar plate. 5 µl of the resuspended (as described above) pellet was dropped on the agar three times, the strain containing the parent plasmid was dropped once and marked as “Ref.” 11. Steps 5 to 10 were repeated another two times. 12. A 10-2 dilution of each culture was plated on LB-agar containing chloramphenicol and incubated over night at 33 °C. The clones were used to inoculate an overnight culture from which a plasmid preparation was conducted. The plasmid DNA was sequenced subsequently.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
In figure 2B, exemplary pictures of a round of selection are shown. Swimming on tryptone agar could be observed every time after one overnight of incubation at 33 °C, but the observed halos at the different attractants varied in size. The halos had mainly a roundish shape and no direct chemotaxis effect towards the attractants could be identified. The reference strain, containing the parent plasmid of the mutagenesis library, also swam but never as fast as the cells containing the library. The observed halos of the BL21 strain with the library and the reference strain had the same size in the negative control where H<SUB>2</SUB>Odd was used instead of an attractant. Therefore, every attractant triggered a chemotactic effect in BL21 cells holding the mutagenesis library. After the third round of selection, the fastest clones were identified and their plasmid DNA was isolated and sequenced.
</P><TABLE align="right"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Fig. 2B:</B><B> Exemplary pictures depicting the third round of selection of the first approach of the library selection assay </B>In this figure, exemplary pictures of the third round of selection of the first approach, thus the first thawed cryostock of the library, are shown. In the next step (see figure 2A), the fastest colony again was cut out three times, incubated and plated on LB-agar plates containing chloramphenicol. The plasmid DNA of 2-4 clones for each attractant was extracted and sequenced. In each of the shown pictures, the clones containing the library were faster than the reference strain except for the negative control containing H<SUP>2</SUP>Odd in the Whatman paper. Here, the same swimming behavior could be observed. The swimming movement of the cells did not seem to be directed, because the halos had a round shape. Thus, no chemotaxis could be observed. In general, the different attractants caused a differently strong swimming behavior; e.g. the attractant caffeine always led to big halos, whereas the halos related to geraniol were smaller. Ref.: Reference strain (BL21 with pSB1C3_J23100_tar_QC).</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE align="right"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption"><B>Fig. 2C:</B><B> Swimming behavior of selected clones versus retransformed clones</B>The plasmid DNA was extracted from the clones that had resulted from the selection library and was transformed into fresh BL21 cells. The fresh BL21 should indicate, whether the before observed swimming behavior was dependent on the mutagenized receptor or the cells themselves. The cultures and 0.3 % tryptone swimming agar plates were prepared as described in the methods section. The halos produced by the selected clones were always bigger than the halos of the retransformed clones. In many cases even the reference strain showed a faster swimming movement than the retransformed BL21 clones. S. clones: Clones, selected through the mutation library. Retrafos: Clones, transformed with the extracted plasmids of the selected clones.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
In order to investigate, whether the observed swimming behavior was related to the vector containing the mutagenized tar or the cells themselves, the isolated and sequenced plasmids were transformed into fresh BL21 cells and applied to a new swimming/chemotaxis assay. Additionally to the respective attractant, the effect of L-aspartate  and H<SUB>2</SUB>Odd was evaluated. In general, the observed halos of the fresh transformed cells were smaller than the ones of the selected clones every time (Figure 2C). Thus, it can be concluded that the cells themselves were also selected for faster swimming abilities. 
L-aspartate  also was an attractant for nearly every clone and the halos were never reduced significantly in size when H<SUB>2</SUB>Odd was applied to the Whatman paper.
</P><H3><SPAN class="mw-headline" id="Separation_Assay_and_RFP-interference">Separation Assay and RFP-interference</SPAN></H3><P>One of the aims of group one was to invent an assay that enables us to separate two strains from each other that have grown and swam together in a mixed culture.  This would be especially beneficial for the determination of the effect of the promoter strength for a tar insert in a vector or to determine whether an attractant affects two different strains in a different way.
In the beginning, it was planned to conduct the separation of the strains via the strong overexpression of RFP of one strain whereas the other strain would not express RFP. The ratio between the red and the white colonies would give us information about the distribution of the different strains in one swimming halo. After the first round of cloning the RFP gene and the tar inserts were on different vectors containing different antibiotic resistances. Therefore, a separation via the resistance marker seemed to be more plausible. 
The separation assay described in the methods section was conducted multiple times and in many different variations, but the achieved outcomes were contradictory (no data shown). 
In some cases when the agar from the middle of the drop was cut out a much higher number of colonies of the non RFP expressing strain was counted, even though the strains were mixed in the same ratio and thus should be present in the same amount. It was suspected that the expression of RFP contributes to the optical density value measured at a wavelength of 600 nm. Consequently a much higher OD600 value is measured than is actually proportional to the number of cells in the culture. Some experiments were conducted to determine the influence of the RFP expression on the OD600 value. Because of the time limit no statistically significant values could be gained and the achieved data is not shown.  Further experiments have to be conducted to determine the RFP interference (e.g. using a counting chamber) and in case of the separation assay one should probably refrain from utilizing RFP expressing strains. 
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><UL><LI>[1] <A href="http://mic.sgmjournals.org/content/74/1/77.full.pdf" class="external text" rel="nofollow">Adler J. 1972. <I>A Method for Measuring Chemotaxis and Use of the Method to Determine Optimum Conditions for Chemotaxis by Escherichia coli.</I> J Gen Microbiol. 74(1):77-91 </A></LI><LI>[2] <A href="http://www.pnas.org/content/95/5/2568.short+html" class="external text" rel="nofollow">Burkart M., Toguchi A., Harshey R.M. 1998. <I>The chemotaxis system, but not chemotaxis, is essential for swarming motility in Escherichia coli</I> 3;95(5):2568-73  </A></LI><LI>[3] <A href="http://www.nature.com/nmeth/journal/v2/n5/full/nmeth0505-377.html" class="external text" rel="nofollow">Farrell I.S., Toroney R., Hazen J.L., Mehl R.A., Chin J.W. 2005. <I> Photo-cross-linking interacting proteins with a genetically encoded benzophenone</I> 2(5):377-84</A></LI><LI>[4] <A href="http://ar.iiarjournals.org/content/29/1/419.full.pdf+html" class="external text" rel="nofollow">Filipiak W., Sporing A., Filipiak W., Mikoviny T., Ager C., Schubert J., Miekisch W., Amann A., and Troppmair J.. 2009. <I>Release of Volatile Organic Compounds from the Lung Cancer Cell Line NCI-H2087 In Vitro.</I> Anticancer Research 29: 419-426.</A></LI></UL><H2><SPAN class="mw-headline" id=".232_-_Speed_Improvement"> #2 - Speed Improvement </SPAN></H2><P>The goal of our group was to enhance the motility of <I>E. coli</I> in order to allow them to reach their targets more efficiently in later applications. This also results in a a faster selection of our Homing Coli and quick generation of results. The BioBricks we designed are therefore primarily directed at flagellum efficiency or number. We selected five different genes that were tested along our project in different strains. <I>E. coli</I> strain BL21 was used as our main object of research because this is a typical lab-strain and therefore usually not very motile. Our goal was to find a way to accelerate BL21 <I>E. coli</I> by transforming them with our constructs. Here is an overview of the genes we tested and the corresponding results.
</P><H3><SPAN class="mw-headline" id="motA_and_motB"><I>motA</I> and <I>motB</I></SPAN></H3><TABLE align="right"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Fig. 3:</B> Illustration of the role of the stator proteins MotA and MotB for the flagellar apparatus. A proton gradient at the inner cell membrane causes protons to flow through the stator-”channel”. This flux of protons causes the flagellum to rotate and allows the cells to swim. Each flagellum can harbor around 8-11 stator components. <B>References:</B> Chevance and Hughes (2008) <I>Coordinating assembly of a bacterial macromolecular machine.</I> Nat Rev Microbiol. 6: 455–465.</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Fig. 4:</B> BL21 <I>E. coli</I> carrying different constructs on LB swimming agar after 12h incubation at 33°C. Cells expressing <I>motB</I> in puc18 under the natural promoter travelled approximately 0.85cm (radius) and those expressing <I>yhjH</I> swam around 0.68cm. The empty vector control and motA transformed cells travelled only about 0.5cm.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>These genes code for proteins that build the stator part of the bacterial flagellum which generates torque by using a proton gradient that exists across the membrane (Fig. 3). Flagellum function depends on copy number of <I>motA</I> and <I>motB</I> (Van Way et al. 2000). Therefore we chose to test these genes, hoping that the number of stator protein copies might have a positive effect on motility (Reid et al. 2006). We first tested our strains with <I>motA</I> and <I>motB</I> constructs containing the natural 3’ and 5’ regions in puc18 because a strong overexpression might have counterproductive effects. High expressed motA might thereby lead to proton leakage and disturb the metabolism of <I>E. coli</I>. 
</P><UL><LI>Results 
</LI><LI>For these two stator-genes it was very hard to get reproducible data. However, after several rounds of testing on different swimming agars and conditions we were able to conclude that overexpression of <I>motB</I> often had a mild positive effect on motility of BL21 <I>E. coli</I> (Fig. 4). However, increased expression on <I>motA</I> seemed to have no or even a slightly negative effect. 
</LI><LI>We did not see very strong effects after testing these two candidates. A possible explanation could be that the number of functional stator elements at one flagellum could not be increased anymore or might just not have a significant impact. Furthermore the expression of <I>motA</I> and <I>motB</I> is usually regulated in concert and if out of balance, their expression might not cause any positive effect for motility. For future projects it might be interesting to test chimeric stator proteins that could harbor higher efficiencies or use other ions for torque generation.
</LI></UL><H3><SPAN class="mw-headline" id="yhjH"><I>yhjH</I></SPAN></H3><P>This gene codes for a phosphodiesterase which reduces the levels of c-di-GMP that are involved in cell motility (Ko and Park, 2000). The protein YcgR is able to bind c-di-GMP and act as a flagellar brake which consequently reduces motility (Paul et al. 2010). The mechanism that we would like to exploit is illustrated in Figure 5. Overexpression of <I>yhjH</I> should therefore result in lower levels of c-di-GMP and increase <I>E. coli'</I>s speed by diminishing the ability of YcgR to act as a brake on the flagellum. The second messenger c-di-GMP furthermore plays a role in the regulation of biofilm development. Expression of <I>yhjH</I> on the other hand represses biofilm formation (Suzuki et al. 2006) which could also serve our purposes and induce swimming motility.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Fig. 5:</B> The <I>yhjH</I> gene encodes for a c-di-GMP phosphodiesterase. The substrate, c-di-GMP, is a second messenger that binds to YcgR, a protein that functions as a flagellar brake and thus down-regulates the motor (A). We suggest that the overexpression of <I>yhjH</I> results in increased c-di-GMP degradation and hence in narrowed braking force. Due to a stronger rotation of the flagellum higher motility could be achieved (B).<B>References:</B><A href="http://www.uniprot.org/uniprot/P37646" class="external free" rel="nofollow">http://www.uniprot.org/uniprot/P37646</A>; <A href="http://www.uniprot.org/uniprot/P76010" class="external free" rel="nofollow">http://www.uniprot.org/uniprot/P76010</A></DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Fig. 6:</B> BL21 <I>E. coli</I> carrying different constructs on <A href="https://2012.igem.org/Team:Goettingen/Project/Materials#Tryptone_swimming_agar" class="external text" rel="nofollow">Tryptone swimming agar</A> after 12h incubation at 33°C. Cells expressing yhjH in puc18 under the natural promoter travelled approximately 0.5cm (radius) whereas no swimming could be detected for the control plasmid carrying <A href="http://partsregistry.org/Part:BBa_K777125" class="external text" rel="nofollow">K777125</A>.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI>Results:
</LI><LI>Expression of <I>yhjH</I> under its natural promoter in puc18 increased motility in almost all of our assays on LB or Tryptone swimming agar (Fig. 4 &amp; Fig. 6). Not only were the <I>yhjH</I> expressing strains among the fastest on these swimming agar plates but also usually the first colonies to start swimming. This might be an indicator that indeed biofilm formation was inhibited and the <I>E. coli</I> were able to spread faster. Compared to our other tested constructs we got the most reliable results for <I>yhjH</I>. However the positive effect on motility was not present when we used M9 minimal medium swimming agar. Here, <I>yhjH</I>-transformed cells usually did not show a strong tendency for swimming. This result suggests that the motility-effects of YhjH are somehow nutrient-dependent and therefore depend on the used media.
</LI></UL><H3><SPAN class="mw-headline" id="fliC"><I>fliC</I></SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Fig. 7:</B> The fliC gene encodes for flagellin, which represents the single subunits of the flagellar filament. The used <I>E. coli</I> strains (DH10B, BL21, XL1 Blue and DH5α) are typical laboratory strains with degenerated flagella and thus reduced motility (A).  We assume that the overexpression of <I>fliC</I> might lead to the incorporation of further flagellin subunits and hence to longer flagella compared to earlier (B).  The electron microscope image (C) features the flagella of the strain MG1655, which still possesses unimpaired, long flagella. Our aim is to restore the phenotype of  the laboratory strains flagella and to receive a length comparable to that of MG1655. <B>References:</B><A href="http://ecocyc.org/ECOLI/NEW-IMAGE?type=POLYPEPTIDE&amp;object=EG10321-MONOMER" class="external free" rel="nofollow">http://ecocyc.org/ECOLI/NEW-IMAGE?type=POLYPEPTIDE&amp;object=EG10321-MONOMER</A></DIV></DIV></DIV><P>Flagellin, the product of the <I>fliC</I> gene is the structural protein that builds up the filament of the bacterial flagellum (Fig. 7). By overexpressing <I>fliC</I> we hope to construct longer flagella which might have a positive impact on swimming speed (Furuo et al. 1997). At the same time we were aware that overlong flagella might cause adverse effects due to an obstructive architecture.
</P><UL><LI>The <I>fliC</I> gene was amplified from DH10B <I>E. coli</I> genomic DNA and contained four forbidden restriction sites. We had to mutate all these sites via overlap PCR. A description for this method can be found <A href="https://static.igem.org/mediawiki/2012/3/34/Overlap_PCR.pdf" class="external text" rel="nofollow">here</A>. 
</LI></UL><UL><LI>Results
</LI><LI>Our overlap PCR for the mutation of three PstI and one SpeI sites worked efficiently and fast. The resulting PCR product was not cut by any BioBrick standard 10 enzymes.
</LI><LI>In order to avoid too strong levels of expression we included 1kb of the upstream region for most of the assays to allow a certain level of regulation in the cells. Like <I>yhjH</I>-transformants these strains showed increased motility in almost all of our assays (Fig. 6). On the minimal medium M9 agar plates they were usually significantly faster and showed swimming motility earlier than any other cells. On Tryptone swimming agar they were in general the runner up behind <I>yhjH</I>-transformed <I>E. coli</I>. Unfortunately, we had trouble cloning our <I>fliC</I> constructs into pSB1C3 because of a problem in our reverse primer suffix. When we had located the source of the problem it was too late to send this part to the Registry in time.
</LI></UL><H3><SPAN class="mw-headline" id="flhDC"><I>flhDC</I></SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Fig. 8:</B> The master operon <I>flhDC</I>, involved in motility, codes for the transcription factor FlhD<SUB>4</SUB>C<SUB>2</SUB> which acts as a regulator of flagellum synthesis. It is suggested that an overexpression of this operon in <I>E. coli</I> results in an increased motility [2]. (A) The flagellar apparatus is  anchored within the bacterial cell membrane. (B) <I>E. coli</I> cells that overexpress the <I>flhDC</I> operon supposedly exhibit an overall increased flagella production. Consequently, the cells should be able to move faster than their untransformed fellows. <B>References:</B> [1] Wei et al. (2001) <I>Positive regulation of motility and flhDC expression by the RNA-binding protein CsrA of Escherichia coli.</I> Molecular Microbiology, 40(1):245-256  [2] Gauger et al. (2007) <I>Role of motility and the flhDC operon in Escherichia coli MG1655 colonization of the mouse intestine.</I> Infect immune 75(7):3315-3324</DIV></DIV></DIV><P>The <I>flhDC</I> operon is the master regulator of motility and chemotaxis in <I>E. coli</I>. This means that it is the main control instance for flagellar synthesis and starts the complex process of flagellar gene synthesis and flagellum assembly (Chevance and Hughes 2008). The <I>flhDC</I> operon codes for the transcriptional regulator FlhD<SUB>4</SUB>C<SUB>2</SUB> which forms heterotetramers and activates class II operons in concert with sigma factor 70. Among the gene products of class II operons are several components of the flagellum and the alternative sigma factor FliA which is essential for the transcription of class III genes. It has been shown that increased expression of flhDC also enhances motility in <I>E. coli</I> (Ling et al. 2010). We wanted to see how different levels of <I>flhDC</I> expression affect the speed of our <I>E. coli</I> lab-strains. In theory, the number of assembled flagellums should increase (Fig. 8) and thereby have a positive impact on motility.
</P><UL><LI>Results
</LI><LI>Our plan was to express <I>flhDC</I> constitutively with a selection of 8 different Anderson-promoters to achieve different levels of transcription. And indeed BL21 <I>E. coli</I> transformed with these constructs showed a higher motility than empty vector controls or WT cells. However, our results were very inconsistent, especially concerning the different promoter strengths when compared to each other. The troubleshooting revealed that we were missing a RBS between the promoters and the ATG within all our constructs. This probably explains the inconsistent results that did not allow us to quantify the effect of our constructs. We hope to finish these experiments with correct constructs soon because we are eager for presentable results!
</LI></UL><H3><SPAN class="mw-headline" id="Electron_microscopy">Electron microscopy</SPAN></H3><P>Overexpression of flagellar genes like <I>fliC</I> or the <I>flhDC</I> operon should result in alterations of the flagellum structure or number, which can be visualized. In the case of <I>fliC</I>, which encodes flagellin, the building block of the flagellum, we expect the emergence of longer filaments. In contrast to this, <I>flhDC</I> as the master regulator is thought to increase the number of flagella when overexpressed, since all genes involved in flagellum assembly should be up-regulated.
In order to visualize these alterations we investigated the <I>E. coli</I> strain BL21, transformed with <I>fliC</I> or <I>flhDC</I> constructs, by using transmission electron microscopy. Both genes were under control of a strong constitutive promoter promoter (J23100). As reference strains we used BL21 cells carrying the same vector with an RFP insert. Furthermore the MG1655 strain, as a more motile strain with non-degenerated flagella, was used. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Fig. 9:</B> Transmission electron microscopy pictures of WT <I>E. coli</I> strain MG1655 (A) and the BL21 strain transformed with K777101, a <I>flhDC</I> construct (B). The single flagella filaments of MG1655 are attached to the cell and visible very clearly. However, our cell densities in the BL21 cultures were not growing very well. Therefore we were not able to get good pictures and had to centrifuge the cells to obtain higher densities. This caused interweaving of the flagella and single filaments were not visible anymore (B).</DIV></DIV></DIV><UL><LI>Results
</LI><LI>For MG1655 nice pictures were obtained since the strain grew very fast and seems to be quite robust (Fig. 9). However, we were not able to obtain meaningful pictures showing flagella of the BL21 strain. Under stress conditions bacteria tend to lose their flagella very easily (Blum, 1971). Apparently the transport form our Lab to the EM-facility was enough to stress the BL21 <I>E. coli</I>. It appears that this demands quite something for MG1655 however, or the pictures we got from this strain were already in a stressed state and the flagella are usually even longer or more in number. We had a look at the different strains with a light microscope in our lab before the EM sessions and they were all motile. This consideration, that the cells were stressed during the transport was supported by their subsequent investigation under a light microscope. The cells were not motile anymore and featured very heterogeneous shapes whereby some formed very long rods. All these appearances are indications for stressed cells. Another problem with BL21 was the very low growth rate, since a high optical density is required in order to obtain an adequate number of cells during preparation. Therefore we had to centrifuge the cultures in some cases which ended up in interwoven filaments (Fig. 9). </LI><LI>In conclusion, we were not able to get informative data about the flagellar structures and morphologies, but we found out, that BL21 is very sensitive and drops its flagella in no time. 
</LI></UL><H3><SPAN class="mw-headline" id="References_2">References</SPAN></H3><UL><LI> Blum J.J. 1971 <I>Existence of a breaking point in cilia and flagella.</I> J. Theor. Biol. 33:257–263.
</LI><LI>Chevance F. F., Hughes K. T. 2008. <I>Coordinating assembly of a bacterial macromolecular machine.</I> Nat Rev Microbiol. 6: 455–465.
</LI><LI>Furuno, M., T. Atsumi, T. Yamada, S. Kojima, N. Nishioka, I. Kawagishi, and M. Homma. 1997. <I>Characterization of polar-flagellar-length mutants in Vibrio alginolyticus.</I> Microbiology. 66: 3632–3636.
</LI><LI>Ko M., Park C. 2000. <I>Two novel flagellar components and H-NS are involved in the motor function of Escherichia coli.</I> J. Mol. Biol. 303: 371-382.
</LI><LI>Ling H., Kang A., Tan M.H., Qi X., Chang M.W. 2010. <I>The absence of the luxS gene Increases swimming motility and flagella synthesis in Escherichia coli K12.</I> Biochem Biophys Res Commun. 401: 521-526.
</LI><LI>Paul, K., Nieto, V., Carlquist, W.C., Blair, D.F., Harshey, R.M. <I>The c-di-GMP Binding Protein YcgR Controls Flagellar Motor Direction and Speed to Affect Chemotaxis by a “Backstop Brake” Mechanism.</I> 2012. Mol Cell. 38: 128–139.
</LI><LI>Reid, S. W., M. C. Leake, J. H. Chandler, C. J. Lo, J. P. Armitage, and R. M. Berry. 2006. <I>The maximum number of torque-generating units in the flagellar motor of Escherichia coli is at least 11.</I> Proc. Natl. Acad. Sci. USA 103: 8066-8071.
</LI><LI>Suzuki, K., Babitzke, P., Kushner, S. R., Romeo, T. 2006. <I>Identification of a novel regulatory protein (CsrD) that targets the global regulatory RNAs CsrB and CsrC for degradation by RNase E.</I> Genes Dev. 20: 2605–2617.
</LI><LI>Van Way, S. M., E. R. Hosking, T. F. Braun, and M. D. Manson. 2000. <I>Mot protein assembly into the bacterial flagellum: a model based on mutational analysis of the motB gene.</I> J. Mol. Biol. 297: 7-24.	
</LI></UL><H2><SPAN class="mw-headline" id=".233_-_Chemoreceptor_Library"> #3 - Chemoreceptor Library </SPAN></H2><H3><SPAN class="mw-headline" id="Prologue"><B>Prologue</B></SPAN></H3><P>The objective of the “Legendary TAR librarians” was the generation of a mutant library of the chemoreceptor Tar (<U>T</U>axis to <U>A</U>spartate and <U>R</U>epellants). Tar is one of five membrane-bound chemoreceptors <I>E. coli</I> uses for chemotaxis [1]. The receptor is well characterized and an X-ray structure is available (see <A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1wat1" class="external text" rel="nofollow">PDB file</A>). Residues of the Tar ligand-binding domain that interact with L-aspartate were identified from the pdb-file. Here, we aimed to identify novel receptors by applying directed evolution to Tar chemoreceptors. To enable <I>E. coli</I> to perceive novel substances five amino acid residues in the ligand binding site were the target. Our method of choice is targeted saturation mutagenesis. In comparison to random mutagenesis, this has the advantage to specifically mutate the selected amino acid residues.
</P><H3><SPAN class="mw-headline" id="Chapter_I:_Preparing_Tar_for_the_Library"><B>Chapter I: Preparing Tar for the Library </B></SPAN></H3><P>The amplification of Tar was carried out directly from the <I>E. coli</I> K12 genome. We introduced the BioBrick standard restriction sites by primer design (see <A href="https://2012.igem.org/Team:Goettingen/Project/Computational_Data#Sequences" class="external text" rel="nofollow">sequencing primers</A>; Tar for + prefix &amp; Tar rev + suffix). The amplified BioBrick part was then inserted into the vector pUC18. To get rid of BioBrick standard restriction sites in <I>tar</I>, a QuikChange reaction was applied. The <I>Xba</I>I site (nt 420-425) was exchanged from TCTAGt to CCTAGt while keeping the codon for the same amino acid. Our next step was to express the quikchanged <I>tar</I> (TAR_QC) under the constitutive <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K777001" class="external text" rel="nofollow">Anderson promoters</A>.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 10: Promoter comparison through a chemotaxis assay. </B> Eight promoters of the „Anderson“ collection were cloned in front of the gene coding for the Tar receptor inside the pSB1C3 backbone. Here, we compared the swimming behavior of these strains in chemotaxis assay with M9-swimming agar plates. The chemotaxis assay was conducted as described in the methods section. L-aspartate served as the attractant in the central Whatman paper. The numbers next to each halo correspond to the promoter activities reported by the Berkley University in 2006. <B>(A)</B> The constructs were transformed in the Δtar strain. The picture was taken after 3 days incubation at 33 °C. The halos of the strongest promoters (J23100, x2547 and J23104, x1831) had a slightly bigger radius than the other promoter constructs. In addition, slight chemotaxis could be observed for these strains. The drop of the construct J23106 was smeared during the application and is therefore not evaluable. <B>(B)</B> The constructs were transformed in the BL21 strain, that naturally expresses the Tar receptor. The picture was taken after 1 day incubation at 33 °C. In general, halo size did not vary strongly. The halos of the strongest promoters (J23100, x2547 and J23104, x1831) had the biggest radius. The halo of  J23106 (x1185) showed the smallest radius, although this promoter has the third stronges activity. Chemotaxis could be observed for every strain.</DIV></DIV></DIV><P>With these various promoter constructs in hand, a swimming assay was performed. This experiment revealed promoter J23100_TAR_QC in pUC18 background giving the best swimming abilities (data not shown). To generate a chemoreceptor library our method of choice requires the absence of the <I>Bsa</I>I restriction site in both, the insert and the vector backbone, of the cloned plasmid. Since pUC18 contains a <I>Bsa</I>I restriction site in the plasmid backbone within the selection marker <I>bla</I> gene, this vector is not appropriate for targeted saturation mutagenesis. Thus, the construct was cloned in the BioBrick vector pSB1C3. 
</P><P>The promoter constructs in the new vector backbone were tested in chemotaxis assays for swimming towards L-aspartate (Fig. 10). Thereby, all eight tested promoter constructs were dropped around in the same distance to an octahedron Whatman paper soaked with L-aspartate. As a control serves the strain BL21 transformed with Tar_QC under the control of the eight promoters. ∆Tar was complemented with Tar_QC. As expected, the strongest promoters (18C = K777001, 18K = K777002) show the strongest movement. It was not possible to evaluate 18O (K777004). All in all, the complementation worked, but not to the expected extend.
</P><P>Further proof came from a SDS-PAGE using where we observed an additional protein band in lysates from cells producing tar from one of three strongest Anderson promoters. The results were consistent with the previous swimming study (data not shown). The construct J23100_TAR_QC gave the best results in these experiments and was therefore used in all further experiments.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 11: Comparison of relative expression rates of the chemoreceptor Tar from constitutive promoters by qrtPCR and relative fluorescence or RFP expressed by the same promoters  (see parts registry, <A href="http://partsregistry.org/Promoters/Catalog/Anderson" class="external text" rel="nofollow">Anderson family</A>).</B> The blue bar indicates the measured expression rates for our constructs (J23100, J23104, J23105, J23106, J23109, J23112, J23113, J23114) and the red ones those for the literature values represented in the parts registry. The measurements are illustrated in a logarithmic application. The standard variation was calculated for our measured values (black error bar; <A href="https://2012.igem.org/Team:Goettingen/Project/Computational_Data#Characterization_of_constitutive_promoters_by_qrtPCR" class="external text" rel="nofollow">raw dataset</A>).</DIV></DIV></DIV><P>Additionally, our goal was to see a correlation between the mRNA expression levels and the motility of our engineered <I>E. coli</I> on swimming plates. Therefore, we used quantitative real-time PCR (qrtPCR) as a powerful tool for quantification of gene expression. We used this method to examine the expression rate of the eight different constitutive promoter constructs from the parts registry that were further confirmed by swimming assays. The reported activities of these promoters are given as the relative fluorescence of these plasmids in strain TG1 (parts registry: <A href="http://partsregistry.org/Part:BBa_J23100" class="external text" rel="nofollow">Part:BBa_J23100</A>). The results are depicted in Figure 11. Here, promoter constructs were cloned into the vector pSB1C3 and expressed in <I>E.coli</I> BL21DE3 grown in LB-media (lysogeny broth). The measurements were performed as a triplet for each construct and a house-keeping gene coding for a 16SrRNA (<I>rrsD</I>) as reference. Additionally, we included H<SUB>2</SUB>O as negative no template control to predict possible contaminations. For the evaluation of our results, the 2<SUP>–ΔΔCT</SUP> (Livak) method was applied. We used the weakest promoter with the lowest expression rate as calibration for the calculations and as reference the housekeeping gene. The data was then compared to the predetermined expression rates from iGEM2006_Berkeley group.
</P><P>Overall, as shown in Figure 11, each tested promoter construct indicated differences in expression rates in comparison to values from the parts registry. In fact, both data-sets were collected by methods which produce data at different points after gene expression. A common trend was detected for the strongest promoters J23100, J23104, J23105 and J23109 together with the weakest promoters J23112 and J23113. Conspicuously, the promoter J23109 revealed for qrtPCRand for relative fluorescence measurements nearly the same expression rates. The expression rates of J23114 and J23106 indicated massive differences in their expression rates and no common trend with the expression values from the parts registry. We detected comparable positioning in the ranking of expression rates (see <A href="https://2012.igem.org/Team:Goettingen/Project/Computational_Data#Raw_Datasets" class="external text" rel="nofollow">characterization of constitutive promoters by qrtPCR</A>) for six out of our eight promoters. The promoters characterized as relatively strong were also, in our case, responsible for higher expression rates and the other way around. In the case of promoter J23114 and J23106, the data-sets exhibited a completely different characterization compared to those in the parts registry.
</P><P>All in all, these results confirm the properties of J23100 as the best swimmer. 
</P><H3><SPAN class="mw-headline" id="Chapter_II:_Designing_the_Library"><B>Chapter II: Designing the Library</B></SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 12: Library design of the quikchanged chemoreceptor Tar. </B><B>(A)</B> Schematic representation of the applied procedure for library generation of the quikchanged Tar_QC under a strong promoter. Saturated PCR mutagenesis was performed in two rounds. In the first step the mutagenesis primers, that randomly mutate three triplets and introduce a <I>Bsa</I>I restriction site (scarless mutagenesis), were designed. In the second step the parent plasmid was amplified with these primers hybridizing to the parent plasmid pSB1C3_18C_TAR_QC. In step three, the parent plasmid was digested with <I>Dpn</I>I and the mutated plasmid with <I>Bsa</I>I to provide sticky ends. <I>Bsa</I>I cuts outside the restriction site. In step four the plasmid was ligated and afterwards transformed with the mutated DNA into electrocompetent DH10B cells. Subsequently, the DNA was isolated for the second round of mutagenesis PCR (step five), where two other triplets were mutagenized repeating the previous steps. After finishing the 2<SUP>nd</SUP> round the plasmid library was isolated from the liquid culture. The 1<SUP>st</SUP> and 2<SUP>nd</SUP> round of the mutation library covered 10<SUP>5</SUP> clones. The two rounds were combined to one library with a final diversity of 2x 10<SUP>5</SUP>. <B>(B)</B> Representative pictures of 0.8% Agarose gels from the 2<SUP>nd</SUP> saturated PCR mutagenesis round of step two after PCR (2), step three after double digest with <I>Dpn</I>I and <I>Bsa</I>I (3) and step four after ligation of the mutant plasmid (4). The size of the plasmid is always 3769 bp. As standard a 1 kb Molecular Weight Marker (M; Fermentas) was used. <B>(C)</B> Overview (right) and detailed (left) PYMOL images of Tar with aspartate in its binding pocket (PDB ID: 1WAT). The mutated amino acid residues involved in ligand binding are labeled. <B>(D)</B> Alignment of the regions involved in ligand binding of the original DNA sequence of Tar_QC (pSB1C3_TAR_QC), the library sequence (TAR_QC Library) and the amino acid sequence (TAR_QC aa-sequence). The upper two lines show the site of mutation, here. K stands for guanine or thymine, N for any possible base. Corresponding to the DNA sequence the amino acid sequence is shown with labeled positions of mutated amino acid residues (*) above. G: guanine, C: cytosine, A: adenine, T: thymine.</DIV></DIV></DIV></DIV><P>The introduction of targeted saturation mutations requires mutant primer design (<A href="https://2012.igem.org/Team:Goettingen/Project/Computational_Data#Primers" class="external text" rel="nofollow">primer mutagenesis</A>). We divided our library generation into two rounds, each mutating different residues (Fig. 12A). Amino acids 149, 150 and 154 were mutated in the first (1<SUP>st</SUP>), amino acids 69 and 73 in the second round (2<SUP>nd</SUP>). Each library generation consists of the following steps: 1. primer design, 2. saturated mutagenesis PCR, 3. <I>Dpn</I>I/<I>Bsa</I>I restriction, 4. ligation, transformation and plasmid isolation, 5. PCR for 2<SUP>nd</SUP> round; the template for 2<SUP>nd</SUP> round being the isolated mutated plasmids of the 1<SUP>st</SUP>. After 2<SUP>nd</SUP> round, the plasmid library is completed. Figure 12B shows gels from the 2<SUP>nd</SUP> round of mutagenesis. The gels of step 2 (PCR), step 3 (digest) and step 4 (ligation) show a band of the expected size at 3769 bp, indicating that the intermediate steps were successful. The primers we used for the mutagenesis PCR randomly mutate five amino acids residues at positions 69, 73, 149, 150 and 154, which are known to bind to L-Aspartate (Fig. 12C). The outcome of the mutations is depicted in Figure 12D.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 13: Characteristics of the generated chemoreceptor Tar_QC library. </B><B>(A)</B> Representation of the diversity reached in the two rounds of Tar_QC library generation. Both, 1<SUP>st</SUP> and 2<SUP>nd</SUP> round, yielded 10<SUP>5</SUP> clones. After combination of the two diversities the merged library comtains 2x 10<SUP>5</SUP> clones. <B>(B)</B> Sequencing chromatogram excerpt of the generated chemoreceptor Tar_QC library in both rounds. Sequencing results of isolated DNA from the liquid cultures the 1<SUP>st</SUP> attempt (top) with mutations for the amino acidsat position 149, 150 and 154 and 2<SUP>nd</SUP> attempt (bottom) with mutations at 69 and 73. Inserted mutations are indicated by the triplets consisting of N and K, whereby K stands for guanine or thymine, N for any possible base. Only in the mutated regions no distinct peaks are present. G: guanine, C: cytosine, A: adenine, T: thymine.</DIV></DIV></DIV><P>In order to examine the property of the generated chemoreceptor J23100_TAR_QClibrary the transformation efficiency was determined. Due to the fact that the first liquid culture of TAR_QC library only revealed 10<SUP>5</SUP> clones (attempt a), a repeat of the 2<SUP>nd</SUP> round of the library (attempt b) was carried out to increase diversity and resulted in the same diversity. Isolated DNA of attempts a and b were merged to gain a final diversity of 2x10<SUP>5</SUP>, covering 6% of the theoretical diversity which would be 3.2<SUP>6</SUP> (Fig. 13A). The comparison of the sequencing chromatograms between the original and the mutated sequence excerpts indicated high quality of the TAR_QC library, because only the expected mutation sites had been changed. Every sequence of the isolated liquid culture DNA, which was not targeted, remained the same (Fig. 13B). 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 14: Sequence alignment reflecting the diversity of generated chemoreceptor Tar_QC library. (A)</B> Sequencing results of randomly picked clones sequenced for 1<SUP>st</SUP> and 2<SUP>nd</SUP> round. <B>(B)</B> Nucleotide (left) and amino acid (right) sequence alignment of the mutated regions for the 1<SUP>st</SUP> round (top) with amino acids mutated at position 149, 150 and 154 and 2<SUP>nd</SUP> round (bottom) at 69 and 73. Sequencesare represented 5´ to 3´ with increasing numbers standing for the nucleotide or translated amino acid position in the indicated mutation regions, respectively. The consensus of 12 or 14 randomly picked clones sequenced for 1<SUP>st</SUP> and 2<SUP>nd</SUP> round is depicted as a pictogram. The height of each letter represents the frequency of each nucleotide at that position. Original sequences: 1<SUP>st</SUP> round tatttcGCTCAGCCAacc / YFAQPT; 2<SUP>nd</SUP> round cgtTCAGCGGTAcgg / RSAVR. G: guanine, C: cytosine, A: adenine, T: thymine.</DIV></DIV></DIV><P>
To exclude the possibility of a bias towards specific amino acids within the library, clones of the transformation efficiency plates were picked randomly for sequencing. The nucleotide and corresponding amino acid sequence are represented in Figure 14A. No clones were picked twice and only two deletions/frameshifts were identified. On the basis of these data the sequencing alignment was done. Figure 14B supports the good quality of the generated library showing the DNA sequence and amino acid diversity. The sequence alignment also reflects the well-targeted mutations as conserved nucleotides are resembled according to their letter size. As shown in Figure 12D, the used mutation primers incorporated NNK at desired positions, whereby N stands for any possible nucleotide and K for guanine or thymine. All introduced nucleotides ‘N’ show at least three diverse nucleotides, whereas all introduced nucleotides ‘K’ showed the expected pattern. The diversity of the nucleotides is reflected in the translated amino acid sequence. 
</P><P>To sum up, our J23100_TAR_QC library is unbiased, well targeted and of good quality.
</P><H3><SPAN class="mw-headline" id="Chapter_III:_Reading_the_Library"><B>Chapter III: Reading the Library</B></SPAN></H3><P><CENTER><TABLE><TBODY><TR><TD><B>Fig. 15: Animation of the combined results of the three focus groups</B></TD></TR></TBODY></TABLE></CENTER></P><P>Based on the knowledge of the well characterized J23100_TAR_QC library, we assumed to have constructed novel chemoreceptors. Due to the modification of the binding site these should be capable of recognizing new chemical substances. Therefore, we performed library selection assays using seven different chemicals. We expected <I>E. coli</I> to perform chemotaxis towards these chemicals as illustrated in Figure 15. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 16: Novel chemoreceptors recognize different tested chemicals. </B> The designed Tar_QC library was transformed into BL21 cells and then tested on swimming agar plates with seven different interesting chemicals. After three rounds of selection of the fastest swimming <I>E. coli</I>s towards these, three clones per chemical were picked. Sequencing results identified two novel chemoreceptors that were found for different chemicals: the novel receptor with amino acid sequence <B>LT</B>AQP<B>D</B>…<B>S</B>SAV<B>P</B> (red boxes) was identified for swimming towards D-aspartate, 2-ethyl-hexanol and sodium cyclamate and the novel receptor with amino acid residues <B>PP</B>AQP<B>P</B>…<B>P</B>SAV<B>T</B> (orange boxes) was identified for swimming towards L-aspartate and caffeine. The green boxes show different identified receptors. The receptor colored in yellow are structural similar to the orange boxes.</DIV></DIV></DIV><P>D-aspartate and L-aspartic acid-4-benzyl ester were chosen because of their structural similarity to D- aspartate. Moreover, the four flavors – vanillin, geraniol, caffeine and sodium cyclamate were interesting to test. The most interesting chemical is 2-ethyl-hexanol, a volatile organic compound related to lung cancer [2]. As mentioned above, the library selection assay was performed (Fig. 2A). After three rounds of selection of the fastest <I>E. coli</I> were sequenced. Figure 16 depicts novel identified chemoreceptors. A particular color code reflects the sequence identity and structure similarity. The novel receptor colored in red with amino acid sequence <B>LT</B>AQP<B>D</B>…<B>S</B>SAV<B>P</B> was identified two times for D-aspartate, 2-ethyl-hexanol and sodium cyclamate. Another novel receptor colored in orange with amino acid sequence <B>PP</B>AQP<B>P</B>…<B>P</B>SAV<B>T</B> was found two times for L-aspartic acid-4-benzyl ester and once for caffeine. However, for caffeine a structural related receptor mutant to the orange colored receptor was identified since the amino acids proline and serine are similar in structure.
</P><H3><SPAN class="mw-headline" id="Epilogue"><B>Epilogue</B></SPAN></H3><P>In conclusion the selection method worked although the selectivity was low. Nevertheless, we hold two novel chemoreceptors in our hands.
All these experiments provide evidence for the functionality of the BioBrick K777001.
</P><H3><SPAN class="mw-headline" id="References_3">References</SPAN></H3><UL><LI>[1] <A href="http://chemotaxis.biology.utah.edu/Parkinson_Lab/publications/PDFs/Hazelbauer%20et%20al.,%202008.pdf" class="external text" rel="nofollow">Hazelbauer G.L., Falke J. J., and Parkinson J. S. 2008. <I>Bacterial chemoreceptors: high-performance signaling in networked arrays.</I> Trends Bichem Sci. 33(1): 9–19.</A></LI><LI>[2] <A href="http://ar.iiarjournals.org/content/29/1/419.full.pdf+html" class="external text" rel="nofollow">Filipiak W., Sporing A., Filipiak W., Mikoviny T., Ager C., Schubert J., Miekisch W., Amann A., and Troppmair J.. 2009. <I>Release of Volatile Organic Compounds from the Lung Cancer Cell Line NCI-H2087 In Vitro.</I> Anticancer Research 29: 419-426.</A></LI></UL><P><TABLE width="900px" cellspacing="0" cellpadding="0"><TBODY><TR><TD><A href="#top">↑ Back to top!</A></TD></TR></TBODY></TABLE></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Goettingen/Notebook/Results">http://2012.igem.org/Team:Goettingen/Notebook/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Goettingen/Notebook/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Goettingen/Notebook/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Goettingen/Notebook/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Goettingen/Notebook/Results&amp;oldid=227345" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>