"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/Project/Bioinformatical Tool
From 2012.igem.org
Home
Homepage
News
Team
Members
Focus Groups
Seminars
Work Impressions
Project
Our Project
Results
Discussion/Outlook
Lab Work Flow
Notebook
Overview
Materials
Methods
Computational Data
Bioinformatical Tools
iGEM
What is iGEM?
Parts Submitted
Attributions
Safety
>
Göttingen
City
University of Göttingen
Max-Planck-Institute
S1-Demo Lab
Human Practice
Our Human Practice Projects
Public and Media
Survey
Synthetic Biology Day
Panel Discussion
Flash Coli
Sponsoring
Our Sponsors
Interested in...?
Deutsch
 /  English 
Contents
1Link List of Bioinformatical Tools
2A plasmid Editor ApE
2.1Primer Design with ApE
2.2Primer Binding with ApE
2.3Find restriction sites and fragment lenghts with ApE
2.4Blasting Sequences
2.5Finding the ORF
2.6General Remarks
3Blasting of Sequences
4Tm calculator
5Double Digest Finder
6Enzyme Finder
7NEBcutter V2.0
8ORF Finder
9RE Base
10WebLogo
Link List of Bioinformatical Tools
Here, an useful link list of computational and bioinformatical tools is provided. All of these programs are regularly used by the students to do the cloning process.
A plasmid Editor ApE - Use of ApE
Blasting of Sequences - BLAST: Basic Local Alignment Search Tool of NCBI
Tm calculator - Calculator for Primer Melting Temperature
Double Digest Finder - Double Digest Finder of Thermo Scientific
Enzyme Finder - Enzyme Finder of NEB
NEBcutter V2.0 - NEBcutter V2.0
ORF Finder - ORF Finder of NCBI
RE Base - The Restriction Enzyme Database RE Base of NEB
WebLogo - WebLogo
A plasmid Editor ApE
File:Goett Ape BG.gif
The Ape Logo
The plasmid Editor ApE by M. Wayne Davis is a free ware program conceived for both Windows (XP, Vista and 7) and Mac (OS X v10.5 and above). It can be downloaded here . The program offers lots of applications required for cloning processes, e.g. construction of plasmid maps, primer design, sequence alignments, management of sequences, ORF finder, Tm calculator, translation of nucleotide sequences and a lot more. ApE is compatible for the handling of common sequencing formatted .seq and .ab1 files. For further information visit the ApE homepage. Moreover, the hoster´s of the program take care about an ApE Wiki where you can find help and advice if questions in the use of ApE pop up.
Primer Design with ApE
Primers depend mainly on the chosen criteria. Yet, the specifity and the tendency to form hair-pins drastically reduces straight forward PCR amplification of genes. ApE also offers a primer design feature.
Highlight the sequence to which the primer should bind. Recognize that given standard minimum and maximum length is 20 bp and 25 bp, respectively. Nevertheless, these options can be altered.
Select "Tools -> Find Primers". A new window Find primers will emerge offering lots of different manipulatable attributes.
Click "OK" to see the possible primers.
Primer Binding with ApE
Fig. 1: ApE window for binding of primers
In this example, ApE is used to find the binding site on Lambda DNA for primer with a specific sequence.
Open ApE and paste the desired sequence, for which the primer is designated, in the sequence field (or open according .ape file, if made before.
Go to Edit>Find (or use Ctrl+F).
Paste the primer sequence and check "also find rev-com of string" (the primer may be identical to (a part of) the opposite DNA strand.) and click the "Find Next" button, as shown in Fig. 1.
The in blue highlighted sequence appears to be the desired primer sequence annealing to the template sequence. Check also if primer is indeed the reverse complement. Below the icon bar, one can find information about the length of the primer 30 nucleotides and the binding site.
For an illustrative description of each steps, please follow this link
Find restriction sites and fragment lenghts with ApE
Start ApE and open desired sequence.
Go to the Enzyme Selector (or use Ctrl+E) and click on the wanted restriction enzymes; the amount of restriction sites will be indicated in brackets.
One can now continue with different options:
searching different restriction enzymes with particular properties applying the various buttons in the "Select panel",
or highlighting the restiction site within the given sequence by pressing "Highlight" button,
or simulating a restriction in silico by klicking on the "Digest" button.
A new window will appear showing the digestion results. Additional information about the different bands can be received by hovering the mouse arrow over the bands, map or text. A convenient way to see the respective sequence behind a digested fragment is by simply klicking on the fragment within the digest window. For a description with the use of pictures for each step, please follow this link .
Blasting Sequences
For blasting two Sequences:
Open two sequences on ApE.
Select "Tools -> Align Two Sequences".
For blasting multiple Sequences:
Open multiple sequences on ApE.
Select "Tools -> Align Sequences".
It is important to note that the opening chronology of the ApE files will matter in the order of the alignment sequences. Those will show up in according sequence from top to bottom. Mismatches will be highlighted in red color, whereas matches will use the respective nucleotide linked with a dash. By double-clicking on any base pair within the sequence alignment, the sequence corresponding to this location will appear in the sequence ApE window.
Finding the ORF
Under "ORFs -> Find Next (or Ctrl+>) / Find Previous (or Ctrl+<)" open reading frames, i. e sequences beginning with a start codon ATG and end with one of the stop codons, become visible.
These ORFs can be translated using "ORFs -> Selection Translate" for direct amino acid sequence in the current displayed selection or by creating a new window "ORFs (or Ctrl+T)-> Translate" listing the aa sequence and enabling to link the amino acids with the nucleotides, respectively. The translated sequence can be chosen in 1- or 3-letter code.
General Remarks
While copying features from one file to another, it is obligatory to have the file containing those features still open.
Addition of extra enzymes is possible only via the "Enzyme Editor (or Ctrl+E) -> Enzymes -> New Enzyme". To save these changes in the program folders ApE, use "Enzyme Editor (or Ctrl+E) -> File -> Save Current Enzymes as Default". For permanent saving exceeding new installing of the program, copy your Default file within the ApE program pathway. After installing paste this file in the corresponding folder.
Features in the DNA sequences can be labeled, selecting "Features (or Ctrl+.) -> New Feature" allowing to mark the standard BioBrick restriction sites obvious or e.g. antibiotic resistances. These annotations are also saved. Moreover, it has a preexisting Feature Library which can be used for easily see important regions.
Blasting of Sequences
This link allows for alignment of two or even multiple given sequences. To become more informed how to do the sequence alignment with BLAST follow here .
Tm calculator
This link provides an online calculation form for the melting temperature Tm of PCR primers referring to the so-called Nearest Neighbour method.
Double Digest Finder
This link helps to find the appropriate reaction conditions while cleaving a DNA substrate with two restriction enzymes simultaneously as double digestion is a common timesaving procedure. The Thermo Scientific Double Digest Finder tool is applied by selecting the two restriction enzymes and submitting the query. It gives the reaction conditions amenable to any two Thermo Scientific restriction enzymes.
Enzyme Finder
This link leads to a tool which allows for selection of restriction enzymes by name, sequence, overhang, or type. It should be noted that the single letter code nomenclature should be entered for restriction sites. The search results appear in a list having enzymes supplied by NEB at the top with corresponding displayed links.
NEBcutter V2.0
This link will lead one to the NEBcutter: a program to cleave DNA with restriction enzymes. By submitting a maximum size of the input file of 1 MByte or a maximum sequence length of 300 kb, a linearized sequence restriction map will be the result containing the marked restriction enzyme sites. The map also indicates the fashion of the sticky or blunted end cutters. Moreover, additional effects like methylation of the DNA sequence are given, too.
ORF Finder
"The ORF Finder (Open Reading Frame Finder) is a graphical analysis tool which finds all open reading frames of a selectable minimum size in a user's sequence or in a sequence already in the database. This tool identifies all open reading frames using the standard or alternative genetic codes. The deduced amino acid sequence can be saved in various formats and searched against the sequence database using the WWW BLAST server. The ORF Finder should be helpful in preparing complete and accurate sequence submissions. It is also packaged with the Sequin sequence submission software" (Quote from: http://www.ncbi.nlm.nih.gov/project/gorf/; 06/29/2012 ).
This link aids to find all open reading frames (ORFs) in a given sequence. It is an ideal alternative to the ApE ORF finding feature if an ApE sequence file has not been constructed yet or if ApE is not installed at the used computer.
RE Base
This link encompasses comprehensive information about restriction enzymes and is the official RE Database by NEB. It provides for instance information of suppliers,recognition sequence, isoschizomers and the restriction enzyme type. In regard of a single enzyme also the organism and organism type with respective growth temperature behind the abbreviation is listed.
Choose the search enzyme names or the recognition sequence search catagory, and type a keyword (e.g. ecor or gaattc) to quickly find info about specific enzymes and click REbase list to find more specialised information. In this example, BamHI was searched within the RE Base:
Acronym: BamH
Prototype: BamHI
Org #: 208
Organism: Bacillus amyloliquefaciens H
Organism type: bacteria
Organism source: ATCC 49763 (ATCC LINK)
Growth Temperature: 37°C
Experimental Evidence: biochemistry
Exhibits star activity
Enzyme gene cloned.
Enzyme gene sequenced.
Crystal data present.
Kinetics data present.
Molecular Weight: 24569
Have also a look at the Enzyme Finder as alternative source of information.
WebLogo
This link encompasses a web based application. It will help you to generate sequence logos easily. To create your own sequence logos it is important to enter the sequence data only in the allowed file type:
FASTA format
CLUSTALW format
Flat format.
↑ Back to top!
Retrieved from " http://2012.igem.org/Team:Goettingen/Project/Bioinformatical_Tool "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
