<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Grenoble-Alpes_selection skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Grenoble-Alpes/selection</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><SPAN class="error">Template loop detected: <A href="/Template:Grenoble-Alpes" title="Template:Grenoble-Alpes">Template:Grenoble-Alpes</A></SPAN><NAV><UL id="ulmenu"><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes">HOME</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#team-members">TEAM MEMBERS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#advisors">ADVISORS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#partners">PARTNERS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Collaborations">COLLABORATIONS</A></LI></UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/biology">BIOLOGY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Hardware">HARDWARE</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Demonstrate">PROOF OF CONCEPT</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Model">
						MODELING
					</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/actors_partnerships">ACTORS &amp; PARTNERSHIPS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ethic">ETHICS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/events">EVENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Safety">SAFETY &amp; SECURITY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ressources">RESSOURCES</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ressources">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols">PROTOCOLS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/components">COMPONENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/medal_criteria">MEDAL CRITERIA</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/characterization">CHARACTERISATION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/site_map">SITE MAP</A></LI></NAV></P><DIV id="container"><DIV id="welcome"><P id="titre-image">TARGET SELECTION</P></DIV><DIV id="sous-menu"><UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description" id="current-nav">PROJECT </A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/biology" id="current-menu">BIOLOGY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/bacteria_choice">BACTERIA CHOICE</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/selection" id="current-el">TARGET SELECTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/phage_lysis">PHAGE LYSIS &amp; DNA EXTRACTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/construction">PROBE CONSTRUCTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/conservation">CONSERVATION</A></LI></UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Hardware">ENGINEERING</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Demonstrate">DEMONSTRATE</A></LI></DIV><DIV id="text-container"><P>Phagyzer aims to detect the presence of pathogenic bacteria such as <I>Pseudomonas aeruginosa</I> (Pyo). It will enable diagnosis by revealing the presence of the bacteria but also if this bacteria has acquired an antibiotic resistance.</P><H3><CENTER><FONT color="#f98d00">How do we do this? </FONT></CENTER></H3><P>At first, the bacterium that we want to detect will be lysed specifically by a phage. Its DNA will be released in the solution and ready to be detected.
To do this, a circular plasmid with a single-stranded portion is constructed. The feature is that the single-stranded part will be the perfect complementary sequence of the target that we want to detect in order to recirculate the DNA probe.
If the DNA is detected, it means that the plasmid re-circularized itself and is ready to be transformed into <I>E.coli</I> in order to produce fluorescence. Finally, if the system detects fluorescence it means that the lysis has been done.</P><P>
 
Thus, the first step is to select the target that characterizes the lysis and resistance markers of P. aeruginosa. We choose to take a fragment of DNA to characterize the lysis, since it will be detectable only if the DNA is released. Regarding resistance markers, we focused on genes that are often the cause of  it.</P><P>These fractions will be the targets of our detection probes. </P><H2><CENTER><FONT color="#f98d00">PSEUDOMONAS AERUGINOSA DETECTION</FONT></CENTER></H2><P>In Pyo, the first studies were possible thanks to the sequencing of the complete genome of the pathogenic strain PAO1 [1], this strain is considered to be the reference strain. This is why in our project, we chose to use this bacterial strain in order to prove the concept of our system.</P><H3><FONT size="5"><FONT color="#f98d00"><CENTER>Target that characterizes the lysis of Pyo</CENTER></FONT></FONT></H3><P>

	The detection of a bacterial lysis will be done by recognizing a gene fragment. Indeed, the gene fragment is intracellular, so when it is detected, it implies that the lysis occurred.</P><P>
	With this in mind, some reading was done on how to characterize the lysis of <I>Pseudomonas aeruginosa</I> by bacteriophages. At first, we decided to work on viral factors genes. However, most of them are also found in other strains than <I>Pseudomonas aeruginosa</I>. We were working on PAO1 strain.</P><P>

	Housekeeping genes are genes that are required for the maintenance of basic cellular functions. They are normally expressed in all bacteria from one strain. We were looking for genes that would be present in most, perhaps even all <I>Pseudomonas aeruginosa</I>. So we choose to focus on this type of genes. </P><P>
	The housekeeping gene used is ProC. Indeed, in 2003, Hakan Savli et al. [3] study “showed that proC and rpoD form the most stable pair in a set of clonally unrelated P. aeruginosa strains with diverse resistance phenotypes.” Moreover, they concluded that this pair could be used as internal controls in relative comparison studies of resistance genes in P. aeruginosa.</P><P>
	Once the gene found, it was inserted into NebCutter to look at the naturally occurring restriction sites. The goal was to find a sequence of about 20-100 nucleotides. This sequence would become the target that we want to detect. </P><P>

	The second objective was to find a specific fragment of <I>Pseudomonas aeruginosa</I>. For this, these fragments are entered in the NCBI database for sequence alignment to judge their relevance. The parameters of the BLAST are : Excluded “<I>Pseudomonas aeruginosa</I> group” and  “Highly similar sequences, megablast”.</P><P>
	By trying several fragments we chose the one that were the most specific to PAO1.</P><P>

	The result (Fig. 1) shows a 100% homology with <I>Pseudomonas</I> sp. AK6U. The AK6U strain of P.A produces rhamnolipid biosurfactants in varying degrees when grown on MgSO 4 [3]. This strain is found in soils contaminated with used lubricating oil, benzene and diesel and not in clinical cases. </P><FIGURE><CENTER><FIGCAPTION>Figure 1: Result of alignment of the target sequence of the detection of bacterial lysis in PAO1.</FIGCAPTION></CENTER></FIGURE><P>	This result isn’t problematic as the specificity will be provided by the phage. Indeed, a phage is specific of a bacterial strain, so a screening of phages will be done before their use in the device. Finally, the target found in ProC gene (822bp) from PAO1 strain (GenBank : AAG03782.1) is located in PA0393 locus. The target is located between nucleotide 766 and nucleotide 802.</P><FIGURE><CENTER><FONT size="3"> 5’- CCCTGAACGCCGCCAGCCAGCGCTCCGCCGAGCTGG-3’</FONT></CENTER><CENTER><FIGCAPTION>Figure 2: Target of lytic biologically selected PAO1 lysis.</FIGCAPTION></CENTER></FIGURE><P>	As for the enzyme surrounding the sequence on both sides it is HaeIII.</P><FIGURE><CENTER><FIGCAPTION>Figure 3: Recognition Site of the HaeIII Enzyme</FIGCAPTION></CENTER></FIGURE><FIGURE><CENTER><FIGCAPTION>Figure 4: Target chosen surrounded by HaeIII, blunt ends restriction enzyme.   </FIGCAPTION></CENTER></FIGURE><P>	All this work has been carried out in order to select the target after extracting Pyo's DNA. Indeed, after the extraction, HaeIII digestion for 15 minutes at 37 ° C in CutSmart Buffer will be performed. The target is now ready to be detected and the next step, the detector activation, can occur.</P><H3><FONT size="5"><FONT color="#f98d00"><CENTER>Target that characterizes a marker of resistance of Pyo</CENTER></FONT></FONT></H3><P>

When selecting the gene fragment to target markers of resistance, the strategy was the same. 
Indeed, we had to choose a fragment with these 3 characteristics but the feature here is to find a short fragment with more than 1 mutation. </P><P>Pyo has developed many resistance mechanisms during evolution. The most common mechanisms are multiple mutations leading to AmpC overexpression (ceftazidime), oprD inactivation (meropenem), modification of type II topoisomerases, as well as overexpression of the efflux pump (ciprofloxacin and meropenem) [4]. </P><P>By doing some reading, we selected the gene that mutated the most and looked for common mutations on it. The selected gene was gyrA (GenBank: AAG06556.1) gene. It encodes for the DNA gyrase subunit A (topoisomerase II) and is on the PA3168 genome PA3168 locus. The gyrase DNA mutation leads to resistance to fluoroquinolones [5] [6]. Fluoroquinolones are antibiotics acting on DNA gyrase, it prevents replication of bacterial DNA and thus bacterial proliferation.</P><P>We noticed in several articles explaining the mutations of the gyrA gene of Pyo clinical strains, that some were redundant. First, at position 83 of the gyrA gene, threonine becomes an isoleucine (Thr83Ile) [5] [6] [7] [11]. 
Moreover, in position 87 an aspartate becomes an asparagine (Asp87Asn) [5] [7] [8]. So we decided to work on these two mutations. The advantage is that they are close in the gyrA gene (12 nucleotides between them), this is an important point because the detection target sequence must not exceed 50 nucleotides. Once the location was found, the gene mutated was inserted in NebCutter to see natural restriction sites.</P><P>Finally, a sequence alignment in BLAST was performed to ensure that the fragment was only found in Pyo. The parameters of the BLAST are : Excluded “<I>Pseudomonas aeruginosa</I> group” and  “ Highly similar sequences, megablast”. The result (Fig.5) shows 4 alignment results with 100% homology. For two of them, the result is normal because it corresponds to the gene coding for the subunit A of DNA gyrase (gyrA). For the 2 others, <I>Pseudomonas Putida</I> is found in most soils and aquatic habitats where there is oxygen. P.P is able to degrade organic solvents such as toluene, it is not found in clinical cases. This result isn’t problematic as the specificity will be provided by the phage.</P><P>Indeed, we used HER18 phage that is it specific against PAO1 strain (taxonomy: txid280701). 
In addition, as previously said, the specificity will be provided by the phage. Indeed, a phage is specific for a bacterial strain, so a selection will have been made before using the engineering system. With all these elements, we can ignore this homology.</P><FIGURE><CENTER><FIGCAPTION>Figure 5: Result of the alignment of the target sequence of resistance detection with PAO1 antibiotics. Parameters : Exclude “<I>Pseudomonas aeruginosa</I> group”  and  “ Highly similar sequences, megablast”.   </FIGCAPTION></CENTER></FIGURE><P>At the end, a fragment of 42 nucleotides was selected, the target is located between nucleotide 229 and nucleotide 271 of GyrA gene. </P><CENTER><FONT size="3">(A): </FONT></CENTER><CENTER>5’ – ACCACCCGCACGGCGACACCGCGGTCTACGACACCATCGTGC – 3’ </CENTER><CENTER>(B): </CENTER><CENTER>5’ – ACCACCCGCACGGCGACATCGCGGTCTACAACACCATCGTGC – 3’ </CENTER><FIGCAPTION>Figure 6: (A) - Non-mutated sequence present in the PAO1 (B) genome - Target B antibiotic resistance of biologically selected PAO1 (Red: mutations).   </FIGCAPTION><FIGURE><CENTER><FIGCAPTION> Figure 7: Target chosen surrounded by RsaI and FspI, two blunt ends restrictions enzymes.     </FIGCAPTION></CENTER></FIGURE><P>The enzymes surrounding the sequence are Ras-I 5 'and FspI side 3' respectively.</P><FIGURE><CENTER><FIGCAPTION> Figure 8: Ras-I enzyme recognition site     </FIGCAPTION></CENTER></FIGURE><FIGURE><CENTER><FIGCAPTION> Figure 9: FspI enzyme recognition site </FIGCAPTION></CENTER></FIGURE><P>All this work has been carried out in order to select the target after extracting Pyo's DNA. Indeed, after the extraction, RsaI and FspI digestions will be performed for 15 minutes at 37 ° C in CutSmart Buffer. The target is now ready to be detected and the next step, the detector activation, can occur.</P><FONT color="#f98d00"><CENTER><H3><FONT size="6">Keep in mind</FONT></H3></CENTER></FONT><DIV style="padding:5px; width: 500px; margin:auto; background-color:#f98d00; border:2px solid #d7906; -moz-border-radius:9px; -khtml-border-radius:9px; -webkit-border-radius:9px; border-radius:9px;"><CENTER><H3><FONT color="white"> We wanted a target responding to 3 criteria: </FONT></H3></CENTER><CENTER><H3><FONT color="white">size: 20 to 100 nucleotide</FONT></H3></CENTER><CENTER><H3><FONT color="white">cut in blunt ends</FONT></H3></CENTER><CENTER><H3><FONT color="white">to be as specific as possible to <I>Pseudomonas aeruginosa</I></FONT></H3></CENTER></DIV><DIV style="padding:3px; padding-left:6px; border:1px dotted #d0d0d0; border-left:4px solid #d0d0d0; margin-left:20px;"><H3><FONT size="4">REFERENCES</FONT></H3><P><FONT size="3">[1] Complete genome sequence of <I>Pseudomonas aeruginosa</I> PAO1, an opportunistic pathogene, MV Olson,Nature volume 406, pages 959-964, 31 august 2000 </FONT></P><P><FONT size="3">[2] Expression stability of six housekeeping genes: A proposal for resistance gene quantification studies of Pseudomoas aeruginosa by real-time quantitative RT-PCR, Savli H, Journal of Medical Microbiology 52(pt:5): 403-8, June 2003 </FONT></P><P><FONT size="3">[3] Stimulation of rhamnolipid biosurfactants production in Pseudomonas aeruginosa AK6U by organosulfur compounds provided as sulfur sources, Wael Ismail, Biotechnol Reports, 7: 55-63, 2015 September </FONT></P><P><FONT size="3">[4] Evolution of Pseudomonas aeruginosa Antimicrobial Resistance and Fitness under Low and High Mutation Rates, Gabriel Cabot, Antimicrobial Agents and Chemotherapy, Volume 60 Number 3, March 2016 </FONT></P><P><FONT size="3">[5] Type II topoisomerase mutations in fluoroquinolone-resistant clinical strains of Pseudomonas aeruginosa isolated in 1998 and 1999: role of target enzyme in the mechanism of fluoroquinolone resistance, Akasaka T, Antimicrobial Agents and Chemotherapy, 45(8): 2263-8, 2001 August </FONT></P><P><FONT size="3">[6] Type II topoisomerase mutations in ciprofloxacin-resistant strains of Pseudomonas aeruginosa, Mouneimné H, Antimicrobial Agents and Chemotherapy, 43 (1): 62-6, 1999 January </FONT></P><P><FONT size="3">[7] GyrA mutations in ciprofloxacin-resistant clinical isolates of Pseudomonas aeruginosa in a Silesian Hospital in Poland, Wydmuch Z, Polish Journal of Microbiology, 54 (3): 201-6, 2005 </FONT></P><P><FONT size="3">[8] The role og gyrA and parC mutations in fluoroquinolones-resistant Pseudomonas aeruginosa isolates from Iran, Braz Journal Microbiol, 47 (4) : 925-930, 2016 Oct-Dec </FONT></P></DIV></DIV></DIV><FOOTER><A href="mailto:Igem.grenoble.alpes@gmail.com">Igem.grenoble.alpes@gmail.com</A></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>