<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Grenoble-Alpes_hybridation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Grenoble-Alpes/hybridation</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><SPAN class="error">Template loop detected: <A href="/Template:Grenoble-Alpes" title="Template:Grenoble-Alpes">Template:Grenoble-Alpes</A></SPAN><NAV><UL id="ulmenu"><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes">HOME</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#team-members">TEAM MEMBERS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#advisors">ADVISORS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#partners">PARTNERS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Collaborations">COLLABORATIONS</A></LI></UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/biology">BIOLOGY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Hardware">HARDWARE</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Demonstrate">PROOF OF CONCEPT</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Model">
						MODELING
					</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/actors_partnerships">ACTORS &amp; PARTNERSHIPS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ethic">ETHICS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/events">EVENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Safety">SAFETY &amp; SECURITY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ressources">RESSOURCES</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ressources">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols">PROTOCOLS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/components">COMPONENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/medal_criteria">MEDAL CRITERIA</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/characterization">CHARACTERISATION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/site_map">SITE MAP</A></LI></NAV></P><DIV id="container"><DIV id="welcome"><P id="titre-image">BACTERIA CHOICE</P></DIV><DIV id="sous-menu"><UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes">HOME</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/project">PROJECT </A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/biology">BIOLOGY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/hybridation" id="current-menu">BACTERIA CHOICE</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/phage_lysis">PHAGE LYSIS &amp; DNA EXTRACTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/selection">TARGET SELECTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/construction">PROBE CONSTRUCTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/conservation">CONSERVATION</A></LI></UL></DIV><DIV id="text-container"><P>We have been working this year on infectious diseases, particularly nosocomial diseases. </P><P>

A nosocomial infection is an infection contracted during hospitalization. It is therefore absent at the time of admission of the patient in the establishment and is declared at least 48 hours after admission. One major problem regarding those infections is that some pathogenic bacterias can develop resistance against antibiotics.</P><P>

Nowadays, antibiotic resistance is one of the biggest threats to global health, food security, and development. The World Health Organization, WHO, published last year, a list of high-priority pathogenic agents for which we need to find new treatments [1]. It lists the 12 most threatening families of bacterias for human health. </P><P>

In this list, <FONT color="#93024b"><I>Pseudomonas aeruginosa</I></FONT> is considered as a first priority pathogen. Indeed, the most worrying strains because of their antibiotic resistance are carbapenem-resistant strains [2]. However, most pathogenic <FONT color="#93024b"><I>Pseudomonas aeruginosa</I></FONT> strains are multi-drug resistant. So it is a challenge to detect these pathogenic strains which represent a particular threat in hospitals or nursing homes, and then find new treatments. </P><P>

Moreover, <FONT color="#93024b"><I>Staphylococcus aureus</I></FONT> is listed as a high priority pathogenic agent because of its multidrug resistance against vancomycin or methicillin antibiotics [3]. </P><P>

This is why, in the project, we decided to work on those two bacteria. </P><DIV class="collapse slide"><H2><CENTER><FONT color="#93024b">PSEUDOMONAS AERUGINOSA</FONT></CENTER></H2><DIV style="padding:5px; background-color:#ffffff; border:2px solid #93024b; -moz-border-radius:9px; -khtml-border-radius:9px; -webkit-border-radius:9px; border-radius:9px;"><P>The gram-negative <FONT color="#93024b"><I>Pseudomonas aeruginosa</I></FONT> bacterium is a common opportunistic pathogen. It can infect humans with compromised natural defenses and causing severe pulmonary disease [4].
</P><P>Opportunistic bacteria can symptomatically infect healthy individual and become pathogenic during a weakening of the body's defenses.</P><P><FONT color="#93024b"><I>Pseudomonas aeruginosa</I></FONT> is likely to cause the following conditions: pneumonia, sepsis, endocarditis, CNS infection, digestive, urinary and skin infections
[5]. </P><P>Various treatments, with different action mechanism, using penicillins, cephalosporins, carbapenems, monobactams, aminoglycosides, fluoroquinolones, and polymyxins have shown a good activity and efficiency against most of P. aeruginosa isolates. All, however, are prone to be compromised by P.aeruginosa acquisition of mutations conferring resistance to these treatments [6]. </P></DIV></DIV><DIV class="collapse slide"><H2><CENTER><FONT color="#93024b">STAPHYLOCOCCUS AUREUS</FONT></CENTER></H2><DIV style="padding:5px; background-color:#ffffff; border:2px solid #93024b; -moz-border-radius:9px; -khtml-border-radius:9px; -webkit-border-radius:9px; border-radius:9px;"><P><FONT color="#93024b"><I>Staphyloccocus aureus</I></FONT> is a common gram-positive bacterium. </P><P><FONT color="#93024b"><I>Staphylococcus aureus</I></FONT> is the main bacteria involved in skin infections with pus formation, particularly in the case of boils, paronychia, folliculitis, sycosis, impetigo, styes and suppurations of wounds and abscesses. Other localized <FONT color="#93024b"><I>Staphylococcus aureus</I></FONT> infections include respiratory tract infections (pneumonia and pleurisy), endocarditis, urinary tract infections, phlebitis, meningitis (sometimes involving S. aureus), ear infections and sinusitis (S aureus sometimes involved) and bone infections: osteomyelitis.</P><P>While it is one of the most common of our natural flora, <FONT color="#93024b"><I>Staphylococcus aureus</I></FONT> is a formidable pathogen that has developed resistance to each new antibiotic introduced for half a century. The plasticity of its genome gives it the ability to adapt to all environmental conditions, including the acquisition of genes for antibiotic resistance and to develop regulatory mechanisms to adapt to increasing concentrations of antibiotics [7].</P></DIV></DIV><P>As you can see, both <FONT color="#93024b"><I>Pseudomonas aeruginosa</I></FONT> and <FONT color="#93024b"><I>Staphylococcus aureus</I></FONT> represents a major threat from an antibiotic resistance point of view. However, we eventually decided to focus on <FONT color="#93024b"><I>Pseudomonas aeruginosa</I></FONT> as we were lacking time and biology staff. Moreover, a laboratory in our university is working on <FONT color="#93024b"><I>Pseudomonas aeruginosa</I></FONT> and was ready to help us during our project. </P><DIV style="padding:3px; padding-left:6px; border:1px dotted #d0d0d0; border-left:4px solid #d0d0d0; margin-left:20px;"><H3><FONT size="4">BIBLIOGRAPHY</FONT></H3><P><FONT size="3">[1] GLOBAL PRIORITY LIST OF ANTIBIOTIC-RESISTANT BACTERIA TO GUIDE RESEARCH, DISCOVERY, AND DEVELOPMENT OF NEW ANTIBIOTICS, WHO, 2018</FONT></P><P><FONT size="3">[2] Carbapenem-resistant Pseudomonas aeruginosa: association with virulence genes and biofilm formation,  2017 Apr-Jun; 48(2): 211–217, Braz J Microbiol</FONT></P><P><FONT size="3">[3] Methicillin-resistant Staphylococcus aureus: a controversial food-borne pathogen, D. Sergelidis and A.S. Angelidis, Letters in Applied Microbiology 64, 409 -- 418, accepted 11
March 2017</FONT></P><P><FONT size="3">[4] International Journal of Biology-Vol. 7, No. 2; 2015-Pseudomonas aeruginosa – Pathogenesis and Pathogenic Mechanisms Alaa Alhazmi</FONT></P><P><FONT size="3">[5] Survey of Pseudomonas aeruginosa and its phages: de novo peptide sequencing as a novel tool to assess the diversity of worldwide collected viruses, Ceyssens PJ, Environmental Microbiology (2009) 11(5), 1303–1313</FONT></P><P><FONT size="3">[6] Multiple Mechanisms of Antimicrobial Resistance in Pseudomonas aeruginosa: Our Worst Nightmare?-Clinical Infectious Diseases, Volume 34, Issue 5, 1 March 2002, Pages 634–640-David M. Livermore)</FONT></P><P><FONT size="3">[7] Community-acquired meticillin-resistant Staphylococcus aureus: an emerging threat. Volume 5, Issue 5, May 2005, Pages 275-286, NicolaZetola, Elsevier</FONT></P></DIV></DIV></DIV></DIV><FOOTER><A href="mailto:Igem.grenoble.alpes@gmail.com">Igem.grenoble.alpes@gmail.com</A></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>