<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Grenoble-Alpes_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Grenoble-Alpes/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><SPAN class="error">Template loop detected: <A href="/Template:Grenoble-Alpes" title="Template:Grenoble-Alpes">Template:Grenoble-Alpes</A></SPAN><NAV><UL id="ulmenu"><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes">HOME</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#team-members">TEAM MEMBERS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#advisors">ADVISORS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#partners">PARTNERS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Collaborations">COLLABORATIONS</A></LI></UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/biology">BIOLOGY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Hardware">HARDWARE</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Demonstrate">PROOF OF CONCEPT</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Model">
						MODELING
					</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/actors_partnerships">ACTORS &amp; PARTNERSHIPS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ethic">ETHICS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/events">EVENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Safety">SAFETY &amp; SECURITY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ressources">RESSOURCES</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ressources">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols">PROTOCOLS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/components">COMPONENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/medal_criteria">MEDAL CRITERIA</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/characterization">CHARACTERISATION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/site_map">SITE MAP</A></LI></NAV></P><DIV id="container"><DIV id="welcome"><P id="titre-image">NOTEBOOK</P></DIV><DIV id="sous-menu"><UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ressources" id="current-nav">RESSOURCES</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Notebook" id="current-menu">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Notebook#biology">TEAM BIO</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Notebook#hardware">TEAM HARDWARE</A></LI></UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols">PROTOCOLS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/components">COMPONENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/medal_criteria">MEDAL CRITERIA</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/characterization">CHARACTERISATION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/site_map">SITE MAP</A></LI></DIV><DIV id="text-container"><P>Here, you can find the Notebook of each team member. To ease the reading, we classified them using the main themes that each person worked on but of course, we helped each other a lot and sometimes you can find mensions of this and of other activities linked with the iGEM competition in the Notebooks.</P><DIV style="padding:3px; padding-left:6px; border:5px solid #9c5899; margin-left:20px; background-color: #9c5899;" id="biology"><CENTER><FONT id="optitl"><FONT size="10"><FONT color="white">BIOLOGY NOTEBOOK</FONT></FONT></FONT></CENTER></DIV><H2><FONT size="6"><CENTER>Biobrick design (Fanny &amp; Elise)</CENTER></FONT></H2><DIV class="collapse slide"><H2><FONT color="#9c5899"><FONT size="5">April </FONT></FONT></H2>

During this month we mainly carried out literature. In fact, we had to define the targets in Pseudomonas Aeruginosa and Staphylococcus Aureus we want to detect. After that, we designed the detection probe of these targets.

</DIV><DIV class="collapse slide"><H2><FONT color="#9c5899"><FONT size="5">May</FONT></FONT></H2>

The first experiments ! 
We didn’t have yet the 2018 distribution kit so we decided to practice transformation on the 2017 distribution kit. We identified the plasmids of interest and recovered them from the plates. 
When we received the distribution kit we started to produce psB1C3-BBa_J04450 and psB1C3-BBa_K592027.

<P><B>05/23/2018</B></P>

With the 2017 distribution kit 

→ psB1C3-BBa_J04450 and psB1C3-BBa_K592027 transformation into DH5alpha<P><B>05/24/2018</B></P>

→ Transformation results : no colonies expect for psB1C3-BBa_J04450 
→ 2 preculures of psB1C3-BBa_J04450 made<P><B>05/25/2018</B></P>

→ Mini prep of the psB1C3-BBa_J04450 preculture 
→ Enzymatic digestion with EcoRI,  XbaI and PstI to check that we produced the right plasmid
→ Culture of the other precultures<P><B>05/28/2018</B></P>

→ Trained the engineers team on the practice of a bacterial transformation
→ Production of Petri dishes with Ampicillin, Chloramphenicol or just LB
→ Midi prep of the culture of psB1C3-BBa_J04450<P><B>05/29/2018</B></P>

With the 2018 distribution kit: 

→ psB1C3-BBa_J04450 and psB1C3-BBa_K592027 transformation into DH5alpha<P><B>05/30/2018</B></P>

→ Results of the transformation : good results
→ preculture using one colonie of each plate<P><B>05/31/2018</B></P>

→ Mini prep of the psB1C3-BBa_J04450 and psB1C3-BBa_K592027 precultures 
→ Enzymatic digestion with EcoRI,  XbaI and PstI to check if we produced the right plasmid and to verify if there is indeed just one digestion site of EcoRI and XbaI (because at first, we wanted to insert our detection probe by cloning technic)</DIV><DIV class="collapse slide"><H2><FONT color="#9c5899"><FONT size="5">June</FONT></FONT></H2><P><B>06/01/2018</B></P>

→ petry dish 
→ preculture of psB1C3-BBa_K592027<P><B>06/04/2018</B></P>

→ preculture of psB1C3-BBa_K592027<P><B>06/05/2018</B></P>

→ Culture of psB1C3-BBa_K592027<P><B>06/06/2018</B></P>

→ Midi prep of psB1C3-BBa_K592027
→ Enzymatic digestion of psB1C3-BBa_K592027 and psB1C3-BBa_J04450 with SphI and NgomIV to make sure they are not in the plasmid<P><B>06/07/2018</B></P>

→ IPTG induction tests of psB1C3-BBa_K592027 <P><B>06/08/2018</B></P>

→ Probes amplification by PCR
→ Overnight enzymatic digestion of psB1C3-BBa_J04450 using EcoRI to linearize the plasmid <P><B>06/11/2018</B></P>

→ After the first digestion, another digestion with XbaI was done to prepare the plasmid for the cloning
→ 1% agarose gel = good
→ Gel extraction using NEB Monarch DNA Gel Extraction Kit
→ Bacterial transformation to verify the quality of the linearization of psB1C3-BBa_J04450<P><B>06/12/2018</B></P>

→ Bacterial transformation results: not as good as we expected
→ Probes amplification by PCR
→ Overnight EcoRI enzymatic digestion of psB1C3-BBa_J04450 to linearize the plasmid <P><B>06/13/2018</B></P>

→ After the first digestion, another digestion was done using XbaI to prepare the plasmid for the cloning
→ 1% agarose gel = good
→ Gel extraction using NEB Monarch DNA Gel Extraction Kit
→ Bacterial transformation to verify the quality of the linearization of psB1C3-BBa_J04450<P><B>06/14/2018</B></P>

→ Bacterial transformation results: not good 

We decided, at that point, to use the Gibson technic to create our clone.

→ Design the primers  we will use for the PCR and to later use them in the Gibson Assembly<P><B>06/15/2018</B></P>

→ Design of primers for the linearization of BBa_J04450 by PCR <P><B>06/16/2018</B></P>

→ Design of primers for the linearization of BBa_J04450 by PCR <P><B>06/18/2018 </B></P>

→ IDT order of the primers
→ Wiki redaction <P><B>06/19/2018</B></P>

→ preculture of psB1C3-BBa_K592027 and psB1C3-BBa_J04450 to produce a stock<P><B>06/20/2018</B></P>

→ Mini prep of the different preculture 
→ Enzymatic digestion to verify to make sure it is the right plasmid<P><B>06/21/2018</B></P>

→ Design of primers for the amplification and addition of the overlaps to the insert probe enabling to detect the lysis of a bacteria<P><B>06/22/2018</B></P>

→ IDT order of the primers 

06/25/2018

→ Wiki redaction<P><B>06/26/2018</B></P>

→ Wiki redaction<P><B>06/27/2018</B></P>

→ Wiki redaction<P><B>06/28/2018</B></P>

→ Primers arrived! 
→ PCR linearization of psB1C3-BBa_J04450
→ Enzymatic digestion of the PCR product with DNP1 in order to eliminate any false positives by digesting the methylated circular plasmid DNA
→ 1% agarose gel = good<P><B>06/29/2018</B></P>

→ PCR amplification of the insert allowing the detection of Pseudomonas aeruginosa lysis 
→ 2% agarose gel = Good</DIV><DIV class="collapse slide"><H2><FONT color="#9c5899"><FONT size="5">July</FONT></FONT></H2><P><B>07/02/2018</B></P>

→ Gibson in order to fuse the detection probe into the psB1C3-BBa_J04450 linearized
→ Bacterial transformation of the PCR products<P><B>07/03/2018</B></P>

→ Bacterial transformation results: GOOD
→ PCR colony 
→ 2% agarose gel = Not good<P><B>07/04/2018</B></P>

For both LYSIS and RESISTANCE

→ PCR colony with other colonies on the dishes
→ 2% agarose gel = Not good 
→ New Gibson with another dilution
→ Bacterial transformation<P><B>07/05/2018</B></P>

→ PCR Colony
→ 2% agarose gel = 2 colonies worked (LYSIS)
→ Preculture<P><B>07/06/2018 = Paris meeting</B></P>

→ Miniprep of the colonies (LYSIS)
→ DNA extraction using Monarch DNA Gel Extraction Kit (LYSIS)

→ BFP amplification (psB1C3-BBa_K592027) (RESISTANCE)
→ 1% agarose gel = Good (RES)<P><B>07/09/2018</B></P>

→ Enzymatic digestion with the restriction site that we added = SphI (LYSIS)
→ 1 % agarose gel = Not good

→ Amplification and addition of the overlap for the resistance insert (RES)
→ 2% agarose gel = Good<P><B>07/10/2018</B></P>

→ Gibson in order to add the resistant insert to J04450 linearized (RES)
→ Bacterial transformation (RES)

→ Same enzymatic digestion with SphI AND NgoMIV (LYSIS)
→ 1% agarose gel = Not good (LYSIS)<P><B>07/11/2018 </B></P>

→ Try to have more purify product for the Gibson technic using Invitrogen PureLink Quick PCR Purification Kit = bad concentration (LYSIS)

→ PCR colony (RES)
→ 1% agarose gel = GOOD (RES)
→ préculture (RES)<P><B>07/12/2018</B></P>

→ DNA extraction (RES)
→ Enzymatic digestion with SpeI and NgoMIV (RES)
→ 1% agarose gel = Good (RES)<P><B>07/13/2018</B></P>

→ PCR linearization of the new J04450 with the resistance insert (RES)
→ 1% agarose gel = Good (RES)<P><B>07/16/2018</B></P>

→ Gibson in order to add the BFP gene to J04450-RES linearized (RES)
→ Transformation (RES)<P><B>07/17/2018</B></P>

→ Colony PCR (RES)
→ 1% agarose gel = Good but the control is contaminated<P><B>07/18/2018</B></P>

→ preculture (RES)<P><B>07/19/2018</B></P>

→ Miniprep (RES)
→ SpeI digestion <P><B>07/23/2018</B></P>

→ Wiki redaction
→ Sequencing primer order<P><B>07/24/2018</B></P>

→ Wiki redaction<P><B>07/25/2018</B></P>

→ Wiki redaction<P><B>07/26/2018</B></P>

→ Wiki redaction<P><B>07/27/2018</B></P>

→ Wiki redaction<P><B>07/30/2018</B></P>

→ Wiki redaction<P><B>07/31/2018</B></P>

→ Wiki redaction</DIV><DIV class="collapse slide"><H2><FONT color="#9c5899"><FONT size="5">August</FONT></FONT></H2><P><B>08/01/2018</B></P>

→ Gibson of the resistance probes in BBa_J04450 
→ Bacterial transformation 
→ Digestion using SphI and NgoMIV for BBa_J04450 not cloned and 2 clones
→ 1% agarose gel = BAD 

→ PCR colony of the lysis insert + BBa_J04450 = Bad results <P><B>08/02/2018</B></P>

→ Bacterial transformation 
→ PCR colony 
→ 2% agarose gel = GOOD 
→ Preculture of some colonies 

→ PCR for linearization of J04450 = GOOD 
→ Digestion using DPN1 to eliminate false positive 
→ 1% Gel = GOOD
→ Invitrogen PureLink Quick PCR Purification Kit done on the PCR sample = BAD<P><B>08/03/2018</B></P>

→ DNA extraction using NucleoSpin Plasmid Mini Prep 
→ Digestion of the DNA extracted using NgoMIV and SpeI
→ 1% agarose gel = GOOD 

→ Gel 1% = GOOD 
→ Digestion using DPN1 
→ New Gibson on J04450 linearized and cleaned up + lysis probe amplification <P><B>08/06/2018</B></P>

→ PCR of the BBa_J04450 to take out the RFP gene and add overlaps
→ 1% agarose gel = GOOD 
→ DPN1 digestion
→ Gibson in order to add BFP gene
→ Bacterial transformation

→ Digestion using DPN1 
→ Invitrogen PureLink Quick PCR Purification Kit of J04450 linearized and digest
→ Gibson 
→ Bacterial transformation<P><B>08/07/2018</B></P>

→ Transformation result = GOOD
 
→ Gibson J04450 + lysis probe = BAD
→ 2% agarose gel J04450 + BFP inserted = GOOD 
→ PCR overnight of the lysis probe amplification<P><B>08/13/2018</B></P>

→ Colony PCR 
→ 2% agarose gel = GOOD
→ Preculture

→ 2% agarose Gel = GOOD
→ Gibson using the latest insert made 
→ Digestion by DPN1 of J04450 linearized 
→ Bacterial transformation
→ Preculture of J04450 + BFP + RES<P><B>08/14/2018</B></P>
→ DNA extraction using NucleoSpin Plasmid Mini Prep 
→ Enzymatic digestion with SpeI

→ PCR colony of the bacterial transformation obtained after the last Gibson for lysis insert
→ 2% agarose gel = BAD
→ Safety form <P><B>08/15/2018</B></P>

→ 1% agarose gel = Not good
→ New enzymatic digestion with RsaI
→ 1% agarose gel = GOOD

→ 1% agarose gel of J04450 linearized = GOOD 
→ 0.8% agarose gel of lysis insert = BAD
→ Monarch DNA Gel Extraction Kit of the J04450 sample that migrated on the 1% gel
→ 2 different PCR using different annealing temperature for the lysis insert 
→ Safety form <P><B>08/16/2018</B></P>

→ Beginning of the in-vitro construction 
→ Strategy one: PCR linearization (first step)
→ Enzymatic digestion by nicking enzymes (second step)
→ Clean Up PCR
→ Hybridization test (different ratio)
→ Transformations

→ 0.8% agarose gel = BAD 
→ IDT order of the lysis insert with the overlap so we will be able to use the Gibson with our J04450 linearized and the insert with overlaps that we ordered<P><B>08/17/2018</B></P>

→ Transformation results = Not good
→ Strategy two : enzymatic digestion by SphI and NgoMIV (first step)
→ Enzymatic digestion by nicking enzymes (seconde step)
→ Hybridization test (different ratio)
→ Transformations

→ Order the lysis insert with overlaps on IDT
→ Safety form 

<P><B>08/20/2018</B></P>

→ Transformation results = a few colonies but not good
→ Strategy two : enzymatic digestion by SphI and NgoMIV (first step)
→ Enzymatic digestion by nicking enzymes (seconde step)
→ Clean Up PCR
→ Hybridization test (different ratio)
→ Transformations

<P><B>08/21/2018</B></P>

→ Transformation results = little better but not good 
→ Strategy one : PCR linearization (first step) = not good 
→ Strategy two : enzymatic digestion by SphI and NgoMIV (first step) = overnight

<P><B>08/22/2018</B></P>

→ Enzymatic digestion by nicking enzymes (seconde step)
→ Hybridization test (different ratio)
→ Transformations

<P><B>08/23/2018</B></P>

→ Transformation results = little better but mais pas assez bien 

<P><B>08/24/2018</B></P>

→ Wiki redaction

<P><B>08/27/2018</B></P>

→ 5 preculture of the transformation

→ Digestion of the J04450 with DPN1
→ 1% agarose gel = GOOD
→ Gibson 
→ Bacterial transformation 



<P><B>08/28/2018</B></P>

→ DNA extraction using NucleoSpin Plasmid Mini Prep
→ Send to sequencing 
→ Strategy two: enzymatic digestion by SphI and NgoMIV (first step) = overnight

→ Preculture of the colonies on the dishes

<P><B>08/29/2018</B></P>

→ Enzymatic digestion by nicking enzymes (seconde step) (wait for the new Clean Up PCR kit)
→ Strategy one: PCR linearization (first step) with another enzyme
→ 1% agarose gel → Gel extraction  

→ Mini prep of the different preculture 
→ Digestion of one of the colony with NgoMIV 
→ Two 1% agarose gel = GOOD <P><B>08/30/2018</B></P>

→ DPN1 digestion  
→ Enzymatic digestion by nicking enzymes (seconde step) = wait for the new Clean Up PCR kit 

→ Linearization of J04450 with the lysis insert by PCR  
→ 1% agarose gel = GOOD  
→ Gel extraction  <P><B>08/31/2018</B></P>

→ Safety form </DIV><DIV class="collapse slide"><H2><FONT color="#9c5899"><FONT size="5">September</FONT></FONT></H2><P><B>09/05/2018</B></P>

→ PCR clean up  
→ hybridization of both resistance and lysis probes  
→ Bacterial transformation  
→ Pre culture for sequencing  <P><B>09/06/2018</B></P>

→ Digestion with Sph I and NgoMIV to activate the probe 
→ Digestion with Nb.BssSI and Nt.BspQI   
→ Sequencing preparation to characterize our biobricks  
→ First test to characterize the biobricks: kinetic tests   <P><B>09/07/2018</B></P>

→ New digestion of other construct using Sph I and NgoMIV  
→ Digestion with Nb.BssSI and Nt.BspQI 
→ New hybridization </DIV><H2><FONT size="6"><CENTER>DNA extraction and bacteriophages (Cécile &amp; Yasmine)</CENTER></FONT></H2><DIV class="collapse slide"><H2><FONT color="#9c5899"><FONT size="5">June</FONT></FONT></H2><P><B>June 18 to 26:</B></P>
→ Bibliographic studies to develop bacteriophage protocols →  How to lyse, amplify and titrate bacteriophages.
→ Goal setting<P><B>06/27/18</B></P>
→ Preparation of the necessary solutions for the protocols :
            	- CaCl2 et MgCl2 → divalent ions required for bacteriophage adsorption (they play a role in the penetration of phage DNA into the cell)
            	- SM Buffer: bacteriophage conservation buffer <P><B>06/28/18</B></P>
→ Petri dish + Bibliography</DIV><DIV class="collapse slide"><H2><FONT color="#9c5899"><FONT size="5">July</FONT></FONT></H2><P><B>07/02/18</B></P>
→ Put E.coli F on dishes from the glycerol stock recovered from the engineers (we have recovered T5 bacteriophages too).
→ Preculture and spread on dishes with 2 different concentrations.
→ Preparation of LB Agar + Top agar<P><B>07/03/18</B></P>
→ Preculture E. coli results: only the 1/10 dilution is exploitable.
→ Spot titration of T5 bacteriophages<P><B>07/04/18</B></P>
→ Spot titration results: 10<SUP>12</SUP> pfu/mL.
→ Interview with Cécile Breyton 
	→ it is easier to carry out a liquid amplification of the bacteriophages rather than to do it in solid medium /! Phages use DNA
	→ Bacteriophages use bacterial DNA to replicate themselves. Can enough DNA be extracted after lysis by bacteriophages?<P><B>07/05/18</B></P>
→ Optic Density monitoring and realization of a growth curve.
→ Amplification of T5 in a preculture of 5mL of E. coli F in LB medium (starting with DO = 0.3)<P><B>07/06/18</B></P>
→ Preparation of solutions for the bacteriophage purification:
            	- NaCl 5M
            	- PEG (20%)/NaCl (2,5M)<P><B>07/09/18</B></P>
→ Attempt to purify.
→ Meeting<P><B>07/10/18</B></P>
→ Continuation of the purification → centrifugation problem ... we use eppendorfs but no pellet is visible ... we consider that there is one anyway and we continue the purification.<P><B>07/11/18</B></P>
Still no pellet. We will titrate it anyway with PEG / NaCl
→ Spot titration of this test.<P><B>07/12/18</B></P>
→ Spot titration results : they are many bacteriophages (arround  10<SUP>10</SUP>pfu/mL)
→ E. coli culture to try an amplification in bigger volumes. 
→ E.coli DNA extraction to test and improve the extraction protocol (used with Lysis Buffer).<P><B>07/13/18</B></P>
→ No water in the lab !!
→ Precipitation attempt (longer centrifugations) → nothing at all (no pellet) !<P><B>07/16/18</B></P>
→ Different purification tests (variation of reagent concentrations and volumes, etc.)<P><B>07/17/18</B></P>
→ First results → pellets of bacteriophages for some tests. <P><B>07/18/18</B></P>
→ End of purification
→ Spot titration of this results.<P><B>07/19/18</B></P>
→ Spot titration results: one of the samples have a titer of 10^15 pfu/mL and one other have a titer greater than 10<SUP>15</SUP> pfu/mL.
→ E.coli F DNA extraction after lysis by T5 bacteriophages.
→ Meeting<P><B>07/20/18</B></P>
→ End of the extraction → It seems correct !! (more than 250ng/µL) (in 4h).
 
→ Cecile’s Holidays from August 21 to 25.<P><B>07/26/18</B></P>
→ Launch of a little amplification.<P><B>07/27/18</B></P>
→ End of the little amplification → OD has increased! Bacteria have recovered!
→ Cultures for bigger amplifications (different dilutions of precultures) --&gt; we pellet them to store them easily in a fridge.<P><B>07/30/18</B></P>
→ Culture was resuspended. Launch of bigger amplifications (OD monitoring). 
→ Petri dish stock<P><B>07/31/18</B></P>
→ PEG Purification (1/2)
→ Meeting
→ BBQ</DIV><DIV class="collapse slide"><H2><FONT color="#9c5899"><FONT size="5">August</FONT></FONT></H2><P><B>08/01/18</B></P>
→ PEG Purification (2/2)
→ Spot titration<P><B>08/02/18</B></P>
→ Spot titration results →  2 samples with a titer higher than 10^20 pfu/mL (new spot titration is necessary for them with higher dilutions)! One sample with a titer of 10^19 pfu/ml → it will be used for the DNA extraction of E. coli. 
→ DNA extraction of E. coli using different concentrations of bacteriophages. <P><B>08/03/18</B></P>
→ End of the extraction → The extraction by higher concentration (10<SUP>17</SUP> pfu/mL) leads to better results than 10<SUP>12</SUP> pfu/mL. The best DNA concentration is obtained after 2 to 6 hours of incubation (T5 bacteriophages with bacteria). <P><B>08/06/18</B></P>
→ Call with Javier (BioMérieux contact) to consider the possibilities open to us to manipulate Pseudomonas a. with HER18 bacteriophages (in class 2 laboratory).
→ Spot titration
→ Preparation of new stock:
                           	- SM Buffer
                           	- PEG (20%)/NaCl(2,5M)
                           	- NaCl 5M
Oops: The autoclave is not available ...<P><B>08/07/18</B></P>
→ Spot titration results → Nothing at all! I need to try again ...
→ Spot titration<P><B>08/08/18</B></P>
→ Appointment at BioMérieux
→ Spot titration results → still not as good as desired ... <P><B>08/09/18</B></P>
→ New attempt of spot titration (vortex between each dilution)
→ Pseudomonas a. PAO1 receipt<P><B>08/10/18</B></P>
→ Spot titration results:  one sample with 10^29 pfu/mL !!<P><B>08/13/18</B></P>
→ HER18 bacteriophages receipt<P><B>08/14/18</B></P>
→ Risk Prevention Plan at BioMérieux<P><B>08/15/18</B></P>
→ Meeting<P><B>08/16/18</B></P>
→ DNA extraction of E.coli with 3 different concentrations of T5. → Best results with the use of a titer of 10<SUP>27</SUP> pfu/mL (the higher titer), and after around 4 hours of incubation (bacteriophages and bacteria) → We have to repeat this test with more precision concerning the time :  we could “tighten” the extractions around 4 hours (after 3h-3h30-4h-4h30-5h-5h30 for example).<P><B>08/17/18</B></P>
→ Preparation of Top Agar stock and sterilization of our products with a pressure cooker.<P><B>08/20/18</B></P>
→ Shipment of T5 Bacteriophages to Munich iGEM team.
→ Start of BioMérieux experiments! →  installation of equipment in the laboratory and planification of the manipulations. Launch of a Pseudomonas a. PAO1 overnight preculture.<P><B>08/21/18</B></P>
→ Fast culture of PAO1 used for the amplification of bacteriophages HER18 (OD monitoring).<P><B>08/22/18</B></P>
→ PEG/NaCl purification (1/2)
→ Overday culture of PAO1 and centrifugation to store the pellet at 4°C overnight.
→ Interview with Pr Olivier Epaulard
→ Meeting (communication)<P><B>08/23/18</B></P>
→ PEG/NaCl purification (2/2)
→ Overday amplification of HER18 bacteriophages with bacterial resuspension of overday culture  (OD monitoring).
→ Spot titration and specificity tests (with T5 and PAK-P3 bacteriophages)<P><B>08/24/18</B></P>
→ PEG/NaCl purification of overday amplification (1/2)
→ Spot titration and specificity tests results → some samples have a titer higher than 10^30 pfu/mL  (new spot titration with higher dilutions is necessary). Only HER18 bacteriophages can lyse PAO1 (no plate with other phages).
→ Bacteria E. coli F shipment to Munich team.<P><B>08/27/18</B></P>
→ PEG/NaCl purification of overday amplification (2/2)
→ Spot titration of  phages from “fast culture” and overday culture (&gt;10^30 pfu/mL). <P><B>08/28/18</B></P>
→ Spot titration results → phages from “fast culture” have a higher titer (10^35 pfu/mL) than the bacteriophages from overday culture.
→ Interview with Pr. Max Maurin<P><B>08/29/18</B></P>
→ DNA extraction of  Pseudomonas aeruginosa PAO1 after lysis by bacteriophages HER18 → Good results! Once again, the use of higher bacteriophage titer leads to better results (higher DNA concentration - in a shorter time -).<P><B>08/30/18</B></P>
→ Meeting with Pierre and Audrey</DIV><H2><FONT size="6"><CENTER>Conservation (Perrine)</CENTER></FONT></H2><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">June</FONT></FONT></H2>
From the first june 2018 to the 26th of this month, I mainly carried out litterature and I helped for experiments on biobricks construction. 
I had to design the freeze-drying and drying protocols, I made some researches and bioinformatics to design a plasmid to overexpress trehalose in DH5α and the primers for PCR to create the biobrick using Gibson Assembly method. I also had to look for a laboratory lyophilizer which worked. <P><B>06/26/2018</B></P>

→ EcoRI-HF drying with trehalose<P><B>06/27/2018</B></P>

→ End of EcoRI-HF drying with trehalose for half of the samples, then stored in a 4°C fridge or at room temperature<P><B>06/28/2018</B></P>

→ End of EcoRI-HF drying with trehalose for the other half of the samples, then stored in a 4°C fridge or at room temperature<P><B>06/28/2018</B></P>

→ End of EcoRI-HF drying with trehalose for the other half of the samples, then stored in a 4°C fridge or at room temperature
→ Digestion of an unknown plasmid (thought to be pSB1C3-BBa_K292015 but the digestion rejected this hypothesis) with dried EcoRI-HF
→ Electrophoresis on agarose gel to check digestion of the plasmid with dried enzymes: good results, dried enzymes digestions with the same pattern as positive controls with enzymes conserved at -20°C digestion</DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">July</FONT></FONT></H2><P><B>07/02/2018</B></P>

→ DH5alpha Genomic DNA extraction with a Nucleospin Tissue kit from Macherey Nagel
→ IDT order of the primers for promoter, terminator and OtsBA gene PCR amplification<P><B>07/03/2018</B></P>

With the 2018 distribution kit 

→ pSB4A5-BBa_J04450 transformation into DH5alpha<P><B>07/04/2018</B></P>

With the 2018 distribution kit 

→ pSB1C3-BBa_I0500 and pSB1C3-BBa_B0015 transformations into DH5alpha
→ Bacterial transformation (07/03/2018) results: Enough colonies on plates
→ Petri dishes
→ Verification of DNA concentration obtained after Genomic DNA extraction (07/02/2018): low concentration probably due to the low amount of bacteria used, purity not really good.<P><B>07/05/2018</B></P>


→ Bacterial transformation (07/04/2018) results: Enough colonies on plates
→ Precultures of Dh5alpha transformed with pSB1C3-BBa_I0500 or pSB1C3-BBa_B0015
→ PCR amplification of OtsBA gene from DH5alpha genomic DNA (07/02/2018)
→ Agarose gel electrophoresis as a verification of PCR amplification: strange results, we did not see a band corresponding to the gene<P><B>07/06/2018</B></P>


→ DNA mini prep from 07/05 precultures to extract pSB1C3-BBa_I0500 and pSB1C3-BBa_B0015 : good concentrations, some contamination with proteins for pSB1C3-BBa_B0015
→ PCR amplification of OtsBA gene from DH5alpha genomic DNA (07/02/2018)
→ Agarose gel electrophoresis as a verification of OtsBA gene PCR amplification: strange results, we did not see a band corresponding to the gene

→ PCR amplification of BBa_B0015 from pSB1C3-BBa_B0015 extracted with mini prep 
→ Agarose gel electrophoresis as a verification of BBa_B0015 PCR amplification: strange results, two bands for the expected size
→ PCR amplification of BBa_I0500 from pSB1C3-BBa_I0500 extracted with mini prep 
→ Agarose gel electrophoresis as a verification of BBa_I0500 PCR amplification: strange results, a band with a size which corresponds to the part size, but this band is also present on the negative control, which means it can’t be our amplified part.<P><B>07/09/2018</B></P>

→ pSB1C3-BBa_I0500 and pSB1C3-BBa_B0015 double digestion with SpeI and XbaI
→ PCR amplifications of BBa_I0500 from digested pSB1C3-BBa_I0500 and BBa_B0015 from digested pSB1C3-BBa_B0015 
→ Agarose gel electrophoresis as a verification of PCR amplifications: not good results, except for BBa_I0500 which seems to have been amplified
→ Precultures and Culture of DH5alpha transformed with pSB1C3-BBa_J04450 or pSB4A5-BBa_J04450 to get large amounts for DNA drying experiment and as a plasmid backbone for trehalose respectively
→ Overnight lysis of DH5alpha E. Coli <P><B>07/10/2018</B></P>

→ Genomic DNA extraction from lysed DH5alpha E. Coli: low concentration of DNA but good purity
→ Midi prep from the cultures of DH5alpha transformed with pSB1C3-BBa_J04450 or pSB4A5-BBa_J04450 (07/09/2018): complete loss of DNA during extraction for pSB4A5-BBa_J04450, good concentration and purity for pSB1C3-BBa_J04450<P><B>07/11/2018</B></P>

→ pSB1C3-BBa_B0015 and pSB1C-BBa_I0500 enzymatic digestions with XbaI and SpeI
→ OtsBA PCR amplification from DH5alpha genomic DNA (07/10/2018)
→ Agarose gel verification of OtsBA PCR amplification: not readable gel
→ PCR amplification of BBa_I0500 and BBa_B0015
→ Preculture of pSB4A5-BBa_J04450 and pSB1C3-BBa_B0015<P><B>07/12/2018</B></P>

→ Mini prep of pSB4A5-BBa_J04450 and pSB1C3-BBa_B0015 : good concentrations en purity
→ Agarose gel verification of 07/11 PCR amplifications of BBa_I0500 and BBa_B0015: not readable gel (hypothesis: not enough Gel red for revelation)
→ Colony PCR from DH5alpha with two different annealing temperatures: not good results<P><B>07/13/2018</B></P>

→ no experiments<P><B>From 07/14/2018 to 07/29/2018</B></P>

Holidays!<P><B>07/30/2018</B></P>

→ Digestion of an unknown plasmid (thought to be pSB1C3-BBa_K292015 but the digestion rejected this hypothesis) with dried EcoRI-HF (06/27/2018) 
→ Electrophoresis on agarose gel to check digestion of the plasmid with dried enzymes: good results, dried enzymes digestions with the same pattern as positive controls with enzymes conserved at -20°C digestion, even 34 days after drying!
→ Preculture of untransformed E. Coli DH5alpha <P><B>07/31/2018</B></P>

→ Culture of untransformed E. Coli DH5alpha (07/30/2018)
→ Competency protocol on untransformed E. Coli DH5alpha</DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">August</FONT></FONT></H2><P><B>08/01/2018</B></P>

→ Transformation of pSB1C3-BBa_J04450 into the made-competent E. Coli DH5alpha (07/31/2018): to check if the bacteria were efficiently made competent, do a fluorescence kinetics, compare transformation with the Peltier module and with ice, and test transformation with different concentrations of plasmid DNA
Not good results for kinetics, nice results for bacteria plated on LB-agar-chloramphenicol, better for transformation on ice than transformation with Peltier module<P><B>08/06/2018</B></P>

→ Preculture and culture of pSB1C3-BBa_J04450 with our BBa_K2629001 biobrick inserted and pSB4A5-BBa_J04450
→ Sucrose and Trehalose stock preparation
→ RsaI, HaeIII and FspI overnight drying with trehalose<P><B>08/07/2018</B></P>

→ Midi prep for extraction of pSB1C3-BBa_J04450-BBa_K2629001 and pSB4A5-BBa_J04450: complete loss of DNA during isopropanol precipitation despite a huge amount of DNA after the first extraction
→ Meeting with a person that could help us with the wiki coding
→ Preculture of untransformed DH5alpha for competency and freeze-drying protocol<P><B>08/08/2018</B></P>

→ Culture of untransformed DH5alpha 
→ Competency protocol on DH5alpha, samples preparation for freeze-drying
→ Beginning of lyophilization protocol with DH5alpha protected with sucrose of trehalose<P><B>08/09/2018</B></P>

→ Freeze-drying fail: the process did not work during the night, leading to the complete defrosting of bacteria
New try of snap-freezing followed by freeze-drying with the defrosted bacteria<P><B>08/10/2018</B></P>

→ Freeze-drying process end<P><B>08/13/2018</B></P>

→ Rehydration of freeze-dried bacteria and transformation with pSB1C3-BBa_J04450 despite we reckoned they all died during the unsuccessful first lyophilization
→ Liquid culture try of non-transformed rehydrated DH5alpha to see if they survived the double lyophilization: no visible growth in three hours compared with the same samples frozen instead of freeze-dried
→ Preculture and culture of pSB1C3-BBa_J04450 with our BBa_K2629001 biobrick inserted and pSB4A5-BBa_J04450 for a new midi prep try<P><B>08/14/2018</B></P>

→ Freeze-dried bacteria transformation results: no colonies for all freeze-dried samples, lots of colonies for frozen samples, which confirmed the process failure led to the death of bacteria
→ Midi prep for extraction of pSB1C3-BBa_J04450-BBa_K2629001 and pSB4A5-BBa_J04450 : really high concentration, good purity
→ Preculture of untransformed TOP10 E. Coli <P><B>08/15/2018</B></P>

→ Culture from 08/14 TOP10 preculture
→ Competency protocol on TOP10 E. Coli (single protocol or double with each step repeated twice)
→ Freeze-drying of TOP10 E. Coli with sucrose and trehalose
→ Transformation to compare TOP10 made competent with single or double competency protocol<P><B>08/16/2018</B></P>

→  End of the lyophilization process
→ Results of transformation: no significant difference between TOP10 made competent one time or twice
→ DNA extraction by phages to help Cécile
→ pSB1C3-BBa_J04450 drying<P><B>08/20/2018</B></P>

→ pSB4A5-BBa_J04450 digestion with dried FspI
→ Resuspension of dried plasmid DNA (dried pSB1C3-BBa_J04450 from 08/16)
→ Rehydration of freeze-dried TOP10 from 08/15
→ Transformation of digested pSB4A5-BBa_J04450 in frozen TOP10 E. Coli stock
→ Transformation of resuspended dried pSB1C3-BBa_J04450 in frozen TOP10 E. Coli stock
→ Transformation of frozen pSB1C3-BBa_J04450 in rehydrated freeze-dried TOP10 E. Coli <P><B>08/21/2018</B></P>

→ Preculture of TOP10 for competency and lyophilization
→ Transformation results: good results for enzymes and DNA drying, nice results for bacteria lyophilization<P><B>08/22/2018</B></P>

→ TOP10 culture and competency protocol
→ TOP10 snap-freezing and freeze-drying
→ E. Coli F preculture for lyophilization (Collaboration with Munich)<P><B>08/23/2018</B></P>

→ Culture of E. Coli F and freeze-drying using sucrose and trehalose
→ End of TOP10 lyophilization process <P><B>08/24/2018</B></P>

→ End of E. Coli F freeze-drying process
→ Verification of E. Coli survival: rehydration and culture of one aliquot which told us they survived lyophilization
→ Transformation try with low bacteria volume (10μL to 20μL bacteria with 20ng to 50ng DNA)<P><B>08/25/2018</B></P>

→ Transformation with low volume verification: good results<P><B>08/27/2018</B></P>

→ Transformation of dried pSB1C3-BBa_J04450 in frozen TOP10 E. Coli stock
→ Transformation of frozen pSB1C3-BBa_J04450 in freeze-dried TOP10 E. Coli (08/23)
→ Preculture of TOP10<P><B>08/28/2018</B></P>

→ TOP10 culture and competency protocol for sensibility and specificity tests
→ Verification of transformations from 08/27: good results</DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">September</FONT></FONT></H2>
Going back to school !
In parallel of attending the courses (which started the third of September), we kept on experiments for two weeks to finish the detection part of the system. We then decided to stop experiments because we still had a lot of things to do before wiki freeze, and it was kind of hard to deal with both our iGEM project and our class. <P><B>09/04/2018</B></P>

→ Sensibility tests: hybridization of plasmid probes with the target and transformation in TOP10
→ TOP10 preculture<P><B>09/05/2018</B></P>

→ Verification of transformations from 09/04: not good results
→ Culture and competency protocol on TOP10
→ Freeze-drying of made-competent TOP10 to try transformation in the automated system<P><B>09/06/2018</B></P>

→ End of lyophilization
→ Fluorescence kinetics to characterize pSB1C3-BBa_J04450: transformation in E. Coli BL21, DH5alpha and TOP10 <P><B>09/07/2018</B></P>

→ FspI drying<P><B>09/11/2018</B></P>

→ Biobricks drying: BBa_K2629000 (which appeared not to be constructed with sequencing results), BBa_K2629001 and BBa_K2629003<P><B>09/12/2018</B></P>

→ End of biobricks drying</DIV><DIV style="padding:3px; padding-left:6px; border:5px solid #9c5899; margin-left:20px; background-color:#9c5899;" id="hardware"><CENTER><FONT id="optitl"><FONT size="10"><FONT color="white">HARDWARE NOTEBOOK</FONT></FONT></FONT></CENTER></DIV><H2><FONT size="6"><CENTER>Temperature control (Xandre)</CENTER></FONT></H2><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">May</FONT></FONT></H2><P><B>05/14/2018</B></P>
→  Summary of what has been done last week
→  sponsorship file finalisation 
→  Peltier unit: bibliography and protocols for future manipulations

To do: 
→ Tomorrow meeting with magIA (diagnostic with magnetic beads - catch DNA)
→ Sign conventions and bring to the TIMC + id + certificate
→ Print the sponsorship file
→ Get the material to experiment on the Peltier unit
→ Bring PC monitor and ex-iGEM team material
→ contact with Julien Travaux (Fablab)<P><B>05/15/2018</B></P>
→ conventions signed - mail to Mme Génin
→ meeting with Sarah from magIA on the subject of magnetic beads (see CR)
→ Experiment on the peltier - we have to find an adequate ventilation (see doc)

To do:
→ Have a look on how to heat up the system
→ look-up thermal conductors
→ theoretical part on peltier + model eventually (?)
→ continue to research on peltier<P><B>05/16/2018</B></P>
→ meeting with Biomerieux
→ Test of the radiator on the peltier ⇒ contact between eppendorf tube and peltier
→ study of iGEM 2017 temperature control<P><B>05/17/2018</B></P>
→ conventions signed
→ research of arduino code 
→ work on comsol modelisation for the magnetic beads
To do:
→ find the arduino code and work that friggin’ peltier
→ get a second pov for the magnetism<P><B>05/18/2018</B></P>

→ wiki peltier unit (find examples)
→ mail to Mr. Toussain about magnetic beads simulation on COMSOL (no answer yet)
→ page for the crowdfunding =&gt; asked for the logo
→ model for heat transfer in my tubes

To do:
→ list of arduino codes for dad
→ mail à Ruty - ok<P><B>05/22/2018</B></P>
→ meeting with Ruty to get more informations about the Peltier module
→ redaction of a notice, have to continue next time to find the part about the peltier (sensor ok)
→ banniere for the crowdfunding page
→ went to see Elsa Genin - will have to go back<P><B>05/23/2018</B></P>
→ peltier Montage  + arduino with LM335 all or nothing (it works !!!!!)

To do:
→ get a soldering iron
→ alim
→ heating resistance to find<P><B>05/24/2018</B></P>
→ test for 30 minutes of the regulation with the captor directly on the peltier =&gt; graph obtained
→ Will have to test if the water can attenuate the pumping phenomenon
→ beginning of the poster<P><B>05/25/2018</B></P>
→ posters (one scientific and the other more informative)
→ redaction of the wiki<P><B>28/05/2018</B></P>
→ test of other peltier modules and curve (not very conclusive)
→ poster
→ diapo for biomerieux
→ training for transformation manipulations with the girls (didn’t work except for Thomas)<P><B>05/29/2018</B></P>
→ test of the IR sensor (at the Peltier surface, in the solution with a temperature goal set to 2°C. Never obtained and 12°C - got some results). Remember that it was chosen for its precision cf datasheet 10° 

To do:
→ test of the temperature evolution of an eppendorf in ice to see what real temperature we will have to obtain
→ test a third peltier to put under to cool better
→ see influence of sensor position
→ see the evolution of the Peltier temperature simultaneously with the LM335<P><B>05/30/2018</B></P>
→ test of the temperature evolution of an eppendorf containing LB (instead of water). We want to observe if the solution is going to freeze or not in contact with the Peltier (to do again just on the Peltier to test)
→ temperature evolution for ice
→ test of the temperature at the surface of the Peltier simultaneously with the temperature of the LB solution.<P><B>05/31/2018</B></P>
→ spent the whole day making the poster and the flyer for the biotech event</DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">June</FONT></FONT></H2><P><B>06/01/2018</B></P>
→ mail sent to Quality teacher for a study on our product
→ new material for my module: peltier ref, copper tube, heating resistances
→ survey about what people think of our project<P><B>06/04/2018</B></P>
→ hygiene and health formation
→ call with B.Commere (meeting next wednesday) have homework to do (list of functions and dangers)<P><B>06/05/2018</B></P>
→ forum biotech (met with lot of people, gave the sponsorship file to sodipro)
→ tested the survey on people
→ meeting with biomerieux, visited the place (wanna work there !!)<P><B>06/06/2018</B></P>
→ started my end of internship report
→ test of a new set-up for the peltier (cardbox + copper)
→ test of the heating resistor
To do:
→ list for B.Commere
→ find out why the peltier montage didn’t work on the heating resistor
→ test an all or nothing loop on the resistor (set temperature to 70°C)
→ think about how we’re going to do with several solutions (one sensor for all or a sensor per tube ??)<P><B>06/07/2018</B></P>
→ tested the resistor, got some encouraging results - will have to test variations of voltage (between 10 and 12 V) to optimize the curves.
→ Meeting with the team - mail to pherecydes pharma - problems of communication with Vincent<P><B>06/08/2018</B></P>
→ work on the website
→ impression of the kicad card
→ work on the website hierarchy with Perrine<P><B>06/11/2018</B></P>
→ work on the AMDEC study
→ work on the pcb (pb it wasn’t contrasted enough on the pdf so the pcb wasn’t really good -&gt; didn’t work yet

To do:
→ see for the french american workshop, deadline 18 juin<P><B>06/12/2018</B></P>
→ printed again the pcb - worked this time (test tomorrow)
→ work on the AMDEC study
→ work on the avatars <P><B>06/13/2018</B></P>
→ finished study AMDEC<P><B>06/14/2018</B></P>
→ wiki on the AMDEC study
→ meeting with B.Commere - new contacts (how to organize a stand for a showroom)
to do:
→ prepare the meeting with Loic (https://www.ncbi.nlm.nih.gov/pubmed/20552933)
→ talk about event of next tuesday (remise des prix de l'innovation)<P><B>06/15/2018</B></P>
→ worked the pipette
→ work on the AMDEC study
→ preparation of the french-american workshop<P><B>06/18/2018</B></P>
→ work with Loic on the HP
→ started the design of the rotating part with paper, took the dimensions
→ meeting with M. Travaux from the fablab who gave us advice on how to shape the rotating part: imprint of the bolts and fix the resistances and the peltier under.<P><B>06/19/2018</B></P>
→ got copper tubes and heating resistors
→ designed on onshape the rotating part (duplicate for each step), 8 buffers and 4 pipette tips
→ fixed a rod on the radiator to fix it to the rotating part
→ 16h-20h: inotrophes - met with pepite osez people - sent a mail. Wil see what happens  next
to do:
→ See if the wood is going to burn in contact with the heating resistor<P><B>06/20/2018</B></P>
→ test of two serial peltier to see what the evolution of temperature is and if the current is enough to cool down both peltiers (this device works in current so putting them “in series” should be ok - they have the same) - however, the resistors work in voltage - we should put them in parallel
→ Everything went to shit (peltier down) - found a new one
→ Back on a hole plaque - it works - will have to test several peltiers in cascade and the intensity to deliver<P><B>06/21/2018</B></P>
→ got an alim that can deliver 10A
→ wood piece for the prototype were impressed and i tried to put the copper tubes in it but the cooling part isn’t fitting -&gt; will have to file
→ test of the peltier with more intensity but the transistor is getting hot very easily so I took a MOS transistor that has the property to support 60A.<P><B>06/22/2018</B></P>
→ file done, we assembled the different parts of the prototype (rotating part, tubes, resistors, peltier and radiator) and put  rode above to see if the rotation is precise (see video).
→ The plate isn’t well equilibrated and the copper tubes need to be improved to fit and support better the wood plate containing the eppendorf tubes.
→ Meeting with the rest of the team to talk about the pherecydes pharma issue (they want to register a patent concerning the use of their phages in our machine)<P><B>06/25/2018</B></P>
→ test of new radiators and fans to cool the peltier
→ meeting with biomérieux: new set-up for heating and cooling to reflect on. Extreme loss of conductivity due to the air with the current set-up (copper tube and nut)
→ Next prototype will be made  in PMMA (T° fusion =  130°C ok for the range we have but we will have to avoid putting the heating resistors directly in contact with the PMMA)<P><B>06/26/2018</B></P>
→ test of the new resistor under 12V (a bit less powerful than the yellow one) but it can go to 70°C
→ Measuring the temperature of aluminum (very reflective material) directly at its surface with an IR sensor is impossible! The temperature given by the sensor is about 30°C different sometimes!
to do:
→ theoretical study on the diffusion along the aluminum piece (on COMSOL maybe)<P><B>06/27/2018</B></P>
→ work on the report and a Gantt diagram<P><B>06/28/2018</B></P>
→ work on the Gantt diagram
→ undid a printer to get pieces from it<P><B>06/29/2018</B></P>
→ work on the Gantt diagram
→ test with a thermal imaging camera (wee really can see the difference of temperature from the hot and cold side of the peltier and the progression of cold diffusion in the copper tube)</DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">July</FONT></FONT></H2><P><B>07/02/2018</B></P>
→ work on the power supply stage
→ got the aluminium block at last ! The heating part works (? further tests needed for the speed)
→ photos with the team<P><B>07/03/2018</B></P>
→ work on the power supply stage (got  the +/-12 V alim), test of the 7805
→ CAD of the piece with the peltier and print next week<P><B>07/04/2018</B></P>
→ test of the watercooling
→ saw M.Ruty for the alimentation - few components to buy to obtain -12V/5V etc
→ alim delivering 10A and 12V from M.Ruty

To do:
→ hole in the rotating part to see if the cables won’t be too annoying to deal with
→ buy my components
→ replace watercooling tubes<P><B>07/05/2018</B></P>
→ command of components done
→ assembly of almost every modules and video (heating, pipette, magnet, rotation)
→ Rq: arduino mega (i2c ports different than on the classic arduino)
→ http://fr.farnell.com/c/optoelectronique-afficheurs/optocoupleurs/optocoupleurs-sortie-transistor?nbre-de-voies=4-voies look for optocoupleur 4 voies)
http://fr.farnell.com/vishay/cny74-4h/optocoupleur-quad-sortie-transistor/dp/1045432
http://fr.farnell.com/vishay/ilq74/optocoupleur-quad/dp/1045424
→ Note for Lucas concerning the HP<P><B>07/16/2018</B></P>
→ update on what happened during my holidays
→ have to find an alternative to the watercooling because it’s too impractical and voluminous.
→ we found a cooling system on a PC - test aren’t very conclusive - we have to carve the aluminum to better fit the peltier module
→ meeting with Loic about the Human Practices - progress - antibiotic resistance ok - video to make - next is the questionnaire<P><B>07/17/2018</B></P>
→ work on the alimentation block
→ “connecteur d’alimentation dc femelle” 
http://www.farnell.com/datasheets/1910033.pdf?_ga=2.128002828.1629805132.1531811727-853834444.1526996106&amp;_gac=1.221859690.1531811727.CjwKCAjwp7baBRBIEiwAPtjwxNcEeQGABwvYjknxpencGIyEQCoizKVB9dJTOAejVPZQOg-WQGWJPxoCO1YQAvD_BwE
center +12V, side &amp; behind = gnd
→ soldered it
→ test of the LT323A -&gt; works
→ FDD15-05S3 wasn’t the good component :( error of command - input current too high
→ test of traco power componant - can’t make it work for the moment<P><B>07/18/2018</B></P>
→ test of a new cooling system (found new radiators and fans in old computers
→ tracopower is working (test with a tl082)<P><B>07/19/2018</B></P>
→ day we moved from Phelma to JR<P><B>07/20/2018</B></P>
→ installation and tour de france<P><B>07/23/2018</B></P>
→ got back to Phelma to retrieve the last pieces (aluminum and 3D printed)
→ with the transport, my temperature regulation doesn’t work anymore and I don’t know why… <P><B>07/24/2018</B></P>
→ glued the magnets to the 3D printed piece because they keep attracting each other and can’t stay in place otherwise
→ tested with the actuator and the arduino, seems to work well, we will have to test how to put it in the final prototype<P><B>07/25/2018</B></P>
→ obviously, with the current system, I will not be able to reach 4 degrees so the other solution is to make research on the influence of the cold step on transformations (cf papers from Clement) 
→ no aluminum because it gets useless when saturated with heat =&gt; not evacuated
→ alimentation seems to work with to fans and two peltiers at the same time
→ test of transformations with a variation of different parameters<P><B>07/26/2018</B></P>
→ success on real-life transformation with my system!
→ can’t make the mosfet work, will have to ask the rest of the team to help
→ work on the aluminum pieces to shape them to fit together
→ went to Biomerieux to get the rest of the aluminum pieces<P><B>07/30/2018</B></P>
→ Discussion with Clement about an experiment testing the purification module with different concentrations of DNA
→ module is working anew, tests are possible
→ pre-culture<P><B>07/31/2018</B></P>
→ protocole of competency for the bacteria</DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">August</FONT></FONT></H2><P><B>08/01/2018</B></P>
→ transformations
→ results on petri dishes and kinetic measurement<P><B>08/02/2018</B></P>
→ count of bacteria colonies and interpretations =&gt; validates the module but it isn’t optimally working
→ failure of the cinetic (no fluo appeared)<P><B>08/03/2018</B></P>
→ kicad modelisation of the whole system
→ test to go to 70°C but failure (will have to see if the new resistors arrive)<P><B>08/06/2018</B></P>
→ wiki all day looong
→ reflexion on the parameters we could optimize in the whole machine<P><B>08/07/2018</B></P>
→ new protocole to test
→ traductions
→ wiki<P><B>08/08/2018</B></P>
→ new day of experimentation on the transformation (test iptg)<P><B>08/09/2018</B></P>
→ data analysis
→ traduction<P><B>08/13/2018</B></P>
→ test of top10 bacteria to see if they emit fluorescence sooner (it’s the case) will have to test the evolution in a tube of 0.5mL → real conditions of the system<P><B>08/14/2018</B></P>
→ competency protocol for top10
→ traduction &amp; wiki<P><B>08/15/2018</B></P>
→ traduction &amp; wiki<P><B>08/16/2018</B></P>
→ pcb for the alimentation system
→ traduction &amp; wiki<P><B>08/17/2018</B></P>
→ pcb rooting
→ traduction &amp; wiki</DIV><H2><FONT size="6"><CENTER>Global structure and purification module (Thomas)</CENTER></FONT></H2><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">May</FONT></FONT></H2><P><B>05/09/2018</B></P> 
→ crowdfunding web page
→ research on protocol for purification, especially magnetic beads
→ in depth description of the specifications of kit for biology process
+design of kit changed accordingly
→ beginning of 3D CAD (on OnShape)<P><B>05/10/2018</B></P> 
→ continue 3D CAD 
→ in depth research magnetic beans purification system + contact with MagIA (a Grenoble companie using magnetic beans)<P><B>05/11/2018</B></P> 
→ continued 3D CAD<P><B>05/12/2018</B></P> 
→ looked for different type of pipette, electronic type chosed for now
→ 3D CAD

<P><B>05/14/2018</B></P> 
→ research on heating and cooling devices
→ 1st real modelisation of our sample plate &amp; association with rotative module (+heating and cooling modules)<P><B>05/15/2018</B></P> 
→ meeting with MagIA CEO =&gt; discussion on magnetic beans and DNA purification
→ first look at the motor for rotative (central) plate : step motor needed, 1st look at how to make it work (H bridge needed for tests)<P><B>05/16/2018</B></P> 
→ meeting with Biomérieux
→ 1st electronic schematic for motor<P><B>05/17/2018</B></P> 
→ control of the motor with arduino
→ beginning of the modelisation for the magnets (DNA purification) on comsol<P><B>05/18/2018</B></P> 
→ step motor with h bridge ⇒ it works
→ 1st modelization of the magnetic gradient that one can obtain using COMSOL
→ meeting with Pierre Schefler =&gt; general overview of the project<P><B>05/22/2018</B></P> 
→ 3D modelisation of magnetic gradient using COMSOL
→ research about electromagnet (instead of permanent one), pb : weak forces, hard to find very small solenoïds with good shapes.<P><B>05/23/2018</B></P> 
→ modelisation of a gradient using 2 solenoids with matlab
→ creation of a homemade electromagnet
→ helped on the temperature measure with Arduino<P><B>05/24/2018</B></P> 
→ construction of 2 solenoids, it made ferromagnetic dust move!
→ meeting  with the instuctors
→ new idea for the sample holder: separate it in 4 parts: easier to clip on the rotative part (thanks Perrine for the idea !)<P><B>05/25/2018</B></P> 
→ meet with Julien Traveaux: 1st look on our modelisation
→ work on homemade electromagnets<P><B>05/28/2018</B></P> 
→ first bacterial transformation ! (with the help of the biologist !)
→ work on a presentation of the project for BioMérieux<P><B>05/29/2018</B></P> 
→ finished handmade electromagnet : good size, good shape
→ dust was moved<P><B>05/30/2018</B></P> 
→ test of magnet beads capture with homemade electromagnet ⇒ not enough, see results
→ use permanent magnet instead OR use hybrid solenoïd/ferrite/permanent magnet</DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">June</FONT></FONT></H2><P><B>06/04/2018</B></P> 
→ work on stepper motor
→ went to the fondation UGA our team’s poster + flyers for the Biotech Forum <P><B>06/05/2018</B></P> 
→ Meeting with Biomérieux<P><B>06/06/2018</B></P> 
→ day off<P><B>06/07/2018</B></P> 
→ magnetic beads from Biomérieux with permanent magnets: ok
→ linear actuator: ok
→ used a cardboard for a first prototype of the rotative plate: bad coupling with the stepper: it misses a few steps from times to times + bad movement from the stepper<P><B>06/08/2018</B></P> 
→ stepper motor: I tried without driver ⇒ pb with backward movements from times to times
⇒ troubleshooting: bad wiring, I will have to test the driver again now<P><B>06/11/2018</B></P>
 →  stepper motor with driver: ok for micro stepping
 → coupler for the stepper motor: printed in 3D, working
next step: use a magnetic or optical switch for position control of the rotative plate<P><B>06/12/2018</B></P>
→ in-depth look of the protocol, in order to know the different buffers to use inside our device (done with the help of Perrine)
→ presentation of our project in front of the Pharmacy University commission<P><B>06/13/2018</B></P>
→ contact Pherecides Pharma in order to have a phone call with them
→ + we prepared a small pitch to give them if we can
→ work on a servo motor in order to bring the permanent magnet in close range of the tubes (containing magnetic beads)
→ Biomérieux gave us second-hand electronic pipette<P><B>06/14/2018</B></P>
→ mail &amp; first contact to Avalun : they are a company using lyophilisation
→ made the electronic pipette work with the arduino : alimentation connected, one button connected &amp; working so far !<P><B>06/15/2018</B></P>
→ meeting with the CTO of Avalun: they got an industrial lyophilization, difficult for us to use without proper training
→ we connected all the buttons of the pipette to our Arduino<P><B>06/18/2018</B></P>
→ I chose and bought a few electronic part: magnetic interrupter for a position calibration, a small servo motor for the magnets<P><B>06/19/2018</B></P>
→ I looked at how to move the pipette up and down, I also looked at where to buy parts for such a linear stage, and where to find it
→ helped Jean on the work with the electronic pipette<P><B>06/20/2018</B></P>
→ 3D modelling of the electronic pipette clamp
→ we borrowed the linear stage of the fablab (motor + moving parts + holding parts), made it work<P><B>06/21/2018</B></P>
→ video conference with Pherecydes Pharma
→ 3D printing and assembly of the pipette clamp (to hold it to the linear stage)<P><B>06/22/2018</B></P>
→ meeting with all the team members
→ helped Xandre on assembling prototype_v0 and moving it with stepper motor + Arduino
→ Gathering information about the patenting process of biologic product-specific use.<P><B>06/25/2018</B></P>
→ Work on the first prototype v0: started on the code to make the pipette move on one axis (bottom-up)
→ also worked on the magnets: the servo can now bring the magnets close to the tubes, or far away.
→ made some videos of the prototype
→ meeting with engineers of Biomérieux: they validate a few things and gave us some new ideas<P><B>06/26/2018</B></P>
→ work on making everything works after each other: now the plate turns, then the pipette goes down, then the liquid is pipetted out etc. : lots of things to optimize, but the basic functions are there!
→ I wrote down the project description that will be published on the 29 of June.<P><B>06/27/2018</B></P>
→ work on putting everything together : added magnetic switch for position calibration.
→ problem: system not able to do small movement in a reliable way<P><B>06/28/2018</B></P>
→ lots of work to do on the mechanical system in order to make it work
→  work on a structure to hold the weight, for now, the weight is held by the motor, which cannot turn the structure in a reliable way<P><B>06/29/2018</B></P>
→ 1st try with 3 rolling balls on a plate in order to withstand the weight of the structure</DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">July</FONT></FONT></H2><P><B>07/02/2018 </B></P>
→ things starts to work on the mechanical aspect, but there is still no way to obtain reliable (very) small movement<P><B>07/03/2018 </B></P>
→ I changed the coupler of the system (the part doing the transmission of the movement btw the motor and the plate), the previous one was in plastic, without a security screw ⇒ replaced with a real coupler: works.<P><B>07/04/2018</B></P>
→ Worked on putting everything together: with the new coupler the rotative plate can do really small movements, meaning that we can now have a real step of calibration (for the position) of the rotative plate<P><B>07/05/2018</B></P>
→ mechanical part of the prototype complete ! everything works together!
→ I made a small video with all the parts connected together: calibration of the position of the pipette+rotative plate, temperature control via Arduino and IR sensor, pipetting with pipette controlled via arduino etc.
→ I will show this video in the Parisian meet’up<P><B>07/07/2018 </B></P>
→  Parisian Meet’up<P><B>07/09/2018 </B></P>
→ came back from Paris in the afternoon, then spent time with Elise (our lead biologist) : I explained her the basics of engineering<P><B> 07/10/2018 </B></P>
→ work on the aluminum blocks (for thermal conductivity)
→ work on electromagnet for capture of Magbeads… doesn’t work, we will have to use permanent magnet<P><B>07/11/2018 </B></P>
→ work on a magnet holder in order to slide in the magnet in a specific aluminum block (one with special hole for it)<P><B> 07/12/2018 </B></P>
→ slides </DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">August</FONT></FONT></H2><P><B>08/06/2018</B></P>

→ got up to date after my vacations
→ work on the design of our plates (the ones for the frame of our device) in 3D<P><B>08/07/2018</B></P>
→ more work on the plates, we will use plexiglass (PMMA) and the laser cutter of a nearby (and still open) FABLAB to create them
→ work on the phage-bacteria interaction modelization ( &amp; discussion with Ruthie from Nottingham)<P><B>08/08/2018</B></P>
→ more work on the modelization (I actually got some interesting results now)
→ Jean &amp; Lucas came back from the FABLAB with our plates, I started mounting them<P><B>08/09/2018</B></P>
→ finished mounting the prototype with the others
→ started to see all the problems with the plate I designed: a lot of rework will have to be done, but with the actual plate under my eyes, it’s quite easy to see them
→ work with the raspberry pi<P><B>08/10/2018</B></P>
→ [I took the morning as a half day off]
→ logistic : made some orders (mechanical parts, touchscreen for rpi) + validation of some other orders (conectics)<P><B>08/13/2018</B></P>
→ redesign of the plates
→ work on the rpi : we will use the kivy library (python) to create the touchscreen gui, some problem with the install though<P><B>08/14/2018</B></P>
→ went to the FABLAB with Jean, we spend the day there using the laser cutter on plexiglass and wood</DIV><H2><FONT size="6"><CENTER>Fluorescence module (Charles)</CENTER></FONT></H2><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">May</FONT></FONT></H2><P><B>05/07/2018 </B></P>

Beginning at 9 am. Organisation of the poles (engineer side). Thought about how to pipette and extract the DNA of interest. Writing a section for “Les trophées de la santé”, it still needs to be completed, especially the financial and repeatability parts. 

To do: wrap this file up, working on phage/bacterium interactions (especially if lytic or lysogenic, needs to go through the R algorithm), rejuvenate the Trello.<P><B>05/08/2018 </B></P>
“Trophée de la santé” wrapped up. Research on phage/bacterium interaction; from the ultra-specific PA phages list, are they lytic or lysogenic? 1st look at the Random forest algorithm, but I did not understand a lot of things. 
Research on fluorescence detection. We work with BFP and RFP (399/456 nm et 584/607 nm). The light source should have a spectrum that covers both excitation wavelengths, to be selected with a filter. A LASER is not a good idea as 2 fluo means 2 laser, which are not that small + a 400 nm LASER is expensive. Here is a schematic representation: 

A dichroic mirror should be placed under the plate to let the incident wavelength to go through to not lose fluo. Conversely to what is schemed, the optical fiber cannot be fixed to the dichroic mirror since the excitation light must be reflected on the mirror. Having this mirror under the plate letting the 395 nm light go through allows to measure the OD if necessary, taking the reflexion on the mirror in consideration. The mirror should be at infinity to focus light on the photomultiplier. 

This device should work, but I still need to think of the excitation light. We could use a mercury-vapour light but it seems its sell is illegal. I also have to think about its power. The geometry should also be considered, but it is hard to know what to do until I do the experiments. I should also have a closer look in the photomultiplier functioning. 


To do: keep working on fluo. Check the crowdfunding file. Write a recored for the Fondation INP. <P><B>05/09/2018 </B></P>
Review of the crowdfunding file for Fondation UGA. It is not finished yet, the communication part has to be written by Vincent. I then worked on the fluorescence unit, especially on the dichroic filter and the photomultiplier. My researches were not really productive, as I did not understand the data sheet: I sent an email to the company to get some explanations (about the window material and the transmission). Concerning the dichroic mirror, I need to do some experiments to make sure I correctly understand how it works, as it changes from a scheme to another. 
Yesterday I wanted a dichroic mirror under the tank, but it might not be necessary, a curved mirror should be enough, as the optic fiber should be located at the image focal point of the mirror. The “laser” will be reflected as well, but the dichroic mirror should deviate the light. If necessary, a filter could be used as well. 
Most importantly, I found how to get a precise wavelength with a white light. It can be done with a monochromator device, it is explained on the bibliography file (I need to go more through this part). It can be really useful as the wavelength can be selected by moving the cristal. 
For the automatisation part, a wheel in the (xy) plan seems to be a good idea. It keeps the pipette and the temperature units (Peltier unit and heating resistor) fixed. 

To do: keep working on the fluorescent unit. 2nd modelling. Onshape. <P><B>05/10/2018 </B></P>
1st approach on Onshape. I modelled a lens, a dichroic filter, a well and the photomultiplier. However, the previous apparatus imagined is not appropriated to the project. First, it is hard to select 2 wavelengths with a dichroic mirror. Moreover, a photomultiplier needs a high voltage alimentation (800 V at least !); it is dangerous enough to work with UV light, I will not risk an electrocution on top of that. 
The new system is presented in the figure below:

 The sample is alternatively lighted by the blue and red LEDs, so that the fluorescences do not overlap. The filters are coherently placed in front of the camera thanks to servomotors (in position OFF, the filter could be in front of the corresponding LEDs, in this case the scheme is wrong). 
The camera takes a picture of the fluorescence, therefore it will probably be measured as last year, with a greyscale. 

To do: Onshape of this new prototype, project report. <P><B>05/11/2018 </B></P>
5/11: Onshape of the fluorescent module. Mail about epifluorescence to Lionel Bastard.

<P><B>05/12/2018 </B></P>
Explored the iGEM’s latest development about fluorescence detection. Munich 2017 used a photoresistor. Cheap, to detect light variation, but highly sensitive to the environment, so cannot be applied if we’re to cold the well during the detection. 
I however found a light captor sensitive from 0.1 lux and quite stable. I need to determine if it will fit to the fluorescence luminance, so I bought it with an Arduino.
Onshape of a second fluorescence detection module. The LEDs are missing, I failed to draw them, but I expect them to be next to the corresponding filter. <P><B>05/14/2018 </B></P>
1st day at the Phelma’s remise! 

I found the PERFECT good LEDs and filters ! The LEDs’ spectra are really narrow, so the noisy light should be entirely filtered. However, the filters are to be tested, especially one which cutoff wavelength is at 605 nm, so very near from the maximum wavelength emission (607 nm). According to its characteristic curve everything should be fine but it still needs to be tested. 

From this day, the tests that I need to do are:
→ Influence of temperature on fluorescence
→ Influence of pH on fluorescence
→ Fluorescence intensity of BFP/RFP
→ Resolution test of the sensor; to be compared with a photoresistance
→ Tests of the LEDs &amp; filters<P><B>05/15/2018 </B></P>
I received my Arduino Uno ! After struggling a lot to plug my several wires, I finally succeeded to:
→ enlight successively 2 LEDs
→ enlight successively 2 series of 2 LEDs

That is the good part of the day. 

Concerning the bad part, the RFP biobrick might not be RFP, but a luminescent protein. The measurement must then be done by absorbance, which changes totally the structure of the module, as the light source is to be under the sample, but we do not have enough space. In a nutshell, this biobrick is a bioshit that I hope we won’t use. <P><B>05/16/2018 </B></P>
Meeting with Javier Yugueros Marcos (Biomérieux), cf report.
Having some fun with a stepper motor that does not rotate… What I and Thomas noticed is that the stabilised alimentation is limited in current. We expected to provide 2 A but we only have 0.04 A. The Arduino might be the reason but that would mean that alimentation is directly connected to the Arduino, that is to say that the H bridge is invisible in the circuit whereas it is a new one… However, for sure something in the circuit limits in current. 
Still working on it tomorrow. <P><B>05/24/2018 </B></P>
Alright I am a “little” late on this notebook, so I’ll try to summarize what I’ve done since 5/17. 

→ the motor finally worked, a little resistance had to be put off the circuit, as it did limit the current. 
→ I wrote my first weekly wiki, I am not sure that everything is useful for the website but whatever
→ The fluorescence unit changed. Quantitative analysis of the fluorescence requires an absorbance measurement to normalise fluorescence, as different concentration of bacteria might give the same fluorescence. So that means that a light detector should be placed under the sample with an excitation filter to get rid of fluorescence and only detect the transmitted light. However, according to Lucas, absorbance is relevant to be measured when light diffusion can be neglected which is not our case, so this is to be thought on. 
→ Anyway I found better filters for each case (fluorescence and absorbance) and better LEDs with high power. 
→ I wrote a few protocols about fluorescence and the impact of different parameters. I expect to work in the lab next week, probably on tuesday
→ We succeed with Thomas to create a magnetic field from a home-made coil, as we attracted a metallic screw. <P><B>05/25/2018 </B></P>
I worked on the magnetic gradient with Thomas. After hours of spooling, we succeeded to attract ferromagnetic dust between 2 coils, but we are not sure that it is due to the gradient, we are not even sure that there is a magnetic gradient. However Thomas took a pretty cool video! <P><B>05/28/2018 </B></P>
Received my light sensor. The related library needs to be modified, as it calculates the light intensity with integers, so the resolution is not 0.1 lux but 1 lux. Thomas did some changes, but it is not enough. Working on it tomorrow. 
1st biology experiments, Elise coached us on a transformation that I probably failed. We’ll see the results tomorrow. 
I was supposed to do my manip on fluorescence, but my culture did not grow well, so I did nothing. <P><B>05/29/2018 </B></P>
 reviewed my sensor, I managed to display float light intensity, with a 0.01 lux accuracy. This is not coherent with the sensor’s datasheet but it doesn’t matter: it just means that I don’t measure exactly a light intensity, but this is the case for all my measures. So I’ll just need to do some statistics and guess what bacteria population give what value. I reused the 2017’s kit to do some tests: I wanted to know if it could detect a LED light. I finally succeeded: the LED was emitting light in a non-optimal path, at almost 2 cm from the sensor, and I got 0.14lux. I tested later with the LED at 3 cm, and I got 0.04lux. I added fluorescent bacteria, excited by the LED. With a 200µL sample of pure bacteria, no changes could be noticed. With a 1-2 mL sample, I noticed a change of 0.2lux (0.3-0.4 lux to 0.6-0.7lux). It seems hard to detect fluorescence with the sensor, but the experimental conditions were not optimal (I don’t know what quantities of bacteria I had, ambient light noise), so further tests have to be done. 
I also did some fluorescence tests (cf protocol) with a solution containing transformed bacteria having expressed a RFP fluorescence.

Although I at first had to review my protocol, forgot to do the controls, struggled with the lector, I finally managed to do the protocols (I won’t perform the pH protocol I think as it is actually written in the database that the proteins are not sensitive to acidity). The data seem at first coherent (fluorescence decreases with dilution, with temperature), I still need to go into the values. This will be done tomorrow. <P><B>05/30/2018 </B></P>  
I analysed the data from my yesterday’s experiments. The conclusions are avalaible here: https://drive.google.com/open?id=1aiut2_jhZVGnPqKSf9THaB-Iu_z0pZb4 . To summarise, fluorescence is indeed proportional to the bacteria concentration, and temperature does not influence enough to vary it.  
I also began to learn HTML5 with OpenClassrooms. It is pretty easy for the moment, but that is only because I didn’t go into CSS3 yet. The most difficult part of HTML5 is, it seems, to address correctly the pictures/links etc. This is all about organisation, but I think it’ll be fine. 
Lionel Bastard came at the end of the day. It surprised him that I did not want a lens in my device, but 1 cm gap between the sensor and the sample eventually convinced him that it might be not necessary. I still follow the same strategy to not use a lens at first and see later if I really need one. Actually this will be largely determined by the solid angle: this is what will tell me if I chose wisely my sensor. He also advised me to use a monopixel sensor, or a photodiode. Another primordial point is to calculate the noise to signal ratio. To do so I’ll have to test my light source on a LB sample (no bacteria in it) and see “how much” light my sensor can detect. To improve this ratio, he advised me to add another emission filter, to improve the noise attenuation.  <P><B>05/31/2018 </B></P>
I found 1cm x 1cm photodiodes with good sensibility. I still need to test my current sensor to make sure it does not suit. This will be done tomorrow. 
I kept on learning HTML5/CSS3. </DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">June</FONT></FONT></H2><P><B>06/13/2018 </B></P>
Let’s summary everything that has been done since last time. 

→ I tested the sensor again: it really does not suit at all, it sucks. It cannot sense fluorescence light. Another sensor has to be found: a photodiode. I ordered one with a large sensor surface and a relatively good responsivity (I cannot compare it to the 1st sensor as its graph is normalised but I expect it to be better)
→ A drawback of the photodiode can be its dark current. The 1st idea was to measure beforehand the light power from fluorescence. It seems easy stated like that but it is actually not, as a light power is usually measured in lux instead of watts which corresponds to the responsivity curve of a photodiode. 
→ The previous issue should be solved with the conditioning circuit:

This circuit can be divided in 3 parts: a transimpedance circuit, the amplificator stage and the memory stage.

I) transimpedance circuit:


As I did not find any photodiodes in LTSpice, I replaced it with an ideal source current: this is in theory the same thing, the idea is just to get a current which is supposed relatively low (the values for the current do not correspond to the following tests)

The aime of this stage is to convert the very low current into a voltage. The output voltage of this stage is: 

As the current is very low, the resistance should be very high. A 400 MΩ resistance is usually expected for that kind of circuit but it is not an easy resistance to find, and I bet it is expensive. For the moment, let’s use a 4.7MΩ resistance: it is not as good as using a 400MΩ resistance as the signal will be more noisy.

II) Memory

The memory stage is the heart of the circuit. Its aim is to hold the voltage related to the dark current so that it can be removed from the experimental current at the amplification stage:



This graph corresponds to an initial current of 600 nA (dark current of the photodiode) which 10 pA is added from the fluorescence (let’s hope it will be greater). 
The voltage at the output of the 1st circuit is in blue and the voltage at the output of the memory stage is in green. We want the green voltage to be equal to the blue voltage during the first 60s (it is the dark current voltage). There is a little difference at first of a few µV so I think we can consider they are equal. When the photodiode is illuminated, which comes out with a greater current and then a voltage lower (this is indeed what we can see at 60s). 

The voltage is stored thanks to a S/H stage built with a capacity and a follower. After 60s, the switch is off, so the voltage from the fluorescence light is not stored in the capacity. However, as the capacitor is no longer supplied, it eventually unloads. A high capacity reduces the unload time: the previous graph is displayed with a 1µF capacity, but the voltage is pretty damn constant with a 100µF capacity:

It is quite important to take care of this stage: if its output voltage is not constant enough, or too far from VD, the output of the amplification stage will not correspond exactly to the voltage of the fluorescence light. 


III) Amplification

This is an amplification stage with a single differential OpAmp. I am not sure yet that we will use this stage exactly or a stage which would look more like the one we used for the BE (with a single resistance to set the gain). 

This is what we obtain at the output:

The variation is very little,  but once again, the current related to fluorescence is there equal to 10pA, it would be really disappointing: this would mean that the light’s power is equal to almost 0.2nW (responsivity of 0.2A/W). 
With a 601 nA current, the output is nearly equal to 1V. 


The entire circuit:

N.B.: a capacity might be placed in parallel of the transimpedance’s resistance. 
There is no filter stage yet: that is because I don’t know what to filter, this will be determined in the lab. 
The bias current of the transimpendance’s AOP must be very low (fA) → model in Chamilo
Very high impedance for the follower
Gain yet to be set as we don’t know the current generated from fluorescence. 
In itself, there are a lot of advice in Bucci’s lesson → must-read


→ Data will be eventually be acquired with an Arduino and sent to a raspberry pi that can monitor a touch-sensitive screen (cf bibliography)<P><B>06/14/2018 </B></P>
 BEGINNING OF THE FIFA WORLD CUP!!! 

Besides that, I tested the memory stage with a 100µF capacitor. There is an unload that is actually non neglectable (200mV/min at the beginning), which means a 3mV/s loss. I don’t know yet what current will give me fluorescence but I hope it will be higher than 3 mV, otherwise, I’ll have to find another way to hold the dark currentvoltage (maybe with the raspberry pi, but I haaaaaate this idea, it is so ugly). </DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">July</FONT></FONT></H2><P><B>07/11/2018 </B></P>
Otherwise, to sum up what I’ve done during the last month :
→ The conditioning circuit works on breadboards, the memory stage is pretty convincing as the capacity unloads 1mV/20s. Working on through-hole components, a polymer capacitor is what suits the best to a memory stage as its capacity can be really high (up to 100µF) but unlike chemical capacitors for instance, they have a high leakage resistance. Ideally, this resistance should have the same impedance as the AOP which is harshly the case with polymer capacitors. X7R capacitors have a much higher impedance, but they are smd components. I then need to work on smd. Another reason to work on smd is that my trans-impendance AOP can only be soldered smd. 
I tried to work on SMD but the circuit doesn’t work: I can’t solder cleanly, so it’s really hard to have something that is functional. Basically, soldering the components in the wrong way means that they must be unsoldered, which destroys the tracks for instance. Actually, the transimpedance and the memory stage eventually worked, but the PCB was so damaged that it was not possible to fix the substractor stage. 
As the circuit worked on breadboard, I tried to solder thru hole components on another PCB. Same issues as last time, it did not work. It is quite concerning as I need to test my photodiode to make sure it will detect fluorescence light. 
I eventually tested my photodiode on on breadboard as the circuit DOES work on it, but the tests didn’t go well as the working conditions were not good (no complete dark, photodiode not stabilised, eppendorf not placed correctly...). I have to 3D print the components that will constitute my module. 
An absorbance measure is not bound to be necessary. This needs to be confirmed by Fred, the fluo expert of Biomérieux. If it is not necessary, the MC	simulation, which is really cool, is not necessary. We’d juste need to calibrate beforehand the photodiode. It should be done this way:
→ We measure the output voltage with 3 solutions: 2 highly concentrated  and a blank, to get a straight line. 
→ Then, several measures of the blank are done : an average voltage is calculated
→ the minimum sensitivity is this average voltage added to 3 times the standard deviation. 
→ We finally complete the line with other concentrations. 

	This will be done when I have the circuit and the 3D pieces.

Finally… there is no 2 kinds of fluorescence anymore… The only fluorescence we will measure is from RFP. There is no BFP anymore because:
→ iGEM’s backbones are not adapted to BFP. 
→ BFP needs 10h of IPTG induction. 

	So there is a filter I don’t need, and I don’t need to change the filter, so the servo motor is useless. Actually it makes everything easier, except that the measures need now to be performed in 2 different pits. And it is quite disappointing, but well, this is how life goes. <P><B>07/31/2018 </B></P>
I have the 1st version of the unit. I’ve used it to do some experiments, that are summed up below:

→ I wanted to evaluate qualitatively how efficient my filter is. To do so, I compared the measured voltage at the trans-impedance stage with and without filter, and I illuminated quite approximately the eppendorf support with 7 LEDs. 8 LEDS were foreseen, but one hole was not correctly designed, so it couldn’t be used to make the wires go through it. By the way, all the holes are not correctly designed, as I can’t insert the LEDs into them. Therefore the LEDs are out of the hole, nearer to the eppendorf but with an uncertain orientation, and they are stabilized with scotch. 
Without filter, the photodiode saturated (4.60V); without filter, the photodiode did not saturate. The filter indeed filters the excitation light. 
→ I tried to measure fluorescence in an eppendorf with fluorescent bacteria. I observed that the voltage was a little lower with the eppendorf in place, which made me think that the bacteria were not fluorescent. A week of holidays later, I tried again this experiment, but this time I saw fluorescence at the fluorescence microscope, and the sample was quite concentrated. However, I saw the same phenomenon. I had 2 hypothesis (with the help of Clément): it could be the LEDs that are not bright enough, or the filter that is too selective. 
I used the UV plate to excitate a petri dish with fluorescent bacteria: the fluorescence can be seen to the naked eye, so the filter indeed lets the fluorescence light by. However, when trying to detect something with the photodiode, I am very near to the saturation value. Worst, this value is still lower than the one I obtain with a petri dish with LB-agar. I do believe that both LEDs and filter are an issue. </DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">August</FONT></FONT></H2><P><B>08/1/2018 </B></P>

New tests with the same UV plate but with different filter, lent by Clément. While trying them, I remembered an advice given by my optics teacher Lionel Bastard, that reducing the habitable area decreases the noise level. Moreover, I was yet very close to the saturation, so these were 2 reasons to hide a great part of the photodiode with black scotch tape. I reached 3.9V with an assembly of 2 filters, but I noticed I had better performance with my own filter as I reached 2.9V. This measures were performed with fluorescent bacteria. I felt confident and performed several measures on this Petri dish, then with a Petri dish with LB agar, and I calculated the dark current of the photodiode. This day was the 1st time I got coherent results about qualitative fluorescence, as the voltage was higher with fluorescent bacteria:

Actually the control is wrong, and new measures were performed with transformed bacteria that are not fluorescent, which changes the voltage, but I didn’t make the calculus yet. 
To confirm the previous result, I wanted to measure fluorescence with another Petri dish which is also highly concentrated in fluorescent bacteria, but still less than the 1st one. Here the results are not good as the voltage is lower than the control voltage. However, I should do this experiment again as I think I went a little too fast through it. 
Now it begins to be shitty: the UV transilluminator I used doesn’t work anymore. So I changed it, but it means I need to do my experiments again as the light excitation is brighter. So I began to do again the experiments but now I have no voltage difference. The voltage is higher (3.9V), so I guess the light should not be too bright. 
So I did the experiments again, and I don’t know why it failed before, but with the light reduced to 70% of the maximum, I can see a voltage difference again:

So I can see something, but with such concentrated bacteria, I’d hope to have higher voltages. Well, this shows how important the amplification stage is. It seems that I could use a gain of 10. But this will be determined undoubtedly with a liquid detection. This is what I tried to perform next but I struggle to have a decent apparatus to do my experiments: the photodiode is not correctly placed on the eppendorf so it saturates. Well, I’m just thinking of it, but it may be time to use again my 3D impressed unit.  <P><B>08/19/2018 </B></P>
Let’s sum up again. 

	Since last time, I made a lot of experiments, with many different conditions. I needed to make sure that my filter was correctly chosen. So I reused the last year apparatus, that I know it had worked well. 

The blue filter is my own filter, the red one is last year’s. The 1st graph displays different value of voltage fluorescence depending on the light source and the emission filter. I have very little differences between the voltage in any case excepting with a very bright light source. I could conclude 2 things from this : either the light source is too bright to be filtered with the red filter, which means mine is better, or the light source is so bright that the fluorescence intensity is much higher, but my filter is too selective; so the red filter is the better one. The second graph compares the voltage obtained with each filter with the control and the fluorescent samples. ΔU is defined as ΔU=ΔUred-ΔUblue. This Δ is always positive, I always have a higher voltage with the red filter, which is coherent with the previous results. I also get similar differences between the samples with different filters and light sources conditions, except for the very bright one: the control voltage is much higher. My conclusion on this is that the filters are equivalent for low bright light sources, but mine is better for high light intensity, which I can get with my LEDs when I supply them with the voltage source. Considering also that I know my filter but not last year’s, I decided to keep mine. 
Yet, I realised something. I don’t exactly remember how, but I got interested in the way my LEDs excite my samples. Either I remembered my past experiments with my 3D printed apparatus or I went on them again, but I was striked by how the light on the sample is less bright that at the level of the LEDs. I made the terrible mistake to think that the viewing angle was not important. Worse than that, I thought that the optical power is unchanging in the light beam, which is dreadfully wrong. I paid too much attention to the optical power, but the light intensity, which is dependent on the solid angle (and Lionel Bastard, my optics teacher, told me so well how important this solid angle is primordial), which depends on the viewing angle, is a better way to choose the light source. What I should actually have done is to take a LED with the highest light intensity possible and the lowest viewing angle, and determine experimentally the optical power with the photodiode. From that, I had several possibilities. I could simply change of LEDs, but I didn’t want to waste my time. I thought of using lenses to focus the light on the sample. It would have basically looked something like that:

It is a very schematic scheme, but it sums up the idea and the issues this idea brings. The LEDs are in the same plane that the photodiode, the focal object plane of the 1st lens, behind the emission filter. A 2 lenses assembly focus the light on the sample which is in the focal image plane of the second lens. The fluorescence light is reemitted on the photodiode which is on the same plane that the photodiode. This system raises 2 issues. Firstly, the filter would have been to selective, the light intensity on the sample wouldn’t have been high enough. Secondly, the viewing angle of the LEDs demand the numerical aperture. To get the full light beam, I would need a lens with a numerical aperture higher than sin(150°/2)=0.9 and a focal of 1.4 cm, which I did not find. 
The last solution before changing of LEDs is to get them closer to the sample. I changed the eppendorf holder for the aluminium plates, in which I installed the LEDs very close to the sample, so that I don’t need to take care of the viewing angle. This arrangement has given me the best results so far, as I could see a sensitive difference between a non-fluorescent sample and a fluorescent one. At this point, I have what seems to be a qualitative fluorescence detector, with a bad sensitivity. The measurements can all be found here under the name “fluo_dilution_quantitative_measurements”

The first experiments were done by hand, which explains the few values in comparison with the last experiments which voltages were measured with the Arduino. The values are significantly different as I faced many issues during the experiments (LEDs short circuit, not perfect dark, auto-inhibit fluorescence phenomenon), which makes difficult to compare them. However, one can learn a few lessons. The control voltage is too high, I have too much noise from the LEDs, which reduces the sensitivity. Also, I have nothing linear, it is much more like a logarithmic function, whereas the voltage is given by:

V=kRDRλP0C with:

	k a coefficient to be experimentally determined in L/(mol.cm)
	RD the trans-impedance stage resistance in Ω
	Rλ the responsivity at λ=610 nm in A/W
	P0 the optical power from the LEDs received by the sample in W 
	C the concentration of bacteria in the sample in mol/L

		Besides the 1st experiment, I always have the same order of magnitude (10<SUP>5</SUP>), and this result is quite near to the one obtained by Munich 2017. 
		To get back to the LEDs, I need to change them. Their emission spectrum overflow too much on the fluorescence spectrum. With yellowish-green LEDs, I hope to have a lower voltage control, and therefore a better sensitivity. 
		Something to be mentioned: the voltage measured with the Arduino is varying a lot, but the mean value is quite good (difference of 5mV with the voltmeter on 10000+ values) and the standard deviation is very low. 
		In case it doesn’t work, I’ll have to change the photodiode for a CMOS sensor, as the fluorescence can be easily seen. </DIV><H2><FONT size="6"><CENTER>Pipetting module &amp; global structure (Jean)</CENTER></FONT></H2><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">June</FONT></FONT></H2><P><B>06/18/2018</B></P> 
Today I worked on the pipette module, more precisely on the control of the pipette with the Arduino controller. 
First I soldered with Thomas all the buttons (Select, Up, Down…) and made the electronic circuit to control them. 
Then I started to write a program to test how to aspire different volumes and different speeds. I realized that the pipette doesn’t really reset so when one restart it, the values of the volume and the speed in/out are still the same as before. That’s a problem because without input from the user, it seems impossible to fix the pipette at a defined volume since we don’t know the value we start from. Moreover, we can’t even try to go at the maximum or minimum value to have a known start point because the pipette does a loop : When it reaches 10μL, if we increase again the next value is 0.20 μL. <P><B>06/19/2018</B></P> 
Still working with the control of the pipette. 

I tried to test different parameters: 
→ How much can we reduce the time for pressing a button? I found that the delay time 175ms for HIGH and LOW levels is a good value. With several tests, I didn’t notice any bug.  
→  How can we change the volume faster? If we keep the button up or down pressed then it goes faster. So I made a graph with the delay time we keep pressed on the button and the delta in volume it implies. !!  
NOTE: the next value after 10μL is 0.20μL <P><B>06/20/2018</B></P> 
The code is now quite well structured with different functions and parameters easy to modify.  
Seems like everything is working perfectly.  
There is : 
→ One function to set the volume with the initial and the wanted volume as parameters
It can use another function to determine the time we need to keep the button up or down pressed to change the volume faster. 
→ One function to set the speeds in and out. 
→ Functions to aspire and dispense the liquid <P><B>06/21/2018 → 06/27/2018</B></P> 
I worked with Charles on the conditioning circuit for the photodiode. We managed to make it work on the breadboard circuit but something is still wrong with the PCB. <P><B>06/28/2018 → 06/29/2018</B></P> 

I worked with Thomas on the mechanics for the circular movement of the carousel. We made some wedges to reduce the weight on the motor itself hoping that it would have an easier and more linear movement.  </DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">July</FONT></FONT></H2><P><B>07/02/2018</B></P> 
Meeting/brainstorming with the engineering team in the morning to take stock of the situation on every modules, what components we need to order, and all the constraints we have.  
→ We have to speed up on the “mechanics” and “pipette”  modules to be able to design the pieces we need on CAD. 
So I started to work on the Eppendorf 1000uL pipette the LMGP gave us. However, there is a problem we didn’t realise before : the eject tip button is mechanical… additionally we need a really small adaptor for the power supply that we don’t have. So, no test can be done for now, and it seems like we have to buy another pipette like we had before but designed for a bigger volume. 
Individual pictures the evening. <P><B>07/03/2018</B></P> 
→ Worked on the mechanics with Thomas. Found a better coupling system for the motor.  
→ Soldered connectors on the PCB plate for the fluorescence. Still doesn’t work. <P><B>07/04/2018</B></P> 
→ New tests on the rotative plate, results are OK. The steps are regular and precise enough. 
→ Found connectors for the pipette Eppendorf. It’s working properly with a sufficient power supply <P><B>07/05/2018 → 07/06//2018</B></P> 
CAD to design a new pipette clamp for the Eppendorf Pipette in case we use it. <P><B>07/11/2018</B></P> 
The pipette clamp has been printed, there is a problem of dimension but with bolts the fixation is OK.  
I sent mails to fablabs in the Grenoble region to find one we’ll be able to access while the buildings of Phelma will be closed. The INRIA fablab answered positively ! <P><B>07/12/2018</B></P> 
I drilled holes in the middle of the rotative wooden plates to make the cables and wires pass inside. 
I also began to look the water cooling system to cool the back side of the Peltier modules.  We’ll try to change the pipes to others less rigid so they can pass inside the hole too.   
I looked for connectors in T on internet to allow the filling of cooling liquid in the pump. 
Bought connectors, pipes, and stuff to test the water cooling with less rigidity. 
Testing that we realized that it would be complicated to use it because the pipes would still apply too much stress on the plates. 
Then we looked for thinner fans like laptops ones. <P><B>07/19/2018</B></P> 
→ Moving from Phelma to Jean Roger buildings. <P><B>07/20/2018</B></P> 
I started to look some connectors and assembled cables to make a “cleaner” and more robust prototype </DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">August</FONT></FONT></H2><P><B>08/06/2018</B></P> 
→ Looked for another pipette biohit e300 to buy on ebay. Bought one in the US that Thomas brother will bring back. 
→ Contacted the fablab of INRIA to use the laser cutting machine to make the plexiglas plates.
→ Looked for connectors and assembled cables. <P><B>08/07/2018</B></P> 
→ Looked again for connectors and assembled cables. 
→ Communication raspberry Pi / Arduino to use a touchscreen. <P><B>08/08/2018</B></P> 
→ Went to INRIA Fablab to make the plexiglas plates with the laser cutting machine. 
→ Made wooden wedges. 
→ Bought screws, stuffs, and fans for Jean Roger buildings :) <P><B>08/09/2018</B></P> 
Started to construct the new prototype with aluminium bars and plexiglas plates. We are missing some pieces to fix the brackets on the aluminium bars.  <P><B>08/10/2018</B></P> 
→ Looked for linear guide for the pipette like the one Julien Traveaux lent to us. 
→ Tested the warming resistance on a plexiglas plate. Good result, it doesn’t deform itself ! <P><B>08/13/2018</B></P>  
→ I designed the plate to support the pipette on Onshape. 
→ I tested the optocoupler, it worked. To use it with the pipette, we just have to change a bit the code : in the setup, let the state at HIGH, and for the function triggerbutton we have to invert HIGH and LOW also. Otherwise the button is triggered non-stop. <P><B>08/14/2018</B></P> 
 Went to INRIA to make plexiglas and wooden plates and wedges. Bought stuff at castorama <P><B>08/15/2018</B></P> 
Built the system with the new plates,  <P><B>08/16/2018</B></P> 
 I designed a piece on Onshape to fix the actuator with the magnet holder on the central plate.  <P><B>08/20/2018</B></P> 
I built the linear guide with the pieces received today and fixed the pipette on it. <P><B>08/22/2018</B></P> 
Soldered the pipette received from the US. Unfortunately, it is not working. We will have to use the pipette Biohit e10 event if it is not the ideal volume. <P><B>08/28/2018</B></P> 
Since we decided to realize a transformation in the machine, I am trying to execute some complex actions like taking a pipette tip. Unfortunately it is not working because the pipette can still move a bit in the pipette holder. <P><B>08/29/2018</B></P> 
Charles wrote a first pseudo-code to make a transformation and I tried to use it in the system. I referenced some angles to move the rotative plate so that samples are placed right below the pipette. <P><B>08/31/2018</B></P> 
Moving from Jean Roger buildings to Phelma ! <P><B>09/06/2018</B></P> 
Drilled a hole on the handle of the pipette to fix it better to the pipette holder so that we can take pipette tips better. </DIV><H2><FONT size="6"><CENTER>3D model &amp; Human Practices (Lucas)</CENTER></FONT></H2><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">July</FONT></FONT></H2><P><B>Week 29 (07/16/18 → 07/22/18) </B></P>
→ Working on 3D modelisation<P><B>Week 30 (07/23/18 → 07/29/18) </B></P>
→ holidays
<P><B>Week 31 (07/30/18 → 08/05/18) </B></P>
→ Monthly meeting (balance sheet meeting) 
→ Contact of Loic (survey and 2 writing) and Terence 
→ Modelisation of the “design” of the system
→ Review of the fluopart
→ Finalisation of the modelisation of the 2 plates 
→ Finalisation of the modelisation of groove profile and brackets (équerres) 
→ Order of groove profile, screw, nuts… from Motedis (2 commands), arrival at Elise’s House
→ IGem registration (600$)
→ Response to Loic, reading of the survey 
→ Lecture of Marion Rannou Thesis about how writing a survey in the best way, trying to help Loic writing the survey
→ Lectures of thesis for moving forward the fluo part
→ Aix Marseille meet up registration _ we decide that I have to go there _ I will have to prepare it for give it useful
→ Help Charles to soldering “micro” leds
→ Reasoning about the experiments for the fluo parts</DIV><DIV class="collapse slide"><H2><FONT size="5"><FONT color="#9c5899">August</FONT></FONT></H2><P><B>Week 32 (08/06/18 → 08/12/18) </B></P>
→ Help Charles to understand the experimental problems he faced (more voltage in the output of the photodiode when we illuminate the control sample than the fluorescent sample). We found it : the location of the sample is not exactly the same during the measurement…. So the experience is invalidated 
→ Design of the first video (explanation of the antibioresistance issue)  with Perrine
→ Mail to Terence for explaining what we want him to do 
→ Try to begin the list of changes in our models and experiments for integrated human practices
→ Organisation of human practices drive
→ Send a mail to Cavailles for having some contact working in the Grenoble CHU for making an interview (script of the mail already ready - but it’s not the case of the script of the interview) 
→ HP, contact with Loic and Terence, explanation of what they do and presentation of what we want them to do (sic) 
→ Rereading of  Oneshape’ s Sketches before going to Inria Fablab (see below) 
→ Creation of the 2 first squared plates  and the 2 first circular plates in Plexiglas in the Inria Fablab thanks to a laser cutting machine
→ Creation of the “wood” holds 
→ Thread (Taraudage in french) of the plates
→ Test of the structure 
→ Great breakthrough this day for Prototype 1 !!!  
→ I participate in iGEM Marseille Meet Up (09/08 and 10/08) <P><B>Week 33 (08/13/18 → 08/19/18)	</B></P>
→ Writing of the report of iGEM MArseille ‘s Meet UP. Team discussion about this meeting is ready for tomorrow (14/10) 
→ Contact and response to Grenoble physician Max Maurin. In the following days I have to write the interview
→ All the afternoon : discussion with EPFL iGEM team member for the HP and in order to participate in “université d’été solidaire et rebelle”. It’s free and open. SO it’s a new event for HP 
→ Selection of workshops to go in the université d’été event 
→ Close of the event “Semaine de l’éthique” because this event won’t be created
→ Writing of the timetable of HP : deadlines and to do list
→ Discussion with Lausanne Team  about HP
→ Writing of HP points to present during morning meeting (see above) 
→ Work on the first video
→ All the morning : Team discussion about HP (finalisation of the timetable)  and Marseille Meetup report 
→ Creation of the survey for physicians
→ Trying to improve the general survey (without success, no progress) 
→ Discussion about the script of the video and the animations we want to create for it
→ Helping Jean for Onshape model of actuator (actionneur) and animations
→ End of the survey for physicians and reviewing by Xandre and Charles
→ Morning : preparation (finding arguments, reading of Testard biography and book abstracts, reading of DU Penser les sciences  lectures, those about research ethics) of Tuesday 23 event 
→ Preparation of the oral for iGEM Spanish Meet Up
→ Modelisation on Onshape 
          → help for Jean modelisation 
                    → Creation of animation and add of pieces for the presentations             
→ I participate in iGEM Spanish Meet Up (08/18, 08/19, 08/20) <P><B>Week 34 (08/20/18 → 08/26/18) </B></P>
→ I participate in iGEM Spanish Meet Up (08/18, 08/19, 08/20) 
→ contact with Karmele (for crowdfunding and paper), the two physicians and Michel Sève (vulgarisation conferences) 
→ Finalisation of the physician script interview 
→ Preparation of the Event “Université d’été”, reading and analysis of the EPFL resume of Testart’s book. Analysis of pages 1 to 13 (of Laurine resume) 
→ End of the preparation of event “Université d’été”, contact with EPFL team 
→ Beginning of organising the notes 
→ Call with Terence for the video : great progress !  
→ I participate in Université d’été with EPFL team  (08/23)  
→ Writing of Epaulard and Falconnet interview, Barcelone Meet Up report, and completing of events folders (add of pictures, write “states of the art”, updates, template) 
→ Interview with Pr. Olivier Epaulard<P><B>Week 35 (08/27/18 → 09/02/18)</B></P>
→ Writing of all interviews and events balance
→ Interview with Pr. Max Maurin </DIV></DIV></DIV><FOOTER><A href="mailto:Igem.grenoble.alpes@gmail.com">Igem.grenoble.alpes@gmail.com</A></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>