<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Grenoble-Alpes_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Grenoble-Alpes/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><SPAN class="error">Template loop detected: <A href="/Template:Grenoble-Alpes" title="Template:Grenoble-Alpes">Template:Grenoble-Alpes</A></SPAN><NAV><UL id="ulmenu"><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes">HOME</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#team-members">TEAM MEMBERS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#advisors">ADVISORS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#partners">PARTNERS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Collaborations">COLLABORATIONS</A></LI></UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/biology">BIOLOGY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Hardware">HARDWARE</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Demonstrate">PROOF OF CONCEPT</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Model">
						MODELING
					</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/actors_partnerships">ACTORS &amp; PARTNERSHIPS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ethic">ETHICS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/events">EVENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Safety">SAFETY &amp; SECURITY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ressources">RESSOURCES</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ressources">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols">PROTOCOLS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/components">COMPONENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/medal_criteria">MEDAL CRITERIA</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/characterization">CHARACTERISATION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/site_map">SITE MAP</A></LI></NAV></P><DIV id="container"><DIV id="welcome"><P id="titre-image">DEMONSTRATE</P></DIV><DIV id="sous-menu"><UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description" id="current-nav">PROJECT </A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/biology">BIOLOGY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Hardware">ENGINEERING</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Demonstrate" id="current-menu">DEMONSTRATE</A></LI></UL></DIV><DIV id="text-container"><P>The following page contains the results and succes that we actually obtained with our system.</P><DIV style="padding:3px; padding-left:6px; border:5px solid #4256f4; margin-left:20px; background-color: #4256f4;" id="THIRD"><CENTER><FONT id="optitl"><FONT size="10"><FONT color="white">TRANSFORMATION MODULE </FONT></FONT></FONT></CENTER></DIV><P>First, we tested the temperature control module on a biological transformation because we knew that the tricky part would be the cooling step of the thermal shock. In fact, the module could only go down to around 11 degrees so we looked out for literature about optimization of the transformation.</P><H2><FONT color=" #4256f4"><CENTER> What literature has to tell us</CENTER></FONT></H2><P> As a reminder, a biological transformation or as it is also called a “temperature shock transformation” is typically a succession of a cold–hot–cold treatment from 0°C to 42°C and back to 0°C to induce the transfer of exogenous genetic material in competent bacteria.</P><P>According to a study [1] about the effect of Heat Shock Temperature, Duration, and Cold Incubation,<EM style="font-style: italic;"> “Escherichia coli BW25113 transformed with the pUC19 cloning vector at cold-step temperatures of 0°C, 4°C, 10°C, and 20°C demonstrated <EM style="font-weight:bold;">no significant difference in transformation efficiency</EM>. When the same strain was <EM style="font-weight:bold;">transformed using approximately a 20°C difference between the cold-step and hot-step treatments:</EM> 0 → 20 → 0°C and 20 → 42 → 20°C, no significant difference in transformation efficiency was observed between the two treatments.”</EM></P><P>Considering the results of this study, we hypothesized that because the module has more than a 20°C difference between the cold-step and the hot-step (in fact, between 11°C and 42°C, there is a difference of 31°C), the transformation should succeed. However, the question was to know how well it would succeed. To test this, a few parameters could be changed: the DNA concentration or the type of competent bacteria for instance.</P><H2><FONT color=" #4256f4"><CENTER> Petri dishes experiments</CENTER></FONT></H2><P>We first tried to transform pSB1C3-BBa_J04450 plasmids in competent DH5α bacteria and Top10 (two different E. coli strains) and growed colonies on petri dishes.</P><P><FONT size="5">Goals:</FONT><UL><LI>Verify that the heating and cooling modules work</LI><LI>Test the efficiency of a transformation as a function of the DNA quantity and plot a curve similar to the following 24<SUP>th</SUP> figure.</LI><LI>Test the efficiency of the module compared to a classical transformation using two different competent bacteria (Top10 and DH5α).</LI></UL><CENTER><FIGCAPTION>Figure 1: Transformation efficiency as a function of DNA concentration[2] </FIGCAPTION></CENTER></P><P>The protocols for these experiments are available in the <U><A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols">protocol page</A></U>.</P><H2><CENTER><FONT color=" #4256f4">Fluorescence kinetic</FONT></CENTER></H2><P><FONT size="5">Goals:</FONT><UL><LI>Test the performance of Top10 competent bacteria (another E.coli strain with different features) during a bacterial transformation in the module by observing the fluorescence with a microplate reader.</LI><LI>Compare the efficiency of Top10 competent bacteria with DH5α competent bacteria.</LI><LI>Get information on how long it takes for bacteria to start producing enough fluorescence to detect it with a microplate reader.</LI></UL></P><P>The protocols for these experiments are available in the the <U><A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols">protocol page</A></U>.</P><H2><FONT color=" #4256f4"><CENTER>Results</CENTER></FONT></H2><H3>Results on Petri dishes</H3><P>The following part describes the results of the first experiment on DH5α and Top10 bacteria, where a classical transformation and the module were compared. As a reminder, the plasmids transformed inside the competent bacteria contain a RFP (Red Fluorescent Protein) gene. Hence, if the transformation is a success, the bacteria will start to emit a red fluorescence after a certain amount of time.</P><P>For the experiment with DH5α, the conditions were not optimal. The ambient temperature was over 35°C and the cooling module could not go under 18°C (so the temperature difference between the cold-step and the hot-step was only 24°C). This surely had an impact on the results in a bad way.After one night, the Petri dishes were photographed and the colonies counted.<CENTER><FIGCAPTION>Figure 2: Photography of the Petri dishes and colonies count</FIGCAPTION></CENTER></P><P>The first observation to make is that there are colonies in the Petri dishes realized with the module, so <STRONG style="font-weight:bold;">the transformation module works</STRONG>. However, there are much less colonies in the dishes from the module than from the classical transformation in this experiment. Hence, it was not as effective as a classical transformation.</P><P>In the following graphics (figures 3 and 4), the number of colonies as a function of the initial DNA concentration used to realize the transformation (dark blue curve) is compared to the efficiency of the transformation (ratio between the number of colonies and the initial concentration of DNA) in light blue on the graphics.</P><P>As we can see, even though the number of transformed colonies is increasing as the DNA concentration is increasing (which is logical because if bacteria are more in contact with DNA, the chances that they will integrate it in their genome is greater), there is a pic of efficiency and then it starts to go down very rapidly.</P><P>This information is interesting for the optimization of the whole system. In fact, if the efficiency is good, a lot of bacteria will have integrated the plasmid containing the RFP and a resistance gene to chloramphenicol. Hence, when the antibiotic is added, less bacteria will be killed and dead bacteria remains will not crowd the medium. The development of fluorescent bacteria will be easier.</P><P><CENTER><FIGCAPTION>Figure 3: Classical transformation efficiency as a function of DNA concentration</FIGCAPTION></CENTER></P><P><CENTER><FIGCAPTION>Figure 4: Module transformation efficiency as a function of DNA concentration</FIGCAPTION></CENTER></P><P>For the experiment with Top10 bacteria, the weather conditions were much better and the cooling module could go to 11°C.
After one night, the Petri dishes were photographed (see figure 5)<CENTER><FIGCAPTION>Figure 5: Photography of Petri dishes for Top10</FIGCAPTION></CENTER></P><P>By observing the two petri dishes on top and comparing them to the two from the bottom of the figure, we can deduce that, on average, there is not a huge difference in the number of colonies (especially for the second set where the number of colonies is approximately the same). This means that for this experiment, the transformation module was as efficient as a classical transformation.</P><P>Moreover, when comparing the Petri dishes made with commercial Top10 and Top10 we made competent, there is not either a significant difference in the number of colonies. The conclusion we can make here is that for our use of those bacteria, the one prepared in our lab are enough and no need to buy commercial Top10.</P><H3>Results of fluorescence kinetic</H3><P>For this experiment, the bacteria stayed the whole time inside the microplate which is not exactly recreating the conditions of the machine (an Eppendorf tube of 0.5mL whose fluorescence is measured by taking a photo and not by taking samples to measure every hour). Nonetheless, it will give us interesting information.
            The three following figures 29, 30 and 31 show the evolution of the OD and fluorescence for 18 hours. A measure was taken every hour for 5 hours then a last one was taken the morning after.<CENTER><FIGCAPTION>[3]
Figure 6: Evolution of the fluorescence and OD for top10 (duplicate 2)</FIGCAPTION></CENTER><CENTER><FIGCAPTION> Figure 7: Evolution of the fluorescence and OD for TOP10 (duplicate 1)</FIGCAPTION></CENTER></P><P>Comparing those two figures shows that Top10 bacteria start producing fluorescence about 3 hours after the first measurement, which is about 5 hours after the transformation. However, the production is rather weak. In one of the well, LB was added overnight and the fluorescence and OD were measured again. The fluorescence was largely superior in this well than in the other one and so was the OD. Hence, this shows that bacteria produce more fluorescence when they are still in the growing phase</P><P><CENTER><FIGCAPTION> Figure 8: Evolution of the fluorescence and OD for DH5alpha</FIGCAPTION></CENTER></P><P>Contrary to Top10 bacteria, the DH5α didn’t show any OD change or fluorescence until the morning after. Meaning that it must take more than 6 hours for them to start producing fluorescence.
            In our system, the main goal is to get a result in the smallest possible amount of time. For this reason, the fluorescence sensor must be very sensitive but the competent bacteria must also be carefully chosen! In this case, Top10 bacteria were chosen.</P><FONT size="6"><FONT color="#4256f4"><CENTER>CONCLUSION</CENTER></FONT></FONT><DIV style="padding:5px; background-color:#ffffff; border:2px solid #4256f4; -moz-border-radius:9px; -khtml-border-radius:9px; -webkit-border-radius:9px; border-radius:9px;"><P>To conclude on this part, because the temperature control module worked on a bacterial transformation, it seemed interesting to test a few parameters to optimize the biological step inside the final machine.
            The first question that was answered is:<STRONG style="font-weight: bold;"> “what competent bacteria should we use?”</STRONG>
            The performances of competent DH5α, commercial competent Top10 and Top10+competency protocol were compared (the parameters of choice being the ratio of transformed bacteria and the time needed to produce enough fluorescence to be detected).
In regards of the results and the price of those bacteria, the good choice would be to use Top10+competency protocol because the only cost is from the first batch of Top10 we put in culture and they are much more efficient than DH5α and there seems to be no difference with commercial competent Top10 bacteria regarding the transformation efficiency.
 
Second question answered:<STRONG style="font-weight: bold;"> “how much DNA should we use for the transformation?”</STRONG>
            The question requires to get back to the precedent biological step in the machine (the hybridization) and to estimate the concentration in probes and the percentage that will recognize a target sequence and form a plasmid that a competent bacterium can integrate. Statistically, this percentage is not very high so it would be better to work with small concentrations of DNA to get a better transformation efficiency and to be as close as possible to the conditions inside of the machine.</P></DIV><DIV style="padding:3px; padding-left:6px; border:5px solid #4256f4; margin-left:20px; background-color: #4256f4;" id="THIRD"><CENTER><FONT id="optitl"><FONT size="10"><FONT color="white">DNA EXTRACTION MODULE </FONT></FONT></FONT></CENTER></DIV><P>One of the main goal of the project was also to realize at least one biological step in our engineered system. Finally, we managed to realize a purification step with BL21 bacteria and using a lysis buffer instead of phages.</P><P>We wrote an entire Arduino computer code able to control the different elements of the system : the pipette, the linear guide, the rotative plate, the heating system, the actuator with the magnet holder. With the coordinated control of these different elements, we successfully managed to realize the transformation. Below is the protocol that the Arduino code executes.</P><P>Protocol realized:
</P><P style="color:#4256f4">Add 200µL of Buffer 1 (Lysis Buffer)</P><LI><P style="color:#4256f4">Add 100µL of bacteria + wait 10 minutes</P></LI><LI><P style="color:#4256f4">20µL of magnetic beads + wait 10 minutes</P></LI><LI><P style="color:#4256f4">Bring magnets next to the tube to attract the magnetic beads and extract the waste liquid</P></LI><LI><P style="color:#4256f4">Add 300µL of Buffer 1 (Lysis Buffer), flush, extract the liquid while magnets attract the beads and throw to the waste.</P></LI><LI><P style="color:#4256f4">(Add 300µL of Buffer 2 (Wash 2), extract while magnets attract the beads and throw to the waste the liquid) * 2 times</P></LI><LI><P style="color:#4256f4">Add 300µL of Buffer 3 (Elution), extract  while magnets attract the beads and throw to the waste the liquid.</P></LI><LI><P style="color:#4256f4">Eluate once more with 30µL of Buffer 3 + heat to 70°C during 5 minutes (DNA detaches from the beads)</P></LI><LI><P style="color:#4256f4">Extract while magnets attract the beads and store the liquid (DNA).</P></LI><P>The Arduino computer code allowing to realize this process is available here in this zip file.</P><A href="https://static.igem.org/mediawiki/2018/5/51/T--Grenoble-Alpes--zipurification.zip" download="" style="position: relative;" class="btn btn-success">Download</A><P>We obtained with a nanodrop absorbance measurement a concentration of 38,4ng/uL and a purity of 1.87. The purity is between 1.8 and 2.0 so it means that the purity is acceptable for DNA. [4]. As for the concentration, we can’t tell if it is consistent or not because we did not measure the OD of the bacteria sample before the experience. Anyway it is not a value close to null so we still can tell that we extracted DNA.</P><P>That proves that our machine works ! And with more time, we could consider realizing the other steps of the biological process in the machine to have a proof of concept for the whole system. We could also spend more time to characterize the efficiency of the machine comparing to the process realized outside the machine.</P><P>Below is a video capturing the purification:
</P><DIV style="padding:3px; padding-left:6px; border:5px solid #4256f4; margin-left:20px; background-color: #4256f4;" id="THIRD"><CENTER><FONT id="optitl"><FONT size="10"><FONT color="white">FLUORESCENCE MODULE </FONT></FONT></FONT></CENTER></DIV><P>Our fluorescence sensor is able to detect fluorescence in samples where the optical density is at least of 0.014. You can find more explanations about our fluorescence unit <A href="https://2018.igem.org/Team:Grenoble-Alpes/fluorescence_module" style="font-weight: bold;">here</A>. </P><P><STRONG style="font-weight: bold;" font-size:1.5em="">References</STRONG><FONT size="3">
[1] Adrian Chu, Cindy Liu, Fabian Tam, Jessica Zhang (Avril 2014) “The Temperature of the Cold Shock during Temperature Shock Transformation Has No Effect on Transformation Efficiency of Escherichia coli” published in the Journal of Experimental Microbiology and Immunology (JEMI), Vol. 18: 87 – 90, consulted in August 2018
[2] Mahipal Singh, Arpita Yadav, Xiaoling Ma and Eugene Amoah (2010) “Plasmid DNA Transformation in Escherichia Coli: Effect of Heat Shock Temperature, Duration, and Cold Incubation of CaCl2 Treated Cells” published in the International Journal of Biotechnology and Biochemistry, Volume 6 Number 4 (p. 561–568), consulted in August 2018
[3] Lloyd Yoo (Avril 2010) &quot;The Effect of rpoH for Heat Shock Gene Expression on Plasmid Transformation” , published in the Journal of Experimental Microbiology and Immunology (JEMI), Vol. 14: 108-11, consulted in August 2018.
[4] Bioblabs, “NEB® 5-alpha Competent E. coli (High Efficiency)&quot; consulted in July 2018 on https://international.neb.com/products/c2987-neb-5-alpha-competent-e-coli-high-efficiency#Product%20Information</FONT></P></DIV></DIV><FOOTER><A href="mailto:Igem.grenoble.alpes@gmail.com">Igem.grenoble.alpes@gmail.com</A></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>