Team:Grenoble-Alpes/conservation
Template loop detected: Template:Grenoble-Alpes
HOME
TEAM
TEAM MEMBERS
ADVISORS
PARTNERS
ATTRIBUTIONS
COLLABORATIONS
PROJECT
DESCRIPTION
BIOLOGY
HARDWARE
PROOF OF CONCEPT
MODELING
HUMAN PRACTICES
OVERVIEW
ACTORS & PARTNERSHIPS
ETHICS
EVENTS
SAFETY & SECURITY
RESSOURCES
OVERVIEW
ACHIEVEMENTS
NOTEBOOK
PARTS
PROTOCOLS
COMPONENTS
MEDAL CRITERIA
CHARACTERISATION
SITE MAP
CONSERVATION
PROJECT
BIOLOGY
BACTERIA CHOICE
TARGET SELECTION
PHAGE LYSIS & DNA EXTRACTION
PROBE CONSTRUCTION
CONSERVATION
ENGINEERING
DEMONSTRATE
The storage of biological components is a major issue for laboratories, which use a lot of energy to power freezers allowing to store bacteria at -80°C or enzymes at -20°C for example. The shipment also represents a big challenge, to preserve cold chain and suitable storage conditions during transport. For instance, companies must send their products in dry ice, which represents an important cost either for laboratories which have to pay it as a supplement with every enzymes shippings for example, or for the companies themselves for whom dry ice represents an increase of package size and thus shipment price.
Our system is not making an exception and one of our main issues was the preservation of the biological tools we used within: restriction enzymes , DNA plasmid probes , and competent bacteria . The storage had to be efficient, long-lasting and adapted to each component so it does not alter its biological properties, but without consuming lots of energy. Indeed, enzymes do not have the same stability or resistance to temperature changes as plasmid DNA or living organisms such as bacteria. Our main goal was to ensure an easy way to conserve the products used in the system, in order to lessen logistical, energizing and repeatability issues.
To achieve this, we had to find the most suitable and appropriate conservation technique for each component by searching in the bibliography, and then test these techniques to improve their efficiency.
I. COMPETENT BACTERIA
The first biological element we needed to conserve was the competent bacteria, which should integrate our DNA plasmids (containing the detection probes)  after hybridization to the target and express the fluorescence encoded by the plasmid.
Our first idea was to genetically modify our competent bacteria so they produce their own cryoprotectant to better resist freeze-drying.  We chose trehalose as cryoprotectant due to its important and well-known effect in cryo-conservation and freeze-drying. Indeed, this carbohydrate is often found in organisms that have great ability to withstand freezing, drying of freeze-drying, such as Saccharomyces cerevisiae[1]. Another argument was the presence of the gene coding for trehalose, known as OtsBA, in E. Coli DH5α genome already (the bacteria we chose to use in our system). The innate presence of the gene in the bacterium genome ensured its good integration and expression in a synthetic system. In the Nature, this particularity allows the bacteria to better resist osmotic stress, but the production of trehalose is usually limited to stressful conditions.
As for us, we thought of using this gene to overproduce trehalose under certain conditions, and not only osmotic stress, so the bacteria produce a sufficient amount of carbohydrates before undergoing freeze-drying and not only because of the stress generated by the process itself.
To do this, we wanted to design a plasmid including the OtsBA gene extracted from E. Coli genome, preceded by an inducible promoter that would act like a switch to produce trehalose only when the bacteria need it.
In this way, we could switch on trehalose production a few hours before freeze-drying, stop induction, proceed with lyophilization, and then use these bacteria for transformation with our detection probes. These modifications could result in an E. Coli strain improved to better resist freeze-drying without having to add cryoprotectant in preparation mix, only needing to add inducer when culturing bacteria and then freeze-dry them. After the freeze-drying process, the plasmid will not be useful anymore, and the switch “off” (namely interruption of induction) will allow to shut the trehalose expression off, preserving the bacteria energy for growth and fluorescence expression only.
The problem with this technique was the use of our bacteria in the system. Indeed, they had to be able to undergo a transformation with a plasmid detection probe, based on the pSB1C3 backbone. This plasmid as a pUC19-derived pMB1 replication origin, which allows a high copy number of plasmids in the cell (between 100 and 300).
The difficulty lies in a phenomenon well-known to anyone who ever wanted to proceed a double transformation: plasmids incompatibility groups. Basically, two plasmids which have the same origin of replication cannot be transformed together, as they will not be replicated by the cell machinery before its division in two daughter-cells, thus leading to an asymmetric repartition of the two plasmids in the following generations.
To ease the choice of plasmid for cloning, replication origins have been classified in incompatibility groups. In this way, molecular biologists know they can use two plasmids in the same transformation only if they are not part of the same group. To design our construction, we needed to pay attention to this criterion for the backbone choice.
We also needed a plasmid that allows the highest production of trehalose, in other words, a plasmid which permits a sufficient expression of trehalose-pathway’s proteins. This could be achieved by the modulation of several conditions. For instance, with a given promoter-RBS pair, DNA might be present in several copies using a high-copy plasmid to raise the number of ARN transcribed and thus the number of proteins synthesized. However, this could lead to a misfolding of the proteins produced, that would not be stable or operative at all. Else, we could use a low-copy plasmid that may provide fewer copies of transcribed DNA  and so a decrease of translation rate, ensuring the right folding of the proteins produced. However, the amount of trehalose produced may not be sufficient for the bacteria to survive freeze-drying.
Here lies the main obstacle to our construction: all of the high copy plasmids found had similar replication origins, that were all from the same incompatibility group and so do not permit a co-transformation with our pSB1C3 probes. The other backbones were not as efficient in replication, leading to tens plasmid copies at best, which may not be enough to produce a large amount of trehalose.
Actually, the mechanism we have described here is just a very small part of the expression machinery. The optimization of such protein expression system is a much more complex and time-consuming process, which is totally different for each protein and requires to try lots of different ORI-promoter-RBS combinations.
Confronted with this fact, we decided to give up the modification of bacteria, because we lacked time to try different backbones-promoter-RBS combinations to find one sufficiently effective in trehalose production. Finally we chose to try freeze-drying bacteria with added carbohydrates instead, an experiment that had already been tested by former teams, including the iGEM Grenoble Alpes 2017 team.
We started from last year’s team results and protocols to optimize the process and adapt it to our system. To be used in our system, bacteria must not only be able to survive lyophilization, conservation at 4°C instead of -80°C, and rehydration, but they also need to efficiently transform a plasmid and express its fluorescent proteins. The goal was to test different kinds of carbohydrates already known for their role as a cryoprotectant and different concentrations of these carbohydrates, to find the best conditions to fulfill all these requirements.
We followed the protocol described in “Process for producing freeze-dried competent cells and use thereof in cloning” (Barnea et al. 2002) [2] to get competent cells capable of surviving freeze-drying and conservation at 4°C. Trehalose and sucrose are recurring carbohydrates in the bibliography as well as in this article when it comes to efficiently freeze-dry a biological element, like proteins or bacteria. Thus, we decided to try different mixes of these carbohydrates to improve our lyophilization survival rates.
2017 Grenoble team tried to increase concentrations of sucrose (40mM, 60mM, 80mM, 100mM), showing that the highest concentration was the most efficient. Therefore we made the choice not to test lower concentrations and we tried to lyophilize E. Coli DH5α with at least 100mM of carbohydrate (sucrose, trehalose or both). We tried to freeze-dry bacteria with six different resuspension mix after competency protocol: 100mM CaCl2 with sucrose (100mM for sample S1, 150mM for S2), trehalose (100mM for T1, 150mM for T2), both sucrose and trehalose (100mM each for sample ST) or no additives (negative control C-). We also did a positive control of each sample (with the same preparation protocol) that we put in the -80°C freezer. We snap-frozen 200μL bacterial samples using liquid nitrogen and put them on the laboratory freeze-dryer.
Our first results were not good due to a technical issue with the lyophilizer, leading to the defrosting of bacteria during the night. We tried to snap-freeze them again and lyophilize them the morning after, even if we were convinced they did not survive. We stopped the freeze-drying process about 20h after the beginning and put the bacteria in a 4°C fridge. Three days after we rehydrated the freeze-dried bacteria with a resuspension solution (ice-cold sterile water, DMSO 7%) in ice, with the same volume as the one bacteria were before freeze-drying (200μL). We defrosted the positive control samples in ice at the same time and then proceeded to a bacterial transformation with 31ng pSB1C3_J04450 plasmid added to 50μL of each bacteria samples. After incubation at 37°C for 2h to allow expression of antibiotic resistance and concentration of bacteria by centrifugation, we plate 25μL each sample half of the petri dish and placed it at 37°C overnight.
To check if the bacteria had survived lyophilization, we also used 50μL of the 150μL left of each resuspended sample to culture bacteria (not transformed) in 1mL LB broth medium. We were then able to measure the difference of 600nm optic density 3 hours later compared to “zero time”, namely the growth of bacteria during these 3 hours. Not surprisingly, all of the lyophilized samples did not grow at all, while frozen samples D.O.600nm had more or less increased, except for negative control. Thus we hypothesized freeze-dried bacterial cells were dead.
The morning after, our thought was confirmed: there were not any colonies for the freeze-dried samples. Nonetheless, results were not pointless: all of the frozen samples had many colonies on plates, except for the negative control without any cryoprotectant, showing the significant role of carbohydrates for survival at low temperatures. We also noticed that increasing concentrations of carbohydrates led to increasing numbers of colonies on plates, a lead that had to be further examined.
Anyway, this experiment showed us that our competency protocol was efficient.
Due to bad results when observing fluorescence with E. Coli DH5α for the engineering part of the project and taking into account a fluorescence kinetics experiment with three different E. Coli strains we did as part of the BBa_J04450 characterization experiment , we carried on experiments with E. Coli Top10 instead. Indeed, we hoped it would increase the fluorescence rate, decrease the time before expression of mRFP (red fluorescent protein) and allow us not to use IPTG as an inducer.
The first experiment we tried on E. Coli Top10 was exactly the same as the one with DH5α, except the fact we tried freeze-drying lower volumes of bacteria (aliquots of 50μL or 100μL instead of 200uL). The idea was to have aliquots with the right amount of bacteria for a transformation in our system (contrary to our experiment with DH5α when we used only 50μL out of the 200μL resuspended aliquot).
Generally, when you use a lyophilized biological component, you have to rehydrate it with the same volume of rehydration buffer as the start volume before lyophilization. As freeze-drying is a stressful process which likely leads to the death of an important portion of bacteria even when using cryoprotectants, we considered increasing the volume (and thus the number) of lyophilized bacteria without changing the final resuspension volume. Thus we could increase the number of living competent bacteria without changing the final volume for the transformation process.
We tried the same competency protocol as the one used for DH5α, and the same carbohydrate concentrations, only changing the final volume of each sample (two aliquots: 50μL and 100μL). In parallel we froze 50μL aliquots of each sample and a positive control with glycerol 15% to compare.
Three days after the end of the freeze-drying process, we tested a transformation with the same plasmid DNA (pSB1C3_J04450 diluted to a final concentration of 31ng/μL). We resuspended all our lyophilized samples with 50μL of ice-cold resuspension solution (even the ones with 100μL start volumes) by pipetting up and down for about 30 seconds and incubating on ice for more than 10 minutes. We also defrosted the frozen ones (including our control with glycerol) in ice and proceeded with transformation protocol.
To better analyze our transformation efficiency with the freeze-dried bacteria, we made cascade dilutions of our positive control with glycerol before spreading on plates.
We also tried to double the competency protocol (incubations with 100mM MgCl2 and 100mM CaCl2 repeated twice), to see if it would improve the number of transformed bacteria. The results were not significantly different between protocols and we decided to continue with the “simple competency protocol” (data not shown).
After overnight growth on LB-agar-chloramphenicol plates, we counted bacterial colonies.
As shown on the figure above, the freeze-drying process was not really efficient compared to the freezing process with the same protectant. This was not a surprise for us, as researchers we met when searching for a freeze-dryer for our experiments told us that they do not usually obtain really good yields after rehydration. We had some colonies on plates though, which was an improvement compared to the first experiment with DH5α.
Surprisingly, we had more colonies when bacteria were frozen and protected with added carbohydrates than with glycerol (C+ with no dilution), sometimes doubling the number of colonies. However, we must note that we the cascade dilution results for C+ are not very precise as the number of colonies does not follow the dilution factor between “no dilution” and “dilution 1/10” samples. This might be due to an error when counting the colonies in the “no dilution” or a bad spreading on plates. Thus we must be careful when comparing results with C+.
The negative control showed that bacteria needed protectant for both freezing and freeze-drying processes, with almost no colonies on plates at all. These results were encouraging as we were able to see an improvement with carbohydrates: exponential for freezing, slighter for lyophilization but still we had something.
When observing the glycerol sample dilutions (positive control C+) which represented our “laboratory reference”, we saw that trehalose samples could be compared to the 1/100 dilution, except for the trehalose 150mM with 100μL start volume which was closer to the 1/50 dilution (a logical observation as the number of bacteria should be twice higher as the same sample with 50μL start volume). As for the sucrose samples, the results were not so good with less than 10 colonies on plates. Finally, the sucrose-trehalose samples were the most promising, with colonies numbers much higher than the other samples (more than three times as high as the highest number of colonies obtained so far with 100μL T2). The results with these conditions were closer to the 1/10 dilution of glycerol control, with even better results for the 100μL sample.
After this experiment, we wondered about the effect of trehalose and sucrose used together: was the improvement due to the combination of both carbohydrates or to the higher concentration of total carbohydrate (100mM each, leading to a total concentration of 200mM added carbohydrates)?
We thus decided to run this experiment again with changes in carbohydrate concentrations, in triplicate and with start volumes of 100uL (which seemed to lead to higher numbers of colonies).
The results for our control are surprising as we obtained more colonies when we had fewer bacteria (more colonies for 50μL than for 100μL start volume of bacteria). This might be due to a too high concentration of bacteria during transformation protocol, that led to the death of a part of bacteria. This phenomenon may also rely on a problem during conservation: a too high volume may have impaired the freezing of bacteria, as they would not have been frozen with the same speed. It is known that in some cases, snap-freezing give better transformation yields than slow freezing processes, especially with cells rendered competent with CaCl2. Hence the slower freezing for bacteria in the center of the tube put in -80°C freezer might have impaired their competency, and thus led to lower transformation yields.
Interestingly, the results obtained with the combination of sucrose and trehalose were not as good as the ones in the former experiment. The increasing concentrations of sucrose and trehalose did not help to enhance the protocol’s efficiency, as we see no significant difference between ST1 (100mM bot carbohydrates) and ST2 samples (200mM both). This mix still gives a higher number of colonies than the carbohydrates alone with concentrations already tested (samples S1 and T1 with 100mM carbohydrate) and also for sample T2 with 200mM trehalose.
Yet it was not the case for the sample S2 with 200mM sucrose which seemed to give the best results so far. However, we can see that the number of colonies obtained with this concentration of sucrose varies a lot, meaning that these results might not be representative of reality but just rely on the samples used. Anyway, we can still say that 200mM sucrose can deliver results sometimes three times as efficient as samples conserved in sucrose and trehalose combined or trehalose alone.
When we compare with our positive control we can see that 200mM sucrose gives similar results to the ones with the same start volume of bacteria diluted 10 times, and in a lesser extent to the ones with 50μL diluted 50 times.
Based on these observations we decided to use 200mM sucrose for the freeze-drying of our competent TOP10 E. Coli to use in the system.
IMPROVEMENTS
To improve our experiments, we should have tried transformations after different and longer conservation periods. We made the supposition that the loss of efficiency compared to the frozen samples was mostly due to the freeze-drying process, and in a lower extent to the conservation itself. That’s the reason why we only tried transformations few days after lyophilization, a rhythm that was more comfortable for us as we did not need to wait for weeks to obtain results and test other freeze-drying conditions.
However, several transformations with the same freeze-dried bacterial stock after different conservation periods would have given more information to confirm our start hypothesis and show that the limiting part of the method truly relied on the lyophilization itself. Until such experiment, the only observations we can make are about the freeze-drying with carbohydrates, and not the all the conservation process.
We can also add that our results are based on an experiment made in triplicate, that might not be representative of the reality. We should have done experiments with a higher number of samples for each condition, and also with different laboratory freeze-dryers to conclude about the efficiency of freeze-drying with carbohydrates.
II. DNA PLASMID PROBES
The second biological product we needed to preserve was our plasmid detection probes. Such as the other DNA molecules, their conservation was not a big issue as the DNA molecule is relatively stable, a mandatory condition to fulfill its biological function of information conservation[3].
Thus, we were able to dry our plasmid overnight under a Biological Safety Cabinet and conserve our samples either at room temperature or in a 4°C fridge.
Three days after drying of the plasmid according to our protocol we proceeded bacterial transformations in E. Coli Top10 with rehydrated plasmids conserved at 4°C or room temperature and the same plasmid DNA stock conserved at -20°C as a positive control. We also did a transformation of this plasmid DNA stock in E. Coli DH5α to check the competency protocol the Top10 were subjected to.
For the positive control we can see there are more colonies for the Top10 compared to DH5α, which tells us our competency protocol was well-functioning on Top10 E. Coli too.
When we compare the positive control with Top10 to our results with our dried DNA samples, we can see that there is no significant difference: there are too many colonies to count but the number seems similar for the sample that was conserved at room temperature. For the sample at 4°C, it is harder to conclude due to a bad spreading of the bacteria on the plate. For the second experiment, we thus decided to compare the transformation efficiency by spreading the same volume of bacteria but on the whole plate this time in order to better separate colonies. Nevertheless, we were satisfied by these results as the transformation with dried plasmids seemed very effective.
The second assay consisted of the same transformation protocol, but 10 days after drying of the plasmid. The samples used were exactly the same as the one in the first experiment and were prepared and dried the same day, only conserved for a longer period at 4°C or room temperature. After the transformation protocol with aliquots of Top10 from the same stock, we plate the same volume with one plate per sample.
As we can see, the results look pretty much like the ones obtained 3 days after drying: there are no significant differences between samples, except for the colonies spreading issue that alter readability and the identification of the colonies, especially for the room-temperature sample (4).
Nevertheless, the huge amount of colonies in all plates, which makes them impossible to count, was enough for us to conclude about the efficiency of this drying method for plasmid conservation.
IMPROVEMENTS
To improve our results, we would have liked to try drying the activated DNA probes we constructed. Indeed, nothing could ensure that their stability would be the same as the one of a double-stranded circular plasmid. The fact that activated probes are firstly linearized and also have single-stranded ends, may have a huge impact on the stability of the DNA molecule, which might not resist to the drying process or to conservation, either at room temperature or 4°C.
Sadly, the construction of these DNA probes took much more time than expected, and we did not manage to try a drying process before the end of the summer.
After this drying we also would have tested a hybridization with targets, to see if the plasmid would still work as expected after drying and conservation at room temperature or 4°C for several days or weeks.
Secondly, we should have tried longer conservation time before transformation, as days are not sufficiently representative of the conservation period required for DNA in laboratories, which ranges from days to decades.
III. RESTRICTION ENZYMES
The last biological elements we needed to conserve in the context of our system were endonucleases. Indeed, to perform detection we had to digest the genomic DNA of the pathogenic bacteria, to target a sequence of its genes that perfectly matches with our probes.
Enzymes and proteins, in general, are temperature-sensitive molecules, that can easily be denatured or degraded if they are not stabilized during temperature changes. For this reason, they are commonly shipped in dry-ice and stored at a temperature below -20°C in laboratories. Such a conservation process was not possible in our automated system, and we had to find a way to conserve the restriction enzymes we needed at 4°C or more.
To achieve this goal, we used trehalose once again. Indeed, this carbohydrate is known to have a huge impact on enzymes thermal stability and more precisely on restriction enzymes stability, allowing their drying and conservation at room temperature [4].
We thus used the protocol described in the article “Extraordinary Stability of Enzymes Dried in Trehalose: Simplified Molecular Biology” (Colaço et al. 1992)[5], and we first tried to dry EcoRI-HF.
Enzymes were dried in a digestion mix containing digestion buffer, sterile water and different concentration of trehalose (except for negative control dried without stabilizer) under a safety cabinet for one or two days. They were then conserved at 4°C or room temperature, rehydrated after one day or after 34 days in sterile water (same volume as start volume, namely 15μL) and used as a digestion mix to digest 1μg plasmid DNA to check their activity. After 1h 37°C digestion, the digestion products were separated by an agarose gel electrophoresis to compare digestion with dried enzymes to undigested plasmids (UD) or plasmid digested by enzymes and buffer conserved at -20°C separately (C+) or together with 0,3M trehalose (C++).
As we can see, there were no significant differences between enzymes samples, which seemed to digest the plasmid with the same specificity (same pattern on the gel). This pattern was clearly different from the undigested plasmid pattern (with the three usual bands obtained for the three forms of an undigested plasmid), which meant that dried enzymes were still active and well-functioning. These results were the same one day and more than one month after drying, as we obtained the same pattern for enzymes conserved in common laboratory conditions (C+ at -20°C) and all dried samples, even samples conserved at room temperature.
Surprisingly, our negative control with enzymes dried without any stabilizer (C-) showed similar results, making think that trehalose might not be useful for restriction enzymes drying at all.
Despite these surprising but positive results, this experiment was not very informative, as it told us a qualitative information (enzymes seemed to work well compared to no digestion), but no quantitative results. What’s more, the wells drying of EcoRI-HF might not be representative of all restriction enzymes, which meant that the drying process might not work efficiently on the endonucleases used in our system.
To pursue, we decide to change the verification protocol to allow a better quantification of enzymes activity and apply it to enzymes used in our system: RsaI, HaeIII and FspI. We dried enzymes with the same protocol, but this time we only tested one concentration of trehalose (0,3M).
To simply evaluate the activity of enzymes, we tried a protocol base on bacterial transformation. The idea was to digest a plasmid chosen to have only one restriction site for a given enzyme, in a known amount and digested for a given time. The digestion products were then transformed into competent bacteria, which allowed to detect the digestion ratio. Indeed, if enzymes worked well, they would digest a huge amount of plasmid, that would be linearized and thus could not be transformed in a bacterium to express its selection marker: there would be only a few colonies on plates. On the contrary, if enzymes are denatured by temperature changes or drying, there are a lot of undigested circular plasmids which can be transformed and lots of colonies to grow on plates.
Thus, this method would allow us to conclude in a quantitative manner and seemed more reliable than just an agarose gel migration.
Due to their really short restriction sites (only 4 nucleotides long), HaeIII and RsaI always had more than one cutting site when searching on few backbones. Such an experiment with these enzymes would not be informative at all, because a digestion on a single of their restriction sites would lead to the linearization of the plasmid, not meaning that enzymes digested every cutting sites. We thus run the experiment with FspI only, and made digestion of pSB4A5_J04450, to allow selection of transformed bacteria on ampicillin medium.
Seven conditions of digestion were tested before transformation, with the same amount of DNA for digestion (1μg) and the same amount extracted from the digestion mix for transformation (47ng):
- plasmid digestion with enzymes and buffer dried together with 0,3M trehalose and conserved at 4°C or room temperature
- plasmid digestion with enzymes and buffer dried together without a stabilizer and conserved at 4°C or room temperature
- plasmid digestion with enzymes and buffer conserved together with 0,3M trehalose in a -20°C freezer
- plasmid digestion with enzymes and buffer conserved separately at -20°C (common laboratory storage conditions) undigested plasmid
Digestion according to these conditions was performed for 1h at 37°C, digestion products were transformed by heat shock in Top10 E. Coli, and bacteria were then spread on LB-agar-ampicillin plates to allow selection. After overnight culture, the number of colonies obtained for each sample was compared to the number of colonies with undigested plasmid, which permits to evaluate the proportion of plasmid digested by enzymes.
As we can see on plates, the difference between FspI dried alone and FspI dried in trehalose appears clearly, in contrast to the experiment with EcoRI-HF. Indeed, we get much more colonies for enzymes dried alone and conserved at room temperature compared to the one conserved at the same temperature but with the addition of trehalose.
The difference, however, seems small between these samples conserved at 4°C. In general, dried enzymes conserved at 4°C and enzymes conserved at room temperature with trehalose obtain similar results as the positive controls conserved at -20°C. In contrast, enzymes dried without a stabilizer and conserved at room temperature obtain results that look more like undigested plasmids.
These results are confirmed by the chart describing colonies numbers. We can see that samples dried with 0,3M trehalose give only a few colonies on plates (58 when conserved at 4°C, 90 at room temperature) compared to the undigested plasmid sample and its 846 colonies. These numbers still are slightly higher than the ones obtained with the positive control without trehalose (39 colonies). However, this difference might be explained by the comparison of both positive controls.
Indeed, the control without carbohydrate seems to allow a better digestion than the one with trehalose (39 colonies versus 56). This may be due to an interaction with trehalose, that might interfere with the enzyme activity. Nevertheless, we considered that this difference was very small and we reckoned it would not be a problem in the context of our system. As a matter of fact, our goal is to digest as much genomic DNA of our pathogenic bacteria as possible, in order to extract the fragment we use for detection. We do not need to reach a total digestion of the entire DNA, but only a sufficient digestion to allow detection.
When looking at the colonies numbers, we can also notice that the sample dried in trehalose and conserved at 4°C is very close to the positive control with trehalose. That means that conservation with trehalose is as effective at -20°C as at 4°C, an improvement which already represents some energy-savings. When conserved at room temperature, enzymes with trehalose nonetheless give higher colonies number, 1.5 times higher than enzymes conserved in trehalose at 4°C or -20°C. Furthermore, this number is really close to the one obtained for enzymes conserved alone at 4°C, then we might suppose that conservation with trehalose at room temperature has the same impact than conservation at 4°C without stabilizer.
Finally, we can see that the negative control without trehalose and conserved at room temperature, has a high number of colonies which draws near the number for undigested plasmids.
This experiment showed us that, despite results obtained with EcoRI-HF, trehalose still seems to have a positive effect on enzymes conservation for days or weeks (in our case 14 days), whether it is conserved at 4°C, or in a lesser extent at room temperature.
IMPROVEMENTS
To improve our results, we must have tried drying of FspI in triplicates or with even more samples. However, the important cost of enzymes and our limited budget has not allowed us to try such an experiment.
We also should have measured the activity of HaeIII and RasI, as the drying results might differ between enzymes.  Finally, our experiments would have been more informative if we performed it after different conservation times between drying and transformation, which would have allowed us to simulate laboratory storage conditions, namely conservation for months or years.
BIBLIOGRAPHY
[1] CROWE, J. H., CROWE, L. M., & CHAPMAN, D. (1984). Preservation of Membranes in Anhydrobiotic Organisms: The Role of Trehalose. Science, 223(4637), 701–703.
[2] Process for producing freeze-dried competent cells and use thereof in cloning, Efrat Barnea, Yael Asscher, Castro Wattad
[3] “Relatively stable molecule”:http://biochemistryrevisited.blogspot.com/2008/01/why-is-dna-and-not-rna-stable-storage.html
[4] Taylor, D. J., Finston, T. L., & Hebert, P. D. N. (1994). The 15% solution for preservation. Trends in Ecology & Evolution, 9(6), 230.
[5] Colaço, C., Sen, S., Thangavelu, M., Pinder, S., & Roser, B. (1992). Extraordinary Stability of Enzymes Dried in Trehalose: Simplified Molecular Biology. Nature Biotechnology, 10(9), 1007–1011.
Igem.grenoble.alpes@gmail.com
