<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Grenoble-Alpes_phage_lysis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Grenoble-Alpes/phage lysis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><SPAN class="error">Template loop detected: <A href="/Template:Grenoble-Alpes" title="Template:Grenoble-Alpes">Template:Grenoble-Alpes</A></SPAN><NAV><UL id="ulmenu"><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes">HOME</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#team-members">TEAM MEMBERS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#advisors">ADVISORS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#partners">PARTNERS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Collaborations">COLLABORATIONS</A></LI></UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/biology">BIOLOGY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Hardware">HARDWARE</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Demonstrate">PROOF OF CONCEPT</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Model">
						MODELING
					</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/actors_partnerships">ACTORS &amp; PARTNERSHIPS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ethic">ETHICS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/events">EVENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Safety">SAFETY &amp; SECURITY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ressources">RESSOURCES</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ressources">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols">PROTOCOLS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/components">COMPONENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/medal_criteria">MEDAL CRITERIA</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/characterization">CHARACTERISATION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/site_map">SITE MAP</A></LI></NAV></P><DIV id="container"><DIV id="welcome"><P id="titre-image">PHAGE LYSIS &amp; DNA EXTRACTION</P></DIV><DIV id="sous-menu"><UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description" id="current-nav">PROJECT </A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/biology" id="current-menu">BIOLOGY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/bacteria_choice">BACTERIA CHOICE</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/selection">TARGET SELECTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/phage_lysis" id="current-el">PHAGE LYSIS &amp; DNA EXTRACTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/construction">PROBE CONSTRUCTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/conservation">CONSERVATION</A></LI></UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Hardware">ENGINEERING</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Demonstrate">DEMONSTRATE</A></LI></DIV><DIV id="text-container"><P>The first main step in our system is the lysis of bacteria by bacteriophages. This step is decisive for all the rest of the system. In fact, this step allows the release of DNA in the medium by lysing the specific bacteria. It is this specific DNA that will be extracted, digested and detected by our plasmid probes. So if the DNA released hybridize to the probes, it gives two essential information:</P><P>
- Which bacterial species are present in the patient sample.</P><P>
- What are the best bacteriophages capable of infecting this bacteria, and thus, treating the patient.</P><H3><FONT size="6"><FONT color="FCBE11"><CENTER>ABOUT BACTERIOPHAGES</CENTER></FONT></FONT></H3><P>But to understand a little more this last point, let’s mention what is a bacteriophage and how it infects bacteria: 
Bacteriophages, also called phages, are bacteria-specific viruses: each bacteriophage is able to infect one or more bacterial strains from a single bacterial species. This specificity is characterized by differences in infection and replication efficiency. This last feature is proportional to the number of lysed bacteria and therefore to the amount of DNA released which will be able to hybridize with the probes. That is how, after the transformation step, we will be able to determine the lysis efficiency by following the fluorescence rate measured by our system. Then we will be able to choose the best bacterium killer bacteriophage.</P><P>However, be careful: some bacteriophages are not just able to lyse bacteria. Indeed, as you see in Figure 1, two types of infection cycles are distinguished: </P><P>
	1) <U>Lytic cycle:</U> the bacteriophage, once inside the bacterial host, uses the machinery of the infected cell to replicate, releasing a great number of new virions after a bacterial burst (in red in Figure 1).</P><P>
	2) <U>Lysogenic cycle:</U> the bacteriophage, once inside the bacterial host, integrates its DNA to the host DNA and becomes a prophage able to transmit itself to the descendants (in blue in Figure 1).</P><FIGURE><CENTER><FIGCAPTION>Figure 1: Infection cycles of bacteriophages   </FIGCAPTION></CENTER></FIGURE><H3><FONT size="5"><FONT color="FCBE11">The lytic cycle</FONT></FONT></H3><P>Lytic bacteriophages, as their name suggests, destroy the bacteria. They divert the bacterial machinery to their advantage to reproduce and multiply.</P><P>
Indeed, bacteriophages, unable to reproduce by their own means, inject their genetic material into host bacteria (red 1. in Figure 1). Thanks to the enzymes and ribosomes of the host, the viral genome can be replicated and translated to form many copies (red 3. in Fig. 1). At the end of the process, the bacteria burst and dozens or even hundreds new bacteriophages - identical to the original - are released in the medium and therefore available to attack other bacteria of the same species (red 4. in the Fig. 1). </P><P>True “professional killers”, lytic bacteriophages are the natural predators of bacteria. It is precisely these lytic bacteriophages that are used for therapeutic purposes to fight against bacterial infections (bacteriophage therapy). For example, in Russia, bacteriophages are used to treat foot infections in patients with type 2 diabetes, known as diabetic foot ulcers and may cause amputation ; as described in the scientific article <I>&quot;Bacteriophage Treatment of Infected Diabetic Foot Ulcers&quot;</I> by scientists of the Institute of Chemical Biology and Fundamental Medicine SB RAS, Laboratory of Molecular Microbiology, Novosibirsk, Russian Federation. Thereby, we also use lytic bacteriophages in our system. </P><H3><FONT size="5"><FONT color="FCBE11">The lysogenic cycle</FONT></FONT></H3><P>Temperate bacteriophages have the property of integrating their genome within the bacterial chromosomes. This phenomenon is called transduction and the bacteriophage cycle is called the lysogenic cycle. Bacteriophages can thus confer new properties to the bacterium, beneficial or not (e.g. virulences genes). For example, lysogenic bacteriophages can “capture” in their genome bacterial genes of interest (resistance to antibiotics, virulence genes…) that they can subsequently propagate within species, or from a species to the other. As a result, bacteriophages contribute to the diversification of bacterial species. </P><P>They can remain quiescent for a long time - such as “dormant agents”, carpeted in the bacterial genome (then called prophages) before starting their reproduction and entering a lytic cycle. These temperate bacteriophages are not used in therapy. During the production of bacteriophages for a therapeutic use, one must even ensure the absence of temperate bacteriophage in the preparation. Although bacteriophages cannot recognize human cells and thus they are considered harmless for human beings, it’s necessary to be careful because bacteriophages are potential gene vectors. These genes can be transferred horizontally to other organism and then, be dangerous to the patient.  </P><P>
In contrast, temperate bacteriophages are valuable adjuncts of molecular biology laboratories, which use them as tools to implant genes into bacteria to modify the genome and allow (for example) the manufacture of molecules by the bacterium, used as “factory”.</P><H3><FONT size="6"><FONT color="FCBE11"><CENTER>HOW TO GET A VERY HIGH TITER OF BACTERIOPHAGES ?</CENTER></FONT></FONT></H3><P>So now, we know how infection cycle works and the importance of bacteriophages. 
But before testing them with their bacterial hosts and extracting DNA, bacteriophages must be prepared! To do this we have to realize 3 steps:</P><P>- bacteriophage amplification <A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols" style="color:grey">[see protocol here]</A></P><P>- bacteriophage purification/concentration <A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols" style="color:grey">[see protocol here]</A></P><P>- plaque assay to determine the bacteriophage titer <A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols" style="color:grey">[see protocol here]</A></P><P>The bacteriophage amplification allows us to get a very high titer of bacteriophages. So we will be able to know if the phage titer influences the lysis efficiency and the amount of extracted DNA. To amplify bacteriophages it is enough to put them with their bacterial host in a liquid medium. To facilitate bacteriophage adsorption we add MgCl2 and CaCl2. Then, bacteriophages infect the bacterial culture and replicate themselves in an impressive number until the death of all the bacteria (Fig. 2). </P><FIGURE><CENTER><FIGCAPTION>Figure 2: Bacteriophage amplification principle   </FIGCAPTION></CENTER></FIGURE><P>Once this done, it is necessary to purify bacteriophages by removing everything that is not needed (bacterial debris in particular). </P><P>For that, we use a NaCl solution which helps bacteriophages to detach from the cell wall and we centrifuge to pellet debris and to keep only the supernatant containing bacteriophages. Then we use a PEG/NaCl solution which helps bacteriophages to crystallize and, after another centrifugation, to pellet <A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols" style="color:grey">[see protocol here]</A>. We can afterward resuspend them in an adequate buffer. This step allows to concentrate the amplified phages and to store them (Fig. 3). The buffer used is named SM Buffer according to the Sodium chloride and the Magnesium sulfate that compose it <A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols" style="color:grey">[see protocol here]</A>. These elements allow the bacteriophages to be stabilized. Then they can be stored at 4°C for a long time.</P><FIGURE><CENTER><FIGCAPTION>Figure 3: Bacteriophage purification principle  </FIGCAPTION></CENTER></FIGURE><P>After these 2 steps, we obtain a very high titered bacteriophage stock. But what does “very high” mean? To ensure that we have succeeded in amplifying bacteriophages and in order to have a quantitative idea of the bacteriophage concentration, we must realize a Plaque Assay (PA) (Fig. 4). Plaque assay protocol is currently considered the “gold standard” to determine phage concentrations. </P><P>
PA allows to detect plaques on a lawn of host bacteria. These plaques are the result of the lysis of the bacterial host by bacteriophages. As a matter of fact, each bacteriophage infects a bacterium and produces new phages which will infect nearby bacteria as well, and then diffuse through the soft agar. This process leads to the apparition of cleared circular zones visible to the naked eye called plaque. </P><P>
In fact, when a bacterium is infected, no other phage can infect it. So, we know that one plaque is the initial infection of a single phage. But to differentiate the plaque from one to another, it is necessary to dilute phages (until we get approximately 1 bacteriophage for 1µL of the sample). In this way, each plaque that comes from one bacteriophage can be distinguished from other plaques.</P><P>
If it is not diluted enough, only a heap of indistinguishable plates will be visible. To ensure that the dilution is sufficient, we recommend to dilute 10<SUP>12</SUP> times in SM Buffer (in serial dilutions) for a basic titer (from a commercial stock for example), and 10<SUP>32</SUP> times if you have previously amplified them.</P><FIGURE><CENTER><FIGCAPTION>Figure 4: Spot titration  </FIGCAPTION></CENTER></FIGURE><FIGURE><CENTER><FIGCAPTION>Figure 5: Result of spot titration of T5 bacteriophage stock  </FIGCAPTION></CENTER></FIGURE><P>In Figure 5, you can see a Plaque Assay of T5 bacteriophages with its bacterial host <I>E. coli</I> F. Only the 10<SUP>-9</SUP> dilution leads to distinguished plaques. We can count approximately 15 plaques. We know that we have spotted 10µL, so we can deduce the titer: </P><P><CENTER><FONT size="4">T =15 pfu/(10<SUP>-3</SUP>mL*10<SUP>-9</SUP>)=15*10<SUP>12</SUP> pfu/mL</FONT></CENTER></P><FIGURE><CENTER><FIGCAPTION> Example of a PA   </FIGCAPTION></CENTER></FIGURE><H3><FONT size="6"><FONT color="FCBE11"><CENTER>EXPERIMENTS &amp; RESULTS</CENTER></FONT></FONT></H3><P>During our experiments, we first worked with T5 bacteriophages and its host <I>E. coli</I> F. </P><P>
We made this choice because <I>E. coli</I> is a bacterium widely used in the laboratory. T5 is a strictly lytic bacteriophage which allows us to get closer to the conditions of our system. As a reminder, we want to detect which are the most effective lytic bacteriophages against the pathogenic bacterium that has infected a patient in order to be able to use them in a bacteriophage therapy.</P><P>
Protocols were perfected until we obtained a very high bacteriophage titer higher than 10<SUP>25</SUP> pfu/mL.</P><P>
Then, we tried to extract and purify the bacterial DNA released after lysis by bacteriophages. For this, we performed the experiment with several bacteriophage titers (the maximum obtained and lower concentrations).</P><P>
For more information about DNA extraction, see explanation <A href="https://2018.igem.org/Team:Grenoble-Alpes/purification_module" style="color:grey">here</A> and protocol <A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols" style="color:grey">here</A>.</P><P>The results of these experiments allowed us to conclude that the phage titer during lysis plays an important role in optimizing the time and yield of this step. Indeed, a very high titer of phages makes it possible to obtain, after a few hours, a concentration of DNA close to the one obtained with a lysis buffer (at time 0) (Table 1).</P><P>In fact, 4 hours after the addition of 50µL of T5 phages in the bacterial samples, we obtained (after extraction and purification with the magnetic beads) an amount of 541.5ng (180.5ng/µL for 30µL of eluant). This amount is satisfactory for the initial quantity of bacteria. Indeed at t=0 we measured an OD at 2.907 which is approximately equal to 1.45*10<SUP>9</SUP> cfu/ml or 2.9 *10<SUP>8</SUP> cfu for each sample of 200µL of <I>E. coli</I>. </P><P>
This results were very motivating and encouraged us to continue the experiments on <I>Pseudomonas</I>. </P><FIGURE><CENTER><FIGCAPTION>Table 1: DNA extraction of <I>E. coli</I> F lysed by T5 bacteriophages   </FIGCAPTION></CENTER></FIGURE><FIGURE><CENTER><FIGCAPTION>Figure 6: Graph showing the concentration of DNA extracted of <I>E. coli</I> F after lysis by T5 bacteriophages depending on three different titers.   </FIGCAPTION></CENTER></FIGURE><P>After that, we wanted to check if similar results could be observed on <I>Pseudomonas aeruginosa</I>, the bacterium that we want to detect for our proof of concept. For that, we worked with <I>Pseudomonas aeruginosa</I> PAO1 strain and one of its specific bacteriophages: HER18.</P><P>
After amplification of the HER18 bacteriophages, we obtained a very high titer of purified bacteriophages exceeding 10<SUP>30</SUP> pfu/mL (Figure 7a). </P><P>
During these experiments, we also tested the specificity of bacteriophages. Indeed phage spots of PAK-P3 (specific to <I>Pseudomonas a.</I> PAK) and T5 were spotted in the plaque assay on a lawn of PAO1 bacteria. No plaque was observed which allowed us to confirm the extreme specificity of the phages (Figure 7b).</P><P>
Because of safety conditions (Class 2 laboratory), we couldn’t take pictures of the plaque assay. Figure 7 displays schematically the obtained results.
</P><FIGURE><CENTER><FIGCAPTION>Figure 7a:  Result of HER18 spot titration   </FIGCAPTION></CENTER></FIGURE><FIGURE><CENTER><FIGCAPTION>Figure 7b:  Result of specificity tests with <I>Pseudomonas a.</I> PAO1   </FIGCAPTION></CENTER></FIGURE><P>After that, we followed the same protocol than with <I>E. coli</I> and T5: DNA extractions after different incubation times (phage + bacteria) with different phage concentrations. The results obtained have once again allowed us to conclude that a very high titer of phages makes it possible to obtain a high yield of DNA in a few hours (Fig. 9, Table 2).</P><FIGURE><CENTER><FIGCAPTION>Figure 8: Graph showing the concentration of DNA extracted of <I>Pseudomonas a.</I> PAO1 after lysis by HER18 bacteriophages depending on two different titers  </FIGCAPTION></CENTER></FIGURE><FIGURE><CENTER><FIGCAPTION>Table2: DNA extraction of <I>Pseudomonas aeruginosa</I> PAO1 lysed by HER18 bacteriophages   </FIGCAPTION></CENTER></FIGURE><P>The quantity of DNA of negative control can be explained by the low vitality of the bacteria (old culture). Indeed it’s possible that they died and released their DNA without phage intervention. </P><P>

The extractions were made exceptionally from 350 μL because the bacteria could not grow. </P><P>
At t=0, the OD was equal to 0.423 which is equivalent to a concentration of 8.8*10<SUP>6</SUP> cfu/mL of <I>Pseudomonas a.</I> PAO1 bacteria. Thus, for each sample of 350µL we had 3.1*10<SUP>6</SUP> cfu to which we have added 50 μL of HER18 bacteriophages at a concentration of 10<SUP>33</SUP> pfu/mL for the titer 1 and 10<SUP>11</SUP> pfu/mL for  the titer 2.</P><P>
Finally, after 4 hours and 30 minutes we obtained, in a sample, a quantity of 2170 ng of DNA (72.2 ng/µL for  30µL of eluant). 
This result (2170 ng of DNA for an initial bacterial quantity of 3.1*10<SUP>6</SUP> cfu) is very satisfactory.</P><P>
Furthermore, they confirmed the results, about the amount of DNA that is possible to obtain and the time necessary to obtain it, previously observed on <I>E. coli</I>. </P><H3><FONT color="#FCBE11"><FONT size="6"><CENTER>CONCLUSION</CENTER></FONT></FONT></H3><DIV style="padding:5px; background-color:#ffffff; border:2px solid #b2b1a6; -moz-border-radius:9px; -khtml-border-radius:9px; -webkit-border-radius:9px; border-radius:9px;"><P>In conclusion, the experiments on <I>E.coli</I> F and T5 as well as on <I>Pseudomonas aeruginosa</I> PAO1 and HER18 allowed us to demonstrate that bacterial lysis by bacteriophages can generate the release of a quantity of DNA close to what is possible to obtain with a lysis buffer.</P><P>
Nonetheless, some additional experiments would have allowed us to confirm our results and go further in the steps of our system. Indeed, the next step would consist in testing the DNA, extracted from the lysis by bacteriophages, with our plasmid probe. Unfortunately, some constraints (time, availability of laboratories) that we  cannot control obliged us to stop here.</P><P>

However, the conclusions of these experiments remain positive: we are able to get a great quantity of DNA thanks to the lysis by bacteriophages. The first step we want to achieve within our system is a success! </P></DIV><DIV style="padding:3px; padding-left:6px; border:1px dotted #d0d0d0; border-left:4px solid #d0d0d0; margin-left:20px;"><H3><FONT size="3">REFERENCES</FONT></H3><P><FONT size="3">[1] M. Poxleitner, W. Pope, D. Jacobs-Sera, V. Sivanathan &amp; G. Hatful, 2017, Phage Discovery Guide. </FONT></P><P><FONT size="3">[2] D. Kim, S. Chung, S. Lee and J. Choi, Relation of microbial biomass to counting units for Pseudomonas aeruginosa, African </FONT></P><P><FONT size="3">[3] Journal of Microbiology Research Vol. 6(21), pp. 4620-4622, 9 June, 2012. </FONT></P><P><FONT size="3">[4] Bonilla, N., Rojas, M. I., Netto Flores Cruz, G., Hung, S.-H., Rohwer, F., &amp; Barr, J. J. (2016). Phage on tap–a quick and efficient protocol for the preparation of bacteriophage laboratory stocks. PeerJ, 4, e2261. doi:10.7717/peerj.2261 </FONT></P><P><FONT size="3">[5] ResearchGate 2018, page consulted on june 2018, https://www.researchgate.net/post/
Phage_precipitation_using_PEG_NaCL_Mechanism_of_TBS_buffer_at_the_second_phase_of_precipitation_and_that_why_we_work_on_ice </FONT></P><P><FONT size="3">[6] F. Ravat, P. Jault, J. Gabart, 2015, Bactériophages et phagothérapie : utilisation de virus naturels pour traiter les infections bactériennes. </FONT></P><P><FONT size="3">[7] R. G. van der Merwe, P. D. van Helden, R. M. Warren, S. L. Sampson and N. C. Gey van Pittius, 2014, Phage-based detection of bacterial pathogens. </FONT></P></DIV></DIV></DIV><FOOTER><A href="mailto:Igem.grenoble.alpes@gmail.com">Igem.grenoble.alpes@gmail.com</A></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>