<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Grenoble-Alpes_construction skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Grenoble-Alpes/construction</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><SPAN class="error">Template loop detected: <A href="/Template:Grenoble-Alpes" title="Template:Grenoble-Alpes">Template:Grenoble-Alpes</A></SPAN><NAV><UL id="ulmenu"><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes">HOME</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#team-members">TEAM MEMBERS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#advisors">ADVISORS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Team#partners">PARTNERS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Collaborations">COLLABORATIONS</A></LI></UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/biology">BIOLOGY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Hardware">HARDWARE</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Demonstrate">PROOF OF CONCEPT</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Model">
						MODELING
					</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/actors_partnerships">ACTORS &amp; PARTNERSHIPS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ethic">ETHICS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/events">EVENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Safety">SAFETY &amp; SECURITY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ressources">RESSOURCES</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/ressources">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/protocols">PROTOCOLS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/components">COMPONENTS</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/medal_criteria">MEDAL CRITERIA</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/characterization">CHARACTERISATION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/site_map">SITE MAP</A></LI></NAV></P><DIV id="container"><DIV id="welcome"><P id="titre-image">PROBE CONSTRUCTION</P></DIV><DIV id="sous-menu"><UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Description" id="current-nav">PROJECT </A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/biology" id="current-menu">BIOLOGY</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/bacteria_choice">BACTERIA CHOICE</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/selection">TARGET SELECTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/phage_lysis">PHAGE LYSIS &amp; DNA EXTRACTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/construction" id="current-el">PROBE CONSTRUCTION</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/conservation">CONSERVATION</A></LI></UL><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Hardware">ENGINEERING</A></LI><LI><A href="https://2018.igem.org/Team:Grenoble-Alpes/Demonstrate">DEMONSTRATE</A></LI></DIV><DIV id="text-container"><P> The following page will describe the lysis and resistance probes construction for <I>Pseudomonas aeruginosa</I>.</P><DIV style="padding:3px; padding-left:6px; border:5px solid #0066cc; margin-left:20px; background-color: #0066cc;"><CENTER><FONT id="optitl"><FONT size="10"><FONT color="white">LYSIS</FONT></FONT></FONT></CENTER></DIV><H3><FONT size="5"><FONT color="#0066cc">1.Plasmid backbone selection </FONT></FONT></H3><P>Among all the backbones proposed by iGEM, we decided to choose one for our probe insertion. In fact, the 2017 IGEM Grenoble team showed by their experiments, that the pSB1C3-BBa_J04450 backbone was the most suitable for the detection of a red fluorescence.</P><FIGURE><CENTER><FIGCAPTION>Figure 1: Backbone tested for hybridization of the target to the probe detecting bacterial lysis   </FIGCAPTION></CENTER></FIGURE><P>The BBa_J04450 biobrick is composed of a RFP gene which is controlled by a LacI promoter inducible by IPTG. Moreover, it has a resistance gene for chloramphenicol to be able to select only bacteria that have been transformed.</P><FIGURE><CENTER><FIGCAPTION>Figure 2: Organisation of the backbone (vector) BBa_J04450 for the cloning.    </FIGCAPTION></CENTER></FIGURE><P>Several teams showed that BBa_J04450 LacI promoter leaks. <A href="https://2017.igem.org/Team:Grenoble-Alpes/LabBook"><FONT color="grey">iGEM Grenoble 2017 </FONT></A> highlights the role of IPTG, at least 180 minutes after induction. IPTG appeared to be important in the detection because it triggers a shorter rise of the fluorescence. As we know, it triggers the transcription of lac operon inducing the expression of the reporter gene.</P><P>

So BBa_J04450 was chosen because this iGEM part enables users to produce mRFP1, a fluorophore which is an engineered mutant of red fluorescent protein from Discosoma striata. This reporter gene is LacI sensitive and can be induced with IPTG. This biobrick was chosen to be able to see fluorescence even if no IPTG was added or adding it to have a more rapid kinetic production.</P><H3><FONT size="5"><FONT color="#0066cc">2. Design of the probe detecting a fragment of P.A characterizing the lysis</FONT></FONT></H3><P>The addition of the bioinformatically designed probe in pSB1C3-BBa_J04450 is done using the Gibson technic. </P><P>

First of all, as a reminder, the DNA fragment to characterize the lysis is :</P><CENTER><FONT size="3"> 5’- CCCTGAACGCCGCCAGCCAGCGCTCCGCCGAGCTGG-3’ </FONT></CENTER><P>The probe has been imagined based on the system of <U><A href="https://2015.igem.org/Team:Cork_Ireland">Cork Ireland 2015 team</A></U> and all the data furnished by New England Biolabs (NEB) website. It contains :</P><UL><LI><P><FONT size="3">Two restriction enzymes producing cohesive end, SphI and NgoMIV. Their goal is to remove the little sequence in between on the bottom strand. Thus creating a perfect complementarity with the target and linearizing the plasmid. But at the end, we trust that using PCR linearization could reduce the background of uncut plasmid.</FONT></P></LI><LI><P><FONT size="3">Two nicking enzymes, Nt.BspQ1 and Nb.BsmI, i.e. enzymes that cut one strand of the double-stranded DNA. Thereby, the top strand is removed, allowing the binding of the target.</FONT></P></LI></UL><FIGURE><CENTER><FIGCAPTION>Figure 3: bioinformatic construction of the lysis detection probe. In grey: random sequence, Blue: SphI and NgoMIV restrictions sites, Highlighted: The detection probe, Green: Nt.BspQI and Nb.BsmI nicking enzymes.    </FIGCAPTION></CENTER></FIGURE><FIGURE><CENTER><FIGCAPTION>Figure 4: Scheme representing the different steps of the in-vitro     </FIGCAPTION></CENTER></FIGURE><H3><FONT size="5"><FONT color="#0066cc">3.  Probe insertion in the psB1C3-BBa_J04450 plasmid - Creation of BBa_K2629000</FONT></FONT></H3><H3><FONT size="4">3.1  Preparation of psB1C3-BBa_J04450</FONT></H3><P>To be able to use the Gibson technic, we had to “prepare” the backbone we chose. To do so, we decided to linearize it by using primers for a PCR. The insertion is between the backbone prefix and the inducible promoter.</P><P>
All the experiments were done by ordering sequences to Integrated DNA Technologies (IDT).</P><FIGURE><CENTER><FIGCAPTION> Figure 5: Localization of the probe insertion     </FIGCAPTION></CENTER></FIGURE><P>This GIF explains the PCR linearization step of the backbone.</P><FIGURE><CENTER><FIGCAPTION> Figure 6: Primers used for the backbone linearization    </FIGCAPTION></CENTER></FIGURE><P>Following the PCR, we decided to digest our product with DPN1. DPN1 only cleaves methylated DNA. As we know, PCR products are not methylated, so the enzyme would just cleave the circularize DNA and so reduce considerably the number of circular DNA plasmid. This is to reduce the number of false positives.</P><P>As we can see on this 1% agarose gel, DPN1 digestion we see that the non digested plasmid has a band at about 3000 bp but also lower bands. When digesting with DPN1, only one clear band can be observed at 3000 bp showing the reduction in the number of non-linearized plasmids.  </P><H3><FONT size="4">3.2. Lysis insert amplification</FONT></H3><P>When using the Gibson technic, DNA fragments contain 20 to 40 base pair overlap with adjacent DNA fragments [1]. Indeed, the technic uses portions of both fragments that are similar so that it fuses them together.</P><FIGURE><CENTER><FIGCAPTION>Figure 7: Gibson technic     </FIGCAPTION></CENTER></FIGURE><P>We first wanted to perform a PCR in order to add the overlap to the probe.</P><FIGURE><CENTER><FIGCAPTION>Figure 8: Addition of the overlap to the insert using PCR     </FIGCAPTION></CENTER></FIGURE><P>After the PCR amplification of the insert it was 143 nucleotides long, so we expected to see on the electrophoresis one band at approximately this size.</P><P>We can see on this electrophoresis for the negative control one strip under 100 nucleotides and for the condition “L” two strips, one at the same size and the other between 100 and 200 nucleotides. We assume that the strips under 100 nucleotides are the primers (approximately 40 nucleotides) and the strip between 100 and 200 nucleotides represent the insert amplification.</P><P>After the overlap addition, we carried out the Gibson technic in order to clone the probe in psB1C3-BBa_J04450.</P><H3><FONT size="4">3.3. Probe insertion</FONT></H3><P>The cloning is performed thanks to the Gibson technic. Indeed, inserts with the overlap are added to the linearized vector and the Gibson Master Mix (this mix contain a 5' exonuclease, a polymerase and a ligase).</P><P>Following this experiment, a colony PCR was done. But this experiment did not show an amplification of the insert in any of the colony we screened. Several hypotheses emerged to explain why the Gibson did not work even though we had pretty good results on the two experiments individually: </P><UL><P><LI><FONT size="3">the PCR to linearize pSB1C3-BBa_J04450 did not work </FONT></LI></P><LI><P><FONT size="4">the PCR to amplify the lysis insert and add the overlap did not work due to the annealing temperatures</FONT></P></LI></UL><P>After trying for a month to linearize the vector, amplify the insert with the overlap and use the Gibson technic without any results. We had to come up with another approach so we decided to design the insert already containing the overlaps and order it to IDT:</P><FIGURE><CENTER><FIGCAPTION>Figure 9: Double stranded insert already containing the overlap ordered on IDT     </FIGCAPTION></CENTER></FIGURE><P>As the insert is not similar to the primers we had designed for the colony PCR anymore, we had to do a digestion of the DNA from the colony newly transformed with the new insert. The digestion is done using NgoM IV. </P><P>As we can see on this electrophoresis gel, after colony PCR, the undigested extractions have the 3 forms in which you can find plasmid DNA (circular, coiled and supercoiled). When it comes to the digested ones, we only see one band at around 3000 bp which shows that the plasmid has been linearized by the NgoMIV enzyme. We can conclude about the presence of the insert into our vector as NgoMIV restriction site was only present in the insert’s sequence.</P><P>The product obtained was kept to use it later for the probe activation process.</P><H3><FONT size="5"><FONT color="#0066cc">4.  Probe activation</FONT></FONT></H3><P>The first approach was to use 4 digestions as told before.</P><H3><FONT size="3"> 4.1.  By four digestions</FONT></H3><P>As the insert contains :</P><UL><P><LI>Two restriction enzymes producing cohesive ends, SphI and NgoMIV. </LI></P><P><LI>Two nicking enzymes, Nt.BspQ1 and Nb.BsmI</LI></P></UL><P>The first approach was to use those enzymes in the activation process of our probe. Indeed, SphI and NgoMIV are used to create a perfect complementarity with the target and linearizing the plasmid. And use Nt.BspQ1 and Nb.BsmI to remove the top strand and allow the binding of the target.</P><P>However, we thought it would be better to use the PCR technic for the linearization because it could reduce the background of non-linearized plasmids. </P><FIGURE><CENTER><FIGCAPTION>Figure 10: Scheme representing the different steps of the in-vitro construction of the probe.
  </FIGCAPTION></CENTER></FIGURE><H3><FONT size="3"> 4.2.  By PCR linearization and two digestions</FONT></H3><P>To use PCR for the linearization of our plasmid, we had to order primers to IDT: </P><FIGURE><CENTER><FIGCAPTION>Figure 11: Primers used for the PCR linearization of the probe     </FIGCAPTION></CENTER></FIGURE><P>Regarding the removal of the top DNA strand, we still use the Nt.BspQ1 and Nb.BsmI. </P><H3><FONT size="5"><FONT color="#0066cc">5.  Detection of the target</FONT></FONT></H3><H3><FONT size="3">     5.1- Transformation tests </FONT></H3><P>Following the activation of the probe by PCR linearization, the detection probes were cleaned up (PureLink Quick PCR Purification kit by Thermofisher Scientific) to eliminate the top strand fragment just removed from the plasmid. The yield following this experiment was very low so we were losing a lot of DNA. The efficiency of the detection probe was checked using design targets from IDT.   </P><P>We did have colonies but not the results we expected, we decided to proceed on the sensibility measure in order to test different ratios probe/target.  </P><H3><FONT size="3">    5.2 - Confirmation by sequencing </FONT></H3><P>In parallel of the experiments, we sent our newly designed probe to sequencing. Unfortunately, the sequencing results we obtained were not the one expected so we had to erase the biobrick from the part registry.   </P><DIV style="padding:3px; padding-left:6px; border:5px solid #0066cc; margin-left:20px; background-color: #0066cc;"><CENTER><FONT id="optitl"><FONT size="10"><FONT color="white">RESISTANCE</FONT></FONT></FONT></CENTER></DIV><H3><FONT size="5"><FONT color="#0066cc">1. Plasmid backbone selection</FONT></FONT></H3><P>When detecting a marker of a bacteria’s antibiotic resistance we decided to use a reporter gene: the Blue Fluorescence Protein (BFP). We found a biobrick, BBa_K592027, containing this reporter gene (BBa_K592100) in the iGEM backbones. However, the data on the biobrick showed better results in another backbone and needs more than 48h to be able to transcript the BFP. This is why we decided to create a new plasmid. </P><P>Indeed, we know the promoter, RBS and terminator of BBa_J04450 work well. So the RFP gene is replaced from psB1C3-BBa_J04450 by the BFP gene from the psB1C3-BBa_K592027. The experiment is explained later. </P><FIGURE><CENTER><FIGCAPTION>Figure 12: Overall view on how the new biobrick BBa_K2629002 is designed     </FIGCAPTION></CENTER></FIGURE><H3><FONT size="5"><FONT color="#0066cc"> 2. Design of the probe that detects a market of resistance of Pyo </FONT></FONT></H3><P>The probe is bioinformatically designed using the same strategy as for the probe design.</P><P>First of all, as a reminder, here is the target to characterize the resistance: </P><CENTER><FONT size="5"> 5’ - ACCACCCGCACGGCGACATCGCGGTCTACAACACCATCGTGC - 3’ </FONT></CENTER><P>The probe has been thought based on the system of Cork Ireland 2015 team and all the data furnished by New England Biolabs (NEB) website. The probe contains :</P><UL><P><LI><FONT size="3">Two restriction enzymes, SphI and NgoMIV, whose goal is to remove the little sequence between on the bottom strand and thus create a perfect complementarity with the target. But at the end, we trust that using PCR linearization could reduce the background of uncut plasmids.</FONT></LI></P><P><LI><FONT size="3">Two nicking enzymes, Nt.BspQ1 and Nb.BssSI, which are enzymes that cut one strand of the double DNA strand. Thereby, the top strand is removed, allowing the target to bind.</FONT></LI></P></UL><FIGURE><CENTER><FIGCAPTION>Figure 13: bioinformatic construction of the resistant detection probe. In grey: random sequence, Bleu: SphI and NgoMIV restrictions sites. Pink: Nt.BspQI and Nb.BssSI nicking enzymes, Highlighted: The detection probe, Red: mutations   </FIGCAPTION></CENTER></FIGURE><FIGURE><CENTER><FIGCAPTION>Figure 14: Scheme representing the different steps of the in-vitro construction of the probe. </FIGCAPTION></CENTER></FIGURE><H3><FONT size="5"><FONT color="#0066cc">3.  Probe insertion in psB1C3- BBa_J04450 </FONT></FONT></H3><H3><FONT size="3">    3.1. Creation of BBa_K2629001 </FONT></H3><H3><FONT size="3"><FONT color="grey">        3.1.1. Linearization of psB1C3-BBa_J04450 </FONT></FONT></H3><P>The first step is the linearization of the psB1C3-BBa_J04450 at a very specific place. To do so, a PCR is performed with specific primers. </P><P>
In order to optimize the experiment, we decided to realize an enzymatic digestion with DPN1. DPN1 cleaves only when it's recognition methylated site. Knowing that PCR products are not methylated, it will reduce considerably the number of false positive. </P><P>For the continuation, the same lot as presented in Figure X was used.</P><H3><FONT size="3"><FONT color="grey">        3.1.2. Probe resistance insert amplification with the addition of the overlaps </FONT></FONT></H3><P>Like the lysis insert, the Gibson technic is used in order to realize the cloning. Thanks to a PCR, overlaps were added to the insert that will enable it to fuse with the linearized plasmid. Indeed, the technic uses portions of both fragments that are similar so that it can fuse them together.</P><P>Let us remind you the probe of 93bp (172bp after the PCR)  : </P><FIGURE><CENTER><FIGCAPTION>Figure 15: PCR amplification strategy with addition of the overlaps of probe resistance insert     </FIGCAPTION></CENTER></FIGURE><P>On the following electrophoresis, we can see two strips for the negative control. One between 100 and 200 nucleotides and the other under 100. The strip under 100 nucleotides represent the primers for the PCR. However, the strip between 100 and 200 nucleotides should not be here. This is due to a contamination following a mishandling.
For the “RI” condition, one strong strip is observed between 100 and 200 nucleotides. 
This represents the insert. Indeed, its size is 172 nucleotides. </P><P>The vector and insert are now ready for the Gibson!</P><H3><FONT size="3"><FONT color="grey">           3.1.3. Probe insertion/ Cloning of the probe into psB1C3-BBa_J04450 </FONT></FONT></H3><P>The cloning is performed using the Gibson technic. Indeed, we amplified the insert and prepared the linearized plasmid. For this Gibson a 1:5 ratio was used. A bacterial transformation was done after the Gibson. Following by a DNA extraction and finally, in order to make sure we inserted the recognition sequence, an enzymatic digestion was done with NgoMIV for which the restriction site is localized into the insert. </P><FIGURE><CENTER><FIGCAPTION>Figure 16: Right- before Gibson technic with no NgoMIV restriction site. Left - after Gibson technic that brings one NgoMIV restriction site (the violet star).
     </FIGCAPTION></CENTER></FIGURE><P>This electrophoresis was performed in order to confirm that the Gibson technic works. We can see for the conditions “CUD” and “CD” the same profile. Indeed, the plasmid doesn’t have NgoMIV restriction site and so the probe wasn’t inserted. Then, for the experimental conditions (1 to 3) we can observe, in the cases of digestion, one strip at 3000 bp. This is what we expected. In fact, by adding NgoMIV restriction site the enzyme cuts one time in the plasmid proving the insertion of the probe.</P><P>Thanks to these results, we can confirm that the BBa_K2629001 worked.</P><H3><FONT size="3"> 3.2. Creation of BBa_K2629003 </FONT></H3><P>After adding the resistance probe, the RFP gene of BBa_J04450 is substituted by the BFP gene of BBa_K592027. </P><H3><FONT size="3"><FONT color="grey">       3.2.1.  BFP amplification </FONT></FONT></H3><P>At first, a PCR amplification of the BFP was realized. To do so, psB1C3-BBa_K592027 was extracted from the plate. Then, the BFP gene (705bp) was amplified thanks to primers. The 16th figure  describes the strategy for the BFP amplification.</P><FIGURE><CENTER><FIGCAPTION>Figure 17: PCR amplification strategy of BFP gene  </FIGCAPTION></CENTER></FIGURE><P>We can see on this electrophoresis, for the negative control, one strip at approximately 700 bp. The strip should not be there, this might be due to a contamination following a mishandling. 
For the “BFP” condition, we can observe several strips and a  strong one at approximately at 700bp. This represents the amplified BFP gene. </P><H3><FONT size="3"><FONT color="grey">  3.2.2.  BBa_K2629001 linearization </FONT></FONT></H3><P>After the insert’s amplification, we proceed with the linearization of the vector using a PCR. However, previously, several steps have been carried out. </P><UL><LI><P><FONT size="3">Linearization of the plasmid</FONT></P></LI><LI><P><FONT size="3"><B>Knockout</B> of the RFP gene into the psB1C3-BBa_J04450 plasmid</FONT></P></LI><LI><P><FONT size="3">Adding the overlap at the extremities for the Gibson</FONT></P></LI></UL><FIGURE><CENTER><FIGCAPTION>Figure 18: PCR linearization with addition of the overlap of psB1C3-BBa_K2629001   </FIGCAPTION></CENTER></FIGURE><P>We can see on this electrophoresis for the negative control one strip under 100 nucleotides. This represents the primers used for the PCR. For the “JR” condition we can observe one strip at approximately 2000 bp. This strip is the BFP insert. </P><P>Now the vector and insert are ready for the Gibson. </P><H3><FONT size="3"><FONT color="grey"> 3.2.3. Cloning of BFP gene into BBa_K2629001</FONT></FONT></H3><P>The cloning is performed using the Gibson technic. Indeed, we amplified the insert and prepared the linearized plasmid. For this Gibson a 1:3 ratio was used. After, a transformation was done in order to realize a colony PCR to verify the success of the cloning. </P><P>We can see on this electrophoresis for the positive control one strip at approximately 700 bp. This is the amplified BFP gene. For the negative control, we can observe one strip at 700 too but it should not be here. This is probably a contamination. </P><P>
Regarding the CFU conditions, we can observe one strong strip at 700 bp. This is certainly the BFP gene. </P><P>In order to be sure, a RsaI digestion was done because the BFP gene brings new restrictions sites.</P><P>This electrophoresis was done in order to confirm the presence of the BFP gene. We chose to realize a RsaI digestion because there are 4 restriction sites have been added in the BFP gene. </P><P>
Without the BFP gene we expected to see strips at approximately 2000 bp, 1000 bp, 530 bp, 200 bp and 100 bp (DC). With the addition of the BFP gene we expected to see 2 strips added at 300 and 450 bp (D1 and D2). </P><P>
This is what we see on this electrophoresis. Therefore, we can confirm the presence of the BFP gene in psB1C3-BBa_K2629001.</P><H3><FONT size="5"><FONT color="#0066cc">4.  Probe activation</FONT></FONT></H3><P>Two strategies were tested to build the probe in vitro to activate it. In order to have more results, both psB1C3-BBa_K2629001 (RFP)  and psB1C3-BBa_K2629003 (BFP) were used for the probe construction. </P><H3><FONT size="4">4.1. Construction by four digestions</FONT></H3><FIGURE><CENTER><FIGCAPTION>Figure 19: Scheme representing the different steps of the in-vitro construction of the probe.  </FIGCAPTION></CENTER></FIGURE><P>All the enzymes were purchased from New England Biolabs. Each digestions have been made 4h at the temperature recommended by the supplier. 
</P><H3><FONT size="4">4.2. By PCR linearization and two digestions</FONT></H3><P>Other methods have been tested in order to find one more efficiency. The second one works with the same nicking enzymes but the first two digestion are replaced by a PCR linearization. 
</P><FIGURE><CENTER><FIGCAPTION>Figure 20: Primers used for the probe linearization  </FIGCAPTION></CENTER></FIGURE><H3><FONT size="5"><FONT color="#0066cc">5.Detection of the target </FONT></FONT></H3><H3><FONT size="4">5.1. Transformation tests</FONT></H3><P>After the activation of the probe by digestion or PCR linearization, the detectors were cleaned up (PureLink Quick PCR Purification kit by Thermofisher Scientific) to remove the top strand of the probes. Efficiency of the detectors was checked with pure targets.
</P><H3><FONT size="4">5.2.  Confirmation by sequencing</FONT></H3><P>Since bacterial transformation does not display full indications of what really happened, sequencing was done to verify the binding of the single strain target to the detector. Four colonies of each previous plate were picked up and cultivated, to extract DNA, which is then sequenced thanks to a unique forward primer (VF2) that binds close to the insertion site of the target. Theoretically, the collected colonies that contain the different plasmid construction. That is why we compared the result of the sequencing with the probe sequence (for BBa_K2629001 &amp; BBa_K2629003) as well as the sequence of the BFP gene (for BBa_K2629003).</P><FIGURE><CENTER><FIGCAPTION>Figure 21: sequencing result. (A): alignment between psB1C3-BBa_K2629001 and the probe after probe activation by digestion. Subject: probe sequence; Query: sequencing result. (B): alignment between psB1C3-BBa_K2629003 and the BFP gene.Subject: BFP gene; Query: sequencing result. (C): alignment between psB1C3-BBa_K2629003 and the probe after probe activation by PCR linearization. Subject: probe sequence; Query: sequencing result.  Primers used for the probe linearization  </FIGCAPTION></CENTER></FIGURE><P>The figure 20A shows the sequence alignment between psB1C3-BBa_K2629001 and the probe after probe activation by digestion. We can observe that the alignment is not completely perfect. Indeed, we can see 3 mutations on the sequencing result. These mutations may correspond to either sequencing error or true mutations. Moreover, the figure 20C shows the sequence alignment between psB1C3-BBa_K2629003 and the probe after probe activation by PCR linearization. We can observe no sequence alignment.</P><P>
Unfortunately, this is not the result that we expected. In fact, thanks to the sequencing result, we can say that the probe wasn’t inserted into psB1C3-BBa_K2629003. It means that the probe construction did not work. Also, we can say that the probe insertion for psB1C3-BBa_K2629001 works but not perfectly. Afterward, the probe construction did not work. There are several explanations that why the construction of the probe did not work. Perhaps the linearization step of the backbone, the insertion of the probe in the backbone or the digestion for the activation of the probe did not work. Regrettably, we did not have the time to make more sensibility tests and more activations.</P><P>
However, the substitution of RFP gene by BFP gene did work well as we can see one the figure 20B, the sequence alignment is perfect.</P><H3><FONT size="4">5.3.  Sensibility and Specificity evaluation</FONT></H3><P>An important aspect in the diagnosis is to estimate sensitivity and specificity. These two parameters estimate the capacity of a system to deliver a result with low false positive cases (positive result while the patient is not ill) and low false negative cases (negative result while the individual is actually sick). </P><P><B>Sensitivity</B></P><P>Sensitivity measures the limit of detection of our test. Here, it is defined by the lowest detectable amount of Pseudomonas aeruginosa that the system can detect to confirm the presence of the pathogen. To estimate sensitivity, the target amount was gradually increased (0 ng; 5 ng; 25 ng; 50 ng; 100 ng and 200 ng), while the detector amount was kept constant (100 ng). </P><P>
As explained before, colonies were observable but we did not know if it was true positive or false positive. Indeed, for the condition 0 ng of the target, a few colonies were present and for the 200 ng of target conditions about thirty colonies were visible. To check if it was false negative we looked at the results of the sequencing to conclude on the probes construction.</P><P> 
These results are coherent with the sequencing result. In fact, we observed that the probes were not activated despite the digestions. It is possible that the detector was not well digested and consequently not well transformed. The biggest problem is that not enough colonies were screened to show that one of them had the detector that received the target.</P><P><B>Specificity</B></P><P>Another limitation driven by the kit is the purity of the sample. Indeed, the detection occurs when the target is mixed with a lot of foreign and unknown DNAs. </P><P>
To estimate specificity, i.e. the ability of the detector to identify the true positive, the detector has to be tested with “false target sequences”, more or less homologous to the original targets. To do so, an algorithm was made by iGEM Grenoble 2017 to give random sequences with 5%, 15%, 25% and 50% randomly modified pairs of nucleotides (length is kept at 42bp). In addition, the probe detects a DNA fragment with mutations causing resistance. As a result, three other controls have been added:</P><UL><LI><P><FONT size="4">One without the 2 mutations</FONT></P></LI><LI><P><FONT size="4">One with the first mutation</FONT></P></LI><LI><P><FONT size="4">One with the second mutation</FONT></P></LI></UL><P>Unfortunately, we did not have the opportunity and the time to carry out these experiments.</P><P>

You can find all the protocols we used in the “protocols” part.</P><H2><FONT color="#0066cc"><CENTER>Comments on the work we did</CENTER></FONT></H2><DIV style="padding:5px; background-color:#ffffff; border:2px solid #0066cc; -moz-border-radius:9px; -khtml-border-radius:9px; -webkit-border-radius:9px; border-radius:9px;"><P>Inspired by the great work done by <A href="https://2015.igem.org/Team:Cork_Ireland"> Cork Ireland iGEM 2015 </A> team followed by the work of <A href="https://2017.igem.org/Team:Grenoble-Alpes">Grenoble-Alpes iGEM 2017</A>, we tryed to design a similar diagnostic tool able to detect a specific sequence of a given bacteria and thus enable to determine whether a patient is infected or not with the bacteria and whether this bacteria is resistant to antibiotics. The detection probes have to be inserted in the automated device we designed. </P><P>
The first detection probe aims to characterize the lysis of the bacteria so here ProC is detected. Besides, for the detection probe of eventual resistant markers, we used the GyrA gene. After talking to several experts we understood our GyrA probe will not be able to discriminate the mutated probes from the natural ones. So we decided to design it do the experiments on it but knowing that it may not be as relevant as we thought it would. </P><P>
Indeed, we could only find 2 single nucleotide mutations (SNPs) characterizing the GyrA mutation and thus, even the unmutated fragment should normally be able to match to the probe we design with the complementary nucleotides to the mutation. We can take the example of miss-match which could occur here. Discrimination using 2 mutations is not enough here. </P><P>
In the first moment, both clonings were successful thanks to the Gibson technic when looking at the electrophoresis gels but, when we characterize them with sequencing only the resistance probe worked out. </P><P>
Different biobricks:</P><P>-psB1C3-BBa_K2629000: probe that characterizes the lysis of the pathogenic bacteria.</P><P>-psB1C3-BBa_K2629001: probe that characterizes a marker of resistance containing the RFP gene.</P><P>-psB1C3-BBa_K2629003:  probe that characterizes a marker of resistance containing the BFP gene.</P><P>The biobricks were then activated by digestions combined with PCR linearization. After the activation, the detectors were tested in order to evaluate their efficiency. Sensitivity tests were attempted. All the transformation results were sent to sequencing in order to prove that the probes detect the target. Unfortunately, we noticed that only the biobrick psB1C3-BBa_K2629001 contained the probe but not psB1C3-BBa_K2629000 and psB1C3-BBa_K2629003. Even though, the biobrick psB1C3-BBa_K2629003 does not have the detection probe it has the BFP gene replacing the RFP, so one part of the cloning worked. This means that the biobrick can be used for another utilization but needs to be further characterized.</P><P>
Importantly, a statistical job has to be done for each step of the detector construction. The process has to be standardized as well as the characterization that has to be completed by another method of construction and sensitivity/specificity tests. Ideally, comparison with a gold standard would be done. </P><P>To conclude, we would like our work to inspire other teams to maybe design probes against other pathological bacteria. Indeed, the purpose of our project is to detect pathological bacteria and eventually determine their resistance to an antibiotic. It would be a great opportunity to work together on fighting antibiotic resistance. </P><P><I>So what are you waiting for?</I> Come and be part of our small contribution to this big issue we are facing today you won’t regret it!</P></DIV><DIV style="padding:3px; padding-left:6px; border:1px dotted #d0d0d0; border-left:4px solid #d0d0d0; margin-left:20px;"><H3><FONT size="5">REFERENCES</FONT></H3><P><FONT size="3">[1] Nat Methods. 2009 May;6(5):343-5. doi: 10.1038/nmeth.1318. Epub 2009 Apr 12. Enzymatic assembly of DNA molecules up to several hundred kilobases.</FONT></P></DIV></DIV></DIV><FOOTER><A href="mailto:Igem.grenoble.alpes@gmail.com">Igem.grenoble.alpes@gmail.com</A></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>