<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ColumbiaNYC_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ColumbiaNYC/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Columbia iGEM 2018</TITLE></P><DIV class="column full_size"><H1>Description</H1><P>Chlamydia and gonorrhea are the two most common sexually transmitted bacterial infections in the world. They are caused by the bacteria <I>Chlamydia trachomatis</I> and <I>Neisseria gonorrhoeae</I>, respectively, affecting millions of people each year. Current methods of detection require the presence of substantial medical resources, as well as a medical professional with the expertise to administer the test. Our proposed detection system can be used in any location, requires no expertise to read, is easily transportable, can be used at one’s own discretion, and is affordable enough for nonprofessional purchase. This makes it especially appealing for those who travel frequently, live in underserved communities, and live in areas that are far away from medical resources. In order to contribute to the effort against these infections, we set out to develop a novel diagnostic test for <I>C. trachomatis</I> and <I>N. gonorrhoeae</I> based on the CRISPR-Cas system. We selected these two pathogens because they are often diagnosed concurrently.

</P><H1>Design</H1><H3>CRISPR-Cas13a</H3><P>The cas proteins Cas12a and Cas13a1, also known as Cpf1 and C2c2, respectively, have been characterized in recent years. In their natural state, they work by detecting particular sequences of dsDNA and ssRNA, respectively, using CRISPR-RNA, or crRNA. Once these sequences have been detected, the cas proteins engage in promiscuous collateral cleavage of ssDNA and ssRNA, respectively, which we will detect using various methods. 
The Zhang Lab at the Broad Institute has developed a system called SHERLOCK (Specific High Sensitivity Enzymatic Reporter UnLOCKing), which detects collateral cleavage of RNA due to CRISPR-Cas13a. Upon recognizing and cleaving target RNA sequences of the invading pathogen, Cas13a collaterally cleaves other RNA sequences including a modified RNA oligonucleotide tagged with a fluorophore and a quencher substructure. Once cleaved, a fluorescent signal is produced, indicating that the target RNA was detected. Our team wanted to harness this CRISPR-Cas system to detect <I>Chlamydia trachomatis</I> and <I>Neisseria gonorrhoeae</I>.

</P><H3>Genes of Interest</H3><P>For each pathogen, we selected genes that were unique to the pathogen to prevent false positives. For <I>Chlamydia trachomatis</I> we chose the <I>ompA</I> gene, which codes for the major outer membrane protein, a “key antigen” of the organism. According to the NIH application BLAST, <I>ompA</I> contains a unique sequence found exclusively in many, if not all, strains of <I>C. trachomatis</I>. For <I>Neisseria gonorrhoeae</I>, we chose the <I>porA</I> pseudogene. 

</P><H3>Detection Method</H3><P>In order to build a cheap, field-deployable, CRISPR-based diagnostic tool, we implemented an <I>Escherichia coli</I> ­based transcription-translation (TX-TL) cell-free expression system. This self-contained diagnostic platform encodes all components for diagnosis, from detection to readout, in a cell-free solution. To supplement our diagnostic platform, we have designed a ratiometric read-out that takes advantage of the preferential cleavage activity of CRISPR-Cas13a. This detection system consists of two different chromoproteins that are expressed and degraded at the same rate. One chromoprotein will be codon-optimized to contain many motifs favored for collateral cleavage by Cas13a, while the other chromoprotein will be codon-optimized to contain little, if no, motifs and will thus be more protected from such cleavage. Upon detection of the gene of interest, Cas13a will exhibit collateral cleavage activity upon mRNAs of both chromoproteins, resulting in a shift in color from an intermediary one.

</P><H3>Sources</H3><P> 
Gootenberg, Jonathan S., et al. “Multiplexed and Portable Nucleic Acid Detection Platform with Cas13, Cas12a, and Csm6.” <I>Science</I>, vol. 360, no. 6387, 2018, pp. 439–444. 

Gootenberg, Jonathan S., et al. “Nucleic Acid Detection with CRISPR-Cas13a/C2c2.” <I>Science</I>, vol. 356, no. 6336, 2017, pp. 438–442.

Nunes, A., et al. “Evolutionary Dynamics of OmpA, the Gene Encoding the Chlamydia Trachomatis Key Antigen.” <I>Journal of Bacteriology</I>, vol. 191, no. 23, 2009, pp. 7182–7192.

Sashital, Dipali G. “Pathogen Detection in the CRISPR–Cas Era.” <I>Genome Medicine</I>, vol. 10, no. 1, 2018.

Shmakov, Sergey, et al. &quot;Discovery and Functional Characterization of Diverse Class 2 CRISPR-Cas Systems.&quot; <I>Molecular Cell</I>, vol. 60, no. 3, 5 Nov. 2015, pp. 385-97.

WHO Guidelines for the Treatment of Chlamydia Trachomatis. World Health Organization, 2016.

WHO Guidelines for the Treatment of Neisseria Gonorrhoeae. World Health Organization, 2016.

Zetsche, Bernd, et al. &quot;Cpf1 Is a Single RNA-Guided Endonuclease of a Class 2 CRISPR-Cas System.&quot; <I>Cell</I>, vol. 163, no. 3, 22 Oct. 2015, pp. 759-71.
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>