<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ASIJ_Tokyo_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ASIJ Tokyo/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="tutorial"><UL><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo" style="">HOME</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Team">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Collaborations">COLLABORATION</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Attributions">ATTRIBUTIONS </A></LI></UL><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Description">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/CRISPR">CRISPR</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Design">DESIGN</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Experiments">EXPERIMENT</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Results">RESULTS</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Improve">IMPROVE</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Activities">OUR ACTIVITIES</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Questions">Q&amp;A</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Future">FUTURE</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Safety">SAFETY</A></LI></DIV></DIV><DIV id="wrapper"><H1 id="header"> EXPERIMENTS </H1><DIV id="everything"><H4>Part I : <A href="https://static.igem.org/mediawiki/2018/c/cc/T--ASIJ_Tokyo--crisprpart1.pdf" target="none"> CRISPR</A></H4><H4>Part II : <A href="https://static.igem.org/mediawiki/2018/3/30/T--ASIJ_Tokyo--crisprpart2.pdf" target="none">CRISPR Confirmation (genotypic)</A></H4><H4>Part III: Designing of the new constructs </H4><H4>Part IV:  <A href="https://static.igem.org/mediawiki/2018/f/fd/T--ASIJ_Tokyo--gibson.pdf" target="none">Gibson Assembly of Constructs</A></H4><H4>Part V: <A href="https://static.igem.org/mediawiki/2018/4/47/T--ASIJ_Tokyo--bevaviorcontrol.pdf" target="none"> Behavior Control Lab Protocol</A><UL class="reason"><LI> To confirm our hypothesis regarding the secretion behavior of A1AT in cells as it differs between the M allele and ZZ allele we conducted a behavior control lab. Through our research, we discovered that antitrypsin of the SERPINA1 E342K (ZZ allele) with point mutation variant polymerizes within the cell, thus resulting in a greater cell mass as the protein is unable to be excreted. A protein of the SERPINA1 (MM) variant will successfully exit the cell, therefore resulting in lesser cell mass. To observe this behavior in our own scenario, each construct was tagged with a distinctly different colored fluorescence reporter, enabling us to see a qualitative difference in where the fluorescence, and therefore the protein, was located. In our experiment, the pattern in the location of fluorescence, due to the mass of the cells, followed the expected behavior. We also tried different fluorescence markers and compared the scenarios of an unmutated protein and a protein we conducted CRISPR on. The completion of the behavior control lab confirmed our hypothesis, provided a qualitative confirmation of the mutationâ€™s successful correction, and decided the fluorescence marker used throughout the rest of the expirement.. </LI></UL></H4><H4>Part VI: Genotypic and Phenotypic Proof of Concept<UL><LI>a. Agarose gel electrophoresis of plasmid constructs (agarose gel electorphoresis protocol)</LI><UL class="reason"><LI> We conducted the electrophoresis as a way to collect more evidence to establish that the CRISPR edited SERPINA1 E342K with point mutation construct was edited successfully and determine if the proteins produced behaved like the unmutated A1AT. The mutated construct, prone to polymerization is expected to have a larger weight, and thus travel less in the gel. On the other hand, the unmutated construct, with no polymerization, is expected to have a lighter molecular weight and thus travel farther in the gel. This method was chosen because the materials were readily available to us and many members of the team have performed it before. 

 </LI></UL><LI>b. Phenotypic Proof (Transformation and growing on selection plates)</LI></UL></H4><H4>Part VII:  Quantifying Protein<UL><LI>a.  <A href="https://static.igem.org/mediawiki/2018/a/a7/T--ASIJ_Tokyo--oggfp.pdf" target="none">GFP protein purification using column chromatography</A></LI><UL class="reason"><LI> Our goal was to isolate A1AT secreted by all constructs so the protein can be accurately analyzed without obstructions. Two methods of purification, as well as several protein tags, were tested out. The two methods or column chromatography were His-tag specific protein purification with nickel column tubes and others were purified using the standard column chromatography method. Utilizing the His-tag was a major player in our success. We purified the proteins before conducting GFP fluorescence readings, the sialic acid assay (ELISA test), and gel electrophoresis. 
 </LI></UL><LI>b.  <A href="https://static.igem.org/mediawiki/2018/5/56/T--ASIJ_Tokyo--ogsialic.pdf" target="none">Sialic Acid Assay - original version of protocol </A></LI><UL class="reason"><LI> The sialic acid content of a cell is a major characteristic differentiating mutated and unmutated variants. Therefore we chose to conduct a sialic acid assay as a means by which to attain quantitative results to determine with greater accuracy if the gene edit was successful. Based on previous literature, we determined that the unmutated constructs would secrete more sialic acid. The mutated constructs were expected to secrete less, due to its tendency to polymerize in the cell, preventing its ability to exit the cell. This assay would allow us to determine the sialic acid levels for each construct and thus allow us to compare the CRISPR edited construct with the unmutated construct and determine based on the proximity of the sialic acid contents if our gene edit was successful. If successful, the CRISPR edited constructs would show sialic acid secretion similar to that of the unmutated construct. 
 </LI></UL></UL></H4><H4>Part VIII: Incorporation of New information to use His-tag<UL><LI>a. Redesign construct to include His-tag</LI><LI>b. Gibson assembly </LI><LI>c. Transformation  </LI><LI>d. <A href="https://static.igem.org/mediawiki/2018/2/2b/T--ASIJ_Tokyo--histag.pdf" target="none">Protein Purification using His Tag Protein Chromatography - 
        redone  Oct 10  (10 trials)</A></LI><LI>e. <A href="https://static.igem.org/mediawiki/2018/8/84/T--ASIJ_Tokyo--esla.pdf" target="none">Sialic Acid Purification redone using new protocol (6 trials) </A></LI><LI>f. Fluoroscence Microplate Reading to Quantify Protein Concentration of Constructs
        using GFP standard curve </LI><UL class="reason"><LI> In another attempt to determine the optimal construct and determine whether the mutation was successfully corrected, we quantified the secretion of A1AT through a GFP reporter system. To create a standard curve showing GFP concentration and relative fluorescence intensity, we used concentrations of 0 uM, 8 uM, 16 uM, 32 uM, and 40 uM GFP in 1 mL PBS. Based on this curve, we took our fluorescence readings from our different constructs and used the equation Relative Fluorescence Intensity= - 0.015 + 0.007625(GFP Concentration), with R2 = 0.991927, to determine the concentration of GFP from the fluorescence intensity measured from each construct. </LI></UL><LI>g. Fluorescence Microplate Reading to Quantify Sialic Acid Concentration 
        using Sialic Acid Standard Curve </LI><UL class="reason"><LI> Since our previous sialic assay was unsuccessful due the extremely complex procedure and lack of conclusive results, we decided to conduct a commercial kit version. A sialic acid assay provides for a direct method to target the sialic acid secreted through a GFP reporter system. Our goal was to quantify the amount of sialic acid secreted by each construct. By doing so, we could compare the CRISPR edited construct with our unmutated construct to determine if our gene edit was successful. The mutated constructs would serve as a negative control, as its tendency to polymerize would result in the least sialic acid secretion. By using an antibody tagged with GFP, we were able to directly target the sialic acid secreted and thus quantify the fluorescence, which was directly correlated to the amount of sialic acid secreted.  And, as expected, after 10 reading of each construct, we were able to find the average fluorescence intensity and thus determine the standard error for each, to conclude that our results were statistically significant. </LI></UL><LI>h. <A href="https://static.igem.org/mediawiki/2018/4/44/T--ASIJ_Tokyo--electro.pdf" target="none">Final Polyacrylamide Gel Confirmation of Constructs </A></LI><UL class="reason"><LI> Because our original electrophoresis was not conclusive, due to a wrong concentration used in gel, we decided to take on a 12% SDS Page Electrophoresis. This would allow us to compare the CRISPR constructs with the unmutated constructs to determine if our gene edit was successful. We once again expect to see a mutated variant with a much greater mass than the unmutated variant and ideally a genetically modified variant similar to an unmutated variant.  </LI></UL></UL></H4><H6>Icon used in the background on the home page made by <A href="https://www.flaticon.com/authors/ddara">dDara</A> from <A href="www.flaticon.com">www.flaticon.com</A></H6></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>