<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ASIJ_Tokyo_CRISPR skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ASIJ Tokyo/CRISPR</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="tutorial"><UL><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo" style="">HOME</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Team">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Collaborations">COLLABORATION</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Attributions">ATTRIBUTIONS </A></LI></UL><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Description">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/CRISPR">CRISPR</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Design">DESIGN</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Experiments">EXPERIMENT</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Results">RESULTS</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Improve">IMPROVE</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Activities">OUR ACTIVITIES</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Questions">Q&amp;A</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Future">FUTURE</A></LI><LI><A href="https://2018.igem.org/Team:ASIJ_Tokyo/Safety">SAFETY</A></LI></DIV></DIV><DIV id="wrapper"><H1 id="header"> CRISPR </H1><DIV id="blurb"><P><I>CRISPR-Cas9 is a technology that enables us to edit parts of the genome by removing, adding, or altering sections of DNA. As a high school team it was a great learning experience to use a tool of such relevance in biotechnology. </I></P></DIV><H2> What is CRISPR </H2><P><B>CRISPR</B> are natural segments of genetic code found in most prokaryotes. CRISPR stands for <B>C</B>lustered <B>R</B>egularly <B>I</B>nterspaced <B>S</B>hort <B>P</B>alindromic <B>R</B>epeats. These repeats are interspaced by unique spacer DNAs, each identical to the DNA of a specific virus. By storing viral DNAs, CRISPR allows bacteria and archaea to recognize and defend themselves against viruses. When a known virus injects its DNA, the CRISPR system uses the stored DNA to transcribe a complementary RNA (<B>crRNA</B>). It also transcribes a CRISPR associated protein (Cas protein) that then binds with the crRNA. This crRNA-Cas complex locates the invading genome by matching it with the complementary crRNA, and then finally utilizes the Cas protein to break it apart.  </P><P> This natural bacterial system, under control, has the potential to modify virtually any organism’s genes. To direct the powers of CRISPR, scientists utilize the Cas9 protein, an RNA-guided enzyme taken from Streptococcus pyogenes. In the case of Cas9, crRNA is bound to a scaffolding RNA (<B>tracrRNA</B>) that keeps it stable. Scientists realized that the Cas9 protein can theoretically cleave any DNA as long as the crRNA is altered to be complementary to what you want to edit. All that is required is to combine the personalized crRNA with a Cas9 specific tracrRNA to allow a stable bond to the Cas9 protein. These modified combinations of crRNA with tracrRNA are referred to as guide RNAs (<B>gRNA</B>). </P><TABLE><TBODY><TR><TD> crRNA-tracrRNA-Cas9 chimera can be simplified into gRNA-Cas9 complex </TD><TD> gRNA-Cas9 complex unwinds DNA and locates target sequence. Nuclease DNA cutters are few basepairs upstream of PAM sequence. </TD><TD> gRNA-Cas9 complex makes desired cleavage using Cas9 nuclease. </TD></TR></TBODY></TABLE><H2> Our Experiment </H2><P> For our experiment, we used CRISPR-Cas9 to edit the Z mutation in SERPINA-1. The mutated Z allele is differentiated from the wild M copy based on a single nucleotide polymorphism (<B>SNP</B>). Thus, we would only have to  alter one base pair, a change from adenine to guanine at position 94378610 on Chromosome 14’s SERPINA1 gene. This slight change would lead to a revertation of the Glu342Lys mutation associated with A1AT deficiency. </P><FIGURE align="right" class="hdrr"><FIGCAPTION> Homology Directed Repair </FIGCAPTION></FIGURE><P> The specific protocol used to create our gRNA was the Zhang protocol. We took the wild-type RNA (M allele) and combined it with a generic scaffolding to create a single gRNA (<B>sgRNA</B>). Next, we took our mutated DNA and added PAM sequences to them as Cas9 requires recognition sites to edit (prevents CRISPR self-targeting). Finally, we put the sgRNA and mutated DNA with PAM in the presence of Cas9. The sgRNA similarity (rather than complete identicality) to the mutated DNA target sequence has two effects. One, it is still able to guide the Cas9 to the mutated DNA target sequence to cause a double-stranded break. Second, it allows homology-directed repair (<B>HDR</B>) to occur. When DNA is cleaved by Cas9, it can simply join together to repair itself. But in the presence of a highly homologous (similar) sequence, this sequence can act as a template to replace the missing bases. And as our sgRNA is highly similar except the wild-type base pair, by this method a Z mutation can be edited to an M allele. </P><H2> Limitations </H2><P> It must be noted that some errors can occur with CRISPR. HDR occurs naturally and because of this, it is in many ways left to chance. This method can leave a mix of both mutated and edited copies. Furthermore, the sgRNA may in some cases guide Cas9 to wrong sequences because it is not completely identical to the target sequence. These off-point mutations further contribute to edited batches of proteins not being entirely the same as the wild type. </P><FIGURE id="elec"><FIGCAPTION> PAGE Electrophoresis of Protein
Mutated Protein: 1, 4, 7
Wild Type: 2, 5, 8
CRISPR: 3, 6, 9

Our CRISPR did not completely repair the mutation to be identical to a Wild-Type protein as seen in differences in staining. We believe we created an MZ type, as the gene is codominant. </FIGCAPTION></FIGURE><DIV id="references"><H3> Sources </H3> Biolabs, N. E. (n.d.). CRISPR/Cas9 &amp; Targeted Genome Editing: New Era in Molecular Biology. Retrieved October 18, 2018, from <A href="https://www.neb.com/tools-and-resources/feature-articles/crispr-cas9-and-targeted-genome-editing-a-new-era-in-molecular-biology"> https://www.neb.com/tools-and-resources/feature-articles/crispr-cas9-and-targeted-genome-editing-a-new-era-in-molecular-biology</A> Andersen, P. (2016, February 18). What is CRISPR? Retrieved October 18, 2018, from <A href="https://www.youtube.com/watch?v=MnYppmstxIs">https://www.youtube.com/watch?v=MnYppmstxIs </A> Cavanaugh, P., &amp; Garrity, A. (n.d.). Homology-Directed Repair. Retrieved October 18, 2018, from <A href="http://sites.tufts.edu/crispr/genome-editing/homology-directed-repair/"> http://sites.tufts.edu/crispr/genome-editing/homology-directed-repair/ </A></DIV></DIV><DIV class="wrapper"><FOOTER><H6>Icon used in the background on the home page made by <A href="https://www.flaticon.com/authors/ddara">dDara</A> from <A href="www.flaticon.com">www.flaticon.com</A></H6></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>