<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Freiburg_Cell_culture skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Freiburg/Cell culture</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="flex-container1"><DIV class="flex-item2"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Introduction">Overview</A><A href="https://2017.igem.org/Team:Freiburg/Motivation">Motivation</A><A href="https://2017.igem.org/Team:Freiburg/CAR">CAR T Cells</A><A href="https://2017.igem.org/Team:Freiburg/Tumor_microenvironment">Tumor Microenvironment</A><A href="https://2017.igem.org/Team:Freiburg/Design">AND Gate</A><A href="https://2017.igem.org/Team:Freiburg/Outlook">Outlook</A><A href="https://2017.igem.org/Team:Freiburg/Achievements">Achievements</A><A href="https://2017.igem.org/Team:Freiburg/References">References</A></DIV></DIV></DIV><DIV class="flex-item3"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Team">Members</A><A href="https://2017.igem.org/Team:Freiburg/Attributions">Attributions</A><A href="https://2017.igem.org/Team:Freiburg/Sponsors">Partners</A><A href="https://2017.igem.org/Team:Freiburg/Contact">Contact</A></DIV></DIV></DIV><DIV class="flex-item4"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Results">Main Project</A><A href="https://2017.igem.org/Team:Freiburg/Model">Modeling</A><A href="https://2017.igem.org/Team:Freiburg/Applied_Design">Applied Design</A><A href="https://2017.igem.org/Team:Freiburg/Demonstrate">Proof of Concept</A><A href="https://2017.igem.org/Team:Freiburg/Basic_Part">BioBricks Basic Part</A><A href="https://2017.igem.org/Team:Freiburg/Improve">BioBrick Improvement</A><A href="https://2017.igem.org/Team:Freiburg/InterLab">Interlab Study</A></DIV></DIV></DIV><DIV class="flex-item5"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Safety">Safety</A><A href="https://2017.igem.org/Team:Freiburg/Cloning">Cloning</A><A href="https://2017.igem.org/Team:Freiburg/Lab_Knockout"><I>HIF1A</I> Knockdown</A><A href="https://2017.igem.org/Team:Freiburg/Cell_culture">Cell Culture</A><A href="https://2017.igem.org/Team:Freiburg/Notebook">Lab Book</A></DIV></DIV></DIV><DIV class="flex-item6"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/HP/Silver">Human Practice</A><A href="https://2017.igem.org/Team:Freiburg/HP/Gold_Integrated">Integrated Human Practice</A><A href="https://2017.igem.org/Team:Freiburg/Collaborations">Collaborations</A></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><DIV class="flex-container"><H1 align="center">Cell Culture</H1><DIV class="item"><P>In cell culture, mammalian cells are grown under controlled conditions outside of their natural environment. This technique poses a powerful <I>in vitro</I> model as transformed cells from a variety of cell types coming from all three germ layers are available. These transformed cell lines have been immortalized and can be easily kept in culture conditions, in contrast to so called primary cells that have freshly been isolated from patients. <I>In vitro</I>, cells can easily be manipulated to study cellular processes, the roles of genes and proteins as well as the cellular responses to environmental cues. A wide range of methods to genetically manipulate cultured cells and assay systems to visualize and evaluate cellular processes have been established and guarantee reliable and reproducible data to be obtained from cell culture experiments. </P><DIV class="item"><DIV class="image_box right"><DIV class="figure" style="width:350px;"><DIV class="figureinner"><DIV class="figurecaption"><P style="font-size:14px;"><STRONG>Figure 1: HEK293T cells</STRONG></P></DIV></DIV></DIV></DIV><P>
For our study we chose human embryonic kidney (HEK293T) cells (<B>Fig. 1</B>) and Jurkat cells (<B>Fig. 2</B>), as well as Chinese hamster ovary (CHO) cells (<B>Fig. 3</B>) as a model. HEK293T is a broadly used cell line that is especially easy to keep in culture and can be genetically manipulated by simple transfection methods. We used this cell line to express and characterize the individual parts of the AND gate and to produce viral particles which were used to stably transduce Jurkat cells. </P></DIV><DIV class="item"><DIV class="image_box left"><DIV class="figure" style="width:350px;"><DIV class="figureinner"><DIV class="figurecaption"><P style="font-size:14px;"><STRONG>Figure 2: Jurkat cells</STRONG></P></DIV></DIV></DIV></DIV><P>
Jurkat cells (<B>Fig. 2</B>) are a cell line of immortalized human T lymphocytes. These cells were used to create a cell line of lymphocytic origin that stably expresses the complete AND gate and CAR.</P></DIV><DIV class="item"><DIV class="image_box right"><DIV class="figure" style="width:350px;"><DIV class="figureinner"><DIV class="figurecaption"><P style="font-size:14px;"><STRONG>Figure 3: Chinese hamster ovary (CHO) cells </STRONG></P></DIV></DIV></DIV></DIV><P>
CHO cells (<B>Fig. 3</B>) are a line of transformed epithelial cells. They were used for some experiments as they possess different expression characteristics compared to HEK293T cells. This allowed us to perform experiments that are not feasible in in HEK293T cells due to high background expression or insufficient transfection efficiency. </P></DIV><DIV class="item"><BUTTON class="accordion">Cell Culture Methods</BUTTON><DIV class="panel panel-default"><DIV class="item"><H3>Transfection</H3><P>Mammalian cell transfection is used to introduce foreign genetic sequences into cells. Depending on cell line and desired effect, there are different ways to transfect mammalian cells. These methods can be broadly classified into two groups: transient and stable transfections. Normally, a selection marker is needed to identify cells that are transfected.</P></DIV><DIV class="item"><H3>Transient Transfection</H3><P>Transient transfections generate short-term alterations in gene and protein expression. In this case, the expression of the introduced gene is temporary because the foreign nucleic acids are not integrated into the host cell genome and therefore will not be passed on by cell division. Transient transfection of cells was achieved by two different methods:
                        </P></DIV><DIV class="item-right"><H4>PEI transfection</H4><P>Polyethylenimine (PEI) is used as a polymeric transfection. PEI condenses DNA into positively charged particles, which bind to anionic cell surface residues and are brought into the cell via endocytosis. Using this method transiently transfected HEK cells were generated.</P></DIV><DIV class="item-right"><H4>PEI transfection of HEK cells</H4><P>For one 10 cm dish:</P><OL><LI>Check cells for confluency (should be at 80 %)</LI><LI>Dilute the 10 µg DNA in 1 ml serum free DMEM</LI><LI>Add 30 µl PEI (1 µg/µl) to the mix while vortexing (PEI:DNA = 3:1)</LI><LI>Incubate for 15 min at RT</LI><LI>Add the mix carefully to the cells</LI></OL></DIV><DIV class="item-right"><H4>Electroporation</H4><P>Electroporation is a method, which is used to introduce target genes into cells transiently in a physical way. To achieve this, an electrical field is generated around the cells to increase the permeability of the cell membrane, so that DNA can enter the cells. During our project we used electroporation to generate transient transfected Jurkat cells.</P></DIV><DIV class="item-right"><H4>BioRad Electroporation of Jurkat cells</H4><OL><LI>Count the cells</LI><LI>Transfer 2 mio cells into falcons and centrifuge (100 g, 3 min)</LI><LI>Discard the supernatant and resuspend in 200 µl P3 Buffer</LI><LI>Add 9 µg DNA</LI><LI>Transfer the mix into an electroporation cuvette</LI><LI>BioRad setting: 12 well plate, 1 well set, 1 pulse, 250 V, 950 µF, ∞ resistance, 20 msec</LI><LI>Add 500 µl RPMI 1640 medium to the cells immediately after electroporation</LI><LI>Plate the cells into 6-well plates with 2 ml medium for incubation</LI><LI>After 24 hours of incubation and then change the medium</LI></OL></DIV><DIV class="item"><H3>Stable transfection</H3><P>The aim of stable transfection is to introduce a gene of interest into the genome of cells, which results in permanent expression of foreign genes in target cells. Lentiviral transduction is used in this project to generate stable cell lines. </P></DIV><DIV class="item-right"><H4 id="LT">Lentiviral Transduction</H4><P>Stable transfection via lentiviral transduction delivers the foreign gene into the host’s genome. This ability was utilized in scientific research for constitutive expression of certain gene in the target cells. Lentiviruses are a subtype of retroviruses. Compared to standard retroviruses, lentiviruses are capable of infecting non-dividing and actively dividing cell types while standard retroviruses can only infect replicating cell types. Although viral vectors are derived from pathogenic viruses, their toxicity and the risk of handling them are reduced via modifications. The necessary components for virus production are splitted into multiple plasmids to increase the safety. The 2nd-generation lentiviral system is used in our project, for which one packaging plasmid and one envelope plasmid are needed for the virus production along with the transfer plasmid containing the gene of interest. 
                        </P></DIV><DIV class="item-right"><H4>Virus production</H4><P>Lentivirus is produced in the packaging cell. HEK293T cell is utilized in our project to produce lentivirus because of its high transfection efficiency.  To achieve this, three plasmids including transfer plasmid containing the gene of interest, packaging and envelope plasmid are transfected via PEI into HEK293T cells. The packaging plasmid pCMVR8.74 and the envelope plasmid pMD2.G are from Addgene.</P></DIV><DIV class="item-right"><H4>PEI transfection for virus production</H4><TABLE><TBODY><TR><TD>Plate size</TD><TD>15 cm</TD></TR><TR><TD>Total DNA/Plate</TD><TD>19.3 µg</TD></TR><TR><TD>Ratio transfer:packaging:envelope</TD><TD>2:1:1</TD></TR><TR><TD>Transfer plasmid</TD><TD>9.56 µg</TD></TR><TR><TD>Packaging plasmid (pCMVR8.74)</TD><TD>4.83 µg</TD></TR><TR><TD>Envelope plasmid (pMD2.G)</TD><TD>4.83 µg</TD></TR></TBODY></TABLE></DIV><DIV class="item"><OL><LI> Make packaging mix:
Mix packaging and envelope plasmids, fill up with sterile filtered 150 mM NaCl to an end volume of 990.4 µl and incubate for 10 min at room temperature</LI><LI>Make PEI mix:
Mix 115.8 µl PEI (7.5 mM) with 884.2 µl sterile filtered 150 mM NaCl and incubate for 10 min at room temperature</LI><LI>Make construct mix:
Add 983,9 µl of the packaging master mix and 1000 µl of the PEI mix to the transfer plasmid</LI><LI>Incubate for 15 min</LI><LI>Pipette 2000 µl of the construct mix dropwise to the HEK293T cells</LI><LI>After 24 hours and 48 hours of incubation, collect the media and add fresh medium to the cells</LI><LI>Supernatant of the collected medium contains virus, which can be directly stored or centrifuged to concentrate virus</LI></OL></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 4: Overview of lentiviral transduction </STRONG>
HEK293T cells as packaging cells for lentivirus production are transfected with transfer, envelope and packaging plasmid. After 48 h  lentiviral particles are collected and can be concentrated for transduction of target cells.
</P></DIV></DIV></DIV></DIV></DIV><DIV class="item"><H3>Concentrate virus</H3><P>To increase the efficacy of lentiviral transduction, viruses can be concentrated.</P><OL><LI>Add 8 ml 20 % Sucrose + 1mM EDTA in one 50 ml Falcon tube</LI><LI> Use one 25 ml pipette and add virus medium slowly over sucrose, optimally bubbles should be formed to generate two layers</LI><LI>Centrifuge at 4000 rpm, 8°C over night</LI><LI>On the next day: Remove the media and put the tubes upside down over 6 paper layers and let them dry for 10 min</LI><LI>Add 100 µl PBS/tube and resuspend virus carefully for about 20 times (with pipette set up with 50 µl) and rotate the tubes while pipetting</LI><LI>Store virus at -80°C</LI></OL></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN><DIV class="modal-content"><DIV class="mySlides"><DIV class="numbertext">lentiviral transduction</DIV></DIV><DIV class="mySlides"><DIV class="numbertext">FACS </DIV></DIV><DIV class="mySlides"><DIV class="numbertext">gating 1 </DIV></DIV><DIV class="mySlides"><DIV class="numbertext">gating 2</DIV></DIV></DIV></DIV><DIV class="item"><H3>Cryo Stocks</H3><P>For long-time storage of cells, cryo stocks were made:</P><P><B>HEK cells:</B> HEK cells were first washed with PBS and then trypsinized. Trypsinization was stopped with RPMI 1640 medium and the sample was centrifuged (100 x g, 3 min). The supernatant was discarded and the pellet was resuspended in RPMI 1640 and the cells counted with a Neubauer chamber. 
After another centrifugation step (100 x g, 3 min) the supernatant was discarded and the pellet resuspended in 1 ml of 90 % FCS and 10 % DMSO. 
The cells were transferred to a cryo tube, which was stored for 2 days at -80 °C in an ice box to ensure gradual freezing of the cells. The tubes were then placed in a liquid nitrogen tank for long term storage.</P><P><B>Jurkat cells:</B> Jurkat cells from a confluent 10 cm dish were transferred to a 15 ml falcon tube and centrifuged (100 g, 3 min). The supernatant was discarded and the pellet resuspended in 2 ml of 90 % FBS and 10 % DMSO. 
Aliquots of 1 ml cell suspension were transferred into a cryo tube and stored for 2 days at -80 °C in an ice box to ensure gradual freezing of the cells. Afterwards the tubes were placed into a liquid nitrogen tank for long term storage.
                        </P></DIV><DIV class="item"><H3>Thawing of the samples</H3><P>To thaw samples, they were taken out of the liquid nitrogen tank and placed into a 37 °C incubator. 15 ml falcon tubes with 5 ml RPMI 1640 were also placed in the 37 °C incubator to heat up.
Afterwards the thawed sample was transferred to the warmed 15 ml falcon tube with RPMI 1640 and centrifuged (100 x g, 3 min). The supernatant was discarded and the pellet resuspended in RPMI 1640. The sample was then transferred to a 6-well plate and placed back into the incubator.

                        </P></DIV><DIV class="item"><H3>Fluorescence-Activated Cell Sorting (FACS)</H3><P>Fluorescence-activated cell sorting was developed by Leonard Herzenberg in 1970 (Dangl &amp; Lanier, 2013) and was a remarkable breakthrough in immunology and cancer biology. Relying on basic principles of flow cytometry, FACS enables separation of cell populations into sub-populations based on fluorescent labelling. To take the advantages of FACS, vectors containing one constitutively expressed reporter gene were cloned, so that cells transfected with these vectors could be sorted. Cell Sorting service was provided at BIOSS from the Toolbox team using Bio-Rad S3eTM Cell Sorter.
                        </P></DIV><DIV class="column"><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Fig.5: Fluorescence-Activated Cell Sorting (FACS)</STRONG> 
 Exemplary sorting result from lentiviral transduced HEK293T cells with a construct containing <I>4xHRE-pTal:eCFP::CMV:mCherry</I>. Cells were excited with FL3 laser channel and sorted for mCherry positive cells. The cells were firstly gated for living, singlets and then gated for mCherry negative (R6) and mCherry positive (R7 and R8) based on wild type HEK293T cells.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="item"><BUTTON class="accordion">Analysis Methods</BUTTON><DIV class="panel panel-default"><DIV class="item"><H3>Plate Reader</H3><P>Plate reader measurements are used to quantify for example the relative expression of fluorescent proteins under different promoters. Another application is tracking and quantification of enzyme activity, which is used for Luciferase or SEAP assays (Ashour <I>et al.</I>, 1987). In this project, the plate reader was used to analyze the spatial and temporal resolution of the promoter activity. </P><P>In this method, a laser is used to excite fluorescent proteins at their absorption maximum. The emitted light is measured by a photosensitive sensor at the emission maximum either detecting from the top or the bottom of the microplate. The latter is prefered for tissue cells as they adhere on the ground of a plate well. Since the measured fluorescence intensity is proportional to the amount of expressed fluorescence proteins, this method is highly suitable for promoter characterizations.</P><P>For plate reader measurements, cells were incubated under different conditions on a black 96 well plate with clear bottom in 200 µl medium. The fluorescence intensity was measured after different incubation times with a Synergy H4 from Biotek.</P></DIV><DIV class="item"><H3>SEAP Assay</H3><P id="seap">
               SEAP (secreted embryonic alkaline phosphatase) assays were performed to measure the induction of the HRE promoter. 
Firstly, 200 µl of cell supernatant was transferred to 1.5 ml Eppendorf tubes. These were heated for 30 min at 65°C, followed by a centrifugation step (1250 x g, 1 min). 
2 µl of 2x SEAP buffer (20 mM homoarginine, 1 mM MgCl<SUB>2</SUB>, 21 % (v/v) diethanolamine, pH 9.8) were filled into wells of a 96 well plate. 40 µl of the heated cell supernatant were added to the wells with 2x SEAP buffer.
The addition of 68 µl of the substrate pNPP (120 mM para-nitrophenyl phosphate dissolved in H<SUB>2</SUB>O) was directly followed by spectroscopic measurement for 180 min, every 30 s at a wavelength of 405 nm in a microplate reader (Biotek Synergy H4 Hybrid Reader).</P><H3>SEAP Assay Analysis</H3><P>
                The averages of absorbance were calculated for all triplicates. Afterwards, the corresponding untransfected control was subtracted from each sample and the controls were set to 0.
The slope was determined for the samples and added in the following formula to determine the relative enzyme activity:</P><DIV><DIV class="col-sm-8"><P><MATH xmlns="http://www.w3.org/1998/Math/MathML"><MO>(</MO><MO>(</MO><MFRAC><MROW><MI>s</MI><MI>l</MI><MI>o</MI><MI>p</MI><MI>e</MI></MROW><MROW><MN>18600</MN><MO>*</MO><MN>0</MN><MO>.</MO><MN>5</MN></MROW></MFRAC><MO>)</MO><MO>*</MO><MN>20</MN><MO>*</MO><MSUP><MN>10</MN><MN>6</MN></MSUP><MO>)</MO><MO>/</MO><MN>80</MN></MATH></P></DIV></DIV></DIV><DIV class="item"><H3>Flow Cytometry</H3><P>
                Flow cytometry was used to measure the induction of all promoters. All experiments were performed in triplicates. 
For the analysis, cells were resuspended and collected in Eppendorf tubes. Following a  centrifugation step (200 x g, 5 min) the supernatant was discarded and the cells were resuspended in 300 µl FACS buffer (10 ml PBS, 100 µl FCS). 
Untransfected cells and cells with either CFP or mCherry were used for the compensation of the channels.
Measurement was performed until either a time limit of 5 min, a total event number of 1 mio or 20,000 living cells was reached.
For the experiments a Gallios flow cytometer from Beckman Coulter was used, as well as the CyAn from Beckman Coulter.</P><H3>Flow Cytometry Analysis</H3><P>
                To analyze the data measured by the flow cytometer, FlowJoTM 10 was used.
First, gates were set up for living cells in the wild type cells, which then were gated for mCherry positive cells and these for CFP positive cells. This was done for each treatment group independently, were the gating was always applied from the corresponding wild type control to the test sample.
Wild type cells that were treated in the same way as the cells containing the construct of interest were used for a gating of living cells, mCherry and CFP positive cells. 

An exemplary gating strategy can be seen in <B>figure 6</B> and <B> figure 7</B>.
                </P><DIV class="column"><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><B> Figure 6: Gating strategy for flow cytometry analysis of mCherry+ CFP+ cells under the population of living, single cells.</B> Exemplary gating for living wild-type Jurkat cells which were treated with forskolin (100 µM) and IBMX (100 µM) for 24h. <B>a)</B> Gating for living cell population. <B>b)</B> Afterwards, the living population was gated for single cells. <B>c)</B> Followed by the gating of mCherry positive cells. <B>d)</B> Depicted is the gating for CFP positive cells under the population of mCherry positive cells. These were however gated underneath the single cell population, considering that the wild type cells do not express mCherry. The gate then was moved under the population of mCherry positive cells.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="column"><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><B>Figure 7: Applied gates from the corresponding wild type control to Jurkat cells which contain the CRE plasmid and were treated with IBMX (100 µM) and forskolin (100 µM). </B>The gating of the wild-type control was shown in figure 1. <B>a)</B> Gating for living cell population. <B>b)</B> Afterwards, the living population was gated for single cells. <B>c)</B> The mCherry positive single cells were determined and then <B>d)</B> gated for CFP positive cells to obtain the cells with successful expression of the construct.
                </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><DIV class="flex-container"><DIV class="col-sm-6"><DIV class="image_box left" style="float:right"><DIV class="figure" style="width:195px"><DIV class="figureinner"><DIV class="figurecaption-car" style="float:right"><P><STRONG>Cloning</STRONG></P></DIV></DIV></DIV></DIV></DIV><DIV class="col-sm-6"><DIV class="image_box right" style="float:left"><DIV class="figure" style="width:180px"><DIV class="figureinner"><DIV class="figurecaption-car" style="float:left"><P><STRONG>Human Practice</STRONG></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>