<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Freiburg_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Freiburg/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="flex-container1"><DIV class="flex-item2"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Introduction">Overview</A><A href="https://2017.igem.org/Team:Freiburg/Motivation">Motivation</A><A href="https://2017.igem.org/Team:Freiburg/CAR">CAR T Cells</A><A href="https://2017.igem.org/Team:Freiburg/Tumor_microenvironment">Tumor Microenvironment</A><A href="https://2017.igem.org/Team:Freiburg/Design">AND Gate</A><A href="https://2017.igem.org/Team:Freiburg/Outlook">Outlook</A><A href="https://2017.igem.org/Team:Freiburg/Achievements">Achievements</A><A href="https://2017.igem.org/Team:Freiburg/References">References</A></DIV></DIV></DIV><DIV class="flex-item3"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Team">Members</A><A href="https://2017.igem.org/Team:Freiburg/Attributions">Attributions</A><A href="https://2017.igem.org/Team:Freiburg/Sponsors">Partners</A><A href="https://2017.igem.org/Team:Freiburg/Contact">Contact</A></DIV></DIV></DIV><DIV class="flex-item4"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Results">Main Project</A><A href="https://2017.igem.org/Team:Freiburg/Model">Modeling</A><A href="https://2017.igem.org/Team:Freiburg/Applied_Design">Applied Design</A><A href="https://2017.igem.org/Team:Freiburg/Demonstrate">Proof of Concept</A><A href="https://2017.igem.org/Team:Freiburg/Basic_Part">BioBricks Basic Part</A><A href="https://2017.igem.org/Team:Freiburg/Improve">BioBrick Improvement</A><A href="https://2017.igem.org/Team:Freiburg/InterLab">Interlab Study</A></DIV></DIV></DIV><DIV class="flex-item5"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Safety">Safety</A><A href="https://2017.igem.org/Team:Freiburg/Cloning">Cloning</A><A href="https://2017.igem.org/Team:Freiburg/Lab_Knockout"><I>HIF1A</I> Knockdown</A><A href="https://2017.igem.org/Team:Freiburg/Cell_culture">Cell Culture</A><A href="https://2017.igem.org/Team:Freiburg/Notebook">Lab Book</A></DIV></DIV></DIV><DIV class="flex-item6"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/HP/Silver">Human Practice</A><A href="https://2017.igem.org/Team:Freiburg/HP/Gold_Integrated">Integrated Human Practice</A><A href="https://2017.igem.org/Team:Freiburg/Collaborations">Collaborations</A></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><DIV class="flex-container"><H1 align="center">Results</H1><DIV class="item"><P>Four milestones had to be achieved over the course of our project:
We wanted to generate stable cell lines that express <A href="https://2017.igem.org/Team:Freiburg/CAR">chimeric antigen receptor (<I>CAR</I>)</A> under control of a logic AND gate. A major part of this work was the generation of cell lines that lack the endogenous <I>HIF1A</I> gene, which is the pivotal point of our CARTEL<SUP>TM</SUP> AND gate. Using these cell lines, we characterized our promoter constructs under various conditions simulating the tumor microenvironment. To improve safety of our system, we validated the functionality of a suicide gene we introduced, testing its efficiency in a viability assay. Below, our results are explained in detail.</P></DIV><DIV class="item"><H2 style="font-size:40px;">Stable Cell Lines</H2><P>During our project, we generated stable cell lines via <A href="https://2017.igem.org/Team:Freiburg/Cell_culture">lentiviral transduction</A> in order to analyse the entire <A href="https://2017.igem.org/Team:Freiburg/Design">AND gate</A>. <A href="https://2017.igem.org/Team:Freiburg/Cell_culture">Jurkat and HEK293T cells</A> have been used in parallel to introduce one of our <A href=" https://2017.igem.org/Team:Freiburg/Cloning">AND gate promoters (<I>pCRE</I>, <I>pHRE</I> or <I>pCTLA4</I>)</A>, <A href="https://2017.igem.org/Team:Freiburg/Lab_Knockout"><I>HIF1A</I> knockdown constructs</A> and a <A href="https://2017.igem.org/Team:Freiburg/HP/Gold_Integrated">suicide gene</A>, respectively.

</P></DIV><DIV class="item"><BUTTON class="accordion">Stable Cell Lines</BUTTON><DIV class="panel panel-default"><DIV class="item"><P><A href=" https://2017.igem.org/Team:Freiburg/Cell_culture">Lentiviral particles</A> were produced by transfecting HEK293T cells with a lentiviral transfer vector along with packaging and envelope plasmids. After incubation, the supernatant containing freshly produced lentiviral particles was collected and used to transduce target cells. </P></DIV><DIV class="item"><P>
After incubation with the lentiviral supernatant, transduced cells were washed several times with PBS to remove residual virus. The washed cells were sorted via <A href=" https://2017.igem.org/Team:Freiburg/Cell_culture">fluorescence activated cell sorting (FACS)</A> for positive and single cells.</P><P>
All stable cell lines show significantly increased expression of the transfection control reporter gene compared to untransduced cells, if analyzed via <A href=" https://2017.igem.org/Team:Freiburg/Cell_culture">flow cytometry</A> (<B>Fig. 2, 4, 6</B>).  
</P></DIV><DIV class="item"><A href="https://2017.igem.org/Team:Freiburg/Cloning"><H3>Stable Cell Lines Transduced with Inducible Promoters</H3></A><P><A href="https://2017.igem.org/Team:Freiburg/Design">CARTEL<SUP>TM</SUP> AND gate</A> aims at the controlled expression of <I>CAR</I>. As the first step of our project, the single promoter activities had to  be examined. Stable Jurkat and HEK293T cell lines containing multiple repeats of enhancer elements (<B>Fig. 2</B>)  were generated. The constructs (<B>Fig. 1</B>) contain a reporter cassette that constitutively expresses <I>mCherry</I>, which was used as selection marker for sorting infected cells.</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 1: Lentiviral vectors for promoter characterization.</STRONG>
The lentiviral vectors for promoter characterization contain <I>mCherry</I> as selection marker. 
                       </P></DIV></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 2: Stable cell lines for promoter characterization.</STRONG>
Comparison of <I>mCherry</I> expression between wild type and transduced cells. Jurkat cells stably expressing <B>a)</B><I>4xHRE:eCFP::CMV:mCherry</I>, <B>b)</B><I>4xCRE:eCFP::CMV:mCherry</I>, or <B>c)</B><I>4xNFATbs-pCTLA4:eCFP::CMV:mCherry</I>. Significant differences were determined using one-tailed student’s t-test (Excel 2017); * p &lt; 0.05, ** p &lt; 0.01, *** p &lt; 0.001.</P></DIV></DIV><DIV class="container"><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV></DIV><DIV class="item">
         	<A href="https://2017.igem.org/Team:Freiburg/Lab_Knockout"><H3><I>HIF1A</I> Depletion </H3></A><P>
To achieve <I>HIF1A</I> depletion, three different shRNAs targeting <I>HIF1A</I> 3'UTR (<B>Fig. 3</B>) were separately transduced via lentiviral particles into Jurkat and HEK293T cells. Stable cell lines were produced and analyzed for eCFP fluorescence using flow cytometry (<B>Fig. 4</B>). The efficiency of knockdown of <I>HIF1A</I> is shown in <I>HIF1A</I> knockdown results section. 

 
</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 3: Lentiviral HIF1A shRNA vectors.</STRONG>
The lentiviral vector for <I>HIF1A</I> knockdown contains neomycin resistance and eCFP as selection marker. 
                       </P></DIV></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 4: HIF1A knockdown stable cell line. </STRONG>
Flow cytometry analysis for eCFP positive cells in Jurkat HIF1A shRNA1 cell line compared to wild type Jurkat cells.
                       </P></DIV></DIV></DIV></DIV></DIV><DIV class="item"><A href="https://2017.igem.org/Team:Freiburg/HP/Gold_Integrated"><H3>Cells Stably Expressing a Suicide Gene (Kill Switch)</H3></A><P>
Jurkat and HEK293T cells were transduced with construct containing the suicide gene <I>herpes simplex virus 1</I> thymidine kinase (<B>Fig. 5</B>). Stable cells were analyzed by flow cytometry for eGFP fluorescence (<B>Fig. 6</B>).
</P><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 5: Lentiviral vector containing the suicide gene <I>HSV-TK</I>.</STRONG>
The lentiviral vector for kill switch contains hygromycin resistance and eGFP as selection marker. 
                       </P></DIV></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 6: Kill switch stable cell line</STRONG>
Flow cytometry plots for <B>a)</B> of WT Jurkat cells and  <B>b)</B><I>HSV-TK</I> transduced Jurkat cells. GFP positive cells (R7 region, highlighted in green) were sorted and seeded for ganciclovir induced killing assay.
                       </P></DIV></DIV></DIV></DIV></DIV><DIV class="item"><P>
All sorting data of our stable cell lines can be found in <A href="https://2017.igem.org/Team:Freiburg/Notebook_Cell_culture">the cell culture lab notebook</A>.
</P></DIV></DIV></DIV></DIV><DIV class="item"><H2 style="font-size:40px;"><I>HIF1A</I> Knockdown</H2><P>The genetic circuit designed to control <A href="https://2017.igem.org/Team:Freiburg/CAR">chimeric antigen receptor (<I>CAR</I>)</A> expression in the <A href="https://2017.igem.org/Team:Freiburg/Tumor_microenvironment">tumor microenvironment</A> relies on exclusive control of hypoxia-inducible factor 1 alpha (<I>HIF1A</I>) expression by the introduced <A href="https://2017.igem.org/Team:Freiburg/Design">AND gate</A>. Various strategies have been employed to obtain HIF1A deficient cells.</P></DIV><DIV class="item"><BUTTON class="accordion"><I>HIF1A</I> Knockdown</BUTTON><DIV class="panel panel-default"><DIV class="item"><H3>Knockdown of <I>HIF1A</I> Reduces HIF1A in Jurkat Cells</H3><P>
HIF1A protein level was assessed in Jurkat cells under normoxic and hypoxic conditions (<B>Fig. 7</B>). Weak HIF1A protein presence was observed under normoxic conditions. Compared to HEK293T cells, lower concentrations of CoCl<SUB>2</SUB> were used to mimic hypoxia, due to toxicity to cells at higher concentrations. The highest HIF1A protein concentration was observed in wild type Jurkat cells at 20 µM CoCl<SUB>2</SUB>. Stable expression of shRNA1 targeting <I>HIF1A</I> lead to a significant reduction in HIF1A protein detection under normoxic and hypoxic conditions.
    </P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 7: Knockdown of <I>HIF1A</I> reduced HIF1A protein in Jurkat cells.</STRONG>
Jurkat wild type (WT) and cells stably expressing a shRNA targeting <I>HIF1A</I> (HIF1A shRNA1) were treated with different concentrations of CoCl<SUB>2</SUB> to mimic hypoxia. After 24 h of induction cells were harvested and Western blot analysis of whole cell lysate was performed. Knockdown cells show no production of HIF1A protein after induction with CoCl<SUB>2</SUB>, whereas accumulation of HIF1A protein can be observed in wild type cells under hypoxic conditions. Anti-GAPDH antibody was used as a loading control. 
                   </P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV><DIV class="item"><H3>Knockdown of <I>HIF1A</I> Reduces HIF1A in HEK293T Cells</H3><P>
        The effect of knocking down <I>HIF1A</I> on HIF1A protein levels was assessed in HEK293T (<B>Fig. 8</B>) under normoxic and hypoxic conditions by Western blotting. In normoxia HIF1A protein can only barely be detected in wild type HEK293T cells. By adding different amounts of CoCl<SUB>2</SUB>, to mimic hypoxia, an increase in HIF1A protein levels can be observed in wild type HEK293T cells at 400 µM CoCl<SUB>2</SUB>. Knockdown of <I>HIF1A</I> significantly decreases HIF1A protein production in all three HEK293T <I>HIF1A</I> shRNA cells lines under hypoxic conditions. 
    </P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 8: Knockdown of <I>HIF1A</I> reduces HIF1A protein in HEK293T cells under hypoxic conditions.  </STRONG>
                   HEK293T  wild type (WT) and stably expressing different shRNA targeting <I>HIF1A</I><B>(a, b, c)</B> were treated with different concentrations of CoCl<SUB>2</SUB> to mimic hypoxia. After 24 h induction cells were harvested and Western blot analysis of whole cell lysate was performed. Knockdown cells show no production of HIF1A protein after induction with CoCl<SUB>2</SUB>, whereas accumulation of HIF1A protein can be observed in wild type cells under hypoxic conditions. Anti-GAPDH antibody was used as a loading control.      
                   </P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV><DIV class="item"><H3><I>HIF1A</I> Reintroduced in HEK293T <I>HIF1A</I> Knockdown Cells is not Affected by the shRNA</H3><P>
    The CARTEL<SUP>TM</SUP> AND gate requires <I>HIF1A</I> expression under the exclusive control of the introduced promoter. To test whether the reintroduced <I>HIF1A</I> is affected by the shRNA used to create the knockdown, a HA tagged <I>HIF1A</I> was transfected transiently into <I>HIF1A</I> knockdown cells. The reintroduced HIF1A-HA is accumulated under hypoxic conditions and is not degraded by the shRNA1 (<B>Fig. 9 a</B>) as well as shRNA2 (<B>Fig. 9 b</B>). 
    </P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 9: Overexpression of <I>HIF1A-HA</I> in HIF1A shRNA1 and shRNA2 knockdown HEK293T cell lines is not affected by the expressed shRNA.</STRONG>
Wild type (WT) and knockdown cells were transfected with <I>CMV:HIF1A-HA</I> one day prior to induction with CoCl<SUB>2</SUB> to simulate hypoxia. Cells were harvested 24 h after induction. An accumulation of HIF1A-HA protein was detected under hypoxic conditions. Endogenous and artificial HIF1A were detected with HIF1A antibody; reintroduced HIF1A-HA was detected via an HA-tag; anti-GAPDH antibody served as a loading control.  
                   </P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV><DIV class="item"><H2>Knockout</H2><P>
    As another approach besides knockdown of <I>HIF1A</I>, various guide RNAs (gRNAs) for Cas9 (<I>S. pyogenes</I>) were used to introduce a knockout in <A href="https://2017.igem.org/Team:Freiburg/Cell_culture">Jurkat cells</A>. Transfected cells were sorted for GFP expression by <A href="https://2017.igem.org/Team:Freiburg/Cell_culture">FACS</A> either directly into single cells to found clonal cultures or manual dilution series of sorted cells was performed to this end. Alternatively, the transfected cells were selected by puromycin and thereafter diluted to find single cells.</P><P>
    Unfortunately, these cultures often did not grow or grew more slowly than standard doubling time as mammalian cells in culture are dependent on growth factors secreted by their peers in order to survive and proliferate. Therefore, we unfortunately were unable to grow clonal knockout cultures to sizes sufficient for analysis. For knockout, mutations were planned to be identified by T7E1 assay on genomic DNA (<B>Fig. 10</B>).</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 10: T7E1 Mutagenesis Assay. </STRONG>
                     Jurkat <I>HIF1A</I> and HEK293T <I>CCR5Δ32</I> gDNA PCR fragments were denatured and reannealed, treated with T7E1 and cleavage fragments visualized by gel electrophoresis. <I>CCR5Δ32</I> fragment length is 568 bp and contains a monoallelic deletion of 32 bp such that cleavage fragments of 244 bp and 292 bp are expected, <I>HIF1A</I> fragment length is 593 bp and contains no mutation.</P></DIV></DIV></DIV></DIV></DIV><DIV class="item"><BUTTON class="accordion">Cloning strategy</BUTTON><DIV class="panel panel-default"><DIV class="item"><H2>Knockdown</H2><P>The miR-E backbone, an optimized lentiviral plasmid for small hairpin RNA (shRNA) expression (Fellman <I>et al.</I>, 2013) was modified to contain eCFP instead of dsRED as fluorescence marker (<B>Fig. 11</B>). A protocol combining <A href="https://2017.igem.org/Team:Freiburg/Cloning">restriction digestion with Gibson cloning</A> was followed. The plasmid was digested to obtain a large part of the backbone and the second part was amplified by <A href="https://2017.igem.org/Team:Freiburg/Cloning">PCR</A> with overlapping ends to the digestion product and the eCFP sequence, which was amplified by PCR with overlapping ends to both backbone parts. These fragments were joined by Gibson assembly.</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 11: Cloning strategy of knockdown construct.</STRONG><B>a)</B> In the knockdown plasmid the fluorescent protein dsRed was replaced with eCFP. Backbone fragments and <I>eCFP</I> sequence were amplified by PCR and combined using Gibson assembly. <B>b)</B> To obtain a <I>HIF1A</I> specific knockdown plasmid an oligomer coding for the shRNA guide sequence, its reverse complement and the loop structure was inserted into the backbone using restriction digestion and ligation of the fragments.</P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV><DIV class="item"><P>Some <I>HIF1A</I> specific shRNA sequences were kindly provided by the Zuber Lab (Vienna), others designed using the shRNA design tool of the <A href="https://portals.broadinstitute.org/gpp/public/analysis-tools/sgrna-design" target="_blank">Genetic Perturbation Platform</A> (GPP; Doench <I>et al.</I>, 2016) (<B>Tab. 1</B>). Oligonucleotides containing the miR-E consensus sequence, the shRNA coding sequence and its reverse complement for formation of hairpin structures were extended by PCR to insert restriction sites. Finally, plasmid and inserts were digested and ligated.</P><P><B>Table 1: Anti-<I>HIF1A</I> shRNA designs.</B>Data obtained by AG Zuber (Vienna) and via GPP for human <I>HIF1A</I> (Binding site in NG_029606).</P></DIV><DIV class="item"><H2>Knockout</H2><P>To target <I>HIF1A</I>, gRNAs were designed using GPP and off-target matches analysed with <A href="https://crispr.cos.uni-heidelberg.de/" target="_blank">CRISPR/Cas Target Online Predictor</A> (CCTOP; Stemmer <I>et al.</I>, 2015). The latter was also used to design dsDNA oligo sequences for cloning. The chosen gRNA sequences are shown (<B>Tab. 2</B>). These were cloned into BsmBI digested Cas9-GFP (Ran <I>et al.</I>, 2013) and Cas9-puromycin (Sanjana <I>et al.</I>, 2014) as dsDNA oligos (<B>Fig. 12</B>). The restriction sites were either present in the plasmid or inserted by site-directed mutagenesis <A href="https://2017.igem.org/Team:Freiburg/Cloning">PCR</A>. A Cas9 plasmid kit targeting <I>HIF1A</I> was sponsored by <A href="http://www.origene.com/CRISPR-CAS9/KN202461/HIF1A.knockout" target="_blank">OriGene</A> (<B>Fig. 13</B>).</P><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 12: Cloning strategy to insert gRNAs into Cas9 plasmids.</STRONG>
                       The guide RNAs (gRNAs) were ligated as dsDNA oligos into BsmBI-digested Cas9 plasmids directly at the end of the contained gRNA scaffold.</P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 13: Overview of anti-<I>HIF1A</I> Cas9 knockout kit plasmid function.</STRONG>
                       Provided were two plasmids, one Cas9 and guide RNA (gRNA) vector, as well as a donor template vector. The gRNA target site is close to the start codon of <I>HIF1A</I>, such that <I>GFP</I> is integrated to be expressed by the endogenous promoter of <I>HIF1A</I>. The repair template furthermore causes integration of puromycin resistance selection marker.</P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV><P><B>Table 2: <I>Cas9</I> anti-<I>HIF1A</I> gRNA designs.</B>Data obtained via GPP for human <I>HIF1A</I>, off-target genes were found using CCTOP.</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="item"><H2 style="font-size:40px;">Promoter Characterization</H2><P>For the CARTEL<SUP>TM</SUP> AND gate we chose the promoters <I>CRE</I>, <I>HRE</I> and <I>CTLA4</I> for inducible expression of <I>CAR</I>. In order to characterize each promoter activity on its own, we generated stable Jurkat and HEK293T cell lines with multiple enhancer elements for each promoter driving the expression of <I>eCFP</I> reporter protein. Cells were exposed to <A href="https://2017.igem.org/Team:Freiburg/Design">low pH, high VEGF concentration and hypoxic conditions</A>, respectively, and the resulting expression of <I>eCFP</I> was analyzed by <A href="https://2017.igem.org/Team:Freiburg/Cell_culture">flow cytometry</A>.</P></DIV><DIV class="item"><BUTTON class="accordion">CRE Promoter</BUTTON><DIV class="panel panel-default"><DIV class="item"><H2>Characterization of the <I>CRE</I> Promoter</H2><P>The cAMP response element-containing promoter (<A href="https://2017.igem.org/Team:Freiburg/Design"><I>pCRE</I></A>), which is pH responsive, was characterized <I>in vitro</I> in <A href="https://2017.igem.org/Team:Freiburg/Cell_culture">Jurkat and HEK293T cell lines</A>. For this purpose, the pH of the media was precisely adjusted. For analysis in HEK293T, <A href="https://2017.igem.org/Team:Freiburg/Cell_culture">PEI transfection</A> of the pH receptor TDAG8, which is not expressed in these cells, was performed (Ausländer <I>et al.</I>, 2014). To generate a high expression, forskolin, which activates the cAMP-producing adenylyl cyclase, and IBMX, which inhibits cAMP-hydrolyzing phosphodiesterases (Bittinger <I>et al.</I>, 2004), were added.</P></DIV><DIV class="item"><BUTTON class="accordion">pH Adjustment of RPMI 1640 Medium</BUTTON><DIV class="panel panel-default"><DIV class="item"><P>To simulate the tumor microenvironment, it was crucial to adjust the pH of the medium. RPMI 1640 medium is strongly buffered by bicarbonate buffer (buffer range: pH = 5.1 - 7.1, <B>Fig. 14</B>) and HEPES (buffer range: pH = 6.8 - 8.2) (Dawson <I>et al.</I>, 1986). Furthermore, the pH in the medium is influenced by the 5 % CO<SUB>2</SUB> atmosphere in the incubator and production of lactic acid by cells (Damaghi <I>et al.</I>, 2013). </P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><B>Figure 14: Bicarbonate buffer.</B>At lower pH, the equilibrium is shifted to CO<SUB>2</SUB> and H<SUB>2</SUB>O causing evaporation of CO<SUB>2</SUB>.</P></DIV></DIV></DIV></DIV></DIV><DIV class="item"><P>To overcome these obstacles, different acids were evaluated. As tumor cells secrete lactic acid it was tested to adjust the pH of the medium (<B>Fig. 15</B>). Bicarbonate buffering leads to instability of original or adjusted pH in RPMI 1640, making use of a titration curve inexpedient. Additionally, lactic acid is a weak acid with a pKa of 3.86 (Narendranath <I>et al.</I>, 2001) and acts as a buffer itself. That makes setting up the pH even more difficult in this highly dynamic buffer system. As an alternative, hydrochloric acid, which completely dissociates with a pKa of -7 (Riedel <I>et al.</I>, 2004), was tested. The setup of the pH is more feasible with hydrochloric acid than with lactic acid.
</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><B>Figure 15: Titration of lactic acid against RPMI 1640.</B>Different concentrations of lactic acid in 2 ml RPMI 1640 have been measured in triplicates.</P></DIV></DIV></DIV></DIV></DIV><DIV class="item"><P>The influence of 5 % CO<SUB>2</SUB> atmosphere on the pH of the medium was analyzed inside the incubator (<B>Fig. 16</B>). The findings suggest that the carbonate buffer is equilibrated with the 5 % CO<SUB>2</SUB> atmosphere after around 60 minutes showing a pH at about 0.3 below the pH outside the incubator. The strong increase at the beginning arises from evaporation of CO<SUB>2</SUB> before entering the incubator as the medium has a higher surface to volume ratio in a well compared to the bottle. The same effect is also responsible for the inconsistent original pH of stock RPMI 1640. The slow decrease of the pH after equilibration of the buffer results from the secretion of metabolites by Jurkat cells (Masters <I>et al.</I>, 2007).</P><P>Based on these results, the pH of the induction media was set 0.3 above the final pH value, so the pH equilibrates to the desired value inside the incubator.
</P></DIV><DIV class="column"><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><B>Figure 16: Dynamic trend of pH in RPMI 1640 medium containing Jurkat cells in incubator.</B>7.5 ml RPMI 1640 containing 250.000 cells per ml were incubated for 24 h in a 6 well plate, pH was measured every 30 seconds.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="item"><H3>Induced Activity of <I>pCRE</I> in Jurkat Cells at Low pH</H3><P>Jurkat <I>4xCRE-pTal:eCFP</I> cells show a 1.5-fold increase in fluorescence at pH 6.5 compared to 7.7 even though the amount of eCFP positive cells remains low (<B>Fig. 17 a</B>). The same trend was obtained via additional stimulation with forskolin and IBMX for which the percentage of eCFP positive cells is about 15-fold higher (<B>Fig. 17 b</B>). These findings show that Jurkat <I>4xCRE-pTal:eCFP</I> cells are pH responsive. As overall expression via <I>CRE</I> remains on a low level, further optimization of the construct is necessary. High expression via induction with forskolin and IBMX indicates that higher expression rates are obtainable. This could be achieved by the use of another minimal promoter or different amounts of enhancer elements. Furthermore, the density of TDAG8 on the membrane could be adapted in order to optimize the signaling of the cells.</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><B>Figure 17: Detection of promoter activity by flow cytometric analysis of eCFP reporter fluorescence.a)</B> Jurkat cells stably transduced with <I>4xCRE-pTal:eCFP</I> were incubated 24 h in pH adjusted RPMI 1640. <B>b)</B> The experiment was repeated inducing with forskolin (10 µM) and IBMX (10 µM). All data points are mean values of triplicates, error bars represent standard deviation. Significant differences were determined using ANOVA; * p &lt; 0.05, ** p &lt; 0.01, *** p &lt; 0.001.</P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV><DIV class="item"><H3>High Basal <I>CRE</I> Promoter Activity in HEK293T Cells</H3><P>In HEK293T containing <I>4xCRE-pTal:eCFP</I> the number of eCFP positive cells was pH and TDAG8 independent at about 90 % (<B>Fig. 18 a</B>). Against our expectations, HEK293T cells without TDAG8 induced with forskolin and IBMX show a higher amount of eCFP positive cells than cells transiently expressing TDAG8. Since high basal expression is observed and the number of fluorescence positive cells does not change with induction, mean fluorescence intensity was evaluated as an alternative data analysis strategy (<B>Fig. 18 b</B>).</P><P>Fluorescence intensity of cells without TDAG8 increases towards lower pH showing a 1.3-fold increase from pH 7.7 to 6.5. As HEK293T are not supposed to have TDAG8 receptors, other pathways might activate the <I>CRE</I> promoter. HEK293T containing transiently introduced TDAG8 show on average 20 % lower fluorescence intensity than cells without the receptor. HEK293T induced additionally with forskolin and IBMX without TDAG8 show a 40 % higher fluorescence intensity than the cells with TDAG8. These findings lead to the conclusion that transiently transfected cells show lower expression caused by the stress of the PEI transfection. Overall, <I>4xCRE-pTal:eCFP</I> in HEK293T cells is leaky, which would have to be optimized in further steps. Another cell line may be better suited for initial promoter analysis. As Jurkat cells are closer to the intended application, <I>4xCRE-pTal:eCFP</I> need to be optimized in this cell line.
</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 18: Flow cytometry analysis of cAMP response element promoter activity.</STRONG>
HEK293T cell line expressing <I>4xCRE-pTal:eCFP</I> with and without transient TDAG8 were induced by pH-adjusted RPMI 1640. <B>a)</B> The percentage of eCFP positive cells and <B>b)</B> mean fluorescence intensity are presented. All data points are mean values of triplicates, error bars represent standard deviation. Significant differences were determined using one-tailed student’s t-test (Excel 2017); * p &lt; 0.05, ** p &lt; 0.01, *** p &lt; 0.001, non-significant differences are not marked.
                       </P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="item"><BUTTON class="accordion">CTLA4 Promoter</BUTTON><DIV class="panel panel-default"><DIV class="item"><H2>Characterization of the <I>CTLA4</I> Promoter</H2><P>The <I>CTLA4</I> promoter (<I>pCTLA4</I>), which is <A href="https://2017.igem.org/Team:Freiburg/Design">VEGF-A</A> responsive, was characterized using <A href="https://2017.igem.org/Team:Freiburg/Cell_culture">Jurkat and HEK293T cells</A>. Stable cell lines were generated containing <I>pCTLA4</I> with quadruple NFAT binding sites (<I>NFATbs</I>) and the TATA like minimal promoter <A href="https://2017.igem.org/Team:Freiburg/Cloning"><I>pTal</I></A> (Mahindhoratep <I>et al.</I>, 2014) expressing <I>eCFP</I> as reporter gene. The VEGF receptor 2 (VEGFR-2) was added by PEI transfection for HEK293T cells, which do not express this gene (Liu <I>et al.</I>, 2014). In order to characterize <I>pCTLA4</I>, transfected cells were induced with different concentrations of VEGF-A for 24 h. To investigate if the promoter can be activated via artificially increasing the intracellular Ca<SUP>2+</SUP> concentration, cells were induced with ionomycin causing influx of Ca<SUP>2+</SUP> into the cells (Bittinger <I>et al.</I>, 2004).</P></DIV><DIV class="item"><H3>VEGF and Ionomycin Combined Induce <I>pCTLA4</I> in Jurkat</H3><P>In Jurkat cells, <I>4xNFATbs-pTal:eCFP</I> expression decreases with increasing VEGF-A concentration (<B>Fig. 19 a</B>). Addition of ionomycin (5 µM) inverts the response on increasing VEGF-A concentration to the expected trend (<B>Fig. 19 b</B>). Generally low expression in Jurkat cells may arise due to transcriptional inactivity in these cells. The influence of ionomycin on the expression of <I>4xNFATbs-pTal:eCFP</I> combined with VEGF-A can not be evaluated as the results do not allow conclusions without further experiments. Results indicate that VEGF-A is not an optimal input for our AND gate. Further experiments have to be performed as the CD4<SUP>+</SUP> Jurkat cell line is not identical to primary T cells.</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><B>Figure 19: Flow cytometry analysis of <I>CTLA4</I> promoter activity.</B>
Jurkat cells stably transduced with <I>4xNFATbs-pTal:eCFP</I> were incubated 24 h with indicated concentrations of <B>a)</B> VEGF-A and <B>b)</B> VEGF-A combinated with ionomycin (5 µM). Data points are mean values of triplicates, error bars represent standard deviation. Significant differences were determined using one-tailed student’s t-test (Excel 2017) followed by Bonferroni-Holm correction; * p &lt; 0.05, ** p &lt; 0.01, *** p &lt; 0.001, non-significant differences are not marked.</P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV><DIV class="item"><H3>The <I>CTLA4</I> Promoter Shows High Activity in HEK293T Cells</H3><P>In HEK293T cells, <I>4xNFATbs-pTal:eCFP</I> expression is already on a high level of 70 % positive cells without any treatment (<B>Fig. 20</B>). Addition of VEGF-A (50 ng/ml) has no significant effect on the amount of eCFP positive cells. HEK293T cells with transiently induced VEGFR-2 show an average 7 % higher expression in general but no response on VEGF-A (50 ng/ml) as well.
High basal expression can originate from high transcriptional activity in HEK293T (Thomas <I>et al.</I>, 2005). Using another minimal promoter could improve the results. Optimization of promoter expression should be done in Jurkat cells as this cell line is closer to the intended application in primary T cells.</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 20: Flow cytometry analysis of <I>CTLA4</I> promoter activity.</STRONG>
HEK293T cells stably transduced with <I>4xNFATbs-pTal:eCFP</I> containing transient VEGFR-2 were incubated 24 h with indicated concentrations of VEGF-A. Data points are mean values of triplicates, error bars represent standard deviation. Significant differences were determined using one-tailed student’s t-test (Excel 2017) followed by Bonferroni-Holm correction; * p &lt; 0.05, ** p &lt; 0.01, *** p &lt; 0.001, non-significant differences are not marked.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="item"><BUTTON class="accordion">HRE Promoter</BUTTON><DIV class="panel panel-default"><DIV class="item"><H2>Characterization of the HRE Promoter</H2><P>
Gene expression specific to low oxygen concentrations (hypoxia) as given in the <A href="https://2017.igem.org/Team:Freiburg/Tumor_microenvironment">tumor microenvironment</A> was tested with promoters containing hypoxia response elements (<I>HREs</I>) (Schödel <I>et al.</I>, 2011). However, it is difficult and expensive to cultivate <A href="https://2017.igem.org/Team:Freiburg/Cell_culture">mammalian cells</A> under hypoxic conditions. The transcription factor hypoxia inducible factor 1 alpha (HIF1A), which binds to <I>HREs</I>, is regulated by proline hydroxylase domain proteins (PHDs). Like in the absence of oxygen, modification of HIF1A by ΡΗDs can also be inhibited by cobalt dichloride (CoCl<SUB>2</SUB>), which was therefore used to mimic hypoxia (Epstein <I>et al.</I>2001).</P><P>
Jurkat and HEK293T cell lines with <A href="https://2017.igem.org/Team:Freiburg/Cell_culture">stable integration</A> of the TATA-like minimal promoter (<I>pTal</I>; Mahindhoratep <I>et al.</I>, 2014), <A href="https://2017.igem.org/Team:Freiburg/Cloning">modified to contain quadruple <I>HRE</I></A>, expressing <I>eCFP</I> were analyzed. Alternatively, single enhancer element promoters were introduced by <A href="https://2017.igem.org/Team:Freiburg/Cell_culture">PEI transfection</A> into CHO-K1, a less transcriptionally active cell line. For analysis, cells were incubated with CoCl<SUB>2</SUB> for 24 h prior to measurement by flow cytometry. Concentrations used vary between the cell lines due to different onset of toxicity.</P><H3><I>pHRE</I> is Activated by Hypoxia in Jurkat</H3><P>
Flow cytometry of Jurkat <I>4xHRE-pTal:eCFP</I> cells shows increasing eCFP fluorescence correlating with moderate rising treatment (<B>Fig. 21</B>). Promoter activity in this most relevant cell line is strongest at 80 μM CoCl<SUB>2</SUB>, decrease of signal at greater levels of induction may stem from toxicity of CoCl<SUB>2</SUB>. Overall low expression is likely due to weak transcriptional activity inherent in this cell line (Michel <I>et al.</I>, 2017).</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><B>Figure 21: Flow cytometry analysis of hypoxia response element promoter in Jurkat.</B> Jurkat cells stably expressing <I>4xHRE-pTal:eCFP</I> were incubated 24 h with indicated concentrations of CoCl<SUB>2</SUB>. Data points are mean values of triplicates, error bars represent standard deviation. Significant differences were determined using one-tailed student’s t-test (Excel 2017) followed by Bonferroni-Holm correction; * p &lt; 0.05, ** p &lt; 0.01, *** p &lt; 0.001, non-significant and decreasing differences are not marked.
</P></DIV></DIV></DIV></DIV></DIV><DIV class="item"><H3><I>pHRE</I> Activity in HEK293T</H3><P>In HEK293T <I>4xHRE-pTal:eCFP</I> expression is already fully activated at 90 % when untreated and decreases with addition of CoCl<SUB>2</SUB> (<B>Fig. 22</B>). Less fluorescence with increasing treatment may represent toxicity. Since it should show low expression without induction, the minimal promoter is leaky in this cell line. High basal expression can originate from high transcriptional activity in HEK293T (Thomas <I>et al.</I>, 2005), which is consistent with findings for the <I>CRE</I> promoter. That construct is also based on <I>pTal</I>, therefore usage of an alternative minimal promoter may improve results. As these cells were a testing system, and Jurkat cells are closer to the intended application of the CARTEL<SUP>TM</SUP> AND gate, promoter expression should be optimized in Jurkat.</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><B>Figure 22: Flow cytometry analysis of hypoxia response element promoter in HEK293T.</B>HEK293T cells stably expressing <I>4xHRE-pTal:eCFP</I> were incubated 24 h with indicated concentrations of CoCl<SUB>2</SUB>. Data points are mean values of triplicates, error bars represent standard deviation. Significant differences were determined using one-tailed student’s t-test (Excel 2017) followed by Bonferroni-Holm correction; * p &lt; 0.05, ** p &lt; 0.01, *** p &lt; 0.001, non-significant and decreasing differences are not marked.</P></DIV></DIV></DIV></DIV></DIV><DIV class="item"><H3>Hypoxia Induces <I>pHRE</I> Expression in CHO-K1</H3><P>As HEK293T proved to be an inadequate promoter activation test system due to leakiness, the constructs <I>1xHRE-pTal:eCFP</I> and <I>4xHRE-pTal:eCFP</I> were transiently transfected into CHO-K1 cells for measurement by flow cytometry (<B>Fig. 23</B>). Rising promoter activity with increased treatment was observed with both constructs. In comparison, fluorescence increase for activation with 80 µM CoCl<SUB>2</SUB> rises from 1.3-fold with single <I>HRE</I> (<B>Fig. 23 a</B>) to 1.5-fold when possessing four <I>HREs</I> (<B>Fig. 23 b</B>). The two constructs significantly differ with p &lt; 0.001, determined by ANOVA (Excel 2017). This indicates that the direction for further improvement of this expression construct, also in T cells, could be addition of even more enhancer elements. Subsequent to optimization of the <I>HRE</I> promoter in Jurkat, the target primary T cells should be tested.</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><B>Figure 23: Flow cytometry analysis of hypoxia response element promoter CHO-K1.a)</B><I>1xHRE-pTal:eCFP</I> or <B>b)</B><I>4xHRE-pTal:eCFP</I> PEI transfected into CHO-K1 cells. Cells were incubated 24 h with indicated concentrations of CoCl<SUB>2</SUB>. Data points are mean values of triplicates, error bars represent standard deviation. Significant differences were determined using one-tailed student’s t-test (Excel 2017) followed by Bonferroni-Holm correction; * p &lt; 0.05, ** p &lt; 0.01, *** p &lt; 0.001, non-significant and decreasing differences are not marked.</P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV><DIV class="item"><P>Hypoxia pathway activation by CoCl<SUB>2</SUB> in accordance with literature (Moroz <I>et al.</I>, 2009) was confirmed via Western Blot assay of HIF1A stabilization (<B>Fig. 24</B>) on samples analyzed in the flow cytometer. Detection of HIF1A excludes the possibility of failure to induce hypoxia. A strong signal at 5 μM CoCl<SUB>2</SUB> indicates, that high fluorescence measured by flow cytometry (<B>Fig. 21</B>) in this condition was caused by accidentally increased dose of this treatment.</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><B>Figure 24: Western Blot detection of HIF1A in Jurkat <I>4xHRE-pTal:eCFP</I>.</B>Cells were induced with CoCl<SUB>2</SUB> for fluorescence analysis by flow cytometry. Following sample preparation for flow cytometry, the remaining cells of each triplicate were pooled and lysed for immunoblot detection with anti-HIF1A antibody. Anti-GAPDH detection is shown as loading control.</P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="item"><H2 style="font-size:40px;">Suicide Gene</H2><P>To improve the safety of the CARTEL<SUP>TM</SUP> cells, a kill switch was implemented in order to specifically eliminate CAR T cells which cause side effects. The suicide gene <I>herpes simplex virus 1</I> thymidine kinase (<I>HSV-TK</I>) can be used to induce apoptosis by application of ganciclovir (GCV).</P></DIV><DIV class="item"><BUTTON class="accordion">Suicide Gene</BUTTON><DIV class="panel panel-default"><DIV class="item"><H2>Ganciclovir Viability Assay</H2><P> The experiment was designed to identify the toxic dose in untransfected cells in addition to testing the response of <I>HSV-TK</I> transduced cells. Therefore a range of concentrations from 0.1 µg/ml to 100 µg/ml was used. Identical numbers of cells were seeded for all conditions and counted in 24 h intervals with trypan blue as cell death marker. The survival rate was calculated as the ratio of living cells to total cells. Medium was changed every two days for Jurkat cells and daily for HEK293T cells.</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 25: Survival curve of Jurkat cells treated with ganciclovir.</STRONG>
Depicted is the survival rate in percent after application of ganciclovir. The assay was done for 120 h by counting living cells of wild type (WT) and stable cell lines containing the suicide gene <I>HSV-TK</I> (SG). All experiments were performed in triplicates, data represents mean ± SD.</P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 26: Survival curve of HEK293T cells treated with ganciclovir.</STRONG>
Depicted is the survival rate in percent after application of ganciclovir. The assay was done for 72 h by counting living cells of wild type (WT) and stable cell lines containing the suicide gene <I>HSV-TK</I> (SG). All experiments were performed in triplicates, data represents mean ± SD.</P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV><DIV class="item"><P>For Jurkat <I>HSV-TK</I> cells with all GCV concentrations used, survival dropped to nearly 0 % at 120 h of treatment (<B>Fig. 25</B>). HEK293T cells expressing the suicide gene were less persistent and reached this survival rate after 72 h (<B>Fig. 26</B>). Wild type (WT) cells of both lines as well as untreated cells did not exhibit significant mortality, except Jurkat WT under 100 µg/ml GCV, probably due to toxicity of this compound.</P></DIV><DIV class="item"><P>The minimum concentration required to induce apoptosis in transduced cells is below the range of concentrations tested here. The physiological levels reached by clinical dosage correspond to the range tested here (Verzeletti <I>et al.</I>, 1998).</P><P>Literature values state that <I>HSV-TK</I> expressing cells have a 1000 times higher affinity to GCV than wild type cells (Dey &amp; Evans, 2011). The difference in affinity between wild type cells and <I>HSV-TK</I> cells is displayed by the divergence of the respective survival rates.</P><P>GCV proved to be toxic to Jurkat WT cells at 100 µg/ml. For HEK293T WT cells the toxic dose was not identified.</P><P>We were able to show that we can add an additional level of safety to our project design by transducing T cells with <I>HSV-TK</I> in combination with our <A href="https://2017.igem.org/Team:Freiburg/Design">CARTEL<SUP>TM</SUP> AND gate</A>. Hereby we addressed the concerns raised by the public with a solution inspired by Prof. Cathomen.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><DIV class="flex-container"><DIV class="col-sm-6"><DIV class="image_box left" style="float:right"><DIV class="figure" style="width:195px"><DIV class="figureinner"><DIV class="figurecaption-car" style="float:right"><P><STRONG>Knockout</STRONG></P></DIV></DIV></DIV></DIV></DIV><DIV class="col-sm-6"><DIV class="image_box right" style="float:left"><DIV class="figure" style="width:180px"><DIV class="figureinner"><DIV class="figurecaption-car" style="float:left"><P><STRONG>Proof of Concept</STRONG></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>