<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Freiburg_Lab_Knockout skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Freiburg/Lab Knockout</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="flex-container1"><DIV class="flex-item2"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Introduction">Overview</A><A href="https://2017.igem.org/Team:Freiburg/Motivation">Motivation</A><A href="https://2017.igem.org/Team:Freiburg/CAR">CAR T Cells</A><A href="https://2017.igem.org/Team:Freiburg/Tumor_microenvironment">Tumor Microenvironment</A><A href="https://2017.igem.org/Team:Freiburg/Design">AND Gate</A><A href="https://2017.igem.org/Team:Freiburg/Outlook">Outlook</A><A href="https://2017.igem.org/Team:Freiburg/Achievements">Achievements</A><A href="https://2017.igem.org/Team:Freiburg/References">References</A></DIV></DIV></DIV><DIV class="flex-item3"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Team">Members</A><A href="https://2017.igem.org/Team:Freiburg/Attributions">Attributions</A><A href="https://2017.igem.org/Team:Freiburg/Sponsors">Partners</A><A href="https://2017.igem.org/Team:Freiburg/Contact">Contact</A></DIV></DIV></DIV><DIV class="flex-item4"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Results">Main Project</A><A href="https://2017.igem.org/Team:Freiburg/Model">Modeling</A><A href="https://2017.igem.org/Team:Freiburg/Applied_Design">Applied Design</A><A href="https://2017.igem.org/Team:Freiburg/Demonstrate">Proof of Concept</A><A href="https://2017.igem.org/Team:Freiburg/Basic_Part">BioBricks Basic Part</A><A href="https://2017.igem.org/Team:Freiburg/Improve">BioBrick Improvement</A><A href="https://2017.igem.org/Team:Freiburg/InterLab">Interlab Study</A></DIV></DIV></DIV><DIV class="flex-item5"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Safety">Safety</A><A href="https://2017.igem.org/Team:Freiburg/Cloning">Cloning</A><A href="https://2017.igem.org/Team:Freiburg/Lab_Knockout"><I>HIF1A</I> Knockdown</A><A href="https://2017.igem.org/Team:Freiburg/Cell_culture">Cell Culture</A><A href="https://2017.igem.org/Team:Freiburg/Notebook">Lab Book</A></DIV></DIV></DIV><DIV class="flex-item6"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/HP/Silver">Human Practice</A><A href="https://2017.igem.org/Team:Freiburg/HP/Gold_Integrated">Integrated Human Practice</A><A href="https://2017.igem.org/Team:Freiburg/Collaborations">Collaborations</A></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><DIV class="flex-container"><H1 align="center"><I>HIF1A</I> Knockdown</H1><DIV class="item"><H2>Elimination of <I>HIF1A</I></H2><P>The genetic circuit designed to control <A href="https://2017.igem.org/Team:Freiburg/CAR">chimeric antigen receptor (CAR)</A> expression in the <A href="https://2017.igem.org/Team:Freiburg/Tumor_microenvironment">tumor microenvironment</A> relies on exclusive control of hypoxia-inducible factor 1 alpha (HIF1A) by the introduced <A href="https://2017.igem.org/Team:Freiburg/Design">AND gate</A>. Various strategies have been employed to obtain HIF1A deficient cells.
</P></DIV><DIV class="item"><H3>CRISPR/Cas9 with Homology-Directed Repair</H3><P>The CRISPR/Cas9 (Ran <I>et al.</I>, 2013) system (clustered regularly interspaced short palindromic repeats / CRISPR-associated protein 9) allows precise editing of genes. The Cas9 endonuclease forms a complex with guide RNA (gRNA) and introduces DNA double strand breaks at sites complementary to the gRNA (<B>Fig. 1</B>). These are repaired by different endogenous repair mechanisms, dependent on the circumstances. One process called homology-directed repair (HDR) can be triggered by co-transformation with a repair template containing regions homologous to the target.</P><P>A plasmid kit with Cas9 from <I>S. pyogenes</I> was used to knockout <I>HIF1A</I> by HDR (sponsored by <A href="http://www.origene.com/CRISPR-CAS9/KN202461/HIF1A.knockout" target="_blank">OriGene</A>). This was conducted with two plasmids containing different gRNAs, both targeting the first exon of human <I>HIF1A</I>, close to the transcription initiation site. The reporter genes for green fluorescent protein (GFP) and puromycin resistance were stably integrated via a repair template.</P></DIV><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 1: Schematic overview of the CRISPR/Cas9 mechanism.</STRONG>
Cas9 endonuclease recognizes the desired target via co-transfected guide RNA (gRNA), which contains 20 nucleotides adjacent to a NGG site complementary to the target gene. The Cas9 induces double strand breaks in the target 3-4 nucleotides upstream of NGG. Endogenous mechanisms repair the break by non-homologous end joining (NHEJ), producing random insertions and deletions, or by homology-directed repair (HDR), which integrates a repair template at the cleavage site.</P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN><DIV class="modal-content"><DIV class="mySlides"><DIV class="numbertext">CRISPER-overview</DIV></DIV><DIV class="mySlides"><DIV class="numbertext">siRNA </DIV></DIV><DIV class="mySlides"><DIV class="numbertext">T7EN </DIV></DIV></DIV></DIV><DIV class="item"><H3>CRISPR/Cas9 induced Non-Homologous End Joining</H3><P>The CRISPR/Cas9 system can also be used without repair template. This forces cells to repair the double strand break by non-homologous end joining (NHEJ), an error-prone process. At the cleavage site nucleotides are randomly added and removed during repair resulting in deletions, substitutions and insertions. This produces stochastically shifts of the translational frame in the gene, ablating the gene product’s function.</P><P>To employ this strategy, gRNAs were designed using the Broad Institute’s Genetic Perturbation Platform (<A href="https://portals.broadinstitute.org/gpp/public/analysis-tools/sgrna-design" target="_blank">GPP</A>; Doench <I>et al.</I>, 2016). Off-target effects were predicted using the program CRISPR/Cas9 Target Online Predictor (<A href="https://crispr.cos.uni-heidelberg.de/" target="_blank">CCTOP</A>; Stemmer <I>et al.</I>, 2015). The most promising candidates were inserted into plasmids coding for Cas9 (<I>S. pyogenes</I>) with either GFP or puromycin resistance as selection marker (Ran <I>et al.</I>, 2013).</P></DIV><DIV class="item"><H3>RNA Interference</H3><P>Another technique to inhibit gene expression is RNA interference, where translation is inhibited by antisense RNA binding to target mRNA (Fire <I>et al.</I>, 1998). This cleaves the mRNA or prevents association of the translational machinery and therefore protein production. Our approach involved stable lentiviral transduction of such short hairpin RNA (shRNA) into cells to obtain downregulation of <I>HIF1A</I> mRNA. These shRNAs rely on the endogenous post-transcrptional silencing (PTGS) machinery (<B>Fig. 2</B>).</P><P>Since the knockdown of <I>HIF1A</I> gene product in our project is used to allow control of this gene’s expression by a synthetic construct, it is crucial that the designed shRNA sequences are complementary to endogenous mRNA but not that of the introduced gene. Therefore, the 3'UTR of this transcript was used as target for shRNA sequence design. The lentiviral transfer plasmid into which shRNA sequences were cloned contained cyan fluorescent protein (CFP) and neomycin resistance markers.</P></DIV><DIV class="column"><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 2: Schematic mechanism of RNA interference.</STRONG> After integration of an shRNA gene into the genome and its transcription, the precursor shRNA is processed by DICER into siRNA. Mature siRNA associates with several proteins into the RNA-induced silencing complex (RISC) which is able to silence the desired gene via blocking translation by the ribosome or by initiating degradation of the target mRNA via PTGS.</P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV></DIV><DIV class="item"><BUTTON class="accordion">Knockdown/-out Methods</BUTTON><DIV class="panel panel-default"><DIV class="item"><H2>Methods</H2><P>CRISPR strategies were applied by transient transfection and knockdown was done via <A href="https://2017.igem.org/Team:Freiburg/Cell_culture">lentiviral transduction</A> followed by fluorescence-activated cell sorting of positive cells. Insertion of mutations was assessed by PCR or T7E1 assay and the absence of gene product was confirmed by Western Blot. The obtained data is described on the <A href="https://2017.igem.org/Team:Freiburg/Results">results page</A>.</P></DIV><DIV class="item"><H3>T7E1 Assay</H3><P>To assess insertion of small mutations T7 endonuclease 1 (T7E1) assay was performed. T7E1 specifically cleaves mismatched dsDNA (Parkinson <I>et al.</I>, 1997). From Cas9 treated clonal <A href="”https://2017.igem.org/Team:Freiburg/Cell_culture”">cells</A> and non-transfected cells gDNA was extracted using KAPA Biosystems Express gDNA extraction kit according to the manufacturer's protocol. From this the Cas9 gRNA target sequences were amplified by <A href="https://2017.igem.org/Team:Freiburg/Cloning">PCR</A> and subsequently purified using PCR purification kit (Quiagen).
For each Cas9 treated clone one pure sample and a 1:1 mix with wild type gDNA PCR fragment was analyzed. The samples were melted for 10 min at 95ºC in NEB2 buffer (1x) and slowly re-annealed by switching off the heat block and letting them cool down in it for at least 2.5 h to form mismatched double strands (<B>Fig. 3</B>). Then 1.5 μl 1:1 dilution of T7E1 enzyme (NEB) with NEB2 (2x) was added to 300 ng annealed PCR fragment in 12 µl NEB2 (1x). The reaction was incubated 20 min at 37ºC and stopped by addition of gel loading dye on ice for subsequent analysis on ethidium bromide stained agarose gel (2 %).
As positive control served the monoallelic <I>CCR5Δ32</I> mutation of HEK293T cells. If only the mix of knockout and untreated fragments is cleaved by T7E1, the mutation is present in both alleles, however, if the pure treated sample is cleaved only one allele was mutated.</P></DIV><DIV class="column"><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 3: Workflow of  T7 endonuclease 1 (T7E1) assay.</STRONG> To investigate the presence of mutations, genomic DNA (gDNA) is extracted from cells. The target sequence of the genome is amplified by PCR. The resulting fragments are denatured and reannealed to allow for mismatched double strand formation caused by mutations. Next, the DNA is treated with T7E1 which cleaves mismatches. Cleaved fragments are visualized by agarose gelelectrophoresis.</P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN></DIV></DIV></DIV><DIV class="item"><H3>Western Blot</H3><H4>Cell Lysis </H4><P>
                HEK and Jurkat cells were induced 24 hours prior to lysis with different concentration of CoCl<SUB>2</SUB> for stabilization of HIF1A. For the lysis cells were collected using the appropriate method. The cells were washed with ice cold PBS and subsequently covered with the according amount of 1x RIPA buffer (<A href="http://www.sigmaaldrich.com/catalog/product/mm/20188?lang=de&amp;region=DE" target="_blank">Sigma Aldrich</A>) containing protease inhibitor cocktail (<A href="http://www.sigmaaldrich.com/catalog/product/sigma/p2714?lang=de&amp;region=DE" target="_blank">Sigma Aldrich</A>) and for detection of HIF1A 100 μM CoCl<SUB>2</SUB>.The cells were sonicated for 10 minutes followed by centrifugation at 20,000 x g and 4°C for 20 minutes. Hereafter the supernatant was transferred into new tubes and the appropriate volume of SDS sample buffer was added. The samples were boiled at 95°C for five minutes and stored at -20°C.
                </P><H4>Amido Black Assay</H4><P>
                Amido black was used to quantify total protein in protein extracts. Therefore, 1 ml of amido black solution was added to 20 μl of the cell lysate following a centrifugation at maximum speed and room temperature for 15 minutes. The supernatant was discarded and 1 ml destain solution was added, after inversion the samples were centrifuged at maximum speed and room temperature for 15 minutes. The supernatant was discarded, subsequently, the pellet was air-dried and resuspended in 1 ml 0.2 M NaOH. The OD<SUB>620</SUB> was measured using a nanodrop spectrometer. The amount of protein in the extract was calculated with a BSA standard curve.
                </P><H4>SDS-PAGE</H4><P>
                SDS-PAGE gels were prepared according to <B>Table 1</B>. The gels were placed into the electrophoresis chamber and the latter was filled with running buffer. Equal amounts of protein was loaded on each lane, 5 μl of <A href="https://www.thermofisher.com/order/catalog/product/26619" target="_blank">Thermo Scientific prestained protein ladder</A> was loaded as a marker. For two gels the power supply was set to 20 mA following 50 mA once the front had entered the running gel. Gels were stopped once the 10 kDa ladder band had reached the bottom.
                </P><P><B>Table1: SDS gels.</B></P><H4>Western Blot</H4><P>
                PVDF membrane (<A href="https://www.carlroth.com/de/de/Aktuelles-Mailing/Sonderpreise/Life-Science/Transfer-%26-Blotting/Membranen/Transfermembranen-aus-PVDF/Roti%3Csup%3E%C2%AE%3C-sup%3E-PVDF/p/000000070002785700020023_de" target="_blank">Carl Roth</A>) was covered with methanol and subsequently equilibrated in blotting buffer for 10 minutes. The blotting sandwich was assembled and placed in the blotting chamber. The blot was run at 250 mA for two hours. Hereafter, the membrane was stained with amido black solution for one minute and washed with deionized water. The amido black stain was followed by blocking with blocking buffer for two hours and three wash steps each lasting 10 minutes. Subsequently, the membrane was incubated overnight with primary antibody at 4°C, followed by three wash steps and incubation with secondary antibodies.<A href="http://www.sigmaaldrich.com/catalog/product/sigma/gerpn2232?lang=de&amp;region=DE&amp;gclid=EAIaIQobChMIh4eeiJ6Z1wIVsTLTCh33bwDDEAAYAiAAEgL7_fD_BwE" target="_blacnk">ECL solution</A> was prepared according to the manufacturer's protocol, spread on the membrane and the luminescence recorded in a camera chamber. </P><H3>Buffer List</H3><H4>Amidoblack destain solution</H4><P>10 % (v/v) acetic acid in EtOH</P><H4>Amidoblack solution</H4><P>10 % (v/v) acetic acid
0.05 % (w/v) amido black
in MeOH</P><H4>Blocking buffer</H4><P>5 % skim milkpowder in TBS-T</P><H4>1 x Blotting Buffer</H4><P>50 mM Tris (121.14 gmol<SUP>-1</SUP>)
100 mM Glycine
0.01 % SDS
10 % MeOH
adjust pH to 8.3</P><H4>100 µg/µl BSA</H4><P>1 g BSA
10 ml ddH<SUB>2</SUB></P><H4> 0.2 NaOH</H4><P>4 g NaOH (40.0 g mol<SUP>-1</SUP>
to 500 ml ddH<SUB>2</SUB></P><H4>60 % Sucrose</H4><P> 60 g Sucrose
to 100 ml ddH<SUB>2</SUB>O</P><H4>20 % SDS</H4><P>40 g SDS
to 200 ml ddH<SUB>2</SUB>O</P><H4>2 M Tris</H4><P>242.28 g Tris (121.14 gmol<SUP>-1</SUP>)
to 1000ml ddH<SUB>2</SUB>O</P><H4>TBS-T</H4><P>20 ml 2 M Tris
125 ml 4 M NaCl
500 µl Tween 20
to 1000 ml ddH<SUB>2</SUB>
adjust pH to 7.5 with HCl</P><H4>1 M Tris pH 8.8</H4><P> 100 ml 2 M Tris
to 200 ml ddH<SUB>2</SUB>O
adjust pH to 8.8 with HCl
autoclave for 25 min. </P><H4>0.5 M Tris pH 6.8</H4><P>50 ml 2 M Tris
to 200 ml ddH<SUB>2</SUB>O
adjust pH to 6.8 with HCl
autoclave for 25 min. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><DIV class="flex-container"><DIV class="col-sm-6"><DIV class="image_box left" style="float:right"><DIV class="figure" style="width:195px"><DIV class="figureinner"><DIV class="figurecaption-car" style="float:right"><P><STRONG>AND Gate</STRONG></P></DIV></DIV></DIV></DIV></DIV><DIV class="col-sm-6"><DIV class="image_box right" style="float:left"><DIV class="figure" style="width:180px"><DIV class="figureinner"><DIV class="figurecaption-car" style="float:left"><P><STRONG>Results</STRONG></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>