<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Freiburg_Cloning skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Freiburg/Cloning</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="flex-container1"><DIV class="flex-item2"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Introduction">Overview</A><A href="https://2017.igem.org/Team:Freiburg/Motivation">Motivation</A><A href="https://2017.igem.org/Team:Freiburg/CAR">CAR T Cells</A><A href="https://2017.igem.org/Team:Freiburg/Tumor_microenvironment">Tumor Microenvironment</A><A href="https://2017.igem.org/Team:Freiburg/Design">AND Gate</A><A href="https://2017.igem.org/Team:Freiburg/Outlook">Outlook</A><A href="https://2017.igem.org/Team:Freiburg/Achievements">Achievements</A><A href="https://2017.igem.org/Team:Freiburg/References">References</A></DIV></DIV></DIV><DIV class="flex-item3"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Team">Members</A><A href="https://2017.igem.org/Team:Freiburg/Attributions">Attributions</A><A href="https://2017.igem.org/Team:Freiburg/Sponsors">Partners</A><A href="https://2017.igem.org/Team:Freiburg/Contact">Contact</A></DIV></DIV></DIV><DIV class="flex-item4"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Results">Main Project</A><A href="https://2017.igem.org/Team:Freiburg/Model">Modeling</A><A href="https://2017.igem.org/Team:Freiburg/Applied_Design">Applied Design</A><A href="https://2017.igem.org/Team:Freiburg/Demonstrate">Proof of Concept</A><A href="https://2017.igem.org/Team:Freiburg/Basic_Part">BioBricks Basic Part</A><A href="https://2017.igem.org/Team:Freiburg/Improve">BioBrick Improvement</A><A href="https://2017.igem.org/Team:Freiburg/InterLab">Interlab Study</A></DIV></DIV></DIV><DIV class="flex-item5"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/Safety">Safety</A><A href="https://2017.igem.org/Team:Freiburg/Cloning">Cloning</A><A href="https://2017.igem.org/Team:Freiburg/Lab_Knockout"><I>HIF1A</I> Knockdown</A><A href="https://2017.igem.org/Team:Freiburg/Cell_culture">Cell Culture</A><A href="https://2017.igem.org/Team:Freiburg/Notebook">Lab Book</A></DIV></DIV></DIV><DIV class="flex-item6"><DIV class="dropdown"><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Freiburg/HP/Silver">Human Practice</A><A href="https://2017.igem.org/Team:Freiburg/HP/Gold_Integrated">Integrated Human Practice</A><A href="https://2017.igem.org/Team:Freiburg/Collaborations">Collaborations</A></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><DIV class="flex-container"><H1 align="center">Cloning</H1><DIV class="item"><H2>Cloning strategy</H2><P>
                    Our aim was to develop a cloning strategy providing us with plasmids that could be used for transient expression, to perform pretests of our CARTEL<SUP>TM</SUP> AND gate, and as templates to generate transfer plasmids for <A href="https://2017.igem.org/Team:Freiburg/Cell_culture">lentiviral transduction</A>. This resulted in the following cloning strategy.
                    </P></DIV><DIV class="column"><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 1: Summary of cloning strategy to generate plasmids suitable for transient transfection. </STRONG>
                     Shown are plasmids type one, that can already be used for promoter characterization pretests. Plasmids type two, that were built as subcloning plasmids to be assembled with plasmids type one into plasmids type three. Plasmids type four were generated by ligating four enhancer elements in succession, built by compatible end assembly, into plasmids type three. </P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="item"><P>
                As a first step, two plasmid types were generated. Plasmid type number one, assembled via Gibson Assembly (for a description see below), contained from 5’ end to 3’ direction one of the enhancers (<A href="https://2017.igem.org/Team:Freiburg/Results">HRE/CRE/pCTLA4</A>), a minimal promoter to ensure better recruitment of the transcription machinery, one reporter protein (<A href="https://2017.igem.org/Team:Freiburg/Cell_culture">eCFP/SEAP/Luciferase</A>) and a polyA signal sequence. The polyA signal was included to ensure better release of the transcribed mRNA from the transcription machinery and longer mRNA half-life (Hager <I>et al.</I>, 2008). These plasmids were already suitable for pretests but were lacking a transfection control or a selection marker for lentiviral transduction (<B>Fig. 1</B>).
                </P><P>
               The second plasmid type, also assembled via Gibson Assembly, contained from 5’ end to 3’ a constitutive promoter (CMV/SV40), a selection marker (mCherry/Zeocin/Neomycin) and a polyA signal sequence. These plasmids sole purpose was to be assembled with type one into type three plasmids. In a second step, plasmids type one and type two were assembled via a Gibson assembly into type three plasmids (<B>Fig. 1</B>). 
                </P><P>
                Considering that one enhancer may only lead to very weak transactivation and therefore low reporter gene expression, plasmids containing an enhancer flanked by compatible restriction sites, suitable for compatible end assembly, were designed. To generate four enhancer elements in succession, compatible end assembly (for a description see below) was performed twice. Additionally, two restriction sites, that were compatible with type three plasmids, were included, so that in a third step, type three plasmids and the four enhancer elements in succession could be combined via T4 ligation (for a description see below) (<B>Fig. 1</B>). In the fourth step, the assembled plasmids were cloned into a transfer plasmid via Gibson Assembly (<B>Fig. 2</B>). 
                </P></DIV><DIV class="column"><DIV class="item"><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 2: Summary of cloning strategy to generate transfer plasmids for lentiviral transduction.</STRONG>
                     Plasmids type four were used as template to clone the constructs into transfer plasmids for lentiviral transduction. </P></DIV></DIV><DIV id="myModal" class="modal"><SPAN class="close cursor" onclick="closeModal()">×</SPAN><DIV class="modal-content"><DIV class="mySlides"><DIV class="numbertext">cloning strategy 2 </DIV></DIV><DIV class="mySlides"><DIV class="numbertext">plasmid overview </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="item"><H2>Cloning Results</H2><P>
The strategy described above was designed to generate compatible plasmids for easy exchange, for  e.g. reporter proteins or selection markers, providing us with our own cloning standard. During the time in the lab the following plasmids were cloned. If two or more plasmids of a certain type shared similar features, they were depicted in an overview scheme (<B>Fig. 3</B>).
                </P><DIV class="image_box middle"><DIV class="figure"><DIV class="figureinner"><DIV class="figurecaption"><P><STRONG>Figure 3: Cloning Results</STRONG>
Depicted are schemes of type one, three, four and transfer plasmids. Type two and other subcloning plasmids are not shown. Plasmids of any type, if they shared similar features, were depicted in an overview scheme.

                     </P></DIV></DIV></DIV></DIV></DIV><DIV class="item"><BUTTON class="accordion">Cloning Methods</BUTTON><DIV class="panel panel-default"><DIV class="item"><H2>Cloning</H2><P>
	If not otherwise stated all methods were done according to the protocol of the manufacturer. 
</P></DIV><DIV class="item"><H3>Cloning methods </H3><H4>Gibson Assembly</H4><P>Plasmids, that were judged to be assembled in the best way via Gibson Assembly, were generated using <A href="https://www.neb.com/products/e5520-nebuilder-hifi-dna-assembly-cloning-kit" target="_blank">NEBuilder HiFi DNA Assembly Kit.</A></P></DIV><DIV class="item"><H4>T4 Ligation</H4><P>Plasmids, that were judged to be assembled in the best way via Ligation, were generated using <A href="https://www.neb.com/products/e5520-nebuilder-hifi-dna-assembly-cloning-kit" target="_blank">Qiagen T4 Ligase.</A></P></DIV><DIV class="item"><H4>Digestion for cloning</H4><P>To extract desired DNA fragments for the cloning process the following protocol has been used (<B>Tab. 1</B>).</P><P><B>Table 1: Components of a restriction digest</B></P><TABLE><TBODY><TR><TH>Components</TH><TH>Concentration</TH></TR><TR><TD>DNA</TD><TD>1-2 µg</TD></TR><TR><TD>appropriate enzyme or enzymes</TD><TD>10 – 20 units</TD></TR><TR><TD>buffer</TD><TD>1x</TD></TR><TR><TD>H20</TD><TD>filled up to 50 µl</TD></TR></TBODY></TABLE><P>If two restriction enzymes were used simultaneously the right conditions were determined by using <A href="http://www.neb.sg/tools-and-resources/interactive-tools/double-digest-finder" target="_blank">'NEB Double Digest Finder'</A> or <A href="https://www.thermofisher.com/de/de/home/brands/thermo-scientific/molecular-biology/thermo-scientific-restriction-modifying-enzymes/restriction-enzymes-thermo-scientific/double-digest-calculator-thermo-scientific.html" target="_blank">'Thermo scientific double digest calculator'</A> online tool. 
The reaction was stopped after 1-4 hours by adding 10x Orange G, 6x NEB Purple or 6x Thermo Scientific loading dye to the sample. After this the sample was analyzed by agarose gel electrophoresis. 
</P></DIV><DIV class="item"><H4>Gel extraction for cloning</H4><P>Extraction of amplified DNA fragments or digested fragments from agarose gels was performed using the <A href="https://www.qiagen.com/us/shop/sample-technologies/dna/dna-clean-up/qiaquick-gel-extraction-kit/#orderinginformation" target="_blank">QIAquick gel extraction kit from Qiagen.</A></P></DIV><DIV class="item"><H4>Site directed mutagenesis</H4><P>To get rid of unwanted restriction sites, because they were forbidden by either the iGEM cloning standard or by our own, a site directed mutagenesis (SDM) was used. Here the exchange of nucleotides is accomplished by performing a PCR with a primer containing the mutated nucleotide. Afterwards, the PCR fragments were assembled via Gibson assembly or T4 ligation.</P></DIV><DIV class="item"><H4>Compatible end assembly</H4><P>To clone multiple repetitive DNA fragments in succession plasmids was generated suitable for a compatible end assembly. These plasmids contained from 5’ end to 3’ XbaI restriction site, an enhancer element, a NheI restriction site and a BglII restriction site. </P><P>To perform the compatible end assembly two different restriction digests of the same plasmid have to be performed. The first contains XbaI and BglII. The second contains NheI and BglII. After separation and extraction of the desired fragments via agarose gel electrophoresis, a T4 ligation is performed. The resulting cloning scar can no longer be cut by either NheI or XbaI. This process can be repeated as often as desired with the resulting plasmids.
</P></DIV><DIV class="item"><H3>PCR methods </H3><H4>PCR</H4><P>Amplification of DNA fragments was performed by using <A href="https://www.neb.com/products/m0491-q5-high-fidelity-dna-polymerase#Product%20Information" target="_blank">NEB Q5 High Fidelity</A> or <A href="https://www.neb.com/products/m0480-onetaq-dna-polymerase#Product%20Information" target="_blank">NEB One Taq polymerase.</A></P></DIV><DIV class="item"><H4>Extension PCR</H4><P>To generate the DNA fragments containing overlapping sequences for Gibson assembly an extension PCR was performed using <A href="https://www.neb.com/products/m0491-q5-high-fidelity-dna-polymerase#Product%20Information" target="_blank">NEB Q5 High Fidelity polymerase.</A></P></DIV><DIV class="item"><H3>Preparation and Handling</H3><H4>Culture conditions</H4><P>Liquid overnight cultures of <I>E. coli</I> strains, regardless of the fact if there were containing plasmids or not, were incubated for 12 – 16 hours in LB medium at 37°C and 200 rpm. The only exception were <I>E. coli</I> strains containing lentiviral transfer plasmids, these were incubated at 30°C to ensure higher plasmid concentrations. Solid cultures were incubated on 1.5 % LB Agar plates at 37°C. 
Selection of <I>E. coli</I> was performed by adding the appropriate antibiotic to the LB medium or LB plates. The following antibiotic concentrations were used: ampicillin at a final concentration of 100 µg/mL, chloramphenicol at 25 µg/mL and kanamycin at 50 µg/mL.
</P></DIV><DIV class="item"><H4>Glycerol stocks</H4><P>To store bacterial strains 500 µL of liquid overnight culture were mixed with 500 µL of 50 % (v/v) glycerol diluted in ddH<SUB>2</SUB>O. Afterwards the cryotube was inverted 4 to 6 times and placed in a -80°C freezer for longterm storage. For overnight cultures, the stock were scraped at the surface with an pipette tip and medium inocculated with it. 
</P></DIV><DIV class="item"><H4>Preparation of chemically competent <I>E. coli</I></H4><P>Competent <I>E. coli</I> were produced using the <A href="http://www.zymoresearch.com/e-coli/transformation-kits-accessories/transformation-kit-buffer-set" target="_blank">Zymo Research Mix &amp; Go <I>E. coli</I> Transformation Kit.</A></P></DIV><DIV class="item"><H4>Transformation</H4><P>Chemically competent <I>E. coli</I> strains were transformed by adding 2 – 10 µL of T4 ligation or Gibson assembly reaction mix containing the appropriate plasmids. Afterwards the transformed cells were incubated for 1 hour in 1000 µL LB medium containing no antibiotics, at 37°C and 300 rpm, followed by spreading on a LB agar plate containing the appropriate antibiotic resistance. The spread cells were incubated overnight at 37°C. 
Subsequently, 3 - 10 colonies were picked per plate and inoculated in 3 - 5 mL of LB medium again containing the appropriate antibiotic and incubated overnight. 
</P></DIV><DIV class="item"><H4>Plasmid preparation</H4><P>The preparation of plasmids was performed using kits from <A href="https://www.qiagen.com/us/shop/sample-technologies/dna/plasmid-dna/qiaprep-spin-miniprep-kit/#orderinginformation" target="_blank">Qiagen</A>, <A href="https://www.jenabioscience.com/molecular-biology/rna-dna-preparation-and-cleanup/plasmid-dna-purification/pp-204-fast-n-easy-plasmid-mini-prep-kit" target="_blank">Jena Bioscience</A>, <A href="https://www.promega.de/resources/protocols/technical-bulletins/101/pureyield-plasmid-miniprep-system-protocol/" target="_blank">Promega</A> and <A href="https://www.zymoresearch.com/dna/plasmid-dna-purification/zyppy-plasmid-miniprep-kit" target="_blank">Zymo Research</A>, according to their provided protocol.
</P></DIV><DIV class="item"><H4>Agarose gels</H4><P>Amplified or digested DNA samples were supplemented with the appropriate amount of 10x Orange G, 6x NEB Purple or 6x Thermo Scientific loading dye and loaded on a 1 - 2 % (w/v) agarose TAE gel.
TAE agarose gels were produced by mixing agarose with 1x TAE buffer. Afterwards the mixture was microwaved until the agarose was completely dissolved. Staining of the DNA was performed by the <A href="https://de.vwr.com/store/product/16927467/dna-rna-dye-peqgreen" target="_blank">PeqGreen DNA dye</A> at the recommended concentration. 
The gel was subjected to electrophoresis at 100 – 130 V for 15 - 30 minutes or until a sufficient separation of the bands was observable. Visualization of the separated DNA fragments in the gel was performed by an ultraviolet light imager. A <A href="https://www.neb.com/products/n3200-2-log-dna-ladder-01-100-kb" target="_blank">NEB 2-log DNA ladder</A> or a <A href="https://www.thermofisher.com/order/catalog/product/10787018" target="_blank">Thermo Scientific 1kb+ DNA ladder</A> were included as molecular size marker.
</P></DIV><DIV class="item"><H2>Screening methods</H2><H3>Test Digestion:</H3><P>
To screen for clones containing the right plasmid, the prepared plasmid was treated for the following protocol.</P><TABLE><TBODY><TR><TD>DNA</TD><TD>0.4 - 0.6 µg</TD></TR><TR><TD>appropriate enzyme or enzymes</TD><TD>5 units</TD></TR><TR><TD>buffer</TD><TD>1x</TD></TR><TR><TD>H<SUB>2</SUB>0</TD><TD>Filled up to 20 µl</TD></TR></TBODY></TABLE><P>
Afterwards, the mixture was analyzed by gel electrophoresis and clones that showed the right band sizes were further analyzed by sequencing. 
</P><H3>Colony PCR:</H3><P>
To perform colony PCR a colony was chosen and transferred to a new LB plate, containing the appropriate antibiotic, with a pipet tip.
Afterwards, the pipet tip was dipped into nuclease free water and the water was used to perform  PCR.</P><H3>Sequencing:</H3><P>Sequencing was performed for the final verification of plasmids. Therefore 20 µl of preperated plasmid DNA with a concentration of 30 – 100 ng and 20 µl of the appropriate primer with a concentration of 10 µM were sent to GATC-Biotech. Sequencing results were analyzed using Geneious version 10.0.9.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><DIV class="flex-container"><DIV class="col-sm-6"><DIV class="image_box left" style="float:right"><DIV class="figure" style="width:195px"><DIV class="figureinner"><DIV class="figurecaption-car" style="float:right"><P><STRONG>Safety</STRONG></P></DIV></DIV></DIV></DIV></DIV><DIV class="col-sm-6"><DIV class="image_box right" style="float:left"><DIV class="figure" style="width:180px"><DIV class="figureinner"><DIV class="figurecaption-car" style="float:left"><P><STRONG>Cell Culture</STRONG></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>