<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF-Lausanne_Brainstorming"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:EPF-Lausanne/Brainstorming">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF-Lausanne/Brainstorming&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF-Lausanne/Brainstorming&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF-Lausanne/Brainstorming&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF-Lausanne/Brainstorming" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF-Lausanne/Brainstorming</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV style="position: relative; left: 190px; top: -134px; padding: 0px; margin: 0px; border: 0px solid white;z-index:1;"><DIV class="wrapper1"><DIV class="wrapper" style="width:650px;"><DIV class="nav-wrapper"><DIV class="nav"><UL id="navigation" style="width:600px"><LI class="#"><A href="https://2009.igem.org/Team:EPF-Lausanne" target="_self"><SPAN class="menu-mid">Home</SPAN></A></LI><LI class="#"><A href="https://2009.igem.org/Team:EPF-Lausanne/Team" target="_self"><SPAN class="menu-mid">Team</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Team" target="_self">Presentation</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Team#Where_we_are_from" target="_self">School</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Team_Management" target="_self">Team Management</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Brainstorming" target="_self">Brainstorming</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Sponsors" target="_self">Sponsors</A></LI></UL></DIV><LI class="#"><A href="https://2009.igem.org/Team:EPF-Lausanne/Project_Abstract" target="_self"><SPAN class="menu-mid">Project</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Project_Abstract" target="_self">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/LOVTAP" target="_self">Theory</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Strategy" target="_self">Strategy</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Experimental_Setup" target="_self">Experimental Setup</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/LOVTAP_Results" target="_self">Results</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Parts" target="_self">Parts submitted to the Registry</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Future_directions" target="_self">Future directions</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/References" target="_self">References</A></LI></DIV><LI class="#"><A href="https://2009.igem.org/Team:EPF-Lausanne/Modeling_overview" target="_self"><SPAN class="menu-mid">Modeling</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Modeling_overview" target="_self">Modeling overview</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Analysis methods" target="_self">Analysis Methods</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Results" target="_self">Results</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Information_&amp;_references" target="_self">References</A></LI></DIV><LI class="#"><A href="https://2009.igem.org/Team:EPF-Lausanne/Social_overview" target="_self"><SPAN class="menu-mid">Human practices</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Social_overview" target="_self">Overview</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Talks" target="_self">Talks</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Survey" target="_self">Survey</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Media" target="_self">Media</A></LI></DIV><LI class="#"><A href="https://2009.igem.org/Team:EPF-Lausanne/Safety" target="_self"><SPAN class="menu-mid">Safety</SPAN></A></LI><LI class="#"><A href="https://2009.igem.org/Team:EPF-Lausanne/Notebook" target="_self"><SPAN class="menu-mid">Lab</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Last_News" target="_self">Last News</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Notebook" target="_self">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:EPF-Lausanne/Protocols" target="_self">Protocols used</A></LI></DIV></DIV></DIV></DIV><DIV class="epfltrick"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Ideas"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Ideas</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#The_BphP_.2F_PpsR_system"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">The BphP / PpsR system</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Implementation"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Implementation</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Strains"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Strains</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#The_pathway"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">The pathway</SPAN></A></LI></UL></TD></TR></TBODY></TABLE></DIV><DIV class="epfl09team"><P><CENTER><FONT size="12" color="#007CBC">Brainstorming</FONT></CENTER></P><H2><SPAN class="mw-headline" id="Ideas">Ideas</SPAN></H2><P><I>Post all of your ideas here! Just a temporary place to share ideas before we get a forum or something else... and maybe also your name, so we know who thought of what if we want to discuss something.</I></P><P><B>1)</B> Bacteria that could be &quot;integrated&quot; (symbiose?) into plants/vegetables/... to enrich them in nutrients? (Gabriela)
</P><P><B>2)</B> Bacteria as a diagnostics tool for still uncurable diseases: sensor for a molecule/hormone/... specific of a disease, and would then release a substance which could create a visible phenotype: could allow early detection of diseases (ideally)  (Gabriela)
</P><P><B>3)</B> Bacteria for the food-processing industry: they could produce some kind of flavouring substance/vitamin/nutrient which is normally added artificially to a mixture (e.g. a juice...), and could be tuned to produce the substance until the optimal dosage is reached? --&gt; more an industry application  (Gabriela)
</P><P><B>4)</B> Sound-sensing bacteria: don't know if it's possible or if a native receptor could be &quot;tweaked&quot; for this, but could be useful if we could use sound waves as a triggering signal for e.g. substance release --&gt; going to try to do some research  (Gabriela)
</P><P><B>5)</B> Bacteria that could bind to a surface making a biofilm and fonctionalize the surface. To make the surface hydrophilic/hydrophobic or change the viscosity of this surface (e.g. with the movement of their flagella..) (Christian)
</P><P><B>6)</B> Use bacteria, like magnetotactic bacteria which produce nanoparticle (Fe3O4 or Fe3S4), to produce another nanoparticle. Or use this bacteria to purify liquid of different toxic metal. Here you have a link to : <A href="http://www.iop.org/EJ/article/0953-8984/18/38/S19/cm6_38_S19.pdf?request-id=28da827d-3a09-4eab-94d8-efadacdf2e76">Biogenic nanoparticles: production, characterization,
and application of bacterial magnetosomes </A> (Christian)
</P><P><B>7)</B> Found a pretty cool thing (though it's from 2002 so a bit old): at MIT they covalently linked a little gold &quot;antenna&quot; to enzymes or DNA, and when they stimulated the bacteria with an electromagnetic field, this disrupted the structure of the protein/DNA the antenna was attached to (because it harvested the energy of the field) so that for example the enzyme wouldn't function anymore (the effect is reversible). Couldn't find the whole article for free but here's a short summary: <A href="http://techon.nikkeibp.co.jp/NEA/archive/200203/172419/">MIT Researchers Merge Electronics with Biology </A> (Gabriela)
</P><P>also: apparently there's some evidence that bacteria do somehow emit electromagnetic/sound waves to stimulate growth? but I haven't found much on the subject yet
</P><P><B>8)</B>Create a biosensor for CO (carbon monoxyde), that would turn red or green if the threshold of toxic CO is reached. Use of dried E. coli cells and a GFP cassette : an inducible promoter sensitive to CO and then the GFP gene so that the E. coli cells would turn green if there is too much CO. This could be then detect and associated with some sort of an alarm. (Caroline)
</P><P><B>9)</B>Engineer cells so they could synthesize ozone (no idea if it is possible), and make them resistant to cold and to create a support for them. An application would be to send these cells in the ozone layer to recreate it. (Caroline)
</P><P><B>10)</B> An application in personal medicine : some treatements are efficient in patient with a certain allele while it has no effect (or an adverse response) in the patients with the other allele. An alternative to sequencing every patient would be to engineer some cells so they bind to the receptor of the drug, and would fluoresce only if not bound. Use of an inducible promoter to make the control (all cells fluoresce at the beginning and when we add the ligand no more fluorescence if it is bound).  (Caroline)
</P><P><B>11)</B> Make heavy metals precipitate (by using a chemical reaction involving the metal ions) with the help of bacteria having such a reaction in their metabolism and refining it (Basile)
</P><P><B>12)</B> Use neuron-similar bacteria (or something like that) to build a logic system with 2 input signals (for example 2 neurotransmitters or other molecules) which would trigger an action potential as the output signal, the advantage compared to presently used &quot;logic systems&quot; being that the output signal would be generated a lot faster than if you have a bistable system based on the equilibrium of chemicals in the cell (Nicolas)
</P><P><B>13)</B> Use bacteria or a micro-organism well-tolerated by humans to use them for applications in skin regeneration and localized drug delivery (and production) for extensive wounds: for example such bacteria could be deposited on the wound and they could produce some sort of protective biofilm or continuously deliver a molecule essential for better cicatrisation... (Basile, Nicolas, Rafael, Gabriela)
</P><P><B>14)</B> A solution to reduce CO2 emissions: use cyanobacteria and modify their tolerance levels for CO2 uptake so that they'd absorb higher quantities and metabolize them: even if the bacteria subsequently die, the CO2 at least wouldn't be released as a gas (it would be transformed into biomass) (Basile)
</P><P><B>15)</B> Help people with an intolerance to gluten (or allergy...) digest it quickly enough for it not to affect their health by making them eat for example a probiotic yoghurt containing bacteria which would provide the enzymes necessary to metabolize the compound. An alternative would be to simply make the bacteria &quot;absorb&quot; the gluten: these would then naturally be eliminated from the organism. (Heidi)
</P><P><B>16)</B> Bacteria that could &quot;fabricate&quot; LDLs with the surface receptors we'd want, and into which we could encapsulate the desired drug... (Nathalie)
</P><P><B>17)</B> Bacteria that could respond differently to different light stimuli, depending on their wavelength (so colour of the light used). This could then have applications in various fields as a tool. (Tù)
</P><H2><SPAN class="mw-headline" id="The_BphP_.2F_PpsR_system"> The BphP / PpsR system </SPAN></H2><P><B>Finally this part of the project was cancelled, due to lack of time.</B></P><P>But it is still of interest to describe it, it could be useful in the future.
PpsR1 is a redox sensitive activator. It binds DNA under anaerobic conditions, and forms a tetramer via a disulfide bond. This interaction is ablated in the mutant PpsR1-C429S; meaning that we should be able to mimic an anaerobic system even under aerobic conditions.
</P><P>PpsR2 is a transcriptional repressor, regulated by BphP.
</P><P>PpsR1 and PpsR2 bind to TGTN_12ACA possibly arranged in tandem with a 7 base spacer. The affinity for both PpsRs are around 100nM.
</P><P>BphP (or BrBphP for the Bradyrhizobium variant) is sensitive to far-red light (~770nm) and controls PpsR1.
</P><P>For more info, see <A href="http://www.jbc.org/cgi/content/full/279/43/44407" class="external autonumber" rel="nofollow">[1]</A>.
</P><P>Similar constituents could possibly be derived from R. Palustris as well with the following model derived from <A href="http://www3.interscience.wiley.com/cgi-bin/fulltext/118512333/HTMLSTART" class="external autonumber" rel="nofollow">[2]</A>:
</P><H2><SPAN class="mw-headline" id="Implementation">Implementation</SPAN></H2><P>There are two modes in which we can use the system. The minimal complement requires PpsR2 and its regulator BphP. We can generate a hybrid system that uses a well known activator or transcription factor, or a constitutively active promoter which will be repressed via PpsR2 upon exposure to far-red light (770nm).
</P><P>It is also possible to reconstitute the entire system including PpsR1-C429S, which will serve as the activator and will be de-coupled from the oxidative state of the cell due to the ablation of the disulfide bond formation.
</P><H2><SPAN class="mw-headline" id="Strains">Strains</SPAN></H2><UL><LI> We contacted Eric Giraud and ask if he could send us the following material:
</LI><LI> Bradyrhizobium ORS278
</LI><LI> The following plasmids:
</LI><LI> pBAD::ppsR1
</LI><LI> pBAD::ppsR2
</LI><LI> pGEM-T::ppsR2/BrbphP
</LI></UL><LI> the PpsR1-C429S construct
</LI><LI> We received the strains in late July, thanks the Eric Giraud's team !
</LI><H2><SPAN class="mw-headline" id="The_pathway">The pathway</SPAN></H2><P>- Sequences of the genes involved in the pathway (minimal genes to  =&gt;<A href="http://www.ncbi.nlm.nih.gov/" class="external text" rel="nofollow">Pubmed</A>)
</P><P><A href="/wiki/images/3/3a/ORS278_BrBphP_genomic_seq.txt" class="internal" title="ORS278 BrBphP genomic seq.txt">ORS278 BrBphP genomic sequence</A></P><P><A href="/wiki/images/6/60/ORS278_PpsR1_genomic_seq.txt" class="internal" title="ORS278 PpsR1 genomic seq.txt">ORS278 PpsR1 genomic sequence</A></P><P><A href="/wiki/images/9/9a/ORS278_PpsR2_genomic_seq.txt" class="internal" title="ORS278 PpsR2 genomic seq.txt">ORS278 PpsR2 genomic sequence</A></P><P><A href="/wiki/images/f/f0/ORS278_PpsR1-C429S_protein_seq.txt" class="internal" title="ORS278 PpsR1-C429S protein seq.txt">ORS278 PpsR2-C429S protein sequence</A></P><P><A href="/wiki/images/6/6b/HmuO_ORS278_Brad.txt" class="internal" title="HmuO ORS278 Brad.txt">ORS278 hmuO genomic sequence</A></P><P><A href="/wiki/images/f/f4/PpsR1_prot_RPalustris.txt" class="internal" title="PpsR1 prot RPalustris.txt"> R.Palustris CGA009 ppsR1 protein sequence</A></P><P><A href="/wiki/images/f/fc/PpsR1_RPalustris.txt" class="internal" title="PpsR1 RPalustris.txt"> R.Palustris CGA009 ppsR1 genomic sequence</A></P><P><A href="/wiki/images/d/d0/PpsR2_prot_RPalustris.txt" class="internal" title="PpsR2 prot RPalustris.txt"> R.Palustris CGA009 ppsR2 protein sequence</A></P><P><A href="/wiki/images/2/22/PpsR2_RPalustris.txt" class="internal" title="PpsR2 RPalustris.txt"> R.Palustris CGA009 ppsR2 genomic sequence</A></P><P><A href="/wiki/images/1/11/CAE001_bphP1.txt" class="internal" title="CAE001 bphP1.txt"> R.Palustris CAE001 bphP1 genomic sequence</A></P><P><A href="/wiki/images/c/cd/CGA009_hmuO.txt" class="internal" title="CGA009 hmuO.txt"> R.Palustris CGA009 hmuO genomic sequence</A></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:394px;"><DIV class="thumbcaption">Organization of the genes downstream the puf operon in Bradyrhizobium and Rps. palustris. BphP: bacteriophytochrome gene; hmuO: heme oxygenase gene.</DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:EPF-Lausanne/Brainstorming">http://2009.igem.org/Team:EPF-Lausanne/Brainstorming</A>&quot;</DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF-Lausanne/Brainstorming" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF-Lausanne/Brainstorming" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF-Lausanne/Brainstorming&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF-Lausanne/Brainstorming&amp;oldid=147399" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>