"
Page
Discussion
View source
History
teams
Log in
 
Team:EPF-Lausanne/LOVTAP
From 2009.igem.org
Home
Team
Presentation
School
Team Management
Brainstorming
Sponsors
Project
Abstract
Theory
Strategy
Experimental Setup
Results
Parts submitted to the Registry
Future directions
References
Modeling
Modeling overview
Analysis Methods
Results
References
Human practices
Overview
Talks
Survey
Media
Safety
Lab
Last News
Notebook
Protocols used
Contents
1 Introduction
2 Project
               
               
               
LovTAP system
Introduction
For organisms to thrive in a changing light environment, they sense and respond to light: the sensory information allows them to move in response to that stimulus. These responses are mediated by phototropins, which are photosensory proteins consisting of a serine-threonine kinase domain and a pair of non-identical Light, Oxygen, or Voltage (LOV) sensitive domains which contain the non-covalently bound chromophore ﬂavin mononucleotide (FMN).
In the article of Strickland et al. , an allosteric switch was created by joining two domains, as explained below. The resulting protein has a domain-domain overlap with a shared helix, this shared helix acting as a rigid lever arm. The disruption of the helical contacts causes a shift in the conformation. Thus, photoexcitation would change the conformation of the protein, in turn changing the stability of the helix-domain contacts. This change impacts the affinity of the shared helix for the two domains, and a signal can be then propagated.
The system is called LovTAP. The light-sensitive input module is the LOV2 domain of Avena sativa phototropin 1 (AsLOV2). Absorption of a photon by AsLOV2 triggers the formation of a covalent adduct between FMN (the flavin mononucleotide) cofactor and a conserved cysteine residue. This formation leads to the displacement and unfolding of a helix in the LOV domain, which is likely to mediate a signal propagation. LOV domains absorb light through a ﬂavin cofactor, photochemically form a covalent bond between the chromophore and a cysteine residue in the protein, and proceed to mediate activation of an attached kinase domain.
The output module was the bacterial transcription factor trp repressor (TrpR). TrpR can bind its operator DNA as a homodimer.
By ligating AsLOV2 to TrpR, Strickland et al. were able to construct an allosteric switch called LovTAP : LOV- and tryptophan-activated protein. This protein protects DNA from digestion when illuminated.
Phototropin switching is regulated at the stage of LOV2-Jalpha interactions, which are mostly intact in the dark. Illumination leads to the formation of a protein-flavin adduct which distorts the LOV2 structure sufficiently to substantially weaken the LOV2-Jalpha interactions, thus freeing the Jalpha-helix and allowing it to unfold ( Yao et al. ).
LovTAP can thus be found in two states : dark-state and light-state. The ground state is referred to as the dark state, and its photoactivated state as the light state. The TrpR domain associates with the shared helix (shared from AsLOV2 and TrpR).
Protein structure of the LOV2 domain and light-induced structural changes
during the photocycle. Taken from the article of Koyama and al. (17)
In the dark state (A in the image below), a few residues of the shared helix presumably dissociate from the TrpR domain and dock against the LOV domain. The steric overlap is then relieved, which decreases the TrpR domain's DNA-binding affinity (when the shared helix contacts the LOV domain, the TrpR domain is in an inactive conformation).
In the light state, the residues re-associate with the TrpR domains (because the photoexcitation disrupts contacts between the shared helix and the LOV domain), which restores DNA-binding affinity and the system is in the active form (B). LovTAP can then bind DNA. But this system is not stable, and the LOV domains returns to the dark state (C), which triggers the dissociation of LovTAP from the DNA (D).
Taken from the article of Strickland and al. (2)
Project
In our project, we want to implement such a system in E. coli, using synthetic biology and then characterize it in vivo ( Strickland et al. have only done the characterization in vitro).
This led to some challenges: Would the in vivo system react as well as in vitro? What would the interferences in such a system be? In their article Strickland et al. used a digestion assay to asses the binding of the LovTAP (the sequence of binding also contained a digestion enzyme recognition site. Thus, binding of the LovTAP would prevent DNA digestion). In our case, as the system was in vivo we wanted to use fluorescent protein expression to quantify the effectiveness of the LovTAP. But, as the binding site for LovTAP is the same as for TrpR, we could expect interference and cross-talk between the natural Trp operon system and our engineered system.
First, we wanted to clone the LovTAP gene into an IPTG inducible BioBrick using a LacI promoter from the registry as well as a RBS and a terminator. Following this, we will clone a readout that would express RFP upon ligation of the LovTAP (after illumination) to its promoter sequence (the trp promoter). As we want to have RFP expression upon illumination (and thus binding of the LovTAP), we will need to create an inverter. We will also try to improve the LovTAP binding on DNA : a major problem is that the conformational change of LOVTAP is weak and the protection assay results show a small difference of LOVTAP binding on DNA between dark state and light state. We will use modeling to determine which amino acid residue we could mutate in order to improve the stability of this binding state.
Retrieved from " http://2009.igem.org/Team:EPF-Lausanne/LOVTAP "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
