Team:Lambert GA/ProjectOverview
JUDGING FORM
MODELING
CALM ELECTROPEN
CALM
BACKGROUND DATA & METHODS RESULTS DISCUSSION COLOR Q SUPPLEMENTARY REFERENCES
QSYSTEM
COLOR Q CHROME Q REFERENCES
ELECTROPEN
PRINCIPLES & DESIGN MODEL & THEORY PRODUCT DESIGN REFERENCES
HUMAN PRACTICES
INTEGRATED HUMAN PRACTICES EDUCATION & ENGAGEMENT COLLABORATIONS
SAFETY
PARTS
BASIC PARTS COMPOSITE PARTS
PROJECT
DESCRIPTION BACKGROUND DESIGN EXPERIMENTS LAB NOTEBOOK INTERLAB RESULTS DEMONSTRATE THE VISION
TEAM
TEAM MEMBERS ATTRIBUTIONS
Your browser does not support video.
P R O J E C T
PROJECT OVERVIEW
OUR PROJECT
CHOLERA SWITCH
WORK FLOW
REFERENCES
DESIGN REFERENCES
PROJECT OVERVIEW
The identification of pathogens in potentially contaminated sources is crucial for the prevention and treatment of affected individuals from across the world. People in developing nations are particularly distressed by infectious diseases as a consequence of poor sanitation and lack of personal hygiene and access to sufficient resources. Approximately 844 million of these people suffer from a lack of access to safe water, of which 3.4 million succumb to the wide range of infectious diseases transmitted through contaminated water sources [1][2]. Numerous organizations have strived to increase access to clean water in impoverished communities within developing nations. However, pathogens are still able to thrive in water sources, allowing for the emergence of endemics and epidemics that devastate large segments of the population.
One disease in particular, Cholera, is notorious for claiming approximately a million lives annually. While its presence is relatively non-existent in developed nations due to sufficient treatments that are easily accessible, this pathogen devastates communities in developing nations. Current strategies to detect Cholera are severely inadequate and inefficient. Consequently, Cholera cases are prevalent in these communities. Therefore, Lambert iGEM hopes to develop a practical, yet efficient solution to ensure that these epidemics can not only be detected but prevented as well.
Current Methods
With the increasing occurrence of epidemics of Cholera, numerous tools have been developed to detect this pathogen in fecal and water samples. However, they significantly vary in cost and precision, making them incompetent for deployment in the field. The most prevalent detection mechanism for Cholera is the Crystal VC Dipstick, an inexpensive tool that can be easily transported and utilized. However, the accuracy range for this device varies between 60% and 99%, which requires additional lab testing for confirmation and act as a deterrent for guaranteed positive/negative results[4]. Immunoassays are also incorporated for confirmation of the Cholera pathogen. However, the precision of these devices comes at a high cost, as the materials necessary to conduct these tests are expensive and difficult to deploy and transport in a field setting, making it inefficient for testing outside the confines of a laboratory[11]. Another common method involves Polymerase Chain Reaction (PCR), which allows for amplification of the target genes of the Cholera pathogen for confirmation of the O1 and O139 strains. However, incorporation of the PCR method requires the utilization of numerous reagents in the field, in addition to a thermocycler and gel electrophoresis apparatus, making it difficult to exploit in the field[11]. The inefficiency of these methods in terms of costs and/or accuracy has driven the 2018 Lambert iGEM team to develop a novel system capable of delivering accurate and precise results for Cholera identification at a significantly lower cost.
Our Project
This overview explains how the multiple parts of our project work together to be a proactive approach to preventing V. cholerae epidemics using Yemen as a test case.  Our process begins with the CALM software predicting outbreaks up to 8 weeks in advance.  Our kit with the necessary hardware, software and biosensor cells are pre-deployed to aid workers.  Text messages are sent to the aid agencies who notify local workers to deploy the testing kits.  Water samples are taken and filtered to extract cells in the size range of V. Cholerae.  The cells are lysed and RNA or DNA is extracted (depending whether NASBA is available in field)  The RNA/DNA is electroporated into the biosensor cells using our Electropen.  Biosensor cells are incubated for 24 hours after which the cell solution is pelleted using the 3-D fuge.  The sample is loaded onto the Chrome Q base and using the Color Q App, the results are quantified and uploaded to AWS server which publishes the results.  The results from the water sampling feeds into our CALM model completing a feedback loop to ensure continual model improvement.
Recognizing the limitations of the current methods utilized for the detection of Cholera, Lambert iGEM has developed a novel platform to detect and analyze data from this pathogen at a fraction of the cost.  While many caveats persist towards pathogen identification in large water sources, access to efficient and inexpensive technology has become the primary prohibitive factor. Inexpensive tests currently utilized for detection have a significant variance in the positive/negative outputs, whereas the precision of other technologies correlates with heightened costs with the lack of access to necessary materials in the field. Lambert iGEM demonstrates the potential for a gene-based detection mechanism for Cholera pathogens that can easily be deployed in the field at drastically lower costs, without sacrificing quality or performance. In tandem, we propose a machine learning model that can predict outbreak inception and spread, allowing for a preventative approach, rather than a reactive one.
Future Implications
While our current technology functions for identification of Cholera, Lambert iGEM hopes to expand this technology to numerous pathogens, establishing a collection of genetic tools for detection and compiling them into a portable synthetic biology toolkit that can be distributed to aid organizations for confirmation of clean water provisions. This detection platform with visible readouts can be integrated into a data collection platform on a global scale, allowing for a proactive response to disease outbreaks and ensuring the safety of the people residing in potentially at-risk areas. Lambert iGEM hopes to revolutionize pathogen detection in order to enhance feasibility, accessibility, affordability, and efficiency.
Cholera Background
Cholera is an acute diarrheal illness caused by an infection of the intestines with the toxigenic bacterium Vibrio cholerae serogroup O1 or O139. V. cholerae O139, first identified in Bangladesh in 1992, has caused numerous outbreaks in the past but recently has only been identified in sporadic cases across Asia. The main form of transmission of the Cholera bacterium is the contamination of water and food by feces from an infected individual. Cholera is prevalent in locations with inadequate water treatment and sanitation infrastructure. The main form of diagnosis for Cholera is a stool sample or rectal swab, which must be sent to a laboratory in order to identify the Cholera bacterium. The key symptom of Cholera is severe diarrhea, which leads to dehydration, pain in the abdominal regions, and lethargy. Approximately one in ten (10%) infected persons will have dire cases of Cholera characterized by watery diarrhea, profuse vomiting, and leg cramps. In these cases, body fluid loss and water-electrolyte imbalance lead to dehydration and shock. Very often, lack of access to treatment can lead to death in a matter of hours. The CDC reports that there are an estimated 2.9 million Cholera cases worldwide.
Unfortunately, even with the existence of Oral Cholera Vaccines (OCVs), an effective tool to combat cholera in developing nations with an 80.2% effectiveness rate, 100,000 Cholera deaths still occur yearly. Other treatments include rehydration therapy, antibiotics, and IV fluids. In the years 2000-2016, the World Health Organization discovered numerous major Cholera epidemics, including Haiti in the Americas, DRC, Somalia and the United Republic of Tanzania in Africa, and Yemen in Asia. These same locations are reported to have poor water infrastructure and lack of access to Cholera treatment centers. As of April 27, 2017, there have been 1,055,788 suspected cases, 612,703 confirmed cases, and 2, 255 deaths from Cholera-related problems. Ultimately, resource allocation has been difficult because Cholera is rapid and sporadic, hindering aid organizations from providing timely solutions.
Timeline
Picture of the distribution of Cholera outbreaks through time and within various geographies as well as the new advancements to effectively reduce Cholera outbreaks worldwide.
How a Toehold Switch Works
Conventional riboregulators serve many synthetic biology purposes including gene regulation, genetic circuits, and biosensors. However, their low dynamic range and susceptibility to crosstalk inhibit their ability to be sensitive towards RNA recognition. In addition, the significant amount of crosstalk that occurs in natural riboswitches diminishes specificity, which is an important aspect to consider in global applications including diagnostics and biotechnology. To address these limitations, toehold switches have revolutionized engineered and systematic riboregulators through their ability to regulate many cell components at the same time. They exhibit far more favorable reaction kinetics and thermodynamics in addition to their dynamic range and versatility [5][1]. As these synthetic riboregulators utilize linear-linear interactions between RNA, they present a reliable and diverse application in synthetic biology offering a potential role in diagnostics. Thus, the 2018 Lambert iGEM team envisions the use of a toehold switch to create a Cholera detector as one of the propelling motivations for this year’s project.
Toehold switches are riboregulators that activate translation in response to a distinct RNA sequence. It is comprised of a switch and a trigger. The switch is composed of a hairpin loop structure that represses translation through its complementary bases in between the ribosome binding site and start codon, which is followed by a 21 nucleotide linker sequence. These sequences ensure that the toehold switch structure will be maintained while coding for low-molecular weight amino acids that would not interfere with the switch’s function. The toehold domains at the beginning of the hairpin are 12 to 18 nucleotides long and are designed to be complementary to the trigger in order to initiate linear RNA binding [5]. The trigger contains complementary sequences to the toehold domain that once it is in the presence of the switch, it will bind to the hairpin stem and unbind the loop. This exposes the ribosome binding site and start codon, allowing translation of the reporter protein to occur.
The 2018 Lambert iGEM team utilized Toehold Switches in two aspects: characterizing a LacZ toehold switch and developing a Cholera switch.
Cholera Switch
To address the lack of a proactive, feasible detection method for Cholera, Lambert iGEM developed a Cholera toehold switch. One of the largest sources of contracting Cholera is through the consumption of contaminated water. To develop a detection method for these contaminated water sources, Lambert iGEM has located the gene ctxB in the toxigenic bacterium Vibrio cholerae serogroup O1 [3]. This gene encodes for the Cholera enterotoxin subunit B which does not contain toxicity by itself but is specific to the pathogenic strain of Cholera [4]. The use of this gene allows for our designed switch to be an accurate detection mechanism while also addressing any safety issues regarding any contact with toxins.
Using a toehold designer software from the Chinese University of Hong Kong iGEM team, we inputted the RNA sequences of ctxB as the target and other parameters including trigger length, toehold domain length, maximum toehold domain base pairs, working temperature, promoter sequence, and linker sequence. The software then generated possible sequence combinations for each of the switch and triggers while also providing additional data that further characterized each component’s efficacy.
Here is all of our Toehold Construct designs.
We also utilized Nupack: an open source software that analyzes nucleic acid structure in response to a given sequence. By inputting the minimum free energy structures for each of the switch and trigger combinations, Nupack showed the possible formed structure allowing us to determine whether or not each switch and trigger would have a high probability of working. The minimum free energy shown below demonstrates the strength of repression for the switch RNA and the single-strandedness of the trigger RNA for the activated complex [2]. According to Green et al. 2014, a negative ∆GRBS-linker values is correlated to lower switch dynamic range.
Through these two softwares, Lambert iGEM was able to design and test three Cholera switches. These switches all consist of a T7 promoter (Part I719005) along with LacZ (Part I732005) as the reporter gene. We performed a series of experiments to clone our toehold switches.
Proof of Concept
In order to utilize LacZ color expression as a biosensor mechanism, Lambert iGEM obtained a LacZ toehold construct assembled with a T7 promoter from the Styczynski Lab at the Georgia Institute of Technology. When assembled with a distinct RNA sequence that is complementary to the trigger sequence, the LacZ operon is induced and subsequently breaks down Xgal and produces galactose and a blue pigment. This color can then be characterized by the shade of the blue pigment expression. This part is transformed in pSB3C5 instead of pSB1C3 because when LacZ is induced in a high copy plasmid such as pSB1C3, it drains the metabolism of a cell.
The construct above displays the proof of concept T7 LacZ switch and trigger. The switch consists of T7 promoter, a strong constitutive promoter, along with LacZ as the reporter gene. The trigger was cloned into a high copy plasmid while the switch was cloned into a low copy plasmid to ensure the replication of two different plasmids for the E. coli cell to produce proportional amounts.
The T7 Toehold LacZ biobrick on a chloramphenicol and Xgal plate expressing blue pigment without trigger sequence as a result of the strong T7 promoter (Image on the left). T7 Toehold LacZ and trigger sequence in a dual plasmid transformation on a carbenicillin, chloramphenicol, and Xgal plate that is expressing a blue pigment due to presence of trigger sequence (Image on the right).
As seen in the figure above, it was observed that the toehold expressed a blue pigment when inoculated into Xgal and Luria Broth (image on the left). Although a lighter shade than when fully induced (image on the right), we hypothesize that this apparent pigment is due to toehold leakiness as a result of the strength of the T7 promoter. The Toehold sequence used in this construct was obtained from the 144 first generation orthogonal toehold switches collection from the 2017 Collins paper titled “ Toehold Switches: De-Novo-Designed Regulators of Gene Expression”.  Following this unique toehold sequence is the LacZ operon. We introduced a base pair wobble in the LacZ gene that substituted an Adenine for a Guanine. The wobble mutation sequence was obtained from the Styczynski Lab at the Georgia Institute of Technology and was used to eliminate the illegal EcoRI site in the LacZ operon.
Wet Lab Summary
Lambert iGEM utilized a toehold switch and a trigger - which corresponds with RNA sequences of the ctxB1 cholera gene - in a biosensor system. If the trigger sequence is present and binds to the toehold switch, then a blue pigment produced by LacZ appears in the cells.
The construct above displays the proof of concept T7 LacZ switch and trigger. The switch consists of T7 promoter, a strong constitutive promoter, along with LacZ as the reporter gene. The trigger was cloned into a high copy plasmid while the switch was cloned into a low copy plasmid to ensure the replication of two different plasmids for the E. coli cell to produce proportional amounts.
Work Flow
The 2018 Lambert iGEM team conducted the following experiments to assemble the proof the concept T7 LacZ toehold switch and Cholera switch.
We ordered three potential Cholera toehold switches and triggers and proceeded to perform PCRs on each of the parts. All switches and triggers were amplified and validated through gel electrophoresis [Figure 1.]. With purified PCR products of the switches, triggers, and their correlating vectors, we ligated each of these parts into their vectors through Gibson Assembly. Dual plasmid transformations and single transformations were conducted with BL21 T7 E. coli
Click on the step to be directed to the procedure!
Miniprep
Nanodrop
Digest
Gel
Ligation
Transformation/electroporation, Plate
Colony PCR (Screening)
Gel
Inoculate correct colony to a liquid culture
Miniprep
Nanodrop
Sequence
Materials
Miniprep: grown culture, microcentrifuge, 2 1.5mL microcentrifuge tubes, mini column and collection tube, Solution I, Solution II, Solution III, HBC Wash Buffer, DNA Wash Buffer, Elution Buffer, micropipette and tips
Nanodrop: nanodrop machine, miniprepped DNA, Kimtech wipes, micropipette and tips
Digest: miniprepped DNA, dH₂O, 10X RE-Mix, standard restriction enzyme, micropipettes and tips
Gel: agarose gel (make one if necessary), 1X TAE Buffer, power supply, chamber and electrodes, ladder, micropipette and tips, DNA
Ligation: vector, parts 1 and 2, ligase buffer, ligase, Antarctic phosphatase, microcentrifuge tube, ice, micropipette and tips
Transformation: ice, ligation mixture, competent cells, incubator, LB media, microcentrifuge tubes, micropipette and tips Plate: agar plate, micropipette and tips, beads
Colony PCR: dH₂O, buffer, VF₂, VR, Q5 polymerase, dNTP, DNA dilution, micropipette and tips, PCR tubes, thermocycler, ice
Gel: agarose gel (make one if necessary), 1X TAE Buffer, power supply, chamber and electrodes, ladder, micropipette and tips, DNA
Inoculate: LB media, dilution, micropipette and tips
Miniprep (using Omega Protocol)
1.1 Grow 1-5mL culture overnight in a 10mL-20mL culture tube.
1.2 Centrifuge at 2500xg for 5 minutes at room temperature. Decant or aspirate and discard the culture media. (Original protocol called for 10,000xg for 1 minute, but the speed and time above seemed to produce better results.)
1.2.1 Original protocol called for 10,000xg for 1 minute, but the speed and time above seemed to produce better results.
1.3 Add 250uL of Solution I mixed with RNase A (pre-added). Vortex to mix thoroughly. Transfer the suspension into a new 1.5mL microcentrifuge tube.
1.4 Add 250uL of Solution II. Invert several times until you get a clear lysate.
1.4.1 Once Solution II is added, do not let it sit for more than 5 minutes!
1.5 Add 350uL of Solution III. Invert several times until white precipitate forms. Centrifuge at 13,000xg or 17,900rcf for 10 minutes. A compact white pellet should form at the bottom of the tube.
1.6 Insert a mini-column into a 2mL collection tube.
1.7 Transfer the clear supernatant into the mini-column using a micropipette. Centrifuge at the maximum speed (13,000xg) for 60 seconds. Discard the filtrate and reuse the collection tube.
1.7.1 Be careful not to get any parts of the pellet! Tilt at an angle with the pellet at the top when micropipetting is advisable.
1.7.2 Think about what you are discarding versus what you want to keep!
1.8 Add 500uL of the HBC Wash Buffer diluted in isopropanol. Centrifuge at maximum speed (13,000xg) for 60 seconds. Discard the filtrate and reuse the collection tube.
1.8.1 All wash buffers will be centrifuged for 1 minute.
1.9 Add 700uL of the DNA Wash Buffer diluted in ethanol. Centrifuge at maximum speed (13,000xg) for 60 seconds. Discard the filtrate and reuse the collection tube.
1.10 Centrifuge the empty mini column at the maximum speed (13,000xg) for 2 minutes to remove the ethanol.
1.11 Transfer the mini-column to a nuclease-free 1.5mL microcentrifuge tube.
1.12 Add 50uL of Elution Buffer (or sterile deionized water). Let it sit in room temperature for 60 seconds. Centrifuge at maximum speed (13,000xg) for 60 seconds.
1.13 Store eluted DNA at -20℃.
Nanodrop
2.1 Vortex before nanodrop.
2.2 Wipe down the nanodrop machine with Kimtech wipes to make it sterile.
2.3 Set the program to analyze nucleic acids [because you are dealing with plasmid DNA].
2.4 Do a blank test to ensure that the platform is sterile.
2.5 Load 1uL of the miniprepped DNA onto the platform.
2.5.1 (Have steady hands. The sample needs to be in the center for best results.)
2.6 Click “measure” on the nanodrop for analysis.
2.7 Write down measurements for the concentration of DNA (in ng/uL), A260, A280, 260/280 (should be around 1.8), and 260/230 (should be around 2.1).
Digest
3.1 Dilute up to 1ug DNA to 17uL with dH₂O.
3.1.1 Take concentration of DNA from nanodrop and convert from ng/uL to ug/uL. Next, set up a proportion to find out how many uL you need to get 1 ug of DNA.
3.1.2 20uL (total reaction) - 2uL RE-Mix - 1uL standard enzyme = uL dH₂O
3.2 Use a microcentrifuge tube to put the reaction in. Put in the contents in this order: water, DNA, enzymes.
3.2.1 Add 2uL of the 10X RE-Mix and 1uL of the standard enzyme.
3.2.1.1 E and X = 10X RE-Mix
3.2.1.2 S and P = standard enzymes
3.3 Incubate at 37℃ for 1 hour for standard enzymes, then at 80℃ for deactivation.
Gel
4.1 Set up the chamber and put in the gel. Make sure the wells of the gel is at the end of the chamber so that the DNA runs to red.
4.2 Pour the TAE buffer evenly to completely cover the gel.
4.3 Using a micropipette, put 3uL of DNA in each well and 6uL for the ladder [if using a thin gel]. Thicker gels will require more DNA to be put in each well.
4.4 Connect the electrodes by closing the box and connecting them to the power supply. Make sure the power supply is set for 120 volts and 60 minutes.
4.5 Turn on the power supply and make sure bubbles are rising on the sides of the chamber.
Ligation
5.1 Use Antarctic phosphatase on the backbone to increase the likelihood of part insertion and decrease backbone closure.Make calculations using a 3:1 molar ratio of insert to backbone. Refer to the two tables below.
5.2 Put in each component in a microcentrifuge tube while on ice. They should be pipetted into the tube in this order: water, DNA, ligase buffer, ligase.
5.3 The ligase buffer should be thawed and resuspended at room temperature.
5.3.1 Gently mix by pipetting up and down and microfuge briefly.
5.4 Incubate at room temperature for 1 hour at 37℃
Transformation
6.1 Thaw materials on ice for 5 minutes.
6.2 Put 10uL of ligation mixture into 100uL competent cells in a microcentrifuge tube.
6.3 Flick the tube to mix.
6.5 Add 200uL of LB media.
6.6 Incubate at 37℃ for one hour.
6.7 Plate 150uL of cells onto a plate. Make sure plate has the correct antibiotic (based on vector backbone)! Grow overnight.
Electroporation
(for 400 ml culture, adjust as appropriate for smaller volumes)
Preparing Electrocompetent cells:
6.1 Grow overnight 5 ml culture
6.2 Dilute 1:100 in fresh media
** preparation: chill big centrifuge to 4C, chill autoclaved water and 10% glycerol soln
6.3 Grow to OD of about 0.6 (isolating in exponential phase most important)
6.4 Pour cells into 50 ml conical tubes, on ice
6.5 Keep on ice for 10 min
6.6 Centrifuge cells (all spins done at 2500 xg, 6 min), discard supernatant
6.7 Add 13 ml of ice cold sterile water to each tube, resuspend by pipetting up and down
6.8 Combine into two total tubes.
6.9 Centrifuge, repeat ice cold water resuspension
6.10 Do same thing twice with 25 ml of ice cold 10% glycerol
6.11 Resuspend in 4 ml (concentrating 100x from initial culture) of ice cold 10% glycerol
6.12 Aliquot into microcentrifuge tubes
6.13 Store in -80 freezer
Electroporation:
6.14 Take 50 ul of electrocompetent cells, and 10-100 ng of PCR product (don’t add more than 2ul) for 6.15 knockouts, 0.1-10 ng of plasmid
6.16 Flick to mix
6.17 Transfer to chilled electroporation cuvette
6.18 Electroporate (machine in Anton’s lab downstairs)
6.18.1 (first setting), target time constant: greater than or equal to 5 ms
6.19 Add 1ml of prewarmed LB to cuvette
6.20 Transfer cell/LB mixture to microcentrifuge tube
6.21 Recover with shaking in incubator for 1.5 hr
Colony PCR
7.1 Pick colonies with a combination of phenotypes i.e. large/small, red/white. Dilute each colony in 40uL dH₂O, 1uL DNA from ligation if transformation is successful.
7.1.1 If necessary, do a quick spin to make sure all the liquid is at the bottom.
7.2 Make the following master mix on ice in this order: 63uL dH₂O, 20uL buffer, 5uL VF₂ primer, 5uL VR primer, 2uL dNTP, 1uL Q5 polymerase.
7.3 Aliquot the master mixes into PCR tubes, then add 1uL of the DNA dilution.
7.3.1 Make sure PCR tubes are labeled properly and carefully!
7.4 Transfer the PCR tubes to a PCR machine and begin thermocycling.
7.4.1 Initial Denaturation: 98℃ for 30 seconds
7.4.2 25-35 Cycles: 98℃ for 5-10 seconds, 50-72℃ for 10-30 seconds, 72℃ for 20-30 seconds/kb
7.4.3 Final Extension: 72℃ for 2 minutes
7.4.4 Hold 4-10℃
Gel
8.1 Set up the chamber and put in the gel. Make sure the wells of the gel is at the end of the chamber so that the DNA runs to red.
8.2 Pour the TAE buffer evenly to completely cover the gel.
8.3 Using a micropipette, put 3uL of DNA in each well and 6uL for the ladder [if using a thin gel]. Thicker gels will require more DNA to be put in each well.
8.4 Connect the electrodes by closing the box and connecting them to the power supply. Make sure the power supply is set for 120 volts and 60 minutes.
8.5 Turn on the power supply and make sure bubbles are rising on the sides of the chamber.
Inoculate Liquid Culture
9.1 Get the remaining 39uL of colony dilution.
9.2 Get LB media and make sure to use the appropriate antibiotic resistance.
9.3 Mix the colony dilution into the media.
9.4 Grow overnight.
Water Collection and Analysis
Water Collection
Prepping the Bottle
1.1 Obtain 4 nalgene bottles (22.75 cm in length, 5.5 cm in width).
1.2 Drill a 1.9 cm (preferably 2 cm) hole in the bottom of the bottle with a ¾ in drill bit.
1.3 Run the bottle through tap water to get any plastic bits out of the bottle. Spray the interior of the bottle with isopropyl alcohol. Rinse with soap and water to remove any alcohol.
1.4 Allow the bottles to dry by placing them on a drying rack.
Prepping Muslin Cloth
Obtain 90 cm by 90 cm muslin cloth.
Cut the muslin cloth into 6 cm. by 6 cm pieces.
Rinse with soap under flowing tap water.
Allow the muslin cloth to dry.
Fold in half.
Place the muslin cloth into an ice bottle.
Fill the bottle ¾th of the way with the cloth
Prepping Cheese Cloth
Obtain a 5.5 meter by 19.4 cm.
Cut the cheese cloth into 20.5 cm by 15 cm.
Rinse the cheesecloth in running tap water and soap.
Create 3-4 folds and pack the ice bottle.
Use a long rod to help push the cloth into the bottle and make the bottle compact.
Fill the bottle ¾th of the way with the cloth.
Prepping T-shirt Cloth
Obtain 1 t-shirt
Cut into 5 cm by 11 cm strips
Fold strips 2 times and pack ice bottle
Fill bottle ½ way with the cloth
Prepping Coffee Cloth
Obtain a pack of large coffee filter
Microwave filters to sterilize
Cut filters in ½
Fold pieces 1-2 times and pack ice bottle
Fill bottle ¾th of the way with the filters
Prepping Carb Cloth
Add 500 ml of dH20 and 17.5g of agar to a stock jar
Autoclave solution
Add 500µl of carb
Pour plates as usual
Creating Liquid Culture
Add 5ml of LB to a 15ml culture tube
Inoculate a few colonies of cells
Obtaining Water
Obtain 1000 ml of creek water from any nearby water body by placing a stock jar within the water and allowing the water to flow into the stock jar.
Filtration
Run distilled water through all the filters until all filters are wet.
Get five 600 ml beakers
Combine the 5 ml liquid culture of E.coli to the 1000ml creek water. Stir the stock jar until the liquid culture is thoroughly mixed with the creek water.
Measure 100ml of the creek water spiked with E. Coli.
Run the 100 ml of creek water through the muslin filter bottle. Make sure a labeled 600 ml beaker is placed under the bottle to catch filtered water.
Repeat steps 4 and 5 with the negative control (no filter), cheesecloth, t-shirt, coffee filters.
Run the creek water spiked with E. coli through the Micron Filter (Positive Control)
Pour 100 ml of the creek water spike with E. Coli to the top cup of the micron filter.
Attach the filtration vacuum to the side of the micron filter. Turn on the vacuum so the water is filtered through the .2 micron filter.
Allow the water to be filtered through the micron filter and stop the vacuum when all the water has filtered to the bottom container of the micron filter.
Serial Dilution and Plating
Obtain 6 centrifuge tubes.
Plate 150 microliters of each of the filtered solutions in the 6 600ml beakers into corresponding carb plates. Use glass beads to spread the cultures throughout the carb plates.
Fill a beaker with 10 mL of distilled water.
Pipette 15 microliters of the filtered water that ran through the muslin and dispense in a centrifuge tube. Label the centrifuge tube, muslin.
Pipette 185 microliters of distilled water and dispense in the same centrifuge tube.
Repeat steps 12 and 13 with the negative control (no filter), cheesecloth, t-shirt, coffee filters, and the micron (positive) filter.
Plate 150 microliters of the solutions from the microcentrifuge to the corresponding carb plates. Use glass beads to spread the cultures throughout the carb plates.
Place the plates in the incubator and incubate at 37 degrees Celsius for 24 hours.
After 24 hours observe the results and note use colony forming units.
Picture of serial dilution and growth on plate (Image on the left) and various cloths used for testing the number of colony forming units (Image on the right).
Data
-
Coffee Filter
T-shirt
Cheese Cloth
Muslin
+
37
89
Too many to count/multiple types
Multiple types of bacteria
Too many to count
0
5
1
Too many to count/multiple types
Multiple types of bacteria
Too many to count
0
Analysis
The negative control showed a limited growth of e.coli colonies. The positive control had no growth meaning the microfilter was effectively filtered out all growth. Having used a blue t-shirt, we found that the water ran through turned blue. For the cheesecloth, only 50ml of water came out and it appeared to be yellow. All tested variables had more growth than the negative control, showing that there was a source of contamination in the experiment. However, coffee filter and muslin appeared to have the least e.coli growth showing they could possibly be effective for filtration. This experiment did not conclusively show whether coffee filter or muslin was more effective so we will repeat the experiment with coffee filters and muslin.
References
[1] Drinking-water. (2018, February 7). Retrieved from http://www.who.int/news-room/fact- sheets/detail/drinking-water
[2] (n.d.). Retrieved from http://www.who.int/water_sanitation_health/takingcharge.html
[3] Berman, J. (2009, October 29). WHO: Waterborne Disease is World's Leading Killer. Retrieved from https://www.voanews.com/a/a-13-2005-03-17-voa34-67381152/274768.html
[4] Learn How to Use the Crystal VC Dipstick Test to Detect Vibrio Cholera in Our New Video | DOVE: Stop Cholera. (n.d.). Retrieved from https://www.stopcholera.org/blog/learn-how-use-crystal-vc-dipstick-test-detect-vibrio-cholera-our-new-video
[5] Cholera - Vibrio cholerae infection. (2018, May 14). Retrieved from https://www.cdc.gov/cholera/diagnosis.html
[6] The Burden of Soil-transmitted Helminths (STH). (2011, June 06). Retrieved from  https://www.cdc.gov/globalhealth/ntd/diseases/sth_burden.html
[7] Water. (2016, April 22). Retrieved from https://www.cdc.gov/parasites/water.html
[8] Collender, P. A., Kirby, A. E., Addiss, D. G., Freeman, M. C., & Remais, J. V. (2015, December). Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4679500/
[9] Action Against Worms. (2008, February). Retrieved from http://www.who.int/neglected_diseases/preventive_chemotherapy/pctnewsletter11.pdf
[10] Pilotte, N., Papaiakovou, M., Grant, J. R., Bierwert, L. A., Llewellyn, S., McCarthy, J. S., & Williams, S. A. (n.d.). Improved PCR-Based Detection of Soil Transmitted Helminth Infections Using a Next-Generation Sequencing Approach to Assay Design. Retrieved from http://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0004578
[11]- Detection of Cholera Toxin [PDF]. (n.d.). Atlanta: Centers for Disease Control and Prevention.  https://www.cdc.gov/cholera/pdf/laboratory-methods-for-the-diagnosis-of-vibrio-cholerae-chapter-7.pdf
Design References
[1]Ausländer, S., & Fussenegger, M. (2014). Toehold gene switches make big footprints. Nature,516(7531), 333-334. doi:10.1038/516333a
[2]Badelt, S., Flamm, C., & Hofacker, I. L. (2016). Computational Design of a Circular RNA with Prionlike Behavior. Artificial Life,22(2), 172-184. doi:10.1162/artl_a_00197
[3]Cholera - Vibrio cholerae infection. (2018, May 11). Retrieved from https://www.cdc.gov/cholera/general/index.html
[4]European Bioinformatics InstituteProtein Information ResourceSIB Swiss Institute of Bioinformatics. (2018, July 18). Cholera enterotoxin subunit B. Retrieved from https://www.uniprot.org/uniprot/P01556
[5]Green, A., Silver, P., Collins, J., & Yin, P. (2014). Toehold Switches: De-Novo-Designed Regulators of Gene Expression. Cell,159(4), 925-939. doi:10.1016/j.cell.2014.10.002
