Team:Lambert GA/Demonstrate
JUDGING FORM
MODELING
CALM ELECTROPEN
CALM
BACKGROUND DATA & METHODS RESULTS DISCUSSION COLOR Q SUPPLEMENTARY REFERENCES
QSYSTEM
COLOR Q CHROME Q REFERENCES
ELECTROPEN
PRINCIPLES & DESIGN MODEL & THEORY PRODUCT DESIGN REFERENCES
HUMAN PRACTICES
INTEGRATED HUMAN PRACTICES EDUCATION & ENGAGEMENT COLLABORATIONS
SAFETY
PARTS
BASIC PARTS COMPOSITE PARTS
PROJECT
DESCRIPTION BACKGROUND DESIGN EXPERIMENTS LAB NOTEBOOK INTERLAB RESULTS DEMONSTRATE THE VISION
TEAM
TEAM MEMBERS ATTRIBUTIONS
Your browser does not support video.
D E M O N S T R A T E
OVERVIEW
PROOF OF CONCEPT RESULTS
PORTABLE KIT
WATER COLLECTION
ELECTROPEN
BIOSENSOR CELLS
Overview
A critical component of characterization and comprehension of synthetic biology, software, and hardware tools exists in the form of mathematical modeling. Here we outline the models developed to facilitate the understanding of CALM (machine learning model) and ElectroPen (ultralow-cost synthetic biology tool), and assist in the prevention of cholera outbreaks. The model for CALM serves to develop a proactive response to potential outbreaks by utilizing predictive algorithms to identify at-risk areas and inform relevant organizations to mount a response. The model for the ElectroPen demonstrates the underlying principles allowing for the functionality of this ultralow-cost electroporator in terms of mechanical, electrical, and biological engineering. With the ElectroPen facilitating infield biosensor tests and CALM predicting cholera outbreaks, the described mathematical modeling serves to describe two key elements of CAPTIVATE and the mechanisms behind their performance.
Proof of Concept Results
In order to utilize LacZ color expression as a biosensor mechanism, Lambert iGEM obtained a LacZ toehold construct assembled with a T7 promoter from the Styczynski Lab at the Georgia Institute of Technology. When assembled with a distinct RNA sequence that is complementary to the trigger sequence, the LacZ operon is induced and subsequently breaks down Xgal and produces galactose and a blue pigment. This color can then be characterized by the shade of the blue pigment expression. This part is transformed in pSB3C5 instead of pSB1C3 because when LacZ is induced in a high copy plasmid such as pSB1C3, it drains the metabolism of a cell.
The construct above displays the proof of concept T7 LacZ switch and trigger. The switch consists of T7 promoter, a strong constitutive promoter, along with LacZ as the reporter gene. The trigger was cloned into a high copy plasmid while the switch was cloned into a low copy plasmid to ensure the replication of two different plasmids for the E. coli cell to produce proportional amounts.
Lambert iGEM used sequences and DNA shared from the Styczynski Lab at the Georgia Institute of Technology to build a T7, Toehold, LacZ switch to be applied as a biosensor. We tested the Biobricked T7 Toehold LacZ Switch BBa_K2550000, the Biobricked T7 Trigger Sequence BBa_K2550001 T7 Toehold LacZ Switch obtained from GATech, and the pSB6A1 T7 Trigger on Luria broth plates with chloramphenicol, carbenicillin, and Xgal antibiotic resistance.
Figure 1: (Top left) Biobricked T7 Toehold LacZ Switch BBa_K2550000  transformed on chloramphenicol, carbenicillin, and Xgal. (Top right) Biobricked Trigger Sequence BBa_K2550001 transformed on chloramphenicol antibiotic resistance. (Bottom left) pSB6A1 T7 Toehold LacZ Switch transformed on chloramphenicol, carbenicillin, and Xgal. (Bottom right) Dual plasmid with biobricked toehold sequence and trigger sequence.
The dual plasmid transformation using the pSB6A1 trigger and biobricked toehold was successful. Since the blue expression was only evident on the dual plasmid plate and not on the plate that contained the toehold, the trigger is successful in inducing LacZ gene expression.
The transformations of the biobricked toehold construct were unsuccessful because they produced a blue color. The blue might be a result of the orthogonal switch not binding as expected or a result of overexpression from a too strong promoter. Since the trigger sequence was not transformed with the construct, the colonies should appear white as the toehold only reveals the RBS and starting sequence of LacZ when the trigger sequence. This could be due to a leaky expression of LacZ under the strong T7 promoter. The dual plasmid transformation confirmed that neither the biobricked trigger nor the biobricked toehold sequence worked in conjunction with each other although they worked in conjunction with similar parts not in pSB1C3.
Figure 2: (Right) Dual Plasmid Transformation of T7 Trigger Sequence and T7 Toehold LacZ on chloramphenicol, carbenicillin, and Xgal antibiotic resistance. (Left) T7 Toehold LacZ transformation on chloramphenicol antibiotic resistance.  These show the expected results of our constructs.
Figure 3: Sequencing results of the trigger sequence that confirmed the part was correct.
We aligned the sequencing results to the original trigger sequence obtained from the second entry in the 144 first generation orthogonal toehold switches collection from the 2017 Green etal paper. Since the sequences matched after alignment, this confirmed that the trigger sequence was correctly cloned.
CAPTIVATE Kit
In recognition of the dire crisis cholera outbreaks have presented to the world, inflicting harm on millions and killing hundreds of thousands, we present a new solution to ensure a proactive response to cholera outbreaks can be mounted by targeting contamination within the water sources. CAPTIVATE constitutes a portable diagnostic kit  to test the water in local areas to determine whether Vibrio Cholerae is present or not. Cholera diagnosis requires This diagnostic kit includes Lambert iGEM’s  hardware components such as the 3-D Fuge, Chrome-Q, Chrome-Q Base, and the Electropen System. The kit also contains the wet-lab components including the Spira Swab (gauze water filter), freeze-dried Biosensor cells in culture tubes, and  a RNA extraction kit. The kit is capable of powering a phone for the Color-Q app and a cooling system needed for certain wet-lab parts of the project through a solar panel. CAPTIVATE portable diagnostic kit could work in conjunction of existing World Health Organization treatment kits which can provide a more efficient assessment to determine if    cholera exists in that water source.
Materials
Scoop (collection jar-disposable water bottle)
Muslin (Muslin will be placed in the 100 mL Nalgene bottle filled with diH2O: This diH2O can be obtained from boiling the water when sterilizing the muslin cloths).
100 ml Nalgene bottle with 2cm.hole at the bottom (will be wrapped around with aluminum foil to keep sterile)
Sterile 100 mL Nalgene bottle
Crochet Hook, or other straight stick that can be sterilized and reused
lysozyme in TE buffer
proteinase K
Rneasy Mini Kit
1M HCl
10 ml sterile distilled H2O (1ml needed for spin columns, and disposal of biosensor cells)
Electropen System (Cuvette and Electropen)
X-gal
Freeze-Dried Cells in a vial
LB in vials
Culture Tubes
Microcentrifuge Tubes
PCR Tubes
50 ml conical tubes
Lego spectrophotometer
1% sodium hydrochlorite
10% glycerol
Chrome-Q Dome & Chrome-Q base
Color-Q app and phone connected to the solar panel.
3-D fuge
pipette/micropipette
Water Collection
In order to use our portable detection kit, a filtration system is needed to isolate pathogenic V. cholerae from water sources. Lambert iGEM has developed a spira swab to carry out these functions at a lower cost. The spira swab consists of a filter packed plastic bottle with a 2-cm hole at the bottom as an outlet for the filtered water. To demonstrate the in-field application of this water filtration method, Lambert iGEM ran a series of experiments to test the efficiencies of different filters (muslin filter, coffee filter, cheese filter, and t-shirt cloth) to determine which would be the most effective filter. By using water from a nearby creek and E. coli liquid cultures, these samples were filtered using the spira swabs containing different filters, and the remaining filtered water samples were plated onto carbenicillin plates. The muslin filter produced the least amount of colony forming units in comparison to the other filters demonstrating its efficiency isolating V. cholerae. Because V. cholerae is larger than E.coli, the spira swab is able to allow V. cholerae to flow through while still letting the muslin filter trap unwanted sediments and bacteria validating the effectiveness of the spira swab in our detection kit.
Picture of a Lambert iGEM member demonstrating a water collection method.
Electropen
Following isolation of the DNA from potential Vibrio cholerae contaminants in the water, the DNA is electroporated into biosensor cells containing the toehold switch to identify the presence or absence of the cholera toxin. In order to conduct electroporation in the field, we developed a portable, 23-cent electroporator called the ElectroPen which weighs only 13g and does not require access to electricity. In order to demonstrate the functionality of the ElectroPen, we conducted proof-of-concept testing using a recombinant plasmid encoding the Green Fluorescent Protein (GFP) and validated our device’s functionality through collaborations with TAS_Taipei iGEM and UGA iGEM. Through UV imaging and plate reader data analysis, we confirmed that in each trial GFP was successfully transformed and expressed by E. coli, and testing was additionally conducted on 3 different cell lines: DH5a, BL21, and Nissile 1917, further verifying the ElectroPen’s performance. With testing of the ElectroPen in different regions across the world in different settings, we further demonstrate the application of this powerful tool in enabling infield electroporation of isolated DNA on E. coli.
Please refer to our Hardware Page for additional modeling and statistical analysis.
Biosensor Cells
Freeze Drying Protocol
These protocols are for using the Biosensor Cells in the field
Add 5 ml of sterile LB media and 5ul of chloramphenicol and ampicillin to a culture tube.
Obtain the desired plate. Take 1-3 colonies from the biosensor cells’ plate with a sterilized inoculating loop. Stir vigorously to ensure colonies are dispensed into solution.
Place culture tubes on a rack in the incubator. Set to shake and temperature to 36-38 C, and grow overnight.
Centrifuge liquid culture and discard the supernatant. Resuspend the pellet in 5ml of Microbial Freeze Drying Buffer with tryptic soy broth.
Aliquot 500ul of the suspension into sterile vials with stopper.
Turn on the lyophilizer and start the condenser. Set the shelf to 4°C.
Center the vials on the shelf. Either manually or with programmed controls, freeze the samples down to -40°C.  This should take about 30-60 minutes, but it is very dependent upon the lyophilizer. If the rate of freezing can be controlled, a practical rate is to drop the temperature by 1°C per minute. The samples should be visually frozen.
Let the samples sit at -40°C for 1 hour to complete freezing.
Turn on the vacuum pump.  Within 10-20 minutes, the vacuum should be under 200 millitorr (mtorr).
After the vacuum is below 200 mtorr, increase the temperature of the shelf for primary drying. The temperature can be up to -15°C. Let continue overnight.
For second drying, raise the shelf temperature to 20°C and dry for 2 hours.
With the stoppering mechanism, pit the stoppers on the vacuum. Turn off vacuum.
Store at 4°C in the dark.
In-Field Preparation of Cells
*Tubes should be on ice at all times unless centrifuging*
Resuspend in 5 ml of LB agar. Incubate for two hour.
Aliquot 1 ml of that liquid culture into 4 ml of sterile LB.
Shake in an incubator overnight. If no incubator present, let sit at room temperature for a day and a half.
Add the 5 ml liquid culture into 500 ml of sterile LB.
Using the portable spectrophotometer, check the optical density every hour. Continue growing until the OD is 0.6.
Pour cells into 50 ml conical tubes, on ice. Keep there for 30 minutes.
Pipet up the bottom 1.5 ml in the tube and transfer to a microcentrifuge tube.
With the 3D-fuge, centrifuge cells for 20 minutes, discard supernatant.
Add 1.5 ml of ice-cold sterile into the microcentrifuge to resuspend. Add into empty 50 ml conical tube.
Add another 11.5 ml of ice-cold sterile milliQ water to each conical tube, pipet up and down.
Combine samples into 2 total conical tubes.
Let sit on ice for 30 minutes. Take bottom 1.5 ml of each sample and transfer into a empty microcentrifuge tube.
With 3D-fuge, centrifuge for 20 minutes, discard supernatant, and repeat steps 9 and 10.
Let sit for 30 minutes on ice.
Transfer bottom 1.5 ml into a microcentrifuge tube. Centrifuge for 20 minutes, discard supernatant, and add 1.5 ml of ice-cold 10% glycerol.
Transfer the microcentrifuge into an empty conical tube. Add 23.5 ml of ice-cold 10% glycerol.
Let sit for 30 minutes on ice. Repeat steps 15 and 16.
Let sit for 30 minutes on ice. Take bottom 1.5 ml and transfer into a microcentrifuge tube. Centrifuge with 3D-fuge for 20 minutes.
Resuspend in 1.5 ml of ice-cold 10% glycerol. Transfer into conical tube and add 2.5 ml of ice-cold 10% glycerol.
Aliquot 1 ml into microcentrifuge tubes. Store in -80 freezer.
