Team:Amoy/Notebook/Protocol
Aomy/Project
TEAM
Member
Amoy
Attributions
PROJECT
Background
Description
Methods
Results
Discussion
Future Work
NOTEBOOK
Notebook
Protocols
Parts
Gallery
INTERLAB
NEWSLETTER
Introduction
Contribution
Discussion
Links
PRACTICES
Promotion
Talk
Communication
Collaborations
JUDGING
Medal Criteria
Acknowledgement
SAFETY
Notebook
Notebook
Protocols
Parts
Gallery
PROTOCOLS
Ⅰ. Molecular Cloning:
1. Antibiotics
Antibiotic
Stock Concentration(mg/mL)
Final Concentration(μg/mL)
Solvent
Ampicillin
50
50
Absolute ethyl alcohol
Chloramphenicol
50
100
ddH2O
2. Transformation
( 1 ) Add 10μL purified plasmid or ligation system into 50 μL fresh competent cells, which is contained in 1.5 mL centrifuge tube. Then mix gently. ( 2 ) Leave on ice for 30 minutes. ( 3 ) Heat shock at 42°C for 60 seconds. ( 4 ) Leave on ice for 2 minutes immediately. ( 5 ) Add 450 μL fresh LB medium into the centrifuge tube. ( 6 ) Incubate for 1 hours under the condition of 37°C, 200 rpm. ( 7 ) Spread 100 μL~150μL culture medium on a LB solid medium plate, which contains appropriate antibiotics. ( 8 ) Incubate overnight at 37°C (about 12~14 hours).
3. Plasmid Extraction (Omega Plasmid minipre Kit I)
( 1 ) Pellet 1.5mL bacteria in two clean 1.5 mL microcentrifuge tubes by centrifugation at 10 ,000 x g for 1 minute at room temperature.  Decant or aspirate medium anddiscard. ( 2 ) Resuspend the bacterial pellet by adding 250µL of Solution I/RNaseA and vortexing(or pipetting up and down). Complete resuspension (novisible cellclumps) of cell pellet is vital for obtaining good yields. ( 3 ) Add 250µL of Solution II and gently mixby inverting and rotating the tube several times to obtain a clear lysate. A 2 minutes incubation maybe necessary. Avoid vigorous mixing as this will shear chromosomal DNA and lower plasmid purity. ( 4 ) Add 350µL of Solution III and mix immediately by inverting the tube several times until a flocculent white precipitate forms. ( 5 ) Centrifuge at ≥13,000  x g for 10 minutes at room temperature. ( 6 ) Add the cleared supernatant by CAREFULLY aspirating it into a clean HiBind®MiniprepColumn (I) assembled in aprovided 2 mL collection tube. Centrifuge for 1 min at 10,000 x g at room temperature to completely pass lysate through the HiBind®Miniprep Column (I). ( 7 ) Discard flow-through liquid and re-use the 2 mL collection tube.Add 500µL of Buffer HB to wash the HiBind®Miniprep Column (I). Centrifuge for 1 min at 10,000x g at room temperature completely pass solution through the HiBind®Miniprep Column (I). ( 8 ) Discard flow-through liquid and  re-use the 2 mL collection tube. Add 700µL of DNA Wash Buffer diluted with absolute ethanol to wash the HiBind®Miniprep Column(I). Centrifuge for 1 minute  at  10,000  x g at room temperature to completely pass solution through the HiBind®Miniprep Column (I) . ( 9 ) Centrifuge the empty column for 2 minutes at 10,000 x g to dry the column matrix. ( 10 ) Place the column into a clean 1.5 mL microcentrifuge tube. Open the lid and put it in the electric dry oven to volatilize alcohol. ( 11 ) Add 60µL 65°C sterile deionized water directly onto the column matrix and let it sit at room temperature for 1-2 minutes. Centrifuge for 2 min at 13,000 x g to elute DNA. ( 12 ) Mark and store in fridge.
4. Digestion for Confirmation
Reaction condition: one and half hours under 37℃.
Components
Volume
Plasmid
7μL
Buffer
1μL
XbaI
1μL
PstI
1μL
5. DNA Gel Electrophoresis
Take 20mL system as example ( 1 ) Dilute 50×TAE to 1×TAE with ddH2O. ( 2 ) Measure 20 mL 1×TAE buffer. ( 3 ) Pour 1×TAE buffer into an erlenmeyer flask. ( 4 ) Weigh 0.2g agarose to make 1% DNA gel. ( 5 ) Pour agarose into the erlenmeyer flask. ( 6 ) Make agarose melt by microwave until completed. ( 7 ) Cool for 5 minutes. ( 8 ) Assemble gel pouring apparatus by inserting gate into slots. ( 9 ) Pour agarose gel into gel tray. ( 10 ) Bring a piece of silver paper on the top of the gel pouring apparatus. ( 11 ) Wait for 30-40 minutes to solidify the DNA agarose gel.
6. Double Enzyme Digestion
Take 100mL system as example, the plasmid concentration is 100ng/μL Quantitative relationship of plasmid and restriction enzyme is about: 1μL to 1μL
Components
Volume
Plasmid
50μL
Buffer
10μL
ddH2O
30μL
Enzyme I
5μL
Enzyme II
5μL
7. Ligation
Take 20μL system as an example and all the concentration of components are 20ng/μL The ratio of gene and backbone is about 3:1 Reaction condition: 8 hours under 16℃.
Components
Volume
Gene
12μL
Backbone
4μL
Buffer
2μL
T4 ligase
2μL
8. Gel Extraction of DNA (Omega Gel Extraction Kit)
( 1 ) Determine the appropriate volume of the gel slice by weighing it in a clean 1.5 mL microcentrifuge tube. Assuming a density of 1g/mL of gel, the volume of gel is derived as follows: a gel slice of mass 0.3g will have a volume of 0.3 mL. Add an equal volume of Binding Buffer (XP2) 1.5mL. Incubate the mixture at 50-55°C for 7 min or until the gel has completely melted. Mix by shaking or vortexing the tube in increments of 2-3 minutes. ( 2 ) Place a HiBind DNA Mini Column into the tube of the vacuum. Turn on vacuum.Add 700μL DNA/agarose solution to the HiBind DNA Mini Column by decanting or pipetting. After the liquid has passed through the column, switch off the vacuum source.Spin at 10000x g for 1minute. ( 3 ) Repeat step 2 with another 700μL of DNA/agarose solution. ( 4 ) Add 300μL of Binding Buffer (XP2)to the HiBind DNA Mini Column and turn on the vacuum. After the buffer has passed through the column, switch off the vacuum source. Spin at 10000x g for 1minute. ( 5 ) Add 700μL of SPW Wash Buffer to the HiBind DNA Mini column and turn on the vacuum source. After the buffer has passed through the column, switch off the vacuum source. Spin at 10000x g for 1minute. IMPORTANT: SPW Wash Buffer must be diluted with absolute ethanol before use. Refer to label for instructions. If refrigerated, SPW Wash Buffer must be brought to room temperature before use. ( 6 ) Repeat step 5 with another 700μL of SPW Wash Buffer. ( 7 ) Place the HiBind DNA Mini Column into a 2mL collection tube (provided) and centrifuge for 2 min at maximal speed ( ≥13,000 x g ) to dry the column matrix. ( 8 ) Place the HiBind DNA Mini Column into a clean 1.5 mL microcentrifuge tube. Depending on desire concentration of the final product, add 60μL water directly onto the column matrix and incubate at room temperature for 1 minute. Centrifuge for 1 min at maximum speed (≥13,000 x g) to elute DNA.
9. Cycle purity (Omega Cycle-Pure Kit)
( 1 ) Perform agarose gel/ethidium bromide electrophoresis to analyze PCR product. ( 2 ) Determine the volume of the PCR reaction, transfer the sample into a clean 1.5mL microcentrifuge tube, and add 4-5 volumes of Buffer CP. For PCR products <200bp add 6 volumes of Buffer CP. ( 3 ) Vortex thoroughly to mix. Briefly spin the tube to collect any drops from the inside of the lid. ( 4 ) Place a HiBind DNA Mini Column into a 2mL collection tube. ( 5 ) Apply the sample from Step 3 to the HiBind DNA column back into the same collection tube. ( 6 ) Discard liquid the place the HiBind DNA column back into the same collection tube. ( 7 ) Wash the HiBind DNA column by adding 700μL of DNA Wash Buffer diluted with absolute ethanol. Centrifuge at 10,000 x g for 1 min at room temperature. ( 8 ) Discard liquid and repeat step 7 by adding 700μL of DNA Wash Buffer diluted with absolute ethanol. ( 9 ) Discard liquid and centrifuge the empty HiBind DNA column for 2 min at maxi speed(≥13,000 x g) to dry the column matrix. ( 10 ) Place the column into a clean 1.5 mL microcentrifuge tube. Add 30μL of ddH20 (preheating to 65℃) directly onto the column matrix and let it sit at room temperature for 2mins. Centrifuge for 1 min at ≥13,000 x g to elute DNA.
10. PCR (Polymerase Chain Reaction)
PCR System
Template
1μL
Forward Primer
1.5μL
Reverse Primer
1.5μL
ddH2O
16μL
Prime Star
20μL
PCR ConditionPreheating:99℃
1
94℃
5min
2
94℃
30s
31 loops
3
52℃
30s
4
72℃
1min/kb
5
72℃
10min
6
15℃
10min
11. Sequence
Primer of leudh Prefix: GTTTCTTCGAATTCGCGGCCGCTTCTAGATGACATTGGAAATCTTC    46bp Tm=44 Suffix: GTTTCTTCCTGCAGCGGCCGCTACTAGTATTACCGGCGACTAAT     44bp
Gene sequence of leudh >LeuDH-A-2 ATGACATTGGAAATCTTCGAATATCTGGAAAAGTATGATTATGAACAGGTTGTGTTTTGTCAGGACAAAGAGAGCGG ACTGAAAGCAATTATCGCCATTCATGACACTACCCTGGGACCGGCTCTCGGCGGTACCCGCATGTGGACCTATGATT CAGAGGAAGCGGCAATCGAAGATGCACTCCGTCTGGCAAAAGGCATGACATATAAAAACGCAGCTGCTGGTTTAAAT CTGGGTGGCGCGAAAACCGTCATTATTGGCGATCCGCGTAAAGATAAATCAGAAGCGATGTTTCGTGCCCTGGGTCG CTACATCCAAGGGCTGAACGGGCGTTATATTACCGCGGAAGATGTAGGCACTACGGTTGACGATATGGATATCATTC ACGAGGAGACTGATTTTGTCACGGGCATCTCTCCGTCCTTCGGGAGCTCCGGAAATCCGTCTCCGGTTACCGCTTAT GGGGTTTACCGCGGGATGAAAGCGGCGGCGAAAGAGGCGTTTGGTACCGACAATTTAGAGGGCAAAGTGATTGCGGT GCAAGGTGTGGGCAATGTGGCCTATCACCTTTGCAAGCACCTGCACGCCGAGGGTGCAAAGTTAATTGTCACCGACA TCAATAAGGAGGCCGTCCAGCGTGCCGTAGAGGAGTTCGGTGCTACGGCGGTGGAGCCTAACGAAATTTATGGGGTG GAATGCGACATTTATGCGCCATGTGCACTCGGCGCAACGGTAAATGATGAAACCATCCCACAGCTTAAAGCGAAAGT GATTGCCGGCAGCGCTAATAATCAGCTTAAAGAAGATCGGCATGGCGATATCATTCATGAAATGGGCATCGTCTACG CCCCGGATTATGTTATTAATGCGGGTGGTGTTATTAACGTGGCGGATGAACTGTACGGCTATAACCGCGAACGTGCA CTGAAACGTGTGGAAAGCATTTACGACACCATTGCCAAGGTGATCGAAATCTCGAAACGGGATGGCATCGCCACGTA CGTAGCCGCGGATCGGTTGGCGGAAGAACGCATCGCCTCGTTGAAAAACAGTCGCTCGACGTATCTGCGCAACGGCC ATGACATCATTAGTCGCCGGtaa
Primer of fdh Prefix: GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAAAATTGTCCTGG     44bp      Tm=42 Suffix: GTTTCTTCCTGCAGCGGCCGCTACTAGTATTACTTTTTATCGTGTT    46bp
Gene sequence of fdh >FDH-B-2 ATGAAAATTGTCCTGGTCCTGTATGACGCGGGCAAACATGCGGCCGATGAGGAGAAACTTTACGGATCTACGGAAAA TAAACTGGGGATCGCCAATTGGCTGAAAGATCAGGGCCACGAACTGATCACCACAAGTGATAAAGAAGGGGAAACAA GCGAATTGGATAAGCATATTCCGGATGCAGATATCATTATTACTACGCCGTTTCATCCAGCATATATCACCAAAGAA CGCCTCGATAAAGCTAAGAACCTGAAGTTGGTGGTAGTCGCAGGGGTGGGGTCGGATCATATTGACCTGGATTACAT TAATCAGACCGGGAAAAAAATTTCTGTGTTAGAAGTTACCGGCAGTAATGTCGTTTCTGTGGCCGAACACGTGGTTA TGACCATGTTGGTTCTGGTGCGCAACTTTGTGCCAGCACATGAACAGATTATCAATCACGACTGGGAGGTTGCCGCG ATCGCAAAAGACGCCTACGATATCGAAGGAAAAACTATCGCTACTATCGGTGCGGGCCGCATCGGTTATCGTGTTTT GGAGCGTCTTCTGCCTTTTAACCCGAAAGAGCTCTTATATTACGATTATCAGGCCTTACCGAAAGAAGCGGAAGAGA AAGTAGGTGCGCGTCGTGTGGAAAATATCGAAGAATTAGTAGCGCAAGCAGATATCGTGACGGTGAACGCGCCTCTC CATGCCGGTACGAAAGGCCTGATTAATAAGGAACTCCTGTCCAAATTCAAAAAAGGTGCGTGGCTTGTGAATACCGC TCGCGGTGCGATTTGCGTCGCTGAAGACGTGGCGGCAGCGCTGGAGAGCGGCCAACTTCGCGGTTATGGCGGTGACG TATGGTTTCCGCAGCCGGCTCCGAAAGACCACCCATGGCGCGACATGCGTAACAAATATGGCGCGGGCAACGCCATG ACCCCGCATTATTCGGGTACCACCCTGGATGCCCAAACCCGGTACGCAGAGGGCACCAAGAATATTCTGGAGTCATT TTTCACGGGCAAATTCGATTATCGGCCGCAGGATATTATTCTGTTGAACGGAGAGTATGTTACGAAGGCCTATGGCA AACACGATAAAAAGtaa
Ⅱ. Characterization:
1. Cultivation and induction
( 1 ) Put the tube with competent cell on the ice. Add 10μL plasmid which was sequenced. ( 2 ) Put the tube back to the ice for 30 minutes after mixing. ( 3 ) Put the bacterial cultures in the 42℃ water for 42 seconds. Then put it in the ice for 2 minutes quickly. ( 4 ) 450mL fresh liquid medium were added. Grown at 37℃，200rmp for 1 hour. ( 5 ) 100μL bacterial cultures were maintained on the agar plate with antibiotic at 37℃ for 12 hours. ( 6 ) The single colonies were picked. Add 10mL liquid mediate with antibiotic. Grown at 37℃， 200rmp for 12 hours.Growth of promising bacterial strain: Cell concentration was determined by measuring the optical density. ( 7 ) Add 10μL，30μL，50μL，70μL，100μL 0.1M IPTG separately to the medium. Incubated at 18 ℃, shaken at 200rmp overnight.
2. Preparation of the cell-free extract
( 1 ) The enzyme expression was induced with the most appropriate concentration of isopropylthiogalactoside at 18°C for 3 h. Cells were centrifuged at 8000×g for 5min. ( 2 ) The column was washed two times with 10 mL of PBS buffer. ( 3 ) The cell pellet was suspended with 7.5 mL of PBS buffer and the cells were broken using Homogenizer. ( 4 ) Ultrasonic breaking cell: power 200w, time 5s, interval 2s, frequency 10. ( 5 ) The cell-free extract : The broken cells were centrifuged at 8000×g for 15min. Supernatant is the cell-free extract.
3. Enzyme and protein assays
In the 96 hole plate, each sample was measured in two parallel samples, each with 220μL of the enzyme reaction system, and the 20μL was added to the reaction system. Set the wavelength of 340nm Eliasa, each 10s reading time, reading 15 points, calculate the absorbance rate of change △A with initial velocity method. When the condition is certain, the amount of enzyme required for catalytic production or consumption of 1 mol NADH per minute is defined as an enzyme activity unit. Calculation of enzyme activity:
K：Sample dilution multiple VT: Total volume (μL) VS: Sample volume (μL) L = Ratio of color cup light（d=0.5cm） ε = molar extinction coefficient(ε=6.22 ) The reaction system of leucine dehydrogenase was: 180μL 1mol/L NH4Cl buffer，10μL 0.5M TMA，10μL 4.4 mM NADH，sample 20μL. The reaction system of formate dehydrogenase was: 180μL 1mol/L PBS buffer，10μL 0.5M Ammonium formate，10μL 4.4 mM NAD+，sample 20μL.
4. Protein SDS-PAGE gel electrophoresis
1. Sampling 160μL in 1.5mL centrifugal tube, adding 40μL 6x protein buffer , boiling water bath 5min, take 20μL as sample. 2. Electrophoresis conditions: 80V, 30min, 120V, 60-80min. 3. Coomassie brilliant blue dye staining at room temperature: 50rmp, 120min. 4. Decolorization: adding Coomassie blue 50rmp, decolorization decolorization liquid, for the night.
5. Catalysis
1. activation radioactivation:500μL fresh liquid bacteria medium were added into 50mL LB medium. Grown at 37℃，200rmp for 12 hours. 2. transfer: 1%-2% was inoculated in 1000mL LB medium (containing 50 μg/mL antibiotic), Grown at 37℃，200rmp for 3h, and the OD was about 0.6. 3. the most suitable concentration of IPTG was added to the culture medium, and induced at 18℃，200rmp for 12h. 4. collect the bacteria, In -80 degree refrigerator preservation. 5. Add the 10mL 1M NH3OH –NH4Cl buffer, smudge cells，add  Trimethylpyruvic acid  (50mM), ammonium formate ，NADH，constant volume to 15mL,  Sampling (500mL) with 1, 2, 4, 6, 8, 10, 12, 24, 48,72 ,96h, In -20 degree refrigerator preservation
6. HPLC analysis of L-tert-leucine
1. The melt cells were centrifuged at 13000×g for 10min. 2. take the supernatant of 200μl joined the 1.5mL EP tube, and add 800μl deionized water. 3. The liquid is pumped out through a syringe of 0.22 m, and is added to the glass bottle, which is detected in the liquid phase detector.
CONTACT US
Email: igemxmu@gmail.com
Website: 2015.igem.org/Team:Amoy
Address: Xiamen University, No. 422, Siming South Road, Xiamen, Fujian, P.R.China 361005
