Team:Munich/Application
-_-
Project
Description
Design
Demonstrate
Measurement
Applied Design
Final Results
Achievements
Wetlab
Protocols
Notebook
Safety
Parts
Improved Part
Part Collection
InterLab
Model/Software
Model
Software
Collaboration
Hardware
Detector
Quake Valve
Paperstrips
Sample Processing
Human Practices
Silver
Gold
Engagement
Collaborations
Team
Team members
Sponsors
Attributions
Gallery
Description
Thanks to advances in molecular biology and biochemistry, scientists have been able to consistently detect lower and lower concentration of molecules 1 , to the point that single molecules can be reliably recognized with methods such as polymerase chain reaction (PCR) 2 , fluorescence in situ hybridization (FISH) 3 and enzyme-linked immunosorbent assays (ELISA) 4 . This has opened doors for synthetic biology to create better and more accurate diagnostic tests that use biomarkers like nucleic acids and proteins as targets 5 , 6 . Through such advances, the field of molecular diagnostics developed. Unfortunately, current standard methods require expensive equipment or trained personnel, which generally limits their usability to hospitals or laboratories. Recently, there has been a push to develop new tests that fuse the reliability of standard methods with affordable platforms such as lab-on-a-chip or paper strips  to overcome this restrictions 7-9 . We wanted to help close this gap and set out to engineer a diagnosis principle for the detection of a wide array of targets that could be used without difficult-to-meet technical requirements.
CascAID
Our project, which we named Cas13a controlled assay for infectious diseases (CascAID), features the recently identified CRISPR/Cas effector Cas13a 10 . Unlike other proteins in the familiy, Cas13a has the unique ability to bind and cleave specific RNA targets rather than DNA ones.  Moreover, after cleaving its target, Cas13a is able to unspecifically cleave RNA molecules. By using this collateral activity from Cas13a, our system is capable of detecting virtually any RNA target. This is done by changing the crRNA in the protein, that is a short RNA sequence that determines what is recognized as target.
Cas13a binds specific target RNA depending on the crRNA sequence. After activation, Cas13a cleaves RNA indiscriminately.
We wanted to start our project by showing that Cas13a's collateral activity could be used to detect the presence of specific RNA. For this, we used the RNAse alert system, as done in a recent publication 11 , to detect RNA digestion. In this assay, the presence of RNAse-like activity is detected by an increase in green fluorescence. Our experiments yielded a convincing proof-of-principle which we went on to model to determine the theoretical detection limit of our system . Moreover, CascAID can be used to detect a wide spectrum of pathogens, as our experiments with gram-positive and viral targets suggested.
Cas13a can be used to detect specific RNA sequences.
Picture of the Thermocycler
For RNA extraction from the samples we tested three methods: extraction with silica beads, extraction with silica membrane and heat lysis. We custom-built an affordable thermocycler for signal amplification by RT-PCR to improve the detection limit. We explored recombinase polymerase amplification (RPA), an isothermal amplification procedure, to use over more conventional PCR methods as its simplicity makes it the more attractive option.
Colorimetric read-outs
To couple CascAID with an easy read-out method we explored three colorimetric read-outs:
AeBlue: The RNA strand in a specially designed RNA/DNA dimer is cut by Cas13a's collateral activity. After digestion, the interaction between the two strands is too weak to hold the dimer and it decays. We can then use the DNA-strand as template to translate the chromoprotein aeBlue .
Intein-Extein: By binding TEV-protease with a RNA-linker we can use Cas13a's collateral activity to regulate the protease's diffusion and use it to cleave a TEV tag separating the intein regions of a modified chromophore. After the first cleavage, the intein segment excises itself 13 , bringing together the halves of the chromophore. Only then is the chromophore functional and produces the colorimetric read-out.
Gold nanoparticles: Other than in the other two colorimetric readouts, aeBlue and Intein-Extein, the only protein involved in the gold nanoparticle (AuNP)-readout is Cas13a, like in our RNase Alert readout. This reduces the necessary fine tuning of the biochemical circuit to a minimum, favoring high robustness of the readout. Due to the phenomenon of Localized Surface Plasmon Resonance, AuNPs appear in a distinct color, ranging from intense red to blue, black and colorless. This property depends on particle size, shape, the immediate environment, and -most critical for our purpose- aggregation state 14 .
In our project we use AuNPs with a diameter of roughly 10 nm, giving them a bright red color in solution. Their small size and therefore high surface-to-volume ratio makes them ideal for functionalization with thiolated compounds, forming covalent Au-S bonds. The first step of our concept is to use these properties to functionalize AuNPs with either 5’- or 3’- thiolated DNA and, through addition of linker- RNA which hybridizes with both thiolated DNA strands, form aggregates, changing the color from red to blue. The design of the linker-RNA includes an uracil-rich, single-stranded segment between the DNA-complementary termini, making it prone to Cas13a-mediated promiscuous cleavage.
It has been shown that, for purely DNA-based hybridization, AuNP aggregates can be spotted on filter paper, dried and severed by addition of a nuclease-containing solution, visible through diffusion of red AuNPs on the paper. Thus, the second part of our concept is to spot RNA-linked AuNPs on paper, dry them alongside the Cas13a mixture and detect specific target RNAs and resulting Cas13a activity with a simple change from blue to red.
Software
To help facilitate the design of crRNA, the sequences that give CascAID its specificity, we developed a software tool that checks crRNA for unwanted secondary structures. This gives valuable insight on whether the sequence is suited to use with Cas13a or whether some modifications are needed. Together with Team Delft's software tool which designs the corresponding crRNA based on the target, we collaborated to develop a powerful tool that suggests crRNA sequences and checks their usability
References
Cohen, Limor, and David R. Walt. "Single-Molecule Arrays for Protein and Nucleic Acid Analysis." Annual Review of Analytical Chemistry 0 (2017).
Nakano, Michihiko, et al. "Single-molecule PCR using water-in-oil emulsion." Journal of biotechnology 102.2 (2003): 117-124.
Taniguchi, Yuichi, et al. "Quantifying E. coli proteome and transcriptome with single-molecule sensitivity in single cells." science 329.5991 (2010): 533-538.
Rissin, David M., et al. "Single-molecule enzyme-linked immunosorbent assay detects serum proteins at subfemtomolar concentrations." Nature biotechnology 28.6 (2010): 595-599.
Pardee, Keith, et al. "Rapid, low-cost detection of Zika virus using programmable biomolecular components." Cell 165.5 (2016): 1255-1266.
Slomovic, Shimyn, Keith Pardee, and James J. Collins. "Synthetic biology devices for in vitro and in vivo diagnostics." Proceedings of the National Academy of Sciences 112.47 (2015): 14429-14435.
Tang, Ruihua, et al. "A fully disposable and integrated paper-based device for nucleic acid extraction, amplification and detection." Lab on a Chip 17.7 (2017): 1270-1279.
Vashist, Sandeep Kumar, et al. "Emerging technologies for next-generation point-of-care testing." Trends in biotechnology 33.11 (2015): 692-705.
Gubala, Vladimir, et al. "Point of care diagnostics: status and future." Analytical chemistry 84.2 (2011): 487-515.
Abudayyeh, Omar O., et al. "C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector." Science 353.6299 (2016): aaf5573.
Gootenberg, Jonathan S., et al. "Nucleic acid detection with CRISPR-Cas13a/C2c2." Science (2017): eaam9321.
https://www.idtdna.com/pages/docs/technical-reports/in_vitro_nuclease_detectionD325FDB69855.pdf (retrieved: 13.10.17)
Anraku, Yasuhiro, Ryuta Mizutani, and Yoshinori Satow. "Protein splicing: its discovery and structural insight into novel chemical mechanisms." IUBMB life 57.8 (2005): 563-574.
Link, Stephan, and Mostafa A. El-Sayed. "Size and temperature dependence of the plasmon absorption of colloidal gold nanoparticles." The Journal of Physical Chemistry B 103.21 (1999): 4212-4217.
Zhao, W., Ali, M.M., Aguirre, S.D., Brook, M.A., and Li, Y. (2008). "Paper-based bioassays using gold nanoparticle colorimetric probes." Analytical Chemistry 80, 8431–8437.
