<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Munich_Targets skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Munich/Targets</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><B style=" color: #ffffff !important">-_-</B></P></DIV><DIV class="dropdown"><BUTTON class="dropbtnProject dropbtn"><A href="/Team:Munich/Description"><P><B>Project</B></P></A></BUTTON><DIV class="dropdown-contentDescription dropdown-content"><A href="/Team:Munich/Description"><P><B>Description</B></P></A></DIV><DIV class="dropdown-contentDesign dropdown-content"><A href="/Team:Munich/Design"><P><B>Design</B></P></A></DIV><DIV class="dropdown-contentDemonstration dropdown-content"><A href="/Team:Munich/Demonstrate"><P><B>Demonstrate</B></P></A></DIV><DIV class="dropdown-contentMeasurement dropdown-content"><A href="/Team:Munich/Measurement"><P><B>Measurement</B></P></A></DIV><DIV class="dropdown-contentApplication dropdown-content"><A href="/Team:Munich/Applied_Design"><P><B>Applied Design</B></P></A></DIV><DIV class="dropdown-contentFinalResults dropdown-content"><A href="/Team:Munich/Results"><P><B>Final Results</B></P></A></DIV><DIV class="dropdown-contentAchievements dropdown-content"><A href="/Team:Munich/Achievements"><P><B>Achievements</B></P></A></DIV></DIV><DIV class="dropdown" valign="center"><BUTTON class="dropbtnWetlab dropbtn" valign="center"><A href="/Team:Munich/Protocols"><P><B>Wetlab</B></P></A></BUTTON><DIV class="dropdown-contentProtocolsMethods dropdown-content"><A href="/Team:Munich/Protocols"><P><B>Protocols</B></P></A></DIV><DIV class="dropdown-contentLabJournal dropdown-content"><A href="/Team:Munich/Notebook"><P><B>Notebook</B></P></A></DIV><DIV class="dropdown-contentSafety dropdown-content"><A href="/Team:Munich/Safety"><P><B>Safety</B></P></A></DIV><DIV class="dropdown-contentParts dropdown-content"><A href="/Team:Munich/Parts"><P><B>Parts</B></P></A></DIV><DIV class="dropdown-contentImprove dropdown-content"><A href="/Team:Munich/Improve"><P><B>Improved Part</B></P></A></DIV><DIV class="dropdown-contentCollection dropdown-content"><A href="/Team:Munich/Part_Collection"><P><B>Part Collection</B></P></A></DIV><DIV class="dropdown-contentInterlab dropdown-content"><A href="/Team:Munich/InterLab"><P><B>InterLab</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnModel dropbtn"><A href="/Team:Munich/Model"><P><B>Model/Software</B></P></A></BUTTON><DIV class="dropdown-contentModel dropdown-content"><A href="/Team:Munich/Model"><P><B>Model</B></P></A></DIV><DIV class="dropdown-contentSoftware dropdown-content"><A href="/Team:Munich/Software"><P><B>Software</B></P></A></DIV><DIV class="dropdown-contentSoftwareCollaborations dropdown-content"><A href="/Team:Munich/Software/Collaborations"><P><B>Collaboration</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnSoftHardware dropbtn"><A href="/Team:Munich/Hardware"><P><B>Hardware</B></P></A></BUTTON><DIV class="dropdown-contentDetector dropdown-content"><A href="/Team:Munich/Hardware/Detector"><P><B>Detector</B></P></A></DIV><DIV class="dropdown-contentQuakeValve dropdown-content"><A href="/Team:Munich/Hardware/QuakeValve"><P><B>Quake Valve</B></P></A></DIV><DIV class="dropdown-contentPaperstrips dropdown-content"><A href="/Team:Munich/Hardware/Paperstrip"><P><B>Paperstrips</B></P></A></DIV><DIV class="dropdown-contentPCRChip dropdown-content"><A href="/Team:Munich/Hardware/SampleProcessing"><P><B>Sample Processing</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnHP dropbtn"><A href="/Team:Munich/HP/Silver"><P><B>Human Practices</B></P></A></BUTTON><DIV class="dropdown-contentHPSilver dropdown-content"><A href="/Team:Munich/HP/Silver"><P><B>Silver</B></P></A></DIV><DIV class="dropdown-contentIHPgold dropdown-content"><A href="/Team:Munich/HP/Gold_Integrated"><P><B>Gold</B></P></A></DIV><DIV class="dropdown-contentPublicEngagement dropdown-content"><A href="/Team:Munich/Engagement"><P><B>Engagement</B></P></A></DIV><DIV class="dropdown-contentCollaborations dropdown-content"><A href="/Team:Munich/Collaborations"><P><B>Collaborations</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnTeam dropbtn"><A href="/Team:Munich/Team"><P><B>Team</B></P></A></BUTTON><DIV class="dropdown-contentTeamMembers dropdown-content"><A href="/Team:Munich/Team"><P><B>Team members</B></P></A></DIV><DIV class="dropdown-contentSponsors dropdown-content"><A href="/Team:Munich/Sponsors"><P><B>Sponsors</B></P></A></DIV><DIV class="dropdown-contentAttributions dropdown-content"><A href="/Team:Munich/Attributions"><P><B>Attributions</B></P></A></DIV><DIV class="dropdown-contentGallery dropdown-content"><A href="/Team:Munich/Gallery"><P><B>Gallery</B></P></A></DIV></DIV><TABLE width="100%" height="100%" cellpadding="0" cellspacing="0" border="0"><TBODY><TR><TD style="background-color: #51a7f9;" colspan="6" align="left"><UL class="menuList" id="menu"><LI><A href="/Team:Munich/Results">Overview</A></LI><LI><A href="/Team:Munich/Cas13a">Cas13a</A></LI><LI><A href="/Team:Munich/Readouts">Readouts</A></LI><LI><A href="/Team:Munich/Targets">Targets</A></LI><LI><A href="/Team:Munich/DetectionOnChip">Detection Chip</A></LI><LI><A href="/Team:Munich/Amplification">Amplification</A></LI></UL></TD></TR><TR><TD colspan="6" align="left" valign="center"><DIV style="margin-top: 40px"><FONT size="7" color="#51a7f9"><B style="color: #51a7f9">Results: Targets</B></FONT></DIV></TD></TR><TR><TD colspan="6"><H3>What worked:</H3><UL class="listResults"><LI>We tested targets from <A class="myLink" href="https://2017.igem.org/Team:Munich/Targets#commonpathogens">common pathogens</A> and showed the <A class="myLink" href="https://2017.igem.org/Team:Munich/Targets#orthog">orthogonality</A> of virus detection versus bacterial detection. </LI></UL></TD></TR><TR><TD colspan="6"><H3>What presented issues:</H3><UL class="listResults"><LI>Detecting Qbeta RNA.</LI></UL></TD></TR><TR><TD colspan="6" align="left"><H3>Modular Design for crRNA</H3><P>Our strategy to create crRNAs, which bind both the Cas13a protein with organism specificity (crRNA Lbu does not bind Cas13a Lsh) and the target sequence with point-mutation sensitivity, is designed to allow easy prototyping of target binding sequences. Our crRNA template for transcription consists of two DNA strands sharing complementary regions that can therefore be amplified using PCR into a full double-stranded template. The non-template (NT) strand contains the T7 promoter site and the scaffold region of the transcript that will specifically bind the Cas13a. The template strand (T) contains part of the scaffold region, so that it is complementary to the NT strand, and the target binding region. In a one-batch reaction using the Klenow DNA polymerase and the T7 RNA polymerase, the two strands bind in the scaffold region, which serves as a primer for Klenow, they get amplified into a complete double stranded DNA template, and finally transcribed into our crRNA <B>(Figure 1)</B>. This modular design allows a fast and cost-effective creation of crRNAs since only the pathogen specific T strand has to be newly designed and synthesized as a oligo to accommodate for new targets. This prototyping gives the opportunity to screen all kind of pathogens with our system easily. This can be combined with our <A class="myLink" href="/Team:Munich/Software">software</A> for target screening, which can be used to verify that the target strand design does not lead to off target binding in human transcriptome and microbiome. 
                </P><DIV class="captionPicture"><P><B>Figure 1:</B> Scheme of the crRNA design, where the template sequence containing the target binding region can be easily exchanged</P></DIV></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>Variety of Detectable Targets</H3><P>Before demonstrating if we could discriminate bacterial and viral sequences, we needed to determine if our detection circuit worked across pathogens. <I>Escherichia coli</I> 16S rRNA had been our first target, as Dr. Pardee had advised us to use the most simple and accessible  RNA sequence. For other targets, we were advised different pathogens through our series of <A class="myLink" href="/Team:Munich/HP/Gold_Integrated">interviews with experts</A>, but we could not easily access most of the pathogens due to safety restrictions in our lab. We therefore looked at <I>B. subtilis</I> as a second bacterial target that shares sequence homology with <I>E. coli</I>, and we used <I>in vitro</I> transcribed RNA for viruses (Norovirus, Hepatitis C, and Qbeta). We describe quickly the sequences chosen and then present our detection results.</P><H4>Noro virus</H4><P>  
Noro virus originally called Norwalk virus, of the family <I>Caliciviridae</I>, is one of the major cause of viral gastroenteritis in humans and it affects patients of all age groups. It is also associated with hospital infections, therefore it is a highly relevant pathogen to test and detect. For our experiments, we took the 5’ UTR of the Noro virus and did <I>in vitro</I> transcription to get the target RNA and the crRNA.  The 5’ UTR of the viruses are very specific to each individual virus so this should be an ideal sequence to specifically detect and differentiate viral RNAs. 
</P><H4>Hepatitis C virus</H4><P>  
HCV is a small single stranded RNA virus of family <I>Flaviviridae</I> which is the major cause of the Hepatitis C and liver cancer. Common setting for transmission of HCV is also intra-hospital (nosocomial) transmission, when practices of hygiene and sterilization are not correctly followed in the clinic. It is estimated that 71 million people world-wide have chronic HCV infection, and there are no known vaccines<SUP><A class="myLink" href="#ref_1">1</A></SUP>. For our experiments, we took the 5’ UTR of the HCV virus and did <I> in vitro </I> transcription to get the target RNA and the crRNA.</P><H4><I>Qbeta</I></H4><P>
The Enterobacteria phage Qbeta belongs to the family of <I>Leviviridae</I> and is one of the most commonly used bacteriophage in research fields. It is an icosahedral virus which uses <I>E. coli</I> as its host. We designed crRNA for the part of this viral sequence and tested it with our Cas13a detection system, which however did not work that well. 
</P><H4><I>Bacillus subtilis</I></H4><P>  
We also focused on trying out our experiment with other bacterial RNAs and for this we chose the gram<SUP>+</SUP><I>B. subtilis</I> since it is widely used in microbiological research.  Plus, we wanted to see if one can detect the difference between the 16s rRNAs of <I>B. subtilis</I> and <I>E. coli</I>. For <I>B. subtilis</I>, we did not perform any <I>in vitro</I> transcription, rather we directly used the bacterial culture for the RNA extraction. However, we did encounter some problems due to possible RNase contamination from <I>in vivo</I> extracted RNA. </P><DIV class="captionPicture"><P><B>Figure 2:</B> The Urea PAGE shows the <I>in vitro</I> transcribed Lbu crRNAs for all detected targets. crRNAs for Lsh, Lwa and Lbu as well as target RNAs were treated the same way. 
</P></DIV></TD></TR><TR><TD class="verticalColumn" colspan="3" align="center" valign="center"><H3 id="commonpathogens">Detection</H3><P>  
When we combined the target RNA from different pathogens and their matching crRNA, we systematically saw an increase of cleaved RNase Alert compared to our controls without targets <B>(Figure 3)</B>. It is worth noting that <I>B. subtilis</I> showed the worst on/off ratio for activation, which we assume to be caused by contamination from the <I>in vivo</I> sample treatment. The Cas13a detection circuit is effective for RNA sequences from different viruses, gram<SUP>+</SUP> and gram<SUP>-</SUP> bacteria, proving its universality. </P></TD><TD colspan="3" align="center" valing="center"><DIV class="captionPicture"><P><B>Figure 3:</B> Detection of bacterial and viral target sequences with their matching crRNA. Concentrations of target when added range from 10nM to 25nM. 
</P></DIV></TD></TR><TR><TD colspan="3" align="center" valing="center"><DIV class="captionPicture"><P><B>Figure 4:</B> Cross-talk between Norovirus crRNA and different targets (indicated on the x-axis). Inset shows time traces of the experiment. 
</P></DIV></TD><TD class="verticalColumn" colspan="3" align="center" valign="center"><H3 id="orthog">Orthogonality of Detection</H3><P>  
We then set out to differentiate viral sequences from bacterial sequences. Assuming the risk of sample contamination from the patient’s own bacteria is high, we decided to first use the crRNA from a virus to screen against different targets. We chose the Norovirus crRNA and looked at the activation of Cas13a under the presence of 30 nM of Norovirus, <I>E. coli</I> or HCV as targets <B>(Figure 4)</B>. We found that only the Norovirus target lead to a great increase in RNase Alert cleavage, and that our detection mechanism is therefore highly orthogonal and specific.
</P><DIV class="captionPicture"><P><B>Figure 5:</B> Scheme representing our crosstalk assays.
</P></DIV></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>CascAID<SUP>+</SUP> Software</H3><P>  
CascAID<SUP>+</SUP> is a potentially universal tool for nucleic acid detection. Fast adaptation of our platform to new targets requires <I>in silico</I> analysis of the crRNA design. 
Therefore, we developed a software that verifies the presence of the right secondary structure and the absence of any off-target effects. This is realised by using established programs  for secondary structure prediction, NUPACK and Mfold, as well as the BLAST program for local alignments. The secondary structure prediction of the designed crRNA is compared to structures that had already been shown to work experimentally, either in the literature or by us. This allows to rule out misfolded crRNA prior to experiment. Furthermore, we constructed a reference transcriptome databank consisting of data from organisms likely to be found in human mucus samples. We then used the BLAST algorithm to search for sequences similar to the designed target sequence in the reference transcriptome to determine possible off-target effects. This rational approach of crRNA cerification allowed us to  save development time and costs during the design of the crRNA for the targets described here.
</P></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>Reproducibility</H3><P>  
The <I>in vitro</I> transcription, purification and quantification of crRNA and target RNA is a highly reproducible method. This procedure is standard to our host lab (Chair for the Physics of Synthetic Biological Systems, TUM) and was used by many of our team members. We can assess that the RNAs were correctly quantified with the reproducibility of the Cas13a detection, that systematically gave us the same response range. We did not have a chance to reproduce systematically our orthogonality tests, but as the sequence specificity of Cas13a was shown in other papers<SUP><A class="myLink" href="#ref_2">2</A></SUP>, we are confident that we can reproduce those results.</P></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>Discussion and Conclusion</H3><P>  
We found that it was quite simple to prototype a variety of targets with our crRNA template system, where the target binding region could be easily exchanged with a new primer. We found that in most cases, the Cas13a detection circuit worked reliably. However, it should be noted that we were unsuccessful at detecting the virus Q5 beta (data not shown), and that <I>in vivo</I>-extracted targets still bear <A class="myLink" href="/Team:Munich/Cas13a">the risk of contamination</A>. We did not experimentally test the point-mutation sensitivity of Cas13a (as this was characterized elsewhere<SUP><A class="myLink" href="#ref_2">2</A></SUP>), nor did we optimise the target sequence within one organism. However, we developed a <A href="/Team:Munich/Software" class="myLink">software</A> that can verify the secondary structure of the crRNA and screen cross-talk between targets and crRNA, so that sequence optimisation can be done more systematically. What we achieved here is a proof-of-principle that our discrimination between viral and bacterial sequences is efficient and simple to implement. We are confident that with the right tools for sequence-optimisation, a general scheme for discriminating between the most common bacterial and viral infections can be built. We have already shown that our readout circuit can be used to detect Norovirus without false positives from other common pathogens.</P></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>References</H3><P><OL style="text-align: left"><LI id="ref_1">World Health Organization: http://www.who.int/mediacentre/factsheets/fs164/en/</LI><LI id="ref_2">Abudayyeh, O. O., Gootenberg, J. S., Konermann, S., Joung, J., Slaymaker, I. M., Cox, D. B., ... &amp; Severinov, K. (2016). C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector. Science, 353(6299), aaf5573.</LI></OL></P></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></BODY></HTML>